<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_Hannover_Description skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><DIV id="top_title"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:Hannover/Description</SPAN></H1></DIV><DIV id="HQ_page"><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><DIV class="full_size container-fluid carousel_div"><DIV class="carousel_container"><DIV id="carousel_myCarousel" class="carousel_myCarousel carousel slide" data-ride="carousel"><A class="left carousel-control" href="#carousel_myCarousel" role="button" data-slide="prev"><SPAN class="sr-only">Previous</SPAN></A><A class="right carousel-control" href="#carousel_myCarousel" role="button" data-slide="next"><SPAN class="sr-only">Next</SPAN></A></DIV></DIV></DIV><NAV class="navbar navbar-inverse menu_wrapper full_size" data-spy="affix" data-offset-top="197" style="z-index: 99; width:100%"><DIV class="container-fluid"><UL id="accordion" class="nav navbar-nav accordion"><LI class="menu_item"><A href="https://2016.igem.org/Team:Hannover">HOME </A></LI><LI class="dropdown menu_item"><A class="dropdown-toggle" data-toggle="dropdown" href="#">TEAM </A></LI><LI><A href="https://2016.igem.org/Team:Hannover/Team"> Team </A></LI><LI><A href="https://2016.igem.org/Team:Hannover/Team#sponsors"> Sponsors </A></LI><LI class="menu_item"><A href="https://2016.igem.org/Team:Hannover/Attributions">Attributions</A></LI></UL><LI class="dropdown menu_item"><A class="dropdown-toggle" data-toggle="dropdown" href="#">PROJECT</A></LI><LI><A href="https://2016.igem.org/Team:Hannover/Description">TALebots </A></LI><LI><A href="https://2016.igem.org/Team:Hannover/Software">Software </A></LI><LI><A href="https://2016.igem.org/wiki/Team:Hannover/InterLab">Interlab Study</A></LI><LI><A href="https://2016.igem.org/Team:Hannover/Notebook"> NOTEBOOK </A></LI><LI class="dropdown menu_item"><A class="dropdown-toggle" data-toggle="dropdown" href="#">PARTS</A></LI><LI><A href="https://2016.igem.org/Team:Hannover/Parts">Parts </A></LI><LI><A href="https://2016.igem.org/Team:Hannover/Basic_Part"> Basic Parts </A></LI><LI><A href="https://2016.igem.org/Team:Hannover/Composite_Part"> Composite Parts </A></LI><LI class="menu_item"><A href="https://2016.igem.org/Team:Hannover/Safety"> SAFETY </A></LI><LI class="dropdown menu_item"><A class="dropdown-toggle" data-toggle="dropdown" href="#">HUMAN PRACTICES</A></LI><LI><A href="https://2016.igem.org/Team:Hannover/HP/Silver">Engagement </A></LI><LI><A href="https://2016.igem.org/Team:Hannover/HP/Gold">Survey </A></LI><LI><A href="https://2016.igem.org/Team:Hannover/Collaborations">iGEM COMMUNITY </A></LI><LI class="dropdown menu_item"><A class="dropdown-toggle" data-toggle="dropdown" href="https://2016.igem.org/Team:Hannover/Achievements">ACHIEVEMENTS</A></LI><LI><A href="https://2016.igem.org/Team:Hannover/Achievements#Bronze"> Bronze </A></LI><LI><A href="https://2016.igem.org/Team:Hannover/Achievements#Silver"> Silver </A></LI><LI><A href="https://2016.igem.org/Team:Hannover/Achievements#Gold"> Gold </A></LI></DIV></NAV><DIV class="container-fluid text-center"><DIV class="row content" style="background-color:rgb(0, 107, 168);"><DIV class="col-sm-2 sidenav"><DIV class="panel-group" style="text-align: left;"><DIV class="panel panel-default"><DIV class="panel-heading"><H4 class="panel-title"><A data-toggle="collapse" href="#collapseMenu"><STRONG>Projects</STRONG></A></H4><DIV id="collapseMenu" class="panel-collapse collapse in"><DIV class="panel-heading"><H4 class="panel-title"><A data-toggle="collapse" href="#collapseTALebots"><STRONG>TALebots </STRONG></A></H4></DIV><DIV id="collapseTALebots" class="panel-collapse collapse in"><UL class="list-group"><LI><A href="https://2016.igem.org/Team:Hannover/Description">Description</A></LI><LI><A href="https://2016.igem.org/Team:Hannover/Design">Design</A></LI><LI><A href="https://2016.igem.org/Team:Hannover/Demonstrate">Demonstrate</A></LI><LI><A href="https://2016.igem.org/Team:Hannover/Proof">Proof</A></LI><LI><A href="https://2016.igem.org/Team:Hannover/Results">Results</A></LI><LI><A href="https://2016.igem.org/Team:Hannover/Reference">Reference</A></LI></UL></DIV><DIV class="panel-heading"><H4 class="panel-title"><A data-toggle="collapse" href="#collapseSW"><STRONG>Software Tools </STRONG></A></H4></DIV><DIV id="collapseSW" class="panel-collapse collapse in"><UL class="list-group"><LI><A href="https://2016.igem.org/Team:Hannover/Software#description">Description</A></LI><LI><A href="https://2016.igem.org/Team:Hannover/Software#phabricator">Phabricator</A></LI><LI><A href="https://2016.igem.org/Team:Hannover/Software#cyberprinter">CyberPrinter</A></LI><LI><A href="https://2016.igem.org/Team:Hannover/Software#talsetter">Modeling: TALsetter</A></LI></UL></DIV><DIV class="panel-heading"><H4 class="panel-title"><A href="https://2016.igem.org/Wiki/Team:Hannover/InterLab"><STRONG>InterLab Study</STRONG></A></H4></DIV></DIV></DIV></DIV></DIV></DIV><DIV class="col-sm-8 text-left main-content"><H1>The project</H1><H2>TALebots</H2><DIV><H2>Overview</H2><P>Twenty years ago, modifying the human genome seemed impossible. But with new technologies, such as CRISPR/Cas9, Zink finger and TALEs (transcription activator-like effectors), modifying DNA is not only made possible but also pretty quick. Several articles in the press also show that these new techniques could help develop new treatments for diseases such as to fight the Zika virus <A href="Reference">(Specter, 2016)</A> and even cure cancer <A href="Reference">(Office, 2015)</A>.</P><P>In fact, Layla, a little girl in Britain, survived leukemia through a treatment that uses genome editing with TALENs. These are TALEs combined with a nuclease (Qasim et al., 2015). When Layla’s disease was unstoppable and her life was close to the end, Waseem Qasim and his team from the University College of London had the idea to use engineered donor blood cells and inject them into Layla. Those cells were engineered to seek and destroy leukemia cells. Using TALENs as molecular scissors, the French company Cellectis had disabled genes in those donor cells to prevent a tissue rejection <A href="Reference">(Qasim et al., 2015)</A>. In 2016, those modified blood cells from Cellectis were also used to cure a second baby of cancer <A href="Reference">(Hirschler, 2016)</A>.</P><P>If we want to use TALEs or TALENs to cure diseases such as cancer, or discover other treatment options, the proteins need to be as stable as they can be. This is where our research stepped in. TALEs by themselves are not very stable and proteases can destroy the link between amino acids, which forms the protein. This instability leads to a problem - the purification of TALEs, as well as the in-vitro application in the lab, is difficult to perform. For this reason, TALE proteins are excluded from lots of applications, because a lot of genetic work usually takes place <I>in vitro</I>.</P><P>Circularizing a protein was first introduced to iGEM by the team from Heidelberg in 2014 with their “Ring of Fire”. Our aim is to develop circular TALEs with the help of a linker in order to stabilize the protein. Thereby, TALEs can be utilized on a daily basis and help enable new techniques of genetic engineering in the lab. This led us to believe that the circularization of TALEs is possible.</P><H2>Background</H2><H3>TALs</H3><P>Transcription activator-like effectors (TALEs) are type III-secreted effector proteins of <I>Xanthomonas ssp</I>. They are used to induce target plant genes and, support bacterial colonization of plant tissue. <I>Xanthomonas</I> injects the protein into plants by using a type III secretion system. Inside the plant cell, the protein reaches the nucleus and binds to promoters of targeted genes. In this way, they activate transcription.</P><FIGURE style="text-align:center;"><FIGCAPTION style="text-align:center;"><SMALL>Figure 1: <I>Xanthomonas</I> infection of a plant (Streubel et al., 2013)</SMALL></FIGCAPTION></FIGURE><P>TALE proteins are able to bind to a specific DNA sequence determined by a central domain of a variable number of repeats which also defines their specificity (Boch et al., 2009). The repeats are very similar but contain a variable pair of amino acids at position 12/13 which determine the DNA-binding specificity. The bond of the protein with the DNA is caused by a bond of each repeat with one DNA base pair and the interaction of amino acid 13 with the base of the DNA-leading strand. In addition, non-canonical repeats in the N-terminal region interact with the DNA in general. The repeats of the TALE can be rearranged to generate a bond to any desired DNA sequence. Furthermore, TALEs contain nuclear localization signals and an acidic transcriptional activation domain, too.</P><FIGURE style="text-align:center;"><FIGCAPTION><SMALL>Figure 2: TAL effectors contain central tandem repeats, NLSs, and an AD (Boch et al., Breaking the Code of DNA Binding Specificity of TAL-Type III Effectors, 2009)</SMALL></FIGCAPTION></FIGURE><P>In order to use TALEs in the biotechnology industry, catalytic domains of endonucleases such as FokI can be fused to the TALE protein. By inducing sequence specific double-strand breaks, TALEs are an effective tool for genome editing (Miller et al., 2011). In addition, TALEs naturally regulate gene expression and can be designed to switch on any gene in a plant and even human (Geissler et al., 2011).</P><P>The TALE system is indeed very similar to the CRISPR/Cas9 technology but offers an even more accurate genome editing. In addition, TALE proteins offer a significant advantage compared to CRISPR/Cas9 - they don't contain any nucleic acid. By an easy change of the amino acid sequence, the protein can be customized to any DNA sequence, and can cheaply be build from preformed building blocks. The function of TALEs could be proven in many different plants, animals, and even human cells. This is why TALEs save time and effort in the lab, allowing multiple ways to alter a genome.</P><FIGURE style="text-align:center;"><FIGCAPTION><SMALL>Figure 3: Crystal structure of a TALE (Streubel et al., 2013)</SMALL></FIGCAPTION></FIGURE><H3>Team Heidelberg: Ring of Fire</H3><DIV class="row"><DIV class="col-sm-10"><P>In 2014, the iGEM team Heidelberg conducted their project “Ring of Fire” <A href="https://2014.igem.org/Team:Heidelberg">(Heidelberg, 2014)</A> and successfully circularized GFP and dihydrofolate reductase by ligating the N- and C-terminal ends through inteins and Sortase circularization. The intein peptides were integrated into proteins. After protein translation, they perform an autocatalytic splicing reaction and link the ends of the protein together. </P></DIV></DIV><P>The team also used the intein <I>Npu</I> DnaE which is naturally split into two fragments and reassembles. </P><P>They successful showed that circularization enhances resistance against proteases and denaturation. </P><P>To perform our experiments, we were able to use the vector construct of the team Heidelberg. For the purpose of optimizing it and stabilizing the circularization, we changed the intein linker sequence. </P><H3>Circularization of TALEs with the help of self-splicing Inteins</H3><P>A circular protein has been shown to be thermally stable and resistant to digestion by exo-proteases or reducing environments <A href="”Reference”">(Williams, 2002)</A>. This is achieved by a smaller size and the reduction of conformational entropy in the denatured state <A href="”Reference”"> (Muir, 2003). </A></P><P>Like the team from Heidelberg, we focused on Inteins to perform the circularization. Inteins are protein domains which perform a cis-splicing reaction followed by their own excision and the break of two peptide bonds. A new bond is formed between the remaining parts (also called exteins). Recent studies discovered Inteins that are split into two halves and only regain activity when binding together, for example, the Npu Intein<A href="”Reference”"> (Wood &amp; Camarero, 2014)</A>. In this way, two proteins can be ligated through Intein-splicing. If a protein is fused to a C-Intein next to a C-Extein and an N-Intein next to an N-Extein, trans-splicing causes the formation of a peptide bond between C-Extein and N-Extein as well as the excision of the combined intein <A href="”Reference”"> (Muir, 2003). </A></P><FIGURE style="text-align:center;"><FIGCAPTION><SMALL>Figure 4: Visualization of our project &quot;TALebots&quot;</SMALL></FIGCAPTION></FIGURE></DIV></DIV></DIV></DIV><FOOTER class="container-fluid text-center"><DIV style="padding:5px;"><H5><STRONG>Sponsors</STRONG></H5><P><SMALL><STRONG>Our project would not have been possible without financial support from multiple sponsors and supporters.</STRONG></SMALL></P></DIV></FOOTER></DIV></DIV></DIV></DIV></DIV></BODY></HTML>