<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_Hannover_Notebook skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><DIV id="top_title"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:Hannover/Notebook</SPAN></H1></DIV><DIV id="HQ_page"><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><DIV class="full_size container-fluid carousel_div"><DIV class="carousel_container"><DIV id="carousel_myCarousel" class="carousel_myCarousel carousel slide" data-ride="carousel"><A class="left carousel-control" href="#carousel_myCarousel" role="button" data-slide="prev"><SPAN class="sr-only">Previous</SPAN></A><A class="right carousel-control" href="#carousel_myCarousel" role="button" data-slide="next"><SPAN class="sr-only">Next</SPAN></A></DIV></DIV></DIV><NAV class="navbar navbar-inverse menu_wrapper full_size" data-spy="affix" data-offset-top="197" style="z-index: 99; width:100%"><DIV class="container-fluid"><UL id="accordion" class="nav navbar-nav accordion"><LI class="menu_item"><A href="https://2016.igem.org/Team:Hannover">HOME </A></LI><LI class="dropdown menu_item"><A class="dropdown-toggle" data-toggle="dropdown" href="#">TEAM </A></LI><LI><A href="https://2016.igem.org/Team:Hannover/Team"> Team </A></LI><LI><A href="https://2016.igem.org/Team:Hannover/Team#sponsors"> Sponsors </A></LI><LI class="menu_item"><A href="https://2016.igem.org/Team:Hannover/Attributions">Attributions</A></LI></UL><LI class="dropdown menu_item"><A class="dropdown-toggle" data-toggle="dropdown" href="#">PROJECT</A></LI><LI><A href="https://2016.igem.org/Team:Hannover/Description">TALebots </A></LI><LI><A href="https://2016.igem.org/Team:Hannover/Software">Software </A></LI><LI><A href="https://2016.igem.org/wiki/Team:Hannover/InterLab">Interlab Study</A></LI><LI><A href="https://2016.igem.org/Team:Hannover/Notebook"> NOTEBOOK </A></LI><LI class="dropdown menu_item"><A class="dropdown-toggle" data-toggle="dropdown" href="#">PARTS</A></LI><LI><A href="https://2016.igem.org/Team:Hannover/Parts">Parts </A></LI><LI><A href="https://2016.igem.org/Team:Hannover/Basic_Part"> Basic Parts </A></LI><LI><A href="https://2016.igem.org/Team:Hannover/Composite_Part"> Composite Parts </A></LI><LI class="menu_item"><A href="https://2016.igem.org/Team:Hannover/Safety"> SAFETY </A></LI><LI class="dropdown menu_item"><A class="dropdown-toggle" data-toggle="dropdown" href="#">HUMAN PRACTICES</A></LI><LI><A href="https://2016.igem.org/Team:Hannover/HP/Silver">Engagement </A></LI><LI><A href="https://2016.igem.org/Team:Hannover/HP/Gold">Survey </A></LI><LI><A href="https://2016.igem.org/Team:Hannover/Collaborations">iGEM COMMUNITY </A></LI><LI class="dropdown menu_item"><A class="dropdown-toggle" data-toggle="dropdown" href="https://2016.igem.org/Team:Hannover/Achievements">ACHIEVEMENTS</A></LI><LI><A href="https://2016.igem.org/Team:Hannover/Achievements#Bronze"> Bronze </A></LI><LI><A href="https://2016.igem.org/Team:Hannover/Achievements#Silver"> Silver </A></LI><LI><A href="https://2016.igem.org/Team:Hannover/Achievements#Gold"> Gold </A></LI></DIV></NAV><DIV class="container-fluid text-center"><DIV class="row content" style="background-color:rgb(0, 107, 168);"><DIV class="col-sm-2 sidenav"><DIV class="panel-group" style="text-align: left;"><DIV class="panel panel-default"><DIV class="panel-heading"><H4 class="panel-title"><A data-toggle="collapse" href="#collapseMenu"><STRONG>Notebook</STRONG></A></H4><DIV id="collapseMenu" class="panel-collapse collapse in"><DIV class="panel-heading"><H4 class="panel-title"><A href="#protocols"><STRONG>Protocols</STRONG></A></H4></DIV></DIV></DIV></DIV></DIV></DIV><DIV class="col-sm-8 text-left main-content"><H1>Notebook</H1><TABLE><TBODY><TR><TH>date</TH><TH>activity</TH><TH>summary</TH></TR><TR><TD>04-21-16</TD><TD>Bianca</TD><TD>preparation of working area</TD><TD>pipets and other stuff autoclaved</TD></TR><TR><TD>04-28-16</TD><TD>Janis, Bianca</TD><TD>testing of pipets</TD><TD>pipets work well</TD></TR><TR><TD>assembly of TALEs</TD><TD>we cloned three different constructs in TALEs and transformed them into <I>E.coli</I>: 1) R-SCTALEN left HA Tag 2) R-TALEN left HA Tag 3) R-TALEN right myc Tag</TD></TR><TR><TD>isolation of plasmids</TD><TD>We isolated the two plasmids 1) iGEM20? and 2) pET 32a+?  out of <I>E.coli</I> with a QIAprep(R) Spin Miniprep Kit. The concentrations were 1) 354 ng/ µl and 2) 43.5 ng/ µl dsDNA measured with a NanoPhotometer</TD></TR><TR><TD>05-03-16</TD><TD>Janis</TD><TD>assembly of TALEs, 2nd try</TD><TD>Cells could not be transformed, propably GGC of vectors did not work. Did the assembly again, this time over night. </TD></TR><TR><TD>PCR/Gels/Prep/Cloning</TD><TD>Primers arrived. Did PCR with N454-F/R from N454-HA and TRX_mod-F/R from PET32. PCR products cloned into PUC57-vector</TD></TR><TR><TD>05-23-16</TD><TD>Bianca</TD><TD>plasmid isolation</TD><TD>Plasmid isolation with QIAprep Spin Miniprep kit. The protocol attached to the kit was used without step 7. </TD></TR><TR><TD>Plasmids: TRXmodule1 (1), TRXmodule2 (2), iGEMN454-1 (3), iGEMN454-2 (4), 1-1, 1-2, 2-2, 3-1, 3-2, 5 FOK DS wos (without stop), 6 FOK RR wos, 7 Sc FOK wos</TD></TR><TR><TD>control digest</TD><TD>Control digest with plasmids (5) - (9). Mastermix for (5)-(9): 3 µl EcoRI-HF, 3 µl BamHI-HF, 12 µl buffer CutSmart and 66 µl H<SUB>2</SUB>O, 14 µl of the Mastermix were added to 6 µl of each plasmid DNA</TD></TR><TR><TD>Control digest with plasmids (1) - (4). Mastermix for (1)-(4): 5 µl BsaI 10 µl CutSmart, 55 µl H<SUB>2</SUB>2O, 14 µl of the Mastermix were added to 6 µl of plasmid DNA</TD></TR><TR><TD>The digests ran for 1 h at 37 °C</TD></TR><TR><TD>agarose gel</TD><TD>After the digest 5 µl loadingbuffer was added to the control digest, 20 µl were loaded on a 1 % agarose gel for 30 min and at 140 V. 7 µl of a 1 kb DNA ladder were used.</TD></TR><TR><TD>05-25-16</TD><TD>Janis</TD><TD>sequencing</TD><TD>Plasmids and primer were pipetted.</TD></TR><TR><TD>05-26-16</TD><TD>Bianca</TD><TD>TNF reaction</TD><TD>500 ng DNA, 1 µl Met, ad 50 µl H<SUB>2</SUB>O. Used DNA: iGEM 1-2 (A and B), MPAx SCL, 90 min at 30 °C</TD></TR><TR><TD>in-vitro-cleavage assay</TD><TD>TC(-): 12,5 µl DNA, 2 µl enzyme, 2 µl NEB3 buffer, 3,5 µl H<SUB>2</SUB>O  (+) Ax7: 10 µl DNA, 2 µl enzyme, 2 µl NEB3 buffer, 6 µl H<SUB>2</SUB>O; 30 min at 37 °C</TD></TR><TR><TD>MiniElute PCR Purification Kit, 100 µl buffer PB, probes frozen with glycerin </TD></TR><TR><TD>05-30-16</TD><TD>Bianca</TD><TD>in-vitro-cleavage assay</TD><TD>2 µl DNA ((+)AX7 und TC(-)), 3 µl NEB3 buffer, 5 µl BSA, 4 µl TNT(-)/Scl + 16 µl H<SUB>2</SUB>O or 13 µl TNT(+) + 7 µl HH<SUB>2</SUB>OO, ad 30 µl HH<SUB>2</SUB>OO</TD></TR><TR><TD>centrifugation for: 1 h at 37 °C, 20 min at 65 °C, 10 min at 7000xg, 3 min at 14000xg, 20 µl supernatant + 5 µl LB</TD></TR><TR><TD>agarose gel</TD><TD>1 % agarose gel, 50 ml liquide agarose + 5 µl RedSafe, 20 min at 140 V</TD></TR><TR><TD>Western blot/preparing probes</TD><TD>at 37 °C, 30- (4 µl) and 30+ (13 µl) with 60 µl Laemmli, after 0, 30, 90 and 180 min. 10 min at 100 °C, stored on ice</TD></TR><TR><TD>repetition of in-vitro-cleavage assay</TD><TD>SCL/TNT(+) with Ax7</TD></TR><TR><TD>25 µl TNT+, 4 µl buffer, 4 µl H<SUB>2</SUB>O, ad 40 µl H<SUB>2</SUB>O</TD></TR><TR><TD>4 µl Scl, 4 µl buffer, 25 µl H<SUB>2</SUB>O, ad 40 µl H<SUB>2</SUB>O</TD></TR><TR><TD>37 °C</TD></TR><TR><TD>06-01-16</TD><TD>Bianca</TD><TD>SDS gel</TD><TD>15 µl probes for westernblot load on SDS gel (probes from 0, 30, 90 and 180 min + and -),40 min at 200 V</TD></TR><TR><TD>westernblot</TD><TD>90 min at 120 mA</TD></TR><TR><TD>membran swing for 1 h in milkpowder and TBST, 1 x in TBST washed, 1 x 5 min in TBST washed, 1 x 10 min in TBST washed</TD></TR><TR><TD>membran with TBST and α-His (antibody) over night (15 µl antibody in TBST (100 µg/500 µl))</TD></TR><TR><TD>06-02-16</TD><TD>Bianca, Janis</TD><TD>membran washed, second antibody</TD></TR><TR><TD>06-03-16</TD><TD>Janis</TD><TD>transformation</TD><TD>chemically competent <I>E.Coli</I> cells were used, for Ax7L, Ax7R, scAx7L, 1-2; 2 µl DNA used and 50 µl competent cells; 20 min on ice, 45 sec at 42 °C, 1 h at 37 °C in 1 ml LB without antibiotic.</TD></TR><TR><TD>cells with Ax7L, Ax7R, scAx7L plated on LB and ampicillin. Cells with 1-2 on LB with chloramphenicol. Stored at 37 °C over night.</TD></TR><TR><TD>06-07-16</TD><TD>Wiebke, Theresa, Sabine</TD><TD>Interlab study</TD><TD>vector transformed into Top10 <I>E.coli</I> via heatshock transfomation. 1 h at 37 °C </TD></TR><TR><TD>06-27-16</TD><TD>Bianca</TD><TD>assembly of TALENs</TD><TD>new vector: p3H001; 4 rudiments with 1) Ax7L-DS, 2) Ax7R-RR, 3) Ax7L-scFOK, 4) Ax7R-scFOK</TD></TR><TR><TD>p3H001 77,0 ng/µl, 2 µl of C-terminal and 2 µl of N-terminal module, 5 µl H<SUB>2</SUB>O, used PCR programm (see protocol)</TD></TR><TR><TD>transformation</TD><TD>transformation into <I>E.coli</I> with electroporation. 200 µl LB and 30 min at 37 °C. Plated on LB chloramphenicol, stored at 37 °C over night.</TD></TR><TR><TD>06-28-16</TD><TD>Bianca</TD><TD>overnight cultures</TD><TD>5 ml LB and 34 mg/ml chloramphenicol with 1 colony from the plates. 3 cultures picked from each plate. Incubate at 37 °C over night.</TD></TR><TR><TD>06-30-16</TD><TD>Bianca, Kim</TD><TD>plasmid isolation</TD><TD>plasmid isolation with QIAprep Spin Miniprep kit. The protocol attached to the kit was used without step 7. Stored at -20 °C</TD></TR><TR><TD>overnight cultures</TD><TD>3 colonies from each plate picked, overnight cultures with 5 ml LB, 34 mg/ml chloramphenicol.</TD></TR><TR><TD>07-01-16</TD><TD>Kim</TD><TD>control digest</TD><TD>control digest with the isolated plasmids from 06-30-16. 10 µl plasmid DNA, 2 µl 10x reaction buffer, 0,5 µl EcoRI, 0,5 µl PstI</TD></TR><TR><TD>1 h at 37 °C and 10 min at 65 °C</TD></TR><TR><TD>agarose gel</TD><TD>loaded with 15 µl probes from control digest and 7 µl 1 kb plus DNA ladder</TD></TR><TR><TD>plasmid isolation</TD><TD>used the overnightcultures from 06-30-16; with QIA Prep Spin Miniprep Kit (without step 7)</TD></TR><TR><TD>07-19-16</TD><TD>Janis</TD><TD>retransformation</TD><TD>plasmids 1.1, 2.1, 3.1, 3.2,  4.1 in Top10 <I>E.coli</I>. Incubation in LB with chloramphenicol (1.1: Ax7L-DS, 2.1: Ax7R-RR, 3.1: Ax7L-scFOK, 3.2: Ax7L-scFOK, 4.1: Ax7R-scFOK)</TD></TR><TR><TD>07-20-16</TD><TD>Kim</TD><TD>plasmid isolation</TD><TD>used the cultures from 07-19-16, with QIA Prep Spin Miniprep Kit (without step 7)</TD></TR><TR><TD>07-21-16</TD><TD>Janis</TD><TD>control digest</TD><TD>used plasmids from 06-20-16, with 4 µl plasmid DNA, 0,5 µl XbaI and 0,5 µl  PstI, 2 µl buffer and 10 µl H<SUB>2</SUB>O. For 1 h at 37 °C and 10 min at 65 °C.</TD></TR><TR><TD>07-28-16</TD><TD>Sabine, Wiebke, Theresa</TD><TD>PCR</TD><TD>run a PCR with 10 µl 5x HF Phusion Buffer, 5 µl dNTPs, 2,5 µl forward Primer, 2,5 µl reverse Primer, 1 µl template TCR5 H2, 0,5 µl Phusion polymerase, 28,5 µl H<SUB>2</SUB>O</TD></TR><TR><TD>1. programm: 98 °C 2 min at the beginnig, denaturation at 98 °C for 10 sec, annealing at 60 °C for 10 sec, extension at 72 °C for 30 sec. steps repeated for 35 times. at the and 72 °C for 5 min. </TD></TR><TR><TD>2. programm: same as the first programm, just the annealing temperature was 66 °C.</TD></TR><TR><TD>agarose gel</TD><TD>probes loaded on 1 % agarose gel, run for 120 V for 20 min, 1 kb plus DNA ladder.</TD></TR><TR><TD>transformation </TD><TD>1.1, 2.1, 3.1, 3.2 and 4.1 in BL21DE3 <I>E.coli</I> with electroporation. 1 h at 37 °C , plated on LB with chloramphenicol, incubation at 37 °C</TD></TR><TR><TD>puc57 in Top10 with heatshock at 42 °C for 40 sec, 200 µl LB added, 1 h at 37 °C. Plated on LB</TD></TR><TR><TD>gel elution</TD><TD>cut the fragment out of the gel. add 300 µl QG buffer, 10 min at 50 °C, add 100 µl Isopropanol, transfered to column, 1 min at max. rpm, add 400 µl QG buffer, 1 min at max. rpm, add 700 µl PE buffer, </TD></TR><TR><TD>1 min at 1500 rpm, centrifuge to dry the column, 1 min at max rpm, add 15 µl EB buffer or H<SUB>2</SUB>O, incubate for one minute, centrifugation for 1 min at max rpm.</TD></TR><TR><TD>cut and ligation</TD><TD>2 µl cut smart buffer, 2 µl ATP, 2 µl puc57, 1 µl SmaI, 1 µl T4 Ligase, 15 µl Elution, 2 µl H<SUB>2</SUB>O, for 2 h at 30 °C</TD></TR><TR><TD>PCR</TD><TD>repeat the PCR, but used 1 min elonation time at 72 °C</TD></TR><TR><TD>agarose gel</TD><TD>load gel with PCR probes, run at 120 V for 20 min</TD></TR><TR><TD>08-01-16</TD><TD>Wiebke, Sabine</TD><TD>Plasmid isolation </TD><TD>Plasmid isolation from overnight cultures with iGem Nterm and GFP puc57  (1, 2, 3, 4)</TD></TR><TR><TD>Used QIA Prep Spin Miniprep Kit without step 7. 4 ml Overnightculture were used.  </TD></TR><TR><TD>control digest</TD><TD>with the isolated plasmids. used 0,5 µl BsaI, 7 µl DNA, 2 µl CutSmart buffer, ad 20 µl (10.5 µl H<SUB>2</SUB>O)  </TD></TR><TR><TD>1 h at 37 °C, 10 min at 65 °C.</TD></TR><TR><TD>agarose gel</TD><TD>1 % agarose gel, used 7 µl 1 kb DNA ladder, 20 µl from Controll digest with 6 µl 5x LD buffer (20 µl loaded on agarose gel)  </TD></TR><TR><TD>Gel at 120 V for 25 min</TD></TR><TR><TD>08-02-16</TD><TD>Wiebke, Sabine, Louise</TD><TD>plasmid isolation</TD><TD>Repeat the plasmid isolation from 08-01-16</TD></TR><TR><TD>control digest </TD><TD>with the isolated plasmids. used 1 µl BsaI, 26 µl DNA, 3 µl CutSmart buffer</TD></TR><TR><TD>agarose gel</TD><TD>1 % agarose gel, used 7 µl 1 kb DNA ladder, 20 µl from control digest with 6 µl 5x LD buffer (20 µl loaded on agarose gel) </TD></TR><TR><TD>08-03-16</TD><TD>Janis</TD></TR><TR><TD>cloning</TD><TD>GFP  - NTH3 Fragment into puc57: 5 µl DNA, 1 µl SmaI, 1 µl T4 Ligase, 1 µl Puc57, 1 µl ATP, 1 µl tango buffer; 1 h at room temperatur (RT)</TD></TR><TR><TD>transformation</TD><TD>puc57 with GFP - NTH3 in <I>E.coli</I> (Top10)</TD></TR><TR><TD>transformation</TD><TD>transformation into <I>E.coli</I> (BL21DE3) with electroporation. 200 µl LB and 30 min at 37 °C. Plated on LB chloramphenicol, stored at 37 °C over night.</TD></TR><TR><TD>Used the DNA 1.1, 2.1, 3.1, 3.2 and 4.1</TD></TR><TR><TD>08-10-16</TD><TD>Kim</TD><TD>overnightcultures</TD><TD>used 4 colonies from 08-03-16 (puc57 with GFP-NTH3 in Top10), 5 ml LB, 5 µl carbenicillin.</TD></TR><TR><TD>08-11-16</TD><TD>Kim</TD><TD>plasmid isolation</TD><TD>plasmid isolation with the overnightcultures from 08-10-16, used 4 ml overnightculture and QIA Prep Spin Miniprep Kit without step 7.</TD></TR><TR><TD>08-12-16</TD><TD>Kim</TD><TD>control digest</TD><TD>plasmids from 08-11-16; 5 µl DNA, 2 µl CutSmart buffer, 0.5 BsaI, 12.5 µl <I>E.coli</I> ; 1 h at 37 °C, 10 min at 65 °C</TD></TR><TR><TD>agarose gel</TD><TD>1 % agarose gel loaded with 20 µl probe (20 µl control digest and 4 µl 6x loading buffer), 7 µl 1 kb DNA ladder; 27 min at 100 V</TD></TR><TR><TD>08-16-16</TD><TD>Theresa, Lousie, Severin</TD><TD>PCR</TD><TD>like the PCR from 07-28-16</TD></TR><TR><TD>agarose gel</TD><TD>1 % agarose gel loaded with 60 µl probe (50 µl PCR probe and 12,5 5x loading buffer), 8 µl DNA ladder; 20 min at 120 V</TD></TR><TR><TD>gel extraction</TD><TD>gel extraction with MiniElute Extraction Kit</TD></TR><TR><TD>cloning</TD><TD>like the cloning from 08-03-16, but CurSmart buffer was used</TD></TR><TR><TD>transformation</TD><TD>10 µl cloning stuff, 50 µl competent <I>E.coli</I> cells; add 200 µl LB for 30 min</TD></TR><TR><TD>10 µl puc57, 50 µl competent <I>E.coli</I> cells; add 200 µl LB for 30 min</TD></TR><TR><TD>08-17-16</TD><TD>Janis</TD><TD>overnightcultures</TD><TD>5 colonies (1-5) picked from transformation (08-16-16), in 5 ml LB with 5 µl Carbenicellin</TD></TR><TR><TD>08-18-16</TD><TD>Janis</TD><TD>plasmid isolation</TD><TD>plasmid isolation (overnight cultures used from 08-17-16) with QIA Prep Spin Miniprep Kit without step 7.</TD></TR><TR><TD>control digest</TD><TD>control digest with the isolated plasmids (06-18-16), same control digest like the digest from 08-12-16 (5 µl DNA, 2 µl CutSmart buffer, 0.5 BsaI, 12.5 µl H<SUB>2</SUB>O; 1 h at 37 °C, 10 min at 65 °C)</TD></TR><TR><TD>assembly</TD><TD>assembly oft GFP-NTH3 into iGem vector; 2 µl iGem vector (50 ng/µl), 1 µl  hexa-repeat LA, 1 µl hexa-repeat AB, 1 µl hexa-repeat BR (each 50 ng/µl), 2 µl N-terminal module (100 ng/µl), 2 µl C-terminal module (100 ng/µl), 2 µl  NEB buffer, </TD></TR><TR><TD>1 µl enzym Mix (2 µl ATP, 2 µl NEB CutSmart, 1 µl T4-DNA Ligase, 1 µl BsaI) and 7 µl H<SUB>2</SUB>O</TD></TR><TR><TD>assembly program: 10 min at 40 °C, 10 min at 16 °C (3 cycles), 20 min at 50 °C, 20 min at 80 °C (Final)</TD></TR><TR><TD>transformation</TD><TD>15 µl assembly, 50 µl competent cells, transformation with electroporation, 200 µl LB, 1 h at 37 °C, plated on LB with chloramphenicol</TD></TR><TR><TD>08-22-16</TD><TD>Kim</TD><TD>overnightcultures</TD><TD>picked 6 colonies from 08-18-16, with LB and Chloramphenicol</TD></TR><TR><TD>08-23-16</TD><TD>Kim</TD><TD>plasmid isolation</TD><TD>plasmid isolation (used 4 ml overnightcultures from 08-22-16) with QIA Prep Spin Miniprep Kit without step 7</TD></TR><TR><TD>control digest</TD><TD>with plasmids (08-23-16), used 7 µl DNA, 2 µl 10x reaction buffer, 0.5 µl XbaI, 0.5 µl PstI, 10 µl H<SUB>2</SUB>O</TD></TR><TR><TD>1 h at 37 °C, 10 min at 65 °C</TD></TR><TR><TD>agarose gel</TD><TD>1 % agarose gel, loaded with 20 µl probes (20 µl control digest and 4 µl 6x loading buffer), 29 min at 100 V</TD></TR><TR><TD>08-24-16</TD><TD>Janis, Kim</TD><TD>transformation</TD><TD>plasmids 1.1, 2.1, 3.1, 3.2,  4.1 in BL21 <I>E.coli</I>., incubation on LB with Chloramphenicol at 37 °C</TD></TR><TR><TD>control digest</TD><TD>repeat control digest from 08-23-16 with 10 µl DNA (2 µl 10x reaction buffer, 0.5 µl XbaI, 0.5 µl PstI, 7 µl H<SUB>2</SUB>O); 1 h at 37 °C, 10 min at 65 °C</TD></TR><TR><TD>agarose gel</TD><TD>1 % agarose gel, loaded with 20 µl probes (20 µl control digest and 4 µl 6x loading buffer), 29 min at 100 V</TD></TR><TR><TD>overnightculures</TD><TD>picked 4 colonies from 08-18-16, with LB and Chloramphenicol</TD></TR><TR><TD>08-25-16</TD><TD>Janis, Kim</TD><TD>plasmid isolation</TD><TD>plasmid isolation (overnightcultures used from 08-24-16 7-10(GFP-NTH3 in iGem vector)) with QIA Prep Spin Miniprep Kit without step 7.</TD></TR><TR><TD>plasmid isolation (overnightcultures interlab study + control 1, + control 2, - control 1, - control 2)</TD></TR><TR><TD>transformation</TD><TD>retransformation plasmids 1.1, 2.1, 3.1, 3.2 and 4.1 in Top 10 <I>E.coli</I>, 50 µl competent cells, 20 min on ice, 45 sec at 42 °C, 1 h at 37 °C in 200 µl LB, plated on LB with chloramphenicol</TD></TR><TR><TD>transformation plasmids for interlab study in Top10 <I>E.coli</I>, 50 µl competent cells, 20 min on ice, 45 sec at 42 °C, 1h at 37 °C in 200 µl LB, plated on LB</TD></TR><TR><TD>08-26-16</TD><TD>Kim</TD><TD>control digest</TD><TD>with plasmids (25-08-16), used 11 µl DNA, 2 µl 10x reaction buffer, 0.5 µl XbaI, 0.5 µl PstI, 6 µl H<SUB>2</SUB>O - different XbaI and PstI stocks used, also repeat control digest with plasmids 1-6 </TD></TR><TR><TD>1 h at 37 °C, 10 min at 65 °C</TD></TR><TR><TD>agarose gel</TD><TD>1 % agarose gel, loaded with 20 µl probes (20 µl control digest and 4 µl 6x loading buffer), 29 min at 100 V</TD></TR><TR><TD>08-29-16</TD><TD>Bianca</TD><TD>Interlab study, transformation</TD><TD>DNA from Göttingen in 10 µl H<SUB>2</SUB>O; 50 µl competent cells (Top10 <I>E.coli</I>), 2 µl DNA, 20 min on ice, 45 sec at 42 °C, 1 ml LB 1 h at 37 °C, plated on LB with chloramphenicol, incubation at 37 °C</TD></TR><TR><TD>overnightcultures</TD><TD>3 colonies each plate (from 08-25-16, Top with 1.1, 2.1, 3.1, 4.1), with 5 ml LB and Chloramphenicol, incubate at 37 °C</TD></TR><TR><TD>3 colonies each plate (from 08-24-16, BL21 with 1.1, 2.1, 3.1, 3.2, 4.1) with 5 ml LB and Chloramphenicol, incubate at 37 °C</TD></TR><TR><TD>08-30-16</TD><TD>Bianca, Theresa</TD><TD>expression</TD><TD>2.5 ml from each overnightculture (from 08-29-16, BL21 with 1.1, 2.1, 3.1, 3.2, 4.1) in 10 ml LB</TD></TR><TR><TD>Incubation, optical density after 60 min: 1.1 - 0.136, 2.1 - 0.122, 3.1 - 0.181, 3.2 - 0.153, 4.1 - 0.134</TD></TR><TR><TD>optical density after 115 min: 1.1 - 0.489, 2.1 - 0.494, 3.1 - 0.596, 3.2 - 0.520, 4.1 - 0.420</TD></TR><TR><TD>optical density after 150 min: 1.1 - 0.728, 2.1 - 0.540, 3.1 - 0.921, 3.2 - 0.792, 4.1 - 0.660</TD></TR><TR><TD>1 ml, centrifuged, pellet stored at -20 °C (pre induction)</TD></TR><TR><TD>add 238 µl IPTG, incubate for 1 h and 40 min at 37 °C, 1 ml centrifuged, pellet stored at -20 °C (after induction)</TD></TR><TR><TD>rest centrifuged, into liquid nitrogen and stored at -80 °C</TD></TR><TR><TD>plasmid isolation</TD><TD>plasmid isolation (overnightcultures re-transformation from 08-29-16) with QIA Prep Spin Miniprep Kit without step 7</TD></TR><TR><TD>overnightcultures</TD><TD>100 µl overnightculture (from 08-26-16 interlab study) with 8 ml LB and chloramphenicol, incubation at 37 °C</TD></TR><TR><TD>08-31-16</TD><TD>Bianca, Theresa, Janis</TD><TD>assembly</TD><TD>with 1 µl GFP-NTH3, 1 µl vector (50 ng/µl), 1 µl HexaI (Repeats 1-6), 1 µl HexaII (Repeats 7-11,5), 1 µl T4-Ligase, 1 µl BsaI, 2 µl 10mM ATP, 2 µl CutSmart buffer, ad 20 µl H<SUB>2</SUB>O</TD></TR><TR><TD>two different vectors were used: iGem vector (with c-terminus c-63) and psaE6 (with c-terminus c-17) and 4 different Hexa combinations each vector</TD></TR><TR><TD>A: iGem vector and Hexa 42 + HDHx</TD></TR><TR><TD>B: iGem vector and Hexa 42 + 2xNG</TD></TR><TR><TD>C: iGem vector and Hexa 42 + 2xNN</TD></TR><TR><TD>D: iGem vector and pJet Hexa3 rep</TD></TR><TR><TD>E: psaE6 and Hexa 42 + HDHx</TD></TR><TR><TD>F: psaE6 and Hexa 42 + 2xNG</TD></TR><TR><TD>G: psaE6 and Hexa 42 + 2xNN</TD></TR><TR><TD>H: psaE6 and pJet Hexa3 rep</TD></TR><TR><TD>interlab study</TD><TD>culture and measurment </TD></TR><TR><TD>09-01-16</TD><TD>Janis, Theresa</TD><TD>transformation</TD><TD>15 µl assambly product, 50 µl competent cells (<I>E.coli</I>, Top10), 20 min on ice, 45 sec at 42 °C, add 200 µl LB, 1 h at 37 °C, plated on LB with Chloramphenicol</TD></TR><TR><TD>SDS gel</TD><TD>prepared 4 gels</TD></TR><TR><TD>control digest</TD><TD>with plasmids (08-25-16re-transformation), used 11 µl DNA, 2 µl 10x reaction buffer, 0.5 µl XbaI, 0.5 µl PstI, 6 µl H<SUB>2</SUB>O</TD></TR><TR><TD>1 h at 37 °C, 10 min at 65 °C</TD></TR><TR><TD>09-05-16</TD><TD>Theresa, Kim</TD><TD>SDS gel</TD><TD>prepared 8 gels</TD></TR><TR><TD>separating gel (10 %): 27.25 ml H<SUB>2</SUB>O, 33.25 Acrylamid (30 %), 25 ml Tris (1.5 M; pH 8.8), 1 ml APS (10 %), 1 ml SDS (10 %), 40 µl TEMED</TD></TR><TR><TD>stacking gel: 34 ml H<SUB>2</SUB>O, 8.5 ml Acrylamid (30 %), 6.25 ml Tris (0.5 M; pH 6.8), 0.5 ml APS (10 %), 0.5 ml SDS (10 %), 50 µl TEMED</TD></TR><TR><TD>first separating gel, than stacking gel; ~1 ml Isopropanol on separating gel, when it is in the chamber (remove it before filling stacking gel in the chamber)</TD></TR><TR><TD>1. H<SUB>2</SUB>O, 2. Tris, 3. SDS, 4. Acrylamid, 5. APS, 6. TEMED; APS and TEMED everytime at last! After adding them be fast</TD></TR><TR><TD>overnightcultures</TD><TD>with colonies from tranformation 09-01-16, 4 colonies of the plates A-D, there was nothing on the plates E-H; 5 ml LB, 5 µl Chloramphenicol and the picked colonie</TD></TR><TR><TD>09-06-16</TD><TD>Theresa, Kim</TD><TD>plasmid isolation</TD><TD>plasmid isolation (overnightcultures  from 09-05-16, A1-3, B1-3, C1-3, D1-3) with QIA Prep Spin Miniprep Kit without step 7</TD></TR><TR><TD>transformation</TD><TD>used plasmids from assembly F, G and H (08-31-16), 10 µl assambly product, 50 µl competent cells (<I>E.coli</I>, Top10), 20 min on ice, 45 sec at 42 °C, add 200 µl LB, 30 min at 37 °C, plated on LB with Chloramphenicol</TD></TR><TR><TD>09-07-16</TD><TD>Theresa, Bianca, Kim</TD><TD>control digest</TD><TD>with plasmids (08-25-16), used 4 µl DNA, 2 µl 10x reaction buffer, 0.5 µl XbaI, 0.5 µl PstI, 13 µl H<SUB>2</SUB>O</TD></TR><TR><TD>1 h at 37 °C, 10 min at 65 °C</TD></TR><TR><TD>agarose gel</TD><TD>1 % agarose gel, loaded with 20 µl probes (20 µl control digest and 4 µl 6x loading buffer), 29 min at 100 V</TD></TR><TR><TD>SDS gel</TD><TD>prepared probes: took a little bit of the pellets (from the expression on 08-30-16), also the pre-induction and the after-induction pellets; 80 µl Leammli, resuspent, 10 min at 100 °C</TD></TR><TR><TD>load gel with 15 µl of the prepared probes, 5 µl Page Ruler best; run for 42 min; gel in coomassi staining solution</TD></TR><TR><TD>transformation</TD><TD>15 µl plasmid DNA (from plasmid isolation 09-06-16, plasmids A3, A4, B1, B2, C2, D1, D2) 50 µl competent cells (<I>E.coli</I>, BL21), 20 min on ice, 45 sec at 42 °C, add 200 µl LB, 35 min at 37 °C, 100 µl plated on LB with Chloramphenicol</TD></TR><TR><TD>09-08-16</TD><TD>Theresa, Kim</TD><TD>SDS gel</TD><TD>gel 3 h in H<SUB>2</SUB>O and 4 h in destaining solution, after that over night in PBS buffer</TD></TR><TR><TD>transformation</TD><TD>repeat transformation from 08-07-16, nothing changed</TD></TR><TR><TD>09-09-16</TD><TD>Kim</TD><TD>overnightcultures</TD><TD>used plates from transformation (09-07-16), each plate (A3, A4, B1, B2, D2) one colonie picked, 5 ml LB and 5 µl Chloramphenicol (34 mg/ml)</TD></TR><TR><TD>09-12-16</TD><TD>Kim</TD><TD>overnightcultures</TD><TD>used plates from transformation (09-07-16), each plate (A3, A4, B1, B2, C2, D2) one colonie picked, 5 ml LB and 5 µl Chloramphenicol (34 mg/ml)</TD></TR><TR><TD>09-13-16</TD><TD>Theresa, Sabine, Wiebke</TD><TD>Expression</TD><TD>Used the overnightcultures from 09-12-16, 100 ml LB and different amounts of overnightcultures, depents on the optical density. Used 1.17 ml A, 1.8 ml B, 3.22 ml C, 2.4 ml D - Start optical density was 0.05</TD></TR><TR><TD>Incubation; optical density after 1 h: A - 0.063, B - 0.11, C - 0.201, D - 0.115</TD></TR><TR><TD>Optical density after 2.5 h: A - 0.452, B - 0.59, C - 0.93, D - 0.756</TD></TR><TR><TD>Optical density after 2.75 h: A - 0.5</TD></TR><TR><TD>Optical density after 3 h: A - 0.59</TD></TR><TR><TD>1 ml, centrifuged, pellet stored at -20 °C (pre induction)</TD></TR><TR><TD>add 200 mM IPTG (1 ml), incubate for 1 h and 30 min at 37 °C, 1 ml centrifuged, pellet stored at -20 °C (after induction)</TD></TR><TR><TD>rest centrifuged 10 min, into liquid nitrogen and stored at -80 °C</TD></TR><TR><TD>09-14-16</TD><TD>Sabine, Wiebke, Louise, Theresa, Kim, Severin</TD><TD>Purification of tales</TD><TD>Resuspensionof sample 1.1.1 in 2ml TE-Buffer and 20 μl PMSF</TD></TR><TR><TD>Ultrasonic treatment on ice: program9, 30%energy for 8 min (30sec on/30 sec off)</TD></TR><TR><TD>Centrifuged at 4°C, 20.000gfor 15 min</TD></TR><TR><TD>Supernatant transferred to column(Gravity flow Strep Tactin PepharoseColumn, Iba), purification as</TD></TR><TR><TD>stated by iba protocol </TD></TR><TR><TD>Flow through was stored as aliquots of 500µl labelled ‘E 1 – 6’ and ‘W 1 – 5’. Flow through of first centrifugation was stored and labelled as ‘supernatant’. Storage at -20°C.</TD></TR><TR><TD>09-15-16</TD><TD>Theresa, Wiebke, Kim, Sabine, Severin</TD><TD>SDS-Gels with samples derived from TALe 1.1.1 (09-14-2016)</TD><TD>50µl of each sample mixed with 5,88µl TCA (final concentration 10%). Incubated for 10 min on ice. Centrifuged at 4°C, 20 000xg for 10 min. Mixed with 500µl of 80% acetone and centrifuged at 4°C, 20 000xg for 10 min.</TD></TR><TR><TD>Incubated for 30 min at 37°C. Mixed with 40µl Laemmli and incubated for 10 min at 80°C. Gels ran for 1 h and 20 min at 200V in TANK buffer. </TD></TR><TR><TD>Coomassie-Staining</TD><TD>Coomassie-gels were set to stain over night.</TD></TR><TR><TD>Western Blot</TD><TD>Other gels were Blotted semi-dry at 70V for 50 min. Poncau-staining was done for 10 min. Incubated for 1 h in PBST with 3% milkpowder. Incubation over night with 1st antibody </TD></TR><TR><TD>(StrepMAB-Classic Cell Culture Supernatant Iga1 anti-strep-tag II monoclonal, 2-1508-025 Iba life science)</TD></TR><TR><TD>09-16-16</TD><TD>Sabine, Kim, Wiebke</TD><TD>Continuation of Western</TD><TD>1st antibody removed by washing in PBST. Incubation in PBST with 2nd antibody (Ecl anti-mouse IgG horseradish peroxidase) for 3 h</TD></TR><TR><TD>Stained with 6 ml 100 mM TRIS (pH 8,5), 30 µl Luminol (22.2 mg/ 500 µl DMSO), 15 µl p-Coumaric acid (7.4 mg/ 500 µl DMSO) and 10 µl 30% H<SUB>2</SUB>O<SUB>2</SUB> for 1 min up to 30 min</TD></TR><TR><TD>Continuation of Coomassie-Staining</TD><TD>Destained for 1,5 h and incubated in water for 1h</TD></TR><TR><TD>09-20-16</TD><TD>Sabine, Wiebke, Kim, Theresa, Bianca, Louise</TD><TD>Purification of tales</TD><TD>Resuspension of sample 1.1.1, B2.1 and C2.1 in 2 ml TE-Buffer and 20 µl PMSF, Ultrasonic treatment on ice: program 9, 30% energy for 8 min (30 sec on/30 sec off). Centrifugation at 4°C, 20 000xg for 15 min </TD></TR><TR><TD>Supernatant transferred to column (Gravity flow Strep Tactin Pepharose Colum, Iba), purification as stated by Iba protocol </TD></TR><TR><TD>Flow through was labelled as ‘E 1 – 6’ and ‘W 1 – 5’ and stored at -20°C</TD></TR><TR><TD>Others</TD><TD>Fluorecence of GFP-TAL fusion proteins in BL21 strains 1.1.1, B1 and C2 and in corresponding raw extracts was documented</TD></TR><TR><TD>expression</TD><TD>Overnight cultures in 200 ml LB of BL21 strains B2 and C2 (09-12-16) inoculated and incubated at 37°C</TD></TR><TR><TD>overnight cultures</TD><TD>Overnight cultures in 5ml LB and master dish of 2.1.1, 3.1.1, 3.2.1 and 4.1.1 (09-12-16) inoculated and incubated at 37°C</TD></TR><TR><TD>09-21-16</TD><TD>Bianca</TD><TD>TEV digestion</TD><TD>According to ProTEV plus protocol (Promega)</TD></TR><TR><TD>2µl Buffer0,4 µl 100mM DTT36,8 µl Protein (E6,5) as collected on 09-14-160,8µl TEV</TD></TR><TR><TD>Incubated at 30°C. Samples of 9,5µl were taken after 0, 1, 2, 4 and 6h and stored at -20°C</TD></TR><TR><TD>Proteinexpression</TD><TD>Overnight cultures of BL21 strains B2 and C2 (09-20-16) were used to inoculate 5x 200ml LB+CA each and grown at 37°C to an OD600 of approx. 0,500.</TD></TR><TR><TD>1ml samples were taken of each flask, the cells pelleted, labelled as ‘before induction’ and stored at -20°C.</TD></TR><TR><TD>Protein expression was induced with 400 µl IPTG, final concentration: 2mM</TD></TR><TR><TD>After 1,5h 1ml samples were taken of each flask, the cells pelleted, labelled as ‘after induction’ and stored at -20°C</TD></TR><TR><TD>and the remaining cells were harvested and stored at -80°C</TD></TR><TR><TD>TCA precipitation</TD><TD>Samples derived from 1.1.1, B2.1 and C2.1 (09-20-16) were mixed with 3 µl NaDOC and incubated for 10min on ice</TD></TR><TR><TD>3 µl TCA were added and samples were incubated for 60 min on ice, Centrifuged at 4°C, 6 500RPM for 20 min, Pellets were resuspended in 1 ml 80% acetone, </TD></TR><TR><TD>Centrifuged at 4 °C, 6,500RPM for 10 min, Supernatant was discarded; samples were dried at 37 °C, 30 µl SDS running-buffer + Lämmli was added, incubated at 80 °C for 10 min, Samples were stored at -20 °C</TD></TR><TR><TD>other</TD><TD>Overnight culture of 1.1.1 was transferred into 30ml fresh LB+CA and incubated over night at 37 °C</TD></TR><TR><TD>09-22-16</TD><TD>Bianca, Janis</TD><TD>Proteinexpression</TD><TD>Overnight culture of 1.1.1 (09-21-16) was used to inoculate 5x 200ml LB+CA each and grown at 37 °C to an OD600 of approx. 0,600. 1 ml samples were taken of each flask, the cells pelleted, labelled as ‘before induction’ and stored at -20 °C.</TD></TR><TR><TD>Proteinexpression was induced with 400 µl IPTG, final concentration: 2 mM</TD></TR><TR><TD>After 1,5h 1 ml samples were taken of each flask, the cells pelleted, labelled as ‘after induction’ and stored at -20 °C and the remaining cells were harvested and stored at -80 °C</TD></TR><TR><TD>SDS-gels</TD><TD>Gels ran for 40 min at 200V in TANK buffer</TD></TR><TR><TD>Western Blot</TD><TD>semi-dry blot at 6,9 V for 45 min</TD></TR><TR><TD>Poncau-staining was done for 10 min</TD></TR><TR><TD>Incubation for 1 h in PBST with 3 % milkpowder</TD></TR><TR><TD>Incubated at 4°C over night with 1st antibody (StrepMAB-Classic Cell Culture Supernatant Iga1 anti-strep-tag II monoclonal, 2-1508-025 Iba life science</TD></TR><TR><TD>Plasmidisolation, With Quigen Mini Prep Spin Column Kit according to protocol.</TD></TR><TR><TD>Masterdish</TD><TD>Petridish was damaged, redone with recent BL21 strains: A3.1, A3.2, A3.3, A4.1, A4.2, A4.3, B1.1, B1.2, B1.3, B2.1, C2.1, C2.2, C2.3, D1.1</TD></TR><TR><TD>09-23-16</TD><TD>Bianca, Janis, Kim</TD><TD>Control digest</TD><TD>Plasmids isolated on 09-22-16 were digested with PstI and XbaI (Set1) as well as PstI, XbaI and BsaI (Set2)</TD></TR><TR><TD>Continuation of Western Blot</TD><TD>1st antibody removed by washing in PBS, Inbubation with 2nd, Incubated in PBST with 2nd antibody (EcL anti-mouse IgG horseradish peroxidase) for 2h, 2nd antibody removed by washing in PBS, </TD></TR><TR><TD>Stained with 6 ml 100mM TRIS (pH 8,5), 30 µl Luminol (22,2 ng/500µl DMSO), 15 µl p-Coumaric acid (7,4 ng/500 µl DMSO) and 10 µl 30% H<SUB>2</SUB>O<SUB>2</SUB> for 1 min up to 30 min</TD></TR><TR><TD>09-26-16</TD><TD>Kim, Bianca</TD><TD>SDS-Gel</TD><TD>Used premade 10% Gel from Biorad, TEV-digestion (09-21-16) samples were mixed with 10 µl Lämmli each, E6 1.1.1, B, C (09-20-16) samples were mixed with 30 µl Lämmli each, Incubation at 100°C for 10m in</TD></TR><TR><TD>Gels ran for 45 min at 150 V in TANK buffer.</TD></TR><TR><TD>Western Blot</TD><TD>Used Towbin-Buffer (25 mM Tris, 192 mM glycine, 20% (v/v) methanol (pH 8.3)) Blotted at 6V for 40min in fastBlot Trans Blot Turbo</TD></TR><TR><TD>Poncau-staining was done for 10 min</TD></TR><TR><TD>Incubated for 1h in blocking solution (1xPBST with 1xRotiBlock, Roth), washed with PBST</TD></TR><TR><TD>Incubated at 4°C over night with 1st antibody</TD></TR><TR><TD>Coomassie staining</TD><TD>Coomassie-gels were set to stain over night </TD></TR><TR><TD>Overnight cultures</TD><TD>3.1.1 and 4.1.1 were inoculated in 25ml LB+CA and grown overnight at 37°C</TD></TR><TR><TD>09-27-16</TD><TD>Bianca, Kim, Wiebke</TD><TD>TEV digestion</TD><TD>According to ProTEV plus protocol (Promega)</TD></TR><TR><TD>2µl buffer, 0,4 µl DTT, 36,8µl sample (1.1.1 E6 as collected on 09-20-16), 0,8 µl TEV</TD></TR><TR><TD>Incubated at 30 °C. Samples of 9,5 µl were taken after 0, 1, 2, 4 and 6 h and stored at -20 °C</TD></TR><TR><TD>continuation of Coomassie-Staining</TD><TD>After incubation with destaining solution no bands, not even marker, were visible. Gel discarded.</TD></TR><TR><TD>continuation of Western Blot</TD><TD>1st antibody removed by washing in PBS</TD></TR><TR><TD>Incubated in PBST with 2nd antibody (EcL anti-mouse IgG horseradish peroxidase) for 1 h</TD></TR><TR><TD>2nd antibody removed by washing in PBS</TD></TR><TR><TD>Stained with 6 ml 100 mM TRIS (pH 8,5), 30 µl Luminol (22,2 ng/500 µl DMSO), 15 µl p-Coumaric acid (7,4 ng/500 µl DMSO) and 10µl 30% H<SUB>2</SUB>Osub&gt;2</TD></TR><TR><TD>Proteinexpression</TD><TD>Overnight cultures of BL21 3.1.1 and BL21 4.1.1 (09-26-16) was used to inoculate 50 ml LB+CA each and grown at 37 °C to an OD600 of approx. 0,600. </TD></TR><TR><TD>1ml samples were taken of each flask, the cells pelleted, labelled as ‘before induction’ and stored at -20 °C.</TD></TR><TR><TD>Proteinexpression was induced with 100 µl IPTG, final concentration: 2mM</TD></TR><TR><TD>After 1,5 h 1 ml samples were taken of each flask, the cells pelleted, labelled as ‘after induction’ and stored at -20 °C and the remaining cells were harvested and stored at -80 °C</TD></TR><TR><TD>09-28-16</TD><TD>Kim, Bianca</TD><TD>Purification of tales</TD><TD>Resuspension of samples 1.1.1 (09-22-16), B and C (09-21-16) as well as 3.1.1 and 4.1.1 (09-27-16) in 2 ml TE-Buffer and 20 µl PMSF, Ultrasonic treatment on ice: program 9, 30% energy for 8 min (30 sec on/30 sec off)</TD></TR><TR><TD>Centrifuged at 4 °C, 20 000xg for 15 min</TD></TR><TR><TD>Supernatant transferred to column (Gravity flow Strep Tactin Pepharose Colum, Iba), purification as stated by Iba protocol</TD></TR><TR><TD>09-29-16</TD><TD>Janis, Bianca, Kim</TD><TD>SDS-Gel</TD><TD>Proben: Raw extract B from 09-21-16, E6 B from 09-20-16, E6 B from 09-28-16</TD></TR><TR><TD>Raw extract C from 09-21-16, E6 C from 09-20-16, E6 C from 09-28-16</TD></TR><TR><TD>Raw extract 1.1.1 from 09-22-16, E6 1.1.1 from 09-22-16, E6 1.1.1 from 09-28-16</TD></TR><TR><TD>Raw extract 3.1.1, E6 3.1.1 from 09-28-16</TD></TR><TR><TD>raw extract 4.1.1, E6 4.1.1 from 09-28-16</TD></TR><TR><TD>TEV 0 from 09-27-16, TEV 1 from 09-27-16, TEV 2 from  09-27-16, TEV 4 from 09-27-16</TD></TR><TR><TD>Coomassie-staining</TD><TD>Incubation overnight at RT</TD></TR><TR><TD>Western Blot</TD><TD>Used Towbin-Buffer (25 mM Tris, 192 mM glycine, 20% (v/v) methanol (pH 8.3)), Blotted at 6V for 40 min in fastBlot Trans Blot Turbo</TD></TR><TR><TD>Poncau-staining was done for 10min</TD></TR><TR><TD>Incubated for 1h in PBST with 3% milk powder, washed with PBST</TD></TR><TR><TD>09-30-16</TD><TD>Kim</TD><TD>Continuation of Coomassie-Staining</TD><TD>Destained Gels</TD></TR><TR><TD>continuation of Western Blot</TD><TD>Washed membrane with PBS</TD></TR><TR><TD>Incubated with 1st antibody (StrepMAB-Classic Cell Culture Supernatant Iga1 anti-strep-tag II monoclonal, 2-1508-025 Iba life science) for 1 h</TD></TR><TR><TD>1stAntibody removed by washing in PBS</TD></TR><TR><TD>Inbubation with 2nd antibody (EcL anti-mouse IgG horseradish peroxidase) for one hour</TD></TR><TR><TD>2nd Antibody removed by washing in PBS</TD></TR><TR><TD>Stained for 1 min with 6 ml 100 mM TRIS (pH 8,5), 30 µl Luminol (22,2 ng/500 µl DMSO), 15 µl p-Coumaric acid (7,4 ng/500 µl DMSO) and 10 µl 30% H<SUB>2</SUB>O<SUB>2</SUB></TD></TR><TR><TD>010-04-16</TD><TD>Kim, Severin, Wiebke</TD><TD>Colonie-PCR</TD><TD>Single Colonies of B, C and 1.1.1 were suspended in 10 µl H<SUB>2</SUB>O and incubated at 95°C for 5min.</TD></TR><TR><TD>1µl sample and 49µl mastermix were prepared and amplified as follows:</TD></TR><TR><TD>50µl Buffer 5xG</TD></TR><TR><TD>5µl dNTPs</TD></TR><TR><TD>10µl Primer 1 (1534)</TD></TR><TR><TD>10µl Primer 2 (1716)</TD></TR><TR><TD>7.5µl DMSO</TD></TR><TR><TD>135µ l H<SUB>2</SUB>O</TD></TR><TR><TD>25µl MgCl<SUB>2</SUB></TD></TR><TR><TD>Program:</TD></TR><TR><TD>35x (30s 98°C, 30 s 59 °C, 30 s 72 °C) and 5 min 72 °C</TD></TR><TR><TD>agarose gel</TD><TD>1 % agarose Gel, Gels ran for 27min at 100V</TD></TR><TR><TD>Proteinexpression</TD><TD>1ml LB+CA was inoculated with strain C as a preculture and later distributed into 10x 10ml LB.</TD></TR><TR><TD>Cultures were grown to an OD600 of approx. 0,5 as well as 0.847</TD></TR><TR><TD>Proteinexpression was induced at 30°C and 37°C with 2mM IPTG for as long as 3h and over night.</TD></TR><TR><TD>10-05-16</TD><TD>Kim, Severin, Wiebke</TD><TD>Liquid cultures were harvested and split into two separate reaction tubes for Dotplot and Immunostain</TD></TR><TR><TD>10-06-16</TD><TD>Kim, Wiebke</TD><TD>TEV digestion</TD><TD>Using samples according to ProTEV plus protocol (Promega)</TD></TR><TR><TD>5µl Buffer</TD></TR><TR><TD>1 µl 100 mM DTT</TD></TR><TR><TD>40 µl Sample 1.1.1 (09-08-16) and C (09-20-16, 20 µl E5 and 20 µl E6)</TD></TR><TR><TD>2 µl TEV</TD></TR><TR><TD>52 µl H<SUB>2</SUB>O</TD></TR><TR><TD>Incubated at RT. Samples of 20 µl were taken after 0, 1, 2, 4 and 6h and stored at -20°C</TD></TR><TR><TD>SDS-Gel</TD><TD>20µl of TEV digestion sample was mixed with 20µl Lämmli and incubated at 80°C for 10min, Gels (10%)  ran for 50min at 150V in TANK buffer.</TD></TR><TR><TD>Western Blot</TD><TD>Used Towbin-Buffer (25 mM Tris, 192 mM glycine, 20% (v/v) methanol (pH 8.3))</TD></TR><TR><TD>Blotted at 25V for 40min in fastBlot Trans Blot Turbo</TD></TR><TR><TD>Poncau-staining was done for 10min</TD></TR><TR><TD>Washed with H<SUB>2</SUB>O</TD></TR><TR><TD>Incubated for 1h in blocking solution (1x PBST with 1x RotiBlock, Roth)</TD></TR><TR><TD>Washed with PBS</TD></TR><TR><TD>Incubated at 4°C over night with 1st antibody</TD></TR><TR><TD>10-07-16</TD><TD>Wiebke, Severin, Kim</TD><TD>Continue the Immunostain.</TD><TD>Wash the Membran with PBS</TD></TR><TR><TD>Incubate at RT with the second antibody for one hour</TD></TR><TR><TD>Wash with PBS</TD></TR><TR><TD>Stained for 1min with 6 ml 100 mM TRIS (pH 8.5), 30 µl Luminol (22.2 ng/500 µl DMSO), 15 µl p-Coumaric acid (7.4 ng/500 µl DMSO) and 10 µl 30% H<SUB>2</SUB>O<SUB>2</SUB></TD></TR><TR><TD>Spotting the Oligonucleotides on a Chip</TD><TD>iGEM.npw as layout</TD></TR><TR><TD>Solutions on Lafontain plates, 10 µl Oligonucleotides (100 µM) + 10 µl array it Micro spotting (2x)</TD></TR><TR><TD>spotting</TD></TR><TR><TD>Aftertreatment: </TD></TR><TR><TD>1) UV-Crosslink for 3 minutes</TD></TR><TR><TD>2) 24 hours in a dark room at room temperature (RT)</TD></TR><TR><TD>3) reducing: 2 min 0.2 % SDS, 2x 1 min H<SUB>2</SUB>O, 5 min NaBH4, 1 min icecold H<SUB>2</SUB>O, 1 min 0.2 % SDS, 2x 1 min H<SUB>2</SUB>O</TD></TR><TR><TD>dry</TD></TR><TR><TD>4) Blocking: 45 min Blocking reagent, 42 °C 5 min H<SUB>2</SUB>O</TD></TR><TR><TD>dry</TD></TR><TR><TD>Overnightcultures</TD><TD>Used glycerin stock from Origami cells, incubate in 20 ml LB medium at RT over the weekend</TD></TR><TR><TD>Used colonies from 1.1.1 and 2.1.1 from masterdish, 50 ml LB medium and 50 µl Chloramphenicol. Incubate over the weekend at RT</TD></TR><TR><TD>10-10-16</TD><TD>Kim, Wiebke, Severin</TD><TD>Purification</TD><TD>Samples of 1.1.1 (09-20-16) and 2.1.1 (08-30-16) were resuspended in 5ml TE-Buffer with 50 µl PMSF. Proteins were purifies according to Iba protocol</TD></TR><TR><TD>One Column of 2.1.1 was incubated over night with 50 mM DTT. (at the 5th washing step, use 2 column bead volume (0.4 ml) and add 50 mM DTT. Incubate over night)</TD></TR><TR><TD>Proteinexpression</TD><TD>Cultures of 1.1.1 and 2.1.1 were inoculated into 5x 200ml LB+CA</TD></TR><TR><TD>Cultures were grown to an OD600 of approx. 0,5-0,6</TD></TR><TR><TD>Proteinexpression was induced at 30°C and 37°C with 2mM IPTG for as long as 1,5h.</TD></TR><TR><TD>Afterwards cells were harvested and stored at -80°C</TD></TR><TR><TD>Prepation of electro competent cells</TD><TD>Take 1 ml or more from the overnight culture from origami cells into 50 ml LB medium. Grow the cells until they reaches a optical density at 600 nm of 0.5 – 0.6. Incubate the bacteria 20 min on ice.</TD></TR><TR><TD>Centrifuge 10 min at 4 °C and discard the supernatant. Add 20 ml cold H<SUB>2</SUB>O and resuspend the pellet. Centrifuge again 10 min at 4 °C.</TD></TR><TR><TD>Add 20 ml cold H<SUB>2</SUB>O and centrifuge for 10 min at 4 °C. Discard the supernatant and add 20 ml cold H<SUB>2</SUB>O. Centrifuge for 10 min at 4 °C. Discard the supernatant and add 10 ml cold glycerin (10 %).</TD></TR><TR><TD>Resuspend the pellet and centrifuge at 4 °C for 10 min. discard the supernatant and resuspend the pellet in 2 ml 10 % cold glycerin</TD></TR><TR><TD>Take 100 µl aliquots into 1.5 µl reaction tubes. Froze them in liquid nitrogen and store at - 80 °C or use them directly for transformation.</TD></TR><TR><TD>transformation</TD><TD>electropotation of competent origami cells with 1 µl plasmid DNA (from 1.1.1 and 2.1.1)</TD></TR><TR><TD>incubate for 30 min in 500 µl LB at 37 °C. Plate 100 µl on LB with Chloramphenicol. incubate at 37 °C over night</TD></TR><TR><TD>10-11-16</TD><TD>Wiebke, Louise, Kim</TD><TD>Continued Purification</TD><TD>More samples of 1.1.1 (09-20-16) and 2.1.1 (08-30-16) were resuspended in 5 ml TE-Buffer with 50 µl PMSF. Proteins were purifies according to Iba protocol</TD></TR><TR><TD>One Column of 1.1.1 and 2.1.1 were incubated overnight with 50 mM DTT.</TD></TR><TR><TD>the incubate column from 10-10-16 were Eluated</TD></TR><TR><TD>Stability test of circularised TALes</TD><TD>Purified protein samples of 2.1.1 were set to incubate with DTT (circularised) or without (control) for 0, 6, 24, 32, 48 h at 4 °C, RT and 37 °C. </TD></TR><TR><TD>In addition, one set of samples was frozen at –20°C and thawed at RT for 30 min each for four continuous times.</TD></TR><TR><TD>The linearized TAL 3HD100 was used as an additional control. After each time a fraction of 10µl was stored at -20°C for analysis.</TD></TR><TR><TD>SDS-Gel</TD><TD>10µl of sample were mixed with 10µl Lämmli and incubated at 80°C for 10min.</TD></TR><TR><TD>10µl of the sample/Lämmli mixture were put onto the Gels (10%) which ran for 50min at 150V in TANK buffer.</TD></TR><TR><TD>western blot</TD><TD>Used Towbin-Buffer (25 mM Tris, 192 mM glycine, 20% (v/v) methanol (pH 8.3))</TD></TR><TR><TD>Blotted at 25V for 40min in fastBlot Trans Blot Turbo</TD></TR><TR><TD>Incubated for 1h in blocking solution (1x PBST with 1x RotiBlock, Roth)</TD></TR><TR><TD>Washed with PBS</TD></TR><TR><TD>Incubated at 4°C over night with 1st antibody</TD></TR><TR><TD>Overnightculture</TD><TD>Used colonies from 1.1.1 and 2.1.1 fromtransformation with origami, 20 ml LB medium and 20 µl Chloramphenicol. Incubate at 37 °C over night</TD></TR><TR><TD>10-12-16</TD><TD>Wiebke, Louise, Sabine, Kim</TD><TD>Proteinexpression</TD><TD>Cultures of 1.1.1 and 2.1.1 were inoculated into 2x 125ml LB+CA</TD></TR><TR><TD>Cultures were grown to an OD600 of approx. 0,5-0,6</TD></TR><TR><TD>Proteinexpression was induced at 30 °C and 37 °C with 2 mM IPTG for as long as 2 h.</TD></TR><TR><TD>Afterwards cells were harvested and stored at -80°C</TD></TR><TR><TD>TEV digestion</TD><TD>Using samples 1.1.1 +/- DTT (10-11-16) and 2.1.1 +/- DTT (11-10-16) according to ProTEV plus protocol (Promega)</TD></TR><TR><TD>Samples were taken after 0, 1, 2, 4 and 6h and stored at -20°C</TD></TR><TR><TD>Continue Immunostain</TD><TD>wash membran with PBS</TD></TR><TR><TD>Incubate second antibody for 1 hour</TD></TR><TR><TD>wash with PBS</TD></TR><TR><TD>Stained for 1min with 6ml 100mM TRIS (pH 8.5), 30µl Luminol (22.2ng/500µl DMSO), 15µl p-Coumaric acid (7.4ng/500µl DMSO) and 10µl 30% H<SUB>2</SUB>O<SUB>2</SUB></TD></TR><TR><TD>10-13-16</TD><TD>Wiebke, Kim, Sabine, Louise</TD><TD>SDS-Gel</TD><TD>TEV digested samples (10-12-16) were mixed 1:1 with Lämmli and incubated at 80°C for 10min.</TD></TR><TR><TD>Gels (10%) ran for 50min at 150V in TANK buffer.</TD></TR><TR><TD>Stability test of circularised TALes</TD><TD>Purified protein samples (10-13-16), 1.1.1 (E5 + E6) and 2.1.1 (E5 + E6), were set to incubate for  0, 15 min, 5 h and 20 h at 4°C, RT and 37 °C.</TD></TR><TR><TD>In addition, one set of samples was frozen at –20 °C and thawed at RT for 30min each for four continuous times. Each time a fraction of 10 µl was stored at -20 °C for analysis.</TD></TR><TR><TD>Western Blot and Immunostain</TD><TD>Used Towbin-Buffer (25 mM Tris, 192 mM glycine, 20% (v/v) methanol (pH 8.3))</TD></TR><TR><TD>Blotted at 25V for 40min in fastBlot Trans Blot Turbo</TD></TR><TR><TD>Incubated for 1h in blocking solution (1x PBST with 1x RotiBlock, Roth)</TD></TR><TR><TD>Washed with PBS</TD></TR><TR><TD>Incubated for 3h at RT with 1st antibody (2 times Flag antibody, to times Strep antibody at TEV digest Western blot)</TD></TR><TR><TD>10-14-16</TD><TD>Wiebke, Kim, Sabine</TD><TD>Continue Immunostain</TD><TD>1stAntibody removed by washing in PBS</TD></TR><TR><TD>Inbubation with 2nd antibody</TD></TR><TR><TD>2nd Antibody removed by washing in PBS</TD></TR><TR><TD>Stained for 1 min with 6 ml 100mM TRIS (pH 8,5), 30µl Luminol (22,2 ng/500 µl DMSO), 15 µl p-Coumaric acid (7,4 ng/500 µl DMSO) and 10 µl 30% H<SUB>2</SUB>O<SUB>2</SUB></TD></TR><TR><TD>Stability test of circularised TALes</TD><TD>Purified protein samples (10-13-16 and 10-14-16) of Origami with DTT were set to incubate for 6 h, 48 h ... at 4 °C, RT and 37 °C.</TD></TR><TR><TD>In addition, one set of samples was frozen at –20 °C and thawed at RT for 30 min each for four continuous times. Each time a fraction of 10 µl was stored at -20 °C for analysis.</TD></TR><TR><TD>Using samples E1 – E6 from 10-13-16 according to ProTEV plus protocol (Promega)</TD></TR><TR><TD>Samples were taken after 0, 1, 2, 4 and 6h and stored at -20°C</TD></TR><TR><TD>10-16-16</TD><TD>Wiebke</TD><TD>stability Test </TD><TD>Probe at 4 pm (68 h for probes without DTT and 54 h for the probes with DTT)</TD></TR><TR><TD>10-17-16</TD><TD>Wiebke, Louise, Sabine, Theresa, Kim</TD><TD>SDS-gels</TD><TD>Used 8 SDS Gels for the probes from the two stability tests with and without DTT. One gel was used for the TEV digest.</TD></TR><TR><TD>Gels run for 50 min at 150 V.</TD></TR><TR><TD>Western blot </TD><TD>Used Towbin-Buffer (25 mM Tris, 192 mM glycine, 20% (v/v) methanol (pH 8.3))</TD></TR><TR><TD>Blotted at 25 V for 40min in fastBlot Trans Blot Turbo</TD></TR><TR><TD>Incubated for 1h in blocking solution (1x PBST with 1x RotiBlock, Roth)</TD></TR><TR><TD>Washed with PBS</TD></TR><TR><TD>Immunostain </TD><TD>Incubated Membrane 1-6 for 2 h at RT with 1st antibody and membrane 7-9 over night at 4 °C </TD></TR><TR><TD>1stAntibody removed by washing in PBS</TD></TR><TR><TD>Inbubation with 2nd antibody for 1 h</TD></TR><TR><TD>2nd Antibody removed by washing in PBS</TD></TR><TR><TD>Stained for 1 min with 6 ml 100mM TRIS (pH 8,5), 30 µl Luminol (22,2 ng/500 µl DMSO), 15 µl p-Coumaric acid (7,4 ng/500 µl DMSO) and 10 µl 30% H<SUB>2</SUB>O<SUB>2</SUB>/td&gt;
  </TD></TR><TR><TD>10-18-16</TD><TD>Kim</TD><TD>Continued immunostain from 10-17-16</TD><TD>1stAntibody removed by washing in PBS</TD></TR><TR><TD>Inbubation with 2nd antibody for 1 h</TD></TR><TR><TD>2nd antibody removed by washing in PBS</TD></TR><TR><TD>Stained for 1 min with 6 ml 100 mM TRIS (pH 8,5), 30 µl Luminol (22,2 ng/500 µl DMSO), 15 µl p-Coumaric acid (7,4 ng/500 µl DMSO) and 10 µl 30% H<SUB>2</SUB>O<SUB>2</SUB></TD></TR></TBODY></TABLE><H2 id="protocols">Protocols</H2><UL><LI><A href="https://static.igem.org/mediawiki/2016/7/7b/T--Hannover--Protocol-iGEM-TALE-in-BL21.pdf">Protocol TALE in BL21</A></LI><LI><A href="https://static.igem.org/mediawiki/2016/2/27/T--Hannover--Protocol-iGEM-TALE-in-Origami.pdf">Protocol TALE in Origami</A></LI><LI><A href="https://static.igem.org/mediawiki/2016/c/c2/T--Hannover--Protocol-iGEM-TALE-with-GFP.pdf">Protocol TALE with GFP</A></LI></UL></DIV></DIV></DIV><FOOTER class="container-fluid text-center"><DIV style="padding:5px;"><H5><STRONG>Sponsors</STRONG></H5><P><SMALL><STRONG>Our project would not have been possible without financial support from multiple sponsors and supporters.</STRONG></SMALL></P></DIV></FOOTER></DIV></DIV></DIV></DIV></DIV></BODY></HTML>