Team:SYSU-CHINA/Notebook/ProtocolsAndMethods
menu
Overview
Inspirations
Cyclebow target
Cyclebow design
Project
Circuit construction
Tests on cells
Cyclebow construction
Parts
Future works
Results
Notes
Protocols and methods
References
Notebook
Model target
Model design
Model
Modelling
Overview
Stem cell therapy survey
CCiC
High School Camp
Bio-festivals
Collaborations
Human Practice
Interlab
Innovative BioBrick construction
Measurement
Medal requirements
Safety
Judging
Team members
Attributions
Team
SYSU-CHINA
Project
Overview
Background
Cyclebow target
Cyclebow design
Results
Circuit construction
Cyclebow construction
Parts
Notebook
Notes
Protocols and methods
References
Modelling
Model target
Model design
Model
Human Practice
Overview
CCiC
High School Camp
Bio-festival
Collaborations
Measurement
Interlab
Innovation BioBrick construction
Judging
Medal requirements
Safety
Team
Team members
Attributions
MOLECULAR CLONING
REVERSE TRANSCRIPTION PCR (RT-PCR)
WESTERN BLOT
MICROPLATE READER (INTERLAB)
CYTOMETRY FLOW ANALYSIS
DAILY CELL TREATMENTS
TRANSIENT TRANSFECTION
LENTIVIRUS TRANSFECTION AND INFECTION
CELL CYCLE CONFIRMATION
SNAP TAG BLOCKING AND DYEING
Protocols
MOLECULAR CLONING
Preparation of Competent E. Coli Cells Using CaCl2
Take one colony and start a 5 ml overnight culture at 37°C, with shaking.
Dilute the overnight culture 1:100 into 50 ml SOB medium.
Grow culture at 37°C with shaking to an OD600=0.4.
Let the culture sit on ice for ~15 min, swirling occasionally.
Pour the culture into a 50 ml Falcon™ tube.
Centrifuge at 3500 rpm for 5 min at 4°C.
Remove as much as possible of the supernatant without disturbing the pellet.
Resuspend the pellet in 100 μL ice-cold 0.1 M CaCl2 with the help of a sterilized loop.
Add 15 ml ice-cold 0.1 M CaCl2. Mix gently by pipetting up and down a few times.
Incubate the cells on ice for 30 min.
Pellet the cells again at 3500 rpm for 5 min at 4°C.
Resuspend the cells in 2 ml ice-cold 0.1 M CaCl2/20% glycerol.
Incubate for 45 min on ice.
Aliquot carefully in 50 μL amounts to chilled 1.5 ml tubes.
Snap freeze in liquid nitrogen any tubes that will not be used for transformation within a few hours.
Store at −80°C.
Plasmid Preparations
OMEGA Plasmid Mini Kit I (Cat. D6943-02). and E.Z.N.A. Endo-free Plasmid Mini Kit Ⅰ(Cat. D6948-02). Protocol refers to the instruction of the kit.
Polymerase Chain Reaction (PCR)
Prepare a mixture of:
DNA template, 100 μg
2 oligodeoxyribonucleotide primers, 0.4 μl *2
10X dNTPs, 2 μl
10X PCR Buffer, 2 μl
rTaq polymerase, 0.1 μl
ddH2O, up to 20 μl
Set PCR program based on melting temperature, fragment length and polymerase type.
Note: We also used Takara PrimeSTAR Max/GXL Mixture. The protocol refers to the instruction of the mixture.
Digestion
Make three mixes: each contains 500 ng of one of the three plasmids and ddH2O to 43 μL.
To each mix, add 5 μL of 10x reaction buffer for restriction enzymes.
Add 1 μL each of the appropriate endonucleases (two per tube) according to Fig. 25 to give a final volume of 50 μL.
Tap on the tubes to mix. If necessary, centrifuge for a few seconds to spin down the liquid.
Incubate at 37°C for 30 min.
Heat-inactivate the enzymes by incubating at 80°C for 20 min.
Gel Analysis
Close the ends of the gel tray.
Insert the comb into the gel tray at one end ~1 cm from the edge.
For a 1% 50 ml agarose gel, weigh 0.5 g of agarose in a 100 ml conical flask. Add 50 ml 1x TBE buffer.
To dissolve the agarose in the buffer, swirl to mix and microwave for a few minutes taking care not to boil the solution out of the flask. Remove the flask occasionally and check whether the agarose has dissolved completely.
Let the agarose solution cool down. Once the solution is touchable, add the DNA stain. Check the stock concentration and add the appropriate amount to give the desired final concentration.
Pour the gel solution into the gel tray. Remove any air bubbles with a pipette tip. Put in comb.
Wait for the gel to solidify while cooling down to room temperature.
Release the gel tray from the tape or casting stand. Place the gel tray into the buffer chamber and remove the comb carefully.
Add 1x TBE buffer until the gel is completely covered.
Take part of DNA samples and mix with loading dye.
Load the size marker mixed in 1x loading dye into a well.
Load samples into the other wells while writing down which lanes have which samples.
Put the lid onto the buffer chamber and connect it to the power supply.
Run the gel at 120 V for 30 min.
Stop the run and bring the gel to a UV table to visualize the gel bands.
Take a picture of the gel.
Gel Extracion
OMEGA Gel Extract Kit (Cat. D2500-02) is used for gel extraction, protocol refers to instruction of the kit.
DNA Clean-up
MicroElute® DNA Clean-Up Kit (Cat. D 6296-02) is used for DNA clean-up, protocol refers to instruction of the kit.
Ligation
Add 2 μL (20 ng) of digestion mixtures to 11 μL of water.
Add 2 μL 10x reaction buffer for T4 DNA ligase.
Add 1 μL of T4 DNA ligase to give a final volume of 20 μL.
Incubate at room temperature (~22°C) for 30 min.
Heat-inactivate the enzymes by heating at 80°C for 20 min.
Cell Glycerol Stock
Mix 600 μL of an overnight culture with 400 μL of 50% glycerol (to give 20% glycerol final).
Place in the −80°C freezer.
Transformation
Turn on a water bath or heating block to 42°C.
Thaw competent cells on ice for 15 min.
Add 5 μL of ligation reaction mixture or controls above to 50 μL of competent cells.
Incubate for 30 min on ice.
Heat shock for 45 s at 42°C.
Incubate for 5 min on ice.
Add 950 μL of SOB media (pre-heated to 37°C).
Incubate for 1–1.5 hr at 37°C, with occasional gentle mixing by inversion of the tubes.
For positive controls, mix gently and plate 100 μL only (=1/10th) on an agar plate containing the appropriate antibiotic as in Step 12.
Spin cells down from remaining 900 μL at 4000 rpm for 5 min.
Discard all but 100 μL of the supernatant and resuspend the pellet in the remaining 100 μL.
Spread the remaining suspension on an agar plate containing the appropriate antibiotic as follows:
Dip the spreader into 95% ethanol.
Put it into the flame for a second.
Let the ethanol burn off outside the flame.
Spread the bacterial suspension evenly out on an agar plate. Continue until all the inoculum has gone into the agar.
Put the plates at 37°C overnight.
REVERSE TRANSCRIPTION PCR (RT-PCR)
RNA Extraction From Cultured Cells
Wash cells with PBS. Prepare cell suspension using Trypsin-EDTA.
Centrifuge at 1,000 rpm for 2 min, remove supernatant.
Add 1ml Trizol, mix by pipetting up and down a few times.
Add 0.2ml of chloroform, shake vigorously then allow to stand for a few minutes until phase start to separate.
Centrifuge at 13,000 g for 15min at 4°C.
Transfer the colorless upper phase to a new clean tube, avoiding the white interphase.
Add 0.6ml of isopropanol, mix by inverting the tube several times gently.
Centrifuge at 13,000 g for 10min at 4°C.
Remove supernatant and wash RNA by adding 0.7ml of 75% ethanol.
Repeat step 8 and 9.
Centrifuge  at  13,000 g  for 10min at 4°C, remove  supernatant  and  invert  the  tube  on  a  clean kimwipe.
Wait for pellet to dry.
Resuspend pellet with DEPC water.
Reverse Transcription
PrimeScript™ 1st Strand cDNA  Synthesis Kit (Cat. 6110A) is use d for reverse  transcription. Protocol refers to instruction of the kit.
WESTERN BLOT
Sample Lysis
Place the cell culture dish on ice and wash the cells with ice-cold PBS.
Aspirate the  PBS, then  add  ice-cold lysis  buffer (1 mL per 107 cells/100 mm dish/150 cm2 flask; 0.5 mL per 5x106 cells/60 mm dish/75 cm2 flask).
Scrape  adherent  cells off  the  dish  using a cold plastic cell scraper, then gently  transfer  the cell suspension into a pre-cooled  microcentrifuge tube. Alternatively cells can be trypsinized and  washed with PBS prior to resuspension in lysis buffer in a microcentrifuge tube.
Maintain constant agitation for 30 min at 4°C.
Centrifuge in a  microcentrifuge at 4°C. You may  have to  vary the  centrifugation  force  and  time depending on the cell type; a guideline  is  20  min at 12,000 rpm but this  must be determined for your experiment (leukocytes need very light centrifugation).
Gently remove the tubes from the centrifuge and place on ice, aspirate the supernatant  and place in a fresh tube kept on ice, and discard the pellet.
Sample preparation
Remove a small  volume of lysate to  perform a protein  quantification assay.Determine the protein concentration for each cell lysate.
Determine how much protein to load and add an equal volume 2X Laemmli sample buffer.
Boil each cell lysate in  sample buffer at 100°C for  5 min. Lysates  can  be  aliquoted  and   stored  at -20°C for future use.
Loading and running the gel
Load equal amounts of protein into the wells of the  SDS-PAGE gel, along  with  molecular  weight marker. Load 20–30 μg of total protein from cell lysate or tissue homogenate, or 10–100 ng of purified protein.
Run the gel for 1–2 h at 100 V. (The time and voltage may require optimization.)
The gel percentage required is dependent on the size of your protein of interest:
Protein size
Gel percentage
4–40 kDa
20%
12–45 kDa
15%
10–70 kDa
12.5%
15–100 kDa
10%
25–100 kDa
8%
Transferring the protein from the gel to the membrane
The membrane can be either nitrocellulose or PVDF. Activate PVDF with methanol for 1 min and rinse with transfer buffer before preparing the stack. The time and voltage of transfer may require some optimization.
Antibody staining
Block the membrane for 1 h at room temperature or overnight at 4°C using blocking buffer.
Incubate the membrane with appropriate dilutions of primary antibody in blocking buffer. We recommend overnight incubation at 4°C; other conditions can be optimized.
Wash the membrane in three washes of TBST, 5 min each.
Incubate the membrane with the recommended dilution of conjugated secondary antibody in blocking buffer at room temperature for 1 h.
Wash the membrane in three washes of TBST, 5 min each.
For signal development, follow the kit manufacturer’s recommendations. Remove excess reagent and cover the membrane in transparent plastic wrap.
Acquire image using darkroom development techniques for chemiluminescence, or normal image scanning methods for colorimetric detection.
MICROPLATE READER (INTERLAB)
OD600 Reference Point
Turn off the path-length correction.
Self-testing of instrument.
Prepare your 96 well plate.
Add 100 μl LUDOX 100 % into wells A1, B1, C1, D1.
Add 100 μl of H2O into A2, B2, C2, D2 .
Measure absorbance 600 nm of all samples in all standard measurement modes in instrument.
Import data into "Abs600" blue cells in provided Excel calibration sheet.(Flashes of per well: 10, Orbit averaging: not applied to our instrument, temperature: 37°C)
FITC calibration
Turn off the path-length correction.
Self-testing of instrument.
Set the gain to 60%.
Set the excitation wavelength to 480nm.
Set the emission wavelength to 509nm.
Set the model to top optic fluorescence reading.
Prepare your 96 well plate.
Spin down FITC stock tube to make sure pellet is at the bottom of tube.
Prepare 2x FITC stock solution (500 μM) by resuspending FITC in 1ml of 1x Phosphate Buffer Saline (PBS).
Incubate the solution at 42°C for 4 hours. Properly dissolved FITC.
(To check this after the incubation period pipetted up and down – if any particulates are visible in the pipette tip continue to incubate overnight.)
Dilute the 2x FITC stock solution in half to make a 1x FITC solution (final concentration is 250 μM).
Add 100 μl of PBS into wells A2, B2, C2, D2....A12, B12, C12, D12.
Add 200 μl of FITC stock solution into A1, B1, C1, D1.
Transfer 100 μl of FITC stock solution from A1 into A2.
Mix A2 by pipetting up and down 3x and transfer 100 μl into A3.
Mix A3 by pipetting up and down 3x and transfer 100 μl into A4.
Mix A4 by pipetting up and down 3x and transfer 100 μl into A5.
Mix A5 by pipetting up and down 3x and transfer 100 μl into A6.
Mix A6 by pipetting up and down 3x and transfer 100 μl into A7.
Mix A7 by pipetting up and down 3x and transfer 100 μl into A8.
Mix A8 by pipetting up and down 3x and transfer 100 μl into A9.
Mix A9 by pipetting up and down 3x and transfer 100 μl into A10.
Mix A10 by pipetting up and down 3x and transfer 100 μl into A11.
Mix A11 by pipetting up and down 3x and transfer 100 μl into liquid waste.
(TAKE CARE NOT TO CONTINUE SERIAL DILUTION INTO COLUMN 12.)
Repeat dilution series for rows B, C, D.
Measure fluorescence of all samples in all standard measurement modes in instrument.
Measure fluorescence of all of your samples.
Import data into "Fluorescence" blue cells in provided Excel calibration sheet.
(Flashes of per well: 10, Orbit averaging: not applied to our instrument, temperature: 37 ℃)
CYTOMETRY FLOW ANALYSIS
Sample preparation
Harvest the cells in the appropriate manner and wash in PBS.
Fix in cold 70% ethanol. Add drop wise to the pellet while vortexing. This should ensure fixation of all cells and minimize clumping.
Fix overnight at 4°C.
Wash 2 X in PBS. Spin at 850 g in a centrifuge and be careful to avoid cell loss when discarding the supernatant especially after spinning out of ethanol.
Treat the cells with ribonuclease. Add 50 µl of a 100 µg/ml sock of RNase. This will ensure only DNA, not RNA, is stained.
Add 200 µl PI (from 50 µg/ml stock solution).
Results analysis
Measure the forward scatter (FS) and side scatter (SS) to identify single cells.
Pulse processing is used to exclude cell doublets from the analysis. This can be achieved either by using pulse area vs. pulse width or pulse area vs. pulse height depending on the type of cytometer.
PI has a maximum emission of 605 nm so can be measured with a suitable bandpass filter.
DAILY CELL TREATMENTS
Subculture and Passage
Remove all medium from culture with a sterile Pasteur pipet. Wash adhering cell monolayer once or twice with a small volume of 37°C PBS to remove any residual FBS that may inhibit the action of trypsin.
Add enough 37°C trypsin/EDTA solution to culture to cover adhering cell layer.
Place plate on a 37°C warming tray 1 to 2 min. Tap bottom of plate on the countertop to dislodge cells. Check culture with an inverted microscope to be sure that cells are rounded up and detached from the surface.
Add isometric 37°C complete medium. Draw cell suspension into a Pasteur pipet and rinse cell layer two or three times to dissociate cells and to dislodge any remaining adherent cells. As soon as cells are detached, add serum or medium containing serum to inhibit further trypsin activity that might damage cells.
Add one quarter volume of cell suspension to fresh plates or flasks that have been appropriately labeled.
Add 4 ml fresh medium to each new culture. Incubate in a humidified 37°C, 5% CO2 incubator.
If necessary, feed subconfluent cultures after 1 or 2 days by removing old medium and adding fresh 37°C medium.
Passage secondary culture when it becomes confluent by repeating steps 1 to 7, and continue to passage as necessary.
Freezing cells
Trypsinize cells from plate.
Transfer cell suspension to a sterile centrifuge tube and add 1 ml complete medium with serum. Centrifuge 5 min at 200 × g, room temperature.
Remove supernatant and add 1 ml of 4°C freezing medium. Resuspend pellet.
Pipet 1 ml aliquots of cell suspension into labeled 2-ml cryovials. Tighten caps on vials.
Place vials 1 hour to overnight in a −70°C freezer, then transfer to liquid nitrogen storage freezer.
TRANSIENT TRANSFECTION
Cell preparation
One day before transfection, plate 0.5-2 x 105 cells in 500 μl of growth medium without antibiotics so that cells will be 70-90% confluent at the time of transfection.
Transfection mix
   For each transfection sample, prepare complexes as follows:
Dilute DNA in 50 μl of Opti-MEM® I Reduced Serum Medium without serum (or other medium without serum). Mix gently.
Mix Lipofectamine® 2000 gently before use, then dilute the appropriate amount in 50 μl of Opti-MEM® I Medium. Incubate for 5 minutes at room temperature. Note: Proceed to Step c within 25 minutes.
After the 5 minute incubation, combine the diluted DNA with diluted Lipofectamine® 2000 (total volume = 100 μl). Mix gently and incubate for 20 minutes at room temperature (solution may appear cloudy). Note: Complexes are stable for 6 hours at room temperature.
Transfection
Add the 100 μl of complexes to each well containing cells and medium. Mix gently by rocking the plate back and forth.
Incubate cells at 37°C in a CO2 incubator for 18-48 hours prior to testing for transgene expression. Medium may be changed after 6-8 hours.
LENTIVIRUS TRANSFECTION AND INFECTION
Cell preparation
Plate 0.5-2 x 105 cells in 500 μl of growth medium without antibiotics so that cells will be 40-70% confluent at the time of transfection.
Transfection mix
Cell cycle determination
In each sterile1. 5 ml tube, dilute 500 ng pMD2G, 1 μg pSPA, and 2μg of pLenti expression plasmid DNA in 200 μ l of opti-MEM. Mix gently.
In other sterile 1.5 ml tube, dilute 2 μl Lipofectamine™ 2000 (mix gently before use) in 200μ l of opti-MEM. Mix gently and incubate for 5 minutes at room temperature.
After incubation, combine the diluted DNA (Step a) with the diluted Lipofectamine™ 2000 (Step b). Mix gently.
Incubate for 15 minutes at room temperature to allow the DNALipo2000 complexes to form.
Add the complex to each 293T cell plate well.
Remove and discard the medium and replace with 2ml DMEM. Incubate cells 48hours at37°C in a humidified 5% CO2 incubator.
Cell cultivation
Set up the target cell line in target cell medium to 6 well plate so that they will be 30% confluent on the next day.
Post-transfection
Harvest virus-containing supernatants and filter the viral supernatants through a  0.45 μm filter in 15 ml sterile tube.
Infect target cells: 1 volume of DMEM (2ml) and 1 volume of filtered  virus-containing supernatants (2ml), with 4 μl polybrene (8mg/ml).
Select cells using drug.
CELL CYCLE CONFIRMATION
Synchronizing cells
Trypsinize cells and count the number of cells in suspension. Plate growing cells at 30% to 50% confluence in two 6 well-plate after incubate 24 hours at 37°C.
Remove medium with a pipet and replate with 2 ml prewarmed DMEM/thymidine and incubate 14 hours at 37°C.
Remove the thymidine-containing medium with a pipet and rinse the dishes twice with 2ml DMEM each time.
After 8 hours, remove medium with a pipet and replate with 2 ml prewarmed DMEM/thymidine and incubate 14 hours at 37°C.
Remove the thymidine-containing medium with a pipet and rinse the dishes twice with 2ml DMEM each time.
Harvest cells every 2 hours.
Staining cells with PI
Centrifugal collection of cells , remove the supernatants and wash cells two times with 60μl pre cold PBS .
Adding the 60μl resuspension into 140μl pre cold 70% ethanol. Mix gently. And store at 4°C overnight.
Centrifugal collection of cells , remove the supernatants and wash cells with 500μl PBS.
Resuspend Cells with 200μl PBS. Add 2μl 10mg/mL RNase A and incubate at 37 °C  for 30 minutes. e. Add 2μl 1mg/ml PI, and stain for 5 minutes away from the light.
SNAP TAG BLOCKING AND DYEING
Preparation
Dissolve one vial of SNAP-tag substrate (50 nmol) in 50 µl of DMSO to yield a labeling stock solution of 1 mM SNAP-tag substrate, and one vial of SNAP-Cell Block (100 nmol) by adding 50 µl of DMSO to give a solution of 2 mM SNAP-Cell Block. Dilute the stock solution 1:200 in medium when using.
For labeling sample: replace the medium on the cells with the SNAP-tag labeling medium and incubate at 37°C, 5% CO2 for 30 minutes. Then wash the cells three times with tissue culture medium with serum and incubate in fresh medium for 30 minutes. Replace the medium one more time to remove unreacted SNAP-tag substrate that has diffused out of the cells.
For block control: replace the medium on one sample of cells with the blocking medium. These are your Blocked Cells. Incubate cell samples for 20 minutes.
For SNAP + Block control: Block cells for 20 minutes in advance before SNAP labeling.
Image the cells using an appropriate filter set. SNAP-tag fusion proteins labeled with SNAP-Cell 505 should have an excitation maximum at 504 nm and an emission maximum at 532 nm, and can be imaged with standard fluorescence filter sets.
Collect cells with trypsin and fix cells with 75% ethanol. Store at 4°C for flow cytometry analysis.
Measuring cell fluorescence by flow cytometry
Set out and adjust cytometer for excitation with blue light and detection of PI emission at red wavelengths.
Measure cell fluorescence by flow cytometry. Use pulse-width/pulse-area signal to discriminate between G2/M cells and the cell doublets, and gate out the latter. Analyze the data using DNA content histogram deconvolution software.
SYSU-CHINA   Sun Yat-sen University
E-mail：sysuchina@outlook.com
Address: No. 135, Xingang Xi Road, Guangzhou, 510275, P. R. China
