<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_Sydney_Australia_Notebook skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:Sydney Australia/Notebook</SPAN></H1><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><DIV id="menuContainer"><DIV id="menuContainer2"><NAV><UL><LI><A href="https://2015.igem.org/Team:Sydney_Australia">Home</A></LI><LI><A href="https://2015.igem.org/Team:Sydney_Australia/achievements">Achievements</A></LI><LI><A href="#">Project</A></LI><LI><A href="https://2015.igem.org/Team:Sydney_Australia/project_overview">Overview</A></LI><LI><A href="https://2015.igem.org/Team:Sydney_Australia/Description">Description</A></LI><LI><A href="https://2015.igem.org/Team:Sydney_Australia/Experiments">Protocols</A></LI><LI><A href="https://2015.igem.org/Team:Sydney_Australia/goldengate">Golden Gate</A></LI><LI><A href="https://2015.igem.org/Team:Sydney_Australia/Results">Results</A></LI><LI><A href="https://2015.igem.org/Team:Sydney_Australia/Notebook">Notebook</A></LI></UL><LI><A href="https://2015.igem.org/Team:Sydney_Australia/Parts">Parts</A></LI><LI><A href="#">Modelling</A></LI><LI><A href="https://2015.igem.org/Team:Sydney_Australia/Modeling">Overview</A></LI><LI><A href="https://2015.igem.org/Team:Sydney_Australia/TransOpt">TransOpt</A></LI><LI><A href="https://2015.igem.org/Team:Sydney_Australia/Modeling/visualisation">Visualisation</A></LI><LI><A href="https://2015.igem.org/Team:Sydney_Australia/Modeling/ElectroStop">ElectroStop</A></LI><LI><A href="https://2015.igem.org/Team:Sydney_Australia/Modeling/Supplementary">Supplementary Material</A></LI><LI><A href="#">Human Practices</A></LI><LI><A href="https://2015.igem.org/Team:Sydney_Australia/Practices">Overview</A></LI><LI><A href="https://2015.igem.org/Team:Sydney_Australia/EthicsMotivation">Motivation &amp; Analysis</A></LI><LI><A href="https://2015.igem.org/Team:Sydney_Australia/Outreach">Outreach</A></LI><LI><A href="https://2015.igem.org/Team:Sydney_Australia/Media">Media</A></LI><LI><A href="https://2015.igem.org/Team:Sydney_Australia/Collaborations">Collaboration</A></LI><LI><A href="https://2015.igem.org/Team:Sydney_Australia/BusinessModel">Business Model</A></LI><LI><A href="https://2015.igem.org/Team:Sydney_Australia/Safety">Safety</A></LI><LI><A href="https://2015.igem.org/Team:Sydney_Australia/Attributions">Attributions</A></LI><LI><A href="https://2015.igem.org/Team:Sydney_Australia/Team">Team</A></LI></NAV></DIV></DIV><DIV id="mainContainer"><DIV id="contentContainer"><P>
Throughout out our project we kept a detailed and through notebook and timeline. We found that having a daily, weekly, and monthly structure greatly helped in our time management. As we are a team in the Southern Hemisphere, we undertook our project during university semesters and thus effective time management was critical in helping us balance university, iGEM, work, and everything else. 
</P><DIV class="center"><DIV class="thumb tnone"><DIV class="thumbinner" style="width:702px;"><DIV class="thumbcaption">Example of our week-by-week schedule</DIV></DIV></DIV></DIV><P>Every day in the lab we would begin by asking ourselves what needed to be done that day and in what order. After every day we evaluated our progress and asked ourselves four questions - 
</P><OL><LI>What did we achieve? </LI><LI>How efficient were we?</LI><LI>What can we improve on?</LI><LI>What is the plan for tomorrow </LI></OL><P>Thus, we knew exactly what experiments needed to be done on each day. This allowed us to work faster and more efficient, it also created an easy transition between the different lab members as one member could pick up where the other left off without having to chase them down to find out the important information about the past experiment.
</P><P>Furthermore, we had weekly meetings and the occassinal skype session. 
</P><CENTER>William Ramsay once said:
<P><I>&quot;Progress is made by trial and failure; the failures are generally a hundred times more numerous than the successesÂ ; yet they are usually left unchronicled&quot;</I></P><P>
This was not the case for our team as we sought to extensively chronicle all the successes and failures. We hope our observations and troubleshooting will help future iGEM teams. 
</P>
See below for a week-by-week account of our work.</CENTER><A class="exp" href="#url" tabindex="1">Week 1: 30th of March - 3rd of April</A><DIV class="exp"><H1>Wet Lab </H1><P>Tuesday</P><UL><LI>Meet up for the first time and were intrdouced to the project. Started to do some research on the project</LI></UL><H1> Modelling</H1><P>Thursday-Friday</P><UL><LI>Researched the possibility of developing a tool for simulation of the translation process. </LI><LI>Harry &amp; Matt discussed this possibility, along with potential areas for improvement.</LI></UL></DIV><A class="exp" href="#url" tabindex="1">Week 2: 6th of April - 10th of April</A><DIV class="exp"><H1>Wet Lab </H1><P>Friday - Sunday </P><UL><LI>Underwent necessary safety training to work independently in the PC2 laboratory</LI></UL><H1>Human Practices </H1><P>Monday - Thursday</P><UL><LI>Started to brainstorm ideas for a human practices project and outreach activities</LI></UL><H1> Modelling</H1><P>Wednesday</P><UL><LI>After continued research over the weekend, met with academic staff in the department to discuss sensible mechanisms that should be included in simulating, and potentially optimising, translation.</LI></UL><P>Thursday-Friday</P><UL><LI>Continued research, focusing on the work of Brakley et al, as well as different aspects of our organisms of interest (and how they compare to yeast).</LI></UL></DIV><A class="exp" href="#url" tabindex="1">Week 3: 13th of April - 17th of April</A><DIV class="exp"><H1>Wet Lab </H1><P>Started designing the G-Blocks for order and began creating electrocompetent cells and various agar plates</P><H1>Human Practices </H1><P>Tuesday - Thursday</P><UL><LI>Developed the Strange Nature question and created the information packages and contacted Rotary clubs</LI></UL><H1> Modelling</H1><P>Wednesday - Friday</P><UL><LI>Begin to develop a translation simulation tool in MATLAB.</LI><LI>Continued review of literature to find appropriate modelling parameters.</LI></UL></DIV><A class="exp" href="#url" tabindex="1">Week 4: 20th of April - 24th of April</A><DIV class="exp"><H1>Wet Lab </H1><P>Continued to work on the G-Blocks with the model created</P><H1>Human Practices </H1><P>Friday - Saturday</P><UL><LI>Developed and worked on the Rotary Presentations</LI></UL><H1> Modelling</H1><P>Monday - Thursday</P><UL><LI>Re-thougth the coding approach being taken, as it may need to be scaled up to treat larger problems in the future. To achieve this we made maximum use of MATLAB's vectorisation to speed up execution. </LI></UL></DIV><A class="exp" href="#url" tabindex="1">Week 5: 27th of April - 1st of May</A><DIV class="exp"><H1>Wet Lab </H1><P>Continued to research and plan the design of the G blocks. Discovered the world of Golden Gate cloning and decided to apply it to our project</P><H1> Modelling</H1><P>Thursday - Sunday</P><UL><LI>Continued to develop aspects of our visualisation tool, now focusing on how we can best display the simulated processes, to give the user some idea of what it is revealing about the cell.</LI></UL></DIV><A class="exp" href="#url" tabindex="1">Week 6: 4th of May - 8th of May</A><DIV class="exp"><H1>Wet Lab </H1><P>Monday - Friday</P><UL><LI>Ordered the G-Blocks and waited for them to arrive</LI></UL><H1>Human Practices </H1><P>Wednesday</P><UL><LI> Discover Uni Day at Sydney University. Our first outreach event went very well!</LI></UL><H1> Modelling</H1><P>Tuesday</P><UL><LI>With the visualisation tool now working, other members of the team test it. This is to improve usability and to reveal any flaws in its operation. A number of suggestions were taken for future improvement</LI></UL><P>Friday</P><UL><LI>Made changes to the program to reflect advice received from all members of the team.</LI></UL></DIV><A class="exp" href="#url" tabindex="2">Week 7: 11th of May - 15th of May</A><DIV class="exp"><H1>Wet Lab</H1><P> Wednesday and Thursday</P><UL><LI>Aim - To replicate the findings of Mai Anh Ly's PhD thesis that showed low levels of expression of the EtnABCD cluster in Pseudomonas Putida KT2440</LI><LI>Result - Confirmed the findings</LI></UL><P>Friday</P><UL><LI>Aim - To transform the pBP-Etn plasmid into pre-made componenet Pseudomonas Putida cells by electroporation</LI><LI>Results - Successful transformation</LI></UL><H1>Human Practices </H1><P>Monday - Thursday</P><UL><LI>Began research on the business proposal and sought contacts in the industry</LI></UL><H1> Modelling</H1><P>Monday-Tuesday</P><UL><LI>Continued testing of visualisation tool, and validation of its capability to demonstrate a wide range of anticipated behaviours.</LI></UL><P>Wednesday - Friday</P><UL><LI>Begun researching translation optimisation processes. A number exist in the literature, as well as via online tools provided by biotech companies.</LI></UL></DIV><A class="exp" href="#url" tabindex="1">Week 8: 18th of May - 22nd of May</A><DIV class="exp"><H1>Wet Lab </H1><P>Monday</P><UL><LI>Aim - Assess if the cells transformed with the wild type EtnABCD cluster have any monooxygenase activity through a Nitrobenzylpyridine assay </LI><LI> Result - Positive test had low (but notable) absorbance compared to control.</LI></UL><H1>Human Practices </H1><P>Thursday - Saturday</P><UL><LI>Continued to work on the business plan and began collating a list of schools to contact for Strange Nature</LI></UL><H1> Modelling</H1><P>Monday - Thursday</P><UL><LI>Continued research of mRNA translation and the factors which effect its performance.</LI><LI>Read literature regarding how different aspects of mRNA sequences are believed to cause certain behaviours during the translation process.</LI></UL><P>Friday</P><UL><LI>Discussed the possibility of making a device which would automatically control electrophoresis experiments.</LI></UL></DIV><A class="exp" href="#url" tabindex="1">Week 9: 25th of May - 29th of May</A><DIV class="exp"><H1>Human Practices </H1><P>Monday</P><UL><LI>Started contacting the media </LI></UL><H1> Modelling</H1><P>Monday - Tuesday</P><UL><LI>Decided to build a device which could control electrophoresis experiments, came up with a preliminary circuit design.</LI></UL><P>Tuesday - Friday</P><UL><LI>Continued to research translation optimisation processes, as well as potentially mathematical approaches to achieve these. </LI><LI>Noticed the work of Spencer et al which presents a new (and so far successful) methodology for predicting translation rates of different codons. </LI></UL><P>Friday</P><UL><LI>Order initial components for electrophoresis device</LI></UL></DIV><A class="exp" href="#url" tabindex="1">Week 10: 1st of June - 5th of June</A><DIV class="exp"><H1>Wet Lab </H1><P>Monday</P><UL><LI>Aim - First PCR of the ordered G-Blocks. </LI><LI> Result - Unsuccessful. High molecular weight smears were present</LI></UL><H1>Human Practices </H1><P>Friday - Sunday </P><UL><LI>Began to write and edit articles for the papers that had agreeded to publish articles</LI></UL><H1> Modelling</H1><P>Tuesday - Thursday</P><UL><LI>Continual research on the work of Spencer et al, as well as investigation of future research which make reference to the initial 2012 paper. </LI></UL></DIV><A class="exp" href="#url" tabindex="1">Week 11: 8th of June - 12th of June</A><DIV class="exp"><H1>Wet Lab </H1><P>Exam time. We took a break from iGEM</P><UL><H1> Modelling</H1><P>Wednesday - Saturday</P><UL><LI>Implemented the standard codon harmonisation methodology using Spencer et al quantisation of rates. This method is based on that proposed by Kudla et al, which has found widespread use, though it makes a wide range of assumptions when quantising translation rates.</LI></UL><A class="exp" href="#url" tabindex="1">Week 12: 14th of June - 19th of June</A><DIV class="exp"><H1> Modelling</H1><P>Monday - Wednesday</P><UL><LI>Testing of the standard codon harmonisation software, comparing results with those produced by various online tools which claim to do the same thing.</LI></UL><P>Thursday</P><UL><LI>Electronic components for the electrophoresis device arrive by mail.</LI></UL></DIV><A class="exp" href="#url" tabindex="1">Week 13: 22nd of June - 26th of June</A><DIV class="exp"><H1>Human Practices </H1><P>Thursday</P><UL><LI>Briefly worked on the business plan</LI></UL><H1> Modelling</H1><P>Monday - Friday</P><UL><LI>Prototype the circuit for use with the electrophoresis device. </LI><LI>Develop code for the device, which by Thursday is working correctly.</LI></UL><P>Wednesday - Friday</P><UL><LI>Continue investigating factors important in mRNA translation.</LI><LI>Research simulated annealing, a statistical approach to optimisation which may be used.</LI></UL></DIV><A class="exp" href="#url" tabindex="1">Week 14: 29th of June - 3rd of July</A><DIV class="exp"><H1>Wet Lab </H1><P>Monday</P><UL><LI>Aim - PCR amplification of pBSIC3 using Phusion and visualization using gel electrophoresis </LI><LI> Result - Several bands present, yet bigger/smaller than expected 2 kb fragment </LI></UL><P> Tuesday</P><UL><LI> Aim - A gel was run with a thermocycling gradient (59.4, 61.3, 63.6, 66.2, 69, 71.6, 74, 76) to determine the best annealing temperature.</LI><LI> Result - The thermocycling gradient showed that 69 was the most suitable temperature. </LI><LI> Aim - PCR amplification (annealing temperature of 69 and 98 cycles) and gel electrophoresis of EtnAB, EtnC, EtnD, Gro1, Gro2 G-blocks using primers iGEM1501 and iGEM1502. The gel was stained with Gel-Red.</LI><LI> Result - The gel electrophoresis showed high molecular weight smears. These were though to be the result of contaminants in the reagent.</LI></UL><P> Wednesday</P><UL><LI> Aim - Second attempt of PCR amplification and gel electrophoresis of pBSIC3 with an annealing temperature of 69 and 35 cycles instead of 98</LI><LI> Result - The gel showed no DNA bands and no smears for all lanes. </LI><LI> Aim - DpnI treatment of pSB1C3 PCR sample (restriction, transformation and column purification)</LI><LI> Result - successful elimination of plasmid using DpnI </LI></UL><P> Thursday </P><UL><LI> Aim - Make up new working stock of primers 1501 and 1502 and the third attempt at running the PCR and gel electrophoresis of  pBSIC3 from 30/6/15 and 1/7/15.</LI><LI> Result - The no-template-control yielded no DNA bands. All other lanes showed smears.</LI></UL><P> Friday </P><UL><LI> Aim - Fourth attempt at PCR amplification and gel electrophoresis of pBSIC3 from the 30/6/15, 1/7/15 and 2/7/15.</LI><LI> Changes to the PCR protocol 
*same conditions for Gro2
*GC buffer instead of HF buffer for Gro2
*Using Taq pol and Taq buffer for Gro2
*1/10 of DNA template
*Shorter primers 
*Same conditions using Abi's G blocks</LI><LI> Results - Large band of correct size was seen with Taq polymerase, using the same primer. Both G-blocks and the primers are working as expected. Next step: eliminate all plasmid, digestions with more DpnI should be performed again with more DpnI</LI></UL><H1>Human Practices </H1><P>Sunday</P><UL><LI>Began developing the activities for APCS</LI></UL><H1> Modelling</H1><P>Monday</P><UL><LI>Test the prototype electrophoresis circuit, now named ElectroStop</LI><LI>ElectroStop demonstrates a capability, after some tuning, to be able to accurately sense the presence of marker dye obstructing its laser.</LI></UL><P>Tuesday - Thursday</P><UL><LI>Adapt the standard codon harmonisation tool to accurately reflect the rate quantisations proposed by Spencer et al, which seems more physically realistic.</LI></UL><P>Thursday - Friday</P><UL><LI>Begin coding a new codon optimisation tool, now named TransOpt, which we hope will optimise many aspects of the translation process.</LI></UL></DIV><A class="exp" href="#url" tabindex="1">Week 15: 6th of July - 10th of July</A><DIV class="exp"><H1>Wet Lab </H1><P>Monday</P><UL><LI>Aim - Testing PCR reactions of gBlocks against varying polymerase activities </LI><LI> Results - Full strength Phusion and Pfu all seemed to produce strong bands </LI></UL><P>Tuesday</P><UL><LI>Aim - PCR of monooxygenase G blocks</LI><LI>Results - Smears and bands of incorrect sizes were observed</LI><LI>Aim - PCR amplification of G blocks with Pfu polymerase</LI><LI>Results - Smears and bands of incorrect sizes were observed</LI></UL><P>Wednesday</P><UL><LI>Aim - Sixth attempt of PCR amplification and gel electrophoresis of EtnAB, EtnC, and EtnD</LI><LI> Results - All G blocks are pure, bands are evident </LI><LI> Aim - Golden Gate assembly of GroEl/ES to pSBIC3 PCR product</LI><LI> Results - No colonies appeared after 20 hours. Colonies appeared in JM109 transformed with GFG of pSBIC3-GroEL/ES</LI></UL><P>Thursday</P><UL><LI>Aim - Repeat PCR from 8/7/15 with a new set of pipettes</LI><LI> Results - Multiple products observed for all lanes including the NTC</LI><LI> Aim - Amplification of pSB1C3 with Phusion</LI><LI> Results - Large clear band was observed around 1/5kb which is the expected size of pCB1C3</LI><LI> Aim - Golden Gate Assembly of GroEL/ES and pUCP24</LI><LI> Results - Cloning was not successful</LI></UL><H1>Human Practices </H1><P>Monday - Thursday</P><UL><LI>Tested out the gels for the APCS and began to create bacterial paintings which could be used at APCS</LI></UL><H1> Modelling</H1><P>Monday - Wednesday</P><UL><LI>Continue developing the code for TransOpt. Now try to make reading of organisms data (e.g. Gene Copy Numbers) as general as possible, such that data from new organisms can be introduced in the future.</LI></UL><P>Thursday - Sunday</P><UL><LI>Design structure for ElectroStop in Solidworks, which will then be 3D Printed.</LI><LI>Order some new electrical components which earlier testing indicated a need for.</LI></UL></DIV><A class="exp" href="#url" tabindex="1">Week 16: 13th of July - 17th of July</A><DIV class="exp"><H1>Wet Lab </H1><P>Friday</P><UL><LI>Aim - Amplification of AE-blue-aac with the primer set used in pervious reactions and a control primer set (igem1501 and igem1502) to determine whether contamination is in the primer</LI><LI> Results - Band of correct size observed for both PCR reactions but there were still smears in all lanes. No difference between control and test PCRs. Cause of smears is unknown </LI></UL><P>Saturday</P><UL><LI>Box PCR for outreach activities</LI></UL><H1>Human Practices </H1><P>Tuesday - Friday</P><UL><LI>Started to wrok on the Strange Nature website and finalised the question. Officially launched Strange Nature</LI></UL><H1> Modelling</H1><P>Monday</P><UL><LI>Testing the visualisation tools: They appear to show scientifically-reasonable results which agree with what we had expected.</LI></UL><P>Tuesday</P><UL><LI>Finalise the design of the ElectroStop structure in Solidworks, ensure measurements match up with all common electrophoresis trays in our lab.</LI></UL><P>Wednesday - Friday</P><UL><LI>Continue development of TransOpt, now designing an energy function for use in the simulated annealing approach.</LI></UL></DIV><A class="exp" href="#url" tabindex="1">Week 17: 20th of July - 24th of July</A><DIV class="exp"><H1>Wet Lab </H1><P>Tuesday</P><UL><LI>Aim - PCR amplification of GroEL/ES and pSB1C3 and gel visualisation</LI><LI> Result - Smears were observed on the gel</LI></UL><P>Wednesday</P><UL><LI>Aim - PCR amplification of GroEL/ES with Phusion</LI><LI> Result - Smears were observed on the gel. Next approach was to try the Pfu</LI></UL><P>Thursday</P><UL><LI>Aim - PCR amplification of GroEL/ES with Pfu</LI><LI> Result - Smears were observed on the gel</LI><LI>Aim - pSB1C3/pUCP24 digestion for use of GG-cloning with Gro</LI><LI> Result - Smears were observed on the gel</LI></UL><P>Friday - Saturday</P><UL><LI>Aim - Digest of pSB1C3 with SpeI/EcoRI and golden gate cloning of pSB1C3/Gro</LI><LI> Result - Colonies appeared which need to be screended and inoculated</LI></UL><H1>Human Practices </H1><P> Monday </P><UL><LI> We attended APCS and spoke to the students about iGEM, synthetic biology, and our project</LI></UL><P>Tuesday</P><UL><LI>We painted on plates with the chromoproteins from the Uppsala team for outreach activities</LI></UL><H1> Modelling</H1><P>Monday</P><UL><LI>Use the Sydney Invention Studio to 3D Print the structure for ElectroStop.</LI><LI>After four hours printing, structural members are finished in VeroWhite plastic, and appear to be manufactured as per design.</LI></UL><P>Wednesday-Sunday</P><UL><LI>Continue development of TransOpt, now focusing on how randomised changes are kept / rejected by the optimisation algorithm.</LI><LI>Extend TransOpt to work with data files from other organisms.</LI><LI>TransOpt's core capabilities finished and working for all organisms we will use in our project.</LI><LI>Utilise TransOpt to produce EtnABCD Sequences for use in main project.</LI></UL></DIV><A class="exp" href="#url" tabindex="1">Week 18: 27th of July - 31st of July</A><DIV class="exp"><H1>Wet Lab </H1><P>Monday</P><UL><LI>Aim - Patch and left junction PCR screening of GroEL/ES transformants followed by gel visualisation</LI><LI> Results - 14 colonies suspected of carrying successful clones, 3 colonies showed clear bands in gel</LI></UL><P>Friday</P><UL><LI>Aim - Colony spanning PCR putative GroEL/ES clones</LI><LI> Results - Cloning was unsuccessful</LI></UL><H1> Modelling</H1><P>Monday</P><UL><LI>New electrical components arrive for ElectroStop.</LI><LI>Testing of new components is successful.</LI></UL><P>Wednesday - Sunday</P><UL><LI>Begin testing TransOpt with a variety of input sequences. </LI><LI>Appears to provide sensible answers, though convergence (due to inefficient programming) is at this stage slow.</LI><LI>Utilise TransOpt to produce fluorescent protein sequences for use in experiments, both via standard codon harmonisation, fast codon utilisation, and the TransOpt method.</LI></UL><P>Friday</P><UL><LI>Solder all electrical components to ElectroStop's protoboard, as prototyping and testing has revealed that the current design functions as necessary.</LI></UL></DIV><A class="exp" href="#url" tabindex="1">Week 19: 3rd of August - 7th of August</A><DIV class="exp"><H1>Wet Lab </H1><P>Monday</P><UL><LI>Aim - Right-junction colony PCR of pSB1C3-Gro JM109 clones</LI><LI> Results - No bands in PCR gel visualisation, therefore the cloning was unsuccessful</LI></UL><P>Wednesday</P><UL><LI>Aim - Digest of pSB1C3 with EcoRI and SpeI to prepare for a second attempt at GG cloning with GroES/EL G blocks</LI><LI> Results - Bands visualized on gel, therefore successful </LI></UL><P>Thursday</P><UL><LI>Aim - GG cloning of pSB1C3, GroEL/ES and transformation</LI><LI> Results - some test colonies were present, so cloning looked to be somewhat successful</LI></UL><P>Friday</P><UL><LI>Aim - colony PCR (left-junction) of pSB1C3-Gro GG clones from yesterday and visualisation</LI><LI> Results - successful: bands as expected </LI></UL><H1>Human Practices </H1><P>Monday</P><UL><LI>Began to look into the LacI co-expression as a solution to the toxicity encountered when having to produce this on an industrial scale</LI></UL><H1> Modelling</H1><P>Monday - Thursday</P><UL><LI>Continue testing of TransOpt, experiment with different values of alpha and beta values to determine how they affect the final outputted result.</LI></UL><P>Thursday - Friday</P><UL><LI>Assemble circuit and 3D printed components to create final prototype of ElectroStop.</LI><LI>Test the ElectroStop device on sample dyes implanted in a gel.</LI><LI>Works effectively on a range of dyes, though purple-tinted ones provide the greatest sensitivity.</LI></UL></DIV><A class="exp" href="#url" tabindex="1">Week 20: 10th of August - 21st of August</A><DIV class="exp"><H1>Wet Lab </H1><P>Monday</P><UL><LI>Aim - spanning colony PCR and broth inoculation of putative GroES/EL clones to confirm assembly and gel visualisation</LI><LI> Result - bands not expected size, therefore deemed inconclusive </LI></UL><P>Tuesday</P><UL><LI>Aim - right junction colony PCR of putative GroEL/ES clones</LI><LI> Result - expected band seen, however as spanning PCR was unsuccessful, cannot confirm Gro assembly. </LI></UL><P>Wednesday</P><UL><LI>Aim - pSB1C3-EtnABCD GG cloning and transformation </LI><LI> Result - some successful colonies  </LI></UL><P>Thursday</P><UL><LI>Aim - spanning colony PCR of putative EtnABCD clones to confirm assembly and gel visualisation </LI><LI> Result - no bands shown in white (putative) colonies so gel inconclusive   </LI></UL><H1>Human Practices </H1><P>Friday</P><UL><LI>Lizzie attended Kambala School and spoke to the girls about iGEM, synthetic biology, and our project. </LI></UL><P> Saturday </P><UL><LI> We attended Science in the Swamp and spoke to over 10 000 about synthetic biology and our project</LI></UL><H1> Modelling</H1><P>Monday - Thursday</P><UL><LI>Continue testing of TransOpt.</LI><LI>Add capability for removal of iGEM restricted sites from the final result, and during the optimisation process.</LI></UL><P>Wednesday - Friday</P><UL><LI>Test ElectroStop with gel runs to ensure switching capability is sufficient.</LI><LI>Success - Appears able to regularly switch the 120-180 V power supplied to the electrophoresis tray.</LI></UL></DIV><A class="exp" href="#url" tabindex="1">Week 21: 24th of August - 28th of August</A><DIV class="exp"><H1>Wet Lab </H1><P>Monday - Wednesday</P><UL><LI>Aim - inoculation, plasmid purification, digestion and visualisation of EtnABCD-pSB1C3 GG cloning </LI><LI> Result - 4/5 clones were identified as positive </LI></UL><P>Thursday</P><UL><LI>Aim - digest using EcoRI and SpeI to confirm insertion of GroES/EL construct followed by gel visualisation </LI><LI> Result - not promising; appears a 400bp fragment inserted </LI></UL><P>Friday</P><UL><LI>Aim - EcoRI-SpeI digestion of EtnABCD-pSB1C3 and pBBR-MCS2 </LI><LI> Result - mix of white, green and mixed green/white large colonies observed </LI></UL><H1>Human Practices </H1><P>Saturday</P><UL><LI>We had a stall at the University of Sydney Open Day and spoke to prospective students about iGEM and out sompetition</LI></UL><H1> Modelling</H1><P>Tuesday - Friday</P><UL><LI>Continue testing of TransOpt</LI><LI>Add capability for graphic results as they are generated.</LI><LI>Add capability for converting output back to base letters (A,T,C,G) and storing  in an easily retrievable text file.</LI><LI>Add final check after optimisation is complete, to ensure that generated sequence is synonymous to original sequence.</LI></UL></DIV><A class="exp" href="#url" tabindex="1">Week 22: 31st of August - 4th of September</A><DIV class="exp"><H1>Wet Lab </H1><P>Wednesday</P><UL><LI>Aim - plasmid purification, digest, gel visualization of EtnABCD-pBBR with EcoRI &amp; EcoRI+SpeI</LI><LI> Result - some clones showed the correct insert </LI><LI> Aim - transformation of positive EtnABCD-pBBR clones into Pseudomonas putida</LI><LI> Result - no test colonies were observed so transformation deemed unsuccessful</LI></UL><P>Thursday - Friday</P><UL><LI>Aim - digestion and ligation of E. coli genomic GroEL/ES into pUCP24/pUS44/pSB1C3 and transformation into JM109</LI><LI> Result - all of these ligations were successful  </LI></UL><P>Wednesday - Thursday</P><UL><LI>Aim - NBP assay to investigate monooxygenase activity in putative clones &amp; replicate </LI><LI> Result - no change in absorbance detected, thus unsuccessful </LI><LI> Aim - digestion of pUS44-lacI with SalI, ligation and transformation into E. coli </LI><LI> Result - some colonies on test plate, therefore successful</LI></UL><H1>Human Practices </H1><P>Wednesday</P><UL><LI>We had a stall at the Investig8 Uni Day and taught the students about synthetic biology and our project</LI></UL><H1> Modelling</H1><P>Monday - Tuesday</P><UL><LI>Collect final results from visualisation tool and form videos / graphs for use on Wiki.</LI><LI>Use visualisation tool on some sequences generated by TransOpt.</LI><LI>Compiling final software for visualisation tool.</LI></UL><P>Tuesday - Thursday</P><UL><LI>Complete final testing of TransOpt.</LI><LI>Begin creating a simplified final version for upload.</LI></UL><P>Tuesday - Friday</P><UL><LI>Experimentally deploy ElectroStop device for usage.</LI></UL></DIV><A class="exp" href="#url" tabindex="1">Week 23: 7th of September - 11th of September</A><DIV class="exp"><H1>Wet Lab </H1><P>Monday</P><UL><LI>Aim - preparing pSB1C3-EtnACBD clones for sequencing efforts</LI><LI> Results - yield and purity were sufficient to go ahead </LI><LI> Aim - PCR screening of putative pSB1C3-lacI clones using left/right junction primers and gel visualisation </LI> Results - a number of potential insert-containing clones. Go on to incoulate </UL><P>Tuesday</P><UL><LI> Aim - isolate, restriction digest and gel visualization of pSB-lacI clones to confirm they contain insert</LI><LI> Results - one clone had correct insert </LI></UL><P>Wednesday</P><UL><LI> Aim - transformation of pSB-lacI clones into chemically competent E. coli JM109-pBBR1MCS2 cells</LI><LI> Results - yesterdayâs successful clone transformed and plated; odd growth on different selective media </LI></UL><P>Thursday</P><UL><LI> Aim - transformation of pBBR-Etn into new electrocompetent Pseudomonas cells</LI><LI> Results - unsuccessful </LI></UL><P>Friday</P><UL><LI> Aim - pSB1C3-Etn and pBBRMCS2 digestion, ligation and transformation into Pseudo K2440 for Etn-pBBR</LI><LI> Results - two clones showed good growth  </LI></UL><P>Saturday</P><UL><LI> Aim - transformation of Pseudo KT440 with lacI-pUS44</LI><LI> Results - a few dozen colonies of pUS44-lacI </LI></UL><H1>Human Practices </H1><P>Friday</P><UL><LI>Contacted more papers</LI></UL><H1> Modelling</H1><P>Monday - Wednesday</P><UL><LI>Final compilation of TransOpt script for upload to Wiki.</LI><LI>Analysis of TransOpt experimental on fluorescent protein.</LI></UL><P>Thursday</P><UL><LI>Use ElectroStop in lab.</LI></UL></DIV><A class="exp" href="#url" tabindex="1">Week 24: 14th of September - 18th of September</A><DIV class="exp"><H1>Wet Lab </H1><P>Tuesday</P><UL><LI>Aim - spanning colony PCR of colonies from Saturdayâs pUS44-lacI clones</LI><LI>Results - showed correct insert</LI><LI> Aim - Pseudo-pUS44-lacI competent cells and colony PCR of successful colonies</LI><LI> Results - 3 colonies showed to be successful </LI></UL><P>Thursday</P><UL><LI>Aim - spanning colony PCR of colonies from Saturdayâs pUS44-lacI clones</LI><LI>Results - a dark purple colour was produced, indicating a positive result</LI><LI> '''Aim - parts submission (lacI, BsFP, etnABCD all in pSB1C3)''' </LI><LI> '''Results - SUCCESS!''' </LI></UL><H1> Modelling</H1><P>Monday - Friday</P><UL><LI>Finished analysing experimental results from fluroprotein: Found that TransOpt optimised sequence showed very poor fluroescence.</LI><LI>Discuss and try to hypothesise causes for TransOpt sub-par performance on this particular protein</LI></UL></DIV></UL></DIV></DIV></DIV></DIV></DIV></DIV></DIV></BODY></HTML>