Team:Sydney Australia/Modeling/Overview
Home
Achievements
Project
Overview
Description
Protocols
Golden Gate
Results
Notebook
Parts
Modelling
Overview
TransOpt
Visualisation
ElectroStop
Supplementary Material
Human Practices
Overview
Motivation & Analysis
Outreach
Media
Collaboration
Business Model
Safety
Attributions
Team
Contents
1 HarOpt: A Codon Harmonisation Model
1.1 Model
1.2 Experimental Validation
1.3 Results & Discussions
2 Supplementary Model: Translation Kinetics
2.1 Model
2.2 Results & Discussions
HarOpt: A Codon Harmonisation Model
It was long thought that the codon redundancies in the genetic code had no evolutionary significance. However, in a paper published by Plotikin et al. 1, it was suggested that the cell uses a variety of mRNA translation regulatory mechanisms such as ribosome binding site (RBS), mRNA folding and codon redundancies to control the translation rate of the ribosome and peptides 2. This was further elaborated in a paper by Kim et al. 3, where it was proposed that the cell intentionally controls the translation rate of ribosomes across different regions of the mRNA to control protein folding. One of the major factors influencing ribosome translation is the choice of of codons 4. By placing rare or common codons in certain segments of the mRNA, the ribosome translation rate can be retarded or quickened, respectively.
With that knowledge, it has been shown that ribosomes move faster over mRNA segments encoding folded protein domains (e.g. alpha-helices) and slower in linker regions between the domains. It has been proposed that this helps secondary structure folding and minimises interference from other regions of the peptide 3. In other words, it allows localised folding to occur by ensuring correct secondary structure formation at the right segments which in turn allows folding in to the correct native 3D conformation. This observation suggests that the translation kinetics profile determined by the codon choices should be maintained in the heterologous host, and thus that the classical method of replacing every codon with its most frequent version in the new host is not the best approach to heterologous expression. This approach to global conservation of codon usage and translation kinetics of an mRNA is referred to as 'codon harmonisation' 4.
Model
Harry to write
- explain how the model works (basically how you explained it in the lab chat)
- put some of the pictures of the translation kinetics graphs as well
Experimental Validation
The B. subtilis flavin-binding fluorescent protein (BsFbFP) was used as a marker to test the validity of the computational algorithm for codon harmonisation. The algorithm (Harrison Steel, unpublished data) generates a harmonised gene sequence for the expression host by considering the tRNA gene copy number (tGCN) profiles in both hosts, and by taking into account the 'wobble effect' which occurs as a result of redundant tRNA-mRNA codon and anticodon base-pairing which has been reported to slow down translation [5].  The particular protein (BsFbFP) used in our study to model codon harmonisation is a promising new alternative to the green fluorescent protein (GFP) family of proteins. BsFbFP is a small protein (137 amino acids), it is functional in many different cellular environments and has high tolerance to changes in pH, oxygen levels and heat, making it applicable to a wide range of experiments [6]. In contrast, GFP is much larger, matures slowly, absolutely requires oxygen for fluorescence, and is sensitive to chemical changes. Furthermore, in the context of our planned experiments, the lack of genome sequence for the GFP source organism Aequorea victoria means that no tGCN profiles are available, which is crucial for the newly-developed harmonisation algorithm.
By measuring changes in fluorescence, the folding of the protein can be determined, and it is expected that higher fluorescence will be detected from the harmonised sequence due to better folding than other variants.
Results & Discussions
- sequences
- statistics of differences in the sequences i.e. WT vs. Harr-monised
- fluorimetric analysis
Supplementary Model: Translation Kinetics
While the codon harmonisation model explained above generates a harmonised sequence - matches the translation kinetics profile of the wild-type to expression host, this model derived from the literature helps explain the influence of the initiation rate dependent on the strength of the RBS as well as the codon usage. Using this model, we can calculate the initiation rate (mRNA-ribosome recruitment rate) and use the harmonised sequences to monitor the ribosomal traffic on the mRNA. The purpose of this model of ribosomal traffic monitoring is to ensure that the mRNA of our gene will provide the ideal environment for optimum translation to yield functional target protein and a successful heterologous expression, consequently. This is because ribosomal traffic has been shown to yield little or non-functional protein 1 and by controlling it, we can express highly functional proteins.
As well as modelling ribosomal traffic by reference to codon usage and RBS-mediated initiation rate, we also model the possible secondary structure that the mRNA can adopt. This is mainly to check that the RBS remains accessible by the ribosome and enable translation pre-initiation complex. Folds at other regions along the path of the ribosome can be overcome with negligible effect on translation rate and fidelity.
Model
Mention the literature and also how you used models for RBS initiation rate and RNA folding in conjunction with each other
Results & Discussions
- the movie is really good to put  - diagrams of RNA fold
1 Plotkin et al. Synonymous but not the same: the causes and consequences of codon bias, Nature Review Genetics, 2011, 12:32-42
2 Brackley CA, Romano MC, Thiel M (2011) The Dynamics of Supply and Demand in mRNA Translation. PLoS Comput Biol 7(10): e1002203.
3 Kim et al. Translational tuning optimizes nascent protein folding in cells, Science, 2015, 348(6233)
4 Angov, E., et al., Heterologous Protein Expression Is Enhanced by Harmonizing the Codon Usage Frequencies of the Target Gene with those of the Expression Host. Plos One, 2008. 3(5).
