<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_Sydney_Australia_Modeling_Supplementary skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:Sydney Australia/Modeling/Supplementary</SPAN></H1><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><DIV id="menuContainer"><DIV id="menuContainer2"><NAV><UL><LI><A href="https://2015.igem.org/Team:Sydney_Australia">Home</A></LI><LI><A href="https://2015.igem.org/Team:Sydney_Australia/achievements">Achievements</A></LI><LI><A href="#">Project</A></LI><LI><A href="https://2015.igem.org/Team:Sydney_Australia/project_overview">Overview</A></LI><LI><A href="https://2015.igem.org/Team:Sydney_Australia/Description">Description</A></LI><LI><A href="https://2015.igem.org/Team:Sydney_Australia/Experiments">Protocols</A></LI><LI><A href="https://2015.igem.org/Team:Sydney_Australia/goldengate">Golden Gate</A></LI><LI><A href="https://2015.igem.org/Team:Sydney_Australia/Results">Results</A></LI><LI><A href="https://2015.igem.org/Team:Sydney_Australia/Notebook">Notebook</A></LI></UL><LI><A href="https://2015.igem.org/Team:Sydney_Australia/Parts">Parts</A></LI><LI><A href="#">Modelling</A></LI><LI><A href="https://2015.igem.org/Team:Sydney_Australia/Modeling">Overview</A></LI><LI><A href="https://2015.igem.org/Team:Sydney_Australia/TransOpt">TransOpt</A></LI><LI><A href="https://2015.igem.org/Team:Sydney_Australia/Modeling/visualisation">Visualisation</A></LI><LI><A href="https://2015.igem.org/Team:Sydney_Australia/Modeling/ElectroStop">ElectroStop</A></LI><LI><A href="https://2015.igem.org/Team:Sydney_Australia/Modeling/Supplementary">Supplementary Material</A></LI><LI><A href="#">Human Practices</A></LI><LI><A href="https://2015.igem.org/Team:Sydney_Australia/Practices">Overview</A></LI><LI><A href="https://2015.igem.org/Team:Sydney_Australia/EthicsMotivation">Motivation &amp; Analysis</A></LI><LI><A href="https://2015.igem.org/Team:Sydney_Australia/Outreach">Outreach</A></LI><LI><A href="https://2015.igem.org/Team:Sydney_Australia/Media">Media</A></LI><LI><A href="https://2015.igem.org/Team:Sydney_Australia/Collaborations">Collaboration</A></LI><LI><A href="https://2015.igem.org/Team:Sydney_Australia/BusinessModel">Business Model</A></LI><LI><A href="https://2015.igem.org/Team:Sydney_Australia/Safety">Safety</A></LI><LI><A href="https://2015.igem.org/Team:Sydney_Australia/Attributions">Attributions</A></LI><LI><A href="https://2015.igem.org/Team:Sydney_Australia/Team">Team</A></LI></NAV></DIV></DIV><DIV id="mainContainer"><DIV id="contentContainer"><DIV style="float: right"><DIV id="toc" class="toc"><DIV id="toctitle"><H2>Contents</H2></DIV><UL><LI class="toclevel-1 tocsection-1"><A href="#Supplementary_Material"><SPAN class="tocnumber">1</SPAN><SPAN class="toctext">Supplementary Material</SPAN></A></LI><LI class="toclevel-1 tocsection-2"><A href="#TransOpt"><SPAN class="tocnumber">2</SPAN><SPAN class="toctext">TransOpt</SPAN></A></LI><LI class="toclevel-1 tocsection-3"><A href="#Visualisation_Tool"><SPAN class="tocnumber">3</SPAN><SPAN class="toctext">Visualisation Tool</SPAN></A></LI><LI class="toclevel-1 tocsection-4"><A href="#ElectroStop"><SPAN class="tocnumber">4</SPAN><SPAN class="toctext">ElectroStop</SPAN></A></LI><LI class="toclevel-1 tocsection-5"><A href="#References"><SPAN class="tocnumber">5</SPAN><SPAN class="toctext">References</SPAN></A></LI></UL></DIV></DIV><H1><SPAN class="mw-headline" id="Supplementary_Material">Supplementary Material</SPAN></H1><P>Maintaining the ideal of open-source research, this page contains the source code for our various modelling efforts. Due to the computational complexity of some of these programs we did not have the time or expertise to produce web-interfacing tools for general usage. That said, for a user with MATLAB already installed these files can be downloaded and operated immediately. 
</P><H1><SPAN class="mw-headline" id="TransOpt">TransOpt</SPAN></H1><P>The complete code for TransOpt is written in MATLAB<SUP><A rel="nofollow" class="external text" href="https://2015.igem.org/Team:Sydney_Australia/Modeling/Supplementary#References">1</A></SUP>(requiring MATLAB2011 or later), and is available in the .zip file below:
</P><DIV align="center"><A rel="nofollow" class="external text" href="https://static.igem.org/mediawiki/2015/e/e0/USYDTransOpt.zip"><B>TransOpt.zip</B></A></DIV><P>The only file which need be run or altered by the user is <I>TransOpt.m</I>, which is the main procedure of the tool. From here the input sequence can be changed, as can the initial and heterologous expression hosts. <I>TransOpt.m</I> makes reference to a function <I>frequency_optimise.m</I> which performs standard codon harmonisation, the results from which are compared to the TransOpt result.
</P><P>The input sequence must be set as a text file containing a sequence of the letters A,T,C,G which is a multiple of three in length.
</P><P>The input GCNs (Gene Copy Numbers) for use in rate quantisation are included for the strains of <I>Pseudomonas putida<SUP><A rel="nofollow" class="external text" href="https://2015.igem.org/Team:Sydney_Australia/Modeling/Supplementary#References">2</A></SUP>, Mycobacterium smegmatis<SUP><A rel="nofollow" class="external text" href="https://2015.igem.org/Team:Sydney_Australia/Modeling/Supplementary#References">3</A></SUP>, Escherichia coli<SUP><A rel="nofollow" class="external text" href="https://2015.igem.org/Team:Sydney_Australia/Modeling/Supplementary#References">4</A></SUP></I> as used in our project. In order to use the tool for different bacterial species, data can be downloaded from the Genomic tRNA Database<SUP><A rel="nofollow" class="external text" href="https://2015.igem.org/Team:Sydney_Australia/Modeling/Supplementary#References">5</A></SUP> and formatted as shown in (for example) <I>pseudogcn.csv</I>.
</P><P>The tool removes iGEM restricted sites from the final mRNA sequence, with a list of these being stored in the <I>badseq</I> variable. If there are additional sites the user wishes to outlaw, they can be added to this list.
</P><H1><SPAN class="mw-headline" id="Visualisation_Tool">Visualisation Tool</SPAN></H1><P>The complete code for our visualisation tool is written in MATLAB<SUP><A rel="nofollow" class="external text" href="https://2015.igem.org/Team:Sydney_Australia/Modeling/Supplementary#References">1</A></SUP>(requiring MATLAB2011 or later), and is available in the .zip file below:
</P><DIV align="center"><A rel="nofollow" class="external text" href="https://static.igem.org/mediawiki/2015/7/75/USYDVisualisationTool.zip"><B>VisualisationTool.zip</B></A></DIV><P>
The only file which need be run or altered by the user is <I>Visualisation.m</I>, which is the main procedure of the tool. From here the input sequence can be changed, as can the expression host. 
</P><P>The input sequence must be set as a text file containing a sequence of the letters A,T,C,G which is a multiple of three in length.
</P><P>The input GCNs (Gene Copy Numbers) for use in rate quantisation are included for the strain of <I>Mycobacterium smegmatis<SUP><A rel="nofollow" class="external text" href="https://2015.igem.org/Team:Sydney_Australia/Modeling/Supplementary#References">3</A></SUP></I> as used in our project. In order to use the tool for different bacterial species, data can be downloaded from the Genomic tRNA Database<SUP><A rel="nofollow" class="external text" href="https://2015.igem.org/Team:Sydney_Australia/Modeling/Supplementary#References">5</A></SUP>. Extending the tool to a different organism also requires replacing <I>utilisation.csv</I>, which is a file (decoded in <I>readSubtracting.m</I>) that reflects which tRNA bind with different codons (via standard Watson-Crick and wobble interactions). 
</P><P>The parameters <I>alpha</I> and <I>beta</I>, which define the hop-on and hop-off rates can be changed on lines 24 and 25 of the code respectively, which yields interesting changes in dynamics. In order to simulate different numbers of mRNA molecules within the cell, the variable <I>mRNA_copies</I> can be altered at the user's discretion. Setting <I>mRNA_copies</I> to one produces a different plot of the translation process, showing the ribosomes moving along a two-dimensional plot of translation rate.
</P><H1><SPAN class="mw-headline" id="ElectroStop">ElectroStop</SPAN></H1><P>The source code for ElectroStop is written in the Arduino programming language<SUP><A rel="nofollow" class="external text" href="https://2015.igem.org/Team:Sydney_Australia/Modeling/Supplementary#References">6</A></SUP>, and is available in the .zip file below:
</P><DIV align="center"><A rel="nofollow" class="external text" href="https://static.igem.org/mediawiki/2015/b/b6/ElectroStop.zip"><B>ElectroStop.zip</B></A></DIV><P>
This software can be run on any of the ATMega328 Arduino variants, but will obviously require the circuit and hardware that makes up the ElectroStop device, as described on the <A rel="nofollow" class="external text" href="https://2015.igem.org/Team:Sydney_Australia/Modeling/ElectroStop">ElectroStop page.</A> Programming an Arduino device with this software will require the Arduino development software version 1.6.5 or later<SUP><A rel="nofollow" class="external text" href="https://2015.igem.org/Team:Sydney_Australia/Modeling/Supplementary#References">6</A></SUP>, as well as the Adafruit SSD1306<SUP><A rel="nofollow" class="external text" href="https://2015.igem.org/Team:Sydney_Australia/Modeling/Supplementary#References">7</A></SUP> and GFX<SUP><A rel="nofollow" class="external text" href="https://2015.igem.org/Team:Sydney_Australia/Modeling/Supplementary#References">8</A></SUP> libraries.
</P><P>The 3D models for the two 3D printed parts of ElectroStop are provided in the .zip file below:
</P><DIV align="center"><A rel="nofollow" class="external text" href="https://static.igem.org/mediawiki/2015/1/14/ElectroStopStructure.zip"><B>ElectroStop Structure.zip</B></A></DIV><P>
This .zip includes .step files for each structural part, which can be opened by most common 3D modelling applications. It also includes .stl (Stereo-Lithography) files, which are used widely in 3D printing, and can be uploaded directly to most 3D printers for immediate manufacturing.
</P><H1><SPAN class="mw-headline" id="References">References</SPAN></H1><P><SUP>1 </SUP> MATLAB, 2015, “MATLAB - The Language of Technical Computing”, <A rel="nofollow" class="external free" href="http://www.mathworks.com/products/matlab/">http://www.mathworks.com/products/matlab/</A>.
</P><P><SUP>2 </SUP> Genomic tRNA Database, 2014, “tRNAscan-SE Analysis of Pseudomonas putida KT2440”, <A rel="nofollow" class="external free" href="http://gtrnadb.ucsc.edu/Pseu_puti_KT2440/">http://gtrnadb.ucsc.edu/Pseu_puti_KT2440/</A>.
</P><P><SUP>3 </SUP> Genomic tRNA Database, 2014, “tRNAscan-SE Analysis of Mycobacterium smegmatis MC2 155”, <A rel="nofollow" class="external free" href="http://gtrnadb.ucsc.edu/Myco_smeg_MC2_155/">http://gtrnadb.ucsc.edu/Myco_smeg_MC2_155/</A>.
</P><P><SUP>4 </SUP> Genomic tRNA Database, 2014, “tRNAscan-SE Analysis of Escherichia coli K12”, <A rel="nofollow" class="external free" href="http://gtrnadb.ucsc.edu/Esch_coli_K12/">http://gtrnadb.ucsc.edu/Esch_coli_K12/</A>.
</P><P><SUP>5 </SUP> The Genomic tRNA Database, 2014, <A rel="nofollow" class="external free" href="http://gtrnadb.ucsc.edu/">http://gtrnadb.ucsc.edu/</A>.
</P><P><SUP>6 </SUP> Arduino, 2015, “Download the Arduino Software - Version 1.6.5”, <A rel="nofollow" class="external free" href="https://www.arduino.cc/en/Main/Software">https://www.arduino.cc/en/Main/Software</A>.
</P><P><SUP>7 </SUP> Adafruit, 2015, “Download the Adafruit SSD1306 Library”, <A rel="nofollow" class="external free" href="https://github.com/adafruit/Adafruit_SSD1306">https://github.com/adafruit/Adafruit_SSD1306</A>.
</P><P><SUP>8 </SUP> Adafruit, 2015, “Download the Adafruit GFX Library”, <A rel="nofollow" class="external free" href="https://github.com/adafruit/Adafruit-GFX-Library">https://github.com/adafruit/Adafruit-GFX-Library</A>.
</P></DIV></DIV></DIV></DIV></DIV></DIV></BODY></HTML>