<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_Sydney_Australia_goldengate skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:Sydney Australia/goldengate</SPAN></H1><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><DIV id="menuContainer"><DIV id="menuContainer2"><NAV><UL><LI><A href="https://2015.igem.org/Team:Sydney_Australia">Home</A></LI><LI><A href="https://2015.igem.org/Team:Sydney_Australia/achievements">Achievements</A></LI><LI><A href="#">Project</A></LI><LI><A href="https://2015.igem.org/Team:Sydney_Australia/project_overview">Overview</A></LI><LI><A href="https://2015.igem.org/Team:Sydney_Australia/Description">Description</A></LI><LI><A href="https://2015.igem.org/Team:Sydney_Australia/Experiments">Protocols</A></LI><LI><A href="https://2015.igem.org/Team:Sydney_Australia/goldengate">Golden Gate</A></LI><LI><A href="https://2015.igem.org/Team:Sydney_Australia/Results">Results</A></LI><LI><A href="https://2015.igem.org/Team:Sydney_Australia/Notebook">Notebook</A></LI></UL><LI><A href="https://2015.igem.org/Team:Sydney_Australia/Parts">Parts</A></LI><LI><A href="#">Modelling</A></LI><LI><A href="https://2015.igem.org/Team:Sydney_Australia/Modeling">Overview</A></LI><LI><A href="https://2015.igem.org/Team:Sydney_Australia/TransOpt">TransOpt</A></LI><LI><A href="https://2015.igem.org/Team:Sydney_Australia/Modeling/visualisation">Visualisation</A></LI><LI><A href="https://2015.igem.org/Team:Sydney_Australia/Modeling/ElectroStop">ElectroStop</A></LI><LI><A href="https://2015.igem.org/Team:Sydney_Australia/Modeling/Supplementary">Supplementary Material</A></LI><LI><A href="#">Human Practices</A></LI><LI><A href="https://2015.igem.org/Team:Sydney_Australia/Practices">Overview</A></LI><LI><A href="https://2015.igem.org/Team:Sydney_Australia/EthicsMotivation">Motivation &amp; Analysis</A></LI><LI><A href="https://2015.igem.org/Team:Sydney_Australia/Outreach">Outreach</A></LI><LI><A href="https://2015.igem.org/Team:Sydney_Australia/Media">Media</A></LI><LI><A href="https://2015.igem.org/Team:Sydney_Australia/Collaborations">Collaboration</A></LI><LI><A href="https://2015.igem.org/Team:Sydney_Australia/BusinessModel">Business Model</A></LI><LI><A href="https://2015.igem.org/Team:Sydney_Australia/Safety">Safety</A></LI><LI><A href="https://2015.igem.org/Team:Sydney_Australia/Attributions">Attributions</A></LI><LI><A href="https://2015.igem.org/Team:Sydney_Australia/Team">Team</A></LI></NAV></DIV></DIV><DIV id="mainContainer"><DIV id="contentContainer"><DIV style="float: right"><DIV id="toc" class="toc"><DIV id="toctitle"><H2>Contents</H2></DIV><UL><LI class="toclevel-1"><A href="#Why_Golden_Gate_Assembly.3F"><SPAN class="tocnumber">1</SPAN><SPAN class="toctext">Why Golden Gate Assembly?</SPAN></A></LI><LI class="toclevel-1"><A href="#Design"><SPAN class="tocnumber">2</SPAN><SPAN class="toctext">Design</SPAN></A></LI><LI class="toclevel-1"><A href="#Method"><SPAN class="tocnumber">3</SPAN><SPAN class="toctext">Method</SPAN></A></LI><LI class="toclevel-2"><A href="#Reaction_mixture_.28total_volume_.3D_15_uL.29"><SPAN class="tocnumber">3.1</SPAN><SPAN class="toctext">Reaction mixture (total volume = 15 uL)</SPAN></A></LI><LI class="toclevel-2"><A href="#Thermocycling"><SPAN class="tocnumber">3.2</SPAN><SPAN class="toctext">Thermocycling</SPAN></A></LI><LI class="toclevel-2"><A href="#Materials"><SPAN class="tocnumber">3.3</SPAN><SPAN class="toctext">Materials</SPAN></A></LI></UL><LI class="toclevel-1"><A href="#Results_.26_Tips"><SPAN class="tocnumber">4</SPAN><SPAN class="toctext">Results &amp; Tips</SPAN></A></LI></DIV></DIV><P>To assembly our multiple G-Blocks, we decided to use the Golden Gate Assembly method compared to the more traditional Gibson Assembly method and/or the classic digestion ligation method. Given this technique is in its infancy, we have provided more information below which we hope this will help future iGEM teams. 
</P><H2><SPAN class="mw-headline" id="Why_Golden_Gate_Assembly.3F">Why Golden Gate Assembly?</SPAN></H2><P>Golden Gate was chosen over other prominent methods as our primary cloning method as we were attempting to clone multiple inserts into the singular vector. Whilst the Gibson Assembly Method has has success in this area, given our inserts were large (upwards of 3kb) we deemed Gibson to be unsuitable for this task. Furthermore, Golden Gate only It is requires two enzymes in the reaction: DNA ligase and BsaI non-palindromic restriction enzyme. 
</P><DIV class="center"><DIV class="thumb tnone"><DIV class="thumbinner" style="width:602px;"><DIV class="thumbcaption"><I>Figure 1</I> General design requirement of Golden Gate Assembly from New England Biolabs Inc. <SUP> 1 </SUP></DIV></DIV></DIV></DIV><H2><SPAN class="mw-headline" id="Design">Design</SPAN></H2><P>The same design protocol as prescribed in NEB website was followed. 
</P><UL><LI> If there are 3 inserts (as in our project), design each end that needs to be put together to have 4bp overlapping overhangs after digestion by BsaI (as shown in above diagram).</LI></UL><DIV class="center"><DIV class="thumb tnone"><DIV class="thumbinner" style="width:452px;"><DIV class="thumbcaption"><I>Figure 2</I> Schematic of the general overlap design of inserts, with matching colours indicative of overlapping overhangs after BsaI digestion.</DIV></DIV></DIV></DIV><UL><LI> The left hand side overhang of the first insert and the right hand side overhang of the last insert need to overlap with the overhangs of the digested vector. </LI></UL><UL><LI> Start by selecting the enzymes you would like to digest your vector e.g. SpeI/EcoRI and design their corresponding overhangs on the inserts to overlap with them. </LI></UL><UL><LI> Make sure to place the BsaI recognition site outside of the ORF regions so that it is digested and removed from the final DNA e.g. place it upstream of the 5' end of insert and downstream of 3' end of insert.</LI></UL><DIV class="center"><DIV class="thumb tnone"><DIV class="thumbinner" style="width:452px;"><DIV class="thumbcaption"><I>Figure 3</I> Final product of the reaction with complementary overlaps of the inserts and vector binding and subsequently ligating together.</DIV></DIV></DIV></DIV><UL><LI> Design primers for PCR linearization of the vector such that they amplified from a few base pairs upstream of the desired cut sites other vector.</LI></UL><UL><LI> Double check that all overhangs overlap and are in the correct overhang type of 3’ or 5’. In other words, reconstruct the final recombinant vector after drafting your designed inserts and vector to see if it yields the desired final product.</LI></UL><UL><LI> In the thermocycling, the 37 <SUP> o </SUP>C cycle is for BsaI digestion and 16 <SUP> o </SUP>C for DNA ligase to join the overlapping overhangs together. </LI></UL><DIV class="center"><DIV class="thumb tnone"><DIV class="thumbinner" style="width:502px;"><DIV class="thumbcaption"><I>Figure 4</I> Diagram combining and reinforcing the whole design protocol together. Bear in mind that the PCR linearised vector needs to be digested already when put in the reaction mixture i.e. vector digestion and Golden Gate reaction occur at two separate steps.</DIV></DIV></DIV></DIV><H2><SPAN class="mw-headline" id="Method">Method</SPAN></H2><H3><SPAN class="mw-headline" id="Reaction_mixture_.28total_volume_.3D_15_uL.29">Reaction mixture (total volume = 15 uL)</SPAN></H3><UL><LI> 1 uL concentrated T4 DNA ligase (2000 U)</LI><LI> 1 uL BsaI-HF (20 U)</LI><LI> 1.5 uL T4 DNA ligase reaction buffer (1X)</LI><LI> adjust volumes of digested vector and inserts such that they are equimolar and their total volume does not exceed 11.5 uL (50 ng of vector is an optimum amount, but there is a wide acceptable range)</LI><LI> make up volume to 15 uL with MiliQ water </LI><LI> mix by gentle tapping </LI></UL><H3><SPAN class="mw-headline" id="Thermocycling">Thermocycling</SPAN></H3><OL><LI> 16 <SUP> o </SUP>C (10 seconds)</LI><LI> 16 <SUP> o </SUP>C (1 minute)</LI><LI> 37 <SUP> o </SUP>C (1 minute)</LI><LI> repeat (2-3) 30-35 times </LI><LI> 16 <SUP> o </SUP>C (1 minute)</LI><LI> 50 <SUP> o </SUP>C (5 minutes)</LI><LI> 80 <SUP> o </SUP>C (5 minutes)</LI></OL><H3><SPAN class="mw-headline" id="Materials">Materials</SPAN></H3><UL><LI> NEB BsaI/BsaI-HF restriction enzyme (20000 U/mL)</LI><LI> NEB T4 DNA ligase concentrated (2000000 U/mL)</LI><LI> NEB 10X T4 DNA ligase reaction buffer (500mM Tris-HCl, 100mM MgCl2, 10mM ATP, 100mM DTT, pH 7.5@25°C)</LI><LI> Insert (equimolar amounts to vector)</LI><LI> Digested PCR linearised vector (e.g. 50 ng)</LI></UL><P>Note: BsaI-HF was used in our experiments; normal BsaI should work as well.
</P><H2><SPAN class="mw-headline" id="Results_.26_Tips">Results &amp; Tips</SPAN></H2><P>We discovered that Golden Gate gives mixed results.
</P><UL><LI>For one of our constructs, it successfully joined three 1-1.5 kbp inserted into digested pSB1C3 using the above method</LI><LI>In the other construct, while the transformation gave positive results, it was discovered that instead of ligating two 2 kbp inserts into pSB1C3, it ligated a 400 bp insert in it (which is the subject of much mystery and speculation)</LI><LI>Design inserts such that they contain the full ORF not divided between two inserts as this can make the design of overlapping regions complicated.</LI><LI>Note that the BsaI cut sites are removed after BsaI digestion, hence, the ligated inserts will not be digested again and the BsaI will go on to digest inserts.</LI></UL><P>Overall, we believe it works well (or at least worth a try) provided that the following are implemented:
</P><UL><LI>Concentrated T4 DNA ligase </LI><LI>Use a maximum total volume of 15 uL </LI><LI>Use equimolar amounts of all inserts and vector (higher ratio may result in mismatched ligation)</LI><LI>Make sure that all DNA samples are pure and free of contaminants (miniprep and DNA extraction kits are adequate compared to midiprep)</LI><LI>Mix sample by pipetting up and down or gentle tapping </LI><LI>Use 5-8 uL for transformation </LI></UL><DIV class="center"><DIV class="thumb tnone"><DIV class="thumbinner" style="width:802px;"><DIV class="thumbcaption">Figure 5. Sequence schematic of the successful golden gate cloning of the EtnABCD gene cluster (purple, yellow, green and blue respectively). Assembled theoretical GBlock construct from golden gate assembly (blue) compared to actual aligned sequence reads obtained from Australian Genome Research Facility Ltd (red).</DIV></DIV></DIV></DIV><P>The reads demonstrate that our golden gate cloning was successful, as each GBlock was cloned in the correct orientation and order, as well showing that no mutations were introduced (<B>100% similarity</B> between our golden gate cloning and theoretical construct).
</P><P>
In conclusion, we believe that Golden Gate is a promising method and while it is at its early stages of optimisation and development, it has great potential for constructing clones with large numbers of inserts. We hope that by sharing our procedures and results we can make available more information for further optimisation and development of this method.
</P><P><SUP> 1 </SUP> Golden Gate Assembly, New England Biolabs Inc. accessed 23/8/2015 
“<A rel="nofollow" class="external free" href="https://www.neb.com/applications/cloning-and-synthetic-biology/dna-assembly-and-cloning/golden-gate-assembly">https://www.neb.com/applications/cloning-and-synthetic-biology/dna-assembly-and-cloning/golden-gate-assembly</A></P></DIV></DIV></DIV></DIV></DIV></DIV></BODY></HTML>