<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_Sydney_Australia_Parts skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:Sydney Australia/Parts</SPAN></H1><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><DIV id="menuContainer"><DIV id="menuContainer2"><NAV><UL><LI><A href="https://2015.igem.org/Team:Sydney_Australia">Home</A></LI><LI><A href="https://2015.igem.org/Team:Sydney_Australia/achievements">Achievements</A></LI><LI><A href="#">Project</A></LI><LI><A href="https://2015.igem.org/Team:Sydney_Australia/project_overview">Overview</A></LI><LI><A href="https://2015.igem.org/Team:Sydney_Australia/Description">Description</A></LI><LI><A href="https://2015.igem.org/Team:Sydney_Australia/Experiments">Protocols</A></LI><LI><A href="https://2015.igem.org/Team:Sydney_Australia/goldengate">Golden Gate</A></LI><LI><A href="https://2015.igem.org/Team:Sydney_Australia/Results">Results</A></LI><LI><A href="https://2015.igem.org/Team:Sydney_Australia/Notebook">Notebook</A></LI></UL><LI><A href="https://2015.igem.org/Team:Sydney_Australia/Parts">Parts</A></LI><LI><A href="#">Modelling</A></LI><LI><A href="https://2015.igem.org/Team:Sydney_Australia/Modeling">Overview</A></LI><LI><A href="https://2015.igem.org/Team:Sydney_Australia/TransOpt">TransOpt</A></LI><LI><A href="https://2015.igem.org/Team:Sydney_Australia/Modeling/visualisation">Visualisation</A></LI><LI><A href="https://2015.igem.org/Team:Sydney_Australia/Modeling/ElectroStop">ElectroStop</A></LI><LI><A href="https://2015.igem.org/Team:Sydney_Australia/Modeling/Supplementary">Supplementary Material</A></LI><LI><A href="#">Human Practices</A></LI><LI><A href="https://2015.igem.org/Team:Sydney_Australia/Practices">Overview</A></LI><LI><A href="https://2015.igem.org/Team:Sydney_Australia/EthicsMotivation">Motivation &amp; Analysis</A></LI><LI><A href="https://2015.igem.org/Team:Sydney_Australia/Outreach">Outreach</A></LI><LI><A href="https://2015.igem.org/Team:Sydney_Australia/Media">Media</A></LI><LI><A href="https://2015.igem.org/Team:Sydney_Australia/Collaborations">Collaboration</A></LI><LI><A href="https://2015.igem.org/Team:Sydney_Australia/BusinessModel">Business Model</A></LI><LI><A href="https://2015.igem.org/Team:Sydney_Australia/Safety">Safety</A></LI><LI><A href="https://2015.igem.org/Team:Sydney_Australia/Attributions">Attributions</A></LI><LI><A href="https://2015.igem.org/Team:Sydney_Australia/Team">Team</A></LI></NAV></DIV></DIV><DIV id="mainContainer"><DIV id="contentContainer"><H2><SPAN class="mw-headline" id="Parts_Outline">Parts Outline</SPAN></H2><TABLE class="wikitable"><TBODY><TR><TH> Part #
</TH><TH> Name
</TH><TH> Type
</TH><TH> Function
</TH><TH> Elements
</TH></TR><TR><TD><A rel="nofollow" class="external text" href="http://parts.igem.org/Part:BBa_K1736000">BBa_K1736000</A></TD><TD> pSB1C3-BsFP
</TD><TD> Basic (Composite)
</TD><TD><I>B. subtilis</I> flavin-binding fluorescent protein,  used as a marker in diverse fluorescence assays
</TD><TD> ORF is harmonised to E.coli codon usage (Angov et al. <SUP>1</SUP> method), has added N-terminal HisTag, and native RBS
</TD></TR><TR><TD><A rel="nofollow" class="external text" href="http://parts.igem.org/Part:BBa_K1736300">BBa_K1736300</A></TD><TD> pSB1C3-P<SUB>LacIQ1</SUB>-LacI
</TD><TD> Basic (Composite)
</TD><TD> LacI repressor protein for regulating genes under control of the lac promoter/operator system
</TD><TD> Wild-type <I>E.coli</I> LacI protein, with its native <I>E. coli</I> RBS, controlled by LacI<SUP>Q1</SUP> strong constitutive promoter
</TD></TR><TR><TD><A rel="nofollow" class="external text" href="http://parts.igem.org/Part:BBa_K1736200">BBa_K1736200</A></TD><TD> pSB1C3-EtnABCD
</TD><TD> Basic (Composite)
</TD><TD> Multicomponent ethene monooxygenase from <I>Mycobacterium</I>, oxidises diverse alkenes to epoxides
</TD><TD> Four ORFs: etnA, etnB, etnC, etn D, codon-optimised for expression in <I>Pseudomonas</I>, each ORF has native RBS upstream
</TD></TR></TBODY></TABLE><H2><SPAN class="mw-headline" id="Characterisation">Characterisation</SPAN></H2><H4><SPAN class="mw-headline" id="BBa_K1736000:_B._subtilis_Fluorescent_Protein_.28BsFP.29"><SPAN style="color:#00ff00"> BBa_K1736000: <I>B. subtilis</I> Fluorescent Protein (BsFP) </SPAN></SPAN></H4><DIV class="center"><DIV class="thumb tnone"><DIV class="thumbinner" style="width:1002px;"><DIV class="thumbcaption">Gblock design of BsFP gene includes the ORF, <I>B.subtilis</I> RBS and an N-terminal HisTag</DIV></DIV></DIV></DIV><P>The <I>Bacillus subtilis</I> flavin-binding fluorescent protein (BsFbFP, or BsFP) is native to the ‘’B. subtilis’’ bacteria, and homologs are also found in several other bacterial genera. This flavin-binding fluoroprotein has a different chromophore to the more widely used GFP family of fluoroproteins, and BsFP has several advantages over the more-traditional fluoroproteins such as GFP (<A rel="nofollow" class="external text" href="http://parts.igem.org/Part:BBa_E0040">BBa_E0040</A>). The BsFP gene is much smaller than GFP (around 700 bp vs 137 bp), and it is capable of folding and fluorescing under anaerobic conditions, unlike GFP and its homologs. The BsFP also folds into a fluorescent form more quickly and efficiently <SUP>2</SUP>. 
</P><P>In this project, we used the BsFP protein to experimentally validate our novel codon harmonisation algorithm, <A rel="nofollow" class="external text" href="https://2015.igem.org/Team:Sydney_Australia/TransOpt">TransOpt</A>. We were inspired to choose this protein as our model system based on the work of previous iGEM teams, who developed related parts <A rel="nofollow" class="external text" href="http://parts.igem.org/Part:BBa_K376004">BBa_K376004</A>, <A rel="nofollow" class="external text" href="http://parts.igem.org/Part:BBa_K1094000">BBa_K1094000</A> and <A rel="nofollow" class="external text" href="http://parts.igem.org/Part:BBa_K660000">BBa_K660000</A>. We tried to improve the function of BsFP by utilising three different codon-optimisation / harmonisation approaches; these were our novel 'in-house' algorithm TransOpt, the standard harmonisation methods proposed by Angov <I>et al.</I><SUP>1</SUP>, and a ‘fast-folding’ method (more information on <A rel="nofollow" class="external text" href="https://2015.igem.org/Team:Sydney_Australia/TransOpt">TransOpt page</A>). 
</P><P>Our model was hypothesised based on previous literature demonstrating that the ribosome translation kinetics profile was influenced by both the copy number of each specific tRNA gene in the host organism, and by the codon redundancy, and that these two factors were paramount in controlling protein folding, which in turn yields protein activity <SUP>3</SUP>. In this experimental validation, we performed fluorescence assays to detect the correctly-folded proteins, with the assumption being that higher fluorescence is characteristic of a better folded protein. We generated the following four sequences: 
</P><UL><LI>BsFP-WT: native sequence from the native host <I>B. subtilis</I></LI><LI>BsFP-fast: all codons were replaced with the E.coli versions that possessed the fastest translation rate, as quantised using the approach of Spencer <I>et al</I><SUP>3</SUP></LI><LI>BsFP-standard: harmonised BsFP generated via standard harmonisation (as proposed by Angov <I>et al</I><SUP>1</SUP>) using the rate quantisation approach of Spencer <I>et al</I><SUP>3</SUP></LI><LI>BsFP-TransOpt: optimised BsFP sequence generated using our new TransOpt algorithm</LI></UL><P>Each variant of BsFP was cloned into a tetracycline-inducible expression vector (pUS212), transformed into <I>E.coli</I>, and clones confirmed to carry the correct constructs were induced with tetracycline, and then both fluorescence (520 nm) and optical desnity (600 nm) were measured in triplicate samples of each culture. The fluorescence data were normalised by dividing by the optical density and subtracting the fluorescence from the <I>E. coli</I> pUS212, containing no BsFP gene.
</P><P>After performing the fluorescence assay, we found that the BsFP subjected to standard harmonisation possessed the highest fluorescence, which was 2 fold higher than the wild-type sequence. The fast-folding and Trans-Opt versions of the gene were less fluorescent than wild type. These data were also qualitatively confirmed by exposing tet-induced patches on plates to a long-wave UV lamp; only the standard-harmonisation clones yielded fluorescence visible with the naked eye. 
</P><DIV class="thumb tleft"><DIV class="thumbinner" style="width:502px;"><DIV class="thumbcaption">Graphs showing the fluorescence and relative fold change in fluorescence (compared to BsFP WT) at 520 nm and 500 nm emission after 460 nm excitation.</DIV></DIV></DIV><DIV class="thumb tright"><DIV class="thumbinner" style="width:480px;"><DIV class="thumbcaption">Plate containing tet-induced E. coli JM109 cells expressing different versions of the BsFP protein, or a vector-only control (no BsFP). Note that WT stands for wild type.</DIV></DIV></DIV><P>Although we failed to demonstrate that the TransOpt algorithm facilitated greater fluorescence in the case of BsFP, we predict that this approach may still be useful for other proteins. Due to its poor fluorescence, we did not submit the TransOpt-harmonised BsFP as a Part. Instead, we submitted the standard-harmonisation BsFP gene to the Registry. This Part is a versatile and widely-applicable alternative to GFP/RFP  type fluoroproteins, due to its high fluorescence, small size, fast maturation, ad oxygen-independence. 
</P><P>This new BsFP Part has the potential to enhance reporter gene assays, improve biomarker tracking in cell tissues, create fluorescent fusion proteins, and many other applications. It is also appropriate to note that the gene encoding the protein is substantially different to the previously submitted parts  <A rel="nofollow" class="external text" href="http://parts.igem.org/Part:BBa_K376004">BBa_K376004</A>, <A rel="nofollow" class="external text" href="http://parts.igem.org/Part:BBa_K1094000">BBa_K1094000</A> and <A rel="nofollow" class="external text" href="http://parts.igem.org/Part:BBa_K660000">BBa_K660000</A>; while these cannot be directly compared in terms of fluorescence intensity, we note that our part is the most rigorously characterised of these. In particular, the ability to easily detect its fluorescence on agar plates is especially convenient. 
</P><P>If you would like to utilise our codon optimisation tools which were used to generate all BsFP variants for this experiment, refer to our modelling <A rel="nofollow" class="external text" href="https://2015.igem.org/Team:Sydney_Australia/Modeling/Supplementary">supplementary materials</A> page for instructions on downloading and using the algorithm.
</P><DIV class="center"><DIV class="thumb tnone"><DIV class="thumbinner" style="width:602px;"><DIV class="thumbcaption">Structure of pSB1C3-BsFP plasmid construct submitted to the registry.</DIV></DIV></DIV></DIV><H4><SPAN class="mw-headline" id="BBa_K1736300:_PLacIQ1-LacI"><SPAN style="color:#00ffff"> BBa_K1736300: P<SUB>LacIQ1</SUB>-LacI </SPAN></SPAN></H4><DIV class="center"><DIV class="thumb tnone"><DIV class="thumbinner" style="width:1002px;"><DIV class="thumbcaption">LacI construct with LacIQ1 strong constitutive promoter and wild-type RBS.</DIV></DIV></DIV></DIV><P>As explained in the project <A rel="nofollow" class="external text" href="https://2015.igem.org/Team:Sydney_Australia/project_overview">description &amp; results</A> page, the P<SUB>LacIQ1</SUB> part was added to the project to tackle the issue of possible toxicity to the host cells due to overexpression of our main enzyme of interest (ethene monooxygenase). This enzyme is known to be very difficult to clone and express in heterologous hosts, and since our primary cloning vector (pBBR1MCS-2) only had the Plac promoter but no LacI repressor, we needed to add the latter part separately. 
</P><P>In order to ensure that we had strong expression and activity of LacI, we placed the LacIQ1 promoter <SUP>4</SUP> upstream of the <I>E. coli</I> LacI gene . In this version of the promoter, the -35 sequence has been modified to make it identical to the <I>E.coli</I> consensus sequence TTGACA, thus making the promoter much stronger than usual. The following Parts are already deposited for P<SUP>LacIQ1</SUP>: <A rel="nofollow" class="external text" href="http://parts.igem.org/Part:BBa_K091112">BBa_K091112</A> and <A rel="nofollow" class="external text" href="http://parts.igem.org/Part:BBa_K091131">BBa_K091131</A>, while for LacI itself, many parts for the gene have also been deposited, such as <A rel="nofollow" class="external text" href="http://parts.igem.org/Part:BBa_C0012">BBa_C0012</A>. However, to date there is no composite part where the LacI gene and LacIQ1 have been placed together, making this a valuable new addition to the registry, which will be useful for future teams and experiments relying on strong expression control of genes under <I>lac</I> regulation. 
</P><P>To confirm that the new composite part is functional, we ran a qualitative experiment to check for the activity of LacI via its control of the reporter gene LacZ based on hydrolysis of the colourless reagent X-gal to yield a blue dye. This system was comprised of three parts: the P<SUB>LacIQ1</SUB>-LacI part cloned in pSB1C3 (CmR), the lacZ alpha fragment in the plasmid pBR1MCS-2 (KmR) under the control of Plac, and the lacZ omega fragment in the chromosome of <I>E. coli</I> JM109, also under control of Plac. 
</P><DIV class="center"><DIV class="thumb tnone"><DIV class="thumbinner" style="width:802px;"><DIV class="thumbcaption">Schematic of the experimental setup for testing LacI activity in <I>E.coli</I>. Note that part of lacZ is coded on a plasmid, and another part is coded on the chromosome.</DIV></DIV></DIV></DIV><P><I>E.coli</I> JM109 cells containing both plasmids were grown on a variety of media to test the interactions between lacI and Plac in this system. These media were:
</P><P>1. LB-Cm-Km-Xgal (repressed, expect white colonies due to lack of IPTG)
</P><P>2. LB-Cm-Km-Xgal-glucose (strongly repressed, expect white colonies due to lack of IPTG and presence of glucose) 
</P><P>3. LB-Cm-Km-Xgal-IPTG (induced, expect blue colonies due to presence of IPTG) 
</P><P>4. LB-Km-Xgal-IPTG (strongly induced, expect blue colonies due to presence of IPTG and loss of plasmid carrying lacI repressor gene) 
</P><DIV class="center"><DIV class="thumb tnone"><DIV class="thumbinner" style="width:802px;"><DIV class="thumbcaption">Plates of the four different media mentioned above to test for the activity of P<SUB>LacIQ1</SUB>-LacI.</DIV></DIV></DIV></DIV><P>The results of this experiment were not clear cut (above). Cells grown on LB-Cm-Km-Xgal-glucose were completely white, as expected, and cells grown on LB-Km-Xgal-IPTG were predominantly blue, as expected, but the other two media yielded a mix of white and blue cells, mostly white. It was notable that addition of IPTG was not sufficient to give uniform positive induction of lacZ. Our interpretation is that in this experimental setup, LacI repression is exceedingly strong. We believe this is because the amount of LacI protein expressed from the strong promoter on a high-copy plasmid will completely repress expression of the chromosomally-encoded omega fragment of LacZ, even when IPTG is present. 
</P><DIV class="center"><DIV class="thumb tnone"><DIV class="thumbinner" style="width:680px;"><DIV class="thumbcaption">Structure of the pSB1C3-PlacIQ1-LacI plasmid construct submitted to the registry.</DIV></DIV></DIV></DIV><H4><SPAN class="mw-headline" id="BBa_K1736200:_Ethene_Monooxygenase_.28EtnABCD.29"><SPAN style="color:#ff00ff"> BBa_K1736200: Ethene Monooxygenase (EtnABCD) </SPAN></SPAN></H4><DIV class="center"><DIV class="thumb tnone"><DIV class="thumbinner" style="width:1002px;"><DIV class="thumbcaption">EtnABCD construct consisting of 4 ORFs corresponding to the 4 monooxygenase subunits, with a RBS upstream of each ORF.</DIV></DIV></DIV></DIV><P>The ethene monooxygenase (MO) enzyme converts alkenes to epoxides, and is found in <I>Mycobacterium</I> strains that grow on ethene (ethylene) as their carbon and energy source <SUP>5</SUP>. This enzyme is of intense interest for bioremediation and biocatalysis <SUP>6</SUP>, but to date it has never been expressed in a heterologous hosts outside the genus <I>Mycobacterium</I><SUP>7</SUP>. A major focus of our project was to enable  expression of ethene MO enzyme in <I>Pseudomonas</I> as an alternative expression host.
</P><P>The etnABCD genes were modified using a novel approach called <A rel="nofollow" class="external text" href="https://2015.igem.org/Team:Sydney_Australia/TransOpt">TransOpt</A>, to give sequences that we expected would be effectively translated in <I>Pseudomonas</I>. These modified sequences were ordered as three Gblocks, and cloned by <A rel="nofollow" class="external text" href="https://2015.igem.org/Team:Sydney_Australia/goldengate">Golden Gate</A> cloning into pSB1C3; we sequenced several clones. While two clones gave bad sequence reads, and one had several large mutations, one clone was very close to the the expected sequence, with just one base change; this resulted in a single amino acid change converting glutamate residue 91 of EtnD to aspartate (E91D). Note that this change is conservative (both negatively charged aa’s), and also an aspartate is found at this position in the EtnD enzyme of other <I>mycobacteria</I>, so we expect the cloned EtnABCD to be functional, if given the correct host environment. 
</P><P>The etnABCD genes were cloned from pSB1C3 into the broad host range vector pBBR1MCS-2, where they are expressed from the <I>lac</I> promoter, and electrotransformed into <I>P. putida</I> KT2440. Note that in this host that lacks LacI, the <I>lac</I> promoter is constitutive. The culture was exposed to ethene, but none of the expected product (ethene oxide) was formed, based on testing with the colorimetric NBP assay. In a similar experiment, the wild-type etnABCD genes cloned in the same plasmid yielded detectable ethene oxide, so all the experimental setup and reagents were working. 
</P><P>We believed that the constitutive expression of ethene MO straight after transformation would put too much stress on the cells, and may have resulted in the growth of deletion mutants or other unexpected incorrect plasmid derivatives. Hence, we introduced LacI to control the expression of the system on a second plasmid, an RSF1010 derivative called pUS44, and reintroduced the pBR1MCS-2/etnABCD ligation mixture. However, after IPTG induction, the NBP assay still failed to detect ethene oxide, which suggests that the inactivity lies within the enzyme itself, not the cells. Testing the assay reagents with a pure epoxide solution (styrene oxide) yielded a strong purple colour, which confirmed our suspicion that the EtnABCD enzyme itself was to blame. 
</P><P>One reason for the ethene MO inactivity could be that our in-house harmonisation tool TransOpt did not yield a sequence which could be properly folded and active in <I>Pseudomonas</I>, since this was the result unexpectedly obtained in parallel experiments with different codon-optimised forms of a fluorescent protein BsFP (see <A rel="nofollow" class="external text" href="https://2015.igem.org/Team:Sydney_Australia/Results">results and discussion</A>). While we could not demonstrate successful enzyme activity with this <I>Pseudomonas</I>-harmonised sequence variant of the ethene MO, we have still deposited it as a Part. Our rationale for Part submission was twofold: 
</P><OL><LI>The EtnABCD sequence was 99.99% correct, and that the single mutation present was a conservative change which was consistent with sequences of other variants of this enzyme.  </LI><LI>This enzyme is notoriously difficult to express in heterologous hosts, and thus further optimisation of elements like host strain, RBS’s, promoter, and accessory proteins may enable functional expression from the Part submitted here </LI></OL><P>We hope that future iGEM teams can further develop this system to yield a functional monooxygenase enzyme.
</P><P>Important notes concerning this part: 
</P><UL><LI>Due to an oversight in Gblock design, the pSB1C3-EtnABCD part submitted is missing the XbaI and NotI cut sites in the BioBrick prefix, but an EcoRI is present here.</LI><LI>A mutation (E91D) exists in the sequence due to an accidental change introduced while removing BsaI cut sites, however, the change is conservative and is present in ethene MO of other <I>Mycobacterium</I> strains, which means that it is unlikely to have had an effect on the function of the enzyme.</LI><LI>There are two slightly different alternative C-terminal ends quoted in GenBank for the EtnD subunit of this enzyme, it is unclear which is the correct annotation. </LI></UL><DIV class="thumb tright"><DIV class="thumbinner" style="width:602px;"><DIV class="thumbcaption">Structure of pSB1C3-EtnABCD plasmid construct submitted to the registry. Note that RBS are placed upstream of each ORF as shown in the above figure.</DIV></DIV></DIV><P><B>References</B></P><P><SUP>1</SUP> Angov E et al., 2008, “Heterologous Protein Expression Is Enhanced by Harmonizing the Codon Usage Frequencies of the Target Gene with those of the Expression Host”, PLoS One, Vol 3.
</P><P><SUP>2</SUP> Mukherjee, A., et al., Characterization of flavin-based fluorescent proteins: an emerging class of fluorescent reporters. PLoS One, 2013. 8(5): p. e64753.
</P><P><SUP>3</SUP> Spencer P et al., 2012, “Silent Substitutions Predictably Alter Translation Elongation Rates and Protein Folding Efficiencies”, Vol 422, pp. 328-335.
</P><P><SUP>4</SUP> Glascock CB. <I>et al.</I> Using chromosomal lacIQ1 to control expression of genes on high-copy-number plasmids in Escherichia coli. Gene. 1998 Nov 26;223(1-2):221-31.
</P><P><SUP>5</SUP> Coleman, N. V. and J. C. Spain (2003). Epoxyalkane:coenzyme M transferase in the ethene and vinyl chloride biodegradation pathways of Mycobacterium strain JS60.Journal of Bacteriology 185: 5536-5545.
</P><P><SUP>6</SUP> Mattes, T. E., A. K. Alexander and N. V. Coleman (2010). Aerobic biodegradation of the chloroethenes: pathways, enzymes, ecology, and evolution. FEMS Microbiol Rev 34: 445-475.
</P><P><SUP>7</SUP> Ly, M.A. et al. (2011). Construction and evaluation of pMycoFos, a fosmid shuttle vector for Mycobacterium spp. with inducible gene expression and copy number control. J Microbiol Methods 86: 320-326.
</P></DIV></DIV></DIV></DIV></DIV></DIV></BODY></HTML>