<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_Sydney_Australia_Results skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:Sydney Australia/Results</SPAN></H1><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><DIV id="menuContainer"><DIV id="menuContainer2"><NAV><UL><LI><A href="https://2015.igem.org/Team:Sydney_Australia">Home</A></LI><LI><A href="https://2015.igem.org/Team:Sydney_Australia/achievements">Achievements</A></LI><LI><A href="#">Project</A></LI><LI><A href="https://2015.igem.org/Team:Sydney_Australia/project_overview">Overview</A></LI><LI><A href="https://2015.igem.org/Team:Sydney_Australia/Description">Description</A></LI><LI><A href="https://2015.igem.org/Team:Sydney_Australia/Experiments">Protocols</A></LI><LI><A href="https://2015.igem.org/Team:Sydney_Australia/goldengate">Golden Gate</A></LI><LI><A href="https://2015.igem.org/Team:Sydney_Australia/Results">Results</A></LI><LI><A href="https://2015.igem.org/Team:Sydney_Australia/Notebook">Notebook</A></LI></UL><LI><A href="https://2015.igem.org/Team:Sydney_Australia/Parts">Parts</A></LI><LI><A href="#">Modelling</A></LI><LI><A href="https://2015.igem.org/Team:Sydney_Australia/Modeling">Overview</A></LI><LI><A href="https://2015.igem.org/Team:Sydney_Australia/TransOpt">TransOpt</A></LI><LI><A href="https://2015.igem.org/Team:Sydney_Australia/Modeling/visualisation">Visualisation</A></LI><LI><A href="https://2015.igem.org/Team:Sydney_Australia/Modeling/ElectroStop">ElectroStop</A></LI><LI><A href="https://2015.igem.org/Team:Sydney_Australia/Modeling/Supplementary">Supplementary Material</A></LI><LI><A href="#">Human Practices</A></LI><LI><A href="https://2015.igem.org/Team:Sydney_Australia/Practices">Overview</A></LI><LI><A href="https://2015.igem.org/Team:Sydney_Australia/EthicsMotivation">Motivation &amp; Analysis</A></LI><LI><A href="https://2015.igem.org/Team:Sydney_Australia/Outreach">Outreach</A></LI><LI><A href="https://2015.igem.org/Team:Sydney_Australia/Media">Media</A></LI><LI><A href="https://2015.igem.org/Team:Sydney_Australia/Collaborations">Collaboration</A></LI><LI><A href="https://2015.igem.org/Team:Sydney_Australia/BusinessModel">Business Model</A></LI><LI><A href="https://2015.igem.org/Team:Sydney_Australia/Safety">Safety</A></LI><LI><A href="https://2015.igem.org/Team:Sydney_Australia/Attributions">Attributions</A></LI><LI><A href="https://2015.igem.org/Team:Sydney_Australia/Team">Team</A></LI></NAV></DIV></DIV><DIV id="mainContainer"><DIV id="contentContainer"><DIV style="float: right"><DIV id="toc" class="toc"><DIV id="toctitle"><H2>Contents</H2></DIV><UL><LI class="toclevel-1"><A href="#Results_Overview"><SPAN class="tocnumber">1</SPAN><SPAN class="toctext">Results Overview</SPAN></A></LI><LI class="toclevel-2"><A href="#Aim"><SPAN class="tocnumber">1.1</SPAN><SPAN class="toctext">Aim</SPAN></A></LI><LI class="toclevel-2"><A href="#Experimental_design"><SPAN class="tocnumber">1.2</SPAN><SPAN class="toctext">Experimental design</SPAN></A></LI><LI class="toclevel-2"><A href="#Summarised_results"><SPAN class="tocnumber">1.3</SPAN><SPAN class="toctext">Summarised results</SPAN></A></LI><LI class="toclevel-2"><A href="#What_did_we_find.3F"><SPAN class="tocnumber">1.4</SPAN><SPAN class="toctext">What did we find?</SPAN></A></LI></UL><LI class="toclevel-1"><A href="#Full_Results"><SPAN class="tocnumber">2</SPAN><SPAN class="toctext">Full Results</SPAN></A></LI><LI class="toclevel-2"><A href="#I._LacI_Characterisation"><SPAN class="tocnumber">2.1</SPAN><SPAN class="toctext"><SPAN> I. LacI Characterisation </SPAN></SPAN></A></LI><LI class="toclevel-3"><A href="#Experiment"><SPAN class="tocnumber">2.1.1</SPAN><SPAN class="toctext">Experiment</SPAN></A></LI><LI class="toclevel-3"><A href="#Characterisation"><SPAN class="tocnumber">2.1.2</SPAN><SPAN class="toctext">Characterisation</SPAN></A></LI><LI class="toclevel-3"><A href="#Purpose"><SPAN class="tocnumber">2.1.3</SPAN><SPAN class="toctext">Purpose</SPAN></A></LI><LI class="toclevel-2"><A href="#II._GroEL-GroES"><SPAN class="tocnumber">2.2</SPAN><SPAN class="toctext"><SPAN> II. GroEL-GroES </SPAN></SPAN></A></LI><LI class="toclevel-3"><A href="#Experiment_2"><SPAN class="tocnumber">2.2.1</SPAN><SPAN class="toctext">Experiment</SPAN></A></LI><LI class="toclevel-3"><A href="#Characterisation_2"><SPAN class="tocnumber">2.2.2</SPAN><SPAN class="toctext">Characterisation</SPAN></A></LI><LI class="toclevel-3"><A href="#Purpose_2"><SPAN class="tocnumber">2.2.3</SPAN><SPAN class="toctext">Purpose</SPAN></A></LI><LI class="toclevel-2"><A href="#III._EtnABCD_Characterisation"><SPAN class="tocnumber">2.3</SPAN><SPAN class="toctext"><SPAN> III. EtnABCD Characterisation </SPAN></SPAN></A></LI><LI class="toclevel-3"><A href="#Experiment_3"><SPAN class="tocnumber">2.3.1</SPAN><SPAN class="toctext">Experiment</SPAN></A></LI><LI class="toclevel-3"><A href="#Characterisation_3"><SPAN class="tocnumber">2.3.2</SPAN><SPAN class="toctext">Characterisation</SPAN></A></LI><LI class="toclevel-3"><A href="#Purpose_3"><SPAN class="tocnumber">2.3.3</SPAN><SPAN class="toctext">Purpose</SPAN></A></LI><LI class="toclevel-1"><A href="#TransOpt:_Experimental_Validation"><SPAN class="tocnumber">3</SPAN><SPAN class="toctext">TransOpt: Experimental Validation</SPAN></A></LI><LI class="toclevel-2"><A href="#Method"><SPAN class="tocnumber">3.1</SPAN><SPAN class="toctext">Method</SPAN></A></LI><LI class="toclevel-2"><A href="#Results_Summary"><SPAN class="tocnumber">3.2</SPAN><SPAN class="toctext">Results Summary</SPAN></A></LI></DIV></DIV><H1><SPAN class="mw-headline" id="Results_Overview">Results Overview</SPAN></H1><H3><SPAN class="mw-headline" id="Aim">Aim</SPAN></H3><P>We aimed to express the enzyme ethene monoxygenase enzyme in <I>Pseudominas Putida</I> and show the oxidation of ethylene to ethylene oxide. This was done by:
</P><OL><LI> Codon Harmonisation (using our inhouse algorithm <A rel="nofollow" class="external text" href="https://2015.igem.org/Team:Sydney_Australia/TransOpt">TransOpt</A>) of <I>Mycobacterium chubuense</I> ethene MO (EtnABCD, or ethene MO) for <I>Pseudomonas putida</I></LI><LI> Expression of EtnABCD in <I>P. putida</I></LI><LI> Co-expression of <I>E.coli</I> GroEL-GroES to enhance folding to ensure highest possible activity </LI><LI> Co-expression of LacI (LacI with constitutive promoter P<SUB>LacIQ1</SUB>) to control EtnABCD expression in <I>P.putida</I></LI><LI> Experimental validation of TransOpt via fluorescence assays using <I>Bacillus subtilis</I> fluorescent protein (BsFP)</LI></OL><H3><SPAN class="mw-headline" id="Experimental_design">Experimental design</SPAN></H3><OL><LI> Cloning EtnABCD Gblocks into pBBRMCS2 vector via <A rel="nofollow" class="external text" href="https://2015.igem.org/Team:Sydney_Australia/goldengate">Golden Gate Assembly</A> and digestion/ligation</LI><LI> Cloning GroEL-GroES into pUCP24 vector via PCR amplification from <I>E.coli</I> genome and digestion/ligation</LI><LI> Cloning LacI-P<SUB>LacIQ1</SUB> into pSB1C3 (note that it carries its own promoter) via PCR amplification and digestion/ligation</LI><LI> Cloning LacI-P<SUB>LacIQ1</SUB> into pUS44 vector for co-expression with EtnABCD (note that pUS44 has different selectable marker so makes screening for presence of both EtnABCD and LacI easier)</LI><LI> Inducing EtnABCD expression in <I>P. putida</I> with IPTG</LI><LI> Assessing if EtnABCD was functional and producing ethylene oxide via a Nito-benzyl-pyridine (NBP) colourimetric assay to detect ethene oxide</LI></OL><H3><SPAN class="mw-headline" id="Summarised_results">Summarised results</SPAN></H3><OL><LI>NBP assay was not successful in detecting ethene oxide after induction of <I>P.putida</I> bearing LacI and EtnABCD, which shows that the enzyme does not work as expected </LI><LI>LacIQ15 partially inhibited <I>lac</I> promoter activity in the absence of IPTG </LI><LI>Fluorescence assay of BsFP showed that protein from standard harmonised algorithm had higher fluorescence than the TransOpt generated, indicating future work is required on the TransOpt algorithm</LI></OL><H3><SPAN class="mw-headline" id="What_did_we_find.3F">What did we find?</SPAN></H3><OL><LI> Unable to make functional heterologous expression of EtnABCD in <I>P.putida</I></LI><LI> Characterisation of LacI and P<SUB>LacIQ1</SUB> constitutive promoter functionality in regulating the <I>lac</I> operon</LI><LI> TransOpt did not successfully optimise heterologous translation and folding, whilst standard harmonisation did </LI></OL><H1><SPAN class="mw-headline" id="Full_Results">Full Results</SPAN></H1> ... for the keen beans out there.
<H3><SPAN class="mw-headline" id="I._LacI_Characterisation"><SPAN style="color:#000000"> I. LacI Characterisation </SPAN></SPAN></H3><H4><SPAN class="mw-headline" id="Experiment">Experiment</SPAN></H4><P>LacI-P<SUB>LacIQ1</SUB> consisting of LacI represser gene (from the <I>lac</I> operon system) and P<SUB>LacIQ1</SUB> promoter upstream of the gene were PCR amplified from the pET15B vector using primers bearing cut sites including SalI, EcoRI and PstI for ligation into pUS44 and pSB1C3 vectors. After PCR clean up using Qiagen PCR kit, we performed digestion and ligation (as prescribed in the protocols page) to generate <A rel="nofollow" class="external text" href="https://2015.igem.org/Team:Sydney_Australia/Parts">pSB1C3-LacI-P<SUB>LacIQ1</SUB></A> (pSB-LacI) plasmid and transformed it into <I>P. putida</I> KT2440 cells followed by screening using chloramphenicol (Cm25) plates. Then, we set up patch colony plates and performed colony PCR screening using Taq polymerase to pick colonies bearing the correct pSB-LacI vectors. To further clarify the PCR results, we also performed a single digest to infer the presence of LacI in the final vector by the total fragment size. 
</P><DIV class="center"><DIV class="thumb tnone"><DIV class="thumbinner" style="width:502px;"><DIV class="thumbcaption">LacI with the upstream strong P<SUB>LacIQ1</SUB> constitutive promoter in pUS44 vector.</DIV></DIV></DIV></DIV><H4><SPAN class="mw-headline" id="Characterisation">Characterisation</SPAN></H4><P>To characterise the LacI repressor gene, we made <I>E.coli</I> JM109 competent cells with the pBBRMCS2 vector containing the LacZ-alpha gene. Then, we transformed the cells with LacI-pSB and incubated them over the following media to test the interactions between lacI and P<SUB>LacIQ1</SUB> in this system. These media were:</P><OL><LI>LB-Cm-Km-Xgal (repressed, expect white colonies due to lack of IPTG)</LI><LI>LB-Cm-Km-Xgal-glucose (strongly repressed, expect white colonies due to lack of IPTG and presence of glucose)</LI><LI>LB-Cm-Km-Xgal-IPTG (induced, expect blue colonies due to presence of IPTG)</LI><LI>LB-Km-Xgal-IPTG (strongly induced, expect blue colonies due to presence of IPTG and loss of plasmid carrying lacI repressor gene)</LI></OL><DIV class="center"><DIV class="thumb tnone"><DIV class="thumbinner" style="width:702px;"><DIV class="thumbcaption">Plates of the four different media mentioned above to test for the activity of P<SUB>LacIQ1</SUB>-LacI.</DIV></DIV></DIV></DIV><P>The results of this experiment were not entirely conclusive (above). Cells grown on LB-Cm-Km-Xgal-glucose were white, as expected, and cells grown on LB-Km-Xgal-IPTG were predominantly blue, as expected, but the other two media yielded a mix of white and blue cells, mostly white. It was notable that addition of IPTG was not sufficient to give uniform positive induction of lacZ. Our interpretation is that in this experimental setup, LacI repression is exceedingly strong. We believe this is because the amount of LacI protein expressed from the strong promoter on a high-copy plasmid will completely repress expression of the chromosomally-encoded omega fragment of LacZ, even when IPTG is present.
</P><H4><SPAN class="mw-headline" id="Purpose">Purpose</SPAN></H4><P>The LacI-P<SUB>LacIQ1</SUB> part was added to the project to tackle the issue of possible toxicity to the host cells due to overexpression of ethene monooxygenase, as a direct outcome from our <A rel="nofollow" class="external text" href="https://2015.igem.org/Team:Sydney_Australia/BusinessModel">HP work</A>. From previous work, it was known that the enzyme is difficult to clone and express in heterologous hosts. Consquently, given primary cloning vector (pBBR1MCS-2) only had the Plac promoter but no LacI repressor, we needed to add the latter part separately. This added a second level of necessary control over the expression of ethene MO. 
</P><H3><SPAN class="mw-headline" id="II._GroEL-GroES"><SPAN style="color:#000000"> II. GroEL-GroES </SPAN></SPAN></H3><H4><SPAN class="mw-headline" id="Experiment_2">Experiment</SPAN></H4><P>GroEL-GroES (GroES) experimental procedure evolved into two different phases due to the unsatisfactory results obtained from phase I necessitating the design of a second approach. 
</P><P><B>Phase I</B></P><P>We started by PCR amplifying the two Glocks containing <I>P. putida</I> GroES open reading frames (ORF). After several attempts, we managed to amplify the Gblocks and used them in the Golden Gate cloning reaction with digested PCR linearised pSB1C3 vector (PCR linearised pSB1C3 primers were designed so as not to amplify the RFP and are more suitable for ligation reactions). To screen for successful clones, we transformed the ligation mix into <I>E. coli</I> JM109 cells on agar plates containing chloramphenicol (Cm). After an overnight incubation, the colonies were patched and screened via PCR colony with primer designed to amplify the GroES-pSB1C3 (Gro-pSB) junctions. The clones showing the correct bands for the junctions were grown, and after miniprep, the plasmid was undergone a single and a double digest to check for the presence of the correct insert based on their size after running on agarose electrophoresis. Unfortunately, the digestion analysis gave negative results for all clones, with unexpected band sizes uncharacteristic of the anticipated 2 kbp of GroES. 
</P><P><B>Phase II</B></P><P>The absence of success in phase I contrived us to try a 'plan B'. In this phase, we amplified the <I>E. coli</I> GroES gene from its chromosomal DNA by PCR, with the primer containing the appropriate cut sites for cloning into pUCP24 vector (note: this <I>E.coli</I> GroES cannot be submitted as a part hence it was not cloned into pSB1C3). Then, using a simple digestion and ligation reaction, pUCP24-GroES (pUC-Gro) recombinant vector was constructed and screened via transformation into JM109 cells in media containing gentamicin (Gm). Clones were obtained, however, due to change in the course of project approach and design, this construct was not further analysed to confirm correct ligation. 
</P><DIV class="center"><DIV class="thumb tnone"><DIV class="thumbinner" style="width:502px;"><DIV class="thumbcaption">pUCP24 recombinant vector containing <I>e.coli</I> GroEL-GroES genomic DNA.</DIV></DIV></DIV></DIV><H4><SPAN class="mw-headline" id="Characterisation_2">Characterisation</SPAN></H4><P>Unfortunately due to the nature of results, lack of time, and change in the overall method design of the project, GroES was not used and characterised. However, we hope future iGEM teams take up this challenge and contribute to the discussion regarding the idea that GroES co-expression in different clones enhances expression and activity of enzymes. 
</P><H4><SPAN class="mw-headline" id="Purpose_2">Purpose</SPAN></H4><P>Due to low ethene oxidation activity of <I>P. putida</I> containing the wild-type EtnABCD in the previous work, we believed that GroES might help with the folding and help increase its activity. This method follows our hypothesis that the low activity was due to incomplete folding into the native conformation. The decision to choose <I>E. coli</I> GroES rather than the initial <I>P. putida</I> was due to reports that the former functions acceptablly in <I>P. putida</I><SUP>1</SUP>. Hence, the decision was made to generate <I>E. coli</I> GroES which can also be used later in <I>E.coli</I> with EtnABCD.  
</P><H3><SPAN class="mw-headline" id="III._EtnABCD_Characterisation"><SPAN style="color:#000000"> III. EtnABCD Characterisation </SPAN></SPAN></H3><P>This is the main part of our experiment. 
</P><H4><SPAN class="mw-headline" id="Experiment_3">Experiment</SPAN></H4><P>As mentioned in the project page and the above experiment, we addressed the problems in the previous work, GroES and other challenges by changing the following in our method: 
</P><UL><LI>Generation of a new harmonised EtnABCD sequence using TransOpt algorithm appropriate and optimised for expression in <I>P.putida</I></LI><LI>Co-expression of LacI repressor protein to control the expression of ethene MO as it is under the control of <I>lac</I> promoter (all part of <I>lac</I> operon gene regulatory system)</LI></UL><P>After running the Golden Gate reaction containing the three G-blocks and digested PCR linearised pSB1C3, followed by transformation into JM109 <I>E. coli</I> and selection on Cm selection plates, we screened the colonies by inoculating them in LB with Cm, performing miniprep and digestion analysis. The two digestion reactions (double and single) were conducted followed by agarose gel electrophoresis. The clones showing the right bands (one large band and two bands in each reaction in single and double digests, respectively) were used to construct pBBRMCS2-EtnABCD (pBBR-Etn) plasmid. The <A rel="nofollow" class="external text" href="https://2015.igem.org/Team:Sydney_Australia/Parts">pSB1C3-EtnABCD</A> (pSB-Etn) vector was digested and ligated with digested pBBR vector. After transformation into <I>P.putida</I> KT2440 strain by electroporation and screening over kanamycin (Kan) plates, the clones were screened via colony PCR. The clones showing results that inferred the presence of Etn in pBBR vector were electrotransformed into <I>P.putida</I> KT2440 competent cells. The transformed competent KT2440 cells were incubated to express ethene MO (constitutively due to the 'leakiness' of <I>lac</I> promoter) and exposed to ethene to check for activity by screening for the presence of ethene oxide. The screening is performed via NBP assay which turns purple upon the presence of ethane oxide. 
</P><P>To add the extra level of control of ethene MO expression, the co-expression of LacI alongside ethene MO was also performed. 
</P><DIV class="center"><DIV class="thumb tnone"><DIV class="thumbinner" style="width:502px;"><DIV class="thumbcaption">EtnABCD-pBBR1MCS2 with ethene MO experssion under <I>lac</I> operon control.</DIV></DIV></DIV></DIV><H4><SPAN class="mw-headline" id="Characterisation_3">Characterisation</SPAN></H4><DIV class="center"><DIV class="thumb tnone"><DIV class="thumbinner" style="width:402px;"><DIV class="thumbcaption">Experimental outline of the characterisation of EtnABCD under the control of LacI repressor protein as part of the <I>lac</I> operon.</DIV></DIV></DIV></DIV><P>
The etnABCD genes were cloned from pSB1C3 into the broad host range vector pBBR1MCS-2, where they were expressed from the lac promoter, and electrotransformed into P. putida KT2440. Note that in the host that lacks LacI, the lac promoter is constitutive. The culture was exposed to ethene, but none of the expected product (ethene oxide) was formed, based on testing with the colorimetric NBP assay. In a similar experiment, the wild-type etnABCD genes cloned in the same plasmid yielded detectable ethene oxide, so all the experimental setup and reagents were working.
</P><P>We believed that the constitutive expression of ethene MO directly after transformation would put too much stress on the cells, resulting in the growth of deletion mutants or other unexpected incorrect plasmid derivatives. Hence, we introduced LacI to control the expression of the system on a second plasmid, an RSF1010 derivative called pUS44, and reintroduced the pBR1MCS-2/etnABCD ligation mixture. However, after IPTG induction, the NBP assay still failed to detect ethene oxide, which suggests that the inactivity lies within the enzyme itself, not the cells. Testing the assay reagents with a pure epoxide solution (styrene oxide) yielded a strong purple colour, which confirmed our suspicion that the EtnABCD enzyme itself was to blame.
</P><P>One reason for the ethene MO inactivity could be that our in-house harmonisation tool TransOpt did not yield a sequence which could be properly folded and active in <I>Pseudomonas</I>, since this was the result unexpectedly obtained in parallel experiments with different codon-optimised forms of a fluorescent protein BsFP (outlined below). 
</P><DIV class="center"><DIV class="thumb tnone"><DIV class="thumbinner" style="width:402px;"><DIV class="thumbcaption">Expected results from the NBP assay. The assay results in a qualitative purple colour in the presence of epoxides (as indicated in tubes 2 and 4), with quantitative analysis enabled by spectrophotometric measurement of the solution. Credit to Samantha Chung</DIV></DIV></DIV></DIV><H4><SPAN class="mw-headline" id="Purpose_3">Purpose</SPAN></H4><P>The motivation for the use of Golden Gate cloning method is outlined in the <A rel="nofollow" class="external text" href="https://2015.igem.org/Team:Sydney_Australia/goldengate">Golden Gate</A> information page. Furthermore, the use of weaker ribosomal binding sites (RBS) upstream of each ORF and harmonisation of ethene MO sequence all aid the efficient translation and folding of the enzyme into its native conformation. Codon harmonisation ensures that the ribosome translation kinetics over ethene MO are conserved, as it affects protein folding, while weaker RBS ensures the inhibition of ribosomal traffic on the mRNA which can adversely affect mRNA translation and protein folding. It is also explained above why the LacI system was introduced into the <I>P. putida</I> expression of ethene MO.
</P><DIV class="center"><DIV class="thumb tnone"><DIV class="thumbinner" style="width:402px;"><DIV class="thumbcaption">Schematic showing the use of <I>lac</I> operon in ctrolling EtnABCD expression in KT2440 <I>p.putida</I> LacI competent cells.</DIV></DIV></DIV></DIV><H1><SPAN class="mw-headline" id="TransOpt:_Experimental_Validation">TransOpt: Experimental Validation</SPAN></H1><P>In order to assess the performance of our novel codon optimisation algorithm we performed an experimental test to investigate its capability to generate a functional mRNA sequence that yields well-folded and highly functional proteins. Though we used this same algorithm to generate an optimised sequence for ethene MO in the target host <I>p.putida</I> (to preserve the translational kinetics profile of the ethene MO mRNA between the native and heterologous host), we saw it appropriate to perform a more simple and targeted validation test, via which we could assess other assumptions made in our work. This is because ethene MO would be too complex a protein for us to (on a single-case basis) reliably analyse and draw conclusions regarding the performance of the algorithm, as there are too great a number of contributing variables to the activity of the enzyme. Hence, we decided to use fluorescent protein as the target of our algorithm. Fluorescent proteins, such as GFP, are used widely in the literature as proteins to test the viability of codon optimisation algorithms (as discussed on the <A rel="nofollow" class="external text" href="https://2015.igem.org/Team:Sydney_Australia/TransOpt">TransOpt page</A>), with the major assumption being that the level of fluorescence is proportional to the efficacy of protein production and folding into the native conformation. 
</P><P>In our study, the <I>b. subtillis</I> flavin-binding fluorescent protein (BsFP) was used as a marker to test the validity of the different methods for codon optimisation. The particular protein (BsFP) is a promising new alternative to the green fluorescent protein (GFP) family of proteins: BsFP is a small protein (137 amino acids), it is functional in many different cellular environments and has high tolerance to changes in pH, oxygen levels and heat, making it applicable to a wide range of experiments. In contrast, GFP is much larger, matures slowly, absolutely requires oxygen for fluorescence, and is sensitive to chemical changes <SUP>1</SUP>. Furthermore, in the context of our planned experiments, the lack of genome sequence for the GFP source organism <I>aequorea victoria</I> means that no tRNA GCN profile is available, which is crucial for the newly-developed harmonisation algorithm. However, the choice of this fluorescent protein does present downsides: It may not include complex folding domains, the presence of which motivated TransOpt's designed capability for regulating translation rate trends. Furthermore, fluorescent markers were the prototypical proteins upon which many of the existing codon optimisation methods have been developed, and so it would not be surprising if standard optimisation methods succeed here, though they may fail on more complex sequences (see <A rel="nofollow" class="external text" href="https://2015.igem.org/Team:Sydney_Australia/TransOpt#Scientific_Background">Scientific Background of TransOpt</A>). Thus, without a complex folding structure, the benefits afforded by TransOpt might be minimal (for example, in comparison to standard codon harmonisation algorithms).
</P><P>The following four BsFP sequences were generated, cloned in to pUS212 vector and induced in tetracycline (TetR) - in low amounts that do not exhibit antibiotic properties - and induced in <I>e.coli</I> to verify the algorithm. We generated all sequences ourselves, as described on the <A rel="nofollow" class="external text" href="https://2015.igem.org/Team:Sydney_Australia/TransOpt">TransOpt page</A>, using a rate quantisation methodology based on the organisms' tRNA Gene Copy Numbers.
</P><UL><LI>BsFP-WT: native sequence from the native host B. subtilis</LI><LI>BsFP-fast: all codons were replaced with the E.coli versions that possessed the fastest translation rate, as quantised using the approach of Spencer <I>et al</I><SUP>3</SUP></LI><LI>BsFP-standard: harmonised BsFP generated via standard harmonisation (as proposed by Angov et al 1) using the rate quantisation approach of Spencer <I>et al</I><SUP>3</SUP></LI><LI>BsFP-TransOpt: optimised BsFP sequence generated using our new TransOpt algorithm</LI></UL><P>By measuring changes in fluorescence, the folding of the protein can be determined. We expect that high fluorescence will be detected from the standard harmonised variant (as demonstrated by past studies), but we hope that the TransOpt variant's fluorescence will be better still, due to superior folding than the other variants. We expect the fast optimised sequence will demonstrate a decreased fluorescence compared to the wild type, as these rate-maximisation approaches to optimisation have been shown to be flawed in past studies (see <A rel="nofollow" class="external text" href="https://2015.igem.org/Team:Sydney_Australia/TransOpt#Existing_Approaches_to_Optimisation">discussion on the TransOpt page</A>). 
</P><H3><SPAN class="mw-headline" id="Method">Method</SPAN></H3><P>
All four BsFP GBlocks were designed to have the BioBrick prefix and suffix at either ends. Firstly, BsFP was cloned into pUS212 expression vector via digestion/ligation. To screen for the recombinant vector, the ligation mix was transformed into JM109 <I>E.coli</I> and screened over agar LB plates containing chloramphenicol (Cm). Then, the colonies were used for colony PCR to check for the presence of the ligated insert and subsequently patched on a new plate with TetR (100 ng/uL) to induce BsFP expression. To gain an immediate qualitative data of the induced patch colonies, they were exposed to wide range UV light and screened for presence of green fluorescent colonies. Using this knowledge and the PCR colony results after running agarose gel electrophoresis, an equal number of appropriate colonies of each BsFP variant was picked, inoculated, and induced in LB broth containing Cm and TetR (TetR is added after the culture reaches the appropriate OD<SUB>600</SUB>. Finally, four hours after the addition of the inducer, the samples were taken and had their OD<SUB>600</SUB> and fluorescence measured with excitation at 460 nm and emission at 500-520 nm, with controls such as GFP and empty pUS212 vector containing no BsFP gene. To normalise the data, the fluorescence arbitrary unit/OD<SUB>600</SUB> (Flou/OD) of the empty pUS212 containing colonies was subtracted from each BsFP variants' Flou/OD. 
</P><DIV class="center"><DIV class="thumb tnone"><DIV class="thumbinner" style="width:602px;"><DIV class="thumbcaption">pSB1C3-BsFP vector construct for all variants.</DIV></DIV></DIV></DIV><P>To develop the <A rel="nofollow" class="external text" href="https://2015.igem.org/Team:Sydney_Australia/Parts">pSB1C3-BsFP</A> plasmid, BsFP Gblock was digested/ligated, transformed and screened by Cm, colony PCR and sequencing. 
</P><H3><SPAN class="mw-headline" id="Results_Summary">Results Summary</SPAN></H3><P>Our raw results are in the figure below, demonstrating that the standard codon harmonisation method and the fast-codon optimised method performed as expected when compared to the wild type. However, the TransOpt method did not function as desired, producing levels of fluorescence indistinguishable (to within experimental error margins) from those in the control case. Aside from the distinct possibility of errors in the experimental procedure  used to prepare this variant for measurement, a number of sequence-based causes may have been responsible for this negative result, as discussed on the <A rel="nofollow" class="external text" href="https://2015.igem.org/Team:Sydney_Australia/TransOpt#Experimental_Validation">TransOpt Experimental Validation page.</A></P><DIV class="center"><DIV class="thumb tnone"><DIV class="thumbinner" style="width:652px;"><DIV class="thumbcaption">Fluorescence measurements (in arbitrary units) for BsFP fluorescent protein variants. From left to right the strains are wild type (WT), fast codon optimised (fast), standard codon harmonisation (standard), TransOpt codon optimisation (TransOpt).</DIV></DIV></DIV></DIV><P>In conclusion, we were able to achieve the following aims:
</P><UL><LI>Demonstrated that the sequence generated via the standard codon harmonisation algorithm was able to give improved fluorescence, supporting the rate quantisation methodology we have employed.</LI><LI>Demonstrated that the sequence generated via selecting fast-translating codons results in decreased fluorescence, supporting our hypothesis that the use of only fast-translating codons may result in slowed (and possibly stalled) translation. </LI></UL><P>However, we came short of reaching the following aim:
</P><UL><LI>Demonstrate that TransOpt improves upon the standard codon harmonisation method.</LI></UL><P>For full analysis of these results, discussion of their implications for the effectiveness of our algorithm, and identification of potential avenues for future improvement, please see the <A rel="nofollow" class="external text" href="https://2015.igem.org/Team:Sydney_Australia/TransOpt">TransOpt</A> page. 
</P><DIV class="center"><DIV class="thumb tnone"><DIV class="thumbinner" style="width:502px;"><DIV class="thumbcaption">Induced colonies exposed to the UV light, with the BsFP optimised using standard codon harmonisation qualitatively showing the highest fluorescence.</DIV></DIV></DIV></DIV><P><B>References</B></P><P><SUP>1</SUP> Venturi V <I>et al.</I> Amplification of the groESL operon in Pseudomonas putida increases siderophore gene promoter activity. <I>Mol Gen Genet.</I> 1994 Oct 17;245(1):126-32.
</P><P><SUP>2</SUP> Mukherjee, A., et al., <I>Characterization of flavin-based fluorescent proteins: an emerging class of fluorescent reporters.</I> PLoS One, 2013. 8(5): p. e64753.
</P><P><SUP>3</SUP> Spencer P <I>et al.</I>, 2012, “Silent Substitutions Predictably Alter Translation Elongation Rates and Protein Folding Efficiencies”, Vol 422, pp. 328-335.
</P></DIV></DIV></DIV></DIV></DIV></DIV></BODY></HTML>