<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_Sydney_Australia_Experiments skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:Sydney Australia/Experiments</SPAN></H1><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><DIV id="menuContainer"><DIV id="menuContainer2"><NAV><UL><LI><A href="https://2015.igem.org/Team:Sydney_Australia">Home</A></LI><LI><A href="https://2015.igem.org/Team:Sydney_Australia/achievements">Achievements</A></LI><LI><A href="#">Project</A></LI><LI><A href="https://2015.igem.org/Team:Sydney_Australia/project_overview">Overview</A></LI><LI><A href="https://2015.igem.org/Team:Sydney_Australia/Description">Description</A></LI><LI><A href="https://2015.igem.org/Team:Sydney_Australia/Experiments">Protocols</A></LI><LI><A href="https://2015.igem.org/Team:Sydney_Australia/goldengate">Golden Gate</A></LI><LI><A href="https://2015.igem.org/Team:Sydney_Australia/Results">Results</A></LI><LI><A href="https://2015.igem.org/Team:Sydney_Australia/Notebook">Notebook</A></LI></UL><LI><A href="https://2015.igem.org/Team:Sydney_Australia/Parts">Parts</A></LI><LI><A href="#">Modelling</A></LI><LI><A href="https://2015.igem.org/Team:Sydney_Australia/Modeling">Overview</A></LI><LI><A href="https://2015.igem.org/Team:Sydney_Australia/TransOpt">TransOpt</A></LI><LI><A href="https://2015.igem.org/Team:Sydney_Australia/Modeling/visualisation">Visualisation</A></LI><LI><A href="https://2015.igem.org/Team:Sydney_Australia/Modeling/ElectroStop">ElectroStop</A></LI><LI><A href="https://2015.igem.org/Team:Sydney_Australia/Modeling/Supplementary">Supplementary Material</A></LI><LI><A href="#">Human Practices</A></LI><LI><A href="https://2015.igem.org/Team:Sydney_Australia/Practices">Overview</A></LI><LI><A href="https://2015.igem.org/Team:Sydney_Australia/EthicsMotivation">Motivation &amp; Analysis</A></LI><LI><A href="https://2015.igem.org/Team:Sydney_Australia/Outreach">Outreach</A></LI><LI><A href="https://2015.igem.org/Team:Sydney_Australia/Media">Media</A></LI><LI><A href="https://2015.igem.org/Team:Sydney_Australia/Collaborations">Collaboration</A></LI><LI><A href="https://2015.igem.org/Team:Sydney_Australia/BusinessModel">Business Model</A></LI><LI><A href="https://2015.igem.org/Team:Sydney_Australia/Safety">Safety</A></LI><LI><A href="https://2015.igem.org/Team:Sydney_Australia/Attributions">Attributions</A></LI><LI><A href="https://2015.igem.org/Team:Sydney_Australia/Team">Team</A></LI></NAV></DIV></DIV><DIV id="mainContainer"><DIV id="contentContainer"><DIV style="float: right"><DIV id="toc" class="toc"><DIV id="toctitle"><H2>Contents</H2></DIV><UL><LI class="toclevel-1 tocsection-1"><A href="#Agarose_Gel_Electrophoresis"><SPAN class="tocnumber">1</SPAN><SPAN class="toctext"><B>Agarose Gel Electrophoresis</B></SPAN></A></LI><LI class="toclevel-1 tocsection-2"><A href="#PCR"><SPAN class="tocnumber">2</SPAN><SPAN class="toctext"><B>PCR</B></SPAN></A></LI><LI class="toclevel-1 tocsection-3"><A href="#Restriction_Digest"><SPAN class="tocnumber">3</SPAN><SPAN class="toctext"><B>Restriction Digest</B></SPAN></A></LI><LI class="toclevel-1 tocsection-4"><A href="#Ligation"><SPAN class="tocnumber">4</SPAN><SPAN class="toctext"><B>Ligation</B></SPAN></A></LI><LI class="toclevel-2 tocsection-5"><A href="#Golden_Gate_Assembly"><SPAN class="tocnumber">4.1</SPAN><SPAN class="toctext"><B>Golden Gate Assembly</B></SPAN></A></LI><LI class="toclevel-2 tocsection-6"><A href="#Heat-shock_Transformation"><SPAN class="tocnumber">4.2</SPAN><SPAN class="toctext"><B>Heat-shock Transformation</B></SPAN></A></LI><LI class="toclevel-2 tocsection-7"><A href="#P.putida_Competent_Cells"><SPAN class="tocnumber">4.3</SPAN><SPAN class="toctext"><B><I>P.putida</I> Competent Cells </B></SPAN></A></LI><LI class="toclevel-2 tocsection-8"><A href="#E.coli_Competent_Cells"><SPAN class="tocnumber">4.4</SPAN><SPAN class="toctext"><B><I>E.coli</I> Competent Cells </B></SPAN></A></LI><LI class="toclevel-2 tocsection-9"><A href="#Transformation_of_P.putida_bacteria_via_Electroporation"><SPAN class="tocnumber">4.5</SPAN><SPAN class="toctext"><B> Transformation of <I>P.putida</I> bacteria via Electroporation </B></SPAN></A></LI><LI class="toclevel-2 tocsection-10"><A href="#IPTG_Induction"><SPAN class="tocnumber">4.6</SPAN><SPAN class="toctext"><B>IPTG Induction</B></SPAN></A></LI><LI class="toclevel-2 tocsection-11"><A href="#NBP_Assay"><SPAN class="tocnumber">4.7</SPAN><SPAN class="toctext"><B>NBP Assay</B></SPAN></A></LI><LI class="toclevel-2 tocsection-12"><A href="#LB_Media"><SPAN class="tocnumber">4.8</SPAN><SPAN class="toctext"><B>LB Media</B></SPAN></A></LI><LI class="toclevel-2 tocsection-13"><A href="#VIOGENE_Gel.2FPCR_DNA.2FRNA_extraction_kit"><SPAN class="tocnumber">4.9</SPAN><SPAN class="toctext"><B>VIOGENE Gel/PCR DNA/RNA extraction kit</B></SPAN></A></LI><LI class="toclevel-2 tocsection-14"><A href="#Bioline_ISOLATE_II_Plasmid_Mini_kit"><SPAN class="tocnumber">4.10</SPAN><SPAN class="toctext"><B>Bioline ISOLATE II Plasmid Mini kit</B></SPAN></A></LI><LI class="toclevel-2 tocsection-15"><A href="#Plasmid_MidiPrep_-_100_mL"><SPAN class="tocnumber">4.11</SPAN><SPAN class="toctext"><B>Plasmid MidiPrep - 100 mL</B></SPAN></A></LI></UL></DIV></DIV><H2><B>Agarose Gel Electrophoresis</B></H2><P>Gel electrophoresis is used to separate DNA based on their size and mobility through agarose. It relies on the fact that DNA is negatively charged, and that a uniform electric field can be applied across the semi-permeable agarose gel. 
</P><P>For a 1% agarose gel:
</P><OL><LI> Add 1 g of agarose to 100 mL TBE.</LI><LI> Gently mix and microwave heat until dissolved.</LI><LI> Prepare gel tray. Depending on the tank, use masking tape or end-formers to seal the two ends of the gel tray. If using end formers, seal the ends by pipetting a small volume of gel solution into the contact between the end formers and tank and allow to set.</LI><LI> Allow the gel solution to cool and add 0.5 µL of Gel Red and pour the gel into the gel tray. Alternatively, exclude the Gel Red and stain with Ethidium Bromide after running the gel.</LI><LI> Insert the well comb into the appropriate position in the gel.</LI><LI> Once set remove the comb, tape/end formers and place into electrophoresis chamber and submerge in TBE.</LI><LI> Mix samples with loading buffer (use 1:5 ratio of loading buffer to sample) and pipette into wells.</LI><LI> Apply 120-180V, depending on time constraints and desired resolution.</LI></OL><H2><B>PCR</B></H2><P>The Polymerase Chain Reaction is used to amplify segments of DNA to high concentrations as a screen for the presence of the target or for further manipulation. The following protocol is readily tweaked depending on the specifics of the template DNA, primers and polymerase of choice.
</P><P>PCR of multiple samples is sped up significantly by making a master mix; the volumes below are for one 50 µL reaction, but the volumes would be multiplied by the number of reactions required and mixed together in a single tube. 
</P><UL><LI>5 µL 10x Pfu buffer/NEB Thermopol Buffer </LI><LI>1 µL dNTPs at 10 uM (final 200mM) </LI><LI>1 µL primer (F) (final 0.5-1uM) </LI><LI>1 µL primer (R) (final 0.5-1uM) </LI><LI>40.5 µL sterile MQ water </LI><LI>0.5 µL Pfu/Taq polymerase </LI></UL><OL><LI>Aliquot master mix into PCR tubes (49 µL) then add 1 µL of the template DNA. Alternatively, if performing a colony PCR, aliquot 50 µL of master mix into a tube and resuspend cells directly from the plate into the tube. (Only dip the toothpick 3-5 times in the master mix).</LI><LI>Start the thermocycler at the setting desired. Cycles must have a peak high enough to denature template DNA, a trough low enough to allow annealing of primer pair, followed by an intermediate stage for the optimal polymerase activity.</LI></OL><P>NB. Keep enzyme on ice and return to the freezer ASAP.
</P><H2><B>Restriction Digest</B></H2><P>Restriction digestion employs restriction enzymes which recognise specific sites in a DNA sequence and will cut the strands at those sites. This may be used for diagnostic purposes by separating DNA into fragments of predictable lengths or for assembly by generating short overlaps at the restriction sites.
</P><OL><LI>Add 5 µL of appropriate 10x buffer to a microcentrifuge tube. The buffer choice depends on the restriction enzyme used and can be checked from readily available tables.</LI><LI>Add DNA sample solution (~1 µg DNA).</LI><LI>Make tube up to 49 µL with reverse osmosis water.</LI><LI>Add restriction enzyme; 10 U activity is sufficient, which is normally the activity of 1 µL restriction enzyme solution. The restriction enzyme volume should not be more than 1/10 of the total reaction volume.</LI><LI>Incubate for an hour at the optimal temperature for restriction enzyme activity; again, this should be checked from relevant data tables.</LI></OL><P>Restriction digests with two enzymes simultaneously are also possible and were performed over the course of the project. This involves the same protocol as above except that the reaction mixture is made up to 48 µL in step 3. This is only possible if the enzymes have compatible reaction buffers and optimal temperatures; if they do not, then you must perform two sequential digests with a purification step in between
</P><H2><B>Ligation</B><P>Ligase facilitates binding between complementary sequences of DNA. Ligation allows fragments of DNA with sticky ends to be joined together, but the enzymes also has other uses like in Gibson Assembly. Ligase is not thermostable, so there is an efficiency trade-off between increasing the rate of ligation and the rate of ligase degradation at higher temperatures. The following protocol is for the ligation we performed most often; that of cloning an PCR product into a plasmid prep. The following volumes thus hold only with the concentration of our plasmid prep. The ligation reaction is quite flexible, and the following protocol can be applied to general ligations, as long as care is taken to maintain a ~3:1 molar ratio of insert:vector, and that the total DNA concentration does not exceed 10 ng/µl. 
</P><OL><LI>Digest plasmid e.g. Use 250 ng in digest volume of 100 µL although there is a wide acceptable range for this. </LI><LI>At the same time as step 1, digest insert(s) e.g. Use 1 µg in 100 µL digest. </LI><LI>Purify both digestion samples using DNA extraction and purification kits (e.g. VIOGENE)</LI><LI>Combine on ice 2 µL of NEB 10X T4 ligase buffer, 8.5 µL purified insert, 8.5 µL of purified vector, and 1 µL of T4 concentrated DNA ligase. Make sure ligase enzyme remains in ice at all times.</LI><LI>Incubate the sample for an hour at room temperature, or overnight at 4°C.</LI></OL><P>To avoid column purification of digestion samples, heat inactivated digestion samples can be used for the ligation. In this case, make sure that:
</P><OL><LI>Digestion is set up in small volume e.g. 25 uL</LI><LI>Check that the restriction enzymes used can be heat inactivated.</LI></OL></H2><H2><B><A rel="nofollow" class="external text" href="https://2015.igem.org/Team:Sydney_Australia/goldengate">Golden Gate Assembly</A></B></H2><P>Golden Gate Assembly (GGA) is used for cloning one or more inserts into a vector by designing overlapping ends which can join desired DNA together after cleavage by non-palindromic restriction enzyme, BsaI, followed by ligation to seal the DNA together. 
</P><P>To set up a 15 uL GGA reaction:
</P><OL><LI> Add 1.5 uL <A rel="nofollow" class="external text" href="https://www.neb.com/products/b0202-t4-dna-ligase-reaction-buffer">10X T4 DNA ligase buffer</A> (50mM Tris-HCl, 10mM MgCl2, 1mM ATP, 10mM DTT, pH 7.5@25°C). </LI><LI> Add 1 uL <A rel="nofollow" class="external text" href="https://www.neb.com/products/r0535-bsai">BsaI</A>/<A rel="nofollow" class="external text" href="https://www.neb.com/products/r3535-bsai-hf">BsaI-HF</A> (200 U)</LI><LI> Add 1 uL <B>concentrated</B><A rel="nofollow" class="external text" href="https://www.neb.com/products/m0202-t4-dna-ligase">T4 DNA ligase</A> (2000 U)</LI><LI> Add equimolar amounts of insert(s) and digested vector e.g. 50 ng cut vector</LI><LI> Make up the solution to 15 uL with MiliQ water</LI><LI> Set up thermocycler to cycle between 16 C (1 minute) and 37 C (1 minute) for 30-35 times </LI><LI> After the termination of heat cycle, incubate sample at each 50 C and 80 C for 5 minutes </LI><LI> Transform 3-8 uL of GGA sample to a suitable host </LI></OL><P>Note: Assign volumes of insert and vector such that to avoid total volume of more 15 uL.
</P><P>Refer to <A rel="nofollow" class="external text" href="https://www.neb.com/applications/cloning-and-synthetic-biology/dna-assembly-and-cloning/golden-gate-assembly">NEB Golden Gate Assembly</A> or <A rel="nofollow" class="external text" href="https://2015.igem.org/Team:Sydney_Australia/goldengate">Sydney iGEM Golden Gate</A> explainer for further information on GGA method and design.
</P><H2><B>Heat-shock Transformation</B></H2><P>Transformation is a technique by which DNA may be inserted into competent cells. Transformation occurs naturally when cells uptake and express exogenous DNA from their environment. Heat shock transformation uses a rapid change in temperature to cause plasmids to enter cells via pores in the membrane. The introduced DNA will often contain a marker gene (usually antibiotic resistance) so that successfully transformed cells can be grown on selective media (normally an antibiotic), which corresponds to the marker gene. 
</P><OL><LI> Set-up a heat-block or water-bath at 42°C.</LI><LI> Retrieve aliquots of cells from -80°C freezer. Thaw on ice.</LI><LI> Add 1 pg-100 ng (1 µL - 5 µL) of plasmid DNA into cell suspension.</LI><LI> Incubate tubes at 42°C for 30-45 seconds.</LI><LI> Return to ice and quickly add 1 mL of LB-broth. Incubate tubes for 1 hour on the 37°C shakers.</LI><LI> Spread-plate 100 µL of cells on LB media containing the appropriate antibiotic. Alternatively, the cells can be diluted or concentrated before plating. For instance, centrifuge the tubes, pour off supernatant, resuspend cells in the final drop remaining in the tube and spread-plate the last 100 µL of cells.</LI><LI> Incubate overnight at 37°C.</LI></OL><H2><B><I>P.putida</I> Competent Cells </B></H2><P>Psudomonas electrocompetent cells are used for transformation of plasmid with or without the gene of interest by electroporation. Transformed Pseudomonas cells are grown at 30°C overnight on selective media to screen for insert. 
</P><OL><LI> Inoculate 50mL LB broth with 1-7 days old Pseudomonas cells and incubate at 30°C in shaker overnight.</LI><LI> Next morning, add entire culture to 500mL LB broth and check OD610. </LI><LI> Incubate the culture at 30°C with shaking and check OD610 at 30mins intervals until it is 0.8~1.0. </LI><LI> After transferring the cells into pre-chilled appropriate tubes, centrifuge at 7500g for 15mins at 4°C.</LI><LI> Resuspend the pellet in 30mL cold electroporation buffer and centrifuge at top speed (~4500rpm or ~3000g) for 15mins at 4°C. (Note: 10%Glycerol can be used as electroporation buffer for Pseudomonas putida)</LI><LI> Repeat step 5 for another two times.</LI><LI> After centrifugation, resuspend pellet in 5mL cold electroporation buffer and place on ice.</LI><LI> Pipette into multiple aliquots and store at -80°C.</LI></OL><H2><B><I>E.coli</I> Competent Cells </B></H2><P>
Competent bacterial cells are used for transformation of plasmids to allow propagation of plasmids and for expression of the genes of interest. 
Streak E.coli strain onto LB agar and incubate overnight at 37°C.
</P><OL><LI>Inoculate 5 mL LB broth with a single colony from LB plate. Incubate at 37°C with shaking.</LI><LI>Measure OD600 of overnight culture. Aseptically inoculate 100 mL LB in 500 mL Schott Bottle with enough culture to obtain OD600 of ~ 0.05.</LI><LI>Grow cells at 37°C, shaking until culture reaches OD600 of ~ 0.5.</LI><LI>Aseptically transfer to centrifuge bottles and keep on ice.</LI><LI>Harvest cells at 4000 rpm, 4°C for 10 min in cold centrifuge.</LI><LI>Working in cold room with cells in ice, pour off supernatant and remove residual liquid by pipetting.</LI><LI>Resuspend pellet gently in 33 mL RF1 solution. Mix by pipetting up and down.</LI><LI>Incubate on wet ice for 1 hour.</LI><LI>Pellet bacteria at 4000 rpm 4°C for 10 min.</LI><LI>Again remove supernatant (work in cold room).</LI><LI>Resuspend pellet in 8 mL RF2 solution and mix.</LI><LI>Incubate on wet ice for 15 min.</LI><LI>Working quickly, aliquot 110 µL of cell suspension into pre-chilled microcentrifuge tubes. Once dispensed, collect all tubes and store in -80°C freezer.</LI></OL><H2><B> Transformation of <I>P.putida</I> bacteria via Electroporation </B></H2><P>Electroporation, or electropermeabilization, is a molecular biology technique in which an electrical field is applied to cells in order to increase the permeability of the cell membrane, to allow DNA of interest to be introduced into electrocompetent Pseudomanas cells. 
</P><OL><LI> Pre-chill electroporation cuvettes on ice for at least 10mins. </LI><LI> On ice, aliquot 50uL of electrocompetent cells into microcentrifuge tubes.</LI><LI> Add 1uL of appropriate purified plasmid for positive control; 2~3uL of ligation mix for test; and no DNA for negative control. </LI><LI> Transfer mixtures into electroporation cuvettes.</LI><LI> In &quot;Exponential&quot; protocol on the electroporator, change the settings to 2500 V, 25 µF, and 200 Ω.</LI><LI> Place the dry cuvette in cuvette holder, press &quot;Pulse&quot; and note down time constant. </LI><LI> Add 1mL LB broth to the cuvette and mix.</LI><LI> Repeat step 6 and 7 for all samples. </LI><LI> Aseptically pour the contents into respective new microcentrifuge tubes and incubate with shaking at 30°C for 1.5 hours.</LI><LI> After incubating, plate the cultures on appropriate selective media plates in 100uL aliquots and as resuspended whole pellet.</LI><LI> Incubate the plates at 30°C overnight and observe results.</LI></OL><H2><B>IPTG Induction</B></H2><OL><LI>Grow the appropriate host by inoculating LB culture with the culture and incubating it while shaking</LI><LI>After reaching an OD<SUB>600</SUB> of ~0.1, add IPTG with final concentration of 1 mM</LI><LI>Incubate for 3-4 hours while shaking</LI></OL><P>Note that the incubating temperature depends on the host.
</P><P>Also, the same protocol can be followed by other inducers such as tetracycline (in low amounts) instead of using IPTG, however, the concentration in the inducing sample may be different.
</P><H2><B>NBP Assay</B></H2><P>NBP or 4-(4-nitrobenzyl)pyridine assay is used to evaluate ethene monoxygenase activity. A chromophore is formed between ethene oxide and the nucleophile 4-(p-nitrobenzyl)pyridine (NBP) producing a violet colour.
</P><OL><LI> Add 2mL of MSM media (20mM K<SUB>2</SUB>HPO<SUB>4</SUB>, 5mM (NH<SUB>4</SUB>)<SUB>2</SUB>SO<SUB>4</SUB>, pH7) to plates containing culture of interest.</LI><LI> Homogenise by scraping the cells and transfer to new eppendorf tubes.</LI><LI> Centrifuge at 15,000g and resuspend in 200uL MSM media and take 10uL to check OD600.</LI><LI> Repeat steps 1 to 3 with negative control.</LI><LI> Transfer cell suspensions to 2mL autosampler vials and inject 200uL ethene gas. </LI><LI> Incubate at 30°C overnight with shaking.</LI><LI> Transfer suspension to microcentrifuge tubes and add 500uL of NBP reagent (100mM in ethylene glycol).</LI><LI> Incubate at 95°C on heating block for an hour. </LI><LI> Centrifuge at 13,000rpm for 5 minutes and transfer 500uL of supernatent to a new tube.</LI><LI> Add 500uL of 1:1 (triethylamine: acetone) and observe for colour change and measure OD600 with deionized water as blank.</LI></OL><H2><B>LB Media</B></H2><P>LB media is a common growth media used for the propagation of bacterial cells.
</P><OL><LI>Mix 10 g Tryptone, 5 g Yeast extract, 5 g NaCl and 1 L of water, or use the same ratio in the required volume.</LI><LI>For LB agar solution, add 15 g to the solution as well.</LI><LI>Autoclave the solution to sterilise.</LI><LI>Antibiotics can be added to make the media selective for antibiotic resistant cells. This must be added after autoclaving.</LI></OL><H2><B>VIOGENE Gel/PCR DNA/RNA extraction kit</B></H2><P>Working with DNA often involves adding enzymes (polymerases, restriction enzymes, ligases), which may subsequently need to be removed before they prevent or contaminate the next stage of work. DNA purification is quick and easy using a store-bought Kit. 
</P><P>Also consult the instruction booklet that comes with the Viogene kit – the protocol below only gives the bare essentials required. This kit purifies fragments of DNA (100bp – 10-kb) from agrose gel or other reactions with enzymes, dNTPs, salts and primers without the need for the phenol/chloroform extraction. The system is based on binding up to 20µg DNA to a silica-based membrane in chaotrophic salts with average recoveries of 60 to 90%.
</P><P><I>Gel extraction -</I></P><OL><LI>Use a clean, sharp scalpel or razor blade to excise the gel slice containing the DNA fragment of interest </LI><LI>Measure the weight of the gel slice (about 50-200mg) and place it into a sterile 1.5ml or 2ml centrifuge tube. Add 0.5ml GP buffer. </LI><LI>Incubate at 60°C for 5 to 10 minutes until the gel is competently dissolved. Invert the tube every 1-2 minutes during incubation </LI><LI>Place a GPTM Column onto a Collection Tube and load no more than 0.6ml dissolved gel mixture into the column. Centrifuge for 30-60 seconds and discard the flow-through.</LI><LI>Repeat step 4 for the rest of the mixture.</LI><LI>Wash the column once with 0.5ml of WN Buffer by centrifuging for 30-60 seconds. Discard the flow through. </LI><LI>Wash the column once with 0.5ml of WS Buffer by centrifuging for 30-60 seconds. Discard the flow through. </LI><LI>Centrifuge the column at full speed for 3 minutes or more to remove residual ethanol </LI><LI>Place the column onto a new 1.5ml centrifuge tube and add 15-20µl of Elution Buffer onto the center of the membrane.</LI><LI>Stand the column for 2-3 minutes and centrifuge at full speed for 1-2 minutes to elute DNA. DNA can be stored at -20°C</LI></OL><P><I>PCR Purification –</I></P><OL><LI>Pipet 10-100µl of PCR product or DNA solution after enzymatic reaction to a new 1.5ml centrifuge tube. Add 0.5ml GP Buffer and mix well </LI><LI>Place a GPTM Column onto a Collection Tube. Add all the mixture from step 1 into the column </LI><LI>Centrifuge for 30-60 seconds. Discard the flow-through </LI><LI>Was the column once with o.5ml WN Buffer by centrifuging for 30 – 60 seconds. Discard the flow-through. </LI><LI>Wash the column once with 0.5ml WS Buffer by centrifuging for 30-60 seconds. Discard the flow-through. </LI><LI>Centrifuge the column at full speed for another 3 minutes or more to remove the residual ethanol.</LI><LI>Place the column onto a new 1.5ml centrifuge tube. Add 15-30µl of Elution Buffer onto the center of the membrane. </LI><LI>Stand the column for 2-3 minutes and centrifuge at full speed fro 1-2 minutes to elute DNA </LI><LI>Store DNA at 4°C or -20°C</LI></OL><H2><A rel="nofollow" class="external text" href="http://www.bioline.com/au/downloads/dl/file/id/1219/isolate_ii_plasmid_mini_kit_protocol.pdf"><B>Bioline ISOLATE II Plasmid Mini kit</B></A></H2><P>The miniprep was performed using the Bioline ISOLATE plasmid mini kit II and the protocol and more information about the kit can be found <A rel="nofollow" class="external text" href="http://www.bioline.com/au/isolate-ii-plasmid-mini-kit.html">here</A>.
</P><P><B>Isolation of High Copy Plasmid DNA from <I>e.coli</I></B></P><OL><LI>Pellet 1-5 mL of a saturated <I>E. coli</I> culture for 30 s and collect pellet, removing as much supernatant as possible</LI><LI>Add 250 uL of Resuspension buffer P1 and resuspend by vortexing/pipetting up and down</LI><LI>Add 250 uL Lysis buffer P2, mixing by gently inverting 6-8 times</LI><LI>After incubation for 5 minutes at room temperature, add 300 uL Neutralisation buffer P3, inverting 6-8 times</LI><LI>Centrifuge the sample for 5 minutes and collect supernatant, taking care not to dissolve the pellet</LI><LI>Place the supernatant - 750 uL at a time - into the spin column placed inside a collection tube</LI><LI>Add 500 uL 50<SUP>o</SUP>C preheated Wash Buffer PW1 and centrifuge for 1 minutes, discarding flow through</LI><LI>Add 600 uL Wash Buffer PW2 and centrifuge for 1 minutes, discarding flow through </LI><LI>Centrifuge for another 2 minutes to remove residual ethanol </LI><LI>Place spin column into a 1.5 mL microcentrifuge tube and add 50 uL Elution buffer directly on to the silica membrane </LI><LI>After 1 minute incubation at room temperature, centrifuge for 1 minute.</LI></OL><P><B>Isolation of Low Copy Plasmid, P1 Constructs or Cosmid DNA from <I>e.coli</I></B></P><OL><LI>Pellet 5-10 mL of a saturated <I>E. coli</I> culture for 30 s and collect pellet, removing as much supernatant as possible</LI><LI>Add 250 uL of Resuspension buffer P1 and resuspend by vortexing/pipetting up and down</LI><LI>Add 250 uL Lysis buffer P2, mixing by gently inverting 6-8 times</LI><LI>After incubation for 5 minutes at room temperature, add 300 uL Neutralisation buffer P3, inverting 6-8 times</LI><LI>Centrifuge the sample for 10 minutes and collect supernatant, taking care not to dissolve the pellet</LI><LI>Place the supernatant - 750 uL at a time - into the spin column placed inside a collection tube</LI><LI>Add 500 uL 50<SUP>o</SUP>C preheated Wash Buffer PW1 and centrifuge for 1 minutes, discarding flow through</LI><LI>Add 600 uL Wash Buffer PW2 and centrifuge for 1 minutes, discarding flow through </LI><LI>Centrifuge for another 2 minutes to remove residual ethanol </LI><LI>Place spin column into a 1.5 mL microcentrifuge tube and add 50 uL Elution buffer directly on to the silica membrane </LI><LI>After 2 minutes of incubation at 70<SUP>o</SUP>C, centrifuge for 1 minute.</LI></OL><P>Note that centrifugations occur at 11000 x g.
</P><H2><B>Plasmid MidiPrep - 100 mL</B></H2><P>This is the process used to extract plasmids from bacterial cells. Plasmids can then be used for screening or further manipulation.
</P><OL><LI> Pellet 100 mL culture in 2 x 50 mL Falcon tubes. Resuspend cells in 4 mL TE buffer, combine resuspended pellets in one tube.</LI><LI> Add 8 mL lysis solution (SDS-OH), mix well by inversion and shaking(~10 sec). Should go viscous. Leave at room temp for 15 min.</LI><LI> Add 6 mL ice-cold precipitation solution (K.Ac). Shake briefly – essential that K.Ac is thoroughly mixed in. Viscosity should disappear, and white precipitate appears. Keep on ice 15 min.</LI><LI> Spin at top speed (4000 rpm) in cold Centaur centrifuge for 15 min. Recover tube immediately and handle gently (pellet is soft and easily resuspended). Pour supernatant into new tube. Try to avoid the white junk, but don’t worry if little bits of it get transferred.</LI><LI> Add an equal volume isopropanol (~15 mL), mix well ice 15 min.</LI><LI> Spin at top speed (4000 rpm) in Centaur centrifuge (doesn’t need to be cold) for 15 min, pour off supernatant, keep pellet.</LI><LI> Add 10 mL 70% ethanol to pellet, resuspend by brief vortexing, leave for 5 min at room temp. Spin 15 min in Centaur centrifuge (doesn’t need to be cold). Pour off supernatant again.</LI><LI> Drain off excess supernatant by gentle tapping on paper towel, then heat at 50°C for approx 10 min to remove ethanol and isopropanol.</LI><LI> Redissolve pellet in 2 mL TE with mixing (tapping tube ~ 1 min , don’t vortex too much from this point onward). Can heat if necessary (eg. 50°C, 10-30 min). Note that plasmid DNA is much more soluble than chromosomal DNA, and dissolves preferentially. Solution should look slightly viscous (traces of chromosomal DNA still present).</LI><LI> Split prep into 2 x 1 mL in Eppendorf tubes. Extract each tube with phenol: chloroform: isoamyl (PCI), as follows. Suck up 500 µL PCI from under the aqueous layer in the reagent bottle, transfer to Eppi tube. Vortex for ~5-10 sec until a uniform milky white emulsion is obtained. Centrifuge 5 min. Transfer top phase (aqueous) to a new Eppi tube. Discard bottom phase (PCI) into phenol waste. Avoid the white junk at the interface between phases.</LI><LI> Repeat solvent extractions using 500 µl chloroform:isoamyl (CI, µl) per tube, as described for PCI above. After mixing &amp; centrifugation, keep top aqueous phase, discard bottom CI phase into waste.</LI><LI> Split DNA prep into 4 equally sized aliquots (~400 µl each) Precipitate DNA by adding 1/10-volume 3M Na-acetate (~40 µl) to each tube, and then add 2 volumes cold ethanol (~1 ml). Incubate &gt;2 hr at -20°C (overnight is fine).</LI><LI> Spin for 10 min, drain off supernatant, rinse pellet with 70% ethanol (as above, but using 500 µl 70% EtOH), drain excess EtOH off, then dry 10 min at 50°C.</LI><LI> Redissolve plasmid in 100 µL EB. Expected yields range from approx 10 µg plasmid per ml culture (eg pUC/pGEM) down to 0.2 µg plasmid per mL culture (eg. RSF1010). Final expected DNA conc. may range from approx 10-500 ng/µL.</LI></OL><P>Solutions: (see Sambrook Appendix 1) [HARRY - WHAT?? do we have a reference to put in here?]
</P><UL><LI>TE (solution I): 10 mM Tris, 10 mM EDTA, pH 8. Autoclaved.</LI><LI>Lysis sol’n (solution II): 0.2 M NaOH, 1% SDS. Prepare fresh from separate stocks (NaOH – 2 M, Autoclaved ; SDS – 10%) </LI><LI>Precipitation sol’n (solution III): 3 M potassium, 5 M acetate, pH 4.8. Autoclaved.</LI><LI>Na-acetate: 3M, pH 4.8 (adjust with conc. acetic acid), autoclaved.</LI><LI>EB (Elution buffer): 5 mM Tris, pH 8. Autoclaved. </LI></UL><P>NOTE: RNAse can be added to TE buffer at the start, or to the EB/TE at the end. RNA doesn’t interfere with most things, but can make gels look messy and obscure small DNA bands. Add RNase from conc., boiled stock (10 mg/ml) to final conc. of ~100 µg/ml.
</P></DIV></DIV></DIV></DIV></DIV></DIV></BODY></HTML>