<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_Sydney_Australia_Modeling_visualisation skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:Sydney Australia/Modeling/visualisation</SPAN></H1><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><DIV id="menuContainer"><DIV id="menuContainer2"><NAV><UL><LI><A href="https://2015.igem.org/Team:Sydney_Australia">Home</A></LI><LI><A href="https://2015.igem.org/Team:Sydney_Australia/achievements">Achievements</A></LI><LI><A href="#">Project</A></LI><LI><A href="https://2015.igem.org/Team:Sydney_Australia/project_overview">Overview</A></LI><LI><A href="https://2015.igem.org/Team:Sydney_Australia/Description">Description</A></LI><LI><A href="https://2015.igem.org/Team:Sydney_Australia/Experiments">Protocols</A></LI><LI><A href="https://2015.igem.org/Team:Sydney_Australia/goldengate">Golden Gate</A></LI><LI><A href="https://2015.igem.org/Team:Sydney_Australia/Results">Results</A></LI><LI><A href="https://2015.igem.org/Team:Sydney_Australia/Notebook">Notebook</A></LI></UL><LI><A href="https://2015.igem.org/Team:Sydney_Australia/Parts">Parts</A></LI><LI><A href="#">Modelling</A></LI><LI><A href="https://2015.igem.org/Team:Sydney_Australia/Modeling">Overview</A></LI><LI><A href="https://2015.igem.org/Team:Sydney_Australia/TransOpt">TransOpt</A></LI><LI><A href="https://2015.igem.org/Team:Sydney_Australia/Modeling/visualisation">Visualisation</A></LI><LI><A href="https://2015.igem.org/Team:Sydney_Australia/Modeling/ElectroStop">ElectroStop</A></LI><LI><A href="https://2015.igem.org/Team:Sydney_Australia/Modeling/Supplementary">Supplementary Material</A></LI><LI><A href="#">Human Practices</A></LI><LI><A href="https://2015.igem.org/Team:Sydney_Australia/Practices">Overview</A></LI><LI><A href="https://2015.igem.org/Team:Sydney_Australia/EthicsMotivation">Motivation &amp; Analysis</A></LI><LI><A href="https://2015.igem.org/Team:Sydney_Australia/Outreach">Outreach</A></LI><LI><A href="https://2015.igem.org/Team:Sydney_Australia/Media">Media</A></LI><LI><A href="https://2015.igem.org/Team:Sydney_Australia/Collaborations">Collaboration</A></LI><LI><A href="https://2015.igem.org/Team:Sydney_Australia/BusinessModel">Business Model</A></LI><LI><A href="https://2015.igem.org/Team:Sydney_Australia/Safety">Safety</A></LI><LI><A href="https://2015.igem.org/Team:Sydney_Australia/Attributions">Attributions</A></LI><LI><A href="https://2015.igem.org/Team:Sydney_Australia/Team">Team</A></LI></NAV></DIV></DIV><DIV id="mainContainer"><DIV id="contentContainer"><DIV style="float: right"><DIV id="toc" class="toc"><DIV id="toctitle"><H2>Contents</H2></DIV><UL><LI class="toclevel-1"><A href="#Visualization_Tool_for_Translation_Kinetics"><SPAN class="tocnumber">1</SPAN><SPAN class="toctext">Visualization Tool for Translation Kinetics</SPAN></A></LI><LI class="toclevel-1"><A href="#Scientific_Background"><SPAN class="tocnumber">2</SPAN><SPAN class="toctext">Scientific Background</SPAN></A></LI><LI class="toclevel-1"><A href="#Model"><SPAN class="tocnumber">3</SPAN><SPAN class="toctext">Model</SPAN></A></LI><LI class="toclevel-1"><A href="#Results_.26_Discussions"><SPAN class="tocnumber">4</SPAN><SPAN class="toctext">Results &amp; Discussions</SPAN></A></LI><LI class="toclevel-1"><A href="#Discussion"><SPAN class="tocnumber">5</SPAN><SPAN class="toctext">Discussion</SPAN></A></LI><LI class="toclevel-1 tocsection-1"><A href="#Conclusion"><SPAN class="tocnumber">6</SPAN><SPAN class="toctext">Conclusion</SPAN></A></LI><LI class="toclevel-1 tocsection-2"><A href="#References"><SPAN class="tocnumber">7</SPAN><SPAN class="toctext">References</SPAN></A></LI></UL></DIV></DIV><H1><SPAN class="mw-headline" id="Visualization_Tool_for_Translation_Kinetics">Visualization Tool for Translation Kinetics</SPAN></H1><P>Whilst <A rel="nofollow" class="external text" href="https://2015.igem.org/Team:Sydney_Australia/TransOpt">TransOpt</A> generates a harmonized sequence, this visualization tool helps demonstrate the influence of a number of factors (including initiation rate and codon usage) on the translation process. This is done for the purpose of monitoring ribosomal traffic,  which has been shown to be a major determinant of protein yield.<SUP><A rel="nofollow" class="external text" href="https://2015.igem.org/Team:Sydney_Australia/Modeling/visualisation#References">1</A></SUP> By visualizing this process the user is able to qualitatively inspect sequences (such as those generated by <A rel="nofollow" class="external text" href="https://2015.igem.org/Team:Sydney_Australia/TransOpt">TransOpt</A>), and infer which tRNA species serve as limiting factors in the translation process.  
</P><H1><SPAN class="mw-headline" id="Scientific_Background">Scientific Background</SPAN></H1><P>During translation, the ribosomal movement rate and resultant impact on protein folding is determined by the interplay of a number of factors, including properties of the mRNA being translated, as well as tRNA concentrations within the cell.<SUP><A rel="nofollow" class="external text" href="https://2015.igem.org/Team:Sydney_Australia/Modeling/visualisation#References">1</A></SUP> It is generally believed that the rate at which a given codon is translated depends on its matching tRNA species' concentration, which is in turn a function of that tRNA's production and charging rates (see <A rel="nofollow" class="external text" href="https://2015.igem.org/Team:Sydney_Australia/TransOpt">TransOpt</A>). This means slow-translating codons typically correspond to rarely produced tRNAs. However, it is also possible for over-use of a specific codon to result in a &quot;tRNA bottleneck&quot;,<SUP><A rel="nofollow" class="external text" href="https://2015.igem.org/Team:Sydney_Australia/Modeling/visualisation#References">2</A></SUP> (also referred to as <I>tRNA starvation</I>) whereby the concentration of its charged variant becomes depleted, thereby slowing translation. Finally, depending on the rate of translation initiation, at any time there are generally several ribosomes translating a single piece of mRNA, which can result in &quot;traffic jams&quot;, where a ribosome's forward movement is prevented by the presence of another in front of it.
</P><P>In order to simulate these effects, our tool is built upon the mathematical model proposed by Brackley et al<SUP><A rel="nofollow" class="external text" href="https://2015.igem.org/Team:Sydney_Australia/Modeling/visualisation#References">2</A></SUP>, which takes into account both the mechanisms for translation along an mRNA, and the corresponding depletion of surrounding charged tRNA species. In our work we extend the design of Brackley et al<SUP><A rel="nofollow" class="external text" href="https://2015.igem.org/Team:Sydney_Australia/Modeling/visualisation#References">2</A></SUP>, by incorporating the translation rate quantisations proposed by Spencer et al<SUP><A rel="nofollow" class="external text" href="https://2015.igem.org/Team:Sydney_Australia/Modeling/visualisation#References">3</A></SUP> (which are used extensively in <A rel="nofollow" class="external text" href="https://2015.igem.org/Team:Sydney_Australia/TransOpt">TransOpt</A>). 
</P><H1><SPAN class="mw-headline" id="Model">Model</SPAN></H1><P>By simulating translation as a stochastic process we can model the way in which ribosomes move along an mRNA, and how the rate of progress interplays with the pools of bare and aminoacylated/charged tRNAs. This process is summarised in <I>Figure 1</I>.
</P><DIV class="center"><DIV class="thumb tnone"><DIV class="thumbinner" style="width:652px;"><DIV class="thumbcaption"><I>Figure 1</I> - Schematic of the ribosomal translation process, including its interplay with the surrounding tRNA concentration trends. Ribosomes (red) traverse the mRNA (blue), with each step reducing the corresponding aminoacylated/charged tRNA (aa-tRNA) population by one. Figure adapted from Brackley et al<SUP><A rel="nofollow" class="external text" href="https://2015.igem.org/Team:Sydney_Australia/Modeling/visualisation#References">2</A></SUP></DIV></DIV></DIV></DIV><P>As ribosomes cover several codons at a given time, they are modelled as blocks of width w (here taken as w=9 <SUP><A rel="nofollow" class="external text" href="https://2015.igem.org/Team:Sydney_Australia/Modeling/visualisation#References">2</A></SUP>) which move along the mRNA. Ribosomes hop on-to the start of the mRNA with rate \(\alpha\), and off with rate \(\beta\), which is assumed constant for all ribosomes. The hop-off rate is assumed to be substantially larger than all other rates, as ribosome exit is generally considered not to be a limiting step in translation<SUP><A rel="nofollow" class="external text" href="https://2015.igem.org/Team:Sydney_Australia/Modeling/visualisation#References">4</A></SUP>. Ribosomes move from codon to codon with hopping rate \(k_{\mu }\), so long as the subsequent codon is free (i.e. no other ribosome covers it). If the subsequent codon is occupied, the ribosome waits until a space appears before advancing again. When a ribosome moves off of a codon the corresponding aa-tRNA pools are reduced by a factor reflecting the codon's possible tRNA binding species, which takes into account non Watson-Crick interactions <SUP><A rel="nofollow" class="external text" href="https://2015.igem.org/Team:Sydney_Australia/Modeling/visualisation#References">3</A></SUP>. As an example, if a ribosome moves over a GCG codon, the aa-tRNA pools are adjusted via:
</P><P>$$ T_{GCG}=T_{GCG}-\frac{1}{1+\frac{1}{3}} \quad and \quad T_{GCA}=T_{GCA}-\frac{\frac{1}{3}}{1+\frac{1}{3}}$$
</P><P>This is reflects the penalising factor (1/3) for non Watson-Crick (wobble) interactions via which some tRNA bind.
The codon hopping rate \(k_{\mu }\)is expressed by:
</P><P>$$k_{\mu }=r\sum_{\widetilde{\mu }}{a_{\widetilde{\mu }}}T_{\widetilde{\mu }}\left(t\right)\ $$ 
</P><P>
This sum over \(\widetilde{\mu }\) is made to account for the contribution of secondary tRNA species via non Watson-Crick tRNA binding (as discussed in <A rel="nofollow" class="external text" href="https://2015.igem.org/Team:Sydney_Australia/TransOpt">TransOpt</A>), where \(T_{\widetilde{\mu }}(t)\) is the number of aa-tRNA molecules of type \(\widetilde{\mu }\) at time t, and \(a_{\widetilde{\mu }}\) is a scaling factor from the Spencer et al<SUP><A rel="nofollow" class="external text" href="https://2015.igem.org/Team:Sydney_Australia/Modeling/visualisation#References">3</A></SUP> rate quantisation. As an example, the rate \(k_{GCG}\) for tRNA binding codon GCG is \(k_{GCG}=r(1\times T_{GCG}+\frac{1}{3}\times T_{GCA})\), where the \(a_{GCA}=\frac{1}{3}\) factor reflects the ability for the tRNA corresponding to GCA to bind via wobble interaction, but with penalising factor \(\frac{1}{3}\ \)<SUP><A rel="nofollow" class="external text" href="https://2015.igem.org/Team:Sydney_Australia/Modeling/visualisation#References">3</A></SUP>. The rate scaling factor r is a set constant as \(r=10\times 41/\overline{T}\) where:
</P><P>$$\overline{T}=\sum^n_{\mu =1}{{\overline{T}}_{\mu }}$$
</P><P>In which \(\overline{T}\) is the total number of tRNAs within the cell (of which there are n types), assumed to be constant, and \({\overline{T}}_{\mu }\) is the total number of tRNA molecules (aminoacylated and bare) of type \(\mu\), also constant. For our model we set a typical value \(\overline{T}=4.3\times {10}^4\) and assume that:
\(\frac{{\overline{T}}_{\mu }}{\overline{T}}=\frac{GCN_{\mu }}{\sum_{\mu '}{GCN_{\mu '}}}\ \)
In which \(GCN_{\mu }\) is the Gene Copy Number for tRNA \(\mu\), and \(\sum_{\mu '}{GCN_{\mu '}}\) is the sum over all tRNA genes, and hence indicates the total number of tRNA encoding genes within the organism's genome. This therefore utilises the same data used in TransOpt for rate quantisation.
</P><P>The recharging of tRNAs (from bare to aminoacylated) is quantised via the Michaelis-Menten equation:
$$R_{\mu }\left(t\right)=\frac{V_{\mu }({\overline{T}}_{\mu }-T_{\mu }\left(t\right))}{K_{m,\mu }+{\overline{T}}_{\mu }-T_{\mu }(t)\ }$$
Which implicitly assumes the availability of amino acid molecules is non-limiting,  \(V_{\mu }\) is the maximum charge rate of tRNA type \(\mu\), and \(K_{m,\mu }\) is the number of bare tRNAs of type \(\mu\) for which the rate Is half-maximum, and is assumed to be<SUP><A rel="nofollow" class="external text" href="https://2015.igem.org/Team:Sydney_Australia/Modeling/visualisation#References">2</A></SUP>:
$$K_{m,\mu }=K_m=\left(1.28\times {10}^{18}\ molecules\ m^{-3}\right)\times \left(2.9\times {10}^{-17}\ m^3\ cell^{-1}\right)=37.12\ molecules$$ 
\(V_{\mu }\) is given by:
$$V_{\mu }=E_{\mu }k_{cat,\mu }$$
In which \(E_{\mu }\) is the number of enzyme molecules present able to recharge tRNA type \(\mu\), defined as \(E_{\mu }={\overline{T}}_{\mu }\times 0.208\), in which \(0.208\ enzymes\ tRNA^{-1}\) is an experimentally determined constant<SUP><A rel="nofollow" class="external text" href="https://2015.igem.org/Team:Sydney_Australia/Modeling/visualisation#References">5</A></SUP>, and \(k_{cat,\mu }\) is the turnover number, the rate at which one enzyme molecule can recharge one tRNA, set as \(k_{cat,\mu }=k_{cat}=6.29\ s^{-1}\).
</P><H1><SPAN class="mw-headline" id="Results_.26_Discussions">Results &amp; Discussions</SPAN></H1><P>We first run the visualisation tool with a single mRNA molecule to demonstrate the impact of &quot;traffic jams&quot; on the translation process.  This shows that the relative magnitudes of the initiation and hopping rates determine ribosomal behaviour to a great degree. With \(\alpha=0.1\ sec^{-1},\ \beta=100\ sec^{-1}\) the initiation rate (\(\alpha\)) becomes a limiting factor in translation, and we see a sparsely populated mRNA , with virtually no interaction between neighboring ribosomes (<I>Figure 2</I>). In the re-design of our mono-oxygenase enzyme (see <A rel="nofollow" class="external text" href="https://2015.igem.org/Team:Sydney_Australia/Description">Project Description</A>) we included weak ribosome binding sites for the purpose of avoiding ribosomal congestion: A weak binding site results in infrequent initiation, corresponding to smaller \(\alpha\), and less congestion.
</P><P><CENTER><VIDEO width="700" height="526" controls=""><SOURCE src="https://static.igem.org/mediawiki/2015/e/eb/1mRNAA0p1.mp4" type="video/mp4">

  Your browser does not support the video tag.
</SOURCE></VIDEO></CENTER></P><DIV class="center" style="width: auto; margin-left: auto; margin-right: auto;"><I>Figure 2</I> - Translation of a single mRNA with \(\alpha=0.1\ sec^{-1}\). Initiation is the limiting step and ribosomal traffic is minimal.</DIV><P>
However, setting \(\alpha=10\ sec^{-1}\) allows regular ribosome initiation, and forms a near-complete coverage of the mRNA molecule (<I>Figure 3</I>). This is what we hoped to avoid via inclusion of a weak ribosome binding site.
</P><P><CENTER><VIDEO width="700" height="526" controls=""><SOURCE src="https://static.igem.org/mediawiki/2015/d/da/1mRNAA10.mp4" type="video/mp4">

  Your browser does not support the video tag.
</SOURCE></VIDEO></CENTER></P><DIV class="center" style="width: auto; margin-left: auto; margin-right: auto;"><I>Figure 3</I> - Translation of a single mRNA with \(\alpha=10\ sec^{-1}\). Initiation is non-limiting and ribosomal traffic jams occur.</DIV><P>
Simultaneously simulation of 100 mRNA molecules allows this to be seen on a greater scale, and highlights the impact of the cellular tRNA concentrations. In In <I>Figure 4</I> 100 mRNA, each corresponding to a single row, are simulated in parallel with parameters \(\alpha=0.1\ sec^{-1}\ \beta=100\ sec^{-1}\), and in <I>Figure 5</I> 100 mRNA are simulated in parallel with parameters \(\alpha=10\ sec^{-1}\ \beta=100\ sec^{-1}\).
</P><P><CENTER><VIDEO width="700" height="526" controls=""><SOURCE src="https://static.igem.org/mediawiki/2015/1/15/100mRNAA0p10.mp4" type="video/mp4">

  Your browser does not support the video tag.
</SOURCE></VIDEO></CENTER></P><DIV class="center" style="width: auto; margin-left: auto; margin-right: auto;"><I>Figure 4</I> - Translation of a 100 mRNA molecules, one per row, with \(\alpha=0.1\ sec^{-1}\). Initiation is limiting, and no ribosomal traffic jams occur. We observe no locations in which ribosomes frequently stall, since the small number of translating ribosomes are unable to substantially impact tRNA concentration within the cell. Yellow rectangles are ribosomes, whilst the background is color-coordinated such that different shades of blue correspond to different codons in the mRNA.</DIV><P><CENTER><VIDEO width="700" height="526" controls=""><SOURCE src="https://static.igem.org/mediawiki/2015/6/63/100mRNAA10.mp4" type="video/mp4">

  Your browser does not support the video tag.
</SOURCE></VIDEO></CENTER></P><DIV class="center" style="width: auto; margin-left: auto; margin-right: auto;"><I>Figure 5</I> - Translation of a 100 mRNA molecules, one per row, with \(\alpha=10\ sec^{-1}\). Initiation is non-limiting and ribosomal traffic jams occur frequently. We can begin to see regular stalling on some slow-translating codons, however, this effect is not particularly strong since the maximal decrease in any of the tRNA concentrations is only on the order of ~20% (<I>Figure 6</I>).  Yellow rectangles are ribosomes, whilst the blue background is color-coordinated such that different shades of blue correspond to different codons in the mRNA.</DIV><P>
The large number of ribosomes involved in simultaneous translation has a significant impact on the simulated cell's tRNA content, which for some heavily-used species can be seen to decrease noticeably during the simulation (<I>Figure 6</I>). In this case, the tRNA that binds with GAC (and via wobble interaction with GAT) is used heavily, and a substantial decrease in its concentration results (leaving ~80% charged and ~20% uncharged). This was not observed for small initiation rates (<I>Figure 4</I>); in that case none of the tRNA species' charged proportion dropped below 98%.
</P><DIV class="center"><DIV class="thumb tnone"><DIV class="thumbinner" style="width:652px;"><DIV class="thumbcaption"><I>Figure 6</I>: Aminoacylated tRNA trends during simulation with 100 mRNA and \(\alpha=10\ sec^{-1}\). One species, which occurs frequently in the mRNA sequence, is noticeably depleted.</DIV></DIV></DIV></DIV><P>The limiting behaviour of aa-tRNA starvation can be demonstrated by decreasing the total number of tRNA, as in <I>Figure 7</I> where \(\overline{T}=4.3\times {10}^3\) (reduced by a factor of 10). Alternatively, the same result can be achieved by multiplying the number of mRNA species being translated by 10, though this results in a much larger (and hence slower) simulation.  Now, the aminoacylated concentration of one of the tRNA species (that which binds GAC) is depleted completely, and can stall translation in certain areas (<I>Figure 8</I>). Other tRNA species decrease initially, but once the GAC aa-tRNA concentration has dropped sufficiently to form a bottleneck in translation, other species are able to recharge to keep up with demand. 
</P><DIV class="center"><DIV class="thumb tnone"><DIV class="thumbinner" style="width:652px;"><DIV class="thumbcaption"><I>Figure 7</I>: Aminoacylated tRNA trends during simulation with 100 mRNA, \(\alpha=10\ sec^{-1}\), and \(\overline{T}=4.3\times {10}^3\). One species, which occurs frequently in the mRNA sequence, is noticeably depleted thereby causing ribosomal stalling.</DIV></DIV></DIV></DIV><P> 
This bottleneck results in the formation of ribosomal traffic jams whenever a codon is encountered that requires one the depleted tRNA species. The is clearly visible in <I>Figure 8</I>, where ribosomes stall temporarily in certain columns. In physical systems, stalling induced by aminoacylated tRNA starvation may have a major impact on protein folding. By including weak ribosome binding sites in our mono-oxygenase enzyme we hoped to decrease the number of ribosomes simultaneously translating each mRNA, thereby preventing these tRNA starvation conditions from occurring. 
</P><P><CENTER><VIDEO width="700" height="526" controls=""><SOURCE src="https://static.igem.org/mediawiki/2015/3/36/100mRNAtentimeslessTBAR.mp4" type="video/mp4">

  Your browser does not support the video tag.
</SOURCE></VIDEO></CENTER></P><DIV class="center" style="width: auto; margin-left: auto; margin-right: auto;"><I>Figure 8</I> - Translation of 100 mRNA with \(\alpha = 10\), and the total number of tRNA set to \(\overline{T}=4.3\times {10}^3\). Ribosomal stalling due to aa-tRNA starvation is evident at certain locations in which the ribosome must wait for a depleted tRNA species in order to proceed. Yellow rectangles are ribosomes, whilst the blue background is color-coordinated such that different shades of blue correspond to different codons in the mRNA. </DIV><H1><SPAN class="mw-headline" id="Discussion">Discussion</SPAN></H1><P>In developing this visualization tool we hoped to illustrate the way in which different features of a mRNA molecule can impact the translation process. Here we have placed particular focus on the translation initiation rate, \(\alpha\), which corresponds to the strength of an mRNA's ribosome binding site. In previous work in the Coleman laboratory it was found that our target mono-oxygenase enzyme showed only very weak expression in Pseudomonas Putida (see <A rel="nofollow" class="external text" href="https://2015.igem.org/Team:Sydney_Australia/Description">Project Description</A>), which we hypothesized might have in part resulted from a poorly designed ribosome binding site. With our visualisation tool we have demonstrated how this may have a substantial impact on translation: First, we find that having a fast initiation rate can result in ribosomal congestion throughout the mRNA, potentially disrupting the process. Second, fast initiation coupled with limited tRNA supplies can result in the complete depletion of certain tRNA species, stalling translation entirely. These issues provided the motivation for selecting a weak ribosome binding site for our enzyme: It was hoped that with slowed initiation a more reliable translation process would result.
</P><P>However, there are obvious down-sides to this approach should excessively fast ribosome initiation not be causing the problems hypothesized above. For example, if issues are occurring further down the mRNA sequence, with our weakened ribosome binding site we are reducing overall production of the protein, leading to a further decrease in observed expression. In order to assess which of these limiting cases is most likely to be occurring, future work could involve testing of a range of mono-oxygenase enzymes, differing only in the strength of their ribosome binding site, in order to assess how this impacts protein expression.
</P><P>tRNA starvation has been identified as playing, in some cases, a major role in translation regulation. In <I>Figure 7</I> this is apparent, where we observe the complete depletion of a single tRNA species, which then results in substantial stalling during translation (<I>Figure 8</I>). Due to the slowed rate of translation this will clearly cause a decrease in overall protein production, and may even result in premature ribosomal termination.<SUP><A rel="nofollow" class="external text" href="https://2015.igem.org/Team:Sydney_Australia/Modeling/visualisation#References">6</A></SUP> In our <A rel="nofollow" class="external text" href="https://2015.igem.org/Team:Sydney_Australia/TransOpt#Experimental_Validation">experimental results for TransOpt</A> the TransOpt optimised sequence failed to achieve successful expression, however, a number of codons were identified which were utilised by this sequence, but in none of the other variants tested. Thus, if the tRNA species responsible for translating one of these codons was produced in only very small quantities (or was depleted for some other reason) in <I>e. coli</I>, it might be possible for a situation like that in <I>Figure 7</I> to occur, where depletion of this single species is able to greatly diminish protein expression. This could then cause the minimal fluorescence observed for the TransOpt sequence variant.
</P><P>Whilst this translation simulation tool provides interesting insight into the translation process, the parameters used in calculations are of substantial uncertainty, and thus few quantitative conclusions can be made. At present parameters such as the total number of tRNA per cell, volume of each cell, and various rates and concentrations, are taken as those set by Brackley et al <SUP><A rel="nofollow" class="external text" href="https://2015.igem.org/Team:Sydney_Australia/Modeling/visualisation#References">2</A></SUP>, which were selected to best represent the conditions in Saccharomyces cerevisiae. However, we have not attempted to draw any quantitative results from this visualisation attempt, hence its name. Rather, we simply aimed to illustrate potential limiting cases that may occur during translation, how they come about, and what impact they may have on eventual expression.
</P><H1><SPAN class="mw-headline" id="Conclusion">Conclusion</SPAN></H1><P>In this work we have extended the model of Brackley et al<SUP><A rel="nofollow" class="external text" href="https://2015.igem.org/Team:Sydney_Australia/Modeling/visualisation#References">2</A></SUP> to include rate quantisations as proposed by Spencer et al<SUP><A rel="nofollow" class="external text" href="https://2015.igem.org/Team:Sydney_Australia/Modeling/visualisation#References">3</A></SUP>, which include factors such as non Watson-Crick tRNA interactions. This results in a model which we believe more accurately describes the tRNA concentration dynamics in bacteria. The resultant tool is able to visualise how the interplay of factors including ribosome binding rate and tRNA concentrations determine translation rates, and illustrates situations in which these factors can combine to potentially cause problems such as stalling and ribosomal traffic jams. Though we refrain from deriving any quantitative results from this model, it remains a useful demonstrative tool, and may be improved in the future to use more physically-realistic parameters, and to account for additional mechanisms within the cell.
</P><H1><SPAN class="mw-headline" id="References">References</SPAN></H1><P><SUP>1 </SUP>  Huang T et al. 2011, “Analysis and Prediction of Translation Rate Based on Sequence and Functional Features of the mRNA”, PLoS One, Vol 6.
</P><P><SUP>2 </SUP>  Brackley C et al. 2011, “The Dynamics of Supply and Demand in mRNA Translation”, PLoS Computational Biology, Vol 7.
</P><P><SUP>3 </SUP>  Spencer P et al., 2012, “Silent Substitutions Predictably Alter Translation Elongation Rates and Protein Folding Efficiencies”, Vol 422, pp. 328-335.
</P><P><SUP>4 </SUP>  Arava Y et al., 2003, “Genome-wide analysis of mrna translation profiles in saccharomyces cerevisiae”, Proceedings of the National Academy of Sciences, Vol 100, pp. 3889-3894.
</P><P><SUP>5 </SUP>  von der Haar T, 2008, “A quantitative estimation of the global translational activity in logarithmically growing yeast cells”, BMC Sys Biol, Vol 2.
</P><P><SUP>6 </SUP>  Keiler K et al. 2015, “Mechanisms of ribosome rescue in bacteria”, Nature Reviews Microbiology, Vol 13.
</P></DIV></DIV></DIV></DIV></DIV></DIV></BODY></HTML>