<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_BroadRun-Baltimore_ProjectDesign skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><DIV id="top_title"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:BroadRun-Baltimore/ProjectDesign</SPAN></H1></DIV><DIV id="HQ_page"><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><P>

.MathJax nobr&gt;span.math&gt;span{border-left-width:0 !important};

</P><DIV id="top_menu" class="backgroundbrhs"><UL id="menu" class="menubrhs"><LI><A href="/Team:BroadRun-Baltimore/Project">Project</A></LI><LI><A href="/Team:BroadRun-Baltimore/Project">Overview</A></LI><LI><A href="/Team:BroadRun-Baltimore/Problem">Problem</A></LI><LI><A href="/Team:BroadRun-Baltimore/Solution">Solution</A></LI><LI><A href="/Team:BroadRun-Baltimore/ProjectDesign">Project Design</A></LI><LI><A href="/Team:BroadRun-Baltimore/Methods">Methods</A></LI><LI><A href="/Team:BroadRun-Baltimore/Results">Testing</A></LI><LI><A href="/Team:BroadRun-Baltimore/Conclusions">Conclusions</A></LI></UL><LI><A href="/Team:BroadRun-Baltimore/Parts">Parts</A></LI><LI><A href="/Team:BroadRun-Baltimore/Notebook">Notebook</A></LI><LI><A href="/Team:BroadRun-Baltimore/Safety">Safety</A></LI><LI><A href="/Team:BroadRun-Baltimore/Model">Model</A></LI><LI><A href="/Team:BroadRun-Baltimore/Practices">Practices</A></LI><LI><A href="/Team:BroadRun-Baltimore/Industry">Industry Comm.</A></LI><LI><A href="/Team:BroadRun-Baltimore/PublicOutreach">Public Outreach</A></LI><LI><A href="/Team:BroadRun-Baltimore/Buildingwbio">Building W/ Biology</A></LI><LI><A href="/Team:BroadRun-Baltimore/Sustainability">Sustainability</A></LI><LI><A href="/Team:BroadRun-Baltimore/Team">Team</A></LI><LI><A href="/Team:BroadRun-Baltimore/Collaborations">Collaboration</A></LI><LI><A href="/Team:BroadRun-Baltimore/Attributions">Attributions</A></LI><LI><A href="/Team:BroadRun-Baltimore/Sponsors">Sponsors</A></LI><LI><A href="/Team:BroadRun-Baltimore/Team">Meet the Team</A></LI><LI><A href="/Team:BroadRun-Baltimore/Judging">Judging</A></LI></DIV><H1> Project Design</H1><H2> Pre-Design considerations</H2><P1>Before designing our project, we had to consider potential risks of releasing engineered self replicating organisms into a broader space, and how to mitigate those risks. Furthermore, we had to consider how different stakeholders might play a role in biosafety practices. We consulted research publications on the biosafety of modified organisms [1] and how to approach risk and public safety using materials published by the <A href="http://www.synbioproject.org">Woodrow Wilson Synthetic Biology Project</A> [2]. </P1><P1>Our thought process for public safety and risk assessment considerations for synthetic biology application in ceiling tile manufacturing is summarized below. 
</P1><P1><U>References </U></P1><P>1.Moe-Behrens, G. H. G., Davis, R., &amp; Haynes, K. A. (2013). Preparing synthetic biology for the world. Frontiers in Microbiology, 4, 5. http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3554958/</P><P>2.Dana GV, Kuiken T, Rejeski D, Snow AA Synthetic biology: Four steps to avoid a synthetic-biology disaster.Nature. 2012 Feb 29; 483(7387):29.</P><H2> Design of Constructs </H2><P1>In continuing our research from last year on developing a solution for the problem faced by ceiling tile we decided to focus on developing a more efficient amylase enzyme and expression system. For this, we designed six constructs; each one a different combination of 2 different promoters and 3 amylase enzymes. Two of the amylase enzymes were beta amylases, and one alpha amylase. </P1><P1>Amylase enzymes that hydrolyze bonds linking large polysaccharides, such as starch and glycogen. Both beta and amylases are able to degrade starch to produce mono and disaccharides as a by-product. Beta amylases typically act on the ends of starch molecules, while alpha amylases typically act on the center of starch molecules. Amylase enzymes from both fungi and bacteria were used, to determine if there was a difference in the efficacy of starch degradation between amylases produced by the two types of organisms. </P1><P1>The three amylases enzymes used were: alpha amylase from <I>Bacillus cereus</I>, beta amylase from <I>Saprolegnia ferax</I>, and alpha amylase from <I>Penicilium</I>. The two promoters used were a TEF1 promoter and a CYC promoter, both strong constitutive promoters. Constructs with the TEF1 promoter were synthesized without a promoter and inserted into a plasmid with the TEF1 promoter. To be able to test the effects of different promoters and enzymes on efficiency of starch degradation, all other parts of the construct (kozak sequence, secretion tag, and terminator) were kept constant across the constructs. Following assembly of genetic parts, constructs were codon optimized for S.cerevisiae, and restriction sites were removed. </P1><P1>Our final construct designs are below. 
<H4>Construct 1</H4><P2>Promoterless</P2><P2>Kozak sequence (BBa_K165002)</P2><P2>Mating Factor Secretion Tag (BBa_K792002)</P2><P2>Alpha amylase coding sequence from the fungus Penicillium </P2><P2>ADH1 Terminator (Part BBa_K392003)</P2><H4>Construct 2</H4><P2>Promoterless</P2><P2>Kozak sequence (Part BBa_K165002)</P2><P2>Mating Factor Secretion Tag (BBa_K792002)</P2><P2>Beta amylase coding sequence from Bacillus cereus</P2><P2>ADH1 Terminator (Part BBa_K392003)</P2><H4>Construct 3</H4><P2>pCyc medium promoter (BBa_I766555)</P2><P2>Kozak sequence (Part BBa_K165002)</P2><P2>Mating Factor Secretion Tag (BBa_K792002)</P2><P2>Alpha amylase coding sequence from Saprolegnia ferax</P2><P2>ADH1 Terminator (Part BBa_K392003)</P2><H4>Construct 4</H4><P2>pCYC Medium Promoter</P2><P2>Kozak sequence (BBa_K165002)</P2><P2>Mating Factor Secretion Tag (BBa_K792002)</P2><P2>Alpha amylase coding sequence from the fungus Penicillium </P2><P2>ADH1 Terminator (Part BBa_K392003)</P2><H4>Construct 5</H4><P2>Promoterless (BBa_I766555)</P2><P2>Kozak sequence (Part BBa_K165002)</P2><P2>Mating Factor Secretion Tag (BBa_K792002)</P2><P2>Alpha amylase coding sequence from Saprolegnia ferax</P2><P2>ADH1 Terminator (Part BBa_K392003)</P2><H4>Construct 6</H4><P2>pCYC Medium Promoter (BBa_I766555)</P2><P2>Kozak sequence (Part BBa_K165002)</P2><P2>Mating Factor Secretion Tag (BBa_K792002)</P2><P2>Beta amylase coding sequence from Bacillus cereus</P2><P2>ADH1 Terminator (Part BBa_K392003)</P2><H2> Design for a Kill Switch</H2><P1>Introduction of genetically engineered organisms outside of controlled laboratory settings carries with it many ethical considerations and safety risks. While our solution is fairly harmless––amylase enzymes pose no risk to the environment or human health and are an enzyme already naturally produced by yeast––a biocontainment mechanism to limit the the engineered cells to the ceiling tile plant would further reduce safety risks. </P1><P1>After extensive research into kill switches, we narrowed our ideas down to these four that would be applicable to our application. We reached out to Dr.Loren Looger, a lab head at Janelia Research Campus of Howard Hughes Medical Institute, and also the founder of a biotech company that designs kill switches for genetically modified organisms. He is currently working with large agrochemical companies to design gene switch mechanisms that respond to agrochemicals and other environmental conditions, for use in agriculture. </P1><P1>With his expertise in molecular engineering and kill switches, in our first meeting we laid out our proposed ideas. From there, we were able to decide upon pursuing a 2 component kill switch; a death gene, and an inhibitor of that gene. Our kill switch would activate death when there is no starch present, thus ensuring that if the cells escaped the ceiling tile plant, they would not survive.  </P1><H4>Components of Kill Switch</H4><P1>After months of extensive literature research, we were able to identify the two proteins needed for our kill switch; Bir1p and Yca1p. 
<P1>	- Bir1p is an inhibitor-of-apoptosis protein (IAP). </P1><P1>	- Nma111p is a yeast serine protease involved in inducing apoptosis. </P1><P1>Proteases are a type of enzyme that specializes in breaking down proteins and peptides. Nma11p is a homolog of the well studied Omi/HtrA2 gene in humans that triggers cell death and has been found to play a significant role in programmed cell suicide in yeast. Activating Nma111p causes irreversible proteolysis (breakdown of proteins) and leads to cell death. </P1><P1>Inhibitor-of-apoptosis proteins (IAPs) are key regulators of caspases, and thus apoptosis. IAPs were originally identified in baculoviruses, as proteins that protected the infected host cell from virus induced death. It has been shown that the Bir1p inhibitor of apoptosis protein binds to and cleaves the serine protease, Nma11p. Thus, Bir1p acts directly upon Nma111p, making it an effective inhibitor of apoptosis in yeast. </P1><P1>The third main component of our kill switch is the regulatory mechanism. Our kill switch needs to be responsive to starch. Thus, we chose the Starch Utilization System (SUS), which is induced by the presence of starch. The SUS operon is a close homolog of the well characterized lac operon. </P1><P1>The SUS operon will be used to control expression of the Birp1 inhibitor of apoptosis protein. When the SUS repressor protein (SUSr) binds to maltose (2 fused glucose molecules) and longer polysaccharides, such as starch, it falls off the SUS operator, allowing expression of the Bir1p protein. In the absence of starch, the repressor binds back onto the operator. </P1><P1>In the presence of starch Bir1p protein will be expressed, inhibit Nma11p and the cell lives. When starch is no longer present and there is only glucose in the environment, the SUS repressor binds back to the DNA; Birp1 expression is blocked, Nma11p is expressed and the cell undergoes apoptosis. </P1><H4>Design</H4><P1>Sequences for the SUSr repressor gene and SUS operator and promoter, were taken from the NCBI Genbank database. The Nma111p gene will be under control of a weak constitutive promoter taken from the iGEM Parts registry, the minimal cyc promoter (Part: BBa_I766557), for example. </P1><P1>With guidance from Dr.Looger, we were able to begin designing the primers and DNA templates for homologous recombination, based upon the following <A href="http://research.stowers.org/jaspersenlab/docs/markerswitching.pdf">protocol for homologous recombination</A>:  </P1><P1>However, soon after starting these designs in early August, with plans to synthesize them and test our kill switch design, we found ourselves without a microbiology lab to work in. Our schools did not have the proper resources to handle complex techniques such as homologous recombination, nor the funding for supplies. We were unable to get approved to work in a laboratory at local universities and research institutions either. While we were not able to carry out our design for the kill switch, we hope that our design may be useful for other iGEM teams looking into kill switches for yeast, an area for which there is not much research. 
</P1><H4>References</H4><P1>1.Madeo, F., Herker, E., Maldener, C., Wissing, S., Lachelt, S., Herlan, M., Fehr, M., Lauber, K., Sigrist, S. J., Wesselborg, S., et al. (2002) A caspase-related protease regulates apoptosis in yeast. Mol. Cell 9, 1– 120.</P1><P1>2.Tammy M. Joska, Mashruwala, A., Boyd, J., Belden, W.  (2014) A universal cloning method based on yeast homologous recombination that is simple, efficient, and versatile” J. Microbiol. Methods 100 C, 46–51.</P1><P1>3.Walter D, Wissing S, Madeo F, et. al. (2006) The inhibitor-of-apoptosis protein Bir1p protects against apoptosis in S. cerevisiae and is a substrate for the yeast homologue of Omi/HtrA2. J Cell Sci;119:1843-51.</P1><P1>4.Owsianowski E, Walter D, Fahrenkrog B (2008) Negative regulation of apoptosis in yeast. Biochim Biophys Acta1783: 1303–1310. </P1></P1></P1></DIV></DIV></DIV></DIV></DIV></BODY></HTML>