<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_BroadRun-Baltimore_Methods skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><DIV id="top_title"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:BroadRun-Baltimore/Methods</SPAN></H1></DIV><DIV id="HQ_page"><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><P>

.MathJax nobr&gt;span.math&gt;span{border-left-width:0 !important};

</P><DIV id="top_menu" class="backgroundbrhs"><UL id="menu" class="menubrhs"><LI><A href="/Team:BroadRun-Baltimore/Project">Project</A></LI><LI><A href="/Team:BroadRun-Baltimore/Project">Overview</A></LI><LI><A href="/Team:BroadRun-Baltimore/Problem">Problem</A></LI><LI><A href="/Team:BroadRun-Baltimore/Solution">Solution</A></LI><LI><A href="/Team:BroadRun-Baltimore/ProjectDesign">Project Design</A></LI><LI><A href="/Team:BroadRun-Baltimore/Methods">Methods</A></LI><LI><A href="/Team:BroadRun-Baltimore/Results">Testing</A></LI><LI><A href="/Team:BroadRun-Baltimore/Conclusions">Conclusions</A></LI></UL><LI><A href="/Team:BroadRun-Baltimore/Parts">Parts</A></LI><LI><A href="/Team:BroadRun-Baltimore/Notebook">Notebook</A></LI><LI><A href="/Team:BroadRun-Baltimore/Safety">Safety</A></LI><LI><A href="/Team:BroadRun-Baltimore/Model">Model</A></LI><LI><A href="/Team:BroadRun-Baltimore/Practices">Practices</A></LI><LI><A href="/Team:BroadRun-Baltimore/Industry">Industry Comm.</A></LI><LI><A href="/Team:BroadRun-Baltimore/PublicOutreach">Public Outreach</A></LI><LI><A href="/Team:BroadRun-Baltimore/Buildingwbio">Building W/ Biology</A></LI><LI><A href="/Team:BroadRun-Baltimore/Sustainability">Sustainability</A></LI><LI><A href="/Team:BroadRun-Baltimore/Team">Team</A></LI><LI><A href="/Team:BroadRun-Baltimore/Collaborations">Collaboration</A></LI><LI><A href="/Team:BroadRun-Baltimore/Attributions">Attributions</A></LI><LI><A href="/Team:BroadRun-Baltimore/Sponsors">Sponsors</A></LI><LI><A href="/Team:BroadRun-Baltimore/Team">Meet the Team</A></LI><LI><A href="/Team:BroadRun-Baltimore/Judging">Judging</A></LI></DIV><H1>Methods</H1><P1>Once we had developed a project design, designed our constructs, and sent them for synthesis we moved onto the first phase of our methods; inserting the synthesized constructs into a plasmid (for the specific protocols used, please visit the Notebook page). </P1><P1>Following the project design phase, the designed constructs were sent for synthesis. While 6 constructs were initially designed, only 4 were able to be synthesized due to size constraints and limitations in synthesis of such long DNA sequences. Due to delays in synthesis, 3 constructs were initially delivered (labeled as Constructs 1, 2, 3). As the 4th construct took over 1 month to be synthesized and delivered, it was not possible to clone this construct into yeast (we did not have access to a microbiology lab after early August and our schools did not have the resources for yeast cloning).  </P1><P1>Below is the first phase of methods: inserting the synthesized constructs into plasmids. </P1><P1>In the above graphic, the first part of the cloning process is summarized. </P1><H4>Amplification PCR </H4><P1>As synthesis often yields a relatively small quantity of DNA, the first step was amplification polymerase chain reaction, which uses primers (small pieces of DNA) to create more copies of the DNA. Synthesized constructs were reconstituted in TE buffer, then combined with PCR reagents (primers, DNTPs, Taq polymerase buffer, Taq polymerase enzyme), before being run in the thermocycler. </P1><H4>Restriction Digest </H4><P1>Following confirmation that the amplification PCR was successful, the PCR products were purified using a PCR purification kit. The amplified DNA constructs from the PCR, and two plasmids were digested using restriction enzymes. Restriction digest ‘cuts’ the DNA with enzymes, which results in ‘sticky ends’ on the constructs and corresponding plasmid. These corresponding ends enable the construct to be inserted into the plasmid. </P1><P1>The two yeast plasmids used were pRS426 and pAG36. The pSB1C3 plasmid was also cut with restriction enzymes, so that the constructs could be inserted for submission to the Parts Registry. For what enzymes were used for each construct/plasmid, and exact quantities of reagents, please see the Notebook. All three constructs and 3 plasmids were digested correctly, verified with a gel. </P1><H4>Restriction Digest Purification</H4><P1>After confirmation that the restriction digest was successful, the DNA samples were purified. The constructs were PCR purified and the plasmids were gel extracted. The DNA was purified to remove the unwanted pieces of DNA leftover from the restriction digest. These unwanted pieces of DNA could prevent the construct from being properly inserted into the plasmid. </P1><P1>The two methods of purification were used because of the different sizes of the pieces of DNA that were left after the restriction digest. The digested constructs had only very small pieces of DNA cut by the enzymes, these small pieces would be washed away during purification. The digested plasmids had large pieces of DNA several thousand base pairs long, thus these pieces had to first be separated by size with gel electrophoresis, then the desired piece of DNA was cut out and purified. This process prevents the plasmid from ligating back on itself, without the construct inside, during ligation. </P1><H4>Ligation </H4><P1>The purified constructs and plasmids were combined with a ligase enzyme and buffer. Constructs 1 and 2, which do not have a promoter, were ligated into the pAG36 yeast plasmid, which contains a TEF1 constitutive promoter. Construct 3, which contains a promoter, was ligated into the pRS426 yeast plasmid. All three constructs were ligated into the pSB1C3 plasmid.</P1><P1>In the above graphic, the second part of the cloning process is summarized. </P1><H4>E.coli Transformation </H4><P1>After ligation, the plasmids containing our construct were transformed into E.coli. The yeast plasmids had to be transformed to E.coli first, in order to verify that the plasmid did indeed contain the construct. </P1><P1>Competent E.coli cells were combined with the DNA samples, heat shocked, then plated onto agar plates with appropriate antibiotics. The antibiotics serve as a selectable marker for the E.coli cells that took up the plasmid. After letting the plates incubate for 24 hours, colonies were present on all plates. </P1><H4>Colony PCR </H4><P1>While the use of antibiotic resistance as a selectable marker eliminates E.coli cells that did not take up the plasmid, colony PCR must be used to determine if the plasmid inside of the cells actually contains the construct. Colony PCR uses primers that bind to the plasmid, just above and below where the construct is supposed to be. During the PCR reaction, many copies of the section of DNA between the two primers is made, and gel electrophoresis verifies how long the section of DNA is. If the plasmid does not contain the construct, then this piece of DNA will be very short. However if it does contain the plasmid, then the section of DNA will be about the same length as the construct. This method is used to ensure that only plasmids with the construct are transformed to yeast and submitted to the Registry. </P1><P1>The PCR reagents used were: forward and reverse primers, DNTPs, Taq polymerase enzyme, Taq buffer, and water. Colonies chosen with toothpicks were stirred into the mix of reagents, and then used to inoculate liquid cultures of LB broth. These liquid cultures were set aside in the incubator, to be used the next day for miniprepping the cells. To ensure that at least one colony from each plate was successful, 7-8 colonies per plate were selected for colony PCR. </P1><P1>After running the samples through the thermocycler, and gel was run on the samples. The gel confirmed that the constructs 1 and 2 had been properly inserted into the pAG36 vector, while construct 3 had not. All 3 constructs had been properly inserted into the pSB1C3 vector. </P1><H4>Miniprep </H4><P1>Once it had been confirmed which colonies were successful, those colonies were mini-prepped, to extract the plasmids inside of the cell. The DNA samples with the pSB1c3 plasmid were submitted to the Registry, those with the yeast plasmids were transformed to yeast. </P1><H4>Yeast Transformation </H4><P1>Following mini-prep, the next step was a yeast transformation. The yeast plasmids used the selectable marker gene URA3, the corresponding yeast strain was used. This means the URA3 yeast strain can’t produce uracil. The URA3 gene in the plasmid codes for uracil. Transformed cultures were plated onto uracil deficient media, so the yeast cells without the plasmid wouldn’t be able to produce uracil and wouldn’t form colonies.</P1><P1>Yeast cultures were prepped by culturing in YPD broth, and using a hemacytometer to ensure the cells were at the right density, before pelleting the cells and resuspending them in a competent cell mixture. </P1><P1><I> Image of yeast cultures being prepped for transformation.</I></P1><P1>Following the preparation, the cultures were transformed with the DNA samples using the lithium acetate and poly ethylene glycol method of transformation. (For more, see the Notebook.) The yeast transformation was successful for the three constructs (2 from 2016, one from 2015). </P1><P1><I>Image of yeast agar plates with many colonies, taken 3 days after transformation.</I></P1><H1> Testing</H1><H2>Phase 1: Testing in Known Starch Concentrations</H2><P1>The first step of testing was to test in a controlled environment, where the yeast were tested in solutions of known starch concentration. Iodine was added to the starch solutions, which causes a color change. The intensity of this color change, which correlates to the amount of starch, was quantified using a spectrophotometer. The spectrophotometer sends a specific wavelength of light through a sample and measures how much light was absorbed by the sample, which correlates to how much of a substance is in the solution. 4 different yeast strains were tested: three genetically modified yeast (three different constructs) and wildtype yeast as a control. A calibration curve was created, which enabled us to correlate the absorbance values from the spectrophotometer with the amount of starch. </P1><H4>Short Term Testing</H4><P1>In this test, the 4 yeast strains were combined with 0.5% starch, in a 1:1 ratio, for 6 hours. Measurements of the amount of starch were taken every hour. Before actually adding starch to the yeast cultures, all cultures were checked using the spectrophotometer, to ensure all had the same cell density. In addition to iodine to the samples for the starch measurement, HCl was added. This was to stop the cell’s metabolic reactions, which affect the iodine and starch reaction. </P1><P1>Starch measurements were made using the following protocol: </P1><UL style="list-style-type:none"><LI style="font-size:20px">1. Combine 10mL of liquid yeast culture with 10mL of .5% starch solution. 
</LI><LI style="font-size:20px">2. Add 0.5mL of this yeast/starch solution to a cuvette. 
</LI><LI style="font-size:20px">3. Add 0.3mL of 1M HCL to the cuvette. 
</LI><LI style="font-size:20px">4. Blank the spectrophotometer by setting the wavelength to 620nm, inserting this cuvette, and setting transmission to 100%. 
</LI><LI style="font-size:20px">5. Add 10µl of iodine and potassium iodide solution (1% iodine, 2% potassium iodide) and mix well. 
</LI><LI style="font-size:20px">6. Measure absorbance using the spectrophotometer. 




<H4>Long Term Testing</H4><P1>In this test, the 4 yeast strains were combined with 2% starch in a 1:8 ratio of yeast to starch. This lower ratio and higher concentration of starch was to ensure that the yeast would not break down all of the starch within just a few hours. This solution was aerated using an air pump and tubing, to keep oxygen levels in the sample high. </P1><P1><I>Setup of long term starch degradation testing. The 4th flask, with wildtype culture, is not pictured.</I></P1><P1>Measurements were made every 24 hours, for 72 hours, using the following protocol: </P1><UL style="list-style-type:none"><LI2 style="font-size:20px">1. Combine 10mL of liquid yeast culture with 10mL of .5% starch solution. 
<LI2 style="font-size:20px">2. Add 0.5mL of this yeast/starch solution to a cuvette. 
<LI2 style="font-size:20px">3. Add 3.5 mL of water and 0.3mL of 1M HCL to the cuvette. 
<LI2 style="font-size:20px">4. Blank the spectrophotometer by setting the wavelength to 620nm, inserting this cuvette, and setting transmission to 100%. 
<LI2 style="font-size:20px">5. Add 10µl of iodine and potassium iodide solution (1% iodine, 2% potassium iodide) and mix well. 
<LI2 style="font-size:20px">6. Measure absorbance using the spectrophotometer.  



<H2>Phase 2: Testing in Industrial Water Samples</H2>

After visiting the Armstrong ceiling tile manufacturing plant, 4 possible problematic areas were identified; aeration basin, secondary clarifier, primary clarifier, and thickener. 6 water samples were collected from these 4 locations; </LI2><P1>1) aeration basin </P1><P1>2) secondary clarifier </P1><P1>3) primary clarifier to the equalization basin</P1><P1>4) thickener to primary clarifier </P1><P1>5) primary clarifier to dry broke</P1><P1>6) thickener</P1><P1><I>Industrial water samples from a ceiling tile manufacturing plant.</I></P1><P1>The six water samples were first tested to determine starch levels. The aeration basin, secondary clarifier, and primary clarifier to equalization basin samples did not contain a detectable level of starch. The thickener to primary clarifier and thickener to dry broke samples contained a small amount of starch, approximately 0.32% and 0.45%, respectively. The primary clarifier to dry broke contained a much higher percentage of starch, approximately 1.3%. Thus, the prototype testing was run with the following three samples; primary clarifier to dry broke, thickener to primary clarifier, and thickener to dry broke. </P1><P1>Construct 3 was found to be most effective in the previous test, thus this genetically modified yeast strain was used in prototype testing. Yeast cultures were mixed into the industrial water sample in a 1:8 ratio, with a total volume of 250ml. To account for starch degradation from other organisms in the water sample, a control without yeast cells was run. The control contained a 1:8 ratio of YPD media (without yeast cells) to industrial water sample. The mechanical agitation was simulated by adding a magnetic stirrer bar and placing the beaker onto a stirrer plate, which kept cells, starch, and other organic compounds suspended in the water sample evenly mixed throughout, as in the ceiling tile plant.</P1><H4>Prototype</H4><P1>In order to simulate the physical conditions of the plant, a prototype was created. Dissolved oxygen levels were simulated by continuously aerating the samples, mimicking the large blowers used in the ceiling tile plant. In addition to aeration, wastewater and process water in the plant is mechanically agitated, usually with large rotating rakes in the clarifiers and thickeners. </P1><P1><I>Setup of prototype with stirrer plates and air pump. </I></P1><P1> Samples were measured at 6, 24, 48, and 72 hours. The same starch measurement protocol as in the long term starch degradation test was used. </P1><H2>Phase 3: Cell Growth Testing</H2><H4>Testing in YPD Media</H4><P1>Two cell growth experiments were run; cell growth in standard yeast media (YPD media) and cell growth in starch media. </P1><H2>Cell Growth in YPD Media </H2><P1>This experiment was run to determine parameters for creating a mathematical model. It was also run to determine if the genetic modifications to the yeast and the increased metabolic strain of constitutively producing amylase enzymes would have an effect on cell growth rates and glucose consumption. YPD media (standard liquid yeast media) was used as a glucose rich substrate. </P1><P1>Due to logistical constraints (shortage of stirrer plates), only 2 cultures were run, wildtype yeast as a control, and Construct 2 yeast (simply referred to as ‘Genetically modified yeast’). 30mL of YPD media was added to 50mL flasks then inoculated with the 2 yeast strains. To prevent settling of the cells, cell cultures were placed on stirrer plates with stirrer bars. Cell cultures were diluted 1:4 with water before measuring in the spectrophotometer for optical density. A standard blood glucose meter was used to measure glucose in the samples, after diluting the samples 1:15. </P1><H4>Testing in Starch Media</H4><P1>To gain insight on the ability of the genetically modified yeast to grow in substrate that contains starch, but no glucose, cell growth testing was completed in a starch media. The media contained starch, the carbon source for the yeast, and heat killed wildtype yeast, a nitrogen and amino acid source. 30mL of media was composed of: 29mL of 1% starch and 1mL of heat killed wildtype yeast. This media was added to a 50mL flask and then inoculated with Construct 2 yeast. Stir bars were added, and the flasks were left on stir plates for 192 hours (8 days). The yeast culture was measured at hours: 0, 24, 48, 54, 72, 78, 168, and 192. </P1><P1>The rationale behind this test was that the genetically modified yeast, which have the unique ability to degrade starch, would be able convert the surrounding starch into glucose, and thus be able to survive in the absence of a direct glucose source. </P1></LI2></LI2></LI2></LI2></LI2></UL></LI></UL></DIV></DIV></DIV></DIV></DIV></BODY></HTML>