<HTML xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en" dir="ltr">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki  ltr ns-0 ns-subject page-Team_Imperial_Functionalisation"><DIV id="globalWrapper"><DIV id="top-section"><DIV id="p-logo"><A href="/Main_Page" title="Main Page">&quot;
	    </A></DIV><DIV id="menubar" class="left-menu noprint"><UL><LI class="selected"><A href="/Team:Imperial/Functionalisation">Page               </A></LI><LI class="new"><A href="/wiki/index.php?title=Talk:Team:Imperial/Functionalisation&amp;action=edit&amp;redlink=1">Discussion               </A></LI><LI><A href="/wiki/index.php?title=Team:Imperial/Functionalisation&amp;action=edit">View source               </A></LI><LI><A href="/wiki/index.php?title=Team:Imperial/Functionalisation&amp;action=history">History               </A></LI><LI style="color:white;cursor:default">teams</LI></UL></DIV><DIV class="right-menu noprint" id="menubar"><UL><LI id="pt-login"><A href="/wiki/index.php?title=Special:UserLogin&amp;returnto=Team:Imperial/Functionalisation" title="You are encouraged to log in; however, it is not mandatory [o]" accesskey="o">Log in</A></LI></UL></DIV><DIV id="search-controls" class="noprint"><FORM action="/Special:Search" id="searchform"> </FORM></DIV></DIV><DIV id="content"><H1 class="firstHeading">Team:Imperial/Functionalisation</H1><DIV id="bodyContent"><H3 id="siteSub" class="noprint">From 2014.igem.org</H3><P><TITLE>Imperial iGEM 2014</TITLE></P><DIV class="wrapper"><DIV class="header"><DIV class="header-banner"><DIV class="left"><A style="text-decoration: none" href="https://2014.igem.org/Team:Imperial"><H1 id="Logo">Aqualose</H1></A></DIV></DIV><DIV id="main_nav" class="pure-menu pure-menu-open pure-menu-horizontal"><UL id="std-menu-items"><LI><A href="#">Project</A></LI><LI><A href="https://2014.igem.org/Team:Imperial/Project">Overview</A></LI><LI><A href="https://2014.igem.org/Team:Imperial/Project_Background">Background</A></LI><LI><A href="https://2014.igem.org/Team:Imperial/Gluconacetobacter"><EM>G. xylinus</EM></A></LI><LI><A href="https://2014.igem.org/Team:Imperial/EColi"><EM>E. coli</EM></A></LI><LI><A href="https://2014.igem.org/Team:Imperial/coculture">Co-culture</A></LI><LI><A href="https://2014.igem.org/Team:Imperial/Functionalisation">Functionalisation</A></LI><LI><A href="https://2014.igem.org/Team:Imperial/Mass_Production_and_Processing">Mass Production and Processing</A></LI><LI><A href="https://2014.igem.org/Team:Imperial/Mechanical_Testing">Mechanical Testing</A></LI></UL><LI><A href="#">Achievements</A></LI><LI><A href="https://2014.igem.org/Team:Imperial/Achievements">Overview</A></LI><LI><A href="https://2014.igem.org/Team:Imperial/Parts">Parts</A></LI><LI><A href="https://2014.igem.org/Team:Imperial/Results">Results</A></LI><LI><A href="https://2014.igem.org/Team:Imperial/Judging_Criteria">Judging Criteria</A></LI><LI><A href="https://2014.igem.org/Team:Imperial/InterLab_Study">Interlab Study</A></LI><LI><A href="#">Application</A></LI><LI><A href="https://2014.igem.org/Team:Imperial/Implementation">Implementation</A></LI><LI><A href="https://2014.igem.org/Team:Imperial/Water_Filtration">Water Filtration</A></LI><LI><A href="#">Policy &amp; Practices</A></LI><LI><A href="https://2014.igem.org/Team:Imperial/Policy_and_Practices">Overview</A></LI><LI><A href="https://2014.igem.org/Team:Imperial/Informing_Design">Informing Design</A></LI><LI><A href="https://2014.igem.org/Team:Imperial/Water_Report">Water Report</A></LI><LI><A href="https://2014.igem.org/Team:Imperial/The_i_in_iGEM">The i in iGEM</A></LI><LI><A href="https://2014.igem.org/Team:Imperial/Outreach">Outreach</A></LI><LI><A href="https://2014.igem.org/Team:Imperial/Collaborations">Collaborations</A></LI><LI><A href="#">Modelling</A></LI><LI><A href="https://2014.igem.org/Team:Imperial/Modelling">Overview</A></LI><LI><A href="https://2014.igem.org/Team:Imperial/Induction_Model">Induction Model</A></LI><LI><A href="https://2014.igem.org/Team:Imperial/CBD_Kinetics_Model">CBD Kinetics Model</A></LI><LI><A href="https://2014.igem.org/Team:Imperial/Nutrient_Diffusion_Simulations">Nutrient Diffusion Simulations</A></LI><LI><A href="https://2014.igem.org/Team:Imperial/BioHackspace_Model">BioHackspace Model</A></LI><LI><A href="#">Notebook</A></LI><LI><A href="https://2014.igem.org/Team:Imperial/Notebook">Overview</A></LI><LI><A href="https://2014.igem.org/Team:Imperial/Lab-book">Lab-book</A></LI><LI><A href="https://2014.igem.org/Team:Imperial/Protocols">Protocols</A></LI><LI><A href="https://2014.igem.org/Team:Imperial/Brainstorming">Brainstorming</A></LI><LI><A href="https://2014.igem.org/Team:Imperial/Safety">Safety</A></LI><LI><A href="https://2014.igem.org/Team:Imperial/Art_and_Design">Art &amp; Design</A></LI><LI><A href="#">Team</A></LI><LI><A href="https://igem.org/Team.cgi?id=1321">Official Profile</A></LI><LI><A href="https://2014.igem.org/Team:Imperial/Students">Students</A></LI><LI><A href="https://2014.igem.org/Team:Imperial/Instructors_and_Advisors">Instructors and Advisors</A></LI><LI><A href="https://2014.igem.org/Team:Imperial/Sponsors">Sponsors</A></LI><LI><A href="https://2014.igem.org/Team:Imperial/Attributions">Attributions</A></LI></DIV></DIV><DIV class="content-wrapper"><DIV class="pure-g"><DIV class="pure-u-1-1 main"><H1>Functionalisation</H1><DIV id="navWrap"><DIV id="subNav"><UL><LI><A data-scroll="" href="#introduction">Introduction</A></LI><LI><A data-scroll="" href="#aim">Aims</A></LI><LI><A data-scroll="" href="#CBDs">CBDs</A></LI><LI><A data-scroll="" href="#binding-partners">Functionalisation proteins</A></LI><LI><A data-scroll="" href="#results">Results</A></LI><LI><A data-scroll="" href="#references">References</A></LI></UL></DIV></DIV></DIV></DIV><SECTION id="overview"><DIV class="pure-g"><DIV class="pure-u-1-2"><H2>Overview</H2><P>Attaching functional proteins to cellulose can expand the properties of cellulose and allow us to selectivity capture specific contaminants in water. By creating fusion proteins of sfGFP and metal-binding proteins with five different cellulose binding domains, three of which are new to the registry, we were able to characterise the relative affinities of the CBDs and show proof-of-concept removal of contaminants from water.</P></DIV><DIV class="pure-u-1-2"><H2>Key Achievements </H2><UL><LI>Added three new cellulose binding domains to the registry</LI><LI>Performed an assay with sfGFP-fusions to determine the strongest potential binders which could be carried forward for further testing</LI><LI>Identified potential metal binding proteins to bind heavy metals from wastewater, including the addition of a new promising polypeptide phytochelatin EC20</LI><LI>Tested metal ion capture ability of the CBD-phytochelatin fusion constructs when attached to cellulose</LI></UL></DIV></DIV></SECTION><SECTION id="introduction"><H2>Introduction</H2><FIGURE class="content-image image-right"><FIGCAPTION>Figure 1. Pictorial summary showing the two cellulose functionalization approaches considered in our project. Strategy 1: Cellulose binding domains are expressed as fusions with the protein of interest, separated optionally by a linker sequence. The cellulose binding domains attach to the cellulose by hydrophobic interactions. Strategy 2: Curli fibrils are expressed in parallel with cellulose to form a composite material. The curli fibres coding sequence is modified to contain a ‘SpyTag’ peptide (Zakeri <I>et al</I> 2012) which binds specifically via a covalent isopeptide bond to its protein partner, ‘SpyCatcher’. This SpyCatcher is fused to the proteins of interest. In our project we only had time to pursue strategy 1 but are still excited about the potential of strategy 2 in the future.</FIGCAPTION></FIGURE><P>Bacterial cellulose is already a useful biomaterial as is due its attractive mechanical properties, with research on applications from loudspeaker diaphragms to protective packaging and wound dressings (Lee 2014). However, our project also aims to extend the properties of cellulose by conjugating functional proteins and peptides to it. This allows us to target specific contaminants in water which can not be filtered by size exclusion alone. (See <A href="https://2014.igem.org/Team:Imperial/Water_Report">water report</A> and <A href="https://2014.igem.org/Team:Imperial/Informing_Design">informing design</A> sections).</P><P>We considered two ways of functionalising cellulose (figure 1) but continued to work fully on the cellulose-binding domain (CBD) method. The curli-fibre method would be an exciting avenue to explore in the future; it is feasible since there has already been research into making a “programmable” biomaterial from curli fibres using a peptide tag approach (Nguyen <I>et al</I> 2014), and in combination with cellulose could produce a composite with robust mechanical properties as well.
                </P></SECTION><SECTION id="aim"><H2>Aims</H2><P>We aimed to make and characterise new cellulose binding domains, and fuse them to functional proteins. </P></SECTION><SECTION id="CBDs"><H2>Cellulose Binding Domains (CBDs)</H2><P>The cellulose binding domain (CBD) is the link between our functional proteins and the cellulose. Therefore, we wanted to use a CBD which had high affinity and strong binding to cellulose. However, weaker CBDs may still be useful since they could have the potential be controllably eluted under specific conditions, thereby enabling regeneration of the filter when saturated, and controlled disposal of the collected contaminants (refer to water report if relevant?).
                </P><P>We started our search by looking into the <A href="http://parts.igem.org/">parts registry</A> and found two CBDs already existed in the registry: CBDclos (<A href="http://parts.igem.org/Part:BBa_K863111">BBa_K863111</A>) and CBDcex (<A href="http://parts.igem.org/Part:BBa_K863101">BBa_K863101</A>). We also made three new CBDs to contribute to the registry: dCBD, CBDcipA, and CBDcenA. All the CBDs are in RFC(25) freiberg fusion format to allow for ease of use in protein fusions.</P><DIV class="accordion"><H3>CBDclos and CBDcex</H3><DIV><P>These parts were biobricked by <A href="https://2012.igem.org/Team:Bielefeld-Germany/Results/cbc">Bielefeld 2012</A>, please see their part pages (<A href="http://parts.igem.org/Part:BBa_K863111">BBa_K863111</A>a and <A href="http://parts.igem.org/Part:BBa_K863101">BBa_K863101</A>) for detailed information. CBDclos is from the cellulolytic bacterium <EM>Clostridium cellulovorans</EM> and CBDcex originates from the cellulolytic bacterium <EM>Cellulomonas fimi</EM>. This is the same organism as for CBDcenA; cex is from the exoglucanase gene and cen is from the endoglucanase gene.</P></DIV><H3>Double CBD (dCBD) with N-terminal linker</H3><DIV><P>This part, <A href="http://parts.igem.org/Part:BBa_K1321340">BBa_K1321340</A> is based on the double cellulose-binding domain construct synthesised and characterised by Linder <I>et al</I>1996, who found that this double CBD had higher affinity for cellulose than either of the two CBDs on their own. The main difference is that our part contains an additional linker sequence on the N-terminus of the protein.
                        </P><P>The two CBDs are from the fungus <I>Trichoderma reesei</I> (<I>Hypocrea jecorina</I>) Exocellobiohydrolase (Exoglucanase) I (cbh1), uniprot ID P62694; and Exocellobiohydrolase (Exoglucanase) II, uniprot ID P07987 (cbh2); with a linker peptide between the two CBDs and at the N-terminus of the protein. Both linkers are the same amino acid sequence and are based on the endogenous linker sequences that exists in cbh1 and cbh2 genes. The linker sequence is PGANPPGTTTTSRPATTTGSSPGP which is the same as used by Linder <I>et al</I> 1996. The first three amino acids are from the cbh2 endogenous linker, and the rest is from the cbh1 endogenous linker. CBDcbh1 is placed C-terminal to CBDcbh2 because naturally CBDcbh1 is a C-terminal domain and CBDcbh2 is an N-terminal domain. Both CBDs are from the <A class="iframe" href="http://www.cazy.org/CBM1.html">CBM family 1</A>. The precise location of the CBD within the cbh genes was slightly different according to the uniprot annotations and the sequence used by Linder <I>et al</I> 1996; we chose to use the sequence from the paper since the protein was expressed and characterised successfully.</P></DIV><H3>CBDcipA with N and C-terminal linker</H3><DIV><P>This CBD is from the Cellulosomal -scaffolding protein A (cipA) of Clostridium thermocellum including the endogenous linker sequences at the N and C-terminus (<A class="iframe" href="http://www.uniprot.org/uniprot/Q06851">UniProt ID Q06851</A>). It is in RFC25 format to allow for easy use in protein fusions.</P><P>The CBD is part of the <A class="iframe" href="http://www.cazy.org/CBM3.html">CBM3 family</A> . This CBD has been used in many application: in fusions with cell adhesion peptides to enhance the properties of cellulose as a cell-growth matrix (Andrade <I>et al </I> 2010a, Andrade <I>et al </I> 2010b), fused to enzymes to remove contaminants from water (Kauffmann <I> et al </I> 2000) and fused to an antimicrobial peptide (Ramos, Domingues &amp; Gama 2010).</P><P>At present the cloning for these constructs is still in progress to correct an illegal EcoRI site which was identified in the parts with this CBD. This can be achieved with a silent mutation via site-directed mutagenesis and we aim to send these parts to the registry once thi
                        </P></DIV><H3>CBDcenA with C-terminal linker</H3><DIV><P>This CBD is from the Endoglucanase A (cenA) gene from <I>Cellulomonas fimi</I> and also contains an endogenous C-terminal linker. The cellulose binding domain region occurs at the N-terminus of the cenA gene (UniProt P07984 link to http://www.uniprot.org/uniprot/P07984) and is from the <A class="iframe" href="http://www.cazy.org/CBM2.html">CBM family 2</A></P><P>The linker sequence (PTTSPTPTPTPTTPTPTPTPTPTPTPTVTP) is Pro-Thr box which has an extended conformation and acts as a hinge region (Gilkes <I>et al</I> 1989). The endoglucanase CBDcenA has 50% homology to the exoglucanase CBDcex (<A class="iframe" href="http://parts.igem.org/Part:BBa_K863101">BBa_K863101</A>) and the linker is highly conserved in both, but the order of the catalytic, linker and cellulose-binding regions is reversed (Warren <I>et al</I> 1986). It has been shown that CBDcenA has the highest binding affinity for crystalline cellulose out of the <I>C. fimi</I> CBDs (Kim <I>et al</I> 2013).
                        </P></DIV></DIV></SECTION><SECTION id="binding-partners"><H2>Functionalisation proteins: Binding partners</H2><H3>Superfolder Green Fluorescent Protein (sfGFP) </H3><FIGURE class="content-image image-right"><FIGCAPTION>sfGFP-CBD fusion parts</FIGCAPTION></FIGURE><P> sfGFP is a more stable mutant of <A href="http://parts.igem.org/Parts:BBa_E0040">mut3GFP</A> and we chose to use this as a robust, easily visible protein to initially fuse to our CBDs. We also designed one of our assays [insert link on 'one of our assays'] for characterising CBD binding using the sfGFP fusion constructs. sfGFP already existed in the registry (<A href="http://parts.igem.org/Part:BBa_I746916">BBa_I746916</A>), detailed information on sfGFP properties is included in this part description) however since we needed to make fusion proteins with our CBDs we modified the part and submitted it in RFC[25] Freiberg fusion format (<A href="http://parts.igem.org/Parts:BBa_K1321337">K1321337</A>). We created the following fusion parts with sfGFP: 

	

</P><H3> Metal Binding Proteins </H3><P>One of the contaminants we can filter out which current size-exclusion technologies can not target, are <A href="https://2014.igem.org/Team:Imperial/Water_Report">heavy metals</A>. We have worked with four different metal binding proteins: the metallothioneins SmtA and fMT, a histidine-rich nickel binding protein (NiBP) from the bacterium <I>Heliobacter pylori</I> and a synthetic phytochelatin (PC) EC20.
                </P><DIV class="accordion"><H3>SmtA and fMT metallothioneins</H3><DIV><P>These two parts already existed in the BioBrick registry, but since we needed to use them as fusions with the CBDs we improved the parts by making them compatible with the <A class="iframe" href="http://parts.igem.org/Assembly_standard_25">RFC(25) Frieburg Fusion BioBrick standard.</A></P><P>Metallothioneins (MTs) are genetically encoded cysteine-rich peptides which serve a role in the detoxification of heavy metals in plants, animals and some prokaryotes (Cobbett, 2002). They have a similar structure and function to phytochelatins, but are produced by different mechanisms (Cobbett, 2002). SmtA is from the cyanobacteria <I>Synechococcus sp.</I> and was used by the <A target="_blank" href="https://2011.igem.org/Team:Tokyo-NoKoGen">Tokyo-NoKoGen 2011</A> iGEM team (who submitted the RFC10 part originally, <A class="iframe" href="http://parts.igem.org/Part:BBa_K519010">BBa_K519010</A>) to confer cadmium (Cd2+) tolerance to <EM>E. coli</EM> cells when expressed. fMT originates from the seaweed <I>Fucus vesiculosus</I> and was submitted by <A target="_blank" href="https://2009.igem.org/Team:Groningen">Groningen 2009 iGEM team </A> and detailed information can be found on the original part page (<A class="iframe" href="http://parts.igem.org/Part:BBa_K190019">BBa_K190019</A>).</P><P>We successfully <A href="https://2014.igem.org/Team:Imperial/Protocols#general">cloned</A> the following CBD fusion constructs:  </P><FIGURE class="content-image image-right image-small"><FIGCAPTION>SmtA-CBD fusion parts</FIGCAPTION></FIGURE><FIGURE class="content-image image-left image-small"><FIGCAPTION>fMT-CBD fusion parts</FIGCAPTION></FIGURE></DIV><H3>Nickel binding protein (NiBP)</H3><DIV><FIGURE class="content-image image-right"><FIGCAPTION>NiBP-CBD fusions parts</FIGCAPTION></FIGURE><P>
This protein is the histidine-rich nickel-binding protein from <I>Heliobacter pylori</I> (GenBank accession ACX98466.1), and already existed as a BioBrick in the registry (<A class="iframe" href="http://parts.igem.org/Part:K1151001">K1151001</A>). Since we needed to use this part in fusion proteins we improved the part by submitting it as a basic part in RFC(25) (link) Freiberg fusion format (<A class="iframe" href="http://parts.igem.org/Part:K1321009">K1321009</A>).
                        </P><P>We successfully <A href="https://2014.igem.org/Team:Imperial/Protocols#general">cloned</A> the following CBD fusion constructs with NiBP:

</P><P>However we discovered late on in the project that the DNA sequence of the NiBP protein in the registry is out of frame, leading to an incorrect protein translation and read-through of the stop codon (figure 2).</P><FIGURE class="content-image"><FIGCAPTION>Figure 2. Alignment of BBa_K1151001 NiBP in the registry to the , showing incorrect amino-acid sequence in red compared to <A target="_blank" href="http://www.ncbi.nlm.nih.gov/protein/261838700">native <I>H.pylori</I></A> sequence.
                            </FIGCAPTION></FIGURE><FIGURE class="content-image image-right"><FIGCAPTION>Figure 3. Single nucleotide deletion in BBa_K1151001 compared to native <I>H.pylori</I> sequence (<A target="_blank" href="http://www.ncbi.nlm.nih.gov/nuccore/261837457">GenBank CP000012.1</A> (link ) bp1391851-1392033, reverse complement) resulting in any downstream nucleotides being out of frame.</FIGCAPTION></FIGURE><P>Further investigation into the error let us to identify a single nucleotide deletion in <A class="iframe" href="http://parts.igem.org/Part:BBa_K1151001">BBa_K1151001</A> compared to the native <I>H.pylori</I> DNA sequence (<A target="_blank" href="http://www.ncbi.nlm.nih.gov/nuccore/261837457">GenBank CP000012.1</A> bp1391851-1392033, reverse complement strand).  As seen in Figure 3 the deletion is a ‘C’ at bp 53 of the native sequence. This error was propagated through all our NiBP constructs and means that any subsequent C-terminal fusions are also out of frame and we were unable to express and characterise these constructs. We have noted this on the description pages for each of the parts and hope that in the future they will still be useful, since the error could be corrected relatively easily by a team who wishes to use them by site-directed-mutagenesis PCR to insert the missing nucleotide.  </P></DIV><H3>Synthetic phytochelatin EC20</H3><DIV><P>This is our new contribution of a metal-binding protein to the parts registry. Synthetic phytochelatins are analogues of phytochelatins (PCs); cysteine-rich peptides of the general structure (γ-GluCys)n-Gly (n~2-5) which bind and sequester heavy metals, with a role in detoxification (Cobbett &amp; Goldsbrough 2002). PCs are found mainly in plants but also in other organisms including diatoms, fungi, algae (including cyanobacteria) and some invertebrates (Cobbett &amp; Goldsbrough 2002, Rea 2012). Natively, the PCs are enzymatically synthesized by phytochelatin synthase (PCS) from a glutathione (GSH) precursor and the amino acids in the resulting peptide are linked by the non-standard gamma peptide bonds (Cobbett &amp; Goldsbrough 2002, Rea 2012). However, genetically encoded analogues (containing normal alpha-peptide bonds) of different number Glu-Cys (EC) repeats have been characterised, with the EC20 having the highest binding (Bae <I>et al</I> 2000).
                        </P><FIGURE class="content-image image-right image-small"><FIGCAPTION>Table 1. Metal ion adsorption increase of C. metallidurans cells with PC EC20 displayed on cell surface, compared to negative control. Taken from Biondo <I>et al</I> 2012</FIGCAPTION></FIGURE><P>The main metal PCs confer tolerance to is Cadmium (Cobbett &amp; Goldsbrough 2002) with a reported stoichiometry of 10 Cd2+ per peptide (Bae <I>et al</I>, . EC20 peptide has still showed good metal ion binding whilst fused to a cellulose binding domain (CBD) for water purification (Xu <I>et al</I> 2002) and when anchored to bacterial cell membrane to confer Cd2+ tolerance (Bae <I>et al</I> 2000, Chaturvedi &amp; Archana 2014). PC EC20 has also been shown to bind many more heavy metals. For example, when displayed on the surface of Cupriavidus metallidurans, the cells capacity to adsorb ions increased by 31 to 219% depending on the ion (table 1, Biondo <I>et al</I> 2012). These binding properties of PC EC20 have been utilised in applications including a regeneratable biosensor of Hg2+, Cd2+, Pb2+, Cu2+ and Zn2+ ions to 100 fM–10 mM (Bontidean <I>et al</I> 2003), and to target CdSe/ZnS quantum dots to streptavidin expressing HeLa cells when biotinylated (Pinaud <I>et al</I> 2004). INSERT TABLE OF THE IONS FROM THE PAPER
                        </P><P>We used this part to <A href="https://2014.igem.org/Team:Imperial/Protocols#general">clone</A> a library of different cellulose-binding domains fused to phytochelatin:<FIGURE class="content-image"><FIGCAPTION>Phytochelatin-CBD fusion parts. </FIGCAPTION></FIGURE></P></DIV></DIV><H3> Other potential proteins </H3><P>During our research we also explored other functional proteins to fuse to the CBDs.
                    </P><DIV class="accordion"><H3>Laccases</H3><DIV><P>Laccases are enzymes which can break down aromatic and phenolic compounds such as modified estrogens which can end up unfiltered in the water supply with unknown consequences (Auriol <I>et al</I> 2008). This method of treating estrogen in water was explored by <A target="_blank" href="https://2012.igem.org/Team:Bielefeld-Germany">Bielefeld 2012 iGEM team</A>, and our system also has potential to address this.
                            </P></DIV><H3>Nanobodies</H3><DIV><P>Nanobodies are fragments of camelid antibodies which can be generated against a variety of antigens and can be easily expressed by organisms such as <I>E. coli </I>. Nanobodies could be used to target viruses, which are often too small to be filtered by size exclusion. For example there are already nanobodies which can bind to <I> Vaccinia </I> virus, ricin, <I>Cholera</I> toxin, Stephylococcal enterotoxin B and H5N1 Influenza (Goldman <I>et al </I> 2006, Ibanez <I>et al </I> 2011) to name a few. Additionally, nanobodies are stable and can be selected to resist proteolytic attach (Harmsen <I>et al </I> 2006). Nanobodies can also be functional when as fusions or conjugates to other proteins for example to target enzymes to specific cells (Cortez-Retamozo 2004), so there is much potential for targeting a wide array of contaminants when fused to cellulose binding domains.

                            </P></DIV></DIV></SECTION><SECTION id="results"><H2>Results</H2><P>The assay for strength of CBD binding utilised our sfGFP-CBD fusion proteins. Strength of binding was assayed between the CBDs by assuming it will be proportional to the level of fluorescence which remains after washing of cellulose coated plates(link to plates protocol)
                </P><DIV class="accordion"><H3>Preliminary Assay</H3><DIV><FIGURE class="content-image image-full"><FIGCAPTION>Figure: </FIGCAPTION></FIGURE><P>text</P></DIV><H3>Full Assay</H3><DIV><FIGURE class="content-image image-full"><FIGCAPTION>Figure: </FIGCAPTION></FIGURE><FIGURE class="content-image image-full"><FIGCAPTION>Figure: </FIGCAPTION></FIGURE><P>text</P></DIV></DIV></SECTION><SECTION id="references"><H2>References</H2><UL><LI>Andrade, F.K., Costa, R., Domingues, L., Soares, R., et al. (2010) Improving bacterial cellulose for blood vessel replacement: Functionalization with a chimeric protein containing a cellulose-binding module and an adhesion peptide. Acta biomaterialia. [Online] 6 (10), 4034–4041. Available from: doi:10.1016/j.actbio.2010.04.023.</LI><LI>Andrade, F.K., Moreira, S.M.G., Domingues, L. &amp; Gama, F.M.P. (2010) Improving the affinity of fibroblasts for bacterial cellulose using carbohydrate-binding modules fused to RGD. Journal of biomedical materials research. Part A. [Online] 92 (1), 9–17. Available from: doi:10.1002/jbm.a.32284.</LI><LI>Auriol, M., Filali-Meknassi, Y., Adams, C.D., Tyagi, R.D., et al. (2008) Removal of estrogenic activity of natural and synthetic hormones from a municipal wastewater: efficiency of horseradish peroxidase and laccase from Trametes versicolor. Chemosphere. [Online] 70 (3), 445–452. Available from: doi:10.1016/j.chemosphere.2007.06.064.</LI><LI>Bae, W., Chen, W., Mulchandani, A. &amp; Mehra, R.K. (2000) Enhanced bioaccumulation of heavy metals by bacterial cells displaying synthetic phytochelatins. Biotechnology and bioengineering. 70 (5), 518–524.</LI><LI>Biondo, R., da Silva, F.A., Vicente, E.J., Souza Sarkis, J.E., et al. (2012) Synthetic phytochelatin surface display in Cupriavidus metallidurans CH34 for enhanced metals bioremediation. Environmental science &amp; technology. 46 (15), 8325–8332.</LI><LI>Bontidean, I., Ahlqvist, J., Mulchandani, A., Chen, W., et al. (2003) Novel synthetic phytochelatin-based capacitive biosensor for heavy metal ion detection. Biosensors and Bioelectronics. 18 (5-6), 547–553.</LI><LI>Chaturvedi, R. &amp; Archana, G. (2014) Cytosolic expression of synthetic phytochelatin and bacterial metallothionein genes in Deinococcus radiodurans R1 for enhanced tolerance and bioaccumulation of cadmium. Biometals : an international journal on the role of metal ions in biology, biochemistry, and medicine. 27 (3), 471–482.</LI><LI>Cobbett, C. &amp; Goldsbrough, P. (2002) Phytochelatins and metallothioneins: roles in heavy metal detoxification and homeostasis. Annual review of plant biology. [Online] 53159–182. Available from: doi:10.1146/annurev.arplant.53.100301.135154.</LI><LI>Cortez-Retamozo, V. (2004) Efficient Cancer Therapy with a Nanobody-Based Conjugate. Cancer Research. [Online] 64 (8), 2853–2857. Available from: doi:10.1158/0008-5472.CAN-03-3935.</LI><LI>Gilkes, N.R., Kilburn, D.G., Miller, R.C. &amp; Warren, R.A. (1989) Structural and functional analysis of a bacterial cellulase by proteolysis. The Journal of biological chemistry. 264 (30), 17802–17808.</LI><LI>Harmsen, M.M., van Solt, C.B., van Zijderveld-van Bemmel, A.M., Niewold, T.A., et al. (2006) Selection and optimization of proteolytically stable llama single-domain antibody fragments for oral immunotherapy. Applied microbiology and biotechnology. [Online] 72 (3), 544–551. Available from: doi:10.1007/s00253-005-0300-7.</LI><LI>Kauffmann, C., Shoseyov, O., Shpigel, E., Bayer, E.A., et al. (2000) Novel Methodology for Enzymatic Removal of Atrazine from Water by CBD-Fusion Protein Immobilized on Cellulose. Environmental Science &amp; Technology. [Online] 34 (7), 1292–1296. Available from: doi:10.1021/es990754h.</LI><LI>Kim, H.-D., Choi, S.-L., Kim, H., Sohn, J.H., et al. (2013) Enzyme-linked assay of cellulose-binding domain functions from Cellulomonas fimi on multi-well microtiter plate. Biotechnology and Bioprocess Engineering. 18 (3), 575–580.</LI><LI>Lee, K.-Y., Buldum, G., Mantalaris, A. and Bismarck, A. (2014), More Than Meets the Eye in Bacterial Cellulose: Biosynthesis, Bioprocessing, and Applications in Advanced Fiber Composites. Macromol. Biosci., 14: 10–32. doi: 10.1002/mabi.201300298</LI><LI>Linder, M.; Salovuori, I.; Ruohonen, L.; Teeri, T.T., 1996. Characterization of a Double Cellulose-binding Domain. SYNERGISTIC HIGH AFFINITY BINDING TO CRYSTALLINE CELLULOSE. Journal of Biological Chemistry, 271(35), pp.21268–21272. Available at: http://www.jbc.org/content/271/35/21268.full</LI><LI>Nguyen, P.Q., Botyanszki, Z., Tay, P.K.R. &amp; Joshi, N.S. (2014) Programmable biofilm-based materials from engineered curli nanofibres. Nature communications. [Online] 54945. Available from: doi:10.1038/ncomms5945.</LI><LI>Pinaud, F., King, D., Moore, H.-P. &amp; Weiss, S. (2004) Bioactivation and cell targeting of semiconductor CdSe/ZnS nanocrystals with phytochelatin-related peptides. Journal of the American Chemical Society. 126 (19), 6115–6123.</LI><LI>Ramos, R., Domingues, L. &amp; Gama, M. (2010) Escherichia coli expression and purification of LL37 fused to a family III carbohydrate-binding module from Clostridium thermocellum. Protein expression and purification. [Online] 71 (1), 1–7. Available from: doi:10.1016/j.pep.2009.10.016.</LI><LI>Warren, R.A., Beck, C.F., Gilkes, N.R., Kilburn, D.G., et al. (1986) Sequence conservation and region shuffling in an endoglucanase and an exoglucanase from Cellulomonas fimi. Proteins. 1 (4), 335–341.</LI><LI>Xu, Z., Bae, W., Mulchandani, A., Mehra, R.K., et al. (2002) Heavy Metal Removal by Novel CBD-EC20 Sorbents Immobilized on Cellulose. Biomacromolecules. [Online] 3 (3), 462–465.</LI><LI>Zakeri, B., Fierer, J.O., Celik, E., Chittock, E.C., et al. (2012) Peptide tag forming a rapid covalent bond to a protein, through engineering a bacterial adhesin. Proceedings of the National Academy of Sciences of the United States of America. [Online] 109 (12), E690–E697. Available from: doi:10.1073/pnas.1115485109.</LI></UL></SECTION></DIV><DIV class="printfooter">
Retrieved from &quot;<A href="http://2014.igem.org/Team:Imperial/Functionalisation">http://2014.igem.org/Team:Imperial/Functionalisation</A>&quot;</DIV></DIV></DIV><DIV id="footer-box" class="noprint"><DIV id="footer"><UL id="f-list"><LI id="t-recentchanges"><A href="/Special:RecentChanges" title="Recent changes">Recent changes</A></LI><LI id="t-whatlinkshere"><A href="/Special:WhatLinksHere/Team:Imperial/Functionalisation" title="List of all wiki pages that link here [j]" accesskey="j">What links here</A></LI><LI id="t-recentchangeslinked"><A href="/Special:RecentChangesLinked/Team:Imperial/Functionalisation" title="Recent changes in pages linked from this page [k]" accesskey="k">Related changes</A></LI><LI id="t-specialpages"><A href="/Special:SpecialPages" title="List of all special pages [q]" accesskey="q">Special pages</A></LI><LI><A href="/Special:Preferences">My preferences</A></LI></UL></DIV><DIV id="footer"><UL id="f-list"><LI id="t-print"><A href="/wiki/index.php?title=Team:Imperial/Functionalisation&amp;printable=yes" title="Printable version of this page [p]" accesskey="p">Printable version</A></LI><LI id="t-permalink"><A href="/wiki/index.php?title=Team:Imperial/Functionalisation&amp;oldid=401737" title="Permanent link to this revision of the page">Permanent link</A></LI><LI id="privacy"><A href="/2014.igem.org:Privacy_policy" title="2014.igem.org:Privacy policy">Privacy policy</A></LI><LI id="disclaimer"><A href="/2014.igem.org:General_disclaimer" title="2014.igem.org:General disclaimer">Disclaimers</A></LI></UL></DIV></DIV></DIV></DIV></BODY></HTML>