"
Page
Discussion
View source
History
teams
Log in
 
Team:Imperial/Functionalisation
From 2014.igem.org
Imperial iGEM 2014
Aqualose
Project
Overview
Background
G. xylinus
E. coli
Co-culture
Functionalisation
Mass Production and Processing
Mechanical Testing
Achievements
Overview
Parts
Results
Judging Criteria
Interlab Study
Application
Implementation
Water Filtration
Policy & Practices
Overview
Informing Design
Water Report
The i in iGEM
Outreach
Collaborations
Modelling
Overview
Induction Model
CBD Kinetics Model
Nutrient Diffusion Simulations
BioHackspace Model
Notebook
Overview
Lab-book
Protocols
Brainstorming
Safety
Art & Design
Team
Official Profile
Students
Instructors and Advisors
Sponsors
Attributions
Functionalisation
Introduction
Aims
CBDs
Functionalisation proteins
Results
References
Overview
Attaching functional proteins to cellulose can expand the properties of cellulose and allow us to selectivity capture specific contaminants in water. By creating fusion proteins of sfGFP and metal-binding proteins with five different cellulose binding domains, three of which are new to the registry, we were able to characterise the relative affinities of the CBDs and show proof-of-concept removal of contaminants from water.
Key Achievements
Added three new cellulose binding domains to the registry
Performed an assay with sfGFP-fusions to determine the strongest potential binders which could be carried forward for further testing
Identified potential metal binding proteins to bind heavy metals from wastewater, including the addition of a new promising polypeptide phytochelatin EC20
Tested metal ion capture ability of the CBD-phytochelatin fusion constructs when attached to cellulose
Introduction
Figure 1. Pictorial summary showing the two cellulose functionalization approaches considered in our project. Strategy 1: Cellulose binding domains are expressed as fusions with the protein of interest, separated optionally by a linker sequence. The cellulose binding domains attach to the cellulose by hydrophobic interactions. Strategy 2: Curli fibrils are expressed in parallel with cellulose to form a composite material. The curli fibres coding sequence is modified to contain a ‘SpyTag’ peptide (Zakeri et al 2012) which binds specifically via a covalent isopeptide bond to its protein partner, ‘SpyCatcher’. This SpyCatcher is fused to the proteins of interest. In our project we only had time to pursue strategy 1 but are still excited about the potential of strategy 2 in the future.
Bacterial cellulose is already a useful biomaterial as is due its attractive mechanical properties, with research on applications from loudspeaker diaphragms to protective packaging and wound dressings (Lee 2014). However, our project also aims to extend the properties of cellulose by conjugating functional proteins and peptides to it. This allows us to target specific contaminants in water which can not be filtered by size exclusion alone. (See water report and informing design sections).
We considered two ways of functionalising cellulose (figure 1) but continued to work fully on the cellulose-binding domain (CBD) method. The curli-fibre method would be an exciting avenue to explore in the future; it is feasible since there has already been research into making a “programmable” biomaterial from curli fibres using a peptide tag approach (Nguyen et al 2014), and in combination with cellulose could produce a composite with robust mechanical properties as well.
Aims
We aimed to make and characterise new cellulose binding domains, and fuse them to functional proteins.
Cellulose Binding Domains (CBDs)
The cellulose binding domain (CBD) is the link between our functional proteins and the cellulose. Therefore, we wanted to use a CBD which had high affinity and strong binding to cellulose. However, weaker CBDs may still be useful since they could have the potential be controllably eluted under specific conditions, thereby enabling regeneration of the filter when saturated, and controlled disposal of the collected contaminants (refer to water report if relevant?).
We started our search by looking into the parts registry and found two CBDs already existed in the registry: CBDclos ( BBa_K863111 ) and CBDcex ( BBa_K863101 ). We also made three new CBDs to contribute to the registry: dCBD, CBDcipA, and CBDcenA. All the CBDs are in RFC(25) freiberg fusion format to allow for ease of use in protein fusions.
CBDclos and CBDcex
These parts were biobricked by Bielefeld 2012 , please see their part pages ( BBa_K863111 a and BBa_K863101 ) for detailed information. CBDclos is from the cellulolytic bacterium Clostridium cellulovorans and CBDcex originates from the cellulolytic bacterium Cellulomonas fimi. This is the same organism as for CBDcenA; cex is from the exoglucanase gene and cen is from the endoglucanase gene.
Double CBD (dCBD) with N-terminal linker
This part, BBa_K1321340 is based on the double cellulose-binding domain construct synthesised and characterised by Linder et al1996, who found that this double CBD had higher affinity for cellulose than either of the two CBDs on their own. The main difference is that our part contains an additional linker sequence on the N-terminus of the protein.
The two CBDs are from the fungus Trichoderma reesei (Hypocrea jecorina) Exocellobiohydrolase (Exoglucanase) I (cbh1), uniprot ID P62694; and Exocellobiohydrolase (Exoglucanase) II, uniprot ID P07987 (cbh2); with a linker peptide between the two CBDs and at the N-terminus of the protein. Both linkers are the same amino acid sequence and are based on the endogenous linker sequences that exists in cbh1 and cbh2 genes. The linker sequence is PGANPPGTTTTSRPATTTGSSPGP which is the same as used by Linder et al 1996. The first three amino acids are from the cbh2 endogenous linker, and the rest is from the cbh1 endogenous linker. CBDcbh1 is placed C-terminal to CBDcbh2 because naturally CBDcbh1 is a C-terminal domain and CBDcbh2 is an N-terminal domain. Both CBDs are from the CBM family 1 . The precise location of the CBD within the cbh genes was slightly different according to the uniprot annotations and the sequence used by Linder et al 1996; we chose to use the sequence from the paper since the protein was expressed and characterised successfully.
CBDcipA with N and C-terminal linker
This CBD is from the Cellulosomal -scaffolding protein A (cipA) of Clostridium thermocellum including the endogenous linker sequences at the N and C-terminus ( UniProt ID Q06851 ). It is in RFC25 format to allow for easy use in protein fusions.
The CBD is part of the CBM3 family . This CBD has been used in many application: in fusions with cell adhesion peptides to enhance the properties of cellulose as a cell-growth matrix (Andrade et al 2010a, Andrade et al 2010b), fused to enzymes to remove contaminants from water (Kauffmann et al 2000) and fused to an antimicrobial peptide (Ramos, Domingues & Gama 2010).
At present the cloning for these constructs is still in progress to correct an illegal EcoRI site which was identified in the parts with this CBD. This can be achieved with a silent mutation via site-directed mutagenesis and we aim to send these parts to the registry once thi
CBDcenA with C-terminal linker
This CBD is from the Endoglucanase A (cenA) gene from Cellulomonas fimi and also contains an endogenous C-terminal linker. The cellulose binding domain region occurs at the N-terminus of the cenA gene (UniProt P07984 link to http://www.uniprot.org/uniprot/P07984) and is from the CBM family 2
The linker sequence (PTTSPTPTPTPTTPTPTPTPTPTPTPTVTP) is Pro-Thr box which has an extended conformation and acts as a hinge region (Gilkes et al 1989). The endoglucanase CBDcenA has 50% homology to the exoglucanase CBDcex ( BBa_K863101 ) and the linker is highly conserved in both, but the order of the catalytic, linker and cellulose-binding regions is reversed (Warren et al 1986). It has been shown that CBDcenA has the highest binding affinity for crystalline cellulose out of the C. fimi CBDs (Kim et al 2013).
Functionalisation proteins: Binding partners
Superfolder Green Fluorescent Protein (sfGFP)
sfGFP-CBD fusion parts
sfGFP is a more stable mutant of mut3GFP and we chose to use this as a robust, easily visible protein to initially fuse to our CBDs. We also designed one of our assays [insert link on 'one of our assays'] for characterising CBD binding using the sfGFP fusion constructs. sfGFP already existed in the registry ( BBa_I746916 ), detailed information on sfGFP properties is included in this part description) however since we needed to make fusion proteins with our CBDs we modified the part and submitted it in RFC[25] Freiberg fusion format ( K1321337 ). We created the following fusion parts with sfGFP:
Metal Binding Proteins
One of the contaminants we can filter out which current size-exclusion technologies can not target, are heavy metals . We have worked with four different metal binding proteins: the metallothioneins SmtA and fMT, a histidine-rich nickel binding protein (NiBP) from the bacterium Heliobacter pylori and a synthetic phytochelatin (PC) EC20.
SmtA and fMT metallothioneins
These two parts already existed in the BioBrick registry, but since we needed to use them as fusions with the CBDs we improved the parts by making them compatible with the RFC(25) Frieburg Fusion BioBrick standard.
Metallothioneins (MTs) are genetically encoded cysteine-rich peptides which serve a role in the detoxification of heavy metals in plants, animals and some prokaryotes (Cobbett, 2002). They have a similar structure and function to phytochelatins, but are produced by different mechanisms (Cobbett, 2002). SmtA is from the cyanobacteria Synechococcus sp. and was used by the Tokyo-NoKoGen 2011 iGEM team (who submitted the RFC10 part originally, BBa_K519010 ) to confer cadmium (Cd2+) tolerance to E. coli cells when expressed. fMT originates from the seaweed Fucus vesiculosus and was submitted by Groningen 2009 iGEM team and detailed information can be found on the original part page ( BBa_K190019 ).
We successfully cloned the following CBD fusion constructs:
SmtA-CBD fusion parts
fMT-CBD fusion parts
Nickel binding protein (NiBP)
NiBP-CBD fusions parts
This protein is the histidine-rich nickel-binding protein from Heliobacter pylori (GenBank accession ACX98466.1), and already existed as a BioBrick in the registry ( K1151001 ). Since we needed to use this part in fusion proteins we improved the part by submitting it as a basic part in RFC(25) (link) Freiberg fusion format ( K1321009 ).
We successfully cloned the following CBD fusion constructs with NiBP:
However we discovered late on in the project that the DNA sequence of the NiBP protein in the registry is out of frame, leading to an incorrect protein translation and read-through of the stop codon (figure 2).
Figure 2. Alignment of BBa_K1151001 NiBP in the registry to the , showing incorrect amino-acid sequence in red compared to native H.pylori sequence.
Figure 3. Single nucleotide deletion in BBa_K1151001 compared to native H.pylori sequence ( GenBank CP000012.1 (link ) bp1391851-1392033, reverse complement) resulting in any downstream nucleotides being out of frame.
Further investigation into the error let us to identify a single nucleotide deletion in BBa_K1151001 compared to the native H.pylori DNA sequence ( GenBank CP000012.1 bp1391851-1392033, reverse complement strand).  As seen in Figure 3 the deletion is a ‘C’ at bp 53 of the native sequence. This error was propagated through all our NiBP constructs and means that any subsequent C-terminal fusions are also out of frame and we were unable to express and characterise these constructs. We have noted this on the description pages for each of the parts and hope that in the future they will still be useful, since the error could be corrected relatively easily by a team who wishes to use them by site-directed-mutagenesis PCR to insert the missing nucleotide.
Synthetic phytochelatin EC20
This is our new contribution of a metal-binding protein to the parts registry. Synthetic phytochelatins are analogues of phytochelatins (PCs); cysteine-rich peptides of the general structure (γ-GluCys)n-Gly (n~2-5) which bind and sequester heavy metals, with a role in detoxification (Cobbett & Goldsbrough 2002). PCs are found mainly in plants but also in other organisms including diatoms, fungi, algae (including cyanobacteria) and some invertebrates (Cobbett & Goldsbrough 2002, Rea 2012). Natively, the PCs are enzymatically synthesized by phytochelatin synthase (PCS) from a glutathione (GSH) precursor and the amino acids in the resulting peptide are linked by the non-standard gamma peptide bonds (Cobbett & Goldsbrough 2002, Rea 2012). However, genetically encoded analogues (containing normal alpha-peptide bonds) of different number Glu-Cys (EC) repeats have been characterised, with the EC20 having the highest binding (Bae et al 2000).
Table 1. Metal ion adsorption increase of C. metallidurans cells with PC EC20 displayed on cell surface, compared to negative control. Taken from Biondo et al 2012
The main metal PCs confer tolerance to is Cadmium (Cobbett & Goldsbrough 2002) with a reported stoichiometry of 10 Cd2+ per peptide (Bae et al, . EC20 peptide has still showed good metal ion binding whilst fused to a cellulose binding domain (CBD) for water purification (Xu et al 2002) and when anchored to bacterial cell membrane to confer Cd2+ tolerance (Bae et al 2000, Chaturvedi & Archana 2014). PC EC20 has also been shown to bind many more heavy metals. For example, when displayed on the surface of Cupriavidus metallidurans, the cells capacity to adsorb ions increased by 31 to 219% depending on the ion (table 1, Biondo et al 2012). These binding properties of PC EC20 have been utilised in applications including a regeneratable biosensor of Hg2+, Cd2+, Pb2+, Cu2+ and Zn2+ ions to 100 fM–10 mM (Bontidean et al 2003), and to target CdSe/ZnS quantum dots to streptavidin expressing HeLa cells when biotinylated (Pinaud et al 2004). INSERT TABLE OF THE IONS FROM THE PAPER
We used this part to clone a library of different cellulose-binding domains fused to phytochelatin:
Phytochelatin-CBD fusion parts.
Other potential proteins
During our research we also explored other functional proteins to fuse to the CBDs.
Laccases
Laccases are enzymes which can break down aromatic and phenolic compounds such as modified estrogens which can end up unfiltered in the water supply with unknown consequences (Auriol et al 2008). This method of treating estrogen in water was explored by Bielefeld 2012 iGEM team , and our system also has potential to address this.
Nanobodies
Nanobodies are fragments of camelid antibodies which can be generated against a variety of antigens and can be easily expressed by organisms such as E. coli . Nanobodies could be used to target viruses, which are often too small to be filtered by size exclusion. For example there are already nanobodies which can bind to Vaccinia virus, ricin, Cholera toxin, Stephylococcal enterotoxin B and H5N1 Influenza (Goldman et al 2006, Ibanez et al 2011) to name a few. Additionally, nanobodies are stable and can be selected to resist proteolytic attach (Harmsen et al 2006). Nanobodies can also be functional when as fusions or conjugates to other proteins for example to target enzymes to specific cells (Cortez-Retamozo 2004), so there is much potential for targeting a wide array of contaminants when fused to cellulose binding domains.
Results
The assay for strength of CBD binding utilised our sfGFP-CBD fusion proteins. Strength of binding was assayed between the CBDs by assuming it will be proportional to the level of fluorescence which remains after washing of cellulose coated plates(link to plates protocol)
Preliminary Assay
Figure:
text
Full Assay
Figure:
Figure:
text
References
Andrade, F.K., Costa, R., Domingues, L., Soares, R., et al. (2010) Improving bacterial cellulose for blood vessel replacement: Functionalization with a chimeric protein containing a cellulose-binding module and an adhesion peptide. Acta biomaterialia. [Online] 6 (10), 4034–4041. Available from: doi:10.1016/j.actbio.2010.04.023.
Andrade, F.K., Moreira, S.M.G., Domingues, L. & Gama, F.M.P. (2010) Improving the affinity of fibroblasts for bacterial cellulose using carbohydrate-binding modules fused to RGD. Journal of biomedical materials research. Part A. [Online] 92 (1), 9–17. Available from: doi:10.1002/jbm.a.32284.
Auriol, M., Filali-Meknassi, Y., Adams, C.D., Tyagi, R.D., et al. (2008) Removal of estrogenic activity of natural and synthetic hormones from a municipal wastewater: efficiency of horseradish peroxidase and laccase from Trametes versicolor. Chemosphere. [Online] 70 (3), 445–452. Available from: doi:10.1016/j.chemosphere.2007.06.064.
Bae, W., Chen, W., Mulchandani, A. & Mehra, R.K. (2000) Enhanced bioaccumulation of heavy metals by bacterial cells displaying synthetic phytochelatins. Biotechnology and bioengineering. 70 (5), 518–524.
Biondo, R., da Silva, F.A., Vicente, E.J., Souza Sarkis, J.E., et al. (2012) Synthetic phytochelatin surface display in Cupriavidus metallidurans CH34 for enhanced metals bioremediation. Environmental science & technology. 46 (15), 8325–8332.
Bontidean, I., Ahlqvist, J., Mulchandani, A., Chen, W., et al. (2003) Novel synthetic phytochelatin-based capacitive biosensor for heavy metal ion detection. Biosensors and Bioelectronics. 18 (5-6), 547–553.
Chaturvedi, R. & Archana, G. (2014) Cytosolic expression of synthetic phytochelatin and bacterial metallothionein genes in Deinococcus radiodurans R1 for enhanced tolerance and bioaccumulation of cadmium. Biometals : an international journal on the role of metal ions in biology, biochemistry, and medicine. 27 (3), 471–482.
Cobbett, C. & Goldsbrough, P. (2002) Phytochelatins and metallothioneins: roles in heavy metal detoxification and homeostasis. Annual review of plant biology. [Online] 53159–182. Available from: doi:10.1146/annurev.arplant.53.100301.135154.
Cortez-Retamozo, V. (2004) Efficient Cancer Therapy with a Nanobody-Based Conjugate. Cancer Research. [Online] 64 (8), 2853–2857. Available from: doi:10.1158/0008-5472.CAN-03-3935.
Gilkes, N.R., Kilburn, D.G., Miller, R.C. & Warren, R.A. (1989) Structural and functional analysis of a bacterial cellulase by proteolysis. The Journal of biological chemistry. 264 (30), 17802–17808.
Harmsen, M.M., van Solt, C.B., van Zijderveld-van Bemmel, A.M., Niewold, T.A., et al. (2006) Selection and optimization of proteolytically stable llama single-domain antibody fragments for oral immunotherapy. Applied microbiology and biotechnology. [Online] 72 (3), 544–551. Available from: doi:10.1007/s00253-005-0300-7.
Kauffmann, C., Shoseyov, O., Shpigel, E., Bayer, E.A., et al. (2000) Novel Methodology for Enzymatic Removal of Atrazine from Water by CBD-Fusion Protein Immobilized on Cellulose. Environmental Science & Technology. [Online] 34 (7), 1292–1296. Available from: doi:10.1021/es990754h.
Kim, H.-D., Choi, S.-L., Kim, H., Sohn, J.H., et al. (2013) Enzyme-linked assay of cellulose-binding domain functions from Cellulomonas fimi on multi-well microtiter plate. Biotechnology and Bioprocess Engineering. 18 (3), 575–580.
Lee, K.-Y., Buldum, G., Mantalaris, A. and Bismarck, A. (2014), More Than Meets the Eye in Bacterial Cellulose: Biosynthesis, Bioprocessing, and Applications in Advanced Fiber Composites. Macromol. Biosci., 14: 10–32. doi: 10.1002/mabi.201300298
Linder, M.; Salovuori, I.; Ruohonen, L.; Teeri, T.T., 1996. Characterization of a Double Cellulose-binding Domain. SYNERGISTIC HIGH AFFINITY BINDING TO CRYSTALLINE CELLULOSE. Journal of Biological Chemistry, 271(35), pp.21268–21272. Available at: http://www.jbc.org/content/271/35/21268.full
Nguyen, P.Q., Botyanszki, Z., Tay, P.K.R. & Joshi, N.S. (2014) Programmable biofilm-based materials from engineered curli nanofibres. Nature communications. [Online] 54945. Available from: doi:10.1038/ncomms5945.
Pinaud, F., King, D., Moore, H.-P. & Weiss, S. (2004) Bioactivation and cell targeting of semiconductor CdSe/ZnS nanocrystals with phytochelatin-related peptides. Journal of the American Chemical Society. 126 (19), 6115–6123.
Ramos, R., Domingues, L. & Gama, M. (2010) Escherichia coli expression and purification of LL37 fused to a family III carbohydrate-binding module from Clostridium thermocellum. Protein expression and purification. [Online] 71 (1), 1–7. Available from: doi:10.1016/j.pep.2009.10.016.
Warren, R.A., Beck, C.F., Gilkes, N.R., Kilburn, D.G., et al. (1986) Sequence conservation and region shuffling in an endoglucanase and an exoglucanase from Cellulomonas fimi. Proteins. 1 (4), 335–341.
Xu, Z., Bae, W., Mulchandani, A., Mehra, R.K., et al. (2002) Heavy Metal Removal by Novel CBD-EC20 Sorbents Immobilized on Cellulose. Biomacromolecules. [Online] 3 (3), 462–465.
Zakeri, B., Fierer, J.O., Celik, E., Chittock, E.C., et al. (2012) Peptide tag forming a rapid covalent bond to a protein, through engineering a bacterial adhesin. Proceedings of the National Academy of Sciences of the United States of America. [Online] 109 (12), E690–E697. Available from: doi:10.1073/pnas.1115485109.
Retrieved from " http://2014.igem.org/Team:Imperial/Functionalisation "
Recent changes
What links here
Related changes
Special pages
My preferences
Printable version
Permanent link
Privacy policy
Disclaimers
