<HTML xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en" dir="ltr">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki  ltr ns-0 ns-subject page-Team_Imperial_Protocols"><DIV id="globalWrapper"><DIV id="top-section"><DIV id="p-logo"><A href="/Main_Page" title="Main Page">&quot;
	    </A></DIV><DIV id="menubar" class="left-menu noprint"><UL><LI class="selected"><A href="/Team:Imperial/Protocols">Page               </A></LI><LI class="new"><A href="/wiki/index.php?title=Talk:Team:Imperial/Protocols&amp;action=edit&amp;redlink=1">Discussion               </A></LI><LI><A href="/wiki/index.php?title=Team:Imperial/Protocols&amp;action=edit">View source               </A></LI><LI><A href="/wiki/index.php?title=Team:Imperial/Protocols&amp;action=history">History               </A></LI><LI style="color:white;cursor:default">teams</LI></UL></DIV><DIV class="right-menu noprint" id="menubar"><UL><LI id="pt-login"><A href="/wiki/index.php?title=Special:UserLogin&amp;returnto=Team:Imperial/Protocols" title="You are encouraged to log in; however, it is not mandatory [o]" accesskey="o">Log in</A></LI></UL></DIV><DIV id="search-controls" class="noprint"><FORM action="/Special:Search" id="searchform"> </FORM></DIV></DIV><DIV id="content"><H1 class="firstHeading">Team:Imperial/Protocols</H1><DIV id="bodyContent"><H3 id="siteSub" class="noprint">From 2014.igem.org</H3><P><TITLE>Imperial iGEM 2014</TITLE></P><DIV class="wrapper"><DIV class="header"><DIV class="header-banner"><DIV class="left"><A style="text-decoration: none" href="https://2014.igem.org/Team:Imperial"><H1 id="Logo">Aqualose</H1></A></DIV></DIV><DIV id="main_nav" class="pure-menu pure-menu-open pure-menu-horizontal"><UL id="std-menu-items"><LI><A href="#">Project</A></LI><LI><A href="https://2014.igem.org/Team:Imperial/Project">Overview</A></LI><LI><A href="https://2014.igem.org/Team:Imperial/Project_Background">Background</A></LI><LI><A href="https://2014.igem.org/Team:Imperial/Gluconacetobacter"><EM>G. xylinus</EM></A></LI><LI><A href="https://2014.igem.org/Team:Imperial/EColi"><EM>E. coli</EM></A></LI><LI><A href="https://2014.igem.org/Team:Imperial/coculture">Co-culture</A></LI><LI><A href="https://2014.igem.org/Team:Imperial/Functionalisation">Functionalisation</A></LI><LI><A href="https://2014.igem.org/Team:Imperial/Mass_Production_and_Processing">Mass Production and Processing</A></LI><LI><A href="https://2014.igem.org/Team:Imperial/Mechanical_Testing">Mechanical Testing</A></LI></UL><LI><A href="#">Achievements</A></LI><LI><A href="https://2014.igem.org/Team:Imperial/Achievements">Overview</A></LI><LI><A href="https://2014.igem.org/Team:Imperial/Parts">Parts</A></LI><LI><A href="https://2014.igem.org/Team:Imperial/Results">Results</A></LI><LI><A href="https://2014.igem.org/Team:Imperial/Judging_Criteria">Judging Criteria</A></LI><LI><A href="https://2014.igem.org/Team:Imperial/InterLab_Study">Interlab Study</A></LI><LI><A href="#">Application</A></LI><LI><A href="https://2014.igem.org/Team:Imperial/Implementation">Implementation</A></LI><LI><A href="https://2014.igem.org/Team:Imperial/Water_Filtration">Water Filtration</A></LI><LI><A href="#">Policy &amp; Practices</A></LI><LI><A href="https://2014.igem.org/Team:Imperial/Policy_and_Practices">Overview</A></LI><LI><A href="https://2014.igem.org/Team:Imperial/Informing_Design">Informing Design</A></LI><LI><A href="https://2014.igem.org/Team:Imperial/Water_Report">Water Report</A></LI><LI><A href="https://2014.igem.org/Team:Imperial/The_i_in_iGEM">The i in iGEM</A></LI><LI><A href="https://2014.igem.org/Team:Imperial/Outreach">Outreach</A></LI><LI><A href="https://2014.igem.org/Team:Imperial/Collaborations">Collaborations</A></LI><LI><A href="#">Modelling</A></LI><LI><A href="https://2014.igem.org/Team:Imperial/Modelling">Overview</A></LI><LI><A href="https://2014.igem.org/Team:Imperial/Induction_Model">Induction Model</A></LI><LI><A href="https://2014.igem.org/Team:Imperial/CBD_Kinetics_Model">CBD Kinetics Model</A></LI><LI><A href="https://2014.igem.org/Team:Imperial/Nutrient_Diffusion_Simulations">Nutrient Diffusion Simulations</A></LI><LI><A href="https://2014.igem.org/Team:Imperial/BioHackspace_Model">BioHackspace Model</A></LI><LI><A href="#">Notebook</A></LI><LI><A href="https://2014.igem.org/Team:Imperial/Notebook">Overview</A></LI><LI><A href="https://2014.igem.org/Team:Imperial/Lab-book">Lab-book</A></LI><LI><A href="https://2014.igem.org/Team:Imperial/Protocols">Protocols</A></LI><LI><A href="https://2014.igem.org/Team:Imperial/Brainstorming">Brainstorming</A></LI><LI><A href="https://2014.igem.org/Team:Imperial/Safety">Safety</A></LI><LI><A href="https://2014.igem.org/Team:Imperial/Art_and_Design">Art &amp; Design</A></LI><LI><A href="#">Team</A></LI><LI><A href="https://igem.org/Team.cgi?id=1321">Official Profile</A></LI><LI><A href="https://2014.igem.org/Team:Imperial/Students">Students</A></LI><LI><A href="https://2014.igem.org/Team:Imperial/Instructors_and_Advisors">Instructors and Advisors</A></LI><LI><A href="https://2014.igem.org/Team:Imperial/Sponsors">Sponsors</A></LI><LI><A href="https://2014.igem.org/Team:Imperial/Attributions">Attributions</A></LI></DIV></DIV><DIV class="content-wrapper"><DIV class="pure-g"><DIV class="pure-u-1-1 main"><H1>Protocols</H1><DIV id="navWrap"><DIV id="subNav"><UL><LI><A data-scroll="" href="#gluconacetobacter"><EM>G. xylinus</EM></A></LI><LI><A data-scroll="" href="#general">General/<EM>E. coli</EM></A></LI><LI><A data-scroll="" href="#functionalisation">Functionalisation</A></LI></UL></DIV></DIV></DIV></DIV><DIV class="pure-g"><DIV class="pure-u-1-1"><P>Here you can find the protocols we used throughout the summer</P><SECTION id="gluconacetobacter"><H2><EM>G. xylinus</EM> Protocols</H2><DIV class="accordion"><H3>Kombucha Protocol</H3><DIV><DIV class="pure-g"><DIV class="pure-u-1-2"><UL><LI>Materials
                                                </LI><LI>Organic cider vinegar</LI><LI>Granulated sugar</LI><LI>One piece of live Kombucha culture</LI><LI>Green tea bag</LI><LI>Distilled water</LI></UL></DIV><DIV class="pure-u-1-2"><OL><LI>Pour distilled water into 5000 ml or 2000 ml conical flasks, up to 4500 ml or 1800 ml respectively</LI><LI>Cover the tops with foil and tape</LI><LI>Boil 2L of water to 100 degrees, in the autoclave.</LI><LI>Spray with antibacterial spray, and set up trays for growth</LI><LI>Pour into container</LI><LI>Add green tea and allow to brew for 15 minutes, 1 tea bag per liter of autoclaved water</LI><LI>Remove tea bags</LI><LI>Add sugar, 100 g per liter</LI><LI>Stir until dissolved</LI><LI>Cool down to below 30°C</LI><LI>Add organic cider vinegar 100 ml per liter of green tea solution</LI><LI>Open bag of living Kombucha culture, take pellicle, hold with blue roll</LI><LI>Add one piece of Kombucha culture pellicle by cutting a chunk with scissors. It will sink to the bottom of the </LI><LI>container. Be careful not to add any liquid Kombucha media.</LI><LI>Cover growth container with blue roll, be careful not to lean over the trays and keep sterile conditions.</LI><LI>Fermentation starts after 48-72h, thin skin and bubbles will be produced and culture will come back out on the surface</LI><LI>When product becomes about 2 cm thick, take it out</LI><LI>Wash with soaped water</LI><LI>Let dry</LI></OL></DIV></DIV></DIV><H3><EM>G. xylinus</EM> HS media and culturing </H3><DIV><DIV class="pure-g"><DIV class="pure-u-1-2"><UL><LI>Materials
                                                </LI><LI>For 500ml media:
                                                        </LI><LI>10g glucose (2%w/v)</LI><LI>2.5g yeast extract (0.5% w/v)</LI><LI>2.5g peptone (0.5% w/v)</LI><LI>1.35g Na2HPO4 (0.27% w/v)</LI><LI>0.75g citric acid (0.15% w/v)</LI><LI>500ml distilled H20</LI><LI>0.5mL cellulase if HS+cellulase media required</LI><LI>7.5g of agar (agar-agar) if making HS-agar plates</LI><LI>(antibiotics as necessary)</LI></UL></DIV><DIV class="pure-u-1-2"><OL><LI>Add 250ml dH20 to glucose in one bottle and 250ml dH20 to the rest in a second bottle. In incompletely distilled water, glucose will form a solid mass, so stir vigorously immediately after adding water. Autoclave both bottles to sterilize media and pour glucose solution in sterile conditions (next to a Bunsen burner or in a flow hood) into the second bottle. Autoclaving glucose separately from amino acids avoids Maillard reaction, which can result in the formation of toxic byproducts in the media.</LI><LI>Streak/inoculate Gluconacetobacter onto plates or into media.</LI><LI>Incubate plates at 30°C inverted. Colonies will appear in 48-72 hours.</LI><LI>Incubate liquid HS-cultures at 30 °C standing. Standing culturing results in low growth rate, but avoids the formation of cellulose non-producing mutants, which have been reported to appear in shaking conditions.</LI><LI>For quick growth, grow with shaking at 180rpm, 30C, with the addition of cellulase. NB! This selects for cellulose non-producing mutants. They can be identified as smooth colonies on plates (cellulose producing colonies have a rough colony morphology - see section <EM>Gluconacetobacter</EM> for images).</LI></OL></DIV></DIV></DIV><H3>Preparing electrocompetent <EM>G. xylinus</EM> cells</H3><DIV><DIV class="pure-g"><DIV class="pure-u-1-2"><UL><LI>Materials
                                                </LI><LI>HS+cellulase media</LI><LI>Ice bucket and ice</LI><LI>Temperature controlled centrifuge</LI><LI>1mM HEPES (ph7.0) 80mL</LI><LI>15% glycerol</LI><LI>50 mL tubes</LI><LI>P1000 pipette and tips</LI><LI>Stripettes and automatic pipette for larger volumes</LI><LI>Shaker at 30 °C, 180rpm</LI></UL></DIV><DIV class="pure-u-1-2"><OL><LI>If the goal of transformation is to produce cellulose-producing transformed G.xylinus, use regular HS media for culturing. If cellulose production after transformation is not primary and speed is required, HS-cellulase medium can be used. Usage of HS-cellulase medium during growth results in the formation of a higher number of cellulose negative mutants, but results in much higher transformation efficiencies, and requires less time. Even with HS-cellulose medium, cellulose producing colonies can be identified on the plate after transformation, as cellulose-producing colonies differ in morphology from cellulose non-producing colonies (see section <EM>Gluconacetobacter</EM> for images of colony morphology).</LI><LI>Inoculate 5ml of HS or HS+cellulase medium with <EM>Gluconacetobacter</EM>.
                                            </LI><LI>Incubate at 30°C, 180 rpm shaking overnight.</LI><LI>Next day, pour 30mL of HS or 15ml of HS+cellulase medium into each of four 50mL tubes. If using regular HS medium, vortex the tubes for 3 minutes to release cells from the cellulose pellicle</LI><LI>To each tube, add 1mL of overnight culture and incubate with shaking at 180rpm, 30C.</LI><LI>Incubate overnight or until OD600 of around 0.4-0.7 is reached. When using HS-cellulase media, OD600 can reach up to 0.6 -0.7, however when using regular HS medium, OD600 measurement is disturbed by the cellulose pellicle, and can reach up to 0.2. Vortex tubes for 3 minutes before taking the measurements.</LI><LI>If using regular HS medium, add 62ul (0.2% v/v) Celluclast cellulase to each tube and incubate at 30C, 180rpm shaking for 2 hours, or until the cellulose pellicle is degraded to completely release the cells. This does not result in higher formation of cel- mutants, as short incubation time and nutrient depletion does not allow for proliferation of cel- mutants.</LI><LI>Before continuing, set up the necessary materials:
                                                </LI><LI>Pre-cool centrifuge to 4°C</LI><LI>Prepare ice bucket</LI><LI>Chill 1mM HEPES buffer and 15% glycerol buffer on ice</LI></OL><LI>Once the cultures reach desired OD600, take them out of incubation and put them on ice for 10 minutes (the tube should feel cool).</LI><LI>From here on, keep cells always cool, at or below 4°C</LI><LI>After cooling, spin the tubes in a refrigerated (4°C) centrifuge for 12min at 4100rpm.</LI><LI>Pour off supernatant carefully, taking care not to pour off the pellet. <EM>G. xylinus</EM> does not pellet as easily as E.coli, most likely due to the buffering effects of cellulose. If the pellet is not attached to the wall after centrifugation, smear the pellet onto the wall of the tube and centrifuge again using longer centrifugation times. Re-suspend bacteria in 10mL HEPES: re-suspend first using 1ml HEPES and a P1000 pipette, then add 9ml of HEPES using a stripette; it is much easier to re-suspend the pellet fully using a P1000. Do not use a vortexer.
                                            </LI><LI>Pool the samples into a single 50ml tube.</LI><LI>Centrifuge again for 14 minutes at 4100 rpm and 4 °C temp. Use a balancer tube with 40ml water.</LI><LI>Pour off supernatant again, re-suspend pellet in 10mL ice-cold HEPES on ice as before.</LI><LI>Centrifuge again for 14 minutes at 4100 rpm, 4 °C.</LI><LI>Pour off supernatant and re-suspend pellet in 4mL ice cold 15% glycerol solution.
                                            </LI><LI>Pipette 50ul aliquots into tubes. Label tubes properly.</LI><LI>Store samples on ice for immediate use or freeze 50ul aliquots in-80°C. According to some reports, the efficiency of electrocompetent cells reduces after each freezing, so immediate use may result in highest efficiencies.
                                            </LI></DIV></DIV></DIV><H3>gDNA extraction from <EM>G.xylinus</EM></H3><DIV><P>This protocol uses the commercially available Qiagen DNA Blood and Tissue kit for gDNA extraction. We have modified it to be amenable for G. xylinus, and deal with the issue of co-purifying contaminants, in order to attain high-quality DNA. If DNA quality is not primary, the original Qiagen protocol may be used. For detailed user-guide and list of required reagents, see Qiagen DNA Blood and Tissue handbook. </P><DIV class="pure-g"><DIV class="pure-u-1-2"><UL><LI>Materials
                                                </LI><LI>HS+cellulase media</LI><LI>Ice bucket and ice</LI><LI>Temperature controlled centrifuge</LI><LI>Qiagen DNA Blood and Tissue Kit</LI><LI>Zymo DNA Clean and Concentrator 5 kit</LI><LI>0.2um filter Millipore filter discs</LI><LI>50ml tubes</LI><LI>1.5ml tubes</LI><LI>Shaker at 30 °C, 180rpm</LI></UL></DIV><DIV class="pure-u-1-2"><OL><LI>Inoculate 4x5ml of HS or HS-cellulase media in 50ml tubes with Gluconacetobacter. I  </LI><LI>Incubate at 30°C, 180 rpm shaking for 48 hours.</LI><LI>Before continuing, set up the necessary materials:
                                                </LI><LI>Pre-cool centrifuge to 4°C</LI><LI>Prepare ice bucket</LI><LI>Chill HS medium on ice</LI></OL><LI>Centrifuge 50ml tubes at 4°C, maximum speed for 10 minutes</LI><LI>Discard the supernatant and resuspend cells in 5ml of cold HS.</LI><LI>Centrifuge and discard the supernatant again as previously</LI><LI>Continue with Qiagen DNA Blood and Tissue kit according to the provided handbook until re-suspension with buffer AL.</LI><LI>After re-suspension, pool the samples by pipetting the mixture onto a spin column, centrifuging, and pipetting and centrifuging the following mixtures onto the same spin column. Alternatively, different columns can be used, however this results in lower concentrations of DNA in the final preparation.
                                            </LI><LI>Continue with DNA Blood and Tissue kit protocol until the elution step.</LI><LI>Pre-heat nuclease-free H20 to approximately 50°C by quick microwaving. Pipet 100ul of heated H20 onto the spin column. Wait 15 minutes before centrifugation </LI><LI>Repeat the elution step as previously described, elute DNA into a new 1.5ml tube.</LI><LI>If high-purity DNA is required, continue with the following steps:</LI><LI>Place a 0.2um filter onto distilled water in a Petri dish. Pipet 30ul of extracted DNA onto the filter</LI><LI>Wait for 30 minutes to dialyze</LI><LI>Pipet DNA into a new tube</LI><LI>Use the Zymo Clean and Concentrator kit to further purify the dialyzed DNA, by following the manufacturers instructions until the elution step </LI><LI>Elute using 25ul of pre-heated water, wait for 15 minutes before centrifugation. If long-term storage is required, use the elution buffer recommended by the manufacturer. </LI><LI>Store the purified DNA at-20°C. </LI></DIV></DIV></DIV><H3>Transformation of <EM>G.xylinus</EM> using electroporation
</H3><DIV><DIV class="pure-g"><DIV class="pure-u-1-2"><UL><LI>Prepare the following
                                                </LI><LI>Plasmid DNA</LI><LI>Electrocompetent <EM> G.xylinus cells</EM> (in 50µl or 100µl aliquots)</LI><LI>Ice bucket and ice</LI><LI>1mm path electrocuvettes</LI><LI>1.5ml microcentrifuge tubes or PCR tubes</LI><LI>An electroporator - set at 2.5kV and 5.9ms</LI><LI>HS+cellulase medium</LI><LI>HS agar plates with appropriate antibiotic</LI><LI>One aliquot of electrocompetent cells for positive control and one aliquot for negative control.</LI><LI>If a plasmid known to work in G. xylinus exists, use this as positive control .</LI></UL></DIV><DIV class="pure-u-1-2"><OL><LI>Set up the electroporator with correct settings: at 2.5kV, 6-8ms, 400ohm resistance, 25microF capacitance.</LI><LI>Prepare 800µL HS+cellulase media containing 1.5ml culture tubes and pre-heat to 30 °C. Prepare SOC medium at 37°C. Prepare a space for shaking.</LI><LI>Prepare ice bucket, place plasmid DNA and electrocuvettes on ice and thaw electrocompetent cells on ice. <B>NB! Make sure DNA is desalinated before use – ionic solution can otherwise cause arcing.</B></LI><LI>Add <B>2 µL</B> of plasmid DNA to 100 µL of concentrated cells in a cold microcentrifuge or PCR tube and mix well by pipetting. Add plasmid DNA also to one aliquot of electrocompetent E.coli cells (positive control; alternatively also add another known plasmid as pos. control).<B>NB! Do not add plasmid DNA to one aliquot of electrocompetent G. xylinus (negative control)</B></LI><LI>Transfer the cell/DNA mixture and positive and negative controls to a cold 0.2-cm electroporation cuvette. Dry any water condensate outside of the cuvette (using labcoat), place the cuvette into the electroporator, and apply the pulse.
                                            </LI><LI>Transfer the pulsed cells into 800 µL of HS+cellulase medium in a culture tube. Transfer E. coli into SOC medium. <B>NB! Prepare everything beforehand and do it quickly.</B></LI><LI>Incubate the culture tubes with shaking (170 rpm) at 30 °C for 4 hours or overnight.</LI><LI>When culturing for 4 hours, spin the culture at max available RPM for 10 minutes, resuspend in 200ul and plate on an HS-agar plate with an appropriate antibiotic concentration. If overnight incubation is required, use a 100ul aliquot of the culture for plating, as you will otherwise find a lawn. (The antibiotic concentration of plates is important, be certain to use the antibiotic concentration specific for the G.xylinus strain (see G.xylinus))</LI><LI>Grow plates at 30°C inverted, colonies will appear in 24-48 hours </LI></OL></DIV></DIV></DIV><H3>Quantification of cellulose production</H3><DIV><OL><LI>Add 50ml of HS medium (or other medium of choice) to 250ml conical flask</LI><LI>Grow G.xylinus in HS medium for 7 days standing, at 30C. Don’t seal the flasks hermetically in order to allow diffusion of oxygen (seal using foam buns)</LI><LI>After 7 days of growth, wash the cellulose twice with distilled water</LI><LI>Add 50ml of 0.1M NaOH to cellulose, incubate at 65C for 4 hours</LI><LI>Wash the cellulose twice using distilled water</LI><LI>Place the formed cellulose pellicle on baking paper and dry the pellicle at 65 degrees for 4 hours-overnight. Before drying, cut out and measure the weight of a piece of baking paper, and dry the pellicle together with the paper. This is because the pellicle will invariably stick to the surface, and removal of it results in loss of cellulose.</LI><LI>Place the pellicle+paper into a vacuum desiccator for 2 hours</LI><LI>Weigh the pellicle+paper using a high-sensitivity scale. Subtract the weight of the paper to determine the weight of cellulose.</LI></OL></DIV><H3>gDNA Library preparation for genome sequencing using Illumina Nextera kit  </H3><DIV><DIV class="pure-g"><DIV class="pure-u-1-1"><P>This is a modified protocol for using Illumina Nextera kit for preparation of gDNA library for genome sequencing. We have modified the original Illumina protocol to be amenable for low sample number and have a lower cost. We used this protocol to prepare gDNA libraries of G. xylinus ATCC 53582 strain and the G. xylinus strain we isolated from Kombucha tea at the beginning of this summer. For the full protocol and required materials, see Illumina Nextera Kit user guide.</P><H4>Preparation</H4><OL><LI>Prepare 200ul PCR tubes, ice bucket, ice</LI><LI>Remove the TD, TDE1, and genomic DNA from -15° to -25°C storage and thaw on ice.</LI><LI>Remove RSB from -15° to -25C storage and thaw at room temperature.
                                            </LI><LI>After thawing, ensure all reagents are adequately mixed by gently inverting the tubes 3–5 times, followed by a brief spin in a microcentrifuge. NOTE: Ensure the reaction is assembled in the order described for optimal performance. Some sources recommend setting up the reaction on ice, as transposons may exhibit a low activity at room temperature</LI><LI>Label the PCR tubes with a smudge resistant pen - important, as labels tend to degrade in a thermocycler.
                                            </LI><LI>Add 20 μl of genomic DNA at 2.5 ng/μl (50 ng total) to each PCR tube
                                            </LI><LI>Add 25 μl of TD Buffer to the wells containing genomic DNA. Change tips between samples. NOTE:Calculate the total volume of TD for all reactions, and divide among an appropriate number of tubes in an 8-well PCR strip tube.</LI><LI>Add 5 μl of TDE1 to the tubes containing genomic DNA</LI><LI>Gently pipette up and down 10 times to mix the reaction.
                                            </LI><LI>Centrifuge at 280g for 1 minute
                                            </LI><LI>Place the PCR tubes into the thermocycler and run the following program:
                                                </LI><LI>Heated lid on</LI><LI>55°C for 5 minutes</LI><LI>Hold at 10°C</LI></OL><LI>While tagmentation is ongoing, label 1.5ml tubes accordingly for the next step and add 180ul of Zymo DNA binding buffer</LI><LI>Also, remove NPM, PPC, and the index primers from -15° to -25°C storage and thaw on a bench at room temperature.</LI><LI>Allow approximately 20 minutes to thaw NPM, PPC, and index primers
                                            </LI><LI>After tagmentation reaction has finished, proceed immediately to Zymo cleanup - transport samples on ice and do not wait before proceeding to Zymo cleanup, as transposons remain active at lower temperatures, albeit at low level, which may result in overtagmentation and fragments shorter than optimal.</LI><H4>Zymo cleanup protocol</H4><OL><LI>Transfer 50 μl of tagmentation reaction into 1.5ml tubes containing Zymo binding buffer</LI><LI>Gently pipette up and down 10x to mix well</LI><LI>Transfer the solution to Zymo spin columns</LI><LI>Centrifuge at 10000rpm for 1 minute.</LI><LI>Discard the flow-through or the collecting tube and place the column into a new collection tube</LI><LI>Wash the Zymo spin column twice by:
                                                </LI><LI>Pipette 300ul of Zymo wash buffer to the tubes</LI><LI>Centrifuge at 10000rpm for 1 minute, discarding the flow through</LI><LI>Repeat the wash step for a total number of 2 washes.</LI><LI>Centrifuge at 10000rpm 1 min to ensure no wash buffer remains.</LI><LI>Add 25 μl of RSB directly to the column matrix in each well. Confirm visually that RSB has absorbed onto the matrix, and is not on the side of the tube. If it is, flick the tube gently to force the liquid onto the matrix</LI><LI>Incubate the tubes for 5 minutes at room temperature. If time is not a limiting factor, increase incubation time to 15 minutes, as this may result in higher elution effieciencies</LI><LI>Centrifuge the tubes at 10000rpm for 1 minute.</LI><LI>Place the tubes on ice until proceeding to PCR</LI></OL><H4>PCR</H4><OL><LI>After NPM, PPC, and Index primers are completely thawed, gently invert each tube 3–5 times to mix and briefly centrifuge the tubes in a microcentrifuge. Use 1.7 ml Eppendorf tubes as adapters for the microcentrifuge.</LI><LI>Label PCR tubes according to your samples</LI><LI>Add 5 μl of index 2 primers (white caps) to each tube respectively.</LI><LI>Add 5 μl of index 1 primers (orange caps) to each tube respectively.</LI><LI>Add 15 μl of NPM (Nextera PCR master mix) to each well of the NAP1 plate containing index primers.</LI><LI>Add 5 μl PPC (PCR primer cocktail) to each well containing index primers and NPM.</LI><LI>Transfer 20 μl of purified tagmented DNA to the corresponding PCR tube</LI><LI>Gently pipette up and down 3–5 times to thoroughly combine the DNA with the PCR mix.</LI><LI>Spin down the tubes with quick centrifugation .</LI><LI>Ensure that the thermocycler lid is heated during the incubation.Run the PCR using the following program:
                                                </LI><LI>72°C for 3 minutes</LI><LI>98°C for 30 seconds</LI><LI>5 cycles of:
                                                        </LI><LI>98°C for 10 seconds</LI><LI>63°C for 30 seconds</LI><LI>72°C for 3 minutes</LI></OL><LI>Hold at 10°C</LI><LI>Ensure that the 72°C step preceeds the rest of the program.
                                            </LI><LI>SAFE STOPPING POINT – can store the DNA in thermocycler overnight, or at 2-8°C for 2 days
                                            </LI><H4>Clean-up of PCR using AMPure beads</H4><OL><LI>Bring the AMPure XP beads to room temperature.</LI><LI>Prepare fresh 80% ethanol from absolute ethanol. (Always prepare fresh 80% ethanol for wash steps. Ethanol can absorb water from the air impacting your results.)</LI><LI>xCentrifuge the PCR tubes containing the limited cycle PCR product at quickly to spin down the liquid.</LI><LI>Label new 1.5ml tubes according to the samples for purification steps</LI><LI>Transfer 50 μl of the PCR product from the PCR tubes into new 1.5ml tubes. Change tips between samples.</LI><LI>Vortex the AMPure XP beads for 30 seconds to ensure that the beads are evenly dispersed.</LI><LI>Add 30 μl of AMPure XP beads to each tube containing the PCR product. For 2x250 runs on the MiSeq, add 25 μl of AMPure XP beads to each tube.</LI><LI>Gently pipette mix up and down (gently, so as not to introduce bubbles) until solution is homogeneous. Make sure to pipette, no to vortex the solution, as this results in liquid collecting under the tube cap. This can't be removed easily as centrifugation also results in sedimentation of AMPure beads.</LI><LI>Incubate the tubes (containing AMPure beads and PCR product) at room temperature without shaking for 5 minutes.</LI><LI>Place the plate on a magnetic stand for 2 minutes or until the supernatant has cleared.</LI><LI>With the tubes on the magnetic stand, carefully remove and discard the supernatant. Pieptte carefully, as AMPure beads may follow the surface of the liquid. If any beads are inadvertently aspirated into the tips, dispense the beads back to the plate and let the plate rest on the magnet for 2 minutes and confirm that the supernatant has cleared.</LI><LI>With the tubes on the magnetic stand, wash the beads with freshly prepared 80% ethanol as follows:
                                                </LI><LI>Add 200 μl of freshly prepared 80% ethanol to each sample well. You should not resuspend the beads at this time.</LI><LI>Incubate the plate on the magnetic stand for 30 seconds or until the supernatant appears clear.</LI><LI>Carefully remove and discard the supernatant.</LI></OL><LI>With the tubes on the magnetic stand, perform a second ethanol wash as follows:

                                                </LI><LI>Add 200 μl of freshly prepared 80% ethanol to each sample.</LI><LI>Incubate the plate on the magnetic stand for 30 seconds or until the supernatant appears clear.</LI><LI>Carefully remove and discard the supernatant. Use a P20 multichannel pipette with fine pipette tips to remove excess ethanol.</LI><LI>With the tubes still on the magnetic stand, allow the beads to air-dry for 15 minutes.
                                            </LI><LI>Remove the tubes from the magnetic stand. Add 32.5 μl of RSB to each well of the NAP2 plate.
                                            </LI><LI>Gently pipette mix up and down until the solution is homogenous, changing tips after each column.
                                            </LI><LI>Incubate at room temperature for 2 minutes.
                                            </LI><LI>Place the tubes on the magnetic stand for 2 minutes or until the supernatant has cleared.</LI><LI>Label new 1.5ml tubes accordingly</LI><LI>Carefully transfer 30 μl of the supernatant into the new tubes. These will contain your purified library.</LI><LI>Store the library in -20C until further processing.</LI></DIV></DIV></DIV></DIV></SECTION><SECTION id="general"><H2>General/<EM>E. coli</EM> Protocols</H2><DIV class="accordion"><H3>General Cloning Workflow</H3><DIV><OL><LI>PCR was often used to generate the insert fragments, for example to add on RFC25 prefix and suffixes to RFC10 parts, or to clone out a desired coding sequence for a new biobrick. In each case primers were designed with overhangs containing the desired prefix and suffix sequences. PCR reactions were set up using Phusion or Q5 (NEB) polymerases, with reactions set up as per manufacturer protocol, with replicates for each construct. PCR programmes varied but typically utilised a touchdown protocol which improves accuracy of primer binding at the beginning of the reaction due to the higher temperatures. ~10% PCR product was ran on agarose gel to confirm it's success, then replicates were pooled and PCR purified (QIAgen kit) </LI><LI>Insert and Vector backbone DNA was digested with the appropriate restriction enzymes, using reaction set-up as recommended in manufacturer protocol</LI><LI>Products were Gel-extracted or PCR purified</LI><LI>Vector backbone was de-phosphorylated to prevent background re-ligation, setting up reaction as per manufacturers recommendation, then heat-kill the enzyme</LI><LI>Optional: PCR purify to remove enzyme and buffer</LI><LI>Ligation reactions for the desired combinations of insert and backbone were set up, including negative control reactions which contain no insert DNA. Reactions components and amounts determined as per the manufacturer recommendation, though halving the total reaction was sometimes carried out</LI><LI>Transformed with ~2-4 ul ligation reaction into DH10B or NEB-5-alpha cells using the general heat shock protocol as explained previously</LI><LI>Picked colonies and innoculated mini-prep cultures. The number picked for each plate depends on the difference between the positive and negative controls. Generally between 2 and 4 were picked. Mini-preps were carried out using the manufacturers protocol (QIAgen QIAspin) </LI><LI>Test digest (no more than 10 ul total reaction volume) performed and products analysed using agarose gel electrophoresis to confirm if correct construct was present. </LI><LI>Alternativley, instead of picking cultures for mini-prepping, colony PCR was used when large numbers of colonies needed to be screened. Then only the positive clones were mini-prepped. </LI><LI>Positive clones were sent for sequencing (Source Bioscience) of the insert using appropriate primers. </LI></OL></DIV><H3>LB Broth Preparation</H3><DIV><OL><LI>Add 25g Luria Broth to 1L demineralised water</LI><LI>Autoclave</LI></OL></DIV><H3>LB Agar Preparation</H3><DIV><OL><LI>Add 25g Luria Broth and 15g Agar to 1L demineralised water</LI><LI>Autoclave</LI></OL></DIV><H3>Preparation of chemically competent E. coli cells</H3><DIV><P> The basis for the protocol is from http://openwetware.org/wiki/TOP10_chemically_competent_cells with a few differences. This also contains the recipe for the CCMB80 buffer used. The protocol is summarised below:
                                <OL><LI>Inoculate 2 ml LB broth with an aliquot (~50 ul) of the desired <I>E. coli</I> from the -80degC freezer stock of cells. 
                                    </LI><LI>Incubate for 2h at 37°C</LI><LI>Add the 2 ml seed culture to 250 ml LB broth and grow at 37degC, shaking (~200 rpm) until OD 600 of 0.3 (~5 hours)
                                    </LI><LI>Centrifuge at 4degC, (in our case 3000 rpm in Heraeus megafuge, Thermo) for 10 minutes</LI><LI>Discard supernatant, then resuspend in 80 ml ice cold CCMB80 buffer (it is easier to resuspend in 1 ml first using a Gilson pipette, then add buffer to the required volume)
     </LI><LI>Place in ice for 20 minutes
                                    </LI><LI>Centrifuge 4degC and discard supernatant</LI><LI>Resuspend in 10 ml CCMB80 buffer
                                    </LI><LI>Test OD 600 of 200 ul SOC media with 50 ul resuspended cells and based on this calculate the amount of CCMB80 buffer needed to add to the resuspended cells to achieve a final yield of OD 600 1.0-1.5.
                                    </LI><LI>Aliquot in volumes as desired (for us ~250 ul) then store at -80 degC</LI></OL></P></DIV><H3>Preparation of electro-competent E. coli cells</H3><DIV><P> To autoclave </P><UL><LI>500 mL LB</LI><LI>500 mL Water</LI><LI>50 mL 10% glycerol </LI><LI>Conical Flasks</LI></UL><P>To grow:5 mL overnight culture containing the required antibiotic, grow under shaking conditions at 37 degC </P><OL><LI>Prepare Eppendorf tubes and keep in the -80 degC freezer until required</LI><LI>Inoculate the autoclaved flasks with 50 mL LB</LI><LI>Add 500 ul of overnight culture into 50 the conical flasks and provide specific antibiotic, if required</LI><LI>Grow for ~1 h and then start taking OD 600 nm readings every half hour. When OD reaches 0.5, proceed to the next step. </LI><LI>Pour culture into falcon tube</LI><LI>Centrifuge for 10 minutes at 4000 rpm and at 4 degC </LI><LI>Discard supernatant and use blue roll remove any left overs. </LI><LI>Add 800 ul of previously chilled, autoclaved water, resuspend cells, then add 9.2 mL to make it up to 10 mL</LI><LI>Centrifuge for 10 minutes at 4000 rpm and at 4 degC </LI></OL></DIV><H3>General Heat-Shock Transformation</H3><DIV><OL><LI>Add DNA to 50 ul cells on ice (no more than 5 ul, i.e. no more than 10% volume of cells)</LI><LI>Incubate on ice 15-30 min</LI><LI>Heat shock 42degC 54 s</LI><LI>Place samples back on ice for 2 minutes</LI><LI>Add 200 ul LB broth, or up to 10x volume of the cells</LI><LI>Incubate at 37°C for 60 minutes, shaking</LI><LI>Optional: Spin down cells, discard supernatant and resuspend in 100-200 ul LB to concentrate</LI><LI>Plate out cells on LB agar, maximum 200 ul</LI><LI>Incubate at 37°C overnight.</LI></OL></DIV><H3>80% Glycerol Preparation</H3><DIV><OL><LI>Add 80ml 99.7% glycerol to 20ml demineralized water</LI><LI>Autoclave</LI></OL></DIV><H3>Glycerol Stock Preparation</H3><DIV><OL><LI>Cultures plated on LB Agar + antibiotic and grown at 37°C overnight.</LI><LI>A 5ml LB culture in LB+antibiotic inoculated from a single, freshly growing colony.</LI><LI>Cultivate for 16h at 37°C, with constant shaking</LI><LI>0.5ml of this culture inoculated into sterile vial</LI><LI>Add 0.5ml of 80% glycerol</LI><LI>Vortex</LI><LI>Spin down</LI><LI>Freeze them at -80 degrees</LI></OL></DIV><H3>QIAprep Spin Miniprep Kit</H3><DIV><DIV class="pure-g"><DIV class="pure-u-1-2"><UL><LI>Materials per sample
                                                </LI><LI>250ul P1 buffer (suspension buffer)</LI><LI>250ul P2 buffer (Lysis buffer)</LI><LI>350ul N3 buffer</LI><LI>750ul PE buffer</LI><LI>500ul PE buffer</LI><LI>Columns</LI></UL></DIV><DIV class="pure-u-1-2"><OL><LI>Spin cells down at 4000rpm for 10 minutes</LI><LI>Discard supernatant (LB)</LI><LI>Resuspend pellet in P1 buffer</LI><LI>Transfer to labeled Eppendorf tube</LI><LI>Add P2 buffer. Solution should turn blue</LI><LI>Invert tubes 4-6 times, then wait for 2 minutes</LI><LI>Stop the reaction by adding N3 buffer and immediately inverting 4-6 times. Solution should turn clear</LI><LI>Centrifuge at 13000 rpm for 10 minutesv
                                                </LI><LI>Decant/pipette supernatant into mini-prep columns. Discard flow-through</LI><LI>Wash with PE buffer (750ul)</LI><LI>Centrifuge at 13000 rpm for 1 minutev
                                                    </LI><LI>Discard flow-through. Add second wash of PE buffer (500ul)v
                                                        </LI><LI>Centrifuge at 13000 rpm for 1 minute</LI><LI>Discard flow-through</LI><LI>Centrifuge empty columns at 13000 rpm for 1 minute to eliminate any excess wash buffer</LI><LI>Discard flow-through</LI><LI>Move columns into a labelled eppendorf</LI><LI>Add 30-40ul distilled water and wait for 2-3minutes</LI><LI>Elute DNA by centrifuging at 13000 rpm for 1 minute, do not discard flow-through. Discard column.</LI><LI>Nanodrop</LI></OL></DIV></DIV></DIV><H3>1% Agarose Gel</H3><DIV><DIV class="pure-g"><DIV class="pure-u-1-2"><UL><LI>Materials
                                                </LI><LI>1g Agarose</LI><LI>100mL 1X TAE buffer</LI><LI>8uL SYBR Safe</LI></UL></DIV><DIV class="pure-u-1-2"><OL><LI>Mix Agarose and 1x TAE buffer</LI><LI>Heat up until Agarose is dissolved</LI><LI>Add SYBR Safe</LI><LI>Pour into gel tray and let cool</LI></OL></DIV></DIV></DIV><H3>Agarose Gel Electrophoresis</H3><DIV><DIV class="pure-g"><DIV class="pure-u-1-2"><UL><LI>Materials
                                                </LI><LI>1% Agarose gel DNA ladder</LI><LI>6x loading dye</LI><LI>Electrophoresis cuvette</LI></UL></DIV><DIV class="pure-u-1-2"><OL><LI>Set gel tray into cuvette, filled with 1x TAE buffer</LI><LI>Inoculate samples, previously dyed with 6x loading dye. Additionally, provided a DNA ladder for further reference of DNA sizes</LI><LI>Run gel at 110V for 30-40min</LI></OL></DIV></DIV></DIV><H3>Overnight Cell Incubation</H3><DIV><DIV class="pure-g"><DIV class="pure-u-1-2"><UL><LI>Materials
                                                </LI><LI>5mL Luria Broth</LI><LI>5ul specific antibiotic</LI><LI>Loops (for colony picking or glycerol stock scraping)</LI></UL></DIV><DIV class="pure-u-1-2"><OL><LI>Add Luria Broth into 50mL tube</LI><LI>Inoculate specific antibiotic</LI><LI>Scrape/pick glycerol stock surface/colony and transfer into falcon tube</LI><LI>Incubate at 37°C overnight</LI></OL></DIV></DIV></DIV><H3>1x TAE buffer</H3><DIV><UL><LI>1x solution contas 40nM Tris, 20mM acetic acid, 1mM EDTA</LI></UL></DIV></DIV></SECTION><SECTION id="functionalisation"><H2>Functionalisation Protocols</H2><DIV class="accordion"><H3>Metal binding assay protocol</H3><DIV><DIV class="pure-g"><DIV class="pure-u-1-1"><P>Metal binding proteins fused to CBDs were bound onto cellulose in 96-well plates and tested against 3 different metals (Nickel, copper, zinc). First, the fusion protein lysate was incubated overnight in the cellulose wells. Following this, the metal salt solutions are added in excess into the wells. Finally, an EDTA step removes the bound metal ions into solution, and the metal concentration in solution is quantified by mass spectrometer. Multiple washes with PBS and water were done between each binding step, ensuring that metal ions read were released from the metal binding proteins.
                                        </P></DIV><DIV class="pure-u-1-2"><UL><LI>Materials
                                                </LI><LI>Cellulose 96-well plate (see ‘Preparing cellulose 96-well plates’) </LI><LI>20mM Zinc chloride/Nickel chloride/Copper chloride solution</LI><LI>1x PBS</LI><LI>Sterile water</LI><LI>Protein(s) of interest in cell lysate(s) (suspended in PBS, see ‘Protein preparation for assays’) or purified protein</LI><LI>Mass spectrometer</LI><LI>Multichannel pipette</LI><LI>25mM EDTA</LI><LI>2ml centrifuge tube</LI></UL></DIV><DIV class="pure-u-1-2"><UL><LI>Methods
                                                </LI><LI>As we are using mass spectrometry to detect metal bound and eluted, samples must be of 2ml volume. We pool the protein-releasing EDTA solution from 4 wells to make one reading. The remaining volume is made up with water. This should be accounted for when planning repeats and negatives.</LI><LI>Apply 200ul cell lysate/purified protein sample in each well on the 96-well plate, accounting for negative controls (wells with no lysate applied).</LI><LI>Incubate overnight (9 to 14 hours) at 4°C.</LI><LI>Remove the 180ul of cell lysate.</LI><LI>Wash thrice with PBS: apply 180ul of 1x PBS into each well. Aspirate 180ul from well and discard. Repeat twice more.</LI><LI>Wash twice with water: apply 180ul of water into each well. Aspirate 180ul from well and discard. Repeat once more.</LI><LI>Apply 180ul of 20mM metal solution.</LI><LI>Incubate for 1 hour at room temperature.</LI><LI>Remove the 180ul of metal solution.</LI><LI>Wash thrice with water: apply 180ul of water into each well. Aspirate 180ul from well and discard.</LI><LI>Apply 180ul of EDTA.</LI><LI>Incubate for 30 minutes at room temperature.</LI><LI>Transfer the liquids in the wells into a 2ml centrifuge tube, pooling the wells as planned. The sample is ready to read with a mass spectrometer</LI></UL></DIV></DIV></DIV><H3>Preparing Cellulose 96-well Plates
</H3><DIV><DIV class="pure-g"><DIV class="pure-u-1-1"><P>40 g of kombucha cellulose was blended with 250 ml sterile water in a conventional kitchen blender. 200 ul of this blend is added into each well of a black 96-well plate. The cellulose is dried in a 37°C incubator overnight until dry and ready to use. 

                                        </P></DIV><DIV class="pure-u-1-2"><UL><LI>Materials
                                                </LI><LI>Black 96-well plate (suitable for fluorescence reading)
</LI><LI>40 g Bacterial cellulose, grown from kombucha isolate (choose an even pellicle)</LI><LI>250 ml sterile water
</LI><LI>Conventional kitchen blender 
</LI><LI>Multichannel pipette 
</LI></UL></DIV><DIV class="pure-u-1-2"><UL><LI>Methods
                                                </LI><LI>Dab dry your bacterial cellulose. Weigh out 40 g. </LI><LI>Add weighed cellulose into blender and add 50 ml of the ddH20. </LI><LI>With a combination of pulse and short blends, try to eliminate large chunks.  Ensure you scoop chunks that are stuck below the blades. </LI><LI>Once the cellulose has begun to homogenise, add the remaining water and blend continuously for 12 mins at the highest setting. </LI><LI>Check that the cellulose is fully homogenised. If not, blend for a further 10 minutes. This blended cellulose mix keeps well indefinitely at room temperature in airtight glassware, using parafilm if needed. </LI><LI>Using a multichannel pipette, aliquot 200 ul of blended cellulose into each well of a black 96-well plate. It helps to mix between aliquots as settling often occurs. </LI><LI>Place in a 37°C incubator until fully dry, usually about 20 hours. Alternatively, place in 60°C oven and monitor for dryness between 3 to 8 hours. </LI><LI>Dry plates are ready for immediate use. </LI></UL></DIV></DIV></DIV><H3>Water filtration protocol</H3><DIV><DIV class="pure-g"><DIV class="pure-u-1-1"><P>Dried cellulose was exposed to the phytochelatin-CBD protein fusion (<A href="http://parts.igem.org/Part:BBa_K1321110">K1321110</A>) to create a smart cellulose filter with a chelating agent. The filter was set up on a coffee press and secured. Nickel chloride solution (250 µM) was added on the top and let to run through. The filtered liquid was collected and analysed using Mass Spectrometry and using the Nickel Assay Kit (Sigma).
                                        </P></DIV><DIV class="pure-u-1-2"><UL><LI>Materials
                                                </LI><LI>Coffee press </LI><LI>ATCC53582 <EM>G. xylinus</EM> produced cellulose </LI><LI>Phytochelatin-CBD in cell lysate</LI><LI>250 µM Nickel chloride solution in MilliQ water</LI><LI>Mass spectrometer</LI><LI>Nickel Assay Kit</LI></UL></DIV><DIV class="pure-u-1-2"><UL><LI>Methods
                                                </LI><LI><EM>G. xylinus</EM> (<A href="http://parts.igem.org/Part:BBa_K1321305">K1321305</A>) produced cellulose with yield treatment method was exposed to cell lysate containing phytochelatin-CBD (K1321110) and dried at 37 <SUP>O</SUP>C.</LI><LI>After the solution with the protein on the cellulose has dried, the cellulose was kept at 4 <SUP>O</SUP>C overnight.</LI><LI>The cellulose with the coated protein was gently soaked with milliQ water and secured on a coffee press, as a control, cellulose without coated protein was also secured on a separate coffee press.</LI><LI>3mL of 250 µM nickel chloride solution was added into each of the two coffee presses. Increased pressure was set up by the use of the press. The solution was let to filter for a sufficient time to collect at least 100 µl of the filtered solution.</LI></UL></DIV></DIV></DIV><H3>CBD binding strength assay, using fluorescence
</H3><DIV><DIV class="pure-g"><DIV class="pure-u-1-1"><P>Our selection of CBDs was cloned with sfGFP to make fluorescent fusion proteins that can be easily detected on a plate reader. We incubated the cell lysate on our cellulose plate overnight, and read the comparative reduction in fluorescence after each of the three washes with either water, ethanol, PBS or BSA.</P></DIV><DIV class="pure-u-1-2"><UL><LI>Materials
                                                </LI><LI>- Cellulose 96-well plate (see protocol ‘Preparing cellulose 96-well plates’)

</LI><LI>Cell lysates containing CBD-sfGFP fusions (see protocol ‘Preparation of crude cell lysate for fusion protein assays’) or purified protein sample</LI><LI>Fluorescence plate reader
</LI><LI>1 x PBS
</LI><LI>5% BSA
</LI><LI>Sterile water</LI><LI>70% ethanol</LI></UL></DIV><DIV class="pure-u-1-2"><UL><LI>Methods
                                                </LI><LI>Apply 200 ul cell lysate/purified protein sample in each well on the 96-well plate, accounting for negative controls (wells with no lysate applied or reading a blank cellulose plate). </LI><LI>Incubate overnight (9 to 14 hours) at 4°C. </LI><LI>Read fluorescence of plate with cell lysate still applied. </LI><LI>Remove the 180 ul of cell lysate. </LI><LI>Read fluorescence of plate with cell lysate removed.</LI><LI>First wash with 1 x PBS for all wells: apply 180u l of PBS into each well. Read fluorescence. Aspirate 180 ul from well and discard. Read fluorescence.</LI><LI>Optional: compare binding strength of CBDs when exposed to different washes. 4 different washes we’ve tried: water, 1 x PBS, 5% BSA, 70% ethanol. </LI><LI>Wash thrice with appropriate reagent: apply 180 ul of wash of choice into each well. Read fluorescence. Aspirate 180 ul from well and discard. Read fluorescence. Repeat twice more. </LI></UL></DIV></DIV></DIV><H3>Preparation of crude cell lysate for fusion protein assays (for constitutive expression)</H3><DIV><DIV class="pure-g"><DIV class="pure-u-1-1"><P>For LacI:
Protein-expressing cells were inoculated into 1litre of LB broth supplemented with antibiotic, and grown in a 37°C shaking incubator overnight until cloudy. The cells were pelleted, then resuspended in 10ml of PBS. The samples were then sonicated, and cell debris pelleted. The resulting clear cell lysate was used for assays and for protein purification. </P><P>For T7 fusions: 
Protein-expressing cells were inoculated into 1litre of LB broth supplemented with antibiotic, and grown in a 37°C shaking incubator to an OD600 between 0.5-1.0 and induced with IPTG. Cultures were returned to the incubator and grown overnight until cloudy. The cells were pelleted, then resuspended in 10ml of PBS. The samples were then sonicated, and cell debris pelleted. The resulting clear cell lysate was used for assays and for protein purification. </P></DIV><DIV class="pure-u-1-2"><UL><LI>Materials
                                                </LI><LI>Verified colony to pick</LI><LI>20 ml LB broth, for seed culture</LI><LI>2 litre conical flask</LI><LI>1 litre autoclaved LB broth, for overnight cell grow up</LI><LI>Foam bung</LI><LI> Aluminium foil</LI><LI>1000x chloramphenicol ( 1ml for each litre of LB)</LI><LI>37°C shaking incubator</LI><LI>Spectrophotometer set for OD<SUB>600</SUB>,/li&gt;
</LI><LI>0.1 M IPTG (for T7 induction)</LI><LI>1 litre capacity cooled centrifuge</LI><LI>1x PBS</LI><LI>50ml Falcon tubes</LI><LI>Stripette and pipette controller</LI><LI>Beaker, half-filled with ice</LI><LI>Sonicator</LI><LI>Cooled 50 ml Falcon centrifuge</LI></UL></DIV><DIV class="pure-u-1-2"><UL><LI>Methods
                                                </LI><LI>Pick your verified colony and inoculate into 20 ml LB broth, supplemented with 20 ul of 1000x chloramphenicol.</LI><LI>Grow for 6-10 hours, or until sufficiently cloudy in a 37°C shaking incubator. </LI><LI>Prepare 1 litre of autoclaved LB broth in a 2 litre conical flask, cover with foam bung, aluminium foil and label with autoclaved tape prior to autoclaving.</LI><LI>When cooled, add 1 ml of 1000x chloramphenicol to broth. Mix well.</LI><LI>Pour all of seed culture into prepared broth. Cover again with the same foam bung and aluminium foil. </LI><LI>If expression is constitutive, place in the shaking 37°C incubator (180 rpm) to grow overnight, between 12-20 hours. </LI><LI>If using T7 promoter, place in a shaking 37°C incubator (180 rpm) and check OD<SUB>600</SUB> periodically. Induce using 2ul of 0.1 M IPTG per ml, when OD<SUB>600</SUB> of culture is between 0.5 and 1.0. Return to the shaking 37°C incubator (180 rpm) to grow overnight, between 12-20 hours. </LI><LI>Check that growth has occurred and media is sufficiently cloudy. </LI><LI>Move culture into 1 litre centrifuge bottle. Balance bottles to the correct 0.1 g. </LI><LI>Centrifuge for 20 mins at 4°C, at the speed of 6000 rpm.</LI><LI>Check for pellet. If present, discard supernatant. </LI><LI>Using a stripette and pipette controller, resuspend in 10 ml 1x PBS, or volume of choice. Transfer to a 50 ml Falcon tube. </LI><LI>Using a beaker small enough to fit under the sonicator and half-filled in ice, embed the resuspended cells in a secure upright position within the ice.</LI><LI>Remove lid of Falcon tube. Submerge the sonicator tip in the resuspended cells, leaving about 0.5 cm from the bottom. It is important that the tip is not touching the bottom of the tube as this disturbs sonication. </LI><LI>Sonicate at 67% for 3:30 minutes in 15 second pulses. Post sonication, keep sample on ice. </LI><LI>Using a cooled 50 ml Falcon centrifuge, pellet down cell debris leaving cell lysate. 6000 rpm for 25 minutes at 4°C is sufficient. Check that lysate is clear and that pellet is present.</LI><LI>Transfer lysate to a new tube. Cell lysate is ready to use, for assay or further purification purposes. Discard cell debris pellet.</LI><LI>Store at 4°C if not using immediately. </LI></UL></DIV></DIV></DIV></DIV></SECTION></DIV></DIV></DIV><DIV class="printfooter">
Retrieved from &quot;<A href="http://2014.igem.org/Team:Imperial/Protocols">http://2014.igem.org/Team:Imperial/Protocols</A>&quot;</DIV></DIV></DIV><DIV id="footer-box" class="noprint"><DIV id="footer"><UL id="f-list"><LI id="t-recentchanges"><A href="/Special:RecentChanges" title="Recent changes">Recent changes</A></LI><LI id="t-whatlinkshere"><A href="/Special:WhatLinksHere/Team:Imperial/Protocols" title="List of all wiki pages that link here [j]" accesskey="j">What links here</A></LI><LI id="t-recentchangeslinked"><A href="/Special:RecentChangesLinked/Team:Imperial/Protocols" title="Recent changes in pages linked from this page [k]" accesskey="k">Related changes</A></LI><LI id="t-specialpages"><A href="/Special:SpecialPages" title="List of all special pages [q]" accesskey="q">Special pages</A></LI><LI><A href="/Special:Preferences">My preferences</A></LI></UL></DIV><DIV id="footer"><UL id="f-list"><LI id="t-print"><A href="/wiki/index.php?title=Team:Imperial/Protocols&amp;printable=yes" title="Printable version of this page [p]" accesskey="p">Printable version</A></LI><LI id="t-permalink"><A href="/wiki/index.php?title=Team:Imperial/Protocols&amp;oldid=382784" title="Permanent link to this revision of the page">Permanent link</A></LI><LI id="privacy"><A href="/2014.igem.org:Privacy_policy" title="2014.igem.org:Privacy policy">Privacy policy</A></LI><LI id="disclaimer"><A href="/2014.igem.org:General_disclaimer" title="2014.igem.org:General disclaimer">Disclaimers</A></LI></UL></DIV></DIV></DIV></DIV></BODY></HTML>