<HTML xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en" dir="ltr">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki  ltr ns-0 ns-subject page-Team_Imperial_EColi"><DIV id="globalWrapper"><DIV id="top-section"><DIV id="p-logo"><A href="/Main_Page" title="Main Page">&quot;
	    </A></DIV><DIV id="menubar" class="left-menu noprint"><UL><LI class="selected"><A href="/Team:Imperial/EColi">Page               </A></LI><LI class="new"><A href="/wiki/index.php?title=Talk:Team:Imperial/EColi&amp;action=edit&amp;redlink=1">Discussion               </A></LI><LI><A href="/wiki/index.php?title=Team:Imperial/EColi&amp;action=edit">View source               </A></LI><LI><A href="/wiki/index.php?title=Team:Imperial/EColi&amp;action=history">History               </A></LI><LI style="color:white;cursor:default">teams</LI></UL></DIV><DIV class="right-menu noprint" id="menubar"><UL><LI id="pt-login"><A href="/wiki/index.php?title=Special:UserLogin&amp;returnto=Team:Imperial/EColi" title="You are encouraged to log in; however, it is not mandatory [o]" accesskey="o">Log in</A></LI></UL></DIV><DIV id="search-controls" class="noprint"><FORM action="/Special:Search" id="searchform">Â </FORM></DIV></DIV><DIV id="content"><H1 class="firstHeading">Team:Imperial/EColi</H1><DIV id="bodyContent"><H3 id="siteSub" class="noprint">From 2014.igem.org</H3><P><TITLE>Imperial iGEM 2014</TITLE></P><DIV class="wrapper"><DIV class="header"><DIV class="header-banner"><DIV class="left"><A style="text-decoration: none" href="https://2014.igem.org/Team:Imperial"><H1 id="Logo">Aqualose</H1></A></DIV></DIV><DIV id="main_nav" class="pure-menu pure-menu-open pure-menu-horizontal"><UL id="std-menu-items"><LI><A href="#">Project</A></LI><LI><A href="https://2014.igem.org/Team:Imperial/Project">Overview</A></LI><LI><A href="https://2014.igem.org/Team:Imperial/Project_Background">Background</A></LI><LI><A href="https://2014.igem.org/Team:Imperial/Gluconacetobacter"><EM>G. xylinus</EM></A></LI><LI><A href="https://2014.igem.org/Team:Imperial/EColi"><EM>E. coli</EM></A></LI><LI><A href="https://2014.igem.org/Team:Imperial/coculture">Co-culture</A></LI><LI><A href="https://2014.igem.org/Team:Imperial/Functionalisation">Functionalisation</A></LI><LI><A href="https://2014.igem.org/Team:Imperial/Mass_Production_and_Processing">Mass Production and Processing</A></LI><LI><A href="https://2014.igem.org/Team:Imperial/Mechanical_Testing">Mechanical Testing</A></LI></UL><LI><A href="#">Achievements</A></LI><LI><A href="https://2014.igem.org/Team:Imperial/Achievements">Overview</A></LI><LI><A href="https://2014.igem.org/Team:Imperial/Parts">Parts</A></LI><LI><A href="https://2014.igem.org/Team:Imperial/Results">Results</A></LI><LI><A href="https://2014.igem.org/Team:Imperial/Judging_Criteria">Judging Criteria</A></LI><LI><A href="https://2014.igem.org/Team:Imperial/InterLab_Study">Interlab Study</A></LI><LI><A href="#">Application</A></LI><LI><A href="https://2014.igem.org/Team:Imperial/Implementation">Implementation</A></LI><LI><A href="https://2014.igem.org/Team:Imperial/Water_Filtration">Water Filtration</A></LI><LI><A href="#">Policy &amp; Practices</A></LI><LI><A href="https://2014.igem.org/Team:Imperial/Policy_and_Practices">Overview</A></LI><LI><A href="https://2014.igem.org/Team:Imperial/Informing_Design">Informing Design</A></LI><LI><A href="https://2014.igem.org/Team:Imperial/Water_Report">Water Report</A></LI><LI><A href="https://2014.igem.org/Team:Imperial/The_i_in_iGEM">The i in iGEM</A></LI><LI><A href="https://2014.igem.org/Team:Imperial/Outreach">Outreach</A></LI><LI><A href="https://2014.igem.org/Team:Imperial/Collaborations">Collaborations</A></LI><LI><A href="#">Modelling</A></LI><LI><A href="https://2014.igem.org/Team:Imperial/Modelling">Overview</A></LI><LI><A href="https://2014.igem.org/Team:Imperial/Induction_Model">Induction Model</A></LI><LI><A href="https://2014.igem.org/Team:Imperial/CBD_Kinetics_Model">CBD Kinetics Model</A></LI><LI><A href="https://2014.igem.org/Team:Imperial/Nutrient_Diffusion_Simulations">Nutrient Diffusion Simulations</A></LI><LI><A href="https://2014.igem.org/Team:Imperial/BioHackspace_Model">BioHackspace Model</A></LI><LI><A href="#">Notebook</A></LI><LI><A href="https://2014.igem.org/Team:Imperial/Notebook">Overview</A></LI><LI><A href="https://2014.igem.org/Team:Imperial/Lab-book">Lab-book</A></LI><LI><A href="https://2014.igem.org/Team:Imperial/Protocols">Protocols</A></LI><LI><A href="https://2014.igem.org/Team:Imperial/Brainstorming">Brainstorming</A></LI><LI><A href="https://2014.igem.org/Team:Imperial/Safety">Safety</A></LI><LI><A href="https://2014.igem.org/Team:Imperial/Art_and_Design">Art &amp; Design</A></LI><LI><A href="#">Team</A></LI><LI><A href="https://igem.org/Team.cgi?id=1321">Official Profile</A></LI><LI><A href="https://2014.igem.org/Team:Imperial/Students">Students</A></LI><LI><A href="https://2014.igem.org/Team:Imperial/Instructors_and_Advisors">Instructors and Advisors</A></LI><LI><A href="https://2014.igem.org/Team:Imperial/Sponsors">Sponsors</A></LI><LI><A href="https://2014.igem.org/Team:Imperial/Attributions">Attributions</A></LI></DIV></DIV><DIV class="content-wrapper"><DIV class="pure-g"><DIV class="pure-u-1-1 main"><H1><EM>E. coli</EM></H1><DIV id="navWrap"><DIV id="subNav"><UL><LI><A data-scroll="" href="#introduction">Introduction</A></LI><LI><A data-scroll="" href="#aim">Aims</A></LI><LI><A data-scroll="" href="#engineering">Engineering</A></LI><LI><A data-scroll="" href="#methods">Materials and Methods</A></LI><LI><A data-scroll="" href="#results">Results</A></LI><LI><A data-scroll="" href="#future">Future Work</A></LI><LI><A data-scroll="" href="#references">References</A></LI></UL></DIV></DIV><SECTION id="overview"><DIV class="pure-g"><DIV class="pure-u-1-2"><H2>Overview</H2><P><EM>Escherichia coli</EM> is the organism of choice for synthetic biology. It is thoroughly characterised, easy to engineer and has numerous parts, control circuits and complex constructs tested in this host.</P><P>Some <EM>E. coli</EM> strains have evolved to produce low amounts of cellulose but the machinery under specific regulatory control (its secretion is generally linked to biofilm formation and stress situations). In order to expand the use of bacterial cellulose as a functional biomaterial that is cost effective to produce, it is desirable for its production machinery to be implemented in an organisms that is easy to engineer and has proven success in large scale bioreactors.</P><P>Here, we confirm that the high output cellulose production machinery of <EM>Gluconacetobacter Xylinus</EM> can be transferred into other organisms. We have proven function of the Acs cellulose producing operon in <EM>Escherichia coli</EM> using Congo Red binding assays.
                            </P></DIV><DIV class="pure-u-1-2"><H2>Key Achievements </H2><UL><LI>Contributed an optimised Asc cellulose production operon to Registry of Standard Parts.</LI><LI>Proved the portability of <EM>Gluconacetobacter xylinus</EM> operon.</LI><LI>Assembled a fully synthetic, functional, cellulose-producing system in <EM>Escherichia coli</EM></LI><LI>Demonstrated the synthesis operon in a two plasmid system for separate induction of genes</LI></UL></DIV></DIV></SECTION><DIV class="pure-g"><DIV class="pure-u-1-2"><FIGURE class="content-image image-full"><FIGCAPTION>Figures 1 &amp; 2: Congo red staining of microbial cellulose in <EM>E. coli</EM></FIGCAPTION></FIGURE></DIV></DIV><SECTION id="Aims"><H2>Aims</H2><P>By transferring the cellulose production operon from its native organism <EM>Gluconacetobacter xylinus</EM> into the lab and industrially friendly host <EM>Escherichia coli</EM>, we aim to improve cellulose production per unit time to achieve more cost effective pellicle growth and to encourage further synthetic biology with microbial cellulose. <EM>E. coli</EM> grows considerably faster plus tolerates higher temperatures and more vigorous aeration. The ease of genetic engineering and the wealth of other characterized Biobricks increases the potential for exciting future projects</P></SECTION><SECTION id="engineering"><H2>Engineering</H2><P>The cellulose production operon in <EM>Gluconacetobacter xylinus</EM> is a 10kb genomic region, consisting of four main elements:</P><UL><LI>acsAB, known as cellulose synthase, is made up of two domains: acsA being the catalytic domain, and acsB being the cyclic-di-GMP-binding domain. This modular enzyme requires the second messenger cyclic-di-GMP in order to carry out efficient catalysis. (Lee et al., 2014)</LI><LI>acsC codes for an oligomeric element that gets embedded in the extracellular membrane and allows transport of the growing polymer across the cell membrane into the extracellular environment. (Lee et al., 2014)</LI><LI>acsD is believed to be a non-essential element, although it is a key player in the control of cellulose crystallinity and required for maximal cellulose productivity. (Lee et al., 2014)</LI></UL><P>The native operon is highly regulated at transcriptional, translational and post-translational levels. Cellulose production responds to a wide set of internal signals and environmental cues. The native ribosomal binding sites (RBS) are predicted to be weak by translation rate calculators (The <A href="https://salis.psu.edu/software/"> Salis Lab RBS calculator </A> and <A href="http://sbi.postech.ac.kr/utr_designer">  Postech UTR designer </A>). This is perhaps to reduce noise in gene expression (strong promoter, weak RBS is less noisy that weak promoter strong RBS). The promoter region has not yet been characterized in <EM>Gluconacetobacter xylinus</EM>, however. (Kawano et al., 2002)</P><P>The first step of our computational design engineering was to identify the coding sequences and extract them from NCBI. We removed some native regulation by codon-optimizing the sequences for expression in <EM>E.coli</EM> and replaced the native RBSâs with the strong B0034 sequence. The 6 base pairs either side of the B0034 RBS were edited with the Salis Lab RBS calculator design function to tune RBS strength output to better preserve the natural stoichiometry.</P><P>Finally, in order to allow for even more customizable fine-tuning of the system, the operon was divided and cloned into two separate expression vectors: the AcsAB element was cloned under the control of the inducible pBAD promoter contained within pSB1C3, a high copy plasmid that would contribute to higher expression of Cellulose Synthase. Alternatively, AcsC and AcsD, believed to be expressed at lower levels in <EM>G.xylinus</EM>, was put under the control of the pLAC promoter and was transferred into a low copy plasmid, pSB3K3 so as to maintain lower levels of expression. This approach provided the following advantages:</P><UL><LI>Reducing the size of the final expression vector avoids complications during assembly and vector take-up, and it also prevents a large construct from becoming a burden for cell growth and vector propagation.</LI><LI>By expressing the cellulose production machinery from two separate expression vectors, the following levels of control become available to us: the first being the changes in vector copy number, which directly affects gene expression levels throughout induction, and the second being the ability to activate the system using two different inducers. These key features of our system allow for a larger scope for fine-tuning and optimization of cellulose production levels.</LI></UL><FIGURE class="content-image"><FIGCAPTION>Figure 3: pSB-pBAD-AB and pLAC-CD</FIGCAPTION></FIGURE><FIGURE class="content-image image-right"><FIGCAPTION>Figure 4: acsABCD fragments</FIGCAPTION></FIGURE><P>With this approach, we allow for our system to be implemented in new range of hosts because a larger set of growth conditions, inducer concentrations and growth phases can be tested in order to achieve the required stoichiometry for the system to become functional.</P><P>In order to ease synthesis, the operon sequence was split into 4 fragments of similar sizes. We took advantage of the presence of two restriction sites: An NcoI present within the AcsAB sequence, and a PvuII site contained within AcsC. Furthermore, the BioBrick prefix and suffix were included and any additional sites removed elsewhere.</P><P>By standard cloning, these fragments could easily be put together to yield the following constructs:</P><FIGURE class="content-image image-small"><FIGCAPTION>Figure 5: acsAB and acsCD constructs</FIGCAPTION></FIGURE><P>Gene Designer served as a very useful tool for assembling constructs and predicting cloning outcomes before proceeding with the experimental procedure. Find above the plasmid maps of our resulting constructs, and refer to Materials and Methods for a deeper insight into the assembly process.</P></SECTION><SECTION id="methods"><H2>Materials and Methods</H2><DIV class="accordion"><H3>1.0 Cell Strains and Media Components</H3><DIV><P><EM>Escherichia coli</EM> strain DH10B were made chemically competent following a standard protocol and were used to clone the pSB-AraC-pBAD and pSB-LacI-pLAC elements. Additionally, High Efficiency chemically competent DH10B <EM>Escherichia coli</EM> were purchased from New England Biolabs for assembly of the Acs cellulose-producing operon, which was codon-optimized for expression in E.coli and synthesized by GeneArt. Both strains were grown overnight in liquid cultures (supplemented with specific antibiotics), or on semi-solid LB-Agar plates, provided with the specific antibiotic, as listed on the table below.</P><P>Luria Broth Miller (LB) and LB Agar were purchased from VWR, prepared using standard protocols and used throughout the experimental procedure. During characterization, Acs-containing strains were supplemented with 1% D-glucose, 0.1% Arabinose and 0.5mM IPTG.
                                </P><FIGURE class="content-image"><TABLE class="tg image-full"><TBODY><TR><TH class="tg-e3zv">Plasmid name</TH><TH class="tg-e3zv">Vector backbone</TH><TH class="tg-e3zv">Antibiotic concentration (ug/ml)</TH></TR><TR><TD class="tg-031e">pSB-AraC-pBAD</TD><TD class="tg-031e">pSB1C3</TD><TD class="tg-031e">Chloramphenicol 50</TD></TR><TR><TD class="tg-031e">pSB-LacI-pLAC</TD><TD class="tg-031e">pSB3K3</TD><TD class="tg-031e">Kanamycin 25</TD></TR><TR><TD class="tg-031e">pSB-AcsAB</TD><TD class="tg-031e">pSB1C3</TD><TD class="tg-031e">Chloramphenicol 50</TD></TR><TR><TD class="tg-031e">pSB-AcsCD</TD><TD class="tg-031e">pSB1C3</TD><TD class="tg-031e">Chloramphenicol 50</TD></TR><TR><TD class="tg-031e">pSB-AraC-pBAD-AcsAB</TD><TD class="tg-031e">pSB1C3</TD><TD class="tg-031e">Chloramphenicol 50</TD></TR><TR><TD class="tg-031e">pSB-LacI-pLAC-CD</TD><TD class="tg-031e">pSB3K3</TD><TD class="tg-031e">Kanamycin 25</TD></TR></TBODY></TABLE><FIGCAPTION>Figure 6: List of antibiotics used, in relation to their respective constructs</FIGCAPTION></FIGURE></DIV><H3>2.0 Chemicals </H3><DIV><P>Congo Red was purchased from Sigma Aldrich (Steiheim, Germany) and diluted in 1x PBS to a concentration of 350 uM. The final Congo Red concentration used during system characterization steps was 20 uM. This was also used, also at a concentration of 20 uM, to make LB Agar Congo Red Assay plates.</P></DIV><H3>3.0 Construct Assembly</H3><DIV><H4>3.1 Cloning of pSB-AraC-pBAD</H4><DIV><P>The AraC and pBAD elements were obtained by PCR, using the Biobricks BBa_K325108, BBa_K325218 and BBa_K325219 as DNA templates, all of which follow the standard structure showed on figure 7. The primers used during this procedure were as follows:
                                    </P><UL><LI>Forward: TACTAGTAGCGGCCGCTGCAG</LI><LI>Reverse: GCTAGCCCAAAAAAACGGGTATGGAG</LI></UL><FIGURE class="content-image"><FIGCAPTION>Figure 7 - Plasmid maps describing both the original (left) and engineered (right) AraC-pBAD-containing parts, including all relevant elements. These were designed and cloned in silico on GeneDesigner.</FIGCAPTION></FIGURE></DIV><P>The amplified elements were DpnI-treated following a standard procedure (New England Biolabs), and were subsequently submitted to a purification step (Qiaquick PCR clean up, Qiagen) prior to self-ligation. Three different template concentrations were tested during ligation (specifically 3 ng, 7.5 ng and 15 ng), and were incubated at room temperature for 1h+. The ligation mix was then transformed into chemically competent DH10B Escherichia coli using a standard chemical transformation protocol and plated out on Chloramphenicol-specific LB plates.</P><H4>3.2 Cloning of pSB3K3-LacI-pLAC</H4><P>A compatible plasmid and induction system was cloned to allow expression of the operon in two parts. The medium/strong promoter J23110 was cloned in front of the quad part inverter Q01121. The inverter has an RBS, the LacI repressor followed by a double terminator then a Lac inducible promoter. The composite part created give constitutive expression of the Lac repressor allowing for induction by IPTG. The parts were then moved into the pSB3k3 backbone to be compatible with pSB1C3.</P><H4>3.3 Assembly of AcsAB and AcsCD </H4><FIGURE class="content-image image-right"><FIGCAPTION>Figure 8: Diagram describing 3-part assembly of AcsAB and AcsCD</FIGCAPTION></FIGURE><P>The AcsABCD operon was designed in silico as described in the previous section. It was ordered for synthesis at GenArt in the form of four separate elements of similar sizes, named AcsA, AcsB, AcsC and AcsD, respectively. After synthesis, the constructs were directly transformed into DH10B <EM>E.coli</EM> using a standard chemical transformation protocol and plated overnight supplied with 50 ug/ml Kanamycin. 5ml LB were inoculated with a freshly growing colony and grown overnight at 37 ï°C, under shaking conditions. Glycerol stocks were produced prior to proceeding with the assembly process.</P><P>In order to meet the Registry standards, we aimed to clone these elements into pSB1C3. To do so, we proceeded as described on figure 8. Two separate rounds of digestion were carried out using EcoRI, NcoI, PvuII and PstI to, and the resulting fragments (AcsA, AcsB, AcsC and AcsD and linearized pSB1C3) were submitted to a three-part ligation after dephosphorylation of the vector backbone (rAPid desphosphorylation, Roche). An equimolar ratio amongst all three parts was established during the ligation procedure, which in turn proceeded at 4 degrees overnight. The NEB High Efficiency Transformation protocol was used on NEB 10B cells (New England Biolabs) to propagate the resulting plasmid. Transformed cells were plated and incubated overnight at 37 ï°C. 5 ml LB supplied with 50 ug/ml Chloramphenicol were inoculated with a selection of colonies for further analysis (refer to Results for additional information).</P><H4>3.4 Assembly of pSB-AraC-pBAD-AcsAB</H4><P>AcsAB was cloned into pSB-AraC-pBAD by Biobrick cloning. The destination vector was linearized using SpeI and PstI (New England Biolabs), and AcsAB (previously digested with XbaI and PstI, and gel extracted using the Minelute Gel Extraction Kit, by Qiagen) was ligated at a 1:1 ratio throughout an overnight ligation at room temperature. The resulting part was transformed into chemically competent DH10B Escherichia coli and plated on LB Agar+Chloramphenicol plates, grown overnight at 37 ï°C. 5 ml LB falcons supplied with 50 ug/ml Chloramphenicol were inoculated with a selection of freshly grown colonies for further restriction analysis (refer to Results for additional information)</P><H4>3.5 Assembly of pSB-LacI-pLAC-CD</H4><P>Similarly to 3.4, AcsCD was cloned by BioBrick cloning into the pSB3K3 low copy expression vector containing the LacI-pLAC regulatory elements. Vector and insert were ligated at a 1:1 ratio overnight at room temperature. After transformation and plating on LB Agar+Kanamycin (25 ug/ml), colonies were screened by Greentaq colony PCR, and were further confirmed during the Congo Red Assay.</P></DIV><H3>Electroporation of pSB-pLAC-CD into electrocompetent AcsAB-containing DH10B cells</H3><DIV><P>DH10B E.coli already containing the AcsAB construct were made electrocompetent using a standard protocol, which can be found in the Protocols section. A second round of transformation was carried out to insert pLAC-AcsCD. 5ul of miniprepped pSB-pLAC-CD (diluted in ddH2O) were added into 50ul electrocompetent DH10B cells already containing the pBAD-inducible AcsAB element. Electroporation was carried out in in 10mm electrocuvettes at 1.8 kV, 200 Ohm resistance and 25 mF capacitance. 500ul SOC medium was added to the electroporated samples and these were recovered at 37 Â°C for 1h before plating on Kanamycin25 + Chloramphenicol 50 LB Agar plates.</P></DIV><H3>Congo Red Characterization</H3><DIV><P>LB Agar Congo Red assay plates were prepared with a final Congo Red concentration of 20uM, and they were supplied with 0.1% Arabinose, 0.5mM IPTG, 50 ug/mL Chloramphenicol, 25 ug/mL Kanamycin and 1% Glucose. Transformed cells containing both AcsAB and AcsCD were plated on Congo Red assay plates and incubated at 37 Â°C. They were qualitatively assayed after 24h and 48h for cellulose production. Finally, 5 mL LB supplied with 0.1% Arabinose, 0.5mM IPTG, 50 ug/mL Chloramphenicol, 25 ug/mL Kanamycin and 1% Glucose were inoculated with freshly grown, cellulose-producing colonies and were grown for 16h at 30 Â°C, 180 rpm. An empty vector control was also grown overnight for comparative analysis. These liquid cultues were supplied with Congo Red, to yield a final concentration of 20uM. They were incubated under static conditions for 2h at room temperature, and were later pelleted down to allow for qualitative analysis of Congo Red binding.</P><P>AcsAB functionality was assayed by inducing AcsAB-containing cells with 0.1% Arabinose in 5mL LB supplied with 1% Glucose and Chloramphenicol. Overnight incubations were set up at 30 Â°C and 37 Â°C, and both empty vector controls and un-induced controls were also assayed. Cells were pelleted at 4000rpm for 10 minutes and the supernatant was kept for further analysis. Cells were resuspended in 1xPBS and sonicated for 3 minutes as on/off cycles of 15 seconds. Both the membrane and soluble samples produced after centrifugation were kept for further analysis. Congo Red was then added to the LB, soluble and membrane samples up to a final concentration of 20uM, and no Congo Red samples were also kept for further inspection. Samples were incubated for 2h at static conditions and at room temperature to allow for Congo Red binding. 96-well plates were inoculated for further absorbance measurements at 490nm.</P></DIV></DIV></SECTION><SECTION id="results"><H2>Results</H2><DIV class="accordion"><H3>Constructs</H3><DIV><P>All constructs were assembled as described in Materials and Methods. As a preliminary verification step, restriction analyses were carried out using the corresponding restriction enzymes (refer to figure legends for more details) and results were visualized on 0.7-1% agarose gels. Final confirmation was obtained by gene sequencing, during which the BioBrick Verification Primers and walking primers were used. These are shown on the table below, please refer to the Registry for further information.</P><FIGURE class="content-image"><TABLE class="tg image-full"><TBODY><TR><TD class="tg-031e">pSB-AraC-pBAD</TD><TD class="tg-031e">Forward: TACTAGTAGCGGCCGCTGCAG
                                                Reverse: GCTAGCCCAAAAAAACGGGTATGGAG</TD></TR><TR><TD class="tg-031e">BioBrick Verification Primers</TD><TD class="tg-031e">Forward: TGCCACCTGACGTCTAAGAA
                                                Reverse: ATTACCGCCTTTGAGTGAGC</TD></TR><TR><TD class="tg-031e">Walking Primer 1</TD><TD class="tg-031e">TCT GAA TTA TGC CAT TGG TCA TAC CG</TD></TR><TR><TD class="tg-031e">Walking Primer 2</TD><TD class="tg-031e">ATA TGT TTC ATG CAG TTG GCA CC</TD></TR><TR><TD class="tg-031e">Walking Primer 3</TD><TD class="tg-031e">GAT GCA TGG GTT GAT TGG GG</TD></TR><TR><TD class="tg-031e">Walking Primer 4</TD><TD class="tg-031e">ATC CAG CAC ATC CGA CCT TTG</TD></TR><TR><TD class="tg-031e">Walking Primer 5</TD><TD class="tg-031e">GTT ACC GCA AGT AAA CTG CAA G</TD></TR><TR><TD class="tg-031e">Walking Primer 6</TD><TD class="tg-031e">GAT GGT CTG ATT CGT CTG GTT A</TD></TR><TR><TD class="tg-031e">Walking Primer 7</TD><TD class="tg-031e">ACG CAG CAC AGG TCA GAC CGG TGA A</TD></TR><TR><TD class="tg-031e">Walking Primer 8</TD><TD class="tg-031e">ATA TTG ATC TGA CCA CCG AACA</TD></TR><TR><TD class="tg-031e">Walking Primer 9</TD><TD class="tg-031e">TGC ACC GCC TGG TGA AAA TGG TT</TD></TR><TR><TD class="tg-031e">Walking Primer 10</TD><TD class="tg-031e">GTG GCT ATG CAA TTC AGA CAG GT</TD></TR><TR><TD class="tg-031e">Walking Primer 11</TD><TD class="tg-031e">TTC ATT CCC AGC GGT CGG TCG AT</TD></TR><TR><TD class="tg-031e">Walking Primer 12</TD><TD class="tg-031e">GTT TGC GTG GTG ATC TTT TT</TD></TR></TBODY></TABLE><FIGCAPTION>Figure 9 - List of primers used by Team E.coli during the experimental procedure</FIGCAPTION></FIGURE><DIV class="pure-g"><DIV class="pure-u-1-2"><FIGURE class="content-image"><FIGCAPTION>Figure 10: Restriction Analysis results of AcsAB. Samples were submitted to an XbaI/PstI double digest, which flank the AcsAB construct. Two bands were displayed as expected: a ~5kb band corresponding to the AcsAB insert, and a ~2kb band corresponding to the pSB1C3 vector backbone. </FIGCAPTION></FIGURE></DIV><DIV class="pure-u-1-2"><FIGURE class="content-image"><FIGCAPTION>Figure 11: Restriction analysis results of AcsCD. Samples were submitted to an XbaI/SpeI double restriction. Positive samples displayed a 4.4kb band corresponding to the AcsCD insert, in addition to a 2kb band that corresponded to the pSB1C3 plasmid backbone. </FIGCAPTION></FIGURE></DIV><DIV class="pure-u-1-2"><FIGURE class="content-image"><FIGCAPTION>Figure 12: Restriction Analysis results of AraC-pBAD. Samples were submitted to an XbaI/PstI double digest. A 2-band pattern was obtained, indicating the presence of a ~2kb DNA fragment corresponding to the pSB1C3 vector backbone, and a ~1kb fragment corresponding to the AraC and pBAD elements being cloned. </FIGCAPTION></FIGURE></DIV><DIV class="pure-u-1-2"><FIGURE class="content-image"><FIGCAPTION>Figure 13: Restriction Analysis results of AraC-pBAD. Samples were submitted to an XbaI/PstI double digest. A 2-band pattern was obtained, indicating the presence of a ~2kb DNA fragment corresponding to the pSB1C3 vector backbone, and a ~1kb fragment corresponding to the AraC and pBAD elements being cloned. </FIGCAPTION></FIGURE></DIV><DIV class="pure-u-1-2"><FIGURE class="content-image"><FIGCAPTION>Figure 14: pLAC-AcsCD colony PCR results. Further details on the procedure can be found on the Methods and Materials section. Those colonies that we testes further have been highlighted with numbers on the pictures. Specifically, colony A1 was used for subcloning into AcsAB containing E.coli DH10B. 
Plates A and B were generated 
</FIGCAPTION></FIGURE></DIV><DIV class="pure-u-1-2"><FIGURE class="content-image"><FIGCAPTION>Figure 15: pLAC-AcsCD colony PCR results. Further details on the procedure can be found on the Methods and Materials section. Those colonies that we testes further have been highlighted with numbers on the pictures. Specifically, colony A1 was used for subcloning into AcsAB containing E.coli DH10B. 
Plates A and B were generated 
</FIGCAPTION></FIGURE></DIV></DIV></DIV><H3>Characterising Cellulose Production</H3><DIV><FIGURE class="content-image image-right"><FIGCAPTION>Figure 16: Congo Red Assay under progress. LB, soluble and non-soluble fractions were assayed.</FIGCAPTION></FIGURE><P>Cellulose Production can be easily assayed using the diazo dye Congo Red. Congo Red specifically binds cellulostic fibrils, and this binding can be measured by spectral analysis, because it causes a shift on the spectral properties of the dye, which can be monitored with a spectrophotometer (Klunk et al., 1999). In addition to this, Congo Red also allows for visual, qualitative analysis: it causes a color change of any cellulose-producing colonies by binding to the growing fibres. Therefore, cellulose production can be easily detected by eye on LB Agar plates and liquid cultures (Jolanta Sereikaite and Vladas-Algirdas Bumelis, 2006).</P><P>We assayed cellulose production on LB Agar Congo Red assay plates. Colonies containing the refactored cellulose production machinery turned red within 24h, and presented a rougher composition and brighter red color after 48h. In contrast to this, colonies presenting empty vector (negative controls), presented no color changes and maintained the characteristics of standard <EM>Escherichia coli</EM> colonies (smoother composition and standard yellow colour).</P><P>To confirm this qualitative assay, a second qualitative assay was carried out, also based on Congo Red binding. This second analysis consisted of incubating overnight cultures (both cellulose producers and empty vector controls) with Congo Red as described in Materials and Methods. Cellulose producers will present growing cellulose fibres on their extracellular surface, hence will dye red in the presence of Congo Red, unlike non-cellulose-producing cells. After spinning down the cells, a red pellet was observed that corresponded to the cellulose producing colonies. In contrast to this, negative controls did not bind Congo Red and hence maintained the standard E.coli characteristics. We consider that this qualitative data supports our design and confirms that the cellulose production operon has been successfully refactored and transferred into <EM>Escherichia coli</EM>.</P><P>In order to gain a better understanding of the activity of the Acs elements in Escherichia coli, it was of interest to explore whether the AcsAB element alone would be enough to achieve cellulose production. AcsAB is embedded in the cytoplasmic membrane and transports the growing polymer through the membrane as it attaches the glucose monomers in the cytoplasm.  In the absence of AcsC and AcsD, no cellulose can be secreted into the environment and as a result, it was assumed that it would instead become accumulated in the cellâs periplasm. For this reason, during characterization we required sonication in order to get the cellulose (if any at all) out of the cells.</P><P>To widen our screening, we set up induced samples and uninduced controls and assayed samples at different temperatures (30Â°C versus 37Â°C) and different conditions (shaking versus static) in the presence or absence of 1% D-Glucose. Furthermore, 2 Biological replicates were assayed, and technical triplicates were also set up during plate reading to support any statistically relevant findings. Although we expected no cellulose to be accumulating in the medium, we still analysed all fractions originating throughout the procedure: LB fractions, soluble fractions and non-soluble fractions (called membrane fractions throughout this study).</P><P>All fractions were incubated in the presence of Congo Red (as described in Materials and Methods) and were later inoculated into a 96-well plate for absorbance measurements at 490nm. Free Congo Red has a maximal absorption at such wavelength. By setting up a control sample, containing solvent (PBS)+ Congo Red, it is possible to measure the total absorbance of the molecule whilst free in solution. When cellulolytic samples are incubated in the presence of Congo Red, less azodye molecules becomes available in the medium due to binding, hence absorbance levels will decrease. The total Congo Red bound will equal the absorbance of Congo Red Control minus the absorbance of our samples of interest. This approach gives you an idea of the amount of Congo Red that may have bound to the cellulositic fibres, but it does not provide exact values of cellulose present.</P><FIGURE class="content-image image-half"><FIGCAPTION>Figure 17: LB fraction - Congo Red Binding</FIGCAPTION></FIGURE><P>As expected, LB fractions all presented similar, non-cellulolytic composition and hence displayed no changes in absorbance, indicating no CR binding. </P><FIGURE class="content-image image-half"><FIGCAPTION>Figure 18: Membrane Fraction - Induction of the pBAD-AcsAB</FIGCAPTION></FIGURE><P>It was observed that interestingly, membrane fractions that had been previously incubated at 30 ï°C presented a dramatic change in Congo Red binding. This proves not only that the AcsAB is active (at least to some extent) in the absence of AcsC and AcsD, but also that it may be optimal for the cellulose synthase to be folded and functional at lower temperatures, as it occurs in the native host G.xylinus. </P></DIV></DIV></SECTION><SECTION id="references"><H2>References</H2><OL><LI>Kawano, S.; Tajima, K., Uemori, Y.; Yamashita, H.; Erata, T.; Munekata, M.; Takai, M. (2002) Cloning of Cellulose Synthesis Related Genes from Acetobacter xylinum ATCC23769 and ATCC53582: Comparison of Cellulose Synthetic Ability Between Strains. Available on: http://dnaresearch.oxfordjournals.org/content/9/5/149.full.pdf
                            </LI><LI>Lee, K.Y; Guldum, G.; Mantalaris, A.; Bismarck, A.; (2014) âMore than meets the eye in Bacterial Cellulose: Biosynthesis, Bioprocessing and Applications in Advanced Fiber Compositesâ <A href="http://onlinelibrary.wiley.com/store/10.1002/mabi.201300298/asset/mabi201300298.pdf?v=1&amp;t=i1ef935d&amp;s=482503feebacfa4a7a977e554b9f410fa6d22a5f&amp;systemMessage=Wiley+Online+Library+will+be+disrupted+on+the+18th+October+from+10%3A00+BST+%2805%3A00+EDT%29+for+essential+maintenance+for+approximately+two+hours+as+we+make+upgrades+to+improve+our+services+to+you">Available here</A></LI><LI>P. Ross, R. Mayer, and M. Benziman (1991) &quot;Cellulose biosynthesis and function in bacteria,&quot; Microbiol Mol Biol Rev, vol. 55, no. 1, pp. 35-58, Mar</LI><LI>Sereikaite, J.; Bumelis, V.A. (2006) Congo Red Interactions with alpha-proteins. Available on: http://www.actabp.pl/pdf/1_2006/87.pdf</LI><LI>W. E. Klunk, R. F. Jacob, and R. P. Mason, âQuantifying amyloid by congo red spectral shift assay,â Methods in Enzymology, vol. 309, pp. 285â305, 1999.</LI><LI>https://2013.igem.org/Team:Toronto/Project/Assays</LI><LI>https://2010.igem.org/Team:Tokyo_Metropolitan/Project/Fiber/Protocol</LI></OL></SECTION></DIV></DIV></DIV><DIV class="printfooter">
Retrieved from &quot;<A href="http://2014.igem.org/Team:Imperial/EColi">http://2014.igem.org/Team:Imperial/EColi</A>&quot;</DIV></DIV></DIV><DIV id="footer-box" class="noprint"><DIV id="footer"><UL id="f-list"><LI id="t-recentchanges"><A href="/Special:RecentChanges" title="Recent changes">Recent changes</A></LI><LI id="t-whatlinkshere"><A href="/Special:WhatLinksHere/Team:Imperial/EColi" title="List of all wiki pages that link here [j]" accesskey="j">What links here</A></LI><LI id="t-recentchangeslinked"><A href="/Special:RecentChangesLinked/Team:Imperial/EColi" title="Recent changes in pages linked from this page [k]" accesskey="k">Related changes</A></LI><LI id="t-specialpages"><A href="/Special:SpecialPages" title="List of all special pages [q]" accesskey="q">Special pages</A></LI><LI><A href="/Special:Preferences">My preferences</A></LI></UL></DIV><DIV id="footer"><UL id="f-list"><LI id="t-print"><A href="/wiki/index.php?title=Team:Imperial/EColi&amp;printable=yes" title="Printable version of this page [p]" accesskey="p">Printable version</A></LI><LI id="t-permalink"><A href="/wiki/index.php?title=Team:Imperial/EColi&amp;oldid=401590" title="Permanent link to this revision of the page">Permanent link</A></LI><LI id="privacy"><A href="/2014.igem.org:Privacy_policy" title="2014.igem.org:Privacy policy">Privacy policy</A></LI><LI id="disclaimer"><A href="/2014.igem.org:General_disclaimer" title="2014.igem.org:General disclaimer">Disclaimers</A></LI></UL></DIV></DIV></DIV></DIV></BODY></HTML>