<HTML xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en" dir="ltr">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki  ltr ns-0 ns-subject page-Team_Imperial_InterLab_Study"><DIV id="globalWrapper"><DIV id="top-section"><DIV id="p-logo"><A href="/Main_Page" title="Main Page">&quot;
	    </A></DIV><DIV id="menubar" class="left-menu noprint"><UL><LI class="selected"><A href="/Team:Imperial/InterLab_Study">Page               </A></LI><LI class="new"><A href="/wiki/index.php?title=Talk:Team:Imperial/InterLab_Study&amp;action=edit&amp;redlink=1">Discussion               </A></LI><LI><A href="/wiki/index.php?title=Team:Imperial/InterLab_Study&amp;action=edit">View source               </A></LI><LI><A href="/wiki/index.php?title=Team:Imperial/InterLab_Study&amp;action=history">History               </A></LI><LI style="color:white;cursor:default">teams</LI></UL></DIV><DIV class="right-menu noprint" id="menubar"><UL><LI id="pt-login"><A href="/wiki/index.php?title=Special:UserLogin&amp;returnto=Team:Imperial/InterLab_Study" title="You are encouraged to log in; however, it is not mandatory [o]" accesskey="o">Log in</A></LI></UL></DIV><DIV id="search-controls" class="noprint"><FORM action="/Special:Search" id="searchform"> </FORM></DIV></DIV><DIV id="content"><H1 class="firstHeading">Team:Imperial/InterLab Study</H1><DIV id="bodyContent"><H3 id="siteSub" class="noprint">From 2014.igem.org</H3><P><TITLE>Imperial iGEM 2014</TITLE></P><DIV class="wrapper"><DIV class="header"><DIV class="header-banner"><DIV class="left"><A style="text-decoration: none" href="https://2014.igem.org/Team:Imperial"><H1 id="Logo">Aqualose</H1></A></DIV></DIV><DIV id="main_nav" class="pure-menu pure-menu-open pure-menu-horizontal"><UL id="std-menu-items"><LI><A href="#">Project</A></LI><LI><A href="https://2014.igem.org/Team:Imperial/Project">Overview</A></LI><LI><A href="https://2014.igem.org/Team:Imperial/Project_Background">Background</A></LI><LI><A href="https://2014.igem.org/Team:Imperial/Gluconacetobacter"><EM>G. xylinus</EM></A></LI><LI><A href="https://2014.igem.org/Team:Imperial/EColi"><EM>E. coli</EM></A></LI><LI><A href="https://2014.igem.org/Team:Imperial/coculture">Co-culture</A></LI><LI><A href="https://2014.igem.org/Team:Imperial/Functionalisation">Functionalisation</A></LI><LI><A href="https://2014.igem.org/Team:Imperial/Mass_Production_and_Processing">Mass Production and Processing</A></LI><LI><A href="https://2014.igem.org/Team:Imperial/Mechanical_Testing">Mechanical Testing</A></LI></UL><LI><A href="#">Achievements</A></LI><LI><A href="https://2014.igem.org/Team:Imperial/Achievements">Overview</A></LI><LI><A href="https://2014.igem.org/Team:Imperial/Parts">Parts</A></LI><LI><A href="https://2014.igem.org/Team:Imperial/Results">Results</A></LI><LI><A href="https://2014.igem.org/Team:Imperial/Judging_Criteria">Judging Criteria</A></LI><LI><A href="https://2014.igem.org/Team:Imperial/InterLab_Study">Interlab Study</A></LI><LI><A href="#">Application</A></LI><LI><A href="https://2014.igem.org/Team:Imperial/Implementation">Implementation</A></LI><LI><A href="https://2014.igem.org/Team:Imperial/Water_Filtration">Water Filtration</A></LI><LI><A href="#">Policy &amp; Practices</A></LI><LI><A href="https://2014.igem.org/Team:Imperial/Policy_and_Practices">Overview</A></LI><LI><A href="https://2014.igem.org/Team:Imperial/Informing_Design">Informing Design</A></LI><LI><A href="https://2014.igem.org/Team:Imperial/Water_Report">Water Report</A></LI><LI><A href="https://2014.igem.org/Team:Imperial/The_i_in_iGEM">The i in iGEM</A></LI><LI><A href="https://2014.igem.org/Team:Imperial/Outreach">Outreach</A></LI><LI><A href="https://2014.igem.org/Team:Imperial/Collaborations">Collaborations</A></LI><LI><A href="#">Modelling</A></LI><LI><A href="https://2014.igem.org/Team:Imperial/Modelling">Overview</A></LI><LI><A href="https://2014.igem.org/Team:Imperial/Induction_Model">Induction Model</A></LI><LI><A href="https://2014.igem.org/Team:Imperial/CBD_Kinetics_Model">CBD Kinetics Model</A></LI><LI><A href="https://2014.igem.org/Team:Imperial/Nutrient_Diffusion_Simulations">Nutrient Diffusion Simulations</A></LI><LI><A href="https://2014.igem.org/Team:Imperial/BioHackspace_Model">BioHackspace Model</A></LI><LI><A href="#">Notebook</A></LI><LI><A href="https://2014.igem.org/Team:Imperial/Notebook">Overview</A></LI><LI><A href="https://2014.igem.org/Team:Imperial/Lab-book">Lab-book</A></LI><LI><A href="https://2014.igem.org/Team:Imperial/Protocols">Protocols</A></LI><LI><A href="https://2014.igem.org/Team:Imperial/Brainstorming">Brainstorming</A></LI><LI><A href="https://2014.igem.org/Team:Imperial/Safety">Safety</A></LI><LI><A href="https://2014.igem.org/Team:Imperial/Art_and_Design">Art &amp; Design</A></LI><LI><A href="#">Team</A></LI><LI><A href="https://igem.org/Team.cgi?id=1321">Official Profile</A></LI><LI><A href="https://2014.igem.org/Team:Imperial/Students">Students</A></LI><LI><A href="https://2014.igem.org/Team:Imperial/Instructors_and_Advisors">Instructors and Advisors</A></LI><LI><A href="https://2014.igem.org/Team:Imperial/Sponsors">Sponsors</A></LI><LI><A href="https://2014.igem.org/Team:Imperial/Attributions">Attributions</A></LI></DIV></DIV><DIV class="content-wrapper"><DIV class="pure-g"><DIV class="pure-u-1-1 main"><H1>Interlab Study</H1><DIV id="navWrap"><DIV id="subNav"><UL><LI><A data-scroll="" href="#introduction">Introduction</A></LI><LI><A data-scroll="" href="#release">I. Release</A></LI><LI><A data-scroll="" href="#protocols">II. Protocols</A></LI><LI><A data-scroll="" href="#measured">III. Measured Quantities</A></LI><LI><A data-scroll="" href="#measurments">IV. Measurements</A></LI><LI><A data-scroll="" href="#extra">Extra</A></LI><LI><A data-scroll="" href="#references">References</A></LI><LI><A data-scroll="" href="#appendix">Appendix</A></LI></UL></DIV></DIV><SECTION id="overview"><DIV class="pure-g"><DIV class="pure-u-1-2"><H2>Overview</H2><P>The characterisation of standard biological parts is a fundamental principle of synthetic biology.  In the interlab study iGEM teams from around the world measure the fluorescence of the same three GFP-coding devices after a set time of growth. This provides a large dataset to analyse variation between labs. Two devices were cloned ourselves as specified, and one was provided by the registry. Using both a plate reader and flow cytometer fluorescence measurements were obtained in absolute units in terms of moles of sodium fluorescein or relative to standard beads. In addition to showing increased fluorescence in the stronger promoter as expected, it was also seen that for the higher copy plasmid the cells appeared to be burdened. Finally, the flow-cytomoter data was used to explore cell to cell variation as part of the extra-credit assignment.   </P></DIV><DIV class="pure-u-1-2"><H2>Key Achievements </H2><UL><LI>The two required devices were cloned from the specified parts in the registry</LI><LI>Fluorescence measurements were obtained from a plate reader and flow cytometer and corresponded with expected resuults.
                                    </LI><LI> Fluorescence was expressed in absolute units of moles of fluorescein or relative to standard beads </LI><LI>Cell-to-cell variation of the devices was explored using the flow-cytometer data</LI></UL></DIV></DIV></SECTION><SECTION id="introduction"><H2>Introduction</H2><P>This year, iGEM invited all the competing teams around the world to measure fluorescence from the same three genetic devices for GFP expression. This exciting experiment will allow the robustness of parts against a variety of different protocols and lab techniques to be assessed. Since the devices are the same, data should match between different labs. In synthetic biology, it is important that part characterisation is be consistent between different labs to be able create well-defined standard parts. We are excited to contribute to this experiment and look forward to seeing the results of any subsequent analysis of data at the iGEM jamboree!
</P></SECTION><SECTION id="release"><H2>Section I: Provenance and Release</H2><DIV class="accordion"><H3>Who did the actual work to acquire these measurements?</H3><DIV><P>Xenia Spencer-Milnes</P></DIV><H3>What other people should be credited for these measurements?</H3><DIV><P>Catherine Ainsworth</P></DIV><H3>On what dates were the protocols run and the measurements taken?</H3><DIV><P>Cloning of constructs was confirmed by 5th September 2014. Transformant plates were from 10-13th September 2014. 16th September 2014 - all samples on flow cytometer and plate reader apart from additional fluorescein standard curve which was measured on 19th September 2014
                                </P></DIV><H3>Do all persons involved consent to the inclusion of this data in publications derived from the iGEM interlab study</H3><DIV><P>Yes</P></DIV></DIV></SECTION><SECTION id="protocols"><H2>Section II: Protocol</H2><DIV class="accordion"><H3>What protocol did you use to prepare samples for measurement?</H3><DIV><P>The two construct built in house were cloned by biobrick cloning to insert E0240 as a back insert into the promoter with backbone. This resulted in the assembly scar sequence TACTAGAG between the promoter and the RBS. Assembly was confirmed by sequencing (Source Bioscience). We used the K823012 part for the J23115 promoter which contains the two mismatched basepairs compared to the original J23115 sequence. 2 mL of LB plus appropriate antibiotic (kanamycin final concentration 50 ug/ml, chloramphenicol 34 ug/ml) in a 14 ml polypropylene round-bottom tube (BD Falcon) was inoculated with a single colony from E.coli DH10B transformant plates. For each construct (labelled ‘A’, ‘B’, ‘C’ and ‘N’ for existing J23101 device [pSB3K3], our cloned J23101 device [pSB1C3], our cloned J23115 device [pSB1C3] and E0240 [pSB1C3] without promoter respectively), three colonies (labelled ‘1’, ‘2’, ‘3’) were picked and grown separately. Cultures were grown in a 37degC growth room at ~230 rpm (VWR Standard analog shaker speed setting 7). Additionally, a non-inoculated LB+antibiotic (chloramphenicol only) tube was grown alongside as a contamination control (labelled ‘O’). Extra media + antibiotic (for chloramphenicol only) was also kept aside for background controls.
                                </P></DIV><H3>What sort of instrument did you use to acquire measurements?</H3><DIV><FIGURE class="content-image image-right"><FIGCAPTION>Figure 1. The flow cytometer system used in the characterisation experiment.</FIGCAPTION></FIGURE><UL><LI>What is the model and manufacturer?
                                        </LI><LI><EM>Plate reader: </EM>(BioTEX Synergy HT) and accompanying software (KC4 v3.2).</LI><LI><EM>Flow cytometer: </EM>FACSCan with a BD FACS Flow Supply System and an Automated Multiwell Sampler (AMS) (Cytek Development) as pictured below, controlled by BD CellQuest™ software on a Mac OS 9.2 computer (figure 1).</LI></UL><LI>How is it configured for your measurements?
                                        </LI><LI><EM>Plate reader: </EM>this was used to measure absorbance at 600 nm, and fluorescence (excitation 485 nm, emission 528 nm, sensitivity 55, optics from top of plate, filter set 1).</LI><LI><EM>Flow cytometer: </EM>this was used to measure fluorescence of individual cells. Each read was terminated at either 45s or 200,000 total events. Laser detection channel FL1 (488 nm) was used with sensitivity at 600 V and optical filter 530/30. Forward-scatter (FSC) detector setting E01, side-scatter (SSC) setting 375. All samples were run on LOW flow setting. The appropriate sensitivity voltage was determined using test wells to confirm that fluorescence peaks across both the high and low strength promoters could be seen clearly.
                                            </LI></DIV><H3>What protocol did you use to take measurements?</H3><DIV><H4>Plate Reader</H4><P>Preparation of samples were as follows: 100 ul cells from each culture, overnight control and media control were transferred in triplicate to a black, flat-bottomed, 96-well Greiner plate on ice. Initially, a sodium fluorescein control was added to the plate in duplicate, at a concentration of 500 ng/ml and final volume 100 ul, by serial dilution in Dulbecco’s Phosphate Buffered Saline pH 7.37 (DPBS) from a 500 ug/ml stock. However, it was decided the readings from this could be improved as a control and a separate control plate was set up at a later date containing varying concentrations (500, 375, 250, 125, 50, 25, 10, 5, 0 ng/ml and PBS only) in triplicate at a final volume of 100 ul. This was set up by pipetting in serial dilution from 3x 1/100 (for the &amp;gt 125 ng/ml concentrations) and 3x 1/1000 (for the &amp;lt 50 ng/ml concentrations) aliquots of 500 ng/ml stock. Confirming a linear relationship between fluorescence of sodium fluorescein at different concentrations and calculating the conversion factor as an average across the different concentrations will provide a more accurate control to convert to absolute fluorescence. The plate was read in the plate reader using the settings described previously. Data was transferred to Excel (Microsoft Office 2007) for analysis.</P><H4>Flow Cytometer</H4><P>Preparation of samples was as follows: Each culture or control was transferred to the plate in triplicate (labelled ‘I’, ‘II’, ‘III’). In a 96-well COSTAR plate, A, B, C and N cultures were pipetted out to a 1/100 concentration using a serial dilution from 1/10, with a total volume of 200 ul per well. Test wells were used to determine the appropriate voltage sensitivity for the the fluorescence read (data is not included). 200 ul of each media and overnight incubation control was added to the plate in triplicate. A well for the standard beads was prepared initially with just 190 ul dH20. The plate layout is shown in figure 2.
                                </P><P>General use of the flow cytometer followed the Standard Operating Procedure (SOP) document specific for the machine, as written by the lab. Settings are as described previously. The plate was loaded into the AMS, and each well was measured for either 45s or up to 200,000 events. Before the run of the samples, wells were pipetted up and down so ensure cells were suspended in solution. Beads (10 ul) were only added to the well (190 ul dH20) just before the read took place, and with vigorous pipetting to ensure they were evenly distributed in the well. Data was analysed using Cyflogic v.1.2.1. then Excel (Microsoft Office 2007).</P><FIGURE class="content-image"><TABLE class="tg image-full"><TBODY><TR><TH class="tg-e3zv">1</TH><TH class="tg-e3zv">2</TH><TH class="tg-e3zv">3</TH><TH class="tg-e3zv">4</TH><TH class="tg-e3zv">5</TH><TH class="tg-e3zv">6</TH><TH class="tg-e3zv">7</TH><TH class="tg-e3zv">8</TH><TH class="tg-e3zv">9</TH><TH class="tg-e3zv">10</TH><TH class="tg-e3zv">11</TH><TH class="tg-e3zv">12</TH></TR><TR><TD class="tg-e3zv">A</TD><TD class="tg-ojj3">A1T-I</TD><TD class="tg-ojj3">B1T-I</TD><TD class="tg-ojj3">C1T-I</TD><TD class="tg-ojj3">N1T-I</TD></TR><TR><TD class="tg-e3zv">B</TD><TD class="tg-ojj3">A1T-II</TD><TD class="tg-dc8v">Beads</TD><TD class="tg-ojj3">B1T-II</TD><TD class="tg-ojj3">C1T-II</TD><TD class="tg-ojj3">N1T-II</TD></TR><TR><TD class="tg-e3zv">C</TD><TD class="tg-ojj3">A1T-III</TD><TD class="tg-ojj3">B1T-III</TD><TD class="tg-ojj3">C1T-III</TD><TD class="tg-ojj3">N1T-III</TD></TR><TR><TD class="tg-e3zv">D</TD></TR><TR><TD class="tg-e3zv">E</TD><TD class="tg-r535">A1-I</TD><TD class="tg-97fa">A2-I</TD><TD class="tg-97fa">A3-I</TD><TD class="tg-zpcz">B1-I</TD><TD class="tg-zpcz">B2-I</TD><TD class="tg-zpcz">B3-I</TD><TD class="tg-nb4k">C1-I</TD><TD class="tg-nb4k">C2-I</TD><TD class="tg-nb4k">C3-I</TD><TD class="tg-jazs">N1-I</TD><TD class="tg-jazs">N2-I</TD><TD class="tg-jazs">N3-I</TD></TR><TR><TD class="tg-e3zv">F</TD><TD class="tg-97fa">A1-II</TD><TD class="tg-97fa">A2-II</TD><TD class="tg-97fa">A3-II</TD><TD class="tg-zpcz">B1-II</TD><TD class="tg-zpcz">B2-II</TD><TD class="tg-zpcz">B3-II</TD><TD class="tg-nb4k">C1-II</TD><TD class="tg-nb4k">C2-II</TD><TD class="tg-nb4k">C3-II</TD><TD class="tg-jazs">N1-II</TD><TD class="tg-jazs">N2-II</TD><TD class="tg-jazs">N3-II</TD></TR><TR><TD class="tg-e3zv">G</TD><TD class="tg-97fa">A1-III</TD><TD class="tg-97fa">A2-III</TD><TD class="tg-97fa">A3-III</TD><TD class="tg-zpcz">B1-III</TD><TD class="tg-zpcz">B2-III</TD><TD class="tg-zpcz">B3-III</TD><TD class="tg-nb4k">C1-III</TD><TD class="tg-nb4k">C2-III</TD><TD class="tg-nb4k">C3-III</TD><TD class="tg-jazs">N1-III</TD><TD class="tg-jazs">N2-III</TD><TD class="tg-jazs">N3-III</TD></TR><TR><TD class="tg-e3zv">H</TD><TD class="tg-m6v9">M-I</TD><TD class="tg-m6v9">M-II</TD><TD class="tg-m6v9">M-III</TD><TD class="tg-qaud">O-I</TD><TD class="tg-qaud">O-II</TD><TD class="tg-qaud">O-III</TD></TR></TBODY></TABLE><FIGCAPTION>Figure 2: Layout of sample plate for flow cytometer to characterise fluorescence of three different constructs expressing GFP.</FIGCAPTION></FIGURE><UL><LI><SPAN style="background-color:#cccccc">Test wells</SPAN> - to determine appropriate voltage (sensitivity).</LI><LI><SPAN style="background-color:#ff0000">Excluded</SPAN> - voltage was changed while sample was running therefore results are invalid ‘
                                    </LI><LI><SPAN style="background-color:#6d9eeb">‘A’ wells</SPAN> - Existing J23101 device, cultures ‘1’, ‘2’, ‘3’, sampled ‘I’, ‘II’, ‘III’</LI><LI><SPAN style="background-color:#b6d7a8">‘B’ wells</SPAN> - Cloned J23101 device, cultures ‘1’, ‘2’, ‘3’, sampled ‘I’, ‘II’, ‘III’</LI><LI><SPAN style="background-color:#ffd966">‘C’ wells</SPAN> - Cloned J23115 device, cultures ‘1’, ‘2’, ‘3’, sampled ‘I’, ‘II’, ‘III’</LI><LI><SPAN style="background-color:#8e7cc3">‘N’ wells</SPAN> - negative control device (no promoter), cultures ‘1’, ‘2’, ‘3’, sampled ‘I’, ‘II’, ‘III’</LI><LI><SPAN style="background-color:#f9cb9c">‘M’ wells</SPAN> - Media plus antibiotic (chloramphenicol only) background control, sampled ‘I’, ‘II’, ‘III’</LI><LI><SPAN style="background-color:#ea9999">‘O’ wells</SPAN> - Non-inoculated culture (chloramphenicol) contamination control, sampled ‘I’, ‘II’, ‘III’
                                    </LI><LI><SPAN style="background-color:#ffff00">Beads</SPAN> - Sphero™ AccuCount Fluorescent particles 5.1 uM, ~1E6 particles/ml as reference for absolute fluorescence. 10 ul in 190 ul dH20.</LI></UL></DIV><H3>What method is used to determine whether to include or exclude each sample from the data set?</H3><DIV><P><EM>Plate reader: all samples were included in the data set.</EM></P><DIV class="pure-g"><DIV class="pure-u-1-2"><P><EM> Flow cytometer</EM>: reads were gated based on side scatter (SSC) to select cells and exclude cell debris, and based on reducing the tails of the distribution of the SSC histogram (e.g. see appendix figures 2-5.1, 2-5.2 and 2-5.3). SSC gating was the same for every sample. Previous experiments in the lab have suggested forward scatter has no relationship with cell population, and this can also be seen qualitatively in our results where the events are evenly distributed for forward scatter (e.g. appendix figures 2-5.1), so this was not gated. Fluorescence was also gated since previous experiments in the lab have suggested that the population at very low or no fluorescence is cell debris rather than non-expressing cells. These gates were adjusted manually for each population to include the main cluster of cells. The resulting gating gives a log-normal distribution, hence the geometric mean is calculated and this value is used as the fluorescence value for that well. For the negative controls, only SSC gating was included since the population with cell debris overlapped in the fluorescence. The beads for fluorescence were gated on both side-scatter and fluorescence to obtain the correct population, based on previous experience, and that they should be a tight group with high SSC and FSC (figure 3A,3B,3C). Note from the bead graphs it can be seen that the water used to dilute the beads was likely contaminated. However, the population of beads is still clearly visible. Additionally, one replicate for culture A1 was not included in the analysis because voltage sensitivity was adjusted whilst the sample was running. Examples of the gating for each construct and the negative construct are shown in the appendix.</P></DIV><DIV class="pure-u-1-2"><FIGURE class="content-image"><FIGCAPTION>Figure 3A: Dot plot showing tight population of standard beads (boxed). Remaining events are thought to be potential contamination from diluent water.</FIGCAPTION></FIGURE><FIGURE class="content-image"><FIGCAPTION>Figure 3B: Gating of standard beads by side scatter (SSC) and fluorescence (FL1) showing tight resolution population</FIGCAPTION></FIGURE><FIGURE class="content-image"><FIGCAPTION>Figure 3C: Histogram of gated standard beads showing high resolution peak. The geometric mean fluorescence is 1391.51.</FIGCAPTION></FIGURE></DIV></DIV></DIV><H3>What exactly were the controls that you used?</H3><DIV><P>A negative control for background cell fluorescence was included as cells containing the device E0240 but without a promoter, to mimic burden of the promoter. The LB media and a non-inoculated overnight culture (using antibiotics chloramphenicol only) were used to control for media-only background, identify any possible contamination and see the debris pattern. In order to obtain absolute values for fluorescence, set standards were measured alongside the samples.</P><P>For the flow cytometer: Sphero™ AccuCount Fluorescent particles 5.1 uM, ~1E6 particles/ml. 10 ul were added to 190 ul dH20.</P><FIGURE class="content-image image-right"><FIGCAPTION>Figure 4: Confirmation of linear relationship between sodium fluorescein concentration and measured fluorescence on a BioTEX Synergy HT plate reader. Gradient of this calibration curve is conversion factor for fluorescence of measured by this plate reader to equivalent ng/ml sodium fluorescein.</FIGCAPTION></FIGURE><P>For the plate reader: The sodium fluorescein control plate was set up as described previously, and measured with same settings as for characterisation samples. A linear relationship was confirmed (figure 4) showing that it is an accurate way representing absolute fluorescence in terms of equivalent ng/ml of sodium fluorescein, at least for the range covered in this calibration curve. The data was processed as follows: 1) Raw fluorescence data was obtained from plate reader, 2) Subtraction of arithmetic mean average of PBS-only wells, 3) Arithmetic mean average fluorescence calculated for each concentration of fluorescein, 4) Mean fluorescence for the concentration of fluorescein for each concentration measured (FL per ng/ml), 5) Arithmetic mean average of FL per ng/ml gives conversion factor for fluorescence of any sample on the plate reader to equivalent ng/ml of sodium fluorescein. N.B. in the calculation of the conversion factor, the 500 ng and 5 ng points were excluded due to pipetting error accumulation for the 5 ng point being small volumes from the serial dilution, and pipetting error for the 500 ng point which was not the starting well for the serial dilution. Data table can be seen in the appendix.</P></DIV><H3>What quantities were measured?</H3><DIV><H4>Plate reader</H4><P>Green fluorescence (485 ex/528 em), absorbance at 600 nm</P><H4>Flow cytometer</H4><P>Green fluorescence (optical filter 530/30, FL1 laser wavelength 488 nm). Also side-scatter (SSC) and forward scatter (FSC) of each cell was recorded as it passed through the laser</P></DIV><H3>How much time did it take to acquire each set of measurements? </H3><DIV><P>Preparation of plates for each machine took between 15 and 30 minutes.</P><H4>Plate reader</H4><P>the read for absorbance is under 60 s for the whole plate, and after a bulb-warming period of 180 s the read for fluorescence is also under 90 s</P><H4>Flow cytometer</H4><P>Priming the machine can take approximately 10 minutes. Each sample takes 45 seconds, with a wash of ~10 s between wells. For a whole plate of 43 samples in total including controls and beads, this takes approximately 43 minutes. The flow cytometer also records side scatter (SSC) which can be indicative of complexity, and forward scatter (FSC) which for this experiment is irrelevant.</P></DIV><H3>How much does it cost to acquire a set of measurements?</H3><DIV><P>The cost of acquiring these measurements was very low because small aliquots were used of reagents already purchased for other purposes. Similarly the machines used are expensive to purchase and maintain but are used primarily by other users and these experiments were conducted during gaps in this other research. Cost to consider however could include:</P><UL><LI>Reagents: Beads (Sphero™ AccuCount Fluorescent Particles 5.1 uM, ~1E6 particles/ml) (3x 10 ul), Sodium fluorescein (3 x 1 ul), PBS (99 ul x 3), 1x COSTAR plate, 1x black plate Greiner, flow cytometer reagents (system buffer, washing reagents etc).</LI><LI>Electricity and maintenance of the cytometer and AMS</LI><LI>Time of staff to train in using or to maintain the machines</LI></UL></DIV><H3>What are the practical limits on the number or rate of measurements taken with this instrument and protocol?</H3><DIV><P>With the plate reader, one 96-well plate could be measured approximately every 2-3 minutes however accounting for plate preparation time this would be approximately 15 minutes. For the flow cytometer there could be one read approximately every hour for one 96-well plate.</P></DIV></DIV></SECTION><SECTION id="measured"><H2>Section III: Measured Quantities</H2><DIV class="accordion"><H3>Units</H3><DIV><H4>What are the units of the measurement?</H4><P>The absorbance and fluorescence have arbitrary units. However, after controlling for background fluorescence (plate reader and flow cytometer) and normalising for absorbance (plate reader only), the data is compared to either standard beads (flow cytometer) or sodium fluorescein (plate reader) to obtain fluorescence in terms of these standards. The units from the plate reader will be equivalent ng/ml fluorescein per OD 600 unit, and for the flow cytometer will be fluorescence as a percentage of the standard bead mean fluorescence. This theoretically could be converted into molecules of GFP with a further calibration. OD 600 units can be converted to absolute cell number by calibrating using a flow cytometer, or by creating a standard curve based on counting colonies on plates of dilutions of different OD 600 values.</P><H4>What is the equivalent unit expressed as a combination of the seven SI base units?</H4><P>For the plate reader, absolute fluorescence could be expressed as equivalent moles of sodium fluorescein using the MW of 376.27. For the flow cytometer, absolute fluorescence is relative to the standard beads, so an SI unit is not applicable.
                                </P></DIV><H3>Precision</H3><DIV><H4>What is the range of possible measured values for this quantity, using your instrument as configured for these measurements?</H4><P>For fluorescence on both the plate reader and the flow cytometer, the sensitivity can be adjusted depending on the sample. For this experiment test wells were done to confirm that for the sensitivity selected, this would include the whole range of the data. For absorbance on the plate reader, readings are accurate between 0 and 1</P><H4>What are the significant figures for these measurement?</H4><P>The plate reader measures absorbance to 3 s.f. For fluorescence this reaches 5 s.f. For the flow cytometer the raw data file records 7 s.f., but we used 3-4.
                                </P><H4>Is the precision the same across the entire range? If not, how does it differ?</H4><P>The precision is the same across the range for the flow cytometer and plate reader as proven by previous lab experiments where different sensitivity settings were tested.</P><H4>How did you determine these answers?</H4><P>For the flow cytometer, this was determined previously in the lab using standard beads of different known fluorescences. For the plate reader, previous tests had compared absorbance readings to a spectrophotometer.</P></DIV><H3>Accuracy</H3><DIV><H4>When was the instrument last calibrated?</H4><P>For the flow cytometer, calibration occurs with every run by using the beads. The plate reader was calibrated approximately 6 months ago.</P><H4>How was the instrument calibrated?</H4><P>The beads are run with every flow cytometer plate reading at the end of the experiment. As the beads are commercially available they can serve as a standard for comparing inter-lab variation. For the plate reader, absorbance at 600 nm measurements are compared to a JENWAY Genova spectrophotometer to obtain a standard curve for converting from plate-reader absorbance to OD 600 with a path-length of 1cm.</P></DIV></DIV></SECTION><SECTION id="measurments"><H2>Section IV: Measurements</H2><DIV class="accordion"><H3>Plate Reader: Direct Measurements</H3><DIV><P>Absorbance at 600 nm was measured three times for each of the three cultures, alongside controls for media and contamination background (table 1).</P><FIGURE class="content-image"><TABLE class="tg image-full"><TBODY><TR><TH class="tg-e3zv">1</TH><TH class="tg-e3zv">2</TH><TH class="tg-e3zv">3</TH><TH class="tg-e3zv">4</TH><TH class="tg-e3zv">5</TH><TH class="tg-e3zv">6</TH><TH class="tg-e3zv">7</TH><TH class="tg-e3zv">8</TH><TH class="tg-e3zv">9</TH><TH class="tg-e3zv">10</TH><TH class="tg-e3zv">11</TH><TH class="tg-e3zv">12</TH></TR><TR><TD class="tg-e3zv">A</TD><TD class="tg-97fa">0.645</TD><TD class="tg-97fa">0.592</TD><TD class="tg-97fa">0.646</TD><TD class="tg-zpcz">0.588</TD><TD class="tg-zpcz">0.496</TD><TD class="tg-zpcz">0.543</TD><TD class="tg-nb4k">0.621</TD><TD class="tg-nb4k">0.581</TD><TD class="tg-nb4k">0.585</TD><TD class="tg-jazs">0.583</TD><TD class="tg-jazs">0.608</TD><TD class="tg-jazs">0.616</TD></TR><TR><TD class="tg-e3zv">B</TD><TD class="tg-97fa">0.643</TD><TD class="tg-97fa">0.612</TD><TD class="tg-97fa">0.644</TD><TD class="tg-zpcz">0.552</TD><TD class="tg-zpcz">0.508</TD><TD class="tg-zpcz">0.528</TD><TD class="tg-nb4k">0.622</TD><TD class="tg-nb4k">0.605</TD><TD class="tg-nb4k">0.623</TD><TD class="tg-jazs">0.583</TD><TD class="tg-jazs">0.612</TD><TD class="tg-jazs">0.593</TD></TR><TR><TD class="tg-e3zv">C</TD><TD class="tg-97fa">0.633</TD><TD class="tg-97fa">0.604</TD><TD class="tg-97fa">0.637</TD><TD class="tg-zpcz">0.529</TD><TD class="tg-zpcz">0.5</TD><TD class="tg-zpcz">0.547</TD><TD class="tg-nb4k">0.599</TD><TD class="tg-nb4k">0.622</TD><TD class="tg-nb4k">0.646</TD><TD class="tg-jazs">0.629</TD><TD class="tg-jazs">0.634</TD><TD class="tg-jazs">0.606</TD></TR><TR><TD class="tg-e3zv">D</TD><TD class="tg-m6v9">0.174</TD><TD class="tg-m6v9">0.164</TD><TD class="tg-m6v9">0.229</TD><TD class="tg-qaud">0.233</TD><TD class="tg-qaud">0.2</TD><TD class="tg-qaud">0.236</TD></TR></TBODY></TABLE><FIGCAPTION>Table 1: Absorbance at 600 nm for 3 devices with different promoters coding for GFP as read in a BioTEX Synergy HT plate reader. Units are arbitrary.</FIGCAPTION></FIGURE><UL><LI><SPAN style="background-color:#6d9eeb">‘A’ wells</SPAN> - Existing J23101 device. Three cultures (horizontal) samples three times (vertical)</LI><LI><SPAN style="background-color:#b6d7a8">‘B’ wells</SPAN> - Cloned J23101 device. Three cultures (horizontal) samples three times (vertical)</LI><LI><SPAN style="background-color:#ffd966">‘C’ wells</SPAN> - Cloned J23115 device. Three cultures (horizontal) samples three times (vertical)</LI><LI><SPAN style="background-color:#8e7cc3">‘N’ wells</SPAN> - negative control device (no promoter). Three cultures (horizontal) samples three times (vertical)
                                    </LI><LI><SPAN style="background-color:#f9cb9c">‘M’ wells</SPAN> - Media plus antibiotic (chloramphenicol only). Sampled three times.</LI><LI><SPAN style="background-color:#ea9999">‘O’ wells</SPAN> - Non-inoculated culture (chloramphenicol). Sampled three times
                                    </LI></UL><P>Fluorescence was also measured (table 2), using settings as described previously. The negative cultures account for 7.75% total fluorescence for ‘A’ wells (blue), 1.15% total fluorescence for ‘B’ wells (green) and 23.2% total fluorescence for ‘C’ wells.</P><FIGURE class="content-image"><TABLE class="tg image-full"><TBODY><TR><TH class="tg-e3zv">1</TH><TH class="tg-e3zv">2</TH><TH class="tg-e3zv">3</TH><TH class="tg-e3zv">4</TH><TH class="tg-e3zv">5</TH><TH class="tg-e3zv">6</TH><TH class="tg-e3zv">7</TH><TH class="tg-e3zv">8</TH><TH class="tg-e3zv">9</TH><TH class="tg-e3zv">10</TH><TH class="tg-e3zv">11</TH><TH class="tg-e3zv">12</TH></TR><TR><TD class="tg-e3zv">A</TD><TD class="tg-97fa">6102</TD><TD class="tg-97fa">5914</TD><TD class="tg-97fa">7191</TD><TD class="tg-zpcz">44832</TD><TD class="tg-zpcz">36996</TD><TD class="tg-zpcz">38835</TD><TD class="tg-nb4k">3182</TD><TD class="tg-nb4k">2940</TD><TD class="tg-nb4k">3023</TD><TD class="tg-jazs">1625</TD><TD class="tg-jazs">1519</TD><TD class="tg-jazs">1436</TD></TR><TR><TD class="tg-e3zv">B</TD><TD class="tg-97fa">7359</TD><TD class="tg-97fa">6652</TD><TD class="tg-97fa">7885</TD><TD class="tg-zpcz">45460</TD><TD class="tg-zpcz">37231</TD><TD class="tg-zpcz">39344</TD><TD class="tg-nb4k">3236</TD><TD class="tg-nb4k">3005</TD><TD class="tg-nb4k">3156</TD><TD class="tg-jazs">1591</TD><TD class="tg-jazs">1712</TD><TD class="tg-jazs">1595</TD></TR><TR><TD class="tg-e3zv">C</TD><TD class="tg-97fa">7221</TD><TD class="tg-97fa">6751</TD><TD class="tg-97fa">7661</TD><TD class="tg-zpcz">43861</TD><TD class="tg-zpcz">36512</TD><TD class="tg-zpcz">39185</TD><TD class="tg-nb4k">3052</TD><TD class="tg-nb4k">3133</TD><TD class="tg-nb4k">3178</TD><TD class="tg-jazs">1719</TD><TD class="tg-jazs">1700</TD><TD class="tg-jazs">1576</TD></TR><TR><TD class="tg-e3zv">D</TD><TD class="tg-m6v9">911</TD><TD class="tg-m6v9">1015</TD><TD class="tg-m6v9">1224</TD><TD class="tg-qaud">1356</TD><TD class="tg-qaud">1147</TD><TD class="tg-qaud">1292</TD><TD class="tg-dc8v">50872</TD><TD class="tg-dc8v">78149</TD></TR></TBODY></TABLE><FIGCAPTION>Table 2: Fluorescence (485 ex/528 em) for 3 devices with different promoters coding for GFP as read in a BioTEX Synergy HT plate reader. Units are arbitrary.</FIGCAPTION></FIGURE><UL><LI><SPAN style="background-color:#6d9eeb">‘A’ wells</SPAN> - Existing J23101 device. Three cultures (horizontal) samples three times (vertical)</LI><LI><SPAN style="background-color:#b6d7a8">‘B’ wells</SPAN> - Cloned J23101 device. Three cultures (horizontal) samples three times (vertical)</LI><LI><SPAN style="background-color:#ffd966">‘C’ wells</SPAN> - Cloned J23115 device. Three cultures (horizontal) samples three times (vertical)</LI><LI><SPAN style="background-color:#8e7cc3">‘N’ wells</SPAN> - negative control device (no promoter). Three cultures (horizontal) samples three times (vertical)
                                    </LI><LI><SPAN style="background-color:#f9cb9c">‘M’ wells</SPAN> - Media plus antibiotic (chloramphenicol only). Sampled three times.</LI><LI><SPAN style="background-color:#ea9999">‘O’ wells</SPAN> - Non-inoculated culture (chloramphenicol). Sampled three times
                                    </LI><LI><SPAN style="background-color:#ffff00">Sodium fluorescein control wells.</SPAN> - Note that these wells were not used in analysis of the data in the end, please see the described sodium fluorescein control plate instead for more details.

                                    </LI></UL></DIV><H3>
Plate Reader: Derived Measurements</H3><DIV><P>Data was processed as follows:</P><OL><LI>Background absorbance was removed by subtracting the average of M and O cells.</LI><LI>These absorbances were multiplied by the conversion factor (8.67) to convert to OD 600 equivalent on a spectrophotometer with a path length of 1 cm. The conversion factor originates from calibration of the plate reader to a spectrophotometer as described previously</LI><LI>Background fluorescence was controlled for by subtracting the average of M and O cells.</LI><LI>Background controlled fluorescence was then divided by background controlled OD 600 to obtain fluorescence per OD unit (FL/OD)</LI><LI>The average FL/OD of the negative control cultures with no promoter was subtracted from the FL/OD of the other constructs to remove background cell fluorescence. The resulting value is fluorescence per OD 600 (arbitrary units).</LI><LI>Dividing these values by the conversion factor as determined from the sodium fluorescein control plate (126.02) gives the absolute fluorescence as the equivalent ng/ml sodium fluorescein per OD unit (table 3, 4). This absolute value should be comparable across different machines which are calibrated in the same way.</LI></OL><FIGURE class="content-image"><TABLE class="tg image-full"><TBODY><TR><TH class="tg-e3zv">A1</TH><TH class="tg-e3zv">A2</TH><TH class="tg-e3zv">A3</TH><TH class="tg-e3zv">B1</TH><TH class="tg-e3zv">B2</TH><TH class="tg-e3zv">B3</TH><TH class="tg-e3zv">C1</TH><TH class="tg-e3zv">C2</TH><TH class="tg-e3zv">C3</TH></TR><TR><TD class="tg-e3zv">I</TD><TD class="tg-031e">9.28</TD><TD class="tg-031e">10.25</TD><TD class="tg-031e">11.52</TD><TD class="tg-031e">103.61</TD><TD class="tg-031e">112.08</TD><TD class="tg-031e">101.3</TD><TD class="tg-031e">3.44</TD><TD class="tg-031e">3.32</TD><TD class="tg-031e">3.48</TD></TR><TR><TD class="tg-e3zv">II</TD><TD class="tg-031e">11.96</TD><TD class="tg-031e">11.36</TD><TD class="tg-031e">13.03</TD><TD class="tg-031e">116.16</TD><TD class="tg-031e">108.3</TD><TD class="tg-031e">107.51</TD><TD class="tg-031e">3.55</TD><TD class="tg-031e">3.21</TD><TD class="tg-031e">3.36</TD></TR><TR><TD class="tg-e3zv">III</TD><TD class="tg-031e">11.97</TD><TD class="tg-031e">11.84</TD><TD class="tg-031e">12.78</TD><TD class="tg-031e">119.98</TD><TD class="tg-031e">109.03</TD><TD class="tg-031e">101.04</TD><TD class="tg-031e">3.38</TD><TD class="tg-031e">3.32</TD><TD class="tg-031e">3.18</TD></TR></TBODY></TABLE><FIGCAPTION>Table 3: Absolute fluorescence in terms of ng/ml fluorescein per OD unit of three different GFP encoding constructs with different promoters and plasmid copy number</FIGCAPTION></FIGURE><FIGURE class="content-image"><TABLE class="tg image-full"><TBODY><TR><TH class="tg-e3zv">A</TH><TH class="tg-e3zv">B</TH><TH class="tg-e3zv">C</TH></TR><TR><TD class="tg-e3zv">Arithmetic mean</TD><TD class="tg-031e">11.55</TD><TD class="tg-031e">108.78</TD><TD class="tg-031e">3.36</TD></TR><TR><TD class="tg-e3zv">S.D.</TD><TD class="tg-031e">1.17</TD><TD class="tg-031e">6.47</TD><TD class="tg-031e">0.12</TD></TR><TR><TD class="tg-e3zv">C.V</TD><TD class="tg-031e">10.15</TD><TD class="tg-031e">5.95</TD><TD class="tg-031e">3.56</TD></TR><TR><TD class="tg-e3zv">Ratio</TD><TD class="tg-031e">3.44</TD><TD class="tg-031e">32.39</TD><TD class="tg-031e">1.00</TD></TR></TBODY></TABLE><FIGCAPTION>Table 4: Comparison of absolute fluorescence in terms of equivalent ng/ml sodium fluorescein for three different constructs expressing GFP. S.D-standard deviation. C.V-coefficient of variation. Ratio - cultures normalised to C.</FIGCAPTION></FIGURE></DIV><H3>Flow Cytometer: Direct Measurements</H3><DIV><P>The negative cultures account for 1.34% total fluorescence for ‘A’ wells (blue), 0.14% total fluorescence for ‘B’ wells (green) and 6.01% total fluorescence for ‘C’ wells. As expected fluorescence is highest for the stronger J23101 promoter (A and B) and for the higher copy number plasmid (B).</P><FIGURE class="content-image"><TABLE class="tg image-full"><TBODY><TR><TH class="tg-e3zv">A1</TH><TH class="tg-e3zv">A2</TH><TH class="tg-e3zv">A3</TH><TH class="tg-e3zv">B1</TH><TH class="tg-e3zv">B2</TH><TH class="tg-e3zv">B3</TH><TH class="tg-e3zv">C1</TH><TH class="tg-e3zv">C2</TH><TH class="tg-e3zv">C3</TH><TH class="tg-e3zv">N1</TH><TH class="tg-e3zv">N2</TH><TH class="tg-e3zv">N3</TH></TR><TR><TD class="tg-e3zv">I</TD><TD class="tg-031e">N/A</TD><TD class="tg-031e">79.71</TD><TD class="tg-031e">70.09</TD><TD class="tg-031e">642.79</TD><TD class="tg-031e">721.65</TD><TD class="tg-031e">670.06</TD><TD class="tg-031e">15.1</TD><TD class="tg-031e">16.77</TD><TD class="tg-031e">16.56</TD><TD class="tg-031e">1.02</TD><TD class="tg-031e">1.01</TD><TD class="tg-031e">1.02</TD></TR><TR><TD class="tg-e3zv">II</TD><TD class="tg-031e">73.92</TD><TD class="tg-031e">78.99</TD><TD class="tg-031e">71.38</TD><TD class="tg-031e">676.39</TD><TD class="tg-031e">784.49</TD><TD class="tg-031e">735.92</TD><TD class="tg-031e">15.57</TD><TD class="tg-031e">17.82</TD><TD class="tg-031e">17.02</TD><TD class="tg-031e">1.02</TD><TD class="tg-031e">1.02</TD><TD class="tg-031e">1.02</TD></TR><TR><TD class="tg-e3zv">III</TD><TD class="tg-031e">75.21</TD><TD class="tg-031e">79.46</TD><TD class="tg-031e">72.19</TD><TD class="tg-031e">682.32</TD><TD class="tg-031e">703.4</TD><TD class="tg-031e">743.01</TD><TD class="tg-031e">16.1</TD><TD class="tg-031e">18.72</TD><TD class="tg-031e">17.75</TD><TD class="tg-031e">1.02</TD><TD class="tg-031e">1.03</TD><TD class="tg-031e">1.01</TD></TR></TBODY></TABLE><FIGCAPTION>Table 5: Fluorescence for each well measured using a flow cytometer and determined using geometric mean (A,B,C) or arithmetic mean (N) for gated events in each well. Excluded sample A1-I because voltage was changed whilst sample was running so is not valid. Units are arbitrary.
                                    </FIGCAPTION></FIGURE></DIV><H3>Flow Cytometer: Derived Measurements</H3><DIV><OL><LI>Background fluorescence was removed by subtracting the negative control cultures.
                                    </LI><LI>These values were divided by the fluorescence of the beads (1391.51, figure 3C) and multiplying by 100 to obtain the absolute fluorescence as percentage of the standard beads (table 6, 7). This should be comparable across machines using the same standard beads.</LI></OL><FIGURE class="content-image"><TABLE class="tg image-full"><TBODY><TR><TH class="tg-e3zv">A1</TH><TH class="tg-e3zv">A2</TH><TH class="tg-e3zv">A3</TH><TH class="tg-e3zv">B1</TH><TH class="tg-e3zv">B2</TH><TH class="tg-e3zv">B3</TH><TH class="tg-e3zv">C1</TH><TH class="tg-e3zv">C2</TH><TH class="tg-e3zv">C3</TH></TR><TR><TD class="tg-e3zv">I</TD><TD class="tg-031e">N/A</TD><TD class="tg-031e">5.66</TD><TD class="tg-031e">4.96</TD><TD class="tg-031e">46.12</TD><TD class="tg-031e">51.79</TD><TD class="tg-031e">48.08</TD><TD class="tg-031e">1.01</TD><TD class="tg-031e">1.13</TD><TD class="tg-031e">1.12</TD></TR><TR><TD class="tg-e3zv">II</TD><TD class="tg-031e">5.24</TD><TD class="tg-031e">5.60</TD><TD class="tg-031e">5.06</TD><TD class="tg-031e">48.54</TD><TD class="tg-031e">56.30</TD><TD class="tg-031e">52.81</TD><TD class="tg-031e">1.05</TD><TD class="tg-031e">1.21</TD><TD class="tg-031e">1.15</TD></TR><TR><TD class="tg-e3zv">III</TD><TD class="tg-031e">5.33</TD><TD class="tg-031e">5.64</TD><TD class="tg-031e">5.11</TD><TD class="tg-031e">48.96</TD><TD class="tg-031e">50.48</TD><TD class="tg-031e">53.32</TD><TD class="tg-031e">1.08</TD><TD class="tg-031e">1.27</TD><TD class="tg-031e">1.20</TD></TR></TBODY></TABLE><FIGCAPTION>Table 6: Absolute fluorescence of three GFP devices with different promoters or plasmid copy number, in terms of percentage compared to standard beads measured with a flow cytometer
                                    </FIGCAPTION></FIGURE><FIGURE class="content-image"><TABLE class="tg image-full"><TBODY><TR><TH class="tg-e3zv">A</TH><TH class="tg-e3zv">B</TH><TH class="tg-e3zv">C</TH></TR><TR><TD class="tg-e3zv">Mean</TD><TD class="tg-031e">5.33</TD><TD class="tg-031e">50.71</TD><TD class="tg-031e">1.14</TD></TR><TR><TD class="tg-e3zv">S.D.</TD><TD class="tg-031e">0.24</TD><TD class="tg-031e">2.74</TD><TD class="tg-031e">0.08</TD></TR><TR><TD class="tg-e3zv">C.V.</TD><TD class="tg-031e">4.51</TD><TD class="tg-031e">5.4</TD><TD class="tg-031e">6.83</TD></TR><TR><TD class="tg-e3zv">Ratio</TD><TD class="tg-031e">4.68</TD><TD class="tg-031e">44.5</TD><TD class="tg-031e">1</TD></TR></TBODY></TABLE><FIGCAPTION>Table 7: Comparison of absolute fluorescence of three GFP devices as measured using a flow cytometer using standard beads as absolute reference control.
                                    </FIGCAPTION></FIGURE><P>Average absolute fluorescence was lowest for construct C (J23115 promoter). The parts registry page (Anderson et al., 2006) shows that J23101 is 4.7 times stronger than J23115. For the supplied J23101 MeasKit our data shows a similar difference of 3.44 times stronger for J23101 than J23115 for the plate reader and agrees with this when the plate reader is used and 4.68 times stronger when the flow cytometer is used (figure 5). There are differences in absolute fluorescence for the two same J23101 constructs A and B because B is in the higher copy number plasmid pSB1C3.</P></DIV><H3>Comparison of Relative Fluorescence for Plate Reader and Flow Cytometer</H3><DIV><FIGURE class="content-image image-right"><FIGCAPTION>Figure 5: Ratio of mean GFP fluorescence for each construct compared to the weakest construct for plate reader and flow cytometry methods.</FIGCAPTION></FIGURE><P>The flow cytometry method gives higher relative fluorescence compared to the J23115 construct than the plate reader method (figure 5).</P></DIV></DIV></SECTION><SECTION id="extra"><H2>Extra Credit Assignment 1: Measurement of Cell-to-Cell Variation for the Three Devices</H2><P>The method of characterising the parts on the flow cytometer measures the fluorescence of each cell individually. In addition to characterising the absolute fluorescence of the different constructs as above, the data can be used to explore the variance of fluorescence between individual cells, since there is a fluorescence reading for each individual cell. This was done by calculating the geometric mean of the whole population for each construct, and the geometric standard deviation (table 8, figure 6). The geometric standard deviation is a good indication of variance in itself because it is a dimensionless multiplicative factor which in essence describes the percentage error from the mean (table 8).</P><FIGURE class="content-image image-right"><FIGCAPTION>Figure 6. Comparing differences in geometric standard deviation of fluorescence of cells measured on a flow cytometer for three devices expressing GFP.</FIGCAPTION></FIGURE><P>From our data it can be seen that the variance for the ‘B’ constructs (Cloned J23101 in pSB1C3) is much larger than the other two constructs. However, this could be due to the cells experiencing burden due to the high copy number and high strength promoter, as evidenced by their lower cell number per well (N, table 8). For example the ‘A’ and ‘C’ constructs have at least twice the number of cell events than ‘B’ (table 8). Hence looking at differences in variation may not be relevant as the sample sizes are so different and a direct comparison cannot be made.</P><P>Solutions to combat this could be to try and obtain similar OD 600 across all constructs by halting growth of the faster cells by incubating on ice and waiting for the burdened cells to reach the same OD 600. However, this still may not be representative as the lower growth rate can have complex, non-linear and often counter-intuitive relationships with gene expression parameters (Klumpp et al, 2009). It could be possible to try and control growth rate, however this would not be possible except for modifying conditions such as temperature, which would then be an extraneous variable compared to the rest of the constructs to be characterised. A better solution would be to have careful consideration of promoter and backbone copy number, for example a weaker promoter when copy number is high, to ensure burden is not experienced by the cells.</P><FIGURE class="content-image"><TABLE class="tg image-full"><TBODY><TR><TH class="tg-e3zv">GeoMean</TH><TH class="tg-e3zv">GeoStDev</TH><TH class="tg-e3zv">N</TH><TH class="tg-e3zv">Relative N</TH></TR><TR><TD class="tg-e3zv">A</TD><TD class="tg-031e">74.55</TD><TD class="tg-031e">1.68</TD><TD class="tg-031e">252273</TD><TD class="tg-031e">2.12641</TD></TR><TR><TD class="tg-e3zv">B</TD><TD class="tg-031e">711.41</TD><TD class="tg-031e">2.03</TD><TD class="tg-031e">118638</TD><TD class="tg-031e">1</TD></TR><TR><TD class="tg-e3zv">C</TD><TD class="tg-031e">16.76</TD><TD class="tg-031e">1.75</TD><TD class="tg-031e">326587</TD><TD class="tg-031e">2.752803</TD></TR></TBODY></TABLE><FIGCAPTION>Table 8: Comparing cell-to-cell variation of three genetic devices coding for GFP with different promoters and backbones. A- Existing J23101 device, B - Cloned J23101 device, C - Cloned J23115 device.
                            </FIGCAPTION></FIGURE></SECTION><SECTION id="references"><H2>References</H2><OL><LI>Anderson, C., Berkley iGEM Team., (2006). <EM>Anderson Promoter Collection.</EM> Registry of Standard Biological Parts. http://parts.igem.org/Promoters/Catalog/Anderson [accessed 19/09/2014]</LI><LI>Klumpp, S., Zhang, Z., Hwa, T., (2009). <EM>Growth rate-dependent global effects on gene expression in bacteria.</EM> Cell 139(7):1366-75. doi: 10.1016/j.cell.2009.12.001.</LI></OL></SECTION><SECTION id="appendix"><H2>Appendix</H2><DIV class="accordion"><H3>Raw data for fluorescein control plate</H3><DIV><P>Raw data for fluorescein control plate (with background DPBS diluent subtracted) to determine conversion factor of fluorescence to equivalent ng/ul of sodium fluorescein. Conversion factor was 126.02, the average of FL/(ng/ml). As noted previously, the 500 ng and 5 ng readings were excluded from the calculation as were deemed too inaccurate due to pipetting error.</P><FIGURE class="content-image"><TABLE class="tg image-full"><TBODY><TR><TH class="tg-e3zv">ng/ml</TH><TH class="tg-e3zv">Replicate I</TH><TH class="tg-e3zv">Replicate II</TH><TH class="tg-e3zv">Replicate III</TH><TH class="tg-e3zv">Mean</TH><TH class="tg-e3zv">S.D.</TH><TH class="tg-e3zv">C.V.</TH><TH class="tg-e3zv">FL/(ng/ml)</TH></TR><TR><TD class="tg-e3zv">500</TD><TD class="tg-031e">39779</TD><TD class="tg-031e">43169</TD><TD class="tg-031e">58234</TD><TD class="tg-031e">47061</TD><TD class="tg-031e">9823.72</TD><TD class="tg-031e">20.87</TD><TD class="tg-031e">94.12</TD></TR><TR><TD class="tg-e3zv">375</TD><TD class="tg-031e">39223</TD><TD class="tg-031e">42055</TD><TD class="tg-031e">54108</TD><TD class="tg-031e">45129</TD><TD class="tg-031e">7904.2</TD><TD class="tg-031e">17.51</TD><TD class="tg-031e">120.34</TD></TR><TR><TD class="tg-e3zv">250</TD><TD class="tg-031e">25511</TD><TD class="tg-031e">29221</TD><TD class="tg-031e">36966</TD><TD class="tg-031e">30566</TD><TD class="tg-031e">5844.74</TD><TD class="tg-031e">19.12</TD><TD class="tg-031e">122.26</TD></TR><TR><TD class="tg-e3zv">125</TD><TD class="tg-031e">13805</TD><TD class="tg-031e">15193</TD><TD class="tg-031e">18957</TD><TD class="tg-031e">15985</TD><TD class="tg-031e">2665.75</TD><TD class="tg-031e">16.68</TD><TD class="tg-031e">127.88</TD></TR><TR><TD class="tg-e3zv">50</TD><TD class="tg-031e">5670</TD><TD class="tg-031e">6042</TD><TD class="tg-031e">7878</TD><TD class="tg-031e">6530</TD><TD class="tg-031e">1182.13</TD><TD class="tg-031e">18.1</TD><TD class="tg-031e">130.6</TD></TR><TR><TD class="tg-e3zv">25</TD><TD class="tg-031e">2812</TD><TD class="tg-031e">2967</TD><TD class="tg-031e">4012</TD><TD class="tg-031e">3264</TD><TD class="tg-031e">652.69</TD><TD class="tg-031e">20</TD><TD class="tg-031e">130.55</TD></TR><TR><TD class="tg-e3zv">10</TD><TD class="tg-031e">1022</TD><TD class="tg-031e">1180</TD><TD class="tg-031e">1533</TD><TD class="tg-031e">1245</TD><TD class="tg-031e">261.63</TD><TD class="tg-031e">21.01</TD><TD class="tg-031e">124.5</TD></TR><TR><TD class="tg-e3zv">5</TD><TD class="tg-031e">554</TD><TD class="tg-031e">808</TD><TD class="tg-031e">776</TD><TD class="tg-031e">713</TD><TD class="tg-031e">138.34</TD><TD class="tg-031e">19.41</TD><TD class="tg-031e">142.53</TD></TR><TR><TD class="tg-e3zv">0</TD><TD class="tg-031e">2</TD><TD class="tg-031e">-2</TD><TD class="tg-031e">0</TD><TD class="tg-031e">0</TD><TD class="tg-031e">2</TD><TD class="tg-031e">N/A</TD><TD class="tg-031e">N/A</TD></TR></TBODY></TABLE><FIGCAPTION>Table 9: Fluorescence of different concentrations of sodium fluorescein (ng/ml) as measured in a BioTEX Synergy HT plate reader with blank DPBS control subtracted. Units are arbitrary. FL. - fluorescence, S.D.- standard deviation, C.V. coefficient of variation (all arithmetic)
                                    </FIGCAPTION></FIGURE></DIV><H3>Example flow cytometry graphs for ‘A’ (Existing J23101 construct) well F1.</H3><DIV><FIGURE class="content-image image-half"><FIGCAPTION>Figure 7: Dot plot of side scatter (SSC) vs forward scatter (FSC) showing population (red, blue) SSC gate for excluding cell debris.</FIGCAPTION></FIGURE><FIGURE class="content-image image-half"><FIGCAPTION>Figure 8: Histogram of SSC before gating
                                    </FIGCAPTION></FIGURE><FIGURE class="content-image image-half"><FIGCAPTION>Figure 9: Histogram of SSC after gating

                                    </FIGCAPTION></FIGURE><FIGURE class="content-image image-half"><FIGCAPTION>Figure 10: Dot plot of SSC vs fluorescence (FL1) showing gating to exclude cell debris (blue)

                                    </FIGCAPTION></FIGURE><FIGURE class="content-image image-half"><FIGCAPTION>Figure 11: Histogram of gated FL-1 showing log-normal distribution


                                    </FIGCAPTION></FIGURE></DIV><H3>Example flow cytometry graphs for ‘B’ (Cloned J23101 construct) well F4.</H3><DIV><FIGURE class="content-image image-half"><FIGCAPTION>Figure 12: Dot plot of side scatter (SSC) vs forward scatter (FSC) showing population (red, blue) SSC gate for excluding cell debris.</FIGCAPTION></FIGURE><FIGURE class="content-image image-half"><FIGCAPTION>Figure 13: Histogram of SSC before gating
                                    </FIGCAPTION></FIGURE><FIGURE class="content-image image-half"><FIGCAPTION>Figure 14: Histogram of SSC after gating

                                    </FIGCAPTION></FIGURE><FIGURE class="content-image image-half"><FIGCAPTION>Figure 15: Dot plot of SSC vs fluorescence (FL1) showing gating to exclude cell debris (blue)

                                    </FIGCAPTION></FIGURE><FIGURE class="content-image image-half"><FIGCAPTION>Figure 16: Histogram of gated FL-1 showing log-normal distribution


                     </FIGCAPTION></FIGURE></DIV><H3>Example flow cytometry graphs for ‘C’ (Cloned J23115 containing the mismatches of K823012 construct) well G9.</H3><DIV><FIGURE class="content-image image-half"><FIGCAPTION>Figure 17: Dot plot of side scatter (SSC) vs forward scatter (FSC) showing population (red, blue) SSC gate for excluding cell debris.</FIGCAPTION></FIGURE><FIGURE class="content-image image-half"><FIGCAPTION>Figure 18: Histogram of SSC before gating
                                    </FIGCAPTION></FIGURE><FIGURE class="content-image image-half"><FIGCAPTION>Figure 19: Histogram of SSC after gating

                                    </FIGCAPTION></FIGURE><FIGURE class="content-image image-half"><FIGCAPTION>Figure 20: Dot plot of SSC vs fluorescence (FL1) showing gating to exclude cell debris (blue)

                                    </FIGCAPTION></FIGURE><FIGURE class="content-image image-half"><FIGCAPTION>Figure 21: Histogram of gated FL-1 showing log-normal distribution


                                    </FIGCAPTION></FIGURE></DIV><H3>Example flow cytometry graphs for ‘N’ (construct with no promoter i.e. E0240 by itself) well F10.</H3><DIV><FIGURE class="content-image image-half"><FIGCAPTION>Figure 22: Dot plot of side scatter (SSC) vs forward scatter (FSC) showing population (red) SSC gate for excluding cell debris.</FIGCAPTION></FIGURE><FIGURE class="content-image image-half"><FIGCAPTION>Figure 23: Histogram of SSC before gating
                                    </FIGCAPTION></FIGURE><FIGURE class="content-image image-half"><FIGCAPTION>Figure 24: Histogram of SSC after gating

                                    </FIGCAPTION></FIGURE><FIGURE class="content-image image-half"><FIGCAPTION>Figure 25: Dot plot of SSC vs fluorescence (FL1). The population of non-fluorescing cells and debris is indistinguishable from debris so no gating is used.

                                    </FIGCAPTION></FIGURE><FIGURE class="content-image image-half"><FIGCAPTION>Figure 26: Histogram of FL-1


                                    </FIGCAPTION></FIGURE></DIV></DIV></SECTION></DIV></DIV></DIV><DIV class="printfooter">
Retrieved from &quot;<A href="http://2014.igem.org/Team:Imperial/InterLab_Study">http://2014.igem.org/Team:Imperial/InterLab_Study</A>&quot;</DIV></DIV></DIV><DIV id="footer-box" class="noprint"><DIV id="footer"><UL id="f-list"><LI id="t-recentchanges"><A href="/Special:RecentChanges" title="Recent changes">Recent changes</A></LI><LI id="t-whatlinkshere"><A href="/Special:WhatLinksHere/Team:Imperial/InterLab_Study" title="List of all wiki pages that link here [j]" accesskey="j">What links here</A></LI><LI id="t-recentchangeslinked"><A href="/Special:RecentChangesLinked/Team:Imperial/InterLab_Study" title="Recent changes in pages linked from this page [k]" accesskey="k">Related changes</A></LI><LI id="t-specialpages"><A href="/Special:SpecialPages" title="List of all special pages [q]" accesskey="q">Special pages</A></LI><LI><A href="/Special:Preferences">My preferences</A></LI></UL></DIV><DIV id="footer"><UL id="f-list"><LI id="t-print"><A href="/wiki/index.php?title=Team:Imperial/InterLab_Study&amp;printable=yes" title="Printable version of this page [p]" accesskey="p">Printable version</A></LI><LI id="t-permalink"><A href="/wiki/index.php?title=Team:Imperial/InterLab_Study&amp;oldid=287046" title="Permanent link to this revision of the page">Permanent link</A></LI><LI id="privacy"><A href="/2014.igem.org:Privacy_policy" title="2014.igem.org:Privacy policy">Privacy policy</A></LI><LI id="disclaimer"><A href="/2014.igem.org:General_disclaimer" title="2014.igem.org:General disclaimer">Disclaimers</A></LI></UL></DIV></DIV></DIV></DIV></BODY></HTML>