<HTML xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en" dir="ltr">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki  ltr ns-0 ns-subject page-Team_KIT-Kyoto_Project_Abstract"><DIV id="globalWrapper"><DIV id="top-section"><DIV id="p-logo"><A href="/Main_Page" title="Main Page">&quot;
	    </A></DIV><DIV id="menubar" class="left-menu noprint"><UL><LI class="selected"><A href="/Team:KIT-Kyoto/Project/Abstract">Page               </A></LI><LI class="new"><A href="/wiki/index.php?title=Talk:Team:KIT-Kyoto/Project/Abstract&amp;action=edit&amp;redlink=1">Discussion               </A></LI><LI><A href="/wiki/index.php?title=Team:KIT-Kyoto/Project/Abstract&amp;action=edit">View source               </A></LI><LI><A href="/wiki/index.php?title=Team:KIT-Kyoto/Project/Abstract&amp;action=history">History               </A></LI><LI style="color:#808080;cursor:default">teams</LI></UL></DIV><DIV class="right-menu noprint" id="menubar"><UL><LI id="pt-login"><A href="/wiki/index.php?title=Special:UserLogin&amp;returnto=Team:KIT-Kyoto/Project/Abstract" title="You are encouraged to log in; however, it is not mandatory [o]" accesskey="o">Log in</A></LI></UL></DIV><DIV id="search-controls" class="noprint"><FORM action="/Special:Search" id="searchform"> </FORM></DIV></DIV><DIV id="content"><H1 class="firstHeading">Team:KIT-Kyoto/Project/Abstract</H1><DIV id="bodyContent"><H3 id="siteSub" class="noprint">From 2010.igem.org</H3><TABLE class="menu" border="0" background="https://static.igem.org/mediawiki/2010/e/e5/HAIKEI-HIDARI.PNG" bordercolor="black" cellspacing="0" cellpadding="0" width="100%"><TBODY><TR><TD align="center"><DIV id="navi_continer"><DIV id="navi"><UL><LI><A href="https://2010.igem.org/Team:KIT-Kyoto/Team">Our Team</A></LI><LI><A href="http://twitter.com/KIT_Kyoto" target="_blank">Twitter</A></LI><LI><A href="http://kitkyoto.blog114.fc2.com/" target="_blank">Blog</A></LI><LI><A href="http://sites.google.com/site/kitkyoto/" target="_blank">Japanese Site</A></LI><LI><A href="http://www.youtube.com/user/KITKyoto" target="_blank">YouTube</A></LI></UL><LI><A href="https://2010.igem.org/Team:KIT-Kyoto/Project/Abstract#Abstract">Abstract</A></LI><LI><A href="https://2010.igem.org/Team:KIT-Kyoto/Project/Abstract#Introduction">Introduction</A></LI><LI><A href="https://2010.igem.org/Team:KIT-Kyoto/Project/Abstract#Materials_.26_Methods">Materials &amp; Methods</A></LI><LI><A href="https://2010.igem.org/Team:KIT-Kyoto/Project/Abstract#Results_.26_Discussion">Results &amp; Discussion</A></LI><LI><A href="https://2010.igem.org/Team:KIT-Kyoto/Project/Abstract#References">References</A></LI><LI><A href="https://2010.igem.org/Team:KIT-Kyoto/Notebook">Lab Note</A></LI><LI><A href="https://2010.igem.org/Team:KIT-Kyoto/Protocol">Protocol</A></LI><LI><A href="https://2010.igem.org/Team:KIT-Kyoto/Notebook/Promoters">Promoters</A></LI><LI><A href="https://2010.igem.org/Team:KIT-Kyoto/DesignNote">Design Note</A></LI><LI><A href="https://2010.igem.org/Team:KIT-Kyoto/Photo">Photograph</A></LI></DIV></DIV></TD></TR></TBODY></TABLE><TABLE border="0" width="965px" align="center"><TBODY><TR><TD><DIV><A href="/Team:KIT-Kyoto/Home" title="Team:KIT-Kyoto/Home">Home</A>&gt;<STRONG class="selflink">Project</STRONG></DIV></TD><TD><DIV align="right">Language : <STRONG class="selflink">English</STRONG> / <A href="/Team:KIT-Kyoto/Project/AbstractJ" title="Team:KIT-Kyoto/Project/AbstractJ">Japanese</A></DIV></TD></TR></TBODY></TABLE><DIV id="NAKAMI"><H2><SPAN class="mw-headline" id="Abstract"> Abstract </SPAN></H2><DL><DD><B>&quot;<I>E.coli</I> Pen&quot;: Draw with your own color.</B></DD><DD>Our team, KIT-Kyoto suggests an “<I>E.coli</I> Pen” as a new Art Tool. This brand-new pen uses no ink but medium in which genetically modified <I>E.coli</I> has been cultured. The Pen is able to express more than four colors in various intensities with single bacterial culture. This will be achieved by constructing plasmids carrying genes coding for four different fluorescent proteins under the control of seven promoters having different sensitivity to oxidative stress. The <I>E.coli</I> carrying these plasmids will produce different colors with various intensities by differentially responding to the gradient of hydrogen peroxide treatment. Different from previous passive BioArt in iGEM, the genetically engineered “<I>E.coli</I> Pen” provides an active and wonderful tool for us to purely enjoy the Art having a feeling for biotechnology.
</DD></DL><H2><SPAN class="mw-headline" id="Introduction"> Introduction </SPAN></H2><DL><DD>Our team, KIT-Kyoto has designed and carried out the project taking account of the key word &quot;Make the science and technology more accessible to the public&quot;. </DD><DD>Recently, it is getting more and more serious problem that people think that the science is too difficult to understand. Moreover, it is expected that people are moving away from the science as the science becomes more and more complicated. Therefore, to overcome such a problem, we would like to offer more opportunities for the public to interact with the science especially for the person who is not interested in the modern science. </DD><DD>In deciding our project, we thought it is the most important that anyone can readily enjoy the science and technology. We therefore decided to focus on &quot;Bio art&quot; in order to invent the artistic products by using biotechnology, since Art is more accessible to the public. Moreover, unifying the two apparently distant academic fields &quot;Science&quot; and &quot;Art&quot; is an educational philosophy of our University, Kyoto Institute of technology. 
</DD></DL><P style="width: 960px; margin:10px auto"><UL class="acc" align="left"><LI><A href="#2"> &gt;&gt;Learn More BioArt</A></LI></UL></P><P> A brand-new form of art has emerged with the revolution of biology in the last ten years, this is Bio-Art. Adding new artistic elements, such as cells, DNA, genomes, proteins, enzymes, etc., this new form of artistic expression has as medium living matter, and the studios of the artists are biological laboratories, while high end technologies of genetic engineering, tissue culture and many others are their new painting brushes.
</P><P> BioArt is considered by most artists to be strictly limited to “living forms,” although there is some debate as to the stages at which matter can be considered to be alive or living. Creating living beings and practicing in the life sciences brings about ethical, social and aesthetic inquiry.
</P><P> Here, we aim at a completely new style of BioArt.  In hitherto BioArt, the appreciator enjoys a work in the conventional way, that is he/she appreciates an artist’s resulting piece looking or touching it. The new style of BioArt we propose here is the enjoyment of making art itself. With the “<I>E.coli</I>  Pen” that we propose here the appreciator himself becomes part of the artistic work he is drawing enjoying the fascination of making the work and the work itself.
    </P><DL><DD><DIV class="thumb tleft"><DIV class="thumbinner" style="width:202px;"><DIV class="thumbcaption">Already existing BioArt:wikipedia</DIV></DIV></DIV>However, an already existing bio art does not meet the key word “Make the science and technology more accessible”, because that the preexisting bio art needs special equipment and the deep knowledge of biotechnology etc. We therefore decided to work on the development of a new art tool by which people could enjoy the bio art easily. </DD><DD>Here we can propose a novel art tool, “<I>E.coli</I> Pen” which draws paintings with innumerable colors as inks of the pen that are produced by a single <I>E. coli</I> bacteria. &quot;<I>E.coli</I> Pen&quot; would further develop the field of the bio art as a new art tool. Furthermore it is useful as an educational tool, since people can enjoy drawings with the heavenly color with happiness and at the same time they can study the molecular mechanism how the colors are made by genetic engineering.</DD></DL><DL><DD>We summarize how to make &quot;<I>E.coli</I> Pen&quot; in the following section.
</DD><DD><B>1. How to make an innumerable color in <I>E. coli</I> bacteria</B></DD><DD><B>1-1. Use of light's three primary colors</B></DD><DD>Principle of light’s three primary colors (red, green, and blue) can be applied to create an innumerable color. The fluorescent protein can be used as a color source. Red Fluorescent Protein (RFP) can be used for Red, Green  Fluorescent Protein (GFP) can be used for Green, and Cyan Fluorescent Protein (CFP) can be used for Blue. The color fluorescence is emitted when ultraviolet rays is irradiated. These fluorescent proteins are widely used in the field of the genetic engineering as the reporter gene. Therefore once we obtain the red, green, and blue fluorescent proteins in our hands, we can create an innumerable color by simply mixing these three fluorescent proteins with various ratios. At first we tried to produce the red, green, and blue fluorescent proteins in <I>E. coli</I>. 
</DD><DD>To achieve this, the multi-gene cassette shown in below was newly designed. 
</DD></DL><DL><DD>In this multi-gene cassette, the genes encoding RFP (Red), GFP (Green) and CFP (Blue) are placed in a tandem array. Each gene can be controlled by different promoter that has different activity. In order to create different color, fluorescent proteins have to be produced in different ratios. This can be achieved by utilizing promoters having different activities.TM is a terminator.
</DD></DL><DL><DD><B>1-2 Selection of an appropriate promoter</B></DD><DD>Then, what kind of promoter is actually appropriate for this purpose? The promoter activity has to be controlled by some regulating substances. The easily available substance that had not been used before in iGEM would be suitable for the purpose. If the substance is related to some human diseases or aging, the promoter responding to the substance may be used for diagnosis of the disease or aging in future, since the fluorescent proteins under the control of the promoter can be used as markers. From these standpoints we decided to use promoters responding to hydrogen peroxide. Hydrogen peroxide is one of the reactive oxygen species that oxidize nucleic acid, lipid and protein to result in aging of human cells and various diseases. Although effective use of oxygen is acquired during evolution, the defense system to the active oxygen generated by consumption of oxygen also has been acquired in the aerobe. There are a set of genes that are involved in this defense system. These genes can be induced by hydrogen peroxide. We have utilized the promoters of these genes to induce the fluorescent proteins. 
</DD></DL><DL><DD><B>1-3 Mechanism of hydrogen peroxide response</B></DD><DD><I>E. coli</I> has two defense mechanisms against the active oxygen . One is OxyR response, and another is SoxRS response. 
</DD><DD>In the OxyR response, OxyR works as a transcription factor. OxyR regulates activation or repression of the gene expression. Without active oxygen, OxyR is in an inactivated form. And with active oxygen, OxyR is activated and the activated OxyR can bind to the specific DNA sequence. OxyR binding sites are found in promoters of a set of genes that respond to the active oxygen. In these active oxygen response genes, we focused on the promoter of <I>ahpC</I>, <I>dps</I>, <I>oxyR</I>, <I>sufA</I>, and <I>yaiA</I>. These genes carry no recognition sequences of <I>EcoR</I>Ⅰ, <I>Xba</I>Ⅰ, <I>Spe</I>Ⅰ,and <I>Pst</I>Ⅰ restriction enzymes. These restriction enzyme sites would be used for cloning into the pSB1C3. The promoters of these genes cloned into pSB6A1 are listed in our Biobrick site.
</DD><DD>In the SoxRS response, SoxR works as a transcription factor. SoxR is constantly expressed in cells. SoxR binds to the promoter of SoxS, and represses its transcription without hydrogen peroxide. With hydrogen peroxide, SoxR activates the SoxS gene transcription. SoxS also works as a transcription factor of two or more downstream genes responsible to the hydorogen peroxide. In these hydrogen peroxide response genes, we focused on the promoter of <I>acrAB</I> and <I>sodA</I>. Both of these genes have no sequence of <I>Eco</I>RⅠ, <I>Xba</I>Ⅰ, <I>Spe</I>Ⅰand <I>Pst</I>Ⅰ restriction enzyme sites. These restriction enzyme sites would be used for cloning into the pSB1C3. The promoters of these two genes cloned into pSB6A1 are listed in our Biobrick site.
</DD><DD><A href="https://2010.igem.org/Team:KIT-Kyoto/Parts" class="external text" rel="nofollow">&gt;&gt;Go to the article of our Biobrick site.</A></DD></DL><DL><DD><B>1-4 Active oxygen response promoters</B></DD><DD>Out of these seven active oxygen response promoters, we here shows the structure of <I>ahpC</I> and <I>sufA</I> promoters. 
</DD></DL><DL><DD><B>ahpC promoter</B></DD><DD><DIV class="thumb tleft"><DIV class="thumbinner" style="width:502px;"><DIV class="thumbcaption">Fig.1 OxyR is the transcriptional regulator for the expression of antioxidant genes in response to oxidative stress.The OxyR protein is constitutively produced and oxidized by hydrogen peroxide.The oxidized form(active form) of OxyR binds to the promoter region(OxyR binding region:OBR) of <I>ahpC</I> gene and promotes the transcription of the <I>ahpC</I> gene.</DIV></DIV></DIV></DD></DL><DL><DD><I>ahpC-F</I> are the genes involved in oxidative stress defense. Transcription of these genes is regulated in response to H<SUB>2</SUB>O<SUB>2</SUB>. The <I>ahpC</I> and <I>ahpF</I> genes encode the small subunit and large subunit of alkyl hydroperoxide reductase, respectively. The complex of AhpC and AhpF reduces alkyl hydroperoxide. The <I>ahpC</I> promoter is controlled by the OxyR protein, an oxidative stress responsive transcription factor. OxyR is constitutively expressed but activated only under oxidative stressed conditions caused by H<SUB>2</SUB>O<SUB>2</SUB>. The activated OxyR binds to the promoter region of the <I>ahpC</I> gene and promotes its transcription.</DD></DL><DL><DD><B>sufA promoter</B></DD><DD><DIV class="thumb tleft"><DIV class="thumbinner" style="width:602px;"><DIV class="thumbcaption">Fig.2 The <I><I>sufA</I></I> operon(<I><I>sufABCDSE</I></I>) is involved in the assembly of Fe-S clusters.Expression of the <I><I>sufA</I></I> gene is induced by superoxide generators and hydrogen peroxide.The <I><I>sufA</I></I>-proximal region also contains the OxyR binding region(OBR),and OxyR functions as one of the activators of the <I><I>sufA</I></I> operon.</DIV></DIV></DIV></DD></DL><DL><DD><I>sufA</I> promoter is controlled by some transcription factors including OxyR, regulates the transcription of <I>sufABCDSE</I> operon encoding an alternative Fe-S cluster assembly system in <I>E.coli</I>. We also constructed expression systems in which the <I>sufA</I> promoter controls the transcription of fluorescent protein genes and measured the fluorescent intensity.
</DD></DL><DL><DD><B>2. How is the pen that efficiently mixes the active oxygen and <I>E. coli</I> made?</B></DD><DD>The <I>E.coli</I> Pen changes the mixing rate for <I>E. coli</I> culture medium and hydrogen peroxide.  The volume of each solution, placed in different tubes within the pen, can be regulated by the frequency and intensity of pressing the pen's piston which can be interchanged among both tubes. For example, to make red ink, one presses the pyston in the <I>E. coli</I> tube three times and then interchanges to the hydrogen peroxide tube and presses the piston twice. Or, to use green ink, press the<I> E. coli</I> tube five times and then the hydrogen peroxide tube only once. In this way, one is able to produce any color. This is a spectacular way of producing new colors that any conventional pen can not realize. The movie of the <I>E.coli</I> Pen is shown in the first page of wiki of our team.
</DD></DL><DL><DD><A href="http://www.youtube.com/user/KITKyoto#p/u/9/DVVq0MWgRFk" class="external text" rel="nofollow">&gt;&gt;System of &quot;<I>E.coli</I> Pen&quot;</A></DD><DD><A href="https://2010.igem.org/Team:KIT-Kyoto/DesignNote" class="external text" rel="nofollow">&gt;&gt;Go to the article of design note.</A></DD></DL><H2><SPAN class="mw-headline" id="Materials_.26_Methods"> Materials &amp; Methods </SPAN></H2><TABLE border="0"><TBODY><TR><TD><B>Materials</B></TD></TR><TR><TD><B>Strains</B></TD></TR><TR><TD><I>Esherichia coli</I></TD><TD>　　DH5 Alpha</TD></TR><TR><TD><B>Plasmids</B></TD></TR><TR><TD>　　　　pSB3K3</TD><TD><A href="http://partsregistry.org/wiki/index.php/Part:pSB3K3" class="external text" rel="nofollow">&gt;&gt;get more information</A></TD></TR><TR><TD>　　　　pSB6A1</TD><TD><A href="http://partsregistry.org/wiki/index.php/Part:pSB6A1" class="external text" rel="nofollow">&gt;&gt;get more information</A></TD></TR><TR><TD>　　　　pSB1C3</TD><TD><A href="http://partsregistry.org/wiki/index.php/Part:pSB1C3" class="external text" rel="nofollow">&gt;&gt;get more information</A></TD></TR><TR><TD>　　　　pSB1A2</TD><TD><A href="http://partsregistry.org/wiki/index.php/Part:pSB1A2" class="external text" rel="nofollow">&gt;&gt;get more information</A></TD></TR><TR><TD>　　　　pSB1AK3</TD><TD><A href="http://partsregistry.org/wiki/index.php/Part:pSB1AK3" class="external text" rel="nofollow">&gt;&gt;get more information</A></TD></TR><TR><TD>　　　　pSB4A5</TD><TD><A href="http://partsregistry.org/wiki/index.php/Part:pSB4A5" class="external text" rel="nofollow">&gt;&gt;get more information</A></TD></TR><TR><TD><B>Methods</B></TD></TR><TR><TD colspan="2">Our standard protocols are listed below. Please click each item for details.</TD></TR><TR><TD><B>List of protocols</B></TD></TR><TR><TD>　　　　Bacterial transformation</TD><TD><A href="https://2010.igem.org/Team:KIT-Kyoto/Protocol#tf" class="external text" rel="nofollow">&gt;&gt;Go protocol</A></TD></TR><TR><TD>　　　　DNA miniprep</TD><TD><A href="https://2010.igem.org/Team:KIT-Kyoto/Protocol#al" class="external text" rel="nofollow">&gt;&gt;Go protocol</A></TD></TR><TR><TD>　　　　PCR</TD><TD><A href="https://2010.igem.org/Team:KIT-Kyoto/Protocol#pcr" class="external text" rel="nofollow">&gt;&gt;Go protocol</A></TD></TR><TR><TD>　　　　DNA digestion by restriction enzymes</TD><TD><A href="https://2010.igem.org/Team:KIT-Kyoto/Protocol#re" class="external text" rel="nofollow">&gt;&gt;Go protocol</A></TD></TR><TR><TD>　　　　Agarose gel electrophoresis</TD><TD><A href="https://2010.igem.org/Team:KIT-Kyoto/Protocol#agar" class="external text" rel="nofollow">&gt;&gt;Go protocol</A></TD></TR><TR><TD>　　　　Isolation of DNA fragments from agarose gel</TD><TD><A href="https://2010.igem.org/Team:KIT-Kyoto/Protocol#gel" class="external text" rel="nofollow">&gt;&gt;Go protocol</A></TD></TR><TR><TD>　　　　DNA ligation</TD><TD><A href="https://2010.igem.org/Team:KIT-Kyoto/Protocol#lig" class="external text" rel="nofollow">&gt;&gt;Go protocol</A></TD></TR><TR><TD><B>Culture media</B></TD></TR><TR><TD>　　　　LB culture medium</TD><TD><A href="https://2010.igem.org/Team:KIT-Kyoto/Protocol#lb" class="external text" rel="nofollow">&gt;&gt;Go protocol</A></TD></TR><TR><TD>　　　2xYT culture medium</TD><TD><A href="https://2010.igem.org/Team:KIT-Kyoto/Protocol#yt" class="external text" rel="nofollow">&gt;&gt;Go protocol</A></TD></TR><TR><TD>　　　　SOB culture medium</TD><TD><A href="https://2010.igem.org/Team:KIT-Kyoto/Protocol#sob" class="external text" rel="nofollow">&gt;&gt;Go protocol</A></TD></TR><TR><TD>　　　　SOC culture medium</TD><TD><A href="https://2010.igem.org/Team:KIT-Kyoto/Protocol#soc" class="external text" rel="nofollow">&gt;&gt;Go protocol</A></TD></TR></TBODY></TABLE><H2><SPAN class="mw-headline" id="Results_.26_Discussion"> Results &amp; Discussion </SPAN></H2><H3><SPAN class="mw-headline" id="1._Susceptibility_to_H2O2_in_Escherichia_coli"> 1. Susceptibility to H<SUB>2</SUB>O<SUB>2</SUB> in <I>Escherichia coli</I></SPAN></H3><DL><DD>In the <I>E.coli</I> Pen, we utilized the hydrogenperoxide (H<SUB>2</SUB>O<SUB>2</SUB>)-responsive system of <I>E.coli</I> cells to make different colors. It is well known that H<SUB>2</SUB>O<SUB>2</SUB> shows the concentration-dependent toxicity for <I>E.coli</I> cells. In order to determine the utilizable concentration of H<SUB>2</SUB>O<SUB>2</SUB> in the <I>E.coli</I> Pen, we monitored the susceptibility of <I>E.coli</I> cells (DH5 Alpha) against H<SUB>2</SUB>O<SUB>2</SUB>.
</DD></DL><DL><DD><B>&lt;Experiment method&gt;</B></DD><DD><B>The 1st day evening </B></DD></DL><DL><DD>Precultivation of DH5 Alpha cells 
</DD><DD>DH5 Alpha cells on a LB plate were picked up with the sterilized toothpick, inoculated into 2 ml of LB (amp-) medium in a test tube, and then precultured at 37˚C (overnight).
</DD></DL><DL><DD><B>The 2nd day</B></DD></DL><DL><DD>Survival test
</DD><DD>All of the precultured DH5 Alpha cells were transferred into 28 ml of a fresh LB (amp-) medium in a 100 ml Erlenmeyer flask. 
</DD><DD>↓ DH5 Alpha cells were cultivated at 37 ˚C until the optical density (OD600) of the medium became 0.4~0.6.
</DD><DD>↓ The culture was equally dispensed (2 ml each) into sample tubes, and then cells were treated with various concentrations of H<SUB>2</SUB>O<SUB>2</SUB> (final concentrations were as follows: 1 pM, 10 pM, 200 pM. 1 nM, 10 nM, 100 nM, 1μM, 10μM, 100μM, and 1 mM) at 37 ˚C for 60 min.
</DD><DD>↓ Samples were diluted by mixing 1 ml of fresh LB medium and 1μl of each sample.
</DD><DD>↓ 10 μl of the diluted sample was spread out on a LB plate (amp-) and incubated at 37 ˚C (overnight).
</DD></DL><DL><DD><B>The 3rd day</B></DD></DL><DL><DD>The number of colonies on the plate was counted and made a graph of the survival curve.
</DD></DL><DL><DD><B>&lt;Results&gt;</B></DD></DL><DL><DD><DIV class="thumb tleft"><DIV class="thumbinner" style="width:302px;"><DIV class="thumbcaption">Fig.3</DIV></DIV></DIV></DD></DL><DL><DD>Representative data from five independent experiments are shown.
</DD></DL><DL><DD><B>&lt;Discussion&gt;</B></DD><DD>As shown in the graph, 1mM of H<SUB>2</SUB>O<SUB>2</SUB> was the lethal concentration for <I>E.coli</I> cells. On the other hand, less than 0.000001mM(=1nM) H<SUB>2</SUB>O<SUB>2</SUB> has no influence on the survival of <I>E.coli</I> cells. These results suggest that a range of 1nM-1mM of H<SUB>2</SUB>O<SUB>2</SUB> is the utilizable concentration for the <I>E.coli</I> Pen.
</DD></DL><DL><DD><B>&lt;References&gt;</B></DD><DD>Application serial number in Japan (P2004-551020) &quot;Ozone generation goes through an antibody or neutrophile&quot; by Novartis Akuchiengezerushafuto and the Scrips Res. Inst.
</DD></DL><DL><DD><DIV class="thumb tleft"><DIV class="thumbinner" style="width:302px;"><DIV class="thumbcaption">Fig.4</DIV></DIV></DIV></DD><DD>Fig.4 shows the concentration of H<SUB>2</SUB>O<SUB>2</SUB> dependent toxicity for the viability of <I>E.coli</I> XL1- blue and O112a, c serotype. Our data is supported by this reference.
</DD></DL><H3><SPAN class="mw-headline" id="2._Compare_the_performances_of_the_high_copy_vector_and_low_copy_vector"> 2. Compare the performances of the high copy vector and low copy vector </SPAN></H3><DL><DD>In making the <I>E.coli</I> Pen, we thought that it was important to select “ink.” The <I>E.coli</I> Pen, which focused on writing or drawing, is a first attempt in iGEM, so we thought that we should start to select an appropriate vector to produce “ink”, fluorescent proteins. As “ink”, it is necessary to keep writing or drawing clearly over the long term, and we therefore selected an expression vector with high transcriptional efficiency.
</DD><DD>In general, pSB6 is a low copy plasmid which makes it more suitable for protein production than the high copy vector. However this information cannot be found in any sites related to iGEM. Consequently, we tried to compare the performances of the high copy vector pSB1 and low copy vector pSB6.
</DD></DL><DL><DD><B>&lt;Experiment method&gt;</B></DD><DD><I>E.coli</I> cells were cultivated at 37 ˚C until the optical density (O.D.600) of the medium became 0.4~0.5. Cells were suspended in PBS to obtain the initial O.D.600 = 0.5 because the LB medium has auto fluorescence, then the fluorescent intensity was measured using Thermo Labosystems Floreskan Ascent CF.
</DD></DL><DL><DD><B>&lt;Results&gt;</B></DD></DL><DL><DD> Vector</DD><DD>Fluorescent intensitypSB1A2(promoter less GFP)pSB1A2(High Copy Vector)pSB6A1(Low Copy Vector)</DD><DD>10.1450.3494.33</DD><DD>20.1460.3244.34</DD><DD>30.1460.3284.42</DD><DD>Ave0.1460.3374.36</DD></DL><DL><DD><DIV class="thumb tleft"><DIV class="thumbinner" style="width:402px;"><DIV class="thumbcaption">Fig.5</DIV></DIV></DIV></DD></DL><DL><DD><B>&lt;Discussion&gt;</B></DD><DD>Clearly the Low copy vector has much higher transcriptional efficiency of [GFP] than the High copy vector.
</DD><DD>So we selected a Low copy vector (pSB6A1) to produce fluorescent proteins.
</DD></DL><H3><SPAN class="mw-headline" id="3._Constructing_an_expression_system"> 3. Constructing an expression system </SPAN></H3><DL><DD>We constructed an expression system in which the active oxygen response promoter controls the transcription of fluorescent protein genes on pSB6, and transformed this vector into DH5 Alpha competent cells. Here we show an expressing system by using <I>ahpC</I> promoter and <I>sufA</I> promoter among 7 promoters which we constructed. We then examined the fluorescent intensity using the cells treated with H<SUB>2</SUB>O<SUB>2</SUB> as follows:
</DD></DL><DL><DD><B>&lt;Experiment method&gt;</B></DD><DD><I>E.coli</I> cells in the log-phase were treated with H<SUB>2</SUB>O<SUB>2</SUB> at 37℃. The fluorescent intensity was measured every 10 minutes until 80minutes. After the treatment with H<SUB>2</SUB>O<SUB>2</SUB>, cells were suspended in PBS to obtain the initial OD600 = 0.5 because the LB medium has auto fluorescence, then the fluorescent intensity was measured using Thermo Labosystems Floreskan Ascent CF. 
</DD></DL><DL><DD><B>&lt;Results&gt;</B></DD><DD><B>3-1. <I>ahpC</I> (BBa_K362001) </B></DD></DL><DL><DD><DIV class="thumb tleft"><DIV class="thumbinner" style="width:402px;"><DIV class="thumbcaption">Fig.6</DIV></DIV></DIV></DD></DL><DL><DD>Fig.6 indicates that the <I>ahpC</I> promoter responded to H<SUB>2</SUB>O<SUB>2</SUB> in a dose-dependent manner and its activity increased with time. Especially, the treatment with 10 μM to 1mM H<SUB>2</SUB>O<SUB>2</SUB> for 40 to 70 min increased the fluorescent intensity remarkably. After 70 min, the intensity decayed with time.
</DD></DL><DL><DD><B>3-2. <I>sufA</I> (BBa_K362005) </B></DD></DL><DL><DD>We also constructed expression systems in which the <I>sufA</I> promoter controls the transcription of fluorescent protein genes and measured the fluorescent intensity.
</DD></DL><DL><DD><DIV class="thumb tleft"><DIV class="thumbinner" style="width:402px;"><DIV class="thumbcaption">Fig.7</DIV></DIV></DIV></DD></DL><DL><DD>Fig.7 shows that the <I>sufA</I> promoter responded to H<SUB>2</SUB>O<SUB>2</SUB> stress in a dose-dependent manner.The <I>sufA</I> promoter region is located upstream of the <I>sufA</I> operon and SufA protein acts as an assembly scaffold. Since levels of SufA protein are maximized quite rapidly, the activity of the <I>sufA</I> promoter was increased and sustained for at least 80 min. We think that the concentration-dependent activation of the <I>sufA</I> promoter also reflects the levels of SufA protein.</DD></DL><DL><DD><B>&lt;Discussion&gt;</B></DD><DD>Our results seem quite reasonable and suggest a possibility that we can adjust the expression levels of fluorescent genes by H<SUB>2</SUB>O<SUB>2</SUB> concentration and by time. Therefore, we expect that lots of colors can be created using this system. Our expression system must be a powerful tool for the construction of the <I>E.coli</I> Pen.
</DD></DL><H3><SPAN class="mw-headline" id="4._Making_Inks"> 4. Making Inks </SPAN></H3><DL><DD>This section consists of two parts. The first one is about making inks with various colors. The other is the actual art works drawn using the inks we made. 
</DD></DL><DL><DD><B>4-1. Making inks with various colors.</B></DD></DL><DL><DD><DIV class="thumb tleft"><DIV class="thumbinner" style="width:321px;"><DIV class="thumbcaption">Fig.8 Autofluorescence of LB medium and PBS.(A) PBS without UV light, (B) LB medium without UV light, (C) PBS under UV light, (D) LB medium under UV light.</DIV></DIV></DIV></DD></DL><DL><DD>Although we initially used the LB medium as a solvent, the colors of inks were not so clear because LB medium has high levels of background fluorescence (autofluorescence). To find the proper solvent for BioArt, we tried out various solutions. Fig.8 shows the comparison of autofluorescence between PBS buffer and LB medium. As shown in Figs. 8C and 8D, PBS buffer has lower levels of autofluorescence than LB medium. As far as we tried out, PBS buffer was one of the best solvent of <I>E.coli</I> cells and we adopted PBS buffer as the solvent for the <I>E.coli</I> Pen.
</DD></DL><DL><DD>We succeeded to make several ink colors such as Red ink, Orange ink, Yellow ink, and Green ink. Additionally, we succeeded to develop various colors by mixing the Green ink and Red ink. 
</DD></DL><DL><DD><DIV class="thumb tleft"><DIV class="thumbinner" style="width:222px;"><DIV class="thumbcaption">Fig.9  Yellow is made by mixing Red ink and Green ink.(A)Red ink by RFP expression under UV light, (B) Green ink by GFP expression under UV light, (C) Yellow ink by mixing RFP:GFP=1:2 under UV light.</DIV></DIV></DIV></DD></DL><DL><DD>To make Green ink and Red ink, we used K362001 on pSB6A1, a GFP expression vector,  and J04450 on pSB1C3, RFP expression vector, respectively.
</DD></DL><P>Furthermore, we mixed Green ink and Red ink to make other colors such as Orange ink and Yellow ink. Fig.9C shows one of a new color ink, Yellow, made by mixing equal amounts of Green ink and Red ink. 
</P><DL><DD><DIV class="thumb tleft"><DIV class="thumbinner" style="width:402px;"><DIV class="thumbcaption">Fig.10 Various colors of our bio inks.(A)Red ink, (B) Orange ink, (C) Yellow ink ,(D)GFP ink, and (E)PBS under UV light.</DIV></DIV></DIV></DD></DL><DL><DD>To make Orange ink, we mixed Green ink and Red ink in a ratio of 1 : 2. To make Yellow ink, we mixed Green ink and Red ink in a ratio of 2 : 1. Of course, we can create various colors by mixing inks in a various ratio. 
</DD></DL><DL><DD><B>4-2.  Art works with bio inks.</B></DD></DL><DL><DD>At first, we used newspaper as drawing papers. However, we found that newspaper has high levels of intrinsic fluorescence and is not suitable for BioArt (Fig.11). Indeed, pictures drawn with bio inks on newspaper were dull and not clear. Therefore, we tried to find the proper paper for BioArt. As shown in Fig.11, ‘‘Kimwipes’’, a type of cleaning tissue commonly used in laboratories, did not show almost any intrinsic fluorescence. Why do some papers have intrinsic fluorescence? It is well known that lignin, commonly derived from wood, has a strong intrinsic fluorescence, and some papers are rich in lignin. That is one of reasons why some papers showed high levels of intrinsic fluorescence. On the other hand, little lignin is contained in Kimwipes. Therefore, we concluded that Kimwipes is the best drawing papers for BioArt with the <I>E.coli</I> Pen.
</DD></DL><DL><DD><DIV class="thumb tleft"><DIV class="thumbinner" style="width:411px;"><DIV class="thumbcaption">Fig.11 Intrinsic fluorescence of newspaper and Kimwipes. Newspaper (A) and Kimwipes (B) under UV light.</DIV></DIV></DIV></DD></DL><DL><DD>Using bio inks and Kimwipes, we drew and painted lots pictures, marks, letters, etc. Various examples of BioArt works are shown as follows. Please enjoy our art works !!!
</DD></DL><DL><DD><DIV class="thumb tleft"><DIV class="thumbinner" style="width:440px;"><DIV class="thumbcaption">Fig.12  Samples are as follows: 1, no ink; 2, PBS; 3, Green ink; 4, Yellow ink; 5, Orange ink; 6, Red ink. Samples were irradiated without UV light, (A) or with UV light (B).</DIV></DIV></DIV></DD></DL><DL><DD>We used these inks in order to make samples. Next figures shows some picture that we drew.
</DD></DL><TABLE><TBODY><TR><TD><DIV align="center"><DIV class="thumb tleft"><DIV class="thumbinner" style="width:276px;"><DIV class="thumbcaption">Fig.13 Maple leaves</DIV></DIV></DIV><DL><DD><DIV class="thumb tleft"><DIV class="thumbinner" style="width:180px;"><DIV class="thumbcaption">Fig.14 Bamboo</DIV></DIV></DIV></DD></DL></DIV></TD></TR><TR><TD><DIV align="center"><DIV class="thumb tleft"><DIV class="thumbinner" style="width:316px;"><DIV class="thumbcaption">Fig.15 Emblem of Samurai ,Toyotomi Hideyosi</DIV></DIV></DIV><DIV class="thumb tleft"><DIV class="thumbinner" style="width:298px;"><DIV class="thumbcaption">Fig.16 Japanese plum</DIV></DIV></DIV></DIV></TD></TR><TR><TD><DIV align="center"><DIV class="thumb tleft"><DIV class="thumbinner" style="width:275px;"><DIV class="thumbcaption">Fig.17 Cherry blossoms</DIV></DIV></DIV><DIV class="thumb tleft"><DIV class="thumbinner" style="width:343px;"><DIV class="thumbcaption">Fig.18 Butterfly</DIV></DIV></DIV></DIV></TD></TR><TR><TD><DIV align="center"><DIV class="thumb tleft"><DIV class="thumbinner" style="width:238px;"><DIV class="thumbcaption">Fig.19 Ukiyoe</DIV></DIV></DIV><DIV class="thumb tleft"><DIV class="thumbinner" style="width:153px;"><DIV class="thumbcaption">Fig.20 Carp</DIV></DIV></DIV><DIV class="thumb tleft"><DIV class="thumbinner" style="width:156px;"><DIV class="thumbcaption">Fig.21 Crane</DIV></DIV></DIV><DIV class="thumb tleft"><DIV class="thumbinner" style="width:171px;"><DIV class="thumbcaption">Fig.22 Ginkgo</DIV></DIV></DIV></DIV></TD></TR></TBODY></TABLE><H2><SPAN class="mw-headline" id="References"> References </SPAN></H2><DL><DD><OL><LI><A href="http://www.ncbi.nlm.nih.gov/pubmed/11443092" class="external text" rel="nofollow">PMID: 11443092</A> Ming Zheng, Xunde Wang, Bernard Doan, Karen A. Lewis, Thomas D. Schneider, Gisela Storz,Computation-Directed Identification of OxyR DNA Binding Sites in <I>Escherichia coli</I>,Journal of Bacteriology,2001 Aug;183(15):4571-79.
</LI><LI><A href="http://www.ncbi.nlm.nih.gov/pubmed/10913087" class="external text" rel="nofollow">PMID: 10913087</A> Urs A. Ochsner, Michel L. Vasil, Eyad Alsabbagh, Kislay Parvatiyar, Daniel Hassett,Role of the Pseudomonas aeruginosa oxyR-recG operon in oxidative stress defense and DNA repair: OxyR-dependent regulation of katB-ankB, ahpB, and ahpC-ahpF.,Journal of Bacteriology,2000 Aug;182(16):4533-44.
</LI><LI><A href="http://ci.nii.ac.jp/naid/110001710476" class="external text" rel="nofollow">NAID: 110001710476</A> Nunoshiba Tatsuo,Mechanisms for the oxidative stress response and oxidative mutagenesis in <I>Escherichia coli</I>.,Environmental mutagen research,2001 June 30;23(1):23-32.
</LI><LI><A href="http://www.ncbi.nlm.nih.gov/pubmed/12876288" class="external text" rel="nofollow">PMID: 12876288</A> Laurent Loiseau, Sandrine Ollagnier-de-Choudens, Laurence Nachin, Mare Fontecave, Frederie Barras,Biogenesis of Fe-S cluster by the bacterial Suf system: SufS and SufE form a new type of cysteine desulfurase.,The Journal of Biological Chemistry,2003 Oct 3;278(40):38352-9.
</LI><LI><A href="http://www.ncbi.nlm.nih.gov/pubmed/18849427" class="external text" rel="nofollow">PMID: 18849427</A> Joon-Hee Lee, Won-Sik Yeo, Jung-Hye Roe,Induction of the sufA operon encoding Fe-S assembly proteins by superoxide generators and hydrogen peroxide: involvement of OxyR, IHF and an unidentified oxidant-responsive factor.,Molecular Microbiology,2008 Dec;190(24):8244-7.
</LI><LI><A href="http://www.ncbi.nlm.nih.gov/pubmed/16621810" class="external text" rel="nofollow">PMID: 16621810</A> Ramakrishnan Balasubramanian,  Gaozhong Shen,  Donald A. Bryant, John H. Golbeck,Regulatory Roles for IscA and SufA in Iron Homeostasis and Redox Stress Responses in the Cyanobacterium Synechococcus sp. Strain PCC 7002.,Journal of Bacteriology,2006 May;188(9):3182-91.
</LI><LI><A href="http://www.ncbi.nlm.nih.gov/pubmed/11443091" class="external text" rel="nofollow">PMID: 11443091</A> Zheng M, Wang X, Templeton LJ, Smulski DR, LaRossa RA, Storz G.,DNA microarray-mediated transcriptional profiling of the Escherichia coli response to hydrogen peroxide.,Journal of Bacteriology,2001 Aug;183(15):4562-70.
</LI><LI><A href="http://www.ncbi.nlm.nih.gov/pubmed/12889026" class="external text" rel="nofollow">PMID: 12889026</A> Lu C, Bentley WE, Rao G.,Comparisons of oxidative stress response genes in aerobic <I>Escherichia coli</I> fermentations.,Biotechnology and Bioengineering,2003 Sep 30;83(7):864-70.
</LI><LI><A href="http://ci.nii.ac.jp/naid/110000056227" class="external text" rel="nofollow">NAID: 110000056227</A> Takebe So,  Azuma Megumi, Yoshida Megumi,  Yoshida Aya,Effect of cAMP-CRP regulation on sodA promoter activity.,Kyoto Women's University food journal,1995 Dec 10;50:37-42.
</LI><LI><A href="http://www.ncbi.nlm.nih.gov/pubmed/10585871" class="external text" rel="nofollow">PMID: 10585871</A> Mitsumoto A, Kim KR, Oshima G, Kunimoto M, Okawa K, Iwamatsu A, Nakagawa Y.,Glyoxalase I is a novel nitric-oxide-responsive protein.,The Biochemical journal,1999 Dec 15;344 Pt 3:837-44.
</LI><LI><A href="http://www.ekouhou.net/%E6%8A%97%E4%BD%93%E3%81%BE%E3%81%9F%E3%81%AF%E5%A5%BD%E4%B8%AD%E7%90%83%E3%82%92%E4%BB%8B%E3%81%99%E3%82%8B%E3%82%AA%E3%82%BE%E3%83%B3%E7%94%9F%E6%88%90/disp-A,2006-506613.html" class="external text" rel="nofollow">PCT/EP2003/012710</A> Novartis, The Scripps Research Institute,Generation of ozone through antibody or neutrophil,2004 May 27.
</LI></OL></DD></DL></DIV><DIV class="printfooter">
Retrieved from &quot;<A href="http://2010.igem.org/Team:KIT-Kyoto/Project/Abstract">http://2010.igem.org/Team:KIT-Kyoto/Project/Abstract</A>&quot;</DIV></DIV></DIV><DIV id="footer-box" class="noprint"><DIV id="footer"><UL id="f-list"><LI id="t-recentchanges"><A href="/Special:RecentChanges" title="Recent changes">Recent changes</A></LI><LI id="t-whatlinkshere"><A href="/Special:WhatLinksHere/Team:KIT-Kyoto/Project/Abstract" title="List of all wiki pages that link here [j]" accesskey="j">What links here</A></LI><LI id="t-recentchangeslinked"><A href="/Special:RecentChangesLinked/Team:KIT-Kyoto/Project/Abstract" title="Recent changes in pages linked from this page [k]" accesskey="k">Related changes</A></LI><LI id="t-specialpages"><A href="/Special:SpecialPages" title="List of all special pages [q]" accesskey="q">Special pages</A></LI><LI><A href="/Special:Preferences">My preferences</A></LI></UL></DIV><DIV id="footer"><UL id="f-list"><LI id="t-print"><A href="/wiki/index.php?title=Team:KIT-Kyoto/Project/Abstract&amp;printable=yes" title="Printable version of this page [p]" accesskey="p">Printable version</A></LI><LI id="t-permalink"><A href="/wiki/index.php?title=Team:KIT-Kyoto/Project/Abstract&amp;oldid=196993" title="Permanent link to this revision of the page">Permanent link</A></LI><LI id="privacy"><A href="/2010.igem.org:Privacy_policy" title="2010.igem.org:Privacy policy">Privacy policy</A></LI><LI id="disclaimer"><A href="/2010.igem.org:General_disclaimer" title="2010.igem.org:General disclaimer">Disclaimers</A></LI></UL></DIV></DIV></DIV></BODY></HTML>