"
Page
Discussion
View source
History
teams
Log in
 
Team:Heidelberg/Notebook/miRNA Kit/October
From 2010.igem.org
Home
Team
Overview
Students
Instructors
Advisors
Institutes
Gallery
miDANCE
Acknowledgements
 
Project
Abstract
Introduction
Synthetic miRNA Kit
Capsid Shuffling
    Homology Based
    ViroBytes
miMeasure
in vivo Study
Summary & Outlook
References
Contributions
 
Parts
Parts
Characterization
 
Modeling
Introduction
miBEAT
miRockdown
miBS designer
mUTING
Descriptions
 
Notebook
Synthetic miRNA Kit
Capsid Shuffling
    Homology Based
    ViroBytes
miMeasure
Binding Site Design
in vivo Study
Modeling
Materials
Methods
Psychological Survey
Philosophical Reflection
Biosafety
 
Human Practices
Psychological Survey
Philosophical Reflection
 
Sponsors
August
M
T
W
T
F
S
S
1
2
3
4
5
6
7
8
9
10
11
12
13
14
15
16
17
18
19
20
21
22
23
24
25
26
27
28
29
30
31
September
M
T
W
T
F
S
S
1
2
3
4
5
6
7
8
9
10
11
12
13
14
15
16
17
18
19
20
21
22
23
24
25
26
27
28
29
30
-
October
M
T
W
T
F
S
S
1
2
3
4
5
6
7
8
9
10
11
13
14
15
16
17
18
19
20
21
22
23
24
25
26
27
28
29
30
-
 
01/10/2010
Gel 101001-1. Digestion of miniprep no. 8, 9, 14 and 16 with EcoRI and PstI. Odd lanes are undigested controls, even lanes are digested minis
repressor construct:
colony 8, 9, 14 and 16 were chosen for miniprep. They consist of the complete TetR construct in pBS 1C3
digestion of the minis with EcoRI and PstI for cloning into pSB1A3 , SAP of pSB1A3 , digestion was tested on gel and proven positive (see Gel 101001-1)
nucleotide removal, then ligation of TetR8 and TetR14 equimolar with SAPed backbone, TetR9 and TetR16 2:1 with not-SAPed backbone, transformation with TOP10 cells (10µl on 50µl E. coli)
Screening 1: shRNA10 single binding sites
Screening 2: shRNA10 single binding sites
Screening of the binding sites cloned into the constructs K3 and K4 containing shRNA10 in pSB1A3 on the previous day. 16 colonies were picked for each perfect, rand9-12 and rand9-22 binding site and colony-PCR was performed according to the colony-PCR standard protocol (Primers Luc2_bs_screen_seq and sh10_ape_xma_screen_rc). The results are shown on the gels Screening1 and Screening 2. Indicated are the construct numbers (either K3 or K4 , which differ in the promoter driving the Luc2 gene) and the binding site property (perfect, 9-12 randomized or 9-22 randomized). Almost all clones show a positive band, although only some have a really bright band at the size of ~ 250 bp. LB cultures for 10 clones for each constructs K3 and K4 were inocculated.
02/10/2010
Gel 101002-1. Colony PCR. All lanes with right amplified fragment except on lane 3, 10, 11, 15 and 23. Lane 1 & 14: 1kb Plus Ladder (Invitrogen)
Gel 101002-2. Test digestions of repressor construct. Upper pattern: EcoRI & PstI. Lower pattern: BamHI. All bands revealing positive results except in the first column (negative control from lane 3 of gel 101002-1). Lane 1 and Lane 10: 1kb Plus Ladder (Invitrogen). Lane 16, 17, 18: incomplete digestion
Miniprep of the LB cultures (constructs K3 and K4 with shRNA10 and the corresponding binding sites) were done and used for transfection of Hek-293 cells
repressor construct:
colony-PCR of 6 colonies for each plate of previous day transformations
analysis, see gel 101002-1
twice test digestion of promising mini-preps, following standard protocol recommendations
first digestion with EcoRI and PstI to check for insert size
second digestion with BamHI to check the insert
see gel 101002-2: some positive results will be send for sequencing and typed T1 and T2 , respectively
moreover: usage of one construct for transfection into HEK cells
03/10/2010
repressor construct:
sequencing results were fine
digestion and purification of T1 and T2 with XmaI and XhoI to clone in binding sites (against ON-targets) following standard protocol recommendations
annealing of binding sites using oligos
D108 & D109: once binding site for miR122
D110 & D111: twice binding site for miR122 right behind each other
protocol:
5 µl of each oligo, 5 µl of Buffer 2 (NEB), 35 µl H2O
95°C, 5'
cool down to room temperature for 2h
incubation on ice, 10'
use 1 µl for ligation with ~ 40 ng of backbone (i. e. digested T1 and T2 )
ligation (including to negative controls with backbone only to check for re-ligation without SAP treatment)
transformation and growth on selective plates (ampicillin)
04/10/2010
repressor construct:
colony-PCR of 6 colonies for each plate of previous day transformations
gel 101004-1 reveals almost positive samples
submission of each a promising mini-prep for sequencing of binding site and correct insert
05/10/2010
repressor construct:
sequencing results from previous day:
once or twice the binding site in each of the T1 constructs, respectively
once the binding site for the T2 construct
re-trial of ligation for two binding sites in T2
planned transfection into HeLa and HUH cells (non-liver and liver to prove ON-targeting) in ratio 6:1 = repressor:operator(tuning) as recommended in the manual
new approach:
purification of PCR amplified shRNA-like miRNA against hAAT ("shhAAT")
digestion of shhAAT and Q2 and Q3 with HindIII and AflII (for backbones sequential because of two inner AflII cutting sites)
ligation in triplicates (using 70 ng of backbone and 10, 15 or 20 ng of insert, respectively)
transformation , over-night growth on selective agar plates (ampicillin)
06/10/2010
waiting for colonies from previous night, which did not grow after all
07/10/2010
repetition of shhAAT cloning
touch-down PCR setup: 8.5 µl ddH2O, 0.5 µl template DNA (pcDNA5, 50 ng/µl), 0.5 µl each primer (i.e. D7 & D16 , 10 µl Phusion PCR MasterMix (2x)
touch-down PCR protocol: like this but with increase of annealing temperature from 68°C to 61°C (i. e. 0.2°C per cycle)
go on like 05/10/2010 but growth on freshly prepared plates
08/10/2010
mini-prep and submission for sequencing for the only promising colony of Q2 containing shhAAT for sequencing
09/10/2010
sequencing the day before was fine but construct contains TetO2, thus: maybe Q2 and Q3 were confused
that would explain why measurements with TetR constructs are not working properly even though sequences were fine
cloning of binding sites for shhAAT (perfect binding site generated via annealing, and randomized ones PCR amplified)
touch-down PCR setup: 17 µl ddH2O, 4 µl each oligo (i.e. respective forward primer + second strand oligo D121 ), 25 µl Phusion PCR MasterMix (2x)
touch-down PCR protocol: like this but with increase of annealing temperature from 70°C to 62°C (i. e. 0.5°C per cycle, 16x) and four additional cycles with 62°C
annealing of perfect binding against shhAAT sites with an oligo-based strategy following the protocol from the 3rd October
digestion of PCR amplified binding sites with AgeI and XhoI, digestion of Q2 backbone containing the shhAAT with XhoI and XmaI, afterwards heat inactivation, nucleotide removal and ligation following standard protocol recommendations
transformation and over-night growth on selective agar plates (ampicillin)
10/10/2010
Gel 101010-1. Colony PCR for construct containing shhAAT and binding sites. Expected is an amplified fragment a bit smaller than 300 bp which can be seen on lane 3, 4, 6, 7, 8 (perfect binding sites), 12, 13, 14, 16 and 17 (randomized binding sites). Lane 1 and Lane 26: 1kb Plus Ladder (Invitrogen)
colony pcr reveals some positive clones (see gel 101010-1) from previous day
inoculation of 5 ml LB Amp cultures for mini-prep over night
ON system test
Dual Luciferase Assay
Tuning construct in which luc2 expression is driven by CMV promoter with TetO2 also expressing either miR122 or miR sAg was cotransfected with Tet repressor under regulation of perfect miR sAg  binding site. Amounts transfected were: 2.5ng of tuning construct , 20ng of miRNA expressing plamid (either miR122 or miR sAg) and 5ng of repressor construct. To keep transfection efficiency the same we used sheared salmon sperm DNA as a stuffer.
In presence of Tet repressor and miR122 that is not targeting it, less of firefly luciferase is expressed, but when miR sAg is expressed is recovered to 60% of original activity.
11/10/2010
mini-prep of ten cultures from previous day and direct transfection to test the system
amount: each 100 µl (c=2.5 ng/µl)
measurement construct for single binding sites:
PCR-based generation of binding sites against miR122 (perfect and randomized)
touch-down PCR setup:
17 µl ddH2O,
4 µl each oligo (i.e. respective forward primer D71, D62 or D5 + second strand oligo D13 ),
25 µl Phusion PCR MasterMix (2x)
touch-down PCR protocol:
like this but with increase of annealing temperature from 70°C to 62°C (i. e. 0.5°C per cycle, 16x)
and four additional cycles with 62°C, elongation time only 10 seconds
digestion of generated binding sites and the pSMB_miMeasure backbone with EcoRI and PstI, then nucleotide removal
ligation with ~ 70 ng vector and between 7.5 ng and 15 ng insert
transformation
12/10/2010
measurement construct for single binding sites:
colony pcr of each eight of load of colonies + two negative controls (sum: 50 samples)
protocol: 16 cycles touch-down pcr from 70°C to 62°C of annealing temperature, then 19 additional cycles with 62°C annealing temperature
elongation time: 90''
no positive results, maybe due to insufficient screening primers (one for BBb standard, the other one for CMV resulting in an 1300 bp fragment)
as a consequence: repetition of entire cloning with SAP treated backbone following day
13/10/2010
measurement construct for single binding sites:
repetition of cloning from 11/10/2010
note: vector concentration of pSMB_miMeasure is okay but after digestion and nucleotide removal it is always pretty low
(loss of 75% of DNA on average whereas other DNA yields to 90% of purified DNA as compared to digested amount)
14/10/2010
cloning of 47 - 58 into pSB_H1
cloning of tuning constructs for in vivo measurement:
cloning of binding sites of miRNA haat with perfect and several imperfect binding
oligo annealing following the standard protocol
digestion of backbone pbsU6 with NotI and XhoI
ligation and transformation
15/10/2010
Screening of M1 to M11 constructs, 4 clones were picked from each plate. 300bp band marks positive clones
Screening of M12 to M22 constructs, 4 clones were picked from each plate. 300bp band marks positive clones
Screening of measurement standard with perfect, 9-12 and 9-22 randomized binding sites. constructs, 8 clones were picked from each plate. 300bp band marks positive clones
screening of plates M1 to M11 of the transformation with the construct pBSU6_SV40_Luc2 with 11 different binding sites for sh hAAT. Four clones from each plate were screened by colony pcr, positives give a band at 300bp. Primers Luc2_bs_screen_seq_fw and reverse primerse are the reverse binding site oligos for the binding sites.
inoculation of miniprep cultures for one positive from each plate.
transformation of miRsAg and mir122 expression constructs
screening of measurement standart cloning from 13th with perfect, 9-12 randomized and 9-22 randomized binding sites for miRNA 122. primers mir122_mimeasure_screen_rev and pSMB_miMeasure_screen_fw. Positives give a band at 300bp.
inoculation of miniprep cultures for positive clones
screening of plates M12 to M22 of the transformation with the measurement plasmid with 11 different binding sites for miRsAg. Four clones from each plate were screened by colony pcr, positives give a band at 300bp. Primers pSMB_miMeasure_Sequencing_rev and forward primerse are the forward primer oligos for the binding sites.
inoculation of miniprep cultures for one positive from each plate.
16/10/2010
mini Prep of mimeasure colonies (4 each)
mini prep of haat constructs
inoculation of miRsAg and mir122 expression constructs
Maxi Preps of constructs M1 till M10
inoculation for Maxipreps of Adenovirus 5, AAV8 rep and cap genes and pBSU6 H1 haat construct
inoculation for Maxiprep of constructs M11 an M23-29
cloning of on targeting constructs with a new strategy...
17/10/2010
annealing of eleven sAg binding sites using oligos
protocol:
5 µl of each oligo, 5 µl of Buffer 2 (NEB), 35 µl H2O
95°C, 5'
cool down to room temperature for 2h
incubation on ice, 10'
use 1 µl for ligation with ~ 40 ng of backbone (i. e. with NotI and XhoI digested pBS_U6 containing hAAt cDNA)
transformation and growth on selective plates (ampicillin)
18/10/2010
colony-PCR of 4 colonies for each plate of previous day transformations using reverse binding site oligo and forward hAAT primer expecting a fragment round about 1300 bp
gel 101018-1 reveals almost positive samples except two (data not yet shown)
19/10/2010
mini-preps of cloned shAAT constructs
dilution and co-transfection with miRsAg expressing construct into HeLa cells for ELISA measurements
23/10/2010
seeding HeLa cells
24/10/2010
Transfection
1) HeLa luc2-miR122BS and miR122 2) for ELISA - pBS_U6 containing hAAT with miRsAg imperfect binding sites, and miRsAg  ratios 1:1 ans 4:1
26/10/2010
Dual Luciferase Assay
Figure 3: Tuning of gene expression through different imperfect miR122 binding sites in psiCHECK-2. Construct was transfected into HeLa cells together with an plasmid expressing miR122. Control without binding site was used for normalization.
Figure 1: Tuning of gene expression through different imperfect shRNA miR binding sites in pBS_U6 Gene expression quantified via dual luciferase assay for constructs containing different imperfect binding sites for shhAAT.
Figure 2: Tuning of gene expression through different imperfect shRNA miR binding sites in pBS_H1. Gene expression quantified via dual luciferase assay for constructs containing different imperfect binding sites for shhAAT.
ELISA was performed on media in which HeLa cells we grown, as hAAT is secreted by cells.
Retrieved from " http://2010.igem.org/Team:Heidelberg/Notebook/miRNA_Kit/October "
Recent changes
What links here
Related changes
Special pages
My preferences
Printable version
Permanent link
Privacy policy
Disclaimers
