"
Page
Discussion
View source
History
teams
Log in
 
Team:Heidelberg/Project/Introduction
From 2010.igem.org
Home
Team
Overview
Students
Instructors
Advisors
Institutes
Gallery
miDANCE
Acknowledgements
 
Project
Abstract
Introduction
Synthetic miRNA Kit
Capsid Shuffling
    Homology Based
    ViroBytes
miMeasure
in vivo Study
Summary & Outlook
References
Contributions
 
Parts
Parts
Characterization
 
Modeling
Introduction
miBEAT
miRockdown
miBS designer
mUTING
Descriptions
 
Notebook
Synthetic miRNA Kit
Capsid Shuffling
    Homology Based
    ViroBytes
miMeasure
Binding Site Design
in vivo Study
Modeling
Materials
Methods
Psychological Survey
Philosophical Reflection
Biosafety
 
Human Practices
Psychological Survey
Philosophical Reflection
 
Sponsors
Introduction
Motivation
Introduction
Motivation Referring to the team abstracts, most of the iGEM Teams are still dealing with what we would call "classical synthetic biology", meaning bacterial engineering. Since the last year's project established to some degree "mammalian synthetic biology", they paved the way for entering the real medical track focussing on gene therapy. The most important issue in terms of gene therapy is regulation of transgene expression. After last year's team characterized synthetic promoters for tuning of gene expression in vitro, we have taken one step further, introducing synthetic microRNAs and their binding sites as a new level of expression control.
Background Micro RNAs (miRNAs), a class of abundant small noncoding RNAs, are key regulators in all kinds of organisms ranging from viruses to mammals. By binding to target sequences most commonly found in the 3' untranslated region (UTR) of the mRNA, miRNAs inhibit the translation of their target mRNAs and thereby adjust the expression of many proteins related to the miRNA expression in a cell (Brenecke et al. 2005) . The importance of miRNA-mediated gene regulation is impressively reflected by the fact that roughly 1% of the human genome codes for miRNAs which target 20-25% of all protein coding genes (Lewis et al. 2005) . Therefore, a large proportion of the transcriptome may be subjected to miRNA-mediated control (Lau et al. 2001) . The broad regulatory scope of miRNAs underlines their key roles in a wide range of biological processes including proliferation, apoptosis, hematopoeisis and oncogenesis (Bushati and Cohen, 2007) . The expression patterns of miRNAs in different cell types, tissues and developmental stages of a cell vary highly, but remain relatively constant within a certain single cell type in a certain stage (Brown et al. 2006 , Gangaragu and Lin 2009) . The special properties of miRNA binding sites and the inhibiting character of miRNAs are excellent devices to be exploited for gene therapy. Not only the fine-tuning capabilities of varying miRNA binding sites but also tissue specificity can play an important role (Brenecke et al., 2005).
A combination of random and rational design of binding sites could become a powerful tool to achieve a narrow range of resulting gene expression knockdown. To ease in silico construction of miRNA binding sites with appropriate characteristics for its target, we wrote a program - the miBS designer. Using all of our theoretical models gives the user the opportunity to calculate knockdown percentages caused by the designed miRNA in the target cell.  The miBS designer allows for in silico construction of binding sites but also rational design is possible. The readily constructed binding sites can then be introduced into the miMeasure standard plasmid. This plasmid has been engineered to enable the easy input of synthetic microRNA binding sites behind one of two fluorescent proteins while the second is used for normalization. Expression of regulated reporter and control from a bidirectional CMV promter guarantee faithful and reproducible measurements in any kind of cell. The fluorescence readout can be used to quantify the regulatory efficiency of the binding site in knockdown percentage. Once the properties of a synthetic binding site are elucidated, they can be used to manipulate and accurately fine-tune gene expression in vitro and in vivo.
After having characterized the binding sites via fluorescent measurement they can be used in our synthetic miRNA Kit. This guarantees at least for individually modifiable but still ready-to-use constructs to interfere genetic circuits with synthetic or endogenous miRNAs. We preciously show, that gene expression can thereby by adjusted - tuned - to an arbitrary level. The miTuner (see sidebar) allows on the simultaneous expression of a synthetic miRNA and a gene of interest that is fused with a designed binding site for this specific miRNA. Our modular kit comes with different parts that can be combined by choice, e. g. different mammalian promoters and characterized binding sites of specific properties. By choosing a certain binding site to tag the GOI, one can tune the expression of this gene. Depending on the GOI, different means for read out of gene expression come into play. At first, we applied dual-luciferase assay, since we used Luciferase as a reporter for a proof-of-principle approach. Later on, semi-quantitative immunoblots were prepared for testing of therapeutic genes. However, all the received information fed our models, thereby creating an integrative feedback loop between experiments and in silico simulation.
Next to that we follow two virus shuffling approaches to re-engineer the AAV capsid gene by directed evolution. This is used for production of a virus that is specifically infected into hepatocytes. We test the hepatocyte specificity of the fittest AAV clone via a luciferase reporter in living mice. We show that organ specific targeting is readily achieved in our mouse model. The following project sections give more background on the individual topics and provide detailed overviews on the corresponding results. A detailed documentation of the laboratory work can further be found in the notebook.
For the first time, we could show miRNA mediated fine tuning and cell targeting in vitro and in vivo. Furthermore we wanted to create a library of synthetic viruses that could be evolved to enhance target specificity of our approach. In the combination of the two, we see doors open to the future of RNA based gene regulatory therapy.
Retrieved from " http://2010.igem.org/Team:Heidelberg/Project/Introduction "
Recent changes
What links here
Related changes
Special pages
My preferences
Printable version
Permanent link
Privacy policy
Disclaimers
