<HTML xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en" dir="ltr">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki  ltr ns-0 ns-subject page-Team_Heidelberg_Parts_Characterization"><DIV id="globalWrapper"><DIV id="top-section"><DIV id="p-logo"><A href="/Main_Page" title="Main Page">&quot;
	    </A></DIV><DIV id="menubar" class="left-menu noprint"><UL><LI class="selected"><A href="/Team:Heidelberg/Parts/Characterization">Page               </A></LI><LI class="new"><A href="/wiki/index.php?title=Talk:Team:Heidelberg/Parts/Characterization&amp;action=edit&amp;redlink=1">Discussion               </A></LI><LI><A href="/wiki/index.php?title=Team:Heidelberg/Parts/Characterization&amp;action=edit">View source               </A></LI><LI><A href="/wiki/index.php?title=Team:Heidelberg/Parts/Characterization&amp;action=history">History               </A></LI><LI style="color:#808080;cursor:default">teams</LI></UL></DIV><DIV class="right-menu noprint" id="menubar"><UL><LI id="pt-login"><A href="/wiki/index.php?title=Special:UserLogin&amp;returnto=Team:Heidelberg/Parts/Characterization" title="You are encouraged to log in; however, it is not mandatory [o]" accesskey="o">Log in</A></LI></UL></DIV><DIV id="search-controls" class="noprint"><FORM action="/Special:Search" id="searchform"> </FORM></DIV></DIV><DIV id="content"><H1 class="firstHeading">Team:Heidelberg/Parts/Characterization</H1><DIV id="bodyContent"><H3 id="siteSub" class="noprint">From 2010.igem.org</H3><DIV id="header"><DIV id="menu"><UL id="nav"><LI id="menu_home"><A class="aHome" href="https://2010.igem.org/Team:Heidelberg">Home</A></LI><LI id="menu_team"><A class="aTeam" href="https://2010.igem.org/Team:Heidelberg/Team">Team</A><TABLE class="menutable" cellpadding="0" cellspacing="0"><TBODY><TR><TD class="lefttdtopteam"><A class="aTeamOverview" href="https://2010.igem.org/Team:Heidelberg/Team">Overview</A><A class="aTeamMembers" href="https://2010.igem.org/Team:Heidelberg/Team/Members">Students</A><A class="aTeamMembers" href="https://2010.igem.org/Team:Heidelberg/Team/Members#Instructors">Instructors</A><A class="aTeamMembers" href="https://2010.igem.org/Team:Heidelberg/Team/Members#Advisors">Advisors</A><A class="aInstitutes" href="https://2010.igem.org/Team:Heidelberg/Team/Institutes">Institutes</A><A class="aGallery" href="https://2010.igem.org/Team:Heidelberg/Team/Gallery">Gallery</A><A class="aDance" href="https://2010.igem.org/Team:Heidelberg/midance">miDANCE</A><A class="amithanks" href="https://2010.igem.org/Team:Heidelberg/Team/miThanks">Acknowledgements</A></TD><TD class="righttdtop"> </TD></TR></TBODY></TABLE></LI><LI id="menu_project"><A class="aProject" href="https://2010.igem.org/Team:Heidelberg/Project">Project</A><TABLE class="menutable" cellpadding="0" cellspacing="0"><TBODY><TR><TD class="lefttdtopproject"><A class="aAbstract" href="https://2010.igem.org/Team:Heidelberg/Project">Abstract</A><A class="aIntroduction" href="https://2010.igem.org/Team:Heidelberg/Project/Introduction">Introduction</A><A class="aProject_miRNA_Kit" href="https://2010.igem.org/Team:Heidelberg/Project/miRNA_Kit">Synthetic miRNA Kit</A><A class="aPro_Capsid_Shuffling" href="https://2010.igem.org/Team:Heidelberg/Project/Capsid_Shuffling">Capsid Shuffling</A><A class="aPro_Homology" href="https://2010.igem.org/Team:Heidelberg/Project/Capsid_Shuffling/Homology_Based">    Homology Based</A><A class="aPro_Virobytes" href="https://2010.igem.org/Team:Heidelberg/Project/Capsid_Shuffling/ViroBytes">    ViroBytes</A><A class="aPro_miMeasure" href="https://2010.igem.org/Team:Heidelberg/Project/miMeasure">miMeasure</A><A class="aPro_Mouse" href="https://2010.igem.org/Team:Heidelberg/Project/Mouse_Infection"><I>in vivo</I> Study</A><A class="aPro_Summary" href="https://2010.igem.org/Team:Heidelberg/Project/Summary">Summary &amp; Outlook</A><A class="aPro_Ref" href="https://2010.igem.org/Team:Heidelberg/Project/References">References</A><A class="aPro_Ref" href="https://2010.igem.org/Team:Heidelberg/Project/Contributions">Contributions</A></TD><TD class="righttdtop"> </TD></TR></TBODY></TABLE></LI><LI id="menu_parts"><A class="aParts" href="https://2010.igem.org/Team:Heidelberg/Parts">Parts</A><TABLE class="menutable" cellpadding="0" cellspacing="0"><TBODY><TR><TD class="lefttdtop"><A class="aparts_ger" href="https://2010.igem.org/Team:Heidelberg/Parts">Parts</A><A class="aparts_char" href="https://2010.igem.org/Team:Heidelberg/Parts/Characterization">Characterization</A></TD><TD class="righttdtop"> </TD></TR></TBODY></TABLE></LI><LI id="menu_modeling"><A class="aModeling" href="https://2010.igem.org/Team:Heidelberg/Modeling">Modeling</A><TABLE class="menutable" cellpadding="0" cellspacing="0"><TBODY><TR><TD class="lefttdtop"><A class="amode_over" href="https://2010.igem.org/Team:Heidelberg/Modeling">Introduction</A><A class="amiGUI" href="https://2010.igem.org/Team:Heidelberg/Modeling/miGUI">miBEAT</A><A class="amirockdown" href="https://2010.igem.org/Team:Heidelberg/Modeling/miRockdown">miRockdown</A><A class="amibsdesigner" href="https://2010.igem.org/Team:Heidelberg/Modeling/miBSdesigner">miBS designer</A><A class="amispec" href="https://2010.igem.org/Team:Heidelberg/Modeling/miSpec">mUTING</A><A class="amodelset" href="https://2010.igem.org/Team:Heidelberg/Modeling/trainingset">Descriptions</A></TD><TD class="righttdtop"> </TD></TR></TBODY></TABLE></LI><LI id="menu_notebook"><A class="aNotebook" href="https://2010.igem.org/Team:Heidelberg/Notebook">Notebook</A><TABLE class="menutable" cellpadding="0" cellspacing="0"><TBODY><TR><TD class="lefttdtop"><A class="aNote_miRNA_kit" href="https://2010.igem.org/Team:Heidelberg/Notebook/miRNA_Kit">Synthetic miRNA Kit</A><A class="aNote_Capsid_Shuffling">Capsid Shuffling</A><A class="aNote_Homology" href="https://2010.igem.org/Team:Heidelberg/Notebook/Capsid_Shuffling/Homology_Based">    Homology Based</A><A class="aNote_Virobytes" href="https://2010.igem.org/Team:Heidelberg/Notebook/Capsid_Shuffling/ViroBytes">    ViroBytes</A><A class="aNote_miMeasure" href="https://2010.igem.org/Team:Heidelberg/Notebook/miMeasure">miMeasure</A><A class="aNote_BSdesign" href="https://2010.igem.org/Team:Heidelberg/Notebook/BSDesign">Binding Site Design </A><A class="aNote_Mouse" href="https://2010.igem.org/Team:Heidelberg/Notebook/Mouse_Infection"><I>in vivo</I> Study</A><A class="aNote_Modeling" href="https://2010.igem.org/Team:Heidelberg/Notebook/Modeling">Modeling</A><A class="aNote_Material" href="https://2010.igem.org/Team:Heidelberg/Notebook/Material">Materials</A><A class="aNote_Methods" href="https://2010.igem.org/Team:Heidelberg/Notebook/Methods">Methods</A><A class="aNote_Psycho" href="https://2010.igem.org/Team:Heidelberg/Human_Practices/Psychology/Notebook">Psychological Survey</A><A class="aNote_Philo" href="https://2010.igem.org/Team:Heidelberg/Human_Practices/Philosophy/Part2">Philosophical Reflection</A><A class="aNote_Biosafety" href="https://2010.igem.org/Team:Heidelberg/Notebook/Biosafety">Biosafety</A></TD><TD class="righttdtop"> </TD></TR></TBODY></TABLE></LI><LI id="menu_human_practices"><A class="aHuman_Practices" href="https://2010.igem.org/Team:Heidelberg/Human Practices">Human Practices</A><TABLE class="menutable" cellpadding="0" cellspacing="0"><TBODY><TR><TD class="lefttdtop"><A class="aPsycho" href="https://2010.igem.org/Team:Heidelberg/Human Practices/Psychology">Psychological Survey</A><A class="aPhilo" href="https://2010.igem.org/Team:Heidelberg/Human Practices/Philosophy">Philosophical Reflection</A></TD><TD class="righttdtop"> </TD></TR></TBODY></TABLE></LI><LI id="menu_support"><A class="aSupport" href="https://2010.igem.org/Team:Heidelberg/Support">Sponsors</A></LI></UL></DIV></DIV><TABLE id="toc" class="toc"><TBODY><TR><TD><DIV id="toctitle"><H2>Contents</H2></DIV><UL><LI class="toclevel-1 tocsection-1"><A href="#Characterization"><SPAN class="tocnumber">1</SPAN><SPAN class="toctext">Characterization</SPAN></A></LI><LI class="toclevel-2 tocsection-2"><A href="#Characterization_of_promoters_in_tuning_constructs_in_T-Rex_cells"><SPAN class="tocnumber">1.1</SPAN><SPAN class="toctext">Characterization of promoters in tuning constructs in T-Rex cells</SPAN></A></LI><LI class="toclevel-2 tocsection-3"><A href="#Characterization_of_BBa_K337035_and_BBa_K337036_in_different_cell_lines"><SPAN class="tocnumber">1.2</SPAN><SPAN class="toctext">Characterization of BBa_K337035 and BBa_K337036 in different cell lines</SPAN></A></LI><LI class="toclevel-2 tocsection-4"><A href="#Characterization_of_part_binding_sites_.28BBa_K337052.2C_BBa_K337053.2C_BBa_K337054.29"><SPAN class="tocnumber">1.3</SPAN><SPAN class="toctext">Characterization of part binding sites (BBa_K337052, BBa_K337053, BBa_K337054)</SPAN></A></LI><LI class="toclevel-2 tocsection-5"><A href="#Characterization_of_synthetic_single_microRNA_binding_sites_.28BBa_K337055.29"><SPAN class="tocnumber">1.4</SPAN><SPAN class="toctext">Characterization of synthetic single microRNA binding sites (BBa_K337055)</SPAN></A></LI><LI class="toclevel-2 tocsection-6"><A href="#Characterization_of_synthetic_microRNA_binding_site_patterns_against_endogenous_miR122_3.1BBa_K337008_and_1.3_BBa_K337000"><SPAN class="tocnumber">1.5</SPAN><SPAN class="toctext">Characterization of synthetic microRNA binding site patterns against endogenous miR122 3.1BBa_K337008 and 1.3 BBa_K337000</SPAN></A></LI><LI class="toclevel-2 tocsection-7"><A href="#Characterization_of_the_bidirectional_CMV_promoter"><SPAN class="tocnumber">1.6</SPAN><SPAN class="toctext">Characterization of the bidirectional CMV promoter</SPAN></A></LI><LI class="toclevel-2 tocsection-8"><A href="#References"><SPAN class="tocnumber">1.7</SPAN><SPAN class="toctext">References</SPAN></A></LI></UL></TD></TR></TBODY></TABLE><P> </P></DIV><H1><SPAN class="mw-headline" id="Characterization">Characterization</SPAN></H1><P>
We characterized intensively four different part groups:
</P><UL><LI> the 8 engineered miTuner construct differing in the promoter combinations (SV40, RSV, CMV, CMV_TetO2) the cassettes on the parts are expressed from (&lt;partinfo&gt;BBa_K337036&lt;/partinfo&gt;, &lt;partinfo&gt;BBa_K337038&lt;/partinfo&gt;, &lt;partinfo&gt;BBa_K337032&lt;/partinfo&gt;, &lt;partinfo&gt;BBa_K337035&lt;/partinfo&gt;, &lt;partinfo&gt;BBa_K337040&lt;/partinfo&gt;, &lt;partinfo&gt;BBa_K337042&lt;/partinfo&gt;, &lt;partinfo&gt;BBa_K337044&lt;/partinfo&gt;, &lt;partinfo&gt;BBa_K337046&lt;/partinfo&gt;)
</LI><LI> the pSMB_miMeasure Standard Plasmid (&lt;partinfo&gt;BBa_K337049&lt;/partinfo&gt;)
</LI><LI> a whole variety of different microRNA binding sites out of which we submitted 6 interesting candidates to the registry. (hsa-mir122 and shRNAmir_hAAT binding sites, parts &lt;partinfo&gt;BBa_K337052&lt;/partinfo&gt;, &lt;partinfo&gt;BBa_K337053&lt;/partinfo&gt;, &lt;partinfo&gt;BBa_K337054&lt;/partinfo&gt;, &lt;partinfo&gt;BBa_K337055&lt;/partinfo&gt;, &lt;partinfo&gt;BBa_K337056&lt;/partinfo&gt;, &lt;partinfo&gt;BBa_K337057&lt;/partinfo&gt;)
</LI><LI> MicroRNA binding site patterns consisting of more than 1 single perfect or imperfect binding sites for microRNA hsa-mir122 (&lt;partinfo&gt;BBa_K337040&lt;/partinfo&gt;, &lt;partinfo&gt;BBa_K337040&lt;/partinfo&gt;, &lt;partinfo&gt;BBa_K337040&lt;/partinfo&gt;, &lt;partinfo&gt;BBa_K337040&lt;/partinfo&gt;, &lt;partinfo&gt;BBa_K337040&lt;/partinfo&gt;, &lt;partinfo&gt;BBa_K337040&lt;/partinfo&gt;)
</LI></UL><H3><SPAN class="mw-headline" id="Characterization_of_promoters_in_tuning_constructs_in_T-Rex_cells">Characterization of promoters in tuning constructs in T-Rex cells</SPAN></H3><P><A href="https://2010.igem.org/Team:Heidelberg/Notebook/Methods#Dual_Luciferase_Assay" class="external text" rel="nofollow">Dual Luciferase Assay</A>
In order to test for promoter efficiency and to check whether the miRNA kit assembly works fine
50ng of each construct with different promoter set-ups (table 1) were transfected into HEK 293 T-REx cells and other cell lines HEK, HeLa, Huh7 in 96-well plate format using FuGENE transfection reagent. As every construct is expressing firefly luciferase (luc2) and renilla luciferase (hRluc) at the same time the setup allows is unaffected by transfection efficiency and cell number. Each sample was transfected and measured by Dual luciferase assay in 8 replicates. As by this time no shRNA has been cloned into plasmid no knock-down of luc2 is expected and the different expression efficiencies allow for characterization of the different promoters. <A href="http://partsregistry.org/wiki/index.php?title=Part:BBa_K337032" class="external text" rel="nofollow">BBa_K337032</A> leads to a relative luciferase unit (RLU) of luc2 to hRluc expression of 6 RLU. <A href="http://partsregistry.org/wiki/index.php?title=Part:BBa_K337035" class="external text" rel="nofollow">BBa_K337035</A> and <A href="http://partsregistry.org/wiki/index.php?title=Part:BBa_K337035" class="external text" rel="nofollow">BBa_K337035</A> are showing a comparable expression of 12 - 13 RLU, which is in line with the knowledge that both luciferases are driven by the CMV promoter.  Hek 293 T-Rex cells stably express the Tet repressor thus allows us to observe very efficient repression of Firefly luciferse expression if a CMV-TetO2 promoter is driving luc2 (<A href="http://partsregistry.org/wiki/index.php?title=Part:BBa_K337038" class="external text" rel="nofollow">BBa_K337038</A> and <A href="http://partsregistry.org/wiki/index.php?title=Part:BBa_K337046" class="external text" rel="nofollow">BBa_K337046</A>). <A href="http://partsregistry.org/wiki/index.php?title=Part:BBa_K337040" class="external text" rel="nofollow">BBa_K337040</A> transfection into Hek T-Rex cells results in an expression of 15 RLU. <A href="http://partsregistry.org/wiki/index.php?title=Part:BBa_K337042" class="external text" rel="nofollow">BBa_K337042</A> and <A href="http://partsregistry.org/wiki/index.php?title=Part:BBa_K337044" class="external text" rel="nofollow">BBa_K337044</A> are constructed in a way that luc2 is driven by the CMV promoter and hRluc is driven by the RSV promoter and show a comparable expression of 17-20 RLU. This leads to the conclusion that the CMV promoter shows comparable expression to the RSV promoter in Hek T-Rex cell lines.

Table 1
</P><TABLE class="wikitable sortable" border="0" style="text-align: center"><TBODY><TR bgcolor="#cccccc"><TH>part</TH><TH>promoter driving luc2 (Firefly)</TH><TH>promoter driving (Renilla)</TH><TH>promoter driving shRNA expression
</TH></TR><TR><TD><A href="http://partsregistry.org/wiki/index.php?title=Part:BBa_K337032" class="external text" rel="nofollow">BBa_K337032</A></TD><TD>RSV</TD><TD>CMV</TD><TD>SV40
</TD></TR><TR><TD><A href="http://partsregistry.org/wiki/index.php?title=Part:BBa_K337035" class="external text" rel="nofollow">BBa_K337035</A></TD><TD>CMV</TD><TD>CMV</TD><TD>SV40
</TD></TR><TR><TD><A href="http://partsregistry.org/wiki/index.php?title=Part:BBa_K337036" class="external text" rel="nofollow">BBa_K337036</A></TD><TD>CMV</TD><TD>CMV</TD><TD>RSV
</TD></TR><TR><TD><A href="http://partsregistry.org/wiki/index.php?title=Part:BBa_K337038" class="external text" rel="nofollow">BBa_K337038</A></TD><TD>CMV TetO2</TD><TD>CMV </TD><TD>RSV
</TD></TR><TR><TD><A href="http://partsregistry.org/wiki/index.php?title=Part:BBa_K337040" class="external text" rel="nofollow">BBa_K337040</A></TD><TD>RSV</TD><TD>RSV</TD><TD>SV40
</TD></TR><TR><TD><A href="http://partsregistry.org/wiki/index.php?title=Part:BBa_K337042" class="external text" rel="nofollow">BBa_K337042</A></TD><TD>CMV</TD><TD>RSV</TD><TD>SV40
</TD></TR><TR><TD><A href="http://partsregistry.org/wiki/index.php?title=Part:BBa_K337044" class="external text" rel="nofollow">BBa_K337044</A></TD><TD>CMV</TD><TD>RSV</TD><TD>RSV
</TD></TR><TR><TD><A href="http://partsregistry.org/wiki/index.php?title=Part:BBa_K337046" class="external text" rel="nofollow">BBa_K337046</A></TD><TD>CMV TetO2</TD><TD>RSV</TD><TD>RSV
</TD></TR></TBODY></TABLE><DIV class="center"><DIV class="thumb tnone"><DIV class="thumbinner" style="width:702px;"><DIV class="thumbcaption">Promoter strength characterization of tuning constructs in HEK 293 T-REx cell line</DIV></DIV></DIV></DIV><DIV class="backtop"><A href="#top">↑</A></DIV><H3><SPAN class="mw-headline" id="Characterization_of_BBa_K337035_and_BBa_K337036_in_different_cell_lines">Characterization of <A href="http://partsregistry.org/wiki/index.php?title=Part:BBa_K337035" class="external text" rel="nofollow">BBa_K337035</A> and <A href="http://partsregistry.org/wiki/index.php?title=Part:BBa_K337036" class="external text" rel="nofollow">BBa_K337036</A> in different cell lines</SPAN></H3><DIV class="thumb tright"><DIV class="thumbinner" style="width:502px;"><DIV class="thumbcaption">Promoter strength characterization in different cell lines</DIV></DIV></DIV><P>If <A href="http://partsregistry.org/wiki/index.php?title=Part:BBa_K337035" class="external text" rel="nofollow">BBa_K337035</A> and <A href="http://partsregistry.org/wiki/index.php?title=Part:BBa_K337036" class="external text" rel="nofollow">BBa_K337036</A> are transfected into different cell lines it is obvious that Hek293T cells are the easiest to transfect with both constructs an expression of 17-22 RLU is to be measured. Hek T-Rex cells are showing and expression level of 12 RLU of both constructs. Hela cells are also showing constant expression levels of  8 RLU with both constructs. A rather low expression  of 2RLU is to be seen by transfecting the 2 constructs in Huh7 cells. This might be due to low transfection efficiency of this cell line in general. All together it is to say that <A href="http://partsregistry.org/wiki/index.php?title=Part:BBa_K337035" class="external text" rel="nofollow">BBa_K337035</A> and <A href="http://partsregistry.org/wiki/index.php?title=Part:BBa_K337036" class="external text" rel="nofollow">BBa_K337036</A> show comparable expression.
</P><H3><SPAN class="mw-headline" id="Characterization_of_part_binding_sites_.28BBa_K337052.2C_BBa_K337053.2C_BBa_K337054.29"> Characterization of part binding sites (<A href="http://partsregistry.org/wiki/index.php?title=Part:BBa_K337052" class="external text" rel="nofollow">BBa_K337052</A>, <A href="http://partsregistry.org/wiki/index.php?title=Part:BBa_K337053" class="external text" rel="nofollow">BBa_K337053</A>, <A href="http://partsregistry.org/wiki/index.php?title=Part:BBa_K337054" class="external text" rel="nofollow">BBa_K337054</A>) </SPAN></H3><P>Regulation of gene expression can be achieved by fusing miRNA binding sites into the 3'UTR of a GOI. In case a referring shRNA miR is endogenously present or co-expressed, the GOI is knocked down. Strength of regulation thereby depends on binding site properties. We are able to tune gene expression linearly over a broad range. This is a first proof of principle for various miRNA-mRNA interaction <I>in vitro</I>. Therefore, we transfected <A href="https://2010.igem.org/Team:Heidelberg/Notebook/Material#Cell_lines" class="external text" rel="nofollow">HeLa cells</A> in principle with our <A href="http://partsregistry.org/Part:BBa_K337036" class="external text" rel="nofollow">pSMB_miTuner Plasmid HD3</A> . It turned out, that no obvious effect of different binding sites on reporter gene expression could be measured (data not shown). We assume that the RSV promoter driving the shRNA miR is too weak for tight regulation of the referring binding site behind the GOI. Only if a sufficient amount of shRNA miR binds to its target, translation is significantly repressed. Thus, we expressed the shRNA miR from a separate plasmid which was always co-transfected with the original tuning construct. The reporter genes - i. e. Luc2 and hRluc - were also expressed from separate plasmids to get a reference as well as a transfection control. This was achieved by co-transfection the tuning construct with corresponding shRNA miRhaat and as a control a miRNA w/o binding sites in the target 3'UTR. The experiment was done in a 96-well plate by plating 5000 Hela cells/well 24h before transfection. Transfection was done using Fugene transfection reagent. 2.5ng of tuning construct were co-transfected with the shRNA miR construct of a concentration of 25 ng (1:10). Then, we conducted a <A href="https://2010.igem.org/Team:Heidelberg/Notebook/Methods#Dual_Luciferase_Assay" class="external text" rel="nofollow">Dual Luciferase Assay</A> for quantification of gene expression. The data preciously shows a tuned expression from almost 0% to 100% (Fig. 1, Fig. 2). Lowest expression refers to complete knockdown through cloning of perfect binding sites into the 3'UTR  to the reporter gene(always green bar on the left hand side of the figures). 100% refers to ordinary expression from a construct without binding sites (always orange column on the right hand side of the figures). In presence of the specific shRNA miR, gene expression was mediated to various levels through interactions with the different imperfect binding sites. Whereas, when an unspecific shRNA miR was expressed, gene expression remained unaffected (see raw data below). The latter aspect reveals, that the binding sites were correctly designed, since they seem to interact specifically with a referring shRNA miR. The constructs were tested in two different backbones: pBS_U6 and pBS_H1. Both are in viral context, meaning that they contain inverted terminal repeats (ITRs). The constructs can be packed into the capsid of an adeno-associated virus (AAV). We chose the data obtained by the construct with the U6 promoter as this promoter is more efficient than the H1 promoter, ensuring that the system is saturated and ensuring that the data is reproducible. The same constructs were also used for <A href="https://2010.igem.org/Team:Heidelberg/Notebook/Methods#Virus_Production" class="external text" rel="nofollow">virus production</A> to infect cells even more efficiently as compared to transfections. Because of the significant data, we decided to inject the viruses into mice to see the tuning effect also <I><A href="https://2010.igem.org/Team:Heidelberg/Project/Mouse_Infection" class="external text" rel="nofollow">in vivo</A></I>. The pBS_H1 construct should be used for mice injections since the expressed shRNA miR against human alpha-1-antitrypsine (hAAT) is cytotoxic in higher concentrations. The pBS_H1 backbone leads to moderate expression ranges, still obviously showing the tuning effect.
</P><DIV class="center"><DIV class="thumb tnone"><DIV class="thumbinner" style="width:602px;"><DIV class="thumbcaption"><B>Figure 1: Tuning of gene expression through different imperfect shRNA miR binding sites in pBS_SV40_Luc2 construct cotransfected with a reference renilla construct.</B> Gene expression quantified via dual luciferase assay for constructs containing different imperfect binding sites for shhAAT. The shhAAT was expressed from a pSB_U6 plasmid</DIV></DIV></DIV></DIV><DIV class="center"><DIV class="thumb tnone"><DIV class="thumbinner" style="width:602px;"><DIV class="thumbcaption"><B>Figure 2: Tuning of gene expression through different imperfect shRNA miR binding sites in pBS_SV40 Luc2 double transfected with a reference renilla construct.</B> The shRNA_hAAT construct was expressed from a pSB_H1 construct.</DIV></DIV></DIV></DIV><P>Strikingly, the order of constructs in terms of knockdown for the imperfect binding sites is similar. The perfect binding site <A href="http://partsregistry.org/wiki/index.php?title=Part:BBa_K337052" class="external text" rel="nofollow">BBa_K337052</A> shows knockdown of about 96%. point mut 10 (1), point mut 10 (2) and point mut 11 <A href="http://partsregistry.org/wiki/index.php?title=Part:BBa_K337053" class="external text" rel="nofollow">BBa_K337053</A> (2) always show strong knockdown, whereas bulge 16-18 <A href="http://partsregistry.org/wiki/index.php?title=Part:BBa_K337054" class="external text" rel="nofollow">BBa_K337054</A> , only seed and bulge 9-12(2)	show only loose down-regulation. Consulting the binding site sequences, the weak knockdown can be addressed to bulges in the supplementary region or complete lack of the 3' region of the binding site. Still high strength could be maintained due to only single nucleotide exchanges in the central region of the binding site.
</P><H3><SPAN class="mw-headline" id="Characterization_of_synthetic_single_microRNA_binding_sites_.28BBa_K337055.29"> Characterization of synthetic single microRNA binding sites (<A href="http://partsregistry.org/wiki/index.php?title=Part:BBa_K337055" class="external text" rel="nofollow">BBa_K337055</A>) </SPAN></H3><P>Binding sites for miR122 were experimentally characterized by cloning them into psiCHECK-2 backbone (Promega). <A href="https://2010.igem.org/Team:Heidelberg/Notebook/Methods#Dual_Luciferase_Assay" class="external text" rel="nofollow">Dual luciferase assay</A> was conducted using this time Renilla Luciferase as a reporter and the other luciferase as a reference for normalization. Figure 3 shows again a broad range of regulation depending on binding site sequence properties.  <A href="http://partsregistry.org/wiki/index.php?title=Part:BBa_K337055" class="external text" rel="nofollow">BBa_K337055</A> (mut 99) is the construct with the perfect binding site and leads to an knock-down percentage of 96%. <A href="http://partsregistry.org/wiki/index.php?title=Part:BBa_K337056" class="external text" rel="nofollow">BBa_K337056</A> (mut 7) is an imperfect binding site which leads to a knockdown percentage of 64%. <A href="http://partsregistry.org/wiki/index.php?title=Part:BBa_K337057" class="external text" rel="nofollow">BBa_K337057</A>(mut 111) is an imperfect binding site with a knockdown efficiency of 24%. All this contributes to a real tuning effect by introducing binding sites with introduced mismatches following the rational design protocol.
</P><DIV class="center"><DIV class="thumb tnone"><DIV class="thumbinner" style="width:602px;"><DIV class="thumbcaption"><B>Figure 3: Tuning of gene expression through different imperfect miR122 binding sites in psiCHECK-2.</B> Construct was transfected into HeLa cells together with an plasmid expressing miR122. Control without binding site was used for normalization.</DIV></DIV></DIV></DIV><H3><SPAN class="mw-headline" id="Characterization_of_synthetic_microRNA_binding_site_patterns_against_endogenous_miR122_3.1BBa_K337008_and_1.3_BBa_K337000"> Characterization of synthetic microRNA binding site patterns against endogenous miR122 3.1<A href="http://partsregistry.org/wiki/index.php?title=Part:BBa_K337008" class="external text" rel="nofollow">BBa_K337008</A> and 1.3 <A href="http://partsregistry.org/wiki/index.php?title=Part:BBa_K337000" class="external text" rel="nofollow">BBa_K337000</A></SPAN></H3><P>We used the miMeasure plasmid with the BB-2 standard to characterise binding sites for the endogenously expressed miR122. We cloned two different synthetic microRNA binding site patterns into our miMeasure construct plasmid: 3.1 BBa_K337008 and 1.3 BBa_K337000. As the binding site is inserted downstream of green fluorescent protein (EGFP), a regulation of EGFP expression is to be expected. miMeasure normalizes knockdown of EGFP to the unregulated blue fluorescent protein (EBFP2). By calculating the ratio of EGFP to EBFP2 we determined the knockdown percentage characteristic of the binding site patterns. </P><P>We tested these constructs in four different set-ups with three different cell lines:  
•	HeLa cells, which do not express miR122 endogenously  
•	HeLa cells cotransfected with miR122 to mimic endogenous expression 
•	HuH7 cells, which are liver cells known to express miR122  
•	HepG2 cells, liver cells known to express low amounts of miR122 (Douglas, 2010) </P><P>EGFP to EBFP2 ratios were measured with flow cytometry and microscopy. Measurement results for the four cell lines and the binding sites 3.1 <A href="http://partsregistry.org/wiki/index.php?title=Part:BBa_K337008" class="external text" rel="nofollow">BBa_K337008</A> and 1.3 <A href="http://partsregistry.org/wiki/index.php?title=Part:BBa_K337000" class="external text" rel="nofollow">BBa_K337000</A> are shown in Fig. 4 for all four cell line setups. As expected, no down regulation of EGFP expression was measured in HeLa cells due to the lack of miR122 therein. This serves as a control for the design of our binding sites as it is clear that they do not cross-react with other endogenously expressed miRNAs but are specific for miR122. Similarly, no down regulation was observed in HepG2 cells. The levels of miR122 expression in those cells are reportedly reduced by 99.5&amp;, therefore also serving as a negative control. Both the construct 1.3 (containing two perfect binding sites with the extra 10bp spacer in between) and 3.1 (containing 3 perfect binding sites) lead to an increase in downregulation effect of EGFP in Huh7 cells and in HeLa cells cotransfected with miR122 in comparison to the single perfect binding site. 
</P><DIV class="center"><DIV class="thumb tnone"><DIV class="thumbinner" style="width:602px;"><DIV class="thumbcaption"><B>Figure 4: miMeasure in four different set-ups with three different cell lines</B> The construct containing synthetic microRNA binding site patterns against endogenous miR122 transfected into different cell lines. The EGFP/EBFP2 ratio for the construct containing no binding site was set to one.</DIV></DIV></DIV></DIV><H3><SPAN class="mw-headline" id="Characterization_of_the_bidirectional_CMV_promoter"> Characterization of the bidirectional CMV promoter </SPAN></H3><P>The bidirectional CMV promoter <A href="http://partsregistry.org/wiki/index.php?title=Part:BBa_K337017" class="external text" rel="nofollow">BBa_K337017</A> is a new standard part out of our <A href="https://2010.igem.org/Team:Heidelberg/Project/miMeasure" class="external text" rel="nofollow">miMeasure construct</A>. 
The promoter was used to express two reporter genes with a comparable transcription rate. Referring to the figures below, this is the case.
</P><DIV class="center"><DIV class="thumb tnone"><DIV class="thumbinner" style="width:602px;"><DIV class="thumbcaption">Left panel: EGFP channel. Middle panel: EBFP channel. Right panel: merge of both the others.</DIV></DIV></DIV></DIV><DIV class="center"><DIV class="thumb tnone"><DIV class="thumbinner" style="width:402px;"><DIV class="thumbcaption">EBFP intensities against EGFP intensities. Correlation coefficient is given.</DIV></DIV></DIV></DIV><H2><SPAN class="mw-headline" id="References">References</SPAN></H2><UL><LI>Douglas D:Small Molecule Modiﬁers of MicroRNA miR-122 Function for the Treatment of Hepatitis C Virus Infection and Hepatocellular Carcinoma. JACS. 2010 May 15; 132(23):7976-81.</LI></UL><DIV class="backtop"><A href="#top">↑</A></DIV></DIV><DIV class="printfooter">
Retrieved from &quot;<A href="http://2010.igem.org/Team:Heidelberg/Parts/Characterization">http://2010.igem.org/Team:Heidelberg/Parts/Characterization</A>&quot;</DIV></DIV><DIV id="footer-box" class="noprint"><DIV id="footer"><UL id="f-list"><LI id="t-recentchanges"><A href="/Special:RecentChanges" title="Recent changes">Recent changes</A></LI><LI id="t-whatlinkshere"><A href="/Special:WhatLinksHere/Team:Heidelberg/Parts/Characterization" title="List of all wiki pages that link here [j]" accesskey="j">What links here</A></LI><LI id="t-recentchangeslinked"><A href="/Special:RecentChangesLinked/Team:Heidelberg/Parts/Characterization" title="Recent changes in pages linked from this page [k]" accesskey="k">Related changes</A></LI><LI id="t-specialpages"><A href="/Special:SpecialPages" title="List of all special pages [q]" accesskey="q">Special pages</A></LI><LI><A href="/Special:Preferences">My preferences</A></LI></UL></DIV><DIV id="footer"><UL id="f-list"><LI id="t-print"><A href="/wiki/index.php?title=Team:Heidelberg/Parts/Characterization&amp;printable=yes" title="Printable version of this page [p]" accesskey="p">Printable version</A></LI><LI id="t-permalink"><A href="/wiki/index.php?title=Team:Heidelberg/Parts/Characterization&amp;oldid=206934" title="Permanent link to this revision of the page">Permanent link</A></LI><LI id="privacy"><A href="/2010.igem.org:Privacy_policy" title="2010.igem.org:Privacy policy">Privacy policy</A></LI><LI id="disclaimer"><A href="/2010.igem.org:General_disclaimer" title="2010.igem.org:General disclaimer">Disclaimers</A></LI></UL></DIV></DIV></BODY></HTML>