<HTML xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en" dir="ltr">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki  ltr ns-0 ns-subject page-Team_Heidelberg_Notebook_Material_Methods"><DIV id="globalWrapper"><DIV id="top-section"><DIV id="p-logo"><A href="/Main_Page" title="Main Page">&quot;
	    </A></DIV><DIV id="menubar" class="left-menu noprint"><UL><LI class="selected"><A href="/Team:Heidelberg/Notebook/Material_Methods">Page               </A></LI><LI class="new"><A href="/wiki/index.php?title=Talk:Team:Heidelberg/Notebook/Material_Methods&amp;action=edit&amp;redlink=1">Discussion               </A></LI><LI><A href="/wiki/index.php?title=Team:Heidelberg/Notebook/Material_Methods&amp;action=edit">View source               </A></LI><LI><A href="/wiki/index.php?title=Team:Heidelberg/Notebook/Material_Methods&amp;action=history">History               </A></LI><LI style="color:#808080;cursor:default">teams</LI></UL></DIV><DIV class="right-menu noprint" id="menubar"><UL><LI id="pt-login"><A href="/wiki/index.php?title=Special:UserLogin&amp;returnto=Team:Heidelberg/Notebook/Material_Methods" title="You are encouraged to log in; however, it is not mandatory [o]" accesskey="o">Log in</A></LI></UL></DIV><DIV id="search-controls" class="noprint"><FORM action="/Special:Search" id="searchform"> </FORM></DIV></DIV><DIV id="content"><H1 class="firstHeading">Team:Heidelberg/Notebook/Material Methods</H1><DIV id="bodyContent"><H3 id="siteSub" class="noprint">From 2010.igem.org</H3><DIV id="header"><DIV id="menu"><UL id="nav"><LI id="menu_home"><A class="aHome" href="https://2010.igem.org/Team:Heidelberg">Home</A></LI><LI id="menu_team"><A class="aTeam" href="https://2010.igem.org/Team:Heidelberg/Team">Team</A><TABLE class="menutable" cellpadding="0" cellspacing="0"><TBODY><TR><TD class="lefttdtopteam"><A class="aTeamOverview" href="https://2010.igem.org/Team:Heidelberg/Team">Overview</A><A class="aTeamMembers" href="https://2010.igem.org/Team:Heidelberg/Team/Members">Students</A><A class="aTeamMembers" href="https://2010.igem.org/Team:Heidelberg/Team/Members#Instructors">Instructors</A><A class="aTeamMembers" href="https://2010.igem.org/Team:Heidelberg/Team/Members#Advisors">Advisors</A><A class="aInstitutes" href="https://2010.igem.org/Team:Heidelberg/Team/Institutes">Institutes</A><A class="aGallery" href="https://2010.igem.org/Team:Heidelberg/Team/Gallery">Gallery</A><A class="aDance" href="https://2010.igem.org/Team:Heidelberg/midance">miDANCE</A><A class="amithanks" href="https://2010.igem.org/Team:Heidelberg/Team/miThanks">Acknowledgements</A></TD><TD class="righttdtop"> </TD></TR></TBODY></TABLE></LI><LI id="menu_project"><A class="aProject" href="https://2010.igem.org/Team:Heidelberg/Project">Project</A><TABLE class="menutable" cellpadding="0" cellspacing="0"><TBODY><TR><TD class="lefttdtopproject"><A class="aAbstract" href="https://2010.igem.org/Team:Heidelberg/Project">Abstract</A><A class="aIntroduction" href="https://2010.igem.org/Team:Heidelberg/Project/Introduction">Introduction</A><A class="aProject_miRNA_Kit" href="https://2010.igem.org/Team:Heidelberg/Project/miRNA_Kit">Synthetic miRNA Kit</A><A class="aPro_Capsid_Shuffling" href="https://2010.igem.org/Team:Heidelberg/Project/Capsid_Shuffling">Capsid Shuffling</A><A class="aPro_Homology" href="https://2010.igem.org/Team:Heidelberg/Project/Capsid_Shuffling/Homology_Based">    Homology Based</A><A class="aPro_Virobytes" href="https://2010.igem.org/Team:Heidelberg/Project/Capsid_Shuffling/ViroBytes">    ViroBytes</A><A class="aPro_miMeasure" href="https://2010.igem.org/Team:Heidelberg/Project/miMeasure">miMeasure</A><A class="aPro_Mouse" href="https://2010.igem.org/Team:Heidelberg/Project/Mouse_Infection"><I>in vivo</I> Study</A><A class="aPro_Summary" href="https://2010.igem.org/Team:Heidelberg/Project/Summary">Summary &amp; Outlook</A><A class="aPro_Ref" href="https://2010.igem.org/Team:Heidelberg/Project/References">References</A><A class="aPro_Ref" href="https://2010.igem.org/Team:Heidelberg/Project/Contributions">Contributions</A></TD><TD class="righttdtop"> </TD></TR></TBODY></TABLE></LI><LI id="menu_parts"><A class="aParts" href="https://2010.igem.org/Team:Heidelberg/Parts">Parts</A><TABLE class="menutable" cellpadding="0" cellspacing="0"><TBODY><TR><TD class="lefttdtop"><A class="aparts_ger" href="https://2010.igem.org/Team:Heidelberg/Parts">Parts</A><A class="aparts_char" href="https://2010.igem.org/Team:Heidelberg/Parts/Characterization">Characterization</A></TD><TD class="righttdtop"> </TD></TR></TBODY></TABLE></LI><LI id="menu_modeling"><A class="aModeling" href="https://2010.igem.org/Team:Heidelberg/Modeling">Modeling</A><TABLE class="menutable" cellpadding="0" cellspacing="0"><TBODY><TR><TD class="lefttdtop"><A class="amode_over" href="https://2010.igem.org/Team:Heidelberg/Modeling">Introduction</A><A class="amiGUI" href="https://2010.igem.org/Team:Heidelberg/Modeling/miGUI">miBEAT</A><A class="amirockdown" href="https://2010.igem.org/Team:Heidelberg/Modeling/miRockdown">miRockdown</A><A class="amibsdesigner" href="https://2010.igem.org/Team:Heidelberg/Modeling/miBSdesigner">miBS designer</A><A class="amispec" href="https://2010.igem.org/Team:Heidelberg/Modeling/miSpec">mUTING</A><A class="amodelset" href="https://2010.igem.org/Team:Heidelberg/Modeling/trainingset">Descriptions</A></TD><TD class="righttdtop"> </TD></TR></TBODY></TABLE></LI><LI id="menu_notebook"><A class="aNotebook" href="https://2010.igem.org/Team:Heidelberg/Notebook">Notebook</A><TABLE class="menutable" cellpadding="0" cellspacing="0"><TBODY><TR><TD class="lefttdtop"><A class="aNote_miRNA_kit" href="https://2010.igem.org/Team:Heidelberg/Notebook/miRNA_Kit">Synthetic miRNA Kit</A><A class="aNote_Capsid_Shuffling">Capsid Shuffling</A><A class="aNote_Homology" href="https://2010.igem.org/Team:Heidelberg/Notebook/Capsid_Shuffling/Homology_Based">    Homology Based</A><A class="aNote_Virobytes" href="https://2010.igem.org/Team:Heidelberg/Notebook/Capsid_Shuffling/ViroBytes">    ViroBytes</A><A class="aNote_miMeasure" href="https://2010.igem.org/Team:Heidelberg/Notebook/miMeasure">miMeasure</A><A class="aNote_BSdesign" href="https://2010.igem.org/Team:Heidelberg/Notebook/BSDesign">Binding Site Design </A><A class="aNote_Mouse" href="https://2010.igem.org/Team:Heidelberg/Notebook/Mouse_Infection"><I>in vivo</I> Study</A><A class="aNote_Modeling" href="https://2010.igem.org/Team:Heidelberg/Notebook/Modeling">Modeling</A><A class="aNote_Material" href="https://2010.igem.org/Team:Heidelberg/Notebook/Material">Materials</A><A class="aNote_Methods" href="https://2010.igem.org/Team:Heidelberg/Notebook/Methods">Methods</A><A class="aNote_Psycho" href="https://2010.igem.org/Team:Heidelberg/Human_Practices/Psychology/Notebook">Psychological Survey</A><A class="aNote_Philo" href="https://2010.igem.org/Team:Heidelberg/Human_Practices/Philosophy/Part2">Philosophical Reflection</A><A class="aNote_Biosafety" href="https://2010.igem.org/Team:Heidelberg/Notebook/Biosafety">Biosafety</A></TD><TD class="righttdtop"> </TD></TR></TBODY></TABLE></LI><LI id="menu_human_practices"><A class="aHuman_Practices" href="https://2010.igem.org/Team:Heidelberg/Human Practices">Human Practices</A><TABLE class="menutable" cellpadding="0" cellspacing="0"><TBODY><TR><TD class="lefttdtop"><A class="aPsycho" href="https://2010.igem.org/Team:Heidelberg/Human Practices/Psychology">Psychological Survey</A><A class="aPhilo" href="https://2010.igem.org/Team:Heidelberg/Human Practices/Philosophy">Philosophical Reflection</A></TD><TD class="righttdtop"> </TD></TR></TBODY></TABLE></LI><LI id="menu_support"><A class="aSupport" href="https://2010.igem.org/Team:Heidelberg/Support">Sponsors</A></LI></UL></DIV></DIV><H1><SPAN class="mw-headline" id="Materials">Materials</SPAN></H1><H2><SPAN class="mw-headline" id="Kits"> Kits </SPAN></H2><CENTER><TABLE class="wikitable sortable" border="0" style="text-align: left"><CAPTION align="top, left"><B>table 1</B>: Used kits and referring suppliers.
</CAPTION><TBODY><TR bgcolor="#cccccc"><TD> Kits</TD><TD>Supplier</TD><TD>Catalog Number
</TD></TR><TR><TD>HiSpeed® Plasmid Maxi Kit (25)</TD><TD>QIAGEN</TD><TD>12663
</TD></TR><TR><TD>CompactPrep Plasmid Maxi Kit (25)</TD><TD>QIAGEN</TD><TD>12763
</TD></TR><TR><TD>RNeasy® Mini Kit (25)</TD><TD>QIAGEN</TD><TD>74106
</TD></TR><TR><TD>QIAquick® Miniprep Kit (250)</TD><TD>QIAGEN</TD><TD>27106
</TD></TR><TR><TD>QIAquick® Gel Extraction Kit (250)</TD><TD>QIAGEN</TD><TD>28706
</TD></TR><TR><TD>QIAquick® Nucleotid Removal Kit (250)</TD><TD>QIAGEN</TD><TD>28306
</TD></TR><TR><TD>QIAquick® PCR Purification Kit (250)</TD><TD>QIAGEN</TD><TD>28106
</TD></TR><TR><TD>GeneMorph® II EZC Clone Domain Mutagenesis Kit</TD><TD>Stratagene</TD><TD>200552-5
</TD></TR></TBODY></TABLE></CENTER><H2><SPAN class="mw-headline" id="Marker"> Marker </SPAN></H2><CENTER><TABLE class="wikitable sortable" border="0" style="text-align: left"><CAPTION align="top, left"><B>table 2</B>: Used DNA Ladder for Gel Electrophoresis.
</CAPTION><TBODY><TR bgcolor="#cccccc"><TD>Name</TD><TD>Supplier</TD><TD>Range
</TD></TR><TR><TD>GeneRuler™ High Range DNA Ladder</TD><TD>Fermentas</TD><TD>10,171-48,502 bp
</TD></TR><TR><TD>GeneRuler™ 1 kb Plus DNA Ladder</TD><TD>Fermentas</TD><TD>75-20,000 bp
</TD></TR></TBODY></TABLE></CENTER><H2><SPAN class="mw-headline" id="Enzymes"> Enzymes </SPAN></H2><CENTER><TABLE class="wikitable sortable" border="0" style="text-align: left"><CAPTION align="top, left"><B>table 3</B>: Used NEB restriction enzymes
</CAPTION><TBODY><TR bgcolor="#cccccc"><TD> Name</TD><TD>Number/ID</TD><TD>NEB buffer1</TD><TD>NEB buffer2</TD><TD>NEB buffer3</TD><TD>NEB buffer4</TD><TD>BSA
</TD></TR><TR><TD>AflII</TD><TD>27</TD><TD>50%</TD><TD>100%</TD><TD>25%</TD><TD>100%</TD><TD>x
</TD></TR><TR><TD>AgeI</TD><TD>1</TD><TD>100%</TD><TD>50%</TD><TD>10%</TD><TD>75%
</TD></TR><TR><TD>ApaI</TD><TD>-</TD><TD>25%</TD><TD>50%</TD><TD>0%</TD><TD>100%</TD><TD>x
</TD></TR><TR><TD>AseI</TD><TD>not</TD><TD>75%</TD><TD>100%</TD><TD>not</TD></TR><TR><TD>AvrII</TD><TD>100%</TD><TD>100%</TD><TD>50%</TD><TD>100%</TD></TR><TR><TD>BamHI</TD><TD>2</TD><TD>75%</TD><TD>100%</TD><TD>100%</TD><TD>100%</TD><TD>x
</TD></TR><TR><TD>BamHI</TD><TD>-</TD><TD>75%</TD><TD>100%</TD><TD>100%</TD><TD>100%</TD><TD>x
</TD></TR><TR><TD>BclI</TD><TD>-</TD><TD>50%</TD><TD>100%</TD><TD>100%</TD><TD>75%</TD></TR><TR><TD>BglI</TD><TD>-</TD><TD>50%</TD><TD>75%</TD><TD>100%</TD><TD>50%</TD></TR><TR><TD>BglII</TD><TD>13</TD><TD>	10%</TD><TD>	75%</TD><TD>	100%</TD><TD>	10%</TD></TR><TR><TD>BsrGI</TD><TD>-</TD><TD>	25%</TD><TD>	100%</TD><TD>	10%</TD><TD>	100%</TD><TD>x
</TD></TR><TR><TD>DpnI	</TD><TD>6</TD><TD>	100%</TD><TD>	100%</TD><TD>	75%</TD><TD>	100%</TD></TR><TR><TD>DpnI	</TD><TD>-</TD><TD>	100%</TD><TD>	100%</TD><TD>	75%</TD><TD>	100%</TD></TR><TR><TD>EcoRI	</TD><TD>-</TD><TD>	100%</TD><TD>	100%</TD><TD>	100%</TD><TD>	100%</TD></TR><TR><TD>EcoRV	</TD><TD>-</TD><TD>	50%</TD><TD>	75%</TD><TD>	100%</TD><TD>	50%</TD><TD>	x
</TD></TR><TR><TD>FseI	</TD><TD>-</TD><TD>	100%</TD><TD>	75%</TD><TD>	0%</TD><TD>	100%</TD></TR><TR><TD>HindIII</TD><TD>-</TD><TD>	10	50%</TD><TD>	100%</TD><TD>	10%</TD><TD>	50%</TD></TR><TR><TD>HindIII</TD><TD>-</TD><TD>50%</TD><TD>	100%</TD><TD>	10%</TD><TD>	50%</TD></TR><TR><TD>KpnI	</TD><TD>11</TD><TD>	100%</TD><TD>	75%</TD><TD>	0%</TD><TD>	50%</TD><TD>	x
</TD></TR><TR><TD>MfeI-HF™</TD><TD>-</TD><TD>	75%</TD><TD>	50%</TD><TD>	10%</TD><TD>	100%</TD></TR><TR><TD>NarI	</TD><TD>26</TD><TD>	100%</TD><TD>	75%</TD><TD>	75%</TD><TD>	100%</TD></TR><TR><TD>NcoI	</TD><TD>12</TD><TD>	100%</TD><TD>	100%</TD><TD>	100%</TD><TD>	100%</TD></TR><TR><TD>NdeI	</TD><TD>-</TD><TD>	75%</TD><TD>	100%</TD><TD>	75%</TD><TD>	100%</TD></TR><TR><TD>NheI	</TD><TD>-</TD><TD>	100%</TD><TD>	100%</TD><TD>	10%</TD><TD>	100%</TD><TD>	x
</TD></TR><TR><TD>NotI	</TD><TD>14</TD><TD>	0%</TD><TD>	50%</TD><TD>	100%</TD><TD>	25%</TD></TR><TR><TD>NsiI	</TD><TD>-</TD><TD>	10%</TD><TD>	75%</TD><TD>	100%</TD><TD>	25%</TD></TR><TR><TD>PciI	</TD><TD>-</TD><TD>	50%</TD><TD>	75%</TD><TD>	100%</TD><TD>	50%</TD><TD>	x
</TD></TR><TR><TD>PstI	</TD><TD>15</TD><TD>	75%</TD><TD>	75%</TD><TD>	100%</TD><TD>	50%</TD></TR><TR><TD>SacI	</TD><TD>16</TD><TD>	100%</TD><TD>	50%</TD><TD>	10%</TD><TD>	100%</TD></TR><TR><TD>ScaI	</TD><TD>18</TD><TD>	not</TD><TD>not</TD><TD>100%</TD><TD>not</TD></TR><TR><TD>SfcI	</TD><TD>19</TD><TD>	75%</TD><TD>	50%</TD><TD>	10%</TD><TD>	100%</TD><TD>	x
</TD></TR><TR><TD>SfcI</TD><TD>-</TD><TD>75%</TD><TD>	50%</TD><TD>	10%</TD><TD>	100%</TD><TD>	x
</TD></TR><TR><TD>SpeI	</TD><TD>-</TD><TD>	75%</TD><TD>	100%</TD><TD>	25%</TD><TD>	100%</TD><TD>	x
</TD></TR><TR><TD>SphI	</TD><TD>-</TD><TD>	100%</TD><TD>	100%</TD><TD>	50%</TD><TD>	100%</TD></TR><TR><TD>SspI	</TD><TD>-</TD><TD>	50%</TD><TD>	100%</TD><TD>	50%</TD><TD>	50%</TD></TR><TR><TD>XbaI	</TD><TD>23	</TD><TD>0%</TD><TD>	100%</TD><TD>	75%</TD><TD>	100%</TD><TD>	x
</TD></TR><TR><TD>XbaI	</TD><TD>56</TD><TD>	0%</TD><TD>	100%</TD><TD>	75%</TD><TD>	100%</TD><TD>	x
</TD></TR><TR><TD>XhoI</TD><TD>24</TD><TD>	75%</TD><TD>	100%</TD><TD>	100%</TD><TD>	100%</TD><TD>	x
</TD></TR><TR><TD>XmaI</TD><TD>25</TD><TD>	25%</TD><TD>	50%</TD><TD>	0%</TD><TD>	100%</TD><TD>	x
</TD></TR></TBODY></TABLE></CENTER><P>NEBuffers are color-coded (NEBuffer 1-yellow, NEBuffer 2-blue, 						
NEBuffer 3-red, NEBuffer 4-green) and supplied as 10X
</P><H2><SPAN class="mw-headline" id="Bacteria"> Bacteria </SPAN></H2><CENTER><TABLE class="wikitable sortable" border="0" style="text-align: left"><CAPTION align="top, left"><B>table 4</B>: Used <I>E. coli</I> Strains.
</CAPTION><TBODY><TR bgcolor="#cccccc"><TD> Name</TD><TD>Purpose</TD><TD>Reference
</TD></TR><TR><TD>DH5α</TD><TD>Amplification of Plasmids</TD><TD>Invitrogen
</TD></TR><TR><TD>SCS110</TD><TD>Amplification of unmethylated Plasmids</TD><TD>Stratagene
</TD></TR><TR><TD>Top10</TD><TD>Amplification of Plasmids</TD><TD>Invitrogen
</TD></TR></TBODY></TABLE></CENTER><H3><SPAN class="mw-headline" id="Media_and_Antibiotics"> Media and Antibiotics </SPAN></H3><UL><LI><B>LB</B> (10 g tryptone, 5 g yeast extract, 10 g sodium chloride, ad 1 L ddH<SUB>2</SUB>O)
</LI></UL><CENTER><TABLE class="wikitable sortable" border="0" style="text-align: left"><CAPTION align="top, left"><B>table 5</B>: Used Antibiotics.
</CAPTION><TBODY><TR bgcolor="#cccccc"><TH> Antibiotic</TH><TH>Final Concentration</TH><TH> Solvent
</TH></TR><TR><TD>Ampicillin</TD><TD>100 µg/ml</TD><TD>ddH<SUB>2</SUB>O
</TD></TR><TR><TD>Chloramphenicol</TD><TD>25 µg/ml</TD><TD>100% Ethanol
</TD></TR><TR><TD>Kanamycin</TD><TD>50 µg/ml</TD><TD>ddH<SUB>2</SUB>O
</TD></TR></TBODY></TABLE></CENTER><H1><SPAN class="mw-headline" id="Methods">Methods</SPAN></H1><H2><SPAN class="mw-headline" id="Cloning">Cloning</SPAN></H2><H3><SPAN class="mw-headline" id="Agarose_Gel_Electrophoresis">Agarose Gel Electrophoresis</SPAN></H3><P>Agarose flat-bed gels in various concentrations (0.6–2% agarose in 0.5 x TAE buffer) and sizes were run to separate DNA fragments in an electrical field (10–20 V/cm) for analytical or preparative use. The desired amount of agarose was boiled in 1 x TAE buffer until it was completely dissolved. After it cooled down to approximately 60°C, ethidium bromide (EtBr) solution (0.5 μg/ml final concentration) was added to the liquid agar, which was then poured in a flat-bed tray with combs. As soon as the agarose solidified, the Running buffer (0.5 x TAE buffer) was added before the DNA in the loading buffer was loaded into the wells and separated electrophoretically. Ethidium bromide intercalates with the DNA’s GC ntss resulting in DNA-EtBr-complex that emits visible light. Therefore, the DNA fragments could be detected on a UV-light tray at 265 nm.
</P><H3><SPAN class="mw-headline" id="Colony_PCR">Colony PCR</SPAN></H3><P>Colony PCRs were performed using Fermentas PCR Master Mix (2x), containing <I>Taq</I> DNA Polymerase.
Bacterial colonies were picked form an LB/Agar-plate and either resuspended in PCR reaction mix and subsequently used to inoculate LB-medium with appropriate antibiotic, or resuspended in water. The water was then taken as &quot;template&quot; for PCR and inoculation of Minepreps.
The PCR was performed using 20 pmol of forward and reverse primer in a total volume of 20 µL.
PCR conditions were set to recommendations made by Fermentas:
</P><TABLE border="1" cellpadding="3" style="border:solid 1px #AAAAAA; border-collapse:collapse; background-color:#F9F9F9; empty-cells:show;"><TBODY><TR><TH>Temp </TH><TH>  Time </TH></TR><TR><TD>95°</TD><TD>03:00</TD></TR><TR><TD style="border-top:solid 1px #000000;">94°</TD><TD style="border-top:solid 1px #000000;border-right:solid 1px #000000;">00:30</TD><TD rowspan="3">x 30 cycles
</TD></TR><TR><TD>45°'*'</TD><TD style="border-right:solid 1px #000000;">00:30
</TD></TR><TR><TD style="border-bottom:solid 1px #000000;">72°</TD><TD style="border-bottom:solid 1px #000000;border-right:solid 1px #000000;">01:00 /kb
</TD></TR><TR><TD>72°</TD><TD>10:00</TD></TR><TR><TD>4°</TD><TD>8</TD></TR></TBODY></TABLE><P>'*'with 45°C at a minimum of primer annealing temp
</P><H3><SPAN class="mw-headline" id="Gel_extraction">Gel extraction</SPAN></H3><P>After gel electrophoresis the digested vector and insert have to be purified from the gel. With the help of a UV lamp, the bands were quickly excised from the gel without exposing the DNA too long to UV light. Afterwards the DNA was purified with the QIAquick Gel extraction kit. Three volumes of buffer QG were added to one volume of gel. The gel fragment was dissolved by incubation for 10 min at 50°C. Afterwards one volume of 100% isopropanol was added. The solution was applied on a QIAquick spin column after this has been placed into a provided 2 ml collection tube. By centrifugation for 1 min at 13.000 rpm the DNA was bound to the column. The flow-through was discarded and the column was placed in the same collection tube. To remove all traces of agarose from the column, 500 µl of wash buffer QC was added followed by centrifugation for 1 min at 13.000 rpm. The flow-through was discarded and the column was washed with 750 µl of buffer PE for 1 min at 13.000 rpm. Afterwards the flow-through was discarded. An additional centrifugation for 1 min at 13.000 rpm helped to remove the residual ethanol. The column was placed into a new 1.5 ml microcentrifuge tube and it was eluted with 30 µl of ddH2O.
</P><H3><SPAN class="mw-headline" id="Large_scale_preparation_of_plasmid_DNA">Large scale preparation of plasmid DNA</SPAN></H3><P>150 ml LB-Medium with 150 µl ampicillin was inoculated with 50 µl of bacteria culture which grew overnight on a shaker at 37°C. The plasmid DNA was isolated using QiAprep Spin MAxiprep kit from Qiagen and the protocol was followed. The overnight culture was centrifuged for 20 min at 4000 rpm at 4°C using an SLA 1500 Rotor. Afterwards the LB-medium was discarded and the pellet was homogeneously resuspended in 10 ml of precooled Buffer P1. After having added 10 ml of Buffer P2 the mixture was inverted 4-6 times and incubated for 5 min at RT before adding 10 ml of chilled Buffer P3. Thereafter the lysate was poured into a prepared QIAfilter Maxi Cartridge and incubated at RT for 10 min. During this time a QIAGEN-tip 500 was equilibrated by applying 10 ml of Buffer QBT and allowing the column to empty by gravity flow. The cell lysate was filtered into the QIAGEN-tip. The cleared lysate entered the resin by gravity flow and after washing with 2 x 30 ml Buffer QC the Plasmid DNA was eluted with 15 ml Buffer QF. After this the DNA was precipitated by adding 10.5 ml isopropanol and centrifuged at 4,000 rpm for 45 min at 4°C. The supernatant was discarded and the DNA pellet was washed with 5 ml ethanol (70%) and centrifuged at 4,000 rpm for 15 min. After air-drying the pellet the DNA was redissolved in H2O. </P><H3><SPAN class="mw-headline" id="Plasmid-DNA_isolation">Plasmid-DNA isolation</SPAN></H3><P>5 ml LB-Medium with 5 µl ampicillin was inoculated with single colonies which grew overnight on a shaker at 37°C. The plasmid DNA was isolated using QiAprep Spin Miniprep kit from Qiagen and following the manufacturer’s protocol. 4 ml of each overnight culture was pelleted in 2 ml microcentrifuge tubes during two steps of centrifugation at 13.000 rpm. Subsequently the pellet was resuspended in 250 µl of chilled buffer P1. 250 µl of lysis buffer P2 was added and the solution was mixed thoroughly by inverting the tube 4-6 times. After adding 350 µl of the neutralization Buffer N3 the solution was mixed immediately and thoroughly by inverting the tube 4-6 times. Thereafter the mixture was centrifuged. The supernatants were applied to a QIAprep column which was put in a 2 ml collection tube. It was centrifuged for 1 min at 13.000 rpm and the flow-through was discarded. After adding 500 µl of wash buffer PB, it was centrifuged for 1 min at 13.000 rpm and the flow-through was discarded. Once more, it was washed with 750 µl of wash buffer PE. In an additional centrifugation for 1 min at 13.000 rpm the residual wash buffer was removed. The QIAprep column was placed into a clean 1.5 ml microcentrifuge tube and the plasmid DNA was eluted in 30 µl ddH2O.</P><H3><SPAN class="mw-headline" id="Purification_of_PCR_product">Purification of PCR product</SPAN></H3><P>One volume of buffer PBI was added to one volume of the PCR sample mix. The sample was applied to a QIAquick column which has been placed into a provided 2 ml collection tube. It was centrifuged for 1 min at 13.000 rpm and the flow-through was discarded and the column was placed in the same collection tube. After this 750 µl of buffer PE was added to wash the column. It was centrifuged for 1 min at 13.000 rpm. The flow-through was discarded and the column was placed in the same collection tube. It was centrifuged for 1 min at 13.000 rpm. Afterwards the QIAquick column was placed into a new 1.5 ml microcentrifuge tube and it was eluted with 40 µl ddH2O.
</P><H3><SPAN class="mw-headline" id="Sequencing">Sequencing</SPAN></H3><P>performed by the GATC Biotech company
</P><H2><SPAN class="mw-headline" id="Cell_culture">Cell culture</SPAN></H2><H3><SPAN class="mw-headline" id="Media">Media</SPAN></H3><UL><LI>HeLa - DMEM, 10% Fetal Bovine Serum, 100 unit Penicillin, 100 µg/ml Streptomycin, 2mM L-Glutamin
</LI><LI>HEK293T - DMEM, 10% Fetal Bovine Serum, 100 unit Penicillin, 100 µg/ml Streptomycin, 2mM L-Glutamin
</LI><LI>HEK T-REx - DMEM, 10% Fetal Bovine Serum, 100 unit Penicillin, 100 µg/ml Streptomycin, 2mM L-Glutamin, 50µg/ml Blasticidin, 50µg/ml Zeocin
</LI><LI>HEK T-REx (with pcDNA5 integrated) - DMEM, 10% Fetal Bovine Serum, 100 unit Penicillin, 100 µg/ml Streptomycin, 2mM L-Glutamin, 50µg/ml Blasticidin, 100µg/ml Hygromycin
</LI><LI>Huh7 - DMEM, 10% Fetal Bovine Serum, 100 unit Penicillin, 100 µg/ml Streptomycin, 2mM L-Glutamin, 1% Non-essential amino acids
</LI><LI>HepG2 - DMEM, 10% Fetal Bovine Serum, 100 unit Penicillin, 100 µg/ml Streptomycin, 2mM L-Glutamin
</LI><LI>Hepa1.6 - DMEM, 10% Fetal Bovine Serum, 100 unit Penicillin, 100 µg/ml Streptomycin, 2mM L-Glutamin
</LI><LI>primary hepatocytes - William's Medium E, 10% Fetal Bovine Serum, 100 unit Penicillin, 100 µg/ml Streptomycin, 2mM L-Glutamin, 100nM Dexamethasone
</LI></UL><H3><SPAN class="mw-headline" id="Passaging">Passaging</SPAN></H3><UL><LI>remove media
</LI><LI>wash cells one time in PBS (10ml)
</LI><LI>remove PBS; add 2 ml of trypsin-EDTA solution and incubate cells for 5 min at 37 °C
</LI><LI>add 5 ml of the according media
</LI><LI>take 1/10th of the cell suspension and plate out on the according dish (either p100 dish, 6 well plate or T-flask)
</LI></UL><H3><SPAN class="mw-headline" id="Coating">Coating</SPAN></H3><P>for 96-well plates
</P><UL><LI>add 30 µl of poly-L-lysine solution (Sigma) to each well (make sure whole surface is covered with solution)
</LI><LI>leave for 30' in the incubator
</LI><LI>remove poly-L-lysine solution
</LI><LI>wash once with 1x PBS
</LI></UL><H3><SPAN class="mw-headline" id="Transfection">Transfection</SPAN></H3><H4><SPAN class="mw-headline" id="FuGENE">FuGENE</SPAN></H4><P>DAY1
</P><UL><LI>seed cells
</LI></UL><P>DAY2 
</P><UL><LI>transfection
</LI><LI>to media (OptiMEM, no FBS) add FuGENE reagent (it is important to add FuGENE to media, not other way round and not to drop FuGENE on the walls of the tube)
</LI><LI>incubate 5 minutes at room temperature
</LI><LI>add DNA
</LI><LI>incubate 15 minutes
</LI><LI>mix well and add dropwise to the well
</LI></UL><TABLE class="wikitable sortable" border="0" style="text-align: center"><TBODY><TR bgcolor="#cccccc"><TH>culture format</TH><TH>number of cells seeded</TH><TH>volume of culture media</TH><TH>amount of DNA</TH><TH>volume of FuGENE</TH><TH>volume of media for FuGENE activation
</TH></TR><TR><TD>96-well</TD><TD>5 000</TD><TD>100µl</TD><TD>50ng</TD><TD>0.3µl</TD><TD>5µl
</TD></TR><TR><TD>24-well</TD><TD>30 000</TD><TD>500µl</TD><TD>200ng</TD><TD>0.6µl</TD><TD>20µl
</TD></TR><TR><TD>6-well</TD><TD>100 000</TD><TD>2500µl</TD><TD>1000ng</TD><TD>3µl</TD><TD>100µl
</TD></TR><TR><TD>labtec chamber</TD><TD>10 000</TD><TD>400µl</TD><TD>200ng</TD><TD>0.4µl</TD><TD>20µl
</TD></TR></TBODY></TABLE><H4><SPAN class="mw-headline" id="PEI">PEI</SPAN></H4><P>150cm<SUP>2</SUP> plate format
</P><UL><LI>mixture 1:
</LI><LI>44ug DNA
</LI><LI>up to 790µl of H2O each flask
</LI><LI>add 790µl of 300mM NaCl
</LI></UL><LI>mixture 2:
</LI><LI>343µl of PEI
</LI><LI>790µl of 300mM NaCl
</LI><LI>add mixture 2 to mixture 1
</LI><LI>incubate 10 min at RT
</LI><LI>add to cells
</LI><H4><SPAN class="mw-headline" id="HBSS">HBSS</SPAN></H4><P>150cm<SUP>2</SUP> flask format
</P><UL><LI>50ugDNA
</LI><LI>2.67ml 250mM CaCl2
</LI><LI>2.67ml 2x HBSS
</LI></UL><UL><LI>warm up both CaCl and HBSS
</LI><LI>add plasmid to CaCl2 solution
</LI><LI>pour 2x HBSS to erlenmeyer flask
</LI><LI>very slowly add CaCl2 to HBSS, mix vigorously during adding (should become turbid)
</LI><LI>incubate for one minute in the room temperature
</LI><LI>add transfection mix to media (1 volume of mix, 2 volumes of media)
</LI><LI>pour on cells
</LI></UL><H2><SPAN class="mw-headline" id="Measurements"> Measurements </SPAN></H2><H3><SPAN class="mw-headline" id="ELISA"> ELISA </SPAN></H3><H4><SPAN class="mw-headline" id="Procedure"> Procedure </SPAN></H4><OL><LI> Coating
</LI><LI> dilute 1st antibody 1:1000 in coating buffer
</LI><LI> add 50 µl per well
</LI><LI> incubate over night at 4°C
</LI><LI> Blocking
</LI><LI> wash three times with 300 µl wash buffer per well
</LI><LI> add 200 µl blocking buffer per well
</LI><LI> incubate more than 1 hour at room temperature
</LI><LI> alternative: over-night at 4°C or freezing at -20°C
</LI><LI> Sample Loading
</LI><LI> 50 µl per well
</LI><LI> incubate 2 hours at 37°C
</LI><LI> wash five times
</LI><LI> Detection
</LI><LI> dilute 2nd antibody 1:1000 in dilution buffer
</LI><LI> add 100 µl per well
</LI><LI> incubate 1 hour at 37°C
</LI><LI> wash five times
</LI><LI> add 100 µl substrate per well
</LI><LI> incubate at room temperature for 5 - 10 min
</LI><LI> stop with 100µl 2M H<SUB>2</SUB>SO<SUB>4</SUB> per well
</LI><LI> read absorbance at 450 nm wavelength
</LI></OL><H4><SPAN class="mw-headline" id="Buffers"> Buffers </SPAN></H4><UL><LI><B>Coating Buffer</B> (0.1M NaHCO<SUB>3</SUB>, 0.1M Na<SUB>2</SUB>CO<SUB>3</SUB>, pH 9.5) 
</LI></UL><UL><LI><B>Diluting/Blocking Buffer (db)</B> (0.25 ml Tween 20, 30 g BSA, add 1x PBS to 500 ml)
</LI></UL><UL><LI><B>Wash Buffer</B> (0.5 ml Tween 20, 1L PBS 1x)
</LI></UL><DL><DD><B>Calibration</B></DD></DL><UL><LI> 960 µl db + 40 µl pooled plasma = 200 ng/ml
</LI><LI> add 500 µl of above into 500 µl of db = 100 ng/ml
</LI><LI> serial dilutions continued down to 1.5625  ng/ml
</LI></UL><DL><DD><B>Detection reagents</B></DD></DL><UL><LI> TMB Substrate Kit (Pierce, Thermo Scientific)
</LI><LI> 2M H<SUB>2</SUB>SO<SUB>4</SUB></LI></UL><H3><SPAN class="mw-headline" id="Microscopy"> Microscopy </SPAN></H3><P>To measure GFP and BFP fluorescence intensity, we used microscopy. Fluorescence was fist evaluated using the Leica DM IRB epifluorescence microscope. Only cells which were positive for transfection were measured. First, the cells were washed with 1x PBS and detached from the plate using Trypsin. 30µl Trypsin was added to each well, incubated for ten minutes at room temperature. Cells were resuspended in 170µl 1%BSA in PBS and replicates for each condition were pooled into 24 well plates.  100-150µl were used for confocal microscopy. 
</P><P>Single images were obtained using the Leica TCS SP5 confocal microscope and camera with the Leica AF6000 imaging software. GFP fluorescence was excited by Argon 488nm laser and measured at 520-560nm, BFP fluorescence was excited by UV laser at 405nm and measured at 440-460nm. Pictures were taken sequentially line by line in three different channels for GFP, BFP and bright field. 
</P><DIV class="center"><DIV class="thumb tnone"><DIV class="thumbinner" style="width:602px;"><DIV class="thumbcaption"><B>HeLa cells two days after transfection with miMeasure</B> (A) fluorescence signal GFP channel, 8bit; (B) fluorescence signal BFP channel, 8bit; (C) merge of channels A and B, RGB (D) cells after segmentation and automated cell counting and annotation</DIV></DIV></DIV></DIV><P>To analyze the fluorescence of single cells, we segmented the images using ImageJ. In 8bit pictures, we set the threshold for each channel to 50, thereby filtering the background. This allows us to annotate cells automatically using the “analyze particles” tool. We could now get the fluorescence intensity for each single cell on each channel (GFP or BFP) as an 8bit output, i.e. a value between 50 and 255. Panel 1 shows an example of one such image in different channels and after segmentation. From the data thus obtained, we calculated the GFP:BFP ratios for each cell using a simple algorithm. We could then visualize the mean of these rations in a bar plot or use all the data to calculate a linear regression curve.
</P><UL><LI> Consumables and Chemicals
</LI></UL><P>PerkinElmer ViewPlate, product number: 6004920
Nunc 24 well microplate, product number: 176740
1x PBS (1.37mM NaCl, 0.27mM KCl, 10mM Na2HPO4, 0.2mM KH2PO4)
0.05% Trypsin-EDTA, Invitrogen GIBCO, product number: 15400054</P><UL><LI> Instruments
</LI></UL><P>Leica DM IRB
Leica SP5
ImageJ version 1.43</P><H3><SPAN class="mw-headline" id="Dual_Luciferase_Assay"> Dual Luciferase Assay </SPAN></H3><P>We measured the knockdown of firefly luciferase using the Promega Dual Luciferase Reporter Assay. 
The DLR™ Assay System provides an efficient mean of performing dual-reporter assays, where the activities of firefly (<I>Photinus pyralis</I>) and Renilla (<I>Renilla reniformis</I>) luciferases (RL) are measured sequentially from a single sample. Firefly and Renilla luciferases can be used as a good reporter system, as those two enzymes have dissimilar enzyme structures and substrate requirements. This allows for selective discrimination between their bioluminescent reactions. The firefly luciferase (FL) reporter is measured first by adding Luciferase Assay Reagent II (LAR II) to generate a stabilized luminescent signal. After quantifying the firefly luminescence, this reaction is quenched, and the Renilla luciferase reaction is simultaneously initiated by adding Stop &amp; Glo® Reagent to the same tube. The Stop &amp; Glo® Reagent also produces a stabilized signal from the Renilla luciferase, which decays slowly over the course of the measurement. Here, Renilla luciferase is used for normalization. The measurements were conducted on the Promega GLOMAX 96 Microplate Luminometer using the Promega standard protocol (<A href="https://2010.igem.org/Team:Heidelberg/Project/References#Materials_and_Methods" class="external text" rel="nofollow">Sherf et al., 1996</A>). 
Twenty hours after transfection, cells were washed with 1x PBS and lysed using 1x Passive Lysis Buffer (5x stock solution diluted with distilled water), shaking for 30 minutes at 37°C. 10µl of the lysate were transferred to a white microplate (LumaPlate) as required for Luminometer measurements. 
</P><P>LAR II reagent was prepared by resuspending Luciferase Assay Substrate in 10ml Luciferase Assay Buffer II. For Stop &amp; Glo reagent, 2.1ml 50x Stop &amp; Glo substrate and 105ml Stop &amp; Glo Buffer were added to the amber Stop &amp; Glo reagent bottle and mixed by vortexing. Reagents where stored in 15ml aliquots at -80°C and thawed freshly prior to each measurement.
</P><P>To set up the Luminometer, the two injectors where flushed with distilled water, 70% ethanol, again water and air, three times each. Afterwards, they were primed three times with substrate reagents. 
</P><P>The activity of the first luciferase (firefly) was measured by adding 25µl of LAR II reagent to the well. The enzyme reacts upon translation without further processing and oxidates beetle luciferin, resulting in photon emission that can be measured. In addition to beetle luciferin, the LAR II reagent contains coenzyme A, which accelerates the reaction and thus creates a prolonged luminescence signal. The luminescence was measured two seconds after addition of the reagent, for ten seconds. Afterwards, 25µl Stop &amp; Glo reagent was added, which is able to quench the firefly luciferase activity and simultaneously contains the substrate for Renilla luciferase, coelenterazine. This second reaction also emits photons upon oxidation of the substrate. Addition of substrates and light emission measurements were conducted automatically by the GLOMAX Luminometer.
</P><H4><SPAN class="mw-headline" id="Consumables_and_Reagents"> Consumables and Reagents </SPAN></H4><P>LumaPlate, PerkinElmer, catalogue number 6005630
Promega Dual-Luciferase® Reporter Assay System, catalogue number E1910
</P><H4><SPAN class="mw-headline" id="Instruments"> Instruments </SPAN></H4><P>Promega GLOMAX 96 Microplate Luminometer
</P><H3><SPAN class="mw-headline" id="Plate_reader_measurements"> Plate reader measurements </SPAN></H3><TABLE id="toc" class="toc"><TBODY><TR><TD><DIV id="toctitle"><H2>Contents</H2></DIV><UL><LI class="toclevel-1 tocsection-1"><A href="#Materials"><SPAN class="tocnumber">1</SPAN><SPAN class="toctext">Materials</SPAN></A></LI><LI class="toclevel-2 tocsection-2"><A href="#Kits"><SPAN class="tocnumber">1.1</SPAN><SPAN class="toctext">Kits</SPAN></A></LI><LI class="toclevel-2 tocsection-3"><A href="#Marker"><SPAN class="tocnumber">1.2</SPAN><SPAN class="toctext">Marker</SPAN></A></LI><LI class="toclevel-2 tocsection-4"><A href="#Enzymes"><SPAN class="tocnumber">1.3</SPAN><SPAN class="toctext">Enzymes</SPAN></A></LI><LI class="toclevel-2 tocsection-5"><A href="#Bacteria"><SPAN class="tocnumber">1.4</SPAN><SPAN class="toctext">Bacteria</SPAN></A></LI><LI class="toclevel-3 tocsection-6"><A href="#Media_and_Antibiotics"><SPAN class="tocnumber">1.4.1</SPAN><SPAN class="toctext">Media and Antibiotics</SPAN></A></LI></UL><LI class="toclevel-1 tocsection-7"><A href="#Methods"><SPAN class="tocnumber">2</SPAN><SPAN class="toctext">Methods</SPAN></A></LI><LI class="toclevel-2 tocsection-8"><A href="#Cloning"><SPAN class="tocnumber">2.1</SPAN><SPAN class="toctext">Cloning</SPAN></A></LI><LI class="toclevel-3 tocsection-9"><A href="#Agarose_Gel_Electrophoresis"><SPAN class="tocnumber">2.1.1</SPAN><SPAN class="toctext">Agarose Gel Electrophoresis</SPAN></A></LI><LI class="toclevel-3 tocsection-10"><A href="#Colony_PCR"><SPAN class="tocnumber">2.1.2</SPAN><SPAN class="toctext">Colony PCR</SPAN></A></LI><LI class="toclevel-3 tocsection-11"><A href="#Gel_extraction"><SPAN class="tocnumber">2.1.3</SPAN><SPAN class="toctext">Gel extraction</SPAN></A></LI><LI class="toclevel-3 tocsection-12"><A href="#Large_scale_preparation_of_plasmid_DNA"><SPAN class="tocnumber">2.1.4</SPAN><SPAN class="toctext">Large scale preparation of plasmid DNA</SPAN></A></LI><LI class="toclevel-3 tocsection-13"><A href="#Plasmid-DNA_isolation"><SPAN class="tocnumber">2.1.5</SPAN><SPAN class="toctext">Plasmid-DNA isolation</SPAN></A></LI><LI class="toclevel-3 tocsection-14"><A href="#Purification_of_PCR_product"><SPAN class="tocnumber">2.1.6</SPAN><SPAN class="toctext">Purification of PCR product</SPAN></A></LI><LI class="toclevel-3 tocsection-15"><A href="#Sequencing"><SPAN class="tocnumber">2.1.7</SPAN><SPAN class="toctext">Sequencing</SPAN></A></LI><LI class="toclevel-2 tocsection-16"><A href="#Cell_culture"><SPAN class="tocnumber">2.2</SPAN><SPAN class="toctext">Cell culture</SPAN></A></LI><LI class="toclevel-3 tocsection-17"><A href="#Media"><SPAN class="tocnumber">2.2.1</SPAN><SPAN class="toctext">Media</SPAN></A></LI><LI class="toclevel-3 tocsection-18"><A href="#Passaging"><SPAN class="tocnumber">2.2.2</SPAN><SPAN class="toctext">Passaging</SPAN></A></LI><LI class="toclevel-3 tocsection-19"><A href="#Coating"><SPAN class="tocnumber">2.2.3</SPAN><SPAN class="toctext">Coating</SPAN></A></LI><LI class="toclevel-3 tocsection-20"><A href="#Transfection"><SPAN class="tocnumber">2.2.4</SPAN><SPAN class="toctext">Transfection</SPAN></A></LI><LI class="toclevel-4 tocsection-21"><A href="#FuGENE"><SPAN class="tocnumber">2.2.4.1</SPAN><SPAN class="toctext">FuGENE</SPAN></A></LI><LI class="toclevel-4 tocsection-22"><A href="#PEI"><SPAN class="tocnumber">2.2.4.2</SPAN><SPAN class="toctext">PEI</SPAN></A></LI><LI class="toclevel-4 tocsection-23"><A href="#HBSS"><SPAN class="tocnumber">2.2.4.3</SPAN><SPAN class="toctext">HBSS</SPAN></A></LI><LI class="toclevel-2 tocsection-24"><A href="#Measurements"><SPAN class="tocnumber">2.3</SPAN><SPAN class="toctext">Measurements</SPAN></A></LI><LI class="toclevel-3 tocsection-25"><A href="#ELISA"><SPAN class="tocnumber">2.3.1</SPAN><SPAN class="toctext">ELISA</SPAN></A></LI><LI class="toclevel-4 tocsection-26"><A href="#Procedure"><SPAN class="tocnumber">2.3.1.1</SPAN><SPAN class="toctext">Procedure</SPAN></A></LI><LI class="toclevel-4 tocsection-27"><A href="#Buffers"><SPAN class="tocnumber">2.3.1.2</SPAN><SPAN class="toctext">Buffers</SPAN></A></LI><LI class="toclevel-3 tocsection-28"><A href="#Microscopy"><SPAN class="tocnumber">2.3.2</SPAN><SPAN class="toctext">Microscopy</SPAN></A></LI><LI class="toclevel-3 tocsection-29"><A href="#Dual_Luciferase_Assay"><SPAN class="tocnumber">2.3.3</SPAN><SPAN class="toctext">Dual Luciferase Assay</SPAN></A></LI><LI class="toclevel-4 tocsection-30"><A href="#Consumables_and_Reagents"><SPAN class="tocnumber">2.3.3.1</SPAN><SPAN class="toctext">Consumables and Reagents</SPAN></A></LI><LI class="toclevel-4 tocsection-31"><A href="#Instruments"><SPAN class="tocnumber">2.3.3.2</SPAN><SPAN class="toctext">Instruments</SPAN></A></LI><LI class="toclevel-3 tocsection-32"><A href="#Plate_reader_measurements"><SPAN class="tocnumber">2.3.4</SPAN><SPAN class="toctext">Plate reader measurements</SPAN></A></LI></TD></TR></TBODY></TABLE><P>  </P><DIV class="printfooter">
Retrieved from &quot;<A href="http://2010.igem.org/Team:Heidelberg/Notebook/Material_Methods">http://2010.igem.org/Team:Heidelberg/Notebook/Material_Methods</A>&quot;</DIV></DIV></DIV><DIV id="footer-box" class="noprint"><DIV id="footer"><UL id="f-list"><LI id="t-recentchanges"><A href="/Special:RecentChanges" title="Recent changes">Recent changes</A></LI><LI id="t-whatlinkshere"><A href="/Special:WhatLinksHere/Team:Heidelberg/Notebook/Material_Methods" title="List of all wiki pages that link here [j]" accesskey="j">What links here</A></LI><LI id="t-recentchangeslinked"><A href="/Special:RecentChangesLinked/Team:Heidelberg/Notebook/Material_Methods" title="Recent changes in pages linked from this page [k]" accesskey="k">Related changes</A></LI><LI id="t-specialpages"><A href="/Special:SpecialPages" title="List of all special pages [q]" accesskey="q">Special pages</A></LI><LI><A href="/Special:Preferences">My preferences</A></LI></UL></DIV><DIV id="footer"><UL id="f-list"><LI id="t-print"><A href="/wiki/index.php?title=Team:Heidelberg/Notebook/Material_Methods&amp;printable=yes" title="Printable version of this page [p]" accesskey="p">Printable version</A></LI><LI id="t-permalink"><A href="/wiki/index.php?title=Team:Heidelberg/Notebook/Material_Methods&amp;oldid=205103" title="Permanent link to this revision of the page">Permanent link</A></LI><LI id="privacy"><A href="/2010.igem.org:Privacy_policy" title="2010.igem.org:Privacy policy">Privacy policy</A></LI><LI id="disclaimer"><A href="/2010.igem.org:General_disclaimer" title="2010.igem.org:General disclaimer">Disclaimers</A></LI></UL></DIV></DIV></DIV></BODY></HTML>