<HTML xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en" dir="ltr">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki  ltr ns-0 ns-subject page-Team_Heidelberg_Notebook_miMeasure_July"><DIV id="globalWrapper"><DIV id="top-section"><DIV id="p-logo"><A href="/Main_Page" title="Main Page">&quot;
	    </A></DIV><DIV id="menubar" class="left-menu noprint"><UL><LI class="selected"><A href="/Team:Heidelberg/Notebook/miMeasure/July">Page               </A></LI><LI class="new"><A href="/wiki/index.php?title=Talk:Team:Heidelberg/Notebook/miMeasure/July&amp;action=edit&amp;redlink=1">Discussion               </A></LI><LI><A href="/wiki/index.php?title=Team:Heidelberg/Notebook/miMeasure/July&amp;action=edit">View source               </A></LI><LI><A href="/wiki/index.php?title=Team:Heidelberg/Notebook/miMeasure/July&amp;action=history">History               </A></LI><LI style="color:#808080;cursor:default">teams</LI></UL></DIV><DIV class="right-menu noprint" id="menubar"><UL><LI id="pt-login"><A href="/wiki/index.php?title=Special:UserLogin&amp;returnto=Team:Heidelberg/Notebook/miMeasure/July" title="You are encouraged to log in; however, it is not mandatory [o]" accesskey="o">Log in</A></LI></UL></DIV><DIV id="search-controls" class="noprint"><FORM action="/Special:Search" id="searchform"> </FORM></DIV></DIV><DIV id="content"><H1 class="firstHeading">Team:Heidelberg/Notebook/miMeasure/July</H1><DIV id="bodyContent"><H3 id="siteSub" class="noprint">From 2010.igem.org</H3><DIV id="header"><DIV id="menu"><UL id="nav"><LI id="menu_home"><A class="aHome" href="https://2010.igem.org/Team:Heidelberg">Home</A></LI><LI id="menu_team"><A class="aTeam" href="https://2010.igem.org/Team:Heidelberg/Team">Team</A><TABLE class="menutable" cellpadding="0" cellspacing="0"><TBODY><TR><TD class="lefttdtopteam"><A class="aTeamOverview" href="https://2010.igem.org/Team:Heidelberg/Team">Overview</A><A class="aTeamMembers" href="https://2010.igem.org/Team:Heidelberg/Team/Members">Students</A><A class="aTeamMembers" href="https://2010.igem.org/Team:Heidelberg/Team/Members#Instructors">Instructors</A><A class="aTeamMembers" href="https://2010.igem.org/Team:Heidelberg/Team/Members#Advisors">Advisors</A><A class="aInstitutes" href="https://2010.igem.org/Team:Heidelberg/Team/Institutes">Institutes</A><A class="aGallery" href="https://2010.igem.org/Team:Heidelberg/Team/Gallery">Gallery</A><A class="aDance" href="https://2010.igem.org/Team:Heidelberg/midance">miDANCE</A><A class="amithanks" href="https://2010.igem.org/Team:Heidelberg/Team/miThanks">Acknowledgements</A></TD><TD class="righttdtop"> </TD></TR></TBODY></TABLE></LI><LI id="menu_project"><A class="aProject" href="https://2010.igem.org/Team:Heidelberg/Project">Project</A><TABLE class="menutable" cellpadding="0" cellspacing="0"><TBODY><TR><TD class="lefttdtopproject"><A class="aAbstract" href="https://2010.igem.org/Team:Heidelberg/Project">Abstract</A><A class="aIntroduction" href="https://2010.igem.org/Team:Heidelberg/Project/Introduction">Introduction</A><A class="aProject_miRNA_Kit" href="https://2010.igem.org/Team:Heidelberg/Project/miRNA_Kit">Synthetic miRNA Kit</A><A class="aPro_Capsid_Shuffling" href="https://2010.igem.org/Team:Heidelberg/Project/Capsid_Shuffling">Capsid Shuffling</A><A class="aPro_Homology" href="https://2010.igem.org/Team:Heidelberg/Project/Capsid_Shuffling/Homology_Based">    Homology Based</A><A class="aPro_Virobytes" href="https://2010.igem.org/Team:Heidelberg/Project/Capsid_Shuffling/ViroBytes">    ViroBytes</A><A class="aPro_miMeasure" href="https://2010.igem.org/Team:Heidelberg/Project/miMeasure">miMeasure</A><A class="aPro_Mouse" href="https://2010.igem.org/Team:Heidelberg/Project/Mouse_Infection"><I>in vivo</I> Study</A><A class="aPro_Summary" href="https://2010.igem.org/Team:Heidelberg/Project/Summary">Summary &amp; Outlook</A><A class="aPro_Ref" href="https://2010.igem.org/Team:Heidelberg/Project/References">References</A><A class="aPro_Ref" href="https://2010.igem.org/Team:Heidelberg/Project/Contributions">Contributions</A></TD><TD class="righttdtop"> </TD></TR></TBODY></TABLE></LI><LI id="menu_parts"><A class="aParts" href="https://2010.igem.org/Team:Heidelberg/Parts">Parts</A><TABLE class="menutable" cellpadding="0" cellspacing="0"><TBODY><TR><TD class="lefttdtop"><A class="aparts_ger" href="https://2010.igem.org/Team:Heidelberg/Parts">Parts</A><A class="aparts_char" href="https://2010.igem.org/Team:Heidelberg/Parts/Characterization">Characterization</A></TD><TD class="righttdtop"> </TD></TR></TBODY></TABLE></LI><LI id="menu_modeling"><A class="aModeling" href="https://2010.igem.org/Team:Heidelberg/Modeling">Modeling</A><TABLE class="menutable" cellpadding="0" cellspacing="0"><TBODY><TR><TD class="lefttdtop"><A class="amode_over" href="https://2010.igem.org/Team:Heidelberg/Modeling">Introduction</A><A class="amiGUI" href="https://2010.igem.org/Team:Heidelberg/Modeling/miGUI">miBEAT</A><A class="amirockdown" href="https://2010.igem.org/Team:Heidelberg/Modeling/miRockdown">miRockdown</A><A class="amibsdesigner" href="https://2010.igem.org/Team:Heidelberg/Modeling/miBSdesigner">miBS designer</A><A class="amispec" href="https://2010.igem.org/Team:Heidelberg/Modeling/miSpec">mUTING</A><A class="amodelset" href="https://2010.igem.org/Team:Heidelberg/Modeling/trainingset">Descriptions</A></TD><TD class="righttdtop"> </TD></TR></TBODY></TABLE></LI><LI id="menu_notebook"><A class="aNotebook" href="https://2010.igem.org/Team:Heidelberg/Notebook">Notebook</A><TABLE class="menutable" cellpadding="0" cellspacing="0"><TBODY><TR><TD class="lefttdtop"><A class="aNote_miRNA_kit" href="https://2010.igem.org/Team:Heidelberg/Notebook/miRNA_Kit">Synthetic miRNA Kit</A><A class="aNote_Capsid_Shuffling">Capsid Shuffling</A><A class="aNote_Homology" href="https://2010.igem.org/Team:Heidelberg/Notebook/Capsid_Shuffling/Homology_Based">    Homology Based</A><A class="aNote_Virobytes" href="https://2010.igem.org/Team:Heidelberg/Notebook/Capsid_Shuffling/ViroBytes">    ViroBytes</A><A class="aNote_miMeasure" href="https://2010.igem.org/Team:Heidelberg/Notebook/miMeasure">miMeasure</A><A class="aNote_BSdesign" href="https://2010.igem.org/Team:Heidelberg/Notebook/BSDesign">Binding Site Design </A><A class="aNote_Mouse" href="https://2010.igem.org/Team:Heidelberg/Notebook/Mouse_Infection"><I>in vivo</I> Study</A><A class="aNote_Modeling" href="https://2010.igem.org/Team:Heidelberg/Notebook/Modeling">Modeling</A><A class="aNote_Material" href="https://2010.igem.org/Team:Heidelberg/Notebook/Material">Materials</A><A class="aNote_Methods" href="https://2010.igem.org/Team:Heidelberg/Notebook/Methods">Methods</A><A class="aNote_Psycho" href="https://2010.igem.org/Team:Heidelberg/Human_Practices/Psychology/Notebook">Psychological Survey</A><A class="aNote_Philo" href="https://2010.igem.org/Team:Heidelberg/Human_Practices/Philosophy/Part2">Philosophical Reflection</A><A class="aNote_Biosafety" href="https://2010.igem.org/Team:Heidelberg/Notebook/Biosafety">Biosafety</A></TD><TD class="righttdtop"> </TD></TR></TBODY></TABLE></LI><LI id="menu_human_practices"><A class="aHuman_Practices" href="https://2010.igem.org/Team:Heidelberg/Human Practices">Human Practices</A><TABLE class="menutable" cellpadding="0" cellspacing="0"><TBODY><TR><TD class="lefttdtop"><A class="aPsycho" href="https://2010.igem.org/Team:Heidelberg/Human Practices/Psychology">Psychological Survey</A><A class="aPhilo" href="https://2010.igem.org/Team:Heidelberg/Human Practices/Philosophy">Philosophical Reflection</A></TD><TD class="righttdtop"> </TD></TR></TBODY></TABLE></LI><LI id="menu_support"><A class="aSupport" href="https://2010.igem.org/Team:Heidelberg/Support">Sponsors</A></LI></UL></DIV></DIV><TABLE cellpadding="5" cellspacing="0" style="text-align: center; color:#4e93a4; border: 3px solid #4e93a4;"><TBODY><TR><TH colspan="7" style="background:#4e93a4;"><A href="https://2010.igem.org/Team:Heidelberg/Notebook/miMeasure/August" class="external text" rel="nofollow"><FONT color="white">August</FONT></A></TH></TR><TR style="background:#4e93a4; color:white"><TD width="20pt"><B>M</B></TD><TD width="20pt"><B>T</B></TD><TD width="20pt"><B>W</B></TD><TD width="20pt"><B>T</B></TD><TD width="20pt"><B>F</B></TD><TD width="20pt"><B>S</B></TD><TD width="20pt"><B>S</B></TD></TR><TR><TD><B>1</B></TD></TR><TR><TD><B>2</B></TD><TD><B>3</B></TD><TD><B>4</B></TD><TD><B>5</B></TD><TD><B>6</B></TD><TD><B>7</B></TD><TD><B>8</B></TD></TR><TR><TD><B>9</B></TD><TD><B>10</B></TD><TD><B>11</B></TD><TD><B>12</B></TD><TD><B>13</B></TD><TD><B>14</B></TD><TD><B>15</B></TD></TR><TR><TD><B>16</B></TD><TD><B>17</B></TD><TD><B>18</B></TD><TD><B>19</B></TD><TD><B>20</B></TD><TD><B>21</B></TD><TD><B>22</B></TD></TR><TR><TD><B>23</B></TD><TD><B>24</B></TD><TD><B>25</B></TD><TD><B>26</B></TD><TD><B>27</B></TD><TD><B>28</B></TD><TD><B>29</B></TD></TR><TR><TD><B>30</B></TD><TD><B>31</B></TD></TR></TBODY></TABLE><TABLE cellpadding="5" cellspacing="0" style="text-align: center; color:#4e93a4; border: 3px solid #4e93a4;"><TBODY><TR><TH colspan="7" style="background:#4e93a4;"><A href="https://2010.igem.org/Team:Heidelberg/Notebook/miMeasure/September" class="external text" rel="nofollow"><FONT color="white">September</FONT></A></TH></TR><TR style="background:#4e93a4; color:white"><TD width="20pt"><B>M</B></TD><TD width="20pt"><B>T</B></TD><TD width="20pt"><B>W</B></TD><TD width="20pt"><B>T</B></TD><TD width="20pt"><B>F</B></TD><TD width="20pt"><B>S</B></TD><TD width="20pt"><B>S</B></TD></TR><TR><TD><B>1</B></TD><TD><B>2</B></TD><TD><B>3</B></TD><TD><B>4</B></TD><TD><B>5</B></TD></TR><TR><TD><B>6</B></TD><TD><B>7</B></TD><TD><B>8</B></TD><TD><B>9</B></TD><TD><B>10</B></TD><TD><B>11</B></TD><TD><B>12</B></TD></TR><TR><TD><B>13</B></TD><TD><B>14</B></TD><TD><B>15</B></TD><TD><B>16</B></TD><TD><B>17</B></TD><TD><B>18</B></TD><TD><B>19</B></TD></TR><TR><TD><B>20</B></TD><TD><B>21</B></TD><TD><B>22</B></TD><TD><B>23</B></TD><TD><B>24</B></TD><TD><B>25</B></TD><TD><B>26</B></TD></TR><TR><TD><B>27</B></TD><TD><B>28</B></TD><TD><B>29</B></TD><TD><B>30</B></TD><TD colspan="7"><SPAN style="color:#ffffff">-</SPAN></TD></TR></TBODY></TABLE><TABLE cellpadding="5" cellspacing="0" style="text-align: center; color:#4e93a4; border: 3px solid #4e93a4;"><TBODY><TR><TH colspan="7" style="background:#4e93a4;"><A href="https://2010.igem.org/Team:Heidelberg/Notebook/miMeasure/October" class="external text" rel="nofollow"><FONT color="white">October</FONT></A></TH></TR><TR style="background:#4e93a4; color:white"><TD width="20pt"><B>M</B></TD><TD width="20pt"><B>T</B></TD><TD width="20pt"><B>W</B></TD><TD width="20pt"><B>T</B></TD><TD width="20pt"><B>F</B></TD><TD width="20pt"><B>S</B></TD><TD width="20pt"><B>S</B></TD></TR><TR><TD><B>1</B></TD><TD><B>2</B></TD><TD><B>3</B></TD></TR><TR><TD><B>4</B></TD><TD><B>5</B></TD><TD><B>6</B></TD><TD><B>7</B></TD><TD><B>8</B></TD><TD><B>9</B></TD><TD><B>10</B></TD></TR><TR><TD><B>11</B></TD><TD><B>12</B></TD><TD><B>13</B></TD><TD><B>14</B></TD><TD><B>15</B></TD><TD><B>16</B></TD><TD><B>17</B></TD></TR><TR><TD><B>18</B></TD><TD><B>19</B></TD><TD><B>20</B></TD><TD><B>21</B></TD><TD><B>22</B></TD><TD><B>23</B></TD><TD><B>24</B></TD></TR><TR><TD><B>25</B></TD><TD><B>26</B></TD><TD><B>27</B></TD><TD><B>28</B></TD><TD><B>29</B></TD><TD><B>30</B></TD><TD><B>31</B></TD></TR></TBODY></TABLE><P> </P></DIV><H3><SPAN class="mw-headline" id="05.2F07.2F2010_-_11.2F07.2F2010">05/07/2010 - 11/07/2010</SPAN></H3><P>
Preperation of competent E. coli Top10 and DH5alpha
Cell Culture Starting
</P><H3><SPAN class="mw-headline" id="12.2F07.2F2010_-_19.2F07.2F2010">12/07/2010 - 19/07/2010</SPAN></H3><P>
Preperation of RNA extracts for miRNA profiling
</P><UL><LI>Hela, Hek and HUH were plated on p100 dishes in their according media (10E6 cells/dish) and grown to 60-70 % confluence
</LI><LI>Infection of Hela p4, HUH-7 and HEK cells with AAV serotype 2 (GFP) and incubation for 48 h
</LI><LI>afterwards, RNA Extraction was performed according to the following protocol:
</LI></UL><P><B>Protocol</B></P><UL><LI>Samples were afterwards analyzed for miRNA expression by using the FEBIT miRNA profiling service
</LI></UL><UL><LI>pSB1AC3 from the registry was transformed into Top10 cells according to the standard transformation protocol; a day later, a 5 ml LB culture was inocculated and incubated for 8 hours; plasmid was extracted by applying the Qiagen MiniPrep Protocol
</LI></UL><P>
design of diraPCR oligos
</P><UL><LI> a diraPCR Designer tool (called miRACLE Designer) was programmed for enabling quick&amp;easy design of diraPCR oligos. The program takes any microRNA guiding strand sequence of choice and constructs synthetic microRNA binding site oligos. Those oligos have the following properties:
</LI></UL><DL><DD><UL><LI> they have a 5' and 3' anealing sequence which is inerte (has no natural microRNA target)
</LI><LI> the microRNA binding site of choice with randomized nucleotides at position 9-12 and a missmatch introduced at position 0
</LI><LI> an inerte spacer sequence of 0, 5 10 or 15 bp length
</LI></UL></DD></DL><P>
The miRACLE Designer constructs a set of 8 ready-to-order oligos and prints them out in the common FASTA format. When performing diraPCR, binding sites will be concatomerized with spacer sequences in between 15 and 30 bp length.
</P><P>design of measurement standard
</P><UL><LI> in order to design an appropriety standard respecting all the rules according to the standard of mammalian synthetic biology (RFC12), we developed a dual reporter measurment construct with the following properties:
</LI></UL><DL><DD><UL><LI> bidirectional CMV promoter driving both reporter genes
</LI><LI> distabilized EGFP and EBFP2 as reporter genes, as they are really similar in their whole structure and amino-acid sequence
</LI><LI> BBB prefix and suffix cloned behind the EGFP for easy swapping of binding site patterns
</LI><LI> unique BamHI and HindIII binding sites introduced behind the EBFP2 reporter for introducing a reference binding site
</LI><LI> strong SV40 terminators behind both reporters
</LI><LI> vector backbone, containing an Amp resistance, Hygromycin resistance, pBR322 origin and an FRT site for stable integration
</LI></UL></DD></DL><H3><SPAN class="mw-headline" id="20.2F07.2F2010">20/07/2010</SPAN></H3><P><B> Dilution of raPCR-Oligos </B></P><UL><LI> oligos (hsa_miR_886_3p_sp0...) and stop oligo (miRaPCR_Stop_fw_Ecori and miRA_PCR_Stop_rev) diluted to a concentration of 100 um
</LI><LI> Set up raPCR-reactions with different amounts of oligos being pooled
</LI></UL><DL><DD><UL><LI>2 ul of each hsa_miR_886_3p oligo
</LI><LI>1.0, 2 or 4 ul of each stop oligo
</LI><LI>25 ul of Phusion PCR MasterMix
</LI><LI>add water to a total volume of 50 ul
</LI></UL></DD></DL><P><B> Seven-Cyle PCR </B></P><UL><LI>run seven-cycle PCR under the following conditions
</LI></UL><P>
................................................
</P><DL><DD>95 °C/ 5 min
</DD></DL><P>................................................
</P><DL><DD>95 °C/ 30 sec
</DD><DD>55 °C/ 45 sec
</DD><DD>72 °C/ 30 sec
</DD></DL><P>................................................ (7x)
</P><DL><DD>4 °C/ forever
</DD></DL><P>................................................
</P><UL><LI> PCR purification of the PCR-product (elution in 42 ul of water)
</LI></UL><P><B> 25-cycle PCR </B></P><UL><LI> run construct amplification PCR according to the following protocol
</LI></UL><DL><DD><UL><LI>20 ul of purified PCR product 
</LI><LI>1 ul of each stop oligo
</LI><LI>3 ul of water
</LI><LI>25 ul of Phusion PCR MasterMix
</LI></UL></DD></DL><P>
................................................
</P><DL><DD>95 °C/ 5 min
</DD></DL><P>................................................
</P><DL><DD>95 °C/ 30 sec
</DD><DD>65 °C/ 45 sec
</DD><DD>72 °C/ 30 sec
</DD></DL><P>................................................ (25x)
</P><DL><DD>72 °C/ 5 min
</DD></DL><P>................................................
</P><DL><DD>4 °C/ forever
</DD></DL><P>................................................
<B> Gel Electrophoresis </B></P><UL><LI> PCR products from the 25 cycle PCR were analyzed on a Gel. Therefor, 5 ul of PCR product were mixed with 1 ul of 6x-loading dye and loaded on a 1.5 % agarose gel. The gel ran at 135 V for 55 min.
</LI></UL><H3><SPAN class="mw-headline" id="21.2F07.2F2010">21/07/2010</SPAN></H3><UL><LI> optimization of cycle number and oligo concentration for raPCR
</LI></UL><DL><DD><UL><LI>0.5 (BS0.5), 1 (BS1) or 1.5 (BS1.5) ul of each binding site oligo
</LI><LI>0 (Stop0), 0.5 (stop0.5) or 1 (stop1) ul of each stop oligo
</LI><LI>25 ul of Phusion PCR MasterMix
</LI><LI>add water to total volume of 25 ul
</LI></UL></DD></DL><P><B> 12-cycle PCR </B></P><UL><LI> run 12-cycle PCR according to the following program
</LI></UL><P>
................................................
</P><DL><DD>95 °C/ 5 min
</DD></DL><P>................................................
</P><DL><DD>95 °C/ 30 sec
</DD><DD>57 °C/ 45 sec
</DD><DD>72 °C/ 45 sec
</DD></DL><P>................................................ (12x)
</P><DL><DD>4 °C/ forever
</DD></DL><P>................................................
</P><UL><LI> add 1 ul of each stop oligo to reactions &quot;Stop0&quot;
</LI><LI> run 3-cycle PCR with the &quot;Stop0&quot; reaction according to the following protocol
</LI></UL><P>................................................
</P><DL><DD>95 °C/ 5 min
</DD></DL><P>................................................
</P><DL><DD>95 °C/ 30 sec
</DD><DD>65 °C/ 45 sec
</DD><DD>72 °C/ 45 sec
</DD></DL><P>................................................ (3x)
</P><DL><DD>4 °C/ forever
</DD></DL><P>................................................
</P><UL><LI>PCR purify all products
</LI></UL><P><B>25-cycle PCR</B></P><UL><LI> run 25-cycle PCR in order to amplify constructs
</LI></UL><DL><DD><UL><LI>20 ul of purified PCR product 
</LI><LI>1 ul of each stop oligo
</LI><LI>3 ul of water
</LI><LI>25 ul of Phusion PCR MasterMix
</LI></UL></DD></DL><DL><DD><UL><LI>for construct Stop0.5/BS1 set up an addition reaction as follows
</LI></UL></DD></DL><DL><DD><UL><LI>1 ul of purified PCR product 
</LI><LI>1 ul of each stop oligo
</LI><LI>22 ul of water
</LI><LI>25 ul of Phusion PCR MasterMix
</LI></UL></DD></DL><UL><LI>run 25-cycle PCR according to the following protocol
</LI></UL><P>
................................................
</P><DL><DD>95 °C/ 5 min
</DD></DL><P>................................................
</P><DL><DD>95 °C/ 30 sec
</DD><DD>65 °C/ 45 sec
</DD><DD>72 °C/ 50 sec
</DD></DL><P>................................................ (25x)
</P><DL><DD>72 °C/ 5 min
</DD></DL><P>................................................
</P><DL><DD>4 °C/ forever
</DD></DL><P>................................................
<B> Gel Electrophoresis </B></P><UL><LI> PCR products from the 25 cycle PCR were analyzed on a Gel. Therefor, 5 ul of PCR product were mixed with 1 ul of 6x-loading dye and loaded on a 1.5 % agarose gel. In addition, 2.5 ul of the 7-cycle PCR (07/21/2010) and 12-cycle PCR were loaded as well. The gel ran at 135 V for 55 min.
</LI></UL><H3><SPAN class="mw-headline" id="22.2F07.2F2010">22/07/2010</SPAN></H3><UL><LI> the PCR products from the previous days' raPCR were loaded on a 1 % agarose gel and the brightest 200, 400, 700 and 1400 bp bands were gel extracted using the Qiagen Gel Extraction Kit (elution in 32 ul of nuclease free water)
</LI><LI> Vector pSB1AC3 (500 ng) and the gel extraction products were digested with EcoRI and SpeI according to the standard digestion protocol
</LI><LI> PCR purification was performed for the digested raPCR products and a gel-extraction was performed for the ~ 3 kb vector band of pSB1AC3 (elution in 32 ul of nuclease-free water each)
</LI><LI> 12 ligation reactions were performed using 5 or 10 ul of insert and 5 ul of vector in each reaction; standard ligation was performed
</LI></UL><H3><SPAN class="mw-headline" id="23.2F07.2F2010">23/07/2010</SPAN></H3><UL><LI>transformation of ligation product (previous day) into E. coli Top10 cells according to the standard transformation protocol
</LI></UL><H3><SPAN class="mw-headline" id="26.2F07.2F2010">26/07/2010</SPAN></H3><UL><LI> of 21 colonies, 7 of each hsa-mir-886_3p 200, 400 and 700 bp band cloning product and inocculation of Miniprep LB cultures
</LI></UL><P>
single binding site synthesis

In order to construct a database of synthetic single binding sites, we developed and optimized a standardized single binding site synthesis protocol;
</P><DIV class="thumb tright"><DIV class="thumbinner" style="width:402px;"><DIV class="thumbcaption">The band at ~60 bp shows the synthesized single binding sites. For mir-886 and mir-122 there were some unspecific products or multimers obtained.</DIV></DIV></DIV><UL><LI>Protocol 1:
</LI></UL><DL><DD>set up the following reaction
<UL><LI>1 ul of each single binding site oligo (100 uM) has-mir-886-3p, hsa-mir-886-3p(perf), hsa-mir-769-5p, hsa-mir-769-5p(perf), hsa-mir-122, hsa-mir-122(perf)
</LI><LI> 1 ul of SBSSSS_EcoRI primer (reverse strand synthesis primer)
</LI><LI> 23 ul of water
</LI><LI> 25 ul of Phusion PCR MasterMix
</LI></UL></DD></DL><P>PCR protocol:

................................................
</P><DL><DD>95 °C/ 5 min
</DD></DL><P>................................................
</P><DL><DD>95 °C/ 30 sec
</DD><DD>60 °C/ 45 sec
</DD><DD>72 °C/ 30 sec
</DD></DL><P>................................................ (30x)
</P><DL><DD>72 °C/ 5 min
</DD></DL><P>................................................
</P><DL><DD>4 °C/ forever
</DD></DL><P>................................................
</P><UL><LI>analysis of the result on a 2 % agarose gel, run 45 min @ 135 V
</LI></UL><H3><SPAN class="mw-headline" id="27.2F07.2F2010">27/07/2010</SPAN></H3><DIV class="thumb tright"><DIV class="thumbinner" style="width:502px;"><DIV class="thumbcaption">The 200, 400 and 1000 bp cloning products were analyzed in an analytic PCR. Only The very first sample of the 200 bp product seems to be positive, as it shows a higher band (~ 300 bp) compared to the religated vector product band at ~ 200 bp</DIV></DIV></DIV><UL><LI> Miniprep of the LB cultures inocculated the previous day
</LI><LI> analytic PCR for detecting positive clones was set up according to the following protocol
</LI></UL><DL><DD><UL><LI> 3 ul of Miniprep product
</LI><LI> 0.5 ul of each standard sequencing primer VF and VR
</LI><LI> 21 ul of water
</LI><LI> 25 ul of PCR Phusion Mastermix
</LI></UL></DD></DL><P>
PCR protocol:

................................................
</P><DL><DD>95 °C/ 5 min
</DD></DL><P>................................................
</P><DL><DD>95 °C/ 30 sec
</DD><DD>60 °C/ 45 sec
</DD><DD>72 °C/ 30 sec
</DD></DL><P>................................................ (30x)
</P><DL><DD>72 °C/ 5 min
</DD></DL><P>................................................
</P><DL><DD>4 °C/ forever
</DD></DL><P>................................................
</P><UL><LI>PCR products were analyzed on a 1 % agarose gel, run for 45 min @ 135 V
</LI></UL><P>
Single Binding Site Synthesis - Optimization
</P><UL><LI>Further optimization of the single binding site synthesis protocol
</LI></UL><DIV class="thumb tright"><DIV class="thumbinner" style="width:402px;"><DIV class="thumbcaption">The unspecific or multimer side products were clearly reduced, but there is still sime unwanted, longer product in case of mir-122.</DIV></DIV></DIV><UL><LI>Protocol 2:
</LI></UL><DL><DD>set up the exact same reaction as on the day before, but using an anealing temperature of 65 °C
</DD></DL><UL><LI>analysis of the result on a 2 % agarose gel, run 45 min @ 135 V
</LI></UL><H3><SPAN class="mw-headline" id="31.2F07.2F2010">31/07/2010</SPAN></H3><UL><LI> ready-to-use vector pSB1A3 from the registry (25 ng/ul) digestion with EcoRI/PstI (NEB Buffer EcoRI + BSA, volume: 40 ul)
</LI><LI> PCR purification of digestion product
</LI><LI> subsequent digestion with DpnI (NEB  Buffer 4, volume: 40 ul)
</LI><LI> 5 ul of digestion product was analyzed on a 1 % agarose gel; unfortunately, there was no digestion product detectable (data not shown)
</LI></UL><UL><LI> transformation of pSB1A3-1 vector (Tom Knight, 2004) from the registry into Top10 cells according to the following protocol:
</LI></UL><DL><DD><UL><LI> 10 ul of nuclease-free water was pipetted into well C1 of Spring 2010 distribution plate 1
</LI><LI> incubation for 10 min at room temperature
</LI><LI> Top10 cells were thawn on ice (10 min) and 2 ul of DNA was added
</LI><LI> incubation on ice for 15 min
</LI><LI> 1 min heat-shock at 42 °C
</LI><LI> incubation on ince for 2 min
</LI><LI> adding 500 ul of LB media; mixing 
</LI><LI> centrifugation of the cells (5 min, 6000 rpm)
</LI><LI> supernatant was removed, pellet resuspended in 50 ul LB media and cells were plated on a LB-Amp plate
</LI></UL></DD></DL><UL><LI> digestion of the 200 bp, 400 bp and 700 bp band from the previous day (corresponding to hsa-mir-886-3p microRNA binding site patterns constructed via diraPCR
</LI><LI> no digestion product was detectable on the gel; therefor raPCR was repeated
</LI></UL><P>
diraPCR for constructing hsa-886-3p binding site patterns
</P><UL><LI> the diraPCR was pipetted according to the following protocol in three repeats:
</LI></UL><DL><DD><UL><LI> 1 ul of each binding site oligo
</LI><LI> 0.5 ul of each stop oligo
</LI><LI> 25 ul of Phusion MasterMix
</LI><LI> add water to final volume of 50 ul
</LI></UL></DD></DL><UL><LI> 12 cycle PCR was performed according to the standard miRACLE PCR protocol
</LI><LI> after PCR purification the subsequent PCR reaction was pipetted as follows:
</LI></UL><DL><DD><UL><LI> 5, 10 or 20 ul of 12 cycle PCR prodcut
</LI><LI> 0.5 ul of each stop oligo
</LI><LI> 25 ul of Phusion MasterMix
</LI><LI> add water to final volume of 50 ul
</LI></UL></DD></DL><UL><LI><DL><DD> the 25 cycle PCR was performed according to the standard miRACLE PCR protocol
</DD></DL></LI></UL></DIV><DIV class="printfooter">
Retrieved from &quot;<A href="http://2010.igem.org/Team:Heidelberg/Notebook/miMeasure/July">http://2010.igem.org/Team:Heidelberg/Notebook/miMeasure/July</A>&quot;</DIV></DIV><DIV id="footer-box" class="noprint"><DIV id="footer"><UL id="f-list"><LI id="t-recentchanges"><A href="/Special:RecentChanges" title="Recent changes">Recent changes</A></LI><LI id="t-whatlinkshere"><A href="/Special:WhatLinksHere/Team:Heidelberg/Notebook/miMeasure/July" title="List of all wiki pages that link here [j]" accesskey="j">What links here</A></LI><LI id="t-recentchangeslinked"><A href="/Special:RecentChangesLinked/Team:Heidelberg/Notebook/miMeasure/July" title="Recent changes in pages linked from this page [k]" accesskey="k">Related changes</A></LI><LI id="t-specialpages"><A href="/Special:SpecialPages" title="List of all special pages [q]" accesskey="q">Special pages</A></LI><LI><A href="/Special:Preferences">My preferences</A></LI></UL></DIV><DIV id="footer"><UL id="f-list"><LI id="t-print"><A href="/wiki/index.php?title=Team:Heidelberg/Notebook/miMeasure/July&amp;printable=yes" title="Printable version of this page [p]" accesskey="p">Printable version</A></LI><LI id="t-permalink"><A href="/wiki/index.php?title=Team:Heidelberg/Notebook/miMeasure/July&amp;oldid=170029" title="Permanent link to this revision of the page">Permanent link</A></LI><LI id="privacy"><A href="/2010.igem.org:Privacy_policy" title="2010.igem.org:Privacy policy">Privacy policy</A></LI><LI id="disclaimer"><A href="/2010.igem.org:General_disclaimer" title="2010.igem.org:General disclaimer">Disclaimers</A></LI></UL></DIV></DIV></BODY></HTML>