"
Page
Discussion
View source
History
teams
Log in
 
Team:Heidelberg/Notebook/miMeasure/September
From 2010.igem.org
Home
Team
Overview
Students
Instructors
Advisors
Institutes
Gallery
miDANCE
Acknowledgements
 
Project
Abstract
Introduction
Synthetic miRNA Kit
Capsid Shuffling
    Homology Based
    ViroBytes
miMeasure
in vivo Study
Summary & Outlook
References
Contributions
 
Parts
Parts
Characterization
 
Modeling
Introduction
miBEAT
miRockdown
miBS designer
mUTING
Descriptions
 
Notebook
Synthetic miRNA Kit
Capsid Shuffling
    Homology Based
    ViroBytes
miMeasure
Binding Site Design
in vivo Study
Modeling
Materials
Methods
Psychological Survey
Philosophical Reflection
Biosafety
 
Human Practices
Psychological Survey
Philosophical Reflection
 
Sponsors
July
M
T
W
T
F
S
S
1
2
3
4
5
6
7
8
9
10
11
12
13
14
15
16
17
18
19
20
21
22
23
24
25
26
27
28
29
30
31
-
August
M
T
W
T
F
S
S
1
2
3
4
5
6
7
8
9
10
11
12
13
14
15
16
17
18
19
20
21
22
23
24
25
26
27
28
29
30
31
September
M
T
W
T
F
S
S
1
2
3
4
5
6
7
8
9
10
11
12
13
14
15
16
17
18
19
20
21
22
23
24
25
26
27
28
29
30
31
-
October
M
T
W
T
F
S
S
1
2
3
4
5
6
7
8
9
10
11
13
14
15
16
17
18
19
20
21
22
23
24
25
26
27
28
29
30
-
 
13/09/2010
Restriction digest of psiCHECK-2 plasmid
This will be used as backbone for raPCR cloning.  Enzymes: XhoI and NotI
Assay:
5 µL 10x NEBuffer 3
5 µL 10x BSA
5 µL plasmid (psiCHECK-2, ~370 ng/µL)
3 µL XhoI
1 µL NotI
18.6 µL H2O
Restriction digest was performed for approx. 5h
raPCR to create binding sites for different miRNAs This random assembly PCR (raPCR) will be done to create binding site patterns for the miRNAs mentioned. In the first PCR step the oligos will basically anneal and constructs of different lengths will form. In the second step, the stop oligos are used as primers to amplify the previously formed constructs.
first tries are: hsa-mir-122, hsa-mir122(ran9-12) and mm-mir-376a/375 (Oligos: ra001-003 and ra006 )
stop-oligos: raPCR_AS13-stop_rev_NotI and  raPCR_AS13-stop_fw_XhoI ( ra017/018 )
spacer: raPCR_AS13-spacer(0) and raPCR_AS13-spacer(10)  ( ra012/013 )
Oligos were used in standard conc. (100µM)
1. PCR
Oligo
mir-122
mir-122(ran9-12)
mir-375/376a
miR
1 or 3 µL
1 or 3 µL
0.5 or 1.5 µL (each)
spacer(0)or (10)
1 µL
1 µL
1 µL
stop
0 or 0.5 µL each
0 or 0.5 µL each
0 or 0.5 µL each
Total: 12 reactions
each reaction was set up in 30 µL, using 2x Phusion Mastermix for 12 cycles
Temp
Time
95°
05:00
95°
00:30
x 12 cycles
57°
00:45
72°
00:45
4°
forever
PCR purification: each PCR was purified using Qiagen PCR purification Kit and eluted in 32 µL
for the next PCR, three assay will tried:
5µL eluate + 1 µL of each stop oligo in 50µL total volume
5µL eluate + 2 µL of each stop oligo in 50µL total volume
20µL eluate + 1 µL of each stop oligo in 50µL total volume
stop-oligos: raPCR_AS13-stop_rev_NotI and  raPCR_AS13-stop_fw_XhoI
2. PCR
In total there were 72 reactions. Each was run with 2x Phusion Mastermix, missing volume was filled with water.
Temp
Time
95°
05:00
95°
00:30
x 25 cycles
65°
00:45
72°
00:50
72°
05:00
4°
forever
DNA was stored in fridge afterwards
14/09/2010
72 PCRs were analyesed on 1% agarose gel
Best looking lanes of spacer(0) and spacer(10) were choosen for preparative gel
preparative gel (1.5%) was made for miR122
Sample for small Binding site patterns (100-250 bp) and large patterns (250-400 bp) were taken.
each were eluted in 1 column (a lot of work for gel extraction...)
15/09/2010
Gel extracted samples were digested with Not/Xho for cloning into psiCheck-2 vector: 5µL template in total 30µL using 1µL xho and 0.6µL Not enzyme
digest was then purified using qiagen nucleotide removal kit, elution in 30µL
due to a problem of the vector-size (it has 8000 instead of 6000 bp), ligation was set-up overnight at roomtemperature
ligation: 2µL buffer 1µL ligase 7µL water 1µL backbone (6000bp) 9µL restriction digest
16/09/2010
Transformation of ligations: 5µL ligation assay in 50µL TOP10 E.coli 25 min on ice 45sec heat shock on 42°C 1.5-2h shaking at 37°C
plated 200µL on Amp-Plates
after incubating ~8h, at 37°C, the plates were incubated overnight at room temperature
17/09/2010
Colonies were visible in reasonable numbers on every plate
colony pcr was performed to check clones
20µL assay PCR assay, using 2x PCR master mix (Taq) from Fermentas
one colony was dissolved in 20µL water
5µL of this bacteria solution was used as PCR template
PCR conditions as recommended from Fermentas
Minipreps will be setup in the evening for overnight cultures (4mL, Amp)
5 for each construct
28/09/2010
send 1.3, 1.5, 1.7, 3.8, 3.2 for sequencing
29/09/2010
got sequencing results of 1.3, 1.5, 1.7, 2.8, 3.2.
1.3 - 3x - all ok
1.5 - 2x - both ok
1.7 - 3x - all ok
2.8 - not ok
3.7 - 2x - both ok seperated by 10 nucleotide spacer
30/09/2010
raPCRfrom above(1, 2, 3, 4)
PCR purification ( nanodrop: c ≈ 100ng/µl)
digested: 2 x 1 µg DNA:
EcoRI
PstI
gel purification (nanodrop: c ≈ 25ng/µl)
Ligation (Quick Ligase and overnight ligation with T4 ligase) into pSB1C3
Vector ≈ 2000 bp
Insert ≈ 200 bp
Transformation
Retrieved from " http://2010.igem.org/Team:Heidelberg/Notebook/miMeasure/September "
Recent changes
What links here
Related changes
Special pages
My preferences
Printable version
Permanent link
Privacy policy
Disclaimers
