<HTML xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en" dir="ltr">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki  ltr ns-0 ns-subject page-Team_Heidelberg_Notebook_Methods"><DIV id="globalWrapper"><DIV id="top-section"><DIV id="p-logo"><A href="/Main_Page" title="Main Page">&quot;
	    </A></DIV><DIV id="menubar" class="left-menu noprint"><UL><LI class="selected"><A href="/Team:Heidelberg/Notebook/Methods">Page               </A></LI><LI class="new"><A href="/wiki/index.php?title=Talk:Team:Heidelberg/Notebook/Methods&amp;action=edit&amp;redlink=1">Discussion               </A></LI><LI><A href="/wiki/index.php?title=Team:Heidelberg/Notebook/Methods&amp;action=edit">View source               </A></LI><LI><A href="/wiki/index.php?title=Team:Heidelberg/Notebook/Methods&amp;action=history">History               </A></LI><LI style="color:#808080;cursor:default">teams</LI></UL></DIV><DIV class="right-menu noprint" id="menubar"><UL><LI id="pt-login"><A href="/wiki/index.php?title=Special:UserLogin&amp;returnto=Team:Heidelberg/Notebook/Methods" title="You are encouraged to log in; however, it is not mandatory [o]" accesskey="o">Log in</A></LI></UL></DIV><DIV id="search-controls" class="noprint"><FORM action="/Special:Search" id="searchform"> </FORM></DIV></DIV><DIV id="content"><H1 class="firstHeading">Team:Heidelberg/Notebook/Methods</H1><DIV id="bodyContent"><H3 id="siteSub" class="noprint">From 2010.igem.org</H3><DIV id="header"><DIV id="menu"><UL id="nav"><LI id="menu_home"><A class="aHome" href="https://2010.igem.org/Team:Heidelberg">Home</A></LI><LI id="menu_team"><A class="aTeam" href="https://2010.igem.org/Team:Heidelberg/Team">Team</A><TABLE class="menutable" cellpadding="0" cellspacing="0"><TBODY><TR><TD class="lefttdtopteam"><A class="aTeamOverview" href="https://2010.igem.org/Team:Heidelberg/Team">Overview</A><A class="aTeamMembers" href="https://2010.igem.org/Team:Heidelberg/Team/Members">Students</A><A class="aTeamMembers" href="https://2010.igem.org/Team:Heidelberg/Team/Members#Instructors">Instructors</A><A class="aTeamMembers" href="https://2010.igem.org/Team:Heidelberg/Team/Members#Advisors">Advisors</A><A class="aInstitutes" href="https://2010.igem.org/Team:Heidelberg/Team/Institutes">Institutes</A><A class="aGallery" href="https://2010.igem.org/Team:Heidelberg/Team/Gallery">Gallery</A><A class="aDance" href="https://2010.igem.org/Team:Heidelberg/midance">miDANCE</A><A class="amithanks" href="https://2010.igem.org/Team:Heidelberg/Team/miThanks">Acknowledgements</A></TD><TD class="righttdtop"> </TD></TR></TBODY></TABLE></LI><LI id="menu_project"><A class="aProject" href="https://2010.igem.org/Team:Heidelberg/Project">Project</A><TABLE class="menutable" cellpadding="0" cellspacing="0"><TBODY><TR><TD class="lefttdtopproject"><A class="aAbstract" href="https://2010.igem.org/Team:Heidelberg/Project">Abstract</A><A class="aIntroduction" href="https://2010.igem.org/Team:Heidelberg/Project/Introduction">Introduction</A><A class="aProject_miRNA_Kit" href="https://2010.igem.org/Team:Heidelberg/Project/miRNA_Kit">Synthetic miRNA Kit</A><A class="aPro_Capsid_Shuffling" href="https://2010.igem.org/Team:Heidelberg/Project/Capsid_Shuffling">Capsid Shuffling</A><A class="aPro_Homology" href="https://2010.igem.org/Team:Heidelberg/Project/Capsid_Shuffling/Homology_Based">    Homology Based</A><A class="aPro_Virobytes" href="https://2010.igem.org/Team:Heidelberg/Project/Capsid_Shuffling/ViroBytes">    ViroBytes</A><A class="aPro_miMeasure" href="https://2010.igem.org/Team:Heidelberg/Project/miMeasure">miMeasure</A><A class="aPro_Mouse" href="https://2010.igem.org/Team:Heidelberg/Project/Mouse_Infection"><I>in vivo</I> Study</A><A class="aPro_Summary" href="https://2010.igem.org/Team:Heidelberg/Project/Summary">Summary &amp; Outlook</A><A class="aPro_Ref" href="https://2010.igem.org/Team:Heidelberg/Project/References">References</A><A class="aPro_Ref" href="https://2010.igem.org/Team:Heidelberg/Project/Contributions">Contributions</A></TD><TD class="righttdtop"> </TD></TR></TBODY></TABLE></LI><LI id="menu_parts"><A class="aParts" href="https://2010.igem.org/Team:Heidelberg/Parts">Parts</A><TABLE class="menutable" cellpadding="0" cellspacing="0"><TBODY><TR><TD class="lefttdtop"><A class="aparts_ger" href="https://2010.igem.org/Team:Heidelberg/Parts">Parts</A><A class="aparts_char" href="https://2010.igem.org/Team:Heidelberg/Parts/Characterization">Characterization</A></TD><TD class="righttdtop"> </TD></TR></TBODY></TABLE></LI><LI id="menu_modeling"><A class="aModeling" href="https://2010.igem.org/Team:Heidelberg/Modeling">Modeling</A><TABLE class="menutable" cellpadding="0" cellspacing="0"><TBODY><TR><TD class="lefttdtop"><A class="amode_over" href="https://2010.igem.org/Team:Heidelberg/Modeling">Introduction</A><A class="amiGUI" href="https://2010.igem.org/Team:Heidelberg/Modeling/miGUI">miBEAT</A><A class="amirockdown" href="https://2010.igem.org/Team:Heidelberg/Modeling/miRockdown">miRockdown</A><A class="amibsdesigner" href="https://2010.igem.org/Team:Heidelberg/Modeling/miBSdesigner">miBS designer</A><A class="amispec" href="https://2010.igem.org/Team:Heidelberg/Modeling/miSpec">mUTING</A><A class="amodelset" href="https://2010.igem.org/Team:Heidelberg/Modeling/trainingset">Descriptions</A></TD><TD class="righttdtop"> </TD></TR></TBODY></TABLE></LI><LI id="menu_notebook"><A class="aNotebook" href="https://2010.igem.org/Team:Heidelberg/Notebook">Notebook</A><TABLE class="menutable" cellpadding="0" cellspacing="0"><TBODY><TR><TD class="lefttdtop"><A class="aNote_miRNA_kit" href="https://2010.igem.org/Team:Heidelberg/Notebook/miRNA_Kit">Synthetic miRNA Kit</A><A class="aNote_Capsid_Shuffling">Capsid Shuffling</A><A class="aNote_Homology" href="https://2010.igem.org/Team:Heidelberg/Notebook/Capsid_Shuffling/Homology_Based">    Homology Based</A><A class="aNote_Virobytes" href="https://2010.igem.org/Team:Heidelberg/Notebook/Capsid_Shuffling/ViroBytes">    ViroBytes</A><A class="aNote_miMeasure" href="https://2010.igem.org/Team:Heidelberg/Notebook/miMeasure">miMeasure</A><A class="aNote_BSdesign" href="https://2010.igem.org/Team:Heidelberg/Notebook/BSDesign">Binding Site Design </A><A class="aNote_Mouse" href="https://2010.igem.org/Team:Heidelberg/Notebook/Mouse_Infection"><I>in vivo</I> Study</A><A class="aNote_Modeling" href="https://2010.igem.org/Team:Heidelberg/Notebook/Modeling">Modeling</A><A class="aNote_Material" href="https://2010.igem.org/Team:Heidelberg/Notebook/Material">Materials</A><A class="aNote_Methods" href="https://2010.igem.org/Team:Heidelberg/Notebook/Methods">Methods</A><A class="aNote_Psycho" href="https://2010.igem.org/Team:Heidelberg/Human_Practices/Psychology/Notebook">Psychological Survey</A><A class="aNote_Philo" href="https://2010.igem.org/Team:Heidelberg/Human_Practices/Philosophy/Part2">Philosophical Reflection</A><A class="aNote_Biosafety" href="https://2010.igem.org/Team:Heidelberg/Notebook/Biosafety">Biosafety</A></TD><TD class="righttdtop"> </TD></TR></TBODY></TABLE></LI><LI id="menu_human_practices"><A class="aHuman_Practices" href="https://2010.igem.org/Team:Heidelberg/Human Practices">Human Practices</A><TABLE class="menutable" cellpadding="0" cellspacing="0"><TBODY><TR><TD class="lefttdtop"><A class="aPsycho" href="https://2010.igem.org/Team:Heidelberg/Human Practices/Psychology">Psychological Survey</A><A class="aPhilo" href="https://2010.igem.org/Team:Heidelberg/Human Practices/Philosophy">Philosophical Reflection</A></TD><TD class="righttdtop"> </TD></TR></TBODY></TABLE></LI><LI id="menu_support"><A class="aSupport" href="https://2010.igem.org/Team:Heidelberg/Support">Sponsors</A></LI></UL></DIV></DIV><TABLE id="toc" class="toc"><TBODY><TR><TD><DIV id="toctitle"><H2>Contents</H2></DIV><UL><LI class="toclevel-1 tocsection-1"><A href="#Methods"><SPAN class="tocnumber">1</SPAN><SPAN class="toctext">Methods</SPAN></A></LI><LI class="toclevel-2 tocsection-2"><A href="#Cloning"><SPAN class="tocnumber">1.1</SPAN><SPAN class="toctext">Cloning</SPAN></A></LI><LI class="toclevel-3 tocsection-3"><A href="#Agarose_Gel_Electrophoresis"><SPAN class="tocnumber">1.1.1</SPAN><SPAN class="toctext">Agarose Gel Electrophoresis</SPAN></A></LI><LI class="toclevel-3 tocsection-4"><A href="#Colony_PCR"><SPAN class="tocnumber">1.1.2</SPAN><SPAN class="toctext">Colony PCR</SPAN></A></LI><LI class="toclevel-3 tocsection-5"><A href="#Plasmid-PCR"><SPAN class="tocnumber">1.1.3</SPAN><SPAN class="toctext">Plasmid-PCR</SPAN></A></LI><LI class="toclevel-3 tocsection-6"><A href="#shRNA-PCR_protocol"><SPAN class="tocnumber">1.1.4</SPAN><SPAN class="toctext">shRNA-PCR protocol</SPAN></A></LI><LI class="toclevel-3 tocsection-7"><A href="#Gel_extraction"><SPAN class="tocnumber">1.1.5</SPAN><SPAN class="toctext">Gel extraction</SPAN></A></LI><LI class="toclevel-3 tocsection-8"><A href="#Large_scale_preparation_of_plasmid_DNA"><SPAN class="tocnumber">1.1.6</SPAN><SPAN class="toctext">Large scale preparation of plasmid DNA</SPAN></A></LI><LI class="toclevel-3 tocsection-9"><A href="#Plasmid-DNA_isolation"><SPAN class="tocnumber">1.1.7</SPAN><SPAN class="toctext">Plasmid-DNA isolation</SPAN></A></LI><LI class="toclevel-3 tocsection-10"><A href="#Preparation_of_competent_E._coli_Top10_and_DH5alpha"><SPAN class="tocnumber">1.1.8</SPAN><SPAN class="toctext">Preparation of competent E. coli Top10 and DH5alpha</SPAN></A></LI><LI class="toclevel-3 tocsection-11"><A href="#Purification_of_PCR_product"><SPAN class="tocnumber">1.1.9</SPAN><SPAN class="toctext">Purification of PCR product</SPAN></A></LI><LI class="toclevel-3 tocsection-12"><A href="#Sequencing"><SPAN class="tocnumber">1.1.10</SPAN><SPAN class="toctext">Sequencing</SPAN></A></LI></UL><LI class="toclevel-2 tocsection-13"><A href="#random_assembly_PCR_.28raPCR.29"><SPAN class="tocnumber">1.2</SPAN><SPAN class="toctext">random assembly PCR (raPCR)</SPAN></A></LI><LI class="toclevel-3 tocsection-14"><A href="#Designing_Oligos"><SPAN class="tocnumber">1.2.1</SPAN><SPAN class="toctext">Designing Oligos</SPAN></A></LI><LI class="toclevel-3 tocsection-15"><A href="#running_the_PCR"><SPAN class="tocnumber">1.2.2</SPAN><SPAN class="toctext">running the PCR</SPAN></A></LI><LI class="toclevel-2 tocsection-16"><A href="#Cell_culture"><SPAN class="tocnumber">1.3</SPAN><SPAN class="toctext">Cell culture</SPAN></A></LI><LI class="toclevel-3 tocsection-17"><A href="#Media"><SPAN class="tocnumber">1.3.1</SPAN><SPAN class="toctext">Media</SPAN></A></LI><LI class="toclevel-3 tocsection-18"><A href="#Passaging"><SPAN class="tocnumber">1.3.2</SPAN><SPAN class="toctext">Passaging</SPAN></A></LI><LI class="toclevel-3 tocsection-19"><A href="#Thawing"><SPAN class="tocnumber">1.3.3</SPAN><SPAN class="toctext">Thawing</SPAN></A></LI><LI class="toclevel-3 tocsection-20"><A href="#Coating"><SPAN class="tocnumber">1.3.4</SPAN><SPAN class="toctext">Coating</SPAN></A></LI><LI class="toclevel-3 tocsection-21"><A href="#Transfection"><SPAN class="tocnumber">1.3.5</SPAN><SPAN class="toctext">Transfection</SPAN></A></LI><LI class="toclevel-4 tocsection-22"><A href="#FuGENE"><SPAN class="tocnumber">1.3.5.1</SPAN><SPAN class="toctext">FuGENE</SPAN></A></LI><LI class="toclevel-4 tocsection-23"><A href="#PEI"><SPAN class="tocnumber">1.3.5.2</SPAN><SPAN class="toctext">PEI</SPAN></A></LI><LI class="toclevel-4 tocsection-24"><A href="#HBSS"><SPAN class="tocnumber">1.3.5.3</SPAN><SPAN class="toctext">HBSS</SPAN></A></LI><LI class="toclevel-2 tocsection-25"><A href="#Virus_Production"><SPAN class="tocnumber">1.4</SPAN><SPAN class="toctext">Virus Production</SPAN></A></LI><LI class="toclevel-3 tocsection-26"><A href="#Quantitative_Realtime_PCR"><SPAN class="tocnumber">1.4.1</SPAN><SPAN class="toctext">Quantitative Realtime PCR</SPAN></A></LI><LI class="toclevel-2 tocsection-27"><A href="#Measurements"><SPAN class="tocnumber">1.5</SPAN><SPAN class="toctext">Measurements</SPAN></A></LI><LI class="toclevel-3 tocsection-28"><A href="#ELISA"><SPAN class="tocnumber">1.5.1</SPAN><SPAN class="toctext">ELISA</SPAN></A></LI><LI class="toclevel-4 tocsection-29"><A href="#Procedure"><SPAN class="tocnumber">1.5.1.1</SPAN><SPAN class="toctext">Procedure</SPAN></A></LI><LI class="toclevel-4 tocsection-30"><A href="#Buffers"><SPAN class="tocnumber">1.5.1.2</SPAN><SPAN class="toctext">Buffers</SPAN></A></LI><LI class="toclevel-3 tocsection-31"><A href="#Microscopy"><SPAN class="tocnumber">1.5.2</SPAN><SPAN class="toctext">Microscopy</SPAN></A></LI><LI class="toclevel-3 tocsection-32"><A href="#Flow_cytometry"><SPAN class="tocnumber">1.5.3</SPAN><SPAN class="toctext">Flow cytometry</SPAN></A></LI><LI class="toclevel-3 tocsection-33"><A href="#Dual_Luciferase_Assay"><SPAN class="tocnumber">1.5.4</SPAN><SPAN class="toctext">Dual Luciferase Assay</SPAN></A></LI><LI class="toclevel-4 tocsection-34"><A href="#Order_of_recording_and_processing_raw_data"><SPAN class="tocnumber">1.5.4.1</SPAN><SPAN class="toctext">Order of recording and processing raw data</SPAN></A></LI><LI class="toclevel-5 tocsection-35"><A href="#nomenclature"><SPAN class="tocnumber">1.5.4.1.1</SPAN><SPAN class="toctext">nomenclature</SPAN></A></LI><LI class="toclevel-4 tocsection-36"><A href="#Consumables_and_Reagents"><SPAN class="tocnumber">1.5.4.2</SPAN><SPAN class="toctext">Consumables and Reagents</SPAN></A></LI><LI class="toclevel-4 tocsection-37"><A href="#Instruments"><SPAN class="tocnumber">1.5.4.3</SPAN><SPAN class="toctext">Instruments</SPAN></A></LI></TD></TR></TBODY></TABLE><P> </P></DIV><H1><SPAN class="mw-headline" id="Methods">Methods</SPAN></H1><H2><SPAN class="mw-headline" id="Cloning">Cloning</SPAN></H2><H3><SPAN class="mw-headline" id="Agarose_Gel_Electrophoresis">Agarose Gel Electrophoresis</SPAN></H3><P>Agarose flat-bed gels in various concentrations (0.6–2% agarose in 0.5 x TAE buffer) and sizes were run to separate DNA fragments in an electrical field (10–20 V/cm) for analytical or preparative use. The desired amount of agarose was boiled in 1 x TAE buffer until it was completely dissolved. After it cooled down to approximately 60°C, ethidium bromide (EtBr) solution (0.5 µg/ml final concentration) was added to the liquid agar, which was then poured in a flat-bed tray with combs. As soon as the agarose solidified, the Running buffer (0.5 x TAE buffer) was added before the DNA in the loading buffer was loaded into the wells and separated electrophoretically. Ethidium bromide intercalates with the DNA’s GC ntss resulting in DNA-EtBr-complex that emits visible light. Therefore, the DNA fragments could be detected on a UV-light tray at 265 nm.
</P><H3><SPAN class="mw-headline" id="Colony_PCR">Colony PCR</SPAN></H3><P>Colony PCRs were performed using Fermentas PCR Master Mix (2x), containing <I>Taq</I> DNA Polymerase.
Bacterial colonies were picked form an LB/Agar-plate and either resuspended in PCR reaction mix and subsequently used to inoculate LB-medium with appropriate antibiotic, or resuspended in water. The water was then taken as &quot;template&quot; for PCR and inoculation of Minepreps.
The PCR was performed using 20 pmol of forward and reverse primer (= 0.2 µL of 100µM) in a total volume of 20 µL.
PCR conditions were set to recommendations made by Fermentas:
</P><TABLE border="1" cellpadding="3" style="border:solid 1px #AAAAAA; border-collapse:collapse; background-color:#F9F9F9; empty-cells:show;"><TBODY><TR><TH>Temp </TH><TH>  Time </TH></TR><TR><TD>95°</TD><TD>03:00</TD></TR><TR><TD style="border-top:solid 1px #000000;">94°</TD><TD style="border-top:solid 1px #000000;border-right:solid 1px #000000;">00:30</TD><TD rowspan="3">x 30 cycles
</TD></TR><TR><TD>45°<B>*</B></TD><TD style="border-right:solid 1px #000000;">00:30
</TD></TR><TR><TD style="border-bottom:solid 1px #000000;">72°</TD><TD style="border-bottom:solid 1px #000000;border-right:solid 1px #000000;">01:00 /kb
</TD></TR><TR><TD>72°</TD><TD>10:00</TD></TR><TR><TD>4°</TD><TD>8</TD></TR></TBODY></TABLE><P><B>*</B>with 45°C at a minimum of primer annealing temp
</P><H3><SPAN class="mw-headline" id="Plasmid-PCR">Plasmid-PCR</SPAN></H3><TABLE border="1" cellpadding="3" style="border:solid 1px #AAAAAA; border-collapse:collapse; background-color:#F9F9F9; empty-cells:show;"><TBODY><TR><TH>Temp </TH><TH>  Time </TH></TR><TR><TD>98°</TD><TD>05:00</TD></TR><TR><TD style="border-top:solid 1px #000000;">98°</TD><TD style="border-top:solid 1px #000000;border-right:solid 1px #000000;">00:10</TD><TD rowspan="3">x 35 cycles
</TD></TR><TR><TD>60°</TD><TD style="border-right:solid 1px #000000;">00:30
</TD></TR><TR><TD style="border-bottom:solid 1px #000000;">72°</TD><TD style="border-bottom:solid 1px #000000;border-right:solid 1px #000000;">02:30</TD></TR><TR><TD>72°</TD><TD>10:00</TD></TR></TBODY></TABLE><H3><SPAN class="mw-headline" id="shRNA-PCR_protocol">shRNA-PCR protocol</SPAN></H3><TABLE border="1" cellpadding="3" style="border:solid 1px #AAAAAA; border-collapse:collapse; background-color:#F9F9F9; empty-cells:show;"><TBODY><TR><TH>Temp </TH><TH>  Time </TH></TR><TR><TD>98°</TD><TD>00:30</TD></TR><TR><TD style="border-top:solid 1px #000000;">98°</TD><TD style="border-top:solid 1px #000000;border-right:solid 1px #000000;">00:10</TD><TD rowspan="3">x 35 cycles
</TD></TR><TR><TD>60°</TD><TD style="border-right:solid 1px #000000;">00:30
</TD></TR><TR><TD style="border-bottom:solid 1px #000000;">72°</TD><TD style="border-bottom:solid 1px #000000;border-right:solid 1px #000000;">00:15
</TD></TR><TR><TD>72°</TD><TD>1:00</TD></TR><TR><TD>4°</TD><TD>forever</TD></TR></TBODY></TABLE><H3><SPAN class="mw-headline" id="Gel_extraction">Gel extraction</SPAN></H3><P>After gel electrophoresis the digested vector and insert have to be purified from the gel. With the help of a UV lamp, the bands were quickly excised from the gel without exposing the DNA too long to UV light. Afterwards the DNA was purified with the QIAquick Gel extraction kit. Three volumes of buffer QG were added to one volume of gel. The gel fragment was dissolved by incubation for 10 min at 50°C. Afterwards one volume of 100% isopropanol was added. The solution was applied on a QIAquick spin column after this has been placed into a provided 2 ml collection tube. By centrifugation for 1 min at 13.000 rpm the DNA was bound to the column. The flow-through was discarded and the column was placed in the same collection tube. To remove all traces of agarose from the column, 500 µl of wash buffer QC was added followed by centrifugation for 1 min at 13.000 rpm. The flow-through was discarded and the column was washed with 750 µl of buffer PE for 1 min at 13.000 rpm. Afterwards the flow-through was discarded. An additional centrifugation for 1 min at 13.000 rpm helped to remove the residual ethanol. The column was placed into a new 1.5 ml microcentrifuge tube and it was eluted with 30 µl of ddH2O.
</P><H3><SPAN class="mw-headline" id="Large_scale_preparation_of_plasmid_DNA">Large scale preparation of plasmid DNA</SPAN></H3><P>150 ml LB-Medium with 150 µl ampicillin was inoculated with 50 µl of bacteria culture which grew overnight on a shaker at 37°C. The plasmid DNA was isolated using QiAprep Spin MAxiprep kit from Qiagen and the protocol was followed. The overnight culture was centrifuged for 20 min at 4000 rpm at 4°C using an SLA 1500 Rotor. Afterwards the LB-medium was discarded and the pellet was homogeneously resuspended in 10 ml of precooled Buffer P1. After having added 10 ml of Buffer P2 the mixture was inverted 4-6 times and incubated for 5 min at RT before adding 10 ml of chilled Buffer P3. Thereafter the lysate was poured into a prepared QIAfilter Maxi Cartridge and incubated at RT for 10 min. During this time a QIAGEN-tip 500 was equilibrated by applying 10 ml of Buffer QBT and allowing the column to empty by gravity flow. The cell lysate was filtered into the QIAGEN-tip. The cleared lysate entered the resin by gravity flow and after washing with 2 x 30 ml Buffer QC the Plasmid DNA was eluted with 15 ml Buffer QF. After this the DNA was precipitated by adding 10.5 ml isopropanol and centrifuged at 4,000 rpm for 45 min at 4°C. The supernatant was discarded and the DNA pellet was washed with 5 ml ethanol (70%) and centrifuged at 4,000 rpm for 15 min. After air-drying the pellet the DNA was redissolved in H2O. </P><H3><SPAN class="mw-headline" id="Plasmid-DNA_isolation">Plasmid-DNA isolation</SPAN></H3><P>5 ml LB-Medium with 5 µl ampicillin was inoculated with single colonies which grew overnight on a shaker at 37°C. The plasmid DNA was isolated using QiAprep Spin Miniprep kit from Qiagen and following the manufacturer’s protocol. 4 ml of each overnight culture was pelleted in 2 ml microcentrifuge tubes during two steps of centrifugation at 13.000 rpm. Subsequently the pellet was resuspended in 250 µl of chilled buffer P1. 250 µl of lysis buffer P2 was added and the solution was mixed thoroughly by inverting the tube 4-6 times. After adding 350 µl of the neutralization Buffer N3 the solution was mixed immediately and thoroughly by inverting the tube 4-6 times. Thereafter the mixture was centrifuged. The supernatants were applied to a QIAprep column which was put in a 2 ml collection tube. It was centrifuged for 1 min at 13.000 rpm and the flow-through was discarded. After adding 500 µl of wash buffer PB, it was centrifuged for 1 min at 13.000 rpm and the flow-through was discarded. Once more, it was washed with 750 µl of wash buffer PE. In an additional centrifugation for 1 min at 13.000 rpm the residual wash buffer was removed. The QIAprep column was placed into a clean 1.5 ml microcentrifuge tube and the plasmid DNA was eluted in 30 µl ddH2O.
</P><H3><SPAN class="mw-headline" id="Preparation_of_competent_E._coli_Top10_and_DH5alpha">Preparation of competent E. coli Top10 and DH5alpha</SPAN></H3><P>Plating of E. coli Top10 and DH5alpha on a agar plate (LB, without Amp); Preperation of competent cells according to the following protocol:
</P><P>First, a 20 ml over night culture was inoculated in antibiotic free LB medium from a fresh single colony and transferred into 400 ml antibiotic free LB  medium the next day. This culture was incubated at 37 °C while shacking until an OD600 of 0.5 – 0.6 was achieved. The culture was than cooled down on ice, centrifuged (8 min, 4 °C, 3500 rpm), the supernatant discarded and the pellet resuspended in 10 ml 100 mM CaCl2. After addition of further 190 ml 100 mM CaCl2 the suspension was incubated on ice for 30 min. The suspension was than again centrifuged (8 min, 4 °C, 3500 rpm), the supernatant discarded, the pellet resuspended in 20 ml 82.5 mM CaCl2 with 17.5 % glycerol and aliquoted. The aliquots were flash frozen in liquid nitrogen and than stored at -80 °C until usage.
</P><H3><SPAN class="mw-headline" id="Purification_of_PCR_product">Purification of PCR product</SPAN></H3><P>One volume of buffer PBI was added to one volume of the PCR sample mix. The sample was applied to a QIAquick column which has been placed into a provided 2 ml collection tube. It was centrifuged for 1 min at 13.000 rpm and the flow-through was discarded and the column was placed in the same collection tube. After this 750 µl of buffer PE was added to wash the column. It was centrifuged for 1 min at 13.000 rpm. The flow-through was discarded and the column was placed in the same collection tube. It was centrifuged for 1 min at 13.000 rpm. Afterwards the QIAquick column was placed into a new 1.5 ml microcentrifuge tube and it was eluted with 40 µl ddH2O.
</P><H3><SPAN class="mw-headline" id="Sequencing">Sequencing</SPAN></H3><P>Performed by the GATC Biotech company

</P><DIV class="backtop"><A href="#top">↑</A></DIV><H2><SPAN class="mw-headline" id="random_assembly_PCR_.28raPCR.29"> random assembly PCR (raPCR) </SPAN></H2><H4><SPAN class="mw-headline" id="Designing_Oligos"> Designing Oligos </SPAN></H4><P>To design Oligos for raPCR, first you need to have your miRNA sequence. This you can find on databases such as  <A href="http://www.microrna.org" class="external text" rel="nofollow">microRNA.org</A> or <A href="http://mirbase.org" class="external text" rel="nofollow">mirbase.org</A>. Then you need to have an innert spacer sequence to seperate the binding sites from each other.
Like we did, you can use following tools to create your own spacer sequence:
</P><DL><DD><A href="http://cbit.snu.ac.kr/~miTarget/" class="external free" rel="nofollow">http://cbit.snu.ac.kr/~miTarget/</A></DD><DD><A href="http://cbcsrv.watson.ibm.com/rna22.html" class="external free" rel="nofollow">http://cbcsrv.watson.ibm.com/rna22.html</A></DD><DD><A href="http://bibiserv.techfak.uni-bielefeld.de/rnahybrid/submission.html" class="external free" rel="nofollow">http://bibiserv.techfak.uni-bielefeld.de/rnahybrid/submission.html</A></DD></DL><P>Or you choose the one we used: <SPAN style="color:#009be1">GCATACATGGACTGC</SPAN><SPAN style="color:#78b41e">CACTGAATCCAACTG</SPAN></P><P>Then the spacer need to be added to the miRNA binding site you want to use. In our approach, the spacer is also used for annealing of the different oligos. Therefore we suggest to split it in two parts. The first half (green) is then added to the 5' end and the second half (blue) is added to the 3' end of the miRBS sequence.
</P><P>For running the PCR you need to create annealing oligos, which could either be the spacer itself (reverse complement of firsthalf elongated with reverse complement of second half).
</P><P>Last but not least you need to have stop oligos. Stop oligos are characterised by having just one annealing sequence so, either first half or second half. Like for normal PCR Primers one annealing sequence should be reverse complemented, so they can also act effeciently as PCR primers. Additionally you can add whatever you want (i.e. the BBb-Standard sites). Those primers you can find in our <A href="https://2010.igem.org/Team:Heidelberg/Notebook/Material/Primer#Primer_Table_raPCR" class="external text" rel="nofollow">primer list</A>.
</P><H4><SPAN class="mw-headline" id="running_the_PCR"> running the PCR </SPAN></H4><P>
We developed a standardized PCR based method we use for the creation of our microRNA binding site patterns. It consists of two subsequent PCR steps: a 12 cycle PCR followed by a 25 cycle PCR. Therefor, order the oligos containing binding sites for the different microRNAs of your choice (refer to step 1). Furthermor, order the following diraPCR stop oligos:
</P><DL><DD><UL><LI>miraPCR_Stop(fw)_EcoRI: 5'-TTTGTCgaattcGGTAGCTATTTCTC-3'
</LI><LI>miraPCR_Stop(rev)_PstI: 5'-TTTGTCctgcagGAGAAATAGCTACC-3'
</LI></UL></DD></DL><UL><LI>Dilute all oligos (including the stop oligos) to a final concentration of 100 um.
</LI><LI>set up a PCR-reaction according to the following protocol. You can direct the length of your plasmid by applying more stop oligos (= shorter PCR fragments, that means patterns with lower amount of miR binding sites) and different amounts of binding site oligos.
</LI></UL><P><B> 12-cycle PCR </B></P><UL><LI>set up a PCR reaction according to the following protocol
</LI></UL><DL><DD><UL><LI>0.1-4 µl of Oligos (recommended: for binding site patterns of the same miRNA, use 3 µL)
</LI><LI>0-0.5 µLof each stop oligo (recommended: for shorter fragments use 0.5 µL)
</LI><LI>1 µL of spacer oligo
</LI><LI>25 µl of Phusion PCR MasterMix
</LI><LI>add water to a final volume of 50 µl
</LI></UL></DD></DL><UL><LI> run 12-cycle PCR according to the following program
</LI></UL><TABLE border="1" cellpadding="3" style="border:solid 1px #AAAAAA; border-collapse:collapse; background-color:#F9F9F9; empty-cells:show;"><TBODY><TR><TH>Temp </TH><TH>  Time </TH></TR><TR><TD>95°</TD><TD>05:00</TD></TR><TR><TD style="border-top:solid 1px #000000;">95°</TD><TD style="border-top:solid 1px #000000;border-right:solid 1px #000000;">00:30</TD><TD rowspan="3">x 12 cycles
</TD></TR><TR><TD>57°</TD><TD style="border-right:solid 1px #000000;">00:45
</TD></TR><TR><TD style="border-bottom:solid 1px #000000;">72°</TD><TD style="border-bottom:solid 1px #000000;border-right:solid 1px #000000;">00:45
</TD></TR><TR><TD>4°</TD><TD>forever</TD></TR></TBODY></TABLE><UL><LI>PCR purify all products applying a PCR purification Kit (Qiagen). We recommend elution in 30 µL for higher DNA concentration.
</LI></UL><P><B>25-cycle PCR</B></P><UL><LI> run 25-cycle PCR in order to amplify constructs; therefor set up a PCR reaction according to the following protocol
</LI></UL><DL><DD><UL><LI>1-20 µl of purified PCR product (for shorter patterns (100-500 bp, 20 ul are recommended)
</LI><LI>0.5-1 µl of each stop oligo 
</LI><LI>25 µl of Phusion PCR MasterMix (2x)
</LI><LI>add water to final volume of 50 µl
</LI></UL></DD></DL><UL><LI>run 25-cycle PCR according to the following protocol
</LI></UL><TABLE border="1" cellpadding="3" style="border:solid 1px #AAAAAA; border-collapse:collapse; background-color:#F9F9F9; empty-cells:show;"><TBODY><TR><TH>Temp </TH><TH>  Time </TH></TR><TR><TD>95°</TD><TD>05:00</TD></TR><TR><TD style="border-top:solid 1px #000000;">95°</TD><TD style="border-top:solid 1px #000000;border-right:solid 1px #000000;">00:30</TD><TD rowspan="3">x 25 cycles
</TD></TR><TR><TD>65°</TD><TD style="border-right:solid 1px #000000;">00:45
</TD></TR><TR><TD style="border-bottom:solid 1px #000000;">72°</TD><TD style="border-bottom:solid 1px #000000;border-right:solid 1px #000000;">00:50
</TD></TR><TR><TD>72°</TD><TD>05:00</TD></TR><TR><TD>4°</TD><TD>forever</TD></TR></TBODY></TABLE><H2><SPAN class="mw-headline" id="Cell_culture">Cell culture</SPAN></H2><H3><SPAN class="mw-headline" id="Media">Media</SPAN></H3><UL><LI>HeLa - DMEM, 10% Fetal Bovine Serum, 100 unit Penicillin, 100 µg/ml Streptomycin, 2mM L-Glutamin
</LI><LI>HEK293T - DMEM, 10% Fetal Bovine Serum, 100 unit Penicillin, 100 µg/ml Streptomycin, 2mM L-Glutamin
</LI><LI>HEK T-REx - DMEM, 10% Fetal Bovine Serum, 100 unit Penicillin, 100 µg/ml Streptomycin, 2mM L-Glutamin, 50µg/ml Blasticidin, 50µg/ml Zeocin
</LI><LI>HEK T-REx (with pcDNA5 integrated) - DMEM, 10% Fetal Bovine Serum, 100 unit Penicillin, 100 µg/ml Streptomycin, 2mM L-Glutamin, 50µg/ml Blasticidin, 100µg/ml Hygromycin
</LI><LI>Huh7 - DMEM, 10% Fetal Bovine Serum, 100 unit Penicillin, 100 µg/ml Streptomycin, 2mM L-Glutamin, 1% Non-essential amino acids
</LI><LI>HepG2 - DMEM, 10% Fetal Bovine Serum, 100 unit Penicillin, 100 µg/ml Streptomycin, 2mM L-Glutamin
</LI><LI>Hepa1.6 - DMEM, 10% Fetal Bovine Serum, 100 unit Penicillin, 100 µg/ml Streptomycin, 2mM L-Glutamin
</LI><LI>primary hepatocytes - William's Medium E, 10% Fetal Bovine Serum, 100 unit Penicillin, 100 µg/ml Streptomycin, 2mM L-Glutamin, 100nM Dexamethasone
</LI></UL><H3><SPAN class="mw-headline" id="Passaging">Passaging</SPAN></H3><UL><LI>remove media
</LI><LI>wash cells one time in PBS (10ml)
</LI><LI>remove PBS; add 2 ml of trypsin-EDTA solution and incubate cells for 5 min at 37 °C
</LI><LI>add 5 ml of the according media
</LI><LI>take 1/10th of the cell suspension and plate out on the according dish (either p100 dish, 6 well plate or T-flask)
</LI></UL><H3><SPAN class="mw-headline" id="Thawing">Thawing</SPAN></H3><P>Thawing of Huh-7, HeLa p4 and HEK-293 cells according to the following protocol
vials from liquid nitrogen were thawn at 37 °C
once, the probe was nearly completely thawn, cells were thrown into pre-warmed DMEM (10 % FCS, L-Glut, P/S) and gentely mixed
cells were spinned down at 800 rpm, 3 min; supernatant was discarded
the pellett was resuspended in 10 ml DMEM an plated on a p100 cell culture dish in the following media according to the different cell lines:
</P><H3><SPAN class="mw-headline" id="Coating">Coating</SPAN></H3><P>For 96-well plates:
</P><UL><LI>add 30 µl of poly-L-lysine solution (Sigma) to each well (make sure whole surface is covered with solution)
</LI><LI>leave for 30' in the incubator
</LI><LI>remove poly-L-lysine solution
</LI><LI>wash once with 1x PBS
</LI></UL><H3><SPAN class="mw-headline" id="Transfection">Transfection</SPAN></H3><H4><SPAN class="mw-headline" id="FuGENE">FuGENE</SPAN></H4><P>DAY1
</P><UL><LI>seed cells
</LI></UL><P>DAY2 
</P><UL><LI>transfection
</LI><LI>to media (OptiMEM, no FBS) add <A href="http://www.roche-applied-science.com/pack-insert/1815091a.pdf" class="external text" rel="nofollow">FuGENE 6 reagent</A> (it is important to add FuGENE to media, not other way round and not to drop FuGENE on the walls of the tube)
</LI><LI>incubate 5 minutes at room temperature
</LI><LI>add DNA
</LI><LI>incubate 15 minutes
</LI><LI>mix well and add dropwise to the well
</LI></UL><TABLE class="wikitable sortable" border="0" style="text-align: center"><TBODY><TR bgcolor="#cccccc"><TH>culture format</TH><TH>number of cells seeded</TH><TH>volume of culture media</TH><TH>amount of DNA</TH><TH>volume of FuGENE</TH><TH>volume of media for FuGENE activation
</TH></TR><TR><TD>96-well</TD><TD>5 000</TD><TD>100µl</TD><TD>50ng</TD><TD>0.3µl</TD><TD>5µl
</TD></TR><TR><TD>24-well</TD><TD>30 000</TD><TD>500µl</TD><TD>200ng</TD><TD>0.6µl</TD><TD>20µl
</TD></TR><TR><TD>6-well</TD><TD>100 000</TD><TD>2500µl</TD><TD>1000ng</TD><TD>3µl</TD><TD>100µl
</TD></TR><TR><TD>labtec chamber</TD><TD>10 000</TD><TD>400µl</TD><TD>200ng</TD><TD>0.4µl</TD><TD>20µl
</TD></TR></TBODY></TABLE><H4><SPAN class="mw-headline" id="PEI">PEI</SPAN></H4><P>150cm<SUP>2</SUP> plate format
</P><UL><LI>mixture 1:
</LI><LI>44ug DNA
</LI><LI>up to 790µl of H2O each flask
</LI><LI>add 790µl of 300mM NaCl
</LI></UL><LI>mixture 2:
</LI><LI>343µl of PEI
</LI><LI>790µl of 300mM NaCl
</LI><LI>add mixture 2 to mixture 1
</LI><LI>incubate 10 min at RT
</LI><LI>add to cells
</LI><H4><SPAN class="mw-headline" id="HBSS">HBSS</SPAN></H4><P>150cm<SUP>2</SUP> flask format
</P><UL><LI>50ugDNA
</LI><LI>2.67ml 250mM CaCl<SUB>2</SUB></LI><LI>2.67ml 2x HBSS
</LI></UL><UL><LI>warm up both CaCl<SUB>2</SUB> and HBSS
</LI><LI>add plasmid to CaCl<SUB>2</SUB> solution
</LI><LI>pour 2x HBSS to erlenmeyer flask
</LI><LI>very slowly add CaCl<SUB>2</SUB> to HBSS, mix vigorously during adding (should become turbid)
</LI><LI>incubate for one minute in the room temperature
</LI><LI>add transfection mix to media (1 volume of mix, 2 volumes of media)
</LI><LI>pour on cells
</LI></UL><DIV class="backtop"><A href="#top">↑</A></DIV><H2><SPAN class="mw-headline" id="Virus_Production"> Virus Production </SPAN></H2><DL><DD><B>Seeding</B></DD></DL><P>Virus production was done in 150cm<SUP>2</SUP> flasks. 0.9 million cells were seeded in 30ml medium per flask. Transfections were done as described <A href="https://2010.igem.org/Team:Heidelberg/Notebook/Methods#Transfection" class="external text" rel="nofollow">above</A>. 
</P><DL><DD><B>Harvesting</B></DD></DL><P>To harvest the cells, cell suspension was decanted into 500ml corning conical centrifuge tubes. Remaining cells were washed with 19ml PBS and also transferred into the tubes. Cells were collected by centrifugation at 1500rpm for 15min at 4°C. Supernatant was aspirated, cells resuspended in 10ml 1xPBS and transferred into a 50ml blue-cap vial. Centrifugaion as above, aspiration of supernatant. Pellet was then resuspended in 5ml virus lysate solution and frozen at -196°C in liquid nitrogen for 5min, then thawed at 37°C. This freeze/thaw cycle was repeated five times.
</P><DL><DD><B>Purification</B></DD></DL><P>Each sample was sonicated in a sonication bath for 1min 20s, 50µg/ml benzonase was added. Samples were kept at 37°C for 30min and vortexed every 10min. Centrifugation for 15min at 3270g, 4°C. Supernatant was transferred in to a new blue cap vial with a 5ml pipette. Gradiant was poured in a Beckman Quick-Seal centrifuge tube and a pasteur pipette plugged into the tube. 5ml of the virus suspension was transferred through the Pasteur pipette into the tube. 1.5ml 15% Iodixanolsolution (in PBS-MK) was poured through the Pasteur pipette in a way thet it presses put the virus suspension. In the same manner, 1.5ml 25% Iodixanolsolution was pipette to become the lower phase, then 1.5ml 40% Iodixanolsolution and finally 1.5ml 60% Iodixanolsolution. The Pasteur pipette was reomeved and the tubes sealed with tube sealer. Ultracentrifulation for 2h at 50.000rpm, 4°C and 71,1 Ti. The virus was recovered from the 40% Iodixanol phase.
</P><P>Seeding medium: DMEM high glucose, 10% FCS, 1% P/S, 1% L-Glutamin
Virus lysate solution
PBS-MK: 1xPBS, 1mM MgCl<SUB>2</SUB>, 2.5mM KCl
Iodixanolsolution
</P><H3><SPAN class="mw-headline" id="Quantitative_Realtime_PCR"> Quantitative Realtime PCR </SPAN></H3><P>To quantitavely measure the amount of AAV rep gene, viruses had to be lysed. First, 10µl TE buffer was mixed with 10µl AAV solution. 20µl 2M NaOH were added and incubated for 30min at 56°C to lyse the viruses. Then 38µl 1M HCl and 922µl H<SUB>2</SUB>O were added. For a negative control, the same procedure was done with 10µl H<SUB>2</SUB>O instead of virus. 
</P><P>For each probe, a 35µl mix of RNase free water, 1x SensiMix II Probe PCR Master Mix, 100pmol/µl forward and reverse primer and 100pm/µl probe was prepared in RNase free water. This was enough for measurement triplicates. To make a RT-PCR standard curve, the standard probes were diluted to concentrations of 3.5x10<SUP>11</SUP> to 3.5x10<SUP>3</SUP> molecules in 10µl. 
</P><P>Thermocycler program was 10min at 95°C initial activation, followed by 40 cycles of 10s at 95°C and 20s at 60°C. 
</P><P>To find out the concentration of viral genomes per ml (<FONT face="Symbol">Y</FONT>) the value of the <FONT face="Symbol">X</FONT> position on the standard curve has to be multiplied as follows: <FONT face="Symbol">Y</FONT> = <FONT face="Symbol">X</FONT>×10²×10²</P><DIV class="backtop"><A href="#top">↑</A></DIV><H2><SPAN class="mw-headline" id="Measurements"> Measurements </SPAN></H2><H3><SPAN class="mw-headline" id="ELISA"> ELISA </SPAN></H3><H4><SPAN class="mw-headline" id="Procedure"> Procedure </SPAN></H4><OL><LI> Coating
</LI><LI> dilute 1st antibody 1:1000 in coating buffer
</LI><LI> add 50 µl per well
</LI><LI> incubate over night at 4°C
</LI><LI> Blocking
</LI><LI> wash three times with 300 µl wash buffer per well
</LI><LI> add 200 µl blocking buffer per well
</LI><LI> incubate more than 1 hour at room temperature
</LI><LI> alternative: over-night at 4°C or freezing at -20°C
</LI><LI> Sample Loading
</LI><LI> 50 µl per well
</LI><LI> incubate 2 hours at 37°C
</LI><LI> wash five times
</LI><LI> Detection
</LI><LI> dilute 2nd antibody 1:1000 in dilution buffer
</LI><LI> add 100 µl per well
</LI><LI> incubate 1 hour at 37°C
</LI><LI> wash five times
</LI><LI> add 100 µl substrate per well
</LI><LI> incubate at room temperature for 5 - 10 min
</LI><LI> stop with 100µl 2M H<SUB>2</SUB>SO<SUB>4</SUB> per well
</LI><LI> read absorbance at 450 nm wavelength
</LI></OL><H4><SPAN class="mw-headline" id="Buffers"> Buffers </SPAN></H4><UL><LI><B>Coating Buffer</B> (0.1M NaHCO<SUB>3</SUB>, 0.1M Na<SUB>2</SUB>CO<SUB>3</SUB>, pH 9.5) 
</LI></UL><UL><LI><B>Diluting/Blocking Buffer (db)</B> (0.25 ml Tween 20, 30 g BSA, add 1x PBS to 500 ml)
</LI></UL><UL><LI><B>Wash Buffer</B> (0.5 ml Tween 20, 1L PBS 1x)
</LI></UL><DL><DD><B>Calibration</B></DD></DL><UL><LI> 960 µl db + 40 µl pooled plasma = 200 ng/ml
</LI><LI> add 500 µl of above into 500 µl of db = 100 ng/ml
</LI><LI> serial dilutions continued down to 1.5625  ng/ml
</LI></UL><DL><DD><B>Detection reagents</B></DD></DL><UL><LI> TMB Substrate Kit (Pierce, Thermo Scientific)
</LI><LI> 2M H<SUB>2</SUB>SO<SUB>4</SUB></LI></UL><H3><SPAN class="mw-headline" id="Microscopy"> Microscopy </SPAN></H3><P>We used fluorescence microscopy to measure EGFP and EBFP2 fluorescence intensity. Transfection efficiency was first evaluated using epifluorescence microscopes (Leica DM IRB or Olympus IX81) directly on the 96-well plates. Only cells which were transfected successfully were measured. Cells were washed with 1x PBS and detached from the plate using trypsin. 30µl trypsin was added to each well. After incubation for ten minutes at room temperature, ells were resuspended in 170µl 1%BSA in PBS and replicates for each condition were pooled into 8-well coverslip chambers.  100-150µl were used for confocal microscopy. 
</P><P>Single cell images were obtained using a Leica TCS SP5 laser scanning confocal microscope (LSCM) and alternatively a Leica TCS SP2 LSCM. EGFP fluorescence was excited by the 488nm laserline of an Argon laser and measured between 520 and 560nm, EBFP2 proteins were excited by UV laser at 405nm and measured between 440 and 460nm. Pictures were taken sequentially line by line in two different channels for EGFP, EBFP2. Bright field was acquired at the same time as the EBFP2 signal from the 405nm laser. 
</P><DIV class="center"><DIV class="thumb tnone"><DIV class="thumbinner" style="width:402px;"><DIV class="thumbcaption"><B>HeLa cells two days after transfection with miMeasure</B> (A) fluorescence signal GFP channel, 8bit; (B) fluorescence signal BFP channel, 8bit; (C) merge of channels A and B, RGB (D) cells after segmentation and automated cell counting and annotation</DIV></DIV></DIV></DIV><P>To analyze the blue and green fluorescence of single cells we analyzed the images with <I>ImageJ</I>. First, we separated the intensities coming from the bright cells from the background by thresholding the images. This allowed us to annotate cells automatically using the “analyze particles” tool. Measurements were redirected to the EGFP or EBFP2 respectively and the mean intensities were for each cell in each channel were saved. Panel 1 shows an example of one such image in different channels and after segmentation. From the data thus obtained, we calculated the EGFP:EBFP2 ratios for each cell using a simple algorithm. The average of these ratios served as the comparison measure and was visualized in bar plots for different constructs and/or conditions. Alternatively we used to individual data points to calculate the correlation between EGFP and EBFP2 expression through a simple linear regression analysis.
</P><UL><LI> Consumables and Chemicals
</LI></UL><P>PerkinElmer ViewPlate, product number: 6004920
Nunc Lab-Tek coverglass chamber, product number: 155411
1x PBS (1.37mM NaCl, 0.27mM KCl, 10mM Na2HPO4, 0.2mM KH2PO4)
0.05% Trypsin-EDTA, Invitrogen GIBCO, product number: 15400054</P><UL><LI> Instruments
</LI></UL><P>Leica DM IRB
Leica SP5
Leica SP2
ImageJ version 1.43</P><DIV class="backtop"><A href="#top">↑</A></DIV><H3><SPAN class="mw-headline" id="Flow_cytometry"> Flow cytometry </SPAN></H3><P>We used flow cytometry to analyse the EGFP and EBFP2 fluorescence intensities. Cells grown in a 96 well plate were washed once with 1xPBS and then trypsinised. Afterwards they were resuspended in PBS + 1% BSA and analyzed. EGFP and EBFP2 cellular intensities were measured with a flow cytometer was performed equipped with two laser diodes, one at 405nm and one at 488nm. The fluorescence of EGFP and EBFP2 was separated at 495nm by a dichroic mirror. Each signal was filtered  through two emission filters: the first with the band pass between 425nm and 475nm to detect EBFP2, the second between 497.5nm and 522.5nm to detect EGFP. The bleed through was compensated by measuring cells only expressing EGFP and accordingly EBFP2. For the measurements 100µl cells in PBS + 1% BSA were pumped by the machine and about 10000 cells were analysed for each construct.
</P><UL><LI> Consumables and Chemicals
</LI></UL><P>Falcon Becton Dickinson Microtest 96, product number: 353072 
1x PBS (1.37mM NaCl, 0.27mM KCl, 10mM Na2HPO4, 0.2mM KH2PO4) 
Bovine Serum Albumine, Sigma, product number: A9647 </P><UL><LI> Instruments
</LI></UL><P>Beckman Coulter Cytomics FC500MPL
</P><UL><LI>Analysis
</LI></UL><P>The reduction of EGFP expression compared to EBFP2 that we observed by flow cytometry could be quantitated as follow: two-dimensional scatter count plots representing cell counts for each EBFP2-EGFP intensity pair on a linear scale were exported as 64X64 pixel images. To ignore non-transfected cells, a square of 3X3 pixel originating from (0,0), where control non-transfected cells accumulate, was put to zero in each count plot. A ratio image of the same size containing the EGFP-EBFP2 intensity ratio for each pixel was generated.  The mean ratio was estimated by multiplying each count plot with the ratio image, measuring the total image intensity and dividing it by the total count of each count plot. For the standard deviation, the mean intensity of a count plot was subtracted to the ratio image. the square of each pixel intensity was multiplied by the corresponding count and the total intensity of the result was divided by the total count. The standard deviation was then calculated by taking the square root of this number. All steps were performed on imageJ. 
</P><DIV class="backtop"><A href="#top">↑</A></DIV><H3><SPAN class="mw-headline" id="Dual_Luciferase_Assay"> Dual Luciferase Assay </SPAN></H3><P>We measured the knockdown of firefly luciferase using the <A href="http://www.promega.com/tbs/tm046/tm046.pdf" class="external text" rel="nofollow">Promega Dual Luciferase Reporter Assay</A>. 
The DLR™ Assay System provides an efficient mean of performing dual-reporter assays, where the activities of firefly (<I>Photinus pyralis</I>) and Renilla (<I>Renilla reniformis</I>) luciferases (RL) are measured sequentially from a single sample. Firefly and Renilla luciferases can be used as a good reporter system, as those two enzymes have dissimilar enzyme structures and substrate requirements. This allows for selective discrimination between their bioluminescent reactions. The firefly luciferase (FL) reporter is measured first by adding Luciferase Assay Reagent II (LAR II) to generate a stabilized luminescent signal. After quantifying the firefly luminescence, this reaction is quenched, and the Renilla luciferase reaction is simultaneously initiated by adding Stop &amp; Glo® Reagent to the same tube. The Stop &amp; Glo® Reagent also produces a stabilized signal from the Renilla luciferase, which decays slowly over the course of the measurement. Here, Renilla luciferase is used for normalization. The measurements were conducted on the Promega GLOMAX 96 Microplate Luminometer using the Promega standard protocol (<A href="https://2010.igem.org/Team:Heidelberg/Project/References#Materials_and_Methods" class="external text" rel="nofollow">Sherf et al., 1996</A>). 
Twenty hours after transfection, cells were washed with 1x PBS and lysed using 1x Passive Lysis Buffer (5x stock solution diluted with distilled water), shaking for 30 minutes at 37°C. 10µl of the lysate were transferred to a white microplate (LumaPlate) as required for Luminometer measurements. 
</P><P>LAR II reagent was prepared by resuspending Luciferase Assay Substrate in 10ml Luciferase Assay Buffer II. For Stop &amp; Glo reagent, 2.1ml 50x Stop &amp; Glo substrate and 105ml Stop &amp; Glo Buffer were added to the amber Stop &amp; Glo reagent bottle and mixed by vortexing. Reagents where stored in 15ml aliquots at -80°C and thawed freshly prior to each measurement.
</P><P>To set up the Luminometer, the two injectors where flushed with distilled water, 70% ethanol, again water and air, three times each. Afterwards, they were primed three times with substrate reagents. 
</P><P>The activity of the first luciferase (firefly) was measured by adding 25µl of LAR II reagent to the well. The enzyme reacts upon translation without further processing and oxidates beetle luciferin, resulting in photon emission that can be measured. In addition to beetle luciferin, the LAR II reagent contains coenzyme A, which accelerates the reaction and thus creates a prolonged luminescence signal. The luminescence was measured two seconds after addition of the reagent, for ten seconds. Afterwards, 25µl Stop &amp; Glo reagent was added, which is able to quench the firefly luciferase activity and simultaneously contains the substrate for Renilla luciferase, coelenterazine. This second reaction also emits photons upon oxidation of the substrate. Addition of substrates and light emission measurements were conducted automatically by the GLOMAX Luminometer.
</P><H4><SPAN class="mw-headline" id="Order_of_recording_and_processing_raw_data"> Order of recording and processing raw data </SPAN></H4><OL><LI> triple-transfection of the following constructs in ratio 10:1:1
</LI><LI> shRNA-like miRNA (shRNA miR) expressing plasmid (25 ng / well)
</LI><LI> reference construct containing hRluc (2.5 ng / well)
</LI><LI> tuning construct with Luc2 fused to binding site referring to shRNA miR (2.5 ng / well)
</LI><DL><DD> note: used backbones for all constructs were either pBS_U6 or pBS_H1
</DD></DL><LI> measurement of bioluminescent signal after substrate addition
<DL><DD> raw data file with following <A name="nomenclature" href="#nomenclature">nomenclature</A>: shRNA_date_samples_ctrl_BB.xls
</DD></DL></LI><LI> first injection: LARII
</LI><LI> quantification of Firefly signal
</LI><LI> second injection: Stop &amp; Glo® Reagent
</LI><LI> quantification of Renilla signal
</LI></OL><LI> calculate signal ratio: Firefly/Renilla
<DL><DD> note: Renilla is used as a reference, Firefly signal determines binding site strength
</DD></DL></LI><LI> controls on every plate: perfect binding site (lower expression limit), no binding site (upper expression limit = 100%) defining overall expression range
</LI><LI> another control: separate triple-transfections of tuning constructs with a shRNA miR expression plasmid that is <I>not</I> corresponding to the binding site on the tuning construct (upper expression limit)
</LI><LI> normalization by control without binding site
</LI><LI> thus, normalized value corresponds to knockdown percentage
</LI><LI> deviation of knockdown percentage for corresponding shRNA miR by referring knockdown percentage of non-corresponding shRNA miR
</LI><LI> calculate mean values and standard deviation (out of eight replicates) for each construct containing imperfect binding site (intermediate expression)
<DL><DD> raw data file with following <A name="nomenclature" href="#nomenclature">nomenclature</A>: plate#_process_BB_shRNA.xls
</DD></DL></LI><LI> note: error propagation has to be taken into account
</LI><LI> grade calculated knockdown percentages in ascending order (lowest value always for perfect binding site, whereas maximum value without binding site)
<DL><DD> raw data file with following <A name="nomenclature" href="#nomenclature">nomenclature</A>: shRNA_BB_final.xlsx
</DD></DL></LI><H5><SPAN class="mw-headline" id="nomenclature"> nomenclature </SPAN></H5><TABLE class="wikitable sortable" border="0" style="text-align: left"><CAPTION align="top, left"><B>table 1</B>: Used nomenclature for raw data files.
</CAPTION><TBODY><TR bgcolor="#cccccc"><TD> abbreviation</TD><TD>meaning</TD><TD>example
</TD></TR><TR><TD>shRNA</TD><TD>expressed shRNA miR</TD><TD>shRNA miR against hAAT (&quot;hAAT&quot;)
</TD></TR><TR><TD>date</TD><TD>date of measurement in format YYYYMMDD</TD><TD>20101026
</TD></TR><TR><TD>samples</TD><TD>internal ID of constructs</TD><TD>M1-M6
</TD></TR><TR><TD>BB</TD><TD>plasmid backbone</TD><TD>either pBS_U6 (&quot;U6&quot;) or pBS_H1 (&quot;H1&quot;)
</TD></TR><TR><TD>plate#</TD><TD>number of measured 96 well plate</TD><TD>plate1
</TD></TR></TBODY></TABLE><H4><SPAN class="mw-headline" id="Consumables_and_Reagents"> Consumables and Reagents </SPAN></H4><P>LumaPlate, PerkinElmer, catalogue number 6005630<A href="http://www.promega.com/tbs/tm046/tm046.pdf," class="external text" rel="nofollow">Promega Dual-Luciferase® Reporter Assay System</A>, catalogue number E1910
</P><H4><SPAN class="mw-headline" id="Instruments"> Instruments </SPAN></H4><P>Promega GLOMAX 96 Microplate Luminometer
</P><DIV class="backtop"><A href="#top">↑</A></DIV></DIV><DIV class="printfooter">
Retrieved from &quot;<A href="http://2010.igem.org/Team:Heidelberg/Notebook/Methods">http://2010.igem.org/Team:Heidelberg/Notebook/Methods</A>&quot;</DIV></DIV><DIV id="footer-box" class="noprint"><DIV id="footer"><UL id="f-list"><LI id="t-recentchanges"><A href="/Special:RecentChanges" title="Recent changes">Recent changes</A></LI><LI id="t-whatlinkshere"><A href="/Special:WhatLinksHere/Team:Heidelberg/Notebook/Methods" title="List of all wiki pages that link here [j]" accesskey="j">What links here</A></LI><LI id="t-recentchangeslinked"><A href="/Special:RecentChangesLinked/Team:Heidelberg/Notebook/Methods" title="Recent changes in pages linked from this page [k]" accesskey="k">Related changes</A></LI><LI id="t-specialpages"><A href="/Special:SpecialPages" title="List of all special pages [q]" accesskey="q">Special pages</A></LI><LI><A href="/Special:Preferences">My preferences</A></LI></UL></DIV><DIV id="footer"><UL id="f-list"><LI id="t-print"><A href="/wiki/index.php?title=Team:Heidelberg/Notebook/Methods&amp;printable=yes" title="Printable version of this page [p]" accesskey="p">Printable version</A></LI><LI id="t-permalink"><A href="/wiki/index.php?title=Team:Heidelberg/Notebook/Methods&amp;oldid=205604" title="Permanent link to this revision of the page">Permanent link</A></LI><LI id="privacy"><A href="/2010.igem.org:Privacy_policy" title="2010.igem.org:Privacy policy">Privacy policy</A></LI><LI id="disclaimer"><A href="/2010.igem.org:General_disclaimer" title="2010.igem.org:General disclaimer">Disclaimers</A></LI></UL></DIV></DIV></BODY></HTML>