<HTML xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en" dir="ltr">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki  ltr ns-0 ns-subject page-Team_Heidelberg_Notebook_miRNA_Kit_October"><DIV id="globalWrapper"><DIV id="top-section"><DIV id="p-logo"><A href="/Main_Page" title="Main Page">&quot;
	    </A></DIV><DIV id="menubar" class="left-menu noprint"><UL><LI class="selected"><A href="/Team:Heidelberg/Notebook/miRNA_Kit/October">Page               </A></LI><LI class="new"><A href="/wiki/index.php?title=Talk:Team:Heidelberg/Notebook/miRNA_Kit/October&amp;action=edit&amp;redlink=1">Discussion               </A></LI><LI><A href="/wiki/index.php?title=Team:Heidelberg/Notebook/miRNA_Kit/October&amp;action=edit">View source               </A></LI><LI><A href="/wiki/index.php?title=Team:Heidelberg/Notebook/miRNA_Kit/October&amp;action=history">History               </A></LI><LI style="color:#808080;cursor:default">teams</LI></UL></DIV><DIV class="right-menu noprint" id="menubar"><UL><LI id="pt-login"><A href="/wiki/index.php?title=Special:UserLogin&amp;returnto=Team:Heidelberg/Notebook/miRNA_Kit/October" title="You are encouraged to log in; however, it is not mandatory [o]" accesskey="o">Log in</A></LI></UL></DIV><DIV id="search-controls" class="noprint"><FORM action="/Special:Search" id="searchform"> </FORM></DIV></DIV><DIV id="content"><H1 class="firstHeading">Team:Heidelberg/Notebook/miRNA Kit/October</H1><DIV id="bodyContent"><H3 id="siteSub" class="noprint">From 2010.igem.org</H3><DIV id="header"><DIV id="menu"><UL id="nav"><LI id="menu_home"><A class="aHome" href="https://2010.igem.org/Team:Heidelberg">Home</A></LI><LI id="menu_team"><A class="aTeam" href="https://2010.igem.org/Team:Heidelberg/Team">Team</A><TABLE class="menutable" cellpadding="0" cellspacing="0"><TBODY><TR><TD class="lefttdtopteam"><A class="aTeamOverview" href="https://2010.igem.org/Team:Heidelberg/Team">Overview</A><A class="aTeamMembers" href="https://2010.igem.org/Team:Heidelberg/Team/Members">Students</A><A class="aTeamMembers" href="https://2010.igem.org/Team:Heidelberg/Team/Members#Instructors">Instructors</A><A class="aTeamMembers" href="https://2010.igem.org/Team:Heidelberg/Team/Members#Advisors">Advisors</A><A class="aInstitutes" href="https://2010.igem.org/Team:Heidelberg/Team/Institutes">Institutes</A><A class="aGallery" href="https://2010.igem.org/Team:Heidelberg/Team/Gallery">Gallery</A><A class="aDance" href="https://2010.igem.org/Team:Heidelberg/midance">miDANCE</A><A class="amithanks" href="https://2010.igem.org/Team:Heidelberg/Team/miThanks">Acknowledgements</A></TD><TD class="righttdtop"> </TD></TR></TBODY></TABLE></LI><LI id="menu_project"><A class="aProject" href="https://2010.igem.org/Team:Heidelberg/Project">Project</A><TABLE class="menutable" cellpadding="0" cellspacing="0"><TBODY><TR><TD class="lefttdtopproject"><A class="aAbstract" href="https://2010.igem.org/Team:Heidelberg/Project">Abstract</A><A class="aIntroduction" href="https://2010.igem.org/Team:Heidelberg/Project/Introduction">Introduction</A><A class="aProject_miRNA_Kit" href="https://2010.igem.org/Team:Heidelberg/Project/miRNA_Kit">Synthetic miRNA Kit</A><A class="aPro_Capsid_Shuffling" href="https://2010.igem.org/Team:Heidelberg/Project/Capsid_Shuffling">Capsid Shuffling</A><A class="aPro_Homology" href="https://2010.igem.org/Team:Heidelberg/Project/Capsid_Shuffling/Homology_Based">    Homology Based</A><A class="aPro_Virobytes" href="https://2010.igem.org/Team:Heidelberg/Project/Capsid_Shuffling/ViroBytes">    ViroBytes</A><A class="aPro_miMeasure" href="https://2010.igem.org/Team:Heidelberg/Project/miMeasure">miMeasure</A><A class="aPro_Mouse" href="https://2010.igem.org/Team:Heidelberg/Project/Mouse_Infection"><I>in vivo</I> Study</A><A class="aPro_Summary" href="https://2010.igem.org/Team:Heidelberg/Project/Summary">Summary &amp; Outlook</A><A class="aPro_Ref" href="https://2010.igem.org/Team:Heidelberg/Project/References">References</A><A class="aPro_Ref" href="https://2010.igem.org/Team:Heidelberg/Project/Contributions">Contributions</A></TD><TD class="righttdtop"> </TD></TR></TBODY></TABLE></LI><LI id="menu_parts"><A class="aParts" href="https://2010.igem.org/Team:Heidelberg/Parts">Parts</A><TABLE class="menutable" cellpadding="0" cellspacing="0"><TBODY><TR><TD class="lefttdtop"><A class="aparts_ger" href="https://2010.igem.org/Team:Heidelberg/Parts">Parts</A><A class="aparts_char" href="https://2010.igem.org/Team:Heidelberg/Parts/Characterization">Characterization</A></TD><TD class="righttdtop"> </TD></TR></TBODY></TABLE></LI><LI id="menu_modeling"><A class="aModeling" href="https://2010.igem.org/Team:Heidelberg/Modeling">Modeling</A><TABLE class="menutable" cellpadding="0" cellspacing="0"><TBODY><TR><TD class="lefttdtop"><A class="amode_over" href="https://2010.igem.org/Team:Heidelberg/Modeling">Introduction</A><A class="amiGUI" href="https://2010.igem.org/Team:Heidelberg/Modeling/miGUI">miBEAT</A><A class="amirockdown" href="https://2010.igem.org/Team:Heidelberg/Modeling/miRockdown">miRockdown</A><A class="amibsdesigner" href="https://2010.igem.org/Team:Heidelberg/Modeling/miBSdesigner">miBS designer</A><A class="amispec" href="https://2010.igem.org/Team:Heidelberg/Modeling/miSpec">mUTING</A><A class="amodelset" href="https://2010.igem.org/Team:Heidelberg/Modeling/trainingset">Descriptions</A></TD><TD class="righttdtop"> </TD></TR></TBODY></TABLE></LI><LI id="menu_notebook"><A class="aNotebook" href="https://2010.igem.org/Team:Heidelberg/Notebook">Notebook</A><TABLE class="menutable" cellpadding="0" cellspacing="0"><TBODY><TR><TD class="lefttdtop"><A class="aNote_miRNA_kit" href="https://2010.igem.org/Team:Heidelberg/Notebook/miRNA_Kit">Synthetic miRNA Kit</A><A class="aNote_Capsid_Shuffling">Capsid Shuffling</A><A class="aNote_Homology" href="https://2010.igem.org/Team:Heidelberg/Notebook/Capsid_Shuffling/Homology_Based">    Homology Based</A><A class="aNote_Virobytes" href="https://2010.igem.org/Team:Heidelberg/Notebook/Capsid_Shuffling/ViroBytes">    ViroBytes</A><A class="aNote_miMeasure" href="https://2010.igem.org/Team:Heidelberg/Notebook/miMeasure">miMeasure</A><A class="aNote_BSdesign" href="https://2010.igem.org/Team:Heidelberg/Notebook/BSDesign">Binding Site Design </A><A class="aNote_Mouse" href="https://2010.igem.org/Team:Heidelberg/Notebook/Mouse_Infection"><I>in vivo</I> Study</A><A class="aNote_Modeling" href="https://2010.igem.org/Team:Heidelberg/Notebook/Modeling">Modeling</A><A class="aNote_Material" href="https://2010.igem.org/Team:Heidelberg/Notebook/Material">Materials</A><A class="aNote_Methods" href="https://2010.igem.org/Team:Heidelberg/Notebook/Methods">Methods</A><A class="aNote_Psycho" href="https://2010.igem.org/Team:Heidelberg/Human_Practices/Psychology/Notebook">Psychological Survey</A><A class="aNote_Philo" href="https://2010.igem.org/Team:Heidelberg/Human_Practices/Philosophy/Part2">Philosophical Reflection</A><A class="aNote_Biosafety" href="https://2010.igem.org/Team:Heidelberg/Notebook/Biosafety">Biosafety</A></TD><TD class="righttdtop"> </TD></TR></TBODY></TABLE></LI><LI id="menu_human_practices"><A class="aHuman_Practices" href="https://2010.igem.org/Team:Heidelberg/Human Practices">Human Practices</A><TABLE class="menutable" cellpadding="0" cellspacing="0"><TBODY><TR><TD class="lefttdtop"><A class="aPsycho" href="https://2010.igem.org/Team:Heidelberg/Human Practices/Psychology">Psychological Survey</A><A class="aPhilo" href="https://2010.igem.org/Team:Heidelberg/Human Practices/Philosophy">Philosophical Reflection</A></TD><TD class="righttdtop"> </TD></TR></TBODY></TABLE></LI><LI id="menu_support"><A class="aSupport" href="https://2010.igem.org/Team:Heidelberg/Support">Sponsors</A></LI></UL></DIV></DIV><TABLE cellpadding="5" cellspacing="0" align="center" style="text-align: center; color:#009be1; border: 1.5px solid #000000;"><TBODY><TR><TH colspan="7" style="background:#f09600;"><A href="https://2010.igem.org/Team:Heidelberg/Notebook/miRNA_Kit/August" class="external text" rel="nofollow"><FONT color="white">August</FONT></A></TH></TR><TR style="background:#f09600; color:white"><TD width="20pt"><B>M</B></TD><TD width="20pt"><B>T</B></TD><TD width="20pt"><B>W</B></TD><TD width="20pt"><B>T</B></TD><TD width="20pt"><B>F</B></TD><TD width="20pt"><B>S</B></TD><TD width="20pt"><B>S</B></TD></TR><TR style="background:#f2f2f2; color:#009be1"><TD><B>1</B></TD></TR><TR style="background:#f2f2f2; color:#009be1"><TD><B>2</B></TD><TD><B>3</B></TD><TD><B>4</B></TD><TD><B>5</B></TD><TD><B>6</B></TD><TD><B>7</B></TD><TD><B>8</B></TD></TR><TR style="background:#f2f2f2; color:#009be1"><TD><B>9</B></TD><TD><B>10</B></TD><TD><B>11</B></TD><TD><B>12</B></TD><TD><B>13</B></TD><TD><B>14</B></TD><TD><B>15</B></TD></TR><TR style="background:#f2f2f2; color:#009be1"><TD><B>16</B></TD><TD><B>17</B></TD><TD><B>18</B></TD><TD><B>19</B></TD><TD><B>20</B></TD><TD><B><A href="https://2010.igem.org/Team:Heidelberg/Notebook/miRNA_Kit/August#21/08/2010" class="external text" rel="nofollow">21</A></B></TD><TD><B><A href="https://2010.igem.org/Team:Heidelberg/Notebook/miRNA_Kit/August#22.2F08.2F2010" class="external text" rel="nofollow">22</A></B></TD></TR><TR style="background:#f2f2f2; color:#009be1"><TD><B><A href="https://2010.igem.org/Team:Heidelberg/Notebook/miRNA_Kit/August#23.2F08.2F2010" class="external text" rel="nofollow">23</A></B></TD><TD><B><A href="https://2010.igem.org/Team:Heidelberg/Notebook/miRNA_Kit/August#24.2F08.2F2010" class="external text" rel="nofollow">24</A></B></TD><TD><B><A href="https://2010.igem.org/Team:Heidelberg/Notebook/miRNA_Kit/August#25.2F08.2F2010" class="external text" rel="nofollow">25</A></B></TD><TD><B><A href="https://2010.igem.org/Team:Heidelberg/Notebook/miRNA_Kit/August#26.2F08.2F2010" class="external text" rel="nofollow">26</A></B></TD><TD><B><A href="https://2010.igem.org/Team:Heidelberg/Notebook/miRNA_Kit/August#27.2F08.2F2010" class="external text" rel="nofollow">27</A></B></TD><TD><B><A href="https://2010.igem.org/Team:Heidelberg/Notebook/miRNA_Kit/August#28.2F08.2F2010" class="external text" rel="nofollow">28</A></B></TD><TD><B><A href="https://2010.igem.org/Team:Heidelberg/Notebook/miRNA_Kit/August#29.2F08.2F2010" class="external text" rel="nofollow">29</A></B></TD></TR><TR style="background:#f2f2f2; color:#009be1"><TD><B><A href="/wiki/index.php?title=Igem2010/Main/synthetic_miR_Kit/August&amp;action=edit&amp;redlink=1" class="new" title="Igem2010/Main/synthetic miR Kit/August (page does not exist)">30</A></B></TD><TD><B><A href="/wiki/index.php?title=Igem2010/Main/synthetic_miR_Kit/August&amp;action=edit&amp;redlink=1" class="new" title="Igem2010/Main/synthetic miR Kit/August (page does not exist)">31</A></B></TD></TR></TBODY></TABLE><TABLE cellpadding="5" cellspacing="0" align="center" style="text-align: center; color:#78b41e; border: 1.5px solid #333333;"><TBODY><TR><TH colspan="7" style="background:#009be1;"><A href="https://2010.igem.org/Team:Heidelberg/Notebook/miRNA_Kit/September" class="external text" rel="nofollow"><FONT color="#ffecba">September</FONT></A></TH></TR><TR style="background:#009be1; color:white"><TD width="20pt"><B>M</B></TD><TD width="20pt"><B>T</B></TD><TD width="20pt"><B>W</B></TD><TD width="20pt"><B>T</B></TD><TD width="20pt"><B>F</B></TD><TD width="20pt"><B>S</B></TD><TD width="20pt"><B>S</B></TD></TR><TR style="background:#f2f2f2; color:#78b41e"><TD><B><A href="https://2010.igem.org/Team:Heidelberg/Notebook/miRNA_Kit/September#01.2F09.2F2010" class="external text" rel="nofollow">1</A></B></TD><TD><B><A href="https://2010.igem.org/Team:Heidelberg/Notebook/miRNA_Kit/September#02.2F09.2F2010" class="external text" rel="nofollow">2</A></B></TD><TD><B><A href="https://2010.igem.org/Team:Heidelberg/Notebook/miRNA_Kit/September#03.2F09.2F2010" class="external text" rel="nofollow">3</A></B></TD><TD><B><A href="https://2010.igem.org/Team:Heidelberg/Notebook/miRNA_Kit/September#04.2F09.2F2010" class="external text" rel="nofollow">4</A></B></TD><TD><B><A href="https://2010.igem.org/Team:Heidelberg/Notebook/miRNA_Kit/September#05.2F09.2F2010" class="external text" rel="nofollow">5</A></B></TD></TR><TR style="background:#f2f2f2; color:#78b41e"><TD><B><A href="https://2010.igem.org/Team:Heidelberg/Notebook/miRNA_Kit/September#06.2F09.2F2010" class="external text" rel="nofollow">6</A></B></TD><TD><B><A href="https://2010.igem.org/Team:Heidelberg/Notebook/miRNA_Kit/September#07.2F09.2F2010" class="external text" rel="nofollow">7</A></B></TD><TD><B><A href="https://2010.igem.org/Team:Heidelberg/Notebook/miRNA_Kit/September#08.2F09.2F2010" class="external text" rel="nofollow">8</A></B></TD><TD><B><A href="https://2010.igem.org/Team:Heidelberg/Notebook/miRNA_Kit/September#09.2F09.2F2010" class="external text" rel="nofollow">9</A></B></TD><TD><B><A href="https://2010.igem.org/Team:Heidelberg/Notebook/miRNA_Kit/September#10.2F09.2F2010" class="external text" rel="nofollow">10</A></B></TD><TD><B><A href="https://2010.igem.org/Team:Heidelberg/Notebook/miRNA_Kit/September#11.2F09.2F2010" class="external text" rel="nofollow">11</A></B></TD><TD><B><A href="https://2010.igem.org/Team:Heidelberg/Notebook/miRNA_Kit/September#12.2F09.2F2010" class="external text" rel="nofollow">12</A></B></TD></TR><TR style="background:#f2f2f2; color:#78b41e"><TD><B><A href="https://2010.igem.org/Team:Heidelberg/Notebook/miRNA_Kit/September#13.2F09.2F2010" class="external text" rel="nofollow">13</A></B></TD><TD><B><A href="https://2010.igem.org/Team:Heidelberg/Notebook/miRNA_Kit/September#14.2F09.2F2010" class="external text" rel="nofollow">14</A></B></TD><TD><B><A href="https://2010.igem.org/Team:Heidelberg/Notebook/miRNA_Kit/September#15.2F09.2F2010" class="external text" rel="nofollow">15</A></B></TD><TD><B><A href="https://2010.igem.org/Team:Heidelberg/Notebook/miRNA_Kit/September#16.2F09.2F2010" class="external text" rel="nofollow">16</A></B></TD><TD><B><A href="https://2010.igem.org/Team:Heidelberg/Notebook/miRNA_Kit/September#17.2F09.2F2010" class="external text" rel="nofollow">17</A></B></TD><TD><B><A href="https://2010.igem.org/Team:Heidelberg/Notebook/miRNA_Kit/September#18.2F09.2F2010" class="external text" rel="nofollow">18</A></B></TD><TD><B><A href="https://2010.igem.org/Team:Heidelberg/Notebook/miRNA_Kit/September#19.2F09.2F2010" class="external text" rel="nofollow">19</A></B></TD></TR><TR style="background:#f2f2f2; color:#78b41e"><TD><B><A href="https://2010.igem.org/Team:Heidelberg/Notebook/miRNA_Kit/September#20.2F09.2F2010" class="external text" rel="nofollow">20</A></B></TD><TD><B><A href="https://2010.igem.org/Team:Heidelberg/Notebook/miRNA_Kit/September#21.2F09.2F2010" class="external text" rel="nofollow">21</A></B></TD><TD><B><A href="https://2010.igem.org/Team:Heidelberg/Notebook/miRNA_Kit/September#22.2F09.2F2010" class="external text" rel="nofollow">22</A></B></TD><TD><B><A href="https://2010.igem.org/Team:Heidelberg/Notebook/miRNA_Kit/September#23.2F09.2F2010" class="external text" rel="nofollow">23</A></B></TD><TD><B><A href="https://2010.igem.org/Team:Heidelberg/Notebook/miRNA_Kit/September#24.2F09.2F2010" class="external text" rel="nofollow">24</A></B></TD><TD><B><A href="https://2010.igem.org/Team:Heidelberg/Notebook/miRNA_Kit/September#25.2F09.2F2010" class="external text" rel="nofollow">25</A></B></TD><TD><B><A href="https://2010.igem.org/Team:Heidelberg/Notebook/miRNA_Kit/September#26.2F09.2F2010" class="external text" rel="nofollow">26</A></B></TD></TR><TR style="background:#f2f2f2; color:#78b41e"><TD><B><A href="https://2010.igem.org/Team:Heidelberg/Notebook/miRNA_Kit/September#27.2F09.2F2010" class="external text" rel="nofollow">27</A></B></TD><TD><B><A href="https://2010.igem.org/Team:Heidelberg/Notebook/miRNA_Kit/September#28.2F09.2F2010" class="external text" rel="nofollow">28</A></B></TD><TD><B><A href="https://2010.igem.org/Team:Heidelberg/Notebook/miRNA_Kit/September#29.2F09.2F2010" class="external text" rel="nofollow">29</A></B></TD><TD><B><A href="https://2010.igem.org/Team:Heidelberg/Notebook/miRNA_Kit/September#22.2F30.2F2010" class="external text" rel="nofollow">30</A></B></TD></TR><TR style="background:#f2f2f2; color:#78b41e"><TD colspan="7"><SPAN style="color:#ffffff">-</SPAN></TD></TR></TBODY></TABLE><TABLE cellpadding="5" cellspacing="0" align="center" style="text-align: center; color:#f09600; border: 1.5px solid #333333;"><TBODY><TR><TH colspan="7" style="background:#78b41e;"><A href="https://2010.igem.org/Team:Heidelberg/Notebook/miRNA_Kit/October" class="external text" rel="nofollow"><FONT color="white">October</FONT></A></TH></TR><TR style="background:#78b41e; color:white"><TD width="20pt"><B>M</B></TD><TD width="20pt"><B>T</B></TD><TD width="20pt"><B>W</B></TD><TD width="20pt"><B>T</B></TD><TD width="20pt"><B>F</B></TD><TD width="20pt"><B>S</B></TD><TD width="20pt"><B>S</B></TD></TR><TR style="background:#f2f2f2; color:#78b41e"><TD><B><A href="https://2010.igem.org/Team:Heidelberg/Notebook/miRNA_Kit/October#01.2F10.2F2010" class="external text" rel="nofollow">1</A></B></TD><TD><B><A href="https://2010.igem.org/Team:Heidelberg/Notebook/miRNA_Kit/October#02.2F10.2F2010" class="external text" rel="nofollow">2</A></B></TD><TD><B><A href="https://2010.igem.org/Team:Heidelberg/Notebook/miRNA_Kit/October#03.2F10.2F2010" class="external text" rel="nofollow">3</A></B></TD></TR><TR style="background:#f2f2f2; color:#f09600"><TD><B><A href="https://2010.igem.org/Team:Heidelberg/Notebook/miRNA_Kit/October#04.2F10.2F2010" class="external text" rel="nofollow">4</A></B></TD><TD><B><A href="https://2010.igem.org/Team:Heidelberg/Notebook/miRNA_Kit/October#05.2F10.2F2010" class="external text" rel="nofollow">5</A></B></TD><TD><B><A href="https://2010.igem.org/Team:Heidelberg/Notebook/miRNA_Kit/October#06.2F10.2F2010" class="external text" rel="nofollow">6</A></B></TD><TD><B><A href="https://2010.igem.org/Team:Heidelberg/Notebook/miRNA_Kit/October#07.2F10.2F2010" class="external text" rel="nofollow">7</A></B></TD><TD><B><A href="https://2010.igem.org/Team:Heidelberg/Notebook/miRNA_Kit/October#08.2F10.2F2010" class="external text" rel="nofollow">8</A></B></TD><TD><B><A href="https://2010.igem.org/Team:Heidelberg/Notebook/miRNA_Kit/October#09.2F10.2F2010" class="external text" rel="nofollow">9</A></B></TD><TD><B><A href="https://2010.igem.org/Team:Heidelberg/Notebook/miRNA_Kit/October#10.2F10.2F2010" class="external text" rel="nofollow">10</A></B></TD></TR><TR style="background:#f2f2f2; color:#f09600"><TD><B><A href="https://2010.igem.org/Team:Heidelberg/Notebook/miRNA_Kit/October#11.2F10.2F2010" class="external text" rel="nofollow">11</A></B></TD><TD><B><A href="https://2010.igem.org/Team:Heidelberg/Notebook/miRNA_Kit/October#13.2F10.2F2010" class="external text" rel="nofollow">13</A></B></TD><TD><B><A href="https://2010.igem.org/Team:Heidelberg/Notebook/miRNA_Kit/October#14.2F10.2F2010" class="external text" rel="nofollow">14</A></B></TD><TD><B><A href="https://2010.igem.org/Team:Heidelberg/Notebook/miRNA_Kit/October#15.2F09.2F2010" class="external text" rel="nofollow">15</A></B></TD><TD><B><A href="https://2010.igem.org/Team:Heidelberg/Notebook/miRNA_Kit/October#16.2F10.2F2010" class="external text" rel="nofollow">16</A></B></TD><TD><B><A href="https://2010.igem.org/Team:Heidelberg/Notebook/miRNA_Kit/October#17.2F10.2F2010" class="external text" rel="nofollow">17</A></B></TD><TD><B><A href="https://2010.igem.org/Team:Heidelberg/Notebook/miRNA_Kit/October#18.2F10.2F2010" class="external text" rel="nofollow">18</A></B></TD></TR><TR style="background:#f2f2f2; color:#f09600"><TD><B><A href="https://2010.igem.org/Team:Heidelberg/Notebook/miRNA_Kit/October#19/10/2010" class="external text" rel="nofollow">19</A></B></TD><TD><B>20</B></TD><TD><B>21</B></TD><TD><B>22</B></TD><TD><B>23</B></TD><TD><B>24</B></TD><TD><B>25</B></TD></TR><TR style="background:#f2f2f2; color:#f09600"><TD><B>26</B></TD><TD><B>27</B></TD><TD><B>28</B></TD><TD><B>29</B></TD><TD><B>30</B></TD></TR><TR style="background:#f2f2f2; color:#f09600"><TD colspan="7"><SPAN style="color:#ffffff">-</SPAN></TD></TR></TBODY></TABLE><P> </P></DIV><H3><SPAN class="mw-headline" id="01.2F10.2F2010">01/10/2010</SPAN></H3><DIV class="thumb tright"><DIV class="thumbinner" style="width:352px;"><DIV class="thumbcaption">Gel 101001-1. Digestion of miniprep no. 8, 9, 14 and 16 with EcoRI and PstI. Odd lanes are undigested controls, even lanes are digested minis</DIV></DIV></DIV><P>repressor construct:
</P><UL><LI> colony 8, 9, 14 and 16 were chosen for miniprep. They consist of the complete TetR construct in pBS 1C3
</LI><LI> digestion of the minis with EcoRI and PstI for cloning into <A href="http://partsregistry.org/Part:pSB1A3" class="external text" rel="nofollow">pSB1A3</A>, SAP of <A href="http://partsregistry.org/Part:pSB1A3" class="external text" rel="nofollow">pSB1A3</A>, digestion was tested on gel and proven positive (see Gel 101001-1)
</LI><LI> nucleotide removal, then ligation of TetR8 and TetR14 equimolar with SAPed backbone, TetR9 and TetR16 2:1 with not-SAPed backbone, transformation with TOP10 cells (10µl on 50µl <I>E. coli</I>)
</LI></UL><DIV class="thumb tright"><DIV class="thumbinner" style="width:352px;"><DIV class="thumbcaption">Screening 1: shRNA10 single binding sites</DIV></DIV></DIV><DIV class="thumb tright"><DIV class="thumbinner" style="width:352px;"><DIV class="thumbcaption">Screening 2: shRNA10 single binding sites</DIV></DIV></DIV><UL><LI>Screening of the binding sites cloned into the constructs <A href="http://partsregistry.org/wiki/index.php?title=Part:BBa_K337036" class="external text" rel="nofollow">K3</A> and <A href="http://partsregistry.org/wiki/index.php?title=Part:BBa_K337038" class="external text" rel="nofollow">K4</A> containing shRNA10 in pSB1A3 on the previous day. 16 colonies were picked for each perfect, rand9-12 and rand9-22 binding site and colony-PCR was performed according to the colony-PCR standard protocol (Primers Luc2_bs_screen_seq and sh10_ape_xma_screen_rc). The results are shown on the gels Screening1 and Screening 2. Indicated are the construct numbers (either <A href="http://partsregistry.org/wiki/index.php?title=Part:BBa_K337036" class="external text" rel="nofollow">K3</A> or <A href="http://partsregistry.org/wiki/index.php?title=Part:BBa_K337038" class="external text" rel="nofollow">K4</A>, which differ in the promoter driving the Luc2 gene) and the binding site property (perfect, 9-12 randomized or 9-22 randomized). Almost all clones show a positive band, although only some have a really bright band at the size of ~ 250 bp. LB cultures for 10 clones for each constructs <A href="http://partsregistry.org/wiki/index.php?title=Part:BBa_K337036" class="external text" rel="nofollow">K3</A> and <A href="http://partsregistry.org/wiki/index.php?title=Part:BBa_K337038" class="external text" rel="nofollow">K4</A> were inocculated.
</LI></UL><H3><SPAN class="mw-headline" id="02.2F10.2F2010">02/10/2010</SPAN></H3><DIV class="thumb tright"><DIV class="thumbinner" style="width:352px;"><DIV class="thumbcaption">Gel 101002-1. Colony PCR. All lanes with right amplified fragment except on lane 3, 10, 11, 15 and 23. Lane 1 &amp; 14: 1kb Plus Ladder (Invitrogen)</DIV></DIV></DIV><DIV class="thumb tright"><DIV class="thumbinner" style="width:352px;"><DIV class="thumbcaption">Gel 101002-2. Test digestions of repressor construct. Upper pattern: EcoRI &amp; PstI. Lower pattern: BamHI. All bands revealing positive results except in the first column (negative control from lane 3 of gel 101002-1). Lane 1 and Lane 10: 1kb Plus Ladder (Invitrogen). Lane 16, 17, 18: incomplete digestion</DIV></DIV></DIV><UL><LI> Miniprep of the LB cultures (constructs <A href="http://partsregistry.org/wiki/index.php?title=Part:BBa_K337036" class="external text" rel="nofollow">K3</A> and <A href="http://partsregistry.org/wiki/index.php?title=Part:BBa_K337038" class="external text" rel="nofollow">K4</A> with shRNA10 and the corresponding binding sites) were done and used for transfection of Hek-293 cells
</LI></UL><P>
repressor construct:</P><UL><LI><A href="/wiki/index.php?title=Igem2010/Main/Protocols/Colony_PCR&amp;action=edit&amp;redlink=1" class="new" title="Igem2010/Main/Protocols/Colony PCR (page does not exist)"> colony-PCR</A> of 6 colonies for each plate of previous day transformations
</LI><LI> analysis, see gel 101002-1
</LI></UL><LI> twice test digestion of promising mini-preps, following <A href="https://2010.igem.org/3A_Assembly" class="external text" rel="nofollow">standard protocol recommendations</A></LI><LI> first digestion with EcoRI and PstI to check for insert size
</LI><LI> second digestion with BamHI to check the insert
</LI><LI> see gel 101002-2: some positive results will be send for sequencing and typed <A href="/wiki/index.php?title=Igem2010/Main/Library/Constructs/pSMB_miTrigger&amp;action=edit&amp;redlink=1" class="new" title="Igem2010/Main/Library/Constructs/pSMB miTrigger (page does not exist)"> T1 and T2</A>, respectively
</LI><LI> moreover: usage of one construct for transfection into HEK cells
</LI><H3><SPAN class="mw-headline" id="03.2F10.2F2010">03/10/2010</SPAN></H3><P>repressor construct: </P><UL><LI> sequencing results were fine
</LI><LI> digestion and purification of <A href="/wiki/index.php?title=Igem2010/Main/Library/Constructs/pSMB_miTrigger&amp;action=edit&amp;redlink=1" class="new" title="Igem2010/Main/Library/Constructs/pSMB miTrigger (page does not exist)"> T1 and T2</A> with XmaI and XhoI to clone in binding sites (against ON-targets) following <A href="https://2010.igem.org/3A_Assembly" class="external text" rel="nofollow">standard protocol recommendations</A></LI><LI> annealing of binding sites using oligos
</LI><LI> D108 &amp; D109: once binding site for miR122
</LI><LI> D110 &amp; D111: twice binding site for miR122 right behind each other
</LI><LI> protocol: 
</LI><LI> 5 µl of each oligo, 5 µl of Buffer 2 (NEB), 35 µl H<SUB>2</SUB>O
</LI><LI> 95°C, 5'
</LI><LI> cool down to room temperature for 2h
</LI><LI> incubation on ice, 10'
</LI><LI> use 1 µl for ligation with ~ 40 ng of backbone (i. e. digested <A href="/wiki/index.php?title=Igem2010/Main/Library/Constructs/pSMB_miTrigger&amp;action=edit&amp;redlink=1" class="new" title="Igem2010/Main/Library/Constructs/pSMB miTrigger (page does not exist)"> T1 and T2</A>)
</LI></UL><LI> ligation (including to negative controls with backbone only to check for re-ligation without SAP treatment)
</LI><LI><A href="https://2010.igem.org/Transformation" class="external text" rel="nofollow">transformation</A> and growth on selective plates (ampicillin)
</LI><H3><SPAN class="mw-headline" id="04.2F10.2F2010">04/10/2010</SPAN></H3><P>repressor construct:</P><UL><LI><A href="/wiki/index.php?title=Igem2010/Main/Protocols/Colony_PCR&amp;action=edit&amp;redlink=1" class="new" title="Igem2010/Main/Protocols/Colony PCR (page does not exist)"> colony-PCR</A> of 6 colonies for each plate of previous day transformations
</LI><LI> gel 101004-1 reveals almost positive samples
</LI></UL><LI> submission of each a promising mini-prep for sequencing of binding site and correct insert
</LI><H3><SPAN class="mw-headline" id="05.2F10.2F2010">05/10/2010</SPAN></H3><P>repressor construct:</P><UL><LI><I>sequencing results</I> from previous day:
</LI><LI> once or twice the binding site in each of the T1 constructs, respectively
</LI><LI> once the binding site for the T2 construct
</LI><LI> re-trial of ligation for two binding sites in T2
</LI></UL><LI> planned transfection into HeLa and HUH cells (non-liver and liver to prove ON-targeting) in ratio 6:1 = repressor:operator(tuning) as recommended in the <A href="http://www.roche-applied-science.com/pack-insert/1815091a.pdf" class="external text" rel="nofollow">manual</A></LI><P>new approach:
</P><UL><LI> purification of PCR amplified shRNA-like miRNA against hAAT (&quot;shhAAT&quot;)
</LI><LI> digestion of shhAAT and <A href="/wiki/index.php?title=Igem2010/Main/Library/Constructs/pSMB_miTrigger&amp;action=edit&amp;redlink=1" class="new" title="Igem2010/Main/Library/Constructs/pSMB miTrigger (page does not exist)"> Q2 and Q3</A> with HindIII and AflII (for backbones sequential because of two inner AflII cutting sites)
</LI><LI> ligation in triplicates (using 70 ng of backbone and 10, 15 or 20 ng of insert, respectively)
</LI><LI><A href="https://2010.igem.org/Transformation" class="external text" rel="nofollow">transformation</A>, over-night growth on selective agar plates (ampicillin)
</LI></UL><H3><SPAN class="mw-headline" id="06.2F10.2F2010">06/10/2010</SPAN></H3><UL><LI> waiting for colonies from previous night, which did not grow after all
</LI></UL><H3><SPAN class="mw-headline" id="07.2F10.2F2010">07/10/2010</SPAN></H3><UL><LI> repetition of shhAAT cloning
</LI><LI> touch-down PCR setup: 8.5 µl ddH<SUB>2</SUB>O, 0.5 µl template DNA (pcDNA5, 50 ng/µl), 0.5 µl each primer (i.e. <A href="/wiki/index.php?title=Igem2010/Main/Primer_database&amp;action=edit&amp;redlink=1" class="new" title="Igem2010/Main/Primer database (page does not exist)"> D7 &amp; D16</A>, 10 µl Phusion PCR MasterMix (2x)
</LI><LI> touch-down PCR protocol: <A href="https://2010.igem.org/Team:Heidelberg/Notebook/Methods#Plasmid-PCR" class="external text" rel="nofollow">like this</A> but with increase of annealing temperature from 68°C to 61°C (i. e. 0.2°C per cycle)
</LI><LI> go on like <A href="/Igem2010/Main/synthetic_miR_Kit/October#05.2F10.2F2010" title="Igem2010/Main/synthetic miR Kit/October"> 05/10/2010</A> but growth on freshly prepared plates
</LI></UL><H3><SPAN class="mw-headline" id="08.2F10.2F2010">08/10/2010</SPAN></H3><UL><LI> mini-prep and submission for sequencing for the only promising colony of <A href="/wiki/index.php?title=Igem2010/Main/Library/Constructs/pSMB_miTrigger&amp;action=edit&amp;redlink=1" class="new" title="Igem2010/Main/Library/Constructs/pSMB miTrigger (page does not exist)"> Q2</A> containing shhAAT for sequencing
</LI></UL><H3><SPAN class="mw-headline" id="09.2F10.2F2010">09/10/2010</SPAN></H3><UL><LI> sequencing the day before was fine but construct contains TetO<SUB>2</SUB>, thus: maybe Q2 and Q3 were confused
</LI><LI> that would explain why measurements with TetR constructs are not working properly even though sequences were fine
</LI></UL><LI> cloning of binding sites for shhAAT (perfect binding site generated via annealing, and randomized ones PCR amplified)
</LI><LI> touch-down PCR setup: 17 µl ddH<SUB>2</SUB>O, 4 µl each oligo (i.e. respective forward primer + second strand oligo<A href="/wiki/index.php?title=Igem2010/Main/Primer_database&amp;action=edit&amp;redlink=1" class="new" title="Igem2010/Main/Primer database (page does not exist)"> D121</A>), 25 µl Phusion PCR MasterMix (2x)
</LI><LI> touch-down PCR protocol: <A href="https://2010.igem.org/Team:Heidelberg/Notebook/Methods#shRNA-PCR_protocol" class="external text" rel="nofollow">like this</A> but with increase of annealing temperature from 70°C to 62°C (i. e. 0.5°C per cycle, 16x) and four additional cycles with 62°C
</LI><LI> annealing of perfect binding against shhAAT sites with an oligo-based strategy following the protocol from the <A href="/Igem2010/Main/synthetic_miR_Kit/October#03.2F10.2F2010" title="Igem2010/Main/synthetic miR Kit/October"> 3<SUP>rd</SUP> October</A></LI><LI> digestion of PCR amplified binding sites with AgeI and XhoI, digestion of Q2 backbone containing the shhAAT with XhoI and XmaI, afterwards heat inactivation, nucleotide removal and ligation following <A href="https://2010.igem.org/3A_Assembly" class="external text" rel="nofollow">standard protocol recommendations</A></LI><LI><A href="https://2010.igem.org/Transformation" class="external text" rel="nofollow">transformation</A> and over-night growth on selective agar plates (ampicillin)
</LI><H3><SPAN class="mw-headline" id="10.2F10.2F2010">10/10/2010</SPAN></H3><DIV class="thumb tright"><DIV class="thumbinner" style="width:352px;"><DIV class="thumbcaption">Gel 101010-1. Colony PCR for construct containing shhAAT and binding sites. Expected is an amplified fragment a bit smaller than 300 bp which can be seen on lane 3, 4, 6, 7, 8 (perfect binding sites), 12, 13, 14, 16 and 17 (randomized binding sites). Lane 1 and Lane 26: 1kb Plus Ladder (Invitrogen)</DIV></DIV></DIV><UL><LI> colony pcr reveals some positive clones (see gel 101010-1) from previous day
</LI><LI> inoculation of 5 ml LB Amp cultures for mini-prep over night
</LI></UL><DIV class="thumb tright"><DIV class="thumbinner" style="width:352px;"><DIV class="thumbcaption">ON system test</DIV></DIV></DIV><UL><LI><A href="https://2010.igem.org/Team:Heidelberg/Notebook/Methods#Dual_Luciferase_Assay" class="external text" rel="nofollow">Dual Luciferase Assay</A></LI></UL><P><A href="http://partsregistry.org/wiki/index.php?title=Part:BBa_K337038" class="external text" rel="nofollow">Tuning construct in which luc2 expression is driven by CMV promoter with TetO2</A> also expressing either miR122 or miR sAg was cotransfected with Tet repressor under regulation of perfect miR sAg  binding site. Amounts transfected were: 2.5ng of <A href="http://partsregistry.org/wiki/index.php?title=Part:BBa_K337038" class="external text" rel="nofollow">tuning construct</A>, 20ng of miRNA expressing plamid (either miR122 or miR sAg) and 5ng of repressor construct. To keep transfection efficiency the same we used sheared salmon sperm DNA as a stuffer.
</P><P>
In presence of Tet repressor and miR122 that is not targeting it, less of firefly luciferase is expressed, but when miR sAg is expressed is recovered to 60% of original activity.
</P><H3><SPAN class="mw-headline" id="11.2F10.2F2010">11/10/2010</SPAN></H3><UL><LI> mini-prep of ten cultures from previous day and direct transfection to test the system
</LI><LI> amount: each 100 µl (c=2.5 ng/µl)
</LI></UL><P>measurement construct for single binding sites:</P><UL><LI> PCR-based generation of binding sites against miR122 (perfect and randomized)
</LI><LI> touch-down PCR setup: 
</LI><LI> 17 µl ddH<SUB>2</SUB>O, 
</LI><LI> 4 µl each oligo (i.e. respective forward primer <A href="/wiki/index.php?title=Igem2010/Main/Primer_database&amp;action=edit&amp;redlink=1" class="new" title="Igem2010/Main/Primer database (page does not exist)"> D71, D62 or D5</A> + second strand oligo<A href="/wiki/index.php?title=Igem2010/Main/Primer_database&amp;action=edit&amp;redlink=1" class="new" title="Igem2010/Main/Primer database (page does not exist)"> D13</A>), 
</LI><LI> 25 µl Phusion PCR MasterMix (2x)
</LI></UL><LI> touch-down PCR protocol: 
</LI><LI><A href="https://2010.igem.org/Team:Heidelberg/Notebook/Methods#shRNA-PCR_protocol" class="external text" rel="nofollow">like this</A> but with increase of annealing temperature from 70°C to 62°C (i. e. 0.5°C per cycle, 16x) 
</LI><LI> and four additional cycles with 62°C, elongation time only 10 seconds
</LI><LI> digestion of generated binding sites and the pSMB_miMeasure backbone with EcoRI and PstI, then nucleotide removal
</LI><LI> ligation with ~ 70 ng vector and between 7.5 ng and 15 ng insert
</LI><LI><A href="https://2010.igem.org/Transformation" class="external text" rel="nofollow">transformation</A></LI><H3><SPAN class="mw-headline" id="12.2F10.2F2010">12/10/2010</SPAN></H3><P>measurement construct for single binding sites:
</P><UL><LI> colony pcr of each eight of load of colonies + two negative controls (sum: 50 samples)
</LI><LI> protocol: 16 cycles touch-down pcr from 70°C to 62°C of annealing temperature, then 19 additional cycles with 62°C annealing temperature
</LI><LI> elongation time: 90''
</LI></UL><LI><U>no positive</U> results, maybe due to insufficient screening primers (one for BBb standard, the other one for CMV resulting in an 1300 bp fragment)
</LI><LI> as a consequence: repetition of entire cloning with <A href="http://www.fermentas.com/templates/files/tiny_mce/coa_pdf/coa_ef0511.pdf" class="external text" rel="nofollow">SAP</A> treated backbone following day
</LI><H3><SPAN class="mw-headline" id="13.2F10.2F2010">13/10/2010</SPAN></H3><P>measurement construct for single binding sites: </P><UL><LI> repetition of cloning from 11/10/2010
</LI><LI><U>note:</U> vector concentration of pSMB_miMeasure is okay but after digestion and nucleotide removal it is always pretty low 
</LI><LI> (loss of 75% of DNA on average whereas other DNA yields to 90% of purified DNA as compared to digested amount)
</LI></UL><H3><SPAN class="mw-headline" id="14.2F10.2F2010">14/10/2010</SPAN></H3><UL><LI> cloning of 47 - 58 into pSB_H1
</LI><LI> cloning of tuning constructs for in vivo measurement:
</LI><LI>cloning of binding sites of miRNA haat with perfect and several imperfect binding
</LI><LI>oligo annealing following the standard protocol
</LI><LI> digestion of backbone pbsU6 with NotI and XhoI
</LI><LI> ligation and transformation
</LI></UL><H3><SPAN class="mw-headline" id="15.2F10.2F2010">15/10/2010</SPAN></H3><DIV class="thumb tright"><DIV class="thumbinner" style="width:352px;"><DIV class="thumbcaption">Screening of M1 to M11 constructs, 4 clones were picked from each plate. 300bp band marks positive clones</DIV></DIV></DIV><DIV class="thumb tright"><DIV class="thumbinner" style="width:352px;"><DIV class="thumbcaption">Screening of M12 to M22 constructs, 4 clones were picked from each plate. 300bp band marks positive clones</DIV></DIV></DIV><DIV class="thumb tright"><DIV class="thumbinner" style="width:352px;"><DIV class="thumbcaption">Screening of measurement standard with perfect, 9-12 and 9-22 randomized binding sites. constructs, 8 clones were picked from each plate. 300bp band marks positive clones</DIV></DIV></DIV><UL><LI> screening of plates M1 to M11 of the transformation with the construct pBSU6_SV40_Luc2 with 11 different binding sites for sh hAAT. Four clones from each plate were screened by colony pcr, positives give a band at 300bp. Primers Luc2_bs_screen_seq_fw and reverse primerse are the reverse binding site oligos for the binding sites.
</LI><LI> inoculation of miniprep cultures for one positive from each plate.
</LI></UL><UL><LI> transformation of miRsAg and mir122 expression constructs
</LI></UL><UL><LI> screening of measurement standart cloning from 13th with perfect, 9-12 randomized and 9-22 randomized binding sites for miRNA 122. primers mir122_mimeasure_screen_rev and pSMB_miMeasure_screen_fw. Positives give a band at 300bp. 
</LI><LI> inoculation of miniprep cultures for positive clones
</LI></UL><UL><LI> screening of plates M12 to M22 of the transformation with the measurement plasmid with 11 different binding sites for miRsAg. Four clones from each plate were screened by colony pcr, positives give a band at 300bp. Primers pSMB_miMeasure_Sequencing_rev and forward primerse are the forward primer oligos for the binding sites.
</LI><LI> inoculation of miniprep cultures for one positive from each plate.
</LI></UL><H3><SPAN class="mw-headline" id="16.2F10.2F2010">16/10/2010</SPAN></H3><UL><LI> mini Prep of mimeasure colonies (4 each)
</LI><LI> mini prep of haat constructs
</LI><LI> inoculation of miRsAg and mir122 expression constructs
</LI><LI> Maxi Preps of constructs M1 till M10
</LI><LI> inoculation for Maxipreps of Adenovirus 5, AAV8 rep and cap genes and pBSU6 H1 haat construct
</LI><LI> inoculation for Maxiprep of constructs M11 an M23-29
</LI><LI> cloning of on targeting constructs with a new strategy...
</LI></UL><H3><SPAN class="mw-headline" id="17.2F10.2F2010">17/10/2010</SPAN></H3><UL><LI> annealing of eleven sAg binding sites using oligos 
</LI><LI> protocol: 
</LI><LI> 5 µl of each oligo, 5 µl of Buffer 2 (NEB), 35 µl H<SUB>2</SUB>O
</LI><LI> 95°C, 5'
</LI><LI> cool down to room temperature for 2h
</LI><LI> incubation on ice, 10'
</LI><LI> use 1 µl for ligation with ~ 40 ng of backbone (i. e. with NotI and XhoI digested pBS_U6 containing hAAt cDNA)
</LI></UL><LI><A href="https://2010.igem.org/Transformation" class="external text" rel="nofollow">transformation</A> and growth on selective plates (ampicillin)
</LI><H3><SPAN class="mw-headline" id="18.2F10.2F2010">18/10/2010</SPAN></H3><UL><LI><A href="https://2010.igem.org/Team:Heidelberg/Notebook/Methods#Colony_PCR" class="external text" rel="nofollow">colony-PCR</A> of 4 colonies for each plate of <A href="/Igem2010/Main/synthetic_miR_Kit/October#17.2F10.2F2010" title="Igem2010/Main/synthetic miR Kit/October"> previous day transformations</A> using reverse binding site oligo and forward hAAT primer expecting a fragment round about 1300 bp
</LI><LI> gel 101018-1 reveals almost positive samples except two (data not yet shown)
</LI></UL><H3><SPAN class="mw-headline" id="19.2F10.2F2010">19/10/2010</SPAN></H3><UL><LI> mini-preps of cloned shAAT constructs
</LI><LI> dilution and co-transfection with miRsAg expressing construct into HeLa cells for ELISA measurements
</LI></UL><H3><SPAN class="mw-headline" id="23.2F10.2F2010">23/10/2010</SPAN></H3><P>seeding HeLa cells
</P><H3><SPAN class="mw-headline" id="24.2F10.2F2010">24/10/2010</SPAN></H3><P><A href="https://2010.igem.org/Team:Heidelberg/Notebook/Methods#Transfection" class="external text" rel="nofollow">Transfection</A></P><P>1) HeLa luc2-miR122BS and miR122
2) for ELISA - pBS_U6 containing hAAT with miRsAg imperfect binding sites, and miRsAg 
ratios 1:1 ans 4:1
</P><H3><SPAN class="mw-headline" id="26.2F10.2F2010">26/10/2010</SPAN></H3><UL><LI><A href="https://2010.igem.org/Team:Heidelberg/Notebook/Methods#Dual_Luciferase_Assay" class="external text" rel="nofollow">Dual Luciferase Assay</A></LI></UL><DIV class="center"><DIV class="thumb tnone"><DIV class="thumbinner" style="width:602px;"><DIV class="thumbcaption"><B>Figure 3: Tuning of gene expression through different imperfect miR122 binding sites in psiCHECK-2.</B> Construct was transfected into HeLa cells together with an plasmid expressing miR122. Control without binding site was used for normalization.</DIV></DIV></DIV></DIV><DIV class="center"><DIV class="thumb tnone"><DIV class="thumbinner" style="width:602px;"><DIV class="thumbcaption"><B>Figure 1: Tuning of gene expression through different imperfect shRNA miR binding sites in pBS_U6</B> Gene expression quantified via dual luciferase assay for constructs containing different imperfect binding sites for shhAAT.</DIV></DIV></DIV></DIV><DIV class="center"><DIV class="thumb tnone"><DIV class="thumbinner" style="width:602px;"><DIV class="thumbcaption"><B>Figure 2: Tuning of gene expression through different imperfect shRNA miR binding sites in pBS_H1.</B> Gene expression quantified via dual luciferase assay for constructs containing different imperfect binding sites for shhAAT.</DIV></DIV></DIV></DIV><UL><LI><A href="https://2010.igem.org/Team:Heidelberg/Notebook/Methods#ELISA" class="external text" rel="nofollow">ELISA</A> was performed on media in which HeLa cells we grown, as hAAT is secreted by cells.
</LI></UL></DIV><DIV class="printfooter">
Retrieved from &quot;<A href="http://2010.igem.org/Team:Heidelberg/Notebook/miRNA_Kit/October">http://2010.igem.org/Team:Heidelberg/Notebook/miRNA_Kit/October</A>&quot;</DIV></DIV><DIV id="footer-box" class="noprint"><DIV id="footer"><UL id="f-list"><LI id="t-recentchanges"><A href="/Special:RecentChanges" title="Recent changes">Recent changes</A></LI><LI id="t-whatlinkshere"><A href="/Special:WhatLinksHere/Team:Heidelberg/Notebook/miRNA_Kit/October" title="List of all wiki pages that link here [j]" accesskey="j">What links here</A></LI><LI id="t-recentchangeslinked"><A href="/Special:RecentChangesLinked/Team:Heidelberg/Notebook/miRNA_Kit/October" title="Recent changes in pages linked from this page [k]" accesskey="k">Related changes</A></LI><LI id="t-specialpages"><A href="/Special:SpecialPages" title="List of all special pages [q]" accesskey="q">Special pages</A></LI><LI><A href="/Special:Preferences">My preferences</A></LI></UL></DIV><DIV id="footer"><UL id="f-list"><LI id="t-print"><A href="/wiki/index.php?title=Team:Heidelberg/Notebook/miRNA_Kit/October&amp;printable=yes" title="Printable version of this page [p]" accesskey="p">Printable version</A></LI><LI id="t-permalink"><A href="/wiki/index.php?title=Team:Heidelberg/Notebook/miRNA_Kit/October&amp;oldid=178228" title="Permanent link to this revision of the page">Permanent link</A></LI><LI id="privacy"><A href="/2010.igem.org:Privacy_policy" title="2010.igem.org:Privacy policy">Privacy policy</A></LI><LI id="disclaimer"><A href="/2010.igem.org:General_disclaimer" title="2010.igem.org:General disclaimer">Disclaimers</A></LI></UL></DIV></DIV></BODY></HTML>