"
Page
Discussion
View source
History
teams
Log in
 
Team:Heidelberg/Notebook/miMeasure/July
From 2010.igem.org
Home
Team
Overview
Students
Instructors
Advisors
Institutes
Gallery
miDANCE
Acknowledgements
 
Project
Abstract
Introduction
Synthetic miRNA Kit
Capsid Shuffling
    Homology Based
    ViroBytes
miMeasure
in vivo Study
Summary & Outlook
References
Contributions
 
Parts
Parts
Characterization
 
Modeling
Introduction
miBEAT
miRockdown
miBS designer
mUTING
Descriptions
 
Notebook
Synthetic miRNA Kit
Capsid Shuffling
    Homology Based
    ViroBytes
miMeasure
Binding Site Design
in vivo Study
Modeling
Materials
Methods
Psychological Survey
Philosophical Reflection
Biosafety
 
Human Practices
Psychological Survey
Philosophical Reflection
 
Sponsors
August
M
T
W
T
F
S
S
1
2
3
4
5
6
7
8
9
10
11
12
13
14
15
16
17
18
19
20
21
22
23
24
25
26
27
28
29
30
31
September
M
T
W
T
F
S
S
1
2
3
4
5
6
7
8
9
10
11
12
13
14
15
16
17
18
19
20
21
22
23
24
25
26
27
28
29
30
-
October
M
T
W
T
F
S
S
1
2
3
4
5
6
7
8
9
10
11
12
13
14
15
16
17
18
19
20
21
22
23
24
25
26
27
28
29
30
31
 
05/07/2010 - 11/07/2010
Preperation of competent E. coli Top10 and DH5alpha Cell Culture Starting
12/07/2010 - 19/07/2010
Preperation of RNA extracts for miRNA profiling
Hela, Hek and HUH were plated on p100 dishes in their according media (10E6 cells/dish) and grown to 60-70 % confluence
Infection of Hela p4, HUH-7 and HEK cells with AAV serotype 2 (GFP) and incubation for 48 h
afterwards, RNA Extraction was performed according to the following protocol:
Protocol
Samples were afterwards analyzed for miRNA expression by using the FEBIT miRNA profiling service
pSB1AC3 from the registry was transformed into Top10 cells according to the standard transformation protocol; a day later, a 5 ml LB culture was inocculated and incubated for 8 hours; plasmid was extracted by applying the Qiagen MiniPrep Protocol
design of diraPCR oligos
a diraPCR Designer tool (called miRACLE Designer) was programmed for enabling quick&easy design of diraPCR oligos. The program takes any microRNA guiding strand sequence of choice and constructs synthetic microRNA binding site oligos. Those oligos have the following properties:
they have a 5' and 3' anealing sequence which is inerte (has no natural microRNA target)
the microRNA binding site of choice with randomized nucleotides at position 9-12 and a missmatch introduced at position 0
an inerte spacer sequence of 0, 5 10 or 15 bp length
The miRACLE Designer constructs a set of 8 ready-to-order oligos and prints them out in the common FASTA format. When performing diraPCR, binding sites will be concatomerized with spacer sequences in between 15 and 30 bp length.
design of measurement standard
in order to design an appropriety standard respecting all the rules according to the standard of mammalian synthetic biology (RFC12), we developed a dual reporter measurment construct with the following properties:
bidirectional CMV promoter driving both reporter genes
distabilized EGFP and EBFP2 as reporter genes, as they are really similar in their whole structure and amino-acid sequence
BBB prefix and suffix cloned behind the EGFP for easy swapping of binding site patterns
unique BamHI and HindIII binding sites introduced behind the EBFP2 reporter for introducing a reference binding site
strong SV40 terminators behind both reporters
vector backbone, containing an Amp resistance, Hygromycin resistance, pBR322 origin and an FRT site for stable integration
20/07/2010
Dilution of raPCR-Oligos
oligos (hsa_miR_886_3p_sp0...) and stop oligo (miRaPCR_Stop_fw_Ecori and miRA_PCR_Stop_rev) diluted to a concentration of 100 um
Set up raPCR-reactions with different amounts of oligos being pooled
2 ul of each hsa_miR_886_3p oligo
1.0, 2 or 4 ul of each stop oligo
25 ul of Phusion PCR MasterMix
add water to a total volume of 50 ul
Seven-Cyle PCR
run seven-cycle PCR under the following conditions
................................................
95 °C/ 5 min
................................................
95 °C/ 30 sec
55 °C/ 45 sec
72 °C/ 30 sec
................................................ (7x)
4 °C/ forever
................................................
PCR purification of the PCR-product (elution in 42 ul of water)
25-cycle PCR
run construct amplification PCR according to the following protocol
20 ul of purified PCR product
1 ul of each stop oligo
3 ul of water
25 ul of Phusion PCR MasterMix
................................................
95 °C/ 5 min
................................................
95 °C/ 30 sec
65 °C/ 45 sec
72 °C/ 30 sec
................................................ (25x)
72 °C/ 5 min
................................................
4 °C/ forever
................................................
Gel Electrophoresis
PCR products from the 25 cycle PCR were analyzed on a Gel. Therefor, 5 ul of PCR product were mixed with 1 ul of 6x-loading dye and loaded on a 1.5 % agarose gel. The gel ran at 135 V for 55 min.
21/07/2010
optimization of cycle number and oligo concentration for raPCR
0.5 (BS0.5), 1 (BS1) or 1.5 (BS1.5) ul of each binding site oligo
0 (Stop0), 0.5 (stop0.5) or 1 (stop1) ul of each stop oligo
25 ul of Phusion PCR MasterMix
add water to total volume of 25 ul
12-cycle PCR
run 12-cycle PCR according to the following program
................................................
95 °C/ 5 min
................................................
95 °C/ 30 sec
57 °C/ 45 sec
72 °C/ 45 sec
................................................ (12x)
4 °C/ forever
................................................
add 1 ul of each stop oligo to reactions "Stop0"
run 3-cycle PCR with the "Stop0" reaction according to the following protocol
................................................
95 °C/ 5 min
................................................
95 °C/ 30 sec
65 °C/ 45 sec
72 °C/ 45 sec
................................................ (3x)
4 °C/ forever
................................................
PCR purify all products
25-cycle PCR
run 25-cycle PCR in order to amplify constructs
20 ul of purified PCR product
1 ul of each stop oligo
3 ul of water
25 ul of Phusion PCR MasterMix
for construct Stop0.5/BS1 set up an addition reaction as follows
1 ul of purified PCR product
1 ul of each stop oligo
22 ul of water
25 ul of Phusion PCR MasterMix
run 25-cycle PCR according to the following protocol
................................................
95 °C/ 5 min
................................................
95 °C/ 30 sec
65 °C/ 45 sec
72 °C/ 50 sec
................................................ (25x)
72 °C/ 5 min
................................................
4 °C/ forever
................................................
Gel Electrophoresis
PCR products from the 25 cycle PCR were analyzed on a Gel. Therefor, 5 ul of PCR product were mixed with 1 ul of 6x-loading dye and loaded on a 1.5 % agarose gel. In addition, 2.5 ul of the 7-cycle PCR (07/21/2010) and 12-cycle PCR were loaded as well. The gel ran at 135 V for 55 min.
22/07/2010
the PCR products from the previous days' raPCR were loaded on a 1 % agarose gel and the brightest 200, 400, 700 and 1400 bp bands were gel extracted using the Qiagen Gel Extraction Kit (elution in 32 ul of nuclease free water)
Vector pSB1AC3 (500 ng) and the gel extraction products were digested with EcoRI and SpeI according to the standard digestion protocol
PCR purification was performed for the digested raPCR products and a gel-extraction was performed for the ~ 3 kb vector band of pSB1AC3 (elution in 32 ul of nuclease-free water each)
12 ligation reactions were performed using 5 or 10 ul of insert and 5 ul of vector in each reaction; standard ligation was performed
23/07/2010
transformation of ligation product (previous day) into E. coli Top10 cells according to the standard transformation protocol
26/07/2010
of 21 colonies, 7 of each hsa-mir-886_3p 200, 400 and 700 bp band cloning product and inocculation of Miniprep LB cultures
single binding site synthesis
In order to construct a database of synthetic single binding sites, we developed and optimized a standardized single binding site synthesis protocol;
The band at ~60 bp shows the synthesized single binding sites. For mir-886 and mir-122 there were some unspecific products or multimers obtained.
Protocol 1:
set up the following reaction
1 ul of each single binding site oligo (100 uM) has-mir-886-3p, hsa-mir-886-3p(perf), hsa-mir-769-5p, hsa-mir-769-5p(perf), hsa-mir-122, hsa-mir-122(perf)
1 ul of SBSSSS_EcoRI primer (reverse strand synthesis primer)
23 ul of water
25 ul of Phusion PCR MasterMix
PCR protocol:
................................................
95 °C/ 5 min
................................................
95 °C/ 30 sec
60 °C/ 45 sec
72 °C/ 30 sec
................................................ (30x)
72 °C/ 5 min
................................................
4 °C/ forever
................................................
analysis of the result on a 2 % agarose gel, run 45 min @ 135 V
27/07/2010
The 200, 400 and 1000 bp cloning products were analyzed in an analytic PCR. Only The very first sample of the 200 bp product seems to be positive, as it shows a higher band (~ 300 bp) compared to the religated vector product band at ~ 200 bp
Miniprep of the LB cultures inocculated the previous day
analytic PCR for detecting positive clones was set up according to the following protocol
3 ul of Miniprep product
0.5 ul of each standard sequencing primer VF and VR
21 ul of water
25 ul of PCR Phusion Mastermix
PCR protocol:
................................................
95 °C/ 5 min
................................................
95 °C/ 30 sec
60 °C/ 45 sec
72 °C/ 30 sec
................................................ (30x)
72 °C/ 5 min
................................................
4 °C/ forever
................................................
PCR products were analyzed on a 1 % agarose gel, run for 45 min @ 135 V
Single Binding Site Synthesis - Optimization
Further optimization of the single binding site synthesis protocol
The unspecific or multimer side products were clearly reduced, but there is still sime unwanted, longer product in case of mir-122.
Protocol 2:
set up the exact same reaction as on the day before, but using an anealing temperature of 65 °C
analysis of the result on a 2 % agarose gel, run 45 min @ 135 V
31/07/2010
ready-to-use vector pSB1A3 from the registry (25 ng/ul) digestion with EcoRI/PstI (NEB Buffer EcoRI + BSA, volume: 40 ul)
PCR purification of digestion product
subsequent digestion with DpnI (NEB  Buffer 4, volume: 40 ul)
5 ul of digestion product was analyzed on a 1 % agarose gel; unfortunately, there was no digestion product detectable (data not shown)
transformation of pSB1A3-1 vector (Tom Knight, 2004) from the registry into Top10 cells according to the following protocol:
10 ul of nuclease-free water was pipetted into well C1 of Spring 2010 distribution plate 1
incubation for 10 min at room temperature
Top10 cells were thawn on ice (10 min) and 2 ul of DNA was added
incubation on ice for 15 min
1 min heat-shock at 42 °C
incubation on ince for 2 min
adding 500 ul of LB media; mixing
centrifugation of the cells (5 min, 6000 rpm)
supernatant was removed, pellet resuspended in 50 ul LB media and cells were plated on a LB-Amp plate
digestion of the 200 bp, 400 bp and 700 bp band from the previous day (corresponding to hsa-mir-886-3p microRNA binding site patterns constructed via diraPCR
no digestion product was detectable on the gel; therefor raPCR was repeated
diraPCR for constructing hsa-886-3p binding site patterns
the diraPCR was pipetted according to the following protocol in three repeats:
1 ul of each binding site oligo
0.5 ul of each stop oligo
25 ul of Phusion MasterMix
add water to final volume of 50 ul
12 cycle PCR was performed according to the standard miRACLE PCR protocol
after PCR purification the subsequent PCR reaction was pipetted as follows:
5, 10 or 20 ul of 12 cycle PCR prodcut
0.5 ul of each stop oligo
25 ul of Phusion MasterMix
add water to final volume of 50 ul
the 25 cycle PCR was performed according to the standard miRACLE PCR protocol
Retrieved from " http://2010.igem.org/Team:Heidelberg/Notebook/miMeasure/July "
Recent changes
What links here
Related changes
Special pages
My preferences
Printable version
Permanent link
Privacy policy
Disclaimers
