"
Page
Discussion
View source
History
teams
Log in
 
Team:Heidelberg/Parts/Characterization
From 2010.igem.org
Home
Team
Overview
Students
Instructors
Advisors
Institutes
Gallery
miDANCE
Acknowledgements
 
Project
Abstract
Introduction
Synthetic miRNA Kit
Capsid Shuffling
    Homology Based
    ViroBytes
miMeasure
in vivo Study
Summary & Outlook
References
Contributions
 
Parts
Parts
Characterization
 
Modeling
Introduction
miBEAT
miRockdown
miBS designer
mUTING
Descriptions
 
Notebook
Synthetic miRNA Kit
Capsid Shuffling
    Homology Based
    ViroBytes
miMeasure
Binding Site Design
in vivo Study
Modeling
Materials
Methods
Psychological Survey
Philosophical Reflection
Biosafety
 
Human Practices
Psychological Survey
Philosophical Reflection
 
Sponsors
Contents
1Characterization
1.1Characterization of promoters in tuning constructs in T-Rex cells
1.2Characterization of BBa_K337035 and BBa_K337036 in different cell lines
1.3Characterization of part binding sites (BBa_K337052, BBa_K337053, BBa_K337054)
1.4Characterization of synthetic single microRNA binding sites (BBa_K337055)
1.5Characterization of synthetic microRNA binding site patterns against endogenous miR122 3.1BBa_K337008 and 1.3 BBa_K337000
1.6Characterization of the bidirectional CMV promoter
1.7References
 
Characterization
We characterized intensively four different part groups:
the 8 engineered miTuner construct differing in the promoter combinations (SV40, RSV, CMV, CMV_TetO2) the cassettes on the parts are expressed from (<partinfo>BBa_K337036</partinfo>, <partinfo>BBa_K337038</partinfo>, <partinfo>BBa_K337032</partinfo>, <partinfo>BBa_K337035</partinfo>, <partinfo>BBa_K337040</partinfo>, <partinfo>BBa_K337042</partinfo>, <partinfo>BBa_K337044</partinfo>, <partinfo>BBa_K337046</partinfo>)
the pSMB_miMeasure Standard Plasmid (<partinfo>BBa_K337049</partinfo>)
a whole variety of different microRNA binding sites out of which we submitted 6 interesting candidates to the registry. (hsa-mir122 and shRNAmir_hAAT binding sites, parts <partinfo>BBa_K337052</partinfo>, <partinfo>BBa_K337053</partinfo>, <partinfo>BBa_K337054</partinfo>, <partinfo>BBa_K337055</partinfo>, <partinfo>BBa_K337056</partinfo>, <partinfo>BBa_K337057</partinfo>)
MicroRNA binding site patterns consisting of more than 1 single perfect or imperfect binding sites for microRNA hsa-mir122 (<partinfo>BBa_K337040</partinfo>, <partinfo>BBa_K337040</partinfo>, <partinfo>BBa_K337040</partinfo>, <partinfo>BBa_K337040</partinfo>, <partinfo>BBa_K337040</partinfo>, <partinfo>BBa_K337040</partinfo>)
Characterization of promoters in tuning constructs in T-Rex cells
Dual Luciferase Assay
In order to test for promoter efficiency and to check whether the miRNA kit assembly works fine 50ng of each construct with different promoter set-ups (table 1) were transfected into HEK 293 T-REx cells and other cell lines HEK, HeLa, Huh7 in 96-well plate format using FuGENE transfection reagent. As every construct is expressing firefly luciferase (luc2) and renilla luciferase (hRluc) at the same time the setup allows is unaffected by transfection efficiency and cell number. Each sample was transfected and measured by Dual luciferase assay in 8 replicates. As by this time no shRNA has been cloned into plasmid no knock-down of luc2 is expected and the different expression efficiencies allow for characterization of the different promoters. BBa_K337032 leads to a relative luciferase unit (RLU) of luc2 to hRluc expression of 6 RLU. BBa_K337035 and BBa_K337035 are showing a comparable expression of 12 - 13 RLU, which is in line with the knowledge that both luciferases are driven by the CMV promoter.  Hek 293 T-Rex cells stably express the Tet repressor thus allows us to observe very efficient repression of Firefly luciferse expression if a CMV-TetO2 promoter is driving luc2 ( BBa_K337038 and BBa_K337046 ). BBa_K337040 transfection into Hek T-Rex cells results in an expression of 15 RLU. BBa_K337042 and BBa_K337044 are constructed in a way that luc2 is driven by the CMV promoter and hRluc is driven by the RSV promoter and show a comparable expression of 17-20 RLU. This leads to the conclusion that the CMV promoter shows comparable expression to the RSV promoter in Hek T-Rex cell lines.
Table 1
part
promoter driving luc2 (Firefly)
promoter driving (Renilla)
promoter driving shRNA expression
BBa_K337032
RSV
CMV
SV40
BBa_K337035
CMV
CMV
SV40
BBa_K337036
CMV
CMV
RSV
BBa_K337038
CMV TetO2
CMV
RSV
BBa_K337040
RSV
RSV
SV40
BBa_K337042
CMV
RSV
SV40
BBa_K337044
CMV
RSV
RSV
BBa_K337046
CMV TetO2
RSV
RSV
Promoter strength characterization of tuning constructs in HEK 293 T-REx cell line
↑
Characterization of BBa_K337035 and BBa_K337036 in different cell lines
Promoter strength characterization in different cell lines
If BBa_K337035 and BBa_K337036 are transfected into different cell lines it is obvious that Hek293T cells are the easiest to transfect with both constructs an expression of 17-22 RLU is to be measured. Hek T-Rex cells are showing and expression level of 12 RLU of both constructs. Hela cells are also showing constant expression levels of  8 RLU with both constructs. A rather low expression  of 2RLU is to be seen by transfecting the 2 constructs in Huh7 cells. This might be due to low transfection efficiency of this cell line in general. All together it is to say that BBa_K337035 and BBa_K337036 show comparable expression.
Characterization of part binding sites ( BBa_K337052 , BBa_K337053 , BBa_K337054 )
Regulation of gene expression can be achieved by fusing miRNA binding sites into the 3'UTR of a GOI. In case a referring shRNA miR is endogenously present or co-expressed, the GOI is knocked down. Strength of regulation thereby depends on binding site properties. We are able to tune gene expression linearly over a broad range. This is a first proof of principle for various miRNA-mRNA interaction in vitro. Therefore, we transfected HeLa cells in principle with our pSMB_miTuner Plasmid HD3 . It turned out, that no obvious effect of different binding sites on reporter gene expression could be measured (data not shown). We assume that the RSV promoter driving the shRNA miR is too weak for tight regulation of the referring binding site behind the GOI. Only if a sufficient amount of shRNA miR binds to its target, translation is significantly repressed. Thus, we expressed the shRNA miR from a separate plasmid which was always co-transfected with the original tuning construct. The reporter genes - i. e. Luc2 and hRluc - were also expressed from separate plasmids to get a reference as well as a transfection control. This was achieved by co-transfection the tuning construct with corresponding shRNA miRhaat and as a control a miRNA w/o binding sites in the target 3'UTR. The experiment was done in a 96-well plate by plating 5000 Hela cells/well 24h before transfection. Transfection was done using Fugene transfection reagent. 2.5ng of tuning construct were co-transfected with the shRNA miR construct of a concentration of 25 ng (1:10). Then, we conducted a Dual Luciferase Assay for quantification of gene expression. The data preciously shows a tuned expression from almost 0% to 100% (Fig. 1, Fig. 2). Lowest expression refers to complete knockdown through cloning of perfect binding sites into the 3'UTR  to the reporter gene(always green bar on the left hand side of the figures). 100% refers to ordinary expression from a construct without binding sites (always orange column on the right hand side of the figures). In presence of the specific shRNA miR, gene expression was mediated to various levels through interactions with the different imperfect binding sites. Whereas, when an unspecific shRNA miR was expressed, gene expression remained unaffected (see raw data below). The latter aspect reveals, that the binding sites were correctly designed, since they seem to interact specifically with a referring shRNA miR. The constructs were tested in two different backbones: pBS_U6 and pBS_H1. Both are in viral context, meaning that they contain inverted terminal repeats (ITRs). The constructs can be packed into the capsid of an adeno-associated virus (AAV). We chose the data obtained by the construct with the U6 promoter as this promoter is more efficient than the H1 promoter, ensuring that the system is saturated and ensuring that the data is reproducible. The same constructs were also used for virus production to infect cells even more efficiently as compared to transfections. Because of the significant data, we decided to inject the viruses into mice to see the tuning effect also in vivo . The pBS_H1 construct should be used for mice injections since the expressed shRNA miR against human alpha-1-antitrypsine (hAAT) is cytotoxic in higher concentrations. The pBS_H1 backbone leads to moderate expression ranges, still obviously showing the tuning effect.
Figure 1: Tuning of gene expression through different imperfect shRNA miR binding sites in pBS_SV40_Luc2 construct cotransfected with a reference renilla construct. Gene expression quantified via dual luciferase assay for constructs containing different imperfect binding sites for shhAAT. The shhAAT was expressed from a pSB_U6 plasmid
Figure 2: Tuning of gene expression through different imperfect shRNA miR binding sites in pBS_SV40 Luc2 double transfected with a reference renilla construct. The shRNA_hAAT construct was expressed from a pSB_H1 construct.
Strikingly, the order of constructs in terms of knockdown for the imperfect binding sites is similar. The perfect binding site BBa_K337052 shows knockdown of about 96%. point mut 10 (1), point mut 10 (2) and point mut 11 BBa_K337053 (2) always show strong knockdown, whereas bulge 16-18 BBa_K337054 , only seed and bulge 9-12(2) show only loose down-regulation. Consulting the binding site sequences, the weak knockdown can be addressed to bulges in the supplementary region or complete lack of the 3' region of the binding site. Still high strength could be maintained due to only single nucleotide exchanges in the central region of the binding site.
Characterization of synthetic single microRNA binding sites ( BBa_K337055 )
Binding sites for miR122 were experimentally characterized by cloning them into psiCHECK-2 backbone (Promega). Dual luciferase assay was conducted using this time Renilla Luciferase as a reporter and the other luciferase as a reference for normalization. Figure 3 shows again a broad range of regulation depending on binding site sequence properties. BBa_K337055 (mut 99) is the construct with the perfect binding site and leads to an knock-down percentage of 96%. BBa_K337056 (mut 7) is an imperfect binding site which leads to a knockdown percentage of 64%. BBa_K337057 (mut 111) is an imperfect binding site with a knockdown efficiency of 24%. All this contributes to a real tuning effect by introducing binding sites with introduced mismatches following the rational design protocol.
Figure 3: Tuning of gene expression through different imperfect miR122 binding sites in psiCHECK-2. Construct was transfected into HeLa cells together with an plasmid expressing miR122. Control without binding site was used for normalization.
Characterization of synthetic microRNA binding site patterns against endogenous miR122 3.1 BBa_K337008 and 1.3 BBa_K337000
We used the miMeasure plasmid with the BB-2 standard to characterise binding sites for the endogenously expressed miR122. We cloned two different synthetic microRNA binding site patterns into our miMeasure construct plasmid: 3.1 BBa_K337008 and 1.3 BBa_K337000. As the binding site is inserted downstream of green fluorescent protein (EGFP), a regulation of EGFP expression is to be expected. miMeasure normalizes knockdown of EGFP to the unregulated blue fluorescent protein (EBFP2). By calculating the ratio of EGFP to EBFP2 we determined the knockdown percentage characteristic of the binding site patterns.
We tested these constructs in four different set-ups with three different cell lines:
• HeLa cells, which do not express miR122 endogenously
• HeLa cells cotransfected with miR122 to mimic endogenous expression
• HuH7 cells, which are liver cells known to express miR122
• HepG2 cells, liver cells known to express low amounts of miR122 (Douglas, 2010)
EGFP to EBFP2 ratios were measured with flow cytometry and microscopy. Measurement results for the four cell lines and the binding sites 3.1 BBa_K337008 and 1.3 BBa_K337000 are shown in Fig. 4 for all four cell line setups. As expected, no down regulation of EGFP expression was measured in HeLa cells due to the lack of miR122 therein. This serves as a control for the design of our binding sites as it is clear that they do not cross-react with other endogenously expressed miRNAs but are specific for miR122. Similarly, no down regulation was observed in HepG2 cells. The levels of miR122 expression in those cells are reportedly reduced by 99.5&, therefore also serving as a negative control. Both the construct 1.3 (containing two perfect binding sites with the extra 10bp spacer in between) and 3.1 (containing 3 perfect binding sites) lead to an increase in downregulation effect of EGFP in Huh7 cells and in HeLa cells cotransfected with miR122 in comparison to the single perfect binding site.
Figure 4: miMeasure in four different set-ups with three different cell lines The construct containing synthetic microRNA binding site patterns against endogenous miR122 transfected into different cell lines. The EGFP/EBFP2 ratio for the construct containing no binding site was set to one.
Characterization of the bidirectional CMV promoter
The bidirectional CMV promoter BBa_K337017 is a new standard part out of our miMeasure construct .  The promoter was used to express two reporter genes with a comparable transcription rate. Referring to the figures below, this is the case.
Left panel: EGFP channel. Middle panel: EBFP channel. Right panel: merge of both the others.
EBFP intensities against EGFP intensities. Correlation coefficient is given.
References
Douglas D:Small Molecule Modiﬁers of MicroRNA miR-122 Function for the Treatment of Hepatitis C Virus Infection and Hepatocellular Carcinoma. JACS. 2010 May 15; 132(23):7976-81.
↑
Retrieved from " http://2010.igem.org/Team:Heidelberg/Parts/Characterization "
Recent changes
What links here
Related changes
Special pages
My preferences
Printable version
Permanent link
Privacy policy
Disclaimers
