<HTML xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en" dir="ltr">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki  ltr ns-0 ns-subject page-Team_Heidelberg_Notebook_miRNA_Kit_September"><DIV id="globalWrapper"><DIV id="top-section"><DIV id="p-logo"><A href="/Main_Page" title="Main Page">&quot;
	    </A></DIV><DIV id="menubar" class="left-menu noprint"><UL><LI class="selected"><A href="/Team:Heidelberg/Notebook/miRNA_Kit/September">Page               </A></LI><LI class="new"><A href="/wiki/index.php?title=Talk:Team:Heidelberg/Notebook/miRNA_Kit/September&amp;action=edit&amp;redlink=1">Discussion               </A></LI><LI><A href="/wiki/index.php?title=Team:Heidelberg/Notebook/miRNA_Kit/September&amp;action=edit">View source               </A></LI><LI><A href="/wiki/index.php?title=Team:Heidelberg/Notebook/miRNA_Kit/September&amp;action=history">History               </A></LI><LI style="color:#808080;cursor:default">teams</LI></UL></DIV><DIV class="right-menu noprint" id="menubar"><UL><LI id="pt-login"><A href="/wiki/index.php?title=Special:UserLogin&amp;returnto=Team:Heidelberg/Notebook/miRNA_Kit/September" title="You are encouraged to log in; however, it is not mandatory [o]" accesskey="o">Log in</A></LI></UL></DIV><DIV id="search-controls" class="noprint"><FORM action="/Special:Search" id="searchform"> </FORM></DIV></DIV><DIV id="content"><H1 class="firstHeading">Team:Heidelberg/Notebook/miRNA Kit/September</H1><DIV id="bodyContent"><H3 id="siteSub" class="noprint">From 2010.igem.org</H3><DIV id="header"><DIV id="menu"><UL id="nav"><LI id="menu_home"><A class="aHome" href="https://2010.igem.org/Team:Heidelberg">Home</A></LI><LI id="menu_team"><A class="aTeam" href="https://2010.igem.org/Team:Heidelberg/Team">Team</A><TABLE class="menutable" cellpadding="0" cellspacing="0"><TBODY><TR><TD class="lefttdtopteam"><A class="aTeamOverview" href="https://2010.igem.org/Team:Heidelberg/Team">Overview</A><A class="aTeamMembers" href="https://2010.igem.org/Team:Heidelberg/Team/Members">Students</A><A class="aTeamMembers" href="https://2010.igem.org/Team:Heidelberg/Team/Members#Instructors">Instructors</A><A class="aTeamMembers" href="https://2010.igem.org/Team:Heidelberg/Team/Members#Advisors">Advisors</A><A class="aInstitutes" href="https://2010.igem.org/Team:Heidelberg/Team/Institutes">Institutes</A><A class="aGallery" href="https://2010.igem.org/Team:Heidelberg/Team/Gallery">Gallery</A><A class="aDance" href="https://2010.igem.org/Team:Heidelberg/midance">miDANCE</A><A class="amithanks" href="https://2010.igem.org/Team:Heidelberg/Team/miThanks">Acknowledgements</A></TD><TD class="righttdtop"> </TD></TR></TBODY></TABLE></LI><LI id="menu_project"><A class="aProject" href="https://2010.igem.org/Team:Heidelberg/Project">Project</A><TABLE class="menutable" cellpadding="0" cellspacing="0"><TBODY><TR><TD class="lefttdtopproject"><A class="aAbstract" href="https://2010.igem.org/Team:Heidelberg/Project">Abstract</A><A class="aIntroduction" href="https://2010.igem.org/Team:Heidelberg/Project/Introduction">Introduction</A><A class="aProject_miRNA_Kit" href="https://2010.igem.org/Team:Heidelberg/Project/miRNA_Kit">Synthetic miRNA Kit</A><A class="aPro_Capsid_Shuffling" href="https://2010.igem.org/Team:Heidelberg/Project/Capsid_Shuffling">Capsid Shuffling</A><A class="aPro_Homology" href="https://2010.igem.org/Team:Heidelberg/Project/Capsid_Shuffling/Homology_Based">    Homology Based</A><A class="aPro_Virobytes" href="https://2010.igem.org/Team:Heidelberg/Project/Capsid_Shuffling/ViroBytes">    ViroBytes</A><A class="aPro_miMeasure" href="https://2010.igem.org/Team:Heidelberg/Project/miMeasure">miMeasure</A><A class="aPro_Mouse" href="https://2010.igem.org/Team:Heidelberg/Project/Mouse_Infection"><I>in vivo</I> Study</A><A class="aPro_Summary" href="https://2010.igem.org/Team:Heidelberg/Project/Summary">Summary &amp; Outlook</A><A class="aPro_Ref" href="https://2010.igem.org/Team:Heidelberg/Project/References">References</A><A class="aPro_Ref" href="https://2010.igem.org/Team:Heidelberg/Project/Contributions">Contributions</A></TD><TD class="righttdtop"> </TD></TR></TBODY></TABLE></LI><LI id="menu_parts"><A class="aParts" href="https://2010.igem.org/Team:Heidelberg/Parts">Parts</A><TABLE class="menutable" cellpadding="0" cellspacing="0"><TBODY><TR><TD class="lefttdtop"><A class="aparts_ger" href="https://2010.igem.org/Team:Heidelberg/Parts">Parts</A><A class="aparts_char" href="https://2010.igem.org/Team:Heidelberg/Parts/Characterization">Characterization</A></TD><TD class="righttdtop"> </TD></TR></TBODY></TABLE></LI><LI id="menu_modeling"><A class="aModeling" href="https://2010.igem.org/Team:Heidelberg/Modeling">Modeling</A><TABLE class="menutable" cellpadding="0" cellspacing="0"><TBODY><TR><TD class="lefttdtop"><A class="amode_over" href="https://2010.igem.org/Team:Heidelberg/Modeling">Introduction</A><A class="amiGUI" href="https://2010.igem.org/Team:Heidelberg/Modeling/miGUI">miBEAT</A><A class="amirockdown" href="https://2010.igem.org/Team:Heidelberg/Modeling/miRockdown">miRockdown</A><A class="amibsdesigner" href="https://2010.igem.org/Team:Heidelberg/Modeling/miBSdesigner">miBS designer</A><A class="amispec" href="https://2010.igem.org/Team:Heidelberg/Modeling/miSpec">mUTING</A><A class="amodelset" href="https://2010.igem.org/Team:Heidelberg/Modeling/trainingset">Descriptions</A></TD><TD class="righttdtop"> </TD></TR></TBODY></TABLE></LI><LI id="menu_notebook"><A class="aNotebook" href="https://2010.igem.org/Team:Heidelberg/Notebook">Notebook</A><TABLE class="menutable" cellpadding="0" cellspacing="0"><TBODY><TR><TD class="lefttdtop"><A class="aNote_miRNA_kit" href="https://2010.igem.org/Team:Heidelberg/Notebook/miRNA_Kit">Synthetic miRNA Kit</A><A class="aNote_Capsid_Shuffling">Capsid Shuffling</A><A class="aNote_Homology" href="https://2010.igem.org/Team:Heidelberg/Notebook/Capsid_Shuffling/Homology_Based">    Homology Based</A><A class="aNote_Virobytes" href="https://2010.igem.org/Team:Heidelberg/Notebook/Capsid_Shuffling/ViroBytes">    ViroBytes</A><A class="aNote_miMeasure" href="https://2010.igem.org/Team:Heidelberg/Notebook/miMeasure">miMeasure</A><A class="aNote_BSdesign" href="https://2010.igem.org/Team:Heidelberg/Notebook/BSDesign">Binding Site Design </A><A class="aNote_Mouse" href="https://2010.igem.org/Team:Heidelberg/Notebook/Mouse_Infection"><I>in vivo</I> Study</A><A class="aNote_Modeling" href="https://2010.igem.org/Team:Heidelberg/Notebook/Modeling">Modeling</A><A class="aNote_Material" href="https://2010.igem.org/Team:Heidelberg/Notebook/Material">Materials</A><A class="aNote_Methods" href="https://2010.igem.org/Team:Heidelberg/Notebook/Methods">Methods</A><A class="aNote_Psycho" href="https://2010.igem.org/Team:Heidelberg/Human_Practices/Psychology/Notebook">Psychological Survey</A><A class="aNote_Philo" href="https://2010.igem.org/Team:Heidelberg/Human_Practices/Philosophy/Part2">Philosophical Reflection</A><A class="aNote_Biosafety" href="https://2010.igem.org/Team:Heidelberg/Notebook/Biosafety">Biosafety</A></TD><TD class="righttdtop"> </TD></TR></TBODY></TABLE></LI><LI id="menu_human_practices"><A class="aHuman_Practices" href="https://2010.igem.org/Team:Heidelberg/Human Practices">Human Practices</A><TABLE class="menutable" cellpadding="0" cellspacing="0"><TBODY><TR><TD class="lefttdtop"><A class="aPsycho" href="https://2010.igem.org/Team:Heidelberg/Human Practices/Psychology">Psychological Survey</A><A class="aPhilo" href="https://2010.igem.org/Team:Heidelberg/Human Practices/Philosophy">Philosophical Reflection</A></TD><TD class="righttdtop"> </TD></TR></TBODY></TABLE></LI><LI id="menu_support"><A class="aSupport" href="https://2010.igem.org/Team:Heidelberg/Support">Sponsors</A></LI></UL></DIV></DIV><TABLE cellpadding="5" cellspacing="0" align="center" style="text-align: center; color:#009be1; border: 1.5px solid #000000;"><TBODY><TR><TH colspan="7" style="background:#f09600;"><A href="https://2010.igem.org/Team:Heidelberg/Notebook/miRNA_Kit/August" class="external text" rel="nofollow"><FONT color="white">August</FONT></A></TH></TR><TR style="background:#f09600; color:white"><TD width="20pt"><B>M</B></TD><TD width="20pt"><B>T</B></TD><TD width="20pt"><B>W</B></TD><TD width="20pt"><B>T</B></TD><TD width="20pt"><B>F</B></TD><TD width="20pt"><B>S</B></TD><TD width="20pt"><B>S</B></TD></TR><TR style="background:#f2f2f2; color:#009be1"><TD><B>1</B></TD></TR><TR style="background:#f2f2f2; color:#009be1"><TD><B>2</B></TD><TD><B>3</B></TD><TD><B>4</B></TD><TD><B>5</B></TD><TD><B>6</B></TD><TD><B>7</B></TD><TD><B>8</B></TD></TR><TR style="background:#f2f2f2; color:#009be1"><TD><B>9</B></TD><TD><B>10</B></TD><TD><B>11</B></TD><TD><B>12</B></TD><TD><B>13</B></TD><TD><B>14</B></TD><TD><B>15</B></TD></TR><TR style="background:#f2f2f2; color:#009be1"><TD><B>16</B></TD><TD><B>17</B></TD><TD><B>18</B></TD><TD><B>19</B></TD><TD><B>20</B></TD><TD><B><A href="https://2010.igem.org/Team:Heidelberg/Notebook/miRNA_Kit/August#21/08/2010" class="external text" rel="nofollow">21</A></B></TD><TD><B><A href="https://2010.igem.org/Team:Heidelberg/Notebook/miRNA_Kit/August#22.2F08.2F2010" class="external text" rel="nofollow">22</A></B></TD></TR><TR style="background:#f2f2f2; color:#009be1"><TD><B><A href="https://2010.igem.org/Team:Heidelberg/Notebook/miRNA_Kit/August#23.2F08.2F2010" class="external text" rel="nofollow">23</A></B></TD><TD><B><A href="https://2010.igem.org/Team:Heidelberg/Notebook/miRNA_Kit/August#24.2F08.2F2010" class="external text" rel="nofollow">24</A></B></TD><TD><B><A href="https://2010.igem.org/Team:Heidelberg/Notebook/miRNA_Kit/August#25.2F08.2F2010" class="external text" rel="nofollow">25</A></B></TD><TD><B><A href="https://2010.igem.org/Team:Heidelberg/Notebook/miRNA_Kit/August#26.2F08.2F2010" class="external text" rel="nofollow">26</A></B></TD><TD><B><A href="https://2010.igem.org/Team:Heidelberg/Notebook/miRNA_Kit/August#27.2F08.2F2010" class="external text" rel="nofollow">27</A></B></TD><TD><B><A href="https://2010.igem.org/Team:Heidelberg/Notebook/miRNA_Kit/August#28.2F08.2F2010" class="external text" rel="nofollow">28</A></B></TD><TD><B><A href="https://2010.igem.org/Team:Heidelberg/Notebook/miRNA_Kit/August#29.2F08.2F2010" class="external text" rel="nofollow">29</A></B></TD></TR><TR style="background:#f2f2f2; color:#009be1"><TD><B><A href="/wiki/index.php?title=Igem2010/Main/synthetic_miR_Kit/August&amp;action=edit&amp;redlink=1" class="new" title="Igem2010/Main/synthetic miR Kit/August (page does not exist)">30</A></B></TD><TD><B><A href="/wiki/index.php?title=Igem2010/Main/synthetic_miR_Kit/August&amp;action=edit&amp;redlink=1" class="new" title="Igem2010/Main/synthetic miR Kit/August (page does not exist)">31</A></B></TD></TR></TBODY></TABLE><TABLE cellpadding="5" cellspacing="0" align="center" style="text-align: center; color:#78b41e; border: 1.5px solid #333333;"><TBODY><TR><TH colspan="7" style="background:#009be1;"><A href="https://2010.igem.org/Team:Heidelberg/Notebook/miRNA_Kit/September" class="external text" rel="nofollow"><FONT color="#ffecba">September</FONT></A></TH></TR><TR style="background:#009be1; color:white"><TD width="20pt"><B>M</B></TD><TD width="20pt"><B>T</B></TD><TD width="20pt"><B>W</B></TD><TD width="20pt"><B>T</B></TD><TD width="20pt"><B>F</B></TD><TD width="20pt"><B>S</B></TD><TD width="20pt"><B>S</B></TD></TR><TR style="background:#f2f2f2; color:#78b41e"><TD><B><A href="https://2010.igem.org/Team:Heidelberg/Notebook/miRNA_Kit/September#01.2F09.2F2010" class="external text" rel="nofollow">1</A></B></TD><TD><B><A href="https://2010.igem.org/Team:Heidelberg/Notebook/miRNA_Kit/September#02.2F09.2F2010" class="external text" rel="nofollow">2</A></B></TD><TD><B><A href="https://2010.igem.org/Team:Heidelberg/Notebook/miRNA_Kit/September#03.2F09.2F2010" class="external text" rel="nofollow">3</A></B></TD><TD><B><A href="https://2010.igem.org/Team:Heidelberg/Notebook/miRNA_Kit/September#04.2F09.2F2010" class="external text" rel="nofollow">4</A></B></TD><TD><B><A href="https://2010.igem.org/Team:Heidelberg/Notebook/miRNA_Kit/September#05.2F09.2F2010" class="external text" rel="nofollow">5</A></B></TD></TR><TR style="background:#f2f2f2; color:#78b41e"><TD><B><A href="https://2010.igem.org/Team:Heidelberg/Notebook/miRNA_Kit/September#06.2F09.2F2010" class="external text" rel="nofollow">6</A></B></TD><TD><B><A href="https://2010.igem.org/Team:Heidelberg/Notebook/miRNA_Kit/September#07.2F09.2F2010" class="external text" rel="nofollow">7</A></B></TD><TD><B><A href="https://2010.igem.org/Team:Heidelberg/Notebook/miRNA_Kit/September#08.2F09.2F2010" class="external text" rel="nofollow">8</A></B></TD><TD><B><A href="https://2010.igem.org/Team:Heidelberg/Notebook/miRNA_Kit/September#09.2F09.2F2010" class="external text" rel="nofollow">9</A></B></TD><TD><B><A href="https://2010.igem.org/Team:Heidelberg/Notebook/miRNA_Kit/September#10.2F09.2F2010" class="external text" rel="nofollow">10</A></B></TD><TD><B><A href="https://2010.igem.org/Team:Heidelberg/Notebook/miRNA_Kit/September#11.2F09.2F2010" class="external text" rel="nofollow">11</A></B></TD><TD><B><A href="https://2010.igem.org/Team:Heidelberg/Notebook/miRNA_Kit/September#12.2F09.2F2010" class="external text" rel="nofollow">12</A></B></TD></TR><TR style="background:#f2f2f2; color:#78b41e"><TD><B><A href="https://2010.igem.org/Team:Heidelberg/Notebook/miRNA_Kit/September#13.2F09.2F2010" class="external text" rel="nofollow">13</A></B></TD><TD><B><A href="https://2010.igem.org/Team:Heidelberg/Notebook/miRNA_Kit/September#14.2F09.2F2010" class="external text" rel="nofollow">14</A></B></TD><TD><B><A href="https://2010.igem.org/Team:Heidelberg/Notebook/miRNA_Kit/September#15.2F09.2F2010" class="external text" rel="nofollow">15</A></B></TD><TD><B><A href="https://2010.igem.org/Team:Heidelberg/Notebook/miRNA_Kit/September#16.2F09.2F2010" class="external text" rel="nofollow">16</A></B></TD><TD><B><A href="https://2010.igem.org/Team:Heidelberg/Notebook/miRNA_Kit/September#17.2F09.2F2010" class="external text" rel="nofollow">17</A></B></TD><TD><B><A href="https://2010.igem.org/Team:Heidelberg/Notebook/miRNA_Kit/September#18.2F09.2F2010" class="external text" rel="nofollow">18</A></B></TD><TD><B><A href="https://2010.igem.org/Team:Heidelberg/Notebook/miRNA_Kit/September#19.2F09.2F2010" class="external text" rel="nofollow">19</A></B></TD></TR><TR style="background:#f2f2f2; color:#78b41e"><TD><B><A href="https://2010.igem.org/Team:Heidelberg/Notebook/miRNA_Kit/September#20.2F09.2F2010" class="external text" rel="nofollow">20</A></B></TD><TD><B><A href="https://2010.igem.org/Team:Heidelberg/Notebook/miRNA_Kit/September#21.2F09.2F2010" class="external text" rel="nofollow">21</A></B></TD><TD><B><A href="https://2010.igem.org/Team:Heidelberg/Notebook/miRNA_Kit/September#22.2F09.2F2010" class="external text" rel="nofollow">22</A></B></TD><TD><B><A href="https://2010.igem.org/Team:Heidelberg/Notebook/miRNA_Kit/September#23.2F09.2F2010" class="external text" rel="nofollow">23</A></B></TD><TD><B><A href="https://2010.igem.org/Team:Heidelberg/Notebook/miRNA_Kit/September#24.2F09.2F2010" class="external text" rel="nofollow">24</A></B></TD><TD><B><A href="https://2010.igem.org/Team:Heidelberg/Notebook/miRNA_Kit/September#25.2F09.2F2010" class="external text" rel="nofollow">25</A></B></TD><TD><B><A href="https://2010.igem.org/Team:Heidelberg/Notebook/miRNA_Kit/September#26.2F09.2F2010" class="external text" rel="nofollow">26</A></B></TD></TR><TR style="background:#f2f2f2; color:#78b41e"><TD><B><A href="https://2010.igem.org/Team:Heidelberg/Notebook/miRNA_Kit/September#27.2F09.2F2010" class="external text" rel="nofollow">27</A></B></TD><TD><B><A href="https://2010.igem.org/Team:Heidelberg/Notebook/miRNA_Kit/September#28.2F09.2F2010" class="external text" rel="nofollow">28</A></B></TD><TD><B><A href="https://2010.igem.org/Team:Heidelberg/Notebook/miRNA_Kit/September#29.2F09.2F2010" class="external text" rel="nofollow">29</A></B></TD><TD><B><A href="https://2010.igem.org/Team:Heidelberg/Notebook/miRNA_Kit/September#22.2F30.2F2010" class="external text" rel="nofollow">30</A></B></TD></TR><TR style="background:#f2f2f2; color:#78b41e"><TD colspan="7"><SPAN style="color:#ffffff">-</SPAN></TD></TR></TBODY></TABLE><TABLE cellpadding="5" cellspacing="0" align="center" style="text-align: center; color:#f09600; border: 1.5px solid #333333;"><TBODY><TR><TH colspan="7" style="background:#78b41e;"><A href="https://2010.igem.org/Team:Heidelberg/Notebook/miRNA_Kit/October" class="external text" rel="nofollow"><FONT color="white">October</FONT></A></TH></TR><TR style="background:#78b41e; color:white"><TD width="20pt"><B>M</B></TD><TD width="20pt"><B>T</B></TD><TD width="20pt"><B>W</B></TD><TD width="20pt"><B>T</B></TD><TD width="20pt"><B>F</B></TD><TD width="20pt"><B>S</B></TD><TD width="20pt"><B>S</B></TD></TR><TR style="background:#f2f2f2; color:#78b41e"><TD><B><A href="https://2010.igem.org/Team:Heidelberg/Notebook/miRNA_Kit/October#01.2F10.2F2010" class="external text" rel="nofollow">1</A></B></TD><TD><B><A href="https://2010.igem.org/Team:Heidelberg/Notebook/miRNA_Kit/October#02.2F10.2F2010" class="external text" rel="nofollow">2</A></B></TD><TD><B><A href="https://2010.igem.org/Team:Heidelberg/Notebook/miRNA_Kit/October#03.2F10.2F2010" class="external text" rel="nofollow">3</A></B></TD></TR><TR style="background:#f2f2f2; color:#f09600"><TD><B><A href="https://2010.igem.org/Team:Heidelberg/Notebook/miRNA_Kit/October#04.2F10.2F2010" class="external text" rel="nofollow">4</A></B></TD><TD><B><A href="https://2010.igem.org/Team:Heidelberg/Notebook/miRNA_Kit/October#05.2F10.2F2010" class="external text" rel="nofollow">5</A></B></TD><TD><B><A href="https://2010.igem.org/Team:Heidelberg/Notebook/miRNA_Kit/October#06.2F10.2F2010" class="external text" rel="nofollow">6</A></B></TD><TD><B><A href="https://2010.igem.org/Team:Heidelberg/Notebook/miRNA_Kit/October#07.2F10.2F2010" class="external text" rel="nofollow">7</A></B></TD><TD><B><A href="https://2010.igem.org/Team:Heidelberg/Notebook/miRNA_Kit/October#08.2F10.2F2010" class="external text" rel="nofollow">8</A></B></TD><TD><B><A href="https://2010.igem.org/Team:Heidelberg/Notebook/miRNA_Kit/October#09.2F10.2F2010" class="external text" rel="nofollow">9</A></B></TD><TD><B><A href="https://2010.igem.org/Team:Heidelberg/Notebook/miRNA_Kit/October#10.2F10.2F2010" class="external text" rel="nofollow">10</A></B></TD></TR><TR style="background:#f2f2f2; color:#f09600"><TD><B><A href="https://2010.igem.org/Team:Heidelberg/Notebook/miRNA_Kit/October#11.2F10.2F2010" class="external text" rel="nofollow">11</A></B></TD><TD><B><A href="https://2010.igem.org/Team:Heidelberg/Notebook/miRNA_Kit/October#13.2F10.2F2010" class="external text" rel="nofollow">13</A></B></TD><TD><B><A href="https://2010.igem.org/Team:Heidelberg/Notebook/miRNA_Kit/October#14.2F10.2F2010" class="external text" rel="nofollow">14</A></B></TD><TD><B><A href="https://2010.igem.org/Team:Heidelberg/Notebook/miRNA_Kit/October#15.2F09.2F2010" class="external text" rel="nofollow">15</A></B></TD><TD><B><A href="https://2010.igem.org/Team:Heidelberg/Notebook/miRNA_Kit/October#16.2F10.2F2010" class="external text" rel="nofollow">16</A></B></TD><TD><B><A href="https://2010.igem.org/Team:Heidelberg/Notebook/miRNA_Kit/October#17.2F10.2F2010" class="external text" rel="nofollow">17</A></B></TD><TD><B><A href="https://2010.igem.org/Team:Heidelberg/Notebook/miRNA_Kit/October#18.2F10.2F2010" class="external text" rel="nofollow">18</A></B></TD></TR><TR style="background:#f2f2f2; color:#f09600"><TD><B><A href="https://2010.igem.org/Team:Heidelberg/Notebook/miRNA_Kit/October#19/10/2010" class="external text" rel="nofollow">19</A></B></TD><TD><B>20</B></TD><TD><B>21</B></TD><TD><B>22</B></TD><TD><B>23</B></TD><TD><B>24</B></TD><TD><B>25</B></TD></TR><TR style="background:#f2f2f2; color:#f09600"><TD><B>26</B></TD><TD><B>27</B></TD><TD><B>28</B></TD><TD><B>29</B></TD><TD><B>30</B></TD></TR><TR style="background:#f2f2f2; color:#f09600"><TD colspan="7"><SPAN style="color:#ffffff">-</SPAN></TD></TR></TBODY></TABLE><P> </P></DIV><H3><SPAN class="mw-headline" id="01.2F09.2F2010">01/09/2010</SPAN></H3><DIV class="thumb tleft"><DIV class="thumbinner" style="width:352px;"><DIV class="thumbcaption">Gel 100901-1</DIV></DIV></DIV><DIV class="thumb tleft"><DIV class="thumbinner" style="width:352px;"><DIV class="thumbcaption">Gel 100901-2</DIV></DIV></DIV><UL><LI> 8 colonies of each plate (see <A href="/wiki/index.php?title=Igem2010/Main/synthetic_miR_Kit/August&amp;action=edit&amp;redlink=1" class="new" title="Igem2010/Main/synthetic miR Kit/August (page does not exist)"> previous day</A>) were picked and colony-PCR was performed. Furthermore, the colonies were used for inocculation of LB miniprep cultures. The numbers behind the construct name refer to vector type (1 = not SAP treated, 2 = dephosphorylated using SAP; see previous day). For hRluc (1300 bp band)), Luc2 (2300 bp band), CMV_TetO (1000 bp band) and the FRT (500 bp band) site, positive colonies could be observed (gel 100901-1 and 100901-2).
</LI><LI> PCR for the construction of the standardized Tet Repressor was performed. The PCR was set up using the Phusion HF MasterMix and standard PCR protocol was applied (annealing temperature 59 °C, 2 replicates). The result of the PCR was analyzed on a 1 % agarose gel (gel 100901-2, lower right side) and the right 700 bp band was clearly visible.
</LI><LI> Digestion of the TetR fragment with EcoRI/PstI was performed according to the standard digestion protocol and the fragment was subsequently ligated into pSB1C3 (SAP treated or non-treated). The ligation mix was transformed into Top10 cells.
</LI></UL><H3><SPAN class="mw-headline" id="02.2F09.2F2010">02/09/2010</SPAN></H3><DIV class="thumb tleft"><DIV class="thumbinner" style="width:352px;"><DIV class="thumbcaption">Gel 100902-1</DIV></DIV></DIV><DIV class="thumb tright"><DIV class="thumbinner" style="width:352px;"><DIV class="thumbcaption">Gel 100902-2</DIV></DIV></DIV><DIV class="thumb tright"><DIV class="thumbinner" style="width:352px;"><DIV class="thumbcaption">Gel 100902-3</DIV></DIV></DIV><UL><LI> 8 colonies were picked from plates 1 and 4 colonies from plates 2 from the previous days cloning and analyzed via colony PCR performed according to the <A href="https://2010.igem.org/Team:Heidelberg/Notebook/Methods#Colony_PCR" class="external text" rel="nofollow">PCR standard protocol</A>. The results of the colony PCR were analyzed on a 1 % agarose gel run for 1 h @ 100 V (gel 100902-1 and 100902-2). The following colonies were afterwards inocculated (LB miniprep cultures):
</LI></UL><DL><DD><UL><LI> TetR (1.4 and 1.6)
</LI><LI> shRNA6 (1.1, 1.4 and 1.8)
</LI><LI> shRNA7 (1.1, 1.5 and 1.6)
</LI><LI> shRNA8 (1.1, 1,2, 1,3)
</LI><LI> shRNA9 (1.2)
</LI></UL></DD></DL><UL><LI> The cultures inocculated the day before were mini prepped by applying a Qiagen Miniprep Kit. Afterwards, a test digestion with NotI was performed and the result was analyzed on a 1 % agarose gel, run for 35 min @ 100 V (gel 100902-3). According to the test digestion result, the following samples were send for sequencing @ GATC:
</LI></UL><DL><DD><UL><LI> CMV_TetO2 (.4, 1.6)
</LI><LI> FRT (1.1, 1.3)
</LI><LI> Luc2 (1.4, 1.6)
</LI><LI> hRluc (1.1, 1.3, 1.4)
</LI></UL></DD></DL><H3><SPAN class="mw-headline" id="03.2F09.2F2010">03/09/2010</SPAN></H3><DIV class="thumb tright"><DIV class="thumbinner" style="width:352px;"><DIV class="thumbcaption">Gel 100903-1</DIV></DIV></DIV><DIV class="thumb tright"><DIV class="thumbinner" style="width:352px;"><DIV class="thumbcaption">Gel 100903-2</DIV></DIV></DIV><UL><LI> Test digestion with NotI of the cloned constructs on the previous day (gel 100903-1). For the TetR constructs the expected 600 bp band and for the shRNA constructs, the expected 200 bp band were clearly visible. For  each construct, one sample was send for sequencing. Sequencing results were correct for all samples send for sequencing.
</LI><LI> Purification of single binding sites against shRNA6-10, miR-375/376/122. For each target, the perfect, 9-12 and 9-22 randomized binding sites were purified and analyzed on an agarose gel (100903-2). All bands were clean, with just a slight amount of side product for the 9-22 randomized binding sites.
</LI></UL><UL><LI> cloning of shRNAs into pcDNA5/FRT/TO
</LI></UL><H3><SPAN class="mw-headline" id="04.2F09.2F2010">04/09/2010</SPAN></H3><UL><LI> mini-prep of pcDNA5 with shRNAs 1, 2, 3, 5, 5* and 6 (each four colonies from two independent transformations)
</LI><LI> PCR of shRNA constructs (including shRNA 7, 8 and 9 as well)
</LI><LI> gel: fine, everywhere amplified sequences below 200 bp 
</LI></UL><LI> digestion of promising tuning constructs with BspEI and StuI
</LI><LI> gel: plasmids seem to be cut only once again (problem maybe BspEI), size is correct
</LI><LI> suggestion: another digestion on monday with SalI and KpnI, then: submission for sequencing
</LI><UL><LI> mini-prep of pcDNA5 with shRNAs 1, 2, 3, 5, 5* and 6 (each four colonies from two independent transformations)
</LI><LI> PCR of shRNA constructs (including shRNA 7, 8 and 9 as well)
</LI><LI> gel: fine, everywhere amplified sequences below 200 bp 
</LI></UL><LI> digestion of promising tuning constructs with BspEI and StuI
</LI><LI> gel: plasmids seem to be cut only once again (problem maybe BspEI), size is correct
</LI><LI> suggestion: another digestion on monday with SalI and KpnI, then: submission for sequencing
</LI><H3><SPAN class="mw-headline" id="05.2F09.2F2010">05/09/2010</SPAN></H3><UL><LI> transformation of final Kit constructs F1 - F16 into E. coli Top10 cells
</LI></UL><H3><SPAN class="mw-headline" id="06.2F09.2F2010">06/09/2010</SPAN></H3><UL><LI> inocculation of miniprep LB cultures for the constructs F1 - F16 from single colonies from the plates prepared the previous day
</LI><LI> digestion of PCR amplified vector backbone pSB1A3 with EcoRI/PstI; after subsequent heat inactivation, digestion with DpnI was performed and the vector was purified by applying a nucleotide removal kit.
</LI></UL><UL><LI> digestion of tuning construct with KpnI and SalI result in weird band pattern on the gel
</LI><LI> PCR and purification of the binding sites looked fine on the gel
</LI><LI> submission of tuning constructs and pcDNA5 containing different miRNAs for sequencing
</LI></UL><H3><SPAN class="mw-headline" id="07.2F09.2F2010">07/09/2010</SPAN></H3><DIV class="thumb tright"><DIV class="thumbinner" style="width:352px;"><DIV class="thumbcaption">Gel 100907-1: Ligation Control. Numbers indicate the F-Construct numbers of the parts assembled</DIV></DIV></DIV><UL><LI> Miniprep of constructs F1 - F16 using a Qiagen Miniprep Kit
</LI><LI> Digestion of 500 ng of each construct with the following enzymes
</LI></UL><DL><DD><UL><LI> F2, F3, F5 with EcoRI/NheI
</LI><LI> F8, F1, F15, F11, F12, F13, F14, F7 with PstI/SpeI
</LI></UL></DD></DL><UL><LI> Ligation of the follwing constructs into pSB1A3 via standard 3A assembly
</LI></UL><DL><DD><UL><LI> F3/F15, F2/F1, F2/F8, F3/F1, F2/F15, F5/F7, F5/F11, F5/F12, F5/F13, F5/F14
</LI></UL></DD></DL><UL><LI> Digestion of 500 ng F6 with XbaI; purification via nucleotide romoval kit
</LI><LI> Annealing of XbaI mutation oligo according to the following protocol
</LI></UL><DL><DD><UL><LI> prepare mixtrure of oligos (5 ul @ 100 mM each) in NEB buffer 2 (volume 50 ul)
</LI><LI> heat up to 95 °C in heating block; cool down to room temperature slowly
</LI><LI> incubate on ice for 10 min
</LI></UL></DD></DL><UL><LI> 0.5, 1 and 3 ul of the annealed oligos were then ligated into 50 ng of pre-digested F6 construct;
</LI></UL><P>All Ligations were controlled by loading 10 ul of ligation reactiong onto a 1 % agarose gel (100907-1), run for 35 min @ 100 V
</P><UL><LI> repetition of TetR and tuning construct cloning
</LI><LI> purification and digestion of PCR amplified binding sites with NotI
</LI></UL><H3><SPAN class="mw-headline" id="08.2F09.2F2010">08/09/2010</SPAN></H3><DIV class="thumb tright"><DIV class="thumbinner" style="width:352px;"><DIV class="thumbcaption">Gel 100908-1: Colony-PCR.</DIV></DIV></DIV><UL><LI> Colony PCR of the previous days' cloning and miniprep cultures were inocculated in parallel; The result of the PCR was analyzed on a 1 % agarsoe gel (100908-1) run for 35 min @ 100 V.
</LI></UL><UL><LI> cloning of shRNAs in pcDNA5 again
</LI><LI> further digestion of binding sites with AscSI (analog to SgfI) and purification
</LI><LI> preparation of psyCHECK2 for insertion of binding sites and shRNAs for first test measurements
</LI><LI> transformation of TetR constructs
</LI><LI> ligation of both inserts for the tuning construct, extraction of the right insert (3000bp) and ligation into the backbone
</LI></UL><H3><SPAN class="mw-headline" id="09.2F09.2F2010">09/09/2010</SPAN></H3><DIV class="thumb tright"><DIV class="thumbinner" style="width:352px;"><DIV class="thumbcaption">Gel 100909-1</DIV></DIV></DIV><DIV class="thumb tright"><DIV class="thumbinner" style="width:352px;"><DIV class="thumbcaption">Gel 100909-2</DIV></DIV></DIV><UL><LI>Miniprep of the cultures inocculated on the previous day followed by test digestion of 300 ng of each constructs with NotI (gel 100909-1). The positive samples were send for sequencing @ GATC. The gel looks weird due to use of 0.05 x TAE buffer instead of 0.5 x.
</LI><LI>The nine Luc2-SV40 constructs were with XbaI and NheI (negative testing), and XhoI and PstI (positive testing) for the induced mutation. The reaction was set up according to the standard protocol, and then the samples were visualized on a 1% agarose gel (gel 100909-2).
</LI></UL><UL><LI> colony PCR of TetR constructs are positive
</LI><LI> inoculate TetR for miniprep
</LI></UL><H3><SPAN class="mw-headline" id="10.2F09.2F2010">10/09/2010</SPAN></H3><DIV class="thumb tright"><DIV class="thumbinner" style="width:352px;"><DIV class="thumbcaption">Tet test digest</DIV></DIV></DIV><DIV class="thumb tright"><DIV class="thumbinner" style="width:352px;"><DIV class="thumbcaption">colony PCR shRNAs</DIV></DIV></DIV><UL><LI> PCR of SV40 Terminator with BamHI site according to standard PCR protocol (annealing @ 58 °C)
</LI><LI> Assembly of the following constructs, again via 3A standard assembly.
</LI></UL><DL><DD><UL><LI> F6/F1, F6/F8, F10/F4, F5/F1, F16/SV40, F5/F8 </LI></UL></DD></DL><P>Subsequent Transformation into DH5alpha cells
</P><UL><LI> miniprep of TetR
</LI><LI> test digest of TetR with BspEI &amp; NotI and with SgfI gave promising results
</LI><LI> YES it was done with marker instead of loading dye but:
</LI><LI> for lanes 2-4 you see a stronger band at 700 pb (size of TetR)
</LI><LI> for landes 5-7 you see a stronger band at 5000 bp as it was linearized with SgfI
</LI></UL><UL><LI> colony PCR of binding site in PsiCheck and tuning construct.. tuning construct looks promising, BS nothing on the gel except for primers
</LI><LI> colony PCR of shRNAs look promising even though we cannot say whether still the old shRNA is inside
</LI><LI> PCR of CMV shRNA which leads to a size of 1200bp
</LI></UL><H3><SPAN class="mw-headline" id="11.2F09.2F2010">11/09/2010</SPAN></H3><DIV class="thumb tright"><DIV class="thumbinner" style="width:352px;"><DIV class="thumbcaption">Gel 100911-1</DIV></DIV></DIV><DIV class="thumb tright"><DIV class="thumbinner" style="width:352px;"><DIV class="thumbcaption">Gel 100911-2</DIV></DIV></DIV><UL><LI> Colonies were picked from the plates from the previous days' cloning and a colony PCR was performed according to the standard protocol. In parallel LB miniprep cultures were inocculated for each colony (gel 100911-1 and 100911-2). 
</LI></UL><UL><LI> miniprep of tuning construct and assorted shRNA BS PsiCheck colonies. Repetition of test PCR (for tuning primer L11 and L13, for BS primer L1 and 86a (dominik)) with miniprep template. Tuning construct looks kind of good on the gel, but I don't want to get you hopes up too soon.. it will be send for sequencing on monday.
</LI><LI> Tet Repressor cloned!!! 
</LI><LI> shRNA 2,3,5,7,8 and 9 seems to be cloned according to test digest
</LI></UL><DIV class="thumb tright"><DIV class="thumbinner" style="width:352px;"><DIV class="thumbcaption">test digest</DIV></DIV></DIV><H3><SPAN class="mw-headline" id="12.2F09.2F2010">12/09/2010</SPAN></H3><DIV class="thumb tright"><DIV class="thumbinner" style="width:352px;"><DIV class="thumbcaption">Gel 100912-1</DIV></DIV></DIV><DIV class="thumb tright"><DIV class="thumbinner" style="width:352px;"><DIV class="thumbcaption">Gel 100912-2</DIV></DIV></DIV><UL><LI> Minipreps were done by applying a Qiagen Miniprep Kit for the positive candidates picked on the previous day. Test digestion was performed by digesting ~ 250 ng of each construct with SpeI/NheI and the result was analyzed on a 1 % agarose gel (100912-1 and100912-2) run for 40 min @ 100 V. For all the constructs we got the right band despite construct F10/F4.3. Positive constructs were send for sequencing @ GATC.
</LI></UL><UL><LI> miniprep of shRNA constructs and Tuning constructs
</LI></UL><H3><SPAN class="mw-headline" id="13.2F09.2F2010">13/09/2010</SPAN></H3><UL><LI> digestion of the following parts:
</LI></UL><DL><DD><UL><LI> F4 (E/N), F5 (S/P), R5 (E/N), R6 (E/N), R1 (S/P), R2 (S/P), R3 (S/P), R4 (S/P), R9 (E/P), R10 (E/P); after heat inactivation subsequent purification using a nucleotide removal kit (elution volume 20 ul).
</LI></UL></DD></DL><UL><LI> ligation of the following parts (50 ng of vector, equal amounts of parts)
</LI></UL><DL><DD><UL><LI> F4/F5 into pSB1A3 (precut E/P)
</LI><LI> R5 with either R1, R2, R3 or R4 into pSB1C3 (precut E/P)
</LI><LI> R6 with either R1, R2, R3 or R4 into pSB1C3 (precut E/P)
</LI><LI> R9 into pSB1C3 (precut E/P)
</LI><LI> R10 into pSB1C3 (precut E/P)
</LI></UL></DD></DL><UL><LI> transformation of 10 ul ligation reaction after heat inactivation into DH5alpha cells
</LI></UL><UL><LI> digestion and cloning of binding sites and PsiCheck.. again
</LI><LI> submission of constructs for sequencing
</LI></UL><H3><SPAN class="mw-headline" id="14.2F09.2F2010">14/09/2010</SPAN></H3><DIV class="thumb tright"><DIV class="thumbinner" style="width:352px;"><DIV class="thumbcaption">Gel 100914-1</DIV></DIV></DIV><DIV class="thumb tright"><DIV class="thumbinner" style="width:352px;"><DIV class="thumbcaption">Gel 100914-2</DIV></DIV></DIV><UL><LI><B>note</B>: for sequential digest of pcDNA5 or shRNAs with ApaI and HindIII use buffer 2
</LI></UL><UL><LI> Colony- PCR for the cloning products done on the previous day (gels 100914-1 and 100914-2 on the right)
</LI><LI> positive clones were Mini-Prepped and test digested
</LI></UL><H3><SPAN class="mw-headline" id="15.2F09.2F2010">15/09/2010</SPAN></H3><DIV class="thumb tright"><DIV class="thumbinner" style="width:352px;"><DIV class="thumbcaption">Gel 100915-1</DIV></DIV></DIV><UL><LI> digestion of TetR construct with AsiSi and NotI or SgfI and NotI respectively
</LI><LI> for cloning of binding sites for miR122, miR375 and miR376
</LI></UL><LI> ligation of the other binding sites for synthetic pri-miRNA (6, 7, 8, 9 and 10) with digested psiCHECK2
</LI><UL><LI> PCR for the introduction of the Kozag-Sequence in front of the hRluc_BGH part; two strategies were performed in parallel using different touchdown-PCR protocols. Protocol nr. 1 was set up as follows:
</LI></UL><DL><DD><UL><LI> 25 ul Phusion HF MasterMix
</LI><LI> 0.5 ul of 100 um primers
</LI><LI> 1 ul of template R12 (50 ng/ul)
</LI><LI> 23 ul of water
</LI></UL></DD></DL><P>
Touchdown PCR was performed according to the following protocol:

................................................
95 °C/5 min 
................................................ (1x) 
95 °C/30 s 
68 °C/45 s (- 0.5 °C/cycle) 
72 °C/1 min 
................................................ (35x) 
72 °C/10 min 
................................................ (1x) 
4 °C/ forever 
................................................ 
Protocol nr. 2 was set up as follows:
</P><DL><DD><UL><LI> 25 ul Phusion HF MasterMix
</LI><LI> 0.1 ul of 100 um primers
</LI><LI> 1 ul of template R12 (50 ng/ul)
</LI><LI> 23.8 ul of water
</LI></UL></DD></DL><P>
Touchdown PCR was performed according to the following protocol:

................................................ 
95 °C/5 min 
................................................ (1x 
95 °C/15 s 
68 °C/45 s (- 0.5 °C/cycle) 
72 °C/1 min 
................................................ (15x) 
95 °C/15 s 
60 °C/30 s 
72 °C/1 min 
................................................ (20x) 
72 °C/10 min 
................................................ (1x) 
4 °C/ forever 
................................................ </P><P>Each PCR was performed in two replicates. The result of the PCRs was analyzed on a 1 % agarose gel (100915-1), run for 1 h @ 100 V. The PCR nr. 2 was subsequently PCR purified by applying a Qiagen PCR purification KIT and used for the cloning
</P><H3><SPAN class="mw-headline" id="16.2F09.2F2010">16/09/2010</SPAN></H3><UL><LI> Cloning of Kozag_hRluc_BGH fragment into pSB1C3 and Constructs R24-R31
</LI><LI> Cloning of the XhoI/XmaI_suffix oligo via NheI/PstI into the TetR construct F16)
</LI></UL><DL><DD><UL><LI> Digestion of PCR fragment with SpeI/PstI according to 3A standard protocol; digestion of destination Vector pSB1C3
</LI><LI> Ligation and Transformation were done according to the standard protocol
</LI></UL></DD></DL><UL><LI> PCR to generate binding site oligos, nucleotid removal, digestion, nucleotid removal
</LI><LI> different digestion and ligation protocols tested for binding sites and psiCHECK2, afterwards transformation
</LI><LI> possibilities: 
</LI><LI> dilution of oligo insert [1:100,1:500], ratio: insert/vector = 5/1
</LI><LI> SAP treatment for oligos to prevent them from forming concatemers 
</LI><LI> SAP treatment of backbone to prevent it from re-ligation
</LI><LI> cut of backbone into two fragments to ease gel excision (if done) and advance ligation
</LI><LI> ligation process: 1h @ RT or over-night @ 4°C in darkness
</LI><LI> note: bs122 confused with bs7
</LI></UL><H3><SPAN class="mw-headline" id="17.2F09.2F2010">17/09/2010</SPAN></H3><UL><LI> colony PCR, mini-prep and digestion of promising psiCHECK or TetR constructs containing binding site
</LI><LI> control plates with horrible number of re-ligations
</LI></UL><DIV class="thumb tright"><DIV class="thumbinner" style="width:352px;"><DIV class="thumbcaption">Gel 100917-1</DIV></DIV></DIV><DIV class="thumb tright"><DIV class="thumbinner" style="width:352px;"><DIV class="thumbcaption">Gel 100917-2</DIV></DIV></DIV><UL><LI> Selection of colonies from the plates (see previous days' cloning) via colony-PCR using the standard sequencing primers; a ~5.5 kb fragment should be visible for the assembled construct K1-K8, if colony-PCR was positive (gel 100917-1 and 100917-2). No sample shows a 5.5 kb band, but that's most likely due to the very difficult amplification of such a long fragment with Fermentas 2x PCR MasterMix (Taq-based). Therefor 3 miniprep cultures were inocculated for each construct (K1-K8). The TetR oligo insertion and the Kozag_hRluc_BGH cloning into pSB1C3 gave the right bands on the gel.
</LI></UL><H3><SPAN class="mw-headline" id="19.2F09.2F2010">19/09/2010</SPAN></H3><UL><LI> Miniprep of the previous days inocculated miniprep cultures
</LI></UL><H3><SPAN class="mw-headline" id="20.2F09.2F2010">20/09/2010</SPAN></H3><DIV class="thumb tright"><DIV class="thumbinner" style="width:352px;"><DIV class="thumbcaption">Gel 100920-1</DIV></DIV></DIV><DIV class="thumb tright"><DIV class="thumbinner" style="width:352px;"><DIV class="thumbcaption">Gel 100920-2</DIV></DIV></DIV><UL><LI> test digestion of the miniprepped clones (previous day) with EcoRI/PstI; For all clones at least 2 digestions gave the exactly expected band (gel 100920-1 and 100920-2); therefor we can say, that the tuning construct is succesfully assembled and ready to be tested
</LI></UL><H3><SPAN class="mw-headline" id="22.2F09.2F2010">22/09/2010</SPAN></H3><UL><LI> digestion of <A href="/wiki/index.php?title=Igem2010/Main/Library/Constructs/pSMB_miTrigger&amp;action=edit&amp;redlink=1" class="new" title="Igem2010/Main/Library/Constructs/pSMB miTrigger (page does not exist)"> R23</A> with EcoRI and PstI in EcoRI Buffer (<A href="http://www.neb.com" class="external text" rel="nofollow">NEB</A>) and ligation into digested <A href="http://partsregistry.org/Part:pSB1C3" class="external text" rel="nofollow">pSB1C3</A>, transformation, growing on selective agar plates (chloramphenicol), following <A href="https://2010.igem.org/3A_Assembly" class="external text" rel="nofollow">standard protocol recommendations</A></LI></UL><P><A href="https://2010.igem.org/Team:Heidelberg/Notebook/Methods#Dual_Luciferase_Assay" class="external text" rel="nofollow">Dual Luciferase Assay</A>
In order to test for promoter efficiency and to check whether the miRNA kit assembly works fine
50ng of each construct with different promoter set-ups were transfected into HEK 293 T-REx cells and other cell lines HEK, HeLa, Huh7 in 96-well plate format using FuGENE transfection reagent. As every construct is expressing firefly luciferase (luc2) and renilla luciferase (hRluc) at the same time the setup allows for standardization of transfection efficiency as only luc2 is tagged with binding sites. Each sample was transfected and measured by Dual luciferase assay in 8 replicates. As by this time no shRNA has been cloned into plasmid no knock-down of luc2 is expected and the different expression efficiencies allow for characterization of the different promoters. <A href="http://partsregistry.org/wiki/index.php?title=Part:BBa_K337032" class="external text" rel="nofollow">BBa_K337032</A> leads to a relative luciferase unit (RLU) of luc2 to hRluc expression of 6 RLU. <A href="http://partsregistry.org/wiki/index.php?title=Part:BBa_K337035" class="external text" rel="nofollow">BBa_K337035</A> and <A href="http://partsregistry.org/wiki/index.php?title=Part:BBa_K337035" class="external text" rel="nofollow">BBa_K337035</A> are showing a comparable expression of 12 - 13 RLU, which is in line with the knowledge that both luciferases are driven by the CMV promoter.  Hek 293 T-Rex cells stably express the Tet repressor thus allows us to observe very efficient repression of Firefly luciferse expression if a CMV-TetO2 promoter is driving luc2 (<A href="http://partsregistry.org/wiki/index.php?title=Part:BBa_K337038" class="external text" rel="nofollow">BBa_K337038</A> and <A href="http://partsregistry.org/wiki/index.php?title=Part:BBa_K337046" class="external text" rel="nofollow">BBa_K337046</A>). <A href="http://partsregistry.org/wiki/index.php?title=Part:BBa_K337040" class="external text" rel="nofollow">BBa_K337040</A> transfection into Hek T-Rex cells results in an expression of 15 RLU. <A href="http://partsregistry.org/wiki/index.php?title=Part:BBa_K337042" class="external text" rel="nofollow">BBa_K337042</A> and <A href="http://partsregistry.org/wiki/index.php?title=Part:BBa_K337044" class="external text" rel="nofollow">BBa_K337044</A> are constructed in a way that luc2 is driven by the CMV promoter and hRluc is driven by the RSV promoter and show a comparable expression of 17-20 RLU. This leads to the conclusion that the CMV promoter shows comparable expression to the RSV promoter in Hek T-Rex cell lines. If transfecting <A href="http://partsregistry.org/wiki/index.php?title=Part:BBa_K337035" class="external text" rel="nofollow">BBa_K337035</A> and <A href="http://partsregistry.org/wiki/index.php?title=Part:BBa_K337036" class="external text" rel="nofollow">BBa_K337036</A> are transfected into different cell lines it is obvious that Hek293T cells are the easiest to transfect with both constructs an expression of 17-22 RLU is to be measured. Hek T-Rex cells are showing and expression level of 12 RLU of both constructs. Hela cells are also showing constant expression levels of  8 RLU with both constructs. A rather high standard deviation in the luciferase expression and also differences between the 2 constructs is to be seen in Huh7 cells. This might be due to low transfection efficiency of this cell line in general. Alltogether it is to say that <A href="http://partsregistry.org/wiki/index.php?title=Part:BBa_K337035" class="external text" rel="nofollow">BBa_K337035</A> and <A href="http://partsregistry.org/wiki/index.php?title=Part:BBa_K337036" class="external text" rel="nofollow">BBa_K337036</A> show comparable expression. 
</P><TABLE class="wikitable sortable" border="0" style="text-align: center"><TBODY><TR bgcolor="#cccccc"><TH>part</TH><TH>promoter driving luc2 (Firefly)</TH><TH>promoter driving (Renilla)</TH><TH>promoter driving shRNA expression
</TH></TR><TR><TD><A href="http://partsregistry.org/wiki/index.php?title=Part:BBa_K337032" class="external text" rel="nofollow">BBa_K337032</A></TD><TD>RSV</TD><TD>CMV</TD><TD>SV40
</TD></TR><TR><TD><A href="http://partsregistry.org/wiki/index.php?title=Part:BBa_K337035" class="external text" rel="nofollow">BBa_K337035</A></TD><TD>CMV</TD><TD>CMV</TD><TD>SV40
</TD></TR><TR><TD><A href="http://partsregistry.org/wiki/index.php?title=Part:BBa_K337036" class="external text" rel="nofollow">BBa_K337036</A></TD><TD>CMV</TD><TD>CMV</TD><TD>RSV
</TD></TR><TR><TD><A href="http://partsregistry.org/wiki/index.php?title=Part:BBa_K337038" class="external text" rel="nofollow">BBa_K337038</A></TD><TD>CMV TetO2</TD><TD>CMV </TD><TD>RSV
</TD></TR><TR><TD><A href="http://partsregistry.org/wiki/index.php?title=Part:BBa_K337040" class="external text" rel="nofollow">BBa_K337040</A></TD><TD>RSV</TD><TD>RSV</TD><TD>SV40
</TD></TR><TR><TD><A href="http://partsregistry.org/wiki/index.php?title=Part:BBa_K337042" class="external text" rel="nofollow">BBa_K337042</A></TD><TD>CMV</TD><TD>RSV</TD><TD>SV40
</TD></TR><TR><TD><A href="http://partsregistry.org/wiki/index.php?title=Part:BBa_K337044" class="external text" rel="nofollow">BBa_K337044</A></TD><TD>CMV</TD><TD>RSV</TD><TD>RSV
</TD></TR><TR><TD><A href="http://partsregistry.org/wiki/index.php?title=Part:BBa_K337046" class="external text" rel="nofollow">BBa_K337046</A></TD><TD>CMV TetO2</TD><TD>RSV</TD><TD>RSV
</TD></TR></TBODY></TABLE><DIV class="center"><DIV class="thumb tnone"><DIV class="thumbinner" style="width:602px;"><DIV class="thumbcaption">Promoter strength characterization in HEK 293 T-REx cell line</DIV></DIV></DIV></DIV><DIV class="center"><DIV class="thumb tnone"><DIV class="thumbinner" style="width:486px;"><DIV class="thumbcaption">Promoter strength characterization in different cell lines</DIV></DIV></DIV></DIV><H3><SPAN class="mw-headline" id="23.2F09.2F2010">23/09/2010</SPAN></H3><DIV class="thumb tright"><DIV class="thumbinner" style="width:352px;"><DIV class="thumbcaption">Gel 100923-1. Analytical gel with following lanes: 1) till 8) colony PCR products with nice bands as expected at ~ 800 bp especially at lane 1, 4, 6 and 8, lane 9) 1kb Plus Ladder (Invitrogen)</DIV></DIV></DIV><UL><LI> minipreps (using Quiagen Kit) and <A href="/wiki/index.php?title=Igem2010/Main/Protocols/Colony_PCR&amp;action=edit&amp;redlink=1" class="new" title="Igem2010/Main/Protocols/Colony PCR (page does not exist)"> colony-PCR</A> of each 8 colonies of <A href="/wiki/index.php?title=Igem2010/Main/Library/Constructs/pSMB_miTrigger&amp;action=edit&amp;redlink=1" class="new" title="Igem2010/Main/Library/Constructs/pSMB miTrigger (page does not exist)"> R23</A> in <A href="http://partsregistry.org/Part:pSB1C3" class="external text" rel="nofollow">pSB1C3</A> (digested, ligated, transformed and plated on <A href="/Igem2010/Main/synthetic_miR_Kit/September#22.2F09.2F2010" title="Igem2010/Main/synthetic miR Kit/September"> previous day</A>)
</LI><LI> PCR product of R23C on analytical gel (see gel 100923-1)
</LI></UL><LI> digestion of <A href="/wiki/index.php?title=Igem2010/Main/Library/Constructs/pSMB_miTrigger&amp;action=edit&amp;redlink=1" class="new" title="Igem2010/Main/Library/Constructs/pSMB miTrigger (page does not exist)"> R23</A> and <A href="/wiki/index.php?title=Igem2010/Main/Library/Constructs/pSMB_miTrigger&amp;action=edit&amp;redlink=1" class="new" title="Igem2010/Main/Library/Constructs/pSMB miTrigger (page does not exist)"> R33</A> with either SpeI and PstI or EcoRI and NheI
</LI><LI> ligation into digested <A href="http://partsregistry.org/Part:pSB1A3" class="external text" rel="nofollow">pSB1A3</A>, <A href="https://2010.igem.org/Transformation" class="external text" rel="nofollow">transformation</A>, growing on selective agar plates (ampicillin), following <A href="https://2010.igem.org/3A_Assembly" class="external text" rel="nofollow">standard protocol recommendations</A></LI><P><A href="https://2010.igem.org/Team:Heidelberg/Notebook/Methods#Dual_Luciferase_Assay" class="external text" rel="nofollow">Dual Luciferase Assay</A> was performed as on [<A href="https://2010.igem.org/Team:Heidelberg/Notebook/miRNA_Kit/September#22.2F09.2F2010" class="external text" rel="nofollow">previous day</A>]. Cells were not harvested after 20h, but 48h.
</P><H3><SPAN class="mw-headline" id="24.2F09.2F2010">24/09/2010</SPAN></H3><UL><LI>inoculation and colony PCR of the transformed (23A+33A)=56C construct from yesterday. Same PCR protocol was used, even though the fragment should be 1100 and therefore elongation was probably not long enough. The PCR showed a band at 300bp, which dominik says is normal if you have religated vector. But nevertheless, miniprep of La1, La3 and La4 were made, because they didn't show a strong negative PCR band. 
</LI><LI> test digestions (actually they are preparative, the only difference is I will test them on the gel as well to be sure I have the right thing)
</LI><LI> R23A from yesterday (miniprep 8 and 8/) is digested with Eco&amp;Nhe, expected fragment is 500bp
</LI><LI> R33 with Spe&amp;Pst, expected fragment 600bp
</LI><LI> R56C with Spe&amp;Pst, expected fragment 1100bp
</LI><LI>R8 with Eco&amp;Nhe, expected frament 1000bp
</LI></UL><LI> ligation of R23A+R33+backbone C
</LI><LI> ligation of R56C + R8 + backbone A
</LI><H3><SPAN class="mw-headline" id="25.2F09.2F2010">25/09/2010</SPAN></H3><UL><LI> PCR amplification of standardised pcDNA5 p55 and p1 with primer D47/48 and D55/69
</LI></UL><H3><SPAN class="mw-headline" id="26.2F09.2F2010">26/09/2010</SPAN></H3><DIV class="thumb tright"><DIV class="thumbinner" style="width:352px;"><DIV class="thumbcaption">Gel 100926-1. Analytical gel with following lanes: 1) R33 E/N (from 23<SUP>rd</SUP> September)  2) R23_1 S/P  3) R23_6  4) R33C  5) R23_4  6) R23_8  7) 1kb Plus Ladder (Invitrogen)  8) R33C E/N  9) R23_4 S/P  9) R23C_8 S/P</DIV></DIV></DIV><DIV class="thumb tright"><DIV class="thumbinner" style="width:302px;"><DIV class="thumbcaption">Gel 100926-2. PCR results of p55 and p1: p1 with primer 47/48 (lane 1), p1 with primer 55/70 (lane 2), p55 with primer 47/48 (lane3), p55 with primer 55/70</DIV></DIV></DIV><UL><LI> miniprep of R23C cultures again (colonies from transformation (done <A href="/Igem2010/Main/synthetic_miR_Kit/September#22.2F09.2F2010" title="Igem2010/Main/synthetic miR Kit/September"> on wednesday</A>))
</LI><LI> check of all digestions on an analytical gel (see gel 100926-1)
</LI><LI> fragment for R23_6 is a bit smaller than expected (rather 400 bp than 500 bp)
</LI><LI> originally digested R23A runs slightly below 500 bp, too (data not shown)
</LI><LI> because of reverse elements and resulting secondary structure?
</LI></UL><LI> repetition of ligation with R23_6 and old R33 (nice band at roughly 600bp)
</LI><LI> including negative control at this time with only backbone <A href="http://partsregistry.org/Part:pSB1A3" class="external text" rel="nofollow">pSB1A3</A></LI><LI><A href="https://2010.igem.org/Transformation" class="external text" rel="nofollow">transformation</A></LI><P> tuning construct:
</P><UL><LI> gel extraction of PCR on <A href="/Igem2010/Main/synthetic_miR_Kit/September#25.2F09.2F2010" title="Igem2010/Main/synthetic miR Kit/September"> p55 and p1</A> (see gel 100926-2)
</LI><LI> recloning of R1,R4 and R7 into standardbackbone <A href="http://partsregistry.org/Part:pSB1A3" class="external text" rel="nofollow">pSB1A3</A></LI><LI> digest 500ng of R1, R4 and R7 with EcoRI and PstI
</LI><LI> heat inactivate 20 min at 80°C
</LI><LI> Nucleotide removal
</LI><LI> ligate 150ng and 250ng of insert into 30ng <A href="http://partsregistry.org/Part:pSB1A3" class="external text" rel="nofollow">pSB1A3</A> 1h at RT and transform ligation
</LI></UL><H3><SPAN class="mw-headline" id="27.2F09.2F2010">27/09/2010</SPAN></H3><DIV class="thumb tright"><DIV class="thumbinner" style="width:352px;"><DIV class="thumbcaption">Gel 100927-1: colony PCR on K1 (lane 1-8), K4 (lane 9-16) and K8 (lane 17-24)</DIV></DIV></DIV><DIV class="thumb tright"><DIV class="thumbinner" style="width:352px;"><DIV class="thumbcaption">Gel 100927-2: colony PCR K1 (lane 1&amp;2), K4 (3&amp;4) und K7 (5&amp;6), K3 digested with BamHI (7), K3 digested NheI and SpeI (8), R33 in pSB1A3 digested with EcoRI and PstI (9), R33 digested with NheI and SpeI (10)</DIV></DIV></DIV><P>
tuning construct:
</P><UL><LI><A href="/wiki/index.php?title=Igem2010/Main/Protocols/Colony_PCR&amp;action=edit&amp;redlink=1" class="new" title="Igem2010/Main/Protocols/Colony PCR (page does not exist)"> colony-PCR</A> on <A href="/wiki/index.php?title=Igem2010/Main/Library/Constructs/pSMB_miTrigger&amp;action=edit&amp;redlink=1" class="new" title="Igem2010/Main/Library/Constructs/pSMB miTrigger (page does not exist)"> K1</A>, <A href="/wiki/index.php?title=Igem2010/Main/Library/Constructs/pSMB_miTrigger&amp;action=edit&amp;redlink=1" class="new" title="Igem2010/Main/Library/Constructs/pSMB miTrigger (page does not exist)"> K4</A> and <A href="/wiki/index.php?title=Igem2010/Main/Library/Constructs/pSMB_miTrigger&amp;action=edit&amp;redlink=1" class="new" title="Igem2010/Main/Library/Constructs/pSMB miTrigger (page does not exist)"> K7</A> cloned into <A href="http://partsregistry.org/Part:pSB1A3" class="external text" rel="nofollow">pSB1A3</A> showed nearly only positives as a band of 400bp is expected by using the primer pair 50/75
</LI><LI> miniprep of <A href="/wiki/index.php?title=Igem2010/Main/Library/Constructs/pSMB_miTrigger&amp;action=edit&amp;redlink=1" class="new" title="Igem2010/Main/Library/Constructs/pSMB miTrigger (page does not exist)"> K1</A>, <A href="/wiki/index.php?title=Igem2010/Main/Library/Constructs/pSMB_miTrigger&amp;action=edit&amp;redlink=1" class="new" title="Igem2010/Main/Library/Constructs/pSMB miTrigger (page does not exist)"> K4</A> and <A href="/wiki/index.php?title=Igem2010/Main/Library/Constructs/pSMB_miTrigger&amp;action=edit&amp;redlink=1" class="new" title="Igem2010/Main/Library/Constructs/pSMB miTrigger (page does not exist)"> K7</A></LI><LI> send for sequencing
</LI><LI> cloning of <A href="/wiki/index.php?title=Igem2010/Main/Library/Constructs/pSMB_miTrigger&amp;action=edit&amp;redlink=1" class="new" title="Igem2010/Main/Library/Constructs/pSMB miTrigger (page does not exist)"> K3</A> into <A href="http://partsregistry.org/Part:pSB1A3" class="external text" rel="nofollow">pSB1A3</A>:
</LI><LI> digest <A href="/wiki/index.php?title=Igem2010/Main/Library/Constructs/pSMB_miTrigger&amp;action=edit&amp;redlink=1" class="new" title="Igem2010/Main/Library/Constructs/pSMB miTrigger (page does not exist)"> K3</A> with PstI and EcoRI
</LI><LI> ligate 200ng with 30ng of vector <A href="http://partsregistry.org/Part:pSB1A3" class="external text" rel="nofollow">pSB1A3</A></LI><LI> transform ligation and plate on an Ampicillin plate
</LI></UL><LI> digest shRNA 6 (F11) and shRNA 7 (F12):
</LI><LI> 1h at 37°C with HindIII
</LI><LI> heat inactivate 20 min at 80°C
</LI><LI> digest 1h at 37°C with AflII
</LI><LI> purify by Nucleotide removal kit and elute in 20µl
</LI><P>
repressor construct:
</P><UL><LI> a lot of religations, thus: a lot of colonies to pick
</LI><LI><A href="/wiki/index.php?title=Igem2010/Main/Protocols/Colony_PCR&amp;action=edit&amp;redlink=1" class="new" title="Igem2010/Main/Protocols/Colony PCR (page does not exist)"> colony-PCR</A> and test digestion with SpeI and PstI of promising mini-preps revealed some positive samples
</LI><LI> ligation with <A href="/wiki/index.php?title=Igem2010/Main/Library/Constructs/pSMB_miTrigger&amp;action=edit&amp;redlink=1" class="new" title="Igem2010/Main/Library/Constructs/pSMB miTrigger (page does not exist)"> R8</A> and <A href="/wiki/index.php?title=Igem2010/Main/Library/Constructs/pSMB_miTrigger&amp;action=edit&amp;redlink=1" class="new" title="Igem2010/Main/Library/Constructs/pSMB miTrigger (page does not exist)"> R11</A>, respectively (each digested with EcoRI and NheI) in <A href="http://partsregistry.org/Part:pSB1C3" class="external text" rel="nofollow">pSB1C3</A> (backbone, digested with EcoRI and PstI), following <A href="https://2010.igem.org/3A_Assembly" class="external text" rel="nofollow">standard protocol recommendations</A></LI><LI> growth of transformed cells on selective agar plates (chloramphenicol) over night
</LI></UL><H3><SPAN class="mw-headline" id="28.2F09.2F2010">28/09/2010</SPAN></H3><DIV class="thumb tright"><DIV class="thumbinner" style="width:316px;"><DIV class="thumbcaption">Gel 100928-1. Colony PCR of K3 construct</DIV></DIV></DIV><DIV class="thumb tright"><DIV class="thumbinner" style="width:352px;"><DIV class="thumbcaption">Gel 100928-2. Test digest of with PstI and EcoRI (lane 1-6), test digest with BamHI (lane 7-12), K1 (lane 1&amp;2), K4 (lane 3&amp;4), K7 (lane 5&amp;6)</DIV></DIV></DIV><DIV class="thumb tright"><DIV class="thumbinner" style="width:352px;"><DIV class="thumbcaption">Gel 100928-3. Digestion of K1 (lane 1-4), K4 (lane 5-8) and K7 (lane 9-12) with HindIII and AflII</DIV></DIV></DIV><P>tuning construct: 
</P><UL><LI> colony PCR of K3 (gel 100928-1)
</LI><LI> inoculation for mini prep of K3
</LI><LI> mini prep of K1, K4 and K7
</LI><LI> test digestion of K1, K4 and K7 with PstI &amp; EcoRI and BamHI (gel 100928-2 and 100928-3)
</LI><LI> digestion of K1, K4 and K7 with HindIII and AflII prior to gel extraction
</LI><LI> ligation of shRNA 6 and 7 into K1, K4 and K7
</LI><LI> transformation of ligation
</LI><LI> plate ligation on ampicillin plates
</LI></UL><P>
repressor construct:
</P><UL><LI> re-do of mini-preps and digestions of (R23+R33)C_1, (R23+R33)C_14, (R23+R33)C_17 and (R23+R33)C_23 (promising)
</LI><LI> check on gel, submission for sequencing <A href="/Igem2010/Main/synthetic_miR_Kit/September#27.2F09.2F2010" title="Igem2010/Main/synthetic miR Kit/September"> next day</A></LI></UL><LI> analysis of transformations from <A href="/Igem2010/Main/synthetic_miR_Kit/September#27.2F09.2F2010" title="Igem2010/Main/synthetic miR Kit/September"> yesterday</A> (i. e. (R23+R33) + R8 or (R23+R33) + R11 in chloramphenicol standard backbone)
</LI><LI> only re-ligations (as assumed from negative control), more colonies <A href="/Igem2010/Main/synthetic_miR_Kit/September#27.2F09.2F2010" title="Igem2010/Main/synthetic miR Kit/September"> next day</A></LI><H3><SPAN class="mw-headline" id="29.2F09.2F2010">29/09/2010</SPAN></H3><DIV class="thumb tright"><DIV class="thumbinner" style="width:352px;"><DIV class="thumbcaption">Gel 100929-1 first gel: double digestion of <A href="http://partsregistry.org/wiki/index.php?title=Part:BBa_K337032" class="external text" rel="nofollow">K1</A> and <A href="http://partsregistry.org/wiki/index.php?title=Part:BBa_K337036" class="external text" rel="nofollow">K3</A> with PstI and EcoRI: <A href="http://partsregistry.org/wiki/index.php?title=Part:BBa_K337032" class="external text" rel="nofollow">K1</A> (lane 1-4), <A href="http://partsregistry.org/wiki/index.php?title=Part:BBa_K337036" class="external text" rel="nofollow">K3</A> (lane 5-8), second gel: digestion of <A href="http://partsregistry.org/wiki/index.php?title=Part:BBa_K337032" class="external text" rel="nofollow">K1</A> and <A href="http://partsregistry.org/wiki/index.php?title=Part:BBa_K337036" class="external text" rel="nofollow">K3</A> with BamHI: <A href="http://partsregistry.org/wiki/index.php?title=Part:BBa_K337032" class="external text" rel="nofollow">K1</A> (lane 1-4), <A href="http://partsregistry.org/wiki/index.php?title=Part:BBa_K337036" class="external text" rel="nofollow">K3</A> (lane 5-8)</DIV></DIV></DIV><DIV class="thumb tright"><DIV class="thumbinner" style="width:352px;"><DIV class="thumbcaption">Gel 1000929-2 digestion of <A href="http://partsregistry.org/wiki/index.php?title=Part:BBa_K337032" class="external text" rel="nofollow">K1</A> and <A href="http://partsregistry.org/wiki/index.php?title=Part:BBa_K337036" class="external text" rel="nofollow">K3</A> with HindIII and AflII: <A href="http://partsregistry.org/wiki/index.php?title=Part:BBa_K337032" class="external text" rel="nofollow">K1</A> (lane 1&amp;2), <A href="http://partsregistry.org/wiki/index.php?title=Part:BBa_K337036" class="external text" rel="nofollow">K3</A>(lane 3-12)</DIV></DIV></DIV><DIV class="thumb tright"><DIV class="thumbinner" style="width:352px;"><DIV class="thumbcaption">gel 100929-3 pcr on binding sites for shRNA 10</DIV></DIV></DIV><P>tuning construct:
</P><UL><LI> mini Prep on <A href="http://partsregistry.org/wiki/index.php?title=Part:BBa_K337036" class="external text" rel="nofollow">K3</A> and <A href="http://partsregistry.org/wiki/index.php?title=Part:BBa_K337032" class="external text" rel="nofollow">K1</A> constructs
</LI><LI> test digestion with PstI &amp; EcoRI and BamHI showed all positive results
</LI><LI> digest <A href="http://partsregistry.org/wiki/index.php?title=Part:BBa_K337036" class="external text" rel="nofollow">K3</A> and <A href="http://partsregistry.org/wiki/index.php?title=Part:BBa_K337032" class="external text" rel="nofollow">K1</A> with HindIII and AflII
</LI></UL><LI> cloning of shRNA 6 and 7 into <A href="http://partsregistry.org/wiki/index.php?title=Part:BBa_K337032" class="external text" rel="nofollow">K1</A>, <A href="http://partsregistry.org/wiki/index.php?title=Part:BBa_K337036" class="external text" rel="nofollow">K3</A>, <A href="http://partsregistry.org/wiki/index.php?title=Part:BBa_K337038" class="external text" rel="nofollow">K4</A> and <A href="http://partsregistry.org/wiki/index.php?title=Part:BBa_K337044" class="external text" rel="nofollow">K7</A>:
</LI><LI> colony PCR on ligation from the 28/09/2010 did not reveal any positives by using the following protocol:
</LI><LI> colony PCR on ligations from the 28/09/2010 did not reveal any positives by using the following protocol:
</LI><LI> start from the beginning again: in doublicates:
</LI><LI> first procedure:
</LI><LI> digest PCR on shRNA 6 and 7 as well as <A href="http://partsregistry.org/wiki/index.php?title=Part:BBa_K337032" class="external text" rel="nofollow">K1</A>, <A href="http://partsregistry.org/wiki/index.php?title=Part:BBa_K337036" class="external text" rel="nofollow">K3</A>, <A href="http://partsregistry.org/wiki/index.php?title=Part:BBa_K337038" class="external text" rel="nofollow">K4</A> and <A href="http://partsregistry.org/wiki/index.php?title=Part:BBa_K337044" class="external text" rel="nofollow">K7</A> with HindIII and AflII
</LI><LI> purify shRNAs with nucleotide removal kit
</LI><LI> purify digested <A href="http://partsregistry.org/wiki/index.php?title=Part:BBa_K337036" class="external text" rel="nofollow">K3</A> and <A href="http://partsregistry.org/wiki/index.php?title=Part:BBa_K337032" class="external text" rel="nofollow">K1</A> with gel extraction kit
</LI><LI> ligate <A href="http://partsregistry.org/wiki/index.php?title=Part:BBa_K337032" class="external text" rel="nofollow">K1</A>, <A href="http://partsregistry.org/wiki/index.php?title=Part:BBa_K337036" class="external text" rel="nofollow">K3</A>, <A href="http://partsregistry.org/wiki/index.php?title=Part:BBa_K337038" class="external text" rel="nofollow">K4</A> and <A href="http://partsregistry.org/wiki/index.php?title=Part:BBa_K337044" class="external text" rel="nofollow">K7</A> with shRNA 6 and 7
</LI><LI> second procedure: 
</LI><LI> digest <A href="http://partsregistry.org/wiki/index.php?title=Part:BBa_K337038" class="external text" rel="nofollow">K4</A> and <A href="http://partsregistry.org/wiki/index.php?title=Part:BBa_K337032" class="external text" rel="nofollow">K1</A> as well as shRNA6 and 7 from PCR product and purify via nucleotide removal kit
</LI><LI> ligate <A href="http://partsregistry.org/wiki/index.php?title=Part:BBa_K337038" class="external text" rel="nofollow">K4</A> and <A href="http://partsregistry.org/wiki/index.php?title=Part:BBa_K337036" class="external text" rel="nofollow">K3</A> with shRNA 6 and 7
</LI><LI> transform all ligations
</LI><LI> cloning of binding sites:
</LI><LI> second strand synthesis PCR on binding sites for shRNA 10
</LI><DIV class="thumb tright"><DIV class="thumbinner" style="width:352px;"><DIV class="thumbcaption">Gel 100929 r-4: colony PCR of repressor construct, band at 350bp points at backbone religation.</DIV></DIV></DIV><DIV class="thumb tright"><DIV class="thumbinner" style="width:352px;"><DIV class="thumbcaption">Gel 100929 r-5: test digestion of miniprep number 14 withe EcoRI and NheI shows expected band at 1100bp. Digestions of R8 and R11 were also positive, DNA cut out with SpeI and PstI runs at 800bp. Lane 1=miniprep 14 undigested, lane2=miniprep 14 digested, lane3=miniprep 23 digested, lane4=R8 undigested, lane5= R8 digested, lane6=R11 undigested, lane7=R11 digested</DIV></DIV></DIV><P>repressor construct:
</P><UL><LI> colony PCR after transformation of P56+R8 and P56+R11 (P56=R23A+R33) yielded no positive results 
</LI><LI> miniprep of colonies 11, 14, 17 and 23 from P56 cloning were repeted because of bad miniprep results yesterday, then digested with S/P
</LI><LI> R8 and R11 were digested with E/N again
</LI><LI> digestion of P56 colony 14 was positive on the gel, digestion of R8 and R11 also gave the right fragments
</LI><LI><A href="http://partsregistry.org/Part:pSB1C3" class="external text" rel="nofollow">pSB1C3</A> backbone was <A href="http://www.fermentas.com/templates/files/tiny_mce/coa_pdf/coa_ef0511.pdf" class="external text" rel="nofollow">SAP</A> treated to reduce religation that was likely to be the problem with previous transformations
</LI><LI> ligation of P56 (14) with R8 and R11 and SAPed backbone and control ligation with backbone only, transformation
</LI></UL><H3><SPAN class="mw-headline" id="30.2F09.2F2010">30/09/2010</SPAN></H3><DIV class="thumb tright"><DIV class="thumbinner" style="width:352px;"><DIV class="thumbcaption">Gel 1000903-2 colony PCR on shRNA10 cloned into K3 and K4</DIV></DIV></DIV><DIV class="thumb tright"><DIV class="thumbinner" style="width:352px;"><DIV class="thumbcaption">Gel 1000930-3 colony PCR on shRNA10 into K3 and K4</DIV></DIV></DIV><P>tuning construct:
</P><UL><LI> colony pcr on shRNA 6 and 7 into construct <A href="http://partsregistry.org/wiki/index.php?title=Part:BBa_K337036" class="external text" rel="nofollow">K3</A> and <A href="http://partsregistry.org/wiki/index.php?title=Part:BBa_K337038" class="external text" rel="nofollow">K4</A></LI><LI> cloning of binding sites for shRNA 10 into <A href="http://partsregistry.org/wiki/index.php?title=Part:BBa_K337036" class="external text" rel="nofollow">K3</A> and <A href="http://partsregistry.org/wiki/index.php?title=Part:BBa_K337038" class="external text" rel="nofollow">K4</A>:
</LI><LI> digestion of binding sites (perfect, imperfect 9-12 and imperfect 9-22) for shRNA10 with xhoI and ageI :) and construct <A href="http://partsregistry.org/wiki/index.php?title=Part:BBa_K337036" class="external text" rel="nofollow">K3</A> and <A href="http://partsregistry.org/wiki/index.php?title=Part:BBa_K337038" class="external text" rel="nofollow">K4</A> in <A href="http://partsregistry.org/Part:pSB1A3" class="external text" rel="nofollow">pSB1A3</A> with xmaI and xhoI
</LI><LI> ligation of binding sites with digested constructs
</LI><LI> transformation of ligation into TOP10 cells
</LI><LI> plate on ampicillin plates ON
</LI></UL><DIV class="thumb tright"><DIV class="thumbinner" style="width:352px;"><DIV class="thumbcaption">Gel 100930-1 colony PCR of Repressor construct with different promoters (R8 or R11). Positive colonies show an amplification product at 2500bp (lanes 1,2,4-8, 10, 13-16)</DIV></DIV></DIV><UL><LI> plenty of colonies on the P56+R8 and P56+R11, NO colonies on control plate with SAPed backbone, colony PCR revealed 50% of positive clones
</LI></UL></DIV><DIV class="printfooter">
Retrieved from &quot;<A href="http://2010.igem.org/Team:Heidelberg/Notebook/miRNA_Kit/September">http://2010.igem.org/Team:Heidelberg/Notebook/miRNA_Kit/September</A>&quot;</DIV></DIV><DIV id="footer-box" class="noprint"><DIV id="footer"><UL id="f-list"><LI id="t-recentchanges"><A href="/Special:RecentChanges" title="Recent changes">Recent changes</A></LI><LI id="t-whatlinkshere"><A href="/Special:WhatLinksHere/Team:Heidelberg/Notebook/miRNA_Kit/September" title="List of all wiki pages that link here [j]" accesskey="j">What links here</A></LI><LI id="t-recentchangeslinked"><A href="/Special:RecentChangesLinked/Team:Heidelberg/Notebook/miRNA_Kit/September" title="Recent changes in pages linked from this page [k]" accesskey="k">Related changes</A></LI><LI id="t-specialpages"><A href="/Special:SpecialPages" title="List of all special pages [q]" accesskey="q">Special pages</A></LI><LI><A href="/Special:Preferences">My preferences</A></LI></UL></DIV><DIV id="footer"><UL id="f-list"><LI id="t-print"><A href="/wiki/index.php?title=Team:Heidelberg/Notebook/miRNA_Kit/September&amp;printable=yes" title="Printable version of this page [p]" accesskey="p">Printable version</A></LI><LI id="t-permalink"><A href="/wiki/index.php?title=Team:Heidelberg/Notebook/miRNA_Kit/September&amp;oldid=174745" title="Permanent link to this revision of the page">Permanent link</A></LI><LI id="privacy"><A href="/2010.igem.org:Privacy_policy" title="2010.igem.org:Privacy policy">Privacy policy</A></LI><LI id="disclaimer"><A href="/2010.igem.org:General_disclaimer" title="2010.igem.org:General disclaimer">Disclaimers</A></LI></UL></DIV></DIV></BODY></HTML>