<HTML xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en" dir="ltr">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki  ltr ns-0 ns-subject page-Team_Heidelberg_Notebook_Homology_Based_October"><DIV id="globalWrapper"><DIV id="top-section"><DIV id="p-logo"><A href="/Main_Page" title="Main Page">&quot;
	    </A></DIV><DIV id="menubar" class="left-menu noprint"><UL><LI class="selected"><A href="/Team:Heidelberg/Notebook/Homology_Based/October">Page               </A></LI><LI class="new"><A href="/wiki/index.php?title=Talk:Team:Heidelberg/Notebook/Homology_Based/October&amp;action=edit&amp;redlink=1">Discussion               </A></LI><LI><A href="/wiki/index.php?title=Team:Heidelberg/Notebook/Homology_Based/October&amp;action=edit">View source               </A></LI><LI><A href="/wiki/index.php?title=Team:Heidelberg/Notebook/Homology_Based/October&amp;action=history">History               </A></LI><LI style="color:#808080;cursor:default">teams</LI></UL></DIV><DIV class="right-menu noprint" id="menubar"><UL><LI id="pt-login"><A href="/wiki/index.php?title=Special:UserLogin&amp;returnto=Team:Heidelberg/Notebook/Homology_Based/October" title="You are encouraged to log in; however, it is not mandatory [o]" accesskey="o">Log in</A></LI></UL></DIV><DIV id="search-controls" class="noprint"><FORM action="/Special:Search" id="searchform"> </FORM></DIV></DIV><DIV id="content"><H1 class="firstHeading">Team:Heidelberg/Notebook/Homology Based/October</H1><DIV id="bodyContent"><H3 id="siteSub" class="noprint">From 2010.igem.org</H3><DIV id="header"><DIV id="menu"><UL id="nav"><LI id="menu_home"><A class="aHome" href="https://2010.igem.org/Team:Heidelberg">Home</A></LI><LI id="menu_team"><A class="aTeam" href="https://2010.igem.org/Team:Heidelberg/Team">Team</A><TABLE class="menutable" cellpadding="0" cellspacing="0"><TBODY><TR><TD class="lefttdtopteam"><A class="aTeamOverview" href="https://2010.igem.org/Team:Heidelberg/Team">Overview</A><A class="aTeamMembers" href="https://2010.igem.org/Team:Heidelberg/Team/Members">Students</A><A class="aTeamMembers" href="https://2010.igem.org/Team:Heidelberg/Team/Members#Instructors">Instructors</A><A class="aTeamMembers" href="https://2010.igem.org/Team:Heidelberg/Team/Members#Advisors">Advisors</A><A class="aInstitutes" href="https://2010.igem.org/Team:Heidelberg/Team/Institutes">Institutes</A><A class="aGallery" href="https://2010.igem.org/Team:Heidelberg/Team/Gallery">Gallery</A><A class="aDance" href="https://2010.igem.org/Team:Heidelberg/midance">miDANCE</A><A class="amithanks" href="https://2010.igem.org/Team:Heidelberg/Team/miThanks">Acknowledgements</A></TD><TD class="righttdtop"> </TD></TR></TBODY></TABLE></LI><LI id="menu_project"><A class="aProject" href="https://2010.igem.org/Team:Heidelberg/Project">Project</A><TABLE class="menutable" cellpadding="0" cellspacing="0"><TBODY><TR><TD class="lefttdtopproject"><A class="aAbstract" href="https://2010.igem.org/Team:Heidelberg/Project">Abstract</A><A class="aIntroduction" href="https://2010.igem.org/Team:Heidelberg/Project/Introduction">Introduction</A><A class="aProject_miRNA_Kit" href="https://2010.igem.org/Team:Heidelberg/Project/miRNA_Kit">Synthetic miRNA Kit</A><A class="aPro_Capsid_Shuffling" href="https://2010.igem.org/Team:Heidelberg/Project/Capsid_Shuffling">Capsid Shuffling</A><A class="aPro_Homology" href="https://2010.igem.org/Team:Heidelberg/Project/Capsid_Shuffling/Homology_Based">    Homology Based</A><A class="aPro_Virobytes" href="https://2010.igem.org/Team:Heidelberg/Project/Capsid_Shuffling/ViroBytes">    ViroBytes</A><A class="aPro_miMeasure" href="https://2010.igem.org/Team:Heidelberg/Project/miMeasure">miMeasure</A><A class="aPro_Mouse" href="https://2010.igem.org/Team:Heidelberg/Project/Mouse_Infection"><I>in vivo</I> Study</A><A class="aPro_Summary" href="https://2010.igem.org/Team:Heidelberg/Project/Summary">Summary &amp; Outlook</A><A class="aPro_Ref" href="https://2010.igem.org/Team:Heidelberg/Project/References">References</A><A class="aPro_Ref" href="https://2010.igem.org/Team:Heidelberg/Project/Contributions">Contributions</A></TD><TD class="righttdtop"> </TD></TR></TBODY></TABLE></LI><LI id="menu_parts"><A class="aParts" href="https://2010.igem.org/Team:Heidelberg/Parts">Parts</A><TABLE class="menutable" cellpadding="0" cellspacing="0"><TBODY><TR><TD class="lefttdtop"><A class="aparts_ger" href="https://2010.igem.org/Team:Heidelberg/Parts">Parts</A><A class="aparts_char" href="https://2010.igem.org/Team:Heidelberg/Parts/Characterization">Characterization</A></TD><TD class="righttdtop"> </TD></TR></TBODY></TABLE></LI><LI id="menu_modeling"><A class="aModeling" href="https://2010.igem.org/Team:Heidelberg/Modeling">Modeling</A><TABLE class="menutable" cellpadding="0" cellspacing="0"><TBODY><TR><TD class="lefttdtop"><A class="amode_over" href="https://2010.igem.org/Team:Heidelberg/Modeling">Introduction</A><A class="amiGUI" href="https://2010.igem.org/Team:Heidelberg/Modeling/miGUI">miBEAT</A><A class="amirockdown" href="https://2010.igem.org/Team:Heidelberg/Modeling/miRockdown">miRockdown</A><A class="amibsdesigner" href="https://2010.igem.org/Team:Heidelberg/Modeling/miBSdesigner">miBS designer</A><A class="amispec" href="https://2010.igem.org/Team:Heidelberg/Modeling/miSpec">mUTING</A><A class="amodelset" href="https://2010.igem.org/Team:Heidelberg/Modeling/trainingset">Descriptions</A></TD><TD class="righttdtop"> </TD></TR></TBODY></TABLE></LI><LI id="menu_notebook"><A class="aNotebook" href="https://2010.igem.org/Team:Heidelberg/Notebook">Notebook</A><TABLE class="menutable" cellpadding="0" cellspacing="0"><TBODY><TR><TD class="lefttdtop"><A class="aNote_miRNA_kit" href="https://2010.igem.org/Team:Heidelberg/Notebook/miRNA_Kit">Synthetic miRNA Kit</A><A class="aNote_Capsid_Shuffling">Capsid Shuffling</A><A class="aNote_Homology" href="https://2010.igem.org/Team:Heidelberg/Notebook/Capsid_Shuffling/Homology_Based">    Homology Based</A><A class="aNote_Virobytes" href="https://2010.igem.org/Team:Heidelberg/Notebook/Capsid_Shuffling/ViroBytes">    ViroBytes</A><A class="aNote_miMeasure" href="https://2010.igem.org/Team:Heidelberg/Notebook/miMeasure">miMeasure</A><A class="aNote_BSdesign" href="https://2010.igem.org/Team:Heidelberg/Notebook/BSDesign">Binding Site Design </A><A class="aNote_Mouse" href="https://2010.igem.org/Team:Heidelberg/Notebook/Mouse_Infection"><I>in vivo</I> Study</A><A class="aNote_Modeling" href="https://2010.igem.org/Team:Heidelberg/Notebook/Modeling">Modeling</A><A class="aNote_Material" href="https://2010.igem.org/Team:Heidelberg/Notebook/Material">Materials</A><A class="aNote_Methods" href="https://2010.igem.org/Team:Heidelberg/Notebook/Methods">Methods</A><A class="aNote_Psycho" href="https://2010.igem.org/Team:Heidelberg/Human_Practices/Psychology/Notebook">Psychological Survey</A><A class="aNote_Philo" href="https://2010.igem.org/Team:Heidelberg/Human_Practices/Philosophy/Part2">Philosophical Reflection</A><A class="aNote_Biosafety" href="https://2010.igem.org/Team:Heidelberg/Notebook/Biosafety">Biosafety</A></TD><TD class="righttdtop"> </TD></TR></TBODY></TABLE></LI><LI id="menu_human_practices"><A class="aHuman_Practices" href="https://2010.igem.org/Team:Heidelberg/Human Practices">Human Practices</A><TABLE class="menutable" cellpadding="0" cellspacing="0"><TBODY><TR><TD class="lefttdtop"><A class="aPsycho" href="https://2010.igem.org/Team:Heidelberg/Human Practices/Psychology">Psychological Survey</A><A class="aPhilo" href="https://2010.igem.org/Team:Heidelberg/Human Practices/Philosophy">Philosophical Reflection</A></TD><TD class="righttdtop"> </TD></TR></TBODY></TABLE></LI><LI id="menu_support"><A class="aSupport" href="https://2010.igem.org/Team:Heidelberg/Support">Sponsors</A></LI></UL></DIV></DIV><TABLE cellpadding="5" cellspacing="0" align="center" style="text-align: center; color:#009be1; border: 1.5px solid #000000;"><TBODY><TR><TH colspan="7" style="background:#f09600;"><A href="https://2010.igem.org/Team:Heidelberg/Notebook/Homology_Based/August" class="external text" rel="nofollow"><FONT color="white">August</FONT></A></TH></TR><TR style="background:#f09600; color:white"><TD width="20pt"><B>M</B></TD><TD width="20pt"><B>T</B></TD><TD width="20pt"><B>W</B></TD><TD width="20pt"><B>T</B></TD><TD width="20pt"><B>F</B></TD><TD width="20pt"><B>S</B></TD><TD width="20pt"><B>S</B></TD></TR><TR style="background:#f2f2f2; color:#009be1"><TD><B>1</B></TD></TR><TR style="background:#f2f2f2; color:#009be1"><TD><B>2</B></TD><TD><B>3</B></TD><TD><B>4</B></TD><TD><B>5</B></TD><TD><B>6</B></TD><TD><B>7</B></TD><TD><B>8</B></TD></TR><TR style="background:#f2f2f2; color:#009be1"><TD><B>9</B></TD><TD><B>10</B></TD><TD><B>11</B></TD><TD><B>12</B></TD><TD><B>13</B></TD><TD><B>14</B></TD><TD><B>15</B></TD></TR><TR style="background:#f2f2f2; color:#009be1"><TD><B>16</B></TD><TD><B>17</B></TD><TD><B>18</B></TD><TD><B>19</B></TD><TD><B>20</B></TD><TD><B>21</B></TD><TD><B>22</B></TD></TR><TR style="background:#f2f2f2; color:#009be1"><TD><B>23</B></TD><TD><B>24</B></TD><TD><B>25</B></TD><TD><B>26</B></TD><TD><B>27</B></TD><TD><B>28</B></TD><TD><B>29</B></TD></TR><TR style="background:#f2f2f2; color:#009be1"><TD><B>30</B></TD><TD><B><A href="https://2010.igem.org/Team:Heidelberg/Notebook/Homology_Based/August#31.2F08.2F2010" class="external text" rel="nofollow">31</A></B></TD><TD colspan="5"><P><SPAN style="color:#ffffff">-</SPAN></P></TD></TR></TBODY></TABLE><TABLE cellpadding="5" cellspacing="0" align="center" style="text-align: center; color:#78b41e; border: 1.5px solid #333333;"><TBODY><TR><TH colspan="7" style="background:#009be1;"><A href="https://2010.igem.org/Team:Heidelberg/Notebook/Homology_Based/September" class="external text" rel="nofollow"><FONT color="#ffecba">September</FONT></A></TH></TR><TR style="background:#009be1; color:white"><TD width="20pt"><B>M</B></TD><TD width="20pt"><B>T</B></TD><TD width="20pt"><B>W</B></TD><TD width="20pt"><B>T</B></TD><TD width="20pt"><B>F</B></TD><TD width="20pt"><B>S</B></TD><TD width="20pt"><B>S</B></TD></TR><TR style="background:#f2f2f2; color:#78b41e"><TD><B><A href="https://2010.igem.org/Team:Heidelberg/Notebook/Homology_Based/September#01.2F09.2F2010" class="external text" rel="nofollow">1</A></B></TD><TD><B><A href="https://2010.igem.org/Team:Heidelberg/Notebook/Homology_Based/September#02.2F09.2F2010" class="external text" rel="nofollow">2</A></B></TD><TD><B><A href="https://2010.igem.org/Team:Heidelberg/Notebook/Homology_Based/September#03.2F09.2F2010" class="external text" rel="nofollow">3</A></B></TD><TD><B>4</B></TD><TD><B>5</B></TD></TR><TR style="background:#f2f2f2; color:#78b41e"><TD><B><A href="https://2010.igem.org/Team:Heidelberg/Notebook/Homology_Based/September#06.2F09.2F2010" class="external text" rel="nofollow">6</A></B></TD><TD><B><A href="https://2010.igem.org/Team:Heidelberg/Notebook/Homology_Based/September#07.2F09.2F2010" class="external text" rel="nofollow">7</A></B></TD><TD><B><A href="https://2010.igem.org/Team:Heidelberg/Notebook/Homology_Based/September#08.2F09.2F2010" class="external text" rel="nofollow">8</A></B></TD><TD><B><A href="https://2010.igem.org/Team:Heidelberg/Notebook/Homology_Based/September#09.2F09.2F2010" class="external text" rel="nofollow">9</A></B></TD><TD><B><A href="https://2010.igem.org/Team:Heidelberg/Notebook/Homology_Based/September#10.2F09.2F2010" class="external text" rel="nofollow">10</A></B></TD><TD><B>11</B></TD><TD><B>12</B></TD></TR><TR style="background:#f2f2f2; color:#78b41e"><TD><B>13</B></TD><TD><B>14</B></TD><TD><B>15</B></TD><TD><B>16</B></TD><TD><B>17</B></TD><TD><B>18</B></TD><TD><B>19</B></TD></TR><TR style="background:#f2f2f2; color:#78b41e"><TD><B><A href="https://2010.igem.org/Team:Heidelberg/Notebook/Homology_Based/September#20.2F09.2F2010" class="external text" rel="nofollow">20</A></B></TD><TD><B><A href="https://2010.igem.org/Team:Heidelberg/Notebook/Homology_Based/September#21.2F09.2F2010" class="external text" rel="nofollow">21</A></B></TD><TD><B><A href="https://2010.igem.org/Team:Heidelberg/Notebook/Homology_Based/September#22.2F09.2F2010" class="external text" rel="nofollow">22</A></B></TD><TD><B><A href="https://2010.igem.org/Team:Heidelberg/Notebook/Homology_Based/September#23.2F09.2F2010" class="external text" rel="nofollow">23</A></B></TD><TD><B><A href="https://2010.igem.org/Team:Heidelberg/Notebook/Homology_Based/September#24.2F09.2F2010" class="external text" rel="nofollow">24</A></B></TD><TD><B><A href="https://2010.igem.org/Team:Heidelberg/Notebook/Homology_Based/September#25.2F09.2F2010" class="external text" rel="nofollow">25</A></B></TD><TD><B><A href="https://2010.igem.org/Team:Heidelberg/Notebook/Homology_Based/September#26.2F09.2F2010" class="external text" rel="nofollow">26</A></B></TD></TR><TR style="background:#f2f2f2; color:#78b41e"><TD><B><A href="https://2010.igem.org/Team:Heidelberg/Notebook/Homology_Based/September#27.2F09.2F2010" class="external text" rel="nofollow">27</A></B></TD><TD><B><A href="https://2010.igem.org/Team:Heidelberg/Notebook/Homology_Based/September#28.2F09.2F2010" class="external text" rel="nofollow">28</A></B></TD><TD><B><A href="https://2010.igem.org/Team:Heidelberg/Notebook/Homology_Based/September#29.2F09.2F2010" class="external text" rel="nofollow">29</A></B></TD><TD><B><A href="https://2010.igem.org/Team:Heidelberg/Notebook/Homology_Based/September#30.2F30.2F2010" class="external text" rel="nofollow">30</A></B></TD></TR><TR style="background:#f2f2f2; color:#78b41e"><TD colspan="7"><P><SPAN style="color:#ffffff">-</SPAN></P></TD></TR></TBODY></TABLE><TABLE cellpadding="5" cellspacing="0" align="center" style="text-align: center; color:#f09600; border: 1.5px solid #333333;"><TBODY><TR><TH colspan="7" style="background:#78b41e;"><A href="https://2010.igem.org/Team:Heidelberg/Notebook/Homology_Based/October" class="external text" rel="nofollow"><FONT color="white">October</FONT></A></TH></TR><TR style="background:#78b41e; color:white"><TD width="20pt"><B>M</B></TD><TD width="20pt"><B>T</B></TD><TD width="20pt"><B>W</B></TD><TD width="20pt"><B>T</B></TD><TD width="20pt"><B>F</B></TD><TD width="20pt"><B>S</B></TD><TD width="20pt"><B>S</B></TD></TR><TR style="background:#f2f2f2; color:#78b41e"><TD><B><A href="https://2010.igem.org/Team:Heidelberg/Notebook/Homology_Based/October#01.2F10.2F2010" class="external text" rel="nofollow">1</A></B></TD><TD><B><A href="https://2010.igem.org/Team:Heidelberg/Notebook/Homology_Based/October#02.2F10.2F2010" class="external text" rel="nofollow">2</A></B></TD><TD><B><A href="https://2010.igem.org/Team:Heidelberg/Notebook/Homology_Based/October#03.2F10.2F2010" class="external text" rel="nofollow">3</A></B></TD></TR><TR style="background:#f2f2f2; color:#78b41e"><TD><B><A href="https://2010.igem.org/Team:Heidelberg/Notebook/Homology_Based/October#04.2F10.2F2010" class="external text" rel="nofollow">4</A></B></TD><TD><B><A href="https://2010.igem.org/Team:Heidelberg/Notebook/Homology_Based/October#05.2F10.2F2010" class="external text" rel="nofollow">5</A></B></TD><TD><B><A href="https://2010.igem.org/Team:Heidelberg/Notebook/Homology_Based/October#06.2F10.2F2010" class="external text" rel="nofollow">6</A></B></TD><TD><B><A href="https://2010.igem.org/Team:Heidelberg/Notebook/Homology_Based/October#07.2F10.2F2010" class="external text" rel="nofollow">7</A></B></TD><TD><B><A href="https://2010.igem.org/Team:Heidelberg/Notebook/Homology_Based/October#08.2F10.2F2010" class="external text" rel="nofollow">8</A></B></TD><TD><B><A href="https://2010.igem.org/Team:Heidelberg/Notebook/Homology_Based/October#09.2F10.2F2010" class="external text" rel="nofollow">9</A></B></TD><TD><B><A href="https://2010.igem.org/Team:Heidelberg/Notebook/Homology_Based/October#10.2F10.2F2010" class="external text" rel="nofollow">10</A></B></TD></TR><TR style="background:#f2f2f2; color:#78b41e"><TD><B><A href="https://2010.igem.org/Team:Heidelberg/Notebook/Homology_Based/October#11.2F10.2F2010" class="external text" rel="nofollow">11</A></B></TD><TD><B><A href="https://2010.igem.org/Team:Heidelberg/Notebook/Homology_Based/October#13.2F10.2F2010" class="external text" rel="nofollow">13</A></B></TD><TD><B><A href="https://2010.igem.org/Team:Heidelberg/Notebook/Homology_Based/October#14.2F10.2F2010" class="external text" rel="nofollow">14</A></B></TD><TD><B><A href="https://2010.igem.org/Team:Heidelberg/Notebook/Homology_Based/October#15.2F10.2F2010" class="external text" rel="nofollow">15</A></B></TD><TD><B>16</B></TD><TD><B>17</B></TD><TD><B>18</B></TD></TR><TR style="background:#f2f2f2; color:#78b41e"><TD><B>19</B></TD><TD><B>20</B></TD><TD><B>21</B></TD><TD><B>22</B></TD><TD><B>23</B></TD><TD><B>24</B></TD><TD><B>25</B></TD></TR><TR style="background:#f2f2f2; color:#78b41e"><TD><B>26</B></TD><TD><B>27</B></TD><TD><B>28</B></TD><TD><B>29</B></TD><TD><B>30</B></TD></TR><TR style="background:#f2f2f2; color:#78b41e"><TD colspan="7"><P><SPAN style="color:#ffffff">-</SPAN></P></TD></TR></TBODY></TABLE><P> </P></DIV><H1><SPAN class="mw-headline" id="October">October</SPAN></H1><H2><SPAN class="mw-headline" id="01.2F10.2F2010">01/10/2010</SPAN></H2><UL><LI> Harvesting of the 50 plates (15 cm) from the previous day and picking of 50 clones were done, and the collected bacteria were incubated in a 700 mL LB culture for 3 hours with shaking, after which a megaprep (Invitrogen) was done.
</LI></UL><UL><LI> mini-preps of the 50 clones were done, and 10 samples were sent for sequencing.
</LI></UL><H2><SPAN class="mw-headline" id="02.2F10.2F2010">02/10/2010</SPAN></H2><UL><LI> Transfection of 13 T flasks (150 cm2, Hek 293 cells) was done, with the plasmid DNA purified the previous day (the shuffled cap genes library) and an adeno-helper plasmid, 35 micrograms from each plasmid for each flask, using HBSS for the transfection. A GFP control for transfection was also used.
</LI></UL><UL><LI> Analysis of the sequencing results for the 10 samples identified good shuffling of the cap genes. More mini-preps were inoculated for the same 50 clones.
</LI></UL><H2><SPAN class="mw-headline" id="03.2F10.2F2010">03/10/2010</SPAN></H2><UL><LI> Mini-preps for the samples from the previous day were done.
</LI></UL><H2><SPAN class="mw-headline" id="05.2F10.2F2010">05/10/2010</SPAN></H2><UL><LI> Harvesting of the cells for virus production was done by:
</LI></UL><P>- Washing the flasks with the medium they contain, and collecting the media from 13 flasks in 500 ml corning conical centrifuge tubes.
</P><P>- The cells were pelleted by centrifuging at 1500 rpm for 15 min at 4 ͦC
- Discard supernatant
- Washing: Resuspend the cells in 1X PBS and transfer to a 50 ml falcon
- Pellet the cells again by centrifuging at 1500 rpm for 15 min at 4 ͦC
- Discard supernatant
</P><UL><LI> Freeze-thaw cylces for cell rupturing:
</LI></UL><P>- Add 20 ml lysis buffer to the cell pellet
- Put in liquid nitrogen for 5 min
- Thaw at 37ͦC
</P><P>The cycles were repeated for 5 times, and the sample was frozen for next day.
</P><H2><SPAN class="mw-headline" id="06.2F10.2F2010">06/10/2010</SPAN></H2><UL><LI> The preparation for virus purification using iodixanol gradient was initiated according to the following:
</LI></UL><P>- The sample from previous day was sonicated in a sonication bath for 1 min 20 sec.
- 50 units/ml benzonase were added to destroy RNA and DNA from non-AAV sources, which could later interfere with qPCR.
- The sample was incubated at 37 ͦC for 30 min, and vortexed every 10 min.
- The sample was then centrifuged for 15 min at 3270 xg
- The supernatant was transferred into a new 50 ml falcon
</P><UL><LI> Virus purification using iodixanol gradient centrifugation:
</LI></UL><P>- A Pasteur pipette was plugged into a Beckman quick-seal centrifuge tube 
- Using a 1000 µl pipette, 20 ml of the virus suspension were added the gradient was poured through the Pasteur pipette in the following order:
</P><PRE> - 7 ml 15% Iodixanol solution 
 - 5 ml 25% Iodixanol solution 
 - 4 ml 40% Iodixanol solution 
 - 4 ml 60% Iodixanol solution 
</PRE><P>- The Pasteur pipette was carefully removed and the tube was sealed. A tare tube was prepared in the same way.
</P><P>- Ultracentrifugation of the sample was carried out at 50,000 rpm for 2:30 hours at 4ͦC.
</P><P>- The virus-containing phase is the 40% iodixanol phase, which was sucked out using a syringe and a needle. The AAV sample was divided into aliquotes, one of them stored at 4 ͦC  for qPCR the next day, and the rest were frozen in liquid nitrogen then transferred to -20 ͦC.
</P><H2><SPAN class="mw-headline" id="07.2F10.2F2010">07/10/2010</SPAN></H2><UL><LI> qPCR of the sample from the previous day was carried out according to the protocol in the Methods section, and the titre of the viral library was determined to be 1.3*10^11 viral genomes/ml.
</LI></UL><UL><LI> The selection rounds of the viruses in the library were initiated on Huh-7 cells, the infection of cells was carried out as follows:
</LI></UL><P>- AAV MOI 100, 10 and 1 were used on Huh-7 cells plated on 6-well plates, and the viruses were allowed to infect cells for 1 hour and 30 min. 
</P><P>- Adeno-virus 5 (MOI 10) was added then in a Biohazard II lab under the superviosion of lab personnel.
</P><P>- Cells were incubated in the Biohazard II lab.
</P><H2><SPAN class="mw-headline" id="08.2F10.2F2010">08/10/2010</SPAN></H2><P>- A non-ITR AAV helper vector was obtained and maxi-preped to clone into it the cap genes from the 50 single clones picked.
</P><H2><SPAN class="mw-headline" id="09.2F10.2F2010">09/10/2010</SPAN></H2><P>- The non-ITR vector was cut with AscI and PacI, and so was the pTR-UF3 containg cap genes from the 50 clones. The bands were purified from the gel.
</P><H2><SPAN class="mw-headline" id="10.2F10.2F2010">10/10/2010</SPAN></H2><UL><LI> The cap genes were ligated into the non-ITR vector, and Hek 293 cells were prepared in 24-well plates for transfection.
</LI></UL><UL><LI> The first selection round Huh-7 cells infect with AAV library and Adeno-virus 5 were monitored under the microscope, and the typical cytopathic effect caused by Adeno-5 infection (roundening of cells) was observed. Cells were harvested by washing the wells with the media (in this case, one well of the 6-well plate was used).
</LI></UL><H2><SPAN class="mw-headline" id="11.2F10.2F2010">11/10/2010</SPAN></H2><UL><LI> The harvested cells from the previous day were incubated at 56 ͦC for 20 min to inactivate adeno virus 5. Then, freeze-thaw cycles (between liquid nitrogen and 37ͦC, see above) were carried out to lyse the cells and get the AAVs out. A 5-minute centrifugation at full speed was then done, and the supernatant containing the 1st selection round AAVs was collected and aliqueted and stored at -20ͦC (initial freezing with liquid nitrogen), or used immediately on the 2nd selection round Huh-7 cells as done previously. This time, instead of MOI for AAV, 100, 10 or 1 microliters of the supernatant from the first selection round were used on 3 wells of a 6-well plate, and the same volumes on the other 3 wells, however, for those the wells were washed and medium changed after 30 min of addition of AAV to increase the selection pressure. For both sets of wells, Adeno-5 (MOI 10) was added after 2 hours of initial addition of AAV.
</LI></UL><UL><LI> A triple transfection (1:1:1) of each non-ITR plasmid with cap genes from the 50 clones (see above) and Adeno-helper plasmid and a YFP construct with ITRs was carried out on Hek 293 cells in 24-well plates using FuGene. This was done to produce AAVs that contain YFP which will be expressed when these viruses successfully infect the target cells on which they will be tested. The viruses with best capsids will be able to infect target cells.
</LI></UL><H2><SPAN class="mw-headline" id="13.2F10.2F2010">13/10/2010</SPAN></H2><UL><LI> Mouse Primary Hepatocytes were received and plated on one 6-well plate plus two 24-well plates, which were previously coated with collagen.
</LI></UL><UL><LI> The 6-well plate with primary hepatocytes was infected with the library AAVs, after 1 hour and a half MOI 10 and 100 Adeno-5 were added, starting the first selection round on these cells
</LI></UL><H2><SPAN class="mw-headline" id="14.2F10.2F2010">14/10/2010</SPAN></H2><UL><LI> Viruses from the 24-well-plate-Hek 293 cells from two days before were collected after monitoring the expression of YFP in those cells. Freeze-thaw cycles were done as previously, and the supernatant after centrifugation was collected. Each well represented one AAV.
</LI></UL><UL><LI> 30 microliters of the supernatant collected for each AAV were used to infect target cells, which were:
</LI></UL><P>- two 24-well plates HepG2 cells
</P><P>- Two 24-well plates Huh-7 cells
</P><P>- Two 24-well plates primary hepatocytes
</P><H2><SPAN class="mw-headline" id="15.2F10.2F2010">15/10/2010</SPAN></H2><UL><LI> AAVs from the second selection round on Huh-7 were collected from the supernatants after freeze-thaw cycles (see previous days), and stored at-20 C to infect the third selection round Huh-7 cells under the same conditions and in the same way as in the second selection round.
</LI></UL><UL><LI> AAVs from the first selection round on primary hepatocytes were harvested using freeze-thaw cycles as before (see previous days), and the supernatants were kept at -20 C for the second selection round (primary hepatocytes are received every Monday).
</LI></UL><H2><SPAN class="mw-headline" id="16.2F10.2F2010">16/10/2010</SPAN></H2><UL><LI> FACS measurements for the 6 24-well plates from two days earlier were done to measure YFP expression for the 48 different clones (AAVs!)in different kinds of cells. Cells were trypsinized (100 microliters trypsin) and then 500 microliters of buffer were added. Each well was measured for 1 min. The results showed no YFP expression for 47 of the samples in HepG2 and Huh-7, whereas one sample was positive (#43). This was confirmed further by microscopy. For the primary hepatocyes no sample showed any positive result.
</LI></UL><P>Clone #43 was later injected into mice with a luc2-SV40 construct.
</P><H2><SPAN class="mw-headline" id="17.2F10.2F2010">17/10/2010</SPAN></H2><UL><LI> Huh-7 cells were infected with AAVs in the supernatant from the second selection round, thus starting the third selection round. The same conditions and volumes as in the second selection round were used.
</LI></UL><H2><SPAN class="mw-headline" id="18.2F10.2F2010">18/10/2010</SPAN></H2><UL><LI> The single sample of AAV that showed a positive result for YFP measurement was sent for sequencing.
</LI></UL><UL><LI> Primary hepatocytes were received and kept at 4 C for seeding the next day on 6-well plates. Collagen was used to coat the plates as before (Polymerization under UV for 1 hour was done).
</LI></UL><H2><SPAN class="mw-headline" id="19.2F10.2F2010">19/10/2010</SPAN></H2><UL><LI> Primary hepatocytes were seeded, and after 5 hours they were infected by AAVs from the first selection round. Adeno virus 5 MOI 100 was used, and varying volumes of the AAV-containing supernatant from the first selection round were applied.
</LI></UL><H2><SPAN class="mw-headline" id="21.2F10.2F2010">21/10/2010</SPAN></H2><UL><LI> Third selection round AAVs from Huh-7 and second selection round AAVs from primary hepatocytes were harvested by freeze-thaw cycles (see above).
</LI></UL><UL><LI> Viral DNA from the first, second and third selection rounds on Huh-7 and first and second selection rounds on primary hepatocytes was extracted using QIAmp MiniElute kit.
</LI></UL><H2><SPAN class="mw-headline" id="22.2F10.2F2010">22/10/2010</SPAN></H2><UL><LI> A PCR reaction using a primer mix was carried out on the DNA extracted the previous day (plus 20 of the clones picked earlier and clones into a non-ITR vector, see above) to amplify the cap genes and introduce SwaI and SpeI sites, which were then used to clone the cap genes into a Wilson wild-type vector. There was a problem with the non-ITR vector used earlier, in the sense that the expression of Cap and Rep genes was not comparable, so not many functional viral particles could be produced, and for this reason the Wilson wildtype vector was used instead.
</LI></UL><P>The conditions for the PCR were the same as the second PCR for amplification of the shuffled capsids (see above). The bands for cap genes (2.2 KB) were purified from the gel, then digested with SwaI and SpeI, together with Wilson wild-type vector (standard 20 microliter digestion in NEB buffer 4, for 2 hours at 25 C). Qiagen nucleotide removal followed for the cap genes, and the vector was gel-purified. A ligation reaction of cap genes into the vector was carried out with NEB Quick Ligase (according to the standard manufacturer's recommendations), for 1 hour 30 min at room temperature.
</P><P><A href="/File:102310_20clones.gif" title="File:102310 20clones.gif">300px</A>
PCR for Huh-7 clones
</P><P>
PCR for Huh-7 clones
</P><P>
PCR for Huh-7 clones
</P><P>
Digested vector
</P><H2><SPAN class="mw-headline" id="23.2F10.2F2010">23/10/2010</SPAN></H2><UL><LI> The vector with cap genes from the previous day for the different selection rounds on Huh-7 and primary hepatocytes was transformed into Top10 cells.
</LI></UL><H2><SPAN class="mw-headline" id="24.2F10.2F2010">24/10/2010</SPAN></H2><UL><LI> 20 clones were picked from each selection round on Huh-7 and primary hepatocytes from the transformation on the previous day and used to inoculate mini-preps.
</LI></UL><H2><SPAN class="mw-headline" id="25.2F10.2F2010">25/10/2010</SPAN></H2><UL><LI> mini-preps were run for the samples from the previous day, and triple transfection of the Wilson construct with cap genes, Adeno-helper plasmid and a YFP construct with ITRs was done on 4 24-well plates of Hek293 cells in replica, each well containing an AAV construct from a different selection round on Huh-7 cells with different conditions during the selection process (washing after 30 min of AAV addition or no washing, see above).
</LI></UL><P>From each selection round 12 samples were transfected, plus AAV wildtypes 1-12.
</P><H2><SPAN class="mw-headline" id="26.2F10.2F2010">26/10/2010</SPAN></H2><UL><LI> The AAVs were harvested from the cells of one set of the plates (see previous day) using freeze-thaw cycles, and 20 microliters of the supernatant after centrifugation were used to infect HeLa, Huh-7 and primary hepatocyte cells in a 96-well plate format. This was done in replicates. The plates are then used for FACS analysis of YFP expression.
</LI></UL><UL><LI> The clones picked were sent for sequencing.
</LI></UL><H2><SPAN class="mw-headline" id="27.2F10.2F2010">27/10/2010</SPAN></H2><UL><LI> A western blot and a dot blot for the other set of 24-well plates from two days earlier (see above) were initiated, with an antibody against a conserved region in cap genes.
</LI></UL></DIV><DIV class="printfooter">
Retrieved from &quot;<A href="http://2010.igem.org/Team:Heidelberg/Notebook/Homology_Based/October">http://2010.igem.org/Team:Heidelberg/Notebook/Homology_Based/October</A>&quot;</DIV></DIV><DIV id="footer-box" class="noprint"><DIV id="footer"><UL id="f-list"><LI id="t-recentchanges"><A href="/Special:RecentChanges" title="Recent changes">Recent changes</A></LI><LI id="t-whatlinkshere"><A href="/Special:WhatLinksHere/Team:Heidelberg/Notebook/Homology_Based/October" title="List of all wiki pages that link here [j]" accesskey="j">What links here</A></LI><LI id="t-recentchangeslinked"><A href="/Special:RecentChangesLinked/Team:Heidelberg/Notebook/Homology_Based/October" title="Recent changes in pages linked from this page [k]" accesskey="k">Related changes</A></LI><LI id="t-specialpages"><A href="/Special:SpecialPages" title="List of all special pages [q]" accesskey="q">Special pages</A></LI><LI><A href="/Special:Preferences">My preferences</A></LI></UL></DIV><DIV id="footer"><UL id="f-list"><LI id="t-print"><A href="/wiki/index.php?title=Team:Heidelberg/Notebook/Homology_Based/October&amp;printable=yes" title="Printable version of this page [p]" accesskey="p">Printable version</A></LI><LI id="t-permalink"><A href="/wiki/index.php?title=Team:Heidelberg/Notebook/Homology_Based/October&amp;oldid=208372" title="Permanent link to this revision of the page">Permanent link</A></LI><LI id="privacy"><A href="/2010.igem.org:Privacy_policy" title="2010.igem.org:Privacy policy">Privacy policy</A></LI><LI id="disclaimer"><A href="/2010.igem.org:General_disclaimer" title="2010.igem.org:General disclaimer">Disclaimers</A></LI></UL></DIV></DIV></BODY></HTML>