"
Page
Discussion
View source
History
teams
Log in
 
Team:Heidelberg/Notebook/miRNA Kit/August
From 2010.igem.org
Home
Team
Overview
Students
Instructors
Advisors
Institutes
Gallery
miDANCE
Acknowledgements
 
Project
Abstract
Introduction
Synthetic miRNA Kit
Capsid Shuffling
    Homology Based
    ViroBytes
miMeasure
in vivo Study
Summary & Outlook
References
Contributions
 
Parts
Parts
Characterization
 
Modeling
Introduction
miBEAT
miRockdown
miBS designer
mUTING
Descriptions
 
Notebook
Synthetic miRNA Kit
Capsid Shuffling
    Homology Based
    ViroBytes
miMeasure
Binding Site Design
in vivo Study
Modeling
Materials
Methods
Psychological Survey
Philosophical Reflection
Biosafety
 
Human Practices
Psychological Survey
Philosophical Reflection
 
Sponsors
August
M
T
W
T
F
S
S
1
2
3
4
5
6
7
8
9
10
11
12
13
14
15
16
17
18
19
20
21
22
23
24
25
26
27
28
29
30
31
September
M
T
W
T
F
S
S
1
2
3
4
5
6
7
8
9
10
11
12
13
14
15
16
17
18
19
20
21
22
23
24
25
26
27
28
29
30
-
October
M
T
W
T
F
S
S
1
2
3
4
5
6
7
8
9
10
11
13
14
15
16
17
18
19
20
21
22
23
24
25
26
27
28
29
30
-
 
21/08/2010
Gel1
all primers for the synthetic miR construction Kit were diluted to a 100 uM concentration
PCR reaction for construction of the following parts were set up:
Gel2
CMV (reverse complementary); template: pCMV_BBB, primer: CMV_rc_BBB_fw/rev
BGH terminator; template: pcDNA5/TO/FRT/miRsAg, primer: BGH_pA_fw/rev
SV40 promoter (reverse complementary); template: pSiCheck-2, primer: SV40_rc_BBB_fw/rev
RSV5' region; template: pBS/U6-GFP, primer: RSV_fw/RSV_rev_midcomp
RSV3' region; template: pBS/U6-GFP, primer: RSV_rev/RSV_fw_midcomp
RSV5' region (reverse complementary); template: pBS/U6-GFP, primer: RSV_rc_rev/RSV_rev_midcomp
RSV3' region (reverse complementary); template: pBS/U6-GFP, primer: RSV_rc_fw/RSV_fw_midcomp
The PCRs were set up according to the following protocol:
0.5 ul (= 50 ng) Template DNA
10 ul Hifi-Buffer
0.5 ul each Primer
1 ul (= 2.5 u) Hifi-Polymerase
37.5 ul water
The PCR was run according to the following protocol:
................................................
95 °C/ 5 min
................................................
94 °C/ 15 sec
57 °C/ 1 min
72 °C/ 1 min
................................................ (35x)
72 °C/ 10 min
................................................
4 °C/ forever
................................................
PCR reaction for the construction of the linear standard plasmids was set up as follows:
1 ul (10 ng) template DNA pSB1A3/pSB1K3/pSB1C3, cut EcoRI/PstI/DpnI
0.5 ul of each primer SB-prep-3p/SB-prep-2Ea
10 ul of Hifi Buffer
2 ul of Hifi Polymerase
36 ul water
The PCR was performed according to the following protocol:
................................................
95 °C/ 5 min
................................................
94 °C/ 15 sec
55 °C/ 1 min
72 °C/ 2.5 min
................................................ (35x)
72 °C/ 10 min
................................................
4 °C/ forever
................................................
The result of the PCRs was analyzed on a 1% agarose gel, run for 1 h @ 100 V (Gel1).
As the result of the first pSB vector PCRs was not ideal due to strong side products, the PCR was repeated in 6 replicates according to the following protocol:
25 ul Phusion PCR MasterMix
1 ul template DNA (pSB1C3, digested with PstI/EcoRI/DpnI)
0.5 ul of each primer SB-prep-3p/SB-prep-2Ea
add water to final volume of 50 ul
The 2.1 kb band was afterwards cut out from the gel and a gel extraction was applied for purification of the DNA (Qiagen Gel Extraction Kit).
PCR was performed, applying the standard protocol (annealing temperature 60 °C). 5 ul of PCR product were analyzed on an agarose gel, run for 1 h @ 100 V.
22/08/2010
Fusion-PCR for the construction of the whole RSV promoter fragment with BBB prefix and suffix:
1 ul (50 ng) of RSV5' template (or RSV5' (reverse complementary))
1 ul (10 ng) of RSV3' template (or RSV3' (reverse complementary))
10 ul buffer
1 ul Hifi Polymerase (2.5 u)
36 ul water
The PCR was performed according to the following protocol:
................................................
95 °C/ 5 min
................................................
94 °C/ 15 sec
58°C/ 1 min
72 °C/ 1 min
................................................ (35x)
72 °C/ 10 min
................................................
4 °C/ forever
................................................
PCR for the construction of linear vector, the PCR was further optimized by applying again the Hifi and the Phusion PCR protocol from the previous day, but using an annealing temperature of 64 °C that time.
The PCR for both fragments was analyzed on a 1 % agarose gel, run for 1 h @100 V (Gel1) and PCR purified applying a Qiagen PCR purification Kit.
Digestion of PCR product and Vectors
1 ug of each vector PCR and insert PCR product was digested with EcoRI/PstI according to the NEB standard digestion protocol (10 u Enzyme, final volume 50 ul). After heat inactivation (20 min @ 80 °C), 2 ul (2 u) of shrimp acidic phosphatase were added to the vector digestions and incubated for 30 min @ 37 °C. After the digestion, a purification was performed using a Qiagen PCR purification kit.
Ligation
Ligations were set up using the Fermentase Ligation Kit. The following vector and insert ratios were used:
Vector 1 (Gel Extracted from previous day) and vector 3 (Phusion PCR product, PCR purified); 4 ul (40 ng):
CMV(reverse complementary): 45 ng
BGH terminator: 27 ng
SV40(reverse complementary): 37 ng
RSV: 50 ng
RSV(reverse complementary): 50 ng
Vector 2 (Hifi PCR product, PCR purified); 4 ul (80 ng):
CMV(reverse complementary): 90 ng
BGH terminator: 55 ng
SV40(reverse complementary): 75 ng
RSV: 100 ng
RSV(reverse complementary): 100 ng
Furthermore, negative controls were set up for each vector.  Transformation in E. coli Top10 cells were performed according to the standard transformation protocol.
23/08/2010
gel1
Gel2
Colony-PCR of the colonies obtained on the agar-plates (cloning previous day) was performed according to the following protocol:
12.5 ul Phusion MasterMix
0.25 ul of each primer
12 ul of water
colony picked from agar-plate
2 colonies were picked for each construct. The PCR was performed according to the standard Phusion PCR protocol. In parallel, a MiniPrep Culture for each colony was inocculated.
PCR for the construction of Luc2, hRluc, CMV, SV40 promoter, SV40 terminator and the BGH-Terminator. Therefor, PCR reactions were set up according to the following protocol:
10 ul Hifi Buffer
0.5 ul of each primer
1 ul (50 ng) of each template
38 ul of water
The combination of templetes and primers were as follows for the mentioned constructs:
CMV: primers CMV_BBB_fw/rev; template: CMV_BBB (Team Heidelberg 2009)
Luc2: primers: Luc2_ter_BBB_fw/Luc2_ter_BamHI_BBB_rev; template: pGL4.23 (Promega)
hRluc: primers: hRluc_ter_BBB_fw/hRluc_ter_BBB_rev; template: pSiCheck-2 (Promega)
SV40 promoter: primer: SV40_BBB_fw/rev; template: pSiCheck-2 (Promega)
SV40 terminator: primer: SV40_term_fw/rev; template: pGL4.23 (Promega)
BGH terminator(reverse complementary): primer: BGH_pA_BamHI_rc_fw/BGH_pA_rc_rev
PCR was performed according to the standard Hifi-Protocol (annealing temperature 57 °C)
24/08/2010
Gel1
Gel2
Gel3
Amplification of the CMV Promoter as well as the amplification of pSB1A3/C3/K3 standard plasmid backbones was performed via Phusion or Hifi-PCR according to the standard protocol (annealing temperature 61 °C).
The PCR product was run on a Gel for 1 h @ 100 V and analyzed an a 1 % agarose gel (gel 1).
Another colony-PCR of the colonies obtained on the agar-plates (cloning two days before) was performed according to the following protocol:
12.5 ul Phusion MasterMix
0.25 ul of each primer
12 ul of water
colony picked from agar-plate
Two colonies each were picked from plate (1) and (2) and four colonies of plate (3) for each construct, respectively.  The PCR was performed according to the standard Phusion PCR protocol.  In parallel, a MiniPrep Culture for each colony was inocculated. The result of the colony-PCR was analyzed on a 1 % agarose gel, run for 1 h @ 100 V (gel2).
a PCR for the construction of synthetic microRNAs and microRNA binding sites was perfomed. For the construction of binding sites, single stranded oligos were ordered and second strand synthesis was performed using the BBB_suffix_reverse primer. The protocol was as follows:
1 ul microRNA binding site oligo
0.5 ul BBB_suffix_reverse primer
23.5 ul water
25 ul Phusion MasterMix
The binding site oligos for the following microRNA binding sites were used:
hsa-miR-375 (perfect, 9-12 mut, 9-22 mut)
hsa-miR-376a (perfect, 9-12 mut, 9-22 mut)
hsa-miR-122 (perfect, 9-12 mut, 9-22 mut)
shRNA-6 (perfect, 9-12 mut, 9-22 mut)
shRNA-7 (perfect, 9-12 mut, 9-22 mut)
shRNA-8 (perfect, 9-12 mut, 9-22 mut)
shRNA-9 (perfect, 9-12 mut, 9-22 mut)
shRNA-10 (perfect, 9-12 mut, 9-22 mut)
PCR for the construction of synthetic shRNAs was performed. Therefore, the pcDNA5/TO/FRT containing the shRNA miRsAg as insert was used as template. For the shRNAs 1,2,3,5,6,7,8,9,10 the synthesis of two intermediate fragments were performed. The 5' fragment was amplified by using the primer shRNA_AflI_BBb_fw and the reverse oligos of the certain shRNA gene. The 3' framents were amplified by using the primer shRNA_HindIII_BBb_rev and the forward primer of the certain shRNA gene. The forward and reverse primer of the shRNA gene introduce the guiding and passanger with certain specificities into the shRNA constuct. The miRsAg is an miR-122 like shRNA and uses the backbone of miR-122, but another guiding and passanger strand. The PCR was set up according to the standard protocol (annealing temperature: 58 °C).
The results of the PCR were analyzed on a 2 % agarose gel, run for 30 min @ 100 V (gel 3)).
25/08/2010
The constructs were tested in a digestion with NotI using the following protocol:
4.0 ul template
2.5 ul NEB buffer 3
2.5 ul BSA
0.5 ul NotI (NEB)
add 25 ul
For RSV, RSV(reverse complementary), BGH(forward) und SV40 promoter (reverse complementary) positives bands were found and samples were send for sequencing.
Cloning of constructs BBb_CMV, Luc2_ter, hRluc, sv40(forward) and BHG(forward):
Digestion of pSB1C3 and the PCR products of the mentioned constructs with EcoRI/PstI (1 ug each) according to the standard digestion protocol. Subsequent purification using a nucleaotide removal kit and ligation overnight at 16 °C.
26/08/2010
The sequencing result for the samples send for sequencing on the privious day was checked. The results were positive for all samples, sequences were accurate.
Transformation of the ligation done on the previous day according to the standard transformation protocol.
27/08/2010
gel1‎
gel2‎
PCR was carried out for the following parts, using the standard HiFi protocol:
1. Backbone amplification using primers backbone_AvrII_fw and backbone_AvrII_rev
2. CMV-TetO2: CMV_TetO2_BBB_fw, CMV_TetO2_BBB_rev
3. Tet-repressor: TetRepressor (from pcDNA6)fw, TetRepressor (from pcDNA6)rew
4. FRT site: FRT_BBB_fw, FRT_BBB_rev
Colony PCR was carried out (according to the standard PhusionMix protocol, annealing temp. 57 C) for CMV, Luc2, hRluc, SV40, SV40 ter and BGH-1 rc. Only one sample for hRluc and one for SV40 ter showed a possible positive result (gel1).
The three standard plasmids (pBS_1A3, pBS_1C3, pBS_1K3) were digested with EcoRI and PstI according to the following protocol:
5 ul of each plasmid (conc. = ?)
3 ul EcoRI buffer
3 ul BSA
0.5 ul of each enzyme
18 ul nuclease-free water
Incubatation: 37 C, 1hr
Heat inactivation: 80 C, 20 min
Then DpnI was added (0.5 ul), and the tubes were incubated at 37 C for 1 hr, then heat-inactivation at 80 C for 20 min.
5 ul of the product were loaded on a 1% agarose gel and analyzed (gel1).
28/08/2010
6 more colonies from the cloning done on the 25th/26th were picked and colony PCR was performed according to the standard colony-PCR protocol. In parallel, Minipreps of each colony were inocculated. The results of the colony-PCR ist shown in the gelpictures.
shRNA construction via PCR; Template: 50 ng of first PCR product; Standard PCR protocol using an annealing temperature of 60 °C.
29/08/2010
gel1
gel2
Vector pcDNA_BBbCMV, PCR product Luc2_ter and shRNA(1-10) with EcoRI/PstI (1 ug each). The BBb_CMV was cut out from the gel (~ 700 bp band) and gel purified (gel1). The PCR product Luc2_ter was purified by applying a nucleotide removal kit.
Ligation reactions were set up using pSB1C3 (amplified using Phusion HF Mastermix (1) or High Fidelity Polymerase (2) cut EcoRI/PstI) (100 ng) and 100 ng CMV, 40 ng shRNA or 300 ng Luc2 constructs. For shRNA6 and CMV the cloning was done in 2 replicates (using either the pSB1C3 amplified via HF Mastermix or High Fidelity Qiagen Kit). The ligation was performed according to the fermentas ligation kit protocol and incubated for 1 h @ room temperature. Afterwards, 10 ul were transformed into Top10 cells.
30/08/2010
Gel1
Gel2
Gel3
The ligation result from the previous days' cloning was analyzed via colony PCR (gel 1-3) and all colonies were inocculated (LB cultures) The construct numbers indicate the number of the clone picked from the agar plate. If cloning was performed in Replicates, the first number indicates the plate (i.e. CMV3) and the second number the clone. As seen in gelpicture 1 the clone CMV3.1 and on gelpicture 3 the clones CMV 2,3,12 and 16 all show the right band length of ~1000 bp. On gelpicture 2 the shRNA10.1 clone shows the right band as well as the Luc2.1.5 clone for the Luc2 luciferase.
31/08/2010
Gel1
Gel2
The LB cultures from the previous day were inocculated and mini prepped. Afterward, a test digestion of all constructs with NotI was performed (gel1) and loaded on a 1 % agarose gel run for 35 min @ 100 V. The samples CMV2, CMV3 (23), CMV3.1, Luc2.1 and sh10.1 showed the right bands on the gel and samples were send for sequencing. All sequencing results showed, that the constructs were right.
pSMB_Measure backbone (igem team HD 2009 measruement standard) was amplified using the ClaI/XhoI or AvrII primers and performing a standard Phusion HF Mastermix PCR protocol (annealing temperature 61 °C. The PCR result was loaded on the agarose gel 1 for control (BcA = AvrII cutting sites, BcB = XhoI/ClaI cutting sites). The PCR products of the used vectors pSB1C3 and the hRluc and Luc2 PCR fragments used in this and subsequent clonings was also loaded on the gel for controlling purposes.
Digestion of the pSB1C3 vector amplified via Phusion HF PCR Mastermix (annealing temperature 61 °C) with EcoRI/PstI for 1 h @ 37 °C. After heat inactivation, 1 ul of DpnI was added to the restriction mix an digestion performed for another hour. The product was SAP (dephosphorylation) or not SAP treated this time. Furthermore, the hRluc, Luc2, CMV_TetO2, sv40 and FRT site PCR products (see 08/27/2010) were digested EcoRI/PstI as well.
Ligation: each PCR fragment was ligated into digested vector backbone pSB1C3. 50 ng of the non-treated and 100 ng of SAP treated vector were used in the ligation reactions. Amount of insert was calculated to be 2 fold (large fragment Luc2 and hRluc) or 3-5 fold of the molar vector amount. Ligation was performed for 1.5 h @ room temperature. The ligation was analyzed on a 1 % agarose gel, run for 35 min @ 100 V. The non-treated vector is indicated via number 1 (i.e. Luc2.1) the SAP-treated vector with nr. 2 (i.e. Luc2.2). As control, the digested insert fragment PCR products and digested vectors were loaded on the gel as well. For hRluc, Luc2 and CMV_TetO2 correct product bands are clearly visible. For sv40, that is not the case. The FRT site is to short, to be able to distinguish clearly between vector fragment and ligated Vector with Insert. Subsequently, a transformation of all 10 ligation reactions into Top10 cells was performed according to the standard transformation protocol.
Retrieved from " http://2010.igem.org/Team:Heidelberg/Notebook/miRNA_Kit/August "
Recent changes
What links here
Related changes
Special pages
My preferences
Printable version
Permanent link
Privacy policy
Disclaimers
