"
Page
Discussion
View source
History
teams
Log in
 
Team:Heidelberg/Notebook/miRNA Kit/September
From 2010.igem.org
Home
Team
Overview
Students
Instructors
Advisors
Institutes
Gallery
miDANCE
Acknowledgements
 
Project
Abstract
Introduction
Synthetic miRNA Kit
Capsid Shuffling
    Homology Based
    ViroBytes
miMeasure
in vivo Study
Summary & Outlook
References
Contributions
 
Parts
Parts
Characterization
 
Modeling
Introduction
miBEAT
miRockdown
miBS designer
mUTING
Descriptions
 
Notebook
Synthetic miRNA Kit
Capsid Shuffling
    Homology Based
    ViroBytes
miMeasure
Binding Site Design
in vivo Study
Modeling
Materials
Methods
Psychological Survey
Philosophical Reflection
Biosafety
 
Human Practices
Psychological Survey
Philosophical Reflection
 
Sponsors
August
M
T
W
T
F
S
S
1
2
3
4
5
6
7
8
9
10
11
12
13
14
15
16
17
18
19
20
21
22
23
24
25
26
27
28
29
30
31
September
M
T
W
T
F
S
S
1
2
3
4
5
6
7
8
9
10
11
12
13
14
15
16
17
18
19
20
21
22
23
24
25
26
27
28
29
30
-
October
M
T
W
T
F
S
S
1
2
3
4
5
6
7
8
9
10
11
13
14
15
16
17
18
19
20
21
22
23
24
25
26
27
28
29
30
-
 
01/09/2010
Gel 100901-1
Gel 100901-2
8 colonies of each plate (see previous day ) were picked and colony-PCR was performed. Furthermore, the colonies were used for inocculation of LB miniprep cultures. The numbers behind the construct name refer to vector type (1 = not SAP treated, 2 = dephosphorylated using SAP; see previous day). For hRluc (1300 bp band)), Luc2 (2300 bp band), CMV_TetO (1000 bp band) and the FRT (500 bp band) site, positive colonies could be observed (gel 100901-1 and 100901-2).
PCR for the construction of the standardized Tet Repressor was performed. The PCR was set up using the Phusion HF MasterMix and standard PCR protocol was applied (annealing temperature 59 °C, 2 replicates). The result of the PCR was analyzed on a 1 % agarose gel (gel 100901-2, lower right side) and the right 700 bp band was clearly visible.
Digestion of the TetR fragment with EcoRI/PstI was performed according to the standard digestion protocol and the fragment was subsequently ligated into pSB1C3 (SAP treated or non-treated). The ligation mix was transformed into Top10 cells.
02/09/2010
Gel 100902-1
Gel 100902-2
Gel 100902-3
8 colonies were picked from plates 1 and 4 colonies from plates 2 from the previous days cloning and analyzed via colony PCR performed according to the PCR standard protocol . The results of the colony PCR were analyzed on a 1 % agarose gel run for 1 h @ 100 V (gel 100902-1 and 100902-2). The following colonies were afterwards inocculated (LB miniprep cultures):
TetR (1.4 and 1.6)
shRNA6 (1.1, 1.4 and 1.8)
shRNA7 (1.1, 1.5 and 1.6)
shRNA8 (1.1, 1,2, 1,3)
shRNA9 (1.2)
The cultures inocculated the day before were mini prepped by applying a Qiagen Miniprep Kit. Afterwards, a test digestion with NotI was performed and the result was analyzed on a 1 % agarose gel, run for 35 min @ 100 V (gel 100902-3). According to the test digestion result, the following samples were send for sequencing @ GATC:
CMV_TetO2 (.4, 1.6)
FRT (1.1, 1.3)
Luc2 (1.4, 1.6)
hRluc (1.1, 1.3, 1.4)
03/09/2010
Gel 100903-1
Gel 100903-2
Test digestion with NotI of the cloned constructs on the previous day (gel 100903-1). For the TetR constructs the expected 600 bp band and for the shRNA constructs, the expected 200 bp band were clearly visible. For  each construct, one sample was send for sequencing. Sequencing results were correct for all samples send for sequencing.
Purification of single binding sites against shRNA6-10, miR-375/376/122. For each target, the perfect, 9-12 and 9-22 randomized binding sites were purified and analyzed on an agarose gel (100903-2). All bands were clean, with just a slight amount of side product for the 9-22 randomized binding sites.
cloning of shRNAs into pcDNA5/FRT/TO
04/09/2010
mini-prep of pcDNA5 with shRNAs 1, 2, 3, 5, 5* and 6 (each four colonies from two independent transformations)
PCR of shRNA constructs (including shRNA 7, 8 and 9 as well)
gel: fine, everywhere amplified sequences below 200 bp
digestion of promising tuning constructs with BspEI and StuI
gel: plasmids seem to be cut only once again (problem maybe BspEI), size is correct
suggestion: another digestion on monday with SalI and KpnI, then: submission for sequencing
mini-prep of pcDNA5 with shRNAs 1, 2, 3, 5, 5* and 6 (each four colonies from two independent transformations)
PCR of shRNA constructs (including shRNA 7, 8 and 9 as well)
gel: fine, everywhere amplified sequences below 200 bp
digestion of promising tuning constructs with BspEI and StuI
gel: plasmids seem to be cut only once again (problem maybe BspEI), size is correct
suggestion: another digestion on monday with SalI and KpnI, then: submission for sequencing
05/09/2010
transformation of final Kit constructs F1 - F16 into E. coli Top10 cells
06/09/2010
inocculation of miniprep LB cultures for the constructs F1 - F16 from single colonies from the plates prepared the previous day
digestion of PCR amplified vector backbone pSB1A3 with EcoRI/PstI; after subsequent heat inactivation, digestion with DpnI was performed and the vector was purified by applying a nucleotide removal kit.
digestion of tuning construct with KpnI and SalI result in weird band pattern on the gel
PCR and purification of the binding sites looked fine on the gel
submission of tuning constructs and pcDNA5 containing different miRNAs for sequencing
07/09/2010
Gel 100907-1: Ligation Control. Numbers indicate the F-Construct numbers of the parts assembled
Miniprep of constructs F1 - F16 using a Qiagen Miniprep Kit
Digestion of 500 ng of each construct with the following enzymes
F2, F3, F5 with EcoRI/NheI
F8, F1, F15, F11, F12, F13, F14, F7 with PstI/SpeI
Ligation of the follwing constructs into pSB1A3 via standard 3A assembly
F3/F15, F2/F1, F2/F8, F3/F1, F2/F15, F5/F7, F5/F11, F5/F12, F5/F13, F5/F14
Digestion of 500 ng F6 with XbaI; purification via nucleotide romoval kit
Annealing of XbaI mutation oligo according to the following protocol
prepare mixtrure of oligos (5 ul @ 100 mM each) in NEB buffer 2 (volume 50 ul)
heat up to 95 °C in heating block; cool down to room temperature slowly
incubate on ice for 10 min
0.5, 1 and 3 ul of the annealed oligos were then ligated into 50 ng of pre-digested F6 construct;
All Ligations were controlled by loading 10 ul of ligation reactiong onto a 1 % agarose gel (100907-1), run for 35 min @ 100 V
repetition of TetR and tuning construct cloning
purification and digestion of PCR amplified binding sites with NotI
08/09/2010
Gel 100908-1: Colony-PCR.
Colony PCR of the previous days' cloning and miniprep cultures were inocculated in parallel; The result of the PCR was analyzed on a 1 % agarsoe gel (100908-1) run for 35 min @ 100 V.
cloning of shRNAs in pcDNA5 again
further digestion of binding sites with AscSI (analog to SgfI) and purification
preparation of psyCHECK2 for insertion of binding sites and shRNAs for first test measurements
transformation of TetR constructs
ligation of both inserts for the tuning construct, extraction of the right insert (3000bp) and ligation into the backbone
09/09/2010
Gel 100909-1
Gel 100909-2
Miniprep of the cultures inocculated on the previous day followed by test digestion of 300 ng of each constructs with NotI (gel 100909-1). The positive samples were send for sequencing @ GATC. The gel looks weird due to use of 0.05 x TAE buffer instead of 0.5 x.
The nine Luc2-SV40 constructs were with XbaI and NheI (negative testing), and XhoI and PstI (positive testing) for the induced mutation. The reaction was set up according to the standard protocol, and then the samples were visualized on a 1% agarose gel (gel 100909-2).
colony PCR of TetR constructs are positive
inoculate TetR for miniprep
10/09/2010
Tet test digest
colony PCR shRNAs
PCR of SV40 Terminator with BamHI site according to standard PCR protocol (annealing @ 58 °C)
Assembly of the following constructs, again via 3A standard assembly.
F6/F1, F6/F8, F10/F4, F5/F1, F16/SV40, F5/F8
Subsequent Transformation into DH5alpha cells
miniprep of TetR
test digest of TetR with BspEI & NotI and with SgfI gave promising results
YES it was done with marker instead of loading dye but:
for lanes 2-4 you see a stronger band at 700 pb (size of TetR)
for landes 5-7 you see a stronger band at 5000 bp as it was linearized with SgfI
colony PCR of binding site in PsiCheck and tuning construct.. tuning construct looks promising, BS nothing on the gel except for primers
colony PCR of shRNAs look promising even though we cannot say whether still the old shRNA is inside
PCR of CMV shRNA which leads to a size of 1200bp
11/09/2010
Gel 100911-1
Gel 100911-2
Colonies were picked from the plates from the previous days' cloning and a colony PCR was performed according to the standard protocol. In parallel LB miniprep cultures were inocculated for each colony (gel 100911-1 and 100911-2).
miniprep of tuning construct and assorted shRNA BS PsiCheck colonies. Repetition of test PCR (for tuning primer L11 and L13, for BS primer L1 and 86a (dominik)) with miniprep template. Tuning construct looks kind of good on the gel, but I don't want to get you hopes up too soon.. it will be send for sequencing on monday.
Tet Repressor cloned!!!
shRNA 2,3,5,7,8 and 9 seems to be cloned according to test digest
test digest
12/09/2010
Gel 100912-1
Gel 100912-2
Minipreps were done by applying a Qiagen Miniprep Kit for the positive candidates picked on the previous day. Test digestion was performed by digesting ~ 250 ng of each construct with SpeI/NheI and the result was analyzed on a 1 % agarose gel (100912-1 and100912-2) run for 40 min @ 100 V. For all the constructs we got the right band despite construct F10/F4.3. Positive constructs were send for sequencing @ GATC.
miniprep of shRNA constructs and Tuning constructs
13/09/2010
digestion of the following parts:
F4 (E/N), F5 (S/P), R5 (E/N), R6 (E/N), R1 (S/P), R2 (S/P), R3 (S/P), R4 (S/P), R9 (E/P), R10 (E/P); after heat inactivation subsequent purification using a nucleotide removal kit (elution volume 20 ul).
ligation of the following parts (50 ng of vector, equal amounts of parts)
F4/F5 into pSB1A3 (precut E/P)
R5 with either R1, R2, R3 or R4 into pSB1C3 (precut E/P)
R6 with either R1, R2, R3 or R4 into pSB1C3 (precut E/P)
R9 into pSB1C3 (precut E/P)
R10 into pSB1C3 (precut E/P)
transformation of 10 ul ligation reaction after heat inactivation into DH5alpha cells
digestion and cloning of binding sites and PsiCheck.. again
submission of constructs for sequencing
14/09/2010
Gel 100914-1
Gel 100914-2
note: for sequential digest of pcDNA5 or shRNAs with ApaI and HindIII use buffer 2
Colony- PCR for the cloning products done on the previous day (gels 100914-1 and 100914-2 on the right)
positive clones were Mini-Prepped and test digested
15/09/2010
Gel 100915-1
digestion of TetR construct with AsiSi and NotI or SgfI and NotI respectively
for cloning of binding sites for miR122, miR375 and miR376
ligation of the other binding sites for synthetic pri-miRNA (6, 7, 8, 9 and 10) with digested psiCHECK2
PCR for the introduction of the Kozag-Sequence in front of the hRluc_BGH part; two strategies were performed in parallel using different touchdown-PCR protocols. Protocol nr. 1 was set up as follows:
25 ul Phusion HF MasterMix
0.5 ul of 100 um primers
1 ul of template R12 (50 ng/ul)
23 ul of water
Touchdown PCR was performed according to the following protocol:
................................................
95 °C/5 min
................................................ (1x)
95 °C/30 s
68 °C/45 s (- 0.5 °C/cycle)
72 °C/1 min
................................................ (35x)
72 °C/10 min
................................................ (1x)
4 °C/ forever
................................................
Protocol nr. 2 was set up as follows:
25 ul Phusion HF MasterMix
0.1 ul of 100 um primers
1 ul of template R12 (50 ng/ul)
23.8 ul of water
Touchdown PCR was performed according to the following protocol:
................................................
95 °C/5 min
................................................ (1x
95 °C/15 s
68 °C/45 s (- 0.5 °C/cycle)
72 °C/1 min
................................................ (15x)
95 °C/15 s
60 °C/30 s
72 °C/1 min
................................................ (20x)
72 °C/10 min
................................................ (1x)
4 °C/ forever
................................................
Each PCR was performed in two replicates. The result of the PCRs was analyzed on a 1 % agarose gel (100915-1), run for 1 h @ 100 V. The PCR nr. 2 was subsequently PCR purified by applying a Qiagen PCR purification KIT and used for the cloning
16/09/2010
Cloning of Kozag_hRluc_BGH fragment into pSB1C3 and Constructs R24-R31
Cloning of the XhoI/XmaI_suffix oligo via NheI/PstI into the TetR construct F16)
Digestion of PCR fragment with SpeI/PstI according to 3A standard protocol; digestion of destination Vector pSB1C3
Ligation and Transformation were done according to the standard protocol
PCR to generate binding site oligos, nucleotid removal, digestion, nucleotid removal
different digestion and ligation protocols tested for binding sites and psiCHECK2, afterwards transformation
possibilities:
dilution of oligo insert [1:100,1:500], ratio: insert/vector = 5/1
SAP treatment for oligos to prevent them from forming concatemers
SAP treatment of backbone to prevent it from re-ligation
cut of backbone into two fragments to ease gel excision (if done) and advance ligation
ligation process: 1h @ RT or over-night @ 4°C in darkness
note: bs122 confused with bs7
17/09/2010
colony PCR, mini-prep and digestion of promising psiCHECK or TetR constructs containing binding site
control plates with horrible number of re-ligations
Gel 100917-1
Gel 100917-2
Selection of colonies from the plates (see previous days' cloning) via colony-PCR using the standard sequencing primers; a ~5.5 kb fragment should be visible for the assembled construct K1-K8, if colony-PCR was positive (gel 100917-1 and 100917-2). No sample shows a 5.5 kb band, but that's most likely due to the very difficult amplification of such a long fragment with Fermentas 2x PCR MasterMix (Taq-based). Therefor 3 miniprep cultures were inocculated for each construct (K1-K8). The TetR oligo insertion and the Kozag_hRluc_BGH cloning into pSB1C3 gave the right bands on the gel.
19/09/2010
Miniprep of the previous days inocculated miniprep cultures
20/09/2010
Gel 100920-1
Gel 100920-2
test digestion of the miniprepped clones (previous day) with EcoRI/PstI; For all clones at least 2 digestions gave the exactly expected band (gel 100920-1 and 100920-2); therefor we can say, that the tuning construct is succesfully assembled and ready to be tested
22/09/2010
digestion of R23 with EcoRI and PstI in EcoRI Buffer ( NEB ) and ligation into digested pSB1C3 , transformation, growing on selective agar plates (chloramphenicol), following standard protocol recommendations
Dual Luciferase Assay
In order to test for promoter efficiency and to check whether the miRNA kit assembly works fine 50ng of each construct with different promoter set-ups were transfected into HEK 293 T-REx cells and other cell lines HEK, HeLa, Huh7 in 96-well plate format using FuGENE transfection reagent. As every construct is expressing firefly luciferase (luc2) and renilla luciferase (hRluc) at the same time the setup allows for standardization of transfection efficiency as only luc2 is tagged with binding sites. Each sample was transfected and measured by Dual luciferase assay in 8 replicates. As by this time no shRNA has been cloned into plasmid no knock-down of luc2 is expected and the different expression efficiencies allow for characterization of the different promoters. BBa_K337032 leads to a relative luciferase unit (RLU) of luc2 to hRluc expression of 6 RLU. BBa_K337035 and BBa_K337035 are showing a comparable expression of 12 - 13 RLU, which is in line with the knowledge that both luciferases are driven by the CMV promoter.  Hek 293 T-Rex cells stably express the Tet repressor thus allows us to observe very efficient repression of Firefly luciferse expression if a CMV-TetO2 promoter is driving luc2 ( BBa_K337038 and BBa_K337046 ). BBa_K337040 transfection into Hek T-Rex cells results in an expression of 15 RLU. BBa_K337042 and BBa_K337044 are constructed in a way that luc2 is driven by the CMV promoter and hRluc is driven by the RSV promoter and show a comparable expression of 17-20 RLU. This leads to the conclusion that the CMV promoter shows comparable expression to the RSV promoter in Hek T-Rex cell lines. If transfecting BBa_K337035 and BBa_K337036 are transfected into different cell lines it is obvious that Hek293T cells are the easiest to transfect with both constructs an expression of 17-22 RLU is to be measured. Hek T-Rex cells are showing and expression level of 12 RLU of both constructs. Hela cells are also showing constant expression levels of  8 RLU with both constructs. A rather high standard deviation in the luciferase expression and also differences between the 2 constructs is to be seen in Huh7 cells. This might be due to low transfection efficiency of this cell line in general. Alltogether it is to say that BBa_K337035 and BBa_K337036 show comparable expression.
part
promoter driving luc2 (Firefly)
promoter driving (Renilla)
promoter driving shRNA expression
BBa_K337032
RSV
CMV
SV40
BBa_K337035
CMV
CMV
SV40
BBa_K337036
CMV
CMV
RSV
BBa_K337038
CMV TetO2
CMV
RSV
BBa_K337040
RSV
RSV
SV40
BBa_K337042
CMV
RSV
SV40
BBa_K337044
CMV
RSV
RSV
BBa_K337046
CMV TetO2
RSV
RSV
Promoter strength characterization in HEK 293 T-REx cell line
Promoter strength characterization in different cell lines
23/09/2010
Gel 100923-1. Analytical gel with following lanes: 1) till 8) colony PCR products with nice bands as expected at ~ 800 bp especially at lane 1, 4, 6 and 8, lane 9) 1kb Plus Ladder (Invitrogen)
minipreps (using Quiagen Kit) and colony-PCR of each 8 colonies of R23 in pSB1C3 (digested, ligated, transformed and plated on previous day )
PCR product of R23C on analytical gel (see gel 100923-1)
digestion of R23 and R33 with either SpeI and PstI or EcoRI and NheI
ligation into digested pSB1A3 , transformation , growing on selective agar plates (ampicillin), following standard protocol recommendations
Dual Luciferase Assay was performed as on [ previous day ]. Cells were not harvested after 20h, but 48h.
24/09/2010
inoculation and colony PCR of the transformed (23A+33A)=56C construct from yesterday. Same PCR protocol was used, even though the fragment should be 1100 and therefore elongation was probably not long enough. The PCR showed a band at 300bp, which dominik says is normal if you have religated vector. But nevertheless, miniprep of La1, La3 and La4 were made, because they didn't show a strong negative PCR band.
test digestions (actually they are preparative, the only difference is I will test them on the gel as well to be sure I have the right thing)
R23A from yesterday (miniprep 8 and 8/) is digested with Eco&Nhe, expected fragment is 500bp
R33 with Spe&Pst, expected fragment 600bp
R56C with Spe&Pst, expected fragment 1100bp
R8 with Eco&Nhe, expected frament 1000bp
ligation of R23A+R33+backbone C
ligation of R56C + R8 + backbone A
25/09/2010
PCR amplification of standardised pcDNA5 p55 and p1 with primer D47/48 and D55/69
26/09/2010
Gel 100926-1. Analytical gel with following lanes: 1) R33 E/N (from 23rd September)  2) R23_1 S/P  3) R23_6  4) R33C  5) R23_4  6) R23_8  7) 1kb Plus Ladder (Invitrogen)  8) R33C E/N  9) R23_4 S/P  9) R23C_8 S/P
Gel 100926-2. PCR results of p55 and p1: p1 with primer 47/48 (lane 1), p1 with primer 55/70 (lane 2), p55 with primer 47/48 (lane3), p55 with primer 55/70
miniprep of R23C cultures again (colonies from transformation (done on wednesday ))
check of all digestions on an analytical gel (see gel 100926-1)
fragment for R23_6 is a bit smaller than expected (rather 400 bp than 500 bp)
originally digested R23A runs slightly below 500 bp, too (data not shown)
because of reverse elements and resulting secondary structure?
repetition of ligation with R23_6 and old R33 (nice band at roughly 600bp)
including negative control at this time with only backbone pSB1A3
transformation
tuning construct:
gel extraction of PCR on p55 and p1 (see gel 100926-2)
recloning of R1,R4 and R7 into standardbackbone pSB1A3
digest 500ng of R1, R4 and R7 with EcoRI and PstI
heat inactivate 20 min at 80°C
Nucleotide removal
ligate 150ng and 250ng of insert into 30ng pSB1A3 1h at RT and transform ligation
27/09/2010
Gel 100927-1: colony PCR on K1 (lane 1-8), K4 (lane 9-16) and K8 (lane 17-24)
Gel 100927-2: colony PCR K1 (lane 1&2), K4 (3&4) und K7 (5&6), K3 digested with BamHI (7), K3 digested NheI and SpeI (8), R33 in pSB1A3 digested with EcoRI and PstI (9), R33 digested with NheI and SpeI (10)
tuning construct:
colony-PCR on K1 , K4 and K7 cloned into pSB1A3 showed nearly only positives as a band of 400bp is expected by using the primer pair 50/75
miniprep of K1 , K4 and K7
send for sequencing
cloning of K3 into pSB1A3 :
digest K3 with PstI and EcoRI
ligate 200ng with 30ng of vector pSB1A3
transform ligation and plate on an Ampicillin plate
digest shRNA 6 (F11) and shRNA 7 (F12):
1h at 37°C with HindIII
heat inactivate 20 min at 80°C
digest 1h at 37°C with AflII
purify by Nucleotide removal kit and elute in 20µl
repressor construct:
a lot of religations, thus: a lot of colonies to pick
colony-PCR and test digestion with SpeI and PstI of promising mini-preps revealed some positive samples
ligation with R8 and R11 , respectively (each digested with EcoRI and NheI) in pSB1C3 (backbone, digested with EcoRI and PstI), following standard protocol recommendations
growth of transformed cells on selective agar plates (chloramphenicol) over night
28/09/2010
Gel 100928-1. Colony PCR of K3 construct
Gel 100928-2. Test digest of with PstI and EcoRI (lane 1-6), test digest with BamHI (lane 7-12), K1 (lane 1&2), K4 (lane 3&4), K7 (lane 5&6)
Gel 100928-3. Digestion of K1 (lane 1-4), K4 (lane 5-8) and K7 (lane 9-12) with HindIII and AflII
tuning construct:
colony PCR of K3 (gel 100928-1)
inoculation for mini prep of K3
mini prep of K1, K4 and K7
test digestion of K1, K4 and K7 with PstI & EcoRI and BamHI (gel 100928-2 and 100928-3)
digestion of K1, K4 and K7 with HindIII and AflII prior to gel extraction
ligation of shRNA 6 and 7 into K1, K4 and K7
transformation of ligation
plate ligation on ampicillin plates
repressor construct:
re-do of mini-preps and digestions of (R23+R33)C_1, (R23+R33)C_14, (R23+R33)C_17 and (R23+R33)C_23 (promising)
check on gel, submission for sequencing next day
analysis of transformations from yesterday (i. e. (R23+R33) + R8 or (R23+R33) + R11 in chloramphenicol standard backbone)
only re-ligations (as assumed from negative control), more colonies next day
29/09/2010
Gel 100929-1 first gel: double digestion of K1 and K3 with PstI and EcoRI: K1 (lane 1-4), K3 (lane 5-8), second gel: digestion of K1 and K3 with BamHI: K1 (lane 1-4), K3 (lane 5-8)
Gel 1000929-2 digestion of K1 and K3 with HindIII and AflII: K1 (lane 1&2), K3 (lane 3-12)
gel 100929-3 pcr on binding sites for shRNA 10
tuning construct:
mini Prep on K3 and K1 constructs
test digestion with PstI & EcoRI and BamHI showed all positive results
digest K3 and K1 with HindIII and AflII
cloning of shRNA 6 and 7 into K1 , K3 , K4 and K7 :
colony PCR on ligation from the 28/09/2010 did not reveal any positives by using the following protocol:
colony PCR on ligations from the 28/09/2010 did not reveal any positives by using the following protocol:
start from the beginning again: in doublicates:
first procedure:
digest PCR on shRNA 6 and 7 as well as K1 , K3 , K4 and K7 with HindIII and AflII
purify shRNAs with nucleotide removal kit
purify digested K3 and K1 with gel extraction kit
ligate K1 , K3 , K4 and K7 with shRNA 6 and 7
second procedure:
digest K4 and K1 as well as shRNA6 and 7 from PCR product and purify via nucleotide removal kit
ligate K4 and K3 with shRNA 6 and 7
transform all ligations
cloning of binding sites:
second strand synthesis PCR on binding sites for shRNA 10
Gel 100929 r-4: colony PCR of repressor construct, band at 350bp points at backbone religation.
Gel 100929 r-5: test digestion of miniprep number 14 withe EcoRI and NheI shows expected band at 1100bp. Digestions of R8 and R11 were also positive, DNA cut out with SpeI and PstI runs at 800bp. Lane 1=miniprep 14 undigested, lane2=miniprep 14 digested, lane3=miniprep 23 digested, lane4=R8 undigested, lane5= R8 digested, lane6=R11 undigested, lane7=R11 digested
repressor construct:
colony PCR after transformation of P56+R8 and P56+R11 (P56=R23A+R33) yielded no positive results
miniprep of colonies 11, 14, 17 and 23 from P56 cloning were repeted because of bad miniprep results yesterday, then digested with S/P
R8 and R11 were digested with E/N again
digestion of P56 colony 14 was positive on the gel, digestion of R8 and R11 also gave the right fragments
pSB1C3 backbone was SAP treated to reduce religation that was likely to be the problem with previous transformations
ligation of P56 (14) with R8 and R11 and SAPed backbone and control ligation with backbone only, transformation
30/09/2010
Gel 1000903-2 colony PCR on shRNA10 cloned into K3 and K4
Gel 1000930-3 colony PCR on shRNA10 into K3 and K4
tuning construct:
colony pcr on shRNA 6 and 7 into construct K3 and K4
cloning of binding sites for shRNA 10 into K3 and K4 :
digestion of binding sites (perfect, imperfect 9-12 and imperfect 9-22) for shRNA10 with xhoI and ageI :) and construct K3 and K4 in pSB1A3 with xmaI and xhoI
ligation of binding sites with digested constructs
transformation of ligation into TOP10 cells
plate on ampicillin plates ON
Gel 100930-1 colony PCR of Repressor construct with different promoters (R8 or R11). Positive colonies show an amplification product at 2500bp (lanes 1,2,4-8, 10, 13-16)
plenty of colonies on the P56+R8 and P56+R11, NO colonies on control plate with SAPed backbone, colony PCR revealed 50% of positive clones
Retrieved from " http://2010.igem.org/Team:Heidelberg/Notebook/miRNA_Kit/September "
Recent changes
What links here
Related changes
Special pages
My preferences
Printable version
Permanent link
Privacy policy
Disclaimers
