<HTML xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en" dir="ltr">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki  ltr ns-0 ns-subject page-Team_Heidelberg_Modeling_descriptions"><DIV id="globalWrapper"><DIV id="top-section"><DIV id="p-logo"><A href="/Main_Page" title="Main Page">&quot;
	    </A></DIV><DIV id="menubar" class="left-menu noprint"><UL><LI class="selected"><A href="/Team:Heidelberg/Modeling/descriptions">Page               </A></LI><LI class="new"><A href="/wiki/index.php?title=Talk:Team:Heidelberg/Modeling/descriptions&amp;action=edit&amp;redlink=1">Discussion               </A></LI><LI><A href="/wiki/index.php?title=Team:Heidelberg/Modeling/descriptions&amp;action=edit">View source               </A></LI><LI><A href="/wiki/index.php?title=Team:Heidelberg/Modeling/descriptions&amp;action=history">History               </A></LI><LI style="color:#808080;cursor:default">teams</LI></UL></DIV><DIV class="right-menu noprint" id="menubar"><UL><LI id="pt-login"><A href="/wiki/index.php?title=Special:UserLogin&amp;returnto=Team:Heidelberg/Modeling/descriptions" title="You are encouraged to log in; however, it is not mandatory [o]" accesskey="o">Log in</A></LI></UL></DIV><DIV id="search-controls" class="noprint"><FORM action="/Special:Search" id="searchform"> </FORM></DIV></DIV><DIV id="content"><H1 class="firstHeading">Team:Heidelberg/Modeling/descriptions</H1><DIV id="bodyContent"><H3 id="siteSub" class="noprint">From 2010.igem.org</H3><DIV id="header"><DIV id="menu"><UL id="nav"><LI id="menu_home"><A class="aHome" href="https://2010.igem.org/Team:Heidelberg">Home</A></LI><LI id="menu_team"><A class="aTeam" href="https://2010.igem.org/Team:Heidelberg/Team">Team</A><TABLE class="menutable" cellpadding="0" cellspacing="0"><TBODY><TR><TD class="lefttdtopteam"><A class="aTeamOverview" href="https://2010.igem.org/Team:Heidelberg/Team">Overview</A><A class="aTeamMembers" href="https://2010.igem.org/Team:Heidelberg/Team/Members">Students</A><A class="aTeamMembers" href="https://2010.igem.org/Team:Heidelberg/Team/Members#Instructors">Instructors</A><A class="aTeamMembers" href="https://2010.igem.org/Team:Heidelberg/Team/Members#Advisors">Advisors</A><A class="aInstitutes" href="https://2010.igem.org/Team:Heidelberg/Team/Institutes">Institutes</A><A class="aGallery" href="https://2010.igem.org/Team:Heidelberg/Team/Gallery">Gallery</A><A class="aDance" href="https://2010.igem.org/Team:Heidelberg/midance">miDANCE</A><A class="amithanks" href="https://2010.igem.org/Team:Heidelberg/Team/miThanks">Acknowledgements</A></TD><TD class="righttdtop"> </TD></TR></TBODY></TABLE></LI><LI id="menu_project"><A class="aProject" href="https://2010.igem.org/Team:Heidelberg/Project">Project</A><TABLE class="menutable" cellpadding="0" cellspacing="0"><TBODY><TR><TD class="lefttdtopproject"><A class="aAbstract" href="https://2010.igem.org/Team:Heidelberg/Project">Abstract</A><A class="aIntroduction" href="https://2010.igem.org/Team:Heidelberg/Project/Introduction">Introduction</A><A class="aProject_miRNA_Kit" href="https://2010.igem.org/Team:Heidelberg/Project/miRNA_Kit">Synthetic miRNA Kit</A><A class="aPro_Capsid_Shuffling" href="https://2010.igem.org/Team:Heidelberg/Project/Capsid_Shuffling">Capsid Shuffling</A><A class="aPro_Homology" href="https://2010.igem.org/Team:Heidelberg/Project/Capsid_Shuffling/Homology_Based">    Homology Based</A><A class="aPro_Virobytes" href="https://2010.igem.org/Team:Heidelberg/Project/Capsid_Shuffling/ViroBytes">    ViroBytes</A><A class="aPro_miMeasure" href="https://2010.igem.org/Team:Heidelberg/Project/miMeasure">miMeasure</A><A class="aPro_Mouse" href="https://2010.igem.org/Team:Heidelberg/Project/Mouse_Infection"><I>in vivo</I> Study</A><A class="aPro_Summary" href="https://2010.igem.org/Team:Heidelberg/Project/Summary">Summary &amp; Outlook</A><A class="aPro_Ref" href="https://2010.igem.org/Team:Heidelberg/Project/References">References</A><A class="aPro_Ref" href="https://2010.igem.org/Team:Heidelberg/Project/Contributions">Contributions</A></TD><TD class="righttdtop"> </TD></TR></TBODY></TABLE></LI><LI id="menu_parts"><A class="aParts" href="https://2010.igem.org/Team:Heidelberg/Parts">Parts</A><TABLE class="menutable" cellpadding="0" cellspacing="0"><TBODY><TR><TD class="lefttdtop"><A class="aparts_ger" href="https://2010.igem.org/Team:Heidelberg/Parts">Parts</A><A class="aparts_char" href="https://2010.igem.org/Team:Heidelberg/Parts/Characterization">Characterization</A></TD><TD class="righttdtop"> </TD></TR></TBODY></TABLE></LI><LI id="menu_modeling"><A class="aModeling" href="https://2010.igem.org/Team:Heidelberg/Modeling">Modeling</A><TABLE class="menutable" cellpadding="0" cellspacing="0"><TBODY><TR><TD class="lefttdtop"><A class="amode_over" href="https://2010.igem.org/Team:Heidelberg/Modeling">Introduction</A><A class="amiGUI" href="https://2010.igem.org/Team:Heidelberg/Modeling/miGUI">miBEAT</A><A class="amirockdown" href="https://2010.igem.org/Team:Heidelberg/Modeling/miRockdown">miRockdown</A><A class="amibsdesigner" href="https://2010.igem.org/Team:Heidelberg/Modeling/miBSdesigner">miBS designer</A><A class="amispec" href="https://2010.igem.org/Team:Heidelberg/Modeling/miSpec">mUTING</A><A class="amodelset" href="https://2010.igem.org/Team:Heidelberg/Modeling/trainingset">Descriptions</A></TD><TD class="righttdtop"> </TD></TR></TBODY></TABLE></LI><LI id="menu_notebook"><A class="aNotebook" href="https://2010.igem.org/Team:Heidelberg/Notebook">Notebook</A><TABLE class="menutable" cellpadding="0" cellspacing="0"><TBODY><TR><TD class="lefttdtop"><A class="aNote_miRNA_kit" href="https://2010.igem.org/Team:Heidelberg/Notebook/miRNA_Kit">Synthetic miRNA Kit</A><A class="aNote_Capsid_Shuffling">Capsid Shuffling</A><A class="aNote_Homology" href="https://2010.igem.org/Team:Heidelberg/Notebook/Capsid_Shuffling/Homology_Based">    Homology Based</A><A class="aNote_Virobytes" href="https://2010.igem.org/Team:Heidelberg/Notebook/Capsid_Shuffling/ViroBytes">    ViroBytes</A><A class="aNote_miMeasure" href="https://2010.igem.org/Team:Heidelberg/Notebook/miMeasure">miMeasure</A><A class="aNote_BSdesign" href="https://2010.igem.org/Team:Heidelberg/Notebook/BSDesign">Binding Site Design </A><A class="aNote_Mouse" href="https://2010.igem.org/Team:Heidelberg/Notebook/Mouse_Infection"><I>in vivo</I> Study</A><A class="aNote_Modeling" href="https://2010.igem.org/Team:Heidelberg/Notebook/Modeling">Modeling</A><A class="aNote_Material" href="https://2010.igem.org/Team:Heidelberg/Notebook/Material">Materials</A><A class="aNote_Methods" href="https://2010.igem.org/Team:Heidelberg/Notebook/Methods">Methods</A><A class="aNote_Psycho" href="https://2010.igem.org/Team:Heidelberg/Human_Practices/Psychology/Notebook">Psychological Survey</A><A class="aNote_Philo" href="https://2010.igem.org/Team:Heidelberg/Human_Practices/Philosophy/Part2">Philosophical Reflection</A><A class="aNote_Biosafety" href="https://2010.igem.org/Team:Heidelberg/Notebook/Biosafety">Biosafety</A></TD><TD class="righttdtop"> </TD></TR></TBODY></TABLE></LI><LI id="menu_human_practices"><A class="aHuman_Practices" href="https://2010.igem.org/Team:Heidelberg/Human Practices">Human Practices</A><TABLE class="menutable" cellpadding="0" cellspacing="0"><TBODY><TR><TD class="lefttdtop"><A class="aPsycho" href="https://2010.igem.org/Team:Heidelberg/Human Practices/Psychology">Psychological Survey</A><A class="aPhilo" href="https://2010.igem.org/Team:Heidelberg/Human Practices/Philosophy">Philosophical Reflection</A></TD><TD class="righttdtop"> </TD></TR></TBODY></TABLE></LI><LI id="menu_support"><A class="aSupport" href="https://2010.igem.org/Team:Heidelberg/Support">Sponsors</A></LI></UL></DIV></DIV><TABLE id="toc" class="toc"><TBODY><TR><TD><DIV id="toctitle"><H2>Contents</H2></DIV><UL><LI class="toclevel-1 tocsection-1"><A href="#miBEAT:"><SPAN class="tocnumber">1</SPAN><SPAN class="toctext">miBEAT:</SPAN></A></LI><LI class="toclevel-2 tocsection-2"><A href="#miRockdown"><SPAN class="tocnumber">1.1</SPAN><SPAN class="toctext">miRockdown</SPAN></A></LI><LI class="toclevel-3 tocsection-3"><A href="#How_to_use_miRockdown"><SPAN class="tocnumber">1.1.1</SPAN><SPAN class="toctext">How to use miRockdown</SPAN></A></LI></UL><LI class="toclevel-2 tocsection-4"><A href="#miBSdesigner"><SPAN class="tocnumber">1.2</SPAN><SPAN class="toctext">miBSdesigner</SPAN></A></LI><LI class="toclevel-3 tocsection-5"><A href="#Input"><SPAN class="tocnumber">1.2.1</SPAN><SPAN class="toctext">Input</SPAN></A></LI><LI class="toclevel-3 tocsection-6"><A href="#Seed_Types"><SPAN class="tocnumber">1.2.2</SPAN><SPAN class="toctext">Seed Types</SPAN></A></LI><LI class="toclevel-3 tocsection-7"><A href="#Supplementary_Region"><SPAN class="tocnumber">1.2.3</SPAN><SPAN class="toctext">Supplementary Region</SPAN></A></LI><LI class="toclevel-3 tocsection-8"><A href="#AU_Content"><SPAN class="tocnumber">1.2.4</SPAN><SPAN class="toctext">AU Content</SPAN></A></LI><LI class="toclevel-3 tocsection-9"><A href="#Sticky_Ends"><SPAN class="tocnumber">1.2.5</SPAN><SPAN class="toctext">Sticky Ends</SPAN></A></LI><LI class="toclevel-3 tocsection-10"><A href="#Output"><SPAN class="tocnumber">1.2.6</SPAN><SPAN class="toctext">Output</SPAN></A></LI><LI class="toclevel-2 tocsection-11"><A href="#mUTING"><SPAN class="tocnumber">1.3</SPAN><SPAN class="toctext">mUTING</SPAN></A></LI><LI class="toclevel-3 tocsection-12"><A href="#Input_2"><SPAN class="tocnumber">1.3.1</SPAN><SPAN class="toctext">Input</SPAN></A></LI><LI class="toclevel-3 tocsection-13"><A href="#Data"><SPAN class="tocnumber">1.3.2</SPAN><SPAN class="toctext">Data</SPAN></A></LI><LI class="toclevel-3 tocsection-14"><A href="#Processing"><SPAN class="tocnumber">1.3.3</SPAN><SPAN class="toctext">Processing</SPAN></A></LI><LI class="toclevel-3 tocsection-15"><A href="#Output_2"><SPAN class="tocnumber">1.3.4</SPAN><SPAN class="toctext">Output</SPAN></A></LI><LI class="toclevel-1 tocsection-16"><A href="#Modeling"><SPAN class="tocnumber">2</SPAN><SPAN class="toctext">Modeling</SPAN></A></LI><LI class="toclevel-2 tocsection-17"><A href="#Parameterization_Concept"><SPAN class="tocnumber">2.1</SPAN><SPAN class="toctext">Parameterization Concept</SPAN></A></LI><LI class="toclevel-2 tocsection-18"><A href="#Neural_Network_Model"><SPAN class="tocnumber">2.2</SPAN><SPAN class="toctext">Neural Network Model</SPAN></A></LI><LI class="toclevel-3 tocsection-19"><A href="#Neural_Network_theory"><SPAN class="tocnumber">2.2.1</SPAN><SPAN class="toctext">Neural Network theory</SPAN></A></LI><LI class="toclevel-3 tocsection-20"><A href="#Model_description"><SPAN class="tocnumber">2.2.2</SPAN><SPAN class="toctext">Model description</SPAN></A></LI><LI class="toclevel-4 tocsection-21"><A href="#Input.2Ftarget_pairs"><SPAN class="tocnumber">2.2.2.1</SPAN><SPAN class="toctext">Input/target pairs</SPAN></A></LI><LI class="toclevel-4 tocsection-22"><A href="#Characteristic_of_the_Network"><SPAN class="tocnumber">2.2.2.2</SPAN><SPAN class="toctext">Characteristic of the Network</SPAN></A></LI><LI class="toclevel-3 tocsection-23"><A href="#Results"><SPAN class="tocnumber">2.2.3</SPAN><SPAN class="toctext">Results</SPAN></A></LI><LI class="toclevel-4 tocsection-24"><A href="#Brief_conclusion"><SPAN class="tocnumber">2.2.3.1</SPAN><SPAN class="toctext">Brief conclusion</SPAN></A></LI><LI class="toclevel-4 tocsection-25"><A href="#Simulation_and_experimental_verification"><SPAN class="tocnumber">2.2.3.2</SPAN><SPAN class="toctext">Simulation and experimental verification</SPAN></A></LI><LI class="toclevel-2 tocsection-26"><A href="#Fuzzy_Logic_Model"><SPAN class="tocnumber">2.3</SPAN><SPAN class="toctext">Fuzzy Logic Model</SPAN></A></LI><LI class="toclevel-3 tocsection-27"><A href="#Why_using_a_fuzzy_inference_system_to_model_binding_site_efficiency.3F"><SPAN class="tocnumber">2.3.1</SPAN><SPAN class="toctext">Why using a fuzzy inference system to model binding site efficiency?</SPAN></A></LI><LI class="toclevel-3 tocsection-28"><A href="#Fuzzy_Model_Concepts"><SPAN class="tocnumber">2.3.2</SPAN><SPAN class="toctext">Fuzzy Model Concepts</SPAN></A></LI><LI class="toclevel-3 tocsection-29"><A href="#Fuzzy_Model_Optimization"><SPAN class="tocnumber">2.3.3</SPAN><SPAN class="toctext">Fuzzy Model Optimization</SPAN></A></LI><LI class="toclevel-3 tocsection-30"><A href="#Result"><SPAN class="tocnumber">2.3.4</SPAN><SPAN class="toctext">Result</SPAN></A></LI><LI class="toclevel-1 tocsection-31"><A href="#Data_Overview"><SPAN class="tocnumber">3</SPAN><SPAN class="toctext">Data Overview</SPAN></A></LI><LI class="toclevel-1 tocsection-32"><A href="#References"><SPAN class="tocnumber">4</SPAN><SPAN class="toctext">References</SPAN></A></LI></TD></TR></TBODY></TABLE><P> </P></DIV><H1><SPAN class="mw-headline" id="miBEAT:">miBEAT:</SPAN></H1><P>miBEAT (<B>mi</B>RNA <B>B</B>inding site <B>E</B>ngineering and <B>A</B>ssembly <B>T</B>ool) is a graphical user interface that has as its back-end a compilation of multiple individual models and scripts which interact with each other to generate constructs. 
</P><H2><SPAN class="mw-headline" id="miRockdown">miRockdown</SPAN></H2><P>miRNA binding sites of varying binding efficiencies can be used to manipulate levels of proteins in a cell. Several tools can predict mRNA knockdown, but our approach aims for the final objective: protein levels (specially interesting for medical applications like gene therapy). This approach, offers a more complete result than the computational tools available, that focus on mRNA levels.
</P><H3><SPAN class="mw-headline" id="How_to_use_miRockdown">How to use miRockdown</SPAN></H3><P>The modeling project was planned in a way such that the model predictions would be available online in the form of a GUI. We made it in the most user-friendly way we could think of: The user only needs to input the desired knockdown percentage (kd%) and choose an sh/miRNA sequence, to get a binding site to satisfy their needs.</P><CENTER><DIV style="font-size:0.95em;"><B>Overview of the miRockdown script flow.</B>
The knockdown percentage (kd%) input invokes the selection of the appropriate experimental BS or theoretical binding site parameters. The miRNA sequence starts the generation of BS sequences. Subsequently, these BS sequences are characterized by a modified TargetScan algorithm and finally the parameters of the theoretical BS are compared with the parameters of the generated BSs and the closest of the generated BSs is given as output.</DIV></CENTER><P>
The results of both of our models and the experimentally verified binding sites are integrated in <A href="https://2010.igem.org/Team:Heidelberg/Modeling/miRockdown" class="external text" rel="nofollow">miRockdown</A> on <A href="https://2010.igem.org/Team:Heidelberg/Modeling/miGUI" class="external text" rel="nofollow">miBEAT</A>. 
For every binding site request of a user he receives the results of the three different approaches. Thus the user can always choose which of the three differently generated binding sites they want to use. 
The binding site with the closest experimentally observed knockdown percentage is displayed, together with its properties and oligos ready to clone into the <A href="https://2010.igem.org/Team:Heidelberg/Project/miRNA_Kit" class="external text" rel="nofollow">miTuner</A>-construct.
The binding sites generated using the models are useful when the user wants to use their own sh/miRNA or when there are not close enough experimentally verified binding sites.
A script integrated into miRockdown will correlate the desired kd% with a database for every model. This database consists of a set of binding site parameter objects spanning the complete range of parameters. Additionally, the database contains the models' knockdown percentage calculated for the whole set of objects.
With the user-chosen sh/miRNA sequence as input, a binding site generator script is invoked, which created more than 2000 different BS on-the-fly by varying the seed-type, 3'pairing, AU content and bulge size. The 3'pairing and the AU content score of the generated BS are characterized by a modified version of the TargetScan Algorithm <SPAN class="referencelink"><A href="http://2010.igem.org/Team:Heidelberg/Modeling/descriptions#References" class="external text" rel="nofollow">(Rodriguez et al., 2007)</A></SPAN>. The input and output functions of the algorithm were adapted for miRockdown, so that no files have to be generated.
Now that the generated binding sites are completely characterized, they are compared with the parameters of the suitable model BS. The generated BS that fits best the parameters of the suitable model BS is selected as the output BS of miRockdown.
</P><DIV class="backtop"><A href="#top">↑</A></DIV><H2><SPAN class="mw-headline" id="miBSdesigner">miBSdesigner</SPAN></H2><P>Having a binding site designer was crucial to complete the computational approach to our project: miBSdesigner is an easy-to-use application to create in silico binding sites for any given miRNA. Using our device, the user will be able to generate binding sites with several different properties.
</P><H3><SPAN class="mw-headline" id="Input">Input</SPAN></H3><P>The user has to input a name for the miRNA to name the primers. The miRNA sequence must be 22 nucleotides long and has to be input in direction 5’ to 3’ (both DNA and RNA sequences are admitted and any extra characters will be removed from the sequence). The user can also enter a spacer inert sequence if he needs to place the binding site further along in the 3’UTR region (it is recommended that the binding site is at least 15 nucleotides away from the stop codon).
Initially the user can choose between a perfect binding site (matching the 22 nucleotides), or an almost perfect binding site (matching all of the nucleotides, but leaving a 4-nucleotide bulge between 9 and 12. 
Apart from these two options, the user can further modify the binding site to meet their individual requirements.
</P><H3><SPAN class="mw-headline" id="Seed_Types">Seed Types</SPAN></H3><P>Figure 1: Interactions between two miRNAs and their binding sites with different types of seeds.
</P><P>
In miBS designer, the user can choose between several types of seed for their binding site (list ordered by increasing efficacy):
</P><P>- 6mer (abundance 21.5%): only the nucleotides 2-7 of the miRNA match with the mRNA.
</P><P>- 7merA1 (abundance 15.1%): the nucleotides 2-7 match with the mRNA, and there is an adenine in position 1.
</P><P>- 7merm8 (abundance 25%): the nucleotides 2-8 match with the mRNA.
</P><P>- 8mer (abundance 19.8%): the nucleotides 2-8 match with the mRNA and there is an adenine in position 1.
</P><P>- Apart from any of these options, the user can decide to create a customized seed with one mismatch included. By inputting a number (between 2-7) in the Customized mismatch position textbox
</P><P>The percentages of abundance are calculated among conserved mammalian sites for a highly conserved miRNA <SPAN class="referencelink"><A href="http://2010.igem.org/Team:Heidelberg/Modeling/descriptions#References" class="external text" rel="nofollow">(Bartel (2009))</A></SPAN>.
</P><H3><SPAN class="mw-headline" id="Supplementary_Region">Supplementary Region</SPAN></H3><P>In miBS designer, the user can choose among several types of supplementary regions, starting with 3 matching nucleotides (14-16), increasing sequentially until 8 (13-20), and then total matching (from 13-22, leaving a bulge)<SPAN class="referencelink"><A href="http://2010.igem.org/Team:Heidelberg/Modeling/descriptions#References" class="external text" rel="nofollow">(Grimson A et al(2007))</A></SPAN>.
In case the user needs some other specific supplementary region, he can customize the sequence by inputting the desired matching nucleotides (in numbers from 9 to 22, separated by commas).
</P><H3><SPAN class="mw-headline" id="AU_Content">AU Content</SPAN></H3><P>In order to allow the user to improve the efficiency of their binding sites, miBS designer offers options to increase the AU content by adding adenine or uracil to positions around the matches (specifically in -1, 0, 1, 8, 9 and 10). The function is designed so that it varies the AU content without introducing new pairings.
</P><H3><SPAN class="mw-headline" id="Sticky_Ends">Sticky Ends</SPAN></H3><P>To facilitate the task of introducing the binding site into a plasmid, the user can add sequences to both ends of the binding site. Initially, the user can choose among the <A href="http://openwetware.org/wiki/The_BioBricks_Foundation:RFC#BBF_RFC_12:_Draft_BioBrick.E2.84.A2_BB-2_standard_for_biological_parts" class="external text" rel="nofollow">RFC-12 standard for biobricks BB2</A>, the XmaI/XhoI restriction enzymes used in our <A href="https://2010.igem.org/Team:Heidelberg/Project/miRNA_Kit" class="external text" rel="nofollow">miTuner</A>-construct, or some custom sequences input by the user. In the last case, the output sequences will not be directly ready for cloning: the user has to either digest the construction prior to ligation, or to process the primers before ordering them to remove the extra nucleotides and create the overhangs.
</P><H3><SPAN class="mw-headline" id="Output">Output</SPAN></H3><P>miBS designer generates the primer needed to integrate the binding site desired into a plasmid, alongside with the primer for the complementary strand. It will also produce specific names for the two primers.
</P><H2><SPAN class="mw-headline" id="mUTING">mUTING</SPAN></H2><P>It is a tool developed to generate binding sites for miRNAs that could be used for tissue targeting based on both on- as well as off-targeting strategy. It takes as input the target and off-target tissues as well as the desired targeting strategy. User can also specify a threshold for difference in the level of relative expression (within a tissue) of miRNAs between target and off-target tissue. The program searches through a database of expression levels to give out a list of possible miRNAs which could be used. Out of these, the desired miRNA can be selected for which the final output is generated in the form of sense and anti-sense oligomers with overhangs that could be used to put binding sites in tandem or into a vector. 
</P><H3><SPAN class="mw-headline" id="Input_2">Input</SPAN></H3><P>The input for the tool is rather simple and consists of five fields.
</P><P><B>Organism</B> – The tool lets you choose between Human, Rat and Mouse as the source organism.
</P><P><B>Target</B> – From a list of tissues, the target (tissue where gene has to be expressed) can be selected.
</P><P><B>Off-target</B> – A list from which multiple off-targets can be selected is available. Here, the tissues from which gene expression has to be excluded can be included.
</P><P><B>Targeting</B> – This options lets you select the targeting strategy you want to employ.
</P><P><B>Threshold</B> – The threshold for difference in the level of relative expression of miRNA in the 	target and off-target tissue can be set here. The default value is 0.001.
</P><H3><SPAN class="mw-headline" id="Data">Data</SPAN></H3><P>The expression data and sequence data that the tool makes use of was recruited from preexisting data sources.
</P><P><B>Sequences</B> – mature miRNA sequences were obtained from mirBase Sequence Database Release 16 <SPAN class="referencelink"><A href="http://2010.igem.org/Team:Heidelberg/Modeling/descriptions#References" class="external text" rel="nofollow">(Griffiths-Jones S. et al.(2008))</A></SPAN>. 
</P><P><B>Expression profiles</B> - miRNA expression profiles were collected from a previously published resource of 172 human, 64 mouse and 16 rat small RNA libraries extracted from major organs and cell types <SPAN class="referencelink"><A href="http://2010.igem.org/Team:Heidelberg/Modeling/descriptions#References" class="external text" rel="nofollow">(Landgraf et al.(2007))</A></SPAN>. The expression values in the data represent the number of cloned mature microRNAs that were sequenced in each library and reported as clone counts. The counts are normalized by the total number of microRNAs that were cloned in each library. These values are then used to calculate the difference in relative miRNA levels for differential expression of the construct.
</P><H3><SPAN class="mw-headline" id="Processing">Processing</SPAN></H3><P>The processing of the data has been done by script written in PERL. After submitting the primary inputs, mentioned above, the tool gives the user a choice of different miRNAs that fulfill the criterion set in the input. These are displayed along with the miRNA expression values in the target (in case of off-targeting) or in the off-targets (in case of on-targeting). The expression values in the off-targets and target in the respective cases are required to be zero. Based on these values, the user can select the most suitable miRNA for their construct.
</P><H3><SPAN class="mw-headline" id="Output_2">Output</SPAN></H3><P>The final output is the binding site for the miRNA selected by the user. It consists of the sense strand and the anti-sense strand that would code the binding site. These are flanked by a spacer sequence that could be used for putting binding sites in tandem and for introducing cloning sites.
</P><H1><SPAN class="mw-headline" id="Modeling">Modeling</SPAN></H1><P>The Neural Network and the Fuzzy Logic Model explained here are the basis of the <A href="https://2010.igem.org/Team:Heidelberg/Modeling/miRockdown" class="external text" rel="nofollow">miRockdown</A> tool. The results of the optimized models are integrated as a database and enable the <A href="https://2010.igem.org/Team:Heidelberg/Modeling/trainingset#miRockdown" class="external text" rel="nofollow">miRockdown</A> output of binding sites, to have confidently predicted protein knockdown efficiency.
</P><H2><SPAN class="mw-headline" id="Parameterization_Concept">Parameterization Concept</SPAN></H2><P>One of the hardest tasks in the development of our models was to come up with good strategy to generate input parameters from the raw data. In our case, the raw data is the binding site sequence and the corresponding sh/miRNA-sequence. The final parameterization concept unites a basic distinction between perfect, bulged (near-perfect) and endogenous miRNA like BS, with the advanced 3'-scoring and AU-content evaluation. The endogenous miRNA like BS parameter is further split into the three <A href="https://2010.igem.org/Team:Heidelberg/Modeling#miRNA_binding_site_features" class="external text" rel="nofollow">seed-types</A>.
The targetscan_scores_50-algorithm <SPAN class="referencelink"><A href="http://2010.igem.org/Team:Heidelberg/Modeling/descriptions#References" class="external text" rel="nofollow">(Rodriguez et al., 2007)</A></SPAN> was used to characterize binding sites in respect to 3'-pairing and AU-content score. TargetScan aligns the miRNA with the mRNA sequence starting from a given seed-position in a way the highest possible 3'-score is reached. Binding from miRNA nucleotide 13-16 will add 1 to the score, pairings outside this region add 0.5. Offsets between bound miRNA and mRNA are also allowed, but will there is a penalty of 0.5 points for an offset higher than 2 nucleotides. The AU-content of 30 nucleotides upstream and downstream of the mRNA seed sequence is rated seed type dependent. The impact of the nucleotides decreases with the distance from the seed. The scoring system is based on a regressions applied to datasets from human, mouse, rat and dog mRNA knockdown <SPAN class="referencelink"><A href="http://2010.igem.org/Team:Heidelberg/Modeling/descriptions#References" class="external text" rel="nofollow">(Grimson et al., 2007)</A></SPAN>.
</P><P>Since all major prior modeling approaches used mRNA levels as training-set, our approach needs to will give a completely new insight into miRNA binding site functionality.
</P><CENTER>3'-pairing-Score vs AU-content-Score/ThreePrime vs BulgeSize/SeedType vs 3'pairing-Score/SeedType vs AU-score/ vs knockdown percentage of the training set data: 
These surface fits show the correlation of increasing 3' Binding Score and AU content Score with increasing knockdown-efficiency of the binding sites. Also the other parameters nicely correlate with our expectations.</CENTER><H2><SPAN class="mw-headline" id="Neural_Network_Model">Neural Network Model</SPAN></H2><H3><SPAN class="mw-headline" id="Neural_Network_theory">Neural Network theory</SPAN></H3><P>Artificial Neural Network usually called (NN), is a computational model that is inspired by the biological nervous system. The network is composed of simple elements called artificial neurons that are interconnected and operate in parallel. In most cases the NN is an adaptive system that can change its structure depending on the internal or and external information that flows into the network during the learning process. The NN can be trained to perform a particular function by adjusting the values of the connection, called weights, between the artificial neurons. Neural Networks have been employed to perform complex functions in various fields, including pattern recognition, identification, classification, speech, vision, and control systems.
Mathematically there are three basic components that describes a single layer network: the synapses of the artificial neurons that are modeled as weights and that represent how strong is the connection between the input and an artificial neuron. An adder, that sum up all the the weighted inputs and finally an activation function, that controls the amplitude of the output of the single layer. Generally there are three type of activation function: threshold, sigmoid, piecewise linear function. For our model the sigmoid function has been used, it can range the output between 0 and 1 or between -1 and 1 <SPAN class="referencelink"><A href="http://2010.igem.org/Team:Heidelberg/Modeling/descriptions#References" class="external text" rel="nofollow">(Kröse et al, 1996)</A></SPAN>.</P><CENTER>
Figure 2: representation of the mathematical model of a biological neuron.</CENTER><P>
During the learning process, difference between the desired output (target) and the network output is minimised. This difference is usually called cost; the cost function is the measure of how far is the network output from the desired value. A common cost function is the mean-squared error and there are several algorithms that can be used to minimise this function. The following figure displays such a loop.
</P><CENTER>
Figure 3: Training of a Neural Network.</CENTER><H3><SPAN class="mw-headline" id="Model_description">Model description</SPAN></H3><H4><SPAN class="mw-headline" id="Input.2Ftarget_pairs">Input/target pairs</SPAN></H4><P>The NN model has been created with the MATLAB NN-toolbox. The input/target pairs used to train the network comprise experimental and literature data <SPAN class="referencelink"><A href="http://2010.igem.org/Team:Heidelberg/Modeling/descriptions#References" class="external text" rel="nofollow">(Bartel et al., 2007)</A></SPAN>. The experimental data were obtained by measuring via luciferase assay the strength of knockdown due to the interaction between the shRNA and the binding site situated on the 3’UTR of luciferase gene (<A href="https://2010.igem.org/Team:Heidelberg/Project/miRNA_Kit" class="external text" rel="nofollow">miTuner</A>). Nearly 30 different rational designed binding sites were tested and the respective knockdown strength calculated.
Each input was represented by a four elements vector. Each element corresponded to a score value related to a specific feature of the binding site (as mentioned in the previous paragraph &quot;Parametrization concept&quot;). The three features used to describe the binding site were: seed type, the 3’pairing contribution and the AU-content. The input/target pair represented the relationship between a particular binding site and the related percentage of knockdown.
Two Neural Network were trained. One was trained with a pool of 45 data coming from literature. The other one was trained with experimental data. The latter network comprised 4 inputs instead of 3. the fourth input represented the size of the bulge in base pairs.  Afterwards both networks were used to predict percentages of knockdown given certain inputs. The predictions were then validated experimentally and compared among the different networks.
</P><H4><SPAN class="mw-headline" id="Characteristic_of_the_Network">Characteristic of the Network</SPAN></H4><P>The neural network comprised two layers (multilayer feedforward Network). The first layer is connected with the input network and it comprised 15 artificial neurons. The second layer is connected to the first one and it produced the output. For the first and the second layer a sigmoid activation function and a linear activation function were used respectively. The algorithm used for minimizing the cost function (sum squared error) was Resilient Back Propagation (RPROP).  RPROP has proven to be one of the best in terms of speed of convergence, moreover it can eliminate the harmful effect of sigmoid activation function used in the hidden layer.</P><P>
Figure 3: schematic illustration of the network components. Hidden represent the first layer and it comprised 15 artificial neurons, while output is the second and last layer producing the output. The symbol “w” was the representation of the weights and “b” of the biases.
</P><H3><SPAN class="mw-headline" id="Results">Results</SPAN></H3><P>Two experiment batches were performed. The network trained only with data coming from literature was used to predict the outcome of the first experiment batch. In Table 1 the simulated and experimental percentage of knockdown are showed.  It becomes clear by looking the results that the bulge size has indeed an effect on the knockdown percentage, in fact the network is able to simulate with high precision when the bulge size is on the range of 3 and 4 nt, but not when it becomes 1 or 0. It is important to underly here that the network was trained with literature values that did not take into consideration the bulge size as a key factor, TargetScan in fact, does not evaluate this binding site feature in the scoring process. The importance of the bulge size is also emphasized in the surface plot in the &quot;Parametrization concept&quot; .</P><CENTER><TABLE border="1" class="wikitable sortable" cellpadding="6" style="border:solid 1px #AAAAAA; border-collapse:collapse; background-color:#F9F9F9; empty-cells:show; font-size:0.9em;"><TBODY><TR><TH align="right"> 3' score </TH><TH> AU-score </TH><TH> bulge </TH><TH> seed type </TH><TH> bulge size </TH><TH> number BS </TH><TH> KD% experimental  </TH><TH> KD% simulated
</TH></TR><TR><TD align="right"> 7.5 </TD><TD> 0.624 </TD><TD> 1 </TD><TD> 3 </TD><TD> 4 </TD><TD> 1 </TD><TD> 0.85 </TD><TD> 0.82
</TD></TR><TR><TD align="right"> 7.5 </TD><TD> 0.595 </TD><TD> 1 </TD><TD> 3 </TD><TD> 4 </TD><TD> 1 </TD><TD> 0.81 </TD><TD> 0.81
</TD></TR><TR><TD align="right"> 7.5 </TD><TD> 0.576 </TD><TD> 1 </TD><TD> 3 </TD><TD> 4 </TD><TD> 2 </TD><TD> 0.92 </TD><TD> 0.8
</TD></TR><TR><TD align="right"> 4 </TD><TD> 0.314 </TD><TD> 0 </TD><TD> 3 </TD><TD> 0 </TD><TD> 1 </TD><TD> 0.69 </TD><TD> 0.56
</TD></TR><TR><TD align="right"> 2.5 </TD><TD> 0.314 </TD><TD> 0 </TD><TD> 3 </TD><TD> 0 </TD><TD> 1 </TD><TD> 0.08 </TD><TD> 0.49
</TD></TR><TR><TD align="right"> 5 </TD><TD> 0.336 </TD><TD> 0 </TD><TD> 2 </TD><TD> 0 </TD><TD> 1 </TD><TD> 0.72 </TD><TD> 0.42
</TD></TR><TR><TD align="right"> 1.5 </TD><TD> 0.327 </TD><TD> 0 </TD><TD> 3 </TD><TD> 0 </TD><TD> 1 </TD><TD> 0.28 </TD><TD> 0.44
</TD></TR><TR><TD align="right"> 2 </TD><TD> 0.327 </TD><TD> 0 </TD><TD> 3 </TD><TD> 0 </TD><TD> 1 </TD><TD> 0.58 </TD><TD> 0.46
</TD></TR><TR><TD align="right"> 2.5 </TD><TD> 0.221 </TD><TD> 0 </TD><TD> 2 </TD><TD> 0 </TD><TD> 1 </TD><TD> 0.34 </TD><TD> 0.28
</TD></TR><TR><TD align="right"> 7.5 </TD><TD> 0.597 </TD><TD> 1 </TD><TD> 3 </TD><TD> 4 </TD><TD> 1 </TD><TD> 0.82 </TD><TD> 0.81
</TD></TR><TR><TD align="right"> 7.5 </TD><TD> 0.603 </TD><TD> 1 </TD><TD> 3 </TD><TD> 4 </TD><TD> 1 </TD><TD> 0.83 </TD><TD> 0.81
</TD></TR><TR><TD align="right"> 7.5 </TD><TD> 0.624 </TD><TD> 1 </TD><TD> 3 </TD><TD> 4 </TD><TD> 1 </TD><TD> 0.77 </TD><TD> 0.82
</TD></TR><TR><TD align="right"> 7.5 </TD><TD> 0.6 </TD><TD> 1 </TD><TD> 3 </TD><TD> 4 </TD><TD> 1 </TD><TD> 0.76 </TD><TD> 0.81
</TD></TR><TR><TD align="right"> 7.5 </TD><TD> 0.603 </TD><TD> 1 </TD><TD> 3 </TD><TD> 4 </TD><TD> 1 </TD><TD> 0.82 </TD><TD> 0.81
</TD></TR><TR><TD align="right"> 5.5 </TD><TD> 0.799 </TD><TD> 1 </TD><TD> 2 </TD><TD style="background-color:#cacaca;"> 3 </TD><TD> 1 </TD><TD style="background-color:#cacaca;"> 0.59 </TD><TD style="background-color:#cacaca;"> 0.63
</TD></TR><TR><TD align="right"> 5.5 </TD><TD> 0.749 </TD><TD> 1 </TD><TD> 2 </TD><TD> 3 </TD><TD> 1 </TD><TD> 0.345 </TD><TD> 0.61
</TD></TR><TR><TD align="right"> 6.5 </TD><TD> 0.799 </TD><TD> 1 </TD><TD> 2 </TD><TD style="background-color:#cacaca;"> 1 </TD><TD> 1 </TD><TD style="background-color:#cacaca;"> 0.9 </TD><TD style="background-color:#cacaca;"> 0.67
</TD></TR><TR><TD align="right"> 6.5 </TD><TD> 0.773 </TD><TD> 1 </TD><TD> 2 </TD><TD> 1 </TD><TD> 1 </TD><TD> 0.775 </TD><TD> 0.67
</TD></TR><TR><TD align="right"> 1.5 </TD><TD> 0.38 </TD><TD> 1 </TD><TD> 2 </TD><TD style="background-color:#cacaca;"> 1 </TD><TD> 1 </TD><TD style="background-color:#cacaca;"> 0.68 </TD><TD style="background-color:#cacaca;"> 0.27
</TD></TR><TR><TD align="right"> 1.5 </TD><TD> 0.38 </TD><TD> 1 </TD><TD> 2 </TD><TD style="background-color:#cacaca;"> 4 </TD><TD> 1 </TD><TD style="background-color:#cacaca;"> 0.21 </TD><TD style="background-color:#cacaca;"> 0.27
</TD></TR></TBODY></TABLE>
Table 1: it shows the simulated data and the experimental results given the features of the binding site. The values in red, underline the discrepancy that occur between the two set of knockdown when the bulge size is the only feature changing. When the bulge size is not 1 the prediction are very precise and within the standard deviation of the experimental values (between 10-25%).</CENTER><H4><SPAN class="mw-headline" id="Brief_conclusion">Brief conclusion</SPAN></H4><P>The bulge size was identified as a very important parameter for knockdown efficiency. This led us to the conclusion of training another Neural Network only with our experimental data and  encompassing the bulge size in the input vector.
</P><DIV class="backtop"><A href="#top">↑</A></DIV><H4><SPAN class="mw-headline" id="Simulation_and_experimental_verification">Simulation and experimental verification</SPAN></H4><H2><SPAN class="mw-headline" id="Fuzzy_Logic_Model">Fuzzy Logic Model</SPAN></H2><H3><SPAN class="mw-headline" id="Why_using_a_fuzzy_inference_system_to_model_binding_site_efficiency.3F">Why using a fuzzy inference system to model binding site efficiency?</SPAN></H3><P>To be able to evaluate the complex features of an shRNA or miRNA binding site and predict a resulting knockdown percentage of the protein we developed a fuzzy inference system (fis). The parameterized properties of the binding sites serve as input and will be processed into the knockdown percentage as the single output. Thus our fuzzy inference system is characterized as a multiple input, single output fuzzy inference system (MISO).
</P><P>Fuzzy Logic is a rule-based approximate artificial reasoning method developed by Lotfi Zadeh in 1965. Its motivation is the observation that humans often think and communicate in a vague way, and yet can make precise decisions <SPAN class="referencelink"><A href="http://2010.igem.org/Team:Heidelberg/Modeling/descriptions#References" class="external text" rel="nofollow">(Nelles O (2000))</A></SPAN>. It has been widely used in engineering and Artificial Intelligence approaches such as Fuzzy Controllers and Fuzzy Expert Systems. Fuzzy Logic has also been used for the modeling of biological pathways <SPAN class="referencelink"><A href="http://2010.igem.org/Team:Heidelberg/Modeling/descriptions#References" class="external text" rel="nofollow">(Bosl et al (2007))</A></SPAN> and to analyze gene regulatory networks <SPAN class="referencelink"><A href="http://2010.igem.org/Team:Heidelberg/Modeling/descriptions#References" class="external text" rel="nofollow">(Laschov et al(2009))</A></SPAN>. Key advantages of Fuzzy logic-based approaches are (i) the ability to construct models based on prior knowledge of the system and experimental data and (ii) encode intermediate states for inputs and outputs, thus improving other logic-approaches that can only deal with ON/OFF states such as Boolean models <SPAN class="referencelink"><A href="http://2010.igem.org/Team:Heidelberg/Modeling/descriptions#References" class="external text" rel="nofollow">(Aldridge et al (2009))</A></SPAN> and (iii) simulations can be derived from both qualitative and quantitative data, both of which can be cast into the form of IF-THEN rules. Thus, FL constitutes a powerful approach for the understanding of heterogeneous datasets.
</P><P>Fuzzy inference systems are based on membership functions (MF). MFs rate input parameters how much they satisfy a criterion on a scale from 0 to 1. There can be one, or multiple MFs per input parameter. Like different criteria applied to an input. The height of persons for example can be evaluated with one MF - how much the person satisfies being tall. On the other hand, there could be 3 MFs, one evaluating the membership to small people, the second to medium sized people and the third one to big people. Changing the shape of the MF gives the opportunity to have either functional dependencies, allowing intermediate states of the membership values, or simple ON/OFF states, where the membership value can be only 0 or 1. Thus different kinds of input parameters can be evaluated with a fuzzy inference system. For the simple height example model the age of the person could be taken as second input and evaluated by a MF that is 0 until the age of 18 and 1 for older persons. Thus the model could differentiate between young and grown-up persons.
</P><P>Simple if-then rules can then be used to combine the input MF to an output MF. The satisfaction of a rule by an object (set of input parameters) is defined by the degree of membership of the object to the different MFs. The higher the satisfaction of the rule, the higher is the membership to the output MF.
The output MF can be a function like the input MF. This is the case in Mamdani method fuzzy inference systems <SPAN class="referencelink"><A href="http://2010.igem.org/Team:Heidelberg/Modeling/descriptions#References" class="external text" rel="nofollow">(Mamdani et al, (1975))</A></SPAN>. We are using a Sugeno method fuzzy inference system <SPAN class="referencelink"><A href="http://2010.igem.org/Team:Heidelberg/Modeling/descriptions#References" class="external text" rel="nofollow">(Sugeno(1985))</A></SPAN>, where the output MF is either a constant or a linear function depending on input parameters. The advantage of a Sugeno fuzzy inference system is, that it is computationally more efficient and easier to optimize or adapt due to the more simple output MF. Due to the non-intuitive combination of the 3'-pairing- and AU-content score, our fuzzy inference system needs to be optimized computationally.
</P><H3><SPAN class="mw-headline" id="Fuzzy_Model_Concepts">Fuzzy Model Concepts</SPAN></H3><P>Our fuzzy inference system can deal with 3 different kinds of shRNA binding sites. Perfect, bulged and endogenous-like binding sites are treated separately, due to the differences in their biological mechanism, as discussed earlier [link to binding site properties].
A perfect binding site is evaluated by a simple ON/OFF input MF evaluating the boolean input of 
</P><P>We came up with different concepts of what kind of input parameters to integrate into the fuzzy inference model and how to evaluate them. Therefore we parameterized the <A href="https://2010.igem.org/Team:Heidelberg/Modeling/trainingset" class="external text" rel="nofollow">properties of a large set of binding sites</A> according to various different BS characteristics.
The targetscan_50_context_scores – Algorithm <SPAN class="referencelink"><A href="http://2010.igem.org/Team:Heidelberg/Modeling/descriptions#References" class="external text" rel="nofollow">(Rodriguez et al., 2007)</A></SPAN> which evaluates binding sites in respect to 3'-pairing and AU-content gives out a score that seems appropriate to distinguish especially between endogenous miRNA like binding sites. A more detailed description on the concept of binding site parameterization can be found under <A href="https://2010.igem.org/Team:Heidelberg/Modeling/trainingset" class="external text" rel="nofollow">Model Training Set</A>.
</P><TABLE class="wikitable"><TBODY><TR><TD><DIV class="thumb tright"><DIV class="thumbinner" style="width:182px;"><DIV class="thumbcaption">Bulged binding sites concept: This model concept evaluates bulged- or &quot;near-perfect&quot; binding sites separately from conventional seed + 3'-pairing binding sites. Rule number 2 considers the bulge-size of the bulged binding site.</DIV></DIV></DIV></TD><TD><DIV class="thumb tright"><DIV class="thumbinner" style="width:182px;"><DIV class="thumbcaption">Bulged binding sites (including AU-content-score) concept: This concept extends the bulged-BS concept with the addition of AU-content score evaluation. Therefore rule number 2 was modified accordingly.</DIV></DIV></DIV></TD><TD><DIV class="thumb tright"><DIV class="thumbinner" style="width:182px;"><DIV class="thumbcaption">Consider low 3' score concept: This model concept takes into consideration, that binding sites with a 3'-score under 3 did not show a significant change in knockdown efficiency compared to a control with only seed pairing <SPAN class="referencelink"><A href="http://2010.igem.org/Team:Heidelberg/Modeling/descriptions#References" class="external text" rel="nofollow">(Grimson et al., 2007)</A></SPAN>. This is realized by rule number 6.</DIV></DIV></DIV></TD></TR></TBODY></TABLE><DIV class="backtop"><A href="#top">↑</A></DIV><H3><SPAN class="mw-headline" id="Fuzzy_Model_Optimization">Fuzzy Model Optimization</SPAN></H3><P>The Fuzzy Logic Toolbox and Global Optimization Toolbox were linked via a script and the output membership functions, as well as the input membership function parameters were optimized.
</P><H3><SPAN class="mw-headline" id="Result">Result</SPAN></H3><TABLE class="wikitable"><TBODY><TR><TD><DIV class="thumb tright"><DIV class="thumbinner" style="width:182px;"><DIV class="thumbcaption">Regression of Fuzzy Logic Model resulst vs Experimental results: The Fuzzy Logic Model is able to predict experimental results.</DIV></DIV></DIV></TD><TD><DIV class="thumb tright"><DIV class="thumbinner" style="width:182px;"><DIV class="thumbcaption">Correlation of AU-content-Score and 3'-pairing Score with knockdown% in the optimized model</DIV></DIV></DIV></TD></TR></TBODY></TABLE><P><A href="http://igem.bioquant.uni-heidelberg.de/igem_2010/FuzzyModelResults.html" class="external text" rel="nofollow">Click here, if you are interested in more recent model optimizations results!</A></P><H1><SPAN class="mw-headline" id="Data_Overview">Data Overview</SPAN></H1><P>The data used for training the models can be accessed <A href="https://2010.igem.org/Team:Heidelberg/Modeling/Data_Overview" class="external text" rel="nofollow">here</A>. It comprises of miRNA and corresponding binding site sequences along with the describing input parameters for the models.
</P><H1><SPAN class="mw-headline" id="References">References</SPAN></H1><P>- Bartel D.P., MicroRNAs: Target Recognition and Regulatory Functions, Cell(136):215-233(2009)
</P><P>- Grimson A, Farh KHF, Johnston WK, Garrett-Engele P, Lim LP, Bartel DP, MicroRNA Targeting Specificity in Mammals: Determinants beyond Seed Pairing, Molecular Cell(27):91-105(2007).
</P><P>- Laschov D, Margaliot M. Mathematical modeling of the lambda switch:a fuzzy logic approach. J Theor Biol. 21:475-89 (2009).
</P><P>- Mamdani, E.H. and S. Assilian, An experiment in linguistic synthesis with a fuzzy logic controller. International Journal of Man-Machine Studies, 7(1):1-13, (1975).
</P><P>- Bosl W. J. Systems biology by the rules: hybrid intelligent systems for pathway modeling and discovery. BMC Systems Biology 1:13 (2007).
</P><P>- Sugeno, M., Industrial applications of fuzzy control, Elsevier Science Pub. Co.,(1985).
</P><P>- Nelles O. Nonlinear System Identification Springer Verlag GmbH &amp; Co., Berlin, (2000).
</P><P>- <A href="http://www.targetscan.org/cgi-bin/targetscan/data_download.cgi?db=vert_50" class="external text" rel="nofollow">targetscan_50_context_scores.pl</A> 
Rodriguez J, Ge R, Walker K, and Bell G., Whitehead Institute for Biomedical Research. (2007,2008) 
</P><P>- Kröse B &amp; van der Smagt P, An introduction to Neural Networks, 8th Ed, (1996).
</P><P>- Aldridge B. B., Saez-Rodriguez J., Muhlich J. L., Sorger P. K., Lauffenburger D. A. Fuzzy logic analysis of kinase pathway crosstalk in TNF/EGF/insulin-induced signaling PLoS Comput Biol.5:e1000340 (2009).
</P><P>- MacKay D.J.C., A Practical Bayesian Framework for Backpropagation Networks, Neural Computation, 4(3):448-472(1992)
</P><P>- Landgraf P., Rusu M., Sheridan R., Sewer A., Iovino N., Aravin A., Pfeffer S., Rice A., Kamphorst A.O., Landthaler M., Lin C., Socci N.D., Hermida L., Fulci V., Chiaretti S., Foa R., Schliwka J., Fuchs U., Novosel A., Muller R.U., Schermer B., Bissels U., Inman J., Phan Q., Chien M., A mammalian microRNA expression atlas based on small RNA library sequencing, Cell. 129:1401-1414 (2007).
</P><P>- Griffiths-Jones S., Saini H.K., van Dongen S., Enright A.J. miRBase: tools for microRNA genomics. Nucleic Acid Research. 36:D154-D158 (2008).
</P></DIV><DIV class="printfooter">
Retrieved from &quot;<A href="http://2010.igem.org/Team:Heidelberg/Modeling/descriptions">http://2010.igem.org/Team:Heidelberg/Modeling/descriptions</A>&quot;</DIV></DIV><DIV id="footer-box" class="noprint"><DIV id="footer"><UL id="f-list"><LI id="t-recentchanges"><A href="/Special:RecentChanges" title="Recent changes">Recent changes</A></LI><LI id="t-whatlinkshere"><A href="/Special:WhatLinksHere/Team:Heidelberg/Modeling/descriptions" title="List of all wiki pages that link here [j]" accesskey="j">What links here</A></LI><LI id="t-recentchangeslinked"><A href="/Special:RecentChangesLinked/Team:Heidelberg/Modeling/descriptions" title="Recent changes in pages linked from this page [k]" accesskey="k">Related changes</A></LI><LI id="t-specialpages"><A href="/Special:SpecialPages" title="List of all special pages [q]" accesskey="q">Special pages</A></LI><LI><A href="/Special:Preferences">My preferences</A></LI></UL></DIV><DIV id="footer"><UL id="f-list"><LI id="t-print"><A href="/wiki/index.php?title=Team:Heidelberg/Modeling/descriptions&amp;printable=yes" title="Printable version of this page [p]" accesskey="p">Printable version</A></LI><LI id="t-permalink"><A href="/wiki/index.php?title=Team:Heidelberg/Modeling/descriptions&amp;oldid=208571" title="Permanent link to this revision of the page">Permanent link</A></LI><LI id="privacy"><A href="/2010.igem.org:Privacy_policy" title="2010.igem.org:Privacy policy">Privacy policy</A></LI><LI id="disclaimer"><A href="/2010.igem.org:General_disclaimer" title="2010.igem.org:General disclaimer">Disclaimers</A></LI></UL></DIV></DIV></BODY></HTML>