<HTML xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en" dir="ltr">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki  ltr ns-0 ns-subject page-Team_Heidelberg_Project_Mouse_Infection"><DIV id="globalWrapper"><DIV id="top-section"><DIV id="p-logo"><A href="/Main_Page" title="Main Page">&quot;
	    </A></DIV><DIV id="menubar" class="left-menu noprint"><UL><LI class="selected"><A href="/Team:Heidelberg/Project/Mouse_Infection">Page               </A></LI><LI class="new"><A href="/wiki/index.php?title=Talk:Team:Heidelberg/Project/Mouse_Infection&amp;action=edit&amp;redlink=1">Discussion               </A></LI><LI><A href="/wiki/index.php?title=Team:Heidelberg/Project/Mouse_Infection&amp;action=edit">View source               </A></LI><LI><A href="/wiki/index.php?title=Team:Heidelberg/Project/Mouse_Infection&amp;action=history">History               </A></LI><LI style="color:#808080;cursor:default">teams</LI></UL></DIV><DIV class="right-menu noprint" id="menubar"><UL><LI id="pt-login"><A href="/wiki/index.php?title=Special:UserLogin&amp;returnto=Team:Heidelberg/Project/Mouse_Infection" title="You are encouraged to log in; however, it is not mandatory [o]" accesskey="o">Log in</A></LI></UL></DIV><DIV id="search-controls" class="noprint"><FORM action="/Special:Search" id="searchform"> </FORM></DIV></DIV><DIV id="content"><H1 class="firstHeading">Team:Heidelberg/Project/Mouse Infection</H1><DIV id="bodyContent"><H3 id="siteSub" class="noprint">From 2010.igem.org</H3><DIV id="header"><DIV id="menu"><UL id="nav"><LI id="menu_home"><A class="aHome" href="https://2010.igem.org/Team:Heidelberg">Home</A></LI><LI id="menu_team"><A class="aTeam" href="https://2010.igem.org/Team:Heidelberg/Team">Team</A><TABLE class="menutable" cellpadding="0" cellspacing="0"><TBODY><TR><TD class="lefttdtopteam"><A class="aTeamOverview" href="https://2010.igem.org/Team:Heidelberg/Team">Overview</A><A class="aTeamMembers" href="https://2010.igem.org/Team:Heidelberg/Team/Members">Students</A><A class="aTeamMembers" href="https://2010.igem.org/Team:Heidelberg/Team/Members#Instructors">Instructors</A><A class="aTeamMembers" href="https://2010.igem.org/Team:Heidelberg/Team/Members#Advisors">Advisors</A><A class="aInstitutes" href="https://2010.igem.org/Team:Heidelberg/Team/Institutes">Institutes</A><A class="aGallery" href="https://2010.igem.org/Team:Heidelberg/Team/Gallery">Gallery</A><A class="aDance" href="https://2010.igem.org/Team:Heidelberg/midance">miDANCE</A><A class="amithanks" href="https://2010.igem.org/Team:Heidelberg/Team/miThanks">Acknowledgements</A></TD><TD class="righttdtop"> </TD></TR></TBODY></TABLE></LI><LI id="menu_project"><A class="aProject" href="https://2010.igem.org/Team:Heidelberg/Project">Project</A><TABLE class="menutable" cellpadding="0" cellspacing="0"><TBODY><TR><TD class="lefttdtopproject"><A class="aAbstract" href="https://2010.igem.org/Team:Heidelberg/Project">Abstract</A><A class="aIntroduction" href="https://2010.igem.org/Team:Heidelberg/Project/Introduction">Introduction</A><A class="aProject_miRNA_Kit" href="https://2010.igem.org/Team:Heidelberg/Project/miRNA_Kit">Synthetic miRNA Kit</A><A class="aPro_Capsid_Shuffling" href="https://2010.igem.org/Team:Heidelberg/Project/Capsid_Shuffling">Capsid Shuffling</A><A class="aPro_Homology" href="https://2010.igem.org/Team:Heidelberg/Project/Capsid_Shuffling/Homology_Based">    Homology Based</A><A class="aPro_Virobytes" href="https://2010.igem.org/Team:Heidelberg/Project/Capsid_Shuffling/ViroBytes">    ViroBytes</A><A class="aPro_miMeasure" href="https://2010.igem.org/Team:Heidelberg/Project/miMeasure">miMeasure</A><A class="aPro_Mouse" href="https://2010.igem.org/Team:Heidelberg/Project/Mouse_Infection"><I>in vivo</I> Study</A><A class="aPro_Summary" href="https://2010.igem.org/Team:Heidelberg/Project/Summary">Summary &amp; Outlook</A><A class="aPro_Ref" href="https://2010.igem.org/Team:Heidelberg/Project/References">References</A><A class="aPro_Ref" href="https://2010.igem.org/Team:Heidelberg/Project/Contributions">Contributions</A></TD><TD class="righttdtop"> </TD></TR></TBODY></TABLE></LI><LI id="menu_parts"><A class="aParts" href="https://2010.igem.org/Team:Heidelberg/Parts">Parts</A><TABLE class="menutable" cellpadding="0" cellspacing="0"><TBODY><TR><TD class="lefttdtop"><A class="aparts_ger" href="https://2010.igem.org/Team:Heidelberg/Parts">Parts</A><A class="aparts_char" href="https://2010.igem.org/Team:Heidelberg/Parts/Characterization">Characterization</A></TD><TD class="righttdtop"> </TD></TR></TBODY></TABLE></LI><LI id="menu_modeling"><A class="aModeling" href="https://2010.igem.org/Team:Heidelberg/Modeling">Modeling</A><TABLE class="menutable" cellpadding="0" cellspacing="0"><TBODY><TR><TD class="lefttdtop"><A class="amode_over" href="https://2010.igem.org/Team:Heidelberg/Modeling">Introduction</A><A class="amiGUI" href="https://2010.igem.org/Team:Heidelberg/Modeling/miGUI">miBEAT</A><A class="amirockdown" href="https://2010.igem.org/Team:Heidelberg/Modeling/miRockdown">miRockdown</A><A class="amibsdesigner" href="https://2010.igem.org/Team:Heidelberg/Modeling/miBSdesigner">miBS designer</A><A class="amispec" href="https://2010.igem.org/Team:Heidelberg/Modeling/miSpec">mUTING</A><A class="amodelset" href="https://2010.igem.org/Team:Heidelberg/Modeling/trainingset">Descriptions</A></TD><TD class="righttdtop"> </TD></TR></TBODY></TABLE></LI><LI id="menu_notebook"><A class="aNotebook" href="https://2010.igem.org/Team:Heidelberg/Notebook">Notebook</A><TABLE class="menutable" cellpadding="0" cellspacing="0"><TBODY><TR><TD class="lefttdtop"><A class="aNote_miRNA_kit" href="https://2010.igem.org/Team:Heidelberg/Notebook/miRNA_Kit">Synthetic miRNA Kit</A><A class="aNote_Capsid_Shuffling">Capsid Shuffling</A><A class="aNote_Homology" href="https://2010.igem.org/Team:Heidelberg/Notebook/Capsid_Shuffling/Homology_Based">    Homology Based</A><A class="aNote_Virobytes" href="https://2010.igem.org/Team:Heidelberg/Notebook/Capsid_Shuffling/ViroBytes">    ViroBytes</A><A class="aNote_miMeasure" href="https://2010.igem.org/Team:Heidelberg/Notebook/miMeasure">miMeasure</A><A class="aNote_BSdesign" href="https://2010.igem.org/Team:Heidelberg/Notebook/BSDesign">Binding Site Design </A><A class="aNote_Mouse" href="https://2010.igem.org/Team:Heidelberg/Notebook/Mouse_Infection"><I>in vivo</I> Study</A><A class="aNote_Modeling" href="https://2010.igem.org/Team:Heidelberg/Notebook/Modeling">Modeling</A><A class="aNote_Material" href="https://2010.igem.org/Team:Heidelberg/Notebook/Material">Materials</A><A class="aNote_Methods" href="https://2010.igem.org/Team:Heidelberg/Notebook/Methods">Methods</A><A class="aNote_Psycho" href="https://2010.igem.org/Team:Heidelberg/Human_Practices/Psychology/Notebook">Psychological Survey</A><A class="aNote_Philo" href="https://2010.igem.org/Team:Heidelberg/Human_Practices/Philosophy/Part2">Philosophical Reflection</A><A class="aNote_Biosafety" href="https://2010.igem.org/Team:Heidelberg/Notebook/Biosafety">Biosafety</A></TD><TD class="righttdtop"> </TD></TR></TBODY></TABLE></LI><LI id="menu_human_practices"><A class="aHuman_Practices" href="https://2010.igem.org/Team:Heidelberg/Human Practices">Human Practices</A><TABLE class="menutable" cellpadding="0" cellspacing="0"><TBODY><TR><TD class="lefttdtop"><A class="aPsycho" href="https://2010.igem.org/Team:Heidelberg/Human Practices/Psychology">Psychological Survey</A><A class="aPhilo" href="https://2010.igem.org/Team:Heidelberg/Human Practices/Philosophy">Philosophical Reflection</A></TD><TD class="righttdtop"> </TD></TR></TBODY></TABLE></LI><LI id="menu_support"><A class="aSupport" href="https://2010.igem.org/Team:Heidelberg/Support">Sponsors</A></LI></UL></DIV></DIV><DIV style="width:340px; padding-top:41px; padding-left:12px; float:right; margin-right:-73px;"><TABLE border="0" cellpadding="0" cellspacing="0" style="width:318px; margin:0px; table-layout:fixed; border-style:none; border-spacing:0; vertical-align:top; background-color:transparent;"><TBODY><TR><TD><DIV id="toctitle"><H2>Contents</H2></DIV><UL><LI class="toclevel-1 tocsection-1"><A href="#construct_schemes"><SPAN class="tocnumber">1</SPAN><SPAN class="toctext">construct schemes</SPAN></A></LI><LI class="toclevel-1 tocsection-2"><A href="#in_vivo_study"><SPAN class="tocnumber">2</SPAN><SPAN class="toctext">in vivo study</SPAN></A></LI><LI class="toclevel-2 tocsection-3"><A href="#Abstract"><SPAN class="tocnumber">2.1</SPAN><SPAN class="toctext">Abstract</SPAN></A></LI><LI class="toclevel-2 tocsection-4"><A href="#Introduction"><SPAN class="tocnumber">2.2</SPAN><SPAN class="toctext">Introduction</SPAN></A></LI><LI class="toclevel-2 tocsection-5"><A href="#Results"><SPAN class="tocnumber">2.3</SPAN><SPAN class="toctext">Results</SPAN></A></LI><LI class="toclevel-3 tocsection-6"><A href="#Strategies"><SPAN class="tocnumber">2.3.1</SPAN><SPAN class="toctext">Strategies</SPAN></A></LI><LI class="toclevel-3 tocsection-7"><A href="#1._Viral_capsid_.2F_Gene_delivery"><SPAN class="tocnumber">2.3.2</SPAN><SPAN class="toctext">1. Viral capsid / Gene delivery</SPAN></A></LI><LI class="toclevel-3 tocsection-8"><A href="#2._miRNA-based_OFF_targeting"><SPAN class="tocnumber">2.3.3</SPAN><SPAN class="toctext">2. miRNA-based OFF targeting</SPAN></A></LI><LI class="toclevel-3 tocsection-9"><A href="#3._miRNA-based_fine-tuning"><SPAN class="tocnumber">2.3.4</SPAN><SPAN class="toctext">3. miRNA-based fine-tuning</SPAN></A></LI><LI class="toclevel-3 tocsection-10"><A href="#Bioluminescence_imaging"><SPAN class="tocnumber">2.3.5</SPAN><SPAN class="toctext">Bioluminescence imaging</SPAN></A></LI></UL><LI class="toclevel-2 tocsection-11"><A href="#Discussion"><SPAN class="tocnumber">2.4</SPAN><SPAN class="toctext">Discussion</SPAN></A></LI><LI class="toclevel-2 tocsection-12"><A href="#Methods"><SPAN class="tocnumber">2.5</SPAN><SPAN class="toctext">Methods</SPAN></A></LI><LI class="toclevel-3 tocsection-13"><A href="#Contructs"><SPAN class="tocnumber">2.5.1</SPAN><SPAN class="toctext">Contructs</SPAN></A></LI><LI class="toclevel-3 tocsection-14"><A href="#Production_of_recombinant_virus"><SPAN class="tocnumber">2.5.2</SPAN><SPAN class="toctext">Production of recombinant virus</SPAN></A></LI><LI class="toclevel-3 tocsection-15"><A href="#Procedure_involving_animals"><SPAN class="tocnumber">2.5.3</SPAN><SPAN class="toctext">Procedure involving animals</SPAN></A></LI><LI class="toclevel-3 tocsection-16"><A href="#in_vivo_animal_imaging"><SPAN class="tocnumber">2.5.4</SPAN><SPAN class="toctext">in vivo animal imaging</SPAN></A></LI><LI class="toclevel-2 tocsection-17"><A href="#References"><SPAN class="tocnumber">2.6</SPAN><SPAN class="toctext">References</SPAN></A></LI></TD></TR></TBODY></TABLE><H3><SPAN class="mw-headline" id="construct_schemes"> construct schemes </SPAN></H3><P> </P></DIV><H1><SPAN class="mw-headline" id="in_vivo_study"><I>in vivo</I> study</SPAN></H1><H2><SPAN class="mw-headline" id="Abstract">Abstract</SPAN></H2><P>Gene therapy offers a great tool for treatment of various human diseases and has shown tremendous promise and success in many recent clinical trials. Nevertheless, two most critical hurdles that remain are 1) the development of delivery vectors that allow for the highly cell- and tissue-specific expression of a gene of interest (GOI), as well as 2) the implementation of novel systems that permit the tightly controlled and fine-tuned expression of this gene in human patients. To fill in these two essential gaps, the iGEM Team 2010 of Heidelberg is introducing molecular evolution of viral gene transfer vectors as well as mi(cro)RNA-based gene regulation technology into the realm of synthetic biology. By combining these two powerful new methodologies, we have succeeded in engineering synthetic viral miRNA vectors that we could demonstrate to provide potent, specific and regulated gene expression in cultured mouse and human hepatocytes. Most importantly, we also document the successful use of our novel approach to achieve tightly controlled and liver-specific expression of a vector-encoded recombinant DNA in intact livers of adult mice. Our work and data thus pave the way for the future further expansion and utilization of our unique miRNA/vector kits in numerous biotechnological and especially biomedical applications.
</P><H2><SPAN class="mw-headline" id="Introduction">Introduction</SPAN></H2><P>Gene therapy offers a fantastic and unique tool for efficient treatment of various human diseases with a genetic cause, ranging from metabolic disorders and cancers to infections with different viral pathogens (Zincarelli, Soltys et al. 2008).  In principal, the underlying concept comprises the initial identification of a certain genetic defect, such as a deletion of a critical tumor suppressor gene which then causes cancer, or a mutation in a gene that is essential for a given metabolic pathway and whose loss or malfunction accordingly triggers symptoms of a disease. Once identified, gene therapists then attempt to correct this specific defect by introducing a correct copy of the missing or mutated gene back into the affected cell, hoping that this newly delivered gene will behave just like the original that it replaces. For transfer of this therapeutic gene sequence, one can choose between physical carriers, such as liposomes or nanoparticles, or alternatively, many groups engineer naturally occurring viruses, such as Adenoviruses, Lentiviruses or Adeno-associated viruses (AAV) as gene delivery vehicles. The latter hold the particular advantage that viruses have evolved to potently and, at least in many cases, specifically transfer their genetic cargo to cells, implying that a recombinant therapeutic DNA (or RNA) replacing these genes will behave just the same. </P><P>However, as one can easily imagine from the complexity of gene expression networks in all cells, especially in those of higher organisms, this whole process of transferring a foreign gene into a cell and expressing it there is everything but trivial, and success is not necessarily guaranteed. In essence, there are several main challenges that still need to be overcome before gene therapy, or more generally gene transfer, in humans can become a routine application, namely, improvements in 1) specificity, 2) potency, 3) control and 4) related to all this, safety. Unsurprisingly, there have been a plethora of attempts in the past to tackle these challenges, and a great deal of success has been reported in the literature. What is curious about these findings and advances, however, is that the vast majority of them have merely focused on the recombinant DNA template itself and have tried to (and often managed to) improve its particular properties, such as stability or encapsidation into the shell of a recombinant virus for delivery to the target cell.</P><P>Still, these particular approaches to improve the therapeutic DNA have taught us an important lesson, namely, that for successful, specific and potent gene transfer into human cells, especially into human patients, DNA is not enough ! Instead, there are two additional levels that are at least equally important and that have therefore been in the center of our team's focus: 1) the carrier used for gene delivery and 2) the control of gene expression inside the cell. More specifically, we have decided to study and improve the above already mentioned Adeno-associated virus (a large family of over 100 naturally occurring different serotypes) as a gene delivery vehicle due to a number of outstanding advantages of this particular virus for use in cell and tissue engineering, including the complete lack of pathogenicity of the parental wildtype viruses plus their capability to infect a wide range of target cells and tissues (<A href="https://2010.igem.org/Team:Heidelberg/Project/Mouse_Infection#References" class="external text" rel="nofollow">Grimm, 2002</A>). On the other hand, this broad tropism can also be a disadvantage in cases where one would like to target a very specific cell type and concurrently avoid off-target gene expression elsewhere in the body. Therefore, several research groups (including the one of our instructor Dr. Grimm) have very recently begun to devise novel methods to molecularly engineer synthetic &quot;designer&quot; AAVs based on naturally occurring capsids which ideally combine and enhance their best properties. For instance, it is now possible to create entire libraries of AAV capsids by inserting short targeting peptides into exposed regions of the virus surface, or alternatively, by randomly point mutating single amino acids in the capsid proteins. The latest and most potent approach is, however, to shuffle entire capsid genes by first fragmenting and subsequently re-assembling the parts into hybrid particles. Ideally, the resulting chimeras will combine the assets of all parental viruses, such as high potency in a given cell type coupled with evasion of neutralization by human anti-AAV antibodies (<A href="https://2010.igem.org/Team:Heidelberg/Project/Mouse_Infection#References" class="external text" rel="nofollow">Grimm et al., 2008</A>). </P><P>Regarding the second critical level, regulation of gene expression following DNA delivery, it would be particularly crucial in the context of many gene transfer/therapy applications to be able to fine-regulate the level of the resulting protein expression, considering that uncontrolled over-production of certain proteins can be highly toxic or even lethal to cells and tissues. Unfortunately, as mentioned above, most approaches in this direction thus far have mainly focused on the DNA itself and accordingly mostly dealt with the development of improved promoter sequences that mediate controlled gene expression under certain conditions. What has been largely ignored until very recently, however, is the fact that gene expression can also be regulated on the RNA level, namely, via RNAi technologies. In this respect, we reasoned that the novel and broadly applicable miRNA tools engineered by our iGEM team should prove extremely useful in combination with gene therapy vectors, considering that they have been devised to permit very tight control over the expression of a given gene on the RNA level (by controlling translation and/or mRNA stability, which is the natural mechanism of miRNA action). Specifically, we wanted to explore the feasibility to fuse a gene of interest (GOI) with binding sites for either a naturally occurring miRNA or for a synthetic miRNA that is co-expressed from a second &quot;tuning&quot; vector. In either case, the binding site can then be designed to match the mi/shRNA either perfectly, which will result in nearly complete knockdown of gene expression, or it can be imperfect, which will yield a more subtle reduction in gene expression (for details, see our synthetic miRNA kit).   </P><P>Accordingly, we utilized our various novel tools to establish an entirely new system for fine-regulated gene expression from viral gene therapy vectors, in which control is exerted no longer on the DNA level, but in addition also on the steps of 1) gene delivery and 2) fine-tuning.       </P><P>Most importantly, as with any novel system for regulated gene expression, there is no guarantee that results obtained in cultured cells are relevant for, and will straight-forwardly translate into, identical finding in living animals or ultimately human patients. Therefore, we reasoned that it would be mandatory following the design and in vitro validation of our novel constructs to also assay and ideally confirm their functionality in living organisms in vivo, i.e., in adult mice. 
In the following, we will summarize our results from the in vivo evaluation of the two main strategies outlined above in the livers of adult mice and will demonstrate how modifying the viral gene vehicle capsid can drastically affect the specificity and level of gene expression in cultured cells versus living animals, and how engineering a given gene expression cassette by adding perfect or imperfect mi/shRNA binding sites can further fine-tune the strength of gene/protein expression at will. 
</P><H2><SPAN class="mw-headline" id="Results">Results</SPAN></H2><H3><SPAN class="mw-headline" id="Strategies">Strategies</SPAN></H3><P>As outlined, we have pursued two fundamentally different strategies to achieve regulated gene expression in mice, which are control via the viral capsid as well as via RNAi/miRNAs. Generally, to measure the success of each individual strategy, it was mandatory to be able to detect and quantify gene expression in living adult mice to begin with. Moreover, we had to define a standard based on which we would evaluate the specificity and efficiency of each new construct and strategy that we wanted to test. As we were going to target the liver of mice (since it is one of the most therapeutically relevant organs plus concurrently very susceptible to gene transfer), we decided to use the particular AAV serotype 8 (AAV8) as our standard, based on previous findings by Dr. Grimm and others that it mediates maximum transduction of livers in adult mice. Additionally, we picked Firefly luciferase as our reporter gene as it can be easily detected and quantified in living mice following a simple intraperitoneal injection of luciferin substrate and using a sensitive camera system. 
Hence, in our very first animal experiment, we infused adult white mice with an AAV8 vector expressing Firefly luciferase from the universally active SV40 promoter and then measured luciferase levels in the liver 5 days after injection. As shown in the center of  Figure 1, we were indeed able to detect a very strong expression in all livers of three independent mice, with an average photon count of 2.77x108 per mouse. In all subsequent experiments, this value served as our reference based on which we aimed to achieve a fine-regulation of in vivo gene expression. 
</P><H3><SPAN class="mw-headline" id="1._Viral_capsid_.2F_Gene_delivery">1. Viral capsid / Gene delivery</SPAN></H3><P>Next, we asked how delivering the exact same luciferase vector to mice from a different viral capsid would alter the expression levels in the liver. For this purpose, we used our lead candidate from the virus shuffling and selection procedure described in the homology based capsid shuffling section. As noted there, this one particular capsid was curious in that it was the only one capable of potently transducing Huh7 and HepG2 cells, strongly suggesting that it might also yield reasonable gene expression in vivo. Indeed, when we performed the experiment with this synthetic capsid under the exact same conditions as with our standard AAV8 capsid, we again noted effective luciferase gene expression in the livers of three independently injected mice. 
Three findings were particularly remarkable in this experiment: 
1) The average photon counts per mouse were 1.01x107 and thus about 27-fold lower than with the AAV8 standard vector. This clearly and impressively proved our theory that simply replacing the viral capsid can dramatically affect the expression of a given gene in vivo and validates our hypothesis that DNA is not enough, but that the vector for DNA delivery is equally important !
2) Intriguingly, when we normalized the luciferase measurement settings for both viruses (wildtype 8 and synthetic iGEM vector) to achieve equal expression data in the liver, we realized that the wildtype virus showed a dramatic off-targeting activity outside the liver. In fact, basically the entire mouse was lighting up positive in the camera at these sensitive settings, confirming published findings that AAV8 has an extremely broad in vivo tropism that includes the brain. In striking contrast, although our own virus gave virtually identical luciferase values in the liver, it was completely inactive anywhere outside of this organ. This clearly validated our strategy to select and test a viral clone that was positive in cell culture on two independent liver cell lines and thus proves the power of our virus evolution/selection strategy to enrich for synthetic capsids with desired gene transduction capabilities.  
3) Curiously, we then found that our synthetic clone clearly outperformed the corresponding AAV8 counterpart on cultured cells, despite the fact that it was about 27-fold less efficient in the liver, as noted above. In fact, it is a well known phenomenon for all wildtype AAV viruses that they either function in cell culture or in mice, but not both. Hence, our finding with our new synthetic clone which gave reasonable expression in vivo AND in vitro is even further proof for our concept to molecularly evolve viral gene transfer vectors and then select synthetic chimeras exhibting properties that do not exist in nature.</P><H3><SPAN class="mw-headline" id="2._miRNA-based_OFF_targeting">2. miRNA-based OFF targeting</SPAN></H3><P>In the above described first set of experiments, we aimed to fine-regulate gene expression by modifying the viral capsids delivering the same gene as was originally expressed from our standard AAV8 vector. In the next step, we asked whether it was also possible to exploit the endogenous expression of cellular miRNAs in order to further control gene expression levels from a viral vector. Generally, this strategy can be exploited in at least two ways: 1) a GOI can be fused with a perfectly matching binding site for such a miRNA, which will typically result in nearly complete inhibition of gene expression, or 2) this site can be imperfect which will yield a gradual reduction of expression. 
A particularly useful application of the first (perfect site) strategy would be to purposely de-target the expression of a viral vector or a gene, respectively, from a certain tissue in which gene expression is not supposed to occur. In order to test this possibility with our system, we engineered our luciferase vector to contain four perfect binding sites for the miR-122, which is specific for liver and highly abundant in hepatocytes. We then again packaged the resulting vector construct into the potent AAV8 capsid and also injected this virus into adult mice. Impressively, and exactly as predicted and hoped for, we found that luciferase expression from this vector was almost 100-fold reduced as compared to the unmodified standard AAV8 vector. In fact, average photon counts per liver (again three independent mice) dropped to 4.95x106 or 1.8%, even though the luciferase gene was still delivered by the same highly potent capsid. </P><P>This obviously validated our hypothesis that cellular miRNAs can be exploited to substantially regulate the expression of a vector-encoded recombinant DNA and is thus further proof for our concept that neither DNA nor the vector are sufficient to achieve the maximum possible level of control over in vivo gene expression.
</P><H3><SPAN class="mw-headline" id="3._miRNA-based_fine-tuning">3. miRNA-based fine-tuning</SPAN></H3><P>As just indicated, we predicted that fusing our luciferase gene with an imperfect binding site for a miRNA would result in a more gradual reduction in gene expression as compared to the perfect site. To test this hypothesis, we generated a series of expression vectors in which the luciferase gene was tagged with perfect or imperfect binding sites for an artificial miRNA (an shRNA directed against the hAAT gene). This miRNA was then packaged into and co-expressed from a second AAV8 vector that we generated in parallel. The results we obtained are shown in Figure1.</P><P>Three important conclusions can be drawn from this experiment:
1) Modifying the luciferase gene with different binding sites has no effect on basal gene expression levels (at least not in this example in our hands) when compared to the standard unmodified AAV8 vector (see figure above), which is remarkable as it suggests that our strategy should be broadly applicable for many other recombinant DNA vectors as well.
2) Tagging the luciferase gene with a perfect binding site and then co-expressing the corresponding artificial miRNA from a second vector resulted in the expected potent down-/fine-regulation of gene expression to about 30% of starting levels from the unmodified standard vector. Note that the effect is less pronounced than what we observed with the perfect sites for miR-122 above, which can however be easily explained with the fact that the miR-122 contained four perfect binding sites versus only one in the present vector for the artificial miRNA. Moreover, one has to take into account that the artifical miRNAs was co-expressed from a second vector in the current setting which certainly resulted in vastly different kinetics as compared to an endogenous miRNA.
3) Tagging luciferase with an imperfect miRNA binding site had the expected intermediate effect on gene expression and resulted in a fine-regulation to about 58% of the levels observed with the unmodified control vector. This is an extremely remarkable result as it clearly shows that using our miTuner strategy, it is possible to achieve highly similar extents of fine-regulation of gene expression in cultured cells as well as in living adult animals. It thereby strongly validates our overall strategy to combine synthetic miRNAs and corresponding binding sites with pre-existing gene expression cassettes in order to fine-tune the latter in a very deliberate, controlled and predictable (from straight-forward in vitro experiments) fashion.  </P><H3><SPAN class="mw-headline" id="Bioluminescence_imaging">Bioluminescence imaging</SPAN></H3><P>Figure 1: Scheme of luciferase </P><H2><SPAN class="mw-headline" id="Discussion">Discussion</SPAN></H2><P>In the above described and documented set of animal experiments, our team has been the very first within the iGEM competition to demonstrate the enormous power and promise of viral vector evolution and miRNA technologies to achieve a level of fine-regulation of in vivo gene expression that has previously been unprecedented. Most importantly, it must be pointed out that we consider our data just as a tip of the iceberg and as a highly promising starting point for numerous further similar developments and improvements that we expect in the future of synthetic biology and biomedicine. 
For instance, we can readily predict from our findings in the liver that it will be possible to evolve many further vectors for the same or for other therapeutically relevant cells and tissues, such as the heart of the brain. As in the liver, it is then very likely that these vectors will yield gene expression levels that differ from any of the naturally occurring viral serotypes and that may prove particularly beneficial for certain applications. 
Likewise, we easily anticipate that our findings with the power of miRNAs – endogenously expressed or co-delivered from a second vector – can be adapted and expanded in other applications, to achieve probably any desired level of fine-regulation of gene expression. Here, we have already shown that we can modulate an originally extremely potent (and for many gene therapy applications TOO potent !) vector in the liver over two orders of magnitude (anywhere from 1.8 to 58.2% in the shown examples), and we have no doubt that similar results can be obtained with other miRNAs and in other cells and tissues as well.
Taken together, we are thus highly optimistic that our work has laid the foundation for a whole new generation of gene delivery vectors and gene expression cassettes that will exhibit substantially improved properties for use in humans and that will thus – by combining synthetic biology with existing biotechnological and biomedical approaches – significantly foster the urgently needed clinical routine implementation of gene therapy in human patients.</P><H2><SPAN class="mw-headline" id="Methods">Methods</SPAN></H2><H3><SPAN class="mw-headline" id="Contructs">Contructs</SPAN></H3><P>The <I>in vivo</I> analysis should enlighten our gene therapy approach using AAV tropism as well as miRNA binding sites as trigger for expression. The following constructs have been subcloned separately into the AAV context to accomplish those tasks: </P><OL><LI> positive control, 
</LI><LI> off-targeting construct, 
</LI><LI> synthetic tuning construct and 
</LI><LI> on-targeting construct. 
</LI></OL><P>
All but one virus were packaged by the AAV rep and cap gene with Adenovirus 5 (Ad5) as a helper plasmid. Accordingly, one virus construct was packaged into a shuffled cap gene from our <A href="https://2010.igem.org/Team:Heidelberg/Project/Capsid_Shuffling/Homology_Based" class="external text" rel="nofollow">homology based capsid shuffling</A> attempt. 
</P><OL><LI> The positive control (see sidebar, fig. 1) consisted of the SV40 promoter driving a firefly luciferase (luc2) gene, thereby leading to an unspecific expression of the luciferase protein in all mice tissues. In addition to packaging this construct into a wild type AAV virus, the positive control was also packaged as a transgene into our <A href="https://2010.igem.org/Team:Heidelberg/Project/Capsid_Shuffling/Homology_Based" class="external text" rel="nofollow">shuffled capsid</A> which after random selection  was already able to <A href="https://2010.igem.org/Team:Heidelberg/Notebook/Homology_Based/October#16/10/2010" class="external text" rel="nofollow">positively transduce Huh7 and HepG2 cells</A><I>in vitro</I>. 
</LI><LI> The off-targeting construct (see sidebar, fig. 2) was composed of an SV40 promoter driving a firefly luciferase (luc2) gene with binding sites against miR-122 behind it. In order to achieve the highest expression in all mice cells but the liver cells - a single perfect binding site of miR-122 was used for <I>in vivo</I> study. 
</LI><LI> The synthetic tuning construct (see sidebar, fig. 3) consisted of two viruses injected at the same time in the mice. The one virus packaged the expression construct of shRNA haat driven by the H1 promoter (&quot;tuning&quot; construct, see sidebar, fig.3). The second virus packaged the following transgene: SV40 promoter driving luc2 with shRNA haat binding site behind it (&quot;tuned&quot; construct, see sidebar, fig. 4). In order to ensure a synthetic tuning effect, a perfect binding site and one with a bulge that was introduced at position 9-12 were used for <I>in vivo</I> experiments, respectively. Those two binding sites should lead to a significant knockdown in the first case and a slight repression of luciferase expression in the latter as compared to the positive control.
</LI><LI> The on-targeting construct consisted of two independent viruses which were co-infected into mice, as well. One of these viruses packaged the Tet Repressor (TetR) driven by an SV40 promoter (&quot;repressor&quot; construct, see sidebar, fig. 5). The expression of TetR is under the control of miR-122 as four binding sites of this miRNA were cloned into the 3’UTR of the gene. The second virus was composed of an SV40 promoter driving the Tet operator (TetO<SUB>2</SUB>) which monitors the expression of luc2 (&quot;operator&quot; construct, see sidebar, fig. 6). With this setup, luc2 expression should be inhibited by the TetR in all mice tissues except for liver cells, where TetR is down-regulated by miRNA 122.
</LI></OL><H3><SPAN class="mw-headline" id="Production_of_recombinant_virus">Production of recombinant virus</SPAN></H3><P>The viruses were produced in HEK 293-T cells and purified on an iodixanol gradient according to <A href="https://2010.igem.org/Team:Heidelberg/Notebook/Methods#Virus_Production" class="external text" rel="nofollow">the virus production protocol</A>. 
</P><P>Before infection, the titer of the viruses was quantified using <A href="https://2010.igem.org/Team:Heidelberg/Notebook/Methods#Quantitative_Realtime_PCR" class="external text" rel="nofollow">quantitative realtime PCR</A>.
</P><H3><SPAN class="mw-headline" id="Procedure_involving_animals">Procedure involving animals</SPAN></H3><P>The mouse experiments were conducted in accordance with the animal facility of the <A href="https://2010.igem.org/Team:Heidelberg/Team/Institutes" class="external text" rel="nofollow">German Cancer Research Center in Heidelberg</A>. Female NMRI mice were obtained from a collaboration with Dr. Oliver Müller. At 8-10 weeks of age, the animals were injected in the tail vein (TV), with ~ 1x10<SUP>11</SUP> particles of AAV-SV40-luciferase in 200µl of 1x phosphate-buffered saline. The mice are transferred to a holding device which restrains the mouse while allowing access to the tail vein. The tails were warmed before the injections and injections were carried out using 27 gauge needles. All the mice recoverd from the injection quickly without loss of mobility or interruption of grooming activity <SPAN class="referencelink"><A href="http://2010.igem.org/Team:Heidelberg/Project/Mouse_Infection#References" class="external text" rel="nofollow">(Zincarelli et al., 2008)</A></SPAN>.
</P><H3><SPAN class="mw-headline" id="in_vivo_animal_imaging">in vivo animal imaging</SPAN></H3><P>Mice were anesthesized in an isofluran chamber. The mice were injected intraperitoneally with 200µl of a 30 mg/ml concentration of D-luciferin. This injection starts the luminescence of luc2. Mice were measured for one to seven minutes post injection under the in vivo bioluminometer.
</P><H2><SPAN class="mw-headline" id="References">References</SPAN></H2><P>Grimm, D. (2002). &quot;Production methods for gene transfer vectors based on adeno-associated virus serotypes.&quot; Methods 28(2): 146-157.
Grimm, D., J. S. Lee, et al. (2008). &quot;In vitro and in vivo gene therapy vector evolution via multispecies interbreeding and retargeting of adeno-associated viruses.&quot; J Virol 82(12): 5887-5911.
Zincarelli, C., S. Soltys, et al. (2008). &quot;Analysis of AAV serotypes 1-9 mediated gene expression and tropism in mice after systemic injection.&quot; Mol Ther 16(6): 1073-1080.</P></DIV><DIV class="printfooter">
Retrieved from &quot;<A href="http://2010.igem.org/Team:Heidelberg/Project/Mouse_Infection">http://2010.igem.org/Team:Heidelberg/Project/Mouse_Infection</A>&quot;</DIV></DIV></DIV><DIV id="footer-box" class="noprint"><DIV id="footer"><UL id="f-list"><LI id="t-recentchanges"><A href="/Special:RecentChanges" title="Recent changes">Recent changes</A></LI><LI id="t-whatlinkshere"><A href="/Special:WhatLinksHere/Team:Heidelberg/Project/Mouse_Infection" title="List of all wiki pages that link here [j]" accesskey="j">What links here</A></LI><LI id="t-recentchangeslinked"><A href="/Special:RecentChangesLinked/Team:Heidelberg/Project/Mouse_Infection" title="Recent changes in pages linked from this page [k]" accesskey="k">Related changes</A></LI><LI id="t-specialpages"><A href="/Special:SpecialPages" title="List of all special pages [q]" accesskey="q">Special pages</A></LI><LI><A href="/Special:Preferences">My preferences</A></LI></UL></DIV><DIV id="footer"><UL id="f-list"><LI id="t-print"><A href="/wiki/index.php?title=Team:Heidelberg/Project/Mouse_Infection&amp;printable=yes" title="Printable version of this page [p]" accesskey="p">Printable version</A></LI><LI id="t-permalink"><A href="/wiki/index.php?title=Team:Heidelberg/Project/Mouse_Infection&amp;oldid=208467" title="Permanent link to this revision of the page">Permanent link</A></LI><LI id="privacy"><A href="/2010.igem.org:Privacy_policy" title="2010.igem.org:Privacy policy">Privacy policy</A></LI><LI id="disclaimer"><A href="/2010.igem.org:General_disclaimer" title="2010.igem.org:General disclaimer">Disclaimers</A></LI></UL></DIV></DIV></BODY></HTML>