"
Page
Discussion
View source
History
teams
Log in
 
Team:Heidelberg/Project/Capsid Shuffling/ViroBytes
From 2010.igem.org
Home
Team
Overview
Students
Instructors
Advisors
Institutes
Gallery
miDANCE
Acknowledgements
 
Project
Abstract
Introduction
Synthetic miRNA Kit
Capsid Shuffling
    Homology Based
    ViroBytes
miMeasure
in vivo Study
Summary & Outlook
References
Contributions
 
Parts
Parts
Characterization
 
Modeling
Introduction
miBEAT
miRockdown
miBS designer
mUTING
Descriptions
 
Notebook
Synthetic miRNA Kit
Capsid Shuffling
    Homology Based
    ViroBytes
miMeasure
Binding Site Design
in vivo Study
Modeling
Materials
Methods
Psychological Survey
Philosophical Reflection
Biosafety
 
Human Practices
Psychological Survey
Philosophical Reflection
 
Sponsors
ViroBytes
Introduction
ViroBytes is a modified BioBytes procedure for "rational shuffling" of capsid genes from natural isolates of Adeno-Associated Virus. Main motivation for the new protocol is unsatisfactory incorporation of certain serotypes (and especially certain parts of AAV capsid regions eg. from AAV5) using conventional shuffling strategies ( Grimm et al., 2008 ). Application of magnetic beads for controlled assembly and the principle of BioByte formation and annealing persists. Sticky overhangs are used for selective combination of the bytes but different method is used for the production of individual ViroBytes.
AAV serotypes 1,2,5,6,8 and 9 were selected as suitable candidates for fragmentation due to their exceptional individual properties. The analysis of Cap gene sequences revealed multiple homology regions which were then used for rational fragment formation. Total number of fragments per Cap gene is eight in our case and all fragments have similar length around ~250bp to assure similar behaviour in the ligation reaction. The Bytes are created and amplified by High-Fidelity PCR using Cap- and FragmentX-specific primers. These primers contain flanking regions with recognition sequence for type II restriction enzyme Bsa1. Precise positioning in front of and behind the homology regions at the ends of each fragmetn ensures that the enzyme forms unique sticky ends with 4nt overhangs. This procedure allows the user to avoid arduous design of overhangs with incorporated uracils further used for USERTM digestion. Also for the purposes of shuffling, the homology regions need to be exploited in order to avoid frame shifts which cannot be easily accomplished using standard BioByte protocol.
ViroBytes Assembly Scheme
Important facts
Shuffling of several (six in our case) wild-type AAV serotypes unfortunately does not allow for standardization of the Virobyte fragments. We use type II restriction enzyme Bsa1 for creating the sticky ends instead. Incorporation of the recognition sequence of Bsa1 at both ends of the ViroBytes allows for higher specificity and thus prevents non-neighbouring fragments from ligating. The homology regions within which the Bsa1 enzyme cuts provide possibility for fragment shuffling without creating sequence frameshifts or deletions leading to nonviable chimeric isolates.
BsaI recognition and cutting sequence:
5'...GGTCTCN/NNNN...3'   3'...CCAGAGNNNNN/...5'
hence the overhang created at each fragment ending: 5'-NNNN
Homology regions employed in our Virobyte approach were selected as follows:
between Anchor and Fragment1 ("Start overhang"):
5'...ATGG...3'   3'...TACC...5'
between Fragment1 and Fragment2 (bp 279-303 relative to AAV1 Cap):
5'...GGAG...3'   3'...CCTC...5'
between Fragment2 and Fragment3 (bp 688-716 ,AAV1):
5'...GACA...3'   3'...CTGT...5'
between Fragment3 and Fragment4 (bp 1013-1016 ,AAV1):
5'...AACC...3'   3'...TTGG...5'
between Fragment4 and Fragment5 (bp 1282-1285 ,AAV1):
5'...CAGC...3'   3'...GTCG...5'
between Fragment5 and Fragment6 (bp 1444-1447 ,AAV1):
5'...AACT...3'   3'...TTGA...5'
between Fragment6 and Fragment7 (bp 1740-1743 ,AAV1):
5'...GTGG...3'   3'...CACC...5'
between Fragment7 and Fragment8 (bp 2040-2043 ,AAV1):
5'...TCAC...3'   3'...AGTG...5'
ViroByte design
Anchor
Anchor for ViroByte assembly was designed according to BioByte Anchor System . We used 5'-biotinylated oligos which were annealed with Anchor_complementary sequences (see Protocols ). The final Anchor construct contains important restriction sites as well as the VirByte_for sequence used for final PCR amplification. First recognition site is HindIII which we used for separating the beads from the assembled DNA before PCR. Second restriction sequence corresponds to PacI which is a common non-cutter within all AAV serotypes used and can be emplyed for further ligation into an appropriate vector. Most importantly the 3'-end forms a sticky end compatible with the so-called "start overhang" of all fragments1.
Annotated sequence of anchor oligos:
Anchor:
5'-Biotin-tctctctctctctctaagctt gtctgagtgactagcattcg ttaattaac+4nt(cap specific)
first 15nt form ssDNA linker (improves efficiency of Anchor binding onto Steptavidin-coated magnetic beads)
italicized is HindIII recognition site
bold is amplification (VirByte_for) sequence
bold italicized is PacI recognition site
Anchor complement:
5'-gttaattaa cgaatgctagtcactcagac aagctt
reverse complimentary strand to Anchors
Final Anchor construct used:
5'-Biotin-tctctctctctctctaagcttgtctgagtgactagcattcgttaattaac+4nt-3'                          3'-ttcgaacagactcactgatcgtaagcaattaattg-5'
Primers
Fragment X forward primers :
5'-gctacggtctcaNNNN+~20-25nt Cap specific
bold is BsaI recognition sequence
NNNN = cap specific homology sequence from all cap genes (also cutting sequence of BsaI) forming the 5'-overhang
Fragment X reverse primers:
5'-gctacggtctcgNNNN
bold is BsaI recognition sequence
Protocols
ViroByte construction
perform ViroByte Phusion HiFi Hot Start PCR to obtain desired fragments with complementary ends
98°C 30s     1x
98°C 10s
72°C 45s     35x
72°C 10min   1x
4°C hold
digestion with Bsa1 - leave digesting longer than in standard protocol (at least ~3hrs)
Anchor preparation
Mix 10ul of Anchor 5 (100uM stock) with 50ul of Anchor 12689 (100ul stock); add 1.08ml of nuclease free water; mix gently
heat on the block to 95degrees for 2 minutes and let cool slowly to room temperature
Anchor docking
after washing the beads and aspiration of the cleared solution the beads are ready for adding of the Anchor
add 40ul of the Anchor solution (~200ng/ul)
resuspend the beads and let incubate at room temperature for 15-20 minutes with occasional flicking in order to maintain the homogeneity of the solution
applying magnet, wash twice with 80ul of wash/binding buffer (0.5 M NaCl; 20 mM Tris HCl (pH 7.5); 1 mM EDTA)  and once with 80ul of 1X T4 ligase buffer
BioByte-like assembly
Bead preparation
Quickly vortex beads stock solution to obtain homogeneous suspension
Aliquot 40 µl of the beads into 1.5ml Eppendorf tubes - gently tap the tubes to collect all the beads at the bottom.
Place the tubes on the magnetic rack (or place a magnet on the side of each tube) and wait for the beads to be cleared out from the solution
Carefully discard the supernatant whilst beads are still attached to the side wall
Resuspend beads in 80ul of wash/binding buffer by gentle flicking (vortexing at high speed is not recommended)
Wash the beads once with 80 µl of 1X T4 Ligase buffer, before addition of ligation premix.
Add the DNA fragments using the ligation premix: 1) 25 µl of Quick Ligase (2X buffer); 2) Amount of DNA corresponding to 200 ng. The optimal DNA concentration for the ViroByte fragments assembly was experimentally established and was in range of 200 ng per fragment. Using higher and lower DNA concentrations did not give any positive assembly results; 3) Fill up with water to obtain the 50 µl total volume.
For the ligation procedure the Quick Ligase was used that showed to be more efficient during the experimental procedure in comparison to the ordinary T4 Ligase.  Optimal ligation time for the Virobytes ligation was between 15-25 minutes.
After the last DNA fragment ligation wash the beads once with Wash/Binding buffer.  Moreover the DNA should be cleaved from the beads by using HindIII restriction enzyme (or the other one depending on your designed cleavage site). For the restriction a standard protocol for the DNA digestion should be used.
Notebook
For further reference please find our lab notebook here .
References
Grimm D, Lee JS, Wang L, Desai T, Akache B, Storm TA, Kay MA. In Vitro and In Vivo Gene Therapy Vector Evolution via Multispecies Interbreeding and Retargeting of Adeno-Associated Viruses. J Virol 2008;82:5887–5911.
Contents
1 ViroBytes
1.1 Introduction
1.2 Important facts
1.3 ViroByte design
1.3.1 Anchor
1.3.2 Primers
1.4 Protocols
1.4.1 ViroByte construction
1.4.2 Anchor preparation
1.4.3 Anchor docking
1.4.4 BioByte-like assembly
1.5 Notebook
1.6 References
   
Retrieved from " http://2010.igem.org/Team:Heidelberg/Project/Capsid_Shuffling/ViroBytes "
Recent changes
What links here
Related changes
Special pages
My preferences
Printable version
Permanent link
Privacy policy
Disclaimers
