<HTML xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en" dir="ltr">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki  ltr ns-0 ns-subject page-Team_Heidelberg_Project_miMeasure"><DIV id="globalWrapper"><DIV id="top-section"><DIV id="p-logo"><A href="/Main_Page" title="Main Page">&quot;
	    </A></DIV><DIV id="menubar" class="left-menu noprint"><UL><LI class="selected"><A href="/Team:Heidelberg/Project/miMeasure">Page               </A></LI><LI class="new"><A href="/wiki/index.php?title=Talk:Team:Heidelberg/Project/miMeasure&amp;action=edit&amp;redlink=1">Discussion               </A></LI><LI><A href="/wiki/index.php?title=Team:Heidelberg/Project/miMeasure&amp;action=edit">View source               </A></LI><LI><A href="/wiki/index.php?title=Team:Heidelberg/Project/miMeasure&amp;action=history">History               </A></LI><LI style="color:#808080;cursor:default">teams</LI></UL></DIV><DIV class="right-menu noprint" id="menubar"><UL><LI id="pt-login"><A href="/wiki/index.php?title=Special:UserLogin&amp;returnto=Team:Heidelberg/Project/miMeasure" title="You are encouraged to log in; however, it is not mandatory [o]" accesskey="o">Log in</A></LI></UL></DIV><DIV id="search-controls" class="noprint"><FORM action="/Special:Search" id="searchform"> </FORM></DIV></DIV><DIV id="content"><H1 class="firstHeading">Team:Heidelberg/Project/miMeasure</H1><DIV id="bodyContent"><H3 id="siteSub" class="noprint">From 2010.igem.org</H3><DIV id="header"><DIV id="menu"><UL id="nav"><LI id="menu_home"><A class="aHome" href="https://2010.igem.org/Team:Heidelberg">Home</A></LI><LI id="menu_team"><A class="aTeam" href="https://2010.igem.org/Team:Heidelberg/Team">Team</A><TABLE class="menutable" cellpadding="0" cellspacing="0"><TBODY><TR><TD class="lefttdtopteam"><A class="aTeamOverview" href="https://2010.igem.org/Team:Heidelberg/Team">Overview</A><A class="aTeamMembers" href="https://2010.igem.org/Team:Heidelberg/Team/Members">Students</A><A class="aTeamMembers" href="https://2010.igem.org/Team:Heidelberg/Team/Members#Instructors">Instructors</A><A class="aTeamMembers" href="https://2010.igem.org/Team:Heidelberg/Team/Members#Advisors">Advisors</A><A class="aInstitutes" href="https://2010.igem.org/Team:Heidelberg/Team/Institutes">Institutes</A><A class="aGallery" href="https://2010.igem.org/Team:Heidelberg/Team/Gallery">Gallery</A><A class="aDance" href="https://2010.igem.org/Team:Heidelberg/midance">miDANCE</A><A class="amithanks" href="https://2010.igem.org/Team:Heidelberg/Team/miThanks">Acknowledgements</A></TD><TD class="righttdtop"> </TD></TR></TBODY></TABLE></LI><LI id="menu_project"><A class="aProject" href="https://2010.igem.org/Team:Heidelberg/Project">Project</A><TABLE class="menutable" cellpadding="0" cellspacing="0"><TBODY><TR><TD class="lefttdtopproject"><A class="aAbstract" href="https://2010.igem.org/Team:Heidelberg/Project">Abstract</A><A class="aIntroduction" href="https://2010.igem.org/Team:Heidelberg/Project/Introduction">Introduction</A><A class="aProject_miRNA_Kit" href="https://2010.igem.org/Team:Heidelberg/Project/miRNA_Kit">Synthetic miRNA Kit</A><A class="aPro_Capsid_Shuffling" href="https://2010.igem.org/Team:Heidelberg/Project/Capsid_Shuffling">Capsid Shuffling</A><A class="aPro_Homology" href="https://2010.igem.org/Team:Heidelberg/Project/Capsid_Shuffling/Homology_Based">    Homology Based</A><A class="aPro_Virobytes" href="https://2010.igem.org/Team:Heidelberg/Project/Capsid_Shuffling/ViroBytes">    ViroBytes</A><A class="aPro_miMeasure" href="https://2010.igem.org/Team:Heidelberg/Project/miMeasure">miMeasure</A><A class="aPro_Mouse" href="https://2010.igem.org/Team:Heidelberg/Project/Mouse_Infection"><I>in vivo</I> Study</A><A class="aPro_Summary" href="https://2010.igem.org/Team:Heidelberg/Project/Summary">Summary &amp; Outlook</A><A class="aPro_Ref" href="https://2010.igem.org/Team:Heidelberg/Project/References">References</A><A class="aPro_Ref" href="https://2010.igem.org/Team:Heidelberg/Project/Contributions">Contributions</A></TD><TD class="righttdtop"> </TD></TR></TBODY></TABLE></LI><LI id="menu_parts"><A class="aParts" href="https://2010.igem.org/Team:Heidelberg/Parts">Parts</A><TABLE class="menutable" cellpadding="0" cellspacing="0"><TBODY><TR><TD class="lefttdtop"><A class="aparts_ger" href="https://2010.igem.org/Team:Heidelberg/Parts">Parts</A><A class="aparts_char" href="https://2010.igem.org/Team:Heidelberg/Parts/Characterization">Characterization</A></TD><TD class="righttdtop"> </TD></TR></TBODY></TABLE></LI><LI id="menu_modeling"><A class="aModeling" href="https://2010.igem.org/Team:Heidelberg/Modeling">Modeling</A><TABLE class="menutable" cellpadding="0" cellspacing="0"><TBODY><TR><TD class="lefttdtop"><A class="amode_over" href="https://2010.igem.org/Team:Heidelberg/Modeling">Introduction</A><A class="amiGUI" href="https://2010.igem.org/Team:Heidelberg/Modeling/miGUI">miBEAT</A><A class="amirockdown" href="https://2010.igem.org/Team:Heidelberg/Modeling/miRockdown">miRockdown</A><A class="amibsdesigner" href="https://2010.igem.org/Team:Heidelberg/Modeling/miBSdesigner">miBS designer</A><A class="amispec" href="https://2010.igem.org/Team:Heidelberg/Modeling/miSpec">mUTING</A><A class="amodelset" href="https://2010.igem.org/Team:Heidelberg/Modeling/trainingset">Descriptions</A></TD><TD class="righttdtop"> </TD></TR></TBODY></TABLE></LI><LI id="menu_notebook"><A class="aNotebook" href="https://2010.igem.org/Team:Heidelberg/Notebook">Notebook</A><TABLE class="menutable" cellpadding="0" cellspacing="0"><TBODY><TR><TD class="lefttdtop"><A class="aNote_miRNA_kit" href="https://2010.igem.org/Team:Heidelberg/Notebook/miRNA_Kit">Synthetic miRNA Kit</A><A class="aNote_Capsid_Shuffling">Capsid Shuffling</A><A class="aNote_Homology" href="https://2010.igem.org/Team:Heidelberg/Notebook/Capsid_Shuffling/Homology_Based">    Homology Based</A><A class="aNote_Virobytes" href="https://2010.igem.org/Team:Heidelberg/Notebook/Capsid_Shuffling/ViroBytes">    ViroBytes</A><A class="aNote_miMeasure" href="https://2010.igem.org/Team:Heidelberg/Notebook/miMeasure">miMeasure</A><A class="aNote_BSdesign" href="https://2010.igem.org/Team:Heidelberg/Notebook/BSDesign">Binding Site Design </A><A class="aNote_Mouse" href="https://2010.igem.org/Team:Heidelberg/Notebook/Mouse_Infection"><I>in vivo</I> Study</A><A class="aNote_Modeling" href="https://2010.igem.org/Team:Heidelberg/Notebook/Modeling">Modeling</A><A class="aNote_Material" href="https://2010.igem.org/Team:Heidelberg/Notebook/Material">Materials</A><A class="aNote_Methods" href="https://2010.igem.org/Team:Heidelberg/Notebook/Methods">Methods</A><A class="aNote_Psycho" href="https://2010.igem.org/Team:Heidelberg/Human_Practices/Psychology/Notebook">Psychological Survey</A><A class="aNote_Philo" href="https://2010.igem.org/Team:Heidelberg/Human_Practices/Philosophy/Part2">Philosophical Reflection</A><A class="aNote_Biosafety" href="https://2010.igem.org/Team:Heidelberg/Notebook/Biosafety">Biosafety</A></TD><TD class="righttdtop"> </TD></TR></TBODY></TABLE></LI><LI id="menu_human_practices"><A class="aHuman_Practices" href="https://2010.igem.org/Team:Heidelberg/Human Practices">Human Practices</A><TABLE class="menutable" cellpadding="0" cellspacing="0"><TBODY><TR><TD class="lefttdtop"><A class="aPsycho" href="https://2010.igem.org/Team:Heidelberg/Human Practices/Psychology">Psychological Survey</A><A class="aPhilo" href="https://2010.igem.org/Team:Heidelberg/Human Practices/Philosophy">Philosophical Reflection</A></TD><TD class="righttdtop"> </TD></TR></TBODY></TABLE></LI><LI id="menu_support"><A class="aSupport" href="https://2010.igem.org/Team:Heidelberg/Support">Sponsors</A></LI></UL></DIV></DIV><DIV style="width:340px; padding-top:41px; padding-left:12px; float:right; margin-right:-73px;"><TABLE border="0" cellpadding="0" cellspacing="0" style="width:318px; margin:0px; table-layout:fixed; border-style:none; border-spacing:0; vertical-align:top; background-color:transparent;"><TBODY><TR style="background:url(https://static.igem.org/mediawiki/2010/7/71/Side_frame_t.png) no-repeat top;"><TD style="padding:30px 25px 0 25px;"><P><B>With miMeasure you can:</B></P><UL><LI>Characterize synthetic/ natural binding sites
</LI><LI>Regulate EGFP with customed BS
</LI><LI>Use EBFP2 as reference
</LI></UL><UL><LI>Clone BS behind the EGFP with the BBb standard 
</LI></UL><UL><LI>Characterize miRNA expression in cell populations or at single cell level by measuring differences in the relative EGFP intensities with standard methods like FACS or microscopy
</LI></UL></TD></TR><TR style="height:40px; background:url(https://static.igem.org/mediawiki/2010/f/fe/Side_frame_b.png) no-repeat top;"><TD> </TD></TR></TBODY></TABLE></DIV><H1><SPAN class="mw-headline" id="miMeasure">miMeasure</SPAN></H1><H3><SPAN class="mw-headline" id="The_miMeasure_standard_plasmid_has_been_engineered_to_enable_the_easy_input_of_synthetic_microRNA_binding_sites_behind_one_of_two_fluorescent_proteins_while_the_second_is_used_for_normalization._Expression_of_regulated_reporter_and_control_from_a_bidirectional_CMV_promter_guarantee_faithful_and_reproducible_measurements_in_any_kind_of_cell._The_fluorescence_readout_can_be_used_to_quantify_the_regulatory_efficiency_of_the_binding_site_in_knockdown_percentage._Once_the_properties_of_a_synthetic_binding_site_are_elucidated.2C_they_can_be_used_to_manipulate_and_accurately_fine-tune_gene_expression_in_vitro_and_in_vivo."> The <B>miMeasure</B> standard plasmid has been engineered to enable the easy input of synthetic microRNA binding sites behind one of two fluorescent proteins while the second is used for normalization. Expression of regulated reporter and control from a bidirectional CMV promter guarantee faithful and reproducible measurements in any kind of cell. The fluorescence readout can be used to quantify the regulatory efficiency of the binding site in knockdown percentage. Once the properties of a synthetic binding site are elucidated, they can be used to manipulate and accurately fine-tune gene expression in vitro and in vivo. </SPAN></H3><H2><SPAN class="mw-headline" id="Introduction">Introduction</SPAN></H2><P>Micro RNAs mainly regulate the translation of their target genes by interacting with regions in the 3' untranslated region (UTR) of their target mRNA. Base-pairing with the miRNA binding site (BS) causes formation of diverse miRNA-mRNA duplexes <SPAN class="referencelink"><A href="http://2010.igem.org/Team:Heidelberg/Project/miMeasure#References" class="external text" rel="nofollow">(reviewed by Fabian et al., 2010)</A></SPAN>. The BS consists of a seven nucleotide long seed region that is perfectly matched to the miRNA, and surrounding regions that matched partially. The seed region is defined as being the minimal required base-pairing that can regulate the mRNA. Apart from the seed region, binding can be unspecific, creating mismatches and bulges. The position and properties of the bulges seem to play a central role in miRNA binding and therefore knockdown efficiency <SPAN class="referencelink"><A href="http://2010.igem.org/Team:Heidelberg/Project/miMeasure#References" class="external text" rel="nofollow">(reviewed by Bartel et al., 2009)</A></SPAN>. Since we were going to use synthetic miRNA BS in our genetherapeutic approach, we had to find a way to study their effects in a standardized manner that would be comparable and reproducible. </P><P>One goal of the iGEM Team Heidelberg 2010 was to test the effects of changes in BS sequences on expression control. Thereby miRNA BS should be characterized. To standardize our measurements of knockdown according to BS specificity, we had to come up with a new standard that is independent from the endogenous cell machinery. We decided to bring synthetic miRNAs into play, hence we engineered BS for them creating an artificial regulatory circuit. Of course there are also differences that arise through the availability of the enzymes involved in the miRNA pathway that may differ slightly from cell to cell. Therefore, constructs containing changes in BS sequences has to be compared to construct containing no binding sites and containing perfect binding sites. Ideally, the miRNA would be stably expressed in the cell line, but a uniform co-transfection also leads to an even distribution of synthetic shRNA-like miRNAs (shRNA miRs). Additionally, miRNA levels can be adjusted by differing transfection ratios. </P><P>The main purpose of our measurement standard, miMeasure, is to express two nearly identical but discernible proteins: one of them tagged with a BS, the other one unregulated (even though the possibility exists to clone in a reference binding site). The two reporters are expressed by a bidirectional CMV promoter to make sure their transcription rate is comparable. We used a destabilized version of GFP, dsEGFP and a dsEBFP2 that was derived from the same sequence (<A href="https://2010.igem.org/Team:Heidelberg/Project/miMeasure#References" class="external text" rel="nofollow">Ai et al., 2007</A>). Thus, we could make sure that both proteins exhibit the same synthesis and degradation properties, making them directly comparable. Hereby, we can also neglect the difference between mRNA and protein knockdown and can take the fluorescence of the marker protein as a direct, linear output of mRNA down-regulation. We included a BBb standard site into our plasmid, which allows to clone BS behind the GFP. If co-transfected with the corresponding shRNA miR, GFP will be down-regulated, while BFP expression is maintained. The ratio of GFP to BFP expression can be used to conclude the knockdown efficiency of the BS. Having destabilized marker proteins with a turnover time of two hours enables us not only to avoid accumulation of marker proteins, which would make the knockdown harder to observe, but also to conduct time-lapse experiments. In the future, this could be for example a way to observe dynamic activity patterns of endogenous miRNAs.

</P><DIV class="backtop"><A href="#top">↑</A></DIV><H2><SPAN class="mw-headline" id="Results">Results</SPAN></H2><H3><SPAN class="mw-headline" id="Analysis_of_Randomized_Binding_Sites_Against_Synthetic_miRNA">Analysis of Randomized Binding Sites Against Synthetic miRNA</SPAN></H3><P>We modified binding sites for the synthetic miRNA by introducing random basepairs surrounding the seed region as described above, thereby changing the knockdown efficiency. In figure 1 we plotted the knockdown percentage of our constructs. The miMeasure construct and negative control were <A href="https://2010.igem.org/Team:Heidelberg/Notebook/Methods#Transfection" class="external text" rel="nofollow">co-transfected</A> with either shRNA miRsAg expressed from a CMV promoter on pcDNA5 or an inert synthetic RNA as control in 1:4 ratio, respectively.
</P><TABLE class="wikitable sortable" border="0" align="center" style="text-align: left"><CAPTION align="top, left"><B>Table 1: Used Binding Sites and Their Features</B></CAPTION><TBODY><TR bgcolor="#009be1"><TD>sequence</TD><TD>binding site feature</TD><TD>Name
</TD></TR><TR><TD>ctcagtttactagtgccatttgttc</TD><TD>perfect binding site against miRsAg</TD><TD>perfect BS
</TD></TR><TR><TD>ctcagtttactagacgcatttgttc</TD><TD>miMeasure with randomised nucleotides 10-12</TD><TD> 10-12 ACG
</TD></TR><TR><TD>ctcagtttactagtaacatttgttc</TD><TD>miMeasure with randomised nucleotides 11-12</TD><TD>11-12 AA
</TD></TR><TR><TD>ctcagtttactagacggatttgttc</TD><TD>miMeasure with randomised nucleotides 9-12</TD><TD>9-12 ACGG
</TD></TR><TR><TD>ctcagtttactagatgtatttgttc</TD><TD>miMeasure with randomised nucleotides 9-12</TD><TD>9-12 ATGT
</TD></TR><TR><TD>ctcagtttactagtggcatttgttc</TD><TD>miMeasure with mutated nucleotide 10</TD><TD>10 G
</TD></TR><TR><TD>ctcagtttactagtgacatttgttc</TD><TD>miMeasure with mutated nucleotide 10</TD><TD>10 A
</TD></TR><TR><TD>ctcagtttactagtaccatttgttc</TD><TD>miMeasure with mutated nucleotide 11</TD><TD>11 A
</TD></TR><TR><TD>ctcagttatgtagtgccatttgttc</TD><TD>miMeasure with mutated nucleotide 16-18</TD><TD>16-18 ATG
</TD></TR></TBODY></TABLE><H4><SPAN class="mw-headline" id="Flow_cytometry_measurements">Flow cytometry measurements</SPAN></H4><P>Hela cells transfected with the constructs (described above) are taken for flow cytometry. Each measurement contains around 10000 cells. The cells are plotted on a logarithmic scale in relation to EGFP and EBFP2 intensity. Each dot intensity represents the count of fluorescent cells for each EGFP/EBFP2 intensity pair. The dots are colour coded, so that the orange dots represent cells cotransfected with different miMeasure constructs and the miRsAg and the blue one represent cells cotransfected with non-specific miRNA (miR-155), respectively. So the blue set of measurements represent the negative control. Before the real measurements, cells transfected with EGFP or EBFP2 alone were measured to establish the gain of the detectors and compensate for fluorescent bleedthrough, especially of EBFP2 into the EGFP channel . This ensured that EGFP and EBFP2 alone shows a perfectly horizontal and vertical distribution respectively (data not shown). For the miMeasure constructs, both population of dots make up a oblique line on the logarithmic scale, which shows the correlation of EGFP and EBFP2 very well. If the two different coloured dots overlap, they become white. Thus both lines overlap almost completely in the negative control containing no binding site, whereas the orange line shifts to the left for the miMeasure construct with the perfect binding site. All the other constructs are like the negative control.
As the difference is subtle on the logarithmic scale, we also observed the cell distribution on a linear scale. All coloured distributions appeared more scattered on the linear scale, but the shifting of the orange dots was more visible for the construct containing the perfect binding site. again all the other constructs have the same range of scattering as the negative control. 
</P><DIV class="center"><DIV class="thumb tnone"><DIV class="thumbinner" style="width:612px;"><DIV class="thumbcaption"><B>Figure 1: GFP/BFP correlation of single transfected Hela cells according to flow cytometry analysis on a logarithmic scale</B>  different binding sites for miRsAg cotransfected with miRsAg or with mi-R155, respectively.  The orange dots represent the cotransfected cells with miRsAg and the blue dots the cotransfected cells with miR-155. Hela cells were used.</DIV></DIV></DIV></DIV>^
<DIV class="center"><DIV class="thumb tnone"><DIV class="thumbinner" style="width:612px;"><DIV class="thumbcaption"><B>Figure 2: GFP/BFP correlation of single transfected Hela cells according to flow cytometry analysis on a linear scale</B>  different binding sites for miRsAg cotransfected with miRsAg or with mi-R155, respectively.  The orange dots represent the cotransfected cells with miRsAg and the blue dots the cotransfected cells with miR-155. Hela cells were used.</DIV></DIV></DIV></DIV><P>Flow cytometry offers an efficient way to estimate the efficiency of expression regulation. Here we could observe that while the perfect binding site generate a partial knockdown, all modified binding site constructs looked similar to the negative control. To characterize potential differences that cannot be identified by flow cytometry, we next imaged the same cells and precisely quantified their fluorescence intensity using laser scanning confocal microscopy at high resolution.
</P><H4><SPAN class="mw-headline" id="Confocal_microscopy_measurements">Confocal microscopy measurements</SPAN></H4><P>We used microscopy analysis to determine the EGFP expression in relation to EBFP2. EBFP2 serves as a normalization for transfection efficiency. Nine miMeasure constructs with different binding sites were designed. Those were cloned downstream of EGFP behind the miMeasure construct, whereas the EBFP2 expression stays unaffected. The GFP/BFP-ratio stand for the level of GFP-expression normalized to one copy per cell.
</P><DIV class="center"><DIV class="thumb tnone"><DIV class="thumbinner" style="width:502px;"><DIV class="thumbcaption"><B>Figure 3: GFP/BFP ratio normalized by the negative control</B> The data are generated by confocal microscopy of Hela cells, which were transfected with different miMeasure constructs M12-M22 including the negative control (miMeasure construct without binding site). The negative control equals to 1.</DIV></DIV></DIV></DIV><P>
Comparing the GFP/BFP-ratio between the constructs, we can see a significant difference of GFP expression between the negative control and the construct containing the perfect binding site. Since the control is not downregulated due to lack of binding sites, we set it as 100% expression on this chart. It can be seen that the perfect binding sites causes the lowest GFP expression, approximately 50%, while other binding sites range in between 55% and 100% of expression. 
</P><DIV class="backtop"><A href="#top">↑</A></DIV><H3><SPAN class="mw-headline" id="Analysis_of_miRaPCR_Generated_Binding_Sites_Against_a_Natural_miRNA">Analysis of miRaPCR Generated Binding Sites Against a Natural miRNA</SPAN></H3><P>The <A href="https://2010.igem.org/Team:Heidelberg/Notebook/BSDesign/July" class="external text" rel="nofollow">miRaPCR</A> generates binding sites out of rationally designed fragments. These are aligned with each other by chance, whereby different  spacer regions are inserted randomly in between. It has been suggested that having more than one binding site of for the same miRNA behind each other can lead to stronger down-regulation than a single one. If imperfect binding sites are aligned, it is also supposed to be stronger than a single one. This is what we tested using MiRaPCR for effortless assembly of binding site fragments.
For our experiments, we took advantage of the high abundance of miRNA 122 in liver cells and tested different combinations of binding sites created by miRaPCR. We transfected HeLa and HuH7 cells with the constructs described in table 2. Since HuH7 cells express miR-122, the constructs will be affected in the HuH7 cells without cotransfecting any miRNAs, whereas miR-122 were cotransfected for the HeLa cells in 2:1 ratio. The result shows the ratio between GFP and BFP normalized to the negative control (miMeasure constructs without binding sites). The miMeasure constructs were also compared to the expression of miMeasure containing one perfect binding site for miRNA 122.
</P><TABLE class="wikitable sortable" border="0" align="center" style="text-align: left"><CAPTION align="top, right"><B>Table2: miRaPCR Designed Binding Sites and Their Features</B></CAPTION><TBODY><TR bgcolor="#009be1"><TD>binding site feature</TD><TD>Name/number</TD></TR><TR><TD>miMeasure with 3 aligned perfect binding sites</TD><TD>miM-1.3-7
</TD></TR><TR><TD>miMeasure with two imperfect binding sites</TD><TD>miM-3.1-8
</TD></TR><TR><TD>miMeasure with randomised nucleotides 9-12</TD><TD>miM-r12
</TD></TR><TR><TD>miMeasure with randomised nucleotides 9-22</TD><TD>miM-r22
</TD></TR></TBODY></TABLE><H4><SPAN class="mw-headline" id="Flow_cytometry">Flow cytometry</SPAN></H4><P>The same constructs in Hela cells were analyzed by flow cytometry, too. Here the orange dots also represent the miMeasure construct transfected with the specific miRNA and the blue dots make up the negative control. The orange dots from the construct containing the perfect and the aligned binding sites have lower EGFP expression compared to EBFP2 expression, since the EGFP trend doesn't correspond to the EBFP2 trend, but shifts to the laft. The orange dots rise with the fluorescence intensity and collapses to zero, when the EBFP2 intensity is high. For the other constructs the EGFP range fully corresponds with the EBFP2 range. 
The linear plot shows the orange and blue dots in more distinct lines. The orange dots from the construct containing the perfect and aligned binding sites assemble along the y-axis, where the EGFP fluorescence intensity is zero. For the other construct the EGFP line fully overlap with the EBFP2.  
</P><DIV class="center"><DIV class="thumb tnone"><DIV class="thumbinner" style="width:622px;"><DIV class="thumbcaption"><B>Figure 4:EGFP2/EBFP correlation of single transfected Hela cells according to flow cytometry analysis</B>  different binding sites for miR122 cotransfected with miR-122 or with miR-155, respectively.  The orange dots represent the cotransfected cells with miR122 and the blue dots the cotransfected cells with miR-155. Hela cells were used.</DIV></DIV></DIV></DIV><DIV class="center"><DIV class="thumb tnone"><DIV class="thumbinner" style="width:622px;"><DIV class="thumbcaption"><B>Figure 5: EGFP/EBFP2 correlation of single transfected Hela cells according to flow cytometry analysis</B>  different binding sites for miR122 cotransfected with miR-122 or with miR-155, respectively.  The orange dots represent the cotransfected cells with miR122 and the blue dots the cotransfected cells with miR-155. Hela cells were used.</DIV></DIV></DIV></DIV><P>
Each cell in the images has a position in the x- and y-axis. So the intensity and number of cells with a certain intensity can be used for <A href="https://2010.igem.org/Team:Heidelberg/Notebook/Methods#Flow_cytometry" class="external text" rel="nofollow">calculation</A> of the mean EGFP/EBFP2 ratio. The calculated data show, that there is a significant EGFP knock-down in the cells cotransfected with miR-122 and the contruct with the perfect binding site or the aligned binding sites, respectively.
</P><DIV class="center"><DIV class="thumb tnone"><DIV class="thumbinner" style="width:531px;"><DIV class="thumbcaption"><B>Figure 6:EGFP/EBFP2 correlation of single transfected Hela cells according to flow cytometry analysis</B>  different binding sites for miR122 cotransfected with miR-122 or with miR-155, respectively.</DIV></DIV></DIV></DIV><H3><SPAN class="mw-headline" id="Confocal_microscopy">Confocal microscopy</SPAN></H3><DIV class="center"><DIV class="thumb tnone"><DIV class="thumbinner" style="width:500px;"><DIV class="thumbcaption"><B>Figure 7: different binding sites for miR122, HeLa cotransfected with miR122 expression plasmid</B></DIV></DIV></DIV></DIV><P>
The EGFP-expression normalized to the EGFB2 expression is set to 100% for the miMeasure construct transfected with the non-matching miRNA (in this case miR-155). The knock-down efficiency of one perfect binding site is around 30%, which also accounts for the three aligned perfect binding sites and the two aligned imperfect ones. The binding site with bulges from position 9-12 and 9-22 don't show any knock-down.  
</P><DIV class="center"><DIV class="thumb tnone"><DIV class="thumbinner" style="width:502px;"><DIV class="thumbcaption"><B>Figure 8: different binding sites for miR122, Huh7 cells</B></DIV></DIV></DIV></DIV><P>
The transfected HeLa are also imaged with the epifluorescent microscope. Large amount of cells in the negative control (miMeasure with perfect binding site cotransfected with miR-155, see a)are green, whereas most cells with the miMeasure construct containing the perfect binding sites (see b) are blue. 
</P><DIV class="center"><DIV class="thumb tnone"><DIV class="thumbinner" style="width:802px;"><DIV class="thumbcaption"><B>Figure 9: epifluorescent microscopy image (10x) of Hela cells transfected with miMeasure</B> miMeasure with a perfect binding site is a) cotransfected with miR-155, which has no specificity to miR-122, b) cotransfected with miR-122, which is complementary to the perfect binding site. EGFP is regulated by miR-122, EBFP2 is unregulated and serves as transfection control.</DIV></DIV></DIV></DIV><P>
The Huh7 cells were also transfected with the 4 different constructs. Here a cotransfection with miR-122 is not necessary, since Huh7 cells express miR-122 themselves. The knock-down of the perfect binding sites are stronger than the knock-down in the Hela cells. Here the knock-down efficiency is 80% for the perfect binding site and the aligned constructs. 
</P><H3><SPAN class="mw-headline" id="Analysis_of_endogenous_miRNA">Analysis of endogenous miRNA</SPAN></H3><P>HepG2, another liver cell line, is also transfected with the constructs containing the perfect and the aligned constructs for miR-122. The cotransfection with miR-155 serves again as a negative control. For this cell line there was only a slight knock-down observed for all of the constructs, it is much less compared to the HuH7 cell line, where the knock-down ranges from 20-40%.
</P><DIV class="center"><DIV class="thumb tnone"><DIV class="thumbinner" style="width:792px;"><DIV class="thumbcaption"><B>Figure 10: relative EGFP expression of transfected HepG2 cells according to confocal microscopy analysis</B>  different binding sites for miR122 are cotransfected with miR-122. The negative control doesn't contain any binding sites.</DIV></DIV></DIV></DIV><P>The EGFP/EBFP2 ratio from each transfected HuH7 cell is calculated for 52 cells. The cells were transfected with the construct carriying the three aligned perfect binding sites against miR-122. The EGFP/EBFP2 ratio in each cell is different and ranges from 0.1 to 6.
</P><DIV class="center"><DIV class="thumb tnone"><DIV class="thumbinner" style="width:852px;"><DIV class="thumbcaption"><B>Figure 11: Single cell analysis of relative EGFP expression of transfected HuH7 cells by confocal microscopy analysis</B> The numbers in the barplot correspond to the cells on the right image.</DIV></DIV></DIV></DIV><H2><SPAN class="mw-headline" id="Discussion">Discussion</SPAN></H2><P>Measurements of synthetic and endogenous miRNA expression level as well as the knock-down efficiency by different miRNAs can characterized via the miMeasure constructs. The fluorescence of EGFP and EBFP2, and therefore the efficiency of knockdown, can be estimated using different methods, for example <A href="https://2010.igem.org/Team:Heidelberg/Notebook/Methods#Flow_cytometry" class="external text" rel="nofollow">flow cytometry</A>, manual and automated fluorescence <A href="https://2010.igem.org/Team:Heidelberg/Notebook/Material_Methods#Microscopy" class="external text" rel="nofollow">microscopy</A> or potentially an automated fluorescence plate reader system. Flow cytometry and fluorescence microscopy offer the possibility to measure knockdown at the single cell level, the first one with the possibility of high throughput, the second with a good precision. Since flow cytometry has a smaller dynamic range, thus even a GFP knock-down of roughly 60% can cause the relative GFP fluorescence intensity values collapsing to zero. 
This quantitative approach offers many possibilities, two of them being expored here: estimating the knockdown capacity of binding sites that differs from a perfect one and determining the presence of endogenous miRNA directly in living cells.
The miMeasure construct can differ between a variation of knock-down efficiencies in the synthetic miRNA binding patterns, which was successfully generated by the <A href="https://2010.igem.org/Team:Heidelberg/Notebook/BSDesign" class="external text" rel="nofollow">miRaPCR</A>. The perfect and aligned binding sites have the greatest knock-down efficiencies with roughly 50%. These experiemtal data corresponds to predictions from the modeling and hypothesis from the liturature. Hence the higher the complementarity from binding site to miRNA the more efficient the miRNA induced knock-down. Alignment of binding sites are shown to have additive effects, either. 
miMeasure can also be used for the characterization of binding sites against synthetic miRNAs. The data only shows a difference between the perfect binding site and the construct without binding site. There were no more subtle differences detected from the slightly modified binding sites. The reason might be, that the synthtic miRNA is much less efficient concerning the knock-down than the natural binding site. So the modified binding sites, which would give an even weaker output, would be out of range. The possible low efficiency of the synthtic miRNA miRsAg could be explained by the fact, that it is assembled by taking the human miR-122 as a template, so that it can be driven by another promoter than Pol1, which is especially weak. So the synthetic miRNA might not expressed in the same amount like the natural miR-122. Real-time PCR could be done to validate the miRNA expression. 
The direct comparison between cell lines enables future miRNA profiling with miMeasure. Several cell lines can be compared directly with each other after cotransfection with the same miRNA. The analysis of the cell lines HeLa, HuH7 and HepG2 gave significant results in miRNA-expression level in those cell lines. This data also corresponds to liturature, which underlies the much higher miR-122 expression in HuH7 cells than in HepG2 cells <SPAN class="referencelink"><A href="http://2010.igem.org/Team:Heidelberg/Project/miMeasure#References" class="external text" rel="nofollow">(Young et al., 2010)</A></SPAN>. Besides the comparison between different cell lines, even single cells can be compared with each other, so that characterization of expression in single cells is possible, which will enable single cell studies as well. The great difference between single cell expression of miR-122 can be due to the carcinogenous property of HuH7. Accoring to <SPAN class="referencelink"><A href="http://2010.igem.org/Team:Heidelberg/Project/miMeasure#References" class="external text" rel="nofollow">(Lin et al., 2008)</A></SPAN> low levels of miR-122 are responsible for reduced apoptosis in liver cells, because miR-122 targets besides many other important genes, an anti-apoptotic gene, Bcl-w.
</P><H2><SPAN class="mw-headline" id="References">References</SPAN></H2><P>Ai HW, Shaner NC, Cheng Z, Tsien RY, Campbell RE. Exploration of new chromophore structures leads to the identification of improved blue fluorescent proteins. Biochemistry. 2007 May 22;46(20):5904-10.
</P><P>Bartel DP. MicroRNAs: target recognition and regulatory functions. Cell. 2009 Jan 23;136(2):215-33.
</P><P>Coulouarn C, Factor VM, Andersen JB, Durkin ME, Thorgeirsson SS. Loss of miR-122 expression in liver cancer correlates with suppression of the hepatic phenotype and gain of metastatic propertiesmiR-122 repression is a marker of tumor progression in HCC. Oncogene. 2009 Oct 8;28(40):3526-36.
</P><P>Fabian MR, Sonenberg N, Filipowicz W. Regulation of mRNA translation and stability by microRNAs. Annu Rev Biochem. 2010;79:351-79.
</P><P>Girard M, Jacquemin E, Munnich A, Lyonnet S, Henrion-Caude A. miR-122, a paradigm for the role of microRNAs in the liver. J Hepatol. 2008 Apr;48(4):648-56.
</P><P>Lin CJ, Gong HY, Tseng HC, Wang WL, Wu JL. miR-122 targets an anti-apoptotic gene, Bcl-w, in human hepatocellular carcinoma cell lines. Biochem Biophys Res Commun. 2008 Oct 24;375(3):315-20.
</P><DIV class="backtop"><A href="#top">↑</A></DIV></DIV><DIV class="printfooter">
Retrieved from &quot;<A href="http://2010.igem.org/Team:Heidelberg/Project/miMeasure">http://2010.igem.org/Team:Heidelberg/Project/miMeasure</A>&quot;</DIV></DIV></DIV><DIV id="footer-box" class="noprint"><DIV id="footer"><UL id="f-list"><LI id="t-recentchanges"><A href="/Special:RecentChanges" title="Recent changes">Recent changes</A></LI><LI id="t-whatlinkshere"><A href="/Special:WhatLinksHere/Team:Heidelberg/Project/miMeasure" title="List of all wiki pages that link here [j]" accesskey="j">What links here</A></LI><LI id="t-recentchangeslinked"><A href="/Special:RecentChangesLinked/Team:Heidelberg/Project/miMeasure" title="Recent changes in pages linked from this page [k]" accesskey="k">Related changes</A></LI><LI id="t-specialpages"><A href="/Special:SpecialPages" title="List of all special pages [q]" accesskey="q">Special pages</A></LI><LI><A href="/Special:Preferences">My preferences</A></LI></UL></DIV><DIV id="footer"><UL id="f-list"><LI id="t-print"><A href="/wiki/index.php?title=Team:Heidelberg/Project/miMeasure&amp;printable=yes" title="Printable version of this page [p]" accesskey="p">Printable version</A></LI><LI id="t-permalink"><A href="/wiki/index.php?title=Team:Heidelberg/Project/miMeasure&amp;oldid=208354" title="Permanent link to this revision of the page">Permanent link</A></LI><LI id="privacy"><A href="/2010.igem.org:Privacy_policy" title="2010.igem.org:Privacy policy">Privacy policy</A></LI><LI id="disclaimer"><A href="/2010.igem.org:General_disclaimer" title="2010.igem.org:General disclaimer">Disclaimers</A></LI></UL></DIV></DIV></BODY></HTML>