"
Page
Discussion
View source
History
teams
Log in
 
Team:Heidelberg/Project/Measurement Standard
From 2010.igem.org
WRONG!!!
Team
Overview
Students
Instructors
Advisors
Institutes
Gallery
miDANCE
 
Project
Abstract
Introduction
Synthetic miRNA Kit
Capsid Shuffling
    Homology Based
    ViroBytes
miMeasure
in vivo Study
Summary & Outlook
References
 
Parts
Parts
Characterization
 
Modeling
Introduction
miBEAT
miRockdown
miBS designer
miCrappallo
Descriptions
 
Notebook
Synthetic miRNA Kit
Capsid Shuffling
    Homology Based
    ViroBytes
miMeasure
in vivo Study
Modeling
Material
Methods
Psychological Survey
Philosophical Reflection
Biosafety
 
Human Practices
Psychological Survey
Philosophical Reflection
 
Sponsors
Measurement Standard
Abstract
With the increasing importance of small RNA molecules in gene therapy the identification and characterization ion of miRNAs and their binding sites become crucial for innovative applications. In order to exploit the miRNA ability to target and regulate specific genes, we constructed a measurement standard not only to characterize existing miRNAs but also to validate potential synthetic shRNA miRNAs for a new therapeutic approach. The synthetic miRNAs we created lack endogenous targets and are thus applicable for gene regulation without any side effects. This openes new possiblities of precise expression tuning.  Our miMeasure plasmid normalizes knockdown of the green fluorescent protein (EGFP) to the blue fluorescent protein (EBFP2). This allows an accurate study of binding site properties, since both fluorescent proteins are combined in the same construct and driven by the same bidirectional promoter. Another advantage is, that any desired binding site can be cloned easily into the miMeasure plasmid with the BB_2 standard. As the binding site is inserted downstream of EGFP,  a regulation of EGFP expression is to be expected.  The percentage of knockdown of each modified binding site can be determined by the comparison of the ratio of EGFP to EBFP2 ratio compared to the ratio of the perfect binding site caused knock-down. The ratio is derived from a linear regression curve. Therefor the knock-down efficiency can be conducted by various basic methods e.g. plate reading, flow cytometry or microscopy.
Introduction
Micro RNAs regulate the translation of their target genes by preferably binding to regions in the 3’ UTR which are called miRNA binding sites (BS)(ref). This miRNA BS consists of a bp seed region at the 5'UTR  that is perfectly matched to the miRNA, and surrounding regions that matched partially. The seed region is defined as being the minimal required basepairing at the 5’ end of the miRNA that can regulate the mRNA. Apart from the seed region, binding can be unspecific, creating bulges between miRNA and mRNA (fig). The position and properties of the bulges seem to play a role in miRNA binding and therefore knockdown efficiency (reviewed in Bartel et al., 2009).
Since we were going to use synthetic miRNA BS in our genetherapeutic approach, we had to find a way to study their effects in a standardized manner that would be comparable and reproducible.
One goal of the iGEM Team Heidelberg 2010 was to test the effects of changes in BS sequence and thereby characterize miRNA BS. To standardize our measurements of knockdown according to BS specificity, we had to come up with a new standard that is independent from the endogenous cell machinery. We decided to bring in synthetic miRNAs and engineer BS for them, simulating naturally occurring miRNAs and miRNA BS without having to worry about the effect of endogenous targets. Of course there are also differences that arise through the availability of the enzymes involved in the miRNA pathway that may differ slightly from cell to cell. Therefore, we also measured the knockdown achieved by the perfect binding site and set this as 100% knockdown efficiency. Ideally, the shRNA would be expressed stably in the cell line, but a uniform co-transfection also leads to an even distribution of shRNA into each cell.
The main idea of our measurement standard, miMeasure, was to express two nearly identical but discernable proteins, one of them tagged with a BS, the other one unregulated. These two reporters were expressed by a bidirectional CMV promoter to make sure their expression rate is identical. We used a destablilized version of GFP, dsEGFP by Clontech (ref) and a dsEBFP2 that was derived from the same sequence. Thus, we could make sure that both proteins exhibit the same synthesis and degradation properties, making them directly comparable. Hereby we can also neglect the difference between mRNA and protein knockdown and can take the fluorescence of the marker protein as a direct, linear output of mRNA knockdown. We included a BBB standard site into our plasmid, which allows to clone BS behind the GFP. If co-transfected with the corresponding shRNA, GFP will be downregulated, while BFP expression is maintained. The ratio of GFP to BFP expression can be used to conclude the knockdown efficiency (in percent, compared to perfect binding site=100% and no binding site=0%) of the BS. Having destabilized marker proteins with a turnover time of two hours enables us not only to avoid accumulation of marker proteins, which would make the knockdown harder to observe, but also to conduct time-lapse experiments. In the future, this could be for example a way to observe the activity patterns of endogenous miRNAs.
Results
Discussion
Methods
To measure GFP and BFP fluorescence intensity, we used microscopy and flow cytometer. Fluorescence was fist evaluated using the Leica DM IRB epifluorescence microscope. Only cells which were positive for transfection were measured.  First, the cells were washed with 1x PBS and detached from the plate using Trypsin. 30µl Trypsin was added to each well, incubated for ten minutes at room temperature. Cells were resuspended in 170µl 1%BSA in PBS and replicates for each condition were pooled into 24 well plates. 200µl from each well were used for FACS measurements, 100-150µl were used for confocal microscopy.
FACS
Microscopy
Single images were obtained using the Leica TCS SP5 confocal microscope and ??? camera with the Leica AF6000 imaging software ??. GFP fluorescence was excited by Argon 488nm laser and measured at xx-xxnm, BFP fluorescence was excited by UV laser at 405nm and measured at xx-xxnm. Pictures were taken sequentially line by line in three different channels for GFP, BFP and bright field.
Data Analysis
To analyze the fluorescence of single cells, we segmented the images using ImageJ. In 8bit pictures, we set the threshold for each channel to 50, thereby filtering the background. This allows us to annotate cells automatically using the “analyze particles” tool. We could now get the fluorescence intensity for each single cell on each channel (GFP or BFP) as an 8bit output, i.e. a value between 50 and 255. Form this data, we calculated the GFP:BFP ratio. We could then visualize the mean of these rations in a bar plot or use all the data to calculate a linear regression curve. (there will be pictures)
Consumables and Chemicals
PerkinElmer ViewPlate, product number: 6004920
Nunc 24 well microplate, product number: 176740
1x PBS (1.37mM NaCl, 0.27mM KCl, 10mM Na2HPO4, 0.2mM KH2PO4)
0.05% Trypsin-EDTA, Invitrogen GIBCO, product number: 15400054
Instruments
Leica DM IRB
Leica SP5
Tecan Infinite M200
FACS machine
ImageJ version 1.43
References
Contents
1 Abstract
2 Introduction
3 Results
4 Discussion
5 Methods
6 References
   
Retrieved from " http://2010.igem.org/Team:Heidelberg/Project/Measurement_Standard "
Recent changes
What links here
Related changes
Special pages
My preferences
Printable version
Permanent link
Privacy policy
Disclaimers
