<HTML xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en" dir="ltr">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki  ltr ns-0 ns-subject page-Team_Heidelberg_Project_miRNA_Kit"><DIV id="globalWrapper"><DIV id="top-section"><DIV id="p-logo"><A href="/Main_Page" title="Main Page">&quot;
	    </A></DIV><DIV id="menubar" class="left-menu noprint"><UL><LI class="selected"><A href="/Team:Heidelberg/Project/miRNA_Kit">Page               </A></LI><LI class="new"><A href="/wiki/index.php?title=Talk:Team:Heidelberg/Project/miRNA_Kit&amp;action=edit&amp;redlink=1">Discussion               </A></LI><LI><A href="/wiki/index.php?title=Team:Heidelberg/Project/miRNA_Kit&amp;action=edit">View source               </A></LI><LI><A href="/wiki/index.php?title=Team:Heidelberg/Project/miRNA_Kit&amp;action=history">History               </A></LI><LI style="color:#808080;cursor:default">teams</LI></UL></DIV><DIV class="right-menu noprint" id="menubar"><UL><LI id="pt-login"><A href="/wiki/index.php?title=Special:UserLogin&amp;returnto=Team:Heidelberg/Project/miRNA_Kit" title="You are encouraged to log in; however, it is not mandatory [o]" accesskey="o">Log in</A></LI></UL></DIV><DIV id="search-controls" class="noprint"><FORM action="/Special:Search" id="searchform"> </FORM></DIV></DIV><DIV id="content"><H1 class="firstHeading">Team:Heidelberg/Project/miRNA Kit</H1><DIV id="bodyContent"><H3 id="siteSub" class="noprint">From 2010.igem.org</H3><DIV id="header"><DIV id="menu"><UL id="nav"><LI id="menu_home"><A class="aHome" href="https://2010.igem.org/Team:Heidelberg">Home</A></LI><LI id="menu_team"><A class="aTeam" href="https://2010.igem.org/Team:Heidelberg/Team">Team</A><TABLE class="menutable" cellpadding="0" cellspacing="0"><TBODY><TR><TD class="lefttdtopteam"><A class="aTeamOverview" href="https://2010.igem.org/Team:Heidelberg/Team">Overview</A><A class="aTeamMembers" href="https://2010.igem.org/Team:Heidelberg/Team/Members">Students</A><A class="aTeamMembers" href="https://2010.igem.org/Team:Heidelberg/Team/Members#Instructors">Instructors</A><A class="aTeamMembers" href="https://2010.igem.org/Team:Heidelberg/Team/Members#Advisors">Advisors</A><A class="aInstitutes" href="https://2010.igem.org/Team:Heidelberg/Team/Institutes">Institutes</A><A class="aGallery" href="https://2010.igem.org/Team:Heidelberg/Team/Gallery">Gallery</A><A class="aDance" href="https://2010.igem.org/Team:Heidelberg/midance">miDANCE</A><A class="amithanks" href="https://2010.igem.org/Team:Heidelberg/Team/miThanks">Acknowledgements</A></TD><TD class="righttdtop"> </TD></TR></TBODY></TABLE></LI><LI id="menu_project"><A class="aProject" href="https://2010.igem.org/Team:Heidelberg/Project">Project</A><TABLE class="menutable" cellpadding="0" cellspacing="0"><TBODY><TR><TD class="lefttdtopproject"><A class="aAbstract" href="https://2010.igem.org/Team:Heidelberg/Project">Abstract</A><A class="aIntroduction" href="https://2010.igem.org/Team:Heidelberg/Project/Introduction">Introduction</A><A class="aProject_miRNA_Kit" href="https://2010.igem.org/Team:Heidelberg/Project/miRNA_Kit">Synthetic miRNA Kit</A><A class="aPro_Capsid_Shuffling" href="https://2010.igem.org/Team:Heidelberg/Project/Capsid_Shuffling">Capsid Shuffling</A><A class="aPro_Homology" href="https://2010.igem.org/Team:Heidelberg/Project/Capsid_Shuffling/Homology_Based">    Homology Based</A><A class="aPro_Virobytes" href="https://2010.igem.org/Team:Heidelberg/Project/Capsid_Shuffling/ViroBytes">    ViroBytes</A><A class="aPro_miMeasure" href="https://2010.igem.org/Team:Heidelberg/Project/miMeasure">miMeasure</A><A class="aPro_Mouse" href="https://2010.igem.org/Team:Heidelberg/Project/Mouse_Infection"><I>in vivo</I> Study</A><A class="aPro_Summary" href="https://2010.igem.org/Team:Heidelberg/Project/Summary">Summary &amp; Outlook</A><A class="aPro_Ref" href="https://2010.igem.org/Team:Heidelberg/Project/References">References</A><A class="aPro_Ref" href="https://2010.igem.org/Team:Heidelberg/Project/Contributions">Contributions</A></TD><TD class="righttdtop"> </TD></TR></TBODY></TABLE></LI><LI id="menu_parts"><A class="aParts" href="https://2010.igem.org/Team:Heidelberg/Parts">Parts</A><TABLE class="menutable" cellpadding="0" cellspacing="0"><TBODY><TR><TD class="lefttdtop"><A class="aparts_ger" href="https://2010.igem.org/Team:Heidelberg/Parts">Parts</A><A class="aparts_char" href="https://2010.igem.org/Team:Heidelberg/Parts/Characterization">Characterization</A></TD><TD class="righttdtop"> </TD></TR></TBODY></TABLE></LI><LI id="menu_modeling"><A class="aModeling" href="https://2010.igem.org/Team:Heidelberg/Modeling">Modeling</A><TABLE class="menutable" cellpadding="0" cellspacing="0"><TBODY><TR><TD class="lefttdtop"><A class="amode_over" href="https://2010.igem.org/Team:Heidelberg/Modeling">Introduction</A><A class="amiGUI" href="https://2010.igem.org/Team:Heidelberg/Modeling/miGUI">miBEAT</A><A class="amirockdown" href="https://2010.igem.org/Team:Heidelberg/Modeling/miRockdown">miRockdown</A><A class="amibsdesigner" href="https://2010.igem.org/Team:Heidelberg/Modeling/miBSdesigner">miBS designer</A><A class="amispec" href="https://2010.igem.org/Team:Heidelberg/Modeling/miSpec">mUTING</A><A class="amodelset" href="https://2010.igem.org/Team:Heidelberg/Modeling/trainingset">Descriptions</A></TD><TD class="righttdtop"> </TD></TR></TBODY></TABLE></LI><LI id="menu_notebook"><A class="aNotebook" href="https://2010.igem.org/Team:Heidelberg/Notebook">Notebook</A><TABLE class="menutable" cellpadding="0" cellspacing="0"><TBODY><TR><TD class="lefttdtop"><A class="aNote_miRNA_kit" href="https://2010.igem.org/Team:Heidelberg/Notebook/miRNA_Kit">Synthetic miRNA Kit</A><A class="aNote_Capsid_Shuffling">Capsid Shuffling</A><A class="aNote_Homology" href="https://2010.igem.org/Team:Heidelberg/Notebook/Capsid_Shuffling/Homology_Based">    Homology Based</A><A class="aNote_Virobytes" href="https://2010.igem.org/Team:Heidelberg/Notebook/Capsid_Shuffling/ViroBytes">    ViroBytes</A><A class="aNote_miMeasure" href="https://2010.igem.org/Team:Heidelberg/Notebook/miMeasure">miMeasure</A><A class="aNote_BSdesign" href="https://2010.igem.org/Team:Heidelberg/Notebook/BSDesign">Binding Site Design </A><A class="aNote_Mouse" href="https://2010.igem.org/Team:Heidelberg/Notebook/Mouse_Infection"><I>in vivo</I> Study</A><A class="aNote_Modeling" href="https://2010.igem.org/Team:Heidelberg/Notebook/Modeling">Modeling</A><A class="aNote_Material" href="https://2010.igem.org/Team:Heidelberg/Notebook/Material">Materials</A><A class="aNote_Methods" href="https://2010.igem.org/Team:Heidelberg/Notebook/Methods">Methods</A><A class="aNote_Psycho" href="https://2010.igem.org/Team:Heidelberg/Human_Practices/Psychology/Notebook">Psychological Survey</A><A class="aNote_Philo" href="https://2010.igem.org/Team:Heidelberg/Human_Practices/Philosophy/Part2">Philosophical Reflection</A><A class="aNote_Biosafety" href="https://2010.igem.org/Team:Heidelberg/Notebook/Biosafety">Biosafety</A></TD><TD class="righttdtop"> </TD></TR></TBODY></TABLE></LI><LI id="menu_human_practices"><A class="aHuman_Practices" href="https://2010.igem.org/Team:Heidelberg/Human Practices">Human Practices</A><TABLE class="menutable" cellpadding="0" cellspacing="0"><TBODY><TR><TD class="lefttdtop"><A class="aPsycho" href="https://2010.igem.org/Team:Heidelberg/Human Practices/Psychology">Psychological Survey</A><A class="aPhilo" href="https://2010.igem.org/Team:Heidelberg/Human Practices/Philosophy">Philosophical Reflection</A></TD><TD class="righttdtop"> </TD></TR></TBODY></TABLE></LI><LI id="menu_support"><A class="aSupport" href="https://2010.igem.org/Team:Heidelberg/Support">Sponsors</A></LI></UL></DIV></DIV><H3><SPAN class="mw-headline" id="Working_Modes"> Working Modes </SPAN></H3><P>The synthetic miR Kit can be applied in three different ways:
</P><DL><DD>I) <B>Tuning</B>: adjusting the expression of the GOI by expressing a synthetic microRNA in the target cell/tissue
</DD></DL><DL><DD>II) <B>Off-Targeting</B>: switching OFF the expression of the GOI in case a certain endogenous microRNA is present in the target cell/tissue
</DD></DL><DL><DD>III) <B>On-Targeting</B>: switching ON the expression of the GOI in case a certain endogenous microRNA is present in the target cell/tissue
</DD></DL><H3><SPAN class="mw-headline" id="Simple_Tuning_Procedure"> Simple Tuning Procedure </SPAN></H3><UL><LI> choose an <A href="/Team:Heidelberg/Project/Introduction" title="Team:Heidelberg/Project/Introduction"> interesting microRNA</A></LI><LI><A href="https://2010.igem.org/Team:Heidelberg/Modeling/miBSdesigner" class="external text" rel="nofollow">create</A> referring binding sites
</LI><LI> order your binding site oligos
</LI><LI><A href="https://2010.igem.org/Team:Heidelberg/Notebook/Methods#Cloning" class="external text" rel="nofollow">clone</A> them into your <A href="https://2010.igem.org/Team:Heidelberg/Parts#final_constructs" class="external text" rel="nofollow">miTuner construct</A></LI><LI><A href="https://2010.igem.org/Team:Heidelberg/Notebook/Methods#Transfection" class="external text" rel="nofollow">transfect</A> your cells
</LI><LI> measure the <A href="/Team:Heidelberg/Project/miRNA_Kit#Results" title="Team:Heidelberg/Project/miRNA Kit"> tuned</A> expression!
</LI></UL><H3><SPAN class="mw-headline" id="Advancement"> Advancement </SPAN></H3><UL><LI> digestion of miR Kit construct with BamHI
</LI><LI> cloning into viral backbone (e. g. <A href="https://2010.igem.org/Team:Heidelberg/Notebook/Material" class="external text" rel="nofollow">pBS_U6</A>)
</LI><LI><A href="https://2010.igem.org/Team:Heidelberg/Notebook/Methods#Virus_Production" class="external text" rel="nofollow">virus production</A></LI><LI> infection of cells
</LI><LI> achievement of specific target cell <A href="https://2010.igem.org/Team:Heidelberg/Project/Capsid_Shuffling" class="external text" rel="nofollow">tropism</A></LI></UL><P>→ further improvement of gene expression tuning
</P><H3><SPAN class="mw-headline" id="Tuning_Raw_Data"> Tuning Raw Data </SPAN></H3><P>For our <I>in vitro</I> tuning, you can have a look even at our unprocessed data with specific <A href="https://2010.igem.org/Team:Heidelberg/Notebook/Methods#nomenclature" class="external text" rel="nofollow">nomenclature</A>:
</P><UL><LI><A href="/wiki/images/0/03/Plate1_process_H1.xls" class="internal" title="Plate1 process H1.xls">Media:Plate1 process H1.xls</A>, </LI><LI><A href="/wiki/images/8/8d/Plate2_process_H1.xls" class="internal" title="Plate2 process H1.xls">Media:Plate2 process H1.xls</A>, </LI><LI><A href="/wiki/images/b/be/Plate3_process_H1.xls" class="internal" title="Plate3 process H1.xls">Media:Plate3 process H1.xls</A>, </LI><LI><A href="/wiki/images/5/57/Haat_20101022_M1-M4_ctrl_H1.xls" class="internal" title="Haat 20101022 M1-M4 ctrl H1.xls">Media:Haat 20101022 M1-M4 ctrl H1.xls</A>, </LI><LI><A href="/wiki/images/f/f6/Haat_20101026_M5-M8_ctrl_H1.xls" class="internal" title="Haat 20101026 M5-M8 ctrl H1.xls">Media:Haat 20101026 M5-M8 ctrl H1.xls</A>, </LI><LI><A href="/wiki/images/5/52/Haat_20101026_M9M22_ctrl_H1.xls" class="internal" title="Haat 20101026 M9M22 ctrl H1.xls">Media:Haat 20101026 M9M22 ctrl H1.xls</A>, </LI><LI><A href="/wiki/images/2/21/HAAT_H1_final.xls" class="internal" title="HAAT H1 final.xls">Media:HAAT H1 final.xls</A>. </LI><LI><A href="/wiki/images/9/90/Plate1_process_U6_haat.xls" class="internal" title="Plate1 process U6 haat.xls">Media:Plate1 process U6 haat.xls</A>,</LI><LI><A href="/wiki/images/8/8f/Plate2_process_U6_haat.xls" class="internal" title="Plate2 process U6 haat.xls">Media:Plate2 process U6 haat.xls</A>,</LI><LI><A href="/wiki/images/9/9c/Plate3_process_U6_haat.xls" class="internal" title="Plate3 process U6 haat.xls">Media:Plate3 process U6 haat.xls</A>,</LI><LI><A href="/wiki/images/9/95/Haat_20101026_plate2_U6.xls" class="internal" title="Haat 20101026 plate2 U6.xls">Media:Haat 20101026 plate2 U6.xls</A>,</LI><LI><A href="/wiki/images/6/60/Haat_20101026_plate1_U6.xls" class="internal" title="Haat 20101026 plate1 U6.xls">Media:Haat 20101026 plate1 U6.xls</A>,</LI><LI><A href="/wiki/images/6/6d/Haat_20101026_plate3_U6.xls" class="internal" title="Haat 20101026 plate3 U6.xls">Media:Haat 20101026 plate3 U6.xls</A>,</LI><LI><A href="/wiki/images/0/0f/HAAT_U6_final.xls" class="internal" title="HAAT U6 final.xls">Media:HAAT U6 final.xls</A>.</LI></UL><P> </P></DIV><DIV class="t1">Synthetic miRNA Kit</DIV><CENTER><H4><SPAN class="mw-headline" id="miTuner_-_a_kit_for_microRNA_based_gene_expression_tuning_in_mammalian_cells">miTuner - a kit for microRNA based gene expression tuning in mammalian cells</SPAN></H4></CENTER><CENTER><I>With the synthetic miRNA kit, we provide a comprehensive mean 
<P>to plan, conduct and evaluate experiments dealing with <A href="https://2010.igem.org/Team:Heidelberg/Parts" class="external text" rel="nofollow">miBricks</A></P>
(i. e. microRNA related Biobricks) as key regulators in mammalian cells.</I></CENTER><H2><SPAN class="mw-headline" id="Abstract">Abstract</SPAN></H2><P>Regulation of any gene of interest has never been as easy as with our <B>miRNA-based expression tuning kit miTuner</B>. Rational design of synthetic miRNA binding sites according to our recommendations enables fine-tuning of gene expression in a range between 5% and 100%. Additionally, we offer Off- and On-targeting switches which effect GOI expression in only one or all but one tissue depending on endogenously occurring miRNAs.
</P><DIV class="thumb tleft"><DIV class="thumbinner" style="width:372px;"><DIV class="thumbcaption"><B>Figure 1</B>: Three modes of gene regulation accomplished by miTuner. <B>a)</B>fine-tuning of gene expression. <B>b) and c)</B> Off- and On-targeting for tissue specific expression of a therapeutic gene</DIV></DIV></DIV><P>The <B>tuning application</B> is based on a dual promoter construct that expresses a GOI controlled by a synthetic miRNA which is expressed from either the same or a second construct (<B>figure 1a</B>). Differing miRNA-binding site interaction efficiencies caused by binding sites of different sequence properties are used to distinctly adjust expression strength of the GOI.
</P><P>For <B>Off-targeting</B>, the GOI is under control of miRNAs that are found in tissue where gene expression is thereupon silenced while the GOI can still be expressed in other tissues as visualized in <B>figure 1b</B>.
</P><P><B>On-targeting</B> is based on the expression of the GOI from a promoter containing a Tet Operator (Tet02) that negatively regulates gene expression in the presence of a Tet Repressor (<B>figure 1c</B>). If the Tet Repressor tagged with binding sites for an endogenous miRNA, that is specifically expressed in the target cells/tissue. In consequence, the TetR is knocked down, releasing the promoter and enabling specific GOI expression.
</P><H2><SPAN class="mw-headline" id="Introduction">Introduction</SPAN></H2><P>MicroRNAs (miRNAs) are short endogenous, non-coding RNAs that mediate gene expression in a diversity of organisms <SPAN class="referencelink"><A href="http://2010.igem.org/Team:Heidelberg/Project/miRNA_Kit#References" class="external text" rel="nofollow">(Bartel, 2004)</A></SPAN>. Although the understanding of their biological functions is progressing remarkably, the exact mechanisms of regulation are still not unambiguously defined. However, it is commonly believed that miRNAs <B>trigger target mRNA regulation</B> by binding to 3’ untranslated region (UTR) of its target <SPAN class="referencelink"><A href="http://2010.igem.org/Team:Heidelberg/Project/miRNA_Kit#References" class="external text" rel="nofollow">(Chekulaeva and Filipowicz, 2009)</A></SPAN>.  Exact principles of expression knockdown mediated by miRNA are still in debate <SPAN class="referencelink"><A href="http://2010.igem.org/Team:Heidelberg/Project/miRNA_Kit#References" class="external text" rel="nofollow">(Eulalio et al., 2008)</A></SPAN>.However, sequence depending <B>binding site properties</B> have an essential impact on miRNA-mRNA interaction.  Depending on pairing specificity translational repression is mediated through the imperfect miRNA-mRNA hybrids. The potential for stringent regulation of transgene expression makes the miRNA world a promising area of gene therapy <SPAN class="referencelink"><A href="http://2010.igem.org/Team:Heidelberg/Project/miRNA_Kit#References" class="external text" rel="nofollow">(Brown et al.,2009)</A></SPAN>. There is a need for tight control of gene expression, since cellular processes are sensitive to expression profiles. Non-mediated gene expression can lead to fatal dysfunction of molecular networks. It is widely known, that miRNAs can adjust such fluctuations <SPAN class="referencelink"><A href="http://2010.igem.org/Team:Heidelberg/Project/miRNA_Kit#References" class="external text" rel="nofollow">(Brenecke et al., 2005)</A></SPAN>. A combination of random and rational <B>design</B> of binding sites could become a <B>powerful tool</B> to achieve a narrow range of resulting gene expression knockdown. To ease <I>in silico</I> construction of miRNA binding sites with appropriate characteristics for its target, we wrote a program - the <A href="https://2010.igem.org/Team:Heidelberg/Modeling/miBSdesigner" class="external text" rel="nofollow">miBS designer</A>. Using all of our <A href="https://2010.igem.org/Team:Heidelberg/Modeling" class="external text" rel="nofollow">theoretical models</A> gives the user the opportunity to calculate knockdown percentages caused by the designed miRNA in the target cell. 
Our <B>synthetic miRNA Kit</B> guarantees at least for individually modifiable but still ready-to-use constructs to interfere genetic circuits with synthetic or endogenous miRNAs. We preciously show, that gene expression can thereby by adjusted - tuned - to an arbitrary level. The <B>miTuner</B> (see sidebar) allows on the simultaneous expression of a synthetic miRNA and a gene of interest that is fused with a designed binding site for this specific miRNA. Our modular kit comes with different parts that can be combined by choice, e. g. different mammalian promoters and characterized binding sites of specific properties. By choosing a certain binding site to tag the GOI, one can tune the expression of this gene. Depending on the GOI, different means for read out of gene expression come into play. At first, we applied <A href="https://2010.igem.org/Team:Heidelberg/Notebook/Methods#Dual_Luciferase_Assay" class="external text" rel="nofollow">dual-luciferase assay</A>, since we used Luciferase as a reporter for a proof-of-principle approach. Later on, semi-quantitative immunoblots were prepared for testing of therapeutic genes. However, all the received information fed our models, thereby creating an <B>integrative feedback loop between experiments and in silico simulation</B>.
</P><H3><SPAN class="mw-headline" id="miTuner_Kit_components"> miTuner Kit components </SPAN></H3><P>The miTuner Kit consists of three basic components: </P><DL><DD>a) A kit of standardized synthetic microRNAs, corresponding binding sites, promoters and luciferase expression constructs as well as measurement constructs assembled in the BB-2 standard. As the miTuner kit was enabled </DD><DD>b) Protocols for engineering synthetic microRNAs, synthetic single microRNA binding sites as well as microRNA binding site patterns 
</DD></DL><P>Please find basic information about the kit components and engineering of the kit <A href="/Team:Heidelberg/Project/miRNA_Kit_engineer" title="Team:Heidelberg/Project/miRNA Kit engineer">here</A></P><DIV class="backtop"><A href="#top">↑</A></DIV><H2><SPAN class="mw-headline" id="Results">Results</SPAN></H2><P>All gene regulatory constructs for tuning, Off- or On-targeting can easily be assembled using <B>BBB standard cloning</B> from our miRNA Kit <A href="https://2010.igem.org/Team:Heidelberg/Parts" class="external text" rel="nofollow">parts</A>. After successful cloning, the constructs can be transfected onto a cell line of choice or transferred into a virus backbone for <A href="https://2010.igem.org/Team:Heidelberg/Project/Mouse_Infection" class="external text" rel="nofollow"><I>in vivo</I></A> experiments. For our <B>proof of principle</B>, we used firefly luciferase normalized to <I>Renilla</I> luciferase on miTuner to characterize knockdown efficiencies of different binding sites and show Off- and On-targeting by mouse infection carried by an AAV virus.
</P><H3><SPAN class="mw-headline" id="miTuner:_Expression_Fine-Tuning_by_Synthetic_miRNAs">miTuner: Expression Fine-Tuning by Synthetic miRNAs</SPAN></H3><P>The data shows a precisely tuned expression from almost 0% to 100% (Fig. 2, Fig. 3). Lowest expression refers to complete knockdown through fusion of perfect binding sites (always green bar on the left hand side of the figures) to the reporter gene. Expression from a construct without binding sites is set as 100% (always orange column on the right hand side of the figures). In presence of the specific shRNA miR, gene expression was mediated to various levels through interactions with the different imperfect binding sites. Whereas, when an unspecific shRNA miR was expressed, gene expression remained unaffected (see raw data below). This reference shows that the binding sites were correctly designed, since they seem to interact specifically with a referring shRNA miR. 
</P><DIV class="center"><DIV class="thumb tnone"><DIV class="thumbinner" style="width:602px;"><DIV class="thumbcaption"><B>Figure 2: Tuning of gene expression through different imperfect shRNA miR binding sites in pBS_SV40 Luc2 double transfected with a reference renilla construct.</B> The shRNA_hAAT construct was expressed from a pBS_H1 construct.</DIV></DIV></DIV></DIV><P>Figure 2 shows the results of Dual-Luciferase measurements of the miTuner plasmid with binding sites against shhAAT behind firefly luciferase. The highest knockdown can be achieved by using a perfect binding site. Single mutations outside the seed region at position 11, 12 or 10-12 lead to knockdown between 10% and 60% compared to unregulated expression. Bulges close to the seed region or changes in the seed region itself lead to very low downregulation. Having only the seed region as a target for the miRNA also leads to a less efficient knockdown compared with binding sites containing flanking regions.
</P><DIV class="center"><DIV class="thumb tnone"><DIV class="thumbinner" style="width:602px;"><DIV class="thumbcaption"><B>Figure 3: Tuning of gene expression through different imperfect shRNA miR binding sites in pBS_SV40_Luc2 construct cotransfected with a reference renilla construct.</B> Gene expression quantified via dual luciferase assay for constructs containing different imperfect binding sites for shhAAT. The shhAAT was expressed from a pBS_U6 plasmid</DIV></DIV></DIV></DIV><P>Figure 3 shows the same assay using binding sites against shhAAT. This time, the shhAAT is driven by a U6 promoter, which is stronger than the H1 promoter used for driving the shRNA in the previous figure. The results are overall similar, with changes in or directly adjacent to the seed region having the highest impact on knockdown efficiency. 
</P><DIV class="center"><DIV class="thumb tnone"><DIV class="thumbinner" style="width:602px;"><DIV class="thumbcaption"><B>Figure 4: Tuning of gene expression through different imperfect miR122 binding sites in psiCHECK-2.</B> Construct was transfected into HeLa cells together with an plasmid expressing miR122. Control without binding site was used for normalization.</DIV></DIV></DIV></DIV><P>We further analyzed binding sites derived from miR122 in the dual luciferase vector PsiCheck2 as can be seen in figure 4. Here we tested sixteen mutated binding sites in order to observe minute fine-tuning between one binding site and the next. Mutated Binding sites 123, 133, 134, 135 and 158 contain 4bp-bulges (non-paired regions) that don not seem to diminish knockdown efficiency much. 107 contains one binding site, while 134 and 135 contain two binding sites for the same miRNA and show a stronger knockdown than 107.
</P><H3><SPAN class="mw-headline" id="Off-Targeting_Using_Endogenous_miRNA">Off-Targeting Using Endogenous miRNA</SPAN></H3><P>Another application of our synthetic miRNA Kit profits of tissue specific endogenous miRNAs expression. These can be exploited for either Off- or On-Targeting. 
To enable Off-Targeting, the GOI expressed on miTuner can be tagged with a miRNA binding site specific for one or a combination of endogenous miRNA of the tissue that is to be excluded from gene expression.
In our experiment, we transfected Huh7 (human hepatoma) cells that endogenously overexpress miR122 with the miTuner construct after cloning different variations of binding sites for miR122 behind firefly luciferase. Figure 5 shows the results of the dual luciferase assay. Perfect binding sites result in almost complete inhibition of expression.
</P><DIV class="center"><DIV class="thumb tnone"><DIV class="thumbinner" style="width:502px;"><DIV class="thumbcaption"><B>Figure 5: Knockdown of reporter gene expression due to endogenous miR122 that interferes with binding sites.</B> Construct transfected to HuH cells to off-target those.</DIV></DIV></DIV></DIV><TABLE class="wikitable sortable" border="0" align="center" style="text-align: left"><CAPTION align="top, left"><B>Table 1: Mutated Binding Sites Against miR122</B></CAPTION><TBODY><TR bgcolor="#009be1"><TD>Sequence</TD><TD>Mutation</TD><TD>Description
</TD></TR><TR><TD>G ACAAACACCATTGTCACACTCCA TCTAGA GC</TD><TD>none</TD><TD>perfect BS
</TD></TR><TR><TD>G ACAAACACCAT_ACGG_ACACTCCAGAGACACAAACACCAT_GAAG_ACACTCCA GC </TD><TD>none</TD><TD>2x perfect BS
</TD></TR><TR><TD>G C*C*CCTG*A*TGGGG*G*CGACACTCCA TCTAGA GC  </TD><TD>point mutations outside seed</TD><TD>HCV5 BS
</TD></TR><TR><TD>TCGA G *AC*T*AA*GGCTGCT*CCAT*CAacactcca TCTAGA GC</TD><TD>one mutation inside seed</TD><TD>Aldo
</TD></TR><TR><TD>TCGA G ACAAACACCATTGTCA*G*A*T*TC*G*A TCTAGA GC </TD><TD>3 mutations in seed</TD><TD>3mutseed
</TD></TR><TR><TD>G ACAAACACCAT_ACGA_ACACTCCA TCTAGA GC </TD><TD>ACGA bulge</TD><TD>bulge region
</TD></TR><TR><TD>TCGA G ACAAACACCAT_GCAG_ACACTCCA TCTAGA GC</TD><TD>GCAG bulge</TD><TD>bulge region
</TD></TR></TBODY></TABLE><H3><SPAN class="mw-headline" id="On-Targeting_Using_Endogenous_miRNA">On-Targeting Using Endogenous miRNA</SPAN></H3><P>In line with the Off-targeting approach, In the case of On-targeting the presence of a certain miRNA in a cell switches on expression of the GOI. This can be accomplished by using a repressor that is targeted by an endogenously expressed miRNA. We exemplified this scenario by using a Tet Repressor fused with a perfect binding site for miRNA 122, a liver-specific miRNA (Jopling et al., 2005). At the same time, the promoter expressing the GOI would be under control of a Tet Operator. Upon presence of the miRNA 122, the Tet Repressor would be knocked down, release the promoter and expression of the GOI could be established. 
</P><DIV class="center"><DIV class="thumb tnone"><DIV class="thumbinner" style="width:485px;"><DIV class="thumbcaption"><B>Figure 6: Activation of gene expression upon Tet Repressor knockdown by liver-specific miR122</B></DIV></DIV></DIV></DIV><P>The miTuner Plasmid  (driving the measurment BBa_K337038 luciferase from CMV_TetO2 promoter) was cotransfected with a TetR expression construct tagged with 4 perfect mir122 binding sites. Rescue of gene expression occurs in case of coexpression of shmir122. In the control experiment (coexpression of miRsAg) did not lead to Luc2 expression rescue, inicating that the on tuning is working. 
</P><DIV class="thumb tleft"><DIV class="thumbinner" style="width:392px;"><DIV class="thumbcaption">Fig. 7: On-Tuning construct pBS_sv40_TetO2_Luc2 was cotransfected with a TetR tagged with 4 perfect miR122 binding sites and either an hcr (mir122 expression) construct or a mir155 control plasmid. Furthermore, a renilla construct was cotransfected also for normalization purposes. Transfection with hcr leads to higher Luc2 expression (rescue of expression) compared to the control, due to TetR knockdown. As positive control for the rescue, DOX was applied for avoid binding of the TetR to the Tet operator.</DIV></DIV></DIV><DIV class="thumb tright"><DIV class="thumbinner" style="width:392px;"><DIV class="thumbcaption">Fig. 8: On-Tuning construct pBS_sv40_TetO2_Luc2 was cotransfected with a control TetR NOT tagged with mir122 binding sites and either an hcr (mir122 expression) construct or a mir155 control plasmid. Furthermore, a renilla construct was cotransfected also for normalization purposes. Transfection with hcr or mir122 lead to comparable expression ratios (NO rescue of expression via hac), indicating that the control TetR construct is not affected by either shRNA. As positive control for the rescue, DOX was applied again in order to avoid binding of the TetR to the Tet operator.</DIV></DIV></DIV><P>
A pBS_SV40_TetO2_Luc2 construct was cotransfected with a Tet repressor construct tagged with 4 perfect mir122 binding sites. Fig. 7 shows a rescue of Luc2 expression in case of shmir122 expression (hcr construct), indicating, that the on-targeting is working. The right picture is the comparable control experiment using a not-binding site tagged TetR construct. As expected, no rescue of gene expression occurs in this control experiment (Fig. 8).
Those results indicate, that the on-tuning is working, in principle. In order to increase to rescue of gene expression, different TetR/pBS_SV40_TetO2_Luc2 ratios could be applied.
</P><DIV class="backtop"><A href="#top">↑</A></DIV><H2><SPAN class="mw-headline" id="Discussion">Discussion</SPAN></H2><P>
The On-construct measurments showed very promising results as well, having a clear firefly (Luc2) expression rescue in case the mir122 binding site TetR is cotransfected with the hcr (mir122) expression construct and not when being co-transfected with the control TetR (not binding site tagged). Therefor, the On-Tuning strategy is working. Adjustment of promoter strengths and ratios of TetR, hcr and the pBS_sv40_Tet02_Luc2 construct should be done in order to increase the rescue of gene expression compared to basal expression level of the Luc2 construct cotransfected with the TetR.
</P><DIV class="backtop"><A href="#top">↑</A></DIV><H2><SPAN class="mw-headline" id="Application_of_miTuner">Application of miTuner</SPAN></H2><H3><SPAN class="mw-headline" id="In_Vitro_Regulation_of_a_Therapeutic_Gene_hAAT"><I>In Vitro</I> Regulation of a Therapeutic Gene hAAT </SPAN></H3><P>We further tested our kit using a gene that is an interesting candidate for gene therapy, i. e. human alpha-1-antitrypsin (hAAT). Tight control of the genetic activity is fundamental, since deficiencies of hAAT can cause emphysema <SPAN class="referencelink"><A href="http://2010.igem.org/Team:Heidelberg/Project/miRNA_Kit#References" class="external text" rel="nofollow">(Lu et al., 2006)</A></SPAN>. With our tuning kit we have a powerful mean at hand to mediate expression levels. In this approach, we tagged hAAT, that we used as our GOI, with binding sites (for miRsAg) that we measured and characterized with our <A href="https://2010.igem.org/Team:Heidelberg/Project/miMeasure" class="external text" rel="nofollow">miMeasure</A> construct beforehand (data not shown). There is some evidence, that the principle works also with this therapeutic gene in HeLa cells (fig. 7). This is a first potential therapeutic approach applying <A href="https://2010.igem.org/Team:Heidelberg/Notebook/Methods#ELISA" class="external text" rel="nofollow">ELISA</A> for measurements.
</P><DIV class="center"><DIV class="thumb tnone"><DIV class="thumbinner" style="width:402px;"><DIV class="thumbcaption"><B>Figure 7: hAAT expression in relative units depending on binding site properties.</B> SV40 driven hAAT was fused to binding sites for miRsAg that was expressed from a co-transfected plasmid in HeLa cells.</DIV></DIV></DIV></DIV><P>It is obvious: different binding sites result in different knockdowns of gene expression. Some imperfect binding sites - e. g. single seed region - indicate even similar expression levels in accordance to the figures shown before. It can be stated, that the tuning idea seems to work for attempts varying in applied miRNAs, binding sites and reporter genes. 
The hAAT as a GOI is worth testing because it is mainly secreted in liver -our target tissue of choice. Efficient transduction can be accomplished by infection with <A href="https://2010.igem.org/Team:Heidelberg/Project/Capsid_Shuffling/Homology_Based" class="external text" rel="nofollow">selected viruses</A>. Dealing with hAAt intertwines our two approaches of specific gene therapy, therefor being a relevant field for future research.
</P><H3><SPAN class="mw-headline" id="In_Vivo_Validation"><I>In Vivo</I> Validation</SPAN></H3><P>The constructs were tested in two different backbones: pBS_U6 and pBS_H1. Both are in viral context, meaning that they contain inverted terminal repeats (ITRs). The constructs can be packed into the capsid of an adeno-associated virus (AAV). Those constructs we also chose for <A href="https://2010.igem.org/Team:Heidelberg/Notebook/Methods#Virus_Production" class="external text" rel="nofollow">virus production</A> to infect cells even more efficiently as compared to transfections. Because of the significant data, we decided to inject the viruses into mice to see the tuning effect also <I><A href="https://2010.igem.org/Team:Heidelberg/Project/Mouse_Infection" class="external text" rel="nofollow">in vivo</A></I>. The pBS_H1 construct should be preferred for mice injections since the expressed synthetic shRNA miR against human alpha-1-antitrypsine (shhAAT) is cytotoxic in higher concentrations. The pBS_H1 backbone leads to moderate expression ranges, still obviously showing the tuning effect.
</P><H3><SPAN class="mw-headline" id="Modeling">Modeling</SPAN></H3><P>After creating a binding site library and testing the miRNA-binding site interaction <I>in vitro</I>, we were able to compute an <A href="https://2010.igem.org/Team:Heidelberg/Modeling/miGUI" class="external text" rel="nofollow"><I>in silico</I> model</A> based on a machine learning approach to predict knockdown efficiencies. A more detailed description of the different binding sites, we characterized can be found in our <A href="https://2010.igem.org/Team:Heidelberg/Project/miMeasure" class="external text" rel="nofollow">measurements</A> page.
</P><H2><SPAN class="mw-headline" id="Methods">Methods</SPAN></H2><H3><SPAN class="mw-headline" id="miTuner:_Expression_Fine-Tuning_by_Synthetic_miRNAs_2">miTuner: Expression Fine-Tuning by Synthetic miRNAs</SPAN></H3><P>The miTuner was <A href="https://2010.igem.org/3A_Assembly" class="external text" rel="nofollow">assembled</A> out of different <A href="https://2010.igem.org/Team:Heidelberg/Parts" class="external text" rel="nofollow">parts</A>. Cloning was done following <A href="https://2010.igem.org/Team:Heidelberg/Notebook/Methods#Cloning" class="external text" rel="nofollow">standard protocols</A>.</P><P>Regulation of gene expression can be achieved by fusing miRNA binding sites right behind a GOI. In case a referring shRNA miR is expressed, the GOI is knocked down. Strength of regulation thereby depends on binding site properties. We are able to tune gene expression linearly over a broad range. This is a first proof of principle for various miRNA-mRNA interactions <I>in vitro</I>. Therefore, we transfected <A href="https://2010.igem.org/Team:Heidelberg/Notebook/Material#Cell_lines" class="external text" rel="nofollow">HeLa cells</A> in principle with our <A href="http://partsregistry.org/Part:BBa_K337036" class="external text" rel="nofollow">pSMB_miTuner Plasmid HD3</A>. It turned out, that there was no obvious effect of different binding sites on reporter gene expression (data not shown). We assume that the RSV driving the shRNA miR is too weak for tight regulation of the referring binding site behind the GOI which is driven by the very strong CMV promoter. Only if a sufficient amount of shRNA miR binds to its target, translation is significantly repressed. Thus, we expressed the shRNA miR from a separate plasmid which was always co-transfected with the original tuning construct. The reporter genes - i. e. hFluc and hRluc - were also expressed from separate plasmids to get a reference as well as a transfection control. Then, we conducted a <A href="https://2010.igem.org/Team:Heidelberg/Notebook/Methods#Dual_Luciferase_Assay" class="external text" rel="nofollow">Dual Luciferase Assay</A> for quantification of gene expression. 
</P><H3><SPAN class="mw-headline" id="On-_and_Off-Targeting">On- and Off-Targeting</SPAN></H3><P>Measurements were done in HeLa cells overexpressing miR122 from plasmid. Besides that, even endogenous miR122 levels were sufficient for off-targeting HuH cells (Fig. 4). A single perfect binding site leads to 95% knockdown, which seems to be maximum, since even a perfect binding site duplicate results in the same reporter gene expression. 
</P><DIV class="backtop"><A href="#top">↑</A></DIV><H2><SPAN class="mw-headline" id="References">References</SPAN></H2><UL><LI>Bartel DP: MicroRNAs: genomics, biogenesis, mechanism, and function. Cell. 2004 Jan 23;116(2):281-97.</LI><LI>Brennecke J, Stark A, Russell RB, Cohen SM. Principles of microRNA-target recognition. PLoS Biol. 2005 Mar;3(3):e85.</LI><LI>Brown BD, Naldini L.: Exploiting and antagonizing microRNA regulation for therapeutic and experimental applications. Nat Rev *Genet. 2009 Aug;10(8):578-8</LI><LI>Chekulaeva M, Filipowicz W.:Mechanisms of miRNA-mediated post-transcriptional regulation in animal cells. Curr Opin Cell Biol. 2009 Jun;21(3):452-60.</LI><LI>Eulalio, A., Huntzinger, E., and Izaurralde, E. (2008). Getting to the root of miRNA-mediated gene silencing. Cell 132, 9-14.
</LI><LI>Bartel, D.P. (2004). MicroRNAs: genomics, biogenesis, mechanism, and function. Cell 116, 281-297.</LI><LI>Jopling CL, Yi M, Lancaster AM, Lemon SM, Sarnow P.: Modulation of hepatitis C virus RNA abundance by a liver-specific MicroRNA. Science. 2005 Sep 2;309(5740):1577-81.</LI></UL><DIV class="backtop"><A href="#top">↑</A></DIV></DIV><DIV class="printfooter">
Retrieved from &quot;<A href="http://2010.igem.org/Team:Heidelberg/Project/miRNA_Kit">http://2010.igem.org/Team:Heidelberg/Project/miRNA_Kit</A>&quot;</DIV></DIV><DIV id="footer-box" class="noprint"><DIV id="footer"><UL id="f-list"><LI id="t-recentchanges"><A href="/Special:RecentChanges" title="Recent changes">Recent changes</A></LI><LI id="t-whatlinkshere"><A href="/Special:WhatLinksHere/Team:Heidelberg/Project/miRNA_Kit" title="List of all wiki pages that link here [j]" accesskey="j">What links here</A></LI><LI id="t-recentchangeslinked"><A href="/Special:RecentChangesLinked/Team:Heidelberg/Project/miRNA_Kit" title="Recent changes in pages linked from this page [k]" accesskey="k">Related changes</A></LI><LI id="t-specialpages"><A href="/Special:SpecialPages" title="List of all special pages [q]" accesskey="q">Special pages</A></LI><LI><A href="/Special:Preferences">My preferences</A></LI></UL></DIV><DIV id="footer"><UL id="f-list"><LI id="t-print"><A href="/wiki/index.php?title=Team:Heidelberg/Project/miRNA_Kit&amp;printable=yes" title="Printable version of this page [p]" accesskey="p">Printable version</A></LI><LI id="t-permalink"><A href="/wiki/index.php?title=Team:Heidelberg/Project/miRNA_Kit&amp;oldid=206886" title="Permanent link to this revision of the page">Permanent link</A></LI><LI id="privacy"><A href="/2010.igem.org:Privacy_policy" title="2010.igem.org:Privacy policy">Privacy policy</A></LI><LI id="disclaimer"><A href="/2010.igem.org:General_disclaimer" title="2010.igem.org:General disclaimer">Disclaimers</A></LI></UL></DIV></DIV></BODY></HTML>