<HTML xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en" dir="ltr">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki  ltr ns-0 ns-subject page-Team_Heidelberg_Project_Capsid_Shuffling_ViroBytes"><DIV id="globalWrapper"><DIV id="top-section"><DIV id="p-logo"><A href="/Main_Page" title="Main Page">&quot;
	    </A></DIV><DIV id="menubar" class="left-menu noprint"><UL><LI class="selected"><A href="/Team:Heidelberg/Project/Capsid_Shuffling/ViroBytes">Page               </A></LI><LI class="new"><A href="/wiki/index.php?title=Talk:Team:Heidelberg/Project/Capsid_Shuffling/ViroBytes&amp;action=edit&amp;redlink=1">Discussion               </A></LI><LI><A href="/wiki/index.php?title=Team:Heidelberg/Project/Capsid_Shuffling/ViroBytes&amp;action=edit">View source               </A></LI><LI><A href="/wiki/index.php?title=Team:Heidelberg/Project/Capsid_Shuffling/ViroBytes&amp;action=history">History               </A></LI><LI style="color:#808080;cursor:default">teams</LI></UL></DIV><DIV class="right-menu noprint" id="menubar"><UL><LI id="pt-login"><A href="/wiki/index.php?title=Special:UserLogin&amp;returnto=Team:Heidelberg/Project/Capsid_Shuffling/ViroBytes" title="You are encouraged to log in; however, it is not mandatory [o]" accesskey="o">Log in</A></LI></UL></DIV><DIV id="search-controls" class="noprint"><FORM action="/Special:Search" id="searchform"> </FORM></DIV></DIV><DIV id="content"><H1 class="firstHeading">Team:Heidelberg/Project/Capsid Shuffling/ViroBytes</H1><DIV id="bodyContent"><H3 id="siteSub" class="noprint">From 2010.igem.org</H3><DIV id="header"><DIV id="menu"><UL id="nav"><LI id="menu_home"><A class="aHome" href="https://2010.igem.org/Team:Heidelberg">Home</A></LI><LI id="menu_team"><A class="aTeam" href="https://2010.igem.org/Team:Heidelberg/Team">Team</A><TABLE class="menutable" cellpadding="0" cellspacing="0"><TBODY><TR><TD class="lefttdtopteam"><A class="aTeamOverview" href="https://2010.igem.org/Team:Heidelberg/Team">Overview</A><A class="aTeamMembers" href="https://2010.igem.org/Team:Heidelberg/Team/Members">Students</A><A class="aTeamMembers" href="https://2010.igem.org/Team:Heidelberg/Team/Members#Instructors">Instructors</A><A class="aTeamMembers" href="https://2010.igem.org/Team:Heidelberg/Team/Members#Advisors">Advisors</A><A class="aInstitutes" href="https://2010.igem.org/Team:Heidelberg/Team/Institutes">Institutes</A><A class="aGallery" href="https://2010.igem.org/Team:Heidelberg/Team/Gallery">Gallery</A><A class="aDance" href="https://2010.igem.org/Team:Heidelberg/midance">miDANCE</A><A class="amithanks" href="https://2010.igem.org/Team:Heidelberg/Team/miThanks">Acknowledgements</A></TD><TD class="righttdtop"> </TD></TR></TBODY></TABLE></LI><LI id="menu_project"><A class="aProject" href="https://2010.igem.org/Team:Heidelberg/Project">Project</A><TABLE class="menutable" cellpadding="0" cellspacing="0"><TBODY><TR><TD class="lefttdtopproject"><A class="aAbstract" href="https://2010.igem.org/Team:Heidelberg/Project">Abstract</A><A class="aIntroduction" href="https://2010.igem.org/Team:Heidelberg/Project/Introduction">Introduction</A><A class="aProject_miRNA_Kit" href="https://2010.igem.org/Team:Heidelberg/Project/miRNA_Kit">Synthetic miRNA Kit</A><A class="aPro_Capsid_Shuffling" href="https://2010.igem.org/Team:Heidelberg/Project/Capsid_Shuffling">Capsid Shuffling</A><A class="aPro_Homology" href="https://2010.igem.org/Team:Heidelberg/Project/Capsid_Shuffling/Homology_Based">    Homology Based</A><A class="aPro_Virobytes" href="https://2010.igem.org/Team:Heidelberg/Project/Capsid_Shuffling/ViroBytes">    ViroBytes</A><A class="aPro_miMeasure" href="https://2010.igem.org/Team:Heidelberg/Project/miMeasure">miMeasure</A><A class="aPro_Mouse" href="https://2010.igem.org/Team:Heidelberg/Project/Mouse_Infection"><I>in vivo</I> Study</A><A class="aPro_Summary" href="https://2010.igem.org/Team:Heidelberg/Project/Summary">Summary &amp; Outlook</A><A class="aPro_Ref" href="https://2010.igem.org/Team:Heidelberg/Project/References">References</A><A class="aPro_Ref" href="https://2010.igem.org/Team:Heidelberg/Project/Contributions">Contributions</A></TD><TD class="righttdtop"> </TD></TR></TBODY></TABLE></LI><LI id="menu_parts"><A class="aParts" href="https://2010.igem.org/Team:Heidelberg/Parts">Parts</A><TABLE class="menutable" cellpadding="0" cellspacing="0"><TBODY><TR><TD class="lefttdtop"><A class="aparts_ger" href="https://2010.igem.org/Team:Heidelberg/Parts">Parts</A><A class="aparts_char" href="https://2010.igem.org/Team:Heidelberg/Parts/Characterization">Characterization</A></TD><TD class="righttdtop"> </TD></TR></TBODY></TABLE></LI><LI id="menu_modeling"><A class="aModeling" href="https://2010.igem.org/Team:Heidelberg/Modeling">Modeling</A><TABLE class="menutable" cellpadding="0" cellspacing="0"><TBODY><TR><TD class="lefttdtop"><A class="amode_over" href="https://2010.igem.org/Team:Heidelberg/Modeling">Introduction</A><A class="amiGUI" href="https://2010.igem.org/Team:Heidelberg/Modeling/miGUI">miBEAT</A><A class="amirockdown" href="https://2010.igem.org/Team:Heidelberg/Modeling/miRockdown">miRockdown</A><A class="amibsdesigner" href="https://2010.igem.org/Team:Heidelberg/Modeling/miBSdesigner">miBS designer</A><A class="amispec" href="https://2010.igem.org/Team:Heidelberg/Modeling/miSpec">mUTING</A><A class="amodelset" href="https://2010.igem.org/Team:Heidelberg/Modeling/trainingset">Descriptions</A></TD><TD class="righttdtop"> </TD></TR></TBODY></TABLE></LI><LI id="menu_notebook"><A class="aNotebook" href="https://2010.igem.org/Team:Heidelberg/Notebook">Notebook</A><TABLE class="menutable" cellpadding="0" cellspacing="0"><TBODY><TR><TD class="lefttdtop"><A class="aNote_miRNA_kit" href="https://2010.igem.org/Team:Heidelberg/Notebook/miRNA_Kit">Synthetic miRNA Kit</A><A class="aNote_Capsid_Shuffling">Capsid Shuffling</A><A class="aNote_Homology" href="https://2010.igem.org/Team:Heidelberg/Notebook/Capsid_Shuffling/Homology_Based">    Homology Based</A><A class="aNote_Virobytes" href="https://2010.igem.org/Team:Heidelberg/Notebook/Capsid_Shuffling/ViroBytes">    ViroBytes</A><A class="aNote_miMeasure" href="https://2010.igem.org/Team:Heidelberg/Notebook/miMeasure">miMeasure</A><A class="aNote_BSdesign" href="https://2010.igem.org/Team:Heidelberg/Notebook/BSDesign">Binding Site Design </A><A class="aNote_Mouse" href="https://2010.igem.org/Team:Heidelberg/Notebook/Mouse_Infection"><I>in vivo</I> Study</A><A class="aNote_Modeling" href="https://2010.igem.org/Team:Heidelberg/Notebook/Modeling">Modeling</A><A class="aNote_Material" href="https://2010.igem.org/Team:Heidelberg/Notebook/Material">Materials</A><A class="aNote_Methods" href="https://2010.igem.org/Team:Heidelberg/Notebook/Methods">Methods</A><A class="aNote_Psycho" href="https://2010.igem.org/Team:Heidelberg/Human_Practices/Psychology/Notebook">Psychological Survey</A><A class="aNote_Philo" href="https://2010.igem.org/Team:Heidelberg/Human_Practices/Philosophy/Part2">Philosophical Reflection</A><A class="aNote_Biosafety" href="https://2010.igem.org/Team:Heidelberg/Notebook/Biosafety">Biosafety</A></TD><TD class="righttdtop"> </TD></TR></TBODY></TABLE></LI><LI id="menu_human_practices"><A class="aHuman_Practices" href="https://2010.igem.org/Team:Heidelberg/Human Practices">Human Practices</A><TABLE class="menutable" cellpadding="0" cellspacing="0"><TBODY><TR><TD class="lefttdtop"><A class="aPsycho" href="https://2010.igem.org/Team:Heidelberg/Human Practices/Psychology">Psychological Survey</A><A class="aPhilo" href="https://2010.igem.org/Team:Heidelberg/Human Practices/Philosophy">Philosophical Reflection</A></TD><TD class="righttdtop"> </TD></TR></TBODY></TABLE></LI><LI id="menu_support"><A class="aSupport" href="https://2010.igem.org/Team:Heidelberg/Support">Sponsors</A></LI></UL></DIV></DIV><H1><SPAN class="mw-headline" id="ViroBytes">ViroBytes</SPAN></H1><H2><SPAN class="mw-headline" id="Introduction">Introduction</SPAN></H2><P>ViroBytes is a modified <A href="https://2009.igem.org/Team:Alberta" class="external text" rel="nofollow">BioBytes</A> procedure for &quot;rational shuffling&quot; of capsid genes from natural isolates of Adeno-Associated Virus. Main motivation for the new protocol is unsatisfactory incorporation of certain serotypes (and especially certain parts of AAV capsid regions eg. from AAV5) using conventional shuffling strategies (<A href="https://2010.igem.org/Team:Heidelberg/Project/References#Capsid_Shuffling" class="external text" rel="nofollow">Grimm et al., 2008</A>). Application of magnetic beads for controlled assembly and the principle of BioByte formation and annealing persists. Sticky overhangs are used for selective combination of the bytes but different method is used for the production of individual ViroBytes.
</P><P>AAV serotypes 1,2,5,6,8 and 9 were selected as suitable candidates for fragmentation due to their exceptional individual properties. The analysis of Cap gene sequences revealed multiple homology regions which were then used for rational fragment formation. Total number of fragments per Cap gene is eight in our case and all fragments have similar length around ~250bp to assure similar behaviour in the ligation reaction. The Bytes are created and amplified by High-Fidelity PCR using Cap- and FragmentX-specific primers. These <A href="https://2010.igem.org/Team:Heidelberg/Notebook/Material/Primer#ViroBytes_primers" class="external text" rel="nofollow">primers</A> contain flanking regions with recognition sequence for type II restriction enzyme Bsa1. Precise positioning in front of and behind the homology regions at the ends of each fragmetn ensures that the enzyme forms unique sticky ends with 4nt overhangs. This procedure allows the user to avoid arduous design of overhangs with incorporated uracils further used for USER<SUP>TM</SUP> digestion. Also for the purposes of shuffling, the homology regions need to be exploited in order to avoid frame shifts which cannot be easily accomplished using standard BioByte protocol.
</P><DIV class="center"><DIV class="thumb tnone"><DIV class="thumbinner" style="width:352px;"><DIV class="thumbcaption">ViroBytes Assembly Scheme</DIV></DIV></DIV></DIV><H2><SPAN class="mw-headline" id="Important_facts">Important facts</SPAN></H2><P>Shuffling of several (six in our case) wild-type AAV serotypes unfortunately does not allow for standardization of the Virobyte fragments. We use type II restriction enzyme Bsa1 for creating the sticky ends instead. Incorporation of the recognition sequence of Bsa1 at both ends of the ViroBytes allows for higher specificity and thus prevents non-neighbouring fragments from ligating. The homology regions within which the Bsa1 enzyme cuts provide possibility for fragment shuffling without creating sequence frameshifts or deletions leading to nonviable chimeric isolates.
</P><P>BsaI recognition and cutting sequence:
</P><PRE>  5'...GGTCTCN/NNNN...3'
  3'...CCAGAGNNNNN/...5'
</PRE><P>hence the overhang created at each fragment ending: 5'-NNNN
</P><P>Homology regions employed in our Virobyte approach were selected as follows:
</P><P>between Anchor and Fragment1 (&quot;Start overhang&quot;): 
</P><PRE>  5'...ATGG...3'
  3'...TACC...5'
</PRE><P>between Fragment1 and Fragment2 (bp 279-303 relative to AAV1 Cap):
</P><PRE>  5'...GGAG...3'
  3'...CCTC...5'
</PRE><P>between Fragment2 and Fragment3 (bp 688-716 ,AAV1):
</P><PRE>  5'...GACA...3'
  3'...CTGT...5'
</PRE><P>between Fragment3 and Fragment4 (bp 1013-1016 ,AAV1):
</P><PRE>  5'...AACC...3'
  3'...TTGG...5'
</PRE><P>between Fragment4 and Fragment5 (bp 1282-1285 ,AAV1):
</P><PRE>  5'...CAGC...3'
  3'...GTCG...5'
</PRE><P>between Fragment5 and Fragment6 (bp 1444-1447 ,AAV1):
</P><PRE>  5'...AACT...3'
  3'...TTGA...5'
</PRE><P>between Fragment6 and Fragment7 (bp 1740-1743 ,AAV1):
</P><PRE>  5'...GTGG...3'
  3'...CACC...5'
</PRE><P>between Fragment7 and Fragment8 (bp 2040-2043 ,AAV1):
</P><PRE>  5'...TCAC...3'
  3'...AGTG...5'
</PRE><H2><SPAN class="mw-headline" id="ViroByte_design">ViroByte design</SPAN></H2><H3><SPAN class="mw-headline" id="Anchor">Anchor</SPAN></H3><P>Anchor for ViroByte assembly was designed according to <A href="https://2009.igem.org/Team:Alberta/DNAanchor" class="external text" rel="nofollow">BioByte Anchor System</A>. We used 5'-biotinylated oligos which were annealed with Anchor_complementary sequences (see <A href="https://2010.igem.org/Team:Heidelberg/Project/Capsid_Shuffling/ViroBytes#Protocols" class="external text" rel="nofollow">Protocols</A>). The final Anchor construct contains important restriction sites as well as the VirByte_for sequence used for final PCR amplification. First recognition site is HindIII which we used for separating the beads from the assembled DNA before PCR. Second restriction sequence corresponds to PacI which is a common non-cutter within all AAV serotypes used and can be emplyed for further ligation into an appropriate vector. Most importantly the 3'-end forms a sticky end compatible with the so-called &quot;start overhang&quot; of all fragments<B>1</B>.
</P><P>Annotated sequence of anchor oligos:
</P><P>Anchor:
</P><P>5'-Biotin-tctctctctctctct<I>aagctt</I><B>gtctgagtgactagcattcg</B><I><B>ttaattaa</B></I>c+4nt(cap specific)
</P><UL><LI>first 15nt form ssDNA linker (improves efficiency of Anchor binding onto Steptavidin-coated magnetic beads)
</LI></UL><UL><LI><I>italicized</I> is HindIII recognition site
</LI></UL><UL><LI><B>bold</B> is amplification (VirByte_for) sequence
</LI></UL><UL><LI><I><B>bold italicized</B></I> is PacI recognition site
</LI></UL><P>Anchor complement:
</P><P>5'-g<I><B>ttaattaa</B></I><B>cgaatgctagtcactcagac</B><I>aagctt</I></P><UL><LI> reverse complimentary strand to Anchors
</LI></UL><P>
Final Anchor construct used:
</P><PRE>   5'-Biotin-tctctctctctctctaagcttgtctgagtgactagcattcgttaattaac+4nt-3'
                         3'-ttcgaacagactcactgatcgtaagcaattaattg-5' 
</PRE><H3><SPAN class="mw-headline" id="Primers">Primers</SPAN></H3><P>Fragment X forward <A href="https://2010.igem.org/Team:Heidelberg/Notebook/Material/Primer#ViroBytes_primers" class="external text" rel="nofollow">primers</A>:
</P><P>5'-gctac<B>ggtctc</B>aNNNN+~20-25nt Cap specific
</P><UL><LI><B>bold</B> is BsaI recognition sequence
</LI></UL><UL><LI> NNNN = cap specific homology sequence from all cap genes (also cutting sequence of BsaI) forming the 5'-overhang
</LI></UL><P>
Fragment X reverse primers:
</P><P>5'-gctac<B>ggtctc</B>gNNNN
</P><UL><LI><B>bold</B> is BsaI recognition sequence
</LI></UL><H2><SPAN class="mw-headline" id="Protocols">Protocols</SPAN></H2><H3><SPAN class="mw-headline" id="ViroByte_construction"><B>ViroByte construction</B></SPAN></H3><UL><LI>perform ViroByte Phusion HiFi Hot Start PCR to obtain desired fragments with complementary ends
</LI><LI>98°C 30s     1x
</LI><LI>98°C 10s
</LI><LI>72°C 45s     35x
</LI><LI>72°C 10min   1x
</LI><LI>4°C hold 
</LI></UL><UL><LI>digestion with Bsa1 - leave digesting longer than in standard protocol (at least ~3hrs)
</LI></UL><H3><SPAN class="mw-headline" id="Anchor_preparation"><B>Anchor preparation</B></SPAN></H3><UL><LI>Mix 10ul of Anchor 5 (100uM stock) with 50ul of Anchor 12689 (100ul stock); add 1.08ml of nuclease free water; mix gently
</LI></UL><UL><LI>heat on the block to 95degrees for 2 minutes and let cool slowly to room temperature
</LI></UL><H3><SPAN class="mw-headline" id="Anchor_docking"><B>Anchor docking</B></SPAN></H3><UL><LI>after washing the beads and aspiration of the cleared solution the beads are ready for adding of the Anchor 
</LI></UL><UL><LI>add 40ul of the Anchor solution (~200ng/ul) 
</LI></UL><UL><LI>resuspend the beads and let incubate at room temperature for 15-20 minutes with occasional flicking in order to maintain the homogeneity of the solution 
</LI></UL><UL><LI>applying magnet, wash twice with 80ul of wash/binding buffer (0.5 M NaCl; 20 mM Tris HCl (pH 7.5); 1 mM EDTA)  and once with 80ul of 1X T4 ligase buffer 
</LI></UL><H3><SPAN class="mw-headline" id="BioByte-like_assembly"><B>BioByte-like assembly</B></SPAN></H3><UL><LI>Bead preparation 
</LI></UL><UL><LI>Quickly vortex beads stock solution to obtain homogeneous suspension 
</LI></UL><UL><LI>Aliquot 40 µl of the beads into 1.5ml Eppendorf tubes - gently tap the tubes to collect all the beads at the bottom.
</LI></UL><UL><LI>Place the tubes on the magnetic rack (or place a magnet on the side of each tube) and wait for the beads to be cleared out from the solution 
</LI></UL><UL><LI>Carefully discard the supernatant whilst beads are still attached to the side wall 
</LI></UL><UL><LI>Resuspend beads in 80ul of wash/binding buffer by gentle flicking (vortexing at high speed is not recommended) 
</LI></UL><UL><LI>Wash the beads once with 80 µl of 1X T4 Ligase buffer, before addition of ligation premix.
</LI></UL><UL><LI>Add the DNA fragments using the ligation premix: 1) 25 µl of Quick Ligase (2X buffer); 2) Amount of DNA corresponding to 200 ng. The optimal DNA concentration for the ViroByte fragments assembly was experimentally established and was in range of 200 ng per fragment. Using higher and lower DNA concentrations did not give any positive assembly results; 3) Fill up with water to obtain the 50 µl total volume.
</LI></UL><UL><LI>For the ligation procedure the Quick Ligase was used that showed to be more efficient during the experimental procedure in comparison to the ordinary T4 Ligase.  Optimal ligation time for the Virobytes ligation was between 15-25 minutes.
</LI></UL><UL><LI>After the last DNA fragment ligation wash the beads once with Wash/Binding buffer.  Moreover the DNA should be cleaved from the beads by using HindIII restriction enzyme (or the other one depending on your designed cleavage site). For the restriction a standard protocol for the DNA digestion should be used.
</LI></UL><H2><SPAN class="mw-headline" id="Notebook">Notebook</SPAN></H2><P>For further reference please find our lab notebook <A href="https://2010.igem.org/Team:Heidelberg/Notebook/Capsid_Shuffling/ViroBytes" class="external text" rel="nofollow">here</A>.
</P><H2><SPAN class="mw-headline" id="References">References</SPAN></H2><P>Grimm D, Lee JS, Wang L, Desai T, Akache B, Storm TA, Kay MA. In Vitro and In Vivo Gene Therapy Vector Evolution via Multispecies Interbreeding and Retargeting of Adeno-Associated Viruses. J Virol 2008;82:5887–5911.
</P><TABLE id="toc" class="toc"><TBODY><TR><TD><DIV id="toctitle"><H2>Contents</H2></DIV><UL><LI class="toclevel-1 tocsection-1"><A href="#ViroBytes"><SPAN class="tocnumber">1</SPAN><SPAN class="toctext">ViroBytes</SPAN></A></LI><LI class="toclevel-2 tocsection-2"><A href="#Introduction"><SPAN class="tocnumber">1.1</SPAN><SPAN class="toctext">Introduction</SPAN></A></LI><LI class="toclevel-2 tocsection-3"><A href="#Important_facts"><SPAN class="tocnumber">1.2</SPAN><SPAN class="toctext">Important facts</SPAN></A></LI><LI class="toclevel-2 tocsection-4"><A href="#ViroByte_design"><SPAN class="tocnumber">1.3</SPAN><SPAN class="toctext">ViroByte design</SPAN></A></LI><LI class="toclevel-3 tocsection-5"><A href="#Anchor"><SPAN class="tocnumber">1.3.1</SPAN><SPAN class="toctext">Anchor</SPAN></A></LI><LI class="toclevel-3 tocsection-6"><A href="#Primers"><SPAN class="tocnumber">1.3.2</SPAN><SPAN class="toctext">Primers</SPAN></A></LI></UL><LI class="toclevel-2 tocsection-7"><A href="#Protocols"><SPAN class="tocnumber">1.4</SPAN><SPAN class="toctext">Protocols</SPAN></A></LI><LI class="toclevel-3 tocsection-8"><A href="#ViroByte_construction"><SPAN class="tocnumber">1.4.1</SPAN><SPAN class="toctext">ViroByte construction</SPAN></A></LI><LI class="toclevel-3 tocsection-9"><A href="#Anchor_preparation"><SPAN class="tocnumber">1.4.2</SPAN><SPAN class="toctext">Anchor preparation</SPAN></A></LI><LI class="toclevel-3 tocsection-10"><A href="#Anchor_docking"><SPAN class="tocnumber">1.4.3</SPAN><SPAN class="toctext">Anchor docking</SPAN></A></LI><LI class="toclevel-3 tocsection-11"><A href="#BioByte-like_assembly"><SPAN class="tocnumber">1.4.4</SPAN><SPAN class="toctext">BioByte-like assembly</SPAN></A></LI><LI class="toclevel-2 tocsection-12"><A href="#Notebook"><SPAN class="tocnumber">1.5</SPAN><SPAN class="toctext">Notebook</SPAN></A></LI><LI class="toclevel-2 tocsection-13"><A href="#References"><SPAN class="tocnumber">1.6</SPAN><SPAN class="toctext">References</SPAN></A></LI></TD></TR></TBODY></TABLE><P>  </P><DIV class="printfooter">
Retrieved from &quot;<A href="http://2010.igem.org/Team:Heidelberg/Project/Capsid_Shuffling/ViroBytes">http://2010.igem.org/Team:Heidelberg/Project/Capsid_Shuffling/ViroBytes</A>&quot;</DIV></DIV></DIV><DIV id="footer-box" class="noprint"><DIV id="footer"><UL id="f-list"><LI id="t-recentchanges"><A href="/Special:RecentChanges" title="Recent changes">Recent changes</A></LI><LI id="t-whatlinkshere"><A href="/Special:WhatLinksHere/Team:Heidelberg/Project/Capsid_Shuffling/ViroBytes" title="List of all wiki pages that link here [j]" accesskey="j">What links here</A></LI><LI id="t-recentchangeslinked"><A href="/Special:RecentChangesLinked/Team:Heidelberg/Project/Capsid_Shuffling/ViroBytes" title="Recent changes in pages linked from this page [k]" accesskey="k">Related changes</A></LI><LI id="t-specialpages"><A href="/Special:SpecialPages" title="List of all special pages [q]" accesskey="q">Special pages</A></LI><LI><A href="/Special:Preferences">My preferences</A></LI></UL></DIV><DIV id="footer"><UL id="f-list"><LI id="t-print"><A href="/wiki/index.php?title=Team:Heidelberg/Project/Capsid_Shuffling/ViroBytes&amp;printable=yes" title="Printable version of this page [p]" accesskey="p">Printable version</A></LI><LI id="t-permalink"><A href="/wiki/index.php?title=Team:Heidelberg/Project/Capsid_Shuffling/ViroBytes&amp;oldid=208395" title="Permanent link to this revision of the page">Permanent link</A></LI><LI id="privacy"><A href="/2010.igem.org:Privacy_policy" title="2010.igem.org:Privacy policy">Privacy policy</A></LI><LI id="disclaimer"><A href="/2010.igem.org:General_disclaimer" title="2010.igem.org:General disclaimer">Disclaimers</A></LI></UL></DIV></DIV></DIV></BODY></HTML>