"
Page
Discussion
View source
History
teams
Log in
 
Team:Heidelberg/Project/Summary
From 2010.igem.org
Home
Team
Overview
Students
Instructors
Advisors
Institutes
Gallery
miDANCE
Acknowledgements
 
Project
Abstract
Introduction
Synthetic miRNA Kit
Capsid Shuffling
    Homology Based
    ViroBytes
miMeasure
in vivo Study
Summary & Outlook
References
Contributions
 
Parts
Parts
Characterization
 
Modeling
Introduction
miBEAT
miRockdown
miBS designer
mUTING
Descriptions
 
Notebook
Synthetic miRNA Kit
Capsid Shuffling
    Homology Based
    ViroBytes
miMeasure
Binding Site Design
in vivo Study
Modeling
Materials
Methods
Psychological Survey
Philosophical Reflection
Biosafety
 
Human Practices
Psychological Survey
Philosophical Reflection
 
Sponsors
Contents
1 Summary
1.1 miTuner
1.2 Capsid shuffling
1.3 Modeling
1.4 Outlook
Summary
miTuner
We have developed a novel method for miRNA-based gene expression tuning in mammalian cells. We show that the miTUNER method allows tuning of protein expression in fine intervals in vivo and in vitro. This enables us to create libraries of protein expression vectors with different strengths without exchanging the promoter. This is highly useful for the usage of certain promoters for an optimal protein expression.
We furthermore demonstrate the ability of the miTuner approach for off-targeting respectively for partial, selective down-regulation of protein expression in vivo and in vitro. The potential of miTuner for on-targeting respectively partial, selective up-regulation has been proven in vitro, in vivo measurements are currently undertaken and will be presented at the Jamboree. To the best of our knowledge, this work is the first demonstration of a miRNA-based gene expression tuner, and it is moreover the first implementation of on-targeting with miRNA-based gene expression control systems.
An RFC describing the miTUNER method has been written (miTuner - a kit for microRNA based gene expression tuning in mammalian cells). Moreover, we describe a new measurement standard (miMeasure – a standard for miRNA binding site characterization in mammalian cells) in a second RFC . We are currently waiting for the assignment of numbers for the submission of these two RFCs.
Capsid shuffling
We have developed a standardized and fast approach towards the creation of AAV-based. We adopted two established methods for the shuffling of capsid genes – homology based shuffling by DNaseI digestion and self-primed PCR. Additionally we introduce ViroBytes, a random assembly protocol based on rationally designed capsid parts. These methods allow for the creation of libraries of randomized synthetic viruses and the consequent screening for novel viruses with improved efficiency and tissue specifity. We have achieved exceptionally selective tissue-specific targeting in vitro and in vivo with hepatocyte specific delivery vectors.
Modeling
Our suite of experimental contributions is complement by a powerful range of, accessible through the miBEAT GUI. This tool combines and connects the output of different models and scripts and then generates a suitable miTuner construct that expresses the gene of interest, miGENE, up to the desired level. miBEAT consists of three subparts; miRockdown, miBS designer and mUTING.
miRockdown is the subpart which contains two computational models that work on different concepts: Neural Network and Fuzzy Logic plus the experimentally obtained data. The models are sequentially associated with a script based on Target Scan algorithm. miRockdown takes as an input the desired knockdown percentage and the sequence of shRNAmir and gives out binding site parameters that are then compared with model predictions to finally generate the appropriate binding site.
miBS designer is available as a stand alone for generating customized binding sites, but a modified version of it is also a part of miBEAT, in charge of generating more than 2000 different binding sites for every miRNA sequences, following more than 135 combinations of regions.
mUTING provides the tissue specific targeting function to the GUI. It uses literature data for miRNA expression in various tissues and can output miRNA binding sites that could be used to differentiate between target and off target tissues.
Apart from all of these tools, our team also developed two independent models:
The Neural Network Model takes inspiration in the biological nervous system to predict its results. It is the appropriate strategy to model complex processes and it is able to learn from experience. Even if the experimental data were not enough to fully train the model, the results agree with the experimental values and the model was able to determine the importance of the bulge size for the knockdown.
The Fuzzy Logic Model is combining the strength of intuitive integration of prior knowledge with a sophisticated Global Genetic optimization Algorithm.
Outlook
We see great potential in the combined usage of synthetic promoters (as proposed by the Heidelberg iGEM team 2009), with miRNA-based post-transcriptional gene expression control and with cell-specific gene delivery. The 3-fold usage of selective gene expression control will allow for very tight coupling of gene expression to target cells, e.g. to cancer cells.
Moreover, the possibility of RNA-based logic gates provides an attractive option for the design and fine-tuning of synthetic networks. As we have demonstrated, miTuner allows for the usage of synthetic miRNAs. This opens up the perspective of engineering orthogonal networks which can be run in parallel to cellular calculation processes. Moreover, the quantitative understanding provided by our in silico tools and our standardized miRNA measurement procedures will be useful for the further investigation of the foundations of the miRNA regulation machinery.
   
Retrieved from " http://2010.igem.org/Team:Heidelberg/Project/Summary "
Recent changes
What links here
Related changes
Special pages
My preferences
Printable version
Permanent link
Privacy policy
Disclaimers
