<HTML xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en" dir="ltr">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki  ltr ns-0 ns-subject page-Team_Yale_Project"><DIV id="globalWrapper"><DIV id="top-section"><DIV id="p-logo"><A href="/Main_Page" title="Main Page">&quot;
	    </A></DIV><DIV id="menubar" class="left-menu noprint"><UL><LI class="selected"><A href="/Team:Yale/Project">Page               </A></LI><LI class="new"><A href="/wiki/index.php?title=Talk:Team:Yale/Project&amp;action=edit&amp;redlink=1">Discussion               </A></LI><LI><A href="/wiki/index.php?title=Team:Yale/Project&amp;action=edit">View source               </A></LI><LI><A href="/wiki/index.php?title=Team:Yale/Project&amp;action=history">History               </A></LI><LI style="color:white;cursor:default">teams</LI></UL></DIV><DIV class="right-menu noprint" id="menubar"><UL><LI id="pt-login"><A href="/wiki/index.php?title=Special:UserLogin&amp;returnto=Team:Yale/Project" title="You are encouraged to log in; however, it is not mandatory [o]" accesskey="o">Log in</A></LI></UL></DIV><DIV id="search-controls" class="noprint"><FORM action="/Special:Search" id="searchform"> </FORM></DIV></DIV><DIV id="content"><H1 class="firstHeading">Team:Yale/Project</H1><DIV id="bodyContent"><H3 id="siteSub" class="noprint">From 2014.igem.org</H3><DIV class="navmenubkg"><DIV id="navmenubox"><UL id="nav"><LI style="width:120px;"><A href="https://2014.igem.org/Team:Yale" title="Home">Home</A></LI><LI style="width:120px;"><A href="https://2014.igem.org/Team:Yale/Project" title="Project">Project</A></LI><LI style="margin-left:14.5px"><A href="https://2014.igem.org/Team:Yale/Project" title="Overview">Overview</A></LI><LI style="width:180px"><A href="https://2014.igem.org/Team:Yale/MaterialsMethods" title="Coating">Materials &amp; Methods</A></LI><LI><A href="https://2014.igem.org/Team:Yale/Results" title="Results">Results</A></LI><LI><A href="https://2014.igem.org/Team:Yale/Project/modeling" title="Modeling">Modeling</A></LI><LI><A href="https://2014.igem.org/Team:Yale/Parts" title="Parts">Submitted Parts</A></LI></UL></DIV><LI style="width:120px;"><A href="https://2014.igem.org/Team:Yale/Team" title="OurLab">About</A></LI><LI style="margin-left:141px;"><A href="https://2014.igem.org/Team:Yale/Team" title="Team">Team</A></LI><LI style="width:180px"><A href="https://igem.org/Team.cgi?year=2014&amp;team_name=Yale" title="Official">Official Team Profile</A></LI><LI style="width:180px;"><A href="https://2014.igem.org/Team:Yale/Acknowledgments" title="Acknowledgments">Acknowledgments</A></LI><LI style="width:140px;"><A href="https://2014.igem.org/Team:Yale/Safety" title="Safety">Safety</A></LI><LI style="width:140px;"><A href="https://2014.igem.org/Team:Yale/Contact" title="Contact">Contact Us</A></LI></DIV><LI style="width:120px;"><A href="https://2014.igem.org/Team:Yale/Outreach" title="Outreach">Outreach</A></LI><LI style="width:120px;"><A href="https://2014.igem.org/Team:Yale/Interlab" title="Interlab">Interlab</A></LI><LI style="width:120px;"><A href="https://2014.igem.org/Team:Yale/Notebook">Notebook</A></LI><LI style="width:150px;"><A href="https://2014.igem.org/Team:Yale/Achievements">Achievements</A></LI></DIV></DIV></DIV><H1 style="margin-top:22px; font-size:48px;">AMPersand: An Anti-Microbial Peptide Coating</H1><DIV class="tinycall"><H1>The Problem</H1></DIV><DIV class="well"><P>
A <STRONG><I> biofilm</I></STRONG> is a community of bacteria attached to a surface that exhibits high resistance to antibiotics and human immunity. Biofilm formation poses a serious threat to the medical and shipping industries in the following ways:
</P><P><OL type="I"><LI><STRONG>Medical Industry</STRONG></LI><LI>Protein adsorption, cell-adhesion, and subsequent biofilm formation have been found to lead to failure of medical implants and infection in patients.<SUP>1</SUP></LI><LI><STRONG> Shipping Industry </STRONG></LI><LI>  Government and industry spend upwards of $5.7 billion annually in the control of marine biofouling. High levels of biofouling result in increased drag and the subsequent loss of hydrodynamic performance.<SUP>2</SUP></LI></OL></P><P><CENTER><I><STRONG>Figure 1.</STRONG><STRONG>(A)</STRONG> Biofilm formation leads to failure of implanted cardiac devices and heart valve infection. <STRONG>(B)</STRONG> Biofilm formation on the hulls of ships leads to loss of optimal hydrodynamic performance and structural failures.</I></CENTER></P><DIV class="tinycall"><H1>Our Solution</H1></DIV><DIV class="well"><P>
To address this issue, we aimed to develop an anti-microbial adhesive peptide composed of two components. We envision these domains can be modulated to suit a variety of functional adhesive applications: 
</P><P><OL type="I"><LI><STRONG>Component 1: Adhesive Domain</STRONG></LI><LI>
Because biofilm formation affects both organic and inorganic substrates, the anti-biofilm coating should show strong adhesion to a variety of surfaces. Mussel adhesive proteins (MAPs), which are secreted by the mussel to help it anchor and survive in the harsh conditions of the intertidal zone, would be ideal for this application. MAP adhesion has been well-characterized and has been investigated in biomimetic adhesive applications in the past. We intend to broaden the scope of their application by looking at their inclusion in the first anti-biofilm adhesive recombinant protein.<SUP>3</SUP> The functional residue in MAPs is L-3,4-dihydroxyphenylalanine (L-DOPA), which is generated by post-translational modification of tyrosine with tyrosinase. Since L-DOPA is a non-standard amino acid, it cannot be incorporated by standard translation systems. However, we intend to be the first group to use a genetically recoded organism (GRO) to incorporate L-DOPA in-situ, eliminating the need for any post-translational modification.</LI><LI><STRONG>Component 2: Anti-Microbial Domain</STRONG></LI><LI>
As our anti-microbial domain, we selected LL-37, a member of the cathelicidin family of peptides, due to the potency of its lipid bilayer disruption by toroidal pore formation. Because this peptide is toxic to the <I>E. coli</I> in which we intend to produce it, we designed a controlled, inducible system that limits basal expression. A novel T7 riboregulation system that controls expression at both the transcriptional and translational levels was designed. This improved system is a precise synthetic switch for the expression of cytotoxic substances.<SUP>4,5</SUP></LI><LI><STRONG>Addressing Environmental Concerns</STRONG></LI><LI> Concerns of environmental toxicity often arise in materials being investigated for anti-fouling activity such as copper paints and Muntz metal. Therefore, we set out to develop an anti-fouling coating with strong adhesive activity to limit leachants into the environment. Additionally, the selection of a MAP, found in a biological organism, as our adhesive domain is crucial to maintaining the soundness of our product's eco-friendliness. 
		</LI></OL></P></DIV><H2 style="border-bottom:none">Video Walkthrough of our Project</H2><DIV class="tinycall"><H1>Project Goals</H1></DIV><DIV class="well"><P><OL type="I"><LI><STRONG>Control Expression of Anti-Microbial Peptides Using an Improved T7 Riboregulation System:</STRONG></LI><LI>
		
		Since we intend to synthesize an anti-microbial peptide, it is possible that the peptide will be toxic to the <I>E. coli</I> used in our synthetic route. To improve our overall protein yield, we designed a plasmid with specific locks in place to control expression of the T7 RNA polymerase, an RNA polymerase from the T7 bacteriophage. When the T7 RNA polymerase is expressed, it can then specifically target the T7 promoter located in a different plasmid upstream of our coding sequence, initiating protein translation. The specific mechanism of our T7 riboregulation system is outlined in a section below.<SUP>6,7</SUP></LI><LI><STRONG>A Modular Anti-Microbial Construct based on Mussel Foot Protein:</STRONG></LI><LI>
		As our adhesive domain, we selected the mussel foot protein (mefp) consensus sequence mefp 1-mgfp 5-mefp-1, which was found to be effective in Lee <I>et al.</I>, 2008.<SUP>8</SUP> At the N-terminus, we included a twin Strep-FLAG tag, used in the purification and isolation of our construct and that can be readily cleaved. The  LL-37 antimicrobial peptide, which is short enough to be inserted via primer overhang, is linked via a 36 residue linker, which we believe is long enough not to engender any unforeseen structural interaction between our domains. On the other side of the foot protein, we included an sfGFP connected by a shorter linker, which will be used to assay presence and yield of construct. Using targeted primers, the construct can be amplified in its entirety, or only with the anti-microbial or GFP segment. Note that the entire construct was designed so that a variety of functional peptide domains can be substituted for LL-37 if desired. A diagram of our entire construct is presented below: </LI></OL></P><P><CENTER><I><STRONG>Figure 2.</STRONG> A diagram illustrating the components in our final construct. The black domain is our anti-microbial peptide, LL-37, while the blue domain represents the recombinant mussel foot protein adhesive component. All other components are labeled accordingly and restriction sites are highlighted to emphasize the modularity of each separate region.</I></CENTER></P><LI><STRONG>Characterize peptide's adhesion and anti-microbial properties:</STRONG></LI><LI> We intend to perform a number of assays to test the erosion resistance of our adhesive coating using an original apparatus designed to introduce erosion by laminar flow through a liquid bath. The specific tests that we investigated for adhesion testing are detailed in the <A href="https://2014.igem.org/Team:Yale/MaterialsMethods">Materials and Methods</A> section. 
</LI><LI> To assess the efficacy of our peptide in inhibiting biofilm formation, we intend to perform a minimum biofilm eradication concentration (MBEC) assay (Innovotech). Further information is provided in our <A href="https://2014.igem.org/Team:Yale/MaterialsMethods">Materials and Methods</A>  section. 
		
		</LI></DIV><DIV class="tinycall"><H1><A href="https://2014.igem.org/Team:Yale/MaterialsMethods">Materials and Methods</A></H1></DIV><DIV class="well"><P>
For a detailed description of our experimental design regarding the T7 expression system, anti-microbial peptide construct, and adhesion assays, see <A href="https://2014.igem.org/Team:Yale/MaterialsMethods">materials and methods.</A></P></DIV><DIV class="tinycall"><H1><A href="https://2014.igem.org/Team:Yale/Results">Results</A></H1></DIV><DIV class="well"><P>
For a detailed description of our results, see <A href="https://2014.igem.org/Team:Yale/Results">results.</A></P></DIV><DIV class="tinycall"><H1><A href="https://2014.igem.org/Team:Yale/Project/modeling">Modeling</A></H1></DIV><DIV class="well"><P>
We modeled theoretical bacterial population survival rates based on various points of anti-microbial peptide induction. See <A href="https://2014.igem.org/Team:Yale/Project/modeling">modeling.</A></P></DIV><DIV class="tinycall"><H1><A href="https://2014.igem.org/Team:Yale/Parts">Submitted Parts</A></H1></DIV><DIV class="well"><P>
Our collection of submitted biobricks consists of: </P><P><UL><LI>Mussel foot protein (MFP) 1-5-1 sequence [combination of Mytilus galloprovincialis Foot Protein 5 (Mgfp-5) and Mytilus Edulis Foot Protein 1 (Mefp-1)].</LI><LI>MFP with superfolder Green Fluorescence Protein (sfGFP).</LI><LI>MFP with our anti-microbial peptide, LL-37.</LI><LI>Entire construct of our anti-microbial adhesive peptide: 2XStrep_Flagtag--LL-37--Mussel Foot Protein--sfGFP.</LI><A href="https://2014.igem.org/Team:Yale/Parts">See here for more information on our Biobricks!</A></UL></P></DIV><DIV class="tinycall"><H1>Additional Background Information</H1></DIV><DIV class="well"><OL type="1"><LI><STRONG>Genomically Recoded <I>E. coli</I></STRONG><P>
Whole-genome recoding was reported for the first time in 2011 by a team including our advisor, Professor Farren Isaacs, when it described replacing all TAG codons in <I>E. coli</I> by TAA. <SUP>9</SUP>  Both TAG and TAA are stop codons (amber and ochre), but their transcripts are bound by different release factors (RF1 binds UAG and UAA, whereas RF2 binds UAA and UGA), not only making this recoding nonlethal but also allowing the deletion of RF1 (ΔprfA) from the recoded strain, given RF2’s response to both remaining stop codons. </P><P>
This genome-wide recoding was enabled by lambda-Red recombineering<SUP>10</SUP> and multiplex automated genome engineering. <SUP>11</SUP>  lambda-Red recombineering uses the lambda prophage’s protein Beta to mediate the homologous recombination of an exogenous single-stranded oligonucleotide, delivered into the cell by electroporation, with the host genome, by annealing that oligonucleotide to single-stranded genomic DNA exposed on the lagging strand of the replication fork. <SUP>12</SUP>  A 30-bp region of homology at either end of a sequence is sufficient to drive recombination of the entire strand, including mismatched regions, and mismatches thus incorporated are passed to progeny at high efficiency (up to 30%) in strains lacking methyl-directed mismatch repair (ΔmutS).<SUP>13</SUP></P><P>
Multiplex automated genome engineering (MAGE) using lambda-Red recombineering can then make many directed mutations across the host genome, as was necessary to replace all 314 TAG codons in the strain EcNR2 (<I>E. coli</I> MG1655 ΔmutS::cat Δ(ybhB-bioAB)::[λcI857 Δ(cro-ea59)::tetR-bla]).<SUP>9</SUP>  To allow recombineering, the lambda prophage was introduced into the host genome by P1 transduction. Because the probability of each target sequence being mutated in a given cycle is at most 30%, attempting to introduce all 314 mutations using a single pool of 314 mutagenic oligonucleotides would much sooner generate any of ~2314 other partially recoded strains before the fully recoded strain, and running enough MAGE cycles to obtain that strain would allow time for spontaneous point mutations to accumulate unchecked by the knocked-out mismatch repair system. To construct the strain we will use 32 separate strains were generated, each with a different recoded sector, and those sectors were assembled by hierarchical conjugation. The recoded strain was validated by sequencing, and its RF1 was deleted to free the TAG codon for reassignment to an orthogonal translation system.<SUP>14</SUP></P></LI><LI><STRONG>Orthogonal Translation Systems</STRONG><P>
An amino acid can be assigned to a particular codon by a two-part translation system: a tRNA with its anticodon loop complementary to the codon, and an aminoacyl synthetase (hereafter called a synthetase) able to charge this tRNA with the amino acid. This system is orthogonal to endogenous translation systems if and only if this synthetase aminoacylates only this tRNA and endogenous synthetases cannot aminoacylate this tRNA. Usually, an unnatural amino acid thus assigned should also be sufficiently dissimilar from naturally encoded amino acids that endogenous synthetases would not charge endogenous tRNAs with the unnatural amino acid, to avoid partially reassigning other codons and disrupting cell function.<SUP>15</SUP></P><P>
Many synthetases include a nonconserved loop specific to their associated tRNA’s anticodon, preventing them from binding a tRNA reassigned to CUA, but the archaeum Methanococcus jannaschii’s tyrosyl-tRNA synthetase (MjtRNAs) and its associated tRNA translation system still bind after the anticodon is mutated, and is an amber suppressor when transformed into <I>E. coli</I>.<SUP>16</SUP>  This MjtRNAs was still aminoacylated by endogenous synthetases, so its specificity for its synthetase was improved by random mutation at eleven positions not directly interacting with its synthetase, followed by negative selection against nonspecific acylation and positive selection for specific acylation. <SUP>17</SUP></P><P>This orthogonal system has since been engineered by directed evolution of its synthetase to incorporate nonstandard amino acids, in place of tyrosine. <SUP>18</SUP> Diverse synthetases were generated by site-directed mutagenesis of five residues in the amino acid-binding pocket, and screened by positive selection, for incorporation of the amino acid into a protein conferring antibiotic resistance, and a negative screen, against mutants resistant to the antibiotic even in the absence of the nonstandard amino acid (by treating colonies on a replica plate lacking the amino acid and then observing which were killed). Selected mutants were then recombined and subjected to further mutagenesis. </P></LI><LI><STRONG>An Improved T7 Expression System</STRONG><P> Our first challenge in making Ampersand was to develop a system that limits basal expression of our peptide, due to its toxicity to the host. Several expression systems have been designed in the past, including the T7 promoter and T7 RNA Polymerase system, to suppress protein expression. The current BL21 (DE3) strain is leaky due to the weak suppressing promoter <I>lacUV5</I> that drives T7 RNA polymerase in the DE3 strains. As a result, low levels of toxic protein are constitutively expressed, ultimately killing the host it was made in and in turn lowering the overall yield of the protein produced.</P><P> In order to reduce the expression of T7 RNA polymerase and create an efficient system for the expression of recombinant proteins in <I>E. coli</I> we chose to introduce two levels of regulation:</P></LI><LI><STRONG>Transcriptional Regulation:</STRONG> We will use pZE21 of the pZ system of vectors developed by Lutz and Buschard with the P<SUB>LlacO</SUB> promoter to inhibit the expression of T7 RNA polymerase.
</LI><LI><STRONG>Translational Regulation:</STRONG> We will use the artificial riboregulatory elements devised by Isaacs <I>et al.</I>. to restrict translation of the mRNA sequence encoding T7 RNA Polymerase.<SUP>19</SUP>  The cis-repressing RNA (crRNA) sequence will be inserted downstream of the promoter driving T7 RNA Polymerase and upstream of the ribosomal binding site (RBS). The crRNA is complimentary to the RBS and forms a stem loop at the 5’ end of the mRNA segment, blocking ribosomal docking and translation. A second promoter, P<SUB>LtetO</SUB> will express the trans-activating RNA (taRNA) capable of undergoing a linear-loop interaction that will expose the RBS and allow for translation of T7 RNA Polymerase. The ribo-regulated T7 RNA Polymerase will be ultimately incorporated into the genome of strain capable of high protein production. A second pZ plasmid will contain the gene of interest expressed by a T7 promoter. <P>
In summary, the benefits to this type of system are its robustness, portability, and efficiency in that it does not require the cell to expend more energy on the constitutive synthesis of another protein. We theorize that by utilizing these two levels of control we will be able to reduce the expression of T7 RNA polymerase and produce a system with zero basal expression of the gene of interest and permit stable cloning of toxic recombinant proteins.<CENTER><I><STRONG>Figure 3.</STRONG> This diagram illustrates the T7 riboregulation system developed to ensure low basal levels of protein expression.<SUP>19</SUP></I></CENTER></P></LI><LI><STRONG>Mussel Adhesion Proteins</STRONG><P>The ability for mussels to survive the harsh conditions of the intertidal zone is directly related to their capability to bind to both organic and inorganic surfaces that they come in contact with. As a result, these mussels have evolved to secrete “adhesion proteins” that enable formation of powerful bonds in an otherwise deleterious aqueous environment. The quantity known as “work of adhesion (W<SUB>A</SUB>)” amongst interfaces in an aqueous environment is significantly less favorable due to the water-surface interactions that must be outcompeted by the adhesive if binding is to occur.<SUP>10</SUP> Therefore, these mussel adhesion proteins offer a promising avenue towards enhanced bioadhesives that retain functionality in aqueous environments. </P><P>The structure responsible for the binding of mussels to these surfaces is known as the byssus, which is composed of a well-characterized assembly of approximately 25-30 proteins.<SUP>20</SUP> The byssus manifests itself as an outgrowth from the mussel’s “foot” composed of threads, which terminate in plaques deposited on the surface. Thus far, a total of 5 proteins, known as mussel foot proteins (mfp’s), were shown to be unique to the plaque. Of these 5 plaque proteins, mfp-3 and mfp-5 have elicited the most investigation for adhesive applications, as both are localized at the interface between the plaque and surface. Additionally, it was shown that all such mfp’s have a uniquely high concentration of tyrosine, which is post-translationally modified to 3,4-dihydroxy-L-phenylalanine (L-Dopa). Atomic force microscopy studies focusing on a single dopa residue showed that its interaction with a metal oxide surface involves a remarkably high strength, reversible coordination bond.<SUP>21</SUP> It has been shown that catechol oxidase enzymes present in mfp secretions are able to convert the catechol groups of L-dopa into reactive orthoquinone functionalities, which undergo irreversible covalent bonding with organic surfaces.<SUP>21,22</SUP> Ultimately, the high degree of surface affinity such dopa-containing peptides possess makes them novel tools in the development of biomimetic adhesive peptides.</P><DIV class="tinycall"><H1>References</H1></DIV></LI><LI>Donlan, R. M. (2001). Biofilms and device-associated infections. Emerg Infect Dis, 7(2), 277-281.
</LI><LI>Hwang, D. S., Gim, Y., Yoo, H. J., &amp; Cha, H. J. (2007). Practical recombinant hybrid mussel bioadhesive fp-151. Biomaterials, 28(24), 3560-3568.
</LI><LI>Isaacs, F. J. (2012). Synthetic biology: Automated design of RNA devices. Nat Chem Biol, 8(5), 413-415.
</LI><LI>Isaacs, F. J., Carr, P. A., Wang, H. H., Lajoie, M. J., Sterling, B., Kraal, L., <I>et al.</I>. (2011). Precise manipulation of chromosomes in vivo enables genome-wide codon replacement. Science, 333(6040), 348-353.
</LI><LI>Nagant, C., Pitts, B., Stewart, P. S., Feng, Y., Savage, P. B., &amp; Dehaye, J. P. (2013). Study of the effect of antimicrobial peptide mimic, CSA-13, on an established biofilm formed by Pseudomonas aeruginosa. Microbiologyopen, 2(2), 318-325.
</LI><LI>Ramos, R., Domingues, L., and Gama, M. (2011) LL-37, a human antimicrobial peptide with immunomodulatory properties. 2, pp.693-1348, In: Science Against Microbial Pathogens: Communicating Current Research and Technological Advances. Formatex Research Center Publications. Badajoz, Spain.
</LI><LI>Salta, M., Wharton, J. A., Stoodley, P., Dennington, S.P., Goodes, L. R., &amp; Werwinski, S., <I>et al.</I>. (2010). Designing biomimetic antifouling surfaces. Philosophical Transactions of the Royal Society A: Mathematical, Physical and Engineering Sciences, 368(1929), 4729-4754.
</LI><LI>H. Lee, S. M. Dellatore, W. M. Miller and P. B. Messersmith, &quot;Mussel-Inspired Surface Chemistry for Multifunctional Coatings&quot;, Science, 318, 426-430, (2007).
Endy, Drew. Standard Registry of Parts. Massachusetts Institute of Technology. partsregistry.org. 2008.
</LI><LI>Isaacs, Farren J., <I>et al.</I> &quot;Precise manipulation of chromosomes in vivo enables genome-wide codon replacement.&quot; <I>Science</I> 333.6040 (2011): 348-353. </LI><LI>Sharan, Shyam K., Lynn C. Thomason, and Sergey G. Kuznetsov. &quot;Recombineering: a homologous recombination-based method of genetic engineering.&quot; <I>Nature protocols</I> 4.2 (2009): 206-223. </LI><LI>Wang, Harris H., <I>et al.</I> &quot;Programming cells by multiplex genome engineering and accelerated evolution.&quot; <I>Nature</I> 460.7257 (2009): 894-898. </LI><LI>Ellis, Hilary M., Daiguan Yu, and Tina DiTizio. &quot;High efficiency mutagenesis, repair, and engineering of chromosomal DNA using single-stranded oligonucleotides.&quot; <I>Proceedings of the National Academy of Sciences</I> 98.12 (2001): 6742-6746. </LI><LI>Costantino, Nina. &amp; Court, Donald L. “Enhanced levels of  Red-mediated recombinants in mismatch repair mutants.” <I>Proceedings of the National Academy of Sciences</I> 100.26 (2003): 15748–15753. </LI><LI>Lajoie, M. J., Rovner, A. J., Goodman, D. B., Aerni, H. R., Haimovich, A. D., Kuznetsov, G., <I>et al.</I> (2013). Genomically recoded organisms expand biological functions. <I>Science</I>, 342(6156), 357-360. </LI><LI>Levine, Melvin, and Harold Tarver. &quot;Studies on ethionine III. Incorporation of ethionine into rat proteins.&quot; <I>Journal of Biological Chemistr</I>y 192.2 (1951): 835-850. </LI><LI>Wang, Lei, <I>et al.</I> &quot;A new functional suppressor tRNA/aminoacyl-tRNA synthetase pair for the in vivo incorporation of unnatural amino acids into proteins.&quot; <I>JOURNAL-AMERICAN CHEMICAL SOCIETY</I>122.20 (2000): 5010-5011. </LI><LI>Wang, Lei, and Peter G. Schultz. &quot;A general approach for the generation of orthogonal tRNAs.&quot; <I> amino acids</I> 3 (2001): 4. </LI><LI>Wang, Lei, <I>et al.</I> &quot;Expanding the genetic code of Escherichia coli.&quot; <I>Science</I> 292.5516 (2001): 498-500. </LI><LI>Isaacs, F. J., <I>et al.</I> (2004). &quot;Engineered riboregulators enable post-transcriptional control of gene expression.&quot; Nature biotechnology 22(7): 841-847.
</LI><LI>BP Lee, PB Messersmith, JN Israelachvili, JH Waite. (2011) Mussel-Inspired Adhesives and Coatings. Annual Review of Materials Research; 41: 99-132. 
</LI><LI>H Lee , NF Scherer, PB Messersmith. (2006) Single-Molecule Mechanics of Mussel Adhesion. Proc Natl Acad Sci; 103:12999-3003.
</LI><LI>M Yu, J Hwang, TJ Deming. (1999) Role of L-3,4-Dihydroxyphenylalanine in Mussel Adhesive Proteins. J. Am. Chem. Soc. 1999, 121, 5825-5826
</LI><LI>Gao, Y., Zorman, S., Gundersen G., Xi, Z., Ma L., Sirinakis G., Rothman J.E., &amp; Zhang Y. (2012) Single Reconstituted Neuronal SNARE Complexes Zipper in Three Distinct Stages. Science (New York, N.Y.) 337(6100):1340-1343

<DIV class="mfooter"><DIV class="footbox" style="">
    Main Campus: 
Molecular, Cellular &amp; Developmental Biology
219 Prospect Street
P.O. Box 208103
New Haven, CT 06520 Phone: 203.432.3783<A href="mailto:igem@yale.edu">igem@yale.edu </A><A href="mailto:natalie.ma@yale.edu">natalie.ma@yale.edu (Graduate Advisor)</A></DIV><CENTER>
    Copyright (c) 2014 Yale IGEM
  </CENTER></DIV><DIV class="printfooter">
Retrieved from &quot;<A href="http://2014.igem.org/Team:Yale/Project">http://2014.igem.org/Team:Yale/Project</A>&quot;</DIV></LI><LI id="t-recentchanges"><A href="/Special:RecentChanges" title="Recent changes">Recent changes</A></LI><LI id="t-whatlinkshere"><A href="/Special:WhatLinksHere/Team:Yale/Project" title="List of all wiki pages that link here [j]" accesskey="j">What links here</A></LI><LI id="t-recentchangeslinked"><A href="/Special:RecentChangesLinked/Team:Yale/Project" title="Recent changes in pages linked from this page [k]" accesskey="k">Related changes</A></LI><LI id="t-specialpages"><A href="/Special:SpecialPages" title="List of all special pages [q]" accesskey="q">Special pages</A></LI><LI><A href="/Special:Preferences">My preferences</A></LI><DIV id="footer"><UL id="f-list"><LI id="t-print"><A href="/wiki/index.php?title=Team:Yale/Project&amp;printable=yes" title="Printable version of this page [p]" accesskey="p">Printable version</A></LI><LI id="t-permalink"><A href="/wiki/index.php?title=Team:Yale/Project&amp;oldid=401159" title="Permanent link to this revision of the page">Permanent link</A></LI><LI id="privacy"><A href="/2014.igem.org:Privacy_policy" title="2014.igem.org:Privacy policy">Privacy policy</A></LI><LI id="disclaimer"><A href="/2014.igem.org:General_disclaimer" title="2014.igem.org:General disclaimer">Disclaimers</A></LI></UL></DIV></OL></DIV></DIV></BODY></HTML>