<HTML xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en" dir="ltr">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki  ltr ns-0 ns-subject page-Team_Yale_Results"><DIV id="globalWrapper"><DIV id="top-section"><DIV id="p-logo"><A href="/Main_Page" title="Main Page">&quot;
	    </A></DIV><DIV id="menubar" class="left-menu noprint"><UL><LI class="selected"><A href="/Team:Yale/Results">Page               </A></LI><LI class="new"><A href="/wiki/index.php?title=Talk:Team:Yale/Results&amp;action=edit&amp;redlink=1">Discussion               </A></LI><LI><A href="/wiki/index.php?title=Team:Yale/Results&amp;action=edit">View source               </A></LI><LI><A href="/wiki/index.php?title=Team:Yale/Results&amp;action=history">History               </A></LI><LI style="color:white;cursor:default">teams</LI></UL></DIV><DIV class="right-menu noprint" id="menubar"><UL><LI id="pt-login"><A href="/wiki/index.php?title=Special:UserLogin&amp;returnto=Team:Yale/Results" title="You are encouraged to log in; however, it is not mandatory [o]" accesskey="o">Log in</A></LI></UL></DIV><DIV id="search-controls" class="noprint"><FORM action="/Special:Search" id="searchform"> </FORM></DIV></DIV><DIV id="content"><H1 class="firstHeading">Team:Yale/Results</H1><DIV id="bodyContent"><H3 id="siteSub" class="noprint">From 2014.igem.org</H3><DIV class="navmenubkg"><DIV id="navmenubox"><UL id="nav"><LI style="width:120px;"><A href="https://2014.igem.org/Team:Yale" title="Home">Home</A></LI><LI style="width:120px;"><A href="https://2014.igem.org/Team:Yale/Project" title="Project">Project</A></LI><LI style="margin-left:14.5px"><A href="https://2014.igem.org/Team:Yale/Project" title="Overview">Overview</A></LI><LI style="width:180px"><A href="https://2014.igem.org/Team:Yale/MaterialsMethods" title="Coating">Materials &amp; Methods</A></LI><LI><A href="https://2014.igem.org/Team:Yale/Results" title="Results">Results</A></LI><LI><A href="https://2014.igem.org/Team:Yale/Project/modeling" title="Modeling">Modeling</A></LI><LI><A href="https://2014.igem.org/Team:Yale/Parts" title="Parts">Submitted Parts</A></LI></UL></DIV><LI style="width:120px;"><A href="https://2014.igem.org/Team:Yale/Team" title="OurLab">About</A></LI><LI style="margin-left:141px;"><A href="https://2014.igem.org/Team:Yale/Team" title="Team">Team</A></LI><LI style="width:180px"><A href="https://igem.org/Team.cgi?year=2014&amp;team_name=Yale" title="Official">Official Team Profile</A></LI><LI style="width:180px;"><A href="https://2014.igem.org/Team:Yale/Acknowledgments" title="Acknowledgments">Acknowledgments</A></LI><LI style="width:140px;"><A href="https://2014.igem.org/Team:Yale/Safety" title="Safety">Safety</A></LI><LI style="width:140px;"><A href="https://2014.igem.org/Team:Yale/Contact" title="Contact">Contact Us</A></LI></DIV><LI style="width:120px;"><A href="https://2014.igem.org/Team:Yale/Outreach" title="Outreach">Outreach</A></LI><LI style="width:120px;"><A href="https://2014.igem.org/Team:Yale/Interlab" title="Interlab">Interlab</A></LI><LI style="width:120px;"><A href="https://2014.igem.org/Team:Yale/Notebook">Notebook</A></LI><LI style="width:150px;"><A href="https://2014.igem.org/Team:Yale/Achievements">Achievements</A></LI></DIV></DIV></DIV><TABLE width="1100px" align="center" style="margin-top:0px"><TBODY><TR><TD><DIV class="callout"><H1 style="margin-top:22px; font-size:50px;">Results</H1></DIV></TD></TR><TR><TD colspan="12"><DIV class="tinycall"><H1>T7 Riboregulation System</H1></DIV><DIV class="well"><OL type="I"><LI><STRONG>T7 RNA polymerase design and creation</STRONG><P>
The T7 Riboregulation System works by a “three-lock system.” The first lock is the cis-repressing RNA (crRNA), which is induced bysopropyl β-D-1-thiogalactopyranoside (IPTG). The second lock is the trans-activating RNA (taRNA), which is induced by anhydrous tetracycline (ATC).  If the taRNA is unlocked, it will bind to the crRNA, removing the hairpin and making the ribosomal binding site accessible for ribosomal binding, leading to translation of a specific protein, in this case, T7 RNA Polymerase. This system was initially developed by Dr. Farren Isaacs, and has been shown to work with chloramphenicol resistance (chloramphenical acetyl transferase gene) in place of the T7 gene. The plasmid was synthesized via Gibson assembly, and confirmed by sequencing.

</P><P><CENTER><I><STRONG>Figure 1.</STRONG> Preliminary gel screening of Mach 1 strains containing transformed pZE21_A12C_T7RNA plasmids created via Gibson Assembly. Used combinations of general pZE21 sequencing primers, F: CAGGGCTTCCCAACCTTAC, R: CGCCTTTGAGTGAGCTGATA, and internal T7 primers, F: TCCCTTACAACATGGACTGGC, R: CCCACCAAGTGTTCTCCAG. The corresponding sizes are labelled on the side. The negative control for the external primers is the ancestor plasmid, which contains chloramphenicol acetyl transferase (CAT) instead of T7, and is 1.1 kb instead of 3.3 kb.</I></CENTER></P><P><CENTER><I><STRONG>Figure 2.</STRONG> Sequencing data for T7 RNA polymerase construct, using the general pZE21 sequencing primers, which amplify upstream of the taRNA and crRNA system. Sequencing done via Keck Biotechnology Resource Laboratory. Gray indicates consensus with the desired sequence. Image made using geneious.</I></CENTER></P><P>Currently the riboregulation system may have an issue with the internal T7 sequence, and while sequencing has been done, no successful data has been obtained.</P></LI><LI><STRONG>Functional Assays for Riboregulated T7 system</STRONG><P>
Experimental plan for the GFP fluorescence assay testing the efficacy of the T7 riboregulation system. The T7 riboregulation system, pZE21_A12C_T7RNA, would express sfGFP behind a T7 promoter, in the plasmid pZA21. Either plasmid, and both plasmids together, were transformed into ECNR2 and induced with either IPTG and ATC. ECNR2 is the ancestral strain. A positive control was the same pZA21_T7sfGFP plasmid in ECNR2, and the same T7 RNA polymerase gene inserted in a regular pZE21 plasmid with a pLtetO promoter, and a negative control with the pZA21_T7sfGFP in ECNR2 without any plasmid that contains T7 RNA. </P><P><CENTER><I><STRONG>Figure 3.</STRONG> The functionalities behind the GFP assay as described above.</I></CENTER></P><P><CENTER><I><STRONG>Figure 4.</STRONG> Conformational assay to test the functionality of the pZA21_T7sfGFP, which is sfGFP placed behind the T7 promoter. The plasmid was transformed into a BL21(DE3) strain, which constitutively expresses T7 RNA polymerase. The strain, as well as untransformed BL21(DE3), were grown overnight and assayed using a Synergy H1 Biotek Platereader. Fluorescence measurement was taken by exciting the cells at 485 nm and detecting at 528 nm, with a bandpass of 4 nm on each side. The optical density was also taken at 600 nm, and the fluorescence data was normalized by dividing fluorescence by optical density. What is shown is the average of 4 replicates each. </I></CENTER></P></LI></OL></DIV></TD></TR><TR class="tableizer-firstrow"><TH>Plasmid</TH><TH>Antibiotic Marker</TH><TH>Inducer Conditions 1</TH><TH>Inducer Conditions 2</TH><TH>Inducer Conditions 3</TH><TH>Inducer Conditions 4</TH></TR><TR><TD>pZE21_A12C_T7RNA, pZA21_T7sfGFP</TD><TD>Kanamycin, Spectinomycin</TD><TD>ATC, IPTG</TD><TD>ATC only</TD><TD>IPTG only</TD><TD>No Inducers</TD></TR><TR><TD>pZE21_A12C_T7RNA</TD><TD>Kanamycin</TD><TD>ATC, IPTG</TD><TD>ATC only</TD><TD>IPTG only</TD><TD>No Inducers</TD></TR><TR><TD>pZA21_T7sfGFP</TD><TD>Spectinomycin</TD><TD>ATC, IPTG</TD><TD>ATC only</TD><TD>IPTG only</TD><TD>No Inducers</TD></TR><TR><TD>ECNR2, no plasmids</TD><TD>None</TD><TD>ATC, IPTG</TD><TD>ATC only</TD><TD>IPTG only</TD><TD>No Inducers</TD></TR></TBODY></TABLE><P><CENTER><I><STRONG>Table 1.</STRONG> Experimental conditions for the GFP assay. Each plasmid combination was paired with each inducer combination, and the conditions were made in six replicates.</I></CENTER></P><DIV class="tinycall"><H1>Ampersand Construct </H1></DIV><DIV class="well"><OL type="I"><LI><STRONG>Synthesis and Transformation</STRONG><P>The AMP-MAP construct, also known as Ampersand construct, was received and cloned into a standard pZE21 plasmid backbone with pLtetO promoter, as the T7 Riboregulation system was incomplete at the time. The construct has been sent for sequencing, and is now awaiting functional assays.
</P><P><CENTER><I><STRONG>Figure 5.</STRONG>  Gel results of the Gibson assembly product of the construct into pZE21 plasmid backbone, with a co-transformed OTS system. Used same universal primers as before, and the results present a difficulty in sequencing: lanes 3 and 4 are the Tyrosine suppressor system WITHOUT the construct, which means sequencing data are unable to be obtained unless performed on a strain without the OTS, which unfortunately interfere with the functional assays planned.</I></CENTER></P></LI><LI><STRONG> Functional Assay</STRONG><P>
Functional assays are ongoing and results will be presented at the jamboree.</P><DIV class="tinycall"><H1>Adhesion Testing </H1></DIV></LI><LI><STRONG>Mass Retention of Mussel Adhesion Proteins (MAPs) Under Stress</STRONG><P>Preliminary proof of concept testing was conducted on a commercially available MAP-based product known as Cell-Tak <SUP>TM</SUP>. Cell-Tak<SUP>TM</SUP> is designed to facilitate cell adhesion to normally non-biocompatible surfaces such as microscope slides and petri dishes. We deposited ~20 µg films of Cell-Tak onto borosilicate substrates and proceeded to erode them under deionized H<SUB>2</SUB>O and 5% acetic acid. The results from this experiment are presented below and illustrate the design of our assay to test a variety of solvent and erosion conditions on MAP films. A microbalance (Mettler Toledo MX5) that can read to uncertainties of 1 µg was used to determine the mass of protein remaining after subjecting the substrate to erosion. An exponential decay curve was fitted to these experiments giving decay rates of 0.002 µg/pass and 0.046 µg/pass for deionized H<SUB>2</SUB>O and 5% acetic acid, respectively. As lower pH reverses the coordination of L-DOPA, it is expected that the acidic conditions engender the higher rate of decay. This experiment presents a preliminary result that validates our ability to apply erosion onto MAP-coated surfaces. We intend to apply a similar protocol to metal and plastic surfaces as well as erode surfaces under different pH conditions to provide a more comprehensive picture of the optimal conditions for mussel adhesion.
</P><P><CENTER><I><STRONG>Figure 5.</STRONG><STRONG>(A)</STRONG> The erosion of Cell-Tak <SUP>TM</SUP> under conditions of DI water erosion. <STRONG>(B)</STRONG> The erosion of Cell-Tak <SUP>TM</SUP> under conditions of 5% acetic acid erosion.</I></CENTER></P></LI><LI><STRONG> Determination of Surface Energies of MAP Films</STRONG><P>
A contact angle measurement of a Cell-Tak<SUP>TM</SUP> was recorded and served as an indication for presence of peptide on surfaces. The contact angle is measured between the surface of the drop and the table-top. Larger contact angles are indicative of more hydrophobic surfaces while shallower contact angles correspond to more wettable surfaces. A contact angle of 25.053º was obtained between an untreated silica substrate and a 2 µL drop of DI H<SUB>2</SUB>O. However, when surfaces were treated with the MAP, the contact angle increased to 62.007º, indicative of an increase in the hydrophobicity of our substrate. This result validates the evolutionary need for mussels to secrete proteins that are resistant to water in order to survive and anchor themselves in constantly wet environments. 
</P><P><CENTER><I><STRONG>Figure 6.</STRONG><STRONG> (A)</STRONG> The profile photograph of a drop on an untreated silica substrate used for contact angle determination. <STRONG>(B)</STRONG> The profile photograph of a Cell-Tak <SUP>TM</SUP> treated surface used for contact angle determination.</I></CENTER></P><DIV class="mfooter"><DIV class="footbox" style="">
    Main Campus: 
Molecular, Cellular &amp; Developmental Biology
219 Prospect Street
P.O. Box 208103
New Haven, CT 06520 Phone: 203.432.3783<A href="mailto:igem@yale.edu">igem@yale.edu </A><A href="mailto:natalie.ma@yale.edu">natalie.ma@yale.edu (Graduate Advisor)</A></DIV><CENTER>
    Copyright (c) 2014 Yale IGEM
  </CENTER></DIV><DIV class="printfooter">
Retrieved from &quot;<A href="http://2014.igem.org/Team:Yale/Results">http://2014.igem.org/Team:Yale/Results</A>&quot;</DIV></LI><LI id="t-recentchanges"><A href="/Special:RecentChanges" title="Recent changes">Recent changes</A></LI><LI id="t-whatlinkshere"><A href="/Special:WhatLinksHere/Team:Yale/Results" title="List of all wiki pages that link here [j]" accesskey="j">What links here</A></LI><LI id="t-recentchangeslinked"><A href="/Special:RecentChangesLinked/Team:Yale/Results" title="Recent changes in pages linked from this page [k]" accesskey="k">Related changes</A></LI><LI id="t-specialpages"><A href="/Special:SpecialPages" title="List of all special pages [q]" accesskey="q">Special pages</A></LI><LI><A href="/Special:Preferences">My preferences</A></LI><DIV id="footer"><UL id="f-list"><LI id="t-print"><A href="/wiki/index.php?title=Team:Yale/Results&amp;printable=yes" title="Printable version of this page [p]" accesskey="p">Printable version</A></LI><LI id="t-permalink"><A href="/wiki/index.php?title=Team:Yale/Results&amp;oldid=400946" title="Permanent link to this revision of the page">Permanent link</A></LI><LI id="privacy"><A href="/2014.igem.org:Privacy_policy" title="2014.igem.org:Privacy policy">Privacy policy</A></LI><LI id="disclaimer"><A href="/2014.igem.org:General_disclaimer" title="2014.igem.org:General disclaimer">Disclaimers</A></LI></UL></DIV></OL></DIV></BODY></HTML>