"
Page
Discussion
View source
History
teams
Log in
 
Team:Yale/Notebook1
From 2014.igem.org
Home
Project
Overview
Materials & Methods
Results
Modeling
Submitted Parts
About
Team
Official Team Profile
Acknowledgments
Safety
Contact Us
Outreach
Interlab
Notebook
Achievements
Lab Notebook
Outline of Our Project
Outline of Our Project
Goal: Create a recombinant protein using a dopamine surface anchor and an anti-fouling head-domain that can be expressed in recoded E. Coli.
Create an expression system
Creating theT7 Riboregulated System
First, create a construct that places the gene for T7 RNA polymerase in the backbone of a plasmid with a cis-repressing and trans-activating RNA.
Obtain T7 RNA Pol: from team strain #5.
Obtain pZE21 backbone: from Ryan
Design primers for PCR and Gibson Assembly (191, 192, 193, 194, 195)
PCR amplify T7 RNA Pol an pZE21 using primers.
DPN1 Digest template DNA.
Run fragments on a gel green Gel
Gel Purification of fragments.
Gibson Assembly of fragments into plasmid construct.
Transform plasmid construct into K12 derivative, ECNR2. Plate on Kan Plates.
Pick colonies from Kan Plates, inoculate liquid culture.
From liquid culture make a frozen stock.
Screen colonies for presence of plasmid of interest
First find which colonies have T7 and pZE21.
Use sequencing primers to amplify sequence of interest. Sequence for verification.
Integrate into genome of ECNR2 or BL21.
Create Promoter for T7 Riboregulated System
Create a construct that installs sfGFP behind promoter for T7.
Obtained T7 promoter from registry and Life Technologies.
obtained sfGFP gene from laboratory stock.
Approach 1: T7 Overhangs Approach
amplify standard pZE21::sfGFP plasmid with primers designed by Stephanie that have T7 as overhangs. Amplification of this plasmid will situate the T7 promoter in front of sfGFP.
DPN1 digest.
Gibson Assemble to circularize into plasmid.
Drop Dialysis to minimize salt.
transform into ECNR2/BL21
Screening for Plasmid
Insert plasmid into BL21 DE3 and screen for expression of sfGFP. BL21 DE3 is a strain that contains the gene for T7 RNA Polymerase.
Approach 2: Insert sfGFP into T7 containing plasmid.
Using psB1A2 with T7 promoter from registry. Transform bacteria to make stock of pSB1AC2.
Using primers designed by Ariel amplify pSB1AC2 to have overhangs with sfGFP.
Amplify sfGFP from pZE21::sfGFP with primers designed by Ariel to have overhangs with pSB1A2 with T7 Promoter.
DPN1 Digest
Gibson Assemble.
Drop Dialysis.
Transform into BL21/576
Screen for ampicillin/carbenicillin resistance.
Insert plasmid into BL21 DE3 and screen for sfGFP expression.
Create constructs
Express Constructs in rE.Coli
Test Constructs using Antimicrobial Assays
Test Constructs using Adhesion Assays.
TroubleShooting (T7)
Okay, so we’re currently having some issues with the transformations, so let’s take a step back and look at what can go wrong, and how we can fix them: 
Template generation: For T7 RNA pol+pZE21 with the cr/ta system, the hairpins on the primers limit the efficacy of PCR and Gibson assembly.
Solutions: First, we should screen the plasmids we have to check and see if they do in fact contain T7. Ariel has designed primers for the test already, and if necessary we can use the pZE21 with CAT as a control, since the band would be a significantly shorter length. 
     We can also try a different method of assembly: restriction enzyme digest (may be difficult because there are none that anneal EXACTLY to the spots we want, some are a few base pairs off), or deactivating CAT’s start codon and placing T7 a little ways downstream of the crRNA sequence, INSIDE CAT (which, without the start codon, would not be translated).
Transformation: We need to drop dialyze longer. Our bacteria doesn’t seem to like the high salt concentration. Do duplicate plates. 
Quality control on plates: be sure to streak a non-transformed plate every time to make sure the antibiotic is working.
Transform into a hardier strain/one better suited for transformations?
Construct Idea: DOPA + Spider silk
Construct Idea: DOPA + Spider silk
Here’s an idea: what if we attach spider silk in between two mussel foot proteins, and then coat a surface with it?
(Stephanie’s idea)
1. Spider silk has been shown to be antimicrobial
2. Spider silk is strong and resistant to degradation.
3. Spider silk is a structural polymer and does not require terminal specificity.
4. Spider silk has been produced with a conjugated protein before by a former iGEM team .
5. Spider silk loses its stickiness over time, and its antimicrobial effects are decreased when exposed to water…DOPA can help 
Spider silk is in the iGEM registry , and sequences can also be found here .
The idea then is to generate spider-silk DOPA peptides that, in coating a surface, creates a mesh on which biofilms cannot form. This may be a good alternative in looking at medical applications, since spider silk is NOT toxic in mammals.
MGFP Purification Protocol
MGFP Purification Protocol
1. Recombinant mgfp-5 in E. coli, extracted w/ unmodified Tyr instead of DOPA
Source:  http://aem.asm.org/content/70/6/3352.full.pdf  (2007)
Extracted mgfp-5 was inserted into plasmid via restriction sites, with an N-terminus His6 tag.
Vector had Amp resistance and IPTG-activated promoter.
Immobilized-metal affinity purification, with Ni-nitrilotriacatate NTA agarose (Qiagen) (10 ml) charged with 10 ml of 0.1 M NiSO4 (Samchun Chemicals) as the afﬁnity puriﬁcation resin.
Pelleted cells, resuspended with  5 ml of buffer B (8 M urea, 10 mM Tris-HCl, 100 mM sodium phosphate [pH 8.0]) per g (wet weight). Centrifuged for 20 min at r.t., then collected supernatant to run on column.
Column had 5 resin vol of Buffer B, and 10 mL of lysate was washed w/ buffer C (8 M urea, 10 mM Tris-HCl, 100 mM sodium phosphate, [pH 6.3]) and buffer D (8 M urea, 10 mM Tris-HCl, 100 mM sodium phosphate [pH 5.9]).
Target protein was eluted with buffer E (8 M urea, 10 mM Tris-HCl, 100 mM sodium phosphate [pH 4.5]).
Eluted protein was drop dialyzed overnight in 5% acetic acid at 4˚C using Spectra/Por molecular porous membrane tubing.
Proteins were centrifuged at 100˚C and ran on an SDS-PAGE.
Percent yield was 7%: future should be better.
2. Same guys as above, now with recombinant fp-151:
Source:  http://www.sciencedirect.com/science/article/pii/S0142961207003651  (2007)
"25% (v/v) acetic acid [dilution] provided a maximum recovery yield (∼54%) and relatively high purity (∼88%) of fl-151"
Antifouling Notes
Antifouling Notes
So there’s different environments to consider with this project: (the proper term for the issue we’re dealing with is “biofouling”)
Ship hulls
Spaceships (interior) (SPACESHIP!)
Pipes
Hospitals (more from Yamini)
Common perpetrators of this foul crime are thiobacilli, pseudomonas, etc. Our best bet is probably some strain of pseudomonas (I can ask Ben for samples)
Factors to consider while choosing antimicrobial factors:
Specificity: we’d probably want a
Size: this is a physical limitation on the effectiveness of creating a conjugated antimicrobial. If we can’t attach the protein to the L-Dopa leg, then it’s pointless.
Toxicity: BIG factor if we want to coat surfaces that people will touch or will come in contact with. For example, we’d want the factor to be relatively safe so the parts that wash off from the ship hull will not then go and kill a plankton population.
Method: so there’s multiple ways to kill bacteria: an antimicrobial protein can actively lyse a cell wall, or, there are mechanical surfaces that actively dissuade cells from growing on surfaces.
A texture antimicrobial factor may be preferable because it is more general and hopefully less toxic if detached from a large surface.
WELL BOY HOWDY LOOK AT THIS HERE PAPER IT’S ALMOST AS IF IT’S TRYING TO SOLVE THE EXACT SAME PROBLEM.
Biomimetic polymers we can look into: cicada wings (other insects also have similar properties), lotus leaves, shark skins, gecko feets (wow these are all cool things! any one of these can be a mascot!)
Here’s a review of hydrophobic surfaces--it looks like our struggle is less about protein function and more about overall structure, which we may be able to space apart with specific designs of the protein foot. These people have a more complicated idea, coating a surface with DOPA and then making the proteins mimic the organization of a lotus leaf. The physics of fluidity may help.
Current findings: this polypeptide-mimic compound that uses PMP1 (huh!) Unfortunately this looks like it needs post-translational modification.
General characteristics of antifouling consensus sequences: electrically neutral, hydrophilic, electron accepting but not electron donating.
http://davidcwhite.org/fulltext/455.pdf
Collection of links on antifouling:
http://www.bimat.org/assets/pdf/nu_05_8dalsin.pdf
http://web.mit.edu/andrew3/Public/Papers/2012/Ivanova/2012_Small_Natural%20Bactericidal%20Surfaces%20Mechanical%20Rupture_Ivanova.pdf
http://www.che.ncsu.edu/genzergroup/pubs/pub-06-11.pdf
http://pubs.acs.org/doi/abs/10.1021/ja303037j
http://science.nchc.org.tw/old_science/nano_mems/nanomidtermreport/Cicada%20Wings%20A%20Stamp%20from%20Nature%20for%20Nanoimprint%20Lithography.pdf
http://cen.acs.org/articles/90/web/2012/06/Lotus-Leaves-Mussels-Inspire-Method.html
Construct Idea: DOPA + Spider silk
Construct Idea: DOPA + Spider silk
Here’s an idea: what if we attach spider silk in between two mussel foot proteins, and then coat a surface with it?
(Stephanie’s idea)
1. Spider silk has been shown to be antimicrobial
2. Spider silk is strong and resistant to degradation.
3. Spider silk is a structural polymer and does not require terminal specificity.
4. Spider silk has been produced with a conjugated protein before by a former iGEM team .
5. Spider silk loses its stickiness over time, and its antimicrobial effects are decreased when exposed to water…DOPA can help 
Spider silk is in the iGEM registry , and sequences can also be found here .
The idea then is to generate spider-silk DOPA peptides that, in coating a surface, creates a mesh on which biofilms cannot form. This may be a good alternative in looking at medical applications, since spider silk is NOT toxic in mammals.
Construct Idea: DOPA + Spider silk
Construct Idea: DOPA + Spider silk
Here’s an idea: what if we attach spider silk in between two mussel foot proteins, and then coat a surface with it?
(Stephanie’s idea)
1. Spider silk has been shown to be antimicrobial
2. Spider silk is strong and resistant to degradation.
3. Spider silk is a structural polymer and does not require terminal specificity.
4. Spider silk has been produced with a conjugated protein before by a former iGEM team .
5. Spider silk loses its stickiness over time, and its antimicrobial effects are decreased when exposed to water…DOPA can help 
Spider silk is in the iGEM registry , and sequences can also be found here .
The idea then is to generate spider-silk DOPA peptides that, in coating a surface, creates a mesh on which biofilms cannot form. This may be a good alternative in looking at medical applications, since spider silk is NOT toxic in mammals.
Construct Idea: DOPA + Spider silk
Construct Idea: DOPA + Spider silk
Here’s an idea: what if we attach spider silk in between two mussel foot proteins, and then coat a surface with it?
(Stephanie’s idea)
1. Spider silk has been shown to be antimicrobial
2. Spider silk is strong and resistant to degradation.
3. Spider silk is a structural polymer and does not require terminal specificity.
4. Spider silk has been produced with a conjugated protein before by a former iGEM team .
5. Spider silk loses its stickiness over time, and its antimicrobial effects are decreased when exposed to water…DOPA can help 
Spider silk is in the iGEM registry , and sequences can also be found here .
The idea then is to generate spider-silk DOPA peptides that, in coating a surface, creates a mesh on which biofilms cannot form. This may be a good alternative in looking at medical applications, since spider silk is NOT toxic in mammals.
{{Template:Team:Yale2014/Templates/Footer}}
Retrieved from " http://2014.igem.org/Team:Yale/Notebook1 "
Recent changes
What links here
Related changes
Special pages
My preferences
Printable version
Permanent link
Privacy policy
Disclaimers
