"
Page
Discussion
View source
History
teams
Log in
 
Team:Yale/MaterialsMethods
From 2014.igem.org
Home
Project
Overview
Materials & Methods
Results
Modeling
Submitted Parts
About
Team
Official Team Profile
Acknowledgments
Safety
Contact Us
Outreach
Interlab
Notebook
Achievements
Materials and Methods
T7 Riboregulation System
Strains, Plasmids, and Reagents
E. coli strains used in this study included BL21(E. coli B F- dcm ompT hsdS(rB- mB-) gal [malB+]K-12 (λS)), BL21(DE3)( F– ompT gal dcm lon hsdSB(rB- mB-) λ(DE3 [lacI lacUV5-T7 gene 1 ind1 sam7 nin5])), ECNR2(ΔmutS:cat.Δ(ybhB-bioAB): [λcI857.Δ(cro-ea59):tetR-bla]), and Mach1(ΔrecA1398 endA1 tonA Φ80ΔlacM15 ΔlacX74 hsdR(r K- mK+). Strains used for transformation were grown in LB Lennox(Cold Spring Harbor Protocols2006).     Cells used for cloning and mini-prep were grown in selective medium of 2XYT (2xYt Medium (7281) 2010) with either kanamycin (American Bioanalytical) or     spectinomycin (Sigma-Aldrich). Kanamycin and spectinomycin were used at 30 mg/mL and 95 mg/mL respectively.
One percent agarose gels were made with 0.5% TBE obtained from American Bio and stained with either ethidium bromide (Sigma-Aldrich) (screening) or SYBR     Safe (Invitrogen) (cloning) in the case of cloning. Gel extraction and purification was completed with QIAprep Gel Extraction Kit following the protocol     provided. PCR purification was accomplished with the QIAquick PCR Purification Kit, following the protocol provided. Plasmid purification was accomplished     using the QIAprep Spin Miniprep Kit and the protocol provided. For all DNA kits provided by QIAgen we used Denville Spin Columns for Nucleic Acid     Purification. The concentration of DNA was measured using a Biotek Synergy HT Multi-Mode microplate Reader with accompanying Take3 Microvolume plates. All     restriction enzymes, and Gibson Assembly Master Mix are from New England Biolabs. Hifi HotStart Readymix and 2GFAST Readymix with loading dye for PCR were     obtained from KAPA Biosystems.
Plasmids
Plasmid pZE21_Y12_a12C_c contains a ColE1 origin of replication and a kanamycin resistance marker. At the multiple cloning site in the reverse direction is     inserted a trans-activating RNA (taRNA) sequence expressed by PLtetO promoter and in the forward direction is a chloramphenicol resistance     marker with a cis-repressing RNA (crRNA) on the 5’ UTR upstream of the ribosomal binding site(RBS). Plasmid pZE21::sfGFP also contains a ColE1 origin of     replication and kanamycin resistance marker. Superfolder GFP (sfGFP) is inserted into the multiple cloning site and is expressed by PLtetO.     pZA21_tRNA_TAGG has a p15a origin of replication and a spectinomycin resistance marker. In its multiple cloning site it contains a 3 Tyrosine ELP     purification tag adjacent to GFP. The recombinant protein expression is driven by PLtetO.
Gibson Assembly of plasmids from PCR products
Fragments were assembled into plasmids using Gibson Assembly (Gibson, Young et al. 2009). Gel purified PCR products with 5’ complimentary overhangs were     combined in equimolar ratios with ten microliters of Gibson Assembly Master Mix and additional nuclease free water to obtain a reaction volume of twenty     microliters. The reaction was carried out in a thermocycler held constant at 50°C over the course of one hour.
Transformation of Plasmids through Electroporation
Transformation was carried out through electroporation. First, the plasmid to be transformed was desalted for an hour using a Millipore (type VSWP) drop     dialysis film with 0.025 μm on ultra-pure Milli-Q water. One mL of liquid culture was centrifuged and washed twice with Milli-Q water. After washing, the     cells were pelleted, the supernatant was removed, and the pellet was re-suspended in 50 μL of a dilute solution of DNA and nuclease free water (American     Bioanalytical). The cells and DNA solutions were transferred to a Bio Rad 0.1 cm gap Gene Pulser cuvettes for electroporation and kept chilled.     Electroporation was conducted with a Bio-Rad Gene Pulser Xcell Electroporator set to 1800 V, 25 μF, and 200 Ω. After electroporation, the cells were     immediately transferred to 1 mL of LB Lennox and placed in a 37° incubator to recover for an hour. The cells were then plated on LB Lennox agar with an     antibiotic selection factor for the transformed plasmid vector.
Assembly of T7 RNA Polymerase in pZE21_Y12_a12C backbone
The plasmid pZE21_Y12_a12C_T7RNAPol was Gibson Assembled from the T7 RNA Polymerase gene obtained from BL21(DE3) and the pZE_Y12_a12C backbone. The T7 RNA     Polymerase gene was obtained from BL21(DE3) through colony PCR with primers pZE21_T7_F and pZE21_T7_R that anneal at either end of the T7 RNA Polymerase     gene and include complimentary base pair overhangs on the 5’ end to pZE21_Y12_a12C at the site for assembly. PCR was completed on a Bio-Rad c1000 thermo     cycler with KAPA Biosystems Hifi Hotstart Readymix following the KAPA Biosystems protocol. It was programmed for an annealing temperature at 58°C and an     annealing time of 00:02:30. After PCR the reaction mixture was treated with Dpn1 restriction enzyme for 1 hour at 37°C to digest template DNA. The linear     fragment was run on a 2% agarose gel stained with SYBR Safe, excised, and gel purified.
pZE_Y12_a12C was obtained as a plasmid, transformed into Mach1 cells for cloning, and purified from liquid culture via mini-prep. The plasmid backbone was     amplified with primers pZgib-F and a12gib-r. The reaction was carried out in Kapa Hifi Hotsart Readymix following the Kapa Biosystems protocol. The     annealing temperature of the reaction was 58°C and the extension time was 00:02:30. The reaction mixture was treated with Dpn1 to digest template DNA and     then run on a 1% agarose gel stained with SYBR Green. The linear fragment was excised and gel purified. Following this, 4000 ng of the linearized backbone     was treated with a single digest with KpnI restriction enzyme, following the NEB protocol to remove the chloramphenicol resistance marker overhang from the     reverse site. The reaction was carried out at 37°C for four hours in a Bio-rad c1000 thermocycler. The reaction mixture was PCR purified with the QIAquick     PCR Purification Kit to yield a linear fragment of the desired backbone vector.
Concentration of the DNA was obtained using the plate reader. The two fragments were assembled with New England Biolabs 2X Gibson Assembly Master Mix     following the NEB Protocol. The resulting plasmid was drop dialyzed for one hour using Millipore drop dialysis filter paper and Milli-Q water. One μL of     desalted DNA was diluted in 49 μL of nuclease free water and used for transformation into Mach1 Cells. Cells were plated on Lb Lennox agar with kanamycin     and allowed to grow overnight in a 37°C incubator.
A dozen colonies were selected for screening. The initial screening with colony PCR was completed with primers pz-integration-f and pz-integration-r and     K2G FAST Readymix with loading dye following the protocol from Kapa Biosystems. The annealing temperature for the reaction was 58°C and the annealing time     given was 0:03:30. The reaction mixture was treated with Dpn1 for one hour to eliminate template DNA and run on a 1% agarose gel stained with ethidium     bromide for 30 minutes at 140V. The gel was imaged on a Bio-Rad Molecular Imager Gel Doc XR+.
A second screening assay was carried out with restriction enzymes KpnI and HindIII in the single and double digest method following the protocols specified     by New England Biolabs. Five clones were grown up in 2XYT media with kanamycin overnight in a 37°C incubator and mini-prepped the following day. Three     hundred nanograms of DNA from each of the clones were used in a double and single digest with KpnI and HindIII. Fragments of the digest were run on a 1%     agarose gel stained with ethidium bromide for 30 minutes at 140V. The gel was imaged using the Molecular Imager Gel Doc XR+. The sequence containing the     taRNA, crRNA, multiple cloning site, and terminator was amplified from purified plasmids using pz-integration-f and pz-integration-r with Hifi Hotstart     Readymix for sequencing. Sequencing was completed by Keck DNA Sequencing Facility. The alignment was analyzed using Geneious.
Assembly of T7 Phage Promoter-super-folder GFP in pZA21 backbone
The plasmid pZA21_T7_sfGFP was assembled from a linear fragment of sfGFP with a RBS and T7 promoter attached to the 5’ untranslated region (UTR) amplified     from plasmid pZE21::sfGFP and the pZA21 backbone amplified from pZA21_tRNA_TAGG. The linear fragment of T7_RBS_sfGFP was PCR amplified from pZE21::sfGFP     with primers pZE21_promT7_F and pZE21_promT7_R and Hifi Hotstart Readymix following the Kapa Biosystems protocol. The annealing temperature for the     reaction was 57°C. The extension time for the reaction was 00:03:00. The reaction was treated with Dpn1 to remove remaining template DNA. Following this,     the reaction mixture was run on a 1% agarose gel stained with SYBR Safe. The band was excised and gel purified following the QIAprep Gel Extraction Kit     protocol. The fragment was circularized using the New England Biolabs 2X Gibson Assembly Master mix and the NEB protocol for Gibson Assembly. The resulting     plasmid was dubbed pZE21::T7sfGFP.
Screening for the replacement of the original PLtetO promoter was verified by sequencing. Sequencing primers for the pZE backbone     pz-integration-f and pz-integration-r were used with Hifi Hotstart Readymix with the NEB protocol to amplify the multiple cloning site of the pZE21     plasmid. The amplified fragment was PCR purified using the QIAquick PCR purification kit and sent to Keck for sequencing. The sequence returned was     analyzed using Geneious.
The pZA21 backbone was amplified with primers T7sfGFP_pZA21_F and T7sfGFP_pZA21_R and Hifi Hotstart Readymix with the NEB protocol. The reaction was     carried out with an annealing temperature of 56°C and an extension time of 00:03:00. The product was treated with Dpn1 to eliminate remaining template DNA.     The reaction mixture was then run on a 1% agarose gel stained with SYBR Safe. The band was excised and gel purified following the QIAprep Gel Extraction     Kit protocol. One thousand nanograms of purified DNA were then treated with a double digest using restriction enzymes AatII and BamHI-HF. The resulting     fragment was PCR purified.
A linear fragment of T7_sfGFP was amplified with overhangs to pZA21 containing the restriction sites AatII and BamHI-HF. T7_sfGFP was amplified from     pZE21::T7sfGFP previously constructed using primers pZE21_T7sfGFP_AatII_F and pZA21_T7sfGFP_BamHI_R and Hifi Hotstart Readymix with the NEB protocol. The     reaction was a two-step PCR amplification. The annealing temperature for step one was 57°C and the annealing temperature for step two was 67°C. The     extension time for both steps is 00:01:10. The reaction mixture was treated with Dpn1 to eliminate remaining template DNA and run on a 1% agarose gel     stained with SYBR safe. The band was excised and gel purified. One thousand nanograms of the purified NDA were treated with a double digest using the     restriction enzymes AatII and BamHI-HF. The resulting fragment was PCR purified.
A ligation reaction using T4 DNA ligase from NEB was conducted with the fragment of PZA21 backbone and T7sfGFP following the NEB protocol. Following the     ligation, 5 μL of the reaction mixture was transformed into Mach1 cells. The colonies were picked the next day and grown up overnight. Liquid culture were     started in 2XYT were started and grown up overnight. The plasmid was mini-prepped and purified. Five plasmid clones were PCR amplified using sequencing     primers pZA_seq_F and pZA_seq_R in Hifi Hotstart Readymix following the NEB protocol. Primers anneal within the pZA21 backbone and amplify the multiple     cloning site. The reaction was carried out with a 56°C annealing temperature and a 00:01:30 annealing time. The reaction mixture was treated with DpnI to     eliminate template DNA and PCR purified. Five microliters of DNA was run on a gel to verify the presence of the fragment. The fragment was sent to Keck DNA     Sequencing Facility and the sequence was analyzed with Geneious.
Anti-Microbial Peptide Construct
Strains, Plasmids, and Reagents
Construct Synthesis and Expression: Strains, Plasmids, and Reagents
The construct sequence was synthesized by Genscript and shipped as  pUC57-Kan_2StrepFLAGLLFP151GFP, and transplanted to the pZE21 plasmid, pZE21_2StrepFLAGLLFP151GFP (BBa_K1396000).   All plasmids were first grown in Mach1, then purified and retransformed into C31POE.ompT.lon.endA.ΔtolC, a recoded strain with all amber stop codons (TAG) replaced, and Release Factor 1 replaced with Streptomycin resistance. It is thus able to encode nonstandard amino acids such as L-DOPA, the incorporation of which is facilitated by the DOPA orthogonal translation system (OTS).
A second strain was made with the Tyrosine suppressor system transformed instead, so the construct can be expressed with tyrosines in the place of L-DOPA, as L-DOPA is very toxic to cells. The construct was then separated into smaller constructs such as pZE21_LLFP151 (BBa_K1396001), pZE21_FP151GFP (BBa_K1396002), pZE21_FP151 (BBa_K1396003).
We hypothesize that we can develop an improved version of the current adhesives by developing a fusion protein of Mgfp-5 with Mefp-1 as the anchoring region for the anti-biofouling peptide. An integral part of developing this peptide is to co-translationally insert L-DOPA into our peptide, which has never been done before with mussel foot proteins (Figure 1). In this process of orthogonal translation, we first will get rid of the UAG stop codon and then transform the strain to synthesize tRNA and tRNA transferase that corresponds to the UAG codon and the L-DOPA non-standard amino acid to develop the genetically recoded organism (GRO). The advantage of this procedure is that we have the ability to skip the time-consuming and inefficient tyrosinase enzyme treatment step.
Figure 1. Integration of L-DOPA into peptide through orthogonal translation.
Protein Purification We plan to purify the protein by using the Twin Strep Tag in tandem with the FLAG tag, which was included in our master construct of the anti-biofouling peptide (Figure 2).
Figure 2. A diagram illustrating the components in our final construct. The Twin Strep and FLAG tags are indicated.
The FLAG tag is perfectly cleavable by the enzyme enterokinase. The FLAG tag is made up of 8 amino acids and works well for low-abundance proteins. It is hydrophilic, so it will most likely not interfere with protein folding and function of the target protein. The Strep tag is also made up of 8 amino acids that will not disturb the protein’s functions. We chose the FLAG tag because it is perfectly cleavable. The protein will be purified in a Strep-TactinR SepharoseR column. In order to address the adhesive L-DOPA component, our final step is to elute with a base to reduce the amount of the anti-biofouling peptide that sticks to the column due to L-DOPA adhesion (Figure 3).
Flag Tag Sequence:  Asp-Tyr-Lys-Asp-Asp-Asp-Asp-Lys
Strep Tag Sequence: Sequence: Trp-Ser-His-Pro-Gln-Phe-Glu-Lys
Figure 3. A diagram illustrating the proposed purification method.
Methods for Assaying Coating Adhesion Properties
Strains, Plasmids, and Reagents
Adhesion:
Subject peptide coated surfaces to liquid erosion:
A number of ASTM assays used in industrial coating testing were investigated, but none offered the level of quantitation desired for our applications. Therefore, an original rig was designed and built to introduce liquid based erosion by laminar flow through a bath. This system directly mimics the drag that a coated surface might experience on a ship's hull. Precise specifications of the rig are provided in a separate section below.
Figure 4. A diagram illustrating the configuration of the erosion rig developed to introduce coated surfaces to liquid erosion.
Assay presence of peptide on eroded surfaces:
Quartz Crystal Microbalance (QCM): A QCM is capable of measuring mass per unit area on a very sensitive scale. The QCM used in these experiments recorded masses with ±1 ng/cm2 uncertainty. The way this instrument works is by measuring change in the resonance frequency, which is converted into a mass estimate on the basis that resonance frequency will decrease with increasing mass. We intend to subject the quartz crystals to varying levels of erosion and determining coating retention from the QCM mass measurement. Alternatively, various flow cell and in-situ erosion techniques can be coupled to the QCM to show the real-time changes in resonance frequency due to loss of mass. Several labs and facilities assisted with planning and execution of this measurement, including Dr. Michael Rooks at the Yale Institute for Nanoscience and Quantum Engineering and Dr. Islam Mosa in the lab of Dr. James Rusling in the Department of Chemistry at the University of Connecticut.
Total Protein Staining and Fluorescence Imaging: Coomassie Blue was used as a total protein stain to determine presence of coating on eroded slides. Adsorbed protein content can theoretically be determined visually from density of stain. Since our construct was designed with an sfGFP domain, we intend to assay presence of our peptide with fluorescence.
Contact Angle Measurement: A contact angle measurement of protein coated silica substrates was conducted as an indicator for presence of peptide, protein hydrophilicity/hydrophobicity, and surface energy. Wetting surfaces show a shallow contact angle, while hydrophobic surfaces show a larger contact angle. A contact angle characterizes the wettability of a surface and Young's equation can be used to determine interfacial energies between the three phases in equilibrium, given below. Note that γXY corresponds to the interfacial energy between phase X and phase Y.
0=γSG – γSL – γLSCos(θC)
This measurement was conducted with the assistance of Dr. Raphael Sarfati, Dr. Katharine Jensen, and Dr. Rostislav Boltyanskiy in the lab of Dr. Eric Dufresne in the Yale Department of Mechanical Engineering.
Fourier Transform Infrared Spectroscopy (FTIR): As a further test to determine if material is adhered to surfaces, we will use Fourier Transform Infrared Spectroscopy (FTIR). The cured adhesive film should exhibit a different spectrum than the uncured adhesive. A notable difference would speak to a change in vibrational bond energies caused by coordination or bonding to our surface.
Assay peptide adhesion strength:
Atomic Force Microscopy (AFM): The standard for measurement of the force of adhesion of MAPs is AFM. This type of measurement is known as a "pull-off" force determination and involves depressing an AFM cantilever functionalized with a 20 µm bead until it comes in contact with a coated substrate surface. The instrument then determines the force required to remove the cantilever from the substrate.
Figure 5. This is an AFM cantilever with a 20 µm silica bead fixed to the tip. By functionalizing the tip, we can control the adhesion interface for which we test our MAP adhesives. In this case, we intend to use a silica bead to measure the adhesion of our coating to a silica interface. This measurement was conducted with the assistance of Dr. Michael Rooks at the Yale Institute for Nanoscience and Quantum Engineering.
Optical Tweezers: While AFM has been used in many MAP studies successfully to measure MAP adhesion force, it comes with some limitations. Inevitably, there is some significant contact area, which makes the adhesion measurement read the adhesive force of multiple proteins. However, the technology exists to measure adhesion on the individual protein level. Some studies have measured the adhesion force of L-DOPA on the single molecule level by chemically linking the L-DOPA residue to the AFM cantilever. However, no such study have looked at adhesion force on the single protein level. Using high intensity lasers, one can engender a repulsive force between two beads in relation to their refractive indices. We intend to link our MAP to a biotin functionalized bead and measure its adhesion to a silica bead substrate.
Figure 6. This diagram illustrates how a DNA handle can be linked to a protein of interest to bind the protein to an optical tweezer bead into which the high intensity laser can be fired to engender a pull force. We intend to conduct a similar protocol with our adhesive peptide.23 This measurement was conducted with the assistance of Dr. Junyi Jiao in the lab of Dr. Yongli Zhang from the Yale Department of Cell Biology.
Preparation of Cell-TakTM films
Cell-TakTM is a mixture of mussel foot proteins 1 and 2 in an approximate 75% to 25% ratio, respectively. The protocol used to deposit Cell-TakTMrelies upon the property of L-DOPA to come out of solution when the pH is increased. Cell-TakTM is constituted in 5% acetic acid. To prepare a film, 10 µL of a 2.36 mg/mL Cell-TakTM solution was spotted onto the substrate. To trigger binding, 10 µL of 0.2 M sodium bicarbonate was added to increase the pH to ~8-9. Films were then allowed to dry for 30 min at 37ºC and then washed with DI water. Prepared films were stored in an incubator until ready for use.
Spin coating was investigated as a new modality for depositing a protein monolayer. However, the hydrophobicity of the protein did not allow for adequate wetting of the substrates by Cell-TakTM for proper spin coating. In addition, a method used for hydrogel preparation involving incubation of a coverslip in Rain-X was tested to deposit protein layers. A coverslip was coated in Cell-TakTM and then covered by the super hydrophobic Rain-X coated coverslip. The coverslips were then pulled apart and the protein coated surface was allowed to dry. However, the consistency in thickness desired for our application made the precipitation preparation the method we chose as our standard.
Main Campus: Molecular, Cellular & Developmental Biology 219 Prospect Street P.O. Box 208103 New Haven, CT 06520
Phone: 203.432.3783
igem@yale.edu
natalie.ma@yale.edu (Graduate Advisor)
Copyright (c) 2014 Yale IGEM
Retrieved from " http://2014.igem.org/Team:Yale/MaterialsMethods "
Recent changes
What links here
Related changes
Special pages
My preferences
Printable version
Permanent link
Privacy policy
Disclaimers
