"
Page
Discussion
View source
History
teams
Log in
 
Team:DTU-Denmark/Repressor Section
From 2010.igem.org
Welcome to the DTU iGEM wiki!
Home
The Team
The Project
Parts submitted
Results
Notebook
Blog
Repressor - Anti-Repressor Section
Construction of BioBricks
Characterization
Strategy
Results
Terminator - Anti-Terminator Section
Characterization
Strategy
Results
Construction of BioBricks
Construction of Parts
Synthetic Promoter Library Section
Construction of BioBricks
Characterization Results
Strategy
Results
Introduction
The Gifsy promoters with their respective repressors and anti-repressors are important regulatory elements in the switch and this is why we wanted to characterize them and submit them as BioBricks. We already know that this system is tight in its natural settings in Salmonella, but to our knowledge no one before us has tried to characterize the system in Escherichia coli. A good way to characterize it in E.coli is to test whether the repression of the pR promoters is tight, and if the anti-repressors indeed do antirepress. Because of previous knowledge of the strength of the promoters in Salmonella all characterization had to be done in low-copy-number plasmids.  We have characterized the following BioBricks:
BBa_K374008
BBa_K374009
BBa_K374010
Construction of BioBricks
BioBricks BBa_K374008 and BBa_K374009
In wildtype Salmonella enterica Serovar Typhimurium (strain ATCC 14028) the Gifsy immunity region, consisting of the two divergent promoters pR and pRM and the repressor gogR/gtgR downstream of the pRM promoter, are present in the chromosome.  We designed a primer pair to amplify the immunity region from Gifsy 1 and Gifsy 2, and by adding the standardized prefix and suffix as a tail, the amplicons were made Biobrick compatible. The amplicons were made by PCR. After the PCR the fragments were cleaned up using a PCR clean-up kit .
The amplicon as well as the linearized backbone plasmid pSB1C3 were digested with the restriction enzymes EcoRI and PstI leaving sticky ends.  Both the digested PCR product and the digested plasmid were run on a gel in order to estimate DNA concentration.
The ligation was made using T4 ligase with a 5:1 ratio of insert to backbone. After the ligation the plasmid was transformed into electrocompetent DH5α E. coli cells. After an hour of recovery in LB media at 37 °C the cells were plated on LB plates containing chloramphenicol (Cam).
The next day five colonies on the plates were selected and restreaked on LB+Cam plates in order to assure pure colonies.  Overnight cultures were made of the transformants by taking one colony from each restreak and inoculating it in LB+Cam at 37 °C over night. Minipreps were made from the overnight cultures and a verification PCR was run on these in order to assure that the plasmid had the expected insert.
Figure 1 Left: GeneRuler DNA ladder mix from Fermentas. Right: Verification PCR of the Biobricks K374008, K374009, K374010. Lanes 1-5 show the expected amplicon of 650 bp from a PCR using the iGEM primer VR as reverse and a forward primer which anneals to the Gifsy2 immunity region. Lanes 6-10 show the expected amplicon of 672 bp from a PCR using the iGEM primer VR as reverse and a forward primer which anneals to the Gifsy2 immunity region. Lanes 11-13 show the expected amplicon of 450 bp from a PCR using the iGEM primer VR as reverse and a forward primer which anneals to the anti-repressor AntO. The Biobricks sent in are the ones corresponding to lanes 4, 9 and 11.
BioBrick BBa_K374010
The Gifsy1 anti-repressor AntO was amplified by PCR using a plasmid containing the anti-repressor as template (Sebastién Lemire). The primers had tails corresponding to the standardized prefix and suffix making the amplicon BioBrick compatible. Once we had the amplicon it was digested, ligated and transformed into DH5α E. coli cells following the same procedure as in the above mentioned BioBricks. The results from the verification PCR can be seen in figure 1.
Finally we decided to have the submitted BioBricks sequenced. To our joy, the results showed that our BioBricks do not contain any mutations.
Characterization
Strategy
In order to characterize the BioBricks we constructed three plasmids for each Gifsy phage using fluorescent proteins as reporter genes, see table 1. We used a BioLector microreactor system for our measurements. The BioLector is a microfermenting system that measures light scattering (a measure of cell density) and fluorescence simultaneously, which is ideal to characterize our constructs.
Table 1: The table shows our characterization plasmids
Plasmid name
BioBrick
Insert
Reporter
Backbone
Source
pIGR01
-
Gifsy1 promoters
GFP
pSB4A5
Gifsy1
pIGR02
-
Gifsy2 promoters
GFP
pSB4A5
Gifsy2
pIGR03
BBa_K374008
Gifsy1 promoters with GogR repressor
GFP
pSB4A5
Gifsy1
pIGR04
BBa_K374009
Gifsy promoters with GtgR repressor
GFP
pSB4A5
Gifsy2
pIGR05
BBa_K374010
pBAD promoter and AntO
RFP
pSB3C5
Gifsy1
pIGR05
-
pBAD promoter and AntT
RFP
pSB3C5
Gifsy2
In the first plasmids (pIGR01 and pIGR02) we cloned a GFP reporter downstream of the pR promoter, because this would give us the level of GFP expression from the pR promoter when the represser is not present, see figure 2.
Figure 2 Characterization plasmid pIGR01/02, containing promoters and GFP reporter. Expression of GFP shows that the pR promoters are working as expected.
The next plasmids (pIGR03 and pIGR04) contain GFP downstream of pR and the repressors (GogR or GtgR) downstream of pRM, see figure 3. These plasmids would allow us to see if the repressors repress the pR promoter – causing expression of GFP to cease. If no GFP is expressed, the repressor system is tight.
Figure 3 Characterization plasmid pIGR03/04, containing promoters, repressor and GFP reporter. No expression of GFP shows that the repressor is working as expected.
The last plasmids, (pIGR05 and pIGR06) contains the arabinose inducible promoter pBAD with antirepressor antO/antT and RFP located downstream, see figure 4. This means that if there is expression of RFP, the anti-repressor is also expressed assuming that both proteins are equally translated.  pIGR05/06 are designed to have a different origin of replication than pIGR03/04 which allows transformation of both plasmids into the same E. coli strain. This enables us to see the influence of the anti-repressor on the repressor. We expect to see expression of both GFP and RFP, as the anti-repressor will hinder the effect of the repressor on the pR promoter.
Figure 4 Characterization plasmid pIGR05/6, containing pBAD promoter, antirepressor and RFP reporter. Expression of RFP shows that the antirepressor is expressed.
All the plasmids were assembled using the iGEM 3A assembly standard , and they were verified using PCR.
Results
Our first discovery was that when growing the E. coli transformants, carrying the pIGR01 or pIGR02 plasmid, we saw a wide variety not only in colony size, but also in level of GFP expression. Colonies with visible expression of GFP were significantly smaller in size indicating that the pR promoter expresses GFP to a level that becomes toxic to the cells,see figure 5.
Figure 5 Agar-plate with colonies of E. coli carrying pIGR01. We se small colonies expressing GFP.
The transformants carrying pIGR03 or pIGR04 did not show any expression of GFP – as expected.
Figure 6 Agar-plate with colonies of E. coli carrying pIGR02 and pIGR03. We se that colonies carrying plasmid pIGR02 are expressing GFP (left), and that colonies carrying pIGR03 are not expressing any GFP(right).
These results were very interesting and must be considered when making the final design for the switch. We might even have to consider the need for a different reporter in our final design of the switch.
Sequencing of characterization plasmids
Because of the differences in colony size and GFP expression that we saw on the transformation plates, see figure 5 and 6, we decided to have the plasmids pIGR01-04 sequenced. Results show that out of three chosen colonies two were mutated for strains carrying pIGR01 and one was mutated for strains carrying pIGR02. For pIGR03 and pIGR04 we chose two colonies carrying each plasmid, and we found one silent mutation in the repressor gogR, but none in the rest.
Biolector
We picked three different colonies from the four constructions that worked (pIGR01, pIGR02, pIGR03 and pIGR04) and ran them on the BioLector.
The results from the BioLector run will be presented here.
BioBrick BBa_K374008 (Gifsy1)
Figure 7 shows the change in biomass over time for the three different strains DH5α, pIGR01 and pIGR03. The growth curves show that the cells without the repressor (pIGR01) grow slower than the cells with the repressor (pIGR03). This might be because the cells without the repressor expresses GFP which might be toxic to the cells. The reference strain DH5α seems to grow at the same rate as pIGR03.
The biomass shift for pIGR01 around (5;200) indicates that the cells change morphology or cell size due to a re-arrangement of metabolism in order to utilize other aspects of the growth media after the exhaustion of others. The drop in biomass around (7;300) can be explained by the complete exhaustion of media.
Figure 7 The change in biomass (Light Scattering Units) over time (hours) for the three different strains.
Figure 8 shows the change in GFP expression over time. The strain only containing the Gifsy1 promoters (pIGR01) show expression of GFP as expected while, the strain containing both the Gifsy1 promoters and the GogR repressor (pIGR03) shows no GFP expression as expected. The DH5α reference strain shows no expression of GFP which is also expected.
Figure 8 The change in GFP (Fluorescence Units) over time (hours) for the three different strains.
Figure 9 shows the normalized GFP expression over time.  When the GFP expression is normalized with biomass any variability in GFP expression due to difference in growth is taken into account.
We would have expected that the specific activity of GFP, and thereby the promoter strength, to be constant at all times. However, a constant rate is not reached until after seven hours.
Figure 9 The normalized change in GFP (FU)/(LSU) over time (hours) for the three different strains.
In Figure 10 the GFP signal is shown according to biomass. As expected we can see a signal of GFP in the strain without the repressor (pIGR01) and no signal in the other two strains (pIGR03 and DH5α).
The linear slope(s) correspond(s) to the specific activity of GFP and thereby also relative promoter strength. The shift in biomass clearly affects this plot. However, the linear slope before and after the shift are equal.
Figure 10 The normalized change in GFP (FU) over time (hours) for the three different strains.
BioBrick BBa_K374009 (Gifsy2)
Figure 11 shows the change in biomass over time. All three strains have approximately the same growth rates.
Figure 11 The change in biomass (LSU) over time (hours) for the three different strains.
Figure 12 shows the GFP signal over time. As expected we see expression from the strain without the repressor and no signal from the strain with the repressor
Figure 12 Expression of GFP (FU) over time (hours) for the three different strains.
In Figure 13 the normalized GFP expression over time is shown and as expected expression occurs solely in the strains without the repressor. However, we would expect a constant rate at all times, but first after five hours we obtain that.
Figure 13 The normalized expression of GFP (FU)/(LSU) over time (hours) for the three different strains.
The GFP signal over biomass is shown in Figure 14. Only the strain without the repressor shows a GFP signal. As for the Gifsy1 we also see a shift in biomass, which can be explained by a metabolic change.
Figure 14 Expression of GFP (FU) over time (hours) for the three different strains.
Conclusion
We have done measurements of the two divergent promoters pR1Gifsy1 and pR2Gifsy2 with and without their repressors GogR and GtgR respectively.
It is clearly demonstrated that both of the repressors represses their respective promoters 100%. In addition to the run in the BioLector the DNA has also been sequenced. The sequencing data for the submitted BioBricks  showed no mutations in neither the promoter region nor the repressor, indicating that the lack of GFP signal is indeed a consequence of repression.
The difference in GFP expressions from Gifsy1 and Gifsy2 indicates that the pR1Gifsy1 promoter is stronger than the pR2Gifsy2. See Figure 8 and Figure 12.
Retrieved from " http://2010.igem.org/Team:DTU-Denmark/Repressor_Section "
Recent changes
What links here
Related changes
Special pages
My preferences
Printable version
Permanent link
Privacy policy
Disclaimers
