"
Page
Discussion
View source
History
teams
Log in
 
Team:DTU-Denmark/Notebook
From 2010.igem.org
Welcome to the DTU iGEM wiki!
Home
The Team
The Project
Parts submitted
Results
Notebook
Blog
Lab Protocols
BioLector
Ligations
PCR product purification
Plasmid purification by miniprep
Restriction/Digestion
Transformation
BioLector
Safety Considerations
October
23-10-2010-Maya-Repressor group
Yesterday Sebastien transformed pIGR06 into a Salmonella strain to test whether the antirepressor works. Today I induced the pBAD promoter, which is infront of the antirepressor, by adding arabinose. After one hour I could see that the cells carrying the antirepressor-plasmid (pIGR06) lysed. This means that the antirepressor works:-)
We tried once more to verify pIGR12, but the bands we got had really weird sizes.
We also made the cells carrying pIGR03 and pIGR04 competent, and transformed pIGR05 and pIGR06 into these.
22-10-2010-Maya-Repressor group
So we haven't been writing in the log for the last days as we have been really busy working in the lab and writing on the wiki. But after having realized that the Antiterminatorgroup still writes the log, I decided to write a little update.
First of all we sent some stuff in for sequencing to find out whether we have mutations, as we thought we had. We got the results yesterday, and we found out that pIGR01, pIGR02, pIGR03, pIGR04 are not mutated. So the measurements we did on the biolector with these strains can indeed be trusted.
Finally we had success in verifying pIGR11 meaning that the antirepressor antO is ready for submission.
We also did a new round of ligation and transformation to get the pIGR05, pIGR06 and pIGR12 that we still haven't been able to verify. We changed the backbone in the pIGR05 and pIGR06 plasmids to pSB3C5, and this seemed to be a good idea because this time we got colonies that we could actually verify by PCR!!!!! :-) This means that we can prepare another biolector run, this time having all the planned constructs. But first we need to transform pIGR05 into a strain containing pIGR03, and pIGR06 into a strain with pIGR04.
We haven't been able to verify pIGR12, so we will do another attempt tomorrow.
We also did another experiment with Salmonella, but unfortunately this failed. We transformed pIGR03 and pIGR04 into Salmonella strains containing the antirepressors under the control of a pBAD promoter. Cultures were launched from the transformed colonies, and after having reached an OD of 2, some culture was transferred to LB+arabinose. The arabinose should induce the pBAD promoter causing expression of the antirepressor, again causing induction of the prophages, and de-repression of the repressor in the pIGR03/pIGR04 constructs which would finally result in GFP detection. We did indeed see lysis of the cells, but unfortunately no GFP expression could be measured when using the fluorometer. This result indicates that our plasmids produces too much repressor for the antirepressor to de-repress.
Lastly, we registered all our biobricks in the partsregistry, and sent them all to the iGEM headquarter.
21-10-2010 - Patrick - Antiterminator group
Seemed like some of the cultures neither grew in the O/N cultures nor on the re-streaked plates, so new colonies were picked in order to have the 24 needed cultures for BioLector measurements.
Once these had grown to what seemed to be exponential phase, OD was measured using Nanodrop and amounts of cultures needed for BioLector were calculated. This followed by aliquoting the calculated amounts of the cultures (which apart from the new cultures, were in stationary phase) into fresh LB media in order to let them grow from ~ exponential phase and measured using BioLector; each culture was ran in duplicates.
After the BioLector was started, we planned out the next set of measurements to run tomorrow which would be of SPL+RFP, J23100+RFP, J23101+RFP and J23116+RFP. O/N cultures were started of these combinations and will be ready for tomorrow's next BioLector run; the chosen colonies were also re-streaked for future reference.
20-10-2010 - Patrick - Antiterminator group
The transformations that were re-done using the old batch of electrocompetent DH5-alpha cells turned out to have worked quite well, the negative ligation control plate had very few colonies whereas the plates containing the transformants had significantly more colonies, so very little background noise in our transformants. The transformations that were done with XL1-blue had tiny colonies on them, so we need to let them grow longer in the incubator before we can make any conclusions on them.
The remainder of the transformation cultures that were used to innoculate fresh LB media were measured using FACS, though as there were difficulties calibrating that spaceship and the uncertainty of FP references, data was obtained but there is bigger certainty that the BioLector measurements using single colonies will give more precise results.
The plates containing colonies from EXII (reference - 19-10-2010) were used to make O/N cultures for BioLector measurements, these colonies were also restreaked for future use.
BioLector measurements are set to take place tomorrow (21-10-2010).
All parts to be sent to iGEM are now ready.
19-10-2010 - Patrick - Antiterminator group
Seeming as the transformants were first plated at 21:00 last night, couldn't make any conclusions after having looked at the plates in the morning; these will be left until the evening and checked again.
As the newest batch of competent cells have some funky contamination, we re-did the transformations from yesterday using our oldest batch, although these are not extremely competent.
Giving this a second thought, the ligations were remade from scratch again, using more DNA content, adding more ligations as well now that J23101 and more of I13507 were digested and ready for ligations.
For these new ligations, a lab technician from another lab was kind enough to give us some XL1-blue electrocompetent cells which as we were told, were awesomely competent! :D
These actually being alot more efficient when wanting to induce high copy number "mode" in pSB2K3 with IPTG as this strain contains lacIq as apposed to DH5-alpha which only contains lacI.
Cultures have been prepared for FACS measurements for tomorrow.
18-10-2010 - Patrick - Antiterminator group
pAT12-pAT16 were digested, cleaned up and run on a gel, they seemed good and were ready to be ligated.
SPL was ligated with combinations between pAT12-pAT16 into pSB2K3; J23100 and J23116 promoters were also ligated with pAT15 into pSB2K3 as references for measuring with SPL.
Once the above mentioned ligations were complete (x10) they along with J23100, J23116 and J23101 were transformed into DH5-alpha cells and then plated in amounts of 20ul, 100ul and 200ul.
Hopefully the plates will give some good results tomorrow.
18-10-2010 Repressor group Annemi, Lisa and Maya
Ran a gel of the verification PCR Lisa did yesterday. Unfortunately the bands were nowhere to be seen.
Did a PCR of the pIGR12 construct (Z2). Looks like we might have an insert in some of them. However in the lanes where our own primerF was used, no bands were visible.
Wrote some descriptions of our BB's GR1, GR2 and Z1 and got it approved by Seb. So we should be ready to send those within the next few days.
Sent the minipreps to sequencing and are now very excited to get the results.
Started O/Ns of the minipreps that were sent for sequencing so we have backups.
Thomas was very kind to do a verification PCR on our pIGR05 and pIGR06 constructs.
17-10-2010  Juliet and Patrick- Antiterminator group
I (Juliet) learned what not to do in the lab today :P but then Patrick and I made a new gel and ran a gel of the (50X) re-verification PCRs that Thomas made last night. We found that most of them were successful so we were able to proceed and make duplicates of PCRs of pAT12-pAT16.
A gel was ran of the PCR's and their clean ups and turned out good, they are ready to be digested, ligated and transformed tomorrow.
Parts to be sent to iGEM HQ have also been registered in the parts registry.
16-10-2010 - Thomas - Antiterminator group
Anastasiya and I did minipreps on the 30 overnight cultures that were started yesterday. We ran the minipreps on a gel and chose 25 plasmids to run verification PCRs on (so we only had to use 2 machines, and not 3). The results from the 50 PCRs were unfortunately inconclusive, so I redid them. I let them run in the machines overnight, so they are ready to be run on a gel tomorrow (amazing how 9-10 hours of work can be described with just 4 lines!).
16-10-2010 Repressor group Annemi
Ran a gel of the varification PCR products Lisa did yesterday. Some bands around 1200 bp and over 3 kb are slightly visible, however they don't look like PCR products. Lisa will redo the PCR tomorrow.
Did a miniprep of the c-colonies 1-13 (except 7 which showed very little growth). Hopefully the minipreps c1-c6 will contain pIGR05 (pBADZ1, RFP), c6-c10 will contain pIGR06 (pBADZ2, RFP) and c11-c13 will contain RFP.
Ran the minipreps on a gel. c11 looks gives a very weak band and no bands are visible in the lane with c13.
15-10-2010 - Thomas - Antiterminator group
Juliet and I did a PCR on pAT15 with primers IG201 and IG202, cleaned it up and ran it on a gel to reconfirm that the insert was correct. It was, so we started a digestion of the PCR product using X and P. We cleaned up the digestion and ran it on a gel to see the concentration. pAT15 is now ready to be ligated to the SPL! :). We also made O/N cultures of pAT12-16 (6 of each, 30 in total).
After the digestion, Manos was ready to help Maya run her samples in the Biolector. Juliet and I helped her set it up, which took alot longer that expected since you have to measure the OD of each sample with the Nanodrop! With Manos' guidance, we loaded their 16 samples in the 48 wells (we did triplicates) and started the measurements.
15-10-2010 Repressor group
Verification PCRs were done. First batch were for verifying some old constructs of the biobricks that we're submitting. But as usual, the PCR failed. Maybe I shouldnt do 40 pcrs at a time (Lisa). Next batch was to verify some newer construcst of BBs as well as a construct with pBADZ2 (pIGR06). Gel needs to be run tomorrow.  Also tried to make a table of all of the restreaks that we have, to get an overview, and we updated the other table of our constructs with which primers and melting temp you should use for verification.
14-10-2010 - Thomas - Antiterminator group
Anastasiya did minipreps of the O/N cultures of pAT13 and pAT15. Meanwhile I did PCR cleanups of the pSB3C5 and pSB3T5 backbone plasmid PCRs that Patrick did yesterday. I then did digestions of pSB3C5, pSB3T5 as well as J23100 and J23116, the two BioBrick promoters we plan to use as references in our measurements. After the minipreps were done, we ran verification PCRs on both plasmids and saw that only pAT15 had the correct insert.
The ligations of pAT12-16 that were done yesterday seemed to have worked this time! Lisa and I restreaked six colonies of each construct, while Patrick did restreaks of the SPL + GFP construct, that also seemed to have worked.
NOTE: It seems the newest batch of competent cells has been contaminated with some unknown E.coli strain that is also Amp resistant. This sucks alot if you are transforming plasmids with only Amp resistance. Sebastien says we just have to deal with it, since it's too late to waste time making a new batch (unless someone has nothing to do). Luckily the unknown E. coli strain seems to grow ALOT faster than DH5alpha, the "bad" colonies are giant compared to the "right" ones, so just avoid those when doing restreaks after a transformation.
13-10-2010 - Patrick - Antiterminator group
A gel was run of pcr cleaned up products: pAT02, pAT08, J23100 and J23116, these turned out good and are ready to be digested.
As the transformed products that were re-plated yesterday didn't turn out to give any promising results, the ligations were remade of pAT12-pAT16. This time all new digestions seemed good and newly made competent cells were used as well. A ligation was made as well of SPL + GFP(+RBS) into pSB4A5, this merely to see the variation in GFP expression amoungst the colonies. The ligations were transformed and left to recover for 2 hours as apposed to the usual one hour, and then plated afterwards.
Hopefully these new ligations work and it was merely a digestion problem before and nothing to do with the usage of pSB2K3 as the backbone nor resistance expression.
12-10-2010 Maya Repressor group
Restreaks of pIGR01 and pIGR07 (isnt it pIGR02? lisa 13-10) looked liked the previous restreaks. The colonies that were really green, were still really green.
I made ligations of pIGR07 and pIGR08, and when I transformed I included that ligation mixture made the 06-10-10 containing pBADz1/z2 + RFP.
Then I spend some time talking to Sebastien about sequencing. I will call the company tomorrow and ask what we need to do.
I also talked to Jacobo about using the biolector Friday, and he will just make sure that he is not using it.
12-10-2010 Patrick Anti-terminator group
Our constructs pAT12-pAT16 didn't seem to work, first reason could be cause of the pSB2K3 plasmid used, perhaps the cells were not left long enough to recover and hence express kanamycin resistance. Another reason could be the digestions made of pAT09-pAT11, as it seems the restriction enzymes were not given enough time, the gel illustrated that ~ half of each product was digested whilst the other half was not.
Bottom line is, the transformed products were re-plated today in case they needed more time for resistance expression; all the products needed for the pAT12-pAT16 constructs were pcr'd and digested and ready for new ligations of pAT12-pAT16 if that is the case.
Tomorrow a test will be made with the SPL, which will be ligated with GFP(+RBS) into pSB4A5.
11-10-2010 Maya, Lisa and Annemi
Another day in hell... Our restreaks with pBADz1/z2 and RFP don't seem to work. They don't grow up on glucose minimal media, and definitely not on ara+IPTG either. When looking at the original transformation plates it can also be seen that the colonies have different sizes indicating accumulation of mutations. This could be because the RFP or the antirepressor are toxic to e.coli, or because of something else. Anyways, we decided to transform the ligation mixture again, and this time plating on LB+Kan so that we don't induce RFP and antirepressor.
RFP and GFP PCR products were purified, digested, purified again and run on a gel. They are ready for a ligation tomorrow.
We also restreaked our pIGR01 and pIGR02 constructs to see whether the very green colonies will stay green meaning that in some way they are stable, or whether they will become white meaning that there is a pressure for accumulations of mutations in order to get rid of GFP.
11-10-2010 Anastasiya and Patrick Anti-terminator group
Started off the day by running digestions made yesterday (pAT09, pAT10, pAT11) on a gel, these were then pcr cleaned up, and then ran on a gel again. On the gel we could see that the products were half digested (half were digested, other half not), thus we repeated the digestions of pAT09, pAT10 and pAT11.
Using the digestions that were purified, we started the new ligation series of pAT12, pAT13, pAT14, pAT15 and pAT16, which are the 2nd final constructs before adding the SPL's.
These were then tranformed and plated.
Digestions were also made of SPL (E+S) and GFP+RBS (X+P) which will be ligated into pSB4A5 (E+P).
Note: pcr clean ups of pAT09, pAT10 and pAT11 were all used up, if more needed of these products, new pcr's should be initiated.
10-10-10 Maya Repressor group
I made O/N cultures of the restreaks from 09-10-10. The restreaks are from the transformations made 06-10-10. There are no O/N of constructs containing pBADz1/pBADz2 as these grow much slower and were therefore not ready.
08-10-2010 Maya P2 lab
100 ul of the O/N cultures originating from pIGR03 and pIGR04 were mixed with 3 ml of top agar and plated on LB plates with Amp. On top of the agar I added a small disc with 5 ul of mitomycin and a disc with 5 ul of water as a control. Ideally mitomycin should induce phage induction and thereby initiate expression of the antirepressor. The antirepressor should then de-repress the repressor allowing expression of GFP.
07-10-2010
Did transformations of the ligations made yesterday and plated these out on the right antibiotic plates. pBADZ1/2 were plated on the four different plates described yesterday. Hopefully we will see red colonies tomorrow.
Did a PCR in parallel with the T-group where we used miniprep 4, 6, 11 and 16 (from october 2nd) and two of the T-group minipreps (which are known to give bands). It will be exciting to see if we can't make a PCR or we have just been unlucky.
07-10-2010 Maya - Still Depressor group
Yesterday I just did some restreaks of the transformations I made 05-10-2010. Today I had a look at these, and it was very obvious that the pIGR01 (G1+GFP) construct in the 421 (WT) strain looked really sick. This is most likely because the pR promoter is too strong and we get too much GFP expression. We thought that the repressor, which is naturally in the salmonella, would repress the pR promoter, but this was not the case.
The other constructs seemed okay.
I had a look at some of the strains in the microscope in order to see whether they expressed GFP. These were:
pIGR02 (G2+GFP) in 421 (WT)
pIGR02 (G2+GFP) in 413
D: pIGR03 (GR1) in 421 (WT)
C expressed a lot of GFP measured at 14 ms. It was very clear that the cells were not doing too good:-)
B expressed 30% less GFP (says Flemming). We would have expected the repressor from the Salmonella prophages to shut down expression from the pR promoter more efficiently.
D hardly expressed any GFP. This was both good and bad. Good because it indicates that we have a correct plasmid since GFP is present. Bad because the repressor in the GR1 construct should shut down expression 100%.
I made O/N of the restreaks from yesterday.
06-10-2010 - Greg and Juliet - Wiki
Greg worked on some of the figures and changed the layout for our poster. I worked on the wiki, changing the layout ~ essentially beautifying it and so the plan is to read through the content and figure out what needs to be edited and added and who should do it (even if it's me) ;) which means that I will probably be bossy and demand things from people, so be prepared! ;)
06-10-2010 - Patrick - Anti-Terminator group
The SPL PCR product was ran on a gel, and looked quite good having a band size just around 1kb, its actual size being 941bp, so thats pretty good, PCR clean up was then performed on it.
The ligation of pAT12 didn't seem to work, so new ligations were made today for pAT12 using different concentrations of the inserts and vector, these were then transformated and plated; they will be checked tomorrow to see if concentration was the issue.
O/N cultures were made of pAT09, pAT10 and pAT11, these need to be mini-prepd tomorrow as well as PCR verified to check if we indeed have pAT09, pAT10 and pAT11.
Competent cells will be made tomorrow, although this will be coordinated with Sebastien.
05-10-2010 - Patrick - Anti-Depressor Group: The Terminators!!
So it seems like our competing team isn't doing too well, so we will try helping them out tomorrow if they need it. :)
During the lab today, started off by running the digestions made yesterday on a gel, namely pAT08(E+S) and pSB2K3(E+P).
They looked good and so proceeded to doing a pcr clean up followed by running them on a gel again in order to determind dna concentration for calculations needed for ligations.
Restreaks were made of the ligations plated yesterday, namely pAT09, pAT10, pAT11; if all looks good tomorrow on the plates then O/N cultures will be made.
The pAT12 construct was made today by ligating the following together:
pAT08(E+S) + pAT02(X+P) + pSB2K3(E+P).
A negative ligation control was made as well which contained only the digested plasmid (pSB2K3) and no insert.
These were then transformed along side negative and positive transformation controls, the positive control had pBR322 as the plasmid for transformation.
After 1hr incubation at 37C, these 4 transformants were plated and hopefully tomorrow we have some good results.
A PCR was also made of the SPL, the way it was designed was that it would have the Chloramphenicol cassette along side it in order to make digest+ligation alot easier as well as having a selectable marker to work with.
The template used in order to obtain the chloramphenicol cassette was pKD3, which was kindly supplied by Sebastien.
Note: I made the PCR using the Phusion enzyme, running a normal PCR program, not the one we've always been running which contains ramp TD settings; I did this PCR on the old PCR machine in the 2nd last lab down the hallway.
The PCR product needs to be run on a gel tomorrow, so lets cross our fingers it will be successful.
05-10-2010 Maya DEPRESSOR group
So today I did a lot of crying... We really need some good results soon!!
I spend the whole afternoon in the p2 lab with Sebastien playing around with Salmonella:-) Hopefully I wont get a stomach infection in the next few days. Well, Sebastien and I made some competent Salmonella cells using the 413 strain (without any phages integrated) and the 421 strain (WT). I then transformed the following plasmids into both strains:
pIGR01 (pSB4A5+G1+GFP): colony 4
pIGR02 (pSB4A5+G2+GFP): colony 6
pIGR03 (pSB4A5+GR1+GFP): colony 11+13
pIGR04 (pSB4A5+GR2+GFP): colony 16+18
All plasmids are from the 02-10-10
I plated 100 ul on Amp plates. The colonies need to be re-streaked tomorrow, and the day after tomorrow, the restreaked colonies can be plated on plates containing phage inducing compounds (this will induce expression of the antirepressor).
What we hope to see from this experiment is whether the G1/G2 construct will stop expression of GFP when transformed into the 421 strain, as this strain produces the repressor which will act on the pR promoter. We also expect the antirepressor to de-repres the repressor in the GR1/GR2 construct, thereby starting expression of GFP. The 413 strain is used as a control.
05-10-2010 Annemi Depressor group!!!
Did a PCR of the mini-preps from October 2nd and 4th (see labbook for further detail) but this time diluted 100 fold in water.
However this resulted in yet another depressing empty gel-picture:(
Sebastien suggested that we let the terminator group do a PCR on some of our mini-preps and some of their own (which have been giving nice gel-pictures in the past).
We will do the same PCRs to see if none in the repressor group are able to do a PCR.
Did a PCR clean-up of the restrictions Lisa did yesterday of the backbones: 4A5 and 1C3 and the new biobricks GR1, GR2, Z1, Z2 and RFP. All were digested with E and P.
It looked good on the gel, so we are ready to do some ligations tomorrow.
Maybe the depressor group will have a succes in the nearest future. A girl can dream...
05-10-2010 Juliet and Greg
Wiki: layout improvements in project description. Refactoring of text in project desc. Updates for blog, front page and team.
04-10-2010 Patrick and Anastasiya
We have done PCR clean up of the PCR products from 03.10 (pAT01-1, pAT01-2, pAT08-1) and run them on the gel to verify their sizes. Glycerol stock solutions of pAT01-1 and pAT08-1 have been prepared. Then we have done restrictions of pAT08 (E, S) and pSB2K3 (E, P). Ligation products from 03.10 (pAT09, pAT10, pAT11) have been transformed.
Tomorrow we will do the following:
restreak pAT09, pAT10 and pAT11
run digestions of pAT08 and pSB2K3 on the gel
do PCR clean up
3 & 4-10-2010 Repressor group Lisa
we did PCR on minipreps - and there are nothing going on in the PCR. What are we doing wrong?
we will try to make a dilution of template DNA.
We could try a digestion check, and maybe we could just use E+P as check enzymes.
we made O/N cultures of restreaks from 2-10-2010. And the interesting one of these were miniprepped - didnt have a look at gelpicture yet.
Digestions were made of products needed for making the plasmids that were submitting as biobricks, these need to be PCR-cleaned and then they should be ready for ligation.
Plating of O/N culture which should contain plasmids with pBADZ2 +RFP together with L-arabinose. If we get red colonies we know that the construct (pBADZ2 +RFP) is actually present
I had a look at colonies from transformations that should contain pIGR01, pIGR02, pIGR03 and pIGR04 in FLemmings microscope. In a few of the transformations with pIGR01 and 02, there are good fluorescence, but in the rest it is poor ( Sebastien says that this could be background fluorescence from E. Coli??). In colonies containing pIGR03 and 04 (containing promoter and repressor) there is also fluorescence, but this is reasonable since the system is not tight..
Tomorrow Maya will transform Salmonella with pIGR03 and 04 and pSLD20 and 22 (containing antirepressors), and hopefully we can get some highly glowing colonies.
04-10-2010  WIKI References (Malthe)
Hi team I have with support from Juliet and Greg, tried to set up a footnote or reference system on our wiki, igem have not intalled the application needed and thus the references have to be typed in with text. se example in the "Project" page.
04-10-2010 - SDU DTU iGEM conference - Malthe
Update: We have booked the seminar room here in 301, to avoid problems with keys ect.  Regarding food i talked to Lene Krøl (info below). we decided that either we get a "factura" and give them and they pay. when this is not possible, when shopping in a supermarket, we pay and get the money refunded, save the receipt.
03-10-2010 - Thomas - Antiterminator group
Juliet and I were in the lab today the group. We ran a gel of the 30 verification PCRs that were done yesterday, as well as the four restriction digests. The verification PCRs showed that pATN and pAT08 were both successfully ligated and that the pAT11 ligation failed. We're unsure about the pAT01 ligation, there were strong bands for all four minipreps done, but one of the inserts was larger than the other three. The one that was larger was also the one we thought was closest to the expected size. We ran three PCRs, two of pAT01 (to further verify it) and one of pAT08 so it will be ready for restriction tomorrow.
The digestions of pAT02, 04, 06 and 07 all looked ok and were used to run three ligations, plasmids pAT09, 10 and 11.
02-10-2010 Anastasiya & Patrick Anti-terminator group
Re-did the digestions on pAT02, pAT04, pAT06 and pAT07 that Thomas managed to kill last night.. :p They were ran on a gel and seem to be good, PCR clean ups were made on them and they need to be ran on a gel again to determine the dna concentration, so this is one of the tasks to do for tomorrow. Freeze solutions were made of pAT02(L1-2), pAT06(L2-4), pAT07(L4-4) and placed into our strain bank in the -80C freezer.
Mini-preps were made of the over-night cultures containing pAT01, pAT08, pAT11 and pATN(N+1C3), these were then ran on a gel as well as used for PCR verification. If no one else from the building takes out the PCR tubes from the PCR machines, then remove them tomorrow and run them on a gel to see if our ligations were successful.
As there was no more space in our functional parts only box, I started a new one called functional parts only box #2 and the new mini-preps can be found in there.
The digestions mentioned above can be found in the working box, which is pretty much filled up, so a new one should be started of those as well.
If all the verification PCR's turn out to be good tomorrow, I suggest getting started on the next set of pATxx series and pushing forward as much as we can to get our construct done.. awesome work so far!!
02-10-2010 Repressor group Annemi
Made minipreps of the overnight cultures from yesterday. Made with 4mL culture. They are stored in the freezer in a blue thingy. Should be easy to find. Ran a gel of the minipreps.
The T-group grabbed the thermal cycler before me, so I didn't make the PCR on the minipreps. But from the gelpicture it looks like we might have an insert:)
Did restreaks of the transformations which had grown.
01-10-2010 - Thomas - Antiterminator group
Ran a gel of PCR products of pAT02, 04, 06 and 07 (from 30-09) and then a PCR clean up and another gel of that. Both gels looked good, each band was the expected size. The purified PCR products were then digested with X and P.
Overnights were also done of the pAT01, pAT08, pAT11 and N+1C3 restreaks from thursday 30-09.
01-10-2010 Repressor group Maya and Lisa
The restriction products from yesterday (G1, G2 and RFP), and the PCR products RFP and GFP were purified and run on a gel. Everthing looked good, and this was the first time that we could actually see G1 and G2 on the gel after restriction. This was probably because two pcr products were pooled in one clean-up, and because the restriction was done in only 50 ul total volume rather than 100 ul.
We re-did some of the ligations that didn’t work yesterday. These were:
L1: pSB4A5 + G1 + GFP (Did work yesterday, but was repeated anyways)
L2: pSB4A5 + G2 + GFP (Did work yesterday, but was repeated anyways)
L5: pSB2K3 + BADZ1 + RFP
L6: pSB2K3 + BADZ2 + RFP
L8: pSB4A5 + RFP
L9: pSB4A5 no insert
L10: pSB2K3 no insert
They were also transformed and plated on Amp and Kan plates in 20 ul and 200 ul.
We made a PCR of the pSB4A5 and the pSB1C3 backbones. These were purified, but the purified products were not run on a gel as we didn’t have the time.
Finally, O/N cultures of the restreaks from yesterday were done. 23 in total. Three of the restreaks containing pSB4A5 and GFP were red, and as they are not supposed to be that, these were of course not included in the O/N’s. Tomorrow minipreps of the O/N’s  and a PCR can be done, and then we will hopefully know whether we have some constructs that are ready for measurements.
01-10-2010 Dry Lab - Anja
meeting summary
reminder BB submission
Mail to Meagan with regard to DNA submission: Low vs High copy number plasmid
another mail to our supervisors > Exam, I also talked to Mogens and he told me they are discussing it and that we will know more quite soon
writing more for the wiki: characterization > the poster group
spaming google groups with messages ;-)
Septermber
30-09-2010 Dry Lab - Anja
nice that the dry lab people also are allowed to write in the Lab blog now :-)
preparing meeting agenda and keeping track, tomorrow I will write a summary and send out reminders
search for more shipping information and when we should send our biobricks
DNA submission, I got more into the details and its important to note, that the registry needs to be filled out before we are able to send the BB
I  also continued to write for the Wiki and this I will do tomorrow as well as we talked about in the meeting, writing a summary about the repressor system and short about characterization for poster purposes
30-09-2010 Anastasiya Antiterminator group
Thomas and I have done 2 PCR verifications on each of the MiniPreps prepared on 29.09, corresponding to ligation products L1-L5 namely of pATO2, pAT06 and pAT07. We run 40 PCR reactions in total. (For further information see our lab book) Then we run them on gel to verify their sizes and see if they worked. It seems like all of them have worked except from pAT06, but likely we have run it twise and at least one of them worked.
Then we have chosen one of L1-L5 for further PCR and we have also done another PCR of Mg03&IC3.
We have also restreaked colonies from L1, L2, L3 and L7 plates from 29.09.
Tomorrow the following should be done:
PCR products should be verified on the gel followed by the PCR clean up
Restrictions of PCR products
Preparation of the overnight cultures of the restreaked L1-L3 (5 of each)
30-09-2010 Maya Repressor group
Annemi's 40 PCR reactions from yesterday didn't give any result, meaning that we still haven't succeeded in ligating our biobricks into pSB1C3. We will put this on pause for now and concentrate on ligating our biobricks into pSB4A5/pSB2K3 with GFP/RFP.
The ligations made yesterday L1-L10 were somewhat successful. All transformations besides the ones containing RFP gave colonies. Today we restreaked these colonies and tomorrow we can do O/N. We also restricted G1, G2, and RFP so we can redo those ligations that didn't work.
Annemi made a PCR og RFP and GFP. These will be purified and run on a gel tomorrow.
29-09-2010 Greg Repressor group
Got in touch with students using the BioLector (Manos and Jacopo). Made a tentative booking for weekend 9-10.10.10 (Friday 8th also possible). Jacopo will explain how to use the equipment a few days before we run the plates. Recommendation is to measure OD of a culture grown outside of Biolector (but in the same medium) to have an estimate of when best to treat with arabinose (this applies to the antirepressor system). Should treat during exponential phase. Also it's important to prepare the plate under the hood, to avoid contamination.
29-09-2010 Annemi Repressor group
Made ligations of the following:
L1: pSB4A5 + G1 + GFP
L2: pSB4A5 + G2 + GFP
L3: pSB4A5 + GR1 + GFP
L4: pSB4A5 + GR2 + GFP
L5: pSB2K3 + BADZ1 + RFP
L6: pSB2K3 + BADZ2 + RFP
L7: pSB4A5 + GFP
L8: pSB4A5 + RFP
L9: pSB4A5 no insert
L10: pSB2K3 no insert
Made a miniprep of I13507 (RFP) and verified it on a gel (thanks to the boys for helping with that:))
Transformed the ligations in electrocompetent cells DH5α
Plated the transformants on plates (20 and 200 µL)
Ran a PCR of the minipreps done yesterday using IG202 as the reverse primer for all (40!!!) reactions. IGR01 used as forward on miniprep 1-5 + 11-15. IGR04 used as forward on miniprep 6-10 + 16-20. IG201 used as forward on 1-20. Further detail on the programs can be seen in the labbook.
Very busy day in the lab and hopefully we will get some results:)
29-09-2010 Thomas & Patrick Antiterminator group
Started off the day by making ligations of our constructs: pAT01, pAT08 and pAT11. These were then transformed and plated; they will be ready for restreaking tomorrow and innoculation of overnight cultures on the subsequent day.   Mini-preps were made from the over-night cultures containing our constructs pAT02, pAT06 and pAT07; these will be PCR verified tomorrow.
28-09-2010 Patrick Antiterminator group
Digestions were made today of nutR and nutR+terminator both with restriction sites of XbaI and PstI; these will be used tomorrow for ligations of constructs pAT08 and pAT11, respectively.
Mini preps of Mg03, Mg03+1C3, N+1C3, nutR+1C3 as well as puc19(one of Sebastien's high copy number plasmids) were made today, a few made in parallel while Sebastien did them as well, intention of this was to test whether mini prep skills were faulty or whether our plasmids are simply in low concentration.
The restreaked plates containing our constructs of pAT02, pAT06 and pAT07 were used for innoculation of overnight cultures; these will be mini-preped tomorrow.
PCR clean ups were made on the pSB2K3 PCR products as well as the digestions of the above mentioned products after digestion was performed.
28-09-2010 Greg Repressor group
Moved last year's layout to the home page. The links have been updated and should work (let me know if they don't).
28-09-2010 Annemi Repressor group
Today the PCR of G1, G2, GR1, GR2, BADZ1 and BADZ2 was redone and it looks good:)
Restrictions with E+S was made on G1, G2, GR1, GR2, BADZ1 and BADZ2 (old PCR products).
6 mL of the transformation overnight culture (still from last week) was spun down. This will hopefully result in a mini-prep with a higher concentration than the ones Maya did... Miniprep was made and they are stored in the working box: no label. Only number (corresponding to the original colony) and date.
Lisa did cleanups of the PCR products and the restrictions.
Maya made restrictions of pSB4A5, pSB2K3, RFP, GFP as well as cleanups of these.
Tomorrow we should be ready to some ligations.
27-09-2010 Maya Repressor group
The PCRs I made the 24-09-2010 were ran on a gel i the weekend and no bands appeared:-( Maybe something is really wrong, or maybe I'm just really bad at making PCRs. Sebastien had a look at the gel picture with the purified plasmids that I used as templates and thinks that they look weird. He thinks the concentration is way lower than normally, and he also thinks they look like cut plasmids. So I'm not sure whether we should attempt a new miniprep or whether we should make a new PCR on the miniprep I did, or start all over... We did after all have a lot of background in this ligation, so maybe its a good idea to start over. Sebstien also mentioned that it could be a problem to try to clone the gifsy repressors into pSB1C3 which is a high copy number plamid, and as far as he understood it, the IGEM headquarter allows submission in other plasmids if you have an explanation for not using pSB1C3. So I guess we could just submit in pSB4A5. Anyways, we can talk about this tomorrow as I'll come in at 8 am.
Today Anja did a PCR of G1, G2, GR1, GR2, Z1 and Z2, but none of them worked, so they have to be redone tomorrow.
27-09-2010 - Anastasiya & Patrick - Anti-terminator group
Restreaks were made of ligation plates of L1 to L5 that can be found in the lab book dated 23rd September 2010. Over night culture media (already containing antibiotics) are ready and stored in the fridge, these will be used for the colonies obtained from the restreaks mentioned above.
The verification PCR's from friday (24/09/2010) were ran on a gel, all except for one band had the right sizes; the band corresponding to the N anti-terminator didn't fit in size so another PCR verification was performed. Another PCR of GFP+RBS was made today as well.
A PCR clean up of RBS-N(X+P) was ran on a gel in order to perform calculations required for ligations.
Along with this gel, PCR amplified plasmid backbones of pSB4A5, pSB1C3 and pSB1A3 were ran to verify that they were still present after they had a little swim on friday.
25-09-2010 Repressor group Maya
Yesterday I was in the lab alone, but fortunately Patrick was kind enough to help me out a bit. I did minipreps of the 20 O/N (5 colonies from each transformation) that Annemi and I did yesterday. These minipreps hopefully contain our PCR products (G1, G2, GR1 and GR2) in the pSB1C3 backbone. After the miniprep I did 40 PCR reactions on the plamids. All plasmids were tested with two sets of primers - the VF1 and VR primers and also with our IGR01/IGR04 & VR primers. Today I'll go the lab and put the PCR products in the fridge, and if I have time I'll run them on a gel as well.
The forward primers that I orderd for the antirepressors also arrived today, so next week we can do a PCR of the antirepressors and ligate the products into pSB1C3.
Unfornately someone forgot to put Annemi's restricted products from the 23-09-2010 in the fridge after having run them on a gel. So these are most likely not working anymore, and have to be redone before we can do the ligations of G1, G2, GR1, GR2 + GFP. Sebastien told us that if restricted products are not kept on ice, the sticky ends possible get degraded.
24-09-2010 - Thomas - Antiterminator group
The transformations of our ligations from yesterday look really good! :). There are only about 8 colonies on the ligation control, and 20+ on the plates with the actual ligations. We restreaked 4 colonies of each of the 5 ligations and put them on the lab bench to let them grow over the weekend. On Monday we should start some overnight cultures of all 20 restreaks so we can miniprep and verify them on Tuesday.
Patrick and I also did PCRs of the pSB2K3 backbone plasmid and the RBS+GFP. In addition to that we did 9 verification PCRs on the minipreps of our ligations from last week, so hopefully we'll get some good bands there! The PCRs will finish just before I leave the lab (wow, it's already 17:30...) so I will put them in the fridge so they can be run on a gel on Monday.
23-09-2010 - Thomas - Antiterminator group
Unfortunately, I was in the lab alone today, due to half my group being on vacation. Since I had to prepare the presentation for the meeting with our supervisors and the 2009 team in the afternoon, not alot got done in the lab. I transformed the ligations that Patrick did yesterday, with Anja's help, and plated them. Anja ran a gel of the PCRs of the plasmids backbones that Patrick did yesterday and they all looked good. Anja also did a miniprep of the MG03 plasmid :).
23-09-2010 Maya Repressor group
Today we got the proof that the plasmids pSLD20 and pSLD22 have been mixed up during O/N or miniprep. This means that the plates we have in the fridge are correct, but that the plasmids are swapped in the glycerol stock. We haven't corrected the naming, but will do that tomorrow.
Annemi and I spend most of the afternoon talking to Sebastien about the problems we have had with the colony PCR and also with the background we got when we did the transformations. As the colony PCR Annemi did yesterday didn't work, Sebastien advised us to do the PCR on the purified plasmids. Therefore we started O/N, and tomorrow I'll do miniprep and a PCR.
Regarding the background we got when we transformed the ligated plasmids, Sebastien came up with several strategies to solve the problem. We talked about them at the meeting today, and they will also be written here in the log tomorrow or the day after.
22-09-2010 Annemi Repressor group
Colony PCR on transformations G1, G2, GR1, GR2. Same approach as Lisa did yesterday, however we marked the colonies this time:)
We used 3 different forward primers: IGR01 (for G1 and GR1), IGR04 (for G2 and GR2) IG201 (for all of the colonies). The same reverse primer IG202 was used in all of the reactions.
Result: only two bands appeared; one which corresponds to the size of amplified G1 and the other one to G2.
Questions: Why didn't the iGEM primers IG201 + IG202) amplify anything? Digested plasmid? Why did Lisa get so much more bands yesterday?
Restrictions:
GFP+rbs: cut with X+P
pSB4A5: cut with E+P
G1: cut with E+S
G2: cut with E+S
GR1: cut with E+S
GR2: cut with E+S
Ran a gel of restrictions. Didn't have enough time to study the picture in detail. However the bands do not look too concentrated.
Did a PCR clean-up -> working box tubes with white stickers provided with: date, DNA, RE-info, 'cleanup'
Next move: Run gel of the cleaned up restrictions.
22-09-2010 Maya Repressor group
We still haven't figured out what happend to the plasmids pSLD20 and pSLD22, so we restreaked both the ones from the glycerol stock and the ones in the fridge.
To Anja:
I  think it would be most efficient if you spend the morning doing dry lab stuff. Could you maybe prepare the meeting agenda? There are a few points on the sheet of paper on the wall in the student room. We will start the meeting with a presentation for the supervisors (Thomas is making that one), and after that (probably at 5 pm when everyone is present) we will have our normal weekly meeting. So if you could make the agenda for the normal meeting.
When that is done, you can just continue on the paragraph you are working on for the wiki.
Annemi and I will come at 1 pm, and hopefully we will do some ligations.
Just send me a message if you have any doubts:-)
Have fun!!
22-09-2010 - Patrick - Terminator group
Another long busy day with many items to take care of.
The digested items from yesterday were purified via pcr clean up, and ran on gel to determine concentration for ligations.
After this, all items needed for ligation constructions of pAT02, pAT06 and pAT07 were ready.
The following ligations were made:
L1: pAT02 - RLN + RBS-RFP + pSB1C3
L2: pAT06 - nutR + B0011 (X+P) + pSB1C3
L3: pAT06 - nutR + B0011 (E+X) ( <-- pSB1A3)
L4: pAT07 - nutR + B1003 (X+P) + pSB1C3
L5: pAT07 - nutR + B1003 (E+X) ( <-- pSB1A3)
L6: Negative ligation control - pSB1C3
L7: Negative ligation control - B0011 (E+X)
L8: Negative ligation control - B1003 (E+X)
Tomorrow (23/09/10) these need to be transformed and plated.
Plasmid purifications were made from the re-streaked ligation plates, the following mini-preps were made:
Mg03 + 1C3
nutR + 1C3
N + 1C3
The gel showed some irregularities as the sizes didn't exactly match, this needs to have a closer look.
A digestion was also made on RBS-N with X+P, this needs to be purified via pcr clean up tomorrow (23/09/10).
Once purified it can be used along with pBAD (E+S digested) and pSB4A5 (E+P digested) to construct pAT01 which should look like the following:
pBAD (E+S)
RBS-N (X+P)
pSB4A5 (E+P)
--> 3A assembly to create pAT01.
PCR duplicates were made today and will be ready tomorrow (23/09/10), the following plasmid backbones were amplified:
pSB4A5
pSB4A5
pSB1C3
pSB1C3
pSB1A3
pSB1A3
(numbering system here follows numbers written on pcr tubes)
- These need to be verified tomorrow (23/09/10) on gel to make sure they were in fact amplified, followed by pcr clean up.
2 extra pcr duplicates should be made of the following:
GFP+RBS
plasmid backbone pSB2K3
Another over night culture was made of Mg03 as a mistake was made yesterday and the wrong anti-biotic was inserted.
This needs to be plasmid purified tomorrow as well.
Find to do list in lab book, same content will be present there as here.
If you have any questions about this, gimme a call or find me in lab 201 tomorrow.
Over and out.
21-09-2010-Lisa - Repressor group
A new PCR with pBAD, BADZ1 and BADZ2 turned out to be succesful, as Annemi wrote (using miniprep 1 as template) - although it seems that there is confusion about which plasmid is pSLD20 and which one is pSLD22. Sebastien says to trust the PCR products! and he might have an explanation for the mix up. Talk to him tomorrow.
You should also check the re-streaking of the strains with the 2 plasmids on CAM-plates - these restreakings were done from the glycerol stock. pSLD22 should have CAM resistance and pSLD 20 KAN resistance. THe re-streaking that Maya did yesterday  came from plates and showed as expected that pSLD 20 has KAN resistance
I did another PCR with pBAD, BADZ1 and BADZ where I used miniprep 2, to check if these templates also seem to be switched. The PCR products needs to be run on a gel - they are probably still in the PCR machine.
I did a colony PCR on 3 colonies from each of the transformations. Check the picture in the lab-book. I didnt have time. The first lane is ladder, the next 5 is Patricks, and the next 15 are ours. I think I forgot to write that the last 3 lanes contain a PCR of the "no insert".
We should change the colony PCR strategy to Annemi's boiling strategy, as this turned out to be quite succesful.
Finally I did a PCR clean-up of the PCR products from the PCR with miniprep 1 as template - products still need to be run on gel, and maybe they are ready for digestion :)
21-09-2010 - Patrick - Terminator group
Today the lab work was ran almost parallel with good success with the repressor group, Lisa and I had a rather successful day. To start off with, the PCR's in the morning.. WORKED!! Those were both from the repressor group and terminator group.
I continued to work on the digestions needed for our construct and did the following ones:
GFP(rbs): E+S
pSB1C3: E+P
pBAD: E+S
pSB4A5: E+P
pSB1A3: E+P
Gels were ran firstly on the PCR's in the morning and turned out successful (both repressor and terminator groups have the gel pictures in their lab books).
The second gel made contained the above mentioned digestions as well as Lisa's colony PCR's. From the gel its hard to evaluate the outcome of the digestions as they were made on pcr products, but non the less bands were visible for all 5 of the digestions run with matching bp sizes. The other 15 bands were repressor groups colony pcr's (both repressor and terminator groups have the gel pictures in their lab books).
Over night cultures were made of the re-streaked ligation plates, tomorrow (22/09/2010) plasmid purifications need to be performed.
Tomorrow ligations on pAT02, pAT06, and pAT07 need to be done.
21-09-2010 - Annemi Repressor group
A new attempt to amplify pBAD, pBADz1 and pBADz2 was made. The PCR was made exactly as Maya made it yesterday (see lab-book for further details).
Patrick and I made some changes in the restriction protocol which can be found in the protocol folder in dropbox.
20-09-2010 - Patrick and Anastasiya - Terminator group
So today we've been doing a bit of work for both the terminator and repressor group.   Firstly we ran 2 pcr's on plasmid backbones (pSB1C3 and pSB2K3) which turned out successful on the gel, they were then purified via pcr clean up. (Please take note here that the old pcr buffer was used, so I suggest from now on when running a pcr, use any buffer you think will work, but use the old buffer as a form of control with an extra pcr product, so that in case it doesn't work but the control does, the problem might be the buffer and not your pcr.)
Digestions that were made on Thursday (16/09/2010) were ran on a gel, most of them weren't very visible, but successful non the less; they were purified as well via pcr clean up.
The ligation plates plated on Wednesday (15/09/2010) were restreaked and will be ready tomorrow for inoculation of over night cultures.
We also helped out Maya, and did pcr clean ups on the pcr products of the plasmid backbones, they all seemed to have worked well (pSB1C3 wasn't as strong as the others though) the only pcr product that appeared to have not worked was that of GFP, details on this can be found in the lab book, but it should have something to do with the pcr buffer again as there were 2 pcr products of GFP, one worked and the other didn't.
20-09-2010 - Maya -Repressor group
The PCR Lisa and I did Friday only worked for the pBAD promoter, not for the pBADAntO or pBADAntT. So today I redid the PCR using each primer set on both pSLD20 and 22 just in case Sebastien messed up the labeling of the plasmids. I also restreaked the pSLD 20 and 22 on plates with CAM and KAN to verify whether Sebastien did indeed mess up labeling. Patrick will run the products on a gel, and place the photo on our desk today. If the PCR is still not working we should talk to sebastien about alternative ways of amplifying the products.
I also transformed the ligations Lisa and I made Friday, and hopefully you should see colonies tomorrow. If there are colonies you should restreak them on new plates tomorrow, so that we get more clean colonies.
The terminator group is still working on amplifying the GFP as the GFP without RBS didn't work. Apparently the plasmid is gone, or they can't find it, so they are redoing one of the PCRs using the PCR product with the RBS as a template.
I made a new box in the freezer for our group, so that we now have a box with verified PCR products and minipreps, and one box with stuff that are not necessarily going to be saved (working box).
One thing you could also do tomorrow is to have a look at that excel sheet called "Parts and plasmid list-repressor group". We need to past in plasmids that we have purified and constructed.
17 -09-2010 - Lisa and Maya - repressor group
Gel of the purification of G1, G2, GR1 and GR2 looked good, we did ligations of these parts with the plasmid pSB1C3 - these ligations should be transformed monday (20-09-2010).
Plasmid purification of overnight cultures with plasmids pSLD20 and pSLD22 also looked good, so we did a PCR to obtain the pieces pBAD, BADZ1 and BADZ2. These PCR products should be checked on gel monday - finally we made glycerol stocks of the overnight cultures (pSLD20 and pSLD22).
16-09-2010 - Maya and Annemi - repressor group
We did a restriction on our PCR products (G1, G2, GR1 and GR2), and of the plasmid pSB1C3. The plasmid is going to be used for submission of the biobricks. The restrictions were purified using the PCR purification kit. The purified products still need to be run on a gel. If the gel looks good tomorrow, a ligation should be done.
The missing primer arrived today meaning that tomorrow a PCR of GFP can be done. Both groups need the GFP.
16-09-2010 - Anja - repressor group
I did the clean up of our PCR products, they are labeled as those. Afterwards I run them on a gel to control the presence of PCR products after the clean up procedure.
I started an O/N of pSLD 20 and pSLD 22 with 100 Amp and incubated them at 37 degrees.
Never take ligase and restriction enzymes from that other lab into our lab - people are panicking if their beloved enzymes aren´t there and want to order immediately new enzymes! ;-)
All tubes for storage have to be labeled with stickers, but don´t use the blue ones. Write clearly: sample, date and what has been done. Otherwise you get in trouble with Annemi ;-)
Use lab book for writing messages to each other.
Write down in lab book, what next days people should continue within the first hours (you don´t have to plan the whole day for them, but at least that they can start easily (and early ;-)).
15/09/2010 - Terminator group - Thomas & Patrick
Today we did digestions of N, NutR, MG03, pSB1C3. After PCR-cleanup the N and NutR digestions seemed to have worked but you can't see a difference from digested/non-digested as they are from pcr products. The MG03 digestion looks bad, but the dna concentration might have been too low. The pSB1C3 also had a low dna concentration. Tomorrow we will still attempt ligations and redo digestion of both MG03 and pSB1C3. We should also pcr more linear pSB1C3 plasmid backbone.
14-09-2010 Anja (Repressor group)
Lab stuff:
Annemi minipreped the overnight BB transformation cultures from yesterday. She minipreped only half of the O/N, because Malthe and Annemi think we have enough plasmid to work with (Correct?).
those minipreps labeled with white stickers are stored in the -20 degree freezer in a box with date 14-09-2010
A back-up preparation of the transformants has also been made. 2x2mL of cells have been centrifuged and the cell residue is kept in the -20 degree freezer (white stickers: date, biobrick/plasmid #, 'cells') This will make it easier to prepare a new mini-prep since we don't have to grow overnight cultures first.
a gel of minipreps has been run and I took a picture. Seems like the mini-preps went well since a band is visible in all of the wells.
Glycerol stocks of transformant cells have been done and stored in the -80 degree freezer
OBS they reorganized the freezer > look into lab book "freezer box location"
Malthe has resuspended our primers, which arrived today :-)
One primer is still missing, Malthe wrote an email to ask for primer IGT02
Ethanol supplies:
There are different ways to get more supplies:
Look for a big 2 L box in the fume hood in our lab next to the sterile bank. It´s refilled by Bjarne.
Go downstairs to room 162 on the 2th floor with a min. 1L bottle. The room is on the right side, go to opposite room, get keys and book from cupboard right to the door. Write in the book, how much you have taken.
Organisation:
to repressor group:
I finished the A3sized paper Maya started yesterday, so Maya maybe you can have a second look at it, feel free to add or change, of course :-)
Plasmids:
to repressor group
With regard to our plasmid pSB4A5, pSB2K3 and others:
I suggest that we make our own linearized plasmid, so we don´t have to worry whether the insert is cut out or not. This is done by using primers IG206 and IG207. These primers have been ordered by Thomas, so there is no time delay, when using this approach.
For further information see parts registry help > protocols > linearized backbones.
Please think about how we can purify our biobrick I_13507 from gel, to get rid of the plasmid, we don´t need. - we just do an ordinary BB transformation
Wiki:
Lisa started on reading articles and hasn´t decided which topic she wants to write about. I signed up for repressors in Juliets wiki sheet.
15. sep. 2010 Annemi
Today we have run PCR´s to amplify the promoter region and the repressor region. We used 2µL of Sebastiens Salmonella DNA number 143 as template. (See the lab book for further detail on the procedure)
The program used to amplify the small fragments (ca. 100 bp) are called IGEM TD 100BP
For the larger fragments we used Sebastiens program: SebFC
The products are run on a gel (3-2-1)
We made transformations with the plasmids pSLD20 and pSLD22 in the DH5α. The cells are plated out on AMP-plates (20µL and 200µL)
13-09-2010 Maya (repressor group)
Today me and Anastasiya did O/N cultures of the biobricks that we transformed last week. Tomorrow we need to do a miniprep on all of these plasmids, and also prepare some glycerol stocks. We made double of the O/N so there should be enough if you want each group to have their own sets of purify plasmids.
If the primers arrive tomorrow you should also resuspend them, organize them in a "repressor group" box and make dilutions.
I've started on another A3-sheet describing the construction of the three different setups in parallel. Maybe you can continue on that or improve it. The purpose of the sheet is to make it easier for us to see which things should be run in parallel. In that way we can do all the PCRs at the same time and so on.
If you have time Lisa, you can continue on the CLC files. I'll try to finish mine today.
I also think we should find out who is going to write what on the wiki, as I think our group should write about the gifsy1 and gifsy2 promoters, the repressors and the antirepressors.
080910 Maya and Annemi (Repressor group)
We now have a strategy on how to do the labwork in the repressor group. We will need to construct six different plasmids:
pIGR01 (GF1) /pIGR02 (GF2): A pSB4A5 backbone plasmid with the two promoters pRM and pR. Downstream of the pR-promoter we will have a GFP.
pIGR03 (GF1) / pIGR04 (GF2): A pSB4A5 backbone plasmid with the two promoters pRM and pR. Downstream of the pR-promoter is a GFP. Downstream of the pRM-promoter  is the gogR and gtgR respectively.
pIGR05 (GF1) / pIGR06 (GF2): A pSB2K3 backbone plasmid with a pBAD/araC promoter. Downstream of the promoter region the anti-repressor antO/antT (GF1 and GF2 respectively) is located. As a reporter the BioBrick RFP (E1010) which is located downstream of the anti-repressor.
The plasmids will be transformed into competent cells and fluorescence will be measured with the BioLector. All of the plasmids will be transformed separately along with the following combinations:
pIGR01/pIGR02 Expected outcome: GFP will be detected
pIGR03/pIGR04 Expected outcome: GFP will not be detected
pIGR05/pIGR06 Expected outcome: RFP will be detected
pIGR05 + pIGR03 Expected outcome: Both GFP and RFP will be detected
pIGR06 + pIGR04 Expected outcome: Both GFP and RFP will be detected
pIGR05 + pIGR01 Expected outcome: Both GFP and RFP will be detected
pIGR06 + pIGR02 Expected outcome: Both GFP and RFP will be detected
As a control we will use a pSB4A5 backbone plasmid only containing GFP.
Overlapping primers which we will use to amplify the two promoters pR and pRM as a BioBrick have been designed.
The primers to amplify the gogR/gtgR and the promoters from the Salmonella chromosome as a BioBrick have been designed.
Primers to amplify pBAD/araC and antT/antO from the pSLD20/22 plasmid as a BioBrick have been designed.
Primers to amplify GFP still need to be designed. How about the RBS and terminator?
August
31-08-2010
IGEM UPDATE: 31-08-2010 (copied to log)  The Result of the Day, and what we talked about.  BB REGISTRATION - REPRESSOR & ANTI-TERMINATOR
Each research group register the biobricks needed and in our excel sheet note when the file have been added to the registry, becouse then no further editing should be done in the excel file.
- see "our biobricks list and sandbox.doc" in Research group.
The needed description is written in the excel sheet    - see "our biobricks list and sandbox.doc" in Research group.
- Enter a long description of the part so that users of your part know what it is, what it does, and how to use it in their projects.
- Enter the source of this part. For example, does it come from some genomic sequence?
- Enter any design considerations you had to deal with during the detailed design of the sequence
- After which we have to enter the sequence and add feature annotations.
FRIDAY MEETING:
AGENDA - So far
Group picture
Wiki presentation, frame set up by Juliet. presentation of what to be filled out.
tech-track
HOMEWORK:
management deadline 20-sept.
Go to our team page, and look at our team roster make sure your information is typed in correct. this will be on your certificat for attendign iGEM.
Come up with 3 good applications for our stable "real" bistable switch system, as we need to select track on friday.. (or think about it) we collect them friday, and put them on the wiki.
AREAS:
Final Tracks
New Application
Food/Energy
Foundational Advance
Health/Medicine
Environment
Manufacturing
Information Processing
Software Tools
tech-track
MANAGEMENT:
Maya and Malthe have made an improved Gant Diagram (again) take a look at it and feel free to add to it.
31_08_V001_MMB_MFK_OVERALL TIME PLAN.xls  in the management folder.
24-08-2010
So the competent cells we made last week have been infected by phages!!! Oh no, this is really bad cuz it's difficult to get rid of phage contamination. Sebastien thinks that it might be a T1 phage. So we are preparing to make a new batch of competent cells. This will be done Friday. All the pipettes have been cleaned thoroughly to get rid of potential contamination.
Still working on the biobrick design. We are reading a lot about biobrick standard measurements, and we have found a lot of interesting papers. The design isn't as easy as initially thought if you want to get some data corresponding to the iGEM standard.
Maya
20-08-2010 (Patrick , Malthe , Anja, Juliet and Maya)
To make it easier for everyone to follow the project I'll try to give an update on what has been going on this week.
First of all we have had two meetings, one group meeting and one meeting with the supervisors. Summaries of the meetings can be found in dropbox in the management folder.
We have spent a lot of time on getting an overview of the project, and we have tried to make a phase diagram of what should be done for next two months.
Juliet got the Wiki up and running and is trying to work out how we can do the modeling with fictional data. She is also having a meeting today with Steen to get an overview of the economy. Remember to let her know whenever you spend some money from our account!!
The characterization of biobricks have a really high priority at the moment. So mainly Malthe and me (Maya) are trying to come up with several stragtegies on how to do this and which biobricks we should focus on. We hope that by next week we have a long list with some thought out projects. The idea is to distribute the projects among the lab group so that each person/group of two gets his/her/their own project. The responsible persons should then finish the design and start on the lab work. Some of the mini projects are most likely not going to work, so that's why it's a good idea to have several.
We have also come up with a new naming system for all our parts as this is really needed when you work with the parts in the lab. This list can be seen in the research group folder-> "our biobrick list" All the parts found in the list should be registered in our "sandbox" in the partsregistry. When we start working on our biobricks, all information should be typed directly into the sandbox. When characterizing a BB Biofab, drew Andy's company have made a data sheet see our wiki
https://2010.igem.org/Team:DTU-Denmark/Project#characterizing_our_Biobricks
Also we are out of competent cells. So today Sebastien will show us how to make our own. A new protocol has been added to the folder.
Social event: Free Funky jazz concert saturday Hezz brothers. http://kglteater.dk/Alle_forestillinger/DKTplus_10_11/Hess_is_more.aspx
Looking forward to everyone is back:-) It really seems like we are on the right track now.
Whats going on in the lab (12/08/10) (Annemi, Patrick)
So if you have done your homework and read yesterdays log ;), you would have a better understanding and update of whats going on with the project. Heres whats actually going on in the lab right now:
From the power-point presentation "Bistable switch design - Biobrick approach" , if you have a look at slides 6 and 9 this will illustrate what we are trying to put together.
We are digesting and ligating by the method of biobrick assembly standard as was mentioned yesterday.
We are digesting and ligating by the method of biobrick assembly standard as was mentioned yesterday.
Initially we ran into problems with the restriction enzymes (FastDigest enzymes from Fermentas), after a third attempt and playing around with the digestion mix, we got them to work.
We are hoping to be successful in our ligations, which we will be transforming today (fingers crossed for tomorrows outcome ;)), the plasmid backbone used for the ligation is pSB1T3, a high-copy number plasmid that has the Tetracycline antibiotic resistance.
Plates containing tetracycline need to be used within 2 weeks after being made as the tetracycline antibiotic is unstable. If you have to make some of these plates as we have to, ask Lisa for the protocol for making plates. You can also ask Bjarne, the lab technician in our lab for assistance. After you have gone through the laborious procedure of making your LB solution, you need to add the antibiotic (the solution can be max 60C otherwise the antibiotic will die).
Today the tetracycline plates made will have a concentration of 10 gamma.
Lab work - follow up (11/08/10) (Lisa, Annemi, Thomas, Anja, Anastasiya & Patrick)
As the log was supposed to be updated according to the successes based upon construction of a template plasmids containing the FP's, the reason why this never occured is because it never worked and a huge change in the project occured.
After having performed the same procedures as was mentioned in the previous log entry, the colony PCR results indicated a zero success rate at constructing the FP template plasmids.
The new biobrick approach (Annemi, Patrick, Thomas, Lisa) :
A new biobrick approach was engineered to put our whole system together. We scratched the idea of using red swap to insert our half switches into the chromosome as these were not biobrick friendly..
Instead we came up with the idea of having "mini-genes" synthesized and thereafter assembled together in the manner of the biobrick assembly standard. This also means that we can in the end submit all of our pieces as biobricks.
All the mini-genes and corresponding primers have already been ordered and we're still awaiting the arival of the mini-genes.
Dr. Thomas has created a powerpoint presentation illustrating how these mini-genes are to be assembled and how everything will look like in the end.
The powerpoint presentation, called "Bistable switch design - Biobrick approach", can be found in the "Step-wise construction and in-silico model" folder in dropbox.
As you all should know, the biobrick assembly standard uses 4 restriction sites, 2 in the prefix: EcoRI (E) & XbaI (X) and 2 in the suffix: SpeI (S) & PstI (P).
Whether having to assemble biobricks taken directly from the dna distribution kit or the mini-genes, the concept would be the same and you need to follow the assembly standard.
If you need to refresh your remember about this procedure, have a look at the file "BioBrick_Assembly_Manual" which can be found in the "Research Group" folder in dropbox.
As we are under quite a bit of pressure to construct our system, the positive side of this approach is that it should be rather easy to assemble everything as long as a lot of thought is put into the digestions and ligations. ;)
In terms of actually activating the system, to start with we won't be using the light-receptor mechanisms, instead we will initially test / activate the system based upon carbohydrate sources, namely rhamnose and arabinose.
July
Lab work (Patrick, Anastasiya and Anja) 06/07/10
After last weeks succesful ligation of CAM and KAN resistance markers into plasmid pSLD3, we successfully constructed plasmids pSLD30 (which contains CAM) and pSLD31 (which contains KAN). All the steps and how we calculated the volumes needed of all the items for both restriction and ligation can be found in the lab book. Long story short, in order to have ample amounts of inserts, (FP's and resistance markers), we performed PCR's.. both de-novo amplifications of the inserts that weren't present as PCR products from Sebastien already, and re-amplications of the PCR products available from Sebastien. Once PCR was completed, we had ample amounts of all the FP's (yGFP, CFP, and CRFP) as well as resistance markers (CAM & KAN).
Next step was to perform a PCR clean up; in order to construct pSLD30 and pSLD31, for now we were only working with the resistance markers.. FP's will come in the picture at a later stage. So, a PCR clean up was performed (follow protocol) on the PCR products of CAM and KAN. These are now ready to use for the next step, which were restrictions.
The primers used when performing PCR on CAM and KAN had tails which contained restriction sites, which brings us to the point using restriction enzymes. The table with all the combinations of restriction enzymes used for each digestion can be found in the lab book, the point was to perform restrictions such that in the next step we could ligate plasmid with insert in the combination of:
pSLD3 + CAM = pSLD30
pSLD3 + KAN = pSLD31
Before proceeding to the next step of ligation, the restricted products (both resistance markers and plasmid) were checked on a gel to make sure restriction occured and that we actually still have material to work with.
The next step was the ligation. Ligation (follow lab book protocol) pretty much doesn't need an explanation as it ligated plasmid with insert.
The ligations were also checked on gel to make sure ligation actually occured.
This was followed by transformation, were we used the method of electroporation to insert our newly constructed plasmids (pSLD30 + 31) into electrocompetent cells, incubate them at 37 degrees in recovery media for 2 hours and then plate them on the corresponding needed plates.
These plates (9 in total) were left over night in the incubator (37C) to grow and have a party. In the lab book you can get an overview of what each of the these 9 plates contained, there were of course several controls as you might have guessed. The plates of most importance were the ones containing the transformations containing plasmids pSLD30 and pSLD31, thus from these plates one colony was used for a restreak on a new plate to have as a "stock" plate, as well as making over-night cultures.
The over-night cultures, obviously having grown over-night were used to perform plasmid purification (follow protocol) in order to obtain pSLD30 and pSLD31, -80 freezing cultures of the strains were also made and registered in the "strain bank" excel file found in dropbox.
So, from A to B, the steps were the following:
PCR
PCR clean-up
Restrictions
Gel-control of restrictions
Ligation
Gel-control of ligation
Transformation
Plating
Over-night cultures
Plasmid purification & -80 freezing cultures
= pSLD30 & pSLD31 plasmids constructed!
Have a look at the lab book in order to get a full detailed explanation of all of these steps and all the small details not mentioned here.
Next step are the insertions of all the FP's (yGFP, CFP & CRFP) into pSLD30 and pSLD31, individually of course.. will make that tomorrow's log.
Over and out (Patrick).
June
Lab work (Thomas, Anja, Patrick and Maya) 25/06/10
Today Thomas joined the lab:-)
The 10-1 and 10-2 dilution from the 24/06/10 yielded several hundred colonies which means that the transformation was indeed alright. As it's Friday today, we will wait with the O/N culture till Sunday.
The O/N of strain SP58 showed growth this time. The plasmid was purified and stored together with the FP plasmids. A -80 freezing culture of this strain was also made.To get an overview of the strains and plasmids we have used so far, have a look in the strain and plasmid bank in dropbox-research group-strain and plasmid bank.
You can't spell funding without fun... (Annemi) - 25/06/10
I had a look at some funding and where we should go from here.
We should definitely apply for the COWI fund. This will take some time since they have a lot of requirements for applying. (see http://www.cowifonden.dk/Vejledning_for_ansoegere.asp)
The ‘Fondeliste’ has been updated according to the funds that Anastasiya and Anja found in the beginning of June.
A document called ‘Company applications’ can be found in the ‘company’ folder. Please write in that document if you send out a company application.
An application to the fund ALECTIA has been send.
Birthday and fun day at the lab? (Maya, Anja, and Patrick) - 24/06/10
So today was yet another day in the lab, only difference being it was my birthday (Patrick). :D
The serial dillutions we plated of our transformants didnt seem to produce a substantial amount of colonies after having plated them yesterday, so we plated a 10-1 and a 10-2 dilution to get a higher number of colonies.
We purified plasmids from strain SP44, SP45 and SP46 using the miniprep kit, and ran the purified plasmids on a gel to check if our plasmid purification was efficient and successful. There is now a miniprep protocol in the protocol folder.
We also made some -80 freezing cultures of strain SP44-46. To keep track of the future freezing cultures we have made a document called strain bank which we should all use when we make some -80 cultures. It's in the research group-> strain and plasmid folder.
The O/N of SP58 didn't show any growth, so a new O/N was started.
Progress (Maya, Anja, and Patrick) - 23/06/10
So we started in the lab yesterday after having gotten all the needed information from Flemming concerning the FP's (fluorescence proteins).
We started constructing some template plasmids with FP's and markers, and so far its been a fun experience thanks to Hassan.
As things are now we have decided to use the fluorescent proteins mCherry and YGFP as the reporters, though the final template plasmids will include the fast-degradable fluorescent proteins of mCherry and YGFP.
Today we started O/N cultures of the following strains: SP44, SP45, SP46 and SP58. SP44-46 contains plasmids with CFP, yGFP and Cherry, and SP58 contains a plasmid with the markers CAM and KAN.
We also made a transformation using electroporation. We transformed plasmid pSLD3 into E.coli strain . pSLD3 is to be used as a recipient plasmid of markers and FPs. After the transformation, we made a serial dilution of the transformed strain and plated these. We have uploaded a transformation protocol in the research group folder- protocol folder.
iGEM the last 2 weeks - 21/06/10
So the last 2 weeks have been quite hectic and busy, and if you're wondering why there hasn't been an update its because alot of things keep coming up while working on the design.
The in-silico design is moving along well, though when every new step comes along we run into issues, just like today we realized that the UV system just might not work after all, long story short - because the excitation wavelength will interfere with the excitation wavelength of the red-light receptor. Solution: good old lov-tap will most likely come into the picture now.
In terms of the reporter genes, we were lucky enough to find Flemming back at CSM today, and so Maya and Juliet managed to get alot of good info from him about flourescence proteins which made us wiser in that area, although we also realized they arnt as straight forward to work with as initially thought, so we need to put in more work in that area as well.
We are set to get into the lab tomorrow to possibly start construction of the template plasmids with the flourescence proteins, that is of course if we have the FP's we want to work with and whether those have the appropriate degradation time.
That's about all for now, stay tuned for more exciting news in the days to come.
Maya, Anja, Malthe and Patrick's log - 08/06/10
We have finished the step-wise construction of our system, and have begun on the in-silico model.
We have uploaded the initial work from the in-silico model in a newly created folder in the research group called step-wise construction and in-silico model.
Maya and Patrick's log - 07/06/10
So today we have been working on the step-wise construction of our system, taking into account how to test each part along the way. We learned a lot about recombineering and the factors entailed in the process and have come up with the initial version of how we are going to construct the system.
As a rule of thumb we also learnt the following combinations and bad-combinations when working with FP's:
RFP can be used together with either GFP or CFP; and YFP together with CFP.
The bad combinations which we should never ever do are the following: RFP with YFP; and YFP with GFP.
We have also created a new folder in the research group in dropbox, called "Synthetic biology" where we have uploaded 2 articles about the first constructed synthetic cell, which we think would be a good idea if everyone had a look at. :)
We have also uploaded Sebastien's presentations from last week, and they can be found in the folder "Presentations from Sebastien" in the research group folder as well.
So, from all the work today we are trying to make a scheme of phases / phase planning of the construction parts that need to be completed in a described order that will be uploaded possibly within this week.
Captain's log 040610
Maya, Thomas, Patrick, Lisa and Annemi have been working on a project description for Sebastien. The document 'project description' can be found in dropbox in the research group folder. From now on this document will be the only one describing the project, therefore you should edit in the document and not create a new one. We have decided that every time we are doing something iGEM related, that it should be posted in this document. Please write the date and your name at the beginning of your log. This will make it easier for all of us to keep track of what is going on in the group. We have tried to make a simple plan of what needs to be done before going to the lab:
Investigate the E. coli chromosome to find out where we can integrate the switch
Find the BioBricks we will be using (DNA sequences)
Install the software CLC main workbench (the program which will be useful for the design of the system)
Design the system: Order of genes, restriction sites, DNA sequences, order of integration of genes into the plasmids
Which plasmids (Sebastien)
Primer design
How to test that the genes have been inserted (antibiotic resistance)
Make a phase planning and talk to Peter (divide into smaller parts and and test these individually)
Maya will start creating an illustration for the system during the weekend. Thomas will delete all irrelevant budgets from the dropbox. He will also send the project description to Sebastien.
Retrieved from " http://2010.igem.org/Team:DTU-Denmark/Notebook "
Recent changes
What links here
Related changes
Special pages
My preferences
Printable version
Permanent link
Privacy policy
Disclaimers
