<HTML xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en" dir="ltr">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki  ltr ns-0 ns-subject page-Team_DTU-Denmark_Modelling"><DIV id="globalWrapper"><DIV id="top-section"><DIV id="p-logo"><A href="/Main_Page" title="Main Page">&quot;
	    </A></DIV><DIV id="menubar" class="left-menu noprint"><UL><LI class="selected"><A href="/Team:DTU-Denmark/Modelling">Page               </A></LI><LI class="new"><A href="/wiki/index.php?title=Talk:Team:DTU-Denmark/Modelling&amp;action=edit&amp;redlink=1">Discussion               </A></LI><LI><A href="/wiki/index.php?title=Team:DTU-Denmark/Modelling&amp;action=edit">View source               </A></LI><LI><A href="/wiki/index.php?title=Team:DTU-Denmark/Modelling&amp;action=history">History               </A></LI><LI style="color:#808080;cursor:default">teams</LI></UL></DIV><DIV class="right-menu noprint" id="menubar"><UL><LI id="pt-login"><A href="/wiki/index.php?title=Special:UserLogin&amp;returnto=Team:DTU-Denmark/Modelling" title="You are encouraged to log in; however, it is not mandatory [o]" accesskey="o">Log in</A></LI></UL></DIV><DIV id="search-controls" class="noprint"><FORM action="/Special:Search" id="searchform"> </FORM></DIV></DIV><DIV id="content"><H1 class="firstHeading">Team:DTU-Denmark/Modelling</H1><DIV id="bodyContent"><H3 id="siteSub" class="noprint">From 2010.igem.org</H3><P><TITLE>Welcome to the DTU iGEM wiki!</TITLE><TABLE class="links"><TBODY><TR><TD align="center"><FONT face="arial" size="3"><A class="mainLinks" href="https://2010.igem.org/Team:DTU-Denmark">Home</A></FONT></TD><TD align="center"><FONT face="arial" size="3"><A class="mainLinks" href="https://2010.igem.org/Team:DTU-Denmark/Team">The Team</A></FONT></TD><TD align="center"><FONT face="arial" size="3"><A class="mainLinks" href="https://2010.igem.org/Team:DTU-Denmark/Project">The Project</A></FONT></TD><TD align="center"><FONT face="arial" size="3"><A class="mainLinks" href="https://2010.igem.org/Team:DTU-Denmark/Parts">Parts submitted</A></FONT></TD><TD align="center"><FONT face="arial" size="3"><A class="mainLinks" href="https://2010.igem.org/Team:DTU-Denmark/Results">Results</A></FONT></TD><TD align="center"><FONT face="arial" size="3"><A class="mainLinks" href="https://2010.igem.org/Team:DTU-Denmark/Notebook" title="Day to day lab activity">Notebook</A></FONT></TD><TD align="center"><FONT face="arial" size="3"><A class="mainLinks" href="https://2010.igem.org/Team:DTU-Denmark/Blog">Blog</A></FONT></TD></TR></TBODY></TABLE><TABLE cellpadding="10px"><TBODY><TR><TD width="163px" height="100%" valign="top"><FONT color="#990000" face="arial" size="3"><UL type="circle"><LI><A href="https://2010.igem.org/Team:DTU-Denmark/Background">Synthetic Biology</A></LI><LI><A href="https://2010.igem.org/Team:DTU-Denmark/Regulatory_sytems">Regulatory Systems</A></LI><UL><FONT size="2"><LI><A href="https://2010.igem.org/Team:DTU-Denmark/Regulatory_sytems#lambda">Lambda Phage</A></LI><LI><A href="https://2010.igem.org/Team:DTU-Denmark/Regulatory_sytems#gifsy">Gifsy Phages</A></LI></FONT></UL><LI><A href="https://2010.igem.org/Team:DTU-Denmark/Switch">The Switch</A></LI><UL><FONT size="2"><LI><A href="https://2010.igem.org/Team:DTU-Denmark/Switch#Biological_Switch">Biological Switches</A></LI><LI><A href="https://2010.igem.org/Team:DTU-Denmark/Switch#Bistable_Switches">Bistable Switches</A></LI><LI><A href="https://2010.igem.org/Team:DTU-Denmark/Switch#Design">Design of our Bi[o]stable Switch</A></LI><LI><A href="https://2010.igem.org/Team:DTU-Denmark/Switch#Engineering">Step-wise Engineering of the Switch</A></LI><LI><A href="https://2010.igem.org/Team:DTU-Denmark/Switch#Applications">Applications</A></LI></FONT></UL><LI><A href="https://2010.igem.org/Team:DTU-Denmark/SPL">Synthetic Promoter Library</A></LI><LI><A href="https://2010.igem.org/Team:DTU-Denmark/SPL#standard">The DTU SPL Standard</A></LI><LI><A href="https://2010.igem.org/Team:DTU-Denmark/SPL#strategy">Strategy</A></LI><LI><A href="https://2010.igem.org/Team:DTU-Denmark/SPL#design">Primer Design</A></LI><LI><A href="https://2010.igem.org/Team:DTU-Denmark/SPL#protocol">Protocol</A></LI></UL><LI><A href="https://2010.igem.org/Team:DTU-Denmark/SPL#advantages">Advantages</A></LI></FONT><LI><A href="https://2010.igem.org/Team:DTU-Denmark/Modelling">Modeling</A></LI><LI><A href="https://2010.igem.org/Team:DTU-Denmark/Modelling#Approach">Modeling Approach</A></LI></TD><TD><FONT color="#990000" face="arial" size="5"><B>Introduction</B></FONT></TD></TR></TBODY></TABLE></P><P align="justify">One of the ideas behind synthetic biology is to engineer artificial systems in biological cells. This can be achieved by using well-characterized and simplified parts that can be found in nature. We will be using parts found in different systems in nature to design a bistable switch, an idea stolen from the world of electronics. A simplified version of our bistable switch is illustrated in Figure 1. </P><TABLE class="https://static.igem.org/mediawiki/2010/9/9e/DTU_modeling1.png" align="center"><CAPTION align="bottom"><P align="justify"><B>Figure 1</B>: Simplified version of our bistable swicth</P></CAPTION></TABLE><P align="justify">This simplified version of our switch, the repressor-repressor switch is analogous SR flip-flop of digital electronics. The SR (set-reset) flip-flop is constructed from a pair of cross-linked NAND or NOR logic gates. An SR flip-flop circuit built with NOR logic gates is illustrated in Figure 2.</P><TABLE class="https://static.igem.org/mediawiki/2010/5/53/SRflipflop.png" align="center"><CAPTION align="bottom"><P align="justify"><B>Figure 2</B>: SR flip-flop</P></CAPTION></TABLE><P align="justify">The circuit, built with the pair of cross-linked NOR logic gates, functions such that output from a gate will only be true if no input is given as demonstrated by the truth table in Table 1. The biological interpretation of this logical operation shows is that the repression by a protein can be prevented either by an inducer (or anti-repressor protein as in our case) (Input 1/2) or by repression of its synthesis by another repressor-protein (Output 2/1). The elements in the circuit do not possess memory in them, but by cross-linking the elements, the circuit is capable of “remembering” and holding its current state. This means that when neither Input 1 nor Input 2 (no inducers) is present, the circuit is bistable with either Output 2 or Output 1 being active (only one of the repressor-proteins are expressed), respectively.</P><TABLE cellpadding="2" cellspacing="0" border="1px" align="center"><FONT size="2"><CAPTION><P align="justify"><B>Table 1</B></P></CAPTION><THEAD><TR><TD align="center"><B>Input 1(Anti-Repressor 1)</B></TD><TD align="center"><B>Input 2(Anti-Repressor 2)</B></TD><TD align="center"><B>Output 1(Reporter 1)</B></TD><TD align="center"><B>Output 2(Reporter 2)</B></TD></TR><TR><TD align="center">0</TD><TD align="center">0</TD><TD align="center">No change</TD><TD align="center">No change</TD></TR><TR><TD align="center">0</TD><TD align="center">1</TD><TD align="center">1</TD><TD align="center">0</TD></TR><TR><TD align="center">1</TD><TD align="center">0</TD><TD align="center">0</TD><TD align="center">1</TD></TR><TR><TD align="center">1</TD><TD align="center">1</TD><TD align="center">Invalid</TD><TD align="center">Invalid</TD></TR></THEAD></FONT></TABLE><FONT color="#990000" face="arial" size="5"><B>Modeling approach</B></FONT><P align="justify">The simplified version of our <A href="https://2010.igem.org/Team:DTU-Denmark/Switch#final" target="_blank">Bi[o]stable switch</A> illustrated in Figure 1 was used in the modeling. A system must possess two or more stable states for it to function as a switch. The essential components of the switch are the two repressors and two repressible promoters. Each promoter is inhibited by the repressor expressed from the opposing promoter. The aim of our project is to design a robust bistable switch that exhibits bi-stability over a wide-range of parameters and that the two states are tolerant of the fluctuations inherent in gene expression. This will allow the latest induced state to remain constant over cell-generations. The bi-stability will arise as a result of the mutually inhibitory arrangement. As denoted in Table 1, the absence of inducers will leave the cell in its current state, where either promoter 1 expresses repressor 1 or promoter 2 expresses repressor 2. The expression of the repressor will prevent expression from the opposing promoter, preventing the spontaneous change of state.</P><P align="justify">The change of state is accomplished by transiently introducing an inducer (or an anti-repressor protein as in our case) of the currently active repressor. Maximal expression from the opposing promoter and establishment of the alternative stable state will ensue (Gardner, T.S. et al, 2000)(Cherry, J., 2000).</P><H3>Michaelis-Menten Function vs Hill Function</H3><P align="justify">In our repressor-repressor switch, the simplest switch is one where each repressor binds to its operator by mass action and the rate of transcription is proportional to the amount of unbound operator. A Michaelis-Menten equation can approximate the fraction of bound operator when the molar quantity of the repressor is greater than the operator. The rate of production of repressor protein <I>x</I> is given by:</P><P align="justify">Where <I>k<SUB>1</SUB></I> is the maximal rate of protein production, <I>K<SUB>D</SUB></I> is the equilibrium constant for dissociation of repressor from its operator site, <I>y</I> is the concentration of repressor protein y</P><P align="justify">By expressing K<SUB><I>d</I></SUB> in terms of repressor y:</P><P align="justify">And conversely, the rate of production for repressor protein 2 is given by:</P><P align="justify">Where similarly, <I>k<SUB>2</SUB></I> is the maximal rate of protein production of repressor y, <I>K<SUB>D</SUB></I> is the equilibrium constant for dissociation of repressor x from its operator site, <I>x</I> is the concentration of repressor protein x</P><P align="justify">This results in the following repressor-repressor system:</P><P align="justify">The resulting nullclines are defined by:</P><P align="justify">At equilibrium:</P><P align="justify">This results in a linear fractional transformation with a negative second derivative everywhere and is unable to have multiple intersections with the diagonal. As the multiple intersections of the second derivative with the diagonal is a requirement for a switch, the repression described can therefore not support a switch. However, the inclusion of the depletion term and more than one operator site per cell makes a switch possible. The Hill function that takes cooperativity of binding when describing the level of repression is:</P><P align="justify">The behavior and conditions for bistability can be elucidated with the following dimensionless model by replacing the Hill equation into the repressor-repressor system, where the depletion term is also taken into account:</P><P align="justify">The parameters k<SUB>1</SUB> and k<SUB>2</SUB> describes the net effect of RNA polymerase binding, open-complex formation, transcription elongation, transcription termination, repressor binding, ribosome binding and translation. The cooperativity, described by n, can be due to the multimerization of the repressor proteins as well as the cooperative binding of repressor multimers to multiple binding sites within the operator. The nullclines of the system are:</P><TABLE class="https://static.igem.org/mediawiki/2010/6/62/DTU_modeling_fi3.png" align="center"><CAPTION align="bottom"><P align="justify"><B>Figure 3</B>: Shows the two stable states and the unstable state of the switch.</P></CAPTION></TABLE><P align="justify">The nullclines intersect at three points, resulting in the two steady states and one unstable state shown in Figure 3. From this it is clear that three intersection points occur due to the sigmoid shape of the graph, which arises when n &gt; 1 showing that cooperativity of binding is necessary. Another key that can be extracted is the importance of similar rates of synthesis of both repressors. If the rates are unbalanced, the nullclines will only intersect once producing only a single stable state. An increase in the rate of repressor synthesis will result in a more robust switch (Gardner, T.S. et al, 2000)(Cherry, J., 2000). Figure 4 illustrates the functions of a good switch.</P><TABLE class="https://static.igem.org/mediawiki/2010/1/14/DTU_modeling_goodswitch.png" align="center"><CAPTION align="bottom"><P align="center"><B>Figure 4</B></P></CAPTION></TABLE><H3>Robustness of the switch</H3><P align="justify">Real switches are prone to stochastic fluctuations or noise, this is a consequence of the lack of separation of the states shown in Figure 5. Poor transversality of the nullclines, illustrated in Figure 6, due to real world fluctuations in parameters can lead to spontaneous change of state and thereby also loss of the information stored by the switch. Systems with highly transverse nullclines are able to withstand movements of the nullclines without spontaneous change of state. Stochastic fluctuations in protein levels occur due to the fact that the number of protein and mRNA molecules are finite. This means that it is more beneficial for a switch to be based on high concentrations of low-affinity repressors compared to a low concentration of high-affinity repressors. The cost of additional protein production is balanced out by the reduction of noise in the system (Gardner, T.S. et al, 2000)(Cherry, J., 2000).</P><TABLE class="https://static.igem.org/mediawiki/2010/2/2b/DTU_modeling_poorsep.png" align="center"><CAPTION align="bottom"><P align="center"><B>Figure 5</B></P></CAPTION></TABLE><TABLE class="https://static.igem.org/mediawiki/2010/7/70/DTU_modeling_poortrans.png" align="center"><CAPTION align="bottom"><P align="center"><B>Figure 6</B></P></CAPTION></TABLE><H3>References</H3><UL><LI>Gardner, T.S., Cantor, C.R. &amp; Collins, J.J. Construction of a genetic toggle switch in Escherichia coli. Nature 403, 339-342 (2000).
</LI><LI>Cherry, J. How to make a Biological Switch. Journal of Theoretical Biology 203, 117-133 (2000).
</LI></UL><DIV class="printfooter">
Retrieved from &quot;<A href="http://2010.igem.org/Team:DTU-Denmark/Modelling">http://2010.igem.org/Team:DTU-Denmark/Modelling</A>&quot;</DIV></DIV></DIV><DIV id="footer-box" class="noprint"><DIV id="footer"><UL id="f-list"><LI id="t-recentchanges"><A href="/Special:RecentChanges" title="Recent changes">Recent changes</A></LI><LI id="t-whatlinkshere"><A href="/Special:WhatLinksHere/Team:DTU-Denmark/Modelling" title="List of all wiki pages that link here [j]" accesskey="j">What links here</A></LI><LI id="t-recentchangeslinked"><A href="/Special:RecentChangesLinked/Team:DTU-Denmark/Modelling" title="Recent changes in pages linked from this page [k]" accesskey="k">Related changes</A></LI><LI id="t-specialpages"><A href="/Special:SpecialPages" title="List of all special pages [q]" accesskey="q">Special pages</A></LI><LI><A href="/Special:Preferences">My preferences</A></LI></UL></DIV><DIV id="footer"><UL id="f-list"><LI id="t-print"><A href="/wiki/index.php?title=Team:DTU-Denmark/Modelling&amp;printable=yes" title="Printable version of this page [p]" accesskey="p">Printable version</A></LI><LI id="t-permalink"><A href="/wiki/index.php?title=Team:DTU-Denmark/Modelling&amp;oldid=207511" title="Permanent link to this revision of the page">Permanent link</A></LI><LI id="privacy"><A href="/2010.igem.org:Privacy_policy" title="2010.igem.org:Privacy policy">Privacy policy</A></LI><LI id="disclaimer"><A href="/2010.igem.org:General_disclaimer" title="2010.igem.org:General disclaimer">Disclaimers</A></LI></UL></DIV></DIV></DIV></BODY></HTML>