"
Page
Discussion
View source
History
teams
Log in
 
Team:DTU-Denmark/SPL
From 2010.igem.org
Welcome to the DTU iGEM wiki!
Home
The Team
The Project
Parts submitted
Results
Notebook
Blog
Synthetic Biology
Regulatory Systems
Lambda Phage
Gifsy Phages
The Switch
Biological Switches
Bistable Switches
Design of our Bi[o]stable Switch
Step-wise Engineering of the Switch
Applications
Synthetic Promoter Library
The DTU SPL Standard
Strategy
Primer Design
Protocol
Advantages
Modeling
Modeling Approach
Introduction to Synthetic Promoter Libraries
Modulation of gene expression of i.e. cellular enzyme activities (Solem and Jensen 2002), as well as regulation of transcription are amongst some of the areas where SPLs are currently being used. SPL provides an alternative method for gene regulation compared to older methods, namely those of gene knockouts and strong over expression. These two methods are usually based upon apparent rate limiting steps within metabolic pathways (Jensen and Hammer 1998).
When working with gene regulation, it is important to elucidate where expression levels are optimal for the given gene being worked on. Under these specifications it is essential to be able to have slight increments in expressional strength when attempting to optimize gene expression. This can be achieved by the usage of an SPL, where the variability in strengths can be achieved by either randomizing the spacer sequences, namely the 17 bases that reside between the -35 and -10 consensus regions, and/or some of the bases within the consensus regions, being the -35 and -10 regions.
The spacer sequences that surround the consensus regions contribute significantly to the strengths of promoters (Hammer et al. 2006). In our design, we decided to both randomize the spacer sequences as well as two bases in both consensus regions as seen in Figure 1 below. N stands for 25% each of A, C, G and T, while S stands for 50% each of C and G, and W stands for 50% A and T.
Figure 1: An SPL designed on the basis of randomizing both the spacer sequences surrounding the consensus regions (-35 and -10 regions) as well as randomizing two bases within each of the consensus regions is illustrated. N stands for 25% each of A, C, G and T, while S stands for 50% each of C and G, and W stands for 50% A and T.
The point of randomizing both areas is to obtain a promoter library that is not biased towards being strong. This is achieved by giving two bases within each of the consensus regions a 50% chance of being their original bases, ensuring that only 1/16 of all promoters will be strong. This is without taking into consideration the fraction of strong promoters obtainable from the randomized spacer sequences.
As previous studies indicate, consensus regions outside of the -35 and -10 regions seem to contribute very little in terms of altering promoter strengths. Mutating a single nucleotide will not change the promoter strength substantially, however mutating many nucleotides in the spacer sequences surrounding the -35 and -10 regions seem to result in the most significant alterations in promoter strengths. This might be due to the three-dimensional structure that forms from the sequences that are arranged from the randomized spacer sequences (Jensen and Hammer 1998).
When wanting to characterize and/or fine tune BioBrick parts and devices, using promoters that are constrained to already set strengths, has the disadvantage that the promoter might induce gene expression that is either too high or too low for the cell to be viable. This problem is nonexistent when using SPL since the SPL will necessarily give you the allowed upper and lower bounds of gene expression for cell viability. Cells with too strong or too weak promoters will simply never grow colonies.
The DTU Synthetic Promoter Library Standard
We utilized the SPL technology to create a new BioBrick Standard in the form of a BioBrick Foundation Request For Comments (BBF RFC). The DTU Synthetic Promoter Library Standard was submitted as BBF RFC 63.
Strategy for Integrating SPL into the BioBrick Assembly Standard
There are many different ways to integrate an SPL into the BioBrick Standard, and a lot of ideas were considered when creating this RFC. However, in the end a method was chosen based on the fact that it would be least time consuming for teams looking to use SPL, and at the same time, be easy to do. Instead of relying on ligations to successfully insert the SPL onto the BioBrick plasmid backbone, a Polymerase Chain Reaction (PCR) method was designed to not only amplify the backbone but also add the SPL onto the linear BioBrick plasmid backbone at a specific chosen site (see Figure 2). Since most teams will probably have to amplify their backbones during the course of a project, this method will only require a small amount of extra work.
Wanting to optimize gene expression and thereafter choosing a promoter that conforms to the strength that efficiently expresses your gene would be best perceivable if the SPL could be easily added and removed from BioBrick parts and/or devices. That is why the SPL will be inserted by PCR in-between the restriction sites EcoRI and XbaI of the BioBrick prefix. This way it is possible to add a part downstream of the SPL by simply ligating a part into the backbone plasmid containing SPL or by using the 3A-assembly standard. Furthermore it is also possible to move the whole insert into another BioBrick plasmid backbone if needed.
Figure 2: The linear BioBrick plasmid backbone with SPL inserted between the EcoRI and XbaI sites of the BioBrick prefix is illustrated.
The design of the SPL leads to the possibility of illegal restriction sites being present within the randomized spacer sequence. If a given promoter is to be used in further ligations it is vital that the promoter is sequenced first to ensure that it does not contain any recognition sites for EcoRI, XbaI, SpeI or PstI. The presence of these recognition sites could lead to the promoter being cut in a future restriction digest
Primer Design
A PCR MUST be used in order to add the SPL onto the BioBrick plasmid backbone. The following primers for amplification of BioBrick plasmid backbones were used as a starting point for the design of our SPL primers:
Primer Suffix-F: 5’-ACTAGTAGCGGCCGCTGCAG-3’
Primer Prefix-R: 5’-TCTAGAAGCGGCCGCGAATTC-3’
The primers were taken from Parts Registry . The restriction enzyme recognition sites are marked with the following colors:
Blue – EcoRI
Green – XbaI
Red – SpeI
Orange – PstI
In order to amplify and add the SPL successfully, the following modifications were made to both of the annealing primers:
Primer SPL Suffix-F: 5’-GTTTCTTCACTAGTAGCGGCCGCTGCAG-3’
For this primer, a tail with the recommended extra bases has been added. For more information click here .
Depending on which backbone needs to be amplified, one of the following SPL primers should be used:
Primer SPL Prefix-R-01: 5’-GTTTCTTCCTCTAGAAGCGGCNNNNATWWTANNNNNNNNNNNNNNNNNTGTSAWNNNNNCG  CGAATTCCAGAAATCATCCTTAGCG-3’
Primer SPL Prefix-R-02:  5’-GTTTCTTCCTCTAGAAGCGGCNNNNATWWTANNNNNNNNNNNNNNNNNTGTSAWNNNNNCG CGAATTCGAGTCACTAAGGGC-3’
These primers have the SPL sequence inserted between the EcoRI and XbaI sites. Furthermore, 14-18 nt have been added to the 3’ end of the primer to ensure that the primers’ annealing sequences are long enough. Table 1 contains a list showing which primer to use with regard to which backbone is chosen.
Figure 3: The primer binding sites on a BioBrick plasmid backbone as well as the final linear plasmid backbone that is generated by the PCR are illustrated.
Table 1: Prefix SPL primers that should be used depending on which BioBrick plamid backbone is selected for amplification are illustrated. The lengths of the BioBrick backbone plasmids are also listed.
BioBrick Plasmid Backbone
Primer II
Primer III
Sizes - bps
pSB1A3
-
2157
pSB1AC3
-
3055
pSB1AK3
-
3189
pSB1AT3
-
3446
pSB1C3
-
2072
pSB1K3
-
2206
pSB1T3
-
2463
pSB2K3
-
4425
pSB3C5
-
2738
pSB3K5
-
2936
pSB3T5
-
3252
pSB4A5
-
3395
pSB4C5
-
3221
pSB4K5
-
3419
pSB4T5
-
3735
Protocol for Adding SPL to a BioBrick Backbone Plasmid
In terms of primer annealing specificity, a touch down ramp PCR (Don et al. 1991)may be used, but as Table 2 illustrates, the melting temperatures (Tm) are relatively high and a standard PCR can therefore be run instead of a touch down ramp PCR.
Table 2: The Tm of the SPL primers. The IDT DNA oligo analyzer was used to calculate the Tm.
Primer
Tm - °C
I) Primer SPL Suffix-F
62.1
II) Primer SPL Prefix-R-01
59.8
III) Primer SPL Prefix-R-02
60
Depending on which BioBrick plasmid backbone is chosen (refer to Table 1 for the list) the selected primer pairs, being either I) & II) or I) & III), should be used for PCR.
It is recommended that a high fidelity polymerase enzyme i.e. Finnzyme’s Phusion polymerase enzyme is used to ensure a minimal amount of mutations occur in the BioBrick plasmid backbone during amplification.
The following amounts of substrates should be used if Phusion polymerase enzyme is used:
Table 3
PCR substrates
Volumes - μL
Total volume
50
Phusion Polymerase (0,02 U/μL)
0.5
x5 Phusion HF buffer
10
dNTP's (5μM)
2
Primer SPL Suffix-F (10μM)
1.25
Template - BioBrick plasmid backbone
1
ddH2O
33.5
For other polymerase enzymes, consult the manual for the polymerase for more specific information on PCR mixtures.
The following program has been optimized for use if Phusion polymerase enzyme is used with the SPL primers:
Table 4
Cycle step
Temperature - ºC
Time
Cycles
Initial denaturation
98
30 sec
1
Denaturation
98
10 sec
-
Annealing
63*
30 sec
20-25
Extension
72
30 sec / kb
-
Final extension
72
10 min
1
Hold
4
forever
1
*) When using Phusion polymerase enzyme for primers that are larger than 20 nt, the annealing temperature should be 3ºC higher than the actual Tm.
For other polymerase enzymes, consult the manual for the polymerase for more specific information on PCR programs.
After PCR is completed, a PCR purification should be performed using a PCR clean up kit (e.g. Macherey-Nagel – NucleoSpin Extract II).
The BioBrick plasmid backbone should now contain the SPL and is ready to be used as a vector in a BioBrick assembly.
Advantages of Using the DTU SPL Standard
An SPL allows the fine tuning of gene expression by creating a promoter library with a wide variety of strengths. For data from our SPL proof of concept experiments, please visit our SPL Results page
The SPL will necessarily give you the allowed upper and lower bounds of gene expression for cell viability.
The method for creating the BioBrick compatible SPL is quick and easy to perform. The method is practically identical to the procedure for generating linear BioBrick plasmid backbones, which many teams will most likely have to do anyway.
Once the SPL is in the linear BioBrick plasmid backbone, it can be used in a variety of experiments. A BioBrick part or device can be ligated into the plasmid backbone and the resulting ligation can be transformed and screened for colonies containing the promoter strength needed.
References
Don, R. H et al. (1991). ‘Touchdown’ PCR to circumvent spurious priming during gene amplification. Nucleic Acids Research. 19 (14): 4008.
Hammer, K., Mijakovic, I. and Jensen, P. R. (2006). Synthetic promoter libraries – tuning of gene expression. Trends in Biotechnology. 24(2): 53-55.
Jensen, P. R. and Hammer, K. (1998). The Sequence of Spacers between the Consensus Sequences Modulates the Strength of Prokaryotic Promoters. Applied and Environmental Microbiology. 64(1): 82-87.
Solem, C. and Jensen, P. R. (2002). Modulation of Gene Expression Made Easy. Applied and Environmental Microbiology. 68(5): 2397-2403.
Retrieved from " http://2010.igem.org/Team:DTU-Denmark/SPL "
Recent changes
What links here
Related changes
Special pages
My preferences
Printable version
Permanent link
Privacy policy
Disclaimers
