"
Page
Discussion
View source
History
teams
Log in
 
Team:DTU-Denmark/AntiTermination Section
From 2010.igem.org
Welcome to the DTU iGEM wiki!
Home
The Team
The Project
Parts submitted
Results
Notebook
Blog
Repressor - Anti-Repressor Section
Construction of BioBricks
Characterization
Strategy
Results
Terminator - Anti-Terminator Section
Characterization
Strategy
Results
Construction of BioBricks
Construction of Parts
Synthetic Promoter Library Section
Construction of BioBricks
Characterization Results
Strategy
Results
Introduction
In this section we focus on the lambda phage nut-site and N protein termination system. In lambda bacteriophage, gene expression is regulated by the suppression of transcription termination which is mediated by the lambda N protein that interacts with the nut site. We have constructed several test plasmids with different terminator strength and references. These plasmids are presented in the Construction of Parts section. The aim of the characterization experiments has been to test:
If it was possible to trigger the feedback mechanisms of the N protein by varying promoter strengths, it would be possible to observe increased read through of the terminator.
If the N-mechanisms can be triggered by induction of N from a seperate inducible plasmid.
Characterization
The characterization strategy and results are presented in this section, for more information on our constructs, see Construction of Parts . For more information about our BioBricks, see Construction of BioBricks .
Strategy
In order to test the funcionality of the antiterminator system the following 6 plasmids were constructed. The test plasmids pAT12-16 and the induction plasmid pAT01 are presented in the figures below.
Figure 1: pAT12: Insert´s organisation in plasmid pAT12 with pSB2K3 as a backbone. As the figure illustrates, there is an RBS followed by the FACS optimised GFP (from pAT03) and the natural lambda nutR (from pAT05). Next, there is the N gene followed by RBS and RFP (from pAT02) downstream of the nutR site.
Figure 2: pAT13, 14, 15: Insert´s organisation in plasmids pAT13-15 with pSB2K3 as the backbone. These plasmids contain RBS followed by the FACS optimised GFP (from pAT03). Next there is a nutR followed by a terminator site, which is different for each of the plasmids. pAT14 and pAT15 contain a single terminator site, namely BBa_B0011 (from pAT09) and BBa_B1003 (from pAT10) respectively, while pAT13 contain both (pAT11). Downstream of the terminator site, there is the N gene followed by RBS and RFP (from pAT02) present in each of the three constructs.
Figure 3: pAT16: Insert´s organisation in plasmid pAT16 with pSB2K3 as the backbone. First there is an RBS and the FACS optimised GFP (from pAT03), followed by nutR and terminator site BBa_B1003 (from pAT07). Downstream of the terminator there is another RBS followed by the RFP and a double terminator site BBa_B0010 and BBa_B0012 (from BBa_I13507).
Figure 4: pAT01:pAT01 contain the arabinose inducible promoter pBAD, and the N plasmid.
As described in the Construction of Parts , several promoters of different strengths have been introduced into the final constructs pAT12-pAT16, all containing GFP and RFP reporters. Together these promoters constitute Synthetic Promoter Library (SPL). Transcription through the terminator site results in the expression of the N antiterminator. If enough N protein is produced, the antitermination is mediated. Our system enables us to measure the promoter strength by measuring the amount of GFP produced. At the same time the amount of transcription past the promoter can be estimated by measuring the amount of RFP produced. This is one of the strategies for the characterization of our final constructs. For a more detailed explanation see the section on Synthetic Promoter Library (SPL).
Strains containing pAT12-16 have also been transformed with pAT01, containing pBAD and ARAC promoters upstream of lambda N-gene with its natural RBS. This has been done in order to induce expression of N protein. Supplement of bacterial cultures containing these plasmids with arabinose enhances the expression of the lambda N-gene and therefore increases the production of the N protein. As a consequence, the N produced in pAT12-16 can be measured and compared. Based on that, we have also tried to characterize our system by testing whether or not it is possible to trigger N-mediated antitermination by coupling the N protein to the pBAD promoter.
The 5 test plasmids were transformed into E. coli Xl1-blue, with the SPL in front of the constructs to give a large range of expressions. The first series of test strains are called series A. Isolated colonies from this transformation (series A) were again transformed with the inducible pAT01 plasmid containing pBAD and ARAC promoters upstream of lambda N-gene with its natural RBS to give the test series N. For an overview of the test strains constructed see the table below.
Table 1: Test cultures used for the characterization experiments. All plasmid constructs presented in the table have been transformed into E. coli Xl1-blue strain. The description of the plasmids can be found in the Construction of Parts section.
Series A
Plasmid
Series N
Plasmids
A
pAT12
AN
pAT12+pAT01
B
pAT13
BN
pAT13+pAT01
C
pAT14
CN
pAT14+pAT01
D
pAT15
DN
pAT15+pAT01
E
pAT16
EN
pAT16+pAT01
Results
In the following sections we present the experiments, results and analysis done on the test strains constructed and presented in the table above.
Fluorescent microscopy
Flourescence microscope was used to investigate the success rate and verify the preformed transformations. We looked at the first transformations done with the test constructs and the SPL, and selected 10 colonies from each construct for further analysis.
Results The results are summarized in the table below.
Table 2: Fluorescent microscopy of constructs A, B and E.
Construct
Green filter
Red Filter
Description
A:
Strong SPL
No Terminator
An example of a colony with a strong promoter from the SPL. The colony has both strong GFP and RFP signals.
A:
Weak SPL
No Terminator
An example of a weak SPL promoter. The colony has weak expression of both GFP and RFP. It can be seen that it is weak due to the background color intensity compared to the stong promoter above
B:
Strong SPL
Strong Terminator
The construct with the strong terminator B0015. The Colony has strong expression of GFP and only very weak expression of RFP proving the high efficiency of the terminator
B:
Multiple SPL
Strong Terminator
This image shows three colonies with weak, medium and strong promoters. The medium promoter in the bottom right corner cannot read through the strong terminator and express the RFP. The Strong promoter in the middle seems to have triggered the positive feedback mechanism and expresses RFP.
E:
No N protein
Weak Terminator
B1003
The control construct E, without the N protein. Again three different strengths of promoters strong, medium and weak. It is seen from this and other E-colonies that the B1003 terminator has a very weak effect. B1003 cannot create a visible difference in florescence, or the anti-terminator effect is triggered.
From the plates we further counted the numbers of strong and weak promoters and investigated if the positive feedback mechanism was triggered. Colonies from construct B, with the strong B0015 terminator and from C with the strong single terminator B0011, it was possible to see a clear termination effect in most of the colonies. Further a few colonies with very strong green florescence also had medium to strong red fluorescence. This is an indication on that the positive feedback mechanism from the N protein works. For B construct 15% of the colonies showed this triggered effect and for construct C 19% shows the triggered effect, see the table below.
Table 3: Triggered effects for the various constructs.
Total 2
n.d.
weak/no
weak/weak
strong/red
A
30
15
0
10
5
E
21
4
5
8
4
Total 2
n.d.
weak/no
strong/non
strong/red
B
39
4
21
8
6
C
21
3
6
8
4
Fluorescent Microscopy of Restreaks
We did a series of restreaks of the successful series A transformations described above. These were used in the Biolector experiments. We used fluorescent microscopy to confirm the BioLector results and the construct's stability.
The efficiency rate was drastically decreased, as was the consistency regarding terminator efficiency. Most of the colonies that had undetectable levels of GFP appeared to gain a mutation that prevented expression of functional GFP, but when isolated, all except for two colonies from each construct showed detectable levels. Only one colony (B10) from the selected and restreaked colonies of the B and C constructs expressed trigger mechanisms and expressed RFP, see Table 4.
Table 4: Examination of fluorescent colonies from the series A transformation after restreaking the selected colonies.
n.d.
mutation
weak/non
weak/weak
strong/red
A1-A10
9
1
0
0
0
E1-E10
6
0
2
0
2
B1-B10
6
1
4
0
1
C1-C10
3
1
5
1
0
Unfortunately only 4 of the 10 selected colonies run in the BioLector showed GFP expression when subsequently examined in the fluorescent microscope, and none of them had RFP expression. This is unfortunately consistent with the data achieved from the actual BioLector run.
Characterization of series B with inducible N protein
Out of the original colonies selected from the transformation of series A constructs, 11 were selected for transformation with pAT01, the inducible pBAD+Nprotein plasmid. This series of 11 strains was successfully transformed with pAT01.
BioLector
Overnight cultures were run in the BioLector and induced with arabinose after approx. 4 hours to induce N protein expression. This was to verify if the trigger mechanism observed using fluorescent microscopy could be induced. Out of the 9 cultures, only one of the E-constructs produced a GFP signal (strong promoter – weak Terminator – no N protein), but none of the constructs  expressed RFP.
Fluorometer
At the same time the overnight cultures were diluted and new overnight cultures were made with a reference and also induced with arabinose to test if a difference in the RFP expression could be seen.  None of the cultures expressed RFP. Data not shown. It was not possible to verify the trigger or inducible mechanism from this experiment. Through visual inspection, it appeared clear that the N induced strains appeared to be lysing.
Construction of BioBricks
In the following section we describe more in detail the method and strategy used for construction of our parts and Biobricks.
General sssemply standard and methods
In order to construct our biobricks, we used a set of forward and reverse primers to amplify a region of interest by using PCR. The fragments created by PCR amplification were purified using a PCR clean-up kit. The amplicon and the linearized backbone plasmid pSB1C3 (containing a chroramphenicol resistance marker) were digested , resulting in sticky ends. This was achieved either by the standard assembly or the three-way ligation approach (3A assembly). Both the digested PCR product and the digested plasmid were run on a gel in order to estimate DNA concentration. T4 ligase was used for ligation with a 5:1 ratio of insert to backbone. After the ligation, the plasmid was transformed into electrocompetent DH5α E. coli cells. After an hour of recovery in LB medium at 37 °C, the cells were plated on LB plates containing chloramphenicol (Cam) and left overnight. Several colonies from the plates were then selected and restreaked on LB+Cam plates in order to assure pure colonies. Overnight cultures of the transformants were made by taking one colony from each restreak and inoculating it in LB+Cam at 37 °C over night. Minipreps were made from the overnight cultures and a verification PCR was run on these in order to make sure that the plasmid had the expected insert.
Construction of BioBrick K374005
This part contains the lambda nutR site, inserted into the backbone plasmid pSB1C3. The lambda nutR site was sythesized by Integrated DNA Technology. In order to construct this part, the standard assembly ligation approach was used. In doing so, the nutR site was digested with restriction enzymes EcoRI and Pst1 and thereafter ligated into pSB1C3. The nutR site was verified by PCR using primers IG201 (VF2 forward primer) and IG004 (lambda nutR reverse primer). The following parts were taken into consideration when calculating the size of BioBrick K374005:
IG201 + nutR + IG004 tail = 140 + 118 + 26 = 284 base pairs.
Construction of BioBrick K374006
This part contains the lambda N-gene that is responsible for the suppression of transcription termination downstream of part BBa_K374005. The lambda N-gene was synthesized by Integrated DNA Technology. As with the construction of K374005, the standard assembly ligation approach was also used in the construction of this part. For size verification, the lambda N-gene was amplified by PCR with primers IG201 and IG006 (lambda N-gene reverse primer). The size of K374006 is therefore:
IG201 + IG006 tail + lambda N gene = 140 + 26 + 402 = 568 base pairs.
Construction of BioBrick K374007
This construct contains the lambda nutR site (BBa_K374005) and the downstream terminator BBa_B0015 (composed of two terminator parts, namely BBa_B0010 and BBa_B0012). The 3A assembly was used in the construction of this part. The nutR site has been digested with the restriction enzymes EcoRI and SpeI. The terminator part BBa_B0015 was, however, digested with Xbal and Pstl. NutR and BBa_B0015 were then ligated into the linearized plasmid pSB1C3 that had been restricted with EcoRl and Pstl. The size of K374007 has been verified by PCR with primers IG201 and IG202 (VR reverse primer) to be the following:
IG201 + IG202 + nutR = 140 + 176 + 255 = 571 base pairs.
Figure 5: Left: GeneRuler DNA ladder mix from Fermentas. Right: Verification PCRs of the Biobricks K374005 (284 base pairs) and K374007 (571 base pairs).
Construction of BioBrick K374013
This part contains lambda N-gene (K374007) with its natural RBS. 3A assembly was used to construct this part. The lambda N-gene was excised with the restriction enzymes EcoRl and Spel. BBa_I13507 (containing RBS and RFP) was cut with Xbal and Pstl. Both parts were then ligated into pSB1C3 that had been restricted with EcoRl and Pstl. Aftertransformation and selection of the transformed colonies, verification PCR with primers IG201 and IG006 was carried out. The estimated size of this part:
IG201 + IG006 tail + N-gene with RBS =140 + 26 + 420 = 586 base pairs.
Construction of BioBricks K37014 and K37015
The 3A assembly approach was used to construct these two parts. The lambda nutR site was exercised with EcoRl and Spel, while recipient vector pSB1C3 has been cut with EcoRl and Pstl. The BioBrick terminator, BBa_B1003 was restricted with Xbal and Pstl and ligated, along with the nutR site, into pSB1C3 to construct K37014. K37015 was constructed by restricting the BioBrick terminator, BBa_B0011, with Xbal and Pstl and ligated, along with the nutR site into pSB1C3. In order to ensure that the plasmid contained the desirable inserts, verification PCRs with primers IG201 and IG004 was carried out. The estimated sizes of the inserts are shown below:
IG201 + IG004 tail + nutR = 284 base pairs.
Figure 6: Left: GeneRuler DNA ladder mix from Fermentas. Right: Verification PCRs of the Biobricks K374013 (586 base pairs), K374015 (284 base pairs, lane 8) and K374014 (284 base pairs, lane 16).
Construction of BioBrick K374016
This construct contains lambda’s natural RBS site (BBa_B0034), followed by a FACS optimized mutant of the Green Fluorescent Protein (BBa_K374012) and nutR site (BBa_K374005). Again, the 3A assembly approach has been used. The RBS-GFP was excised with EcoRl and Spel, while the nutR site with Xbal and Pstl. RBS-GFP  and nutR were then ligated into pSB1C3 (with EcoRl and Pstl sticky ends). After transformation, the verification PCR with primers IG201  and IG004 was performed. The estimated size of this part includes the sizes of the following parts:
IG201 + GFP + nutR + IG004 tail + RBS + biobrick scar = 140+717+118+2618+6=1025 base pairs
Figure 7: Left: GeneRuler DNA ladder mix from Fermentas. Right: Verification PCRs of the Biobricks K374016 (1025 base pairs) and K374006 (568 base pairs).
Construction of Parts
The step-wise construction of the test plasmids and intermediate constructs are presented in this section. Some of these parts have been submitted to the parts registry as BioBricks. All of the parts constructed in our terminator system are presented in table 1. A more detailed description of the parts submitted as BioBricks and the experimental procedure of setting up these parts is presented in the Construction of BioBricks section below.
Table 5: Plasmids constructed for the characterization of the antiterminator function. Inserts and backbone plasmids are specified in the table, as well as the inserted reporter proteins. Parts submitted to the parts registry are identified. (S) indicates that the physical DNA has been submitted. Test plasmids and intermediate constructs are indentified as (Test-p) and (Int) respectively
Plasmid/part name Type Insert Reporter Backbone pAT01Test-p.pBAD and ARAC promoters, RBS and N-gene (natural lambda DNA)---pSB4A5 pATNK374006, SLambda N-gene---pSB1C3 pATN2K374011N with natural RBS------ pATN3K374012FACS optimized GFPmut2GFP--- pAT02K374013K374011 (lambda N-gene with its natural RBS), followed by BBa_B0034 (RBS) and BBa_E1010 (RFP)RFPpSB1C3 pAT03Int.BBa_B0034 (RBS) and K374012 (FACS optimised GFP)GFPpSB1C3 pAT04K374007, SpAT05 with BBa_B0015---pSB1C3 pAT05K374005, SLambda nutR---pSB1C3 pAT06K374015, SpAT05 with BBa_B0011---pSB1C3 pAT07K374014, SpAT05 with BBa_B1003---pSB1C3 pAT08K374016, SpAT03 and pAT05GFPpSB1C3 pAT09Int.pAT03 and pAT06GFPpSB1A3 pAT10Int.pAT03 and pAT07GFPpSB1A3 pAT11Int.pAT03 and pAT04GFPpSB1C3 pAT12Test-p.pAT08 and pAT02GFP, RFPpSB2K3 pAT13Test-p.pAT11 and pAT02GFP, RFPpSB2K3 pAT14Test-p.pAT09 and pAT02GFP, RFPpSB2K3 pAT15Test-p.pAT10 and pAT02GFP, RFPpSB2K3 pAT16Test-p.pAT10 with BBa_I13507pAT10 with BBa_I13507pSB2K3
The plasmids and parts have been constructed using existing BioBricks yet we have also submitted new genes that are not in the parts registry, these genes are:
The lambda N protein from e. Coli EMG2.
The lambda NutR site, from e. Coli EMG2.
A GFP – derived from GFP, called GFPmut2.
From these parts the following test plasmids pAT12 to pAT16, where constructed and are presented below.
Construction details
Content and information pertaining to the construction of our constructs are listed below.
pATN, pAT05
Natural Lambda N-gene (encoding the N protein) and nutR have been inserted into the backbone plasmid pSB1C3 to construct pATN and pAT05 respectively. These parts have been submitted to the parts registry.
pAT02
pAT02 containing RFP reporter, has been constructed by inserting K374011 (lambda N-gene with its natural RBS), followed by BBa_B0034 (RBS) and BBa_E1010 (RFP) into pSB1C3. This part has also been submitted to the parts registry.
pAT03
pAT03 has been constructed by inserting RBS site BBa_B0034 and K374012 (FACS optimised GFP) into the recipient plasmid pSB1C3.
pAT04, pAT06, pAT07
pAT05 with the lambda nutR insert has been used to construct pAT06, pAT07 and pAT04, also containing different terminator sites. Plasmids pAT06 and pAT07 contain a single terminator site, but with different terminator strengths. Thus, pAT06 contains BBa_B0011 with medium terminator strength, while pAT07 contains BBa_B1003, which is a strong terminator site. Plasmid pAT04 contains both BBa_B0011 and BBa_B1003. These plasmids have been constructed in order to test termination efficiency and have also been submitted to the parts registry.
pAT08, pAT09, pAT10 and pAT11
plasmids pAT05, pAT06, pAT07 and pAT04 have been used along with pAT03 to construct pAT08, pAT09, pAT10 and pAT11 respectively. pAT08-pAT11 contain a GFP reporter that is important for the construction of pAT12-15.
pAT12, pAT13, pAT15
pAT08 has been used to construct pAT12, pAT09 to construct pAT014, pAT10 to construct pAT15 and pAT11 has been used to construct pAT13. All of these new constructs also contain pAT02. The organisation of inserts in plasmid pAT12 is shown in figure 1, while the organisation of inserts in plasmids pAT13-15 is presented in figure 2.
pAT16
pAT16 has been constructed by inserting pAT10 and BBa_I13507 into pSB2K3 backbone. The insert organisation of pAT16 is shown in figure 3.
pAT12-pAT16
The organization of inserts in plasmids pAT12-16 have been shown in the Characterization Strategy section
Retrieved from " http://2010.igem.org/Team:DTU-Denmark/AntiTermination_Section "
Recent changes
What links here
Related changes
Special pages
My preferences
Printable version
Permanent link
Privacy policy
Disclaimers
