<HTML xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en" dir="ltr">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki  ltr ns-0 ns-subject page-Team_DTU-Denmark_Regulatory_sytems"><DIV id="globalWrapper"><DIV id="top-section"><DIV id="p-logo"><A href="/Main_Page" title="Main Page">&quot;
	    </A></DIV><DIV id="menubar" class="left-menu noprint"><UL><LI class="selected"><A href="/Team:DTU-Denmark/Regulatory_sytems">Page               </A></LI><LI class="new"><A href="/wiki/index.php?title=Talk:Team:DTU-Denmark/Regulatory_sytems&amp;action=edit&amp;redlink=1">Discussion               </A></LI><LI><A href="/wiki/index.php?title=Team:DTU-Denmark/Regulatory_sytems&amp;action=edit">View source               </A></LI><LI><A href="/wiki/index.php?title=Team:DTU-Denmark/Regulatory_sytems&amp;action=history">History               </A></LI><LI style="color:#808080;cursor:default">teams</LI></UL></DIV><DIV class="right-menu noprint" id="menubar"><UL><LI id="pt-login"><A href="/wiki/index.php?title=Special:UserLogin&amp;returnto=Team:DTU-Denmark/Regulatory_sytems" title="You are encouraged to log in; however, it is not mandatory [o]" accesskey="o">Log in</A></LI></UL></DIV><DIV id="search-controls" class="noprint"><FORM action="/Special:Search" id="searchform"> </FORM></DIV></DIV><DIV id="content"><H1 class="firstHeading">Team:DTU-Denmark/Regulatory sytems</H1><DIV id="bodyContent"><H3 id="siteSub" class="noprint">From 2010.igem.org</H3><P><TITLE>Welcome to the DTU iGEM wiki!</TITLE><TABLE class="links"><TBODY><TR><TD align="center"><FONT face="arial" size="3"><A class="mainLinks" href="https://2010.igem.org/Team:DTU-Denmark">Home</A></FONT></TD><TD align="center"><FONT face="arial" size="3"><A class="mainLinks" href="https://2010.igem.org/Team:DTU-Denmark/Team">The Team</A></FONT></TD><TD align="center"><FONT face="arial" size="3"><A class="mainLinks" href="https://2010.igem.org/Team:DTU-Denmark/Project">The Project</A></FONT></TD><TD align="center"><FONT face="arial" size="3"><A class="mainLinks" href="https://2010.igem.org/Team:DTU-Denmark/Parts">Parts submitted</A></FONT></TD><TD align="center"><FONT face="arial" size="3"><A class="mainLinks" href="https://2010.igem.org/Team:DTU-Denmark/Results">Results</A></FONT></TD><TD align="center"><FONT face="arial" size="3"><A class="mainLinks" href="https://2010.igem.org/Team:DTU-Denmark/Notebook" title="Day to day lab activity">Notebook</A></FONT></TD><TD align="center"><FONT face="arial" size="3"><A class="mainLinks" href="https://2010.igem.org/Team:DTU-Denmark/Blog">Blog</A></FONT></TD></TR></TBODY></TABLE><TABLE cellpadding="10px"><TBODY><TR><TD width="163px" height="100%" valign="top"><FONT color="#990000" face="arial" size="3"><UL type="circle"><LI><A href="https://2010.igem.org/Team:DTU-Denmark/Background">Synthetic Biology</A></LI><LI><A href="https://2010.igem.org/Team:DTU-Denmark/Regulatory_sytems">Regulatory Systems</A></LI><UL><FONT size="2"><LI><A href="https://2010.igem.org/Team:DTU-Denmark/Regulatory_sytems#lambda">Lambda Phage</A></LI><LI><A href="https://2010.igem.org/Team:DTU-Denmark/Regulatory_sytems#gifsy">Gifsy Phages</A></LI><LI><A href="https://2010.igem.org/Team:DTU-Denmark/Regulatory_sytems#antitermination">Anti-Termination systems</A></LI></FONT></UL><LI><A href="https://2010.igem.org/Team:DTU-Denmark/Switch">The Switch</A></LI><UL><FONT size="2"><LI><A href="https://2010.igem.org/Team:DTU-Denmark/Switch#Biological_Switch">Biological Switches</A></LI><LI><A href="https://2010.igem.org/Team:DTU-Denmark/Switch#Bistable_Switches">Bistable Switches</A></LI><LI><A href="https://2010.igem.org/Team:DTU-Denmark/Switch#Design">Design of our Bi[o]stable Switch</A></LI><LI><A href="https://2010.igem.org/Team:DTU-Denmark/Switch#Engineering">Step-wise Engineering of the Switch</A></LI><LI><A href="https://2010.igem.org/Team:DTU-Denmark/Switch#Applications">Applications</A></LI></FONT></UL><LI><A href="https://2010.igem.org/Team:DTU-Denmark/SPL">Synthetic Promoter Library</A></LI><LI><A href="https://2010.igem.org/Team:DTU-Denmark/SPL#standard">The DTU SPL Standard</A></LI><LI><A href="https://2010.igem.org/Team:DTU-Denmark/SPL#strategy">Strategy</A></LI><LI><A href="https://2010.igem.org/Team:DTU-Denmark/SPL#design">Primer Design</A></LI><LI><A href="https://2010.igem.org/Team:DTU-Denmark/SPL#protocol">Protocol</A></LI></UL><LI><A href="https://2010.igem.org/Team:DTU-Denmark/SPL#advantages">Advantages</A></LI></FONT><LI><A href="https://2010.igem.org/Team:DTU-Denmark/Modelling">Modeling</A></LI><LI><A href="https://2010.igem.org/Team:DTU-Denmark/Modelling#Approach">Modeling Approach</A></LI></TD><TD><FONT color="#990000" face="arial" size="5"><B>The lambda phage</B></FONT></TD></TR></TBODY></TABLE></P><P align="justify">The temperate bacteriophage lambda found to infect <I>E. coli</I>, is the best-studied phage with regard to phage structure and regulation. Once lambda infects a cell, it can chose to either follow the lytic cycle or the lysogenic cycle described in Figure 1.</P><TABLE class="http://upload.wikimedia.org/wikipedia/commons/5/5a/Phage2.JPG" align="center"><CAPTION align="bottom"><P align="justify"><B>Figure 1</B>: The two distinct development pathways of a prophage life cycle [wikipedia].In the lytic pathway the phage uses the bacterial molecular machinery to make many viral copies for infection of other cells before lysing the host bacterium. In contrast to the lytic pathway, the phage integrates its DNA into the bacterial genome in the lysogenic pathway. The lysogenic state is very stable, which means that the prophage can be replicated along with the bacterial genome for generations.
Despite the stability of the lysogenic state, the lytic state is readily induced when the bacteria are irradiated with ultraviolet light.</P></CAPTION></TABLE><P align="justify">The organization of the phage chromosome is shown in Figure 2.</P><TABLE class="http://upload.wikimedia.org/wikipedia/commons/e/e4/Bacteriophage_lambda_genome.png" align="center"><CAPTION align="bottom"><P align="justify"><B>Figure 2</B>: Schematic representation of the genome of the bacteriophage lambda [wikipedia].</P></CAPTION></TABLE><H3>The Lysogenic Pathway</H3><P align="justify">The rightward promoter/operator region in lambda prophages forms a switch with the two genes <I>cro</I> and <I>cI</I> under mutually exclusive expression in the lambdoid phage.</P><P align="justify">This regulatory region consists of the promoters pRM and pR and the three sub-operator sites, OR1, OR2 and OR3, in-between the promoter. The interesting feature about these two promoters is that they are orientated in a back-to-back fashion and are also called divergent or bidirectional promoters. The sub-operator sites are shared regulatory regions that influence the expression of two oppositely oriented genes. This type of promoter arrangement is common in prokaryotes and is also found in humans and other higher species. The pR promoter is very strong and has greater similarity to the promoter consensus sequence than the very weak pRM promoter [Strainic et al.,1999].</P><P align="justify">The <I>cI</I> gene encodes cI, the lambda repressor protein. The presence of this protein stabilizes the lysogenic state and causes immunity to superinfections by other lambda phages. cI has a dual function by acting as repressor and activator. It represses transcription from the pR promoter while up-regulating its own expression from the pRM promoter.</P><P align="justify">A dimer is formed by the cI repressor and binds to DNA in the helix-turn-helix binding motif. The cI repressor binds to all three operator sites in the order OR1 = OR2 &gt; OR3, because of its different intrinsic affinities for the operator sites.</P><P align="justify">The repressor binds with highest affinity to OR1 and that stimulates binding of more cI to OR2 by a mechanism called positive cooperative binding. Binding to OR1 and OR2 blocks binding of RNA polymerase to the pR promoter, so switching to the lytic cycle is prevented. At high repressor concentrations cI down-regulates its own expression by binding to OR3, so all three operator sites are occupied and expression from the pRM promoter is limited i.e cI, while the expression from cro genes still is inhibited. It is important to understand, that regulation of the switch is solely dependent on repressor concentrations and not by other regulatory proteins e.g. anti-repressors. The Cro protein does not bind with positive cooperation to the three operator sites whilst cI does. The result is that cI and the lysogenic state is more stable, because it can outcompete the cro protein. </P><H3>The Lytic Pathway</H3><P align="justify">The lytic pathway of lambda is induced by the SOS response after DNA damage in <I>E. coli</I> by e.g. UV light. This is achieved when the repressor protein cI is cleaved by a protein expressed during SOS response, RecA [Lemire et al.,2008]. After induction of the lytic cycle, transcription through the pL and pR promoters is instigated. This results in the expression of the N protein from the pL promoter. The N protein participates in anti-termination, a process that prevents termination of transcription by termination signal. This allows transcription through termination signals located downstream of the early genes.</P><P align="justify">The <I>nut</I>-site plays an important role in anti-termination as it promotes the formation of an anti-termination complex on the nascent mRNA, consisting of the lambda N protein and four <I>E. coli</I> proteins, NusA, NusB, NusE and NusG. The <I>nut</I>-site is made up of regions, the conserved <I>boxA</I> region and the variable <I>boxB</I> region. 

NusA and the N protein is necessary for anti-termination effect. NusA native to e-coli binds to the spacer region, between boxA and boxB, and the N protein binds to boxB. NusA has a 50% greater affinity for <I>nutL</I>, found downstream of the pL promoter, compared to <I>nutR</I>, found downstream of the pR promoter. This mode of anti-termination is also found in other lambda-related phages such as P21 and P22. Under normal conditions, these N analog proteins show specificity for <I>nut</I>-sites of the respective phages, but this specificity is lost when the N proteins are overexpressed.</P><TABLE class="https://static.igem.org/mediawiki/2010/0/05/Anti-termination-complex.PNG" align="center"><CAPTION align="bottom"><P align="justify"><B>Figure 3</B>: The anti-termination complex is illustrated. The anti-termination complex consists of RNA-polymerase, the hosts proteins, NusA, NusB, NusE and NusG and (Burmann et.al. 2010).</P></CAPTION></TABLE><P align="justify">Expression of the Q protein is observed as a result of the anti-termination by the N protein during transcription from the pR promoter. Q is another protein that is capable of causing anti-termination. However, unlike the N-protein, it prevents termination of transcription by the binding to DNA sequences just upstream of the late pR' promoter, resulting in transcription of the late genes.</P><P align="justify">Few papers describe and test the distance between the nut-site and the termination steam loop. NusA and N have been found to be sufficient for anti-termination under in vitro conditions, however, N is unable of singularly inducing anti-termination. In the tested construct, <I>nutL</I> was placed 200nt upstream of the lambda right terminator (Whalen et.al.1988). The required distance between <I>boxB</I>, within the nut-site, to the terminator stem loop has yet to be determined but it was placed 26 bp from the beginning of the terminator stem loop.</P><FONT color="#990000" face="arial" size="5"><B>The Gifsy phages: Gifsy1 and Gifsy2</B></FONT><P align="justify">Gifsy 1 and Gifsy 2 are temperate phages present in the vast majority of <I>Salmonella enterica</I> serovar Typhimurium strains; the genomic positioning of the prophages is illustrated in Figure 2. This strain of pathogenic bacteria infects a broad spectrum of animal species, from reptiles to mammals [1].</P><TABLE class="https://static.igem.org/mediawiki/2010/3/35/RegSysFig1_DTU.png" align="center"><CAPTION align="bottom"><P align="justify"><B>Figure 2</B>: The positions of the prophage inserts into the Salmonella genome [Lemire et al., 2008].</P></CAPTION></TABLE><P align="”justify”">Similar to the lambda phage, the Gifsy phages follow either lytic cycle or lysogenic cycle after infecting <I>S. enterica</I>.</P><P align="justify">Salmonella strains harbor different subsets of prophages and therefore have differences in prophage distribution, lysogeny thereby contributes to the genetic diversity of Salmonella genomes. Since phages are able to switch between these two different developmental states, they are a very interesting example of a natural genetic switch and this is why we choose to use the key regulatory elements from Gifsy-1 and Gifsy-2 prophages to construct our own bistable switch in <I>E.coli</I> [Lemire et al.,2008].</P><H3>Chromosomal organization</H3><P align="justify">The overall gene organization of Gifsy-1 and Gifsy-2 prophages is typical of the lambdoid phage family that is illustrated in Figure 2 [Lemire et al.,2008].</P><P align="justify">The regulation region of the lambda phage, also called immunity region, is of particular interest to us. This region includes the three promoters pR, pRM and pL, the left- and rightward operator and the <I>cro</I> and <I>cI</I> gene responsible for controlling the switch between lysogenic and lytic growth. The Gifsy chromosome and its immunity region are organized in a very similar way to that of the lambda phage as illustrated by Figure 4 [Lemire et al.,2008].</P><TABLE class="https://static.igem.org/mediawiki/2010/4/43/RegSysFig3_DTU.png" align="center"><CAPTION align="bottom"><P align="justify"><B>Figure 4</B>: The immunity region in Gifsy phages is illustrated [Lemire et al.,2008].</P></CAPTION></TABLE><H3>Regulation of promoters and repressors</H3><P align="justify">GogR and GtgR are repressor proteins found in Gifsy-1 and Gifsy-2, respectively. These repressor proteins are analogous to the lambda repressor protein, cI, previously described. The Gifsy repressors (136 aa) are much smaller than the lambda repressor cI (237 aa) and lack the typical cleavage motif [Lemire et al.,2008]. The mechanism by which the repressors GogR and GtgR regulate the Gifsy promoters, pR and pRM is analogous to that of cI in the lambda phage with the exception of the mode of lytic induction.</P><P align="justify">As previously mentioned, GogR and GtgR lack the cI cleavage site and are inactivated by binding of small anti-repressor proteins, called AntO and AntT. The Gifsy genes encoding these proteins are located outside the immunity region and are under the direct control of the LexA protein. This protein is the major regulator of the SOS regulon and this regulon is activated by the cleavage of LexA by RecA. The interesting point is that Gifsy and lambda prophage regulation is an integral part of the SOS response i.e. once the SOS response is triggered, the lytic pathway is also induced [Lemire et al.,2008].</P><H3>References</H3><P align="justify"><UL><LI> Lemire, S., N. Figueroa-Bossi, and L. Bossi. 2008. Prophage contribution to Salmonella virulence and diversity, p. 159-192. In M. Hensel and H. Schmidt (ed.), Horizontal gene transfer in the evolution of bacterial pathogenesis.. Cambridge University, Cambridge, England.</LI><LI> Lemire, S., Figueroa-Bossi, N. &amp; Bossi, L. A Singular Case of Prophage Complementation in Mutational Activation of recET Orthologs in Salmonella enterica Serovar Typhimurium. J. Bacteriol. 190, 6857-6866 (2008).</LI><LI> Promoter Interference in a Bacteriophage Lambda Control Region: Effects of a Range of Interpromoter Distances
Michael G. Strainic, Jr., Jennifer J. Sullivan, Julio Collado-Vides, and Pieter L. deHaseth</LI><LI> Bacteriophage lambda. July 2010, 10:55 UTC. In Wikipedia: The Free Encyclopedia. Wikimedia Foundation Inc. Encyclopedia on-line. Available from http://en.wikipedia.org/wiki/Bacteriophage_lambda. Internet. Retrieved 10 August 2010.</LI><LI> Bacteriophage lysogenic and lytic cycle. 21 October 2008, 22:34 UTC. In Wikipedia: The Free Encyclopedia. Wikimedia Foundation Inc. Encyclopedia on-line. Available from http://en.wikipedia.org/wiki/File:Phage2.JPG. Internet. Retrieved 10 August 2010.</LI><LI> Schematic representation of the genome of the bacteriophage lambda. 21 March 2009, 14:58 UTC. In Wikipedia: The Free Encyclopedia. Wikimedia Foundation Inc. Encyclopedia on-line. Available from http://en.wikipedia.org/wiki/File:Bacteriophage_lambda_genome.png. Internet. Retrieved 10 August 2010.</LI><LI> Lou, C. et al. Synthesizing a novel genetic sequential logic circuit: a push-on push-off switch. Mol Syst Biol 6,  (2010).</LI><LI> Hensel, M. &amp; Schmidt, H. Horizontal gene transfer in the evolution of pathogenesis. P 5 (Cambridge University Press: 2008).</LI><LI> Burmann. B., Schweimer. K., Luo. X., Wahl. M., Stitt. B., Gottesmann. M., Rösch. P., &quot; A NusE:NusG Complex Links Transcription and Translation&quot; Science 2010.</LI><LI> Nudler. E., Gottesman. M.E., &quot;Transcription termination and anti-termination in E.coli&quot;, Genes to Cells (2002) </LI><LI> Prasch. S., Jurk. M., Washburn. R.S., Gottesman. M.E., Wöhrl. B., Rösch. P., &quot;Rna-binding specificity of E. coli NusA&quot; Nucleic Acids Research 2009.</LI><LI> Whalen. W., Ghosh. B., Das. A., &quot;NusA protein is necessary and sufficient in vitro for phage lambda N gene product to suppress a rho-independent terminator placed downstream of nutL. Proc.Natl. Acad sci. 1988</LI></UL></P><DIV class="printfooter">
Retrieved from &quot;<A href="http://2010.igem.org/Team:DTU-Denmark/Regulatory_sytems">http://2010.igem.org/Team:DTU-Denmark/Regulatory_sytems</A>&quot;</DIV></DIV></DIV><DIV id="footer-box" class="noprint"><DIV id="footer"><UL id="f-list"><LI id="t-recentchanges"><A href="/Special:RecentChanges" title="Recent changes">Recent changes</A></LI><LI id="t-whatlinkshere"><A href="/Special:WhatLinksHere/Team:DTU-Denmark/Regulatory_sytems" title="List of all wiki pages that link here [j]" accesskey="j">What links here</A></LI><LI id="t-recentchangeslinked"><A href="/Special:RecentChangesLinked/Team:DTU-Denmark/Regulatory_sytems" title="Recent changes in pages linked from this page [k]" accesskey="k">Related changes</A></LI><LI id="t-specialpages"><A href="/Special:SpecialPages" title="List of all special pages [q]" accesskey="q">Special pages</A></LI><LI><A href="/Special:Preferences">My preferences</A></LI></UL></DIV><DIV id="footer"><UL id="f-list"><LI id="t-print"><A href="/wiki/index.php?title=Team:DTU-Denmark/Regulatory_sytems&amp;printable=yes" title="Printable version of this page [p]" accesskey="p">Printable version</A></LI><LI id="t-permalink"><A href="/wiki/index.php?title=Team:DTU-Denmark/Regulatory_sytems&amp;oldid=207763" title="Permanent link to this revision of the page">Permanent link</A></LI><LI id="privacy"><A href="/2010.igem.org:Privacy_policy" title="2010.igem.org:Privacy policy">Privacy policy</A></LI><LI id="disclaimer"><A href="/2010.igem.org:General_disclaimer" title="2010.igem.org:General disclaimer">Disclaimers</A></LI></UL></DIV></DIV></DIV></BODY></HTML>