<HTML xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en" dir="ltr">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki  ltr ns-0 ns-subject page-Team_UiOslo_Norway_Notebook"><DIV id="globalWrapper"><DIV id="top-section"><DIV id="p-logo"><A href="/Main_Page" title="Main Page">&quot;
	    </A></DIV><DIV id="menubar" class="left-menu noprint"><UL><LI class="selected"><A href="/Team:UiOslo_Norway/Notebook">Page               </A></LI><LI class="new"><A href="/wiki/index.php?title=Talk:Team:UiOslo_Norway/Notebook&amp;action=edit&amp;redlink=1">Discussion               </A></LI><LI><A href="/wiki/index.php?title=Team:UiOslo_Norway/Notebook&amp;action=edit">View source               </A></LI><LI><A href="/wiki/index.php?title=Team:UiOslo_Norway/Notebook&amp;action=history">History               </A></LI><LI style="color:white;cursor:default">teams</LI></UL></DIV><DIV class="right-menu noprint" id="menubar"><UL><LI id="pt-login"><A href="/wiki/index.php?title=Special:UserLogin&amp;returnto=Team:UiOslo_Norway/Notebook" title="You are encouraged to log in; however, it is not mandatory [o]" accesskey="o">Log in</A></LI></UL></DIV><DIV id="search-controls" class="noprint"><FORM action="/Special:Search" id="searchform"> </FORM></DIV></DIV><DIV id="content"><H1 class="firstHeading">Team:UiOslo Norway/Notebook</H1><DIV id="bodyContent"><H3 id="siteSub" class="noprint">From 2014.igem.org</H3><P><TITLE>UiOslo IGEM 2014</TITLE></P><DIV class="navbar navbar-default navbar-static-top"><DIV class="navbar-inner"><DIV class="container"><DIV class="collapse navbar-collapse navHeaderCollapse"><UL class="nav navbar-nav navbar-right"><LI><A href="/Team:UiOslo_Norway/">Home</A></LI><LI class="dropdown"><A href="/Team:UiOslo_Norway/Project" class="dropdown-toggle" data-toggle="dropdown">
							Project </A></LI><LI><A href="/Team:UiOslo_Norway/Project">Project Details</A></LI><LI><A href="/Team:UiOslo_Norway/Project/Plan">Project Plan</A></LI><LI><A href="/Team:UiOslo_Norway/Project/Prospect">Prospect for the Future</A></LI></UL><LI class="dropdown"><A href="#" class="dropdown-toggle" data-toggle="dropdown">
							Team </A></LI><LI><A href="/Team:UiOslo_Norway/Team">Our Team</A></LI><LI><A href="/Team:UiOslo_Norway/Album">Album</A></LI><LI><A href="/Team:UiOslo_Norway/Acknowledgements">Acknowlegdements</A></LI><LI class="dropdown"><A href="/Team:UiOslo_Norway/Achievements" class="dropdown-toggle" data-toggle="dropdown">
							Achievements </A></LI><LI><A href="/Team:UiOslo_Norway/Achievements">Achievements</A></LI><LI><A href="/Team:UiOslo_Norway/Achievements/Parts">Parts</A></LI><LI class="dropdown"><A href="/Team:UiOslo_Norway/Notebook" class="dropdown-toggle" data-toggle="dropdown">
							Notebook </A></LI><LI><A href="/Team:UiOslo_Norway/Notebook">Notebook</A></LI><LI><A href="/Team:UiOslo_Norway/Notebook/Protocols">Protocols</A></LI><LI><A href="/Team:UiOslo_Norway/Notebook/PartsUsed">Parts Used</A></LI><LI><A href="/Team:UiOslo_Norway/Notebook/ProductsUsed">Products Used</A></LI><LI class="dropdown"><A href="/Team:UiOslo_Norway/HumanPractice" class="dropdown-toggle" data-toggle="dropdown">Human Practice </A></LI><LI><A href="/Team:UiOslo_Norway/HumanPractice">Human Practice</A></LI><LI><A href="/Team:UiOslo_Norway/HumanPractice/Media">Media</A></LI><LI><A href="/Team:UiOslo_Norway/HumanPractice/MediaplanAndTroubleshooting">Mediaplan and Troubleshooting</A></LI><LI><A href="/Team:UiOslo_Norway/HumanPractice/Events">Events</A></LI><LI><A href="/Team:UiOslo_Norway/HumanPractice/Collaboration">Collaboration</A></LI><LI><A href="/Team:UiOslo_Norway/HumanPractice/Comic">Adventures in Synthetic Biology</A></LI><LI class="dropdown"><A href="/Team:UiOslo_Norway/Safety" class="dropdown-toggle" data-toggle="dropdown">Safety </A></LI><LI><A href="/Team:UiOslo_Norway/Safety">Safety</A></LI><LI><A href="/Team:UiOslo_Norway/Safety/Ethics">Ethics and Concerns</A></LI></DIV></DIV></DIV></DIV><DIV class="container"><DIV class="jumbotron text-center"><H1 style="font-family:Syncopate; color: blue;">Notebook</H1><P>Welcome to our notebook! Here you can see what we've been up to in the lab each week. Click below to navigate between weeks.</P></DIV><DIV class="row text-center"><DIV class="col-sm-2"><DIV>
			June
		</DIV><DIV><UL class="nav nav-pills"><LI><A href="#" class="weekLink w22-link">Week 22</A></LI><LI><A href="#" class="weekLink w23-link">Week 23</A></LI><LI><A href="#" class="weekLink w24-link">Week 24</A></LI><LI><A href="#" class="weekLink w25-link">Week 25</A></LI><LI><A href="#" class="weekLink w26-link">Week 26</A></LI></UL></DIV></DIV><DIV class="col-sm-2"><DIV>
			July
		</DIV><DIV><UL class="nav nav-pills"><LI><A href="#" class="weekLink w27-link">Week 27</A></LI><LI><A href="#" class="weekLink w28-link">Week 28</A></LI><LI><A href="#" class="weekLink w29-link">Week 29</A></LI><LI><A href="#" class="weekLink w30-link">Week 30</A></LI><LI><A href="#" class="weekLink w31-link">Week 31</A></LI></UL></DIV></DIV><DIV class="col-sm-2"><DIV>
			August
		</DIV><DIV><UL class="nav nav-pills"><LI><A href="#" class="weekLink w31-link">Week 31</A></LI><LI><A href="#" class="weekLink w32-link">Week 32</A></LI><LI><A href="#" class="weekLink w33-link">Week 33</A></LI><LI><A href="#" class="weekLink w34-link">Week 34</A></LI><LI><A href="#" class="weekLink w35-link">Week 35</A></LI></UL></DIV></DIV><DIV class="col-sm-2"><DIV>
			September
		</DIV><DIV><UL class="nav nav-pills"><LI><A href="#" class="weekLink w36-link">Week 36</A></LI><LI><A href="#" class="weekLink w37-link">Week 37</A></LI><LI><A href="#" class="weekLink w38-link">Week 38</A></LI><LI><A href="#" class="weekLink w39-link">Week 39</A></LI><LI><A href="#" class="weekLink w40-link">Week 40</A></LI></UL></DIV></DIV><DIV class="col-sm-2"><DIV>
			October
		</DIV><DIV><UL class="nav nav-pills"><LI><A href="#" class="weekLink w40-link">Week 40</A></LI><LI><A href="#" class="weekLink w41-link">Week 41</A></LI></UL></DIV></DIV></DIV><DIV class="weekDiv" id="w22div"><DIV class="row"><H1>Week 22</H1></DIV><DIV class="row"><H2>Sunday</H2></DIV><DIV class="row"><DIV class="col-md-12 borderTopRow person-box"><H4>Håkon</H4><P>Designing primers and indentifying parts to use in iGEM.</P></DIV></DIV></DIV><DIV class="weekDiv" id="w23div"><DIV class="row"><H1>Week 23</H1></DIV><DIV class="row"><H2>Monday</H2></DIV><DIV class="row"><DIV class="col-md-12 borderTopRow person-box"><H4>Håkon</H4><P>Checking primers and adjusting sequences to the right melting temperatur.</P></DIV></DIV></DIV><DIV class="weekDiv" id="w24div"><DIV class="row"><H1>Week 24</H1></DIV><DIV class="row"><H2>Tuesday</H2></DIV><DIV class="row"><DIV class="col-md-12 borderTopRow person-box"><H4>Elina, Håkon</H4><P>Get to know the lab. Made gel loading buffer and TBE-buffers - 10X-stock and 1X-user.</P></DIV></DIV><DIV class="row"><H2>Wednesday</H2></DIV><DIV class="row"><DIV class="col-md-12 borderTopRow person-box"><H4>Elina, Håkon</H4><P>Growed two different E. coli strains (BL21 and AB2848) of bacteria on agar. Preparing isolation of Lac Z beta from the bacteria.</P></DIV></DIV><DIV class="row"><H2>Thursday</H2></DIV><DIV class="row"><DIV class="col-md-12 borderTopRow person-box"><H4>William, Håkon</H4><P>PCR on BL21 and AB2848 strain on bacteria to retrieve LacZ-beta gene. Used LacZ-beta STOP primer.</P><P>Agarose gel electrophoresis result: Band about 3 kb for the BL21 strain. Probably LacZ-beta.</P></DIV></DIV><DIV class="row"><H2>Friday</H2></DIV><DIV class="row"><DIV class="col-md-12 borderTopRow person-box"><H4>William, Håkon</H4><P>PCR clean-up using Machery-Nagel PCR clean-up kit. Followed protocol from the kit.</P><P>PCR amplification of the cleaned DNA with and without stop codon.</P></DIV></DIV></DIV><DIV class="weekDiv" id="w25div"><DIV class="row"><H1>Week 25</H1></DIV><DIV class="row"><H2>Tuesday</H2></DIV><DIV class="row"><DIV class="col-md-12 borderTopRow person-box"><H4>William</H4><P>PCR clean-up using Machery-Nagel PCR clean-up kit on DNA with and without stop codon.</P><P>We did step 1 of making E. coli TOP10 cells competent (incubation of cells from freeze stock in LB medium with streptomycin.</P><P>We analyzed the DNA concentration using NanoDrop:</P><UL><LI>LacZ-beta with stop codon: 230 ng/µl</LI><LI>LacZ-beta without stop codon: 300 ng/µl</LI></UL></DIV></DIV><DIV class="row"><H2>Wednesday</H2></DIV><DIV class="row"><DIV class="col-md-12 borderTopRow person-box"><H4>William, Håkon, Sumaya</H4><P>Finished Jack's protocol for making competent cells.</P><P>Attempted to transform bacteria with these parts:</P><UL><LI>LacZ-alpha 22D (plate 2)</LI><LI>10X His-tag 5P (plate 3)</LI><LI>pBad strong promoter 14A (plate 3)</LI><LI>T1 terminator 1D (plate 1)</LI><LI>Double terminator 3D (plate 1)</LI></UL><P>Parts were resuspended in accordance to iGEM protocol and for the transformation we used Jack's protocol.</P><P>Plates for E. coli growth were made and the transformation products were plated.</P></DIV></DIV><DIV class="row"><H2>Thursday</H2></DIV><DIV class="row"><DIV class="col-md-12 borderTopRow person-box"><H4>William, Håkon, Sumaya</H4><P>Growth of 10X his-tag and double terminator transformed bacteria observed. Overnight (O/N) cultures were made of these.</P><P>LacZ-alpha, pBad and T1 were attempted transformed again.</P><P>New batch of LB-plates were made with chloramphenicol.</P></DIV></DIV><DIV class="row"><H2>Friday</H2></DIV><DIV class="row"><DIV class="col-md-12 borderTopRow person-box"><H4>William, Håkon, Sumaya</H4><P>Restriction cutting of LacZ-beta STOP cleaned PCR product, LacZ-beta NONSTOP cleaned PCR-product and pSB1C3 were attempted using restriction enzymes XbaI and PstI. Incubation in accordance with iGEM restriction digest protocol. The tube with pSB1C3 was not mixed with water before pipetting.</P><P>The concentration of the digested products were measured in NanoDrop:</P><UL><LI>LacZ-beta stop: 14.4 ng/µl</LI><LI>LacZ-beta non stop: 23.3 ng/µl</LI><LI>pSB1C3: 26.5 ng/µl</LI></UL><P>Ligation was attempted on the digestions using T4 DNA ligase and following the iGEM ligation protocol.</P><P>Freeze stocks of the O/N cultures of 10X His-tag and double terminator were made.</P><P>Minipreps using Promega Wizard Plus SV miniprep kit was attempted on saturated 10X his-tag and double terminator cultures.</P><P>Cultures observed on pBad, LacZ-alpha and T1-terminator plates. O/N cultures made during the weekend.</P></DIV></DIV></DIV><DIV class="weekDiv" id="w26div"><DIV class="row"><H1>Week 26</H1></DIV><DIV class="row"><H2>Monday</H2></DIV><DIV class="row"><DIV class="col-md-12 borderTopRow person-box"><H4>Sumaya</H4><P>The pBad, LacZ-alpha and T1-terminator O/N cultures were miniprepped.</P><P>Agarose gel electrophoresis showed bands in all lanes (picture 12).</P><P>Freeze stocks made of pBad, LacZ-alpha and T1-terminator transformed bacteria.</P><P>The ligated products from the 20th were used in a transformation attempt.</P></DIV></DIV><DIV class="row"><H2>Tuesday</H2></DIV><DIV class="row"><DIV class="col-md-12 borderTopRow person-box"><H4>William, Sumaya</H4><P>No colonies seen on the LacZ-beta stop and non stop plates.</P><P>Preformed new digestion of parts using pSB1C3 solved in 30 μl dH2O and XbaI and PstI.</P></DIV></DIV><DIV class="row"><H2>Wednesday</H2></DIV><DIV class="row"><DIV class="col-md-12 borderTopRow person-box"><H4>Sumaya</H4><P>The digestions were cleaned up using the Machery-Nagel PCR clean-up kit.</P><P>Ligation was done following the iGEM ligation protocol, but running two parallells – one with and one without heat shock.</P></DIV></DIV><DIV class="row"><H2>Thursday</H2></DIV><DIV class="row"><DIV class="col-md-12 borderTopRow person-box"><H4>William, Håkon</H4><P>Made O/N culture of T1 terminator from freeze stock in order to obtain pSB1C3 plasmids in case the pSB1C3 from the tube in the kit is defective.</P><P>We tested the restriction digest protocol by using a part from the kit we don't need (1A). Due to issues with the agarose gel we started over using another part (1K) that we left it in 37C for 4 hours and then on 4C overnight.</P></DIV></DIV><DIV class="row"><H2>Friday</H2></DIV><DIV class="row"><DIV class="col-md-12 borderTopRow person-box"><H4>William, Sumaya</H4><P>Miniprep of the T1 terminator O/N culture. 3 MP's made in parallel with NanoDrop DNA concentrations:</P><UL><LI>T1-1: 15.6 ng/µl</LI><LI>T1-2: 16.7 ng/µl</LI><LI>T1-3: 16.2 ng/µl</LI></UL><P>The T1-1 was digested using XbaI and PstI following the iGEM digestion protocol.</P><P>The product was run on a gel and we cut out the pSB1C3 part using a scalpel and stored it in a eppendorf tube in the fridge.</P></DIV></DIV><DIV class="row"><H2>Saturday</H2></DIV><DIV class="row"><DIV class="col-md-12 borderTopRow person-box"><H4>Håkon</H4><P>Made freeze stock of Lac-Z beta stop transformed Top 10 cells.</P><P>PCR was preformed on LacZ-beta with stop and non stop reverse primers using Q5 polymerase.</P><P>Miniprep of Lac Z beta stop.</P><P>Restriction cutting of Lac-Z beta stop with EcoRI. Cut and uncut samples were teste don gel with negative results for both.</P></DIV></DIV></DIV><DIV class="weekDiv" id="w27div"><DIV class="row"><H1>Week 27</H1></DIV><DIV class="row"><H2>Monday</H2></DIV><DIV class="row"><DIV class="col-md-12 borderTopRow person-box"><H4>William</H4><P>We used the NucleoSpin gel clean-up kit and protocol in order to isolate the XbaI/PstI-cut pSB1C3 from the gel piece. Concentration measured with NanoDrop: 10-15 ng/µl.</P><P>PCR was done on LacZ-beta with stop and non stop reverse primers  using Q5 polymerase.</P></DIV></DIV><DIV class="row"><H2>Tuesday</H2></DIV><DIV class="row"><DIV class="col-md-12 borderTopRow person-box"><H4>Håkon</H4><P>Testing Lac Z beta stop on gel, and transformation of Lac Z beta stop into competent cells.</P></DIV></DIV><DIV class="row"><DIV class="col-md-12 borderTopRow person-box"><H4>Sumaya</H4><P>Transformation of E.coli with LacZ-beta stop (without EcoRI site) was attempted and transformation of flourecent protiens was attempted.</P><P>These flourecent protiens were found in the iGEM kit: </P><UL><LI>cjBlue, green chromoprotein placed 2I and 2K (will be referred to these code from now on) from Kit Plate 4.</LI><LI>amilCP, blue chromoprotein  placed 19E (will also be referred to this code) from Kit Plate 1</LI><LI>**highly** engineered mutant of red fluorescent protein from Discosoma striata (coral) placed 11N (will also be referred to this code) from Kit Plate 3 </LI><LI>green fluorescent protein derived from jellyfish Aequeora victoria wild-type GFP placed 13L (will also be referred to this code) from Kit Plate 4</LI></UL><P>Digestion of Lac Z beta-stop with only the EcoR1 enzyme was also attempted.</P></DIV></DIV><DIV class="row"><H2>Wednesday</H2></DIV><DIV class="row"><DIV class="col-md-12 borderTopRow person-box"><H4>Vilde</H4><P>Made N-Histag from primers. Used the program called “gradient” in the PCR machine. Started at 95 deegree and went slovly down to 55 degrees. Found out that this was the wrong program to use.</P></DIV></DIV><DIV class="row"><DIV class="col-md-12 borderTopRow person-box"><H4>Håkon</H4><P>Blunt end ligation of pure linear pSB1C3 to control the ligation reaction.</P></DIV></DIV><DIV class="row"><DIV class="col-md-12 borderTopRow person-box"><H4>Sumaya</H4><P>Ran PCR using the TaKaRA PCR setup and ran agorose gel to check if the transformations had work.</P><P>These were tested:</P><UL><LI>His-tag (His)</LI><LI>Double terminator (DT)</LI><LI>pBAD promotor (pBad)</LI><LI>Lac Z alpha (α)</LI><LI>Terminator 1 (T1)</LI><LI>2I</LI><LI>2K</LI><LI>19E</LI><LI>11N</LI></UL><P>All of these had bands.</P></DIV></DIV><DIV class="row"><H2>Thursday</H2></DIV><DIV class="row"><DIV class="col-md-12 borderTopRow person-box"><H4>William, Håkon</H4><P>O/N cultures were made of the colonies of transformed pSB1C3/LacZ-beta non-stop.</P><P>PCR was run with VR2 and VF primers for confirmation.</P></DIV></DIV><DIV class="row"><DIV class="col-md-12 borderTopRow person-box"><H4>Vilde</H4><P>Made new N-term His-tag primers. Used Waterbath this time. Put the waterbath at 95 degrees, mixed the Forward and reverse oligoes for the N-his term and put them in the waterbath. Let the temperature slowly go down to about 50 degrees.</P></DIV></DIV><DIV class="row"><DIV class="col-md-12 borderTopRow person-box"><H4>Vilde</H4><P>Made a big O/N culture of the pSB1C3 C-term His to isolate the plasmid backbone tomorrow. Made a 50ml O/N culture with chloramphenicole. 50ul Chloranphenicole in 50ml LB medium.</P></DIV></DIV><DIV class="row"><DIV class="col-md-12 borderTopRow person-box"><H4>Izadora</H4><P>A PCR on BL21 and AB2848 strains of bacteria was repeated to retrieve LacZ-beta gene. The LacZ-beta Non-Stop and LacZ-alpha Non-Stop primers were used.</P><P>The fragment-size in the agarose gel (0.7%) was like expected.</P><P>A miniprep was performed on O/N cultures with colorful parts from the Kit (19E, 11N, 2I and 2K) using Promega Wizard Plus SV miniprep kit.</P></DIV></DIV><DIV class="row"><H2>Friday</H2></DIV><DIV class="row"><DIV class="col-md-12 borderTopRow person-box"><H4>William, Håkon</H4><P>3A assembly was attempted with the bricks pBad and 2I, 2K, 19E and 11N, using pSB1A3.</P></DIV></DIV><DIV class="row"><DIV class="col-md-12 borderTopRow person-box"><H4>Vilde</H4><P>Miniprepped lacZ-beta Non-stop pSB1c3 (Wizard Miniprep). Final Concentration of 95 ng/ul.</P></DIV></DIV><DIV class="row"><DIV class="col-md-12 borderTopRow person-box"><H4>Vilde</H4><P>Quickchange mutagenesis of LacZ beta non stop. One of the parts we are making contains an EcoRI site. We will do a mutagenesis to get read of this site.  Used 7,5ul primers and 6 min elongation. Followed the quickchange protocol and used  pfu Turbo enzyme from agilent technologies.</P></DIV></DIV><DIV class="row"><DIV class="col-md-12 borderTopRow person-box"><H4>Vilde</H4><P>A3 assembly of lacZ Alpha Stop and LacZ beta NS into pBAd.</P></DIV></DIV><DIV class="row"><H2>Saturday</H2></DIV><DIV class="row"><DIV class="col-md-12 borderTopRow person-box"><H4>Vilde</H4><P>No colonies after the mutagenesis or A3 assembly. Troubleshooted it.</P></DIV></DIV></DIV><DIV class="weekDiv" id="w28div"><DIV class="row"><H1>Week 28</H1></DIV><DIV class="row"><H2>Monday</H2></DIV><DIV class="row"><DIV class="col-md-12 borderTopRow person-box"><H4>William, Håkon</H4><P>Batch of LB agar plates w/Ampicillin was made.</P></DIV></DIV><DIV class="row"><DIV class="col-md-12 borderTopRow person-box"><H4>Vilde</H4><P>Made primers for Autotransporters.</P></DIV></DIV><DIV class="row"><DIV class="col-md-12 borderTopRow person-box"><H4>Vilde</H4><P>Ordered primers for Autotransporters, Signaling peptide, Primers to amplify linearized plasmid backbone and VR, VF2 primers.</P><P>The primes contains the standard prefix and suffix. Non of these genes contains any of the restriction sites for the restriction enzymes in the iGEM assembly protocol.</P><UL><LI>Invefull Long</LI><LI>Invfull Short</LI><LI>Ehaj1 – Short</LI><LI>Ehaj2 – long</LI><LI>Signalling peptide</LI></UL></DIV></DIV><DIV class="row"><DIV class="col-md-12 borderTopRow person-box"><H4>Vilde</H4><P>Assembled lacZ beta Stop and NS (Cleaned up PCR products) into pSB1C3 plasmid.</P><P>Cut both plasmid and parts with EcoRI and PstI. Ligated for 1h at RT. Transformed 2 ul.</P></DIV></DIV><DIV class="row"><DIV class="col-md-12 borderTopRow person-box"><H4>Sumaya</H4><P>Made 2 overnight cultures (O/N) from two different colonies of pBad-beta non-stop, His 1- α-stop, His 1- α-stop, pBad-2K, pBad-2I, pBad-19E, and pBad-11N</P><P>Also made the agrose gel for the gel electrophorese.</P></DIV></DIV><DIV class="row"><DIV class="col-md-12 borderTopRow person-box"><H4>Izadora</H4><P>A PCR on colonies from O/N cultures following the TaKaRa® enzyme protocol was performed, but increasing the extension time to 4:20min (when using iGEM’s primers VF and VR2).</P><P>Samples below:</P><UL><LI>pBAD - LacZ-beta Non Stop</LI><LI>HisTag 1 - LacZ-alpha Stop</LI><LI>HisTag 2 (in a different concentration) - LacZ-alpha Stop</LI><LI>pBAD - 2K</LI><LI>pBAD - 2I</LI><LI>pBAD - 19E</LI><LI>pBAD - 11N</LI></UL><P>An A3 assembly in the pSB1A3 plasmid following iGEM protocol was done, but the ligation taking place at 16°C O/N as shown below:</P><UL><LI>Part A: pBADPart B: LacZ-alpha Non Stop</LI></UL></DIV></DIV><DIV class="row"><H2>Tuesday</H2></DIV><DIV class="row"><DIV class="col-md-12 borderTopRow person-box"><H4>William, Håkon</H4><P>The results of the 3A assembly was poor, with either no growth at all or carpet growth.</P><P>The O/N cultures from the day before was made without using antibiotics, so the cultures was transferred to fresh medium with antibiotics to try to select for the right bacteria.</P><P>2I, 11N, 2K and 19E was plated on LB agar with 0.1% arabinose.</P><P>New batch of LB agar was made, using LB broth powder.</P></DIV></DIV><DIV class="row"><DIV class="col-md-12 borderTopRow person-box"><H4>Vilde</H4><P>No growth observed from the assembly into pSB1C3. We think there may be something wrong with the linearized plasmid. We will try to amplify the plasmid by PCR from another part and use that one in the assembly work.</P></DIV></DIV><DIV class="row"><DIV class="col-md-12 borderTopRow person-box"><H4>Vilde</H4><UL><LI>Found out that the iGEM scar contains a Stop codon</LI><LI>Found tRNA amber Stop – part in the kit</LI><LI>Found a constitutive promoter</LI><LI>Found plasmid (from jack) with another origin of replication</LI></UL></DIV></DIV><DIV class="row"><DIV class="col-md-12 borderTopRow person-box"><H4>Izadora</H4><P>An analytical gel electrophoresis was performed in a 0.7% agarose gel with samples from last PCR and O/N ligated products from A3 assembly. The fragment-sizes in the gel were like expected just for BAD -11N and HisTag 1 - LacZ-alpha Stop.</P><P>The gel electrophoresis was repeated with increased volume of sample (20μL) and all fragments presented the expected size.</P><P>A Transformation with the O/N ligated product from A3 assembly was attempted using the agar plates with the correct antibiotic at the right working concentration. </P></DIV></DIV><DIV class="row"><H2>Wednesday</H2></DIV><DIV class="row"><DIV class="col-md-12 borderTopRow person-box"><H4>Vilde</H4><UL><LI>Transformed Amber stop – tRNA and promoter</LI><LI>Amber stop tRNA - Part: K228001, 14E kit plate 3</LI><LI>constitutive promoter – Part J23119, 17O kit plate 3 </LI></UL></DIV></DIV><DIV class="row"><DIV class="col-md-12 borderTopRow person-box"><H4>Sumaya</H4><P>Miniprepped pBad Lac Z beta non-stop as the others had failed.</P><P>Attempted a mutagenisis of the pBad-Lac Z beta non-stop.</P></DIV></DIV><DIV class="row"><DIV class="col-md-12 borderTopRow person-box"><H4>Izadora</H4><P>There was no growing in the transformation.</P></DIV></DIV><DIV class="row"><H2>Thursday</H2></DIV><DIV class="row"><DIV class="col-md-12 borderTopRow person-box"><H4>Vilde</H4><P>Got colonies from transformation of the amber stop tRNA and promoter. Tested the colonies on PCR with the VF and VR2 primers. Made O/N cultures in LB chloramphenicole medium.</P></DIV></DIV><DIV class="row"><DIV class="col-md-12 borderTopRow person-box"><H4>Izadora</H4><P>The transformation worked for samples 17O, 14E, 13L, but not in pBAD - LacZ-alpha Non Stop.</P><P>A PCR clean-up of PCR products of LacZ-beta and -alpha Non Stop from both AB2848 and BL21 strains was done, following Machery-Nagel PCR clean-up kit’s protocol.</P><P>The Concentrations were measured in NanoDrop (NanoDrop 1000 3.8.1) for each cleaned sample:</P><UL><LI>LacZ-beta Non Stop (AB2848): 109.8 ng/µl</LI><LI>LacZ-beta Non Stop (BL21): 36.4 ng/µl</LI><LI>LacZ-alpha Non Stop (AB2848): 74.8 ng/µl</LI><LI>LacZ-alpha Non Stop (BL21): 91.9 ng/µl</LI></UL><P>After that, an A3 assembly (as in 07.07.14) was attempted with PCR cleaned-digested products, digested pBAD and pSB1A3 following iGEM’s protocol with O/N ligation.</P></DIV></DIV><DIV class="row"><H2>Friday</H2></DIV><DIV class="row"><DIV class="col-md-12 borderTopRow person-box"><H4>William, Sumaya, Izadora</H4><P>O/N cultures was miniprepped and freeze stocks were made of:</P><UL><LI>pBad + 19E in pSB1A3</LI><LI>pBad + 2K in pSB1A3</LI><LI>pBad + 2I in pSB1A3</LI><LI>pBad + 11N in pSB1a3</LI><LI>13L in pSB1A2</LI><LI>His alpha stop in psB1A3</LI><LI>19E in pSB1C3</LI></UL><P>All in TOP10 E.coli cells.</P><P>New transformation was set up using the following ligation products:</P><UL><LI>LacZ alpha NS (from AB strain) + pBAD + pSB1A3</LI><LI>LacZ alpha NS (from BL strain) + pBAD + pSB1A3</LI><LI>LacZ beta NS (from AB strain) + pBAD + pSB1A3</LI><LI>LacZ alpha NS (from BL strain) + pBAD + pSB1A3</LI></UL><P>The NCM17 E.coli strain was received from Yale E.coli Repository, plated and overnight cultured.</P></DIV></DIV><DIV class="row"><DIV class="col-md-12 borderTopRow person-box"><H4>Vilde</H4><P>Miniprepped and made freeze stock of Amber tRNA and constitutive promoter.</P></DIV></DIV><DIV class="row"><DIV class="col-md-12 borderTopRow person-box"><H4>Izadora</H4><P>A frozen stock was prepared with O/N cultures and Glycerol 60% (1/1).</P><P>There were no colonies from transformation with pBAD-LacZ-beta Non Stop (Mutated with EcoR1).</P><P> A transformation with O/N ligated products and with part 17O (constitutive promoter) was attempted with 2μL of ligated products and 30μL of TOP 10 competent cells.</P><P>50mL of SOC medium was prepared with: 10mM of MgSO4 and 2% of Glucose. The volume was completed to 50mL with LB medium and the final medium was sterilized with syringe filter (0,2μm).</P><P>The SOC medium was tested for contamination in test tubes by O/N incubation at 37°C and 180rpm.</P><P>The strain NCM17 without LacZ gene has arrived and was plated out in agar-kanamycin (Adgene®, working concentration = 50μg/mL) and test tubes without antibiotics by O/N incubation at 37°C and 180rpm (in the case of test tube only).</P></DIV></DIV></DIV><DIV class="weekDiv" id="w29div"><DIV class="row"><H1>Week 29</H1></DIV><DIV class="row"><H2>Monday</H2></DIV><DIV class="row"><DIV class="col-md-12 borderTopRow person-box"><H4>William</H4><P>Freeze stock was made of NCM17 strain.</P></DIV></DIV><DIV class="row"><DIV class="col-md-12 borderTopRow person-box"><H4>Vilde</H4><P>Tested the function of the pfu Turbo enzyme. The enzyme was forgotten at RT for 48h. I therefore tested if it was still functional by running a pcr on genomic DNA with Actin primers. I used the pfuTurbo protocol 600410. Results: No bands. The PCR was repeated with different primers and substrate but still showed no results. We concluded with that the enzyme is not functional anymore.</P></DIV></DIV><DIV class="row"><DIV class="col-md-12 borderTopRow person-box"><H4>Izadora</H4><P>One colony of NCM17 strain was picked up and incubated in a test tube by O/N incubation following the same protocol. The plate was stored at 4°C.</P><P>There were no colonies from transformation of 11.07, but for part 17O.</P><P>For sample 17O, O/N culture with chloramphenicol (Adgene®, working concentration = 25μl/mL) was prepared as before.</P></DIV></DIV><DIV class="row"><H2>Tuesday</H2></DIV><DIV class="row"><DIV class="col-md-12 borderTopRow person-box"><H4>William</H4><P>O/N culture of bacteria transformed with 17O was miniprepped.</P></DIV></DIV><DIV class="row"><DIV class="col-md-12 borderTopRow person-box"><H4>Vilde</H4><P>Cleanup-day in the lab! We tested all the minipreps we got in the freezer. Used Takara enzyme and general reaction mix. Tested them all on PCR and run a gel afterwords. We found out that all our plasmids contain the gene we expected.</P></DIV></DIV><DIV class="row"><DIV class="col-md-12 borderTopRow person-box"><H4>Izadora</H4><P>The gel electrophoresis was performed in agarose gel (o.7%) with digested and ligated parts.</P><P>The fragment-sizes were like as expected just for ladder and LacZ-alpha and -beta Non Stop digested products, but not for pSB1A3 and pSB1C3.</P><P>A PCR test was performed in all miniprep that we have so far using TaKaRa® enzyme and following its protocol with VF/VR2 program.</P><P>Using the Promega Wizard Plus SV miniprep kit, a miniprep was performed in O/N cultures with part 17O, storing it at -20°C. Frozen stocks of cells were prepared for O/N cultures with 60% Glycerol (1/1) and stored at -80°C.</P></DIV></DIV><DIV class="row"><H2>Wednesday</H2></DIV><DIV class="row"><DIV class="col-md-12 borderTopRow person-box"><H4>William</H4><P>Made 20 LB plates with kanamycin.</P><P>The miniprep of LacZ beta NS in pSB1C3 was run on a gel. The bond was the right size.</P></DIV></DIV><DIV class="row"><DIV class="col-md-12 borderTopRow person-box"><H4>Vilde, Izadora</H4><P>A3 assembly of constitutive promoter and amber stop tRNA. Followed protocol from iGEM kit. With some small changes – used only ¼ of the reaction volume.</P><UL><LI>Part A: Constitutive promoter</LI><LI>Part B: amber stop tRNA.</LI><LI>Linearized plasmid: pSBA1A3</LI></UL><P>Since the RNA made from the amber stop tRNA gene will not be translated into proteins, a RBS in front of the gene is not necessary. I therefore  ligated the promoter directly to the gene.</P><UL><LI>Digestion buffer: NEB 3.1</LI><LI>ligation time: 1h at RT.</LI><LI>Transformed 3 ul of the ligated product.</LI></UL></DIV></DIV><DIV class="row"><DIV class="col-md-12 borderTopRow person-box"><H4>Elina</H4><P>N-His-Tag to pSB1C3</P></DIV></DIV><DIV class="row"><H2>Thursday</H2></DIV><DIV class="row"><DIV class="col-md-12 borderTopRow person-box"><H4>William</H4><P>The NCM17 strain was made competent (Using Jack Leo’s protocol) and put in freezer.</P></DIV></DIV><DIV class="row"><DIV class="col-md-12 borderTopRow person-box"><H4>Vilde</H4><P>No growth observed after the A3 assembly. Troubelshooting.</P></DIV></DIV><DIV class="row"><DIV class="col-md-12 borderTopRow person-box"><H4>Vilde</H4><P>New Assembly of amber stop tRNA and the constitutive promoter. Tried the assembly again, but this time with a larger reaction volume, 1/2 of the original volume. Also used the cutsmart buffer and not the NEB 3.1 buffer. I also changed and used the pSB1K3 plasmid instead of the pSB1A3 plasmid.</P></DIV></DIV><DIV class="row"><DIV class="col-md-12 borderTopRow person-box"><H4>Vilde</H4><P>Received primers for autotransporters and signaling peptide. Made stock solution and user stocks (5uM) of the primers.</P></DIV></DIV><DIV class="row"><DIV class="col-md-12 borderTopRow person-box"><H4>Izadora</H4><P>A PCR on 4 different colonies from transformation with pSB1K3 was done with TaKaRa® enzyme, following its protocol.</P><P>No fragments were observed in the agarose gel 0.7% after gel electrophoresis, but the ladder.</P><P>100mLof LB medium with kanamycin was prepared (Adgene®, working concentration = 50μg/μL) and sterilized with a syringe filter (0,2μm).</P><P>6L of LB medium and 6L of LB/agar 1% were prepared following the manufacturer’s protocol.</P><P>The 4 colonies were incubated in test tubes by O/N incubation as before.</P></DIV></DIV><DIV class="row"><H2>Friday</H2></DIV><DIV class="row"><DIV class="col-md-12 borderTopRow person-box"><H4>Vilde</H4><P>Growth from the A3 assembly of amber stop tRNA and constitutive promoter. Checked colonies on PCR with VR and VF2 primers and made O/N cultures of positive colonies.</P></DIV></DIV><DIV class="row"><DIV class="col-md-12 borderTopRow person-box"><H4>Vilde, Sumaya</H4><P>Minipreped and made freeze stock of Izadoras O/N cultures of LacZalpha in pSB1K3. Used wizard miniprep protocol.</P></DIV></DIV></DIV><DIV class="weekDiv" id="w30div"><DIV class="row"><H1>Week 30</H1></DIV><DIV class="row"><H2>Monday</H2></DIV><DIV class="row"><DIV class="col-md-12 borderTopRow person-box"><H4>Vilde, Sumaya</H4><P>A3 assembly of His-Tags into LacZ beta. Used A3 assembly protocol from iGEM.</P><OL><LI>Part A:  N-term HisPart B: B-Stop</LI><LI>Part A: Beta-Non StopPart B: C-term His</LI><LI>Part A: C-term hisPart B : T1 terminator</LI></OL><P>Found out that these lacZ beta parts still contains the EcoRI site. The samples will not be used. Assembly nr.3 (C-term-his to T1 terminator) will be used in future work.</P></DIV></DIV><DIV class="row"><DIV class="col-md-12 borderTopRow person-box"><H4>Vilde, Sumaya</H4><P>We found out that the promoters we have been using do not contain a RBS in front of it. We will therefore need to make new constructs with a RBS in between the promoter and the gene. I found and transformed these RBS from the kit:</P><UL><LI>RBS 4G kit plate 4 in psb1C3</LI><LI>RBS 1N Kit plate 4 in pSB1A3</LI><LI>RBS-LacZ alpha 22D kit plate 3 in pSB1C3</LI></UL></DIV></DIV><DIV class="row"><DIV class="col-md-12 borderTopRow person-box"><H4>Vilde, Sumaya</H4><P>Made O/N culture of the strain we will use to express the amber-stop tRNA. It contains CAM resistance. Place 69# in Dirks stock.</P></DIV></DIV><DIV class="row"><DIV class="col-md-12 borderTopRow person-box"><H4>Vilde, Sumaya</H4><P>Plated out bacteria from Dirks stock witch contains the plasmid that contains the autotransporters with the signaling peptide.</P><UL><LI>Invfull #289</LI><LI>Ehaj1 #261</LI><LI>Ehaj2 #262</LI></UL><P>All these plasmids got amp resistance. </P></DIV></DIV><DIV class="row"><H2>Tuesday</H2></DIV><DIV class="row"><DIV class="col-md-12 borderTopRow person-box"><H4>Elina</H4><P>Put N-His-Tag into pSB1C3</P></DIV></DIV><DIV class="row"><DIV class="col-md-12 borderTopRow person-box"><H4>Sumaya</H4><P>The O/N of 19P from the day before had grown and was miniprepped.</P></DIV></DIV><DIV class="row"><DIV class="col-md-12 borderTopRow person-box"><H4>Izadora</H4><P>An A3 assembly in both pSB1C3 and pSB1A3 with (CutSmart for digestion and 1hour of ligation), but the parts were cut with wrong enzymes by mistake.</P></DIV></DIV><DIV class="row"><H2>Wednesday</H2></DIV><DIV class="row"><DIV class="col-md-12 borderTopRow person-box"><H4>Vilde</H4><P>Made constructs of Autotransporters and signaling peptide. Used the Q5 protocol and did PCR on bacteria colonies I grew up 21.07. Tested the PCR results on a gel. Got bands for every constructs at the correct length. I made: Ehaj1, Ehaj2, Inventful short, Inventful long, Signalling peptide.</P></DIV></DIV><DIV class="row"><DIV class="col-md-12 borderTopRow person-box"><H4>Vilde</H4><P>3A Assembly of autotransporters and signaling peptide.</P><P>Followed the protocol and the manual from the iGEM kit. Used pSB1K3 as linear plasmid.</P><OL><LI>Part A: Signaling peptidePart B: Inventful long</LI><LI>Part A: Signaling peptidePart B: Inventful short</LI><LI>Part A: Ehaj1 shortPart B: T1 Terminator</LI><LI>Part A: Ehaj2 longPart B: T1 terminator</LI></OL></DIV></DIV><DIV class="row"><DIV class="col-md-12 borderTopRow person-box"><H4>Elina</H4><P>Use PCR to make new plasmids.</P></DIV></DIV><DIV class="row"><DIV class="col-md-12 borderTopRow person-box"><H4>Izadora</H4><P>A miniprep was done using Promega Wizard Plus SV miniprep kit and frozen stocks of cells were prepared with the O/N cultures below:</P><UL><LI>InvFull (pSB1A3)</LI><LI>Ehaj (pSB1A3)</LI><LI>Ehaj1 (pSB1A3)</LI><LI>LacZ-beta Stop (pSB1K3)</LI><LI>RBS - 4E1 (pSB1C3)</LI><LI>CtermHisTag - T1 (pSB1K3)</LI><LI>LacZ-alpha - RBS (pSB1C3)</LI><LI>RBS - 4G (pSB1C3)</LI><LI>RBS - 1N (pSB1A3)</LI><LI>LacZ-beta Non Stop - CtermHisTag (pSB1K3)</LI><LI>NtermHisTag - LacZ-beta Stop (pSB1K3)</LI><LI>LacZ-beta Non Stop (pSB1K3)</LI></UL></DIV></DIV><DIV class="row"><DIV class="col-md-12 borderTopRow person-box"><H4>Izadora</H4><P>The O/N cultures with cells of samples 4 to 12 were prepared again.</P><P>The concentration of PCR products from plasmids were measured in NanoDrop (NanoDrop 1000 3.8.1):</P><UL><LI>pSB1A3: 1135.1ng/μL</LI><LI>pSB1C3: 717.6ng/μL</LI><LI>pSB1K3: 1072.3ng/μL</LI><LI>pSB1T3: 684.3ng/μL</LI></UL></DIV></DIV><DIV class="row"><H2>Wednesday</H2></DIV><DIV class="row"><DIV class="col-md-12 borderTopRow person-box"><H4>Izadora</H4><P>A miniprep was performed using Promega Wizard Plus SV miniprep kit with samples 4 to 12, but eluting the plasmid DNA in 50μL of Nuclease free water.</P><P>A PCR clean-up with NucleoSpin® Gel and PCR clean-up of plasmid PCR products was performed following the manufacturer’s protocol (Macherey-Nagel).</P><P>An A3 assembly was done as previous protocol, ligating and transforming as following:</P><UL><LI>
					pSB1A3Part A: pBADPart B: RBS 4G
				</LI><LI>
					pSB1A3Part A: 17OPart B: RBS 4G
				</LI><LI>
					pSB1A3Part A: LacZ-alpha Non StopPart B: CtermHisTag - T1
				</LI><LI>
					pSB1K3Part A: NtermHisTag - LacZ-alpha StopPart B: T1
				</LI><LI>
					pSB1K3Part B: LacZ-alpha Non Stop
				</LI></UL></DIV></DIV><DIV class="row"><H2>Thursday</H2></DIV><DIV class="row"><DIV class="col-md-12 borderTopRow person-box"><H4>Vilde</H4><P>Got Colonies from the 3A assembly of autotransporters. Made O/N cultures.</P></DIV></DIV><DIV class="row"><DIV class="col-md-12 borderTopRow person-box"><H4>Vilde</H4><P>Assembled parts into pSB1C3 – The shipping plasmid. Used linearized plasmids pSB1C3 from the kit. Cut both plasmid and parts with EcoRI and PstI. Digestion and ligation followed protocol. Ligated these constructs into pSb1C3: Ehaj1 short, Ehaj2 long, Invful long, Invful short, Signaling peptide.  Transformed into E.coli. No growth observed the next day.</P></DIV></DIV><DIV class="row"><DIV class="col-md-12 borderTopRow person-box"><H4>Vilde</H4><P>Run LacZ Beta Stop and Non Stop on gel to make sure they were in a linear plasmid. No results on the PCR.</P></DIV></DIV><DIV class="row"><DIV class="col-md-12 borderTopRow person-box"><H4>Vilde</H4><P>Tested lacZ beta colonies on PCR. Used TaKaRa setup and protocol. Tested LacZbeta Stop, lacZ beta non-stop, lacz beta with N and with C-term his.  Nothing was observed in the gel. Not even the ladder so we might ha forgot the DNA Dye.</P></DIV></DIV><DIV class="row"><DIV class="col-md-12 borderTopRow person-box"><H4>Sumaya</H4><P>Did a PCR cleanup first for the PCR products of T1, 11N, 19E, 2K and 2I.</P><P>Followed the A3 assembly protocol found on the iGEM site</P><OL><LI>Part A: 11Part B: T1</LI><LI>Part A:19EPart B: T1</LI><LI>Part A: 2KPart B: T1</LI><LI>Part A: 2I Part B: T1</LI></OL><P>These were digested, ligated into pSB1K3 and transformed. </P></DIV></DIV><DIV class="row"><H2>Friday</H2></DIV><DIV class="row"><DIV class="col-md-12 borderTopRow person-box"><H4>Sumaya</H4><P>Got carpet growth from yesterdays transformations. The tranformation was done again.</P><P>Prepared O/N of Vildes tranformation.</P></DIV></DIV><DIV class="row"><DIV class="col-md-12 borderTopRow person-box"><H4>Izadora</H4><P>Colonies were got for all transformations, so O/N cultures in test tubes (2 colonies of each) with the correct antibiotics were prepared following the same conditions as previously. They are shown below:</P><UL><LI>Signaling peptide - InvFull long</LI><LI>Signaling peptide - InvFull short</LI><LI>Ehaj 1 short - T1</LI><LI>Ehaj 2 long - T1</LI></UL><P>LB medium was prepared with kanamycin as previously.</P></DIV></DIV><DIV class="row"><H2>Saturday</H2></DIV><DIV class="row"><DIV class="col-md-12 borderTopRow person-box"><H4>Izadora</H4><P>Frozen stocks of cells were prepared and miniprep was attempted using Promega Wizard Plus SV miniprep kit with following O/N cultures:</P><UL><LI>LacZ-alpha Non Stop (pSB1K3)</LI><LI>17O - RBS (pSB1A3)</LI><LI>Ehaj 1 - T1 (pSB1K3)</LI><LI>Ehaj 2 - T1 (pSB1K3)</LI><LI>InvFull long (pSB1K3)</LI><LI>InvFull short (pSB1K3)</LI><LI>NtermHisTag  - LacZ-alpha Stop - T1 (pSB1K3)</LI></UL><P>Frozen stocks were prepared and stored at -80°C for:</P><UL><LI>2I - T1 (pSB1K3)</LI><LI>2K - T1 (pSB1K3)</LI><LI>19E - T1 (pSB1K3)</LI><LI>11N - T1(pSB1K3)</LI></UL></DIV></DIV><DIV class="row"><H2>Sunday</H2></DIV><DIV class="row"><DIV class="col-md-12 borderTopRow person-box"><H4>Vilde</H4><P>Transformed the part for galactosidase into e.coli</P></DIV></DIV><DIV class="row"><DIV class="col-md-12 borderTopRow person-box"><H4>Sumaya</H4><P>Izadora had made O/Ns of my tranformations. These O/Ns  were positive. Made a freezestock and miniprepped them using the Wizard Plus kit.</P></DIV></DIV></DIV><DIV class="weekDiv" id="w31div"><DIV class="row"><H1>Week 31</H1></DIV><DIV class="row"><H2>Monday</H2></DIV><DIV class="row"><DIV class="col-md-12 borderTopRow person-box"><H4>Vilde</H4><P>Mutagenesis of lacZ beta Stop and non-stop. Used QuickChange protocol and setup with pfuTurbo enzyme and dpnI treated the sample. Used LacZBeta Non-Stop-pSBiK3 and LacZbeta Stop- PSB1K3 as template. I Transformed 3ul into bacteria. No colonies was observed the day after.</P></DIV></DIV><DIV class="row"><DIV class="col-md-12 borderTopRow person-box"><H4>Vilde</H4><P>3A assembly. Used standard protocol and procedure. Ligated them all into the psb1T3 plasmid.</P><OL><LI>Part A: alpha NonStopPart B: Ehaj1 short – T1</LI><LI>Part A: Alpha NonStopPartB: Ehaj2 Long – T1</LI><LI>Part A: Beta NonStopPartB: Ehaj1 short – T1</LI><LI>Part A: Beta NonStopPartB: Ehaj2 long – T1</LI><LI>Part A: Signalling peptide (SP) - Invful shortPartB: Alpha Stop</LI><LI>Part A: SP- invful shortPart B: Beta Stop</LI><LI>Part A: Sp-invful longPart B: Alpha Stop</LI><LI>Part A: Sp-invful longPart B: Beta Stop</LI></OL></DIV></DIV><DIV class="row"><DIV class="col-md-12 borderTopRow person-box"><H4>Vilde</H4><P>Assembled the Amber stop tRNA into plasmid. I will assemble the promoter-AmberStop tRNA into the plasmid; pACYCDuet1. I cut both plasmid and part with EcorRI and PstI. The plasmid contains a multiple cloning site that will be used to set in our part. The plasmid contains chloramphenicol resistance.</P></DIV></DIV><DIV class="row"><DIV class="col-md-12 borderTopRow person-box"><H4>Izadora</H4><P>An A3 assembly was performed as previous protocol, ligating and transforming as following:</P><UL><LI>
					pSB1T3Part A: 17O - RBS Part B: 2I - T1
				</LI><LI>
					pSB1T3Part A: 17O - RBS Part B: 2K - T1
				</LI><LI>
					pSB1T3Part A: 17O - RBS Part B: 19E - T1
				</LI><LI>
					pSB1T3Part A: 17O - RBSPart B: 11N - T1
				</LI><LI>
					pSB1T3Part A: 17O - RBSPart B: NtermHisTag - LacZ-alpha Stop
				</LI><LI>
					pSB1T3Part A: LacZ-alpha Non StopPart B: CtermHisTag - T1
				</LI><LI>
					pSB1T3 and pSB1A3Part A: pBADPart B: RBS
				</LI></UL><P>The eppendorf tubes and pipette tips were autoclaved in dry program for 30min at 120°C.</P><P>Plates with agar-tetracycline (Working concentration = 10μg/mL) were prepared.</P></DIV></DIV><DIV class="row"><H2>Tuesday</H2></DIV><DIV class="row"><DIV class="col-md-12 borderTopRow person-box"><H4>Vilde</H4><P>Made O/N cultures from the 3A assembly done yesterday.</P></DIV></DIV><DIV class="row"><DIV class="col-md-12 borderTopRow person-box"><H4>Izadora</H4><P>Just few colonies were got from transformations, so the positive colonies were incubated O/N as previously.</P><P>LB-tetracycline, SOC and LB-ampicilin medium were prepared as previously.</P><P>Ligation with previous digested products was repeated as before and the transformation as well.</P></DIV></DIV><DIV class="row"><H2>Wednesday</H2></DIV><DIV class="row"><DIV class="col-md-12 borderTopRow person-box"><H4>Vilde</H4><P>No growth in the O/N culture except from the promoter-Amberstop-tRNA in the pAC-plasmid. We found out that we had used the work tetracycline antibiotic.</P></DIV></DIV><DIV class="row"><DIV class="col-md-12 borderTopRow person-box"><H4>Vilde</H4><P>PCR of lacZ beta from the galactosidase biobrick from iGEM kit. This galactosidase gene does not contain the EcoRI restriction enzyme in the end of the gene. This means we do not have to do mutagenesis to take away the restriction site. I use Q5 protocol and setup. Used primers: LaczBeta forward and LacZBeta reverse – Stop and Non-Stop. Used both galatosidace biobricks found in the kit as template. For the part 9E kit plate 2 (galactosidase w. RBS) I got clear bond for both parts. We will use these parts for future work.</P></DIV></DIV><DIV class="row"><DIV class="col-md-12 borderTopRow person-box"><H4>Sumaya</H4><P>Miniprepped pBad-RBS and promotor -amber tRNA O/Ns.</P><P>Digested the pBad-RBS and pSB1T3 and ligated thus:</P><UL><LI>pBad-RBS + 11N-T1</LI><LI> pBad-RBS + 19E-T1</LI><LI> pBad-RBS + 2K-T1</LI><LI> pBad-RBS + 2I-T1</LI></UL></DIV></DIV><DIV class="row"><DIV class="col-md-12 borderTopRow person-box"><H4>Izadora</H4><P>Colonies were got just for pBAD - RBS in pSB1A3. </P><P>A PCR in these colonies was performed following TaKaRa® enzyme’s protocol (VF/VR2 program).</P><P>The plates were prepared with anhydrotetracycline by mistake. For that reason plates and stock solution with tetracycline hydrochloride (15μg/mL) were prepared.</P><P>The previous transformation was repeated in the new tetracycline medium.</P></DIV></DIV><DIV class="row"><H2>Thursday</H2></DIV><DIV class="row"><DIV class="col-md-12 borderTopRow person-box"><H4>Vilde</H4><P>Made O/N culture of RBS-4G in pSB1C3. We will use this to isolate the plasmid, cut it and run it on a gel. We will isolate the plasmid from the gel and use it in the assembly.</P></DIV></DIV><DIV class="row"><DIV class="col-md-12 borderTopRow person-box"><H4>Vilde</H4><P>Meeting with supervisor about Boston/Giant jamboree and problems in the labwork.</P></DIV></DIV><DIV class="row"><DIV class="col-md-12 borderTopRow person-box"><H4>Sumaya</H4><P>Attemped tranformation of yesterdays ligations.</P><P>Ran a PCR of minipreps:</P><UL><LI>2K-T1</LI><LI>19E-T1</LI><LI>2I-T1</LI><LI>11N-T1</LI><LI>Ehaj2-T0</LI><LI>Ehaj1-T1</LI><LI>Inv Ful short</LI><LI>Inv Ful long</LI></UL><P>All were in pSB1K3</P></DIV></DIV><DIV class="row"><H2>Friday</H2></DIV><DIV class="row"><DIV class="col-md-12 borderTopRow person-box"><H4>Vilde</H4><P>PCR Clean up. Cleaned up PCR products before digestion. Used NovoSpin Gel PCR and clean up kit. Washed two times and eluted in 20ul of dH2O. I cleaned this PCR products:</P><UL><LI>Invful Long</LI><LI>Invful Short</LI><LI>Ehaj1</LI><LI>Ehaj2</LI><LI>Signaling Peptide</LI><LI>LacZ beta Stop</LI><LI>LacZ Beta Non Stop</LI><LI>LacZ Alpha Non Stop</LI></UL><P>Checked concentrations on Nanodrop: ca 40ng/ul</P></DIV></DIV><DIV class="row"><DIV class="col-md-12 borderTopRow person-box"><H4>Vilde</H4><P>Digested samples for future 3A assembly and ligation into pSB1C3. Digested with EcoRi and SpeI:</P><UL><LI>Promoter-RBS</LI><LI>LacZBeta Stop</LI><LI>LacZBeta Non Stop</LI><LI>LacZAlpha Non Stop</LI><LI>LacZAlpha Stop</LI></UL><P>Digested with XbaI and PstI</P><UL><LI>SP-invfull short</LI><LI>SP-Invfull long</LI><LI>T1 (Terminator)</LI><LI>Ehaj1 - T1</LI><LI>Ehaj2 – T1</LI><LI>C-term his – T1</LI></UL><P>Digested with EcoRI and SpeI</P><UL><LI>pSB1A3</LI><LI>pSB1K3</LI><LI>Samples I PCR cleaned up earlier today</LI></UL><P>Digestion: 30 min at 37 degrees and inactivation of the enzymes by 20 min at 80 degrees.</P></DIV></DIV><DIV class="row"><DIV class="col-md-12 borderTopRow person-box"><H4>Vilde</H4><P>PCR Clean-Up. Used Nucleospin gel and PCR clean up and cleaned up all the digested samples.</P></DIV></DIV><DIV class="row"><DIV class="col-md-12 borderTopRow person-box"><H4>Vilde</H4><P>Run gel on Sumayas PCR from yesterday. No Gel results.</P></DIV></DIV><DIV class="row"><DIV class="col-md-12 borderTopRow person-box"><H4>Vilde</H4><P>Bought T4 ligase.</P></DIV></DIV><DIV class="row"><DIV class="col-md-12 borderTopRow person-box"><H4>Sumaya</H4><P>Only the transformation of  pBad-RBS-11N-T1 and  pBad-RBS-2I-T1. O/Ns were made of these two. </P><P>Made 6 agar plates with L-arabinose and tetracycline. And replated the  pBad-RBS-11N-T1 and and  pBad-RBS-2I-T1 onto two of these to see if they would show colour the day after.</P></DIV></DIV><DIV class="row"><DIV class="col-md-12 borderTopRow person-box"><H4>Sumaya</H4><P>No colour reaction was observed from yesterdays tranformation on the platesa with L-arabinose. Since the colour reaction did not happen the O/Ns were not used and the plates discarded.</P></DIV></DIV><DIV class="row"><DIV class="col-md-12 borderTopRow person-box"><H4>Izadora</H4><P>A Miniprep from RBS 4G was performed using NucleoSpin® plasmid kit and following the manufacturer’s protocol (Macherey-Nagel).</P></DIV></DIV><DIV class="row"><H2>Saturday</H2></DIV><DIV class="row"><DIV class="col-md-12 borderTopRow person-box"><H4>Izadora</H4><P>The DNA concentration was measured by NanoDrop (NanoDrop 1000 3.8.1) in the following samples:</P><UL><LI>InvFull long: 40.9ng/μL</LI><LI>Ehaj 1: 45.9ng/μL</LI><LI>LacZ-beta Stop: 41.0ng/μL</LI><LI>LacZ-beta Stop: 37.6ng/μL</LI><LI>Signal peptide: 65.2ng/μL</LI><LI>RBS - pSB1C3: 32.3ng/μL</LI></UL><P>An A3 assembly was done using RBS - pSB1C3, but after digestion the digested products were tested in a gel electrophoresis with agarose gel (0.7%).</P><P>No fragments were seen, but the ladder.</P><P>In an attempt to correct subsequent failures of A3 assembly, the iGEM’s protocol was changed and 500ng of DNA was used, instead of 250ng suggested. For that reason, the digestion was performed at 37°C for 1hour.</P><P>In addition to that, the transformation efficiency kit from iGEM was used following the protocol to check the efficiency of the competent cells being used.</P></DIV></DIV></DIV><DIV class="weekDiv" id="w32div"><DIV class="row"><H1>Week 32</H1></DIV><DIV class="row"><H2>Monday</H2></DIV><DIV class="row"><DIV class="col-md-12 borderTopRow person-box"><H4>William, Håkon</H4><P>The N-terminal His tag was made by mixing reverse and forward primers, putting the solution in a 96oC waterbath and left to cool.</P><P>Made O/N of TOP10 and NCM17.</P><P>Made O/N of double terminator in pSB1C3 (too much medium compared to bacteria - no growth).</P></DIV></DIV><DIV class="row"><DIV class="col-md-12 borderTopRow person-box"><H4>Vilde</H4><P>3A assembly. Followed protocol from iGEM.</P><OL><LI>PartA: Promoter RBS, PartB: Sp Invful long, Plasmid: km</LI><LI>A: Promoter RBS, B: Sp-invful short, P: km</LI><LI>A: Promoter-RBS, B: Signalling peptide, P: Km</LI><LI>A: LacZ Beta Stop, B: T1, P: amp</LI><LI>A: LacZAlpha Stop, B: T1, P: Amp</LI><LI>A: LacZAlpha NS, B: Ehaj1-T1, P: Km</LI><LI>A: LacZAlpha NS, B: Ehaj2-T1, P: Km</LI><LI>A: LacZBeta NS, B: Ehaj1-T1, P: Km</LI><LI>A: LacZBeta Ns, B: Ehaj2-T1, P: Km</LI><LI>A: LacZAlpha NS, B: C-term His-T1, P: amp</LI><LI>A: LacZBeta NS, B: C-term His- T1, P: amp</LI><LI>A: Promoter-RBS, B: AlphaNS, P:km</LI><LI>A: promoter-RBS, B: Beta Ns, P: km</LI></OL></DIV></DIV><DIV class="row"><DIV class="col-md-12 borderTopRow person-box"><H4>Izadora</H4><P>No colonies grown in the plates from the transformation efficiency test. Therefore, new TOP 10 competent cells were prepared.</P></DIV></DIV><DIV class="row"><H2>Tuesday</H2></DIV><DIV class="row"><DIV class="col-md-12 borderTopRow person-box"><H4>William</H4><P>O/N cultures of TOP10 and NCM17 was diluted and Jack’s protocol was followed in order to make them competent.</P></DIV></DIV><DIV class="row"><DIV class="col-md-12 borderTopRow person-box"><H4>Vilde</H4><P>Made O/N cultures of colonies from yesterday’s 3A assembly. I got colonies on assembly: 1,4,5,6,7,9 and 13.</P></DIV></DIV><DIV class="row"><DIV class="col-md-12 borderTopRow person-box"><H4>Vilde</H4><P>3A assembly. Due to few colonies from yesterday’s assembly, I suspect something wrong with the parts we already got. I will thus make some of them again from scratch by 3A assembly. Followed iGEM’s protocol and procedure. I ligated them all in to the pSB1K3 plasmid from the kit.</P><OL><LI>Part A: C-term his, Part B: Terminator1 (T1)</LI><LI>A. C-term his, B. Double Terminator (DT)</LI><LI>A: Ehaj1, B: Dt</LI><LI>A: Ehaj2, B: DT</LI><LI>A: LacZAlpha Stop, B: DT</LI><LI>A: LacZBeta Stop, B. Dt</LI><LI>A: signaling peptide (SP), B; T1</LI><LI>A: SP, B: DT</LI><LI>A: Ehaj1, B: T1</LI><LI>A: Ehaj2, B: T1</LI></OL><P>Transformed the ligated products into competent TOP10 E.coli Cells.</P></DIV></DIV><DIV class="row"><DIV class="col-md-12 borderTopRow person-box"><H4>Izadora</H4><P>A Miniprep was performed using Promega Wizard Plus SV miniprep kit in the O/N cultures below:</P><UL><LI>CtermHisTag - T1</LI><LI>LacZ-beta pSB1C3</LI><LI>pBAD - RBS</LI><LI>CtermHisTag</LI><LI>T1</LI><LI>RBS 4G</LI></UL><P>The double of lysis buffer was used.</P><P>The DNA concentrations of the isolated plasmid DNAS were measured in NanoDrop (NanoDrop 1000 3.8.1):</P><UL><LI>CtermHisTag - T1: 15.9ng/μL</LI><LI>LacZ-beta pSB1C3: 40.1ng/μL</LI><LI>pBAD - RBS: 17.6ng/μL</LI><LI>CtermHisTag: 80.3ng/μL</LI><LI>T1: 49.3ng/μL</LI><LI>RBS 4G: 24.4ng/μL</LI></UL><P>The 280/260 value for samples 1 and 3 were too lower than 1.80, indicating presence of contaminations.</P><P>A gel electrophoresis in agarose gel (0.7%) with these plasmid DNAs was performed and the fragment-sizes were like expected, but no fragments for samples 1 and 3 were could be seen.</P><P>Another digestion was performed in RNS 4G and T1 and they were checked by gel electrophoresis in agarose gel (0.7%) before being used for ligation.</P><P>An O/N culture was prepared with pBAD in LB/chloramphenicol as previously.</P></DIV></DIV><DIV class="row"><H2>Wednesday</H2></DIV><DIV class="row"><DIV class="col-md-12 borderTopRow person-box"><H4>William</H4><P>The digestion of the 3A assembly protocol was done with:</P><P>N-His tag, pBad and constitutive promoter (17O) as part A’s (cut with EcoRI and SpeI). LacZ alpha stop, lacZ beta stop and RBS as part B’s (cut with PstI and XbaI). pSB1A3 and pSB1C3 as plasmids (cut with PstI, EcoRI).</P></DIV></DIV><DIV class="row"><DIV class="col-md-12 borderTopRow person-box"><H4>Vilde</H4><P>3A assembly. Tested colonies from 3A assembly from 04.08 and 05.08 by  PCR. Used TaKaRa enzyme and followed 50ul setup. Used elongation 4,3 min and annealing 55degrees. The gel showed only one positive colonies. The gel showed a lot of religated plasmids (bonds of 250bp). I troubleshootet the problem and found out that I will start to de phosphorylated the plasmid to prevent relegation.</P></DIV></DIV><DIV class="row"><DIV class="col-md-12 borderTopRow person-box"><H4>Vilde</H4><P>Minipreped O/N cultures from ligation 04.08. Wanted to test them on PCR to be sure that it is nothing wrong with the way we are doing PCR to test colonies.  Tested the concentration on nanodrop – all samples had a concentration between 50-80ng/ul and good 260/280 values. However all the samples had low 230/260 values that indicates that there are co-purified contaminants in the sample.</P></DIV></DIV><DIV class="row"><DIV class="col-md-12 borderTopRow person-box"><H4>Vilde</H4><P>Digested the minipreps and run gel on them. Run gel on the samples minipreped. The results showed that only Alpha-NS-Ehaj1 gave us the expected size of fragments in the gel. However for the smallest parts (RBS, T1 etc) we will not expect to see the bond from the part after digestion only the linearized plasmid. We can thus not say if these parts are correct or just re-ligated plasmids.</P></DIV></DIV><DIV class="row"><DIV class="col-md-12 borderTopRow person-box"><H4>Vilde</H4><P>Tested the one positive colony on PCR to be completely sure it is correct. Used gene specific primers (for Ehaj and LacZAlpha) combined with the iGEM VR and VF2 primers to be sure the fusion protein is in the plasmid.</P></DIV></DIV><DIV class="row"><H2>Thursday</H2></DIV><DIV class="row"><DIV class="col-md-12 borderTopRow person-box"><H4>William, Håkon</H4><P>Colonies from a plate with growth of LacZ-beta transformed bacteria were put in tubes, mixed with 20 µl water, heated to 96oC for 10 minutes and then centrifuged at 11.000 rpm for 10 minutes.</P><P>A PCR was set up in order to amplify the LacZ beta fragment, but there was no result on the gel.</P></DIV></DIV><DIV class="row"><DIV class="col-md-12 borderTopRow person-box"><H4>Vilde</H4><P>Run gel on PCR from yesterday. Run the gel on the PCR from  06.08. I got bonds as expected and can thus say that  the part LacZalpha NS – Ehaj1 is in the plasmid.</P></DIV></DIV><DIV class="row"><DIV class="col-md-12 borderTopRow person-box"><H4>Vilde, Håkon</H4><P>I tested more of the 3A assembly colonies from 05.08 on PCR. Hopefully not all of them are re-ligated plasmids. Used TaKaRa setup (25ul). I tested 3 different colonies from CtermHis –T1 and three from C-termHis-DT. Run the PCR with Håkon.</P></DIV></DIV><DIV class="row"><DIV class="col-md-12 borderTopRow person-box"><H4>Vilde</H4><P>Negative PCR results. Found out that Håkon had used a too short elongation time for my samples to be amplified. Have to do a new PCR on the colonies.</P></DIV></DIV><DIV class="row"><DIV class="col-md-12 borderTopRow person-box"><H4>Vilde</H4><P>Dephosphorylated pSB1C3. Used the pSB1CR Izadora cut (with EcoRI and PstI)and purified from gel yesterday. Used 17ul of purified pSB1C3, 2ul of 10X buffer and 1ul of phosphatase. Let the reaction in 37 degrees for 15 minutes and heat inactivated the enzymes by leaving the sample at 75degrees for 5 minutes. Stored the sample at -20 degrees.</P></DIV></DIV><DIV class="row"><DIV class="col-md-12 borderTopRow person-box"><H4>Vilde</H4><P>Assembled parts into dephosphorylated pSB1C3. Used the dephosphorylated pSB1C3 and parts digested 01.08. Ligation reaction sample: 2ul pSB1C3, 2ul Part, 1ul 10X buffer, 0,5uL enzyme and 4,5ul DH2O. Transformed 5ul and plated out on CAM plates. These parts will be the new biobrikcs we will send to the iGEM register.</P></DIV></DIV><DIV class="row"><DIV class="col-md-12 borderTopRow person-box"><H4>Izadora</H4><P>An A3 assembly was performed making a dephosphorylation with the enzyme FastAP Thermosensitive Alkaline Phosphatase in the plasmids before ligating it and following the manufacturer’s protocol (Thermo Scientific). </P><P>The digestion enzymes that iGEM’s protocol suggests were used and the buffers as following:</P><UL><LI>
					Part A: Cut Smart bufferPart B: Cut Smart bufferPlasmid: NEB 3.1 buffer
				</LI><LI>
					Part A: NtermHisTag Part B: LacZ-alpha Stop
				</LI><LI>
					Part A: LacZ-alpha Non StopPart B: CtermHisTag
				</LI><LI>
					Part A: CtermHisTagPart B: T1
				</LI><LI>
					Part A: NtermHisTag 
				</LI></UL><P>Together with the A3 assembly the re-ligation or circularization of the following linearized plasmids without dephosphorylation was attempted: pSB1A3, pSB1K3, pSB1T3.</P></DIV></DIV><DIV class="row"><H2>Friday</H2></DIV><DIV class="row"><DIV class="col-md-12 borderTopRow person-box"><H4>Izadora</H4><P>The isolated plasmid DNAs with T1 and RBS 4G were tested by gel electrophoresis in an agarose gel (0.7%) and the expected fragment-sizes were obtained.</P><P>After that, the fragmentes were isolated by cutting them out from the gel and purifying them with the Kit NucleoSpin Gel and PCR clean up, following the manufacturer’s protocol (Macherey-Nagel).</P><P>The plasmid isolated in this process was pSB1C3.</P><P>200mL of SOC medium was prepared as previously.</P></DIV></DIV></DIV><DIV class="weekDiv" id="w33div"><DIV class="row"><H1>Week 33</H1></DIV><DIV class="row"><H2>Monday</H2></DIV><DIV class="row"><DIV class="col-md-12 borderTopRow person-box"><H4>Izadora</H4><P>A PCR was performed as the TaKaRa® enzyme’s protocol in the positive transformations: 3 (pSB1A3 and pSB1K3), 1 (pSB1K3), 4 (pSB1K3) and the all three plasmids, as well. Plus, the following samples: LacZ-beta Stop and Non Stop.</P><P>A gel electrophoresis in agarose gel (1%) was performed with PCR samples.</P><P>200mL of LB/kanamycin medium was prepared as previously.</P><P>The O/N cultures with colonies that presented the correct fragments in the gel were prepared and incubated as previously.</P></DIV></DIV><DIV class="row"><H2>Tuesday</H2></DIV><DIV class="row"><DIV class="col-md-12 borderTopRow person-box"><H4>Izadora</H4><P>With the positive O/N cultures, frozen stock cells were prepared and stored at -80°C, miniprep were performed using Promega Wizard Plus SV miniprep kit and DNA concentration were measured as below:</P><UL><LI>pSB1K3: 35.2ng/μL</LI><LI>pSB1A3: 40.5ng/μL</LI></UL><P>A gel electrophoresis in agarose gel (0.7%) was performed with these samples and the expected fragment-sizes were obtained.</P><P>The isolated plasmid DNAs were stored at -20°C.</P><P>An A3 assembly with pSB1A3 and pSB1K3 was performed as the last protocol with:</P><UL><LI>
					Part A: pBADPart B: RBS
				</LI></UL></DIV></DIV><DIV class="row"><H2>Wednesday</H2></DIV><DIV class="row"><DIV class="col-md-12 borderTopRow person-box"><H4>Izadora</H4><P>A PCR was performed in 5 different colonies from last transformation following TaKaRa®’s enzyme protocol. The fragment-sizes were obtained like expected.</P><P>O/N cultures with the 5 colonies tested were prepared as previously.</P><P>In order to remake empty stocks of isolated plasmid DNAs, O/N cultures of it were also prepared as previously.</P></DIV></DIV><DIV class="row"><H2>Thursday</H2></DIV><DIV class="row"><DIV class="col-md-12 borderTopRow person-box"><H4>Izadora</H4><P>A miniprep was performed using Promega Wizard Plus SV miniprep kit in the positive O/N cultures from yesterday.</P></DIV></DIV></DIV><DIV class="weekDiv" id="w34div"><DIV class="row"><H1>Week 34</H1></DIV><DIV class="row"><H2>Saturday</H2></DIV><DIV class="row"><DIV class="col-md-12 borderTopRow person-box"><H4>William, Håkon</H4><P>The N-terminal His tag was made by mixing reverse and forward primers, putting the solution in a 96oC waterbath and left to cool (it did not work the last time(4.8.14)).</P><P>The LacZ (19E) and the LacZ (9B) was attempted amplified using PCR. Did not work this time either.</P></DIV></DIV></DIV><DIV class="weekDiv" id="w35div"><DIV class="row"><H1>Week 35</H1></DIV><DIV class="row"><H2>Sunday</H2></DIV><DIV class="row"><DIV class="col-md-12 borderTopRow person-box"><H4>William, Håkon</H4><P>DNA concentrations was measured using NanoDrop:</P><UL><LI>LacZ beta non stop - 15.2 ng/µl</LI><LI>LacZ beta stop - 19.0 ng/µl</LI><LI>EhajI - 15.9 ng/µl</LI><LI>EhajII - 22.9 ng/µl</LI><LI>Signal peptide - 56.2 ng/µl</LI></UL><P>A ligation was set up using all the parts above cut with EcoRI and PstI.</P><P>The plasmid used was pSB1C3. Followed the T4 ligase protocol.</P><P>The ligation mixtures was mixed with TOP10 competent cells for transformation (Jack’s protocol).</P><P>The transformation mixtures was plated onto chloramphenicol plates and incubated until the next day (16h).</P></DIV></DIV></DIV><DIV class="weekDiv" id="w36div"><DIV class="row"><H1>Week 36</H1></DIV></DIV><DIV class="weekDiv" id="w37div"><DIV class="row"><H1>Week 37</H1></DIV><DIV class="row"><H2>Tuesday</H2></DIV><DIV class="row"><DIV class="col-md-12 borderTopRow person-box"><H4>Vilde</H4><P>PCR on colonies. Tested LacZ Beta Non-stop, LacZ-bet Stop and the autotransporter Ehaj1 all in pSB1c3 plasmid backbone. Used VF and VR2 primers. Used Q5 polymerase and followed protocol supplied with the enzyme. Bands of around 500bp were observed for all three samples. This may be religated plasmid backbone.</P></DIV></DIV><DIV class="row"><DIV class="col-md-12 borderTopRow person-box"><H4>Vilde</H4><P>O/N cultures. Made Overnight cultures from the colonies tested even if the PCR did not show expected results. I will try to do another PCR and/Or sequence the isolated plasmid DNA to check if it is religated plasmid I observe.</P></DIV></DIV><DIV class="row"><DIV class="col-md-12 borderTopRow person-box"><H4>Vilde</H4><P>Made new LB Cam medium</P></DIV></DIV><DIV class="row"><DIV class="col-md-12 borderTopRow person-box"><H4>Vilde</H4><P>Made O/N culture of LacZ-gene from iGEM kit that I will use to make constructs.</P></DIV></DIV><DIV class="row"><H2>Wednesday</H2></DIV><DIV class="row"><DIV class="col-md-12 borderTopRow person-box"><H4>Vilde</H4><P>Minipreped the bacterial O/N culture containing LacZ-Gene from iGEM kit.</P></DIV></DIV><DIV class="row"><DIV class="col-md-12 borderTopRow person-box"><H4>Vilde</H4><P>It seems like we have a problem with the digestion of our constructs and or plasmid since we get religated plasmids. I tested all the restriction enzymes stepwise with the corresponding buffer. I used the protocol and setup supplied with the enzyme. I used the EcoRI buffer for EcoRI enzyme, NEB3.1 for SpeI and Cutsmart for XbaI and PstI. I cut a known plasmid from the iGEM kit. Analysis of  cut constructs in a gel shows that all restriction enzymes are functional, but has to be done with correct buffer.</P><P>I cut the linear plasmid backbone in a stepwise matter</P></DIV></DIV><DIV class="row"><DIV class="col-md-12 borderTopRow person-box"><H4>Vilde</H4><P>Cut pSB1C3. Cut one sample of the plasmid backbone with EcoRI and one with PstI. Used correct buffer for both enzymes. I cut the samples at 37 degrees for 1 hour.</P></DIV></DIV><DIV class="row"><DIV class="col-md-12 borderTopRow person-box"><H4>Vilde</H4><P>PCR cleaned up of the cur plasmid backbone samples eluted the samples in 20ul dH2O.</P></DIV></DIV><DIV class="row"><DIV class="col-md-12 borderTopRow person-box"><H4>Vilde</H4><P>Cut the plasmids backbone with the other enzyme using supplied protocol and buffer. Stored the stepwise cut plasmid backbone in -20 freezer for usage in ligation later.</P></DIV></DIV><DIV class="row"><H2>Friday</H2></DIV><DIV class="row"><DIV class="col-md-12 borderTopRow person-box"><H4>Vilde</H4><P>PCR to amplify new constructs from plasmid DNA. Used gene specific primers and template corresponding to the construct I will make. Used the whole LacZ gene as template for LacZbeta NS, lacZBeta Stop, LazC alpha NS. Used isolated plasmids containing the autotransporter gene to amplify the autotransporter and signaling peptide. I used Q5 polymerase and a 50ul reaction volume. The samples were cut and ligated into already cut plasmid (10.09) and transformed into Top10 cells.</P></DIV></DIV><DIV class="row"><DIV class="col-md-12 borderTopRow person-box"><H4>Vilde</H4><P>Sent samples to sequencing. Ehaj1 pSB1c3 with VF2 and VR primers, pBAd-RBS with VR primers. Used 5ul DNA  (80ng/ul) and 5ul 5mM primers</P></DIV></DIV><DIV class="row"><DIV class="col-md-12 borderTopRow person-box"><H4>Vilde</H4><P>Tested colonies on PCR. Tested colonies with constructs made and transformed 12.09. Only Ehaj2 pSB1C3 showed correct size. The other showed religated plasmid. I will test more colonies.</P></DIV></DIV><DIV class="row"><DIV class="col-md-12 borderTopRow person-box"><H4>Vilde</H4><P>Sat up new PCR to screen for positive colonies from 12.09. Found LacZ Beta non-stop and LacZ alpha non-stop as positive colonies.</P></DIV></DIV><DIV class="row"><H2>Sunday</H2></DIV><DIV class="row"><DIV class="col-md-12 borderTopRow person-box"><H4>Vilde</H4><P>Made O/n cultures of positive colonies.</P></DIV></DIV><DIV class="row"><DIV class="col-md-12 borderTopRow person-box"><H4>Vilde</H4><P>Miniprepped Ehaj2 pSB1C3 and send the sample to sequencing.</P></DIV></DIV></DIV><DIV class="weekDiv" id="w38div"><DIV class="row"><H1>Week 38</H1></DIV><DIV class="row"><H2>Monday</H2></DIV><DIV class="row"><DIV class="col-md-12 borderTopRow person-box"><H4>Vilde</H4><P>Minipreped lacZ beta non-stop pSB1C3 and lazC alpha non-stop pSB1C3. Send to sequencing the 19.09</P></DIV></DIV><DIV class="row"><H2>Thursday</H2></DIV><DIV class="row"><DIV class="col-md-12 borderTopRow person-box"><H4>Vilde</H4><P>Did a new PCR to amplify constructs not already made. Used Q5 polymerase, Gene specific primers and template DNA containing the gene of interest. I used an annealing temperature of 53 degrees.</P></DIV></DIV><DIV class="row"><H2>Friday</H2></DIV><DIV class="row"><DIV class="col-md-12 borderTopRow person-box"><H4>Vilde</H4><P>Dephosphorylated already stepwise cut pSB1C3 to prevent re-ligation.</P></DIV></DIV><DIV class="row"><DIV class="col-md-12 borderTopRow person-box"><H4>Vilde</H4><P>Ligated the dephosphorylated, cut pSB1C3 into constructs made 18.09; LacZ Beta Non stop, LacZ beta Stop,LacZ alpha NS, Invful (autotransporter) and Signaling peptide. Followed protocol supplied by the T4 ligase and transformed into TOP10 cells.</P></DIV></DIV><DIV class="row"><DIV class="col-md-12 borderTopRow person-box"><H4>Vilde</H4><P>Checked sequencing results. Seems like the LacZ beta gene is correct. It does not contain the EcoRI site in the middle of the gene.</P></DIV></DIV><DIV class="row"><H2>Saturday</H2></DIV><DIV class="row"><DIV class="col-md-12 borderTopRow person-box"><H4>Vilde</H4><P>Checked colonies on PCR (from 19.09). Used TaKaRa PCR and VF2, VR primers. The colonies did not give bands as expected.</P></DIV></DIV><DIV class="row"><DIV class="col-md-12 borderTopRow person-box"><H4>Vilde</H4><P>Checked Sequencing results. It seems like our constructs do not contain the whole suffix. I made new reverse primers to test if this lack of the whole suffix may be due to wrong primers.</P></DIV></DIV></DIV><DIV class="weekDiv" id="w39div"><DIV class="row"><H1>Week 39</H1></DIV><DIV class="row"><H2>Thursday</H2></DIV><DIV class="row"><DIV class="col-md-12 borderTopRow person-box"><H4>Vilde</H4><P>Got the new primers and set up PCR to make new constructs. Used Q5 and gene specific primers. Made: LacZBeta NS, LacZ Beta Stop, LacZ alpha Ns, Ehaj1, Ehaj2 and signaling peptide. All were positive except lacZ alpha non stop.</P></DIV></DIV><DIV class="row"><DIV class="col-md-12 borderTopRow person-box"><H4>Vilde</H4><P>Stepwise digestion of the PCR products and pSB1C3. Cut the constructs and the plasmid backbone stepwise using the correct buffer to each enzyme and a PCR cleanup in between.</P></DIV></DIV><DIV class="row"><DIV class="col-md-12 borderTopRow person-box"><H4>Vilde</H4><P>Dephosphorylated the plasmid backbone to prevent relegation</P></DIV></DIV><DIV class="row"><DIV class="col-md-12 borderTopRow person-box"><H4>Vilde</H4><P>Ligated the cut constructs into the cut and dephosphorylated plasmid backbone. Used T4 ligase and followed the protocol supplied with the enzyme. Transformed only LacZ beta Non stop due to lack of competent cells.</P></DIV></DIV><DIV class="row"><H2>Friday</H2></DIV><DIV class="row"><DIV class="col-md-12 borderTopRow person-box"><H4>Vilde</H4><P>Growth was observed for the transformed LacZ Beta Non-Stop from 25.09.</P></DIV></DIV><DIV class="row"><DIV class="col-md-12 borderTopRow person-box"><H4>Vilde</H4><P>Showed Håkon And William what I have been doing so they can continue making the constructs after I move to Germany.</P></DIV></DIV></DIV><DIV class="weekDiv" id="w40div"><DIV class="row"><H1>Week 40</H1></DIV><DIV class="row"><H2>Monday</H2></DIV><DIV class="row"><DIV class="col-md-12 borderTopRow person-box"><H4>Håkon</H4><P>O/N of top 10 e.coli with pASK-IBA3 plasmid were prepared.</P></DIV></DIV><DIV class="row"><H2>Tuesday</H2></DIV><DIV class="row"><DIV class="col-md-12 borderTopRow person-box"><H4>Håkon</H4><P>PCR was preformed on Lac-Z alpha stop using VF2 and VR primers.</P><P>Miniprep of O/N culture with pASK-IBA3 was preformed.</P><P>Lac-Z beta and Lac-Z alpha were measured in nanodrop.</P></DIV></DIV><DIV class="row"><H2>Wednesday</H2></DIV><DIV class="row"><DIV class="col-md-12 borderTopRow person-box"><H4>Håkon</H4><P>Sequencial cutting with EcorI and PstI was preformed followed by alkaline phophatase treatment.</P><P>Ligation reaction was put on incubation for 1 hour and transformation preformed directly after.</P></DIV></DIV><DIV class="row"><H2>Thursday</H2></DIV><DIV class="row"><DIV class="col-md-12 borderTopRow person-box"><H4>Håkon</H4><P>New transformation was preformed using our standard protocol and competent NCM17 E.coli cell with deleted Lac operon.</P></DIV></DIV><DIV class="row"><H2>Saturday</H2></DIV><DIV class="row"><DIV class="col-md-12 borderTopRow person-box"><H4>Håkon</H4><P>Colonies were selected and O/N were made.</P></DIV></DIV><DIV class="row"><H2>Sunday</H2></DIV><DIV class="row"><DIV class="col-md-12 borderTopRow person-box"><H4>Håkon</H4><P>Miniprep was preformed on all O/N cultures. No gowth in the O/N cultures!</P></DIV></DIV></DIV><DIV class="weekDiv" id="w41div"><DIV class="row"><H1>Week 41</H1></DIV><DIV class="row"><H2>Wednesday</H2></DIV><DIV class="row"><DIV class="col-md-12 borderTopRow person-box"><H4>Håkon</H4><P>Sequencial digestion with EcorI and PstI using the correct buffers and PCR- clean-up between the two digestions. Digestions tested on gel, but results was negative.</P></DIV></DIV></DIV></DIV><DIV class="footer"><DIV class="container"><P class="text-muted text-center">Created using Bootstrap and jQuery.</P></DIV></DIV><DIV class="printfooter">
Retrieved from &quot;<A href="http://2014.igem.org/Team:UiOslo_Norway/Notebook">http://2014.igem.org/Team:UiOslo_Norway/Notebook</A>&quot;</DIV></DIV></DIV><DIV id="footer-box" class="noprint"><DIV id="footer"><UL id="f-list"><LI id="t-recentchanges"><A href="/Special:RecentChanges" title="Recent changes">Recent changes</A></LI><LI id="t-whatlinkshere"><A href="/Special:WhatLinksHere/Team:UiOslo_Norway/Notebook" title="List of all wiki pages that link here [j]" accesskey="j">What links here</A></LI><LI id="t-recentchangeslinked"><A href="/Special:RecentChangesLinked/Team:UiOslo_Norway/Notebook" title="Recent changes in pages linked from this page [k]" accesskey="k">Related changes</A></LI><LI id="t-specialpages"><A href="/Special:SpecialPages" title="List of all special pages [q]" accesskey="q">Special pages</A></LI><LI><A href="/Special:Preferences">My preferences</A></LI></UL></DIV><DIV id="footer"><UL id="f-list"><LI id="t-print"><A href="/wiki/index.php?title=Team:UiOslo_Norway/Notebook&amp;printable=yes" title="Printable version of this page [p]" accesskey="p">Printable version</A></LI><LI id="t-permalink"><A href="/wiki/index.php?title=Team:UiOslo_Norway/Notebook&amp;oldid=323686" title="Permanent link to this revision of the page">Permanent link</A></LI><LI id="privacy"><A href="/2014.igem.org:Privacy_policy" title="2014.igem.org:Privacy policy">Privacy policy</A></LI><LI id="disclaimer"><A href="/2014.igem.org:General_disclaimer" title="2014.igem.org:General disclaimer">Disclaimers</A></LI></UL></DIV></DIV></DIV></BODY></HTML>