<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_Evry_Paris-Saclay_Demonstrate skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><DIV id="top_title"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:Evry Paris-Saclay/Demonstrate</SPAN></H1></DIV><DIV id="HQ_page"><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><DIV class="container"> <DIV class="collapse navbar-collapse" id="navbarResponsive"><UL class="navbar-nav ml-auto"><LI class="nav-item"><A class="nav-link" href="https://2018.igem.org/Team:Evry_Paris-Saclay">Home</A></LI><LI class="nav-item dropdown"><A class="nav-link dropdown-toggle" href="https://2018.igem.org/Team:Evry_Paris-Saclay/Description" id="navbarDropdownProject" aria-haspopup="true" aria-expanded="false">
                Project
              </A><DIV class="dropdown-menu dropdown-menu-right" aria-labelledby="navbarDropdownProject"><A class="dropdown-item" href="https://2018.igem.org/Team:Evry_Paris-Saclay/Description">Description</A><A class="dropdown-item" href="https://2018.igem.org/Team:Evry_Paris-Saclay/Design">Design overview</A><A class="dropdown-item" href="https://2018.igem.org/Team:Evry_Paris-Saclay/Improve">The peptide</A><A class="dropdown-item" href="https://2018.igem.org/Team:Evry_Paris-Saclay/Model">The receptor</A><A class="dropdown-item" href="https://2018.igem.org/Team:Evry_Paris-Saclay/Demonstrate">The promoter</A><A class="dropdown-item" href="https://2018.igem.org/Team:Evry_Paris-Saclay/Futur_perspective">Future perspectives</A><SPAN class="separationMenu">--------------</SPAN><A class="dropdown-item" href="https://2018.igem.org/Team:Evry_Paris-Saclay/InterLab">Interlab</A><SPAN class="separationMenu">--------------</SPAN><A class="dropdown-item" href="https://2018.igem.org/Team:Evry_Paris-Saclay/Notebook">Notebook</A></DIV></LI><LI class="nav-item dropdown"><A class="nav-link dropdown-toggle" href="https://2018.igem.org/Team:Evry_Paris-Saclay/Parts" id="navbarDropdownParts" aria-haspopup="true" aria-expanded="false">
                Parts
              </A><DIV class="dropdown-menu dropdown-menu-right" aria-labelledby="navbarDropdownParts"><A class="dropdown-item" href="https://2018.igem.org/Team:Evry_Paris-Saclay/Parts#basic_parts">Basic Parts</A><A class="dropdown-item" href="https://2018.igem.org/Team:Evry_Paris-Saclay/Parts#composite_parts">Composite Parts</A><A class="dropdown-item" href="https://2018.igem.org/Team:Evry_Paris-Saclay/Parts#part_collection">Part Collection</A></DIV></LI><LI class="nav-item dropdown"><A class="nav-link dropdown-toggle" href="https://2018.igem.org/Team:Evry_Paris-Saclay/Human_Practices" id="navbarDropdownHumanPractices" aria-haspopup="true" aria-expanded="false">
                Human Practices
              </A><DIV class="dropdown-menu dropdown-menu-right" aria-labelledby="navbarDropdownHumanPractices"><A class="dropdown-item" href="https://2018.igem.org/Team:Evry_Paris-Saclay/Human_Practices#overview">Overview</A><A class="dropdown-item" href="https://2018.igem.org/Team:Evry_Paris-Saclay/Human_Practices#meeting_professionals">Meeting with professionals</A><A class="dropdown-item" href="https://2018.igem.org/Team:Evry_Paris-Saclay/Human_Practices#applications">Applications</A><A class="dropdown-item" href="https://2018.igem.org/Team:Evry_Paris-Saclay/Public_Engagement">Education &amp; public engagement</A><A class="dropdown-item" href="https://2018.igem.org/Team:Evry_Paris-Saclay/Human_Practices#integrated_human_practices">Integrated human practices</A><A class="dropdown-item" href="https://2018.igem.org/Team:Evry_Paris-Saclay/Collaborations">Collaborations</A></DIV></LI><LI class="nav-item"><A class="nav-link" href="https://2018.igem.org/Team:Evry_Paris-Saclay/Achievement">Achievement</A></LI><LI class="nav-item dropdown"><A class="nav-link dropdown-toggle" href="https://2018.igem.org/Team:Evry_Paris-Saclay/Team" id="navbarDropdownTeam" aria-haspopup="true" aria-expanded="false">
                Team
              </A><DIV class="dropdown-menu dropdown-menu-right" aria-labelledby="navbarDropdownTeam"><A class="dropdown-item" href="https://2018.igem.org/Team:Evry_Paris-Saclay/Team">Team Members</A><A class="dropdown-item" href="https://2018.igem.org/Team:Evry_Paris-Saclay/Attributions">Attributions</A><A class="dropdown-item" href="https://2018.igem.org/Team:Evry_Paris-Saclay/Sponsors">Sponsors</A></DIV></LI></UL></DIV></DIV><DIV class="container"><DIV class="col-md-3" style="width:auto;"><P style="color: white; text-align:left;" class="bibliographie"><B font-weight:900="">TO CONTACT US</B> Genopole Campus 1, Batiment 6, 91030 Evry Cedex, France +33 7 69 96 68 31<A href="mailto:igemevry@gmail.com"> igemevry@gmail.com</A></P></DIV><DIV class="col-md-3" style="width:auto;"><P style="color: white; text-align:left;" class="bibliographie">© Copyright 2018Design &amp; Developpement byIGEM EVRY GENOPOLE</P></DIV></DIV><DIV class="contentEvry" style="text-align:center; padding-left:auto; padding-right:auto;"><H1 style="font-weight:800; text-align:center;">THE PROMOTER</H1><P style="font-size:15px;">The switch from lytic-to-lysogenic cycle of phage phi3T is based on the expression of a single transcript, AimX [1]. The expression of AimX is controlled by the transcriptional regulator AimR (<A href="http://parts.igem.org/Part:BBa_K2279000">BBa_K2279000</A>) which is inactivated upon binding of the ‘arbitrium’ hexapeptide SAIRGA.
The transcription of AimX is driven from the intergenic region between AimP and AimX which we will refer to as pAimX(full) promoter (<A href="http://parts.igem.org/Part:BBa_K2675020">BBa_K2675020</A>). This corresponds to nucleotides 70182 to 70313 of Phi3T phage genome (<A href="https://www.ncbi.nlm.nih.gov/nuccore/KY030782">GenBank KY030782.1</A>).
</P><H2 style="font-weight:800; text-align:center;" id="basic_parts">THE NATURAL PROMOTER(S)</H2><P style="font-size:15px;">We have analysed the sequence of the pAimX(full) promoter using various available tools like the <A href="http://www.fruitfly.org/seq_tools/promoter.html">Neural Network Promoter Prediction web server</A>, <A href="http://linux1.softberry.com/berry.phtml?topic=bprom&amp;group=programs&amp;subgroup=gfindb">BPROM websever</A> [2] or <A href="http://rna.igmors.u-psud.fr/toolbox/arnold/index.php">ARNold</A> [3, 4].
These sequence analyses revealed the presence of three putative promoters that we denote pAimX(1), pAimX(2) and pAimX(3) (Figure 1), but also of an inverted repeat sequence potentially forming the stem-loop structure of a terminator.</P><FIGURE><FIGCAPTION style="font-size:12px;font-family: 'Nunito', sans-serif;"><B>Figure 1</B>: The predicted features of pAimx(full) and pAimX(short) promoters of phage phi3T’s AimX gene. -35 boxes are marked in green, -10 boxes in red, and the transcription start sites in blue and indicated with +1. The inverted repeat sequences potentially forming the stem-loop structure of a terminator are marked by open horizontal arrows.
</FIGCAPTION></FIGURE><P style="font-size:15px;">To investigate the promoter activity of pAimX(full) sequence in <EM>E. coli</EM>, we have placed the reporter sfGFP with or without the LVAtag (<A href="http://parts.igem.org/Part:BBa_K2675005">BBa_K2675005</A> and <A href="http://parts.igem.org/Part:BBa_K2675006">BBa_K2675006</A>) under its control and thus constructed the composite parts <A href="http://parts.igem.org/Part:BBa_K2675050">BBa_K2675050</A> and <A href="http://parts.igem.org/Part:BBa_K2675060">BBa_K2675060</A>.
Considering the fact that the potential internal terminator overlaps mainly with the pAimX(2) putative promoter, we suspect that the AimX transcription could probably be initiated from the pAimX(3) putative promoter. To investigate this possibility in <EM>E. coli</EM>, we also placed the reporter sfGFP with or without the LVAtag (<A href="http://parts.igem.org/Part:BBa_K2675005">BBa_K2675005</A> and <A href="http://parts.igem.org/Part:BBa_K2675006">BBa_K2675006</A>) under the control of this short version of the pAimX promoter (pAimX(3) = pAimX(short) promoter) and thus constructed the <A href="http://parts.igem.org/Part:BBa_K2675051">BBa_K2675051</A> and <A href="http://parts.igem.org/Part:BBa_K2675061">BBa_K2675061</A>.</P><P style="font-size:15px;">Fluorescence measurements of sfGFP in <EM>E. coli</EM> cells harbouring the 4 expression cassettes (<A href="http://parts.igem.org/Part:BBa_K2675050">BBa_K2675050</A>, <A href="http://parts.igem.org/Part:BBa_K2675051">BBa_K2675051</A>, <A href="http://parts.igem.org/Part:BBa_K2675060">BBa_K2675060</A> and <A href="http://parts.igem.org/Part:BBa_K2675061">BBa_K2675061</A>) in both pSB1C3 and pSB4K5 backbones were unable to detect any significant sfGFP expression (Figure 3). This observation was visually confirmed when bacteria were spotted on plates (Figure 4).
Even the presence of AimR did not have any effect on sfGFP expression from the pAimX(full) promoter, nor the pAimX(short) promoter (Figure 3). 
</P><P style="font-size:15px;">From these data, we conclude that pAimX(full) and pAimX(short) are not active promoters in <EM>E. coli</EM>. The reasons may be multiple: sub-optimal -35 and/ or -10 boxes sequences, sub-optimal distance between the consensus sequence motifs (typically 17+/-1 bp), or very low basal promoter activity that cannot be enhanced by AimR since it does not function as an activator outside the native host (<EM>Bacillus</EM>).
</P><H2 style="font-weight:800; text-align:center;" id="basic_parts">THE ENGINEERED PROMOTERS</H2><P style="font-size:15px;">To improve the pAimx(full) and pAimX(short) promoter activity in <EM>E. coli</EM>, we decided to substitute the -35 and -10 boxes of these promoters with those from the <A href="http://parts.igem.org/Part:BBa_J23110">J23110</A> promoter (Figure 2).</P><P style="font-size:15px;"><A href="http://parts.igem.org/Part:BBa_J23110">J23110</A> is a constitutive promoter from the Anderson Library with a medium transcription strength compared to the other members of this promoter family. In our system, when using sfGFP or sfGFP-LVAtag (<A href="http://parts.igem.org/Part:BBa_K2675056">BBa_K2675056</A> and <A href="http://parts.igem.org/Part:BBa_K2675066">BBa_K2675066</A>) as reporter downstream of <A href="http://parts.igem.org/Part:BBa_J23110">J23110</A> promoter, an intense fluorescence was detected (Figures 3 and 4).</P><FIGURE><FIGCAPTION style="font-size:12px;font-family: 'Nunito', sans-serif;"><B>Figure 2</B>: pAimx(full)- and pAimX(short)-J23110 hybrid promoters. These mutated pAimX(full) and pAimx(short) promoters of phage phi3T AimX gene have modified -35 and -10 boxes. The mutated -35 and -10 boxes have the same sequences as <A href="http://parts.igem.org/Part:BBa_J23110">BBa_J23110</A> promoter. -35 boxes are in green, -10 boxes in red and the transcription start sites in blue and indicated with +1. The inverted repeat sequences potentially forming the stem-loop structure of a terminator are marked by open horizontal arrows.
</FIGCAPTION></FIGURE><P style="font-size:15px;">The results presented in Figures 3 and 4 show that the pAimx(full)- and pAimX(short)-J23110 hybrid promoters are not active in <EM>E. coli</EM>. We were unable to detect any significant sfGFP expression regardless of the DNA copy number present in the cell (sfGFP on a high copy plasmid pSB1C3 or on a low copy plasmid pSB4K5), and of the presence of AimR expressed by <A href="http://parts.igem.org/Part:BBa_K2675040">BBa_K2675040</A> (from the high copy plasmid pSB1C3 or from the low copy plasmid pSB3T5).</P><FIGURE><FIGCAPTION style="font-size:12px;font-family: 'Nunito', sans-serif;"><B>Figure 3</B>: <EM>In vivo</EM> characterization of sfGFP expression by <EM>E. coli</EM> cells harbouring the expression cassettes of sfGFP and sfGFP-LVAtag under the control of pAimX(full) promoter (<A href="http://parts.igem.org/Part:BBa_K2675050">BBa_K2675050</A> and <A href="http://parts.igem.org/Part:BBa_K2675060">BBa_K2675060</A>), pAimX(short) promoter (<A href="http://parts.igem.org/Part:BBa_K2675051">BBa_K2675051</A> and <A href="http://parts.igem.org/Part:BBa_K2675061">BBa_K2675061</A>), pAimx(full-1)-J23110 hybrid promoter (<A href="http://parts.igem.org/Part:BBa_K2675053">BBa_K2675053</A> and <A href="http://parts.igem.org/Part:BBa_K2675063">BBa_K2675063</A>), pAimx(full-2)-J23110 hybrid promoter (<A href="http://parts.igem.org/Part:BBa_K2675054">BBa_K2675054</A> and <A href="http://parts.igem.org/Part:BBa_K2675064">BBa_K2675064</A>), pAimx(full-3)-J23110 hybrid promoter (<A href="http://parts.igem.org/Part:BBa_K2675055">BBa_K2675055</A> and <A href="http://parts.igem.org/Part:BBa_K2675065">BBa_K2675065</A>) and pAimX(short)-J23110 hybrid promoter (<A href="http://parts.igem.org/Part:BBa_K2675052">BBa_K2675052</A> and <A href="http://parts.igem.org/Part:BBa_K2675062">BBa_K2675062</A>) in pSB1C3 or pSB4K5 backbones. The negative control has been performed with an empty pSB1C3 or pSB3K4 and the positive control with <A href="http://parts.igem.org/Part:BBa_K2675056">BBa_K2675056</A> and <A href="http://parts.igem.org/Part:BBa_K2675066">BBa_K2675066</A>. The AimR was expressed as <A href="http://parts.igem.org/Part:BBa_K2675040">BBa_K2675040</A> on pSB3T5. Fluorescence values were normalised by OD<SUB>600nm</SUB>.
</FIGCAPTION></FIGURE><FIGURE><FIGCAPTION style="font-size:12px;font-family: 'Nunito', sans-serif;"><B>Figure 4</B>: Pictures of <EM>E. coli</EM> cells harbouring the expression cassettes of sfGFP and sfGFP-LVAtag under the control of pAimX(full) promoter (<A href="http://parts.igem.org/Part:BBa_K2675050">BBa_K2675050</A> and <A href="http://parts.igem.org/Part:BBa_K2675060">BBa_K2675060</A>), pAimX(short) promoter (<A href="http://parts.igem.org/Part:BBa_K2675051">BBa_K2675051</A> and <A href="http://parts.igem.org/Part:BBa_K2675061">BBa_K2675061</A>), pAimx(full-1)-J23110 hybrid promoter (<A href="http://parts.igem.org/Part:BBa_K2675053">BBa_K2675053</A> and <A href="http://parts.igem.org/Part:BBa_K2675063">BBa_K2675063</A>), pAimx(full-2)-J23110 hybrid promoter (<A href="http://parts.igem.org/Part:BBa_K2675054">BBa_K2675054</A> and <A href="http://parts.igem.org/Part:BBa_K2675064">BBa_K2675064</A>), pAimx(full-3)-J23110 hybrid promoter (<A href="http://parts.igem.org/Part:BBa_K2675055">BBa_K2675055</A> and <A href="http://parts.igem.org/Part:BBa_K2675065">BBa_K2675065</A>) and pAimX(short)-J23110 hybrid promoter (<A href="http://parts.igem.org/Part:BBa_K2675052">BBa_K2675052</A> and <A href="http://parts.igem.org/Part:BBa_K2675062">BBa_K2675062</A>) in pSB1C3 or pSB4K5 backbones. The negative control has been performed with an empty pSB1C3 or pSB3K4 and the positive one with <A href="http://parts.igem.org/Part:BBa_K2675056">BBa_K2675056</A> and <A href="http://parts.igem.org/Part:BBa_K2675066">BBa_K2675066</A>. In <A href="http://parts.igem.org/Part:BBa_K2675056">BBa_K2675056</A> and <A href="http://parts.igem.org/Part:BBa_K2675066">BBa_K2675066</A> sfGFP-LVAtag and sfGFP were equipped by a custom made RBS (<A href="http://parts.igem.org/Part:BBa_K2675017">BBa_K2675017</A>) and placed under the control of the constitutive promoter (<A href="http://parts.igem.org/Part:BBa_J23110">BBa_J23110</A>). The AimR was expressed as <A href="http://parts.igem.org/Part:BBa_K2675040">BBa_K2675040</A> from the high copy plasmid pSB1C3 or the low copy plasmid pSB3T5. We can clearly see on the pictures a strong sfGFP-LVAtag and sfGFP expression by <A href="http://parts.igem.org/Part:BBa_K2675056">BBa_K2675056</A> and <A href="http://parts.igem.org/Part:BBa_K2675066">BBa_K2675066</A> and no significant expression by all other constructs.
</FIGCAPTION></FIGURE><P style="font-size:15px;">This absence of sfGFP expression while using the hybrid pAimx(full)- and pAimX(short)-J23110 as promoters can be explained by the fact that -35 and -10 boxes are not the only elements of a promoter that can affect transcription initiation. For example, UP elements (sequence upstream of the -35 box), spacing between -35 and -10 boxes, non-template nucleotide at -5, spacing between the -10 box and the +1, or 5’UTR sequence, can also have significant effects on gene expression [5, 6].</P><P style="font-size:15px;">Studies showed that an A/T rich sequences upstream of the -35 box tend to enhance transcription for some promoters [7]. Since the transcription of AimX is probably initiated from the pAimX(3) putative promoter closest to the gene, we analysed the -60 to -40 sequence upstream of this promoter and found it to be only 50% A/T.</P><P style="font-size:15px;">Given that the +1 nucleotides represented in Figure 1 are software prediction, it is unclear if our substitutions were even at the right positions. The distance between the -10 box and the +1 nucleotide can also affect transcription levels. It has been showed that the preferred +1 nucleotide is typically located at 7-9 nucleotides downstream of the -7 base of the -10 box [8]. However, in our hybrid promoters, pAimx(full)- and pAimX(short)-J23110, the +1 nucleotide is only 6 bases downstream of the equivalent position in the -10 box.</P><H2 style="font-weight:800; text-align:center;" id="basic_parts">THE NATURAL PROMOTER(S) TURNED INTO AN OPERATOR</H2><P style="font-size:15px;">AimR was described to be a transcriptional regulator of the pAimX promoter in B. subtilis [1]. The exact binding site of AimR to DNA was not precisely described, but it was mapped by ChIPseq in the intergenic region between AimP and AimX genes, a.k.a. the pAimX(full) promoter (<A href="http://parts.igem.org/Part:BBa_K2675020">BBa_K2675020</A>). </P><P style="font-size:15px;">pAimX(full) and pAimX(short) sequences, as well as their hybrid versions, proved not to be active promoters in <EM>E. coli</EM>. However, assuming that AimR retains its binding ability to its specific target sequence in <EM>E. coli</EM>, we considered converting the pAimX into an operator for regulation of an <EM>E. coli</EM> promoter.</P><P style="font-size:15px;">To do this, we needed to place a constitutive <EM>E. coli</EM> promoter upstream of pAimX and a reporter gene downstream of pAimX. In this design, transcription will be initiated at the upstream constitutive <EM>E. coli</EM> promoter and, in the absence of AimR, the RNA polymerase should be able to transcribe the pAimX region, the RBS and the reporter sequence up to the terminator. In the presence of AimR, AimR will bind to its specific recognition sequence mapped in pAimX(full) region and thus it should prevent the RNA polymerase from continuing the transcription. The expected results are:
- in the absence of AimR: expression of the reporter gene
- in the presence of AimR: no expression of the reporter gene
</P><P style="font-size:15px;">Thus, AimR will be converted from an activator in <EM>Bacillus</EM> to a repressor in <EM>E. coli</EM>. 
We considered this activator-to-repressor functional conversion following discussions with several scientists who are used to working with the classical repressor/ inducer systems like LacI/ IPTG (see Integrated human practices).
</P><P style="font-size:15px;">We chose the <A href="http://parts.igem.org/Part:BBa_J23110">J23110</A> constitutive promoter from the Anderson Library that is reported to have a medium activity compared to the other members of this promoter family. A medium strength promoter allows more easily to detect variation in its activity (inhibition or activation) than a low or high strength promoter.</P><P style="font-size:15px;">Then we placed the pAimX(full) and pAimX(short) sequences (<A href="http://parts.igem.org/Part:BBa_K2675020">BBa_K2675020</A>&amp;<A href="http://parts.igem.org/Part:BBa_K2675021">BBa_K2675021</A>) between <A href="http://parts.igem.org/Part:BBa_J23110">J23110</A> and sfGFP-LVAtag (<A href="http://parts.igem.org/Part:BBa_K2675006">BBa_K2675006</A>), and we thus constructed 2 composite parts: <A href="http://parts.igem.org/Part:BBa_K2675057">BBa_K2675057</A> and <A href="http://parts.igem.org/Part:BBa_K2675058">BBa_K2675058</A> respectively.
During the cloning process, we selected a colony that, upon sequencing, revealed to have a truncated version of the J23110 with only the -10 box (<A href="http://parts.igem.org/Part:BBa_K2675026">BBa_K2675026</A>) upstream of pAimX(short). As this colony was green under blue light (with amber filter) due to sfGFP expression, we continued characterising this cloning artefact for its interesting promoter activity (<A href="http://parts.igem.org/Part:BBa_K2675059">BBa_K2675059</A>).</P><P style="font-size:15px;">The results presented in Figures 5 and 6 show that the pAimX(short), but not pAimX(full) acts as expected as an operator for <A href="http://parts.igem.org/Part:BBa_J23110">J23110</A> promoter in <EM>E. coli</EM>. Indeed, we can observe the fluorescent sfGFP expressed when this reporter gene is placed downstream of the composite promoter J23110 + pAimX(short) (<A href="http://parts.igem.org/Part:BBa_K2675058">BBa_K2675058</A>). Moreover, the fluorescence decreases in the presence of AimR suggesting that AimR acts as predicted for this design: as a repressor. The same trend is observed also when the truncated version of J23110 in present upstream of pAimX(short) (<A href="http://parts.igem.org/Part:BBa_K2675059">BBa_K2675059</A>).</P><P style="font-size:15px;">However, no significant sfGFP expression was observed when the pAimX(full) sequence was inserted between between the J23110 promoter and sfGFP (<A href="http://parts.igem.org/Part:BBa_K2675059">BBa_K2675059</A>). One plausible explanation is that transcription is initiated from the upstream <A href="http://parts.igem.org/Part:BBa_J23110">J23110</A> promoter, but it ends prematurely at an internal terminator predicted to exist in the pAimX(full) sequence (see arrows marking a hairpin loop in Figure 1). In addition to proving that AimR can act as an effective repressor in <EM>E. coli</EM>, these experiments help identify the precise site of AimR binding to DNA: the pAimX(short) promoter (<A href="http://parts.igem.org/Part:BBa_K2675021">BBa_K2675021</A>).</P><FIGURE><FIGCAPTION style="font-size:12px;font-family: 'Nunito', sans-serif;"><B>Figure 5</B>: <EM>In vivo</EM> characterization of sfGFP expression by <EM>E. coli</EM> cells harbouring the expression cassettes of sfGFP-LVAtag under the control of pAimX(full) promoter (<A href="http://parts.igem.org/Part:BBa_K2675050">BBa_K2675050</A>), pAimX(short) promoter (<A href="http://parts.igem.org/Part:BBa_K2675051">BBa_K2675051</A>), J23110 + pAimX(full) promoter (<A href="http://parts.igem.org/Part:BBa_K2675057">BBa_K2675057</A>), J23110 + pAimX(short) promoter (<A href="http://parts.igem.org/Part:BBa_K2675058">BBa_K2675058</A>), J23110 truncated promoter (-10 only) + pAimX(short) promoter (<A href="http://parts.igem.org/Part:BBa_K2675059">BBa_K2675059</A>) in pSB4K5 backbone. The AimR was expressed as <A href="http://parts.igem.org/Part:BBa_K2675040">BBa_K2675040</A> on pSB3T5. An empty pSB3T5 served as negative control. Fluorescence values were normalised by OD<SUB>600nm</SUB>.
</FIGCAPTION></FIGURE><FIGURE><FIGCAPTION style="font-size:12px;font-family: 'Nunito', sans-serif;"><B>Figure 6</B>: Pictures of <EM>E. coli</EM> cells harbouring the expression cassettes of sfGFP-LVAtag under the control of pAimX(full) promoter (<A href="http://parts.igem.org/Part:BBa_K2675050">BBa_K2675050</A>), pAimX(short) promoter (<A href="http://parts.igem.org/Part:BBa_K2675051">BBa_K2675051</A>), J23110 + pAimX(full) promoter (<A href="http://parts.igem.org/Part:BBa_K2675057">BBa_K2675057</A>), J23110 + pAimX(short) promoter (<A href="http://parts.igem.org/Part:BBa_K2675058">BBa_K2675058</A>), J23110 truncated promoter (-10 only) + pAimX(short) promoter (<A href="http://parts.igem.org/Part:BBa_K2675059">BBa_K2675059</A>), J23110 promoter (<A href="http://parts.igem.org/Part:BBa_K2675056">BBa_K2675056</A>) in pSB1C3 or pSB4K5 backbones. The negative control has been performed with an empty pSB1C3 or pSB3K4 and the positive one with <A href="http://parts.igem.org/Part:BBa_K2675056">BBa_K2675056</A>.
</FIGCAPTION></FIGURE><P style="font-size:15px;">In <EM>Bacillus</EM> cells, AimR is inactivated upon binding of the ‘arbitrium’ hexapeptide SAIRGA [1]. To test if the repressor activity of AimR in <EM>E. coli</EM> is inhibited in the presence of the SAIRGA hexapeptide, we added in the culturing media increasing concentrations of the peptide. We can see on the graph presented in Figure 7 that fluorescence intensity and SAIRGA concentration are not really correlated since increasing concentration of SAIRGA doesn’t influence fluorescence output, neither in the absence nor in the presence of AimR. Although the fold-repression (the ratio between the Fluorescence/OD<SUB>600nm</SUB> values of the samples without AimR over those with AimR) increases slightly with increasing SAIRGA concentration, the variance of the fluorescence data are too high to draw a clear conclusion (Figures 8).</P><P style="font-size:15px;">Instability of the SAIRGA in the medium or its inefficient import into <EM>E. coli</EM> may be other reasons why its effect on AimR is not obvious. The import of peptides in <EM>E. coli</EM> is dependent on the expression of transmembrane transporters in the cell plasma membrane. Considering that SAIRGA is a peptide signal, its import may be realised through the oligopeptide permease system (Opp) that is an ABC transporter capable of importing large oligopeptides into bacteria without selection for size, composition, sequence or charge [9]. The Opp system can be induced by supplementing the media with L-Leucine [9]. So, we decided to test the effect of SAIRGA peptide in culturing media supplemented with this amino acid.</P><P style="font-size:15px;">We also tested the effect of CaCl<SUB>2</SUB>, know to have a permeabilizing effect on <EM>E. coli</EM> plasma membrane.</P><P style="font-size:15px;">The results presented in Figure 9 are encouraging : the represion fold (the ratio between the Fluorescence/OD<SUB>600nm</SUB> values of the samples without AimR over those with AimR) is slightly increasing with increasing SAIRGA concentration.</P><FIGURE><FIGCAPTION style="font-size:12px;font-family: 'Nunito', sans-serif;"><B>Figure 7</B>: <EM>In vivo</EM> characterization of sfGFP expression by <EM>E. coli</EM> cells harbouring the expression cassettes of sfGFP-LVAtag under the control of J23110 + pAimX(short) promoter (<A href="http://parts.igem.org/Part:BBa_K2675058">BBa_K2675058</A>) and J23110 truncated promoter (-10 only) + pAimX(short) promoter (<A href="http://parts.igem.org/Part:BBa_K2675059">BBa_K2675059</A>) in pSB4K5 backbone in the presence of increasing concentrations of SAIRGA hexapeptide in the culturing medium. The AimR was expressed as <A href="http://parts.igem.org/Part:BBa_K2675040">BBa_K2675040</A> on pSB3T5. An empty pSB3T5 served as negative control. Fluorescence values were normalised by OD<SUB>600nm</SUB>.
</FIGCAPTION></FIGURE><FIGURE><FIGCAPTION style="font-size:12px;font-family: 'Nunito', sans-serif;"><B>Figure 8</B>: <EM>In vivo</EM> characterization of sfGFP expression by <EM>E. coli</EM> cells harbouring the expression cassettes of sfGFP-LVAtag under the control of J23110 + pAimX(short) promoter (<A href="http://parts.igem.org/Part:BBa_K2675058">BBa_K2675058</A>) and J23110 truncated promoter (-10 only) + pAimX(short) promoter (<A href="http://parts.igem.org/Part:BBa_K2675059">BBa_K2675059</A>) in pSB4K5 backbone in the presence of increasing concentrations of SAIRGA hexapeptide in the culturing medium. The AimR was expressed as <A href="http://parts.igem.org/Part:BBa_K2675040">BBa_K2675040</A> on pSB3T5. An empty pSB3T5 served as negative control. The results are present as the ratio between the Fluorescence/OD<SUB>600nm</SUB> values of the samples without AimR over those with AimR (represion fold).
</FIGCAPTION></FIGURE><FIGURE><FIGCAPTION style="font-size:12px;font-family: 'Nunito', sans-serif;"><B>Figure 9</B>: <EM>In vivo</EM> characterization of sfGFP expression by <EM>E. coli</EM> cells harbouring the expression cassettes of sfGFP-LVAtag under the control of J23110 + pAimX(short) promoter (<A href="http://parts.igem.org/Part:BBa_K2675058">BBa_K2675058</A>) and J23110 truncated promoter (-10 only) + pAimX(short) promoter (<A href="http://parts.igem.org/Part:BBa_K2675059">BBa_K2675059</A>) in pSB4K5 backbone in the presence of increasing concentrations of SAIRGA hexapeptide in the culturing medium supplemented or ot with 1 mM L-Leucine or 0.1 mM of CaCl<SUB>2</SUB>. The AimR was expressed as <A href="http://parts.igem.org/Part:BBa_K2675040">BBa_K2675040</A> on pSB3T5. An empty pSB3T5 served as negative control. The results are present as the ratio between the Fluorescence/OD<SUB>600nm</SUB> values of the samples without AimR over those with AimR (represion fold).
</FIGCAPTION></FIGURE><P style="font-size:15px;">As the effect of externally added SAIRGA peptide was not very convincing, we tried expressing the SAIRGA peptide directly inside the cell. The results presented in Figure 10 do not show any effect of the co-expression, most probably because the SAIRGA peptide is expressed with specific secretion and protease-cleavage tags that remain unprocessed inside the cell.</P><FIGURE><FIGCAPTION style="font-size:12px;font-family: 'Nunito', sans-serif;"><B>Figure 10</B>: <EM>In vivo</EM> characterization of sfGFP expression by <EM>E. coli</EM> cells harbouring the expression cassette of sfGFP-LVAtag under the control of J23110 + pAimX(short) promoter (<A href="http://parts.igem.org/Part:BBa_K2675058">BBa_K2675058</A>) in pSB4K5 backbone in the presence of various SAIRGA expression plasmids (<A href="http://parts.igem.org/Part:BBa_K2675041">BBa_K2675041</A>, <A href="http://parts.igem.org/Part:BBa_K2675042">BBa_K2675042</A>, <A href="http://parts.igem.org/Part:BBa_K2675043">BBa_K2675043</A> and <A href="http://parts.igem.org/Part:BBa_K2675044">BBa_K2675044</A>) in pSB4K5 backbone. The AimR was expressed as <A href="http://parts.igem.org/Part:BBa_K2675040">BBa_K2675040</A> on pSB3T5. Empty pSB1C3, pSB3T5 and pSB4K5 served as negative controls. Fluorescence values were normalised by OD<SUB>600nm</SUB>.
</FIGCAPTION></FIGURE><H2 style="font-weight:800; text-align:center;" id="basic_parts">THE INTERNAL TERMINATOR</H2><P style="font-size:15px;">Sequence analysis of the pAimX(full) promoter (<A href="http://parts.igem.org/Part:BBa_K2675020">BBa_K2675020</A>) revealed the presence of an inverted repeat sequence potentially forming the stem-loop structure of a terminator (Figure 1).
The results presented above in Figures 5 and 6 show that no fluorescent sfGFP is expressed when pAimX(full) sequence is inserted between between a constitutive promoter (<A href="http://parts.igem.org/Part:BBa_J23110">J23110</A>) and the reporter gene sfGFP-LVAtag (<A href="http://parts.igem.org/Part:BBa_K2675059">BBa_K2675059</A>).
</P><P style="font-size:15px;">To test if this predicted terminator is indeed responsible for this effect, we randomised the left arm of the terminator stem or the poly-U tail as two different strategies of disrupting the predicted terminator. Since intrinsic termination depends on both a strong hairpin and a downstream polyU signal [10], we expect that disrupting either of them should restore fluorescence from these constructs. From the library we generated, we selected 8 colonies that appeared to be fluorescent on plates.</P><P style="font-size:15px;">Sequencing revealed various mutations and insertions in the terminator region (Figure 11) that lead to various degrees of sfGFP expression initiated at the upstream <A href="http://parts.igem.org/Part:BBa_J23110">J23110</A> promoter (Figures 12 and 13). Terminator versions 1, 2, 7 and 8 where termination efficiency is most severely reduced were designed to disrupt the polyU tail, while versions 3 to 6 were designed to disrupt the hairpin loop. These results suggest that the predicted hairpin loop in the pAimX(full) promoter is indeed an active terminator in <EM>E. coli</EM>, and possibly also in <EM>Bacillus</EM>. If the predicted hairpin in the pAimX(full) sequence acts like an intrinsic terminator in <EM>Bacillus</EM> it is possible that the activating function of AimR is in fact an antitermination function, consistent with other phages where the antitermination mechanism is used for infection cycle regulation [11]. That said, the fact that in our experiments expression from the composite promoter J23110 + pAimX(short) is regulated by AimR (Figures 5 and 6), while that from J23110 + pAimX(full) is not, rules out an antitermination mechanism for regulation by AimR at least in <EM>E. coli</EM>.</P><FIGURE><FIGCAPTION style="font-size:12px;font-family: 'Nunito', sans-serif;"><B>Figure 11</B>: pAimX(full) promoters of phage phi3T with mutations in the putative terminator. These mutations disrupted also the pAimX(2) and pAimX(3) putative promoters. -35 boxes are in green, -10 boxes in red and the transcription start sites in blue and indicated with + 1. The inverted repeat sequences potentially forming the stem-loop structure of a terminator are marked by open horizontal arrows.
</FIGCAPTION></FIGURE><FIGURE><FIGCAPTION style="font-size:12px;font-family: 'Nunito', sans-serif;"><B>Figure 12</B>: <EM>In vivo</EM> characterization of sfGFP expression by <EM>E. coli</EM> cells harbouring the expression cassettes of sfGFP-LVAtag under the control of the composite promoter J23110 + pAimX(full) (<A href="http://parts.igem.org/Part:BBa_K2675057">BBa_K2675057</A>), and its mutants where the internal terminator has been disrupted  (<A href="http://parts.igem.org/Part:BBa_K2675190">BBa_K2675190</A> to <A href="http://parts.igem.org/Part:BBa_K2675197">BBa_K2675197</A>). Fluorescence values were normalised by OD<SUB>600nm</SUB>.
</FIGCAPTION></FIGURE><FIGURE><FIGCAPTION style="font-size:12px;font-family: 'Nunito', sans-serif;"><B>Figure 13</B>: Pictures of <EM>E. coli</EM> cells harbouring the expression cassettes of sfGFP-LVAtag under the control of pAimX(full) promoter (<A href="http://parts.igem.org/Part:BBa_K2675050">BBa_K2675050</A>), pAimX(short) promoter (<A href="http://parts.igem.org/Part:BBa_K2675051">BBa_K2675051</A>), J23110 + pAimX(full) promoter (<A href="http://parts.igem.org/Part:BBa_K2675057">BBa_K2675057</A>), J23110 + pAimX(short) promoter (<A href="http://parts.igem.org/Part:BBa_K2675058">BBa_K2675058</A>), J23110 truncated promoter (-10 only) + pAimX(short) promoter (<A href="http://parts.igem.org/Part:BBa_K2675059">BBa_K2675059</A>) and a series of eight pAimX(full) derived promoters with with mutations in the putative terminator (<A href="http://parts.igem.org/Part:BBa_K2675190">BBa_K2675190</A> to <A href="http://parts.igem.org/Part:BBa_K2675197">BBa_K2675197</A>).
</FIGCAPTION></FIGURE><H2 style="font-weight:800; text-align:center;" id="basic_parts">CONCLUSIONS</H2><P style="font-size:15px;">We have successfully transferred the ‘Arbitrium’ communication system from a <EM>Bacillus</EM> phage to <EM>E. coli</EM>. While doing so, we have engineered new “composite” promoters that convert the transcriptional activator AimR from <EM>Bacillus</EM> into a repressor in <EM>E. coli</EM>. We have precisely identified the  AimR binding region in the pAimX(full) promoter. We have also identified a putative intrinsic terminator in the pAimX(full) promoter sequence, and verified its terminator activity in E. coli. Further, we have ruled out a transcription antitermination mechanism at this terminator site. We also have promising data that indicates conditional regulation of AimR repressor activity by the small signalling peptide SAIRGA.</P><H2 class="anchor" style="font-weight:800; text-align:center;" id="references">REFERENCES</H2><P style="font-size:15px;" class="bibliographie">[1] Erez Z, Steinberger-Levy I, Shamir M, Doron S, Stokar-Avihail A, Peleg Y, Melamed S, Leavitt A, Savidor A, Albeck S, Amitai G, Sorek R. Communication between viruses guides lysis-lysogeny decisions. Nature (2017) 541, 488-493.</P><P style="font-size:15px;" class="bibliographie">[2] Solovyev V, Salamov A. Automatic Annotation of Microbial Genomes and Metagenomic Sequences. In Metagenomics and its Applications in Agriculture, Biomedicine and Environmental Studies (Ed. R.W. Li), Nova Science Publishers (2011) p. 61-78.</P><P style="font-size:15px;" class="bibliographie">[3] Gautheret D, Lambert A. Direct RNA motif definition and identification from multiple sequence alignments using secondary structure profiles. J Mol Biol (2001) 313, 1003-1011.</P><P style="font-size:15px;" class="bibliographie">[4] Macke T, Ecker D, Gutell R, Gautheret D, Case DA, Sampath R. RNAMotif – A new RNA secondary structure definition and discovery algorithm. Nucleic Acids Res (2001) 29, 4724–4735.</P><P style="font-size:15px;" class="bibliographie">[5] Hook-Barnard IG, Hinton DM. Transcription initiation by mix and match elements: flexibility for polymerase binding to bacterial promoters. Gene Regul Syst Bio (2007) 1, 275-293.</P><P style="font-size:15px;" class="bibliographie">[6] Mutalik VK, Guimaraes JC, Cambray G, Mai QA, Christoffersen MJ, Martin L, Yu A, Lam C, Rodriguez C, Bennett G, Keasling JD, Endy D, Arkin AP. Quantitative estimation of activity and quality for collections of functional genetic elements. Nat Methods (2013) 10, 347-353.</P><P style="font-size:15px;" class="bibliographie">[7] Chan B, Busby S. Recognition of nucleotide sequences at the Escherichia coli galactose operon P1 promoter by RNA polymerase. Gene (1989) 84, 227-236.</P><P style="font-size:15px;" class="bibliographie">[8] Liu J, Turnbough CL Jr. Effects of transcriptional start site sequence and position on nucleotide-sensitive selection of alternative start sites at the pyrC promoter in Escherichia coli. J Bacteriol (1994) 176, 2938-2945.</P><P style="font-size:15px;" class="bibliographie">[9] Maio A, Brandi L, Donadio S, Gualerzi CO. The Oligopeptide Permease Opp Mediates Illicit Transport of the Bacterial P-site Decoding Inhibitor GE81112. Antibiotics (Basel) (2016) 5, 17.</P><P style="font-size:15px;" class="bibliographie">[10] Chen YJ, Liu P, Nielsen AA, Brophy JA, Clancy K, Peterson T, Voigt CA. Characterization of 582 natural and synthetic terminators and quantification of their design constraints. Nat Methods (2013) 10, 659-664.</P><P style="font-size:15px;" class="bibliographie">[11] Herskowitz I, Hagen D. The lysis-lysogeny decision of phage lambda: explicit programming and responsiveness. Annu Rev Genet (1980) 14, 399-445.</P></DIV><DIV class="container"><DIV class="row"><TABLE class="direction" style="border: none; width:80%;"><TBODY><TR><TH><A href="https://2018.igem.org/Team:Evry_Paris-Saclay/Model" style="color:black;">Previous</A></TH><TH><A href="https://2018.igem.org/Team:Evry_Paris-Saclay/Futur_perspective" style="color:black;">Next</A></TH></TR></TBODY></TABLE></DIV></DIV></DIV></DIV></DIV></DIV></DIV></BODY></HTML>