<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_OUC-China_Interlab skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:OUC-China/Interlab</SPAN></H1><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><P>&lt;!DOCTYPE html&gt;<TITLE>Team:OUC-China Member</TITLE></P><DIV class="container head-container"><SPAN class="menu"> Menu</SPAN><DIV class="banner-top"><UL class="nav banner-nav"><LI><A href="https://2015.igem.org/Team:OUC-China">HOME</A></LI><LI class="dropdown1"><A class="down-scroll" href="#">PROJECT</A></LI><LI><A href="https://2015.igem.org/Team:OUC-China/Project/Overview">Overview</A></LI><LI><A href="https://2015.igem.org/Team:OUC-China/Project/Magnetic_Receiver">Magnetic Receiver</A></LI><LI><A href="https://2015.igem.org/Team:OUC-China/Project/Thermosensitive_Regulator">Thermosensitive Regulator</A></LI><LI><A href="https://2015.igem.org/Team:OUC-China/Project/Captor">Captor</A></LI><LI><A href="https://2015.igem.org/Team:OUC-China/Project/Results">Results</A></LI><LI><A href="https://2015.igem.org/Team:OUC-China/Parts">Parts</A></LI><LI><A href="https://2015.igem.org/Team:OUC-China/Project/Achievements">Achievements</A></LI></UL><LI class="dropdown1"><A href="https://2015.igem.org/Team:OUC-China/Interlab">INTERLAB</A></LI><LI class="dropdown1"><A class="down-scroll" href="#">MODELING</A></LI><LI><A href="https://2015.igem.org/Team:OUC-China/Modeling">Overview</A></LI><LI><A href="https://2015.igem.org/Team:OUC-China/Modeling/Magnetic_Receiver">Magnetic Receiver</A></LI><LI><A href="https://2015.igem.org/Team:OUC-China/Modeling/Heating">Heating</A></LI><LI><A href="https://2015.igem.org/Team:OUC-China/Modeling/Thermosensitive_Regulator">Thermosensitive Regulator</A></LI><LI class="dropdown1"><A class="down-scroll" href="#">HUMAN PRACTISE</A></LI><LI><A href="https://2015.igem.org/Team:OUC-China/Practices">Overview</A></LI><LI><A href="https://2015.igem.org/Team:OUC-China/Human_Practice/Safety">Safety</A></LI><LI><A href="https://2015.igem.org/Team:OUC-China/Human_Practice/Education">Education</A></LI><LI><A href="https://2015.igem.org/Team:OUC-China/Human_Practice/Transportation">Transportation</A></LI><LI><A href="https://2015.igem.org/Team:OUC-China/Human_Practice/Communication">Communication</A></LI><LI><A href="https://2015.igem.org/Team:OUC-China/Human_Practice/Guide_Of_Logo_Design">Guide of Logo Design</A></LI><LI class="dropdown1"><A class="down-scroll" href="#">NOTE BOOK</A></LI><LI><A href="https://2015.igem.org/Team:OUC-China/Note_Book/Lab_Journal">Lab Journal</A></LI><LI><A href="https://2015.igem.org/Team:OUC-China/Note_Book/Protocol">Protocol</A></LI><LI class="dropdown1"><A class="down-scroll" href="#">TEAM</A></LI><LI><A href="https://2015.igem.org/Team:OUC-China/Team/Member">Member</A></LI><LI><A href="https://2015.igem.org/Team:OUC-China/Attributions">Attribution</A></LI><LI><A href="https://2015.igem.org/Team:OUC-China/Collaborations">Collaboration</A></LI></DIV></DIV><DIV class="container-fluid member-head"><DIV class="container"><H2 class="text-center">InterLab</H2></DIV></DIV><DIV class="container-fluid"><DIV class="text-content container" id="text-content"><ARTICLE><H2>InterLab Study</H2><P>
                    It is vital but difficult to characterize a biological part in Synthetic Biology. A series of well characterized parts can be very useful in design of new systems and prediction of function, while comparing the data from different labs could be a challenge. The InterLab Study provides us a chance to share the analyses of data set varying from team to team, whose goal is to obtain fluorescence data of three specific genetic devices.
                </P><H2>Overview</H2><P>
                    This is the first time that we took part in InterLab Study. We successfully measured fluorescence from three devices for GFP expression, and got the date as expected. At the very beginning, the fluorescence of GFP was observed with our self-made blue light generator whose function was integrated into Captor for convenience. Then, we measured at least three technical replicates and three biological replicates with plate reader and flow cytometry for extra credit. In addition, we tried to obtain data of these devices with Raman Spectrometer and gain some experience we would like to share here.And we inserted Ribo J between promoter and RBS in order to measure in a more accurate way.
                </P><H2>Achievements</H2><P>
                    Fluorescence was measured with our cheap and convenient self-made Captor.Fluorescence measurements were obtained from a plate reader in absolute units.Flow cytometer data was used to explore instantcell-to-cell variation as part of the extra credit assignment relative to standard beads.Ribo J data (added between the promoter and RBS) was used to reduce the influence of the genetic context.A new measurement of Raman Spectrometer was first introduced in InterLab Study, and some experience we gained was shared here.The three required promoters was characterized and submitted to the registry.We used our <A href="https://2015.igem.org/Team:OUC-China/Project/Captor" style="color:#0000FF;text-decoration: underline;">Captor</A> to have a qualitative observation of the expression of GFP as expected and provided a easy but great image analysis method.
                </P><H2>Cloning</H2><H3>Devices</H3><P>
                    Device 1: J23101 + I13504 (B0034-E0040-B0015)Device 2: J23106 + I13504 (B0034-E0040-B0015)Device 3: J23117 + I13504 (B0034-E0040-B0015)Device 4: J23101 + Ribo J + I13504 (B0034-E0040-B0015)Device 5: J23106 + Ribo J + I13504 (B0034-E0040-B0015)Device 6: J23117 + Ribo J + I13504 (B0034-E0040-B0015)Positive Control: BBa_I0270Negative Control 1: E. coli K-12 DH5-alpha without plasmidNegative Control 2: BBa_R0040 (pTetR)
                </P><H3>DNA Assembly Method</H3><P>We used 3A Assembly to assemble parts.</P><H3>Protocol</H3><P>Following the protocol we used in <A href="https://2015.igem.org/Team:OUC-China/Note_Book/Protocol" style="color:#0000FF;text-decoration: underline;">Notebook</A>.</P><H3>Validation</H3><P>All of our devices’ sequencing results were validated by GENEWIZ.</P><H2>Growing Cells for Measurement</H2><H3>Type of Agar</H3><P>
                    AmpicillinStock: 100mg/ml in 50% ethanolWorking: 100ug/ml
                </P><P>
                    ChloramphenicolStock: 35mg/ml in 100% ethanolWorking: 35ug/ml
                </P><H3>Equipment and Parameter</H3><P>
                    Incubator: HZQ-QSmart Incubator, Harbin Donglian Electronic Technology Development CO. Ltd.Diameter:26mmSpeed: 210rpmTemperature: 37 degree centigradeDimensions of Test Tube: 12 mLVolume of Test Tubes: 10 mL
                </P><H3>Protocol</H3><P>
                    1. Streak out 1 plate per device and control2. Incubate plates overnight (18 hours later, individual colonies are clearly visible) at 37 degree centigrade3. Inoculate liquid culture with experimental devices and controls4. Incubate liquid cultures5. Incubate for 12 hours (37 degree centigrade, 220 rpm)
                </P><H2>Plate Reader Measurement</H2><H3>Equipment</H3><P>
                    Varioskan Flash Multimode ReaderThermo ScientificRead Speed: 6.4 well per second
                </P><H3>Software</H3><P>
                    Run Software Version: Skanlt Software 2.4.5 RE for Varioskan FlashCurrent Software Version: Skanlt Software 2.4.5 RE for Varioskan Flash
                </P><H3>Protocol</H3><P>
                    Date: 2015 Aug 121. Set our instrument to read OD6002. Setup a 96-well plate with our cultures3. Take the measurement and record it4. Calculate the dilution required for each sample (OD600=0.5)5. Dilute each sample6. Remeasure our sample on OD6007. Recalculate our dilution and remeasure until it's within 5% of 0.5.
                </P><H3>Layout</H3><P>
                    101: J23101+I13504; 106: J23106+I13504; 117: J23117+I13504; 117R: J23101+Ribo J+I13504; D: E. coli K-12 DH5α without plasmid; R: R0040; I: I0270; 1: Technical replicate 1; 2: Technical replicate 2; 3: Technical replicate 3; A:Biological replicate 1; B: Biological replicate 2; C: Biological replicate 3; M9: M9 liquid media; \: Blank control'SF: Sodium Fluorescein (concentration of 0, 10, 20, 40, 80, 160, 320, 640 ng/mL)
                </P><H3>Unit</H3><P>
                    Three technical replicates of M9 liquid media was measured, which were called background value.A series of concentration of sodium fluorescein was measured, which are 0, 10, 20, 40, 80, 160, 320, 640 ng/mL . A calibration curve was defined.
                </P><H3>Dataset</H3><P>
                    All samples were cut the mean of background value, and then compared with the calibration curve to get the final dataset in units of fluorescein.
                </P><P>TR: Technical ReplicateBR: Biological Replicate</P><H2>Flow Cytometer Measurement</H2><H3>Equipment</H3><P>Epics-XLBeckman Coulter</P><H3>Protocol</H3><P>
                    Date: 2015 Aug 121. Start up the instrument and warm up it until it shows “Awaiting Sample”2. Calibrate the instrument with check beads3. Mix the fluorescent beads and negative sample (E coli. without plasmid) and dilute it to 1 mL4. Measure the mix and regulate voltage until the points appear in the appropriate location and set gates</P><P>5. Measure the other samples in three biological replicates and three technical replicates6. Use the Cleanase and ddH2O to clean the instrument as instruction7. Process the data
                </P><H3>Unit</H3><P>
                    The fluorescence of GFPwas compared with the 2.00μm Fluoresbrite Yellow Green (YG) Microspheres.
                </P><H3>Dataset</H3><H2>Raman Spectrometer</H2><H3>Overview</H3><P>
                    Raman spectra is a kind of scattered spectrums, which can show the quality of different chemical bonds of a molecule. In other words, we can gain the expression of GFP straightly through the quality of its chemical bonds instead of the fluorescence of it under exciting light. We introduced this technology into InterLab Study for the first time, which could be a potential way of measuring GFP.
                </P><H3>Equipment</H3><P>
                    Raman Spectrometer in Qingdao Institute of Bioenergy and Bioprocess Technology (QIBEBT), Chinese Academy of Sciences (CAS)
                </P><H3>Protocol</H3><P>
                    Date: 2015 Aug 211. Grow the samples in LB liquid media with the appropriate antibiotic2. When OD=0.5, scour off the LB with 0.85% normal saline three times3. Use capillary to load the culture4. Measure the GFP with Raman Spectrometer
                </P><H3>Dataset</H3><P>
                    One of the challenges of using Raman spectroscopy for biological applications is the inherent fluorescence generated by many biological molecules that underlies the measured spectra. As a matter of fact, we cannot tell the GFP apart from total proteins. Here's our result.
                </P><P>
                    The first step of data processing is choosing an internal referencesetting the baseline，which means removing.the gently varied cruve and aligning the bottom of peak.The second step is substracting background.The third step is normalizing. We hypothesized the amount of substance at focus points of laser are the same. We found the peak of phenylalanine are similar. So we treat phenylalanine as an internal reference because of the following reasons：1. The peak of phenylalanine (1001/cm) is clear.2. Phenylanine is a kind of aromatic amino acids which was synthesized through specific metabolic pathways after the populist acid synthesis. The production of phenylalanine is stable in our samples because there is no circuit having obvious influence on the mentioned pathways.3. Phenylalanine is a characteristic component of living things, which can help us tell cells from the samples.
                </P><P>Interestingly, we can see from the figure that the expression of GFP is roughly related to the strength of promoter through the result from plate reader. We thought it is highly likely that the difference of total protein intensity is caused by the GFP expression. Though the data is not ideal, we still think it is a potential way to measure the specific protein, especially when the specific protein has no fluorescence. The Raman Spectrometer is a convenient equipment that can measure the sample without any other dye, just a database of Raman spectrum of millions substances.
                </P><P>
                    Because of the busyness of our main project, we didn't repeat this measurement. But we have a summary of its announcements and this may help following teams.1. Dry the sample rinsed by ddH2O on a CaF2 plate before measurement in order to reduce the noise.2. Focusing on the cells is a hard job which may lead to an inaccurate measuring result.3. It needs a standard protein sample to gain the absolute results.
                </P><H2>Captor</H2><H3>Protocol</H3><P>
1. Prepare a 6-well plate containing 2mL LB agar per well with appropriate concentration of antibiotics; 
2. Add 20μL culture of target E. coli in the center of the agar, whose OD value is about 0.5, into each well; 
3. Move it into constant temperature incubator (37°C) carefully. 
4. Incubate it overnight. 
</P><H3>Dataset</H3><P>We found that it is hard to take fluorescent photo by Digital Single Lens Reflex because of the blue light.</P><P>To solve this problem, we use photoshop to process the image. We adjust the RGB parameter to make it in low blue value and high green value. Then we enhanced contrast ratio and luminance. Finally, we obtained the following image, which can be easily seen the green circles. The extent of the green is consist with the promoter strength. 
</P><DIV style="width:60%;margin:0px auto;"><P>
As we can see from the picture A we took with a digital camera, the fluorescence of is not clear.
In order to solve this problem, we used photoshop to edit the picture by just adjusting the ratio of RGB (low Blue and high Green), enhanceing luminance and contrast ratio. The picture B was processed as this method and the result is clear and consist with the expectation.
</P><H2>Tips</H2><H3>Reduce Auto-fluorescence</H3><P>
                    Before the measurement, we exposed the samples under an incandescent light bulb for 1 min. Here we provided a universal method to decrease the errors in GFP measurement. As we all know, one big problem in measurement is the high background value of fluorescence, which causes results to be inaccurate. A main source of background is the auto-fluorescence from the samples. When the expression of GFP is low. The auto-fluorescence can shield the target signal. Because the auto-fluorescence can quench quickly under wide-spectrum visible light, while the fluorescence of GFP will not be influenced.
                </P><H3>Reduce Material Fluorescence</H3><P>
                    Another way to reduce background value is reducing material fluorescence. We chose the 96-well assay plate in black and clear bottom, which is an advice for other teams who want to measure the fluorescence.
                </P><H3>M9 Medium</H3><P>
                    The LB medium is a common medium used to grow E. coli. But the faint yellow background can be a severe interference when measuring fluorescence. M9 medium is a kind of colorless medium that can avoid the problem. Particularly, we added casein enzymatic hydrolysates to provide it with enough amino acid. 
                </P><H3>Ribo J</H3><P>
                    Promoter parts are often defined by a relatively short (~50 bp) sequence, but regions 100 bp or more upstream can affect promoter strength, and the effect of remote sequences can be reduced by including an insulator region.Ribo J is a kind of ribozyme-based insulator part that can buffer synthetic circuits from genetic context. It is a useful tool for measuring the strength of promoters which can make the disruption from genetic context to be reduced.As a matter of fact, our results from plate reader and flow cytometer show that the expression of GFP under J23117 promoter have a little difference with or without Ribo J. It proved that the Ribo J did has the function of buffering.
                </P><H2>Safety</H2><P>
                    E. coli K-12 DH5-alpha is the chassis we used to conduct the InterLab Study, whose biosafety level is BSL 1.Our Personal Protective Equipment (PPE) includes disposable latex gloves, disposable PE gloves, lab coats, long pants, disposable sterile masks, shoe covers, etc.Any operation of our experiments abide by the principles mentioned in the Safety page of our Wiki.
                </P><H2>Problems &amp; Challenges</H2><H3>We Got False Products of I13504 through PCR</H3><P>
                    When constructing our devices, we used PCR to amplify the fragment of part I13504. We use VF2 as forward primer, VR as reserve primer, r Taq (Takara, Dalian) as DNA Polymerase, the pure plasmid (pSB1C3) of I13504 as template in our PCR system. The annealing temperature is 55 degree centigrade, extension time is 1 minute and cycle times is 35. Then, we used XbaI (NEB) and PstⅠ(NEB) to digest the PCR product and use SanPrep Column PCR Product Purification Kit(Sangon Biotech, Shanghai) to gain enzyme-digested product. After ligation, we transformed DNA into DH5-alpha. But the colonies didn’t turn green at all after 1.5d under the condition of LB and 37 degree centigrade. According to the sequencing data (Genewiz, Beijing), the product was not as expected, with right promoter sequence and wrong GFP sequence.As a response, we speculated the false priming site causing it. So, we extracted the plasmid contained the part and then digested the plasmid instead of amplifying by PCR. Also, we strongly recommend the strength of promoter or RBS in a few constructed circuits should be tested.
                </P><H2>Responsible Person</H2><P>
                    Creating the Devices: Qikai Qin ,Jinyang LiangMeasuring &amp; Data Processing: Qikai QinTechnical Support of Raman Spectrometer: Yetian SuTechnical Support of Flow Cytometer: Yu Ma
                </P><DIV style="background-color:#f2f2f2"><H2>References</H2><P style="font-size:13px;">
                            [1] Lou C, Stanton B, Chen Y J, et al. Ribozyme-based insulator parts buffer synthetic circuits from genetic context[J]. Nature biotechnology, 2012, 30(11): 1137-1142.[2] Zhang P, Ren L, Zhang X, et al. Raman-Activated Cell Sorting Based on Dielectrophoretic Single-Cell Trap and Release[J]. Analytical chemistry, 2015, 87(4): 2282-2289.
                        </P></DIV></DIV></ARTICLE></DIV></DIV><FOOTER><DIV class="container"><ADDRESS class="center-block">
                Follow us on Facebook@ <A href="https://www.facebook.com/oucigem">iGEM OUC</A>Mail to: <A href="mailto:#">oucigem@163.com</A></ADDRESS></DIV></FOOTER></DIV></DIV></DIV></DIV></BODY></HTML>