Team:Vanderbilt/Background/Abstract
Abstract
Problem
Mutation
Project Description
Every Gene Harbors a Vulnerability
At the foundation of the field of synthetic biology lies a seemingly irreconcilable paradox. By applying our mastery over life’s genetic code, we aim to turn biological systems into devices that we can control and program in predictable ways. Yet the same genetic elements that we incorporate into our designs owe their existence to the unpredictability of evolution operating to mutate and change DNA.
Evolution and mutation work hand in hand to select against the maintenance of synthetic DNA sequences, taking the biological engineer’s best-formulated designs and eroding them to nothing. No matter if it is a complex gene circuit or single-gene expression, a single mutation is enough to render an entire system nonfunctional. As soon as a mutant cell emerges, its mutation frees it from the metabolic burden of sustaining transgenes, allowing it to outcompete all the remaining cells that are functioning as intended [1]. Within days the population ceases to be what it was engineered to be [2]. Nature defeats the engineer’s attempts at harnessing the vast potential for biological machines to be re-purposed as agents for good.
How to Tame Evolution
How is it even possible to combat processes as fundamental as mutation and natural selection? The dogma in biology has been that random mutation will strike any gene sequence and initiate a process of selection as a logical consequence of the variations that mutation introduces. But that basic dogma is wrong. As soon as we realize that subtle mistake, the way is opened to bring mutation itself under the synthetic biologist’s control.
Decades of genomics and biochemistry research has established that mutation is not truly random. Certain DNA sequences are “hotspots” more prone to mutation than others, while others are resistant against mutagenic damage [3]. Our idea is to rationally modulate the sequence composition of synthetic genes to reduce or eliminate motifs with high mutation risk, substituting them out for mutation-resistant sequences. When combined with gene synthetic technologies, our process becomes a simple and reliable optimization that is universally applicable to genes expressed in any organism
An Algorithmic Approach to Control Mutation
We have developed a computational algorithm that returns control back into the hands of synthetic biologists. By making synonymous substitutions that preserve gene function, substantial proportions of mutagenic sites can be eliminated from any sequence. Our robust algorithmic strategy for generating mutation-resistant genes has potential for improving the safety and stability of transgenes, which we are demonstrating by performing multiple independent techniques to measure stability at the level of sequences in vitro up to the function of cell populations.
To complement our applied research, we are also taking advantage of our new software tools to begin to answer a lingering question in the field: why would a site’s risk of mutation depend so greatly on the base-composition of the nucleotides surrounding it? By synthesizing sequences with tailored patterns of mutation “hotspots”, we may be able to further improve the performance of our own algorithm, and may provide insight into the nature of mutagenesis to advance the field of cancer research.
While any single engineered change to reduce mutation may still fail, when our innovative approaches to modulating evolutionary stability are taken in combination, they offer an unprecedented hope for overcoming evolutionary entropy. More than a victory for synthetic biology, we prove that through rational design principles- exactly what mutation most virulently tries to uproot- and with enough clever innovations, it is possible to defend against what seemed like an inevitability of nature.
Reference
1. Ceroni, F., Algar, R., Stan, G.-B., and Ellis, T. (2015). Quantifying cellular capacity identifies gene expression designs with reduced burden. Nature Methods 12, 415–418.
2. Sleight, S.C., Bartley, B.A., Lieviant, J.A., and Sauro, H.M. (2010). Designing and engineering evolutionary robust genetic circuits. Journal of Biological Engineering 4, 12.
3. Das, S., Duggal, P., Roy, R., Myneedu, V.P., Behera, D., Prasad, H.K., and Bhattacharya, A. (2012). Identification of Hot and Cold spots in genome of Mycobacterium tuberculosis using Shewhart Control Charts. Scientific Reports 2.
