<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_Vanderbilt_Background_Mutation skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><DIV id="top_title"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:Vanderbilt/Background/Mutation</SPAN></H1></DIV><DIV id="HQ_page"><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><DIV class="menu_wrapper"><DIV class="collapsable_menu_control"> MENU ▤ </DIV><UL id="accordion" class="accordion"><LI class="menu_item"><A href="https://2016.igem.org/Team:Vanderbilt"><U> Abstract</U></A></LI><LI class="menu_item"> Our Team
			</LI><LI><A href=" https://2016.igem.org/Team:Vanderbilt/About"> About </A></LI><LI><A href=" https://2016.igem.org/Team:Vanderbilt/Team"> Team   </A></LI><LI><A href="https://2016.igem.org/Team:Vanderbilt/Attributions"> Attributions</A></LI></UL><LI class="menu_item"> Outreach
			</LI><LI><A href="https://2016.igem.org/Team:Vanderbilt/Collaborations"> Collaborations</A></LI><LI><A href="https://2016.igem.org/Team:Vanderbilt/Interlab"> Interlab</A></LI><LI><A href="https://2016.igem.org/Team:Vanderbilt/Community"> Community</A></LI><LI><A href="https://2016.igem.org/Team:Vanderbilt/Education"> Education</A></LI><LI class="menu_item"><I><U>The Problem</U></I></LI><LI class="menu_item"> Background 
			</LI><LI><A href="https://2016.igem.org/Team:Vanderbilt/Background/Problem"> Introduction </A></LI><LI><A href="https://2016.igem.org/Team:Vanderbilt/Background/Mutation"> Mutation </A></LI><LI class="menu_item"><I><U>Our Solution</U></I></LI><LI class="menu_item"> Software 
			</LI><LI><A href="https://2016.igem.org/Team:Vanderbilt/Software/Algorithm"> Algorithm</A></LI><LI><A href="https://2016.igem.org/Team:Vanderbilt/Software/Performance"> Performance</A></LI><LI><A href="https://2016.igem.org/Team:Vanderbilt/Software/Design"> Design </A></LI><LI class="menu_item"> Experiments 
			</LI><LI><A href="https://2016.igem.org/Team:Vanderbilt/Experiments/qPCR"> qPCR </A></LI><LI><A href="https://2016.igem.org/Team:Vanderbilt/Experiments/Immunoblotting"> Immunoblotting </A></LI><LI><A href="https://2016.igem.org/Team:Vanderbilt/Experiments/LCMS"> LC-MS </A></LI><LI><A href="https://2016.igem.org/Team:Vanderbilt/Experiments/Evolution"> Evolution </A></LI></DIV><DIV class="content_wrapper"><H2><SPAN class="mw-headline" id="Targeting_mutagenesis">Targeting mutagenesis</SPAN></H2><H4> Radiological mutagens </H4><P> Ultraviolet radiation from solar and industrial sources represents a major threat to the genomic integrity of engineered organisms. It is well established that UV does not inflict nucleotide-level DNA damage in a random manner. Certain sequence motifs are rife substrates for mutagenesis upon exposure to the normal range of 290-400 nm UV present in the environment. In particular, genomic repeats of two pyrimidines known as dipyrimidine sites are hotspots of UV-mediated genetic insults. At the molecular level, absorption of UV radiation at dipyrimidine sites catalyzes a series of photochemical reactions that can generate three stable chemical species: the cyclobutane pyrimidine dimer (CPD), pyrimidine(6-4)pyrimidine photoproduct (64PP), and the Dewar isomer (Douki &amp; Sage, 2016). These first two chemical products are bulky covalently linked complexes of adjacent bases that inhibit DNA synthesis by obstructing the passage of replication enzymes across the template strand. The preponderance of mutations generated by these photoproducts involve the conversion of C to T or CC to TT at a pyrimidine dimer in a pattern known as the UV signature. </P><P> Several models attempt to explain the resolution of these modified pyrimidine photoproducts into stable UV signature mutations. The cytosine residues at CPD sites are particularly susceptible to spontaneous deamination, a reaction that can demonstrably convert di-cytosine CPD’s to di-uracil CPD’s. During a round of replication, these former CC nucleotides are swapped for TT through erroneous base-pairing with deaminated di-uracil CPD’s. An alternative model for explaining TT to CC mutations at CPD sites involves the activity of error-prone DNA polymerases, including the tendency of E. coli pol V to incorrectly add adenine across from a CPD site (Ikehata &amp; Ono, 2011). The mechanism by which 64PP and Dewar isomers transform into UV signature mutations is less clear, but it is thought to involve error-prone DNA polymerase activity at the sites of these lesions (Ikehata &amp; Ono, 2011).    </P><H4> Oxidative Mutagenesis </H4><P>
In addition to directly catalyzing the formation of dipyrimidine photoproducts, UV radiation also primes the genome for a second class of environmentally induced DNA mutations.  While oxidation of nucleotides by endogenous produced reactive oxygen species (ROS) threatens cells even during states of basal metabolic activity,  UV radiation exacerbates cellular conditions by stimulating photosensitive biological molecules such as riboflavin and porphyrin to increase ROS production. When the normal balance of ROS and anti-oxidant chemistry is tipped in favor of oxidation, free radicals wreak havoc on the genome. Shown below is a survey of the wide array of reduction and oxidation products generated by common ROS species such as hydroxide and superoxide radicals.</P><P> In particular, the hydroxide free radical is involved in a reaction with guanine residues that produces 8-hydroxyguanine (8OH-G). This common oxidative lesion is associated with both UV-dependent and UV-independent environmental insults through the ROS mutagenesis pathway. If left unrepaired, 8OH-G will readily base pair with dATP during replication to facilitate a stable G to T transversion that is a hallmark of oxidative mutagenesis. Our decision to analyze 8OH-G oxidative mutations is rooted in the rich scientific literature linking the chemical mechanism and physiological consequences of these ROS-induced DNA lesions. Reported hotspots of 8OH-G oxidative mutation include the ras and p53 loci in cancer cells, and tissue-specific accumulation of 8OH-G has been associated with pathological conditions such as coronary heart disease (Cooke et al., 2003). </P><H4> Mutation and Natural Selection</H4><P>  
Rejection of foreign DNA in gene therapy and unintended effects of GMOs are driven by the effect of metabolic load and selective pressures on genes which push them toward inactivation or deletion.  Increasing the metabolic load of the cell he been shown to decrease the rate of replication, which is correlated with decreased Darwinian fitness. Cells that must produce progeny and are less able to replicate under a given set of environmental conditions are likely to be overtaken in a heterogeneous population where other clones are not under burdened by the same load (Ceroni et al., 2014). Moreover, the cost of translating active proteins and producing secondary metabolites elevates the chemical stress experienced by the cell for a further decrease in fitness. </P><P> Though the metabolic load itself is deleterious to a cell’s ability to survive, an even greater danger is posed by increasing cellular load through stress-induced mutagenesis. The stress of an augmented rate of protein production pushes the cell to initiate a global response system that alters gene expression and metabolism. Part of this changed gene expression signature includes increased transcription of error prone DNA polymerases and down-regulation of error correcting enzymes. Taken together, these changes also facilitate the movement of insertion sequences. Switching into a mutation-prone state under stressful conditions due increased metabolic load may facilitate adaptive evolution, but such a state is counterproductive to the ideal environment of stable expression sought by synthetic biologists.</P><P><I>Reference</I></P><P>Ceroni, F., Algar, R., Stan, G. B., &amp; Ellis, T. (2015). Quantifying cellular capacity identifies gene expression designs with reduced burden. Nature methods, 12(5), 415-418. </P><P>Cooke, M. S., Evans, M. D., Dizdaroglu, M., &amp; Lunec, J. (2003). Oxidative DNA damage: mechanisms, mutation, and disease. The FASEB Journal, 17(10), 1195-1214. </P><P> David SS, O'shea VL, Kundu S. Base-excision repair of oxidative DNA damage. Nature. 2007;447(7147):941-50. </P><P>Douki, T., &amp; Sage, E. (2016). Dewar valence isomers, the third type of environmentally relevant DNA photoproducts induced by solar radiation. Photochemical &amp; Photobiological Sciences, 15(1), 24-30.</P><P> Ikehata, H., &amp; Ono, T. (2011). The mechanisms of UV mutagenesis. Journal of radiation research, 52(2), 115-125. </P></DIV></DIV></DIV></DIV></DIV></DIV></BODY></HTML>