Team:Vanderbilt/Experiments/Immunoblotting
MENU ▤
Abstract
Our Team
About
Team
Attributions
Outreach
Collaborations
Interlab
Community
Education
The Problem
Background
Introduction
Mutation
Our Solution
Software
Algorithm
Performance
Design
Experiments
qPCR
Immunoblotting
LC-MS
Evolution
Immunoblotting
Summary
Using antibodies sensitive to picogram amounts of specific DNA mutagenic lesions, we found that our sequence-optimization strategy could protect common genes from up to half of mutagenic damage upon UV exposure at a biochemical level. Our synthetic sequences with defined pyrimidine levels gave additional support to the surprising conclusion that mutations may not be linearly correlated with the number of mutation-prone sequence motifs. We also developed a protocol for performing analogous analyses on oxidized DNA to cross-validate our LC-MS results.
Methodology
Antibodies evolved as nature’s solution to the problem of achieving the highest and most selective binding affinity possible to any of even structurally similar molecules. Beyond distinguishing different proteins, their specificity can extend to recognizing DNA bases with particular mutagenic modifications. We employed monoclonal antibodies that recognize UV-dimerized or oxidized nucleotides (Noblanc et al., 2012; Sinha et al., 2001).
Our mutagenized DNA samples were immobilized to a membrane and probed with mutation-specific antibody. For UV experiments, our antibody recognized cyclobutane pyrimidine dimers, and for oxidation experiments our antibody recognized the three most common forms of oxidized guanine (8-hydroxy-2'-deoxyguanosine, 8-hydroxyguanine, and 8-hydroxyguanosine). A fluorescently-conjugated secondary antibody gave a quantitative output signal. All samples were measured before and after membrane-binding to asses that DNA levels were equal. Any differences in antibody-linked fluorescent would therefore be indicative of different incidences of mutation.
Reduction in UV-Induced Mutation
Previous investigators using these DNA-damage antibodies focused on extracted genomic DNA or fixed cells. The short synthetic gene fragments we were working with posed protocol challenges that were not previously addressed in the literature. Different membrane types have different DNA binding efficiencies, so we began by testing binding under different conditions. We used a SYBR stain to label our < 1 kb gene fragments and observed how much stain remained after binding and simulated wash steps.
We found that uncharged nitrocellulose membranes were best at binding and retention with a membrane-baking protocol to avoid crosslinking when binding DNA (figure 5). In contrast, most charged membranes we tested resulted in minimal binding capacity despite being reported to perform better with many genomic samples [Brown, T. (2001). Hybridization Analysis of DNA Blots. In Current Protocols in Molecular Biology, (John Wiley & Sons, Inc.) ]. We also optimized antibody conditions, first for our pyrimidine dimer antibody, by constructing a dose-curve. The UV dose range we were interested in was within the dynamic range of the antibody and produced a signal that was linearly related to UV exposure (adjusted R2 = 0.983, p < 0.001 ). Consequently, any differences between samples would scale linearly to their degree of UV-mutation.
We noticed that the decrease in mutation often exceeded what we would have had predicted based on the 35% reduction in mutagenic sequence motifs. Intrigued by this observation, we synthesized DNA sequences with set numbers of pyrimidine motifs and analyzed if their rates of mutation were directly correlated with the number of sites. We found the relationship was better approximated by an exponential function (figure 5), where halving the number of sites can reduce mutation up to six-fold.
Conclusions
Using antibodies against irradiated and oxidized DNA nucleotides that we demonstrated give sensitive and quantitative measurement, we found that our algorithmic sequence optimization significantly reduces the occurrence of mutation for standard genes such as RFP. From our artificial sequences with defined number of hotspots, we discovered that frequency of mutagenic sites in a sequence is non-linearly correlated with that sequence’s risk of mutation. We have begun expanding our assay to oxidized DNA samples and other optimized constructs to study this phenomenon in greater detail.
