<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_Vanderbilt_Software_Algorithm skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><DIV id="top_title"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:Vanderbilt/Software/Algorithm</SPAN></H1></DIV><DIV id="HQ_page"><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><DIV class="menu_wrapper"><DIV class="collapsable_menu_control"> MENU ▤ </DIV><UL id="accordion" class="accordion"><LI class="menu_item"><A href="https://2016.igem.org/Team:Vanderbilt"><U> Abstract</U></A></LI><LI class="menu_item"> Our Team
			</LI><LI><A href=" https://2016.igem.org/Team:Vanderbilt/About"> About </A></LI><LI><A href=" https://2016.igem.org/Team:Vanderbilt/Team"> Team   </A></LI><LI><A href="https://2016.igem.org/Team:Vanderbilt/Attributions"> Attributions</A></LI></UL><LI class="menu_item"> Outreach
			</LI><LI><A href="https://2016.igem.org/Team:Vanderbilt/Collaborations"> Collaborations</A></LI><LI><A href="https://2016.igem.org/Team:Vanderbilt/Interlab"> Interlab</A></LI><LI><A href="https://2016.igem.org/Team:Vanderbilt/Community"> Community</A></LI><LI><A href="https://2016.igem.org/Team:Vanderbilt/Education"> Education</A></LI><LI class="menu_item"><I><U>The Problem</U></I></LI><LI class="menu_item"> Background 
			</LI><LI><A href="https://2016.igem.org/Team:Vanderbilt/Background/Problem"> Introduction </A></LI><LI><A href="https://2016.igem.org/Team:Vanderbilt/Background/Mutation"> Mutation </A></LI><LI class="menu_item"><I><U>Our Solution</U></I></LI><LI class="menu_item"> Software 
			</LI><LI><A href="https://2016.igem.org/Team:Vanderbilt/Software/Algorithm"> Algorithm</A></LI><LI><A href="https://2016.igem.org/Team:Vanderbilt/Software/Performance"> Performance</A></LI><LI><A href="https://2016.igem.org/Team:Vanderbilt/Software/Design"> Design </A></LI><LI class="menu_item"> Experiments 
			</LI><LI><A href="https://2016.igem.org/Team:Vanderbilt/Experiments/qPCR"> qPCR </A></LI><LI><A href="https://2016.igem.org/Team:Vanderbilt/Experiments/Immunoblotting"> Immunoblotting </A></LI><LI><A href="https://2016.igem.org/Team:Vanderbilt/Experiments/LCMS"> LC-MS </A></LI><LI><A href="https://2016.igem.org/Team:Vanderbilt/Experiments/Evolution"> Evolution </A></LI></DIV><DIV class="content_wrapper"><H2><SPAN class="mw-headline" id="MutOpt_Algorithm">MutOpt Algorithm</SPAN></H2><H4> Summary </H4><P>Our software is the first tool that gives synthetic biologists the ability to reduce the risk of mutation for any given gene sequence. Users input any gene which the program then scans for sequence motifs that are prone to mutate at elevated rates. These motifs are removed by synonymous substitutions that are designed not to effect gene function or expression. Parameters can be customized to tailor the optimization to protect against different mutagens, as well as ensuring parts are compatible with assembly standards, and allowing for mutation maximization for directed evolution experiments. </P><H4> Software Application </H4><P align="middle"><I> Our software is hosted at: <A href="https://cosmicexplorer.github.io/mutation-optimizer/"> cosmicexplorer.github.io/mutation-optimizer </A></I></P><P>To use our software tools, user simply need to input a gene sequence into the left-hand field and press “Go!”. Both nucleotide and amino acid sequences are accepted as inputs. By default, sequences are optimized broadly against all mutation types, eliminating a variety of sites including those for oxidation, irradiation, deamination, alkylation, polymerase errors, and recombination. Any of these optimization parameters can be customized to concentrate on specific mutagens, ensure compatibility for RFC assembly standards, and directed evolution. </P><P> In the output window, the changed nucleotides are highlighted to visualize the changes made to the gene. The display also includes two metrics to compare the relative effectiveness of optimization for a particular gene. We combine the mutation types that the user includes as input (or keeps as defaults) into a single mutability score that captures how at-risk the inputted gene is to mutate. Following optimization, the software calculates the score again and returns the percent mutability change, which gives an indication of how many predicted mutation-prone sites were successfully removed by the algorithm. </P><H4> Program Parameters </H4><P>  We analyzed the literature for reports or nucleotide sequences that appeared to mutate at rates higher than what would be expected by chance (Rogdozin and Pavlov 2003). In particular we concentrated on short (&lt; 5 bp) motifs with multiple studies-worth of validation that could reasonably be targeted by sense substitutions. In all, we selected sequences associated with ultraviolet dimers, insertion sequences, deamination, oxidation, RNA hairpins, microhomologies, polymerase errors, alkylation, and sites non-specific to mutagens. </P><P> The program analyzes each codon of the gene and determines what synonymous substitutions can be made and calculates the number of mutagenic sites per possible construction of codons. For overall optimization, sites are assigned scores based on their relative frequency and severity. The scores are also adjusted to account for codon usage biases in the species selected. After experimentally testing multiple iterations, we developed a custom set of heuristics that achieved the best expression levels for optimized sequences. </P><P><I>Reference</I></P><P> Rogozin, I.B., and Pavlov, Y.I. (2003). Theoretical analysis of mutation hotspots and their DNA sequence context specificity. Mutat. Res. 544, 65–85 </P></DIV></DIV></DIV></DIV></DIV></DIV></BODY></HTML>