<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_Munich_Description skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><DIV id="top_title"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:Munich/Description</SPAN></H1></DIV><DIV id="HQ_page"><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><P><TITLE>Phactory</TITLE></P><H5 class="d-inline ml-3">Phactory</H5><DIV class="collapse navbar-collapse justify-content-around" id="collapsibleNavbar"><UL class="navbar-nav"><LI id="homeButton" class="nav-item mx-3"><A class="nav-link" href="https://2018.igem.org/Team:Munich">HOME</A></LI><LI class="nav-item dropdown mx-3"><A class="nav-link dropdown-toggle" href="#" id="navbardrop" data-toggle="dropdown">
                            TEAM
                        </A><DIV class="dropdown-menu"><A class="dropdown-item" href="https://2018.igem.org/Team:Munich/Team">Team Members</A><A class="dropdown-item" href="https://2018.igem.org/Team:Munich/Collaborations">Collaborations</A><A class="dropdown-item" href="https://2018.igem.org/Team:Munich/Sponsors">Sponsors</A><A class="dropdown-item" href="https://2018.igem.org/Team:Munich/Attributions">Attributions</A></DIV></LI><LI class="nav-item dropdown mx-3"><A class="nav-link dropdown-toggle" href="#" id="navbardrop" data-toggle="dropdown">
                            PROJECT
                        </A><DIV class="dropdown-menu"><A class="dropdown-item" href="https://2018.igem.org/Team:Munich/Description">Description</A><A class="dropdown-item" href="https://2018.igem.org/Team:Munich/Design">Design</A><A class="dropdown-item" href="https://2018.igem.org/Team:Munich/Notebook">Notebook</A><A class="dropdown-item" href="https://2018.igem.org/Team:Munich/Protocols">Protocols</A><A class="dropdown-item" href="https://2018.igem.org/Team:Munich/InterLab">InterLab</A><A class="dropdown-item" href="https://2018.igem.org/Team:Munich/Model">Model</A><A class="dropdown-item" href="https://2018.igem.org/Team:Munich/Results">Results</A><A class="dropdown-item" href="https://2018.igem.org/Team:Munich/Demonstrate">Demonstrate</A></DIV></LI><LI class="nav-item dropdown mx-3"><A class="nav-link dropdown-toggle" href="#" id="navbardrop" data-toggle="dropdown">
                            PARTS
                        </A><DIV class="dropdown-menu"><A class="dropdown-item" href="https://2018.igem.org/Team:Munich/Parts">Parts Overview</A><A class="dropdown-item" href="https://2018.igem.org/Team:Munich/Basic_Part">Basic Parts</A><A class="dropdown-item" href="https://2018.igem.org/Team:Munich/Improve">Improved Parts</A><A class="dropdown-item" href="https://2018.igem.org/Team:Munich/Composite_Part">Composite Parts</A></DIV></LI><LI class="nav-item mx-3"><A class="nav-link" href="https://2018.igem.org/Team:Munich/Safety">SAFETY</A></LI><LI class="nav-item dropdown mx-3"><A class="nav-link dropdown-toggle" href="#" id="navbardrop" data-toggle="dropdown">
                            HUMAN PRACTICES
                        </A><DIV class="dropdown-menu"><A class="dropdown-item" href="https://2018.igem.org/Team:Munich/Human_Practices">Human Practices</A><A class="dropdown-item" href="https://2018.igem.org/Team:Munich/Public_Engagement">Education &amp; Engagement</A><A class="dropdown-item" href="https://2018.igem.org/Team:Munich/Oracoli">Oracoli</A><A class="dropdown-item" href="https://2018.igem.org/Team:Munich/European_Meetup">European Meetup</A></DIV></LI><LI class="nav-item dropdown mx-3"><A class="nav-link dropdown-toggle" href="#" id="navbardrop" data-toggle="dropdown">
                            AWARDS
                        </A><DIV class="dropdown-menu"><A class="dropdown-item" href="https://2018.igem.org/Team:Munich/Applied_Design">Applied Design</A><A class="dropdown-item" href="https://2018.igem.org/Team:Munich/Entrepreneurship">Entrepreneurship</A><A class="dropdown-item" href="https://2018.igem.org/Team:Munich/Hardware">Hardware</A><A class="dropdown-item" href="https://2018.igem.org/Team:Munich/Measurement">Measurement</A><A class="dropdown-item" href="https://2018.igem.org/Team:Munich/Model">Model</A><A class="dropdown-item" href="https://2018.igem.org/Team:Munich/Software">Software</A></DIV></LI><LI class="nav-item mx-3"><A class="nav-link" href="https://igem.org/2018_Judging_Form?team=Munich" target="_blank">JUDGING FORM â‡—</A></LI></UL></DIV><DIV class="pictureTitle container-fluid text-center mb-0 align-items-center text-light"><DIV class="display-2 mb-0">
                     Project Description
                    </DIV></DIV><DIV id="conceptFlow" class="text-dark bg-whites"><DIV class="container"><DIV id="conceptFlowTitle" class="pb-3 pt-5 mb-3 text-center" style="">How does Phactory work?</DIV><DIV class="row align-items-center mb-5"><DIV class="col-10 col-md-8"><H5>Abstract</H5><P class="text-justify">Antimicrobial resistance is a major emerging threat as reported by the WHO<SUP><A href="https://www.who.int/mediacentre/news/releases/2018/antibiotic-resistance-found/en">1</A></SUP>. Worldwide implementation of bacteriophage therapy, a 100-year old treatment employing the natural enemies of bacteria, is impeded by the lack of common manufacturing procedures which meet international quality and safety standards<SUP><A href="https://doi.org/10.2174/138920110790725401">2</A></SUP>. Based on synthetic biology we created Phactory, a cell-free molecular assembly line for bacteriophages. We demonstrate expression of several phages including T7, MS2 and 3S at clinically relevant concentrations. Exploiting the open nature of cell-free systems, Phactory enables modular composition of bacteriophages with engineered proteins while remaining GMO-free. We developed a quality control structure utilizing state-of-the-art bioinformatics, as well as purification and encapsulation protocols. To expand our production variety while reducing cost, we optimized and engineered home-made E. Coli cell-extract. Compared to traditional manufacturing procedures, Phactory requires 2.5% of the production volume and demands no special biosafety regulations to yield bacteriophages ready for therapy.</P></DIV></DIV><DIV class="row align-items-center how-it-works d-flex"><DIV class="col-10 col-md-8"><H5>Cell-Free Expression System</H5><P class="text-justify">The molecular heart of Phactory is a cell-free expression system<SUP><A href="https://pubs.acs.org/doi/abs/10.1021/sb200016s">3</A></SUP>. Screening over 50 parameters, we optimized a self-made chassis for assembly of bacteriophages. From a culture of the Rosetta2 (DE3) <I>E. coli</I> strain, cell extract is prepared by following improved cultivation and lysis protocols. We optimized our strain for therapeutic bacteriophage production by knocking out the myristoyltransferase <I>msbB</I> in the host strain to disrupt the lipid A biosynthesis pathway. This modification diminished endotoxin content in the final extract preparation below the detection limit. Lyophilization of the cell-free system paves the way for storage at room temperature and easy distribution.</P><H3>Characterizing All Variables of a Cell-Free System</H3><P class="text-justify">
            Manufacturing a pharmaceutical product requires fulfillment of the utmost standards. To ensure a constantly high-grade product, extensive quality control measures have been implemented in Phactory. Expression capability of the cell-free system is assessed on the basis of a malachite green aptamer and an mTurquoise construct, enabling independent evaluation of transcription and translation, complemented by an overall protein content measurement.</P></DIV></DIV><DIV class="row align-items-center how-it-works d-flex"><DIV class="col-10 col-md-8"><H5>Synthetic Phage Manufacturing</H5><P class="text-justify">Purified linear DNA templates are assembled to functional bacteriophages in Phactory. Creating independence from the living bacterial host, our cell-free system is capable of assembling any desired phage during a typical incubation time of four hours. Apart from combining the two components of our cell extract, phage assembly requires inhibition of the recombinase complex RecBCD by addition of purified GamS protein<SUP><A href="https://www.sciencedirect.com/science/article/pii/S0022283607006171">4</A></SUP>. Incorporation of dNTPs increases phage yield by allowing DNA amplification throughout the reaction.</P></DIV></DIV><DIV class="row align-items-center how-it-works d-flex"><DIV class="col-10 col-md-8"><H5>Modular Phage Composition</H5><P class="text-justify">A distinguishing feature of Phactory is the possibility to incorporate engineered proteins into the phage proteome by directly adding them into the assembly reaction. Without requiring genetic engineering, tailor-made phages can be assembled. As a proof-of-concept, we used a modified capsid protein containing eYFP to create a fluorescent T4 phage. Using an identical manufacturing principle, bioluminescent phages produced in Phactory could serve as an imaging platform for diagnosis or research purposes. Histidine tagging of capsid proteins allowed us to isolate assembled phages.
</P></DIV></DIV><DIV class="row align-items-center how-it-works d-flex"><DIV class="col-10 col-md-8"><H5>Quality Control</H5><H3>Quantification of DNA-Contamination Using Sequ-Into</H3><P class="text-justify">
           To assess the quality of our phage DNA, we performed third generation sequencing using nanopore technology.
A high DNA quality is especially critical for many therapeutic bacteriophages whose genome sequence is not known.
We performed de novo assembly of T7, 3S, FFP, and NES bacteriophages.
In our initial sequencing attempts, we noticed that traditional purification methods for phage DNA retained large amounts of contamination from E. coli genomes.
By sequencing different phage batches and determining the level of contamination, we could evaluate and enhance our purification protocols drastically, up to 96% purity<SUP><A href="https://academic.oup.com/hmg/article/19/R2/R227/641295">5</A><SUP>.</SUP></SUP></P><P class="text-justify">
            Determining contamination levels from sequencing data requires knowledge of bioinformatics command line tools and programming skills.
In a clinical setting, it has to be done repeatedly and quickly, ideally in the lab on the same computer of the sequencing experiment to allow for real time testing.
So we implemented a straightforward, easy-to-use cross-platform desktop application offering a visualized statistical overview of contamination and simplifying downstream analysis: Sequ-Into.
We adapted it further to become a valuable addition to general third generation sequencing lab routines.
</P><H3>Assessing Phage Functionality</H3><P class="text-justify">Receiving functional phages is crucial for curing patients. A special focus of Phactory rests upon monitoring of phage functionality. We implemented a plaque assay to ensure that the phages are capable of infecting and eliminating bacteria. 
            </P><H3>Purity</H3><P class="text-justify">Phactory demands minimal purification efforts. We demonstrated absence of detectable immunogenic endotoxins in our modified cell extract preparation with the FDA-validated Limulus Amebocyte Lysate (LAL) test.
    </P></DIV></DIV><DIV class="row align-items-center how-it-works d-flex"><DIV class="col-10 col-md-8"><H5>Encapsulation</H5><P class="text-justify">The final component of Phactory is a hardware-supported microencapsulation in monodisperse spheres consisting of an alginate and CaCO<SUB>3</SUB> polymer. Protecting the phages against low pH and pepsin, our packaging approach allows gastric passage and intestinal delivery. In addition, the encapsulation enhances storage capabilities of the phage product.</P></DIV></DIV><DIV id="phareferences" class="row"><DIV class="col-12"><H2>References</H2><OL><LI id="ref_1"><A href="https://www.who.int/mediacentre/news/releases/2018/antibiotic-resistance-found/en">High levels of antibiotic resistance found worldwide, new data shows</A></LI><LI id="ref_2"><A href="https://doi.org/10.2174/138920110790725401">Phage Therapy in Clinical Practice: Treatment of Human Infections, Current Pharmaceutical Biotechnology (2010) 11: 69</A></LI><LI id="ref_3"><A href="https://pubs.acs.org/doi/abs/10.1021/sb200016s">An E. coli Cell-Free Expression Toolbox: Application to Synthetic Gene Circuits and Artificial Cells</A></LI><LI id="ref_4"><A href="https://www.sciencedirect.com/science/article/pii/S0022283607006171">The Protein GamS Inhibits RecBCD Binding to dsDNA Ends</A></LI><LI id="ref_5"><A href="https://academic.oup.com/hmg/article/19/R2/R227/641295">A window into third-generation sequencing</A></LI><LI id="ref_6"><A href="https://ispe.org/initiatives/regulatory-resources/gmp">Good Manufacturing Practice (GMP)</A></LI><LI id="ref_7"><A href="https://www.fda.gov/drugs/guidancecomplianceregulatoryinformation/guidances/ucm314718.htm">Guidance for Industry: Pyrogen and Endotoxins Testing: Questions and Answers</A></LI></OL></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></BODY></HTML>