<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_Munich_Model skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><DIV id="top_title"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:Munich/Model</SPAN></H1></DIV><DIV id="HQ_page"><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><P><TITLE>Phactory</TITLE></P><H5 class="d-inline ml-3">Phactory</H5><DIV class="collapse navbar-collapse justify-content-around" id="collapsibleNavbar"><UL class="navbar-nav"><LI id="homeButton" class="nav-item mx-3"><A class="nav-link" href="https://2018.igem.org/Team:Munich">HOME</A></LI><LI class="nav-item dropdown mx-3"><A class="nav-link dropdown-toggle" href="#" id="navbardrop" data-toggle="dropdown">
                            TEAM
                        </A><DIV class="dropdown-menu"><A class="dropdown-item" href="https://2018.igem.org/Team:Munich/Team">Team Members</A><A class="dropdown-item" href="https://2018.igem.org/Team:Munich/Collaborations">Collaborations</A><A class="dropdown-item" href="https://2018.igem.org/Team:Munich/Sponsors">Sponsors</A><A class="dropdown-item" href="https://2018.igem.org/Team:Munich/Attributions">Attributions</A></DIV></LI><LI class="nav-item dropdown mx-3"><A class="nav-link dropdown-toggle" href="#" id="navbardrop" data-toggle="dropdown">
                            PROJECT
                        </A><DIV class="dropdown-menu"><A class="dropdown-item" href="https://2018.igem.org/Team:Munich/Description">Description</A><A class="dropdown-item" href="https://2018.igem.org/Team:Munich/Design">Design</A><A class="dropdown-item" href="https://2018.igem.org/Team:Munich/Notebook">Notebook</A><A class="dropdown-item" href="https://2018.igem.org/Team:Munich/Protocols">Protocols</A><A class="dropdown-item" href="https://2018.igem.org/Team:Munich/InterLab">InterLab</A><A class="dropdown-item" href="https://2018.igem.org/Team:Munich/Model">Model</A><A class="dropdown-item" href="https://2018.igem.org/Team:Munich/Results">Results</A><A class="dropdown-item" href="https://2018.igem.org/Team:Munich/Demonstrate">Demonstrate</A></DIV></LI><LI class="nav-item dropdown mx-3"><A class="nav-link dropdown-toggle" href="#" id="navbardrop" data-toggle="dropdown">
                            PARTS
                        </A><DIV class="dropdown-menu"><A class="dropdown-item" href="https://2018.igem.org/Team:Munich/Parts">Parts Overview</A><A class="dropdown-item" href="https://2018.igem.org/Team:Munich/Basic_Part">Basic Parts</A><A class="dropdown-item" href="https://2018.igem.org/Team:Munich/Improve">Improved Parts</A><A class="dropdown-item" href="https://2018.igem.org/Team:Munich/Composite_Part">Composite Parts</A></DIV></LI><LI class="nav-item mx-3"><A class="nav-link" href="https://2018.igem.org/Team:Munich/Safety">SAFETY</A></LI><LI class="nav-item dropdown mx-3"><A class="nav-link dropdown-toggle" href="#" id="navbardrop" data-toggle="dropdown">
                            HUMAN PRACTICES
                        </A><DIV class="dropdown-menu"><A class="dropdown-item" href="https://2018.igem.org/Team:Munich/Human_Practices">Human Practices</A><A class="dropdown-item" href="https://2018.igem.org/Team:Munich/Public_Engagement">Education &amp; Engagement</A><A class="dropdown-item" href="https://2018.igem.org/Team:Munich/Oracoli">Oracoli</A><A class="dropdown-item" href="https://2018.igem.org/Team:Munich/European_Meetup">European Meetup</A></DIV></LI><LI class="nav-item dropdown mx-3"><A class="nav-link dropdown-toggle" href="#" id="navbardrop" data-toggle="dropdown">
                            AWARDS
                        </A><DIV class="dropdown-menu"><A class="dropdown-item" href="https://2018.igem.org/Team:Munich/Applied_Design">Applied Design</A><A class="dropdown-item" href="https://2018.igem.org/Team:Munich/Entrepreneurship">Entrepreneurship</A><A class="dropdown-item" href="https://2018.igem.org/Team:Munich/Hardware">Hardware</A><A class="dropdown-item" href="https://2018.igem.org/Team:Munich/Measurement">Measurement</A><A class="dropdown-item" href="https://2018.igem.org/Team:Munich/Model">Model</A><A class="dropdown-item" href="https://2018.igem.org/Team:Munich/Software">Software</A></DIV></LI><LI class="nav-item mx-3"><A class="nav-link" href="https://igem.org/2018_Judging_Form?team=Munich" target="_blank">JUDGING FORM ⇗</A></LI></UL></DIV><DIV class="pictureTitle container-fluid text-center mb-0 align-items-center text-light"><DIV class="display-2 mb-0">
                    Model
                    </DIV></DIV><DIV class="phaContainer"><MAIN class="post-content"><H2>Cell-free Biosynthesis of Bacteriophages</H2><DIV class="row"><DIV class="col-xs-12 col-sm-12 "><P>The seemingly simple experimental procedures of the optimized framework of Phactory may lead to the false impression that cell-free biosynthesis of bacteriophages is a trivial process. In fact, the production of phages represents one of the most complicated self-assembly processes that have been realized in a synthetic context (<A href="#phareferences">1,2</A>).

We chose the T7 phage as a model system for cell-free phage production, because it is well characterized in the literature and our experiments. The phage assembly reaction resembles a temporally orchestrated process, which involves the expression of more than 40 proteins. This genetic program, encoded by the phage genome, is executed in a test tube containing hundreds of proteins, ribosomes, tRNAs and small molecules.

Due to the complexity of this process, variations in the quality of our cell extract cause irreproducible results. To optimize bacteriophage titers, we created a model based on ordinary differential equations to understand the essential features of a phage assembly reaction in a quantitative manner.
</P></DIV></DIV><H2>Assumptions and Parameter Estimation</H2><DIV class="row"><DIV class="col-xs-12 col-sm-12 "><P>	A phage assembly reaction follows a temporal program (<A href="#phareferences">1</A>). First, the genes controlled by <I>E. coli</I> promoters, including the T7 DNA and RNA polymerase are expressed. Once the T7 RNA polymerase is present, the transcription of genes under a T7 promoter, e.g. the capsid proteins, starts. Our experiments revealed that simultaneously the phage genome being is replicated by the T7 DNA polymerase. As soon as a sufficient amount of capsid proteins is produced, they self-assemble to form empty phage capsids. In a final step the phage genome is packed into the empty capsids forming functional phages. In the following paragraphs we develop a set of ordinary differential equations that describe the phage assembly reaction, give analytical results where appropriate and estimate realistic reaction rate constants.
</P></DIV></DIV><H3>Transcription/ Translation Kinetics</H3><DIV class="row"><DIV class="col-xs-12 col-sm-12 "><P>The kinetics of cell-free protein expression reactions has been described in the literature (<A href="#phareferences">3</A>). mRNA $r$ is being produced from a gene $g$ at a rate $\alpha_{TX}(t)$, while being degraded with a rate $\delta_r=0.05$ 1/min (<A href="#phareferences">4</A>):

$$ \frac{dr}{dt}=\alpha_{TX}(t) \cdot g - \delta_r \cdot r \ .$$

In order to account for the finite lifetime of cell-free reactions, we assume that the production rate $\alpha_{TX}(t)=\alpha_{TX,0} \cdot e^{-t/\tau} \cdot P_r$ decreases exponentially with a half-life of $\tau=2$ hours (<A href="#phareferences">4</A>). Here, $P_r$ is the RNA polymerase concentration. In case of E. coli RNA polymerase we chose $\alpha_{TX,0}=10^{-4}$ 1/nM/s (<A href="#phareferences">3</A>), while the T7 RNA polymerase is 10 times faster $\alpha_{TX,0}=10^{-3}$ 1/nM/s.
Qualitatively, this means that following an initial increase, mRNA concentration approaches the steady state given by $\alpha_{TX}(t)/\delta_r$, which decreases with time. This leads to a pulse in mRNA concentration (figure). 

Translation of a protein $p$ follows a similar reaction, where $\alpha_{TL}(t)=\alpha_{TL,0} \cdot e^{-t/\tau} \cdot R$ is proportional to the ribosome concentration $R$:

 $$ \frac{dp}{dt}=\alpha_{TL}(t) \cdot r \ . $$

Due to the absence of dilution by bacterial growth, protein degradation rates in cell-free systems are in absence of degradation tags are slow compared to the reaction time scale and can be neglected (<A href="#phareferences">3</A>).
For the translation rate we chose $\alpha_{TL,0} \cdot R=10^{-2}$ 1/s (<A href="#phareferences">3</A>). Assuming that the ribosome concentration is in the order of $10^3$ nM (<A href="#phareferences">3</A>), $\alpha_{TL,0} = 10^{-5}$ 1/nM/s. With the same lifetime $\tau = 2$ hours, this leads to a saturating curve (figure).
</P></DIV></DIV><H3>Replication of the Phage Genome</H3><DIV class="row"><DIV class="col-xs-12 col-sm-12 "><P>Our qPCR results indicated that upon addition of dNTPs to the reaction, the T7 DNA polymerase can replicate the phage genome, resulting in an approximately 10-fold amplification. On the other hand, linear DNA is rapidly degraded in cell-extract in the absence of inhibitors. We therefore introduced the following equation:

$$\frac{dg}{dt}=k_g(t)\cdot g - \delta_g \cdot g  = (k_g(t) - \delta_g) \cdot g \ , $$

where $k_g(t) = k_{g,0} \cdot P_d \cdot e^{-t/\tau_d}$, $\tau_d = 1$ hour, $k_{g,0} = 3 \cdot 10^{-5}$ 1/nM/s, and $\delta_g=0-10^{-5}$ (<A href="#phareferences">5</A>) for inhibited and non-inhibited degradation of linear phage DNA by the RecBCD complex, respectively (Figure 1, 2). At steady state, we get $g/g_0=\mathrm{exp}((k_g(t)-\delta_g)\cdot t)$, which indicates that DNA replication only occurs if the production rate exceeds the degradation.</P></DIV></DIV><DIV class="row"><DIV class="col-xs-12 col-sm-6"><VIDEO autoplay="" loop="" poster="https://static.igem.org/mediawiki/2018/0/0c/T--Munich--RecBCD_poster.png" style="width:100%;"><SOURCE src="https://static.igem.org/mediawiki/2018/4/41/T--Munich--Model_RecBCD_GamS.mp4">
Your browser does not support the video tag.
</SOURCE><B>Figure 1</B> RecBCD inhibited by GamS
    </VIDEO></DIV><DIV class="col-xs-12 col-sm-6"><VIDEO autoplay="" loop="" poster="https://static.igem.org/mediawiki/2018/4/40/T--Munich--Model_RecBCD_DNA_poster.png" style="width:100%;"><SOURCE src="https://static.igem.org/mediawiki/2018/e/e5/T--Munich--Model_Joe_RecBCD_DNA.mp4">
Your browser does not support the video tag.
</SOURCE><B>Figure 2</B> RecBCD degrading linear DNA.
    </VIDEO></DIV></DIV><H3>Capsid Assembly and DNA Packing</H3><DIV class="row"><DIV class="col-xs-12 col-sm-12 "><P>   One phage capsid $c$ is assembled from 420 capsid proteins $p_c$ (<A href="#phareferences">6</A>). The thermodynamics of such self-assembly processes are analogous to micelle formation from lipids (<A href="#phareferences">7</A>). An important result is, that assemblies cannot be formed below a critical monomer concentration $K_c$. We assume that $K_c \approx 10$ nM, which is a typical dissociation constant for protein-protein interactions. In our model we approximate this sharp threshold-like behavior by a steep Hill function with $n=10$:

$$ \frac{dc}{dt} = \frac{k_{assembly} \cdot p_c^n}{K_c^n + p_c^n} \ . $$

As phage assembly does not involve any catalytic reactions, we assume that the assembly rate $k_{assembly}$ is fast compared to gene expression and choose $k_{assembly}=10^{-3}$ nM/s. 

Finally, the assembled capsids are packed with DNA to form active phages $c^*$. We consider this by the following:

$$\frac{dc^*}{dt} = k_{packing} \cdot c \cdot g \ .$$

Again, we assume that $k_{packing}$ is fast compared to gene expression $k_{packing}=10^{-3}$ 1/nM/s.</P></DIV></DIV><H2>Equations</H2><DIV class="row"><DIV class="col-xs-12 col-sm-12 ">
The complete set of model equations is shown below.
        <H3>Reaction Fluxes</H3><P> \begin{align*}
                 [E.Coli\ Translation] &amp;=  k_{TL,E.Coli}\cdot[E.Coli\ mRNA]\cdot[Ribosome]\\
                 [E.Coli\ mRNA\ Degradation] &amp;= k_{deg,E.ColiRNA}\cdot[E.Coli\ mRNA]\\
                 [E.Coli\ Transcription] &amp;= k_{TX,E.Coli}\cdot[E.Coli\ RNAP]\cdot[Phage\ DNA]\cdot e^{-\frac{t}{7200}}\\
                 [T7\ Genome\ Replication] &amp;= k_{DNA}\cdot[T7\ DNAP]\cdot[Phage\ DNA]\cdot e^{-\frac{t}{3600}}\\
                 [DNA\ Degradation] &amp;= k_{deg,PhageDNA}\cdot[Phage DNA]\\
                 [T7\ Transscription] &amp;= k_{TX,T7}\cdot e^{-\frac{t}{7200}}\cdot[T7\ RNAP]\cdot [Phage\ DNA]\\
                 [T7\ mRNA\ Degradation] &amp;= k_{deg,T7RNA}\cdot [T7\ mRNA]\\
                 [T7\ Translation] &amp;= k_{TL,T7}\cdot[T7\ mRNA]\cdot [Ribosome]\\
                 [Phage\ Packaging] &amp;= k_{packaging}\cdot[Procapsid]\cdot [Phage\ DNA]\\
                 [Phage\ Assembly] &amp;=k_{assembly}\cdot \frac{Capsid^{10}}{10^{10}+Capsid^{10}}\\
                 [T7\ DNAP\ Translation] &amp;= k_{TL,E.Coli}\cdot[E.Coli\ mRNA]\cdot[Ribosome]
                 \end{align*}</P><H3>Ordinary Differential Equation System</H3><P>\begin{align*}
                 \frac{d[E.Coli\ mRNA]}{dt} &amp;=\ -[E.Coli\ mRNA\ Degradation] + [E.Coli\ Transcription]\\
                 \frac{d[Capsid]}{dt} &amp;=\ [E.Coli\ Translation] - 420\cdot[Phage\ Assembly]\\
                 \frac{d[Phage\ DNA]}{dt} &amp;=\ -[Phage\ Packaging] - [DNA\ Degradation] + [T7\ Genome Replication]\\
                 \frac{d[Phage]}{dt} &amp;=\ [Phage\ Packaging]\\
                 \frac{d[T7\ DNAP]}{dt} &amp;=\ [T7\ DNAP\ Translation]\\
                 \frac{d[T7\ RNAP]}{dt} &amp;=\ [E.Coli\ Translation]\\
                 \frac{d[T7\ mRNA]}{dt} &amp;=\ [T7\ Transscription] - [T7\ mRNA\ Degradation]\\
                 \frac{d[Procapsid]}{dt} &amp;=\ -[Phage\ Packaging] + [Phage\ Assembly]\\
                 \end{align*}
            </P></DIV></DIV><H2>Results</H2><DIV class="row"><DIV class="col-xs-12 col-sm-12 "><P>We used the MATLAB SimBiology toolbox to analyze the reactions numerically.</P></DIV></DIV><H3>Phage Assembly Follows a Temporal Program</H3><DIV class="row"><DIV class="col-xs-12 col-sm-6 "><P>Evaluating our ODE model we can now follow the time traces of each reaction species in our system. Looking at the mRNA levels, it becomes evident that mRNA transcribed by <I>E. coli</I> RNA polymerase is produced instantly and rapidly reaches a low steady-state level. mRNA transcribed by the T7 RNA polymerase is produced after a time delay and reaches a much higher steady-state level, consistent with the faster transcription rate. Both mRNA levels decrease as the steady state level decreases due to the finite lifetime of the cell extract.</P><P>Following protein levels, we see that production of the T7 DNA and RNA polymerases starts first, while production of the T7 RNA polymerase controlled capsid proteins sets in later. When the free capsid proteins exceed the critical concentration $K_c$, they start to be assembled into full capsids and the free capsid protein concentration approaches a steady-state. All protein levels approach a final level at the end of the reaction. </P></DIV><DIV class="col-xs-12 col-sm-6 "><FIGURE class="figure"><FIGCAPTION class="figure-caption"><B>Figur 3</B> mRNA levels over time in the cell-extract. </FIGCAPTION></FIGURE></DIV></DIV><DIV class="row"><DIV class="col-xs-12 col-sm-12 "><P>Further, it is instructive to inspect the time traces of free DNA and fully packed active phages. As soon as sufficient T7 DNA polymerase is present, the phage genomes are being amplified to approximately 10 times the initial level. The assembly and packing of functional phages, however, only kicks in after an initial time delay. Importantly, this allows the genome to be freely transcribed and replicated, before the negative feedback introduced by DNA packing, impedes further phage production.
 </P></DIV></DIV><DIV class="row"><DIV class="col-xs-12 col-sm-6 "><FIGURE class="figure"><FIGCAPTION class="figure-caption"><B>Figure 4</B>T7 polymerase generation by the E.coli in-house polymerase</FIGCAPTION></FIGURE></DIV><DIV class="col-xs-12 col-sm-6 "><FIGURE class="figure"><FIGCAPTION class="figure-caption"><B>Figure 5</B>Phage production and DNA levels over time </FIGCAPTION></FIGURE></DIV></DIV><H3>DNA Stability</H3><DIV class="row"><DIV class="col-xs-12 col-sm-6 "><P> To analyze the influence of DNA stability, we screened a range of DNA degradation rates as shown in the figure on the right.
                     Phages are produced until a critical degradation rate of $\delta_{g,c}\approx 2 \cdot 10^{-4}$ is reached.
                     This results in lower concentrations of proteins in downstream reactions and resultingly prohibits the formation of procapsids.</P></DIV><DIV class="col-xs-12 col-sm-6"><FIGURE class="figure"><FIGCAPTION class="figure-caption"><B>Figure 6</B>Influence of DNA degradation on the phage production</FIGCAPTION></FIGURE></DIV></DIV><H3>Impact of Positive Feedback via DNA Replication</H3><DIV class="row"><DIV class="col-xs-12 col-sm-6 "><P>To evaluate the influence of the positive feedback loop introduced by DNA replication, we conducted experiments with and without addition of dNTPs to the reaction mixture. With our model we can simulate this experiment by setting the DNA replication rate to zero.</P><P>The bar plot on the right shows enhanced phage packaging of over one decade by the integrated feedback loop of the T7 DNA  polymerase.</P></DIV><DIV class="col-xs-12 col-sm-6"><FIGURE class="figure"><FIGCAPTION class="figure-caption"><B>Figure 7</B>Evaluation of DNA polymerase in the phage production.</FIGCAPTION></FIGURE></DIV></DIV><H2>Conclusions</H2><DIV class="row"><DIV class="col-xs-12 col-sm-12 "><P>Based on the results, we identified that enhancing DNA stability and maximizing protein yield are the most relevant parameters to improve phage titers. After optimizing our protocols accordingly, we reached $10^7$ phages per 10 µL reaction - sufficient to treat up to 10 patients.</P></DIV></DIV><DIV id="phareferences" class="row"><H3>References</H3><DIV class="col-12"><OL><LI id="ref_1"><A href="https://pubs.acs.org/doi/abs/10.1021/sb300049p">  Shin, J., Jardine, P., &amp; Noireaux, V. (2012). Genome Replication, Synthesis, and Assembly of the Bacteriophage T7 in a Single Cell-Free Reaction. ACS Synthetic Biology, 1(9), 408–413.</A></LI><LI id="ref_2"><A href="https://www.jove.com/video/56144/synthesis-infectious-bacteriophages-an-e-coli-based-cell-free">  Rustad, M., Eastlund, A., Marshall, R., Jardine, P., &amp; Noireaux, V. (2017). Synthesis of infectious bacteriophages in an E. coli-based cell-free expression system.
</A></LI><LI id="ref_3"><A href="https://doi.org/10.1103/PhysRevLett.106.048104">  Karzbrun, E., Shin, J., Bar-Ziv, R. H., &amp; Noireaux, V. (2011). Coarse-grained dynamics of protein synthesis in a cell-free system. Physical Review Letters, 106(4).</A></LI><LI id="ref_4"><A href="https://doi.org/10.1021/sb400203p">Siegal-Gaskins, D., Tuza, Z. A., Kim, J., Noireaux, V., &amp; Murray, R. M. (2014). Gene Circuit Performance Characterization and Resource Usage in a Cell-Free “Breadboard.” ACS Synthetic Biology, 3(6), 416–425. </A></LI><LI id="ref_5"><A https:="">
Sun, Z. Z., Yeung, E., Hayes, C. A., Noireaux, V., &amp; Murray, R. M. (2014). Linear DNA for rapid prototyping of synthetic biological circuits in an Escherichia coli based TX-TL cell-free system. ACS Synthetic Biology, 3(6), 387–397.  </A></LI><LI id="ref_6"><A href="http://www.jbc.org/content/286/1/234.full">Ionel, A., Velázquez-Muriel, J. A., Luque, D., Cuervo, A., Castón, J. R., Valpuesta, J. M., … Carrascosa, J. L. (2011). Molecular Rearrangements Involved in the Capsid Shell Maturation of Bacteriophage T7. Journal of Biological Chemistry, 286(1), 234–242. </A></LI><LI id="ref_7"><A href="https://books.google.de/books?hl=de&amp;lr=&amp;id=vgyBJbtNOcoC&amp;oi=fnd&amp;pg=PP1&amp;dq=israelachvili&amp;ots=_PQtchA1Q9&amp;sig=Bw7OdRovpmTt5FKFaZCesKrPM18#v=onepage&amp;q=israelachvili&amp;f=false">Israelachvili, J. N. (2011). Intermolecular and surface forces. Academic press.</A></LI></OL></DIV></DIV></MAIN></DIV></DIV></DIV></DIV></DIV></DIV></BODY></HTML>