<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_Munich_preResults skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><DIV id="top_title"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:Munich/preResults</SPAN></H1></DIV><DIV id="HQ_page"><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><P><TITLE>Phactory</TITLE></P><H5 class="d-inline ml-3">Phactory</H5><DIV class="collapse navbar-collapse justify-content-around" id="collapsibleNavbar"><UL class="navbar-nav"><LI id="homeButton" class="nav-item mx-3"><A class="nav-link" href="https://2018.igem.org/Team:Munich">HOME</A></LI><LI class="nav-item dropdown mx-3"><A class="nav-link dropdown-toggle" href="#" id="navbardrop" data-toggle="dropdown">
                            TEAM
                        </A><DIV class="dropdown-menu"><A class="dropdown-item" href="https://2018.igem.org/Team:Munich/Team">Team Members</A><A class="dropdown-item" href="https://2018.igem.org/Team:Munich/Collaborations">Collaborations</A><A class="dropdown-item" href="https://2018.igem.org/Team:Munich/Sponsors">Sponsors</A><A class="dropdown-item" href="https://2018.igem.org/Team:Munich/Attributions">Attributions</A></DIV></LI><LI class="nav-item dropdown mx-3"><A class="nav-link dropdown-toggle" href="#" id="navbardrop" data-toggle="dropdown">
                            PROJECT
                        </A><DIV class="dropdown-menu"><A class="dropdown-item" href="https://2018.igem.org/Team:Munich/Description">Description</A><A class="dropdown-item" href="https://2018.igem.org/Team:Munich/Design">Design</A><A class="dropdown-item" href="https://2018.igem.org/Team:Munich/Notebook">Notebook</A><A class="dropdown-item" href="https://2018.igem.org/Team:Munich/Protocols">Protocols</A><A class="dropdown-item" href="https://2018.igem.org/Team:Munich/InterLab">InterLab</A><A class="dropdown-item" href="https://2018.igem.org/Team:Munich/Model">Model</A><A class="dropdown-item" href="https://2018.igem.org/Team:Munich/Results">Results</A><A class="dropdown-item" href="https://2018.igem.org/Team:Munich/Demonstrate">Demonstrate</A></DIV></LI><LI class="nav-item dropdown mx-3"><A class="nav-link dropdown-toggle" href="#" id="navbardrop" data-toggle="dropdown">
                            PARTS
                        </A><DIV class="dropdown-menu"><A class="dropdown-item" href="https://2018.igem.org/Team:Munich/Parts">Parts Overview</A><A class="dropdown-item" href="https://2018.igem.org/Team:Munich/Basic_Part">Basic Parts</A><A class="dropdown-item" href="https://2018.igem.org/Team:Munich/Improve">Improved Parts</A><A class="dropdown-item" href="https://2018.igem.org/Team:Munich/Composite_Part">Composite Parts</A></DIV></LI><LI class="nav-item mx-3"><A class="nav-link" href="https://2018.igem.org/Team:Munich/Safety">SAFETY</A></LI><LI class="nav-item dropdown mx-3"><A class="nav-link dropdown-toggle" href="#" id="navbardrop" data-toggle="dropdown">
                            HUMAN PRACTICES
                        </A><DIV class="dropdown-menu"><A class="dropdown-item" href="https://2018.igem.org/Team:Munich/Human_Practices">Human Practices</A><A class="dropdown-item" href="https://2018.igem.org/Team:Munich/Public_Engagement">Education &amp; Engagement</A><A class="dropdown-item" href="https://2018.igem.org/Team:Munich/Oracoli">Oracoli</A><A class="dropdown-item" href="https://2018.igem.org/Team:Munich/European_Meetup">European Meetup</A></DIV></LI><LI class="nav-item dropdown mx-3"><A class="nav-link dropdown-toggle" href="#" id="navbardrop" data-toggle="dropdown">
                            AWARDS
                        </A><DIV class="dropdown-menu"><A class="dropdown-item" href="https://2018.igem.org/Team:Munich/Applied_Design">Applied Design</A><A class="dropdown-item" href="https://2018.igem.org/Team:Munich/Entrepreneurship">Entrepreneurship</A><A class="dropdown-item" href="https://2018.igem.org/Team:Munich/Hardware">Hardware</A><A class="dropdown-item" href="https://2018.igem.org/Team:Munich/Measurement">Measurement</A><A class="dropdown-item" href="https://2018.igem.org/Team:Munich/Model">Model</A><A class="dropdown-item" href="https://2018.igem.org/Team:Munich/Software">Software</A></DIV></LI><LI class="nav-item mx-3"><A class="nav-link" href="https://igem.org/2018_Judging_Form?team=Munich" target="_blank">JUDGING FORM ⇗</A></LI></UL></DIV><DIV class="pictureTitle container-fluid text-center mb-0 align-items-center text-light"><DIV class="display-2 mb-0">
                     Results
                    </DIV></DIV><DIV class="phaContainer"><MAIN class="post-content"><H2>Optimizing a Cell-Free Expression System</H2><DIV class="row"><DIV class="col-12"><P>
The first part of our project is the optimization of a cell-free expression system as a manufacturing platform for bacteriophages. For our purpose, it is necessary to produce a high quality cell extract, in a reproducible and easy manner.

We focused on achieving the following goals for our cell extract:
</P></DIV></DIV><DIV class="row"><DIV class="col-12"><UL style=""><LI>increase protein content</LI><LI>find reproducible methods of quality control</LI><LI>produce cell-extract that allows phage assembly</LI></UL></DIV></DIV><DIV class="row"><DIV class="col-12"><P>
Considering a possible commercial use of our product we decided to furthermore evaluate the potential for scaling up of our preparation protocol.

We chose to try several different approaches to achieve these goals
</P></DIV></DIV><DIV class="row"><DIV class="col-12"><UL style=""><LI>test cell cultivation in a bioreactor to enable upscaling</LI><LI>find optimal lysis conditions that produce high-quality extract</LI><LI>test scaling up of cell lysis</LI></UL></DIV></DIV><DIV class="row"><DIV class="col-12"><P>
We found that:
</P></DIV></DIV><DIV class="row"><DIV class="col-12"><UL style=""><LI>bioreactor cultivation allows for upscaling of cell extract production</LI><LI>sonication and lysozyme improve the performance of cell extract</LI><LI>cultivation and extract preparation barely impacts cell extract composition </LI><LI>lyophilization is a good choice for cell-extract storage</LI></UL></DIV></DIV><H3>
Bioreactor Cultivation Allows For Upscaling Of Cell Extract Production
</H3><DIV class="row"><DIV class="col-12 col-md-6"><P>Cultivating bacteria is the first step in cell extract preparation. The cell extract preparation protocol of Sun et al.  uses shaking flask cultivation for biomass production. It recommends cell harvest in the mid-log growth phase at OD 1.8-2.0 to produce high-quality cell extract. We recorded growth curves for shaking flask cultivation and bioreactor cultivation in a lab-scale bioreactor, to compare biomass production under different culture conditions.
</P></DIV><DIV class="col-12 col-md-6"><FIGURE class="figure"><FIGCAPTION class="figure-caption">Growth curves for cell cultivation in a shaking flask and a 2 L bioreactor. Growth curves were approximated by fitting a logistic function (line).</FIGCAPTION></FIGURE></DIV></DIV><DIV class="row"><DIV class="col-12 col-md-6"><P>The growth curve from shaking flask cultivation, revealed that the mid-log growth phase correlates to an OD of around 2. In bioreactor cultivation the mid-log growth phase is prolonged, due to the better aeration of the medium, and corresponds to an OD between 4 and 6. We harvested cells at OD 4, 5 and 6 to test which harvest point is most suitable to produce high quality cell extract.
</P></DIV><DIV class="col-12 col-md-6"><FIGURE class="figure"><FIGCAPTION class="figure-caption">Protein content (grey) and fluorescence intensity (blue) in cell extract in different cultivation conditions.</FIGCAPTION></FIGURE></DIV></DIV><DIV class="row"><DIV class="col-12"><P>
The test reveals that the protein content is higher when cells where harvested at higher OD, with OD 5 yielding the highest protein content. In addition, the cell extract from OD 5 produces the highest fluorescence intensity in the TXTL test. Compared to a cell extract from shaking flask cultivation the signal is reduced to about 60 %. The most likely cause for this result, is that a higher fraction of proteins in the OD 5 cell extract is inactive, possibly due to protein damage during cell lysis (at the time of this experiment, cell lysis was not yet fully optimized).
</P><P>
  The comparison cultivation methods revealed that cultivation in a bioreactor is an applicable alternative to shaking flask cultivation and offers great potential for upscaling of cell extract preparation. In our 2 L lab-scale bioreactor we were able to produce 20 g of cell pellet, compared to 4-5 g that 2 L of shaking flask cultivation yield.
</P></DIV></DIV><H3>
Sonication and Lysozyme Improve the Performance of Cell-Extract
</H3><H5>Sonication</H5><DIV class="row"><DIV class="col-12"><P>We tested three different cell lysis methods: beat beating, French press and sonication. Beat beating is an inexpensive and widely employed method but upscaling is problematic and the preparation is tedious and time-consuming. High-pressure cell disruption in a so called ‘French-press’ is also used but these devices are very expensive and not widely prevalent. This contradicts our effort to provide a generally-applicable protocol for preparation of home-made cell extract. A sonication device on the other hand is available in most biochemical labs as it is a commonly used method for cell lysis in protein purification protocols. Nevertheless, we started by comparing all three lysis methods with settings commonly applied for cell lysis.
</P></DIV></DIV><DIV class="row"><DIV class="col-12 col-md-6"><P>Our initial test showed that cell lysis by sonication yields extract with up to 2.8 and 4.4 fold higher expression than cell lysis by bead beating and French press respectively. These findings, combined with the limited options of optimization for both bead beating and French press lysis as well as restricted potential for scaling up of bead beating lead to the decision to focus our optimization efforts on sonication as lysis method.</P></DIV><DIV class="col-12 col-md-6"><FIGURE class="figure"><FIGCAPTION class="figure-caption"><P>Comparison of different cell lysis methods for cell extract preparation.</P></FIGCAPTION></FIGURE></DIV></DIV><DIV class="row"><DIV class="col-12"><P>Cell lysis by sonication has two main parameters that can be varied: the amplitude, which correlates to the power emitted by the sonication device and cycle number, representing the total time of sonication. Based on Yong-Chan Kwon &amp; Michael C. Jewett   we decided to use 10 s pulses with 15 s pause between pulses to prevent excessive heating of the sample.
</P></DIV></DIV><DIV class="row"><DIV class="col-12"><FIGCAPTION class="figure-caption">Screening of different amplitudes and cycle numbers for sonication.</FIGCAPTION></DIV></DIV><DIV class="row"><DIV class="col-12"><P>
  Our screening revealed that increase of amplitude as well as increase of cycle number leads to higher protein content. The TXTL test showed high fluorescence intensity only for settings with low cycle numbers. This indicates that prolonged sonication damages the cellular machinery, declining the ability for protein expression.
</P><P>
  The highest-performing extracts where those obtained by sonication at 4 cycles at 33 % amplitude and 5 cycles at 10 % amplitude. Therefore we focused our next steps on further optimization of those settings.
</P><H5>Lysozyme</H5><DIV class="row"><DIV class="col-12"><P>In the last step we made an effort to scale up the sonication step. Preliminary experiments (results not shown) indicated 4 mL as the maximal sample volume that could efficiently be lysed with the device we had at our disposal. As indicated in the publication by Yong-Chan Kwon &amp; Michael C. Jewett2 the number of sonication cycles was increased in proportion to the increase of sample volume. The presented results shown in the figure below shows that this approach was successful: Using higher sample volumes for sonication increased protein content and expression quality of the cell extract. This proves that upscaling of cell extract preparation with cell lysis by sonication is indeed feasible.
    </P></DIV></DIV><DIV class="row"><DIV class="col-12"><FIGCAPTION class="figure-caption">Comparison of cell lysis conditions: increase of sample volume and use of lysozyme.</FIGCAPTION></DIV></DIV><DIV class="row"><DIV class="col-12"><P>
      Even though previous attempts by Shresta et al.[ref 3] to use Lysozyme in cell extract preparation have been unsuccessful, we tested the effect of adding the cell-wall degrading enzyme at 1 mM concentration to the cell lysis reaction. In our experiments lysozyme increases the efficiency of cell lysis for all tested settings. Moreover, the expression quality was likewise increased after Lysozyme addition, indicating that the remaining enzyme does not interfere with protein biosynthesis.
    </P></DIV></DIV><H5>Cell Extract Processing</H5><DIV class="row"><DIV class="col-12"><P>
      Cultivation and cell lysis are the apparent most crucial steps in preparation of cell extract but there are other steps that can have a great impact onto cell extract quality. Some of our findings regarding those steps are:
    </P></DIV></DIV><DIV class="row"><DIV class="col-12"><UL style=""><LI>Freezing cell pellets overnight for next-day processing does not interfere with extract quality but confers a major simplification of cell extract preparation</LI><LI>Preliminary results indicate that omitting the run-off reaction after lysis does not significantly impair the extract quality</LI><LI>Dialysis of cell extract can be substituted by diafiltration in centrifugal filters without reducing expression yield of the resulting extract </LI></UL></DIV></DIV><H3>Characterizing Cell Extract Quality </H3><DIV class="row"><DIV class="col-12"><P>
In order to validate the success of our optimization efforts, adequate characterization of cell extract is indispensable.
Here we will give a short review about the tools for quality assessment we implemented. Further details about characterization
of these tools are available on the <A href="https://2018.igem.org/Team:Munich/Measurement">Measurement</A> page.
</P><P>
We had initially defined increase of protein content in cell extract as optimization goal. However we realized early on, that protein content alone is no measure for extract quality, because it provides no information about the activity of the cellular machinery.
</P></DIV></DIV><DIV class="row"><DIV class="col-12 col-md-6"><P>
To assess functionality of cell extract, testing of protein expression is necessary, which can be achieved by expressing fluorescent proteins and measuring the fluorescence time trace in a plate reader. To find the optimal quality control we compared the performance of different fluorescent proteins in cell extract.
</P><P>
We chose expression of mTurquoise – a variant of Cyan Fluorescent Protein (CFP) – as our favorite quality control for cell extract, as it results in high fluorescence intensity in a reproducible manner.
</P></DIV></DIV><DIV class="row"><DIV class="col-12 col-md-6"><P>
In addition, to examining protein expression we decided to analyze transcription uncoupled from translation by transcribing an RNA aptamer that binds a fluorescent dye.
</P><P>
We compared the fluorescence time trace of a Malachite Green binding aptamer in different cell extract samples. The fluorescence time traces decline after 30-50min indicating, that RNA degradation starts to prevail over transcription. Differences in the observed kinetics can be explained by variations in cell extract composition.
</P></DIV></DIV><H3>
Cultivation and Extract Preparation Barely Impacts Cell Extract Composition
</H3><DIV class="row"><DIV class="col-12 col-md-6"><P>
    The main question was, if the different performance of the cell extracts (P15, E10, myTXTL), especially concerning phage titers, is due to a divergent composition or caused by a discrepancy in activity of the relevant proteins e.g. transcription and translation related proteins
    Therefore, one student was sent to Bavarian Biomolecular Mass Spectrometry Centre (BayBioMS), where he analyzed the samples under the supervision of Dr. Christina Ludwig.
    </P><P>The results of the Mass Spectrometry gave us insight into the composition of the different TX-TLs. We were able to identify 1771 proteins in all the extracts. The results indicate a high homogeneity between all three TX-TLs, as illustrated in the heatmap.
Based on the LFQ values (Label Free Quantification) a volcano plot of two samples (Arbor/E10, Arbor/P15 and E10/P15) was generated. In general, there is no protein more frequent in Arbor TX-TL in comparison to E10 and P15. In contrast, some translation related proteins were slightly more abundant in E10 and P15 like certain tRNA ligases (PheS and PheT). The RecBCD subunits (the dsDNA degrading complex) is of similar abundance in all TX-TLs, showing the importance to consider its negative impact on the assembly. The results were further validated with DAVID, a tool for finding metabolic pathways based on proteomic data. The identified pathways indicate no correlation with transcription/translation related pathways (data not shown).
</P><P>
We concluded that the performance difference is most likely due to decreased activity during TX-TL preparation rather than a change in composition.
</P></DIV><DIV class="col-12 col-md-6"><FIGURE class="figure"><FIGCAPTION class="figure-caption">Illustration of the abundance of all sequenced proteome of duplicates from P15, E10 and myTXTL based on LFQ values. Heatmap of the label-free quantification intensities of all proteins identified. Diagram shows duplicate samples from the three cell extracts analyzed. Green indicates high expression, red indicates low expression. </FIGCAPTION></FIGURE></DIV></DIV><DIV class="row"><DIV class="col-12 col-md-4"><FIGURE class="figure"><FIGCAPTION class="figure-caption">Arbor p15</FIGCAPTION></FIGURE></DIV><DIV class="col-12 col-md-4"><FIGURE class="figure"><FIGCAPTION class="figure-caption">Arbor e10</FIGCAPTION></FIGURE></DIV><DIV class="col-12 col-md-4"><FIGURE class="figure"><FIGCAPTION class="figure-caption">e10 p15</FIGCAPTION></FIGURE></DIV><DIV class="col-12"><FIGCAPTION class="figure-caption">Volcano plots of myTXTL vs. P15 (A), myTX-TL vs. E10 (B) and E10 vs. P15 (C),  LC-MS/MS measurements of the E. coli proteome lysates were performed in duplicates on a Q-Exactive HFX instrument in data-dependent acquisition mode (Top18 method). Total protein injection amount was 0.5 µg. Gradient length was 1h. The data was then analyzed using MaxQuant and the Uniprot E. coli K12 fasta file. The LFQ data was filtered for proteins that occur in both duplicates of at least one cell lysate group (Arbor E10 P14).</FIGCAPTION></DIV></DIV><H3>
Lyophilization Is a Good Choice for Cell-Extract Storage
</H3><DIV class="row"><DIV class="col-12"><P>
With regard to global application of Phactory it is crucial to enable long-term storage and shipping of our cell extract. In order to fulfill these requirements we chose to freeze-dry our cell extract. Lyophilized extract can be stored at ambient temperature and can be reactivated by addition of only nuclease-free water.

Initial tests for the optimal lyophilization conditions revealed several important points:
</P><UL style=""><LI>cell extract quality is only preserved when cell-extract is mixed with the TXTL reaction buffer prior to lyophilization</LI><LI>the preservation of quality does not depend on the size of lyophilized extract aliquots</LI></UL></DIV></DIV><DIV class="row"><DIV class="col-12 col-md-6"><P>We tested the expression quality of several of our home-made cell extracts from fresh and lyophilized aliquots. This comparison also includes the commercially available myTXTL® cell-free expression system (arbor biosciences) and highlights two important results of our efforts to optimize production of cell extract:
</P><UL style=""><LI>our improved cell extract can be lyophilized and retains 40-70 % of its performance</LI><LI>our final settings for cell extract preparation yield extract that clearly outperforms the commercial system, with up to 9-fold higher protein expression</LI></UL></DIV><DIV class="col-12 col-md-6"><FIGURE class="figure"><FIGCAPTION class="figure-caption">Our two tested samples of cell extract retained 70 and 90 % of expression quality respectively after lyophilization.</FIGCAPTION></FIGURE></DIV></DIV><H2>
Synthetic Phage Manufacturing
</H2><DIV class="row"><DIV class="col-12"><P>
Phages were manufactured based on three components: the cell extract, an energy solution (ATP, GTP, NAD+ etc.) and a supplement
solution (aminoacids, tRNAs, pholic acid etc.). The only additional component for phage assembly is pure phage DNA. We expanded the assembly platform by
</P></DIV></DIV><DIV class="row"><DIV class="col-12"><UL style=""><LI>producing the first clinically relevant phages at therapeutic concentrations</LI><LI>manufacturing of therapeutic phages independent of a living pathogen</LI><LI>achieving similar phage titers as the commercial cell extract</LI><LI>determining the amount of DNA necessary for sufficient phage assembly</LI><LI>calculating the amount of DNA produced in the cell extract by replication</LI></UL></DIV></DIV><H3>
DNA Purification
</H3><DIV class="row"><DIV class="col-12 col-md-6"><P> An essential part of the bacteriophage manufacturing is the DNA. For the extraction of the bacteriophage DNA we started with a Phenol-Chloroform extraction protocol (link). To purity was controlled with an agarose gel, where two distinct bands where present in for all four samples the proper high mass band and a low mass band. 
</P><P> To reduce the amount of the low mass bands the protocol was changed to a column based technique (link). With this technique the incorrect band where significantly reduced in all samples, which was proven with an agarose gel.</P></DIV><DIV class="col-12 col-md-6"><FIGURE class="figure"><FIGCAPTION class="figure-caption"> The agarose gel shows the DNA of a phenol chloroform purification of the bacteriophage genomes from left to right T7, GEC-3S, T4 and NES. The ladder in the first lane from left is the 1kb extended range ladder from NEB.</FIGCAPTION></FIGURE></DIV><DIV class="col-12 col-md-6"><FIGURE class="figure"><FIGCAPTION class="figure-caption"> The agarose gel shows the DNA of bacteriophages which was column purified. The bands are from left to right, a the 1kb extended range ladder from NEB the CLB-P2, CLP-P3, T4, T5, T7, NES, FFP and GEC-3S bacteriophage.
</FIGCAPTION></FIGURE></DIV></DIV><H3>
Home-Made Cell Extract Achieves Phage Titer Comparable to the Commercial System
</H3><DIV class="row"><DIV class="col-12 col-md-6"><P>We determined the concentration of phages assembled in our cell extracts P15 and E10, which had the most promising
  results in our cell extract quality control. We compared our extracts to the commercial extract (myTXTL) and saw a similar performance.
</P><P>
  All in all our extract has the same phage assembly potential as the commercial cell extract. Moreover, lyophylization, which makes
  the cell extract more long lived, does not reduce the effiency of bacteriophage assembly. We concluded that phages could be assembled on site in a durable lyophilized cell extract.
</P></DIV><DIV class="col-12 col-md-6"><FIGURE class="figure"><FIGCAPTION class="figure-caption">Comparison of assembled T7 phage in our extracts (P15 and E10) in comparison to the commercial extract (myTXTL). Blue bars indicate an assembly within fresh extract, whereas red indicates an assembly in a lyophilized cell extract.</FIGCAPTION></FIGURE></DIV></DIV><H3>
Bacteriophages Can Be Assembled Independent of the Living Host
</H3><DIV class="row"><DIV class="col-12"><P>
Phactory has is the ability to assemble various bacteriophages, in a bacteria-independent manner. To underline this feature and demonstrate universal applicability, we assembled a variety of different E. coli phages, both DNA and RNA-based.
</P></DIV></DIV><DIV class="row"><DIV class="col-12 col-md-6"><P>The successful assembly of all phages was confirmed by plaque assay and transmission electron microscopy (TEM). In addition, DNA encoding for NES and FFP phages was used to perform assembly of these phages in our cell extract. However, we were not in possession of the respective host bacterial strains and therefore could not demonstrate successful assembly.
</P><P>
Pathogenic bacteria such as salmonella, pseudomonas and staphylococcus are prone to develop multi-drug resistance and pose an urgent or serious threat (Centers for Disease Control and Prevention, 2013. Antibiotic/Antimicrobial Resistance.). Therefore, to fulfill this medical need, phages specific for these bacteria should be assembled next in our cell-free system. Potentially, this would require co-expression of the respective sigma factors that are needed for initiation of transcription.
</P></DIV><DIV class="col-12 col-md-6"><FIGURE class="figure"><FIGCAPTION class="figure-caption"> Overview of the successfully assembled phages. MS2 (RNA-phage), T4 (DNA phage), T5 (DNA phage), T7 (DNA phage), CLB-P2 (clinically relevant), CLB-P2 (clinically relevant), GEC-3S P2 (clinically relevant).</FIGCAPTION></FIGURE></DIV></DIV><DIV class="row"><DIV class="col-12 col-md-3"><FIGURE class="figure"><FIGCAPTION class="figure-caption">Phage MS2.</FIGCAPTION></FIGURE></DIV><DIV class="col-12 col-md-3"><FIGURE class="figure"><FIGCAPTION class="figure-caption">Phage T4.</FIGCAPTION></FIGURE></DIV><DIV class="col-12 col-md-3"><FIGURE class="figure"><FIGCAPTION class="figure-caption">Phage T5.</FIGCAPTION></FIGURE></DIV><DIV class="col-12 col-md-3"><FIGURE class="figure"><FIGCAPTION class="figure-caption">Phage T7.</FIGCAPTION></FIGURE></DIV></DIV><DIV class="row"><DIV class="col-12 col-md-3"><FIGURE class="figure"><FIGCAPTION class="figure-caption">Phage CLB-P2.</FIGCAPTION></FIGURE></DIV><DIV class="col-12 col-md-3"><FIGURE class="figure"><FIGCAPTION class="figure-caption">Phage CLB-P3.</FIGCAPTION></FIGURE></DIV><DIV class="col-12 col-md-3"><FIGURE class="figure"><FIGCAPTION class="figure-caption">Phage GEC-3S.</FIGCAPTION></FIGURE></DIV></DIV><DIV class="row"><DIV class="col-12"><FIGCAPTION class="figure-caption">TEM class averages of phages assembled in cell extract. We assembled the phages MS2, T4, T5, T7 and the clinically relevant phages CLB-P2, CLB-P3, and GEC-3S.</FIGCAPTION></DIV></DIV><H3>DNA Concentration Determines Phage Titer</H3><H5>Bacteriophage Titers Correlate With DNA Concentration</H5><DIV class="row"><DIV class="col-12"><P>To prove the influence of the DNA concentration on the bacteriophage titer, cell extract reactions were prepared with varying T4 DNA concentrations. The bacteriophage production was performed. The titer of the bacteriophages was measured with the top agar method and the formed plaques were counted. The increase in DNA concentration results also in an increase in the bacteriophage concentration. This increase is nonlinear and our model predicted. This finding is probably due to a critical concentration of phage proteins, which have to be reached for capsid head assembly (similar to a critical micelle concentration).
</P></DIV></DIV><DIV class="row"><DIV class="col-12 col-md-6"><FIGURE class="figure"><FIGCAPTION class="figure-caption">Assembly of the T4 phage depending on the DNA concentration.</FIGCAPTION></FIGURE></DIV><DIV class="col-12 col-md-6"><FIGURE class="figure"><FIGCAPTION class="figure-caption">Assembly of the T7 phage depending on the DNA concentration.</FIGCAPTION></FIGURE></DIV></DIV><H5>Cell-Free Systems Replicate Phage Genomes</H5><DIV class="row"><DIV class="col-12 col-md-6"><P>The DNA sequence added to the cell-free system serves as the template for the required phage. We saw, that DNA-Polymerases can amplify the DNA segment, multiplying the amount of DNA in the cell-free reaction.
</P><P>
  To assess this effect and its dependence on deoxynucleotide triphosphates (dNTPs), we performed an absolute quantification of T7 DNA in the cell-free reaction by quantitative PCR (qPCR). A standard curve with a serial dilution of T7 DNA. We used the TXTL qPCR protocol (add link).
  We used two selected cell extracts from us (P15 and E10), which reached similar phage titers compared to the commercial cell extract  (myTXTL) in previous experiments.
   We compared the replication potential in comparison to myTXTL.
</P><P>
  The addition of dNTP to the reference reaction leads to an increase in DNA concentration by a factor of 15 in the reaction after 4 hours (290 ng compared to 19 ng). This is higher than in the myTXTL reaction without additional dNTPs, in which there is a 1.8-fold increase in DNA (91 ng compared to 51 ng) after the 4-hour reaction.
 The home-made cell extracts P10 and E15 however do not resemble this behavior.
</P><P>
It would be desirable to increase DNA amplification in our cell extracts. We therefore conducted a cause analysis, focusing on the T7 replication system. A more than 250-fold increase in processivity of the T7 DNA polymerase is achieved by its binding behavior to E. coli thioredoxin1. We suspected reduced presence of this factor in our cell extract. Thioredoxin could be added to a phage assembly reaction to further test these assumptions. However, our proteome analysis did not confirm that there were low levels of thioredoxin present in our cell extract.
</P></DIV><DIV class="col-12 col-md-6"><FIGURE class="figure"><FIGCAPTION class="figure-caption">DNA replication within cell extracts, E15 and P15 are self prepared cell extracts, which reached similar phage titers compared to the commercial cell extract (myTXTL). ctrl = control without dNTPs. </FIGCAPTION></FIGURE></DIV></DIV><H3>Modelling Phage Production in Cell Extract</H3><H2>
Modular Bacteriophage Composition
</H2><H3>
Protein engineering
</H3><DIV class="row"><DIV class="col-12"><P>
In the TX-TL system should be possible to modify phage proteins without altering their genome. This was attempted by modifying HOC (highly immunogenic capsid protein), which is part of the capsid protein structure of the T4-phage. Therefore, His-TEV-YFP-HOC was separately expressed and the purified protein was applied to our phage assembly.
</P></DIV></DIV><DIV class="row"><DIV class="col-12"><P>
For protein modification of the T4 capsid protein HOC, a plasmid for protein expression was cloned and His-YFP-HOC (82kDa) was expressed. The plasmid was transformed into BL21, expressed and puryfied by nickel affinity chromatography and gelfiltration.
        </P><P>FILM</P><P>MODEL</P></DIV></DIV><DIV class="row mt-3"><DIV class="col-12"><DIV id="accordion"><DIV class="card" style="background-color:#017FD4;"><DIV class="card-header" id="headingOne"><H5 class="mb-0"><BUTTON class="btn btn-link collapsed" data-toggle="collapse" data-target="#collapseOne" aria-expanded="false" aria-controls="collapseOne"><SPAN class="pl-3" style="color: white;">Protein Purificaton</SPAN></BUTTON></H5></DIV><DIV id="collapseOne" class="collapse" aria-labelledby="headingOne" data-parent="#accordion"><DIV class="card-body bg-white"><FIGURE class="figure"><FIGCAPTION class="figure-caption">Elution peaks of His-HOC-YFP from nickel chromatography (A) and gelfiltration (B) with corresponding 15% silverstained SDS-PAGE.</FIGCAPTION></FIGURE></DIV></DIV></DIV></DIV></DIV></DIV><DIV class="row"><DIV class="col-12"><P>
After the two chromatography steps, the purified protein had a final concentration of 70µM and no by-products were visible by silver staining. The CD-spectra (minima of 218nm) of the protein corresponds to the model and the Ramachandran plot indicates that the protein was not denatured during purification. Then, the purified protein was intentionally denatured by thermal transition. Thereby, it was confirmed that the protein is stable below 40°C. This is of importance, as phage assembly is performed at 29°C.</P></DIV></DIV><DIV class="row mt-3"><DIV class="col-12"><DIV id="accordion"><DIV class="card" style="background-color:#017FD4;"><DIV class="card-header" id="headingTwo"><H5 class="mb-0"><BUTTON class="btn btn-link collapsed" data-toggle="collapse" data-target="#collapseTwo" aria-expanded="false" aria-controls="collapseTwo"><SPAN class="pl-3" style="color: white;">Protein Characterization</SPAN></BUTTON></H5></DIV><DIV id="collapseTwo" class="collapse" aria-labelledby="headingTwo" data-parent="#accordion"><DIV class="card-body bg-white"><FIGURE class="figure"><FIGCAPTION class="figure-caption">CD spectra of His-YFP-HOC with the specific minimum at 218 nm.</FIGCAPTION></FIGURE><FIGURE class="figure"><FIGCAPTION class="figure-caption">Thermal transition of His-YFP-HOC in correlation with the absorbance.</FIGCAPTION></FIGURE></DIV><DIV class="card-body bg-white"><DIV class="col-12 col-md-6"><FIGURE class="figure"><FIGCAPTION class="figure-caption">Ramachandran plot based on the amino acid sequence.</FIGCAPTION></FIGURE></DIV></DIV></DIV></DIV></DIV></DIV></DIV><DIV class="row"><DIV class="col-12 col-md-6"><P>The purified protein was added to the assembly mix. Bacteria were transfected with the modified phages. Unbound phages and protein were removed by centrifugation. Fluorescence was measured in dependence of the proximity to the bacteria. Theoretically, YFP intensity should correlate with the binding of YFP-HOC modified phages to the bacteria. </P></DIV></DIV><DIV class="row"><DIV class="col-12 col-md-6"><FIGCAPTION class="figure-caption">Bacteria infected with (A) and without (B) YFP modified phages </FIGCAPTION></DIV></DIV><DIV class="row"><DIV class="col-12"><P>
    The purified protein was added to the assembly mix. Bacteria were transfected with the modified phages. Unbound phages and protein were removed by centrifugation. Fluorescence was measured in dependence of the proximity to the bacteria. Theoretically, YFP intensity should correlate with the binding of YFP-HOC modified phages to the bacteria.
  </P></DIV></DIV><H3>
Phage Genome Engineering
</H3><DIV class="row"><DIV class="col-12 col-md-6"><P>An additional advantage of Phactory is the possibility of rebooting bacteriophages from their genomic template, which is especially important for the genetic engineering of bacteriophages. With the self-made cell extract it was possible to manufacture an engineered MS2 RNA phage, where a polyhistidine-tag was added on the phage RNA polymerase. The genome template was generated via simple cloning methods such as gibson assembly and PCR amplification. After purification of the engineered genome, phages were assembled in our self-made P15 cell extract. A plaque assay confirmed the successful assembly of functional phages with a titer of 3 × 10<SUP>7</SUP> PFU/ml.</P><P>To test the his-tag modification the Phages were amplified in a bacterial culture flask. After lysis of the bacteria the tagged polymerase was purified from the supernatant by nickel affinity chromatography. An SDS-PAGE proved, that the 62 kDa Protein remained in the Nickel column due to the successfully engineering of an inserted His-tag. </P></DIV><DIV class="col-12 col-md-6"><FIGURE class="figure"><FIGCAPTION class="figure-caption">SDS gel of the His-tag purified MS2 RNA dependent RNA polymerase</FIGCAPTION></FIGURE></DIV></DIV><H2>
Quality Control
</H2><DIV class="row"><DIV class="col-12"><P>
Quality control covers several aspects of phage manufacturing including phage functionality, endoxin levels and DNA purity. We found that
</P></DIV></DIV><DIV class="row"><DIV class="col-12"><UL style=""><LI>we are able to produce functional phages in our cell extract</LI><LI>the cell extract of our optimized strain has an endotoxin content below the detection limit and our regular self-produced cell extract has fewer endotoxins than the commercial cell extract</LI><LI>next-generation sequencing allowed us to accurately quantify contamination</LI><LI>phenol-chloroform extraction leads to a large amount of contaminating DNA which complicates phage assembly</LI><LI>next-generation sequencing helped us to improve our purification protocols, leading to improved phage assembly</LI></UL></DIV></DIV><H3>
Assessing Phage Functionality by Plaque Assays
</H3><DIV class="row"><DIV class="col-12"><P>We performed a Plaque Assay to determine the activity of the viable phages (titer) in our assembly batch. By creating serial dilutions, we were able to calculate a plaque forming units/milliliter (PFU/ml) value. The <A href="https://static.igem.org/mediawiki/2018/2/23/T--Munich--AgarOverlayPlaqueAssay_.pdf "> plaque assay protocol </A> was used.
</P></DIV></DIV><DIV class="row"><DIV class="col-12 col-md-12"><FIGURE class="figure"><FIGCAPTION class="figure-caption"> Plaque Assay of manufactured T7 DNA -bacteriophage (top) and MS2 RNA-bacteriophage (bottom) in our self-produced cell extract P15</FIGCAPTION></FIGURE></DIV></DIV><H3>Assessing Endotoxin Levels</H3><H5>Msb-B Knockouts Reduce Endotoxin Levels By 49-Fold</H5><DIV class="row"><DIV class="col-12 col-md-6"><P>Endotoxins are pyrogens deriving from gram-negative bacteria. Their mini from any pharmaceutical product is mandatory. Therefore, or Phactory, we engineered an E. coli strain lacking lipid A, a major endotoxin component and used this bacterium to produce our cell extract To evaluate endotoxin content of different cell extracts, a Limulus Amebocyte Lysate (LAL)-test was performed according to the <A href="https://www.genscript.com/product/documents?cat_no=L00350&amp;catalogtype=Document-PROTOCOL">supplier manual</A>. As a reference, we compared the cell extract from our msbB-deficient strain (K2) to a cell extract from a wild-type strain (K4) as well as a commercial cell-free system (myTXTL, Arbor Biosciences). A solution with live <I>E. coli</I> served as a positive control.</P><P>Compared to the K4 strain our msbB-deficient K2 cell extract had 49-fold reduced endotoxin levels (0.06 EU/ml compared to 2.94 EU/ml). Other cell extracts such as the P15 cell extract (3.83 EU/ml) and the commercial myTXTL (4.65 EU/ml) had even higher endotoxin contents.</P></DIV><DIV class="col-12 col-md-6"><FIGURE class="figure"><FIGCAPTION class="figure-caption">Endotoxin content in different cell-extracts determined by LAL-Test. Error bars indicate standard deviation of the measured plateau values. Error bars indicate SD</FIGCAPTION></FIGURE></DIV></DIV><DIV class="row"><DIV class="col-12"><P>A calibration curve using known endotoxin concentrations is required for the LAL-Test. A dilution series ranging from 0.625 EU/ml to 5 EU/ml. The fitting curve is used to interpolate the concentrations in the unknown sample. The linear fit of the calibration curve had a R2 of 0.98, an intersection with the y-axis at 0.38 and a slope of 0.39 ml/EU. These values are in accordance with the requirements of the LAL-Test manufacturer. </P></DIV></DIV><DIV class="row"><DIV class="col-12"><P>Removal of endotoxins is impeded by their tendency to form stable interactions with other biomolecules2. Our method of preventing the lipid A biosynthesis is therefore superior to extensive isolation steps required for removing endotoxins in conventional phage production. </P></DIV></DIV><H3>
Nanopore Sequencing enhances DNA purification
</H3><DIV class="row"><DIV class="col-12 col-md-6"><P>Besides the cell extract, the DNA quality is the key to phage assembly. Unpure DNA (Host DNA, proteins contamination) inhibits the assembly of the phage within the cell lysate. 
</P><P>Our first attempts with classical chloroform/phenol extraction failed because of low molecular DNA (agarose gel). Therefore, we decided to search for other purification protocols, DNAseI and the Norgene kit (46800) tremendously increased DNA purity. With this DNA, we were finally able to achieve high titers by assembly.
</P><P>As the results indicate, the main contaminant in chloroform phenole extratction is <I>E. coli</I>, as the origin for the native phage DNA. DNAseI and the Norgene kit led to reduction of <I>E. coli</I> contamination. 
</P></DIV><DIV class="col-12 col-md-6"><FIGURE class="figure"><FIGCAPTION class="figure-caption"> Illustration of DNA composition after purification, 3S phage DNA was purified from the <I>E. coli</I>  host by chloroform-phenol (CM) and column extraction with and without DNAseI treatment after cell lysis.</FIGCAPTION></FIGURE></DIV></DIV><H3>
    Sequ-Into Allows For Accurate Detection of DNA Contamination Levels
    </H3><DIV class="row"><DIV class="col-12"><TABLE class="table"><P>
        The laboratory team sequenced T7, 3S, NES and FFP bacteriophages genomes (preparation <A href="https://2018.igem.org/Team:Munich/Protocols">protocols)</A>.
    </P><THEAD><TR><TH scope="col">Experiment</TH><TH scope="col">Sequencing Time</TH><TH scope="col">Reads Sequenced</TH><TH scope="col">Base-pairs sequenced</TH></TR></THEAD><TBODY><TR><TH scope="row">T7</TH><TD>12h 02min</TD><TD>424,198</TD><TD>1.27 × 10<SUP>9</SUP></TD></TR><TR><TH scope="row">3S</TH><TD>3h 42min</TD><TD>77,092</TD><TD>2.53 × 10<SUP>8</SUP></TD></TR><TR><TH scope="row">NES</TH><TD>3h 58min</TD><TD>39,501</TD><TD>2.31 × 10<SUP>8</SUP></TD></TR><TR><TH scope="row">NFFP</TH><TD>5h 42min</TD><TD>27,633</TD><TD>1.23 × 10<SUP>8</SUP></TD></TR></TBODY></TABLE><P>
        The main task after receiving genomes was to assemble the phage genomes from the Nanopore reads without the reference
        genome and gain the origin sequence. There are several tools available, but for minION Nanopore recommended tools are
        canu and miniasm, of which we finally used the latter.
    </P><P>
        Despite an excellent lab team we have been confronted with a major problem: contamination. Contamination is a problem
        regarding the therapeutic usage of the final phage extract, but also an important factor in the bioinformatics pipeline
        because contamination-based reads can confuse the assembly process. We thus developed <A href="https://2018.igem.org/Team:Munich/Software">sequ-into</A> to detect
        the contamination and also get rid of contamination-originated reads. In the context of our project, several DNA
        purification protocols were evaluated with sequ-into and allowed iterative engineering cycles in Phactory, leading to
        unparalleled purity of up to 96% (bases sequenced).
    </P><P>
        While evaluating the sequencing data with sequ-into we noticed that in the first 10% of the sequencing time of
        each experiment, more non-target reads were sequenced than in any latter time interval.
        Moreover we found that this also holds true for the first x sequenced reads, which allowed us to setup sequ-into such
        that it only analyses the first 1000 reads of a minION sequencing experiment, speeding up computation and enabling an
        en-run analysis.
    </P><DIV class="row"><DIV class="col-12 col-md-6"><FIGURE class="figure"><FIGCAPTION class="figure-caption">More non-target reads sequenced in the first 10% of the sequencing time of each experiment.</FIGCAPTION></FIGURE></DIV><DIV class="col-12 col-md-6"><FIGURE class="figure"><FIGCAPTION class="figure-caption">Also in the first x sequenced reads.</FIGCAPTION></FIGURE></DIV></DIV><P>
        After getting rid of the contamination we noticed a non-uniform coverage in the phage genome assemblies after
        re-aligning the reads to the assembly.
    </P><P>
        Rearranging the middle part which initially was “over-expressed” and re-assembling led us to the expected uniform
        coverage of the de-novo genome sequence after re-aligning the input reads.
        Finally, we could predict coding-sequences and visualize all descriptive statistics in genome diagrams.
    </P></DIV></DIV><DIV class="row"><DIV class="col-12 col-md-4"><FIGURE class="figure"><FIGCAPTION class="figure-caption">3S</FIGCAPTION></FIGURE></DIV><DIV class="col-12 col-md-4"><FIGURE class="figure"><FIGCAPTION class="figure-caption">FFP</FIGCAPTION></FIGURE></DIV><DIV class="col-12 col-md-4"><FIGURE class="figure"><FIGCAPTION class="figure-caption">NES</FIGCAPTION></FIGURE></DIV></DIV><P>More details at <A href="https://2018.igem.org/Team:Munich/DataAnalysis">Data Analysis</A> page.</P><H2>
Encapsulation
</H2><DIV class="row"><DIV class="col-12"><P>
After successful bacteriophage production and toxicity evaluation the final step of the manufacturing pipeline is the application.
While bacteriophages possess a relatively short have-life when orally administered. The encapsulation in alginate protects phages from the gastric fluid.
(Quelle: Colom, Joan, et al. &quot;Microencapsulation with alginate/CaCO 3: A strategy for improved phage therapy.&quot; Scientific reports 7 (2017): 41441.),
</P><P>
We constructed a simple <A href="https://2018.igem.org/Team:Munich/Hardware">device</A> that allows us to encapsulate our bacteriophages in alginate droplets.
</P></DIV></DIV><H3>Droplets are Monodisperse</H3><DIV class="row"><DIV class="col-12 col-md-4"><FIGURE class="figure"><FIGCAPTION class="figure-caption">Alginate composition of 1.8 % Alginate and 0.2 % low-viscosity Alginate delivers spherical Alginate microspheres.</FIGCAPTION></FIGURE></DIV><DIV class="col-12 col-md-4"><FIGURE class="figure"><FIGCAPTION class="figure-caption">Alginate encapsulated phages, DNA stained with SYBR Gold.</FIGCAPTION></FIGURE></DIV><DIV class="col-12 col-md-4"><FIGURE class="figure"><FIGCAPTION class="figure-caption">Monodisperse Alginate Droplets at 150X magnification.</FIGCAPTION></FIGURE></DIV></DIV><DIV class="row"><DIV class="col-12 col-md-6"><P>
  In order to achieve defined phage concentrations and therefore defined doses, we optimized the monodispersity of our alginate droplets.
</P><P>
  In our initial attempts to create alginate droplets the size within a batch often varied significantly. Additionally, due to
  aggregation a lot of droplets were lost. Optimization of parameters such as flow rate,
  alginate concentration and N2 pressure led to an increase of monodispersity for all tested sizes (50-300 μm).
  Specifically, an alginate concentration of 1.8 % alginate and 0.2 % low-viscosity alginate proved to be ideal. Pressure and flow rate determine the droplet sizes.
</P></DIV><DIV class="col-12 col-md-6"><FIGURE class="figure"><FIGCAPTION class="figure-caption">Monodispersity of alginate droplets with three different diameters before and after optimization of the hardware device.</FIGCAPTION></FIGURE></DIV></DIV><H3>
Bacteriophages Encapsulated In Alginate Can Withstand Gastric Acid
</H3><DIV class="row"><DIV class="col-12 col-md-6"><P>The main problem of oral application is the acidic environment in the gastric fluid, necessitating protective measures against degradation. The other requirement of phage protection is the release of functional phages in the intestines. For this reason we compared the behavior of the encapsulated phages and non-encapsulated phages in simulated gastric fluid (SGF) and simulated intenstinal fluid (SIF).
</P><P>
In SGF, the number of active non-encapsulated phages decreases by more than 99.99 % within an hour. This shows the urgent need of a form of protection against degradation to make oral application of bacteriophages possible. As a reference, we used phages that were chemically released by citrate from alginate droplets.
</P></DIV><DIV class="col-12 col-md-6"><FIGURE class="figure"><FIGCAPTION class="figure-caption">Non-encapsulated phages are highly degraded within an hour in simulated gastric fluid.</FIGCAPTION></FIGURE></DIV></DIV><DIV class="row"><DIV class="col-12 col-md-6"><P>
In comparison, the encapsulated phages were tested in SGF for the same time as the non-encapsulated phages. Afterwards, the same droplets were exposed for two hours to simulated intestinal fluid to test the release of functional bacteriophages in this environment.
</P><P>The encapsulated phages were barely released in an hour of exposure to SGF. After transfering the capsules to SIF the number of active phages reached that of the undegraded reference.
 This indicates that the encapsulation of bacteriophages in alginate capsules enables the possibility of an oral application. Further experiments could test the alginate capsules in an animal model system.
</P></DIV><DIV class="col-12 col-md-6"><FIGURE class="figure"><FIGCAPTION class="figure-caption">A small amount of encapsulated phages are released in SGF. In SIF, functional phages are released in high concentration.</FIGCAPTION></FIGURE></DIV></DIV><DIV id="phareferences" class="row"><H2>References [FIX THIS!]</H2><DIV class="col-12"><OL><LI><A href="http://www.sciencedirect.com/science/article/pii/S0092867415012003?via%3Dihub">Zetsche, B, Gootenberg, J.S., Abudayyeh, O.O., Slaymaker, I.M., Makarova, K.S., Essletzbichler, P., Volz, S.E., van der Oost, J., Regev, Aviv, Koonin, E.V., Zhang, F., 2015. Cpf1 Is a Single RNA-Guided Endonuclease of a Class 2 CRISPR-Cas System. Cell 163, 759-771. </A></LI></OL></DIV></DIV></DIV></DIV></MAIN></DIV></DIV></DIV></DIV></DIV></DIV></BODY></HTML>