<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_Munich_Measurement skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><DIV id="top_title"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:Munich/Measurement</SPAN></H1></DIV><DIV id="HQ_page"><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><P><TITLE>Phactory</TITLE></P><H5 class="d-inline ml-3">Phactory</H5><DIV class="collapse navbar-collapse justify-content-around" id="collapsibleNavbar"><UL class="navbar-nav"><LI id="homeButton" class="nav-item mx-3"><A class="nav-link" href="https://2018.igem.org/Team:Munich">HOME</A></LI><LI class="nav-item dropdown mx-3"><A class="nav-link dropdown-toggle" href="#" id="navbardrop" data-toggle="dropdown">
                            TEAM
                        </A><DIV class="dropdown-menu"><A class="dropdown-item" href="https://2018.igem.org/Team:Munich/Team">Team Members</A><A class="dropdown-item" href="https://2018.igem.org/Team:Munich/Collaborations">Collaborations</A><A class="dropdown-item" href="https://2018.igem.org/Team:Munich/Sponsors">Sponsors</A><A class="dropdown-item" href="https://2018.igem.org/Team:Munich/Attributions">Attributions</A></DIV></LI><LI class="nav-item dropdown mx-3"><A class="nav-link dropdown-toggle" href="#" id="navbardrop" data-toggle="dropdown">
                            PROJECT
                        </A><DIV class="dropdown-menu"><A class="dropdown-item" href="https://2018.igem.org/Team:Munich/Description">Description</A><A class="dropdown-item" href="https://2018.igem.org/Team:Munich/Design">Design</A><A class="dropdown-item" href="https://2018.igem.org/Team:Munich/Notebook">Notebook</A><A class="dropdown-item" href="https://2018.igem.org/Team:Munich/Protocols">Protocols</A><A class="dropdown-item" href="https://2018.igem.org/Team:Munich/InterLab">InterLab</A><A class="dropdown-item" href="https://2018.igem.org/Team:Munich/Model">Model</A><A class="dropdown-item" href="https://2018.igem.org/Team:Munich/Results">Results</A><A class="dropdown-item" href="https://2018.igem.org/Team:Munich/Demonstrate">Demonstrate</A></DIV></LI><LI class="nav-item dropdown mx-3"><A class="nav-link dropdown-toggle" href="#" id="navbardrop" data-toggle="dropdown">
                            PARTS
                        </A><DIV class="dropdown-menu"><A class="dropdown-item" href="https://2018.igem.org/Team:Munich/Parts">Parts Overview</A><A class="dropdown-item" href="https://2018.igem.org/Team:Munich/Basic_Part">Basic Parts</A><A class="dropdown-item" href="https://2018.igem.org/Team:Munich/Improve">Improved Parts</A><A class="dropdown-item" href="https://2018.igem.org/Team:Munich/Composite_Part">Composite Parts</A></DIV></LI><LI class="nav-item mx-3"><A class="nav-link" href="https://2018.igem.org/Team:Munich/Safety">SAFETY</A></LI><LI class="nav-item dropdown mx-3"><A class="nav-link dropdown-toggle" href="#" id="navbardrop" data-toggle="dropdown">
                            HUMAN PRACTICES
                        </A><DIV class="dropdown-menu"><A class="dropdown-item" href="https://2018.igem.org/Team:Munich/Human_Practices">Human Practices</A><A class="dropdown-item" href="https://2018.igem.org/Team:Munich/Public_Engagement">Education &amp; Engagement</A><A class="dropdown-item" href="https://2018.igem.org/Team:Munich/Oracoli">Oracoli</A><A class="dropdown-item" href="https://2018.igem.org/Team:Munich/European_Meetup">European Meetup</A></DIV></LI><LI class="nav-item dropdown mx-3"><A class="nav-link dropdown-toggle" href="#" id="navbardrop" data-toggle="dropdown">
                            AWARDS
                        </A><DIV class="dropdown-menu"><A class="dropdown-item" href="https://2018.igem.org/Team:Munich/Applied_Design">Applied Design</A><A class="dropdown-item" href="https://2018.igem.org/Team:Munich/Entrepreneurship">Entrepreneurship</A><A class="dropdown-item" href="https://2018.igem.org/Team:Munich/Hardware">Hardware</A><A class="dropdown-item" href="https://2018.igem.org/Team:Munich/Measurement">Measurement</A><A class="dropdown-item" href="https://2018.igem.org/Team:Munich/Model">Model</A><A class="dropdown-item" href="https://2018.igem.org/Team:Munich/Software">Software</A></DIV></LI><LI class="nav-item mx-3"><A class="nav-link" href="https://igem.org/2018_Judging_Form?team=Munich" target="_blank">JUDGING FORM ⇗</A></LI></UL></DIV><DIV class="pictureTitle container-fluid text-center mb-0 align-items-center text-light"><DIV class="display-2 mb-0">
                     Measurement
                    </DIV></DIV><DIV class="phaContainer"><MAIN class="post-content"><H2>Cell-Free Systems as Universal Expression Platform In Synthetic Biology</H2><DIV class="row"><DIV class="col-12"><P>Genetic modification of microorganisms is an essential method in the field of synthetic biology. By ‘thinking outside of the cell’ a cell free transcription/translation (TX-TL) system has emerged to be a true alternative by expanding the capabilities of a natural biological system from in vivo to in vitro <SUP><A href="#phareferences">1</A></SUP>. While commercialized TX-TL systems exhibit good reproducible results, they are only available at high cost. Contrary to commercial systems, home-made TX-TL systems are affordable but come with the drawback of high batch-to-batch variations after time-consuming preparation steps. Further investigations in the optimization of the protocol for TX-TL preparation are necessary to create an easily accessible platform for the research community. Our iGEM team developed a protocol for the easy and high-quality production of a cell extract, which can be reproduced in any laboratory without the need for expensive equipment. This TX-TL system can be used to measure protein expression, genetic circuit performance and other synthetic biology tools.</P></DIV></DIV><H2>Optimization of TX-TL Preparation and Quality Control</H2><H3>Translation Efficiency</H3><DIV class="row"><DIV class="col-12"><P>To obtain a high quality cell extract, we focused on the optimization of several key factors:
    The culture conditions were optimized by screening for the optimal cell density to harvest cells. Assessing several different lysis methods, we identified sonication as the cheapest, most accessible and reproducible way for the production of our TX-TL protein expression system. Furthermore, the upscaling of TX-TL preparation was demonstrated by using a bioreactor for cell cultivation. 
    Elaborate steps like the removal of the endotoxin Lipid A from the TX-TL were rendered obsolete by designing a suitable mutant strain for TX-TL preparation. For this purpose, TX-TL was prepared with a strain lacking the msbB gene to disrupt the lipid A biosynthesis pathway. By this the lipid A concentration could be lowered by a factor of 49 compared to the cell extract prepared using the wild-type strain (0.06 EU/mL).</P><P>The produced TX-TL went through further quality control checkpoints consisting of measuring  protein content, as well as an activity assay via expression level analysis. The protein concentration was determined by using a commercial BCA protein assay kit. Compared to the commercial TX-TL, our home-made TX-TL shows a similar protein content of about 16 mg/mL. </P><P>However, a distinct and functional quality control requires the determination of the protein expression rate. This was done by expressing fluorescent proteins (GFP, YFP, RFP, CFP) and recording the corresponding signal using standard lab equipment like a plate reader. Extending the mRNA of the fluorescent protein mTurquoise2 (mTQ2) with the sequence for the malachite green aptamer (MG), we planned to determine the transcription and translation rate in parallel. Unfortunately, we could not express a functional fused mTQ2-MG and instead measured the transcription and translation separately by using the single components on two different plasmids.</P></DIV></DIV><DIV class="row"><DIV class="col-12"><FIGCAPTION class="figure-caption">Fluorescence curves of the different fluorescent proteins in cell extract. Proteins analysed were GFP, YFP, RFP and mTQ.</FIGCAPTION></DIV></DIV><DIV class="row mt-5"><DIV class="col-12"><P>Analysing early results, we observed that functional GFP was expressed, but the measured signal did not show a coherent trend. This phenomenon was due to several autofluorescent metabolites produced in the cell extract<SUP><A href="#phareferences">3</A></SUP>. Therefore GFP had to be excluded from further measurements.</P></DIV></DIV><DIV class="row"><DIV class="col-12"><P>Moreover, mTQ2 showed the fastest maturation time and lowest signal-to-noise ratio of the tested fluorescent proteins. Due to its long fluorescent lifetimes (&gt; 3.7 ns) and high quantum yields (&gt; 0.8) it delivers significant results for the quality control of the translation performance of our TX-TL<SUP><A href="#phareferences">4</A></SUP>.</P></DIV></DIV><DIV class="row"><DIV class="col-12"><P>The expression of mTurquoise2 showed a five times higher fluorescent signal in our home-made TX-TL than in the commercialized TX-TL and therefore confirms that our optimized protocol produces cell extract of high translation efficiency. Although the overall protein expression rate can be observed with this method, the transcription rate has to be determined using a different approach.</P></DIV></DIV><H3>Transcription Efficiency</H3><DIV class="row"><DIV class="col-12"><P>For an excellent quality control, the transcription efficiency of our home made TX-TL was analyzed separately. We used a plasmid containing a malachite green aptamer downstream of a T7 promoter. 
The malachite green aptamer, a small RNA transcript, is able to bind a specific ligand, malachite green and enhances its initial fluorescence &gt;2000 fold<SUP><A href="#phareferences">5</A></SUP>.</P><P>By observing the fluorescence signal over time the transcription levels of different TX-TL batches can be compared.</P></DIV></DIV><H2>Cell Extract – A Platform For Qualified Comparison Of BioBrick<SUP>TM</SUP></H2><DIV class="row"><DIV class="col-12"><P>These results prove our home-made TX-TL as a high-quality expression platform regarding transcription as well as translation. Thus, it forms a perfect platform for testing various biological parts listed in the iGEM Registry. The standardized expression system allows a more reliable comparison of BioBricks for iGEM teams around the world.
</P></DIV></DIV><H2>Data Analysis</H2><DIV class="row"><DIV class="col-12"><P>One of the most essential aspects when it comes to manufacturing therapeutics and, thus, a major
            consideration while designing Phactory is quality control. To tackle this issue from a bioinformatician’s 
            point-of-view, we analysed <A href="https://nanoporetech.com/products/minion">minION Oxford Nanopore</A> 
            sequencing data in order to draw conclusions on purity, origin, and functionality of the bacteriophages genome. 
        </P><P>For this purpose, the wetlab team sequenced several phage genomes: T7, 3S, NES, FFP. Analysis of the results 
            has been performed using an in-house developed software poreSTAT. For each sequencing sample, it is 
            determined how many bases are sequenced, what base-pair yield has been achieved and how many minION pores
            were used.
        </P><P>
            Considering the sequence in which the reads have been acquired, it can nicely be seen how the used chip 
            gets worn out with each sequencing experiment. While for the first sequencing experiment many pores are 
            show high average read lengths, the used pores average read lengths decreases with the number of 
            sequencing experiments.
        </P></DIV></DIV><DIV class="row"><DIV class="col-12 col-md-6"><FIGURE class="figure"><FIGCAPTION class="figure-caption">T7</FIGCAPTION></FIGURE></DIV><DIV class="col-12 col-md-6"><FIGURE class="figure"><FIGCAPTION class="figure-caption">3S</FIGCAPTION></FIGURE></DIV></DIV><DIV class="row"><DIV class="col-12 col-md-6"><FIGURE class="figure"><FIGCAPTION class="figure-caption">NES</FIGCAPTION></FIGURE></DIV><DIV class="col-12 col-md-6"><FIGURE class="figure"><FIGCAPTION class="figure-caption">FFP</FIGCAPTION></FIGURE></DIV></DIV><DIV class="row"><DIV class="col-12"><TABLE class="table"><P>The read and base-pair yield is summarized in the following table:</P><THEAD><TR><TH scope="col">Experiment</TH><TH scope="col">Sequencing Time</TH><TH scope="col">Reads Sequenced</TH><TH scope="col">Base-pairs sequenced</TH></TR></THEAD><TBODY><TR><TH scope="row">T7</TH><TD>12h 02min</TD><TD>424,198</TD><TD>1.27 × 10<SUP>9</SUP></TD></TR><TR><TH scope="row">3S</TH><TD>3h 42min</TD><TD>77,092</TD><TD>2.53 × 10<SUP>8</SUP></TD></TR><TR><TH scope="row">NES</TH><TD>3h 58min</TD><TD>39,501</TD><TD>2.31 × 10<SUP>8</SUP></TD></TR><TR><TH scope="row">NFFP</TH><TD>5h 42min</TD><TD>27,633</TD><TD>1.23 × 10<SUP>8</SUP></TD></TR></TBODY></TABLE></DIV></DIV><H3>Phage Genome Assembly</H3><DIV class="row"><DIV class="col-12"><P>
            Genome assembly refers to aligning and merging fragments in order to reconstruct the original sequence. 
            Particularly Nanopore sequencing is well suitable for genome assembly since its long reads allow to reduce 
            the ambiguity of highly similar and repetitive sequences. The read distributions of the sequencing experiments 
            are shown in Figure 2.
        </P></DIV></DIV><DIV class="row"><DIV class="col-12 col-md-6"><FIGURE class="figure"><FIGCAPTION class="figure-caption">Figure 2a: T7</FIGCAPTION></FIGURE></DIV><DIV class="col-12 col-md-6"><FIGURE class="figure"><FIGCAPTION class="figure-caption">Figure 2b: 3S</FIGCAPTION></FIGURE></DIV></DIV><DIV class="row"><DIV class="col-12 col-md-6"><FIGURE class="figure"><FIGCAPTION class="figure-caption">Figure 2c: NES</FIGCAPTION></FIGURE></DIV><DIV class="col-12 col-md-6"><FIGURE class="figure"><FIGCAPTION class="figure-caption">Figure 2d: FFP</FIGCAPTION></FIGURE></DIV></DIV><DIV class="row"><DIV class="col-12"><P>
            During the initial screening it was observable that the read lengths do not approach the expected 
            length of the phage genomes in the 50-70 kbp range for T7 and 100 kbp range for the remaining phages. 
            Smaller reads generally lead to more ambiguity and but are inevitable due to experimental limitations.
        </P><P>
            The usual assembly algorithms require the reference genome. As bacteriophage genomes can be highly mosaic, 
            i.e. the genome of many phage species appear to be composed of numerous individual modules, we could not use 
            the reference from the database. Consequently, we were forced to perform a <I>de novo</I> assembly, a method of 
            creating the original sequence by aligning and merging fragments without the aid of a reference genome.
        </P><P>
            There are several tools available for <I>de novo</I> genome assembly. Unfortunately, most approaches have been 
            developed for short-read genome assembly (2nd generation sequencing, Illumina) and employ a de bruijn graph 
            approach. 
        </P><P>
            Here we have 3rd generation sequencing data which must be handled totally different from old short-read 
            sequencing data: the reads are less perfect in terms of sequencing errors. While short-reads nowadays have 
            error-rates of about 1% (e.g. 1 base out of 100 is reported incorrectly), this error is up to 15% for 
            nanopore sequencing data using newest sequencing chemistry (R9.4 at the time of wiki-freeze). 
        </P><P>
            Assemblers suitable for 3rd generation sequencing data are nominal, <A href="https://github.com/marbl/canu">canu</A>  and <A href="https://github.com/lh3/miniasm">miniasm</A> being the 
            most prevalent ones. Both rely on a overlap-layout-consensus approach, which, historically, can be seen 
            as the father of all assemblers (see celera assembler).
        </P><P>
            We first used canu to assemble our genomes. The main limitation to any approach in 3rd generation 
            sequencing data analysis is contamination. Particularly for assembly, contamination is disastrous 
            because it can lead the assembler into faulty assemblies – depending on the phylogenetic distance of 
            the original sample and the contamination. In the worst case, the assembler is the collapses all 
            organism sequences or gives no output sequence at all.
        </P><P>
            We thus developed <A href="https://2018.igem.org/Team:Munich/Software">sequ-into</A> to first detect 
            the contamination and also get rid of contamination-originated 
            reads. More on the performance and finding while using sequ-into can be found at 
            our <A href="https://2018.igem.org/Team:Munich/Software">Software</A> page.
        </P><P>
            After eliminating contamination we noticed non-uniform coverage of the sequence  
            in the phage genome assemblies after re-aligning the reads to the assembly, which can be seen in Figure 3.
        </P></DIV></DIV><DIV class="row"><DIV class="col-12"><FIGURE class="figure"><FIGCAPTION class="figure-caption">Figure 3: non-unform coverage of FFP sequence</FIGCAPTION></FIGURE><P>
            In theory, we should observe uniform coverage over the full genome since there is no bias for read 
            template generation during sample preparation (due to random primers).
        </P><P>
            However, the first half of the sequence has a lower coverage than the remaining part and there is 
            a high-coverage region in the middle and end of the assembled genome.
        </P><P>
            Terminal repeats are common in phage genomes. Therefore, it can happen that double coverage within 
            the sequence occurs due to missassembly of the genome. As a result, initial and terminal residues 
            of the sequences are forcematched in the interim of the assembled sequence rather than at its edges. 
        </P><P>
            We thus tried to use the other assembler, miniasm, which is known for very fast assemblies but little 
            error correction. However, this error correction can be achieved by combining miniasm with <A href="https://github.com/lh3/minimap">minimap</A>
            for read mapping and <A href="https://github.com/isovic/racon">racon</A> for polishing the sequences.
        </P></DIV></DIV><DIV class="card mt-2"><DIV class="card-header" id="heading2" style="background-color: #018DCD;"><H5 class="mb-0"><BUTTON class="btn btn-link collapsed" data-toggle="collapse" data-target="#collapse2" aria-expanded="false" aria-controls="collapse2"><SPAN class="text-white font-weight-bold">The assembly pipeline</SPAN></BUTTON></H5></DIV><DIV id="collapse2" class="collapse" aria-labelledby="heading2" data-parent="#accordion"><DIV class="card-body"><DIV class="row mt-2"><CODE style="width: 100%">
#!/usr/bin/env sh
INREADS=$1
ASMFOLDER=$2
ASMPREFIX=$3
THREADS=$4
if [ -z &quot;$4&quot; ]
then
    THREADS=4
fi
# path to used executables
MINIMAP2=minimap2
MINIASM=miniasm
GRAPHMAP=graphmap
RACON=racon
# first we must overlap all reads with each other
$MINIMAP2 -x ava-ont -t$THREADS $INREADS $INREADS &gt; $ASMFOLDER/$ASMPREFIX.paf
# then miniasm can create alignment
$MINIASM -f $INREADS $ASMFOLDER/$ASMPREFIX.paf &gt; $ASMFOLDER/$ASMPREFIX.gfa
# extract unitigs (an assembly of fragments for which there are no #competing choices in terms of internal overlaps) from miniasm
awk '$1 ~/S/ {print &quot;&gt;&quot;$2&quot;\n&quot;$3}' $ASMFOLDER/$ASMPREFIX.gfa &gt; $ASMFOLDER/$ASMPREFIX.unitigs.fasta
# align reads with unitigs
$MINIMAP2 $ASMFOLDER/$ASMPREFIX.unitigs.fasta $INREADS &gt; $ASMFOLDER/$ASMPREFIX.unitigs.paf
# find contigs (contiguous join of unitigs)
$RACON $INREADS $ASMFOLDER/$ASMPREFIX.unitigs.paf $ASMFOLDER/$ASMPREFIX.unitigs.fasta &gt; $ASMFOLDER/$ASMPREFIX.contigs.fasta
~/progs/minimap2/minimap2 -x map-ont -a -t$THREADS $ASMFOLDER/$ASMPREFIX.contigs.fasta $INREADS &gt; $ASMFOLDER/$ASMPREFIX.reads.mm2.sam
$GRAPHMAP align -r $ASMFOLDER/$ASMPREFIX.contigs.fasta -d $INREADS -o $ASMFOLDER/$ASMPREFIX.reads.gm.sam</CODE><P>And can be started simply from the command-line using: <CODE>./assemble.sh FQ_file PATH_TO_ASM_FOLDER PREFIX_of_output&gt;</CODE></P></DIV></DIV></DIV></DIV><DIV class="row"><DIV class="col-12"><P>
    This finally led to a reasonable assembly after rearranging the aforementioned sequences which initially 
    were overrepresented in the sequence.  
</P></DIV></DIV><H3>Phage Genome Annotation</H3><DIV class="row"><DIV class="col-12"><P>
        After having a core genome we want to check how many protein-coding genes we can find in the genome. 
        Two of the programs widely used in the field are glimmer and genemark.
    </P><P>
        Since genermark has been shown to perform better in benchmark tests, we used this tool for the 
        genome annotation. We ran the tool on the assembled genome in FASTA format generating a gene annotation file 
        (gff3) for the genome highlighting all coding sequences.
        
        For easier and more compact usage, we transformed the genome in fasta format with the annotation in gff3 
        into the embl flat file format.
    </P></DIV></DIV><DIV class="row"><DIV class="col-12"><TABLE class="table"><P>
                In conclusion, we can describe the assembled genomes as follows:
            </P><THEAD><TR><TH scope="col">Genome</TH><TH scope="col">#Genes reference</TH><TH scope="col">#Genes</TH><TH scope="col">#Genes+</TH><TH scope="col">#Genes-</TH><TH scope="col">%CG content</TH><TH scope="col">Genome Length</TH></TR></THEAD><TBODY><TR><TH scope="row">T7</TH><TD><A href="https://www.ncbi.nlm.nih.gov/genome/10265?genome_assembly_id=222363">60</A></TD><TD>70</TD><TD>69</TD><TD>1</TD><TD>0.4833</TD><TD>39,684</TD></TR><TR><TH scope="row">3S</TH><TD><A href="https://www.ncbi.nlm.nih.gov/genome/?term=3S+phage">277</A></TD><TD>423</TD><TD>305</TD><TD>118</TD><TD>0.4019</TD><TD>164,860</TD></TR><TR><TH scope="row">NES</TH><TD>969</TD><TD>821</TD><TD>148</TD><TD>0.3396</TD><TD>373,576</TD></TR><TR><TH scope="row">FFP</TH><TD>534</TD><TD>353</TD><TD>181</TD><TD>0.3522</TD><TD>368,471</TD></TR></TBODY></TABLE><P>It can be seen that the number of detected genes using genemark is higher than the number of the genes 
        in the reference. This is most likely due to incorrectly sequenced bases leading to an early stop codon. 
        More sophisticated polishing steps or higher quality Illumina sequencing reads could possibly avoid this.
    </P><P>
        Using the embl flat file format we visualized the phage genomes in a circular genome diagram plot.
    </P></DIV></DIV><DIV class="row"><DIV class="col-12 col-md-4"><FIGURE class="figure"><FIGCAPTION class="figure-caption">3S</FIGCAPTION></FIGURE></DIV><DIV class="col-12 col-md-4"><FIGURE class="figure"><FIGCAPTION class="figure-caption">FFP</FIGCAPTION></FIGURE></DIV><DIV class="col-12 col-md-4"><FIGURE class="figure"><FIGCAPTION class="figure-caption">NES</FIGCAPTION></FIGURE></DIV></DIV><DIV class="row"><DIV class="col-12"><P>
        Here we must note several things. For the 3S genome, we can see that at certain positions we see a high 
        decrease in the coverage (at 58kbp, 72kbp and 110kbp). At these positions no reads align to the reference genome. 
        Since this realignment was performed using <A href="https://github.com/isovic/graphmap">graphmap</A>, while the inital assembly pipeline uses <A href="https://github.com/lh3/minimap2">minimap2</A> internally, 
        this could also be an alignment artifact.
    </P><P>
        For the NES genome we can see a similar behaviour at 330kbp. Additionally we can see two spikes at the ends of 
        the genome. Finally the FFP genome again has the same problems as the NES genome ends.
    </P></DIV></DIV><H3>Conclusion</H3><DIV class="row"><DIV class="col-12"><P>
        Sequ-into has been shown to be able to allow rapid genome assembly from a Nanopore sequencing sample such 
        as encountered in Phactory manufacturing experiments. We use state-of-the-art 3rd generation sequencing 
        data analysis tools within our framework to overcome difficulties frequently experienced by scientists in 
        Nanopore applications. In result, we not only introduce our step by step evolved results of the analysis, 
        assembly and finally annotation of the bacteriophage genomes but also make clear how to deal with Nanopore 
        sequencing data already widely used in synthetic biology and rising. Furthermore our custom phage genome of 
        the bacteriophage 3S allows further analysis and usage in the manufacturing cycle, for example, a fast 
        contamination detection using sequ-into.
    </P></DIV></DIV><DIV id="phareferences" class="row"><H2>References</H2><DIV class="col-12"><OL><LI><A href="">Hodgman, C. E., Jewett, M. C. (2012). Cell-Free Synthetic Biology: Thinking Outside the Cell. MetabEng. 2012 May; 14(3): 261–269.</A></LI><LI><A href="">Kremers, G-J., Gilbert, S. G., Cranfill, P. J., Davidson, M. W. &amp; Piston, D. W. (2011). Fluorescent proteins at a glance. J Cell Sci. 124(2): 157–160.</A></LI><LI><A href="http://iopscience.iop.org/article/10.1088/2050-6120/4/4/042005">Galbán, J., Sanz-Vicente, I., Navarro, J., &amp; de Marcos, S. (2016). The intrinsic fluorescence of FAD and its application in analytical chemistry: a review. Methods and applications in fluorescence, 4(4), 042005.</A></LI><LI><A href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3316892">Goedhart, J., Von Stetten, D., Noirclerc-Savoye, M., Lelimousin, M., Joosen, L., Hink, M. A., ... &amp; Royant, A. (2012). Structure-guided evolution of cyan fluorescent proteins towards a quantum yield of 93%. Nature communications, 3, 751.
</A></LI><LI><A href="  https://pubs.acs.org/doi/abs/10.1021/ja037994o">Babendure, J. R., Adams, S. R., &amp; Tsien, R. Y. (2003). Aptamers switch on fluorescence of triphenylmethane dyes. Journal of the American Chemical Society, 125(48), 14716-14717.
</A></LI><LI><A href="https://www.ncbi.nlm.nih.gov/pubmed/28298431">Koren S, Walenz BP, Berlin K, Miller JR, Bergman NH, Phillippy AM. Canu: scalable and accurate long-read assembly via adaptive k-mer weighting and repeat separation. Genome Research. 2017;27(5):722-736. doi:10.1101/gr.215087.116.
</A></LI><LI><A href="https://www.nature.com/articles/ncomms11307"> Sovic, I., Šikić, M., Wilm, A., Fenlon, S.N., Chen, S.L., &amp; Nagarajan, N. (2016). Fast and sensitive mapping of nanopore sequencing reads with GraphMap. Nature communications, 7, 11307.
</A></LI><LI><A href="https://arxiv.org/abs/1512.01801"> Heng Li; Minimap and miniasm: fast mapping and de novo assembly for noisy long sequences, Bioinformatics, Volume 32, Issue 14, 15 July 2016, Pages 2103–2110.
</A></LI><LI><A href="https://academic.oup.com/bioinformatics/article/24/8/1035/212858"> Gennady Denisov, Brian Walenz, Aaron L. Halpern, Jason Miller, Nelson Axelrod, Samuel Levy, Granger Sutton; Consensus generation and variant detection by Celera Assembler, Bioinformatics, Volume 24, Issue 8, 15 April 2008, Pages 1035–1040.
</A></LI><LI><A href="https://academic.oup.com/nar/article/27/23/4636/1064678"> Arthur L. Delcher, Douglas Harmon, Simon Kasif, Owen White, Steven L. Salzberg; Improved microbial gene identification with GLIMMER, Nucleic Acids Research, Volume 27, Issue 23, 1 December 1999, Pages 4636–4641.
</A></LI><LI><A href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC1160247/"> Besemer J, Borodovsky M. GeneMark: web software for gene finding in prokaryotes, eukaryotes and viruses. Nucleic Acids Research. 2005;33(Web Server issue):W451-W454. doi:10.1093/nar/gki487.
</A></LI><LI><A href="https://doi.org/10.1093/bfgp/elr035"> Zhenyu Li, Yanxiang Chen, Desheng Mu, Jianying Yuan, Yujian Shi, Hao Zhang, Jun Gan, Nan Li, Xuesong Hu, Binghang Liu, Bicheng Yang, Wei Fan; Comparison of the two major classes of assembly algorithms: overlap–layout–consensus and de-bruijn-graph, Briefings in Functional Genomics, Volume 11, Issue 1, 1 January 2012, Pages 25–37.
</A></LI></OL></DIV></DIV></MAIN></DIV></DIV></DIV></DIV></DIV></DIV></BODY></HTML>