<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_SCU-China_modeling_algorithm skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><DIV id="top_title"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:SCU-China/modeling/algorithm</SPAN></H1></DIV><DIV id="HQ_page"><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><P><TITLE>Team:SCU-China - 2018</TITLE></P><DIV style="width:100%"><DIV><UL class="navbar" id="navbar"><LI><A href="https://2018.igem.org/Team:SCU-China">Home</A></LI><LI><A>Project</A></LI><LI><A href="https://2018.igem.org/Team:SCU-China/project/overview">Overview</A></LI><LI><A href="https://2018.igem.org/Team:SCU-China/project/regulation">Precise Regulation</A></LI><LI><A href="https://2018.igem.org/Team:SCU-China/project/circuits">Logic Circuits</A></LI><LI><A href="https://2018.igem.org/Team:SCU-China/project/indigo">CRISProgrammer of Indigo</A></LI><LI><A href="https://2018.igem.org/Team:SCU-China/project/parts">Parts</A></LI><LI><A href="https://2018.igem.org/Team:SCU-China/project/interlab">Interlab</A></LI></UL><LI><A>Modeling</A></LI><LI><A href="https://2018.igem.org/Team:SCU-China/modeling/prediction">Mismatch</A></LI><LI><A href="https://2018.igem.org/Team:SCU-China/modeling/algorithm">Orthogonality</A></LI><LI><A>Human practice</A></LI><LI><A href="https://2018.igem.org/Team:SCU-China/practice/collaboration">Collaboration</A></LI><LI><A href="https://2018.igem.org/Team:SCU-China/practice/meet">Meet Up</A></LI><LI><A href="https://2018.igem.org/Team:SCU-China/practice/education">Education &amp; Public</A></LI><LI><A>Judging</A></LI><LI><A href="https://2018.igem.org/Team:SCU-China/judging/human">Integrated Human Practice</A></LI><LI><A href="https://2018.igem.org/Team:SCU-China/judging/improve">Improve</A></LI><LI><A href="https://2018.igem.org/Team:SCU-China/judging/model">Model</A></LI><LI><A href="https://2018.igem.org/Team:SCU-China/judging/demonstrate">Demonstrate</A></LI><LI><A href="https://2018.igem.org/Team:SCU-China/attribution">Attribution</A></LI><LI><A href="https://2018.igem.org/Team:SCU-China/safety">Safety</A></LI><LI><A href="https://2018.igem.org/Team:SCU-China/notebook">Notebook</A></LI></DIV><DIV class="content"><DIV class="main"><DIV class="sidebar "><UL><LI><A href="orthogon">Orthogonality</A></LI><LI><A href="2018.igem.org/Team:SCU-China/modeling/prediction">Mismatch</A></LI></UL></DIV><DIV class="maincontent" id="orthogon"><DIV class="phrase">
                In our experiment, we need to construct a sgRNA sequence that is orthogonal to both the genome and the plasmid.
                The algorithm steps are as follows:
                <OL><LI>Sequence summary: E.coli genome is sequence α, with length m; All plasmid sequences that need to be
                        orthogonal are spliced into a sequence β of length n.
                    </LI><LI>Genome orthogonal screening: A randomly generated sgRNA sequence λ that ends in NGG together with all
                        23 consecutive sequences on sequence α are ruled as heterotopic or, and their total weighted scores
                        are obtained. Then reverse α and do the same thing. Eventually, we can get two score sequences of
                        length m-22. The distribution function with the total weight points in the entire sample space was
                        calculated in advance by means of parameter estimation, and then different threshold values of the
                        orthogonal exponential threshold μ_1were selected according to different orthogonality requirements.
                        The threshold value used in this experiment was 63 (90%). Retrieve two socre sequences.If there is a
                        score value below μ1, it is considered not to satisfy genomic orthogonality;if the conditions are met,
                        proceed to the next step. In addition, we used some empirical weight values - Weights 1 to 10 are assigned
                        at a time from 10 digits away from the PAM end,10 in the seed region;t he 10 sites in the seed region
                        assign an average value of 10; The 21st bit weight is 0,22 and 23 bit weight is 20.
                        <DIV class="showimg">Fig i</DIV><DIV class="showimg">Fig 2</DIV></LI><LI>Plasmid orthogonal screening: Repeat algorithm in step 2 using the sequence from step 2 and
                        sequence β. And remove the weight score of two plasmid spliced sequences that do not exist in the
                        bacteria.The screening was done with threshold value μ_2 = 43.Record the filtered sequence.
                    </LI><LI>Repeat steps 1, 2, and 3 until the number of recorded sequences reaches 600.</LI><LI>Verify the orthogonality of all the obtained sequences among each other, and finally obtain the
                        20 sequences that are most orthogonal to the genome and plasmid. The result matrix obtained by mutual
                        verification is as follows:
                        <DIV class="showimg">Fig 3</DIV>
                        Each pane shows the value of the orthogonality between two sequences.So it is clear we should choose the sequences on the top left.
                        
                        All codes were written in Fortran and can be find <A href="https://static.igem.org/mediawiki/2018/e/ea/T--SCU-China--fortrancode.txt">here</A>.
                    </LI></OL><DIV class="subtitle">References</DIV><DIV class="block">[1] Bikard, D., Jiang, W., Samai, P., Hochschild, A., Zhang, F., &amp; Marraffini, L. A.
                    (2013). Programmable repression and activation of bacterial gene expression using an engineered
                    crispr-cas system. Nucleic Acids Research, 41(15), 7429-7437.</DIV></DIV></DIV></DIV></DIV></DIV><DIV class="footer">SCU_China 2018 iGEM Team</DIV></DIV></DIV></DIV></DIV></DIV></BODY></HTML>