Team:SCU-China/project/regulation
Team:SCU-China - 2018
Home
Project
Overview
Precise Regulation
Logic Circuits
CRISProgrammer of Indigo
Parts
Interlab
Modeling
Mismatch
Orthogonality
Human practice
Collaboration
Meet Up
Education & Public
Judging
Integrated Human Practice
Improve
Model
Demonstrate
Attribution
Safety
Notebook
·Background
·Mismatch
·Logic Circuits
·Future Work
Background
Do more with less – let’s not re-invent the wheel – reusable genetic modules. This year, SCU_China iGEM team is taking advantage of dCas9 protein to regulate the expression of specific metabolic pathway. However, metabolic regulation is convoluted and precise, the change in expression level of one intermediate product or one transcription factor does a domino effect on final metabolic production amount people required. In plants, the various degree of over-expression of a transcription factor spots at the turning point of metabolic pathway, the ultimate products will be disparate. Some secondary metabolic products are unstable and prone to oxidize in extracellular environment. And the feature prevents manufacturing industry to take advantage of them easily. Crucial chemical modification or further enzyme-catalyze reaction is required to protect them. Since cannot economically create the pristine intracellular environment to store secondary metabolic products, protection strategy should be seriously considered. In order to protect and obtain the definite products, the precise regulation of enzyme is necessary. The dCas9 protein with its own special characters may do help to solve the complicated problem.
Mismatch
CRISPRi (by using dCas9-sgRNA complex to block the transcription elongation in promoter region or coding sequence region) targeting is based on the base-pairing principle. Previously reported that single or multiple mutations introduced into base-paring region of sgRNA can have disparate levels of repression. Besides, the 12-nt sequence that targets the region adjacent to the NGG PAM motif site constitutes a seed region that is all-important for effective gene regulation (Figure 1). Mismatches in seed region generally reduce the repression by 79-90%. A single mismatch in other 8-nt sequence, only causes a modest less than 50% decreased repression, though1. But with the gradually increasing mismatches in base-repairing region of sgRNA, the repression                         level can be regulated inch by inch2
Inspired by the detail, SCU_China iGEM team would like to apply the trait of CRISPR/dCas9 system to the precise regulation of genes. Above all, because of the huge data and preparatory work our team should do, if we decided to pick up the idea; our team model it at first. Several factors are taken into consideration: 1) The position of mismatches. 2) The number of base mismatches. 3) The continuous base mismatches. 4) Thermodynamics structure relationship of mismatches. When the mutations happen on seed region, the repression level is persistently low. While the mismatches happen on off-seed region, once the amounts of that live up to 3, the repression level slumps immediately. Unfortunately, it seems the mismatch strategy cannot work perfectly like our team wanted based on the modeling results .
Logic Circuits
With inspiration of our last-year project Rhythmic Production of Melatonin in E.coli, the powerful logic circuits can help us achieve abundance of regulation goals. Our team turn to alternative ways, the intricate logic, to precisely regulate gene expression. Like repressilator, the cycling-repression system which couples with quorum sensing system, exerting end-product itself to generate periodically3. In our design (Figure 2), under the control of inducible promoter, the repression level can be influenced by the amount of inducer. More complicated the logic circuits are, the more complex function they can possess
Future Work: Versatile Promoter
In our experiment, we were impeded by a significant problem. The expression of sgRNA requires that it should exactly transcript from the transcription start site. If there were any other nucleotides which are not linked to target, at the 5’ end of sgRNA, it would have an impact on the efficiency of repression1. Thus, to know more about the mechanism of a certain promoter is important. Nevertheless, the demand to affirm transcription start site of different promoters grows as the more sophisticated the logic circuits are. It’s a huge work for everyone who wants to use CRISProgrammer.                         Combining the need for well-known promoters and precise regulation, team SCU_China designed versatile promoters for standardized and modularized use.                         The sgRNA targets the (-35) ~ (-10) region of a promoter will have the most demonstrated effect2. Beyond the region, the repression level decreases because of the distinctive distance between the core promoter region and the blocking site (Figure 3).
By modeling, our team used an algorithm to generate a set of orthogonal sgRNAs, in order to reduce the crosstalk to the maximum extent. We filter the genome of Escherichia coli (BL21), the sequence of backbone of our minimid, pSB1C3, to insure the orthogonality. Then we synthesize the set of sgRNAs as a PAM-rich biobrick, adding a certain promoter whose transcription start site have been acknowledged just behind (Figure 4).                         The versatile promoters are general to every gene, and individuals can design the customized new promoters to meet their specific regulation requirement.
Reference
1. Larson, M. H., Gilbert, L. A., Wang, X., Lim, W. A., Weissman, J. S., & Qi, L. S. (2013). Crispr interference (crispri) for sequence-specific control of gene expression. Nature Protocols, 8(11), 2180-2196.
2. Bikard, D., Jiang, W., Samai, P., Hochschild, A., Zhang, F., & Marraffini, L. A. (2013). Programmable repression and activation of bacterial gene expression using an engineered crispr-cas system. Nucleic Acids Research, 41(15), 7429-7437.
3. Gao, X. J., & Elowitz, M. B. (2016). Synthetic biology: precision timing in a cell. Nature, 538(7626).
SCU_China 2018 iGEM Team
