<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_SCU-China_project_circuits skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><DIV id="top_title"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:SCU-China/project/circuits</SPAN></H1></DIV><DIV id="HQ_page"><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><P><TITLE>Team:SCU-China - 2018</TITLE></P><DIV style="width:100%"><DIV><UL class="navbar" id="navbar"><LI><A href="https://2018.igem.org/Team:SCU-China">Home</A></LI><LI><A>Project</A></LI><LI><A href="https://2018.igem.org/Team:SCU-China/project/overview">Overview</A></LI><LI><A href="https://2018.igem.org/Team:SCU-China/project/regulation">Precise Regulation</A></LI><LI><A href="https://2018.igem.org/Team:SCU-China/project/circuits">Logic Circuits</A></LI><LI><A href="https://2018.igem.org/Team:SCU-China/project/indigo">CRISProgrammer of Indigo</A></LI><LI><A href="https://2018.igem.org/Team:SCU-China/project/parts">Parts</A></LI><LI><A href="https://2018.igem.org/Team:SCU-China/project/interlab">Interlab</A></LI></UL><LI><A>Modeling</A></LI><LI><A href="https://2018.igem.org/Team:SCU-China/modeling/prediction">Mismatch</A></LI><LI><A href="https://2018.igem.org/Team:SCU-China/modeling/algorithm">Orthogonality</A></LI><LI><A>Human practice</A></LI><LI><A href="https://2018.igem.org/Team:SCU-China/practice/collaboration">Collaboration</A></LI><LI><A href="https://2018.igem.org/Team:SCU-China/practice/meet">Meet Up</A></LI><LI><A href="https://2018.igem.org/Team:SCU-China/practice/education">Education &amp; Public</A></LI><LI><A>Judging</A></LI><LI><A href="https://2018.igem.org/Team:SCU-China/judging/human">Integrated Human Practice</A></LI><LI><A href="https://2018.igem.org/Team:SCU-China/judging/improve">Improve</A></LI><LI><A href="https://2018.igem.org/Team:SCU-China/judging/model">Model</A></LI><LI><A href="https://2018.igem.org/Team:SCU-China/judging/demonstrate">Demonstrate</A></LI><LI><A href="https://2018.igem.org/Team:SCU-China/attribution">Attribution</A></LI><LI><A href="https://2018.igem.org/Team:SCU-China/safety">Safety</A></LI><LI><A href="https://2018.igem.org/Team:SCU-China/notebook">Notebook</A></LI></DIV><DIV class="content"><DIV class="main"><DIV class="sidebar" id="sidebar"><UL><LI><A href="#logic">·Logic circuits</A></LI><LI><A href="#single">·Single inhibition circuits</A></LI><LI><A href="#basic">·Basic logic circuits</A></LI><LI><A href="#refer">·Reference</A></LI></UL></DIV><DIV class="maincontent"><DIV class="phrase" id="logic"><DIV class="subtitle">Logic Circuits</DIV><DIV class="text">
                        dCas9 is a Cas9 nuclease mutant that loses nuclease activity but retains DNA-binding activity. It is directed by sgRNA to the spacer sequence in target DNA site. Binding to promoter, dCas9 inhibits transcription initiation because of steric effect on RNA polymerase. Similarly, dCas9 can also interrupt transcription elongation when it is directed to open reading frame. Therefore, dCas9 can be engineered as a programmable transcription repressor and target any specific site in genome and plasmids of E. coli. Based on simple modulation of guide RNA motif, CRISPR-dCas9 system has great potential in efficient control of gene expression pattern.
                        
                        In natural gene-regulation mechanism, transcription factors bind to specific DNA sequences upstream of genes and can either repress or activate their expression. Many transcription factors are exploited to modulate gene expression but all require the engineering of specific promoter sequences that can be bound by these regulators. On the other hand, reprogrammed Zinc Finger DNA-binding proteins and Transcription Activator–Like Effectors can bind to specific sequences and have been used to direct transcription activation and repression in the past decade. However, producing sequence specificity of these proteins is a time-consuming and expensive process.
                        
                        In CRISPR-dCas9 system, sequence specificity is determined by a small RNA molecule, sgRNA. This technology provides a specific, easy-to-use and highly modular programmable method for global regulation of gene expression and shows potential in facilitating the study of gene networks and the development of synthetic biology applications.
            </DIV></DIV><DIV class="phrase" id="single"><DIV class="subtitle">Single inhibition circuits</DIV><DIV class="text">
                        Team SCU-China gained a plasmid containing dCas9 and green fluorescent protein (GFP) gene from the University of Science and Technology of China. In this plasmid, the dCas9 gene is controlled by an arabinose induced promoter.
                        
                        We used the self-contained GFP of dCas9 plasmid as a reporter and searched all NGG PAM sites in both template strand and non-template strand of GFP gene. Under this circumstance, dCas acts as transcription terminator by blocking mRNA elongation. In order to reduce off-target effect to the greatest extent, we BLAST all spacer sequence generated by GFP open reading frame with the genome of E. coli BL21 strain and only remained the sequences whose ‘seed region’(12nt of guide RNA motif 3’ terminal) don’t exist in E. coli genome. Finally, one of these sgRNAs was selected and cloned into part ‘sgRNA generator’ (BBa_K1689000) by Golden Gate Assembly.
                        
                        The engineering of sgRNAs targeting coding sequence is limited as it requires being designed de novo. To achieve modularity and standardization, team SCU-China modified a series of promoters which can be targeted by a specific sgRNA so that the sgRNA/promoter pairs could be used as Biobricks. We installed the spacer sequence and NGG PAM site right in front of the 5’ terminal of a constitutive promoter J23101 (BBa_J23101). In this case, dCas9 works as a transcription repressor because it prevents RNA polymerase binding to promoter region sterically so that blocks transcription initiation.
                        
                        In terms of the sgRNA1/spacer1-J23101 pair, the spacer sequence, spacer1-J23101, is a part of a verified PAM rich sequence. We selected part J23101-GFP (BBa_J364000) as a reporter to test the function of the modified promoter and measure the repression level. The modified promoter does not influence the function of GFP gene as the GFP fluorescence intensity of the modified part (<A href="https://2018.igem.org/Team:SCU-China/project/parts">BBa_K2611001</A>) is even stronger than the original one (Figure 1).
                        <DIV class="showimg"> Figure 1. Fluorescence of spacer-J23101 GFP and spacer-J23101 RFP</DIV>
                        The fluorescent intensity of GFP is measured at exciting light 485nm and emitting light 530nm. The fluorescent intensity of RFP is measured at exciting light 584nm and emitting light 607nm. Spacer sequences of the modified parts come from a reported PAM rich sequence and a specific algorithm respectively. We measured the fluorescent intensity of bacteria solutions after 26h cultivation in LB medium.
                        
                        We combined the improved part, BBa_K2611001, and the corresponding sgRNA1 into one reporter plasmid by enzyme digestion and ligation and co-transformed it together with dCas9 ( Delta-GFP) plasmid into E. coli BL21 strain (Figure 2). Cultivated 20h later, a 40% reduction of fluorescent intensity was observed when dCas9 is induced by arabinose (Figure 3).
                        <DIV class="showimg">Figure 2. Co-transform dCas9plasmid and reporter plasmid into E. coli DH5α</DIV>
                        Lane 1:DL15000 DNA Marker, lane 2-5: co-transformed plasmids, lane 6-9: sgRNA, lane 10-13:dCas9 plasmid, lane 14: sgRNA positive control, lane 15: dCas9 positive control. The theoretical size of the dCas9 plasmid is 10689bp. The theoretical size of the sgRNA plasmid is 2235bp. The extracted plasmid has three forms, and the supercoiled plasmid runs faster so that supercoiled bands are smaller than normal size in DNA electrophoresis gel.
                        <DIV class="showimg">Figure 3. GFP fluorescence of co-transformed E. coli</DIV>
                        The arabinose was added in logarithmic phase. After 12h later of induction, we measured the fluorescent intensity of bacteria solution at exciting light 485nm and emitting light 530nm. The average relative fluorescent intensity of contrast group (without arabinose) and treated group (with arabinose) are 59.44 and 35.95 respectively.
                        
                        Altogether, these results demonstrate that directing dCas9 to different gene regions can prevent both the initiation and elongation of transcription. More importantly, the sgRNA/promoter pairs provide a more modular and programmable tool to use dCas9 as transcription repressor.
                    </DIV></DIV><DIV class="phrase" id="basic"><DIV class="subtitle">Basic logic circuits</DIV><DIV class="text">
                        After we demonstrated that dCas9 works well in single inhibition logic, team SCU-China started to construct some basic logic circuits such as series circuit and parallel circuit.
                        In both basic logic circuits and more complex logic circuits, there are definitely more than one kinds of sgRNA function in a cell simultaneously. To reduce the crosstalk to the maximum extent, we used a specific <A href="">algorithm</A> to generate a set of orthogonal sgRNAs.
                        
                        In series circuit, the reporter plasmid contains two sgRNAs and a fluorescent protein. The first sgRNA, sgRNA1, is controlled by an IPTG inducible pLac promoter. The second sgRNA, sgRNA2, follows a modified T7 promoter which is inserted by spacer sequence of sgRNA1 so that it can be repressed by sgRNA1-dCas9 complex. At last, the promoter for GFP responds to sgRNA2 as it is inserted by spacer sequence of sgRNA2. (Figure 4)
                        <DIV class="showimg">Figure 4. Series circuit</DIV>
                        Without IPTG, GFP is repressed by sgRNA2 so that bacteria only express very weak background fluorescence. When pLac promoter is induced by IPTG, the expression of sgRNA2 is inhibited by sgRNA1. Therefore, GFP will be expressed and accumulated.
                        
                        We have successfully constructed spacer1-T7-sgRNA2-spacer2-J23101-gfp and T7-sgRNA1 in vector pSB1C3 up to now (Figure 5). Next, we would change promoter of T7-sgRNA1 to pLac promoter and combine these two parts into one plasmid.
                        <DIV class="showimg">Figure 5. spacer1-T7-sgRNA2-spacer2-J23101-gfp and T7-sgRNA1 in vector pSB1C3
                            a, lane 1 : Trans 5k DNA Marker, lane 2-4: spacer1-T7-sgRNA2-spacer2-J23101-gfp in vector pSB1C3. The theoretical size of spacer1-T7-sgRNA2-spacer2-J23101-gfp plasmid is 3184bp. b, lane 1: DL5000 DNA Marker, lane 2-3: T7-sgRNA1in vector pSB1C3. The theoretical size of the plasmid is 2252bp. The extracted plasmid has three forms, and the supercoiled plasmid runs faster so that supercoiled bands are smaller than normal size in DNA electrophoresis gel.
                        </DIV>
                        In parallel circuits, the reporter plasmid also contains two sgRNAs and a fluorescent protein. The two sgRNAs are controlled by aTc inducible pTet and IPTG inducible pLac promoter respectively. The promoter for GFP is modified to respond to both sgRNA1 and sgRNA2. In this case, with aTc or IPTG induction, the expression of GFP will be inhibited. GFP will be expressed solely when no inducer is added. (Figure 6)
                        <DIV class="showimg">Figure 6. Parallel circuit</DIV>
                        We have successfully constructed T7-sgRNA1-spacer1&amp;2-J23101-gfp and T7-sgRNA2 in vector pSB1C3 up to now (Figure 7). Next, we would change promoters of T7-sgRNA1-spacer1&amp;2-J23101-gfp and T7-sgRNA2 to pTet and pLac promoter respectively following combine these two parts into one plasmid.
                        <DIV class="showimg">Figure 7. T7-sgRNA1-spacer1&amp;2-J23101-gfp and T7-sgRNA2 in vector pSB1C3
                            a, lane 1: DL5000 DNA Marker, lane 2-4: T7-sgRNA1-spacer1&amp;2-J23101-gfp in vector pSB1C3. The theoretical size of spacer1-T7-sgRNA2-spacer2-J23101-gfp plasmid is 3184bp. b, lane 1: DL5000 DNA Marker, lane 2-3: T7-sgRNA2 in vector pSB1C3. The theoretical size of the plasmid is 2252bp. The extracted plasmid has three forms, and the supercoiled plasmid runs faster so that supercoiled bands are smaller than normal size in DNA electrophoresis gel.
                        </DIV></DIV></DIV><DIV class="phrase" id="refer"><DIV class="subtitle">Reference</DIV><DIV class="text"><DIV class="block">1. Bikard, D., Jiang, W., Samai, P., Hochschild, A., Zhang, F., &amp; Marraffini, L. A. (2013). Programmable repression and activation of bacterial gene expression using an engineered crispr-cas system. Nucleic Acids Research, 41(15), 7429-7437.</DIV><DIV class="block">2. Didovyk, A., Borek, B., Hasty, J., &amp; Tsimring, L. (2016). Orthogonal modular gene repression in escherichia coli using engineered crispr/cas9. Acs Synthetic Biology, 5(1), 81-88.</DIV><DIV class="block">Didovyk, A., Borek, B., Tsimring, L., &amp; Hasty, J. (2016). Transcriptional regulation with crispr-cas9: principles, advances, and applications. Current Opinion in Biotechnology, 40, 177-184.</DIV><DIV class="block">4. Larson, M. H., Gilbert, L. A., Wang, X., Lim, W. A., Weissman, J. S., &amp; Qi, L. S. (2013). Crispr interference (crispri) for sequence-specific control of gene expression. Nature Protocols, 8(11), 2180-2196.</DIV></DIV></DIV></DIV></DIV></DIV></DIV><DIV class="footer">SCU_China 2018 iGEM Team</DIV></DIV></DIV></DIV></DIV></DIV></BODY></HTML>