Team:SCU-China/project/indigo
Team:SCU-China - 2018
Home
Project
Overview
Precise Regulation
Logic Circuits
CRISProgrammer of Indigo
Parts
Interlab
Modeling
Mismatch
Orthogonality
Human practice
Collaboration
Meet Up
Education & Public
Judging
Integrated Human Practice
Improve
Model
Demonstrate
Attribution
Safety
Notebook
· Indigo Synthesis
· Data processing
· Model building and training
· Result
· Reference
Indigo Synthesis
Indigo is a deep and rich color close to the color wheel blue, which was cultivated in East Asia, Egypt,                         India, and Peru in antiquity. The earliest direct evidence for the use of indigo dates to around 4000 B.C.                         and comes from Huaca Prieta, in contemporary Peru.1 It is widely used in the food, pharmaceutical, printing                         and dyeing industries. However, at present, indigo industrial production often uses a large number of chemical                         reducing agents, such as sodium dithionite. Sodium dithionite decomposes to form sulfates and sulfites, which                         can corrode equipment in dyeing plants and wastewater treatment facilities. Many dyeing plants avoid the extra                         cost of wastewater treatment by dumping waste dyes into rivers, as rivers have a negative impact on ecology 2.
In order to improve this situation, we have explored the Tammy M Hsu’s and 2013 IGEM_Berkeley’s study1 and produced                         a system to express indigo in E. coli with a protecting group strategy (Figure 1).
Results
FMO
Referring to the indigo synthesis strategy of Hack Sun Choi3, we use FMO to synthesize indoxyl. FMO can efficiently oxidize indole to indophenol, and then Indophenol can be oxidized by oxygen in air or liquid to Leucoindigol, which will be spontaneously transformed to indigo (Figure 2).
For better purification and identification, we inserted the coding sequence of each enzyme into pET21αvector containing an N-terminal 6x His tag. Results of colony PCR and sequencing demonstrated that we successfully constructed these plasmids (Figure.3).
Recombinant bacteria E. The coli BL21 was streaked in an LB solid medium containing ampicillin, and cultured at 37 ° C for 12 hours, and single colonies were picked and inoculated into 10 mL of LB liquid medium at 37 ° C overnight. Take 100 uL of bacterial solution into 5 ml of LB liquid medium. After incubating at 37 ° C for 3 h, 5 mM IPTG was added, the temperature was adjusted to 37 ° C, and shaking was incubated at 300 rpm for 3 h, and then an appropriate amount of hydrazine solution (ethanol preparation) was added. The bacterial solution was shaken at 37 ° C, 300 rpm for 4 to 24 hours. We successfully got our indigo produced by E.coli. (Figure.4) Besides observing the extracted indigo by eye, we performed an absorbance measurement in the UV-Vis spectra under 640nm and made its standard curve (Figure5).
We detected flavin-containing monooxygenases (FMO) with a 6xHis-tag by SDS-page (Figure 6) and Western-blot (Figure 7). And all the results showed that FMO has been expressed successfully under the regulation of promoter pT7 in BL21. The protein of FMO-His tag is about 54kDa.
To have a higher expression level of indigo, we optimized the fermentation conditions. The concentration of Indole, IPTG and time were investigated by single factor experiment. (Figure 8&Figure9&Figure 10)
When the concentration of lanthanum in the medium reached 150 mg/L, the amount of indigo synthesis and conversion were the highest. Since ruthenium inhibits both host cell and enzyme catalysis, further increases in substrate concentration products are reduced.
Experiments have shown that IPTG can induce the expression of flavin monooxygenase. Considering the production needs, we studied the effect of lactose concentration on the synthesis of indigo. The results are shown in Figure 7 (the concentration of strontium is 150mg/L, the same below). When the final concentration of IPTG was 0.5 mM, the concentration of indigo in the conversion solution was the highest.
The results of the relationship between transformation time and indigo are shown in Figure 10. After 3 h, the concentration of indigo in the medium tends to be the maximum stable value.
UGT
Indoxyl can be spontaneously oxidized to indigo. But indigo cannot be used in dying for that indigo cannot be embedded in the voids of fabric fibers. In industry, indoxyl products must be reduced by a reducing agent before dyeing, then it can be used. To prevent the step which will make a lot of pollution, we want to prevent indoxyl from oxidating into indigo.
To achieve the goal, we need to produce UDP-glucuronosyltransferaseii (UGT) in our E.coli which can protect indoxyl by changing it into indicant(Figure 11).
We insert the coding sequence of UGT into the pET21a plasmid by enzymes digest and ligation and it is induced by IPTG. Then we successfully detect a 43 kDa protein with his-tag by western blot after induced by IPTG and we believe that this is UGT (Figure 7).
The molecular weight of UGT is expected to be about 43 kDa. Our Western Blot result shows that we had successfully expressing the UGT in E.coli which means it has the potential to be used into the industry.
BGL
In order to take advantage of the protected Indican, we should deprotect. In this way, indoxyl can get access into fiber, after oxidation, can dye the clothes. Even the enzyme BGL can work to deprotect, we chose a better enzyme BGL-A as our deprotection enzyme according to Hsu, T. M et al., BGL-A is the enzyme function in the midstream of the environment-friendly indigo synthesis. Gene we used this year is the originally found in Bacillus circulans. It is a beta-glucosidase which would de-protected the indican and then release the final product, indigo. Similar enzymes can also be detected in E. coli strains which may cause the advanced hydrolysis of indican to a certain extent. After synthesis the gene by GENERAL BIOSYSTEMS, we first constructed the BGL-A in pET21-a and then transformed DH5 alpha with the plasmid (oligos and detailed plasmid sequence can be found in supplementary material). Then the E. coli BL21 was used as our expression system. However, after several attempts to optimizing the expression condition, we didn’t get a complete protein. Our instructor Hongbin Yu noticed that codons were not well-optimized for E. coli BL21, thus a relatively short product may be synthesized instead of the real product. New BGL-A sequence is still under experiments and please make the presentation as the standard. Even if we can’t get a BGL-A expression, we can also try to modulate the endogenous BGL-A by our CRISProgrammer system to realize the indigo synthesis.
Design of Regulation in Practice
Here, we report the Indi.coli system to achieve the on/ off (storage/ dyeing) switch of indoxyl bio-synthesis pathway. The combination of indoxyl synthesis switch and the customized CRISProgrammer, is a great evidence that our CRISProgrammer system can regulate the metabolism pathway in an engineerable, customizable, and adjustable manner. (Figure 1).
Figure 1. The logic circuit of Indi.coli.
P0 represents for an inducible promoter, while p1-p4 represent for constitutive promoters that can be targeted by sgRNA orthogonally. One constitutive promoter can be suppressed by related sgRNA in same colors. The function of three enzymes are shown below each of them, respectively. Indoxyl in blue dashed outline can be oxidized into indigo naturally.
At the start point, indole is added in the system as substrate, while no inducer is added. The enzymes FMO (flavin-containing monooxygenases) is constitutively expressed, catalyzing the reaction from indole into indican. Since the expression of BGL (β-glucosidase) is inhibited by sgRNA4, indican could not be deglycosylation and would be accumulated. This is the transport and storage stage of indi.coli (Figure 2a).
When inducer is added to the system, sgRNA1 controlled by inducible promoter p0 is highly expressed. The expression of the sgRNA2, sgRNA4, and UGT are inhibited, while BGL is produced. In this situation, the indican accumulated from last stage would be deprotected into indoxyl, which can be easily oxidized into indigo. In the dyeing stage, Indoxyl is embedded in the voids of fabric fibers and reoxidized to form indigo, which makes the fabric dyed successfully. (Figure 2b).
Figure 2. The mechanism of indoxyl production in indi.coli.                         a)The transport and storage stage. b)The indoxyl production and dyeing stage.
By achieving the indi.coli system, the feasibility of combining the regulator parts (sgRNA logic circuits) with the metabolism pathway is tentatively proved. Building customized sgRNA regulatory logic circuits as input, then we could obtain customized outputs under specific signals, our CRISProgrammer is demonstrated to have great potential to be a powerful platform to design tunable, complex logic circuits.
CRISProgrammer in Indigo
In fermentation engineering, the most optimal and pragmatic strategy always contains the least times to adding additional raw materials into the fermentation cylinder. It requires that artificial manipulation should be as less as possible (Dr. Fei Huang mentioned the point). However, the time-space specific expression of genes during fermentation process tangle the transfer of intermediate products into different system. In this year project, SCU-China iGEM team put CRISProgrammer into practice by focusing on the spatio-temporality production of indigo.
According to the strategy, SCU_China iGEM team designed a genetic circuit to mimic the regulation process, in order to get rid of the complicated operation to transfer intermediate products into another fermentation system.
According to the design of regulation, our team deliberately substitute the key enzymes with reporter genes. It looks like a ‘bistable switch’, having two kinds of output with correspondent input. Our bistable switch consists of two main modules, the regulatory module and the reporter module (Figure 1).
In bistable switch system, the sgRNA (J23) and the reporter gene, mRFP1, are both controlled by T7 promoter. The sgRNA (T7) is under the control of pTet inducible promoter, while GFP is controlled by a constitutive promoter, J23100. The whole system is designed to function in BL21 E. coli strain, which can synthesis T7 RNA polymerase.
Since J23100 is a constitutive promoter (Figure 2), the start point of the whole module is supposed to have certain GFP expressed. When the plasmid is transformed into BL21 strain, the T7 promoter is activated. Then the expression level of mRFP1 is raised. Meanwhile, the expression of the sgRNA(J23) ‘closes’ the activating-form of J23100 promoter and the expression of GFP is inhibited. In the normal stage, the system exhibits high mRFP1 level and low GFP level.
When inducer (tetracycline) is added, pTet promoter is activated. The sgRNA (T7) is highly expressed, inhibiting the expression of mRFP1 and sgRNA(J23). The inhibition stage of GFP is ‘withdrawn’. At this stage, the system shows high GFP level and low mPFP1 level.
The bistable switch is designed as a minimal circuit to demonstrate the feasibility to build a whole CRISProgrammer system, which enables orthogonal regulation and demonstrate the regulation process of indigo syntesis.
Results
Reporter
The Reporter Module contains two fluorescence proteins as its reporters (Figure 3). The construction of Reporter module involves two steps. First, rrnB T1 terminator and T7te terminator (BBa_B0010, BBa_B0012) were added behind the mRFP1 coding sequence. Our team is afraid that without the terminator in the transcriptional unit of mRFP1, the elongation will process, and the GFP will expression simultaneously. All the basic parts come from the previous exist plasmids in the
registry (BBa_K199118). Then, 3A assembly method was used to build the functional reporter by inserting T7-RFP fragment into J23100-GFP-pSB1C3 backbone. The experiment details are shown in the notebook .
We have successfully constructed the reporter plasmid and verified that it could function well (T7 promoter-RFP-J23100-GFP):
T7 promoter-RFP-J23100-GFP-pSB1C3 has been successfully constructed. The normal function of the GFP and mRFP1 have been verified.
dCas9 Expression Validation
We obtained the plasmid p-GFP-dCas9-VVD from Haiyan Liu's lab, University of Science and technology of China, and are to modify the plasmid so it could be of our use. First, we have to remove the trailing VVD sequence in frame with the dCas9 ORF and add a 6x His-Tag to dCas9 for downstream detection; second, we choose to delete the GFP coding sequence for we want a solely dCas9-expressing plasmid. What’s more, we decided to keep the backbone for it carried rop sequence, which makes it low-copy and therefore thought to be exerting less effect on the E. coli viability. After adding 6x His-Tag, we would transform the plasmid into DH5α and do Western Blot to verify the expression of dCas9.
In the cloning process, traditional digest-ligate workflow and homology recombination techniques are used. We are working on constructing the vector recently. Identifying the fusion dCas9 protein could be successfully expressed and verifying its biological function will be our next step.
The origin pdCas9 offer by Haiyan Liu's lab at University of Science and technology of China was called as p-GFP-dCas9-VVD in the following. Sequencing Independent Ligation Coloning (SILC) was used for adding 6×His tag and deleting GFP and VVD coding sequence.
Primer A is design to PCR a 4kb fragment which contains TcR and rop sequence. Primer B is supposed to generate a 6kb fragment which contains ori-ara/ pBAD promoter-dCas9 coding sequence (from 5’ terminal to 3’ terminal), meanwhile, 6x His-Tag is also added to the N-terminal of dCas9 coding sequence. (Primer A and Primer B are both represent one pair of primers, which contain homologous sequence for recombination for 15-20bp.)
The electrophoretogram of ligation result is shown as follows.
The Functional Test
The basic circuit T7-sgRNA (J23100)-J23100-GFP-pSB1C3 is a minimal circuit built to verify whether our dCas9/ sgRNA system can function well. sgRNA (J23100) is short for the small guide RNA which are designed to target the J23100 promoter specifically. he basic circuit and the vector of dCas9 are in two backbones. To confirm our dCas9/ sgRNA regulatory system is feasible, we choose to co-transform these two plasmids into the E. coli BL21 strain at the same time.
In this circuit, sgRNA (J23100) is controlled by a T7 promoter without lac operon (which is equal to a constitutive promoter) and GFP is controlled by J23100 promoter, therefore their expression is both sustainable. In the dCas9 circuit, the expression of dCas9 is under the control of ara/ pBAD promoter, so the arabinose is a trigger to the inhibition effect of GFP expression in whole system. If the expression of GFP is significantly suppressed after inducer added, it is fair enough to demonstrate that our dCas9 protein and sgRNA present their biological function successfully.
We tried to use electro-transformation to co-transform these two plasmids (vector of dCas9 without GFP, the basic circuit) for several times, but the efficiency was quite low. Therefore, we are working on assemble the two functional devices into one plasmid in the following days.
We designed and synthesized the 20bp variable sequence of gRNA (J23100), which is critical for its specific targeting. Next, we use primer annealing to insert this sequence to the sgRNA generator plasmid (BBa_K1689000), which was submitted by 2015 Peking iGEM team, to successfully built the T7-gRNA (J23100)-pSB1C3 vector. Then, we chose 3A assembly method to build the basic circuit by inserting J23100-GFP fragment (BBa_J364007) into T7-gRNA (J23100)-pSB1C3 backbone.
The successful ligation of T7-gRNA (J23)-pSB1C3 backbone and J23100-GFP inserts using 3A assembly is shown as follows.
Bacteria Genome Editing——Knock in
In order to minimize the plasmid applied in CRISProgrammer, we are supposed to design a modularization workflow by applying CRISPR/Cas9 system to write (knock in) and erase (knock out) parts in genome.
We tried to apply Red/ET4 system, which comes from Lambda bacteriophage, and RecA protein 5,6, which could improve the stability of single strand DNA, to assist the
HR process. Besides, two plasmids system or two step method will work at low efficiency and is time-consuming, so we try to combine all parts into one big plasmid which could shorten the period of workflow. There should be no plasmid or antibiotic gene remaining in bacteria after genome editing, therefore, we used pSC101 temperature sensitive origin of replication7, the plasmid containing which would replicate at a low efficiency under 37℃ and finally loses after 1 day’s incubation. Considering the cytotoxicity of Cas9 protein, and the controllability of knock in system, we used arabinose (Ara) promoter to control the expression of Cas9 protein and recombination related proteins, because the leaky expression of Cas9 protein could not only slow down the grow of bacteria, but also decrease the efficiency of transformation. Providing glucose only should inhibit gene transcription. While providing arabinose only, the downstream expression system could be active.
In our design, we are going to make a modulization workflow so that it could be applied to any targeted gene.
In order to test the practicality of knocking-in, we used mRFP1 as a reporter to build a consecutive workflow. In light of the limited time, we used the plasmid original targeted site as our knock in site located in poxB8 which encodes the Pyruvate Oxidase of Escherichia coli, it9 has been verified that the loss of this gene won’t influence the vitality of E.coli in normal culture condition. In addition, we use Modified Gibson Assembly (see notebook) to adjoin the long plasmid backbone with the small insertion part by short homologous arms in vitro.
Besides we chose universal primers between BioBrick prefix and suffix so that we can easily clone the prats from any registry for knock-in. Therefore, we prepared the part we are going to knock-in including GFP+RFP with PAM rich targeted promoters and GFP with PAM targeted promoters. As for dCas9 protein, we make 2 expression cassettes, one contains the only dcas9 protein, the other contains short homologous arms between dCas9 protein and GFP with PAM rich targeted promoters
Results
In knock in part, we used Modified Gibson Assembly to construct the donor DNA and backbone which is prepared by high speed PCR enzymes (Primer STAR○R GXL). To get a good electroporation transformation efficiency, we purify the reaction product by column (E.Z.N.A.® Plasmid Mini Kit I). After SOC culture at 30℃ for 1h, we plated the electro transformation production on plate with 1%(w/v) glucose, and 50ng/ul kanamycin. Finally, we selected several colonies for screening and re-plated them.
After successfully plasmid construction, we selected new single colony for amplification and cultured colony with solid L-Ara (1%) media overnight and selected colony randomly for colony PCR which we get 2 in 14 has knock-in feature.
Then we re-plated the colony on plate. Interestingly not all the colony turn red at the same time.
In order to test the elimination of plasmid, we cultured the E.coli on 37℃ for 1 day, and then plated it on plasmid on K+ plate compared by those without elimination. Those still have plasmid (right) survived better than the left. And the plasmid purification from 6 ml colony culture shows no plasmid remaining.
4.2 Donor part preparing
At the same time, we design the universal primers so that we can use these primers to amplify any sequence on between prefix and suffix. Thus, we used these primers to amplified the GFP, RFP+GFP, dCas9 and dCas9 with GFP for future cloning.
In this part, we successfully verified the knock in function of our plasmid, and test the Golden Gate assembly plasmid construction. Besides here are some drawbacks of this methods.1 It’s hard to transformation efficiency, for with only the colony PCR it could be a low efficiency for colony selection when construction plasmid. 2 The periodic expression of genome may result in the desynchrony expression of RFP on genome. 3 the exogenous gene in Bacteria genome may result in long fragment homologous recombination within genome range. We only get positive result for the first few days. After several days culture after losing its plasmid, the same site colony will smear or abnormal length fragment will rise (see notebook) which may suggest the HR in genome range.
Reference
1. https://en.wikipedia.org/wiki/Indigo#In_culture
2. Hsu, T. M., Welner, D. H., Russ, Z. N., Cervantes, B., Prathuri, R. L., & Adams, P. D., et al. (2018). Employing a biochemical protecting group for a sustainable indigo dyeing strategy. Nature Chemical Biology, 14(3).
3. Choi, H. S., Kim, J. K., Cho, E. H., Kim, Y. C., Kim, J. I., & Kim, S. W. (2003). A novel flavin-containing monooxygenase from methylophaga sp strain sk1 and its indigo synthesis in escherichia coli. Biochemical & Biophysical Research Communications, 306(4), 930-936.
4. Tu, Q., Herrmann, J., Hu, S., Raju, R., Bian, X., & Zhang, Y., et al. (2016). Genetic engineering and heterologous expression of the disorazol biosynthetic gene cluster via red/et recombineering. Sci Rep, 6, 21066.
5. Benzinger R, Enquist LW, Skalka A. Transfection of Escherichia coli spheroplasts. V. Activity of recBC nuclease in rec+ and rec minus spheroplasts measured with different forms of bacteriophage DNA. J Virol. 1975;15(4):861–71.
6. Wang J, Sarov M, Rientjes J, Hu J, Hollak H, Kranz H, Xie Y, Stewart AF, Zhang Y. An improved recombineering approach by adding RecA to λ red recombination. Mol Biotechnol. 2006;32(1):43–53.
7. Zhao, D., Yuan, S., Xiong, B., Sun, H., Ye, L., & Li, J., et al. (2016). Development of a fast and easy method for escherichia coli genome editing with crispr/cas9. Microbial Cell Factories, 15(1), 205.
8. Grabau, C., & Jr, J. E. C. (1984). Molecular cloning of the gene (poxb) encoding the pyruvate oxidase of escherichia coli, a lipid-activated enzyme. Journal of Bacteriology, 160(3), 1088.
9. Chang, Y. Y., & Jr, J. E. C. (1983). Genetic and biochemical analyses of escherichia coli strains having a mutation in the structural gene (poxb) for pyruvate oxidase. Journal of Bacteriology, 154(2), 756-762.
SCU_China 2018 iGEM Team
