Team:SCU-China/InterLab
Team:SCU-China - 2018
Home
Project
Overview
Precise Regulation
Logic Circuits
CRISProgrammer of Indigo
Parts
Interlab
Modeling
Mismatch
Orthogonality
Human practice
Integrated Human Practice
Collaboration
Meet Up
Education & Public
Judging
Integrated Human Practice
Improve
Model
Demonstrate
Attribution
Safety
Notebook
Introduction
Materials and Equipment
Details of Methods
Acknowledgement and Collaboration
Introduction
We have participated in 2018 InterLab work and uploaded the data and questionnaires. In this page, we
listed the materials and equipment we used in the InterLab Study;
gave an introduction to the details of our experiment procedure, for the official protocol was a little brief;
showed our experiment results and relative discussion;
and made a sincere acknowledgement to whom have helped us.
Materials and Equipment
Table 1 and table 2 show the detail information of the materials and equipment we used this year respectively.
Table 1. Details of experimental material
Table 2. Details of experimental equipment
Details of Methods
Our interlab study was successfully done by following the official protocol provided on 2018 iGEM interlab website, including
Transformation Protocol
Plate Reader Protocol
Details of Transformation Experiment
ddH2O was used to dissolve the plasmid in kit. After the certain DNA was dissolved, both of the 2018 iGEM kit and the dissolved plasmid were kept in -20 degrees centigrade fridge.
When we prepared transformation experiment, trans5-alpha competent cells and the dissolved plasmid were taken out and defrosted on the ice. The step of mixing the competent cells with DNA was done outside the biosafety cabinet, while adding the liquid LB to help the bacterial recovery was done inside it.
Solid LB contained 2% agar, and chloramphenicol was added before making plates. Parafilm was used to prevent the fast evaporation of H2O during plates storing.
Details of Preservation and Verification
After the successful transformation, single colonies for each device was picked, and the plates were kept in 4 degrees centigrade fridge. Bacterial glycerol stocks were prepared with the final glycerol concentration being 25%.
All the single colonies we used for further studies were sequenced correctly. Our bacterial culture was sent to Chengdu Tsingke Biological Technology Co., Ltd, and the universal primer VF2 and VR were choosed for sequencing. Sequencing results were aligned with sequences provided with iGEM Registry and all of them were right.
Details of Measurement Study
When we did the measurement. The cabinet was shined by ultraviolet before start for 30 minutes. One person (name C for convenience) took 500μl from each culture tube to a EP tube, then gave it to one person out of the cabinet (name B) to test the original OD600. Then, C prepared enough 50ml tubes with 12ml liquid LB in it (chloramphenicol preadded), and B calculated the dilution way according to the excel provided on official website. When the calculation was done, C piped out certain volume of LB and added bacterial culture, then shook to mix and took 500μl as the sample of 0 hour. After the first sampling, we took out the culture from incubator (220rpm, 37℃) every 2 hours, with the first sampling time denoting 0 hour.
Plates with black frame and transparent, flat bottom were used to do the measurement. Since we did the experiment many times resulting from some accidents, we recycled the plates. Plates were cleaned by distilled water and 75% ethanol. Plate reader and relative software was manipulated by a teacher due to the requirement of the lab.
Result
A) Eight plasmids provided in 2018 kit plate 7(Positive Control, Negative Control, Test Device 1, Test Device 2,                             Test Device 3, Test Device 4, Test Device 5, and Test Device 6) were successfully transformed into E.coli Trans5-alpha                             strains. Name of plasmids and corresponded parts name are shown in Table 3. Short for name will be used to refer                             instead of parts name for convenience.
Table 3. Name of plasmids and corresponded parts name
Here we show our result in table 4, representing for OD600 reference point; figure 1-3, representing                             for the particle standard curve, fluorescent standard curve and fluorescence per OD and fluorescence                             per particle graph respectively. Data were calculated in the excel provided by iGEM official website.                             Blank control had been considered before plotting.
B) Calibration Experiments
Calibration1: OD 600 Reference Point
Table 4. Calibration2, OD600 reference point
Calibration2: Particle Standard Curve
Figure 1. Calibration2, Particle Standard Curve and Its Log Scale
Left panel: Particle Standard Curve
Right panel: Fluorescein Standard Curve (log scale)
Calibration3: Fluorescence Standard Curve
Dilution serious of fluorescein were prepared and measured in a black frame transparent bottom 96 well plate. A standard curve is generated to correct the cell based readings to an equivalent fluorescein concentration.
Figure2. Calibration3, Fluorescence Standard Curve and Its Log Scale.
Left panel: Particle Standard Curve. Fluorescein Standard Curve under ex485/em530.
Right panel: Fluorescein Standard Curve (log scale)
C) Measurement Experiments
a) Cell Measurement Experiments
Figure3. Fluorescence per OD and Fluorescence per Particle Graph                                 NC and PC means the negative control and positive control respectively. Biological replicates                                 were considered together and error bar shows the standard error of all the eight replicates.
According to these figures, we can conclude that in the aspect of promoter strength, J23101 is                             greater than J23106 and J23106 is greater than J23117, what’s more, J23100 is greater than J23104 and J23104 is greater than J23116.
We can also see that the growth of D5 was relatively influenced, suggesting J23104 may have a                             negative effect on bacterial growth, though the protein expression is relatively high.
b) Colony Forming Units per 0.1 OD600 E. coli cultures The series dilution instructions:
Figure4.                                 The series dilution instructions
CFU/mL/OD Calculation Instructions:                             Based on the assumption that 1 bacterial cell gives rise to 1 colony, colony forming units (CFU)                             per 1mL of an OD600 = 0.1 culture can be calculated as follows:
Count the colonies on each plate with fewer than 300 colonies.
Multiple the colony count by the Final Dilution Factor on each plate.
Example using Dilution 4 from above
# colonies          x         Final Dilution Factor           = CFU/mL
125             x          (8 x 105)               = 1 x 108 CFU/mL
in Starting Sample (OD600 = 0.1)
The CFU/mL/OD Calculation is shown as follows:
Table 5. The CFU/mL/OD Calculation
Acknowledgement and Collaboration
We sincerely gratitude the work done by our team members, Fuqiang Yang, Minyue Bao, Tianyi Zhang, Xiaolong Liu, Yanling Zhong, Yibing Tao, and Ziru Wang (names are listed in no particular order) in InterLab study. We also appreciated that the microplate reader (ThermoScientific Varioskan Flash) provided by Dr. Kang from the Center of Growth, Metabolism and Aging in Sichuan University, using which we got all of our plate reader experiment data.
We helped Shishi High School Team this year by sharing them some of                         our plasmids from 2018 iGEM kit for interlab study. And their feedback showed that the                         transformation was successful.
SCU_China 2018 iGEM Team
