<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_SCU-China_project_overview skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><DIV id="top_title"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:SCU-China/project/overview</SPAN></H1></DIV><DIV id="HQ_page"><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><P><TITLE>Team:SCU-China - 2018</TITLE></P><DIV style="width:100%"><DIV><UL class="navbar" id="navbar"><LI><A href="https://2018.igem.org/Team:SCU-China">Home</A></LI><LI><A>Project</A></LI><LI><A href="https://2018.igem.org/Team:SCU-China/project/overview">Overview</A></LI><LI><A href="https://2018.igem.org/Team:SCU-China/project/regulation">Precise Regulation</A></LI><LI><A href="https://2018.igem.org/Team:SCU-China/project/circuits">Logic Circuits</A></LI><LI><A href="https://2018.igem.org/Team:SCU-China/project/indigo">CRISProgrammer of Indigo</A></LI><LI><A href="https://2018.igem.org/Team:SCU-China/project/parts">Parts</A></LI><LI><A href="https://2018.igem.org/Team:SCU-China/project/interlab">Interlab</A></LI></UL><LI><A>Modeling</A></LI><LI><A href="https://2018.igem.org/Team:SCU-China/modeling/prediction">Mistatch</A></LI><LI><A href="https://2018.igem.org/Team:SCU-China/modeling/algorithm">Orthogonality</A></LI><LI><A>Human practice</A></LI><LI><A href="https://2018.igem.org/Team:SCU-China/practice/collaboration">Collaboration</A></LI><LI><A href="https://2018.igem.org/Team:SCU-China/practice/meet">Meet Up</A></LI><LI><A href="https://2018.igem.org/Team:SCU-China/practice/education">Education &amp; Public</A></LI><LI><A>Judging</A></LI><LI><A href="https://2018.igem.org/Team:SCU-China/judging/human">Integrated Human Practice</A></LI><LI><A href="https://2018.igem.org/Team:SCU-China/judging/improve">Improve</A></LI><LI><A href="https://2018.igem.org/Team:SCU-China/judging/model">Model</A></LI><LI><A href="https://2018.igem.org/Team:SCU-China/judging/demonstrate">Demonstrate</A></LI><LI><A href="https://2018.igem.org/Team:SCU-China/attribution">Attribution</A></LI><LI><A href="https://2018.igem.org/Team:SCU-China/safety">Safety</A></LI><LI><A href="https://2018.igem.org/Team:SCU-China/notebook">Notebook</A></LI></DIV><DIV class="content"><DIV class="main"><DIV class="sidebar" id="sidebar"><UL><LI><A href="#part1">·Computer Programming</A></LI><LI><A href="#part2">·CRISPR-dCas9</A></LI><LI><A href="#part3">·CRISPrpgrammer</A></LI></UL></DIV><DIV class="maincontent"><DIV class="text">Synthetic biology is now developing into its zenith time, and the concepts of BioBricks and standardized assembly methods have proven to be efficient, universally compatible and highly extensible. This year, Team SCU_China plans to bring the standardization in synthetic biology to a completely new level.</DIV><DIV class="phrase" id="part1"><DIV class="subtitle">Call-and-return in Computer Programming</DIV><DIV class="text">In modern computer programming, a complete program is decomposed into different modules with relatively
                    independent scopes, and within each module the operations are further grouped into different functions
                    or subroutines. During the execution of a program, different functions in different modules are
                    “called” (or used) with some parameters given; after the “callee” (the function that has been
                    called) completed its operations and/or calculations, the results are returned to its “caller”
                    (the function that calls). Such hierarchical calls for and returns from functions enables the computer
                    engineers not only to divide and conquer large and complex programs, but also to allow the common and
                    already written modules and functions, such as the functions for adding, subtracting, file reading/writing etc.,
                    to be reused directly instead of coding for them again. The principle is intuitively called “Do not re-invent
                    the wheel”. There is already a plethora of handy “libraries”, which are the collections of commonly
                    used functions, could be directly used and thus facilitates fast program development.（Figure 1）
                    </DIV><DIV style="margin:0 auto;width:430px" class="showimg">Figure 1. This figure depicts how the functions are called and return during the execution of a program. Functions can be called hierarchically, and one function could be called more than once (not shown).</DIV></DIV><DIV class="phrase" id="part2"><DIV class="subtitle">CRISPR-dCas9</DIV><DIV class="text">
                    CRISPR technique, known as the bacterial immune system against invading viruses, has become an indispensable tool in biological research. Mainly, there are three different types of the CRISPR systems, including type I, II, III. Types I and III loci contain multiple Cas proteins forming complexes with crRNA to recognize and destroy target nucleic acids. In contrast, type II usually has reduced number of Cas proteins. And the CRISPR Cas9 system belonging to type II, after being optimized and artificially designed, is being widely used in gene editing, epigenetic regulation, chromatin engineering, and imaging. CRISPR Cas9 technique consists of sgRNAs, Cas9 nucleases, and perhaps of repair templates. The sgRNA has a scaffold motif and a 20-nt guide RNA motif which binds to Cas9 and guides this nuclease to the spacer sequence in target DNA respectively. Protospacer-adjacent motifs (PAMs) is an essential element in the non-target DNA strand for target recognition and cleavage. The location of PAM varies from different types of CRISPR Cas systems. And the composition of PAM can also vary widely according to the Cas9 deriving from what bacteria species. The CRISPR Cas9 (type II) used in our project is from Streptococcus pyogenes of which the PAM is located in the 3’ end of the protospacer and its sequence is usually NGG. Wild Cas9 nucleases are activated once binding to both sgRNA and PAM and produce double strand break (DSB) of target DNA.
                    
                    Besides, the mutation of Cas9 nuclease enables it to have diverse function. One of these mutated Cas9 that has been widely used in gene regulation is ‘dead’ Cas9 (dCas9). Cas9 nucleases carry out strand-specific cleavage by using the conserved HNH and RuvC nuclease domains. Both of the DNA cleaving catalytic residues (D10A in the RuvC domain and H840A in the HNH domain) mutation abolishes dCas9 cleavage activity but does not impair DNA binding activity. When dCas9 binds to genes regardless of promoter and transcriptional regions, it becomes the barrier preventing RNA polymerase from transcription initiation and elongation (Figure 2). Taking advantage of this characteristic, CRISPR dCas9 technique is used to regulate transcriptional rates thereby gene expression rates. And in some complex systems, CRISPR dCas9 technique is used as switches to construct orthogonal systems to fulfill the spatiotemporal regulation of a series of genes.
                    </DIV><DIV class="showimg">Figure 2. dCas9-mediated repression in E. coli. The sgRNA directs dCas9 to promoter or open reading frame regions to prevent RNA polymerase binding or elongation, respectively</DIV></DIV><DIV class="phrase" id="part3"><DIV class="subtitle">CRISPrpgrammer</DIV><DIV class="text">
                    Inspired by the modularization and “Do not re-invent the wheel” philosophy in computer programming, this year SCU_China iGEM team came up with the idea of using the dCas9 protein to manipulate the expression of proteins and implement complex logic in a single E. coli cell to achieve the similar “call-and-return” controlling paradigm.
                    
                    Moreover, we want to construct versatile “library strains” that contains many, and maybe redundant coding sequences of commonly used proteins (like enzymes for industrial production etc.) which are dormant by default; such sequences could be on a large plasmid transformed in advance or integrated into the genome. When we need to use the function of particular proteins whose coding sequences are already exist in the cell (to “call” them), we just simply transform a much smaller (compared to the CDS) “Minimid” containing expression cassettes of sgDNAs targeting the desired proteins into the library strain, and then the dCas9-sgRNA complex would initiate/inhibit the transcription and then expression of the proteins of interest.
                    
                    This way, we could not only simplify the construction of plasmids (for it is small and could even be directly synthesized), but also provide a rapid way of constructing different strains for production and a possible method to avoid genetic pollution. What’s more, utilizing dCas9 system also enables us to implement complex logics in one cell. Our prospect is that using the system, finally we could really literally “program” the cell, i.e. using computer language to code the logic, and then use a “genetic compiler” to convert the logic to nucleotide sequences (for “Minimid”) containing a series of well-designed sgDNAs.
                    </DIV></DIV></DIV></DIV></DIV></DIV><DIV class="footer">SCU_China 2018 iGEM Team</DIV></DIV></DIV></DIV></DIV></DIV></BODY></HTML>