<HTML xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en" dir="ltr">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki  ltr ns-0 ns-subject page-Team_Virginia_Commonwealth_Internal_Papers"><DIV id="globalWrapper"><DIV id="top-section"><DIV id="p-logo"><A href="/Main_Page" title="Main Page">&quot;
	    </A></DIV><DIV id="menubar" class="left-menu"><UL><LI class="selected"><A href="/Team:Virginia_Commonwealth/Internal/Papers">Page               </A></LI><LI class="new"><A href="/wiki/index.php?title=Talk:Team:Virginia_Commonwealth/Internal/Papers&amp;action=edit&amp;redlink=1">Discussion               </A></LI><LI><A href="/wiki/index.php?title=Team:Virginia_Commonwealth/Internal/Papers&amp;action=edit">View source               </A></LI><LI><A href="/wiki/index.php?title=Team:Virginia_Commonwealth/Internal/Papers&amp;action=history">History               </A></LI><LI style="color:#808080;cursor:default">teams</LI></UL></DIV><DIV class="right-menu" id="menubar"><UL><LI id="pt-login"><A href="/wiki/index.php?title=Special:UserLogin&amp;returnto=Team:Virginia_Commonwealth/Internal/Papers" title="You are encouraged to log in; however, it is not mandatory [o]" accesskey="o">Log in</A></LI></UL></DIV><DIV id="search-controls"><FORM action="/Special:Search" id="searchform"> </FORM></DIV></DIV><DIV id="content"><H1 class="firstHeading">Team:Virginia Commonwealth/Internal/Papers</H1><DIV id="bodyContent"><H3 id="siteSub">From 2009.igem.org</H3><DIV id="header"><TITLE>Virginia Commonwealth iGEM 2009</TITLE><DIV id="colortab" class="ddcolortabs"><UL><LI><A href="https://2009.igem.org/Team:Virginia_Commonwealth" title="Home"><SPAN>Home</SPAN></A></LI><LI><A href="https://2009.igem.org/Team:Virginia_Commonwealth/Overview" title="Overview"><SPAN>Overview</SPAN></A></LI><LI><A href="https://2009.igem.org/Team:Virginia_Commonwealth/Design" title="Part design"><SPAN>Design</SPAN></A></LI><LI><A href="https://2009.igem.org/Team:Virginia_Commonwealth/Methods" title="Characterization methods" rel="dropmenu2_a"><SPAN>Methods</SPAN></A></LI><LI><A href="https://2009.igem.org/Team:Virginia_Commonwealth/Results" title="Results" rel="dropmenu1_a"><SPAN>Results</SPAN></A></LI><LI><A href="https://2009.igem.org/Team:Virginia_Commonwealth/BioBricks" title="Our submitted BioBrick parts" rel="dropmenu1_a"><SPAN>BioBricks</SPAN></A></LI><LI><A href="https://2009.igem.org/Team:Virginia_Commonwealth/Notebook" title="Daily eNotebook"><SPAN>Notebook</SPAN></A></LI><LI><A href="https://2009.igem.org/Team:Virginia_Commonwealth/People" title="Meet the team"><SPAN>People</SPAN></A></LI><LI><A href="https://2009.igem.org/Team:Virginia_Commonwealth/Internal" title="Brainstorming and discussion"><SPAN>Internal</SPAN></A></LI></UL></DIV><DIV class="ddcolortabsline"> </DIV></DIV><H2><SPAN class="mw-headline" id="Reflections.2Fresponses_to_the_papers_we.27ve_read_together">Reflections/responses to the papers we've read together</SPAN></H2><P>Please share with the team what was meaningful about the reading, how it might be applied to the team's research and what questions you may have.  Links to papers can be found in the <A href="https://2009.igem.org/Team:Virginia_Commonwealth/Literature" class="external text" rel="nofollow"><I>literature compilation</I></A> section and available on blackboard.
</P><UL><LI><I><B>Extreme genetic engineering: an introduction to synthetic biology</B></I> by the ETC Group
</LI><LI>There seems to be so many different areas of research that are considered synthetic biology.  Which areas apply to iGEM and which area(s) will the VCU team be involved in? - <A href="/User:GMcArthurIV" title="User:GMcArthurIV">GMcArthurIV</A> 16:44, 15 May 2009 (UTC)
</LI><LI>The New Synthetic Energy Agenda section on page 27 reflected the type of work the VCU team will be doing.  The cellulose to biofuel and bio remediation to biofuel projects would fall well under this area of syn bio.  Obviously our projects will pertain to biofuel production but the section regarding synbiosafety cannot be ignored - <A href="/wiki/index.php?title=User:Bussingkm&amp;action=edit&amp;redlink=1" class="new" title="User:Bussingkm (page does not exist)">Bussingkm</A> 21:34, 3 June 2009 (UTC)
</LI><LI>What you've mentioned certainly fits within the scope of the research currently going on in the Systems Biological Engineering Lab, but let's talk a bit more broadly.  This paper lists five major research areas in synthetic biology: minimal cells, assembly-line DNA, artifical cells, pathway engineering and novel genetics.  Which areas do you think we could or will be involved in? - <A href="/User:GMcArthurIV" title="User:GMcArthurIV">GMcArthurIV</A> 00:08, 4 June 2009 (UTC)
</LI></UL><LI>As you could guess from my comment below, I think the effects of the cellular &quot;environment&quot; on the functions of individual genetic parts is fascinating and of critical importance, in the same way that your boundaries must be defined for any engineered system.  To this end, I think the minimal genome work is very interesting.  In what situations is it better (or not) to pay attention to all the varied species nature provides us rather than engineering rational systems in the carefully designed, minimal environment provided by a minimal or artificial cell?  My work with thermocellum and the lab's work in general with <I>T. fusca</I> and others tends to rely on existing systems.  At what point do we scrap all that and build the pathways we want into a blank or nearly blank cell? - <A href="/User:Chris.m.gowen" title="User:Chris.m.gowen"> Chris</A> 02:38, 4 June 2009 (UTC)
</LI><LI>When we can actually pare down the models we have to a bare minimum. There's a whole lot of bugs and gray area still in our models and other work. I think the minimal cell projects are coming from the other end; starting blank and adding systems until it fulfills all definitions of a cell. Our work is very important, as the next step after minimal construction will be to direct modification towards a specific purpose. That will proceed much faster if the networks have already been deconstructed. - <A href="/User:Jalvin" title="User:Jalvin">Jalvin</A> 16:33, 4 June 2009 (UTC)
</LI><LI>Why is DNA synthesis technology so important to the development of synthetic biology? Or is it at all?  - <A href="/User:GMcArthurIV" title="User:GMcArthurIV">GMcArthurIV</A> 00:12, 4 June 2009 (UTC)
</LI><LI>I think improvements in DNA synthesis technology are very important to the development of synthetic biology.  With these improvements comes the ability to generate DNA code faster and at a lower cost.  Since the building blocks of synthetic biology are comprised of this DNA code, improvements in the synthesis technology go hand in hand with the development of synthetic biology. - <A href="/wiki/index.php?title=User:Albergca&amp;action=edit&amp;redlink=1" class="new" title="User:Albergca (page does not exist)">Albergca</A> 14:20, 5 June 2009 (UTC)
</LI><UL><LI><I><B>The iGEM competition: building with biology</B></I> by James Brown
</LI><LI>Does this [the iGEM competition] sound exciting to you? - <A href="/User:GMcArthurIV" title="User:GMcArthurIV">GMcArthurIV</A> 00:12, 4 June 2009 (UTC)
</LI></UL><UL><LI><I><B>Foundations for engineering biology</B></I> by Drew Endy
</LI><LI>Drew Endy lists <I>standardization</I> as a key foundational element to enable biological systems.  In what ways do you think we can standardize biological research? - <A href="/User:GMcArthurIV" title="User:GMcArthurIV">GMcArthurIV</A> 00:08, 4 June 2009 (UTC)
</LI><LI>Cellular devices developed for a standard strain seem to be advantageous for many reasons.  For instance, if the devices are interchangeable with other strains and potentially other organisms, time and money can be saved by not having to redevelop the devices.  A standardized stripped down device also lends itself to the idea of abstraction and decoupling.  These minimal devices can be built up and combined into more complex devices or systems capable of achieving more complicated goals. - <A href="/wiki/index.php?title=User:Albergca&amp;action=edit&amp;redlink=1" class="new" title="User:Albergca (page does not exist)">Albergca</A> 14:54, 5 June 2009 (UTC) 
</LI></UL><LI> I think this paper is hilarious. &quot;...microprocessors and other electronic systems are not built directly from chunks of metal and silicon lying about the countryside.&quot;  He alludes to the standardization of a cellular &quot;chassis&quot; when he talks about the different groups using different strains of ecoli to develop different devices (p450, right column).  Do you think it makes more sense to use a standard strain or to more clearly define the differences between strains or even different organisms.  For example, wouldn't it be nice if work on e. coli could be reliably translated for use in <I>T. fusca</I>? - <A href="/User:Chris.m.gowen" title="User:Chris.m.gowen">chris</A> 02:17, 4 June 2009 (UTC)
</LI><UL><LI><I><B>The promise and perils of synthetic biology</B></I> by J. Tucker and R. Zilinskas
</LI><LI>What are some of the major risks associated with synthetic biology research projects? - <A href="/User:GMcArthurIV" title="User:GMcArthurIV">GMcArthurIV</A> 00:08, 4 June 2009 (UTC)
</LI><LI>The risks range from accidental to deliberate and include accidental release, testing in an open environment, and the deliberate misuse of the technology.  Self-regulation seems to be the best avenue to address these risks early before any unforeseen mishaps. - <A href="/wiki/index.php?title=User:Albergca&amp;action=edit&amp;redlink=1" class="new" title="User:Albergca (page does not exist)">Albergca</A> 15:04, 5 June 2009 (UTC)
</LI><LI>Anyone with a decent knowledge of genomic engineering and access to University equipment and literature could easily turn into a crazed bio-terrorist. It's not too difficult to isolate something like MRSA and just grow massive vats of it, putting it into spray bottles and hosing down public transport and other high-volume/distance transmission venues. Not that <I>I've</I> ever thought of that. - <A href="/User:Jalvin" title="User:Jalvin">Jalvin</A> 16:45, 5 June 2009 (UTC)
</LI></UL><UL><LI><I><B>Refinement and standardization of synthetic biological parts and devices</B></I></LI><LI>What does it mean to <I>refine</I> a biological part? - <A href="/User:GMcArthurIV" title="User:GMcArthurIV">GMcArthurIV</A> 11:28, 8 June 2009 (UTC)
</LI><LI>In the context of this article, refinement of a biological part refers to establishing a widely accessible, standardized library of data regarding a biological part or device.  This data informs engineers how the device will interact with the cell once it is transformed.  The device must consistently perform as expected in the standard conditions and cannot be majorly effected by differing variables from situation to situation.  A refined device is analogous to the engine in a car.  If we take that engine out of one car and move it to another than the power output and fuel consumption of the engine will be about the same.  Also, if we change another part of the car, such as the steering wheel than the performance of the engine will not be effected.  I think that refinement of devices is a major step for this industry and should be a major part of our goals this summer.  Refinement of these devices will eventually lead to abstraction. - <A href="/wiki/index.php?title=User:Bussingkm&amp;action=edit&amp;redlink=1" class="new" title="User:Bussingkm (page does not exist)">Bussingkm</A> 14:51, 8 June 2009 (UTC)
</LI><LI>Refinement refers to taking a natural part and making it compatible with some specific assembly standard.  For instance, in terms of the BioBrick assembly standard, a natural part must have the BioBrick restriction endonuclease sites removed to produce a synthetic part that can be used in BioBrick assembly. - <A href="/wiki/index.php?title=User:Albergca&amp;action=edit&amp;redlink=1" class="new" title="User:Albergca (page does not exist)">Albergca</A> 15:27, 8 June 2009 (UTC)
</LI><LI>To refine a part is to find out more information about it, and organize it in a way that makes it more likely that the part, when used in conjunction with something else, will still have the desired function. It wants to get away from the “ad hoc” method so results can be more predictable. This article suggested datasheets with compatibility information on it. - <A href="/wiki/index.php?title=User:Trentay&amp;action=edit&amp;redlink=1" class="new" title="User:Trentay (page does not exist)">Trentay</A> 15:01, 8 June 2009 (UTC)
</LI></UL><LI>Also, I found the signaling device that they used as an example rather confusing.  Just thought you should know. - <A href="/wiki/index.php?title=User:Bussingkm&amp;action=edit&amp;redlink=1" class="new" title="User:Bussingkm (page does not exist)">Bussingkm</A> 14:56, 8 June 2009 (UTC)
</LI><UL><LI><I><B>Measuring the activity of BioBrick promoters using an in vivo reference standard</B></I></LI><LI>Is the method of measuring/characterizing promoters presented in this paper the method we should use in our work? - <A href="/User:GMcArthurIV" title="User:GMcArthurIV">GMcArthurIV</A> 11:28, 8 June 2009 (UTC)
</LI><LI>This is a clear winner for the method of measuring and characterizing promoters.  This method reduces irregularities due to different lab conditions and tools. An added bonus to using this method is that iGEM has built and distributed a kit for this method which is designed for undergraduate researchers. I think that this method is an excellent step toward standardization of practice in the field.  - <A href="/wiki/index.php?title=User:Bussingkm&amp;action=edit&amp;redlink=1" class="new" title="User:Bussingkm (page does not exist)">Bussingkm</A> 14:38, 8 June 2009 (UTC)
</LI><LI> This can be helpful. I do remember the article saying that the promoters that were tested and compared to the reference standard, had to be “standardized” first (p.9 under Standard promoter definition). Would we have to do something like this before we can use the RPU measurement talked about in this paper? Is that what the kit is for? Also, it is still unclear to me as to when this type of measurement is needed. - <A href="/wiki/index.php?title=User:Trentay&amp;action=edit&amp;redlink=1" class="new" title="User:Trentay (page does not exist)">Trentay</A> 15:01, 8 June 2009 (UTC)
</LI><LI>I think the measurement of relative promoter strength as well as the measurement of absolute promoter strength are both useful tools in characterizing promoters.  Obviously reducing the variation in the measurement of promoter activity is important when working towards the standardization of quantifiable information.  However, it is also important to characterize the absolute strength of promoters based on environmental differences, which only an absolute measurement can accomplish. - <A href="/wiki/index.php?title=User:Albergca&amp;action=edit&amp;redlink=1" class="new" title="User:Albergca (page does not exist)">Albergca</A> 15:27, 8 June 2009 (UTC)
</LI></UL><UL><LI><I><B>Engineering BioBrick vectors from BioBrick parts</B></I></LI><LI>This paper demonstrates the versatility of the BioBrick standard in addition to walking through the steps necessary to perform BioBrick assembly. - <A href="/User:GMcArthurIV" title="User:GMcArthurIV">GMcArthurIV</A> 11:28, 8 June 2009 (UTC)
</LI><LI>I just had a few questions: What is the Gateway system? What is ligation reaction? I’ve seen these terms a couple of times and wanted to know what they mean. - <A href="/wiki/index.php?title=User:Trentay&amp;action=edit&amp;redlink=1" class="new" title="User:Trentay (page does not exist)">Trentay</A> 15:45, 8 June 2009 (UTC)
</LI><LI>The Gateway system (by Invitrogen) is a cloning method that provides a &quot;gateway&quot; into any expression system (e.g., shuttle genetic cassette from <I>E. coli</I> plasmid to a mammalian vector).  More information can be found here: <A href="http://www.invitrogen.com/site/us/en/home/Products-and-Services/Applications/Cloning/Gateway-Cloning.html" class="external free" rel="nofollow">http://www.invitrogen.com/site/us/en/home/Products-and-Services/Applications/Cloning/Gateway-Cloning.html</A> - <A href="/User:GMcArthurIV" title="User:GMcArthurIV">GMcArthurIV</A> 19:35, 9 June 2009 (UTC)
</LI></UL><LI>“…the processes of DNA design and DNA synthesis cannot be easily decoupled. Improvements in commercial DNA synthesis are needed that free the process from dependence on in vivo DNA propagation and replication.” (p.6) Have any other methods been developed since this paper was written? - <A href="/wiki/index.php?title=User:Trentay&amp;action=edit&amp;redlink=1" class="new" title="User:Trentay (page does not exist)">Trentay</A> 15:45, 8 June 2009 (UTC)
</LI><UL><LI><I><B>Synthetic biology: lessons from the history of synthetic organic chemistry</B></I></LI><LI>&quot;For example, man-made materials, even at the nanoscale, are currently templated or built using directed assembly— exactly the opposite of how biological molecules create structure and function. Biological molecules are self-assembling systems that can adapt to change, show robust homeostasis, and can self-repair.&quot;  The author makes these points in a different context, but what other ways are biological molecules and systems unique from chemical systems and how can we take advantage of these and other unique qualities of biological molecules (e.g. self-assembly, robustness, excellent specificity)?  Put another way, what can biologicals do that chemicals can't?  Also, <A href="http://xkcd.com/435/" class="external text" rel="nofollow">this</A>. - <A href="/User:Chris.m.gowen" title="User:Chris.m.gowen">chris</A> 04:18, 9 June 2009 (UTC)
</LI><LI>Biological molecules are unique in that once they are built and viable they never need to be repaired.  Once a biofactory is synthesized to produce a metabolite all that needs to be done is to grow it, feed it, and harvest the product.  This is very cheap compared to producing a chemical via traditional methods.  In a chemical plant, if one link in the chain fails than the process must be halted and repaired.  In a biological production system, since all cells are for the most part independent from each other, if one cell fails there is no damage to the overall system.  Also, biological molecules reproduce very fast and cheaply as oppose to chemical factories which are built very slowly and expensively. - <A href="/wiki/index.php?title=User:Bussingkm&amp;action=edit&amp;redlink=1" class="new" title="User:Bussingkm (page does not exist)">Bussingkm</A> 14:54, 9 June 2009 (UTC)
</LI></UL><LI>I think that these passages put things into a nice perspective. In this Commentary, we consider the historical role of synthetic approaches in the development of modern organic chemistry in order to extract some lessons that might help guide the development of synthetic biology. A critical lesson here is that a complete understanding of chemical principles was not a prerequisite for the emergence of synthetic chemistry. The history of organic chemistry suggests that synthesis will be a necessary complement to analysis in order for biologists to truly understand the mechanisms of complex living systems. - <A href="/wiki/index.php?title=User:Bussingkm&amp;action=edit&amp;redlink=1" class="new" title="User:Bussingkm (page does not exist)">Bussingkm</A> 14:54, 9 June 2009 (UTC)
</LI><UL><LI><I><B>Systems biology as a foundation for genome-scale synthetic biology</B></I></LI><LI>We've talked a lot about how we'll use <I>synthetic</I> biology concepts to design and build our projects, but in what ways can we use the methodologies developed by <I>systems</I> biology? - <A href="/User:GMcArthurIV" title="User:GMcArthurIV">GMcArthurIV</A> 21:18, 8 June 2009 (UTC)
</LI><LI>I like the point made that systems biology can help synthetic biologists design systems that will have to function in the context of biological variability (aka evolution).  I view this as similar to the way a chemical engineer uses thermodynamic models to design for process equilibria that are stable in the presence of input perturbations. - <A href="/User:Chris.m.gowen" title="User:Chris.m.gowen">chris</A> 04:18, 9 June 2009 (UTC)
</LI><LI>We can help systems biology by characterizing parts like promoters and terminators and get quantitative data. That way the information could be incorporated into modeling programs to give more predictable results for synthetic biologists using the program. - <A href="/wiki/index.php?title=User:Trentay&amp;action=edit&amp;redlink=1" class="new" title="User:Trentay (page does not exist)">Trentay</A> 15:56, 9 June 2009 (UTC)
</LI></UL><LI> They make the case for systems biology developing to eventually meet the needs of synthetic biologists (and vice-versa), but do you think it is currently well-enough developed to accomplish this?  What would you minimally require in order to trust predictions from an <I>in silico</I> reconstruction? - <A href="/User:Chris.m.gowen" title="User:Chris.m.gowen">chris</A> 04:18, 9 June 2009 (UTC)
</LI><LI>I'm not sure of what to make of this passage: &quot;Particularly, reconstruction methodology [23] highlights how to efficiently identify cellular constituent — genes, for instance — by making use of genome context analysis that cannot be simply annotated by homology-based search and, thereby, improves the reconstructed metabolic network.&quot; -<A href="/wiki/index.php?title=User:Bussingkm&amp;action=edit&amp;redlink=1" class="new" title="User:Bussingkm (page does not exist)">Bussingkm</A> 15:21, 9 June 2009 (UTC)
</LI><UL><LI><I><B>Biology by design: reduction and synthesis of cellular components and behaviour</B></I></LI><LI>This may be a bit of review at this point.  Was there anything new or important that stuck out to you while reading this? - <A href="/User:GMcArthurIV" title="User:GMcArthurIV">GMcArthurIV</A> 21:24, 9 June 2009 (UTC) 
</LI></UL><UL><LI><I><B>Genetic parts to program bacteria</B></I> (discussion on Thursday, June 11)
</LI><LI>How do you think we could use synthetic genetic circuits to control metabolic pathways? - <A href="/User:GMcArthurIV" title="User:GMcArthurIV">GMcArthurIV</A> 18:57, 10 June 2009 (UTC)
</LI><LI>We could use a genetic circuit to switch the metabolic pathway on or off using an external signal.  This would be useful to minimize the stress on the cell and colony while it is growing, then once the colony is at the desired size all of the cell's resources could be used for the metabolite.  We would also have to down regulate the replication function within the cell. -<A href="/wiki/index.php?title=User:Bussingkm&amp;action=edit&amp;redlink=1" class="new" title="User:Bussingkm (page does not exist)">Bussingkm</A> 15:38, 11 June 2009 (UTC)
</LI><LI>The article mentions using synthetic genetic circuits to produce a series of metabolic proteins &quot;at the precise time and amount required for the maximal synthesis of a drug or energetic compound, while diverting the flux from competing pathways&quot; (pg 554).  Using the information-processing common to electric circuits, a cell can be programmed to perform a series of coordinated tasks such as maximizing certain metabolic pathways. - <A href="/wiki/index.php?title=User:Albergca&amp;action=edit&amp;redlink=1" class="new" title="User:Albergca (page does not exist)">Albergca</A> 15:57, 11 June 2009 (UTC)
</LI></UL><UL><LI><I><B>Toward scalable parts families for predictable design of biological circuits</B></I> - <FONT color="red"><B>discussion on Monday, June 15</B></FONT></LI></UL><UL><LI><I><B>Engineering frameworks with synthetic biology frameworks</B></I> - <FONT color="red"><B>discussion on Monday, June 15</B></FONT></LI></UL><UL><LI><I><B>Synthetic biology for synthetic chemistry</B></I> - <FONT color="red"><B>discussion on Monday, June 15</B></FONT></LI></UL><DIV class="printfooter">
Retrieved from &quot;<A href="http://2009.igem.org/Team:Virginia_Commonwealth/Internal/Papers">http://2009.igem.org/Team:Virginia_Commonwealth/Internal/Papers</A>&quot;</DIV></DIV></DIV><DIV id="footer-box"><DIV id="footer"><UL id="f-list"><LI id="t-recentchanges"><A href="/Special:RecentChanges" title="Recent changes">Recent changes</A></LI><LI id="t-whatlinkshere"><A href="/Special:WhatLinksHere/Team:Virginia_Commonwealth/Internal/Papers" title="List of all wiki pages that link here [j]" accesskey="j">What links here</A></LI><LI id="t-recentchangeslinked"><A href="/Special:RecentChangesLinked/Team:Virginia_Commonwealth/Internal/Papers" title="Recent changes in pages linked from this page [k]" accesskey="k">Related changes</A></LI><LI id="t-specialpages"><A href="/Special:SpecialPages" title="List of all special pages [q]" accesskey="q">Special pages</A></LI><LI><A href="/Special:Preferences">My preferences</A></LI></UL></DIV><DIV id="footer"><UL id="f-list"><LI id="t-print"><A href="/wiki/index.php?title=Team:Virginia_Commonwealth/Internal/Papers&amp;printable=yes" title="Printable version of this page [p]" accesskey="p">Printable version</A></LI><LI id="t-permalink"><A href="/wiki/index.php?title=Team:Virginia_Commonwealth/Internal/Papers&amp;oldid=7795" title="Permanent link to this revision of the page">Permanent link</A></LI><LI id="privacy"><A href="/2009.igem.org:Privacy_policy" title="2009.igem.org:Privacy policy">Privacy policy</A></LI><LI id="disclaimer"><A href="/2009.igem.org:General_disclaimer" title="2009.igem.org:General disclaimer">Disclaimers</A></LI></UL></DIV></DIV></DIV></BODY></HTML>