<HTML xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en" dir="ltr">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki  ltr ns-0 ns-subject page-Team_Virginia_Commonwealth_Internal_Project_ideas"><DIV id="globalWrapper"><DIV id="top-section"><DIV id="p-logo"><A href="/Main_Page" title="Main Page">&quot;
	    </A></DIV><DIV id="menubar" class="left-menu"><UL><LI class="selected"><A href="/Team:Virginia_Commonwealth/Internal/Project_ideas">Page               </A></LI><LI class="new"><A href="/wiki/index.php?title=Talk:Team:Virginia_Commonwealth/Internal/Project_ideas&amp;action=edit&amp;redlink=1">Discussion               </A></LI><LI><A href="/wiki/index.php?title=Team:Virginia_Commonwealth/Internal/Project_ideas&amp;action=edit">View source               </A></LI><LI><A href="/wiki/index.php?title=Team:Virginia_Commonwealth/Internal/Project_ideas&amp;action=history">History               </A></LI><LI style="color:#808080;cursor:default">teams</LI></UL></DIV><DIV class="right-menu" id="menubar"><UL><LI id="pt-login"><A href="/wiki/index.php?title=Special:UserLogin&amp;returnto=Team:Virginia_Commonwealth/Internal/Project_ideas" title="You are encouraged to log in; however, it is not mandatory [o]" accesskey="o">Log in</A></LI></UL></DIV><DIV id="search-controls"><FORM action="/Special:Search" id="searchform"> </FORM></DIV></DIV><DIV id="content"><H1 class="firstHeading">Team:Virginia Commonwealth/Internal/Project ideas</H1><DIV id="bodyContent"><H3 id="siteSub">From 2009.igem.org</H3><DIV id="header"><TITLE>Virginia Commonwealth iGEM 2009</TITLE><DIV id="colortab" class="ddcolortabs"><UL><LI><A href="https://2009.igem.org/Team:Virginia_Commonwealth" title="Home"><SPAN>Home</SPAN></A></LI><LI><A href="https://2009.igem.org/Team:Virginia_Commonwealth/Overview" title="Overview"><SPAN>Overview</SPAN></A></LI><LI><A href="https://2009.igem.org/Team:Virginia_Commonwealth/Design" title="Part design"><SPAN>Design</SPAN></A></LI><LI><A href="https://2009.igem.org/Team:Virginia_Commonwealth/Methods" title="Characterization methods" rel="dropmenu2_a"><SPAN>Methods</SPAN></A></LI><LI><A href="https://2009.igem.org/Team:Virginia_Commonwealth/Results" title="Results" rel="dropmenu1_a"><SPAN>Results</SPAN></A></LI><LI><A href="https://2009.igem.org/Team:Virginia_Commonwealth/BioBricks" title="Our submitted BioBrick parts" rel="dropmenu1_a"><SPAN>BioBricks</SPAN></A></LI><LI><A href="https://2009.igem.org/Team:Virginia_Commonwealth/Notebook" title="Daily eNotebook"><SPAN>Notebook</SPAN></A></LI><LI><A href="https://2009.igem.org/Team:Virginia_Commonwealth/People" title="Meet the team"><SPAN>People</SPAN></A></LI><LI><A href="https://2009.igem.org/Team:Virginia_Commonwealth/Internal" title="Brainstorming and discussion"><SPAN>Internal</SPAN></A></LI></UL></DIV><DIV class="ddcolortabsline"> </DIV></DIV><P>Use this space to brainstorm project ideas.  Go crazy with it. Feel free to make comments and suggestions to existing ideas.  Please sign your comments by inserting &quot;~~~~&quot; after each so that we can keep track of who is communicating and when the information was posted.
</P><TABLE id="toc" class="toc"><TBODY><TR><TD><DIV id="toctitle"><H2>Contents</H2></DIV><UL><LI class="toclevel-1 tocsection-1"><A href="#Biofuels"><SPAN class="tocnumber">1</SPAN><SPAN class="toctext">Biofuels</SPAN></A></LI><LI class="toclevel-2 tocsection-2"><A href="#Overview"><SPAN class="tocnumber">1.1</SPAN><SPAN class="toctext">Overview</SPAN></A></LI><LI class="toclevel-2 tocsection-3"><A href="#Comment"><SPAN class="tocnumber">1.2</SPAN><SPAN class="toctext">Comment</SPAN></A></LI></UL><LI class="toclevel-1 tocsection-4"><A href="#Bioprocessing"><SPAN class="tocnumber">2</SPAN><SPAN class="toctext">Bioprocessing</SPAN></A></LI><LI class="toclevel-2 tocsection-5"><A href="#Overview_2"><SPAN class="tocnumber">2.1</SPAN><SPAN class="toctext">Overview</SPAN></A></LI><LI class="toclevel-2 tocsection-6"><A href="#Comments"><SPAN class="tocnumber">2.2</SPAN><SPAN class="toctext">Comments</SPAN></A></LI><LI class="toclevel-1 tocsection-7"><A href="#Bio-remediation:_Carbon_fixation"><SPAN class="tocnumber">3</SPAN><SPAN class="toctext">Bio-remediation: Carbon fixation</SPAN></A></LI><LI class="toclevel-2 tocsection-8"><A href="#Overview_3"><SPAN class="tocnumber">3.1</SPAN><SPAN class="toctext">Overview</SPAN></A></LI><LI class="toclevel-2 tocsection-9"><A href="#Comments_2"><SPAN class="tocnumber">3.2</SPAN><SPAN class="toctext">Comments</SPAN></A></LI><LI class="toclevel-1 tocsection-10"><A href="#Preemptive_Part_Optimization"><SPAN class="tocnumber">4</SPAN><SPAN class="toctext">Preemptive Part Optimization</SPAN></A></LI><LI class="toclevel-2 tocsection-11"><A href="#Overview_4"><SPAN class="tocnumber">4.1</SPAN><SPAN class="toctext">Overview</SPAN></A></LI><LI class="toclevel-2 tocsection-12"><A href="#Comments_3"><SPAN class="tocnumber">4.2</SPAN><SPAN class="toctext">Comments</SPAN></A></LI><LI class="toclevel-1 tocsection-13"><A href="#Systematic_characterization_of_promoters"><SPAN class="tocnumber">5</SPAN><SPAN class="toctext">Systematic characterization of promoters</SPAN></A></LI><LI class="toclevel-2 tocsection-14"><A href="#Overview_5"><SPAN class="tocnumber">5.1</SPAN><SPAN class="toctext">Overview</SPAN></A></LI><LI class="toclevel-2 tocsection-15"><A href="#Comments_4"><SPAN class="tocnumber">5.2</SPAN><SPAN class="toctext">Comments</SPAN></A></LI><LI class="toclevel-1 tocsection-16"><A href="#Development_of_thermostable_enzymes"><SPAN class="tocnumber">6</SPAN><SPAN class="toctext">Development of thermostable enzymes</SPAN></A></LI><LI class="toclevel-2 tocsection-17"><A href="#Overview_6"><SPAN class="tocnumber">6.1</SPAN><SPAN class="toctext">Overview</SPAN></A></LI><LI class="toclevel-2 tocsection-18"><A href="#Comments_5"><SPAN class="tocnumber">6.2</SPAN><SPAN class="toctext">Comments</SPAN></A></LI><LI class="toclevel-1 tocsection-19"><A href="#Extracellular_enzyme_scaffold"><SPAN class="tocnumber">7</SPAN><SPAN class="toctext">Extracellular enzyme scaffold</SPAN></A></LI><LI class="toclevel-2 tocsection-20"><A href="#Overview_7"><SPAN class="tocnumber">7.1</SPAN><SPAN class="toctext">Overview</SPAN></A></LI><LI class="toclevel-2 tocsection-21"><A href="#Problem_areas"><SPAN class="tocnumber">7.2</SPAN><SPAN class="toctext">Problem areas</SPAN></A></LI><LI class="toclevel-2 tocsection-22"><A href="#Organisms_with_cellulosomes"><SPAN class="tocnumber">7.3</SPAN><SPAN class="toctext">Organisms with cellulosomes</SPAN></A></LI><LI class="toclevel-2 tocsection-23"><A href="#Comments_6"><SPAN class="tocnumber">7.4</SPAN><SPAN class="toctext">Comments</SPAN></A></LI><LI class="toclevel-1 tocsection-24"><A href="#Yeast_that_make_gluten_free_beer"><SPAN class="tocnumber">8</SPAN><SPAN class="toctext">Yeast that make gluten free beer</SPAN></A></LI><LI class="toclevel-2 tocsection-25"><A href="#Overview_8"><SPAN class="tocnumber">8.1</SPAN><SPAN class="toctext">Overview</SPAN></A></LI><LI class="toclevel-2 tocsection-26"><A href="#Comments_7"><SPAN class="tocnumber">8.2</SPAN><SPAN class="toctext">Comments</SPAN></A></LI><LI class="toclevel-1 tocsection-27"><A href="#Bacterial_interaction"><SPAN class="tocnumber">9</SPAN><SPAN class="toctext">Bacterial interaction</SPAN></A></LI><LI class="toclevel-2 tocsection-28"><A href="#Overview_9"><SPAN class="tocnumber">9.1</SPAN><SPAN class="toctext">Overview</SPAN></A></LI><LI class="toclevel-1 tocsection-29"><A href="#Enhanced_regulation_of_cellular_replication"><SPAN class="tocnumber">10</SPAN><SPAN class="toctext">Enhanced regulation of cellular replication</SPAN></A></LI><LI class="toclevel-2 tocsection-30"><A href="#Overview_10"><SPAN class="tocnumber">10.1</SPAN><SPAN class="toctext">Overview</SPAN></A></LI><LI class="toclevel-1 tocsection-31"><A href="#Regulate_insulin_gene_expression_in_insulin_producing_E._coli"><SPAN class="tocnumber">11</SPAN><SPAN class="toctext">Regulate insulin gene expression in insulin producing E. coli</SPAN></A></LI><LI class="toclevel-2 tocsection-32"><A href="#Overview_11"><SPAN class="tocnumber">11.1</SPAN><SPAN class="toctext">Overview</SPAN></A></LI><LI class="toclevel-2 tocsection-33"><A href="#Recombinant_Protein_Production_and_Transport_Outside_of_the_Cell"><SPAN class="tocnumber">11.2</SPAN><SPAN class="toctext">Recombinant Protein Production and Transport Outside of the Cell</SPAN></A></LI><LI class="toclevel-2 tocsection-34"><A href="#Redesign_Characterized_Promoters_into_Environment-Responsive_Promoters"><SPAN class="tocnumber">11.3</SPAN><SPAN class="toctext">Redesign Characterized Promoters into Environment-Responsive Promoters</SPAN></A></LI></TD></TR></TBODY></TABLE><H2><SPAN class="mw-headline" id="Biofuels">Biofuels</SPAN></H2><H3><SPAN class="mw-headline" id="Overview">Overview</SPAN></H3><H3><SPAN class="mw-headline" id="Comment">Comment</SPAN></H3><H2><SPAN class="mw-headline" id="Bioprocessing">Bioprocessing</SPAN></H2><H3><SPAN class="mw-headline" id="Overview_2">Overview</SPAN></H3><H3><SPAN class="mw-headline" id="Comments">Comments</SPAN></H3><H2><SPAN class="mw-headline" id="Bio-remediation:_Carbon_fixation">Bio-remediation: Carbon fixation</SPAN></H2><H3><SPAN class="mw-headline" id="Overview_3">Overview</SPAN></H3><P>CO<SUB>2</SUB> levels are rising and will likely have significant effects on the biosphere. Many propose that the proper action to take is to eliminate and eventually sequester large amounts of CO<SUB>2</SUB> in order to abate or attempt to reverse the effects.
</P><H3><SPAN class="mw-headline" id="Comments_2">Comments</SPAN></H3><UL><LI>Fixation via photosynthesis
</LI><LI>Fixation via carbonic anhydrase, base for limestone synthesis
</LI><LI>?
</LI><LI>During our brainstorming session, we discussed bio-remediation for carbon fixation, but also respect to oil spills. Any ideas? <A href="/wiki/index.php?title=User:MandM&amp;action=edit&amp;redlink=1" class="new" title="User:MandM (page does not exist)">MandM</A> 16:22, 7 June 2009 (UTC)
</LI></UL><H2><SPAN class="mw-headline" id="Preemptive_Part_Optimization">Preemptive Part Optimization</SPAN></H2><P>Much time in (synthetic) biology is spent optimizing devices after importing parts from a model organism into a different model. A focused effort could eliminate these problems. This project would likely involve generating a bank of part/brick/device combinations and their relative efficiencies within a model organism and other organisms.  
</P><H3><SPAN class="mw-headline" id="Overview_4">Overview</SPAN></H3><H3><SPAN class="mw-headline" id="Comments_3">Comments</SPAN></H3><H2><SPAN class="mw-headline" id="Systematic_characterization_of_promoters">Systematic characterization of promoters</SPAN></H2><H3><SPAN class="mw-headline" id="Overview_5">Overview</SPAN></H3><P>Significant knowledge gaps remain in the functional and dynamic characterization of several gene promoters, and engineering decisions depend on detailed quantitative information.  
</P><H3><SPAN class="mw-headline" id="Comments_4">Comments</SPAN></H3><UL><LI>George, do you have any good papers about this? <A href="/User:Chris.m.gowen" title="User:Chris.m.gowen">Chris.m.gowen</A> 01:08, 22 April 2009 (UTC)
</LI><LI>Yeah, there is a recent paper from Endy <I>et al.</I> titled <A href="http://www.jbioleng.org/content/pdf/1754-1611-3-4.pdf" class="external text" rel="nofollow"><I>Measuring the activity of BioBrick promoters using an in vivo reference standard</I></A>.  In addition, there is a promoter measurement kit that has been developed by Jason Kelly (Ginkgo Bioworks) that is distributed by the Registry (I believe it was used in the paper mentioned above).  More information, including a protocol, is available <A href="http://partsregistry.org/Measurement" class="external text" rel="nofollow">here</A>. <A href="/User:GMcArthurIV" title="User:GMcArthurIV">GMcArthurIV</A> 15:18, 17 May 2009 (UTC)
</LI><LI>Here is another good paper by Adams and Hatfield titled <A href="http://www.jbc.org/cgi/reprint/259/12/7399" class="external text" rel="nofollow">Effects of promoter strengths and growth conditions on copy number of transcription-fusion vectors</A>.  This idea of quantifying promoter strength seems very interesting and important to the idea of standardization.  I also like the idea of using a reference promoter to minimize the variances in promoter strength due to environmental and instrumental differences. - <A href="/wiki/index.php?title=User:Albergca&amp;action=edit&amp;redlink=1" class="new" title="User:Albergca (page does not exist)">Albergca</A> 22:58, 4 June 2009 (UTC)
</LI></UL><H2><SPAN class="mw-headline" id="Development_of_thermostable_enzymes">Development of thermostable enzymes</SPAN></H2><H3><SPAN class="mw-headline" id="Overview_6">Overview</SPAN></H3><P>Bacteria and Archea occupy nearly every nook and cranny on Earth; from deep beneath Arctic ice in the absence of light, to temperatures over boiling in Yellowstone. As thermodynamics tells us, reaction rates increase proportional to temperature. Therefore, an industrial enzymatic process normally carried out at 37<SUP>o</SUP>C, could be run at higher temperatures for a faster turnover. Additionally, incubating microbes is very energy-intensive, and the knowledge of thermostable properties can also be applied to utilize thermophilic enzymes in meso- or even psychrophiles. This could lead to Quiagen kits that don't require any incubation, yet can proceed at the same rate.
</P><H3><SPAN class="mw-headline" id="Comments_5">Comments</SPAN></H3><UL><LI>I think looking at homologous proteins across temperature preferences would be a good start. Usually dense G-C regions are more thermostable in terms of T<SUB>m</SUB>, similar substitutions in amino acid sequences and tertiary/quarternary structure. Also, prions would be neat to look at, because they don't denature at 60°C, but require an extreme of 134°C for a while. Also, partially denatured prions can renature and become active and infections. See: <A href="http://www.pnas.org/content/83/3/576.full.pdf" class="external text" rel="nofollow">Isolation of a thermostable enzyme variant by cloning and selection in a thermophile</A><A href="/User:Jalvin" title="User:Jalvin">Jalvin</A> 20:39, 28 May 2009 (UTC)
</LI><LI>Interesting idea, Joe.  That's generally true about GC content, although a strange anomaly is that C. thermocellum has a genome-wide GC content of only ~39%.  Fun Fact.  <A href="/User:Chris.m.gowen" title="User:Chris.m.gowen">Chris.m.gowen</A> 18:26, 29 May 2009 (UTC)
</LI></UL><H2><SPAN class="mw-headline" id="Extracellular_enzyme_scaffold">Extracellular enzyme scaffold</SPAN></H2><H3><SPAN class="mw-headline" id="Overview_7">Overview</SPAN></H3><P>A cell-associated protein scaffold which could accommodate various enzymes or binding proteins can create a cellular assembly line by drastically reducing inefficiencies of product and reactant diffusion and by aligning binding sites to targets.  This concept is what makes the cellulosome of <I>Clostridium thermocellum</I> and other organisms so efficient at breaking down cellulose.  See the <A href="http://www.springerlink.com/content/785bm9d7uugugrdn/" class="external text" rel="nofollow">review</A> by Schwarz for an overview of this system.  A protein known as scaffoldin is a consistent part of each cellulosome, and provides the basic structure to which each catalytic enzyme attaches.  In many cases a cohesin module can bind a wide range of proteins to the scaffoldin, and a dockerin module binds the scaffoldin to the cell surface.  If we could export the scaffoldin and a dockerin module to <I>E. coli</I>, it may be possible to add the cohesin domain to the non-catalytic regions of desirable enzymes and create a synthetic &quot;reaction line.&quot;
</P><H3><SPAN class="mw-headline" id="Problem_areas">Problem areas</SPAN></H3><UL><LI> Protein folding may be quite different in <I>E. coli</I>, especially from thermophilic source organisms.
</LI><LI> The dockerin module binds to the cell wall of the gram positive source organisms, which ecoli of course doesn't have...
</LI><LI> Integrating a foreign cohesin domain into native enzymes is in the realm of protein engineering, with which I am not familiar
</LI><LI> Here are some programs that may provide a way to verify protein docking <A href="http://openwetware.org/wiki/Wikiomics:Docking_toolbox#Protein-protein_docking" class="external text" rel="nofollow">here</A>; <A href="/User:Chris.m.gowen" title="User:Chris.m.gowen">Chris.m.gowen</A> 01:46, 22 April 2009 (UTC)
</LI></UL><LI> Others?
</LI><H3><SPAN class="mw-headline" id="Organisms_with_cellulosomes">Organisms with cellulosomes</SPAN></H3><P><I>Clostridium thermocellum</I><I>C. acetobutylicum</I></P><H3><SPAN class="mw-headline" id="Comments_6">Comments</SPAN></H3><H2><SPAN class="mw-headline" id="Yeast_that_make_gluten_free_beer">Yeast that make gluten free beer</SPAN></H2><H3><SPAN class="mw-headline" id="Overview_8">Overview</SPAN></H3><P>Gluten is a protein (proteins?) in wheat that is in most breads and beers.  Many people have allergies to these proteins, and so have to carefully avoid gluten-containing products.  Maybe we could engineer a pathway to specifically break down gluten in the brewing process?
</P><H3><SPAN class="mw-headline" id="Comments_7">Comments</SPAN></H3><P>Using yeast may be more trouble than we want for the first year. <A href="/User:Chris.m.gowen" title="User:Chris.m.gowen">chris</A></P><P>I'm not 21. <A href="/User:Jalvin" title="User:Jalvin">Jalvin</A> 20:08, 28 May 2009 (UTC)
</P><H2><SPAN class="mw-headline" id="Bacterial_interaction">Bacterial interaction</SPAN></H2><H3><SPAN class="mw-headline" id="Overview_9">Overview</SPAN></H3><P>Some kind of social behavior, like quorum sensing or self assembly based on communicating only with neighboring cells would be cool.  This would be a good way to take advantage of Advait's cellular automata expertise before he leaves.  This would also be somthing that would require good quantitative characterization of our genes and could be combined with the promoter characterization stuff, maybe.
</P><H2><SPAN class="mw-headline" id="Enhanced_regulation_of_cellular_replication">Enhanced regulation of cellular replication</SPAN></H2><H3><SPAN class="mw-headline" id="Overview_10">Overview</SPAN></H3><P>The implications of the ability to control cellular replication are tremendous.  The possibility of increasing or decreasing the rate of the cell division cycle via gene manipulation has very broad scope, from increasing metabolite production to decreasing or terminating unbounded cell growth.  In the arena of biofuels, it would be of enormous value to create a large mass of cells capable of producing certain metabolites of interest in a short time period.  Since cell replication uses large amounts of energy, it would be beneficial to reduce or terminate cell replication once a large amount of cells have been produced.  An idea mentioned by George involves manipulating the cascades of quorum sensing to achieve these purposes.
</P><H2><SPAN class="mw-headline" id="Regulate_insulin_gene_expression_in_insulin_producing_E._coli">Regulate insulin gene expression in insulin producing E. coli</SPAN></H2><H3><SPAN class="mw-headline" id="Overview_11">Overview</SPAN></H3><P>Chemical production companies find it useful to control the rate that they can produce their product.  This makes it possible for them to cope with varying rate of demand.  Insulin is a valuable drug that is mass-produced.  A simple way to control insulin production by E. coli would be a valuable tool.  We could use bio-bricks to insert the proper DNA sequences into E. coli to produce insulin then regulate that gene’s expression by manipulating either the promoter or the ribosomal binding site.
-Kevin
</P><UL><LI>Does anyone have any insight on the metabolic pathway for insulin and the enzymes that are involved.  Maybe like a paper to read or something?  
</LI><LI>Interesting suggestion, Kevin.  I don't know what promoter/RBS combination is used in front of the coding region for insulin in current biotechnological practice, but it could be possible that more powerful combinations exist and could be predicted using quality part characterization.  Since in this case the protein itself is the product of interest, pathway engineering as we discussed in the meeting today wouldn't apply the same way, except that an adequate supply of ATP and amino acids would be necessary. <A href="/User:Chris.m.gowen" title="User:Chris.m.gowen">chris</A> 20:09, 8 June 2009 (UTC)
</LI></UL><H3><SPAN class="mw-headline" id="Recombinant_Protein_Production_and_Transport_Outside_of_the_Cell">Recombinant Protein Production and Transport Outside of the Cell</SPAN></H3><UL><LI>A problem that is protein based drugs are not transported out of the cell, the cell must be destroyed to harvest the protein.  We could implant a transporter that can be found in a Gram positive organism into the membrane of the cell to transport protein across the membrane.  
</LI></UL><P>There are 7 or 8 secretion systems that are very widely studied that could be looked at for this.
</P><H3><SPAN class="mw-headline" id="Redesign_Characterized_Promoters_into_Environment-Responsive_Promoters">Redesign Characterized Promoters into Environment-Responsive Promoters</SPAN></H3><P>Once we have a list of characterized promoters we could look at their design and find a similarity between them and a promoter that is responsive to either a chemical signal or an environmental trigger.  If we can redesign a promoter into a responsive promoter than we should do that for all of our characterized promoters and see if they have the same relative gene expression intensity after redesign.  If so than we could explore redesigning them to respond to a multitude of different signals.  If we do this than we would have a diverse toolbox of promoters that would be a huge step toward standardization and characterization.  -<A href="/wiki/index.php?title=User:Bussingkm&amp;action=edit&amp;redlink=1" class="new" title="User:Bussingkm (page does not exist)">Bussingkm</A> 15:50, 11 June 2009 (UTC)
</P><DIV class="printfooter">
Retrieved from &quot;<A href="http://2009.igem.org/Team:Virginia_Commonwealth/Internal/Project_ideas">http://2009.igem.org/Team:Virginia_Commonwealth/Internal/Project_ideas</A>&quot;</DIV></DIV></DIV><DIV id="footer-box"><DIV id="footer"><UL id="f-list"><LI id="t-recentchanges"><A href="/Special:RecentChanges" title="Recent changes">Recent changes</A></LI><LI id="t-whatlinkshere"><A href="/Special:WhatLinksHere/Team:Virginia_Commonwealth/Internal/Project_ideas" title="List of all wiki pages that link here [j]" accesskey="j">What links here</A></LI><LI id="t-recentchangeslinked"><A href="/Special:RecentChangesLinked/Team:Virginia_Commonwealth/Internal/Project_ideas" title="Recent changes in pages linked from this page [k]" accesskey="k">Related changes</A></LI><LI id="t-specialpages"><A href="/Special:SpecialPages" title="List of all special pages [q]" accesskey="q">Special pages</A></LI><LI><A href="/Special:Preferences">My preferences</A></LI></UL></DIV><DIV id="footer"><UL id="f-list"><LI id="t-print"><A href="/wiki/index.php?title=Team:Virginia_Commonwealth/Internal/Project_ideas&amp;printable=yes" title="Printable version of this page [p]" accesskey="p">Printable version</A></LI><LI id="t-permalink"><A href="/wiki/index.php?title=Team:Virginia_Commonwealth/Internal/Project_ideas&amp;oldid=7248" title="Permanent link to this revision of the page">Permanent link</A></LI><LI id="privacy"><A href="/2009.igem.org:Privacy_policy" title="2009.igem.org:Privacy policy">Privacy policy</A></LI><LI id="disclaimer"><A href="/2009.igem.org:General_disclaimer" title="2009.igem.org:General disclaimer">Disclaimers</A></LI></UL></DIV></DIV></DIV></BODY></HTML>