<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_TU_Darmstadt_Project_Caprolactone skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><DIV id="top_title"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:TU Darmstadt/Project/Caprolactone</SPAN></H1></DIV><DIV id="HQ_page"><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><P class="title">combimer</P><UL><LI><A href="https://2018.igem.org/Team:TU_Darmstadt">Home</A></LI><LI class="multiple"><A>People</A></LI><LI><A href="https://2018.igem.org/Team:TU_Darmstadt/Team">Team</A></LI><LI><A href="https://2018.igem.org/Team:TU_Darmstadt/Collaborations">Collaborations</A></LI><LI><A href="https://2018.igem.org/Team:TU_Darmstadt/Sponsoring">Sponsoring</A></LI><LI><A href="https://2018.igem.org/Team:TU_Darmstadt/Attributions">Attributions</A></LI></UL><LI class="multiple"><A>Project</A></LI><LI><A href="https://2018.igem.org/Team:TU_Darmstadt/Project/Background">Background</A></LI><LI><A href="https://2018.igem.org/Team:TU_Darmstadt/Project/Monomers">Monomers</A></LI><LI class="sub"><A href="https://2018.igem.org/Team:TU_Darmstadt/Project/Glycolic_acid/E_coli">Glycolic Acid 
                 Production in <I>E. coli</I></A></LI><LI class="sub"><A href="https://2018.igem.org/Team:TU_Darmstadt/Project/Glycolic_acid/S_cerevisiae">Glycolic Acid Production in 
                <I>S. cerevisiae</I></A></LI><LI class="sub"><A href="https://2018.igem.org/Team:TU_Darmstadt/Project/Caprolactone">Caprolactone Production in <I>E. coli</I></A></LI><LI><A href="https://2018.igem.org/Team:TU_Darmstadt/Project/Polymers">Polymers</A></LI><LI class="sub"><A href="https://2018.igem.org/Team:TU_Darmstadt/Project/Analysis">Analytical Background</A></LI><LI class="sub"><A href="https://2018.igem.org/Team:TU_Darmstadt/Project/PLGA">PLGA</A></LI><LI class="sub"><A href="https://2018.igem.org/Team:TU_Darmstadt/Project/PLGC">PLGC</A></LI><LI class="sub"><A href="https://2018.igem.org/Team:TU_Darmstadt/Project/Applications">Applications</A></LI><LI><A href="https://2018.igem.org/Team:TU_Darmstadt/Project/Upscaling">Upscaling</A></LI><LI><A href="https://2018.igem.org/Team:TU_Darmstadt/Notebook">Notebook</A></LI><LI class="multiple"><A>Achievements</A></LI><LI><A href="https://2018.igem.org/Team:TU_Darmstadt/InterLab">InterLab</A></LI><LI><A href="https://2018.igem.org/Team:TU_Darmstadt/Composite_Part">Parts</A></LI><LI><A href="https://2018.igem.org/Team:TU_Darmstadt/Demonstrate">Proof of Concept</A></LI><LI><A href="https://2018.igem.org/Team:TU_Darmstadt/Applied_Design">Applied Design</A></LI><LI><A href="https://2018.igem.org/Team:TU_Darmstadt/Model">Modeling</A></LI><LI class="multiple"><A>Human Practices</A></LI><LI><A href="https://2018.igem.org/Team:TU_Darmstadt/Human_Practices">Human Practices</A></LI><LI><A href="https://2018.igem.org/Team:TU_Darmstadt/Public_Engagement">Education &amp; Engagement</A></LI><LI><A href="https://2018.igem.org/Team:TU_Darmstadt/Safety">Safety</A></LI><DIV class="igem_2018_team_content"><DIV class="igem_2018_team_column_wrapper"><DIV id="toc" class="toc"><DIV id="toctitle"><H2>Contents</H2></DIV><UL><LI class="toclevel-1 tocsection-1"><A href="#Caprolactone_Production_in_E..C2.A0coli"><SPAN class="tocnumber">1</SPAN><SPAN class="toctext">Caprolactone Production in <I>E. coli </I></SPAN></A></LI><LI class="toclevel-2"><A href="#Abstract"><SPAN class="tocnumber">1.1</SPAN><SPAN class="toctext">Abstract</SPAN></A></LI><LI class="toclevel-2 tocsection-2"><A href="#Introduction"><SPAN class="tocnumber">1.2</SPAN><SPAN class="toctext">Introduction</SPAN></A></LI><LI class="toclevel-2 tocsection-3"><A href="#Methods"><SPAN class="tocnumber">1.3</SPAN><SPAN class="toctext">Methods</SPAN></A></LI><LI class="toclevel-3 tocsection-4"><A href="#Cloning"><SPAN class="tocnumber">1.3.1</SPAN><SPAN class="toctext">Cloning</SPAN></A></LI><LI class="toclevel-3 tocsection-5"><A href="#SDS-PAGE_and_Western_Blot"><SPAN class="tocnumber">1.3.2</SPAN><SPAN class="toctext">SDS-PAGE and Western Blot</SPAN></A></LI><LI class="toclevel-3 tocsection-6"><A href="#Purification"><SPAN class="tocnumber">1.3.3</SPAN><SPAN class="toctext">Purification</SPAN></A></LI><LI class="toclevel-3 tocsection-7"><A href="#Enzyme_Assays"><SPAN class="tocnumber">1.3.4</SPAN><SPAN class="toctext">Enzyme Assays</SPAN></A></LI><LI class="toclevel-3 tocsection-8"><A href="#Growth_Assay"><SPAN class="tocnumber">1.3.5</SPAN><SPAN class="toctext">Growth Assay</SPAN></A></LI><LI class="toclevel-4 tocsection-9"><A href="#M9_Minimal_Medium"><SPAN class="tocnumber">1.3.5.1</SPAN><SPAN class="toctext">M9 Minimal Medium</SPAN></A></LI></UL><LI class="toclevel-2 tocsection-10"><A href="#Results_and_Discussion"><SPAN class="tocnumber">1.4</SPAN><SPAN class="toctext">Results and Discussion</SPAN></A></LI><LI class="toclevel-3 tocsection-11"><A href="#Cloning_2"><SPAN class="tocnumber">1.4.1</SPAN><SPAN class="toctext">Cloning</SPAN></A></LI><LI class="toclevel-3 tocsection-12"><A href="#SDS-PAGE_and_Western_Blot_2"><SPAN class="tocnumber">1.4.2</SPAN><SPAN class="toctext">SDS-PAGE and Western Blot</SPAN></A></LI><LI class="toclevel-3 tocsection-13"><A href="#Purification_2"><SPAN class="tocnumber">1.4.3</SPAN><SPAN class="toctext">Purification</SPAN></A></LI><LI class="toclevel-3 tocsection-14"><A href="#Enzyme_Assays_2"><SPAN class="tocnumber">1.4.4</SPAN><SPAN class="toctext">Enzyme Assays</SPAN></A></LI><LI class="toclevel-3 tocsection-15"><A href="#Growth_Assay_2"><SPAN class="tocnumber">1.4.5</SPAN><SPAN class="toctext">Growth Assay</SPAN></A></LI><LI class="toclevel-4 tocsection-16"><A href="#M9_Minimal_Medium_2"><SPAN class="tocnumber">1.4.5.1</SPAN><SPAN class="toctext">M9 Minimal Medium</SPAN></A></LI><LI class="toclevel-2 tocsection-17"><A href="#Outlook"><SPAN class="tocnumber">1.5</SPAN><SPAN class="toctext">Outlook</SPAN></A></LI><LI class="toclevel-2 tocsection-18"><A href="#Team"><SPAN class="tocnumber">1.6</SPAN><SPAN class="toctext">Team</SPAN></A></LI><LI class="toclevel-2 tocsection-19"><A href="#References"><SPAN class="tocnumber">1.7</SPAN><SPAN class="toctext">References</SPAN></A></LI></DIV><H2><SPAN class="mw-headline" id="Caprolactone_Production_in_E..C2.A0coli">Caprolactone Production in <I>E. coli </I></SPAN></H2><DIV class="abstract"><H3><SPAN class="mw-headline" id="Abstract">Abstract</SPAN></H3><P>Our aim was to produce ε-caprolactone in <I>E. coli</I>. ε-caprolactone can be used in Poly(lactide-co-glycolide-co-caprolactone) abbreviated PLGC to tailor degradation time by enhancing diffusion rates. ε-caprolactone is non-toxic and can be mixed with different polymers to form materials with the desired set of properties, making it a prime candidate for future medical applications. In order to produce ε-caprolactone, the cyclohexanone monooxygenase (<I>chmo</I>) was introduced into <I>E. coli</I>. CHMO utilizes NADPH as a co-factor to produce ε-caprolactone from cyclohexanone. To replenish the NADPH used up in this process, as well as oxidize cyclohexanol to cyclohexanone, the alcohol dehydrogenase (<I>adh</I>) was as well introduced into <I>E. coli</I>. In the end we successfully characterized our enzymes and verified their function through a NADPH-dependent assay. We therefore created a pair of BioBricks allowing the community to produce ε-caprolactone, and submitted them to the iGEM Registry  (<A rel="nofollow" class="external text" href="http://parts.igem.org/Part:BBa_K2770010">CHMO</A> and <A rel="nofollow" class="external text" href="http://parts.igem.org/Part:BBa_K2770011">ADH</A>). 
</P></DIV><H3><SPAN class="mw-headline" id="Introduction">Introduction</SPAN></H3><P>ε-caprolactone is a seven-membered circular lactone. It is used as a building block for several polymers, like in our case PLGC. The ε-caprolactone decreases the degradation time of a polymer by making ester bonds easier accessible for water molecules<SUP id="cite_ref-1" class="reference"><A href="#cite_note-1">[1]</A></SUP>. Therefore, polymers with caprolactones have a shorter lifetime than polymers without them. Right now, ε-caprolactone is derived from petrochemicals and further chemicals, which are harmful for living beings and their environment. In order to produce ε-caprolactone in <I>E. coli</I>, we used the two enzymes, cyclohexanone monooxygenase (CHMO) and alcohol dehydrogenase (ADH). 
</P><DIV class="caption"><P><B>Figure 1:</B> Reaction sequence of ADH and CHMO.
</P></DIV><P>The most important enzyme for the production of ε-caprolactone in <I>E. coli</I> is cyclohexanone monooxygenase (CHMO). The variant used in this project is derived from <I>Acinetobacter calcoaceticus</I> and has a molecular weight of 60.9 kDa. CHMO is a Baeyer-Villiger monoxygenase, which catalyses the direct conversion from a ketone into an ester. In our case, CHMO catalyzes the conversion from cyclohexanone into ε-caprolactone. The enzyme requires oxygen as well as the co-factor NADPH to catalyze this reaction. Since the wildtype of the enzyme is unstable and therefore consequently unsuitable for large scale application, a double mutant of CHMO has been used in this project (C376L/M400I)<SUP id="cite_ref-Hauptpaper_Capris_2-0" class="reference"><A href="#cite_note-Hauptpaper_Capris-2">[2]</A></SUP>. These mutations lead to an higher tolerance against oxidation, and consequently lead to an enhanced stability and functionality of CHMO. 
<VIDEO width="830" height="340" autoplay="autoplay" loop=""><SOURCE src="https://static.igem.org/mediawiki/2018/6/6b/T--TU_Darmstadt--CHMO.mp4" type="video/mp4">
   Your browser does not support the video tag.
</SOURCE></VIDEO></P><DIV class="caption"><P><B>Figure 2:</B> Structure of the NADPH-dependent cyclohexanone monooxygenase with a molecular weight of 60.9 kDa..
</P></DIV><P>The other enzyme we used is alcohol dehydrogenase (ADH) from <I>Lactobacillus kefir</I> which has a molecular weight of 26.8 kDa. ADH has two functions. First, it converts cyclohexanol into cyclohexanone and secondly, it recycles the cofactor NADP<SUP>+</SUP> to NADPH, enabling further CHMO activity. Hence, the combination of these two enzymes makes the process self-sustaining. 
<VIDEO width="830" height="340" autoplay="autoplay" loop=""><SOURCE src="https://static.igem.org/mediawiki/2018/a/ab/T--TU_Darmstadt--ADH.mp4" type="video/mp4">
   Your browser does not support the video tag.
</SOURCE></VIDEO></P><DIV class="caption"><P><B>Figure 3:</B> Structure of the NADP<SUP>+</SUP>-dependent alcohol dehydrogenase with a molecular weight of 26.8 kDa.
</P></DIV><P>
In order to achieve a maximal output the enzymes should be used in a 1:10 ratio (ADH:CHMO)<SUP id="cite_ref-Hauptpaper_Capris_2-1" class="reference"><A href="#cite_note-Hauptpaper_Capris-2">[2]</A></SUP>.
</P><H3><SPAN class="mw-headline" id="Methods">Methods</SPAN></H3><P>The following methods have been applied in our lab work: 
</P><DIV class="caption"><P><B>Figure 4:</B> Schematic representation of work flow. 
</P></DIV><H4><SPAN class="mw-headline" id="Cloning">Cloning</SPAN></H4><P>The sequences for <I>adh</I> and <I>chmo</I>, with the two named mutations, were modified with a His-tag, ordered from Integrated DNA Technologies (IDT) and inserted into the pSB1C3 plasmid. For this purpose, the BioBrick assembly (BBa) was used. A T7 lac promotor (BBa_K921000) and a B0034-based ribosomal binding site (BBa_K2380024) were inserted upstream of the coding sequence via the BBa as well. <I>E. coli</I> TOP10 were transformed with generated plasmids and positive colonies were identified via colony PCR and DNA sequencing.
</P> <DIV class="caption"><P><B>Figure 5 (left):</B> pSB1C3 plasmid including expression cassette of <I>adh</I>.
</P><P><B>Figure 6 (right):</B> pSB1C3 plasmid including expression cassette of <I>chmo</I>.
</P></DIV><H4><SPAN class="mw-headline" id="SDS-PAGE_and_Western_Blot">SDS-PAGE and Western Blot</SPAN></H4><P>To verify that <I>ADH</I> and <I>CHMO</I> were heterologously produced, a SDS-PAGE and western blot were performed. The resulting bands were compared to the molecular weight of ADH and CHMO.
</P><H4><SPAN class="mw-headline" id="Purification">Purification</SPAN></H4><P>After expression of <I>adh</I> and <I>chmo</I> in <I>E. coli</I> BL21 by induction of the T7 lac promotor with IPTG, an ÄKTA chromatography system (GE Healthcare, Illinois, USA) was used to purify the desired His-tagged enzymes. 
</P><H4><SPAN class="mw-headline" id="Enzyme_Assays">Enzyme Assays</SPAN></H4><P>Enzyme activity was measured by following the formation or depletion of NADPH at 340 nm. When CHMO is active NADPH is being depleted while when ADH is active NADP<SUP>+</SUP> gets converted into NADPH. For testing the CHMO and ADH, thioanisole or cyclohexanole was used as a substrate, respectively<SUP id="cite_ref-capri_assay_3-0" class="reference"><A href="#cite_note-capri_assay-3">[3]</A></SUP>. The changes in absorption over time were used to calculate the specific enzyme activity utilizing a calibration curve of NADPH. When testing the enzymes together equal amounts of each enzyme were used and cyclohexanol was added as a substrate.
</P><H4><SPAN class="mw-headline" id="Growth_Assay">Growth Assay</SPAN></H4><P>The aim of the growth assay is to find out in what way the cloned plasmids and the enzyme production affects the growth efficiency of our expression stain <I>E. coli</I> BL21. The tests were performed in a 24 well plade. The OD<SUB>600</SUB> was measured every 10 minutes for a period of 20 hours at 37 °C. 
</P><H5><SPAN class="mw-headline" id="M9_Minimal_Medium">M9 Minimal Medium</SPAN></H5><P>The growth of <I>E. coli</I> BL21 was studied in M9 minimal media with and without a supplemented carbon source. M9 minimal medium consists of numerous salts to achieve an isotonic environment, but no carbon source. Hence, bacterial growth in M9 minimal media is inhibited. However, by supplementing either glucose, cyclohexanon or ε-caprolactone in M9 media, the potential metabolization of these compounds of interest as a carbon source can be investigated. Therefore, the OD<SUB>600</SUB> of <I>E. coli</I> cultures with the different carbon sources of interest were measured for 14 hours at 37 °C using a Tecan Spark. 
</P><H3><SPAN class="mw-headline" id="Results_and_Discussion">Results and Discussion</SPAN></H3><H4><SPAN class="mw-headline" id="Cloning_2">Cloning</SPAN></H4><P>After ligating <I>adh</I> into the pSB1C3 vector through BioBrick assembly (BBa)<SUP id="cite_ref-biobrick_4-0" class="reference"><A href="#cite_note-biobrick-4">[4]</A></SUP>, a cPCR was performed using VF2 and VR primers. The PCR products (1197 bp and 1996 bp) were successfully detected after running the samples on an agarose gel. The ligation of <I>adh</I> (756 bp) and <I>chmo</I> (1629 bp) into pSB1C3 was also verified via DNA sequencing.  
</P><H4><SPAN class="mw-headline" id="SDS-PAGE_and_Western_Blot_2">SDS-PAGE and Western Blot</SPAN></H4><P>The enzymes ADH and CHMO could not be detected using SDS-PAGE or western analysis. A  possible reason for this is that the translation was somehow hindered, or the enzymes were degraded before a measurement could be done. Therefore, a different construct was used to determine whether there was a production of both enzymes. <I>E. coli</I> BL21  were transformed with the plasmid pRSFDuet (provided to us by working groupe Prof. Dr. Bornscheuer) encoding the genes of N-terminally His-tagged <I>adh</I> and <I>chmo</I> under the control of a T7 lac promoter. Positive colonies were identified through white-red selection, cultured, and protein expression was induced by addition of IPTG. Samples were taken after 16 hours and separated via SDS-PAGE. We demonstrated successful protein production via western blot (Figure 7). For this, we used primary Rabbit anti His-tag antibodies and secondary Mouse anti rabbit antibodies conjugated to a horse radish peroxidase (HRP). 
</P><DIV class="caption"><P><B>Figure 7:</B> Western blot. Lane 1: Protein prestained Ladder (PageRuler™ Prestained Protein Ladder, 10 to 180 kDa, Thermo Fisher Scientific). Lane 2: Negative control. Lane 3: Overexpressed genes <I>adh</I> and <I>chmo</I> in pRSFDuet in <I>E. coli</I> BL21. 
</P></DIV><P>The western blot shows two bands at approximately 60 and 30 kDa, which meet the expected molecular mass of CHMO (62.5 kDa) and ADH (28.5 kDa). There are two conceivable reasons for the different intensities of the bands. First, the ADH possesses an S-Tag, which has a lower affinity towards the anti His-tag antibodies than His-tags. The main reason, however, is the different level of expression for the <I>adh</I> and <I>chmo</I>. The optimal ratio for ε-caprolactone production of the two enzymes is 1:10 (ADH:CHMO), which is why the promotors are adjusted to this ratio by design<SUP id="cite_ref-geile_capris_5-0" class="reference"><A href="#cite_note-geile_capris-5">[5]</A></SUP>. Thus, the different strengths of the bands have fulfilled our expectations.
</P><H4><SPAN class="mw-headline" id="Purification_2">Purification</SPAN></H4><P>After a successful western analysis of the enzymes on vector pRSFDuet, they were purified using FPLC (Äkta Purifier). There were 3 distinguished peaks detectable in the chromatogram (Figure 8).
</P><DIV class="caption"><P><B>Figure 8:</B> Chromatogram of the ÄKTA purification of ADH and CHMO.
</P></DIV><P>The wash fraction 3 and fractions 13-22 were collected in tubes. ADH was expected to be highly concentrated in fractions 13-16 and CHMO in 17-22. Fraction 3, 13, 14, 17, 19, 20, 21 were analyzed via SDS-PAGE (Figure 9). Also, a sample of the pellet after the cell disruption and a sample of crude protein extract before ÄKTA purification was applied.
</P><DIV class="caption"><P><B>Figure 9:</B> SDS-PAGE. Lanes, from left to right: PageRuler Prestained Protein Ladder (10 to 180 kDa), Pellet after disruption, supernatant before ÄKTA purification, fraction 3 (wash fraction), fraction 13 (ADH), fraction 14 (ADH), fraction 17 (CHMO), fraction 19 (CHMO), fraction 20 (CHMO), fraction 21 (CHMO).
</P></DIV><P>The SDS-PAGE shows, that there is a high concentration of CHMO in the chosen samples, but no ADH in the expected fractions. ADH could, however, be collected in the wash fraction. Fraction 3 (wash fraction) was used for later ADH assays and fraction 19 for CHMO assays.
</P><H4><SPAN class="mw-headline" id="Enzyme_Assays_2">Enzyme Assays</SPAN></H4><P>Enzyme activity was measured by following the formation or depletion of NADPH at 340 nm. After mixing either 0.2 or 1 µM of the enzyme with the substrate of choice in the reaction buffer, 300 µM NADP<SUP>+</SUP> or NADPH was added, mixed in a well, and the extinction was measured over time (Tecan Infinite M200 PRO). 10 mM cyclohexanol was used as a substrate for ADH activity measurements and 0.5 mM thioanisole was used for CHMO. The changes in absorption over time were used to calculate the specific enzyme activity utilizing a calibration curve of NADPH.
</P><DIV class="caption"><P><B>Figure 10:</B> NADPH dependent enzyme assay showing the degradation and formation of NADPH using enzymes CHMO and ADH over 30 minutes.
</P></DIV><P>The depletion of NADPH using CHMO and its formation by ADH was observed, individually (Figure 10) 200 nM and 1 µM of the respective enzyme were used for the assays. It is shown, that higher enzyme concentrations result in faster formation/degradation of NADPH. An assay with NADPH/NADP<SUP>+</SUP> without added enzyme was used as a negative/positive control. These controls’ absorption didn’t change over time, indicating an activity of our enzymes.
The mean specific enzyme activity (n (product in µmol))/(t (in min)*m (enzyme in mg)) of CHMO and ADH was determined to be 0.17 and 0.22.
The assays successfully proofed the activity of our tested enzymes. Future experiments could determine optimal temperature and pH environments for the highest activity.
The significance of these assays was determined using the Student’s t-test for 0 min, 5 min, 10 min, 20 min and 30 min. (Figure 11 and 12)
</P><DIV class="caption"><P><B>Figure 11 (left):</B> Students' T-test of the NADPH dependent CHMO assay.
</P><P><B>Figure 12 (right):</B> Students' T-test of the NADPH dependent ADH assay.
</P></DIV><P>
The assays were proven to be significant with a p-value below 0.05, except for 0 min. This could be due to a suboptimal mixing of the components in the reaction-well after this short time.
Next, an assay with both enzymes in one reaction was done, to see, whether an equilibrium of formation and degradation of NADPH could be observed over time (Figure 9).
</P><DIV class="caption"><P><B>Figure 13:</B> NADPH dependent assay with ADH and CHMO in one reaction.
</P></DIV><P>For this assay 1 µM of both CHMO and ADH was used, as well as cyclohexanol. As expected, it can be observed, that after a short period of degradation or formation, the NADPH levels are normalized and there is almost no rise of drop of NADPH levels over time, because ADH produces NADPH, and CHMO uses it as a substrate.
</P><H4><SPAN class="mw-headline" id="Growth_Assay_2">Growth Assay</SPAN></H4><P>The aim of the growth assay is to find out in what way the cloned plasmids affect the growth efficiency of our expression strain <I>E. coli</I> BL21. For this reasons, different colonies with different vectors cloned inside where tested. <I>E. coli</I> BL21 untransformed was used as a positive control. The gene <I>adh</I> in <I>E. coli</I> BL21 was tested both induced as not induced. The same was done with <I>chmo</I> in <I>E. coli</I> BL21. <I>E. coli</I> BL21 transformed with a vector with both <I>adh</I> and <I>chmo</I> was tested induced and not induced as well. 
</P><DIV class="caption"><P><B>Figure 14:</B> Growth assay with <I>E. coli</I> BL21 with different vectors and induced or not induced.
</P></DIV><P>In figure 14 the results of the growth assay are shown. It can be seen, that the <I>E. coli</I> BL21 colony with the induced <I>adh</I> plasmids grows better than the other colonies including the positive control <I>E. coli</I> BL21 without an added plasmid. It can furthermore be seen that the induced <I>chmo</I> strain and the positive control reach the lowest plateau with an OD<SUB>600</SUB> of 0.5 while the <I>E. coli</I> BL21 with <I>adh</I> induced grows until an OD<SUB>600</SUB> of 0.7.
We expected the positive control without an added plasmid and without inducing to grow best since it does not have a metabolic burden added. It was also expected, that the induced <I>E. coli</I> BL21 strain with <I>chmo</I> and <I>adh</I> added has the lowest growth rate due to the large added plasmid and to the two enzymes which take effort to be produced additionally. In the experiment this colony shows an average growth with a final OD<SUB>600</SUB> of 0.55. Possible source of error might be that a mistake occoured when inoculating the LB-media in the well with <I>E. coli</I> BL21.
As a conclusion it can still be said that the added plasmid and the protein production do not seem to affect the growth of <I>E. coli</I> BL21 severely. 
This means for us that our modified<I>E. coli</I> BL21 is still able to grow properly and also produce ε - caprolactone.
</P><H5><SPAN class="mw-headline" id="M9_Minimal_Medium_2">M9 Minimal Medium</SPAN></H5><P>The goal of the M9 growth assay was, to find out whether our expression strain <I>E. coli</I> BL21 was able to use the produced product and intermediate as a carbon source. In our case, the substrate of the production was cyclohexanol, which is converted into the intermediate product cyclohexanone by ADH, and then into ε-caprolactone by CHMO. It is already known, that cyclohexanol decreases the growth rate of <I>E. coli</I> BL21<SUP id="cite_ref-6" class="reference"><A href="#cite_note-6">[6]</A></SUP>. For this reason, cyclohexanol has not been tested as a possible carbon source for <I>E. coli</I> BL21. 
</P><DIV class="caption"><P><B>Figure 15:</B> M9 growth assay of <I>E. coli</I> BL21 on M9-minimal media with either glucose, cyclohexanone or caprolactone added as a carbon source.
</P></DIV><P>In figure 15, the absorbance at 600 nm (OD<SUB>600</SUB>) was plotted against time in hours. It can be seen, that the OD<SUB>600</SUB> of both the media with caprolactone and with cyclohexanone, respectively, only slightly change and do not increase. At the same time, the OD<SUB>600</SUB> of the minimal medium with glucose increases. 
The OD<SUB>600</SUB> correlates with the number of cells in the media. As significant cell growth can only be observed in glucose-containing media, we concluded that cyclohexanone and ε-caprolactone are no suitable carbon sources for <I>E. coli</I> BL21. 
With this knowledge it can be said, that our products and intermediates are not metabolized by <I>E. coli</I> BL21, which makes it a suitable organism for the production of &amp;epsilon-caprolactone in this regard.
</P><H3><SPAN class="mw-headline" id="Outlook">Outlook</SPAN></H3><P>There are several things we did in our project to enhance the activity of the cyclohexanone monooxygenase (CHMO), which is catalyzing the ε-caprolactone-forming step. First of all, we used the double mutant C376L/M400I of CHMO. This mutation leads to a higher tolerance against molecular oxygen as well as an enhanced stability, which leads to a longer period of ε-caprolactone production. Combining CHMO with the alcohol dehydrogenase (ADH) is another way how we want to upscale the production of ε-caprolactone. CHMO is NADPH-dependent and since NADPH is an expensive chemical, its use in industry processes is economically unfeasible. The function of ADH is to regenerate the NADPH. By adding the enzyme to the organism, it makes the process self-sustaining. The highest product concentration can be reached when using ADH and CHMO in a ratio of 1:10<SUP id="cite_ref-Hauptpaper_Capris_2-2" class="reference"><A href="#cite_note-Hauptpaper_Capris-2">[2]</A></SUP>.
There are a lot more things which can be done to enhance the product yields from CHMO, which we were unfortunately unable to do because of limited time.
One problem of the CHMO is product inhibition. This problem can be faced by additionally adding CAL-A lipase from <I>Candida Antarctica</I>. CAL-A lipase performs an enzymatic ring opening polymerisation of the ε-caprolactone thus creating ε-caprolactone oligomers and can thereby avoid the inhibition which leads to higher yields<SUP id="cite_ref-Hauptpaper_Capris_2-3" class="reference"><A href="#cite_note-Hauptpaper_Capris-2">[2]</A></SUP>. Cyclohexanol is the substrate added to enable the reaction to take place.
The effectiveness of CHMO can be improved further by adding bacterial haemoglobin from <I>Vitreoscilla stercoraria</I> (Lee et al. 2014). CHMO does not only require NADPH to catalyze the conversion from cyclohexanone into ε-caprolactone but does also need oxygen in order to work efficiently. The haemoglobin is supplying the enzyme with oxygen which leads to a higher yield.
</P><P>CHMO has the potential to produce high amounts of ε-caprolactone. Especially when the process gets further optimisied, it is worth considering it as an alternative to the chemical production of ε-caprolactone.
</P><H3><SPAN class="mw-headline" id="Team">Team</SPAN></H3><DIV class="caption"><P><B>Group pictures of the &quot;Capri&quot; team</B>Left picture: Jana Anton and Sergei Senger  Right picture: Sergei Senger, Leon Schröder, Levin Hafa and Jana Anton
</P></DIV><H3><SPAN class="mw-headline" id="References">References</SPAN></H3><OL class="references"><LI id="cite_note-1"><SPAN class="mw-cite-backlink"><A href="#cite_ref-1">↑</A></SPAN><SPAN class="reference-text">Diederik J. Opperman,The Royal Society of Chemistry 2015, 2015.</SPAN></LI><LI id="cite_note-Hauptpaper_Capris-2"><SPAN class="mw-cite-backlink">↑ <SUP><A href="#cite_ref-Hauptpaper_Capris_2-0">2.0</A></SUP><SUP><A href="#cite_ref-Hauptpaper_Capris_2-1">2.1</A></SUP><SUP><A href="#cite_ref-Hauptpaper_Capris_2-2">2.2</A></SUP><SUP><A href="#cite_ref-Hauptpaper_Capris_2-3">2.3</A></SUP></SPAN><SPAN class="reference-text"> Sandy Schmidt, Angewandte Chemie, Wiley Online Library, 2015.</SPAN></LI><LI id="cite_note-capri_assay-3"><SPAN class="mw-cite-backlink"><A href="#cite_ref-capri_assay_3-0">↑</A></SPAN><SPAN class="reference-text"> Friso S. Aalbers, Appl Microbiol Biotechnol, 2017. </SPAN></LI><LI id="cite_note-biobrick-4"><SPAN class="mw-cite-backlink"><A href="#cite_ref-biobrick_4-0">↑</A></SPAN><SPAN class="reference-text"> Reshma P Shetty Journal of Biological Engineering, 2008. </SPAN></LI><LI id="cite_note-geile_capris-5"><SPAN class="mw-cite-backlink"><A href="#cite_ref-geile_capris_5-0">↑</A></SPAN><SPAN class="reference-text"> Anna Kohl, Elsevier, 2018. </SPAN></LI><LI id="cite_note-6"><SPAN class="mw-cite-backlink"><A href="#cite_ref-6">↑</A></SPAN><SPAN class="reference-text"> Na-Rae Lee, Biotechnology and Bioprocess Engineering, November 2015, Volume 20, Issue 6, pp 1088–1098 . </SPAN></LI></OL></DIV></DIV></DIV></DIV></DIV></DIV></DIV></BODY></HTML>