<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_TU_Darmstadt_Project_Glycolic_acid_E_coli skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><DIV id="top_title"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:TU Darmstadt/Project/Glycolic acid/E coli</SPAN></H1></DIV><DIV id="HQ_page"><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><P class="title">combimer</P><UL><LI><A href="https://2018.igem.org/Team:TU_Darmstadt">Home</A></LI><LI class="multiple"><A>People</A></LI><LI><A href="https://2018.igem.org/Team:TU_Darmstadt/Team">Team</A></LI><LI><A href="https://2018.igem.org/Team:TU_Darmstadt/Collaborations">Collaborations</A></LI><LI><A href="https://2018.igem.org/Team:TU_Darmstadt/Sponsoring">Sponsoring</A></LI><LI><A href="https://2018.igem.org/Team:TU_Darmstadt/Attributions">Attributions</A></LI></UL><LI class="multiple"><A>Project</A></LI><LI><A href="https://2018.igem.org/Team:TU_Darmstadt/Project/Background">Background</A></LI><LI><A href="https://2018.igem.org/Team:TU_Darmstadt/Project/Monomers">Monomers</A></LI><LI class="sub"><A href="https://2018.igem.org/Team:TU_Darmstadt/Project/Glycolic_acid/E_coli">Glycolic Acid 
                 Production in <I>E. coli</I></A></LI><LI class="sub"><A href="https://2018.igem.org/Team:TU_Darmstadt/Project/Glycolic_acid/S_cerevisiae">Glycolic Acid Production in 
                <I>S. cerevisiae</I></A></LI><LI class="sub"><A href="https://2018.igem.org/Team:TU_Darmstadt/Project/Caprolactone">Caprolactone Production in <I>E. coli</I></A></LI><LI><A href="https://2018.igem.org/Team:TU_Darmstadt/Project/Polymers">Polymers</A></LI><LI class="sub"><A href="https://2018.igem.org/Team:TU_Darmstadt/Project/Analysis">Analytical Background</A></LI><LI class="sub"><A href="https://2018.igem.org/Team:TU_Darmstadt/Project/PLGA">PLGA</A></LI><LI class="sub"><A href="https://2018.igem.org/Team:TU_Darmstadt/Project/PLGC">PLGC</A></LI><LI class="sub"><A href="https://2018.igem.org/Team:TU_Darmstadt/Project/Applications">Applications</A></LI><LI><A href="https://2018.igem.org/Team:TU_Darmstadt/Project/Upscaling">Upscaling</A></LI><LI><A href="https://2018.igem.org/Team:TU_Darmstadt/Notebook">Notebook</A></LI><LI class="multiple"><A>Achievements</A></LI><LI><A href="https://2018.igem.org/Team:TU_Darmstadt/InterLab">InterLab</A></LI><LI><A href="https://2018.igem.org/Team:TU_Darmstadt/Composite_Part">Parts</A></LI><LI><A href="https://2018.igem.org/Team:TU_Darmstadt/Demonstrate">Proof of Concept</A></LI><LI><A href="https://2018.igem.org/Team:TU_Darmstadt/Applied_Design">Applied Design</A></LI><LI><A href="https://2018.igem.org/Team:TU_Darmstadt/Model">Modeling</A></LI><LI class="multiple"><A>Human Practices</A></LI><LI><A href="https://2018.igem.org/Team:TU_Darmstadt/Human_Practices">Human Practices</A></LI><LI><A href="https://2018.igem.org/Team:TU_Darmstadt/Public_Engagement">Education &amp; Engagement</A></LI><LI><A href="https://2018.igem.org/Team:TU_Darmstadt/Safety">Safety</A></LI><DIV class="igem_2018_team_content"><DIV class="igem_2018_team_column_wrapper"><DIV id="toc" class="toc"><DIV id="toctitle"><H2>Contents</H2></DIV><UL><LI class="toclevel-1 tocsection-1"><A href="#Glycolic_Acid_Production_in_E..C2.A0coli"><SPAN class="tocnumber">1</SPAN><SPAN class="toctext">Glycolic Acid Production in <I>E. coli</I></SPAN></A></LI><LI class="toclevel-2"><A href="#Abstract"><SPAN class="tocnumber">1.1</SPAN><SPAN class="toctext">Abstract</SPAN></A></LI><LI class="toclevel-2 tocsection-2"><A href="#Introduction"><SPAN class="tocnumber">1.2</SPAN><SPAN class="toctext">Introduction</SPAN></A></LI><LI class="toclevel-2 tocsection-3"><A href="#Methods"><SPAN class="tocnumber">1.3</SPAN><SPAN class="toctext">Methods</SPAN></A></LI><LI class="toclevel-3 tocsection-4"><A href="#Cloning"><SPAN class="tocnumber">1.3.1</SPAN><SPAN class="toctext">Cloning</SPAN></A></LI><LI class="toclevel-3 tocsection-5"><A href="#SDS-PAGE_and_Western_Blot"><SPAN class="tocnumber">1.3.2</SPAN><SPAN class="toctext">SDS-PAGE and Western Blot</SPAN></A></LI><LI class="toclevel-3 tocsection-6"><A href="#Purification"><SPAN class="tocnumber">1.3.3</SPAN><SPAN class="toctext">Purification</SPAN></A></LI><LI class="toclevel-3 tocsection-7"><A href="#Enzyme_Assays"><SPAN class="tocnumber">1.3.4</SPAN><SPAN class="toctext">Enzyme Assays</SPAN></A></LI><LI class="toclevel-3 tocsection-8"><A href="#HPLC_Analysis"><SPAN class="tocnumber">1.3.5</SPAN><SPAN class="toctext">HPLC Analysis</SPAN></A></LI><LI class="toclevel-3 tocsection-9"><A href="#M9_Growth_Assay"><SPAN class="tocnumber">1.3.6</SPAN><SPAN class="toctext">M9 Growth Assay</SPAN></A></LI></UL><LI class="toclevel-2 tocsection-10"><A href="#Results_and_Discussion"><SPAN class="tocnumber">1.4</SPAN><SPAN class="toctext">Results and Discussion</SPAN></A></LI><LI class="toclevel-3 tocsection-11"><A href="#Cloning_and_Expression"><SPAN class="tocnumber">1.4.1</SPAN><SPAN class="toctext">Cloning and Expression</SPAN></A></LI><LI class="toclevel-3 tocsection-12"><A href="#AceA_Enzyme_Assay"><SPAN class="tocnumber">1.4.2</SPAN><SPAN class="toctext">AceA Enzyme Assay</SPAN></A></LI><LI class="toclevel-3 tocsection-13"><A href="#YcdW_Enzyme_Assay"><SPAN class="tocnumber">1.4.3</SPAN><SPAN class="toctext">YcdW Enzyme Assay</SPAN></A></LI><LI class="toclevel-3 tocsection-14"><A href="#HPLC_Analysis_2"><SPAN class="tocnumber">1.4.4</SPAN><SPAN class="toctext">HPLC Analysis</SPAN></A></LI><LI class="toclevel-3 tocsection-15"><A href="#Growth_Assay"><SPAN class="tocnumber">1.4.5</SPAN><SPAN class="toctext">Growth Assay</SPAN></A></LI><LI class="toclevel-4 tocsection-16"><A href="#M9_Minimal_Medium"><SPAN class="tocnumber">1.4.5.1</SPAN><SPAN class="toctext">M9 Minimal Medium</SPAN></A></LI><LI class="toclevel-2 tocsection-17"><A href="#Outlook"><SPAN class="tocnumber">1.5</SPAN><SPAN class="toctext">Outlook</SPAN></A></LI><LI class="toclevel-2 tocsection-18"><A href="#Team"><SPAN class="tocnumber">1.6</SPAN><SPAN class="toctext">Team</SPAN></A></LI><LI class="toclevel-2 tocsection-19"><A href="#References"><SPAN class="tocnumber">1.7</SPAN><SPAN class="toctext">References</SPAN></A></LI></DIV><H2><SPAN class="mw-headline" id="Glycolic_Acid_Production_in_E..C2.A0coli">Glycolic Acid Production in <I>E. coli</I></SPAN></H2><DIV class="abstract"><H3><SPAN class="mw-headline" id="Abstract">Abstract</SPAN></H3><P>Our aim was to produce glycolic acid for the production of biodegradable polymers. Hence, we overexpressed the genes <I>aceA</I> (isocitrate lyase) and <I>ycdW</I> (glyoxylate reductase) in <I>Escherichia coli</I> (<I>E. coli</I>) to synthesize glycolic acid from the glyoxylate cycle intermediate isocitrate. In the end, we successfully characterized our enzymes and verified the production of glycolic acid through both, NADPH and phenylhydrazine-dependent assays, as well as HPLC analysis. We therefore created a pair of BioBricks allowing the community to produce glycolic acid, and submitted them to the iGEM Registry 
(<A rel="nofollow" class="external text" href="http://parts.igem.org/Part:BBa_K2770003">BBa_K2770003</A>&amp;<A rel="nofollow" class="external text" href="http://parts.igem.org/Part:BBa_K2770002">BBa_K2770002</A>).
</P></DIV><H3><SPAN class="mw-headline" id="Introduction">Introduction</SPAN></H3><P><I>Escherichia coli</I> is the most frequently used and a well-established host organism for biotechnological applications. Especially its stable expression system of heterologous genes, fast growth rates and its overall well characterized metabolome and proteome make it a currently indispensable organism in biotechnology.
</P><P>In our project, we modified the glyoxylate cycle of <I>E. coli</I> to produce glycolic acid. For this purpose, we overexpressed two native <I>E. coli</I> genes encoding the isocitrate lyase AceA and the glyoxylate reductase YcdW.
</P><P>The isocitrate lyase catalyzes the conversion of isocitrate into glyoxylate and succinate. AceA has a tetrameric structure with a molecular mass of 47.5 kDa per subunit. <SUP id="cite_ref-UNIACEA_1-0" class="reference"><A href="#cite_note-UNIACEA-1">[1]</A></SUP></P><P><VIDEO width="830" height="340" autoplay="autoplay" loop=""><SOURCE src="https://static.igem.org/mediawiki/2018/9/97/T--TU_Darmstadt--AceA.mp4" type="video/mp4">
   Your browser does not support the video tag.
</SOURCE></VIDEO></P><DIV class="caption"><P><B>Figure 1:</B> Tetrameric structure of the isocitrate lyase (AceA) with a molecular mass of 47.5 kDa per subunit.
</P></DIV><P>Subsequently, the glyoxylate produced by the isocitrate lyase AceA is further converted into glycolic acid by the NADPH-dependent glyoxylate reductase YcdW. It has a molecular weight of 35.3 kDa<SUP id="cite_ref-YCDWUNI_2-0" class="reference"><A href="#cite_note-YCDWUNI-2">[2]</A></SUP>.
</P><P><VIDEO width="830" height="340" autoplay="autoplay" loop=""><SOURCE src="https://static.igem.org/mediawiki/2018/a/af/T--TU_Darmstadt--ycdW.mp4" type="video/mp4">
   Your browser does not support the video tag.
</SOURCE></VIDEO></P><DIV class="caption"><P><B>Figure 2:</B> Structure of the NADPH-dependent glyoxylate reductase (YcdW) with a molecular mass of 35.3 kDa.
</P></DIV><P>Through production of both key enzymes, we achieved the synthesis of glycolic acid in <I>E. coli</I>. We assess our accomplishment as a crucial step towards a more environment-friendly production of biodegradable polymers.
</P><DIV class="caption"><P><B>Figure 3:</B> Reaction sequence of AceA and YcdW.
</P></DIV><H3><SPAN class="mw-headline" id="Methods">Methods</SPAN></H3><P>The following methods have been applied in our lab work:
</P><DIV class="caption"><P><B>Figure 4:</B> Schematic representation of work flow.
</P></DIV><H4><SPAN class="mw-headline" id="Cloning">Cloning</SPAN></H4><P>The sequences of <I>aceA</I> and <I>ycdW</I> were modified with a His-tag, ordered from Integrated DNA Technologies (IDT) and inserted into the pSB1C3 plasmid. For this purpose, the BioBrick assembly (BBa) was used. A T7 lac promotor (<A rel="nofollow" class="external text" href="http://parts.igem.org/Part:BBa_K921000">BBa_K921000</A>) and a B0034-based ribosomal binding site (<A rel="nofollow" class="external text" href="http://parts.igem.org/Part:BBa_K2380024">BBa_K2380024</A>) were inserted upstream of the coding sequence via the BBa as well. <I>E. coli</I> TOP10 were transformed with generated plasmids and positive colonies were identified via colony PCR and DNA sequencing.
</P><DIV class="caption"><P><B>Figure 5 (left):</B> pSB1C3 plasmid including expression cassette of <I>aceA</I>.
</P><P><B>Figure 6 (right):</B> pSB1C3 plasmid including expression cassette of <I>ycdW</I>.
</P></DIV><H4><SPAN class="mw-headline" id="SDS-PAGE_and_Western_Blot">SDS-PAGE and Western Blot</SPAN></H4><P>To verify that AceA and YcdW were produced, a SDS-PAGE was performed, followed by a western blot. The resulting bands were compared to the molecular weight of AceA and YcdW.
</P><H4><SPAN class="mw-headline" id="Purification">Purification</SPAN></H4><P>After production of AceA and YcdW in <I>E. coli</I> BL21 by induction of the T7 lac promotor with IPTG, an ÄKTA chromatography system (GE Healthcare, Illinois, USA) was used to purify the desired His-tagged enzymes. 
</P><H4><SPAN class="mw-headline" id="Enzyme_Assays">Enzyme Assays</SPAN></H4><P>The purified enzymes were spectrophotometrically assayed for their activity using a plate reader. For the assay of the isocitrat lyase (AceA), the enzyme activity was tested via a phenylhydrazine-dependent reaction. As soon as AceA converts isocitrate into glyoxylate, the product reacts with phenylhydrazin to glyoxylate-phenylhydrazone, which possesses an absorption maximum at 324 nm. The change in absorption at 324 nm was measured over time. A calibration curve was created to calculate the glyoxylate-phenylhydrazone conversion rate. Therefore, the absorption of different concentrations of glyoxylate and phenylhyrazine was used.<SUP id="cite_ref-ROBERTSON_3-0" class="reference"><A href="#cite_note-ROBERTSON-3">[3]</A></SUP></P><DIV class="caption"><P><B>Figure 7:</B> Reaction mechanism of phenylhydrazine-dependent assay for the verification of AceA enzyme activity.
</P></DIV><P>The assay for the glyoxylate reductase YcdW is based on the different absorption maxima of NADPH (absorption maxima at 260 nm and 340 nm) and NADP<SUP>+</SUP> (maximum at 260 nm). During the reaction the enzyme uses NADPH as a cofactor. NADPH is converted into NADP<SUP>+</SUP> which leads to a decrease of absorption at 340 nm. By measuring the absorption level at 340 nm over time, it is possible to quantify the enzyme activity and glycolic acid production. A calibration curve was created to calculate the NADPH conversion rate. Therefore, the absorption of different NADPH concentrations was measured.<SUP id="cite_ref-NU.C3.91EZ_4-0" class="reference"><A href="#cite_note-NU.C3.91EZ-4">[4]</A></SUP></P><DIV class="caption"><P><B>Figure 8:</B> Reaction mechanism of NADPH-dependent conversion of glyoxylate into glycolic acid performed by YcdW.
</P></DIV><H4><SPAN class="mw-headline" id="HPLC_Analysis">HPLC Analysis</SPAN></H4><P>To detect the produced monomer glycolic acid, as well as the precursors, high-performance liquid chromatography (HPLC) was employed. An organic acid separation column as stationary phase and sulfuric acid as mobile phase allowed the separation of isocitrate, glyoxylate and glycolic acid. Signals were recorded by a refractive index detector.
</P><H4><SPAN class="mw-headline" id="M9_Growth_Assay">M9 Growth Assay</SPAN></H4><P>The growth of <I>E. coli</I> BL21 transformed with an YcdW-generating plasmid was studied in M9 minimal media with and without a supplemented carbon source. M9 minimal medium consists of numerous salts to achieve an isotonic environment, but no carbon source. Hence, bacterial growth in M9 minimal media is inhibited. However, by supplementing either glucose, glyoxylate or glycolic acid in M9 media, the potential metabolization of these compounds of interest as a carbon source can be investigated. Therefore, the OD<SUB>600</SUB> of <I>E. coli</I> cultures with the different carbon sources of interest were measured over a period of four days and compared to a carbon source-deficient culture.
</P><H3><SPAN class="mw-headline" id="Results_and_Discussion">Results and Discussion</SPAN></H3><H4><SPAN class="mw-headline" id="Cloning_and_Expression">Cloning and Expression</SPAN></H4><P>The <I>aceA</I> and the <I>ycdW</I> genes were cloned using <I>E. coli</I> TOP 10. Furthermore, the plasmids were successfully introduced into BL21 cells to enable protein production under the control of an IPTG inducible T7-lac promotor (BBA_K921000). An attached His-tag was used to purify the encoded enzymes via an ÄKTA system. The subsequent SDS-PAGE showed bands of the expected protein sizes. We also proved the successful production and purification of the enzymes via western blot (Figure 9). The expected molecular weights were 47.5 kDa for AceA and 35.3 kDa for YcdW, respective bands could be detected on the western blot.
</P><DIV class="caption"><P><B>Figure 9:</B> Western blot analysis of purified proteins AceA &amp; YcdW. Thermo Scientific PageRuler Prestained Protein Ladder was used. The nitrocellulose membrane was incubated with an anti-His antibody. For protein detection, a mouse anti-rabbit antibody conjugated with a horseradish-peroxidase was used.
</P></DIV><P>As seen in figure 9, the western blot shows the expected signals. This indicates the successful purification and production of AceA and YcdW in <I>E. coli</I>. Hence, we started the protein characterization via <I>in vitro</I> enzyme assays.
</P><H4><SPAN class="mw-headline" id="AceA_Enzyme_Assay">AceA Enzyme Assay</SPAN></H4><P>We verified the enzyme activity of AceA via a phenylhydrazine-dependent assay with the collected protein fractions from the ÄKTA purification. Isocitrate was included as a substrate. By adding phenylhydrazine to the reaction, glyoxylate produced during the assay (reaction time of 30 minutes) will further react to glyoxylate-phenylhydrazone, which absorbs light at a wavelength of 324 nm (see <A rel="nofollow" class="external text" href="https://2018.igem.org/Team:TU_Darmstadt/Project/Glycolic_acid/E_coli#Enzyme_assays">methods</A>). Hence, we were able to indirectly measure glyoxylate production over time. We also included a negative control without enzyme, which is expected to stay on a constant absorption level, because no isocitrate is converted into glyoxylate. We used different enzyme concentrations (0.03 µg/µL; 0.15 µg/µL; 0.3 µg/µL; 0.6 µg/µL) and temperatures (21 °C/25 °C/37 °C) to screen for the best conditions.
</P><P>Referring to Robertson <I>et al.</I><SUP id="cite_ref-ROBERTSON_3-1" class="reference"><A href="#cite_note-ROBERTSON-3">[3]</A></SUP>, we first carried out assays with different enzyme concentrations at 25 °C. Figure 10 shows the calibration curve for the AceA assay, which is used to calculate the substrate turnover per time. To create the calibration curve, the absorption of different glyoxylate-phenylhydrazone concentrations was measured and compared to the resulting assay absorptions (Figure 11). For the AceA enzyme assay it was expected that a higher concentration would result in a faster substrate conversion.
</P><DIV class="caption"><P><B>Figure 10 (left):</B> Calibration curve for AceA enzyme assay with different glyoxylate concentrations (0 mM, 0.125 mM, 0.1875 mM, 0.25 mM, 0.375 mM, 0.5 mM, 0.75 mM, 1 mM, 1.5 mM, 2 mM, 3 mM and 4 mM).
</P><P><B>Figure 11 (right):</B> AceA enzyme assay at 25 °C and 324 nm. The graph shows the average absorption in correlation to the time (min). Reactions with different enzyme concentration were measured, as well as a negative control sample without enzyme.
</P></DIV><P>All used samples reach an absorption (324 nm) of approximately 3.7, but with different substrate conversion rates. Higher absorption values than 3.7 are not detectable by the plate reader. Hence, it is not possible to determine the exact yield. The negative control stays on a constant absorption level, which indicates that the increase in absorption of the enzyme containing samples is based on enzymatic activity. 
As expected, figure 11 shows that a higher enzyme concentration leads to a faster turnover of glyoxylate. The highest concentration of 0.6 µg/µL results in a substrate conversion rate of 0.912 µM/s. Thus, we decided to carry out further analysis regarding the optimal temperature with an enzyme concentration of 0.6 µg/µL.
</P><P>To study the enzyme activity at different temperatures, we next performed the assay at 21 °C and 37 °C. The substrate conversion per time (µM/s) is shown in table 1. To achieve this, the substrate conversion values were recorded in the linear range of the particular curve in a time frame of 3.5 minutes. It should be noted that the determination does not start at the exact same time for all samples, but the time frame of 3.5 minutes was retained. Three replicates were measured for every assay. 
</P><DIV class="caption"><P><B>Table 1:</B> Substrate conversion of AceA per time (µM/s) in correlation to different temperatures (21 °C, 25 °C and 37 °C).
</P></DIV><TABLE class="wikitable"><TBODY><TR><TH> Temperature [°C]
</TH><TH> Substrate conversion
<P>per second [µM/s]
</P></TH></TR><TR><TD> 21
</TD><TD> 0.82
</TD></TR><TR><TD> 25
</TD><TD> 0.912
</TD></TR><TR><TD> 37
</TD><TD> 0.914
</TD></TR></TBODY></TABLE><P>
Table 1 shows that the enzyme performs best at a temperature of 25 °C and 37 °C with a substrate conversion of 0.912 µM/s and 0.914 µM/s. At a temperature of 21 °C the enzyme shows its lowest conversion rate of 0.82 µM/s. 
Referring to Robertson <I>et al.</I><SUP id="cite_ref-ROBERTSON_3-2" class="reference"><A href="#cite_note-ROBERTSON-3">[3]</A></SUP> a temperature of 25 °C was used for further analysis. 
</P><P>A Student's t-test was performed to check the significance of our data. For this test, five data points from the 25 °C assay were chosen (0, 5, 10, 20 and 30 minutes) and plotted in figure 12 for a better visualization.
</P><DIV class="caption"><P><B>Figure 12:</B> AceA assay absorption for 0, 5, 10, 20 and 30 minutes with standard error. n=3
</P></DIV><P>The absorptions, as well as the particular standard errors, are also shown in table 2. Significance is indicated by *.
</P><DIV class="caption"><P><B>Table 2:</B> Calculated significance of AceA data sets for 0, 5, 10, 20 and 30 minutes, 25 °C. The table also shows the absorption as well as the particular standard error. A= Absorption, *= Significance (p-value &lt; 0.05). n= 3.
</P></DIV><TABLE class="wikitable"><TBODY><TR><TH> AceA [µg/µL]
</TH><TH> A (t=0 min)
</TH><TH> A (t=5 min)
</TH><TH> A (t=10 min)
</TH><TH> A (t=20 min)
</TH><TH> A (t=30 min)
</TH></TR><TR><TD>0.03
</TD><TD> 1.046±0.016*
</TD><TD> 1.665±0.018
</TD><TD> 1.32±0.058
</TD><TD> 1.645±0.258
</TD><TD> 1.937±0.249
</TD></TR><TR><TD> 0.15
</TD><TD> 1.071±0.009*
</TD><TD> 1.609±0.1*
</TD><TD> 2.456±0.222*
</TD><TD> 3.639±0.114*
</TD><TD> 3.732±0,003*
</TD></TR><TR><TD> 0.3
</TD><TD> 1.089±0.003
</TD><TD> 1.793±0.04*
</TD><TD> 2.84±0.091*
</TD><TD> 3.19±0.017*
</TD><TD> 3.732±0.012*
</TD></TR><TR><TD> 0.6
</TD><TD> 1.175±0.023
</TD><TD> 2.698±0.225*
</TD><TD> 3.795±0.083*
</TD><TD> 3.909±0.027*
</TD><TD> 3.759±0.02*
</TD></TR><TR><TD> neg.
</TD><TD>1.105±0.005
</TD><TD>1.13+0.007
</TD><TD>1.154+0.007
</TD><TD>1.191±0.006
</TD><TD>1.209±0.006
</TD></TR></TBODY></TABLE><P>
As shown in table 2, most of the data points show a p-value &lt; 0.05, which indicates that the data is significant. Non-significant p-values appear in the data set of the lowest enzyme concentration.
</P><P>After the characterization of the <I>in vitro</I> enzyme activity, the production of glyoxylate was analyzed via high-performance liquid chromatography (HPLC).
</P><H4><SPAN class="mw-headline" id="YcdW_Enzyme_Assay">YcdW Enzyme Assay</SPAN></H4><P>We verified the enzyme activity of YcdW via a NADPH assay with the collected protein fractions from the ÄKTA purification. YcdW oxidizes NADPH to reduce glyoxylate to glycolic acid. NADPH absorbs light at a wavelength of 340 nm, but NADP<SUP>+</SUP> does not. Hence, the absorbance at 340 nm decreases when YcdW is actively turning glyoxylate into glycolic acid. In this experiment we included a positive control, which consisted of NADP<SUP>+</SUP> with enzyme as well as a negative control, including NADPH without enzyme. For both the positive and negative control it was expected, that they stay on a constant absorption level. For the negative control, containing NADPH without the desired enzyme, the absorption should stay constant because no NADPH should be converted into NADP<SUP>+</SUP>. For the positive control, containing NADP<SUP>+</SUP>, the absorption is expected to stay on a constant absorption level, which should be lower than the absorption for NADPH at a wavelength of 340 nm. Different enzyme concentrations (0.1 µg/µL; 0.2 µg/µL; 0.4 µg/µL) and temperatures (21 °C/25 °C/37 °C) were tested to find optimal assay conditions.
</P><P>Referring to Nuñez <I>et al.</I><SUP id="cite_ref-NU.C3.91EZ_4-1" class="reference"><A href="#cite_note-NU.C3.91EZ-4">[4]</A></SUP>, we carried out assays with different enzyme concentrations at 25 °C. Figure 13 shows the calibration curve for the YcdW assay, which is used to calculate the substrate turnover per time. To create the calibration curve, the absorption of different NADPH concentrations was measured and compared to the resulting assay absorptions (Figure 14). For the YcdW assay, it was expected that a higher enzyme concentration would result in a faster substrate conversion. 
</P><DIV class="caption"><P><B>Figure 13 (left):</B> Calibration curve for YcdW enzyme assay with different NADPH concentrations (1 mM, 0.5 mM, 0.25 mM, 0.125 mM and 0 mM).
</P><P><B>Figure 14 (right):</B> NADPH dependent enzyme assay of YcdW at a temperature of 25 °C. The graph shows the average absorption at 340 nm in correlation to the time (min). Different enzyme concentration were used, as well as a postive (NADP<SUP>+</SUP>) and negative control (NADPH without enzyme).
</P></DIV><P>As expected, figure 14 shows that a higher enzyme concentration leads to a faster turnover of glyoxylate. The highest concentration of 0.4 µg/µL reaches an absorption of 0.29 after approximately 15 minutes, which equals a total substrate conversion rate of 0.345 µM/s. Hence, we decided to carry out further analysis regarding the optimal temperature with an enzyme concentration of 0.4 µg/µL. The positive and negative control both stayed on a constant absorption level, which indicates that the absorption decrease of the enzyme containing samples is based on enzymatic activity. However, figure 14 also shows that the enzyme does not reach the level of the positive control, which is presumably due to residual NADPH in the sample. A possible reason could be a too small amount of glyxoylate when starting the assay.
</P><P>To study the enzyme activity at different temperatures we performed the assay at 21 °C and 37 °C. The substrate conversion per time (µM/s) is shown in table 3. Therefore, the substrate conversion values were recorded in the linear range of the particular curve in a time frame of 3.5 minutes. It should be noted that the determination does not start at the exact same time for all samples, but the time frame of 3.5 minutes was retained. Three replicates were measured for every assay.
</P><DIV class="caption"><P><B>Table 3:</B> Substrate conversion of YcdW per time (µM/s) in correlation to different temperature (21 °C, 25 °C and 37 °C).
</P></DIV><TABLE class="wikitable"><TBODY><TR><TH> Temperature [°C]
</TH><TH> Substrate conversion
<P>per second [µM/s]
</P></TH></TR><TR><TD> 21
</TD><TD> 0.31
</TD></TR><TR><TD> 25
</TD><TD> 0.345
</TD></TR><TR><TD> 37
</TD><TD> 0.26
</TD></TR></TBODY></TABLE><P>
Table 3 shows that the enzyme performs best at a temperature of 25 °C with a substrate conversion of 0.345 µM/s, followed by a conversion rate of 0.31 µM/s at 21 °C. At a temperature of 37 °C the enzyme shows its lowest conversion rate of 0.26 µM/s. 
</P><P>After showing that YcdW performs best at a temperature of 25 °C, a Student's t-test was performed to check the significance of our data. Five different data sets (0, 5, 10, 20, 30 min) were chosen. For a better visualization the data sets were plotted (Figure 15).
</P><DIV class="caption"><P><B>Figure 15:</B> YcdW assay absorption for 0, 5, 10, 20 and 30 minutes with standard error. n=3
</P></DIV><P>
The absorptions and corresponding standard errors are also shown in table 4. Significance is indicated by *.
</P><DIV class="caption"><P><B>Table 4:</B> Calculated significance of YcdW data sets for 0, 5, 10, 20 and 30 minutes, 25 °C. The table also shows the NADPH-absorption as well as the particular standard error. A= Absorption, *= Significance (p-value &lt; 0.05), n= 3.
</P></DIV><TABLE class="wikitable"><TBODY><TR><TH> YcdW [µg/µL]
</TH><TH> A (t=0 min)
</TH><TH> A (t=5 min)
</TH><TH> A (t=10 min)
</TH><TH> A (t=20 min)
</TH><TH> A (t=30 min)
</TH></TR><TR><TD> 0.1
</TD><TD> 0.691±0.0003*
</TD><TD> 0.635±0.01*
</TD><TD> 0.58±0.008*
</TD><TD> 0.451±0.004*
</TD><TD> 0.326±0.005*
</TD></TR><TR><TD> 0.2
</TD><TD> 0.693±0.009
</TD><TD> 0.623±0.018*
</TD><TD> 0.563±0.028*
</TD><TD> 0.429±0.056*
</TD><TD> 0.31±0.08*
</TD></TR><TR><TD> 0.4
</TD><TD> 0.868±0.043
</TD><TD> 0.57±0.07
</TD><TD> 0.38±0.064*
</TD><TD> 0.3±0.06*
</TD><TD> 0.293±0.054*
</TD></TR><TR><TD> pos.
</TD><TD> 0.096±0.0008*
</TD><TD> 0.098±0.001*
</TD><TD> 0.099±0.001*
</TD><TD> 0.1±0.001*
</TD><TD> 0.101±0.001*
</TD></TR><TR><TD> neg.
</TD><TD>0.724±0.0017
</TD><TD>0.719±0.002
</TD><TD>0.714±0.002
</TD><TD>0.705±0.003
</TD><TD>0.696±0.003
</TD></TR></TBODY></TABLE><P>
As shown in table 4, most of the data points show a p-value &lt; 0.05, which indicates that the data is significant.
</P><P>The results show clearly that the glyoxylate reductase YcdW performs best at 25 °C and the highest concentration of 0.4 µg/µL, which allows a further characterization of the reaction and production of the desired monomer glycolic acid via high-performance liquid chromatography (HPLC).
</P><H4><SPAN class="mw-headline" id="HPLC_Analysis_2">HPLC Analysis</SPAN></H4><P>In order to verify the production of glycolic acid (product of YcdW) and its precursor glyoxylate (product of AceA) HPLC was employed. Therefore, three <I>in vitro</I> assays were performed as described in section <A rel="nofollow" class="external text" href="https://2018.igem.org/Team:TU_Darmstadt/Project/Glycolic_acid/E_coli#Enzyme_assays">1.3.4</A>. For the YcdW assay, previous HPLC-analysis showed a low conversion rate of gyloxylate into glycolic acid, which indicated a NADPH-shortage. Hence, NADPH was used in a higher concentration. Additionally an <I>in vitro</I> assay, containing both enzymes was carried out in the YcdW reaction buffer to analyze the simultaneous performance of both enzymes, starting from isocitrate as a substrate. In order to analyze the <I>in vivo</I> production of glycolic acid, <I>ycdW</I>-transformed <I>E. coli</I> cells were disrupted and centrifuged. The supernatant was sterile-filtered and analyzed. In total, four samples were analyzed via HPLC:
</P><P>1.	<I>In vitro</I> YcdW assay with NADPH surplus (Figure 16)
</P><P>2.	<I>In vitro</I> AceA assay (Figure 17)
</P><P>3.	<I>In vitro</I> YcdW with AceA in YcdW reaction buffer (Figure 18)
</P><P>4.	Analysis of disrupted <I>ycdW</I>-transformed <I>E. coli</I> (Figure 19)
</P><P>
To compare the resulting retention times from our samples, we used external references, which showed the following retention times:
</P><P>1.	Isocitrate: 10.25 min
</P><P>2.	Glyoxylate: 12.193 min
</P><P>3.	Glycolic acid: 15.749 min
</P><P>
For assay 1 (<I>in vitro</I> YcdW assay) a peak after 15.701 minutes confirms the production of glycolic acid from glyoxylate (12.210 min, Figure 16).
</P><DIV class="caption"><P><B>Figure 16:</B> HPLC analysis of YcdW <I>in vitro</I> enzyme assay with NADPH surplus. Glycolic acid peak can be seen at 15.701 minutes and glyoxylate peak at 12.210 minutes.
</P></DIV><P>
Figure 17 (<I>in vitro</I> AceA assay) indicates the production of glyoxylate with a retention time of 12.195 minutes from isocitrate (10.240 min).
</P><DIV class="caption"><P><B>Figure 17:</B> HPLC analysis of AceA <I>in vitro</I> assay. Production of glyoxylate is indicated by a peak at 12.195 min. The results show a peak at 10.240 minutes, which correlates to isocitrat.
</P></DIV><P>
While testing the parallel performance of YcdW and AceA, a white precipitate was noticed. Since the HPLC (Figure 18) shows no peaks for neither glyoxylate nor glycolic acid, it is likely that AceA precipitated in the YcdW reaction buffer (potassium phosphate), resulting in a loss of catalytic activity.
</P><DIV class="caption"><P><B>Figure 18:</B> HPLC analysis of <I>in vitro</I> assay containing both desired enzymes in the YcdW reaction buffer. No expected retention times can be seen.
</P></DIV><P>
The cell lysate (Figure 19) shows, besides others, peaks corresponding to the retention time of glyoxylate (12.066 min) and glycolic acid (15.568 min). 
It can therefore be concluded that the production of glycolic acid in <I>E. coli</I> was successful.
</P><DIV class="caption"><P><B>Figure 19:</B> HPLC analysis of supernatant of disrupted <I>ycdW</I>-transformed <I>E. coli</I>. Glyoxylate and glycolic acid peaks can be seen at 12.066 and 15.568 minutes.
</P></DIV><P>
The results show, that a simultaneous <I>in vitro</I> performance of both enzymes in the YcdW potassium phosphate buffer, starting from isocitrate as a substrate, does not result in the production of glycolic acid (Figure 19). To switch from a successive <I>in vitro</I> synthesis of glycolic acid to a simultaneous one, it is necessary to find an optimized buffer for maximum performance of both enzymes. The <I>in vitro</I> productions of glycolic acid and glyoxylate as well as the <I>in vivo</I> production of glycolic acid were successful (Figures 16, 17 and 18). 
</P><H4><SPAN class="mw-headline" id="Growth_Assay">Growth Assay</SPAN></H4><P>To find out in which way the cloned plasmids (<I>ycdW</I> x pSB1C3 and <I>aceA</I> x pSB1C3) effect the growth efficiency of our expression strain <I>E. coli</I> BL21, we performed a growth assay. The results showed no significance. However, in other experiments, these cells showed no distinguishable difference in growth compared to non-transformed cells. Hence, we suspect that the overexpression of <I>ycdW</I> and <I>aceA</I> has no great influence on growth. To proof this hypothesis, further growth assays must be carried out.
</P><H5><SPAN class="mw-headline" id="M9_Minimal_Medium">M9 Minimal Medium</SPAN></H5><P>The strain used for protein production (<I>E. coli</I> BL21) was investigated regarding its ability to use the produced monomers as a carbon source. Since this would result in a decrease of the desired product amount, it would be detrimental to this projects goal of providing monomers for polymerisation. 
Since <I>E. coli</I> show a significantly elongated lag phase when grown in M9 minimal medium (Paliy <I>et al.</I><SUP id="cite_ref-PALIY_5-0" class="reference"><A href="#cite_note-PALIY-5">[5]</A></SUP>), the OD<SUB>600</SUB> was measured over the course of 120 hours.
An increase in the OD<SUB>600</SUB> indicating growth of the culture was only observed in media supplemented with glucose. Since no growth was observed neither in glycolic acid nor in glyoxylate containing media, it can be concluded, that no significant degradation of the desired product is taking place.
</P><DIV class="caption"><P><B>Figure 20:</B> Growth assay of <I>ycdW</I>-transformed <I>E. coli</I> in M9 minimal medium. Growth was only observed in media supplemented with glucose. No growth was observe neither in glycolic acid nor in glyoxylate containing media.
</P></DIV><H3><SPAN class="mw-headline" id="Outlook">Outlook</SPAN></H3><P>We aimed for a more sustainable, environment-friendly production of glycolic acid, which can be used as component of the biodegradable polymer PLGA. To achieve this, we genetically engineered <I>E. coli</I> and demonstrated the <I>in vivo</I> and <I>in vitro</I> bioproduction of glycolic acid. Hence, our approach enables the future production of PLGA polymers independently from fossil resources.
</P><P>However, to make this process profitable, production titers have to be increased. Therefore, we suggest the following optimization steps:
Currently, we use a two-plasmid system, harboring the genes <I>ycdW</I> and <I>aceA</I>, to express the biosynthesis pathway in <I>E. coli</I>. To minimize the metabolic burden on <I>E. coli</I>, both enzymes shall be either encoded on one single plasmid or their genes shall be genomically integrated in <I>E. coli</I>. Furthermore, counteracting enzymes, <I>e.g.</I> <I>aceB</I><SUP id="cite_ref-ACEBUNI_6-0" class="reference"><A href="#cite_note-ACEBUNI-6">[6]</A></SUP>, lower the metabolite pool of our pathway and thereby decrease product titers. Hence, we suggest the deletion of such counteracting enzymes.
</P><P>To further increase titers, it is possible to fine-tune the expression ratios of our genes, considering their enzyme activities, co-factor dependencies, titers of glycolic acid and the growth rate of <I>E. coli</I>. Several synthetic metabolic channeling approaches may also increase titers.<SUP id="cite_ref-DUEBER_7-0" class="reference"><A href="#cite_note-DUEBER-7">[7]</A></SUP><SUP id="cite_ref-LEE_8-0" class="reference"><A href="#cite_note-LEE-8">[8]</A></SUP></P><P>For an enhanced <I>in vitro</I> production, we suggest the optimization of the reaction buffer, including its pH, to simultaneously use both key enzymes. We also see space for improvements in fine-tuning substrate and co-factor concentrations. To quantify the performance and output of our two approaches (<I>in vivo</I> and <I>in vitro</I> production), a quantitative analysis of our HPLC runs should be performed next. The first step towards this will be the creation of a calibration curve. 
</P><P>Considering all these aspects, we are optimistic that an optimized glycolic acid production in <I>E. coli</I> has the potential to replace the petrochemical-dependent PLGA production.
</P><H3><SPAN class="mw-headline" id="Team">Team</SPAN></H3><DIV class="caption"><P><B>Group picture of the &quot;E. coli&quot; team</B>From left to right: Elena Nickels, Maria Musillo, Benedict Spannenkrebs, Peter Gockel and Luca Brenker
</P></DIV><H3><SPAN class="mw-headline" id="References">References</SPAN></H3><OL class="references"><LI id="cite_note-UNIACEA-1"><SPAN class="mw-cite-backlink"><A href="#cite_ref-UNIACEA_1-0">↑</A></SPAN><SPAN class="reference-text"> UniProtKB - P0A9G6 (ACEA_ECOLI), 10/14/2018 <A rel="nofollow" class="external autonumber" href="https://www.uniprot.org/uniprot/P0A9G6">[1]</A>.</SPAN></LI><LI id="cite_note-YCDWUNI-2"><SPAN class="mw-cite-backlink"><A href="#cite_ref-YCDWUNI_2-0">↑</A></SPAN><SPAN class="reference-text"> UniProtKB - P75913 (GHRA_ECOLI), 10/14/2018 <A rel="nofollow" class="external autonumber" href="https://www.uniprot.org/uniprot/P75913">[2]</A>.</SPAN></LI><LI id="cite_note-ROBERTSON-3"><SPAN class="mw-cite-backlink">↑ <SUP><A href="#cite_ref-ROBERTSON_3-0">3.0</A></SUP><SUP><A href="#cite_ref-ROBERTSON_3-1">3.1</A></SUP><SUP><A href="#cite_ref-ROBERTSON_3-2">3.2</A></SUP></SPAN><SPAN class="reference-text"> Eugene F. Roberston and Henry C. Reeves, Purification and characterization of isocitrate lyase from <I>Escherichia coli</I>, Current Microbiology Vol.14 (1987), pp. 347-350 <A rel="nofollow" class="external autonumber" href="https://link.springer.com/content/pdf/10.1007%2FBF01568702.pdf">[3]</A>.</SPAN></LI><LI id="cite_note-NU.C3.91EZ-4"><SPAN class="mw-cite-backlink">↑ <SUP><A href="#cite_ref-NU.C3.91EZ_4-0">4.0</A></SUP><SUP><A href="#cite_ref-NU.C3.91EZ_4-1">4.1</A></SUP></SPAN><SPAN class="reference-text"> M. Felisa Nuñez, M. Teresa Pellicer, Josefa Badia, Juan Aguilar and Laura Baldoma, Biochemical characterization of the 2-ketoacid reductases encoded by <I>ycdW</I> and <I>yiaE</I> genes in <I>Escherichia coli</I>, Biochemical Journal, 2001, 354, 707-715 <A rel="nofollow" class="external autonumber" href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC1221703/">[4]</A>.</SPAN></LI><LI id="cite_note-PALIY-5"><SPAN class="mw-cite-backlink"><A href="#cite_ref-PALIY_5-0">↑</A></SPAN><SPAN class="reference-text"> Oleg Paliy and Thusitha S. Gunasekera, Growth of <I>E. coli</I> BL21 in minimal media with different gluconeogenic carbon sources and salt contents, Applied Microbiology Biotechnology, 2007, 73: 1169-1172 <A rel="nofollow" class="external autonumber" href="http://www.wright.edu/~oleg.paliy/Papers/BL21/Paliy_AMB2007.pdf">[5]</A>.</SPAN></LI><LI id="cite_note-ACEBUNI-6"><SPAN class="mw-cite-backlink"><A href="#cite_ref-ACEBUNI_6-0">↑</A></SPAN><SPAN class="reference-text"> UniProtKB - P08997 (MASY_ECOLI), 10/16/2018 <A rel="nofollow" class="external autonumber" href="https://www.uniprot.org/uniprot/P08997">[6]</A>.</SPAN></LI><LI id="cite_note-DUEBER-7"><SPAN class="mw-cite-backlink"><A href="#cite_ref-DUEBER_7-0">↑</A></SPAN><SPAN class="reference-text"> John E. Dueber, Gabriel C. Wu, G Reza Malmirchegini, Tae Seok Moon, Christopher J. Petzold, Adeeti V. Ullal, Kristala L. J. Prather and Jay D. Keasling, Synthetic protein scaffolds provide modular control over metabolic flux, Nature Biotechnology, 27, pages 753–759 (2009) <A rel="nofollow" class="external autonumber" href="https://www.nature.com/articles/nbt.1557">[7]</A>.</SPAN></LI><LI id="cite_note-LEE-8"><SPAN class="mw-cite-backlink"><A href="#cite_ref-LEE_8-0">↑</A></SPAN><SPAN class="reference-text">Matthew J. Lee, Judith Mandell, Lorna Hodgson, Dominic Alibhai, Jordan M. Fletcher, Ian R. Brown, Stefanie Frank, Wei-Feng Xue, Paul Verkade, Derek N. Woolfson and Martin J. Warren, Engineered synthetic scaffolds for organizing proteins within the bacterial cytoplasm, nature chemical biology, 14, pages 142–147 (2018) <A rel="nofollow" class="external autonumber" href="https://www.nature.com/articles/nchembio.2535">[8]</A>.</SPAN></LI></OL></DIV></DIV></DIV></DIV></DIV></DIV></DIV></BODY></HTML>