<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_TU_Darmstadt_Notebook_LJ-Colis skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><DIV id="top_title"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:TU Darmstadt/Notebook/LJ-Colis</SPAN></H1></DIV><DIV id="HQ_page"><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><P class="title">combimer</P><UL><LI><A href="https://2018.igem.org/Team:TU_Darmstadt">Home</A></LI><LI class="multiple"><A>People</A></LI><LI><A href="https://2018.igem.org/Team:TU_Darmstadt/Team">Team</A></LI><LI><A href="https://2018.igem.org/Team:TU_Darmstadt/Collaborations">Collaborations</A></LI><LI><A href="https://2018.igem.org/Team:TU_Darmstadt/Sponsoring">Sponsoring</A></LI><LI><A href="https://2018.igem.org/Team:TU_Darmstadt/Attributions">Attributions</A></LI></UL><LI class="multiple"><A>Project</A></LI><LI><A href="https://2018.igem.org/Team:TU_Darmstadt/Project/Background">Background</A></LI><LI><A href="https://2018.igem.org/Team:TU_Darmstadt/Project/Monomers">Monomers</A></LI><LI class="sub"><A href="https://2018.igem.org/Team:TU_Darmstadt/Project/Glycolic_acid/E_coli">Glycolic Acid 
                 Production in <I>E. coli</I></A></LI><LI class="sub"><A href="https://2018.igem.org/Team:TU_Darmstadt/Project/Glycolic_acid/S_cerevisiae">Glycolic Acid Production in 
                <I>S. cerevisiae</I></A></LI><LI class="sub"><A href="https://2018.igem.org/Team:TU_Darmstadt/Project/Caprolactone">Caprolactone Production in <I>E. coli</I></A></LI><LI><A href="https://2018.igem.org/Team:TU_Darmstadt/Project/Polymers">Polymers</A></LI><LI class="sub"><A href="https://2018.igem.org/Team:TU_Darmstadt/Project/Analysis">Analytical Background</A></LI><LI class="sub"><A href="https://2018.igem.org/Team:TU_Darmstadt/Project/PLGA">PLGA</A></LI><LI class="sub"><A href="https://2018.igem.org/Team:TU_Darmstadt/Project/PLGC">PLGC</A></LI><LI class="sub"><A href="https://2018.igem.org/Team:TU_Darmstadt/Project/Applications">Applications</A></LI><LI><A href="https://2018.igem.org/Team:TU_Darmstadt/Project/Upscaling">Upscaling</A></LI><LI><A href="https://2018.igem.org/Team:TU_Darmstadt/Notebook">Notebook</A></LI><LI class="multiple"><A>Achievements</A></LI><LI><A href="https://2018.igem.org/Team:TU_Darmstadt/InterLab">InterLab</A></LI><LI><A href="https://2018.igem.org/Team:TU_Darmstadt/Composite_Part">Parts</A></LI><LI><A href="https://2018.igem.org/Team:TU_Darmstadt/Demonstrate">Proof of Concept</A></LI><LI><A href="https://2018.igem.org/Team:TU_Darmstadt/Applied_Design">Applied Design</A></LI><LI><A href="https://2018.igem.org/Team:TU_Darmstadt/Model">Modeling</A></LI><LI class="multiple"><A>Human Practices</A></LI><LI><A href="https://2018.igem.org/Team:TU_Darmstadt/Human_Practices">Human Practices</A></LI><LI><A href="https://2018.igem.org/Team:TU_Darmstadt/Public_Engagement">Education &amp; Engagement</A></LI><LI><A href="https://2018.igem.org/Team:TU_Darmstadt/Safety">Safety</A></LI><DIV class="igem_2018_team_content"><DIV class="igem_2018_team_column_wrapper"><H2><SPAN class="mw-headline" id="Production_of_glycolic_acid_in_E._coli">Production of glycolic acid in <I>E. coli</I></SPAN></H2><P><B><U> April </U></B><B> Week 1 (2<SUP>nd</SUP> of April – 8<SUP>th</SUP> of April) </B>
We found out about petrochemicals and were upset...<B> Week 2 (9<SUP>th</SUP> of April – 15<SUP>th</SUP> of April) </B>
Thinking about a solution, we discovered the possibilities of <I> E. coli</I><B> Week 3 (16<SUP>th</SUP> of April – 22<SUP>nd</SUP> of April) </B>
Planned all the theoretical lab work for our part of the project.<B> Week 4 (23<SUP>rd</SUP> of April – 29<SUP>th</SUP> of April) </B>
Preparing the wet lab to get started. 
<B><U> May </U></B><B> Week 5 (30<SUP>th</SUP> of April – 6<SUP>th</SUP> of Mai) </B>
To get our coding gene <I>aceA</I> out of <I>E. coli</I> we designed our own primers (in total eight), which we mixed partially together. Due this we differentiate between seven samples A to G. The samples are as followed:</P><TABLE class="wikitable"><TBODY><TR><TH>Sample
</TH><TH>Primer
</TH></TR><TR><TD>A
</TD><TD>AceA-fw + AceA-rev
</TD></TR><TR><TD>B
</TD><TD>AceA-rev + BB prefix
</TD></TR><TR><TD>C
</TD><TD>AceA-rev + BB suffix
</TD></TR><TR><TD>D
</TD><TD>AceA-rev + GS linker-AceA-fw
</TD></TR><TR><TD>E
</TD><TD>AceA-rev + His-GS linker-fw
</TD></TR><TR><TD>F
</TD><TD>AceA-rev + BB prefix-His tag-fw
</TD></TR><TR><TD>G
</TD><TD>AceA-rev + BB prefix-RBS-His-fw
</TD></TR></TBODY></TABLE><P><B> Week 6 (7<SUP>th</SUP> of May – 13<SUP>th</SUP> of May) </B>
We finally started with labwork. At first we performed a PCR with our samples A, C and D and was followed by a PCR Clean-up of these samples.<B> Week 7 (14<SUP>th</SUP> of May – 20<SUP>th</SUP> of May) </B>
In this week, we performed a restriction digest of sample A and pSB1C3 with EcoRI and PstI. The backbone of pSB1C3 was dephosphorylated. We then ligated the sample A into the plasmid pSB1C3 via the standard T4 ligation protocol. Thereafter a transformation of the ligation product into <I>E. coli</I> TOP 10 was performed.  <B> Week 8 (21<SUP>st</SUP> of May – 27<SUP>th</SUP> of May) </B>
First, we performed a colony PCR (cPCR) on sample A-transformed colonies. However, an agarose gel-electrophoresis showed that that none of the screened colonies was transformed with the correct ligation product. To examine the transformed ligation product closer, we subsequently prepared an overnight culture and extracted and purified plasmids the following day. To check our restriction digest we performed a restriction analyze of the sample A with the restriction enzymes XbaI and SpeI. The gel-electrophoreses showed that a re-ligation of PsB1C3 happened. <B><U> June </U></B><B> Week 9 (28<SUP>th</SUP> of May – 3<SUP>rd</SUP> of June) </B>
A transformation with our sample A in <I>E. coli</I> was at first performed. Colonies were screened via colony PCR. A gel-electrophoresis showed that the cPCR was not successful. We then prepared an overnight culture, followed by a plasmid preparation. To check our restriction digest we performed a restriction analyze of the sample A with EcoRI, PstI, XbaI and SpeI. A gel-electrophoresis showed that only the backbone pSB1C3 was cut properly.<B> Week 10 (4<SUP>th</SUP> of June – 10<SUP>th</SUP> of June) </B>
In this week we performed several PCR with the samples A, B and E with different settings to screen for feasible conditions, until we obtained positive results. The respective samples were purified and PCRs for sample were repeated.<B> Week 11 (11<SUP>th</SUP> of June – 17<SUP>th</SUP> of June) </B>
We started working on our second gene <I>ycdW</I> by performing a PCR this week. A agarose gel electrophoresis showed that this was not successful. For the gene <I>aceA</I> we performed a restriction digest of purified PCR products from last week with EcoRI-Hf and PstI-Hf. We were troubleshooting about the ligation of <I>aceA</I> into pSB1C3 and concluded to try the ligtaion with different insert-backbone ratios (6:1 and 3:1). The ligation product was then transformed in <I>E. coli</I> TOP 10 and positive colonies were screened for via cPCR. Subsequently we performed a DNA-preparation with the positive samples and send it for DNA sequencing.<B> Week 12 (18<SUP>th</SUP> of June – 24<SUP>th</SUP> of June) </B>
For our <I>ycdW</I> construct, we digested the encoding gBock with EcoRI and PstI. The digestion product was ligated into a dephosphorylated pSB1C3 backbone. We transformed <I>E. coli</I> TOP 10 with the ligation product, screened for positive transformants via cPCR and subsequent agarose gel electrophoresis of the PCR products. Positive colonies were inoculated in overnight cultures and respective plasmids were extracted and purified the following day. Sequence integrity was validated via DNA sequencing. 
For <I>aceA</I> we have done a PCR of sample G and E, which was not successful.</P><P><B><U> July </U></B><B> Week 13 (25<SUP>th</SUP> of June – 1<SUP>st</SUP> of July) </B>
This week we have done a few PCRs with the samples D and E from <I>aceA</I>, as well as overnight cultures of pSB1C3-<I>ycdW</I>.<B> Week 14 (2<SUP>nd</SUP> of July – 8<SUP>th</SUP> of July) </B>
After positive sequencing results of pSB1C3-<I>ycdW</I> returned we made glycerol stocks. <B> Week 15 (9<SUP>th</SUP> of July – 15<SUP>th</SUP> of July) </B>
This week we took a break from the lab and learned for our exams. <B> Week 16 (16<SUP>th</SUP> of July – 22<SUP>nd</SUP> of July) </B>
The exams were still ongoing. <B> Week 17 (23<SUP>rd</SUP> of July – 29<SUP>th</SUP> of July) </B>
We returned to the lab to clone <I>aceA</I> from a gBlock into pSB1C3 and transformed it into <I>E. coli</I> TOP 10. We checked it via cPCR and gel-electrophoresis. The positive samples we send in for sequencing. <B><U> August </U></B><B> Week 18 (30<SUP>th</SUP> of July – 5<SUP>th</SUP> of August) </B>
This week, we prepared everything to analyze gene expression of <I>ycdW</I> in <I>E. coli</I> BL21 via SDS-PAGE. Therefore, we transformed pSB1C3-<I>ycdW</I> into <I>E. coli</I> BL21 and inoculate the starter and main culture. <B> Week 19 (6<SUP>th</SUP> of August – 12<SUP>th</SUP> of August) </B>
This week, we verified <I>ycdW</I> via SDS-PAGE.  To see if induction of 1mM IPTG has an impact and to verify our SDS-PAGE again we set on a new preparation for a new SDS-Page. A second SDS-Page was made, which proves the positive results from before. Beside this we repeated the cloning of <I>aceA</I> into pSB1C3 using a gBlock and send the samples for sequencing. <B> Week 20 (13<SUP>th</SUP> of August – 19<SUP>th</SUP> of August) </B>
After positive sequencing results of <I>aceA</I> returned, we prepared glycerol stocks of <I>E. coli</I> TOP10 as well as <I>E. coli</I> BL21. Subsequently, we performed a SDS-PAGE with <I>aceA</I> with and without inducing 0.5mM IPTG. And it proves the activity of our cloned <I>aceA</I>.<B> Week 21 (20<SUP>th</SUP> of August – 26<SUP>th</SUP> of August) </B>
To specifically confirm the functional protein expression from the previous SDS-PAGE with both enzymes, we now performed our first Western Blot, which succeeded. <B><U> September </U></B><B> Week 22 (27<SUP>th</SUP> of August – 2<SUP>nd</SUP> of September) </B>
Due the positive results from the previous SDS-PAGE and western blot, we purified our AceA and YcdW via Äkta pure. Subsequently the enzyme buffer was exchange via Amicon Ultra-15 Centrifugal Filters. <B> Week 23 (3<SUP>rd</SUP> of September – 9<SUP>th</SUP> of September) </B> 
We analyzed our purified proteins AceA and YcdW via SDS-PAGE and western blot. <B> Week 25 (10<SUP>th</SUP> of September – 16<SUP>th</SUP> of September) </B>
This is the week we started with our enzyme assays. Therefore, we buffer exchanged YcdW via PD-10 Desalting Columns. Subsequently we started with the NADPH-dependent assay. This assay was repeated twice to verify our results. To see if <I>E. coli</I> can metabolize glycolic acid, we performed a M9Minimal Media assay. Besides, we repeated the western blot with and without induction of <I>aceA</I> and <I>ycdW</I> with 1mM IPTG. <B> Week 26 (17<SUP>th</SUP> of September – 23<SUP>rd</SUP> of September) </B>
We continued with our enzyme assays. First, we performed the M9 –assay for <I>ycdW</I> about three times. Afterwards we repeated the NADPH assay with an enhanced NADPH concentration. For AceA we conducted phenylhydrazin-dependent assay with three replicates. <B> Week 27 (24<SUP>th</SUP> of September – 30<SUP>th</SUP> of September) </B>
We recorded the calibration curve for the enzyme assays of AceA and YcdW. We then buffer exchanged AceA into a potassium phosphate buffer. <B><U> October </U></B><B> Week 28 (1<SUP>st</SUP> of October – 7<SUP>th</SUP> of October) </B>
We finally purified our last plasmids and checked their sequence integrity via DNA sequencing. We also shipped our plasmids to Boston, to submit them to the iGEM registry. Furthermore, we run an <I>in vivo</I> assay to produce glycolic acid and analyzed production via HPLC. We also recorded growth curves of <I>aceA</I> and <I>ycdW</I> in <I>E. coli</I>.<B> Week 29 (8<SUP>th</SUP> of October – 14<SUP>th</SUP> of October) </B>
We are totally done. <B> Week 30 (15<SUP>th</SUP> of October – 21<SUP>st</SUP> of October) </B>
… and cleaning of the Lab<B> Week 31 (22<SUP>nd</SUP> of October – 28<SUP>th</SUP> of October) </B>
Flying to Boston and attend the Giant Jamboree!
</P><P><A rel="nofollow" class="external text" href="//2018.igem.org/Team:TU_Darmstadt/FUN">♦</A></P></DIV></DIV></DIV></DIV></DIV></DIV></DIV></BODY></HTML>