<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_TU_Darmstadt_Project_Glycolic_acid_S_cerevisiae skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><DIV id="top_title"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:TU Darmstadt/Project/Glycolic acid/S cerevisiae</SPAN></H1></DIV><DIV id="HQ_page"><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><P class="title">combimer</P><UL><LI><A href="https://2018.igem.org/Team:TU_Darmstadt">Home</A></LI><LI class="multiple"><A>People</A></LI><LI><A href="https://2018.igem.org/Team:TU_Darmstadt/Team">Team</A></LI><LI><A href="https://2018.igem.org/Team:TU_Darmstadt/Collaborations">Collaborations</A></LI><LI><A href="https://2018.igem.org/Team:TU_Darmstadt/Sponsoring">Sponsoring</A></LI><LI><A href="https://2018.igem.org/Team:TU_Darmstadt/Attributions">Attributions</A></LI></UL><LI class="multiple"><A>Project</A></LI><LI><A href="https://2018.igem.org/Team:TU_Darmstadt/Project/Background">Background</A></LI><LI><A href="https://2018.igem.org/Team:TU_Darmstadt/Project/Monomers">Monomers</A></LI><LI class="sub"><A href="https://2018.igem.org/Team:TU_Darmstadt/Project/Glycolic_acid/E_coli">Glycolic Acid 
                 Production in <I>E. coli</I></A></LI><LI class="sub"><A href="https://2018.igem.org/Team:TU_Darmstadt/Project/Glycolic_acid/S_cerevisiae">Glycolic Acid Production in 
                <I>S. cerevisiae</I></A></LI><LI class="sub"><A href="https://2018.igem.org/Team:TU_Darmstadt/Project/Caprolactone">Caprolactone Production in <I>E. coli</I></A></LI><LI><A href="https://2018.igem.org/Team:TU_Darmstadt/Project/Polymers">Polymers</A></LI><LI class="sub"><A href="https://2018.igem.org/Team:TU_Darmstadt/Project/Analysis">Analytical Background</A></LI><LI class="sub"><A href="https://2018.igem.org/Team:TU_Darmstadt/Project/PLGA">PLGA</A></LI><LI class="sub"><A href="https://2018.igem.org/Team:TU_Darmstadt/Project/PLGC">PLGC</A></LI><LI class="sub"><A href="https://2018.igem.org/Team:TU_Darmstadt/Project/Applications">Applications</A></LI><LI><A href="https://2018.igem.org/Team:TU_Darmstadt/Project/Upscaling">Upscaling</A></LI><LI><A href="https://2018.igem.org/Team:TU_Darmstadt/Notebook">Notebook</A></LI><LI class="multiple"><A>Achievements</A></LI><LI><A href="https://2018.igem.org/Team:TU_Darmstadt/InterLab">InterLab</A></LI><LI><A href="https://2018.igem.org/Team:TU_Darmstadt/Composite_Part">Parts</A></LI><LI><A href="https://2018.igem.org/Team:TU_Darmstadt/Demonstrate">Proof of Concept</A></LI><LI><A href="https://2018.igem.org/Team:TU_Darmstadt/Applied_Design">Applied Design</A></LI><LI><A href="https://2018.igem.org/Team:TU_Darmstadt/Model">Modeling</A></LI><LI class="multiple"><A>Human Practices</A></LI><LI><A href="https://2018.igem.org/Team:TU_Darmstadt/Human_Practices">Human Practices</A></LI><LI><A href="https://2018.igem.org/Team:TU_Darmstadt/Public_Engagement">Education &amp; Engagement</A></LI><LI><A href="https://2018.igem.org/Team:TU_Darmstadt/Safety">Safety</A></LI><DIV class="igem_2018_team_content"><DIV class="igem_2018_team_column_wrapper"><DIV id="toc" class="toc"><DIV id="toctitle"><H2>Contents</H2></DIV><UL><LI class="toclevel-1 tocsection-1"><A href="#Glycolic_Acid_Production_in_S..C2.A0cerevisiae"><SPAN class="tocnumber">1</SPAN><SPAN class="toctext">Glycolic Acid Production in <I>S. cerevisiae</I></SPAN></A></LI><LI class="toclevel-2"><A href="#Abstract"><SPAN class="tocnumber">1.1</SPAN><SPAN class="toctext">Abstract</SPAN></A></LI><LI class="toclevel-2 tocsection-2"><A href="#Introduction"><SPAN class="tocnumber">1.2</SPAN><SPAN class="toctext">Introduction</SPAN></A></LI><LI class="toclevel-2 tocsection-3"><A href="#Methods"><SPAN class="tocnumber">1.3</SPAN><SPAN class="toctext">Methods</SPAN></A></LI><LI class="toclevel-3 tocsection-4"><A href="#Cloning"><SPAN class="tocnumber">1.3.1</SPAN><SPAN class="toctext">Cloning</SPAN></A></LI><LI class="toclevel-3 tocsection-5"><A href="#Deletion"><SPAN class="tocnumber">1.3.2</SPAN><SPAN class="toctext">Deletion</SPAN></A></LI><LI class="toclevel-3 tocsection-6"><A href="#SDS-PAGE_and_Western_Blot"><SPAN class="tocnumber">1.3.3</SPAN><SPAN class="toctext">SDS-PAGE and Western Blot</SPAN></A></LI><LI class="toclevel-3 tocsection-7"><A href="#Purification"><SPAN class="tocnumber">1.3.4</SPAN><SPAN class="toctext">Purification</SPAN></A></LI><LI class="toclevel-3 tocsection-8"><A href="#Enzyme_Assays"><SPAN class="tocnumber">1.3.5</SPAN><SPAN class="toctext">Enzyme Assays</SPAN></A></LI><LI class="toclevel-3 tocsection-9"><A href="#HPLC_Analysis"><SPAN class="tocnumber">1.3.6</SPAN><SPAN class="toctext">HPLC Analysis</SPAN></A></LI></UL><LI class="toclevel-2 tocsection-10"><A href="#Results_and_Discussion"><SPAN class="tocnumber">1.4</SPAN><SPAN class="toctext">Results and Discussion</SPAN></A></LI><LI class="toclevel-3 tocsection-11"><A href="#Deletions"><SPAN class="tocnumber">1.4.1</SPAN><SPAN class="toctext">Deletions</SPAN></A></LI><LI class="toclevel-3 tocsection-12"><A href="#Cloning_and_Expression"><SPAN class="tocnumber">1.4.2</SPAN><SPAN class="toctext">Cloning and Expression</SPAN></A></LI><LI class="toclevel-3 tocsection-13"><A href="#Enzyme_Assay"><SPAN class="tocnumber">1.4.3</SPAN><SPAN class="toctext">Enzyme Assay</SPAN></A></LI><LI class="toclevel-3 tocsection-14"><A href="#Growth_Assays_and_HPLC_Analysis"><SPAN class="tocnumber">1.4.4</SPAN><SPAN class="toctext">Growth Assays and HPLC Analysis</SPAN></A></LI><LI class="toclevel-2 tocsection-15"><A href="#Outlook"><SPAN class="tocnumber">1.5</SPAN><SPAN class="toctext">Outlook</SPAN></A></LI><LI class="toclevel-2 tocsection-16"><A href="#Team"><SPAN class="tocnumber">1.6</SPAN><SPAN class="toctext">Team</SPAN></A></LI><LI class="toclevel-2 tocsection-17"><A href="#References"><SPAN class="tocnumber">1.7</SPAN><SPAN class="toctext">References</SPAN></A></LI></DIV><H2><SPAN class="mw-headline" id="Glycolic_Acid_Production_in_S..C2.A0cerevisiae">Glycolic Acid Production in <I>S. cerevisiae</I></SPAN></H2><DIV class="abstract"><H3><SPAN class="mw-headline" id="Abstract">Abstract</SPAN></H3><P>For the production of a biodegradable polymer, our goal was to produce glycolic acid as one of the monomers in <I>Saccharomyces cerevisiae</I>. Glycolic acid production from glyoxylate was initiated by the heterologous expression of glyoxylate reductase 1 (<I>AtGLYR1</I>). To increase the yield of glycolic acid, we intervened in the glyoxylate cycle by deleting the malate synthase 1 (<I>MLS1</I>) and the isocitrate dehydrogenase 2 (<I>IDP2</I>)<SUP id="cite_ref-Finnland_1-0" class="reference"><A href="#cite_note-Finnland-1">[1]</A></SUP>. We produced <I>At</I>GLYR1 in <I>S. cerevisiae</I>, used western blots for verification, purified it via <I>Strep</I>-tag affinity chromatography and characterized its activity through an NADPH assay.
</P></DIV><H3><SPAN class="mw-headline" id="Introduction">Introduction</SPAN></H3><P>By modifying the natural glyoxylate cycle, we aim to generate a biosynthetic pathway to produce glycolic acid in <I>S. cerevisiae</I> using the strains CEN.PK-1C and H3847<SUP id="cite_ref-Finnland_1-1" class="reference"><A href="#cite_note-Finnland-1">[1]</A></SUP> (see <A rel="nofollow" class="external text" href="https://2018.igem.org/Team:TU_Darmstadt/Project/Monomers">monomer subpage</A>).
</P><DIV class="caption"><P><B>Figure 1:</B> Reaction mechanism of ICL1 and <I>At</I>GLYR1.
</P></DIV><P><I>GOR1</I> is the natural occurring gene in <I>S. cerevisiae</I>, which encodes the enzyme converting glyoxylate into glycolic acid by reducing the aldehyde group in a NADPH-dependent reaction. Since the gene is not expressed<SUP id="cite_ref-GOR1_2-0" class="reference"><A href="#cite_note-GOR1-2">[2]</A></SUP> under normal growth conditions and the encoded glyoxylate reductase shows moderate affinity to the substrate glyoxylate, we used GLYR1 from <I>Arabidopsis thaliana</I> instead. Previous studies reported an enhanced affinity of <I>At</I>GLYR1 to glyoxylate <SUP id="cite_ref-affinity_AtGLYR1_3-0" class="reference"><A href="#cite_note-affinity_AtGLYR1-3">[3]</A></SUP> and also enhanced activity <SUP id="cite_ref-Finnland_1-2" class="reference"><A href="#cite_note-Finnland-1">[1]</A></SUP>. The enzyme has a molecular weight of 30.7 kDa <SUP id="cite_ref-FAZ_4-0" class="reference"><A href="#cite_note-FAZ-4">[4]</A></SUP>.
</P><P><VIDEO width="830" height="340" autoplay="autoplay" loop=""><SOURCE src="https://static.igem.org/mediawiki/2018/8/84/T--TU_Darmstadt--GLYR1.mp4" type="video/mp4">
   Your browser does not support the video tag.
</SOURCE></VIDEO></P><DIV class="caption"><P><B>Figure 2:</B> Structure of the NADPH-dependent glyoxylate reductase (<I>At</I>GLYR1) with a molecular weight of 30.7 kDa.
</P></DIV><P>Accumulating glyoxylate benefits the yield of glycolic acid. The isocitrate lyase 1, ICL1, catalyzes the reaction from isocitrate to glyoxylate. In the formation of glyoxylate, succinate occurs as a byproduct of the reaction. By overexpressing <I>ICL1</I>, a high substrate concentration is ensured. ICL1 has a molecular weight of 62.4 kDa<SUP id="cite_ref-Daten_ICL1_5-0" class="reference"><A href="#cite_note-Daten_ICL1-5">[5]</A></SUP>.
</P><P><VIDEO width="830" height="340" autoplay="autoplay" loop=""><SOURCE src="https://static.igem.org/mediawiki/2018/b/b0/T--TU_Darmstadt--Icl1.mp4" type="video/mp4">
   Your browser does not support the video tag.
</SOURCE></VIDEO></P><DIV class="caption"><P><B>Figure 3:</B> Structure of the isocitrate lyase 1 (ICL1) with a molecular weight of 62.4 kDa.
</P></DIV><P>The accumulation of glyoxylate is supported by the deletion of <I>MLS1</I> and <I>IDP2</I>, since the corresponding enzymes degrade glyoxylate and isocitrate in unwanted side reactions. Consequently, previous deletions of both genes in <I>S. cerevisiae</I> significantly increased glycolic acid production titers<SUP id="cite_ref-Finnland_1-3" class="reference"><A href="#cite_note-Finnland-1">[1]</A></SUP>.
MLS1, a malate synthase, catalyzes the conversion from glyoxylate to malate, and IDP2, 
an isocitrate dehydrogenase, catalyzes the oxidation of isocitrate to α-ketoglutarate. 
</P><P>As a result of our metabolic engineering approach, we were able to prove the activity of the glyoxylate reductase <I>At</I>GLYR1 and the <I>in vitro</I> production of glycolic acid.  We supposed a successful glycolic acid production <I>in vivo</I>. However, further HPLC analysis will be necessary.
</P><H3><SPAN class="mw-headline" id="Methods">Methods</SPAN></H3><P>The following methods have been applied in our lab work:
</P><DIV class="caption"><P><B>Figure 4:</B> Schematic representation of work flow.
</P></DIV><H4><SPAN class="mw-headline" id="Cloning">Cloning</SPAN></H4><P>Firstly, the sequence of <I>AtGLYR1</I> was modified with a <I>Strep</I>-tag on the 3´end and 20 base pairs overhangs at both ends of the gene, which are complementary with the alcohol dehydrogenase promotor (ADH) of the pAT423 shuttle vector. The sequence was ordered from Integrated DNA Technologies (IDT) and inserted into the pAT423 plasmid using the Gibson assembly method. <I>E. coli</I> TOP10 were transformed with generated plasmids and positive colonies were identified via colony PCR and DNA sequencing.
</P><DIV class="caption"><P><B>Figure 5:</B> pAT423 plasmid including expression cassette of <I>AtGLYR1</I>.
</P></DIV><H4><SPAN class="mw-headline" id="Deletion">Deletion</SPAN></H4><P><I>MLS1</I> and <I>IDP2</I> were deleted from the genome of the <I>S. cerevisiae</I> using deletion cassettes, which were integrated into the genome by homologous recombination. Both deletion cassettes (<I>KanMX</I> cassette and <I>Leucin</I> cassette) were first amplified via PCR. The primers were binding to the loxP-sites and had a 40 bp homologous region to the gene of interest. After that, <I>S. cerevisiae</I> cells were transformed with the linearized amplified deletion cassettes, with the result that <I>MLS1</I> was deleted using the <I>KanMX</I> and <I>IDP2</I> using the <I>Leucin</I> cassette.
</P><H4><SPAN class="mw-headline" id="SDS-PAGE_and_Western_Blot">SDS-PAGE and Western Blot</SPAN></H4><P>To verify that <I>At</I>GLYR1 was heterologously produced, a SDS-PAGE was performed, followed by a western blot. The resulting bands were compared to the expected molecular weight of 32.9 kDa.
</P><H4><SPAN class="mw-headline" id="Purification">Purification</SPAN></H4><P>After expression of <I>AtGLYR1</I> in <I>S. cerevisiae</I>, a GE Healthcare ÄKTA Pure machine was used to purify the desired <I>Strep</I>-tagged enzyme.    
</P><H4><SPAN class="mw-headline" id="Enzyme_Assays">Enzyme Assays</SPAN></H4><P>The assay for the glyoxylate reductase <I>At</I>GLYR1 is based on the different absorption maxima of NADPH (absorption maxima at 260 and 340 nm) and NADP<SUP>+</SUP> (maximum at 260 nm). During the reaction, the enzyme uses NADPH as a co-factor. NADPH is converted into NADP<SUP>+</SUP>, which leads to a decrease of absorption at 340 nm. By measuring the absorption at 340 nm over time, it is possible to quantify the enzyme activity and infer glycolic acid production. To calculate the NADPH conversion rate, a calibration curve was created by measuring the absorption of different NADPH concentrations.  
</P><DIV class="caption"><P><B>Figure 6:</B> Reaction mechanism of NADPH-dependent conversion of glyoxylate into glycolic acid performed by <I>At</I>GLYR1.
</P></DIV><H4><SPAN class="mw-headline" id="HPLC_Analysis">HPLC Analysis</SPAN></H4><P>To detect the produced monomer glycolic acid, as well as its precursors, high-performance liquid chromatography (HPLC) was employed. An organic acid separation column as stationary phase and sulfuric acid as mobile phase were used. This allowed the separation of glycolic acid, isocitrate and glyoxylate. Signals were recorded by a refractive index detector.
</P><H3><SPAN class="mw-headline" id="Results_and_Discussion">Results and Discussion</SPAN></H3><H4><SPAN class="mw-headline" id="Deletions">Deletions</SPAN></H4><P>To increase the yield of glycolic acid in <I>S. cerevisiae</I>, the genes <I>MLS1</I> (malate synthase) and <I>IDP2</I> (cytosolic isocitrate dehydrogenase) were deleted. MLS1 converts glyoxylate to malate. Therefore, the amount of substrate for <I>At</I>GLYR1 is minimized. IDP2 converts isocitrate to α–ketoglutarate. Isocitrate is also used by ICL1 as a substrate to produce glyoxylate. Since glyoxylate is the precursor of glycolic acid, as much isocitrate as possible should be converted to glyoxylate instead of α–ketoglutarate. To bypass these problems, the deletions were performed by transformation of CEN.PK-1C cells with a deletion cassette containing homologous regions to the flanking areas of the genes of interest (for a more detailed explanation see <A rel="nofollow" class="external text" href="https://2018.igem.org/Team:TU_Darmstadt/Notebook">Methods</A>). To determine whether or not the deletion of <I>MLS1</I> and <I>IDP2</I> worked, a PCR with two sets of primers was carried out. The primer sets contained the same forward primer but different reverse primers. If the deletion was not successful, only the reverse primer within the gene of interest would bind. If the deletion was successful, the reverse primer would bind within deletion cassette. Therefore, two PCRs per colony were performed. To see if the gene of interest or the deletion cassette was part of the genome, the colony PCR product was analyzed via agarose gel electrophoresis (Figure 7). 
</P><DIV class="caption"><P><B>Figure 7:</B> Gel electrophoretic separation for control of deletions. <I>KanMX</I> = rev primer binds in the <I>KanMX</I> cassette, MLS1 = rev primer binds in the <I>MLS1</I> sequence, Leu = rev primer binds in the <I>Leucin</I> cassette, IDP2 = rev primer binds in the <I>IDP2</I> sequence. 2-log ladder from NEB was used.
</P></DIV><P>Three different colonies were screened (Figure 7). The expected fragment sizes were 1650 bp if the <I>KanMX</I> cassette was successfully replaced <I>MLS1</I>, and 1012 bp if not. Colony 1 shows the two signals for <I>KanMX</I> as well as <I>MLS1</I>. If the deletion of <I>MLS1</I> had worked, only a fragment for <I>KanMX</I> should be visible. Therefore, colony 1 is a mixed culture. Colony 2 and 5 only show a fragment for <I>KanMX</I> but not for <I>MLS1</I>. This means that the deletion was performed successfully. 
<I>IDP2</I> was deleted using a <I>Leucin</I> cassette instead of <I>KanMX</I>. The expected fragment sizes for the cassette and <I>IDP2</I> were 1156 bp and 1498 bp, respectively. All three tested colonies show a clear fragment for <I>Leucin</I> at the expected size. The deletion of <I>IDP2</I> was therefore successful. Colony 1 was not used for further experimental procedures. 
</P><H4><SPAN class="mw-headline" id="Cloning_and_Expression">Cloning and Expression</SPAN></H4><P>To produce glycolic acid in <I>S. cerevisiae</I>, a glyoxylate reductase from <I>Arabidopsis thaliana</I> is needed to be expressed. The native glyoxylate reductase GOR1 does not have an affinity to glyoxylate as high as <I>At</I>GLYR1 from <I>A. thaliana</I>. A <I>S. cerevisiae</I> codon optimized version of the sequence coding for <I>At</I>GLYR1 containing a <I>Strep</I>-tag was ordered from IDT. The sequence was inserted into a shuttle vector (pAT423), which can be used for gene expression in <I>S. cerevisiae</I>, as well as in <I>E. coli</I>. The cloning was done using the Gibson assembly method. The accuracy of the cloning was confirmed by sequencing.
</P><P>To test whether <I>At</I>GLYR1 was produced in <I>S. cerevisiae</I>, we performed a western blot. We tested three colonies which were grown on selective medium after transformation with <I>AtGLYR1</I> x pAT423. As a negative control, untransformed CEN.PK-1C cells were used. The resulting western blot is shown in Figure 8. <I>At</I>GLYR1 was designed to contain a <I>Strep-tag</I>. Therefore, an anti-<I>Strep</I> antibody was used to detect the tagged protein with an expected molecular mass of 31.9 kDa. 
</P><DIV class="caption"><P><B>Figure 8:</B> Western blot for <I>At</I>GLYR1. Three colonies were screened for the production of <I>At</I>GLYR1 in <I>S. cerevisiae</I>. Thermo Scientific PageRuler Prestained Protein Ladder was used as protein standard. 1-3 = CEN.PK-1C was transformed using <I>AtGLYR1</I> x pAT423; - = Negative control of untransformed CEN.PK-1C.
</P></DIV><P>The western blot shows the results of three colonies, which were screened for the production of <I>At</I>GLYR1. Colony 1 and 3 produced <I>At</I>GLYR1. Colony 2 shows no band at the expected height. 
A corresponding positive colony was used for further experimental procedures. After gene expression for up to 22h, the protein was purified via StrepTactin columns using an ÄKTA system.
</P><H4><SPAN class="mw-headline" id="Enzyme_Assay">Enzyme Assay</SPAN></H4><P>The purified enzyme was subjected to a determination of its biological activity. The assay was based on the oxidization of NADPH. While the purified <I>At</I>GLYR1 oxidizes NADPH, glyoxylate is being reduced to glycolic acid. The conversion of NADPH to NADP<SUP>+</SUP> can be recognized through a decrease in absorption at 340 nm. Since the conversion of NADPH comes along with the reduction of glyoxylate, this assay can proof the activity of the enzyme and the possible <I>in vitro</I> production of glycolic acid. 
For the calibration curve, the absorption of different concentrations of NADPH was measured and used to calculate the substrate turnover rate (Figure 9). The purified <I>At</I>GLYR1 was assayed at 30 °C (Figure 10).
</P><DIV class="caption"><P><B>Figure 9 (left):</B> Calibration curve for <I>At</I>GLYR1 enzyme assay. Different NADPH concentrations (1 mM, 0.5 mM, 0.25 mM, 0.125 mM and 0 mM) were measured and plotted against their average absorption. n=3. 
</P><P><B>Figure 10 (right):</B> NADPH dependent enzyme assay of <I>At</I>GLYR1 at 30 °C. The graph shows the average absorption at 340 nm in correlation to the time (min). For the enzyme assay 2.7 ng/µL were used, positive (NADP<SUP>+</SUP>) and negative control (NADPH) were included. n=3.
</P></DIV><P>The average absorption of both, the positive and negative controls, stays constant over the time. After about 30 minutes of reaction time, the curve almost reaches the absoprtion level of the positive control, consisting of pure NADP<SUP>+</SUP>. It can be assumed that the decrease of the average absorption in the enzyme containing sample is due to the functionality and activity of <I>At</I>GLYR1, which oxidizes NADPH. The concentration of NADPH at the start of the assay was the same as for the negative control. Even though the enzyme is added to the reaction tube as the last component, the enzyme starts to work right before the first measurement point is recorded, so that the initial value was only 0.9 instead of 1.2. 
</P><P>The substrate conversion values were recorded in the linear range of the particular curve in a time frame of four minutes. Thus, 0.53 µM/s was the calculated substrat conversion rate of <I>At</I>GLYR1 at 30 °C. 
</P><P>Next, a Student’s t-test was performed to see whether or not these results were significant. For this purpose, five data sets (0, 5, 10, 20 and 30 minutes) were chosen (Figure 11).
</P><DIV class="caption"><P><B>Figure 11:</B> Enzyme assay absorption for 0, 5, 10, 20 and 30 minutes with standard error. n=3.
</P></DIV><P>Table 1 includes the values for figure 11. It also shows the NADPH-absorption at chosen times, as well as the associated standard error. 
</P><DIV class="caption"><P><B>Table 1:</B> Calculated significance of <I>At</I>GLYR1 data sets for 0, 5, 10, 20 and 30 minutes at 30 °C. The table also shows the NADPH-absorption, as well as the standard error. A= Absorption, *= Significance (p-value &lt; 0.05), n= 3.
</P></DIV><TABLE class="wikitable"><TBODY><TR><TH><I>At</I>GLYR1 [µg/mL]
</TH><TH> A (t=0 min)
</TH><TH> A (t=5 min)
</TH><TH> A (t=10 min)
</TH><TH> A (t=20 min)
</TH><TH> A (t=30 min)
</TH></TR><TR><TD> 2.7
</TD><TD> 0.879±0.023*
</TD><TD> 0.433±0.028*
</TD><TD> 0.27±0.021*
</TD><TD> 0.179±0.007*
</TD><TD> 0.16±0.002*
</TD></TR><TR><TD> neg.
</TD><TD> 1.209±0.019*
</TD><TD> 1.204±0.019*
</TD><TD> 1.197±0.019*
</TD><TD> 1.184±0.018*
</TD><TD> 1.172±0.018*
</TD></TR><TR><TD> pos.
</TD><TD> 0.139±0,001
</TD><TD> 0.139±0,001
</TD><TD> 0.139±0,001
</TD><TD> 0.139±0,001
</TD><TD> 0.14±0,001
</TD></TR></TBODY></TABLE><P>Both figure 11 and table 1 show the significance of the enzymes’ activity.
</P><H4><SPAN class="mw-headline" id="Growth_Assays_and_HPLC_Analysis">Growth Assays and HPLC Analysis</SPAN></H4><P>The deletion of <I>MLS1</I> and <I>IDP2</I> and the heterologous production of <I>At</I>GLYR1 in <I>S. cerevisiae</I> could affect its growth. For this reason, we measured the growth of the strain CEN.PK-1C (<I>At</I>GLYR1 x pAT423 Δ<I>MLS1</I> Δ<I>IDP2</I>) in YPD- and in YPEG-medium for approximately 25 hours by recording OD<SUB>600</SUB> values using a TECAN 200 pro reader. The same measurement was carried out with the wildtype strain. The YPD-medium contains glucose as a C-source, while YPEG-medium consists of ethanol and glycerol as C-sources.
The growth of the wildtype was observed in the two different media. The stationary phase in YPG-medium was achieved after 10 hours (OD<SUB>600</SUB> = 1.0) and in YPEG-medium after approximately 19 hours (OD<SUB>600</SUB> = 0.55). The modified CEN.PK-1C strain reached the start of the stationary phase after 12.5 hours in both media. However, the growth is clearly reduced in YPEG-medium (OD<SUB>600</SUB> = 0.9, figure 12).
</P><DIV class="caption"><P><B>Figure 12:</B> Growth assay of CEN.PK-1C wildtype and CEN.PK-1C (<I>At</I>GLYR1 x pAT423 Δ<I>MLS1</I> Δ<I>IDP2</I>) in YPD- and YPEG-medium for 25 hours by OD<SUB>600</SUB>.
</P></DIV><P>We concluded, that <I>S. cerevisiae</I> growths better in YPD- than in YPEG-medium and that the modified strain growths faster than the wildtype. For this reason, we suppose that <I>S. cerevisiae</I> indeed produces higher amounts of glycolic acid than unmodified strains but uses the produced glycolic acid as a new form of energy for faster growth. In order to prove that the modified strain produced glycolic acid, we analyzed the sterile-filtered supernatant of an overnight culture of CEN.PK-1C (<I>At</I>GLYR1xpAT423 Δ<I>MLS1</I> Δ<I>IDP2</I>) and the supernatant of another overnight culture of CEN.PK-1C (<I>At</I>GLYR1xpAT423 Δ<I>MLS1</I> Δ<I>IDP2</I>), after cell disruption via sonication using HPLC (Figure 13). The peak of the glycolic acid standard was observed after 15.7 minutes. In our HPLC measures we observed a peak after 16 minutes, which cannot be assigned to the glycolic acid peak. However, it could be possible that <I>S. cerevisiae</I> modified the produced glycolic acid, so that the peak of this modified glycolic acid molecule was observed after 16 minutes.
</P><DIV class="caption"><P><B>Figure 13:</B> Elution profile for the different molecules of an overnight culture of CEN.PK-1C (<I>At</I>GLYR1xpAT423 Δ<I>MLS1</I> Δ<I>IDP2</I>) after the HPLC-analysis. A significantly peak is observed after 16 minutes, which could correspond to a modified glycolic acid molecule.
</P></DIV><H3><SPAN class="mw-headline" id="Outlook">Outlook</SPAN></H3><P>Due to metabolic engineering of the glyoxylate cycle in <I>S. cerevisiae</I>, we were able to produce glycolic acid <I>in vitro</I> as one of our monomers for PLGA- and PGLC-synthesis. We also supposed, that its <I>in vivo</I> production was successful. However, further analysis via HPLC has to be carried out in order to validate our assumption. It has been shown that yeast acts as a suitable organism for genetic modifications and production of small organic acids. Yet, an emerging number of different yeast species have established themselves as microbial cell factories with at present unknown potential to further increase the yield of metabolic products. Recent work with <I>Kluveromyces lactis</I><SUP id="cite_ref-Finnland_1-4" class="reference"><A href="#cite_note-Finnland-1">[1]</A></SUP> suggests that a higher titer of glycolic acid can be received by implementation of similar biosynthetic approaches on different organisms. 
Obtaining a quantitative analysis of the so produced monomers would therefore be of particular relevance for the comparison of the chosen production strains. In this regard, the currently applied method <SUP id="cite_ref-Zahoor_6-0" class="reference"><A href="#cite_note-Zahoor-6">[6]</A></SUP> including high-performance liquid chromatography with use of a refractive index detector for signal recording can be complemented with external standards in order to create a quantifying calibration curve.  
</P><P>Furthermore, the construction of a BioBrick-compliant shuttle vector for overexpression, protein purification with <I>Strep</I>-tag affinity chromatography and subsequent activity assays, similar to those performed on AceA in <I>E. coli</I>, is the next step in the process of glycolic acid production. For the construction of a stable <I>AtGLYR1</I>-encoding strain, the glyoxylate reductase gene could be integrated into the genome via homologous recombination, as transformed populations can suffer from plasmid instability <SUP id="cite_ref-Zhang_7-0" class="reference"><A href="#cite_note-Zhang-7">[7]</A></SUP>.
Since the reaction catalyzed by <I>At</I>GLYR1 is depending on NADPH, the effect of co-factor limitation and regenerating systems, like we used for the production of ε-caprolactone, could also be investigated. Coordinating the two-gene expression ratio of heterologous <I>At</I>GLYR1 and overexpressed ICL1 is one potential future outlook. 
</P><H3><SPAN class="mw-headline" id="Team">Team</SPAN></H3><DIV class="caption"><P><B>Group picture of the &quot;Yeast&quot; team</B>From left to right: Victor Macarron Palacios, Elena Nickels, Jonathan Achter, Theresa Wörmann, Christian Hake, Jamina Gerhardus and Yannik Käseberg
</P></DIV><H3><SPAN class="mw-headline" id="References">References</SPAN></H3><OL class="references"><LI id="cite_note-Finnland-1"><SPAN class="mw-cite-backlink">↑ <SUP><A href="#cite_ref-Finnland_1-0">1.0</A></SUP><SUP><A href="#cite_ref-Finnland_1-1">1.1</A></SUP><SUP><A href="#cite_ref-Finnland_1-2">1.2</A></SUP><SUP><A href="#cite_ref-Finnland_1-3">1.3</A></SUP><SUP><A href="#cite_ref-Finnland_1-4">1.4</A></SUP></SPAN><SPAN class="reference-text"> Koivistoinen, O. M., Kuivanen, J., Barth, D., Turkia, H., Pitkänen, J.-P., Penttilä, M., &amp; Richard, P. (2013). Glycolic acid production in the engineered yeasts <I>Saccharomyces cerevisiae</I> and <I>Kluyveromyces lactis</I>. Microbial Cell Factories, 12, 82.
 <A rel="nofollow" class="external autonumber" href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3850452/">[1]</A>. </SPAN></LI><LI id="cite_note-GOR1-2"><SPAN class="mw-cite-backlink"><A href="#cite_ref-GOR1_2-0">↑</A></SPAN><SPAN class="reference-text"> Rintala, E., Pitkänen, J., Vehkomäki, M., Penttilä, M., &amp; Ruohonen, L. (2007). The ORF YNL274c (GOR1) codes for glyoxylate reductase in <I>Saccharomyces cerevisiae</I>. Yeast, 24, 129-136.<A rel="nofollow" class="external autonumber" href="https://onlinelibrary.wiley.com/doi/epdf/10.1002/yea.1434">[2]</A>. </SPAN></LI><LI id="cite_note-affinity_AtGLYR1-3"><SPAN class="mw-cite-backlink"><A href="#cite_ref-affinity_AtGLYR1_3-0">↑</A></SPAN><SPAN class="reference-text"> Hoover, G. J., Van Cauwenberghe, O. R., Breitkreuz, K. E., Clark, S. M., Merrill, A. R., &amp; Shelp, B. J. (2007). Characteristics of an <I>Arabidopsis</I> glyoxylate reductase: general biochemical properties and substrate specificity for the recombinant protein, and developmental expression and implications for glyoxylate and succinic semialdehyde metabolism in planta. Canadian Journal of Botany, 85, 883-895. <A rel="nofollow" class="external autonumber" href="http://www.nrcresearchpress.com/doi/abs/10.1139/B07-081#.W8CLm_ZCTIV">[3]</A>. </SPAN></LI><LI id="cite_note-FAZ-4"><SPAN class="mw-cite-backlink"><A href="#cite_ref-FAZ_4-0">↑</A></SPAN><SPAN class="reference-text"> UniProtKB - Q9LSV0 (GLYR1_ARATH), 10/15/2018 <A rel="nofollow" class="external autonumber" href="https://www.uniprot.org/uniprot/Q9LSV0">[4]</A>. </SPAN></LI><LI id="cite_note-Daten_ICL1-5"><SPAN class="mw-cite-backlink"><A href="#cite_ref-Daten_ICL1_5-0">↑</A></SPAN><SPAN class="reference-text"> ICL1 / YER065C Overview, 10/15/2018 <A rel="nofollow" class="external autonumber" href="https://www.yeastgenome.org/locus/S000000867#protein">[5]</A>. </SPAN></LI><LI id="cite_note-Zahoor-6"><SPAN class="mw-cite-backlink"><A href="#cite_ref-Zahoor_6-0">↑</A></SPAN><SPAN class="reference-text"> Zahoor, A., Otten, A., &amp; Wendisch, V. F. (2014). Metabolic engineering of Corynebacterium glutamicum for glycolate production. Journal of Biotechnology, 192, 366-375 <A rel="nofollow" class="external autonumber" href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3962153/">[6]</A>. </SPAN></LI><LI id="cite_note-Zhang-7"><SPAN class="mw-cite-backlink"><A href="#cite_ref-Zhang_7-0">↑</A></SPAN><SPAN class="reference-text"> Zhang, Z., Moo-Young, M., &amp; Christi, Y. (1996). Plasmid stability in recombinant Saccharomyces cerevisiae. Biotechnology Advances, 14(4), 401-435. <A rel="nofollow" class="external autonumber" href="https://www.ncbi.nlm.nih.gov/pubmed/14540156">[7]</A>. </SPAN></LI></OL></DIV></DIV></DIV></DIV></DIV></DIV></DIV></BODY></HTML>