<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_SUSTech_Shenzhen_Parts_Bronze skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><DIV id="top_title"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:SUSTech Shenzhen/Parts/Bronze</SPAN></H1></DIV><DIV id="HQ_page"><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><DIV style="background-color: white"><DIV class="igem_2018_team_column_wrapper"><DIV class="megamenu skyblue"><LI><CENTER><A class="color5" href="https://2018.igem.org/Team:SUSTech_Shenzhen" style="width:13%">Home</A></CENTER></LI><LI><CENTER><A class="color5" href="#" style="width:20%">Project</A></CENTER><DIV class="megapanel"><DIV class="row"><DIV class="col2"><DIV class="h_nav"><A href="https://2018.igem.org/Team:SUSTech_Shenzhen/Project_Overview"><H4>Overview</H4></A></DIV></DIV></DIV></DIV></LI><LI><A href="https://2018.igem.org/Team:SUSTech_Shenzhen/Motivations_Novelty">Motivations &amp; Novelty</A></LI><LI><A href="https://2018.igem.org/Team:SUSTech_Shenzhen/Project_design">Project Design</A></LI><LI><A href="https://2018.igem.org/Team:SUSTech_Shenzhen/Methods">Methods</A></LI></DIV></DIV><DIV class="col2"><DIV class="h_nav"><A href="#"><H4>Results</H4></A><UL><LI><A href="https://2018.igem.org/Team:SUSTech_Shenzhen/Engineered_cells">Engineered Cells</A></LI><LI><A href="https://2018.igem.org/Team:SUSTech_Shenzhen/Engineered_systems">Engineered Systems</A></LI><LI><A href="https://2018.igem.org/Team:SUSTech_Shenzhen/Outreach">Outreach</A></LI></UL></DIV></DIV><DIV class="col2"><DIV class="h_nav"><A href="#"><H4>For judges</H4></A><UL><LI><A href="https://2018.igem.org/Team:SUSTech_Shenzhen/Demonstrate">Demonstration</A></LI><LI><A href="https://2018.igem.org/Team:SUSTech_Shenzhen/Achievements">Achievements</A></LI><LI><A href="https://2018.igem.org/Team:SUSTech_Shenzhen/References">References</A></LI></UL></DIV></DIV></DIV></DIV><LI><CENTER><A class="color5" href="#" style="width:25%">Hardware &amp; Modeling</A></CENTER><DIV class="megapanel"><DIV class="row"><DIV class="col2"><DIV class="h_nav"><A href="https://2018.igem.org/Team:SUSTech_Shenzhen/Hardware"><H4>DOUBLE EMULSION SYSTEM</H4></A></DIV></DIV></DIV></DIV></LI><LI><A href="https://2018.igem.org/Team:SUSTech_Shenzhen/Hardware/Double_Emulsion_System">The Principle</A></LI><LI><A href="https://2018.igem.org/Team:SUSTech_Shenzhen/Hardware/Double_Emulsion_System#Components">Components and Reagents</A></LI><LI><A href="https://2018.igem.org/Team:SUSTech_Shenzhen/Hardware/Double_Emulsion_System#Protocols">Protocols</A></LI><LI><A href="https://2018.igem.org/Team:SUSTech_Shenzhen/Hardware/Double_Emulsion_System#Analysis">Analysis and Advantages</A></LI><LI><A href="https://2018.igem.org/Team:SUSTech_Shenzhen/Hardware/Double_Emulsion_System#Troubleshooting">Troubleshooting and Future</A></LI></DIV></DIV><DIV class="col2"><DIV class="h_nav"><A href="https://2018.igem.org/Team:SUSTech_Shenzhen/Hardware"><H4>MICRO-WELL SYSTEM</H4></A><UL><LI><A href="https://2018.igem.org/Team:SUSTech_Shenzhen/Hardware/Micro-well_system">Fabrication and Principles</A></LI><LI><A href="https://2018.igem.org/Team:SUSTech_Shenzhen/Hardware/Micro-well_system#Aim">Aim of the System</A></LI><LI><A href="https://2018.igem.org/Team:SUSTech_Shenzhen/Hardware/Micro-well_system#Protocols">Protocols</A></LI><LI><A href="https://2018.igem.org/Team:SUSTech_Shenzhen/Hardware/Micro-well_system#Analysis">Analysis</A></LI><LI><A href="https://2018.igem.org/Team:SUSTech_Shenzhen/Hardware/Micro-well_system#Troubleshooting">Troubleshooting and Future</A></LI></UL></DIV></DIV><DIV class="col2"><DIV class="h_nav"><A href="https://2018.igem.org/Team:SUSTech_Shenzhen/Model"><H4>Modeling</H4></A><UL><LI><A href="https://2018.igem.org/Team:SUSTech_Shenzhen/Model">Overview</A></LI><LI><A href="https://2018.igem.org/Team:SUSTech_Shenzhen/Model#Genes_prediction_model_of_Wnt_secretion">Genes Prediction of Wnt Secretion</A></LI><LI><A href="https://2018.igem.org/Team:SUSTech_Shenzhen/Model#Kinetic_model_of_Wnt_signaling">Kinetics of Wnt Signaling Pathway</A></LI><LI><A href="https://2018.igem.org/Team:SUSTech_Shenzhen/Model#Model_of_encapsulation">Model of Encapsulation</A></LI></UL></DIV></DIV></DIV></DIV><LI><CENTER><A class="color5" href="https://2018.igem.org/Team:SUSTech_Shenzhen/Human_Practices" style="width:22%">Human Practices</A></CENTER><DIV class="megapanel"><DIV class="row"><DIV class="col2"><DIV class="h_nav"><A href="https://2018.igem.org/Team:SUSTech_Shenzhen/Human_Practices"><H4>Overview &amp; CCC Project</H4></A></DIV></DIV></DIV></DIV></LI><LI><A href="https://2018.igem.org/Team:SUSTech_Shenzhen/Human_Practices">Overview</A></LI><LI><A href="https://2018.igem.org/Team:SUSTech_Shenzhen/HP/Module_1">HUH?</A></LI><LI><A href="https://2018.igem.org/Team:SUSTech_Shenzhen/HP/Module_2">EMMM...</A></LI><DIV class="col2"><DIV class="h_nav"><A href="#"><H4> </H4></A><UL><LI><A href="https://2018.igem.org/Team:SUSTech_Shenzhen/HP/Module_3">OOOPS-</A></LI><LI><A href="https://2018.igem.org/Team:SUSTech_Shenzhen/HP/Module_4">DING~</A></LI><LI><A href="https://2018.igem.org/Team:SUSTech_Shenzhen/HP/Module_5">BANG!</A></LI></UL></DIV></DIV><DIV class="col2"><DIV class="h_nav"><A href="#"><H4>Required events</H4></A><UL><LI><A href="https://2018.igem.org/Team:SUSTech_Shenzhen/Collaborations">Collaboration</A></LI><LI><A href="https://2018.igem.org/Team:SUSTech_Shenzhen/Public_Engagement">Education &amp; Public Engagement</A></LI><LI><A href="https://2018.igem.org/Team:SUSTech_Shenzhen/HP/Safety_Responsibility">Safety &amp; Responsibility</A></LI><LI><A href="https://2018.igem.org/Team:SUSTech_Shenzhen/Integrated">Integrated Human Practices</A></LI></UL></DIV></DIV><LI><CENTER><A class="color5" href="#" style="width:20%">Others</A></CENTER><DIV class="megapanel"><DIV class="row"><DIV class="col2"><DIV class="h_nav"><A href="#"><H4>Parts</H4></A></DIV></DIV></DIV></DIV></LI><LI><A href="https://2018.igem.org/Team:SUSTech_Shenzhen/Parts">Overview</A></LI><LI><A href="https://2018.igem.org/Team:SUSTech_Shenzhen/Improve#Gold">Gold</A></LI><LI><A href="https://2018.igem.org/Team:SUSTech_Shenzhen/Improve#Silver">Silver</A></LI><LI><A href="https://2018.igem.org/Team:SUSTech_Shenzhen/Improve#Bronze">Bronze</A></LI><DIV class="col2"><DIV class="h_nav"><A href="#"><H4>Team</H4></A><UL><LI><A href="https://2018.igem.org/Team:SUSTech_Shenzhen/Members">Members</A></LI><LI><A href="https://2018.igem.org/Team:SUSTech_Shenzhen/Attributions">Attributions</A></LI><LI><A href="https://2018.igem.org/Team:SUSTech_Shenzhen/Attributions#Acknowledgement">Acknowledgement</A></LI></UL></DIV></DIV><DIV class="col1_5"><DIV class="h_nav"><A href="#"><H4>Notebook</H4></A><UL><LI><A href="https://2018.igem.org/Team:SUSTech_Shenzhen/Calendars">Calendars</A></LI><LI><A href="https://2018.igem.org/Team:SUSTech_Shenzhen/Protocols">Protocols</A></LI><LI><A href="https://2018.igem.org/Team:SUSTech_Shenzhen/Materials">Materials</A></LI></UL></DIV></DIV><DIV class="welcome" id="Bronze"><DIV class="container"><H2>Bronze part</H2></DIV><DIV class="why Bronze"><DIV class="container"><H4>pSB1C3_BM-1_BBa_I714031</H4><DIV class="camp"><DIV class="col-md-4 minist-right"><P>For this part, we have found a point error (G-&gt;A) in its sequence, which may potentially cause unexpected functional damage. Therefore, we intended to fix the point mutation by artificial mutagenesis. The total length of this biobrick, BBa_I714031, is 278bp, inside which located a point error in No.108bp.Thus, we decided to utilize PCR to generate mutations. The forward primer (BM1_mut_F) is 5’-cgcctcgtctctcatgccg gcggtagccggctgcc-3’, and the reverse primer (BM1_mut_R) is 5’-gcgaggcag ccggctaccgccggcatgagagacgaggcg-3’.</P><P>First, we designed the PCR experiment as follows:<SPAN style="white-space:pre;font-size: 100%">Plasmid_BBa_I714031			2 μLBM1_mut_F					2 μLBM1_mut_R					2 μL2×phanta Max Master Mix		25μLH2O						19 μLTotal						50 μL</SPAN></P></DIV><DIV class="col-md-3 minist-right"><P>For negative control, the 2×phanta Max Master Mix was replaced by H2O of the same volume. PCR program was 95°C for 30s, then went to 25 cycles, which included degradation at 95°C for 15s，annealing at 63°C for 15s, extension at 72°C for 2min, then followed by a 5-min 72°C completely extension. Then we run a gel to see whether PCR is successful, in which 15kb marker and 1% gel are used to run for 30min. The gel image of our results was shown in Fig1.</P><P style="text-align:center">Figure1. Linearization PCR results of Plasmid_BBa_I714031. First lane: Marker. Second lane: experimental group. Third lane: negative control (without enzymes)</P></DIV><DIV class="col-md-5 minist-right"><P>After that, extraction of DNA sample from gel was done. Then we used DpnI, which cuts the methylated GATC sequence, to digest the methylated origin templates and left the newly synthesized DNA. Finally, we directly transformed the digestion products into DH5α, due to the repair system of E.coli, the linearized plasmids was fixed to circular. Then we did the miniprep and sent to Sanger sequencing. The sequencing result shown we had successfully generated the point mutation in No.108bp, which means the single error was repaired. (Fig2) </P><P style="text-align:center">Figure2. Alignment of mutagenesis product and original sequence.</P></DIV></DIV></DIV><DIV class="container" style="margin-top: 50px; "><H4>pSB1C3_BM2_BBa_K1592002</H4><DIV class="camp"><DIV class="col-md-4 minist-right"><P>For this part, we have found a point error(T-&gt;C), which may potentially cause some unexpected functional damages. Therefore, we intended to fix the point mutation by artificial mutagenesis. The total length of this biobrick, BBa_K1592002, is 366bp, inside which located a point error in No.189bp. So, we decided to utilize PCR to generate mutation. The forward primer (BM2_mut_F) is 5’-cgaggccgctgctggtgctc acgccactgccggtgccattg-3’, and the reverse primer (BM2_mut_R) is 5’-caatggcaccggca gtggcgtgagcaccagcagcggcctcg-3’.</P><P>First, we designed the PCR experiment as follows:<SPAN style="white-space:pre;font-size: 100%">plasmid_ BBa_K1592002		2 μLBM2_mut_F					2 μLBM2_mut_R					2 μL2×phanta Max Master Mix		25μLH2O						19 μLTotal						50 μL</SPAN></P></DIV><DIV class="col-md-3 minist-right"><P>For negative controls, the 2×phanta Max Master Mix was replaced by H2O of the same volume.
				The PCR program was set at 95°C for 30s, then went to 25 cycles, which included degradation at 95°C for 15s，annealing at 63°C for 15s, extension at 72°C for 2min, following with a 5-min 72°C completely extension. Then we run a gel to see whether the PCR is successful, in which 5kb marker and 1% gel are used to run for 30min. The gel image was shown in Fig1. 
				</P><P style="text-align:center">Figure1. Linearization PCR results of Plasmid_BBa_I714031. First lane: Marker. Second lane: experimental group. Third lane: negative control (NO enzyme)</P></DIV><DIV class="col-md-5 minist-right"><P>After that, extraction of DNA sample from gel was done. Then we used DpnI, which cut the methylated GATC sequence, to digest the methylated origin templates and left the newly synthesized DNA. Finally, we directly transformed the digestion product into DH5α, due to the repair system of E.coli, the linearized plasmids was fixed to circular. Then we did the miniprep and sent it to Sanger sequencing. The sequencing result showed that we had successfully generate the point mutation in No.189bp, which means the single error was repaired. (Fig2)</P><P style="text-align:center">Figure2. Alignment of mutagenesis product and original sequence. </P></DIV></DIV></DIV></DIV></DIV><DIV class="welcome" id="Silver"><DIV class="container" style="margin-bottom: 50px"><H2>Silver part</H2><P>For Validated Part in Silver Medal, we added three parts (<A href="#">BBa_K2591010</A>, <A href="#">BBa_K2591011</A>, and <A href="#">BBa_K2591012</A>), consisting of gRNA of genes together with scaffold, all of which were well characterized, validated, and used in our project.</P></DIV><DIV class="why Silver"><DIV class="container"><DIV class="camp" style="margin-top: 0px"><DIV class="col-md-4 minist-right"><P>Naturally, single guide RNA (sgRNA) is a small RNA which guide CRISRR-Cas protein family to target the exogenous sequence in prokaryotes, and assists them to defend phages. Nowadays, the artificial CRISPR-Cas9 system can achieve the modification casually in the gene level and is an excellent way to introduce mutation. The mechanism is shown in Fig1, the Cas9 nuclease is targeted to genomic DNA by a sgRNA consisting of a 20-nt guide sequence (blue) and a scaffold (red). The guide sequence pairs with the DNA target (blue bar on top strand), directly upstream of a require a 5’-NGG adjacent motif (PAM; pink). Cas9 mediates a double strand break in the upstream of the PAM (red triangle). (Ran, et al, Nat. Protoc., 2013)  </P><P style="text-align:center">Figure1. Schematic of the RNA-guided Cas9 nuclease. (Ran, et al, Nat. Protoc., 2013) </P><P>Wnt3A protein is a secretion protein function in the development and proliferation of the normal cell, and more importantly, canonical Wnt pathway plays a role in the induction of cancer. In WNT secretion process, Porcupine O-acyltransferase (PORCN), an essential enzyme to function in the Wnt3A protein secretion. Wntless WNT ligand secretion mediator (WLS) is a functional protein for the secretion of the Wnt3A protein and assists the modification for the mature of Wnt3A protein. </P></DIV><DIV class="col-md-4 minist-right"><P>In our project, we design the gRNAs for PORCN, WLS, and Wnt3a. If we transfect their gRNA into cells, we can inhibit the donor cells from secreting the Wnt3A proteins normally. Knockout of PORCN and WLS can block the Wnt3a secretion through disturbing its maturation process, while knockout of Wnt3a blocks the process from the origin, which works as a positive control. And we have integrated each gRNA together with the scaffold into the backbone pSB1C3 to get a new part (<A href="#">BBa_K2591010</A>, <A href="#">BBa_K2591011</A>, <A href="#">and BBa_K2591012</A>). (Fig2)</P><P style="text-align:center">Figure2. Diagram of pSB1C3_gRNA_scaffold.</P><P>As mentioned above, knockout of these three genes will impair the Wnt3a secretion activity in the cells. Therefore, we first introduced sgRNA into the L-Wnt3A-Cas9-mCherry cell line, which continuously expresses Cas9 protein. After that, we coculture them with our Wnt3a reception cell, 293R-TCF-EGFP cell, then observed the fluorescence after one day to validate our part. As a result, we did not see green fluorescent protein at the green filed(475nm), which means the Wnt3A protein cannot be secreted from the L cell normally, resulting no reception in 293R-TCF-EGFP cell. And all of the gRNA designed for these genes could target specific sequence and did mutate it with Cas9 protein.  </P></DIV><DIV class="col-md-4 minist-right"><P>We also observed the validated results using FACS. Validation images are shown in Fig4 and Fig5.</P><P style="text-align:center">Figure4. Knockout of POR, WLS and WNT show no Wnt secretion, fluorescence validation. First row: normal 293R cell line. Second to fourth rows: 293R cell line cultured with L-cell with POR, WLS and WNT knockout. Fifth row: 293R cell line cultured with L-cell with no target knockout. Column legends indicated the observation field, merged means artificial adding of red and green. </P><P style="text-align:center">Figure5. Knockout of POR, WLS and WNT show no Wnt secretion, FACS validation. (a) Coculture of L cell-NON-target-KO with 293R-TCF-GFP in 24 hours. (b) Coculture of L cell-NON-target-KO with 293R-TCF-GFP in 0 hours. (c, d, e) Coculture of L cell-POR (WLS, WNT)-KO with 293R-TCF-GFP in 24 hours. </P></DIV></DIV></DIV></DIV></DIV><DIV class="welcome" id="Gold"><DIV class="container" style="margin-bottom: 50px"><H2>Gold part</H2></DIV><DIV class="why Gold"><DIV class="container"><H4>TCF_TATA_GFP</H4><H4>Brief introduction</H4><P>This improved part, TCF_TATA_GFP, is designed to generate a reporter system that can senses Wnt3a signal, and is improved from normal GFP part, BBa_E0040. This part can be stably transfected into our receptor cells. When receptor cells with this part received a Wnt3a signal, beta-catenin would be released and bind to TCF, activating downstream GFP expression, as a consequence, green fluorescence can be observed. In the contrast, when there is no Wnt3a signal, the expression of GFP is shut down. In a word, Wnt3a works as a switch here in this part. (picture concerning Wnt signals is shown as Fig 1)
			</P><DIV class="camp" style=""><H4>Usage and biology</H4><DIV class="col-md-4 minist-right"><H3>Wnt pathway TCF TRE </H3><P>The WNT signaling pathway is one of the most essential cellular signal transduction pathways, which begins with reception of WNT ligands in the plasma membrane, initiating a series of downstream events. If there is no reception of WNT ligands, the subunits of disruption complex is free with the ability to bind the downstream protein β-catenin in cytoplasm, mediating its degradation, which means there is no more downstream events.(Fig1)</P><P style="text-align:center">Figure1. Condition of no WNT signal</P><P>However, when WNT ligands bind to the receptor Frizzled, the disruption complex is fixed by WNT-Frizzled complex in plasma membrane, leading to the release and accumulation of β-catenin.</P></DIV><DIV class="col-md-4 minist-right"><P> After that, the β-catenin can finally translocate into the nucleus and act as a co-activator of transcriptional factor TCF. The β-catenin-TCF complex then interacts with TCF transcriptional response elements (TCF TRE) and starts the transcription of WNT target genes. (Fig2) </P><P style="text-align:center">Figure2. The WNT-on Stage</P><P>In the absence of β-catenin, TCF acts as a repressor of WNT target genes by occupying the TCF TRE, though β-catenin can convert the TCF into a transcriptional activator of the same genes.This novel process can be extracted from nature and makes up a reporter system when we replace the target genes into a fluorescence gene like GFP.  </P></DIV><DIV class="col-md-4 minist-right"><P>When this whole construct stably be inserted to a receptor cell line, it can sense WNT signals and give green fluorescence. </P><P style="text-align:center">Figure3. GFP followed by TCF TRE</P><P>The part we construct is improved from GFP part in Registry (link to GFP part http://parts.igem.org/Part:BBa_I714891), in front of GFP, we add two TCF transcriptional response elements and a TATA box. (Fig.4) </P><P style="text-align:center">Figure4. The construct of TCF-TATA-GFP in pSB1C3</P></DIV></DIV></DIV><DIV class="container"><DIV class="camp" style=""><H4>Characterization </H4><DIV class="col-md-4 minist-right"><P>We first designed primers and construct the plasmid in Snapgene, then according to the  protocols which we write previously, we did some molecular cloning experiments. Finally we got the plasmid and sent to Sanger sequencing. Forward(VF2) and reverse(VR).</P><P style="text-align:center">Figure4. Forward and reverse sequencing results aligned with designed plasmid</P><P> Sequencing results show that we have successfully got the right plasmid.(Fig.4) After verification in sequence level, we designed experiments to verify its function.</P></DIV><DIV class="col-md-4 minist-right"><P> We have stably integrated this construct into the genome of our reporter cell line by lentiviral infection. At the same time, we have used the original part(BBa_E0040) to do the same experiments, the fluorescence images are shown in Fig5. And for TCF-GFP cells, we added Wnt3a to the culture medium, after several hours, we can observe green fluorescence under microscope(Fig.6) and also, we can see a shift in fluorescence activated cell sorting(FACS) experiments.(Fig.7)</P><P style="text-align:center">Figure5. Fluorescence image of original GFP part (BBa_E0040)</P></DIV><DIV class="col-md-4 minist-right"><P style="text-align:center">Figure6.  Fluorescence image of WNT activation by Chir99021(GSK3β inhibitor) and Wnt3a</P><P style="text-align:center">Figure7. FACS results show GFP intensity shift after adding Wnt3a</P></DIV></DIV></DIV><DIV class="container"><H4>Application</H4><P>This TCF-TATA-GFP construct can be used to indicate the activation of WNT pathway, which largely related carcinogenesis. Therefore, it’s a powerful indicator in cancer pathway researches.(Akiri, et al, 2009) Besides this, it can indicate the extracellular Wnt3a protein, acting as perfect reporter system. 
			</P><P>This method of transcriptional response element(TRE) coupled with a fluorescence gene gives a new window for biosensor in synthetic biology. Especially when the chassis organisms are eukaryotic cells. It may inspire more iGEMers to design project based on eukaryotic cells. </P></DIV></DIV><DIV class="why Gold" style="padding-top: 0px"><DIV class="container"><H4>loxP-STOP-loxP(LSL)</H4><H4>Background</H4><P>At the very beginning, we wanted to use the Cre-loxp system to switch on the giving signals of our donor cells. However, we found it’s not practical at all because the signal giving system is hard to be modified by generating some loxP sites and some other complex reasons. we then devoted ourselves to and improved the existing parts, BGH poly A mammalian terminator(<A href="#">BBa_K678019</A>) and loxP site(<A href="#">BBa_J61046</A>), generating a loxP-stop-loxP construct, which can act as a switch to control the transcription of downstream sequence.
			</P><DIV class="camp" style=""><H4>Introduction</H4><DIV class="col-md-4 minist-right"><P>The part called loxP-STOP-loxP(LSL)  is a combination of the loxP site and the bGH poly A terminating signal. Generally, Cre-loxP system is a widely used system to generate deletions, translocations and inversions at specific sites of DNA. It consists of two directional 34bp loxP sequence. When these two loxP sites possess the same direction, the sequence between them will be excised by Cre recombinase(Fig1). BGH poly A is shown in Fig1.</P></DIV><DIV class="col-md-4 minist-right"><P style="text-align:center">Fig 1. BGH poly A</P></DIV><DIV class="col-md-4 minist-right"><P>BGH poly A, as one of the mammalian terminators, mediates the polyadenylation of mRNA; we improved this part by adding two loxP site with the same direction both before and after it, which means the BGH poly A terminator can be deleted under Cre recombinase as a result.</P></DIV></DIV></DIV><DIV class="container"><DIV class="camp" style=""><H4>Usage and biology</H4><DIV class="col-md-4 minist-right"><P>In the field of molecular biology and cell biology, basic concepts of transcription termination in eukaryotes are widely applied to the technology of molecular cloning. For the DNA polymerase II, the transcription termination is always mediated by the process called polyadenylation, which is initiated by poly A signal in the 3’ downstream region of coding sequence. BGH poly A signal is a commonly used terminator in molecular cloning to build a normal expression construct initiated by Pol II. </P><P> Cre-loxP recombination system is a popularly used system to generate deletions, translocations and inversions at specific sites of DNA. This system is relatively simple compared to other mechanic switches, it only need two LoxP sites and one single Cre enzyme, and then, inversion or deletion happens.  In details, it consists of two directional 34bp loxP sequence.When these two loxP sites possess the same direction, the sequence between them will be excised by Cre recombinase; </P></DIV><DIV class="col-md-4 minist-right"><P>while if these two loxP sites possess opposite directions, the sequence between them will be inverted.  This method is implemented both in eukaryotic and prokaryotic cells. Particularly, Cre-loxP system is used in transgenic mice as well as neurobiology, with a great potential usage in synthetic biology as a switch.</P><P style="text-align:center">Figure1. Deletion event mediated by Cre recombinase when two LoxP sequence possess the same direction.</P><P>In the case that BGH polyA signal is flanked by two LoxP site, in which the construct is leaded by a promoter and followed by a functional gene.</P></DIV><DIV class="col-md-4 minist-right"><P> Withoutthe Cre recombinase, transcription happens but terminates before coding sequence, producing a short piece of mRNA, which goes to abortion. (Fig2) On the contrast, when we introduce Cre recombinase, the recombination happens, and BGH polyA signal sequence is deleted as the result. This coding sequence can be effectively transcribed to a much longer mRNA and thus the functional protein can be expresses well. (Fig3)</P><P style="text-align:center">Figure2. Shorter mRNA terminated by BGH polyA before coding sequence and the abortion of mRNA</P><P style="text-align:center">Figure3. Longer mRNA terminated by BGH polyA after coding sequence and expression of functional protein</P></DIV></DIV></DIV><DIV class="container"><H4>Application</H4><P>LSL can be insert in front of any functional gene or some switches in synthetic biology. It can mediate time-specific or stage-specific expressions of some essential genes. For example, scientists have used this strategy to activate some oncogenes in specific stages. Also, for members in the iGEM community, when  constructing a circuit, they can  work with the global turn-on button, LSL, to make their contributions.
			</P></DIV><DIV class="container"><H4>References</H4><P>Akiri, G., Cherian, M. M., Vijayakumar, S., Liu, G., Bafico, A., &amp; Aaronson, S. A. (2009). Wnt pathway aberrations including autocrine Wnt activation occur at high frequency in human non-small-cell lung carcinoma. <I>Oncogene</I>, 28(21), 2163.
			</P><P>Blauwkamp, T. A., Nigam, S., Ardehali, R., Weissman, I. L., &amp; Nusse, R. (2012). Endogenous Wnt signalling in human embryonic stem cells generates an equilibrium of distinct lineage-specified progenitors. <I>Nature communications</I>, 3, 1070.</P><P>LEF, G. W., &amp; GFP, T. D. (2002). LEF/TCF-Dependent, Fluorescence-Based Re-porter Gene Assay For Wnt Signaling. <I>BioTechniques</I>, 32, 1000-1002.</P><P>Jackson EL; Willis N; Mercer K; Bronson RT; Crowley D; Montoya R; Jacks T; Tuveson DA. 2001. Analysis of lung tumor initiation and progression using conditional expression of oncogenic K-ras. <I>Genes Dev</I> 15(24):3243-8PubMed: 11751630MGI: J:73445 Sequence and Features</P></DIV></DIV></DIV><DIV class="footer"><DIV class="col-md-3 contact-left"><H4>Address</H4><P><A href="http://www.sustc.edu.cn/en">
						  No 1088, Xueyuan Rd., Xili, 
						  Nanshan District, Shenzhen, 
						  Guangdong, China
					</A></P></DIV><DIV class="col-md-3 contact-left"><H4>Contact us</H4><P><A href="mailto:igem@sustc.edu.cn">Mail to us</A></P><P><A href="https://twitter.com/iGEM_SUSTech">Twitter</A></P></DIV><DIV class="copyright">
			Licensed under <A href="http://creativecommons.org/licenses/by/4.0/">CC BY 4.0</A>.
			   ©2018 SUSTech_Shenzhen
		</DIV></DIV></BODY></HTML>