<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_SUSTech_Shenzhen_Model_Modeling skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><DIV id="top_title"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:SUSTech Shenzhen/Model/Modeling</SPAN></H1></DIV><DIV id="HQ_page"><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><DIV style="background-color: white"><DIV class="igem_2018_team_column_wrapper"><DIV class="megamenu skyblue"><LI><CENTER><A class="color5" href="https://2018.igem.org/Team:SUSTech_Shenzhen" style="width:13%">Home</A></CENTER></LI><LI><CENTER><A class="color5" href="#" style="width:20%">Project</A></CENTER><DIV class="megapanel"><DIV class="row"><DIV class="col2"><DIV class="h_nav"><A href="https://2018.igem.org/Team:SUSTech_Shenzhen/Project_Overview"><H4>Overview</H4></A></DIV></DIV></DIV></DIV></LI><LI><A href="https://2018.igem.org/Team:SUSTech_Shenzhen/Motivations_Novelty">Motivations &amp; Novelty</A></LI><LI><A href="https://2018.igem.org/Team:SUSTech_Shenzhen/Project_design">Project Design</A></LI><LI><A href="https://2018.igem.org/Team:SUSTech_Shenzhen/Methods">Methods</A></LI></DIV></DIV><DIV class="col2"><DIV class="h_nav"><A href="#"><H4>Results</H4></A><UL><LI><A href="https://2018.igem.org/Team:SUSTech_Shenzhen/Engineered_cells">Engineered Cells</A></LI><LI><A href="https://2018.igem.org/Team:SUSTech_Shenzhen/Engineered_systems">Engineered Systems</A></LI><LI><A href="https://2018.igem.org/Team:SUSTech_Shenzhen/Outreach">Outreach</A></LI></UL></DIV></DIV><DIV class="col2"><DIV class="h_nav"><A href="#"><H4>For judges</H4></A><UL><LI><A href="https://2018.igem.org/Team:SUSTech_Shenzhen/Demonstrate">Demonstration</A></LI><LI><A href="https://2018.igem.org/Team:SUSTech_Shenzhen/Achievements">Achievements</A></LI><LI><A href="https://2018.igem.org/Team:SUSTech_Shenzhen/References">References</A></LI></UL></DIV></DIV></DIV></DIV><LI><CENTER><A class="color5" href="#" style="width:25%">Hardware &amp; Modeling</A></CENTER><DIV class="megapanel"><DIV class="row"><DIV class="col2"><DIV class="h_nav"><A href="https://2018.igem.org/Team:SUSTech_Shenzhen/Hardware"><H4>DOUBLE EMULSION SYSTEM</H4></A></DIV></DIV></DIV></DIV></LI><LI><A href="https://2018.igem.org/Team:SUSTech_Shenzhen/Hardware/Double_Emulsion_System">The Principle</A></LI><LI><A href="https://2018.igem.org/Team:SUSTech_Shenzhen/Hardware/Double_Emulsion_System#Components">Components and Reagents</A></LI><LI><A href="https://2018.igem.org/Team:SUSTech_Shenzhen/Hardware/Double_Emulsion_System#Protocols">Protocols</A></LI><LI><A href="https://2018.igem.org/Team:SUSTech_Shenzhen/Hardware/Double_Emulsion_System#Analysis">Analysis and Advantages</A></LI><LI><A href="https://2018.igem.org/Team:SUSTech_Shenzhen/Hardware/Double_Emulsion_System#Troubleshooting">Troubleshooting and Future</A></LI></DIV></DIV><DIV class="col2"><DIV class="h_nav"><A href="https://2018.igem.org/Team:SUSTech_Shenzhen/Hardware"><H4>MICRO-WELL SYSTEM</H4></A><UL><LI><A href="https://2018.igem.org/Team:SUSTech_Shenzhen/Hardware/Micro-well_system">Fabrication and Principles</A></LI><LI><A href="https://2018.igem.org/Team:SUSTech_Shenzhen/Hardware/Micro-well_system#Aim">Aim of the System</A></LI><LI><A href="https://2018.igem.org/Team:SUSTech_Shenzhen/Hardware/Micro-well_system#Protocols">Protocols</A></LI><LI><A href="https://2018.igem.org/Team:SUSTech_Shenzhen/Hardware/Micro-well_system#Analysis">Analysis</A></LI><LI><A href="https://2018.igem.org/Team:SUSTech_Shenzhen/Hardware/Micro-well_system#Troubleshooting">Troubleshooting and Future</A></LI></UL></DIV></DIV><DIV class="col2"><DIV class="h_nav"><A href="https://2018.igem.org/Team:SUSTech_Shenzhen/Model"><H4>Modeling</H4></A><UL><LI><A href="https://2018.igem.org/Team:SUSTech_Shenzhen/Model">Overview</A></LI><LI><A href="https://2018.igem.org/Team:SUSTech_Shenzhen/Model#Genes_prediction_model_of_Wnt_secretion">Genes Prediction of Wnt Secretion</A></LI><LI><A href="https://2018.igem.org/Team:SUSTech_Shenzhen/Model#Kinetic_model_of_Wnt_signaling">Kinetics of Wnt Signaling Pathway</A></LI><LI><A href="https://2018.igem.org/Team:SUSTech_Shenzhen/Model#Model_of_encapsulation">Model of Encapsulation</A></LI></UL></DIV></DIV></DIV></DIV><LI><CENTER><A class="color5" href="https://2018.igem.org/Team:SUSTech_Shenzhen/Human_Practices" style="width:22%">Human Practices</A></CENTER><DIV class="megapanel"><DIV class="row"><DIV class="col2"><DIV class="h_nav"><A href="https://2018.igem.org/Team:SUSTech_Shenzhen/Human_Practices"><H4>Overview &amp; CCC Project</H4></A></DIV></DIV></DIV></DIV></LI><LI><A href="https://2018.igem.org/Team:SUSTech_Shenzhen/Human_Practices">Overview</A></LI><LI><A href="https://2018.igem.org/Team:SUSTech_Shenzhen/HP/Module_1">HUH?</A></LI><LI><A href="https://2018.igem.org/Team:SUSTech_Shenzhen/HP/Module_2">EMMM...</A></LI><DIV class="col2"><DIV class="h_nav"><A href="#"><H4> </H4></A><UL><LI><A href="https://2018.igem.org/Team:SUSTech_Shenzhen/HP/Module_3">OOOPS-</A></LI><LI><A href="https://2018.igem.org/Team:SUSTech_Shenzhen/HP/Module_4">DING~</A></LI><LI><A href="https://2018.igem.org/Team:SUSTech_Shenzhen/HP/Module_5">BANG!</A></LI></UL></DIV></DIV><DIV class="col2"><DIV class="h_nav"><A href="#"><H4>Required events</H4></A><UL><LI><A href="https://2018.igem.org/Team:SUSTech_Shenzhen/Collaborations">Collaboration</A></LI><LI><A href="https://2018.igem.org/Team:SUSTech_Shenzhen/Public_Engagement">Education &amp; Public Engagement</A></LI><LI><A href="https://2018.igem.org/Team:SUSTech_Shenzhen/HP/Safety_Responsibility">Safety &amp; Responsibility</A></LI><LI><A href="https://2018.igem.org/Team:SUSTech_Shenzhen/Integrated">Integrated Human Practices</A></LI></UL></DIV></DIV><LI><CENTER><A class="color5" href="#" style="width:20%">Others</A></CENTER><DIV class="megapanel"><DIV class="row"><DIV class="col2"><DIV class="h_nav"><A href="#"><H4>Parts</H4></A></DIV></DIV></DIV></DIV></LI><LI><A href="https://2018.igem.org/Team:SUSTech_Shenzhen/Parts">Overview</A></LI><LI><A href="https://2018.igem.org/Team:SUSTech_Shenzhen/Improve#Gold">Gold</A></LI><LI><A href="https://2018.igem.org/Team:SUSTech_Shenzhen/Improve#Silver">Silver</A></LI><LI><A href="https://2018.igem.org/Team:SUSTech_Shenzhen/Improve#Bronze">Bronze</A></LI><DIV class="col2"><DIV class="h_nav"><A href="#"><H4>Team</H4></A><UL><LI><A href="https://2018.igem.org/Team:SUSTech_Shenzhen/Members">Members</A></LI><LI><A href="https://2018.igem.org/Team:SUSTech_Shenzhen/Attributions">Attributions</A></LI><LI><A href="https://2018.igem.org/Team:SUSTech_Shenzhen/Attributions#Acknowledgement">Acknowledgement</A></LI></UL></DIV></DIV><DIV class="col1_5"><DIV class="h_nav"><A href="#"><H4>Notebook</H4></A><UL><LI><A href="https://2018.igem.org/Team:SUSTech_Shenzhen/Calendars">Calendars</A></LI><LI><A href="https://2018.igem.org/Team:SUSTech_Shenzhen/Protocols">Protocols</A></LI><LI><A href="https://2018.igem.org/Team:SUSTech_Shenzhen/Materials">Materials</A></LI></UL></DIV></DIV><DIV class="services"><DIV class="container"><DIV class="SUSTech_title">Modeling</DIV><DIV class="Overview ＆ formulation"><H3 style="color:#1cd5b5;font-weight:bold;margin-bottom:50px;"> Overview ＆formulation</H3><P>      In our wet lab experiments, we developed a novel Two-Cell system to study cell-cell communication. Millions of donor cells with single gene alteration convey the signal to the receptor cells, according to the response 
		of receptor cells, we can back to investigate the alterations of interest. </P><P>      However, the fantastic engineering approach does have some limitations, first, when future iGEM teams move further to design larger screening, like whole genome screening, our initial methodology of capturing two cells
		within one droplet could potentially become less applicable, because of cell death during the long emulsion time. Nevertheless, this drawback can be overcome by further improve our
<A href="https://2018.igem.org/Team:SUSTech_Shenzhen/Hardware/Double_Emulsion_System" style="color:#1cd5b5;font-style:italic;text-decoration:underline">hardware</A>, also, 
		the reverse approach by modeling may open up a new window. We can use pre-existing knowledge and bioinformatic method to narrow down the screening size previously. </P><P>      Another thing we found in experiments is the Wnt response capability of Wnt receptor cell is weak and it needs time to send out response. Then, we wanted to see 
		is it possible to use computational modeling to simulate the 
		biological process of Wnt signaling and help to optimize experiments. </P><P>      And we can never work with an engineered system without any quantification, that will largely blot out our understanding of our engineered system. So, we want to build a encapsulation model from our own experimental
		data to see whether we can improve our system from theoretical perspective.</P><P>      So, for modeling, we have three aims:</P><DIV class="welcome-btm" style="padding-top:30px;padding-bottom:30px;"><DIV class="col-md-4 welcome-left"><H5 style="color:#1cd5b5">1</H5><P>Narrow down the screening size and explore potential secretion-related candidate genes through co-expression analysis.</P></DIV><DIV class="col-md-4 welcome-left"><H5 style="color:#1cd5b5">2</H5><P>Define the importance of key features in signaling pathway to optimize our construction of receptor cell.</P></DIV><DIV class="col-md-4 welcome-left"><H5 style="color:#1cd5b5">3</H5><P>Find out the appropriate Droplet Generation Rate (DGR) and estimate the efficiency of the system.</P></DIV></DIV><P>      Before any biological modeling, the most important thing is formulating the scenario,
		and for these secretion and reception can be regarded as two systems, one secret Wnt after a cellular process, the other give response after signal process. (Fig1) </P><P>      After simplification, the diagram is like Fig2. For the right part, Wnt gene as the origin, 
		known or unknown factors influencing the process to mature Wnt protein. For the left part, Wnt protein becomes the input. And the output is Wnt pathway target genes.</P><H4 style="font-weight:bold;text-align:center;font-size:0.8em">Figure1. Real scenario of Wnt pathway.</H4><H4 style="font-weight:bold;text-align:center;font-size:0.8em">Figure2. Formulated diagram of Wnt pathway.</H4></DIV><DIV id="Genes_prediction_model_of_Wnt_secretion" style="padding-top:100px;"><H3>Genes prediction model of Wnt secretion</H3><P>      For Wnt secretion part, we want to use co-expression analysis to search Wnt secretion-related candidate genes to narrow down our screen size o
		f gRNA library. This means we use these known factors, which are known enzymes involving in Wnt protein maturation, like PORCN and WLS, to fish
		new unknown factors.</P><P>      And the basic principal of co-expression of two genes is shown in Fig 1, it compares the expression patterns of two genes across samples, define their co-expressing correlation valves. The higher the valve, the more similar they are, they may 
		undertake the same function. (Zhang, et al, 2005) And once we get the co-expressing gene list, we can design a more specific gRNA library to do engineered screening.
		</P><H4 style="font-weight:bold;text-align:center;font-size:0.8em">Figure3. basic principle of co-expression of genes</H4><P>      The co-expression model is based on Pearson correlation coefficient between two genes. The formula is shown as below in Fig4. </P><H4 style="font-weight:bold;text-align:center;font-size:0.8em">Figure4. The formula of Pearson’s correlation coefficient. xi and yi are two genes’ expression level index with i. n is the sample size. ¯x is the mean expression level of all samples, and analogously for ¯y.</H4><P style="padding-top:50px">      We have collected a large amount of gene expression datasets of different samples from online databases (Dam, et al, 2014), we regarded it as n samples. And for each sample, it consisted of a large gene list which couples with their expression level. We regarded it as m genes. Therefore, it’s a m x n matrix, for the expression pattern of each gene, it’s actually a n dimension vector. Pearson’s correlation coefficient measures the tendency of these two vectors of gene expression level to increase or decrease together, giving a value of their overall correspondence.</P><H4 style="font-weight:bold;text-align:center;font-size:0.8em">Figure5. The pipeline of heatmap generation</H4><P>      After calculating Pearson’s correlation coefficient of each pair of genes from expression matrix, we generate a correlation matrix. And we apply heatmap to present the results (Fig5), each column is one sample’s gene expression, each row is one gene’s expression level across samples. The genes with high Pears
		on’s correlation coefficient cluster together automatically, so do the samples. (Fig6) Then, we focus on the gene cluster around our reference genes, PORCN 
		and WLS.</P><H4 style="font-weight:bold;text-align:center;font-size:0.8em">Figure6. Part of a small heatmap generated from gene expression matrix. (18 samples only) Each column shows one sample’s gene expression; each row shows one gene’s expression level across samples. WLS is in highlighted color. 
			(data from NCBI, Schoumacher, et al, 2014)</H4><P>      After a series heatmaps were generated, we extract two gene lists based on the workflow diagram of Fig7. We compare each genes’ coefficient with WLS, rank them, set a threshold, select out the genes with coefficient larger than threshold to obtain a new list, in which every gene is considered as potential related genes to WLS. Because we have lots of samples actually, which will intrinsically to get lower Pearson’s correlation coefficient as the dimension of vectors of gene expression gets higher. Therefore, final Pearson’s correlation coefficient we get is different from Fig5 shows, which only contains 18 samples. They are no more than 0.7.
		</P><H4 style="font-weight:bold;text-align:center;font-size:0.8em">Figure7. Workflow diagram of potential gene selection. (WLS as an example.)</H4><P>      So, for WLS related genes, we set the threshold as 0.3, through which we get 215 genes. And for PORCN related genes, 
		we set the threshold as 0.25, through which we get 108 genes. Top10 genes and their annotations are shown in Table1 and Table2. From these two tables, we can
		see even the sample size is large, we can always get some highly related genes. However, from the annotations of these potential genes, we can not draw a 
		strong conclusion without further verification. This point does reveal the complexity of biological phenomena. There is even an uncharacterized gene that has high correlation coefficient with PORCN. Besides their annotated functions, these genes may have potential relation to WLS and PORCN, and they may play roles in Wnt secretion pathway.
		</P><H4 style="font-weight:bold;text-align:center;font-size:0.8em;margin-top:100px">Table1. Top10 related genes to WLS generated from correlation model. Annotations show the validated functions of genes.</H4><H4 style="font-weight:bold;text-align:center;font-size:0.8em;margin-top:20px;">Table2. Top10 related genes to WLS generated from correlation model. Annotations show the validated functions of genes.  </H4><P>      Now that we have two potential Wnt secretion-related gene lists, but we are still uncertain to say they are related to Wnt secretion. In order to further verify the results, we can use these gene lists to narrow down our gRNA library and screening 
size, then use our engineered system to perform double emulsion <A href="https://2018.igem.org/Team:SUSTech_Shenzhen/Outreach" style="color:#1cd5b5;font-style:italic;text-decoration:underline">experiments</A>, hopefully, we can verify these genes with high efficiency. 		
		</P><P>      On the other hand, we try to further narrow down the size of these two gene lists to find out the candidate genes through modeling. First, we merge these two lists to form one gene list. And then the improving method is identifying the differentially expressed genes between Wnt high secretion cell and Wnt low secretion cell. The intersection of these differentially expressed genes and our previous co-expressed genes should have much higher potential to be Wnt secretion-related genes.
		</P></DIV><DIV id="Kinetic_model_of_Wnt_signaling" style="padding-top:100px;"><H3>Kinetic model of Wnt signaling</H3><P>      For the left part of Fig1 and Fig2, we wanted to define the importance of key features in signaling pathway to optimize our construction of receptor cell line. The changes of output can be regarded as accumulation of the changes of input and pathway components. Obviously, the 
		change of input (Wnt ligand) contributes the most to the whole signaling pathway. However, when we fix the Wnt ligand, how much do the other components contribute?		
		</P><P>      To answer this question, we integrated our experimental observation and data pre-existing knowledge (MacDonald, et al, 2009; Clevers, 2006; Anastas ＆ Moon, 2013), and gene expression data from online database[TCGA, 2012] to deeply analyze Wnt signaling pathway, then construct a contribution model to update our understanding of this pathway. Once we know the contribution coefficient, we can adjust the signaling pathway through knock down or overexpression experiments to make the reception cell more sensitive. 
		</P><P>      Wnt signaling pathway is presented on the KEGG database (Kanehisa, et al, 2000) and Pathway-Common (Cerami, et al, 2010). We picked genes from Wnt canonical pathway according to previous researches as our pathway components. Also, some useful information was found on GTEx, TCGA and NCBI databases, including gene expression data and gene description.
		</P><P>      Finally, we collected the gene expression data of 50 different normal samples. And we have selected one input gene, WNT2, one output gene, C-MYC, which is one of WNT pathway targets, and x component genes, they are SFRP, DKK1, LRP5, LRP6, FZD2, BAMBI, DVL1, DVL2, DVL3, GSK3B, APC, AXIN1, CTNNB1, CSNK1A1, TCF7, C-MYC. Besides, in order to test the feasibility of our method in the last procedure, we introduced three control genes, they are: CFL1, NPM1, CALR. All of them are housekeeping genes, which means their expression keeping in a relatively stable level and have no effects on Wnt signaling pathway.The picture of formulated wnt canonical signaling pathway including our selected genes is shown in Fig8. 
		</P><H4 style="font-weight:bold;text-align:center;font-size:0.8em">Figure8. Formulated wnt canonical signaling pathway. Rounded rectangle represents components in signaling pathway, oval represents input and output. </H4><P>      After a series of essential formulation done, we began our real modeling. </P><DIV class="step1"><H4 style="color:#1cd5b5;padding-left:100px;padding-top:50px">Step1. Data Pre-processing</H4><P><SPAN style="color:#1cd5b5;font-weight:bold;font-size:1em;padding-left:100px;">Purpose:</SPAN> to make sure all the sample under the similar WNT exposing condition, to fix Wnt ligand input.</P><P>      In the beginning, we used grey relation analysis (GRA) to select the samples that under the same physiological condition, which means they should possess
			the same extracellular environment. GRA is one kind of methods to describe curves similarity quantitively. Fig9 explains how it works. </P><H4 style="font-weight:bold;text-align:center;font-size:0.8em">Figure9. Example of Grey Relation Coefficient, the more similar two curves are, the higher GRA coefficient they have.</H4><P>      We chose the first sample as our reference sample and calculate the grey relation coefficient of the other 49 samples compared to the first sample. The results are represented as distribution histogram (Fig10). After that, normal function was applied to fit it. Result shows all samples are under very similar condition. All of them can be used in our following processes. 
			</P><H4 style="font-weight:bold;text-align:center;font-size:0.8em">Figure10. The standardized results of grey relation analysis.</H4></DIV><DIV class="step2"><H4 style="color:#1cd5b5;padding-left:100px;padding-top:30px">Step2. Cluster Analysis</H4><P><SPAN style="color:#1cd5b5;font-weight:bold;font-size:1em;padding-left:100px;">Purpose:</SPAN>pre-define some correlation parameters based on pre-existing gene connections.</P><P>      For gene pathway analysis, it’s very common to use cluster analysis to verify the connection between genes. In order to better simulate their inner relationship, we chose hierarchical clustering as our analysis method.</P><P>      We possessed 50 samples’ gene expression data, after filtering, each sample contains 17 genes that we are interested. This indicates a 17 × 50 matrix. Each gene is a 50 dimension vector. If we normalize it and calculate the distance between each pairs of genes, an 17 × 17 distance matrix will be generated. We picked out the gene pair that has the smallest distance and put them all together, their average valve will 
			be treated as a new column. Then, calculate the distance again and find another pair, repeat it until we cluster them all. Our cluster result are as shown in Fig11.
			</P><H4 style="font-weight:bold;text-align:center;font-size:0.8em">Figure11. The clustering result for our selected genes</H4><P>      It’s very surprised for us to get this result. As the figure shows, DVL1/2/3 are clustered together, the distance between LRP5 and LRP6 is also very small. Moreover, their relationship is nearly coincided with the canonical pathway of Wnt signaling. This result further encouraged us to continue. The distance between genes was then extracted and used in the following step.
			</P></DIV><DIV class="step3"><H4 style="color:#1cd5b5;padding-left:100px;padding-top:30px">Step3. Pathway Analysis</H4><P><SPAN style="color:#1cd5b5;font-weight:bold;font-size:1em;padding-left:100px;">Purpose:</SPAN>define the contribution strength of pathway components</P><P>      The relationship between genes is very complex and the basic assumption here is every two genes are linked. The concrete interactive network is under the mist through. These complicated effects include but not limited to: feedback regulation, transgenic interaction, coexpression effect.</P><P>      It’s important to quantify these all effect in one form. Correlation coefficient is the basic math analysis method, which reveals their mathmatic relationship in a total black box. The correlation coefficients between of our selected genes were calculated. Results are as shown in Fig12.
			</P><H4 style="font-weight:bold;text-align:center;font-size:0.8em">Figure12. The correlation coeficient matrix between selected genes.</H4><P>      However, a single correlation coefficient will cause lots of error. Combing with the cluster, which reveals genes relationship in a relatvie accurate way. We modified correlation with their distance data and normalize it. This is the new correlation coefficient for unknown relationships.
				While for those known gene relationships, we modify it manually according to the previous results.
			</P><P>      Let’s back to our assumption: Every two genes are linked. It’s necessary to say: The strength of the link was quantified by modified correlation coefficient.If every two genes are linked, it means one gene can cause some effect on all other genes through many pathways.
			</P><P>      Considering component gene A, B, C and output gene. If we want to calculate the effect caused by A. Here are five conditions:
			</P><DIV style="padding-left:100px;"><OL class="wel"><LI>A → output</LI><LI>A → B → output</LI><LI>A → C → output</LI><LI>A → B → C → output</LI><LI>A → C → B → output</LI></OL></DIV><P>      If we calculate one gene acts on output through 1 gene. We need to solve the equation below to find out direct pathway coefficient:
			</P><P>      Due to the limits of the computing power, we can’t list all possible ways. Nevertheless, it’s enough to calculate one gene acts on output through 4 genes. (Correlation coefficient is always less than 1, more than 4 genes connection can be ignored.)
			</P><P>      We solve the equations and find out P value for each pairs of component gene to output gene(C-MYC). P is treated as direct effect from genes to Wnt signaling pathway. The final results are shown in the Fig13. 
			</P><H4 style="font-weight:bold;text-align:center;font-size:0.8em">Figure13. Genes contribution to Wnt signaling pathway based on pathway analysis</H4><P>      As the result shows, our control genes are at a low level around zero, which means they do not affect too much. They do have a value can reflect the systematic errors of our methods. The TCF7 was the biggest contributor to Wnt signaling pathway. Nevertheless, other genes effects are still very close to known pathway relationship, which reveals the correctness of our methods.
			</P><P>      And the results actually can help us construct our reception cell line, to adjust the condition of signaling pathway so as to maximize the sensitivity of Wnt response. 
			</P></DIV></DIV><DIV id="Model_of_encapsulation" style="padding-top:100px"><H3>Model of encapsulation</H3><P>      Since we apply the double emulsion system to encapsulate the cells, we want to estimate the efficiency of the system. That is, we aim to deduce the total time in which the required cells are encapsulated. 
		</P><P>      The experience of experiments tells us that the number of particles containing in one droplet obeys the Poisson 
	Distribution. Hence, the first step is to estimate the coefficient of the distribution. Assumerefers to the number of particles containing in droplet
	and  .
	Then the moment estimator ofis  . After many times of trials with single channel feed, we conclude that  . Hence, we are able to draw the distribution 
	probability of single channel feed. (Fig14)
		</P><H4 style="font-weight:bold;text-align:center;font-size:0.8em">Figure14. Particle number distribution of single channel feed.</H4><P>      With the condition of double channel feed, we assume the two feeding channels are independent and the new Poisson Distribution follows  . Then we are able to draw the distribution probability of double channel feed. (Fig15) As we can see, the probability of two particle encapsulated in one droplet is 1.64%. That means if 10000 droplets are produced, about 164 of them are exactly what we need. 
		</P><H4 style="font-weight:bold;text-align:center;font-size:0.8em">Figure15. Particle number distribution of double channel feed.</H4><P>      The second step is to optimize the droplet generation rate. With larger droplet generation rate (The number of droplets produced per second), less time is required to produce the sample. We denote the Droplet Generation Rate as DGR, the number of the encapsulated cells required as N, the Poisson Distribution with single channel feed as  , the Poisson Distribution with double channel feed as  . Then the required time is calculated by the formula in Fig16.
		</P><H4 style="font-weight:bold;text-align:center;font-size:0.8em">Figure16. The formula of requiring time.</H4><P>      And we draw the graph of the above function, DGR as x-axis and required time as y-axis. (Fig18) We can infer from the graph that the time required is inverse proportional to DGR. When DGR is large than 3500, the time changes little. Hence, we may modify the system where DGR is around 3500 to make it more effective. 
		</P><H4 style="font-weight:bold;text-align:center;font-size:0.8em">Figure17. The graph of function in Fig16, required time vs. DGR.</H4><H4 style="color:#1cd5b5;padding-top:30px;font-weight:bold">Discussion</H4><P>      Our modeling and analysis are all based on scientific logic, real data, and literature conclusion, we are focusing on the improvement and complement for our experimental design and engineered system. Also, both the co-expression methods to narrow down screen size the optimization methods for reception cell line are able to give considerations and inspirations for future iGEM team that wants to extend our ideas, designing more exciting projects related to intercellular communication. What’s more, the simulation model of double emulsion system can provide ways to make
		it more applicable for other teams and other fields. We do look forward to seeing it is applied to various fields, like biosensor, biosynthesis and drug screening. 
		</P><H4 style="color:#1cd5b5;padding-top:30px;font-weight:bold">References</H4><DIV class="reference"><P>      Zhang, Bin and Horvath, Steve (2005) &quot;A General Framework for Weighted Gene CoExpression Network Analysis,&quot; <I>Statistical Applications in Genetics and Molecular Biology</I>: Vol. 4: Iss. 1, Article 17. DOI: 10.2202/1544-6115.1128</P><P>      van Dam, S., Craig, T., &amp; de Magalhaes, J. P. (2014). GeneFriends: a human RNA-seq-based gene and transcript co-expression database. <I>Nucleic acids research</I>, 43(D1), D1124-D1132.</P><P>      Schoumacher, M., Hurov, K. E., Lehár, J., Yan-Neale, Y., Mishina, Y., Sonkin, D., ... &amp; Cooke, V. G. (2014). Inhibiting Tankyrases sensitizes KRAS mutant cancer cells to MEK inhibitors via FGFR2 feedback signaling. <I>Cancer research</I>, canres-0138.</P><P>      Anastas, J. N., &amp; Moon, R. T. (2013). WNT signalling pathways as therapeutic targets in cancer. <I>Nature Reviews Cancer</I>, 13(1), 11.</P><P>      Clevers, H. (2006). Wnt/β-catenin signaling in development and disease. <I>Cell</I>, 127(3), 469-480.</P><P>      MacDonald, B. T., Tamai, K., &amp; He, X. (2009). Wnt/β-catenin signaling: components, mechanisms, and diseases. <I>Developmental cell</I>, 17(1), 9-26.</P><P>      Cancer Genome Atlas Network. (2012). Comprehensive molecular characterization of human colon and rectal cancer. <I>Nature</I>, 487(7407), 330.</P><P>      Cerami, E. G., Gross, B. E., Demir, E., Rodchenkov, I., Babur, Ö., Anwar, N., ... &amp; Sander, C. (2010). Pathway Commons, a web resource for biological pathway data. <I>Nucleic acids research</I>, 39(suppl_1), D685-D690.</P><P>      Kanehisa, M., &amp; Goto, S. (2000). KEGG: kyoto encyclopedia of genes and genomes. <I>Nucleic acids research</I>, 28(1), 27-30.</P></DIV></DIV></DIV></DIV><DIV class="footer"><DIV class="footer"><DIV class="col-md-3 contact-left"><H4>Address</H4><P><A href="http://www.sustc.edu.cn/en">
						  No 1088, Xueyuan Rd., Xili, 
						  Nanshan District, Shenzhen, 
						  Guangdong, China
					</A></P></DIV><DIV class="col-md-3 contact-left"><H4>Contact us</H4><P><A href="mailto:igem@sustc.edu.cn">Mail to us</A></P><P><A href="https://twitter.com/iGEM_SUSTech">Twitter</A></P></DIV><DIV class="copyright">
			Licensed under <A href="http://creativecommons.org/licenses/by/4.0/">CC BY 4.0</A>.
			   ©2018 SUSTech_Shenzhen
		</DIV></DIV></DIV></BODY></HTML>