<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_SUSTech_Shenzhen_Protocols skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><DIV id="top_title"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:SUSTech Shenzhen/Protocols</SPAN></H1></DIV><DIV id="HQ_page"><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><DIV style="background-color: white"><DIV class="igem_2018_team_column_wrapper"><DIV class="megamenu skyblue"><LI><CENTER><A class="color5" href="https://2018.igem.org/Team:SUSTech_Shenzhen" style="width:13%">Home</A></CENTER></LI><LI><CENTER><A class="color5" href="#" style="width:20%">Project</A></CENTER><DIV class="megapanel"><DIV class="row"><DIV class="col2"><DIV class="h_nav"><A href="https://2018.igem.org/Team:SUSTech_Shenzhen/Project_Overview"><H4>Overview</H4></A></DIV></DIV></DIV></DIV></LI><LI><A href="https://2018.igem.org/Team:SUSTech_Shenzhen/Motivations_Novelty">Motivations &amp; Novelty</A></LI><LI><A href="https://2018.igem.org/Team:SUSTech_Shenzhen/Project_design">Project Design</A></LI><LI><A href="https://2018.igem.org/Team:SUSTech_Shenzhen/Methods">Methods</A></LI></DIV></DIV><DIV class="col2"><DIV class="h_nav"><A href="#"><H4>Results</H4></A><UL><LI><A href="https://2018.igem.org/Team:SUSTech_Shenzhen/Engineered_cells">Engineered Cells</A></LI><LI><A href="https://2018.igem.org/Team:SUSTech_Shenzhen/Engineered_systems">Engineered Systems</A></LI><LI><A href="https://2018.igem.org/Team:SUSTech_Shenzhen/Outreach">Outreach</A></LI></UL></DIV></DIV><DIV class="col2"><DIV class="h_nav"><A href="#"><H4>For judges</H4></A><UL><LI><A href="https://2018.igem.org/Team:SUSTech_Shenzhen/Demonstrate">Demonstration</A></LI><LI><A href="https://2018.igem.org/Team:SUSTech_Shenzhen/Achievements">Achievements</A></LI><LI><A href="https://2018.igem.org/Team:SUSTech_Shenzhen/References">References</A></LI></UL></DIV></DIV></DIV></DIV><LI><CENTER><A class="color5" href="#" style="width:25%">Hardware &amp; Modeling</A></CENTER><DIV class="megapanel"><DIV class="row"><DIV class="col2"><DIV class="h_nav"><A href="https://2018.igem.org/Team:SUSTech_Shenzhen/Hardware"><H4>DOUBLE EMULSION SYSTEM</H4></A></DIV></DIV></DIV></DIV></LI><LI><A href="https://2018.igem.org/Team:SUSTech_Shenzhen/Hardware/Double_Emulsion_System">The Principle</A></LI><LI><A href="https://2018.igem.org/Team:SUSTech_Shenzhen/Hardware/Double_Emulsion_System#Components">Components and Reagents</A></LI><LI><A href="https://2018.igem.org/Team:SUSTech_Shenzhen/Hardware/Double_Emulsion_System#Protocols">Protocols</A></LI><LI><A href="https://2018.igem.org/Team:SUSTech_Shenzhen/Hardware/Double_Emulsion_System#Analysis">Analysis and Advantages</A></LI><LI><A href="https://2018.igem.org/Team:SUSTech_Shenzhen/Hardware/Double_Emulsion_System#Troubleshooting">Troubleshooting and Future</A></LI></DIV></DIV><DIV class="col2"><DIV class="h_nav"><A href="https://2018.igem.org/Team:SUSTech_Shenzhen/Hardware"><H4>MICRO-WELL SYSTEM</H4></A><UL><LI><A href="https://2018.igem.org/Team:SUSTech_Shenzhen/Hardware/Micro-well_system">Fabrication and Principles</A></LI><LI><A href="https://2018.igem.org/Team:SUSTech_Shenzhen/Hardware/Micro-well_system#Aim">Aim of the System</A></LI><LI><A href="https://2018.igem.org/Team:SUSTech_Shenzhen/Hardware/Micro-well_system#Protocols">Protocols</A></LI><LI><A href="https://2018.igem.org/Team:SUSTech_Shenzhen/Hardware/Micro-well_system#Analysis">Analysis</A></LI><LI><A href="https://2018.igem.org/Team:SUSTech_Shenzhen/Hardware/Micro-well_system#Troubleshooting">Troubleshooting and Future</A></LI></UL></DIV></DIV><DIV class="col2"><DIV class="h_nav"><A href="https://2018.igem.org/Team:SUSTech_Shenzhen/Model"><H4>Modeling</H4></A><UL><LI><A href="https://2018.igem.org/Team:SUSTech_Shenzhen/Model">Overview</A></LI><LI><A href="https://2018.igem.org/Team:SUSTech_Shenzhen/Model#Genes_prediction_model_of_Wnt_secretion">Genes Prediction of Wnt Secretion</A></LI><LI><A href="https://2018.igem.org/Team:SUSTech_Shenzhen/Model#Kinetic_model_of_Wnt_signaling">Kinetics of Wnt Signaling Pathway</A></LI><LI><A href="https://2018.igem.org/Team:SUSTech_Shenzhen/Model#Model_of_encapsulation">Model of Encapsulation</A></LI></UL></DIV></DIV></DIV></DIV><LI><CENTER><A class="color5" href="https://2018.igem.org/Team:SUSTech_Shenzhen/Human_Practices" style="width:22%">Human Practices</A></CENTER><DIV class="megapanel"><DIV class="row"><DIV class="col2"><DIV class="h_nav"><A href="https://2018.igem.org/Team:SUSTech_Shenzhen/Human_Practices"><H4>Overview &amp; CCC Project</H4></A></DIV></DIV></DIV></DIV></LI><LI><A href="https://2018.igem.org/Team:SUSTech_Shenzhen/Human_Practices">Overview</A></LI><LI><A href="https://2018.igem.org/Team:SUSTech_Shenzhen/HP/Module_1">HUH?</A></LI><LI><A href="https://2018.igem.org/Team:SUSTech_Shenzhen/HP/Module_2">EMMM...</A></LI><DIV class="col2"><DIV class="h_nav"><A href="#"><H4> </H4></A><UL><LI><A href="https://2018.igem.org/Team:SUSTech_Shenzhen/HP/Module_3">OOOPS-</A></LI><LI><A href="https://2018.igem.org/Team:SUSTech_Shenzhen/HP/Module_4">DING~</A></LI><LI><A href="https://2018.igem.org/Team:SUSTech_Shenzhen/HP/Module_5">BANG!</A></LI></UL></DIV></DIV><DIV class="col2"><DIV class="h_nav"><A href="#"><H4>Required events</H4></A><UL><LI><A href="https://2018.igem.org/Team:SUSTech_Shenzhen/Collaborations">Collaboration</A></LI><LI><A href="https://2018.igem.org/Team:SUSTech_Shenzhen/Public_Engagement">Education &amp; Public Engagement</A></LI><LI><A href="https://2018.igem.org/Team:SUSTech_Shenzhen/HP/Safety_Responsibility">Safety &amp; Responsibility</A></LI><LI><A href="https://2018.igem.org/Team:SUSTech_Shenzhen/Integrated">Integrated Human Practices</A></LI></UL></DIV></DIV><LI><CENTER><A class="color5" href="#" style="width:20%">Others</A></CENTER><DIV class="megapanel"><DIV class="row"><DIV class="col2"><DIV class="h_nav"><A href="#"><H4>Parts</H4></A></DIV></DIV></DIV></DIV></LI><LI><A href="https://2018.igem.org/Team:SUSTech_Shenzhen/Parts">Overview</A></LI><LI><A href="https://2018.igem.org/Team:SUSTech_Shenzhen/Improve#Gold">Gold</A></LI><LI><A href="https://2018.igem.org/Team:SUSTech_Shenzhen/Improve#Silver">Silver</A></LI><LI><A href="https://2018.igem.org/Team:SUSTech_Shenzhen/Improve#Bronze">Bronze</A></LI><DIV class="col2"><DIV class="h_nav"><A href="#"><H4>Team</H4></A><UL><LI><A href="https://2018.igem.org/Team:SUSTech_Shenzhen/Members">Members</A></LI><LI><A href="https://2018.igem.org/Team:SUSTech_Shenzhen/Attributions">Attributions</A></LI><LI><A href="https://2018.igem.org/Team:SUSTech_Shenzhen/Attributions#Acknowledgement">Acknowledgement</A></LI></UL></DIV></DIV><DIV class="col1_5"><DIV class="h_nav"><A href="#"><H4>Notebook</H4></A><UL><LI><A href="https://2018.igem.org/Team:SUSTech_Shenzhen/Calendars">Calendars</A></LI><LI><A href="https://2018.igem.org/Team:SUSTech_Shenzhen/Protocols">Protocols</A></LI><LI><A href="https://2018.igem.org/Team:SUSTech_Shenzhen/Materials">Materials</A></LI></UL></DIV></DIV><DIV class="services Protocols" style="margin: 50px 0;text-indent:0px"><DIV class="SUSTech_title" style="color:black;font-size:6em">Protocols</DIV><DIV class="container"><H4 class="click-pro">Cell Counting</H4><DIV class="pro-detail" style="display: none;"><P style="font-size: 120%;font-weight: bolder">Materials</P><P>Fibro Media warm</P><P>Hemocytometer with cover slip</P><P>Eppendorf pipettes</P><P>0.1% Trypan blue</P><P>Cell suspension</P><P>Microcentrifuge tube</P><P>Microscope</P><P style="font-size: 120%;font-weight: bolder">Procedure</P><P>1. Warm Fibro Media in water bath.</P><P>2. Spray down hood with alcohol.</P><P>3. Spray hemocytometer and cover slip with alcohol and allow to air dry.</P><P>4. Using appropriate Eppendorf pipettes, combine 450 ul of 0.1% trypan blue and 50 ul of suspended cells in a microcentrifuge tube. Mix well.</P><P>5. Inject 50 ul of trypan blue/cell mixture into each side of hemocytometer.</P><P>6. Count cells in 5 out of the 9 boxes of the grid using 10x objective with phase contrast. Only two of the four sides of a box should be counted. If a cell is stained blue, then it is dead and should not be counted.</P><P>7. Count both sides of the hemocytometer. Add the totals and multiply by 10,000 for amount of cells per ml.</P><P>8. Decide how many cells per ml of media are needed. Add appropriate amount of media for desired concentration of cells.</P></DIV></DIV><DIV class="container"><H4 class="click-pro">CRISPR gene modification</H4><DIV class="pro-detail" style="display: none;"><P>1. Acquire the genomic sequence of target gene from NCBI or other databases. </P><P>2. Identify the Exon containing the start codon (ATG) and find the sequence 100bp downstream of ATG. </P><P>3. Design sgRNA using online tool developed by Zhang Lab( Ran, et al, 2013). </P><P>4. Synthesize sgRNA oligos with the following format:
Top: 5’-CACCgNNNNNNNNNNNNNNNNNNNN-3’
Bottom: 5’-AAACNNNNNNNNNNNNNNNNNNNNc-3’
Note: Bbs1 enzyme digestion site. g is only added when the first base it not g.</P><P>5. Suspend oligos in H2O at 100 uM. </P><P>6. Anneal the oligos (20 ul):
Top:                                              9 ul
Bottom:                                 9 ul  
10X annealing buffer:          2 ul
Incubate at 95 oC for 5 min
Cool down to 25 oC at 5 oC/min
10X annealing buffer (100 mM Tris-HCl, pH 8.0, 10 mM EDTA, pH 8.0, 1 M NaCl)
</P><P>Then followed by Ligation in Molecular Cloning.</P></DIV></DIV><DIV class="container"><H4 class="click-pro">Cell Transfection</H4><DIV class="pro-detail" style="display: none;"><P>1. Cell seeding and Cell density at transfection. Cells should be plated 18–24 hours before transfection to ensure that the cells are actively dividing and reach the appropriate cell density (generally 70–90% confluence) at the time of transfection. </P><P>2. Cell passage number. It is critical to transfect primary cells at early passages. </P><P>3. DNA expression vector selection. Promoter (CMV vs. EF1α), type of GFP (Enhanced GFP vs. Emerald GFP) and size of expression vector (5 kb vs. 8 kb) dramatically affect transgene expression in particular cell types. </P><P>4. DNA Preparation. Plasmid DNA must be sterile and free from phenol and other contaminants. </P><P>5. Ratio of GeneXPlus Reagent to DNA. Depending on the cell type, the optimal ratio of DNA (μg) to GeneXPlus Transfection Reagent (μL) varies from 1:1 to 1:3. A DNA (μg) to reagent (μL) ratio of 1:2 is recommended as a starting point. </P><P>6. Complex formation conditions. Prepare GeneXPlus Transfection Reagent and DNA complexes in serum-free growth medium (e.g. Opti-MEM Medium). </P><P>7. Presence of antibiotics and other known inhibitors: Antibiotics can inhibit transfection complex formation and therefore should be excluded from the complex formation step. However, GeneXPlus has been optimized for intracellular delivery of nuclei acids into cultured mammalian cells in the presence of serum. Culture medium containing polyanions such as heparin, heparin sulfate or dextran sulfate can also inhibit transfection. Medium containing these chemicals should not be used for transfection; however, the medium can be replaced with medium containing polyanions 24 hours after transfection. </P><P>8. Post-transfection incubation time. The optimal incubation time is generally 24–72 hours post transfection, but will vary depending on the goal of the experiment, nature of the plasmid used, and cell doubling time.</P></DIV></DIV><DIV class="container"><H4 class="click-pro">Flow cytometry</H4><DIV class="pro-detail" style="display: none;"><P style="font-size: 120%;font-weight: bolder">Reagents and equipment：</P><P>Fc Receptor Blocking Reagents (These include Fc receptor blocking antibodies or IgG solutions)</P><P>Flow Cytometry Red Blood Cell Lysis Solution</P><P>Flow Cytometry Human Lyse Buffer (10X; Catalog # FC002) or Flow Cytometry Mouse Lyse Buffer (10X; Catalog # FC003) or equivalent</P><P>Flow Cytometry Staining Buffer</P><P>R&amp;D Systems, Catalog # FC001, or an equivalent solution containing BSA and sodium azide</P><P>Fluorochrome-conjugated antibodies suitable for use in flow cytometry</P><P>Isotype Control Antibodies </P><P>Materials Required：</P><P>FACS™ Tubes (5 mL round-bottom polystyrene tubes)</P><P>Pipette Tips and Pipettes</P><P>Centrifuge</P><P>Vortex</P><P style="font-size: 120%;font-weight: bolder">Procedure</P><P>Sample preparation: </P><P>For staining peripheral blood cells, whole blood should be collected in evacuated tubes containing EDTA or heparin as the anticoagulant. Contaminating serum components should be removed by washing the cells three times in an isotonic phosphate buffer (supplemented with 0.5% BSA) by centrifugation at 500 x g for 5 minutes. For staining cell lines or cells from activated cell cultures, the cells should be centrifuged at 500 x g for 5 minutes and washed three times in an isotonic PBS buffer (supplemented with 0.5% BSA) to remove any residual growth factors that may be present in the culture medium. Adherent cell lines may require pretreatment with 0.5 mM EDTA to facilitate removal from their substrates. Cells that require trypsinization to enable removal from their substrates should be further incubated in medium for 6-10 hours on a rocker platform to enable regeneration of the receptors. The use of the rocker platform will prevent reattachment to the substrate.</P><P><SPAN style="font-size: 120%;font-weight: bolder">Note</SPAN>: Titration experiments should be performed to determine optimal reagent amounts.</P><P>Harvest cells and aliquot up to 1 x 106 cells/100 μL into FACS tubes. Fc-block cells with blocking IgG (1 μg IgG/106 cells) for 15 minutes at room temperature.</P><P><SPAN style="font-size: 120%;font-weight: bolder">Note</SPAN>: Do not wash excess blocking IgG from this reaction.</P><P>Add conjugated antibody (10 μL/106 cells, or a previously titrated amount) and vortex. Incubate cells for 30 minutes at room temperature in the dark.</P><P>Remove any unbound antibody by washing the cells in Flow Cytometry Staining Buffer. Centrifuge the suspended cells at 300 x g for 5 minutes and decant the buffer. Resuspend the cells by adding 2 mL of Flow Cytometry Staining Buffer. Repeat the wash two times.</P><P><SPAN style="font-size: 120%;font-weight: bolder">Note</SPAN>: If using whole blood, samples should go through a red blood cell lysis step at this point using Flow Cytometry Human or Mouse Lysis Buffer.</P><P>Lysis of Red Blood Cells: Add 2 mL of 1X Human (Catalog # FC002) or Mouse (Catalog # FC003) Lyse Buffer to each tube, vortex, and incubate in the dark at room temperature for 10 minutes. Centrifuge and wash cells in Flow Cytometry Staining Buffer as described in step 4 above.</P><P><SPAN style="font-size: 120%;font-weight: bolder">Note</SPAN>: If an unconjugated primary antibody was used, incubation with an appropriate secondary antibody should occur now. Dilute the secondary antibody in Flow Cytometry Staining Buffer, starting with the suggested concentration in the product datasheet. Incubate for 20-30 minutes in the dark and wash as in step 4.</P><P>Resuspend the cells in 200 – 400 μL of Flow Cytometry Staining Buffer for final flow cytometric analysis.</P><P><SPAN style="font-size: 120%;font-weight: bolder">Note</SPAN>: For a negative control, a separate set of cells should be stained with an isotype control antibody using the steps outlined above.</P></DIV></DIV><DIV class="container"><H4 class="click-pro">Gibson Assembly</H4><DIV class="pro-detail" style="display: none;"><P>1.	Set up the following reaction on ice:</P><TABLE><TBODY><TR><TD colspan="3">Recommended Amount of Fragments Used for Assembly</TD></TR><TR><TD>2-3 Fragment Assembly</TD><TD>4-6 Fragment Assembly</TD><TD>Positive Control**</TD></TR><TR><TD>Total Amount of Fragments</TD><TD>0.02–0.5 pmols*X μl</TD><TD>0.2–1 pmols*X μl</TD><TD>10 μl</TD></TR><TR><TD>Gibson Assembly Master Mix (2X)</TD><TD>10 μl</TD><TD>10 μl</TD><TD>10 μl</TD></TR><TR><TD>Deionized H2O</TD><TD>10-X μl</TD><TD>10-X μl</TD><TD>0</TD></TR><TR><TD>Total Volume</TD><TD>20 μl***</TD><TD>20 μl***</TD><TD>20 μl</TD></TR></TBODY></TABLE><P>* Optimized cloning efficiency is 50–100 ng of vectors with 2–3 fold of excess inserts. Use 5 times more of inserts if size is less than 200 bps. Total volume of unpurified PCR fragments in Gibson Assembly reaction should not exceed 20%.</P><P>** Control reagents are provided for 5 experiments.</P><P>*** If greater numbers of fragments are assembled, additional Gibson Assembly Master Mix may be required.</P><P>2.	Incubate samples in a thermocycler at 50°C for 15 minutes when 2 or 3 fragments are being assembled or 60 minutes when 4-6 fragments are being assembled. Following incubation, store samples on ice or at –20°C for subsequent transformation.</P><P>Note: Extended incubation up to 60 minutes may help to improve assembly efficiency in some cases (for further details see FAQ section).</P><P>3.	Transform NEB 5-alpha Competent E. coli cells (provided with the kit) with 2 μl of the assembly reaction, following the transformation protocol.</P></DIV></DIV><DIV class="container"><H4 class="click-pro">Molecular cloning</H4><DIV class="pro-detail" style="display: none;"><P style="font-size: 120%;font-weight: bolder">Plasmid DNA extraction</P><P>1. 5 ml LB medium containing proper antibiotics were inoculated with a single bacterial colony. The tube was incubated at 37 ˚C overnight with vigorous shaking at 360 rpm. </P><P>2. Pellet bacteria from the culture at 10,000 x g for 5 minutes at room temperature. </P><P>3. Discard the supernatant. </P><P>4. Resuspend bacterial pellet in a total of 1 ml ice-cooled solution I (50 mM). Pipet up and down or vortex as necessary to fully resuspend the bacteria. </P><P>5. Add 2 ml room temperature 0.2 N NaOH/1.0% SDS to the suspension. Mix thoroughly by repeated gentle inversion. Do not vortex. </P><P>6. Add 1.5 ml ice-cold Solution III to the lysate. Mix thoroughly by repeated gentle inversion. Do not vortex. </P><P>7. Centrifuge at 15,500 x g for 30 minutes at 4C. </P><P>8. Recover resulting supernatant. </P><P>9. Add 2.5 volume isopropanol to precipitate the plasmid DNA. Mix thoroughly by repeated gentle inversion. Do not vortex. </P><P>10. Centrifuge at 15,500 x g for 30 minutes at 4C. </P><P>11. Removal of resulting supernatant. The pellet is plasmid DNA. </P><P>12. Rinse the pellet in ice-cold 70% EtOH and air-dry for about 10 minutes to allow the EtOH to evaporate. </P><P>13. Add ddH2O or TE to dissolve the pellet. After addition of 2ul RNase A (10mg/ml), the mixture was incubated for 20 minutes at room temperature to remove RNA.</P><P>(Note: plasmid isolation kit can be used)</P><P style="font-size: 120%;font-weight: bolder">Enzyme digestion</P><P>1.Add components to a clean tube in the order shown:</P><P style="text-indent:4em">1 µL DNA (concentration 1 µg/µL)</P><P style="text-indent:4em">2 µL 10x buffer</P><P style="text-indent:4em">1 µL restriction enzyme</P><P style="text-indent:4em">16 µL sterile water</P><P>2.Incubate the reaction at digestion temperature (usually 37°C) for 1 hour.</P><P>3.Stop the digestion by heat inactivation (65°C for 15 minutes) or addition of 10mM final concentration EDTA.</P><P>4.The digested DNA is ready for use in research applications. </P><P style="font-size: 120%;font-weight: bolder">Agarose gel electrophoresis</P><P>1. Pour enough running buffer into the electrophoresis tank. (The surface should be higher than the top of the gel and not overflow)</P><P>2. Prepare the suitable concentration of agarose solution and microwave it untill a boiling. Add in 5-10µl of gold view dye when it's cooled down to 55°C</P><P>3. Choose the suitable gel tank, pour the fluid agarose gel and insert the comb</P><P>4. After a complete solidification put the agarose gel in the electrophoresis tank.</P><P>5. Mix the sample with loading buffer sufficiently and load them into the sample lane together with the marker (usually marker in the first lane).</P><P>6. Set an appropriate voltage and run the electrophoresis.</P><P>7. After approximately 35min (80V), 25min (100V), put the agarose gel in an UV detector and record the picture.</P><P style="font-size: 120%;font-weight: bolder">Gel extraction</P><P>1.Weigh enough microcentrifuge tubes for the number of gel slices you will cut out.</P><P>2.Using gel extraction kit</P><P style="font-size: 120%;font-weight: bolder">T4 ligation</P><P>Set up the following reaction in a microcentrifuge tube on ice.</P><P>(T4 DNA Ligase should be added last. Note that the table shows a ligation using a molar ratio of 1:3 vector to insert for the indicated DNA sizes.) </P><P style="text-indent:4em">COMPONENT	20μl REACTION</P><P style="text-indent:4em">T4 DNA Ligase Buffer (10X)	2μl</P><P style="text-indent:4em">Vector DNA (4 kb)	50 ng (0.020pmol)</P><P style="text-indent:4em">Insert DNA (1 kb)	37.5 ng (0.060pmol)</P><P style="text-indent:4em">Nuclease-free water	to 20μl</P><P style="text-indent:4em">T4 DNA Ligase	1μl</P><P>The T4 DNA Ligase Buffer should be thawed and resuspended at room temperature.</P><P>Gently mix the reaction by pipetting up and down and microfuge briefly.</P><P>For cohesive (sticky) ends, incubate at 16°C overnight or room temperature for 10 minutes.</P><P>For blunt ends or single base overhangs, incubate at 16°C overnight or room temperature for 2 hours (alternatively, high concentration T4 DNA Ligase can be used in a 10minutes ligation).</P><P>Heat inactivate at 65°C for 10 minutes.</P><P>Chill on ice and transform 1-5μl of the reaction into 50μl competent cells.</P><P style="font-size: 120%;font-weight: bolder">Transformation</P><P>1. Thaw competent cells on ice.</P><P>2. Chill about 5 ng (2μl) of the ligation mixture in a 1.5 ml microcentrifuge tube.</P><P>3. Add 50 µl of competent cells to the DNA. Mix gently by pipetting up and down or flicking the tube 4–5 times to mix the cells and DNA. Do not vortex.</P><P>4. Place the mixture on ice for 30 minutes. Do not mix.</P><P>5. Heat shock at 42°C for 45 seconds. Do not mix.</P><P>6. Add 950 µl of room temperature media to the tube. </P><P>7. Place tube at 37°C for 60 minutes. Shake vigorously (250 rpm) or rotate. </P><P>8. Warm selection plates to 37°C.</P><P>9. Spread 50–100 µl of the cells and ligation mixture onto the plates. </P><P>10. Incubate overnight at 37°C.</P><P>11. Pick single clones and grow in growth medium (like LB) supplied by appropriate antibiotics for about 14-16 hours. </P></DIV></DIV><DIV class="container"><H4 class="click-pro">Mycoplasma Detection</H4><DIV class="pro-detail" style="display: none;"><P style="font-size: 120%;font-weight: bolder">Gel electrophoresis </P><P>1.	Prepare a 3% agarose gel.</P><P>2.	Prepare samples: Add 10 μL of the PCR product to 1.5 μL loading buffer. Mix thoroughly.</P><P>3.	Load samples and a DNA marker (e.g., 100 bp ladder) onto the gel.</P><P>4.	Electrophorese until the tracking dye migrates 60-70% the length of the gel.</P><P>5.	Stain the gel with ethidium bromide or similar stain and view with UV illumination.</P><P>Results: A test sample that is positive for the presence of mycoplasma shows a distinct band at 434 to 468 bp. The positive control samples exhibit a 464-bp band. There should be no visible band in the negative control lane.</P><P style="font-size: 120%;font-weight: bolder">Sample preparation </P><P>1. Cell Harvest:
			</P><P>A. Suspension cells: Count cells. 104 - 105 cells are needed for the assay.</P><P>B. Adherent cells: Scrape the cells into the existing culture media and suspend. Do not treat cells with trypsin or EDTA as these agents disrupt mycoplasma.</P><P>2. Transfer 1 mL cell suspension (104 to 105 cells) into the Sample Lysis Tubes and centrifuge at 13,000 rpm for 3 minutes at 4°C. 
			</P><P>3. Carefully remove and discard the supernatant.</P><P>4. Resuspend the cell pellet with 50 ul Lysis Buffer by vortexing.</P><P>5. Incubate the resuspended cell pellet at 37°C for 15 minutes to lyse the cells and degrade the proteins.</P><P>6. Heat the samples at 95°C for 10 minutes to inactivate the protease.</P><P>7. Spin down cell debris at 13,000 rpm for 5 minutes at 4°C. Transfer supernatant to a new microcentrifuge tube. Do NOT use the tubes provided with the kit as these are needed for remaining kit assays.</P><P>8. Samples are now ready for PCR. If desired, these extracts may be stored at -80°C for up to six months.</P><P style="font-size: 120%;font-weight: bolder">PCR preparation</P><P>1.	Prepare a PCR + Primers Mix by combining universal PCR Mix with universal primers.</P><P>2.	Prepare the reaction mixtures in PCR tubes.</P><P>3.	Mix gently by pipetting the reaction mixes up and down a few times. Cap tubes and centrifuge briefly to bring fluid to the bottom of the tube.</P><P>4.	Place the tubes in a thermal cycler.</P><P>5.	Set the temperature and cycling parameters for PCR.</P></DIV></DIV><DIV class="container"><H4 class="click-pro">Stable Cell Lines Construction</H4><DIV class="pro-detail" style="display: none;"><P>Step 1: Transfect the cells using the desired transfection method. If the selectable marker is on a separate vector, use a 5:1 to 10:1 molar ratio of plasmid containing the gene of interest to plasmid containing the selectable marker.</P><P>Step 2: Forty-eight hours after transfection, passage the cells at several different dilutions (e.g., 1:100, 1:500) in medium containing the appropriate selection drug. For effective selection, cells should be subconfluent, because confluent, non-growing cells are resistant to the effects of antibiotics like Geneticin. Suspension cells can be selected in soft agar or in 96-well plates for single-cell cloning.For the next two weeks, replace the drug-containing medium every 3 to 4 days (or as needed).</P><P>Step 3: During the second week, monitor cells for distinct “islands” of surviving cells. Depending on the cell type, drug-resistant clones will appear in 2–5 weeks. Cell death should occur after 3–9 days in cultures transfected with the negative control plasmid.</P><P>Step 4: Isolate large (500–1,000 cells), healthy colonies using cloning cylinders or sterile toothpicks, and continue to maintain cultures in medium containing the appropriate drug (for the isolation of clones in suspension culture, see Freshney, 1993).</P><P>Step 5: Transfer single cells from resistant colonies into the wells of 96-well plates to confirm that they can yield antibiotic-resistant colonies. Ensure that only one cell is present per well after the transfer.</P><P>For more details: <A href="https://static.igem.org/mediawiki/2018/b/bc/T--SUSTech_Shenzhen--notebook.pdf">Stable Transfection | Thermo Fisher Scientific - CN</A></P></DIV></DIV><DIV class="container"><H4 class="click-pro">Suspension cell culture</H4><DIV class="pro-detail" style="display: none;"><P>1.Take 150/250 ml conical flask containing autoclaved 40/60 ml liquid medium</P><P>2. Transfer 3-4 pieces of pre-established callus tissue (approx. wt. 1 gm. each) from the culture tube using the spoon headed spat­ula to conical flasks.</P><P>3. Flame the neck of conical flask, close the mouth of the flask with a piece of allumini­um foil or a cotton plug. Cover the closure with a piece of brown paper.</P><P>4. Place the flasks within the clamps of a ro­tary shaker moving at the 80-120 rpm (rev­olution per minute)</P><P>5. After 7 days, pour the contents of each flask through the sterilized sieve pore diameter -60µ- 100µ and collect the filtrate in a big sterilized container. The filtrate contains only free cells and cell aggregates.</P><P>6. Allow the filtrate to settle for 10-15 min. or centrifuge the filtrate at 500 to 1,000 rpm and finally pour off the supernatant.</P><P>7. Re-suspend the residue cells in a requisite volume of fresh liquid medium and dispense the cell suspension equally in several ster­ilized flasks (150/250 ml). Place the flasks on shaker and allow the free cells and cell aggregates to grow.</P><P>8. At the next subculture, repeat the previous steps but take only one-fifth of the residual cells as the inoculum and dispense equally in flasks and again place them on shaker.</P><P>9. After 3-4 subcultures, transfer 10 ml of cell suspension from each flask into new flask containing 30 ml fresh liquid medium.</P><P>10. To prepare a growth curve of cells in sus­pension, transfer a definite number of cells measured accurately by a hemocytometer to a definite volume of liquid medium and incubates on shaker. Pipette out very little aliquot of cell suspension at short intervals of time (1 or 2 days interval) and count the cell number. Plot the cell count data of a passage on a graph paper and the curve will indicate the growth pattern of suspension culture.</P></DIV></DIV><DIV class="container"><H4 class="click-pro">Viral infection</H4><DIV class="pro-detail" style="display: none;"><P>Day 1: Plate 250,000--‐500,000 cells in one well of a 6--‐well plate.</P><P>Day 2: Mix 100 ul un-concentrated virus (or 2 ul concentrated virus) in a tube with 900 ul regular growth media and 1 ul polybrene (10 mg/ml stock). Vortex. Aspirate media from your cells. Replace with virus-containing media.</P><P>Day 3: If your construct contains a fluorescent reporter, check cells. Please note that it can take up to two days for cells to transduce. Depending on the confluency of your cells, remove virus-containing media, and either refresh with new media or passage your cells.</P></DIV></DIV><DIV class="container"><H4 class="click-pro">Western Blot</H4><DIV class="pro-detail" style="display: none;"><P style="font-size: 120%;font-weight: bolder">Sample prep (based on a typical cell culture scenario)</P><P>1.	Place the cell culture dish in ice and wash the cells with ice-cold Tris-buffered saline (TBS).</P><P>2.	Aspirate the TBS, then add ice-cold RIPA buffer (1 ml per 100 mm dish).</P><P>3.	Scrape adherent cells off the dish using a cold plastic cell scrape and gently transfer the cell suspension into a precooled microcentrifuge tube.</P><P>4.	Maintain constant agitation for 30 min at 4°C.</P><P>5.	If necessary, sonicate 3 times for 10-15 sec to complete cell lysis and shear DNA to reduce sample viscosity.</P><P>6.	Spin at 16,000 xg for 20 min in a 4°C precooled centrifuge.</P><P>7.	Gently remove the centrifuge tube and place it on ice. Transfer the supernatant to a fresh tube, also kept on ice, and discard the pellet.</P><P>8.	Remove a small volume (10-20 ul) of lysate to perform a protein assay. Determine the protein concentration foreach cell lysate.</P><P>9.	If necessary, aliquot the protein samples for long-term storage at -20°C. Repeated freeze and thaw cycles cause protein degradation and should be avoided.</P><P>10.	Take 20 ug of each sample and add an equal volume of 2x Laemmli sample buffer.</P><P>11.	Boil each cell lysate in sample buffer at 95°C for 5 min.</P><P>12.	Centrifuge at 16,000xg in a microcentrifuge for 1 min.</P><P style="font-size: 120%;font-weight: bolder">Protein separation by gel electrophoresis</P><P>1.	Load equal amounts of protein (20 g) into the wells of a mini (8.6 *6.7 cm) or midi (13.3*8.7 cm) format SDS-PAGE gel, along with molecular weight markers.</P><P>2.	Run the gel for 5 min at 50V.</P><P>3.	Increase the voltage to 100-150 V to finish the run in about 1 hr.</P><P style="font-size: 120%;font-weight: bolder">Transferring the protein from the gel to the membrane</P><P>1.	Place the gel in 1x transfer buffer for 10-15 min.</P><P>2.	Assemble the transfer sandwich and make sure n air bubbles are trapped in the sandwich. The blot should be on the cathode and the gel on the anode.</P><P>3.	Place the cassette in the transfer tank and place an ice block in the tank.</P><P>4.	Transfer overnight in a cold room at a constant current of 10 mA.</P><P style="font-size: 120%;font-weight: bolder">Antibody incubation</P><P>1.	Briefly rinse the blot in water and stain it with Ponceau S solution to check the transfer quality.</P><P>2.	Rinse off the Ponceau S stain with three washes with TBST.</P><P>3.	Block in 3% BSA in TBST at room temperature for 1 hr.</P><P>4.	Incubate overnight in the primary antibody solution against the target protein at 4°C.</P><P>5.	Rinse the blot 3-5 times for 5 min with TBST.</P><P>6.	Incubation in the HRP-conjugated secondary antibody solution for 1 hour at room temperature.</P><P>7.	Rinse the blot 3-5 times for 5 min with TBST.</P><P style="font-size: 120%;font-weight: bolder">Imaging and data analysis</P><P>1.	Apply the chemiluminescent substrate to the blot according to the manufacturer’s recommendation.</P><P>2.	Capture the chemiluminescent signals using a CCD camera-based imager.</P><P>3.	Use image analysis software to read the band intensity of the target proteins.</P><P style="font-size: 120%;font-weight: bolder">Stripping and reprobing</P><P>1.	Warm the buffer to 50°C.
			</P><P>2.	Add the buffer to the membrane in a container designated for stripping. Incubate at 50°C for up to 45 min with some agitation.</P><P>3.	Rinse the blot under running water for 1 hr.</P><P>4.	Transfer the membrane to a clean container, wash 5 times for 5 min with TBST.</P><P>5.	Block in 3% BSA in TBST at room temperature for 1 hr.</P><P>6.	Incubate overnight in the primary antibody solution at 4°C.</P><P>7.	Rinse the blot 3-5 times for 5 min with TBST.</P><P>8.	Incubation in the HRP-conjugated secondary antibody solution for 1hr at room temperature.</P><P>9.	Rinse the blot 3-5 times for 5 min with TBST.</P><P style="font-size: 120%;font-weight: bolder">Imaging and data analysis</P><P>1.	Apply the chemiluminescent substrate to the blot following the manufacture’s suggestions.</P><P>2.	Capture the chemiluminescent signals using a CCD camera-based imager.</P><P>3.	Use image analysis software to read the band intensity of the loading control proteins.</P><P>4.	Use the loading control protein levels to normalize the target protein levels.</P></DIV></DIV></DIV><DIV class="footer"><DIV class="col-md-3 contact-left"><H4>Address</H4><P><A href="http://www.sustc.edu.cn/en">
						  No 1088, Xueyuan Rd., Xili, 
						  Nanshan District, Shenzhen, 
						  Guangdong, China
					</A></P></DIV><DIV class="col-md-3 contact-left"><H4>Contact us</H4><P><A href="mailto:igem@sustc.edu.cn">Mail to us</A></P><P><A href="https://twitter.com/iGEM_SUSTech">Twitter</A></P></DIV><DIV class="copyright">
			Licensed under <A href="http://creativecommons.org/licenses/by/4.0/">CC BY 4.0</A>.
			   ©2018 SUSTech_Shenzhen
		</DIV></DIV></BODY></HTML>