<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_Grenoble-Alpes_Demonstrate skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><DIV id="top_title"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:Grenoble-Alpes/Demonstrate</SPAN></H1></DIV><DIV id="HQ_page"><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><DIV class="navi"><LABEL for="menumobile" class="menumobile">MENU</LABEL><UL><LI class="menu"><A href="https://2017.igem.org/Team:Grenoble-Alpes">Home</A></LI><LI class="menu"><A href="https://2017.igem.org/Team:Grenoble-Alpes/Team">Team</A></LI><LI><A href="https://2017.igem.org/Team:Grenoble-Alpes/Team">Members</A></LI><LI><A href="https://2017.igem.org/Team:Grenoble-Alpes/Attributions">Attributions</A></LI></UL><LI class="menu"><A href="https://2017.igem.org/Team:Grenoble-Alpes/Description"> Project </A></LI><LI><A href="https://2017.igem.org/Team:Grenoble-Alpes/Description">Description</A></LI><LI><A href="https://2017.igem.org/Team:Grenoble-Alpes/Demonstrate">Results</A></LI><LI class="menu"><A href="https://2017.igem.org/Team:Grenoble-Alpes/LabBook">Lab</A></LI><LI><A href="https://2017.igem.org/Team:Grenoble-Alpes/LabBook">LabBook</A></LI><LI><A href="https://2017.igem.org/Team:Grenoble-Alpes/Protocols">Protocols</A></LI><LI><A href="https://2017.igem.org/Team:Grenoble-Alpes/Composite_Part">Composite part</A></LI><LI><A href="https://2017.igem.org/Team:Grenoble-Alpes/Parts">Parts</A></LI><LI class="menu"><A href="https://2017.igem.org/Team:Grenoble-Alpes/Measurement">Engineering</A></LI><LI><A href="https://2017.igem.org/Team:Grenoble-Alpes/Measurement">Detection</A></LI><LI><A href="https://2017.igem.org/Team:Grenoble-Alpes/Fluid_Transfer">Fluid Transfer</A></LI><LI><A href="https://2017.igem.org/Team:Grenoble-Alpes/Temperature_Control">Temperature control</A></LI><LI><A href="https://2017.igem.org/Team:Grenoble-Alpes/Hardware">Hardware</A></LI><LI class="menu"><A href="https://2017.igem.org/Team:Grenoble-Alpes/Software">App</A></LI><LI class="menu"><A href="https://2017.igem.org/Team:Grenoble-Alpes/Safety">Safety</A></LI><LI class="menu"><A href="https://2017.igem.org/Team:Grenoble-Alpes/HP/Silver">Human Practices</A></LI><LI><A href="https://2017.igem.org/Team:Grenoble-Alpes/HP/Silver">Human Practices</A></LI><LI><A href="https://2017.igem.org/Team:Grenoble-Alpes/HP/Gold_Integrated">Integrated Human Practices</A></LI><LI><A href="https://2017.igem.org/Team:Grenoble-Alpes/Collaborations">Collaborations</A></LI><LI class="menu"><A href="https://2017.igem.org/Team:Grenoble-Alpes/Notebook">Notebook</A></LI><LI><A href="https://2017.igem.org/Team:Grenoble-Alpes/Notebook">Notebook</A></LI><LI><A href="https://2017.igem.org/Team:Grenoble-Alpes/Medals">Medal Criteria</A></LI></DIV><TITLE> TEAM </TITLE><HEADER><H3> Our results </H3></HEADER><DIV class="popup_geoloc" onclick="Function_geo()"><SPAN class="popuptext" id="Popup_geo"> The Aiguilles d’Arves are not only the apex the Maurienne valley, they are also its emblem.  Credits: Estelle Vincent
                  </SPAN></DIV><H3 style="text-align:center; color:#F49C20; font-size:2.5vw; padding-top:1%; padding-bottom:1.75%; font-weight:bold;"><A style="color:#F49C20; text-decoration:none;" href="#ancre_main">Main results</A>  <A style="color:#F49C20; text-decoration:none;" href="#ancre_proof">The proof of concept</A>  <A style="color:#F49C20; text-decoration:none;" href="#ancre_impro">Improvements</A></H3><DIV style="background:#F49C20; margin-top:3%; margin-bottom:1%; padding-bottom: 1%;"><H3 id="ancre_main" style="text-align:center; color:white; font-size:2.5vw; padding-top:2%; padding-bottom:1.75%;"> Main results </H3></DIV><DIV style="padding-bottom:0.5%;"><H5> SnapLab project was divided into five main missions : establishing an extraction process to get the target, constructing the DNA detector, finding solutions for the conservation of the biological content within the kit, creating a detection system using a smartphone to give a diagnosis only with the portable kit and adapting the kit to biological constraints in parallel of its modelisation.</H5></DIV><DIV style="text-align:center; padding-bottom:2.5%; padding-top:1%;"><VIDEO src="https://static.igem.org/mediawiki/2017/0/00/GrenobleAlpesfinalanalysis.mp4" poster="https://static.igem.org/mediawiki/2017/a/a3/4_Image_Kit_Analyse_Vue_Eclatee_ok.png" controls="" width="50%">
            Your browner cannot show this video.
            </VIDEO></DIV><H2 id="ancre_target" style="padding-top:2%; padding-bottom:1%; color:#F49C20; font-size:2vw;"> About the target preparation </H2><H5> About the extraction, the problematic was the following: how to extract, from a stool sample, the small DNA target sequence ? Investigations were done about <I>Vibrio cholerae’s</I> functioning. How does it grow in a laboratory ? What are the tanks, how does contamination happen ? What is its physiopathology ? In parallel, current techniques for DNA extraction were reviewed. Finally, a few protocols for <I>Vibrio cholerae</I> (and especially CTX genome) extractions were established thanks to these data. For safety reasons, practical tests were done on E. coli bacteria (strain: DH5α). Also, some parts of the protocol were done in kit-like conditions. For example, steps for which liquids had to go through a silica membrane were done thanks to a piston, to avoid centrifugation, since it would not be possible within the kit. Results were very promising and the main conclusions are : (i) extracting genomic DNA without centrifugation is possible and (ii) elution with TE buffer gives better concentrations than with water. </H5><H5> At the end of the extraction process, <I>Vibrio cholerae</I> DNA is theoretically digested with AluI enzyme, then the targeted sequence is isolated and denatured. At this stage, <STRONG>the target is ready to be detected!</STRONG></H5><H2 id="ancre_plasmid" style="padding-top:2%; padding-bottom:1%; color:#F49C20; font-size:2vw;"> About the construction of the plasmid detector </H2><H5>The second biological mission was the construction of the plasmid detector. Inspired by the genius of <A style="text-decoration: none; color:#F49C20;" href="https://2015.igem.org/Team:Cork_Ireland">Cork Ireland iGEM 2015</A> team, who designed an amazing DNA detector for Human Papillomavirus, Grenoble Alpes 2017 team created a similar biological system to detect the specific target sequence of <I>Vibrio cholerae</I> (more precisely: 39bp found in RstA gene of bacteriophage CTX). A probe was designed according to the target sequence, and inserted into an appropriate plasmid backbone carrying a fluorescent reporter gene. This insertion was more complicated than expected. The main conclusions for this mission are : 
<LI>(i) an insertion of sequence inside a plasmid vector through a unique restriction site requires at least three dephosphorylation steps to avoid self-ligation</LI><LI>(ii) the best ratio insert:vector in that case appeared to be a molar ratio 1:1 </LI><LI>(iii) an insertion between two different restriction sites is much easier than one in a unique site </LI>
The constructed detector was activated and then tested with the target, with successful results. Few false positives were observed but were suspected to be due to a bad digestion of the detector probe prior to the contact with the target. Still, there is a notable difference of colonies’ number when the target is present or not. 
Evaluation of sensitivity and specificity was also attempted. Sensitivity, which is the limit of detection of the test, was tested through the counting of transformed bacteria. An important difference between absence or presence of the target was observed as soon as 6ng of target was in contact with 100ng of detector. However, a sequencing of all colonies would have been more accurate, to ensure that the counting was not taking false positives into account. Specificity, which is the proportion of positives identified as such, brought more issues. The tests consisted in the evaluation of the detector’s ability to recognize only the original target and not the false sequences more or less homologous. Unfortunately, results were too heterogeneous to draw any conclusions. </H5><H5>As a conclusion, a <A style="text-decoration: none; color:#F49C20;" href="http://parts.igem.org/Part:BBa_K2299000">new biobrick</A> was built for which the proof of concept for <STRONG>its ability to detect cholera’s pathogen DNA was successfully achieved</STRONG>. However, its characterization should be completed by statistical analyses.</H5><H2 id="ancre_cons" style="padding-top:2%; padding-bottom:1%; color:#F49C20; font-size:2vw;"> About the detector and bacteria’s conservation</H2><H5> The last biological mission was a huge challenge: finding a way to conserve the DNA detector, the enzymes and the competent bacteria within the kit, i.e at ambient temperature, so they would all keep their respective properties. </H5><H5> About the plasmid detector, it is a chance that DNA is a relatively stable molecule. Few tests were made on BBa_J04450 plasmid to dry it:
<LI>in a spectrophotometer cuvette, overnight, at RT, under fume hood; </LI><LI>in an open petri dish, overnight, at RT, under fume hood; </LI><LI>in a 1,5mL tube, for 40 min, at RT, with a speed-vacuum. </LI>
After 35 days of waiting, the resuspension and transformation of E. coli bacteria (strain: top 10) were realised. Results showed that the three drying methods were comparatively efficient. Indeed, even one month after the drying and RT conservation, transformation gave lots of colonies, even if the number was reduced in comparison to the positive control (small loss of DNA during the drying process). Finally the speed-vacuum method was favoured because of its rapidity.The conservation of the enzymes appeared to be a real issue. Unfortunately, since they are very sensitive to their environment, the tests for their lyophilisation did not give the expected results. However, there is still hope to find a way to conserve them in the kit’s conditions. </H5><H5> Tests for the conservation of the competent bacteria, on the contrary, were very promising. The chosen method was the freeze-drying. Bacteria (E coli JM109) were made competent thanks to a classical treatment of CaCl<SUB>2</SUB> (100mM) and MgCl<SUB>2</SUB> (100mM) but with the addition of a cryoprotective agent, sucrose. After one night of freeze-dry at -50°C in a lyophilisator, transformation tests were done with these bacteria and BBa_J04450 plasmid. The main conclusions are: (i) <STRONG>lyophilisation of competent bacteria is possible</STRONG> , (ii) it leads to a loss of competency of about 35% and (iii) 100mM of sucrose appeared to be the best concentration for a good cryoprotection. </H5><H2 id="ancre_detection" style="padding-top:2%; padding-bottom:1%; color:#F49C20; font-size:2vw;"> About the detection</H2><H5> The first engineering mission Grenoble Alpes 2017 team was to detect the target using the plasmid detector inside a small portable kit. In other words, the kit has to detect cholera by measuring the fluorescence of the feces' sample. Thus the kit has to provide a technique to capture and quantify the fluorescence emitted by the sample, but being as precise as possible to actually give a reliable diagnosis. This is where engineering studies come in, interfering with biological studies!
The fluorescence <A style="text-decoration: none; color:#F49C20;" href="https://2017.igem.org/Team:Grenoble-Alpes/Measurement">detection system</A> created by the Grenoble Alpes team was applied on SnapLab kit. Since it is proved that this system can measure fluorescence, it was useful for two upstream works:
<LI>define a threshold above which the analysis can lead to a positive diagnosis (in fact, false positives have been considered to define the minimum of fluorescence that lead to a positive diagnosis) </LI><LI>define the duration of one analysis</LI></H5><H5>Thanks to this work, a precise analysis has been made, serving as a proof of concept for the entire SnapLab project. </H5><H5 style="font-size:1.5vw;"><I>Threshold</I></H5><H5>To define the threshold, the exact same test presented <A style="text-decoration: none; color:#F49C20;" href="https://2017.igem.org/Team:Grenoble-Alpes/Measurement#detectiontest">here</A> has been made but this time, the plasmid used was not the targeted plasmid but a non targeted plasmid that emits fluorescence. In fact, biological steps reveals that some bacteria grow after the transformation without the presence of targeted plasmid. These bacteria are called false positives and are used in this experiment. The tubes’ composition was as follows.</H5><CENTER><TABLE><CAPTION> Table 1: Tube's composition (volumes in mL)  </CAPTION><TBODY><TR><TD> 1 </TD><TD> 2 </TD><TD> 3</TD><TD> 4 </TD><TD> 5 </TD><TD> 6 </TD><TD> 7 </TD></TR><TR><TH> False positives </TH><TD> 0.00 </TD><TD> 0.025 </TD><TD> 0.05 </TD><TD> 0.075 </TD><TD> 0.1 </TD><TD> 0.125 </TD><TD> 0.150 </TD></TR><TR><TH> Non fluorescent bacteria JM109 </TH><TD> 0.50 </TD><TD> 0.437 </TD><TD> 0.375 </TD><TD> 0.312 </TD><TD> 0.250 </TD><TD> 0.187 </TD><TD> 0.125 </TD></TR><TR><TH> LB </TH><TD> 0 </TD><TD> 0.038 </TD><TD> 0.075 </TD><TD> 0.113 </TD><TD> 0.150 </TD><TD> 0.188 </TD><TD> 0.225 </TD></TR></TBODY></TABLE></CENTER><H5>The processed pictures taken of each tubes gave the following results. What was interesting in this experiment was to compare these results with the ones obtained with <A style="text-decoration: none; color:#F49C20;" href="https://2017.igem.org/Team:Grenoble-Alpes/Measurement#detectiontest">the exact same experiment with targeted plasmid.</A></H5><DIV style="text-align:center; padding-bottom:1%; padding-top:2%;"><H5 class="tfg">Figure 1: Comparison between real positives and false positives</H5></DIV><H5>As it can be seen, the fluorescence of a false positive is much smaller than real positives. By comparing both maximum levels of fluorescence, it is possible to define a threshold, which is <STRONG>the half of the real positives maximum of fluorescence. </STRONG></H5><H5 style="font-size:1.5vw;"><I>Duration of one analysis</I></H5><H5>Once the threshold has been characterised, the determination of the duration of the analysis is possible. To do that, a liquid bacterial transformation has been tracked (final step of the biological part). Directly after realising the heat shock essential to the bacterial transformation, bacteria have been transferred inside an Eppendorf tube, and the smartphone captured bacteria’s emitted fluorescence every 30 minutes. In that way, the level of fluorescence of the tube has been plotted as a function of time. It is important to notice that the tube has been kept at 37°C during the whole analysis. </H5><H5>Thus, the duration of an analysis can be determined. In fact, it is possible to visualize the increase of fluorescence in real time, and then to be able to give a diagnosis after a certain time that is determined here. Photographs have been taken during 5 hours, giving the following results.</H5><DIV style="text-align:center; padding-bottom:1%; padding-top:2%;"><H5 class="tfg">Figure 2: Bacterial transformation tracking </H5></DIV><H5>The fluorescence value corresponding to half its maximum value is obtained after 3 hours. At this stage, it can be affirmed that after the heat shock necessary for the bacterial transformation, one analysis lasts 3 hours. <STRONG>A diagnosis can be made after 3 hours once the biological part is proceeded!</STRONG></H5><H2 id="ancre_kit" style="padding-top:2%; padding-bottom:1%; color:#F49C20; font-size:2vw;"> About the kit itself </H2><H5> The SnapLab kit has been conceived to run all the analysis on its own. First, an extraction column has been modeled to prepare the target. Second, a microfluidic system using pumps has also been modeled to drop the different reagents needed for the analysis itself. In fact, before the bacterial transformation, a few biological steps must be realised within the kit. Then, a temperature control system has been created to lead the heat shock, but also to stabilise the temperature within the kit at 37°C for biological conservation. Thanks to the integrated smartphone and the connected Arduino card, the kit is conceived to be automatised by the SnapLab application. <STRONG>SnapLab kit’s modelling is perfectly fulfilled, making it operational to receive sample to eventually conclude on a diagnosis, when everything is created and assembled.</STRONG></H5><DIV style="background:#F49C20; margin-top:4%; margin-bottom:1%; padding-bottom: 1%;"><H3 id="ancre_proof" style="text-align:center; color:white; font-size:2.5vw; padding-top:2%; padding-bottom:1.75%;"> The proof of concept </H3></DIV><H5> As a final proof of concept of SnapLab kit, Grenoble Alpes 2017 team made an ultimate experiment. A tracking of a bacterial transformation with the kit was once again made but this time more conditions have been respected. The linearised plasmid (plasmid + detector digested) was dried. It was resuspended with the target (following the <A style="text-decoration: none; color:#F49C20;" href="https://2017.igem.org/Team:Grenoble-Alpes/Protocols">protocol</A> for detection). Plasmid was used for transformation with lyophilised competent bacteria. All the biological requirements were respected here, this test was actually the same as a real analysis (except for the DNA extraction part). </H5><H5>To follow the transformation, photos were taken during 5 hours and then processed. But even after this time, no fluorescence was detected. After 72 hours, the team decided to try again and took one picture of the sample inside the kit. This time, fluorescence was actually detected (figure 3), and that led to the final proof of concept. </H5><DIV style="text-align:center; padding-bottom:1%; padding-top:2%;"><H5 class="tfg">Figure 3: Fluorescence of a sample captured, respecting all the kit's conditions</H5></DIV><H5><STRONG>Grenoble Alpes team proved that SnapLab kit is able to detect cholera!</STRONG></H5><DIV style="background:#F49C20; margin-top:4%; margin-bottom:1%; padding-bottom: 1%;"><H3 id="ancre_impro" style="text-align:center; color:white; font-size:2.5vw; padding-top:2%; padding-bottom:1.75%;"> Improvements </H3></DIV><H5> Although the team is really satisfied with the achievements it has accomplished, there are still, as in all projects, many ways to improve SnapLab kit. </H5><H5>The first point is not really an improvement but a necessity. Indeed, even if the SnapLab was entirely modelled with OnShape software, only the detection part has been 3D-printed. Obviously, the first thing to do is thus to print the whole kit (i.e. the extraction part and the pumps).
<LI>Once the extraction part is printed, the work would consist in making several experiments to test the DNA’s extraction from a sample by passing it through the filters. This part has been tested but not in the kit’s conditions.</LI><LI>Concerning the pumps, one has already been printed, and works thanks to the Arduino card acting on a motor. The objective here is still to print the fluid transfer part of the kit, which is composed of two pumps working thanks to motors and the Arduino card. The whole interest is being able to automate the biologic part of the analysis, and to verify whether the pumps are able to deliver a certain amount with enough precision or not. </LI>
Automating the kit is also a major point of improvement. To conserve the biological matter and create a heat shock, the temperature control has to be inserted within the kit. In fact, the electronic system is operational and has been tested on an operating panel. For now, the electronic components have been printed on a surface-mount device (SMD), but its control and integration within the kit, as well as integrating the Peltier device, remains something to do. Another thermal aspect to consider is the heat conduction. In fact, it has to be studied to make sure that all the biological compounds can actually be warmed and cooled on the allocated time. This is essential to make SnapLab kit self-sufficient.</H5><H5>In line with the two previous points, the whole automatisation of SnapLab kit has to be realised.The first step would be to ensure the communication of the Arduino card with the smartphone : all the actions would be coded on the Arduino card and controlled thanks to the application. At this stage, the application sends a signal to the Arduino card when it has to start a program. This is actually what happens with the pumps. The communication is made but not necessarily at the right moment. Plus, some Arduino programs still need to be coded. It is also worth noting that the communication between the Arduino card and the smartphone is one-sided. The phone can communicate with the Arduino card but not the other way around because of the Bluetooth module. For a simple utilisation, this module can be kept, but it is something that will have to be investigated for displaying temperature on the phone, that is to say replace the module with a two-sided communication module. </H5><H5>The fifth point is quite interesting especially for the use of  the kit in remote places like African countries. Instead of plugging the kit to a power source, the idea would be to conceive a system with solar panels. The only thing that needs to be taken into consideration is the electrical consumption of the kit, to make sure this renewable system would deliver enough power to the kit. </H5><H5>Another line of research concerns the database. For now, SnapLab has its own database, but the team believes that it would be more useful if that database was connected to an existing one. In fact, more diagnosis would be gathered and it would be a first step towards an epidemiological study. The bigger and more complete the database is, the more useful it would be to detect new infectious sites.</H5><H5>One last axis of improvement concerning the engineering part is about the optical filters. In fact, by changing the filter, other fluorescences could be detected and the spectra of use of SnapLab could be wider. Thus, <STRONG>SnapLab detection part could be used like spectrophotometers. </STRONG></H5><H5>However, changing filters is obviously not enough to detect other diseases. Consequently, a work on the plasmid detector could be done. In fact, to diagnose a disease, a target taken from the infected sample hybridises a plasmid especially conceived to detect a precise illness. Exploring the possibilities to create other detectors could be a real improvement for SnapLab. <STRONG>A larger panel of pathogens could be detected inside the kit</STRONG> and that would be one of the biggest improvement for this project !</H5><H5>But still, the biological part contains a multitude of axis of research which would enhance SnapLab kit, in the three different biological steps. The first step, i.e. the DNA extraction step, could be entirely automated, which means that from a feces’ sample, DNA would come out from the extraction column without needing any intervention from the user. That being said, a test with feces samples has not been realised so far, but it would be something to do. The last thing that could be interesting to do would be to enhance the yield of genomic DNA extraction, thus exploring other lysis buffers or changing parameters in the process. </H5><H5>About the biological detection, the study of the specificity of the detector has not been completely finished. Though, it is really important to quantify the specificity, as well as the sensitivity, to estimate the analysis’ accuracy and its limits. All in all, statistics have to be done. Moreover, it would be great to ensure a minimum number of false positives due to the bad digestions of the probe. </H5><H5>The last biological step on the components’ conservation still raises questions for the enzymes’ conservation. Even if plasmid detectors and competent bacteria have been successfully conserved by drying and lyophilisation respectively, a work on the dry-freezing of the enzyme is thought to be continued. Then, enzymes’ conservation achieved, a bacterial transformation should be performed with the dried detector, lyophilised bacteria and lyophilised enzymes. So far, a bacterial transformation has been successfully done with dried detectors and lyophilised bacteria only. What would also be interesting is a test of the transformation after a long-term conservation. Indeed, lyophilised bacteria and dried detector have been conserved for a few weeks but it was not long enough to actually assess for sure whether the time alters their properties or not. Is it necessary to perform drying and lyophilisation regularly because a long conservation is not suitable ? This has to be further studied. </H5><H5 style="padding-bottom:2%;">To conclude on SnapLab project, processing the detection in vitro from the DNA extraction of contaminated feces to the transformation of biological stored components still remains. Then, a whole detection from A to Z should be performed within the kit. But even though time does not allow the Grenoble Alpes team to finalise the project, <STRONG>a  proof of concept has been made and our team hopes that this project will inspire many people to help tackle the remaining challenges that SnapLab faces to be fully operational.</STRONG></H5><DIV class="credit"> 
     Designed by iGEM Grenoble-Alpes 2017 team | <A style="text-decoration:none; color:white;" href="https://2017.igem.org/Team:Grenoble-Alpes/Site_Map">Site Map </A></DIV></DIV></DIV></DIV></DIV></DIV></BODY></HTML>