Team:Heidelberg/Sandbox
Achievements
Results
Medal fulfillments
AptaBodies
Real time S.M.S.
Cystic fibrosis
Labeling and vizualization
Modeling and Software validation
Standardization
Project
Overview
AptaBodies
Real time S.M.S.
Cystic fibrosis
Labelling and Visualization
HRP based detection
Software
MAWS
JAWS
Wiki Tools
Notebook
AptaBodies
Real time S.M.S.
Cystic fibrosis
Labeling and vizualization
Pulse chase
Materials & Methods
Safety
Parts
Team
Members
Supervisors
Collaboration
Attributions
Sponsors
Modeling
Aptamer Sensors
In vitro transcription
Outreach
iGEM Team Heidelberg 2015
Achievements
Results
Project
Software
Notebook
Parts
Team
Modeling
Outreach
Short introduction
“The sea is everything. […] The sea is the vast reservoir of Nature. The globe began with sea, so to  speak”, Nemo said while the “Nautilus” was cruising with a school of hammerhead sharks deep  beneath the waves.
And the captain was right. Deep in the ocean billions of years ago the miracle of nature took place as  a pool of small molecules evolved to self-replicating lifeforms. The flagship role in this development  was probably taken by the most versatile class of molecules in the history of life: RNA.
Seemingly random nucleotides happened to be in the right order to form the first biocatalysts that  made life on the blue planet possible. Today we know those miracles of nature as ribozymes. Inspired  by this, humanity took evolution into their own hands to create aptamers – nucleic acids capable of  encaging molecules. This allows for the detection of virtually anything. Still this process has been  tedious and time consuming much like fishing with a rod in an ocean. We want to revolutionize this  former evolutionary process and want to make it swift like a shark tracking down its prey.
Yet to really bring out the strengths of these simple yet powerful molecules just comprised of A, U, C  and G we want to combine aptamers and ribozymes to create a toolset for the synthetic biologist to  create allosteric ribozymes able to sense a variety of molecules. Therefore, we hope to introduce the  true origins of life and the capabilities of functional RNA to iGEM.
Join us as we sail forth into new  waters of synthetic biology.
Further content
Test
Test
Notebook
Supporting iGEM Team Paris Bettencourt
iGEM Paris Bettencourt has set up a visual databse that should help various iGEM teams collaborate and find common ground. In that database all teams can enter techniques, keywords, organism and many more thev are knowdledgeable in or have been working on. As common dots are connected automatically, it's easy so see, whom to ask for a collaboration.
Though we came a little late to the party, we really loved the idea and hoped, that the database has helped various teams!
" Rhizi is an open-sourced web of knowledge where nodes are sources of information (e.g. scientific articles, questions, blog posts) and edges are the links between them. The goal of Rhizi is to create information rich links between knowledge, open questions and ideas, through encouraging users to vote, comment, and create new relationships."
http://cri-paris.org/research/rhizi/
Collecting impressions from the community
It’s done. We’ve finally decided upon a logo. It’s beautiful, it’s blue, it’s fishy *wink*. The next step was to choose a color scheme and a general style for the wiki. As we feared that our own ideas might be too similar we thought of asking a broader public how they would design a website based on our logo. In order to do so, we headed to the ’Neckarwiese’. During summer, a big part of Heidelbergs population decideds to go there and enjoy the sun, have a BBQ or just relax – so we caught them off guard.
We have had prepared a little presentatio with which we introduced them to iGEM and Synthetic Biology. Afterwards we explained them what we planned on doing, showed them our logo and asked, how they would design a website, based on what they’ve just heard. The general consens was to go for a blue theme, maybe add a little algae green  and generally stick to maritime iconography. Unexpectedly we actually met someone who knew his stuff: Natalie, a student of the history of arts from Berlin. She gave us useful advice on how play with the different colors, what to avoid and even added a little touch of history to it. Lucky catch!
With fresh ideas we headed back to the lab and continued our experiments, being envious of all the people that could continue to sleep in the sun.
We’ve been lucky and actually met someone knowledgable in arts. Her name is Nati and she studies the history of art in Berlin.
Panel Discussion
Synthetic biology - engineering life with its most fundamental units by using DNA BioBricks and other modularly combinable parts, has a potential beyond scope and can improve the quality of life for everyone and mankind as a whole. The ultimate goal of researchers in synthetic biology is not only the understanding of life itself and how it functions, but applying the acquired knowledge to make a change within their community.
To investigate the current state of the research and its acceptance in society, we talked to scientists and asked them how their work has influenced their community. Nonetheless, it is of absolute necessity to include the entire society. We decided to do this by organizing a panel discussion evening dedicated to the topic 'Synthetic biology - Bricks for a healthy life?', i.e. synthetic biology in medicine.
As research in general and especially synthetic biology relies on a community, on interaction between researchers and the exchange of ideas and expertise, we asked experts and researchers from different fields to join us for this evening and we distributed flyers and placards to invite the broad society. However, we wanted to go a step further: The panel discussion was not limited to the audience in Heidelberg, it was simultaneously translated from German to English and broadcasted it via a live stream.
Despite the great enhancements synthetic biology can achieve, engineering with building blocks that are so close to the basic principles of life itself comes with a range of ethical questions and security precautions to consider. We see it as our immanent responsibility as iGEM team to address these questions and to take concerns very seriously. Hence, we invited Prof. Dr. Axel Bauer, who has been member of the German ethics council for several years and Dr. Joachim Boldt who works as assessor for the German ethics commitee for ethical implications of synthetic biology. Both of them are highly involved in the field of medical ethics also due to being professor for this subject. In order to review the safety concerns, we were very glad to have Dr. Harald König, who works at the Institute for Technology Assessment and System Analysis, as our guest.
Different experts were invited to an interdisciplinary talk evening
Politics and law play a very big role when discussing the application of synthetic biology in real life. Researchers need to obey the juridical boundaries and on the contrary the legislature has to react to novel developments and find a compromise between many different opinions, some being more conservative, others more progressive. To reflect this interweaving, we additionally invited local politicians, such as Prof. Dr. Nicole Marmé who is member of the city council of Heidelberg. Dr. Stephan Brandt, chairman of the department for Biotechnological Innovation, Nanotechnology and Genetic Engineering, investigates how laws need to be adapted to the most recent findings in synthetic biology and genetics and we are very glad he joined us for this evening. Finally, the  topic that stands in the center of this evening is, after all, research in synthetic biology. For that reason, we asked Dr. Dirk Grimm, who works on the CRISPR-Cas system and who knows the cutting edge developments to be the scientific representative in the panel.
Jasmin and Max gave a great introduction to synthetic biology in medicine
After a lot of planning, organization and set up of all the required technical equipment (thanks again to Dr. Jens Wagner from the Physics Department), the discussion evening could start with a brief introduction to the topic given by Jasmin and Max from our team. As we wanted the main part of the evening to be an open discussion, we deliberately made the introduction very compact. For the remaining 1.5 hours of the evening, Tim navigated our guests, as well as the audience through the discussion.
The involvement of the audience was amazing, proving that this topic is indeed highly interesting to a large part of the society. Most eminently, we were very happy that additional to the approximately 100 guests that joined us physically in our institute, we had almost 400 viewers online, among them also other iGEM teams, such as iGEM Team Cambrige (link). Besides watching, our followers on twitter were also very engaged in asking questions that were then addressed by Tim and the invited experts.
Luckily, many people came to our talk evening. Even more joined us online via our simultaneously translated live stream and asked questions with #askigemheidelberg
The topics discussed ranged from green to red biotechnology and were contemplated in a highly interdisciplinary way (with the focus on medical applications nonetheless). Besides, and in correspondence to the initialization of the “Community lab” track in iGEM, we addressed biohacking and the implications of it on society, the scientific community and the communication between the two entities. Question were asked about ethical problems and implications of in vivo and in vitro technologies, but also about dreams and wishes of the scientists regarding future developments in this field. The question iGEM team Cambridge nicely summarizes the last part of the discussion: “How can synthetic biologists better communicate their research to the public?” This includes the role of politicians and law makers, as well as the responsibility of everyone who is involved in research to put a focus on the outreach and the interaction, not only with the scientific community, but also with the broader public.
So far, many iGEM teams have organized discussion evenings and invited people from the broad society to join an interdisciplinary evening. This approach is great and helps a lot to improve the communication between scientists and the public. Nonetheless, there are still barriers to overcome:
The interested population of one city is not representing the entire society. Hence, we decided to provide the opportunity to join us online via live stream. This should not be limited to watching the discussion passively, that is why everyone could ask questions via twitter by #askigemheidelberg. These questions were then shown in the discussion, so that our invited guests could reply or reflect on them.
The lingua franca in research is English, however not everybody is capable of speaking English fluently. Therefore, we deliberately chose German as the language the discussion was held in. This way, everyone who was interested had the possibility to follow. In order to keep the event international and also understandable for those who watched online, we translated the event simultaneously to English.
week number 38
▼2015-09-14  Test bitte löschen
10 x <i>in vitro</i> transcription buffer
50 mM Tris ph 7.5, 100 mM NaCl, 20 mM MgCl<sub>2</sub>, 0,01 % SDS
 
week number 37
▼2015-09-10  DNAzyme Activity: DNAzyme with ATP aptamer and calculated Kanamycin aptamer
Samples:
10-23 DNAzyme: xxfs032xx
7-18 DNAzyme: xxfs033xx
10-23 DNAzyme with ATP aptamer with linker: xxfs019xx
7-18 DNAzyme with ATP aptamer with linker: xxfs027xx
10-23 DNAzyme with ATP aptamer A: xxfs017xx, B: xxfs018xx
7-18 DNAzyme with ATP aptamer A: xxfs025, B: xxfs026xx
10-23 DNAzyme with calculated Kan aptamer: xxfs034xx
7-18 DNAzyme with calculated Kan aptamer candidate I: xxfs035xx
7-18 DNAzyme with calculated Kan aptamer candidate II: xxfs036xx
7-18 DNAzyme with calculated Kan aptamer candidate III: xxfs037xx
Stock solutions and conditions:
 
cStock
cFinal
Tris HCl ph 7.5
1 M
50 mM
DNAzyme (A)
10 µM
500 nM
DNAzyme B
10 µM
500 nM
Substrate
1 µM
200 nM
NaCl
1 M
100 mM
MgCl2
1 M
20 mM
SDS
20 %
0,01 %
Adenosine in H2O:DMSO 1:2
33 mM
5 mM
H2O
 
ad 25 µL
 
Pipetting scheme:
#
 
 
Tris HCl ph 7.5
DNAzyme A
DNAzyme B
Substrate
NaCl
MgCl2
SDS
Adenosine
H2O
Final
1
FS032
10-23D
1,25
1,25
0,00
5,00
2,50
0,50
1,25
3,75
9,50
25,00
2
FS033
7-18D
1,25
1,25
0,00
5,00
2,50
0,50
1,25
3,75
9,50
25,00
3
FS019
10-23DmLink
1,25
1,25
0,00
5,00
2,50
0,50
1,25
3,75
9,50
25,00
4
FS027
7-18DmLink
1,25
1,25
0,00
5,00
2,50
0,50
1,25
3,75
9,50
25,00
5
FS017+18
10-23D_A+B
1,25
1,25
6,25
5,00
2,50
0,50
1,25
3,75
3,25
25,00
6
FS025+26
7-18D_A+B
1,25
1,25
6,25
5,00
2,50
0,50
1,25
3,75
3,25
25,00
7
FS032
10-23D
1,25
1,25
0,00
5,00
2,50
0,50
1,25
0,00
13,25
25,00
8
FS033
7-18D
1,25
1,25
0,00
5,00
2,50
0,50
1,25
0,00
13,25
25,00
9
FS019
10-23DmLink
1,25
1,25
0,00
5,00
2,50
0,50
1,25
0,00
13,25
25,00
10
FS027
7-18DmLink
1,25
1,25
0,00
5,00
2,50
0,50
1,25
0,00
13,25
25,00
11
FS017+18
10-23D_A+B
1,25
1,25
6,25
5,00
2,50
0,50
1,25
0,00
7,00
25,00
12
FS025+26
7-18D_A+B
1,25
1,25
6,25
5,00
2,50
0,50
1,25
0,00
7,00
25,00
13
FS032
-10-23D
1,25
1,25
0
0
2,5
0,5
1,25
0
18,25
25,00
14
FS033
-7-18D
1,25
1,25
0
0
2,5
0,5
1,25
0
18,25
25,00
15
FS019
-10-23DmLink
1,25
1,25
0
0
2,5
0,5
1,25
0
18,25
25,00
16
FS027
-7-18DmLink
1,25
1,25
0
0
2,5
0,5
1,25
0
18,25
25,00
17
FS017+18
-10-23D_A+B
1,25
1,25
6,25
0
2,5
0,5
1,25
0
12
25,00
18
FS025+26
-7-18D_A+B
1,25
1,25
6,25
0
2,5
0,5
1,25
0
12
25,00
19
Adenosine
Substrate only
1,25
0
0
5
2,5
0,5
1,25
3,75
10,75
25,00
 
 
#
 
 
Tris HCl ph 7.5
DNAzyme A
DNAzyme B
Substrate
NaCl
MgCl2
SDS
Kan
H2O
Final
20
FS032
10-23D
1,25
1,25
0
5
2,5
0,5
1,25
1,25
12
25
21
FS033
7-18D
1,25
1,25
0
5
2,5
0,5
1,25
1,25
12
25
22
FS034
Kan
1,25
1,25
0
5
2,5
0,5
1,25
1,25
12
25
23
FS035
Kan I
1,25
1,25
0
5
2,5
0,5
1,25
1,25
12
25
24
FS036
Kan II
1,25
1,25
0
5
2,5
0,5
1,25
1,25
12
25
25
FS037
Kan III
1,25
1,25
0
5
2,5
0,5
1,25
1,25
12
25
26
FS032
10-23D
1,25
1,25
0
5
2,5
0,5
1,25
0
13,25
25
27
FS033
7-18D
1,25
1,25
0
5
2,5
0,5
1,25
0
13,25
25
28
FS034
Kan
1,25
1,25
0
5
2,5
0,5
1,25
0
13,25
25
29
FS035
Kan I
1,25
1,25
0
5
2,5
0,5
1,25
0
13,25
25
30
FS036
Kan II
1,25
1,25
0
5
2,5
0,5
1,25
0
13,25
25
31
FS037
Kan III
1,25
1,25
0
5
2,5
0,5
1,25
0
13,25
25
32
Kan
Substrate only
1,25
0
0
5
2,5
0,5
1,25
1,25
13,25
25,00
33
 
Substrate only
1,25
0
0
5
2,5
0,5
1,25
0
14,5
25,00
 
Results and Outlook:
Positive controls worked, Adenosine dependency could be detected for one candidate.
▼2015-09-10  Click reaction of Label-, Label AU RNA [1, 2, 3]
For 51.5µL
Cstock
Cfinal
V[µL]
Phosphate Buffer - pH 7, 0.1M
100mM
50mM
25
Alexa 488 azide
10µM
400nM
2
RNA
1µM
200nM
10
CuSO4
20mM
1mM
2.5
THPTA
50mM
5mM
5
NaAsc
100mM
1mM
0.5
H2O
 
 
6.5
 
Alexa 488 azide was solved in DMSO
Incubation at 37 °C for 12-14 hours (overnight)
Beispieltitel 1
The use of restriction enzymes is no option for cloning if functional nucleic acids contain the recognition sites for enzymes used in standard restriction cloning, as described in RFC 10, or other suitable restriction enzymes. Although techniques using Type IIs restriction enzymes like Golden Gate assembly do not leave scars after cloning, a Type IIs recognition site within the functional RNA sequence may greatly reduce the efficiency and fidelity of a Golden Gate based cloning attempt.                                              Therefore, the method of choice for reliable assembly of the sequences into a standardized vector has to be based on homology at the interfaces of the parts to be fused. As the DNA templates for functional RNA are commonly synthesized de novo, the addition of overhangs upstream and downstream the functional sequence does not appear to be challenging. Equally those extensions can be added during PCR with primer overhangs.
Beispielbild 1
Cooles neues Feature
Google hat neue Effekte
The term “functional RNA” covers a wide range of noncoding natural and synthetic RNA. These RNAs do not need to be translated into protein and are able to fulfil their function either by simple Watson-Crick basepairing interactions or by a more complex formation of secondary structures. Noteworthy classes of functional RNA relevant to the synthetic biologist include:
The term “functional RNA” covers a wide range of noncoding natural and synthetic RNA. These RNAs do not need to be translated into protein and are able to fulfil their function either by simple Watson-Crick basepairing interactions or by a more complex formation of secondary structures. Noteworthy classes of functional RNA relevant to the synthetic biologist include:
Beispielbild 1
Cooles neues Feature
Google hat neue Effekte
Beispielbild 1
Cooles neues Feature
Google hat neue Effekte
Comparison of the cost for the user of antibodies and aptabodies
Cost of antibodies for single use (smalles available size)
The prices of antibodies and aptabodies both in development as well as per use differs drastically. To get an accurate number of cost per use for antibodies, the prices of 4896 aptabodies offered by Thermo Fisher Scientific have been collected and plotted into a histogram. Thermo Fisher Scientific offers only one size for purchase, this equals then the price for the end user for single use. The aptabodies have been sampled pseudo-randomly from the Thermo Fisher Scientific database of "40,000+" antibodies, no currency conversion has been performed and shipping is not considered.
For comparison, the approximate calculated price of an aptabody is shown, see below how this cost is calculated.
Back to Introduction
Copper-catalyzed azide-alkyne cycloaddition (CuAAC)
Sharpless described the copper-catalyzed azide-alkyne cycloaddition (CuAAC)
Kolb2001
also called click chemistry. This is due to the simple concept. It works under many different conditions with high yields and no byproducts.
Mckay2014
A highly energetic azide reacts with an alkyne enabling a selective reaction that links these reactive groups to one another via a dipolar cycloaddition (Fig. 1). However this reaction requires a lot of activation energy.
Zhang2005
Without catalyst the reaction is slow and results in a 1,4, 1,5 triazole regioisomer. To increase the reaction rate and to avoid this byproduct it is necessary to add copper as a catalyst. Copper (Cu(I)) proofed to be a suitable catalyst that rapidly yields a 1,2,3 triazole heterocycle.
Figure 1. Copper-catalyzed azide-alkyne cycloaddition (CuAAC)
Reaction scheme of the CuAAC. Adapted from: Source
Figure 2. Alkyne modification and click reaction of RNA
RNA was 3' modified with alkyne modified nucleotides using yeast Poly(A) Polymerase at 37 °C. Afterwards RNA was precipitated and used for copper click reaction with Alexa 488 azide.
The advantages of a click reaction are that it is very simple and works under many different conditions, as well as that the reaction results in high yields with no byproducts. The highly energetic azides react with alkynes enabling a selective reaction that links reactive groups to one another. To obtain the oxidation state of the copper sodium ascorbate is added to the reaction. Furthermore a ligand like THPTA is necessary to keep the Cu(I) stabilized in aqueous solution.
In order to use the above explained advantages of click chemistry for the labelling of DNA and RNA azide or alkyne modified nucleotides have to be incorporated into the sequence (Fig. 2). Martin et al. have shown that yeast Poly(A) Polymerase is able to incorporate modified nucleotides with small moieties                                                                                        to the 3’ terminus. To obtain an internal modification it is necessary to ligate two part of DNA or RNA to each other via splinted ligation.
Winz2012
Summary
The specific monitoring of small molecules in biochemical reactions is problem that scientist tried to solve since many years. In this project we developed small molecule sensor (SMS) enabling us to analyze biochemical reactions in real-time using a fluorescent readout. Our implemented JAWS software generates us the best ATP-binding Aptamer which is fused to the Spinach Aptamer. In presence of ATP fluorescence will be emitted. This tool enables the sensing of small molecules like ATP that are part of biochemical reactions.
Major Achievements:
Validation of the JAWS Software
Sensing of ATP concentrations in biochemical reactions
Establishment of a dual-fluorescent read-out system to sense small molecules in real-time
Detailed monitoring of the ATP consumption as well as RNA strand synthesis
Abstract
The specific monitoring of small molecules in biochemical reactions is problem that scientist tried to solve since many years. In this project we developed small molecule sensor (SMS) enabling us to analyze biochemical reactions in real-time using a fluorescent readout. Using this innovative method, we were able to analyze classical in vitro transcriptions in detail by monitoring the ATP consumption as well as RNA strand synthesis simultaneously. Using a Spinach RNA Aptamer fused to an ATP-binding Aptamer RNA we can specifically sense ATP concentrations in real-time. Our implemented JAWS software generates us the best ATP-binding Aptamer. Thus we can even detect small changes in the concentration of ATP during RNA synthesis. To validate the JAWS software as well as to show the general feasibility of our fluorescent tool-box system we analyzed transcription efficiencies of different RNA polymerases, the influence of the buffer as well as the effect of inhibitors like heparin on transcription.  Finally, the combination of the JAWS Software and the fluorescent readout enables the scientific community the possibility to target specifically any small molecule of interest in vivo and in vitro.
Introduction
Small molecules are known to regulate many cellular functions. Hence, the development of innovative techniques to analyze metabolic pathways became an important field in research. Those assays require a variety of tools allowing the user to detect small molecules even within live cells
Fernandez-Suarez2008
Tyagi2009
. Such methods hold promise to solve the mechanisms of transcription, translation, localization and the function of non-coding RNA.  The most common ways to analyze small molecules or cellular pathways include protein-based methods like GFP or molecule reactive probes which have been engineered in the past. Having a fluorescent readout seems to be a valuable implement to provide time-resolved information in vivo and in vitro. Recently, Paige et al. discovered an RNA that mimics the green fluorescent protein (GFP) so called “Spinach”
Paige2011
Strack2013
Strack2015
. This Spinach aptamer was generated by systematic evolution of ligands by exponential enrichment (SELEX). In presence of the 3’5-difluoro-4-hydroxybenzylidne imidazolinone dye (DFHBI), RNA forms a stable Spinach-DFHBI RNA aptamer-complex, which is fluorescent. Since then Spinach has been successfully applied by several laboratories to image RNA in live cells
Buxbaum2015
Dean2014
. Moreover, this RNA has been used as a tool to monitor RNA synthesis in real-time
Höfer2013
Pothoulakis
in vitro or to sense different small molecule levels in vivo
Kellenberger2015
Kellenberger2013
. A prominent example is the sensing of ci-di-GMP concentrations in live cells. To do so, Kellenberger et al. attached a ci-di-GMP aptamer to the Spinach aptamer. In presence of a small molecule (c-di-GMP) the aptamer forms a functional stem which results in the formation of a fluorescent Spinach-DFHBI RNA aptamer-complex. Thus, small molecule concentrations can be determined by a fluorescence read-out system.
In our project we are interested in small molecules that are difficult to sense using common techniques. Here we will describe an innovative system that uses the Spinach2 fused to a specific aptamer to detect small molecules. In 2013 the Jaffrey Lab developed Spinach2 which shows in comparison to Spinach a better folding efficiency and thermostability.
Strack2013
. To generate aptamers that specifically bind to a small molecule, we will use our software JAWS. Using this set up we will be able to show that our software is capable to support time consuming methods like SELEX, to identify Aptamers that bind specifically to small molecules. As an interesting target, we will sense the small molecule adenosine triphosphate (ATP) in biochemical reactions. A common method that is performed in thousands of laboratories is a in vitro transcriptions To study the function of ribonucleic acids, RNA is generally prepared by in vitro transcription
Beckert2011
. Using bacteriophage DNA dependent RNA polymerases (T7, T3, Sp6), a variety of different RNAs can be enzymatically synthesized in the lab. In this context we want to establish a new biochemical readout method, called real-time SMS, to record simultaneously small molecules (ATP) and enzymatic kinetics (RNA polymerase) using Spinach2-ATP-Aptamer system.
AIMS AND METHODOLOGIES
Common techniques to sense small molecules in biochemical reactions are most likely connected to radioactive labeling. The exposure to radioactive sources is known to result in damage of the genetic information and is therefore more and more banished from lab work. However because of its sensitivity, radioactive chemicals are needed to monitor small changes of those molecules.
A common method that is performed in many laboratories is an in vitro transcription (Fig. 1A). This method is usually performed in a black-box manner. To analyze the decrease of nucleotide trisphosphate over time as well as to determine the success of the in vitro transcription, scientists use radioactive labelled nucleotide triphosphates and perform time demanding acrylamide gel electrophoresis.
Fig. 1. in vitro transcription and fluorescent RNA Aptamers.
A
A DNA template is transcribed by and RNA Polymerase (RNAP) into RNA. Tools to analyze the kinetic of the polymerase or consumption of the nucleotide triphosphates during the reaction in real-time are still missing.
B
Secondary structure of the Spinach Aptamer. The Spinach RNA consists of three stem loops that are important for the binding of the dye DFHBI. In presence of the dye a fluoresced signal is emitted.
C
Secondary structure of the Malachite Green Aptamer. In presence of malachite green dye and the Aptamer fluorescence is emitted.
To establish such tool box, two different fluorescent RNA constructs will be applied: The first construct is a fusion of an ATP aptamer
Sassanfar1993
and Spinach2
Strack2013
(Fig. 1B), which we will call Spinach2-ATP-Aptamer system . To improve the binding of the ATP aptamer to ATP we apply our own implemented JAWS software. Using our software, nucleotides which form the stem region of the ATP aptamer can be predicted, which will improve binding properties of this RNA to ATP. The fusion of this aptamer to the Spinach2 enables us a fluorescent read out in real-time. Thus, in presence of this small molecule, the ATP aptamer will form a tight stem loop, which results in a new structural conformation of the Spinach aptamer. Finally after binding of the ATP, the Spinach2-ATP-Aptamer is able to interact with the DFHBI. Hereby fluorescence can be measured at 500 nm if the Spinach was excited at 460 nm. We will generate using our JAWS software different Spinach2-ATP-Aptamers. To validate the software as well as to identify the best Spinach2-ATP-Aptamer system, we will analyze them concerning their fluorescent turn-on effect in presence of ATP.
Fig. 2. Small molecule sensing in real time (Real time SMS).
General overview of our real time SMS tool-box, which allows the monitoring of the ATP consumption during in vitro transcription reaction. For this purpose we designed a DNA template containing a promoter (T7) according to the applied RNA polymerase, RNA of interest (ROI), Hammerhead ribozyme (HHR) and Malachite Green Aptamer (MGA). If the Malachite Green Aptamer is transcribed, an increase in fluorescence can be monitored. The second part of the toolbox is a fusion of Spinach2 to an ATP-binding Aptamer, predicted by our JAWS software. In presence of ATP, the ATP Aptamer forms a stem, enabling the DFHBI dye to bind to the Spinach Aptamer thereby triggering fluorescence emission.
The second important part of the toolbox is a DNA template containing a promoter according to the applied RNA polymerase and a Malachite Green Aptamer (Fig. 1C), which is excited at 630 nm and emits at 652 nm in presents of malachite green dye. If the Malachite Green Aptamer is transcribed, an increase in fluorescence can be monitored. The second part of the toolbox allows a direct analysis of the success of the in vitro transcription in real-time. In an advanced set up, this system can be extended by DNA template encoding for the RNA of interest (ROI) and a hammerhead ribozyme (HHR). Both fragments are inserted between the promotor and the Malachite Green Aptamer. Inducing the hammerhead ribozyme allows cleavage during the in vitro transcription. Hereby two RNA fragments emerge, the ROI and the hammerhead ribozyme fused to Malachite Green Aptamer (HHR-MGA). Using such setup, the emitted fluorescence of the HHR-MGA will be independent on the applied ROI. Thus the fluorescence induced by the Malachite Green Aptamer can be applied to compare efficiencies of different RNAs at the same time.
The combination of two aptamers that are able to turn on fluorescence by binding to DFHBI or malachite green provide us the possibility to monitor simultaneously the ATP consumption as well as RNA strand synthesis during in vitro transcription in real-time (Fig. 2). The application of the JAWS software generates us the best ATP-aptamer to identify even small changes during RNA synthesis.
To validate the JAWS-generated aptamers as well as to show the functionality of the Spinach2-ATP-Aptamer system we will apply different bacteriophage RNA polymerases (T7, Sp6, T3) that are commonly used. In addition inhibitors (Heparin) of polymerases as well as the effect of different buffer compositions will be analyzed by our approach.
RNA-Spinach2-Template preparation
Spinach2-DNA templates for in vitro transcription were amplified by extension-PCR and purified using Quiaquick PCR Purification Kit or ethanol precipitation. in vitro transcription was performed in presence of 40 mM Tris-HCl pH 8.1, 1 mM spermidine, 20 mM MgCl2, 0.01% Triton-X100, 4 mM each NTP, 10 mM DTT, 5 % DMSO and 0.1 mg/mL T7 RNA Polymerase. DNA was removed by 1 U/ µL DNase I digest. RNA was purified by denaturing PAGE and eluted in 0.3 M NaOAc pH 5.5. RNA was isopropanol precipitated and concentration determined by NanoDrop measurements.
Purified RNA was renatured in 40 mM HEPES KOH pH 7.5, 125 mM KCl and 3 mM MgCl2 and 20 % RNA. Proper folding was achieved by heating up the RNA to 95 °C for 3 min and then letting it cool down to room temperature.
Functionality of the renatured Spinach2 was determined using 500 nM renatured RNA, 1 mM of ligand (e.g. ATP) 100 µM DFHBI (Lucerna) and 0.2 U/µL of Ribolock RNase Inhibitor (Thermo Scientific). The fluorescence spectrum was measured using a spectro fluorometer (JASCO). Then following settings were applied to measure Spinach 2 fluorescence: Ex= 460 nm and Em=475-600 nm, high sensitivity and 37 °C.
Thin Layer Chromatography to Analyze ATP Consumption During in vitro Transcription
Thin layer chromatography will be applied to analyze ATP consumption during in vitro transcription. To analyze ATP consumptions during the RNA transcription, ATP is converted to AMP after the transcription process by apyrase. in vitro transcription was performed in presence of 40 mM Tris-HCl pH 8.1, 1 mM spermidine, 20 mM MgCl2, 0.01% Triton-X100, 4 mM each NTP, 10 mM DTT, 5 % DMSO and 0.1 mg/mL T7 RNA Polymerase. As DNA template we used the ATP AptamerJAWS1 Spinach2. 5 µL samples were taken every 5 min from the reaction and heated up to 95 °C for 10 min to inactivate the T7 RNA polymerase. To convert ATP to AMP 0.01 U Apyrase and 4 mM CaCl2 were added to each sample and incubated for 1 h at 30 °C. 2 µL of each sample was spotted on a fluorophore coated TLC Plate (ALUGRAM®Xtra SIL G/UV254, Macherey Nagel). TLC was run 4 h 45 min 4:6 Ammonium acetat: ethanol and analyzed with a UV lamp.
Malachite Green Aptamer Template Preparation for in vitro Transcription
Malachite Green Aptamer template (10 µM) was hybridized by heating up a forward and reverse oligo to 95 °C. DNA was cooled down to room temperature. dsDNA was then stored at -20 °C.
Malachite Green Aptamer Activity
Malachite Green Aptamer activity was tested by transcribing the prepared DNA-Template in vitro with T7 polymerase as described above and in presence of 1 mM malachite green (Sigma). To ensure synchronic initiation a master mix containing the buffer, enzymes, dye and template were pipetted separately. The assay was performed in 384 well micro titer plates (black, flat round, transparent bottom [Corning, 3540]) on a 20 µL scale. Evaporation during transcription was prevented by using a sealing tape (#232701, Nunc). Measurements on micro titer plates were performed in a microplate reader (Tecan Safire 2). Following parameters were chosen for the assay setup: Ex= 630 nm and Em=652 nm, 10 nm excitation/ emission bandwidth, high sensitivity flash mode and 40 µs integration time. The reaction was measured every 30 sec at 37 °C. To ensure the functionality of the assay, samples of the in vitro transcription were analyzed on a 10 % 8 M urea PAGE.
Time-resolved in vitro Transcription
To describe in vitro transcription time-resolved in terms of NTP consumption, following conditions were applied: 500 nM renatured ATP Aptamer Spinach2 RNA, 10 µM Malachite Green Aptamer DNA template, 40 mM Tris pH 8.1, 1 mM spermidine, 20 mM MgCl2, 0.01% Triton X-100, 4 mM each NTP, 10 mM DTT, 5 % DMSO, 100 µM DFHBI, 1 mM malachite green, 0.1 mg/mL T7 RNA Polymerase, 0.2 U/µL Ribolock RNase and 0.1 U Pyrophosphatase (Thermo Scientific). The assay was performed in 384 well micro titer plates (black, flat round, transparent bottom [Corning, 3540]) on a 20 µL scale. Evaporation during transcription was prevented by using a sealing tape (#232701, Nunc). Measurements on micro titer plates were performed in a microplate reader (Tecan Safire 2).
Influence of Inhibitors on in vitro Transcription
To analyze the influence of inhibitors on the in vitro transcription, heparin (0.7 mg/mL and 1.3 mg/mL) was applied. in vitro transcriptions were performed similar to “time-resolved in vitro Transcription” as described above.
Quantification of RNA Concentrations using Malachite Green Aptamer Fluorescence Read-Out
The T7-Malachite Green DNA template was applied for in vitro transcription as described above. RNA was purified by denaturing PAGE, eluted and recovered by NaOAc/isopropanol precipitation. The RNA was purified from remaining salts using Amicon Ultra-0.5 mL centrifugal filters 3K (Merck Millipore). For preparation of a calibration curve, RNA solutions of different concentrations were refolded (95°C 3min, in 1x renaturing buffer) 7. After addition of 100 µM DFHBI, 100 µM malachite green, 10 mM DTT, 4 mM ATP, 4 mM GTP, 4mM CTP, 4 mM UTP, 1x transcription buffer (1mM spermidine), 0.0017 U pyrophosphatase, 0.46 U Ribolock, 0.075 % glycerol fluorescence, fluorescence was measured on micro titer plates using microplate reader (Tecan Safire 2). Following parameters were chosen for the assay setup: Ex= 630 nm and Em=652 nm, 10 nm excitation/ emission bandwidth, high sensitivity flash mode and 40 µs integration time. The reaction was measured every 30 sec at 37 °C.
Results
Establishment of a System to Sense Small Molecule Using the Spinach2 Aptamer
Fig. 3. Fusion of Aptamers to Spinach to generate fluorescent small molecule sensors.
A
For the design of a new small molecule sensor, the second stem of the Spinach2 aptamer can be exchanged by an aptamer that binds specifically to a small molecule. We fused an ATP-binding Aptamer (yellow) to the Spinach.
B
We applied our JAWS software to predict the best ATP-Aptamers, that can form the best stem structure (blue and red highlighted) in presence of ATP.
C
The JAWS predicted stems can be fused to the Spinach Aptamer. The software can be validated by analyzing the fluorescence emission in presence of ATP and DFHBI (Fig.4B and C).
To establish a stable, reproducible system in lab that is able to sense small molecules in real time using a fluorescent readout, we applied the already published c-di-GMP Spinach2 system first
Kellenberger2013
. To design new small molecule sensors, we adapted the system described by Kellenberger et al. We exchanged the second stem of Spinach2 by three different ATP Aptamer systems (Fig. 3). All of them are based on an ATP Aptamer described by Szostak. ATP Aptamer Spinach2 is the original aptamer that was generated by Szostak. ATP AptamerJAWS1 Spinach2 and the ATP AptamerJAWS2 Spinach2 contain the core sequence of the original ATP Aptamer, but have a modified stem, generated by our software JAWS. In presence of the ATP ligand, the ATP aptamer should change its tertiary structure and forms together with the Spinach2 a stem. The stem formations should finally result in an increase of the fluorescence. Using this read-out system we can compare the JAWS-generated Aptamers with the original one, described by Szostak. If our JAWS software works nicely, we should get a better fluorescent read-out with ATP AptamerJAWS1 Spinach2 or ATP AptamerJAWS2 Spinach2 in comparison to the original ATP Aptamer Spinach2 construct.
Fig. 4. Establishment of a system to sense small molecule using the Spinach2 Aptamer.
A
Emission spectrum of the original Spinach2 Aptamer, which was applied as an internal control.
B
As another internal control, we reproduce the data for the c-di-GMP Spinach2 system, published by Kellenberger et al.. Indeed, highest fluorescence maximum for the c-di-GMP Spinach2 system was measured in presence of the ligand.
C
Analysis of the fluorescent properties of our ATP Aptamer Spinach2 constructs. The Spinach2 containing the Szostak ATP Aptamer shows the lowest fluorescence of all three ATP Aptamer Spinach2 variations. The JAWS-generated ATP AptamerJAWS1 Spinach2 and the ATP AptamerJAWS2 Spinach2 show higher fluorescence maxima in presence of ATP.
To analyze the functionality of our small molecule sensor system, we amplified all constructs, including, Spinach2, c-di-GMP Spinach2 and ATP Aptamer Spinach2 sensors by PCR and transcribed them in vitro successfully. The RNA was analyzed for its specific binding to a small molecule by a highly sensitive fluorometer. Figure 4 shows the results of the fluorescence measurements. All Spinach2 systems show fluorescence maxima at 500 nm in presence of the ligand. The original Spinach2
Paige2011
was applied as an internal control and shows the highest fluorescence intensity (Fig. 4A). As another internal control we tried to reproduce the data for the c-di-GMP Spinach2 system (Fig. 4B). Indeed, highest fluorescence maximum for the c-di-GMP Spinach2 system was measured in presence of the ligand. Thus we were able to reproduce the data shown by Kellenberger et al. In addition, we observed a new second peak for the c-di-GMP Spinach2. The second maxima appears at Em= 552 nm. The red light shift also exists in the spectrum of the c-di-GMP Spinach2 without ligand with a lower peak. Finally, we analyzed the fluorescent properties of our new ATP Aptamer Spinach2 constructs (Fig. 4C). The Spinach2 containing the Szostak ATP Aptamer shows the lowest fluorescence of all three ATP Aptamer Spinach2 variations. The JAWS-generated ATP AptamerJAWS1 Spinach2 and the ATP AptamerJAWS2 Spinach2 show higher fluorescence maxima in presence of the ligand. Due to its perfect fluorescence properties we applied the ATP AptamerJWAS1 for further studies.
Common Techniques of Sensing Small Molecules in Biochemical Reactions
Analyzing biochemical reactions depend on radioactive labeling. Yet, many laboratories try to avoid the usage of radioactivity since it is connected to affect the health. Our established ATP Aptamer Spinach2 sensor system enables the sensing low concentrated small molecules like ATP within in vitro assays. In this context the change of ATP concentration during the in vitro transcription reaction is an attractive target to study. NTPs are genially consumed by a RNA polymerase over time. To monitor the decrease of ATP during a successful in vitro transcription, we applied thin layer chromatography first (Fig. 5). To analyze ATP and not GTP/CTP or UTP by the TLC, the remaining ATP was converted to AMP via an apyrase reaction. However, we cannot identify significant changes of the ATP concentration on the TLC plate because of the low sensitivity of this method.
Fig. 5. Analysis of an in vitro transcription by thin layer chromatography.
ATP consumption during the transcription was monitored by TLC. ATP was converted to AMP by an apyrase reaction in order to separate it from other NTPs. Using TLC the consumption of ATP by the polymerase cannot be monitored properly.
Highly Sensitive Sensing of Small Molecules via Spinach in Biochemical Reactions in Real time
Fig. 6. Real time monitoring of the RNA synthesis using a Malachite Green Aptamer.
A
To analyze RNA synthesis in real time we applied a DNA template encoding for the RNA of interest (ROI) and a hammerhead ribozyme (HHR). Both fragments are inserted between the promotor (T7) and the Malachite Green Aptamer (MGA). Inducing the hammerhead ribozyme allows cleavage during the in vitro transcription. Hereby two RNA fragments emerge, the ROI and the hammerhead ribozyme fused to Malachite Green Aptamer (HHR-MGA). Using such setup, the emitted fluorescence of the HHR-MGA will be independent on the applied ROI. Thus the fluorescence is induced by the Malachite Green Aptamer that can be applied to compare efficiencies of different RNAs at the same time.
B
As a proof of principle we performed transcriptions and monitored the malachite green fluorescence signal in real time.
C
To confirm the results of the fluorescence measurements as well as the transcription efficiency is not hampered by the malachite green dye, all in vitro transcription reactions were analyzed using denaturing acrylamide gels.
Common techniques like TLC have a very poor limit of detection. Especially in our studies, a highly sensitive method is needed to visualize ATP concentration in biochemical reactions. Thus, we can apply our ATP AptamerJAWS1 Spinach2 system so sense consumption of ATP in real time (Fig. 6). To show the general feasibility of our system we titrated the T7 RNA polymerase concentration in in vitro transcription reaction. We monitored the fluorescence signal of the ATP AptamerJAWS1 Spinach2 in regular intervals. In presence of a high RNA polymerase concentration we would expect the highest turn-off effect of the fluorescence. In Figure 6B we show the results of the real time measurements of the spinach fluorescence. As expected we can show that the ATP signal decreases rapidly in presence of a high concentration of polymerase. In contrast, if less RNA polymerase was applied, the measured fluorescence is much higher. To confirm the results of the fluorescence measurements, we analyzed the in vitro transcription reaction using denaturing acrylamide gels in addition (Fig. 6C). Similar to the fluorescent readout, we were able to identify the most intense bands in presence of high T7 RNA polymerase.
in vitro Transcription Using a Fluorescent RNA of Interest
To substitute the time-consuming analysis of in vitro transcription by denaturing acrylamide gel electrophoresis, we established a second read-out system. Using this system we want to monitor RNA synthesis in real time (Fig. 7). For this purpose we applied a DNA transcription template which encodes the Malachite Green Aptamer
Baugh2000
. This aptamer binds to the malachite green dye and differs theoretically from the ATP Aptamer Spinach2 in its excitation and emission spectra.
Fig. 7. Real time monitoring of the RNA synthesis using a Malachite Green Aptamer.
A
To analyze RNA synthesis in real time we applied a DNA template encoding for the RNA of interest (ROI) and a hammerhead ribozyme (HHR). Both fragments are inserted between the promotor (T7) and the Malachite Green Aptamer (MGA). Inducing the hammerhead ribozyme allows cleavage during the in vitro transcription. Hereby two RNA fragments emerge, the ROI and the hammerhead ribozyme fused to Malachite Green Aptamer (HHR-MGA). Using such setup, the emitted fluorescence of the HHR-MGA will be independent on the applied ROI. Thus the fluorescence is induced by the Malachite Green Aptamer that can be applied to compare efficiencies of different RNAs at the same time.
B
As a proof of principle we performed transcriptions and monitored the malachite green fluorescence signal in real time.
C
To confirm the results of the fluorescence measurements as well as the transcription efficiency is not hampered by the malachite green dye, all in vitro transcription reactions were analyzed using denaturing acrylamide gels.
To analyze the functionality and fluorescent properties of the Malachite Green Aptamer we performed in vitro transcriptions and measured fluorescence in regular intervals. We prepared two negative controls (Blank) which were either depleted with T7 RNA Polymerase or without malachite green. The functional setup contains DNA template, polymerase as well as malachite green dye to monitor the transcription in real time by an increase of the fluorescence signal. If our Malachite-Green Aptamer read-out system functions properly, we expect an increase at an emission wavelength of 650 nm only for functional setup. The blank values should stay at zero. The results (Fig. 7) indicate that we do not see any increase of fluorescence for the blanks. All measured values show a fluctuation around 0. The graph of the functional aptamer with malachite green and T7 polymerase shows an increase of fluorescence over time. We were able to record a classical enzyme kinetic curve of a polymerase using the malachite green system (Fig. 7B). Furthermore, to confirm our data and to show, that the transcription efficiency is not hampered by the malachite green dye, we performed denaturing acrylamide gel electrophoresis (Fig. 7C). The gel shows similar band intensities in presence as well as in absence of the dye if T7 polymerase was added to the reaction. Thus malachite green does not influence our transcription efficiency.
in vitro Transcription recording the Nucleotide Consumption in Correlation to de novo RNA Synthesis
Fig. 8 Simultaneous monitoring of ATP consumption and RNA synthesis during transcription.
in vitro transcription reaction was performed in presence of the ATP AptamerJAWS1 Spinach2/DFHBI and Malachite Green Aptamer DNA template/malachite green dye. Fluorescence was measured in regular intervals. During the reaction we were able to sense the decrease of ATP AptamerJAWS1 Spinach2 fluorescence and an increase of the fluorescence corresponding to the Malachite Green Aptamer. Controls (w/o T7 RNA polymerase) show no change in the fluorescence signal over time. Thus ATP consumption and RNA strand synthesis can be analyzed in real time.
Fig. 9. Real time monitoring of RNA synthesis and ATP consumption in presence of inhibitors.
A
Model of the transcription using our Spinach/Malachite Green readout system in presence of a polymerase inhibitor (I) such as heparin. The inhibitor results in a stop of the transcription.
B
We determined the influence of the inhibitor heparin on a T7 RNA polymerase transcription. For this purpose we applied our new fluorescent readout system. Fluorescence emission of ATP AptamerJAWS1 Spinach2 and Malachite Green Aptamer is not increasing over time. For the positive controls (w/o heparin) the fluorescence increases as expected over time.
As we were able to establish a highly sensitive fluorescent read out for ATP concentrations as well as for the RNA synthesis, we wanted to combine both methods. We performed in vitro transcription reactions in presence of the ATP AptamerJAWS1 Spinach2/DFHBI and Malachite Green Aptamer DNA template/malachite green dye. Fluorescence was measured in regular intervals. During the reaction we were able to sense the decrease of ATP AptamerJAWS1 Spinach2 fluorescence and an increase of the fluorescence corresponding to the Malachite Green Aptamer (Fig. 8). The results show that the transcription reaction reaches its maximum after 140 min.
In further studies, we added heparin to the in vitro transcription which is known to be a potent inhibitor of the T7 RNA polymerase
Sastry1997
. Here we expect a stable level of the ATP signal and reduced fluorescence at 650 nm (Malachite green). In Figure 9 we can show that indeed heparin results in low fluorescence signal and ATP concentrations stay at the same level over time. Thus transcription reaction is inhibited by this molecule.
High Throughput Buffer Test under Low Input Conditions
Fig. 10. Application of the Spinach/Malachite Green readout system to improve buffer conditions for the transcription.
As DNA template we applied the Malachite Green Aptamer with T7 promotor. Using our fluorescent read out system we can determine different transcription efficiencies as well as ATP consumption in real time.
A
In presence of the HEPES buffer no increase of the Malachite Green Aptamer fluorescence signal was monitored. Furthermore the signal of the ATP Aptamer Spinach2 stays also at the same level as the blank.
B
Tris-HCl buffer show a decrease of Spinach2 fluorescence and an increase of the Malachite Green Aptamer signal.
C
The transcription buffer shows the highest slope of Malachite Green Aptamer fluorescence and the fastest decrease of the ATP Aptamer Spinach2 fluorescence.
Adjusting an in vitro transcription to its optimal conditions is usually connected to big efforts. The efficiency and the final RNA yield of each transcription depend on the applied buffer conditions, polymerase as well as DNA template. In our project several RNAs were generated by in vitro transcription. However, we could not achieve high efficiency for some constructs such as the “substrate of RNA-cleaving DNAzyme”. To show the general applicability of our Spinach/Malachite green system to optimize transcription conditions in the shortest time possible, we tested different buffers systems first (Fig. 10). Especially buffers are known to have a huge influence on the activity of the polymerase which can probably improve the transcription of the DNA template “substrate of RNA-cleaving DNAzyme”.
For the buffer screen we applied six different buffer systems: Three transcription buffers pH 8.1 with different spermidine concentrations ranging from 0 mM to 10 mM, Tris-HCl buffer pH 8.1, HEPES buffer pH 7.5 (similar to the renaturing buffer) and a buffer including BSA pH 8.1. All buffers were analyzed using the same conditions as described in previous assays. As DNA template we applied the Malachite Green Aptamer with T7 promotor. Using our fluorescent read out system we can determine different transcription efficiencies. In presence of the HEPES buffer no increase of the Malachite Green Aptamer fluorescence signal was monitored (Fig. 10A). Furthermore the signal of the ATP Aptamer Spinach2 stays also at the same level as the blank. The BSA buffer and the Tris-HCl buffer (Fig. 10B) show a decrease of Spinach2 fluorescence and an increase of the Malachite Green Aptamer signal. The transcription buffer (1 mM and 10 mM) shows the highest slope of Malachite Green Aptamer fluorescence and the fastest decrease of the ATP Aptamer Spinach2 fluorescence (Fig. 10C). Thus we can detect huge differences in the transcription by applying our Spinach/Malachite Green set up. In addition we tested DNA template “substrate of RNA-cleaving DNAzyme”, which was difficult to transcribe (Fig. 11).
Fig.11. Monitoring of the ATP consumption and RNA synthesis of difficult transcription templates.
A
Using our fluorescent tool box we can demonstrate again that substrate of RNA-cleaving DNAzyme causes problems during transcription. We can identify a slight decrease of Spinach fluorescence and increase in the Malachite Green signal.
B
We were applying a DNA template containing a HHR in front of the Malachite Green Aptamer, we analyzed cleavage by denaturing acrylamide gel electrophoresis. We could identify the cleaved HHR-MGA RNA by high sensitive Sybr Gold staining.
Using our fluorescent tool box we can demonstrate again that substrate of RNA-cleaving DNAzyme causes problems during transcription. We can identify a slight decrease of Spinach fluorescence and increase in the Malachite Green signal. Thus this RNA yield of this transcription is low (Fig. 11A). As we were applying a DNA template containing a HHR in front of the Malachite Green Aptamer, we analyzed cleavage by denaturing acrylamide gel electrophoresis (Fig 11B). We could identify the cleaved HHR-MGA RNA by high sensitive Sybr Gold staining.
Employing different Polymerases for Universal Setup Conditions
Fig. 12. Determination of the transcription efficiency of different polymerases.
A
We applied our Spinach/Malachitgreen setup to different RNA polymerase such as T7 Phi 2.5, T3, Sp6 and E. coli.
B
ATP consumption (ATP-AptamerJWAS1 Spinach fluorescence) and RNA synthesis (Malachite Green fluorescence) were measured in real time. ATP was fast consumed and RNA transcribed by T7 and T3 polymerases. SP6 shows activity as well. E. coli RNA polymerase was not showing any activity.
To overcome the dependence of the setup to apply a specific polymerase we designed new transcription templates containing different promoters for a subset of commercial available RNA polymerase such as T7 Phi 2.5, T3, Sp6 and E. coli (T7A1) (Fig. 12A). All polymerases have been tested for 12 hours at 37 °C and were measured in regular time intervals using TECAN Safire. The results show that the decrease of ATP AptamerJWAS1 Spinach2 activity is connected to the speed of the RNA polymerase. The data indicates that the reduction of ATP goes faster for T7 and T7Phi 2.5 than for the other candidates (Fig. 12B). Therefore the fluorescence of ATP AptamerJAWS1 Spinach2 decreases. The T3 polymerase is a little bit slower than the T7 RNA polymerase and does not reach the same bottom point. Nevertheless the graph decreases still over time and the polymerase seem to be active even after 500 min of the reaction. The Sp6 shows activity as well. The Sp6 leads to higher Malachite Green Aptamer production by using less ATP or leading to more functional RNA, while the T3 has a lower ATP Aptamer Spinach2 activity than Malachite Green Aptamer.
Establishment of Titration Curves for Quantification of the in vitro Transcription
Fig. 13. Calculation of RNA yields in real time using the Spinach/Malachite Green system.
A
Malachite Green Aptamer RNA was titrated to different concentrations (10 µM, 7.5 µM, 5 µM, 2.5 µM, 1 µM, 0.5 µM and 0 µM of RNA) and the fluorescence determined using a multiwell-plate reader. A calibration curve was calculated and applied to quantify RNA yields in real time.
B
As an example we calculated the RNA amount of the different transcriptions in presence of different DNA template concentrations (0-2.5 nM). As expected best RNA yields were determined in presence of the highest DNA template concentration.
The quantification of fluorescence is necessary to estimate the total amount of RNA synthetized in the in vitro transcription. For this purpose we transcribed the Malachite Green Aptamer and purified it by denaturing PAGE. Finally Malachite Green Aptamer RNA was titrated to different concentrations (10 µM, 7.5 µM, 5 µM, 2.5 µM, 1 µM, 0.5 µM and 0 µM of RNA) and the fluorescence determined using a multiwell-plate reader in presence of the dye and all other transcription components (Fig. 13). We were able to obtain a calibration curve (Fig. 13A) with a correlation coefficient of 0.99 which enables us to calculate the final RNA yields of different in vitro transcription reactions (Fig. 13B). As an example we calculated the RNA amount of the different transcriptions in presence of different DNA template concentrations. As expected best RNA yields were determined in presence of the highest DNA template concentration.
Discussion and Outlook
To sense specifically small molecules in biochemical reactions is problem that scientist tried to solve since many years.
In this project we developed an innovative method that can be applied as a tool box, enabling us to sense small molecules in real-time (real-time SMS) using a fluorescent readout at two different emission wavelengths. Using a spinach aptamer fused to an ATP aptamer we can sense ATP concentrations in real-time during a biochemical reaction. Thus we were able to analyze classical in vitro transcriptions in detail by monitoring simultaneously the ATP consumption as well as RNA strand synthesis.
A major achievement of our fluorescent tool-box was the validation of our own implemented JAWS software. ATP Aptamer-stems were successfully predicted which were causing a high turn-on effect of the fluorescence in presence of the small molecule. The ATP AptamerJAWS2 Spinach RNA shows a stronger interaction with ATP resulting in a higher fluorescence maximum than the ATP AptamerJAWS1 Spinach RNA. However, the tight binding to the small molecule ATP can result in a decreased bioavailability of this NTP for the polymerase. This results in decreased transcription efficiency. Due to this effect we decided to apply the ATP AptamerJAWS1 Spinach for our fluorescent read-out system. Nevertheless live-cell imaging might benefit from the good binding properties of ATP AptamerJAWS2 Spinach RNA. Imaging of ATP in cells might be also an interesting target to study.
Another important achievement was the monitoring of the change of ATP concentrations during the in vitro transcription. We can show that the concentration of used amount of polymerase changes the final RNA yield. We increased the polymerase concentration and observed low Spinach and high malachite green fluorescence levels. Thus ATP was consumed and RNA was successfully transcribed. In addition we were able to show the influence of different buffer systems, inhibitors and different polymerase. Using this huge amount of generated Data, we were able to model in vitro transcription reactions and to analyze the speed of nucleotide incorporation. By application of our tool-box, a in vitro transcription reaction can be optimized within a few hours. Thus difficult transcription targets can be easily optimized.
Furthermore, we were able to set-up a method to quantify the RNA yields in real-time. Similar to classical colorimetric assays like Bradford, we successfully applied instead of BSA, our Malachite Green Aptamer RNA to generate a calibration curve with a really good correlation coefficient. In addition, the JAWS generated ATP aptamer Spinach can be applied in several other biochemical reactions that depend on ATP. In interesting target would be the Poly(A) polymerase
Balbo2007
, that attaches several adenosine moieties to the 3’-end of RNAs in eukaryotes.
Furthermore, by using the JAWS software we can generate new sensors that are specifically binding to new interesting small molecules for example rape drugs, cofactors like NAD(H) or FAD. The generated aptamers could be fused to the Spinach and binding can be monitored by an increase of the fluorescence in presence of the small molecule.
Those generated aptamers might by also applicable for live-cell imaging to sense small molecules. We have already demonstrated for ATP AptamerJAWS2 Spinach RNA a good binding efficiency, which might be a good candidate for imaging experiments as well.
What is synthetic biology
What’s there better than asking those successfully working in synthetic biology for many years?
We embarked on realizing a video project that, for the first time, collected the opinions of renowned synthetic biologists on crucial questions about the field they (and we) work in. It has been challenging to recruit them for this project, but eventually we managed to convince several of them to share with us their thoughts on synthetic biology.
The project was structured this way: we sent personalized emails to invite very famous synthetic biologists to participate in the video project. We sent them a list of questions (Table 1) from which to choose, and asked them to film themselves while replying to one of these questions.
Can you define synthetic biology?
How does synthetic biology impact society?
What has been the biggest struggle you have had working on synthetic biology ?
Throughout the years how did your opinion on synthetic biology change? Did you accomplish something that you thought would never be possible ?
What is the biggest risk associated to synthetic biology?
Which is the best way in your opinion to educate the new generations in synthetic biology?
How far is a future where synthetic biology applications are part of everyday life?
What would you say to convince a student to undertake studies in synthetic biology?
What are the qualities that distinguish a synthetic biologist from other scientists?
What distinguishes synthetic biology from other disciplines?
Table I. List of questions on synthetic biology and its impact on society we asked synthetic biologists to reply to in a short video.
Beatrix Suess (Technical University Darmstadt, Germany)
What would you say to convince a student to undertake studies in synthetic biology?
“Synthetic Biology is characterized by its enormous interdisciplinarity, bringing together scientists from different disciplines. […] I feel that it is necessary to implement this thinking in the curriculum of our universities.”
Luis Serrano (Design of Biological Systems, Barcelona, Spain)
Can you define synthetic biology?
“For me synthetic biology differs from biotechnology that it uses rational engineering and design.”
Victor De Lorenzo (Centro National de Biotechnología, Madrid, Spain)
Throughout the years how did your opinion on synthetic biology change? Did you accomplish something that you thought would never be possible?
“My view of synthetic biology has changed in the last few years from being just a mere extention of molecular genetics […] into something that allows us to understand and also to reprogram biological systems with a degree of predictability.”
It was very interesting to us to find out whether other renowned scientists working in other fields could be able to at least answer the simple question “what is synthetic biology?”. We, therefore, tried to involve other non-synthetic biologists in our video project. Some of them found the time to reply showing interest in the project, but eventually did not get to the point to really contribute a video due to their tight schedule. To bypass the problem of having to ask people per mail and not in person, we had the idea to interview speakers that visited a conference here in Heidelberg. We were surprised to see how well they could explain concepts related to synthetic biology!! Yet, to be honest, the conference was on systems biology, a field very close to synthetic biology, so it is perhaps not too surprising that they showed such high competence in this topic.
Eytran Ruppin (University of Maryland, USA)
How does synthetic biology impact medicine?
“Synthetic Biology is at a stage[…] where there is a lot of promise or a lot of hype and yet you need to deliver.”
Gaudenz Danuser (Harvard University, Boston, USA)
Which is the best way in your opinion to educate the new generations in synthetic biology?
“Synthetic Biology is happening and therefore we need students who can do it and critically think about it.”
Ioannis Xenarios (University of Lausanne, Switzerland
How does synthetic biology impact society?
“A better access to a certain type biotechnology products at a cost that is reasonable. And ultimately, […] help mankind take the next challenge.”
Philippe Bastien (Max-Planck Institute of Molecular Physiology, Dortmund, Germany)
What would you say to convince a student to undertake studies in synthetic biology?
“I would try to convince you that curiosity is what makes it really fascinating.”
Roy Wollmann (University of California, San Diego, USA)
How far is a future where synthetic biology applications are part of everyday life?
“Tremendous effect on the scientific community: The availability of new tools […] will open up a whole new field of experiments that we can do that we weren’t able to do before.”
Ulricke Gaul (Ludwig-Maximilians University, Munich, Germany)
What is the biggest risk associated to synthetic biology?
“There is always the danger when you create something new that has never existed before, that if you release it into the world at large that it might harm something.”
Uri Alon (Weizmann Institute of Science, Rechovot, Israel
How far is a future where synthetic biology applications are part of everyday life?
“It’s hard for me to believe that the century will end without a profound incorporation of understanding of biology into everyday’s life.”
Walter Kolch (University College Dublin, Ireland)
What are the qualities that distinguish a synthetic biologist from other scientists?
“A broad way of thinking and an open way of thinking, […] you also need to be inventive and actually finding the right applications. You need to be an allrounder.”
Beyond the systems biologists, we actually received written answers to our questions (Table 1) from the director general of the EMBL, Dr. Ian Mattaj. He showed quite some knowledge of what synthetic biology is, and gave us some interesting points of views, such as the feeling that synthetic biology applications are already pervasive in our daily life (we would not have expected this statement!). Here you find all his replies:
Q: Do you know what synthetic biology is?
A: Yes
Q: How does synthetic biology impact society in your opinion?
A: Synthetic Biology will lead to the way that many types of “manufacturing” are done, in the widest sense of the word. This has both the potential for good, for example in easing shortages or developing processes that produce fewer pollutants or unwanted side-effects, but will also change how some people work.
Q: Throughout the years did your opinion on synthetic biology change?
A: No
Q: Did you hear of great accomplishments in the field that you thought would never be possible?
A: Thus far not. There was always a very large potential and so nothing I have seen has gone beyond the obviously possible so far.
Q: What is the biggest risk associated to synthetic biology?
A: I would say a lack of acceptance in society caused by a lack of information that is broadly accessible to enable members of the public to assess risk (cf GMO debate).
Q: How far is a future where synthetic biology applications are part of everyday life?
A: It is here now.
Q: Would you support students to undertake studies in synthetic biology?
A: Yes, there have been PhD students in EMBL already who have carried out projects that would be defined as synthetic biology.
Discussion
In performing this video project, we learnt a lot about how to approach famous and busy scientists, how they reason and how they feel about synthetic biology. In listening to their inspiring words, we got even more enthusiastic about synthetic biology and feel compelled to positively impact society in the future with our innovative ideas. It would have been fantastic to be able to collect more information on the opinion of scientists totally outside of the field, but this was somehow not possible. While the answers we got from Ian Mattaj clearly show that scientists working in unrelated field can have an opinion on synthetic biology, we should consider that the EMBL is a research institute where synthetic biology has been present. We fear that most scientists working in institutes where synthetic biology projects were never performed would not be able to define synthetic biology or would fail to see the difference between synthetic biology and biotechnology (well, we saw how for instance even some of the system biologists we interviewed still spoke of synthetic biology as biotechnology, so this distinction seems to be fuzzy even for those working in closely related field!). Synthetic biologists have still to go a long way to explain their approaches and goals to other researchers and to the lay public. We think that the iGEM foundation, by pushing teams to perform human practice projects during the iGEM competition, strongly accelerates the spread of knowledge on this fascinating new field of research and pushes us students to spend time reflecting on how important it is to explain what we do to the outside world .
