Team:Heidelberg/notebook/pc
Achievements
Results
Medal fulfillments
AptaBodies
Real time S.M.S.
Cystic fibrosis
Labeling and vizualization
Modeling and Software validation
Standardization
Project
Overview
AptaBodies
Real time S.M.S.
Cystic fibrosis
Labelling and Visualization
HRP based detection
Software
MAWS
JAWS
Wiki Tools
Notebook
AptaBodies
Real time S.M.S.
Cystic fibrosis
Labeling and vizualization
Pulse chase
Materials & Methods
Safety
Parts
Team
Members
Supervisors
Collaboration
Attributions
Sponsors
Modeling
Aptamer Sensors
In vitro transcription
Outreach
iGEM Team Heidelberg 2015
Achievements
Results
Project
Software
Notebook
Parts
Team
Modeling
Outreach
week number 26
▼2015-06-22  E.coli Trafo with Plasmids form the iGEM registry
To build the protein construct the mTagBFP BBa_K592100 from the iGEM registry were used.
1. With a pipette tip, punch a hole through the foil cover into the corresponding well of the part that you want. Do not remove the foil cover.
2. Pipette 10µl of dH2O (distilled water) into the well. Pipette up and down a few times and let sit for 5 minutes to make sure the dried DNA is fully resuspended.
3. Thawing 50µl chemical competent E. coli on ice.
4. Add to 50µl competent cells:
1µl DNA from the registry
10µl KCM 5x
39µl H2O
5. Incubate on ice for 30 minutes
6. Heat shock at 42°C for 2 minutes
7. Incubate on ice for 2 minutes
8. Add 900 µl of LB or 2x YT Medium
9. Incubate on 37°C for 60min
10. Centrifuge 5min at 1000g
11. Take 900µl of supernatant and throw away
12. Resuspend pellet in remaining media
13. Plate out on agar with the appropriate antibiotic and grow overnight at 37°C
 
▼2015-06-24  QIAprep Spin Miniprep Kit
Miniprep with mTagBFP BBa-K592100 and the pUB23-S-ßGal plasmid was performed.
1. Prepare o/n culture
2. Perform mini prep according to manufacturer’s protocol
Durchführung
▼2015-06-25  E. coli glycerol stocks
Glycerol stocks from E.coli containing mTag BFP BBa_K592100 or pUB23-S-ßGal were made.
1. Grow up an overnight culture of strains of interest
2. Transfer 500µl into a safe lock reaction tube
3. Add 500µl of 40% sterile glycerol solution
4. Freeze slowly at -80°C
Durchführung
▼2015-06-25  Fragment amplification for protein construct (PCR)
To obtain the required Gibson Fragments (Backbone (p413-GPD), mTagBFP, Ubiquitin, sfGFP) PCRs were performed.
 
reaction mixture:
12.5µl Q5 High-Fidelity 2x Master Mix
0.5µl DNA
1.25µl fwd primer 10µM
1.25µl rev primer 10µM
9.5µl H2O
 
Program:
95°C 2min
95°C 30sec
63-66°C 30sec (mTagBFP: 63°C; Ub: 65°C; sfGFP: 64°C; p415-GDP: 66°C)
72°C 30sec
72°C 5min
35 cycles
 
gel:
TAE buffer
1% w/v agarose
 
loading samples:
5µl Ladder
25µl PCR sample
5µl Loading buffer
▼2015-06-27  Fragment amplification for protein construct (PCR) (continued)
Results
PCRs for Gibson fragments mTagBFP, Ubiquitin and sfGFP worked because a bands are visible with the expected size. No p415-GPD product was visible. Therefore the p415-GPD PCR need to be optimised.
Gel Extraction
mTagBFP, Ubiquitin and sfGFP fragments were separated in 1% agaros.
Gel extraction was performed according to QIAGEN QIAquick Gel Extraction Protocol. (Elution in 30µl EB)
▼2015-06-27  Phusion PCR with DMSO and GC Buffer
To amplify the p415-GPD plasmid with Gibson overhangs an optimized gradient PCR program was used.
 
reaction mixture:
total:10µl
5.0µl Phusion 2x Master Mix
0.2µl DNA
0.5µl xxJD007xx 10µM
0.5µl xxJD008xx10µM
0.5µl DMSO
 
1.3 H2O
 
programm:
(PHGRAD)
95°C 2:00min
95°C 0:30min
58°C-70°C 0:30min
72°C 3:00min
72°C 5:00
35 cycles
 
gel:
TAE buffer
1% w/v agarose
 
loading samples:
5µl PCR sample
2µl Loading buffer
 
Results
no product. PCR need to be repeated.
 
▼2015-06-27  p415-GPD test digest
To check if the right plasmid is in the tube a KpnI digest was performed.
Set up reaction according to protocol:
ddH2O for a final volume of 20 µl
2 µl of 10x 1.1 Reaction Buffer
0.5 µl of KpnI
1 µl of DNA
Incubate at 37°C for 60'
Load on gel (add loading dye first)
 
Results
The gel shows the expected bands. The p415 plasmid is in the tube.
 
 
week number 27
▼2015-06-29  p415 PCR
Another p415 PCR was performed to amplify p415 for Gibson assembly.
 
reaction mixture:
12.5µl Phusion 2x Master Mix
1.2µl DNA 1:10 50ng
1.25µl xxJD007xx 10µM
1.25µl xxJD008xx 10µM
one with 1.25µl DMSO one without
add H2O to 25µl
 
program:
95°C 5min
98°C 30sec
63°C 30sec
72°C 2min
72°C 5min
35 cycles
 
Results
still not working
 
▼2015-07-02  p413 and sfGFP PCR
Because the amplification problems with the p415 plasmid could not be solved, the p413-GPD plasmid was used for the protein construct. Therefor sfGFP and p413 were amplified with new primers.
To be created: sfGFP, p413-GPD Gibson fragments
PCR (25 µl):
12.5 µl Q5 Master Mix
0.5 µl DNA Template (pMaM17; p413-GPD)
1.25µl fwd primer 10µM
1.25µl rev primer 10µM
9.5µl H2O
Cycler Program:
Initial denaturation: 2'@95°C
35x:
Denaturation: 30''@95°C
Annealing: 30'' (sfGFP: 64°C p413-GPD: 66°C)
Elongation: 3'@72°C
Final Elongation: 5'@72°C
Store @4°C
Results
sfGFP was used for gel extraction. p413 PCR need to be repeated.
 
 
 
Gel Extraction
The sfGFP Gibsonfragment was separated in 1% Agarose.
Gel extraction was performed according to QIAGEN QIAquick Gel Extraction Protocol. (Elution in 30µl EB)
 
▼2015-07-03  p413 PCR repeated
p413 amplification
PCR (25 µl):
12.5 µl Q5 Master Mix
0,5 µl DNA Template (p413-GPD)
1.25µl fwd primer 10µM
1.25µl rev primer 10µM
9.5µl H2O
Cycler Program:
Initial denaturation: 2'@95°C
35x:
Denaturation: 30''@95°C
Annealing: 30'' (p413-GPD: 64°C)
Elongation: 3'@72°C
Final Elongation: 5'@72°C
Store @4°C
 
 
 Results
The p413 products were used for gel extraction.
 
Gel Extraction
The backbone p413 was separated in 1% Agarose.
Gel extraction was performed according to QIAGEN QIAquick Gel Extraction Protocol. (Elution in 30µl EB)
After extraction due to minimal concentration the sample was lyophilized to obtain DNA pellet.
 
▼2015-07-04  Gibson Assembly
Samples mTagBFP, Ubiquitin, sfGFP and p413 were used for Gibson assembly. The reaction was carried out in the p413 tube containing the p413 pellet.
 
aIG
Size [bp]
c [ng/µl]
Volume [µl]
GFP
714
16,8
2,55
Ub
228
13,6
1,01
mTagBFP
731
34,5
1,27
 
Add water to a final volume of 10µl.
Add 10µl Gibson Assembly Master Mix.
Incubate for 60min at 50°C
 
2.5µl DNA was used for KCM Transformation
 
KCM Transformation
1. Take 50µl chemical competent E. coli from -80 freezer and thaw on ice
2. Add (as master mix):
2,5µl DNA
10µl KCM 5x
37,5µl H2O
3. Incubate on ice for 30 minutes
4. Heat shock at 42°C for 1 minute
5. Incubate on ice for 2 minutes
6. Add 900 µl of LB or 2x YT Medium
7. Incubate on 37°C for 60min
8. Centrifuge 5min at 1000g
9. Take 900µl of supernatant and throw away
10. Resuspend pellet in remaining media
11. Plate out on agar with antibiotics
 
Results
No E.coli were transformed. Therefore Gibson fragments were checked on an agarose gel.
 
▼2015-07-04  Fragment test for Gibson
p413, mTagBFP, Ub,GFP samples with Gibson overhangs after Gel Extraction were loaded on an 1% agarose gel.
 
Results
No fragments can be detected. Gibson PCR need to be repeated.
week number 28
▼2015-07-07  Fragment amplification
To obtain the required Gibson Fragments (Backbone (p413-GPD), mTagBFP, Ubiquitin, sfGFP) PCRs were performed.
For the backbone we performed the reaction with Thermo 2x Phusion Flash PCR Mastermix and with NEB Q5 2x High-Fidelity Mastermix.
The insert fragments were amplified using the Thermo 2x Phusion Flash PCR Mastermix.
All PCRs were performed as 25 µl reactions and set up in duplicates.
For the backbone:
PCR (25 µl):
12,5 µl NEB Q5 2x High-Fidelity Mastermix/ Thermo 2x Phusion Flash PCR Mastermix
0,25 µl DNA Template (p413-GPD)
1.25µl fwd primer 10µM
1,25µl rev primer 10µM
9,75 µl H2O
Cycler Program:
Initial denaturation: 2'@95°C
35x:
Denaturation: 30''@95°C
Annealing: 30''@66°C
Elongation: 3'@72°C
 
Final Elongation: 5'@72°C
Hold @4°C
 
For the inserts:
PCR (25 µl):
12,5 µl 2x Phusion Flash PCR Mastermix
0,5 µl DNA Template (BBa_K592100 prepped from Registry, pUB23-S-ßGal, pMaM17)
1.25µl fwd primer 10µM
1,25µl rev primer 10µM
9.5µl H2O
Cycler Program:
Initial denaturation: 2'@95°C
35x:
Denaturation: 30''@95°C
Annealing: 30'' (mTagBFP: 63°C; Ub: 65°C sfGFP: 64°C)
Elongation: 30''@72°C
 
Final Elongation: 5'@72°C
Hold @4°C
 The products will then be separated on a gel and purified.
 
▼2015-07-08  Fragments gel
To purify the fragments of the protein construct for Gibson assembly we chose to seperate on a gel: 
The samples from the PCR are loaded together with 6x NEB Purple Loading Dye on a 1% agarose TAE gel. 
Load scheme (25 µl per PCR reaction): 
#1: 5 µl NEB 2-Log DNA Ladder 
#2: p413-GPD (amplified with Phusion Flash) 
#3: p413-GPD (amplified with Phusion Flash) 
#4: p413-GPD (amplified with NEB Q5) 
#5: p413-GPD (amplified with NEB Q5) 
#6: mTagBFP (amplified with Phusion Flash) 
#7: mTagBFP (amplified with Phusion Flash) 
#8: Ubiquitin (amplified with Phusion Flash) 
#9: Ubiquitin (amplified with Phusion Flash) 
#10: sfGFP (amplified with Phusion Flash) 
#11: sfGFP (amplified with Phusion Flash)
▼2015-07-08  fragment test for Gibson
p413, mTagBFP, Ub,GFP samples with Gibson overhangs after Gel Extraction were loaded on a gel.
Results
BFP, Ub and GFP worked, p413 need to be repeated.
 
▼2015-07-10  p413 PCR
50µl in total:
 22.5 µl H2O
10 µl 5x buffer
2 µl 50 mM MgCl2
2x 2.5 µl primer
5 µl 10 mM dNTPs
5 µl 5 M betaine
0.25 µl template (~80 ng)
0.5 µl Velocity polymerase
 
1: 97 °C: 2 min
2: 97 °C: 30 s
3: 60 °C: 30 s
4: 72 °C: 3.5 min; -> 2 34x
5: 72 °C: 10 min
6: 4 °C inf
 
 
▼2015-07-10  p413 precipitation
150µl absolute ethanol (freezer)
25µl acetat
mix gently
incubate for 30min by 4°C
centrifuge 20min by 4°C
remove supernatant
add 100µl 70% enthanol
remove enthanol and let pellet dry
solve pellet in 20µl ddH2O
 
▼2015-07-11  DpnI digest of p413
20µl p413 precipitation
2µl CutSmart
1.5µl DpnI
37°C 60min
65°C 1min
gel:
1µl p413 digested
1µl loading buffer
5µl 2log Ladder
 
▼2015-07-11  Gibson assembly
0.62µl p413 digested
3.01µl mTagBFP 1:10
2.19µl Ubiquintin 1:10
3.09µl sfGFP 1:10
1.09µl H2O
10µl Gibson MM
60min 50°C
▼2015-07-11  E.coli transformation
Take 50µl chemical competent E. coli from -80 freezer and thaw on ice
Add (as master mix):
2,5µl DNA
10µl KCM 5x
37,5µl H2O
 
Incubate on ice for 30 minutes
Heat shock at 42°C for 1 minute
Incubate on ice for 2 minutes
Add 900 µl of LB or 2x YT Medium
Incubate on 37°C for 60min
Centrifuge 5min at 1000g
Take 900µl of supernatant and throw away
Resuspend pellet in remaining media
Plate out on agar with antibiotics (1:1 / 1:10)
Results
No colonies visible
▼2015-07-12  Colony PCR
5µl OneTag MasterMix
0.5µl primer 1:10
0.5µl primer 1:10
4µl H2O
Program:
3min 95°C
0:30min 95°C
0:30min 51°C
2min 68°C
3min 68°C
∞ 4°C
week number 29
▼2015-07-14  Yeast trafo
Because the protein construct could not be assembled via Gibson in E. coli a yeast transformation with the Gibson fragments was performed.
p413-GPD: 750ng (25µl)
GFP 500ng
Ubi 200ng
BFP 500ng
50µl competent yeast
300µl PEG
Add fragments and PEG to competent yeast
Incubate 40min at 42°C
Centrifuge at 500g for 3min
Remove supernatant
Resolve pellet in 100µl H20
Plate on –His plates 1:100 and rest
GFP fluorescence could be detected
 
▼2015-07-15  Gibson assembly of the protein construct after PCR purification of the DpnI digest
To increase the Gibson efficiency we purified our DpnI digest of the Backbone (p413-GPD) with the QIAGEN QIAquick PCR Purification Kit. Purification was performed according to manufacturers protocol. DNA was eluted with 30 µl H2O. (obtained concentration: 30.3 ng/µL)
With the insert fragments kept and the purified Backbone we performed a 20µL Gibson assembly reaction.
Solution
Amount
NEB Gibson Assembly Master Mix
10 µL
Backbone (p413-GPD)
2.18 µL
sfGFP (1:10 dilution)
2.91 µL
Ubiquitin (1:10 dilution)
2.06 µL
mTagBFP (1:10 dilution)
2.84 µL
 
Incubation: 1h at 50 °C
The reaction was transformed in aliquots of 0.5 µL and 2 µL into E.coli TOP10
Results:
On the plate transformed with 0.5 µL of Gibson reaction no colonies were observed. The plate with 2 µL of Gibson mix carried 3 colonies that went into colony PCR screening.
▼2015-07-15  Aussagekräftiger Titel
 
Equipment and reagents
TE: 10 mM Tris-HCl pH 7.5, 1 mM EDTA add 1.25µl of 10mg/ml Zymolyase and 875µl ß mercaptoethanol to 250µl TE before use
S buffer: 10 mM K2H PO4 pH 7.2, 10mM EDTA add 1.25µl of 10mg/ml Zymolyase and 875µl ß mercaptoethanol to 250µl S buffer before use
Lysis solution: 25mM Tris-HCl pH 7.5, 25 mM EDTA, 2.5% SDS
3 M natrium acetate pH 5.5
 
1. Scrape a large mass of yeast from a plate and resuspend it in 1ml of TE in a microcentrifuge tube (the OD600 of this suspension should be between two and five). Yeast from a fresh, two to three day-old plate work best. The yeast can also be obtained from a 2 ml overnight liquid culture.
2. Spin briefly to pellet the cells. Resuspend the yeast in 250µl of S buffer with Zymolyase and ß mercaptoethanol.
3. Incubate at 37°C for 30min
4. Add 55µl lysis solution. Vortex to mix.
5. Incubate at 65°C for 30min.
6. Add 83 µl 3 M sodium acetate. Chill on ice for 10 min
7. Spin in a microcentrifuge for 10 min. Pour the supernatant into a new tube.
8. Precipitate the DNA by adding 400µl cold ethanol. Incubate on ice for 10 min, spin for 10 min and pour off the supernatant.
9. Wash the pellet by centrifugation with 0.5 ml of 70% ethanol and dry the pellet.
10. Dissolve the pellet in 40 µl sterile water. Use 1-2 µl of this crude yeast mini-prep to transform E. coli by electroporation.
 
week number 30
▼2015-07-25  Yeast cell lysates using TCA
Cell sediment from 2-3 OD of cells was resuspended in 1 ml of cold water. Alternatively, resuspend cells (a match head) from a fresh plate in 1 ml water. Alternatively, just use 1 ml of your cells in the medium (does only work for synthetic media, rich media contain too much  TCA precipitable material). Add 150 µl of 1.85 M NaOH, 7.5% ß-mercaptoethanol to the resuspended cells. Incubate 10 min on ice. Than add 150 µl 55% TCA (w/v) to the resuspended cells. Incubate for 10 min on ice. Centrifuge at 4°C for 15 min in a microfuge (14000 rpm). Remove supernatant, centrifuge again and remove remaining supernatant. Resuspend precipitated proteins in 100-200 µl HU-buffer by vortexing and heating sample for 10-15 min at 65°C. Spin samples for 5-10 min at RT, full speed in Eppi centrifuge. Load sample (10-15 µl) onto SDS-PAGE.
HU-buffer (high-urea buffer):
8 M urea, 5% SDS, 200 mM NaHPO4 pH 6.8, 0.1 mM EDTA, bromphenol blue (according  to your taste), freeze at –20°C, before use, adjust to 15mg/ml DTT
▼2015-07-25  Western Blot
15µl Cell lysates were loaded on a 12% SDS Gel:
Resolivng Gel:
6.6 ml water
8 ml acrylamide mix 30
5 ml Tris 1.5 M pH 8.8
200µl 10% SDS
200µl 10% APS
20µl TEMED
Stacking Gel:
3.4 ml water
830 µl Acrylamid mix 30
630 µl Tris pH 6.8
50 µl 10% SDS
50µl 10% APS
5 µl TEMED
Western Blot
Cut pvdf membrane (9x6cm) and Blotting paper to the dimensions of the gel. One membrane and two pieces of extra thick filter paper per gel are needed
Activate membrane 5min in Methanol
Equilibrate gel in transfer buffer for 5min
Assembly:
1. Place a pre-soaked sheet of extra thick filter paper onto the platinum anode. Roll a pipet or test tube over the surface of the filter paper (like a rolling pin) to exclude all air bubbles.
2. Carefully place the equilibrated gel on top of the transfer membrane, aligning the gel on the center of the membrane. Transfer will be incomplete if any portion of the gel is outside the blotting media. Roll out all air bubbles.
3. Place the other sheet of pre-soaked filter paper on top of the gel, carefully removing air bubbles from between the gel and filter paper.
4. run Blot at 25 V for 1h
Prepare around 50ml 5% milk in TBT-T.
After blotting wash membrane three times with TBT-T for 5min
Block 1h or overnight with 5% milk, keep milk afterwards
Wash membrane three times, 5min with TBT-T
Prepare dilution of first antibody in 5 ml milk in a Falcon
Put membrane in the Falcon and let it roll at 4°C overnight
Wash membrane three times, 5min with TBT-T
Prepare dilution of second antibody in 5 ml milk in a Falcon
Put membrane in the Falcon and let it roll at 4°C for 1 h
Wash membrane three times, 5min with TBT-T
Prepare strain solution: 2.5 ml each in one Falcon
Put membrane in the Falcon and let it roll at 4°C for 5min
Pipet 50 µl H20 on a clear film, add membrane, pipet 50µl on the membrane and close clear film
 
week number 31
▼2015-07-28  p415 digest
p415 BsaI and BamHI digest:
0.5 µl BsaI
0.5µl BamHI
1 µl DNA
2 µl Cut Smart
16 µl water
1h 37°C
 
▼2015-07-28  p415 PCR DpnI digest, DNA precipitation und trafo in e.coli
DpnI digest:
0.5 µl DpnI
4.5 µl Cut Smart
45 µl DNA (Q5 p415, Ph p415, Ve p415)
37°C 60min
DNA precipitation:
20µl NaAc 3M pH 5,5
10 min on ice
Centrifuge for 10 min at 4°C max
Pour the supernatant into a new tube
Add 2.5 volumes of 96% cold ethanol 
10min on ice
Centrifuge max for 10 min at 4°C
Remove supernatant
Wash pellet with 500µl 70% ethanol
Dry pellet
Dissolve pellet in 10 µl sterile water
KCM Trafo
With p415 mutated BsaI side after PCR (Q5, Ph,Ve) DpnI degest and precipitation
 
▼2015-07-29  Chemical competent E. coli cells (CaCl method)
For Golden Gate Cloning preparation, one shot survival cells were made competent.
Chemical competent E. coli cells (CaCl method)
════════════════════════════════════════════════
• for ~ 200x 100 µl aliquotes
 Materials
 ─────────────
• 100 mM CaCl2 (aq) - autoclaved
• Glycerol
Day 1
 ─────────
 • grow bacteria over night in 50 ml medium at 37°C
 Day 2
 ─────────
1) transfer 20-30 ml of the ON culture into 400 ml LB
2) grow at 37°C until an OD of 0.5-0.6 is reached (3-5 hours), put
CaCl_{2} on ice, prepare aliquoting buffer (20 ml CaCl_{2} with 10%
glycerol)
3) centrifuge bacteria for 30 min at 3750 rpm, 4°C
• *work on ice and as quick as possible from here on*
• ==> you will need 8 falcons (50 ml) in order to do so
4) discard supernatant, resuspend bacteria in 5 ml CaCl_{2} / falcon,
pool 2 falcons, add up to 50 ml CaCl_{2}
• ==> you should have 4 falcons from here on
5) leave on ice for 30 min
6) centrifuge for 20 min at 3750 rpm, 4°C
7) discard supernatant, resuspend in 10 ml aliquoting buffer for all 4
falcons, add another 10 ml aliquoting buffer
• *Do not vortex*
8) pipette 100 µl per eppendorf tube (pre-cooled on ice), put aliquots
in -80 °C freezer
Competence of E.coli were tested.
 
▼2015-07-31  p415-GPD BsaI mutation PCR
22.3µl water
10µl 5x buffer
2µl 50mM MgCl2
2x 2.5µl pimer
5µl 10mM dNTPs
5µl DMSO
0.3 µl DNA
0.5µl Velocity
25µl Q5 MM
2x 2.5 µl primer
0.3 µl DNA
5µl DMSO
14.7µl water
Program:
97°C 3min
97°C 30sec
55°C 30sec
72°C 4min
72°C 5min
1-3 25x
 
▼2015-08-01  Miniprep p415 for “general plasmids”
▼2015-08-01  p415-GDP open PCR
Durchführung
P415-GDP open PCR (to get ccdB inside later on)
Velocity:
10 µl 5x buffer
2 µl 50mM MgCl2
2x 2.5 µl primer (JD 24,25)
5 µl 10mM dNTP
0.2 DNA
0.5 Velocity
27.3 µl water
 
Q5
25 µl Q5
2x 2.5 µl primer
0.2 DNA
14.6 µl water
 
Program:
97°C 3min
97°C 30 sec
72/68.6/61.4°C 30 sec
72°C 4min
30x
72°C 5min
4 for ever
Gel extraction with Q5 61.4 samples for Gibson.
▼2015-08-01  Gibson
Gibson:
One with ccdBg(106.5ng/µl) and one with ccdBk(359ng/µl) both with p415 (12.5ng/µl)
Gibson ccdBg:
P415 6.62µl
ccdBg 0.39µl
water 2.99µl
Gibson MM 10µl
 
Gibson ccdBk:
P415 6.62 µl
ccdBk 1:2 0.23
water 3.14 µl
Gibson MM 3.14 µl
 1h 50°C
 
▼2015-08-01  Yeast trafo with positive control construct
Method adapted from
Knop M, Siegers K, Pereira G, Zachariae W, Winsor B, Nasmyth KSchiebel E.  Epitope tagging of yeast genes using a PCR-based strategy: more tags and improved practical routines. Yeast, 15:963-972.
The transformation protocol was based on the LiOAc method (Schiestl & Gietz, 1989).
The following solutions were used:
SORB: 100 mM LiOAc, 10 mM Tris/HCl pH 8 (from 1 M stock), 1 mM EDTA/NaOH pH 8 (from 0.5 M stock), 1 M sorbitol (special grade for molecular biology from Merck), adjusted with diluted acetic acid to pH 8, sterile filtered (can be stored at room temperature for several months, a volume of 0.5 l was usually prepared).
PEG: 100 mM LiOAc, 10 mM Tris/HCl pH8 (from 1 M stock), 1 mM EDTA/NaOH (from 0.5 M stock) pH8 , 40% PEG3350 (Sigma), sterile filtered (can be stored at 4°C for several months, a volume of 50 to 100 ml was usually prepared).
Carrier DNA: Salmon sperm DNA (10 mg/ml, Gibco BRL, 2 kb average lenght) was boild for 10 min and cooled on ice. Such prepared carrier-DNA can be stored at -20°C and repeatedly used.
Growth of yeast cells:
Yeast cells were inoculated from a fresh pre-culture and grown for at least 6-8 hours or over night to a density of 1 to 2 OD600 at 30°C (approx. 4*107 cells per ml) in YPAD medium.
Competent yeast cells: Yeast cells were harvested by centrifugation (500 g, 5 min, room temperature), washed once with 0.1 to 0.5 vol sterile water (room temperature) and once with 0.1 to 0.2 vol SORB (room temperature). SORB was completely removed by aspiration. The cells were finally resuspended in a total volume of 360 ml SORB per 50 ml of cell culture and 40 ml of carrier DNA (0°C) was added. The cells were aliquoted into appropriate portions (e.g. 50 ml, at room temperature) and placed at -80°C (no shock freezing).
Transformation: Usually, 10 ml of competent cells were used for the transformation of plasmid DNA and 50 ml of cells for the transformation of a PCR product. The DNA was placed into a sterile 1.5 ml tube (maximal 2 ml plasmid DNA per 10 ml of cells), the competent cells were thawed and added. The suspension was mixed well before a 6 fold volume of PEG was added. Cells were mixed throughout and incubated at room temperature for approx. 30 min (up to 2 hours is no problem). DMSO was added (1/9 vol to make a final concentration of approx. 10%). The cells were placed in a water bath at 42°C for 5-20 min (15 min works best with most strains). The cells were sedimented (2-3 min at 2000 rpm), the supernatant was removed (since it slows down growth on the plates) and the cells were resuspended in 100 to 200 ml of liquid medium (YPD or selective medium, for kanMX6 selection see below).
Selection for transformants:
In cases were auxotrophy markers were used for selection of the plasmid or PCR product, cells were plated onto synthetic complete medium lacking the corresponding amino acid (His3MX6: SC-HIS plates; klTRP1: SC-TRP plates). In cases, where PCR-products containing the dominant resistance marker kanMX6 were used, the cells were resuspended in approx. 3 ml of YPAD, incubated on a shaker for 2-3 hours at 23 or 30˚C, harvested and spread on a G418 plate (further details are given in the text).
 
1µl DNA
10 µl cells
60 µl PEG
7.7 µl DMSO
 
 
 
▼2015-08-02  KCM Trafo with Gibson Assembly (see 2015-08-01)
ccdB resistant E. coli and top10 as control
▼2015-08-02  PCR to open p415 and p416 for ccdB
Velocity:
2µl buffer
2x 0.5µl pimer
1µl dNTP
1µl p415 1:10 or p416 1:10
0.2 Velocity
0/1/3 µl MgCl2
Add water to 10µl final volume
Q5
5µl Q5 MM
2µl GC buffer 
1µl p415 1:10 or p416 1:10
2x 0.5µl primer
1µl water
Program:
97°C 3min
97°C 30 sec    
67/63.9/59.6°C 30 sec
72°C 4min
30x
72°C 5min
4°C for ever
3x3 with ve, 3x with Q5, one p416ve and one p416Q5
 
week number 32
▼2015-08-04  Colony PCR with ccdBg and ccdBk
5µl One Tag
2x 0.5 µl primer
4 µl water
 
Program:
4min 95°C
30 sec 95°C
30 sec 55°C
2min 68°C
3min 68°C
 
 
A product with 784bp is expected when the ccdB is in p415
Positive colonies were grown for a test digest.
Entryvector test digest with BsaI
600ng p415
0.5 µl BsaI
 
Results:
If the entryvector assembled correctly bands with a side of 4854bp, 2050bp, 577bp and 123bp were expected. Entryvector 1, 2, 4, 5, 6 and 7 can be used.
▼2015-08-04  KCM Trafo
KCM Trafo with correct entryvectors in DH5α to check if it is functional. The original ccdB pSB1C3 plasmid was also transformed as a control.
Results:
With colonies which grow a miniprep and BsaI test digest were done.
 
▼2015-08-06  PCR ribozyme fragments
10µl 5xbuffer
2µl 50mM MgCl2
2x2.5 µl primer
5µl 10mM dNTP
1µl DNA 0.5 µl Velocity
26.5µl water
Program:
97°C 2min
97°C 30sec
61°C 30sec
72°C 45°C
72°C 5min
 
▼2015-08-07  PCR PCR ribozyme fragments optimised
0µl MgCl2
Phusion
Velocity
 
5µl 5x buffer
 
2x 1.25µl primer
 
2.5 µl dNTP
0.5 µl DNA
0.5 DNA
2x 1.25µl primer
0,25 Velocity
9.5 µl water
14.25 water
 
 
 
 
 
1.5µl MgCl2
Phusion
Velocity
 
5µl 5x buffer
 
2x 1.25µl primer
 
2.5 µl dNTP
0.5 µl DNA
0.5 DNA
2x 1.25µl primer
0,25 Velocity
8 µl water
12.75 water
1.5 µl MgCl2
1.5µl MgCl2
 
 
 
 
From some samples a gel extraction was done.
week number 33
▼2015-08-10  PCR optimized
Ve
Ve with DMSO
Phusion
5µl 5xbuffer
5µl 5xbuffer
5µl 5xbuffer
2x1.25µl primer
2x1.25µl primer
2x1.25µl primer
2.5µl 10mM dNTP
2.5µl 10mM dNTP
 
0.5 µl DNA
0.5 µl DNA
0.5 µl DNA
0.25 Ve
0.25 Ve
12.5 MM
14.25µl water
13.25µl water
9.5 µl water
 
1µl DMSO
 
 
 
Program:
95°C 2min
95°C 30sec
63-54°C 30sec
72°C 45 sec
72°C 5min
 
PCR purification with two of this samples gave to less DNA.
▼2015-08-11  Fragment check for Golden Gate
Fragment SH11 does not have the right side. New primers to build SH11 were ordered.
▼2015-08-11  Golden Gate
construct assembly, Golden Gate [Flo's recept?]
> ═════════════════════════════════════════════════
>
> • 15 µl raction mixture
> • 20 - 30 ng Backbone-Fragment
> • 1.5x des Insert-Fragments (in molar)
> • 1.5 µl NEBuffer 2.1
> • 1 µl NEB T4 Ligase (400 U / µl)
> • 0.75 µl BbsI (5 U / µl)
> • 1 mM ATP, 1 mM DTT (1.5 µl of 10 mM ATP/DTT stock)
>
> • Cycler program:
> ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━
> cycle temp time purpose
> ────────────────────────────────────────────────────────
> 50x (or less) 37 °C 2:00 restriction reaction
> – 16 °C 5:00 ligation reaction
> ────────────────────────────────────────────────────────
> 1x 50 °C 5:00 ligase inactivation
> ────────────────────────────────────────────────────────
> 1x 80 °C 5:00 restr. enzyme inactivation
> ────────────────────────────────────────────────────────
> 1x 12 °C inf hold
▼2015-08-12  E.coli Trafo
The Entryvector contains a BsaI restriction side in the Amp. resistance. That is why an additional ligation step was added.
1µl T4 DNA Ligase
20°C 20min
65°C 10min
 
With this product DH5alpha Transformation was performed following the usual protocol. Transformation was done with 0.5µl, 2µl and rest DNADurchführung
▼2015-08-13  Colony PCR
All Transformations worked well. Colony PCR was performed.
5µl OneTag MM
2x 0.5µl primer
4µl water
95°C 4min
95°C 30 sec
55°C 30 sec
68°C 2min
68°C 3min
 
Fragments with the side of 1094bp were expected.
Fragments with the side of 1112bp were expected.
 
▼2015-08-14  Miniprep, test digest, Trafo
With all 16 colonies Mini Prep and a test digest were performed.
PStI Hf
BsaI Hf
1µl DNA
2µl SmartCut
2x 0.5µl Enzyme
16.5 µl water
 
 
 
Expected Fragments:
935 bp, 2038bp, 4892bp
 
 
 
Expected Fragments:
951 bp, 2043bp, 4924bp
 
Sample SH09L1 and JD28L1 were used for yeast transformation. For this co-transformation the Entryvector g2 and either SH09L1 or JD28L1 were used.
500ng from each DNA
7.7 µl DMSO
 
Glycerol stocks were made from Entryvector g2, g5, SH09L1 and JD28L1
 
▼2015-08-15  Cloning ccdB in p415 via restriction enzymes
p415 digest:
2µl CutSmart
1µl PstI HF
1µl HindIII HF
1.9 µl p415 (1µg)
14.1 nuclease free water
 
ccdBg:
2µl CutSmart
1µl Psti HF
1µl HindIII HF
9.38 µl ccdBg
6.62 water
 
ccdBk:
2µl CutSmart
1µl Psti HF
1µl HindIII HF
2.7 µl ccdBk
13.3 µl water
 
At 37°C over night
 
PCR purification
 
Ligation
ccdBg:
2 µl 10x T4 DNA Buffer
1µl p415 (50ng= 0.0111pmol)
0,52 µl ccdB (25.8ng= 0.0557pmol)
1µl T4 Ligase
15.48 µl water
 
E. coli trafo with ccdB resistant E.coli ccdBg or ccdBk or efficient check
Efficient check with Transformation Efficiency Kit BBa_J04450 psB1C3
 
 
ccdBg or ccdBk:
5 µl DNA
10 µl KCM 5x
35 µl water
Rest see protocol “KCM Transformation E.coli
▼2015-08-15  yeast colony PCR
Colonies from PC SH09L1 and PC JD28L1 plates were taken, put in –His-Leu medium and a colony pcr was performed.
Colony PCR to check yeast transformation
Sample preparation:
Pick a small amount of cells with pipet tips and transfer in 5µl water
Add 20µl 20mM NaOH each
Heat samples at 99°C for 10min using PCR machine
Vortex samples vigorously for 10-20 s
Spin tubes for 1min to pellet brocken cells
PCR reaction:
10µl 2xMM
2x0.5µl primer
0.4 DMSO
8.6 µl supernatant of cell lysis
Run PCR:
94°C 2min
94°C 30s
55°C 30s
72°C 1min
Repeat step 2 to 4 35x
72°C 5min
Products were loaded on a gel
 
Bands by 1782bp and 1092bp or 1103bp were expected. Clear products were only visible for the ribozyme constructs. That is why yeast transformation was repeated.
 
▼2015-08-15  PCR
PCR to check if p413-myc-BFP-Ub-GFP is contaminated.
12.5 µl Phusion Master Mix
1.25 µl xxDH_08p413GPD revxx
1.25 µl xxDH_07p413GPD fwdxx
5µl p413 empty or with protein construct
Add water to 25µl 
Program:
97°C 5min
97°C 30sec
52°C 30sec
72°C 1.5min
Step 2 to 4 35 times
72°C 3min
4°C for ever
Results: The sample with p413 with protein construct is not contaminated.
 
▼2015-08-15  PCR optimization for the p415 series of plasmids
To solve the problems observed with the p415-GPD plamid in PCR reactions we tried to set up an reaction with 1M betaine and 5% DMSO. This mixture has proved to be capable to solve PCR problems with ribozyme secondary structures. Therefore we hoped to solve the issues involving p415-GPD
Setup in 50 µL reactions:
 
Volume
2x Phusion Flash PCR Mastermix
25 µL
Template DNA
1 µL
fwd. Primer (DH07)
2.5 µL
rev. Primer (DH08)
2.5 µL
ddH2O
6.5 µL
Betaine (5 M)
10 µL
DMSO
2.5 µL
 
Annealing temperatures were varied in a gradient.
#
Temperature
1
72 °C
2
68.8 °C
3
62.0 °C
4
59.4 °C
5
56 °C
 
Program (Phusion):
2 min
98 °C
35x
15 s
98 °C
20 s
Gradient
2 min 20 s
72 °C
5 min
72 °C
infinite
4 °C
 
Analysis on a 1% agarose gel showed for the reactions with annealing at 62 °C, 59.4 °C and 56 °C the presence of a band at the expected size of ~7kb. The reaction with annealing at 62 °C showed not only the clearest appearance but also the highest yield for the wanted product.
▼2015-08-16  Test digest of received plasmids
To verify we were given valid plasmids from Ilia, we performed a test digest with BsaI and EcoRI. These two enzymes should produce, according to a Geneious virtual digest, a restriction fragment pattern easily distinguished.
To be analyzed:
In DNA minipreps: p413-ADH (#1/#2), p413-cyc1, p413-Gal1, p413-TEF (#1/#2), p415-ADH (#1/#2), p415-cyc1, p415-Gal1, p415-TEF (#1/#2)
Reactions were set up as following:
 
Volume
Miniprep DNA from Plasmids
1 µL
BsaI-HF (NEB)
0.2 µL
EcoRI-HF (NEB)
0.2 µL
CutSmart Buffer (NEB)
1 µL
ddH2O
7.6 µL
 
After digestion for 1 min at 37 °C the reaction mixture was analyzed via 1% agarose gel.
The restriction fragments appeared with sizes according to the expected fragments. Yet for the p413/p415-TEF digestions it appeared that there has been a swap between a p413-TEF and a p415-TEF sample. Before use of a p41x-TEF it is necessary to recheck for the correct backbone.
▼2015-08-16  PCR optimization for BFP site-directed mutagenesis
The SalI site in the BFP of our protein construct (negative and myc-tag positive) has to be removed, to allow for easy recloning into a library of p413 Vectors with different strength promoters via BamHI and SalI.
To find the perfect conditions we resorted to PCR optimization.
Temperature conditions: 72 °C, 66 °C, 60 °C
Polymerases: 2x Phusion Flash PCR Mastermix (Thermo), Q5 2x High-Fidelity Mastermix (NEB)
Buffer solution: Standard Mastermix conditions or addition of 1 M Betaine and 5 % DMSO to counter secondary structure behavior.
Standard buffer conditions:                               
 
Volume
2x Polymerase Mastermix
5 µL
Template DNA
0.2 µL
fwd. Primer (DH61)
0.5 µL
rev. Primer (DH62)
0.5 µL
ddH2O
3.8 µL
 
DMSO/Betaine buffer conditions:
 
Volume
2x Polymerase Mastermix
5 µL
Template DNA
0.2 µL
fwd. Primer (DH61)
0.5 µL
rev. Primer (DH62)
0.5 µL
ddH2O
1.3 µL
Betaine (5 M)
2 µL
DMSO
0.5 µL
 
Reactions were prepared as mastermixes and distributed into 0.2 ml PCR tubes. Cyclinge programs were chosen differently for each polymerase and according to manufacturer’s guideline.
Program (Phusion Flash):
15 s
98 °C
35x
5 s
98 °C
5 s
Gradient (72 °C, 66 °C, 60 °C)
2 min
72 °C
5 min
72 °C
infinite
4 °C
 
Program (Q5):
1 min
98 °C
35x
10
98 °C
20 s
Gradient (72 °C, 66 °C, 60 °C)
4 min
72 °C
5 min
72 °C
infinite
4 °C
 
The reactions were then mixed with NEB Purple Loading dye and analyzed on a 1% Agarose Gel.
The strongest and most defined bands, to be expected around 7.5 kb were observed with Phusion Flash polymerase in Mastermix buffer at 66° C annealing temperature. We will therefore try to perform the site directed mutagenesis under these conditions.
week number 34
▼2015-08-17  Repetition of site-directed mutagenesis to remove SalI site for recloning
The  reaction was repeated for PC (2.1) and myc-Pos (#1) with triplicates at annealing temperatures 60 °C, 64 °C, 66 °C in 50 µL reactions:
 
Volume
2x Phusion Flash PCR Mastermix
25 µL
Template DNA (1 ng/µL Dilution)
1 µL
fwd. Primer (DH61)
2.5 µL
rev. Primer (DH62)
2.5 µL
ddH2O
19 µL
 Program:
15 s
98 °C
35x
5 s
98 °C
5 s
Gradient (60 °C, 64 °C, 66 °C)
2 min
72 °C
5 min
72 °C
infinite
4 °C
 
Following PCR each 50 µL reaction mix was digested with 0,25 µL DpnI and 0,25 µL SalI to destroy template DNA and linearize the product. The reaction mix was incubated 90 min at 37 °C and heat inactivated for 5 min at 80 °C.
For verification 5µL of each reaction were loaded onto a 1% Agarose gel.
The gel showed, apart from little blur, no sign of the expected product. Restaining of the gel with ethidium bromide solution did not improve gel image.
Further processing of the product was aborted.
▼2015-08-18  Yeast co-transformation
PC/p415, pos/p415, SH09L/PC, JD28L/PC
PC=PC2.1 pos=#1 SH09L=SH09L1 JD28L=JD28L1
1µl each were taken
Sequencing
15µl 30-50ng/µl
SH09L1
SH09L5
JD28L1
JD28L2
 
▼2015-08-19  Ribozyme assay
Following yeast strains were used:
Yeast containing the protein construct in p413 only
Yeast containing the positive construct in p413
Yeast containing the p413 with the protein construct and p415 with the ribozyme (SH09)
Yeast containing the p413 with the protein construct and p415 with theophylline inducible ribozyme (JD28)
 An overnight culture of each was diluted to OD=0.15
Cells grow approx. 4h @ 30 deg shaker until they reach OD of 0.5
1ml t0 samples of each were taken, two the samples 150 µl of 1.85 M NaOH, 10µl ß-mercaptoethanol were added and then shock froze in liquid nitrogen
 
After t0 samples were taken, theophylline was added the following:
PC (protein construct): 0mM theophylline
Pos (positive construct): 0mM theophylline
SH09-0 (ribozyme): 0mM theophylline
SH09-5: 5mM theophylline
JD28-0 (inducible ribozyme): 0mM theophylline
JD28-2.5 : 2.5mM theophylline
JD28-5: 5mM theophylline
JD28-10: 10mM theophylline
 
After two hours (approx. OD=0.5) 1ml t1 samples were taken and treated as previously described. Cultures were diluted in such a way that the theophylline concentration remainders the same.
T2 (4h) T3 (5.5h) and t4 (16.5h) samples were taken as described above.
Then all samples were treaded the following:
Then add 150 µl 55% TCA (w/v) to the cells. Incubate for 10 min on ice. Centrifuge at 4°C for 15 min in a microfuge (14000 rpm). Remove supernatant, centrifuge again and remove remaining supernatant. Resuspend precipitated proteins in 100-200 µl HU-buffer by vortexing and heating sample for 10-15 min at 65°C. Spin samples for 5-10 min at RT, full speed in Eppi centrifuge. Load sample (10-15 µl) onto SDS-PAGE. SDS-PAGE run at 160V max ampere for 90min.
Than a semi-dry blot was performed as described above.
Membranes were treaded with anti-myc antibody (for more details see Western blot protocol above)
Results: no activity of any ribozyme, neither SH09 nor JD28, can be observed.
▼2015-08-22  colony PCR
A colony PCR was performed to check if both ribozyme and protein construct are in the tested yeast.
Weak bands in the expected side were detected, also the positive control only gave weak bands.
week number 35
▼2015-08-25  RNA extraction
Because the functionality of the ribozyme couldn’t be shown on protein level, the next step is to show functionality on RNA level. Therefore RT-PCR was performed with positive control samples and SH09 samples. Old means that the samples were also used for the Western Blot described above.
RNA extraction
1. Grow yeast cells in 10ml of desired medium to mid-exponential phase (OD600=1.0).
2. Transfer culture to 50-ml centrifuge tube and centrifuge cells 3min at 1500g, 4°C
3. Discard supernatant, resuspend pellet in 1ml ice-cold water. Transfer to clean 1.5-ml microcentrifuge tube. Microcentrifuge 10 sec at 4°C, and remove supernatant.
4. Resuspend pellet in 300µl RNA buffer: 0.5M NaCl, 200mM Tris Cl, pH 7.5, 10mM EDTA
Disrupt the cells
5. Add a volume of chilled acid-washed glass beads equivalent to 200ml water
6. Add 300µl of 25:24:1 phenol/chloroform/isoamyl alcohol equilibrated with RNA buffer.
7. Close the cap, then invert and shake up and dwon to ensure that the beads are suspended. Vortex vigorously for 2min at highest speed.
8. Microcentrifuge 1min at room temperature. Transfer aqueous layer to a clean microcentrifuge tube.
9. Add an equal volume of 25:24:1 phenol/chloroform/isoamyl alcohol. Vortex vigorously 10 sec.
10. Repeat step 8.
Precipitate the RNA
0.1 volume of 3M NaAC
2.5 volume of ethanol
Incubate at least 2 hours at -20°C, centrifuge for 30min at 4°C full speed. Discard supernatant and resuspend pellet in 20µl water.
DNA digest, Precipitate the RNA as described above.
▼2015-08-26  RT-PCR
4µl RNA
1µl dNTPmix (10mM each)
1µl primer (2µM)
7µl water
 
65°C, 5min, on ice, spin down
4µl 5xFirst buffer
1µl 0.1M DTT
1µl RNase inhinitor
1µl SuperScript III RT or for –RT control 1µl water
 
55°C, 1h for gene specific primer: xxJD33RTmycxx, xxJD34RTgfpxx
50°C oligo dT
70°C, 15min
150mM NaOH, 37°C, 15min
150mM HCl
DNA precipitation with EtOH and NaOAc as described above.
▼2015-08-27  PCR with cDNA
With all four samples (pos old, pos new, ribozyme old, ribozyme new) PCRs with following primers were performed:
xxDH076xx and xxJD034xx
xxDH076xx and xxJD033xx
SH021 and JD033xx
Program:
94°C 4min
94°C 30 sec
60°C 30sec
72°C 45 sec
72°C 10min
4°C for ever
 
Samples were sent for sequencing.
▼2015-08-28  Sequence information
Sequence of the ribozyme can be confirmed. Sequencing doesn’t show any ribozyme product: GFP-myc in ribozyme samples.
week number 36
▼2015-08-31  Labeling of RNA with alkyne-NTP
The labeling RNA was modified with alkyne NTPs by Terminale Dinucleotidyl Transferase (TdT) and as a second approach, by Poly A Polymerase (yeast), affimetrix.
The modified NTPs are from Jena Bioscience:
5-Ethynyl-dUTP (5-EdUTP)
C8-Alkyne-dUTP
C8-Alkyne-dCTP
 
cStock
cFinal
V[µL]
TdT buffer
10 x
1 x
2.5
Labeling RNA
84,97µM
15 µM
4.41
Alkyne-NTP
10 mM
400 µM
1
TdT
20,000 u/mL
80 u/mL
1
CoCl2
2.5 mM
0.25 mM
2.5
H2O
 
Add 25 µL
13.59
Final
 
 
25
 
 
cStock
cFinal
V[µL]
PAP (yeast)
600 u/µL
24 u/µL
1
PAP buffer
5 x
1 x
5
5-EdUTP/ C8-Alkyne-dUTP/C8-Alkyne-dCTP
10 mM
400 µM
1
Labeling RNA
84.97µM
15µM
4.41
H2O
 
Add 25 µL
13.59
Final
 
 
25
 
All compounds were mixed as indicated in the tables.
Reactions were incubated for 2 hours at 37 °C
Samples were stored at -20 °C oN
▼2015-09-01  Biotin modification of substrate RNA
For 25µL
Cstock
Cfinal
V [µL]
TdT- Buffer
10x
1x
2.5
RNA
71.804µM
10µM
3.48
Biotin - NTP
1mM
200µM
5
TdT
600U/µL
24U/µL
1
CoCl2
2.5mM
0.5mM
5
MQ H2O
 
 
8.02
 
All samples were mixed together and incubated for 2 hours at 37 °C.
▼2015-09-01  Click reaction (modified)
new Alexa 488 with DMSO
 
For 50µL
Cstock
Cfinal
V[µL]
Phosphate Buffer - pH 7, 0.1M
100mM
50mM
25
Alexa 488 azide
10µM
400nM
2
RNA
1µM
200nM
10
CuSO4
50mM
1mM
1
THPTA
50mM
5mM
5
NaAsc
100mM
1mM
0.5
H2O
 
 
6.5
 
All samples were mixed together and then incubated for 2hours at 37 °C.
▼2015-09-02  In vitro transcription of Labeling RNA AU + substrate HRP
 
cStock
cFinal
V[µL]
Transcription buffer
10 x
1 x
20
ATP
100 mM
4 mM
8
GTP
100 mM
4 mM
8
CTP
100 mM
4 mM
8
UTP
100 mM
4 mM
8
DTT
1 M
10 mM
2
DMSO
 
5 %
10
FS_53/54 (Label AU)
 
 
20
T7 RNA Polymerase
2 mg/mL
0,1 mg/mL
5
H2O
 
ad 100 µL
106
Final
 
 
400
 
 
cStock
cFinal
V[µL]
Transcription buffer
10 x
1 x
20
ATP
100 mM
4 mM
8
GTP
100 mM
4 mM
8
CTP
100 mM
4 mM
8
UTP
100 mM
4 mM
8
DTT
1 M
10 mM
2
DMSO
 
5 %
10
FS_50/51 (Substrate HRP)
 
 
20
T7 RNA Polymerase
2 mg/mL
0,1 mg/mL
5
H2O
 
ad 100 µL
106
Final
 
 
200
 
All compounds were mixed together and incubated for 3 hours at 37 °C. Afterwards, 2µL DNAseI was added for 30min at 37 °C.
week number 37
▼2015-09-10  DNAzyme Activity: DNAzyme with ATP aptamer and calculated Kanamycin aptamer
Samples:
10-23 DNAzyme: xxfs032xx
7-18 DNAzyme: xxfs033xx
10-23 DNAzyme with ATP aptamer with linker: xxfs019xx
7-18 DNAzyme with ATP aptamer with linker: xxfs027xx
10-23 DNAzyme with ATP aptamer A: xxfs017xx, B: xxfs018xx
7-18 DNAzyme with ATP aptamer A: xxfs025, B: xxfs026xx
10-23 DNAzyme with calculated Kan aptamer: xxfs034xx
7-18 DNAzyme with calculated Kan aptamer candidate I: xxfs035xx
7-18 DNAzyme with calculated Kan aptamer candidate II: xxfs036xx
7-18 DNAzyme with calculated Kan aptamer candidate III: xxfs037xx
Stock solutions and conditions:
 
cStock
cFinal
Tris HCl ph 7.5
1 M
50 mM
DNAzyme (A)
10 µM
500 nM
DNAzyme B
10 µM
500 nM
Substrate
1 µM
200 nM
NaCl
1 M
100 mM
MgCl2
1 M
20 mM
SDS
20 %
0,01 %
Adenosine in H2O:DMSO 1:2
33 mM
5 mM
H2O
 
ad 25 µL
 
Pipetting scheme:
#
 
 
Tris HCl ph 7.5
DNAzyme A
DNAzyme B
Substrate
NaCl
MgCl2
SDS
Adenosine
H2O
Final
1
FS032
10-23D
1,25
1,25
0,00
5,00
2,50
0,50
1,25
3,75
9,50
25,00
2
FS033
7-18D
1,25
1,25
0,00
5,00
2,50
0,50
1,25
3,75
9,50
25,00
3
FS019
10-23DmLink
1,25
1,25
0,00
5,00
2,50
0,50
1,25
3,75
9,50
25,00
4
FS027
7-18DmLink
1,25
1,25
0,00
5,00
2,50
0,50
1,25
3,75
9,50
25,00
5
FS017+18
10-23D_A+B
1,25
1,25
6,25
5,00
2,50
0,50
1,25
3,75
3,25
25,00
6
FS025+26
7-18D_A+B
1,25
1,25
6,25
5,00
2,50
0,50
1,25
3,75
3,25
25,00
7
FS032
10-23D
1,25
1,25
0,00
5,00
2,50
0,50
1,25
0,00
13,25
25,00
8
FS033
7-18D
1,25
1,25
0,00
5,00
2,50
0,50
1,25
0,00
13,25
25,00
9
FS019
10-23DmLink
1,25
1,25
0,00
5,00
2,50
0,50
1,25
0,00
13,25
25,00
10
FS027
7-18DmLink
1,25
1,25
0,00
5,00
2,50
0,50
1,25
0,00
13,25
25,00
11
FS017+18
10-23D_A+B
1,25
1,25
6,25
5,00
2,50
0,50
1,25
0,00
7,00
25,00
12
FS025+26
7-18D_A+B
1,25
1,25
6,25
5,00
2,50
0,50
1,25
0,00
7,00
25,00
13
FS032
-10-23D
1,25
1,25
0
0
2,5
0,5
1,25
0
18,25
25,00
14
FS033
-7-18D
1,25
1,25
0
0
2,5
0,5
1,25
0
18,25
25,00
15
FS019
-10-23DmLink
1,25
1,25
0
0
2,5
0,5
1,25
0
18,25
25,00
16
FS027
-7-18DmLink
1,25
1,25
0
0
2,5
0,5
1,25
0
18,25
25,00
17
FS017+18
-10-23D_A+B
1,25
1,25
6,25
0
2,5
0,5
1,25
0
12
25,00
18
FS025+26
-7-18D_A+B
1,25
1,25
6,25
0
2,5
0,5
1,25
0
12
25,00
19
Adenosine
Substrate only
1,25
0
0
5
2,5
0,5
1,25
3,75
10,75
25,00
 
 
#
 
 
Tris HCl ph 7.5
DNAzyme A
DNAzyme B
Substrate
NaCl
MgCl2
SDS
Kan
H2O
Final
20
FS032
10-23D
1,25
1,25
0
5
2,5
0,5
1,25
1,25
12
25
21
FS033
7-18D
1,25
1,25
0
5
2,5
0,5
1,25
1,25
12
25
22
FS034
Kan
1,25
1,25
0
5
2,5
0,5
1,25
1,25
12
25
23
FS035
Kan I
1,25
1,25
0
5
2,5
0,5
1,25
1,25
12
25
24
FS036
Kan II
1,25
1,25
0
5
2,5
0,5
1,25
1,25
12
25
25
FS037
Kan III
1,25
1,25
0
5
2,5
0,5
1,25
1,25
12
25
26
FS032
10-23D
1,25
1,25
0
5
2,5
0,5
1,25
0
13,25
25
27
FS033
7-18D
1,25
1,25
0
5
2,5
0,5
1,25
0
13,25
25
28
FS034
Kan
1,25
1,25
0
5
2,5
0,5
1,25
0
13,25
25
29
FS035
Kan I
1,25
1,25
0
5
2,5
0,5
1,25
0
13,25
25
30
FS036
Kan II
1,25
1,25
0
5
2,5
0,5
1,25
0
13,25
25
31
FS037
Kan III
1,25
1,25
0
5
2,5
0,5
1,25
0
13,25
25
32
Kan
Substrate only
1,25
0
0
5
2,5
0,5
1,25
1,25
13,25
25,00
33
 
Substrate only
1,25
0
0
5
2,5
0,5
1,25
0
14,5
25,00
 
Results and Outlook:
Positive controls worked, Adenosine dependency could be detected for one candidate.
▼2015-09-10  Click reaction of Label-, Label AU RNA [1, 2, 3]
For 51.5µL
Cstock
Cfinal
V[µL]
Phosphate Buffer - pH 7, 0.1M
100mM
50mM
25
Alexa 488 azide
10µM
400nM
2
RNA
1µM
200nM
10
CuSO4
20mM
1mM
2.5
THPTA
50mM
5mM
5
NaAsc
100mM
1mM
0.5
H2O
 
 
6.5
 
Alexa 488 azide was solved in DMSO
Incubation at 37 °C for 12-14 hours (overnight)
week number 38
▼2015-09-14  Test bitte löschen
10 x <i>in vitro</i> transcription buffer
50 mM Tris ph 7.5, 100 mM NaCl, 20 mM MgCl<sub>2</sub>, 0,01 % SDS
 
