<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_Heidelberg_notebook_cf skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:Heidelberg/notebook/cf</SPAN></H1><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><DIV class="container" id="menucontainer"><DIV id="navcontainer-right-outer"><DIV id="navcontainer-right-inner"><UL class="nav navbar-nav navbar-right"><LI><A href="https://2015.igem.org/Team:Heidelberg/Achievements" class="menubox"><P class="menutext">Achievements</P></A></LI><LI class="dropdown"><A href="https://2015.igem.org/Team:Heidelberg/Results" class="dropdown-toggle menubox" data-toggle="dropdown" role="button" aria-haspopup="true" aria-expanded="false"><P class="menutext">Results
                        </P></A></LI><LI class="dropdownbox"><A href="https://2015.igem.org/Team:Heidelberg/Results#Medalfulfillments"><P class="menutext dropdown-text">Medal fulfillments</P></A></LI><LI class="dropdownbox"><A href="https://2015.igem.org/Team:Heidelberg/Results#AB"><P class="menutext dropdown-text">AptaBodies</P></A></LI><LI class="dropdownbox"><A href="https://2015.igem.org/Team:Heidelberg/Results#RTSMS"><P class="menutext dropdown-text">Real time S.M.S.</P></A></LI><LI class="dropdownbox"><A href="https://2015.igem.org/Team:Heidelberg/Results#LandV"><P class="menutext dropdown-text">Cystic fibrosis</P></A></LI><LI class="dropdownbox"><A href="https://2015.igem.org/Team:Heidelberg/Results#CF"><P class="menutext dropdown-text">Labeling and vizualization</P></A></LI><LI class="dropdownbox"><A href="https://2015.igem.org/Team:Heidelberg/Results#Modelling"><P class="menutext dropdown-text">Modeling and Software validation</P></A></LI><LI class="dropdownbox"><A href="https://2015.igem.org/Team:Heidelberg/Results/Standardization"><P class="menutext dropdown-text">Standardization</P></A></LI></UL><LI class="dropdown"><A href="https://2015.igem.org/Team:Heidelberg/Overview" class="dropdown-toggle menubox" data-toggle="dropdown" role="button" aria-haspopup="true" aria-expanded="false"><P class="menutext">Project
                        </P></A></LI><LI class="dropdownbox"><A href="https://2015.igem.org/Team:Heidelberg/Overview"><P class="menutext dropdown-text">Overview</P></A></LI><LI class="dropdownbox"><A href="https://2015.igem.org/Team:Heidelberg/Project/AB"><P class="menutext dropdown-text">AptaBodies</P></A></LI><LI class="dropdownbox"><A href="https://2015.igem.org/Team:Heidelberg/project/rtsms"><P class="menutext dropdown-text">Real time S.M.S.</P></A></LI><LI class="dropdownbox"><A href="https://2015.igem.org/Team:Heidelberg/project/cf"><P class="menutext dropdown-text">Cystic fibrosis</P></A></LI><LI class="dropdownbox"><A href="https://2015.igem.org/Team:Heidelberg/project/rd"><P class="menutext dropdown-text">Labelling and Visualization</P></A></LI><LI class="dropdownbox"><A href="https://2015.igem.org/Team:Heidelberg/project/hlpd"><P class="menutext dropdown-text">HRP based detection</P></A></LI><LI class="dropdown"><A href="https://2015.igem.org/Team:Heidelberg/Software" class="dropdown-toggle menubox" data-toggle="dropdown" role="button" aria-haspopup="true" aria-expanded="false"><P class="menutext">Software
                        </P></A></LI><LI class="dropdownbox"><A href="https://2015.igem.org/Team:Heidelberg/software/maws"><P class="menutext dropdown-text">MAWS</P></A></LI><LI class="dropdownbox"><A href="https://2015.igem.org/Team:Heidelberg/software/jaws"><P class="menutext dropdown-text">JAWS</P></A></LI><LI class="dropdownbox"><A href="
https://2015.igem.org/Team:Heidelberg/WikiTools"><P class="menutext dropdown-text">Wiki Tools</P></A></LI><LI class="dropdown"><A href="https://2015.igem.org/Team:Heidelberg/notebook/mm" class="dropdown-toggle menubox" data-toggle="dropdown" role="button" aria-haspopup="true" aria-expanded="false"><P class="menutext">Notebook
                        </P></A></LI><LI class="dropdownbox"><A href="https://2015.igem.org/Team:Heidelberg/notebook/ab"><P class="menutext dropdown-text">AptaBodies</P></A></LI><LI class="dropdownbox"><A href="https://2015.igem.org/Team:Heidelberg/notebook/rtsms"><P class="menutext dropdown-text">Real time S.M.S.</P></A></LI><LI class="dropdownbox"><A href="https://2015.igem.org/Team:Heidelberg/notebook/cf"><P class="menutext dropdown-text">Cystic fibrosis</P></A></LI><LI class="dropdownbox"><A href="https://2015.igem.org/Team:Heidelberg/notebook/rd"><P class="menutext dropdown-text">Labeling and vizualization</P></A></LI><LI class="dropdownbox"><A href="https://2015.igem.org/Team:Heidelberg/notebook/pc"><P class="menutext dropdown-text">Pulse chase</P></A></LI><LI class="dropdownbox"><A href="https://2015.igem.org/Team:Heidelberg/notebook/mm"><P class="menutext dropdown-text">Materials &amp; Methods</P></A></LI><LI class="dropdownbox"><A href="https://2015.igem.org/Team:Heidelberg/Safety"><P class="menutext dropdown-text">Safety</P></A></LI><LI><A href="https://2015.igem.org/Team:Heidelberg/Parts" class="menubox"><P class="menutext">Parts</P></A></LI><LI class="dropdown"><A href="https://2015.igem.org/Team:Heidelberg/Team" class="dropdown-toggle menubox" data-toggle="dropdown" role="button" aria-haspopup="true" aria-expanded="false"><P class="menutext">Team
                        </P></A></LI><LI class="dropdownbox"><A href="https://2015.igem.org/Team:Heidelberg/Team"><P class="menutext dropdown-text">Members</P></A></LI><LI class="dropdownbox"><A href="https://2015.igem.org/Team:Heidelberg/Supervisors"><P class="menutext dropdown-text">Supervisors</P></A></LI><LI class="dropdownbox"><A href="https://2015.igem.org/Team:Heidelberg/Collaborations"><P class="menutext dropdown-text">Collaboration</P></A></LI><LI class="dropdownbox"><A href="https://2015.igem.org/Team:Heidelberg/Attributions"><P class="menutext dropdown-text">Attributions</P></A></LI><LI class="dropdownbox"><A href="https://2015.igem.org/Team:Heidelberg/Sponsors"><P class="menutext dropdown-text">Sponsors</P></A></LI><LI class="dropdown"><A href="https://2015.igem.org/Team:Heidelberg/Modeling" class="dropdown-toggle menubox" data-toggle="dropdown" role="button" aria-haspopup="true" aria-expanded="false"><P class="menutext">Modeling
                        </P></A></LI><LI class="dropdownbox"><A href="https://2015.igem.org/Team:Heidelberg/Modeling/aptakinetics"><P class="menutext dropdown-text">Aptamer Sensors</P></A></LI><LI class="dropdownbox"><A href="https://2015.igem.org/Team:Heidelberg/Modeling/rtsms"><P class="menutext dropdown-text">In vitro transcription</P></A></LI><LI><A href="https://2015.igem.org/Team:Heidelberg/Practices" class="menubox"><P class="menutext">Outreach</P></A></LI></DIV></DIV></DIV><NAV id="myNavmenu" class="navmenu navmenu-default navmenu-fixed-left offcanvas" role="navigation"><A class="navmenu-brand" href="https://2015.igem.org/Team:Heidelberg">iGEM Team Heidelberg 2015</A><UL class="nav navmenu-nav"><LI><A href="https://2015.igem.org/Team:Heidelberg/Achievements">Achievements</A></LI><LI><A href="https://2015.igem.org/Team:Heidelberg/Results">Results</A></LI><LI><A href="https://2015.igem.org/Team:Heidelberg/Overview">Project</A></LI><LI><A href="https://2015.igem.org/Team:Heidelberg/Software">Software</A></LI><LI><A href="https://2015.igem.org/Team:Heidelberg/notebook/mm">Notebook</A></LI><LI><A href="https://2015.igem.org/Team:Heidelberg/Parts">Parts</A></LI><LI><A href="https://2015.igem.org/Team:Heidelberg/Team">Team</A></LI><LI><A href="https://2015.igem.org/Team:Heidelberg/Modeling">Modeling</A></LI><LI><A href="https://2015.igem.org/Team:Heidelberg/Practices">Outreach</A></LI></UL></NAV><DIV class="container"><DIV class="content"><DIV class="row"><DIV class="col-lg-12"><DIV class="panel panel-default"><DIV class="panel-heading"><H3 class="basicheader">week number 42</H3></DIV><DIV class="panel-body"><DIV class="row"><DIV class="col-lg-12"><DIV class="notebookentry"><H3 class="notebookheading">▼2015-10-18  In vitro transcription of P1, P2 </H3><DIV class="notebooktext"><P> </P><P>For every reaction:</P><TABLE border="1" cellpadding="0" cellspacing="0"><TBODY><TR><TD style="height:21px; width:201px"><P>Stock solution</P></TD><TD style="height:21px; width:150px"><P>Final concentration</P></TD><TD style="height:21px; width:73px"><P>[µl]</P></TD></TR><TR><TD style="width:201px"><P>ATP 100mM</P></TD><TD style="width:150px"><P>4 mM</P></TD><TD style="width:73px"><P>8</P></TD></TR><TR><TD style="width:201px"><P>CTP 100mM</P></TD><TD style="width:150px"><P>4 mM</P></TD><TD style="width:73px"><P>8</P></TD></TR><TR><TD style="width:201px"><P>UTP 100mM</P></TD><TD style="width:150px"><P>4 mM</P></TD><TD style="width:73px"><P>8</P></TD></TR><TR><TD style="width:201px"><P>GTP 100mM</P></TD><TD style="width:150px"><P>4 mM</P></TD><TD style="width:73px"><P>8</P></TD></TR><TR><TD style="width:201px"><P>DTT 1M</P></TD><TD style="width:150px"><P>1 mM</P></TD><TD style="width:73px"><P>2</P></TD></TR><TR><TD style="width:201px"><P>DMSO</P></TD><TD style="width:150px"><P>5%</P></TD><TD style="width:73px"><P>10</P></TD></TR><TR><TD style="width:201px"><P>10x Transcription buffer</P></TD><TD style="width:150px"><P>1x</P></TD><TD style="width:73px"><P>20</P></TD></TR><TR><TD style="width:201px"><P>DNA</P></TD><TD style="width:150px"><P>1 µg</P></TD><TD style="width:73px"><P>20</P></TD></TR><TR><TD style="width:201px"><P>ddH20</P></TD><TD style="width:150px"><P> </P></TD><TD style="width:73px"><P>106</P></TD></TR><TR><TD style="width:201px"><P>T7 RNA polymerase</P></TD><TD style="width:150px"><P> </P></TD><TD style="width:73px"><P>3</P></TD></TR></TBODY></TABLE><P> </P><P> </P><UL><LI>Reaction was incubated for 3 h at 37 °C</LI><LI>After 1.5 h another 2 µL T7 RNA Polymerase were added</LI><LI>Addition of 2 µL DNase I and further incubation at 37 °C for 20 min</LI></UL><UL><LI>The constructs containing hammerhead- or HDV-ribozymes was heated up to 95° for 5 min, so that there is a coplete cleavage.</LI></UL><P> </P><P> </P><P>RNA purification by precipitation:</P><UL><LI>Samples was mixed with 200 µL of 2 x loading dye and purified over a 10 % PAGE</LI><LI>Bands were visualized by UV shadowing and suitable bands were excised</LI><LI>RNA was eluted out of the gel using 0.3 M NaAc pH 5.5 in three elution steps</LI><LI>Gel parts were filtered of and 2.5 volumes of -20 °C EtOH were added, sample was stored at -20 °C oN to let the RNA precipitate</LI><LI>Spin sample at 16,000 g for 30 min, discard the supernatant</LI><LI>Washed the pellet twice with 70 % EtOH and dissolved RNA in 20 µL of MQ water</LI></UL><P> </P></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV><DIV class="container"><DIV class="content"><DIV class="row"><DIV class="col-lg-12"><DIV class="panel panel-default"><DIV class="panel-heading"><H3 class="basicheader">week number 37</H3></DIV><DIV class="panel-body"><DIV class="row"><DIV class="col-lg-12"><DIV class="notebookentry"><H3 class="notebookheading">▼2015-09-07  Cloning of ribozyme 12-1 (Ribozyme without Insert) from pSB1C3 into p413-GPD</H3><DIV class="notebooktext"><P>Procedures:</P><P> </P><P>Digestion:</P><P> </P><P>Description:</P><P> </P><P>Materials and Chemicals:</P><P>4 µl of pSB1C3 with pcat-MCS and ribozyme 12, about 2 µg of DNA in total</P><P>2 µl Cutsmart Buffer</P><P>1 µl BamI-HF</P><P>1 µl SalI-HF</P><P>12 µl ddH2O</P><P> </P><P>The digest was incubated for 1 hour at 37°C while mixing slowly</P><P> </P><P>The lower band with cut out insert (ribozyme 12) was purificated by gel extraction</P><P> </P><P>Ligation:</P><P> </P><P>Description:</P><P> </P><P>Materials and Chemicals:</P><P>1 µl of p413-Vector DNA</P><P>6 µl of ribozyme 12 insert (purificated by gel extraction)</P><P>1 µl of T4-Ligase</P><P>2 µl of T4-Ligase Buffer</P><P>10 µl ddH2O</P><P> </P><P> </P><P>The ligation was incubated for 30 minutes at room temperature (about 25°C)</P><P> </P><P>KCM Transformation:</P><P> </P><P>Description:</P><P> </P><P>Steps:</P><P> </P><OL><LI>Take 50µl chemical competent E. coli from -80 freezer and thaw on ice</LI></OL><P> </P><OL><LI>Add (as master mix):</LI></OL><P>                2,5µl DNA</P><P>                10µl KCM 5x</P><P>                37,5µl H2O</P><P> </P><OL><LI>Incubate on ice for 30 minutes</LI></OL><P> </P><OL><LI>Heat shock at 42°C for 1 minute</LI></OL><P> </P><OL><LI>Incubate on ice for 2 minutes</LI></OL><P> </P><OL><LI>Add 900 µl of LB or 2x YT Medium</LI></OL><P> </P><OL><LI>Incubate on 37°C for 60min</LI></OL><P> </P><OL><LI>Centrifuge 5min at 1000g</LI></OL><P> </P><OL><LI>Take 900µl of supernatant and throw away</LI></OL><P> </P><OL><LI>Resuspend pellet in remaining media</LI></OL><P> </P><OL><LI>Plate out on agar with antibiotics (1:1 / 1:10)</LI></OL><P> </P><P>Used DNA: Ligation of p413 and ribozyme 12-1</P><P>10 µl instead of 2,5 µl were used</P><P> </P></DIV></DIV></DIV></DIV><DIV class="row"><DIV class="col-lg-12"><DIV class="notebookentry"><H3 class="notebookheading">▼2015-09-09  PCR of Type-1 Ribozymes for in vitro assay  </H3><DIV class="notebooktext"><TABLE border="1" cellpadding="0" cellspacing="0"><TBODY><TR><TD style="width:90px"><P> </P></TD><TD style="width:98px"><P>Ribozyme CFTR1 A</P></TD><TD style="width:43px"><P>[µl]</P></TD><TD style="width:119px"><P>Ribozyme CFTR1 C</P></TD><TD style="width:49px"><P>[µl]</P></TD><TD style="width:91px"><P>Ribozyme CFTR1 T</P></TD><TD style="width:50px"><P>[µl]</P></TD></TR><TR><TD style="width:90px"><P>Primer Fwd</P></TD><TD style="width:98px"><P>DH_82</P></TD><TD style="width:43px"><P>5</P></TD><TD style="width:119px"><P>DH_82</P></TD><TD style="width:49px"><P>5</P></TD><TD style="width:91px"><P>DH_82</P></TD><TD style="width:50px"><P>5</P></TD></TR><TR><TD style="width:90px"><P>Primer Rev</P></TD><TD style="width:98px"><P>DH_33</P></TD><TD style="width:43px"><P>5</P></TD><TD style="width:119px"><P>DH_83</P></TD><TD style="width:49px"><P>5</P></TD><TD style="width:91px"><P>DH_83</P></TD><TD style="width:50px"><P>5</P></TD></TR><TR><TD style="height:42px; width:90px"><P>Template</P></TD><TD style="height:42px; width:98px"><P>Ribozyme CFTR1 DE A</P></TD><TD style="height:42px; width:43px"><P>0,5</P></TD><TD style="height:42px; width:119px"><P>Ribozyme CFTR1 DE C</P></TD><TD style="height:42px; width:49px"><P>0,5</P></TD><TD style="height:42px; width:91px"><P>Ribozyme CFTR1 DE T</P></TD><TD style="height:42px; width:50px"><P>0,5</P></TD></TR><TR><TD style="width:90px"><P>ddH2O</P></TD><TD style="width:98px"><P> </P></TD><TD style="width:43px"><P>14,5</P></TD><TD style="width:119px"><P> </P></TD><TD style="width:49px"><P>14,5</P></TD><TD style="width:91px"><P> </P></TD><TD style="width:50px"><P>14,5</P></TD></TR><TR><TD style="width:90px"><P>Q5 Polymerase</P></TD><TD style="width:98px"><P> </P></TD><TD style="width:43px"><P>25</P></TD><TD style="width:119px"><P> </P></TD><TD style="width:49px"><P>25</P></TD><TD style="width:91px"><P> </P></TD><TD style="width:50px"><P>25</P></TD></TR></TBODY></TABLE><P> </P><P>Conditions:</P><TABLE border="1" cellpadding="0" cellspacing="0"><TBODY><TR><TD style="width:201px"><P>Step</P></TD><TD style="width:129px"><P>Temperature [°C]</P></TD><TD style="width:59px"><P>Time</P></TD><TD style="width:59px"><P>Cycles</P></TD></TR><TR><TD style="width:201px"><P>Initial denaturation</P></TD><TD style="width:129px"><P>98</P></TD><TD style="width:59px"><P>1:00</P></TD><TD style="width:59px"><P>1</P></TD></TR><TR><TD style="height:19px; width:201px"><P>Denturation</P></TD><TD style="height:19px; width:129px"><P>98</P></TD><TD style="height:19px; width:59px"><P>0:10</P></TD><TD rowspan="3" style="height:19px; width:59px"><P>35</P></TD></TR><TR><TD style="width:201px"><P>Annealing</P></TD><TD style="width:129px"><P>66</P></TD><TD style="width:59px"><P>0:15</P></TD></TR><TR><TD style="width:201px"><P>Extension</P></TD><TD style="width:129px"><P>72</P></TD><TD style="width:59px"><P>0:15</P></TD></TR><TR><TD style="width:201px"><P>Final Extension</P></TD><TD style="width:129px"><P>72</P></TD><TD style="width:59px"><P>1:00</P></TD><TD style="width:59px"><P>1</P></TD></TR><TR><TD style="width:201px"><P>Hold</P></TD><TD style="width:129px"><P>4</P></TD><TD style="width:59px"><P> </P></TD><TD style="width:59px"><P>Cycles</P></TD></TR></TBODY></TABLE><P> </P><P>After PCR DNA was purified by Qiagen PCR Purification Kit ans stored at -20°C.</P><P> </P></DIV></DIV></DIV></DIV><DIV class="row"><DIV class="col-lg-12"><DIV class="notebookentry"><H3 class="notebookheading">▼2015-09-10  In vitro transcription of CFTR1-type ribozymes</H3><DIV class="notebooktext"><H1> </H1><P>For every reaction:</P><TABLE border="1" cellpadding="0" cellspacing="0"><TBODY><TR><TD style="height:21px; width:201px"><P>Stock solution</P></TD><TD style="height:21px; width:150px"><P>Final concentration</P></TD><TD style="height:21px; width:73px"><P>[µl]</P></TD></TR><TR><TD style="width:201px"><P>ATP 100mM</P></TD><TD style="width:150px"><P>4 mM</P></TD><TD style="width:73px"><P>8</P></TD></TR><TR><TD style="width:201px"><P>CTP 100mM</P></TD><TD style="width:150px"><P>4 mM</P></TD><TD style="width:73px"><P>8</P></TD></TR><TR><TD style="width:201px"><P>UTP 100mM</P></TD><TD style="width:150px"><P>4 mM</P></TD><TD style="width:73px"><P>8</P></TD></TR><TR><TD style="width:201px"><P>GTP 100mM</P></TD><TD style="width:150px"><P>4 mM</P></TD><TD style="width:73px"><P>8</P></TD></TR><TR><TD style="width:201px"><P>DTT 1M</P></TD><TD style="width:150px"><P>1 mM</P></TD><TD style="width:73px"><P>2</P></TD></TR><TR><TD style="width:201px"><P>DMSO</P></TD><TD style="width:150px"><P>5%</P></TD><TD style="width:73px"><P>10</P></TD></TR><TR><TD style="width:201px"><P>10x Transcription buffer</P></TD><TD style="width:150px"><P>1x</P></TD><TD style="width:73px"><P>20</P></TD></TR><TR><TD style="width:201px"><P>DNA</P></TD><TD style="width:150px"><P>1 µg</P></TD><TD style="width:73px"><P>20</P></TD></TR><TR><TD style="width:201px"><P>ddH20</P></TD><TD style="width:150px"><P> </P></TD><TD style="width:73px"><P>106</P></TD></TR><TR><TD style="width:201px"><P>T7 RNA polymerase</P></TD><TD style="width:150px"><P> </P></TD><TD style="width:73px"><P>3</P></TD></TR></TBODY></TABLE><P> </P><P> </P><UL><LI>Reaction was incubated for 3 h at 37 °C</LI><LI>After 1.5 h another 2 µL T7 RNA Polymerase were added</LI><LI>Addition of 2 µL DNase I and further incubation at 37 °C for 20 min</LI></UL><UL><LI>Reaction was heated up to 95° for 5 min, so that there is a coplete cleavage.</LI></UL><P> </P><P>RNA purification by precipitation:</P><UL><LI>Samples was mixed with 200 µL of 2 x loading dye and purified over a 10 % PAGE</LI><LI>Bands were visualized by UV shadowing and suitable bands were excised</LI><LI>RNA was eluted out of the gel using 0.3 M NaAc pH 5.5 in three elution steps</LI><LI>Gel parts were filtered of and 2.5 volumes of -20 °C EtOH were added, sample was stored at -20 °C oN to let the RNA precipitate</LI><LI>Spin sample at 16,000 g for 30 min, discard the supernatant</LI><LI>Washed the pellet twice with 70 % EtOH and dissolved RNA in 20 µL of MQ water</LI></UL><P> </P><P> </P><P> </P><P> </P><P> </P></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV><DIV class="container"><DIV class="content"><DIV class="row"><DIV class="col-lg-12"><DIV class="panel panel-default"><DIV class="panel-heading"><H3 class="basicheader">week number 35</H3></DIV><DIV class="panel-body"><DIV class="row"><DIV class="col-lg-12"><DIV class="notebookentry"><H3 class="notebookheading">▼2015-08-25  Plasmid prep of colonies containing c(p413-GPD; sfGFP_delY)</H3><DIV class="notebooktext"><P><STRONG>Goal: Verifying the sequences of the picked cultures</STRONG></P><UL><LI>Plasmids of the cultures grown overnight were purified according to the manufacturers protocol.</LI><LI>Sequences were confirmed as correct by sequencing for colonies 3-2, 4-1,4-2 and 5-1.</LI><LI>Culture 4-1 was used for further experiments.</LI></UL></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV><DIV class="container"><DIV class="content"><DIV class="row"><DIV class="col-lg-12"><DIV class="panel panel-default"><DIV class="panel-heading"><H3 class="basicheader">week number 34</H3></DIV><DIV class="panel-body"><DIV class="row"><DIV class="col-lg-12"><DIV class="notebookentry"><H3 class="notebookheading">▼2015-08-17  PCR of Part 1 and 2 of the CFTR2 Ribozyme Insert for in vitro transcription  </H3><DIV class="notebooktext"><P> </P><TABLE border="1" cellpadding="0" cellspacing="0"><TBODY><TR><TD style="width:90px"><P> </P></TD><TD style="width:86px"><P>P1</P></TD><TD style="width:86px"><P>[µl]</P></TD><TD style="width:86px"><P>P2</P></TD><TD style="width:85px"><P>[µl]</P></TD></TR><TR><TD style="width:90px"><P>Primer Fwd</P></TD><TD style="width:86px"><P>DH_54</P></TD><TD style="width:86px"><P>5</P></TD><TD style="width:86px"><P>DH_56</P></TD><TD style="width:85px"><P>5</P></TD></TR><TR><TD style="width:90px"><P>Primer Rev</P></TD><TD style="width:86px"><P>DH_55</P></TD><TD style="width:86px"><P>5</P></TD><TD style="width:86px"><P>DH_57</P></TD><TD style="width:85px"><P>5</P></TD></TR><TR><TD style="width:90px"><P>Template</P></TD><TD style="width:86px"><P>Insert CFTR2</P></TD><TD style="width:86px"><P>0,5</P></TD><TD style="width:86px"><P>Insert CFTR2</P></TD><TD style="width:85px"><P>0,5</P></TD></TR><TR><TD style="width:90px"><P>ddH2O</P></TD><TD style="width:86px"><P> </P></TD><TD style="width:86px"><P>14,5</P></TD><TD style="width:86px"><P> </P></TD><TD style="width:85px"><P>14,5</P></TD></TR><TR><TD style="width:90px"><P>Q5 Polymerase</P></TD><TD style="width:86px"><P> </P></TD><TD style="width:86px"><P>25</P></TD><TD style="width:86px"><P> </P></TD><TD style="width:85px"><P>25</P></TD></TR></TBODY></TABLE><P> </P><P> </P><P> </P><P>Conditions:</P><TABLE border="1" cellpadding="0" cellspacing="0"><TBODY><TR><TD style="width:201px"><P>Step</P></TD><TD style="width:129px"><P>Temperature [°C]</P></TD><TD style="width:59px"><P>Time</P></TD><TD style="width:59px"><P>Cycles</P></TD></TR><TR><TD style="width:201px"><P>Initial denaturation</P></TD><TD style="width:129px"><P>98</P></TD><TD style="width:59px"><P>1:00</P></TD><TD style="width:59px"><P>1</P></TD></TR><TR><TD style="width:201px"><P>Denturation</P></TD><TD style="width:129px"><P>98</P></TD><TD style="width:59px"><P>0:10</P></TD><TD rowspan="3" style="width:59px"><P>35</P></TD></TR><TR><TD style="width:201px"><P>Annealing</P></TD><TD style="width:129px"><P>66</P></TD><TD style="width:59px"><P>0:15</P></TD></TR><TR><TD style="width:201px"><P>Extension</P></TD><TD style="width:129px"><P>72</P></TD><TD style="width:59px"><P>0:15</P></TD></TR><TR><TD style="width:201px"><P>Final Extension</P></TD><TD style="width:129px"><P>72</P></TD><TD style="width:59px"><P>1:00</P></TD><TD style="width:59px"><P>1</P></TD></TR><TR><TD style="width:201px"><P>Hold</P></TD><TD style="width:129px"><P>4</P></TD><TD style="width:59px"><P> </P></TD><TD style="width:59px"><P> </P></TD></TR></TBODY></TABLE><P> </P><P>After PCR DNA was purified by ethanol precipitation and stored at -20°C</P><P> </P><P> </P></DIV></DIV></DIV></DIV><DIV class="row"><DIV class="col-lg-12"><DIV class="notebookentry"><H3 class="notebookheading">▼2015-08-19  Labeling of P1 with G-Azide</H3><DIV class="notebooktext"><P>Reaction was performed with 2-fold concentration of labeling nucleotide and a longer incubation time in comparison to first labeling experiment.</P><P> </P><TABLE border="1" cellpadding="0" cellspacing="0"><TBODY><TR><TD style="width:151px"><P> </P></TD><TD style="width:151px"><P>Reaction [µl]</P></TD><TD style="width:151px"><P>Control without PAP[µl]</P></TD><TD style="width:151px"><P>Control without template [µl]</P></TD></TR><TR><TD style="width:151px"><P>PAP (Poly-A-Polymerase, yeast)</P></TD><TD style="width:151px"><P>1</P></TD><TD style="width:151px"><P>/</P></TD><TD style="width:151px"><P>1</P></TD></TR><TR><TD style="width:151px"><P>RNAse inhibitor</P></TD><TD style="width:151px"><P>1</P></TD><TD style="width:151px"><P>1</P></TD><TD style="width:151px"><P>1</P></TD></TR><TR><TD style="width:151px"><P>PAP buffer 5x</P></TD><TD style="width:151px"><P>4</P></TD><TD style="width:151px"><P>4</P></TD><TD style="width:151px"><P>4</P></TD></TR><TR><TD style="height:41px; width:151px"><P>Nucleotide G-Azide, 100µM</P></TD><TD style="height:41px; width:151px"><P>4</P></TD><TD style="height:41px; width:151px"><P>4</P></TD><TD style="height:41px; width:151px"><P>4</P></TD></TR><TR><TD style="height:22px; width:151px"><P>P1-RNA</P></TD><TD style="height:22px; width:151px"><P>5 (= 50µM)</P></TD><TD style="height:22px; width:151px"><P>5 (= 50µM)</P></TD><TD style="height:22px; width:151px"><P>/</P></TD></TR><TR><TD style="width:151px"><P>ddH2O</P></TD><TD style="width:151px"><P>5</P></TD><TD style="width:151px"><P>6</P></TD><TD style="width:151px"><P>10</P></TD></TR></TBODY></TABLE><P> </P><UL><LI>Reaction was incubated for 2 h at 37 °C</LI></UL><UL><LI>Heat inactivation at 65°C for 10min</LI><LI>Purification by ethanol precipitation</LI></UL><P> </P></DIV></DIV></DIV></DIV><DIV class="row"><DIV class="col-lg-12"><DIV class="notebookentry"><H3 class="notebookheading">▼2015-08-20  Splinted Ligation of modified P1 and P2  </H3><DIV class="notebooktext"><P> </P><TABLE border="1" cellpadding="0" cellspacing="0"><TBODY><TR><TD style="width:151px"><P> </P></TD><TD style="width:151px"><P>Reaction [µl]</P></TD><TD style="width:151px"><P>Control without Ligase [µl]</P></TD><TD style="width:151px"><P>Control without Splint [µl]</P></TD></TR><TR><TD style="height:21px; width:151px"><P>P1 RNA, labeled</P></TD><TD style="height:21px; width:151px"><P>6 (=1µM)</P></TD><TD style="height:21px; width:151px"><P>6 (=1µM)</P></TD><TD style="height:21px; width:151px"><P>6 (=1µM)</P></TD></TR><TR><TD style="width:151px"><P>P2 RNA</P></TD><TD style="width:151px"><P>0,5 (=0,5µM)</P></TD><TD style="width:151px"><P>0,5 (=0,5µM)</P></TD><TD style="width:151px"><P>0,5 (=0,5µM)</P></TD></TR><TR><TD style="height:21px; width:151px"><P>Splint</P></TD><TD style="height:21px; width:151px"><P>0,7 (=2,25µM)</P></TD><TD style="height:21px; width:151px"><P>0,7 (=2,25µM)</P></TD><TD style="height:21px; width:151px"><P>/</P></TD></TR><TR><TD style="width:151px"><P>10x Ligation buffer</P></TD><TD style="width:151px"><P>3</P></TD><TD style="width:151px"><P>3</P></TD><TD style="width:151px"><P>3</P></TD></TR><TR><TD style="width:151px"><P>T4 DNA ligase</P></TD><TD style="width:151px"><P>1</P></TD><TD style="width:151px"><P>/</P></TD><TD style="width:151px"><P>1</P></TD></TR><TR><TD style="width:151px"><P>ddH2O</P></TD><TD style="width:151px"><P>13,8</P></TD><TD style="width:151px"><P>15,8</P></TD><TD style="width:151px"><P>15,5</P></TD></TR></TBODY></TABLE><P> </P><P> </P><UL><LI>Reaction was heaten up to 95°C for 30s without T4 DNA ligase and then cooled down to room temperature</LI><LI>After 15min  T4 DNA ligase was added.</LI><LI>Reaction was incubated for 1 h at 37 °C</LI><LI>Heat inactivation at 80°C for 10min</LI></UL></DIV></DIV></DIV></DIV><DIV class="row"><DIV class="col-lg-12"><DIV class="notebookentry"><H3 class="notebookheading">▼2015-08-20  Copper Click – Test of Labeled Part 1 and Splinted ligation</H3><DIV class="notebooktext"><H1> </H1><P>To check if the azide-modified NTPs are incorporated the reactive azide is clicked to an alkyne activated FAM-alkyne fluorophore under copper catalyzed 1,3-dipolar azide-alkyne cycloaddition (CuAAC). Samples are after the reaction separated on a 20 % denaturing PAGE and visualized first in the fluorescent channel for FAM and afterwards stained with SYBR</P><P>Gold to visualize the RNA. Controls were performed as well.</P><P> </P><P> </P><P>Reaction conditions:</P><TABLE border="1" cellpadding="0" cellspacing="0"><TBODY><TR><TD style="width:245px"> </TD><TD style="width:113px"><P><STRONG>cStock</STRONG></P></TD><TD style="width:104px"><P><STRONG>cFinal</STRONG></P></TD><TD style="width:47px"><P><STRONG>V[µL]</STRONG></P></TD></TR><TR><TD style="width:245px"><P><STRONG>Phosphate buffer pH 7.0</STRONG></P></TD><TD style="width:113px"><P>100 mM</P></TD><TD style="width:104px"><P>50 mM</P></TD><TD style="width:47px"><P>12,5</P></TD></TR><TR><TD style="width:245px"><P><STRONG>FAM alkyne</STRONG></P></TD><TD style="width:113px"><P>10 µM</P></TD><TD style="width:104px"><P>400 nM</P></TD><TD style="width:47px"><P>1</P></TD></TR><TR><TD style="width:245px"><P><STRONG>Azide modified RNA</STRONG></P></TD><TD style="width:113px"><P>1 µM</P></TD><TD style="width:104px"><P>200 nM</P></TD><TD style="width:47px"><P>5</P></TD></TR><TR><TD style="width:245px"><P><STRONG>Cu(II)SO4</STRONG></P></TD><TD style="width:113px"><P>2 mM</P></TD><TD style="width:104px"><P>100 µM</P></TD><TD style="width:47px"><P>1.25</P></TD></TR><TR><TD style="width:245px"><P><STRONG>THPTA</STRONG></P></TD><TD style="width:113px"><P>5 mM</P></TD><TD style="width:104px"><P>500 µM</P></TD><TD style="width:47px"><P>2.5</P></TD></TR><TR><TD style="width:245px"><P><STRONG>Sodium ascorbate</STRONG></P></TD><TD style="width:113px"><P>10 mM</P></TD><TD style="width:104px"><P>1 mM</P></TD><TD style="width:47px"><P>2.5</P></TD></TR><TR><TD style="width:245px"><P><STRONG>H2O</STRONG></P></TD><TD style="width:113px"> </TD><TD style="width:104px"><P>Ad 25</P></TD><TD style="width:47px"><P>0.25</P></TD></TR><TR><TD style="width:245px"><P><STRONG>Final</STRONG></P></TD><TD style="width:113px"> </TD><TD style="width:104px"><P> </P></TD><TD style="width:47px"><P><STRONG><EM>25</EM></STRONG></P></TD></TR></TBODY></TABLE><P>Conditions as Winz, 2012.</P><P> </P><UL><LI>All compounds were mixed in the given order</LI><LI>Cu(II)SO4 and THPTA and H2O were mixed before adding to the mixture to let THPTA chelat the Cu</LI><LI>Lastly sodium ascorbate was added to reduce the Cu(II) to Cu(I)</LI><LI>Reaction was incubated for 1 h at 37 °C and afterwards put at -20 °C</LI></UL><P>Checking success on a 20 % denaturing PAGE</P><UL><LI>Samples were mixed with an appropriate amount of 2 x loading dye and separated on a 20 % denaturing PAGE</LI><LI>Gel was scanned with a Biorad ChemDoc, GE in fluorescent mode using the pre-set FAM parameters</LI><LI>Afterwards gel was stained using SYBR gold and scanned with the Typoon in SYBR Gold mode</LI></UL><P><STRONG>Result and Outlook:</STRONG></P><P>Labeling of RNA with azide and splinted ligation was also at the second tryonly partially successful. The repair-oligonucleotide is ordered as RNA-synthesis.</P><P> </P></DIV></DIV></DIV></DIV><DIV class="row"><DIV class="col-lg-12"><DIV class="notebookentry"><H3 class="notebookheading">▼2015-08-22  Mutation PCR of pMaM17</H3><DIV class="notebooktext"><P><STRONG>Goal: Deleting a Y in sfGFP from pMaM17 rendering it non-functional. </STRONG></P><UL><LI>PCR with primers xxhb003xx and xxhb004xx was performed using a standard protocol with 68°C as annealing temperature. </LI><LI>PCR was performed using Phusion Flash MasterMix.</LI><LI>300 µl were split amongst 4 tubes. </LI><LI>PCR was followed by a DpnI digest.</LI><LI>Samples were purified by gel extraction according to manufacturs protocol. </LI><LI>NanoDrop showed a concentration of 743 ng/µl after purification, with 29 µl sample left to use.</LI></UL></DIV></DIV></DIV></DIV><DIV class="row"><DIV class="col-lg-12"><DIV class="notebookentry"><H3 class="notebookheading">▼2015-08-22  Digestion of pMaM(sfGFP_delY)</H3><DIV class="notebooktext"><P><STRONG>Goal: Digest pMaM17(sfGFP_delY) with BamI and SalI</STRONG></P><UL><LI>5 µl of previous PCR product (pMaM17(sfGFP_delY) were added to 11 µl ddH20, 1 µl BamI, 1 µl SalI and 2 µl CutSmart buffer.</LI><LI>Sample rested for 1 hour at room temperature.</LI><LI>Heat inactivation was performed at 95 °C for 1 min.</LI><LI>Purification was performed according to manufacturers protocol.</LI></UL></DIV></DIV></DIV></DIV><DIV class="row"><DIV class="col-lg-12"><DIV class="notebookentry"><H3 class="notebookheading">▼2015-08-22  Digestion of p413-GPD</H3><DIV class="notebooktext"><P><STRONG>Goal: Digest p413-GPD with BamI and SalI</STRONG></P><UL><LI>3 µl of p413-GPD plasmid were added to 13 µl ddH20, 1 µl BamI, 1 µl SalI and 2 µl CutSmart buffer.</LI><LI>Sample rested for 1 hour at room temperature.</LI><LI>Heat inactivation was performed at 95 °C for 1 min.</LI><LI>Linearized plasmid was purified by performing gel extraction according to manufacturers protocol.</LI></UL></DIV></DIV></DIV></DIV><DIV class="row"><DIV class="col-lg-12"><DIV class="notebookentry"><H3 class="notebookheading">▼2015-08-22  Ligation of p413-GPD_digested and sfGFP_delY_digested</H3><DIV class="notebooktext"><P><STRONG>Goal: Ligation of p413-GPD and sfGFP_delY, both previously digested with BamI and SalI</STRONG></P><UL><LI>100 ng of digested p413-GPD plasmid and 500 ng of digested sfGFP_delY were ligated using T4 DNA ligase in a 20 µl assay.</LI><LI>Sample rested for 1 hour at room temperature.</LI><LI>Purification was performed according to manufacturers protocol.</LI><LI>NanoDrop determined a concentration of 18.9 ng/µl with 19 µl sample remaining.</LI></UL></DIV></DIV></DIV></DIV><DIV class="row"><DIV class="col-lg-12"><DIV class="notebookentry"><H3 class="notebookheading">▼2015-08-22  Transformation of E. coli with sfGFP_delY in p413-GPD</H3><DIV class="notebooktext"><P><STRONG>Goal: Transforming E. coli with sfGFP_delY inserted into p413-GPD after a digest with BamI and SalI</STRONG></P><UL><LI><P>4 x 50 µl of chemically competent E. coli were thawed on ice</P></LI><LI><P>4 samples were prepared, each containing 10 µl KCM.</P></LI><LI><P>Ligated plasmid was added as 5 µl, 4 µl, 3 µl and 2 µl, one volume per sample.</P></LI><LI><P>Samples were filled up to 50 µl with ddH20.</P></LI><LI><P>50 µl of hawed cells were added to each sample.</P></LI><LI><P> </P><P>Samples were held at 42 °C for 2 min.</P></LI></UL><P> </P><UL><LI>Samples were incubated on ice for 2 min</LI><LI>900 ul of media was added to each sample.</LI><LI>Samples were incubated at 37 °C for 60 min.</LI><LI>Cells were sedimented by centrifuging at 1000 g for 5 min.</LI><LI>Supernatant was removed.</LI><LI>Cells were resuspended in media.</LI><LI>Cells were spread on one plate each.</LI><LI>Cells incubated overnight at 37°C. </LI></UL></DIV></DIV></DIV></DIV><DIV class="row"><DIV class="col-lg-12"><DIV class="notebookentry"><H3 class="notebookheading">▼2015-08-23  Colony picking of E. coli transfected with c(p413-GPD; sfGFP_delY)</H3><DIV class="notebooktext"><P><STRONG>Goal: Confirming a culture containing the correct plasmid with sfGFP_delY ligated into p413-GPD.</STRONG></P><UL><LI>Colonies were picked from plates</LI><LI><TABLE border="1" cellpadding="1" cellspacing="1" style="width:500px"><TBODY><TR><TD>Concetration of plasmid in spread sample</TD><TD>Picked colonies</TD></TR><TR><TD>2 µl</TD><TD>1</TD></TR><TR><TD>3 µl</TD><TD>2</TD></TR><TR><TD>4 µl</TD><TD>3</TD></TR><TR><TD>5 µl</TD><TD>2</TD></TR></TBODY></TABLE></LI><LI><P>Picked colonies were grown overnight as a liquid culture.</P></LI></UL></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV><DIV class="container"><DIV class="content"><DIV class="row"><DIV class="col-lg-12"><DIV class="panel panel-default"><DIV class="panel-heading"><H3 class="basicheader">week number 33</H3></DIV><DIV class="panel-body"><DIV class="row"><DIV class="col-lg-12"><DIV class="notebookentry"><H3 class="notebookheading">▼2015-08-10  PCR of Part 1 and 2 of the CFTR2 Ribozyme Insert for in vitro transcription </H3><DIV class="notebooktext"><P>Durchführung</P><P> </P><TABLE border="1" cellpadding="0" cellspacing="0"><TBODY><TR><TD style="width:90px"><P> </P></TD><TD style="width:86px"><P>P1</P></TD><TD style="width:86px"><P>[µl]</P></TD><TD style="width:86px"><P>P2</P></TD><TD style="width:85px"><P>[µl]</P></TD></TR><TR><TD style="width:90px"><P>Primer Fwd</P></TD><TD style="width:86px"><P>DH_54</P></TD><TD style="width:86px"><P>5</P></TD><TD style="width:86px"><P>DH_56</P></TD><TD style="width:85px"><P>5</P></TD></TR><TR><TD style="width:90px"><P>Primer Rev</P></TD><TD style="width:86px"><P>DH_55</P></TD><TD style="width:86px"><P>5</P></TD><TD style="width:86px"><P>DH_57</P></TD><TD style="width:85px"><P>5</P></TD></TR><TR><TD style="width:90px"><P>Template</P></TD><TD style="width:86px"><P>Insert CFTR2</P></TD><TD style="width:86px"><P>0,5</P></TD><TD style="width:86px"><P>Insert CFTR2</P></TD><TD style="width:85px"><P>0,5</P></TD></TR><TR><TD style="width:90px"><P>ddH2O</P></TD><TD style="width:86px"><P> </P></TD><TD style="width:86px"><P>14,5</P></TD><TD style="width:86px"><P> </P></TD><TD style="width:85px"><P>14,5</P></TD></TR><TR><TD style="width:90px"><P>Q5 Polymerase</P></TD><TD style="width:86px"><P> </P></TD><TD style="width:86px"><P>25</P></TD><TD style="width:86px"><P> </P></TD><TD style="width:85px"><P>25</P></TD></TR></TBODY></TABLE><P> </P><P> </P><P> </P><P>Conditions:</P><TABLE border="1" cellpadding="0" cellspacing="0"><TBODY><TR><TD style="width:201px"><P>Step</P></TD><TD style="width:129px"><P>Temperature [°C]</P></TD><TD style="width:59px"><P>Time</P></TD><TD style="width:59px"><P>Cycles</P></TD></TR><TR><TD style="width:201px"><P>Initial denaturation</P></TD><TD style="width:129px"><P>98</P></TD><TD style="width:59px"><P>1:00</P></TD><TD style="width:59px"><P>1</P></TD></TR><TR><TD style="width:201px"><P>Denturation</P></TD><TD style="width:129px"><P>98</P></TD><TD style="width:59px"><P>0:10</P></TD><TD rowspan="3" style="width:59px"><P>35</P></TD></TR><TR><TD style="width:201px"><P>Annealing</P></TD><TD style="width:129px"><P>66</P></TD><TD style="width:59px"><P>0:15</P></TD></TR><TR><TD style="width:201px"><P>Extension</P></TD><TD style="width:129px"><P>72</P></TD><TD style="width:59px"><P>0:15</P></TD></TR><TR><TD style="width:201px"><P>Final Extension</P></TD><TD style="width:129px"><P>72</P></TD><TD style="width:59px"><P>1:00</P></TD><TD style="width:59px"><P>1</P></TD></TR><TR><TD style="width:201px"><P>Hold</P></TD><TD style="width:129px"><P>4</P></TD><TD style="width:59px"><P> </P></TD><TD style="width:59px"><P> </P></TD></TR></TBODY></TABLE><P> </P><P>After PCR DNA was purified by ethanol precipitation and stored at -20°C</P><P> </P><P> </P></DIV></DIV></DIV></DIV><DIV class="row"><DIV class="col-lg-12"><DIV class="notebookentry"><H3 class="notebookheading">▼2015-08-10  PCR of Ribozyme target for in vitro assay  </H3><DIV class="notebooktext"><TABLE border="1" cellpadding="0" cellspacing="0"><TBODY><TR><TD style="width:90px"><P> </P></TD><TD style="width:86px"><P>Target</P></TD><TD style="width:86px"><P>[µl]</P></TD></TR><TR><TD style="width:90px"><P>Primer Fwd</P></TD><TD style="width:86px"><P>DH_58</P></TD><TD style="width:86px"><P>5</P></TD></TR><TR><TD style="width:90px"><P>Primer Rev</P></TD><TD style="width:86px"><P>DH_59</P></TD><TD style="width:86px"><P>5</P></TD></TR><TR><TD style="width:90px"><P>Template</P></TD><TD style="width:86px"><P>CFTRtestconstruct</P></TD><TD style="width:86px"><P>0,5</P></TD></TR><TR><TD style="width:90px"><P>ddH2O</P></TD><TD style="width:86px"><P> </P></TD><TD style="width:86px"><P>14,5</P></TD></TR><TR><TD style="width:90px"><P>Q5 Polymerase</P></TD><TD style="width:86px"><P> </P></TD><TD style="width:86px"><P>25</P></TD></TR></TBODY></TABLE><P> </P><P> </P><P> </P><P> </P><P>Conditions:</P><TABLE border="1" cellpadding="0" cellspacing="0"><TBODY><TR><TD style="width:201px"><P>Step</P></TD><TD style="width:129px"><P>Temperature [°C]</P></TD><TD style="width:59px"><P>Time</P></TD><TD style="width:59px"><P>Cycles</P></TD></TR><TR><TD style="width:201px"><P>Initial denaturation</P></TD><TD style="width:129px"><P>98</P></TD><TD style="width:59px"><P>2:00</P></TD><TD style="width:59px"><P>1</P></TD></TR><TR><TD style="width:201px"><P>Denturation</P></TD><TD style="width:129px"><P>98</P></TD><TD style="width:59px"><P>0:20</P></TD><TD rowspan="3" style="width:59px"><P>35</P></TD></TR><TR><TD style="width:201px"><P>Annealing</P></TD><TD style="width:129px"><P>66</P></TD><TD style="width:59px"><P>0:20</P></TD></TR><TR><TD style="width:201px"><P>Extension</P></TD><TD style="width:129px"><P>72</P></TD><TD style="width:59px"><P>1:00</P></TD></TR><TR><TD style="width:201px"><P>Final Extension</P></TD><TD style="width:129px"><P>72</P></TD><TD style="width:59px"><P>3:00</P></TD><TD style="width:59px"><P>1</P></TD></TR><TR><TD style="width:201px"><P>Hold</P></TD><TD style="width:129px"><P>4</P></TD><TD style="width:59px"><P> </P></TD><TD style="width:59px"><P>Cycles</P></TD></TR></TBODY></TABLE><P> </P><P>After PCR DNA was purified by Qiagen PCR Purification Kit ans stored at -20°C.</P><P> </P><P> </P></DIV></DIV></DIV></DIV><DIV class="row"><DIV class="col-lg-12"><DIV class="notebookentry"><H3 class="notebookheading">▼2015-08-11  In vitro transcription of P1, P2, Ribozyme 2 A/C/T and Ribozyme Target  </H3><DIV class="notebooktext"><P> </P><P>For every reaction:</P><TABLE border="1" cellpadding="0" cellspacing="0"><TBODY><TR><TD style="height:21px; width:201px"><P>Stock solution</P></TD><TD style="height:21px; width:150px"><P>Final concentration</P></TD><TD style="height:21px; width:73px"><P>[µl]</P></TD></TR><TR><TD style="width:201px"><P>ATP 100mM</P></TD><TD style="width:150px"><P>4 mM</P></TD><TD style="width:73px"><P>8</P></TD></TR><TR><TD style="width:201px"><P>CTP 100mM</P></TD><TD style="width:150px"><P>4 mM</P></TD><TD style="width:73px"><P>8</P></TD></TR><TR><TD style="width:201px"><P>UTP 100mM</P></TD><TD style="width:150px"><P>4 mM</P></TD><TD style="width:73px"><P>8</P></TD></TR><TR><TD style="width:201px"><P>GTP 100mM</P></TD><TD style="width:150px"><P>4 mM</P></TD><TD style="width:73px"><P>8</P></TD></TR><TR><TD style="width:201px"><P>DTT 1M</P></TD><TD style="width:150px"><P>1 mM</P></TD><TD style="width:73px"><P>2</P></TD></TR><TR><TD style="width:201px"><P>DMSO</P></TD><TD style="width:150px"><P>5%</P></TD><TD style="width:73px"><P>10</P></TD></TR><TR><TD style="width:201px"><P>10x Transcription buffer</P></TD><TD style="width:150px"><P>1x</P></TD><TD style="width:73px"><P>20</P></TD></TR><TR><TD style="width:201px"><P>DNA</P></TD><TD style="width:150px"><P>1 µg</P></TD><TD style="width:73px"><P>20</P></TD></TR><TR><TD style="width:201px"><P>ddH20</P></TD><TD style="width:150px"><P> </P></TD><TD style="width:73px"><P>106</P></TD></TR><TR><TD style="width:201px"><P>T7 RNA polymerase</P></TD><TD style="width:150px"><P> </P></TD><TD style="width:73px"><P>3</P></TD></TR></TBODY></TABLE><P> </P><P> </P><UL><LI>Reaction was incubated for 3 h at 37 °C</LI><LI>After 1.5 h another 2 µL T7 RNA Polymerase were added</LI><LI>Addition of 2 µL DNase I and further incubation at 37 °C for 20 min</LI></UL><UL><LI>The constructs containing hammerhead- or HDV-ribozymes was heaten up to 95° for 5 min, so that there is a coplete cleavage.</LI></UL><P> </P><P> </P><P>RNA purification by precipitation:</P><UL><LI>Samples was mixed with 200 µL of 2 x loading dye and purified over a 10 % PAGE</LI><LI>Bands were visualized by UV shadowing and suitable bands were excised</LI><LI>RNA was eluted out of the gel using 0.3 M NaAc pH 5.5 in three elution steps</LI><LI>Gel parts were filtered of and 2.5 volumes of -20 °C EtOH were added, sample was stored at -20 °C oN to let the RNA precipitate</LI><LI>Spin sample at 16,000 g for 30 min, discard the supernatant</LI><LI>Washed the pellet twice with 70 % EtOH and dissolved RNA in 20 µL of MQ water</LI></UL><P> </P><P> </P><P> </P></DIV></DIV></DIV></DIV><DIV class="row"><DIV class="col-lg-12"><DIV class="notebookentry"><H3 class="notebookheading">▼2015-08-11  Labeling of P1 with G-Azide</H3><DIV class="notebooktext"><TABLE border="1" cellpadding="0" cellspacing="0"><TBODY><TR><TD style="width:151px"><P> </P></TD><TD style="width:151px"><P>Reaction [µl]</P></TD><TD style="width:151px"><P>Control without PAP[µl]</P></TD><TD style="width:151px"><P>Control without template [µl]</P></TD></TR><TR><TD style="width:151px"><P>PAP (Poly-A-Polymerase, yeast)</P></TD><TD style="width:151px"><P>1</P></TD><TD style="width:151px"><P>/</P></TD><TD style="width:151px"><P>1</P></TD></TR><TR><TD style="width:151px"><P>RNAse inhibitor</P></TD><TD style="width:151px"><P>1</P></TD><TD style="width:151px"><P>1</P></TD><TD style="width:151px"><P>1</P></TD></TR><TR><TD style="width:151px"><P>PAP buffer 5x</P></TD><TD style="width:151px"><P>4</P></TD><TD style="width:151px"><P>4</P></TD><TD style="width:151px"><P>4</P></TD></TR><TR><TD style="width:151px"><P>Nucleotide G-Azide, 100µM</P></TD><TD style="width:151px"><P>2</P></TD><TD style="width:151px"><P>2</P></TD><TD style="width:151px"><P>2</P></TD></TR><TR><TD style="width:151px"><P>P1-RNA</P></TD><TD style="width:151px"><P>3 (= 5µM)</P></TD><TD style="width:151px"><P>3</P></TD><TD style="width:151px"><P>/</P></TD></TR><TR><TD style="width:151px"><P>ddH2O</P></TD><TD style="width:151px"><P>9</P></TD><TD style="width:151px"><P>10</P></TD><TD style="width:151px"><P>12</P></TD></TR></TBODY></TABLE><P> </P><P> </P><UL><LI>Reaction was incubated for 2 h at 37 °C</LI></UL><UL><LI>Heat inactivation at 65°C for 10min</LI><LI>Purification by ethanol precipitation</LI></UL></DIV></DIV></DIV></DIV><DIV class="row"><DIV class="col-lg-12"><DIV class="notebookentry"><H3 class="notebookheading">▼2015-08-12  Splinted Ligation of modified P1 and P2 </H3><DIV class="notebooktext"><TABLE border="1" cellpadding="0" cellspacing="0"><TBODY><TR><TD style="width:151px"><P> </P></TD><TD style="width:151px"><P>Reaction [µl]</P></TD><TD style="width:151px"><P>Control without Ligase [µl]</P></TD><TD style="width:151px"><P>Control without Splint [µl]</P></TD></TR><TR><TD style="height:21px; width:151px"><P>P1 RNA, labeled</P></TD><TD style="height:21px; width:151px"><P>12 (=0,5µM)</P></TD><TD style="height:21px; width:151px"><P>12 (=0,5µM)</P></TD><TD style="height:21px; width:151px"><P>12 (=0,5µM)</P></TD></TR><TR><TD style="width:151px"><P>P2 RNA</P></TD><TD style="width:151px"><P>0,5 (=0,5µM)</P></TD><TD style="width:151px"><P>0,5 (=0,5µM)</P></TD><TD style="width:151px"><P>0,5 (=0,5µM)</P></TD></TR><TR><TD style="height:21px; width:151px"><P>Splint</P></TD><TD style="height:21px; width:151px"><P>0,7 (=2,25µM)</P></TD><TD style="height:21px; width:151px"><P>0,7 (=2,25µM)</P></TD><TD style="height:21px; width:151px"><P>/</P></TD></TR><TR><TD style="width:151px"><P>10x Ligation buffer</P></TD><TD style="width:151px"><P>3</P></TD><TD style="width:151px"><P>3</P></TD><TD style="width:151px"><P>3</P></TD></TR><TR><TD style="width:151px"><P>T4 DNA ligase</P></TD><TD style="width:151px"><P>1</P></TD><TD style="width:151px"><P>/</P></TD><TD style="width:151px"><P>1</P></TD></TR><TR><TD style="width:151px"><P>ddH2O</P></TD><TD style="width:151px"><P>11,8</P></TD><TD style="width:151px"><P>12,8</P></TD><TD style="width:151px"><P>12,5</P></TD></TR></TBODY></TABLE><P> </P><P> </P><UL><LI>Reaction was heaten up to 95°C for 30s without T4 DNA ligase and then cooled down to room temperature</LI><LI>After 15min  T4 DNA ligase was added.</LI><LI>Reaction was incubated for 1 h at 37 °C</LI><LI>Heat inactivation at 80°C for 10min</LI></UL></DIV></DIV></DIV></DIV><DIV class="row"><DIV class="col-lg-12"><DIV class="notebookentry"><H3 class="notebookheading">▼2015-08-13  Transfer of the ribozymes and inserts into yeast and mammalian vectors</H3><DIV class="notebooktext"><P>Procedures:</P><P> </P><P>Digestion of pSB1C3 + MCS +pcat with and without ribozymes:</P><P> </P><P>Description</P><P> </P><P>Reaction mix:</P><P>1 µg of DNA</P><P>0,1 µl of BamHI-HF</P><P>0,1 µl of SalI-HF</P><P>2 µl Cutsmart</P><P>ad 20 µl ddH2O</P><P> </P><P>Digestion for 1 hour at 37°C</P><P> </P><P>After the reaction the samples were given on a 0,8% Agarose gel and were then purified by gel extraction.</P><P> </P><P>Ligation of the ribozymal fragments into p413</P><P> </P><P>Description:</P><P> </P><P>Chemicals:</P><P>1 µl of Vector DNA (about 25 ng) dephosporylated</P><P>5 µl of Insert DNA (about 10 ng)</P><P>2 µl T4-Ligation Buffer</P><P>1 µl T4-Ligase</P><P>10 µl ddH2O</P><P> </P><P>The Mix was incubated for 30 minutes at room temperature</P><P> </P><P>Transformation of p413 + Ribozymes + Inserts:</P><P> </P><P>Description:</P><P> </P><P>Steps:</P><P> </P><OL><LI>Take 50µl chemical competent E. coli from -80 freezer and thaw on ice</LI></OL><P> </P><OL><LI>Add (as master mix):</LI></OL><P>                2,5µl DNA</P><P>                10µl KCM 5x</P><P>                37,5µl H2O</P><P> </P><OL><LI>Incubate on ice for 30 minutes</LI></OL><P> </P><OL><LI>Heat shock at 42°C for 1 minute</LI></OL><P> </P><OL><LI>Incubate on ice for 2 minutes</LI></OL><P> </P><OL><LI>Add 900 µl of LB or 2x YT Medium</LI></OL><P> </P><OL><LI>Incubate on 37°C for 60min</LI></OL><P> </P><OL><LI>Centrifuge 5min at 1000g</LI></OL><P> </P><OL><LI>Take 900µl of supernatant and throw away</LI></OL><P> </P><OL><LI>Resuspend pellet in remaining media</LI></OL><P> </P><OL><LI>Plate out on agar with antibiotics (1:1 / 1:10)</LI></OL><P> </P><P>The prior ligations were used for the transformation.</P><P> </P><P>Colony PCR of ribozymes + Inserts:</P><P> </P><P> </P><P> </P><TABLE border="0" cellpadding="0" cellspacing="0" style="width:792px"><TBODY><TR><TD style="height:18px; width:60px"><P>96-well</P></TD><TD style="height:18px; width:60px"> </TD><TD style="height:18px; width:60px"> </TD><TD style="height:18px; width:60px"> </TD><TD style="height:18px; width:60px"> </TD><TD style="height:18px; width:60px"> </TD><TD style="height:18px; width:60px"> </TD><TD style="height:18px; width:60px"> </TD><TD style="height:18px; width:60px"> </TD><TD style="height:18px; width:60px"> </TD><TD style="height:18px; width:70px"> </TD><TD style="height:18px; width:60px"> </TD><TD style="height:18px; width:60px"> </TD></TR><TR><TD style="height:18px; width:60px"> </TD><TD style="height:18px; width:60px"> </TD><TD style="height:18px; width:60px"> </TD><TD style="height:18px; width:60px"> </TD><TD style="height:18px; width:60px"> </TD><TD style="height:18px; width:60px"> </TD><TD style="height:18px; width:60px"> </TD><TD style="height:18px; width:60px"> </TD><TD style="height:18px; width:60px"> </TD><TD style="height:18px; width:60px"> </TD><TD style="height:18px; width:70px"> </TD><TD style="height:18px; width:60px"> </TD><TD style="height:18px; width:60px"> </TD></TR><TR><TD style="height:18px; width:60px"> </TD><TD style="height:18px; width:60px"><P>1</P></TD><TD style="height:18px; width:60px"><P>2</P></TD><TD style="height:18px; width:60px"><P>3</P></TD><TD style="height:18px; width:60px"><P>4</P></TD><TD style="height:18px; width:60px"><P>5</P></TD><TD style="height:18px; width:60px"><P>6</P></TD><TD style="height:18px; width:60px"><P>7</P></TD><TD style="height:18px; width:60px"><P>8</P></TD><TD style="height:18px; width:60px"><P>9</P></TD><TD style="height:18px; width:70px"><P>10</P></TD><TD style="height:18px; width:60px"><P>11</P></TD><TD style="height:18px; width:60px"><P>12</P></TD></TR><TR><TD style="height:18px; width:60px"><P>1</P></TD><TD style="height:18px; width:60px"><P>1-2a</P></TD><TD style="height:18px; width:60px"><P>1-2d</P></TD><TD style="height:18px; width:60px"><P>3-4a 1</P></TD><TD style="height:18px; width:60px"><P>3-4a 2</P></TD><TD style="height:18px; width:60px"><P>3-4a 3</P></TD><TD style="height:18px; width:60px"><P>3-4b 1</P></TD><TD style="height:18px; width:60px"><P>3-4b 2</P></TD><TD style="height:18px; width:60px"><P>3-5b 1</P></TD><TD style="height:18px; width:60px"><P>3-5b 2</P></TD><TD style="height:18px; width:70px"><P>3-5b 3</P></TD><TD style="height:18px; width:60px"> </TD><TD style="height:18px; width:60px"> </TD></TR><TR><TD style="height:18px; width:60px"><P>2</P></TD><TD style="height:18px; width:60px"><P>4-1a 1</P></TD><TD style="height:18px; width:60px"><P>4-1a 2</P></TD><TD style="height:18px; width:60px"><P>7-2b</P></TD><TD style="height:18px; width:60px"><P>4-1b 1</P></TD><TD style="height:18px; width:60px"><P>4-1b 2</P></TD><TD style="height:18px; width:60px"><P>4-1b 3</P></TD><TD style="height:18px; width:60px"><P>5-2c</P></TD><TD style="height:18px; width:60px"><P>6-1b 1</P></TD><TD style="height:18px; width:60px"><P>6-1b 2</P></TD><TD style="height:18px; width:70px"><P>6-1b 3</P></TD><TD style="height:18px; width:60px"><P>6-1c</P></TD><TD style="height:18px; width:60px"><P>6-1d</P></TD></TR><TR><TD style="height:18px; width:60px"><P>3</P></TD><TD style="height:18px; width:60px"><P>7-2a 1</P></TD><TD style="height:18px; width:60px"><P>7-2a 2</P></TD><TD style="height:18px; width:60px"><P>4-1a 3</P></TD><TD style="height:18px; width:60px"><P>7-2e</P></TD><TD style="height:18px; width:60px"><P>8-1a 1</P></TD><TD style="height:18px; width:60px"><P>8-1a 2</P></TD><TD style="height:18px; width:60px"><P>8-1a 3</P></TD><TD style="height:18px; width:60px"><P>9-1a</P></TD><TD style="height:18px; width:60px"><P>9-1b 1</P></TD><TD style="height:18px; width:70px"><P>9-1b 2</P></TD><TD style="height:18px; width:60px"><P>9-1b 3</P></TD><TD style="height:18px; width:60px"> </TD></TR><TR><TD style="height:18px; width:60px"><P>4</P></TD><TD style="height:18px; width:60px"><P>10-3a 1</P></TD><TD style="height:18px; width:60px"><P>10-3a 2</P></TD><TD style="height:18px; width:60px"><P>10-3a 3</P></TD><TD style="height:18px; width:60px"><P>10-3c 1</P></TD><TD style="height:18px; width:60px"><P>10-3c 2</P></TD><TD style="height:18px; width:60px"><P>10-3c 3</P></TD><TD style="height:18px; width:60px"><P>10-5a 1</P></TD><TD style="height:18px; width:60px"><P>10-5a 2</P></TD><TD style="height:18px; width:60px"><P>10-5a 3</P></TD><TD style="height:18px; width:70px"><P>11-1c 1</P></TD><TD style="height:18px; width:60px"><P>11-1c 2</P></TD><TD style="height:18px; width:60px"><P>11-1c 3</P></TD></TR><TR><TD style="height:18px; width:60px"><P>5</P></TD><TD style="height:18px; width:60px"><P>12-1b 1</P></TD><TD style="height:18px; width:60px"><P>12-1b 2</P></TD><TD style="height:18px; width:60px"><P>12-1b 3</P></TD><TD style="height:18px; width:60px"><P>12-1c 1</P></TD><TD style="height:18px; width:60px"><P>12-1c 2</P></TD><TD style="height:18px; width:60px"><P>12-1c 3</P></TD><TD style="height:18px; width:60px"><P>12-1d 1</P></TD><TD style="height:18px; width:60px"><P>12-1d 2</P></TD><TD style="height:18px; width:60px"><P>12-1d 3</P></TD><TD style="height:18px; width:70px"><P>13-1a 1</P></TD><TD style="height:18px; width:60px"><P>13-1a 2</P></TD><TD style="height:18px; width:60px"><P>13-1a 3</P></TD></TR><TR><TD style="height:18px; width:60px"><P>6</P></TD><TD style="height:18px; width:60px"><P>13-1b 1</P></TD><TD style="height:18px; width:60px"><P>13-1b 2</P></TD><TD style="height:18px; width:60px"><P>13-1b 3</P></TD><TD style="height:18px; width:60px"><P>14-1c 1</P></TD><TD style="height:18px; width:60px"><P>14-1c 2</P></TD><TD style="height:18px; width:60px"><P>14-1c 3</P></TD><TD style="height:18px; width:60px"><P>14-1e 1</P></TD><TD style="height:18px; width:60px"><P>14-1e 2</P></TD><TD style="height:18px; width:60px"><P>14-1e 3</P></TD><TD style="height:18px; width:70px"><P>15-2b 1</P></TD><TD style="height:18px; width:60px"><P>15-2b 2</P></TD><TD style="height:18px; width:60px"><P>15-2b 3</P></TD></TR><TR><TD style="height:18px; width:60px"><P>7</P></TD><TD style="height:18px; width:60px"><P>15-3a 1</P></TD><TD style="height:18px; width:60px"><P>15-3a 2</P></TD><TD style="height:18px; width:60px"><P>15-3a 3</P></TD><TD style="height:18px; width:60px"><P>15-3b 1</P></TD><TD style="height:18px; width:60px"><P>15-3b 2</P></TD><TD style="height:18px; width:60px"><P>15-3b 3</P></TD><TD style="height:18px; width:60px"><P>15-3d 1</P></TD><TD style="height:18px; width:60px"><P>15-3d 2</P></TD><TD style="height:18px; width:60px"><P>15-3d 3</P></TD><TD style="height:18px; width:70px"><P>16-3c 1</P></TD><TD style="height:18px; width:60px"><P>16-3c 2</P></TD><TD style="height:18px; width:60px"><P>16-3c 3</P></TD></TR><TR><TD style="height:18px; width:60px"><P>8</P></TD><TD style="height:18px; width:60px"><P>16-3e</P></TD><TD style="height:18px; width:60px"><P>16-3d 1</P></TD><TD style="height:18px; width:60px"><P>16-3d 2</P></TD><TD style="height:18px; width:60px"><P>16-3d 3</P></TD><TD style="height:18px; width:60px"> </TD><TD style="height:18px; width:60px"> </TD><TD style="height:18px; width:60px"> </TD><TD style="height:18px; width:60px"> </TD><TD style="height:18px; width:60px"> </TD><TD style="height:18px; width:70px"> </TD><TD style="height:18px; width:60px"> </TD><TD style="height:18px; width:60px"> </TD></TR></TBODY></TABLE><P> </P><P>3 Colonies were picked from every plate which showed 3 or more colonies.</P><P> </P><P>Colonies in the plates after transformation:</P><P> </P><TABLE border="1" cellpadding="0" cellspacing="0" style="width:251px"><TBODY><TR><TD style="height:20px; width:123px"><P>Ribozyme/Colony</P></TD><TD style="height:20px; width:128px"><P>Number of colonies</P></TD></TR><TR><TD style="height:20px; width:123px"><P>1-2a</P></TD><TD style="height:20px; width:128px"><P>1</P></TD></TR><TR><TD style="height:20px; width:123px"><P>1-2d</P></TD><TD style="height:20px; width:128px"><P>1</P></TD></TR><TR><TD style="height:20px; width:123px"><P>1-2e</P></TD><TD style="height:20px; width:128px"><P>0</P></TD></TR><TR><TD style="height:20px; width:123px"><P>3-4a</P></TD><TD style="height:20px; width:128px"><P>&gt;3</P></TD></TR><TR><TD style="height:20px; width:123px"><P>3-4b</P></TD><TD style="height:20px; width:128px"><P>2</P></TD></TR><TR><TD style="height:20px; width:123px"><P>3-4d</P></TD><TD style="height:20px; width:128px"><P>0</P></TD></TR><TR><TD style="height:20px; width:123px"><P>3-5a</P></TD><TD style="height:20px; width:128px"><P>0</P></TD></TR><TR><TD style="height:20px; width:123px"><P>3-5b</P></TD><TD style="height:20px; width:128px"><P>&gt;3</P></TD></TR><TR><TD style="height:20px; width:123px"><P>3-5e</P></TD><TD style="height:20px; width:128px"><P>0</P></TD></TR><TR><TD style="height:20px; width:123px"><P>4-1a</P></TD><TD style="height:20px; width:128px"><P>&gt;3</P></TD></TR><TR><TD style="height:20px; width:123px"><P>4-1b</P></TD><TD style="height:20px; width:128px"><P>3</P></TD></TR><TR><TD style="height:20px; width:123px"><P>5-2c</P></TD><TD style="height:20px; width:128px"><P>1</P></TD></TR><TR><TD style="height:20px; width:123px"><P>5-2e</P></TD><TD style="height:20px; width:128px"><P>0</P></TD></TR><TR><TD style="height:20px; width:123px"><P>6-1b</P></TD><TD style="height:20px; width:128px"><P>&gt;3</P></TD></TR><TR><TD style="height:20px; width:123px"><P>6-1c</P></TD><TD style="height:20px; width:128px"><P>1</P></TD></TR><TR><TD style="height:20px; width:123px"><P>6-1d</P></TD><TD style="height:20px; width:128px"><P>1</P></TD></TR><TR><TD style="height:20px; width:123px"><P>7-2a</P></TD><TD style="height:20px; width:128px"><P>2</P></TD></TR><TR><TD style="height:20px; width:123px"><P>7-2b</P></TD><TD style="height:20px; width:128px"><P>1</P></TD></TR><TR><TD style="height:20px; width:123px"><P>7-2c</P></TD><TD style="height:20px; width:128px"><P>0</P></TD></TR><TR><TD style="height:20px; width:123px"><P>7-2e</P></TD><TD style="height:20px; width:128px"><P>1</P></TD></TR><TR><TD style="height:20px; width:123px"><P>8-1a</P></TD><TD style="height:20px; width:128px"><P>3</P></TD></TR><TR><TD style="height:20px; width:123px"><P>9-1a</P></TD><TD style="height:20px; width:128px"><P>1</P></TD></TR><TR><TD style="height:20px; width:123px"><P>9-1b</P></TD><TD style="height:20px; width:128px"><P>3</P></TD></TR><TR><TD style="height:20px; width:123px"><P>9-1c</P></TD><TD style="height:20px; width:128px"><P>0</P></TD></TR><TR><TD style="height:20px; width:123px"><P>10-2a</P></TD><TD style="height:20px; width:128px"><P>0</P></TD></TR><TR><TD style="height:20px; width:123px"><P>10-3a</P></TD><TD style="height:20px; width:128px"><P>3</P></TD></TR><TR><TD style="height:20px; width:123px"><P>10-3c</P></TD><TD style="height:20px; width:128px"><P>&gt;3</P></TD></TR><TR><TD style="height:20px; width:123px"><P>10-5a</P></TD><TD style="height:20px; width:128px"><P>&gt;3</P></TD></TR><TR><TD style="height:20px; width:123px"><P>10-5b</P></TD><TD style="height:20px; width:128px"><P>0</P></TD></TR><TR><TD style="height:20px; width:123px"><P>10-5c</P></TD><TD style="height:20px; width:128px"><P>0</P></TD></TR><TR><TD style="height:20px; width:123px"><P>11-1c</P></TD><TD style="height:20px; width:128px"><P>3</P></TD></TR><TR><TD style="height:20px; width:123px"><P>12-1b</P></TD><TD style="height:20px; width:128px"><P>&gt;3</P></TD></TR><TR><TD style="height:20px; width:123px"><P>12-1c</P></TD><TD style="height:20px; width:128px"><P>&gt;3</P></TD></TR><TR><TD style="height:20px; width:123px"><P>12-1d</P></TD><TD style="height:20px; width:128px"><P>&gt;3</P></TD></TR><TR><TD style="height:20px; width:123px"><P>12-2a</P></TD><TD style="height:20px; width:128px"><P>0</P></TD></TR><TR><TD style="height:20px; width:123px"><P>13-1a</P></TD><TD style="height:20px; width:128px"><P>&gt;3</P></TD></TR><TR><TD style="height:20px; width:123px"><P>13-1b</P></TD><TD style="height:20px; width:128px"><P>&gt;3</P></TD></TR><TR><TD style="height:20px; width:123px"><P>14-1c</P></TD><TD style="height:20px; width:128px"><P>&gt;3</P></TD></TR><TR><TD style="height:20px; width:123px"><P>14-1e</P></TD><TD style="height:20px; width:128px"><P>&gt;3</P></TD></TR><TR><TD style="height:20px; width:123px"><P>15-2b</P></TD><TD style="height:20px; width:128px"><P>&gt;3</P></TD></TR><TR><TD style="height:20px; width:123px"><P>15-3a</P></TD><TD style="height:20px; width:128px"><P>&gt;3</P></TD></TR><TR><TD style="height:20px; width:123px"><P>15-3b</P></TD><TD style="height:20px; width:128px"><P>&gt;3</P></TD></TR><TR><TD style="height:20px; width:123px"><P>15-3d</P></TD><TD style="height:20px; width:128px"><P>&gt;3</P></TD></TR><TR><TD style="height:20px; width:123px"><P>16-2b</P></TD><TD style="height:20px; width:128px"><P>0</P></TD></TR><TR><TD style="height:20px; width:123px"><P>16-3c</P></TD><TD style="height:20px; width:128px"><P>&gt;3</P></TD></TR><TR><TD style="height:20px; width:123px"><P>16-3d</P></TD><TD style="height:20px; width:128px"><P>3</P></TD></TR><TR><TD style="height:20px; width:123px"><P>16-3e</P></TD><TD style="height:20px; width:128px"><P>1</P></TD></TR><TR><TD style="height:20px; width:123px"><P>Religation Control</P></TD><TD style="height:20px; width:128px"><P>0</P></TD></TR></TBODY></TABLE><P> </P><P> </P><P>Results:</P><P> </P><TABLE border="0" cellpadding="0" cellspacing="0" style="width:131%"><TBODY><TR><TD colspan="3" style="height:25px; width:16.66%"><P>96-well</P></TD><TD colspan="3" style="height:25px; width:16.66%"><P>Positive Colonies green, negative Colonies red</P></TD><TD colspan="3" style="height:25px; width:16.66%"><P>Red font: Maybe positive</P></TD><TD colspan="3" style="height:25px; width:16.66%"> </TD><TD colspan="3" style="height:25px; width:16.66%"> </TD><TD colspan="3" style="height:25px; width:16.7%"> </TD></TR><TR><TD style="height:25px; width:7.68%"> </TD><TD style="height:25px; width:7.7%"> </TD><TD colspan="2" style="height:25px; width:7.7%"> </TD><TD style="height:25px; width:7.7%"> </TD><TD colspan="2" style="height:25px; width:7.68%"> </TD><TD style="height:25px; width:7.7%"> </TD><TD colspan="2" style="height:25px; width:7.7%"> </TD><TD style="height:25px; width:7.7%"> </TD><TD colspan="2" style="height:25px; width:7.68%"> </TD><TD style="height:25px; width:7.7%"> </TD><TD colspan="2" style="height:25px; width:7.7%"> </TD><TD style="height:25px; width:7.7%"> </TD><TD style="height:25px; width:7.66%"> </TD></TR><TR><TD style="height:25px; width:7.68%"> </TD><TD style="height:25px; width:7.7%"><P>1</P></TD><TD colspan="2" style="height:25px; width:7.7%"><P>2</P></TD><TD style="height:25px; width:7.7%"><P>3</P></TD><TD colspan="2" style="height:25px; width:7.68%"><P>4</P></TD><TD style="height:25px; width:7.7%"><P>5</P></TD><TD colspan="2" style="height:25px; width:7.7%"><P>6</P></TD><TD style="height:25px; width:7.7%"><P>7</P></TD><TD colspan="2" style="height:25px; width:7.68%"><P>8</P></TD><TD style="height:25px; width:7.7%"><P>9</P></TD><TD colspan="2" style="height:25px; width:7.7%"><P>10</P></TD><TD style="height:25px; width:7.7%"><P>11</P></TD><TD style="height:25px; width:7.66%"><P>12</P></TD></TR><TR><TD style="height:25px; width:7.68%"><P>1</P></TD><TD style="height:25px; width:7.7%"><P>1-2a</P></TD><TD colspan="2" style="height:25px; width:7.7%"><P>1-2d</P></TD><TD style="height:25px; width:7.7%"><P>3-4a 1</P></TD><TD colspan="2" style="height:25px; width:7.68%"><P>3-4a 2</P></TD><TD style="height:25px; width:7.7%"><P>3-4a 3</P></TD><TD colspan="2" style="height:25px; width:7.7%"><P>3-4b 1</P></TD><TD style="height:25px; width:7.7%"><P>3-4b 2</P></TD><TD colspan="2" style="height:25px; width:7.68%"><P>3-5b 1</P></TD><TD style="height:25px; width:7.7%"><P>3-5b 2</P></TD><TD colspan="2" style="height:25px; width:7.7%"><P>3-5b 3</P></TD><TD style="height:25px; width:7.7%"> </TD><TD style="height:25px; width:7.66%"> </TD></TR><TR><TD style="height:25px; width:7.68%"><P>2</P></TD><TD style="height:25px; width:7.7%"><P>4-1a 1</P></TD><TD colspan="2" style="height:25px; width:7.7%"><P>4-1a 2</P></TD><TD style="height:25px; width:7.7%"><P>7-2b</P></TD><TD colspan="2" style="height:25px; width:7.68%"><P>4-1b 1</P></TD><TD style="height:25px; width:7.7%"><P>4-1b 2</P></TD><TD colspan="2" style="height:25px; width:7.7%"><P>4-1b 3</P></TD><TD style="height:25px; width:7.7%"><P>5-2c</P></TD><TD colspan="2" style="height:25px; width:7.68%"><P>6-1b 1</P></TD><TD style="height:25px; width:7.7%"><P>6-1b 2</P></TD><TD colspan="2" style="height:25px; width:7.7%"><P>6-1b 3</P></TD><TD style="height:25px; width:7.7%"><P>6-1c</P></TD><TD style="height:25px; width:7.66%"><P>6-1d</P></TD></TR><TR><TD style="height:25px; width:7.68%"><P>3</P></TD><TD style="height:25px; width:7.7%"><P>7-2a 1</P></TD><TD colspan="2" style="height:25px; width:7.7%"><P>7-2a 2</P></TD><TD style="height:25px; width:7.7%"><P>4-1a 3</P></TD><TD colspan="2" style="height:25px; width:7.68%"><P>7-2e</P></TD><TD style="height:25px; width:7.7%"><P>8-1a 1</P></TD><TD colspan="2" style="height:25px; width:7.7%"><P>8-1a 2</P></TD><TD style="height:25px; width:7.7%"><P>8-1a 3</P></TD><TD colspan="2" style="height:25px; width:7.68%"><P>9-1a</P></TD><TD style="height:25px; width:7.7%"><P>9-1b 1</P></TD><TD colspan="2" style="height:25px; width:7.7%"><P>9-1b 2 + 6-1b 3</P></TD><TD style="height:25px; width:7.7%"><P>9-1b 3</P></TD><TD style="height:25px; width:7.66%"> </TD></TR><TR><TD style="height:25px; width:7.68%"><P>4</P></TD><TD style="height:25px; width:7.7%"><P>10-3a 1</P></TD><TD colspan="2" style="height:25px; width:7.7%"><P>10-3a 2</P></TD><TD style="height:25px; width:7.7%"><P>10-3a 3</P></TD><TD colspan="2" style="height:25px; width:7.68%"><P>10-3c 1</P></TD><TD style="height:25px; width:7.7%"><P>10-3c 2</P></TD><TD colspan="2" style="height:25px; width:7.7%"><P>10-3c 3</P></TD><TD style="height:25px; width:7.7%"><P>10-5a 1</P></TD><TD colspan="2" style="height:25px; width:7.68%"><P>10-5a 2</P></TD><TD style="height:25px; width:7.7%"><P>10-5a 3</P></TD><TD colspan="2" style="height:25px; width:7.7%"><P>11-1c 1</P></TD><TD style="height:25px; width:7.7%"><P>11-1c 2</P></TD><TD style="height:25px; width:7.66%"><P>11-1c 3</P></TD></TR><TR><TD style="height:25px; width:7.68%"><P>5</P></TD><TD style="height:25px; width:7.7%"><P>12-1b 1</P></TD><TD colspan="2" style="height:25px; width:7.7%"><P>12-1b 2</P></TD><TD style="height:25px; width:7.7%"><P>12-1b 3</P></TD><TD colspan="2" style="height:25px; width:7.68%"><P>12-1c 1</P></TD><TD style="height:25px; width:7.7%"><P>12-1c 2</P></TD><TD colspan="2" style="height:25px; width:7.7%"><P>12-1c 3</P></TD><TD style="height:25px; width:7.7%"><P>12-1d 1</P></TD><TD colspan="2" style="height:25px; width:7.68%"><P>12-1d 2</P></TD><TD style="height:25px; width:7.7%"><P>12-1d 3</P></TD><TD colspan="2" style="height:25px; width:7.7%"><P>13-1a 1</P></TD><TD style="height:25px; width:7.7%"><P>13-1a 2</P></TD><TD style="height:25px; width:7.66%"><P>13-1a 3</P></TD></TR><TR><TD style="height:25px; width:7.68%"><P>6</P></TD><TD style="height:25px; width:7.7%"><P>13-1b 1</P></TD><TD colspan="2" style="height:25px; width:7.7%"><P>13-1b 2</P></TD><TD style="height:25px; width:7.7%"><P>13-1b 3</P></TD><TD colspan="2" style="height:25px; width:7.68%"><P>14-1c 1</P></TD><TD style="height:25px; width:7.7%"><P>14-1c 2</P></TD><TD colspan="2" style="height:25px; width:7.7%"><P>14-1c 3</P></TD><TD style="height:25px; width:7.7%"><P>14-1e 1</P></TD><TD colspan="2" style="height:25px; width:7.68%"><P>14-1e 2</P></TD><TD style="height:25px; width:7.7%"><P>14-1e 3</P></TD><TD colspan="2" style="height:25px; width:7.7%"><P>15-2b 1</P></TD><TD style="height:25px; width:7.7%"><P>15-2b 2</P></TD><TD style="height:25px; width:7.66%"><P>15-2b 3</P></TD></TR><TR><TD style="height:25px; width:7.68%"><P>7</P></TD><TD style="height:25px; width:7.7%"><P>15-3a 1</P></TD><TD colspan="2" style="height:25px; width:7.7%"><P>15-3a 2</P></TD><TD style="height:25px; width:7.7%"><P>15-3a 3</P></TD><TD colspan="2" style="height:25px; width:7.68%"><P>15-3b 1</P></TD><TD style="height:25px; width:7.7%"><P>15-3b 2</P></TD><TD colspan="2" style="height:25px; width:7.7%"><P>15-3b 3</P></TD><TD style="height:25px; width:7.7%"><P>15-3d 1</P></TD><TD colspan="2" style="height:25px; width:7.68%"><P>15-3d 2</P></TD><TD style="height:25px; width:7.7%"><P>15-3d 3</P></TD><TD colspan="2" style="height:25px; width:7.7%"><P>16-3c 1</P></TD><TD style="height:25px; width:7.7%"><P>16-3c 2</P></TD><TD style="height:25px; width:7.66%"><P>16-3c 3</P></TD></TR><TR><TD style="height:25px; width:7.68%"><P>8</P></TD><TD style="height:25px; width:7.7%"><P>16-3e</P></TD><TD colspan="2" style="height:25px; width:7.7%"><P>16-3d 1</P></TD><TD style="height:25px; width:7.7%"><P>16-3d 2</P></TD><TD colspan="2" style="height:25px; width:7.68%"><P>16-3d 3</P></TD><TD style="height:25px; width:7.7%"> </TD><TD colspan="2" style="height:25px; width:7.7%"> </TD><TD style="height:25px; width:7.7%"> </TD><TD colspan="2" style="height:25px; width:7.68%"> </TD><TD style="height:25px; width:7.7%"> </TD><TD colspan="2" style="height:25px; width:7.7%"> </TD><TD style="height:25px; width:7.7%"> </TD><TD style="height:25px; width:7.66%"> </TD></TR><TR><TD> </TD><TD> </TD><TD> </TD><TD> </TD><TD> </TD><TD> </TD><TD> </TD><TD> </TD><TD> </TD><TD> </TD><TD> </TD><TD> </TD><TD> </TD><TD> </TD><TD> </TD><TD> </TD><TD> </TD><TD> </TD></TR></TBODY></TABLE><P> </P><DIV class="col-lg-12"><DIV class="imagewrapper"><DIV class="imagewrappercaption"><STRONG>
                Figure 8: Colony PCR row 1
            </STRONG></DIV></DIV></DIV><DIV class="col-lg-12"><DIV class="imagewrapper"><DIV class="imagewrappercaption"><STRONG>
                Figure 9: Colony PCR row 2
            </STRONG></DIV></DIV></DIV><DIV class="col-lg-12"><DIV class="imagewrapper"><DIV class="imagewrappercaption"><STRONG>
                Figure 10: Colony PCR row 3
            </STRONG></DIV></DIV></DIV><DIV class="col-lg-12"><DIV class="imagewrapper"><DIV class="imagewrappercaption"><STRONG>
                Figure 11: Colony PCR row 4
            </STRONG></DIV></DIV></DIV><DIV class="col-lg-12"><DIV class="imagewrapper"><DIV class="imagewrappercaption"><STRONG>
                Figure 13: Colony PCR row 5
            </STRONG></DIV></DIV></DIV><DIV class="col-lg-12"><DIV class="imagewrapper"><DIV class="imagewrappercaption"><STRONG>
                Figure 14: Colony PCR row 6
            </STRONG></DIV></DIV></DIV><DIV class="col-lg-12"><DIV class="imagewrapper"><DIV class="imagewrappercaption"><STRONG>
                Figure 14: Colony PCR row 7
            </STRONG></DIV></DIV></DIV></DIV></DIV></DIV></DIV><DIV class="row"><DIV class="col-lg-12"><DIV class="notebookentry"><H3 class="notebookheading">▼2015-08-13  Copper Click – Test of Labeled Part 1 and Splinted ligation</H3><DIV class="notebooktext"><P>To check if the azide-modified NTPs are incorporated the reactive azide is clicked to an alkyne activated FAM-alkyne fluorophore under copper catalyzed 1,3-dipolar azide-alkyne cycloaddition (CuAAC). Samples are after the reaction separated on a 20 % denaturing PAGE and visualized first in the fluorescent channel for FAM and afterwards stained with SYBR</P><P>Gold to visualize the RNA. Controls were performed as well.</P><P>Reaction conditions:</P><TABLE border="1" cellpadding="0" cellspacing="0"><TBODY><TR><TD style="width:245px"> </TD><TD style="width:113px"><P><STRONG>cStock</STRONG></P></TD><TD style="width:104px"><P><STRONG>cFinal</STRONG></P></TD><TD style="width:47px"><P><STRONG>V[µL]</STRONG></P></TD></TR><TR><TD style="width:245px"><P><STRONG>Phosphate buffer pH 7.0</STRONG></P></TD><TD style="width:113px"><P>100 mM</P></TD><TD style="width:104px"><P>50 mM</P></TD><TD style="width:47px"><P>12,5</P></TD></TR><TR><TD style="width:245px"><P><STRONG>FAM alkyne</STRONG></P></TD><TD style="width:113px"><P>10 µM</P></TD><TD style="width:104px"><P>400 nM</P></TD><TD style="width:47px"><P>1</P></TD></TR><TR><TD style="width:245px"><P><STRONG>Azide modified RNA</STRONG></P></TD><TD style="width:113px"><P>1 µM</P></TD><TD style="width:104px"><P>200 nM</P></TD><TD style="width:47px"><P>5</P></TD></TR><TR><TD style="width:245px"><P><STRONG>Cu(II)SO4</STRONG></P></TD><TD style="width:113px"><P>2 mM</P></TD><TD style="width:104px"><P>100 µM</P></TD><TD style="width:47px"><P>1.25</P></TD></TR><TR><TD style="width:245px"><P><STRONG>THPTA</STRONG></P></TD><TD style="width:113px"><P>5 mM</P></TD><TD style="width:104px"><P>500 µM</P></TD><TD style="width:47px"><P>2.5</P></TD></TR><TR><TD style="width:245px"><P><STRONG>Sodium ascorbate</STRONG></P></TD><TD style="width:113px"><P>10 mM</P></TD><TD style="width:104px"><P>1 mM</P></TD><TD style="width:47px"><P>2.5</P></TD></TR><TR><TD style="width:245px"><P><STRONG>H2O</STRONG></P></TD><TD style="width:113px"> </TD><TD style="width:104px"><P>Ad 25</P></TD><TD style="width:47px"><P>0.25</P></TD></TR><TR><TD style="width:245px"><P><STRONG>Final</STRONG></P></TD><TD style="width:113px"> </TD><TD style="width:104px"><P> </P></TD><TD style="width:47px"><P><STRONG><EM>25</EM></STRONG></P></TD></TR></TBODY></TABLE><P>Conditions as Winz, 2012.</P><P> </P><UL><LI>All compounds were mixed in the given order</LI><LI>Cu(II)SO4 and THPTA and H2O were mixed before adding to the mixture to let THPTA chelat the Cu</LI><LI>Lastly sodium ascorbate was added to reduce the Cu(II) to Cu(I)</LI><LI>Reaction was incubated for 1 h at 37 °C and afterwards put at -20 °C</LI></UL><P>Checking success on a 20 % denaturing PAGE</P><UL><LI>Samples were mixed with an appropriate amount of 2 x loading dye and separated on a 20 % denaturing PAGE</LI><LI>Gel was scanned with a Biorad ChemDoc, GE in fluorescent mode using the pre-set FAM parameters</LI><LI>Afterwards gel was stained using SYBR gold and scanned with the Typoon in SYBR Gold mode</LI></UL><P><STRONG>Result and Outlook:</STRONG></P><P>Labeling of RNA with azide and splinted ligation was only partially successful. Labeling and splinted ligation was repeated with different conditions.</P></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV><DIV class="container"><DIV class="content"><DIV class="row"><DIV class="col-lg-12"><DIV class="panel panel-default"><DIV class="panel-heading"><H3 class="basicheader">week number 32</H3></DIV><DIV class="panel-body"><DIV class="row"><DIV class="col-lg-12"><DIV class="notebookentry"><H3 class="notebookheading">▼2015-08-03  Digest – Ribozymes (Sequencing) (3.8.15)
</H3><DIV class="notebooktext"><TABLE border="1" cellpadding="0" cellspacing="0"><TBODY><TR><TD style="width:307px"><P> </P></TD><TD style="width:307px"><P>Volume for 30µl [µl]</P></TD></TR><TR><TD style="width:307px"><P>DNA</P></TD><TD style="width:307px"><P>25</P></TD></TR><TR><TD style="width:307px"><P>CutSmart</P></TD><TD style="width:307px"><P>2,5</P></TD></TR><TR><TD style="width:307px"><P>Bmt</P></TD><TD style="width:307px"><P>0,5</P></TD></TR><TR><TD style="width:307px"><P>BamHI –HF</P></TD><TD style="width:307px"><P>0,5</P></TD></TR><TR><TD style="width:307px"><P>ddH2O</P></TD><TD style="width:307px"><P>1,5</P></TD></TR></TBODY></TABLE><UL><LI>5µl 2Log-Ladder (NEB)</LI><LI>5µl Purple Loading Dye (NEB)</LI></UL></DIV></DIV></DIV></DIV><DIV class="row"><DIV class="col-lg-12"><DIV class="notebookentry"><H3 class="notebookheading">▼2015-08-04  Ribozyme 1 _CFTR 1 T_ Digest and Ligation</H3><DIV class="notebooktext"><P>j PCR, PCR Purification</P><P>k Digest, Gel</P><P>l Ligation</P><P>m Transformation</P><P> </P><P> </P><P><STRONG>Digest Rib 1A, B</STRONG></P><P> </P><TABLE border="1" cellpadding="0" cellspacing="0"><TBODY><TR><TD style="width:307px"><P> </P></TD><TD style="width:307px"><P>Volume for 30µl [µl]</P></TD></TR><TR><TD style="width:307px"><P>DNA</P></TD><TD style="width:307px"><P>25</P></TD></TR><TR><TD style="width:307px"><P>CutSmart</P></TD><TD style="width:307px"><P>2,5</P></TD></TR><TR><TD style="width:307px"><P>Bmt</P></TD><TD style="width:307px"><P>0,5</P></TD></TR><TR><TD style="width:307px"><P>BamHI –HF</P></TD><TD style="width:307px"><P>0,5</P></TD></TR><TR><TD style="width:307px"><P>ddH2O</P></TD><TD style="width:307px"><P>1,5</P></TD></TR></TBODY></TABLE><P> </P><P><STRONG>Ligation 1A, B</STRONG></P><P> </P><TABLE border="1" cellpadding="0" cellspacing="0"><TBODY><TR><TD style="width:205px"><P>Ribozyme</P></TD><TD style="width:205px"><P>[ng/µl]</P></TD><TD style="width:205px"><P>[µl]</P></TD></TR><TR><TD style="width:205px"><P>1A</P></TD><TD style="width:205px"><P>20,4</P></TD><TD style="width:205px"><P>0,20</P></TD></TR><TR><TD style="width:205px"><P>1B</P></TD><TD style="width:205px"><P>13,4</P></TD><TD style="width:205px"><P>0,30</P></TD></TR></TBODY></TABLE><P> </P><TABLE border="1" cellpadding="0" cellspacing="0"><TBODY><TR><TD style="width:307px"><P> </P></TD><TD colspan="2" style="width:307px"><P>Concentration [µl]</P></TD></TR><TR><TD style="width:307px"><P>Backbone</P></TD><TD colspan="2" style="width:307px"><P>2</P></TD></TR><TR><TD style="width:307px"><P>Insert</P></TD><TD colspan="2" style="width:307px"><P>4ng</P></TD></TR><TR><TD style="width:307px"><P>T4- Ligase</P></TD><TD colspan="2" style="width:307px"><P>1</P></TD></TR><TR><TD style="width:307px"><P>Buffer</P></TD><TD colspan="2" style="width:307px"><P>1</P></TD></TR><TR><TD style="width:307px"><P>ddH2O</P></TD><TD style="width:153px"><P>5,8 (Rib 1A)</P></TD><TD style="width:153px"><P>5,7 (Rib 1B)</P></TD></TR></TBODY></TABLE></DIV></DIV></DIV></DIV><DIV class="row"><DIV class="col-lg-12"><DIV class="notebookentry"><H3 class="notebookheading">▼2015-08-06  Insert cloning into the pSB1C3 Vector with MCS, pcat and Ribozyme</H3><DIV class="notebooktext"><P>Procedures:</P><P> </P><P>Digestion:</P><P> </P><P>Description:</P><P> </P><P>Steps:</P><P> </P><OL><LI>Set up reaction according to protocol:</LI></OL><P>                ddH2O for a final volume of 20 µl</P><P>                2 µl of 10x Reaction Buffer (e.g. NEB CutSmart)</P><P>                0.5 µl of selected Enzyme(s)</P><P>                ca. 1 µl of mini prep DNA (Range 200-1000 ng)</P><P> </P><OL><LI>Incubate at 37°C for 60'</LI></OL><P> </P><P>The digestion was made in two steps in order to trigger the exonuclease activity of one of the endonucleases. The cutting sites of BglII and SalI are to narrow to each other for a endonuclease activity.</P><P> </P><P>Step 1: Digestion with BglII</P><P> </P><P>Components:</P><P> </P><P>1 µg of Vector DNA</P><P>2 µl Cutsmart</P><P>1 µl of BglII</P><P>ad 20 µl ddH2O</P><P> </P><P> </P><P>Step 2: Digestion with SalI, triggering of SalI's exonuclease activity</P><P> </P><P>Components:</P><P> </P><P>8 µl of SalI-HF</P><P>10 µl of Cutsmart</P><P>52µl ddH2O</P><P>30 µl DNA</P><P> </P><P>Aterwards a dephosphorylation and a PCR purification was made</P><P> </P><P>Step 1: BglII digestion:</P><P> </P><TABLE border="1" cellpadding="0" cellspacing="0" style="width:320px"><TBODY><TR><TD style="height:20px; width:80px"><P>Nummer</P></TD><TD style="height:20px; width:80px"><P>c</P></TD><TD style="height:20px; width:80px"><P>µl/µg</P></TD><TD style="height:20px; width:80px"><P>µl ddH20</P></TD></TR><TR><TD style="height:20px; width:80px"><P>7-2</P></TD><TD style="height:20px; width:80px"><P>224</P></TD><TD style="height:20px; width:80px"><P>5</P></TD><TD style="height:20px; width:80px"><P>12,5</P></TD></TR><TR><TD style="height:20px; width:80px"><P>8-1</P></TD><TD style="height:20px; width:80px"><P>210</P></TD><TD style="height:20px; width:80px"><P>5</P></TD><TD style="height:20px; width:80px"><P>12,5</P></TD></TR><TR><TD style="height:20px; width:80px"><P>8-2</P></TD><TD style="height:20px; width:80px"><P>519</P></TD><TD style="height:20px; width:80px"><P>2</P></TD><TD style="height:20px; width:80px"><P>15,5</P></TD></TR><TR><TD style="height:20px; width:80px"><P>8-3</P></TD><TD style="height:20px; width:80px"><P>471</P></TD><TD style="height:20px; width:80px"><P>3</P></TD><TD style="height:20px; width:80px"><P>14,5</P></TD></TR><TR><TD style="height:20px; width:80px"><P>9-1</P></TD><TD style="height:20px; width:80px"><P>205</P></TD><TD style="height:20px; width:80px"><P>5</P></TD><TD style="height:20px; width:80px"><P>12,5</P></TD></TR><TR><TD style="height:20px; width:80px"><P>9-2</P></TD><TD style="height:20px; width:80px"><P>507</P></TD><TD style="height:20px; width:80px"><P>2</P></TD><TD style="height:20px; width:80px"><P>15,5</P></TD></TR><TR><TD style="height:20px; width:80px"><P>9-5</P></TD><TD style="height:20px; width:80px"><P>448</P></TD><TD style="height:20px; width:80px"><P>3</P></TD><TD style="height:20px; width:80px"><P>14,5</P></TD></TR><TR><TD style="height:20px; width:80px"><P>10-1</P></TD><TD style="height:20px; width:80px"><P>226</P></TD><TD style="height:20px; width:80px"><P>5</P></TD><TD style="height:20px; width:80px"><P>12,5</P></TD></TR><TR><TD style="height:20px; width:80px"><P>10-2</P></TD><TD style="height:20px; width:80px"><P>220</P></TD><TD style="height:20px; width:80px"><P>5</P></TD><TD style="height:20px; width:80px"><P>12,5</P></TD></TR><TR><TD style="height:20px; width:80px"><P>10-3</P></TD><TD style="height:20px; width:80px"><P>369</P></TD><TD style="height:20px; width:80px"><P>3</P></TD><TD style="height:20px; width:80px"><P>14,5</P></TD></TR><TR><TD style="height:20px; width:80px"><P>10-5</P></TD><TD style="height:20px; width:80px"><P>542</P></TD><TD style="height:20px; width:80px"><P>2</P></TD><TD style="height:20px; width:80px"><P>15,5</P></TD></TR><TR><TD style="height:20px; width:80px"><P>11-1</P></TD><TD style="height:20px; width:80px"><P>307</P></TD><TD style="height:20px; width:80px"><P>4</P></TD><TD style="height:20px; width:80px"><P>13,5</P></TD></TR><TR><TD style="height:20px; width:80px"><P>12-1</P></TD><TD style="height:20px; width:80px"><P>534</P></TD><TD style="height:20px; width:80px"><P>2</P></TD><TD style="height:20px; width:80px"><P>15,5</P></TD></TR><TR><TD style="height:20px; width:80px"><P>12-2</P></TD><TD style="height:20px; width:80px"><P>106</P></TD><TD style="height:20px; width:80px"><P>10</P></TD><TD style="height:20px; width:80px"><P>7,5</P></TD></TR><TR><TD style="height:20px; width:80px"><P>12-4</P></TD><TD style="height:20px; width:80px"><P>499</P></TD><TD style="height:20px; width:80px"><P>3</P></TD><TD style="height:20px; width:80px"><P>14,5</P></TD></TR><TR><TD style="height:20px; width:80px"><P>13-1</P></TD><TD style="height:20px; width:80px"><P>221</P></TD><TD style="height:20px; width:80px"><P>5</P></TD><TD style="height:20px; width:80px"><P>12,5</P></TD></TR><TR><TD style="height:20px; width:80px"><P>14-1</P></TD><TD style="height:20px; width:80px"><P>542</P></TD><TD style="height:20px; width:80px"><P>2</P></TD><TD style="height:20px; width:80px"><P>15,5</P></TD></TR><TR><TD style="height:20px; width:80px"><P>14-3</P></TD><TD style="height:20px; width:80px"><P>303</P></TD><TD style="height:20px; width:80px"><P>4</P></TD><TD style="height:20px; width:80px"><P>13,5</P></TD></TR><TR><TD style="height:20px; width:80px"><P>15-2</P></TD><TD style="height:20px; width:80px"><P>407</P></TD><TD style="height:20px; width:80px"><P>3</P></TD><TD style="height:20px; width:80px"><P>14,5</P></TD></TR><TR><TD style="height:20px; width:80px"><P>15-3</P></TD><TD style="height:20px; width:80px"><P>317</P></TD><TD style="height:20px; width:80px"><P>4</P></TD><TD style="height:20px; width:80px"><P>13,5</P></TD></TR><TR><TD style="height:20px; width:80px"><P>3-4</P></TD><TD style="height:20px; width:80px"><P>130</P></TD><TD style="height:20px; width:80px"><P>8</P></TD><TD style="height:20px; width:80px"><P>9,5</P></TD></TR><TR><TD style="height:20px; width:80px"><P>3-5</P></TD><TD style="height:20px; width:80px"><P>120</P></TD><TD style="height:20px; width:80px"><P>9</P></TD><TD style="height:20px; width:80px"><P>8,5</P></TD></TR><TR><TD style="height:20px; width:80px"><P>4-1</P></TD><TD style="height:20px; width:80px"><P>129</P></TD><TD style="height:20px; width:80px"><P>8</P></TD><TD style="height:20px; width:80px"><P>9,5</P></TD></TR><TR><TD style="height:20px; width:80px"><P>5-2</P></TD><TD style="height:20px; width:80px"><P>64</P></TD><TD style="height:20px; width:80px"><P>16</P></TD><TD style="height:20px; width:80px"><P>1,5</P></TD></TR><TR><TD style="height:20px; width:80px"><P>6-1</P></TD><TD style="height:20px; width:80px"><P>89</P></TD><TD style="height:20px; width:80px"><P>12</P></TD><TD style="height:20px; width:80px"><P>5,5</P></TD></TR><TR><TD style="height:20px; width:80px"><P>16-3</P></TD><TD style="height:20px; width:80px"><P>85</P></TD><TD style="height:20px; width:80px"><P>12</P></TD><TD style="height:20px; width:80px"><P>5,5</P></TD></TR></TBODY></TABLE><P> </P><P>    Ligation</P><P> </P><P>Description:</P><P> </P><P>Materials and chemicals:</P><P>2 µl 10x T4 Ligase Buffer: thawed and resuspended at room temperature</P><P>1 parts Vector DNA (mol not l)</P><P>3 part Insert DNA (mol not l)</P><P>20 µl Nuclease free water</P><P>1 µl T4 DNA Ligase</P><P> </P><P>Steps:</P><P> </P><OL><LI>Set up the reaction in a microcentrifuge tube on ice. T4 DNA Ligase should be added last. The molar ratio of vector to insert should be 1:3.</LI></OL><P> </P><OL><LI>Gently mix the reaction by pipetting up and down and microfuge briefly.</LI></OL><P> </P><OL><LI>For cohesive (sticky) ends, incubate at 16°C overnight or room temperature for 10 minutes.</LI></OL><P>                For blunt ends or single base overhangs, incubate at 16°C overnight or room temperature for                2 hours. Alternatively a high concentration of T4 Ligase can be used in a 10 minute ligation.</P><P> </P><OL><LI>Heat inactivate at 80°C for 10 minutes</LI></OL><P> </P><OL><LI>Chill on ice and transform 1-5 µl of the reaction into 50 µl of competent cells.</LI></OL><P> </P><P>Chemicals:</P><P>25 ng cutted Vector DNA</P><P>8 ng cutted Insert DNA</P><P>2 µl DNA-Ligase Buffer</P><P>1 µl DNA-Ligase</P><P>ad 20 µl ddH2O</P><P> </P><TABLE border="1" cellpadding="0" cellspacing="0" style="width:678px"><TBODY><TR><TD style="height:15px; width:59px"><P>Nr</P></TD><TD style="height:15px; width:122px"><P>Ribozym</P></TD><TD style="height:15px; width:57px"><P>c</P></TD><TD style="height:15px; width:68px"><P>Insert Nr</P></TD><TD style="height:15px; width:68px"><P>µl Insert</P></TD><TD style="height:15px; width:75px"><P>µl Vector</P></TD><TD style="height:15px; width:75px"><P>µl ddH20</P></TD><TD style="height:15px; width:83px"><P>T4-Ligase-Buffer</P></TD><TD style="height:15px; width:71px"><P>T4-Ligase</P></TD></TR><TR><TD style="height:15px; width:59px"><P>3-4</P></TD><TD style="height:15px; width:122px"><P>CFTR 1 C</P></TD><TD style="height:15px; width:57px"><P>8,5</P></TD><TD style="height:15px; width:68px"><P>C1V</P></TD><TD style="height:15px; width:68px"><P>0,2</P></TD><TD style="height:15px; width:75px"><P>3</P></TD><TD style="height:15px; width:75px"><P>13,8</P></TD><TD style="height:15px; width:83px"><P>2</P></TD><TD style="height:15px; width:71px"><P>1</P></TD></TR><TR><TD style="height:15px; width:59px"><P>3-5</P></TD><TD style="height:15px; width:122px"><P>CFTR 1 C</P></TD><TD style="height:15px; width:57px"><P>16,3</P></TD><TD style="height:15px; width:68px"><P>C1V</P></TD><TD style="height:15px; width:68px"><P>0,2</P></TD><TD style="height:15px; width:75px"><P>2</P></TD><TD style="height:15px; width:75px"><P>14,8</P></TD><TD style="height:15px; width:83px"><P>2</P></TD><TD style="height:15px; width:71px"><P>1</P></TD></TR><TR><TD style="height:15px; width:59px"><P>4-1</P></TD><TD style="height:15px; width:122px"><P>CFTR 2 T</P></TD><TD style="height:15px; width:57px"><P>15,2</P></TD><TD style="height:15px; width:68px"><P>C2V</P></TD><TD style="height:15px; width:68px"><P>0,2</P></TD><TD style="height:15px; width:75px"><P>2</P></TD><TD style="height:15px; width:75px"><P>14,8</P></TD><TD style="height:15px; width:83px"><P>2</P></TD><TD style="height:15px; width:71px"><P>1</P></TD></TR><TR><TD style="height:15px; width:59px"><P>5-2</P></TD><TD style="height:15px; width:122px"><P>CFTR 2 A</P></TD><TD style="height:15px; width:57px"><P>15,2</P></TD><TD style="height:15px; width:68px"><P>C2V</P></TD><TD style="height:15px; width:68px"><P>0,2</P></TD><TD style="height:15px; width:75px"><P>2</P></TD><TD style="height:15px; width:75px"><P>14,8</P></TD><TD style="height:15px; width:83px"><P>2</P></TD><TD style="height:15px; width:71px"><P>1</P></TD></TR><TR><TD style="height:15px; width:59px"><P>6-1</P></TD><TD style="height:15px; width:122px"><P>CFTR 2 C</P></TD><TD style="height:15px; width:57px"><P>21,4</P></TD><TD style="height:15px; width:68px"><P>C2V</P></TD><TD style="height:15px; width:68px"><P>0,2</P></TD><TD style="height:15px; width:75px"><P>2</P></TD><TD style="height:15px; width:75px"><P>14,8</P></TD><TD style="height:15px; width:83px"><P>2</P></TD><TD style="height:15px; width:71px"><P>1</P></TD></TR><TR><TD style="height:15px; width:59px"><P>7-2</P></TD><TD style="height:15px; width:122px"><P>CFTR 1 DE T</P></TD><TD style="height:15px; width:57px"><P>7,2</P></TD><TD style="height:15px; width:68px"><P>C1V</P></TD><TD style="height:15px; width:68px"><P>0,2</P></TD><TD style="height:15px; width:75px"><P>4</P></TD><TD style="height:15px; width:75px"><P>12,8</P></TD><TD style="height:15px; width:83px"><P>2</P></TD><TD style="height:15px; width:71px"><P>1</P></TD></TR><TR><TD style="height:15px; width:59px"><P>8-1</P></TD><TD style="height:15px; width:122px"><P>CFTR 1 DE A</P></TD><TD style="height:15px; width:57px"><P>3,8</P></TD><TD style="height:15px; width:68px"><P>C1V</P></TD><TD style="height:15px; width:68px"><P>0,2</P></TD><TD style="height:15px; width:75px"><P>7</P></TD><TD style="height:15px; width:75px"><P>9,8</P></TD><TD style="height:15px; width:83px"><P>2</P></TD><TD style="height:15px; width:71px"><P>1</P></TD></TR><TR><TD style="height:15px; width:59px"><P>8-2</P></TD><TD style="height:15px; width:122px"><P>CFTR 1 DE A</P></TD><TD style="height:15px; width:57px"><P>10,7</P></TD><TD style="height:15px; width:68px"><P>C1V</P></TD><TD style="height:15px; width:68px"><P>0,2</P></TD><TD style="height:15px; width:75px"><P>3</P></TD><TD style="height:15px; width:75px"><P>13,8</P></TD><TD style="height:15px; width:83px"><P>2</P></TD><TD style="height:15px; width:71px"><P>1</P></TD></TR><TR><TD style="height:15px; width:59px"><P>8-3</P></TD><TD style="height:15px; width:122px"><P>CFTR 1 DE A</P></TD><TD style="height:15px; width:57px"><P>16,6</P></TD><TD style="height:15px; width:68px"><P>C1V</P></TD><TD style="height:15px; width:68px"><P>0,2</P></TD><TD style="height:15px; width:75px"><P>2</P></TD><TD style="height:15px; width:75px"><P>14,8</P></TD><TD style="height:15px; width:83px"><P>2</P></TD><TD style="height:15px; width:71px"><P>1</P></TD></TR><TR><TD style="height:15px; width:59px"><P>9-1</P></TD><TD style="height:15px; width:122px"><P>CFTR 1 DE C</P></TD><TD style="height:15px; width:57px"><P>4,2</P></TD><TD style="height:15px; width:68px"><P>C1V</P></TD><TD style="height:15px; width:68px"><P>0,2</P></TD><TD style="height:15px; width:75px"><P>6</P></TD><TD style="height:15px; width:75px"><P>10,8</P></TD><TD style="height:15px; width:83px"><P>2</P></TD><TD style="height:15px; width:71px"><P>1</P></TD></TR><TR><TD style="height:15px; width:59px"><P>9-2</P></TD><TD style="height:15px; width:122px"><P>CFTR 1 DE C</P></TD><TD style="height:15px; width:57px"><P>5,7</P></TD><TD style="height:15px; width:68px"><P>C1V</P></TD><TD style="height:15px; width:68px"><P>0,2</P></TD><TD style="height:15px; width:75px"><P>5</P></TD><TD style="height:15px; width:75px"><P>11,8</P></TD><TD style="height:15px; width:83px"><P>2</P></TD><TD style="height:15px; width:71px"><P>1</P></TD></TR><TR><TD style="height:15px; width:59px"><P>9-5</P></TD><TD style="height:15px; width:122px"><P>CFTR 1 DE C</P></TD><TD style="height:15px; width:57px"><P>8,9</P></TD><TD style="height:15px; width:68px"><P>C1V</P></TD><TD style="height:15px; width:68px"><P>0,2</P></TD><TD style="height:15px; width:75px"><P>3</P></TD><TD style="height:15px; width:75px"><P>13,8</P></TD><TD style="height:15px; width:83px"><P>2</P></TD><TD style="height:15px; width:71px"><P>1</P></TD></TR><TR><TD style="height:15px; width:59px"><P>10-1</P></TD><TD style="height:15px; width:122px"><P>CFTR 2 DE T</P></TD><TD style="height:15px; width:57px"><P>5</P></TD><TD style="height:15px; width:68px"><P>C2V</P></TD><TD style="height:15px; width:68px"><P>0,2</P></TD><TD style="height:15px; width:75px"><P>5</P></TD><TD style="height:15px; width:75px"><P>11,8</P></TD><TD style="height:15px; width:83px"><P>2</P></TD><TD style="height:15px; width:71px"><P>1</P></TD></TR><TR><TD style="height:15px; width:59px"><P>10-2</P></TD><TD style="height:15px; width:122px"><P>CFTR 2 DE T</P></TD><TD style="height:15px; width:57px"><P>5,7</P></TD><TD style="height:15px; width:68px"><P>C2V</P></TD><TD style="height:15px; width:68px"><P>0,2</P></TD><TD style="height:15px; width:75px"><P>5</P></TD><TD style="height:15px; width:75px"><P>11,8</P></TD><TD style="height:15px; width:83px"><P>2</P></TD><TD style="height:15px; width:71px"><P>1</P></TD></TR><TR><TD style="height:15px; width:59px"><P>10-3</P></TD><TD style="height:15px; width:122px"><P>CFTR 2 DE T</P></TD><TD style="height:15px; width:57px"><P>10</P></TD><TD style="height:15px; width:68px"><P>C2V</P></TD><TD style="height:15px; width:68px"><P>0,2</P></TD><TD style="height:15px; width:75px"><P>3</P></TD><TD style="height:15px; width:75px"><P>13,8</P></TD><TD style="height:15px; width:83px"><P>2</P></TD><TD style="height:15px; width:71px"><P>1</P></TD></TR><TR><TD style="height:15px; width:59px"><P>10-5</P></TD><TD style="height:15px; width:122px"><P>CFTR 2 DE T</P></TD><TD style="height:15px; width:57px"><P>10,5</P></TD><TD style="height:15px; width:68px"><P>C2V</P></TD><TD style="height:15px; width:68px"><P>0,2</P></TD><TD style="height:15px; width:75px"><P>3</P></TD><TD style="height:15px; width:75px"><P>13,8</P></TD><TD style="height:15px; width:83px"><P>2</P></TD><TD style="height:15px; width:71px"><P>1</P></TD></TR><TR><TD style="height:15px; width:59px"><P>11-1</P></TD><TD style="height:15px; width:122px"><P>CFTR 2 DE a</P></TD><TD style="height:15px; width:57px"><P>15</P></TD><TD style="height:15px; width:68px"><P>C2V</P></TD><TD style="height:15px; width:68px"><P>0,2</P></TD><TD style="height:15px; width:75px"><P>2</P></TD><TD style="height:15px; width:75px"><P>14,8</P></TD><TD style="height:15px; width:83px"><P>2</P></TD><TD style="height:15px; width:71px"><P>1</P></TD></TR><TR><TD style="height:15px; width:59px"><P>12-1</P></TD><TD style="height:15px; width:122px"><P>CFTR 2 DE C</P></TD><TD style="height:15px; width:57px"><P>13,6</P></TD><TD style="height:15px; width:68px"><P>C2V</P></TD><TD style="height:15px; width:68px"><P>0,2</P></TD><TD style="height:15px; width:75px"><P>2</P></TD><TD style="height:15px; width:75px"><P>14,8</P></TD><TD style="height:15px; width:83px"><P>2</P></TD><TD style="height:15px; width:71px"><P>1</P></TD></TR><TR><TD style="height:15px; width:59px"><P>12-2</P></TD><TD style="height:15px; width:122px"><P>CFTR 2 DE C</P></TD><TD style="height:15px; width:57px"><P>9</P></TD><TD style="height:15px; width:68px"><P>C2V</P></TD><TD style="height:15px; width:68px"><P>0,2</P></TD><TD style="height:15px; width:75px"><P>3</P></TD><TD style="height:15px; width:75px"><P>13,8</P></TD><TD style="height:15px; width:83px"><P>2</P></TD><TD style="height:15px; width:71px"><P>1</P></TD></TR><TR><TD style="height:15px; width:59px"><P>13-1</P></TD><TD style="height:15px; width:122px"><P>GFP 1 DE</P></TD><TD style="height:15px; width:57px"><P>2,9</P></TD><TD style="height:15px; width:68px"><P>G1V</P></TD><TD style="height:15px; width:68px"><P>0,2</P></TD><TD style="height:15px; width:75px"><P>9</P></TD><TD style="height:15px; width:75px"><P>7,8</P></TD><TD style="height:15px; width:83px"><P>2</P></TD><TD style="height:15px; width:71px"><P>1</P></TD></TR><TR><TD style="height:15px; width:59px"><P>14-1</P></TD><TD style="height:15px; width:122px"><P>GFP 2 DE</P></TD><TD style="height:15px; width:57px"><P>17,6</P></TD><TD style="height:15px; width:68px"><P>G2V</P></TD><TD style="height:15px; width:68px"><P>0,2</P></TD><TD style="height:15px; width:75px"><P>2</P></TD><TD style="height:15px; width:75px"><P>14,8</P></TD><TD style="height:15px; width:83px"><P>2</P></TD><TD style="height:15px; width:71px"><P>1</P></TD></TR><TR><TD style="height:15px; width:59px"><P>14-3</P></TD><TD style="height:15px; width:122px"><P>GFP 2 DE</P></TD><TD style="height:15px; width:57px"><P>6</P></TD><TD style="height:15px; width:68px"><P>G2V</P></TD><TD style="height:15px; width:68px"><P>0,2</P></TD><TD style="height:15px; width:75px"><P>5</P></TD><TD style="height:15px; width:75px"><P>11,8</P></TD><TD style="height:15px; width:83px"><P>2</P></TD><TD style="height:15px; width:71px"><P>1</P></TD></TR><TR><TD style="height:15px; width:59px"><P>15-2</P></TD><TD style="height:15px; width:122px"><P>GFP 1</P></TD><TD style="height:15px; width:57px"><P>14,3</P></TD><TD style="height:15px; width:68px"><P>G1V</P></TD><TD style="height:15px; width:68px"><P>0,2</P></TD><TD style="height:15px; width:75px"><P>2</P></TD><TD style="height:15px; width:75px"><P>14,8</P></TD><TD style="height:15px; width:83px"><P>2</P></TD><TD style="height:15px; width:71px"><P>1</P></TD></TR><TR><TD style="height:15px; width:59px"><P>15-3</P></TD><TD style="height:15px; width:122px"><P>GFP 1</P></TD><TD style="height:15px; width:57px"><P>12,1</P></TD><TD style="height:15px; width:68px"><P>G1V</P></TD><TD style="height:15px; width:68px"><P>0,2</P></TD><TD style="height:15px; width:75px"><P>3</P></TD><TD style="height:15px; width:75px"><P>13,8</P></TD><TD style="height:15px; width:83px"><P>2</P></TD><TD style="height:15px; width:71px"><P>1</P></TD></TR><TR><TD style="height:15px; width:59px"><P>16-3</P></TD><TD style="height:15px; width:122px"><P>GFP 2</P></TD><TD style="height:15px; width:57px"><P>14</P></TD><TD style="height:15px; width:68px"><P>G2V</P></TD><TD style="height:15px; width:68px"><P>0,2</P></TD><TD style="height:15px; width:75px"><P>2</P></TD><TD style="height:15px; width:75px"><P>14,8</P></TD><TD style="height:15px; width:83px"><P>2</P></TD><TD style="height:15px; width:71px"><P>1</P></TD></TR></TBODY></TABLE><P> </P><TABLE border="1" cellpadding="0" cellspacing="0" style="width:160px"><TBODY><TR><TD style="height:20px; width:80px"><P>Insert Nr</P></TD><TD style="height:20px; width:80px"><P>c</P></TD></TR><TR><TD style="height:20px; width:80px"><P>C1V</P></TD><TD style="height:20px; width:80px"><P>49,8</P></TD></TR><TR><TD style="height:20px; width:80px"><P>C2V</P></TD><TD style="height:20px; width:80px"><P>41,4</P></TD></TR><TR><TD style="height:20px; width:80px"><P>G1V</P></TD><TD style="height:20px; width:80px"><P>46,6</P></TD></TR><TR><TD style="height:20px; width:80px"><P>G2V</P></TD><TD style="height:20px; width:80px"><P>40,6</P></TD></TR></TBODY></TABLE><P> </P><P> </P><P>KCM Transformation:</P><P> </P><P>Description:</P><P> </P><P>Steps:</P><P> </P><OL><LI>Take 50µl chemical competent E. coli from -80 freezer and thaw on ice</LI></OL><P> </P><OL><LI>Add (as master mix):</LI></OL><P>                2,5µl DNA</P><P>                10µl KCM 5x</P><P>                37,5µl H2O</P><P> </P><OL><LI>Incubate on ice for 30 minutes</LI></OL><P> </P><OL><LI>Heat shock at 42°C for 1 minute</LI></OL><P> </P><OL><LI>Incubate on ice for 2 minutes</LI></OL><P> </P><OL><LI>Add 900 µl of LB or 2x YT Medium</LI></OL><P> </P><OL><LI>Incubate on 37°C for 60min</LI></OL><P> </P><OL><LI>Centrifuge 5min at 1000g</LI></OL><P> </P><OL><LI>Take 900µl of supernatant and throw away</LI></OL><P> </P><OL><LI>Resuspend pellet in remaining media</LI></OL><P> </P><OL><LI>Plate out on agar with antibiotics (1:1 / 1:10)</LI></OL><P> </P><P>Colony PCR:</P><P> </P><P>Description:</P><P> </P><P>Materials and Chemicals:</P><P>PCR-tubes</P><P>Forward primer</P><P>Reverse primer</P><P>Masermix (dNTPs, Polymerase, buffer)</P><P>Water</P><P>Thermocycler</P><P> </P><P>Endvolume: 10 µl</P><P> </P><P>Steps:</P><P> </P><OL><LI>Pick colonies from plates. Solute one colony in about 20 µl of water.</LI></OL><P> </P><OL><LI>Give the colonies into 10 µl colony PCR solution with OneTaq Mastermix (which should be diluted to 1x in the end (e.g. you need 5 µl of 2x mastermix for 10 µl)) and primer (between 0,1 and 1 µl)</LI></OL><P> </P><P>Use the Thermocycler with an appropriate PCR program for at least 25 cycles</P><P> </P><P>Results:</P><P> </P><P>The colony PCR showed at least one positive clone for every ribozyme.</P><P> </P><P>Results:</P><P> </P><P>The plates gave a high number of clones. Out of them either new plates were made or single colonies were found, that were given into a colony PCR. Positive Ribozyme colonies were minipreped and gave a yield of 150-325 ng/µl DNA.</P><P> </P><P>First colony PCR: (Clones are named: x-y A: x = 1-15 (number of ribozyme), y = number of the colony, A: Colony capital letter (A-E))</P><P> </P><TABLE border="1" cellpadding="0" cellspacing="0"><TBODY><TR><TD style="width:302px"><P>Colony ID</P></TD><TD style="width:302px"><P>Positve/Negative/Multiple Bands</P></TD></TR><TR><TD style="width:302px"><P>15-3 A</P></TD><TD style="width:302px"><P>Pos</P></TD></TR><TR><TD style="width:302px"><P>15-3 B</P></TD><TD style="width:302px"><P>Pos</P></TD></TR><TR><TD style="width:302px"><P>15-3 C</P></TD><TD style="width:302px"><P>Neg</P></TD></TR><TR><TD style="width:302px"><P>15-3 D</P></TD><TD style="width:302px"><P>Pos</P></TD></TR><TR><TD style="width:302px"><P>14-3 A</P></TD><TD style="width:302px"><P>Neg</P></TD></TR><TR><TD style="width:302px"><P>14-3 B</P></TD><TD style="width:302px"><P>Neg</P></TD></TR><TR><TD style="width:302px"><P>12-1 A</P></TD><TD style="width:302px"><P>Pos</P></TD></TR><TR><TD style="width:302px"><P>12-1 B</P></TD><TD style="width:302px"><P>Pos</P></TD></TR><TR><TD style="width:302px"><P>12-1 C</P></TD><TD style="width:302px"><P>Pos</P></TD></TR><TR><TD style="width:302px"><P>12-1 D</P></TD><TD style="width:302px"><P>Pos</P></TD></TR><TR><TD style="width:302px"><P>12-1 E</P></TD><TD style="width:302px"><P>Neg</P></TD></TR><TR><TD style="width:302px"><P>10-3 A</P></TD><TD style="width:302px"><P>Double Band</P></TD></TR><TR><TD style="width:302px"><P>10-3 B</P></TD><TD style="width:302px"><P>Neg</P></TD></TR><TR><TD style="width:302px"><P>10-3 C</P></TD><TD style="width:302px"><P>Pos</P></TD></TR><TR><TD style="width:302px"><P>10-2 A</P></TD><TD style="width:302px"><P>Pos</P></TD></TR><TR><TD style="width:302px"><P>10-2 B</P></TD><TD style="width:302px"><P>Pos</P></TD></TR><TR><TD style="width:302px"><P>10-2 C</P></TD><TD style="width:302px"><P>Neg</P></TD></TR><TR><TD style="width:302px"><P>9-1 A</P></TD><TD style="width:302px"><P>Double Band</P></TD></TR><TR><TD style="width:302px"><P>9-1 B</P></TD><TD style="width:302px"><P>Double Band</P></TD></TR><TR><TD style="width:302px"><P>9-1 C</P></TD><TD style="width:302px"><P>Pos</P></TD></TR><TR><TD style="width:302px"><P>8-1 A</P></TD><TD style="width:302px"><P>Pos</P></TD></TR><TR><TD style="width:302px"><P>8-1 B</P></TD><TD style="width:302px"><P>Neg</P></TD></TR><TR><TD style="width:302px"><P>6-1 A</P></TD><TD style="width:302px"><P>Neg</P></TD></TR><TR><TD style="width:302px"><P>6-1 B</P></TD><TD style="width:302px"><P>Pos</P></TD></TR><TR><TD style="width:302px"><P>6-1 C</P></TD><TD style="width:302px"><P>Pos</P></TD></TR><TR><TD style="width:302px"><P>6-1 D</P></TD><TD style="width:302px"><P>Pos</P></TD></TR><TR><TD style="width:302px"><P>6-1 E</P></TD><TD style="width:302px"><P>Neg</P></TD></TR><TR><TD style="width:302px"><P>4-1 A</P></TD><TD style="width:302px"><P>Pos</P></TD></TR><TR><TD style="width:302px"><P>4-1 B</P></TD><TD style="width:302px"><P>Pos</P></TD></TR><TR><TD style="width:302px"><P>10-5 A</P></TD><TD style="width:302px"><P>Pos</P></TD></TR><TR><TD style="width:302px"><P>10-5 B</P></TD><TD style="width:302px"><P>Pos</P></TD></TR><TR><TD style="width:302px"><P>10-5 C</P></TD><TD style="width:302px"><P>Double Band</P></TD></TR><TR><TD style="width:302px"><P>10-5 D</P></TD><TD style="width:302px"><P>Neg</P></TD></TR><TR><TD style="width:302px"><P>10-5 E</P></TD><TD style="width:302px"><P>Neg</P></TD></TR><TR><TD style="width:302px"><P>11-1 A</P></TD><TD style="width:302px"><P>Neg</P></TD></TR><TR><TD style="width:302px"><P>11-1 B</P></TD><TD style="width:302px"><P>Neg</P></TD></TR><TR><TD style="width:302px"><P>11-1 C</P></TD><TD style="width:302px"><P>neg</P></TD></TR><TR><TD style="width:302px"><P>13-1 A</P></TD><TD style="width:302px"><P>Pos</P></TD></TR><TR><TD style="width:302px"><P>13-1 B</P></TD><TD style="width:302px"><P>Pos</P></TD></TR><TR><TD style="width:302px"><P>15-2 A</P></TD><TD style="width:302px"><P>Neg</P></TD></TR><TR><TD style="width:302px"><P>15-2 B</P></TD><TD style="width:302px"><P>Triple Band</P></TD></TR><TR><TD style="width:302px"><P>15-2 C</P></TD><TD style="width:302px"><P>Neg</P></TD></TR></TBODY></TABLE><P> </P><P> </P><P> </P></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV><DIV class="container"><DIV class="content"><DIV class="row"><DIV class="col-lg-12"><DIV class="panel panel-default"><DIV class="panel-heading"><H3 class="basicheader">week number 31</H3></DIV><DIV class="panel-body"><DIV class="row"><DIV class="col-lg-12"><DIV class="notebookentry"><H3 class="notebookheading">▼2015-07-27  PCR amplification of the ribozymes CFTR2 A/T/C for in vitro transcription</H3><DIV class="notebooktext"><P>Procedures:</P><P> </P><P>Amplification PCR 1</P><P> </P><P>Description</P><P> </P><P>Chemicals:</P><P>5 µl Q5 Master Mix</P><P>0,5 µl Ribozymes from stock solution</P><P>1 µl Primer fwd</P><P>1 µl Primer rev</P><P>2,5 µl ddH2O</P><P> </P><P>Program:</P><P> </P><P>98°C for 1 minute</P><P>------------------------------</P><P>98°C for 10 seconds</P><P>60°C for 20 seconds</P><P>72°C for 30 seconds</P><P>Repeat 35 times</P><P>------------------------------</P><P>72°C for 2 minutes</P><P>------------------------------</P><P>4°C for holding</P><P> </P><P>Results:</P><P> </P><P>After recognizing the success of the first PCR on a gel, a second PCR was made with to get more DNA.</P><P> </P><P>Amplification PCR 2:</P><P> </P><P>Description:</P><P> </P><P>Chemicals:</P><P> </P><P>25 µl Q5 Master Mix</P><P>0,5 µl Ribozymes from prior PCR reaction</P><P>5 µl Primer fwd</P><P>5 µl Primer rev</P><P>14,5 µl ddH2O</P><P> </P><P>Program:</P><P> </P><P>98°C for 30 seconds</P><P>------------------------------</P><P>98°C for 10 seconds</P><P>60°C for 15 seconds</P><P>72°C for 20 seconds</P><P>Repeat 35 times</P><P>------------------------------</P><P>72°C for 1 minutes</P><P>------------------------------</P><P>4°C for holding</P></DIV></DIV></DIV></DIV><DIV class="row"><DIV class="col-lg-12"><DIV class="notebookentry"><H3 class="notebookheading">▼2015-07-28  Digest-Ribozyme amplification</H3><DIV class="notebooktext"><TABLE border="1" cellpadding="0" cellspacing="0"><TBODY><TR><TD style="width:307px"><P> </P></TD><TD style="width:307px"><P>Volume [µl]</P></TD></TR><TR><TD style="width:307px"><P>DNA</P></TD><TD style="width:307px"><P>15</P></TD></TR><TR><TD style="width:307px"><P>CutSmart</P></TD><TD style="width:307px"><P>3</P></TD></TR><TR><TD style="width:307px"><P>BamHI-HF</P></TD><TD style="width:307px"><P>0.3</P></TD></TR><TR><TD style="width:307px"><P>BmtI-HF</P></TD><TD style="width:307px"><P>0.3</P></TD></TR><TR><TD style="width:307px"><P>ddH2O</P></TD><TD style="width:307px"><P>11.4</P></TD></TR></TBODY></TABLE><P> </P><P><STRONG>Conditions</STRONG></P><P> </P><UL><LI>Duration: 1h</LI><LI>Temperature: 37°C</LI><LI>350 rpm</LI></UL><P> </P><P><STRONG>Concentration</STRONG></P><P> </P><TABLE border="1" cellpadding="0" cellspacing="0" style="width:614px"><TBODY><TR><TD style="width:307px"><P>Ribozyme</P></TD><TD style="width:307px"><P>Concentration [ng/µl]</P></TD></TR><TR><TD style="width:307px"><P><STRONG>1</STRONG> CFTR 1 T</P></TD><TD style="width:307px"><P>8,9</P></TD></TR><TR><TD style="width:307px"><P><STRONG>2</STRONG> CFTR 1 A</P></TD><TD style="width:307px"><P>15,9</P></TD></TR><TR><TD style="width:307px"><P><STRONG>3</STRONG> CFTR 1 C</P></TD><TD style="width:307px"><P>22,9</P></TD></TR><TR><TD style="width:307px"><P><STRONG>4</STRONG> CFTR 2 T</P></TD><TD style="width:307px"><P>1,5</P></TD></TR><TR><TD style="width:307px"><P><STRONG>5</STRONG> CFTR 2 A</P></TD><TD style="width:307px"><P>15,6</P></TD></TR><TR><TD style="width:307px"><P><STRONG>6</STRONG> CFTR 2 C</P></TD><TD style="width:307px"><P>31,9</P></TD></TR><TR><TD style="width:307px"><P><STRONG>15</STRONG> GFP 1</P></TD><TD style="width:307px"><P>14,2</P></TD></TR><TR><TD style="height:18px; width:307px"><P><STRONG>16</STRONG> GFP 2</P></TD><TD style="height:18px; width:307px"><P>17,8</P></TD></TR></TBODY></TABLE><UL><LI>PCR Purification Kit</LI></UL></DIV></DIV></DIV></DIV><DIV class="row"><DIV class="col-lg-12"><DIV class="notebookentry"><H3 class="notebookheading">▼2015-07-28  Ligation (28.07.15)</H3><DIV class="notebooktext"><TABLE border="1" cellpadding="0" cellspacing="0"><TBODY><TR><TD style="width:307px"><P> </P></TD><TD style="width:307px"><P>Volume [µl]</P></TD></TR><TR><TD style="width:307px"><P>Backbone</P></TD><TD style="width:307px"><P>10ng -&gt; 2</P></TD></TR><TR><TD style="width:307px"><P>Insert</P></TD><TD style="width:307px"><P>4ng</P></TD></TR><TR><TD style="width:307px"><P>T4-Ligase</P></TD><TD style="width:307px"><P>1</P></TD></TR><TR><TD style="width:307px"><P>Buffer</P></TD><TD style="width:307px"><P>1</P></TD></TR><TR><TD style="width:307px"><P>ddH2O</P></TD><TD style="width:307px"><P>to 20µl</P></TD></TR></TBODY></TABLE><P> </P><P><STRONG>Conditions</STRONG></P><UL><LI>Duration: 15min</LI><LI>Temperature:  25°C</LI><LI>350 rpm</LI></UL></DIV></DIV></DIV></DIV><DIV class="row"><DIV class="col-lg-12"><DIV class="notebookentry"><H3 class="notebookheading">▼2015-07-28  KCM - Transformation</H3><DIV class="notebooktext"><P>See also LabGuru protocol</P></DIV></DIV></DIV></DIV><DIV class="row"><DIV class="col-lg-12"><DIV class="notebookentry"><H3 class="notebookheading">▼2015-07-29  Picking colonies</H3><DIV class="notebooktext"><P><STRONG>Picked colonies</STRONG></P><P> </P><TABLE border="1" cellpadding="0" cellspacing="0" style="width:107.96%"><TBODY><TR><TD style="height:47px; width:16.66%"><P>Ribozyme</P></TD><TD style="height:47px; width:16.66%"><P>Colony1</P></TD><TD style="height:47px; width:16.66%"><P>Colony 2</P></TD><TD style="height:47px; width:16.66%"><P>Colony 3</P></TD><TD style="height:47px; width:16.68%"><P>Colony 4</P></TD><TD style="height:47px; width:16.68%"><P>Colony 5</P></TD></TR><TR><TD style="height:28px; width:16.66%"><P><STRONG>1</STRONG> CFTR 1 T</P></TD><TD style="height:28px; width:16.66%"><P>/</P></TD><TD style="height:28px; width:16.66%"><P>/</P></TD><TD style="height:28px; width:16.66%"><P>/</P></TD><TD style="height:28px; width:16.68%"><P>/</P></TD><TD style="height:28px; width:16.68%"><P>/</P></TD></TR><TR><TD style="height:30px; width:16.66%"><P><STRONG>2</STRONG> CFTR 1 A</P></TD><TD style="height:30px; width:16.66%"><P>2-1</P></TD><TD style="height:30px; width:16.66%"><P>2-2</P></TD><TD style="height:30px; width:16.66%"><P>2-3</P></TD><TD style="height:30px; width:16.68%"><P>2-4</P></TD><TD style="height:30px; width:16.68%"><P>2-5</P></TD></TR><TR><TD style="height:30px; width:16.66%"><P><STRONG>3</STRONG> CFTR 1 C</P></TD><TD style="height:30px; width:16.66%"><P>3-1</P></TD><TD style="height:30px; width:16.66%"><P>3-2</P></TD><TD style="height:30px; width:16.66%"><P>3-3</P></TD><TD style="height:30px; width:16.68%"><P>3-4</P></TD><TD style="height:30px; width:16.68%"><P>3-5</P></TD></TR><TR><TD style="height:30px; width:16.66%"><P><STRONG>4</STRONG> CFTR 2 T</P></TD><TD style="height:30px; width:16.66%"><P>4-1</P></TD><TD style="height:30px; width:16.66%"><P>4-2</P></TD><TD style="height:30px; width:16.66%"><P>4-3</P></TD><TD style="height:30px; width:16.68%"><P>4-4</P></TD><TD style="height:30px; width:16.68%"><P>4-5</P></TD></TR><TR><TD style="height:30px; width:16.66%"><P><STRONG>5</STRONG> CFTR 2 A</P></TD><TD style="height:30px; width:16.66%"><P>5-1</P></TD><TD style="height:30px; width:16.66%"><P>5-2</P></TD><TD style="height:30px; width:16.66%"><P>5-3</P></TD><TD style="height:30px; width:16.68%"><P>5-4</P></TD><TD style="height:30px; width:16.68%"><P>5-5</P></TD></TR><TR><TD style="height:30px; width:16.66%"><P><STRONG>6</STRONG> CFTR 2 C</P></TD><TD style="height:30px; width:16.66%"><P>6-1</P></TD><TD style="height:30px; width:16.66%"><P>6-2</P></TD><TD style="height:30px; width:16.66%"><P>6-3</P></TD><TD style="height:30px; width:16.68%"><P>6-4</P></TD><TD style="height:30px; width:16.68%"><P>6-5</P></TD></TR><TR><TD style="height:28px; width:16.66%"><P><STRONG>15</STRONG> GFP 1</P></TD><TD style="height:28px; width:16.66%"><P>15-1</P></TD><TD style="height:28px; width:16.66%"><P>15-2</P></TD><TD style="height:28px; width:16.66%"><P>15-3</P></TD><TD style="height:28px; width:16.68%"><P>15-4</P></TD><TD style="height:28px; width:16.68%"><P>15-5</P></TD></TR><TR><TD style="height:31px; width:16.66%"><P><STRONG>16</STRONG> GFP 2</P></TD><TD style="height:31px; width:16.66%"><P>16-1</P></TD><TD style="height:31px; width:16.66%"><P>16-2</P></TD><TD style="height:31px; width:16.66%"><P>16-3</P></TD><TD style="height:31px; width:16.68%"><P>16-4</P></TD><TD style="height:31px; width:16.68%"><P>16-5</P></TD></TR></TBODY></TABLE></DIV></DIV></DIV></DIV><DIV class="row"><DIV class="col-lg-12"><DIV class="notebookentry"><H3 class="notebookheading">▼2015-07-29  colony PCR</H3><DIV class="notebooktext"><P><STRONG>Colony PCR</STRONG></P><P> </P><TABLE border="1" cellpadding="0" cellspacing="0"><TBODY><TR><TD style="width:307px"><P> </P></TD><TD style="width:307px"><P>Volume [ng/µl]</P></TD></TR><TR><TD style="width:307px"><P>Primer fwd</P></TD><TD style="width:307px"><P>0,5</P></TD></TR><TR><TD style="width:307px"><P>Primer rvs</P></TD><TD style="width:307px"><P>0,5</P></TD></TR><TR><TD style="width:307px"><P>OneTaq Polymerase</P></TD><TD style="width:307px"><P>5</P></TD></TR><TR><TD style="width:307px"><P>ddH2O</P></TD><TD style="width:307px"><P>4</P></TD></TR></TBODY></TABLE></DIV></DIV></DIV></DIV><DIV class="row"><DIV class="col-lg-12"><DIV class="notebookentry"><H3 class="notebookheading">▼2015-07-30  Repetition Ribozyme 1 and 15</H3><DIV class="notebooktext"><P><STRONG>PCR (GEL image)</STRONG></P><P> </P><P><STRONG>Digest (GEL image)</STRONG></P><P> </P><TABLE border="1" cellpadding="0" cellspacing="0"><TBODY><TR><TD style="width:307px"><P>Ribozyme</P></TD><TD style="width:307px"><P>Concentration [ng/µl]</P></TD></TR><TR><TD style="width:307px"><P><STRONG>1</STRONG> CFTR 1 T</P></TD><TD style="width:307px"><P>17,4</P></TD></TR><TR><TD style="width:307px"><P><STRONG>15</STRONG> GFP 1</P></TD><TD style="width:307px"><P>6</P></TD></TR><TR><TD style="width:307px"><P>Backbone</P></TD><TD style="width:307px"><P>37</P></TD></TR></TBODY></TABLE><P> </P><P> </P><P><STRONG>Ligation</STRONG></P><P> </P><TABLE border="1" cellpadding="0" cellspacing="0"><TBODY><TR><TD style="width:307px"><P>ddH2O</P></TD><TD style="width:307px"><P>To 20µl</P></TD></TR><TR><TD style="width:307px"><P>Backbone</P></TD><TD style="width:307px"><P>50ng</P></TD></TR><TR><TD style="width:307px"><P>Insert</P></TD><TD style="width:307px"><P>20ng</P></TD></TR><TR><TD style="width:307px"><P>T4 Ligase</P></TD><TD style="width:307px"><P>1µl</P></TD></TR><TR><TD style="width:307px"><P>Buffer</P></TD><TD style="width:307px"><P>2µl</P></TD></TR></TBODY></TABLE><P> </P><P><STRONG>Transformation</STRONG></P><P> </P><P> </P></DIV></DIV></DIV></DIV><DIV class="row"><DIV class="col-lg-12"><DIV class="notebookentry"><H3 class="notebookheading">▼2015-07-30  Mutation correction PCR of p415-GPD</H3><DIV class="notebooktext"><P>Description:</P><P> </P><P>In order to correct a stop-codon mutation in the CFTR-testconstruct a mutation correction PCR was made.</P><P> </P><P>Mutation correction PCR:</P><P> </P><P>Description:</P><P> </P><P>Chemicals:</P><P>0,2 µl p415-GPD CFTR-test</P><P>1 µl Primer fwd</P><P>1 µl Primer rev</P><P>2,8 µl ddH2O</P><P>5 µl Q5 Master Mix</P><P> </P><P>Program:</P><P> </P><P>98°C for 2 minutes</P><P>--------------------------</P><P>98°C for 30 seconds</P><P>Annealing temperature for 30 seconds 35 cycles</P><P>72°C for 4 minutes</P><P>--------------------------</P><P>72°C for 5 minutes</P><P>--------------------------</P><P>4°C for holding</P><P> </P><P>The reaction was set in a triplet with 3 different annealing temperatures: 72°C (2step), 70°C and 68°C</P><P> </P><P>DpnI digestion:</P><P> </P><P>Description:</P><P> </P><P>1 µl of DnpI was given on the PCR mix after the PCR reaction. The mix was incubated at 37°C for 4 hours and then inactivated at 65°C for 20 minutes.</P><P> </P><P>The DNA was given on a gel after the DpnI-digestion</P><P> </P><P>Results:</P><P> </P><P>The gel picture showed just smear.</P></DIV></DIV></DIV></DIV><DIV class="row"><DIV class="col-lg-12"><DIV class="notebookentry"><H3 class="notebookheading">▼2015-07-31  Miniprep – QIAGEN Kit (31.7.15)_concentrations</H3><DIV class="notebooktext"><TABLE border="1" cellpadding="0" cellspacing="0" style="width:100.0%"><TBODY><TR><TD style="width:33.32%"><P>Ribozyme + number</P></TD><TD style="width:33.34%"><P>Concentration [ng/µl]</P></TD><TD style="width:33.34%"><P>Volume for Sequencing(40ng) [µl]</P></TD></TR><TR><TD style="width:33.32%"><P>1</P></TD><TD style="width:33.34%"><P>/</P></TD><TD style="width:33.34%"><P>/</P></TD></TR><TR><TD style="width:33.32%"><P><STRONG>2_</STRONG>2-2</P></TD><TD style="width:33.34%"><P>99</P></TD><TD style="width:33.34%"><P>6,06</P></TD></TR><TR><TD style="width:33.32%"><P>3_3-4/5</P></TD><TD style="width:33.34%"><P>90,6</P></TD><TD style="width:33.34%"><P>6,62</P></TD></TR><TR><TD style="width:33.32%"><P>4_4-1/4-2</P></TD><TD style="width:33.34%"><P>116,9</P></TD><TD style="width:33.34%"><P>5,13</P></TD></TR><TR><TD style="width:33.32%"><P>5_5-1/2</P></TD><TD style="width:33.34%"><P>99,3</P></TD><TD style="width:33.34%"><P>6,04</P></TD></TR><TR><TD style="width:33.32%"><P>6_6-1</P></TD><TD style="width:33.34%"><P>87,3</P></TD><TD style="width:33.34%"><P>6,87</P></TD></TR><TR><TD style="width:33.32%"><P>15</P></TD><TD style="width:33.34%"><P> </P></TD><TD style="width:33.34%"><P> </P></TD></TR><TR><TD style="width:33.32%"><P>16_16-3</P></TD><TD style="width:33.34%"><P>89,8</P></TD><TD style="width:33.34%"><P>6,68</P></TD></TR></TBODY></TABLE></DIV></DIV></DIV></DIV><DIV class="row"><DIV class="col-lg-12"><DIV class="notebookentry"><H3 class="notebookheading">▼2015-07-31  Colony PCR and mini prep of Ribozymes</H3><DIV class="notebooktext"><P><STRONG>Colony PCR</STRONG></P><P> </P><P><STRONG>Miniprep – QIAGEN Kit </STRONG></P><P> </P><TABLE border="1" cellpadding="0" cellspacing="0" style="width:100.0%"><TBODY><TR><TD style="width:33.32%"><P><A name="OLE_LINK7">Ribozyme + number</A></P></TD><TD style="width:33.34%"><P>Concentration [ng/µl]</P></TD><TD style="width:33.34%"><P>Volume for Sequencing(40ng) [µl]</P></TD></TR><TR><TD style="width:33.32%"><P>1</P></TD><TD style="width:33.34%"><P>/</P></TD><TD style="width:33.34%"><P>/</P></TD></TR><TR><TD style="width:33.32%"><P><STRONG>2_</STRONG>2-2</P></TD><TD style="width:33.34%"><P>99</P></TD><TD style="width:33.34%"><P>6,06</P></TD></TR><TR><TD style="width:33.32%"><P>3_3-4/5</P></TD><TD style="width:33.34%"><P>90,6</P></TD><TD style="width:33.34%"><P>6,62</P></TD></TR><TR><TD style="width:33.32%"><P>4_4-1/4-2</P></TD><TD style="width:33.34%"><P>116,9</P></TD><TD style="width:33.34%"><P>5,13</P></TD></TR><TR><TD style="width:33.32%"><P>5_5-1/2</P></TD><TD style="width:33.34%"><P>99,3</P></TD><TD style="width:33.34%"><P>6,04</P></TD></TR><TR><TD style="width:33.32%"><P>6_6-1</P></TD><TD style="width:33.34%"><P>87,3</P></TD><TD style="width:33.34%"><P>6,87</P></TD></TR><TR><TD style="width:33.32%"><P>15</P></TD><TD style="width:33.34%"><P> </P></TD><TD style="width:33.34%"><P> </P></TD></TR><TR><TD style="width:33.32%"><P>16_16-3</P></TD><TD style="width:33.34%"><P>89,8</P></TD><TD style="width:33.34%"><P>6,68</P></TD></TR></TBODY></TABLE></DIV></DIV></DIV></DIV><DIV class="row"><DIV class="col-lg-12"><DIV class="notebookentry"><H3 class="notebookheading">▼2015-07-31  Repairing the deletion in the HDV region of the twin ribozymes with assembly PCR + Cloning</H3><DIV class="notebooktext"><P>Description:</P><P> </P><P>In order to repair a deletion in the DE inserts and ribozymes, 3 PCRs were made. In the first PCR the parts were elongated with a Primer overhang that carries the DNA fragment to fill up the deletion. The second was an assembly PCR. The third has been done to amplify the fragments.</P><P> </P><TABLE border="1" cellpadding="0" cellspacing="0" style="width:225px"><TBODY><TR><TD style="height:21px; width:67px"><P>ID #</P></TD><TD style="height:21px; width:159px"><P>Ribozyme</P></TD></TR><TR><TD style="height:21px; width:67px"><P><STRONG>1</STRONG></P></TD><TD style="height:21px; width:159px"><P>CFTR 1 T</P></TD></TR><TR><TD style="height:21px; width:67px"><P><STRONG>2</STRONG></P></TD><TD style="height:21px; width:159px"><P>CFTR 1 A</P></TD></TR><TR><TD style="height:21px; width:67px"><P><STRONG>3</STRONG></P></TD><TD style="height:21px; width:159px"><P>CFTR 1 C</P></TD></TR><TR><TD style="height:21px; width:67px"><P><STRONG>4</STRONG></P></TD><TD style="height:21px; width:159px"><P>CFTR 2 T</P></TD></TR><TR><TD style="height:21px; width:67px"><P><STRONG>5</STRONG></P></TD><TD style="height:21px; width:159px"><P>CFTR 2 A</P></TD></TR><TR><TD style="height:21px; width:67px"><P><STRONG>6</STRONG></P></TD><TD style="height:21px; width:159px"><P>CFTR 2 C</P></TD></TR><TR><TD style="height:21px; width:67px"><P><STRONG>7</STRONG></P></TD><TD style="height:21px; width:159px"><P>CFTR 1 DE T</P></TD></TR><TR><TD style="height:21px; width:67px"><P><STRONG>8</STRONG></P></TD><TD style="height:21px; width:159px"><P>CFTR 1 DE A</P></TD></TR><TR><TD style="height:21px; width:67px"><P><STRONG>9</STRONG></P></TD><TD style="height:21px; width:159px"><P>CFTR 1 DE C</P></TD></TR><TR><TD style="height:21px; width:67px"><P><STRONG>10</STRONG></P></TD><TD style="height:21px; width:159px"><P>CFTR 2 DE T</P></TD></TR><TR><TD style="height:21px; width:67px"><P><STRONG>11</STRONG></P></TD><TD style="height:21px; width:159px"><P>CFTR 2 DE A</P></TD></TR><TR><TD style="height:21px; width:67px"><P><STRONG>12</STRONG></P></TD><TD style="height:21px; width:159px"><P>CFTR 2 DE C</P></TD></TR><TR><TD style="height:21px; width:67px"><P><STRONG>13</STRONG></P></TD><TD style="height:21px; width:159px"><P>GFP 1 DE</P></TD></TR><TR><TD style="height:21px; width:67px"><P><STRONG>14</STRONG></P></TD><TD style="height:21px; width:159px"><P>GFP 2 DE</P></TD></TR><TR><TD style="height:21px; width:67px"><P><STRONG>15</STRONG></P></TD><TD style="height:21px; width:159px"><P>GFP 1</P></TD></TR><TR><TD style="height:21px; width:67px"><P><STRONG>16</STRONG></P></TD><TD style="height:21px; width:159px"><P>GFP 2</P></TD></TR></TBODY></TABLE><P> </P><P> </P><P>PCR: Division and Elongation</P><P> </P><P>Description</P><P> </P><P>PCR-Mix:</P><P>5 µl Primer fwd</P><P>5 µl Primer rev</P><P>0,5 µl Template</P><P>14,5 µl ddH2O</P><P>25 µl Q5-MasterMix</P><P> </P><P>Program:</P><P> </P><P>98°C for 30 seconds</P><P>------------------------------------------------</P><P>98°C for 10 seconds</P><P>68°C for 10 seconds 35 Cycles</P><P>72°C for 15 seconds</P><P>------------------------------------------------</P><P>72°C for 30 seconds</P><P>------------------------------------------------</P><P>4°C for holding</P><P> </P><P>The PCR to make two parts is made in two different tubes, one for the first and one for the second part.</P><P> </P><P>First part:</P><P> </P><TABLE border="1" cellpadding="0" cellspacing="0"><TBODY><TR><TD style="width:201px"><P>Ribozyme/Insert</P></TD><TD style="width:201px"><P>Fwd Primer</P></TD><TD style="width:201px"><P>Rev Primer</P></TD></TR><TR><TD style="width:201px"><P>CFTR 1 DE T</P></TD><TD style="width:201px"><P>DH_39</P></TD><TD style="width:201px"><P>MJ_15</P></TD></TR><TR><TD style="width:201px"><P>CFTR 1 DE A</P></TD><TD style="width:201px"><P>DH_39</P></TD><TD style="width:201px"><P>MJ_15</P></TD></TR><TR><TD style="width:201px"><P>CFTR 1 DE C</P></TD><TD style="width:201px"><P>DH_39</P></TD><TD style="width:201px"><P>MJ_15</P></TD></TR><TR><TD style="width:201px"><P>CFTR 2 DE T</P></TD><TD style="width:201px"><P>DH_40</P></TD><TD style="width:201px"><P>MJ_15</P></TD></TR><TR><TD style="width:201px"><P>CFTR 2 DE A</P></TD><TD style="width:201px"><P>DH_40</P></TD><TD style="width:201px"><P>MJ_15</P></TD></TR><TR><TD style="width:201px"><P>CFTR 2 DE C</P></TD><TD style="width:201px"><P>DH_40</P></TD><TD style="width:201px"><P>MJ_15</P></TD></TR><TR><TD style="width:201px"><P>GFP 1 DE</P></TD><TD style="width:201px"><P>DH_41</P></TD><TD style="width:201px"><P>MJ_18</P></TD></TR><TR><TD style="width:201px"><P>GFP 2 DE</P></TD><TD style="width:201px"><P>DH_41</P></TD><TD style="width:201px"><P>MJ_18</P></TD></TR><TR><TD style="width:201px"><P>Insert CFTR 1</P></TD><TD style="width:201px"><P>DH_18a</P></TD><TD style="width:201px"><P>MJ_17</P></TD></TR><TR><TD style="width:201px"><P>Insert CFTR 2</P></TD><TD style="width:201px"><P>DH_18a</P></TD><TD style="width:201px"><P>MJ_16</P></TD></TR><TR><TD style="width:201px"><P>Insert GFP 1</P></TD><TD style="width:201px"><P>DH_20a</P></TD><TD style="width:201px"><P>MJ_20</P></TD></TR><TR><TD style="width:201px"><P>Insert GFP 2</P></TD><TD style="width:201px"><P>DH_22a</P></TD><TD style="width:201px"><P>MJ_19</P></TD></TR></TBODY></TABLE><P> </P><P>Second Part:</P><P> </P><TABLE border="1" cellpadding="0" cellspacing="0"><TBODY><TR><TD style="width:151px"><P>Riobzyme/Insert</P></TD><TD style="width:151px"><P>Fwd-Primer</P></TD><TD style="width:151px"><P>Rev-Primer</P></TD><TD style="width:151px"><P>Tubes</P></TD></TR><TR><TD style="width:151px"><P>CFTR 1 DE T</P></TD><TD style="width:151px"><P>MJ_21</P></TD><TD style="width:151px"><P>DH_48</P></TD><TD style="width:151px"><P>4</P></TD></TR><TR><TD style="width:151px"><P>Insert CFTR 1</P></TD><TD style="width:151px"><P>MJ_21</P></TD><TD style="width:151px"><P>DH_27a</P></TD><TD style="width:151px"><P>2</P></TD></TR></TBODY></TABLE><P> </P><P> </P><P>Assembly PCR</P><P> </P><P>Description:</P><P> </P><P>PCR-Mix:</P><P>1:1 Molar ratio of part 1 and part 2 (from the first PCR)</P><P>25 µl Q5-MasterMix</P><P>ad 40 µl ddH2O</P><P> </P><P>PCR-program:</P><P> </P><P>98°C for 1 minute</P><P>--------------------------------------------</P><P>98°C for 10 seconds</P><P>70°C with a ramp rate of 1°C per second for 30 seconds</P><P>72°C for 30 seconds</P><P>Repeat 3 Times</P><P>--------------------------------------------</P><P>72°C for 30 seconds</P><P>-------------------------------------------</P><P>4°C for holding</P><P> </P><P> </P><TABLE border="1" cellpadding="0" cellspacing="0"><TBODY><TR><TD style="width:47px"><P>Frag-ment 1</P></TD><TD style="width:66px"><P>Length</P></TD><TD colspan="2" style="width:57px"><P>ng/µl</P></TD><TD style="width:47px"><P>nmol</P></TD><TD style="width:47px"><P>Frag-ment 2</P></TD><TD style="width:57px"><P>Length</P></TD><TD style="width:57px"><P>ng/µl</P></TD><TD style="width:47px"><P>nmol</P></TD><TD style="width:47px"><P>Frag-ment 2 Ratio</P></TD><TD style="width:49px"><P>F1 µl</P></TD><TD style="width:36px"><P>F2 µl</P></TD><TD style="width:47px"><P>ddH2O µl</P></TD></TR><TR><TD style="width:47px"><P>2T</P></TD><TD style="width:66px"><P>226</P></TD><TD style="width:55px"><P>142</P></TD><TD colspan="2" style="width:49px"><P>1,017</P></TD><TD style="width:47px"><P>Rib</P></TD><TD style="width:57px"><P>80</P></TD><TD style="width:57px"><P>327,2</P></TD><TD style="width:47px"><P>6,62</P></TD><TD style="width:47px"><P>0,15</P></TD><TD style="width:49px"><P>1,5</P></TD><TD style="width:36px"><P>0,2</P></TD><TD style="width:47px"><P>23,3</P></TD></TR><TR><TD style="width:47px"><P>2A</P></TD><TD style="width:66px"><P>226</P></TD><TD style="width:55px"><P>184,9</P></TD><TD colspan="2" style="width:49px"><P>1,324</P></TD><TD style="width:47px"><P>Rib</P></TD><TD style="width:57px"><P>80</P></TD><TD style="width:57px"><P>327,2</P></TD><TD style="width:47px"><P>6,62</P></TD><TD style="width:47px"><P>0,2</P></TD><TD style="width:49px"><P>1,5</P></TD><TD style="width:36px"><P>0,3</P></TD><TD style="width:47px"><P>23,2</P></TD></TR><TR><TD style="width:47px"><P>2C</P></TD><TD style="width:66px"><P>226</P></TD><TD style="width:55px"><P>117,5</P></TD><TD colspan="2" style="width:49px"><P>0,841</P></TD><TD style="width:47px"><P>Rib</P></TD><TD style="width:57px"><P>80</P></TD><TD style="width:57px"><P>327,2</P></TD><TD style="width:47px"><P>6,62</P></TD><TD style="width:47px"><P>0,12</P></TD><TD style="width:49px"><P>1,5</P></TD><TD style="width:36px"><P>0,2</P></TD><TD style="width:47px"><P>23,3</P></TD></TR><TR><TD style="width:47px"><P>G1</P></TD><TD style="width:66px"><P>227</P></TD><TD style="width:55px"><P>199,3</P></TD><TD colspan="2" style="width:49px"><P>1,42</P></TD><TD style="width:47px"><P>Rib</P></TD><TD style="width:57px"><P>80</P></TD><TD style="width:57px"><P>327,2</P></TD><TD style="width:47px"><P>6,62</P></TD><TD style="width:47px"><P>0,2</P></TD><TD style="width:49px"><P>1,5</P></TD><TD style="width:36px"><P>0,3</P></TD><TD style="width:47px"><P>23,2</P></TD></TR><TR><TD style="width:47px"><P>G2</P></TD><TD style="width:66px"><P>243</P></TD><TD style="width:55px"><P>166,5</P></TD><TD colspan="2" style="width:49px"><P>1,109</P></TD><TD style="width:47px"><P>Rib</P></TD><TD style="width:57px"><P>80</P></TD><TD style="width:57px"><P>327,2</P></TD><TD style="width:47px"><P>6,62</P></TD><TD style="width:47px"><P>0,16</P></TD><TD style="width:49px"><P>1,5</P></TD><TD style="width:36px"><P>0,3</P></TD><TD style="width:47px"><P>23,2</P></TD></TR><TR><TD style="width:47px"><P>IC1</P></TD><TD style="width:66px"><P>109</P></TD><TD style="width:55px"><P>120,2</P></TD><TD colspan="2" style="width:49px"><P>1,784</P></TD><TD style="width:47px"><P>Ins</P></TD><TD style="width:57px"><P>87</P></TD><TD style="width:57px"><P>205,6</P></TD><TD style="width:47px"><P>6,62</P></TD><TD style="width:47px"><P>0,46</P></TD><TD style="width:49px"><P>1,5</P></TD><TD style="width:36px"><P>0,7</P></TD><TD style="width:47px"><P>22,8</P></TD></TR><TR><TD style="width:47px"><P>IC2</P></TD><TD style="width:66px"><P>118</P></TD><TD style="width:55px"><P>128,3</P></TD><TD colspan="2" style="width:49px"><P>1,759</P></TD><TD style="width:47px"><P>Ins</P></TD><TD style="width:57px"><P>87</P></TD><TD style="width:57px"><P>205,6</P></TD><TD style="width:47px"><P>6,62</P></TD><TD style="width:47px"><P>0,46</P></TD><TD style="width:49px"><P>1,5</P></TD><TD style="width:36px"><P>0,7</P></TD><TD style="width:47px"><P>22,8</P></TD></TR><TR><TD style="width:47px"><P>1T</P></TD><TD style="width:66px"><P>216</P></TD><TD style="width:55px"><P>150,2</P></TD><TD colspan="2" style="width:49px"><P>1,125</P></TD><TD style="width:47px"><P>Rib</P></TD><TD style="width:57px"><P>80</P></TD><TD style="width:57px"><P>327,2</P></TD><TD style="width:47px"><P>6,62</P></TD><TD style="width:47px"><P>0,17</P></TD><TD style="width:49px"><P>1,5</P></TD><TD style="width:36px"><P>0,3</P></TD><TD style="width:47px"><P>23,2</P></TD></TR><TR><TD style="width:47px"><P>1A</P></TD><TD style="width:66px"><P>216</P></TD><TD style="width:55px"><P>121</P></TD><TD colspan="2" style="width:49px"><P>0,906</P></TD><TD style="width:47px"><P>Rib</P></TD><TD style="width:57px"><P>80</P></TD><TD style="width:57px"><P>327,2</P></TD><TD style="width:47px"><P>6,62</P></TD><TD style="width:47px"><P>0,14</P></TD><TD style="width:49px"><P>1,5</P></TD><TD style="width:36px"><P>0,2</P></TD><TD style="width:47px"><P>23,3</P></TD></TR><TR><TD style="width:47px"><P>1C</P></TD><TD style="width:66px"><P>216</P></TD><TD style="width:55px"><P>98,5</P></TD><TD colspan="2" style="width:49px"><P>0,737</P></TD><TD style="width:47px"><P>Rib</P></TD><TD style="width:57px"><P>80</P></TD><TD style="width:57px"><P>327,2</P></TD><TD style="width:47px"><P>6,62</P></TD><TD style="width:47px"><P>0,11</P></TD><TD style="width:49px"><P>1,5</P></TD><TD style="width:36px"><P>0,2</P></TD><TD style="width:47px"><P>23,3</P></TD></TR><TR><TD style="width:47px"><P>IG1</P></TD><TD style="width:66px"><P>114</P></TD><TD style="width:55px"><P>145,7</P></TD><TD colspan="2" style="width:49px"><P>2,067</P></TD><TD style="width:47px"><P>Ins</P></TD><TD style="width:57px"><P>87</P></TD><TD style="width:57px"><P>205,6</P></TD><TD style="width:47px"><P>6,62</P></TD><TD style="width:47px"><P>0,54</P></TD><TD style="width:49px"><P>1,5</P></TD><TD style="width:36px"><P>0,8</P></TD><TD style="width:47px"><P>22,7</P></TD></TR><TR><TD style="width:47px"><P>IG2</P></TD><TD style="width:66px"><P>115</P></TD><TD style="width:55px"><P>151,3</P></TD><TD colspan="2" style="width:49px"><P>2,128</P></TD><TD style="width:47px"><P>Ins</P></TD><TD style="width:57px"><P>87</P></TD><TD style="width:57px"><P>205,6</P></TD><TD style="width:47px"><P>6,62</P></TD><TD style="width:47px"><P>0,56</P></TD><TD style="width:49px"><P>1,5</P></TD><TD style="width:36px"><P>0,8</P></TD><TD style="width:47px"><P>22,7</P></TD></TR><TR><TD> </TD><TD> </TD><TD> </TD><TD> </TD><TD> </TD><TD> </TD><TD> </TD><TD> </TD><TD> </TD><TD> </TD><TD> </TD><TD> </TD><TD> </TD></TR></TBODY></TABLE><P> </P><P> </P><P>PCR: Amplification</P><P> </P><P>Description:</P><P> </P><P>PCR-Mix:</P><P> </P><P>5 µl Primer fwd and rev are given into the assembly PCR mix after the assembly PCR</P><P> </P><P> </P><P>Program:</P><P> </P><P>98°C for 60 seconds</P><P>----------------------------------</P><P>98°C for 10 seconds</P><P>68°C for 15 seconds 35 Cycles</P><P>72°C for 20 seconds</P><P>------------------------------------</P><P>72°C for 30 seconds</P><P>------------------------------------</P><P>4°C for holding</P><P> </P><P>Digestion:</P><P> </P><P>Description:</P><P> </P><P>Steps:</P><P> </P><OL><LI>Set up reaction according to protocol:</LI></OL><P>                ddH2O for a final volume of 20 µl</P><P>                2 µl of 10x Reaction Buffer (e.g. NEB CutSmart)</P><P>                0.5 µl of selected Enzyme(s)</P><P>                Ca. 1 µl of mini prep DNA (Range 200-1000 ng)</P><P> </P><OL><LI>Incubate at 37°C for 60'</LI></OL><P> </P><P>Digestion of the ribozymes was made with BamHI and BmtI.</P><P> </P><P>Ligation</P><P> </P><P>Description:</P><P> </P><P>Materials and chemicals:</P><P>2 µl 10x T4 Ligase Buffer: thawed and resuspended at room temperature</P><P>1 parts Vector DNA (mol not l)</P><P>3 part Insert DNA (mol not l)</P><P>20 µl Nuclease free water</P><P>1 µl T4 DNA Ligase</P><P> </P><P>Steps:</P><P> </P><OL><LI>Set up the reaction in a microcentrifuge tube on ice. T4 DNA Ligase should be added last. The molar ratio of vector to insert should be 1:3.</LI></OL><P> </P><OL><LI>Gently mix the reaction by pipetting up and down and microfuge briefly.</LI></OL><P> </P><OL><LI>For cohesive (sticky) ends, incubate at 16°C overnight or room temperature for 10 minutes.</LI></OL><P> </P><OL><LI>For blunt ends or single base overhangs, incubate at 16°C overnight or room temperature for 2 hours. Alternatively a high concentration of T4 Ligase can be used in a 10 minute ligation.</LI></OL><P> </P><OL><LI>Heat inactivate at 80°C for 10 minutes</LI></OL><P> </P><OL><LI>Chill on ice and transform 1-5 µl of the reaction into 50 µl of competent cells.</LI></OL><P> </P><P>The ribozymes were ligated into a cutted and dephoshorylated pSB1C3 with MCS and pcat.</P><P> </P><P>KCM Transformation:</P><P> </P><P>Description:</P><P> </P><P>Steps:</P><P> </P><OL><LI>Take 50µl chemical competent E. coli from -80 freezer and thaw on ice</LI></OL><P> </P><OL><LI>Add (as master mix):</LI></OL><P>                2,5µl DNA</P><P>                10µl KCM 5x</P><P>                37,5µl H2O</P><P> </P><OL><LI>Incubate on ice for 30 minutes</LI></OL><P> </P><OL><LI>Heat shock at 42°C for 1 minute</LI></OL><P> </P><OL><LI>Incubate on ice for 2 minutes</LI></OL><P> </P><OL><LI>Add 900 µl of LB or 2x YT Medium</LI></OL><P> </P><OL><LI>Incubate on 37°C for 60min</LI></OL><P> </P><OL><LI>Centrifuge 5min at 1000g</LI></OL><P> </P><OL><LI>Take 900µl of supernatant and throw away</LI></OL><P> </P><OL><LI>Resuspend pellet in remaining media</LI></OL><P> </P><OL><LI>Plate out on agar with antibiotics (1:1 / 1:10)</LI></OL><P> </P><P>Colony PCR:</P><P> </P><P>Description:</P><P> </P><P>Materials and Chemicals:</P><P>PCR-tubes</P><P>Forward primer</P><P>Reverse primer</P><P>Masermix (dNTPs, Polymerase, buffer)</P><P>Water</P><P>Thermocycler</P><P> </P><P>Endvolume: 10 µl</P><P> </P><P>Steps:</P><P> </P><OL><LI>Pick colonies from plates. Solute one colony in about 20 µl of water.</LI></OL><P> </P><OL><LI>Give the colonies into 10 µl colony PCR solution with OneTaq Mastermix (which should be diluted to 1x in the end (e.g. you need 5 µl of 2x mastermix for 10 µl)) and primer (between 0,1 and 1 µl)</LI></OL><P> </P><P>Use the Thermocycler with an appropriate PCR program for at least 25 cycles</P><P> </P><P>Results:</P><P> </P><P>The colony PCR showed at least one positive clone for every ribozyme</P><P> </P><DIV class="col-lg-12"><DIV class="imagewrapper"><DIV class="imagewrappercaption"><STRONG>
         Figure 6: Colony PCR of the cloned DE ribozymes
         </STRONG><P class="basictext">
            Positive Colonies: 7-2, 8-1, 8-2, 8-3, 9-1, 9-2, 9-5, 10-1, 10-2, 10-3, 10-5, 11-1, 12-1, 12-2
         </P></DIV></DIV></DIV><P> </P><DIV class="col-lg-12"><DIV class="imagewrapper"><DIV class="imagewrappercaption"><STRONG>
         Figure 7: Colony PCR of the cloned De ribozymes 
         </STRONG><P class="basictext">
            Positive Colonies: 12-4, 13-1, 14-1, 14-3, 15-2, 15-3
         </P></DIV></DIV></DIV><P> </P></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV><DIV class="container"><DIV class="content"><DIV class="row"><DIV class="col-lg-12"><DIV class="panel panel-default"><DIV class="panel-heading"><H3 class="basicheader">week number 30</H3></DIV><DIV class="panel-body"><DIV class="row"><DIV class="col-lg-12"><DIV class="notebookentry"><H3 class="notebookheading">▼2015-07-24  Yeast transformation with p415-GPD with CFTR-testconstruct without mutation correction</H3><DIV class="notebooktext"><P>Description</P><P> </P><P>Yeast was transformated with p415-GPD with the CFTR-testconstruct from a miniprep of an E.coli culture. Transformed cells should turn red.</P><P> </P><P>Procedures:</P><P> </P><P>Yeast transformation 24.07</P><P> </P><P>Description:</P><P> </P><P>Materials and chemicals:</P><P>10 µl of cells for transformation with a plasmid, 50 µl of cells for transformation with a PCR product</P><P>2 µl of plasmid DNA per 10 µl of cells (200 ng of DNA)</P><P>6 equivalents of PEG to the yeast cells</P><P>1/9 of DMSO to the volume of plasmids, yeast cells and PEG</P><P>100 - 200 µl of liquid medium</P><P> </P><P>Steps:</P><P> </P><OL><LI>Give the plasmid DNA into a sterile 1.5 ml tube (2 µl per 10 µl of cells. Add the thawed competent cells.</LI></OL><P> </P><OL><LI>Mix the suspension well, then add the PEG.</LI></OL><P> </P><OL><LI>Incubate for 30 mins at room temperature while mixing (the cells can be incubated up to 2 hours).</LI></OL><P> </P><OL><LI>Add the DMSO to get a final concentration of 10% DMSO.</LI></OL><P> </P><OL><LI>Place the yeast in a 42°C water bath for 5-20 minutes. NO Thermomixer.</LI></OL><P> </P><OL><LI>Centrifuge cells for 2-3 minutes at 2000 rpm/500g</LI></OL><P> </P><OL><LI>Discard the supernatant and resuspend the yeast in the liqiud YPD medium</LI></OL><P> </P><P>The transformation was repeated on 27.07.2015</P><P> </P><P>Results:</P><P> </P><P>Transformation 1:</P><P>The first transformation resulted in cultures with an ununsual behaivior, maybe the plates were contaminated with another organism. The cells didn't turned red</P><P> </P><P>Transformation 2:</P><P>A test run on a FACS showed that the transformed cells are functional and express the CFTR-testconstruct</P></DIV></DIV></DIV></DIV><DIV class="row"><DIV class="col-lg-12"><DIV class="notebookentry"><H3 class="notebookheading">▼2015-07-25  Mutation correction PCR p415-GPD with CFTR-testconstruct</H3><DIV class="notebooktext"><P>Description</P><P> </P><P>In order to correct the mutation in the mCherry part of the CFTR-testcontruct, a mutation PCR was made.</P><P> </P><P>Procedures:</P><P> </P><P>Mutation correction PCR 1:</P><P> </P><P>Description</P><P> </P><P>Chemicals:</P><P>0,1 µl p415-GPD with CFTR-testconstruct (about 650 ng/µl)</P><P>1 µl Primer fwd</P><P>1 µl Primer rev</P><P>2,9 µl ddH2O</P><P>5 µl Q5</P><P> </P><P>Program:</P><P> </P><P>2-step PCR:</P><P> </P><P>98°C</P><P>-----------------------</P><P>98°C</P><P>72°C</P><P>------------------------</P><P>72°C</P><P>------------------------</P><P>4°C for holding</P><P> </P><P>Mutation correction PCR 2</P><P> </P><P>Description:</P><P> </P><P>Chemicals:</P><P>0,1 µl p415-GPD with CFTR-testcontruct</P><P>1 µl Primer fwd</P><P>1 µl Primer rev</P><P>2,9 µl ddH2O</P><P>5 µl Q5</P><P> </P><P>Program:</P><P> </P><P>98°C for 1 minute</P><P>------------------------</P><P>98°C for 20 s</P><P>Anneal for 20 s</P><P>72°C for 4 minutes</P><P>Repeat cycle 35 times</P><P>-------------------------</P><P>72°C for 5 minutes</P><P>-------------------------</P><P>4°C for holding</P><P> </P><P>Tried annealing temperatures:</P><P>72°C (2-step-PCR)</P><P>70,5°C</P><P>68,8°C</P><P>65,6°C</P><P>64°C</P><P> </P><P>Annealing temperatures were made by gradient</P><P> </P><P>Results:</P><P> </P><P>Mutation PCR 1:</P><P>The Gel showed there was just smear</P><P> </P><P>Mutation PCR 2:</P><P>Every Annealing temperature resulted in smear and no detectable fragment</P><P> </P><DIV class="col-lg-12"><DIV class="imagewrapper"><DIV class="imagewrappercaption"><STRONG>
                Figure 5: The second mutation PCR of the CFTR-testconstruct
            </STRONG></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV><DIV class="container"><DIV class="content"><DIV class="row"><DIV class="col-lg-12"><DIV class="panel panel-default"><DIV class="panel-heading"><H3 class="basicheader">week number 29</H3></DIV><DIV class="panel-body"><DIV class="row"><DIV class="col-lg-12"><DIV class="notebookentry"><H3 class="notebookheading">▼2015-07-14  Inserting twin ribozymes into pSB1C3 + MCS + pcat</H3><DIV class="notebooktext"><P>Description:</P><P> </P><P>After validating the MCS + pcat fragment, the ribozymes were cloned into pSB1C3</P><P> </P><P>Procedures:</P><P> </P><P>2-step digest:</P><P> </P><P>Description:</P><P> </P><P>Steps:</P><P> </P><OL><LI>Set up reaction according to protocol:</LI></OL><P>                ddH2O for a final volume of 20 µl</P><P>                2 µl of 10x Reaction Buffer (e.g. NEB CutSmart)</P><P>                0.5 µl of selected Enzyme(s)</P><P>                ca. 1 µl of mini prep DNA (Range 200-1000 ng)</P><P> </P><OL><LI>Incubate at 37°C for 60'</LI></OL><P> </P><OL><LI>Load on gel (add loading dye first)</LI></OL><P> </P><P>Two digestion steps were made:</P><P> </P><P>Step 1:</P><P> </P><OL><LI>Set up following mix:</LI></OL><P>                25 µl pSB1C3-MCS-pcat</P><P>                1 µl BmtI</P><P>                5 µl CutSmart 10x Buffer</P><P>                19 µl ddH2O</P><P> </P><OL><LI>Digestion for 1 hour at 37°C</LI></OL><P> </P><OL><LI>PCR-purification with 2 step elution in 35 µl water</LI></OL><P> </P><P> </P><P>Step 2:</P><P> </P><OL><LI>Set up following mix:</LI></OL><P>                35 µl from the PCR-purification</P><P>                10 µl BamHF</P><P>                5 µl CutSmart</P><P> </P><OL><LI>Digestion for 4 hours at 37°C</LI></OL><P> </P><OL><LI>PCR purification with 2 step elution in 30 µl Elution Buffer</LI></OL><P> </P><P>    Ligation:</P><P> </P><P>Description:</P><P> </P><P>Materials and chemicals:</P><P>2 µl 10x T4 Ligase Buffer: thawed and resuspended at room temperature</P><P>1 parts Vector DNA (mol not l)</P><P>3 part Insert DNA (mol not l)</P><P>20 µl Nuclease free water</P><P>1 µl T4 DNA Ligase</P><P> </P><P>Steps:</P><P> </P><OL><LI>Set up the reaction in a microcentrifuge tube on ice. T4 DNA Ligase should be added last. The molar ratio of vector to insert should be 1:3.</LI></OL><P> </P><OL><LI>Gently mix the reaction by pipetting up and down and microfuge briefly.</LI></OL><P> </P><OL><LI>For cohesive (sticky) ends, incubate at 16°C overnight or room temperature for 10 minutes.</LI></OL><P> </P><OL><LI>For blunt ends or single base overhangs, incubate at 16°C overnight or room temperature for 2 hours. Alternatively a high concentration of T4 Ligase can be used in a 10 minute ligation.</LI></OL><P> </P><OL><LI>Heat inactivate at 80°C for 10 minutes</LI></OL><P> </P><OL><LI>Chill on ice and transform 1-5 µl of the reaction into 50 µl of competent cells.</LI></OL><P> </P><P>We made one ligation for every ribozyme and one as a religation control (17 in total)</P><P> </P><P>Ligation mixes:</P><P> </P><P>Not DE-ribozymes (without Hammerhead and Hepatitis Delta Virus ribozyme on the ends):</P><P>1 µl Backbone (35 ng)</P><P>2,5 µl Insert (12,5 ng)</P><P>1 µl T4-Ligase</P><P>2 µl T4-Ligation Buffer 10x</P><P>13,5 µl ddH2O</P><P> </P><P>DE-ribozymes (with HH and HDV):</P><P>1 µl Backbone</P><P>4 µl Insert</P><P>1 µl T4-Ligase</P><P>2 µl T4-Ligation Buffer 10x</P><P>12 µl ddH2O</P><P> </P><P>Religation Control:</P><P>1 µl Backbone</P><P>1 µl T4-Ligase</P><P>2 µl T4-Ligation Buffer 10x</P><P>16 µl ddH2O</P><P> </P><P>KCM Transformation:</P><P> </P><P>Description:</P><P> </P><P>Steps:</P><P> </P><OL><LI>Take 50µl chemical competent E. coli from -80 freezer and thaw on ice</LI></OL><P> </P><OL><LI>Add (as master mix):</LI></OL><P>                2,5µl DNA</P><P>                10µl KCM 5x</P><P>                37,5µl H2O</P><P> </P><OL><LI>Incubate on ice for 30 minutes</LI></OL><P> </P><OL><LI>Heat shock at 42°C for 1 minute</LI></OL><P> </P><OL><LI>Incubate on ice for 2 minutes</LI></OL><P> </P><OL><LI>Add 900 µl of LB or 2x YT Medium</LI></OL><P> </P><OL><LI>Incubate on 37°C for 60min</LI></OL><P> </P><OL><LI>Centrifuge 5min at 1000g</LI></OL><P> </P><OL><LI>Take 900µl of supernatant and throw away</LI></OL><P> </P><OL><LI>Resuspend pellet in remaining media</LI></OL><P> </P><OL><LI>Plate out on agar with antibiotics (1:1 / 1:10)</LI></OL><P> </P><P>We made 1 transformation for every ligation (17 in total), one for each ribozyme and one as a religation control</P><P> </P><P>Colony PCR:</P><P> </P><P>Description:</P><P> </P><P>Materials and Chemicals:</P><P>PCR-tubes</P><P>Forward primer</P><P>Reverse primer</P><P>Masermix (dNTPs, Polymerase, buffer)</P><P>Water</P><P>Thermocycler</P><P> </P><P>Endvolume: 10 µl</P><P> </P><P>Steps:</P><P> </P><OL><LI>Pick colonies from plates. Solute one colony in about 20 µl of water.</LI></OL><P> </P><OL><LI>Give the colonies into 10 µl colony PCR solution with OneTaq Mastermix (which should be diluted to 1x in the end (e.g. you need 5 µl of 2x mastermix for 10 µl)) and primer (between 0,1 and 1 µl)</LI></OL><P> </P><OL><LI>Use the Thermocycler with an appropriate PCR program for at least 25 cycles</LI></OL><P> </P><P>We picked 12 colonies from every ribozyme plate and got no satisfying results</P></DIV></DIV></DIV></DIV><DIV class="row"><DIV class="col-lg-12"><DIV class="notebookentry"><H3 class="notebookheading">▼2015-07-15  Digestion of pSB1C3-MCS-pcat with BmtI and BamHI 2 step</H3><DIV class="notebooktext"><P>Description:</P><P> </P><P>We made a 2 step digestion with BmtI and BamHI for 1 and 4 hours to proper cut the pSB1C3 in the MCS region, because the BmtI and BamHI cutting sites are next to each other and the exonuclease activity of the BamHI needs to be triggered.    </P><P> </P><P>Procedures:</P><P> </P><P>2 step ligation:</P><P> </P><P>Description:</P><P> </P><P>First digetion with BmtI:</P><P> </P><P>Steps:</P><P> </P><OL><LI>Make a mix of following chemicals in a 1,5 ml centrifuge tube:</LI></OL><P>                25 µl pSB1C3-MCS-pcat</P><P>                1 µl BmtI</P><P>                5 µl CutSmart 10x Buffer</P><P>                19 µl ddH2O</P><OL><LI>Incubate for 1 hour at 37°C</LI><LI>Make a PCR purification</LI></OL><P> </P><P> </P><P>Second digestion with BamHI:</P><P> </P><P>Steps:</P><P> </P><OL><LI>Make a mix of following chemicals:</LI></OL><P>                35 µl of DNA solution from the prior PCR purification</P><P>                10 µl BamHI</P><P>                5 µl CutSmart 10x Buffer</P><OL><LI>Incubate for 4 hours at 37°C</LI><LI>Make a PCR purification</LI></OL><P> </P><P>Results:</P><P> </P><P>Nanodrop result: 24,6 ng/µl</P><P> </P></DIV></DIV></DIV></DIV><DIV class="row"><DIV class="col-lg-12"><DIV class="notebookentry"><H3 class="notebookheading">▼2015-07-15  Digestion of the ribozyme fragments</H3><DIV class="notebooktext"><P>Description:</P><P>The ribozymes were cut for the insertion into pSB1C3 with MCS/pcat.</P><P> </P><P>Procedures:</P><P> </P><P>Ribozyme digestion:</P><P> </P><P>Description:</P><P> </P><P>Steps</P><P> </P><OL><LI>Chemicals (per tube):</LI></OL><P>                6 µl DNA solution (~60 ng)</P><P>                1 µl Cutsmart</P><P>                0,1 µl BamHI</P><P>                01, µl BmtI</P><P>                2,8 µl ddH2O</P><P> </P><OL><LI>Incubation for 1 hour at 37°C</LI></OL><P> </P><OL><LI>PCR-purificaiton, with 2 step-elution</LI></OL><P> </P><OL><LI>Lyophylisation</LI></OL><P> </P><OL><LI>Solute dried DNA in 10 µl water</LI></OL><P> </P><P>Results:</P><P> </P><P>Nanodrop results:</P><P> </P><TABLE border="1" cellpadding="0" cellspacing="0"><TBODY><TR><TD style="width:85px"><P><STRONG>Number</STRONG></P></TD><TD style="width:284px"><P><STRONG>Ribozyme</STRONG></P></TD><TD style="width:236px"><P><STRONG>Nanodrop-Result</STRONG></P></TD></TR><TR><TD style="width:85px"><P><STRONG>1</STRONG></P></TD><TD style="width:284px"><P>CFTR 1 T</P></TD><TD style="width:236px"><P>19,4</P></TD></TR><TR><TD style="width:85px"><P><STRONG>2</STRONG></P></TD><TD style="width:284px"><P>CFTR 1 A</P></TD><TD style="width:236px"><P>77</P></TD></TR><TR><TD style="width:85px"><P><STRONG>3</STRONG></P></TD><TD style="width:284px"><P>CFTR 1 C</P></TD><TD style="width:236px"><P>94,8</P></TD></TR><TR><TD style="width:85px"><P><STRONG>4</STRONG></P></TD><TD style="width:284px"><P>CFTR 2 A</P></TD><TD style="width:236px"><P>97</P></TD></TR><TR><TD style="width:85px"><P><STRONG>5</STRONG></P></TD><TD style="width:284px"><P>CFTR 2 C</P></TD><TD style="width:236px"><P>91,1</P></TD></TR><TR><TD style="width:85px"><P><STRONG>6</STRONG></P></TD><TD style="width:284px"><P>GFP 1</P></TD><TD style="width:236px"><P>77,3</P></TD></TR><TR><TD style="width:85px"><P><STRONG>7</STRONG></P></TD><TD style="width:284px"><P>GFP 1 DE</P></TD><TD style="width:236px"><P>38,6</P></TD></TR><TR><TD style="width:85px"><P><STRONG>8</STRONG></P></TD><TD style="width:284px"><P>CFTR 2 DE C</P></TD><TD style="width:236px"><P>55,3</P></TD></TR><TR><TD style="width:85px"><P><STRONG>9</STRONG></P></TD><TD style="width:284px"><P>CFTR 1 DE T</P></TD><TD style="width:236px"><P>72,7</P></TD></TR><TR><TD style="width:85px"><P><STRONG>10</STRONG></P></TD><TD style="width:284px"><P>CFTR 1 DE A</P></TD><TD style="width:236px"><P>60,6</P></TD></TR><TR><TD style="width:85px"><P><STRONG>11</STRONG></P></TD><TD style="width:284px"><P>GFP 2 DE</P></TD><TD style="width:236px"><P>65,6</P></TD></TR><TR><TD style="width:85px"><P><STRONG>12</STRONG></P></TD><TD style="width:284px"><P>CFTR 1 DE C</P></TD><TD style="width:236px"><P>56,3</P></TD></TR><TR><TD style="width:85px"><P><STRONG>13</STRONG></P></TD><TD style="width:284px"><P>CFTR 2 DE T</P></TD><TD style="width:236px"><P>100,2</P></TD></TR><TR><TD style="width:85px"><P><STRONG>14</STRONG></P></TD><TD style="width:284px"><P>CFTR 1 DE A</P></TD><TD style="width:236px"><P>29,3</P></TD></TR><TR><TD style="width:85px"><P><STRONG>15</STRONG></P></TD><TD style="width:284px"><P>CFTR 2 T</P></TD><TD style="width:236px"><P>62,5</P></TD></TR><TR><TD style="width:85px"><P><STRONG>16</STRONG></P></TD><TD style="width:284px"><P>GFP 2</P></TD><TD style="width:236px"><P>94,2</P></TD></TR></TBODY></TABLE><P> </P></DIV></DIV></DIV></DIV><DIV class="row"><DIV class="col-lg-12"><DIV class="notebookentry"><H3 class="notebookheading">▼2015-07-15  Ligation of the cutted pSB1C3-MCS-pcat with ribozyme fragments</H3><DIV class="notebooktext"><P>Procedures:</P><P> </P><P>Ligation:</P><P> </P><P>Description:</P><P> </P><P>Materials and chemicals:</P><P>2 µl 10x T4 Ligase Buffer: thawed and resuspended at room temperature</P><P>1 parts Vector DNA (mol not l)</P><P>3 part Insert DNA (mol not l)</P><P>20 µl Nuclease free water</P><P>1 µl T4 DNA Ligase</P><P> </P><P>Steps:</P><P> </P><OL><LI>Set up the reaction in a microcentrifuge tube on ice. T4 DNA Ligase should be added last. The molar ratio of vector to insert should be 1:3.</LI></OL><P> </P><OL><LI>Gently mix the reaction by pipetting up and down and microfuge briefly.</LI></OL><P> </P><OL><LI>For cohesive (sticky) ends, incubate at 16°C overnight or room temperature for 10 minutes.</LI></OL><P> </P><OL><LI>For blunt ends or single base overhangs, incubate at 16°C overnight or room temperature for 2 hours. Alternatively a high concentration of T4 Ligase can be used in a 10 minute ligation.</LI></OL><P> </P><OL><LI>Heat inactivate at 80°C for 10 minutes</LI></OL><P> </P><OL><LI>Chill on ice and transform 1-5 µl of the reaction into 50 µl of competent cells.</LI></OL><P> </P><P>Pipetting sheet:</P><P> </P><P>For fragment numbers see digestion of ribozyme fragments protocol</P><P> </P><P>In one tube:</P><P>40 ng of pSB1C3 (about 1,5 µl)</P><P>13 ng of non-DE-ribozymes or 22 ng of DE ribozymes</P><P>1 µl Ligase</P><P>2 µl Buffer</P><P> </P><P>Mastermix: pSB1C3, Ligase used 5x concentrated ligase (2 Million U per ml) diluted with ddH2O, Buffer</P><P> </P><P>Everything in µl:</P><P> </P><TABLE border="1" cellpadding="0" cellspacing="0"><TBODY><TR><TD style="width:151px"><P>Fragment-Number</P></TD><TD style="width:151px"><P>Mastermix</P></TD><TD style="width:151px"><P>Insert</P></TD><TD style="width:151px"><P>ddH2O</P></TD></TR><TR><TD style="width:151px"><P>1</P></TD><TD style="width:151px"><P>4,5</P></TD><TD style="width:151px"><P>0,7</P></TD><TD style="width:151px"><P>14,8</P></TD></TR><TR><TD style="width:151px"><P>2</P></TD><TD style="width:151px"><P>4,5</P></TD><TD style="width:151px"><P>0,2</P></TD><TD style="width:151px"><P>15,3</P></TD></TR><TR><TD style="width:151px"><P>3</P></TD><TD style="width:151px"><P>4,5</P></TD><TD style="width:151px"><P>0,2</P></TD><TD style="width:151px"><P>15,3</P></TD></TR><TR><TD style="width:151px"><P>4</P></TD><TD style="width:151px"><P>4,5</P></TD><TD style="width:151px"><P>0,2</P></TD><TD style="width:151px"><P>15,3</P></TD></TR><TR><TD style="width:151px"><P>5</P></TD><TD style="width:151px"><P>4,5</P></TD><TD style="width:151px"><P>0,2</P></TD><TD style="width:151px"><P>15,3</P></TD></TR><TR><TD style="width:151px"><P>6</P></TD><TD style="width:151px"><P>4,5</P></TD><TD style="width:151px"><P>0,2</P></TD><TD style="width:151px"><P>15,3</P></TD></TR><TR><TD style="width:151px"><P>7</P></TD><TD style="width:151px"><P>4,5</P></TD><TD style="width:151px"><P>0,6</P></TD><TD style="width:151px"><P>14,9</P></TD></TR><TR><TD style="width:151px"><P>8</P></TD><TD style="width:151px"><P>4,5</P></TD><TD style="width:151px"><P>0,4</P></TD><TD style="width:151px"><P>15,1</P></TD></TR><TR><TD style="width:151px"><P>9</P></TD><TD style="width:151px"><P>4,5</P></TD><TD style="width:151px"><P>0,3</P></TD><TD style="width:151px"><P>15,2</P></TD></TR><TR><TD style="width:151px"><P>10</P></TD><TD style="width:151px"><P>4,5</P></TD><TD style="width:151px"><P>0,4</P></TD><TD style="width:151px"><P>15,1</P></TD></TR><TR><TD style="width:151px"><P>11</P></TD><TD style="width:151px"><P>4,5</P></TD><TD style="width:151px"><P>0,4</P></TD><TD style="width:151px"><P>15,1</P></TD></TR><TR><TD style="width:151px"><P>12</P></TD><TD style="width:151px"><P>4,5</P></TD><TD style="width:151px"><P>0,4</P></TD><TD style="width:151px"><P>15,1</P></TD></TR><TR><TD style="width:151px"><P>13</P></TD><TD style="width:151px"><P>4,5</P></TD><TD style="width:151px"><P>0,3</P></TD><TD style="width:151px"><P>15,2</P></TD></TR><TR><TD style="width:151px"><P>14</P></TD><TD style="width:151px"><P>4,5</P></TD><TD style="width:151px"><P>0,8</P></TD><TD style="width:151px"><P>14,7</P></TD></TR><TR><TD style="width:151px"><P>15</P></TD><TD style="width:151px"><P>4,5</P></TD><TD style="width:151px"><P>0,2</P></TD><TD style="width:151px"><P>15,3</P></TD></TR><TR><TD style="width:151px"><P>16</P></TD><TD style="width:151px"><P>4,5</P></TD><TD style="width:151px"><P>0,2</P></TD><TD style="width:151px"><P>15,3</P></TD></TR></TBODY></TABLE><P> </P></DIV></DIV></DIV></DIV><DIV class="row"><DIV class="col-lg-12"><DIV class="notebookentry"><H3 class="notebookheading">▼2015-07-16  Digestion of pSB1C3-MCS-pcat with BmtI and BamHI 2 step</H3><DIV class="notebooktext"><P>Description:</P><P> </P><P>We made a 2 step digestion with BmtI and BamHI for 1 and 4 hours to proper cut the pSB1C3 in the MCS region, because the BmtI and BamHI cutting sites are next to each other and the exonuclease activity of the BamHI needs to be triggered.</P><P> </P><P>Procedures:</P><P> </P><P>2 step ligation:</P><P> </P><P>Description:</P><P> </P><P>First digetion with BmtI:</P><P> </P><P>Steps:</P><P> </P><OL><LI>Make a mix of following chemicals in a 1,5 ml centrifuge tube:</LI></OL><P>                25 µl pSB1C3-MCS-pcat</P><P>                1 µl BmtI</P><P>                5 µl CutSmart 10x Buffer</P><P>                19 µl ddH2O</P><P> </P><OL><LI>Incubate for 1 hour at 37°C</LI></OL><P> </P><OL><LI>Make a PCR purification</LI></OL><P> </P><P>Second digestion with BamHI:</P><P> </P><P>Steps:</P><P> </P><OL><LI>Make a mix of following chemicals:</LI></OL><P>                35 µl of DNA solution from the prior PCR purification</P><P>                10 µl BamHI</P><P>                5 µl CutSmart 10x Buffer</P><P> </P><OL><LI>Incubate for 4 hours at 37°C</LI></OL><P> </P><OL><LI>Make a PCR purification</LI></OL></DIV></DIV></DIV></DIV><DIV class="row"><DIV class="col-lg-12"><DIV class="notebookentry"><H3 class="notebookheading">▼2015-07-18  Outamplification and transformation of the CFTR-testconstruct</H3><DIV class="notebooktext"><P>Description:</P><P> </P><P>In order to change the cloning system from Gibson to restriction cloning, the CFTR fragment has been first mutated (single base mutation on sequencing data) and then outamplified. Afterwards it has been cutted, ligated with the vector and then transformed into E. coli</P><P> </P><P>Mutation PCR #1:</P><P> </P><P>Description:</P><P> </P><P>Materials and chemicals:</P><P>25 µl Phusion flash</P><P>1 µl Primer fwd Dh_23</P><P>1 µl Primer rev DH_24</P><P>0,5 µl p413+CFTR-construct (which is inserted the other way around)</P><P>22,5 µl water</P><P> </P><P>PCR:</P><P> </P><P>98°C for 1 min</P><P>--------------------</P><P>98°C for 10 sec</P><P>72°C for 2 minutes</P><P>(Repeat 34 times)</P><P>---------------------</P><P>72°C for 4 minutes</P><P>4°C for holding</P><P> </P><P>Mutation PCR #2</P><P> </P><P>Description:</P><P> </P><P>Materials and chemicals:</P><P>5 µl Phusion flash</P><P>1 µl Primer fwd Dh_23</P><P>1 µl Primer rev DH_24</P><P>0,5 µl p413+CFTR-construct (which is inserted the other way around)</P><P>2,5 µl water</P><P> </P><P>PCR:</P><P> </P><P>98°C for 1 min</P><P>--------------------</P><P>98°C for 10 sec</P><P>Annealing temperature for 30 sec</P><P>72°C for 1,5 min</P><P> (Repeat 34 times)</P><P>---------------------</P><P>72°C for 4 minutes</P><P>4°C for holding</P><P> </P><P>4 samples with different annealing temperatures were made:</P><P> </P><P>1: 72°C</P><P>2: 70°C</P><P>3: 68°C</P><P>4: 65°C</P><P> </P><P>Materials and chemicals:</P><P>5 µl Phusion flash</P><P>1 µl Primer fwd Dh_23</P><P>1 µl Primer rev DH_24</P><P>0,5 µl p413+CFTR-construct (which is inserted the other way around)</P><P>2,5 µl water</P><P> </P><P>PCR:</P><P> </P><P>98°C for 1 min</P><P>--------------------</P><P>98°C for 10 sec</P><P>Annealing temperature for 30 sec</P><P>72°C for 1,5 min</P><P> (Repeat 34 times)</P><P>---------------------</P><P>72°C for 4 minutes</P><P>4°C for holding</P><P> </P><P>4 samples with different annealing temperatures were made:</P><P> </P><P>1: 72°C</P><P>2: 70°C</P><P>3: 68°C</P><P>4: 65°C</P><P> </P><P> </P><P>Materials and chemicals:</P><P>5 µl Phusion flash</P><P>1 µl Primer fwd Dh_23</P><P>1 µl Primer rev DH_24</P><P>0,5 µl p413+CFTR-construct (which is inserted the other way around)</P><P>2,5 µl water</P><P> </P><P>PCR:</P><P> </P><P>98°C for 1 min</P><P>--------------------</P><P>98°C for 10 sec</P><P>Annealing temperature for 30 sec</P><P>72°C for 1,5 min</P><P>(Repeat 34 times)</P><P>---------------------</P><P>72°C for 4 minutes</P><P>4°C for holding</P><P> </P><P>4 samples with different annealing temperatures were made:</P><P> </P><P>1: 72°C</P><P>2: 70°C</P><P>3: 68°C</P><P>4: 65°C</P><P> </P><P>The 65°C annealing temperature gave the best results and further work was made with the sample 4.</P><P> </P><P>Outamplification and rebuild of the end sites for the CFTR construct</P><P> </P><P>Description:</P><P> </P><P>Materials and chemicals:</P><P>25 µl Phusion Flash</P><P>1 µl DH_25</P><P>1 µl DH_26</P><P>1 µl DNA from Sample 4 of mutation PCR #2</P><P>22 µl ddH2O</P><P> </P><P>PCR-program:</P><P>98°C for 1 min</P><P>------------------------</P><P>98°C for 10 s</P><P>64°C for 15 s</P><P>72°C for 20 s</P><P> (Repeat 34 times)</P><P>-----------------------</P><P>72°C for 1 min</P><P>4°C for holding</P><P> </P><P>CFTR-Digestion:</P><P> </P><P>Digestion CFTR-construct:</P><P> </P><P>26 µl CFTR-Test construct</P><P>5 µl BamHI</P><P>5 µl HindIII</P><P>4 µl CutSmart 10x</P><P>Digest 3 hours at 37°C</P><P> </P><P>Digestion p413-GPD:</P><P> </P><P>10 µl p413-GPD</P><P>1 µl BamHI</P><P>1 µl HindIII</P><P>2 µl CutSmart 10x</P><P>6 µl ddH2O</P><P> </P><P>Digest 60 minutes at 37°C</P><P> </P><P>Ligation:</P><P> </P><P>Description:</P><P> </P><P>Materials and chemicals:</P><P>2 µl 10x T4 Ligase Buffer: thawed and resuspended at room temperature</P><P>1 parts Vector DNA (mol not l)</P><P>3 part Insert DNA (mol not l)</P><P>20 µl Nuclease free water</P><P>1 µl T4 DNA Ligase</P><P> </P><P>Steps:</P><P> </P><OL><LI>Set up the reaction in a microcentrifuge tube on ice. T4 DNA Ligase should be added last. The molar ratio of vector to insert should be 1:3.</LI></OL><P> </P><OL><LI>Gently mix the reaction by pipetting up and down and microfuge briefly.</LI></OL><P> </P><OL><LI>For cohesive (sticky) ends, incubate at 16°C overnight or room temperature for 10 minutes.</LI></OL><P> </P><OL><LI>For blunt ends or single base overhangs, incubate at 16°C overnight or room temperature for 2 hours. Alternatively a high concentration of T4 Ligase can be used in a 10 minute ligation.</LI></OL><P> </P><OL><LI>Heat inactivate at 80°C for 10 minutes</LI></OL><P> </P><OL><LI>Chill on ice and transform 1-5 µl of the reaction into 50 µl of competent cells</LI></OL><P> </P><P>KCM Transformation:</P><P> </P><P>Description:</P><P> </P><P>Steps:</P><P> </P><OL><LI>Take 50µl chemical competent E. coli from -80 freezer and thaw on ice</LI></OL><P> </P><OL><LI>Add (as master mix):</LI></OL><P>                2,5µl DNA</P><P>                10µl KCM 5x</P><P>                37,5µl H2O</P><P> </P><OL><LI>Incubate on ice for 30 minutes</LI></OL><P> </P><OL><LI>Heat shock at 42°C for 1 minute</LI></OL><P> </P><OL><LI>Incubate on ice for 2 minutes</LI></OL><P> </P><OL><LI>Add 900 µl of LB or 2x YT Medium</LI></OL><P> </P><OL><LI>Incubate on 37°C for 60min</LI></OL><P> </P><OL><LI>Centrifuge 5min at 1000g</LI></OL><P> </P><OL><LI>Take 900µl of supernatant and throw away</LI></OL><P> </P><OL><LI>Resuspend pellet in remaining media</LI></OL><P> </P><OL><LI>Plate out on agar with antibiotics (1:1 / 1:10)</LI></OL><P> </P><P>Colony PCR:</P><P> </P><P>Description:</P><P> </P><P>Materials and Chemicals:</P><P>PCR-tubes</P><P>Forward primer</P><P>Reverse primer</P><P>Masermix (dNTPs, Polymerase, buffer)</P><P>Water</P><P>Thermocycler</P><P> </P><P>Endvolume: 10 µl</P><P> </P><P>Steps:</P><P> </P><OL><LI>Pick colonies from plates. Solute one colony in about 20 µl of water.</LI></OL><P> </P><OL><LI>Give the colonies into 10 µl colony PCR solution with OneTaq Mastermix (which should be diluted to 1x in the end (e.g. you need 5 µl of 2x mastermix for 10 µl)) and primer (between 0,1 and 1 µl)</LI></OL><P> </P><OL><LI>Use the Thermocycler with an appropriate PCR program for at least 25 cycles</LI></OL><P> </P></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV><DIV class="container"><DIV class="content"><DIV class="row"><DIV class="col-lg-12"><DIV class="panel panel-default"><DIV class="panel-heading"><H3 class="basicheader">week number 27</H3></DIV><DIV class="panel-body"><DIV class="row"><DIV class="col-lg-12"><DIV class="notebookentry"><H3 class="notebookheading">▼2015-07-01  Ribozyme amplification</H3><DIV class="notebooktext"><P> </P><TABLE align="left" border="0" cellpadding="0" cellspacing="0" style="width:364px"><TBODY><TR><TD style="height:29px; width:67px"><P>Number</P></TD><TD style="height:29px; width:159px"><P>Ribozyme</P></TD><TD style="height:29px; width:81px"><P>Primer fwd</P></TD><TD style="height:29px; width:57px"><P>Primer rvs</P></TD></TR><TR><TD style="height:20px; width:67px"><P><STRONG>1</STRONG></P></TD><TD style="height:20px; width:159px"><P>CFTR 1 T</P></TD><TD rowspan="3" style="height:20px; width:81px"><P>DH_35</P></TD><TD rowspan="3" style="height:20px; width:57px"><P>DH_45</P></TD></TR><TR><TD style="height:20px; width:67px"><P><STRONG>2</STRONG></P></TD><TD style="height:20px; width:159px"><P>CFTR 1 A</P></TD></TR><TR><TD style="height:20px; width:67px"><P><STRONG>3</STRONG></P></TD><TD style="height:20px; width:159px"><P>CFTR 1 C</P></TD></TR><TR><TD style="height:20px; width:67px"><P><STRONG>4</STRONG></P></TD><TD style="height:20px; width:159px"><P>CFTR 2 T</P></TD><TD style="height:20px; width:81px"><P>DH_36</P></TD><TD style="height:20px; width:57px"><P>DH_46</P></TD></TR><TR><TD style="height:20px; width:67px"><P><STRONG>5</STRONG></P></TD><TD style="height:20px; width:159px"><P>CFTR 2 A</P></TD><TD style="height:20px; width:81px"><P>DH_37</P></TD><TD rowspan="2" style="height:20px; width:57px"><P>DH_47</P></TD></TR><TR><TD style="height:20px; width:67px"><P><STRONG>6</STRONG></P></TD><TD style="height:20px; width:159px"><P>CFTR 2 C</P></TD><TD style="height:20px; width:81px"><P>DH_38</P></TD></TR><TR><TD style="height:20px; width:67px"><P><STRONG>7</STRONG></P></TD><TD style="height:20px; width:159px"><P>CFTR 1 DE T</P></TD><TD rowspan="3" style="height:20px; width:81px"><P>DH_39</P></TD><TD rowspan="8" style="height:20px; width:57px"><P>DH_48</P></TD></TR><TR><TD style="height:20px; width:67px"><P><STRONG>8</STRONG></P></TD><TD style="height:20px; width:159px"><P>CFTR 1 DE A</P></TD></TR><TR><TD style="height:20px; width:67px"><P><STRONG>9</STRONG></P></TD><TD style="height:20px; width:159px"><P>CFTR 1 DE C</P></TD></TR><TR><TD style="height:20px; width:67px"><P><STRONG>10</STRONG></P></TD><TD style="height:20px; width:159px"><P>CFTR 2 DE T</P></TD><TD rowspan="3" style="height:20px; width:81px"><P>DH_40</P></TD></TR><TR><TD style="height:20px; width:67px"><P><STRONG>11</STRONG></P></TD><TD style="height:20px; width:159px"><P>CFTR 2 DE A</P></TD></TR><TR><TD style="height:20px; width:67px"><P><STRONG>12</STRONG></P></TD><TD style="height:20px; width:159px"><P>CFTR 2 DE C</P></TD></TR><TR><TD style="height:20px; width:67px"><P><STRONG>13</STRONG></P></TD><TD style="height:20px; width:159px"><P>GFP 1 DE</P></TD><TD style="height:20px; width:81px"><P>DH_41</P></TD></TR><TR><TD style="height:20px; width:67px"><P><STRONG>14</STRONG></P></TD><TD style="height:20px; width:159px"><P>GFP 2 DE</P></TD><TD style="height:20px; width:81px"><P>DH_42</P></TD></TR><TR><TD style="height:20px; width:67px"><P><STRONG>15</STRONG></P></TD><TD style="height:20px; width:159px"><P>GFP 1</P></TD><TD style="height:20px; width:81px"><P>DH_43</P></TD><TD rowspan="2" style="height:20px; width:57px"><P>DH_49</P></TD></TR><TR><TD style="height:20px; width:67px"><P><STRONG>16</STRONG></P></TD><TD style="height:20px; width:159px"><P>GFP 2</P></TD><TD style="height:20px; width:81px"><P>DH_44</P></TD></TR><TR><TD style="height:20px; width:67px"><P> </P><P> </P></TD><TD style="height:20px; width:159px"><P> </P></TD><TD style="height:20px; width:81px"><P> </P></TD><TD style="height:20px; width:57px"><P> </P></TD></TR></TBODY></TABLE><P><STRONG>PCR - amplification </STRONG></P><P> </P><TABLE border="1" cellpadding="0" cellspacing="0"><TBODY><TR><TD style="width:205px"><P> </P></TD><TD style="width:205px"><P> for 10µl [µl]</P></TD><TD style="width:205px"><P>for 50µl [µl]</P></TD></TR><TR><TD style="width:205px"><P>Template</P></TD><TD style="width:205px"><P>1</P></TD><TD style="width:205px"><P>5</P></TD></TR><TR><TD style="width:205px"><P>Primer fwd</P></TD><TD style="width:205px"><P>1</P></TD><TD style="width:205px"><P>5</P></TD></TR><TR><TD style="width:205px"><P>Primer rvs</P></TD><TD style="width:205px"><P>0.5</P></TD><TD style="width:205px"><P>0.5</P></TD></TR><TR><TD style="width:205px"><P>ddH2O</P></TD><TD style="width:205px"><P>2.5</P></TD><TD style="width:205px"><P>14.5</P></TD></TR><TR><TD style="width:205px"><P>Q5 Polymerase</P></TD><TD style="width:205px"><P>5</P></TD><TD style="width:205px"><P>25</P></TD></TR></TBODY></TABLE><P> </P><P> </P><P><STRONG>PCR conditions</STRONG></P><P> </P><TABLE border="1" cellpadding="0" cellspacing="0"><TBODY><TR><TD style="width:205px"><P> </P></TD><TD style="width:205px"><P>Time [s]</P></TD><TD style="width:205px"><P>Temperature [°C]</P></TD></TR><TR><TD style="width:205px"><P>Initial denaturation</P></TD><TD style="width:205px"><P>30</P></TD><TD style="width:205px"><P>98</P></TD></TR><TR><TD style="width:205px"><P>Denaturation</P></TD><TD style="width:205px"><P>10</P></TD><TD style="width:205px"><P>98</P></TD></TR><TR><TD style="width:205px"><P>Annealing</P></TD><TD style="width:205px"><P>10</P></TD><TD style="width:205px"><P>66</P></TD></TR><TR><TD style="width:205px"><P>Elongation</P></TD><TD style="width:205px"><P>15</P></TD><TD style="width:205px"><P>72</P></TD></TR><TR><TD style="width:205px"><P>Final extension</P></TD><TD style="width:205px"><P>30</P></TD><TD style="width:205px"><P>72</P></TD></TR><TR><TD style="height:25px; width:205px"><P>Hold</P></TD><TD style="height:25px; width:205px"><P>∞</P></TD><TD style="height:25px; width:205px"><P>4</P></TD></TR></TBODY></TABLE><P> </P><P> </P><P><STRONG>PCR Purification</STRONG></P><UL><LI>QIAGEN Kit</LI></UL></DIV></DIV></DIV></DIV><DIV class="row"><DIV class="col-lg-12"><DIV class="notebookentry"><H3 class="notebookheading">▼2015-07-05  pSB1C3-vector assembly with a new artificial multiple cloning site and pcat promotor</H3><DIV class="notebooktext"><P>Description:</P><P>In this experiment, we wanted to insert a Multiple Cloning Site (MCS) and a pCat promotor into the pSB1C3-vector</P><P> </P><P>Procedures:</P><P> </P><P>Oligo-Annealing:</P><P> </P><P>Materials and chemicals:</P><P>    5 µl forward oligo</P><P>    5 µl reverse oligo</P><P>    95 µl sterile water</P><P> </P><P>Steps:</P><P> </P><OL><LI>Incubate phosphorylated oligos at 95°C for 3 minutes</LI></OL><P> </P><OL><LI>Cool the reaction down slowly for 30 minutes to one hour</LI></OL><P> </P><OL><LI>Check the concentration on the nanodrop</LI></OL><P> </P><P>Ligation:</P><P> </P><P>        Materials and chemicals:</P><P> </P><P>        2 µl 10x T4 Ligase Buffer: thawed and resuspended at room temperature</P><P>        1 parts Vector DNA (mol not l)</P><P>        3 part Insert DNA (mol not l)</P><P>        1 µl T4 DNA Ligase</P><P>        To 20µl nuclease free water</P><P> </P><P>Steps:</P><P> </P><OL><LI>Set up the reaction in a microcentrifuge tube on ice. T4 DNA Ligase should be added last. The molar ratio of vector to insert should be 1:3.</LI></OL><P> </P><OL><LI>Gently mix the reaction by pipetting up and down and microfuge briefly.</LI></OL><P> </P><OL><LI>For cohesive (sticky) ends, incubate at 16°C overnight or room temperature for 10 minutes.</LI></OL><P>                For blunt ends or single base overhangs, incubate at 16°C overnight or room temperature for                2 hours. Alternatively a high concentration of T4 Ligase can be used in a 10 minute ligation.</P><P> </P><OL><LI>Heat inactivate at 80°C for 10 minutes</LI></OL><P> </P><OL><LI>Chill on ice and transform 1-5 µl of the reaction into 50 µl of competent cells.</LI></OL><P> </P><P>Notes:</P><P> </P><P>We used several ligation strategies and insert to backbone concentrations. We used a annealing product for the ligation</P><P> </P><P>20 µl test digest</P><P> </P><P>Description:</P><P> </P><P>Steps</P><OL><LI>Set up reaction according to protocol:</LI></OL><P>                              ddH2O for a final volume of 20 µl</P><P>                              2 µl of 10x Reaction Buffer (e.g. NEB CutSmart)</P><P>                              0.5 µl of selected Enzyme(s)</P><P>                              ca. 1 µl of mini prep DNA (Range 200-1000 ng)</P><P> </P><OL><LI>Incubate at 37°C for 60'</LI></OL><P> </P><OL><LI>Load on gel (add loading dye first)</LI></OL><P> </P><P>Notes:</P><P> </P><P>We wanted to verify our results by digestion with:</P><P>                1. An enzyme that is exclusively on pSB1C3</P><P>                2. An enzyme that is exclusively on our new inserted multiple cloning site</P><P> </P><P>Oligo Phosphorylation:</P><P> </P><P>Description:</P><P> </P><P>Materials and Chemicals:</P><P> </P><P>        2 µl 100 µM oligo stock</P><P>        2 µl 10x T4 DNA ligase buffer</P><P>        1 µl T4 polynucleotide kinase</P><P>        15 µl sterile water</P><P> </P><P>Steps:</P><P> </P><OL><LI>Pipette the Chemicals in one micro centrifuge tube</LI></OL><P> </P><OL><LI>Incubate the reaction mixture at 37°C for 1 hour</LI></OL><P> </P><OL><LI>Heat inactivate the polynucleotide kinases at 65°C for 20 minutes.</LI></OL><P> </P><P>KCM Transformation:</P><P> </P><P>Description:</P><P> </P><P>Steps:</P><P> </P><OL><LI>Take 50µl chemical competent E. coli from -80 freezer and thaw on ice</LI></OL><P> </P><OL><LI>Add (as master mix):</LI></OL><P>                2,5µl DNA</P><P>                10µl KCM 5x</P><P>                37,5µl H2O</P><P> </P><OL><LI>Incubate on ice for 30 minutes</LI></OL><P> </P><OL><LI>Heat shock at 42°C for 2 minutes</LI></OL><P> </P><OL><LI>Incubate on ice for 2 minutes</LI></OL><P> </P><OL><LI>Add 900 µl of LB or 2x YT Medium</LI></OL><P> </P><OL><LI>Incubate on 37°C for 60min</LI></OL><P> </P><OL><LI>Centrifuge 5min at 1000g</LI></OL><P> </P><OL><LI>Take 900µl of supernatant and throw away</LI></OL><P> </P><OL><LI>Resuspend pellet in remaining media</LI></OL><P> </P><OL><LI>Plate out on agar with antibiotics (1:1 / 1:10)</LI></OL><P> </P><P> </P><P>Results:</P><P> </P><P>None of our ligations worked to assemble the pSB1C3, MCS and pCat. We used a annealing product without considering that our product has no 5'phosphate and can therefore not be ligated by the T4 ligase. Therefore we used the T4 polynucleotide kinase and tried to ligate again, but the ligation did not worked again.</P><P> </P><P>Conclusion:</P><P> </P><P>We ordered new primers with a 5' phosphate, to try the ligation of the PCR-products (MCS and pCat) with the pSB1C3</P></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV><DIV class="container"><DIV class="content"><DIV class="row"><DIV class="col-lg-12"><DIV class="panel panel-default"><DIV class="panel-heading"><H3 class="basicheader">week number 26</H3></DIV><DIV class="panel-body"><DIV class="row"><DIV class="col-lg-12"><DIV class="notebookentry"><H3 class="notebookheading">▼2015-06-22  Miniprep and cryostock of culture 10 in colony PCR</H3><DIV class="notebooktext"><P>Description:</P><P> </P><P>Procedures:</P><P> </P><P>Qiaprep Spin Miniprep Kit:</P><P> </P><P>Description:</P><P> </P><P>    Steps:</P><OL><LI>Prepare o/n culture</LI><LI>Perform mini prep according to manufacturers protocol</LI></OL><P> </P><P>E.coli glycerol stocks:</P><P> </P><P>Description:</P><P> </P><P>    Steps:</P><OL><LI>Grow up an overnight culture of strains of interest</LI><LI>Transfer 500µl into a safe lock reaction tube</LI><LI>Add 500µl of 40% sterile glycerol solution</LI><LI>Freeze slowly at -80°C</LI></OL><P> </P><P>Results:</P><P> </P><P>Miniprep: 451,5ng/µl</P><P>2 stocks were frozen at -80°</P></DIV></DIV></DIV></DIV><DIV class="row"><DIV class="col-lg-12"><DIV class="notebookentry"><H3 class="notebookheading">▼2015-06-23  Insert of MCS + promotor in cutted pSB1C3</H3><DIV class="notebooktext"><P>Description:</P><P> </P><P>Procedures:</P><P> </P><P>Ligation:</P><P> </P><P>Description:</P><P> </P><P>        Materials and chemicals:</P><P> </P><P>        2 µl 10x T4 Ligase Buffer: thawed and resuspended at room temperature</P><P>       1 parts Vector DNA (mol not l)</P><P>       3 part Insert DNA (mol not l)</P><P>       20 µl Nuclease free water</P><P>        1 µl T4 DNA Ligase</P><P> </P><P>    Steps:</P><P> </P><P>Set up the reaction in a microcentrifuge tube on ice. T4 DNA Ligase should be added last. The molar ratio of vector to insert should be 1:3.</P><P> </P><P>        Gently mix the reaction by pipetting up and down and microfuge briefly.</P><P> </P><P>        For cohesive (sticky) ends, incubate at 16°C overnight or room temperature for 10 minutes.</P><P> </P><P>For blunt ends or single base overhangs, incubate at 16°C overnight or room temperature for 2 hours. Alternatively a high concentration of T4 Ligase can be used in a 10 minute ligation.</P><P> </P><P>        Heat inactivate at 80°C for 10 minutes</P><P> </P><P>        Chill on ice and transform 1-5 µl of the reaction into 50 µl of competent cells.</P><P> </P><P> </P><P>Notes:</P><P> </P><P>        Used fragments/chemicals:</P><P> </P><P>        1 µl cutted pSB1C3</P><P>        0,1 µl MCS</P><P>        0,1 µl pCat</P><P>        2 µl T4 Buffer</P><P>        1 µl T4 Ligase</P><P>        Filled up to 20 µl with water</P><P> </P><P>        Incubated for 2 hours at room temperature</P><P> </P><P>Results:</P><P> </P><P>Plated transformated bacteria (with ligation product) did not grew on a LB-Agar with CM</P></DIV></DIV></DIV></DIV><DIV class="row"><DIV class="col-lg-12"><DIV class="notebookentry"><H3 class="notebookheading">▼2015-06-23  PCR: Site-change of ribozyme constructs</H3><DIV class="notebooktext"><P>Description:</P><P> </P><P>Aim: Get every fragment to the same cloning standart.</P><P>BamHI -----------------------------/ BmtI</P><P> </P><P>Procedures:</P><P> </P><P>23.06.2015:</P><P> </P><P>Description</P><P> </P><P>        PCR mix:</P><P> </P><P>       Polymerase Mastermix 2x: 25 µl</P><P>        Primer:        fwd: 1 µl</P><P>                              rev 1 µl</P><P>        Template-DNA 1 µl</P><P>        ddH20 22 µl</P><P> </P><TABLE border="1" cellpadding="0" cellspacing="0"><TBODY><TR><TD style="width:121px"><P>Number</P></TD><TD style="width:121px"><P>Fragment Name</P></TD><TD style="width:121px"><P>T Anneal</P></TD><TD style="width:121px"><P>Fwd</P></TD><TD style="width:121px"><P>Rev</P></TD></TR><TR><TD style="width:121px"><P>1</P></TD><TD style="width:121px"><P>CFTR 1 T</P></TD><TD style="width:121px"><P>58°C</P></TD><TD style="width:121px"><P>DH13</P></TD><TD style="width:121px"><P>DH14</P></TD></TR><TR><TD style="width:121px"><P>2</P></TD><TD style="width:121px"><P>CFTR 1 A</P></TD><TD style="width:121px"><P>58°C</P></TD><TD style="width:121px"><P>DH13</P></TD><TD style="width:121px"><P>DH14</P></TD></TR><TR><TD style="width:121px"><P>3</P></TD><TD style="width:121px"><P>CFTR 1 C</P></TD><TD style="width:121px"><P>58°C</P></TD><TD style="width:121px"><P>DH13</P></TD><TD style="width:121px"><P>DH14</P></TD></TR><TR><TD style="width:121px"><P>4</P></TD><TD style="width:121px"><P>CFTR 2 A</P></TD><TD style="width:121px"><P>61°C</P></TD><TD style="width:121px"><P>DH13</P></TD><TD style="width:121px"><P>DH15</P></TD></TR><TR><TD style="width:121px"><P>5</P></TD><TD style="width:121px"><P>CFTR 2 C</P></TD><TD style="width:121px"><P>61°C</P></TD><TD style="width:121px"><P>DH13</P></TD><TD style="width:121px"><P>DH15</P></TD></TR><TR><TD style="width:121px"><P>6</P></TD><TD style="width:121px"><P>GFP 1</P></TD><TD style="width:121px"><P>61°C</P></TD><TD style="width:121px"><P>DH16</P></TD><TD style="width:121px"><P>DH17</P></TD></TR></TBODY></TABLE><P> </P><P> </P><P>Results:</P><P> </P><P>1: 1 band + smear: PCR settings were not right --&gt; many sideproducts were synthesized</P><P>2: 2 bands + smear: As above, lower band is our desired product</P><P>3: 2 bands: Higher product yield than 1 and 2, higher purity than 1 and 2</P><P>4: One small band: Low yield and higher purity than 3</P><P>5: One small band: Lower yield than 4</P><P>6: Smear: No visible product band</P><P> </P><P>The PCR settings were wrong for our target DNA.</P><P> </P><DIV class="col-lg-12"><DIV class="imagewrapper"><DIV class="imagewrappercaption"><STRONG>
                Figure 2: Site change of ribozyme constructs: First PCR
            </STRONG><P class="basictext">
                Positive colonies exhibit a clear band with a length of about 200 bp. Bigger products seem to be side products
            </P></DIV></DIV></DIV><P> </P><P>24.06.2015:</P><P> </P><P>Description:</P><P> </P><P>        PCR-Mix:</P><P>       25 µl Polymerase Master Mix 2x</P><P>        1 µl Primer fwd</P><P>        1 µl Primer rev</P><P>        1 µl Template-DNA</P><P>        22 µl ddH2O</P><P> </P><TABLE border="1" cellpadding="0" cellspacing="0"><TBODY><TR><TD style="width:86px"><P>Number</P></TD><TD style="width:86px"><P>Template</P></TD><TD style="width:86px"><P>T Anneal</P></TD><TD style="width:86px"><P>Fwd</P></TD><TD style="width:86px"><P>Rev</P></TD><TD style="width:86px"><P>C(ng/µl)</P></TD><TD style="width:86px"><P>Comment</P></TD></TR><TR><TD style="width:86px"><P>1</P></TD><TD style="width:86px"><P>CFTR 1 T</P></TD><TD style="width:86px"><P>60°C</P></TD><TD style="width:86px"><P>DH13</P></TD><TD style="width:86px"><P>DH14</P></TD><TD style="width:86px"><P>24</P></TD><TD style="width:86px"><P>Number 1 + 1a in one tube</P></TD></TR><TR><TD style="width:86px"><P>2</P></TD><TD style="width:86px"><P>CFTR 1 A</P></TD><TD style="width:86px"><P>60°C</P></TD><TD style="width:86px"><P>DH13</P></TD><TD style="width:86px"><P>DH14</P></TD><TD style="width:86px"><P>30,5</P></TD><TD style="width:86px"><P>Number 2 + 2a in one tube</P></TD></TR><TR><TD style="width:86px"><P>3</P></TD><TD style="width:86px"><P>CFTR 1 C</P></TD><TD style="width:86px"><P>60°C</P></TD><TD style="width:86px"><P>DH13</P></TD><TD style="width:86px"><P>DH14</P></TD><TD style="width:86px"><P>25</P></TD><TD style="width:86px"><P>Number 3 + 3a in one tube</P></TD></TR><TR><TD style="width:86px"><P>4</P></TD><TD style="width:86px"><P>CFTR 2 A</P></TD><TD style="width:86px"><P>60°C</P></TD><TD style="width:86px"><P>DH13</P></TD><TD style="width:86px"><P>DH15</P></TD><TD style="width:86px"><P>8,1</P></TD><TD style="width:86px"><P> </P></TD></TR><TR><TD style="width:86px"><P>5</P></TD><TD style="width:86px"><P>CFTR 2 C</P></TD><TD style="width:86px"><P>60°C</P></TD><TD style="width:86px"><P>DH13</P></TD><TD style="width:86px"><P>DH15</P></TD><TD style="width:86px"><P>8,2</P></TD><TD style="width:86px"><P> </P></TD></TR><TR><TD style="width:86px"><P>6</P></TD><TD style="width:86px"><P>GFP 1</P></TD><TD style="width:86px"><P>60°C</P></TD><TD style="width:86px"><P>DH16</P></TD><TD style="width:86px"><P>Dh17</P></TD><TD style="width:86px"><P>-0,6</P></TD><TD style="width:86px"><P> </P></TD></TR><TR><TD style="width:86px"><P>1a</P></TD><TD style="width:86px"><P>CFTR 1 T</P></TD><TD style="width:86px"><P>57°C</P></TD><TD style="width:86px"><P>DH13</P></TD><TD style="width:86px"><P>DH14</P></TD><TD style="width:86px"><P>24</P></TD><TD style="width:86px"><P> </P></TD></TR><TR><TD style="width:86px"><P>2a</P></TD><TD style="width:86px"><P>CFTR 1 A</P></TD><TD style="width:86px"><P>57°C</P></TD><TD style="width:86px"><P>DH13</P></TD><TD style="width:86px"><P>DH14</P></TD><TD style="width:86px"><P>30,5</P></TD><TD style="width:86px"><P> </P></TD></TR><TR><TD style="width:86px"><P>3a</P></TD><TD style="width:86px"><P>CFTR 1 C</P></TD><TD style="width:86px"><P>57°C</P></TD><TD style="width:86px"><P>DH13</P></TD><TD style="width:86px"><P>DH14</P></TD><TD style="width:86px"><P>25</P></TD><TD style="width:86px"><P> </P></TD></TR><TR><TD style="width:86px"><P>4a</P></TD><TD style="width:86px"><P>CFTR 2 A</P></TD><TD style="width:86px"><P>57°C</P></TD><TD style="width:86px"><P>DH13</P></TD><TD style="width:86px"><P>DH15</P></TD><TD style="width:86px"><P>7,2</P></TD><TD style="width:86px"><P> </P></TD></TR><TR><TD style="width:86px"><P>5a</P></TD><TD style="width:86px"><P>CFTR 2 C</P></TD><TD style="width:86px"><P>57°C</P></TD><TD style="width:86px"><P>DH13</P></TD><TD style="width:86px"><P>DH15</P></TD><TD style="width:86px"><P> </P></TD><TD style="width:86px"><P> </P></TD></TR><TR><TD style="width:86px"><P>6a</P></TD><TD style="width:86px"><P>GFP 1</P></TD><TD style="width:86px"><P>57°C</P></TD><TD style="width:86px"><P>DH16</P></TD><TD style="width:86px"><P>DH17</P></TD><TD style="width:86px"><P> </P></TD><TD style="width:86px"><P> </P></TD></TR></TBODY></TABLE><P> </P><P> </P><P>Results:</P><P>1/2 = 1/1a</P><P>3/4 = 2/2a</P><P>5/6 = 3/3a</P><P>7/8 = 4/4a</P><P>9/10 = 5/5a</P><P>10/11 = 6/6a</P><P> </P><P> </P><P>1/2: 60°C is the optimal temperature for our PCR</P><P>3/4: 60°C and 57°C yield equal amounts of DNA</P><P>5/6: 60°C and 57°C yield equal amounts of DNA</P><P>7/8: 60°C yields no DNA, 57°C yields less DNA than 1/2</P><P>9/10: 60°C yields no DNA, 57°C yields less DNA than 7/8</P><P>11/12: Smear, no band at 60°C and 57°C</P><P> </P><P>The Annealing Temperatures for most products are help to get a high yield. Just for 9/10 and 11/12 the yields are not optimal.</P><P> </P><DIV class="col-lg-12"><DIV class="imagewrapper"><DIV class="imagewrappercaption"><STRONG>
                Figure 3: Site change of ribozyme constructs: First PCR
            </STRONG><P class="basictext">
                Positive colonies exhibit a clear band with a length of about 200 bp. Bigger products seem to be side products.
            </P></DIV></DIV></DIV><P> </P><P>25.06.2015:</P><P> </P><P>Description:</P><P> </P><P>PCR-Mix:</P><P> </P><P>Like in the former protocols</P><P> </P><TABLE border="1" cellpadding="0" cellspacing="0"><TBODY><TR><TD style="width:121px"><P>Number</P></TD><TD style="width:121px"><P>Template</P></TD><TD style="width:121px"><P>T Anneal</P></TD><TD style="width:121px"><P>Fwd</P></TD><TD style="width:121px"><P>Rev</P></TD></TR><TR><TD style="width:121px"><P>5.1</P></TD><TD style="width:121px"><P>CFTR 2 C</P></TD><TD style="width:121px"><P>55°C</P></TD><TD style="width:121px"><P>DH13</P></TD><TD style="width:121px"><P>DH15</P></TD></TR><TR><TD style="width:121px"><P>5.2</P></TD><TD style="width:121px"><P>CFTR 2 C</P></TD><TD style="width:121px"><P>53°C</P></TD><TD style="width:121px"><P>DH13</P></TD><TD style="width:121px"><P>DH15</P></TD></TR><TR><TD style="width:121px"><P>6.1</P></TD><TD style="width:121px"><P>GFP 1</P></TD><TD style="width:121px"><P>55°C</P></TD><TD style="width:121px"><P>DH16</P></TD><TD style="width:121px"><P>DH17</P></TD></TR><TR><TD style="width:121px"><P>6.2</P></TD><TD style="width:121px"><P>GFP 1</P></TD><TD style="width:121px"><P>53°C</P></TD><TD style="width:121px"><P>DH16</P></TD><TD style="width:121px"><P>DH17</P></TD></TR></TBODY></TABLE><P> </P><P> </P></DIV></DIV></DIV></DIV><DIV class="row"><DIV class="col-lg-12"><DIV class="notebookentry"><H3 class="notebookheading">▼2015-06-23  Digest of pSB1C3 BBa_J04450</H3><DIV class="notebooktext"><P>Description:</P><P> </P><P>Procedure:</P><P> </P><P>20 µl test-digest:</P><P> </P><P>Description</P><P> </P><P>    Steps:</P><P> </P><OL><LI>Set up reaction according to protocol:</LI></OL><P>                ddH2O for a final volume of 20 µl</P><P>                2 µl of 10x Reaction Buffer (e.g. NEB CutSmart)</P><P>                0.5 µl of selected Enzyme(s)</P><P>                ca. 1 µl of mini prep DNA (Range 200-1000 ng)</P><P> </P><OL><LI>Incubate at 37°C for 60'</LI></OL><P> </P><OL><LI>Load on gel (add loading dye first)</LI></OL><P> </P><P>Notes:</P><P> </P><P>        Selected Enzymes:</P><P>        EcoRI/SpeI</P><P> </P><P>        Buffer:</P><P>        Cutsmart</P><P> </P><P>        Heat inactivation at 60°C</P><P> </P><P>        Everything was given on the gel</P><P> </P><P>        A fragment with about 2 kb was cut out</P><P> </P><P>        Gel elution: Result: 50 ng/µl DNA</P><P> </P><P>Results:</P><P> </P><P>50 ng/µl DNA</P><P> </P><P>5:</P><P>Title: Yeast Transformation with p413-GPD and CFTR construct 2</P><P>Author: Hendrik</P><P>Date: 23.06.2015</P><P> </P><P>Description:</P><P> </P><P>Procedures:</P><P> </P><P> </P><P>Yeast transformation:</P><P> </P><P> </P><P>Description:</P><P> </P><P>               10 µl of cells for transformation with a plasmid, 50 µl of cells for transformation with a PCR                      product</P><P>               2 µl of plasmid DNA per 10 µl of cells</P><P>               6 equivalents of PEG</P><P>               1/9 equivalents of DMSO</P><P>               100 - 200 µl of liquid medium</P><P>   </P><P>               Steps:</P><P> </P><OL><LI>Give the plasmid DNA into a Eppi. Add the competent cells.</LI><LI>Mix, then add the PEG.</LI><LI>Incubate for 30 mins at room temperature while mixing</LI><LI>Add the DMSO</LI><LI>Place the yeast in a 42°C water bath for 5-20 minutes</LI><LI>Centrifuge cells for 2-3 minutes at 2000 rpm</LI><LI>Discard the supernatant and resuspend the yeast in the liqiud YPD medium</LI></OL><P> </P><P>Notes:</P><P> </P><P>        10 µl of Yeast and 100 µl of SD-His medium was taken.</P><P>       2 transformations were made</P><P>        After transformation the yeast was plated.</P><P> </P><P>Results:</P><P> </P><P>Some yeast grew on the first plate, but not on the second plate (I took some of the original biofilm and made a fractionated plating). Also the biofilm on the first plate exhibited no further growth. Maybe the wrong yeast medium was token (The writing on the flasks didn't survive the autoclave). Another plating was been made to see if this is right. The yeast on the SD-His and SD-Leu plate exhibited no growth. The yeast in the SD-His/Leu plate grew spotlike and maybe forms colonies.</P></DIV></DIV></DIV></DIV><DIV class="row"><DIV class="col-lg-12"><DIV class="notebookentry"><H3 class="notebookheading">▼2015-06-23  Yeast Transformation with p413-GPD and CFTR construct 2</H3><DIV class="notebooktext"><P>Description:</P><P> </P><P>Procedures:</P><P> </P><P> </P><P>Yeast transformation:</P><P> </P><P> </P><P>Description:</P><P> </P><P>               10 µl of cells for transformation with a plasmid, 50 µl of cells for transformation with a PCR                      product</P><P>               2 µl of plasmid DNA per 10 µl of cells</P><P>               6 equivalents of PEG</P><P>               1/9 equivalents of DMSO</P><P>               100 - 200 µl of liquid medium</P><P>   </P><P>               Steps:</P><P> </P><OL><LI>Give the plasmid DNA into a Eppi. Add the competent cells.</LI><LI>Mix, then add the PEG.</LI><LI>Incubate for 30 mins at room temperature while mixing</LI><LI>Add the DMSO</LI><LI>Place the yeast in a 42°C water bath for 5-20 minutes</LI><LI>Centrifuge cells for 2-3 minutes at 2000 rpm</LI><LI>Discard the supernatant and resuspend the yeast in the liqiud YPD medium</LI></OL><P> </P><P>Notes:</P><P> </P><P>        10 µl of Yeast and 100 µl of SD-His medium was taken.</P><P>       2 transformations were made</P><P>        After transformation the yeast was plated.</P><P> </P><P>Results:</P><P> </P><P>Some yeast grew on the first plate, but not on the second plate (I took some of the original biofilm and made a fractionated plating). Also the biofilm on the first plate exhibited no further growth. Maybe the wrong yeast medium was token (The writing on the flasks didn't survive the autoclave). Another plating was been made to see if this is right. The yeast on the SD-His and SD-Leu plate exhibited no growth. The yeast in the SD-His/Leu plate grew spotlike and maybe forms colonies.</P></DIV></DIV></DIV></DIV><DIV class="row"><DIV class="col-lg-12"><DIV class="notebookentry"><H3 class="notebookheading">▼2015-06-27  Test if transformed cells have the MCS + pcat insert</H3><DIV class="notebooktext"><P>Description:</P><P> </P><P>Procedures were done on 5 ml overnight E.coli culture with estimated MCS+pCat insert in pSB1C3</P><P>Five colonies were picked from the original plate and were given in 5 snapcaps with 5 ml LB medium each.</P><P>The 5 cultures are named 1-5</P><P> </P><P>Procedures:</P><P> </P><P>QIAprep Spin Miniprep Kit:</P><P> </P><P>Description:</P><P> </P><P>    Steps:</P><P> </P><OL><LI>Prepare o/n culture</LI></OL><P> </P><OL><LI>Perform mini prep according to manufacturers protocol</LI></OL><P> </P><P>Notes:</P><P> </P><P>After the miniprep 2 ml of overnight culture were transferred to 100 ml of fresh LB medium with 1:1000 Chloramphenicol</P><P> </P><P>Results:</P><P> </P><P>Nanodrop results:</P><P>1: c = 9,5 ng/µl</P><P>2: c = 7,7 ng/µl</P><P>3: c = 81,5 ng/µl</P><P>4: c = 48,5 ng/µl</P><P>5: c = 64 ng/µl</P><P> </P><P>A test digestion will be made with 3-5. The concentrations for 1-2 are too low.</P><P> </P><P>20 µl test-digest:</P><P> </P><P>Description</P><P> </P><P>    Steps:</P><P> </P><OL><LI>Set up reaction according to protocol:</LI></OL><P>                ddH2O for a final volume of 20 µl</P><P>                2 µl of 10x Reaction Buffer (e.g. NEB CutSmart)</P><P>                0.5 µl of selected Enzyme(s)</P><P>                ca. 1 µl of mini prep DNA (Range 200-1000 ng)</P><P> </P><OL><LI>Incubate at 37°C for 60'</LI></OL><P> </P><OL><LI>Load on gel (add loading dye first)</LI></OL><P> </P><P>Notes:</P><P> </P><P>               Digest with EcoRI and SpeI</P><P>               Heat inactivation at 65°C for 20 minutes</P><P>               The gel picture should have one band when the plasmid have no insert (the natural plasmid   has no SpeI cutting site) and two bands when the insert is in the plasmid (the MCS has one              cutting site for SpeI).</P><P> </P><P>Results:</P><P> </P><P>The agarose gel shows, that none of our colonies has got the MCS + pCat fragment.</P><P> </P><DIV class="col-lg-12"><DIV class="imagewrapper"><DIV class="imagewrappercaption"><STRONG>
                Figure 4: Digestion of pSB1C3 to test if MCS and pcat are inserted
            </STRONG><P class="basictext">The target fragment exhibits 2 bands, the non-cut fragment exhibits just one band. The reason for the larger bands (2 and 4) is unknown.
            </P></DIV></DIV></DIV><P> </P></DIV></DIV></DIV></DIV><DIV class="row"><DIV class="col-lg-12"><DIV class="notebookentry"><H3 class="notebookheading">▼2015-06-27  Transformation test for pSB1C3 + MCS and pcat</H3><DIV class="notebooktext"><P>Description:</P><P> </P><P>Procedures were done on 5 ml overnight E.coli culture with estimated MCS+pCat insert in pSB1C3</P><P>Five colonies were picked from the original plate and were given in 5 snapcaps with 5 ml LB medium each.</P><P>The 5 cultures are named 1-5</P><P> </P><P>Procedures:</P><P> </P><P>QIAprep Spin Miniprep Kit:</P><P> </P><P>Description:</P><P> </P><P>    Steps:</P><P> </P><OL><LI>Prepare o/n culture</LI></OL><P> </P><OL><LI>Perform mini prep according to manufacturers protocol</LI></OL><P> </P><P>Notes:</P><P> </P><P>After the miniprep 2 ml of overnight culture were transferred to 100 ml of fresh LB medium with 1:1000 Chloramphenicol</P><P> </P><P>Results:</P><P> </P><P>Nanodrop results:</P><P>1: c = 9,5 ng/µl</P><P>2: c = 7,7 ng/µl</P><P>3: c = 81,5 ng/µl</P><P>4: c = 48,5 ng/µl</P><P>5: c = 64 ng/µl</P><P> </P><P>A test digestion will be made with 3-5. The concentrations for 1-2 are too low.</P><P> </P><P>20 µl test-digest:</P><P> </P><P>Description</P><P> </P><P>    Steps:</P><P> </P><OL><LI>Set up reaction according to protocol:</LI></OL><P>                ddH2O for a final volume of 20 µl</P><P>                2 µl of 10x Reaction Buffer (e.g. NEB CutSmart)</P><P>                0.5 µl of selected Enzyme(s)</P><P>                ca. 1 µl of mini prep DNA (Range 200-1000 ng)</P><P> </P><OL><LI>Incubate at 37°C for 60'</LI></OL><P> </P><OL><LI>Load on gel (add loading dye first)</LI></OL><P> </P><P>Notes:</P><P> </P><P>               Digest with EcoRI and SpeI</P><P>               Heat inactivation at 65°C for 20 minutes</P><P>               The gel picture should have one band when the plasmid have no insert (the natural plasmid   has no SpeI cutting site) and two bands when the insert is in the plasmid (the MCS has one              cutting site for SpeI).</P><P> </P><P>Results:</P><P> </P><P>The agarose gel shows, that none of our colonies has got the MCS + pCat fragment.</P></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV><DIV class="container"><DIV class="content"><DIV class="row"><DIV class="col-lg-12"><DIV class="panel panel-default"><DIV class="panel-heading"><H3 class="basicheader">week number 25</H3></DIV><DIV class="panel-body"><DIV class="row"><DIV class="col-lg-12"><DIV class="notebookentry"><H3 class="notebookheading">▼2015-06-19  Quantitative Gel of the PCR of the Gibson assembly fragments</H3><DIV class="notebooktext"><P>Description:</P><P>We performed a gel quantification of DNA yielded in the PCR of our Gibson fragments.</P><P> </P><P>Procedures:</P><P> </P><P>Quantitative DNA gel:</P><P> </P><P>Description:</P><P> </P><P>Steps:</P><OL><LI>Prepare an evenly stained agarose gel.</LI><LI>Load a defined amount of the sample</LI><LI>Load the DNA Ladder according to manufacturers protocol (e.g. NEB 2-log: 10µl for ng values according to sheet)</LI><LI>Document and analyze the gel</LI></OL><P> </P><P>Notes:</P><P> </P><P>DNA samples were prepared by 1 hour of digestion with Dpn1 at 60 °C, followed by purification using a qiagen kit.</P><P>Samples were mixed with 3 parts of 6x loading dye, and loaded together with 2 log ladder.</P><P>Samples were: 1/2: PCR product of mCherry</P><P>3/4: CFTR testing construct</P><P>5/6: GFP</P><P>7/8: p413 backbone</P><P>Samples were yielded from PCR with Phusion polymerase.</P><P> </P><P>Results summary:</P><TABLE border="1" cellpadding="0" cellspacing="0"><TBODY><TR><TD style="width:86px"><P>Lane</P></TD><TD style="width:86px"><P>Lane 1</P></TD><TD style="width:86px"><P>Lane 2</P></TD><TD style="width:86px"><P>Lane 3</P></TD><TD style="width:86px"><P>Lane 4</P></TD><TD style="width:86px"><P>Lane 5</P></TD><TD style="width:86px"><P>Lane 6</P></TD></TR><TR><TD style="width:86px"><P>DNA</P></TD><TD style="width:86px"><P>2-log</P></TD><TD style="width:86px"><P>mCherry</P></TD><TD style="width:86px"><P>CFTR</P></TD><TD style="width:86px"><P>GFP</P></TD><TD style="width:86px"><P>P413</P></TD><TD style="width:86px"><P>2-log</P></TD></TR><TR><TD style="width:86px"><P>Basepairs</P></TD><TD style="width:86px"><P> </P></TD><TD style="width:86px"><P>~800</P></TD><TD style="width:86px"><P>200-300</P></TD><TD style="width:86px"><P>~800</P></TD><TD style="width:86px"><P>~6000</P></TD><TD style="width:86px"><P> </P></TD></TR><TR><TD style="width:86px"><P>DNA Yield</P></TD><TD style="width:86px"><P> </P></TD><TD style="width:86px"><P>~60 ng/µl</P></TD><TD style="width:86px"><P>~60 ng/µl</P></TD><TD style="width:86px"><P>~60 ng/µl</P></TD><TD style="width:86px"><P>~60 ng/µl</P></TD><TD style="width:86px"><P> </P></TD></TR></TBODY></TABLE><P> </P><P>Conclusion:</P><P>The bright bands yielded in this gel, in comparison to the previous quantitative gel of the PCR with OneTaq are indicative of a higher suitability of Phusion polymerase for constructs of this composition. Especially backbone p413 yielded a bright band under UV light in our present experiment, whereas in the previous experiment, there were no bands at all. This indicates, that we should choose Phusion for the amplification of larger fragments.</P><P>In the following quantification, all the bands yielded a concentration of about 60 ng/µl, fairly sufficient for a following Gibson Assembly.</P></DIV></DIV></DIV></DIV><DIV class="row"><DIV class="col-lg-12"><DIV class="notebookentry"><H3 class="notebookheading">▼2015-06-21  Colony PCR of E.coli transformated with Gibson-Plasmids</H3><DIV class="notebooktext"><P>Procedures:</P><P> </P><P>PCR:</P><P> </P><P>Description:</P><P> </P><P>       Materials and Chemicals:</P><P> </P><P>        PCR-tubes</P><P>        Forward primer</P><P>        Reverse primer</P><P>        Mastermix (dNTPs, Polymerase, buffer)</P><P>        Water</P><P>        Thermocycler</P><P> </P><P>        Endvolume: 10 µl</P><P> </P><P>Steps:</P><OL><LI>Pick colonies from plates. Solute one colony in about 20 µl of water.</LI><LI>Give concentrated mastermix (which should be diluted to 1x in the end (e.g. you need 5 µl of 2x mastermix for 10 µl)), primer (concentration between and µl). Fill the volume to 10 µl with the water with E.coli</LI><LI>Use the Thermocycler with the usual PCR program for at least 25 cycles.</LI><LI>If the result is positive, take the remaining water to seed LB-medium with our E.coli</LI></OL><P> </P><P>Notes:</P><P> </P><P>        Plasmids: p413-GPD</P><P>       Insert: mCherry-CFTR-Tags-GFP</P><P> </P><P>        Chemicals(one tube):</P><P>        0,4 µl forward Primer</P><P>        0,4 µl reverse Primer</P><P>        5 µl Mastermix</P><P>        4,2 µl water with E.coli</P><P> </P><P>Results:</P><P> </P><P>Culture 10 seems to be the only picked culture wich exhibits an insert</P><P> </P><DIV class="col-lg-12"><DIV class="imagewrapper"><DIV class="imagewrappercaption"><STRONG>
                Figure 1: Colony PCR of E.coli transformated with Gibson-Plasmids
            </STRONG><P class="basictext">
                Positive Colonies exhibit an insert with the CFTR-testconstruct that is about 1,5 kb long.
            </P></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></BODY></HTML>