<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_Heidelberg_notebook_pc skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:Heidelberg/notebook/pc</SPAN></H1><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><DIV class="container" id="menucontainer"><DIV id="navcontainer-right-outer"><DIV id="navcontainer-right-inner"><UL class="nav navbar-nav navbar-right"><LI><A href="https://2015.igem.org/Team:Heidelberg/Achievements" class="menubox"><P class="menutext">Achievements</P></A></LI><LI class="dropdown"><A href="https://2015.igem.org/Team:Heidelberg/Results" class="dropdown-toggle menubox" data-toggle="dropdown" role="button" aria-haspopup="true" aria-expanded="false"><P class="menutext">Results
                        </P></A></LI><LI class="dropdownbox"><A href="https://2015.igem.org/Team:Heidelberg/Results#Medalfulfillments"><P class="menutext dropdown-text">Medal fulfillments</P></A></LI><LI class="dropdownbox"><A href="https://2015.igem.org/Team:Heidelberg/Results#AB"><P class="menutext dropdown-text">AptaBodies</P></A></LI><LI class="dropdownbox"><A href="https://2015.igem.org/Team:Heidelberg/Results#RTSMS"><P class="menutext dropdown-text">Real time S.M.S.</P></A></LI><LI class="dropdownbox"><A href="https://2015.igem.org/Team:Heidelberg/Results#LandV"><P class="menutext dropdown-text">Cystic fibrosis</P></A></LI><LI class="dropdownbox"><A href="https://2015.igem.org/Team:Heidelberg/Results#CF"><P class="menutext dropdown-text">Labeling and vizualization</P></A></LI><LI class="dropdownbox"><A href="https://2015.igem.org/Team:Heidelberg/Results#Modelling"><P class="menutext dropdown-text">Modeling and Software validation</P></A></LI><LI class="dropdownbox"><A href="https://2015.igem.org/Team:Heidelberg/Results/Standardization"><P class="menutext dropdown-text">Standardization</P></A></LI></UL><LI class="dropdown"><A href="https://2015.igem.org/Team:Heidelberg/Overview" class="dropdown-toggle menubox" data-toggle="dropdown" role="button" aria-haspopup="true" aria-expanded="false"><P class="menutext">Project
                        </P></A></LI><LI class="dropdownbox"><A href="https://2015.igem.org/Team:Heidelberg/Overview"><P class="menutext dropdown-text">Overview</P></A></LI><LI class="dropdownbox"><A href="https://2015.igem.org/Team:Heidelberg/Project/AB"><P class="menutext dropdown-text">AptaBodies</P></A></LI><LI class="dropdownbox"><A href="https://2015.igem.org/Team:Heidelberg/project/rtsms"><P class="menutext dropdown-text">Real time S.M.S.</P></A></LI><LI class="dropdownbox"><A href="https://2015.igem.org/Team:Heidelberg/project/cf"><P class="menutext dropdown-text">Cystic fibrosis</P></A></LI><LI class="dropdownbox"><A href="https://2015.igem.org/Team:Heidelberg/project/rd"><P class="menutext dropdown-text">Labelling and Visualization</P></A></LI><LI class="dropdownbox"><A href="https://2015.igem.org/Team:Heidelberg/project/hlpd"><P class="menutext dropdown-text">HRP based detection</P></A></LI><LI class="dropdown"><A href="https://2015.igem.org/Team:Heidelberg/Software" class="dropdown-toggle menubox" data-toggle="dropdown" role="button" aria-haspopup="true" aria-expanded="false"><P class="menutext">Software
                        </P></A></LI><LI class="dropdownbox"><A href="https://2015.igem.org/Team:Heidelberg/software/maws"><P class="menutext dropdown-text">MAWS</P></A></LI><LI class="dropdownbox"><A href="https://2015.igem.org/Team:Heidelberg/software/jaws"><P class="menutext dropdown-text">JAWS</P></A></LI><LI class="dropdownbox"><A href="
https://2015.igem.org/Team:Heidelberg/WikiTools"><P class="menutext dropdown-text">Wiki Tools</P></A></LI><LI class="dropdown"><A href="https://2015.igem.org/Team:Heidelberg/notebook/mm" class="dropdown-toggle menubox" data-toggle="dropdown" role="button" aria-haspopup="true" aria-expanded="false"><P class="menutext">Notebook
                        </P></A></LI><LI class="dropdownbox"><A href="https://2015.igem.org/Team:Heidelberg/notebook/ab"><P class="menutext dropdown-text">AptaBodies</P></A></LI><LI class="dropdownbox"><A href="https://2015.igem.org/Team:Heidelberg/notebook/rtsms"><P class="menutext dropdown-text">Real time S.M.S.</P></A></LI><LI class="dropdownbox"><A href="https://2015.igem.org/Team:Heidelberg/notebook/cf"><P class="menutext dropdown-text">Cystic fibrosis</P></A></LI><LI class="dropdownbox"><A href="https://2015.igem.org/Team:Heidelberg/notebook/rd"><P class="menutext dropdown-text">Labeling and vizualization</P></A></LI><LI class="dropdownbox"><A href="https://2015.igem.org/Team:Heidelberg/notebook/pc"><P class="menutext dropdown-text">Pulse chase</P></A></LI><LI class="dropdownbox"><A href="https://2015.igem.org/Team:Heidelberg/notebook/mm"><P class="menutext dropdown-text">Materials &amp; Methods</P></A></LI><LI class="dropdownbox"><A href="https://2015.igem.org/Team:Heidelberg/Safety"><P class="menutext dropdown-text">Safety</P></A></LI><LI><A href="https://2015.igem.org/Team:Heidelberg/Parts" class="menubox"><P class="menutext">Parts</P></A></LI><LI class="dropdown"><A href="https://2015.igem.org/Team:Heidelberg/Team" class="dropdown-toggle menubox" data-toggle="dropdown" role="button" aria-haspopup="true" aria-expanded="false"><P class="menutext">Team
                        </P></A></LI><LI class="dropdownbox"><A href="https://2015.igem.org/Team:Heidelberg/Team"><P class="menutext dropdown-text">Members</P></A></LI><LI class="dropdownbox"><A href="https://2015.igem.org/Team:Heidelberg/Supervisors"><P class="menutext dropdown-text">Supervisors</P></A></LI><LI class="dropdownbox"><A href="https://2015.igem.org/Team:Heidelberg/Collaborations"><P class="menutext dropdown-text">Collaboration</P></A></LI><LI class="dropdownbox"><A href="https://2015.igem.org/Team:Heidelberg/Attributions"><P class="menutext dropdown-text">Attributions</P></A></LI><LI class="dropdownbox"><A href="https://2015.igem.org/Team:Heidelberg/Sponsors"><P class="menutext dropdown-text">Sponsors</P></A></LI><LI class="dropdown"><A href="https://2015.igem.org/Team:Heidelberg/Modeling" class="dropdown-toggle menubox" data-toggle="dropdown" role="button" aria-haspopup="true" aria-expanded="false"><P class="menutext">Modeling
                        </P></A></LI><LI class="dropdownbox"><A href="https://2015.igem.org/Team:Heidelberg/Modeling/aptakinetics"><P class="menutext dropdown-text">Aptamer Sensors</P></A></LI><LI class="dropdownbox"><A href="https://2015.igem.org/Team:Heidelberg/Modeling/rtsms"><P class="menutext dropdown-text">In vitro transcription</P></A></LI><LI><A href="https://2015.igem.org/Team:Heidelberg/Practices" class="menubox"><P class="menutext">Outreach</P></A></LI></DIV></DIV></DIV><NAV id="myNavmenu" class="navmenu navmenu-default navmenu-fixed-left offcanvas" role="navigation"><A class="navmenu-brand" href="https://2015.igem.org/Team:Heidelberg">iGEM Team Heidelberg 2015</A><UL class="nav navmenu-nav"><LI><A href="https://2015.igem.org/Team:Heidelberg/Achievements">Achievements</A></LI><LI><A href="https://2015.igem.org/Team:Heidelberg/Results">Results</A></LI><LI><A href="https://2015.igem.org/Team:Heidelberg/Overview">Project</A></LI><LI><A href="https://2015.igem.org/Team:Heidelberg/Software">Software</A></LI><LI><A href="https://2015.igem.org/Team:Heidelberg/notebook/mm">Notebook</A></LI><LI><A href="https://2015.igem.org/Team:Heidelberg/Parts">Parts</A></LI><LI><A href="https://2015.igem.org/Team:Heidelberg/Team">Team</A></LI><LI><A href="https://2015.igem.org/Team:Heidelberg/Modeling">Modeling</A></LI><LI><A href="https://2015.igem.org/Team:Heidelberg/Practices">Outreach</A></LI></UL></NAV><DIV class="container"><DIV class="content"><DIV class="row"><DIV class="col-lg-12"><DIV class="panel panel-default"><DIV class="panel-heading"><H3 class="basicheader">week number 26</H3></DIV><DIV class="panel-body"><DIV class="row"><DIV class="col-lg-12"><DIV class="notebookentry"><H3 class="notebookheading">▼2015-06-22  E.coli Trafo with Plasmids form the iGEM registry</H3><DIV class="notebooktext"><P>To build the protein construct the mTagBFP BBa_K592100 from the iGEM registry were used.</P><P>1. With a pipette tip, punch a hole through the foil cover into the corresponding well of the part that you want. Do not remove the foil cover.</P><P>2. Pipette 10µl of dH2O (distilled water) into the well. Pipette up and down a few times and let sit for 5 minutes to make sure the dried DNA is fully resuspended.</P><P>3. Thawing 50µl chemical competent E. coli on ice.</P><P>4. Add to 50µl competent cells:</P><P>1µl DNA from the registry</P><P>10µl KCM 5x</P><P>39µl H2O</P><P>5. Incubate on ice for 30 minutes</P><P>6. Heat shock at 42°C for 2 minutes</P><P>7. Incubate on ice for 2 minutes</P><P>8. Add 900 µl of LB or 2x YT Medium</P><P>9. Incubate on 37°C for 60min</P><P>10. Centrifuge 5min at 1000g</P><P>11. Take 900µl of supernatant and throw away</P><P>12. Resuspend pellet in remaining media</P><P>13. Plate out on agar with the appropriate antibiotic and grow overnight at 37°C</P><P> </P></DIV></DIV></DIV></DIV><DIV class="row"><DIV class="col-lg-12"><DIV class="notebookentry"><H3 class="notebookheading">▼2015-06-24  QIAprep Spin Miniprep Kit</H3><DIV class="notebooktext"><P>Miniprep with mTagBFP BBa-K592100 and the pUB23-S-ßGal plasmid was performed.</P><P>1. Prepare o/n culture</P><P>2. Perform mini prep according to manufacturer’s protocol</P><P>Durchführung</P></DIV></DIV></DIV></DIV><DIV class="row"><DIV class="col-lg-12"><DIV class="notebookentry"><H3 class="notebookheading">▼2015-06-25  E. coli glycerol stocks</H3><DIV class="notebooktext"><P>Glycerol stocks from E.coli containing mTag BFP BBa_K592100 or pUB23-S-ßGal were made.</P><P>1. Grow up an overnight culture of strains of interest</P><P>2. Transfer 500µl into a safe lock reaction tube</P><P>3. Add 500µl of 40% sterile glycerol solution</P><P>4. Freeze slowly at -80°C</P><P>Durchführung</P></DIV></DIV></DIV></DIV><DIV class="row"><DIV class="col-lg-12"><DIV class="notebookentry"><H3 class="notebookheading">▼2015-06-25  Fragment amplification for protein construct (PCR)</H3><DIV class="notebooktext"><P>To obtain the required Gibson Fragments (Backbone (p413-GPD), mTagBFP, Ubiquitin, sfGFP) PCRs were performed.</P><P> </P><P>reaction mixture:</P><P>12.5µl Q5 High-Fidelity 2x Master Mix</P><P>0.5µl DNA</P><P>1.25µl fwd primer 10µM</P><P>1.25µl rev primer 10µM</P><P>9.5µl H2O</P><P> </P><P>Program:</P><P>95°C 2min</P><P>95°C 30sec</P><P>63-66°C 30sec (mTagBFP: 63°C; Ub: 65°C; sfGFP: 64°C; p415-GDP: 66°C)</P><P>72°C 30sec</P><P>72°C 5min</P><P>35 cycles</P><P> </P><P>gel:</P><P>TAE buffer</P><P>1% w/v agarose</P><P> </P><P>loading samples:</P><P>5µl Ladder</P><P>25µl PCR sample</P><P>5µl Loading buffer</P></DIV></DIV></DIV></DIV><DIV class="row"><DIV class="col-lg-12"><DIV class="notebookentry"><H3 class="notebookheading">▼2015-06-27  Fragment amplification for protein construct (PCR) (continued)</H3><DIV class="notebooktext"><P>Results</P><P>PCRs for Gibson fragments mTagBFP, Ubiquitin and sfGFP worked because a bands are visible with the expected size. No p415-GPD product was visible. Therefore the p415-GPD PCR need to be optimised.</P><P><STRONG>Gel Extraction</STRONG></P><P>mTagBFP, Ubiquitin and sfGFP fragments were separated in 1% agaros.</P><P>Gel extraction was performed according to QIAGEN QIAquick Gel Extraction Protocol. (Elution in 30µl EB)</P></DIV></DIV></DIV></DIV><DIV class="row"><DIV class="col-lg-12"><DIV class="notebookentry"><H3 class="notebookheading">▼2015-06-27  Phusion PCR with DMSO and GC Buffer</H3><DIV class="notebooktext"><P>To amplify the p415-GPD plasmid with Gibson overhangs an optimized gradient PCR program was used.</P><P> </P><P>reaction mixture:</P><P>total:10µl</P><P>5.0µl Phusion 2x Master Mix</P><P>0.2µl DNA</P><P>0.5µl xxJD007xx 10µM</P><P>0.5µl xxJD008xx10µM</P><P>0.5µl DMSO</P><P> </P><P>1.3 H2O</P><P> </P><P>programm:</P><P>(PHGRAD)</P><P>95°C 2:00min</P><P>95°C 0:30min</P><P>58°C-70°C 0:30min</P><P>72°C 3:00min</P><P>72°C 5:00</P><P>35 cycles</P><P> </P><P>gel:</P><P>TAE buffer</P><P>1% w/v agarose</P><P> </P><P>loading samples:</P><P>5µl PCR sample</P><P>2µl Loading buffer</P><P> </P><P>Results</P><P>no product. PCR need to be repeated.</P><P> </P></DIV></DIV></DIV></DIV><DIV class="row"><DIV class="col-lg-12"><DIV class="notebookentry"><H3 class="notebookheading">▼2015-06-27  p415-GPD test digest</H3><DIV class="notebooktext"><P>To check if the right plasmid is in the tube a KpnI digest was performed.</P><P>Set up reaction according to protocol:</P><UL><LI>ddH2O for a final volume of 20 µl</LI><LI>2 µl of 10x 1.1 Reaction Buffer</LI><LI>0.5 µl of KpnI</LI><LI>1 µl of DNA</LI></UL><P>Incubate at 37°C for 60'</P><P>Load on gel (add loading dye first)</P><P> </P><P>Results</P><P>The gel shows the expected bands. The p415 plasmid is in the tube.</P><P> </P><P> </P></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV><DIV class="container"><DIV class="content"><DIV class="row"><DIV class="col-lg-12"><DIV class="panel panel-default"><DIV class="panel-heading"><H3 class="basicheader">week number 27</H3></DIV><DIV class="panel-body"><DIV class="row"><DIV class="col-lg-12"><DIV class="notebookentry"><H3 class="notebookheading">▼2015-06-29  p415 PCR</H3><DIV class="notebooktext"><P>Another p415 PCR was performed to amplify p415 for Gibson assembly.</P><P> </P><P>reaction mixture:</P><P>12.5µl Phusion 2x Master Mix</P><P>1.2µl DNA 1:10 50ng</P><P>1.25µl xxJD007xx 10µM</P><P>1.25µl xxJD008xx 10µM</P><P>one with 1.25µl DMSO one without</P><P>add H2O to 25µl</P><P> </P><P>program:</P><P>95°C 5min</P><P>98°C 30sec</P><P>63°C 30sec</P><P>72°C 2min</P><P>72°C 5min</P><P>35 cycles</P><P> </P><P>Results</P><P>still not working</P><P> </P></DIV></DIV></DIV></DIV><DIV class="row"><DIV class="col-lg-12"><DIV class="notebookentry"><H3 class="notebookheading">▼2015-07-02  p413 and sfGFP PCR</H3><DIV class="notebooktext"><P>Because the amplification problems with the p415 plasmid could not be solved, the p413-GPD plasmid was used for the protein construct. Therefor sfGFP and p413 were amplified with new primers.</P><P>To be created: sfGFP, p413-GPD Gibson fragments</P><P>PCR (25 µl):</P><P>12.5 µl Q5 Master Mix</P><P>0.5 µl DNA Template (pMaM17; p413-GPD)</P><P>1.25µl fwd primer 10µM</P><P>1.25µl rev primer 10µM</P><P>9.5µl H2O</P><P>Cycler Program:</P><P>Initial denaturation: 2'@95°C</P><P>35x:</P><P>Denaturation: 30''@95°C</P><P>Annealing: 30'' (sfGFP: 64°C p413-GPD: 66°C)</P><P>Elongation: 3'@72°C</P><P>Final Elongation: 5'@72°C</P><P>Store @4°C</P><P>Results</P><P>sfGFP was used for gel extraction. p413 PCR need to be repeated.</P><P> </P><P> </P><P> </P><P>Gel Extraction</P><P>The sfGFP Gibsonfragment was separated in 1% Agarose.</P><P>Gel extraction was performed according to QIAGEN QIAquick Gel Extraction Protocol. (Elution in 30µl EB)</P><P> </P></DIV></DIV></DIV></DIV><DIV class="row"><DIV class="col-lg-12"><DIV class="notebookentry"><H3 class="notebookheading">▼2015-07-03  p413 PCR repeated </H3><DIV class="notebooktext"><P>p413 amplification</P><P>PCR (25 µl):</P><P>12.5 µl Q5 Master Mix</P><P>0,5 µl DNA Template (p413-GPD)</P><P>1.25µl fwd primer 10µM</P><P>1.25µl rev primer 10µM</P><P>9.5µl H2O</P><P>Cycler Program:</P><P>Initial denaturation: 2'@95°C</P><P>35x:</P><P>Denaturation: 30''@95°C</P><P>Annealing: 30'' (p413-GPD: 64°C)</P><P>Elongation: 3'@72°C</P><P>Final Elongation: 5'@72°C</P><P>Store @4°C</P><P> </P><P> </P><P> Results</P><P>The p413 products were used for gel extraction.</P><P> </P><P>Gel Extraction</P><P>The backbone p413 was separated in 1% Agarose.</P><P>Gel extraction was performed according to QIAGEN QIAquick Gel Extraction Protocol. (Elution in 30µl EB)</P><P>After extraction due to minimal concentration the sample was lyophilized to obtain DNA pellet.</P><P> </P></DIV></DIV></DIV></DIV><DIV class="row"><DIV class="col-lg-12"><DIV class="notebookentry"><H3 class="notebookheading">▼2015-07-04  Gibson Assembly</H3><DIV class="notebooktext"><P>Samples mTagBFP, Ubiquitin, sfGFP and p413 were used for Gibson assembly. The reaction was carried out in the p413 tube containing the p413 pellet.</P><P> </P><TABLE border="0" cellpadding="0" cellspacing="0" style="width:339px"><TBODY><TR><TD style="height:16px; width:121px"><P>aIG</P></TD><TD style="height:16px; width:56px"><P>Size [bp]</P></TD><TD style="height:16px; width:75px"><P>c [ng/µl]</P></TD><TD style="height:16px; width:87px"><P>Volume [µl]</P></TD></TR><TR><TD style="height:16px; width:121px"><P>GFP</P></TD><TD style="height:16px; width:56px"><P>714</P></TD><TD style="height:16px; width:75px"><P>16,8</P></TD><TD style="height:16px; width:87px"><P>2,55</P></TD></TR><TR><TD style="height:16px; width:121px"><P>Ub</P></TD><TD style="height:16px; width:56px"><P>228</P></TD><TD style="height:16px; width:75px"><P>13,6</P></TD><TD style="height:16px; width:87px"><P>1,01</P></TD></TR><TR><TD style="height:16px; width:121px"><P>mTagBFP</P></TD><TD style="height:16px; width:56px"><P>731</P></TD><TD style="height:16px; width:75px"><P>34,5</P></TD><TD style="height:16px; width:87px"><P>1,27</P></TD></TR></TBODY></TABLE><P> </P><P>Add water to a final volume of 10µl.</P><P>Add 10µl Gibson Assembly Master Mix.</P><P>Incubate for 60min at 50°C</P><P> </P><P>2.5µl DNA was used for KCM Transformation</P><P> </P><P>KCM Transformation</P><P>1. Take 50µl chemical competent E. coli from -80 freezer and thaw on ice</P><P>2. Add (as master mix):
2,5µl DNA</P><P>10µl KCM 5x</P><P>37,5µl H2O</P><P>3. Incubate on ice for 30 minutes</P><P>4. Heat shock at 42°C for 1 minute</P><P>5. Incubate on ice for 2 minutes</P><P>6. Add 900 µl of LB or 2x YT Medium</P><P>7. Incubate on 37°C for 60min</P><P>8. Centrifuge 5min at 1000g</P><P>9. Take 900µl of supernatant and throw away</P><P>10. Resuspend pellet in remaining media</P><P>11. Plate out on agar with antibiotics</P><P> </P><P>Results</P><P>No E.coli were transformed. Therefore Gibson fragments were checked on an agarose gel.</P><P> </P></DIV></DIV></DIV></DIV><DIV class="row"><DIV class="col-lg-12"><DIV class="notebookentry"><H3 class="notebookheading">▼2015-07-04  Fragment test for Gibson</H3><DIV class="notebooktext"><P>p413, mTagBFP, Ub,GFP samples with Gibson overhangs after Gel Extraction were loaded on an 1% agarose gel.</P><P> </P><P>Results</P><P>No fragments can be detected. Gibson PCR need to be repeated.</P></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV><DIV class="container"><DIV class="content"><DIV class="row"><DIV class="col-lg-12"><DIV class="panel panel-default"><DIV class="panel-heading"><H3 class="basicheader">week number 28</H3></DIV><DIV class="panel-body"><DIV class="row"><DIV class="col-lg-12"><DIV class="notebookentry"><H3 class="notebookheading">▼2015-07-07  Fragment amplification</H3><DIV class="notebooktext"><P>To obtain the required Gibson Fragments (Backbone (p413-GPD), mTagBFP, Ubiquitin, sfGFP) PCRs were performed.</P><P>For the backbone we performed the reaction with Thermo 2x Phusion Flash PCR Mastermix and with NEB Q5 2x High-Fidelity Mastermix.</P><P>The insert fragments were amplified using the Thermo 2x Phusion Flash PCR Mastermix.</P><P>All PCRs were performed as 25 µl reactions and set up in duplicates.</P><P>For the backbone:</P><P>PCR (25 µl):</P><P>12,5 µl NEB Q5 2x High-Fidelity Mastermix/ Thermo 2x Phusion Flash PCR Mastermix</P><P>0,25 µl DNA Template (p413-GPD)</P><P>1.25µl fwd primer 10µM</P><P>1,25µl rev primer 10µM</P><P>9,75 µl H2O</P><P>Cycler Program:</P><P>Initial denaturation: 2'@95°C</P><P>35x:</P><P>Denaturation: 30''@95°C</P><P>Annealing: 30''@66°C</P><P>Elongation: 3'@72°C</P><P> </P><P>Final Elongation: 5'@72°C</P><P>Hold @4°C</P><P> </P><P>For the inserts:</P><P>PCR (25 µl):</P><P>12,5 µl 2x Phusion Flash PCR Mastermix</P><P>0,5 µl DNA Template (BBa_K592100 prepped from Registry, pUB23-S-ßGal, pMaM17)</P><P>1.25µl fwd primer 10µM</P><P>1,25µl rev primer 10µM</P><P>9.5µl H2O</P><P>Cycler Program:</P><P>Initial denaturation: 2'@95°C</P><P>35x:</P><P>Denaturation: 30''@95°C</P><P>Annealing: 30'' (mTagBFP: 63°C; Ub: 65°C sfGFP: 64°C)</P><P>Elongation: 30''@72°C</P><P> </P><P>Final Elongation: 5'@72°C</P><P>Hold @4°C</P><P> The products will then be separated on a gel and purified.</P><P> </P></DIV></DIV></DIV></DIV><DIV class="row"><DIV class="col-lg-12"><DIV class="notebookentry"><H3 class="notebookheading">▼2015-07-08  Fragments gel</H3><DIV class="notebooktext"><P>To purify the fragments of the protein construct for Gibson assembly we chose to seperate on a gel: 
The samples from the PCR are loaded together with 6x NEB Purple Loading Dye on a 1% agarose TAE gel. 
Load scheme (25 µl per PCR reaction): 
#1: 5 µl NEB 2-Log DNA Ladder 
#2: p413-GPD (amplified with Phusion Flash) 
#3: p413-GPD (amplified with Phusion Flash) 
#4: p413-GPD (amplified with NEB Q5) 
#5: p413-GPD (amplified with NEB Q5) 
#6: mTagBFP (amplified with Phusion Flash) 
#7: mTagBFP (amplified with Phusion Flash) 
#8: Ubiquitin (amplified with Phusion Flash) 
#9: Ubiquitin (amplified with Phusion Flash) 
#10: sfGFP (amplified with Phusion Flash) 
#11: sfGFP (amplified with Phusion Flash)</P></DIV></DIV></DIV></DIV><DIV class="row"><DIV class="col-lg-12"><DIV class="notebookentry"><H3 class="notebookheading">▼2015-07-08  fragment test for Gibson</H3><DIV class="notebooktext"><P>p413, mTagBFP, Ub,GFP samples with Gibson overhangs after Gel Extraction were loaded on a gel.</P><P>Results
BFP, Ub and GFP worked, p413 need to be repeated. </P></DIV></DIV></DIV></DIV><DIV class="row"><DIV class="col-lg-12"><DIV class="notebookentry"><H3 class="notebookheading">▼2015-07-10  p413 PCR</H3><DIV class="notebooktext"><P>50µl in total:</P><P> 22.5 µl H2O</P><P>10 µl 5x buffer</P><P>2 µl 50 mM MgCl2</P><P>2x 2.5 µl primer</P><P>5 µl 10 mM dNTPs</P><P>5 µl 5 M betaine</P><P>0.25 µl template (~80 ng)</P><P>0.5 µl Velocity polymerase</P><P> </P><P>1: 97 °C: 2 min</P><P>2: 97 °C: 30 s</P><P>3: 60 °C: 30 s</P><P>4: 72 °C: 3.5 min; -&gt; 2 34x</P><P>5: 72 °C: 10 min</P><P>6: 4 °C inf</P><P> </P><P> </P></DIV></DIV></DIV></DIV><DIV class="row"><DIV class="col-lg-12"><DIV class="notebookentry"><H3 class="notebookheading">▼2015-07-10  p413 precipitation</H3><DIV class="notebooktext"><P>150µl absolute ethanol (freezer)</P><P>25µl acetat</P><P>mix gently</P><P>incubate for 30min by 4°C</P><P>centrifuge 20min by 4°C</P><P>remove supernatant</P><P>add 100µl 70% enthanol</P><P>remove enthanol and let pellet dry</P><P>solve pellet in 20µl ddH2O</P><P> </P></DIV></DIV></DIV></DIV><DIV class="row"><DIV class="col-lg-12"><DIV class="notebookentry"><H3 class="notebookheading">▼2015-07-11  DpnI digest of p413</H3><DIV class="notebooktext"><P>20µl p413 precipitation
2µl CutSmart
1.5µl DpnI
37°C 60min
65°C 1min</P><P>gel:
1µl p413 digested
1µl loading buffer
5µl 2log Ladder </P></DIV></DIV></DIV></DIV><DIV class="row"><DIV class="col-lg-12"><DIV class="notebookentry"><H3 class="notebookheading">▼2015-07-11  Gibson assembly</H3><DIV class="notebooktext"><P>0.62µl p413 digested</P><P>3.01µl mTagBFP 1:10</P><P>2.19µl Ubiquintin 1:10</P><P>3.09µl sfGFP 1:10</P><P>1.09µl H2O</P><P>10µl Gibson MM</P><P>60min 50°C</P></DIV></DIV></DIV></DIV><DIV class="row"><DIV class="col-lg-12"><DIV class="notebookentry"><H3 class="notebookheading">▼2015-07-11  E.coli transformation</H3><DIV class="notebooktext"><P>Take 50µl chemical competent E. coli from -80 freezer and thaw on ice</P><P>Add (as master mix):</P><P>2,5µl DNA</P><P>10µl KCM 5x</P><P>37,5µl H2O</P><P> </P><P>Incubate on ice for 30 minutes</P><P>Heat shock at 42°C for 1 minute</P><P>Incubate on ice for 2 minutes</P><P>Add 900 µl of LB or 2x YT Medium</P><P>Incubate on 37°C for 60min</P><P>Centrifuge 5min at 1000g</P><P>Take 900µl of supernatant and throw away</P><P>Resuspend pellet in remaining media</P><P>Plate out on agar with antibiotics (1:1 / 1:10)</P><P>Results</P><P>No colonies visible</P></DIV></DIV></DIV></DIV><DIV class="row"><DIV class="col-lg-12"><DIV class="notebookentry"><H3 class="notebookheading">▼2015-07-12  Colony PCR</H3><DIV class="notebooktext"><P>5µl OneTag MasterMix</P><P>0.5µl primer 1:10</P><P>0.5µl primer 1:10</P><P>4µl H2O</P><P>
Program:</P><P>3min 95°C</P><P>0:30min 95°C</P><P>0:30min 51°C</P><P>2min 68°C</P><P>3min 68°C</P><P>∞ 4°C</P></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV><DIV class="container"><DIV class="content"><DIV class="row"><DIV class="col-lg-12"><DIV class="panel panel-default"><DIV class="panel-heading"><H3 class="basicheader">week number 29</H3></DIV><DIV class="panel-body"><DIV class="row"><DIV class="col-lg-12"><DIV class="notebookentry"><H3 class="notebookheading">▼2015-07-14  Yeast trafo</H3><DIV class="notebooktext"><P>Because the protein construct could not be assembled via Gibson in E. coli a yeast transformation with the Gibson fragments was performed.
p413-GPD: 750ng (25µl)
GFP 500ng
Ubi 200ng
BFP 500ng
50µl competent yeast
300µl PEG
Add fragments and PEG to competent yeast
Incubate 40min at 42°C
Centrifuge at 500g for 3min
Remove supernatant
Resolve pellet in 100µl H20
Plate on –His plates 1:100 and rest</P><P>GFP fluorescence could be detected </P></DIV></DIV></DIV></DIV><DIV class="row"><DIV class="col-lg-12"><DIV class="notebookentry"><H3 class="notebookheading">▼2015-07-15  Gibson assembly of the protein construct after PCR purification of the DpnI digest</H3><DIV class="notebooktext"><P>To increase the Gibson efficiency we purified our DpnI digest of the Backbone (p413-GPD) with the QIAGEN QIAquick PCR Purification Kit. Purification was performed according to manufacturers protocol. DNA was eluted with 30 µl H2O. (obtained concentration: 30.3 ng/µL)</P><P>With the insert fragments kept and the purified Backbone we performed a 20µL Gibson assembly reaction.</P><TABLE border="1" cellpadding="0" cellspacing="0"><TBODY><TR><TD style="width:302px"><P>Solution</P></TD><TD style="width:302px"><P>Amount</P></TD></TR><TR><TD style="width:302px"><P>NEB Gibson Assembly Master Mix</P></TD><TD style="width:302px"><P>10 µL</P></TD></TR><TR><TD style="width:302px"><P>Backbone (p413-GPD)</P></TD><TD style="width:302px"><P>2.18 µL</P></TD></TR><TR><TD style="width:302px"><P>sfGFP (1:10 dilution)</P></TD><TD style="width:302px"><P>2.91 µL</P></TD></TR><TR><TD style="width:302px"><P>Ubiquitin (1:10 dilution)</P></TD><TD style="width:302px"><P>2.06 µL</P></TD></TR><TR><TD style="width:302px"><P>mTagBFP (1:10 dilution)</P></TD><TD style="width:302px"><P>2.84 µL</P></TD></TR></TBODY></TABLE><P> </P><P>Incubation: 1h at 50 °C</P><P>The reaction was transformed in aliquots of 0.5 µL and 2 µL into E.coli TOP10</P><P>Results:</P><P>On the plate transformed with 0.5 µL of Gibson reaction no colonies were observed. The plate with 2 µL of Gibson mix carried 3 colonies that went into colony PCR screening.</P></DIV></DIV></DIV></DIV><DIV class="row"><DIV class="col-lg-12"><DIV class="notebookentry"><H3 class="notebookheading">▼2015-07-15  Aussagekräftiger Titel</H3><DIV class="notebooktext"><P> </P><P>Equipment and reagents</P><P>TE: 10 mM Tris-HCl pH 7.5, 1 mM EDTA add 1.25µl of 10mg/ml Zymolyase and 875µl ß mercaptoethanol to 250µl TE before use</P><P>S buffer: 10 mM K2H PO4 pH 7.2, 10mM EDTA add 1.25µl of 10mg/ml Zymolyase and 875µl ß mercaptoethanol to 250µl S buffer before use</P><P>Lysis solution: 25mM Tris-HCl pH 7.5, 25 mM EDTA, 2.5% SDS</P><P>3 M natrium acetate pH 5.5</P><P> </P><P>1. Scrape a large mass of yeast from a plate and resuspend it in 1ml of TE in a microcentrifuge tube (the OD600 of this suspension should be between two and five). Yeast from a fresh, two to three day-old plate work best. The yeast can also be obtained from a 2 ml overnight liquid culture.</P><P>2. Spin briefly to pellet the cells. Resuspend the yeast in 250µl of S buffer with Zymolyase and ß mercaptoethanol.</P><P>3. Incubate at 37°C for 30min</P><P>4. Add 55µl lysis solution. Vortex to mix.</P><P>5. Incubate at 65°C for 30min.</P><P>6. Add 83 µl 3 M sodium acetate. Chill on ice for 10 min</P><P>7. Spin in a microcentrifuge for 10 min. Pour the supernatant into a new tube.</P><P>8. Precipitate the DNA by adding 400µl cold ethanol. Incubate on ice for 10 min, spin for 10 min and pour off the supernatant.</P><P>9. Wash the pellet by centrifugation with 0.5 ml of 70% ethanol and dry the pellet.</P><P>10. Dissolve the pellet in 40 µl sterile water. Use 1-2 µl of this crude yeast mini-prep to transform E. coli by electroporation.</P><P> </P></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV><DIV class="container"><DIV class="content"><DIV class="row"><DIV class="col-lg-12"><DIV class="panel panel-default"><DIV class="panel-heading"><H3 class="basicheader">week number 30</H3></DIV><DIV class="panel-body"><DIV class="row"><DIV class="col-lg-12"><DIV class="notebookentry"><H3 class="notebookheading">▼2015-07-25  Yeast cell lysates using TCA</H3><DIV class="notebooktext"><P>Cell sediment from 2-3 OD of cells was resuspended in 1 ml of cold water. Alternatively, resuspend cells (a match head) from a fresh plate in 1 ml water. Alternatively, just use 1 ml of your cells in the medium (does only work for synthetic media, rich media contain too much  TCA precipitable material). Add 150 µl of 1.85 M NaOH, 7.5% ß-mercaptoethanol to the resuspended cells. Incubate 10 min on ice. Than add 150 µl 55% TCA (w/v) to the resuspended cells. Incubate for 10 min on ice. Centrifuge at 4°C for 15 min in a microfuge (14000 rpm). Remove supernatant, centrifuge again and remove remaining supernatant. Resuspend precipitated proteins in 100-200 µl HU-buffer by vortexing and heating sample for 10-15 min at 65°C. Spin samples for 5-10 min at RT, full speed in Eppi centrifuge. Load sample (10-15 µl) onto SDS-PAGE.</P><P>HU-buffer (high-urea buffer):
8 M urea, 5% SDS, 200 mM NaHPO4 pH 6.8, 0.1 mM EDTA, bromphenol blue (according  to your taste), freeze at –20°C, before use, adjust to 15mg/ml DTT</P></DIV></DIV></DIV></DIV><DIV class="row"><DIV class="col-lg-12"><DIV class="notebookentry"><H3 class="notebookheading">▼2015-07-25  Western Blot</H3><DIV class="notebooktext"><P>15µl Cell lysates were loaded on a 12% SDS Gel:
Resolivng Gel:
6.6 ml water
8 ml acrylamide mix 30
5 ml Tris 1.5 M pH 8.8
200µl 10% SDS
200µl 10% APS
20µl TEMED</P><P>Stacking Gel:
3.4 ml water
830 µl Acrylamid mix 30
630 µl Tris pH 6.8
50 µl 10% SDS
50µl 10% APS
5 µl TEMED</P><P>
Western Blot
Cut pvdf membrane (9x6cm) and Blotting paper to the dimensions of the gel. One membrane and two pieces of extra thick filter paper per gel are needed
Activate membrane 5min in Methanol
Equilibrate gel in transfer buffer for 5min
Assembly:
1. Place a pre-soaked sheet of extra thick filter paper onto the platinum anode. Roll a pipet or test tube over the surface of the filter paper (like a rolling pin) to exclude all air bubbles.
2. Carefully place the equilibrated gel on top of the transfer membrane, aligning the gel on the center of the membrane. Transfer will be incomplete if any portion of the gel is outside the blotting media. Roll out all air bubbles.
3. Place the other sheet of pre-soaked filter paper on top of the gel, carefully removing air bubbles from between the gel and filter paper.
4. run Blot at 25 V for 1h</P><P>Prepare around 50ml 5% milk in TBT-T.
After blotting wash membrane three times with TBT-T for 5min
Block 1h or overnight with 5% milk, keep milk afterwards
Wash membrane three times, 5min with TBT-T
Prepare dilution of first antibody in 5 ml milk in a Falcon
Put membrane in the Falcon and let it roll at 4°C overnight
Wash membrane three times, 5min with TBT-T
Prepare dilution of second antibody in 5 ml milk in a Falcon
Put membrane in the Falcon and let it roll at 4°C for 1 h
Wash membrane three times, 5min with TBT-T
Prepare strain solution: 2.5 ml each in one Falcon
Put membrane in the Falcon and let it roll at 4°C for 5min
Pipet 50 µl H20 on a clear film, add membrane, pipet 50µl on the membrane and close clear film</P><P> </P></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV><DIV class="container"><DIV class="content"><DIV class="row"><DIV class="col-lg-12"><DIV class="panel panel-default"><DIV class="panel-heading"><H3 class="basicheader">week number 31</H3></DIV><DIV class="panel-body"><DIV class="row"><DIV class="col-lg-12"><DIV class="notebookentry"><H3 class="notebookheading">▼2015-07-28  p415 digest</H3><DIV class="notebooktext"><P>p415 BsaI and BamHI digest:
0.5 µl BsaI
0.5µl BamHI
1 µl DNA
2 µl Cut Smart
16 µl water
1h 37°C </P></DIV></DIV></DIV></DIV><DIV class="row"><DIV class="col-lg-12"><DIV class="notebookentry"><H3 class="notebookheading">▼2015-07-28  p415 PCR DpnI digest, DNA precipitation und trafo in e.coli</H3><DIV class="notebooktext"><P>DpnI digest:
0.5 µl DpnI
4.5 µl Cut Smart
45 µl DNA (Q5 p415, Ph p415, Ve p415)
37°C 60min</P><P>DNA precipitation:
20µl NaAc 3M pH 5,5
10 min on ice
Centrifuge for 10 min at 4°C max
Pour the supernatant into a new tube
Add 2.5 volumes of 96% cold ethanol 
10min on ice
Centrifuge max for 10 min at 4°C
Remove supernatant
Wash pellet with 500µl 70% ethanol
Dry pellet
Dissolve pellet in 10 µl sterile water</P><P>KCM Trafo
With p415 mutated BsaI side after PCR (Q5, Ph,Ve) DpnI degest and precipitation </P></DIV></DIV></DIV></DIV><DIV class="row"><DIV class="col-lg-12"><DIV class="notebookentry"><H3 class="notebookheading">▼2015-07-29  Chemical competent E. coli cells (CaCl method)</H3><DIV class="notebooktext"><P>For Golden Gate Cloning preparation, one shot survival cells were made competent.
Chemical competent E. coli cells (CaCl method)
════════════════════════════════════════════════</P><P>• for ~ 200x 100 µl aliquotes</P><P> Materials ─────────────</P><P>• 100 mM CaCl2 (aq) - autoclaved• Glycerol</P><P>
Day 1 ─────────</P><P> • grow bacteria over night in 50 ml medium at 37°C</P><P> Day 2 ─────────</P><P>1) transfer 20-30 ml of the ON culture into 400 ml LB
2) grow at 37°C until an OD of 0.5-0.6 is reached (3-5 hours), put
CaCl_{2} on ice, prepare aliquoting buffer (20 ml CaCl_{2} with 10%
glycerol)
3) centrifuge bacteria for 30 min at 3750 rpm, 4°C• *work on ice and as quick as possible from here on*• ==&gt; you will need 8 falcons (50 ml) in order to do so
4) discard supernatant, resuspend bacteria in 5 ml CaCl_{2} / falcon,
pool 2 falcons, add up to 50 ml CaCl_{2}• ==&gt; you should have 4 falcons from here on
5) leave on ice for 30 min
6) centrifuge for 20 min at 3750 rpm, 4°C
7) discard supernatant, resuspend in 10 ml aliquoting buffer for all 4
falcons, add another 10 ml aliquoting buffer• *Do not vortex*
8) pipette 100 µl per eppendorf tube (pre-cooled on ice), put aliquots
in -80 °C freezer</P><P>Competence of E.coli were tested. </P></DIV></DIV></DIV></DIV><DIV class="row"><DIV class="col-lg-12"><DIV class="notebookentry"><H3 class="notebookheading">▼2015-07-31  p415-GPD BsaI mutation PCR</H3><DIV class="notebooktext"><P>22.3µl water
10µl 5x buffer
2µl 50mM MgCl2
2x 2.5µl pimer
5µl 10mM dNTPs
5µl DMSO
0.3 µl DNA
0.5µl Velocity</P><P>25µl Q5 MM
2x 2.5 µl primer
0.3 µl DNA
5µl DMSO
14.7µl water</P><P>Program:
97°C 3min
97°C 30sec
55°C 30sec
72°C 4min
72°C 5min
1-3 25x </P></DIV></DIV></DIV></DIV><DIV class="row"><DIV class="col-lg-12"><DIV class="notebookentry"><H3 class="notebookheading">▼2015-08-01  Miniprep p415 for “general plasmids”</H3></DIV></DIV></DIV><DIV class="row"><DIV class="col-lg-12"><DIV class="notebookentry"><H3 class="notebookheading">▼2015-08-01  p415-GDP open PCR </H3><DIV class="notebooktext"><P>Durchführung</P><P>P415-GDP open PCR (to get ccdB inside later on)</P><P>Velocity:</P><P>10 µl 5x buffer</P><P>2 µl 50mM MgCl2</P><P>2x 2.5 µl primer (JD 24,25)</P><P>5 µl 10mM dNTP</P><P>0.2 DNA</P><P>0.5 Velocity</P><P>27.3 µl water</P><P> </P><P>Q5</P><P>25 µl Q5</P><P>2x 2.5 µl primer</P><P>0.2 DNA</P><P>14.6 µl water</P><P> </P><P>Program:</P><P>97°C 3min</P><P>97°C 30 sec</P><P>72/68.6/61.4°C 30 sec</P><P>72°C 4min</P><P>30x</P><P>72°C 5min</P><P>4 for ever</P><P>Gel extraction with Q5 61.4 samples for Gibson.</P></DIV></DIV></DIV></DIV><DIV class="row"><DIV class="col-lg-12"><DIV class="notebookentry"><H3 class="notebookheading">▼2015-08-01  Gibson</H3><DIV class="notebooktext"><P>Gibson:</P><P>One with ccdBg(106.5ng/µl) and one with ccdBk(359ng/µl) both with p415 (12.5ng/µl)</P><P>Gibson ccdBg:</P><P>P415 6.62µl</P><P>ccdBg 0.39µl</P><P>water 2.99µl</P><P>Gibson MM 10µl</P><P> </P><P>Gibson ccdBk:</P><P>P415 6.62 µl</P><P>ccdBk 1:2 0.23</P><P>water 3.14 µl</P><P>Gibson MM 3.14 µl</P><P> 1h 50°C</P><P> </P></DIV></DIV></DIV></DIV><DIV class="row"><DIV class="col-lg-12"><DIV class="notebookentry"><H3 class="notebookheading">▼2015-08-01  Yeast trafo with positive control construct</H3><DIV class="notebooktext"><P>Method adapted from</P><P>Knop M, Siegers K, Pereira G, Zachariae W, Winsor B, Nasmyth KSchiebel E. <STRONG>Epitope tagging of yeast genes using a PCR-based strategy: more tags and improved practical routines.</STRONG><EM>Yeast</EM>, <STRONG>15:</STRONG>963-972.</P><P>The transformation protocol was based on the LiOAc method (Schiestl &amp; Gietz, 1989).</P><P><EM>The following solutions were used</EM>:</P><P><STRONG>SORB</STRONG>: 100 mM LiOAc, 10 mM Tris/HCl pH 8 (from 1 M stock), 1 mM EDTA/NaOH pH 8 (from 0.5 M stock), 1 M sorbitol (special grade for molecular biology from Merck), adjusted with diluted acetic acid to pH 8, sterile filtered (can be stored at room temperature for several months, a volume of 0.5 l was usually prepared).</P><P><STRONG>PEG</STRONG>: 100 mM LiOAc, 10 mM Tris/HCl pH8 (from 1 M stock), 1 mM EDTA/NaOH (from 0.5 M stock) pH8 , 40% PEG3350 (Sigma), sterile filtered (can be stored at 4°C for several months, a volume of 50 to 100 ml was usually prepared).</P><P><STRONG>Carrier DNA</STRONG>: Salmon sperm DNA (10 mg/ml, Gibco BRL, 2 kb average lenght) was boild for 10 min and cooled on ice. Such prepared carrier-DNA can be stored at -20°C and repeatedly used.</P><P><EM>Growth of yeast cells</EM>:</P><P>Yeast cells were inoculated from a fresh pre-culture and grown for at least 6-8 hours or over night to a density of 1 to 2 OD600 at 30°C (approx. 4*107 cells per ml) in YPAD medium.</P><P><EM>Competent yeast cells</EM>: Yeast cells were harvested by centrifugation (500 g, 5 min, room temperature), washed once with 0.1 to 0.5 vol sterile water (room temperature) and once with 0.1 to 0.2 vol SORB (room temperature). SORB was completely removed by aspiration. The cells were finally resuspended in a total volume of 360 ml SORB per 50 ml of cell culture and 40 ml of carrier DNA (0°C) was added. The cells were aliquoted into appropriate portions (e.g. 50 ml, at room temperature) and placed at -80°C (<STRONG>no</STRONG> shock freezing).</P><P><EM>Transformation</EM>: Usually, 10 ml of competent cells were used for the transformation of plasmid DNA and 50 ml of cells for the transformation of a PCR product. The DNA was placed into a sterile 1.5 ml tube (maximal 2 ml plasmid DNA per 10 ml of cells), the competent cells were thawed and added. The suspension was mixed well before a 6 fold volume of PEG was added. Cells were mixed throughout and incubated at room temperature for approx. 30 min (up to 2 hours is no problem). DMSO was added (1/9 vol to make a final concentration of approx. 10%). The cells were placed in a water bath at 42°C for 5-20 min (15 min works best with most strains). The cells were sedimented (2-3 min at 2000 rpm), the supernatant was removed (since it slows down growth on the plates) and the cells were resuspended in 100 to 200 ml of liquid medium (YPD or selective medium, for <EM>kanMX6</EM> selection see below).</P><P><EM>Selection for transformants</EM>:</P><P>In cases were auxotrophy markers were used for selection of the plasmid or PCR product, cells were plated onto synthetic complete medium lacking the corresponding amino acid (<EM>His3MX6</EM>: SC-HIS plates; <EM>klTRP1</EM>: SC-TRP plates). In cases, where PCR-products containing the dominant resistance marker <EM>kanMX6</EM> were used, the cells were resuspended in approx. 3 ml of YPAD, incubated on a shaker for 2-3 hours at 23 or 30˚C, harvested and spread on a G418 plate (further details are given in the text).</P><P> </P><P>1µl DNA
10 µl cells
60 µl PEG
7.7 µl DMSO </P><P> </P><P> </P></DIV></DIV></DIV></DIV><DIV class="row"><DIV class="col-lg-12"><DIV class="notebookentry"><H3 class="notebookheading">▼2015-08-02  KCM Trafo with Gibson Assembly (see 2015-08-01)</H3><DIV class="notebooktext"><P>ccdB resistant E. coli and top10 as control</P></DIV></DIV></DIV></DIV><DIV class="row"><DIV class="col-lg-12"><DIV class="notebookentry"><H3 class="notebookheading">▼2015-08-02  PCR to open p415 and p416 for ccdB</H3><DIV class="notebooktext"><P>Velocity:
2µl buffer
2x 0.5µl pimer
1µl dNTP
1µl p415 1:10 or p416 1:10
0.2 Velocity
0/1/3 µl MgCl2
Add water to 10µl final volume</P><P>Q5
5µl Q5 MM
2µl GC buffer 
1µl p415 1:10 or p416 1:10
2x 0.5µl primer
1µl water</P><P>Program:
97°C 3min
97°C 30 sec   
67/63.9/59.6°C 30 sec
72°C 4min
30x
72°C 5min
4°C for ever</P><P>3x3 with ve, 3x with Q5, one p416ve and one p416Q5 </P></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV><DIV class="container"><DIV class="content"><DIV class="row"><DIV class="col-lg-12"><DIV class="panel panel-default"><DIV class="panel-heading"><H3 class="basicheader">week number 32</H3></DIV><DIV class="panel-body"><DIV class="row"><DIV class="col-lg-12"><DIV class="notebookentry"><H3 class="notebookheading">▼2015-08-04  Colony PCR with ccdBg and ccdBk</H3><DIV class="notebooktext"><P>5µl One Tag</P><P>2x 0.5 µl primer</P><P>4 µl water</P><P> </P><P>Program:</P><P>4min 95°C</P><P>30 sec 95°C</P><P>30 sec 55°C</P><P>2min 68°C</P><P>3min 68°C</P><P> </P><P> </P><P>A product with 784bp is expected when the ccdB is in p415</P><P>Positive colonies were grown for a test digest.</P><P>Entryvector test digest with BsaI</P><P>600ng p415</P><P>0.5 µl BsaI</P><P> </P><P>Results:</P><P>If the entryvector assembled correctly bands with a side of 4854bp, 2050bp, 577bp and 123bp were expected. Entryvector 1, 2, 4, 5, 6 and 7 can be used.</P></DIV></DIV></DIV></DIV><DIV class="row"><DIV class="col-lg-12"><DIV class="notebookentry"><H3 class="notebookheading">▼2015-08-04  KCM Trafo</H3><DIV class="notebooktext"><P>KCM Trafo with correct entryvectors in DH5α to check if it is functional. The original ccdB pSB1C3 plasmid was also transformed as a control.</P><P>Results:
With colonies which grow a miniprep and BsaI test digest were done. </P></DIV></DIV></DIV></DIV><DIV class="row"><DIV class="col-lg-12"><DIV class="notebookentry"><H3 class="notebookheading">▼2015-08-06  PCR ribozyme fragments</H3><DIV class="notebooktext"><P>10µl 5xbuffer
2µl 50mM MgCl2
2x2.5 µl primer
5µl 10mM dNTP
1µl DNA 0.5 µl Velocity
26.5µl water</P><P>Program:
97°C 2min
97°C 30sec
61°C 30sec
72°C 45°C
72°C 5min </P></DIV></DIV></DIV></DIV><DIV class="row"><DIV class="col-lg-12"><DIV class="notebookentry"><H3 class="notebookheading">▼2015-08-07  PCR PCR ribozyme fragments optimised</H3><DIV class="notebooktext"><P>0µl MgCl2</P><TABLE border="0" cellpadding="0" cellspacing="0"><TBODY><TR><TD style="width:201px"><P>Phusion</P></TD><TD style="width:201px"><P>Velocity</P></TD></TR><TR><TD style="width:201px"><P> </P></TD><TD style="width:201px"><P>5µl 5x buffer</P></TD></TR><TR><TD style="width:201px"><P> </P></TD><TD style="width:201px"><P>2x 1.25µl primer</P></TD></TR><TR><TD style="width:201px"><P> </P></TD><TD style="width:201px"><P>2.5 µl dNTP</P></TD></TR><TR><TD style="width:201px"><P>0.5 µl DNA</P></TD><TD style="width:201px"><P>0.5 DNA</P></TD></TR><TR><TD style="width:201px"><P>2x 1.25µl primer</P></TD><TD style="width:201px"><P>0,25 Velocity</P></TD></TR><TR><TD style="width:201px"><P>9.5 µl water</P></TD><TD style="width:201px"><P>14.25 water</P></TD></TR><TR><TD style="width:201px"><P> </P></TD><TD style="width:201px"><P> </P></TD></TR><TR><TD style="width:201px"><P> </P></TD><TD style="width:201px"><P> </P></TD></TR></TBODY></TABLE><P> </P><P>1.5µl MgCl2</P><TABLE border="0" cellpadding="0" cellspacing="0"><TBODY><TR><TD style="width:201px"><P>Phusion</P></TD><TD style="width:201px"><P>Velocity</P></TD></TR><TR><TD style="width:201px"><P> </P></TD><TD style="width:201px"><P>5µl 5x buffer</P></TD></TR><TR><TD style="width:201px"><P> </P></TD><TD style="width:201px"><P>2x 1.25µl primer</P></TD></TR><TR><TD style="width:201px"><P> </P></TD><TD style="width:201px"><P>2.5 µl dNTP</P></TD></TR><TR><TD style="width:201px"><P>0.5 µl DNA</P></TD><TD style="width:201px"><P>0.5 DNA</P></TD></TR><TR><TD style="width:201px"><P>2x 1.25µl primer</P></TD><TD style="width:201px"><P>0,25 Velocity</P></TD></TR><TR><TD style="width:201px"><P>8 µl water</P></TD><TD style="width:201px"><P>12.75 water</P></TD></TR><TR><TD style="width:201px"><P>1.5 µl MgCl2</P></TD><TD style="width:201px"><P>1.5µl MgCl2</P></TD></TR><TR><TD style="width:201px"><P> </P></TD><TD style="width:201px"><P> </P></TD></TR></TBODY></TABLE><P> </P><P> </P><P>From some samples a gel extraction was done.</P></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV><DIV class="container"><DIV class="content"><DIV class="row"><DIV class="col-lg-12"><DIV class="panel panel-default"><DIV class="panel-heading"><H3 class="basicheader">week number 33</H3></DIV><DIV class="panel-body"><DIV class="row"><DIV class="col-lg-12"><DIV class="notebookentry"><H3 class="notebookheading">▼2015-08-10  PCR optimized</H3><DIV class="notebooktext"><TABLE border="1" cellpadding="0" cellspacing="0"><TBODY><TR><TD style="width:201px"><P>Ve</P></TD><TD style="width:201px"><P>Ve with DMSO</P></TD><TD style="width:201px"><P>Phusion</P></TD></TR><TR><TD style="width:201px"><P>5µl 5xbuffer</P></TD><TD style="width:201px"><P>5µl 5xbuffer</P></TD><TD style="width:201px"><P>5µl 5xbuffer</P></TD></TR><TR><TD style="width:201px"><P>2x1.25µl primer</P></TD><TD style="width:201px"><P>2x1.25µl primer</P></TD><TD style="width:201px"><P>2x1.25µl primer</P></TD></TR><TR><TD style="width:201px"><P>2.5µl 10mM dNTP</P></TD><TD style="width:201px"><P>2.5µl 10mM dNTP</P></TD><TD style="width:201px"><P> </P></TD></TR><TR><TD style="width:201px"><P>0.5 µl DNA</P></TD><TD style="width:201px"><P>0.5 µl DNA</P></TD><TD style="width:201px"><P>0.5 µl DNA</P></TD></TR><TR><TD style="width:201px"><P>0.25 Ve</P></TD><TD style="width:201px"><P>0.25 Ve</P></TD><TD style="width:201px"><P>12.5 MM</P></TD></TR><TR><TD style="width:201px"><P>14.25µl water</P></TD><TD style="width:201px"><P>13.25µl water</P></TD><TD style="width:201px"><P>9.5 µl water</P></TD></TR><TR><TD style="width:201px"><P> </P></TD><TD style="width:201px"><P>1µl DMSO</P></TD><TD style="width:201px"><P> </P></TD></TR></TBODY></TABLE><P> </P><P> </P><P>Program:</P><P>95°C 2min</P><P>95°C 30sec</P><P>63-54°C 30sec</P><P>72°C 45 sec</P><P>72°C 5min</P><P> </P><P>PCR purification with two of this samples gave to less DNA.</P></DIV></DIV></DIV></DIV><DIV class="row"><DIV class="col-lg-12"><DIV class="notebookentry"><H3 class="notebookheading">▼2015-08-11  Fragment check for Golden Gate</H3><DIV class="notebooktext"><P>Fragment SH11 does not have the right side. New primers to build SH11 were ordered.</P></DIV></DIV></DIV></DIV><DIV class="row"><DIV class="col-lg-12"><DIV class="notebookentry"><H3 class="notebookheading">▼2015-08-11  Golden Gate</H3><DIV class="notebooktext"><P>construct assembly, Golden Gate [Flo's recept?]&gt; ═════════════════════════════════════════════════&gt;&gt;• 15 µl raction mixture&gt;• 20 - 30 ng Backbone-Fragment&gt;• 1.5x des Insert-Fragments (in molar)&gt;• 1.5 µl NEBuffer 2.1&gt;• 1 µl NEB T4 Ligase (400 U / µl)&gt;• 0.75 µl BbsI (5 U / µl)&gt;• 1 mM ATP, 1 mM DTT (1.5 µl of 10 mM ATP/DTT stock)&gt;&gt;• Cycler program:&gt; ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━&gt; cycle temp time purpose&gt; ────────────────────────────────────────────────────────&gt; 50x (or less) 37 °C 2:00 restriction reaction&gt;– 16 °C 5:00 ligation reaction&gt; ────────────────────────────────────────────────────────&gt; 1x 50 °C 5:00 ligase inactivation&gt; ────────────────────────────────────────────────────────&gt; 1x 80 °C 5:00 restr. enzyme inactivation&gt; ────────────────────────────────────────────────────────&gt; 1x 12 °C inf hold</P></DIV></DIV></DIV></DIV><DIV class="row"><DIV class="col-lg-12"><DIV class="notebookentry"><H3 class="notebookheading">▼2015-08-12  E.coli Trafo</H3><DIV class="notebooktext"><P>The Entryvector contains a BsaI restriction side in the Amp. resistance. That is why an additional ligation step was added.</P><P>1µl T4 DNA Ligase</P><P>20°C 20min</P><P>65°C 10min</P><P> </P><P>With this product DH5alpha Transformation was performed following the usual protocol. Transformation was done with 0.5µl, 2µl and rest DNADurchführung</P></DIV></DIV></DIV></DIV><DIV class="row"><DIV class="col-lg-12"><DIV class="notebookentry"><H3 class="notebookheading">▼2015-08-13  Colony PCR</H3><DIV class="notebooktext"><P>All Transformations worked well. Colony PCR was performed.
5µl OneTag MM
2x 0.5µl primer
4µl water</P><P>95°C 4min
95°C 30 sec
55°C 30 sec
68°C 2min
68°C 3min</P><P> </P><P>Fragments with the side of 1094bp were expected.</P><P>Fragments with the side of 1112bp were expected.</P><P> </P></DIV></DIV></DIV></DIV><DIV class="row"><DIV class="col-lg-12"><DIV class="notebookentry"><H3 class="notebookheading">▼2015-08-14  Miniprep, test digest, Trafo</H3><DIV class="notebooktext"><P>With all 16 colonies Mini Prep and a test digest were performed.</P><P>PStI Hf</P><P>BsaI Hf</P><P>1µl DNA</P><P>2µl SmartCut</P><P>2x 0.5µl Enzyme</P><P>16.5 µl water</P><P> </P><P> </P><P> </P><P>Expected Fragments:</P><P>935 bp, 2038bp, 4892bp</P><P> </P><P> </P><P> </P><P>Expected Fragments:</P><P>951 bp, 2043bp, 4924bp</P><P> </P><P>Sample SH09L1 and JD28L1 were used for yeast transformation. For this co-transformation the Entryvector g2 and either SH09L1 or JD28L1 were used.</P><P>500ng from each DNA</P><P>7.7 µl DMSO</P><P> </P><P>Glycerol stocks were made from Entryvector g2, g5, SH09L1 and JD28L1</P><P> </P></DIV></DIV></DIV></DIV><DIV class="row"><DIV class="col-lg-12"><DIV class="notebookentry"><H3 class="notebookheading">▼2015-08-15  Cloning ccdB in p415 via restriction enzymes</H3><DIV class="notebooktext"><P>p415 digest:</P><P>2µl CutSmart</P><P>1µl PstI HF</P><P>1µl HindIII HF</P><P>1.9 µl p415 (1µg)</P><P>14.1 nuclease free water</P><P> </P><P>ccdBg:</P><P>2µl CutSmart</P><P>1µl Psti HF</P><P>1µl HindIII HF</P><P>9.38 µl ccdBg</P><P>6.62 water</P><P> </P><P>ccdBk:</P><P>2µl CutSmart</P><P>1µl Psti HF</P><P>1µl HindIII HF</P><P>2.7 µl ccdBk</P><P>13.3 µl water</P><P> </P><P>At 37°C over night</P><P> </P><P>PCR purification</P><P> </P><P>Ligation</P><P>ccdBg:</P><P>2 µl 10x T4 DNA Buffer</P><P>1µl p415 (50ng= 0.0111pmol)</P><P>0,52 µl ccdB (25.8ng= 0.0557pmol)</P><P>1µl T4 Ligase</P><P>15.48 µl water</P><P> </P><P>E. coli trafo with ccdB resistant E.coli ccdBg or ccdBk or efficient check</P><P>Efficient check with Transformation Efficiency Kit BBa_J04450 psB1C3</P><P> </P><P> </P><P>ccdBg or ccdBk:</P><P>5 µl DNA</P><P>10 µl KCM 5x</P><P>35 µl water</P><P>Rest see protocol “KCM Transformation E.coli</P></DIV></DIV></DIV></DIV><DIV class="row"><DIV class="col-lg-12"><DIV class="notebookentry"><H3 class="notebookheading">▼2015-08-15  yeast colony PCR</H3><DIV class="notebooktext"><P>Colonies from PC SH09L1 and PC JD28L1 plates were taken, put in –His-Leu medium and a colony pcr was performed.
Colony PCR to check yeast transformation
Sample preparation:
Pick a small amount of cells with pipet tips and transfer in 5µl water
Add 20µl 20mM NaOH each
Heat samples at 99°C for 10min using PCR machine
Vortex samples vigorously for 10-20 s
Spin tubes for 1min to pellet brocken cells</P><P>PCR reaction:
10µl 2xMM
2x0.5µl primer
0.4 DMSO
8.6 µl supernatant of cell lysis</P><P>Run PCR:
94°C 2min
94°C 30s
55°C 30s
72°C 1min
Repeat step 2 to 4 35x
72°C 5min</P><P>Products were loaded on a gel 
Bands by 1782bp and 1092bp or 1103bp were expected. Clear products were only visible for the ribozyme constructs. That is why yeast transformation was repeated. </P></DIV></DIV></DIV></DIV><DIV class="row"><DIV class="col-lg-12"><DIV class="notebookentry"><H3 class="notebookheading">▼2015-08-15  PCR</H3><DIV class="notebooktext"><P>PCR to check if p413-myc-BFP-Ub-GFP is contaminated.
12.5 µl Phusion Master Mix
1.25 µl xxDH_08p413GPD revxx
1.25 µl xxDH_07p413GPD fwdxx
5µl p413 empty or with protein construct
Add water to 25µl </P><P>Program:
97°C 5min
97°C 30sec
52°C 30sec
72°C 1.5min
Step 2 to 4 35 times
72°C 3min
4°C for ever</P><P>Results: The sample with p413 with protein construct is not contaminated. </P></DIV></DIV></DIV></DIV><DIV class="row"><DIV class="col-lg-12"><DIV class="notebookentry"><H3 class="notebookheading">▼2015-08-15  PCR optimization for the p415 series of plasmids</H3><DIV class="notebooktext"><P>To solve the problems observed with the p415-GPD plamid in PCR reactions we tried to set up an reaction with 1M betaine and 5% DMSO. This mixture has proved to be capable to solve PCR problems with ribozyme secondary structures. Therefore we hoped to solve the issues involving p415-GPD</P><P>Setup in 50 µL reactions:</P><TABLE border="1" cellpadding="0" cellspacing="0"><TBODY><TR><TD style="width:307px"><P> </P></TD><TD style="width:307px"><P>Volume</P></TD></TR><TR><TD style="width:307px"><P>2x Phusion Flash PCR Mastermix</P></TD><TD style="width:307px"><P>25 µL</P></TD></TR><TR><TD style="width:307px"><P>Template DNA</P></TD><TD style="width:307px"><P>1 µL</P></TD></TR><TR><TD style="width:307px"><P>fwd. Primer (DH07)</P></TD><TD style="width:307px"><P>2.5 µL</P></TD></TR><TR><TD style="width:307px"><P>rev. Primer (DH08)</P></TD><TD style="width:307px"><P>2.5 µL</P></TD></TR><TR><TD style="width:307px"><P>ddH2O</P></TD><TD style="width:307px"><P>6.5 µL</P></TD></TR><TR><TD style="width:307px"><P>Betaine (5 M)</P></TD><TD style="width:307px"><P>10 µL</P></TD></TR><TR><TD style="width:307px"><P>DMSO</P></TD><TD style="width:307px"><P>2.5 µL</P></TD></TR></TBODY></TABLE><P> </P><P>Annealing temperatures were varied in a gradient.</P><TABLE border="1" cellpadding="0" cellspacing="0"><TBODY><TR><TD style="width:307px"><P>#</P></TD><TD style="width:307px"><P>Temperature</P></TD></TR><TR><TD style="width:307px"><P>1</P></TD><TD style="width:307px"><P>72 °C</P></TD></TR><TR><TD style="width:307px"><P>2</P></TD><TD style="width:307px"><P>68.8 °C</P></TD></TR><TR><TD style="width:307px"><P>3</P></TD><TD style="width:307px"><P>62.0 °C</P></TD></TR><TR><TD style="width:307px"><P>4</P></TD><TD style="width:307px"><P>59.4 °C</P></TD></TR><TR><TD style="width:307px"><P>5</P></TD><TD style="width:307px"><P>56 °C</P></TD></TR></TBODY></TABLE><P> </P><P>Program (Phusion):</P><TABLE border="1" cellpadding="0" cellspacing="0"><TBODY><TR><TD colspan="2" style="width:409px"><P>2 min</P></TD><TD style="width:205px"><P>98 °C</P></TD></TR><TR><TD rowspan="3" style="width:205px"><P>35x</P></TD><TD style="width:205px"><P>15 s</P></TD><TD style="width:205px"><P>98 °C</P></TD></TR><TR><TD style="width:205px"><P>20 s</P></TD><TD style="width:205px"><P>Gradient</P></TD></TR><TR><TD style="width:205px"><P>2 min 20 s</P></TD><TD style="width:205px"><P>72 °C</P></TD></TR><TR><TD colspan="2" style="width:409px"><P>5 min</P></TD><TD style="width:205px"><P>72 °C</P></TD></TR><TR><TD colspan="2" style="width:409px"><P>infinite</P></TD><TD style="width:205px"><P>4 °C</P></TD></TR></TBODY></TABLE><P> </P><P>Analysis on a 1% agarose gel showed for the reactions with annealing at 62 °C, 59.4 °C and 56 °C the presence of a band at the expected size of ~7kb. The reaction with annealing at 62 °C showed not only the clearest appearance but also the highest yield for the wanted product.</P></DIV></DIV></DIV></DIV><DIV class="row"><DIV class="col-lg-12"><DIV class="notebookentry"><H3 class="notebookheading">▼2015-08-16  Test digest of received plasmids</H3><DIV class="notebooktext"><P>To verify we were given valid plasmids from Ilia, we performed a test digest with BsaI and EcoRI. These two enzymes should produce, according to a Geneious virtual digest, a restriction fragment pattern easily distinguished.</P><P>To be analyzed:</P><P>In DNA minipreps: p413-ADH (#1/#2), p413-cyc1, p413-Gal1, p413-TEF (#1/#2), p415-ADH (#1/#2), p415-cyc1, p415-Gal1, p415-TEF (#1/#2)</P><P>Reactions were set up as following:</P><TABLE border="1" cellpadding="0" cellspacing="0"><TBODY><TR><TD style="width:307px"><P> </P></TD><TD style="width:307px"><P>Volume</P></TD></TR><TR><TD style="width:307px"><P>Miniprep DNA from Plasmids</P></TD><TD style="width:307px"><P>1 µL</P></TD></TR><TR><TD style="width:307px"><P>BsaI-HF (NEB)</P></TD><TD style="width:307px"><P>0.2 µL</P></TD></TR><TR><TD style="width:307px"><P>EcoRI-HF (NEB)</P></TD><TD style="width:307px"><P>0.2 µL</P></TD></TR><TR><TD style="width:307px"><P>CutSmart Buffer (NEB)</P></TD><TD style="width:307px"><P>1 µL</P></TD></TR><TR><TD style="width:307px"><P>ddH2O</P></TD><TD style="width:307px"><P>7.6 µL</P></TD></TR></TBODY></TABLE><P> </P><P>After digestion for 1 min at 37 °C the reaction mixture was analyzed via 1% agarose gel.</P><P>The restriction fragments appeared with sizes according to the expected fragments. Yet for the p413/p415-TEF digestions it appeared that there has been a swap between a p413-TEF and a p415-TEF sample. Before use of a p41x-TEF it is necessary to recheck for the correct backbone.</P></DIV></DIV></DIV></DIV><DIV class="row"><DIV class="col-lg-12"><DIV class="notebookentry"><H3 class="notebookheading">▼2015-08-16  PCR optimization for BFP site-directed mutagenesis</H3><DIV class="notebooktext"><P>The SalI site in the BFP of our protein construct (negative and myc-tag positive) has to be removed, to allow for easy recloning into a library of p413 Vectors with different strength promoters via BamHI and SalI.</P><P>To find the perfect conditions we resorted to PCR optimization.</P><P>Temperature conditions: 72 °C, 66 °C, 60 °C</P><P>Polymerases: 2x Phusion Flash PCR Mastermix (Thermo), Q5 2x High-Fidelity Mastermix (NEB)</P><P>Buffer solution: Standard Mastermix conditions or addition of 1 M Betaine and 5 % DMSO to counter secondary structure behavior.</P><P>Standard buffer conditions:                               </P><TABLE border="1" cellpadding="0" cellspacing="0"><TBODY><TR><TD style="width:307px"><P> </P></TD><TD style="width:307px"><P>Volume</P></TD></TR><TR><TD style="width:307px"><P>2x Polymerase Mastermix</P></TD><TD style="width:307px"><P>5 µL</P></TD></TR><TR><TD style="width:307px"><P>Template DNA</P></TD><TD style="width:307px"><P>0.2 µL</P></TD></TR><TR><TD style="width:307px"><P>fwd. Primer (DH61)</P></TD><TD style="width:307px"><P>0.5 µL</P></TD></TR><TR><TD style="width:307px"><P>rev. Primer (DH62)</P></TD><TD style="width:307px"><P>0.5 µL</P></TD></TR><TR><TD style="width:307px"><P>ddH2O</P></TD><TD style="width:307px"><P>3.8 µL</P></TD></TR></TBODY></TABLE><P> </P><P>DMSO/Betaine buffer conditions:</P><TABLE border="1" cellpadding="0" cellspacing="0"><TBODY><TR><TD style="width:307px"><P> </P></TD><TD style="width:307px"><P>Volume</P></TD></TR><TR><TD style="width:307px"><P>2x Polymerase Mastermix</P></TD><TD style="width:307px"><P>5 µL</P></TD></TR><TR><TD style="width:307px"><P>Template DNA</P></TD><TD style="width:307px"><P>0.2 µL</P></TD></TR><TR><TD style="width:307px"><P>fwd. Primer (DH61)</P></TD><TD style="width:307px"><P>0.5 µL</P></TD></TR><TR><TD style="width:307px"><P>rev. Primer (DH62)</P></TD><TD style="width:307px"><P>0.5 µL</P></TD></TR><TR><TD style="width:307px"><P>ddH2O</P></TD><TD style="width:307px"><P>1.3 µL</P></TD></TR><TR><TD style="width:307px"><P>Betaine (5 M)</P></TD><TD style="width:307px"><P>2 µL</P></TD></TR><TR><TD style="width:307px"><P>DMSO</P></TD><TD style="width:307px"><P>0.5 µL</P></TD></TR></TBODY></TABLE><P> </P><P>Reactions were prepared as mastermixes and distributed into 0.2 ml PCR tubes. Cyclinge programs were chosen differently for each polymerase and according to manufacturer’s guideline.</P><P>Program (Phusion Flash):</P><TABLE border="1" cellpadding="0" cellspacing="0"><TBODY><TR><TD colspan="2" style="width:409px"><P>15 s</P></TD><TD style="width:205px"><P>98 °C</P></TD></TR><TR><TD rowspan="3" style="width:205px"><P>35x</P></TD><TD style="width:205px"><P>5 s</P></TD><TD style="width:205px"><P>98 °C</P></TD></TR><TR><TD style="width:205px"><P>5 s</P></TD><TD style="width:205px"><P>Gradient (72 °C, 66 °C, 60 °C)</P></TD></TR><TR><TD style="width:205px"><P>2 min</P></TD><TD style="width:205px"><P>72 °C</P></TD></TR><TR><TD colspan="2" style="width:409px"><P>5 min</P></TD><TD style="width:205px"><P>72 °C</P></TD></TR><TR><TD colspan="2" style="width:409px"><P>infinite</P></TD><TD style="width:205px"><P>4 °C</P></TD></TR></TBODY></TABLE><P> </P><P>Program (Q5):</P><TABLE border="1" cellpadding="0" cellspacing="0"><TBODY><TR><TD colspan="2" style="width:409px"><P>1 min</P></TD><TD style="width:205px"><P>98 °C</P></TD></TR><TR><TD rowspan="3" style="width:205px"><P>35x</P></TD><TD style="width:205px"><P>10</P></TD><TD style="width:205px"><P>98 °C</P></TD></TR><TR><TD style="width:205px"><P>20 s</P></TD><TD style="width:205px"><P>Gradient (72 °C, 66 °C, 60 °C)</P></TD></TR><TR><TD style="width:205px"><P>4 min</P></TD><TD style="width:205px"><P>72 °C</P></TD></TR><TR><TD colspan="2" style="width:409px"><P>5 min</P></TD><TD style="width:205px"><P>72 °C</P></TD></TR><TR><TD colspan="2" style="width:409px"><P>infinite</P></TD><TD style="width:205px"><P>4 °C</P></TD></TR></TBODY></TABLE><P> </P><P>The reactions were then mixed with NEB Purple Loading dye and analyzed on a 1% Agarose Gel.</P><P>The strongest and most defined bands, to be expected around 7.5 kb were observed with Phusion Flash polymerase in Mastermix buffer at 66° C annealing temperature. We will therefore try to perform the site directed mutagenesis under these conditions.</P></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV><DIV class="container"><DIV class="content"><DIV class="row"><DIV class="col-lg-12"><DIV class="panel panel-default"><DIV class="panel-heading"><H3 class="basicheader">week number 34</H3></DIV><DIV class="panel-body"><DIV class="row"><DIV class="col-lg-12"><DIV class="notebookentry"><H3 class="notebookheading">▼2015-08-17  Repetition of site-directed mutagenesis to remove SalI site for recloning</H3><DIV class="notebooktext"><P>The  reaction was repeated for PC (2.1) and myc-Pos (#1) with triplicates at annealing temperatures 60 °C, 64 °C, 66 °C in 50 µL reactions:</P><TABLE border="1" cellpadding="0" cellspacing="0"><TBODY><TR><TD style="width:307px"><P> </P></TD><TD style="width:307px"><P>Volume</P></TD></TR><TR><TD style="width:307px"><P>2x Phusion Flash PCR Mastermix</P></TD><TD style="width:307px"><P>25 µL</P></TD></TR><TR><TD style="width:307px"><P>Template DNA (1 ng/µL Dilution)</P></TD><TD style="width:307px"><P>1 µL</P></TD></TR><TR><TD style="width:307px"><P>fwd. Primer (DH61)</P></TD><TD style="width:307px"><P>2.5 µL</P></TD></TR><TR><TD style="width:307px"><P>rev. Primer (DH62)</P></TD><TD style="width:307px"><P>2.5 µL</P></TD></TR><TR><TD style="width:307px"><P>ddH2O</P></TD><TD style="width:307px"><P>19 µL</P></TD></TR></TBODY></TABLE><P> Program:</P><TABLE border="1" cellpadding="0" cellspacing="0"><TBODY><TR><TD colspan="2" style="width:409px"><P>15 s</P></TD><TD style="width:205px"><P>98 °C</P></TD></TR><TR><TD rowspan="3" style="width:205px"><P>35x</P></TD><TD style="width:205px"><P>5 s</P></TD><TD style="width:205px"><P>98 °C</P></TD></TR><TR><TD style="width:205px"><P>5 s</P></TD><TD style="width:205px"><P>Gradient (60 °C, 64 °C, 66 °C)</P></TD></TR><TR><TD style="width:205px"><P>2 min</P></TD><TD style="width:205px"><P>72 °C</P></TD></TR><TR><TD colspan="2" style="width:409px"><P>5 min</P></TD><TD style="width:205px"><P>72 °C</P></TD></TR><TR><TD colspan="2" style="width:409px"><P>infinite</P></TD><TD style="width:205px"><P>4 °C</P></TD></TR></TBODY></TABLE><P> </P><P>Following PCR each 50 µL reaction mix was digested with 0,25 µL DpnI and 0,25 µL SalI to destroy template DNA and linearize the product. The reaction mix was incubated 90 min at 37 °C and heat inactivated for 5 min at 80 °C.</P><P>For verification 5µL of each reaction were loaded onto a 1% Agarose gel.</P><P>The gel showed, apart from little blur, no sign of the expected product. Restaining of the gel with ethidium bromide solution did not improve gel image.</P><P>Further processing of the product was aborted.</P></DIV></DIV></DIV></DIV><DIV class="row"><DIV class="col-lg-12"><DIV class="notebookentry"><H3 class="notebookheading">▼2015-08-18  Yeast co-transformation</H3><DIV class="notebooktext"><P>PC/p415, pos/p415, SH09L/PC, JD28L/PC
PC=PC2.1 pos=#1 SH09L=SH09L1 JD28L=JD28L1
1µl each were taken</P><P>Sequencing
15µl 30-50ng/µl
SH09L1
SH09L5
JD28L1
JD28L2 </P></DIV></DIV></DIV></DIV><DIV class="row"><DIV class="col-lg-12"><DIV class="notebookentry"><H3 class="notebookheading">▼2015-08-19  Ribozyme assay</H3><DIV class="notebooktext"><P>Following yeast strains were used:</P><P>Yeast containing the protein construct in p413 only</P><P>Yeast containing the positive construct in p413</P><P>Yeast containing the p413 with the protein construct and p415 with the ribozyme (SH09)</P><P>Yeast containing the p413 with the protein construct and p415 with theophylline inducible ribozyme (JD28)</P><P> An overnight culture of each was diluted to OD=0.15</P><P>Cells grow approx. 4h @ 30 deg shaker until they reach OD of 0.5</P><P>1ml t0 samples of each were taken, two the samples 150 µl of 1.85 M NaOH, 10µl ß-mercaptoethanol were added and then shock froze in liquid nitrogen</P><P> </P><P>After t0 samples were taken, theophylline was added the following:</P><P>PC (protein construct): 0mM theophylline</P><P>Pos (positive construct): 0mM theophylline</P><P>SH09-0 (ribozyme): 0mM theophylline</P><P>SH09-5: 5mM theophylline</P><P>JD28-0 (inducible ribozyme): 0mM theophylline</P><P>JD28-2.5 : 2.5mM theophylline</P><P>JD28-5: 5mM theophylline</P><P>JD28-10: 10mM theophylline</P><P> </P><P>After two hours (approx. OD=0.5) 1ml t1 samples were taken and treated as previously described. Cultures were diluted in such a way that the theophylline concentration remainders the same.</P><P>T2 (4h) T3 (5.5h) and t4 (16.5h) samples were taken as described above.</P><P>Then all samples were treaded the following:</P><P>Then add 150 µl 55% TCA (w/v) to the cells. Incubate for 10 min on ice. Centrifuge at 4°C for 15 min in a microfuge (14000 rpm). Remove supernatant, centrifuge again and remove remaining supernatant. Resuspend precipitated proteins in 100-200 µl HU-buffer by vortexing and heating sample for 10-15 min at 65°C. Spin samples for 5-10 min at RT, full speed in Eppi centrifuge. Load sample (10-15 µl) onto SDS-PAGE. SDS-PAGE run at 160V max ampere for 90min.</P><P>Than a semi-dry blot was performed as described above.</P><P>Membranes were treaded with anti-myc antibody (for more details see Western blot protocol above)</P><P>Results: no activity of any ribozyme, neither SH09 nor JD28, can be observed.</P></DIV></DIV></DIV></DIV><DIV class="row"><DIV class="col-lg-12"><DIV class="notebookentry"><H3 class="notebookheading">▼2015-08-22  colony PCR </H3><DIV class="notebooktext"><P>A colony PCR was performed to check if both ribozyme and protein construct are in the tested yeast.</P><P>Weak bands in the expected side were detected, also the positive control only gave weak bands.</P></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV><DIV class="container"><DIV class="content"><DIV class="row"><DIV class="col-lg-12"><DIV class="panel panel-default"><DIV class="panel-heading"><H3 class="basicheader">week number 35</H3></DIV><DIV class="panel-body"><DIV class="row"><DIV class="col-lg-12"><DIV class="notebookentry"><H3 class="notebookheading">▼2015-08-25  RNA extraction</H3><DIV class="notebooktext"><P>Because the functionality of the ribozyme couldn’t be shown on protein level, the next step is to show functionality on RNA level. Therefore RT-PCR was performed with positive control samples and SH09 samples. Old means that the samples were also used for the Western Blot described above.</P><P>RNA extraction</P><P>1. Grow yeast cells in 10ml of desired medium to mid-exponential phase (OD600=1.0).</P><P>2. Transfer culture to 50-ml centrifuge tube and centrifuge cells 3min at 1500g, 4°C</P><P>3. Discard supernatant, resuspend pellet in 1ml ice-cold water. Transfer to clean 1.5-ml microcentrifuge tube. Microcentrifuge 10 sec at 4°C, and remove supernatant.</P><P>4. Resuspend pellet in 300µl RNA buffer: 0.5M NaCl, 200mM Tris Cl, pH 7.5, 10mM EDTA</P><P>Disrupt the cells</P><P>5. Add a volume of chilled acid-washed glass beads equivalent to 200ml water</P><P>6. Add 300µl of 25:24:1 phenol/chloroform/isoamyl alcohol equilibrated with RNA buffer.</P><P>7. Close the cap, then invert and shake up and dwon to ensure that the beads are suspended. Vortex vigorously for 2min at highest speed.</P><P>8. Microcentrifuge 1min at room temperature. Transfer aqueous layer to a clean microcentrifuge tube.</P><P>9. Add an equal volume of 25:24:1 phenol/chloroform/isoamyl alcohol. Vortex vigorously 10 sec.</P><P>10. Repeat step 8.</P><P>Precipitate the RNA</P><P>0.1 volume of 3M NaAC</P><P>2.5 volume of ethanol</P><P>Incubate at least 2 hours at -20°C, centrifuge for 30min at 4°C full speed. Discard supernatant and resuspend pellet in 20µl water.</P><P>DNA digest, Precipitate the RNA as described above.</P></DIV></DIV></DIV></DIV><DIV class="row"><DIV class="col-lg-12"><DIV class="notebookentry"><H3 class="notebookheading">▼2015-08-26  RT-PCR</H3><DIV class="notebooktext"><P>4µl RNA</P><P>1µl dNTPmix (10mM each)</P><P>1µl primer (2µM)</P><P>7µl water</P><P> </P><P>65°C, 5min, on ice, spin down</P><P>4µl 5xFirst buffer</P><P>1µl 0.1M DTT</P><P>1µl RNase inhinitor</P><P>1µl SuperScript III RT or for –RT control 1µl water</P><P> </P><P>55°C, 1h for gene specific primer: xxJD33RTmycxx, xxJD34RTgfpxx</P><P>50°C oligo dT</P><P>70°C, 15min</P><P>150mM NaOH, 37°C, 15min</P><P>150mM HCl</P><P>DNA precipitation with EtOH and NaOAc as described above.</P></DIV></DIV></DIV></DIV><DIV class="row"><DIV class="col-lg-12"><DIV class="notebookentry"><H3 class="notebookheading">▼2015-08-27  PCR with cDNA</H3><DIV class="notebooktext"><P>With all four samples (pos old, pos new, ribozyme old, ribozyme new) PCRs with following primers were performed:</P><P>xxDH076xx and xxJD034xx</P><P>xxDH076xx and xxJD033xx</P><P>SH021 and JD033xx</P><P>Program:</P><P>94°C 4min</P><P>94°C 30 sec</P><P>60°C 30sec</P><P>72°C 45 sec</P><P>72°C 10min</P><P>4°C for ever</P><P> </P><P>Samples were sent for sequencing.</P></DIV></DIV></DIV></DIV><DIV class="row"><DIV class="col-lg-12"><DIV class="notebookentry"><H3 class="notebookheading">▼2015-08-28  Sequence information</H3><DIV class="notebooktext"><P>Sequence of the ribozyme can be confirmed. Sequencing doesn’t show any ribozyme product: GFP-myc in ribozyme samples.</P></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV><DIV class="container"><DIV class="content"><DIV class="row"><DIV class="col-lg-12"><DIV class="panel panel-default"><DIV class="panel-heading"><H3 class="basicheader">week number 36</H3></DIV><DIV class="panel-body"><DIV class="row"><DIV class="col-lg-12"><DIV class="notebookentry"><H3 class="notebookheading">▼2015-08-31  Labeling of RNA with alkyne-NTP</H3><DIV class="notebooktext"><P>The labeling RNA was modified with alkyne NTPs by Terminale Dinucleotidyl Transferase (TdT) and as a second approach, by Poly A Polymerase (yeast), affimetrix.</P><P>The modified NTPs are from Jena Bioscience:</P><UL><LI>5-Ethynyl-dUTP (5-EdUTP)</LI><LI>C8-Alkyne-dUTP</LI><LI>C8-Alkyne-dCTP</LI></UL><TABLE border="1" cellpadding="0" cellspacing="0"><TBODY><TR><TD style="width:217px"> </TD><TD style="width:85px"><P><STRONG>cStock</STRONG></P></TD><TD style="width:104px"><P><STRONG>cFinal</STRONG></P></TD><TD style="width:95px"><P><STRONG>V[µL]</STRONG></P></TD></TR><TR><TD style="width:217px"><P><STRONG>TdT buffer</STRONG></P></TD><TD style="width:85px"><P>10 x</P></TD><TD style="width:104px"><P>1 x</P></TD><TD style="width:95px"><P>2.5</P></TD></TR><TR><TD style="width:217px"><P><STRONG>Labeling RNA</STRONG></P></TD><TD style="width:85px"><P>84,97µM</P></TD><TD style="width:104px"><P>15 µM</P></TD><TD style="width:95px"><P>4.41</P></TD></TR><TR><TD style="width:217px"><P><STRONG>Alkyne-NTP</STRONG></P></TD><TD style="width:85px"><P>10 mM</P></TD><TD style="width:104px"><P>400 µM</P></TD><TD style="width:95px"><P>1</P></TD></TR><TR><TD style="width:217px"><P><STRONG>TdT</STRONG></P></TD><TD style="width:85px"><P>20,000 u/mL</P></TD><TD style="width:104px"><P>80 u/mL</P></TD><TD style="width:95px"><P>1</P></TD></TR><TR><TD style="width:217px"><P><STRONG>CoCl2</STRONG></P></TD><TD style="width:85px"><P>2.5 mM</P></TD><TD style="width:104px"><P>0.25 mM</P></TD><TD style="width:95px"><P>2.5</P></TD></TR><TR><TD style="width:217px"><P><STRONG>H2O</STRONG></P></TD><TD style="width:85px"> </TD><TD style="width:104px"><P>Add 25 µL</P></TD><TD style="width:95px"><P>13.59</P></TD></TR><TR><TD style="width:217px"><P><STRONG>Final</STRONG></P></TD><TD style="width:85px"> </TD><TD style="width:104px"> </TD><TD style="width:95px"><P><STRONG><EM>25</EM></STRONG></P></TD></TR></TBODY></TABLE><P> </P><TABLE border="1" cellpadding="0" cellspacing="0"><TBODY><TR><TD style="width:217px"> </TD><TD style="width:85px"><P><STRONG>cStock</STRONG></P></TD><TD style="width:104px"><P><STRONG>cFinal</STRONG></P></TD><TD style="width:95px"><P><STRONG>V[µL]</STRONG></P></TD></TR><TR><TD style="width:217px"><P><STRONG>PAP (yeast)</STRONG></P></TD><TD style="width:85px"><P>600 u/µL</P></TD><TD style="width:104px"><P>24 u/µL</P></TD><TD style="width:95px"><P>1</P></TD></TR><TR><TD style="width:217px"><P><STRONG>PAP buffer</STRONG></P></TD><TD style="width:85px"><P>5 x</P></TD><TD style="width:104px"><P>1 x</P></TD><TD style="width:95px"><P>5</P></TD></TR><TR><TD style="width:217px"><P><STRONG>5-EdUTP/ C8-Alkyne-dUTP/C8-Alkyne-dCTP</STRONG></P></TD><TD style="width:85px"><P>10 mM</P></TD><TD style="width:104px"><P>400 µM</P></TD><TD style="width:95px"><P>1</P></TD></TR><TR><TD style="width:217px"><P><STRONG>Labeling RNA</STRONG></P></TD><TD style="width:85px"><P>84.97µM</P></TD><TD style="width:104px"><P>15µM</P></TD><TD style="width:95px"><P>4.41</P></TD></TR><TR><TD style="width:217px"><P><STRONG>H2O</STRONG></P></TD><TD style="width:85px"> </TD><TD style="width:104px"><P>Add 25 µL</P></TD><TD style="width:95px"><P>13.59</P></TD></TR><TR><TD style="width:217px"><P><STRONG>Final</STRONG></P></TD><TD style="width:85px"> </TD><TD style="width:104px"> </TD><TD style="width:95px"><P><STRONG><EM>25</EM></STRONG></P></TD></TR></TBODY></TABLE><P> </P><UL><LI>All compounds were mixed as indicated in the tables.</LI><LI>Reactions were incubated for 2 hours at 37 °C</LI><LI>Samples were stored at -20 °C oN</LI></UL></DIV></DIV></DIV></DIV><DIV class="row"><DIV class="col-lg-12"><DIV class="notebookentry"><H3 class="notebookheading">▼2015-09-01  Biotin modification of substrate RNA</H3><DIV class="notebooktext"><TABLE border="1" cellpadding="0" cellspacing="0"><TBODY><TR><TD style="width:153px"><P><EM>For 25µL</EM></P></TD><TD style="width:153px"><P>Cstock</P></TD><TD style="width:153px"><P>Cfinal</P></TD><TD style="width:153px"><P>V [µL]</P></TD></TR><TR><TD style="width:153px"><P>TdT- Buffer</P></TD><TD style="width:153px"><P>10x</P></TD><TD style="width:153px"><P>1x</P></TD><TD style="width:153px"><P>2.5</P></TD></TR><TR><TD style="width:153px"><P>RNA</P></TD><TD style="width:153px"><P>71.804µM</P></TD><TD style="width:153px"><P>10µM</P></TD><TD style="width:153px"><P>3.48</P></TD></TR><TR><TD style="width:153px"><P>Biotin - NTP</P></TD><TD style="width:153px"><P>1mM</P></TD><TD style="width:153px"><P>200µM</P></TD><TD style="width:153px"><P>5</P></TD></TR><TR><TD style="width:153px"><P>TdT</P></TD><TD style="width:153px"><P>600U/µL</P></TD><TD style="width:153px"><P>24U/µL</P></TD><TD style="width:153px"><P>1</P></TD></TR><TR><TD style="width:153px"><P>CoCl2</P></TD><TD style="width:153px"><P>2.5mM</P></TD><TD style="width:153px"><P>0.5mM</P></TD><TD style="width:153px"><P>5</P></TD></TR><TR><TD style="width:153px"><P>MQ H2O</P></TD><TD style="width:153px"><P> </P></TD><TD style="width:153px"><P> </P></TD><TD style="width:153px"><P>8.02</P></TD></TR></TBODY></TABLE><P> </P><P>All samples were mixed together and incubated for 2 hours at 37 °C.</P></DIV></DIV></DIV></DIV><DIV class="row"><DIV class="col-lg-12"><DIV class="notebookentry"><H3 class="notebookheading">▼2015-09-01  Click reaction (modified)</H3><DIV class="notebooktext"><UL><LI>new Alexa 488 with DMSO</LI></UL><P> </P><TABLE border="1" cellpadding="0" cellspacing="0"><TBODY><TR><TD style="width:153px"><P><EM>For 50µL</EM></P></TD><TD style="width:153px"><P>Cstock</P></TD><TD style="width:153px"><P>Cfinal</P></TD><TD style="width:153px"><P>V[µL]</P></TD></TR><TR><TD style="width:153px"><P>Phosphate Buffer - pH 7, 0.1M</P></TD><TD style="width:153px"><P>100mM</P></TD><TD style="width:153px"><P>50mM</P></TD><TD style="width:153px"><P>25</P></TD></TR><TR><TD style="width:153px"><P>Alexa 488 azide</P></TD><TD style="width:153px"><P>10µM</P></TD><TD style="width:153px"><P>400nM</P></TD><TD style="width:153px"><P>2</P></TD></TR><TR><TD style="height:28px; width:153px"><P>RNA</P></TD><TD style="height:28px; width:153px"><P>1µM</P></TD><TD style="height:28px; width:153px"><P>200nM</P></TD><TD style="height:28px; width:153px"><P>10</P></TD></TR><TR><TD style="width:153px"><P>CuSO4</P></TD><TD style="width:153px"><P>50mM</P></TD><TD style="width:153px"><P>1mM</P></TD><TD style="width:153px"><P>1</P></TD></TR><TR><TD style="width:153px"><P>THPTA</P></TD><TD style="width:153px"><P>50mM</P></TD><TD style="width:153px"><P>5mM</P></TD><TD style="width:153px"><P>5</P></TD></TR><TR><TD style="width:153px"><P>NaAsc</P></TD><TD style="width:153px"><P>100mM</P></TD><TD style="width:153px"><P>1mM</P></TD><TD style="width:153px"><P>0.5</P></TD></TR><TR><TD style="width:153px"><P>H2O</P></TD><TD style="width:153px"><P> </P></TD><TD style="width:153px"><P> </P></TD><TD style="width:153px"><P>6.5</P></TD></TR></TBODY></TABLE><P> </P><P>All samples were mixed together and then incubated for 2hours at 37 °C.</P></DIV></DIV></DIV></DIV><DIV class="row"><DIV class="col-lg-12"><DIV class="notebookentry"><H3 class="notebookheading">▼2015-09-02  In vitro transcription of Labeling RNA AU + substrate HRP </H3><DIV class="notebooktext"><TABLE border="1" cellpadding="0" cellspacing="0"><TBODY><TR><TD style="width:217px"> </TD><TD style="width:85px"><P><STRONG>cStock</STRONG></P></TD><TD style="width:104px"><P><STRONG>cFinal</STRONG></P></TD><TD style="width:95px"><P><STRONG>V[µL]</STRONG></P></TD></TR><TR><TD style="width:217px"><P><STRONG>Transcription buffer</STRONG></P></TD><TD style="width:85px"><P>10 x</P></TD><TD style="width:104px"><P>1 x</P></TD><TD style="width:95px"><P>20</P></TD></TR><TR><TD style="width:217px"><P><STRONG>ATP</STRONG></P></TD><TD style="width:85px"><P>100 mM</P></TD><TD style="width:104px"><P>4 mM</P></TD><TD style="width:95px"><P>8</P></TD></TR><TR><TD style="width:217px"><P><STRONG>GTP</STRONG></P></TD><TD style="width:85px"><P>100 mM</P></TD><TD style="width:104px"><P>4 mM</P></TD><TD style="width:95px"><P>8</P></TD></TR><TR><TD style="width:217px"><P><STRONG>CTP</STRONG></P></TD><TD style="width:85px"><P>100 mM</P></TD><TD style="width:104px"><P>4 mM</P></TD><TD style="width:95px"><P>8</P></TD></TR><TR><TD style="width:217px"><P><STRONG>UTP</STRONG></P></TD><TD style="width:85px"><P>100 mM</P></TD><TD style="width:104px"><P>4 mM</P></TD><TD style="width:95px"><P>8</P></TD></TR><TR><TD style="width:217px"><P><STRONG>DTT</STRONG></P></TD><TD style="width:85px"><P>1 M</P></TD><TD style="width:104px"><P>10 mM</P></TD><TD style="width:95px"><P>2</P></TD></TR><TR><TD style="width:217px"><P><STRONG>DMSO</STRONG></P></TD><TD style="width:85px"> </TD><TD style="width:104px"><P>5 %</P></TD><TD style="width:95px"><P>10</P></TD></TR><TR><TD style="width:217px"><P><STRONG>FS_53/54 </STRONG><EM>(Label AU)</EM></P></TD><TD style="width:85px"><P> </P></TD><TD style="width:104px"><P> </P></TD><TD style="width:95px"><P>20</P></TD></TR><TR><TD style="width:217px"><P><STRONG>T7 RNA Polymerase</STRONG></P></TD><TD style="width:85px"><P>2 mg/mL</P></TD><TD style="width:104px"><P>0,1 mg/mL</P></TD><TD style="width:95px"><P>5</P></TD></TR><TR><TD style="width:217px"><P><STRONG>H2O</STRONG></P></TD><TD style="width:85px"> </TD><TD style="width:104px"><P>ad 100 µL</P></TD><TD style="width:95px"><P>106</P></TD></TR><TR><TD style="width:217px"><P><STRONG>Final</STRONG></P></TD><TD style="width:85px"> </TD><TD style="width:104px"> </TD><TD style="width:95px"><P><STRONG><EM>400</EM></STRONG></P></TD></TR></TBODY></TABLE><P> </P><TABLE border="1" cellpadding="0" cellspacing="0"><TBODY><TR><TD style="width:217px"> </TD><TD style="width:85px"><P><STRONG>cStock</STRONG></P></TD><TD style="width:104px"><P><STRONG>cFinal</STRONG></P></TD><TD style="width:95px"><P><STRONG>V[µL]</STRONG></P></TD></TR><TR><TD style="width:217px"><P><STRONG>Transcription buffer</STRONG></P></TD><TD style="width:85px"><P>10 x</P></TD><TD style="width:104px"><P>1 x</P></TD><TD style="width:95px"><P>20</P></TD></TR><TR><TD style="width:217px"><P><STRONG>ATP</STRONG></P></TD><TD style="width:85px"><P>100 mM</P></TD><TD style="width:104px"><P>4 mM</P></TD><TD style="width:95px"><P>8</P></TD></TR><TR><TD style="width:217px"><P><STRONG>GTP</STRONG></P></TD><TD style="width:85px"><P>100 mM</P></TD><TD style="width:104px"><P>4 mM</P></TD><TD style="width:95px"><P>8</P></TD></TR><TR><TD style="width:217px"><P><STRONG>CTP</STRONG></P></TD><TD style="width:85px"><P>100 mM</P></TD><TD style="width:104px"><P>4 mM</P></TD><TD style="width:95px"><P>8</P></TD></TR><TR><TD style="width:217px"><P><STRONG>UTP</STRONG></P></TD><TD style="width:85px"><P>100 mM</P></TD><TD style="width:104px"><P>4 mM</P></TD><TD style="width:95px"><P>8</P></TD></TR><TR><TD style="width:217px"><P><STRONG>DTT</STRONG></P></TD><TD style="width:85px"><P>1 M</P></TD><TD style="width:104px"><P>10 mM</P></TD><TD style="width:95px"><P>2</P></TD></TR><TR><TD style="width:217px"><P><STRONG>DMSO</STRONG></P></TD><TD style="width:85px"> </TD><TD style="width:104px"><P>5 %</P></TD><TD style="width:95px"><P>10</P></TD></TR><TR><TD style="width:217px"><P><STRONG>FS_50/51 </STRONG><EM>(Substrate HRP)</EM></P></TD><TD style="width:85px"><P> </P></TD><TD style="width:104px"><P> </P></TD><TD style="width:95px"><P>20</P></TD></TR><TR><TD style="width:217px"><P><STRONG>T7 RNA Polymerase</STRONG></P></TD><TD style="width:85px"><P>2 mg/mL</P></TD><TD style="width:104px"><P>0,1 mg/mL</P></TD><TD style="width:95px"><P>5</P></TD></TR><TR><TD style="width:217px"><P><STRONG>H2O</STRONG></P></TD><TD style="width:85px"> </TD><TD style="width:104px"><P>ad 100 µL</P></TD><TD style="width:95px"><P>106</P></TD></TR><TR><TD style="width:217px"><P><STRONG>Final</STRONG></P></TD><TD style="width:85px"> </TD><TD style="width:104px"> </TD><TD style="width:95px"><P><STRONG><EM>200</EM></STRONG></P></TD></TR></TBODY></TABLE><P> </P><P>All compounds were mixed together and incubated for 3 hours at 37 °C. Afterwards, 2µL DNAseI was added for 30min at 37 °C.</P></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV><DIV class="container"><DIV class="content"><DIV class="row"><DIV class="col-lg-12"><DIV class="panel panel-default"><DIV class="panel-heading"><H3 class="basicheader">week number 37</H3></DIV><DIV class="panel-body"><DIV class="row"><DIV class="col-lg-12"><DIV class="notebookentry"><H3 class="notebookheading">▼2015-09-10  DNAzyme Activity: DNAzyme with ATP aptamer and calculated Kanamycin aptamer</H3><DIV class="notebooktext"><P>Samples:</P><UL><LI>10-23 DNAzyme: xxfs032xx</LI><LI>7-18 DNAzyme: xxfs033xx</LI><LI>10-23 DNAzyme with ATP aptamer with linker: xxfs019xx</LI><LI>7-18 DNAzyme with ATP aptamer with linker: xxfs027xx</LI><LI>10-23 DNAzyme with ATP aptamer A: xxfs017xx, B: xxfs018xx</LI><LI>7-18 DNAzyme with ATP aptamer A: xxfs025, B: xxfs026xx</LI><LI>10-23 DNAzyme with calculated Kan aptamer: xxfs034xx</LI><LI>7-18 DNAzyme with calculated Kan aptamer candidate I: xxfs035xx</LI><LI>7-18 DNAzyme with calculated Kan aptamer candidate II: xxfs036xx</LI><LI>7-18 DNAzyme with calculated Kan aptamer candidate III: xxfs037xx</LI></UL><P>Stock solutions and conditions:</P><TABLE border="1" cellpadding="0" cellspacing="0"><TBODY><TR><TD style="width:198px"> </TD><TD style="width:104px"><P><STRONG>cStock</STRONG></P></TD><TD style="width:104px"><P><STRONG>cFinal</STRONG></P></TD></TR><TR><TD style="width:198px"><P><STRONG>Tris HCl ph 7.5</STRONG></P></TD><TD style="width:104px"><P>1 M</P></TD><TD style="width:104px"><P>50 mM</P></TD></TR><TR><TD style="width:198px"><P><STRONG>DNAzyme (A)</STRONG></P></TD><TD style="width:104px"><P>10 µM</P></TD><TD style="width:104px"><P>500 nM</P></TD></TR><TR><TD style="width:198px"><P><STRONG>DNAzyme B</STRONG></P></TD><TD style="width:104px"><P>10 µM</P></TD><TD style="width:104px"><P>500 nM</P></TD></TR><TR><TD style="width:198px"><P><STRONG>Substrate</STRONG></P></TD><TD style="width:104px"><P>1 µM</P></TD><TD style="width:104px"><P>200 nM</P></TD></TR><TR><TD style="width:198px"><P><STRONG>NaCl</STRONG></P></TD><TD style="width:104px"><P>1 M</P></TD><TD style="width:104px"><P>100 mM</P></TD></TR><TR><TD style="width:198px"><P><STRONG>MgCl2</STRONG></P></TD><TD style="width:104px"><P>1 M</P></TD><TD style="width:104px"><P>20 mM</P></TD></TR><TR><TD style="width:198px"><P><STRONG>SDS</STRONG></P></TD><TD style="width:104px"><P>20 %</P></TD><TD style="width:104px"><P>0,01 %</P></TD></TR><TR><TD style="width:198px"><P><STRONG>Adenosine in H2O:DMSO 1:2</STRONG></P></TD><TD style="width:104px"><P>33 mM</P></TD><TD style="width:104px"><P>5 mM</P></TD></TR><TR><TD style="width:198px"><P><STRONG>H2O</STRONG></P></TD><TD style="width:104px"><P> </P></TD><TD style="width:104px"><P>ad 25 µL</P></TD></TR></TBODY></TABLE><P> </P><P>Pipetting scheme:</P><TABLE border="1" cellpadding="0" cellspacing="0"><TBODY><TR><TD style="height:31px; width:64px"><P><STRONG>#</STRONG></P></TD><TD style="height:31px; width:64px"><P> </P></TD><TD style="height:31px; width:64px"><P> </P></TD><TD style="height:31px; width:64px"><P><STRONG>Tris HCl ph 7.5 </STRONG></P></TD><TD style="height:31px; width:64px"><P><STRONG>DNAzyme A</STRONG></P></TD><TD style="height:31px; width:64px"><P><STRONG>DNAzyme B</STRONG></P></TD><TD style="height:31px; width:64px"><P><STRONG>Substrate</STRONG></P></TD><TD style="height:31px; width:64px"><P><STRONG>NaCl</STRONG></P></TD><TD style="height:31px; width:64px"><P><STRONG>MgCl2</STRONG></P></TD><TD style="height:31px; width:64px"><P><STRONG>SDS</STRONG></P></TD><TD style="height:31px; width:64px"><P><STRONG>Adenosine</STRONG></P></TD><TD style="height:31px; width:64px"><P><STRONG>H2O</STRONG></P></TD><TD style="height:31px; width:64px"><P><STRONG>Final</STRONG></P></TD></TR><TR><TD style="height:20px; width:64px"><P>1</P></TD><TD style="height:20px; width:64px"><P>FS032</P></TD><TD style="height:20px; width:64px"><P>10-23D</P></TD><TD style="height:20px; width:64px"><P>1,25</P></TD><TD style="height:20px; width:64px"><P>1,25</P></TD><TD style="height:20px; width:64px"><P>0,00</P></TD><TD style="height:20px; width:64px"><P>5,00</P></TD><TD style="height:20px; width:64px"><P>2,50</P></TD><TD style="height:20px; width:64px"><P>0,50</P></TD><TD style="height:20px; width:64px"><P>1,25</P></TD><TD style="height:20px; width:64px"><P>3,75</P></TD><TD style="height:20px; width:64px"><P>9,50</P></TD><TD style="height:20px; width:64px"><P>25,00</P></TD></TR><TR><TD style="height:20px; width:64px"><P>2</P></TD><TD style="height:20px; width:64px"><P>FS033</P></TD><TD style="height:20px; width:64px"><P>7-18D</P></TD><TD style="height:20px; width:64px"><P>1,25</P></TD><TD style="height:20px; width:64px"><P>1,25</P></TD><TD style="height:20px; width:64px"><P>0,00</P></TD><TD style="height:20px; width:64px"><P>5,00</P></TD><TD style="height:20px; width:64px"><P>2,50</P></TD><TD style="height:20px; width:64px"><P>0,50</P></TD><TD style="height:20px; width:64px"><P>1,25</P></TD><TD style="height:20px; width:64px"><P>3,75</P></TD><TD style="height:20px; width:64px"><P>9,50</P></TD><TD style="height:20px; width:64px"><P>25,00</P></TD></TR><TR><TD style="height:20px; width:64px"><P>3</P></TD><TD style="height:20px; width:64px"><P>FS019</P></TD><TD style="height:20px; width:64px"><P>10-23DmLink</P></TD><TD style="height:20px; width:64px"><P>1,25</P></TD><TD style="height:20px; width:64px"><P>1,25</P></TD><TD style="height:20px; width:64px"><P>0,00</P></TD><TD style="height:20px; width:64px"><P>5,00</P></TD><TD style="height:20px; width:64px"><P>2,50</P></TD><TD style="height:20px; width:64px"><P>0,50</P></TD><TD style="height:20px; width:64px"><P>1,25</P></TD><TD style="height:20px; width:64px"><P>3,75</P></TD><TD style="height:20px; width:64px"><P>9,50</P></TD><TD style="height:20px; width:64px"><P>25,00</P></TD></TR><TR><TD style="height:20px; width:64px"><P>4</P></TD><TD style="height:20px; width:64px"><P>FS027</P></TD><TD style="height:20px; width:64px"><P>7-18DmLink</P></TD><TD style="height:20px; width:64px"><P>1,25</P></TD><TD style="height:20px; width:64px"><P>1,25</P></TD><TD style="height:20px; width:64px"><P>0,00</P></TD><TD style="height:20px; width:64px"><P>5,00</P></TD><TD style="height:20px; width:64px"><P>2,50</P></TD><TD style="height:20px; width:64px"><P>0,50</P></TD><TD style="height:20px; width:64px"><P>1,25</P></TD><TD style="height:20px; width:64px"><P>3,75</P></TD><TD style="height:20px; width:64px"><P>9,50</P></TD><TD style="height:20px; width:64px"><P>25,00</P></TD></TR><TR><TD style="height:20px; width:64px"><P>5</P></TD><TD style="height:20px; width:64px"><P>FS017+18</P></TD><TD style="height:20px; width:64px"><P>10-23D_A+B</P></TD><TD style="height:20px; width:64px"><P>1,25</P></TD><TD style="height:20px; width:64px"><P>1,25</P></TD><TD style="height:20px; width:64px"><P>6,25</P></TD><TD style="height:20px; width:64px"><P>5,00</P></TD><TD style="height:20px; width:64px"><P>2,50</P></TD><TD style="height:20px; width:64px"><P>0,50</P></TD><TD style="height:20px; width:64px"><P>1,25</P></TD><TD style="height:20px; width:64px"><P>3,75</P></TD><TD style="height:20px; width:64px"><P>3,25</P></TD><TD style="height:20px; width:64px"><P>25,00</P></TD></TR><TR><TD style="height:20px; width:64px"><P>6</P></TD><TD style="height:20px; width:64px"><P>FS025+26</P></TD><TD style="height:20px; width:64px"><P>7-18D_A+B</P></TD><TD style="height:20px; width:64px"><P>1,25</P></TD><TD style="height:20px; width:64px"><P>1,25</P></TD><TD style="height:20px; width:64px"><P>6,25</P></TD><TD style="height:20px; width:64px"><P>5,00</P></TD><TD style="height:20px; width:64px"><P>2,50</P></TD><TD style="height:20px; width:64px"><P>0,50</P></TD><TD style="height:20px; width:64px"><P>1,25</P></TD><TD style="height:20px; width:64px"><P>3,75</P></TD><TD style="height:20px; width:64px"><P>3,25</P></TD><TD style="height:20px; width:64px"><P>25,00</P></TD></TR><TR><TD style="height:20px; width:64px"><P>7</P></TD><TD style="height:20px; width:64px"><P>FS032</P></TD><TD style="height:20px; width:64px"><P>10-23D</P></TD><TD style="height:20px; width:64px"><P>1,25</P></TD><TD style="height:20px; width:64px"><P>1,25</P></TD><TD style="height:20px; width:64px"><P>0,00</P></TD><TD style="height:20px; width:64px"><P>5,00</P></TD><TD style="height:20px; width:64px"><P>2,50</P></TD><TD style="height:20px; width:64px"><P>0,50</P></TD><TD style="height:20px; width:64px"><P>1,25</P></TD><TD style="height:20px; width:64px"><P>0,00</P></TD><TD style="height:20px; width:64px"><P>13,25</P></TD><TD style="height:20px; width:64px"><P>25,00</P></TD></TR><TR><TD style="height:20px; width:64px"><P>8</P></TD><TD style="height:20px; width:64px"><P>FS033</P></TD><TD style="height:20px; width:64px"><P>7-18D</P></TD><TD style="height:20px; width:64px"><P>1,25</P></TD><TD style="height:20px; width:64px"><P>1,25</P></TD><TD style="height:20px; width:64px"><P>0,00</P></TD><TD style="height:20px; width:64px"><P>5,00</P></TD><TD style="height:20px; width:64px"><P>2,50</P></TD><TD style="height:20px; width:64px"><P>0,50</P></TD><TD style="height:20px; width:64px"><P>1,25</P></TD><TD style="height:20px; width:64px"><P>0,00</P></TD><TD style="height:20px; width:64px"><P>13,25</P></TD><TD style="height:20px; width:64px"><P>25,00</P></TD></TR><TR><TD style="height:20px; width:64px"><P>9</P></TD><TD style="height:20px; width:64px"><P>FS019</P></TD><TD style="height:20px; width:64px"><P>10-23DmLink</P></TD><TD style="height:20px; width:64px"><P>1,25</P></TD><TD style="height:20px; width:64px"><P>1,25</P></TD><TD style="height:20px; width:64px"><P>0,00</P></TD><TD style="height:20px; width:64px"><P>5,00</P></TD><TD style="height:20px; width:64px"><P>2,50</P></TD><TD style="height:20px; width:64px"><P>0,50</P></TD><TD style="height:20px; width:64px"><P>1,25</P></TD><TD style="height:20px; width:64px"><P>0,00</P></TD><TD style="height:20px; width:64px"><P>13,25</P></TD><TD style="height:20px; width:64px"><P>25,00</P></TD></TR><TR><TD style="height:20px; width:64px"><P>10</P></TD><TD style="height:20px; width:64px"><P>FS027</P></TD><TD style="height:20px; width:64px"><P>7-18DmLink</P></TD><TD style="height:20px; width:64px"><P>1,25</P></TD><TD style="height:20px; width:64px"><P>1,25</P></TD><TD style="height:20px; width:64px"><P>0,00</P></TD><TD style="height:20px; width:64px"><P>5,00</P></TD><TD style="height:20px; width:64px"><P>2,50</P></TD><TD style="height:20px; width:64px"><P>0,50</P></TD><TD style="height:20px; width:64px"><P>1,25</P></TD><TD style="height:20px; width:64px"><P>0,00</P></TD><TD style="height:20px; width:64px"><P>13,25</P></TD><TD style="height:20px; width:64px"><P>25,00</P></TD></TR><TR><TD style="height:20px; width:64px"><P>11</P></TD><TD style="height:20px; width:64px"><P>FS017+18</P></TD><TD style="height:20px; width:64px"><P>10-23D_A+B</P></TD><TD style="height:20px; width:64px"><P>1,25</P></TD><TD style="height:20px; width:64px"><P>1,25</P></TD><TD style="height:20px; width:64px"><P>6,25</P></TD><TD style="height:20px; width:64px"><P>5,00</P></TD><TD style="height:20px; width:64px"><P>2,50</P></TD><TD style="height:20px; width:64px"><P>0,50</P></TD><TD style="height:20px; width:64px"><P>1,25</P></TD><TD style="height:20px; width:64px"><P>0,00</P></TD><TD style="height:20px; width:64px"><P>7,00</P></TD><TD style="height:20px; width:64px"><P>25,00</P></TD></TR><TR><TD style="height:20px; width:64px"><P>12</P></TD><TD style="height:20px; width:64px"><P>FS025+26</P></TD><TD style="height:20px; width:64px"><P>7-18D_A+B</P></TD><TD style="height:20px; width:64px"><P>1,25</P></TD><TD style="height:20px; width:64px"><P>1,25</P></TD><TD style="height:20px; width:64px"><P>6,25</P></TD><TD style="height:20px; width:64px"><P>5,00</P></TD><TD style="height:20px; width:64px"><P>2,50</P></TD><TD style="height:20px; width:64px"><P>0,50</P></TD><TD style="height:20px; width:64px"><P>1,25</P></TD><TD style="height:20px; width:64px"><P>0,00</P></TD><TD style="height:20px; width:64px"><P>7,00</P></TD><TD style="height:20px; width:64px"><P>25,00</P></TD></TR><TR><TD style="height:20px; width:64px"><P>13</P></TD><TD style="height:20px; width:64px"><P>FS032</P></TD><TD style="height:20px; width:64px"><P>-10-23D</P></TD><TD style="height:20px; width:64px"><P>1,25</P></TD><TD style="height:20px; width:64px"><P>1,25</P></TD><TD style="height:20px; width:64px"><P>0</P></TD><TD style="height:20px; width:64px"><P>0</P></TD><TD style="height:20px; width:64px"><P>2,5</P></TD><TD style="height:20px; width:64px"><P>0,5</P></TD><TD style="height:20px; width:64px"><P>1,25</P></TD><TD style="height:20px; width:64px"><P>0</P></TD><TD style="height:20px; width:64px"><P>18,25</P></TD><TD style="height:20px; width:64px"><P>25,00</P></TD></TR><TR><TD style="height:20px; width:64px"><P>14</P></TD><TD style="height:20px; width:64px"><P>FS033</P></TD><TD style="height:20px; width:64px"><P>-7-18D</P></TD><TD style="height:20px; width:64px"><P>1,25</P></TD><TD style="height:20px; width:64px"><P>1,25</P></TD><TD style="height:20px; width:64px"><P>0</P></TD><TD style="height:20px; width:64px"><P>0</P></TD><TD style="height:20px; width:64px"><P>2,5</P></TD><TD style="height:20px; width:64px"><P>0,5</P></TD><TD style="height:20px; width:64px"><P>1,25</P></TD><TD style="height:20px; width:64px"><P>0</P></TD><TD style="height:20px; width:64px"><P>18,25</P></TD><TD style="height:20px; width:64px"><P>25,00</P></TD></TR><TR><TD style="height:20px; width:64px"><P>15</P></TD><TD style="height:20px; width:64px"><P>FS019</P></TD><TD style="height:20px; width:64px"><P>-10-23DmLink</P></TD><TD style="height:20px; width:64px"><P>1,25</P></TD><TD style="height:20px; width:64px"><P>1,25</P></TD><TD style="height:20px; width:64px"><P>0</P></TD><TD style="height:20px; width:64px"><P>0</P></TD><TD style="height:20px; width:64px"><P>2,5</P></TD><TD style="height:20px; width:64px"><P>0,5</P></TD><TD style="height:20px; width:64px"><P>1,25</P></TD><TD style="height:20px; width:64px"><P>0</P></TD><TD style="height:20px; width:64px"><P>18,25</P></TD><TD style="height:20px; width:64px"><P>25,00</P></TD></TR><TR><TD style="height:20px; width:64px"><P>16</P></TD><TD style="height:20px; width:64px"><P>FS027</P></TD><TD style="height:20px; width:64px"><P>-7-18DmLink</P></TD><TD style="height:20px; width:64px"><P>1,25</P></TD><TD style="height:20px; width:64px"><P>1,25</P></TD><TD style="height:20px; width:64px"><P>0</P></TD><TD style="height:20px; width:64px"><P>0</P></TD><TD style="height:20px; width:64px"><P>2,5</P></TD><TD style="height:20px; width:64px"><P>0,5</P></TD><TD style="height:20px; width:64px"><P>1,25</P></TD><TD style="height:20px; width:64px"><P>0</P></TD><TD style="height:20px; width:64px"><P>18,25</P></TD><TD style="height:20px; width:64px"><P>25,00</P></TD></TR><TR><TD style="height:20px; width:64px"><P>17</P></TD><TD style="height:20px; width:64px"><P>FS017+18</P></TD><TD style="height:20px; width:64px"><P>-10-23D_A+B</P></TD><TD style="height:20px; width:64px"><P>1,25</P></TD><TD style="height:20px; width:64px"><P>1,25</P></TD><TD style="height:20px; width:64px"><P>6,25</P></TD><TD style="height:20px; width:64px"><P>0</P></TD><TD style="height:20px; width:64px"><P>2,5</P></TD><TD style="height:20px; width:64px"><P>0,5</P></TD><TD style="height:20px; width:64px"><P>1,25</P></TD><TD style="height:20px; width:64px"><P>0</P></TD><TD style="height:20px; width:64px"><P>12</P></TD><TD style="height:20px; width:64px"><P>25,00</P></TD></TR><TR><TD style="height:20px; width:64px"><P>18</P></TD><TD style="height:20px; width:64px"><P>FS025+26</P></TD><TD style="height:20px; width:64px"><P>-7-18D_A+B</P></TD><TD style="height:20px; width:64px"><P>1,25</P></TD><TD style="height:20px; width:64px"><P>1,25</P></TD><TD style="height:20px; width:64px"><P>6,25</P></TD><TD style="height:20px; width:64px"><P>0</P></TD><TD style="height:20px; width:64px"><P>2,5</P></TD><TD style="height:20px; width:64px"><P>0,5</P></TD><TD style="height:20px; width:64px"><P>1,25</P></TD><TD style="height:20px; width:64px"><P>0</P></TD><TD style="height:20px; width:64px"><P>12</P></TD><TD style="height:20px; width:64px"><P>25,00</P></TD></TR><TR><TD style="height:20px; width:64px"><P>19</P></TD><TD style="height:20px; width:64px"><P>Adenosine</P></TD><TD style="height:20px; width:64px"><P>Substrate only</P></TD><TD style="height:20px; width:64px"><P>1,25</P></TD><TD style="height:20px; width:64px"><P>0</P></TD><TD style="height:20px; width:64px"><P>0</P></TD><TD style="height:20px; width:64px"><P>5</P></TD><TD style="height:20px; width:64px"><P>2,5</P></TD><TD style="height:20px; width:64px"><P>0,5</P></TD><TD style="height:20px; width:64px"><P>1,25</P></TD><TD style="height:20px; width:64px"><P>3,75</P></TD><TD style="height:20px; width:64px"><P>10,75</P></TD><TD style="height:20px; width:64px"><P>25,00</P></TD></TR></TBODY></TABLE><P> </P><P> </P><TABLE border="1" cellpadding="0" cellspacing="0"><TBODY><TR><TD style="height:31px; width:64px"><P>#</P></TD><TD style="height:31px; width:64px"> </TD><TD style="height:31px; width:64px"> </TD><TD style="height:31px; width:64px"><P>Tris HCl ph 7.5</P></TD><TD style="height:31px; width:64px"><P>DNAzyme A</P></TD><TD style="height:31px; width:64px"><P>DNAzyme B</P></TD><TD style="height:31px; width:64px"><P>Substrate</P></TD><TD style="height:31px; width:64px"><P>NaCl</P></TD><TD style="height:31px; width:64px"><P>MgCl2</P></TD><TD style="height:31px; width:64px"><P>SDS</P></TD><TD style="height:31px; width:64px"><P>Kan</P></TD><TD style="height:31px; width:64px"><P>H2O</P></TD><TD style="height:31px; width:64px"><P>Final</P></TD></TR><TR><TD style="height:20px; width:64px"><P>20</P></TD><TD style="height:20px; width:64px"><P>FS032</P></TD><TD style="height:20px; width:64px"><P>10-23D</P></TD><TD style="height:20px; width:64px"><P>1,25</P></TD><TD style="height:20px; width:64px"><P>1,25</P></TD><TD style="height:20px; width:64px"><P>0</P></TD><TD style="height:20px; width:64px"><P>5</P></TD><TD style="height:20px; width:64px"><P>2,5</P></TD><TD style="height:20px; width:64px"><P>0,5</P></TD><TD style="height:20px; width:64px"><P>1,25</P></TD><TD style="height:20px; width:64px"><P>1,25</P></TD><TD style="height:20px; width:64px"><P>12</P></TD><TD style="height:20px; width:64px"><P>25</P></TD></TR><TR><TD style="height:20px; width:64px"><P>21</P></TD><TD style="height:20px; width:64px"><P>FS033</P></TD><TD style="height:20px; width:64px"><P>7-18D</P></TD><TD style="height:20px; width:64px"><P>1,25</P></TD><TD style="height:20px; width:64px"><P>1,25</P></TD><TD style="height:20px; width:64px"><P>0</P></TD><TD style="height:20px; width:64px"><P>5</P></TD><TD style="height:20px; width:64px"><P>2,5</P></TD><TD style="height:20px; width:64px"><P>0,5</P></TD><TD style="height:20px; width:64px"><P>1,25</P></TD><TD style="height:20px; width:64px"><P>1,25</P></TD><TD style="height:20px; width:64px"><P>12</P></TD><TD style="height:20px; width:64px"><P>25</P></TD></TR><TR><TD style="height:20px; width:64px"><P>22</P></TD><TD style="height:20px; width:64px"><P>FS034</P></TD><TD style="height:20px; width:64px"><P>Kan</P></TD><TD style="height:20px; width:64px"><P>1,25</P></TD><TD style="height:20px; width:64px"><P>1,25</P></TD><TD style="height:20px; width:64px"><P>0</P></TD><TD style="height:20px; width:64px"><P>5</P></TD><TD style="height:20px; width:64px"><P>2,5</P></TD><TD style="height:20px; width:64px"><P>0,5</P></TD><TD style="height:20px; width:64px"><P>1,25</P></TD><TD style="height:20px; width:64px"><P>1,25</P></TD><TD style="height:20px; width:64px"><P>12</P></TD><TD style="height:20px; width:64px"><P>25</P></TD></TR><TR><TD style="height:20px; width:64px"><P>23</P></TD><TD style="height:20px; width:64px"><P>FS035</P></TD><TD style="height:20px; width:64px"><P>Kan I</P></TD><TD style="height:20px; width:64px"><P>1,25</P></TD><TD style="height:20px; width:64px"><P>1,25</P></TD><TD style="height:20px; width:64px"><P>0</P></TD><TD style="height:20px; width:64px"><P>5</P></TD><TD style="height:20px; width:64px"><P>2,5</P></TD><TD style="height:20px; width:64px"><P>0,5</P></TD><TD style="height:20px; width:64px"><P>1,25</P></TD><TD style="height:20px; width:64px"><P>1,25</P></TD><TD style="height:20px; width:64px"><P>12</P></TD><TD style="height:20px; width:64px"><P>25</P></TD></TR><TR><TD style="height:20px; width:64px"><P>24</P></TD><TD style="height:20px; width:64px"><P>FS036</P></TD><TD style="height:20px; width:64px"><P>Kan II</P></TD><TD style="height:20px; width:64px"><P>1,25</P></TD><TD style="height:20px; width:64px"><P>1,25</P></TD><TD style="height:20px; width:64px"><P>0</P></TD><TD style="height:20px; width:64px"><P>5</P></TD><TD style="height:20px; width:64px"><P>2,5</P></TD><TD style="height:20px; width:64px"><P>0,5</P></TD><TD style="height:20px; width:64px"><P>1,25</P></TD><TD style="height:20px; width:64px"><P>1,25</P></TD><TD style="height:20px; width:64px"><P>12</P></TD><TD style="height:20px; width:64px"><P>25</P></TD></TR><TR><TD style="height:20px; width:64px"><P>25</P></TD><TD style="height:20px; width:64px"><P>FS037</P></TD><TD style="height:20px; width:64px"><P>Kan III</P></TD><TD style="height:20px; width:64px"><P>1,25</P></TD><TD style="height:20px; width:64px"><P>1,25</P></TD><TD style="height:20px; width:64px"><P>0</P></TD><TD style="height:20px; width:64px"><P>5</P></TD><TD style="height:20px; width:64px"><P>2,5</P></TD><TD style="height:20px; width:64px"><P>0,5</P></TD><TD style="height:20px; width:64px"><P>1,25</P></TD><TD style="height:20px; width:64px"><P>1,25</P></TD><TD style="height:20px; width:64px"><P>12</P></TD><TD style="height:20px; width:64px"><P>25</P></TD></TR><TR><TD style="height:20px; width:64px"><P>26</P></TD><TD style="height:20px; width:64px"><P>FS032</P></TD><TD style="height:20px; width:64px"><P>10-23D</P></TD><TD style="height:20px; width:64px"><P>1,25</P></TD><TD style="height:20px; width:64px"><P>1,25</P></TD><TD style="height:20px; width:64px"><P>0</P></TD><TD style="height:20px; width:64px"><P>5</P></TD><TD style="height:20px; width:64px"><P>2,5</P></TD><TD style="height:20px; width:64px"><P>0,5</P></TD><TD style="height:20px; width:64px"><P>1,25</P></TD><TD style="height:20px; width:64px"><P>0</P></TD><TD style="height:20px; width:64px"><P>13,25</P></TD><TD style="height:20px; width:64px"><P>25</P></TD></TR><TR><TD style="height:20px; width:64px"><P>27</P></TD><TD style="height:20px; width:64px"><P>FS033</P></TD><TD style="height:20px; width:64px"><P>7-18D</P></TD><TD style="height:20px; width:64px"><P>1,25</P></TD><TD style="height:20px; width:64px"><P>1,25</P></TD><TD style="height:20px; width:64px"><P>0</P></TD><TD style="height:20px; width:64px"><P>5</P></TD><TD style="height:20px; width:64px"><P>2,5</P></TD><TD style="height:20px; width:64px"><P>0,5</P></TD><TD style="height:20px; width:64px"><P>1,25</P></TD><TD style="height:20px; width:64px"><P>0</P></TD><TD style="height:20px; width:64px"><P>13,25</P></TD><TD style="height:20px; width:64px"><P>25</P></TD></TR><TR><TD style="height:20px; width:64px"><P>28</P></TD><TD style="height:20px; width:64px"><P>FS034</P></TD><TD style="height:20px; width:64px"><P>Kan</P></TD><TD style="height:20px; width:64px"><P>1,25</P></TD><TD style="height:20px; width:64px"><P>1,25</P></TD><TD style="height:20px; width:64px"><P>0</P></TD><TD style="height:20px; width:64px"><P>5</P></TD><TD style="height:20px; width:64px"><P>2,5</P></TD><TD style="height:20px; width:64px"><P>0,5</P></TD><TD style="height:20px; width:64px"><P>1,25</P></TD><TD style="height:20px; width:64px"><P>0</P></TD><TD style="height:20px; width:64px"><P>13,25</P></TD><TD style="height:20px; width:64px"><P>25</P></TD></TR><TR><TD style="height:20px; width:64px"><P>29</P></TD><TD style="height:20px; width:64px"><P>FS035</P></TD><TD style="height:20px; width:64px"><P>Kan I</P></TD><TD style="height:20px; width:64px"><P>1,25</P></TD><TD style="height:20px; width:64px"><P>1,25</P></TD><TD style="height:20px; width:64px"><P>0</P></TD><TD style="height:20px; width:64px"><P>5</P></TD><TD style="height:20px; width:64px"><P>2,5</P></TD><TD style="height:20px; width:64px"><P>0,5</P></TD><TD style="height:20px; width:64px"><P>1,25</P></TD><TD style="height:20px; width:64px"><P>0</P></TD><TD style="height:20px; width:64px"><P>13,25</P></TD><TD style="height:20px; width:64px"><P>25</P></TD></TR><TR><TD style="height:20px; width:64px"><P>30</P></TD><TD style="height:20px; width:64px"><P>FS036</P></TD><TD style="height:20px; width:64px"><P>Kan II</P></TD><TD style="height:20px; width:64px"><P>1,25</P></TD><TD style="height:20px; width:64px"><P>1,25</P></TD><TD style="height:20px; width:64px"><P>0</P></TD><TD style="height:20px; width:64px"><P>5</P></TD><TD style="height:20px; width:64px"><P>2,5</P></TD><TD style="height:20px; width:64px"><P>0,5</P></TD><TD style="height:20px; width:64px"><P>1,25</P></TD><TD style="height:20px; width:64px"><P>0</P></TD><TD style="height:20px; width:64px"><P>13,25</P></TD><TD style="height:20px; width:64px"><P>25</P></TD></TR><TR><TD style="height:20px; width:64px"><P>31</P></TD><TD style="height:20px; width:64px"><P>FS037</P></TD><TD style="height:20px; width:64px"><P>Kan III</P></TD><TD style="height:20px; width:64px"><P>1,25</P></TD><TD style="height:20px; width:64px"><P>1,25</P></TD><TD style="height:20px; width:64px"><P>0</P></TD><TD style="height:20px; width:64px"><P>5</P></TD><TD style="height:20px; width:64px"><P>2,5</P></TD><TD style="height:20px; width:64px"><P>0,5</P></TD><TD style="height:20px; width:64px"><P>1,25</P></TD><TD style="height:20px; width:64px"><P>0</P></TD><TD style="height:20px; width:64px"><P>13,25</P></TD><TD style="height:20px; width:64px"><P>25</P></TD></TR><TR><TD style="height:20px; width:64px"><P>32</P></TD><TD style="height:20px; width:64px"><P>Kan</P></TD><TD style="height:20px; width:64px"><P>Substrate only</P></TD><TD style="height:20px; width:64px"><P>1,25</P></TD><TD style="height:20px; width:64px"><P>0</P></TD><TD style="height:20px; width:64px"><P>0</P></TD><TD style="height:20px; width:64px"><P>5</P></TD><TD style="height:20px; width:64px"><P>2,5</P></TD><TD style="height:20px; width:64px"><P>0,5</P></TD><TD style="height:20px; width:64px"><P>1,25</P></TD><TD style="height:20px; width:64px"><P>1,25</P></TD><TD style="height:20px; width:64px"><P>13,25</P></TD><TD style="height:20px; width:64px"><P>25,00</P></TD></TR><TR><TD style="height:20px; width:64px"><P>33</P></TD><TD style="height:20px; width:64px"> </TD><TD style="height:20px; width:64px"><P>Substrate only</P></TD><TD style="height:20px; width:64px"><P>1,25</P></TD><TD style="height:20px; width:64px"><P>0</P></TD><TD style="height:20px; width:64px"><P>0</P></TD><TD style="height:20px; width:64px"><P>5</P></TD><TD style="height:20px; width:64px"><P>2,5</P></TD><TD style="height:20px; width:64px"><P>0,5</P></TD><TD style="height:20px; width:64px"><P>1,25</P></TD><TD style="height:20px; width:64px"><P>0</P></TD><TD style="height:20px; width:64px"><P>14,5</P></TD><TD style="height:20px; width:64px"><P>25,00</P></TD></TR></TBODY></TABLE><P> </P><P>Results and Outlook:</P><P>Positive controls worked, Adenosine dependency could be detected for one candidate.</P></DIV></DIV></DIV></DIV><DIV class="row"><DIV class="col-lg-12"><DIV class="notebookentry"><H3 class="notebookheading">▼2015-09-10  Click reaction of Label-, Label AU RNA [1, 2, 3] </H3><DIV class="notebooktext"><TABLE border="1" cellpadding="0" cellspacing="0"><TBODY><TR><TD style="width:153px"><P><EM>For 51.5µL</EM></P></TD><TD style="width:153px"><P>Cstock</P></TD><TD style="width:153px"><P>Cfinal</P></TD><TD style="width:153px"><P>V[µL]</P></TD></TR><TR><TD style="width:153px"><P>Phosphate Buffer - pH 7, 0.1M</P></TD><TD style="width:153px"><P>100mM</P></TD><TD style="width:153px"><P>50mM</P></TD><TD style="width:153px"><P>25</P></TD></TR><TR><TD style="width:153px"><P>Alexa 488 azide</P></TD><TD style="width:153px"><P>10µM</P></TD><TD style="width:153px"><P>400nM</P></TD><TD style="width:153px"><P>2</P></TD></TR><TR><TD style="height:28px; width:153px"><P>RNA</P></TD><TD style="height:28px; width:153px"><P>1µM</P></TD><TD style="height:28px; width:153px"><P>200nM</P></TD><TD style="height:28px; width:153px"><P>10</P></TD></TR><TR><TD style="width:153px"><P>CuSO4</P></TD><TD style="width:153px"><P>20mM</P></TD><TD style="width:153px"><P>1mM</P></TD><TD style="width:153px"><P>2.5</P></TD></TR><TR><TD style="width:153px"><P>THPTA</P></TD><TD style="width:153px"><P>50mM</P></TD><TD style="width:153px"><P>5mM</P></TD><TD style="width:153px"><P>5</P></TD></TR><TR><TD style="width:153px"><P>NaAsc</P></TD><TD style="width:153px"><P>100mM</P></TD><TD style="width:153px"><P>1mM</P></TD><TD style="width:153px"><P>0.5</P></TD></TR><TR><TD style="width:153px"><P>H2O</P></TD><TD style="width:153px"><P> </P></TD><TD style="width:153px"><P> </P></TD><TD style="width:153px"><P>6.5</P></TD></TR></TBODY></TABLE><P> </P><UL><LI>Alexa 488 azide was solved in DMSO</LI><LI>Incubation at 37 °C for 12-14 hours (overnight)</LI></UL></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV><DIV class="container"><DIV class="content"><DIV class="row"><DIV class="col-lg-12"><DIV class="panel panel-default"><DIV class="panel-heading"><H3 class="basicheader">week number 38</H3></DIV><DIV class="panel-body"><DIV class="row"><DIV class="col-lg-12"><DIV class="notebookentry"><H3 class="notebookheading">▼2015-09-14  Test bitte löschen</H3><DIV class="notebooktext"><P>10 x &lt;i&gt;in vitro&lt;/i&gt; transcription buffer</P><P>50 mM Tris ph 7.5, 100 mM NaCl, 20 mM MgCl&lt;sub&gt;2&lt;/sub&gt;, 0,01 % SDS </P></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></BODY></HTML>