Team:Heidelberg/Parts
Achievements
Results
Medal fulfillments
AptaBodies
Real time S.M.S.
Cystic fibrosis
Labeling and vizualization
Modeling and Software validation
Standardization
Project
Overview
AptaBodies
Real time S.M.S.
Cystic fibrosis
Labelling and Visualization
HRP based detection
Software
MAWS
JAWS
Wiki Tools
Notebook
AptaBodies
Real time S.M.S.
Cystic fibrosis
Labeling and vizualization
Pulse chase
Materials & Methods
Safety
Parts
Team
Members
Supervisors
Collaboration
Attributions
Sponsors
Modeling
Aptamer Sensors
In vitro transcription
Outreach
iGEM Team Heidelberg 2015
Achievements
Results
Project
Software
Notebook
Parts
Team
Modeling
Outreach
Modularity and Standardization
As functional DNA in iGEM has been rarely mentioned we want to introduce it to iGEM. We want to provide standard parts that everyone can use to make the work with functional DNA and also RNA accessible for the whole community. To simplify RNA work we designed a new BBF RFC 110 to standardize the processes necessary for this.
Standard for RNA work
RFC 110
Overview of BBF RFC 110
The coloning into RFC 110 is performed without restiction sites to avoid problems with consered restriction sites in functional sequences. We provide primers to linearize the plasmid in a two step PCR. Furthermore we provide 5' and 3'  extension sequences that have to be added to the insert. These extensions carry the homology that is needed for CPEC or Gibson Assembly. Plasmid can be transformed into E. coli BL21 DE3 for in vivo experiments or plamids can be use as template for in vitro RNA transcription. Alternatively another set of Primers is provided for extraction of a DNA template for in vitro transcription.
This new BBF RFC 110 is very important for the work with RNA as the cloning strategy does not rely on any restriction site. We designed it this way because if you work with a functional RNA the sequence is extremely essential. In most cases the sequence is laboriously optimized with several cycles of SELEX. If the functional RNA contains a cut sites usually one cannot simply mutate it as it is possible for proteins. Any mutation alters the effector sequence directly and thus can have severe effects on the functionality.
Standard for DNA work
Furthermore we have standardized the work with functional DNA. Has functional DNA usually is ssDNA it can NOT be cloned into a plasmid for storage. But every single strand functional DNA can simply be ordered as oligos.
In our project we mainly combined aptamer with a catalytic nucleic acid and thus create an aptazyme. However some constructs are more complicated than this (see HRP based detection). The parts we use can be combined in different constructs. All DNA parts described by us can be combined in may ways to create devices with a variety of functions and as such are very useful platform for the iGEM community.
<groupparts>iGEM015 Heidelberg</groupparts>
Parts
We would like to point out several of our parts as they were crucial and fundamental for our work. Below this section, a list of all entered parts can be found.
BBa_K1614007 - Favorite Part
We deliberately chose BBa_K1614007 - the HRP-mimicking DNAzyme - as favorite part, even though we cannot physically submit it, as it is a 17 bp long DNA-oligomer. Despite its small size, this functional DNA is capable of forming a G-quadruplex. Upon incorporation of hemin, a chemiluminescent reaction can be performed. We used this striking feature for many optical readouts, namely the AptaBody approach, the Labelling and Visualization and the Validation of the Software. This part is hence so central to our work that we would find it inappropriate to not appreciate and acknowledge this part aptly.
BBa_K1614002 - The Basis of our RFC 110
A highly relevant series of parts that is based upon BBa_K1614002 are all parts associated to our RFC (as also described above). In our RFC 110, we describe a standardized procedure for the production of small functional RNA. The storage of the constructs, as well as the integration into the in vitro transcription - cassette is suggested to be performed in BBa_K1614002. Therefore, this part is also central to our work and by including it into the registry, we want to enable the entire community of synthetic biology to generate small functional RNA in a standardized, yet versatile way.
BBa_K1614019 - A Part we Improved
One of the main strengths of synthetic biology, compared to other scientific disciplines is the strong community it is build upon. Yet this community is nothing without active participation and interaction. We therefore see it as a very important responsibility of all iGEM-teams to constantly improve and newly apply parts submitted to the registry. As we were working with functional nucleotides, we soon took notice of BBa_K1330000, better known as Spinach2.1 from DTU Denmark iGEM team 2014. This modified version of Spinach2 was made compatible with the RFC 10 standard without change of functionality.
Envisionning the introduction of the power of aptamers and aptazymes to the iGEM community, we improved part BBa_K1330000 by making it depending on and thereby capable of sensing ATP - we call it BBa_K1614019. We show a possible application of this improved part in the optimization of in vitro transcription conditions and furthermore enhance the readout and the activity by software engineering. Besides, one could imagine sensing other small molecules in a standardized way.
BBa_K1614016 - Malachite Green aptamer
Establishing a read out system that would give the information that the DNA was fully transcribed into RNA was provided by the Malachite Green Aptamer. Therefore we provide a new biobrick which contains a hammerhead ribozyme fused to the Malachite Green Aptamer (BBa_K1614016). This aptamer can bind malachite green dye which lead to a fluorescence at 652 nm if excited at 630 nm. This part was used for the detection of fully extended RNA during in vitro transcription in presence of malachite green dye. The ligand dependent aptamer folds correctly during the transcription thereby emitting light that was measured in a 384 well plate format. Experiments have shown that the Malachite Green Aptamer can be used as a universal read out system by adding a Hammerhead Ribozyme that cleaves at the 3â€™ end of the RNA of Interest. This setup allows the cleavage of an RNA of Interest (ROI) from the Hammerhead Ribozyme-Malachite Green Aptamer.
Experiments with the Substrate of the RNA cleaving DNAzyme have shown that the Hammerhead Ribozyme-Malachite Green Aptamer cleaves from the substrate. Hence, the substrate and the Hammerhead Ribozyme-Malachite Green Aptamer could be purified on a denaturing polyacrylamide gel electrophoresis.
