<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_Heidelberg_project_cf skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:Heidelberg/project/cf</SPAN></H1><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><DIV class="container" id="menucontainer"><DIV id="navcontainer-right-outer"><DIV id="navcontainer-right-inner"><UL class="nav navbar-nav navbar-right"><LI><A href="https://2015.igem.org/Team:Heidelberg/Achievements" class="menubox"><P class="menutext">Achievements</P></A></LI><LI class="dropdown"><A href="https://2015.igem.org/Team:Heidelberg/Results" class="dropdown-toggle menubox" data-toggle="dropdown" role="button" aria-haspopup="true" aria-expanded="false"><P class="menutext">Results
                        </P></A></LI><LI class="dropdownbox"><A href="https://2015.igem.org/Team:Heidelberg/Results#Medalfulfillments"><P class="menutext dropdown-text">Medal fulfillments</P></A></LI><LI class="dropdownbox"><A href="https://2015.igem.org/Team:Heidelberg/Results#AB"><P class="menutext dropdown-text">AptaBodies</P></A></LI><LI class="dropdownbox"><A href="https://2015.igem.org/Team:Heidelberg/Results#RTSMS"><P class="menutext dropdown-text">Real time S.M.S.</P></A></LI><LI class="dropdownbox"><A href="https://2015.igem.org/Team:Heidelberg/Results#LandV"><P class="menutext dropdown-text">Cystic fibrosis</P></A></LI><LI class="dropdownbox"><A href="https://2015.igem.org/Team:Heidelberg/Results#CF"><P class="menutext dropdown-text">Labeling and vizualization</P></A></LI><LI class="dropdownbox"><A href="https://2015.igem.org/Team:Heidelberg/Results#Modelling"><P class="menutext dropdown-text">Modeling and Software validation</P></A></LI><LI class="dropdownbox"><A href="https://2015.igem.org/Team:Heidelberg/Results/Standardization"><P class="menutext dropdown-text">Standardization</P></A></LI></UL><LI class="dropdown"><A href="https://2015.igem.org/Team:Heidelberg/Overview" class="dropdown-toggle menubox" data-toggle="dropdown" role="button" aria-haspopup="true" aria-expanded="false"><P class="menutext">Project
                        </P></A></LI><LI class="dropdownbox"><A href="https://2015.igem.org/Team:Heidelberg/Overview"><P class="menutext dropdown-text">Overview</P></A></LI><LI class="dropdownbox"><A href="https://2015.igem.org/Team:Heidelberg/Project/AB"><P class="menutext dropdown-text">AptaBodies</P></A></LI><LI class="dropdownbox"><A href="https://2015.igem.org/Team:Heidelberg/project/rtsms"><P class="menutext dropdown-text">Real time S.M.S.</P></A></LI><LI class="dropdownbox"><A href="https://2015.igem.org/Team:Heidelberg/project/cf"><P class="menutext dropdown-text">Cystic fibrosis</P></A></LI><LI class="dropdownbox"><A href="https://2015.igem.org/Team:Heidelberg/project/rd"><P class="menutext dropdown-text">Labelling and Visualization</P></A></LI><LI class="dropdownbox"><A href="https://2015.igem.org/Team:Heidelberg/project/hlpd"><P class="menutext dropdown-text">HRP based detection</P></A></LI><LI class="dropdown"><A href="https://2015.igem.org/Team:Heidelberg/Software" class="dropdown-toggle menubox" data-toggle="dropdown" role="button" aria-haspopup="true" aria-expanded="false"><P class="menutext">Software
                        </P></A></LI><LI class="dropdownbox"><A href="https://2015.igem.org/Team:Heidelberg/software/maws"><P class="menutext dropdown-text">MAWS</P></A></LI><LI class="dropdownbox"><A href="https://2015.igem.org/Team:Heidelberg/software/jaws"><P class="menutext dropdown-text">JAWS</P></A></LI><LI class="dropdownbox"><A href="
https://2015.igem.org/Team:Heidelberg/WikiTools"><P class="menutext dropdown-text">Wiki Tools</P></A></LI><LI class="dropdown"><A href="https://2015.igem.org/Team:Heidelberg/notebook/mm" class="dropdown-toggle menubox" data-toggle="dropdown" role="button" aria-haspopup="true" aria-expanded="false"><P class="menutext">Notebook
                        </P></A></LI><LI class="dropdownbox"><A href="https://2015.igem.org/Team:Heidelberg/notebook/ab"><P class="menutext dropdown-text">AptaBodies</P></A></LI><LI class="dropdownbox"><A href="https://2015.igem.org/Team:Heidelberg/notebook/rtsms"><P class="menutext dropdown-text">Real time S.M.S.</P></A></LI><LI class="dropdownbox"><A href="https://2015.igem.org/Team:Heidelberg/notebook/cf"><P class="menutext dropdown-text">Cystic fibrosis</P></A></LI><LI class="dropdownbox"><A href="https://2015.igem.org/Team:Heidelberg/notebook/rd"><P class="menutext dropdown-text">Labeling and vizualization</P></A></LI><LI class="dropdownbox"><A href="https://2015.igem.org/Team:Heidelberg/notebook/pc"><P class="menutext dropdown-text">Pulse chase</P></A></LI><LI class="dropdownbox"><A href="https://2015.igem.org/Team:Heidelberg/notebook/mm"><P class="menutext dropdown-text">Materials &amp; Methods</P></A></LI><LI class="dropdownbox"><A href="https://2015.igem.org/Team:Heidelberg/Safety"><P class="menutext dropdown-text">Safety</P></A></LI><LI><A href="https://2015.igem.org/Team:Heidelberg/Parts" class="menubox"><P class="menutext">Parts</P></A></LI><LI class="dropdown"><A href="https://2015.igem.org/Team:Heidelberg/Team" class="dropdown-toggle menubox" data-toggle="dropdown" role="button" aria-haspopup="true" aria-expanded="false"><P class="menutext">Team
                        </P></A></LI><LI class="dropdownbox"><A href="https://2015.igem.org/Team:Heidelberg/Team"><P class="menutext dropdown-text">Members</P></A></LI><LI class="dropdownbox"><A href="https://2015.igem.org/Team:Heidelberg/Supervisors"><P class="menutext dropdown-text">Supervisors</P></A></LI><LI class="dropdownbox"><A href="https://2015.igem.org/Team:Heidelberg/Collaborations"><P class="menutext dropdown-text">Collaboration</P></A></LI><LI class="dropdownbox"><A href="https://2015.igem.org/Team:Heidelberg/Attributions"><P class="menutext dropdown-text">Attributions</P></A></LI><LI class="dropdownbox"><A href="https://2015.igem.org/Team:Heidelberg/Sponsors"><P class="menutext dropdown-text">Sponsors</P></A></LI><LI class="dropdown"><A href="https://2015.igem.org/Team:Heidelberg/Modeling" class="dropdown-toggle menubox" data-toggle="dropdown" role="button" aria-haspopup="true" aria-expanded="false"><P class="menutext">Modeling
                        </P></A></LI><LI class="dropdownbox"><A href="https://2015.igem.org/Team:Heidelberg/Modeling/aptakinetics"><P class="menutext dropdown-text">Aptamer Sensors</P></A></LI><LI class="dropdownbox"><A href="https://2015.igem.org/Team:Heidelberg/Modeling/rtsms"><P class="menutext dropdown-text">In vitro transcription</P></A></LI><LI><A href="https://2015.igem.org/Team:Heidelberg/Practices" class="menubox"><P class="menutext">Outreach</P></A></LI></DIV></DIV></DIV><NAV id="myNavmenu" class="navmenu navmenu-default navmenu-fixed-left offcanvas" role="navigation"><A class="navmenu-brand" href="https://2015.igem.org/Team:Heidelberg">iGEM Team Heidelberg 2015</A><UL class="nav navmenu-nav"><LI><A href="https://2015.igem.org/Team:Heidelberg/Achievements">Achievements</A></LI><LI><A href="https://2015.igem.org/Team:Heidelberg/Results">Results</A></LI><LI><A href="https://2015.igem.org/Team:Heidelberg/Overview">Project</A></LI><LI><A href="https://2015.igem.org/Team:Heidelberg/Software">Software</A></LI><LI><A href="https://2015.igem.org/Team:Heidelberg/notebook/mm">Notebook</A></LI><LI><A href="https://2015.igem.org/Team:Heidelberg/Parts">Parts</A></LI><LI><A href="https://2015.igem.org/Team:Heidelberg/Team">Team</A></LI><LI><A href="https://2015.igem.org/Team:Heidelberg/Modeling">Modeling</A></LI><LI><A href="https://2015.igem.org/Team:Heidelberg/Practices">Outreach</A></LI></UL></NAV><DIV class="container"><DIV class="content"><DIV class="row"><DIV class="col-lg-12"><DIV class="panel panel‐default"><DIV class="panel‐heading"><H3 class="basicheader"> Abstract </H3></DIV><DIV class="panel‐body"><DIV class="row"><DIV class="col-lg-12"><P class="basictext">
Cystic fibrosis (CF) is a genetic disorder occurring in about 1:2000 to 1:3000 people in Europe and about 1:100000 to 1:350000 in Japan <X-REF>WHO-2004</X-REF>, it is caused by a wide range of mutations (http://www.genet.sickkids.on.ca/StatisticsPage.html, Statistics Page of the CFTR mutation database [accessed 29.08.2015]). Most people affected by this defect have Caucasian origin <X-REF>WHO-2004</X-REF>. The life of cystic fibrosis patients is threatened and the life quality is severely impaired. Patients suffer from several symptoms: High chloride sweat, infertility in male patients, digestion problems and obstipation, but the most severe symptom is the thickening of the mucus in the lung, which leads to lung obstipation, respiration problems and bacterial infections <X-REF>Andersen-1958</X-REF>. The life expectancy of a new-born cystic fibrosis patient is merely 40 years low, mostly because of the lung disease <X-REF>MacKenzie-2010</X-REF>. We report here the application of man-made functional RNA, called twin-ribozymes <X-REF>Mueller2003</X-REF>, to medicate CF-patients. These twin-ribozymes could correct a specific mutation on the RNA level occurring in about 80% of the patients, by replacing the segment with the mutation (Fig. 1).
</P><DIV class="col-lg-12"><DIV class="imagewrapper"><DIV class="imagewrappercaption"><STRONG>
                Figure 1: Project overview of cystic fibrosis
            </STRONG><P class="basictext">
                A defect mRNA of a cystic fiborosis patient, with the δF508-mutation leads to a misfolded cystic fibrosis transmembrane conductance regulator protein (CFTR). Therefore the patient has to suffer from multiple severe symptoms. In order to repair the mutation, twin-ribozymes were used. Twin-ribozymes are able to cut out a mutated region in the mRNA and insert a repair fragment. This leads to a functional CFTR-protein.
            </P></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV><DIV class="container"><DIV class="content"><DIV class="row"><DIV class="col-lg-12"><DIV class="panel panel‐default"><DIV class="panel‐heading"><H3 class="basicheader"> Introduction </H3></DIV><DIV class="panel‐body"><DIV class="row"><DIV class="col-lg-12"><H3 class="subheader"> Cystic Fibrosis and Causes of the Disease </H3><P class="basictext">
Cystic fibrosis (CF) is a recessive monogenetic disorder, which is present in every ethnicity worldwide. Especially Europeans are affected with 1:2000 to 1:3000 people exhibiting cystic fibrosis <X-REF>WHO-2004</X-REF>, while Japanese people are least affected with an incidence rate of 1:100000 to 1:350000. In Germany the incidence rate for cystic fibrosis is 1:3300, in the USA it is 1:3500. Ireland has the highest incidence rate with 1:1800 people <X-REF>WHO-2004</X-REF>. 
</P><P class="basictext">
The disease can be a result of multiple different genetic mutations. At the moment 2001 mutations, which result in CF, are known. The most abundant mutation is the ΔF508, also called Phe508del or F508del mutation (http://www.genet.sickkids.on.ca/StatisticsPage.html, Statistics Page of the CFTR mutation database [accessed 29.08.2015]), which occurs in about 70% of western, central, northern and north-eastern Europe. In Germany, the prevalence of the mutation lies between 63.5% and 74.2% depending on the region. On the Faroe Islands 100% of cystic fibrosis patients exhibit the ΔF508 mutation <X-REF>WHO-2004</X-REF>.
</P><DIV class="col-lg-4"><DIV class="imagewrapper"><DIV class="imagewrappercaption"><STRONG>
                Figure 2: A crystal structure model of the CFTR-channel obtained by Rosenberg, et al
            </STRONG></DIV></DIV></DIV><P class="basictext">
The mutations leading to CF occur in one particular gene: The Cystic Fibrosis Transmembrane Conductance Regulator (CFTR) gene (for crystal structure see Fig. 2). The CFTR-gene encodes for a transmembrane anion channel, expressed in epithelial cells. The CFTR-protein is part of the ATP-binding cassette protein superfamily. Anions, especially chloride ions, are able to pass the membrane passively along the concentration gradient. ATP helps to stabilize the open channel to establish a higher anion flux through the channel. The CFTR consists of 5 domains: Two nucleotide binding domains (NBD1 and NBD2), two transmembrane domains (TMD1 and TMD2) and one regulatory domain (RD). The CFTR-channel is moreover regulated by protein kinase A (PKA) and ATP <X-REF>Odolczyk-2014</X-REF>.
</P><P class="basictext">
Out of the 2001 mutations leading to CF the most common ΔF508-mutation leads to a misfolding of the CFTR-protein. This leads to a faster degradation, a reduction in function and consequently a damping in cell membrane integration rate <X-REF>Amaral-2015</X-REF>. Other mutations can cause missplicing, loss of function or even completely absence of the protein <X-REF>Amaral-2015</X-REF>, <X-REF>WHO-2004</X-REF>.
</P><H3 class="subheader"> Symptoms of Cystic Fibrosis </H3><P class="basictext">
In the 1950s, most patients died in infancy or childhood <X-REF>Andersen-1958</X-REF>, in 2010 the suggested life expectancy of new-born CF-patients was 39 years <X-REF>MacKenzie-2010</X-REF>. This is caused by the manifold symptoms patients suffer from: High chloride sweat, male infertility, digestion problems and obstipation, pancreatic insuffency, as well as liver problems, leading to liver cirrhosis <X-REF>Andersen-1958</X-REF>. The most severe symptoms are the respiratory problems, which significantly reduce the life expectancy and gravely impairs the life quality of cystic fibrosis patients <X-REF>Andersen-1958</X-REF>, <X-REF>Cutting-2015</X-REF>. They are caused by thickening of mucus, which provides an ideal environment for bacterial infections <X-REF>Cutting-2015</X-REF>. These infections will be fought by the human immune system, which leads to lysis of both, bacteria and immune cells and thereby to a further thickening of the mucus by released bacterial and cellular lysis products such as free DNA, parts of the cell walls and cell membranes <X-REF>MacKenzie-2010</X-REF>. Through the development of strategies and medications to fight the disorder, the child mortality of CF-patients could be strongly reduced.
</P><H3 class="subheader"> Approaches in Medication and Therapy </H3><P class="basictext">
The classical approach to medicate cystic fibrosis is a mixture of nebulization therapy, physiotherapy, physical exercise, dietaries and antibiotic therapy <X-REF>Matthews-1964</X-REF>. In the nebulization therapy water is evaporated and dispensed in the breathing air of cystic fibrosis patients. This results in a higher fluidity of the lung mucus. With the nebulization therapy patients can cough up the thick mucus more easily, which results in a better respiratory function. Medication, like bronchodilatic drugs, mucolytics and antibiotics, was often given into the nebula <X-REF>Matthews-1964</X-REF>. Another therapy is the physiotherapy and physical exercises in which helps with the removal of the mucus by massaging the airway or by loosen the mucus through higher lung function in physical exercise <X-REF>Matthews-1964</X-REF>. Dietaries were used to overcome the digestion problems and high chloride excretion <X-REF>Matthews-1964</X-REF>, <X-REF>Haack2013</X-REF>. Antibiotics were often permanently used to fight lung infections, even though the bacteria showed resistances <X-REF>Matthews-1964</X-REF>. Another option is the transplantation of organs, like the lungs <X-REF>Yankaskas1998</X-REF>. The classical way to diagnose cystic fibrosis is an electrical measurement of the chloride sweat <X-REF>Gibson1959</X-REF>. An early diagnosis helps to stop the progression of CF through an earlier treatment. Without the sweat test the detection of the defects in infants or young children can be hard. This especially affects patients with less severe symptoms such as chronic cough, chronic diarrhoea and malnutrition.
</P><P class="basictext">
The used drugs to fight cystic fibrosis belong mostly to the following classes: Bronchodilatorica, mucolytics, antibiotics and enzymes. Prominent examples of CF medications are dornase alfa <X-REF>Jones2010</X-REF> and pancreatic enzymes <X-REF>Haack2013</X-REF>, <X-REF>Somaraju2014</X-REF>. The most reknown example for a mucolytic used in CF treatment is Dornase alfa, a human recombinant DNAse (hrDNAse) <X-REF>Jones2010</X-REF>. Especially long DNA strands strongly increase the viscosity of the mucus. In the 1950s the loosening of mucus by a bovine DNAse was proved the first time <X-REF>Somaraju2014</X-REF>. The medication of CF with bovine dornase was damped because of adverse effects like bronchospasms <X-REF>Raskin1968</X-REF>. In 1990 Dornase alfa was produced for the first time. It displayed significantly lower side effects than the bovine dornase, because of the decreased immune response due to its nature as human protein <X-REF>Somaraju2014</X-REF>. There are also novel approaches to treat CF such as medication to stabilize the ΔF508-CFTR-protein to prevent it from degradation <X-REF>Fuller2000</X-REF>.
</P><P class="basictext">
Other approaches to treat cystic fibrosis want to cure cystic fibrosis, by targeting its cause, with gene therapy. Gene therapy approaches to medicate CF can be divided in two categories: Non-viral gene therapy <X-REF>Alton2015</X-REF> and viral gene therapy <X-REF>Drumm1990</X-REF>. Non-viral gene therapy for cystic fibrosis works via the application of the CFTR-gene. One trial for non-viral gene therapy, working via the application of a CFTR-gene liposome complex, is currently in the clinical study phase 2b. Here the patient gets a statistically relevant, but only small effect. The gene-liposome complex has been applied every 28 days for 1 year <X-REF>Alton2015</X-REF>. The non-viral gene therapy approaches show almost no adverse effects <X-REF>Alton2015</X-REF>. Viral-gene therapy also aims either to deliver the CFTR-gene into the cells <X-REF>Crystal1994</X-REF>. The two most used vectors are adeno- <X-REF>Crystal1994</X-REF> and adeno-associated virus vectors <X-REF>Aitken2001</X-REF>. Viral vectors lead often to inflammatory responses or common cold symptoms. This is a reaction of immune system on the administration of the viral particles <X-REF>Crystal1994</X-REF>, <X-REF>Zuckerman1999</X-REF>. Also patients develop immunities against the viral particles after at least 3 applications <X-REF>Harvey1999</X-REF>.
</P><DIV class="col-lg-6"><DIV class="imagewrapper"><DIV class="imagewrappercaption"><STRONG>
                Figure 3: Schematic of a Twin-ribozyme reaction for cystic fibrosis medication
            </STRONG><P class="basictext">
                The reaction consists of two steps: First the ribozymes cleaves at its target site and releases the cut out RNA fragment. Second the ribozyme binds the insert-fragment with three additional bases coding for phenylalanine and inserts it into the target mRNA.
            </P></DIV></DIV></DIV><H3 class="subheader"> Transsplicing Ribozymes and Twinzymes </H3><P class="basictext">
Amongst all small self-cleaving ribozymes, the hairpin ribozyme is very well characterized <X-REF>Fedor2000</X-REF><X-REF>Mueller2012</X-REF>, with our knowledge about it being second only to the extensive characterization of the hammerhead ribozyme <X-REF>Hammann2012</X-REF><X-REF>Birikh1996</X-REF>. It is capable of both cleavage and ligation, the mechanisms and structural requirements of which are well-studied<X-REF>Fedor2000</X-REF>. This knowledge of both structure and activity of the hairpin allows us to modify and adapt it to the needs of specific goals. As of this time, numerous adaptions of the hairpin including, but not limited to trans-cleavage enabled ribozymes <X-REF>PerezRuiz1999</X-REF><X-REF>Hampel1989</X-REF>, efficient ligases <X-REF>Paul2002</X-REF><X-REF>Kazakov2006</X-REF><X-REF>Vlassov</X-REF>, combinations of both<X-REF>Drude2007</X-REF><X-REF>Balke2014</X-REF> and ribozymes controlled by the addition of ligands<X-REF>Meli2003</X-REF><X-REF>NajafiShoushtari2007</X-REF>. Also, the hairpin ribozyme itself has been redesigned numerous times, based on structural data, yielding different or even more efficient versions, by stabilizing its active conformation <X-REF>Komatsu1995</X-REF><X-REF>Komatsu1996</X-REF>, and expanding its repertoire of cleavable sequences by the readjustment of tertiary interactions<X-REF>Drude2011</X-REF><X-REF>Balke2014</X-REF>. The hairpin ribozyme itself consists of two stems, each containing a bulge loop, termed A and B respectively<X-REF>Müller2011</X-REF>, where loop A contains the cleavage site, with an extended consensus sequence YNGUHN, and loop B establishes tertiary interactions vital to ribozyme activity <X-REF>Kath-Schorr2012</X-REF>. From a map of those interactions, as given in <X-REF>Sumita2013</X-REF> and a target for cleavage and ligation, one could in principle use structure-based design to engineer a corresponding trans-acting ribozyme to catalyse that reaction. Furthermore, the hairpin ribozyme has been shown to be capable of working as multiple copies in tandem, either to both self-cleave and trans-cleave, or to trans-cleave and ligate a target strand at multiple sites<X-REF>Balke2014</X-REF><X-REF>Drude2007</X-REF><X-REF>Welz2000</X-REF>. To this end, one can use both the standard version of the ribozyme, and the reverse-linked ribozyme<X-REF>Komatsu1995</X-REF><X-REF>Welz2000</X-REF><X-REF>Schmidt2000</X-REF><X-REF>Balke2011</X-REF><X-REF>Balke2014</X-REF><X-REF>Drude2007</X-REF>, which allows for additional flexibility when designing these so-called 'Twin Ribozymes' or 'Twinzyme' (Fig. 3).
</P><H3 class="subheader"> Using Twinzymes as Therapeutical Approach </H3><P class="basictext">
Twin ribozymes are a new application for the treatment of cystic fibrosis. Through mRNA-editing, we could establish a ribozymal gene therapy and revolutionize the treatment of CF. Compared to the viral and non-viral therapy our ribozymal gene-therapy/mRNA-editing has several benefits. Like the non-viral therapy most problems with the immune systems can be evaded by application through liposomes or nanoparticles, like it was done for siRNA <X-REF>Sioud2003</X-REF>. 
</P></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV><DIV class="container"><DIV class="content"><DIV class="row"><DIV class="col-lg-12"><DIV class="panel panel‐default"><DIV class="panel‐heading"><H3 class="basicheader"> Idea and Methodology </H3></DIV><DIV class="panel‐body"><DIV class="row"><DIV class="col-lg-12"><H3 class="subheader"> Idea </H3><P class="basictext">
Over the last decades, numerous new techniques for the editing of DNA inside living cells have been developed. Tools like Cas9-based homologous recombination enable synthetic biologists to locate targets inside the genome and change complete sections of the DNA with greater than ever ease. Many chronical and hereditary diseases are caused by mutations in the DNA genome that are transcribed into mRNA and translated into altered proteins. As they are not able to fulfill their task correctly, the metabolism of the cell, and in the end the wellbeing of the patient is infringed by these changes. This challenge in mind, the new DNA editing tools gave hope to repair specific regions in the genome that are responsible for the malicious phenotypes. Despite vast improvements in the ability to target virtually any gene, the DNA editing gene therapy still has to overcome many challenges and and mediate certain risks. The strategy of changing the cells’ DNA requires the system to be highly reliable. A genetic instability caused by the unwanted insertion of the therapeutic gene can create permanent damage and may lead to an increased cancer risk or even turn out fatal. 
A promising way to circumvent this risks, however displaying the full potential of curing a disease by repair of the damaged sequences, is given by the idea of RNA gene therapy. In this strategy the required correction is done on mRNA level. As this type of therapy can be applied transiently without viral-vectors, the side effects can be considered to be lower <X-REF>Sioud2003</X-REF><X-REF>Alton2015</X-REF>. The straightforward strategy to change mRNAs relies on the use of mRNA editing ribozymes like trans-splicing ribozymes or highly engineered RNA cleaving and ligating systems. 
</P><H3 class="subheader"> Construction of Ribozymes </H3><P class="basictext">
For our goal of treating cystic fibrosis, specifically repairing the deletion of phenylalanine 508, which is the prevalent cause for the disease, we engineered a series of Twinzymes to specifically excise and replace the affected RNA sequence. To this end, we selected cut sites in the target mRNA with a distance of 20bp at most, as distances above that cause the base-pairing between substrate and ribozyme to be too strong to permit efficient dissociation of the cleavage product. That constraint could potentially be relaxed by the introduction of mismatches between ribozyme and substrate. As no sites corresponding to the extended consensus sequence were found adjacent to the ΔF508 deletion site under said constraint, we used structure-based design to engineer ribozymes corresponding to target sites close to the consensus, differing by a substitution of the active site U by A. This mutation on its own results in a loss of cleavage and ligation activity<X-REF>PerezRuiz1999</X-REF>. We modified the ribozyme sequence to restore the intramolecular interactions lost due to the mutation using a previously published interaction map as reference<X-REF>Sumita2013</X-REF>. Ribozymes were thus designed for two target sites conforming to the maximum distance constraint. For short distances to the active site (&lt;20 nucleotides), a reverse-linked design of loop B is used. The two ribozymes are linked by a sequence complementary to the target in its edited state, forming the Twinzyme. (If we want to insert bases, there will be a bulge on the ribozyme strand, where the bases are to be inserted <X-REF>Balke2014</X-REF>. If we want to delete bases, there will be a bulge on the substrate strand, where bases are to be deleted<X-REF>Drude2007</X-REF>. Following these easy design principles, one can target any sequence compatible with the extended consensus. For sequences outside that consensus, interaction maps and crystal structures have to be consulted.)
</P><H3 class="subheader"> Assembly of Ribozymes </H3><P class="basictext">
For the assembly of ribozymes with the repair oligonucleotide, we designed a ribozyme assembly vector. This vector is comprised of the Hammerhead – Twinzyme - HDV cassette and various numbers of Hammerhead - repair oligonucleotide - HDV cassettes all transcribed into a single RNA. The Twinzyme and the repair oligonucleotides are released from the transcribed mRNA by the self-cleaving activity of the Hammerhead and HDV ribozymes.
</P><P class="basictext">
The ribozyme assembly vector was designed by inserting a pCat promoter into the pSB1C3 plasmid along with a new cloning site containing restriction sites for insertion of the ribozyme and the repair oligonucleotide. By transferring this cassette to yeast or mammalian backbones, expression in different chassis-systems is possible. In order to make insertion of multiple repair-oligonucleotide cassettes in a sequential fashion possible, we decided to use recognition sites of isocaudomeric restriction enzymes.
</P><H3 class="subheader"> Ribozyme Screening </H3><P class="basictext">
Sequencing of the assembled ribozyme construct proved extremely difficult, most likely due to extensive secondary structure formation. Neither one of several commercial sequencing services nor any other laboratory could obtain sequences of the plasmids. We therefore decided to screen a large number of candidates for functionality under the assumption that constructs possessing a correct sequence would be among the most efficient ones. We chose yeast as a eucaryotic model due to its rapid growth and easy handling.</P><DIV class="col-lg-6"><DIV class="imagewrapper"><DIV class="imagewrappercaption"><STRONG>
                Figure 4: Application of a fluorescent reporter assay to measure Twinzymes's activity.
            </STRONG><P class="basictext">
               We expext severeal outcomes if the ribozyme interacts with the target: <B>(A)</B> Stop codon is inserted at the ΔF508-mutation site, <B>(B)</B> construct is cleaved, <B>(C)</B> a frameshift occurs. A his-tag is expressed, that can be seen on a Western Blot or <B>(D)</B> no cleavage or insertion is occuring.
            </P></DIV></DIV></DIV><P class="basictext">We designed a fluorescent reporter to measure the Twinzyme's activity. To account for differences in protein expression due to variations in plasmid copy number, a ratiometric approach utilizing a fusion of two fluorescent proteins, mCherry and sfGFP, was chosen (Fig. 4). The two FPs are linked by a fragment from the CFTR protein containing the ΔF508 mutation followed by a frameshift test region. The Twinzyme is designed to insert a stop codon into the CFTR fragment, thus decreasing the GFP/mCherry fluorescence intensity ratio. In case the Twinzyme does not function as expected and a frameshift is introduced, the frameshift test region will translate into a hexahistidine stretch detectable by immunoblotting in either of two possible reading frames.</P><P class="basictext">Both the reporter protein and the ribozymes were expressed from yeast centromeric plasmids under the control of the strong GPD promoter. A positive control expressing only the fluorescent reporter as well as a non-fluorescent negative control used for autofluorescence correction were included in the screening. Ribozyme activity was assayed by high-throughput flow cytometry. </P><DIV class="col-lg-6"><DIV class="imagewrapper"><DIV class="imagewrappercaption"><STRONG>
                Figure 5: Twin-ribozyme click assay
            </STRONG><P class="basictext">
                The ribozyme and an RNA-insert with an Azido-group is inserted into the cleaved mRNA. Afterwards a fluorophore that is alkine-modified is applied with a catalyzator and Copper. If the insert is in the mRNA, it can be seen on the gel, although the fragment was elongated with only 3 bases.
            </P></DIV></DIV></DIV><H3 class="subheader"><I>In vitro</I> Assay </H3><P class="basictext">
As the functionality of the twin ribozyme has only been demonstrated in <I>in vitro</I> setups until now, an important approach to take is to test the ribozyme reaction in a gel based assay.
To visualize the reaction, an azide-modified repair-oligonucleotide should be inserted in the substrate RNA (the region of the CFTR around the d508 mutation) by our ribozyme-candidates. Thereby, we want to prove that the ribozyme is not only able to excise, but also to insert the repair-oligonucleotide. This approach should be seen as orthogonal, yet complementary insofar as by either the <I>in vitro</I> or the <I>in vivo</I> approach alone, there are possible outcomes that are difficult to distinguish. By combining and comparing the results obtained by both strategies, we not only want to be able to explain what the ribozymes are able to perform and if they are suitable as possible RNA editing agents, but furthermore that the results obtained are reproducible in different systems.
</P><P class="basictext"><B>Construction of internal-labeled repair-oligonucleotide-sequences:</B>
We wanted to build a modified repair oligonucleotide, based on our repair-oligonucleotide sequences in the ribozyme vectors. To achieve this, the 27bp-long repair-oligonucleotide-part of the type 2 – ribozymes are amplified in two separate parts – with one missing G-nucleotide in the middle of the sequence. Using overhangs at the PCR primers, a T7 promoter was added to 5´ of each part to facilitate the following in-vitro transcription to RNA fragments. 
After purification of in-vitro transcription, a single azide-modified G-nucleotide was added to the 3’ of part 1 by using a poly-A-poymerase. The two parts were then ligated using splinted ligation with a DNA-splint covering the site of ligation. (Conditions as described in <X-REF>Winz2012</X-REF>).
Due to low efficiency of the performed labeling and ligation, we decided to order the repair oligonucleotides with an internal G-azide modification for using them in the ribozyme reaction.
</P><P class="basictext"><B>Ribozyme reaction:</B>
According to the protocol provided to us by Darko Balke <X-REF>Balke2012</X-REF>, the ribozyme reaction was performed in a 40mM TRIS buffer with 10mM MgCl2. Six different ribozymes, the belonging repair oligonucleotides and the substrate RNA were incubated in equimolar concentration at 37° for eight hours. Afterwards a FAM-alkine fluorphore was added by copper click to the G-azide. By gel analysis, we could quantify how much of the repair oligonucleotide was inserted and compare these results with <I>in vivo</I> work.
</P></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV><DIV class="container"><DIV class="content"><DIV class="row"><DIV class="col-lg-12"><DIV class="panel panel‐default"><DIV class="panel‐heading"><H3 class="basicheader"> Results </H3></DIV><DIV class="panel‐body"><DIV class="row"><DIV class="col-lg-12"><P class="basictext">Results from Yeast assay: 

Since Sanger sequencing of the cloned ribozymes proved impossible due to extensive secondary structure formation, several clones of each construct were used for all subsequent steps under the assumption that clones possessing the correct sequence would be among the most efficiently performing ones. As this led to an exponential increase in samples, we only assayed constructs expressing one copy of the repair oligonucleotide cassette. 
We screened the 12 CFTR ribozymes in both technical and biological replicates by high-throughput flow cytometry including a positive control expressing only the fluorescent reporter and a non-fluorescent negative control to correct for auto fluorescence. Only weak, if any, changes in GFP/mCherry fluorescence intensity could be detected (Fig. 1), indicating low cleavage and insertion efficiency. Given that we only assayed a kinetically suboptimal 1:1:1 ratio of reporter to ribozyme to repair oligonucleotide, and a repair efficiency of 5 - 10 % would suffice to completely alleviate clinical symptoms of cystic fibrosis, we selected the most promising candidates (CFTR 1C (4a), CFTR 2A (2c), CFTR 2C (1d), CFTR 2 DE T (2b), CFTR 2 DE T (3c), CFTR 2 DE C (1a 1 (a)), and CFTR 2 DE C (1c)) for a follow-up screen. While no significant change in GFP/mCherry ratio was observed, fluorescence intensities of both FPs dropped significantly in construct 2 DE C (1a 1(a)) (Figure 2), indicating dramatically reduced expression of the reporter construct. We therefore hypothesized that the ribozyme cleaves the target mRNA with high efficiency, but only a small minority of cleavage events is followed by an insertion reaction, rendering it difficult to observe. The cleaved, mRNA lacking the protective polyA-tail would be rapidly degraded, leading to the observed drop in expression. To determine the frequency of insertion events, we performed RNA-Sequencing of yeasts co-expressing the fluorescent reporter and construct 2DE C (1a 1(a)). While only 50 % of reporter construct mRNA was observed in ribozyme-expressing cells compared to the control (figure), indicating extremely high ribozyme activity, we found no evidence of insertion reactions.</P><DIV class="col-lg-12"><DIV class="imagewrapper"><DIV class="imagewrappercaption"><STRONG>
                Figure 5: Boxplot of the first FACS screening with all ribozyme candidates
            </STRONG><P class="basictext">
                Yeast cells co-expressing the fluorescent reporter and a ribozyme were grown in 96-well plates to log phase and analysed by flow cytometry. GFP and mCherry intensities were corrected for auto fluorescence using non-fluorescent cells. For each sample, the distribution of GFP/mCherry ratios is visualized.  The black line denotes the median, the box ranges from the 1st to the 3rd quartile, dashed whiskers indicate 1.5x the interquartile range. The mean is depicted by a red line, solid whiskers represent the mean ± standard deviation, yellow lines the mean ± standard error of the mean.
            </P></DIV></DIV></DIV><DIV class="col-lg-12"><DIV class="imagewrapper"><DIV class="imagewrappercaption"><STRONG>
                Figure 2: Intensity ratios of GFP and mCherry against the positive control and each other
            </STRONG><P class="basictext">
                The yeast fluorescence of the yeast GFP (first diagraph) and RFP (second diagraph) was plotted against the positive control (first and second diagraph) and against each other (third diagraph). The calculation of the ratio was made the following: ratio = (exp(Iribo) - exp(Ineg,ctr)) / (exp(Ipos,ctr) - exp(Ineg,ctr)). An especially high drop in fluorescence is seen for ribozyme CFTR 2 DE C 1, but not for CFTR 2 DE C 2
            </P></DIV></DIV></DIV><DIV class="col-lg-6"><DIV class="imagewrapper"><DIV class="imagewrappercaption"><STRONG>
                Figure 7: Relation of mRNA in the control and the yeast with CFTR 2 DE ribozyme
            </STRONG><P class="basictext">
                The control is shown left and the CFTR 2 DE ribozyme is shown right. A cleavge of about 55% of the CFTR-test-construct mRNA is visible
            </P></DIV></DIV></DIV><DIV class="col-lg-12"><P class="basictext">Next Generation Sequencing: More exactly cells reporter as reference construct and cells with reporter and the ribozyme CFTR 2 DE C were screened with 75 Million reads of 70 bases each and data alligned to yeast BY4741s genome and the experssed reporter construct. 
The baseline important for comparing the data was generated by analysing the housekeeping genes oft the yeast strain with more than 1000 allignments. 
Th data shows that the candidate has a very high in-vivo cleavage activity, in two biological replicates at least 50% of mRNA has been cleaved without a ligation. In our target region there was in 803 reads no fragment found with the ATG insertion.  
</P><P class="basictext">
Click results: 
The in vitro assay was checked on a gel by scanning the fluorophore and the whole RNA by SYBR gold stain. As seen in the gel, the click reaction was not successful - so we could not see if there is a replacement by the ribozyme. With new click catalysts we have already shown a functional click of labeled RNA and habe to repeat the in vitro reaction.
</P><P class="basictext">In conclusion: 
We were able to show restored cleavage activity <I>in vivo</I> for non-consensus target sites involving a GA motif in the active site, instead of the usual GU. We have thus extended the hairpin ribozyme to sequences of the type YNGHHN, which were previously known to have only imparied funcionality. Furthermore, we were able to show the expression and activity of twin hairpin ribozymes in vivo, using a yeast promoter, as well as a hammerhead-HDV expression cassete. To date, the twin hairpin ribozyme was only known to cleave in vitro, which makes this not a trivial exercise, but a result in it's own right</P></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV><DIV class="container"><DIV class="content"><DIV class="row"><DIV class="col-lg-12"><DIV class="panel panel‐default"><DIV class="panel‐heading"><H3 class="basicheader"> Discussion and Outlook </H3></DIV><DIV class="panel‐body"><DIV class="row"><DIV class="col-lg-12"><P class="basictext">
Here we aimed to develop a novel, non-permanent gene therapy for cystic fibrosis by editing the CFTR mRNA using a twin hairpin ribozyme. We rationally engineered a new ribozyme targeting a fragment of the CFTR mRNA and, for the first time, showed cleavage of the mRNA to proceed at an extremely high rate in vivo. Even though no insertion events could be detected, our engineered ribozyme represents a first step towards that goal. Assuming that the rationally designed Twinzyme is able to catalyze fragment exchanges in principle, its failure to do so in vivo may result from a non-optimal ribozyme to repair oligonucleotide ratio. In theory, decreasing this ratio would result in a higher probability of a repair oligo binding to a ribozyme molecule currently engaging its substrate, thus enabling a trans-splicing reaction. This hypothesis could be easily tested in mammalian cells by transfection with different amounts of plasmids expressing either the ribozyme or the repair oligonucleotide. 
One must also consider that the Twin-ribozyme construct, synthesized in vitro and subsequently cloned into the expression vector, could not be sequenced. Since two active sites are required to perform the insertion reaction, and in vitro DNA synthesis is known to be highly error-prone, only one of the two active sites may be functional in the 2DEC construct, enabling cleavage of the target mRNA, but not insertion of the fragment. Given that a functional version of the Twin-ribozyme is now known, high-throughput screening of several hundred clones may be performed to identify a functional candidate. Alternatively, a yeast library could be generated from the DNA pool obtained from in vitro synthesis, and functional candidates could be isolated by fluorescence-aided cell sorting (FACS).
</P></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV><DIV class="container"><DIV class="content"><DIV class="row"><DIV class="col-lg-12"><DIV class="panel panel‐default"><DIV class="panel‐heading"><H3 class="basicheader"> References </H3></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></BODY></HTML>