<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_Heidelberg_Modeling_rtsms skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:Heidelberg/Modeling/rtsms</SPAN></H1><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><DIV class="container" id="menucontainer"><DIV id="navcontainer-right-outer"><DIV id="navcontainer-right-inner"><UL class="nav navbar-nav navbar-right"><LI><A href="https://2015.igem.org/Team:Heidelberg/Achievements" class="menubox"><P class="menutext">Achievements</P></A></LI><LI class="dropdown"><A href="https://2015.igem.org/Team:Heidelberg/Results" class="dropdown-toggle menubox" data-toggle="dropdown" role="button" aria-haspopup="true" aria-expanded="false"><P class="menutext">Results
                        </P></A></LI><LI class="dropdownbox"><A href="https://2015.igem.org/Team:Heidelberg/Results#Medalfulfillments"><P class="menutext dropdown-text">Medal fulfillments</P></A></LI><LI class="dropdownbox"><A href="https://2015.igem.org/Team:Heidelberg/Results#AB"><P class="menutext dropdown-text">AptaBodies</P></A></LI><LI class="dropdownbox"><A href="https://2015.igem.org/Team:Heidelberg/Results#RTSMS"><P class="menutext dropdown-text">Real time S.M.S.</P></A></LI><LI class="dropdownbox"><A href="https://2015.igem.org/Team:Heidelberg/Results#LandV"><P class="menutext dropdown-text">Cystic fibrosis</P></A></LI><LI class="dropdownbox"><A href="https://2015.igem.org/Team:Heidelberg/Results#CF"><P class="menutext dropdown-text">Labeling and vizualization</P></A></LI><LI class="dropdownbox"><A href="https://2015.igem.org/Team:Heidelberg/Results#Modelling"><P class="menutext dropdown-text">Modeling and Software validation</P></A></LI><LI class="dropdownbox"><A href="https://2015.igem.org/Team:Heidelberg/Results/Standardization"><P class="menutext dropdown-text">Standardization</P></A></LI></UL><LI class="dropdown"><A href="https://2015.igem.org/Team:Heidelberg/Overview" class="dropdown-toggle menubox" data-toggle="dropdown" role="button" aria-haspopup="true" aria-expanded="false"><P class="menutext">Project
                        </P></A></LI><LI class="dropdownbox"><A href="https://2015.igem.org/Team:Heidelberg/Overview"><P class="menutext dropdown-text">Overview</P></A></LI><LI class="dropdownbox"><A href="https://2015.igem.org/Team:Heidelberg/Project/AB"><P class="menutext dropdown-text">AptaBodies</P></A></LI><LI class="dropdownbox"><A href="https://2015.igem.org/Team:Heidelberg/project/rtsms"><P class="menutext dropdown-text">Real time S.M.S.</P></A></LI><LI class="dropdownbox"><A href="https://2015.igem.org/Team:Heidelberg/project/cf"><P class="menutext dropdown-text">Cystic fibrosis</P></A></LI><LI class="dropdownbox"><A href="https://2015.igem.org/Team:Heidelberg/project/rd"><P class="menutext dropdown-text">Labelling and Visualization</P></A></LI><LI class="dropdownbox"><A href="https://2015.igem.org/Team:Heidelberg/project/hlpd"><P class="menutext dropdown-text">HRP based detection</P></A></LI><LI class="dropdown"><A href="https://2015.igem.org/Team:Heidelberg/Software" class="dropdown-toggle menubox" data-toggle="dropdown" role="button" aria-haspopup="true" aria-expanded="false"><P class="menutext">Software
                        </P></A></LI><LI class="dropdownbox"><A href="https://2015.igem.org/Team:Heidelberg/software/maws"><P class="menutext dropdown-text">MAWS</P></A></LI><LI class="dropdownbox"><A href="https://2015.igem.org/Team:Heidelberg/software/jaws"><P class="menutext dropdown-text">JAWS</P></A></LI><LI class="dropdownbox"><A href="
https://2015.igem.org/Team:Heidelberg/WikiTools"><P class="menutext dropdown-text">Wiki Tools</P></A></LI><LI class="dropdown"><A href="https://2015.igem.org/Team:Heidelberg/notebook/mm" class="dropdown-toggle menubox" data-toggle="dropdown" role="button" aria-haspopup="true" aria-expanded="false"><P class="menutext">Notebook
                        </P></A></LI><LI class="dropdownbox"><A href="https://2015.igem.org/Team:Heidelberg/notebook/ab"><P class="menutext dropdown-text">AptaBodies</P></A></LI><LI class="dropdownbox"><A href="https://2015.igem.org/Team:Heidelberg/notebook/rtsms"><P class="menutext dropdown-text">Real time S.M.S.</P></A></LI><LI class="dropdownbox"><A href="https://2015.igem.org/Team:Heidelberg/notebook/cf"><P class="menutext dropdown-text">Cystic fibrosis</P></A></LI><LI class="dropdownbox"><A href="https://2015.igem.org/Team:Heidelberg/notebook/rd"><P class="menutext dropdown-text">Labeling and vizualization</P></A></LI><LI class="dropdownbox"><A href="https://2015.igem.org/Team:Heidelberg/notebook/pc"><P class="menutext dropdown-text">Pulse chase</P></A></LI><LI class="dropdownbox"><A href="https://2015.igem.org/Team:Heidelberg/notebook/mm"><P class="menutext dropdown-text">Materials &amp; Methods</P></A></LI><LI class="dropdownbox"><A href="https://2015.igem.org/Team:Heidelberg/Safety"><P class="menutext dropdown-text">Safety</P></A></LI><LI><A href="https://2015.igem.org/Team:Heidelberg/Parts" class="menubox"><P class="menutext">Parts</P></A></LI><LI class="dropdown"><A href="https://2015.igem.org/Team:Heidelberg/Team" class="dropdown-toggle menubox" data-toggle="dropdown" role="button" aria-haspopup="true" aria-expanded="false"><P class="menutext">Team
                        </P></A></LI><LI class="dropdownbox"><A href="https://2015.igem.org/Team:Heidelberg/Team"><P class="menutext dropdown-text">Members</P></A></LI><LI class="dropdownbox"><A href="https://2015.igem.org/Team:Heidelberg/Supervisors"><P class="menutext dropdown-text">Supervisors</P></A></LI><LI class="dropdownbox"><A href="https://2015.igem.org/Team:Heidelberg/Collaborations"><P class="menutext dropdown-text">Collaboration</P></A></LI><LI class="dropdownbox"><A href="https://2015.igem.org/Team:Heidelberg/Attributions"><P class="menutext dropdown-text">Attributions</P></A></LI><LI class="dropdownbox"><A href="https://2015.igem.org/Team:Heidelberg/Sponsors"><P class="menutext dropdown-text">Sponsors</P></A></LI><LI class="dropdown"><A href="https://2015.igem.org/Team:Heidelberg/Modeling" class="dropdown-toggle menubox" data-toggle="dropdown" role="button" aria-haspopup="true" aria-expanded="false"><P class="menutext">Modeling
                        </P></A></LI><LI class="dropdownbox"><A href="https://2015.igem.org/Team:Heidelberg/Modeling/aptakinetics"><P class="menutext dropdown-text">Aptamer Sensors</P></A></LI><LI class="dropdownbox"><A href="https://2015.igem.org/Team:Heidelberg/Modeling/rtsms"><P class="menutext dropdown-text">In vitro transcription</P></A></LI><LI><A href="https://2015.igem.org/Team:Heidelberg/Practices" class="menubox"><P class="menutext">Outreach</P></A></LI></DIV></DIV></DIV><NAV id="myNavmenu" class="navmenu navmenu-default navmenu-fixed-left offcanvas" role="navigation"><A class="navmenu-brand" href="https://2015.igem.org/Team:Heidelberg">iGEM Team Heidelberg 2015</A><UL class="nav navmenu-nav"><LI><A href="https://2015.igem.org/Team:Heidelberg/Achievements">Achievements</A></LI><LI><A href="https://2015.igem.org/Team:Heidelberg/Results">Results</A></LI><LI><A href="https://2015.igem.org/Team:Heidelberg/Overview">Project</A></LI><LI><A href="https://2015.igem.org/Team:Heidelberg/Software">Software</A></LI><LI><A href="https://2015.igem.org/Team:Heidelberg/notebook/mm">Notebook</A></LI><LI><A href="https://2015.igem.org/Team:Heidelberg/Parts">Parts</A></LI><LI><A href="https://2015.igem.org/Team:Heidelberg/Team">Team</A></LI><LI><A href="https://2015.igem.org/Team:Heidelberg/Modeling">Modeling</A></LI><LI><A href="https://2015.igem.org/Team:Heidelberg/Practices">Outreach</A></LI></UL></NAV><DIV class="container"><DIV class="content"><DIV class="row"><DIV class="col-lg-12"><DIV class="panel panel‐default"><DIV class="panel‐heading"><H3 class="basicheader"> Studying determinants of polymerase efficiency based on an aptamer sensor </H3></DIV><DIV class="panel‐body"><DIV class="row"><DIV class="col-lg-12"><P class="basictext">
Our subproject on small molecule sensing facilitates quantitatively studying in vitro transcription (IVT) by ATP-spinach and malachite green RNA-aptamers. Here, we apply mathematical modeling to understand mechanistic details of this process and demonstrate that our approach can be used as a tool for basic research. 
After adding an RNA polymerase to DNA templates, the polymerase binds to the template and starts consuming ATP by incorporating it into transcripts containing the malachite green aptamer. While the concentration of ATP could be monitored by fluorescence of the Spinach2-ATP-Aptamer, the transcript yield was monitored by malachite green fluorescence. This enabled us to follow IVT quantitatively and time-resolved. In particular, we could study the inaccuracy of polymerases reflected by an excess of consumed ATP molecules over the number of ATP molecules in synthesized malachite green aptamers.  
To this end, we implemented a mathematical model that describes the formation of &quot;active templates&quot; $T^*$ from unbound DNA-templates $T$ and polymerases $P$, and the consumption of ATP $A$ for the synthesis of malachite green aptamers $M$ (Figure 1A). Because malachite green aptamers contain $n_{A,M}=10$ adenine nucleotides, the rate, at which malachite green is produced, is at least by this factor lower than the rate, at which ATP is consumed. The production of premature abortion products that result from the detachment of the polymerase from the template before completing the transcript, however, leads to an even larger number $n_A&gt;n_{A,M}$. By calibrating the model with experimental data, we estimated this number to characterize this polymerase inaccuracy. For this purpose, we used datasets that were recorded with the T7 RNA polymerase. First, as depicted in Figure 1A, we tried to explain this inaccuracy by a constant number $n_A$ that was independent from DNA-template, ATP or polymerase concentrations. Then, we extended the model step-wise until the experimental data could be explained by the model. The step-wise extensions are listed in Table 1 while Table 2 contains the model equations for each variant.
</P><DIV class="imagewrappercaption"><STRONG>Figure 1.</STRONG> IVT model reactions and fits to experimental data. (A) Model reactions describing reversible assembly of templates $T$ and polymerase $P$ to active templates $T^*$ that incorporate ATP $A$ into malachite green RNA-aptamers $M$ but also into abortion products, leading to a higher number $n_A$ of consumed than ATP molecules $n_{A,M}$ incorporated in malachite green aptamers. (B) Model fits to data at two different polymerase concentrations. </DIV> <P class="basictext">
Counter-intuitively, malachite green fluorescence showed a linear increase while the ATP-spinach fluorescence intensity was exponentially decreasing. Furthermore, it was surprising that doubling the amount of polymerase increased the production of malachite green by even more than two-fold. Because of these two unexpected findings, our basic model with constant values for $n_A$ could not explain the data. However, both phenomena could be explained by an optimal model variant (Figure 1B, 1A, Table 1), in which the polymerase inaccuracy increased with increasing ratios between ATP and active templates. Figure 2A visualizes the improvement in fit quality from the basic model to the optimal model variant (variant 4) in values of the Akaike information criterion (AIC) that accounts for the distance between the model and the experimental data and additionally penalizes for the number of model parameters to favor parsimonious model topologies.  
Next, we tried if the optimal model, variant 4, could be simplified without losing fit quality. Leaving out degradation reactions for the polymerase $P$ strongly decreased fit quality (Figure 2B). Furthermore, assuming a fast binding of the polymerase to its template, which can be reflected in the model by a steady state of active template formation, resulted in a large AIC value increase. Leaving out ATP degradation, however, resulted only in a slight decrease in fit quality indicated by a small increase in the corresponding AIC value. We applied the rank-based Kruskal-Wallis test and found that, nevertheless, the small AIC value increase was significant ($p = 1.57\cdot10^{-4}$). This indicated that the optimal model could not be further reduced without losing fit quality. Essentially, in the optimal model variant, the rate of malachite green synthesis was dependent on a consumed number of ATP molecules $n_A=n_{A,0} A /T^{*l}$ for each malachite green aptamer molecule. In Figure 2C, the number $n_A$ is shown for different ratios between ATP and active template concentrations using the best fit parameters of the optimal model variant. The model thus predicts a high sensitivity of $n_A$ for changes of the $A /T^{*}$ ratio at values below $A /T^{*}\approx10$ and a low sensitivity of $n_A$ at higher ratios in the range above $A /T^{*}\approx30$ to $50$. </P><DIV class="imagewrappercaption"><STRONG>Figure 2.</STRONG> IVT inaccuracy depends on the ATP to active template ratio. (A) A basic model with constant numbers of $n_A$ and synthesis parameters $k_{syn,M}$, was extended to variants with $n_A$ and $k_{syn,M}$ depending on the polymerase concentration (variant 2), $A$- and $T^*$-dependent $n_A$ with exponents $k$ and $l$ (variant 3) or only an exponent for $T^*$ (variant 4). Fitting improvement is indicated by decreasing Akaike information criterion (AIC) values. (B) Reducing the optimal variant 4 by assuming a steady state for $T^*$, no degradation of $P$ or no degradation of $A$ strongly worsened model fits. (C) Model variant 4 can explain increasing inefficiency (higher $n_A$) with decreasing $A/T^*$ ratios.</DIV> <P class="basictext">
Taken together, our setup was suitable for studying the phenomenon of polymerase inaccuracy based on a mathematical model. We have learned that the inaccuracy of an RNA polymerase increases with an increasing ratio between ATP and active templates in a non-linear manner. Furthermore, we learned that the kinetics of polymerase binding to the DNA-template is relevant for the transcription dynamics. In the future, our approach might facilitate quantitative studies of the interaction between polymerases and promoters as well as the impact of DNA-modifications on the transcription dynamics.</P><DIV class="imagewrappercaption"><STRONG>Table 1.</STRONG> Stepwise changes from the basic model variant 1 to the optimal variant 4 and from variant 4 to variants 4a to 4c</DIV><TABLE border="1" cellpadding="0" cellspacing="0"><TBODY><TR><TD style="width:92px;height:60px;"><P><STRONG>Model variant</STRONG></P></TD><TD style="width:350px;height:60px;"><P><STRONG>Subsequent modifications relative to the previous variant</STRONG></P></TD><TD style="width:200px;height:60px;"><P><STRONG>Changes in fitting quality</STRONG></P></TD></TR><TR><TD style="width:92px;"><P>
							1</P></TD><TD style="width:350px;"><P>
							$k_{syn}$ and $n_A$ independent from polymerase concentrations</P></TD></TR><TR><TD style="width:92px;"><P>
							2</P></TD><TD style="width:350px;"><P>
							Individual $k_{syn}$ and $n_A$ values for different polymerase concentrations</P></TD><TD style="width:200px;"><P>
							improvement</P></TD></TR><TR><TD style="width:92px;"><P>
							3</P></TD><TD style="width:350px;"><P>
							$n_A$ depends on function of $T^*$ and $A$</P><P>
							$n_A=n_{A,0} A^{k} /T^{*l}$</P></TD><TD style="width:200px;"><P>
							improvement, $k\approx0$</P><P> </P></TD></TR><TR><TD style="width:92px;"><P>
							4, best model</P></TD><TD style="width:350px;"><P>
							Setting $k=0$</P></TD><TD style="width:200px;"><P>
							improvement</P></TD></TR><TR><TD style="width:92px;"><P>
							4a</P></TD><TD style="width:350px;"><P>
							No degradation of $P$ in variant 4</P></TD><TD style="width:200px;"><P>
							decrease</P></TD></TR><TR><TD style="width:92px;"><P>
							4b</P></TD><TD style="width:350px;"><P>
							No degradation of $A$ in variant 4</P></TD><TD style="width:200px;"><P>
							decrease</P></TD></TR><TR><TD style="width:92px;"><P>
							4c</P></TD><TD style="width:350px;"><P>
							Binding of $P$ to $T$ in steady state in variant 4</P></TD><TD style="width:200px;"><P>
							decrease</P></TD></TR></TBODY></TABLE><P> </P><P> </P><DIV class="imagewrappercaption"><STRONG>Table 2.</STRONG> Model equations for the basic model and variants 1 to 4c</DIV><TABLE border="1" cellpadding="0" cellspacing="0"><TBODY><TR><TD style="width:55px;"><P><STRONG>Model species</STRONG></P></TD><TD style="width:151px;"><P><STRONG>Variant</STRONG></P></TD><TD style="width:436px;"><P><STRONG>Equation</STRONG></P></TD></TR><TR><TD rowspan="3" style="width:55px;"><P>
							$P$</P></TD><TD style="width:151px;"><P>
							Variants 1 to 4, 4c</P></TD><TD style="width:436px;"><P>
							$\frac{d[P]}{dt}=-k_{on}[T][P]+k_{off}[T^*]-k_{deg,P}[P]$</P></TD></TR><TR><TD style="width:151px;"><P>
							Variant 4a</P></TD><TD style="width:436px;"><P>
							$[P](t)=[P](t_{0})\exp\left(-k_{deg,P}t\right)$</P></TD></TR><TR><TD style="width:151px;"><P>
							Variant 4b</P></TD><TD style="width:436px;"><P>
							$\frac{d[P]}{dt}=-k_{on}[T][P]+k_{off}[T^*]$</P></TD></TR><TR><TD rowspan="2" style="width:55px;"><P>
							$T$</P></TD><TD style="width:151px;"><P>
							Variants 1 to 4, 4b, 4c</P></TD><TD style="width:436px;"><P>
							$\frac{d[T]}{dt}=-k_{on}[T][P]+k_{off}[T^*]$</P></TD></TR><TR><TD style="width:151px;"><P>
							Variant 4a</P></TD><TD style="width:436px;"><P>
							$[T]=[T_{tot}]-[T^*]$</P></TD></TR><TR><TD rowspan="2" style="width:55px;"><P>
							$T^*$</P></TD><TD style="width:151px;"><P>
							Variants 1 to 4, 4b, 4c</P></TD><TD style="width:436px;"><P>
							$\frac{d[T^*]}{dt}=k_{on}[T][P]-k_{off}[T^*]$</P></TD></TR><TR><TD style="width:151px;"><P>
							Variant 4a</P></TD><TD style="width:436px;"><P>
							$[T^*]=\frac{[T_{tot}][P]}{K_{d,P}}$</P></TD></TR><TR><TD rowspan="3" style="width:55px;"><P>
							$A$</P></TD><TD style="width:151px;"><P>
							Variants 2 to 4, 4a, 4b</P></TD><TD style="width:436px;"><P>
							$\frac{d[A]}{dt}=-k_{syn}[A][T^*]-k_{deg,A}[A]$</P></TD></TR><TR><TD style="width:151px;"><P>
							Variant 1</P></TD><TD style="width:436px;"><P>
							$\frac{d[A]}{dt}=-k_{syn}\frac{[A][T^*]}{K_{m,T}+[T^*]}-k_{deg,A}[A]$</P><P> </P></TD></TR><TR><TD style="width:151px;"><P>
							Variant 4c</P></TD><TD style="width:436px;"><P>
							$\frac{d[A]}{dt}=-k_{syn}[A][^*]$</P></TD></TR><TR><TD rowspan="4" style="width:55px;"><P>
							$M$</P></TD><TD style="width:151px;"><P>
							Variant 2</P></TD><TD style="width:436px;"><P>
							$\frac{d[M]}{dt}=\frac{k_{syn}}{n_{A}}[A][T^*]$</P></TD></TR><TR><TD style="width:151px;"><P>
							Variants 1</P></TD><TD style="width:436px;"><P>
							$\frac{d[M]}{dt}=\frac{k_{syn}}{n_{A}}\frac{[A][T^*]}{K_{m,T}+[T^*]}$</P></TD></TR><TR><TD style="width:151px;"><P>		Variant 3</P></TD><TD style="width:436px;"><P>
							$\frac{d[M]}{dt}=\frac{k_{syn}}{n_{A,0}\frac{[A]^{k}}{[T^*]^{l}}}[A][T^*]=\frac{k_{syn}}{n_{A,0}}[A]^{1-k}[T^*]^{1+j}$</P></TD></TR><TR><TD style="width:151px;"><P>
							Variants 4, 4a, 4b, 4c</P></TD><TD style="width:436px;"><P>
							$\frac{d[M]}{dt}=\frac{k_{syn}}{n_{A,0}\frac{[A]}{[T*]^{l}}}[A][T^*]=\frac{k_{syn}}{n_{A,0}}[T^*]^{1+j}$</P></TD></TR></TBODY></TABLE><P> </P><P> </P></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></BODY></HTML>