Team:Heidelberg/notebook/cf
Achievements
Results
Medal fulfillments
AptaBodies
Real time S.M.S.
Cystic fibrosis
Labeling and vizualization
Modeling and Software validation
Standardization
Project
Overview
AptaBodies
Real time S.M.S.
Cystic fibrosis
Labelling and Visualization
HRP based detection
Software
MAWS
JAWS
Wiki Tools
Notebook
AptaBodies
Real time S.M.S.
Cystic fibrosis
Labeling and vizualization
Pulse chase
Materials & Methods
Safety
Parts
Team
Members
Supervisors
Collaboration
Attributions
Sponsors
Modeling
Aptamer Sensors
In vitro transcription
Outreach
iGEM Team Heidelberg 2015
Achievements
Results
Project
Software
Notebook
Parts
Team
Modeling
Outreach
week number 42
▼2015-10-18  In vitro transcription of P1, P2
 
For every reaction:
Stock solution
Final concentration
[µl]
ATP 100mM
4 mM
8
CTP 100mM
4 mM
8
UTP 100mM
4 mM
8
GTP 100mM
4 mM
8
DTT 1M
1 mM
2
DMSO
5%
10
10x Transcription buffer
1x
20
DNA
1 µg
20
ddH20
 
106
T7 RNA polymerase
 
3
 
 
Reaction was incubated for 3 h at 37 °C
After 1.5 h another 2 µL T7 RNA Polymerase were added
Addition of 2 µL DNase I and further incubation at 37 °C for 20 min
The constructs containing hammerhead- or HDV-ribozymes was heated up to 95° for 5 min, so that there is a coplete cleavage.
 
 
RNA purification by precipitation:
Samples was mixed with 200 µL of 2 x loading dye and purified over a 10 % PAGE
Bands were visualized by UV shadowing and suitable bands were excised
RNA was eluted out of the gel using 0.3 M NaAc pH 5.5 in three elution steps
Gel parts were filtered of and 2.5 volumes of -20 °C EtOH were added, sample was stored at -20 °C oN to let the RNA precipitate
Spin sample at 16,000 g for 30 min, discard the supernatant
Washed the pellet twice with 70 % EtOH and dissolved RNA in 20 µL of MQ water
 
week number 37
▼2015-09-07  Cloning of ribozyme 12-1 (Ribozyme without Insert) from pSB1C3 into p413-GPD
Procedures:
 
Digestion:
 
Description:
 
Materials and Chemicals:
4 µl of pSB1C3 with pcat-MCS and ribozyme 12, about 2 µg of DNA in total
2 µl Cutsmart Buffer
1 µl BamI-HF
1 µl SalI-HF
12 µl ddH2O
 
The digest was incubated for 1 hour at 37°C while mixing slowly
 
The lower band with cut out insert (ribozyme 12) was purificated by gel extraction
 
Ligation:
 
Description:
 
Materials and Chemicals:
1 µl of p413-Vector DNA
6 µl of ribozyme 12 insert (purificated by gel extraction)
1 µl of T4-Ligase
2 µl of T4-Ligase Buffer
10 µl ddH2O
 
 
The ligation was incubated for 30 minutes at room temperature (about 25°C)
 
KCM Transformation:
 
Description:
 
Steps:
 
Take 50µl chemical competent E. coli from -80 freezer and thaw on ice
 
Add (as master mix):
                2,5µl DNA
                10µl KCM 5x
                37,5µl H2O
 
Incubate on ice for 30 minutes
 
Heat shock at 42°C for 1 minute
 
Incubate on ice for 2 minutes
 
Add 900 µl of LB or 2x YT Medium
 
Incubate on 37°C for 60min
 
Centrifuge 5min at 1000g
 
Take 900µl of supernatant and throw away
 
Resuspend pellet in remaining media
 
Plate out on agar with antibiotics (1:1 / 1:10)
 
Used DNA: Ligation of p413 and ribozyme 12-1
10 µl instead of 2,5 µl were used
 
▼2015-09-09  PCR of Type-1 Ribozymes for in vitro assay
 
Ribozyme CFTR1 A
[µl]
Ribozyme CFTR1 C
[µl]
Ribozyme CFTR1 T
[µl]
Primer Fwd
DH_82
5
DH_82
5
DH_82
5
Primer Rev
DH_33
5
DH_83
5
DH_83
5
Template
Ribozyme CFTR1 DE A
0,5
Ribozyme CFTR1 DE C
0,5
Ribozyme CFTR1 DE T
0,5
ddH2O
 
14,5
 
14,5
 
14,5
Q5 Polymerase
 
25
 
25
 
25
 
Conditions:
Step
Temperature [°C]
Time
Cycles
Initial denaturation
98
1:00
1
Denturation
98
0:10
35
Annealing
66
0:15
Extension
72
0:15
Final Extension
72
1:00
1
Hold
4
 
Cycles
 
After PCR DNA was purified by Qiagen PCR Purification Kit ans stored at -20°C.
 
▼2015-09-10  In vitro transcription of CFTR1-type ribozymes
 
For every reaction:
Stock solution
Final concentration
[µl]
ATP 100mM
4 mM
8
CTP 100mM
4 mM
8
UTP 100mM
4 mM
8
GTP 100mM
4 mM
8
DTT 1M
1 mM
2
DMSO
5%
10
10x Transcription buffer
1x
20
DNA
1 µg
20
ddH20
 
106
T7 RNA polymerase
 
3
 
 
Reaction was incubated for 3 h at 37 °C
After 1.5 h another 2 µL T7 RNA Polymerase were added
Addition of 2 µL DNase I and further incubation at 37 °C for 20 min
Reaction was heated up to 95° for 5 min, so that there is a coplete cleavage.
 
RNA purification by precipitation:
Samples was mixed with 200 µL of 2 x loading dye and purified over a 10 % PAGE
Bands were visualized by UV shadowing and suitable bands were excised
RNA was eluted out of the gel using 0.3 M NaAc pH 5.5 in three elution steps
Gel parts were filtered of and 2.5 volumes of -20 °C EtOH were added, sample was stored at -20 °C oN to let the RNA precipitate
Spin sample at 16,000 g for 30 min, discard the supernatant
Washed the pellet twice with 70 % EtOH and dissolved RNA in 20 µL of MQ water
 
 
 
 
 
week number 35
▼2015-08-25  Plasmid prep of colonies containing c(p413-GPD; sfGFP_delY)
Goal: Verifying the sequences of the picked cultures
Plasmids of the cultures grown overnight were purified according to the manufacturers protocol.
Sequences were confirmed as correct by sequencing for colonies 3-2, 4-1,4-2 and 5-1.
Culture 4-1 was used for further experiments.
week number 34
▼2015-08-17  PCR of Part 1 and 2 of the CFTR2 Ribozyme Insert for in vitro transcription
 
 
P1
[µl]
P2
[µl]
Primer Fwd
DH_54
5
DH_56
5
Primer Rev
DH_55
5
DH_57
5
Template
Insert CFTR2
0,5
Insert CFTR2
0,5
ddH2O
 
14,5
 
14,5
Q5 Polymerase
 
25
 
25
 
 
 
Conditions:
Step
Temperature [°C]
Time
Cycles
Initial denaturation
98
1:00
1
Denturation
98
0:10
35
Annealing
66
0:15
Extension
72
0:15
Final Extension
72
1:00
1
Hold
4
 
 
 
After PCR DNA was purified by ethanol precipitation and stored at -20°C
 
 
▼2015-08-19  Labeling of P1 with G-Azide
Reaction was performed with 2-fold concentration of labeling nucleotide and a longer incubation time in comparison to first labeling experiment.
 
 
Reaction [µl]
Control without PAP[µl]
Control without template [µl]
PAP (Poly-A-Polymerase, yeast)
1
/
1
RNAse inhibitor
1
1
1
PAP buffer 5x
4
4
4
Nucleotide G-Azide, 100µM
4
4
4
P1-RNA
5 (= 50µM)
5 (= 50µM)
/
ddH2O
5
6
10
 
Reaction was incubated for 2 h at 37 °C
Heat inactivation at 65°C for 10min
Purification by ethanol precipitation
 
▼2015-08-20  Splinted Ligation of modified P1 and P2
 
 
Reaction [µl]
Control without Ligase [µl]
Control without Splint [µl]
P1 RNA, labeled
6 (=1µM)
6 (=1µM)
6 (=1µM)
P2 RNA
0,5 (=0,5µM)
0,5 (=0,5µM)
0,5 (=0,5µM)
Splint
0,7 (=2,25µM)
0,7 (=2,25µM)
/
10x Ligation buffer
3
3
3
T4 DNA ligase
1
/
1
ddH2O
13,8
15,8
15,5
 
 
Reaction was heaten up to 95°C for 30s without T4 DNA ligase and then cooled down to room temperature
After 15min  T4 DNA ligase was added.
Reaction was incubated for 1 h at 37 °C
Heat inactivation at 80°C for 10min
▼2015-08-20  Copper Click – Test of Labeled Part 1 and Splinted ligation
 
To check if the azide-modified NTPs are incorporated the reactive azide is clicked to an alkyne activated FAM-alkyne fluorophore under copper catalyzed 1,3-dipolar azide-alkyne cycloaddition (CuAAC). Samples are after the reaction separated on a 20 % denaturing PAGE and visualized first in the fluorescent channel for FAM and afterwards stained with SYBR
Gold to visualize the RNA. Controls were performed as well.
 
 
Reaction conditions:
 
cStock
cFinal
V[µL]
Phosphate buffer pH 7.0
100 mM
50 mM
12,5
FAM alkyne
10 µM
400 nM
1
Azide modified RNA
1 µM
200 nM
5
Cu(II)SO4
2 mM
100 µM
1.25
THPTA
5 mM
500 µM
2.5
Sodium ascorbate
10 mM
1 mM
2.5
H2O
 
Ad 25
0.25
Final
 
 
25
Conditions as Winz, 2012.
 
All compounds were mixed in the given order
Cu(II)SO4 and THPTA and H2O were mixed before adding to the mixture to let THPTA chelat the Cu
Lastly sodium ascorbate was added to reduce the Cu(II) to Cu(I)
Reaction was incubated for 1 h at 37 °C and afterwards put at -20 °C
Checking success on a 20 % denaturing PAGE
Samples were mixed with an appropriate amount of 2 x loading dye and separated on a 20 % denaturing PAGE
Gel was scanned with a Biorad ChemDoc, GE in fluorescent mode using the pre-set FAM parameters
Afterwards gel was stained using SYBR gold and scanned with the Typoon in SYBR Gold mode
Result and Outlook:
Labeling of RNA with azide and splinted ligation was also at the second tryonly partially successful. The repair-oligonucleotide is ordered as RNA-synthesis.
 
▼2015-08-22  Mutation PCR of pMaM17
Goal: Deleting a Y in sfGFP from pMaM17 rendering it non-functional. 
PCR with primers xxhb003xx and xxhb004xx was performed using a standard protocol with 68°C as annealing temperature. 
PCR was performed using Phusion Flash MasterMix.
300 µl were split amongst 4 tubes.
 
PCR was followed by a DpnI digest.
Samples were purified by gel extraction according to manufacturs protocol.
 
NanoDrop showed a concentration of 743 ng/µl after purification, with 29 µl sample left to use.
▼2015-08-22  Digestion of pMaM(sfGFP_delY)
Goal: Digest pMaM17(sfGFP_delY) with BamI and SalI
5 µl of previous PCR product (pMaM17(sfGFP_delY) were added to 11 µl ddH20, 1 µl BamI, 1 µl SalI and 2 µl CutSmart buffer.
Sample rested for 1 hour at room temperature.
Heat inactivation was performed at 95 °C for 1 min.
Purification was performed according to manufacturers protocol.
▼2015-08-22  Digestion of p413-GPD
Goal: Digest p413-GPD with BamI and SalI
3 µl of p413-GPD plasmid were added to 13 µl ddH20, 1 µl BamI, 1 µl SalI and 2 µl CutSmart buffer.
Sample rested for 1 hour at room temperature.
Heat inactivation was performed at 95 °C for 1 min.
Linearized plasmid was purified by performing gel extraction according to manufacturers protocol.
▼2015-08-22  Ligation of p413-GPD_digested and sfGFP_delY_digested
Goal: Ligation of p413-GPD and sfGFP_delY, both previously digested with BamI and SalI
100 ng of digested p413-GPD plasmid and 500 ng of digested sfGFP_delY were ligated using T4 DNA ligase in a 20 µl assay.
Sample rested for 1 hour at room temperature.
Purification was performed according to manufacturers protocol.
NanoDrop determined a concentration of 18.9 ng/µl with 19 µl sample remaining.
▼2015-08-22  Transformation of E. coli with sfGFP_delY in p413-GPD
Goal: Transforming E. coli with sfGFP_delY inserted into p413-GPD after a digest with BamI and SalI
4 x 50 µl of chemically competent E. coli were thawed on ice
4 samples were prepared, each containing 10 µl KCM.
Ligated plasmid was added as 5 µl, 4 µl, 3 µl and 2 µl, one volume per sample.
Samples were filled up to 50 µl with ddH20.
50 µl of hawed cells were added to each sample.
 
Samples were held at 42 °C for 2 min.
 
Samples were incubated on ice for 2 min
900 ul of media was added to each sample.
Samples were incubated at 37 °C for 60 min.
Cells were sedimented by centrifuging at 1000 g for 5 min.
Supernatant was removed.
Cells were resuspended in media.
Cells were spread on one plate each.
Cells incubated overnight at 37°C.
 
▼2015-08-23  Colony picking of E. coli transfected with c(p413-GPD; sfGFP_delY)
Goal: Confirming a culture containing the correct plasmid with sfGFP_delY ligated into p413-GPD.
Colonies were picked from plates
Concetration of plasmid in spread sample
Picked colonies
2 µl
1
3 µl
2
4 µl
3
5 µl
2
Picked colonies were grown overnight as a liquid culture.
week number 33
▼2015-08-10  PCR of Part 1 and 2 of the CFTR2 Ribozyme Insert for in vitro transcription
Durchführung
 
 
P1
[µl]
P2
[µl]
Primer Fwd
DH_54
5
DH_56
5
Primer Rev
DH_55
5
DH_57
5
Template
Insert CFTR2
0,5
Insert CFTR2
0,5
ddH2O
 
14,5
 
14,5
Q5 Polymerase
 
25
 
25
 
 
 
Conditions:
Step
Temperature [°C]
Time
Cycles
Initial denaturation
98
1:00
1
Denturation
98
0:10
35
Annealing
66
0:15
Extension
72
0:15
Final Extension
72
1:00
1
Hold
4
 
 
 
After PCR DNA was purified by ethanol precipitation and stored at -20°C
 
 
▼2015-08-10  PCR of Ribozyme target for in vitro assay
 
Target
[µl]
Primer Fwd
DH_58
5
Primer Rev
DH_59
5
Template
CFTRtestconstruct
0,5
ddH2O
 
14,5
Q5 Polymerase
 
25
 
 
 
 
Conditions:
Step
Temperature [°C]
Time
Cycles
Initial denaturation
98
2:00
1
Denturation
98
0:20
35
Annealing
66
0:20
Extension
72
1:00
Final Extension
72
3:00
1
Hold
4
 
Cycles
 
After PCR DNA was purified by Qiagen PCR Purification Kit ans stored at -20°C.
 
 
▼2015-08-11  In vitro transcription of P1, P2, Ribozyme 2 A/C/T and Ribozyme Target
 
For every reaction:
Stock solution
Final concentration
[µl]
ATP 100mM
4 mM
8
CTP 100mM
4 mM
8
UTP 100mM
4 mM
8
GTP 100mM
4 mM
8
DTT 1M
1 mM
2
DMSO
5%
10
10x Transcription buffer
1x
20
DNA
1 µg
20
ddH20
 
106
T7 RNA polymerase
 
3
 
 
Reaction was incubated for 3 h at 37 °C
After 1.5 h another 2 µL T7 RNA Polymerase were added
Addition of 2 µL DNase I and further incubation at 37 °C for 20 min
The constructs containing hammerhead- or HDV-ribozymes was heaten up to 95° for 5 min, so that there is a coplete cleavage.
 
 
RNA purification by precipitation:
Samples was mixed with 200 µL of 2 x loading dye and purified over a 10 % PAGE
Bands were visualized by UV shadowing and suitable bands were excised
RNA was eluted out of the gel using 0.3 M NaAc pH 5.5 in three elution steps
Gel parts were filtered of and 2.5 volumes of -20 °C EtOH were added, sample was stored at -20 °C oN to let the RNA precipitate
Spin sample at 16,000 g for 30 min, discard the supernatant
Washed the pellet twice with 70 % EtOH and dissolved RNA in 20 µL of MQ water
 
 
 
▼2015-08-11  Labeling of P1 with G-Azide
 
Reaction [µl]
Control without PAP[µl]
Control without template [µl]
PAP (Poly-A-Polymerase, yeast)
1
/
1
RNAse inhibitor
1
1
1
PAP buffer 5x
4
4
4
Nucleotide G-Azide, 100µM
2
2
2
P1-RNA
3 (= 5µM)
3
/
ddH2O
9
10
12
 
 
Reaction was incubated for 2 h at 37 °C
Heat inactivation at 65°C for 10min
Purification by ethanol precipitation
▼2015-08-12  Splinted Ligation of modified P1 and P2
 
Reaction [µl]
Control without Ligase [µl]
Control without Splint [µl]
P1 RNA, labeled
12 (=0,5µM)
12 (=0,5µM)
12 (=0,5µM)
P2 RNA
0,5 (=0,5µM)
0,5 (=0,5µM)
0,5 (=0,5µM)
Splint
0,7 (=2,25µM)
0,7 (=2,25µM)
/
10x Ligation buffer
3
3
3
T4 DNA ligase
1
/
1
ddH2O
11,8
12,8
12,5
 
 
Reaction was heaten up to 95°C for 30s without T4 DNA ligase and then cooled down to room temperature
After 15min  T4 DNA ligase was added.
Reaction was incubated for 1 h at 37 °C
Heat inactivation at 80°C for 10min
▼2015-08-13  Transfer of the ribozymes and inserts into yeast and mammalian vectors
Procedures:
 
Digestion of pSB1C3 + MCS +pcat with and without ribozymes:
 
Description
 
Reaction mix:
1 µg of DNA
0,1 µl of BamHI-HF
0,1 µl of SalI-HF
2 µl Cutsmart
ad 20 µl ddH2O
 
Digestion for 1 hour at 37°C
 
After the reaction the samples were given on a 0,8% Agarose gel and were then purified by gel extraction.
 
Ligation of the ribozymal fragments into p413
 
Description:
 
Chemicals:
1 µl of Vector DNA (about 25 ng) dephosporylated
5 µl of Insert DNA (about 10 ng)
2 µl T4-Ligation Buffer
1 µl T4-Ligase
10 µl ddH2O
 
The Mix was incubated for 30 minutes at room temperature
 
Transformation of p413 + Ribozymes + Inserts:
 
Description:
 
Steps:
 
Take 50µl chemical competent E. coli from -80 freezer and thaw on ice
 
Add (as master mix):
                2,5µl DNA
                10µl KCM 5x
                37,5µl H2O
 
Incubate on ice for 30 minutes
 
Heat shock at 42°C for 1 minute
 
Incubate on ice for 2 minutes
 
Add 900 µl of LB or 2x YT Medium
 
Incubate on 37°C for 60min
 
Centrifuge 5min at 1000g
 
Take 900µl of supernatant and throw away
 
Resuspend pellet in remaining media
 
Plate out on agar with antibiotics (1:1 / 1:10)
 
The prior ligations were used for the transformation.
 
Colony PCR of ribozymes + Inserts:
 
 
 
96-well
 
 
 
 
 
 
 
 
 
 
 
 
 
 
 
 
 
 
 
 
 
 
 
 
 
 
1
2
3
4
5
6
7
8
9
10
11
12
1
1-2a
1-2d
3-4a 1
3-4a 2
3-4a 3
3-4b 1
3-4b 2
3-5b 1
3-5b 2
3-5b 3
 
 
2
4-1a 1
4-1a 2
7-2b
4-1b 1
4-1b 2
4-1b 3
5-2c
6-1b 1
6-1b 2
6-1b 3
6-1c
6-1d
3
7-2a 1
7-2a 2
4-1a 3
7-2e
8-1a 1
8-1a 2
8-1a 3
9-1a
9-1b 1
9-1b 2
9-1b 3
 
4
10-3a 1
10-3a 2
10-3a 3
10-3c 1
10-3c 2
10-3c 3
10-5a 1
10-5a 2
10-5a 3
11-1c 1
11-1c 2
11-1c 3
5
12-1b 1
12-1b 2
12-1b 3
12-1c 1
12-1c 2
12-1c 3
12-1d 1
12-1d 2
12-1d 3
13-1a 1
13-1a 2
13-1a 3
6
13-1b 1
13-1b 2
13-1b 3
14-1c 1
14-1c 2
14-1c 3
14-1e 1
14-1e 2
14-1e 3
15-2b 1
15-2b 2
15-2b 3
7
15-3a 1
15-3a 2
15-3a 3
15-3b 1
15-3b 2
15-3b 3
15-3d 1
15-3d 2
15-3d 3
16-3c 1
16-3c 2
16-3c 3
8
16-3e
16-3d 1
16-3d 2
16-3d 3
 
 
 
 
 
 
 
 
 
3 Colonies were picked from every plate which showed 3 or more colonies.
 
Colonies in the plates after transformation:
 
Ribozyme/Colony
Number of colonies
1-2a
1
1-2d
1
1-2e
0
3-4a
>3
3-4b
2
3-4d
0
3-5a
0
3-5b
>3
3-5e
0
4-1a
>3
4-1b
3
5-2c
1
5-2e
0
6-1b
>3
6-1c
1
6-1d
1
7-2a
2
7-2b
1
7-2c
0
7-2e
1
8-1a
3
9-1a
1
9-1b
3
9-1c
0
10-2a
0
10-3a
3
10-3c
>3
10-5a
>3
10-5b
0
10-5c
0
11-1c
3
12-1b
>3
12-1c
>3
12-1d
>3
12-2a
0
13-1a
>3
13-1b
>3
14-1c
>3
14-1e
>3
15-2b
>3
15-3a
>3
15-3b
>3
15-3d
>3
16-2b
0
16-3c
>3
16-3d
3
16-3e
1
Religation Control
0
 
 
Results:
 
96-well
Positive Colonies green, negative Colonies red
Red font: Maybe positive
 
 
 
 
 
 
 
 
 
 
 
 
 
 
 
 
 
1
2
3
4
5
6
7
8
9
10
11
12
1
1-2a
1-2d
3-4a 1
3-4a 2
3-4a 3
3-4b 1
3-4b 2
3-5b 1
3-5b 2
3-5b 3
 
 
2
4-1a 1
4-1a 2
7-2b
4-1b 1
4-1b 2
4-1b 3
5-2c
6-1b 1
6-1b 2
6-1b 3
6-1c
6-1d
3
7-2a 1
7-2a 2
4-1a 3
7-2e
8-1a 1
8-1a 2
8-1a 3
9-1a
9-1b 1
9-1b 2 + 6-1b 3
9-1b 3
 
4
10-3a 1
10-3a 2
10-3a 3
10-3c 1
10-3c 2
10-3c 3
10-5a 1
10-5a 2
10-5a 3
11-1c 1
11-1c 2
11-1c 3
5
12-1b 1
12-1b 2
12-1b 3
12-1c 1
12-1c 2
12-1c 3
12-1d 1
12-1d 2
12-1d 3
13-1a 1
13-1a 2
13-1a 3
6
13-1b 1
13-1b 2
13-1b 3
14-1c 1
14-1c 2
14-1c 3
14-1e 1
14-1e 2
14-1e 3
15-2b 1
15-2b 2
15-2b 3
7
15-3a 1
15-3a 2
15-3a 3
15-3b 1
15-3b 2
15-3b 3
15-3d 1
15-3d 2
15-3d 3
16-3c 1
16-3c 2
16-3c 3
8
16-3e
16-3d 1
16-3d 2
16-3d 3
 
 
 
 
 
 
 
 
 
 
 
 
 
 
 
 
 
 
 
 
 
 
 
 
 
 
 
Figure 8: Colony PCR row 1
Figure 9: Colony PCR row 2
Figure 10: Colony PCR row 3
Figure 11: Colony PCR row 4
Figure 13: Colony PCR row 5
Figure 14: Colony PCR row 6
Figure 14: Colony PCR row 7
▼2015-08-13  Copper Click – Test of Labeled Part 1 and Splinted ligation
To check if the azide-modified NTPs are incorporated the reactive azide is clicked to an alkyne activated FAM-alkyne fluorophore under copper catalyzed 1,3-dipolar azide-alkyne cycloaddition (CuAAC). Samples are after the reaction separated on a 20 % denaturing PAGE and visualized first in the fluorescent channel for FAM and afterwards stained with SYBR
Gold to visualize the RNA. Controls were performed as well.
Reaction conditions:
 
cStock
cFinal
V[µL]
Phosphate buffer pH 7.0
100 mM
50 mM
12,5
FAM alkyne
10 µM
400 nM
1
Azide modified RNA
1 µM
200 nM
5
Cu(II)SO4
2 mM
100 µM
1.25
THPTA
5 mM
500 µM
2.5
Sodium ascorbate
10 mM
1 mM
2.5
H2O
 
Ad 25
0.25
Final
 
 
25
Conditions as Winz, 2012.
 
All compounds were mixed in the given order
Cu(II)SO4 and THPTA and H2O were mixed before adding to the mixture to let THPTA chelat the Cu
Lastly sodium ascorbate was added to reduce the Cu(II) to Cu(I)
Reaction was incubated for 1 h at 37 °C and afterwards put at -20 °C
Checking success on a 20 % denaturing PAGE
Samples were mixed with an appropriate amount of 2 x loading dye and separated on a 20 % denaturing PAGE
Gel was scanned with a Biorad ChemDoc, GE in fluorescent mode using the pre-set FAM parameters
Afterwards gel was stained using SYBR gold and scanned with the Typoon in SYBR Gold mode
Result and Outlook:
Labeling of RNA with azide and splinted ligation was only partially successful. Labeling and splinted ligation was repeated with different conditions.
week number 32
▼2015-08-03  Digest – Ribozymes (Sequencing) (3.8.15)
 
Volume for 30µl [µl]
DNA
25
CutSmart
2,5
Bmt
0,5
BamHI –HF
0,5
ddH2O
1,5
5µl 2Log-Ladder (NEB)
5µl Purple Loading Dye (NEB)
▼2015-08-04  Ribozyme 1 _CFTR 1 T_ Digest and Ligation
j PCR, PCR Purification
k Digest, Gel
l Ligation
m Transformation
 
 
Digest Rib 1A, B
 
 
Volume for 30µl [µl]
DNA
25
CutSmart
2,5
Bmt
0,5
BamHI –HF
0,5
ddH2O
1,5
 
Ligation 1A, B
 
Ribozyme
[ng/µl]
[µl]
1A
20,4
0,20
1B
13,4
0,30
 
 
Concentration [µl]
Backbone
2
Insert
4ng
T4- Ligase
1
Buffer
1
ddH2O
5,8 (Rib 1A)
5,7 (Rib 1B)
▼2015-08-06  Insert cloning into the pSB1C3 Vector with MCS, pcat and Ribozyme
Procedures:
 
Digestion:
 
Description:
 
Steps:
 
Set up reaction according to protocol:
                ddH2O for a final volume of 20 µl
                2 µl of 10x Reaction Buffer (e.g. NEB CutSmart)
                0.5 µl of selected Enzyme(s)
                ca. 1 µl of mini prep DNA (Range 200-1000 ng)
 
Incubate at 37°C for 60'
 
The digestion was made in two steps in order to trigger the exonuclease activity of one of the endonucleases. The cutting sites of BglII and SalI are to narrow to each other for a endonuclease activity.
 
Step 1: Digestion with BglII
 
Components:
 
1 µg of Vector DNA
2 µl Cutsmart
1 µl of BglII
ad 20 µl ddH2O
 
 
Step 2: Digestion with SalI, triggering of SalI's exonuclease activity
 
Components:
 
8 µl of SalI-HF
10 µl of Cutsmart
52µl ddH2O
30 µl DNA
 
Aterwards a dephosphorylation and a PCR purification was made
 
Step 1: BglII digestion:
 
Nummer
c
µl/µg
µl ddH20
7-2
224
5
12,5
8-1
210
5
12,5
8-2
519
2
15,5
8-3
471
3
14,5
9-1
205
5
12,5
9-2
507
2
15,5
9-5
448
3
14,5
10-1
226
5
12,5
10-2
220
5
12,5
10-3
369
3
14,5
10-5
542
2
15,5
11-1
307
4
13,5
12-1
534
2
15,5
12-2
106
10
7,5
12-4
499
3
14,5
13-1
221
5
12,5
14-1
542
2
15,5
14-3
303
4
13,5
15-2
407
3
14,5
15-3
317
4
13,5
3-4
130
8
9,5
3-5
120
9
8,5
4-1
129
8
9,5
5-2
64
16
1,5
6-1
89
12
5,5
16-3
85
12
5,5
 
    Ligation
 
Description:
 
Materials and chemicals:
2 µl 10x T4 Ligase Buffer: thawed and resuspended at room temperature
1 parts Vector DNA (mol not l)
3 part Insert DNA (mol not l)
20 µl Nuclease free water
1 µl T4 DNA Ligase
 
Steps:
 
Set up the reaction in a microcentrifuge tube on ice. T4 DNA Ligase should be added last. The molar ratio of vector to insert should be 1:3.
 
Gently mix the reaction by pipetting up and down and microfuge briefly.
 
For cohesive (sticky) ends, incubate at 16°C overnight or room temperature for 10 minutes.
                For blunt ends or single base overhangs, incubate at 16°C overnight or room temperature for                2 hours. Alternatively a high concentration of T4 Ligase can be used in a 10 minute ligation.
 
Heat inactivate at 80°C for 10 minutes
 
Chill on ice and transform 1-5 µl of the reaction into 50 µl of competent cells.
 
Chemicals:
25 ng cutted Vector DNA
8 ng cutted Insert DNA
2 µl DNA-Ligase Buffer
1 µl DNA-Ligase
ad 20 µl ddH2O
 
Nr
Ribozym
c
Insert Nr
µl Insert
µl Vector
µl ddH20
T4-Ligase-Buffer
T4-Ligase
3-4
CFTR 1 C
8,5
C1V
0,2
3
13,8
2
1
3-5
CFTR 1 C
16,3
C1V
0,2
2
14,8
2
1
4-1
CFTR 2 T
15,2
C2V
0,2
2
14,8
2
1
5-2
CFTR 2 A
15,2
C2V
0,2
2
14,8
2
1
6-1
CFTR 2 C
21,4
C2V
0,2
2
14,8
2
1
7-2
CFTR 1 DE T
7,2
C1V
0,2
4
12,8
2
1
8-1
CFTR 1 DE A
3,8
C1V
0,2
7
9,8
2
1
8-2
CFTR 1 DE A
10,7
C1V
0,2
3
13,8
2
1
8-3
CFTR 1 DE A
16,6
C1V
0,2
2
14,8
2
1
9-1
CFTR 1 DE C
4,2
C1V
0,2
6
10,8
2
1
9-2
CFTR 1 DE C
5,7
C1V
0,2
5
11,8
2
1
9-5
CFTR 1 DE C
8,9
C1V
0,2
3
13,8
2
1
10-1
CFTR 2 DE T
5
C2V
0,2
5
11,8
2
1
10-2
CFTR 2 DE T
5,7
C2V
0,2
5
11,8
2
1
10-3
CFTR 2 DE T
10
C2V
0,2
3
13,8
2
1
10-5
CFTR 2 DE T
10,5
C2V
0,2
3
13,8
2
1
11-1
CFTR 2 DE a
15
C2V
0,2
2
14,8
2
1
12-1
CFTR 2 DE C
13,6
C2V
0,2
2
14,8
2
1
12-2
CFTR 2 DE C
9
C2V
0,2
3
13,8
2
1
13-1
GFP 1 DE
2,9
G1V
0,2
9
7,8
2
1
14-1
GFP 2 DE
17,6
G2V
0,2
2
14,8
2
1
14-3
GFP 2 DE
6
G2V
0,2
5
11,8
2
1
15-2
GFP 1
14,3
G1V
0,2
2
14,8
2
1
15-3
GFP 1
12,1
G1V
0,2
3
13,8
2
1
16-3
GFP 2
14
G2V
0,2
2
14,8
2
1
 
Insert Nr
c
C1V
49,8
C2V
41,4
G1V
46,6
G2V
40,6
 
 
KCM Transformation:
 
Description:
 
Steps:
 
Take 50µl chemical competent E. coli from -80 freezer and thaw on ice
 
Add (as master mix):
                2,5µl DNA
                10µl KCM 5x
                37,5µl H2O
 
Incubate on ice for 30 minutes
 
Heat shock at 42°C for 1 minute
 
Incubate on ice for 2 minutes
 
Add 900 µl of LB or 2x YT Medium
 
Incubate on 37°C for 60min
 
Centrifuge 5min at 1000g
 
Take 900µl of supernatant and throw away
 
Resuspend pellet in remaining media
 
Plate out on agar with antibiotics (1:1 / 1:10)
 
Colony PCR:
 
Description:
 
Materials and Chemicals:
PCR-tubes
Forward primer
Reverse primer
Masermix (dNTPs, Polymerase, buffer)
Water
Thermocycler
 
Endvolume: 10 µl
 
Steps:
 
Pick colonies from plates. Solute one colony in about 20 µl of water.
 
Give the colonies into 10 µl colony PCR solution with OneTaq Mastermix (which should be diluted to 1x in the end (e.g. you need 5 µl of 2x mastermix for 10 µl)) and primer (between 0,1 and 1 µl)
 
Use the Thermocycler with an appropriate PCR program for at least 25 cycles
 
Results:
 
The colony PCR showed at least one positive clone for every ribozyme.
 
Results:
 
The plates gave a high number of clones. Out of them either new plates were made or single colonies were found, that were given into a colony PCR. Positive Ribozyme colonies were minipreped and gave a yield of 150-325 ng/µl DNA.
 
First colony PCR: (Clones are named: x-y A: x = 1-15 (number of ribozyme), y = number of the colony, A: Colony capital letter (A-E))
 
Colony ID
Positve/Negative/Multiple Bands
15-3 A
Pos
15-3 B
Pos
15-3 C
Neg
15-3 D
Pos
14-3 A
Neg
14-3 B
Neg
12-1 A
Pos
12-1 B
Pos
12-1 C
Pos
12-1 D
Pos
12-1 E
Neg
10-3 A
Double Band
10-3 B
Neg
10-3 C
Pos
10-2 A
Pos
10-2 B
Pos
10-2 C
Neg
9-1 A
Double Band
9-1 B
Double Band
9-1 C
Pos
8-1 A
Pos
8-1 B
Neg
6-1 A
Neg
6-1 B
Pos
6-1 C
Pos
6-1 D
Pos
6-1 E
Neg
4-1 A
Pos
4-1 B
Pos
10-5 A
Pos
10-5 B
Pos
10-5 C
Double Band
10-5 D
Neg
10-5 E
Neg
11-1 A
Neg
11-1 B
Neg
11-1 C
neg
13-1 A
Pos
13-1 B
Pos
15-2 A
Neg
15-2 B
Triple Band
15-2 C
Neg
 
 
 
week number 31
▼2015-07-27  PCR amplification of the ribozymes CFTR2 A/T/C for in vitro transcription
Procedures:
 
Amplification PCR 1
 
Description
 
Chemicals:
5 µl Q5 Master Mix
0,5 µl Ribozymes from stock solution
1 µl Primer fwd
1 µl Primer rev
2,5 µl ddH2O
 
Program:
 
98°C for 1 minute
------------------------------
98°C for 10 seconds
60°C for 20 seconds
72°C for 30 seconds
Repeat 35 times
------------------------------
72°C for 2 minutes
------------------------------
4°C for holding
 
Results:
 
After recognizing the success of the first PCR on a gel, a second PCR was made with to get more DNA.
 
Amplification PCR 2:
 
Description:
 
Chemicals:
 
25 µl Q5 Master Mix
0,5 µl Ribozymes from prior PCR reaction
5 µl Primer fwd
5 µl Primer rev
14,5 µl ddH2O
 
Program:
 
98°C for 30 seconds
------------------------------
98°C for 10 seconds
60°C for 15 seconds
72°C for 20 seconds
Repeat 35 times
------------------------------
72°C for 1 minutes
------------------------------
4°C for holding
▼2015-07-28  Digest-Ribozyme amplification
 
Volume [µl]
DNA
15
CutSmart
3
BamHI-HF
0.3
BmtI-HF
0.3
ddH2O
11.4
 
Conditions
 
Duration: 1h
Temperature: 37°C
350 rpm
 
Concentration
 
Ribozyme
Concentration [ng/µl]
1 CFTR 1 T
8,9
2 CFTR 1 A
15,9
3 CFTR 1 C
22,9
4 CFTR 2 T
1,5
5 CFTR 2 A
15,6
6 CFTR 2 C
31,9
15 GFP 1
14,2
16 GFP 2
17,8
PCR Purification Kit
▼2015-07-28  Ligation (28.07.15)
 
Volume [µl]
Backbone
10ng -> 2
Insert
4ng
T4-Ligase
1
Buffer
1
ddH2O
to 20µl
 
Conditions
Duration: 15min
Temperature:  25°C
350 rpm
▼2015-07-28  KCM - Transformation
See also LabGuru protocol
▼2015-07-29  Picking colonies
Picked colonies
 
Ribozyme
Colony1
Colony 2
Colony 3
Colony 4
Colony 5
1 CFTR 1 T
/
/
/
/
/
2 CFTR 1 A
2-1
2-2
2-3
2-4
2-5
3 CFTR 1 C
3-1
3-2
3-3
3-4
3-5
4 CFTR 2 T
4-1
4-2
4-3
4-4
4-5
5 CFTR 2 A
5-1
5-2
5-3
5-4
5-5
6 CFTR 2 C
6-1
6-2
6-3
6-4
6-5
15 GFP 1
15-1
15-2
15-3
15-4
15-5
16 GFP 2
16-1
16-2
16-3
16-4
16-5
▼2015-07-29  colony PCR
Colony PCR
 
 
Volume [ng/µl]
Primer fwd
0,5
Primer rvs
0,5
OneTaq Polymerase
5
ddH2O
4
▼2015-07-30  Repetition Ribozyme 1 and 15
PCR (GEL image)
 
Digest (GEL image)
 
Ribozyme
Concentration [ng/µl]
1 CFTR 1 T
17,4
15 GFP 1
6
Backbone
37
 
 
Ligation
 
ddH2O
To 20µl
Backbone
50ng
Insert
20ng
T4 Ligase
1µl
Buffer
2µl
 
Transformation
 
 
▼2015-07-30  Mutation correction PCR of p415-GPD
Description:
 
In order to correct a stop-codon mutation in the CFTR-testconstruct a mutation correction PCR was made.
 
Mutation correction PCR:
 
Description:
 
Chemicals:
0,2 µl p415-GPD CFTR-test
1 µl Primer fwd
1 µl Primer rev
2,8 µl ddH2O
5 µl Q5 Master Mix
 
Program:
 
98°C for 2 minutes
--------------------------
98°C for 30 seconds
Annealing temperature for 30 seconds 35 cycles
72°C for 4 minutes
--------------------------
72°C for 5 minutes
--------------------------
4°C for holding
 
The reaction was set in a triplet with 3 different annealing temperatures: 72°C (2step), 70°C and 68°C
 
DpnI digestion:
 
Description:
 
1 µl of DnpI was given on the PCR mix after the PCR reaction. The mix was incubated at 37°C for 4 hours and then inactivated at 65°C for 20 minutes.
 
The DNA was given on a gel after the DpnI-digestion
 
Results:
 
The gel picture showed just smear.
▼2015-07-31  Miniprep – QIAGEN Kit (31.7.15)_concentrations
Ribozyme + number
Concentration [ng/µl]
Volume for Sequencing(40ng) [µl]
1
/
/
2_2-2
99
6,06
3_3-4/5
90,6
6,62
4_4-1/4-2
116,9
5,13
5_5-1/2
99,3
6,04
6_6-1
87,3
6,87
15
 
 
16_16-3
89,8
6,68
▼2015-07-31  Colony PCR and mini prep of Ribozymes
Colony PCR
 
Miniprep – QIAGEN Kit 
 
Ribozyme + number
Concentration [ng/µl]
Volume for Sequencing(40ng) [µl]
1
/
/
2_2-2
99
6,06
3_3-4/5
90,6
6,62
4_4-1/4-2
116,9
5,13
5_5-1/2
99,3
6,04
6_6-1
87,3
6,87
15
 
 
16_16-3
89,8
6,68
▼2015-07-31  Repairing the deletion in the HDV region of the twin ribozymes with assembly PCR + Cloning
Description:
 
In order to repair a deletion in the DE inserts and ribozymes, 3 PCRs were made. In the first PCR the parts were elongated with a Primer overhang that carries the DNA fragment to fill up the deletion. The second was an assembly PCR. The third has been done to amplify the fragments.
 
ID #
Ribozyme
1
CFTR 1 T
2
CFTR 1 A
3
CFTR 1 C
4
CFTR 2 T
5
CFTR 2 A
6
CFTR 2 C
7
CFTR 1 DE T
8
CFTR 1 DE A
9
CFTR 1 DE C
10
CFTR 2 DE T
11
CFTR 2 DE A
12
CFTR 2 DE C
13
GFP 1 DE
14
GFP 2 DE
15
GFP 1
16
GFP 2
 
 
PCR: Division and Elongation
 
Description
 
PCR-Mix:
5 µl Primer fwd
5 µl Primer rev
0,5 µl Template
14,5 µl ddH2O
25 µl Q5-MasterMix
 
Program:
 
98°C for 30 seconds
------------------------------------------------
98°C for 10 seconds
68°C for 10 seconds 35 Cycles
72°C for 15 seconds
------------------------------------------------
72°C for 30 seconds
------------------------------------------------
4°C for holding
 
The PCR to make two parts is made in two different tubes, one for the first and one for the second part.
 
First part:
 
Ribozyme/Insert
Fwd Primer
Rev Primer
CFTR 1 DE T
DH_39
MJ_15
CFTR 1 DE A
DH_39
MJ_15
CFTR 1 DE C
DH_39
MJ_15
CFTR 2 DE T
DH_40
MJ_15
CFTR 2 DE A
DH_40
MJ_15
CFTR 2 DE C
DH_40
MJ_15
GFP 1 DE
DH_41
MJ_18
GFP 2 DE
DH_41
MJ_18
Insert CFTR 1
DH_18a
MJ_17
Insert CFTR 2
DH_18a
MJ_16
Insert GFP 1
DH_20a
MJ_20
Insert GFP 2
DH_22a
MJ_19
 
Second Part:
 
Riobzyme/Insert
Fwd-Primer
Rev-Primer
Tubes
CFTR 1 DE T
MJ_21
DH_48
4
Insert CFTR 1
MJ_21
DH_27a
2
 
 
Assembly PCR
 
Description:
 
PCR-Mix:
1:1 Molar ratio of part 1 and part 2 (from the first PCR)
25 µl Q5-MasterMix
ad 40 µl ddH2O
 
PCR-program:
 
98°C for 1 minute
--------------------------------------------
98°C for 10 seconds
70°C with a ramp rate of 1°C per second for 30 seconds
72°C for 30 seconds
Repeat 3 Times
--------------------------------------------
72°C for 30 seconds
-------------------------------------------
4°C for holding
 
 
Frag-ment 1
Length
ng/µl
nmol
Frag-ment 2
Length
ng/µl
nmol
Frag-ment 2 Ratio
F1 µl
F2 µl
ddH2O µl
2T
226
142
1,017
Rib
80
327,2
6,62
0,15
1,5
0,2
23,3
2A
226
184,9
1,324
Rib
80
327,2
6,62
0,2
1,5
0,3
23,2
2C
226
117,5
0,841
Rib
80
327,2
6,62
0,12
1,5
0,2
23,3
G1
227
199,3
1,42
Rib
80
327,2
6,62
0,2
1,5
0,3
23,2
G2
243
166,5
1,109
Rib
80
327,2
6,62
0,16
1,5
0,3
23,2
IC1
109
120,2
1,784
Ins
87
205,6
6,62
0,46
1,5
0,7
22,8
IC2
118
128,3
1,759
Ins
87
205,6
6,62
0,46
1,5
0,7
22,8
1T
216
150,2
1,125
Rib
80
327,2
6,62
0,17
1,5
0,3
23,2
1A
216
121
0,906
Rib
80
327,2
6,62
0,14
1,5
0,2
23,3
1C
216
98,5
0,737
Rib
80
327,2
6,62
0,11
1,5
0,2
23,3
IG1
114
145,7
2,067
Ins
87
205,6
6,62
0,54
1,5
0,8
22,7
IG2
115
151,3
2,128
Ins
87
205,6
6,62
0,56
1,5
0,8
22,7
 
 
 
 
 
 
 
 
 
 
 
 
 
 
 
PCR: Amplification
 
Description:
 
PCR-Mix:
 
5 µl Primer fwd and rev are given into the assembly PCR mix after the assembly PCR
 
 
Program:
 
98°C for 60 seconds
----------------------------------
98°C for 10 seconds
68°C for 15 seconds 35 Cycles
72°C for 20 seconds
------------------------------------
72°C for 30 seconds
------------------------------------
4°C for holding
 
Digestion:
 
Description:
 
Steps:
 
Set up reaction according to protocol:
                ddH2O for a final volume of 20 µl
                2 µl of 10x Reaction Buffer (e.g. NEB CutSmart)
                0.5 µl of selected Enzyme(s)
                Ca. 1 µl of mini prep DNA (Range 200-1000 ng)
 
Incubate at 37°C for 60'
 
Digestion of the ribozymes was made with BamHI and BmtI.
 
Ligation
 
Description:
 
Materials and chemicals:
2 µl 10x T4 Ligase Buffer: thawed and resuspended at room temperature
1 parts Vector DNA (mol not l)
3 part Insert DNA (mol not l)
20 µl Nuclease free water
1 µl T4 DNA Ligase
 
Steps:
 
Set up the reaction in a microcentrifuge tube on ice. T4 DNA Ligase should be added last. The molar ratio of vector to insert should be 1:3.
 
Gently mix the reaction by pipetting up and down and microfuge briefly.
 
For cohesive (sticky) ends, incubate at 16°C overnight or room temperature for 10 minutes.
 
For blunt ends or single base overhangs, incubate at 16°C overnight or room temperature for 2 hours. Alternatively a high concentration of T4 Ligase can be used in a 10 minute ligation.
 
Heat inactivate at 80°C for 10 minutes
 
Chill on ice and transform 1-5 µl of the reaction into 50 µl of competent cells.
 
The ribozymes were ligated into a cutted and dephoshorylated pSB1C3 with MCS and pcat.
 
KCM Transformation:
 
Description:
 
Steps:
 
Take 50µl chemical competent E. coli from -80 freezer and thaw on ice
 
Add (as master mix):
                2,5µl DNA
                10µl KCM 5x
                37,5µl H2O
 
Incubate on ice for 30 minutes
 
Heat shock at 42°C for 1 minute
 
Incubate on ice for 2 minutes
 
Add 900 µl of LB or 2x YT Medium
 
Incubate on 37°C for 60min
 
Centrifuge 5min at 1000g
 
Take 900µl of supernatant and throw away
 
Resuspend pellet in remaining media
 
Plate out on agar with antibiotics (1:1 / 1:10)
 
Colony PCR:
 
Description:
 
Materials and Chemicals:
PCR-tubes
Forward primer
Reverse primer
Masermix (dNTPs, Polymerase, buffer)
Water
Thermocycler
 
Endvolume: 10 µl
 
Steps:
 
Pick colonies from plates. Solute one colony in about 20 µl of water.
 
Give the colonies into 10 µl colony PCR solution with OneTaq Mastermix (which should be diluted to 1x in the end (e.g. you need 5 µl of 2x mastermix for 10 µl)) and primer (between 0,1 and 1 µl)
 
Use the Thermocycler with an appropriate PCR program for at least 25 cycles
 
Results:
 
The colony PCR showed at least one positive clone for every ribozyme
 
Figure 6: Colony PCR of the cloned DE ribozymes
Positive Colonies: 7-2, 8-1, 8-2, 8-3, 9-1, 9-2, 9-5, 10-1, 10-2, 10-3, 10-5, 11-1, 12-1, 12-2
 
Figure 7: Colony PCR of the cloned De ribozymes
Positive Colonies: 12-4, 13-1, 14-1, 14-3, 15-2, 15-3
 
week number 30
▼2015-07-24  Yeast transformation with p415-GPD with CFTR-testconstruct without mutation correction
Description
 
Yeast was transformated with p415-GPD with the CFTR-testconstruct from a miniprep of an E.coli culture. Transformed cells should turn red.
 
Procedures:
 
Yeast transformation 24.07
 
Description:
 
Materials and chemicals:
10 µl of cells for transformation with a plasmid, 50 µl of cells for transformation with a PCR product
2 µl of plasmid DNA per 10 µl of cells (200 ng of DNA)
6 equivalents of PEG to the yeast cells
1/9 of DMSO to the volume of plasmids, yeast cells and PEG
100 - 200 µl of liquid medium
 
Steps:
 
Give the plasmid DNA into a sterile 1.5 ml tube (2 µl per 10 µl of cells. Add the thawed competent cells.
 
Mix the suspension well, then add the PEG.
 
Incubate for 30 mins at room temperature while mixing (the cells can be incubated up to 2 hours).
 
Add the DMSO to get a final concentration of 10% DMSO.
 
Place the yeast in a 42°C water bath for 5-20 minutes. NO Thermomixer.
 
Centrifuge cells for 2-3 minutes at 2000 rpm/500g
 
Discard the supernatant and resuspend the yeast in the liqiud YPD medium
 
The transformation was repeated on 27.07.2015
 
Results:
 
Transformation 1:
The first transformation resulted in cultures with an ununsual behaivior, maybe the plates were contaminated with another organism. The cells didn't turned red
 
Transformation 2:
A test run on a FACS showed that the transformed cells are functional and express the CFTR-testconstruct
▼2015-07-25  Mutation correction PCR p415-GPD with CFTR-testconstruct
Description
 
In order to correct the mutation in the mCherry part of the CFTR-testcontruct, a mutation PCR was made.
 
Procedures:
 
Mutation correction PCR 1:
 
Description
 
Chemicals:
0,1 µl p415-GPD with CFTR-testconstruct (about 650 ng/µl)
1 µl Primer fwd
1 µl Primer rev
2,9 µl ddH2O
5 µl Q5
 
Program:
 
2-step PCR:
 
98°C
-----------------------
98°C
72°C
------------------------
72°C
------------------------
4°C for holding
 
Mutation correction PCR 2
 
Description:
 
Chemicals:
0,1 µl p415-GPD with CFTR-testcontruct
1 µl Primer fwd
1 µl Primer rev
2,9 µl ddH2O
5 µl Q5
 
Program:
 
98°C for 1 minute
------------------------
98°C for 20 s
Anneal for 20 s
72°C for 4 minutes
Repeat cycle 35 times
-------------------------
72°C for 5 minutes
-------------------------
4°C for holding
 
Tried annealing temperatures:
72°C (2-step-PCR)
70,5°C
68,8°C
65,6°C
64°C
 
Annealing temperatures were made by gradient
 
Results:
 
Mutation PCR 1:
The Gel showed there was just smear
 
Mutation PCR 2:
Every Annealing temperature resulted in smear and no detectable fragment
 
Figure 5: The second mutation PCR of the CFTR-testconstruct
week number 29
▼2015-07-14  Inserting twin ribozymes into pSB1C3 + MCS + pcat
Description:
 
After validating the MCS + pcat fragment, the ribozymes were cloned into pSB1C3
 
Procedures:
 
2-step digest:
 
Description:
 
Steps:
 
Set up reaction according to protocol:
                ddH2O for a final volume of 20 µl
                2 µl of 10x Reaction Buffer (e.g. NEB CutSmart)
                0.5 µl of selected Enzyme(s)
                ca. 1 µl of mini prep DNA (Range 200-1000 ng)
 
Incubate at 37°C for 60'
 
Load on gel (add loading dye first)
 
Two digestion steps were made:
 
Step 1:
 
Set up following mix:
                25 µl pSB1C3-MCS-pcat
                1 µl BmtI
                5 µl CutSmart 10x Buffer
                19 µl ddH2O
 
Digestion for 1 hour at 37°C
 
PCR-purification with 2 step elution in 35 µl water
 
 
Step 2:
 
Set up following mix:
                35 µl from the PCR-purification
                10 µl BamHF
                5 µl CutSmart
 
Digestion for 4 hours at 37°C
 
PCR purification with 2 step elution in 30 µl Elution Buffer
 
    Ligation:
 
Description:
 
Materials and chemicals:
2 µl 10x T4 Ligase Buffer: thawed and resuspended at room temperature
1 parts Vector DNA (mol not l)
3 part Insert DNA (mol not l)
20 µl Nuclease free water
1 µl T4 DNA Ligase
 
Steps:
 
Set up the reaction in a microcentrifuge tube on ice. T4 DNA Ligase should be added last. The molar ratio of vector to insert should be 1:3.
 
Gently mix the reaction by pipetting up and down and microfuge briefly.
 
For cohesive (sticky) ends, incubate at 16°C overnight or room temperature for 10 minutes.
 
For blunt ends or single base overhangs, incubate at 16°C overnight or room temperature for 2 hours. Alternatively a high concentration of T4 Ligase can be used in a 10 minute ligation.
 
Heat inactivate at 80°C for 10 minutes
 
Chill on ice and transform 1-5 µl of the reaction into 50 µl of competent cells.
 
We made one ligation for every ribozyme and one as a religation control (17 in total)
 
Ligation mixes:
 
Not DE-ribozymes (without Hammerhead and Hepatitis Delta Virus ribozyme on the ends):
1 µl Backbone (35 ng)
2,5 µl Insert (12,5 ng)
1 µl T4-Ligase
2 µl T4-Ligation Buffer 10x
13,5 µl ddH2O
 
DE-ribozymes (with HH and HDV):
1 µl Backbone
4 µl Insert
1 µl T4-Ligase
2 µl T4-Ligation Buffer 10x
12 µl ddH2O
 
Religation Control:
1 µl Backbone
1 µl T4-Ligase
2 µl T4-Ligation Buffer 10x
16 µl ddH2O
 
KCM Transformation:
 
Description:
 
Steps:
 
Take 50µl chemical competent E. coli from -80 freezer and thaw on ice
 
Add (as master mix):
                2,5µl DNA
                10µl KCM 5x
                37,5µl H2O
 
Incubate on ice for 30 minutes
 
Heat shock at 42°C for 1 minute
 
Incubate on ice for 2 minutes
 
Add 900 µl of LB or 2x YT Medium
 
Incubate on 37°C for 60min
 
Centrifuge 5min at 1000g
 
Take 900µl of supernatant and throw away
 
Resuspend pellet in remaining media
 
Plate out on agar with antibiotics (1:1 / 1:10)
 
We made 1 transformation for every ligation (17 in total), one for each ribozyme and one as a religation control
 
Colony PCR:
 
Description:
 
Materials and Chemicals:
PCR-tubes
Forward primer
Reverse primer
Masermix (dNTPs, Polymerase, buffer)
Water
Thermocycler
 
Endvolume: 10 µl
 
Steps:
 
Pick colonies from plates. Solute one colony in about 20 µl of water.
 
Give the colonies into 10 µl colony PCR solution with OneTaq Mastermix (which should be diluted to 1x in the end (e.g. you need 5 µl of 2x mastermix for 10 µl)) and primer (between 0,1 and 1 µl)
 
Use the Thermocycler with an appropriate PCR program for at least 25 cycles
 
We picked 12 colonies from every ribozyme plate and got no satisfying results
▼2015-07-15  Digestion of pSB1C3-MCS-pcat with BmtI and BamHI 2 step
Description:
 
We made a 2 step digestion with BmtI and BamHI for 1 and 4 hours to proper cut the pSB1C3 in the MCS region, because the BmtI and BamHI cutting sites are next to each other and the exonuclease activity of the BamHI needs to be triggered.    
 
Procedures:
 
2 step ligation:
 
Description:
 
First digetion with BmtI:
 
Steps:
 
Make a mix of following chemicals in a 1,5 ml centrifuge tube:
                25 µl pSB1C3-MCS-pcat
                1 µl BmtI
                5 µl CutSmart 10x Buffer
                19 µl ddH2O
Incubate for 1 hour at 37°C
Make a PCR purification
 
 
Second digestion with BamHI:
 
Steps:
 
Make a mix of following chemicals:
                35 µl of DNA solution from the prior PCR purification
                10 µl BamHI
                5 µl CutSmart 10x Buffer
Incubate for 4 hours at 37°C
Make a PCR purification
 
Results:
 
Nanodrop result: 24,6 ng/µl
 
▼2015-07-15  Digestion of the ribozyme fragments
Description:
The ribozymes were cut for the insertion into pSB1C3 with MCS/pcat.
 
Procedures:
 
Ribozyme digestion:
 
Description:
 
Steps
 
Chemicals (per tube):
                6 µl DNA solution (~60 ng)
                1 µl Cutsmart
                0,1 µl BamHI
                01, µl BmtI
                2,8 µl ddH2O
 
Incubation for 1 hour at 37°C
 
PCR-purificaiton, with 2 step-elution
 
Lyophylisation
 
Solute dried DNA in 10 µl water
 
Results:
 
Nanodrop results:
 
Number
Ribozyme
Nanodrop-Result
1
CFTR 1 T
19,4
2
CFTR 1 A
77
3
CFTR 1 C
94,8
4
CFTR 2 A
97
5
CFTR 2 C
91,1
6
GFP 1
77,3
7
GFP 1 DE
38,6
8
CFTR 2 DE C
55,3
9
CFTR 1 DE T
72,7
10
CFTR 1 DE A
60,6
11
GFP 2 DE
65,6
12
CFTR 1 DE C
56,3
13
CFTR 2 DE T
100,2
14
CFTR 1 DE A
29,3
15
CFTR 2 T
62,5
16
GFP 2
94,2
 
▼2015-07-15  Ligation of the cutted pSB1C3-MCS-pcat with ribozyme fragments
Procedures:
 
Ligation:
 
Description:
 
Materials and chemicals:
2 µl 10x T4 Ligase Buffer: thawed and resuspended at room temperature
1 parts Vector DNA (mol not l)
3 part Insert DNA (mol not l)
20 µl Nuclease free water
1 µl T4 DNA Ligase
 
Steps:
 
Set up the reaction in a microcentrifuge tube on ice. T4 DNA Ligase should be added last. The molar ratio of vector to insert should be 1:3.
 
Gently mix the reaction by pipetting up and down and microfuge briefly.
 
For cohesive (sticky) ends, incubate at 16°C overnight or room temperature for 10 minutes.
 
For blunt ends or single base overhangs, incubate at 16°C overnight or room temperature for 2 hours. Alternatively a high concentration of T4 Ligase can be used in a 10 minute ligation.
 
Heat inactivate at 80°C for 10 minutes
 
Chill on ice and transform 1-5 µl of the reaction into 50 µl of competent cells.
 
Pipetting sheet:
 
For fragment numbers see digestion of ribozyme fragments protocol
 
In one tube:
40 ng of pSB1C3 (about 1,5 µl)
13 ng of non-DE-ribozymes or 22 ng of DE ribozymes
1 µl Ligase
2 µl Buffer
 
Mastermix: pSB1C3, Ligase used 5x concentrated ligase (2 Million U per ml) diluted with ddH2O, Buffer
 
Everything in µl:
 
Fragment-Number
Mastermix
Insert
ddH2O
1
4,5
0,7
14,8
2
4,5
0,2
15,3
3
4,5
0,2
15,3
4
4,5
0,2
15,3
5
4,5
0,2
15,3
6
4,5
0,2
15,3
7
4,5
0,6
14,9
8
4,5
0,4
15,1
9
4,5
0,3
15,2
10
4,5
0,4
15,1
11
4,5
0,4
15,1
12
4,5
0,4
15,1
13
4,5
0,3
15,2
14
4,5
0,8
14,7
15
4,5
0,2
15,3
16
4,5
0,2
15,3
 
▼2015-07-16  Digestion of pSB1C3-MCS-pcat with BmtI and BamHI 2 step
Description:
 
We made a 2 step digestion with BmtI and BamHI for 1 and 4 hours to proper cut the pSB1C3 in the MCS region, because the BmtI and BamHI cutting sites are next to each other and the exonuclease activity of the BamHI needs to be triggered.
 
Procedures:
 
2 step ligation:
 
Description:
 
First digetion with BmtI:
 
Steps:
 
Make a mix of following chemicals in a 1,5 ml centrifuge tube:
                25 µl pSB1C3-MCS-pcat
                1 µl BmtI
                5 µl CutSmart 10x Buffer
                19 µl ddH2O
 
Incubate for 1 hour at 37°C
 
Make a PCR purification
 
Second digestion with BamHI:
 
Steps:
 
Make a mix of following chemicals:
                35 µl of DNA solution from the prior PCR purification
                10 µl BamHI
                5 µl CutSmart 10x Buffer
 
Incubate for 4 hours at 37°C
 
Make a PCR purification
▼2015-07-18  Outamplification and transformation of the CFTR-testconstruct
Description:
 
In order to change the cloning system from Gibson to restriction cloning, the CFTR fragment has been first mutated (single base mutation on sequencing data) and then outamplified. Afterwards it has been cutted, ligated with the vector and then transformed into E. coli
 
Mutation PCR #1:
 
Description:
 
Materials and chemicals:
25 µl Phusion flash
1 µl Primer fwd Dh_23
1 µl Primer rev DH_24
0,5 µl p413+CFTR-construct (which is inserted the other way around)
22,5 µl water
 
PCR:
 
98°C for 1 min
--------------------
98°C for 10 sec
72°C for 2 minutes
(Repeat 34 times)
---------------------
72°C for 4 minutes
4°C for holding
 
Mutation PCR #2
 
Description:
 
Materials and chemicals:
5 µl Phusion flash
1 µl Primer fwd Dh_23
1 µl Primer rev DH_24
0,5 µl p413+CFTR-construct (which is inserted the other way around)
2,5 µl water
 
PCR:
 
98°C for 1 min
--------------------
98°C for 10 sec
Annealing temperature for 30 sec
72°C for 1,5 min
 (Repeat 34 times)
---------------------
72°C for 4 minutes
4°C for holding
 
4 samples with different annealing temperatures were made:
 
1: 72°C
2: 70°C
3: 68°C
4: 65°C
 
Materials and chemicals:
5 µl Phusion flash
1 µl Primer fwd Dh_23
1 µl Primer rev DH_24
0,5 µl p413+CFTR-construct (which is inserted the other way around)
2,5 µl water
 
PCR:
 
98°C for 1 min
--------------------
98°C for 10 sec
Annealing temperature for 30 sec
72°C for 1,5 min
 (Repeat 34 times)
---------------------
72°C for 4 minutes
4°C for holding
 
4 samples with different annealing temperatures were made:
 
1: 72°C
2: 70°C
3: 68°C
4: 65°C
 
 
Materials and chemicals:
5 µl Phusion flash
1 µl Primer fwd Dh_23
1 µl Primer rev DH_24
0,5 µl p413+CFTR-construct (which is inserted the other way around)
2,5 µl water
 
PCR:
 
98°C for 1 min
--------------------
98°C for 10 sec
Annealing temperature for 30 sec
72°C for 1,5 min
(Repeat 34 times)
---------------------
72°C for 4 minutes
4°C for holding
 
4 samples with different annealing temperatures were made:
 
1: 72°C
2: 70°C
3: 68°C
4: 65°C
 
The 65°C annealing temperature gave the best results and further work was made with the sample 4.
 
Outamplification and rebuild of the end sites for the CFTR construct
 
Description:
 
Materials and chemicals:
25 µl Phusion Flash
1 µl DH_25
1 µl DH_26
1 µl DNA from Sample 4 of mutation PCR #2
22 µl ddH2O
 
PCR-program:
98°C for 1 min
------------------------
98°C for 10 s
64°C for 15 s
72°C for 20 s
 (Repeat 34 times)
-----------------------
72°C for 1 min
4°C for holding
 
CFTR-Digestion:
 
Digestion CFTR-construct:
 
26 µl CFTR-Test construct
5 µl BamHI
5 µl HindIII
4 µl CutSmart 10x
Digest 3 hours at 37°C
 
Digestion p413-GPD:
 
10 µl p413-GPD
1 µl BamHI
1 µl HindIII
2 µl CutSmart 10x
6 µl ddH2O
 
Digest 60 minutes at 37°C
 
Ligation:
 
Description:
 
Materials and chemicals:
2 µl 10x T4 Ligase Buffer: thawed and resuspended at room temperature
1 parts Vector DNA (mol not l)
3 part Insert DNA (mol not l)
20 µl Nuclease free water
1 µl T4 DNA Ligase
 
Steps:
 
Set up the reaction in a microcentrifuge tube on ice. T4 DNA Ligase should be added last. The molar ratio of vector to insert should be 1:3.
 
Gently mix the reaction by pipetting up and down and microfuge briefly.
 
For cohesive (sticky) ends, incubate at 16°C overnight or room temperature for 10 minutes.
 
For blunt ends or single base overhangs, incubate at 16°C overnight or room temperature for 2 hours. Alternatively a high concentration of T4 Ligase can be used in a 10 minute ligation.
 
Heat inactivate at 80°C for 10 minutes
 
Chill on ice and transform 1-5 µl of the reaction into 50 µl of competent cells
 
KCM Transformation:
 
Description:
 
Steps:
 
Take 50µl chemical competent E. coli from -80 freezer and thaw on ice
 
Add (as master mix):
                2,5µl DNA
                10µl KCM 5x
                37,5µl H2O
 
Incubate on ice for 30 minutes
 
Heat shock at 42°C for 1 minute
 
Incubate on ice for 2 minutes
 
Add 900 µl of LB or 2x YT Medium
 
Incubate on 37°C for 60min
 
Centrifuge 5min at 1000g
 
Take 900µl of supernatant and throw away
 
Resuspend pellet in remaining media
 
Plate out on agar with antibiotics (1:1 / 1:10)
 
Colony PCR:
 
Description:
 
Materials and Chemicals:
PCR-tubes
Forward primer
Reverse primer
Masermix (dNTPs, Polymerase, buffer)
Water
Thermocycler
 
Endvolume: 10 µl
 
Steps:
 
Pick colonies from plates. Solute one colony in about 20 µl of water.
 
Give the colonies into 10 µl colony PCR solution with OneTaq Mastermix (which should be diluted to 1x in the end (e.g. you need 5 µl of 2x mastermix for 10 µl)) and primer (between 0,1 and 1 µl)
 
Use the Thermocycler with an appropriate PCR program for at least 25 cycles
 
week number 27
▼2015-07-01  Ribozyme amplification
 
Number
Ribozyme
Primer fwd
Primer rvs
1
CFTR 1 T
DH_35
DH_45
2
CFTR 1 A
3
CFTR 1 C
4
CFTR 2 T
DH_36
DH_46
5
CFTR 2 A
DH_37
DH_47
6
CFTR 2 C
DH_38
7
CFTR 1 DE T
DH_39
DH_48
8
CFTR 1 DE A
9
CFTR 1 DE C
10
CFTR 2 DE T
DH_40
11
CFTR 2 DE A
12
CFTR 2 DE C
13
GFP 1 DE
DH_41
14
GFP 2 DE
DH_42
15
GFP 1
DH_43
DH_49
16
GFP 2
DH_44
 
 
 
 
 
PCR - amplification
 
 
 for 10µl [µl]
for 50µl [µl]
Template
1
5
Primer fwd
1
5
Primer rvs
0.5
0.5
ddH2O
2.5
14.5
Q5 Polymerase
5
25
 
 
PCR conditions
 
 
Time [s]
Temperature [°C]
Initial denaturation
30
98
Denaturation
10
98
Annealing
10
66
Elongation
15
72
Final extension
30
72
Hold
∞
4
 
 
PCR Purification
QIAGEN Kit
▼2015-07-05  pSB1C3-vector assembly with a new artificial multiple cloning site and pcat promotor
Description:
In this experiment, we wanted to insert a Multiple Cloning Site (MCS) and a pCat promotor into the pSB1C3-vector
 
Procedures:
 
Oligo-Annealing:
 
Materials and chemicals:
    5 µl forward oligo
    5 µl reverse oligo
    95 µl sterile water
 
Steps:
 
Incubate phosphorylated oligos at 95°C for 3 minutes
 
Cool the reaction down slowly for 30 minutes to one hour
 
Check the concentration on the nanodrop
 
Ligation:
 
        Materials and chemicals:
 
        2 µl 10x T4 Ligase Buffer: thawed and resuspended at room temperature
        1 parts Vector DNA (mol not l)
        3 part Insert DNA (mol not l)
        1 µl T4 DNA Ligase
        To 20µl nuclease free water
 
Steps:
 
Set up the reaction in a microcentrifuge tube on ice. T4 DNA Ligase should be added last. The molar ratio of vector to insert should be 1:3.
 
Gently mix the reaction by pipetting up and down and microfuge briefly.
 
For cohesive (sticky) ends, incubate at 16°C overnight or room temperature for 10 minutes.
                For blunt ends or single base overhangs, incubate at 16°C overnight or room temperature for                2 hours. Alternatively a high concentration of T4 Ligase can be used in a 10 minute ligation.
 
Heat inactivate at 80°C for 10 minutes
 
Chill on ice and transform 1-5 µl of the reaction into 50 µl of competent cells.
 
Notes:
 
We used several ligation strategies and insert to backbone concentrations. We used a annealing product for the ligation
 
20 µl test digest
 
Description:
 
Steps
Set up reaction according to protocol:
                               ddH2O for a final volume of 20 µl
                               2 µl of 10x Reaction Buffer (e.g. NEB CutSmart)
                               0.5 µl of selected Enzyme(s)
                               ca. 1 µl of mini prep DNA (Range 200-1000 ng)
 
Incubate at 37°C for 60'
 
Load on gel (add loading dye first)
 
Notes:
 
We wanted to verify our results by digestion with:
                1. An enzyme that is exclusively on pSB1C3
                2. An enzyme that is exclusively on our new inserted multiple cloning site
 
Oligo Phosphorylation:
 
Description:
 
Materials and Chemicals:
 
        2 µl 100 µM oligo stock
        2 µl 10x T4 DNA ligase buffer
        1 µl T4 polynucleotide kinase
        15 µl sterile water
 
Steps:
 
Pipette the Chemicals in one micro centrifuge tube
 
Incubate the reaction mixture at 37°C for 1 hour
 
Heat inactivate the polynucleotide kinases at 65°C for 20 minutes.
 
KCM Transformation:
 
Description:
 
Steps:
 
Take 50µl chemical competent E. coli from -80 freezer and thaw on ice
 
Add (as master mix):
                2,5µl DNA
                10µl KCM 5x
                37,5µl H2O
 
Incubate on ice for 30 minutes
 
Heat shock at 42°C for 2 minutes
 
Incubate on ice for 2 minutes
 
Add 900 µl of LB or 2x YT Medium
 
Incubate on 37°C for 60min
 
Centrifuge 5min at 1000g
 
Take 900µl of supernatant and throw away
 
Resuspend pellet in remaining media
 
Plate out on agar with antibiotics (1:1 / 1:10)
 
 
Results:
 
None of our ligations worked to assemble the pSB1C3, MCS and pCat. We used a annealing product without considering that our product has no 5'phosphate and can therefore not be ligated by the T4 ligase. Therefore we used the T4 polynucleotide kinase and tried to ligate again, but the ligation did not worked again.
 
Conclusion:
 
We ordered new primers with a 5' phosphate, to try the ligation of the PCR-products (MCS and pCat) with the pSB1C3
week number 26
▼2015-06-22  Miniprep and cryostock of culture 10 in colony PCR
Description:
 
Procedures:
 
Qiaprep Spin Miniprep Kit:
 
Description:
 
    Steps:
Prepare o/n culture
Perform mini prep according to manufacturers protocol
 
E.coli glycerol stocks:
 
Description:
 
    Steps:
Grow up an overnight culture of strains of interest
Transfer 500µl into a safe lock reaction tube
Add 500µl of 40% sterile glycerol solution
Freeze slowly at -80°C
 
Results:
 
Miniprep: 451,5ng/µl
2 stocks were frozen at -80°
▼2015-06-23  Insert of MCS + promotor in cutted pSB1C3
Description:
 
Procedures:
 
Ligation:
 
Description:
 
        Materials and chemicals:
 
        2 µl 10x T4 Ligase Buffer: thawed and resuspended at room temperature
        1 parts Vector DNA (mol not l)
        3 part Insert DNA (mol not l)
        20 µl Nuclease free water
        1 µl T4 DNA Ligase
 
    Steps:
 
Set up the reaction in a microcentrifuge tube on ice. T4 DNA Ligase should be added last. The molar ratio of vector to insert should be 1:3.
 
        Gently mix the reaction by pipetting up and down and microfuge briefly.
 
        For cohesive (sticky) ends, incubate at 16°C overnight or room temperature for 10 minutes.
 
For blunt ends or single base overhangs, incubate at 16°C overnight or room temperature for 2 hours. Alternatively a high concentration of T4 Ligase can be used in a 10 minute ligation.
 
        Heat inactivate at 80°C for 10 minutes
 
        Chill on ice and transform 1-5 µl of the reaction into 50 µl of competent cells.
 
 
Notes:
 
        Used fragments/chemicals:
 
        1 µl cutted pSB1C3
        0,1 µl MCS
        0,1 µl pCat
        2 µl T4 Buffer
        1 µl T4 Ligase
        Filled up to 20 µl with water
 
        Incubated for 2 hours at room temperature
 
Results:
 
Plated transformated bacteria (with ligation product) did not grew on a LB-Agar with CM
▼2015-06-23  PCR: Site-change of ribozyme constructs
Description:
 
Aim: Get every fragment to the same cloning standart.
BamHI -----------------------------/ BmtI
 
Procedures:
 
23.06.2015:
 
Description
 
        PCR mix:
 
        Polymerase Mastermix 2x: 25 µl
        Primer:        fwd: 1 µl
                               rev 1 µl
        Template-DNA 1 µl
        ddH20 22 µl
 
Number
Fragment Name
T Anneal
Fwd
Rev
1
CFTR 1 T
58°C
DH13
DH14
2
CFTR 1 A
58°C
DH13
DH14
3
CFTR 1 C
58°C
DH13
DH14
4
CFTR 2 A
61°C
DH13
DH15
5
CFTR 2 C
61°C
DH13
DH15
6
GFP 1
61°C
DH16
DH17
 
 
Results:
 
1: 1 band + smear: PCR settings were not right --> many sideproducts were synthesized
2: 2 bands + smear: As above, lower band is our desired product
3: 2 bands: Higher product yield than 1 and 2, higher purity than 1 and 2
4: One small band: Low yield and higher purity than 3
5: One small band: Lower yield than 4
6: Smear: No visible product band
 
The PCR settings were wrong for our target DNA.
 
Figure 2: Site change of ribozyme constructs: First PCR
Positive colonies exhibit a clear band with a length of about 200 bp. Bigger products seem to be side products
 
24.06.2015:
 
Description:
 
        PCR-Mix:
        25 µl Polymerase Master Mix 2x
        1 µl Primer fwd
        1 µl Primer rev
        1 µl Template-DNA
        22 µl ddH2O
 
Number
Template
T Anneal
Fwd
Rev
C(ng/µl)
Comment
1
CFTR 1 T
60°C
DH13
DH14
24
Number 1 + 1a in one tube
2
CFTR 1 A
60°C
DH13
DH14
30,5
Number 2 + 2a in one tube
3
CFTR 1 C
60°C
DH13
DH14
25
Number 3 + 3a in one tube
4
CFTR 2 A
60°C
DH13
DH15
8,1
 
5
CFTR 2 C
60°C
DH13
DH15
8,2
 
6
GFP 1
60°C
DH16
Dh17
-0,6
 
1a
CFTR 1 T
57°C
DH13
DH14
24
 
2a
CFTR 1 A
57°C
DH13
DH14
30,5
 
3a
CFTR 1 C
57°C
DH13
DH14
25
 
4a
CFTR 2 A
57°C
DH13
DH15
7,2
 
5a
CFTR 2 C
57°C
DH13
DH15
 
 
6a
GFP 1
57°C
DH16
DH17
 
 
 
 
Results:
1/2 = 1/1a
3/4 = 2/2a
5/6 = 3/3a
7/8 = 4/4a
9/10 = 5/5a
10/11 = 6/6a
 
 
1/2: 60°C is the optimal temperature for our PCR
3/4: 60°C and 57°C yield equal amounts of DNA
5/6: 60°C and 57°C yield equal amounts of DNA
7/8: 60°C yields no DNA, 57°C yields less DNA than 1/2
9/10: 60°C yields no DNA, 57°C yields less DNA than 7/8
11/12: Smear, no band at 60°C and 57°C
 
The Annealing Temperatures for most products are help to get a high yield. Just for 9/10 and 11/12 the yields are not optimal.
 
Figure 3: Site change of ribozyme constructs: First PCR
Positive colonies exhibit a clear band with a length of about 200 bp. Bigger products seem to be side products.
 
25.06.2015:
 
Description:
 
PCR-Mix:
 
Like in the former protocols
 
Number
Template
T Anneal
Fwd
Rev
5.1
CFTR 2 C
55°C
DH13
DH15
5.2
CFTR 2 C
53°C
DH13
DH15
6.1
GFP 1
55°C
DH16
DH17
6.2
GFP 1
53°C
DH16
DH17
 
 
▼2015-06-23  Digest of pSB1C3 BBa_J04450
Description:
 
Procedure:
 
20 µl test-digest:
 
Description
 
    Steps:
 
Set up reaction according to protocol:
                ddH2O for a final volume of 20 µl
                2 µl of 10x Reaction Buffer (e.g. NEB CutSmart)
                0.5 µl of selected Enzyme(s)
                ca. 1 µl of mini prep DNA (Range 200-1000 ng)
 
Incubate at 37°C for 60'
 
Load on gel (add loading dye first)
 
Notes:
 
        Selected Enzymes:
        EcoRI/SpeI
 
        Buffer:
        Cutsmart
 
        Heat inactivation at 60°C
 
        Everything was given on the gel
 
        A fragment with about 2 kb was cut out
 
        Gel elution: Result: 50 ng/µl DNA
 
Results:
 
50 ng/µl DNA
 
5:
Title: Yeast Transformation with p413-GPD and CFTR construct 2
Author: Hendrik
Date: 23.06.2015
 
Description:
 
Procedures:
 
 
Yeast transformation:
 
 
Description:
 
                10 µl of cells for transformation with a plasmid, 50 µl of cells for transformation with a PCR                       product
                2 µl of plasmid DNA per 10 µl of cells
                6 equivalents of PEG
                1/9 equivalents of DMSO
                100 - 200 µl of liquid medium
   
                Steps:
 
Give the plasmid DNA into a Eppi. Add the competent cells.
Mix, then add the PEG.
Incubate for 30 mins at room temperature while mixing
Add the DMSO
Place the yeast in a 42°C water bath for 5-20 minutes
Centrifuge cells for 2-3 minutes at 2000 rpm
Discard the supernatant and resuspend the yeast in the liqiud YPD medium
 
Notes:
 
        10 µl of Yeast and 100 µl of SD-His medium was taken.
        2 transformations were made
        After transformation the yeast was plated.
 
Results:
 
Some yeast grew on the first plate, but not on the second plate (I took some of the original biofilm and made a fractionated plating). Also the biofilm on the first plate exhibited no further growth. Maybe the wrong yeast medium was token (The writing on the flasks didn't survive the autoclave). Another plating was been made to see if this is right. The yeast on the SD-His and SD-Leu plate exhibited no growth. The yeast in the SD-His/Leu plate grew spotlike and maybe forms colonies.
▼2015-06-23  Yeast Transformation with p413-GPD and CFTR construct 2
Description:
 
Procedures:
 
 
Yeast transformation:
 
 
Description:
 
                10 µl of cells for transformation with a plasmid, 50 µl of cells for transformation with a PCR                       product
                2 µl of plasmid DNA per 10 µl of cells
                6 equivalents of PEG
                1/9 equivalents of DMSO
                100 - 200 µl of liquid medium
   
                Steps:
 
Give the plasmid DNA into a Eppi. Add the competent cells.
Mix, then add the PEG.
Incubate for 30 mins at room temperature while mixing
Add the DMSO
Place the yeast in a 42°C water bath for 5-20 minutes
Centrifuge cells for 2-3 minutes at 2000 rpm
Discard the supernatant and resuspend the yeast in the liqiud YPD medium
 
Notes:
 
        10 µl of Yeast and 100 µl of SD-His medium was taken.
        2 transformations were made
        After transformation the yeast was plated.
 
Results:
 
Some yeast grew on the first plate, but not on the second plate (I took some of the original biofilm and made a fractionated plating). Also the biofilm on the first plate exhibited no further growth. Maybe the wrong yeast medium was token (The writing on the flasks didn't survive the autoclave). Another plating was been made to see if this is right. The yeast on the SD-His and SD-Leu plate exhibited no growth. The yeast in the SD-His/Leu plate grew spotlike and maybe forms colonies.
▼2015-06-27  Test if transformed cells have the MCS + pcat insert
Description:
 
Procedures were done on 5 ml overnight E.coli culture with estimated MCS+pCat insert in pSB1C3
Five colonies were picked from the original plate and were given in 5 snapcaps with 5 ml LB medium each.
The 5 cultures are named 1-5
 
Procedures:
 
QIAprep Spin Miniprep Kit:
 
Description:
 
    Steps:
 
Prepare o/n culture
 
Perform mini prep according to manufacturers protocol
 
Notes:
 
After the miniprep 2 ml of overnight culture were transferred to 100 ml of fresh LB medium with 1:1000 Chloramphenicol
 
Results:
 
Nanodrop results:
1: c = 9,5 ng/µl
2: c = 7,7 ng/µl
3: c = 81,5 ng/µl
4: c = 48,5 ng/µl
5: c = 64 ng/µl
 
A test digestion will be made with 3-5. The concentrations for 1-2 are too low.
 
20 µl test-digest:
 
Description
 
    Steps:
 
Set up reaction according to protocol:
                ddH2O for a final volume of 20 µl
                2 µl of 10x Reaction Buffer (e.g. NEB CutSmart)
                0.5 µl of selected Enzyme(s)
                ca. 1 µl of mini prep DNA (Range 200-1000 ng)
 
Incubate at 37°C for 60'
 
Load on gel (add loading dye first)
 
Notes:
 
                Digest with EcoRI and SpeI
                Heat inactivation at 65°C for 20 minutes
                The gel picture should have one band when the plasmid have no insert (the natural plasmid   has no SpeI cutting site) and two bands when the insert is in the plasmid (the MCS has one              cutting site for SpeI).
 
Results:
 
The agarose gel shows, that none of our colonies has got the MCS + pCat fragment.
 
Figure 4: Digestion of pSB1C3 to test if MCS and pcat are inserted
The target fragment exhibits 2 bands, the non-cut fragment exhibits just one band. The reason for the larger bands (2 and 4) is unknown.
 
▼2015-06-27  Transformation test for pSB1C3 + MCS and pcat
Description:
 
Procedures were done on 5 ml overnight E.coli culture with estimated MCS+pCat insert in pSB1C3
Five colonies were picked from the original plate and were given in 5 snapcaps with 5 ml LB medium each.
The 5 cultures are named 1-5
 
Procedures:
 
QIAprep Spin Miniprep Kit:
 
Description:
 
    Steps:
 
Prepare o/n culture
 
Perform mini prep according to manufacturers protocol
 
Notes:
 
After the miniprep 2 ml of overnight culture were transferred to 100 ml of fresh LB medium with 1:1000 Chloramphenicol
 
Results:
 
Nanodrop results:
1: c = 9,5 ng/µl
2: c = 7,7 ng/µl
3: c = 81,5 ng/µl
4: c = 48,5 ng/µl
5: c = 64 ng/µl
 
A test digestion will be made with 3-5. The concentrations for 1-2 are too low.
 
20 µl test-digest:
 
Description
 
    Steps:
 
Set up reaction according to protocol:
                ddH2O for a final volume of 20 µl
                2 µl of 10x Reaction Buffer (e.g. NEB CutSmart)
                0.5 µl of selected Enzyme(s)
                ca. 1 µl of mini prep DNA (Range 200-1000 ng)
 
Incubate at 37°C for 60'
 
Load on gel (add loading dye first)
 
Notes:
 
                Digest with EcoRI and SpeI
                Heat inactivation at 65°C for 20 minutes
                The gel picture should have one band when the plasmid have no insert (the natural plasmid   has no SpeI cutting site) and two bands when the insert is in the plasmid (the MCS has one              cutting site for SpeI).
 
Results:
 
The agarose gel shows, that none of our colonies has got the MCS + pCat fragment.
week number 25
▼2015-06-19  Quantitative Gel of the PCR of the Gibson assembly fragments
Description:
We performed a gel quantification of DNA yielded in the PCR of our Gibson fragments.
 
Procedures:
 
Quantitative DNA gel:
 
Description:
 
Steps:
Prepare an evenly stained agarose gel.
Load a defined amount of the sample
Load the DNA Ladder according to manufacturers protocol (e.g. NEB 2-log: 10µl for ng values according to sheet)
Document and analyze the gel
 
Notes:
 
DNA samples were prepared by 1 hour of digestion with Dpn1 at 60 °C, followed by purification using a qiagen kit.
Samples were mixed with 3 parts of 6x loading dye, and loaded together with 2 log ladder.
Samples were: 1/2: PCR product of mCherry
3/4: CFTR testing construct
5/6: GFP
7/8: p413 backbone
Samples were yielded from PCR with Phusion polymerase.
 
Results summary:
Lane
Lane 1
Lane 2
Lane 3
Lane 4
Lane 5
Lane 6
DNA
2-log
mCherry
CFTR
GFP
P413
2-log
Basepairs
 
~800
200-300
~800
~6000
 
DNA Yield
 
~60 ng/µl
~60 ng/µl
~60 ng/µl
~60 ng/µl
 
 
Conclusion:
The bright bands yielded in this gel, in comparison to the previous quantitative gel of the PCR with OneTaq are indicative of a higher suitability of Phusion polymerase for constructs of this composition. Especially backbone p413 yielded a bright band under UV light in our present experiment, whereas in the previous experiment, there were no bands at all. This indicates, that we should choose Phusion for the amplification of larger fragments.
In the following quantification, all the bands yielded a concentration of about 60 ng/µl, fairly sufficient for a following Gibson Assembly.
▼2015-06-21  Colony PCR of E.coli transformated with Gibson-Plasmids
Procedures:
 
PCR:
 
Description:
 
        Materials and Chemicals:
 
        PCR-tubes
        Forward primer
        Reverse primer
        Mastermix (dNTPs, Polymerase, buffer)
        Water
        Thermocycler
 
        Endvolume: 10 µl
 
Steps:
Pick colonies from plates. Solute one colony in about 20 µl of water.
Give concentrated mastermix (which should be diluted to 1x in the end (e.g. you need 5 µl of 2x mastermix for 10 µl)), primer (concentration between and µl). Fill the volume to 10 µl with the water with E.coli
Use the Thermocycler with the usual PCR program for at least 25 cycles.
If the result is positive, take the remaining water to seed LB-medium with our E.coli
 
Notes:
 
        Plasmids: p413-GPD
        Insert: mCherry-CFTR-Tags-GFP
 
        Chemicals(one tube):
        0,4 µl forward Primer
        0,4 µl reverse Primer
        5 µl Mastermix
        4,2 µl water with E.coli
 
Results:
 
Culture 10 seems to be the only picked culture wich exhibits an insert
 
Figure 1: Colony PCR of E.coli transformated with Gibson-Plasmids
Positive Colonies exhibit an insert with the CFTR-testconstruct that is about 1,5 kb long.
