Team:Heidelberg/notebook/rd
Achievements
Results
Medal fulfillments
AptaBodies
Real time S.M.S.
Cystic fibrosis
Labeling and vizualization
Modeling and Software validation
Standardization
Project
Overview
AptaBodies
Real time S.M.S.
Cystic fibrosis
Labelling and Visualization
HRP based detection
Software
MAWS
JAWS
Wiki Tools
Notebook
AptaBodies
Real time S.M.S.
Cystic fibrosis
Labeling and vizualization
Pulse chase
Materials & Methods
Safety
Parts
Team
Members
Supervisors
Collaboration
Attributions
Sponsors
Modeling
Aptamer Sensors
In vitro transcription
Outreach
iGEM Team Heidelberg 2015
Achievements
Results
Project
Software
Notebook
Parts
Team
Modeling
Outreach
week number 26
▼2015-06-25  PCR of the Fragments E, B-Hammerhead Ribozyme (Hammerhead ribozyme: HHR), EGS-Hammerhead Ribozyme and M1
Aim: Amplification of the DNA-primers for the in vitro-transcription
 
Procedure:
Make a 1:10 thinner of each primer by using each 9 µL water and 1 µL primer
 
 
Fragment primer
Forward Primer
Reverse Primer
Lenght with out T7 promotor [bp]
Fragment E
FS01, FS02
FS 13
FS 14
66
Fragment B
FS04, FS05
FS 13
FS 15
54
Fragment EGS
FS10, FS
FS 13
FS 16
65
Fragment M1
genomic DNA
FS 08
FS 09
 
 
Fill up the following ingredients in a PCR-tube and mix them well:
1 µL of 1 µM fragment  
19 µL of water
2,5 µL of 10 µM reverse primer
2,5 µL of 10 µM foward primer
25 µL of 2x PCR Mastermix Phuion Flash
Put the samples into a PCR-cycler (cycle is discribed below):
98 °C for 2:30 min (denaturation)
98 °C for 0:15 min
55 °C for 0:15 min (annealing)
72 °C for 0:20 min (polymerisation)
Go to step 2 and repeat for 39x
72 °C for 2:00 min
4 °C for cool down
Time total: 1 h 20 min 54 sec
Analyze the PCR-products by using an 1 % agarose gel with ethidumbromid:
1 % agarose
1 x TAE
20 µL/L Ethidumbromid for gel
Put 5 µL of each PCR-product which was mixed with 2 µL Loading Dye in a pocket
Fill 5 µL of a 2 log Ladder in the last pocket
Let the gel run for 30 min at 120 V.
Analyze the gel by using UV-light.
 
Conclusion:
The PCR has worked, therefore the fragments are amplified
Furthermore the primers (forward and reverse) were designed right since there is no visible ladder for them
▼2015-06-26  Precipitation of the PCR-DNA
Aim: Cleaning the DNA as a prepartional step for the in vitro transcripton
 
Procedure:
Mix each PRC-Product with 0.1x V of a 3 M NaAc and 2.5x V of absolute ethanol
Vortex the mixture well
Put the samples in the freezer for overnight
On the next day, spin the DNA-Ethanol mixture at 4 °C with 13000 rpm for 30 min in a centrifuge
Remove the ethanol from the pellet and rehydrate the pellet with 1 ml of 70 % absolute ethanol.
Vortex again.
Repeat the steps 4 to 6 for one more time and for the last cylce just 4
Remove the ethanol from the pellet and rehydrate the pellet with 100 µL millipore water.
Vortex the sample again.
▼2015-06-27  in vitro-transcription of the fragments E, B-HHR, EGS-HHR and M1 out of the DNA that was generated out of the PCR form the 25th June 2015
Aim: transcribe the PCR-samples into RNA to have the single components for the construct
Procedure for 100 µL transcription:
Mix together
4 µL of 100 mM each NTP (ATP, GTP, CTP and UTP)
10 µL of 10 x transcription buffer
1 µL of 1 M DTT
5 % of DMSO
5 µL of T7 ( 2 mg/mL)
10 µL of the DNA-template
and 53 µL millipore water
Use a pipett to mix it up and down (do not vortex)
Put the samples into a heatblock at 37 °C for 3 hours
Add  to the mixture 5 µL of T7 polymerase and 2 µL of 1 M MgCl2 addtionally after 2 hours
Add 4 µL DNase I to the mixture and let it incubate at 37 °C for 20 min.
▼2015-06-27  Quantative Gel for RNA-Extraction
Aim: Purifying the RNA that was transcribed in the previous in vitro assay
Procedure:
Make a 8 M urea polyacrylamid gel with  Roth© mastermix, 400 µL APS and 40 µL TEMED.
Mix the sample with 2x Loading Dye to get a 1x Loading Dye concentration
Clean the pockets with TBA buffer
Load the entire in vitro-Transcript with Loading Dye in a pocket
Fill TBA buffer in all reservoirs of the gel system
Run the gel at 300 V for 3 hours
Analyze the gel through UV-Shadowing by using a TLC-plate underneath your gel
 
week number 27
▼2015-07-02  Producing fragment A by using M1 cleavage
Aim: Making a proof of principle that the M1 can cleave fragments with an external guiding strand
Procedure:
Measuring the transcribed RNA by using a NanoVue at 260 nm
Mix 5 µL of 5x buffer for fragment A, 9 µl of a 7.13 µM solution of E (RNA-Stock) and 11 µL of a 5,7 µL EGS-RNA solution
Mix 5 µL of 5x buffer, 17 µL of 1.87 µM M1-RNA solution and 3 µL millipore water
Heat up each of the mixtures to 95 °C for 5 min
Mix the
Incubate the mixture at 37 °C for 2 hours
Put the sample, E and EGS on a gel, therefore use the procedure discribed on 27th June 2015.
Conclusion:
Making A did not work because we should be able to see three different. According to the results from the UV-shadowing there is just one line.
Next step:
Optimize the assay by heating up the single fragments, thus the fragments can reach the correct secondary structure.
week number 28
▼2015-07-12  Hammerhead cleavage No. 1
Aim: Creating a functional external guiding strand and a functional B without their hammerheads
Procedure for 100 µL transcription:
Mix together
4 µL of 100 mM each NTP (ATP, GTP, CTP and UTP)
10 µL of 10 x transcription buffer
1 µL of 1 M DTT
5 % of DMSO
5 µL of T7 ( 2 mg/mL)
10 µL of the DNA-template
and 53 µL millipore water
Use a pipett to mix it up and down (do not vortex)
Put the samples into a heatblock at 37 °C for 3 hours
Add  to the mixture 5 µL of T7 polymerase and 2 µL of 1 M MgCl2 addtionally after 2 hours
Add 4 µL DNase I to the mixture and let it incubate at 37 °C for 20 min.
Increase the tempeature to 95 °C and heat the samples up to 95 °C
Once it has hit 95 °C take the sample out and let them cool down to room temperature for 1 hour
 
In the meanwhile one can prepare the analytical gel:
Make a 8 M urea  20 % polyacrylamid gel  with  Roth© mastermix, 400 µL APS and 40 µL TEMED.
Mix the sample with 2x Loading Dye to equal a conentration of 1x Loading Dye
Clean the pockets with TBA buffer
Load the entire in vitro-Transcript with Loading Dye in a pocket
Fill TBA buffer in all reservoirs of the gel system
Run the gel at 20 W for 4 hours
Analyze the gel through UV-Shadowing by using a TLC-plate underneath your gel
Result:
The cleavage of B is successfully performed. EGS did not work. Therfore we will try it again. This time with two heatshocks.
week number 29
▼2015-07-15  Hammerhead cleavage of the transcripts from the 12th July 2015: No. 2
Aim: Showing that the EGS can cleave
Procedure:
Use the transcribed RNA from the 12th of July and heat up it up from 18 °C to 95 °C
Cool them down on ice for 1 hour
Put them back into a Thermoblock and heat it up form 10 °C to 95 °C
Cool them down on ice for 1 hour
 
In the meanwhile one can prepare the analytical gel:
Make a 8 M urea 20% polyacrylamid gel  with  Roth© mastermix, 400 µL APS and 40 µL TEMED.
Mix the sample with 10 µL 2x Loading Dye to get a 1x Loading Dye concentration
Clean the pockets with TBA buffer
Load the entire in vitro-Transcript with Loading Dye in a pocket
Fill TBA buffer in all reservoirs of the gel system
Run the gel at 20 W for 4 hours
Analyze the gel through UV-Shadowing by using a TLC-plate underneath your gel
Results:
For the second time the cleavage of the EGS did not work causing us to design a new external guiding strand that is able to cleave its hammerhead ribozyme.
week number 30
▼2015-07-23  PCR of the Fragments that are necessary for the Spinach II: proof of principle fragments
Aim: Amplification of the DNA-Spinach-primers for the in vitro-transcription
PCR III (Adavanced Style)/ Procedure (100 µL PCR):
 
 
Fragments
Forward Primer
Reverse Primer
Lenght with T7 promotor [bp]
Spinach II
MS01, MS02
MS08
MS09
95
Spinach II with Theophylline Aptamer
MS01, MS05
MS08
MS09
82
Spinach II with Theophyliine Aptamer Mutant
MS01, MS06
MS08
MS09
82
 
Fill up the following ingredients in a PCR-tube and mix them well:
41 µL of water
50 µL of 2x PCR Mastermix Phuion Flash
1 µL of 100 µM foward primer
1 µL of 100 µM reverse primer
1 µL of 100 µM fragment
5 µL of DMSO
Put the samples into a PCR-cycler (cycle is discribed below):
95 °C for 4 min (denaturation)
95 °C for 0:30 min
62 °C for 0:30 min (annealing)
72 °C for 1 min (polymerisation)
Go to step 2 and repeat for 39x
72 °C for 5:00 min
4 °C for cool down
Analyze the PCR-products by using an 1 % agarose gel with ethidumbromid:
1 % agarose
1 x TAE
20 µL/L Syber green
Put 5 µL of each PCR-product which was mixed with 2 µL Loading Dye in a pocket
Fill 5 µL of a 2 log Ladder in the last pocket
Let the gel run for 30 min at 120 V.
Analyze the gel by using UV-light.
 
Conclusion:
The gel shows that we were able to amplify the primers and to receive DNA-fragments for the spinach variations.
▼2015-07-24  Extension PCR to obtain dsDNA of Substrates for 7-18 and 10-23 DNAzyme for in vitro transcription
For the DNAzymes 7-18 and 10-23 the overlapping oligos were purchased at Sigma Aldrich, Deisenhofen, Germany. For 7-18 DNAzyme FS022 5' GGGCTAATACGACTCACTATAGGTTTTTTTTTGACTGCCGTAGGTTGCCCG 3' and FS023 5' CGGCCGCTGCTCTACCGACTGAGCTATCCGGGCGGGCAACCTACGGCAGTCAAAAAAAAACC 3' and for 10-23 DNAzyme FS028 5' GGGCTAATACGACTCACTATAGGTTTTTTTTTGACTGCCGTAGGTTGCCCG 3' and FS029 5' CGGCCGCTGCTCTACCGACTGAGCTATCCGGGCGGGCAACCTACGGCAGTCAAAAAAAAACC 3' were anealed and extended under PCR conditions (see table).
 
 
cStock
cFinal
V[µL]
Primer 1
100 mM
2 µM
2
Primer 2
100 mM
2 µM
2
DMSO
 
5 %
5
Phusion Flash Master mix
2 x
1 x
50
H2O
 
ad 100 µL
41
Final
 
 
100
PCR settings:
 
Temperture [°C]
Time
Cycles
Initial denaturation
95
4 min
1
Denaturation
95
30 s
39
Annealing
62
30 s
Elongation
72
1 min
Final elongation
72
5 min
1
Hold
4
∞
1
Size of the samples were checked on a 1 % agarose in TAE gel and visualized with Ethidiumbromide (EtBr) under UV light (see gel picture).
 
Results:
DNA is used for in vitro transcription as next steps.
Comment: Both constructs are actually the same as the substrate sequence Wang and coworkers, 2002 used is the same for both DNAzymes.
week number 31
▼2015-07-27  In vitro transcription of substrate RNA
All solutions were mixed as indicated in the table:
 
cStock
cFinal
V[µL]
Transcription buffer
10 x
1 x
10
ATP
100 mM
4 mM
4
GTP
100 mM
4 mM
4
CTP
100 mM
4 mM
4
UTP
100 mM
4 mM
4
DTT
1 M
10 mM
1
DMSO
 
5 %
5
DNA Template (PCR 2015-07-24)
 
 
10
T7 RNA Polymerase
2 mg/mL
0,1 mg/mL
5
H2O
 
ad 100 µL
53
Final
 
 
100
 
Reaction was incubated for 3 h at 37 °C
After 1.5 h another 2 µL T7 RNA Polymerase were added
Addition of 2 µL DNase I and further incubation at 37 °C for 20 min
▼2015-07-28  Substrate RNA purification by precipitation
Sample was mixed with 100 µL of 2 x loading dye and purified over a 10 % PAGE
Bands were visualized by UV shadowing and suitable bands were excised
RNA was eluted out of the gel using 0.3 M NaAc pH 5.5 in three elution steps
Gel parts were filtered of and 2.5 volumes of -20 °C EtOH were added, sample was stored at -20 °C oN to let the RNA precipitate
Spin sample at 16,000 g for 30 min, discard the supernatant
Washed the pellet twice with 70 % EtOH and dissolved RNA in 20 µL of MQ water
▼2015-07-29  Modification of HRP with N3
Sequence of HRP ~44 kDa, Sigma Aldrich:
>tr|K7ZW28|K7ZW28_ARMRU Peroxidase OS=Armoracia rusticana GN=HRP_A2A PE=3 SV=1
MAVTNLSTTCDGLFIISLLVIVSSLFGTSSAQLNATFYSGTCPNASAIVRSTIQQAFQSD
TRIGASLIRLHFHDCFVNGCDASILLDDSGSIQSEKNAGPNANSARGFNVVDNIKTALEN
TCPGVVSCSDILALASEASVSLTGGPSWTVLLGRRDSLTANLAGANSAIPSPFEGLSNIT
SKFSAVGLNTNDLVALSGAHTFGRARCGVFNNRLFNFSGTGNPDPTLNSTLLSSLQQLCP
QNGSASTITNLDLSTPDAFDNNYFANLQSNNGLLQSDQELFSTTGSATIAVVTSFASNQT
LFFQAFAQSMINMGNISPLTGSNGEIRLDCKKVNGS
 
All compounds were mixed as indicated in the table. Reaction was incubated at first for a couple of hours at 4 °C and then over night at 18 °C. Afterwards modified HRP was washed by the addition of MQ water and subsequent spinning at 2095 x g through an Amicon UltraFree-MC centrifugal filter with a 10 kDa cutoff. The flow through was discarded and the steps were repeated till a total volume of 30 mL water was used for washing. The sample was concentrated to a volume of ~270 µL as last step. Then it was stored at -20 °C.
 
cStock
V[µL]
HRP ~44 kDa
10 mg/mL
40
H2O
 
160
K2CO3
2 mg/mL
100
Cu(II)SO4
1 mg/mL
25
Imidazol-sulfonyl-azide (209,61 g/mol)
2 mg/mL
15
Final
 
340
Conditions as Stiin, 2008.
All compounds were mixed as indicated in the table. Reaction was incubated at first for a couple of hours at 4 °C and then over night at 18 °C. Afterwards modified HRP was washed by the addition of MQ water and subsequent spinning at 2095 x g through an Amicon UltraFree-MC centrifugal filter with a 10 kDa cutoff. The flow through was discarded and the steps were repeated till a total volume of 30 mL water was used for washing. The sample was concentrated to a volume of ~270 µL as last step. Then it was stored at -20 °C.
Results:
Modification has to be proved.
When modification is confirmed N3-HRP can be used for click chemistry reaction.
▼2015-07-30  In vitro transcription of labeling RNA
Annealing of oligos:
Two complementary primers were purchased at Sigma Aldrich:
FS021: 5‘ GGGCTAATACGACTCACTATAGGCGGCCGCGGGTCCAGGGTTCAAGTCCCTGTTCGGGCGCCA 3’
FS024: 5’ TGGCGCCCGAACAGGGACTTGAACCCTGGACCCGCGGCCGCCTATAGTGAGTCGTATTAGCCC 3’
They were annealed by mixing equal amount of both and heating it upto 95 °C for 5 min and letting it cool to rt.
In vitro Transcription:
All solutions were mixed as indicated in the table:
 
cStock
cFinal
V[µL]
Transcription buffer
10 x
1 x
10
ATP
100 mM
4 mM
4
GTP
100 mM
4 mM
4
CTP
100 mM
4 mM
4
UTP
100 mM
4 mM
4
DTT
1 M
10 mM
1
DMSO
 
5 %
5
DNA Template (PCR 2015-07-24)
 
 
10
T7 RNA Polymerase
2 mg/mL
0,1 mg/mL
5
H2O
 
ad 100 µL
53
Final
 
 
100
 
Reaction was incubated for 3 h at 37 °C
After 1.5 h another 2 µL T7 RNA Polymerase were added
Addition of 2 µL DNase I and further incubation at 37 °C for 20 min
 
Outlook:
As next step sample is purified over 10 % PAGE
 
▼2015-07-30  DNazyme Activity Assay
Substrate and DNAzyme was incubated together as follows:
DNAzyme:
10-23 DNAzyme: FS032, purchased at Sigma Aldrich
7-18 DNAzyme: FS 033, purchased at Sigma Aldrich
DNAzymes were designed according to Wang, 2002.
 
 
cStock
cFinal
V[µL]
Tris HCl ph 7.5
1 M
50 mM
1.25
DNAzyme
10 µM
5 µM
12.5
Substrate
10 nM
2 nM
5
NaCl
1 M
100 mM
2.5
MgCl2
1 M
20 mM
0.5
SDS
20 %
0,01 %
1,25
H2O
 
ad 25 µL
2
Final
 
 
25
 
All compounds were mixed except the substrate. The samples were heated up to 95 °C for 5 min and cooled on ice
Substrate was added and the reaction was incubated at 37 °C for 1.5 h
Reaction was stopped by storing at -20 °C
Samples were separated on a 20 % denaturing PAGE, samples were thawed just before loading on the gel and mixed with appropriate volume of 2x loading dye
Gel was stained with SYBR Gold and imaged with a Typhoon Scanner, GE
Results:
Amount of RNA substrate was too low to be detected by SYBR Gold. Amount of DNAzyme was too high.
Assay will be repeated with a DNase I digestion step after the reaction, to get rid of the sDNA and get a better signal from the substrate.
▼2015-08-01  Labeling RNA purification by precipitation
Sample was mixed with 100 µL of 2 x loading dye and purified over a 10 % PAGE
Bands were visualized by UV shadowing and suitable bands were excised
RNA was eluted out of the gel using 0.3 M NaAc pH 5.5 in three elution steps
Gel parts were filtered of and 2.5 volumes of -20 °C EtOH were added, sample was stored at -20 °C oN to let the RNA precipitate
Spin sample at 16,000 g for 30 min, discard the supernatant
Washed the pellet twice with 70 % EtOH and dissolved RNA in 20 µL of MQ water
week number 32
▼2015-08-04  DNAzyme Activity Assay with DNase I digest and precipitation
Substrate and DNAzyme was incubated together as follows:
DNAzyme:
10-23 DNAzyme: FS032, purchased at Sigma Aldrich
7-18 DNAzyme: FS 033, purchased at Sigma Aldrich
DNAzymes were designed according to Wang, 2002.
 
 
cStock
cFinal
V[µL]
Tris HCl ph 7.5
1 M
50 mM
1.25
DNAzyme
10 µM
5 µM
12.5
Substrate
10 nM
2 nM
5
NaCl
1 M
100 mM
2.5
MgCl2
1 M
20 mM
0.5
SDS
20 %
0,01 %
1,25
H2O
 
ad 25 µL
2
Final
 
 
25
 
All compounds were mixed except the substrate. The samples were heated up to 90 °C for 5 min and cooled on ice
Substrate was added and the reaction was incubated at 37 °C for 1.5 h
2 µL DNase I were added and everything was further incubated at 37 °C for 30 min
Afterwards 2.5 µL of 3 M NaAc pH 5.5 were added to a final concentration of 0.3 M and 75 µL ice cold EtOH, sample was stored at -20 °C oN to precipitate the nucleic acid
Sample was centrifuged for 30 min at 14,000 g to pellet the precipitate, pellet was washed with 70 % EtOH twice
Afterwards it was dried and dissolved in 10 µL MQ H2O and mixed with an appropriate volume of 2x loading dye
Samples were separated on a 20 % denaturing PAGE, stained with SYBR Gold and imaged on a Typhoon Scanner, GE
Results:
Amount of DNAzyme was reduced but did not disappear. Amount of RNA was still under the detection limit.
▼2015-08-06  MALDI-TOF-MS of N3 modified HRP
The purified and modified HRP was spotted in a 1:4 ratio with sinapinic acid matrix on an Opti-TOF 384 well MALDI plateand measured on an AB Sciex TOF/TOF 5800 system in linear mode. The mass of the protein could be found but there was no significant shift when comparing modified and unmodified protein.
Outlook:
Samples will be check for modification using an alkyne modified fluorophore and separated on a SDS-PAGE 
▼2015-08-08  N3 HRP click test with FAM-Alkyne
N3 modified HRP was incubated with alkyne FAM:
 
cStock
cFinal
V[µL]
Phosphate buffer pH 7.0
100 mM
50 mM
50
FAM-Alkyne
1 mM
50 µM
5
N3 HRP
1 mM/ 100 µM/ 10 µM/ 1 µM
50 µM/ 5 µM/ 500 nM/ 50nM
5
Cu(II)SO4
20 mM
100 µM
0,5
THPTA
50 mM
500 µM
1
Sodium ascorbate
100 mM
1 mM
1
H2O
 
Ad 100
37,5
Final
 
 
100
Conditions as Winz, 2012.
As negative control N3 HRP w/o alkyne FAM was incubated and also a sample of alkyne FAM w/o N3 HRP was loaded onto the gel.
Samples were analyzed on an 14 % SDS-PAGE
Gel was scanned an Typhoon Scanner using FAM mode
Staining with silver nitrate stain according to http://proteomics.rockefeller.edu/ms_silverStaining
Results:
N3 modification reaction was probably not that efficient. Only a very faint band could be visualized with fluorescence scan at the correct size. The amount of protein was under the detection limit of silver nitrate staining.
N3 activation has to be repeated.
▼2015-08-09  DNAzyme Activity with inceased amount of substrate
Substrate and DNAzyme was incubated together as follows:
DNAzyme:
10-23 DNAzyme: FS032, purchased at Sigma Aldrich
7-18 DNAzyme: FS 033, purchased at Sigma Aldrich
Control: All compounds but substrate, all compound but DNAzyme
t=0 the same amount of substrate as in all samples was mixed with water and loading dye
DNAzymes were designed according to Wang, 2002.
 
 
cStock
cFinal
V[µL]
Tris HCl ph 7.5
1 M
50 mM
1.25
DNAzyme
10 µM
5 µM
12.5
Substrate
1 µM/ 100 nM
200 nM/ 20 nM
5
NaCl
1 M
100 mM
2.5
MgCl2
1 M
20 mM
0.5
SDS
20 %
0,01 %
1,25
H2O
 
ad 25 µL
2
Final
 
 
25
 
All compounds were mixed except the substrate. The samples were heated up to 90 °C for 5 min and cooled on ice
Substrate was added and the reaction was incubated at 37 °C for 1.5 h
5 µL DNase I were added and everything was further incubated at 37 °C for 30 min
Samples were mixed with an appropriate volume of 2x loading dye and separated on a 20 % denaturing PAGE, stained with SYBR Gold and imaged on a Typhoon Scanner, GE
Results:
The DNAzymes cleave the substrate. The band at 62 bp of the substrate disappears for both DNAzymes and a band at ~40 bp appears a cleavage takes place.
week number 33
▼2015-08-10  In vitro Transcription of substrate and labeling RNA
In vitro Transcription:
As DNA templates the PCR (24th July 2015) and the annealed primers (30th July 2015) were used.
All solutions were mixed as indicated in the table:
 
cStock
cFinal
V[µL]
Transcription buffer
10 x
1 x
10
ATP
100 mM
4 mM
4
GTP
100 mM
4 mM
4
CTP
100 mM
4 mM
4
UTP
100 mM
4 mM
4
DTT
1 M
10 mM
1
DMSO
 
5 %
5
DNA Template (PCR 2015-07-24)
 
 
10
T7 RNA Polymerase
2 mg/mL
0,1 mg/mL
5
H2O
 
ad 100 µL
53
Final
 
 
100
 
Reaction was incubated for 3 h at 37 °C
After 1.5 h another 2 µL T7 RNA Polymerase were added
Addition of 2 µL DNase I and further incubation at 37 °C for 20 min
Substrate RNA purification by precipitation:
Sample was mixed with 100 µL of 2 x loading dye and purified over a 10 % PAGE
Bands were visualized by UV shadowing and suitable bands were excised
RNA was eluted out of the gel using 0.3 M NaAc pH 5.5 in three elution steps
Gel parts were filtered of and 2.5 volumes of -20 °C EtOH were added, sample was stored at -20 °C oN to let the RNA precipitate
Spin sample at 16,000 g for 30 min, discard the supernatant
Washed the pellet twice with 70 % EtOH and dissolved RNA in 20 µL of MQ water
 
▼2015-08-11  DNAzyme Activity Assay with candidates with ATP aptamer
Samples:
10-23 DNAzyme: FS032
7-18 DNAzyme: FS 033
10-23 DNAzyme with ATP aptamer with linker: FS019
7-18 DNAzyme with ATP aptamer with linker: FS027
10-23 DNAzyme with ATP aptamer A: FS017, B: FS018
7-18 DNAzyme with ATP aptamer A: FS025, B: FS026
t=0: 7-18 DNAzyme und equal amounts of substrate in water and loading dye
DNAzymes were designed according to Wang, 2002. All were purchased as oligos at Sigma Aldich
Substrate and DNAzyme (FS032 and FS033) was incubated together as follows:
 
cStock
cFinal
V[µL]
Tris HCl ph 7.5
1 M
50 mM
1.25
DNAzyme
10 µM
5 µM
1.25
Substrate
1 µM
200 nM
5
NaCl
1 M
100 mM
2.5
MgCl2
1 M
20 mM
0.5
SDS
20 %
0,01 %
1,25
H2O
 
ad 25 µL
2
Final
 
 
25
 
Substrate and DNAzyme (FS019 and FS027) was incubated together as follows:
 
cStock
cFinal
V[µL]
Tris HCl ph 7.5
1 M
50 mM
1.25
DNAzyme
10 µM
5 µM
1.25
Substrate
1 µM
200 nM
5
NaCl
1 M
100 mM
2.5
MgCl2
1 M
20 mM
0.5
SDS
20 %
0,01 %
1,25
ATP
100 mM
5 mM
1,25
H2O
 
ad 25 µL
12
Final
 
 
25
 
Substrate and DNAzyme (FS032 and FS033) was incubated together as follows:
 
cStock
cFinal
V[µL]
Tris HCl ph 7.5
1 M
50 mM
1.25
DNAzyme A
10 µM
5 µM
1.25
DNAzyme B
10 µM
5 µM
6.25
Substrate
1 µM
200 nM
5
NaCl
1 M
100 mM
2.5
MgCl2
1 M
20 mM
0.5
SDS
20 %
0,01 %
1,25
ATP
100 mM
5 mM
1.25
H2O
 
ad 25 µL
5.75
Final
 
 
25
All compounds were mixed except the substrate. The samples were heated up to 95 °C for 5 min and cooled on ice
Substrate was added and the reaction was heated up to 95 °C for 1 min and then incubated at 37 °C for 2 h
5 µL DNase I were added and everything was further incubated at 37 °C for ~30 min
Samples were mixed with an appropriate volume of 2x loading dye and separated on a 20 % denaturing PAGE, stained with SYBR Gold and imaged on a Typhoon Scanner, GE
Results:
The DNAzymes w/o aptamer cleaved the substrate. The band at 62 bp of the substrate disappears for both DNAzymes and a band at ~40 bp appears. For the DNAzymes with ATP aptamer no clear result could be observed.
Assay will be repeated with and w/o the presence of ATP and further controls.
▼2015-08-13  DNAzyme Activity of DNAzymes with ATP aptamer
 
Samples:
10-23 DNAzyme: FS032
7-18 DNAzyme: FS 033
10-23 DNAzyme with ATP aptamer with linker: FS019
7-18 DNAzyme with ATP aptamer with linker: FS027
10-23 DNAzyme with ATP aptamer A: FS017, B: FS018
7-18 DNAzyme with ATP aptamer A: FS025, B: FS026
DNAzymes were designed according to Wang, 2002. All were purchased as oligos at Sigma Aldich
Stock solutions and conditions:
 
cStock
cFinal
Tris HCl ph 7.5
1 M
50 mM
DNAzyme (A)
10 µM
500 nM
DNAzyme B
10 µM
500 nM
Substrate
1 µM
200 nM
NaCl
1 M
100 mM
MgCl2
1 M
20 mM
SDS
20 %
0,01 %
ATP
100 mM
5 mM
H2O
 
ad 25 µL
 
 
 
Pipetting scheme:
#
 
 
Tris HCl ph 7.5
DNAzyme A
DNAzyme B
Substrate
NaCl
MgCl2
SDS
ATP
H2O
Final
1
FS032
10-23D
2.5
2.5
0
10
5
1
2.5
2.5
24
50
2
FS033
7-18D
2.5
2.5
0
10
5
1
2.5
2.5
24
50
3
FS019
10-23DmLink
2.5
2.5
0
10
5
1
2.5
2.5
24
50
4
FS027
7-18DmLink
2.5
2.5
0
10
5
1
2.5
2.5
24
50
5
FS017+18
10-23D_A+B
2.5
2.5
12.5
10
5
1
2.5
2.5
11.5
50
6
FS025+26
7-18D_A+B
2.5
2.5
12.5
10
5
1
2.5
2.5
11.5
50
7
 
 
2.5
2.5
0
10
5
1
2.5
0
26.5
50
8
 
 
2.5
2.5
0
10
5
1
2.5
0
26.5
50
9
FS019
10-23DmLink
2.5
2.5
0
10
5
1
2.5
0
26.5
50
10
FS027
7-18DmLink
2.5
2.5
0
10
5
1
2.5
0
26.5
50
11
FS017+18
10-23D_A+B
2.5
2.5
12.5
10
5
1
2.5
0
14
50
12
FS025+26
7-18D_A+B
2.5
2.5
12.5
10
5
1
2.5
0
14
50
13
FS032
-10-23D
1.25
1.25
0
0
2.5
0.5
1.25
0
18.25
25
14
FS033
-7-18D
1.25
1.25
0
0
2.5
0.5
1.25
0
18.25
25
15
FS019
-10-23DmLink
1.25
1.25
0
0
2.5
0.5
1.25
0
18.25
25
16
FS027
-7-18DmLink
1.25
1.25
0
0
2.5
0.5
1.25
0
18.25
25
17
FS017+18
-10-23D_A+B
1.25
1.25
6.25
0
2.5
0.5
1.25
0
12
25
18
FS025+26
-7-18D_A+B
1.25
1.25
6.25
0
2.5
0.5
1.25
0
12
25
19
 
Substrate only
1.25
0
0
5
2.5
0.5
1.25
0
14.5
25
 
 
 
 
 
 
 
 
 
 
 
 
 
 
 
 
 
 
 
 
 
 
 
 
 
 
 
 
 
 
 
 
All compounds were mixed except the substrate. The samples were heated up to 95 °C for 5 min and cooled on ice
Substrate was added and the reaction was heated up to 95 °C for 1 min
Afterwards 25 µL of samples 1-12 were stored at -20 °C (t=0)
The remaining 25 µL and all other samples were incubated at 37 °C for 1.5 h
12.5 µL of samples 13-19 were taken and stored at -20 °C
5 µL and 2.5 µL DNase I were added to 1-12 and 13-19 and everything was further incubated at 37 °C for ~30 min
Samples were stored at -20 °C
Samples were mixed with an appropriate volume of 2x loading dye
25µL were loaded and separated on a 20 % denaturing PAGE
Gel was stained with SYBR Gold and imaged with a Typhoon Scanner, GE
 
Results:
Non-modified DNAzymes cleaved Substrate, while the candidates that contain an ATP aptamer do not lead to detectable cleavage. 7-18 DNAzyme (non-modified) seems to work better in the absence of ATP.
▼2015-08-15  Aussagekräftiger Titel
In vitro Transcription:
As DNA templates the PCR (24th July 2015) and the annealed primers (30th July 2015) were used.
All solutions were mixed as indicated in the table:
 
cStock
cFinal
V[µL]
Transcription buffer
10 x
1 x
20
ATP
100 mM
4 mM
8
GTP
100 mM
4 mM
8
CTP
100 mM
4 mM
8
UTP
100 mM
4 mM
8
DTT
1 M
10 mM
2
DMSO
 
5 %
10
DNA Template (PCR 2015-07-24)
 
 
20
T7 RNA Polymerase
2 mg/mL
0,1 mg/mL
5
H2O
 
ad 100 µL
106
Final
 
 
200
 
Reaction was incubated for 3 h at 37 °C
After 1.5 h another 2 µL T7 RNA Polymerase were added
Addition of 5 µL DNase I and further incubation at 37 °C for 20 min
Substrate RNA purification by precipitation:
Sample was mixed with 100 µL of 2 x loading dye and purified over a 10 % PAGE
Bands were visualized by UV shadowing and suitable bands were excised
RNA was eluted out of the gel using 0.3 M NaAc pH 5.5
Gel parts were filtered of and 2.5 volumes of -20 °C EtOH were added, sample was stored at -20 °C for 2 h to let the RNA precipitate
Spin sample at 16,000 g for 30 min, discard the supernatant
Dissolved RNA in ~50 µL of MQ water
Concentration was determined with an Nanodrop Spectrometer and the extinction coefficient (calculated with IDT Oligo Analyzer)
Substrate RNA: A260: 5.401 à 9.44 µM
Labeling RNA: A260: 10.121 à 26,45 µM
 
week number 34
▼2015-08-17  Modification of substrate RNA with biotin-NTP and labeling RNA with alkyne-NTP
Substrate RNA is labeled with biotin modified NTP using Terminale Dinucleotidyl Transferase (TdT), Thermo Fisher and labeling RNA with alkyne NTPs with Poly A Polymerase (yeast), affimetrix.
Modified NTPs, all from Jena Bioscience:
Biotin-16-dUTP
5-Ethynyl-dUTP (5-EdUTP)
C8-Alkyne-dUTP
C8-Alkyne-dCTP
 
cStock
cFinal
V[µL]
TdT buffer
10 x
1 x
2.5
Substrate RNA
37,2 µM
21 µM
14.2
Biotin-NTP
1 mM
150 µM
3.75
TdT
20,000 u/mL
80 u/mL
1
CoCl2
2.5 mM
0.25 mM
2.5
H2O
 
Ad 25 µL
1.05
Final
 
 
25
 
 
cStock
cFinal
V[µL]
PAP (yeast)
600 u/µL
24 u/µL
1
PAP buffer
5 x
1 x
5
5-EdUTP/ C8-Alkyne-dUTP/C8-Alkyne-dCTP
10 mM
400 µM
1
Labeling RNA
51.4
15 µM
3.8
H2O
 
Ad 25 µL
1.05
Final
 
 
25
 
All compounds were mixed as indicated in the tables.
Reactions were incubated for 2 h at 37 °
Samples were stored at -20 °C oN
RNA was precipitated by the addition of 0.1 volume of 3 M NaAc and 2.5 volume of ice cold EtOH
Sample was stored at -20 °C for 2 h so that the RNA can precipitate
Afterwards precipitate was pelleted by centrifugation, supernatant was discarded and pellet was dried and dissolved in 10 µL H2O
Substrate RNA: A260: 6.599 à 11.54 µM
Labeling RNA (5-EdUTP): A260: 7.123 à 18.61 µM
Labeling RNA C8-Alkyne-dUTP: A260: 6.458 à 16.87 µM
Labeling RNA C8-Alkyne-dCTP: A260: 6.378 à 16.67 µM
 
Results and Outlook:
Success of modification is checked on PAGE via a streptavidin shift for the biotin modification and via clicking Alexa 488 azide to thte alkyne modified RNA.
▼2015-08-18  Click chemistry: click alkyne modified labeling RNA to Alexa 488 azide; copper-catalyzed 1,3-dipolar azide-alkyne cycloaddition (CuAAC)
To check if the alkyne modified NTPs are incorporated the reactive alkyne is clicked to an azide activated Alexa 488 azide fluorophore under copper catalyzed 1,3-dipolar azide-alkyne cycloaddition (CuAAC). Samples are after the reaction separated on a 20 % denaturing PAGE and visualized first in the fluorescent channel for Alexa 488 and afterwards stained with SYBR Gold to visualize the RNA. Appropriate controls were performed as well.
Reaction condition can be found in the table:
 
cStock
cFinal
V[µL]
Phosphate buffer pH 7.0
100 mM
50 mM
12,5
Alexa 488 azide
10 µM
400 nM
1
Alkyne modified RNA
1 µM
200 nM
5
Cu(II)SO4
2 mM
100 µM
1.25
THPTA
5 mM
500 µM
2.5
Sodium ascorbate
10 mM
1 mM
2.5
H2O
 
Ad 25
0.25
Final
 
 
25
Conditions as Winz, 2012.
All compounds were mixed in the given order
Cu(II)SO4 and THPTA and H2O were mixed before adding to the mixture to let THPTA chelat the Cu
Lastly sodium ascorbate was added to reduce the Cu(II) to Cu(I)
Reaction was incubated for 2 h at 37 °C and afterwards put at -20 °C
Checking success on a 20 % denaturing PAGE
Samples were mixed with an appropriate amount of 2 x loading dye and separated on a 20 % denaturing PAGE
Gel was scanned with a Typhoon scanner, GE in fluorescent mode using the pre-set Alexa 488 program
Afterwards gel was stained using SYBR gold and scanned with the Typoon in SYBR Gold mode
Steptavidin-shift in 20 % PAGE to check the success of biotin modification 2015-08-18
Result and Outlook:
Labeling of RNA with alkyne was successful and CuCCA as well. More RNA will be labeled.
▼2015-08-18  Steptavidin-shift in 20 % PAGE to check the success of biotin modification 2015-08-18
Biotin modified RNA was incubated with equimolar amounts of streptavidin before it was mixed with appropriate amounts of 2 x loading dye and loaded onto a 20 % denaturing PAGE to let biotin and streptavidin bind.
Result and Outlook:
No shift was detected. Modification reaction is repeated with more Co2+ in the reaction buffer to increase the tailing efficiency of TdT
▼2015-08-21  KW 34 – Biotin labelled substrate RNA with terminal deoxynucleotidyl transferase
For 25µL
Cstock
Cfinal
V [µL]
TdT- Buffer
10x
1x
2.5
RNA
25.05µM
10µM
10
Biotin - NTP
1mM
200µM
5
TdT
600U/µL
24U/µL
1
CoCl2
2.5mM
0.5mM
5
MQ H2O
 
 
1.5
 
The above listed substances were all mixed together and then incubated for 2 hours at 37 °C.
▼2015-08-21  Biotin labelled substrate RNA with Poly(A)Polymerase (yeast)
For 25µL
Cstock
Cfinal
V [µL]
H2O
 
 
12.14
PAP- Buffer
5x
1x
5
Biotin - NTP
1mM
100µM
2.5
RNA
28.7µM
5µM
4.36
PAP
600U/µL
24U/µL
1
 
The substances were mixed together as listed in the table above.
Eventually, having a volume of 25µl the mixture was incubated for two hours at 37 °C.
 
The Biotin labelled RNA was purified via Ethanol precipitation. Hereby sodium acetate and ethanol were added to the reagent.
 
 
Volume [µL]
NaOAc (3M, pH=5.5)
2.5
EtOH
75
 
Afterwards, the Biotin labelled RNA was stored at -20 °C.
Later, the tube was centrifuged at 14.000 rpm and 4 °C for 30 min.
The supernatant was thrown away and the pellet was dissolved in 10µl MQ H2O.
▼2015-08-23  KW 35 - Diazo transfer reaction with HRP and Imidazole-1-sulfonyl azide hydrochloride
For 340µL
Cstock
V[µL]
HRP
10mg/mL
40
MQ H2O
 
160
K2CO3
2mg/mL
100
Cu(II)SO4•5H2O
1mg/mL
25
Imidazole-1-sulfonyl azide hydrochloride
2mg/mL
15
 
The listed substances were added chronologically and then stored for 14h and 48h at RT.
After that the solution was transferred to a Amicon UltraFree-MC centrifugal filter with a 10kDa cut off.
For a better yield HRP was purified via TCA precipitation.
 
1 volume TCA (100% (w/v) to 4 volumes protein sample
Incubate for 10min at 4 °C
Centrifuge at 14K rpm for 5min
Remove the supernatant
Let the pellet dry in a heat block for 5-10min at 95 °C
 
In spite of the precipitation the concentration was very low.
Therefore, for the next approach an Amicon UltraFree-MC centrifugal filter with a 3kDa cut off was used.
week number 35
▼2015-08-25  Click Chemistry – RNA
For 100µL
Cstock
Cfinal
V[µL]
Phosphate Buffer - pH 7, 0.1M
100mM
50mM
50
Alexa 488 azide
10µM
400nM
4
RNA
1µM
200nM
20
CuSO4
50mM
100µM
0.2
THPTA
50mM
500µM
1
NaAsc
500mM
1mM
0.2
H2O
 
 
24.6
 
As the first click chemistry attempt showed positive results for one RNA sample, the reaction was repeated. All substances were mixed together as listed in the table above.
▼2015-08-25  Primer extension for substrate RNA with AU Linker
Oligos xxfs040xx and xxfs041xx (for Substrate RNA with AU linker) were annealed and extended using Phusion Flash:
 
cStock
cFinal
V[µL]
Primer 1
100 mM
2 µM
2
Primer 2
100 mM
2 µM
2
DMSO
 
5 %
5
Phusion Flash Master mix
2 x
1 x
50
H2O
 
ad 100 µL
41
Final
 
 
100
PCR settings:
 
Temperture [°C]
Time
Cycles
Initial denaturation
95
3 min
1
Denaturation
95
30 s
5
Annealing
62
30 s
Elongation
72
30 min
Final elongation
72
2 min
1
Hold
12
∞
1
Size of the samples were checked on a 1 % agarose in TAE gel and visualized with Ethidiumbromide (EtBr) under UV light.
▼2015-08-27  click chemistry - modified
As there were no positive results, the experiment was repeated, however, with altered concentrations:
 
For 100µL
Cstock
Cfinal
V[µL]
Phosphate Buffer - pH 7, 0.1M
100mM
50mM
50
Alexa 488 azide
10µM
400nM
4
RNA
1µL
200nM
20
CuSO4
50mM
100µM
1
THPTA
50mM
500µM
5
NaAsc
500mM
1mM
0.2
H2O
 
 
24.6
▼2015-08-28  Diazo transfer – HRP
For 340µL
Cstock
V[µL]
HRP
10mg/mL
40 and 100
MQ H2O
 
160 and 100
K2CO3
2mg/mL
100
Cu(II)SO4•5H2O
1mg/mL
25
Imidazole-1-sulfonyl azide hydrochloride
2mg/mL
15
 
Two samples were made with either 40µL HRP and 160µL MQ H2O or 100µL HRP and 100µL MQ H2O.
For a higher concentration each sample was centrifuged at 14k rpm for 10-15min with an Amicon UltraFree- MC centrifugal filter mixed with in total 2-3ml MQ H2O.
▼2015-08-28  Click chemistry – HRP (28-08-2015)
For 100µL
Cstock
Cfinal
V[µL]
Phosphate Buffer - pH 7, 0.1M
100mM
50mM
50
Alexa 488 azide
10µM
40µL: 2346,59nM
100µL: 465,91nM
40µL: 23,47
100µL: 4,66
Protein
40µL: 9386,36nM
100µL: 1863,63nM
40µL: 469,318nM
100µL: 93,1818nM
5
CuSO4
50mM
100µM
1
THPTA
50mM
500µM
5
NaAsc
10mM
1mM
10
H2O
 
 
40µL: 5,53
100µL: 24,34
 
The samples were incubated for 2 hours at 37 °C.
Afterwards they were analysed via SDS PAGE.
▼2015-08-30  Annealing of Primers to dsDNA for the ivT of labeling RNA
Equal amounts of Primers xxfs021xx and xxfs024xx were mixed and heated up 95 °C for 5 min. Sample was allowed to cool to RT.
Outlook:
dsDNA is used for ivT
week number 36
▼2015-08-31  Labeling of RNA with alkyne-NTP
The labeling RNA was modified with alkyne NTPs by Terminale Dinucleotidyl Transferase (TdT) and as a second approach, by Poly A Polymerase (yeast), affimetrix.
The modified NTPs are from Jena Bioscience:
5-Ethynyl-dUTP (5-EdUTP)
C8-Alkyne-dUTP
C8-Alkyne-dCTP
 
cStock
cFinal
V[µL]
TdT buffer
10 x
1 x
2.5
Labeling RNA
84,97µM
15 µM
4.41
Alkyne-NTP
10 mM
400 µM
1
TdT
20,000 u/mL
80 u/mL
1
CoCl2
2.5 mM
0.25 mM
2.5
H2O
 
Add 25 µL
13.59
Final
 
 
25
 
 
cStock
cFinal
V[µL]
PAP (yeast)
600 u/µL
24 u/µL
1
PAP buffer
5 x
1 x
5
5-EdUTP/ C8-Alkyne-dUTP/C8-Alkyne-dCTP
10 mM
400 µM
1
Labeling RNA
84.97µM
15µM
4.41
H2O
 
Add 25 µL
13.59
Final
 
 
25
 
All compounds were mixed as indicated in the tables.
Reactions were incubated for 2 hours at 37 °C
Samples were stored at -20 °C oN
▼2015-09-01  Biotin modification of substrate RNA
For 25µL
Cstock
Cfinal
V [µL]
TdT- Buffer
10x
1x
2.5
RNA
71.804µM
10µM
3.48
Biotin - NTP
1mM
200µM
5
TdT
600U/µL
24U/µL
1
CoCl2
2.5mM
0.5mM
5
MQ H2O
 
 
8.02
 
All samples were mixed together and incubated for 2 hours at 37 °C.
▼2015-09-01  Click reaction (modified)
new Alexa 488 with DMSO
 
For 50µL
Cstock
Cfinal
V[µL]
Phosphate Buffer - pH 7, 0.1M
100mM
50mM
25
Alexa 488 azide
10µM
400nM
2
RNA
1µM
200nM
10
CuSO4
50mM
1mM
1
THPTA
50mM
5mM
5
NaAsc
100mM
1mM
0.5
H2O
 
 
6.5
 
All samples were mixed together and then incubated for 2hours at 37 °C.
▼2015-09-02  In vitro transcription of Labeling RNA AU + substrate HRP
 
cStock
cFinal
V[µL]
Transcription buffer
10 x
1 x
20
ATP
100 mM
4 mM
8
GTP
100 mM
4 mM
8
CTP
100 mM
4 mM
8
UTP
100 mM
4 mM
8
DTT
1 M
10 mM
2
DMSO
 
5 %
10
FS_53/54 (Label AU)
 
 
20
T7 RNA Polymerase
2 mg/mL
0,1 mg/mL
5
H2O
 
ad 100 µL
106
Final
 
 
400
 
 
cStock
cFinal
V[µL]
Transcription buffer
10 x
1 x
20
ATP
100 mM
4 mM
8
GTP
100 mM
4 mM
8
CTP
100 mM
4 mM
8
UTP
100 mM
4 mM
8
DTT
1 M
10 mM
2
DMSO
 
5 %
10
FS_50/51 (Substrate HRP)
 
 
20
T7 RNA Polymerase
2 mg/mL
0,1 mg/mL
5
H2O
 
ad 100 µL
106
Final
 
 
200
 
All compounds were mixed together and incubated for 3 hours at 37 °C. Afterwards, 2µL DNAseI was added for 30min at 37 °C.
week number 37
▼2015-09-10  DNAzyme Activity: DNAzyme with ATP aptamer and calculated Kanamycin aptamer
Samples:
10-23 DNAzyme: xxfs032xx
7-18 DNAzyme: xxfs033xx
10-23 DNAzyme with ATP aptamer with linker: xxfs019xx
7-18 DNAzyme with ATP aptamer with linker: xxfs027xx
10-23 DNAzyme with ATP aptamer A: xxfs017xx, B: xxfs018xx
7-18 DNAzyme with ATP aptamer A: xxfs025, B: xxfs026xx
10-23 DNAzyme with calculated Kan aptamer: xxfs034xx
7-18 DNAzyme with calculated Kan aptamer candidate I: xxfs035xx
7-18 DNAzyme with calculated Kan aptamer candidate II: xxfs036xx
7-18 DNAzyme with calculated Kan aptamer candidate III: xxfs037xx
Stock solutions and conditions:
 
cStock
cFinal
Tris HCl ph 7.5
1 M
50 mM
DNAzyme (A)
10 µM
500 nM
DNAzyme B
10 µM
500 nM
Substrate
1 µM
200 nM
NaCl
1 M
100 mM
MgCl2
1 M
20 mM
SDS
20 %
0,01 %
Adenosine in H2O:DMSO 1:2
33 mM
5 mM
H2O
 
ad 25 µL
 
Pipetting scheme:
#
 
 
Tris HCl ph 7.5
DNAzyme A
DNAzyme B
Substrate
NaCl
MgCl2
SDS
Adenosine
H2O
Final
1
FS032
10-23D
1,25
1,25
0,00
5,00
2,50
0,50
1,25
3,75
9,50
25,00
2
FS033
7-18D
1,25
1,25
0,00
5,00
2,50
0,50
1,25
3,75
9,50
25,00
3
FS019
10-23DmLink
1,25
1,25
0,00
5,00
2,50
0,50
1,25
3,75
9,50
25,00
4
FS027
7-18DmLink
1,25
1,25
0,00
5,00
2,50
0,50
1,25
3,75
9,50
25,00
5
FS017+18
10-23D_A+B
1,25
1,25
6,25
5,00
2,50
0,50
1,25
3,75
3,25
25,00
6
FS025+26
7-18D_A+B
1,25
1,25
6,25
5,00
2,50
0,50
1,25
3,75
3,25
25,00
7
FS032
10-23D
1,25
1,25
0,00
5,00
2,50
0,50
1,25
0,00
13,25
25,00
8
FS033
7-18D
1,25
1,25
0,00
5,00
2,50
0,50
1,25
0,00
13,25
25,00
9
FS019
10-23DmLink
1,25
1,25
0,00
5,00
2,50
0,50
1,25
0,00
13,25
25,00
10
FS027
7-18DmLink
1,25
1,25
0,00
5,00
2,50
0,50
1,25
0,00
13,25
25,00
11
FS017+18
10-23D_A+B
1,25
1,25
6,25
5,00
2,50
0,50
1,25
0,00
7,00
25,00
12
FS025+26
7-18D_A+B
1,25
1,25
6,25
5,00
2,50
0,50
1,25
0,00
7,00
25,00
13
FS032
-10-23D
1,25
1,25
0
0
2,5
0,5
1,25
0
18,25
25,00
14
FS033
-7-18D
1,25
1,25
0
0
2,5
0,5
1,25
0
18,25
25,00
15
FS019
-10-23DmLink
1,25
1,25
0
0
2,5
0,5
1,25
0
18,25
25,00
16
FS027
-7-18DmLink
1,25
1,25
0
0
2,5
0,5
1,25
0
18,25
25,00
17
FS017+18
-10-23D_A+B
1,25
1,25
6,25
0
2,5
0,5
1,25
0
12
25,00
18
FS025+26
-7-18D_A+B
1,25
1,25
6,25
0
2,5
0,5
1,25
0
12
25,00
19
Adenosine
Substrate only
1,25
0
0
5
2,5
0,5
1,25
3,75
10,75
25,00
 
 
#
 
 
Tris HCl ph 7.5
DNAzyme A
DNAzyme B
Substrate
NaCl
MgCl2
SDS
Kan
H2O
Final
20
FS032
10-23D
1,25
1,25
0
5
2,5
0,5
1,25
1,25
12
25
21
FS033
7-18D
1,25
1,25
0
5
2,5
0,5
1,25
1,25
12
25
22
FS034
Kan
1,25
1,25
0
5
2,5
0,5
1,25
1,25
12
25
23
FS035
Kan I
1,25
1,25
0
5
2,5
0,5
1,25
1,25
12
25
24
FS036
Kan II
1,25
1,25
0
5
2,5
0,5
1,25
1,25
12
25
25
FS037
Kan III
1,25
1,25
0
5
2,5
0,5
1,25
1,25
12
25
26
FS032
10-23D
1,25
1,25
0
5
2,5
0,5
1,25
0
13,25
25
27
FS033
7-18D
1,25
1,25
0
5
2,5
0,5
1,25
0
13,25
25
28
FS034
Kan
1,25
1,25
0
5
2,5
0,5
1,25
0
13,25
25
29
FS035
Kan I
1,25
1,25
0
5
2,5
0,5
1,25
0
13,25
25
30
FS036
Kan II
1,25
1,25
0
5
2,5
0,5
1,25
0
13,25
25
31
FS037
Kan III
1,25
1,25
0
5
2,5
0,5
1,25
0
13,25
25
32
Kan
Substrate only
1,25
0
0
5
2,5
0,5
1,25
1,25
13,25
25,00
33
 
Substrate only
1,25
0
0
5
2,5
0,5
1,25
0
14,5
25,00
 
Results and Outlook:
Positive controls worked, Adenosine dependency could be detected for one candidate.
▼2015-09-10  Click reaction of Label-, Label AU RNA [1, 2, 3]
For 51.5µL
Cstock
Cfinal
V[µL]
Phosphate Buffer - pH 7, 0.1M
100mM
50mM
25
Alexa 488 azide
10µM
400nM
2
RNA
1µM
200nM
10
CuSO4
20mM
1mM
2.5
THPTA
50mM
5mM
5
NaAsc
100mM
1mM
0.5
H2O
 
 
6.5
 
Alexa 488 azide was solved in DMSO
Incubation at 37 °C for 12-14 hours (overnight)
