Team:Dundee Schools/Experiments
MENU ▤
HOME
TEAM
Meet the Agents
Collaborations
PROJECT
Description
Design
Experiments
Proof of Concept
Results
Notebook
PARTS
Parts
Basic Parts
Composite Parts
Part Collection
SAFETY
ATTRIBUTIONS
HUMAN PRACTICES
Human Practices
Silver
Gold
Integrated Practices
Engagement
AWARDS
Modelling
Medals
Dundee Schools
Experiments
Standard PCR reaction (final volume = 50 μl)
Component
Volume (μl)
5x Herculase buffer
10
100 μM forward primer
0.5
100 μM reverse primer
0.5
10 mM dNTPs
0.5
DMSO
2.5
H2O
34.5
gDNA
1
Herculase II
0.5
Agarose Gel Electrophoresis
DNA products were run on a 1% (w/v) agarose gel prepared in 1 x tris-acetate-EDTA (TAE) buffer containing GelRed (in a ratio of 1:20,000) to stain the nucleic acids. The sample was mixed with 10 x DNA loading dye and loaded on the gel with the DNA marker (Roche). Gels were then run in 1 x TAE buffer for 30 min at 100 V. After sufficient separation the gel was visualised under UV light on the BioRad GelDoc.
TAE Buffer (pH 7.6): 40mM Tris, 20mM acetic acid, 1mM EDTA
Gel Extraction of DNA products
DNA fragments were excised from agarose gels and the QIAquick Gel Extraction Kit protocol was followed (Qiagen) according to manufacturer’s instructions. DNA eluted in 50 μl of ddH2O.
Double Restriction Digests
Restriction digests were performed to prepare DNA for ligation.
60 μl reactions following clean up after PCR were set up:
50 μl PCR product
1.5 μl restriction enzyme 1
1.5 μl restriction enzyme 2
6 μl cognate digestion buffer for enzyme(s)
1 μl ddH2O
Digests were carried out using enzymes supplied by Roche and New England BioLabs and incubated at 37°C for 3 hours. A 2.5 μl aliquot of alkaline phosphatase was added to restriction digests of vectors at 2 hours and a second aliquot at 2.5 hours.
PCR purification
Following restriction digest, DNA products were then cleaned using the QIAquick PCR purification kit (Qiagen) according to the manufacturer’s instructions and eluted in 50 μl ddH2O.
Transformation and Competent Cells
Add 100μl of thawed (defrosted) competent cells into a chilled clean Eppendorf with up to 10 uL of the DNA you wish to transform
Rest on ice for 5 minutes
Heat shock at 42°C for 90 seconds
Rest on ice for 5 minutes
Add 1μl liquid broth and incubate at 37°C for 1 hour
Spin down at full speed for 10 minutes and remove supernatant
Add 50μl LB and plate
Leave in hot room over night
Overnight Culture
Add 10μl liquid broth in tube
Add 10μl antibiotic
Scrape colonies and put clean pipette tip in solution
Store in hot room overnight
Mini Prep
For isolation of plasmid DNA, the QIAprep Spin Miniprep Kit (Qiagen) was followed according to the manufacturer’s instructions and DNA eluted in 50 μl of ddH2O.
Colony PCR
Take a clean petri dish and spot 12 μL H2O on to the plastic for every colony you wish to test.
Pick up your colony and resuspend it in the spot of water you just made
Take 3 μL of this and spot it on to a fresh LB plate (or other appropriate)
Take another 3 μL and add to PCR reaction mix
In the PCR program, add an extra 15 min lysis step at 95oC
Making spiRNA
Design primers to amplify the entire spiRNA plasmid (BBa_K1963003 or BBa_K1963004) that add a 24nt sequence of perfect homology to target gene
PCR amplify the plasmid using these primers and a DNA polymerase that gives a product with blunt ends
PCR purify this and ligate 5 μL of the purified sample in a 10 μL reaction
Transform in to JM110 competent cells
Miniprep and sequence the resulting plasmid
SDS
Resolving gel
Recipe for 12% acrylamide gel:
Water - 2.1 ml
Acrylamide (30% stock) - 4 ml
Tris HCl, 1M pH 8.8 - 3.75 ml
SDS 10% w/v - 100 μl
APS 10% w/v - 50 μl
TEMED - 5 μl
After casting the resolving gel, you should add a layer of isopropanol on top, the overlay, to get rid of any bubbles and keep a straight line on the top. Once the resolving gel has set you can pour it off by tipping the cassette or by using some blotting paper to sook it out. Then add the stacking gel right to the top and insert the comb immediately.
Stacking gel
Water - 2.16 ml
Acrylamide (30% stock) - 400 μl
Tris HCl, 1M pH 6.8 - 375 μl
SDS 10% w/v - 30 μl
APS 10% w/v - 30 μl
TEMED 3 μl
Running gel
Run the gel in SDS running buffer for an hour at 200V.
Check regularly for bubbles in the wells that could affect the mobility of samples in these lanes.
Western blot
Take overnight cultures and dilute 1/50 with LB
Grow to an OD600 of 0.4
Induce the samples with different concentrations of rhamnose: 0.1, 0.2, 0.4 and 0.5% (v/v)
Incubate the samples and take out 1ml of the sample each hour: 0hrs, 1hr, 2hrs, 3hrs, 4hrs and 16hrs
Take each sample and spin down in centrifuge for 10 mins to separate cell and supernatant
Run samples on SDS gel
Place gel between 2x filter paper, 1 PVDF membrane, 2x filter paper (bottom to top) all soaked in transfer buffer (0.6g tris, 2.9 glycine, 40 ml methanol, make up to 200 ml with water)
Assemble this in semi-dry blot apparatus and transfer at 20V for 20 mins
Take membrane which should now have protein transferred to it and block in 5% skim milk (made up in PBS). Shake RT at least an hour.
Wash 3x in TBS tween (0.1% tween) 10 minutes each
Add antibody – 3μL in 30ml TBStween, incubate with shaking 1 hour RT
Wash 3x ten minutes in TBST
Take 1ml BioRad HRP detection solution and pipette on to membrane, incubate 2 min RT and develop with GeneGnome.
Swimming motility assay
Make 0.25% agar LB plates
Place a single colony from a streak plate of your desired strain on the surface of the plate
Incubate plate face up at 37oC for approximately 36 hours
Image plates
Using ImageJ, measure the colony area with the Wand tool set to Legacy mode.
