<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_UPMC-Paris_Notebook skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><DIV id="top_title"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:UPMC-Paris/Notebook</SPAN></H1></DIV><DIV id="HQ_page"><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><DIV id="Menucontent"><DIV class="container" id="main" role="main"><UL class="menu"><LI><A href="#s1">Project |</A></LI><LI><A href="https://2016.igem.org/Team:UPMC-Paris/Project/overview">Overview</A></LI><LI><A href="https://2016.igem.org/Team:UPMC-Paris/Design">Design</A></LI><LI><A href="https://2016.igem.org/Team:UPMC-Paris/Experiments">Experiments</A></LI><LI><A href="https://2016.igem.org/Team:UPMC-Paris/Application">Application</A></LI><LI><A href="https://2016.igem.org/Team:UPMC-Paris/Proof">Proof</A></LI></UL><LI><A href="#s2">LAB Life |</A></LI><LI><A href="https://2016.igem.org/Team:UPMC-Paris/La_Paillasse">La Paillasse</A></LI><LI><A href="https://2016.igem.org/Team:UPMC-Paris/Safety">Safety</A></LI><LI><A href="https://2016.igem.org/Team:UPMC-Paris/Notebook">Notebook</A></LI><LI><A href="https://2016.igem.org/Team:UPMC-Paris/Material">Material</A></LI><LI><A href="#s3">Parts |</A></LI><LI><A href="https://2016.igem.org/Team:UPMC-Paris/Overview">Overview</A></LI><LI><A href="https://2016.igem.org/Team:UPMC-Paris/Description">Description</A></LI><LI><A href="https://2016.igem.org/Team:UPMC-Paris/Basic_Part">Basic</A></LI><LI><A href="https://2016.igem.org/Team:UPMC-Paris/Composite_Part">Composite</A></LI><LI><A href="https://2016.igem.org/Team:UPMC-Paris/Part_Collection">Collection</A></LI><LI><A href="#s4">Human Practice |</A></LI><LI><A href="https://2016.igem.org/Team:UPMC-Paris/HP/Overview">Overview</A></LI><LI><A href="https://2016.igem.org/Team:UPMC-Paris/Integrated_Practices">Integrated Practice</A></LI><LI><A href="https://2016.igem.org/Team:UPMC-Paris/HP/Silver">Investigations</A></LI><LI><A href="https://2016.igem.org/Team:UPMC-Paris/HP/Gold">Adaptations</A></LI><LI><A href="https://2016.igem.org/Team:UPMC-Paris/Collaborations">Team Work |</A></LI><LI><A href="#s5">About Us</A></LI><LI><A href="https://2016.igem.org/Team:UPMC-Paris/Team">Team</A></LI><LI><A href="https://2016.igem.org/Team:UPMC-Paris/Attributions">Attributions</A></LI><LI><A href="https://2016.igem.org/Team:UPMC-Paris/Achievement">Achievement</A></LI></DIV></DIV><DIV id="footer"><DIV id="content2"><DIV id="text"><DIV class="image" id="Week1Pic" onclick="ShowHide(week1)"><H1 class="week">Week 1 :</H1></DIV><DIV id="week1" style="display: none;"><H3><U>Competence of Bsu9716:</U></H3><P><I>Bacteria from 2/8/16 in fridge (Dj)</I>
50 ml LB and 10 uL Kan added (so total 150 mL and 15 uL Kan) and cells back in incubator
Spin (4000 rpm 10 min) 50 ml X 2 and used pellet for competency
Spinned again same way after pulling both pellet together
Changed medium (50 mL LB + 5 uL Kan) and put it back at 4 degrees<I>Stock solutions (Dj)</I></P><TABLE><TBODY><TR><TH>Name </TH><TH>Mass (g)</TH><TH>Total Volume (mL)</TH></TR><TR><TD>1.2% MgSO4</TD><TD>0.1023</TD><TD>12.5</TD></TR><TR><TD>10% (w/v) Bacto yeast extract</TD><TD>1.00960.1023</TD><TD>10</TD></TR><TR><TD>1% (w/v) casamino acids</TD><TD>0.1039</TD><TD>10</TD></TR><TR><TD>0.1 M CaCl2 (110.98 g/mol)</TD><TD>0.1792</TD><TD>15</TD></TR></TBODY></TABLE><P><I>SpC  medium made by Djamila</I></P><OL><LI> ➟ T base 20 ml</LI><LI> ➟ 50% (w/v) filtered glucose 0.2 ml</LI><LI> ➟ 1.2% (w/v) MgSO4 0.3 ml</LI><LI> ➟ 10% (w/v) Bacto yeast extract 0.4 ml</LI><LI> ➟ 1% (w/v) casamino acids 0.5 ml</LI></OL><P>ODi 0.5 and put back at 37 degrees with shaking
ODf 0.8
<I>SpII medium made by Sofiane</I></P><OL><LI> ➟ T base 200 ml</LI><LI> ➟ 50% (w/v) filtered glucose 2 ml</LI><LI> ➟ 1.2% (w/v) MgSO4 14 ml</LI><LI> ➟ 10% (w/v) Bacto yeast extract 2 ml</LI><LI> ➟ 1% (w/v) casamino acids 2 ml</LI><LI> ➟ 0.1 M CaCl2 1 ml</LI></OL><P>2 mL of SpC BSU in SpII and 90 min incubation 37 degrees with shaking<SPAN style="text-decoration: underline;">Note:</SPAN>
Originally 1.2% of the dihydrate form so made the calculations in function of that; and Sofiane had to make a second batch
50% (w/v) glucose was filtered
No tryptophan added for the cell growth
Plates from 03/08/2016 (Zinebe)
Contains lawn and colonies ⇒ Not sure which are the B. subtilis.
Diluted some of the lawn and the colony each in 200 uL LB and plated it on LB-Kan then 37 degrees overnight<SPAN style="text-decoration: underline;">Note:</SPAN>
LB used for dilution was contaminated
When picking the colony with a tip some of the lawn was scraped with it
Pick primers and autoclaved MilliQ at school
Plates (from 04/08/16) (DJ)
Both plates had lawns on them however the plate containing the colony (along with some lawn and labelled Isolate) was green which was weird
Took plate from 3/8/16 and streaked on a plate divided in 3 a single colony, the lawn, and for the last portion, 10 uL of the B subtilis solution in the fridge was diluted in 200 uL of LB before streaking it.
The rest of B subtilis solution in the fridge was spinned, diluted in 10ml LB and divided in 2 x 50 mL tubes, 5 uL of Kan was added in each. Both were put back in the fridge
Tris HCl made by Kevin
All streaks grew well so we put them in fridge</P></DIV><DIV class="image" id="Week2Pic" onclick="ShowHide(week2)"><H1 class="week">Week 2 :</H1></DIV><DIV id="week2" style="display: none;"><P style="text-align:center;"><B>Nothing done at the bench this week because of material issues</B></P></DIV><DIV class="image" id="Week3Pic" onclick="ShowHide(week3)"><H1 class="week">Week 3 :</H1></DIV><DIV id="week3" style="display: none;"><P><U>Tris-EDTA preparation (Sofiane):</U></P><OL><LI> ➟ 10mL 1M Tris pH 7.5 + 2mL 0,5M EDTA pH 8.0</LI><LI> ➟ 0,5M EDTA : 18,6g EDTA + 100 mL H2O. Ajuste pH 8.0 with solid NaOH (2,6g)</LI><LI> ➟ 1M Tris : 30,285g + 205 mL H2O. Ajust pH 7.5 with HCl. </LI></OL><P><I>Note :</I> pH ajustment failed, I add 25mL of 12% HCl and 12 mL of 20% HCl to have a pH 7,6.
Concentration is not good anymore. We have to make this solution again. Maybe buy TE Buffer? Verification of TE we have is needed!<SPAN style="text-decoration: underline;"><STRONG>Competency test (Djamila/Zinebe)</STRONG></SPAN><SPAN style="text-decoration: underline;">Chloramphenicol (cm 25 mg/mL) preparation:</SPAN>
0.25 g diluted in 10 mL EtOH 95%; aliquots at -20
Note: first dilution in milliQ water by mistake
Got rid of water and tried to let it dry but without success so will probably just add directly EtOH in to make more Cm<SPAN style="text-decoration: underline;">Cm (Cf = 5 ug/mL) LB plates:</SPAN>
70 mL melted LB agar + 14 uL c<SPAN style="text-decoration: underline;">Note:</SPAN> agar cooled too much before pouring plate (only got 3 plates) so made more 100 mL melted LB agar + 20 uL cm<SPAN style="text-decoration: underline;"><STRONG>Competency test (CT)</STRONG></SPAN>
Made SpII-EGTA solution (solutions made yesterday by Sofiane<SPAN style="text-decoration: underline;">T base 200 ml :</SPAN></P><OL><LI> ➟ 50% (w/v) glucose 2 ml</LI><LI> ➟ 1.2% (w/v) MgSO4·3H2O 14 ml</LI><LI> ➟ 10% (w/v) Bacto yeast extract 2 ml</LI><LI> ➟ 1% (w/v) casamino acids 2 ml</LI><LI> ➟ 4 ml EGTA (0.1 M, pH 8.0)</LI></OL><P>Aliquots put at -20°C
After thawing competent BSU9716 at 37C in incubator (water bath had issues), added 500 uL of SpII-EGTA and mixed up and down.
Added 50 uL bacteria in 5 eppendorf tubes after spinning CT plasmids, added 1 uL of DNA in the appropriate tube (50pg/ul, 20pg/ul, 10pg/ul, 5pg/ul, 0.5pg/ul).
Bacteria+DNA mixture incubated at 37°C with shaking (150 rpm) for an hour
Spreaded the bacteria on Cm LB agar plate then put at 37°C overnight
At this point from the rest of competent bacteria, 100 uL spread on Kan plates<SPAN style="text-decoration: underline;">Note:</SPAN>
Should have spread the transformed bacteria on Cm and Kan plates
Competency test kit contains five vials of purified DNA from BBa_J04450 (RFP construct) in plasmid backbone pSB1C3 (cm resistant). Each vial contains DNA at a different concentration: 50pg/ul, 20pg/ul, 10pg/ul, 5pg/ul, 0.5pg/ul.</P></DIV><DIV class="image" id="Week4Pic" onclick="ShowHide(week4)"><H1 class="week">Week 4 :</H1></DIV><DIV id="week4" style="display: none;"><P>Plasmidic DNA Extraction on the liquid cultures made Sunday by Nassim and Wilhelm (Zinebe)
    --&gt; Followed protocol: “Monarch plasmid miniprep kit” from NEB <I>Note: First centrifugation of 10 mL, but only 5ml needed, so we re-suspended them in 5mL and centrifuge again.</I><U>DNA concentration obtained using Nanodrop</U></P><TABLE><TBODY><TR><TH>Plasmid Name</TH><TH>Plasmid Concentration (µg/mL)</TH><TH>260/280</TH><TH>260/230</TH></TR><TR><TD>pMAD</TD><TD>75.136</TD><TD>1.969</TD><TD>2.556</TD></TR><TR><TD>pBS1C</TD><TD>81.240</TD><TD>1.725</TD><TD>0.740</TD></TR></TBODY></TABLE><P>
Competency test check (Djamila) done on Friday 19th failed (Boys checked on Saturday) ie bacteria but no rfp from igem plasmid next step? 
BSU9716 Plates from Sunday 20th(Wilhelm/Nassim) 
No growth on Kan/Cm and Amp, plate back on Kan, Cm and Amp individually (from “loan plate”) and also frozen from initial BSU batch on LB then 37 degrees overnight
Oligo dilution in TE 1X (Sofiane) final concentration 100 uM, at -20<U>BSU9716 Plates from Monday 20th(Djamila)</U>
Cells on Kan but not on Amp and Cm; should had have it on Kan and Cm so don’t use anymore “loan plate” as BSU9716 source. 
Frozen on LB grew, so put it (chose 4 isolates and 2 locations on loan) on Kan/Cm and 37C overnight (Sofiane) 
Made more LB agar 15g agar 1L LB
Liquid solutions (not from isolates but loan for bsu and directly from tube in which came from) of pEN and BSU9716 in 10 mL of LB gent (10 ug/mL) and Kan/Cm (50//25ug/mL), respectively, for tomorrow and plated both on appropriate antibiotic<U><B>Preparing lysozyme aliquot (Zinebe) : </B></U>
Concentration = 20g/L : 
 → 0,0220 g lysozyme in 1,1 mL miliQ water (mix with vortex) 
Conservation dans le congélo -20°C<U>pEN plasmid extraction (Djamila)</U>
 --&gt; Using the “Monarch Plasmid Miniprep Kit”, according to the NEB protocol 
Note: Mistakenly eluted with 130 uL instead of 30 uL
</P><TABLE><TBODY><TR><TH>Plasmid Name</TH><TH>Plasmid Concentration (µg/mL)</TH><TH>260/280</TH><TH>260/230</TH></TR><TR><TD>pEN-Gent</TD><TD>33.197</TD><TD>1.872</TD><TD>1.515</TD></TR></TBODY></TABLE><P><U>Plates from 24/08/16 (Djamila)</U>
Bsu9716 no isolates and pEN single colonies, but on both plate there is a lot of bubbles
gDNA extraction
plasmids glycerol stock
gBlocks/biobrick resuspension
gblocks digestion<U><I>E. coli</I> DH5alpha Transformation with pBS1C using the following protocol :</U><OL><LI> ➟ Put 1μL of circular plasmid or all of a ligation reaction of plasmid DNA in a microtube. Gently add ~100μL of competent cells. Do NO DNA control tube with cells and no DNA.</LI><LI> ➟ Incubate for 30 mins on ice.</LI><LI> ➟ Heat shock for 1 mins @ 42°C. Put back on ice.</LI><LI> ➟ Add 900 μL of LB to tubes. Incubate @ 37°C for 30 mins.</LI><LI> ➟ Plate 490 μL of the cells in LBAmp (100µg/ml plates). Plate 100 μL of the NO DNA control in a LB plate (to check for quality of cells). Grow O/N.</LI><LI>   ➟ 2 boites de LBamp à 100 µg/mL avec du e.coli DH5 alpha + pBS1C </LI><LI>   ➟ 1 boite de LB sans Amp avec que du e.coli DH5 alpha </LI></OL></P><P><U>Transformation from 26/08 worked:</U> Red on transformed and nothing red on negative control 
Made 2 bottles of 1X TAE (100 mL TAE 10X and 900 mL water osmosee) 
Set up PCR programs for deleted fragments amplification</P></DIV><DIV class="image" id="Week5Pic" onclick="ShowHide(week5)"><H1 class="week">Week 5 :</H1></DIV><DIV id="week5" style="display: none;"><P><U>Gradient PCR for pEN plasmid</U> (Gent amplification) 
61.6 degrees middle and +2 gradient 3 on left and 3 on right tube 4 in whole 7
Dilution Primers: final concentration 5 uM (2.5 uL primer+47.5 uL nuclease free water) 
Dilution pEN: 1/10 (2 uL pEN + 18 uL Elution Buffer EB) final concentration 3.32 ug/mL<TABLE><TBODY><TR><TH>Component</TH><TH>20 μl Reaction</TH><TH>Final Concentration</TH></TR><TR><TD>Nuclease free water</TD><TD>3.9 μl</TD><TD> - </TD></TR><TR><TD>2X Phusion Master Mix</TD><TD>10 μl</TD><TD>1X</TD></TR><TR><TD>5 μM Forward Primer	</TD><TD>2 μl</TD><TD>0.5 μM</TD></TR><TR><TD>5 μM Reverse Primer	</TD><TD>2 μl</TD><TD>0.5 μM</TD></TR><TR><TD>Template DNA (3.3 ug/mL or ng/uL)</TD><TD>1.5 μl</TD><TD>5ng</TD></TR><TR><TD>DMSO (optional)</TD><TD>0.6 μl</TD><TD>3%</TD></TR></TBODY></TABLE></P><P>
Program Gent: IGEM1601<TABLE><TBODY><TR><TH>Step</TH><TH>Temperature</TH><TH>Time</TH></TR><TR><TD>Denaturation</TD><TD>98 °C</TD><TD>30s</TD></TR><TR><TD>Annealing and Elongation</TD><TD>98 °C61.6°C_ Gradient 2 °C72°C</TD><TD>30s15s15s</TD></TR><TR><TD>Final Elongation</TD><TD>72 °C</TD><TD>5min</TD></TR><TR><TD>Hold</TD><TD>4 °C</TD><TD> - </TD></TR></TBODY></TABLE></P><P><U>BSU9716 liquid culture prep</U><U>1.2% Gel of Gent amplification PCR product (Djamila)</U>
1.2 g Agarose 100 mL TAE 1X 10 uL (10000 X) Gel red
Samples: 
5 uL DNA,  4 uL loading dye, 11 uL MilliQ H2O Run 90V 45 min then 120V 55 min
Gent resistance gene size: 534 bp Band looks like it’s around the expected size</P><P><B>Note:</B> Have same amount of DNA in all well and this was supposed to be a gradient PCR. Therefore, it looks like all temperature are working well. Next PCR simple instead of gradient then if don’t work will do gradient
Run PCR for fragment A and B<TABLE><TBODY><TR><TH>Component</TH><TH>20 μl Reaction</TH><TH>Final Concentration</TH></TR><TR><TD>Nuclease free water</TD><TD>4.4 μl</TD><TD> - </TD></TR><TR><TD>2X Phusion Master Mix</TD><TD>10 μl</TD><TD>1X</TD></TR><TR><TD>5 μM Forward Primer	</TD><TD>2 μl</TD><TD>0.5 μM</TD></TR><TR><TD>5 μM Reverse Primer	</TD><TD>2 μl</TD><TD>0.5 μM</TD></TR><TR><TD>Template DNA</TD><TD>1 μl</TD><TD>Unsure</TD></TR><TR><TD>DMSO (optional)</TD><TD>0.6 μl</TD><TD>3%</TD></TR></TBODY></TABLE></P><P>
Program A IGEM1602
</P><TABLE><TBODY><TR><TH>Step</TH><TH>Temperature</TH><TH>Time</TH></TR><TR><TD>Denaturation</TD><TD>98 °C</TD><TD>30s</TD></TR><TR><TD>Annealing and Elongation</TD><TD>98°C66.2°C72°C</TD><TD>30s15s15s</TD></TR><TR><TD>Final Elongation</TD><TD>72°C</TD><TD>5min</TD></TR><TR><TD>Hold</TD><TD>4°C</TD><TD> - </TD></TR></TBODY></TABLE><P>
Program B: IGEM1603 65.9 (remember to change the name it’s IGEM1604 on the thermos cycler) </P><TABLE><TBODY><TR><TH>Step</TH><TH>Temperature</TH><TH>Time</TH></TR><TR><TD>Denaturation</TD><TD>98°C</TD><TD>30s</TD></TR><TR><TD>Annealing and Elongation</TD><TD>98°C65.9°C72°C</TD><TD>30s15s15s</TD></TR><TR><TD>Final Elongation</TD><TD>72°C</TD><TD>5min</TD></TR><TR><TD>Hold</TD><TD>4°C</TD><TD> - </TD></TR></TBODY></TABLE><P><B>Note:</B> issue with PCR of B; when looked at the TC to check evolution there was an error and we assumed the beginning of the PCR was done so we just did the final elongation step. 
Run gel for fragment A and B (second half of the 1.2% Gel of the morning) 
3 uL of DNA + 4 uL of loading dye + 13 uL milliQ H2O
Run 140V 30 min 160V 30 min
Band around 500 bp for A and nothing for B. Not surprising since maybe no PCR was done on it. Start over the B tomorrow and join A and Gent. Maybe purify do bigger volume and purify PCR before joining them. </P><P>
31/08/16<U>PCR Fragments A, B and Gent</U>
Used diluted samples and oligos done for previous PCR</P><TABLE><TBODY><TR><TH>Component</TH><TH>60 μl Reaction</TH><TH>Final Concentration</TH></TR><TR><TD>Nuclease free water</TD><TD>13.2 μl</TD><TD> - </TD></TR><TR><TD>2X Phusion Master Mix</TD><TD>30 μl</TD><TD>1X</TD></TR><TR><TD>5 μM Forward Primer	</TD><TD>6 μl</TD><TD>0.5 μM</TD></TR><TR><TD>5 μM Reverse Primer	</TD><TD>6 μl</TD><TD>0.5 μM</TD></TR><TR><TD>Template DNA</TD><TD>3 μl</TD><TD>Unsure</TD></TR><TR><TD>DMSO (optional)</TD><TD>1.8 μl</TD><TD>3%</TD></TR></TBODY></TABLE><P>
PCR programs: A IGEM1602, B IGEM1603 and Gent IGEM1601 without gradient
1.2% Gel (prepared the 30/08) 
5 uL of DNA + 3.5 uL of loading dye + 11.5 uL milliQ H2O
Have a band for each sample around 500 bp as expected but faint for B, and even for A compared to yesterday’s results. </P><P><U>PCR purification</U>
Used 40 uL of each sample. Followed QIAquick PCR Purification Kit Protocol. <B>Note:</B> Samples centrifuged at 12000 rpm<U>Recombinant PCR AG</U>
Used diluted samples and oligos done for previous PCR</P><TABLE><TBODY><TR><TH>Component</TH><TH>60 μl Reaction 1</TH><TH>60 μl Reaction 2</TH><TH>Final Concentration</TH></TR><TR><TD>Nuclease free water</TD><TD> - </TD><TD> - </TD><TD> - </TD></TR><TR><TD>2X Phusion Master Mix</TD><TD>30 μl</TD><TD>30 μl</TD><TD>1X</TD></TR><TR><TD>5 μM Forward Primer	</TD><TD>6 μl</TD><TD>6 μl</TD><TD>0.5 μM</TD></TR><TR><TD>5 μM Reverse Primer	</TD><TD>6 μl</TD><TD>6 μl</TD><TD>0.5 μM</TD></TR><TR><TD>Template DNA A</TD><TD>6.6 μl</TD><TD>8.2 μl</TD><TD>Unsure</TD></TR><TR><TD>Template DNA Gent</TD><TD>6.6 μl</TD><TD>6 μl</TD><TD>Unsure</TD></TR><TR><TD>DMSO (optional)</TD><TD>1.8 μl</TD><TD>1.8 μl</TD><TD>3%</TD></TR></TBODY></TABLE><P>
PCR programs: A-Gent IGEM1601 without gradient (Ta= 61.6) <U>Recombinant PCR AGB</U>
Used purified AG 1 and 2 </P><TABLE><TBODY><TR><TH>Component</TH><TH>60 μl Reaction 1</TH><TH>60 μl Reaction 2</TH><TH>Final Concentration</TH></TR><TR><TD>Nuclease free water</TD><TD>1.2 μl</TD><TD>1.2 μl</TD><TD> - </TD></TR><TR><TD>2X Phusion Master Mix</TD><TD>30 μl</TD><TD>30 μl</TD><TD>1X</TD></TR><TR><TD>5 μM Forward Primer	</TD><TD>6 μl</TD><TD>6 μl</TD><TD>0.5 μM</TD></TR><TR><TD>5 μM Reverse Primer	</TD><TD>6 μl</TD><TD>6 μl</TD><TD>0.5 μM</TD></TR><TR><TD>Template DNA A</TD><TD>6 μl</TD><TD>6 μl</TD><TD>Unsure</TD></TR><TR><TD>Template DNA Gent</TD><TD>6 μl</TD><TD>6 μl</TD><TD>Unsure</TD></TR><TR><TD>DMSO (optional)</TD><TD>1.8 μl</TD><TD>1.8 μl</TD><TD>3%</TD></TR></TBODY></TABLE><P>
PCR programs: A-Gent IGEM1604 (Ta= 61.8) <U>PCR purif (Zinebe)</U>
Used 40 μl of each sample (AG1, 2 and AGB 1, 2). Followed QIAquick PCR Purification Kit Protocol. <B>Note: </B>Samples centrifuged at 12000 rpm<U>Gel electrophoresis (Djamila) </U>

5 uL DNA,  4 μl loading dye, 11 μl MilliQ H2O Run 90V 45 min then 120V 55 min</P><P><B>Note: </B>none of the bands are where they are supposed to be. Plan was to ligate into plasmid but first start over gel and check bands. After that may need to start over recombinant PCR
02/09/16
M1 Meeting at UPMC with IONIS</P><TABLE><TBODY><TR><TH>PCR purified samples concentration</TH><TH>Concentration (ng/μl)</TH></TR><TR><TD>AG1</TD><TD>113.1</TD></TR><TR><TD>AG2</TD><TD>60</TD></TR><TR><TD>AGB1</TD><TD>20</TD></TR><TR><TD>AGB2</TD><TD>43.5</TD></TR><TR><TD>A</TD><TD>19.8</TD></TR><TR><TD>G</TD><TD>35.6</TD></TR><TR><TD>B</TD><TD>13.4</TD></TR></TBODY></TABLE><P><B>Note:</B>
Did the blank with water as there was no EB at Peronnet’s (elution in EB) 
No idea on purity (took pictures to have all purity on file but don’t have the phone anymore and the picures with) 
Biobricks digestion ligation Gel</P></DIV><DIV class="image" id="Week6Pic" onclick="ShowHide(week6)"><H1 class="week">Week 6 - 8 :</H1></DIV><DIV id="week6" style="display: none;">

Fragment Resuspension in RNase Free Water to 10 ng/uL. -20°C stockExtraction plasmidic DNA from many colonies, eluted with elution buffer heated to 50°C. DNA stock at 4°C.Elimination of the RFP cassette from the pSB1C3 plasmid. LacI and Pigment fragments digestion and ligation to pSB1C3.E. coli transformation and plasmid amplification inside.Plasmid miniprep followed by PCR fragment amplification and purification. PCR primers are VF et VR2 from the iGEM site.Many bacteria cultures in order to have something relevant. Followed by plasmid extraction, PCR amplification and gel purification.PCR on purified fragments were good.Resuspension of operators fragments and digestion of operators fragments, lacI Part, Pigment Part</DIV><DIV class="image" id="Week7Pic" onclick="ShowHide(week7)"><H1 class="week">Week 9 - 10 :</H1></DIV><DIV id="week7" style="display: none;"><P>Parts with different operators-pigment combination were produced by digestion/ligation of our biobrick parts previously created in order to complete our collection. This was made in &lt;i&gt;E. coli.&lt;/i&gt; We didn’t succeed to obtain all our construction when we transform B. subtilis with our constructs with a plasmid backbone pBS1C</P></DIV><DIV class="image" id="Week8Pic" onclick="ShowHide(week8)"><H1 class="week">Week 11 - End :</H1></DIV><DIV id="week8" style="display: none;">
Many cultures of B. subtilis transformed with all of our constructs were made. Because of some technical issues we had to change our B. subtilis stock and strain.At a point we suspected some of our restriction enzymes not to work properly causing the lack of bacterial growth…</DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></BODY></HTML>