<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_AFCM-Egypt_Description skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><DIV id="top_title"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:AFCM-Egypt/Description</SPAN></H1></DIV><DIV id="HQ_page"><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><P><TITLE>A F C M</TITLE></P><DIV class="container" style="width:95%"><DIV class="navbar-header" style="width:100%"><BUTTON type="button" id="navbar-toggle" class="navbar-toggle collapsed" data-toggle="collapse" data-target="#navbar" aria-expanded="false" aria-controls="navbar"><SPAN class="sr-only">Toggle navigation</SPAN></BUTTON><DIV class="collapse navbar-collapse" id="navbar"><DIV class="row text-center" style="margin:auto;float:none;width:fit-content"><UL id="menu" style="float: none;
    margin-right: auto;
    margin-left: auto;
    margin-top: 0px;
    display: list-item;"><LI class="HOME"><A href="https://2017.igem.org/Team:AFCM-Egypt">HOME</A></LI><LI class="Meet"><A>TEAM </A></LI><LI><A href="https://2017.igem.org/Team:AFCM-Egypt/Meet_the_team">Meet The Team</A></LI><LI><A href="https://2017.igem.org/Team:AFCM-Egypt/Collaborations">Collaborations</A></LI></UL><LI class="PROJECT"><A>PROJECT </A></LI><LI><A href="https://2017.igem.org/Team:AFCM-Egypt/Description">Project Description</A></LI><LI><A href="https://2017.igem.org/Team:AFCM-Egypt/Experiments">Experiment</A></LI><LI><A href="https://2017.igem.org/Team:AFCM-Egypt/Results">Results</A></LI><LI><A href="https://2017.igem.org/Team:AFCM-Egypt/Demonstrate">Proof of concept</A></LI><LI class="PARTS"><A>PARTS </A></LI><LI><A href="https://2017.igem.org/Team:AFCM-Egypt/Basic_Part">Parts</A></LI><LI><A href="https://2017.igem.org/Team:AFCM-Egypt/Basic_Part">Basic Parts</A></LI><LI><A href="https://2017.igem.org/Team:AFCM-Egypt/Composite_Part">Composite Parts</A></LI><LI><A href="https://2017.igem.org/Team:AFCM-Egypt/Improve">Improve</A></LI><LI class="Design"><A href="https://2017.igem.org/Team:AFCM-Egypt/Design">Design</A></LI><LI class="MODELING"><A href="https://2017.igem.org/Team:AFCM-Egypt/Model">MODELING</A></LI><LI class="SOFTWARE"><A href="https://2017.igem.org/Team:AFCM-Egypt/Software">Software</A></LI><LI class="SAFETY"><A href="https://2017.igem.org/Team:AFCM-Egypt/Safety">SAFETY</A></LI><LI class="HUMAN"><A>HUMAN PRACTICE </A></LI><LI><A href="https://2017.igem.org/Team:AFCM-Egypt/HP/Silver">Silver</A></LI><LI><A href="https://2017.igem.org/Team:AFCM-Egypt/HP/Gold">Gold</A></LI><LI><A href="https://2017.igem.org/Team:AFCM-Egypt/HP/Integrated">Integrated</A></LI><LI class="ATTRIBUTIONS"><A href="https://2017.igem.org/Team:AFCM-Egypt/Attributions">ATTRIBUTIONS</A></LI><LI class="CRITERIA"><A href="https://2017.igem.org/Team:AFCM-Egypt/Medal_criteria">MEDAL CRITERIA</A></LI></DIV></DIV></DIV></DIV><TITLE>A F C M</TITLE><SECTION class="price-table"><DIV class="container section-wrapper"><DIV class="row" style="margin-bottom: 50px;"><DIV class="col-md-6 text-center" style="float: none; margin: auto; width: 85%; background-color: rgba(0, 0, 0, 0.4);"><P style="text-align: center;"> </P><P style="text-align: center;"><SPAN style="font-family: 'times new roman', times; color: #ffffff;"><STRONG><SPAN style="font-size: 28pt; color: #5f9ea0;">Introduction</SPAN></STRONG></SPAN><SPAN style="font-family: 'times new roman', times; font-size: 14pt; color: #ffffff;">Liver cancer is a leading cause of cancer deaths worldwide, accounting for more than</SPAN><SPAN style="font-family: 'times new roman', times; font-size: 14pt; color: #ffffff;">600,000 deaths each year. The American Cancer Society’s estimates -of primary liver</SPAN><SPAN style="font-family: 'times new roman', times; font-size: 14pt; color: #ffffff;">cancer and intrahepatic bile duct cancer in the United States for 2016 are about 39,230</SPAN><SPAN style="font-family: 'times new roman', times; font-size: 14pt; color: #ffffff;">of newly diagnosed cases - and 27,170 died people of these cancers.(i)</SPAN><SPAN style="color: #ffffff;"><SPAN style="font-family: 'times new roman', times; font-size: 14pt;"><STRONG>In Egypt,</STRONG> liver </SPAN><SPAN style="font-family: 'times new roman', times; font-size: 14pt;">cancer is a serious if not the most serious cancer problem. It is ranked the first among</SPAN></SPAN><SPAN style="font-family: 'times new roman', times; font-size: 14pt; color: #ffffff;">cancers in males (33.6%) and next to breast cancer among females based upon results</SPAN><SPAN style="color: #ffffff;"><SPAN style="font-family: 'times new roman', times; font-size: 14pt;">of  National Cancer Registry Program (NCRP 2008-2011).(ii) </SPAN><SPAN style="font-family: 'times new roman', times; font-size: 14pt;">The rising rates of HCC in </SPAN><SPAN style="font-family: 'times new roman', times; font-size: 14pt;">Egypt are due to the high prevalence of hepatitis B virus (HBV) and hepatitis C virus </SPAN></SPAN><SPAN style="font-family: 'times new roman', times; font-size: 14pt; color: #ffffff;">infection (HCV) among Egyptian populationiii. Therefore; we need effective strategies</SPAN><SPAN style="font-family: 'times new roman', times; font-size: 14pt; color: #ffffff;">for early detection and better management of HCC which will be of great value in</SPAN><SPAN style="color: #ffffff;"><SPAN style="font-family: 'times new roman', times; font-size: 14pt;">developing countries with limited resources and high incidence rates of HCC, such as </SPAN><SPAN style="font-family: 'times new roman', times; font-size: 14pt;">Egypt.</SPAN></SPAN><SPAN style="font-family: 'times new roman', times; font-size: 14pt; color: #ffffff;">It is well known that the human genome is actively transcribed; however, </SPAN><SPAN style="font-family: 'times new roman', times; font-size: 14pt; color: #ffffff;">there are only about 20, 000 protein-coding genes, accounting for about 2%</SPAN><SPAN style="font-family: 'times new roman', times; font-size: 14pt; color: #ffffff;">of the genome, and the rest of the transcripts are non-coding RNAs</SPAN><SPAN style="font-family: 'times new roman', times; font-size: 14pt; color: #ffffff;">including microRNAs and long non-coding RNAs (lncRNAs). LncRNAs </SPAN><SPAN style="font-family: 'times new roman', times; font-size: 14pt; color: #ffffff;">play an important part in the regulation of gene expression, including </SPAN><SPAN style="font-family: 'times new roman', times; font-size: 14pt; color: #ffffff;">chromatin modification, transcription and post-transcriptional processing. It </SPAN><SPAN style="font-family: 'times new roman', times; font-size: 14pt; color: #ffffff;">has been confirmed that dysregulation of lncRNAs is accompanied by a </SPAN><SPAN style="font-family: 'times new roman', times; font-size: 14pt; color: #ffffff;">number of human pathological diseases, mainly tumors.(iv)</SPAN><SPAN style="font-family: 'times new roman', times; font-size: 14pt; color: #ffffff;">A most commonly used approach for gene functional study is knockdown </SPAN><SPAN style="font-family: 'times new roman', times; font-size: 14pt; color: #ffffff;">by RNA inference (RNAi). However, many lncRNAs are localized to the </SPAN><SPAN style="color: #ffffff;"><SPAN style="font-family: 'times new roman', times; font-size: 14pt;">nucleus (v</SPAN><SPAN style="font-family: 'times new roman', times; font-size: 14pt;">), which can make it difficult to achieve robust knockdown. Thus, </SPAN></SPAN><SPAN style="font-family: 'times new roman', times; font-size: 14pt; color: #ffffff;">genetic editing at the genomic level provides a better alternative because it </SPAN><SPAN style="font-family: 'times new roman', times; font-size: 14pt; color: #ffffff;">targets the genomic DNA.  There are several genetic tools available for this </SPAN><SPAN style="font-family: 'times new roman', times; font-size: 14pt; color: #ffffff;">purpose, including zinc finger nuclease (ZFN) and transcription activationlike </SPAN><SPAN style="font-family: 'times new roman', times; font-size: 14pt; color: #ffffff;">element nuclease (TALEN) (vi). Recently, a novel genetic engineering</SPAN><SPAN style="font-family: 'times new roman', times; font-size: 14pt; color: #ffffff;">tool called clustered regularly interspaced short palindromic repeats </SPAN><SPAN style="font-family: 'times new roman', times; font-size: 14pt; color: #ffffff;">(CRISPR)/CRISPR‐associated (Cas) system is more advanced because of  </SPAN><SPAN style="font-family: 'times new roman', times; font-size: 14pt; color: #ffffff;">easy generation and high efficiency of gene targeting. Importantly, it only</SPAN><SPAN style="font-family: 'times new roman', times; font-size: 14pt; color: #ffffff;">requires changing the sequence of the guide RNA (gRNA). </SPAN><SPAN style="color: #ffffff;"><SPAN style="font-family: 'times new roman', times; font-size: 14pt;">CRISPR/Cas9 has rapidly gained popularity due to its superior simplicity</SPAN><SPAN style="font-family: 'times new roman', times; font-size: 14pt;">(</SPAN><SPAN style="font-family: 'times new roman', times; font-size: 14pt;">vii). In this system, a single guide RNA (sgRNA) complexes with Cas9 </SPAN></SPAN><SPAN style="font-family: 'times new roman', times; font-size: 14pt; color: #ffffff;">nuclease, which can recognize a variable 20-nucleotide target sequence </SPAN><SPAN style="font-family: 'times new roman', times; font-size: 14pt; color: #ffffff;">adjacent to a 5′-NGG-3′ protospacer adjacent motif (PAM) and introduce a </SPAN><SPAN style="font-family: 'times new roman', times; font-size: 14pt; color: #ffffff;">DSB in the target DNA (viii). The induced DSB (DNA double stranded </SPAN><SPAN style="font-family: 'times new roman', times; font-size: 14pt; color: #ffffff;">break) then triggers DNA repair process mainly via two distinct </SPAN><SPAN style="font-family: 'times new roman', times; font-size: 14pt; color: #ffffff;">mechanisms, namely, the non-homologous end joining (NHEJ) and the </SPAN><SPAN style="font-family: 'times new roman', times; font-size: 14pt; color: #ffffff;">homology-directed repair (HDR) pathways.</SPAN><SPAN style="font-family: 'times new roman', times; font-size: 14pt; color: #ffffff;">CRISPR loci and their associated cas (CRISPR-associated) genes provide adaptive </SPAN><SPAN style="font-family: 'times new roman', times; font-size: 14pt; color: #ffffff;">immunity against viruses (phages) and other mobile genetic elements in bacteria and </SPAN><SPAN style="font-family: 'times new roman', times; font-size: 14pt; color: #ffffff;">archaea. While most of the early work has largely been dominated by examples of </SPAN><SPAN style="font-family: 'times new roman', times; font-size: 14pt; color: #ffffff;">CRISPR-Cas systems directing the cleavage of phage or plasmid DNA, recent studies </SPAN><SPAN style="font-family: 'times new roman', times; font-size: 14pt; color: #ffffff;">have revealed a more complex landscape where CRISPR-Cas loci might be involved </SPAN><SPAN style="font-family: 'times new roman', times; font-size: 14pt; color: #ffffff;">in gene regulation. </SPAN><SPAN style="font-family: 'times new roman', times; font-size: 14pt; color: #ffffff;">In human cells, efficient knock-in of foreign DNA into a selected genomic </SPAN><SPAN style="font-family: 'times new roman', times; font-size: 14pt; color: #ffffff;">locus is anticipated to facilitate various applications, ranging from gene </SPAN><SPAN style="font-family: 'times new roman', times; font-size: 14pt; color: #ffffff;">function study to therapeutic genome editing. Currently, most studies have </SPAN><SPAN style="color: #ffffff;"><SPAN style="font-family: 'times new roman', times; font-size: 14pt;">focused on HDR-based strategies (ix</SPAN><SPAN style="font-family: 'times new roman', times; font-size: 14pt;">,</SPAN><SPAN style="font-family: 'times new roman', times; font-size: 14pt;">x</SPAN><SPAN style="font-family: 'times new roman', times; font-size: 14pt;">).In a recent study by Merkle et al., the </SPAN><SPAN style="font-family: 'times new roman', times; font-size: 14pt;">efficiency of CRISPR/Cas9-induced HDR-mediated knock-in was estimated</SPAN></SPAN><SPAN style="color: #ffffff;"><SPAN style="font-family: 'times new roman', times; font-size: 14pt;">to be around 1 × 10−5 without pre-selection (xi</SPAN><SPAN style="font-family: 'times new roman', times; font-size: 14pt;">,</SPAN><SPAN style="font-family: 'times new roman', times; font-size: 14pt;">xii). </SPAN></SPAN><SPAN style="font-family: 'times new roman', times; font-size: 14pt; color: #ffffff;">Scientists constructed a universal reporter system, by targeting </SPAN><SPAN style="font-family: 'times new roman', times; font-size: 14pt; color: #ffffff;">the GAPDH locus in human genome with a promoterless fluorescent </SPAN><SPAN style="font-family: 'times new roman', times; font-size: 14pt; color: #ffffff;">reporter. Through systematic investigation into the potentials of both HDR</SPAN><SPAN style="font-family: 'times new roman', times; font-size: 14pt; color: #ffffff;">and NHEJ repair in mediating CRISPR/Cas9-induced reporter integration, </SPAN><SPAN style="font-family: 'times new roman', times; font-size: 14pt; color: #ffffff;">researchers demonstrated that CRISPR/Cas9-induced NHEJ can mediate </SPAN><SPAN style="font-family: 'times new roman', times; font-size: 14pt; color: #ffffff;">reporter  knock-in more efficiently than HDR-based strategy, in various  </SPAN><SPAN style="font-family: 'times new roman', times; font-size: 14pt; color: #ffffff;">human cells types including human ESCs. This finding paves a new path for </SPAN><SPAN style="font-family: 'times new roman', times; font-size: 14pt; color: #ffffff;">efficient genome editing in human ESCs and somatic cells, and it offers a </SPAN><SPAN style="color: #ffffff;"><SPAN style="font-family: 'times new roman', times; font-size: 14pt;">great potential in their subsequent applications.(</SPAN><SPAN style="font-family: 'times new roman', times; font-size: 14pt;">xiii)</SPAN></SPAN><SPAN style="font-family: 'times new roman', times; font-size: 28pt; color: #5f9ea0;">Aim of the work</SPAN><SPAN style="font-family: 'times new roman', times; font-size: 14pt; color: #ffffff;">   1- To analyze circRNA and disease databases to select significantly relevant</SPAN><SPAN style="font-family: 'times new roman', times; font-size: 14pt; color: #ffffff;">circRNA for HCC.</SPAN><SPAN style="font-family: 'times new roman', times; font-size: 14pt; color: #ffffff;">   2- To analyze circRNA- miRNA interaction databases to retrieve competing</SPAN><SPAN style="font-family: 'times new roman', times; font-size: 14pt; color: #ffffff;">endogenous RNA specific for HCC.</SPAN><SPAN style="font-family: 'times new roman', times; font-size: 14pt; color: #ffffff;">  3- To characterize the expression of the serum cirRNA-associated ceRNA</SPAN><SPAN style="font-family: 'times new roman', times; font-size: 14pt; color: #ffffff;">genes in HepG2 cell line to evaluate their role in pathogenesis of HCC.</SPAN><SPAN style="font-family: 'times new roman', times; font-size: 14pt; color: #ffffff;">4- To compare beteen the efficacy of a CRISPR &amp; non-CRISPR based</SPAN><SPAN style="font-family: 'times new roman', times; font-size: 14pt; color: #ffffff;">synthetic circuit on modulating cirRNA-associated ceRNA related HCC</SPAN><SPAN style="font-family: 'times new roman', times; font-size: 14pt; color: #ffffff;">expression using HepG2 cell line.</SPAN></P><P style="text-align: center;"><SPAN style="color: #5f9ea0;"><STRONG><SPAN style="font-family: 'times new roman', times; font-size: 28pt;">What is the role of ceRNA?</SPAN></STRONG></SPAN></P><P style="text-align: center;"><SPAN style="color: #ffffff; font-size: 11pt;"><STRONG><SPAN style="font-family: 'times new roman', times;">fig(1): components of CIRcRNA-associated CeRNA</SPAN></STRONG></SPAN></P><P style="text-align: center;"><SPAN style="font-size: 14pt; color: #ffffff;"><SPAN style="font-family: 'times new roman', times;">Given the intricate interplay among the diverse RNA species, our team this year will disseminate its proposal: being a modification for last year's with the application of the &quot;Crispr gene-editing tool&quot;.Well what is the story? And how does the deregulation affect cancer cell growth?RNA transcripts, like the long non-coding RNA and circular RNA, act as competing endogenous RNAs (ceRNAs) or natural microRNA sponges they communicate with and co-regulate each other by competing for binding to shared microRNAs, a family of small non-coding RNAs thatare important post-transcriptional regulators of gene expression. Such competing endogenous RNAs (ceRNAs) regulate the distribution of miRNA molecules on their targets and thereby impose an additional level of post-transcriptional regulation. On the same note, this regulation is scientifically effective way in manipulating critical roles in both normal physiology and tumorigenesis. (As shown in the below figure)Therefore, Competitive endogenous RNAs (ceRNAs) act as molecular sponges for a microRNA through their miRNA binding sites (also referred to as miRNA response elements, MRE), thereby de-repressing all target genes of the respective miRNA family.</SPAN></SPAN></P><P style="text-align: center;"><SPAN style="font-size: 14pt; color: #ffffff;"><SPAN style="font-family: 'times new roman', times;">  CircRNA         has-cric-0000064     Target geneTRIM   miRNA      mirna-1825miRNAs were revealed to repress their target genes via binding imperfectly to miRNA response elements (MREs) on the 3 ′ untranslated regions (3 ′ -UTRs) of target RNA transcripts and causing to reduced expression of their target proteins either by mRNA breakdown ortranslational repression. Because each miRNA could target hundreds of genes and vice versa, each gene can be targeted by many miRNAs; such molecules are critically mentioned in the fine-tuned regulation of gene expression. RNAs functioning as in this course are named ceRNA. Some coding RNAs, pseudogenes, ncRNAs, and circular RNAs could work as ceRNAs. CeRNAs having common MREs can compete for binding of miRNA. They delineate a concealed RNA B jargon: a network of interactions of different RNA types that regulate gene expression. It was suggested that these ceRNAs can talk to each other via their ability to compete for binding of miRNA. This cross-talk produces comprehensive cis - and Trans -orga- nizing communication acrossall the transcriptome. Moreover, ceRNA networks further depend on the subcellular dispersionand tissue particularity of RNAs and miRNAs found in a specific cell type at a specific. The concentration of miRNAs is an important factor for ceRNA activity. If there are a less number of miRNAs than their targets, the ceRNA activity is reduced as the targets will remain largelyunrepressed. Also, if there are more miRNAs as compared to their targets, there would have been no cross- regulation due to almost a universal repression of the targets.</SPAN></SPAN></P><P style="text-align: center;"><SPAN style="font-size: 14pt; color: #ffffff;"><SPAN style="font-family: 'times new roman', times;"><STRONG><SPAN style="font-size: 11pt;">fig(2): Principle of CeRNAs</SPAN></STRONG></SPAN></SPAN></P><P style="text-align: center;"><SPAN style="color: #5f9ea0;"><SPAN style="font-size: 14pt;"><SPAN style="font-family: 'times new roman', times;"><SPAN style="font-size: 28pt;">Then, why ceRNA in Hepatocellular carcinoma?</SPAN><SPAN style="color: #ffffff;">Hepatocellular carcinoma (HCC) is originally the fifth most common cancer worldwide and the</SPAN><SPAN style="color: #ffffff;"> third cause of cancer mortality. The Eastern and South-Asia, Middle and Western Africa are considered</SPAN><SPAN style="color: #ffffff;"> to be of high incidence rate. In Egypt, HCC is one of the health problems facing the</SPAN><SPAN style="color: #ffffff;">health authorities. Egypt is the sixth largest country in the middle east and Arab world, it is the</SPAN><SPAN style="color: #ffffff;"> Third largest country in Africa, it is the fifteenth most populous nation in the world about 90 million</SPAN><SPAN style="color: #ffffff;"> inhabitants. According to a study published by El-Zayadi et al: they reported almost 2 folds increase</SPAN><SPAN style="color: #ffffff;"> in HCC among chronic liver disease patients over a decade. Also, according to Ibrahim et al, </SPAN><SPAN style="color: #ffffff;">HCC is the first most common cancer in males and second most common cancer in females. Given this high prevalence of a disease-statistics, we followed the lead of Ji-hang Yuan et al, through the ceRNA world, who demonstrated that lncRNA-ATB acts as a key regulator of TGF-β signaling pathways and  revealed roles </SPAN><SPAN style="color: #ffffff;">of TGF-β in regulating long noncoding RNAs. </SPAN></SPAN></SPAN><SPAN style="font-size: 14pt;"><SPAN style="font-family: 'times new roman', times;"><SPAN style="color: #ffffff;">The findings of this study have  significant implications  </SPAN><SPAN style="color: #ffffff;">regarding our understanding of HCC metastasis pathogenesis. As direct targets of lncRNA-ATB, miR-200-ZEB</SPAN><SPAN style="color: #ffffff;"> and IL-11 mediated the role of lncRNAATB in local invasion and distant colonization, respectively.</SPAN><SPAN style="color: #ffffff;"> Similarly, Zhang J et al showed the lncRNA expression patterns and a complex ceRNA network</SPAN><SPAN style="color: #ffffff;"> in HCC, and identified a complex cancer specific ceRNA network, which includes 14 lncRNAs</SPAN><SPAN style="color: #ffffff;"> and 17 miRNAs in HCC.</SPAN></SPAN></SPAN></SPAN></P><P style="text-align: center;"><SPAN style="font-size: 14pt; color: #5f9ea0;"><SPAN style="font-family: 'times new roman', times;"><SPAN style="font-size: 28pt;">What is next?</SPAN><SPAN style="color: #ffffff;">Via our bioinformatics analysis, we led through into a long process of dry lab work, deciding a suitable and effectively tested regulatory pathway to do our hypothesis. Our modeled ceRNA contained, as provided </SPAN><SPAN style="color: #ffffff;">in the figure, the circular RNA (has-cric-0000064 ) competing for shared microRna </SPAN><SPAN style="color: #ffffff;">(mir-1825) and sequestrate it within the cell as they have MREs (microRNA sponge); </SPAN><SPAN style="color: #ffffff;">lastly deregulating our target gene (TRIM) And yes! A sophisticated and effective molecular level control.</SPAN><SPAN style="font-size: 28pt;">What is our aim for a ceRNA</SPAN>  <SPAN style="color: #ffffff;"> <SPAN style="font-size: 15pt;"> 1- To analyze circRNA and disease databases to select significantly</SPAN></SPAN><SPAN style="color: #ffffff; font-size: 15pt;">relevant circRNA for HCC.</SPAN><SPAN style="color: #ffffff; font-size: 15pt;">2- To analyze circRNA-miRNA interaction databases to retrieve</SPAN><SPAN style="color: #ffffff; font-size: 15pt;">competing endogenous RNA specific for HCC.</SPAN><SPAN style="color: #ffffff; font-size: 15pt;">3- To characterize the expression of cirRNA-associated ceRNA</SPAN><SPAN style="color: #ffffff; font-size: 15pt;">genes in HepG2 cell line to evaluate their role in</SPAN><SPAN style="color: #ffffff; font-size: 15pt;">pathogenesis of HCC.</SPAN><SPAN style="color: #ffffff; font-size: 15pt;">         4- To compare between the efficacy of knocking in of circular RNA using</SPAN><SPAN style="color: #ffffff; font-size: 15pt;">synthetic circuits and crisper techniques.</SPAN></SPAN></SPAN></P><P style="text-align: center;"><SPAN style="font-size: 14pt; color: #5f9ea0;"><SPAN style="font-family: 'times new roman', times;"><SPAN style="font-size: 28pt;">CRISPR</SPAN><SPAN style="color: #ffffff;">In order to simply explain to you what CRISPR is and what we can use it in or how does it solve</SPAN><SPAN style="color: #ffffff;"> a problem, we have to briefly tell you the headache that’s been bothering us. Nowadays we have major</SPAN><SPAN style="color: #ffffff;"> medical therapies that we are using to manage disease processes and its pathogenic or misregulated</SPAN><SPAN style="color: #ffffff;">proteins or molecules associated with the disease. Worth to mention, we know that these proteins</SPAN><SPAN style="color: #ffffff;"> are encoded and affected by changes in the genes and their sequences, yet we haven’t got enough</SPAN><SPAN style="color: #ffffff;">treatment therapies on that scale. So, we need to improve our impact on that molecular level by using</SPAN><SPAN style="color: #ffffff;"> genome (gene) editing tools such as TALENs, ZFN and of course our precious CRISPR/CAS9.</SPAN><SPAN style="color: #ffffff;">CRISPR, a new genome editing tool, but why CRISPR specifically rather than the other tools. In a matter of</SPAN><SPAN style="color: #ffffff;"> fact this question is answered plainly, because it fulfills the criteria that we are looking for. These criteria</SPAN><SPAN style="color: #ffffff;"> are that it edits the genomes with exceptional precision, efficacy and flexibility. Also we have to</SPAN><SPAN style="color: #ffffff;"> mention that it wasn’t invented by scientists but that it's naturally occurring in bacteria </SPAN><SPAN style="color: #ffffff;">(Streptococcus pyogenes) as way of self-defense against viruses that prey on bacteria.</SPAN><SPAN style="color: #ffffff;">When we undeclared the mechanism of defense of those bacteria we understood how CRISPR/CAS9 works</SPAN><SPAN style="color: #ffffff;">and that it's a part of the bacteria's</SPAN> </SPAN></SPAN><SPAN style="font-size: 14pt; color: #ffffff;"><SPAN style="font-family: 'times new roman', times;">immune system. The CRISPR part in the bacteria would keep pieces andparts of those vicious viruses around it so that it can recognize them the next time they encounter. Not only that, CRISPER then ignites CAS9 which is one of its famous associated proteins to assault the violating virus by snipping parts of their DNA at specific sites. Usually the encoding genes for CAS are settled close to those of CRISPR. Now that we understood what CRISPR is and how it works, we can put a simple map in our heads by imagining that CRISPR is a collection of DNAsequence that tells it associates (helpers) what to do. One of CRISPR'S loyal comrade is CAS9 that is responsible for snipping the DNA at specific sites that CRISPR assign's it to by guiding it by providing it by (logically called) guide RNA. We start to wonder how we are going to apply it in medical therapy. It's really obvious that we are going to guide our tool to snip the unwanted DNA in order to regain the normal regulation of the genes and proteins and so that the disease is cured. That is going to happen by inserting aguide RNA to match the undesirable gene and let the CAS9 snip it off. Always put in mind that the DNA is a long sequence of bases, we can't just insert all of the genome for CAS9 to detect and snip, it doesn’t work like that. In fact, CAS9 can only take up to about 20 base long sequenceto be recognized. Ordering the matching guide RNA after using an online tool to design the targeted sequence. When scientists put the editing tools in trial the other tools were provento be more specific yet, there was a huge downside, Scientists have to create a designed protein each time and create several variations beforefinding one that might work. CRISPR saved all the time and is also more likely to work. Besides that it can be used in all kind of organisms. In order to use such technique we had to totally understand how it worksand we couldn’t have done this unless for those who started using it and simplified the ideas for us like Genetic Home References.xiv We can't neglect those who matured this technique and helped to enlighten our minds to carry on with our theory and experiments. We first read that, Jie Wang et al. mentioned in their paper about using CRISPR/CAS9 to inhibit hepatitis B virus replication that, these results suggested that CRISPR/Cas9 system could efficiently destroy HBV expressing templates (genotypes A-D) without apparent cytotoxicity. It may be a potential approach for eradication of persistent HBV cccDNA in chronic HBV infection patients.xv  Also, we found that, Panpan Hou et al. published that they found two sites in CXCR4 that can be targeted effectively and specifically by the CRISPR/Cas9 system, resulting in co-receptor CXCR4 ablation. We also show that the modified cells are resistant to X4 type HIV-1 infectionand this may provide us with an alternative approach of gene therapy for treating AIDS. Although lenti-CRISPR/Cas9 provides powerful means to disrupt CXCR4, the optimized delivery methods using adenovirusneed to be explored as to further improve their specificity and minimize the concern for therapeutic safety. Due to the variation in viral  infection, co-disruption of CCR5 and CXCR4 should be tested using lentioradenovirus mediated CRISPR/Cas9 system in the future. Furthermore, the successful disruption of CXCR4 in Rhesus macaque CD4+ T cells may accelerate gene therapy studies </SPAN></SPAN><SPAN style="font-size: 14pt; color: #ffffff;"><SPAN style="font-family: 'times new roman', times;">for AIDS in nonhuman primate models.xvi </SPAN></SPAN></P><P style="text-align: center;"><SPAN style="font-size: 14pt; color: #ffffff;"><SPAN style="font-family: 'times new roman', times;">As well as, Kit-San Yuen et al. CRISPR/Cas9-mediated genome editing of Epstein–Barr virus in human cells published that CRISPR/Cas9-mediated editing of the EBV genome</SPAN></SPAN><SPAN style="font-size: 14pt; color: #ffffff;"><SPAN style="font-family: 'times new roman', times;"> in human cells provides a new technology platform for the genetic study of EBV. In particular, it will facilitate rapid analysis of the roles of individual EBV genes in viral replication,persistence and transformation. Compared with BAC clones, it will be more difficult to obtain large amounts of CRISPR/Cas9-edited viral DNA for the assessment of genome integrity by restriction mapping. However, the new method also has several advantages and is highlycomplementary to the existing BAC technology. First, CRISPR/Cas9 technology is applicable to any EBV strain, whereas EBV BACs are currently available for only three strains. Second, CRISPR/Cas9-mediated editing is performed completely in human cells, whereas EBV BACs are constructed and produced in Escherichia coli.xvii</SPAN></SPAN></P><P style="text-align: center;"><STRONG><SPAN style="color: #ffffff;">fig(3): Principle of CRISPR </SPAN></STRONG></P><P style="text-align: center;"><SPAN style="color: #5f9ea0;"><STRONG><SPAN style="font-family: 'times new roman', times; font-size: 28pt;">IGEM Hypothesis</SPAN></STRONG></SPAN></P><P style="text-align: center;"><SPAN style="color: #ffffff;"><STRONG><SPAN style="font-family: 'times new roman', times; font-size: 28pt;"><SPAN style="color: #5f9ea0;">References</SPAN></SPAN></STRONG></SPAN></P><P style="text-align: center;"><SPAN style="color: #ffffff;">iAmerican Cancer Society. Cancer Facts &amp; Figures 2016 . Atlanta, Ga: American </SPAN><SPAN style="color: #ffffff;">CancerSociety; 2015.</SPAN><SPAN style="color: #ffffff;">iiAmal S. Ibrahim, Hussein M. Khaled, Nabiel NH Mikhail, Hoda Baraka, and</SPAN><SPAN style="color: #ffffff;">HossamKamel, “Cancer Incidence in Egypt: Results of the National Population-Based</SPAN><SPAN style="color: #ffffff;">Cancer Registry Program,” Journal of Cancer Epidemiology, vol. 2014, Article ID 437971,</SPAN><SPAN style="color: #ffffff;">18 pages, 2014. doi:10.1155/2014/437971</SPAN><SPAN style="color: #ffffff;">iiiAbdelgawad, I.A.; Mossallam, G.I.; Radwan, N.H.; Elzawahry, H.M. and Elhifnawy,</SPAN><SPAN style="color: #ffffff;">N.M.(2013):Can Glypican3 be diagnostic for early hepatocellular carcinoma among</SPAN><SPAN style="color: #ffffff;">Egyptian patients?. Asian Pac J Cancer Prev.; 14(12):7345-9.</SPAN><SPAN style="color: #ffffff;">YU, FU-JUN et al. “Long Non-Coding RNAs and Hepatocellular Carcinoma.” Molecular and iv</SPAN><SPAN style="color: #ffffff;">Clinical Oncology 3.1 (2015): 13–17. PMC. Web. 9 Feb. 2017.</SPAN><SPAN style="color: #ffffff;">v Bhaya D. Davison M. Barrangou R. CRISPR-Cas systems in bacteria and archaea:</SPAN><SPAN style="color: #ffffff;">versatile small RNAs for adaptive defense and regulation Annu. Rev. Genet. 2011 45</SPAN><SPAN style="color: #ffffff;">273297</SPAN><SPAN style="color: #ffffff;">vii Cox D.B. Platt R.J. Zhang F. Therapeutic genome editing: prospects and challenges Nat.</SPAN><SPAN style="color: #ffffff;">Med. 2015 21 121131</SPAN><SPAN style="color: #ffffff;">viii Xiangjun He, Chunlai Tan, Feng Wang, Yaofeng Wang, Rui Zhou, Dexuan Cui, Wenxing</SPAN><SPAN style="color: #ffffff;">You, Hui Zhao, Jianwei Ren, Bo Feng; Knock-in of large reporter genes in human cells via</SPAN></P><P style="text-align: center;"><SPAN style="color: #ffffff;">CRISPR/Cas9-induced homology-dependent and independent DNA repair. Nucl Acids</SPAN><SPAN style="color: #ffffff;">Res 2016; 44 (9): e85. doi: 10.1093/nar/gkw064</SPAN><SPAN style="color: #ffffff;">ix Kan Y. Ruis B. Lin S. Hendrickson E.A. The mechanism of gene targeting in human</SPAN><SPAN style="color: #ffffff;">somatic cells PLoS Genet. 2014 10 e1004251</SPAN><SPAN style="color: #ffffff;">x Mao Z. Bozzella M. Seluanov A. Gorbunova V. Comparison of nonhomologous end</SPAN><SPAN style="color: #ffffff;">joining and homologous recombination in human cells DNA Repair (Amst.) 2008 7</SPAN><SPAN style="color: #ffffff;">17651771</SPAN><SPAN style="color: #ffffff;">xi Merkle F.T. Neuhausser W.M. Santos D. Valen E. Gagnon J.A. Maas K. Sandoe J. Schier</SPAN><SPAN style="color: #ffffff;">A.F. Eggan K. Efficient CRISPR-Cas9-mediated generation of knockin human pluripotent</SPAN><SPAN style="color: #ffffff;">stem cells lacking undesired mutations at the targeted locus Cell Rep. 2015 11 875883.</SPAN><SPAN style="color: #ffffff;">xii Rong Z. Zhu S. Xu Y. Fu X. Homologous recombination in human embryonic stem</SPAN><SPAN style="color: #ffffff;">cells using CRISPR/Cas9 nickase and a long DNA donor template Protein Cell 2014 5</SPAN><SPAN style="color: #ffffff;">258260</SPAN><SPAN style="color: #ffffff;">xiii Thomson J.A. Itskovitz-Eldor J. Shapiro S.S. Waknitz M.A. Swiergiel J.J. Marshall</SPAN><SPAN style="color: #ffffff;">V.S. Jones J.M. Embryonic stem cell lines derived from human blastocysts Science 1998</SPAN><SPAN style="color: #ffffff;">282 11451147</SPAN><SPAN style="color: #ffffff;">xiv gupta, r. M., &amp; musunuru, k. (2014, october 1). Expanding the genetic editing tool kit: zfns,</SPAN><SPAN style="color: #ffffff;">talens, and crispr-cas9. Journal of clinical investigation. American society for clinical</SPAN><SPAN style="color: #ffffff;">investigation. Hsu pd, lander es, zhang f. Development and applications of crispr-cas9 for</SPAN><SPAN style="color: #ffffff;">genome engineering. Cell. 2014 jun 5;157(6):1262-78. Doi:10.1016/j.cell.2014.05.010.</SPAN><SPAN style="color: #ffffff;">Review. Pubmed: 24906146. Free full-text available from pubmed central: pmc4343198.</SPAN><SPAN style="color: #ffffff;">Komor ac, badran ah, liu dr. Crispr-based technologies for the manipulation of eukaryotic</SPAN><SPAN style="color: #ffffff;">genomes. Cell. 2017 apr 20;169(3):559. Doi:10.1016/j.cell.2017.04.005. Pubmed: 28431253.</SPAN><SPAN style="color: #ffffff;">Lander es. The heroes of crispr. Cell. 2016 jan 14;164(1-2):18-28.</SPAN><SPAN style="color: #ffffff;">Doi:10.1016/j.cell.2015.12.041. Review. Pubmed: 26771483.</SPAN><SPAN style="color: #ffffff;">xv wang j, xu z-w, liu s, et al. Dual grnas guided crispr/cas9 system inhibits hepatitis b virus</SPAN><SPAN style="color: #ffffff;">replication. World journal of gastroenterology : wjg. 2015;21(32):9554-9565.</SPAN><SPAN style="color: #ffffff;">Doi:10.3748/wjg.v21.i32.9554.</SPAN><SPAN style="color: #ffffff;">xvi hou, p., chen, s., wang, s., yu, x., chen, y., jiang, m., … guo, d. (2015). Genome editing of</SPAN><SPAN style="color: #ffffff;">cxcr4 by crispr/cas9 confers cells resistant to hiv-1 infection. Scientific reports, 5, 15577.</SPAN><SPAN style="color: #ffffff;">xvii  yuen, k.-s., chan, c.-p., wong, n.-h. M., ho, c.-h., ho, t.-h., lei, t., … jin, d.-y. (2015).</SPAN><SPAN style="color: #ffffff;">Crispr/cas9-mediated genome editing of epstein-barr virus in human cells. Journal of general</SPAN><SPAN style="color: #ffffff;">virology, 96(pt_3), 626–636.</SPAN></P><P style="text-align: center;"><STRONG><SPAN style="font-family: 'times new roman', times; font-size: 28pt;"> </SPAN></STRONG></P></DIV></DIV></DIV></SECTION></DIV></DIV></DIV></DIV></DIV></BODY></HTML>