Team:AFCM-Egypt/Description
A F C M
Toggle navigation
HOME
TEAM
Meet The Team
Collaborations
PROJECT
Project Description
Experiment
Results
Proof of concept
PARTS
Parts
Basic Parts
Composite Parts
Improve
Design
MODELING
Software
SAFETY
HUMAN PRACTICE
Silver
Gold
Integrated
ATTRIBUTIONS
MEDAL CRITERIA
A F C M
 
Introduction
Liver cancer is a leading cause of cancer deaths worldwide, accounting for more than
600,000 deaths each year. The American Cancer Society’s estimates -of primary liver
cancer and intrahepatic bile duct cancer in the United States for 2016 are about 39,230
of newly diagnosed cases - and 27,170 died people of these cancers.(i)
In Egypt, liver cancer is a serious if not the most serious cancer problem. It is ranked the first among
cancers in males (33.6%) and next to breast cancer among females based upon resultsof  National Cancer
Registry Program (NCRP 2008-2011).(ii) The rising rates of HCC in Egypt are due to the high prevalence of
hepatitis B virus (HBV) and hepatitis C virus infection (HCV) among Egyptian populationiii.
Therefore; we need effective strategiesfor early detection and better management of HCC which will be
of great value indeveloping countries with limited resources and high incidence rates of HCC, such as Egypt.
It is well known that the human genome is actively transcribed; however, there are only about 20, 000 protein-coding
genes, accounting for about 2%of the genome, and the rest of the transcripts are non-coding RNAs
including microRNAs and long non-coding RNAs (lncRNAs). LncRNAs play an important part
in the regulation of gene expression, including chromatin modification, transcription
and post-transcriptional processing. It has been confirmed that dysregulation of lncRNAs is accompanied
by a number of human pathological diseases, mainly tumors.(iv)
A most commonly used approach for gene functional study is knockdown by RNA inference (RNAi).
However, many lncRNAs are localized to the nucleus (v), which can make it difficult to achieve robust
knockdown. Thus, genetic editing at the genomic level provides a better alternative because it targets the
genomic DNA.  There are several genetic tools available for this purpose, including zinc finger nuclease (ZFN)
and transcription activationlike element nuclease (TALEN) (vi). Recently, a novel genetic engineering
tool called clustered regularly interspaced short palindromic repeats (CRISPR)/CRISPR‐associated (Cas) system
is more advanced because of  easy generation and high efficiency of gene targeting. Importantly, it only
requires changing the sequence of the guide RNA (gRNA). CRISPR/Cas9 has rapidly gained popularity
due to its superior simplicity(vii). In this system, a single guide RNA (sgRNA) complexes with Cas9 nuclease,
which can recognize a variable 20-nucleotide target sequence adjacent to a 5′-NGG-3′ protospacer adjacent motif (PAM)
and introduce a DSB in the target DNA (viii). The induced DSB (DNA double stranded break)
then triggers DNA repair process mainly via two distinct mechanisms, namely, the non-homologous end joining (NHEJ)
and the homology-directed repair (HDR) pathways.
CRISPR loci and their associated cas (CRISPR-associated) genes provide adaptive immunity
against viruses (phages) and other mobile genetic elements in bacteria and archaea. While most of
the early work has largely been dominated by examples of CRISPR-Cas systems directing
the cleavage of phage or plasmid DNA, recent studies have revealed a more complex landscape
where CRISPR-Cas loci might be involved in gene regulation. In human cells,
efficient knock-in of foreign DNA into a selected genomic locus is anticipated to facilitate
various applications, ranging from gene function study to therapeutic genome editing. Currently,
most studies have focused on HDR-based strategies (ix,x).
In a recent study by Merkle et al., the efficiency of CRISPR/Cas9-induced HDR-mediated
knock-in was estimatedto be around 1 × 10−5 without pre-selection (xi,xii). Scientists constructed
a universal reporter system, by targeting the GAPDH locus in human genome with a
promoterless fluorescent reporter. Through systematic investigation into the potentials of both HDR
and NHEJ repair in mediating CRISPR/Cas9-induced reporter integration, researchers 
demonstrated that CRISPR/Cas9-induced NHEJ can mediate reporter  knock-in more
efficiently than HDR-based strategy, in various  human cells types including human ESCs.
This finding paves a new path for efficient genome editing in human ESCs and somatic cells,
and it offers a great potential in their subsequent applications.(xiii)
Aim of the work
     1- To analyze circRNA and disease databases to select significantly relevant
circRNA for HCC.
     2- To analyze circRNA- miRNA interaction databases to retrieve competing
endogenous RNA specific for HCC.
  3- To characterize the expression of the serum cirRNA-associated ceRNA
genes in HepG2 cell line to evaluate their role in pathogenesis of HCC.
4- To compare beteen the efficacy of a CRISPR & non-CRISPR based
synthetic circuit on modulating cirRNA-associated ceRNA related HCC
expression using HepG2 cell line.
What is the role of ceRNA?
fig(1): components of CIRcRNA-associated CeRNA
Given the intricate interplay among the diverse RNA species, our team this year will disseminate
its proposal: being a modification for last year's with the application of the "Crispr gene-editing tool".
Well what is the story? And how does the deregulation affect cancer cell growth?
RNA transcripts, like the long non-coding RNA and circular RNA, act as competing endogenous RNAs
(ceRNAs) or natural microRNA sponges they communicate with and co-regulate each other
by competing for binding to shared microRNAs, a family of small non-coding RNAs that
are important post-transcriptional regulators of gene expression. Such competing endogenous RNAs
(ceRNAs) regulate the distribution of miRNA molecules on their targets and thereby impose
an additional level of post-transcriptional regulation. On the same note, this regulation is scientifically
effective way in manipulating critical roles in both normal physiology and tumorigenesis.
(As shown in the below figure)
Therefore, Competitive endogenous RNAs (ceRNAs) act as molecular sponges for a microRNA
through their miRNA binding sites (also referred to as miRNA response elements, MRE), thereby
de-repressing all target genes of the respective miRNA family.
   CircRNA
                 has-cric-0000064
        Target gene
TRIM
    miRNA
          mirna-1825
miRNAs were revealed to repress their target genes via binding imperfectly to miRNA response 
elements (MREs) on the 3 ′ untranslated regions (3 ′ -UTRs) of target RNA transcripts and
causing to reduced expression of their target proteins either by mRNA breakdown or
translational repression. Because each miRNA could target hundreds of genes and vice versa,
each gene can be targeted by many miRNAs; such molecules are critically mentioned in the
fine-tuned regulation of gene expression. RNAs functioning as in this course are named
ceRNA. Some coding RNAs, pseudogenes, ncRNAs, and circular RNAs could work as ceRNAs.
CeRNAs having common MREs can compete for binding of miRNA. They delineate a concealed
RNA B jargon: a network of interactions of different RNA types that regulate gene expression.
It was suggested that these ceRNAs can talk to each other via their ability to compete for binding
of miRNA. This cross-talk produces comprehensive cis - and Trans -orga- nizing communication across
all the transcriptome. Moreover, ceRNA networks further depend on the subcellular dispersion
and tissue particularity of RNAs and miRNAs found in a specific cell type at a specific.
The concentration of miRNAs is an important factor for ceRNA activity. If there are a less number
of miRNAs than their targets, the ceRNA activity is reduced as the targets will remain largely
unrepressed. Also, if there are more miRNAs as compared to their targets, there would have been
no cross- regulation due to almost a universal repression of the targets.
fig(2): Principle of CeRNAs
Then, why ceRNA in Hepatocellular carcinoma?
Hepatocellular carcinoma (HCC) is originally the fifth most common cancer worldwide and the
third cause of cancer mortality. The Eastern and South-Asia, Middle and Western Africa are considered
to be of high incidence rate. In Egypt, HCC is one of the health problems facing the
health authorities. Egypt is the sixth largest country in the middle east and Arab world, it is the
Third largest country in Africa, it is the fifteenth most populous nation in the world about 90 million
inhabitants. According to a study published by El-Zayadi et al: they reported almost 2 folds increase
in HCC among chronic liver disease patients over a decade. Also, according to Ibrahim et al,
HCC is the first most common cancer in males and second most common cancer in females.
Given this high prevalence of a disease-statistics, we followed the lead of Ji-hang Yuan et al, through
the ceRNA world, who demonstrated that lncRNA-ATB acts as a key regulator of TGF-β
signaling pathways and  revealed roles of TGF-β in regulating long noncoding RNAs. 
The findings of this study have  significant implications  regarding our understanding
of HCC metastasis pathogenesis. As direct targets of lncRNA-ATB, miR-200-ZEB
and IL-11 mediated the role of lncRNAATB in local invasion and distant colonization, respectively.
Similarly, Zhang J et al showed the lncRNA expression patterns and a complex ceRNA network
in HCC, and identified a complex cancer specific ceRNA network, which includes 14 lncRNAs
and 17 miRNAs in HCC.
What is next?
Via our bioinformatics analysis, we led through into a long process of dry lab work, deciding
a suitable and effectively tested regulatory pathway to do our hypothesis. Our modeled ceRNA contained,
as provided in the figure, the circular RNA (has-cric-0000064 ) competing for
shared microRna (mir-1825) and sequestrate it within the cell as they have MREs (microRNA sponge);
lastly deregulating our target gene (TRIM) And yes! A sophisticated and effective molecular level control.
What is our aim for a ceRNA
      1- To analyze circRNA and disease databases to select significantly
relevant circRNA for HCC.
2- To analyze circRNA-miRNA interaction databases to retrieve
competing endogenous RNA specific for HCC.
3- To characterize the expression of cirRNA-associated ceRNA
genes in HepG2 cell line to evaluate their role in
pathogenesis of HCC.
                 4- To compare between the efficacy of knocking in of circular RNA using
synthetic circuits and crisper techniques.
CRISPR
In order to simply explain to you what CRISPR is and what we can use it in or how does it solve
a problem, we have to briefly tell you the headache that’s been bothering us. Nowadays we have major
medical therapies that we are using to manage disease processes and its pathogenic or misregulated
proteins or molecules associated with the disease. Worth to mention, we know that these proteins
are encoded and affected by changes in the genes and their sequences, yet we haven’t got enough
treatment therapies on that scale. So, we need to improve our impact on that molecular level by using
genome (gene) editing tools such as TALENs, ZFN and of course our precious CRISPR/CAS9.
CRISPR, a new genome editing tool, but why CRISPR specifically rather than the other tools. In a matter of
fact this question is answered plainly, because it fulfills the criteria that we are looking for. These criteria
are that it edits the genomes with exceptional precision, efficacy and flexibility. Also we have to
mention that it wasn’t invented by scientists but that it's naturally occurring in bacteria
(Streptococcus pyogenes) as way of self-defense against viruses that prey on bacteria.
When we undeclared the mechanism of defense of those bacteria we understood how CRISPR/CAS9 works
and that it's a part of the bacteria's immune system. The CRISPR part in the bacteria would keep pieces and
parts of those vicious viruses around it so that it can recognize them the next time they encounter.
Not only that, CRISPER then ignites CAS9 which is one of its famous associated proteins to assault
the violating virus by snipping parts of their DNA at specific sites. Usually the encoding genes for CAS
are settled close to those of CRISPR. Now that we understood what CRISPR is and how it works,
we can put a simple map in our heads by imagining that CRISPR is a collection of DNA
sequence that tells it associates (helpers) what to do. One of CRISPR'S loyal comrade is CAS9 that is
responsible for snipping the DNA at specific sites that CRISPR assign's it to by guiding it by
providing it by (logically called) guide RNA.
We start to wonder how we are going to apply it in medical therapy. It's really obvious that we are
going to guide our tool to snip the unwanted DNA in order to regain the normal regulation of the genes
and proteins and so that the disease is cured. That is going to happen by inserting a
guide RNA to match the undesirable gene and let the CAS9 snip it off. Always put in mind that the
DNA is a long sequence of bases, we can't just insert all of the genome for CAS9 to detect and snip,
it doesn’t work like that. In fact, CAS9 can only take up to about 20 base long sequence
to be recognized. Ordering the matching guide RNA after using an online tool to design the
targeted sequence. When scientists put the editing tools in trial the other tools were proven
to be more specific yet, there was a huge downside, Scientists have to create a designed protein each time
and create several variations before
finding one that might work. CRISPR saved all the time and is also more likely to work. Besides that it can
be used in all kind of organisms. In order to use such technique we had to totally understand how it works
and we couldn’t have done this unless for those who started using it and simplified the ideas for us
like Genetic Home References.xiv 
We can't neglect those who matured this technique and helped to enlighten our minds to
carry on with our theory and experiments. We first read that, Jie Wang et al. mentioned in their
paper about using CRISPR/CAS9 to inhibit hepatitis B virus replication that,
these results suggested that CRISPR/Cas9 system could efficiently
destroy HBV expressing templates (genotypes A-D) without apparent cytotoxicity. It may be a
potential approach for eradication of persistent HBV cccDNA in chronic HBV infection patients.xv
  Also, we found that, Panpan Hou et al.
published that they found two sites in CXCR4 that can be targeted effectively and specifically
by the CRISPR/Cas9 system, resulting in co-receptor
CXCR4 ablation. We also show that the modified cells are resistant to X4 type HIV-1 infection
and this may provide us with an alternative approach of gene therapy for treating AIDS.
Although lenti-CRISPR/Cas9 provides powerful means to disrupt CXCR4,
the optimized delivery methods using adenovirus
need to be explored as to further improve their specificity and minimize the concern for therapeutic
safety. Due to the variation in viral  infection, co-disruption of CCR5 and CXCR4 should be tested using lentior
adenovirus mediated CRISPR/Cas9 system in the future. Furthermore, the successful disruption
of CXCR4 in Rhesus macaque CD4+ T cells may accelerate gene therapy studies
for AIDS in nonhuman primate models.xvi
As well as, Kit-San Yuen et al. CRISPR/Cas9-mediated genome editing of Epstein–Barr virus
in human cells published that CRISPR/Cas9-mediated editing of the EBV genome
in human cells provides a new technology platform for the genetic study of EBV.
In particular, it will facilitate rapid analysis of the roles of individual EBV genes in viral replication,
persistence and transformation. Compared with BAC clones, it will be more difficult to obtain
large amounts of CRISPR/Cas9-edited viral DNA for the assessment of genome integrity by
restriction mapping. However, the new method also has several advantages and is highly
complementary to the existing BAC technology. First, CRISPR/Cas9 technology is applicable
to any EBV strain, whereas EBV BACs are currently available for only three strains. Second,
CRISPR/Cas9-mediated editing is performed completely in human cells, whereas
EBV BACs are constructed and produced in Escherichia coli.xvii
fig(3): Principle of CRISPR 
IGEM Hypothesis
References
iAmerican Cancer Society. Cancer Facts & Figures 2016 . Atlanta, Ga: American CancerSociety; 2015.
iiAmal S. Ibrahim, Hussein M. Khaled, Nabiel NH Mikhail, Hoda Baraka, and
HossamKamel, “Cancer Incidence in Egypt: Results of the National Population-Based
Cancer Registry Program,” Journal of Cancer Epidemiology, vol. 2014, Article ID 437971,
18 pages, 2014. doi:10.1155/2014/437971
iiiAbdelgawad, I.A.; Mossallam, G.I.; Radwan, N.H.; Elzawahry, H.M. and Elhifnawy,
N.M.(2013):Can Glypican3 be diagnostic for early hepatocellular carcinoma among
Egyptian patients?. Asian Pac J Cancer Prev.; 14(12):7345-9.
YU, FU-JUN et al. “Long Non-Coding RNAs and Hepatocellular Carcinoma.” Molecular and iv
Clinical Oncology 3.1 (2015): 13–17. PMC. Web. 9 Feb. 2017.
v Bhaya D. Davison M. Barrangou R. CRISPR-Cas systems in bacteria and archaea:
versatile small RNAs for adaptive defense and regulation Annu. Rev. Genet. 2011 45
273297
vii Cox D.B. Platt R.J. Zhang F. Therapeutic genome editing: prospects and challenges Nat.
Med. 2015 21 121131
viii Xiangjun He, Chunlai Tan, Feng Wang, Yaofeng Wang, Rui Zhou, Dexuan Cui, Wenxing
You, Hui Zhao, Jianwei Ren, Bo Feng; Knock-in of large reporter genes in human cells via
CRISPR/Cas9-induced homology-dependent and independent DNA repair. Nucl Acids
Res 2016; 44 (9): e85. doi: 10.1093/nar/gkw064
ix Kan Y. Ruis B. Lin S. Hendrickson E.A. The mechanism of gene targeting in human
somatic cells PLoS Genet. 2014 10 e1004251
x Mao Z. Bozzella M. Seluanov A. Gorbunova V. Comparison of nonhomologous end
joining and homologous recombination in human cells DNA Repair (Amst.) 2008 7
17651771
xi Merkle F.T. Neuhausser W.M. Santos D. Valen E. Gagnon J.A. Maas K. Sandoe J. Schier
A.F. Eggan K. Efficient CRISPR-Cas9-mediated generation of knockin human pluripotent
stem cells lacking undesired mutations at the targeted locus Cell Rep. 2015 11 875883.
xii Rong Z. Zhu S. Xu Y. Fu X. Homologous recombination in human embryonic stem
cells using CRISPR/Cas9 nickase and a long DNA donor template Protein Cell 2014 5
258260
xiii Thomson J.A. Itskovitz-Eldor J. Shapiro S.S. Waknitz M.A. Swiergiel J.J. Marshall
V.S. Jones J.M. Embryonic stem cell lines derived from human blastocysts Science 1998
282 11451147
xiv gupta, r. M., & musunuru, k. (2014, october 1). Expanding the genetic editing tool kit: zfns,
talens, and crispr-cas9. Journal of clinical investigation. American society for clinical
investigation. Hsu pd, lander es, zhang f. Development and applications of crispr-cas9 for
genome engineering. Cell. 2014 jun 5;157(6):1262-78. Doi:10.1016/j.cell.2014.05.010.
Review. Pubmed: 24906146. Free full-text available from pubmed central: pmc4343198.
Komor ac, badran ah, liu dr. Crispr-based technologies for the manipulation of eukaryotic
genomes. Cell. 2017 apr 20;169(3):559. Doi:10.1016/j.cell.2017.04.005. Pubmed: 28431253.
Lander es. The heroes of crispr. Cell. 2016 jan 14;164(1-2):18-28.
Doi:10.1016/j.cell.2015.12.041. Review. Pubmed: 26771483.
xv wang j, xu z-w, liu s, et al. Dual grnas guided crispr/cas9 system inhibits hepatitis b virus
replication. World journal of gastroenterology : wjg. 2015;21(32):9554-9565.
Doi:10.3748/wjg.v21.i32.9554.
xvi hou, p., chen, s., wang, s., yu, x., chen, y., jiang, m., … guo, d. (2015). Genome editing of
cxcr4 by crispr/cas9 confers cells resistant to hiv-1 infection. Scientific reports, 5, 15577.
xvii  yuen, k.-s., chan, c.-p., wong, n.-h. M., ho, c.-h., ho, t.-h., lei, t., … jin, d.-y. (2015).
Crispr/cas9-mediated genome editing of epstein-barr virus in human cells. Journal of general
virology, 96(pt_3), 626–636.
 
