<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_Toronto_Model skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><DIV id="top_title"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:Toronto/Model</SPAN></H1></DIV><DIV id="HQ_page"><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><DIV id="navBarBackground" class="navBarSpacer" style="background-color=black;"><UL id="sdt_menu" class="sdt_menu"> style=&quot;position:fixed; background-color=black; padding-bottom: 86px;&quot;&gt;
	
	<LI><A href="https://2018.igem.org/Team:Toronto/WetLab"><SPAN class="sdt_wrap"><SPAN class="sdt_link">WetLab</SPAN><SPAN class="sdt_descr">Laboratory Experiments</SPAN></SPAN></A><DIV class="sdt_box" style="z-index: 99; opacity: 1.0; height:170px;"><A href="https://2018.igem.org/Team:Toronto/WetLab/ExperimentProtocols">Protocols</A><A href="https://2018.igem.org/Team:Toronto/WetLab/Notebook">Notebook</A><A href="https://2018.igem.org/Team:Toronto/WetLab/Results">Results</A><A href="https://2018.igem.org/Team:Toronto/WetLab/Parts">Parts</A><A href="https://2018.igem.org/Team:Toronto/InterLab">InterLab</A></DIV></LI><LI><A href="https://2018.igem.org/Team:Toronto/DryLab"><SPAN class="sdt_wrap"><SPAN class="sdt_link">DryLab</SPAN><SPAN class="sdt_descr">Computational Projects</SPAN></SPAN></A><DIV class="sdt_box" style="z-index: 99; opacity: 1.0;"><A href="https://2018.igem.org/Team:Toronto/Model">Model</A></DIV></LI><LI><A href="https://2018.igem.org/Team:Toronto/PolicyAndPractices"><SPAN class="sdt_wrap"><SPAN class="sdt_link">P &amp; P</SPAN><SPAN class="sdt_descr">Policy and Practices</SPAN></SPAN></A><DIV class="sdt_box" style="z-index: 99; opacity: 1.0;"><A href="https://2018.igem.org/Team:Toronto/Human_Practices">Human Practices</A><A href="https://2018.igem.org/Team:Toronto/Public_Engagement">Engagement</A><A href="https://2018.igem.org/Team:Toronto/PolicyAndPractices/Outreach">Outreach</A></DIV></LI><LI><A href="https://2018.igem.org/Team:Toronto/Project"><SPAN class="sdt_wrap"><SPAN class="sdt_link">Project</SPAN><SPAN class="sdt_descr">Team Behind the Project</SPAN></SPAN></A><DIV class="sdt_box" style="z-index: 99; opacity: 1.0;"><A href="https://2018.igem.org/Team:Toronto/Project/Description">Description</A><A href="https://2018.igem.org/Team:Toronto/Demonstrate">Demonstrate</A><A href="https://2018.igem.org/Team:Toronto/Project/Team">Team</A><A href="https://2018.igem.org/Team:Toronto/Collaborations">Collaborations</A><A href="https://2018.igem.org/Team:Toronto/Attributions">Attributions</A></DIV></LI><LI><A href="https://2018.igem.org/Team:Toronto"><SPAN class="sdt_wrap"><SPAN class="sdt_link">iGEM</SPAN><SPAN class="sdt_descr">iGEM Home Page</SPAN></SPAN></A><DIV class="sdt_box" style="z-index: 99; opacity: 1.0;"><A href="https://2018.igem.org/Main_Page">iGEM Home</A><A href="https://2018.igem.org/Team:Toronto">iGEM Toronto</A></DIV></LI></UL></DIV><DIV style="margin-top: -7px;"><DIV class="backgroundTertiaryHandler"><NAV class="navigation" id="mainNav" style="border-style: solid;"><A class="navigation__link" href="#1">1. Introduction</A><A class="navigation__link" href="#2">2. A Differential Bioreactor Model</A><A class="navigation__link" href="#9">3. A Mechanistic Buoyancy Model: Maximum Carrying Capacity Determination</A><A class="navigation__link" href="#15">4. Growth Dynamics</A><A class="navigation__link" href="#19">5. Discussion and Future Directions</A><A class="navigation__link" href="#20">6. Appendix</A><A class="navigation__link" href="#21">7. References</A></NAV><DIV class="page-section hero" id="1"><H3 style="font-size: 50px; line-height: 60px; text-align: center;">Drylab Model</H3><H1 id="introduction">1. Introduction</H1><P>The goal of the 2018 iGEM team was to explore an alternative waste-water cleaning mechanism that is both environmentally friendly and economically feasible. We hypothesized that one could genetically engineer <EM>E. coli </EM> to bind to waste particles, and then float to the surface of the reactor, allowing for easy <EM>E. coli </EM> removal. We:
</P><P>Create a generic dynamic bioreactor system that utilizes <EM>E. coli </EM> flotation. This model should be so generic that the bioreactor can be used for a variety of applications, with different cells and different starting conditions. Then we characterized the performance of the system with many different initial starting conditions and parameters.</P><LI><P>Benchmark <EM>E. coli </EM> flotation results, using a cellular tracking algorithm we developed. These results can be coupled with our ODE buoyancy model, which allows us to determine the maximum carrying capacity of our particular strain of cells. This is necessary for the bioreactor, it allows us to determine which genetically engineered <EM>E. coli </EM> strain can perform for what type of waste particle. This model also allows the wet lab to measure how certain genetic variations affect the flotation. Rather than having flotation or no flotation, we can now quantify the characteristics of flotation.</P></LI><LI><P>Characterize the growth dynamics of a particular strain of cells. This data is necessary for the bioreactor.</P></LI><STRONG>Note:</STRONG> In the very end of this paper, we included a nomenclature defining all the variables used.
				</DIV><DIV class="page-section" id="2"><H1 id="a-differential-bioreactor-model">2. A Differential Bioreactor Model</H1></DIV><DIV class="page-section" id="3"><H2>2.1 Goals</H2><UL><LI><P>Explore a possible application for our genetically engineered <EM>E. coli</EM> biomass that utilizes flotation.</P></LI><LI><P>Develop a generic bioreactor that can be reused in many different conditions and for a variety of purposes.</P></LI></UL></DIV><DIV class="page-section" id="4"><H2 id="mathematical-formulation">2.2 Mathematical Formulation</H2><H3 id="concepts">Concepts</H3><P><STRONG>Monod Equation [7]</STRONG> 
In order to run a bioreactor, we need to understand at what rate the <EM>E. coli</EM> grow, and how they respond to environmental conditions. Bacterial growth can be separated into four phases:</P><OL><LI><P>The Lag Phase: Little growth is observed.</P></LI><LI><P>The Exponential Phase: Exponential growth is observed after the cells get used to their environment, and so long as the limiting substrate is still in surplus.</P></LI><LI><P>The Stationary Phase: No growth is observed, death rate and growth rate are approximately equal.</P></LI><LI><P>Death Phase: There is no more substrate, thus the population dies.</P></LI></OL><P>If <SPAN class="math inline"><EM>N</EM></SPAN> is the concentration of biomass population, then the exponential growth phase is given by: 
	$$\dot{N} = \mu N$$
	where <SPAN class="math inline"><EM>μ</EM></SPAN> is the specific growth rate, the “rate of increase of cell concentrations per unit cell concentration” (<SPAN class="math inline"><EM>h</EM><SUP> − 1</SUP></SPAN>) [7]. <SPAN class="math inline"><EM>μ</EM></SPAN> depends on the substrate concentration; the relation is given by the Monod equation: 
	$$ \mu(S) = \mu_m \frac{S}{K_S + S}$$
	where <SPAN class="math inline"><EM>S</EM></SPAN> is the substrate concentration (in mol <SPAN class="math inline"><EM>L</EM><SUP> − 1</SUP></SPAN>), and <SPAN class="math inline"><EM>K</EM><SUB><EM>S</EM></SUB></SPAN> is the Monod Constant (in mol <SPAN class="math inline"><EM>L</EM><SUP> − 1</SUP></SPAN>). As the substrate is consumed, more biomass is being created at the rate of: 
	$$\frac{dN}{dS} = -\gamma \frac{dS}{dt}$$
	where <SPAN class="math inline"><EM>t</EM></SPAN> is time. <SPAN class="math inline"><EM>γ</EM></SPAN> should be interpreted as “the ratio of the mass of cells formed to the mass of substrate consumed” [7].</P><H3 id="the-model">The Model</H3><FIGURE><FIGCAPTION style="text-align:center"><STRONG>Figure 1. The Bioreactor Design</STRONG></FIGCAPTION></FIGURE><P>Call the ’Main Reactor’ (M.R) and the ’Flotation Tank’ (F.T). Let <SPAN class="math inline"><EM>X</EM><SUB><EM>u</EM></SUB></SPAN> be the concentration of <EM>E. coli</EM> (in mol <SPAN class="math inline"><EM>L</EM><SUP> − 1</SUP></SPAN>), <SPAN class="math inline"><EM>X</EM><SUB><EM>w</EM></SUB></SPAN> the concentration of particle bound to the surface of the <EM>E. coli</EM> (in mol <SPAN class="math inline"><EM>L</EM><SUP> − 1</SUP></SPAN>). Below we will outline the steps of the bioreactor seen in Figure 1.</P><OL><LI><P>All the substrate, waste and cellular population is homogeneously distributed between the F.T and the M.R. Initially there is no particle bound to any of the cells, i.e <SPAN class="math inline"><EM>X</EM><SUB><EM>w</EM></SUB> = 0</SPAN>.</P></LI><LI><P>When activated, the bioreactor begins to pump solution from the M.R to the F.T at rate <SPAN class="math inline"><EM>D</EM></SPAN>, this is represented by <SPAN class="math inline"><EM>D</EM>(<EM>N</EM> + <EM>X</EM><SUB><EM>u</EM></SUB> + <EM>X</EM><SUB><EM>w</EM></SUB> + <EM>S</EM>)</SPAN>.</P></LI><LI><P>In the F.T, the cells are floating on the surface, and are scraped off and removed. Thus only <SPAN class="math inline"><EM>X</EM><SUB><EM>u</EM></SUB></SPAN> and <SPAN class="math inline"><EM>S</EM></SPAN> remain within the solution, which is pumped back into M.R at rate <SPAN class="math inline"><EM>D</EM></SPAN>. In practice, a cellular filter alike the one used for skimming could be placed at the exit of the F.T to capture any dead cells.</P></LI><LI><P>Within M.R the unbound particle binds to the <EM>E. coli</EM> with some rate constant <SPAN class="math inline"><EM>α</EM></SPAN>, and detaches with some rate <SPAN class="math inline"><EM>β</EM></SPAN> (equilibrium process).</P></LI><LI><P>This process operates until the particle concentration is as low as desired.</P></LI></OL><H3 id="bioreactor-assumptions">Bioreactor Assumptions</H3><UL><LI><P>Our cell surface engineering technique has specificity to only one substance in the waste-water (i.e. the substance desired for removal).</P></LI><LI><P>We will only observe exponential bacterial growth within the reactor, i.e. the biomass has been cultured to sufficiently large optical density before being introduced into the bioreactor.</P></LI><LI><P>Suspended particles attach to the surface of our <EM>E. coli</EM> at a rate proportional to the particle concentration and the ratio of unused area available on the surface.</P></LI><LI><P>Bound particles detach from the surface of our <EM>E. coli</EM> back into the waste-water at a rate proportional to the particle concentration bound to the <EM>E. coli</EM>.</P></LI><LI><P>Once the <EM>E. coli</EM> that are pumped out of the main chamber end up in the flotation tank, they are immediately removed – no more growth nor unbinding of particle occurs.</P></LI><LI><P>We ignore all the spatial properties of pipes between the main reactor and flotation tank, as soon as part of the solution leaves the main take, it immediately arrives in the flotation tank, and vice versa.</P></LI><LI><P>The pipe leading into the flotation tank sprinkles the solution over the flotation tank so lightly that the <EM>E. coli</EM> cells do not sink into the tank – they float on the surface, allowing for immediate removal.</P></LI><LI><P>The solution within the main reactor is continuously mixed; it can be considered a homogeneous solution).</P></LI><LI><P>All the <EM>E. coli</EM> in the flotation tank float.</P></LI><LI><P>A higher dilution rate (<SPAN class="math inline"><EM>D</EM></SPAN>) does not affect the detachment rate.</P></LI></UL></DIV><DIV class="page-section" id="5"><H2 id="equations">2.3 Equations</H2><P>The dynamics within the bioreactor are given by: <SPAN class="math display">
				    $$\begin{aligned}\dot{N} &amp;= N_i \mu (S) - N \cdot \left( D + \kappa \right) \label{N}\end{aligned}$$
            <FIGCAPTION style="text-align:right"><STRONG>Equation 1.</STRONG></FIGCAPTION>
            $$\begin{aligned}
				    \dot{S} &amp;= -\mu (S) N \gamma^{-1} \label{S}
            \end{aligned}$$
            <FIGCAPTION style="text-align:right"><STRONG>Equation 2.</STRONG></FIGCAPTION>
            $$\begin{aligned}
				    \dot{X}_u &amp;= \beta X_w - \alpha X_u \cdot \left(\frac{\delta N - X_w}{\delta N} \right) \label{Xu}
            \end{aligned}$$
            <FIGCAPTION style="text-align:right"><STRONG>Equation 3.</STRONG></FIGCAPTION>
            $$\begin{aligned}
				    \dot{X}_w &amp;= - \beta X_w + \alpha X_u \cdot \left(\frac{\delta N - X_w}{\delta N} \right) - D X_w \label{Xw}
            \end{aligned}$$
            <FIGCAPTION style="text-align:right"><STRONG>Equation 4.</STRONG></FIGCAPTION></SPAN> Where, <SPAN class="math inline"><EM>κ</EM></SPAN> is the death rate, <SPAN class="math inline"><EM>δ</EM></SPAN> the maximum carrying capacity of the cells, <SPAN class="math inline"><EM>α</EM></SPAN> the rate constant for binding, <SPAN class="math inline"><EM>β</EM></SPAN> the rate constant for unbinding.
				Equation <A href="#N" data-reference-type="ref" data-reference="N">[1]</A> represents the change in <EM>E. coli</EM> cell concentration, equation <A href="#S" data-reference-type="ref" data-reference="S">[2]</A> represents the change in limiting substrate concentration, equation <A href="#Xu" data-reference-type="ref" data-reference="Xu">[3]</A> represents the change in unbound particle concentration, and equation <A href="#Xw" data-reference-type="ref" data-reference="Xw">[4]</A> represents the change in concentration of particle bound to the surface of the <EM>E. coli</EM>.
				In equation <A href="#Xu" data-reference-type="ref" data-reference="Xu">[3]</A> we have <SPAN class="math inline"><EM>β</EM><EM>X</EM><SUB><EM>w</EM></SUB></SPAN> [9] which represents particle unbinding from the <EM>E. coli</EM> population, it depends on <SPAN class="math inline"><EM>β</EM></SPAN>: the kinetic detachment rate, and <SPAN class="math inline"><EM>X</EM><SUB><EM>w</EM></SUB></SPAN>: the concentration of particle that is bound to the surface of the <EM>E. coli</EM>. We are also removing particles from the solution by <SPAN class="math inline">$$\alpha X_u \cdot \left(\frac{\delta N - X_w}{\delta N} \right)$$</SPAN>. This depends on the kinetic binding rate <SPAN class="math inline"><EM>α</EM></SPAN>, the concentration of unbound particle within the solution (<SPAN class="math inline"><EM>X</EM><SUB><EM>u</EM></SUB></SPAN>), and the proportion of available binding sites on the <EM>E. coli</EM>, this is represented by <SPAN class="math inline">$$\left(\frac{\delta N - X_w}{\delta N} \right)$$</SPAN>. Equation <A href="#Xw" data-reference-type="ref" data-reference="Xw">[4]</A> is the negative of equation <A href="#Xu" data-reference-type="ref" data-reference="Xu">[3]</A> but we also remove <SPAN class="math inline"><EM>X</EM><SUB><EM>w</EM></SUB></SPAN> at some dilution rate <SPAN class="math inline"><EM>D</EM></SPAN>.</P></DIV><DIV class="page-section" id="6"><H2 id="numerical-solution">2.4 Numerical Solution</H2><FIGURE><FIGCAPTION style="text-align:center"><STRONG>Figure 2. Bioreactor Numerical Simulation</STRONG></FIGCAPTION></FIGURE><P>We numerically solved the ODE system composed of equations <A href="#N" data-reference-type="ref" data-reference="N">[1]</A> - <A href="#Xw" data-reference-type="ref" data-reference="Xw">[4]</A> using the ODE45 solver in MATLAB (see the end of the paper of the MATLAB code), using the initial conditions <SPAN class="math inline"><EM>N</EM><SUB><EM>i</EM></SUB> = 0.4, <EM>S</EM> = 0.63, <EM>X</EM><SUB><EM>u</EM></SUB> = 0.5, <EM>X</EM><SUB><EM>w</EM></SUB> = 0</SPAN> and parameters:</P><UL><LI><P><SPAN class="math inline"><EM>γ</EM> = 1.1 (<EM>u</EM><EM>n</EM><EM>i</EM><EM>t</EM><EM>l</EM><EM>e</EM><EM>s</EM><EM>s</EM>)</SPAN></P></LI><LI><P><SPAN class="math inline"><EM>κ</EM> = 0.01 <EM>h</EM><SUP> − 1</SUP></SPAN></P></LI><LI><P><SPAN class="math inline"><EM>D</EM> = 3 <EM>h</EM><SUP> − 1</SUP></SPAN></P></LI><LI><P><SPAN class="math inline"><EM>β</EM> = 0.03 <EM>h</EM><SUP> − 1</SUP></SPAN></P></LI><LI><P><SPAN class="math inline"><EM>α</EM> = 1.5 <EM>h</EM><SUP> − 1</SUP></SPAN></P></LI><LI><P><SPAN class="math inline"><EM>μ</EM><SUB><EM>m</EM></SUB> = 0.8 <EM>h</EM><SUP> − 1</SUP></SPAN></P></LI><LI><P><SPAN class="math inline"><EM>K</EM><SUB><EM>s</EM></SUB> = 2.87 × 10<SUP> − 7</SUP> <EM>m</EM><EM>o</EM><EM>l</EM> <EM>L</EM><SUP> − 1</SUP></SPAN></P></LI><LI><P><SPAN class="math inline"><EM>δ</EM> = 1.5 (<EM>u</EM><EM>n</EM><EM>i</EM><EM>t</EM><EM>l</EM><EM>e</EM><EM>s</EM><EM>s</EM>)</SPAN></P></LI></UL><P>The above simulation was done for removing Cobalt from mining waste-water effluent. It utilizes a metal binding mechanism outlined in [2] which gave us a range of viable <SPAN class="math inline"><EM>α</EM></SPAN>. We got <SPAN class="math inline"><EM>K</EM><SUB><EM>s</EM></SUB></SPAN> and <SPAN class="math inline"><EM>μ</EM><SUB><EM>m</EM></SUB></SPAN> from [8], <SPAN class="math inline"><EM>β</EM>,</SPAN> and <SPAN class="math inline"><EM>κ</EM></SPAN> from [9]. We estimated reasonable values for <SPAN class="math inline"><EM>D</EM>, <EM>γ</EM></SPAN>. See §3.5 for calculating the value of <SPAN class="math inline"><EM>δ</EM></SPAN>.
				These results are incredibly promising. They show that the bioreactor can theoretically be useful at solving real problems, and that it operates within a reasonable amount of time (6.5 hours).</P></DIV><DIV class="page-section" id="7"><H2 id="sensitivity-analysis">2.5 Sensitivity Analysis</H2><P>In order to understand the dynamics within the bioreactor, we performed a sensitivity analysis that estimates the relative effect of a single parameter on the performance of the system. Unfortunately, it is not possible to define an explicit function relating the different parameters to each other, we must perform a ’naive’ analysis; we vary one parameter while holding all other constant, and see how this change affects the performance of the system. Performance is measured by how long it takes the bioreactor to remove the large majority of the particle that is desired for bioremediation. We set some small <SPAN class="math inline"><EM>ε</EM> = 10<SUP> − 4</SUP><EM>m</EM><EM>o</EM><EM>l</EM> <EM>L</EM><SUP> − 1</SUP></SPAN> to define a low threshold of acceptable particle concentration. For §2.5.1 - §2.5.6, we kept all the parameters at:</P><UL><LI><P><SPAN class="math inline"><EM>γ</EM> = 1.1 (<EM>u</EM><EM>n</EM><EM>i</EM><EM>t</EM><EM>l</EM><EM>e</EM><EM>s</EM><EM>s</EM>)</SPAN></P></LI><LI><P><SPAN class="math inline"><EM>κ</EM> = 0.01 <EM>h</EM><SUP> − 1</SUP></SPAN></P></LI><LI><P><SPAN class="math inline"><EM>D</EM> = 3 <EM>h</EM><SUP> − 1</SUP></SPAN></P></LI><LI><P><SPAN class="math inline"><EM>β</EM> = 0.03 <EM>h</EM><SUP> − 1</SUP></SPAN></P></LI><LI><P><SPAN class="math inline"><EM>α</EM> = 1.5 <EM>h</EM><SUP> − 1</SUP></SPAN></P></LI><LI><P><SPAN class="math inline"><EM>μ</EM><SUB><EM>m</EM></SUB> = 0.8 <EM>h</EM><SUP> − 1</SUP></SPAN></P></LI><LI><P><SPAN class="math inline"><EM>K</EM><SUB><EM>s</EM></SUB> = 2.87 × 10<SUP> − 7</SUP> <EM>m</EM><EM>o</EM><EM>l</EM> <EM>L</EM><SUP> − 1</SUP></SPAN></P></LI><LI><P><SPAN class="math inline"><EM>δ</EM> = 1.5 (<EM>u</EM><EM>n</EM><EM>i</EM><EM>t</EM><EM>l</EM><EM>e</EM><EM>s</EM><EM>s</EM>)</SPAN></P></LI></UL><P>and only varied the parameter whose effect was measured.</P><H3 id="the-effect-of-initial-metal-concentration-on-bioreactor-performance">The Effect of Maximum Carrying Capacity on Bioreactor Performance</H3><FIGURE><FIGCAPTION style="text-align:center"><STRONG>Figure 3: Maximum Carrying Capacity (<SPAN class="math inline"><EM>X</EM><SUB><EM>u</EM></SUB></SPAN>) vs. Bioreactor Performance</STRONG></FIGCAPTION></FIGURE><P>(See Figure 3) The higher the initial metal concentration within the bioreactor, the longer it takes to operate.</P><H3 id="the-effect-of-dilution-rate-on-bioreactor-performance">The Effect of Dilution Rate on Bioreactor Performance</H3><FIGURE><FIGCAPTION style="text-align:center"><STRONG>Figure 4: Dilution Rate (<SPAN class="math inline"><EM>D</EM></SPAN>) vs. Bioreactor Performance</STRONG></FIGCAPTION></FIGURE><P>(See Figure 4) The higher the dilution rate, the better the performance. Above dilution rate <SPAN class="math inline"><EM>D</EM> = 1</SPAN> it slowly approaches an asymptote. We should not have a <SPAN class="math inline"><EM>D</EM></SPAN> that is too large because it is energy costly. Also, a larger <SPAN class="math inline"><EM>D</EM></SPAN> leads to faster deterioration of the bioreactor.</P><H3 id="the-effect-of-initial-metal-concentration-on-bioreactor-performance">The Effect of Initial Metal Concentration on Bioreactor Performance</H3><FIGURE><FIGCAPTION style="text-align:center"><STRONG>Figure 5: Initial Metal Concentration (<SPAN class="math inline"><EM>X</EM><SUB><EM>u</EM></SUB></SPAN>) vs. Bioreactor Performance</STRONG></FIGCAPTION></FIGURE><P>(See Figure 5) The higher the initial metal concentration within the bioreactor, the longer it takes to operate.</P><H3 id="the-effect-of-attachment-rate-on-bioreactor-performance">The Effect of Attachment Rate on Bioreactor Performance</H3><FIGURE><FIGCAPTION style="text-align:center"><STRONG>Figure 6: Attachment Rate (<SPAN class="math inline"><EM>α</EM></SPAN>) vs. Bioreactor Performance </STRONG></FIGCAPTION></FIGURE><P>(See Figure 6) The higher the attachment rate (<SPAN class="math inline"><EM>α</EM></SPAN>), the better the performance of the system. It does approach an asymptote quite quickly though. This is because of the maximum carrying capacity, at some point, there is just no more available space on the <EM>E. coli</EM> surface to bind more particle, regardless of the attachment rate.</P><H3 id="the-effect-of-detachment-rate-on-bioreactor-performance">The Effect of Detachment Rate on Bioreactor Performance</H3><FIGURE><FIGCAPTION style="text-align:center"><STRONG>Figure 7: Detachment Rate (<SPAN class="math inline"><EM>β</EM></SPAN>) vs. Bioreactor Performance </STRONG></FIGCAPTION></FIGURE><P>(See Figure 7) The lower the detachment rate (<SPAN class="math inline"><EM>β</EM></SPAN>), the better the performance of the system.</P><H3 id="the-effect-of-initial-cellular-population-on-bioreactor-performance">The Effect of Initial Cellular Population on Bioreactor Performance</H3><FIGURE><FIGCAPTION style="text-align:center"><STRONG>Figure 8: Initial Cellular Population (<SPAN class="math inline"><EM>N</EM><SUB><EM>i</EM></SUB></SPAN>) vs. Bioreactor Performance</STRONG></FIGCAPTION></FIGURE><P>(See Figure 8) The higher the initial <EM>E. coli</EM> population (<SPAN class="math inline"><EM>N</EM><SUB><EM>i</EM></SUB></SPAN>), the better the system performs. After some <SPAN class="math inline"><EM>N</EM><SUB><EM>i</EM></SUB> ≈ 1</SPAN>, it approaches an asymptote.</P><H3 id="the-effect-of-maximum-specific-growth-rate-on-bioreactor-performance">The Effect of Maximum Specific Growth Rate on Bioreactor Performance</H3><FIGURE><FIGCAPTION style="text-align:center"><STRONG>Figure 9: Maximum Specific Growth Rate (<SPAN class="math inline"><EM>μ</EM><SUB><EM>m</EM></SUB></SPAN>) vs. Bioreactor Performance</STRONG></FIGCAPTION></FIGURE><P>(See Figure 9) The higher the maximum specific growth rate, the better the performance of the system.</P></DIV><DIV class="page-section" id="8"><H2 id="hypothetical-applications">2.6. Hypothetical Applications</H2><P>The numerical simulations above were all based on parameters values specific to cobalt removal from mining waste-water. If the <EM>E. coli</EM> are coupled with a binding mechanism that is not specific to cobalt, then this generic bioreactor can be used for a large variety of applications (one simply has to change <SPAN class="math inline"><EM>α</EM></SPAN> and <SPAN class="math inline"><EM>β</EM></SPAN>). One very promising application is the removal of pharmaceuticals, such as penicillin, from municipal waste-water. The reactor is viable for many different binding methods, strains of <EM>E. coli</EM>, and initial concentrations of waste, as displayed in §2.5. This reactor is independent of volume, all the parameters are relative to each other, this system would perform the same with 1 <SPAN class="math inline"><EM>L</EM></SPAN> of effluent as with <SPAN class="math inline">1 × 10<SUP>10</SUP></SPAN> L of effluent. Evidently, the size of the chambers would need to be scaled appropriately.
				</P></DIV><DIV class="page-section" id="9"><H1 id="a-mechanistic-buoyancy-model-maximum-carrying-capacity-determination">3. A Mechanistic Buoyancy Model: Maximum Carrying Capacity Determination</H1></DIV><DIV class="page-section" id="10"><H2 id="goal">3.1 Goal</H2><P>The buoyancy model has 2 main goals:</P><OL><LI><P>Estimate the mean buoyant force experienced by the genetically engineered <EM>E. coli</EM> biomass per unit mass as a result of gas vesicle formation from ARG1 over-expression. This effectively determines a mechanical upper bound for carrying capacity of the biomass.</P></LI><LI><P>Characterize and quantify flotation observations from wet lab experimentation. It is not well known what the role of some of the secondary gas vesicle proteins (GVPs) in the ARG1 construct is for gas vesicle formation, and being able to quantify flotation facilities comparison of different combinations of secondary GVPs for optimization of a gene construct specifically engineered for biomass flotation.</P></LI></OL></DIV><DIV class="page-section" id="11"><H2 id="mathematical-formulation-1">3.2 Mathematical Formulation</H2><H3 id="concepts-1">Concepts</H3><UL><LI><P>Newtonian mechanics: $$\vec{F} = m\vec{a} \equiv m\frac{\mathrm{d}\vec{v}}{\mathrm{t}}$$</P></LI><LI><P><SPAN class="math inline"><EM>F</EM>≡</SPAN> vector sum of forces on body</P></LI><LI><P><SPAN class="math inline"><EM>m</EM>≡</SPAN> mass of body</P></LI><LI><P><SPAN class="math inline"><EM>a</EM>≡</SPAN> acceleration as a function of time</P></LI><LI><P><SPAN class="math inline"><EM>v</EM>≡</SPAN> velocity as a function of time</P></LI></UL><LI><P>Stokes-Einstein Drag: $$\vec{F_D} = -6\pi \eta R \vec{v}$$</P></LI><LI><P>FD≡drag force (note it opposes direction of velocity)</P></LI><LI><P><SPAN class="math inline"><EM>η</EM>≡</SPAN>viscosity of fluid medium</P></LI><LI><P><SPAN class="math inline"><EM>R</EM>≡</SPAN>(approximate) radius of body in spherical approximation.</P></LI><LI><P>Integrating factor method to solve first order linear ODE: <SPAN class="math display">$$\label{ODE}
				        \frac{\mathrm{d}y}{\mathrm{d}t}+f(t)y = g(t)$$</SPAN><FIGCAPTION style="text-align:right"><STRONG>Equation 5.</STRONG></FIGCAPTION>
                Let <SPAN class="math inline"><EM>μ</EM>(<EM>t</EM>) = <EM>e</EM><SUP><EM>β</EM><EM>t</EM></SUP></SPAN>. Multiply both sides by <SPAN class="math inline"><EM>μ</EM>(<EM>t</EM>)</SPAN>.
				Chain rule <SPAN class="math inline">$$\implies \frac{\mathrm{d}}{\mathrm{d}t}\big(y(t)\mu(t)\big) = h(t)\mu(t)$$</SPAN>
				 
				Integrate, divide by <SPAN class="math inline"><EM>μ</EM>(<EM>t</EM>)</SPAN><SPAN class="math inline"> ⟹ <EM>y</EM>(<EM>t</EM>)</SPAN>.</P></LI><H3 id="assumptions">Assumptions</H3><UL><LI><P>The biomass separates into clumps that can be approximated by spheres.</P></LI><LI><P>Buoyant force is constant.</P></LI><LI><P>Cell motility and in-plane motion (motion perpendicular to the vertical axis) is negligible. This effectively reduces our system to one dimension.</P></LI></UL><H3 id="the-model-1">The Model</H3><P>Stokes-Einstein, gravitational force near the surface of the earth <SPAN class="math inline"><EM>F</EM><SUB><EM>g</EM></SUB> = <EM>m</EM><EM>g</EM></SPAN> (<SPAN class="math inline"><EM>g</EM> ≈ 9.81 <EM>m</EM><EM>s</EM><SUP> − 2</SUP>)</SPAN>, Newton’s second law lets us write: <SPAN class="math display">$$\label{Fnet}
				F_{net} = F_g + F_B + F_D \implies m\frac{\mathrm{d}v}{\mathrm{d}t} = -mg + F_B - 6\pi \eta R v$$</SPAN><FIGCAPTION style="text-align:right"><STRONG>Equation 6.</STRONG></FIGCAPTION> Rearrange... <SPAN class="math display">$$\label{theODE}
				\frac{\mathrm{d}v}{\mathrm{d}t} + \frac{6\pi \eta R}{m}v = \big(\frac{F_B}{m} - g\big)$$</SPAN><FIGCAPTION style="text-align:right"><STRONG>Equation 7.</STRONG></FIGCAPTION> Simplify notation with <SPAN class="math inline">$$\alpha := \frac{6\pi \eta R}{m}$$</SPAN> and <SPAN class="math inline">$$\beta := \big(\frac{F_B}{m} - g\big)$$</SPAN> we have: <SPAN class="math display">$$\frac{\mathrm{d}v}{\mathrm{d}t} + \alpha v = \beta$$</SPAN><FIGCAPTION style="text-align:right"><STRONG>Equation 8.</STRONG></FIGCAPTION> Observe the model is now in the form of equation <A href="#ODE" data-reference-type="ref" data-reference="ODE">[5]</A>.</P></DIV><DIV class="page-section" id="12"><H2 id="analytic-solution">3.3 Analytic Solution</H2><P>Now to get a solution we can directly apply the integrating factor method as described in the Concepts sections.
				 
				The integrating factor is: 
				$$\begin{equation}\label{mu}
    		\mu(t) = \mathrm{e}^{\alpha t}
				\end{equation}$$ <FIGCAPTION style="text-align:right"><STRONG>Equation 9.</STRONG></FIGCAPTION> Multiplying both sides we have: <SPAN class="math display">$$\label{integrate}\frac{\mathrm{d}}{\mathrm{d}t}\big(v\mathrm{e}^{\alpha t} \big) = \beta \mathrm{e}^{\alpha t}$$</SPAN><FIGCAPTION style="text-align:right"><STRONG>Equation 10.</STRONG></FIGCAPTION> Now we integrate, divide by integrating factor, and impose an initial condition of <SPAN class="math inline"><EM>v</EM>(<EM>t</EM> = 0) = 0 <EM>m</EM><EM>s</EM><SUP> − 1</SUP></SPAN> (not moving initially): <SPAN class="math display">$$\label{v}
				    v(t) = \frac{\beta}{\alpha}\big(1 - \mathrm{e}^{-\alpha t} \big) \equiv \big(\frac{F_B- mg}{6\pi\eta R} \big)\big(1 - \mathrm{e}^{-\frac{6\pi \eta R}{m}t} \big)$$</SPAN><FIGCAPTION style="text-align:right"><STRONG>Equation 11.</STRONG></FIGCAPTION> This is a closed form time-dependent solution for velocity.
				 
				Now we integrate one more time and impose an initial condition of <SPAN class="math inline"><EM>z</EM>(<EM>t</EM> = 0) = 0 <EM>m</EM></SPAN>: <SPAN class="math display">$$\label{z}
				z(t) = \bigg(\frac{F_B - mg}{g\pi\eta R} \bigg)\bigg(t + \frac{m}{6\pi \eta R}\big(\mathrm{e}^{-\frac{6 \pi \eta R}{m}t} - 1 \big) \bigg)$$</SPAN><FIGCAPTION style="text-align:right"><STRONG>Equation 12.</STRONG></FIGCAPTION> Finally, we have a closed form time-dependent solution for vertical displacement.
				 <STRONG>NB:</STRONG><SPAN class="math inline"><EM>m</EM>, <EM>g</EM>, <EM>η</EM>, <EM>R</EM></SPAN> are considered to be known, empirical constants. <SPAN class="math inline"><EM>F</EM><SUB><EM>B</EM></SUB></SPAN> is left as a parameter that can be determined in a curve fit regression to a time-series of vertical displacement data for a floating biomass.</P></DIV><DIV class="page-section" id="13"><H2 id="determining-buoyant-force-temporal-tracking-algorithm">3.4 Determining Buoyant Force: Temporal Tracking Algorithm</H2><P>Now that we have an expression for vertical displacement as a function of time which depends on the buoyant force as a parameter, we want to determine the magnitude of this buoyant force. In principle, this is not a difficult task as it can be solved using the built-in least-squares optimization curve-fit functions, in say, MATLAB. The difficulty lies in acquiring experimental data to fit to. Our proposed solution to this is to acquire images from a stationary point of view at evenly spaced, small time intervals and perform image segmentation and analysis techniques to track the vertical position of floating biomass frame-to-frame. In reality, the biomass will be clumped into many clusters, so clustering and labeling algorithms are employed. The advantage of using a visual tracking algorithm to observe vertical displacement is that researchers can exactly characterize flotation, benchmark their results, and calculate how different variables affect flotation.</P><FIGURE><FIGCAPTION>Cultured <EM>E. coli</EM> expressing RFP</FIGCAPTION></FIGURE><FIGURE><FIGCAPTION>Binarized ROI</FIGCAPTION></FIGURE><FIGURE><FIGCAPTION>Labeled ROI</FIGCAPTION></FIGURE><FIGCAPTION style="text-align:center"><STRONG>Figure 10</STRONG></FIGCAPTION><P>Below is pseudo-code for a stochastic temporal tracking algorithm that is intended to maintain consistent cluster labeling from frame-to-frame and account for clusters combining and splitting. The time interval between frames should be chosen to be sufficiently small such that the probability of more than one binding/un-binding events occurring between any two frames can be assumed to be zero.</P><UL><LI><P>Load directory with images.</P></LI><LI><P>For <SPAN class="math inline"><EM>i</EM></SPAN> up to number of frames (i.e. time steps):</P></LI><LI><P>Read pixel data from image file.</P></LI><LI><P>Manually set crop margins to ROI encapsulating the region of flotation. (apply same cropping margins in subsequent frames programatically).</P></LI><LI><P>Convert from RGB to grayscale, perform thresholding using Otsu’s method.</P></LI><LI><P>Binarize image based off of threshold, segmentation complete.</P></LI><LI><P>Cluster and label binary image.</P></LI><LI><P>Centroid clusters. Store labels <SPAN class="math inline"><EM>j</EM> = 1, 2, 3, ..., <EM>N</EM></SPAN>, positions <SPAN class="math inline">(<EM>x</EM><SUB><EM>i</EM></SUB><SUP>(<EM>j</EM>)</SUP>, <EM>y</EM><SUB><EM>i</EM></SUB><SUP>(<EM>j</EM>)</SUP>)</SPAN>, and approximate radius for each of the <SPAN class="math inline"><EM>N</EM></SPAN> objects identified in segmentation. Note that the radius is updated at each step to account for changes in cluster morphology affecting magnitude of drag force at each time step.</P></LI><LI><P>if <SPAN class="math inline"><EM>i</EM>&gt; 1</SPAN>:</P></LI><LI><P>For all <SPAN class="math inline"><EM>N</EM></SPAN> labels <SPAN class="math inline"><EM>j</EM></SPAN> in frame <SPAN class="math inline"><EM>i</EM></SPAN> find position <SPAN class="math inline">(<EM>x</EM><SUB><EM>i</EM> − 1</SUB><SUP>(<EM>j</EM><SUP>′</SUP>)</SUP>, <EM>y</EM><SUB><EM>i</EM> − 1</SUB><SUP>(<EM>j</EM><SUP>′</SUP>)</SUP>)</SPAN> with <SPAN class="math inline"><EM>y</EM><SUB><EM>i</EM></SUB> ≥ <EM>y</EM><SUB><EM>i</EM> − 1</SUB></SPAN> s.t. the distance <SPAN class="math inline"><EM>d</EM>((<EM>x</EM><SUB><EM>i</EM> − 1</SUB><SUP>(<EM>j</EM><SUP>′</SUP>)</SUP>, <EM>y</EM><SUB><EM>i</EM> − 1</SUB><SUP>(<EM>j</EM><SUP>′</SUP>)</SUP>), (<EM>x</EM><SUB><EM>i</EM></SUB><SUP>(<EM>j</EM>)</SUP>, <EM>y</EM><SUB><EM>i</EM></SUB><SUP>(<EM>j</EM>)</SUP>))</SPAN> is minimized.</P></LI><LI><P>The label <SPAN class="math inline"><EM>j</EM></SPAN> in the <SPAN class="math inline"><EM>i</EM><SUP><EM>t</EM><EM>h</EM></SUP></SPAN> frame is considered to be a child of the label <SPAN class="math inline"><EM>j</EM>′</SPAN> in the <SPAN class="math inline">(<EM>i</EM> + 1)<SUP><EM>t</EM><EM>h</EM></SUP></SPAN> frame.</P></LI><LI><P>End if-statement.</P></LI></UL><LI><P>End for-loop.</P></LI><LI><P>Define function handle for solution to ODE model for buoyancy.</P></LI><LI><P>Convert cluster tracking data from pixels to spacial units (based on pixel size of image), curve-fit to each set of cluster branches.</P></LI><LI><P>Take average of curve-parameter determination of <SPAN class="math inline"><EM>F</EM><SUB><EM>B</EM></SUB></SPAN> for each branch, analyze distribution, significance of fit, variance of mean.</P></LI><P>But wait! We passively assumed that the number of clusters <SPAN class="math inline"><EM>N</EM><SUP>′</SUP></SPAN> in frame <SPAN class="math inline"><EM>i</EM> − 1</SPAN> was greater than the number of clusters <SPAN class="math inline"><EM>N</EM></SPAN> in frame <SPAN class="math inline"><EM>i</EM></SPAN>. Not to worry, small modification...</P><OL><LI><P><SPAN class="math inline"><EM>N</EM> = <EM>N</EM><SUP>′</SUP></SPAN>:</P></LI><LI><P>If this is the case, the logic holds and we simply have a child label <SPAN class="math inline"><EM>j</EM><SUP>′</SUP></SPAN> that is mapped to parent label <SPAN class="math inline"><EM>j</EM></SPAN>.</P></LI><LI><P><SPAN class="math inline"><EM>N</EM>&gt;<EM>N</EM><SUP>′</SUP></SPAN>:</P></LI><LI><P>Physically, this corresponds to a cluster splitting.</P></LI><LI><P>How do we handle this? No modification of the pseudo-code algorithm is needed. There will simply be two objects with labels <SPAN class="math inline"><EM>j</EM><SUB>1</SUB>, <EM>j</EM><SUB>2</SUB></SPAN>, respectively, to which some <SPAN class="math inline"><EM>j</EM><SUP>′</SUP></SPAN> is mapped to. A <EM>branch</EM> in the tracking tree.</P></LI><LI><P><SPAN class="math inline"><EM>N</EM>&lt;<EM>N</EM><SUP>′</SUP></SPAN></P></LI><LI><P>Non-trivial case.</P></LI><LI><P>Switch search: For all <SPAN class="math inline"><EM>N</EM><SUP>′</SUP></SPAN> labels <SPAN class="math inline"><EM>j</EM><SUP>′</SUP></SPAN> in frame <SPAN class="math inline"><EM>i</EM> + 1</SPAN> find position <SPAN class="math inline">(<EM>x</EM><SUB><EM>i</EM></SUB><SUP>(<EM>j</EM>)</SUP>, <EM>y</EM><SUB><EM>i</EM></SUB><SUP>(<EM>j</EM>)</SUP>)</SPAN> with <SPAN class="math inline"><EM>y</EM><SUB><EM>i</EM></SUB> ≥ <EM>y</EM><SUB><EM>i</EM> − 1</SUB></SPAN> s.t. the distance <SPAN class="math inline"><EM>d</EM>((<EM>x</EM><SUB><EM>i</EM> − 1</SUB><SUP>(<EM>j</EM><SUP>′</SUP>)</SUP>, <EM>y</EM><SUB><EM>i</EM> − 1</SUB><SUP>(<EM>j</EM><SUP>′</SUP>)</SUP>), (<EM>x</EM><SUB><EM>i</EM></SUB><SUP>(<EM>j</EM>)</SUP>, <EM>y</EM><SUB><EM>i</EM></SUB><SUP>(<EM>j</EM>)</SUP>))</SPAN> is minimized.</P></LI><LI><P>There will exist a label <SPAN class="math inline"><EM>j</EM></SPAN> in the <SPAN class="math inline"><EM>i</EM><SUP><EM>t</EM><EM>h</EM></SUP></SPAN> frame such that two labels <SPAN class="math inline"><EM>j</EM><SUB>1</SUB><SUP>′</SUP>, <EM>j</EM><SUB>2</SUB><SUP>′</SUP></SPAN> in the <SPAN class="math inline">(<EM>i</EM> + 1)<SUP><EM>t</EM><EM>h</EM></SUP></SPAN> frame map to it. In other words, two clusters combine.</P></LI></OL><P>Note: The position tracking effectively forms a directed acyclic graph (DAG)</P><FIGURE><FIGCAPTION style="text-align:center"><STRONG>Figure 11: A few potential layers of a simple example with 3 initial clusters (nodes) to illustrate the 3 cases.</STRONG></FIGCAPTION></FIGURE></DIV><DIV class="page-section" id="14"><H2 id="discussion-on-maximum-carrying-capacity">3.5 Discussion On Maximum Carrying Capacity</H2><P>For this discussion, consider the buoyant force <SPAN class="math inline"><EM>F</EM><SUB><EM>B</EM></SUB></SPAN> to be a known, constant value.
				 
				In order for the biomass to float the net force on it must be greater than zero. By equation <A href="#Fnet" data-reference-type="ref" data-reference="Fnet">[6]</A> we have the following constraint on buoyant force: 

        $$\begin{equation}\label{forceconstraint}
        F_B \geq mg + 6\pi \eta R v(t), \ \forall t
        \end{equation}$$

        <FIGCAPTION style="text-align:right"><STRONG>Equation 13.</STRONG></FIGCAPTION> 

        The right side of the above inequality has strictly positive monotonicity with respect to <SPAN class="math inline"><EM>v</EM></SPAN>. <SPAN class="math inline"><EM>v</EM></SPAN> is a strictly increasing function so its max on a closed, bounded interval is at the upper bound. Let <SPAN class="math inline"><EM>v</EM><SUB><EM>m</EM><EM>a</EM><EM>x</EM></SUB> = <EM>v</EM>(<EM>t</EM><SUB><EM>f</EM></SUB>)</SPAN>, where <SPAN class="math inline"><EM>t</EM><SUB><EM>f</EM></SUB></SPAN> is the solution to <SPAN class="math inline"><EM>z</EM>(<EM>t</EM><SUB><EM>f</EM></SUB>) = <EM>h</EM></SPAN>, <SPAN class="math inline"><EM>h</EM></SPAN> being the height required from the start point to the top of container, <SPAN class="math inline"><EM>z</EM></SPAN> being as given in equation <A href="#z" data-reference-type="ref" data-reference="z">[12]</A>. The inequality in equation <A href="#forceconstraint" data-reference-type="ref" data-reference="forceconstraint">[13]</A> is thus equivalent to the following: 

        $$\begin{equation}
        F_B \geq m_{\mathrm{E. \ coli}}g + 6\pi \eta R_{\mathrm{E. \ coli}} v_{\max}
        \end{equation}$$

        <FIGCAPTION style="text-align:right"><STRONG>Equation 14.</STRONG></FIGCAPTION>
        
        Taking <SPAN class="math inline"><EM>m</EM><SUB><EM>E</EM>. <EM>c</EM><EM>o</EM><EM>l</EM><EM>i</EM></SUB></SPAN> to be the approximate mass of an <EM>E. coli</EM> cell (without bound particle), <SPAN class="math inline"><EM>R</EM><SUB><EM>E</EM>. <EM>c</EM><EM>o</EM><EM>l</EM><EM>i</EM></SUB></SPAN> to be the approximate radius of an <EM>E. coli</EM> cell in a spherical approximation, this constraint on buoyant force influences goals in the wet lab by setting a minimum requirement for observed buoyant force in modifying ARG1 for optimized flotation performance.
				Once ARG1 is optimized for flotation and an estimate for <SPAN class="math inline"><EM>F</EM><SUB><EM>B</EM></SUB></SPAN> is obtained, the maximal carrying capacity can be obtained as follows: 

        <SPAN class="math display">$$m \leq \frac{F_B - 6\pi \eta R_{\mathrm{E. \ coli}} v_{\mathrm{max}}}{g}$$</SPAN><FIGCAPTION style="text-align:right"><STRONG>Equation 15.</STRONG></FIGCAPTION>

        Here <SPAN class="math inline"><EM>m</EM> = <EM>m</EM><SUB><EM>E</EM>. <EM>c</EM><EM>o</EM><EM>l</EM><EM>i</EM></SUB> + <EM>m</EM><SUB><EM>b</EM><EM>o</EM><EM>u</EM><EM>n</EM><EM>d</EM></SUB></SPAN> (sum of cell mass with mass of bound particle). Solving for <SPAN class="math inline"><EM>m</EM><SUB><EM>b</EM><EM>o</EM><EM>u</EM><EM>n</EM><EM>d</EM></SUB></SPAN>: <SPAN class="math display">$$m_{\mathrm{bound}} \leq \frac{F_B - 6\pi \eta R_{\mathrm{E. \ coli}} v_{\mathrm{max}}}{g} - m_{\mathrm{E.\ coli}}$$</SPAN><FIGCAPTION style="text-align:right"><STRONG>Equation 16.</STRONG></FIGCAPTION>
        Diving by the molecular or atomic mass, denote this 
        <SPAN class="math inline"><EM>m</EM><SUB><EM>p</EM><EM>a</EM><EM>r</EM><EM>t</EM><EM>i</EM><EM>c</EM><EM>l</EM><EM>e</EM></SUB></SPAN> of the substance to be sequestered we have the following value for maximal carrying capacity <SPAN class="math inline"><EM>δ</EM></SPAN> in the bioreactor model: <SPAN class="math display">$$\delta = \frac{1}{m_{particle}}\big[\frac{F_B - 6\pi \eta R_{\mathrm{E. \ coli}} v_{\mathrm{max}}}{g} - m_{\mathrm{\textit{E. coli}}}\big]$$</SPAN><FIGCAPTION style="text-align:right"><STRONG>Equation 17.</STRONG></FIGCAPTION>
        However, in the bioreactor model, the effluent is pumped from the main reactor in such a way that cells will not sink, so <SPAN class="math inline"><EM>v</EM><SUB><EM>m</EM><EM>a</EM><EM>x</EM></SUB></SPAN> can be set to 0 and we see: <SPAN class="math display">$$\delta \approx \frac{1}{m_{particle}}\big[\frac{F_B}{g} - m_{\mathrm{\textit{E. coli}}}\big]$$</SPAN><FIGCAPTION style="text-align:right"><STRONG>Equation 18.</STRONG></FIGCAPTION>
        See §2.3. This could influence the choice of expression levels in future cell surface engineering techniques. Too many surface receptors could result in accumulation of too much particle mass, eliminating the ability for flotation.</P></DIV><DIV class="page-section" id="15"><H1 id="growth-dynamics">4. Growth Dynamics</H1></DIV><DIV class="page-section" id="16"><H2 id="objective">4.1 Objective</H2><P>Our objective for this section is to describe the growth dynamics of BL21 <EM>E. coli</EM> strain in lysogeny broth (LB). We have obtained two sets of observations from our wet lab team:</P><OL><LI><P>25 observations with <SPAN class="math inline"><EM>Δ</EM><EM>t</EM> = 30</SPAN>mins</P></LI><LI><P>40 observations with <SPAN class="math inline"><EM>Δ</EM><EM>t</EM> = 30</SPAN>mins</P></LI></OL></DIV><DIV class="page-section" id="17"><H2 id="approach">4.2 Approach</H2><P>To model all the observations, we will use the Gompertz function[10], which can be expressed in the following way: 
					$$f(t) = ae^{-be^{-ct}}$$
					with: 
					$$b, c &gt;0$$
					The reason for using this model is that it best describes the lag phase, the exponential phase and stationary phase.</P></DIV><DIV class="page-section" id="18"><H2 id="results">4.3 Results</H2><P>The observations were given to us in <SPAN class="math inline"><EM>O</EM><EM>D</EM><SUB>600</SUB></SPAN>. We know that for bacterial cell cultures <SPAN class="math inline"><EM>O</EM><EM>D</EM><SUB>600</SUB></SPAN> of <SPAN class="math inline">1.0 = 8.0 × 10<SUP>8</SUP></SPAN>cells<SPAN class="math inline">/</SPAN>ml [6].</P><UL><LI><P>For the first set of observations we fit the curve to our data.</P><FIGURE><FIGCAPTION style="text-align:center"><STRONG>Figure 12: Fitted data</STRONG></FIGCAPTION></FIGURE><P>with the following parameters:</P><PRE style="line-height: 1.5em;"><CODE>     General model:
				     ans(x) = a*exp(-b*exp(-c*x))
				     Coefficients (with 95% confidence bounds):
				       a =   3.876e+08  (2.782e+08, 4.97e+08)
				       b =       18.97  (-22.7, 60.64)
				       c =      0.4938  (0.1035, 0.884)</CODE></PRE></LI><LI><P>For the second set of observations, we fit the same model. However, we obtain different parameters, since we enter the stationary phase after the exponential phase, which we did not observe in the first data set. But we also notice that they fall into the 95% confidence interval for the parameters of the first observations. Hence we can assume, with some certainty, that the parameters we got for this curve fitting process would also fit for the previous one.</P><FIGURE><FIGCAPTION style="text-align:center"><STRONG>Figure 13: Fitted data</STRONG></FIGCAPTION></FIGURE><P>The parameters are as follows:</P><PRE style="line-height: 1.5em;"><CODE> General model:
				     ans(x) = a*exp(-b*exp(-c*x))
				     Coefficients (with 95% confidence bounds):
				       a =   5.707e+08  (5.355e+08, 6.059e+08)
				       b =       5.127  (3.785, 6.469)
				       c =      0.1989  (0.1603, 0.2376)</CODE></PRE></LI></UL></DIV><DIV class="page-section" id="19"><H1 id="discussion-and-future-directions">5. Discussion and Future Directions</H1><P>Based on the results of our differential bioreactor model we postulate that a bioreactor of this design could perform at appropriately small time-scales with a sufficiently optimized flotation construct in a bioremediation context. This bodes well for future laboratory endeavours where the bioreactor schema along with an engineered cell-line optimized for flotation from gas vesicle formation could be tested in a small scale laboratory model of the system to test its empirical performance. This is useful for model validation, and proof of concept.
				An improvement to the bioreactor could be seen in the form of incorporation of an experimental growth dynamics model as demonstrated in the Growth Dynamics section with the Goempertz equation. The Monod equation was chosen initially for this model because it is scalable and its parameters are known for BL21; however, it only describes the exponential phase of growth and it is not guaranteed that the biomass will remain in exponential phase during the bioreactor process. Another shortcoming is that it is a single-substrate model and in practice, this would not be the case as the likely nutrient source would be from carbohydrates, lipids, and organics present in the waste-water effluent. For future application-based analysis using the bioreactor model, Goempertz coefficients could be determined for a biomass of industrially relevant size for the volume demands dictated by the industry requiring bioremediation using a similar experimentation and analysis technique as described in the Growth Dynamics section.
				To further enrich the analysis of the behaviour of the bioreactor model, in addition to the existing sensitivity analysis, identification and characterization of stable points resulting from different combinations of parameter values could be performed.
				It would also be informative to apply the stochastic temporal tracking algorithm to real flotation data (images), once obtained, to estimate the buoyant force for ARG1 and compare different modifications of ARG1 to determine an optimal gene combination for flotation.
				A next major step is coupling cellular flotation assays with cell surface engineering for application-based testing for different bioremediation tasks. Much research has been done relating to selecting peptide sequences for metal binding [4][5]. Existing techniques would include constitutively expressing cell-surface protein along with OD regulated IPTG induction ARG1, or optimised constructs of ARG1 for flotation.</P></DIV><DIV class="page-section" id="20"><H1 id="appendix">6. Appendix</H1><H2 id="variable-and-parameter-definitions-and-units">Variable and Parameter Definitions and Units</H2><UL><LI><P><SPAN class="math inline"><EM>γ</EM></SPAN> Yield Constant (unitless)</P></LI><LI><P><SPAN class="math inline"><EM>κ</EM></SPAN><EM>E</EM>. coli Death Rate (<SPAN class="math inline"><EM>h</EM><SUP>−</SUP>1</SPAN>)</P></LI><LI><P><SPAN class="math inline">D = F/V</SPAN> Dilution Rate (<SPAN class="math inline"><EM>h</EM><SUP>−</SUP>1</SPAN>)</P></LI><LI><P><SPAN class="math inline"><EM>F</EM></SPAN> Flow Rate (<SPAN class="math inline"><EM>L</EM><EM>h</EM><SUP>−</SUP>1</SPAN>)</P></LI><LI><P><SPAN class="math inline"><EM>V</EM></SPAN> Total Volume (<SPAN class="math inline"><EM>L</EM></SPAN>)</P></LI><LI><P><SPAN class="math inline"><EM>β</EM></SPAN> Detachment Rate (<SPAN class="math inline"><EM>h</EM><SUP>−</SUP>1</SPAN>)</P></LI><LI><P><SPAN class="math inline"><EM>α</EM></SPAN> Attachment Rate (<SPAN class="math inline"><EM>h</EM><SUP>−</SUP>1</SPAN>)</P></LI><LI><P><SPAN class="math inline"><EM>μ</EM><SUB><EM>m</EM></SUB></SPAN> Maximum Specific Growth Rate (<SPAN class="math inline"><EM>h</EM><SUP>−</SUP>1</SPAN>)</P></LI><LI><P><SPAN class="math inline"><EM>K</EM><SUB><EM>s</EM></SUB></SPAN> Monod Constant (mol <SPAN class="math inline"><EM>L</EM><SUP>−</SUP>1</SPAN>)</P></LI><LI><P><SPAN class="math inline"><EM>δ</EM></SPAN> Maximum Carrying Capacity Constant (unitless)</P></LI><LI><P><SPAN class="math inline"><EM>N</EM><SUB><EM>i</EM></SUB></SPAN> Initial Cellular Concentration (mol <SPAN class="math inline"><EM>L</EM><SUP>−</SUP>1</SPAN>)</P></LI><LI><P><SPAN class="math inline"><EM>S</EM><SUB><EM>i</EM></SUB></SPAN> Initial Limiting Substrate Concentration (mol <SPAN class="math inline"><EM>L</EM><SUP>−</SUP>1</SPAN>)</P></LI><LI><P><SPAN class="math inline"><EM>X</EM><EM>u</EM><SUB><EM>i</EM></SUB></SPAN> Initial Particle Concentration within Waste-water (mol <SPAN class="math inline"><EM>L</EM><SUP>−</SUP>1</SPAN>)</P></LI><LI><P><SPAN class="math inline"><EM>X</EM><EM>w</EM></SPAN> Concentration of Bound Particle (mol <SPAN class="math inline"><EM>L</EM><SUP>−</SUP>1</SPAN>)</P></LI><LI><P><SPAN class="math inline"><EM>η</EM></SPAN> Dynamic viscosity of waste-water (<SPAN class="math inline"><EM>k</EM><EM>g</EM> <EM>m</EM><EM>s</EM><SUP>−</SUP>1</SPAN>)</P></LI><LI><P><SPAN class="math inline"><EM>m</EM><SUB><EM>p</EM></SUB><EM>a</EM><EM>r</EM><EM>t</EM><EM>i</EM><EM>c</EM><EM>l</EM><EM>e</EM></SPAN> Atomic or molecular mass of substance to be sequestered (<SPAN class="math inline"><EM>g</EM> <EM>m</EM><EM>o</EM><EM>l</EM><SUP>−</SUP>1</SPAN>)</P></LI><LI><P><SPAN class="math inline"><EM>m</EM><SUB><EM>E</EM></SUB>. <EM>c</EM><EM>o</EM><EM>l</EM><EM>i</EM></SPAN> Aproximate mass of <EM>E</EM>. coli cell</P></LI><LI><P><SPAN class="math inline"><EM>R</EM><SUB><EM>E</EM></SUB>. <EM>c</EM><EM>o</EM><EM>l</EM><EM>i</EM></SPAN> Approximate radius of <EM>E</EM>. coli cell in spherical approximation (<SPAN class="math inline"><EM>m</EM></SPAN>)</P></LI><LI><P><SPAN class="math inline"><EM>v</EM><SUB><EM>m</EM></SUB><EM>a</EM><EM>x</EM></SPAN> Maximum achievable velocity based on vertical displacement specification.</P></LI></UL><H2 id="the-matlab-bioreactor-ode-code">The MATLAB Bioreactor ODE Code</H2><PRE style="line-height: 1.5em;"><CODE>
gamma = 1.1; % yield constant (?)
kappa = 0.01; % death rate h^-1
D = 3; % dilution/flow rate h^-1
beta = 0.03; % detachment rate of particle from biomass surface h^-1
alpha = 1.5; % attachment rate h^-1
mu_m = 0.8; % maximum specific growth rate (monod equation) unitless
K_s = 2.87*10^(-7); % monod constont mol L^-1
delta = 1.5; % maximum carrying capacity
N_i = 0.4; % initial cellular concentration L^-1
S_i = 0.63; % initial substrate concentration L^-1
Xu_i = 0.5; % initial unbound metal concentration L^-1
Xw_i = 0; % initial bound metal concentration L^-1
epsilon = 0.0001; % Acceptable metal concentration threshold

f = @(t,x) [N_i * ((mu_m * x(2)) / (K_s + x(2))) - x(1) * (D + kappa);
    -((mu_m * x(2)) / (K_s + x(2))) * x(1) / gamma;
    beta * x(4) - alpha * x(3) * (delta * x(1) - x(4))/(delta * x(1));
    -(beta * x(4) - alpha * x(3) * (delta * x(1) - x(4))/(delta * x(1))) - D*x(4)];

[t,xa] = ode45(f,[0 50],[N_i S_i Xu_i Xw_i]);
fprintf('ode solved\n')

ind = find(xa(:,3) &lt;= epsilon);
xap = xa(1:ind,:);
tp = t(1:ind);

figure;
plot(tp,xap(:,1)); % Cell concetration mol L^-1
hold on 
plot(tp,xap(:,2)); % Substrate concentration
hold on
plot(tp,xap(:,3)) % Unbound metal concentration mol L^-1
hold on
plot(tp,xap(:,4)) % Bound metal concentration mol L^-1
hold on
set(findall(gca, 'Type', 'Line'),'LineWidth',2);

legend('E. coli Pop', 'Substrate','Unbound Particle', 'Bound particle in MR')
xlabel('Time (h)'), ylabel('mol/L')
</CODE></PRE></DIV><DIV class="page-section" id="21"><H1>7. References</H1><P>[1] Oladeji, S. O., &amp; Saeed, M. D. (2015). <EM>Assessment of cobalt levels in wastewater, soil and vegetable samples grown along Kubanni stream channels in Zaria, Kaduna State, Nigeria</EM>. African Journal of Environmental Science and Technology, 9(10), 765-772. doi:10.5897/ajest2015.1969</P><P>[2] Whittaker MM, Mizuno K, Bächinger HP, Whittaker JW. <EM>Kinetic Analysis of the Metal Binding Mechanism of Escherichia coli Manganese Superoxide Dismutase</EM>. Biophysical Journal. 2006;90(2):598-607. doi:10.1529/biophysj.105.071308.</P><P>[3] <EM>Monod substrate affinity constant (Ks) for strain ML30 of <EM>E. coli</EM> growing on glucose as the only source of carbon and energy</EM>. (n.d.). Retrieved October 13, 2018 from http://bionumbers.hms.harvard.edu/bionumber.aspx?s=n&amp;v=1&amp;id=111051.</P><P>[4] N., Cetinel, S., Omar, S. I., Tuszynski, J. A., &amp; Montemagno, C. (2017). <EM>A computational method for selecting short peptide sequences for inorganic material binding.</EM> Proteins: Structure, Function, and Bioinformatics, 85(11), 2024-2035. doi:10.1002/prot.25356</P><P>[5] Li, Pengsong &amp; Tao, Huchun. (2013). <EM>Cell surface engineering of microorganisms towards adsorption of heavy metals.</EM> Critical reviews in microbiology. 41. 10.3109/1040841X.2013.813898.</P><P>[6] <EM>Bacterial cell number (OD600)</EM> (n.d.). Retrieved October 13, 2018 from http://www.labtools.us/bacterial-cell-number-od600/</P><P>[7] Bengt Carlsson. (2009). <EM>An introduction to modeling of bioreactors.</EM> Dept of Systems and Control, Information Technology Uppsala University. Retrieved October 13, 2018 from https://www.it.uu.se/edu/course/homepage/modynsyst/vt11/Lecture/DynSystBior2009.pdf.</P><P>[8] <EM>Monod substrate affinity constant (Ks) for strain ML30 of <EM>E. coli</EM> growing on glucose as the only source of carbon and energy</EM>. (n.d.). Retrieved October 13, 2018</P><P>[9] Chen, L., &amp; Chai, L. (2005). <EM>Mathematical Model and Mechanisms for Biofilm Wastewater Treatment Systems</EM>. World Journal of Microbiology and Biotechnology, 21(8-9), 1455-1460. doi:10.1007/s11274-005-6565-2</P><P>[10] Parolini, N, &amp; Carcano, S. (2009). <EM>A MODEL FOR CELL GROWTH IN BATCH BIOREACTORS</EM>. Faculty of Systems Engineering, Polytechnic University of Milan. Retrieved October 13, 2018 from https://www.politesi.polimi.it/bitstream/10589/2082/1/2010_07_Carcano.pdf.</P></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></BODY></HTML>