"
Page
Discussion
View source
History
teams
Log in
 
Team:HKUST/Group2
From 2009.igem.org
Salt and Soap template
Home
Our Team
Project description
Background
Experimental design
Lab Notebook
Experimental result
Parts Submitted
Protocol list
Other resources
Future plan
Gallery
Consolidation
Acknowledgement
Welcome
Gene deletion
We are using a PCR-based gene deletion strategy (Baudin et al., Nucl. Acids Res. 21, 3329-3330, 1993 and Wach et al., Yeast 10, 1793-1808, 1994). Basically, knock-out fragments are generated by PCR, transformed into yeast strain, and integrated into the yeast genome by homologous recombination. Successful deletion replaces the yeast ORF with antibiotic resistant gene present in the knock-out fragment which serves as the selection marker.
Cassette plasmids, PCR primers and PCR conditions for amplification of knock-out fragments are chosen or designed according to a gene deletion toolbox (C. Janke et al., Yeast 2004; 21: 947–962.)
I. Amplification of knock-out fragment
1. Primer design
The figure illustrates the design of the primers S1 and S2 that are used for the amplification of the cassettes. S1-primer, 45–55 bases upstream of the ATG (including ATG = start codon) of the gene, followed by 5’-CGTACGCTGCAGGTCGAC-3’; S2-primer, the reverse complement of 45–55 bases downstream of the STOP-codon including STOP) of the gene, followed by 5’-ATCGATGAATTCGAGCTCG-3’.
2. PCR conditions
(1) Amplification of FAR1 knock-out fragment (1822bp)
(2)Amplification of GPA1 knock-out fragment (1439bp)
(3) Amplification of ARO9 knock-out fragment (1822bp)
II. Deletion Strain Confirmation
The correct replacement of the gene with hphNT1 or natNT2 is verified in the mutants by the appearance of PCR products of the expected size using primers that span the left and right junctions of the deletion module within the genome. Four ORF-specific confirmation primers (A, B, C, and D primers) are chosen for each ORF disruption.
1. Primer design
The figure illustrates the primers designed for colony PCR that is performed to confirm successful deletion of yeast ORF. The "A" and "D" primers are positioned 200-400 bp from the start and stop codons of the gene, respectively. The "B" and "C" primers are located within the coding region of the ORF and, when used with the A or D primers, give product sizes between 250-1000 bp. The "hphB" and "hphC" primers are internal to the hphNT1 module while the "natB" and "natC" primers are internal to the natNT2 module.
2. PCR    For each colony growing on the selective plate of each deletion, four set of primers are used to confirm the successful deletion of ORF. The junctions of the disruption are verified by amplification of genomic DNA using primers "A" and "hphB"(or “natB”) and primers "hphC"(or “natC”) and "D". Deletion of the ORF is verified by the absence of a PCR product using primers "A" with "B" and "C" with "D".
iGEM 2009
Retrieved from " http://2009.igem.org/Team:HKUST/Group2 "
Recent changes
What links here
Related changes
Special pages
My preferences
Printable version
Permanent link
Privacy policy
Disclaimers
