<HTML xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en" dir="ltr">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki  ltr ns-0 ns-subject page-Team_HKUST_Back1"><DIV id="globalWrapper"><DIV id="top-section"><DIV id="p-logo"><A href="/Main_Page" title="Main Page">&quot;
	    </A></DIV><DIV id="menubar" class="left-menu"><UL><LI class="selected"><A href="/Team:HKUST/Back1">Page               </A></LI><LI class="new"><A href="/wiki/index.php?title=Talk:Team:HKUST/Back1&amp;action=edit&amp;redlink=1">Discussion               </A></LI><LI><A href="/wiki/index.php?title=Team:HKUST/Back1&amp;action=edit">View source               </A></LI><LI><A href="/wiki/index.php?title=Team:HKUST/Back1&amp;action=history">History               </A></LI><LI style="color:#808080;cursor:default">teams</LI></UL></DIV><DIV class="right-menu" id="menubar"><UL><LI id="pt-login"><A href="/wiki/index.php?title=Special:UserLogin&amp;returnto=Team:HKUST/Back1" title="You are encouraged to log in; however, it is not mandatory [o]" accesskey="o">Log in</A></LI></UL></DIV><DIV id="search-controls"><FORM action="/Special:Search" id="searchform"> </FORM></DIV></DIV><DIV id="content"><H1 class="firstHeading">Team:HKUST/Back1</H1><DIV id="bodyContent"><H3 id="siteSub">From 2009.igem.org</H3><P><TITLE>Salt and Soap template</TITLE></P><DIV id="containerxx"><DIV id="borderxx"><DIV id="mainxx"><DIV id="leftxx"><DIV id="menuxx"><UL><LI><A href="https://2009.igem.org/Team:HKUST">Home</A></LI><LI><A href="https://2009.igem.org/Team:HKUST/Team">Our Team</A></LI><LI><A href="https://2009.igem.org/Team:HKUST/Project">Project Description</A></LI><B><SPAN style="color:green"><LI>Main Parts</LI></SPAN></B><LI><A href="https://2009.igem.org/Team:HKUST/OdorantSensing">Odorant Sensing</A></LI><LI><A href="https://2009.igem.org/Team:HKUST/AttractantProduction">Attractant Production</A></LI><LI><A href="https://2009.igem.org/Team:HKUST/ToxinProduction">Toxin Production</A></LI><B><SPAN style="color:green"><LI>Resources</LI></SPAN></B><LI><A href="https://2009.igem.org/Team:HKUST/Lab Notebook">Lab Notebook</A></LI><LI><A href="https://2009.igem.org/Team:HKUST/Parts">Parts Submitted </A></LI><LI><A href="https://2009.igem.org/Team:HKUST/Protocols">Protocol List</A></LI><LI><A href="https://2009.igem.org/Team:HKUST/Resourses">Other Resources</A></LI></UL></DIV><DIV id="menubottom"><UL><LI><A href="https://2009.igem.org/Team:Gallery">Gallery</A></LI><LI><A href="https://2009.igem.org/Team:Biosafety">Biosafety</A></LI><LI><A href="https://2009.igem.org/Team:Acknowledgement">Acknowledgement</A></LI></UL></DIV></DIV><DIV id="rightxx"><DIV class="contentodS_b"><H3>a</H3></DIV><DIV class="contentxx"><P>GPCRs</P>
  G protein-coupled receptors (GPCRs) are the main sensors of cells. They respond to ligands such as hormones, neurotransmitters, light, odorants, taste substances, pheromones, and a variety of medicines. GPCRs comprise one of the largest protein superfamilies: it is estimated that there are 1050 and 160 GPCR genes in the genomes of <EM>Caenorhabditis elegans</EM> and <EM>Drosophila melanogaster</EM>, corresponding to 5.5% and 1% of their total genes, respectively[1]. Identifying GPCR ligands may lead to the discovery of novel hormones and neurotransmitters, and thus contribute to the development of new medicines. In fact, GPCRs are known to target 30-60% of present medicines[2]. 
GPCRs are seven transmembrane proteins (Fig1). When stimulated by extracellular ligands, the receptors act through heterotrimeric G proteins to regulate intracellular effector proteins. The G proteins are composed of Gα,Gβ and Gγ subunits. In the inactive state, the Gα-subunit is bound to GDP and associated with the Gβ and Gγ dimer. Upon ligand binding, the receptor stimulates the release of GDP from Gα, allowing the binding of GTP to the subunit. Gα-GTP is released from the Gβ and Gγ dimer, and the dissociated subunits regulate the activity of effector proteins such as adenylate cyclase, phospholipase C, mitogen-activated protein kinase (MAP kinase) cascades and ion channels. 
Fig.1 Model of GPCRs. Adopted from <EM>Hinako Suga, Tatsuya Haga, 2007</EM>.[3] 
  Olfactory receptors (OR) are also seven transmembrane proteins. Insect OR gene families have been identified in the fruit fly <EM>Drosophila melanogaster</EM>, the malaria vector mosquito <EM>Anopheles gambiae</EM>, the yellow fever vector mosquito <EM>Aedes aegypti</EM>, the honeybee <EM>Apis mellifera</EM>, the silkmoth <EM>Bombyx mori</EM>, and the flour beetle <EM>Tribolium castaneum</EM>.[4] Though in vertebrates ORs consist largely of GPCRs, in insects there remains doubt whether ORs belong to GPCRs. However, various studies have been carried out on <EM>Drosophila</EM> and <EM>C. elegans</EM> G-protein coupled olfactory receptors, and have uncovered much of their structures and functions. 
Based on these facts, in this iGEM project, we choose to study the <EM>C.elegans</EM> ODR-10 GPCR and the rat RI7 GPCR to demonstrate the working principle in our design. ODR-10 protein is an odorant receptor in <EM>C. elegans</EM>. It responds to a volatile odorant diacetyl and triggers chemotaxis of <EM>C.elegans</EM> towards the molecule. RI7 is a rat odorant receptor that has been functionally expressed in yeast. We will show that the C.elegans odorant receptor can also be functionally expressed in yeast and later we could potentially extend this study method to other insect ORs or other GPCRs.<P>Yeast Pheromone Sensing Pathway</P><EM>Saccharomyces cerevisiae</EM> contains two GPCR-based signalling pathways, one that mediates pheromone response during mating and one that senses glucose to regulate filamentous differentiation in diploid cells and invasive growth in haploid cells[5]. There appears to be little crosstalk between these two pathways. By providing a null background, the yeast pheromone response pathway provides a robust and experimentally tractable system to study heterologous GPCRs. 
Haploid <EM>S. cerevisiae</EM> exists in one of two mating types(a and α), and mating occurs between cells of opposite types and is controlled by the exchange of mating pheromones; a-cells secrete a-factor and express the α-factor receptor (Ste2), whereas α-cells secrete α-factor and express the a-factor receptor (Ste3). Both receptors couple to the same heterotrimeric G protein composed of Gα(Gpa1), Gβ(Ste4) and Gγ (Ste18) subunits. Binding of the mating pheromone to the receptor results in the release of the Gβγ dimer, which then interacts with effectors to bring about conjugation (Fig.2A). A key role of Gβγ is to activate a MAP kinase cascade in which sequential activity of Ste11 (MEK kinase), Ste7 (MAP kinase kinase or MEK) and Fus3 (MAP kinase) leads to activation of a transcription factor (Ste12) that induces gene expression in preparation for mating. Fus3 also activates the cyclindependent kinase inhibitor Far1 to bring about cell cycle arrest[6].<P>The Expression of Heterologous GPCRs in <EM>S. cerevisiae</EM></P><EM>S. cerevisiae</EM> is chosen in this project as the ideal system to study heterologous GPCR for several reasons(Box 1). 
   To functionally express heterologous GPCRs, we need to do some modifications to the strain to improve the localization of the receptor to the cell membrane and the coupling of the heterologous receptor to the Gα subunit (Fig.2B). <P>Modifying GPCRs</P>
  Though some GPCRs may be degraded or retained within intracellular compartments, most GPCRs can be expressed unmodified in yeast. Rat I7 GPCR is one of them, which has been shown to be functionally expressed in the yeast system[8]. Sequence analysis of ORs and GPCRs have shown that the N termini of these receptors are involved in plasma membrane localization, whereas the C termini define the specificity for G protein interaction; the spanning transmembrane domains two (TMII) to seven (TMVII) are the ligand-binding regions. It is possible that a chimeric OR can be expressed using the N termini and C termini of known GPCRs that can be functional expressed, such as RI7, and the ligand binding region of our desired ORs.(For more details, see Experiment Design). <P>Modifying the G protein</P>
  The yeast Gα (Gpa1) has been shown to interact with several non-yeast GPCRs. This functional coupling can be improved by replacing Gpa1 with mammalian Gα of the related GPCR, since yeast Gpa1, Ste4 and Ste18 are structurally and functionally similar to mammalian Gα. For example, rat Gαolf  and RI7 are both expressed in yeast and RI7 is effectively coupled to Gαolf[8]. <P>Developing Reporter Genes</P> 
  Most yeast GPCR assays use reporter genes under the control of the Ste12-inducible FUS1 promoter. We can either introduce β-galactosidase to provide a quantitative assay suitable for high-throughput analysis; or we can incorporate GFP or luciferase under the FUS1 promoter to fast detect the signal transduction. This method has also been verified in previous studies[8].<LI><A href="https://2009.igem.org/Team:HKUST/Back1">Background</A></LI><LI><A href="https://2009.igem.org/Team:HKUST/Group1">Experimental Design</A></LI><LI><A href="https://2009.igem.org/Team:HKUST/Part1">Parts Design</A></LI><LI><A href="https://2009.igem.org/Team:HKUST/Result1">Experimental Results</A></LI><LI><A href="https://2009.igem.org/Team:HKUST/Future1">Future Work</A></LI><LI><A href="https://2009.igem.org/Team:HKUST/Ref1">References</A></LI></DIV></DIV></DIV></DIV><DIV id="footerxx"><DIV id="copyright"><SPAN> iGEM 2009 </SPAN></DIV></DIV></DIV><DIV class="printfooter">
Retrieved from &quot;<A href="http://2009.igem.org/Team:HKUST/Back1">http://2009.igem.org/Team:HKUST/Back1</A>&quot;</DIV></DIV></DIV><DIV id="footer-box"><DIV id="footer"><UL id="f-list"><LI id="t-recentchanges"><A href="/Special:RecentChanges" title="Recent changes">Recent changes</A></LI><LI id="t-whatlinkshere"><A href="/Special:WhatLinksHere/Team:HKUST/Back1" title="List of all wiki pages that link here [j]" accesskey="j">What links here</A></LI><LI id="t-recentchangeslinked"><A href="/Special:RecentChangesLinked/Team:HKUST/Back1" title="Recent changes in pages linked from this page [k]" accesskey="k">Related changes</A></LI><LI id="t-specialpages"><A href="/Special:SpecialPages" title="List of all special pages [q]" accesskey="q">Special pages</A></LI><LI><A href="/Special:Preferences">My preferences</A></LI></UL></DIV><DIV id="footer"><UL id="f-list"><LI id="t-print"><A href="/wiki/index.php?title=Team:HKUST/Back1&amp;printable=yes" title="Printable version of this page [p]" accesskey="p">Printable version</A></LI><LI id="t-permalink"><A href="/wiki/index.php?title=Team:HKUST/Back1&amp;oldid=169047" title="Permanent link to this revision of the page">Permanent link</A></LI><LI id="privacy"><A href="/2009.igem.org:Privacy_policy" title="2009.igem.org:Privacy policy">Privacy policy</A></LI><LI id="disclaimer"><A href="/2009.igem.org:General_disclaimer" title="2009.igem.org:General disclaimer">Disclaimers</A></LI></UL></DIV></DIV></DIV></BODY></HTML>