"
Page
Discussion
View source
History
teams
Log in
 
Team:UC Chile/Side Project
From 2013.igem.org
Wiki-IGEM
Hidden
Links
Team
Team Members
PSB Lab
Photo gallery
Human
Practices
Home
More
Collaborations
Judging Criteria
Acknowledgments
Project
Whateversisoma
Biobricks
Biosafety
Side project
Methodology
Protocols
Lab Notebook
Modelling
Mathematical Model
Software
Home
Overview
Achievements
Side Project
Agrobacterium tumefaciens is a bacterium with an infective method in which a DNA fragment is randomly inserted into the plant genome. Biotechnology has taken advantage of this characteristic to generate transgenic plants.  However, this is a rather inefficient procedure because the insertion is not site-specific and can occur in between essential genes of the plant.
In this project, we designed a system that can insert any DNA sequence of interest in specific sites on the plant’s genome. This was going to be achieved by the utilization of a modified binary vector that we pretended to add to Parts Registry. This plasmid would have the sequences for TALENs, proteins that can recognize a specific sequence of interest in the plant’s genome and generate a double strand break in the DNA, connected with a linker to the sequence of the VirD2 protein that would take the DNA of interest inside the plant’s nucleus.
This binary vector couldn’t be finished for the Regional Jamboree because we had problems with the construction process of the specific TALENs for our project.
Figure 1. General schema of the project.
The image shows the gene that codifies the GFP been recognized by the TALENs, previous to the insertion of the gen of interest (G.O.I)
Introduction
Agrobacterium tumefaciens is a pathogenic alpha proteobacterium that causes the crown gall disease when it is induced by hormones secreted by wounds in plants that result in the production of tumors. The infection causes the synthesis of several proteins that manage to insert a fragment of single strand DNA called the T-DNA in the plant genome. This DNA fragment codifies for metabolites that the bacterium needs in order to survive parasitizing the plant. Biotechnology had taken advantage of the insertion system of the T-DNA and had modified the vector that contains this sequence, the T-plasmid, with the objective of using this bacterium to generate transgenic plants.
The system for transformation of plants with A. tumefaciens has the T-plasmid modified in various aspects: Because the T-plasmid is really long and difficult to manipulate, the T-DNA sequence and the genes that cause the crown gall tumor have been deleted. Instead of that, the DNA sequence that you want to insert in the plant is placed by using a “Binary Vector”, a small plasmid designed for the insertion of a sequence of interest flanked by the recognition sites for the cleavage of the T-DNA by the proteins encoded in the T-plasmid.
The procedure of infection in wild type bacteria is the following: as we can see in FIGURE X, once the bacteria enters the plant the phenolic compounds that the plant secretes induce the transcription and then the translation of the Vir regulon. In it, there are codified the proteins involved in the infection, such as VirB complex, which is in charge of the secretion into the vegetal cell or VirE2 that moves the T-DNA throughout the vegetal cytoplasm guiding it to the nucleus. In particular, the protein that we want to emphasize is the VirD2; it protects the T-strand in the 5’ and guides it out of the bacteria and into the vegetal cell.
Figure 2. Infection system done by A. tumefaciens
Agrobacterium tumefaciens enters the plant and then the vir regulon is induced. VirD2 protein protects and carries the T-DNA inside the nucleus of the plant for its posterior insertion.
Once inside the plant cell, the mature T-complex, which is the T-strand ligated to the VirD2 and VirE2 proteins, enters the nucleus. Inside it, VirD2 recognizes the proteins of the repairing machinery of double strand breaks in the DNA such as the Homologous Recombination (HR) or Non Homologous End Joining (NHEJ) system. Taking advantage of the wound in the genome, the T-DNA is inserted.
Figure 3. Insertion of the gen of interest in a random site of the plant’s genome
By taking advantage of the wound on the genome, the gen of interest is inserted.
The fact that VirD2 recognizes the repairing machinery implies that randomness plays an important role in the insertion of the T-strand. This generates multiple possible sites where the T-DNA could be inserted; because of this, the transformation generates different genotypes in the plants. This variable insertion is the reason why huge screening procedures are needed when you want to produce recombinant plants, wasting time and scientific consumptions.
The problem and the possible solution
Given the problem of random insertion, we decided to work on A. tumefaciens, in order to give it the capability of transform site specifically. We were going to direct the DNA using TAL Effector nucleases (TALENs). These proteins are site specific and can also generate a double strand break in the DNA; using this method it is possible to induce in some way the double strand repair machinery in the exact place that you want you insertion to be. To make sure that the T-strand is in the same place that the NHEJ machinery, we wanted to unite the TALENs with the VirD2 protein.
The idea of our system was that the TALEN protein entered the vegetal nucleus and interacted with the specific site of the plant’s genome, and then produced the double strand break as shown in the Figure X. At the same time, the TALENs are associated with the VirD2 protein, and this is united with the T-DNA strand, so it can be inserted in the break recently done by the TALEN. With this, we hoped that the T-strand insertion is site specific.
Figure 4. Proposed mechanism of insertion
The association of a TALEN with the virD2 protein should work for recognition inside the plants genome, the generation of a break, and the posterior site specific insertion of the gen of interest
For our biobricks, the idea was to modify the binary vector, so in it, we have had the TALEN’s sequence (ready to be modified at will) united with the VirD2 sequence.
Experimental design
Figure 5. General schema of the experimental design
First we needed to knock out the native VirD2 protein from the T-plasmid, to make sure that the only protein of that kind that enters the plant cell is associated with our TALEN system. To achieve this goal we tried to use homologous recombination. The design of this DNA segment had the sequences of VirD1 followed by a Chloramphenicol resistance cassette that would replace the VirdD2 and then VirD3 (Figure XI,). In the T-plasmid, VirD2 sequence is between of the VirD1 and VirD3 sequences, so the last ones are the homologous regions for the recombination. We chose an antibiotic resistance for later selection in medium.
The second and most important step was to design the correct TALENs. We used a transformed Nicotiana benthamiana plant that contains a specific GFP, mgfp5, in the genome. We designed the TALENs to recognize the mgfp5 sequence as a control in plant transformation. The recognition sequence was chosen using a software created by Felipe Erices, one of our team members, that finds the optimum sequence to cut the gene inside the plant’s genome. An important variable in this recognition is that the sequence is present hopefully once in the whole genome. We used the same software to optimize the TALEN recognition sequence on this DNA sequence.
The last step was to generate the optimized binary vector that would contain the corresponding TALEN sequence united with the VirD2 sequence to the Vir regulon promoter. This new binary vector would work altogether with the modified Agrobacterium tumefaciens that has the T-plasmid (VirD2 knock-out) for a specific DNA insertion for plant transformation.
Construct for homologous recombination
Figure 6. Construct generated by virD2 deletion
Chloramphenicol is replacing the native VirD2 protein, to avoid interference with our own recombinant VirD2.
This is an intermediate plasmid that we used to build the homologous recombination fragment.  In order to get a section with a longer region of homology we decided that instead of generating a linear PCR product by adjusting the primers to contain the region of homology, we would create a plasmid by using Gibson Assembly that would allow us to have 600bp of homology in each side of the antibiotic resistant cassette. This construct was done successfully. After the cloning, we tried to knockout VirD2 from Agrobacterium tumefaciens by transforming competent cells (freeze and thaw protocol) with both the linearized fragment from the RH plasmid and with the whole construct.
TALENs construction
For the assembly of the TALENs we purchased Addgene: Golden Gate TALEN Kit 2.0.  However, even after several attempts, we couldn’t construct the TALENs proteins so we stopped our work on this project.
Figure 7. Half of a TALEN.
We successfully made half of a TALEN
Notebook
In this section you will find the complete description of our everyday work in this project and detailed information of our results.  Click here .
References:
1. Magori, S., & Citovsky, V. (2011). Epigenetic control of Agrobacterium T-DNA integration. Biochimica et Biophysica Acta, 1809(8), 388-394. Elsevier B.V.
2. Citovsky, V., Kozlovsky, S. V., Lacroix, B., Zaltsman, A., Dafny-Yelin, M., Vyas, S., Tovkach, A., et al. (2007). Biological systems of the host cell involved in Agrobacterium infection. Cellular Microbiology, 9(1), 9-20.
3. Mahfouz, M. M., Li, L., Shamimuzzaman, M., Wibowo, A., Fang, X., & Zhu, J.-K. (2011). De novo-engineered transcription activator-like effector (TALE) hybrid nuclease with novel DNA binding specificity creates double-strand breaks.Proceedings of the National Academy of Sciences of the United States of America, 108(6), 2623-2628.
4. Zhang, F., Cong, L., Lodato, S., Kosuri, S., Church, G. M., & Arlotta, P. (2011). LETTErs Efficient construction of sequence-specific TAL effectors for modulating mammalian transcription. Nature Biotechnology, 29(2), 149-154. Nature Publishing Group, a division of Macmillan Publishers Limited. All Rights Reserved.
5. Cermak, T., Doyle, E. L., Christian, M., Wang, L., Zhang, Y., Schmidt, C., Baller, J. A., et al. (2011). Efficient design and assembly of custom TALEN and other TAL effector-based constructs for DNA targeting. Nucleic Acids Research,39(12), e82. Oxford University Press.
6. Pelczar, P., Kalck, V., Gomez, D., & Hohn, B. (2004). Agrobacterium proteins VirD2 and VirE2 mediate precise integration of synthetic T-DNA complexes in mammalian cells. EMBO Reports, 5(6), 632-637.
7. Gelvin, S. B. (2000). AGROBACTERIUM AND PLANT GENES INVOLVED IN T-DNA TRANSFER AND INTEGRATION. Annual Review of Plant Physiology and Plant Molecular Biology, 51(1), 223-256.
8. Grindley, N. D. F., Whiteson, K. L., & Rice, P. A. (2006). Mechanisms of site-specific recombination. Annual Review of Biochemistry, 75(1), 567-605. Annual Reviews.
9. Mysore, K. S., Bassuner, B., Deng, X. B., Darbinian, N. S., Motchoulski, A., Ream, W., & Gelvin, S. B. (1998). Role of the Agrobacterium tumefaciens VirD2 protein in T-DNA transfer and integration. Molecular plantmicrobe interactions MPMI, 11(7), 668-683. The American Phytopathological Society.
10. Sawitzke, J. A., Thomason, L. C., Bubunenko, M., Li, X., & Court, D. L. (2011). Recombineering : Using Drug Cassettes to Knock out Genes in vivo.redrecombineeringncifcrfgov, Chapter 1, 1-14.
11. Bochtler, M. (2012). Structural basis of the TAL effector-DNA interaction.Biological chemistry, 393(10), 1055-66.
12. Bogdanove, A. J., & Voytas, D. F. (2011). TAL Effectors: Customizable Proteins for DNA Targeting. Science, 333(6051), 1843-1846. American Association for the Advancement of Science.
13. Scholze, H., & Boch, J. (2011). TAL effectors are remote controls for gene activation. Current Opinion in Microbiology, 14(1), 47-53. Elsevier Ltd.
14. Satoh, K., Kikuchi, M., Ishaque, A. M., Ohba, H., Yamada, M., Tejima, K., Onodera, T., et al. (2012). The role of Deinococcus radiodurans RecFOR proteins in homologous recombination. DNA Repair, 11(4), 410-8. Elsevier B.V.
15. Durant, S., & Karran, P. (2003). Vanillins—a novel family of DNA-PK inhibitors.Nucleic Acids Research, 31(19), 5501-5512. Oxford University Press.
Retrieved from " http://2013.igem.org/Team:UC_Chile/Side_Project "
Recent changes
What links here
Related changes
Special pages
My preferences
Printable version
Permanent link
Privacy policy
Disclaimers
