<HTML xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en" dir="ltr">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki  ltr ns-0 ns-subject page-Team_UC_Chile_Purification"><DIV id="globalWrapper"><DIV id="top-section"><DIV id="p-logo"><A href="/Main_Page" title="Main Page">&quot;
	    </A></DIV><DIV id="menubar" class="left-menu noprint"><UL><LI class="selected"><A href="/Team:UC_Chile/Purification">Page               </A></LI><LI class="new"><A href="/wiki/index.php?title=Talk:Team:UC_Chile/Purification&amp;action=edit&amp;redlink=1">Discussion               </A></LI><LI><A href="/wiki/index.php?title=Team:UC_Chile/Purification&amp;action=edit">View source               </A></LI><LI><A href="/wiki/index.php?title=Team:UC_Chile/Purification&amp;action=history">History               </A></LI><LI style="color:white;cursor:default">teams</LI></UL></DIV><DIV class="right-menu noprint" id="menubar"><UL><LI id="pt-login"><A href="/wiki/index.php?title=Special:UserLogin&amp;returnto=Team:UC_Chile/Purification" title="You are encouraged to log in; however, it is not mandatory [o]" accesskey="o">Log in</A></LI></UL></DIV><DIV id="search-controls" class="noprint"><FORM action="/Special:Search" id="searchform"> </FORM></DIV></DIV><DIV id="content"><H1 class="firstHeading">Team:UC Chile/Purification</H1><DIV id="bodyContent"><H3 id="siteSub" class="noprint">From 2013.igem.org</H3><P><TITLE> Wiki-IGEM </TITLE></P><DIV id="hidden-links">HiddenLinks</DIV><DIV id="Header"><DIV id="MENU"><DIV class="new_menu"><DIV class="h1 hexa" onclick="aux(1)"><DIV class="text">Team</DIV><UL class="sub oculto"><LI class="pos1 sub_item oculto">Team Members</LI><LI class="pos2 sub_item oculto">PSB Lab</LI><LI class="pos3 sub_item oculto">Photo gallery</LI></UL></DIV><DIV class="h2 hexa" onclick="aux(2)"><DIV class="text">HumanPractices</DIV><UL class="sub oculto"><LI class="pos1 sub_item oculto">Home</LI><LI class="pos2 sub_item oculto">Synbio web</LI></UL></DIV><DIV class="h3 hexa" onclick="aux(3)"><DIV class="text">More</DIV><UL class="sub oculto"><LI class="pos1 sub_item oculto">Acknowledgments</LI><LI class="pos2 sub_item oculto">Biosafety</LI></UL></DIV><DIV class="h4 hexa sele" onclick="aux(4)"><DIV class="text">Project</DIV><UL class="sub "><LI class="pos1 sub_item ">Description</LI><LI class="pos2 sub_item ">Formation</LI><LI class="pos3 sub_item ">Targeting</LI><LI class="pos4 sub_item ">Purification</LI><LI class="pos5 sub_item ">Disruption</LI><LI class="pos5 sub_item ">In vitro Channelling</LI></UL></DIV><DIV class="h5 hexa" onclick="aux(5)"><DIV class="text">Methodology</DIV><UL class="sub oculto"><LI class="pos1 sub_item oculto">Lab Notebook</LI><LI class="pos2 sub_item oculto">Protocols</LI></UL></DIV><DIV class="h6 hexa" onclick="aux(6)"><DIV class="text">Results</DIV><UL class="sub oculto"><LI class="pos1 sub_item oculto">BioBricks</LI><LI class="pos2 sub_item oculto">Mathematical Model</LI><LI class="pos3 sub_item oculto">Wet Lab</LI></UL></DIV><DIV class="h7 hexa " onclick="location.href='https://2013.igem.org/Team:UC_Chile/Overview'"><DIV class="text">Home</DIV><UL class="sub oculto"><LI class="pos1 sub_item oculto">Overview</LI></UL></DIV></DIV></DIV></DIV><DIV id="Container"><DIV class="gray_box"><H1>Carboxysome Purification</H1><DIV class="normal-text">
An easy mechanism for the purification of reengineered microcompartments is essential for good results in our in vitro channeling system.   We have designed a novel methodology for the purification of assembled and functional Carboxysomes. This technique is based on traditional extraction methods like the sucrose gradient centrifugation and new procedures like biotinylation of proteins and isolation by interaction with streptavidin- coated magnetic beads. 
</DIV></DIV><H3>INTRODUCTION</H3><DIV class="normal-text">
Our project is based on the ability of Carboxysome to retain its metabolic properties in vitro. Also,  it has an easy purification. This feature can be personalized or standarized with an industrial production goal and it could give high yield of your protein complex If it is needed.
In order to achieve this aim, we thought about different approaches:</DIV><DIV class="normal-text"><H4>1) Sucrose gradient centrifugation</H4>
From literature, we know that Carboxysomes can be purified using traditional methods such as sucrose gradient centrifugation <A href="#Referencias">(1)</A>.  In this reported protocol, the cells are disrupted by sonication and after that they are centrifuged and resuspended with different buffers in several steps.  The carboxysome-enriched solution is loaded onto 10 to 50% (wt/vol) linear sucrose density gradients. Consequently, the prominent band near the middle can be isolated and should contain the carboxysome structures.
Employing this protocol, the proteins obtained are separated according to size and density, the specificity of this technique is not high enough and it is most likely submitted to human error.</DIV><DIV class="normal-text"><H4>2) Magnetic Carboxysome</H4>
During 2011, the Dundee iGEM team worked with Pdu, another published bacterial microcompartment.
In general, they tried to make Pdu magnetic for bioremediation processes. This strategy looked for an easy way of recover a modified microorganism. To achieve this, they attached bacterioferritin (Bfr) from Escherichia coli to Pdu40. Also, they include a tag for destination to the inside of Pdu. Bfr forms a large storage protein for iron and iron oxides. This should concentrate the particles inside the BMC and make it simple to separate with a magnet<A href="#Referencias">(2)</A>.
We thought about attaching a protein such as Bfr to one of the proteins of the shell in the Carboxysome in order for the microcompartment to be coated with metallic particles and thus, be able to be purified using its magnetic properties. However, proteins from Magnetosome associated systems or from the electron transport chain (ferritin, etc.) exceed the size and the complexity to be used for such a purpose.
Then, after some searching for alternative mechanisms with high yield of purification, we decided to mainly focus on different affinity chromatography strategy.</DIV><DIV class="normal-text"><H4>3) Biotinylation</H4>
In 2010, the Freiburg Bioware iGEM team designed Virus Construction Kit for Therapy. In their project, they tried two different approaches for the purification of the virus capsides. One methodology was based on HisTag affinity purification and the other one was based on biotinylation of capsid. According to their results the detection limit of the biotinylated viral capsids was about 10 fold more sensitive <A href="#Referencias">(4)</A>.
The principal advantages of this procedure is that with this mechanism you can isolate a protein complex in an assembled way and have the protein aggregate ready for further analysis; it also shows good yield results.
How does biotinylation works?

In nature, E. coli biotin holoenzyme synthetase, BirA catalyzes the transfer of biotin to the epsilon amino group of a specific lysine residue of the biotin carboxyl carrier protein (BCCP) subunit of acetyl-CoA carboxylase<A href="#Referencias">(4)</A>.

Schatz et al <A href="#Referencias">(5)</A>.  Identified the 15 amino acid peptide 5’ GLNDIFEAQKIEWHE 3’ as the most efficient sequence to produce biotinylation. Therefore, if it is added to any protein of interest and in the presence of the BirA gene, the protein of interest could be selectively tagged.

Using this approach, we learnt from our host lab PSBL about the INTACT (isolation of nuclei tagged in specific cell types) method for the purification of nuclei from plants <A href="#Referencias">(6)</A>. This technique purifies biotin-labeled nuclei using streptavidin-coated magnetic beads, without the need for specialized equipment. With this methodology, you generate a nuclear targeting fusion protein, which consists of three parts: a specific domain for nuclear envelope association in plants, a green fluorescent protein (GFP) for visualization and the biotin ligase recognition peptide. The expression of this construct and the BirA gene in plants allows purification of specific nuclei.
 
The INTACT method supposedly takes 2 days and should be adaptable to any organism that is amenable to transformation. So, we decided to combine both: 
<UL><LI>- The protocol for the sucrose gradient centrifugation (early steps): Disruption and centrifugation steps and the use of the same buffers in which the carboxysome is stable.</LI><LI>- The INTACT protocol (biotinylation and later steps): Fluorescence visualization and streptavidin- coated magnetic beads isolation process.</LI></UL>
 Using this method, we were able to create a new methodology for the extraction and purification of carboxysomes. 
 <DIV class="img_center_legend"><DIV class="legend">Our dreamed purification system</DIV></DIV></DIV><H3>Genetic Experimental Design</H3><DIV class="normal-text">
The general idea was to use: our construct the Carboxysome BioBrick (<A href="http://parts.igem.org/Part:Bba_K1113100">Bba_K1113100</A>) + RFP( <A href="http://parts.igem.org/Part:BBa_E1010">Bba_E1010</A>) and to add to the RFP the biotinylation acceptor peptide 5’ GLNDIFEAQKIEWHE 3’. Using this approach,  we could visualize the microcompartment using microscopy (to check the formation and correct induction) and allow carboxysomes to get affinity bound to streptavidin beads.
<DIV class="img_center_legend"><DIV class="legend">Construct for the biotinylation of the Carboxysome</DIV></DIV></DIV><H3>Protocols</H3><DIV class="normal-text">
For more information about the protocols used for the extraction and purification of the carboxysomes, click <A href="https://2013.igem.org/Team:UC_Chile/Protocols">here</A>.
</DIV><H3>Further Work</H3><DIV class="normal-text">
The characterization of the purification system needs to be tested in more detail:
<UL><LI>- The improvement on the efficiency between using traditional methodologies and our innovative system of purification.</LI><LI>- A yield quantification of the functional and assembled carboxysomes.</LI></UL>
We also need to verify if this system can be scalable for industrial production by checking costs and advantages of the production inside the carboxysomes.
</DIV><DIV class="Referencias"><B>References:</B><UL><LI><SPAN>1. So, A. K.-C., Espie, G. S., Williams, E. B., Shively, J. M., Heinhorst, S., &amp; Cannon, G. C. (2004). A novel evolutionary lineage of carbonic anhydrase (epsilon class) is a component of the Carboxysome shell. Journal of bacteriology, 186(3), 623–630.</SPAN></LI><LI><SPAN>2. The university of Dundee iGEM (2011). How our system works. Retrieved: September 2013, from</SPAN><A href="https://2011.igem.org/Team:Dundee/Results">https://2011.igem.org/Team:Dundee/Results</A></LI><LI><SPAN>3. Virus construction kit for therapy. (2010). Virus construction kit - The manual. Retrieved: September, 2013, from </SPAN><A href="https://2010.igem.org/Team:Freiburg_Bioware">https://2010.igem.org/Team:Freiburg_Bioware</A></LI><LI><SPAN>4. Cronan, J. E., Jr. (1990). Biotination of proteins in vivo. A post-translational modification to label, purify, and study proteins. The Journal of biological chemistry, 265(18), 10327–10333.</SPAN></LI><LI><SPAN>5. Beckett, D., Kovaleva, E., &amp; Schatz, P. J. (1999). A minimal peptide substrate in biotin holoenzyme synthetase-catalyzed biotinylation.Protein science: a publication of the Protein Society, 8(4), 921–929. doi:</SPAN><A href="10.1110/ps.8.4.921">10.1110/ps.8.4.921</A></LI><LI><SPAN>6. Deal, R. B., &amp; Henikoff, S. (2011). The INTACT method for cell type-specific gene expression and chromatin profiling in Arabidopsis thaliana. Nature protocols,6(1), 56–68. doi:</SPAN><A href="10.1038/nprot.2010.175">10.1038/nprot.2010.175</A></LI></UL></DIV></DIV><DIV class="printfooter">
Retrieved from &quot;<A href="http://2013.igem.org/Team:UC_Chile/Purification">http://2013.igem.org/Team:UC_Chile/Purification</A>&quot;</DIV></DIV></DIV><DIV id="footer-box" class="noprint"><DIV id="footer"><UL id="f-list"><LI id="t-recentchanges"><A href="/Special:RecentChanges" title="Recent changes">Recent changes</A></LI><LI id="t-whatlinkshere"><A href="/Special:WhatLinksHere/Team:UC_Chile/Purification" title="List of all wiki pages that link here [j]" accesskey="j">What links here</A></LI><LI id="t-recentchangeslinked"><A href="/Special:RecentChangesLinked/Team:UC_Chile/Purification" title="Recent changes in pages linked from this page [k]" accesskey="k">Related changes</A></LI><LI id="t-specialpages"><A href="/Special:SpecialPages" title="List of all special pages [q]" accesskey="q">Special pages</A></LI><LI><A href="/Special:Preferences">My preferences</A></LI></UL></DIV><DIV id="footer"><UL id="f-list"><LI id="t-print"><A href="/wiki/index.php?title=Team:UC_Chile/Purification&amp;printable=yes" title="Printable version of this page [p]" accesskey="p">Printable version</A></LI><LI id="t-permalink"><A href="/wiki/index.php?title=Team:UC_Chile/Purification&amp;oldid=318434" title="Permanent link to this revision of the page">Permanent link</A></LI><LI id="privacy"><A href="/2013.igem.org:Privacy_policy" title="2013.igem.org:Privacy policy">Privacy policy</A></LI><LI id="disclaimer"><A href="/2013.igem.org:General_disclaimer" title="2013.igem.org:General disclaimer">Disclaimers</A></LI></UL></DIV></DIV></DIV></BODY></HTML>