<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_Austin_LASA_Design skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><DIV id="top_title"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:Austin LASA/Design</SPAN></H1></DIV><DIV id="HQ_page"><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><TEXTAREA id="page">
const p = t =&gt; h('p', null, t);
h(g.Page, {title: 'Design, Development, and Results', prev: 'https://2018.igem.org/Team:Austin_LASA/Description', next: 'https://2018.igem.org/Team:Austin_LASA/Notebook', selector: [2, 1]},
  h(g.Section, {title: 'What will be used to detect HIV?'},
    p('Because our final Cas12a detection assay relies on sequence recognition between the crRNA spacer sequence and a target strand, we only needed to choose specific sequences from HIV’s genome to work with.'),
  ),
  h(g.Section, {title: 'Choosing a Gene'},
    p('In the end, we chose to work with one of HIV’s genes to simplify our work in the lab. Because HIV is a BSL3 organism, we were unable to work with HIV or any HIV-infected human cells in the lab per iGEM’s Biosafety Standards. Instead, we worked with synthetic viral DNA that we ordered as gblocks through IDT.'),
    p('We ultimately focused on Rev, a non-hazardous gene in HIV’s genome which codes for a transportation protein in HIV. The sequence we found for Rev was specifically from HIV1, and as a result, our subsequent work in the lab focused on the detection of HIV-1.'),
    p('Rev served as the viral sequence we would attempt to amplify with loop-mediated isothermal amplification (LAMP) and subsequently detect with our Cas12a assay.')
  ),
  h(g.Section, {title: 'Designing crRNA'},
    p('Once we decided on an HIV gene to detect, we went ahead and designed crRNA sequences for our Cas12a assay. crRNA sequences are the basis of our detection assay. However, Cas12a crRNA design is still a field of much study, and it was very difficult to find specific information on how to design crRNA. Ultimately, the design of our crRNA was broken up into two components: the hairpin sequence and spacer sequence.'),
    p('We decided on two hairpin sequences, one for AsCas12a and one for LbCas12a. Our LbCas12a sequence was taken from the Doudna Lab. Our AsCas12a sequence was taken from the Finkelstein Lab. We would test our AsCas12a crRNA with purified AsCas12a enzyme, and we would later attempt to clone a gene circuit expressing LbCas12a to use with our LbCas12a crRNA in our cellular reagents.'),
    p('We used several criteria when choosing our spacer sequence. The most important criteria for choosing our spacer sequence was that it needed to be around 24bp following Cas12a’s PAM site, TTTN. From there, we distinguished between spacer sequences based on the following:'),
    h('ul', null,
      h('li', null, 'GC content'),
      h('li', null, 'Dimer formation'),
      h('li', null, 'Similarities to Cas9 sgRNA design')
    ),
    p('Ultimately, we designed 7 different crRNA sequences, and ultimately used 4.'),
  ),
  h(g.Section, {title: 'Can we detect our gene? Testing out crRNA: Cas12a Assay with Purified Enzyme'},
    p('Following the design of our crRNA, we conducted a Cas12a assay with purified AsCas12a and several of our crRNAs.'),
    p('(See Experiments page for more information on carrying out our Cas12a assay.)')
  ),
  h(g.Section, {title: 'Can we amplify our gene isothermally to increase sensitivity? LAMP Primer and Amplicon Design'},
    p('Loop-mediated isothermal amplification (LAMP) is an isothermal amplification method that relies on specific primers to continuously create kinks in a LAMP amplicon that contain repeats of the template DNA.'),
    p('A LAMP reaction requires a primer set of four specific primers. Rather than only designing one primer set, we were advised to initially test a large number of primer sets. Our lab has found that LAMP primers will not amplify well, and it must be experimentally confirmed which LAMP primer sets work best. Additionally, during our primer design phase, we realized we would only be able to design primers for regions of a couple hundred base pairs, as opposed to the full 2.7kb of the Rev gene. As a result, using PrimerDesign, we designed six different LAMP primer sets that amplified regions of Rev that also contained our crRNA sequences.'),
    p('However, LAMP amplicons have a very special structure. The largest bulk of the resultant amplicons in a LAMP reaction have a continuous zig-zag shape with hairpin loops at the corners. Our lab has previously done work with oligonucleotide strand displacement with LAMP amplicons and has found they work best at hairpin loops. In our project, we wanted to see whether having the target sequence in the linear region or in the loop region would affect Cas12a’s ability to recognize the sequence.'),
    p('Therefore, when designing our LAMP primer sets and resultant amplicons, we designed primers such that, for each primer set, there would be a target sequence corresponding to one of our designed crRNAs in the linear region of the amplicon and in the loop region of the amplicon.')
  ),
  h(g.Section, {title: 'LAMP Reactions with Purified Enzyme'},
    p('We first decided to verify that our team could successfully carry out LAMP reactions of our Rev gblock with purified enzyme. We also wanted to verify which of our LAMP primer sets worked (or worked best) before we moved on to collecting more data for our kinetics analysis.'),
    p('The results from our initial LAMP tests showed that four of our primer sets were able to amplify regions of Rev relatively well, whereas two of our primer sets were unable to do so.'),
    h(g.Image, {src: 'https://static.igem.org/mediawiki/2018/b/bc/T--Austin_LASA--573_amp.svg', position: 'center'}),
    h(g.Image, {src: 'https://static.igem.org/mediawiki/2018/f/f5/T--Austin_LASA--153_amp.svg', position: 'center'}),
    h(g.Image, {src: 'https://static.igem.org/mediawiki/2018/e/ef/T--Austin_LASA--113_amp.svg', position: 'center'}),
    h(g.Image, {src: 'https://static.igem.org/mediawiki/2018/8/8a/T--Austin_LASA--11_amp.svg', position: 'center'}),
    h(g.Image, {src: 'https://static.igem.org/mediawiki/2018/e/ef/T--Austin_LASA--12_amp.svg', position: 'center'}),
    h(g.Image, {src: 'https://static.igem.org/mediawiki/2018/d/de/T--Austin_LASA--14_amp.svg', position: 'center'}),
    p('We can see from the above data that only primer sets 573, 11, 12, and 14 had significant amplification. We ultimately determined that primer set 14 was the most effective (See the Model page for data and methodology).'),
    p('We continued to test our LAMP primers in a cell-free system to gain more insight on the kinetics of our LAMP reactions. Using our data from LAMP reactions in a cell-free system, we wanted to use modeling to gain insight on:'),
    h('ol', {type: 'a'},
      h('li', null, 'How much DNA could be amplified in LAMP reactions with purified enzyme?', p('Being able to model how much DNA our LAMP reactions were able to amplify would allow us to later determine the sensitivity of our Cas12a detection assay.')),
      h('li', null, 'Being able to model how much DNA our LAMP reactions with purified enzyme were able to amplify would allow us to later compare between LAMP reactions with purified enzyme and cellular reagents.')
    ),
    p('Using the data we collected from reactions on our four LAMP primer sets, we were able to create a modeling system to gain more insight on LAMP.'),
    p('(See our Experiments page for more information on how to carry out a LAMP reaction. See our Modeling page for more information on our kinetic analysis of LAMP.)')
  ),
  h(g.Section, {title: 'Can we carry out our Cas12a assay with LAMP amplicons? Testing out LAMP Amplicons: Cas12a Assay with LAMP Amplicons'},
    p('After verifying that we could detect Rev with Cas12a and we could amplify regions of Rev isothermally with LAMP, we wanted to test the two together. This was for three reasons:'),
    h('ol', {type: 'a'},
      h('li', null, 'Could we verify the modeling we did with LAMP to determine the sensitivity of our Cas12a detection assay?', p('With LAMP data, we were able to set up a model to estimate how much DNA was amplified in our LAMP reactions. We then used these estimates to approximate how much of our LAMP reactions would be necessary in our Cas12a assay. We could verify all of our modeling with this step.')),
      h('li', null, 'Would the Cas12a reactions with LAMP amplicons would occur similarly to our initial Cas12a reactions with gblocks?', p('Because similar Cas12a reactions have not been carried out with LAMP amplicons, we do not know definitively that they will be the same. Therefore, we would need to test this to verify.')),
      h('li', null, 'If in the case the reactions did occur without any hiccups, would the placement of the target sequence (i.e. linear region versus loop region) affect the reaction?', p('As described earlier, our team was curious if the placement of the target sequence impacted the Cas12a reaction. If b. worked, we could try to see if there was an impact on the reaction as a result of target sequence placement.')),
    ),
    p('Unfortunately, we did not have the time to carry out this reaction in the lab. For continuation of our project, this step would need to be carried out.')
  ),
  h(g.Section, {title: 'Can we translate our system to cellular reagents?'},
    p('After we finally demonstrated that our Cas12a and LAMP reactions could be carried out with purified enzymes (which, as you can see from earlier, was unfortunately not the case), we wanted to translate our system to cellular reagents.'),
    p('We planned to do so by designing circuits to express Bst and LbCas12a enzymes in bacteria. This bacteria would be grown, induced to produce enzyme, after which the bacteria would be lyophilized. Unfortunately, we were unable to get to this step. However, we did design potential circuits to be used in cellular reagents.'),
    p('(See our Experiments page for more information on how to create cellular reagents).')
  ),
  h(g.Section, {title: 'LAMP Cellular Reagents'},
    p('Our lab has already developed LAMP cellular reagents. Therefore, we were advised by our supervisors to focus on developing Cas12a cellular reagents.')
  ),
  h(g.Section, {title: 'Cas12a Cellular Reagents'},
    p('We designed Cas12a gene circuits that expressed LbCas12a and a corresponding crRNA. We did not have time to clone these circuits, but we did include them in our parts collection.'),
    p('Each circuit contains a transcriptional unit which expressed LbCas12a. Adjacent and in the opposite direction is a transcriptional unit coding for a crRNA. Unfortunately, we did not have the time to clone these circuits and create cellular reagents.')
  ),
  h(g.Section, {title: 'In the end, what did we do?'},
    p('We were able to detect our viral DNA with four different specifically designed crRNA sequences with purified enzyme.'),
    p('We were able to amplify regions of our viral DNA (Rev) with four different LAMP primer sets with purified enzyme.'),
    p('We did not have the time to detect our LAMP amplicons with our Cas12a assay.'),
    p('We did not have the time to create cellular reagents with which to test our LAMP and Cas12a reactions.')
  ),
  h(g.Section, {title: 'What are the future directions of our kit?'},
    p('To continue our project, we would need to proceed by the steps laid out above.'),
    p('We would need to verify that we can detect our LAMP amplicons with our Cas12a assay with purified enzyme.'),
    p('Then, we would need to create cellular reagents to use in LAMP and Cas12a reactions and to further test these cellular reagents.'),
    p('Once we could successfully use cellular reagents in LAMP and Cas12a reactions, we would need to rigorously test our cellular reagents to verify that they fit diagnostic kit guidelines and could actually be used in resource-constrained settings like those in developing countries.')
  )
);
</TEXTAREA></DIV></DIV></DIV></DIV></DIV></BODY></HTML>