Team:Hamburg/Notebook
Home
Team
Collaborations
Project
Description Experiments Proof of Concept Demonstrate Notebook
Parts
Basic Part Composite Parts Part Collection
Attributions
Human Practices
Silver Gold Integrated Practices
Safety
Awards
10 mL SOC Medium
2 g Bacto-Tryptone
0.019 g KCl
0.5 g Bacto-Yeast extract
0.05 g NaCl
0.207 g MgCl2 · 6 H2O
0.036 g Glucose
were dissolved in 10 mL H2O and stored at 4°C in 1 mL aliquots.
100 mL TB PIPES Buffer
1.87 g KCl
0.17 g CaCl2
15.1 g PIPES
were dissolved in 100 mL H2O. NaOH was added to adjust the pH to 6.7.
1 L LB Medium
20.3 g Bacto-Tryptone
5 g Bacto-Yeast extract
10.1 g NaCl
were dissolved in 1 L H2O and autoclaved for 20 min at 120°C 15 pound-force per square inch (psi), to neutralize any bacterial contaminants.
15 LB-Agar / CAmp Plates
10 g Bacto-Tryptone
10 g NaCl
15 g Agar
5 g Bacto-Yeast extract
Were dissolved in 950 mL H2O and autoclaved for 20 min at 120°C 15 psi.
1 mL Chloramphenicol (25 mg/mL)
was added to the LB-Agar at 55°C and poured into 15 sterile petri dishes. The plates were stored at 4°C.
1 L 100 mM CaCl2 Buffer
11.1 g CaCl2
was dissolved in 1 L H2O and autoclaved for 20 min at 120°C 15 psi.
100 mL 85 mM CaCl2, 15% Glycerol Buffer
85 mL 100mM CaCl2
15 mL Glycerol
were mixed and autoclaved for 20 min at 120°C 15 psi.
1 L 100 mM MgCl2
20.33 g MgCl2 · 6 H2O
was dissolved in 1 L H2O and autoclaved for 20 min at 120°C 15 psi.
10 LB Agar / Kanamycin plates
20 g LB-Agar
Was dissolved in 500 mL H2O and autoclaved at 120°C 15 psi.
1 mL Kanamycin (25 mg/mL)
was added to the LB-Agar at 55°C and poured into 10 sterile petri dishes. The plates were stored at 4°C.
500 mL LB-Agar
25.0 g, 25.1 g, 24.99 g and 25.13 g LB Medium
were dissolved in 4x 1 L H2O and autoclaved at 120°C 15 psi.
Transformation of pcDNA3 and pBAD/Rluc
150 µL competent JW 3367-3 cells were split into 3x50 µL. 57 ng pcDNA3 and 80 ng pBAD/Rluc were added to 1 competent cell tube each and mixed. The cells were incubated on ice for 1.5 h, heat shock at 42°C for 1 min and chilled on ice for 3 min. 950 µL pre-warmed LB medium was added and the cells were incubated at 37°C and 900 rpm for 1h.
The cells were harvested by centrifugation at 3000g for 1 min, resuspended in 50 µL LB medium and 40 µL were plated onto LB Agar/Amp plates. The remaining 10 µL were plated onto different plates. All plates were incubated overnight at 37°C.
Transformation Efficiency
plasmid
number of colonies
pcDNA3
> 1000
pcDNA3 (1:5)
408
pBAD/Rluc
350
pBAD/Rluc
75
The negative controls did not show cell growth.
Preparation of 0.9 % NaCl Solution
1.8 g NaCl was solved in 200 mL water.
NaCl solution was mixed with a magnetic stirrer.
NaCl solution was filtrated over a 0.22 μm sterile filter into a glass flask.
Preparation of 2.5 % Alginate Solution
2.5 g Alginic Acid was slowly dissolved in 100 mL of the 0.9 % NaCl solution
The solution was mixed with a magnetic stirrer.
The solution was autoclaved for 20 min at 121 °C.
Preparation of a CaCl2 Solution (100 mM)
1.1098 g CaCl2 was solved in 100 mL water.
CaCl2 solution was filtrated over a 0.22 μm sterile filter into a glass flask.
Preparation of a CaCl2 Solution (100 mM)
1.1098 g CaCl2 was solved in 100 mL water.
CaCl2 solution was filtrated over a 0.22 μm sterile filter into a glass flask.
Transformation of JW 3367-3 (160427DW01) with fluorescence reporter systems: BBa_E0430, BBa_E0240, BBa_R0082
To 4 x 50 μL competent JW 3367-3 were added:
0.5 μL Part BBa_E0430: enhanced yellow fluorescent protein derived from A. victoria GFP (160607DW02)
0.5 μL Part BBa_E0240: GFP (160607DW02)
0.5 μL Part BBa_R0082: Promoter OmpR (160505DC02) as positive control
nothing as negative control
The cells were incubated on ice for 45 min, heat shocked at 42 °C for 1 min and put on ice for 5 minutes. 950 μL LB media was added to each tube. The cells were incubated at 37 °C, 900 RPM for 1.5 h and brought out on LB-Agar/CAmp plates.
The plates contained:
LOT#
plate
# of colonies
160607DW03
Part:BBa_E0430
19
160607DW04
Part:BBa_E0240
154
160607DW05
Par:BBa_R0082 (positive control)
36
160607DW06
negative control
0
Fabrication of Alginate Beads
4 mL of 2.5 % Alginic Acid solution was dyed with 2 droplets of red food color.
50 mL of CaCl2 solution was filtrated over a 0.22 μm sterile filter into a beaker.
Alginic Acid was pipetted into the CaCl2 solution.
Alginate Beads formed.
Different sized Alginate Beads were produced using 1000 μL, 200 μL and 10 μL Eppendorf pipettes.
Polymerized alginate beads were separated with a coffee filter.
Preparation of a CaCl2 Solution (100 mM)
4.44 g CaCl2 was solved in 400 mL water.
CaCl2 solution was filtrated over a 0.22 μm sterile filter into a glass flask
Fabrication of Alginate Beads
50 mL of CaCl2 solution was filtrated over a 0.22 μm sterile filter into a beaker.
Alginic acid was pipetted into the CaCl2 solution.
Alginate beads formed.
Different sized alginate beads were produced using 1000 μL, 200 μL and 10 μL Eppendorf pipettes.
Polymerized alginate beads were separated with a coffee filter.
The beads were stored in different places and the stability was assessed.
storage
stability
RT
very stable
fridge
very stable
freezer
not stable (without glycerol)
Images were captured with an optical muscope.
Beads were put in the muwave to test stability: aliginate capsules were stable.
Minipreparation of BBa_E0240 and BBA_E0430
The overnight cultures from the day before were centrifuged for 3 min at 5000 x g. The supernatant was discarded. For the further steps see the protocol from the GeneJet Plasmid Miniprep Kit from Thermo Scientific. #K0503. All centrifugation steps were performed at 13000 x g. In the end the columns were eluted with 20 μL elution buffer twice. The dsDNA concentration was measured at the NanoDrop.
LOT#
Part
DNA concentration
160615CL01
BBA_E0430
159.66 ng/μL
160615PW01
BBA_E0240
145.56 ng/μL
Restriction of BBa_E0240 (GFP), BBA_E0430 (eYFP) and BBa_R0082 (OmpR)
BBa_E0240
BBa_E0430
BBa_R0082
Volume
3.44 μL
3.13 μL
1.37 μL
XbaI
1 μL
1 μL
1 μL
SpeI
1 μL
1 μL
1 μL
10x NEB 2.1 Buffer
5 μL
5 μL
5 μL
water
39.76 μL
39.87 μL
42.63 μL
LOT%
160615CL02
160615CL03
160615PW02
The restriction preparations were incubated at 37 °C for 10 min and afterwards heat-inactivated at 80 °C for 20 min.
Ligation of Restricted BBa_E0240 (GFP), BBA_E0430 (eYFP) with BBa_R0082 (OmpR)
BBa_E0240 with BBa_R0082
BBa_E0430 with BBa_R0082
Volume Reporters
2 μL BBa_E0240 restricted
2 μL BBa_E0430 restricted
Volume OmpR
2 μL
2 μL
10x T4 DNA Ligase Buffer
2 μL
2 μL
T4 DNA Ligase
1 μL
1 μL
water
13 μL
13 μL
LOT%
160615PW04
160615PW03
The ligation preparations were incubated at RT for 10 min and afterwards heat-inactivated at 80 °C for 20 min.
Transformation of JW 3367-3 with Ligation Preparations
14 μL BBa_E0240 ligated with OmpR was added to 150 μL competent JW 3367-3 cells and mixed. 14 μL BBa_E0430 ligated with OmpR was added to 150 μL competent JW 3367-3 cells and mixed. The cells were incubated on ice for 1.5 h, heat shocked at 42 °C for 1 min and chilled on ice for 3 min. 950 μL pre-warmed LB medium was added to each tube and the cells were incubated at 37 °C and 900 rpm for 1 h. The cells were brought onto LB Agar plates with Chloramphenicol (CAmp) as a selection antibiotic and stored at 37 °C overnight.
160615PW05: LB-Agar CAmp plates with JW 3357-3 transformed with BBa_R0082/BBa_E0240
160615CL04: LB-Agar CAmp plates with JW 3357-3 transformed with BBa_R0082/BBa_E0430
Fabrication of LB-Agar-Plates with Chloramphenicol
50 g  of LB-Agar (Lucia/Miller) was solved in 1000 mL water.
Autoclaved for 1.5 h at 121 °C.
Added 250 μL of chloramphenicol.
LB-Agar solution was prepared under sterile conditions in petri dishes and after the cooling process was it  stored at 4 °C in the fridge
Transformation of DH5α and BL21 Cells
Competent cells were took from the -80 °C freezer onto ice for 30 min (JW 3367-3 / 160427DW01).
2.5 μL of plasmid pET_Duet_1 with a Hat t7 promoter and a yellow fluorescent protein (YFP) insert was prepared on ice for 1.5 h.
Heat-shock at 42 °C for 45 s.
Transformed cells chilled on ice for 3 min.
850 μL LB-Medium was warmed up to 37 °C.
After the 3 min the LB-Medium was added to the transformed cells.
Incubation of the cells at 37 °C, 900 rpm for 1 h.
Centrifugation at 4500 rpm for 2 min, afterwards 950 μL were removed.
The pelleted cells resuspended with the rest (~50 μL) of the LB-Medium.
50 μL of the cell solution on a LB-Agar-Plate with Ampicillin was plated out.
Plates was stored overnight at 37 °C.
160615ST01 => BL21 E. coli Strain
160615ST02 => BL21 E. coli Strain
160615ST03 => DH5α E. coli Strain
Preparation of Bacteria for Alginate Encapsulation
Cultures (160615ST01, 160615ST02, 160615ST03) were picked from the incubator.
1 mL LB-Medium with bacteria was pipetted in a cuvette and OD600 was measured versus a blank measurement of LB-Medium.
OD600(DH5α)=2.422
OD600(BL21)=1.224
Cells were pipetted from the liquid culture to 1 mL into 1.5 mL Eppendorf tubes.
Isolation of Plasmid pSB1C3-LacZα (160616DC01)
Isolation using the Thermo Scientific Purification Kit.
Measurement of plasmid concentrations.
160620CSW01: 45,62 ng/μl
160620CSW02: 63,97 ng/μl
Restriction of OmpR-ColA and BBa_I732018 (LacZα)
Restriction of OmpR-ColA with SpeI and PstI.
Restriction of BBa-I732018 with PstI and XbaI.
BBA_I732018
OmpR
Volume Plasmid DNA
7.8 μL
2.24 μL
XbaI
1 μL
1 μL
PstI
1 μL
1 μL
10x NEB 2.1 Buffer
5 μL
5 μL
water
35.2 μL
40.5 μL
The restriction preparations were incubated at 37 °C for 10 min and afterwards heat-inactivated at 80 °C for 20 min.
Purification of Restriction Reactions
Preparation of 1.75% agarose gel. 10 μL restriction reaction with 2 μL loading dye were added.
Lane 1
Lane 2
Lane 3
2 Log DNA Ladder
BBa_I732018 (insert)
OmpR (vector)
10 μL
12 μL
12 μL
Restriction of BBa_E0240 (GFP) and BBa_R0082 (OmpR)
BBa_E0240
BBa_R0082 Colony B
500 ng Plasmid
3.4 μL
3.2 μL
XbaI
1 μL
-
SpeI
1 μL
1 μL
10X Cutsmart Buffer
5 μL
2 μL
water
19.6 μL
13.8 μL
The restriction preparations were incubated at 37 °C for 30 min and afterwards heat-inactivated at 80 °C for 20 min.
Restriction of BBa_R0082 (OmpR) with BcuI
In the end the plasmid with OmpR should be restricted with SpeI (or BcuI from Thermo Fisher) and PstI. Because the buffer from NEB and Thermo Fisher are not compatible we decided to first restrict the plasmid OmpR with BcuI and afterwards purify it to loose the buffer and enzymes. Then we would perform a second restriction.
There were three preparations of the OmpR restriction.
Plasmid
3 x 3.2 μL
BcuI (SpeI)
3 x 1 μL
10X FastDigest Buffer
3 x 2 μL
Water
3 x 13.8 μL
The restriction preparations were purified with the GeneJet PCR Purification Kit von Thermo Scientific. Centrifugations were carried out at 13000 x g and in the end just 20 μL elution buffer was added to the column. Afterwards the concentration were measured at the NanoDrop.
Preparation
DNA concentration
I
85.44 ng/ μL
II
21.57 ng/ μL
III
23.97 ng/ μL
Restriction of BBa_I732018 (LacZ-160620CSW01) / BBa_E0430 (eYFP-160615CL01) / BBa_E0240 (GFP-160615PW01) / BBa_R0082 (OmpR-160511DC02)
LacZ
eYFP
GFP
OmpR Prep. I-III
OmpR Control 1
OmpR Control 2
OmpR Control 3
OmpR Control 4
OmpR Control 5
Plasmid
10.9 μL
3.1 μL
3.4 μL
3 x 20 μL
3.2 μL
3.2 μL
3.2 μL
3.2 μL
3.2 μL
XbaI
1 μL
1 μL
1 μL
-
-
-
-
1 μL
-
PstI
1 μL
1 μL
1 μL
3 x 1 μL
-
-
-
-
1 μL
10X NEB 3.1
2.5 μL
2.5 μL
2.5 μL
3 x 2.5 μL
-
2.5 μL
2.5 μL
2.5 μL
2.5 μL
10X NEB 2.1
-
-
-
-
2.5 μL
-
-
-
-
water
9.6 μL
17.4 μL
17.5 μL
3 x 1.5 μL
18.3 μL
18.3 μL
19.3 μL
18.3 μL
18.3 μL
BcuI
-
-
-
-
1 μL
1 μL
-
-
-
The restriction preparations were incubated at 37 °C for 30 min and afterwards heat-inactivated at 80 °C for 20 min.
Control of the restriction of BBa_I732018, BBa_E0430, BBa_E0240, BBa_R0082 and controls
A 1.7 % agarose gel was prepared with 10 μL RedSafe and loaded using the following setup:
2-Log DNA Ladder (NEB)
BBa_I732018
BBa_E0430
BBa_E0240
BBa_R0082 1
BBa_R0082 2
BBa_R0082 3
BBa_R0082 unrestricted
BBa_R0082 Control XbaI
BBa_R0082 Control PstI
BBa_R0082 Control BcuI in 3.1 NEB
BBa_R0082 Control BcuI in 2.1 NEB
Gel control of the restriction of BBa_I732018, BBa_E0430, BBa_E0240, BBa_R0082 and controls
The restriction was performed to receive the BBa_I732018, BBa_E0430 and BBa_E0240 genes as well as restricting the BBa_R0082 Plasmid to ligate the Plasmid with  the restricted genes of BBa_I732018, BBa_E0430 and BBa_E0240.  Therefore the lanes were cut out of the gel and afterwards purified with the GeneJet Gel Extraction Kit from Thermo Fisher Scientific. The centrifugation was performed at 13000 g for 1 min.
Afterwards the preparations were measured at the NanoDrop.
Concentration
BBa_I732018
22.24 ng/μL
BBa_E0430
22.40 ng/μL
BBa_E0240
23.57 ng/μL
BBa_R0082 1
25.42 ng/μL
BBa_R0082 2
32.28 ng/μL
BBa_R0082 3
24.33 ng/μL
Restriction of BBa_I732018 / BBa_E0430 / BBa_E0240 / BBa_R0082
BBa_I732018
BBa_E0430
BBa_E0240
Control BBa_R0082
Plasmid
10.9 μL
3.1 μL
3.4 μL
1.4 μL
XbaI
1 μL
1 μL
1 μL
-
PstI
1 μL
1 μL
1 μL
-
BcuI
-
-
-
1 μL
10x NEB 3.1
2.5 μL
2.5 μL
2.5 μL
2.5 μL
H2O
9.6 μL
17.4 μL
17.1 μL
19.1 μL
Incubation for 1 h at 37 °C. Heat-inactivation for 20 min at 80 °C.
Control of the restriction of BBa_I732018, BBa_E0430, BBa_E0240, BBa_R0082 and controls
A 2.2 % agarose gel with 10 μL RedSafe was prepared and loaded using the following setup:
2-Log DNA Ladder (NEB)
BBa_I732018
BBa_E0430
BBa_E0240
Control BBa_R0082 with BcuI
Control of the restriction of BBa_I732018, BBa_E0430, BBa_E0240, BBa_R0082 and controls
The lanes were cut out of the gel and afterwards purified with the GeneJet Gel Extraction Kit from Thermo Fisher Scientific. The centrifugation was performed at 13000 g for 1 min.
Afterwards the preparations were measured at the NanoDrop.
BBa_I732018
c=100 ng/μL
BBa_E0430
c=160 ng/μL
BBa_E0240
c=160 ng/μL
Control with gel
c=110 ng/μL
Ligation of restricted BBa_I732018, BBa_E0430, BBa_E0240-genes with BBa_R0082
Ligation mixtures consisted of the following components:
Amount [μ]
BBa_E0430
BBa_E0240
BBa_I732018
V (Insert)
3.57 μL
3.39 μL
1.20 μL
V (Vektor)
2 μL
2 μL
2 μL
V (T4 Ligase)
1 μL
1 μL
1 μL
V (H20)
11.43 μL
11.61 μL
13.80 μL
The preparations were incubated at 16 °C overnight.
Transformation of JW3367-3 with the ligated plasmids
Competent cells were split into aliquots and incubated on ice with the following DNA samples:
Aliquot amount
DNA
DNA amount [μL]
150 μL
BBa_R0082/BBa_I732018 ligation
10
150 μL
BBa_R0082/BBa_E0430 ligation
10
150 μL
BBa_R0082/BBa_E0240 ligation
10
50 μL
pcDNA
0.5
50 μL
pSB1C3/BBa_R0082
1
50 μL
pSB1C3/BBa_E0430
1
50 μL
pSB1C3/BBa_E0240
1
50 μL
pSB1C3/BBa_I732018
1
The cells were incubated on ice for 1 h, heat-shocked at 42 °C for 1 min and put back on ice for another 5 min. Ad 1 mL pre-warmed LB medium was added and the cells were incubated at 37 °C / 900 rpm for 1 h on the ThermoMix. The cells were brought onto LB/Agar plates according to the resistances coded in the transformed plasmids in following order:
Plasmid
Amount [μL]
pcDNA3
150
pSB1C3/BBa_R0082/BBa_E0240
100
50
pSB1C3/BBa_R0082/BBa_E0430
100
50
pSB1C3/BBa_R0082/BBa_I732018
100
50
pSB1C3/BBa_E0240
100
50
pSB1C3/BBa_E0430
100
50
pSB1C3/BBa_I732018
100
50
pSB1C3/BBa_R0082
100
50
negativ control
100
50
Colony PCR with pSB1C3/BBa_R0082/BBa_I732018, pSB1C3/BBa_R0082/BBa_E0430 and pSB1C3/BBa_R0082/BBa_E0240
(All of the transformed cells (except for the negative control preparations) formed single colonies on the according plates. There was not one cell colony on the negative control plates.)
The mixture of the Colony PCR tubes was done as follows:
BBa_E0240
BBa_E0430
BBa_I732018
V (DreamTaq-Buffer)
1.5 μL
1.5 μL
1.5 μL
V (dNTPs)
0.5 μL
0.5 μL
0.5 μL
V (H20)
5.9 μL
5.9 μL
5.9 μL
V (VF2)
1 μL
1 μL
1 μL
V (VR)
1 μL
1 μL
1 μL
Taq
0.1 μL
0.1 μL
0.1 μL
Assignment of the plates with the colonies that were picked for the PCR and the PCR tubes was as described in the following table:
Tubes
Plate
1, 2, 3, 4
pSB1C3/BBa_R0082/BBa_E0240
5, 6, 7, 8
pSB1C3/BBa_R0082/BBa_E0430
9, 10, 11, 12
pSB1C3/BBa_R0082/BBa_I732018
Glycerol Stock of pSB1C3/BBa_R0082/BBa_E0430 and pSB1C3/BBa_R0082/BBa_E0240
Made a Glycerol Stock of the overnight cultures from the day before. Therefore mixed 250 μL Glycerol with 250 μL water and 500 μL culture. The stocks were stored at -80 °C.
Stock of BBa_R0082/BBa_E0240
Stock of BBa_R0082/BBa_E0430 A
Stock of BBa_R0082/BBa_E0430 B
Stock of BBa_R0082/BBa_E0430 C
Stock of BBa_R0082/BBa_E0430 D
Minipreparation of pSB1C3/BBa_R0082/BBa_E0430 and pSB1C3/BBa_R0082/BBa_E0240
The over night cultures from the day before were centrifuged for 3 min at 5000 g. The supernatant was discarded. For the further steps see the protocol from the GeneJet Plasmid Miniprep Kit from Thermo Scientific. #K0503. All centrifugation steps were performed at 13000 g. In the end the columns were eluted with 20 μL Elutionbuffer twice.
Miniprep of BBa_R0082/BBa_E0240
Miniprep of BBa_R0082/BBa_E0430 A
Miniprep of BBa_R0082/BBa_E0430 B
Miniprep of BBa_R0082/BBa_E0430 C
Miniprep of BBa_R0082/BBa_E0430 D
The dsDNA concentration was measured at the NanoDrop.
BBa_R0082/BBa_E0240
188.46 μL
BBa_R0082/BBa_E0430 A
219.83 μL
BBa_R0082/BBa_E0430 B
321.86 μL
BBa_R0082/BBa_E0430 C
223.68 μL
BBa_R0082/BBa_E0430 D
339.88 μL
Colony PCR with pSB1C3/BBa_R0082/BBa_I732018
The PCR was performed with 4 colonies. The mixture of the Colony PCR tubes was done as follows:
The PCR Program TAQSCREEN was used.
BBa_I732018
V (DreamTaq Buffer)
1.0 μL
V (dNTPs)
0.5 μL
V (H2O)
6.4 μL
V (VF2)
1.0 μL
V (VR)
1.0 μL
Taq
0.1 μL
Tubes
Plates
1, 2
pSB1C3/BBa_R0082/BBa_I732018 (colonies 1 and 2)
3, 4
pSB1C3/BBa_R0082/BBa_I732018 (colonies 3 and 4)
Gel electrophoresis in 2.2 % agarose gel and 1:10000 RedSafe with the samples from the colony PCR
Potential difference: 100 V
Duration: ˜45 min
Control of the Colony PCR of pSB1C3/BBa_R0082/BBa_I732018
Sequencing ofpSB1C3/BBa_R0082/BBa_E0430 and pSB1C3/BBa_R0082/BBa_E0240
2.4 μL of pSB1C3/BBa_R0082/BBa_E0240 once with 5 μL VF2 and once with 5 μL VR
2.2 μL of pSB1C3/BBa_R0082/BBa_E0430 1 once with 5 μL VF2 and once with 5 μL VR
1.6 μL of pSB1C3/BBa_R0082/BBa_E0430 2 once with 5 μL VF2 and once with 5 μL VR
2.2 μL of pSB1C3/BBa_R0082/BBa_E0430 3 once with 5 μL VF2 and once with 5 μL VR
1.5 μL of pSB1C3/BBa_R0082/BBa_E0430 4 once with 5 μL VF2 and once with 5 μL VR
Preparation of overnight cultures
Plates from the 5.07.2016 were picked and put into overnight culture
pcDNA
pSB1C3/BBa_E0240
pSB1C3/BBa_E0430
pSB1C3/BBa_I732018
pSB1C3/BBa_R0082
LB-Agar Plates
Each 35 CAmp and Amp plates were prepared, with 100 μL Ampicillin and 33 μg/L Chloramphenicol, using Roth: LB-Broth chemical mix.
Plates were stored in 4 °C freezer until needed.
1 L LB-Medium
Was prepared as well, using Roth: LB-medium chemical mix and stored in 4 °C freezer.
Miniprep of overnight cultures
Liquid culture from JW 3367-3
Colony of pSB1C3/BBa_E0240 GFP plate
Colony from pSB1C3/BBa_E0430 YFP plate
Colony from pSB1C3/BBa_I732018 LacZΑ plate
Colony from pSB1C3/BBa_I732018 LacZΑ plate
Colony from pSB1C3/BBa_R0082
Colony from pBAD/Rluc plate
Colony from pcDNA3 plate
Miniprep was performed according to the GeneJET Miniprep Kit Protocol by ThermoScientific. Storage of plasmid eluates at -20°C.
Restriction of EnvZ-LRR, pSB1C3, BBa_I732018 and pSB1C3/BBa_R0082
First restriction
EnvZ-LRR 3'
EnvZ-LRR 5'
pSB1C3
pSB1C3/BBa_R0082
pSB1C3/BBa_I732018
Plasmid/DNA
10  μL
10  μL
2  μL
20  μL
11  μL
PvuI
1  μL
1  μL
-
-
-
BcuI
1  μL
-
1  μL
-
-
XbaI
-
-
-
1  μL
1  μL
Buffer
2  μL FastDigest
2  μL FastDigest
2  μL FastDigest
2  μL NEB2.1 Buffer
2  μL NEB2.1 Buffer
water
16  μL
17  μL
15  μL
6  μL
6  μL
Total
30  μL
30  μL
20  μL
30  μL
21  μL
Purification of restricted DNA EnvZ-LRR 5’ and pSB1C3
Protocol see manufacturer's instructions of Thermo Scientific Purification Kit
Restriction of EnvZ-LRR 5’ and pSB1C3 with EcoRI
Second restriction of the two samples was needed because EcoRI needs a different buffer than PvuI and BcuI.
EnvZ-LRR 5'
pSB1C3
Plasmid/DNA
50  μL
50  μL
EcoRI
2  μL
2  μL
NEB2.1 Buffer
6  μL
6  μL
water
2  μL
2  μL
Total:
60  μL
60  μL
Ligation of pSB1C3 with EnvZ-LRR3’ and EnvZ-LRR5’
1:10 (Vector:Insert) Ligation
Amount [ μL]
EnvZ-LRR3’ (Insert)
5.7
EnvZ-LRR5’ (Insert)
5.7
pSB1C3 (Vector)
5.7
T4 Ligase
1
DNA Ligase Buffer
2
1:5 (Vector:Insert) Ligation
Amount [ μL]
EnvZ-LRR3’ (Insert)
4.25
EnvZ-LRR5’ (Insert)
4.25
pSB1C3 (Vector)
8.5
T4 Ligase
1
DNA Ligase Buffer
2
All samples were left for incubation at 12° C overnight.
Ligation of pSB1C3/BBa_R0082 with pSB1C3/BBa_I732018
1:10 (Vector:Insert) Ligation
Amount [ μL]
pSB1C3/BBa_I732018
8.5
pSB1C3/BBa_R0082
8.5
T4 Ligase
1
DNA Ligase Buffer
2
All samples were left for incubation at 12° C overnight.
Transformation of competent JW-33673 cells with pSB1C3-EnvZ-LRR 1:10 and pSB1C3-EnvZ-LRR 1:5
Amount JW-33673 cells
Plasmid
Amount plasmid
150  μL
pSB1C3-EnvZ-LRR 1:10
10  μL
150  μL
pSB1C3-EnvZ-LRR 1:5
10  μL
75  μL
pSB1C3 1:4 (positive control)
1  μL
75  μL
- (negative control)
-
All plates were put in incubator at 37° C overnight.
Colony PCR of pSB1C3/EnvZ-LRR 3’
The ligation preparations of EnvZ-LRR and pSB1C3 were amplified by colony PCR after preparing the following master mix:
1x
7x
Dreamtaq-Buffer
1  μL
7  μL
dNTPs (10 mM)
0.5  μL
3.5  μL
water
6.4  μL
44.8  μL
VF2 primer (10 mM)
1  μL
7  μL
VR primer (10 mM)
1  μL
7  μL
DreamTaq-Polymerase
0.1  μL
0.7  μL
4 colonies from the 1:10 and 2 colonies from the 1:5 ligation were picked and LB cultures were prepared in parallel.
The following PCR setup was used:
1.
98° C
2:00
2.
98° C
0:10
3.
55° C
0:45
4.
72° C
3:15
5. GOTO 2 repeat 34x
6. HOLD at 4° C
Restriction of BBa_R0082 and BBa_I732018
Because of buffer difficulties, Omp had to be restricked sequentielly with two restriction enzyms BcuI (ThermoF.) and PstI (NEB). After the first restriction, the restricted vector was purified via the PCR Purification Kit, following the respective protocol . Pippeting of the samples was done, as described in the following table:
1. Restriction of BBa_R0082
Intredient
Volume
1:10 diluted pSB1C3/BBa_R0082 (363.67 ng/μL)
1.3 μL
BcuI
1.0 μL
Fast Digest Buffer (T.F.)
5.0 μL
H2O
42.8 μL
2. Restriction of BBa_I732018
Intredient
Volume
BBa_I732018 (33.89 ng/μL)
8.0 μL
XbaI
1.0 μL
PstI
1.0 μL
NEBuffer 3.1
5.0 μL
H2O
35.0 μL
3. Second Restriction of BBa_R0082
Ingredient
Volume
BBa_R0082 (Restricded with BcuI)
10 μL
PstI
1.0 μL
NEBuffer 3.1
5.0 μL
H2O
34.0 μL
Minipreparation of the Over-Night-Cultures from pSB1C3-EnvZ-LRR
From each sample a cryostock was prepared, following the respective protocol .
The rest of the samples were miniprapareted, following the respective protocol .
Gel control of the mini-prep, the restriction products BBa_R0082 and BBa_I732018
200 mL gel with 2.0 μL redsafe at 150 V, t:0.75 h
Ladder
Mini-prep product 1
Mini-prep product 2
Mini-prep product 3
Mini-prep product 4
Mini-prep product 5
Mini-prep product 6
Mini-prep product 7
Restrection product BBa_R0082
Restricton product BBa_I732018
Control of the mini-prep, the restriction products BBa_R0082 and BBa_I732018
Restriction of BBa_I732018 and pSB1A3 and sequential restriction of BBa_R0082 with BcuI and EcoRI
The BBa_I732018 and BBa_R0082 genes should be inserted into pSB1A3. Therefore BBa_I732018 and pSB1A3 were restricted like shown in the table below:
BBa_I732018
BBa_R0082
Plasmid DNA
2.0 μL (700 ng)
4.0 μL (100 ng)
XbaI
1.0 μL
0.5 μL
PstI
1.0 μL
0.5 μL
10x NEB 2.1 Buffer
2.0 μL
2.0 μL
H2O
14.0 μL
13.0 μL
The Plasmid with BBa_R0082 should be restricted with SpeI (or BcuI from Thermo Fisher) and EcoRI. Because the buffer from NEB and Thermo Fisher are not compatible. The BBa_R0082 plasmid is first restricted with BcuI and afterwards purify it to loose the Buffer and Enzyme. Only then the second restriction.
First restriction:
BBa_R0082
Plasmid DNA
2.0 μL (720 ng)
BcuI
1.0 μL
10x Fast Digest Buffer
2.0 μL
H2O
15 μL
There were two preparations of the BBa_R0082 restriction.
The restriction products were purified with the GeneJet PCR Purification Kit, following the respective protocol . Centrifugations were carried out at 12000 g.  For preperation I 50 μL of Elutions buffer was added to the column. For Preperation II 15 μL of Elutions buffer was added to the column. Afterwards the concentration was measured at the NanoDrop.
Preparation I
8.34 ng/μL
Preparation II
23.61 ng/μL
Second Restriction
Preparation I
Preparatin II
Plasmid DNA
29 μL
29 μL
EcoRI
1 μL
1 μL
10x NEBuffer 2.1
2.0 μL
2.0 μL
H2O
18 μL
32 μL
Heat-inactivation: 80 °C, 20 min
Sequential restriction of BBa_R0082 with BcuI and PstI
The Plasmid with BBa_R0082 should be restricted with SpeI (or BcuI from Thermo Fisher) and PstI. Because the buffer from NEB and Thermo Fisher are not compatible we decided to first restrict the Plasmid BBa_R0082 with BcuI and afterwards purify it to loose the Buffer and Enzyme. Then we would perform a second restriction.
First restriction:
BBa_R0082
Plasmid DNA
4.0 μL
BcuI
1.0 μL
10x Fast Digest Buffer
2.0 μL
H2O
13 μL
The restriction product was purified with the GeneJet PCR Purification Kit von Thermo Scientific, following the respective protocol . Centrifugations were carried out at 12000 g.  Washing was performed with using only 50 μL Wash buffer and for elution only 50 μL of Elutions buffer was added to the column. The concentration was measured at the NanoDrop.
Purified extract (BBa_R0082) are c=15.62 ng/μL
Second Restriction:
Purified extract (BBa_R0082)
Plasmid DNA
40.0 μL
PstI
1.0 μL
10x NEBuffer 2.1
5.0 μL
H2O
4.0 μL
Heat-inactivation: 80 °C, 20 min
Ligation of restricted BBa_I732018-gene with BBa_R0082
Ligation mixtures consisted of the following components:
Preparation I
Preparation II
Praparation III
BBa_I732018
2.74 μL
(Insert:Vector=3:1)
3.706 μL
(Insert:Vector=5:1)
5.78 μL
(Insert:Vector=7:1)
BBa_R0082
2.0 μL (31.24 ng)
2.0 μL (31.24 ng)
2.0 μL (31.24 ng)
T4 Ligase
1.0 μL (31.24 ng)
1.0 μL (31.24 ng)
1.0 μL (31.24 ng)
H2O
12.26 μL
11.29 μL (31.24 ng)
9.22 μL (31.24 ng)
The preparations were incubated at 16 °C overnight.
Transformation of the Ligation preparations of pSB1C3/BBa_R0082/BBa_I732018 and the Parts BBa_K081005, BBa_K608006 and BBa_K608007
The parts BBa_K608006 (Plate 1, 5E), BBa_K081005 (Plate 3, 16E) and BBa_K608007 (Plate 1, EG) were resuspended in 10 μL dest. H2O (DNAse free).
10 μL of the Ligation Preparations pSB1C3/BBa_R0082/BBa_I732018 with Insert:Vector of 3:1, 5:1 and 7:1  were added to 150 μL competent cells and mixed. 150 μL competent JW3367-3 cells were split into 3x 50 μL. 1 μL of the resuspended Parts were added to 1 competent cell tube each and mixed. The cells were incubated on ice for 1 h, heat shock at 42 °C for 1 min and chilled on ice for 3 min. 850 μL pre-warmed LB medium was added and the cells were incubated at 37 °C 900 rpm for 1 h.
The cells were harvested by centrifugation at 4500 rpm for 1 min, resuspended in 50 μL LB medium and 50 μL were plated onto LB Agar/Camp plates. All plates were incubated over night at 37 °C.
Transformation following the respective protocol
Transformation of DH5a and JW 3367-3 with pSB1C3/EnvZ-NOD1 and pSB1C3/BBa_R0082/BBa_I732018
Competent DH5a and JW 3367-3 were thawed at 4° C, split and inoculated with DNA as following:
Volume of cells
Cell line
Transfected with
Amount of DNA
50 µL
DH5α
pSB1C3/BBa_R0082/BBa_I732018
50 ng
50 µL
JW 3367-3
pSB1C3/BBa_R0082/BBa_I732018
50 ng
50 µL
JW 3367-3
pSB1C3/BBa_R0082/BBa_I732018
50 ng
150 µL
JW 3367-3
pSB1C3/EnvZ-NOD1 1:10
10 µL
150 µL
JW 3367-3
pSB1C3/EnvZ-NOD1 1:5
10 µL
150 µL
JW 3367-3
pSB1C3/EnvZ-NOD1 1:5
2 µL
50 µL
DH5α
none
none
50 µL
JW3367-3
none
none
50 µL
JW3367-3
none
none
Cells and DNA were incubated at 4° C for 1 h, heat shocked at 42 ° C for 1 minute and put on ice at 4°C for 5 more minutes. Add 1 mL of 37° C pre-warmed.
Transformation of DH5α with eYFP (pSB1C3/BBa_R0082/BBa_E0430) and GFP (pSB1C3/BBa_R0082/BBa_E0240) reporter constructs
150 µL competent DH5ɑ cells were thawed at 4°C split into three 50 µL tubes. Plasmids were added:
cells
Plasmid
amount
DH5α
pSB1C3/BBa_R0082/BBa_E0430
50 ng
DH5α
pSB1C3/BBa_R0082/BBa_E0240
50 ng
DH5α (negative control)
none
none
The cells were incubated at 4°C for 1h, heat shocked at 42°C for 1 min and put back on ice at 4°C for 5 minutes. 950 µL 37°C warm LB broth was added to each tube and the cells were incubated at 37°C / 900 RPM on the thermomix for 1 h.
Cells were centrifuged, resuspended in 50 µL LB broth, brought onto LB-Agar/CAmp Plates and incubated at 37°C overnight.
Fabrication of ED01 SU-8 Master (1st attempt)
A 4-inch Si-wafer was used to fabricate a master(stamp) for polydimethylsiloxane (PDMS) printing. Channels of the final microfluidic (MF) chip have 125 μm width according to CAD file.
SU-8 photoresist was chosen to serve as the imprinting material
To acquire square sections of the MF-chip channels, the Master was processed according to the datasheet for typical processes with 100 μm resist thickness
The 4-inch Si-wafer was baked in an oven at 130 °C for 10 minutes
SU-8 (Gersteltec GM1075) was spin coated on the Si-wafer directly after baking using the following parameters:
1) 100 rpm/second acceleration to 1700 rpm (for 17 seconds)
2) 100 seconds spinning at 1700 rpm
3) spin peak with 400 rpm/second acceleration to 2100 rpm (for 1 second)
4) -100 rpm/second deceleration to 0 rpm (for 21 seconds)
The Si-wafer was kept at 40 °C for 30 minutes to allow for relaxation of the resis edges
appearing gas bubbles in the resist were popped with a sterile syringe tip
The wafer was soft baked for 2 minutes at 120 °C with temperature ramps of 4 °C/minute up and down
The wafer was illuminated for 16 seconds with an exposure dose of 13 mW/cm2 (208 mJ/cm2 - instead of recommended 200 mJ/cm2)
After exposure the wafer was stored for a delay time of 1 hour to let the created initiator molecules diffuse until homogeneity was reached
The wafer was PEBd for 30 minutes at 95 °C with 4 °C/minute temperature ramps up and down from room temperature
After RT was reached, the wafer was put into used propylene glycol methyl ether acetate (PGMEA) for 3 minutes for development of the SU-8 resist. Slight detachment of the SU-8 from the Si-wafer was observed
Development was cancelled by rinsing with isopropanol after removing the wafer from PGMEA. Strong white liquid formation occurred whilst rinsing with isopropanol
SU-8 was undeveloped
The wafer was kept in isopropanol for about 1 hour
After removal from the isopropanol bath the wafer remained underdeveloped
The wafer was put back in used PGMEA for 3 minutes. Partial detachment of the remaining structures after completed development became clearly visible. The structures remained intact.
Rinsing with isopropanol showed slight to no white traces, indicating completed development
The imprint structures were clearly visible
The smallest features of the SU-8 Master detached completely and eventually rested randomly across the Master’s Si-wafer surface
Examination of the shape of the structures with a profilometer (DektakXT; Bruker) showed a SU-8 thickness of 65 +/- 1 μm (as opposed to 100 μm according to the datasheet)
SU-8 Master for the Encapsulation Device (ED01-Master01) was not suitable for channel formation with square-like sections
Fabrication of ED01 SU-8 Master (2nd attempt)
A 4-inch Si-wafer was put in an oven at 130 °C overnight from the seventh to the eighth of August
GM1060 photoresist (SU-8 thin application; Gersteltec) was used as a base adhesion layer for the GM1075 photoresist (SU-8 thick application; Gersteltec) to increase structure layer attachment to the substrate
The parameters used for the GM1060-SU-8 base layer were derived from the GM1060 datasheet typical processes for 6 μm layer thickness except for an adjusted spin speed of 6000 rpm and the cancellation of the process after the post-exposure bake (PEB) due to the lack of need for structuring
After dehydration of the Si-wafer in the oven the base layer was spinned on the Si-wafer with the following parameters:
Spinning:
100 rpm/second to 6000 rpm (for 60 seconds)
6000 rpm for 40 seconds
-100 rpm/second to 0 rpm (for 60 seconds)
Relaxation:
5 minutes at RT
Soft bake:
2 °C/minute from RT to 65 °C
5 minutes at 65 °C
2 °C/minute from 65 °C to 95 °C
5 minutes at 95 °C
coli down to 50 °C
Exposure:
20 seconds with 13 mW/cm2 (260 mJ/cm2 instead of recommended 200 mJ/cm2) on i-line (365 nm)
Delay:
10 minutes
PEB:
Same as soft bake except 15 minutes at 95 °C
After the soft bake small white dots (bubbles) were visible in or on the SU-8 layer, but they were too small to pop open, as well was too many; temperature profile may be irrelevant or relaxation time should be longer to allow for more gentle solvent evaporation
After the PEB the structural GM1075-SU-8 layer was applied and processed
The fabrication of the structural layer of ED01-Master02 was performed similiar to ED01-Master01, but according to the datasheet parameters for the typical process for 150 μm layer thickness with slight modifications to reach the desired 100 μm thickness. Following parameters were used:
Spinning:
100 rpm/second to 1200 rpm (for 12 seconds)
1200 rpm for 100 seconds
400 rpm/second to 1600 rpm (for 1 second)
-100 rpm/second to 0 rpm (for 16 seconds)
Relaxation:
30 minutes at 40 °C
Soft bake:
4 °C/minute from 40 to 120 °C
5 minutes at 120 °C
-4 °C/minute from 120 °C to RT
Exposure:
20 seconds with 13 mW/cm2 (260 mJ/cm2 instead of recommended 200 mJ/cm2 for 100 μm) on i-line (365 nm)
Delay:
1 hour
PEB:
4 °C/minute from 40 to 95 °C
40 minutes at 95 °C; instead of 30 minutes recommended for 100 μm
-4 °C/minute from 95 °C to RT
Slight to no white traces were observed during post-development isopropanol rinse; structures remained attached and intact remaining clear
Examination of ED01-Master02 and the shape of the structures with a profilometer (DektakXT; Bruker) showed a SU-8 thickness of 87 +/- 1 μm (as opposed to 150 μm according to the datasheet); structures appeared clear and the edges sharp; the measured properties are sufficient to serve as a stamp for the ED01-chips
To create castings of PDMS from the ED01-Master02 72.37 g of PDMS oligomer (Sylgard 184 silicone elastomer base) and 7.26 g of initiator (Sylgard 184 elastomer curing agent) were thoroughly mixed
The ED01-Master02 was put into a glass petri dish which was covered with aluminium foil
The PDMS mixture was poured over the ED01-Master02 and the dish was put into a desiccator to remove air bubbles from inside the PDMS
Subsequently the dish including the mixture was covered to prevent dust from entering and incubated in an oven at 65 °C overnight to allow it to polymerize completely
LB - medium
25 g LB - medium
add to 1 L H2O
-> autoclave
LB - agar
10 g LB - agar
add 250 mL H2O
-> autoclave
LB agar plates
heat 250 mL LB - agar (is hot from the autoclave)
Cool down to approx. 50 °C
add 250 µL chloramphenicol
Preparation of 13 plates under steril conditions
-> store at 4 °C
Mark as LB CAmp Plates 160809
Plating of JW 3367-3 and DH5α with fluorescence reporters
To use
3 plates (LB CAmp Plates iGEM 09.08.2015)
Cells LOT# 190707CL03 (JW 3367-3 with pSB1C3/BBa_R0082/BBa_E0430) (Omp/eYFP)
Cells LOT# 160803KP03 (DH5&alpha with pSB1C3/BBa_R082/BBa_E0240) (Omp/GFP)
Cells LOT# 160803KP02 (DH5&alpha with pSB1C3/BBa_R082/BBa_E0430) (Omp/eYFP)
Now work under sterile conditions:
For DH5α/eYFP: pick one colony from the plate 160803KP02, put it in 50 µL LB medium, put the 50 µL LB medium with cells on the LB - Agar Plate and grease it -> stored at 37 °C overnight
For DH5α/GFP: the same as the DH5α/eYFP
For JW 3367-3/eYFP: warm up the cryostock, than pick 5 µL up and put it on 45 µL LB medium, put the 50 µL to the plate and grease it  ->  stored at 37 °C overnight
Fabrication of ED01 MF-Chips
Petri dish with cured PDMS on ED01-Master02 was removed from the oven (65 °C) after overnight incubation
PDMS was removed from the Master by fragile pulling
PDMS was cut with scalpel and broken by hand to retrieve three replicated channel systems in one PDMS piece
Access channels for syringe tubes (input and output) were cut with a biopsy cutter / hole punch (0.75 mm diameter) and rinsed with isopropanol and dried with a N2-gun
3 glass object slides and 3 triple-device PDMS pieces were plasma treated in 02-plasma for 100 seconds at approx. 10 W and 0.38 mbar
The glass object slides and PDMS pieces were brought into contact with their plasma treated surfaces by pressing them together by hand, forming the ED01 MF-Chips
The ED01 MF-chips were kept still for 3 hours until stable binding of glass and PDMS was tested
One of the three created ED01 MF-chips bonded completely with all three devices appearing intact
Fluorescence microscopy of plates
Fluorescence microscopy of bacteria on plates:
DH5α/pSB1C3/BBa_R0082/BBa_E0240 (Omp/GFP). Magnification factor: 40x; exposure: 75,31 ms.
JW3367-3/pSB1C3/BBa_R0082/BBa_E0240 (Omp/GFP). Magnification factor: 40x; exposure: 75,31 ms.
JW3367-3/pSB1C3/BBa_R0082/BBa_E0240 (Omp/GFP). Magnification factor: 40x; exposure: 1 s.
Note: As expected, DH5α cells showed higher fluorescence than JW3367-3, which are lacking the EnvZ receptor required for activation of ompR.
FACS
Following cell lines were measured with FACS:
DH5α/pSB1C3/BBa_R0082/BBa_E0240 (Omp/GFP)
DH5α/pSB1C3/BBa_R0082/BBa_E0430 (Omp/eYFP)
JW3367-3/pSB1C3/BBa_R0082/BBa_E0240 (Omp/GFP)
JW3367-3/pSB1C3/BBa_R0082/BBa_E0430 (Omp/eYFP)
DH5α (negative control)
JW3367-3 (negative control)
As buffer 100 mM MgCL2 was used and the cells were diluted 1:100.
Setup: Cytometry was conducted with the FACSCalibur System configured as follows.
Detector
Voltage
Mode
FSC
E02
Lin
SSC
628
Lin
FL1
705
Log
Threshold FSC: 0                         Mode: Hi
Preparation of Alginic Acid Solution
Second creation of alginate solution 2.5 % ; soluted for about 12 hours
Preparation encapsulated JW cells from 22.08.16
The OD600 of the overnight culture was measured and amounts to 4.770
Due to the encapsulation protocol which requires an OD of about 1.5, 2 mL of the main culture were diluted with 4 mL of fresh LB medium without any antibiotics
OD600 of the diluted culture equaled to 2.00
The 6 mL of bacterial solution were centrifuged in 2 mL tubes separately for 5 minutes at 4500 rpm. Due to unexpected waiting time the mixtures were centrifuged for another 2 minutes before separating the pellet from the fluid
The 3 tubes with bacterial pellets were resuspended with 333 μL 0.9 % NaCl each and thereafter reunited
1 mL of diluted bacteria were mixed with 2 mL of 2.5 % alginate solution
Encapsulation was managed through pipetting the alginate-bacteria solution in 100 mL BaCl2
The BaCl2 was sterile filtered for reuse
The capsules were washed with 100 mL dist. water
About 10 capsules were crushed immediately and set into culture with 5 mL LB cAmp
Mistake: JW do not have cAmp resistance, therefore no growth was possible
Mistake was identified after first measurement (1.5 hours)
Measurements of the intact capsules in LB cAmp as well did not show any growth after 1.5 hours and 2 hours
Analogous to the BaCl2 encapsulation, CaCl2 encapsulation was performed; a bacterial solution of OD600 = 1.83 was used
Incubation with LB medium (antibiotics free) did not show growth
Encapsulated bacteria could be seen using a light microscope; bacteria supposedly strongly bound; new dilution/capsule-crushing methods needed
Overnight culture of crushed parts was fixed
OD600 Measurement of Overnight Cultures from 23.08.16
No visible indication for bacterial growth were observed in the overnight cultures
Restriction of BBa_C0082 and the Anderson Promoters for 3A Assembly
Part
BBa_C0082
BBa_K081005
BBa_K081006
BBa_K081007
DNA
48 µL
16 µL
10.3 µL
14.1 µL
NEB Buffer 2.1
6 µL
2 µL
2 µL
2 µL
XbaI
2 µL
-
-
-
PstI
2 µL
1 µL
1 µL
1 µL
H2O
2 µL
1 µL
6.7 µL
2.9 µL
Total
60 µL
20 µL
20 µL
20 µL
The DNA was restricted at 37 °C for 45 minutes. Enzymes were heat-inactivated at 65 °C for 20 minutes. DNA was purified using the GeneJET PCR Purification Kit, eluting the DNA with 8 µL elution buffer for the promoters and 27 µL for the BBa_C0082.
A second restriction was prepared:
Part
BBa_K081005
BBa_K081006
BBa_K081007
DNA
8 µL
8 µL
8 µL
FastDigest Buffer
1 µL
1 µL
1 µL
BcuI
1 µL
1 µL
1 µL
Total
10 µL
10 µL
10 µL
The DNA was restricted at 37 °C for 30 minutes and purified using the GeneJET PCR Purification Kit, eluting with 8 µL.
Ligation of BBa_C0082 with the Anderson Promoters
Three ligation reactions were set up as following:
Part
BBa_K081005
BBa_K081006
BBa_K081007
Anderson Promoter
8 µL
8 µL
8 µL
BBa_C0082
9 µL
9 µL
9 µL
T4 DNA Ligase
1 µL
1 µL
1 µL
Ligation Buffer
2 µL
2 µL
2 µL
Total
20 µL
20 µL
20 µL
The ligation was conducted overnight at 16 °C.
Transformation of JW3367-3 with BBa_C0082/Anderson Promotor ligation product and BBa_B0015
Transformation was conducted according to the Zhang Gong Protocol for Transformation. 2  µL of DNA were added to the competent cells of each BBa_B0015 and BBa_C0082/Anderson plasmid transformation.
New colony PCRs of Anderson Promoters
All Colony PCRs of the Anderson promoters were negative. Screen of 15 colonies from each plate.
A: pSB1C3/BBa_K081005 → Colony  1 - 15
B: pSB1C3/BBa_K608006 → Colony 16 - 30
C: pSB1C3/BBa_K608007 → Colony 31 - 45
TAQSCREEN:
98 °C → 5 min
98 °C → 10 s
56 °C → 30 s
72 °C → 2:15 min
Miniprep of  Terminator (BBa_B0015)
The overnight culture of JW 3367-3 cells containing pSB1C3/BBa_B0015 was prepared and prepped using the ThermoScientific GeneJET Plasmid Miniprep Kit, following the respective protocol , eluting with 100 μL each. The resulting sample contained:
Plasmid
DNA concentration [ng/μL]
pSB1C3/BBa_B0015
46.42
Gel electrophoresis of colony PCR of BBa_K081005, BBa_K608006 and BBa_K608007
Control of the colony PCR of BBa_K081005, BBa_K608006 and BBa_K608007
Positive results for A16, B16, C9, C10, C11
Overnight cultures for positive colonies were produced and incubated overnight at 37 °C.
Restriction of pSB1C3/BBa_R0082/BBa_I732018 and BBa_B0015
First restriction
Components
Volume [μL]
BBa_R0082/BBa_I732018
14.5
PstI
1.0
NEB Buffer 3.1
5.0
water
29.5
Total:
50.0
Second restriction
Components
Volume [μL]
BBa_B0015
21.5
PstI
1.0
XbaI
1.0
NEB 3.1 Buffer
5.0
water
21.5
Total:
50.0
Incubation for 10 minutes at 37 °C each and heat inactivation of the first restriction at 80 °C for 20 minutes. Purification of BBa_B0015 fragment using the GeneJET Gel Extraction Kit, with concentrations of 89.55 ng/µL and 91.46 ng/μL. A cryostock of JW3367-3/pSB1C3/BBa_B0015 was prepared.
Encapsulation of JW cells in alginate
Preparations:
-Sodium citrate 55 mM
-JW cell culture
Encapsulation:
-OD600 = 1.6 of JW cell culture
-3 tubes with each 1 mL of culture centrifuged 5 minutes at 4500 rpm
-Pellet was resuspended with 1 mL 0.9 % NaCl each
-Encapsulation was performed by pipetting of 1 mL NaCl-cells suspension with 2 mL alginate into 100 mL CaCl2
Solution of Beads:
-10 mL Na-citrate were incubated with beads for approx. 1 hour
-Na-citrate-beads sliution was centrifuged for 5 minutes at 4500 rpm
-Pellet was resuspended in LB / NaCl medium (several tubes - several pellets)
Cultivation:
-beads were cultivated and thereafter lysed in Na-citrate (up) - cultivated again (37 °C LB/CaCl2)
-OD measurement revealed growth in the JWs cultivated in LB medium
-One culture was cultivated overnight. Another one was lysed overnight
Evaluation of Encapsulated  JW cells from 06.09.16
OD6600 measurement:
-LB-JW (marked red): OD600 = 4.918
-LB-JW (marked green) diluted ½ : OD600 = 3.550; undiluted around 7
-Na-citrate-JW (marked red): OD600 = 0.002
-Na-citrate-JW (marked green): OD600 = 0.006
The vitality of the cells in the mixture with sodium citrate and LB has been confirmed via microscopy
Miniprep of BBa_K081007/BBa_C0082
The overnight culture of JW3367-3 cells containing pSB1C3/BBa_K081007/BBa_C0082 was prepared and prepped using the ThermoScientific GeneJET Plasmid Miniprep Kit, following the respective protocol , eluting two times.
Colony
Concentration of first elution [ng/μL]
Concentration of second elution [ng/μL]
9c
148.73
22.79
10c
185.27
13.55
11c
161.75
23.91
Restriction of pSB1C3/BBa_K081007/BBa_C0082
Restriction following the respective protocol .
pSB1C3/BBa_K081007/BBa_C0082
Colony 9c
pSB1C3/BBa_K081007/BBa_C0082
Colony 10c
pSB1C3/BBa_K081007/BBa_C0082
Colony 11c
Plasmid [in μL]
13.5
10.8
12.4
EcoRI [in μL]
2.0
2.0
2.0
10x CutSmart [in μL]
5.0
5.0
5.0
H2O [in μL]
29.5
32.2
28.6
Incubation was performed at 37 °C for 15 minutes and then loaded on an agarose gel (1 %) for 1 hour at 120 V.
Control of the restriction of BBa_K608007/BBa_C0082
Upper and two of the lower bands were cut out and extracted, using the GeneJET Gel Extraction Kit, following the respective protocol .
Restriction of pSB1C3/BBa_R0082/BBa_1732018
Restriction following the respective protocol .
pSB1C3/BBa_R0082/BBa_I732018
Plasmid
15.0 μL
EcoRI
1.0 μL
10x CutSmart
5.0 L
H2O
29.0 μL
Heat inactivation for 20 min (65 °C).
+ 1 μL BcuI
PCR Purification with the ThermoScientific GeneJET PCR Purification Kit, following the respective protocol
Restriction of pSB1C3/BBa_B0015
Restriction following the respective protocol .
pSB1C3/BBa_B0015
Plasmid
11.2 μL
EcoRI
1.0 μL
XbaI
1.0 μL
10x CutSmart
5.0 μL
H2O
31.8 μL
Heat inactivation for 20 min (65 °C).
Ligation
The consolidated extracted bands of plasmid pSB1C3/BBa_K081007/BBa_C0082 were self-ligated with the following reaction:
Components
Volume [μL]
pSB1C3/BBa_K081007/BBa_C0082
15.0 μL (100 ng)
Ligase T4
1.0 μL
Ligase Buffer
2.0 μL
H2O
Add 20μL
Incubation at RT for 25 minutes. Heat inactivation at 65 °C for 10 minutes. Storage at -20 °C over weekend.
Cryostock and Miniprep of pSB2K3/BBa_C0082
Cryostock: 500 μL of the ONC and 500 μL of 1:1 Glycerol-water mixture. Storage at -80 °C.
Miniprep was conducted according to the GeneJET Miniprep Kit Protocol, following the respective protocol .
Concentrations in first elution 156.66 ng/μL and 26.15 ng/μL.
Restriction of Anderson Promoters (BBa_K081005, BBa_K081006, BBa_K081007) with Taz (BBa_C0082)
Part
BBa_C0082
BBa_K081005
BBa_K081006
BBa_K081007
Concentration [ng/µL]
156.66
73.35
112.41
81.74
Volume [µL for 500 ng]
3.6
6.9
4.5
6.2
NEB Buffer 3.1 [µL]
6
2
2
2
XbaI [µL]
2
-
-
-
PstI [µL]
2
1
1
1
H2O [µL]
40.4
10.1
12.5
10.8
Total [µL]
60
20
20
20
The DNA was restricted for 1 h for the Anderson promoters and 1 hour and 15 minutes for Taz at 37 °C. Enzymes were heat-inactivated for 20 minutes at 65 °C.                             The restrictions of the three Anderson promotors were purified using the GeneJET PCR Purification Kit. DNA was eluted with 9 µL.                             A second restriction was set up:
Part
BBa_K081005
BBa_K081006
BBa_K081007
Volume [µL]
17
17
17
FastDigest Buffer [µL]
2
2
2
BcuI [µL]
1
1
1
Total: [µL]
20
20
20
The DNA was restricted at 37 °C for 30 minutes. The DNA was purified using the GeneJET PCR Purification Kit and eluted with 10 µL. The concentration was measured at the Nanodrop.
Part
BBa_K081005
BBa_K081006
BBa_K081007
Concentration [ng/µL]
10.53
7.67
7.34
Gel electrophoresis of BBa_C0082 (restricted with XbaI and PstI)
A 2% gel was run for 45 minutes at 130 V.
The lower band at 1.5 kb was cut out and the DNA was extracted from the gel with the GeneJET Gel Extraction Kit. DNA was eluted with 32 µL.                             The DNA-concentration of Taz was 181 ng/µL.
Ligation of BBa_R0082/BBa_I732018 with Terminator BBa_B0015
Components
Volume [µL]
BBa_R0082/BBa_I732018 (Insert)
15
BBa_B0015
0.5
T4 Ligase
1
T4 DNA Ligase Buffer
2
water
1
Total:
20
Ligation of Anderson Promoters with Taz
BBa_K081005
BBa_K6081006
BBa_K6081007
Vector (Anderson) [µL]
9
9
9
Insert (BBa_C0082) [µL]
2
2
2
T4 Ligase [µL]
1
1
1
T4 DNA Ligase Buffer [µL]
2
2
2
H2O [µL]
6
6
6
Total: [µL]
20
20
20
Incubation for 15 minutes at RT and heat-inactivation for 10 minutes at 65 °C.
Fabrication of DD01-Chamber-Mask01
Cultivation of Frozen Capsules
Light microscopy of the capsule rest/remains and the lysed capsules showed high viability
Encapsulation with glycerol-alginate mix does not work, capsules get fragile and start releasing bacteria
New encapsulation of bacteria in three different approaches:
1) Regular encapsulation; frozen on pure glycerol
2) Regular encapsulation; frozen on glycerol / LB
3) Encapsulation with alginate-glycerol mix; frozen on 0.9 % NaCl
Production of 50 mL 1 M Tris-HCL Buffer
6.057 g TRIS
Adjust to pH 8 with HCl
Fill up to 50 µL with water
Production of 0.5 L TE Buffer
5 mL 1M TRIS-HCl at pH 8
2 mL 0.5 M EDTA(Na2) also at pH 8
Add 494 mL water
PCR of mTaz and NOD1/EnvZ 3’/5’ OEPCR fragments
mTaz
NOD1/EnvZ 5’ OEPCR
NOD1/EnvZ 3’ OEPCR
Q5 2x Mastermix (Anderson) [µL]
25
Template [µL]
2.5
1.5
1.5
T4 Ligase [µL]
1
1
1
2.5 µL fw primer (10 mM)
GCAAAAAGAATTCGCGG
GCAAAAAGAATTCGCGG
GGCAGAATCGATCAATAAAGA
2.5 µL rev primer (10 mM)
GCTTTTCTGCAGCGG
ATATCTTTATTGATCGATTCTGCC
GCTTTTCTGCAGCGG
H2O [µL]
Add to 50 µL
The following PCR setup was used:
1.
98° C
0:30
2.
98° C
0:10
3.
47° C
0:20
4.
72° C
1:00
5.
GOTO 2 repeat 28x
6.
72 °C
2:00
7.
HOLD at 4° C
Gel electrophoresis of the recombinant PCR of NOD1/EnvZ 3'5' and PCR of mTaz
10 µL of PCR product was mixed with 2 µL of 6x loading dye and run on an 1% agarose gel at 130 V for 40 minutes.
>
Bands confirmed presence of RPCR 5’ and 3’ fragment as well as mTaz. However, a second band in mTaz (~600 bp) could be observed.                             The PCR reaction solutions were purified using the PCR Purification Kit. Elution in 35 µL.
mTaz concentration: 73.64 ng/µL
5’ RPCR concentration: 69.92 ng/µL
3’ RPCR concentration: 77.95 ng/µL
All PCR products were stored at -20 °C.
Cryostock of the overnight culture
DH5α, Anderson 05
DH5α, Anderson 06
DH5α, Anderson 07
The cryostocks were stored at -80 °C.
Miniprep of the overnight culture
GeneJET Miniprep Kit Protocol was used to the letter, and elution volume was 35 µL.
Construct
Concentration [ng/µL]
DH5α, Anderson 05
28.01
DH5α, Anderson 05
30.27
DH5α, Anderson 05
42.99
DH5α, Anderson 06
67.88
DH5α, Anderson 06
41.91
DH5α, Anderson 06
38.04
DH5α, Anderson 07
41.34
DH5α, Anderson 07
72.14
DH5α, Anderson 07
30.26
BBa_I732018/BBa_R0082/BBa_B0015
29.99
BBa_I732018/BBa_R0082/BBa_B0015
55.97
BBa_I732018/BBa_R0082/BBa_B0015
31.69
Sequential Restriction of OmpR/GFP and OmpR/YFP
First the DNA concentrations of the plasmids were measured.
Construct
DNA Concentration [ng/µL]
BBa_R0082/BBa_E0240
90.63
BBa_R0082/BBa_E0430
143.74
First restriction with EcoRI
BBa_R0082/BBa_E0240
BBa_R0082/BBa_E0430
Plasmid (1 µg) [µL]
11
7
EcoRI [µL]
1
1
10x CutSmart [µL]
2
2
H2O [µL]
2
2
The restriction preparations were incubated at 37 °C for 15 min and then heat-inactivated at 65 °C for 20 min.                             The restriction preparations were purified using the GeneJET PCR Purification Kit. DNA was eluted with 20 µL Elution Buffer. Afterwards DNA concentration was measured at the NanoDrop.
DNA concentration [ng/µL]
Purified BBa_R0082/BBa_E0240
90.39
Purified BBa_R0082/BBa_E0430
81.81
Second restriction with Bcu I
Purified OmpR/GFP
Purified OmpR/YFP
Plasmid (1 µg) [µL]
12
13
BcuI [µL]
1
1
FD Buffer [µL]
2
2
ddH2O[µL]
5
4
The restriction preparations were incubated at 37 °C for 10 min.                             The restriction preparations were purified using the GeneJET PCR Purification Kit. DNA was eluted with 20 µL Elution Buffer. Afterwards DNA concentration was measured at the NanoDrop.
DNA concentration [ng/µL]
260/230
280/260
Purified BBa_R0082/BBa_E0240
90.39
0.76
1.77
Purified BBa_R0082/BBa_E0430
34.25
1.96
1.79
Ligation of BBa_R0082/BBa_E0430 and BBa_R0082/BBa_E0240 with BBa_B0015 (Terminator)
A 3:1 (insert:vector) ligation was performed:
OmpR/GFP
OmpR/YFP
Vector: restricted BBa_B0015 ( 160909DK05) [µL]
1
1
Insert (71 ng) [µL]
3
2
T4 Ligase [µL]
1
1
T4 Ligase Buffer [µL]
2
2
ddH2O[µL]
13
14
The ligation preparations were incubated at RT for 50 min and then heat-inactivated at 65 °C for 10 min.
Measurement of cultivated frozen capsules
OD600 measurements were performed on all three defrosted setups from 13th September
OD600 = 2.346
OD600 = 2.406
OD600 = 2.458
Chloramphenicol Stock Solution
A Chloramphenicol (CAmp) stock solution was prepared following the respective protocol .
LB-Agar Plates
LB-Agar plates were prepared following the respective protocol .
Transformation of JW 3367-3 with Fluorescence Reporters and B0015 - Quantification
Average amount of colonies for C2987/BBa_E0240/B0015
1 μL DNA, 30 μL Trafovolume
1 μL DNA, 60 μL Trafovolume
1 μL DNA, 90 μL Trafovolume
~100 colonies
~200 colonies
~250 colonies
Average amount of colonies for JW3367-3 BBa_E0430/B0015
1 μL DNA
2 μL DNA
5 μL DNA
50 μL Trafovolume
20 colonies
not done
33 colonies
65 μL Trafovolume
not done
21 colonies
not done
90 μL Trafovolume
25 colonies
~35 colonies
~50 colonies
LB-Agar Plates
LB-Agar plates were prepared following the respective protocol .
Vitality examination of encapsulated frozen bacteria via light microscopy
Bacteria from the setups from the 13th September were examined via light microscopy
1) E. coli vital
2) E. coli vital
3)  alive but slightly lower activity
Validation of LacZα Reporter System by β-Galactosidase Activity
20 μL of JW3367-3 cell transformed with pSB1C3/BBa_R0082/BBa_I732018 were plated onto a LB-Agar/CAmp plate containing 10 μL 10 mg/mL X-Gal and incubated one night at 37°C.
Transformation of JW 3367-3 with pSB1C3/BBa_I732020 (lacZα/Terminator)
150 μL competent JW 3367-3 cells were split into 3 x 50 μL and inoculated with DNA:
(+) postive control: 1 μL pSB1C3/BBa_R0082/BBa_I732018
(-) negative control: no addition of DNA
1 μL pSB1C3/BBa_I732020, resuspended from iGEM Distribution Kit Plate 3, 2F
Transformation was carried out using the respective protocol .
PCR purification of the linearized plasmids pSB1C3, pSB1A3, pSB1T3
A PCR was performed following the GeneJet PCR Purification Kit Protocoll.                             The following concentrations were measured with Nanodrop
Plasmid
Concentration [ng/µL]
pSB1C3
a
158.77
b
184.20
c
132.83
pSB1A3
189.96
pSB1T3
145.48
PCR amplification of EnvZ Fusionprotein_Tai/RFC25, EnvZ_First_Transmembrane and NOD_1LRRRFC25
Volume [µL]
Q5 2x Mastermix
25
EnvZ 5' fw
2.5
Biobrick 3'
2.5
Template
1
H2O
19
The PCR preparations were mixed according to scheme above. As a template the EnvZ Fusionsprotein_TailRFC25 was used in preparation A.                             In preparation B the EnvZ_First_Transmembrane and in C the NOD_LRR_RFC25 were used.
The following PCR setup was used:
1.
98° C
0:30
2.
98° C
0:10
3.
47° C
0:20
4.
72° C
1:00
5.
GOTO 2 repeat 30x
6.
72 °C
3:00
7.
HOLD at 4° C
Analysis of the PCR via gel electrophoresis (plasmid pSB1C3, pSB1A3, pSB1T3)
An agarosegel (1%) was used to analyze the PCR reaction of the plasmids pSB1C3, pSB1A3 and pSB1T3.                             The first lane contained pSB1C3 a, the second pSB1C3 b,the third pSB1C3 c, the fourth pSB1A3 and the fifth pSB1T3.                             The gel was incubated with ethidiumbromide for 30 minutes before running it at 150 V for 30 minutes.                             All preparations contained DNA fragments contained DNA fragments with the expected length.
Analysis of the PCR via gel electrophoresis (EnvZ Fusionprotein_Tail RFC25, EnvZ_First Transmembrane and NOD1_LRR RFC25)
A Agarosegel (1%) was used to analyze the PCR reaction.                             The first lane contained the EnvZ Fusionprotein_TailRFC25, the second EnvZ First Transmembrane and the third lane NOD1_LRR RFC25.                             The gel was incubated with ethidiumbromide for 30 minutes before running it at 150 V for 30 minutes.                             All preparations contained DNA fragments contained DNA fragments with the expected length.
Restriction of pSB1A3, mTaz, EnvZ, pSB1C3 and Reporter Systems
Following scheme was used:
Part
pSB1C3
pSB1A3
mTaz
EnvZSH
EmvZFH
NOD1LRR
pSB1C3/BBa_R0082
pSB1C3/BBa_I532020
pSB1C3/BBa_K081005
pSB1C3/BBa_K608006
pSB1C3/BBa_E0430
pSB1C3/BBa_E0240
Volume [μL]
17
14.1
13.3
7.9
7.7
13.8
5.0
25.8
18
18
2.9
4.7
Buffer
5 μL NEB Buffer 2.1
5 μL NEB Buffer 2.1
5 μL NEB Buffer 2.1
5 μL NEB Buffer 2.1
5 μL NEB Buffer 2.1
5 μL NEB Buffer 2.1
5 μL Thermo FD
5 μL NEB Buffer 2.1
5 μL Thermo FD
5 μL Thermo FD
5 μL NEB Buffer 2.1
5 μL NEB Buffer 2.1
EcoRI [μL]
1
1
1
1
1
1
-
-
-
-
-
-
XbaI [μL]
-
-
-
-
1
1
-
-
-
-
-
-
BcuI [μL]
-
-
-
-
-
-
1
-
1
1
-
-
PstI [μL]
1
1
1
1
1
1
-
1
-
-
1
1
H2O [μL]
26
28.9
29.7
35.1
35.3
29.2
39
17.2
26
26
40.1
38.3
A second restriction was set up:
Part
pSB1C3/BBa_R0082 restricted with BcuI
pSB1C3/BBa_I532020 restricted with XbaI and PstI
pSB1C3/BBa_K081005 restricted with BcuI
pSB1C3/BBa_K608006 restricted with BcuI
Volume [μL]
50
25.8
50
50
NEB 2.1 Buffer [μL]
6
6
6
6
EcoRI [μL]
2
-
1.2
1.2
PstI [μL]
-
1
-
-
XbaI [μL]
-
1
-
-
H2O [μL]
2.8
17.2
2.8
2.8
Ligation of pSB1A3 and pSB1C3 with the Part Precursors and Fluorescence Reporters as well as pSB1C3 with BBa_R0082 and BBa_I732020
Part 1
5 μL pSB1C3
5 μLpSB1C3
5 μLpSB1C3
5 μLpSB1C3
5 μLpSB1A3
5 μLpSB1A3
5 μLpSB1A3
Part 2
10 μ mTaz
25 μ EnvZ SH
25 μ EnvZ FH
25 μ NOD1-LRR
20 μ BBa_E0430
20 μ BBa_E0240
30 μ BBa_R0082
Part 3
-
-
-
-
-
-
BBa_I732020
T4 Ligase Buffer [μL]
5
5
5
5
5
5
25
T4 Ligase[μL]
2.5
2.5
2.5
2.5
2.5
2.5
10
H2O [μL]
27.5
12.5
12.5
12.5
17.5
17.5
25
Transformation of JW3367-3 with the Ligation Preparations
Competent JW 3367-3 cells were thawed on ice at 4°C and inoculated with the ligation preparations. For each ligation preparation 150 μL competent cells were used, for the positive control (transformed with pcDNA3) and negative control 75 μL were used.                             Transformation was carried out using the respective protocol .
Fabrication of Diagnostic Device DD01 (SU-8) Master02 continued
Continuation after application of base / adhesion layer from the 27th of September
Structural layer of SU-8 (GM1075)
-Spin coating:
1) 0 - 1000 rpm (100 rpm/second acceleration) in 10 s
2) 1000 rpm for 100 seconds
3) 1000 - 1400 rpm (400 rpm/second acceleration) in 1 s
4) 1400 - 0 rpm (-100 rpm/second acceleration) in 14 seconds
-30 minutes relaxation time on hotplate at 40 °C:
The wafer was covered with a crystallization dish, but allowed for breathing due to evaporating SU-8 solvent (PGMEA), otherwise condensation occurs on covering dish and curing during prebake would be insufficient due to excessive vapour pressure
-Soft bake:
1) 40 - 120 °°C in about 10 minutes (avg +8 °C/minute)
2) 120 °C for 12 minutes
3) 120 °C - 40 °C in about 20 minutes (avg -4 °C/minute)
-i-line exposure 3) 120 °C - 40 °C in about 20 °minutes (avg -4 °C/minute)
-1 hour delay time at RT in wafer box before PEB
-PEB:
1) RT - 120 °C in about 11 minutes (avg 9 °C/minute); accidentally - 2 bubbles created on structure-free space
2) 120 °C - 95 °C in about 3 minutes (avg -8 °C/minute)
3) 95 °C for 50 minutes
4) 95 °C - 36 °C in about 25 minutes (avg -2.4 °C/minute)
-Develop:
-In PGMEA (used) for about 8 minutes while constantly agitating the PGMEA and the wafer in the beaker
-Wafer was rinsed with isopropanol to stop the development which caused white traces on wafer
-Isopropanol-rinsed wafer was washed the with new, used PGMEA and subsequently rinsed with isopropanol which caused the previous white traces to vanish, however, lines of residue from isopropanol evaporation remained
-Examination of the profile (using the DektakXT; Bruker) showed a thickness of approx. 110 μm; comparison between the depth of the large filter structure holes and the fine filter structure holes (as visible in the live video in the DektakXT using a ruler) showed similar depth (7 mm at 100x magnification) and can be checked in the captured images
Bonding of Diagnostic Chip (Channel and Chamber) PDMS Components (PDMS batch from the 24th of September on DD01-Master01)
Holes were drilled with 0.75 hole punch through the channel component
Droplets of isopropanol were dripped on both components and gently rubbed with the fingertips to clean the PDMS components bonding surface; PDMS was subsequently dried using a N2-gun and leaving it at ambient air
Effects of cleaning were examined under a light microscope:
1) 40 μm square-shaped filter structures (of about 100 μm height) sticked onto each other with the side walls; therefore the filter structure’s columns were bent which resulted in increased filter size due to locally increased width of the filter channels
2) Touching the filter structures with a sharp metal syringe tip causes unsticking of the filter structures from one another
3) Wetting with isopropanol caused resticking and resticking persisted upon drying; after completed drying and touching unsticking recurred
Both components bonding surfaces were O2-plasma treated simultaneously using the parameters: 100 seconds; about 25 W; 0,4 mbar O2
Surfaces were examined using a light microscope; filter structures still remained largely intact and in the same state as before the O2-plasma
Wetting of complete bonding surface of chamber component with H2O (demineralised) droplets(5 - 10 droplets)
Surfaces were once again examined using a light microscope: filter structures still intact
PDMS component’s bonding surfaces were brought together with remaining intermediate H2O layer
Channel and chamber components were aligned under a light microscope using tweezers to align; used too much water - too slippery, next time use 1 - 2 droplets (formed by a syringe tip with diameter of about 0.6 mm); alignment markers / triangles do not lock into one another because the PDMS is too rigid and does not bent enough to compensate the 100 μm height difference (10:1 oligomer:initiator ratio used)
The chips were put into an oven overnight at 40 °C to cure and bond without further touching or pressing
Preparation of LB-Agar
1 x 40 g LB-agar were put in 1 L H2O.
The suspension was autoclaved.
Preparation of LB-Agar/Tetracycline-Plates
1 mL of tetracycline stock solution were added into 1 L liquid  LB-agar.
The solution was poured into petri dishes under sterile conditions.
Plates were stored at 4°C.
Preparation of Tetracycline Stock Solution
3 x 10 mg tetracycline were solved in 1 mL H2O.
Steril filtered with a 0.22 μm filter.
Stored at -20°C.
Preparation of LB-Medium
2 x 25 g LB-medium were put in 1 L H2O.
The suspension was autoclaved.
Stored at 4°C.
PCR of EnvZ RFC25
Following scheme was used for the different preparations.
Template
1 μL
Primer fw
2.5 μL 
Primer rv
2.5 μL 
10 x Dream Taq buffer
5 μL
dNTPs (10 mM)
1 μL
Taq polymerase
0.1 μL
H2O
39.5 μL
50 μL
There were three preparations:
Template
Primer fw
Primer rv
EnvZ RFC25 TM1
RFC23 TMFirst fw
RFC23 TMFirst rv
Nod1/EnvZ RFC25 Tail
RFC23 EnvZ Tail fw
RFC23 EnvZ Tail rv
Nod1/LRR RFC25
RFC23 NOD1-LRR fw
RFC23 NOD1-LRR rv
Preparations of Competent Cells DH5α
Overnight cultures of the DH5α were splitted into two 1000 mL Erlenmeyer flask in 500 mL LB-medium and grown at 37°C, 250 rpm.
The OD was checked till it reached 0.4
Time after incubation start [min]
OD (1.=flask 1; 2.=flask 2 )
20
1. 0.044; 2. 0.026
50
1. 0.053; 2. 0.030
80
1. 0.055; 2. 0.026
120
1. 0.278; 2. 0.144
2. 0.352
2. 0.420 -> done
Cells were chilled on ice for at least 5 min.
Cells were centrifuged at 5000 rpm for 10 min at 4°C.
Flow-through was discarded.
Cells were resuspended in 40 mL cold Ca/glycerol buffer on ice.
Cells were chilled on ice for at least 30 min.
Cells were centrifuged at 4000 rpm for 10 min at 4°C.
Flow-through was discarded.
Cells were resuspended in 40 mL cold Ca/glycerol buffer and chilled on ice for 90 min.
Cells were collected by centrifugation (5000 rpm, 15 min, 4°C)
Cells were resuspended in 6 mL cold Ca/glycerol buffer on ice.
Aliquots of 150 μL each were made.
Aliquoted competent cells were frozen in liquid N2 and stored at -80°C.
PCR of EnvZ RFC25
Following scheme was used for the different preparations.
Template
1 μL
Primer fw
2.5 μL 
Primer rv
2.5 μL 
10 x Dream Taq buffer
5 μL
dNTPs (10 mM)
1 μL
Taq polymerase
0.1 μL
H2O
39.5 μL
50 μL
There were three preparations:
Template
Primer fw
Primer rv
EnvZ RFC25 TM1
RFC23 TMFirst fw
RFC23 TMFirst rv
Nod1/EnvZ RFC25 Tail
RFC23 EnvZ Tail fw
RFC23 EnvZ Tail rv
Nod1/LRR RFC25
RFC23 NOD1-LRR fw
RFC23 NOD1-LRR rv
PCR Purification of ENvZ/LRR
The PCR products were purified using the respective protocol .                             DNA concentrations were measured:
DNA
DNA concentration [ng/μL]
EnvZ RFC25 TM1
36.36
Nod1/EnvZ RFC25 Tail
97.89
Nod1/LRR RFC25
33.72
PCR of mTaz
Following scheme was used:
Template
1 μL
Primer EnvZ 5' fw
2.5 μL 
Primer EnvZ 3' rv
2.5 μL 
Q5 Master Mix 2x
25 μL
H2O
19 μL
50 μL
Following programme was used for the PCR:
1. 98 °C
30 sec
2. 98 °C
10 sec
3. 47 °C
20 sec
3. 72 °C
1.5 min
4. GOTO 2.
Repeat 34x
5. 72 °C
3 min
4 °C
∞
PCR Purification of mTaz
The PCR product was purified using the respective protocol .                             DNA concentration was measured:
DNA
DNA concentration [ng/μL]
mTaz
104.36
PCR of mTaz
Following scheme was used:
Template
1 μL
Primer EnvZ 5' fw
2.5 μL 
Primer EnvZ 3' rv
2.5 μL 
Q5 Master Mix 2x
25 μL
H2O
19 μL
50 μL
Following programme was used for the PCR:
1. 98 °C
30 sec
2. 98 °C
10 sec
3. 47 °C
20 sec
3. 72 °C
1.5 min
4. GOTO 2.
Repeat 34x
5. 72 °C
3 min
4 °C
∞
The PCR products were loaded onto a 1%-gel. There was one band at the expected size 1.500 bp along with four other bands, that weren't the expected size.
PCR of mTaz
Following scheme was used:
Template
1 μL
Primer EnvZ 5' fw
2.5 μL 
Primer EnvZ 3' rv
2.5 μL 
10 Pfu buffer
5 μL
dNTPs (10 mM each)
1.5 μL
Pfu polymerase
1 μL
H2O
36.5 μL
50 μL
Following programme was used for the PCR:
1. 98 °C
30 sec
2. 98 °C
10 sec
3. 47 °C
20 sec
3. 72 °C
4 min
4. GOTO 2.
Repeat 34x
5. 72 °C
3 min
4 °C
∞
Restriction of pSB1C3, pSB1A3, mTaz and Taz
Part
EnvZ TM first
NOD1-LRR
EnvZ Tail
Volume [μL]
3.8
8.4
2.8
NEB Buffer 2.1 [μL]
2
2
2
EcoRI [μL]
0.5
0.5
0.5
PstI [μL]
1
1
1
1
1
1
H2O [μL]
3
3
9.5
31.8
13
20.1
Total [μL]
50
50
50
50
50
50
The DNA was restricted for 1 h at 37 °C. Enzymes were heat-inactivated for 20 minutes at 80°C.
Ligation
Part 1
pSB1C3
pSB1C3
pSB1A3
pSB1A3
pSB1A3
pSB1A3
pSB1A3
pSB1A3
Part 2
mTaz (IDT)
mTaz aliquot 2
BBa_K081005
BBa_K608006
BBa_K608007
BBa_K081005
BBa_K608006
BBa_K608007
Part 3
-
-
mTaz
mTaz
mTaz
Taz
Taz
Taz
T4 Ligase Buffer [μL]
6
6
6
6
6
6
6
6
T4 [μL]
1.2
1.2
1.2
1.2
1.2
1.2
1.2
1.2
H2O [μL]
2.1
38.3
7.6
7.6
7.6
7.6
7.6
7.6
Restriction of pSB1C3, pSB1A3, mTaz and Taz
Part
pSB1C3
pSB1A3
mTaz aliquot 1
mTaz aliquot 2
mTaz (IDT)
mTaz
Volume [μL]
40
40
33.5
11.2
30
22.9
NEB Buffer 2.1 [μL]
5
5
5
5
5
5
EcoRI [μL]
1
1
-
1
1
-
XbaI [μL]
-
-
1
-
-
1
PstI [μL]
1
1
1
1
1
1
H2O [μL]
3
3
9.5
31.8
13
20.1
Total [μL]
50
50
50
50
50
50
The DNA was restricted for 1 h at 37 °C. Enzymes were heat-inactivated for 20 minutes at 80°C.
Transformation of JW3367-3 and NEB competent E. coli with ligations
Following plasmids were transformed into following cells:
cells
Plasmids
NEB cc
pSB1C3/mTaz(IDT)
NEB cc
pSB1A3/BBa_K081005/mTaz
NEB cc
pSB1A3/BBa_K608006/mTaz
NEB cc
pSB1A3/BBa_K608007/mTaz
JW3367-3
pSB1C3/mTaz
JW3367-3
pSB1A3/BBa_K081005/mTaz
JW3367-3
pSB1A3/BBa_K608006/mTaz
JW3367-3
pSB1A3/BBa_K608007/mTaz
JW3367-3
pSB1A3/BBa_K081005/BBa_C0082
JW3367-3
pSB1A3/BBa_K608006/BBa_C0082
JW3367-3
pSB1A3/BBa_K608007/BBa_C0082
JW3367-3
(+) positive control: pcDNA (1:20)
JW3367-3
(-) negative control: - (plated onto CAmp plate)
JW3367-3
(-) negative control: - (plated onto Amp plate)
DH5α
competent cell control: pcDNA (1:20)
Transformation was carried out using the respective protocol . At the end cells were spun down, resuspended in 50 μL LB-medium and plated onto LB-agar plates with the suitable antibiotic.
Colony PCRs of cells transformed with pSB1C3/mTaz
Following mastermix for 30 PCR reactions was set up:
VF2 Primer fw
15 μL 
VR Primer rv
15 μL 
10 x Dream Taq buffer
30 μL
dNTPs (10 mM)
6 μL
Dream Taq polymerase
7.5 μL
H2O
226.5 μL
total
300 μL
28 x 10 μL of the mastermix was pipetted into PCR tubes, which were inoculated with:
Colony numbering
Content
1 - 2
old mTaz from 27.09.
3 - 28
different colonies from the transformations from 03.10.
Positive results (bands with expected sizes) for 4 - 6, 10, 13, 15, 20, 24 and 26.
Colony PCRs of cells transformed with pSB1C3/mTaz
Following mastermix for 90 PCR reactions was set up:
VF2 Primer fw
45 μL 
VR Primer rv
45 μL 
10 x Dream Taq buffer
90 μL
dNTPs (10 mM)
18 μL
Dream Taq polymerase
22.5 μL
H2O
679.5 μL
total
900 μL
80 x 10 μL of the mastermix was pipetted into PCR tubes, which were inoculated with:
Colony numbering
Content
29
NEB cc pSB1C3/mTaz (IDT)
30 - 31
NEB cc pSB1A3/BBa_K608007/mTaz 2
32 - 36
NEB cc pSB1A3/BBa_K081005/mTaz 2
37 - 44
NEB cc pSB1A3/BBa_K608007/mTaz 1
45 - 52
NEB cc pSB1A3/BBa_K608006/mTaz 1
53 - 60
NEB cc pSB1A3/BBa_K608007/mTaz 1
61 - 68
JW3367-3 pSB1A3/BBa_K081005/mTaz
69-76
JW3367-3 pSB1A3/BBa_K608006/mTaz
77 - 84
JW3367-3 pSB1A3/BBa_K608007/mTaz
85 - 92
JW3367-3 pSB1A3/BBa_K081005/BBa_C0082
93-100
JW3367-3 pSB1A3/BBa_K608006/BBa_C0082
101 - 108
JW3367-3 pSB1A3/BBa_K608007/BBa_C0082
Positive results (bands with expected sizes) for 29, 31, 37, 55, 68.
Minipreparation of Overnight Cultures
The minipreparation was carried out following the instructions of the GeneJET Plasmid Miniprep Kit .
Plasmid
DNA concentration
pSB1C3/mTaz colony 4
158.29 ng/μL
pSB1C3/mTaz colony 5
143.37 ng/μL
pSB1C3/mTaz colony 6
141.55 ng/μL
pSB1C3/mTaz colony 10
98.89 ng/μL
pSB1C3/mTaz colony 13
134.68 ng/μL
pSB1C3/mTaz colony 15
188.91 ng/μL
pSB1C3/mTaz colony 20
176.11 ng/μL
pSB1C3/mTaz colony 24
165.42 ng/μL
Restriction of EnvZ TM first, NOD1-LRR and EnvZ Tail for RFC23 Assembly
Part
EnvZ TM first
NOD1-LRR
EnvZ Tail
Volume [μL]
3.8
8.4
2.8
NEB Buffer 2.1 [μL]
2
2
2
EcoRI [μL]
0.5
0.5
0.5
PstI [μL]
0.5
0.5
0.5
H2O [μL]
13.2
8.6
14.2
Total [μL]
20
20
20
The DNA was restricted for 20  at 37°C. Enzymes were heat-inactivated for 30 minutes at 80°C.
Ligation of pSB1C3 with the EnvZ Parts
Three ligations were set up using restricted, dephosphorylated linear pSB1C3 and the restricted EnvZ parts.
Part 1
10.5 μL pSB1C3
4.6 μL pSB1C3
4.9 μL pSB1C3
Part 2
10 μL EnvZ TM first
10 μL NOD1-LRR
10 μL EnvZ Tail
T4 Ligase Buffer [μL]
2.5
2
2
T4 Ligase [μL]
0.5
0.5
0.5
H2O [μL]
1.5
2.9
2.6
Ligations were incubated at 22°C for 1 h. Ligase was heat inactivated at 80°C for 20 min.
Transformation of JW3367-3 with pSB1C3/EnvZ-parts
8 x 150 μL competent JW 3367-3 cells were thawed on ice at 4°C and inoculated with DNA as following:
cell amount [μL]
DNA
DNA amount [ng]
150
pSB1C3/EnvZ TM first ligation
200
150
pSB1C3/NOD1-LRR ligation
200
150
pSB1C3/EnvZ Tail ligation
200
50
pSB1C3/BBa_B0015
50 ng
50
- (negative control)
-
Transformation was carried out using the respective protocol . At the end cells were spun down, resuspended in 50 μL LB-medium and plated onto LB-agar/CAmp-plates.
Colony PCRs
Following mastermix for 40 PCR reactions was set up:
VF2 Primer fw
20 μL 
VR Primer rv
20 μL 
10 x Dream Taq buffer
40 μL
dNTPs (10 mM)
8 μL
Dream Taq polymerase
2 μL
H2O
310 μL
total
400 μL
33 x 10 μL of the mastermix was pipetted into PCR tubes, which were inoculated with:
Colony numbering
Content
1 - 3
NEB cc pSB1A3/BBa_K081005/mTaz 2
4 - 6
NEB cc pSB1A3/BBa_K081005/mTaz 1
7 - 9
NEB cc pSB1A3/BBa_K608006/mTaz
10 - 12
NEB cc pSB1A3/BBa_K608007/mTaz 1
13 - 15
JW3367-3 pSB1A3/BBa_K081005/mTaz
16 - 18
JW3367-3 pSB1A3/BBa_K608006/mTaz
19 - 21
JW3367-3 pSB1A3/BBa_K608007/mTaz
22 - 24
JW3367-3 pSB1A3/BBa_K081005/BBa_C0082
25 - 27
JW3367-3 pSB1A3/BBa_K608006/BBa_C0082
28 - 30
JW3367-3 pSB1A3/BBa_K608007/BBa_C0082
31
NEB cc pSB1A3/BBa_K608007/mTaz 2
32
NEB cc pSB1A3/BBa_K608007/mTaz
33
JW3367-3 pSB1C3/BBa_R0082/BBa_I7302020
Positive results (bands with expected sizes) for 12, 22, 25, 26, 31 and 32.
LB-Agar Plates with antibodies
CAmp/Amp
700&nbp;mL LB-agar were mixed with 700&nbp;μL CAmp and 700&nbp;μAmp. The solution was distributed onto petri dishes.
Amp
700&nbp;mL LB-agar were mixed with 700&nbp;μAmp. The solution was distributed onto petri dishes.
CAmp
600&nbp;mL LB-agar were mixed with 600&nbp;μL CAmp The solution was distributed onto petri dishes.
Transformation of JW3367-3 with the (m)Taz ligations and pSB1C3/OmpR/GFP
8 x 150 μL competent JW 3367-3 cells were thawed on ice at 4°C and inoculated with DNA as following:
cell amount [μL]
DNA
DNA amount [μL/100 ng]
150
pSB1A3/BBa_K081005/mTaz + pSB1C3/BBa_R0082/BBa_E0240
1.16 + 1.16
150
pSB1A3/BBa_K081005/BBa_C0082 + pSB1C3/BBa_R0082/BBa_E0240
1.39 + 1.16
150
pSB1A3/BBa_K608006/BBa_C0082 + pSB1C3/BBa_R0082/BBa_E0240
0.77 + 1.16
150
pSB1A3/BBa_K608007/BBa_C0082 + pSB1C3/BBa_R0082/BBa_E0240
1.58 + 1.16
negative control: 150
-
-
negative control: 150
pSB1A3/BBa_K608006/BBa_C0082
1.54/200 ng
negative control: 150
pSB1C3/BBa_R0082/BBa_E0240
2.32/200 ng
positive control: 150
pSB1A3/BBa_K608006/BBa_C0082 (plated onto LB-agar/Amp plates)
1.54/200 ng
positive control: 150
pSB1C3/BBa_R0082/BBa_E0240 (plated onto LB-agar/CAmp plates)
2.32/200 ng
Transformation was carried out using the respective protocol . At the end cells were spun down, resuspended in 50 μL LB-medium and plated onto LB-agar/CAmp/Amp plates (if not stated otherwise).
Colony PCRs of cells transformed with pSB1C3/mTaz
Following mastermix for 20 PCR reactions was set up:
VF2 Primer fw
10 μL 
VR Primer rv
10 μL 
10 x Dream Taq buffer
20 μL
dNTPs (10 mM)
4 μL
Dream Taq polymerase
1 μL
H2O
155 μL
total
200 μL
18 x 10 μL of the mastermix was pipetted into PCR tubes, which were inoculated with:
Colony numbering
Content
1 - 6
JW3367-3/pSB1C3/NOD1-LRR
7 - 12
JW3367-3/pSB1C3/EnzZ Tail
13 - 18
JW3367-3/pSB1C3/EnvZ TM first
Positive results (bands with expected sizes) for 6, 7, 10, 14 and 15.
Minipreparation of Overnight Cultures from the PCRs from 05.10. and 06.10
The minipreparation was carried out following the instructions of the GeneJET Plasmid Miniprep Kit .
Plasmid
DNA concentration
pSB1A3/BBa_K608006/BBa_C0082
93.04 ng/μL
pSB1A3/BBa_K608006/BBa_C0082
129.88 ng/μL
pSB1A3/BBa_K608007/mTaz
98.40 ng/μL
pSB1A3/BBa_K608007/mTaz
63.21 ng/μL
pSB1A3/BBa_K081005/mTaz
86.03 ng/μL
pSB1A3/BBa_K081005/mTaz
90.59 ng/μL
pSB1A3/BBa_K081005/BBa_C0082
71.80 ng/μL
Preparation of Aspartate Solution
2.5 g in 100 mL H2O
Measurement of Aspartate sensor (BBa_C0082)
16 samples JW 3367-3 cells with pSB1A3/BBa_K608006/BBa_C0082 and pSB1C3/BBa_R0082/BBa_E0240 were inoculated in 5 mL LB-medium with CAmp/Amp resistance and grown until OD600=0.178.
Incremental Aspartate concentrations were given for pairs of three, leading to concentrations of 0.0, 0.001, 0.005, 0.01, 0.025 [g/mL].
Incubation was performed for 30, 60, 120 minutes after which 10.000  cells werde measured on the FAC.
Making BBa_K081005/mTaz, BBa_K608006/mTaz and BBa_K608007/mTaz competent cells
JW 3367-3/pSB1A3/BBa_K081005/mTaz (colony 68)
NEB competent cells/pSB1A3/BBa_K608007/mTaz (colony 55)
NEB competent cells/pSB1A3/BBa_K081005/mTaz (colony 37)
NEB competent cells/pSB1A3/BBa_K608007/mTaz (colony 31)
This samples were incolated overnight in LB/Amp-medium. Cryostocks were produced, following the respective protocol and the cells were incolated in 50 mL LB/Amp-medium flasks until OD600=0.27. The cultered were chilled on ice for 5 min and collected after centrifugation at 5000 rpm, 10 min, 4 °C.
The cells were gently resuspended in 4 mL cold Ca/Glycerol Buffer, as desribed in the Zang Gong Competent Cell Protocol , and chilled on ice for 20 minutes.
Afterwards they were centrifuged at 4500 rpm, 4 °C, 10 min.
The add Ca/Glycerol Buffer was discarded and new 4 mL Buffer were added. The cells were then incubated for 60 minutes and the centrifuged at 5000 rpm, 4 °C, 10 min. The Buffer was discarded and 600 μL Ca/Glycerol Buffer was added per 50 mL falcon.
The competent cells were stored in 150 μL aliquots.
Transformation
according to Zang Gong Competent Cell Protocol
cell amount [μL]
DNA
DNA amount[μL;ng]
NEB competent cells/pSB1A3/BBa_K608007/mTaz (colony 31)
50.0
pSB1C3/BBa_R0082/BBa_E0240
0.58;50
NEB competent cells/pSB1A3/BBa_K081005/mTaz (colony 37)
50.0
pSB1C3/BBa_R0082/BBa_E0240
0.58;50
NEB competent cells/pSB1A3/BBa_K608007/mTaz (colony 55)
50.0
pSB1C3/BBa_R0082/BBa_E0240
0.58;50
JW 3367-3/pSB1A3/BBa_K081005/mTaz (colony 68)
100.0
pSB1C3/BBa_R0082/BBa_E0240
2.32;200
NEB competent cells/pSB1A3/BBa_K608007/mTaz (colony 31)
50.0
-
-
NEB competent cells/pSB1A3/BBa_K081005/mTaz (colony 37)
50.0
-
-
JW 3367-3
50.0
pSB1A3/BBa_K608007/mTaz
pSB1C3/BBa_R0082/BBa_E0240
0.79; 50
0.58; 50
JW 3367-3
50.0
pSB1A3/BBa_K081005/mTaz
pSB1C3/BBa_R0082/BBa_E0240
0.59; 50
0.58; 50
JW 3367-3
50.0
pSB1A3/BBa_K081005/BBa_C0082
pSB1C3/BBa_R0082/BBa_E0240
0.70; 50
0.58; 50
JW 3367-3
50.0
pSB1A3/BBa_K608006/BBa_C0082
pSB1C3/BBa_R0082/BBa_E0240
0.39; 50
0.58; 50
JW 3367-3
50.0
-
-
JW 3367-3 (negativ)
50.0
pSB1A3/BBa_K608006/BBa_C0082
0.39; 50
JW 3367-3 (negativ)
50.0
pSB1C3/BBa_R0082/BBa_E0240
0.58; 50
JW 3367-3 (positiv)
50.0
pSB1A3/BBa_K608006/BBa_C0082
0.39; 50
JW 3367-3 (positiv)
50.0
pSB1C3/BBa_R0082/BBa_E0240
0.58; 50
The CAmp/Amp plates are preparated with 20 μL (40 %) Amp, the same for the Amp-plates for positiv and negativ control.
Plates incubated at 37 °C
Colony PCR I
PCR reaction Master Mix (20x)
amount [μL]
10x DreamTaq Buffer
40.0
dNTPs, 10 mM
8.0
VF2 Primer, 10 μM
20.0
VR Primer, 10 μM
20.0
DreamTaq
2.0
H2O
310.0
400.0
Colony
cell with plasmids
1
NEB Competent Cells with pSB1A3/BBa_K081005/mTaz and pSB1C3/BBa_R0082/BBa_E0240
2
NEB Competent Cells with pSB1A3/BBa_K081005/mTaz and pSB1C3/BBa_R0082/BBa_E0240
3
NEB Competent Cells with pSB1A3/BBa_K081005/mTaz and pSB1C3/BBa_R0082/BBa_E0240
4
NEB Competent Cells with pSB1A3/BBa_K081005/mTaz and pSB1C3/BBa_R0082/BBa_E0240
5
JW 3367-3 with pSB1A3/BBa_K081005/mTaz and pSB1C3/BBa_R0082/BBa_E0240
6
JW 3367-3 with pSB1A3/BBa_K081005/mTaz and pSB1C3/BBa_R0082/BBa_E0240
7
JW 3367-3 with pSB1A3/BBa_K081005/mTaz and pSB1C3/BBa_R0082/BBa_E0240
8
JW 3367-3 with pSB1A3/BBa_K081005/mTaz and pSB1C3/BBa_R0082/BBa_E0240
9
JW 3367-3 with pSB1A3/BBa_K081005/mTaz and pSB1C3/BBa_R0082/BBa_E0240
10
JW 3367-3 with pSB1A3/BBa_K608006/BBa_C0082 and pSB1C3/BBa_R0082/BBa_E0240
11
JW 3367-3 with pSB1A3/BBa_K608006/BBa_C0082 and pSB1C3/BBa_R0082/BBa_E0240
12
JW 3367-3 with pSB1A3/BBa_K608006/BBa_C0082 and pSB1C3/BBa_R0082/BBa_E0240
13
JW 3367-3 with pSB1A3/BBa_K608006/BBa_C0082 and pSB1C3/BBa_R0082/BBa_E0240
14
JW 3367-3 with pSB1A3/BBa_K608006/BBa_C0082 and pSB1C3/BBa_R0082/BBa_E0240
15
JW 3367-3 with pSB1A3/BBa_K608006/BBa_C0082
Colonies were picked and resuspended in 4 μL H2O
3 μL H2O were transformed into 5 mL LB/CAmp/Amp-medium, the rest was used as template for the PCR.
Program: TAQSCREEN
result: all samples are positiv!
Colony PCR II
PCR reaction Master Mix (20x)
amount [μL]
10x DreamTaq Buffer
40.0
dNTPs, 10 mM
8.0
VF2 Primer, 10 μM
20.0
VR Primer, 10 μM
20.0
DreamTaq
2.0
H2O
310.0
400.0
Colony
cell with plasmids
1
NEB Competent Cells with pSB1A3/BBa_K608007/mTaz and pSB1C3/BBa_R0082/BBa_E0240
2
NEB Competent Cells with pSB1A3/BBa_K608007/mTaz and pSB1C3/BBa_R0082/BBa_E0240
3
NEB Competent Cells with pSB1A3/BBa_K608007/mTaz and pSB1C3/BBa_R0082/BBa_E0240
4
NEB Competent Cells with pSB1A3/BBa_K608007/mTaz and pSB1C3/BBa_R0082/BBa_E0240
5
JW 3367-3 with pSB1A3/BBa_K608007/mTaz and pSB1C3/BBa_R0082/BBa_E0240
6
JW 3367-3 with pSB1A3/BBa_K081005/mTaz and pSB1C3/BBa_R0082/BBa_E0240
7
JW 3367-3 with pSB1A3/BBa_K081005/mTaz and pSB1C3/BBa_R0082/BBa_E0240
8
JW 3367-3 with pSB1A3/BBa_K081005/mTaz and pSB1C3/BBa_R0082/BBa_E0240
9
JW 3367-3 with pSB1A3/BBa_K081005/mTaz and pSB1C3/BBa_R0082/BBa_E0240
10
JW 3367-3 with pSB1A3/BBa_K081005/mTaz and pSB1C3/BBa_R0082/BBa_E0240
11
JW 3367-3 with pSB1A3/BBa_K081005/BBa_C0082 and pSB1C3/BBa_R0082/BBa_E0240
12
JW 3367-3 with pSB1A3/BBa_K081005/BBa_C0082 and pSB1C3/BBa_R0082/BBa_E0240
13
JW 3367-3 with pSB1A3/BBa_K081005/BBa_C0082 and pSB1C3/BBa_R0082/BBa_E0240
14
JW 3367-3 with pSB1A3/BBa_K081005/BBa_C0082 and pSB1C3/BBa_R0082/BBa_E0240
15
JW 3367-3 with pSB1A3/BBa_K081005/BBa_C0082 and pSB1C3/BBa_R0082/BBa_E0240
Colonies were picked and resuspended in 4 μL H2O
3 μL H2O were transformed into 5 mL LB/CAmp/Amp-medium, the rest was used as template for the PCR.
Program: TAQSCREEN
Inoculate for FACS
2x 50 μL JW 3367-3 with pSB1A3/BBa_K081005/mTaz and pSB1C3/BBa_R0082/BBa_E0240
2x 50 μL JW 3367-3 with pSB1A3/BBa_K608006/BBa_C0082 and pSB1C3/BBa_R0082/BBa_E0240
2x 50 μL JW 3367-3 with pSB1A3/BBa_K608007/BBa_C0082 and pSB1C3/BBa_R0082/BBa_E0240
negative controls
20x 50 μL in 45 mL JW 3367-3/pSB1C3/BBa_R0082/BBa_E0240
20x 500 ΜL in 45 mL JW 3367-3/pSB1A3/BBa_K081005/mTaz
Preparation of DAP solutions
DAP solution was prepared in a 1 g/ 100 mL concentration on a 40 mL volumina with autoclaved water.
FACs
Since ONCs failed to grow (fast enough) the following cultures were prepared for FACs measurement.
NEB Competent Cells with pSB1A3/BBa_K081005/mTaz and pSB1C3/BBa_R0082/BBa_E0240
JW 3367-3 with pSB1C3/BBa_K608007/mTaz and pSB1C3/BBa_R0082/BBa_E0240
JW 3367-3 with pSB1C3/BBa_K608007/mTaz and pSB1C3/BBa_R0082/BBa_E0240
JW 3367-3 with pSB1C3/BBa_K608006/mTaz and pSB1C3/BBa_R0082/BBa_E0240
Preparation included centrifugation, resuspended in new LB-medium, splitting the culture into nine samples and treating them with:
0; 0.1; 1.0; 5.0; 10.0 [mmol/L] of DAP and Aspartate.
At timepoints of 0, 30, 90 minutes 1 mL culture was centrifuged at 3000 rpm for 15 minutes to get cell pellett. The supernatant was discarted and the cells werde resuspended in 1 mL PBS.
The cells were centrifuged at 13400 rpm for 1 minuts.
The supernatant were discarded and the cells resuspended in 1 mL PBS for FACs analysis.
Preparations of FACs Samples
JW 3367-3 cells
DH5α cells
JW 3367-3/pSB1C3/BBa_R0082/BBa_E0240
DH5α/pSB1C3/BBa_R0082/BBa_E0240
Cells were splitted in nine  5 mL cultures after centrifugation and resuspention in 45 mL LB-medium.
Cell cultures 3 and 4 were treated with CAmp.
At timepoints of 0, 30, 90 minutes 1 mL culture was centrifuged at 3000 rpm for 15 minutes to get cell pellett. The supernatant was discarted and the cells werde resuspended in 1 mL PBS.
The cells were centrifuged at 13400 rpm for 1 minuts.
The supernatant were discarded and the cells resuspended in 1 mL PBS for FACs analysis.
Get constructs from pSB1K3 into pSB1C3
mTaz and BBa_C0082 with Anderson Promoters [BBa_K081005, BBa_K608006, BBa_K608007] (respectivly)
Restriction
5 μL pSB1C3 (50 ng/μL)
3.4 μL pSB1A3/BBa_K608007/mTaz
2.3 μL pSB1A3/BBa_K081005/mTaz
2.8 μL pSB1A3/BBa_K081007/BBa_C0082
2.1 μL pSB1A3/BBa_K608006/BBa_C0082
Add 0.5 μL PstI and EcoRI and 2 μL 2.1NEB Buffer for each reaction, add H2O to 20 μL
Incubate for 20 min (37 °C); heat inactivate for 20 min (80 °C)
Dephosphorylation of Vector
Add 0.5 μL FastAP and 2 μL AP Buffer
Incubate for 10 min (37 μC), heat inactivation 5 min (75 °C)
Ligation
Vektor and Insert (1:5)
4.5 μL pSB1C3 (50 ng)
20.0 μL of each BBa_C0082 and mTaz Restriction sample
1.0 μL T4 Ligase
3.0 μL T4 Buffer
1.5 μL H2O
Incubation for 1 h (22 °C), heat inactivation for 10 min (65 °C)
Transformation
Transformation following the Zhang Gongs Transformation protocol
25.0 μL of each ligation product.
Colony PCR and Over Night Cultures
Master Mix
10x DreamTaq Buffer
35.0 μL
dNTPs, 10 mM
7.0 μL
VF2 Primer, 10 μM
5.0 μL
VR Primer, 10 μM
5.0 μL
DreamTaq
2.0 μL
H2O
331.0 μL
to 350 μL
Eight colonies of each construct were picked.
positiv bands
pSB1C3/BBa_K081005/mTaz colonies 1, 2, 4, 6 and 7
pSB1C3/BBa_K608006/BBa_C0082 colonies 10, 11, 12, 13, 14, 15 and 16
pSB1C3/BBa_K081005/BBa_C0082 colonies 17, 18, 19, 20, 21, 22 and 24
pSB1C3/BBa_K608007/mTaz colonies 25, 28, 29, 30, 31 and 32
first three of each sample there minipreped, following the respective protocol
of each cryostock were generated, following the respective protocol
Fabrication of ED02 (SU-8) Master01
4-inch Si-wafer was dehydrated for 20 minutes at 130 °C in oven
Base / adhesion layer of SU-8 (GM1060):
-Spin coating:
1) 0 - 4000 rpm (100 rpm/second acceleration) in 40 s
2) 4000 rpm for 40 seconds
3) 4000 - 0 rpm (-100 rpm/second acceleration) in 40 seconds
-30 minutes relaxation time at RT
-Soft bake:
1) 37 °C - 65 °C in about 5 minutes (avg +6 °C/minute)
2) 65 °C for 5 minutes
3) 65 °C - 95 °C in about 5 minutes (avg +5 °C/minute)
4) 95 °C for 5 minutes
5) 95 °C - 49 °C in about 13 minutes (avg -3.5 °C/minute)
i-line exposure (13 mW/cm2) for 30 seconds without mask (complete exposure)
-10 minutes delay time before PEB
-PEB:
1) 36 °C - 65 °C in about 5 minutes (avg +6 °C/minute)
2) 65 °C for 5 minutes
3) 65 °C - 95 °C in about 5 minutes (avg +6 °C/minute)
4) 95 °C for 15 minutes
5) 95 °C - 45 °C in about 15 minutes (avg -3.3 °C/minute)
Structural layer of SU-8 (GM1075):
-Spin coating:
1) 0 - 1000 rpm (100 rpm/second acceleration) in 10 s
2) 1000 rpm for 100 seconds
3) 1000 - 1400 rpm (10000 rpm/second acceleration) in 1 s
4) 1400 - 0 rpm (-100 rpm/second acceleration) in 14 seconds
-30 minutes relaxation time on hotplate at 40 °C; covered with glass dish
-Soft bake:
1) 40 - 120 °C in about 9 minutes (avg +9 °C/minute)
2) 120 °C for 5 minutes
3) 120 °C - 37 °C in about 25 minutes (avg -3.3 °C/minute)
-i-line exposure (13 mW/cm2 at 365 nm) for 30 seconds (corresponding 390 mJ/cm2 instead of recommended 350 mJ/cm2) using the ED02-Mask01
-1 hour delay time at RT in wafer box before PEB
-PEB:
1) 28 °C- 95 °C in about 7 minutes (avg 9.5 °C/minute);
2) 95 °C for 30 minutes
3) 95 °C - 41 °C in about 16 minutes (avg -2.4 °C/minute)
-Develop:
-In PGMEA (new) for about 8 minutes while constantly agitating the PGMEA and the wafer in the beaker
-Wafer was rinsed with isopropanol to stop the development which caused white traces on wafer
-Isopropanol-rinsed wafer was washed the with new, used PGMEA and subsequently rinsed with isopropanol which caused the previous white traces to vanish, however, lines of residue from isopropanol evaporation remained
-Examination of the profile (using the DektakXT; Bruker) showed a thickness of approx. 110 μm; comparison between the depth of the large filter structure holes and the fine filter structure holes (as visible in the live video in the DektakXT using a ruler) showed similar depth (7 mm at 100x magnification) and can be checked in the captured images
Sponsored by:
