Team:Hamburg/Experiments
Home
Team
Collaborations
Project
Description Experiments Proof of Concept Demonstrate Notebook
Parts
Basic Part Composite Parts Part Collection
Attributions
Human Practices
Silver Gold Integrated Practices
Safety
Awards
Alginate Encapsulation
To encapsulate our diagnostic bacteria we utilize a microfluidic encapsulation chip. The first task performed by the encapsulation chip is the pinch-off of the solution of alginic acid and bacteria by an oil, the shear fluid, creating an emulsion. In the second and last step the polymerization initiating CaCl2 solution is injected into the pinched-off droplets. By the time the droplets exit the chip they are polymerized.  The size of the produced alginate encapsulated bacteria droplets is tuneable by using different flow rates of sunflower oil (shear fluid). The following videos show the dependence of the droplet size from the shear flow rates. To precisely test the influence of the flow rate on the droplet size we have used a syringe pump system. To produce big droplets, we have used a shear fluid flow rate of 15.000 µL/h and an alginate bacteria solution flow rate of 4.000 µL/h. Smaller droplets can be produced with a shear fluid flow rate of 8.000 µL/h and an alginate bacteria solution flow rate of 4.000 µL/h.
Protocols
Biolab Protocols
Agarose gel
AG Ignatova
0.5 g Agarose
50 mL TAE-buffer (gel room)
Heat in microwave until agarose is dissolved
Transfer into gel slide, tip pipet into EtBr, then into gel
Cool for 30 minutes
Backbone dephosphorylation
AG Ignatova
10 μL
pSB1C3 (cut)
1 μL
FastAP
4 μL
FastAP Buffer
→ 37 °C for 30 minutes
→ purify with PCR clean up kit
Chloramphenicol Stock
AG Ignatova
34 mg/mL preparation of Choramphenicol Stock (3 in EtOH)
dissolve 34 mg of Chloramphenicol in 1 mL 100% ethanol
filter through a 0.22 μL filter to sterilize
use at 1:1000 dilution in LB or LB-agar
mark as CAmp (top) and CAmp/dd.mm.yy/34 mg/mL (side) in Antibiotics
store at -20 °C
Competent Cells
Zhang Gong
Buffers and solutions:
Ca/glycerol buffer: 60mM CaCl2, 10mM PIPES, 150mL glycerol, Fill water up to 1L, pH=7.0
Filter sterilization. Avoid autoclaving.
Protocol:
Grow the Cells at 37°C till OD=0.2-0.4
Chill the culture on ice for 5min.
Cllect cells by centrifugation, 6000rpm 10min 4°C
Gently resuspend the cells from 500ml LB in 40ml cold Ca/glycerol buffer on ice.
Incubate the cells on ice for 5~30min.
Repeat step 3 and 4.
Incubate the resuspended cells in Ca/glycerol buffer on ice for 30min. (The longer, the better)
Collect cells by centrifugation, 6000rpm 10min 4°C
Resuspend cells in 6ml Ca/glycerol buffer.
Aliquote 150µl/tube. Freeze in liquid N2 and store at -80°C.
GeneJET PCR Purification Kit
#K0701, #K0702
Thermo SCIENTIFIC Product Information
PURIFICATION PROTOCOLS
Note
Read IMPORTANT NOTES on p.3 before starting.
All purification steps ahould be carried out at room temperature.
All centrifugations should be carried out in a table-top microcentrifuge at >12000 x g (10000-14000 rpm, depending on the rotor type).
Protocol A. DNA purification using centrifuge
Step
Procedure
1
Add a 1:1 volume of Binding Buffer to completed PCR mixture (e.g. for every 100 µL of Binding Buffer). Mix thoroughly. Check the color of the solution. A yellow color indicates an optimal pH for DNA binding. If the color of the solution is orange or violet, add 10 µL of 3 M sodium acetate, pH 5.2 solution and mix. The color of the mix will become yellow.
2
for DNA ≥500 bp
Optional: if the DNA fragment is ≥ 500 bp, add a 1:2 volume of 100% isopropanol (e.g., 100 µL of isopropanol should be added to 100 µL of PCR mixture combined with 100 µL of Binding Buffer). Mix thoroughly.
Note. If PCR mixture contains primer-dimers, purification without isopropanol is recommended. However, the yield of the target DNA fragment will be lower.
3
Transfer up to 800 µL of the solution from step 1 (or optional step 2) to the GeneJET purification column. Centrifuge for 30-60 s. Discard the flow-through.
Note. If the total volume exceeds 800 µL, the solution can be added to the column in stages. After the addition of 800 µL of solution, centrifuge the column for 30-60 s and discard flow-through. Repeat until the entire solution has been added to the column membrane.
Close the bag with GeneJET Purification Columns tightly after each use!
4
Add a 700 µL of Wash Buffer (diluted with the ethanol as described on p.3) to the GeneJET purification column. Centrifuge for 30-60 s.
Discard the flow-through and place the purification column back into the collection tube.
5
Centrifuge the empty GeneJET purification column for an additional 1 min to completely remove any residual wash buffer.
Note. This step is essential as the presence of residual ethanol in the DNA sample may inhibit subsequent reactions.
6
Transfer the GeneJET purification column to a clean 1.5 mL microcentrifuge tube (not included).
Add 50 µL of Elution Buffer to the center of the GeneJET purification column membrane and centrifuge for 1 min.
Note.
For low DNA amounts the elution volumes can be reduced to increase DNA concentration. An elution volume between 20-50 µL does not significantly reduce the DNA yield. However, elution volumes less than 10 µL are not recommended.
If DNA fragment is > 10 kb, prewarm Elution Buffer to 65 °C before applying to column.
If the elution volume is 10 µL and DNA amount is ≥5 µg, incubate column for 1 min at room temperature before centrifugation.
7
Discard the GeneJET purification column and store the purified DNA at -20 °C.
GeneJET Plasmid Miniprep Kit
#K0502, #K0503
Thermo SCIENTIFIC Product Information
Growth of Bacterial Cultures
Pick a single colony from a freshly streaked selective plate to inoculate 1-5 mL of LB medium supplemented with the appropriate selection antibiotic. Incubate for 12-16 hours at 37 °C while shaking at 200-250 rpm. Use a tube or flask with a volume of least 4 times the culture volume.
Havest the bacterial culture by centrifugation at 8000 rpm (6800 x g) in a microcentrifuge for 2 min at room temperature. Decant the supernetant and remove all remaining medium.
Do not overload the column:
For high-copy-number plasmids (see Table 1), do not process more than 5 mL of bacterial culture. If more than 5 mL of such a culture are processed, the GeneJET spin column capacity (20 μL of dsDNA) will be exceeded and no increase in plasmid yield will be obtained.
For low-copy-number plasmids (see Table 1), it may be necessary to process larger volumes of bacterial culture (up to 10 mL) to recover a sufficient quantity of DNA.
Table 1. Copy numbers of various vectors
High-copy
300-700 copies per cell
Low-copy
10-50 copies per cell
Very low-copy
up to 5 copies per cell
pUC vectors
pBluescript vectors
pGEM vectors
pTZ vectors
pJET vectors
pBR322 and derivatives
pACYC and derivatives
pSC101 and derivatives
PURIFICATION PROTOCOLS
Note
Read IMPORTANT NOTES on p.3 before starting.
All purification steps should be carried out at room temperature.
All centrifugations should be carried out in a table-top microcentrifuge at >12000 x g (10000-14000 rpm, depending on the rotor type).
Use 1-5 mL of E. coli culture in LB media for purification of higt-copy plasmids.
For low-copy plasmids use up to 10 mL of culture.
Protocol A. Plasmid DNA purification using centrifuges
Step
Procedure
1
Resuspend the pelleted cells in 250 μL of the Resuspension Solution. Transfer the cell suspension to a microcentrifuge tube. The bacteria should be resuspended completely by vortexing or pipetting up and down until no cell clumps remain.
Note. Ensure RNase A has been added to the Resuspension Solution (as describe on p.3)
2
Add 250 μL of the Lysis Solution and mix thoroughly by inverting the tube 4-6 times until the solution becomes viscous and slightly clear.
Note. Do not vortex to avoid shearing of chromosomal DNA. Do not incubate for more than 5 min to avoid denaturation of supercoiled plasmid DNA.
3
Add 350 μL of the Neutralization Solution and mix immediately and thoroughly by inverting the tube 4-6 times.
Note. It is important to mix thoroughly and gently after the addition of the Neutralization Solution to avoid localized precipitation of bacterial cell debris.
The neutralized bacterial lysate should become cloudy.
4
Centrifuge for 5 min to pellet cell debris and chromosomal DNA.
5
Transfer the supernatant to the supplied GeneJET spin column by decanting or pipetting. Avoid disturbing or transferring the white prepipitate.
Note. Close the bag with GeneJET Spin Columns tightly after each use!
6
Centrifuge for 1 min. Discard the flow-through and place the column back into the same collection tube.
Note. Do not add bleach to the flow-through, see p.8 for Safety Information.
7
for EndA+ strains only
Optional: use this preliminary washing step only if EndA+ strains which have high level of nuclease activity are used.
Wash the GeneJET spin column by adding 500 μL of Wash Solution I (#R1611, diluted)
8
Add 500 μL of Wash Solution (diluted with ethanol prior to first use as described on p.3) to the GeneJET spin column. Centriifuge for 30-60 seconds and discard the flow-through. Place the column back into the same collection tube.
9
Repeat the wash procedure (step 8) using 500 μL of the Wash Solution.
10
Discard the flow-trough and centrifuge for an additional 1 min to remove residual Wash Solution. This step is essential to avoid residual ethanol in plasmid preps.
11
Transfer the GeneJET spin column into a fresh 1.5 mL microcentrifuge tube (not included). Add 50 μL of the Elution Buffer to the center of GeneJET spin column membrane to elute the plasmid DNA. Take care not to contact the membrane with the pipette tip. Incubate for 2 min at room temperature and centrifuge for 2 min.
Note. An additional elution step (optional) with Elution Buffer or water will recover residual DNA from the membrane and increase the overall yield by 10-20%
For elution of plasmids or cosmids >20 kb, prewarm Elution Buffer to 70 °C before applying to silica membrane.
12
Discard the column and store the purified plasmid DNA at -20 °C.
Glycerol stocks
AG Ignatova
from liquid culture
500 μL (50% Glycerol, 50% H2O)
500 μL Cells
Store at -80 °C
LB-Agar Plates (+ AB)
AG Ignatova
Heat 250 mL LB-Agar
Cool down to ca. 50°C
Add 250 μL Chloramphenicol/ 12.5 μL Kanamycin (50 μg/mL)/ 25μL Ampicillin (80 μg/mL)
Preparation of 20 plates under sterile conditions
Store at 4 °C, mark as LB CAmp/Kan/Amp Plates iGEM, dd.mm.yyyy
Ligation
AG Ignatova
1 μL
pSB-1C3 vector (23 ng)
1 μL/1.5 μL
GroEl-Insert (3.5 ng) / miRNA-Insert (3.3 ng)
2 μL
T4 Buffer
1 μL
T4 Ligase
5 μL
ddH20 (6 μL for control)
15 min at room temperature, then 10 min at 65 °C
Media
Tejas
Content
LB-Medium
M9Y Medium
M9 minimal medium
LB Medium
Yeast extrakt
5 g/L
Trypton
10 g/L
NaCl
10 g/L
RO-water add 1000 mL
pH 7.5
autoclave: 20 min 121 °C
M9Y Medium (with yeast extract)
Na2HPO4
6 g/L
KH2PO4
3 g/L
NaCl
0.5 g/L
NH4Cl
1 g/L
Yeast extract
2.5 g/L
RO-water ad 1000 mL
pH 7.4
autoclave: 20 min 121 °C
After autocalving add following substances:
1 M MgSO4
2 mL
20% Glucose
15 mL (end conc. 3 g/L)
0.1 M CaCl2
1 mL
Glucose and MgSO4 autoclave separately, CaCl2 to be steril filtered.
M9 minimal medium
M9 salts (part 1) 10x:
Na2HPO4
68 g
KH2PO4
30 g
NaCl
5 g (do not add for osmotic stress experiment)
NH4Cl
10 g
Add water to 1 L. Adjust pH=7.4. Autoclave
M9 (part 2) 10x:
MgSO4
10 mM
Glucose
30 g
Add water to 1 L. Filter sterilization.
CaCl2 100 mM stock solution. Filter sterilization
AA-Met mixture 250 mg/L each, pH=7.4. Filter sterilization
To use:
10x M9 salts (part 1)
1 mL
10x M9 (part 2)
1 mL
5x AA
2 mL for stress experiment don’t add!
CaCl2 100 mM
0.1 mL
Water
up to 10 mL
PCR protocols
Zhang Gong
Content
PCR protocols
Fragment amplification with Herculase
Fragment amplification with Pfu
mutagenesis
Fragment amplification with Herculase
Template
1 μL
Primers (100˜200 μM)
1 μL each
10x Herc buffer
5 μL
Water
39.5 μL
dNTPs (10 mM each)
1.5 μL
Herculase
0.5˜1 μL
50 μL
95 °C
2 min
95 °C
30 sec
10 cycles
55 °C
45˜60 sec
72 °C
1 min/kb
95 °C
30 sec
20 cycles
55 °C
45˜60 sec
72 °C
1 min/kb + 10 sec/cycle
4 °C
-
Fragment amplification with Pfu
Tamplate
1 μL
Primers (100˜200 μM)
1 μL each
10x Pfu buffer
5 μL
Water
39.5 μL
dNTPs (10 mM each)
1.5 μL
Pfu polymerase
1 μL
50 μL
95 °C
2 min
95 °C
30 sec
30 cycles
55 °C
45˜60 sec
72 °C
2 min/kb + 1 min
4 °C
-
Mutagenesis
Tamplate (1:5 dilution)
1 μL
Primers (10 μM)
1 μL each
10x Pfu buffer
5 μL
Water
39.5 μL
dNTPs (10 mM each)
1.5 μL
Pfu polymerase
1 μL
50 μL
95 °C
2 min
95 °C
30 sec
18 cycles
55 °C
45˜60 sec
68 °C
2 min/kb + 1 min
4 °C
-
DpnI 1 μl, 37 °C 1 h.  Then transform 5 μL into DH5α.
Note:
If you get too less colonies, you may think to increase the cycles to 20, but no more! Otherwise you might get mistakes in the final sequence.
You can try both Pfu and Herculase to perform muatgenesis. Pfu will give less yield but more fidelity, so you should preferably use Pfu. If you cannot succeed with Pfu then you can consider trying Herculase. Just keep the same protocol!
Pick colony
AG Ignatova
Inoculate each picked colony into 5 mL of liquid LB-(CAmp/ respective antibioticum)-medium, respectively
Tubes in the 37 °C incubation room, on rotator!
Preparation for sequencing
AG Ignatova
Primer concentration: 100 μM → Concentration needed: 5 μM (dilute at 1:20)
take 50 nm of sample mix with 5 μL of (5 μM primer, only forward)
Restriction
AG Ignatova
20 μL
pSB-1C3 (500 ng)
1 μL
EcoRI
1 μL
PstI
3 μL
FD Buffer
5 μL
ddH2O
20 minutes at 37 °C, following clean up with kit
SOC medium
Dagmar Stang
Bacto-Tryptone
2.000g 
Bacto-Hefe-Extract
0.500 g
NaCl
0.050 g
KCl
0.019 g
MgCl2 * (H2O)6
0.203 g
(without (H2O)6 → 0.095 g)
Glucose
0.036 g
pH 7.0 with pH-Meter
→ 10 mL (fridge)
TB PIPES Buffer
AG Ignatova
for 100 mL:
KCl
1.865 g
CaCl x 2 H2O
0.220 g
0.5 M PIPES
2 mL
pH 6.7
MgCl2 x 2 H20
0.889 g
ddH2O
→ 100 mL
Transformation Efficiency 'Competent Cell Test Kit'
iGEM HQ
http://parts.igem.org/Help:Transformation_Efficiency_Kit
spin down DNA (0.5, 5, 10, 20 & 50 pg/μL RFP Construct BBa_J04450, pSB1C3) from Competent Cell Test Kit (30 sec, 10.000 rpm)
thaw competent cells on ice, label one 2 mL tube for each concentration + pre-chill
1 μL of DNA in each tube, respectively
add 50 μL of competent cells to each tube, incubate for 30 min on ice
pre-heat (42 °C), heat shock for 1 min
incubate for 5 min on ice
add 200 μL SOC media, incubate for 2 h at 37 °C
pipet 20 μL on each plate (triplets), respectively
incubate overnight at 37 °C
count colonies, calculate transformation efficiency
Nanolab Protocols
Fabrication of Alginate Capsules
iGEM-Team TU Eindhoven
Buffers and solutions:
100mM CaCl2/BaCl2 solution, 0.9% NaCl solution, 2.5% alginic acid solution
Protocol:
Preparation of 0.9% NaCl Solution
Dissolve 0.9g NaCl in 100ml MiliQ water
Mix/vortex so that all NaCl goes into solution
Filter into a glass flask using a syringe with a 0.22µm filter for sterilization
Preparation of 2.5% Alginate Solution
Dissolve 2.5g alginic acid slowly in 100ml NaCl solution. During the entire dissolving process a mixing magnet should mixing the solution carefully
Autoclave the alginic acid solution at 121°C for 30min
Preparation of 100mM CaCl2/BaCl2 solution
Dissolve 1.1098g of CaCl2/BaCl2 in 100ml MiliQ water
Mix so that all CaCl2/BaCl2 goes into solution
Filter into a glass flask using a syringe with 0.22µm filter for sterilization
Production of Alginate Beads
Mix 1ml bacteria in 0.9% NaCl solution with 4ml alginic acid solution
Put suspension in a sterile syringe and use it to produce droplets in 100ml of the 100mM CaCl2/BaCl2 solution or use the microfluidic device
Put the alginate beads over a filter into a falcon tube for storage
Fabrication of a Chrome-Mask
xxxxxx
Protocol:
Design structure with AutoCAD and save it as a dxf file
Load the dxf file to the Heidelberg Laserwriter
Clean chrome-mask with acetone in ultrasonic bath
Spin-coat mask with photoresist S1813 at 4000rpm and 2000rpm acceleration for 1min
Pre-bake at 90°C for 2min
Adjust chrome-mask on the Laserwriter stage
Turn the vacuum on with the white valves
Convert the dxf file to a job data for the Laserwriter
Select a proper filter and laser head
Start exposure
Develop mask in MF319 for 1min
Chrome-etching for 1-3min
Control the etching process with an optical microscope
Fabrication of PDMS-Devices
Martin Trebbin
Put pour PDMS oligomer solution (SYLGARD 184) and 1/10 (of its weight) curing agent on the SU-8 master template inside an aluminum foil coated petri dish
Stir well, remove big bubbles with a pipette tip an then expose inside a desiccator for 1-2h
Cure in oven at 75°C for 2h
Cutting
Rough cut
Fine cut
Drill holes with a biops-cutter with a 0.75mm needle diameter
Clean PDMS pieces with isopropanol
Rinse isopropanol on the pieces, gently rub the surface with a finger
Dry with air jet
Leave the cleaned pieces in a fume hood to dry in ambient atmosphere
Activate PDMS surface with the low pressure plasma chamber (13.56MHz, 10Watt)
Power on
Ventilation of the chamber
Open hatch carefully and insert sample in the middle of the chamber
Turn ventilation off
Turn pump on
Open gas valve
Wait until pressure sets stable at 0.38mbar
Set power up to 10Watt and time to 100s
Press generate
Turn the pump off and the ventilation on
Open hatch and remove sample with gloves
Attach the bottom/up PDMS pieces onto another
Drop filtered Milipore water on the bottom
Put the upper part (with the holes) on the bottom part
Adjust the two pieces with the help of an optical microscope
Attach the PDMS to glass
Dry out oven at 40°C for 2h or overnight
Sponsored by:
