<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_Hamburg_Description skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><DIV id="top_title"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:Hamburg/Description</SPAN></H1></DIV><DIV id="HQ_page"><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><DIV class="content"><DIV id="main"><DIV class="navbar"><UL><LI class="navitem"><DIV class="dropdown"><A class="nav-link" href="https://2016.igem.org/Team:Hamburg" title="">Home</A></DIV></LI><LI class="navitem"><DIV class="dropdown"><A class="nav-link" href="https://2016.igem.org/Team:Hamburg/Team#main" title="">Team</A><DIV class="dropdown-content"><A class="dropdown-link" href="https://2016.igem.org/Team:Hamburg/Collaborations#main" title="">Collaborations</A></DIV></DIV></LI><LI class="navitem"><DIV class="dropdown"><A class="nav-link" href="https://2016.igem.org/Team:Hamburg/Description#main" title="">Project</A><DIV class="dropdown-content"><A class="dropdown-link" href="https://2016.igem.org/Team:Hamburg/Description#main" title="">Description</A><A class="dropdown-link" href="https://2016.igem.org/Team:Hamburg/Experiments#main" title="">Experiments</A><A class="dropdown-link" href="https://2016.igem.org/Team:Hamburg/Proof#main" title="">Proof of Concept</A><A class="dropdown-link" href="https://2016.igem.org/Team:Hamburg/Demonstrate#main" title="">Demonstrate</A><A class="dropdown-link" href="https://2016.igem.org/Team:Hamburg/Notebook#main" title="">Notebook</A></DIV></DIV></LI><LI class="navitem"><DIV class="dropdown"><A class="nav-link" href="https://2016.igem.org/Team:Hamburg/Parts#main" title="">Parts</A><DIV class="dropdown-content"><A class="dropdown-link" href="https://2016.igem.org/Team:Hamburg/Basic_Part#main" title="">Basic Part</A><A class="dropdown-link" href="https://2016.igem.org/Team:Hamburg/Composite_Part#main" title="">Composite Parts</A><A class="dropdown-link" href="https://2016.igem.org/Team:Hamburg/Part_Collection#main" title="">Part Collection</A></DIV></DIV></LI><LI class="navitem"><DIV class="dropdown"><A class="nav-link" href="https://2016.igem.org/Team:Hamburg/Attributions#main" title="">Attributions</A></DIV></LI><LI class="navitem"><DIV class="dropdown"><A class="nav-link" href="https://2016.igem.org/Team:Hamburg/HP/Gold#main" title="">Human Practices</A><DIV class="dropdown-content"><A class="dropdown-link" href="https://2016.igem.org/Team:Hamburg/HP/Silver#main" title="">Silver</A><A class="dropdown-link" href="https://2016.igem.org/Team:Hamburg/HP/Gold#main" title="">Gold</A><A class="dropdown-link" href="https://2016.igem.org/Team:Hamburg/Integrated_Practices#main" title="">Integrated Practices</A></DIV></DIV></LI><LI class="navitem"><DIV class="dropdown"><A class="nav-link" href="https://2016.igem.org/Team:Hamburg/Safety#main">Safety</A></DIV></LI><LI class="navitem"><DIV class="dropdown"><A class="nav-link" href="https://2016.igem.org/Team:Hamburg/Awards" title="">Awards</A></DIV></LI></UL></DIV><DIV id="main-content"><DIV class="container-content-text"><H1>
        100 Million
    </H1><P>
        People are infected with <I>Chlamydia trachomatis</I> every year which makes it the second most abundant sexually transmitted disease worldwide<A id="page-reference-1" href="#reference-1" class="reference">
            1<SPAN class="reference-tooltip">
                Global incidence and prevalence of selected curable sexually transmitted infections – 2008. <I>WHO, Department of Reproductive Health and Research </I>(2011).
            </SPAN></A>.
        Present all over the world, its prevalence ranges between 2.3% in the WHO western pacific region and 25.7% in Africa (Fig. 1, Table 1)<A id="page-reference-2" href="reference-1" class="reference">
            2<SPAN class="reference-tooltip">
                Global Prevalence and Incidence of Selected Curable Sexually Transmitted Infections – Overview and Estimates. <I>WHO</I> (2001).
            </SPAN></A>.
    </P><P class="img-content-description">
        Fig. 1 Prevalence of <I>Chlamydia trachomatis</I> infections in men and women aged 15-49 years by WHO region 1999.
    </P><H3>
        Table 1: Incidence and Prevalence of <I>Chlamydia trachomatis</I> infections in men and women aged 15-49 years by WHO region 1999.
    </H3><TABLE><THEAD><TR><TH>
                    Region
                </TH><TH>
                    Population (15-49 year old) (millions)
                </TH><TH>
                    Incidence (millions)
                </TH><TH>
                    Prevalence
                </TH></TR></THEAD><TBODY><TR><TD>
                    African Region
                </TD><TD>
                    269
                </TD><TD>
                    69
                </TD><TD>
                    25.7%
                </TD></TR><TR><TD>
                    Region of the Americas
                </TD><TD>
                    416
                </TD><TD>
                    52
                </TD><TD>
                    12.5%
                </TD></TR><TR><TD>
                    Eastern Mediterranean Region
                </TD><TD>
                    165
                </TD><TD>
                    10
                </TD><TD>
                    6.1%
                </TD></TR><TR><TD>
                    European Region
                </TD><TD>
                    408
                </TD><TD>
                    39
                </TD><TD>
                    9.6%
                </TD></TR><TR><TD>
                    South-East Asian Region
                </TD><TD>
                    995
                </TD><TD>
                    151
                </TD><TD>
                    12.5%
                </TD></TR><TR><TD>
                    Western Pacific Region
                </TD><TD>
                    826
                </TD><TD>
                    19
                </TD><TD>
                    2.3%
                </TD></TR><TR><TD>
                    total
                </TD><TD>
                    3079
                </TD><TD>
                    340
                </TD><TD>
                    11.0%
                </TD></TR></TBODY></TABLE><P>
        While an infection may be asymptomatic but contagious at first, when untreated severe and partly incurable symptoms arise<A id="page-reference-3" href="reference-3" class="reference">
            3<SPAN class="reference-tooltip">
                Mania-Pramanik, J., Kerkar. S., Sonawane, S., Mehta, P., Salvi, V. Current Chlamydia trachomatis In-fection, A Major Cause of Infertility. <I>J. Reprod. Infertil.</I><B>4</B>, 204-210 (2012).
            </SPAN></A>.
        Symptoms range from urinary urgency and painful urinating to pelvic inflammatory disease, including cervicitis and sapingitis in females, ultimately result-ing in infertility.
        Infected males suffer from epididymitis and prostatic inflammation. Urethritis, reactive arthritis and trachoma are found in both genders<A id="page-reference-4" href="reference-4" class="reference">
            4<SPAN class="reference-tooltip">
                Da Ros, C., da Silva Schmitt, C. Global epidemiology of sexually transmitted diseases. <I>Asian. J. Androl.</I><B>10</B>, 110-114 (2008).
            </SPAN></A><A id="page-reference-5" href="reference-5" class="reference">
            5<SPAN class="reference-tooltip">
                Wagenlehner, FM., Naber, KG., Weidner, W. Chlamydial infections and prostatitis in men. <I>BJU Int.</I><I>98</I>, 687-90 (2006).
            </SPAN></A><A id="page-reference-6" href="reference-6" class="reference">
            6<SPAN class="reference-tooltip">
                Siala, M. et al. Analysis of bacterial DNA in synovial tissue of Tunisian patients with reactive and un-differentiated arthritis by broad-range PCR, cloning and sequencing. <I>Arthritis Research &amp; Therapy</I><I>10</I>, R40 (2008).
            </SPAN></A>.
        Trachoma, if untreated, leads to blindness.
    </P><TABLE id="table-symptoms"><TBODY><TR><TD><P><I>C. trachomatis</I> causes trachoma, an infection of the eye causing visual impairment and blindness if not treated correctly. Trachoma is responsible for the visual impairment of 2.2 million people worldwide, of whom 1.2 million are blind<A id="page-reference-7" href="reference-7" class="reference">
                        7<SPAN class="reference-tooltip">
                            Pascolini, D., Mariotti S. P. Global estimates of visual impairment: 2010. <I>Br. J. Ophthalmol.</I><B>96</B>, 614-618 (2012).
                        </SPAN></A>.
                </P></TD></TR><TR><TD><P>
                    Transmitted sexually, <I>C. trachomatis</I> causes pelvic inflammation and urethritis, which leads to infertility in women<A href="reference-1" class="reference">
                        2<SPAN class="reference-tooltip">
                            Global Prevalence and Incidence of Selected Curable Sexually Transmitted Infections – Overview and Estimates. <I>WHO</I> (2001).
                        </SPAN></A>.
                </P></TD></TR><TR><TD><P>
                    Chlamydia may be transmitted to infants during birth. It is estimated that up to 4000 new-born babies become blind due to infection upon birth every year<A id="page-reference-8" href="reference-8" class="reference">
                        8<SPAN class="reference-tooltip">
                            Global strategy for the prevention and control of sexually transmitted infections: 2006-2015. Geneva, <I>World Health Organization</I> (2007).
                        </SPAN></A>.
                </P></TD></TR></TBODY></TABLE><H2>Simple Treatment - Difficult Diagnosis</H2><P><I>C. trachomatis</I> are bacteria that are sensitive to Azithromycine, a broad-band, low-cost antibi-otic drug<A id="page-reference-9" href="#reference-9" class="reference">
            9<SPAN class="reference-tooltip">
                Geisler, W. M. et al. Azithromycin versus Doxycycline for Urogenital Chlamydia trachomatis Infec-tion. <I>N. Engl. J. Med.</I><B>373</B>, 2512-2521 (2015).
            </SPAN></A>.
        While treating people affected with Chlamydia is easy, the much more difficult task lies in recogniz-ing the infection in the first place.
        Usual diagnostic tests for C.
        trachomatis include PCR, Anti-body-based tests and cell culture with costs ranging between $ 20 and $ 140 in industrial coun-tries.
        As part of our Human Practices we visited <I>Blantyre Institute for Community Ophthalmology</I> (BICO) in Malawi,
        a domestic country in central Africa. Founded by Khumbo Kalua in 2008, BICO plays an important role in monitoring and fighting the <I>C. trachomatis</I> endemic within the rural parts of Malawi.
        In an interview with Khumbo Kalua we learned that due to the lack of an inexpensive diagnostic method they need to treat people without testing them,
        having to accept the side-effects of an unnecessary treatment.
    </P><P class="quote">
        “We have an ointment which we give to babies. It costs about $ 0.30 and the PCR costs $ 1 . So for every PCR that you do you can actually buy treatment for three children.”
    </P><P class="quote-reference">- Khumbo Kalua, Head of <I>Blantyre Institute for Community Ophthalmology</I></P><P>
        In addition to the economical aspect there is the impossibility to test everyone who would need a test. In field missions, scientists and medical staff at BICO collect thousands of samples, store them frozen but they don not have the capacities to test them in time for a selective treatment.
    </P><P class="quote">
        “We can’t let the patients wait, so we treat the patients and then a year later we find out it was not chlamydia.”
    </P><P class="quote-reference">- Khumbo Kalua</P><P>
        In our opinion it is inacceptable that a disease so simple to treat creates such an immense burden for so many people. To solve this issue, our goal is to develop a simple, fast and economic diagnostic method to detect <I>Chlamydia trachomatis</I>.
    </P><H2>
        A diagnostic system that produces itself
    </H2><P>
        To create a diagnostic system that is simple and economic to produce, costs have to be kept as low as possible. Using genetically modified bacteria that are able to sense the presence of chlamydia would be an optimal solution due to the low maintenance costs of bacteria and the fact that after initial assembly no further manufactural production of the diagnostic system is needed.
    </P><H2>
        What makes them special
    </H2><P>
        Chlamydia differ from almost all other bacteria by the composition of their peptidoglycane. While almost all gram-negative bacteria produce and incorporate <I>meso</I>-2,6-Diaminopimelic acid (mDAP), a non-proteinogenic amino acid that acts as a linker between peptidoglycane layers, <I>C. trachomatis</I> misses the gene for peptidoglycane incorporation (murE). Therefore it is secreted by Chlamydia trachomatis<A id="page-reference-10" href="reference-10" class="reference">
            10<SPAN class="reference-tooltip">
                Henrichfreise, B. <I>et al.</I> Functional conservation of the lipid II biosynthesis pathway in the cell wall-less bacteria Chlamydia and Wolbachia: why is lipid II needed? <I>Mol. Microbiol.</I><B>73</B>, 913-923 (2009).
            </SPAN></A>.
        Our aim was to create a receptor capable of sensing mDAP in the medium as a proof of concept of the possibility of bacteria sensing specific pathogens in a patient sample.
    </P><H2>mTaz – An EnvZ Fusion Protein designed to detect mDAP secreted by <I>Chlamydia trachomatis</I>.</H2><P>
        To detect <I>Chlamydia trachomatis</I> we designed and assembled a receptor based on the Tar-EnvZ chimeric receptor capable of
        detecting aspartate in the medium as described by Utsumi <I>et al.</I>
        1989<A class="reference" id="page-reference-11" href="#reference-11">
            11<SPAN class="reference-tooltip">
                Utsumi, R. <I>et al.</I> Activation of Bacterial Porin Gene Expression by a Chimeric Signal Transducer in Response to Aspartate. <I>Science</I><B>245</B>, 1246-1249 (1989).
            </SPAN></A>. Tar, or Methyl-accepting chemotaxis protein II, is a signal
        transducer in <I>E. coli</I> chemotaxis, being activated upon the recognition of
        aspartate<A class="reference" id="page-reference-12" href="#reference-12">
            12<SPAN class="reference-tooltip">
                Reader, R. W., Tso, W. W., Springer, M. S., Goy, M. F., Adler, J. Pleiotropic aspartate taxis and serine taxis mu-tants of Escherichia coli. <I>J. Gen. Microbiol</I><B>111</B>, 363-374 (1979).
            </SPAN></A>. It has been shown that Tar is sensible to mutations
        altering its ligand specificity, sometimes resulting in strong activity upon recognition of other amino
        acids<A class="reference" id="page-reference-13" href="#reference-13">
            13<SPAN class="reference-tooltip">
                Derr, P., Boder, E., Goulian, M. Changing the Specificity of a Bacterial Chemoreceptor. <I>. Mol. Biol.</I>J <B>355</B>, 923-932 (2006).
            </SPAN></A>. EnvZ is part of the well-studied and commonly
        used two-component system EnvZ/OmpR in <I>E coli</I>, offering a simple way of signal
        transduction<A class="reference" id="page-reference-14" href="#reference-14">
            14<SPAN class="reference-tooltip">
                Lan, C. Y., Igo, M. M. Differential expression of the OmpF and OmpC porin proteins in Escherichia coli K-12 depends upon the level of active OmpR. <I>J. Bacteriol.</I><B>180</B>, 171.174 (1998).
            </SPAN></A><A class="reference" id="page-reference-15" href="#reference-15">
            15<SPAN class="reference-tooltip">
                Mizuno, T., Mizushima, S. Signal transduction and gene regulation through the phosphorylation of two regu-latory components: The molecular basis for the osmotic regulation of the porin genes. <I>Mol. Microbiol.</I><B>4</B>, 1077-1082 (1990).
            </SPAN></A><A class="reference" id="page-reference-16" href="#reference-16">
            16<SPAN class="reference-tooltip">
                Russo, F. D., Silhavy, T.J. EnvZ controls the concentration of phosphorylated OmpR to mediate osmoregula-tion of the porin genes. <I>J. Mol. Biol.</I><B>222</B>, 567-580 (1991).
            </SPAN></A><A class="reference" id="page-reference-17" href="#reference-17">
            17<SPAN class="reference-tooltip">
                Slauch, J. M., et al. EnvZ functions through OmpR to control porin gene expression in Escherichia coli K-12. <I>J. Bacteriol.</I><B>170</B>, 439-441 (1988).
            </SPAN></A>.
        Its periplasmic domain was shown to be exchangeable with different sensor domains as in the Tar/EnvZ chimeric receptor capable of detecting
        aspartate in the medium<A class="reference" id="page-reference-18" href="#reference-18">
            18<SPAN class="reference-tooltip">
                Yoshida, T., Phadtare, S., Inouye, M. The design and development of Tar-EnvZ chimeric receptors. <I>Methods Enzymol.</I><B>423</B>, 166-183 (2007).
            </SPAN></A>.
    </P><H3>
        Two fairly alike ligands
    </H3><P><I>meso</I>-2,6-Diaminopimelic acid (mDAP) is a non-proteinogenic amino acid that is synthesized and secreted by <I>Chlamydia trachomatis</I><A class="reference" id="page-reference-19" href="#reference-19">
            19<SPAN class="reference-tooltip">
                Henrichfreise, B. et al. Functional conservation of the lipid II biosynthesis pathway in the cell wall-less bacte-ria Chlamydia and Wolbachia: why is lipid II needed? <I>Mol. Microbiol.</I><B>73</B>, 913-923 (2009).
            </SPAN></A>. mDAP has a relatively high chemical similarity to aspartate, sharing a Tanimoto coefficient of T=0.8 in a molecular fingerprint analysis ]T &lt;0: chemically very different structures; T=1: identity] (Fig. 1). As a proof of concept, a molecular docking model proposes few mutations which would change the Tar specificity towards mDAP. As input for molecular docking we used the crystal structure of the ligand binding domain of Tar bound to aspartate (PDB <A href="http://www.rcsb.org/pdb/explore/explore.do?structureId=4Z9H">4Z9H</A>)<A class="reference" id="page-reference-20" href="#reference-20">
            20<SPAN class="reference-tooltip">
                Mise, T. Structural Analysis of the Ligand-Binding Domain of the Aspartate Receptor Tar from Escherichia coli. <I>Biochemistry</I><B>55</B>, 3708-3713 (2016).
            </SPAN></A>.

        </P><P class="img-content-description">
        Fig. 1 <B>Comparison of the structures of <I>meso</I>-2,6-Diaminopimelic acid (mDAP) and aspartate. A: </B>mDAP, <B>B:</B> aspartate. In molecular count fingerprint analysis, aspartate and mDAP share a Tanimoto coefficient of T=0.8.
    </P><H2>Design considerations</H2><H3>The Sequence</H3>
    As a base for the part sequence, the BioBrick representation of the original Taz, <A href="http://parts.igem.org/Part:BBa_C0082">BBa_C0082</A>, was used. The BBa_C0082 BioBrick features a small part of the vector (64 nucleotides) it was derived from on its 3’ end, which we have removed along with the stop codon inside the Part for further fusion protein design in RFC 12, 21, 23 and 25 assembly standards.

    <H3>Mutagenesis based on docking results</H3>
    We used the Glide<A class="reference" id="page-reference-21" href="#reference-21">
        21<SPAN class="reference-tooltip">
            Small-Molecule Drug Discovery Suite 2016-3: Glide, version 7.2, Schrödinger, LLC, New York, NY (2016).
        </SPAN></A> algorithm, Maestro and the Schrodinger Suite for molecular docking of aspartate and mDAP to the Tar ligand binding domain. Because of its structural similarity to aspartate, mDAP showed some affinity to the native Tar aspartate binding site (Table 1). Since mDAP is larger than aspartate, Y149 led to structural interference, not allowing mDAP to fully enter the binding site (Fig. 2). To create the additional space needed for mDAP binding, we substituted Y149 with smaller amino acids. The best results were obtained with serine. Because of the additional size of mDAP, the bigger challenge was to reduce the affinity of aspartate rather than to raise the affinity of mDAP. We identified R64 as a key residue for aspartate binding which hovewer was not needed to bind mDAP. To minimally interfere with folding and function performed a constitutive substitution and substituted it with lysine, creating the R64K Y149S double mutant (Fig 2 C), called mTaz hereafter. Using mTaz, a higher docking score was achieved for mDAP than for aspartate (Table 1).

    <H3>Table 1: Glide Docking Scores of mDAP and aspartate docked to Tar (4Z9H) and mTaz</H3><TABLE><TBODY><TR><TH rowspan="2">Receptor Model</TH><TH colspan="2">Glide Docking Score</TH></TR><TR><TD>mDAP</TD><TD>aspartate</TD></TR><TR><TD>4Z9H</TD><TD>-4.595</TD><TD>-7.413</TD></TR><TR><TD>mTaz</TD><TD>-6.576</TD><TD>-5.995</TD></TR></TBODY></TABLE><P class="img-content-description">
        Fig. 2: <B><I>meso</I>-Diaminopimelic Acid (mDAP) and Aspartate docked to the structure of the Tar ligand binding domain (PDB: 4Z9H) and mTaz. A:</B> Aspartate in its native conformation at the receptor site of Tar. <B>B:</B> mDAP docked to the receptor site of Tar. <B>C:</B> mDAP docked to mTaz. Y149 was changed to serine to make room for mDAP to enter the ligand binding site. R64 was changed to lysine to weaken the receptor's affinity to aspartate.
    </P><H2>
        Validation proves to be difficult
    </H2><P>
        To achieve a minimal amount of noise, we used the ΔEnvZ <I>E. coli</I> strain JW 3367-3. We created two fluorescence reporters responding to activation by OmpR, Parts <A href="http://parts.igem.org/Part:BBa_K1909013">BBa_K1909013</A> and <A href="http://parts.igem.org/Part:BBa_K1909014">BBa_K1909014</A>.
    </P><P>
        BBa_K1909013 is a composite part consisting of <A href="http://parts.igem.org/Part:BBa_R0082">BBa_R0082 (Promoter (OmpR, positive))</A>, <A href="http://parts.igem.org/Part:BBa_B0032">BBa_B0032</A> (medium rbs), <A href="http://parts.igem.org/Part:BBa_E0040">BBa_E0040</A> (GFP), <A href="http://parts.igem.org/Part:BBa_B0010">BBa_B0010</A> and <A href="http://parts.igem.org/Part:BBa_B0012">BBa_B0012</A>, two terminator sequences. It induces production of GFP upon activation by OmpR. OmpR is the transcription factor activated by EnvZ at high osmolarity levels. 
        In order to characterize this part, we transformed both wildtype and JW 3367-3 <I>E. coli</I> strains with pSB1C3/BBa_K1909013. To induce expression, cells were incubated in LB broth containing 1% NaCl at 37°C, 220 RPM until OD<SUB>600</SUB> reached 0.2. Cells were diluted to 1:100 and washed using 100 mM MgCl<SUB>2</SUB>. Expression level of GFP was analysed by flow cytometry. Untransformed cells of both strains were used as negative controls. 
        Negative controls showed only few to no fluorescence as expected (Fig. 3). GFP expression proved to be significantly higher in DH5α (green) cells compared to ΔEnvZ cells (pink). Differences in GFP expression was confirmed by KS tests (p &lt;= 0.001).
    </P><P class="img-content-description"><B>Fig. 2 FACS analysis of GFP expression in JW 3367-3 and DH5α <I>E. coli</I> strains. Grey:</B> DH5α and JW 3367-3 cells without reporter system. <B>Pink:</B> JW 3367-3/pSB1C3/BBa_K1909013, <B>Green:</B> DH5α/BBa_K19090013.
    </P><P>
        Part BBa_K1909014 was validated using the same experimental setup. Negative controls showed only few to no fluorescence as expected (Fig. 4). eYFP expression proved to be signifi-cantly higher in DH5α (green) cells compared to ΔEnvZ cells (pink). Differences in eYFP ex-pression was confirmed by KS tests (p &lt;= 0.001).
    </P><P class="img-content-description"><B>Fig. 3 FACS analysis of eYFP expression in JW 3367-3 and DH5α <I>E. coli</I> strains. Grey:</B> DH5α and JW 3367-3 cells without reporter system. <B>Pink:</B> JW 3367-3/pSB1C3/BBa_K1909014, <B>Green:</B> DH5α/BBa_K19090014.
    </P><P>
        We submitted parts BBa_K1909013 and BBa_K1909014 to the registry as our Silver medal criterion 1 parts. 
        In order to validate mTaz we created the parts <A href="http://parts.igem.org/Part:BBa_K1909007">BBa_K1909007</A> to <A href="http://parts.igem.org/Part:BBa_K1909009">BBa_K1909009</A>. They feature different combinations of Anderson Promoters, ribosome binding sites and mTaz. Best results were achieved using BBa_K1909009 as a combination of a medium Anderson Promoter <A href="http://parts.igem.org/Part:BBa_J3110">BBa_J23110</A>, and a weak ribosome binding site (<A href="http://parts.igem.org/Part:BBa_J3110">BBa_B0031</A>). mTaz, as a large transmembrane protein, proved to slow down growth of cells drastically. The combination with the lowest expression rate therefore yielded the best results. As positive controls, we used the same combination of Anderson promoters and ribosome binding sites with the original Taz1 receptor, resulting in Parts <A href="http://parts.igem.org/Part:BBa_K1909010">BBa_K1909010</A> to <A href="http://parts.igem.org/Part:BBa_K1909012">BBa_K1909012</A>. 
        We didn not manage to assemble parts BBa_K1909008 as well as BBa_K1909012, so they remain theoretical.
        We used flow cytometry to assess the functionality of mTaz.
    </P><P>
        mTaz was characterized by growing <I>E. coli</I> transformed with pSB1C3/BBa_K1909013 and pSB1A3/BBa_K1909002 in LB medium for an hour. Cells were inoculated with 0.1 mM mDAP as well as no mDAP as negative control and incubated for 30 min at 37°C.
        GFP expression induced by mTaz activation was measured by flow cytometry (Fig. 4A).
    </P><P class="img-content-description">
        Fig. 3: <B>mTaz pathway in <I>E. coli</I>.</B> Upon mDAP binding mTaz is autophosphorylated, transphosphorylating OmpR, which acts as a transcription factor, activating GFP expression.
    </P><P>
        Statistical analysis using K-S-test was used to evaluate significance of the results. The shift in fluorescence induced by 0.1 mM mDAP was statistically very significant (p&lt;0.001, Fig. 4B).
    </P><P class="img-content-description">
        Fig. 4: <B>Validation of mTaz using flow cytometry. A:</B> GFP fluorescence histogram. <B>Yellow:</B><I>E. coli</I> incubated in LB medium with 0.1 mM mDAP for 30 min at 37°C. <B>Black:</B> Negative control, <I>E. coli</I> incubated in LB medium without mDAP for 30 min at 37°C. <B>B:</B> K-S-statistics indicating statistical significance of fluorescence difference between negative control and experiment, p&lt;0.001.
    </P><H2>The usual problem with GMOs in field tests</H2><P>
        We designed our diagnostic system for use in the field, so scientists and doctors like the ones working at BICO can use them wherever they go. The obvious problem arising is the containment and safety of genetically engineered organisms outside the laboratory. Another problem is the sensitivity of the test to environmental influences. You cannot carry a live diagnostic system in the wild and expect reproducible results. We defined a set of requirements for a containment system for bacteria in field diagnostics:
    </P><TABLE id="table-requirements-device"><TBODY><TR><TD>
                A diagnostic device using modified bacteria needs to contain all bacteria inside it without releasing them under normal use conditions.
            </TD></TR><TR><TD>
                It has to provide a defined environment for bacteria to guarantee reproducible test results independent of outside parameters.
            </TD></TR><TR><TD>
                It has to be economic and may not cost more than an antibiotic treatment given due to the test results it produces.
       </TD></TR></TBODY></TABLE><P>
        In Microfluidics we saw potential to achieve all of our goals to develop a biosafe, easy to use and cheap diagnostic method to detect Chlamydia trachomatis. Polydimethylsiloxane (PDMS) is a cheap polymer solution. With photolithography and the use of cheap polymers, we can produce the microfluidic chip for $ 0.10. Another huge advantage is the easy usage of these devices. The medical personnel only needs syringes and can input the samples directly onto our chip. We also have invented a lab-in-phone device, which should be able to detect very sensitive fluorescence signals. The lab-in-phone device can be used with a smartphone in the field. The combination of nanotechnology and synthetic biology tools leads to a marketable product which is accustomed for the use in developing countries.
    </P><H2>Photolithography as a Tool for Rapid Microfluidic Chip Fabrication</H2><P>
        Therefore we use the principle of photolithography. We have produced microfluidic chips. Photolithography is a powerful technique for the fabrication of multiple chips. First we designed our microfluidic structures on AutoCAD 2010. This file is uploaded to the Laserwriter. Then we coated a chrome mask with a positive photoresist. A positive photoresist depolymerizes at exposed areas. The Laserwriter exposes the photoresist coated chrome mask at the designated areas of the structure. After this step the chrome mask is developed. Then the mask is etched by a chrome etch solution. The areas of the chrome mask which are not covered by photoresist are removed. This produced mask is finally cleaned with acetone, isopropanol and deionized water. The mask is the foundation for the production of the master template. We have coated a silicon wafer with a negative photoresist. A negative photoresist polymerizes when it is exposed. The chrome mask is aligned in a mask aligner. On the stage, below the mask holder of this machine, the coated wafer takes place. The mask aligner irradiates the full area of the chrome mask. The photoresist on the silicon wafer will be exposed by the illumination passing through the transparent areas of the mask. Afterwards, solidification of the exposed areas by light induced polymerization takes place. The unexposed, soluble photoresist is removed with propylene glycol methyl ether acetate (PGMEA). This yields the desired 3D structure on top of the wafer. This structured silicon wafer is called the master template and can be used as a stamp for mass production of chips. The next step is mixing PDMS with a curing agent. This reactive, viscous solution is put into a basin with the master template. After the PDMS is cured in an oven, it can easily be stripped off from the mold. The last step is the cutting and bonding of the microfluidic replica onto a glass slide. The bonding is realized by using an air plasma oven. Inside of it the plasma creates highly reactive hydroxyl groups or oxygen radicals on the glass and PDMS. When two activated surfaces are stuck together a covalent bond is formed.

    </P><H2>Alginate Encapsulation for Biosafety</H2><P>
        For our genetically modified E. coli strain the restrictions for handling genetically modified organisms (GMOs) apply. While taking care of biosafety, an easy-to-use integrated application came into our minds. Therefore we have produced a microfluidic chip, which encapsulates the bacteria into an alginate matrix. In the blueprint below descriptions to each structure by hovering over the elements are depicted.
    </P><P class="img-content-description">
        Fig. 6: <B>Blueprint of our Microfluidic Deivce for bacteria encapsulation.</B> A bacteria / alginate solution is split into droplets of 50 µM diameter by sunflower oil. Calcium solution is added to the droplets to let the alginate form solid capsules.<I>Click on the image for an interactive view of all elements of the design</I></P><P>
        Alginate is an anionic polysaccharide from the brown algae<A id="page-reference-22" href="#reference-22" class="reference">22<SPAN class="reference-tooltip">Draget, K. I. <I>et al.</I> Alginate from Algae. <I>Biopolymers</I><B>6</B> (2005).</SPAN></A>.
        The bacteria solution is mixed with 2.5% alginic acid. This solution is sheared into small droplets at a tapering structure with a carrier fluid (sunflower oil). At the T-junction, after the tapering structure and the junction of the pinch-off of the droplets, a calcium chloride solution (CaCl2) is jetting into the droplets and initiates the polymerization process of the alginic acid to alginate. The free calcium ions bind to the negatively charged oxygen groups (deprotonated) from the polysaccharide backbone of the alginic acid.  The pore size of the matrix is tuneable by using different concentrations of the polymerization initiator. This way we can ensure that the bacteria cannot escape the matrix. After a short washing step of the capsules with a 0.9% natrium chloride solution (NaCl) all weakly bonded bacteria are removed. Then the biosafety encapsulated bacteria are bonded into the microfluidic diagnostic chip.
    </P><H2>Usage of our Diagnostic Chip</H2><P>
        The diagnostic chip is also made of PDMS and contains three reservoirs (testing chambers) hosting the immobilized bacteria which are trapped inside the alginate capsules, ready to be probed with serum. The capsules themselves remain trapped inside the chambers due to the size of their diameter (100 µm) which prevents them from escaping through the 20 µm filters that border the chambers at the openings of the inlet and outlet channels.
    </P><P class="img-content-description">Fig. 7: <B>Blueprint of our Diagnostic Chip.</B> Three chambers, resembling positive and negative controls alongside the diagnostic chamber, are filled with encapsulated diagnostic bacteria. A patient sample can be applied to the chip using a standard syringe.<I>Click on the image for an interactive view of all elements of the design</I></P><P>
        The three chambers serve for diagnostic testing and as positive and negative control, respectively. Each chamber contains the same bacteria encapsulated in alginate. To address the chambers, they are connected to 115 µm wide inlet and outlet channels while being secured from flow-induced capsule elution by the 20 µm filters on both sides.
    </P><P>
        The liquid-leading input channels are protected from blockage caused by particles from outside at the chip entrances by a 120 µm wide, rough filter structure as opposed to the fine filter structures at the chamber-channel-interface, which serve biosafety by prevention of capsule leakage out of the chip.
    </P><P>
        The diagnostic chamber is probed with the patient's serum which passes through the rough filter, leaving all macroscopic particles behind, is led through the channels which spread into a wider area before passing the fine filter to decrease fluid resistance and inhibit filter blockage due to high throughput. Upon passing the fine filter, the serum is now exposed to the capsules and the contained diagnostic bacteria. If the serum contains chlamydia trachomatis the mDAP released by the pathogen is expected to reach the receptors of the immobilized bacteria by diffusing through the pores of the alginate capsules. This starts a signalling cascade which leads to the expression of GFP and results in a fluorescence signal. After a resting time, the fine filter at the chamber exit is passed and the serum enters the outlet channel.
    </P><P>
        For the positive control chamber the procedure is the same, except the testing liquid is a mDAP solution and enters through its own dedicated chip input and inlet channel. Using concentrations of mDAP high enough could deliver an upper fluorescence threshold that could be used to quantify the degree of infection when compared with the diagnostic chamber.
    </P><P>
        The negative control chamber remains closed and unaddressable, being free of connecting channels. This gives a fluorescence signal control for comparison with the test chambers - to find lower detection thresholds and background fluorescence intensity of the bacteria and the chip itself.
    </P><P>
        The two outlet channels of the chamber of diagnosis and the positive control join together into a single output leading out of the chip. At the junction of the outlet channels and the chip output however the channels taper from 115 µm width to 50 µm. By narrowing the channel exits the outflow is prevented from leaking into the respectively other channels and chambers.
    </P><P>
        Each chamber has dimensions of 2 x 5 mm, with a height of n x 100 µm resulting in a volume of n x 1 µL  and allowing for up to n x 1000 capsules per chamber. The depth of the chambers is stackable to multiples of 100 µm by modification of the master mold. This tunable depth of bacteria-containing medium (corresponding with an increase of the fluorescent  agent) can be used to amplify the fluorescence intensity by roughly the number of layers. In order to assess this fluorescence signal it first has to be read out and thus it is convenient that PDMS has a rather low intrinsic fluorescence at the emission peak of GFP (509 nm), supplying only an insignificant background signal<A id="page-reference-23" href="#reference-23" class="reference">23<SPAN class="reference-tooltip">Cesaro-Tadic, S., Dernick, G., Juncker, D., Buurman, G., Kropshofer, H., Michel, B., Fattinger, C., Delamarche, E. High-Sensitivity Miniaturized Immunoassays for Tumor Necrosis Factor α using Microfluidic Systems. <I>Lab Chip</I><B>4</B>, 563-569 (2004).</SPAN></A>.
        The chip can either be made of only PDMS, with two halves bonded onto each other or bonded on glass, e. g. a microscope slide which would make it chunkier, but easier to handle especially while read-out. The chip itself is roughly 2 x 1 x 0.4 cm large.  These dimensions are chosen to read it out by simply looking at it or analysing it in a lab-in-phone setup. For this we designed such a device.
    </P><H2>Fluorescence Detection via a Lab-in-Phone Device</H2><P>
        This lab-in-phone device makes it possible to measure the expressed fluorescence protein with a smartphone camera. Our device is designed for the Motorola ATRIX MB865, that is mostly used in field operations from the BICO. We especially focused on this phone, because our human practises team discovered this phone is widely used for collecting data in the field on <I>Chlamydia trachomatis</I> infections. This smartphone has an 8 megapixel camera with an autofocus function. Our designed lab-in-phone device contains a filter, which filters the excited light from the emitted one, and a lens which enhances the resolution. The device also contains a small box for electronics. The idea is that the LED or laser diodes with a narrow wavelength spectrum excite the expressed fluorophore. In case of using GFP the excitation wavelength is at 495 nm and emitting takes place at 509 nm. Then the light goes to an aspherical lens. This type of lenses have the advantage that they bundle the light much better than concave or convex lenses. The same effect can be reached by multi-lens-systems. The second variant is a very expensive one.
    </P><H2>References</H2><OL><LI id="reference-1"><A href="#page-reference-11">↑</A><A target="_blank" href="http://apps.who.int/iris/bitstream/10665/75181/1/9789241503839_eng.pdf">
                Global incidence and prevalence of selected curable sexually transmitted infections – 2008. <I>WHO, Department of Reproductive Health and Research </I>(2011).
            </A></LI><LI id="reference-2"><A href="#page-reference-11">↑</A><A target="_blank" href="http://www.who.int/hiv/pub/sti/who_hiv_aids_2001.02.pdf">
                Global Prevalence and Incidence of Selected Curable Sexually Transmitted Infections – Overview and Estimates. <I>WHO</I> (2001).
            </A></LI><LI id="reference-3"><A href="#page-reference-11">↑</A><A target="_blank" href="https://www.ncbi.nlm.nih.gov/pubmed/23926547">
                Mania-Pramanik, J., Kerkar. S., Sonawane, S., Mehta, P., Salvi, V. Current Chlamydia trachomatis In-fection, A Major Cause of Infertility. <I>J. Reprod. Infertil.</I><B>4</B>, 204-210 (2012).
            </A></LI><LI id="reference-4"><A href="#page-reference-11">↑</A><A target="_blank" href="https://www.ncbi.nlm.nih.gov/pubmed/18087650">
                Da Ros, C., da Silva Schmitt, C. Global epidemiology of sexually transmitted diseases. <I>Asian. J. Androl.</I><B>10</B>, 110-114 (2008).
            </A></LI><LI id="reference-5"><A href="#page-reference-11">↑</A><A target="_blank" href="https://www.ncbi.nlm.nih.gov/pubmed/16536754">
                Wagenlehner, FM., Naber, KG., Weidner, W. Chlamydial infections and prostatitis in men. <I>BJU Int.</I><I>98</I>, 687-90 (2006).
            </A></LI><LI id="reference-6"><A href="#page-reference-11">↑</A><A target="_blank" href="https://arthritis-research.biomedcentral.com/articles/10.1186/ar2398">
                Siala, M. et al. Analysis of bacterial DNA in synovial tissue of Tunisian patients with reactive and un-differentiated arthritis by broad-range PCR, cloning and sequencing. <I>Arthritis Research &amp; Therapy</I><I>10</I>, R40 (2008).
            </A></LI><LI id="reference-7"><A href="#page-reference-11">↑</A><A target="_blank" href="https://www.ncbi.nlm.nih.gov/pubmed/22133988">
                Pascolini, D., Mariotti S. P. Global estimates of visual impairment: 2010. <I>Br. J. Ophthalmol.</I><B>96</B>, 614-618 (2012).
            </A></LI><LI id="reference-8"><A href="#page-reference-11">↑</A><A target="_blank" href="http://apps.who.int/iris/bitstream/10665/43853/1/9789241563475_eng.pdf">
                Global strategy for the prevention and control of sexually transmitted infections: 2006-2015. Geneva, <I>World Health Organization</I> (2007).
            </A></LI><LI id="reference-9"><A href="#page-reference-11">↑</A><A target="_blank" href="http://www.nejm.org/doi/full/10.1056/NEJMoa1502599#t=article">
                Geisler, W. M. et al. Azithromycin versus Doxycycline for Urogenital Chlamydia trachomatis Infection. <I>N. Engl. J. Med.</I><B>373</B>, 2512-2521 (2015).
            </A></LI><LI id="reference-10"><A href="#page-reference-11">↑</A><A target="_blank" href="https://www.ncbi.nlm.nih.gov/pubmed/19656295">
                Henrichfreise, B. <I>et al.</I> Functional conservation of the lipid II biosynthesis pathway in the cell wall-less bacteria Chlamydia and Wolbachia: why is lipid II needed? <I>Mol. Microbiol.</I><B>73</B>, 913-923 (2009).
            </A></LI><LI id="reference-11"><A href="#page-reference-11">↑</A><A target="_blank" href="https://www.ncbi.nlm.nih.gov/pubmed/2476847">
                Utsumi, R. <I>et al.</I> Activation of Bacterial Porin Gene Expression by a Chimeric Signal Transducer in Response to Aspartate. <I>Science</I><B>245</B>, 1246-1249 (1989).
            </A></LI><LI id="reference-12"><A href="#page-reference-12">↑</A><A target="_blank" href="https://www.ncbi.nlm.nih.gov/pubmed/383889">
                Reader, R. W., Tso, W. W., Springer, M. S., Goy, M. F., Adler, J. Pleiotropic aspartate taxis and serine taxis mu-tants of Escherichia coli. <I>J. Gen. Microbiol</I><B>111</B>, 363-374 (1979).
            </A></LI><LI id="reference-13"><A href="#page-reference-13">↑</A><A target="_blank" href="https://www.ncbi.nlm.nih.gov/pubmed/16359703">
                Derr, P., Boder, E., Goulian, M. Changing the Specificity of a Bacterial Chemoreceptor. <I>. Mol. Biol.</I>J <B>355</B>, 923-932 (2006).
            </A></LI><LI id="reference-14"><A href="#page-reference-14">↑</A><A target="_blank" href="https://www.ncbi.nlm.nih.gov/pubmed/9422609">
                Lan, C. Y., Igo, M. M. Differential expression of the OmpF and OmpC porin proteins in Escherichia coli K-12 depends upon the level of active OmpR. <I>J. Bacteriol.</I><B>180</B>, 171.174 (1998).
            </A></LI><LI id="reference-15"><A href="#page-reference-15">↑</A><A target="_blank" href="https://www.ncbi.nlm.nih.gov/pubmed/1700256">
                Mizuno, T., Mizushima, S. Signal transduction and gene regulation through the phosphorylation of two regu-latory components: The molecular basis for the osmotic regulation of the porin genes. <I>Mol. Microbiol.</I><B>4</B>, 1077-1082 (1990).
            </A></LI><LI id="reference-16"><A href="#page-reference-16">↑</A><A target="_blank" href="https://www.ncbi.nlm.nih.gov/pubmed/1660927">
                Russo, F. D., Silhavy, T.J. EnvZ controls the concentration of phosphorylated OmpR to mediate osmoregula-tion of the porin genes. <I>J. Mol. Biol.</I><B>222</B>, 567-580 (1991).
            </A></LI><LI id="reference-17"><A href="#page-reference-17">↑</A><A target="_blank" href="https://www.ncbi.nlm.nih.gov/pubmed/2826400">
                Slauch, J. M., et al. EnvZ functions through OmpR to control porin gene expression in Escherichia coli K-12. <I>J. Bacteriol.</I><B>170</B>, 439-441 (1988).

            </A></LI><LI id="reference-18"><A href="#page-reference-18">↑</A><A target="_blank" href="https://www.ncbi.nlm.nih.gov/pubmed/17609131">
                Yoshida, T., Phadtare, S., Inouye, M. The design and development of Tar-EnvZ chimeric receptors. <I>Methods Enzymol.</I><B>423</B>, 166-183 (2007).
            </A></LI><LI id="reference-19"><A href="#page-reference-19">↑</A><A target="_blank" href="https://www.ncbi.nlm.nih.gov/pubmed/19656295">
                Henrichfreise, B. et al. Functional conservation of the lipid II biosynthesis pathway in the cell wall-less bacte-ria Chlamydia and Wolbachia: why is lipid II needed? <I>Mol. Microbiol.</I><B>73</B>, 913-923 (2009).
            </A></LI><LI id="reference-20"><A href="#page-reference-20">↑</A><A target="_blank" href="https://www.ncbi.nlm.nih.gov/pubmed/27292793">
                Mise, T. Structural Analysis of the Ligand-Binding Domain of the Aspartate Receptor Tar from Escherichia coli. <I>Biochemistry</I><B>55</B>, 3708-3713 (2016).
            </A></LI><LI id="reference-21"><A href="#page-reference-21">↑</A><A target="_blank" href="https://www.schrodinger.com/glide">
                Small-Molecule Drug Discovery Suite 2016-3: Glide, version 7.2, Schrödinger, LLC, New York, NY (2016).
            </A></LI><LI id="reference-22"><A href="#page-reference-21">↑</A><A target="_blank" href="https://www.schrodinger.com/glide">
                Draget, K. I. <I>et al.</I> Alginate from Algae. <I>Biopolymers</I><B>6</B> (2005).
           </A></LI></OL></DIV></DIV></DIV></DIV><DIV id="footer"><DIV id="footer-sponsors">
                    Sponsored by:
                    </DIV></DIV><DIV id="fullscreen-diagnostic" class="hidden"><DIV class="fullscreen-content"><DIV id="feature-diagnostic-input" class="blueprint-feature hidden" style="top:27.2%; left:18.5%;"><H3>
                    Sample Input
                </H3><P>
                    First inlet for the patient sample, which consists of a semen sample or vaginal
                    swap specimen. This sample flows into the reservoir of the encapsulated positive
                    control bacteria. Second inlet for the patient sample, which also consists of a semen sample or vaginal
                    swap specimen. This sample flows into the reservoir of the encapsulated diagnostic
                    bacteria. The second inlet is necessary to prevent a contamination from the positive control reservoir into the diagnostic one.
                </P></DIV><DIV id="feature-diagnostic-roughfilter" class="blueprint-feature hidden" style="top:22%; left:25%;"><H3>
                    Rough Filter Structure
                </H3><P>
                    The rough filter structures protect the reservoirs of coarse impurities. These can be
                    risen by the fabrication process of cutting the connections or plugging-in of the tubes from the syringes. Another aspect is to slow down the speed of flow.
                </P></DIV><DIV id="feature-diagnostic-finefilter" class="blueprint-feature hidden" style="top:75%; left:36%;"><H3>
                    Fine Filter Structure
                </H3><P>
                    The fine filter structures guarantee that the encapsulated bacteria can not escape
                    out of the reservoirs.
                </P></DIV><DIV id="feature-diagnostic-positive" class="blueprint-feature hidden" style="top:52.6%; left:50%;"><H3>
                    Resevoir for the Positive Control
                </H3><P>
                    For the positive control we use as the testing liquid a mDAP solution which enters through its own dedicated chip input and inlet channel. Using concentrations of mDAP high enough could deliver an upper fluorescence threshold that could be used to quantify the degree of infection when compared with the diagnostic chamber.
                </P></DIV><DIV id="feature-diagnostic-negative" class="blueprint-feature hidden" style="top:50%; left:50%;"><H3>
                    Reservoir for the Negative Control
                </H3><P>
                    The negative control chamber remains closed and unaddressable, being free of connecting channels. This gives a fluorescence signal control for comparison with the test chambers - to find lower detection thresholds and background fluorescence intensity of the bacteria and the chip itself.
                </P></DIV><DIV id="feature-diagnostic-experiment" class="blueprint-feature hidden" style="top:27%; left:50%;"><H3>
                    Reservoir for the Diagnostic Bacteria
                </H3><P>
                    Reservoir for the diagnostic bacteria. The E. coli strain JW 3367, which is
                    provided with a self-built receptor, is used. The bacteria is encapsulated in an
                    alginate matrix.
                </P></DIV><DIV id="feature-diagnostic-outlet" class="blueprint-feature hidden" style="top:50%; left:65.3%;"><H3>
                    Output
                </H3><P>
                    The tapering in front of the outlet rises the pressure and makes it unlikely for the
                    fluid to flow through it. Instead it rests in the reservoir. Only in high pressure
                    situations the fluid can pass the outlet.
                </P></DIV></DIV></DIV><DIV id="fullscreen-encapsulation" class="hidden"><DIV class="fullscreen-content"><DIV id="feature-encapsulation-inlet1" class="blueprint-feature hidden" style="top:63.5%; left:19.2%;"><H3>
                   Shearing Medium Input
                </H3><P>
                    The inlet for shearing medium contains a special oil solution. This is also called the carrier fluid. The carrier fluid has a high viscosity.    
                </P></DIV><DIV id="feature-encapsulation-roughfilter" class="blueprint-feature hidden" style="top:55%; left:28%;"><H3>
                    Rough Filter Structure
                </H3><P>
                    The rough filter structures protect the reservoirs of coarse impurities. These can be
                    risen by the fabrication process of cutting the connections or plugging-in of the tubes from the syringes. Another aspect is to slow down the speed of flow.
                </P></DIV><DIV id="feature-encapsulation-splitstructure" class="blueprint-feature hidden" style="top:63.5%; left:31.8%;"><H3>
                    Bifurcation
                </H3><P>
                    The bifurcation splits the shearing medium in two equal flows. These are needed
                    later for creating small separate droplets of the bacteria and alginic acid solution.
                </P></DIV><DIV id="feature-encapsulation-inlet2" class="blueprint-feature hidden" style="top:32.2%; left:49.8%;"><H3>
                    Bacteria Suspension Input
                </H3><P>
                    The inlet contains the bacteria suspension, a 0.9% NaCl and 2.5% alginic acid
                    solution.
                </P></DIV><DIV id="feature-encapsulation-inlet3" class="blueprint-feature hidden" style="top:63.5%; left:49.8%;"><H3>
                    Input of Polymerization Initiator
                </H3><P>
                    The inlet for barium chloride or calcium chloride, which later initiate the
                    polymerization of the alginic acid.
                </P></DIV><DIV id="feature-encapsulation-tjunction" class="blueprint-feature hidden" style="top:63.5%; left:67.6%;"><H3>
                    Tapering Structure
                </H3><P>
                        The tapering supports the production of small loads of bacteria suspension, which is
                        separated by the carrier fluid. A higher flowrate of the carrier fluid leads to smaller droplets (and vice versa).
                    </P></DIV><DIV id="feature-encapsulation-reactor" class="blueprint-feature hidden" style="top:63.5%; left:75.2%;"><H3>
                    T-Junction
                </H3><P>
                    At the T-junction the barium chloride or calcium chloride solution is added to the
                    fluid.The barium or calcium ions initiate the polymerization process. The ions bind to
                    the negatively charged oxygen groups from the alginic acid. A silicone like capsule is
                    formed.
                </P></DIV><DIV id="feature-encapsulation-meander" class="blueprint-feature hidden" style="top:53%; left:61%;"><H3>
                    Meander Structure
                </H3><P>
                    The meander structure ensures, that the barium or calcium chloride solution is well
                    mixed with the alginic acid. 
                </P></DIV><DIV id="feature-encapsulation-outlet" class="blueprint-feature hidden" style="top:36.3%; left:61.4%;"><H3>
                    Output for Alginate Capsules
                </H3><P>
                    The outlet releases alginate encapsulated bacteria.
                </P></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></BODY></HTML>