<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_UGA-Georgia_Description skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:UGA-Georgia/Description</SPAN></H1><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><DIV id="mainContainer"><DIV id="menuContainer"><UL><LI><A href="https://2015.igem.org/Team:UGA-Georgia">HOME</A></LI><LI><A href="https://2015.igem.org/Team:UGA-Georgia/Team">TEAM</A></LI><LI><A href="#">PROJECT
            					</A></LI><LI><A href="https://2015.igem.org/Team:UGA-Georgia/Description">Description</A></LI><LI><A href="https://2015.igem.org/Team:UGA-Georgia/Experiments">Experiments &amp; Protocols</A></LI><LI><A href="https://2015.igem.org/Team:UGA-Georgia/Results">Results</A></LI><LI><A href="https://2015.igem.org/Team:UGA-Georgia/Design">Design</A></LI></UL><UL><LI><A href="#">PARTS
            					</A></LI><LI><A href="https://2015.igem.org/Team:UGA-Georgia/Parts">Team Parts</A></LI></UL><LI><A href="https://2015.igem.org/Team:UGA-Georgia/Notebook">NOTEBOOK</A></LI><LI><A href="https://2015.igem.org/Team:UGA-Georgia/Attributions">ATTRIBUTIONS</A></LI><LI><A href="https://2015.igem.org/Team:UGA-Georgia/Collaborations">COLLABORATIONS</A></LI><LI><A href="https://2015.igem.org/Team:UGA-Georgia/Practices">HUMAN PRACTICES</A></LI><LI><A href="https://2015.igem.org/Team:UGA-Georgia/Safety">SAFETY</A></LI><LI><A href="https://2015.igem.org/Team:UGA-Georgia/Modeling">MODELING</A></LI><LI><A href="https://2015.igem.org/Team:UGA-Georgia/Measurement">MEASUREMENT</A></LI><LI><A href="https://2015.igem.org/Team:UGA-Georgia/Software">SOFTWARE</A></LI></DIV><DIV id="contentContainer"><H2> Project Description </H2><P><I>Methanococcus maripaludis</I> is a model organism for Archaea, which affords researchers the beneficial qualities such as (1) producing methane used as biogas and (2) manufacturing isoprenoids as precursors for high-value biochemicals. However, there are few genetic tools available for metabolic engineering Archaea. Our goal is to develop some useful tools for synthetic biology of Archaea. Building on our past <I>M. maripaludis</I> projects, which created and characterized a mCherry reporter system and a recombinant mutant making geraniol, our team is now working to (1) create, characterize and model a ribosome-binding site (RBS) library using the mCherry reporter system and (2) model geraniol production of the recombinant <I>M. maripaludis</I> using flux balance analyses. Preliminary results have shown varying levels of expression in the RBS library, and increased geraniol yield from some growth substrates. Additionally, our team has initiated an Archaeal InterLab Study to further characterize the reproducibility of our mCherry reporter system.</P><H2>Why an Archaea in Synthetic Biology?</H2><P>The goal of the UGA iGEM team is to establish the feasibility of Archaea in the field of synthetic biology. Compared to Bacteria and eukaryotes, Archaea have many unique properties such as an isoprenoid based membrane and a diverse autotrophic metabolism <SUP>[1,2]</SUP>. Importantly, methanogenic archaea are the only group of organisms that produce methane, a renewable biofuel. However, most Archaea have not been as extensively studied as many of the traditional chassis organism such as E. coli and yeast.  Therefore our work aims to elucidate more light work of an Archaea in Synthetic Biology.</P><H3>But why <I>M. mariplaudis</I>?</H3><P>Most Archaea are extremophiles, which means that are inherently difficult organisms to work with.  <I>M. mariplaudis</I> is Anaerobic Archaea that was isolated from eastern coast salt marshes. The greatest obstacle in working with this Archaea is that it is anaerobic and although this produces significant challenges, <I>M. mariplaudis</I> is more practical to work with then other Archaea, such as the ones that live in extremely hot or acidic environments. Moreover this organism has many characteristics that make it an ideal organism for study. These include its rapid growth, a complete genome sequence, and a robust set of genetic manipulation techniques. Finally, <I>M. mariplaudis</I> produces methane using simple and inexpensive substrates such as Hydrogen, CO<SUB>2</SUB> and formate <SUP>[3]</SUP>. Therefore, a role of this organism in the H<SUB>2</SUB> economy can be envisaged </P><P align="center"> Figure 1. Potential role of methanogens in a H<SUB>2</SUB> economy powered by the sun. Hydrolysis of H<SUB>2</SUB>O using electricity generated by solar panels produces large amount of H<SUB>2</SUB> as the energy carrier <SUP>[4]</SUP>. An upgraded electrical process can also be used to produce formate as a liquid fuel <SUP>[5]</SUP>. Methanogens convert H<SUB>2</SUB> and CO<SUB>2</SUB> (or formate) to high-value biochemicals, CH<SUB>4</SUB> and H<SUB>2</SUB>O. CH<SUB>4</SUB> can be used as a fuel for cooking and transportation among other things.  H<SUB>2</SUB>O can be returned to the hydrolysis step for production of more H<SUB>2</SUB>.</P><H2 align="center"> Overview of the Projects</H2><H3>The Ribosome Binding Site Project</H3><P>The Ribosome Binding site project was the most time intensive project taken on in the summer semester. This project involves first transforming plasmids containing the mCherry gene and that have mutations to the Ribosome Binding Site in the plasmids. This project was such a large undertaking that it was split into three different subprojects with different group leaders heading each subproject: transformation, screening and sequencing. The ultimate goal of this project is to facilitate future endeavor with Archaea in the field of synthetic biology by providing researchers more options to vary protein expression levels.</P><P><FONT size="3"><B><I>Transformation</I></B></FONT>; The transformation team had started with the wild type <I>M. maripaludis</I> and purified many mutant colonies that carried site-mutated ribosome binding site for expression mCherry in the pMEV4 plasmid.</P><P><FONT size="3"><B><I>Screening</I></B></FONT>; The screening team determined expression levels of mCherry fluorescence for each of the mutants by harvesting cell extracts and using plate reader to measure fluorescence.</P><P><FONT size="3"><B><I>Sequencing</I></B></FONT>; Since the location of the mutations were unknown, the goal of the sequencing team was to determine the sequence of the mutations and map the site mutations <I>in silico</I> using Geneious.</P><H3>The UTR Modeling Project</H3><P>The UTR model was used to determine the level of translational efficiency for various Ribosome Binding Site Sequences, by assessing the specific regions for mRNA folding <SUP>[6]</SUP>. The model was developed for Bacteria and has never been validated in Archaea before. Therefore, we decided to analyze whether the model would fit well with our experimental data generated from the ribosomal binding site project.</P><H3>The Flux Balance Modeling Project</H3><P>We also retrieved a metabolic flux model of other <I>M. maripaludis</I> researcher based in Singapore <SUP>[7]</SUP>. We then added our reaction for geraniol synthase to the model. Our goal was to determine how changing the substrate concentration would affect the formation of biomass and geraniol. The <I>in silico</I> data generated from this project will be useful to determine the concentrations of various substrates needed to maximize geraniol production.</P><H4>References</H4><OL><LI>Albers SV, Meyer BH (2011) The archaeal cell envelope. Nature reviews Microbiology 9 (6):414-426. doi:10.1038/nrmicro2576</LI><LI>Berg IA, Kockelkorn D, Ramos-Vera WH, Say RF, Zarzycki J, Hugler M, Alber BE, Fuchs G (2010) Autotrophic carbon fixation in archaea. Nature reviews Microbiology 8 (6):447-460. doi:10.1038/nrmicro2365</LI><LI>Jones WJ, Paynter MJB, Gupta R (1983) Characterization of Methanococcus maripaludis sp. nov., a new methanogen isolated from salt marsh sediment. Archives of Microbiology 135 (2):91-97. doi:10.1007/BF00408015</LI><LI>Liu J, Liu Y, Liu N, Han Y, Zhang X, Huang H, Lifshitz Y, Lee ST, Zhong J, Kang Z (2015) Water splitting. Metal-free efficient photocatalyst for stable visible water splitting via a two-electron pathway. Science 347 (6225):970-974. doi:10.1126/science.aaa3145</LI><LI>White JL, Herb JT, Kaczur JJ, Majsztrik PW, Bocarsly AB (2014) Photons to formate: Efficient electrochemical solar energy conversion via reduction of carbon dioxide. J CO2 Util 7:1-5. doi:http://dx.doi.org/10.1016/j.jcou.2014.05.002</LI><LI>Seo SW, Yang JS, Kim I, Yang J, Min BE, Kim S, Jung GY (2013) Predictive design of mRNA translation initiation region to control prokaryotic translation efficiency. Metabolic engineering 15:67-74. doi:10.1016/j.ymben.2012.10.006</LI><LI>Goyal N, Widiastuti H, Karimi IA, Zhou Z (2014) A genome-scale metabolic model of Methanococcus maripaludis S2 for CO2 capture and conversion to methane. Molecular bioSystems 10 (5):1043-1054. doi:10.1039/C3MB70421A</LI></OL></DIV></DIV></DIV></DIV></DIV></DIV></BODY></HTML>