<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_UiOslo_Norway_Lab skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><DIV id="top_title"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:UiOslo Norway/Lab</SPAN></H1></DIV><DIV id="HQ_page"><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><DIV class="container"><DIV class="navbar-header"><BUTTON type="button" class="navbar-toggle" data-toggle="collapse" data-target=".navbar-collapse"><SPAN class="sr-only">Toggle navigation</SPAN></BUTTON></DIV><DIV class="collapse navbar-collapse"><UL class="nav navbar-nav navbar-right"><LI class="active"><A href="https://2017.igem.org/Team:UiOslo_Norway">Home</A></LI><LI class="dropdown"><A href="#">Project</A></LI><LI><A href="https://2017.igem.org/Team:UiOslo_Norway/Description">Description</A></LI><LI><A href="https://2017.igem.org/Team:UiOslo_Norway/Lab">Methods</A></LI><LI><A href="https://2017.igem.org/Team:UiOslo_Norway/Parts">Parts</A></LI><LI><A href="https://2017.igem.org/Team:UiOslo_Norway/Results">Results</A></LI><LI><A href="https://2017.igem.org/Team:UiOslo_Norway/Discussion">Discussion</A></LI><LI><A href="https://2017.igem.org/Team:UiOslo_Norway/Safety">Safety</A></LI></UL><LI class="dropdown"><A href="https://2017.igem.org/Team:UiOslo_Norway/Modelling">Modelling</A></LI><LI class="dropdown"><A href="https://2017.igem.org/Team:UiOslo_Norway/InterLab">InterLab</A></LI><LI class="dropdown"><A href="#">Notebook</A></LI><LI><A href="https://2017.igem.org/Team:UiOslo_Norway/Notebook">Work Notebook</A></LI><LI><A href="https://2017.igem.org/Team:UiOslo_Norway/Timeline">Timeline</A></LI><LI><A href="https://2017.igem.org/Team:UiOslo_Norway/Meetings">Meetings</A></LI><LI class="dropdown"><A href="#">Human Practices</A></LI><LI><A href="https://2017.igem.org/Team:UiOslo_Norway/Engagement">Public Engagement</A></LI><LI><A href="https://2017.igem.org/Team:UiOslo_Norway/Integrated_Practices">Integrated Practices</A></LI><LI><A href="https://2017.igem.org/Team:UiOslo_Norway/Collaborations">Collaborations</A></LI><LI><A href="https://2017.igem.org/Team:UiOslo_Norway/Sponsors">Sponsors</A></LI><LI class="dropdown"><A href="#">Team</A></LI><LI><A href="https://2017.igem.org/Team:UiOslo_Norway/Team">Team Members</A></LI><LI><A href="https://2017.igem.org/Team:UiOslo_Norway/Attributions">Attributions</A></LI><LI><A href="https://2017.igem.org/Team:UiOslo_Norway/Achievements">Achievements</A></LI></DIV></DIV><H1 class="h1-font-other">Methods</H1><DIV class="bootstrap-overrides padding-left padding-right"><DIV id="tab-container"><DIV class="row"><DIV class="col-md-12"><UL id="tab2" class="nav nav-pills"><LI class="active"><A href="#tab-item1" data-toggle="tab">Gibson</A></LI><LI><A href="#tab-item2" data-toggle="tab">Transformation</A></LI><LI><A href="#tab-item3" data-toggle="tab">PCR</A></LI><LI><A href="#tab-item4" data-toggle="tab">Gel</A></LI><LI><A href="#tab-item5" data-toggle="tab">Miniprep</A></LI><LI><A href="#tab-item6" data-toggle="tab">French press</A></LI><LI><A href="#tab-item7" data-toggle="tab">sfGFP purification</A></LI><LI><A href="#tab-item8" data-toggle="tab">Growth protocol</A></LI><LI><A href="#tab-item9" data-toggle="tab">Interlab</A></LI><LI><A href="#tab-item10" data-toggle="tab">NTNU Collaboration</A></LI><LI><A href="#tab-item11" data-toggle="tab">LED</A></LI><LI><A href="#tab-item12" data-toggle="tab">Biolaser</A></LI></UL><DIV class="tab-content"><DIV class="tab-pane fade active in" id="tab-item1"><DIV class="bootstrap-overrides">
    Gibson Assembly allows for successful assembly of multiple DNA fragments, regardless of fragment length or end compatibility. (1). The method
    was invented in 2009 by Daniel G. Gibson, of the J. Craig Venter Institute. The assembly reaction is carried out in
    one single reaction-tube, all at once, at 50° Celsius for 15-60 minutes. The process involves three different 
    enzymatic actions. A 5’ exonuclease creates overhangs, enabling matched fragments to anneal. Then a DNA polymerase
    fills gap between the annealed strands and the 5´ end. Finally, a DNA ligase seals the gaps between the filled in 
    gap and the annealed strands. <H3><A class="bodyURLcolor" href="https://www.neb.com/protocols/2012/12/11/gibson-assembly-protocol-e5510">Protocol used for Gibson </A></H3><B>Modifications</B>
 (Used for insertion of nmt1, cyc1, sfGFP and composite part into submition vector pSB1C3 and insertion of composite part into yeast vector)
    (i)    Volume Changes: V<SUB>insert</SUB> = x V<SUB>vector</SUB> = y  V<SUB>gibson</SUB> = x+y V<SUB>water</SUB> = 0µl
    
    (ii)   Incubation for 1h, not 15 min 
    
    (iii)  Before transformation: One transformation with x ul concentrated Gibson solution and one transformation with
           Gibson solution diluted 1:3 and transformation with 3*x ul diluted Gibson solution. </DIV></DIV><DIV class="tab-pane fade" id="tab-item2"><DIV class="bootstrap-overrides"><B> E.coli TOP10 </B>(Used for nmt1, cyc1, sfGFP and composite part) : 
         One Shot® TOP10 E. coli are provided at a transformation efficiency of 1 x 109 cfu/µg supercoiled DNA and are 
         ideal for high-efficiency cloning and plasmid propagation. They allow stable replication of high-copy number 
         plasmids.<A class="bodyURLcolor" href="https://www.thermofisher.com/us/en/home/references/protocols/cloning/competent-cells-protocol/routine-cloning-using-top10-competent-cells.html">Chemical Transformation Procedure </A><B>Modifications </B>
          In Step 5, Incubate for exactly 30-45 seconds in the 42°C water bath. Do not mix or shake.
          In Step 7, Add 200-250 µl of rom temperatured S.O.C medium to each vial. S.O.C is a rich medium; sterile 
          technique must be practiced to avoid contamination <B> E.coli DH5Alpha </B> (Used for the Interlab and for purification of sfGFP):  <A href="http://parts.igem.org/Help:Protocols/Transformation">Protocol</A>
         Modifications: 
          Step 15 and Step 16 not done </DIV></DIV><DIV class="tab-pane fade" id="tab-item3"><DIV class="bootstrap-overrides">
                           The goal of PCR is to amplify a section of DNA of interest for DNA analysis (e.g. gene insertion, sequencing, etc). The amplification rate is exponential.

<UL><LI>Tag polimerase (25µl reaction) Used for colony PCR for cyc1</LI><A class="bodyURLcolor" href="https://static.igem.org/mediawiki/2017/d/d9/T--UiOslo_Norway--5-PRIME_HotMasterMix-Protocol.pdf
">Protocol Here</A><B>Specifications</B><TABLE class="customers"><TBODY><TR><TD>10X Standard Taq Reaction Buffer</TD><TD>2.5 μl</TD></TR><TR><TD>10 mM dNTPs</TD><TD>0.5 µl</TD></TR><TR><TD>10 µM VF2</TD><TD>0.5 µl</TD></TR><TR><TD>10 µM VR</TD><TD>0.5 µl</TD></TR><TR><TD>Template DNA</TD><TD>variable</TD></TR><TR><TD>Taq DNA Polymerase</TD><TD>0.125 µl</TD></TR><TR><TD>Nuclease-free water to</TD><TD>25 µl</TD></TR></TBODY></TABLE><TABLE class="customers"><TBODY><TR><TD>1) 95°C </TD><TD>30 sec</TD></TR><TR><TD>2) 95°C </TD><TD>30 sec</TD></TR><TR><TD>3) 63°C</TD><TD>1 min</TD></TR><TR><TD>4) 68°C</TD><TD>1 min/kb</TD></TR><TR><TD>5) 2/30X</TD></TR><TR><TD>6) 68°C</TD><TD>5 min</TD></TR><TR><TD>7) 10°C </TD><TD>forever</TD></TR></TBODY></TABLE><LI> Phusion polymerase (20µl reaction) Used for colony PCR for nmt1 and composite part : </LI><A class="bodyURLcolor" href="https://www.neb.com/protocols/1/01/01/pcr-protocol-m0530
">Protocol Here</A><B>Specifications</B><TABLE class="customers"><TBODY><TR><TD>15X Phusion HF or GC Buffer</TD><TD> 4 µl</TD></TR><TR><TD>10 mM dNTPs</TD><TD>0.4 µl</TD></TR><TR><TD>10 µM VF2</TD><TD>1 µl</TD></TR><TR><TD>10 µM VR</TD><TD>1 µl</TD></TR><TR><TD>Template DNA</TD><TD>variable</TD></TR><TR><TD>Phusion DNA Polymerase</TD><TD>0.2 µl</TD></TR><TR><TD>Nuclease-free water to</TD><TD>20 µl</TD></TR></TBODY></TABLE><TABLE class="customers"><TBODY><TR><TD>1) 95°C </TD><TD>30 sec</TD></TR><TR><TD>2) 98°C </TD><TD>30 sec</TD></TR><TR><TD>3) 63°C</TD><TD>1 min</TD></TR><TR><TD>4) 72°C</TD><TD>1 min/kb</TD></TR><TR><TD>5) 2/30X</TD></TR><TR><TD>6) 72°C</TD><TD>5 min</TD></TR><TR><TD>7) 10°C </TD><TD>forever</TD></TR></TBODY></TABLE><LI>  5 PRIME HotMasterMix (50µl and 10µl reaction) Used for colony PCR for nmt1 and composite part : </LI><A class="bodyURLcolor" href="https://www.genetargetsolutions.com.au/wp-content/uploads/2016/06/5-Prime-Hot-Master-Mix-Manual.pdf
">Protocol Here</A><B>Specifications</B><TABLE class="customers"><TABLE class="customers"><TBODY><TR><TD>10 µM VF2</TD><TD>0.4 µl</TD></TR><TR><TD>10 µM VR</TD><TD>0.4 µl</TD></TR><TR><TD>Template DNA</TD><TD>variable</TD></TR><TR><TD>Phusion DNA Polymerase</TD><TD>4 µl</TD></TR><TR><TD>Nuclease-free water to</TD><TD>10 µl</TD></TR></TBODY></TABLE><TABLE class="customers"><TBODY><TR><TD>1) 92°C </TD><TD>2 min</TD></TR><TR><TD>2) 94°C </TD><TD>20 sec</TD></TR><TR><TD>3) 55°C</TD><TD>30 sec</TD></TR><TR><TD>4) 70°C</TD><TD>1 min/kb</TD></TR><TR><TD>5) 2/30X</TD></TR><TR><TD>6) 70°C</TD><TD>5 min</TD></TR><TR><TD>7) 10°C </TD><TD>forever</TD></TR></TBODY></TABLE>

          Primers for amplification of composite part: 
          Fw: aaaaagaattcgcggccgcttc
          Rev: aaaaactgagcggccgctactag
          <DIV class="tab-pane fade" id="tab-item4"><DIV class="bootstrap-overrides">
                        For making a small 1% gel:
 <UL>Weigh out 0.5 g of agarose and mix it with 50 ml of 1x TAE buffer in a 100 ml Erlenmeyer flask.</UL><UL> Dissolve the agarose by bringing the mixture to the boiling point in a microwave oven, followed by mixing (by swirling the flask). Repeat the heating and mixing until all the agarose has dissolved. </UL><UL>Cool the agarose solution to ~50 o C by leaving it on the bench for ~20 min (or you may accelerate the cooling by applying cold water from the tap to the outside of the flask).  
</UL><UL>Using gloves, add 5 l GelRed (10 000x). Swirl the flask gently to mix, try to avoid bubbles.</UL><UL>Pour the gel carefully into the mold. Bubbles may be removed/punctured by using a pipette tip. </UL></DIV></DIV><DIV class="tab-pane fade " id="tab-item5"><DIV class="bootstrap-overrides"><A class="bodyURLcolor" href="https://static.igem.org/mediawiki/2017/8/85/T--UiOslo_Norway--miniPrep.pdf"> Protocol </A><B> Modifications: </B><UL>During the first attempt ethanol was not added to the PE buffer, which resulted in an unsuccessful miniprepl
</UL><UL>In the second attempt 72/4% ethanol was added, as opposed to the recommended 96-100%, resulting in a successful miniprep
</UL></DIV></DIV><DIV class="tab-pane fade " id="tab-item6"><DIV class="bootstrap-overrides"><A class="bodyURLcolor" href="https://static.igem.org/mediawiki/2017/8/85/T--UiOslo_Norway--miniPrep.pdf"> Protocol </A><B> Load cells into Cell: </B><OL><LI>Attach flow valve to big straight hole. Look at the nylon ball at he end and make sure it’s not distorted and misshapen. Make sure valve is closed. Attach dispenser + small tubing to the other hole (small and slanted).</LI><LI>Examine the piston to make sure the o-rings are not nicked or distorted. Push piston into cell to line.</LI><LI>Remove cap on other end.</LI><LI>Pour sample into cell. Fill completely (1 mL for small cell).</LI><LI>Put cap on the end. Firmly push the cap down – the best way is to use the heel of your hand and hit the cap hard until it’s completely seated.</LI><LI>Turn setup over. Put in Press with piston up.</LI><LI>Mini cell can be loaded in your hand, but the large cell is too heavy to load this way. There is a black 3 columned stand next to the Press (may be pushed back on the counter). Put the large cell upside down (piston down). Fill the cell the same way as the mini cell.</LI></OL><B> Run: </B><OL><LI>Turn the Ratio Selector to Down position and turn the Pressure Increase control fully counterclockwise. The press needs to go down enough for the cell and extended piston.</LI><LI>Pump on.</LI><LI>Pressure Increase clockwise to 800 (turn knob) for mini-cell and 1000 for large cell.</LI><LI>Pump off.</LI><LI>For mini-cell turn Ratio Selector to medium.</LI><LI>Put cell on the stand – make sure you can turn the flow valve on the cell, that it’s not blocked by anything. Make sure the cell is aligned properly so that the piston squarely strikes under the upper platen. Make sure the piston handles are perpendicular to the bar and it’s screws (if it’s not, as the piston is pushed down the handles will run into the screws and something will break). Swing the bar across the cell and make sure it is completely against the cell. If it’s not, the cell could pop off the stand when pressure is applied. If the screws on the bar get in the way – unscrew them enough to slide over cell and then tighten them).</LI><LI>Pump on.</LI><LI>When the pressure gets to 800, slowly release cells by tapping (not hitting) the flow valve (black handle) with a pen – cells must come out slowly, drop by drop (pump on still), 15 drops/min. Several labs feel that tapping with a pen gives a more consistent and reproducible release of the cells. The drop rate tends to increase near the end of the run. As you approach the end of the run, you may want to close the flow valve slightly by turning it clockwise before opening it again. Also, there might be air bubbles in the sample and these tend to squirt into the collection tube and if you aren’t careful where you hold your collection tube, you could lose your sample. Be very careful that the tubing is in your collection tube and not pointing towards your face. </LI><LI>Pump off.</LI><LI>Turn the Ratio Selector down.</LI><LI>If you are done for the day, turn the Pressure Increase control fully counterclockwise. </LI><LI>Pump on.</LI><LI>Wash cell with H<SUB>2</SUB>O.</LI></OL></DIV></DIV><DIV class="tab-pane fade" id="tab-item7"><DIV class="bootstrap-overrides-with-pictures"><DIV class="modelling-content">

                                This experiment was conducted with the supervision of <A href="http://www.mn.uio.no/ibv/english/people/aca/awrobel/">Agnieszka Wrobel</A>. This procedure was done 2 times (First Experiment and Second Experiment). <B>Materials</B><UL><LI> pASK-IBA3 sfGFP (Amp100) </LI><LI> BL21 (DE3) Gold competent cells (50µl per sample) </LI><LI> 1.5mL Microtubes </LI><LI> SOC Media (950µL per sample) </LI><LI> Petri plates w/ LB agar and ampicillin (100µg/ml) (2 per sample) </LI><LI> Following buffers were made (and sterilized): </LI><UL><LI> Running buffer (per 1 L) (40 mM sodium phosphate (7,11g), 400 mM NaCl (23,4g), 20 mM imidazole (1,36g), pH 8.0 – made 2L </LI><LI> Elution buffer (per 1 l)(40 mM sodium phosphate (7,11g), 400 mM NaCl (23,4g), 500 mM imidazole (34g), pH 8.0 – made 1L </LI><LI> 1 M MgCl2 – used 20µl </LI><LI> 1M MnCl2 - used 20 µL </LI><LI> LB medium – 2L </LI></UL></UL><B>Equipment</B><UL><LI>Floating Foam Tube Rack </LI><LI>Ice &amp; ice bucket </LI><LI>Lab Timer </LI><LI>42°C water bath </LI><LI>37°C incubator </LI><LI>Sterile spreader or glass beads </LI><LI>Pipettes and tips (10µL, 20µL, 200µL, 1000µL) </LI><LI>Microcentrifuge </LI><LI>Pipetboy controller </LI><LI>ÄKTA start (chromatography) </LI><LI>Frac30 fraction collector </LI><LI>French Pressure Cell </LI></UL><H3>Method</H3><B>Day 1</B>
The transformation was done as given in the iGEM protocol for transformation (<A href="http://parts.igem.org/Help:Protocols/Transformation"> Single Tube Transformation Protocol </A>) with some modifications:
<UL><LI>The cells that we used were <I> E. coli </I> BL21 Gold </LI><LI>The plasmid that we used to transform the cells was pASK-IBA3 sfGFP (Amp100) (glycerol stock 709) </LI><LI>This plasmid was not resuspended as it was already dissolved in water. </LI><LI>The incubation after transformation was done at 220rpm. </LI><LI>We did not make the overnight culture and did not run the PCR (points 15. and 16. In the protocol). </LI><LI>We let the cells grow on plates at 37ºC until the next day (19 plates in total). </LI></UL><B> Day 2 </B><OL><LI>We made 10 ml of an overnight culture of E.coli BL21 Gold (pASK-IBA3 sfGFP) Amp100 from each petri plate. </LI><LI>LB medium for the next day was already prepared by the researches at the laboratory (2L). </LI></OL><B> Day 3 </B><OL><LI>We inoculated 800 mL of LB/Amp100 (100µg/ml) with 5 mL of the overnight culture of E.coli BL21 (DE3) Gold (pASK-IBA3 sfGFP) Amp100 (x 2 bottles) </LI><LI>We grew the cells until OD600=0.5 at 37ºC. This took us 3 hrs. </LI><LI>We prepared 12% SDS-PAGE gel. </LI><LI>Induce the cells with AHTC 0.2 µg/mL  </LI><LI>Then we grew cells for another 2 hours at 30ºC </LI><LI>We then spun the cells: 4500 rpm, 20 min, RT </LI><LI>The pellet was resuspended in 20 mL running buffer </LI><LI>The cells were frozen at -80ºC. </LI></OL><B> Day 4 </B><OL><LI>Cells in the tubes were thawed. </LI><LI>Protease inhibitor was added (1 tablet, thermos scientific – prod. 88266) </LI><LI>1 mM MgCl2 and 1 mM MnCl2 was added as well (20 µL of 1M stock/ per 20 mL) </LI><LI>20 µL of DNaseI solution (10mg/mL)  was added to each tube (10 µg/mL) </LI><LI>200 µL of Lysozyme solution (10mg/mL) was added to each tube (0.1 mg/mL) </LI><LI>French press: The samples were kept on ice at all times. We took out 30 µL of the protein sample to run it on a 12% SDS-Page gel. (<A href="#tab-item6"> Protocol for French Press </A> ) </LI><LI>Ultracentrifuged 20 000 g, 35 min, 4 C. Balance tubes.  </LI><LI>For affinity chromatography we used supernatant. We took out 30 µL of the protein sample to run it on a 12% SDS-Page gel. </LI><LI>Filter sterilize the supernatant through the 0.2 uM filter before applying to the column. </LI><LI>Affinity chromatography: </LI><UL><LI><B> Running buffer: </B></LI><UL><LI> 40 mM sodium phosphate </LI><LI> 400 mM NaCl </LI><LI> 20 mM imidazole </LI><LI> pH 8.0 </LI></UL><LI><B> Elution buffer: </B></LI><UL><LI> 40 mM sodium phosphate </LI><LI> 400 mM NaCl </LI><LI> 500 mM imidazole </LI><LI> pH 8.0 </LI></UL><LI> We first prepared the column His-trap-FF 5 mL. Then, the machine (ÄKTA system, GE Healthcare Life Sciences) for purification was prepared. </LI><LI>Settings (UNICORN start 1.0) for sfGFP purification. Variable list. </LI><LI>The machine was then switched on and the affinity chromatography was conducted using UNICORN Control Software. </LI></UL><LI>After this, 30 µL of protein sample was taken before dialysis to run it on a 12% gel (SDS-PAGE). </LI><LI>Dialysis was performed with 1L of 1xPBS buffer O/N at 4°C for 48 hours. We used water to wet the membrane tubing before starting the dialysis. </LI></OL>
Solutions prepared for dialysis. <B> Day 5 </B><OL><LI>We took 30 µL of our protein after dialysis for SDS-PAGE. </LI><LI>Concentrating the protein: </LI><UL><LI>Vivaspin Protein Concentrator Spin Columns that we picked had molecular weight cut off 15 000 kDa since the size of our protein was 27 kDa. </LI><LI>The columns were centrifuged for 15 min, 4000 rpm, 4°C. There was precipitation. </LI><LI>Then we continued with the centrifugation until we had following final volumes: </LI><UL><LI>Column nr. 1: 1 mL (4x15min) </LI><LI>Column nr. 2: 1.9 mL (5x15min) </LI></UL></UL><LI>Measurement of the protein concentration: </LI><UL><LI>The protein was centrifuged at max speed, 5 min (each tube). </LI><LI>We made 50 µL aliquots into Eppendorf tubes. 30 µL of our protein was taken for running on the gel. </LI><LI>We had three different batches (1 from column nr. 1 and batch nr. 2 and nr. 3 from column nr. 2): </LI><UL><LI>Batch nr. 1: 1 mL of protein solution, added 166ul of 60% glycerol </LI><LI>Batch nr. 2: 1 mL of protein solution, added 166ul of 60% glycerol </LI><LI>Batch nr. 3:  877 µl of protein solution, added 146ul of 60% glycerol </LI></UL></UL><LI>We did not have liquid nitrogen at that moment so we just froze the samples at -80ºC. </LI></OL> 

The entire procedure was repeated (Second Experiment) once more with some changes: <UL><LI>We used 6L of the running buffer. </LI><LI>Dialysis was done conducted. </LI><LI>Aliquots in the end were not made. The samples were stored in cold room at 4ºC in test tubes. </LI><LI>SDS-PAGE was not conducted either. </LI></UL></DIV><DIV class="modelling-picture"><FIGURE><FIGCAPTION>Solutions prepared for dialysis</FIGCAPTION></FIGURE></DIV></DIV></DIV><DIV class="tab-pane fade" id="tab-item8"><DIV class="bootstrap-overrides"><H2>Growth protocol: yeast with GFP strain 680 (nmt1-GFP-ppk18)</H2>

1. Wake the cells by plating them and incubate on 32 Celsius for 48 hours
        -the yeast cells are stored and hibernating at -80 celcius
2. Make a liquid culture with thiamine to supress the expression of GFP and let the cells further incubate on 25 Celsius for 24 hours on shaker.
        - 50 mL EMM
        - 25 uL thimamine 
3. Measure OD of the culture and check for contamination under a microscope.
        -dilute the culture morning and night with more medium to make sure the cells don't starve. 
4. Spinn the culture to extract the thiamine from the solution, make a new liquid culture and incubate on 25 Celsius for another 24 hours on shaker.
        -this will induce the expression of GFP
5. Measure OD of the culture and check for contamination again.
        -dilute the culture to be at around 0.05 OD as this gives the best results with the laser.  
                        </DIV></DIV><DIV class="tab-pane fade " id="tab-item9"><DIV class="bootstrap-overrides">
                            96-Well Transformation Protocol: <A class="bodyURLcolor" href="http://parts.igem.org/Help:Protocols/Transformation#96-Well_Transformation_Protocol
"> Protocol </A>

Plate reader protocol:<A class="bodyURLcolor" href="https://static.igem.org/mediawiki/2017/0/0b/T--UiOslo_Norway--InterLab_2017_Plate_Reader_Protocol.pdf">Protocol</A></DIV></DIV><DIV class="tab-pane fade " id="tab-item10"><DIV class="bootstrap-overrides">
                           This protocol for this collaborative project composed of four different days in which the methodology applied for each day has described in bellow. This Experiment has been done in BSL2 lab facility under supervision of <A href="http://www.mn.uio.no/ibv/english/people/aca/awrobel/">Agnieszka Wrobel</A>. 


<H3>Materials:</H3><UL><LI>Plasmid pLA230 </LI><LI>Plasmid MP6 </LI><LI>Top10 chemically competent cells (100 µl frozen stock) </LI><LI>LB Media (950µL per sample) </LI><LI>Kanamycin:  50 µg/ml (final concentration, recommended)</LI><LI>Chloramphenicol:  25  µg/ml (final concentration, recomended)</LI><LI>Petri plates LB agar (+Kanamycin)</LI><LI>Petri plates LB agar (+Chloramphenicol)</LI><LI>Petri plates LB agar (+Ampicilin)</LI><LI>LB medium (autoclaved)</LI><LI>D(+) Glucose. (20 mM final conc.)  VWR</LI><LI>L(+) Arabinose. (20 mM final conc.)  Sigma Aldrich</LI><LI>Yeast extract </LI><LI>NaCl</LI><LI>Tryptone</LI><LI>NaOH</LI><LI>EtOH</LI><LI>ddH2O</LI><LI>Microtubes (1.5 ml)</LI><LI>Incubator (37 degree celcius)</LI><LI>Fume Hood</LI><LI>Spreader</LI><LI>Erlenmyer glass Flasks (50 ml)</LI><LI>Plastic tubes (15 ml)</LI><LI>A high resolution Camera</LI><LI>Ice (and ice bucket) </LI><LI>Timer </LI><LI>Pipettes and tips (10µL- 1000µL) </LI><LI>Samplers (2.5 – 1000)</LI><LI>Autoclaved glass tubes/Erlenmeyer</LI></UL><H3> Methodology </H3><H4> Day 1: Transformation of  pLA230 and MP6 into TOP 10  </H4>
The transformation was performed mainly according to  Single Tube Transformation Protocol ) but with minor modifications as follows:

<UL><LI>Thawing chemically frozen competent cells TOP10 on ice (~5 min after) </LI><LI>Adding 1/2 µL of vector (pMP6) and vector, to the 50/100  µL of thawed cells on ice separately.</LI><LI>Incubation on Ice:  30 min.  </LI><LI>Heat-Shock:  30 sec   42 degree Celsius</LI><LI>Incubation on Ice: 5 min.</LI><LI>Immediately adding liquid LB medium to tubes (900  µL)</LI><LI>Overnight (O/N) incubation (16 hours): 37ºC.</LI></UL><H4>Day 2:     Inoculation (with and without arabinose)</H4><UL><LI>Making two working stock solution from D-Glucose and L-Arabinose (200 mM)</LI><LI>After 16 hours O/N:  splitting into Erlenmeyer 50  and adding glucose in one bottle and L-arabinose in the other (final concentration of 20mM in the culture).</LI><LI>Incubation with agitation (800 rpm)  24 hours.</LI></UL><H4>Day 3:     Making serial dilutions and plating on LA (+/- Antibiotic)</H4><UL><LI>Making serial dilutions: For tests (10^0 μ -0-10 μ -3) and for controls on LA + Glucose plates (10^ -5 - 5 ^ -8). </LI><LI>Plating (100 μl dilution per plate) on LA + Glucose + Amp plates (10 μ -0-10 μ -3) and on LA + Glucose plates (control) (10 -5 -5 ^ -8). </LI><LI>Incubation 37 degree Celsius (Overnight)  </LI></UL><H4>Day 4:     Counting the number of colonies   </H4><UL><LI>Counting the cells (under fume hood)</LI><LI>Documenting (taking photos from plates) </LI></UL></DIV></DIV><DIV class="tab-pane fade " id="tab-item11"><DIV class="bootstrap-overrides-with-pictures"><DIV class="modelling-content">
                                For our light source we used a blue LED lamp with the properties 20mA and 3.2V, a resistance of 1kΩ and a PHYWE power supply that allowed us to vary our voltage between 0-30V. The LED and the resistance were mounted to a circuit board using a soldering iron, before the wires were soldered to each side of the board. 
                                We made four different light sources:
                                <UL><LI>One single blue LED </LI><LI>A double blue LED </LI><LI>One single green LED </LI><LI>One single red LED </LI></UL></DIV><DIV class="modelling-picture"><FIGURE><FIGCAPTION> Hilde (physicist) making one of the LED circuitboards </FIGCAPTION></FIGURE><FIGURE><FIGCAPTION> One of the blue LED circuitboards. We wanted to make a double to test if we could increase the intensity transmitted on the sample </FIGCAPTION></FIGURE></DIV></DIV></DIV><DIV class="tab-pane fade" id="tab-item12"><DIV class="bootstrap-overrides-with-pictures"><DIV class="modelling-content">

                                At the very beginning of our process of making a biolaser we startet by checking that our LED circuit could make the sfGFP sample fluoresce. After verifying that it indeed could, we began the process of mounting our selected equipment to a rail. We wanted to optimize the light hitting our chosen gain medium (being the sfGFP and later yeast), which would hopefully result in that only a small part of the light was lost before hitting the medium. We continued with the mounting and added one and one element to our setup until we had the final setup with four lenses, two color filters, two mirrors, a spectrometer and a CCD-camera. The method for this was adding one element, check with a white paper where the light hit, before adding a second element and repeat the process.
                            </DIV><DIV class="modelling-picture"><FIGURE><FIGCAPTION> Elisabeth (physicist) handling some frozen sfGFP. We wanted to have the sample in a container that would not interfere as much with the light. </FIGCAPTION></FIGURE><FIGURE><FIGCAPTION>Tubes of the sfGFP, kept on ice while we had them in the physics lab to make sure they would keep cool </FIGCAPTION></FIGURE><FIGURE><FIGCAPTION> One of the set ups where we tried to send the light in on the sample, diagonal to the optical path. </FIGCAPTION></FIGURE><FIGURE><FIGCAPTION> Another angle of the same set up as mentioned above, sending the light in diagonal at the sample to the optical path </FIGCAPTION></FIGURE><FIGURE><FIGCAPTION> One of the inital set ups before we got the mirrors and filters from Thorlabs </FIGCAPTION></FIGURE><FIGURE><FIGCAPTION> Overview of our final set up from the end where you can see the spectrometer, CCD camera and the corresponding computer and lenses. </FIGCAPTION></FIGURE><FIGURE><FIGCAPTION> How we aligned the mirrors, with the sample and lenses in the final set up. </FIGCAPTION></FIGURE><FIGURE><FIGCAPTION> A closer look at our final set up wiht focus on the sample and mirrors. </FIGCAPTION></FIGURE><FIGURE><FIGCAPTION> Our LED, filter and lenses with our sample and mirrors behind in our final set up. </FIGCAPTION></FIGURE><FIGURE><FIGCAPTION> Another look at our final set up taken beside the spectrometer</FIGCAPTION></FIGURE></DIV></DIV></DIV><DIV class="footer-bootstrap-overrides"><DIV class="findUs"><DIV class="map"><P><A href="https://www.google.no/maps/place/Problemveien+7,+0313+Oslo/@59.9401331,10.7180943,17z/data=!3m1!4b1!4m5!3m4!1s0x46416de7a8e9c1a1:0x917a1b33171c58b1!8m2!3d59.9401331!4d10.720283">Problemveien 7, Oslo, Norway </A></P></DIV><DIV class="mail"><P><A href="mailto:uioslonorway@gmail.com">uioslonorway@gmail.com</A></P></DIV></DIV></DIV></TABLE></UL></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></BODY></HTML>