<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_NEFU_China_Description skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><DIV id="top_title"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:NEFU China/Description</SPAN></H1></DIV><DIV id="HQ_page"><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><DIV id="nav"><UL class="navList"><LI><A href="https://2016.igem.org/Team:NEFU_China#">Home</A></LI><LI><A href="https://2016.igem.org/Team:NEFU_China/Team">Team</A><DIV class="secondMenu"><A href="https://2016.igem.org/Team:NEFU_China/Team">Member</A><A href="https://2016.igem.org/Team:NEFU_China/Teams">Collaboration</A><A href="https://2016.igem.org/Team:NEFU_China/Team">Attribution</A></DIV></LI><LI><A href="https://2016.igem.org/Team:NEFU_China/Description">Project</A><DIV class="secondMenu"><A href="https://2016.igem.org/Team:NEFU_China/Description">Description</A><A href="https://2016.igem.org/Team:NEFU_China/Description">Design</A><A href="https://2016.igem.org/Team:NEFU_China/Description">Experiment&amp;Protocol</A><A href="https://2016.igem.org/Team:NEFU_China/Description">Demonstrate Hardware</A><A href="https://2016.igem.org/Team:NEFU_China/Description">Judging criteria</A></DIV></LI><LI><A href="https://2016.igem.org/Team:NEFU_China/Parts">Parts</A></LI><LI><A href="https://2016.igem.org/Team:NEFU_China/Results">Results</A><DIV class="secondMenu"><A href="https://2016.igem.org/Team:NEFU_China/Results">AMB-1</A><A href="https://2016.igem.org/Team:NEFU_China/Results"><I>E. coli</I></A><A href="https://2016.igem.org/Team:NEFU_China/Results">Purification</A><A href="https://2016.igem.org/Team:NEFU_China/Results">Modeling &amp; Proof</A><A href="https://2016.igem.org/Team:NEFU_China/Results">Future application</A></DIV></LI><LI><A href="https://2016.igem.org/Team:NEFU_China/Notebook">Notebook</A></LI><LI><A href="https://2016.igem.org/Team:NEFU_China/Human_Practices">Human practice</A><DIV class="secondMenu"><A href="https://2016.igem.org/Team:NEFU_China/Human_Practices">Safety</A><A href="https://2016.igem.org/Team:NEFU_China/Human_Practices">Education</A><A href="https://2016.igem.org/Team:NEFU_China/Human_Practices">Questionnaire</A><A href="https://2016.igem.org/Team:NEFU_China/Human_Practices">iGEMer</A><A href="https://2016.igem.org/Team:NEFU_China/Human_Practices">Community</A></DIV></LI><LI style="display: none;"><A href="http://www.yaolibio.com">BioDog</A></LI></UL></DIV><DIV class="headPic" style="display:"><DIV class="wt1200"><DIV class="miao"><UL><LI><A href="#Business">Description</A></LI><LI><A href="#Sos">Design</A></LI><LI><A href="#Plan">Experiment &amp; Protocol</A></LI><LI><A href="#wujin">Demonstrate &amp; Hardware</A></LI><LI><A href="#shiyan">Judging criteria</A></LI></UL></DIV></DIV></DIV><DIV><ARTICLE class="content wt960"><H3 id="our-business-idea" class="mt20">Description</H3><DIV class="row"><P>
          	Protein purification is important to both biopharmaceutical production and scientific research. Producing recombinant	 proteins with high purity is essential for protein drug-based therapies, protein structural analyses and functional studies. Most currently used protein purification techniques, such as commercial resins conjugated by various ligands, different types of chromatography and ultracentrifugation, require expensive investment, special device or tedious procedure. Meanwhile, they have different flaws in getting purified proteins, such as low specificity, long-term operating procedures and harsh conditions to sacrifice protein activity; thus, many of them, even with their combinations, cannot meet the required protein purity and quality for many applications. For instance, affinity chromatography is reputed for its high affinity, but tightly bound proteins typically require relatively extreme eluting conditions to disassociate from affinity columns; ultracentrifugation or size exclusion separates samples based on molecular weight difference, but they can co-purify other proteins and large molecules of lipid or nuclear acids. Thus, developing fast, easy to operate and economic approaches to purify recombinant proteins with high quality is urgently needed.
        </P><P style="text-align:center;">
         	Fig.1 Purification progress by using the Nickel column
        </P><P>
			Our team developed a novel protein purification approach that eliminates most weaknesses of other techniques. Our method is based on the highly specific recognition and covalent conjugation between Spytag (a peptide with 13 amino acids) and Spycatcher (a peptide with 116 amino acids). We also take the advantage of a magnetic bacterium AMB-1 that can produce magnet particles, also called Magnetosome, covered by a bilayer phospholipid membrane, in which Mms13 protein is anchored. We generate Spycatcher-Mms13 and express this fusion protein in AMB-1 bacteria to make Spycatcher linked to the Magnetosome. A protein of interest with a Spytag at its N- or C-terminus is expressed in <I>E. coli</I> and present in bacterial lysate. The Spycatcher-Mms13-linked Magnetosome can specifically and covalently conjugate to the Spytag-tagged protein in the bacterial lysate and they can be simply co-purified by a magnet. Due to covalent linkage, stringent washing can be used to achieve high purity of the recombinant protein that is subsequently released by cleavage of a designated protease.
		</P><P>
			Overall, our novel approach for protein purification is reliable and highly specific. It provides a better solution to reach high protein purity and quality by remedying many drawbacks of currently used methods.
		</P><H3>Design</H3><DIV class="row"><P>
        	Magnetic bacteria <I>Magnetospirillum magneticum</I> strain AMB-1 is a gram-negative, facultative anaerobic bacterium. It produces a unique organelle known as Magnetosome using iron and other metal ions. Magnetosome consists of uniform nano-sized magnetic particles (50 to 100 nm) arranged as chains in cytoplasm and embraced by a stable lipid bilayer membrane, and exhibits strong ferrimagnetism.
  		 </P><P style="text-align:center;">
			 Fig.2 Transmission electron microscope image of (A) AMB-1 and (B) Magnetosome
		 </P><P>
			Magnetosome has been used in various biomedical applications because it is easy to handle and can be used to separate target molecules from reaction mixes. Several proteins are identified to tightly bind to the membrane of Magnetosome, such as MagA, Mms16, and Mms13. Among them, Mms13 has the highest affinity to the magnetic core, and its anchoring efficiency is also better than other Mms proteins. Thus, a protein of interest can be expressed on the Magnetosome when fused with Mms13. The magnetic particles can be easily extracted and purified from bacterial lysate of AMB-1 transformant using a magnet.
		 </P><P style="text-align:center;">
			 Fig.3 Proteins on the Magnetosomeâ€™s membrane and AMB-1 modification by transformed with Mms13 protein.
		 </P><P>
		    Choosing an appropriate tag is crucial for detecting and purifying a recombinant protein. However, the interaction of tags with corresponding ligands is typically reversible and mostly has low affinity. Streptococcus pyogenes fibro-nectin-binding protein FbaB contains a domain with a spontaneous isopeptide bond between Lys31 and Asp117. This domain can be splitted into two fragments: the small one (13 amino acids) is named as Spytag, while the big one (116 amino acids) is Spycatcher. When they meet together, the two parts can specifically associate and spontaneously form an isopeptide bond. The reaction can take place in diverse conditions, even in relatively extreme pH and temperatures.
         </P><P style="text-align:center;">
		 	Fig.4 Highly specific recognition and covalent conjugation with Spychatcher and Spytag.
		 </P><P>
        	We used this feature of Spytag and Spycatcher to design a handy tool for protein purification. In the magnetic bacterium AMB-1, Spycatcher-Mms13 fusion protein is expressed and the Mms13 part can tightly associate with the Magnetosome. As a result, the Spycatcher in the fusion protein is anchored in the Magnetosome, which can be isolated from AMB-1 bacterial through sonication and centrifugation. Due to fast growing and easy genetic engineering of <I>E. coli</I> we utilize it to produce recombinant proteins of interest with a Spytag at either the N- or C- terminus. To easily release a captured recombinant protein, we add a cleaving site for the tobacco etch virus (TEV) protease between the protein of interest and Spytag. Now the isolated magnetosome linked by Spycatcher can be used to specifically bind to the Spytag tagged recombinant protein in bacterial lysate of <I>E. coli</I>. After Spytag and Spycatcher form an irreversible covalent bond via spontaneous isopeptide linkage, they can be brought down together with the Magnetosome in a magnetic field. Extensive and stringent washing can be applied to remove proteins of nonspecific binding and the protein of interest can be released by cleavage of TEV protease.
        </P><P style="text-align:center;">
			Fig.5 Steps to purify protein in a new way
		</P><P>
			One caveat in the design above is that the TEV protease is still present in the final elution of the recombinant protein. The following two strategies can be used to circumvent this problem.
		</P><P>
			First, in the design shown above, we can express a TEV protease-Mms13 fusion protein in AMB-1; thus, the TEV protease can be easily removed after it fulfills its task. Second, we can use a self-cleavable sequence, intein, to replace the TEV site. In this case, the recombinant protein can be released by DTT(DL-Dithiothreitol)-mediated cleavage.
		</P><H2>Experiment&amp;Protocol</H2><TABLE border="1" align="center"><TBODY><TR><TD style="text-align:center;" colspan="2">AMB-1</TD><TD style="text-align:center;" colspan="2"><I>E. coli</I></TD></TR><TR><TD style="text-align:center;">Incubation</TD><TD style="text-align:center;">Wet-lab</TD><TD style="text-align:center;">Prokaryotic Expression</TD><TD style="text-align:center;">Protein Purification</TD></TR><TR><TD style="text-align:center;">StrainMediumBacteria recoveryBacteria incubationCollect cellsUltracentrifugationMagnetosomes extractionExam the purified MagnetosomeStorage and sterilization of purified Magnetosome</TD><TD style="text-align:center;">Electrocompetent cellsTransformation by ElecroporationTransformants incubation</TD><TD style="text-align:center;">MediumPCRGel extractionPCR purificationRestriction digesAgarose gel ElectrophoresisLigationTransformationColony PCRIdentificationSequence</TD><TD style="text-align:center;">GST protein Expression and purificationMagneGST protein purificationBradford protein assay SDS-PAGE</TD></TR></TBODY></TABLE><P><B>Strain</B></P><P>
				The <I>Magnetospirillum magneticum</I> strain AMB-1(ATCC 700264) was purchased from ATCC and keep in -80â„ƒ.
			</P><P><B>Medium</B></P><P>
				For the enrichment of the AMB-1 strain, we use the medium contained the following ingredients:
			</P><P align="center">Magnetic Spirillum Growth Medium (1 L)</P><P align="center">ï¼ˆpH=6.7Â±0.1ï¼‰</P><TABLE border="1" align="center"><TBODY><TR><TD style="text-align:center;">Component</TD><TD style="text-align:center;">Mass</TD></TR><TR><TD style="text-align:center;">Distilled water</TD><TD style="text-align:center;">1 l</TD></TR><TR><TD style="text-align:center;">Wolfe's Vitamin Solution</TD><TD style="text-align:center;">10 ml</TD></TR><TR><TD style="text-align:center;">Wolfe's Mineral Solution</TD><TD style="text-align:center;">5 ml</TD></TR><TR><TD style="text-align:center;">Ferric Quinate</TD><TD style="text-align:center;">5 ml</TD></TR><TR><TD style="text-align:center;">KH<SUB>2</SUB>PO<SUB>4</SUB></TD><TD style="text-align:center;">0.68 g</TD></TR><TR><TD style="text-align:center;">NaNO<SUB>3</SUB></TD><TD style="text-align:center;">0.12 g</TD></TR><TR><TD style="text-align:center;">Tartaric acid</TD><TD style="text-align:center;">0.37 g</TD></TR><TR><TD style="text-align:center;">Succinic acid</TD><TD style="text-align:center;">0.37 g</TD></TR><TR><TD style="text-align:center;">Sodium acetate</TD><TD style="text-align:center;">0.05 g</TD></TR><TR><TD style="text-align:center;">0.1% Resazurin</TD><TD style="text-align:center;">0.45 ml</TD></TR><TR><TD style="text-align:center;">Ascorbic acid</TD><TD style="text-align:center;">0.035 g</TD></TR></TBODY></TABLE><P style="text-align:center;">
				Autoclave medium at 121â„ƒ for 15 minutes.
			</P><P>Wolfe's Vitamin Solution</P><TABLE border="1" align="center"><TBODY><TR><TD style="text-align:center;">Component</TD><TD style="text-align:center;">Mass</TD></TR><TR><TD style="text-align:center;">Biotin</TD><TD style="text-align:center;">2 mg</TD></TR><TR><TD style="text-align:center;">Folic acid</TD><TD style="text-align:center;">2 mg</TD></TR><TR><TD style="text-align:center;">Pyridoxine hydrochloride</TD><TD style="text-align:center;">10 mg</TD></TR><TR><TD style="text-align:center;">Thiamine . HCl</TD><TD style="text-align:center;">5 mg</TD></TR><TR><TD style="text-align:center;">Riboflavin</TD><TD style="text-align:center;">5 mg</TD></TR><TR><TD style="text-align:center;">Nicotinic acid</TD><TD style="text-align:center;">5 mg</TD></TR><TR><TD style="text-align:center;">Calcium D-(+)-pantothenate</TD><TD style="text-align:center;">5 mg</TD></TR><TR><TD style="text-align:center;">Vitamin B<SUB>12</SUB></TD><TD style="text-align:center;">0.1 g</TD></TR><TR><TD style="text-align:center;">p-Aminobenzoic acid</TD><TD style="text-align:center;">5 mg</TD></TR><TR><TD style="text-align:center;">Thioctic acid</TD><TD style="text-align:center;">5 mg</TD></TR><TR><TD style="text-align:center;">Distilled water</TD><TD style="text-align:center;">1 l</TD></TR></TBODY></TABLE><P style="text-align:center;">
				Filtered through 0.22 Î¼l filter
			</P><P>Wolfe's Mineral Solution</P><TABLE border="1" align="center"><TBODY><TR><TD style="text-align:center;">Component</TD><TD style="text-align:center;">Mass</TD></TR><TR><TD style="text-align:center;">Nitrilotriacetic acid</TD><TD style="text-align:center;">1.5 g</TD></TR><TR><TD style="text-align:center;">MgSO<SUB>4</SUB>Â·7H<SUB>2</SUB>O</TD><TD style="text-align:center;">3 g</TD></TR><TR><TD style="text-align:center;">MnSO<SUB>4</SUB>Â·H<SUB>2</SUB>O</TD><TD style="text-align:center;">0.5 g</TD></TR><TR><TD style="text-align:center;">NaCl</TD><TD style="text-align:center;">1 g</TD></TR><TR><TD style="text-align:center;">FeSO<SUB>4</SUB>Â·7H<SUB>2</SUB>O</TD><TD style="text-align:center;">0.1 g</TD></TR><TR><TD style="text-align:center;">CoCl<SUB>2</SUB>Â·6H<SUB>2</SUB>O</TD><TD style="text-align:center;">0.1 g</TD></TR><TR><TD style="text-align:center;">CaCl<SUB>2</SUB></TD><TD style="text-align:center;">0.1 g</TD></TR><TR><TD style="text-align:center;">ZnSO<SUB>4</SUB>Â·7H<SUB>2</SUB>O</TD><TD style="text-align:center;">0.1 g</TD></TR><TR><TD style="text-align:center;">CuSO<SUB>4</SUB>Â·5H<SUB>2</SUB>O</TD><TD style="text-align:center;">0.01 g</TD></TR><TR><TD style="text-align:center;">AlK(SO<SUB>4</SUB>)<SUB>2</SUB>Â·12H<SUB>2</SUB>O</TD><TD style="text-align:center;">0.01 g</TD></TR><TR><TD style="text-align:center;">H<SUB>3</SUB>BO<SUB>3</SUB></TD><TD style="text-align:center;">0.01 g</TD></TR><TR><TD style="text-align:center;">Na<SUB>2</SUB>MoO<SUB>4</SUB>Â·2H<SUB>2</SUB>O</TD><TD style="text-align:center;">0.01 g</TD></TR><TR><TD style="text-align:center;">Distilled water</TD><TD style="text-align:center;">1 l</TD></TR></TBODY></TABLE><P style="text-align:center;">
				Autoclave medium at 121â„ƒ for 15 minutes.
			</P><P><B>Bacteria recovery</B></P><P>1.Keep?vial?frozen?until?ready?to?use,?then?thaw.?Thaw?contents?of?the?vial?in?a?37 â„ƒ?water?bath?with gentle?agitation. </P><P>2.Inoculate?a?single?tube?of?MSGM?with?the?entire?contents?of?the?thawed?vial.?Be?sure?the?entiretube?is?filled?with?medium?to?achieve?microaerophilic?conditions.</P><P>3.Incubate?at?30 Â°C?for 7 days.</P><P>4.When?broth?becomes?cloudy?and?a?small?gray?sediment?is?visible?at?the?bottom?of?the?tube,?cells?can be?harvested.?Divide?contents?of?
the?tube?into?two?tubes?and?spin?in?a?tabletop?centrifuge. </P><P>5.Discard?the?supernatant?liquid.?Do?not?bubble?or?shake?cells.?Resuspend?cells?in?a?125?mL?flask.</P><P>6.Incubate?as?directed?above.</P><P><B>Bacteria incubation</B></P><P>1. Add 1 ml AMB-1 bacterium solution into a 100 ml serum bottle supplemented with 75 ml MSGM.</P><P>2. Incubate at 30â„ƒ under microaerophilic conditions.</P><P><B>Cells collection</B></P><P>1. Make sure bacterium solution in the logarithmic growth phase.</P><P>2. Transform it into 50 ml tubes, centrifugation at 8000 r for 5 min.</P><P>3. Discard the supernatant, wash the cells with 1 ml PBS, centrifugation at 8000 r for 5 min.</P><P>4. Repeat step 3.</P><P>5. Resuspend with 1 ml sterile water.</P><P><B>Ultracentrifugation</B></P><P>1. Harvest cells by centrifugation at 10,000 Ã—g and washed twice with 100 mM phosphate buffer (pH 7.0). </P><P>2. Resuspend cells in 100 ml of the same buffer. </P><P>3. Disrupt the cells by sonication at 4â„ƒ for 5 s. </P><P>4. Remove cell debris and unbroken cells by centrifugation for 20 min at 10,000Ã—g at 4 â„ƒ. </P><P>5. The supernatant was used as the crude enzyme.</P><P><B>Magnetosomes extraction</B></P><P>1. AMB-1 strain is concentrated by centrifugation at 10,000g for 20 min at 4â„ƒ.</P><P>2. Put AMB-1 suspended in 10 ml of phosphate buffer saline (PBS; pH 7.4), disrupted by low level ultrasonication.</P><P>3. Use the optical microscope to examine the effect of disruption.</P><P>4. Collect Magnetosomes from the bottom of a tube by multifunctional magnetic corpuscle separator which can produce an inhomogeneous magnetic field (see device). </P><P>5. Remove the supernatant.</P><P>6. Wash Magnetosomes eight to ten times with PBS while agitating via low level ultrasonication.</P><P><B>Exam the purified Magnetosomes</B></P><P>The purified Magnetosomes was suspended in 20 Î¼l PBS and put onto 300-mesh carbon-coated copper grids. Samples were observed through TEMï¼ˆTransmission electron microscopyï¼‰ at an accelerating voltage of 300 kV.</P><P><B>Storage and sterilization of purified Magnetosomes</B></P><P>The purified Magnetosomes were lyophilized using a freeze drier for 20 h. The lyophilized Magnetosomes were stored at -20â„ƒ for further assay.</P><P><B>Electrocompetent cells (AMB-1)</B></P><P>1. Put 200 ml bacterium solution (in the logarithmic growth phase) into a falcon tube, ice bath for 10 min.</P><P>2. Centrifugation (8000 rpm, 30 min) at 4â„ƒ, discard supernatant.</P><P>3. Resuspend with 10 mM TES (pH 7.5), wash cells twice with 1 ml sterile water.</P><P>4. Centrifugation (3000 rpm, 12 min)</P><P>5. All the bacterium suspend in 2 ml TES, divide the bacterium solution into two 1.5 ml falcon tubes, keep in -40â„ƒ.</P><P><B>Transformation by elecroporation</B></P><P>1. Mix 40 Î¼l electrocompetent cells with 2 Î¼l plasmid DNA.</P><P>2. Carefully?transfer?the?cell?mix?into?a?chilled?cuvette?without?introducing?bubbles?and?make?sure?that?the cells?deposit
across?the?bottom?of?the?cuvette.</P><P>3. Electroporate?using?the?following?conditions?for?BTX?ECM?630?and?Bio-Rad?GenePulser?electroporators:?2 kV,?200 Î©,?25 Î¼F?
electric?pulse?in?a?0.2?cm?cuvette.</P><P>4. Immediately?add?500Î¼l MSGM medium to the?cuvette, harvest the cells by shake cultivation (100 rpm) at 27â„ƒ for one night.</P><P>5. Dilute the cells as appropriate then spread 250 Î¼l cells onto a pre-warmed MSGM plate.</P><P>6. Incubate the plates at 30â„ƒ for 1 to 2 days under anarobic condition.</P><P>7. Use isolated colonies to check the correct insertion.</P><P><B>Transformants incubation</B></P><P>The transformants were cultured in MSGM containing 50 Î¼g/ ml ampicillin. </P><P><B>Medium</B></P><P>For the enrichment of the <I>E. coli</I> strain, we use the medium containing the following ingredients:</P><P><B>LB medium (500 ml)</B></P><TABLE border="1" align="center"><TBODY><TR><TD style="text-align:center;">Component</TD><TD style="text-align:center;">Mass</TD></TR><TR><TD style="text-align:center;">Tryptone</TD><TD style="text-align:center;">5 g</TD></TR><TR><TD style="text-align:center;">MgSO<SUB>4</SUB>Â·7H<SUB>2</SUB>O</TD><TD style="text-align:center;">3 g</TD></TR><TR><TD style="text-align:center;">Yeast Extract</TD><TD style="text-align:center;">2.5 g</TD></TR><TR><TD style="text-align:center;">NaCl</TD><TD style="text-align:center;">5 g</TD></TR><TR><TD style="text-align:center;">Distilled water</TD><TD style="text-align:center;">500 ml</TD></TR></TBODY></TABLE><P><B>PCR</B></P><P>1. Set up a small box with ice, put DNA and polymerase mix into it before going into the Bio-rad S1000TM thermocycler cycler.</P><P>2. Add the following reagent to a PCR tube.(50 Î¼l).</P><TABLE border="1" align="center"><TBODY><TR><TD style="text-align:center;">2Ã— Taq Master Mix  (Enzyme)</TD><TD>25 Î¼l</TD></TR><TR><TD style="text-align:center;">Template (5ng/ Î¼l)</TD><TD>1 Î¼l</TD></TR><TR><TD style="text-align:center;">Forward Primer (10 Î¼M)</TD><TD>1 Î¼l</TD></TR><TR><TD style="text-align:center;">Reverse Primer (10 Î¼M)</TD><TD>1 Î¼l</TD></TR><TR><TD style="text-align:center;">ddH<SUB>2</SUB>O</TD><TD>22 Î¼l</TD></TR></TBODY></TABLE><P>3. Program the thermocycler as follows:</P><P style="text-align:center;">Temperature: Time</P><TABLE border="1" align="center"><TBODY><TR><TD>94â„ƒ</TD><TD style="text-align:center;">5 min</TD></TR><TR><TD>94â„ƒ</TD><TD style="text-align:center;">30 s</TD></TR><TR><TD>58â„ƒ</TD><TD style="text-align:center;">30 s</TD></TR><TR><TD>72â„ƒ</TD><TD style="text-align:center;">ï¼ˆ60 sï¼‰Ã— (1 kb)</TD></TR><TR><TD>72â„ƒ</TD><TD style="text-align:center;">10 min</TD></TR><TR><TD>4â„ƒ</TD><TD style="text-align:center;">forever</TD></TR></TBODY></TABLE><P><B>Gel Extraction</B></P><P>For the Gel Extraction we used TIANprep Midi Plasmid Kit according to manufacturerâ€™s instruction</P><P><B>PCR Purification</B></P><P>1. Add 5 volumes of Buffer PB to 1 volume of the PCR sample and mix.</P><P>2. If pH indicator has been added to Buffer PB, check that the color of the mixture is yellow.</P><P>3. Place a QIAquick spin column in a provided 2 ml collection tube.</P><P>4. To bind DNA, apply the sample to the QIAquick column and centrifuge for 30â€“60 s.</P><P>5. Discard flow-through. Place the QIAquick column back into the same tube.</P><P>6. To wash, add 0.75 ml Buffer PE to the QIAquick column and centrifuge for 30â€“60 s.</P><P>7. Discard flow-through and place the QIAquick column back in the same tube.</P><P>8. Centrifuge the column for an additional 1 min.</P><P>9. Place QIAquick column in a clean 1.5 ml microcentrifuge tube.</P><P>10. To elute DNA, add 50 Î¼l Buffer EB (10 mM TrisÂ·Cl, pH 8.5) or water (pH 7.0â€“8.5) to the center of the QIAquick membrane andcentrifuge the column for 1 min.</P><P><B>Restriction Digest</B></P><P>1. Add the following reagents to a tube (Total: 30 Î¼l).</P><TABLE border="1" align="center"><TBODY><TR><TD style="text-align:center;">DNA</TD><TD>500-1000 ng</TD></TR><TR><TD style="text-align:center;">Enzyme (NEB)</TD><TD style="text-align:center;">1 Î¼l</TD></TR><TR><TD style="text-align:center;">10Ã—Buffer</TD><TD style="text-align:center;">3 Î¼l</TD></TR></TBODY></TABLE><P style="text-align:center;">Fill the rest with water.</P><P>2. Pipette up and down thoroughly.</P><P>3. Put at 37â„ƒ for about 4 hours (according to different enzymes).</P><P><B>Agarose Gel Electrophoresis (0.7%)</B></P><P>1. Weigh 0.21 g of agarose.</P><P>2. Add 30 ml of TAE 1X. </P><P>3. Heat up until the solution is homogeneous.</P><P>4. Add 1 Î¼l Gelred into solution and mix well.</P><P>5. Put the solution into bed for polymerize, make sure &quot;comb&quot; is well placed and the solution is balanced.</P><P>6. 20 min later, use the gel.</P><P><B>Ligation (Total: 15Î¼l)</B></P><TABLE border="1" align="center"><TBODY><TR><TD style="text-align:center;">Vector</TD><TD rowspan="2">Varied from different desired  combinations of insert and backbone</TD></TR><TR><TD style="text-align:center;">Insert fragment</TD></TR><TR><TD style="text-align:center;">T4 ligase(NEB)</TD><TD style="text-align:center;">0.5 Î¼l</TD></TR><TR><TD style="text-align:center;">10Ã—Ligation Buffer</TD><TD style="text-align:center;">1.5 Î¼l</TD></TR></TBODY></TABLE><P style="text-align:center;">Include negative control reaction which contains no insert DNA. </P><P style="text-align:center;">Fill the rest with water, put at room temperature for 4 hours.</P><P><B>Transformationï¼š</B></P><P>1. Put Trans1-T1 Phage Resistant Chemically Competent Cells on ice for melting.</P><P>2. Add 50 Î¼l electrocompetent cells into tubes, mix with 5Î¼l ligation product thoroughly, incubate the cells on ice for 30 min.</P><P>3. Heat shock the cells for exactly 30 s at 42 â„ƒ water bath.</P><P>4. Place back on ice for 2 min.</P><P>5. Add 250 Î¼l of SOC (37â„ƒ to RT) medium to each tube, shake the tubes at 37 â„ƒ, 220 rpm for 60 min.</P><P>6. Discard 200 Î¼l supernatant, peptide up and down, resuspend the sediment. </P><P>7. Plate 100 Î¼l for a reaction product, incubate plates upside down overnight at 37â„ƒ.</P><P><B>Colony PCR (Total: 20 Î¼l)</B></P><P>1. Pick colonies as the template for colony PCR. The number picked for each plate depends on the difference between the positive and negative controls.</P><TABLE border="1" align="center"><TBODY><TR><TD style="text-align:center;">2Ã—Taq Master Mix (Vazyme)</TD><TD>10 Î¼l</TD></TR><TR><TD style="text-align:center;">Template</TD><TD>colony</TD></TR><TR><TD style="text-align:center;">Forward Primer (100 Î¼M)</TD><TD>0.05 Î¼l</TD></TR><TR><TD style="text-align:center;">Reverse Primer (100 Î¼M)</TD><TD>0.05 Î¼l</TD></TR></TBODY></TABLE><P style="text-align:center;"> Fill the rest with water. </P><P>2. Test digest performed and products analysed using agarose gel electrophoresis to confirm if correct construct was present.</P><P>3. Use colony PCR to enlarge colony numbers. Then only the positive clones were mini-prepped.</P><P><B>Mini-prep</B></P><P>For the plasmid extraction we used PurePlasmid Mini kit according to manufacturer's instructions.</P><P><B>Identification (Total: 10 Î¼l)</B></P><TABLE border="1" align="center"><TBODY><TR><TD style="text-align:center;">DNA</TD><TD>500-1000 ng</TD></TR><TR><TD style="text-align:center;">Enzyme (NEB)</TD><TD style="text-align:center;">1 Î¼l</TD></TR><TR><TD style="text-align:center;">10Ã—Buffer</TD><TD style="text-align:center;">1 Î¼l</TD></TR></TBODY></TABLE><P style="text-align:center;"> Fill the rest with water. </P><P>2. Pipette up and down thoroughly.</P><P>3. Put at 37â„ƒ for about 4 h (according to different enzymes)</P><P><B>Sequence</B></P><P>Positive clones were sent for sequencing of the insert using appropriate primers.</P><P><B>GST protein expression and purification</B></P><P><B>Expression:</B></P><P>1. Inoculate the bacteria and let them grow overnight at 37â„ƒ.</P><P>2. On the second day, use LB medium dilute the bacteria by about 1/50 and let them grow at 37â„ƒ for 2-3 h until the A600nm reaches 0.6-0.8.</P><P>3. Add IPTG till final concentration of 0.1 mM to 2.0 mM and continue growing for another 3-4 h. Alternatively, the bacteria can also grown at room temperature.</P><P>4. Temperature overnight after adding IPTG. (Lower temperature and lower IPTG concentration could reduce the formation of inclusion bodies and consequently increase the solubility of the synthesized protein).</P><P>5. The bacteria with centrifugation 3.000 gÃ—10 min at 4â„ƒ. Wash the pelleted bacteria by PBS and centrifugate in the same way. The bacteria pellet can be stored at -80â„ƒ.</P><P>6. Resuspend the pellet in Bacteria Lysis Buffer or STE buffer.</P><P>7. Add lysozyme till final concentration of 1 mg/ml and leave the bacteria on ice forat least 15 min. Just before the sonication, add 100 Î¼l of 1 M DTT and 1.4 ml of 10% Sarkosyl. Mix thoroughly and sonicate for 1 min.</P><P>8. Centrifuge at 16,000 rpm for 20 min to pellet the debris on the SS34 rotor.</P><P>9. The supernatant can be stored at -80â„ƒ freezer.</P><P><B>Purification:</B></P><P>1. The Glutathione-Agarose beads should be hydrolyted for at least 2 h by PBS or water. Then use 1% non-fat milk to block the bead for at least 30 min.</P><P>2. Wash the bead by PBS for 4-5 times to remove the milk. Add NaN3 to prevent the bacteria from growing.</P><P>3. To pull down the GST protein, add 20 Î¼l of the blocked bead to each sample in eppendorf tubes.</P><P>4. Rotate the tubes at 4â„ƒ for at least 15 min.</P><P>5. Spin the tubes at 6,500 rpm for 10-20 s.</P><P>6. Aspirate the supernatant and wash with 1 ml of PBS.</P><P>7. Repeat the step 6 for 6-9 times.</P><P>8. Use the Elution Buffer to elute the bound GST protein. </P><P><B>MagneGST Protein Purification</B></P><P><B>Preparation of Cells:</B></P><P>Bacterial cultures can be grown in tubes or flasks. Grow the culture containing the appropriate GST-fusion protein (GST-spycatcher) to an O.D.600 between 0.4 and 0.6, then induce protein expression. For IPTG induction, add IPTG till final concentration of 0.4mM-1mM and incubate at 37â„ƒ for 3 h or at 16â„ƒ overnight. Depending upon the expression system, other methods of induction and growth conditions can be used. Induction conditions often require optimization. For 1ml bacterial cultures, pellet cells by centrifugation at 16,000 g for 3 min and carefully remove the supernatant. For culture volumes larger than 1 ml, pellet the cells by centrifugation at 3,000 g for 20 min. Carefully remove the supernatant. The cell pellet can be frozen at â€“20â„ƒ orâ€“70â„ƒ for long-term storage, or the cells can be lysed immediately.</P><P><B>Cell Lysis:</B></P><P>1. Prepare cell pellets from bacterial culture. Remove all growth medium. Recommended: Freeze the cell pellets at â€“20â„ƒ (15â€“20 min is usually sufficient) or on dry ice for 5â€“10 min. Freezing cell pellets will increase efficiency of cell lysis of certain strains, such as the BL21 series of strains, or when cultures are grown to a high density. For cultures of O.D.600 &gt;2, more lysis buffer will be required.</P><P>2.Resuspend the pellet in MagneGST Cell Lysis Reagent at room temperature (20â€“25â„ƒ) by pipetting or gentle mixing. (for 500 ml of bacteria culture, use 10 ml of MagneGST Cell Lysis Reagent)</P><P>3.Incubate the cell suspension at room temperature for 20â€“30 min on a rotating platform or shaker and then sonicate for 1 min.</P><P><B>Particle Equilibration:</B></P><P>1. Thoroughly resuspend the MagneGST particles by inverting the bottle to obtain a uniform suspension.</P><P>2. Pipet 100 Î¼l of MagneGST particles into a 1.5 ml tube.</P><P>3. Place the tube in the magnetic stand, and allow the MagneGST particles to be captured by the magnet.</P><P>4. Carefully remove and discard the supernatant.</P><P>5. Remove the tube from the magnetic stand. Add 250 Î¼l of MagneGST Binding/Wash Buffer to the particles, and resuspend by pipetting or inverting.</P><P>6. Repeat Particle Equilibration Steps 3â€“5 times.</P><P><B>Binding:</B></P><P>1. After the final wash, gently resuspend the particles in 100 Î¼l of MagneGST Binding/Wash Buffer.
Note: After being washed and resuspended in the appropriate volume of MagneGST Binding/Wash Buffer, the
MagneGST particles can be stored at 4â„ƒ overnight. Do not allow the particles to dry.</P><P>2. Add 600 Î¼l of cell lysate, prepared as described above, to the particles. Mix gently by pipetting or inverting. </P><P>3. Incubate with gentle mixing on a rotating platform or shaker for 30 min at 4â„ƒ or room temperature.
Note: Do not allow particles to settle for more than a few minutes, as this will reduce binding efficiency.</P><P><B>Washing:</B></P><P>1. Place the tube in the magnetic stand, and allow the MagneGST particles to be captured by the magnet.</P><P>2. Carefully remove the supernatant. Save the supernatant (flowthrough) for SDS-PAGE analysis, if desired.</P><P>3. Remove the tube from the magnetic stand. Add 250 Î¼l of MagneGST Binding/Wash Buffer to the particles, and mix gently by pipetting or inverting. Incubate at room temperature or 4â„ƒ for 5 min. Occasionally mix by inverting the tube.</P><P>4. Place the tube in the magnetic stand, and allow the MagneGST particles to be captured by the magnet.</P><P>5. Carefully remove the supernatant. </P><P>6. Remove the tube from the magnetic stand. Add 250 Î¼l of MagneGST Binding/Wash Buffer to the particles, and mix gently by pipetting or inverting. The incubation is not necessary at this step.</P><P>7. Place the tube in the magnetic stand, and allow the MagneGST particles to be captured by the magnet.</P><P>8. Carefully remove the supernatant. Save the supernatant if analysis of wash solution is desired.</P><P>9. Repeat Washing Steps 6â€“8 times.</P><P>10. Peptide 200 Î¼l sample for SDS-PAGE analysis.</P><P><B>Formation of a stable Spytag/ Spycatcher complex:</B></P><P>1. Centrifugation (6,500 rpm, 30 s), discard supernatant.</P><P>2. Add 600 Î¼l protein tagged by Spytag, incubate at 37â„ƒ for 15 min.</P><P>3. Repeat washing step.</P><P>4. Peptide 300 Î¼l for SDS-PAGE analysis.</P><P>5. Add TEV protease to the rest solution, release protein of interest.</P><P>The reaction system is as follows:</P><TABLE border="1" align="center"><TBODY><TR><TD style="text-align:center;">Fusion protein</TD><TD style="text-align:center;">20 Î¼g</TD></TR><TR><TD style="text-align:center;">20Ã—rTEV Buffer</TD><TD style="text-align:center;">7.5 Î¼l</TD></TR><TR><TD style="text-align:center;">0.1 M DTT</TD><TD style="text-align:center;">1.5 Î¼l</TD></TR><TR><TD style="text-align:center;">rTEV protease</TD><TD style="text-align:center;">1 Î¼l</TD></TR><TR><TD style="text-align:center;">H<SUB>2</SUB>O</TD><TD style="text-align:center;">up to 150 Î¼l</TD></TR></TBODY></TABLE><P>6. Incubate at 4â„ƒ for 5-6 h.</P><P><B>Bradford Protein Assay</B></P><P>For the Bradford protein assay we used Bradford Protein Assay Kit according to manufacturer's instructions.</P><P><B>SDS-PAGE</B></P><P><B>Casting the gel </B></P><P>1. Select and cast the appropriate base (separating) gel. The amounts of reagents required for two gels with dimensions 0.07 Ã— 14 Ã— 14 cm and containing 10% and 16% acrylamide are given in the table that follows. Do not degas the gel mixtures, because the gel buffer contains SDS. Overlay the poured gels with several drops of water. Leave the gels for about 30 min to polymerize. (Alternatively, precast gels can be purchased from vendors, and the protocol can be started from Step 3.)The freshly prepared ammonium persulfate (APS) solution and TEMED should be added last, immediately before pouring the gels, because these polymerize the gels.</P><P>2. Overlay the polymerized separating gel (10% or 16%, or 16%/6 M urea) directly with a 4% sample (stacking) gel prepared as indicated in the table in Step 1, except if resolution of proteins &lt;5 kDa is desired. If resolution of proteins &lt;5 kDa is desired, then use AB-6 instead of AB-3 for the separating gel and overlay the separating gel with a 1-cm 10% gel, made up as described in the table. The 16% separating gel and the overlaid 10% â€˜spacer gelâ€™ can be polymerized together if no glycerol is added to the 10% acrylamide gel mixture (the common role of glycerol in SDS gels is to increase the density of solutions and to facilitate gel casting; it has no obvious effect on protein separation). Introducing a 10% â€˜spacer gelâ€™ between 4% stacking and 16% separating gels considerably sharpens the bands for proteins and peptides of 1â€“5 kDa.</P><P>3. Adjust protein concentrations so that a suitable amount of protein can be loaded onto the gel. Concentrate samples, preferentially by techniques that do not increase the salt concentration (such as ultrafiltration). Roughly 0.2â€“1 Î¼g of protein for each protein band (in 0.7 Ã— 5 mm gel strips) is sufficient for Coomassie staining. Accordingly, the desired protein concentration in the sample is 0.1 mg/mlâ€“1 for each protein band. For silver staining, 100-fold less protein may be sufficient. Depending on the requirements of protein detection and analysis, concentration of the sample may be necessary.The maximal protein load can be limited by large amounts of neutral detergent in the sample and by high concentrations of lipid when solubilizing biological membranes. SDS must always be in large excess over neutral detergents and/or lipids. For isolated mitochondrial membranes (70% protein, 30% lipid), for example, the optimized maximal protein load of sample wells 0.7 Ã— 5 mm is 20 Î¼g. Increasing the applied amount of protein to 40 Î¼g, for example, may cause the individual protein bands to disappear in a diffuse background. Around 0.4% neutral detergent in the sample can be tolerated for direct mixing with SDS-containing sample incubation buffers and application to SDS gels. The incubation buffers and volumes described in Step 4 set a tenfold excess of SDS over the neutral detergent. Setting the SDS/neutral detergent ratio to &lt;10 may result in a surprising result after staining: normally separated, large proteins may be detected in the upper gel areas, but the lower gel areas may be completely clear with no small proteins detectable.</P><P>4. Mix samples with SDS-containing sample buffers. The volume and buffer to be used depend on the origins of the samples. For low-density samples such as elution fractions from chromatographic columns, add 5 Î¼l of reducing or nonreducing sample incubation buffer A or B (see REAGENTS) to 15 Î¼l of sample. For high-density samples such as fractions from sucrose density gradients, add 5 Î¼l of sample buffer C or D to 15 Î¼l of sample. For pellet samples, resuspend the pellet in 15â€“20 Î¼l of buffer A/4 or B/4.</P><P><B>Sample preparation and protein loading</B></P><P>5. Incubate samples at 37 Â°C for 15 min or for up to 60 min for samples that were in pellet form. Avoid boiling samples, because membrane proteins can irreversibly aggregate in SDS at temperatures &gt;50 Â°C.</P><P>6. Mount the gels in the vertical electrophoresis apparatus (see EQUIPMENT), and add anode buffer as the lower electrode buffer and cathode buffer as the upper electrode buffer (anodes and cathodes are commonly marked red and black, respectively, by the suppliers).</P><P>7. Load samples under the cathode buffer. Apply 10-Î¼l sample volumes to 0.7 Ã— 5 mm sample wells.For the optimal resolution of peptides of 1â€“5 kDa, reduce the amount of SDS in the incubation buffers, for example, by fourfold and reduce the volume applied to 5 Î¼l. Reducing the amount of SDS facilitates electrophoretic separation of peptides from the bulk of SDS. To avoid streaking of peptide bands, peptides and bulk SDS must be separated before the faster-migrating bulk SDS and the peptides immediately following reach the separating gel. The sample volume applied for Tricineâ€“SDS-PAGE should not substantially exceed 10 Î¼l, because stacking of proteins in the range 20â€“100 kDa is less efficient than in Laemmliâ€“SDS-PAGE.</P><P><B>Electrophoresis conditions</B></P><P>8. Set running conditions appropriate to your type of gel; guidance on appropriate running conditions are given in the table below. Start electrophoresis with an initial voltage of 30 V and maintain at this voltage until the sample has completely entered the stacking gel. The next appropriate voltage step can then be applied. The initial current may be as high as 80 mA for a 0.7mm 10% gel. Gels may warm up, but the temperature should not exceed 35â€“40 Â°C. Approaching the end of the run, voltage can be gradually increased to shorten the total time of electrophoresis. Fast runs give better results than overnight runs, especially with 10% acrylamide gels. Alternatively, a constant power of 10 W per gel might be set to ensure an even distribution of heat. The specific settings for electrophoretic runs depend considerably on the apparatus used and its cooling capacity, the length and thickness of the gel, and the acrylamide concentration of the gel. The settings given above can be regarded only as general hints to approach reasonable experimental conditions. For initial runs, it seems advisable to test considerably lower voltage and wattage settings.</P><TABLE border="1" align="center"><TBODY><TR><TD>4% sample gel</TD><TD>10% gel</TD><TD>16% gel</TD><TD>16%  6M urea</TD></TR><TR><TD style="text-align:center;">AB-3</TD><TD style="text-align:center;">ml</TD><TD style="text-align:center;">1</TD><TD style="text-align:center;">6</TD><TD style="text-align:center;">10</TD><TD style="text-align:center;">10</TD></TR><TR><TD style="text-align:center;">Gel Buffer (3Ã—)</TD><TD style="text-align:center;">ml</TD><TD style="text-align:center;">3</TD><TD style="text-align:center;">10</TD><TD style="text-align:center;">10</TD><TD style="text-align:center;">10</TD></TR><TR><TD style="text-align:center;">glycerol</TD><TD style="text-align:center;">g</TD><TD style="text-align:center;">-</TD><TD style="text-align:center;">3</TD><TD style="text-align:center;">3</TD><TD style="text-align:center;">-</TD></TR><TR><TD style="text-align:center;">Urea</TD><TD style="text-align:center;">g</TD><TD style="text-align:center;">-</TD><TD style="text-align:center;">-</TD><TD style="text-align:center;">-</TD><TD style="text-align:center;">10.8</TD></TR><TR><TD>Add water to final volume</TD><TD style="text-align:center;">ml</TD><TD style="text-align:center;">12</TD><TD style="text-align:center;">30</TD><TD style="text-align:center;">30</TD><TD style="text-align:center;">30</TD></TR><TR><TD style="text-align:center;">Polymerize by adding:</TD></TR><TR><TD style="text-align:center;">APS (10%)</TD><TD style="text-align:center;">Î¼l</TD><TD style="text-align:center;">90</TD><TD style="text-align:center;">150</TD><TD style="text-align:center;">100</TD><TD style="text-align:center;">100</TD></TR><TR><TD style="text-align:center;">TEMED</TD><TD style="text-align:center;">Î¼l</TD><TD style="text-align:center;">9</TD><TD style="text-align:center;">15</TD><TD style="text-align:center;">10</TD><TD style="text-align:center;">10</TD></TR></TBODY></TABLE><P style="text-align:center;"><B>REAGENTS</B></P><TABLE border="1" align="center"><TBODY><TR><TD style="text-align:center;">Buffer A/4</TD><TD style="text-align:center;">buffer A diluted with 3 volumes of water</TD></TR><TR><TD style="text-align:center;">Buffer C</TD><TD style="text-align:center;">buffer A without glycerol</TD></TR><TR><TD style="text-align:center;">Buffer B/4</TD><TD style="text-align:center;">buffer B diluted with 3 volumes of water</TD></TR><TR><TD style="text-align:center;">Buffer D</TD><TD style="text-align:center;">buffer B without glycerol</TD></TR></TBODY></TABLE><P style="text-align:center;"><B>Buffer A</B></P><TABLE border="1" align="center"><TBODY><TR><TD style="text-align:center;">Component</TD><TD style="text-align:center;">Mass</TD></TR><TR><TD style="text-align:center;">SDS</TD><TD style="text-align:center;">12 %</TD></TR><TR><TD style="text-align:center;">Fusion protein</TD><TD style="text-align:center;">6 %</TD></TR><TR><TD style="text-align:center;">mercaptoethanol</TD><TD style="text-align:center;">7.5 Î¼l</TD></TR><TR><TD style="text-align:center;">glycerol</TD><TD style="text-align:center;">30 %</TD></TR><TR><TD style="text-align:center;">Coomassie blue G-250</TD><TD style="text-align:center;">0.05 %</TD></TR><TR><TD style="text-align:center;">Tris/HCl (pH 7.0)</TD><TD style="text-align:center;">Tris/HCl (pH 7.0)</TD></TR></TBODY></TABLE><P style="text-align:center;"><B>Buffer B</B></P><TABLE border="1" align="center"><TBODY><TR><TD style="text-align:center;">Component</TD><TD style="text-align:center;">Mass</TD></TR><TR><TD style="text-align:center;">SDS</TD><TD style="text-align:center;">12 %</TD></TR><TR><TD style="text-align:center;">glycerol</TD><TD style="text-align:center;">30 %</TD></TR><TR><TD style="text-align:center;">Coomassie blue G-250</TD><TD style="text-align:center;">0.05 %</TD></TR><TR><TD style="text-align:center;">Tris/HCl (pH 7.0)</TD><TD style="text-align:center;">150 mM</TD></TR></TBODY></TABLE><P style="text-align:center;"><B>Electrode buffer</B></P><TABLE border="1" align="center"><TBODY><TR><TD style="text-align:center;">Component</TD><TD style="text-align:center;">Mass</TD></TR><TR><TD style="text-align:center;">SDS</TD><TD style="text-align:center;">12 %</TD></TR><TR><TD style="text-align:center;">Tris/ HCl </TD><TD style="text-align:center;">300 mM</TD></TR><TR><TD style="text-align:center;">Acetic acid (pH 8.6)</TD><TD style="text-align:center;">100 mM</TD></TR></TBODY></TABLE><H1><B>Device</B></H1><H3><B>Electromagnet separator</B></H3><P>We try to design a rapid and continuous tool for Magnetosome separation. We poured crude AMB-1 cell lysate into a tube and switch it on, 220 / 110v alternating current through the rectifier into a 24V DC power, acting on the electromagnet. All Magnetosomes were adhered to one side of the tube by the magnetic force of the electromagnetic system. The waste solution was empty by releasing the clip at the bottom of the tube, and then used distilled water to wash Magnetosomes, the process was repeated more than three times, and the Magnetosomes were allowed to bind at the side of the tube by magnetic force. The final step was turning off the device, using PBS to resuspend Magnetosomes.</P><P style="text-align:center;">Fig.6 Electromagnet separator</P><H3><B>The pulsed magnetic field generator</B></H3><P>According to reference, the pulsed magnetic field could enhance Magnetosomes formation. So we design this device, the pulsed magnetic field generator, it is composed of the pulse signal generating circuit and the magnetic field generating part. The circuit generates pulse signal, and pass through DC electromagnet coils, which provide a highly homogenous pulsed magnetic field. The instrument was kept at 30â„ƒ in a constant temperature chamber for AMB-1 incubation.</P><P style="text-align:center;">Fig.7 The pulsed magnetic field generator</P><H3>Judging criteria</H3><H3><P><B>Bronze</B></P><P>1. Register the team, have a great summer, and plan to have fun at the Giant Jamboree.</P><P>Our team has been registered, we have had the best summer in lab and in human practice activities, and we expect to have lots of fun at the Giant Jamboree in Boston!</P><P>2. Meet all deliverables on the Requirements page.</P><P>1) We design our team wiki to let people understand our case better.</P><P>2) We prepare poster and presentation for Jamboree.</P><P>3) We submit parts, safety forms and judging forms completely and in time.</P><P>4) We submit DNA samples of our new parts to Registry</P><P>3. Create a page on our team wiki with clear attribution of each aspect of our project. </P><P>Our team consists of students who focus on experiment, website design, art and writing. We have clearly show work done by the students and done by others, including host labs, advisors, instructors, sponsors, professional website designers, artists, and commercial services in our team wiki. </P><P>4. Document at least one new standard BioBrick Part or Device central to our project and submit this part to the iGEM Registry and document a new application of a BioBrick part from a previous iGEM year. </P><P>Our new BioBricks can be found in the Parts page.</P><P><B>Sliver</B></P><P>1. Experimentally validate that at least one new BioBrick Part or Device of your own design and construction works as expected. </P><P>We document the characterization of this part in the Partâ€™s and Deviceâ€™s Registry entry and they can be searched in partsâ€™, human practiceâ€™s and resultsâ€™ pages of our team wiki. We have validated : </P><P>2. Help any registered iGEM team from high school, a different track, another university, or another institution in a significant way.  </P><P>We have helped NEU_China to examine the effectiveness of the biobricks. For more details, see our Collaboration part.</P><P>3. Human Practices in iGEM. Demonstrate how your team has identified, investigated and addressed one or more of these issues in the context of your project. </P><P>Our activity focus on education, experimentersâ€™ metal health, public perception. Our human practice is really interesting, see our human practice page in our wiki.</P><P><B>Gold </B></P><P>1. Integrated Human Practices. </P><P>We have expanded on our human practice by integrating our investigated issues like questionnaire about protein purification in many labs of different universities and scientific centers at first. After our experiments, we give them feedback of our new approach comparing with other approaches. </P><P>2. Improve a previous part or project.</P><P>We improved the function OR characterization of an existing BioBrick Part and enter this information in the Registry. </P><P>3. Proof of concept.</P><P>Our all parts consist of a BioBrick device and the device shows its real function as we expected. See more detail on our device part.</P><P>4. Demonstrate our work. </P><P>Our system works in the real simulated conditions in lab.</P></H3><H3>References</H3><P>1. Honda, T., Yasuda, T., Tanaka, T., Hagiwara, K., Arai, T. and Yoshino, T. 2015. Functional expression of 
full-length TrkA in the prokaryotic host Magnetospirillum magneticum AMB-1 by using a magnetosome display system. Appl Environ Microbiol 81:1472-6.</P><P>2. Li, L., Fierer, J.O., Rapoport, T.A. and Howarth, M. 2014. Structural analysis and optimization of the covalent association between SpyCatcher and a peptide Tag. J Mol Biol 426:309-17.</P><P>3. Lin, Z., Zhao, Q., Zhou, B., Xing, L. and Xu, W. 2015. Cleavable self-aggregating tags (cSAT) for protein
expression and purification. Methods Mol Biol 1258:65-78.</P><P>4. Sugamata, Y., Tanaka, T., Matsunaga, T. and Yoshino, T. 2014. Functional expression of an scFv on bacterial magnetic particles by in vitro docking. Biochem Biophys Res Commun 445:1-5</P><P>5. Wang, X., Liang, L., Song, T. and Wu, L. 2009. Magnetosome formation and expression of mamA, mms13, mms6 and magA in Magnetospirillum magneticum AMB-1 exposed to pulsed magnetic field. Curr Microbiol 59:221-6.</P><P>6. Yoshino, T., Kato, F., Takeyama, H., Nakai, M., Yakabe, Y. and Matsunaga, T. 2005. Development of a novel method for screening of estrogenic compounds using nano-sized bacterial magnetic particles displaying estrogen receptor. Analytica Chimica Acta 532:105-111.</P><P>7. Yoshino, T. and Matsunaga, T. 2006. Efficient and stable display of functional proteins on bacterial magnetic particles using mms13 as a novel anchor molecule. Appl Environ Microbiol 72:465-71.</P><P>8. Arakaki, A., Webb, J. and Matsunaga, T. 2003. A novel protein tightly bound to bacterial magnetic particles in <I>Magnetospirillum magneticum</I> strain AMB-1. J Biol Chem 278:8745-50.</P><P>9. Wang, X. and Liang, L. 2009. Effects of static magnetic field on magnetosome formation and expression of mamA, mms13, mms6 and magA in Magnetospirillum magneticum AMB-1.</P><P>10. Pan, Y., Li, N., Mu, J., Zhou, R., Xu, Y., Cui, D., Wang, Y. and Zhao, M. 2015. Biogenic magnetic nanoparticles from Burkholderia sp. YN01 exhibiting intrinsic peroxidase-like activity and their applications. Appl Microbiol Biotechnol 99:703-15.</P><P>11. Liu, Y., Li, G.R., Guo, F.F., Jiang, W., Li, Y. and Li, L.J. 2010. Large-scale production of magnetosomes by chemostat culture of Magnetospirillum gryphiswaldense at high cell density. Microb Cell Fact 9:99.</P><DIV class="row mt50"><DIV class="Partner"><H1> Partner</H1></DIV></DIV></DIV></DIV></ARTICLE></DIV></DIV></DIV></DIV></DIV></DIV></BODY></HTML>