Team:NEFU China/Notebook
Home
Team
Member Collaboration Attribution
Project
Description Design Experiment&Protocol Demonstrate Hardware Judging criteria
Parts
Results
AMB-1 E. coli Purification Modeling & Proof Future application
Notebook
Human practice
Safety Education Questionnaire iGEMer Community
BioDog
The 2016 NEFU iGEM team was recruited at the very beginning of November 2015, just followed the end of the iGEM competition of 2015. After our iGEM team was established, the old guys in our team 2015 give a series of classes to teach every member in our team a foundational knowledge about molecular biology, microbiology and safety. During this period, we have also learned some classic projects of former team which gives us a lot of inspiration on designing our project. The study of fundamental knowledge lasted 3 months.
March to June
We started to design our own project since the middle of March. We put forward three ideas this year, we were divided into three groups to improve the project respectively. We searched and studied lots of papers to build the theoretical framework for each topic during this period.
July to August
We started to learn how to do experiment in our lab in June. In order to avoid lab accident, we all received laboratory safety training before we formally step into the lab. We attended Central China iGEM Consortium (CCiC) at the end of August. During the conference we showed our project and communicated with other teams and gained a lot of inspiration to develop a better system.
September
In this month we started to focus on the group 1’s project. Since the project of group 1 is the most maturity one, we decided to use their project to apply this year’s competition. We had also developed cooperation with NEU_China in this month. Meanwhile we had organized several social activities in this month, in order to expand the influence of iGEM
October
Time flies. We seized the time to make all the experiments done and put great efforts for the wiki building and preparing the presentation for the Giant Jamboree.
Use restriction enzyme to digest the pET-14b backbone. (Vector: pET-14b-9×Z domain)
Using gel electrophoresis to confirm the concentration ratio between the backbone and the insert. (Vector: pET-14b-Pmsp3-Mms13-EGFP)
Enzyme digestion for detection. (vector: pET-14b-Pmsp3-Mms13-EGFP)
Digestion of intein. (Vector: pET-14b-Amilcp-intein-Spytag)
Using gel electrophoresis to confirm the concentration ratio between the backbone(pET-14b) and the insert. (Vector: Amilcp: Intein=2:20:1) (Vector: pET-14b-Amilcp-intein-Spytag)
Enzyme digestion (9x Z domain) (vector: pET-14b-9×Z domain)
BamH I
Kpn I
10x
CutSmart
Buffer
plasmid
Temperature
Time
1µl
1µl
3µl
25µl
37°C
4h
Enzyme digestion (pET14b plasmid) (Vector: pET-14b-9×Z domain)
KpnI
BamHI
10x
CutSmart
Buffer
plasmid
ddH2O
Temperature
Time
1µl
1µl
3µl
2µl
23µl
37°C
4h
Enzyme digestion (Vector: pET-14b-Pmsp3-EGFP)
EGFP
KpnI
NotI
10*smart
buffer
H2O
25µl
1µl
1µl
3µl
0µl
Enzyme digestion (Vector: pET-14b-Pmsp3-EGFP)
Pet14b-Pmsp3
KpnI
NotI
10*smart
buffer
H2O
5µl
1µl
1µl
3µl
20µl
Ligation (Vector: pET-14b-Pmsp3-EGFP)
pet14b-pmsp3
10*buffer
T4 Ligase
H2O
Insert:
EGFP
1µl
1.5µl
0.5µl
11.2µl
0.8µl
E. coli transformed with vector (pET-14b-Pmsp3-EGFP) cultured on LB-ampicillin medium for 14h.
EGFP
Primers
Primer F
CGGGTACCGTGAGCAAGGGCGAGGA
Primer R
ATAAGAATGCGGCCGCTTACTTGTACAGCTCGTCCATG
PCR system (50µl)
Parameters
Procedure
Temperature
Time
2×Taq Master Mix
25µl
Predenature
94℃
5min
PrimerF (10µM)
1 µl
Denature
94℃
30sec
PrimerR (10µM)
1 µl
Annealing
55℃
30sec
ddH2O
22µl
Extension
72℃
60sec
Template
1µl
Final
Elongation
72℃
10min
dNTPs
Included
in
premix
Final Hold
4℃
∞
Buffer
Cycle
30 cycles
Mms13
Primers
Primer F
GGGGTACCCCCTTTCACCTTGCCCC
Primer R
CCGCTCGAGGGCCAGTTCGTCCCGC
PCR system (50µl)
Parameters
Procedure
Temperature
Time
2×Taq Master Mix
25µl
Predenature
94℃
5min
PrimerF (10µM)
1 µl
Denature
94℃
30sec
PrimerR (10µM)
1 µl
Annealing
60℃
30sec
ddH2O
22µl
Extension
72℃
45sec
Template
1µl
Final
Elongation
72℃
10min
dNTPs
Included
in
premix
Final Hold
4℃
∞
Buffer
Cycle
30 cycles
pmsp3（488bp）
Primers
Primer F
GAAGATCTACTAGTACGTTGAATCCCAGCG
Primer P
GCTCTAGAGGGAAACCGTTGTGGTCTCATAGAAAGGCTCCCTCATAG
PCR system (50µl)
Parameters
Procedure
Temperature
Time
2×Taq Master Mix
25µl
Predenature
94℃
5min
PrimerF (10µM)
1 µl
Denature
94℃
30sec
PrimerR (10µM)
1 µl
Annealing
55℃
30sec
ddH2O
22µl
Extension
72℃
30sec
Template
1µl
Final
Elongation
72℃
10min
dNTPs
Included
in
premix
Final Hold
4℃
∞
Buffer
Cycle
30 cycles
Colony PCR (pmsp3-1 pmsp3-2)
Primers
Primer F
AGCCCGATCTTCCCCATC
Primer P
TTCAGCAAAAAACCCCTCAA
PCR system (10µl)
Parameters
Procedure
Temperature
Time
10×Dream Taq Buffer
1µl
Predenature
94℃
5min
PrimerF (100µM)
0.05 µl
Denature
94℃
30sec
PrimerR (100µM)
0.05 µl
Annealing
58℃
30sec
ddH2O
8.4µl
Extension
72℃
30sec
Template:
Pick a single colony
and dip in H2O
Final
Elongation
72℃
10min
dNTPs
0.2
Final Hold
4℃
∞
Taq Enzyme
0.05
Cycle
30 cycles
amilcp
Primers
Primer F
CGGAATTCAGTGTGATCGCTAAACAAATGAC
Primer P
CGGGATCCTTATTAGGCGACCACAGGTT
PCR system (50µl)
Parameters
Procedure
Temperature
Time
2×Taq Master Mix
25µl
Predenature
94℃
5min
PrimerF (10µM)
1 µl
Denature
94℃
30sec
PrimerR (10µM)
1 µl
Annealing
50℃
40sec
ddH2O
22µl
Extension
72℃
30sec
Template
1µl
Final
Elongation
72℃
10min
dNTPs
Included
in
premix
Final Hold
4℃
∞
Buffer
Cycle
30 cycles
EGFP+Pmsp3
Primers
Primer F
CGGGTACCGTGAGCAAGGGCGAGGA
Primer P
ATAAGAATGCGGCCGCTTACTTGTACAGCTCGTCCATG
PCR system (25µl)
Parameters
Procedure
Temperature
Time
2×Taq Master Mix
12.5µl
Predenature
94℃
5min
PrimerF (10µM)
1 µl
Denature
94℃
30sec
PrimerR (10µM)
1 µl
Annealing
58℃
30sec
ddH2O
9.5µl
Extension
72℃
60sec
Template
1µl
Final
Elongation
72℃
10min
dNTPs
Included
in
premix
Final Hold
4℃
∞
Buffer
Cycle
30 cycles
Colony PCR (EGFP+Pmsp3)
Primers
Primer F
AGCCCGATCTTCCCCATC
Primer P
TTCAGCAAAAAACCCCTCAA
PCR system (10µl)
Parameters
Procedure
Temperature
Time
10×Dream Taq Buffer
1µl
Predenature
94℃
5min
PrimerF (100µM)
0.05 µl
Denature
94℃
30sec
PrimerR (100µM)
0.05 µl
Annealing
58℃
30sec
ddH2O
8.4µl
Extension
72℃
30sec
Template:
Pick a single colony
and dip in H2O
Final
Elongation
72℃
10min
dNTPs
0.2
Final Hold
4℃
∞
Taq Enzyme
0.05
Cycle
30 cycles
16S rRNA Gene cloning
1.Each sample of bacteria (1-5ml) was in a tube. Centrifugation was carried out at 12,000 rpm for 1 min at 4°C. The supernatant was discarded, and the pellet was processed for each procedure as follows.
2.   Add 180μL Buffer GTL to the pellet to suspend.
3.  20μL Proteinase K was added to the tube and vortex. Then the samples were incubated at 56°C for 30 min until the solution is clear.
4.  Add 20μL Buffer GL. The mixture was then briefly mixed on a vortex mixer and centrifuged at 10,000 g for 5 min. Then add 200μL anhydrous ethanol and still vortex.
5. Add the solution obtained from step 4 (include the pellet) to the adsorption column with collection tube, centrifuge at 12000rpm for 1 min, discard the effluent. Adsorption column was put back in the collection tube.
6. Add 500μL Buffer GW1 to the adsorption column, centrifuge at 12000rpm for 1 min. Discard the effluent. Adsorption column was put back in the collection tube.
7. Add 500μL Buffer GW2 to the adsorption column, centrifuge at 12000rpm for 1 min. Discard the effluent. Adsorption column was put back in the collection tube.
8. Centrifuge at 12,000rpm for 2 min, discard the effluent. Put the adsorption column at room temperature until dry out.
9. Place an adsorption column in a new tube, add the 50μLBuffer GE to the column. Put at room temperature for 2-5 min. Centrifuge at 12,000rpm for 1 min. Collect DNA solution, store at -20°C.
Make competence AMB-1 and transformation
Growth
1.Add 1ml bacterial solution to 100ml LB media.
2.Grow at 30°C, 200rpm
3.after 3 and half hours, the OD600 is 0.44
4.after 5 hours, OD600 is 0.85
Make competence cells
1.200ml bacterial solution, stored in ice for 10 min.
2.Centrifuged at 8000 rpm for 30 min at 4℃ .
3.Wash bacteria twice with 272mmol sucrose 10mM TES (PH 7.5). Centrifuged at 3000 rpm for 12 min
4.Suspend in 2ml TES, put in 1.5ml tube, stored in -40℃
Electroporation
Bacteria cultured on the MSGM media. Single colony was spotted.
Fig. 1 AMB-1 cultured on the MSGM media. Single colony was spotted.
Fig. 2 AMB-1 cultured on the LB media. Single colony was spotted.
