<HTML xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en" dir="ltr">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki  ltr ns-0 ns-subject page-Team_DTU-Denmark_Bioinformatic"><DIV id="globalWrapper"><DIV id="top-section"><DIV id="p-logo"><A href="/Main_Page" title="Main Page">&quot;
	    </A></DIV><DIV id="menubar" class="left-menu noprint"><UL><LI class="selected"><A href="/Team:DTU-Denmark/Bioinformatic">Page               </A></LI><LI class="new"><A href="/wiki/index.php?title=Talk:Team:DTU-Denmark/Bioinformatic&amp;action=edit&amp;redlink=1">Discussion               </A></LI><LI><A href="/wiki/index.php?title=Team:DTU-Denmark/Bioinformatic&amp;action=edit">View source               </A></LI><LI><A href="/wiki/index.php?title=Team:DTU-Denmark/Bioinformatic&amp;action=history">History               </A></LI><LI style="color:#808080;cursor:default">teams</LI></UL></DIV><DIV class="right-menu noprint" id="menubar"><UL><LI id="pt-login"><A href="/wiki/index.php?title=Special:UserLogin&amp;returnto=Team:DTU-Denmark/Bioinformatic" title="You are encouraged to log in; however, it is not mandatory [o]" accesskey="o">Log in</A></LI></UL></DIV><DIV id="search-controls" class="noprint"><FORM action="/Special:Search" id="searchform"> </FORM></DIV></DIV><DIV id="content"><H1 class="firstHeading">Team:DTU-Denmark/Bioinformatic</H1><DIV id="bodyContent"><H3 id="siteSub" class="noprint">From 2011.igem.org</H3><DIV id="navigation"><DIV class="navigation"><DIV class="centering"><UL class="topnav" id="topnav"><LI class="home navitem1"><A href="/Team:DTU-Denmark">Home</A></LI><LI class="vision navitem2"><A href="/Team:DTU-Denmark/Vision" title="Test: A soft introduction to the idea and it's future applications.">Vision</A></LI><LI class="project navitem3"><A href="/Team:DTU-Denmark/Project">Project</A></LI><LI><A href="/Team:DTU-Denmark/Project">Overview</A></LI><LI><A href="/Team:DTU-Denmark/Bioinformatic">Bioinformatics</A></LI><LI><A href="/Team:DTU-Denmark/Project_testing_sRNA">Experiment: Testing sRNA</A></LI><LI><A href="/Team:DTU-Denmark/Project_improving_araBAD">Experiment: Improving araBAD</A></LI><LI><A href="/Team:DTU-Denmark/Modeling">Modeling</A></LI><LI><A href="/Team:DTU-Denmark/Data">Data page</A></LI><LI><A href="/Team:DTU-Denmark/Attributions">Attributions</A></LI></UL><LI class="technical_stuff navitem4"><A href="/Team:DTU-Denmark/Technical_stuff">Technical stuff</A></LI><LI><A href="/Team:DTU-Denmark/Parts">Parts</A></LI><LI><A href="/Team:DTU-Denmark/Notebook">Notebook</A></LI><LI><A href="/Team:DTU-Denmark/Protocols">Protocols</A></LI><LI><A href="/Team:DTU-Denmark/Technical_stuff_math">Math</A></LI><LI><A href="/Team:DTU-Denmark/Matlab">Matlab</A></LI><LI><A href="/Team:DTU-Denmark/Safety">Safety</A></LI><LI class="background navitem5"><A href="/Team:DTU-Denmark/Background">Background</A></LI><LI><A href="/Team:DTU-Denmark/Background_sRNA">Small RNA</A></LI><LI><A href="/Team:DTU-Denmark/Background_the_natural_system">The chitobiose system</A></LI><LI><A href="/Team:DTU-Denmark/Background_synthetic_biology">Synthetic biology</A></LI><LI class="team navitem6"><A href="/Team:DTU-Denmark/Team">Team</A></LI></DIV></DIV></DIV><DIV id="heading"><DIV class="centering"><H1>

Bioinformatics

            </H1></DIV></DIV><DIV id="innercontent"><DIV class="centering"><DIV class="whitebox article"><TABLE id="toc" class="toc"><TBODY><TR><TD><DIV id="toctitle"><H2>Contents</H2></DIV><UL><LI class="toclevel-1 tocsection-1"><A href="#Motivation"><SPAN class="tocnumber">1</SPAN><SPAN class="toctext">Motivation</SPAN></A></LI><LI class="toclevel-1 tocsection-2"><A href="#Approach"><SPAN class="tocnumber">2</SPAN><SPAN class="toctext">Approach</SPAN></A></LI><LI class="toclevel-1 tocsection-3"><A href="#Results"><SPAN class="tocnumber">3</SPAN><SPAN class="toctext">Results</SPAN></A></LI><LI class="toclevel-1 tocsection-4"><A href="#Alignment"><SPAN class="tocnumber">4</SPAN><SPAN class="toctext">Alignment</SPAN></A></LI><LI class="toclevel-1 tocsection-5"><A href="#Species"><SPAN class="tocnumber">5</SPAN><SPAN class="toctext">Species</SPAN></A></LI><LI class="toclevel-1 tocsection-6"><A href="#References"><SPAN class="tocnumber">6</SPAN><SPAN class="toctext">References</SPAN></A></LI></UL></TD></TR></TBODY></TABLE><H2><SPAN class="mw-headline" id="Motivation"> Motivation </SPAN></H2><P>A bioinformatic study was performed to investigate the flexibility when engineering sRNA regulation genetically. The aim was to elucidate sequence and structure conservation for relevant sRNA homologs among 24 bacterial species representative of the diversity within Enterobacteriaceae.
</P><H2><SPAN class="mw-headline" id="Approach"> Approach </SPAN></H2><P>Firstly, <B>ChiP homologs</B> including its ribosome binding site (RBS) were identified in the 24 species by BlastP using the <I>E. coli</I> protein as bait. Then, to identify <B>putative ChiXs</B> a local BlastN search was performed with the sequence flanking the RBS as query and the given genome as target (window size of 7). Hits in coding genes were excluded since we restrict ChiX-homologs to be in intergenic regions as is the case in <I>E.coli</I> and <I>Salmonella</I><SPAN class="superscript"><A href="#References">[4]</A></SPAN><SPAN class="superscript"><A href="#References">[2]</A></SPAN>. In addition, hits which did not seem to have a putative -35 and -10 region in a reasonable distance from the putative start-site were also excluded, leaving one hit for each of the 24 sequences. Next, the <B>secondary structure</B> of the 24 putative ChiX homologs were analysed by RNAfold (default parameter setting) from the Vienna RNA package version 2.0.0<SPAN class="superscript"><A href="#References">[3]</A></SPAN> and sequence lengths were selected for further analysis  so  they start with one or two nucleotides  upstream of the first stemloop and end with four or five  Ts downstream of the second stem-loop. In the next step, a  <B>structural alignment</B> was made by the LocARNA server, also from the Vienna RNA package version 2.0.0<SPAN class="superscript"><A href="#References">[3]</A></SPAN> (using default parameter settings), and finally, the RNA secondary structure and sequence conservation were visualized by RNAlogo webserver<SPAN class="superscript"><A href="#References">[1]</A></SPAN>. Afterwards, the percentage of G/C and A/T (out of 24) was manually calculated. Note that at gap positions these two numbers do not sum to 100.
</P><H2><SPAN class="mw-headline" id="Results"> Results </SPAN></H2><P>From the sequence logo it can be seen that the structure, but not the sequence, is conserved in the first stemloop, whereas both the structure and sequence is conserved in the second stemloop. The length between the two stemloops varies a few nucleotides among the species, which is seen from the introduced gaps in the alignment. Another interesting feature is the conserved A/U stretch around position 40 which resemble the characteristic Hfq binding motif<SPAN class="superscript"><A href="#References">[5]</A></SPAN>. The sequence which base-pair with the ChiP RBS is perfectly conserved, this might be due to evolutionary selection for a strong RBS rather than functionally constraints (AAAGAGG is not a bad RBS, and is somewhat similar to the RBS BioBricks: BBa_B0030, BBa_B0034, BBa_B0035, and BBa_B0064 which has a relative strength of 0.35-1.124). Consequently, we expect that complementary mutations can freely be made to match the RBS in any mRNA of interest.
</P><DIV class="center"><DIV class="thumb tnone"><DIV class="thumbinner" style="width:667px;"><DIV class="thumbcaption"><B>RNALogo graph</B> generated with RNALogo<SPAN class="superscript"><A href="#References">[1]</A></SPAN> as well as sequence logo and percentage of A/U (green) and G/C (yellow)nucleotides out of 24 sequences. Notice, that perfectly conserved nucleotides are highlighted with pink in the graph.</DIV></DIV></DIV></DIV><P><B>To conclude</B>, the study indicates several constrains for engineering a novel sRNA derived from ChiX. First of all must the sRNA contain a Hfq binding motif, a terminal poly-U stretch and two stemloops (the first for which only the structure seems important in contrast to the second for which both the structure and the majority of the nucleotides appear essential). Taking these restrictions into account we believe it is possible to rationally design a sRNA to target any gene of interest.
</P><H2><SPAN class="mw-headline" id="Alignment"> Alignment </SPAN></H2><P>The used structural alignment generated by the LocARNA server from the Vienna RNA package version 2.0.0<SPAN class="superscript"><A href="#References">[3]</A></SPAN>.
</P><PRE>Erwinia_billingi   -----CCCCG-CUGGCAACCUGAUUGCUACCGG---AUAACUAAAAUCAUA-AAAAAAUUCCUCUUUGA-CGGGCCGAUAGCAAUAUUGGCCAUUUUU
Pantoea_sp__At_9   AUUGGCCCGCGCCAGCCA-GUAAUGGCUGUGUG---AUAACCAAAAUCAUA-AACAAAUUCCUCUUUGA-CGGGCCGAUAGUAAUAUUGGCCUUCUUU
Serratia_proteam   -C---UAUUGUAACCGUUAACA--GGGUUACAAUGCGUA-UAACUACAAUACAAGAAAUUCCUCUUUGA-CUGGCCAGUAGCGAUAUUGGCCACUUUU
Yersinia_enteroc   -CU-GCUC--AUAUUUUCCGCAAGGAAUGUGAGGGCUUAAUAACUAAAAUAAUGAAAAUUCCUCUUUGA-CUGGCCGAUAGCGAUAUCGGCCAUUUUU
Yersinia_pesti_1   --UGGCGCUCACAUUAUCCGCAAGGAGUGUGAGUGCUUAAUAACAAAAAUAAUGAAAAUUCCUCUUUGA-CUGGCCGGUAGUGAUAUCGGCCAUUUUU
Yersinia_pestis_   --UGGCGCUCACAUUAUCCGCAAGGAGUGUGAGUGCUUAAUAACAAAAAUAAUGAAAAUUCCUCUUUGA-CUGGCCGGUAGUGAUAUCGGCCAUUUUU
Yersinia_pseudot   -CUGGUGCUCACAUUAUCCGCAAGGAGUGUGAGUGCUUAAUAACAAAAAUAAUGAAAAUUCCUCUUUGA-CUGGCCGGUAGUGAUAUCGGCCAUUUUU
Citrobacter_rode   --------ACCGUC--GCUUAA-AGCGGCGGC----AUAACAAU--AAUGA--UGAAAUUCCUCUUUGA-CGGGCCAAUAGCGAUAUUGGCCAUUUUU
Cronobacter_saka   -----A--ACCGUC--CGCUAAGGCGCACGGC----AUAACGACAAUAACG--AAAAGUUCCUCUUUGA-CGGGCCAGUAGCGAUACUGGCCUUCUUU
Cronobacter_turi   --------ACCGUU--CGCUAACGCGCACGGC----AUAACGAUAAUAACG---AAAGUUCCUCUUUGA-CGGGCCAGUAGCGAUACUGGCCUUCUUU
Edwardsiella_ict   -----ACCGGGCAC--CCCUUGGGGGGCGCCC-GGAAUAAUAAU---AUC---UGAAGUUCCUCUUUGACUG-GCCAAUAGCAAUAUUGGCCUUUUUU
Edwardsiella_tar   -----A--CCGGGU--CCCUAUGGGGGCCCGGCAUAAUAAUAAU---AUC---UGAAGUUCCUCUUUGACUG-GCCAAUAGCGAUAUUGGCCUUUUUU
Enterobacter_aer   -----A--UCCGGG--AUGUAU--AUCCCGGG----AUAAUAAU--AAUGA--UGAAAUUCCUCUUUGA-CGGGCCAAUAGCAAUAUUGGCCAUUUUU
Enterobacter_clo   -----A--UCCGGA--GUGCGA--ACUCCGGG----AUAAUAAU--AACGA--UGAAAUUCCUCUUUGA-CGGGCCAAUAGAAAUAUUGGCCAUUUUU
Enterobacter_sp_   -----A--ACCGAG--GGUCU---CCUUCGGC----AUAAUAAU--AACGA--UGAAAUUCCUCUUUGA-CGGGCCAAUAGAAAUAUUGGCCAUUUUU
Escherichia_coli   -----C--ACCGUC--GCUUAA-AGUGACGGC----AUAAUAAUAAAAAAA--UGAAAUUCCUCUUUGA-CGGGCCAAUAGCGAUAUUGGCCAUUUUU
Klebsiella_pne_1   -----A--UCCGGG--AUGCAA--AUCCCGGG----AUAAUAAU--AAUGA--UGAAAUUCCUCUUUGA-CGGGCCAAUAGCAAUAUUGGCCAUUUUU
Klebsiella_pneum   -----A--UCCGGG--AUGCAA--AUCCCGGG----AUAAUAAU--AAUGA--UGAAAUUCCUCUUUGA-CGGGCCAAUAGCAAUAUUGGCCAUUUUU
Klebsiella_varii   -----A--UCCGGG--AUGCAA--AUCCCGGG----AUAAUAAU--AAUGA--UGAAAUUCCUCUUUGA-CGGGCCAAUAGCAAUAUUGGCCAUUUUU
Salmonella_bongo   -----A--UCCGAA--GUGAAA--GCUUCGGG----AUAAUAAU--AAUGA--UGAAAUUCCUCUUUGA-CGGGCCAAUAGCAAUAUUGGCCAUUUUU
Salmonella_enter   -----A--UCCGAA--GCGAAA--GCGUCGGG----AUAAUAAU--AACGA--UGAAAUUCCUCUUUGA-CGGGCCAAUAGCGAUAUUGGCCAUUUUU
Shigella_flexner   --------ACCGUC--GCUUAA-AGUGACGGC----AUAAUAAUAAAAAAA--UGAAAUUCCUCUUUGA-CGGGCCAAUAGCGAUAUUGGCCAUUUUU
Shigella_sonnei_   --------ACCGUC--GCUUAA-AGUGACGGC----AUAAUAAUAAAAAAA--UGAAAUUCCUCUUUGA-CGGGCCAAUAGCGAUAUUGGCCAUUUUU
citrobacter_kose   -----A--ACCAGG--GCGCUA-CGUCCUGGC----AUAAUAAU--AACGA--UGAAAUUCCUCUUUGA-CGGGCCAAUAGAAAUAUUGGCCAUUUUU
</PRE><H2><SPAN class="mw-headline" id="Species"> Species </SPAN></H2><P>Below are the names and accession numbers listed for the 24 species included in the analysis.
</P><TABLE><TBODY><TR><TD><B>Species</B></TD><TD><B>Accession Number</B></TD></TR><TR><TD><I>Citrobacter koseri</I></TD><TD>	NC_009792
</TD></TR><TR><TD><I>Citrobacter rodentium</I></TD><TD>	NC_013716
</TD></TR><TR><TD><I>Cronobacter sakazakii</I></TD><TD>	CP000783
</TD></TR><TR><TD><I>Cronobacter turicensis</I></TD><TD>	FN543093
</TD></TR><TR><TD><I>Edwardsiella ictaluri</I></TD><TD>	CP001600
</TD></TR><TR><TD><I>Edwardsiella tarda</I></TD><TD>	CP002154
</TD></TR><TR><TD><I>Enterobacter aerogenes</I></TD><TD>	CP002824
</TD></TR><TR><TD><I>Enterobacter cloacae</I></TD><TD>	CP002272
</TD></TR><TR><TD><I>Enterobacter sp.</I></TD><TD>	CP000653
</TD></TR><TR><TD><I>Erwinia billingiae</I></TD><TD>	FP236843
</TD></TR><TR><TD><I>Escherichia coli</I></TD><TD>	AE005174
</TD></TR><TR><TD><I>Klebsiella pneumonia</I></TD><TD>	CP002910
</TD></TR><TR><TD><I>Klebsiella pneumonia</I></TD><TD>	CP000647
</TD></TR><TR><TD><I>Klebsiella variicola</I></TD><TD>	CP001891
</TD></TR><TR><TD><I>Pantoea sp.</I></TD><TD>		CP002433
</TD></TR><TR><TD><I>Salmonella bongori</I></TD><TD>	FR877557
</TD></TR><TR><TD><I>Salmonella enteric</I></TD><TD>	NC_010067
</TD></TR><TR><TD><I>Serratia proteamaculans</I></TD><TD>	CP000826
</TD></TR><TR><TD><I>Shigella flexneri</I></TD><TD>	AE014073
</TD></TR><TR><TD><I>Shigella sonnei</I></TD><TD> NC_007384
</TD></TR><TR><TD><I>Yersinia enterocolitica</I></TD><TD>	FR729477
</TD></TR><TR><TD><I>Yersinia pestis</I></TD><TD>	 	CP000308
</TD></TR><TR><TD><I>Yersinia pestis</I></TD><TD>	NC_005810
</TD></TR><TR><TD><I>Yersinia pseudotuberculosis</I></TD><TD>	NC_006155
</TD></TR></TBODY></TABLE></DIV><DIV class="whitebox article"><H2><SPAN class="mw-headline" id="References"> References </SPAN></H2><P>[1] Chang, Tzu-Hao, Jorng-Tzong Horng, and Hsien-Da Huang. “RNALogo: a new approach to display structural RNA alignment.” Nucleic Acids Research 36, no. Suppl. S (2008): W91-W96.
</P><P>[2] Figueroa-Bossi, Nara, Martina Valentini, Laurette Malleret, and Lionello Bossi. “Caught at its own game: regulatory small RNA inactivated by an inducible transcript mimicking its target.” Genes &amp; Development 23, no. 17 (2009): 2004 -2015. <A href="http://genesdev.cshlp.org/content/23/17/2004.abstract" class="external free" rel="nofollow">http://genesdev.cshlp.org/content/23/17/2004.abstract</A>.
</P><P>[3] Hofacker, Ivo L. “Vienna RNA secondary structure server.” Nucleic Acids Research 31, no. 13 (July 2003): 3429 -3431. <A href="http://nar.oxfordjournals.org/content/31/13/3429.abstract" class="external free" rel="nofollow">http://nar.oxfordjournals.org/content/31/13/3429.abstract</A>.
</P><P>[4] Overgaard, Martin, Jesper Johansen, Jakob Møller‐Jensen, and Poul Valentin‐Hansen. “Switching off small RNA regulation with trap‐mRNA.” Molecular Microbiology 73, no. 5 (September 2009): 790-800. <A href="http://onlinelibrary.wiley.com/doi/10.1111/j.1365-2958.2009.06807.x/abstract" class="external free" rel="nofollow">http://onlinelibrary.wiley.com/doi/10.1111/j.1365-2958.2009.06807.x/abstract</A>.
</P><P>[5] Vogel, Jörg, and Ben F Luisi. “Hfq and its constellation of RNA.” Nature reviews. Microbiology 9, no. 8 (2011): 578-589.
</P></DIV></DIV></DIV><DIV id="footer-panel"><DIV class="centering"><DIV class="footerBox"><H4>This project is part of iGEM</H4><P>Read more on the <A href="https://2011.igem.org/Main_Page">iGEM Main Page</A></P></DIV><DIV class="footerBox"><H4>Sponsors</H4><P>Thanks to:</P></DIV><DIV class="footerBox"><H4>How to customize an iGEM wiki?</H4><P>Learn all the tricks in our <A href="/Team:DTU-Denmark/How_to_customize_an_iGEM_wiki">wiki guide</A></P></DIV></DIV></DIV><DIV class="printfooter">
Retrieved from &quot;<A href="http://2011.igem.org/Team:DTU-Denmark/Bioinformatic">http://2011.igem.org/Team:DTU-Denmark/Bioinformatic</A>&quot;</DIV></DIV></DIV><DIV id="footer-box" class="noprint"><DIV id="footer"><UL id="f-list"><LI id="t-recentchanges"><A href="/Special:RecentChanges" title="Recent changes">Recent changes</A></LI><LI id="t-whatlinkshere"><A href="/Special:WhatLinksHere/Team:DTU-Denmark/Bioinformatic" title="List of all wiki pages that link here [j]" accesskey="j">What links here</A></LI><LI id="t-recentchangeslinked"><A href="/Special:RecentChangesLinked/Team:DTU-Denmark/Bioinformatic" title="Recent changes in pages linked from this page [k]" accesskey="k">Related changes</A></LI><LI id="t-specialpages"><A href="/Special:SpecialPages" title="List of all special pages [q]" accesskey="q">Special pages</A></LI><LI><A href="/Special:Preferences">My preferences</A></LI></UL></DIV><DIV id="footer"><UL id="f-list"><LI id="t-print"><A href="/wiki/index.php?title=Team:DTU-Denmark/Bioinformatic&amp;printable=yes" title="Printable version of this page [p]" accesskey="p">Printable version</A></LI><LI id="t-permalink"><A href="/wiki/index.php?title=Team:DTU-Denmark/Bioinformatic&amp;oldid=158825" title="Permanent link to this revision of the page">Permanent link</A></LI><LI id="privacy"><A href="/2011.igem.org:Privacy_policy" title="2011.igem.org:Privacy policy">Privacy policy</A></LI><LI id="disclaimer"><A href="/2011.igem.org:General_disclaimer" title="2011.igem.org:General disclaimer">Disclaimers</A></LI></UL></DIV></DIV></DIV></BODY></HTML>