"
Page
Discussion
View source
History
teams
Log in
 
Team:DTU-Denmark-2/Project/Other assembly systems
From 2011.igem.org
Home
About us
The Team
Supervisors
Gallery
The Project
Abstract
Introduction
The Plug 'n' Play
Assembly system
Customization
Other assembly systems
Achievements
Attributions
Results
Background
Characterization
Proof of concept
Mammalian Cells
Fungi
Data page
Submitted Biobricks
Collaboration
Safety
Safety
Safety rules
Notebook
Lab notes
Protocols
Our Partners
Sponsors
Advisors
Acknowledgements
Other assembly systems
Comparison
Standard BioBrick Assembly
3A assembly
Gateway system
Gibson Assembly
In fusion
References
There are a wide range of techniques that facilitate cloning, and they all have their strengths and weaknesses. Here we compare the different techniques to the Plug 'n' Play assembly standard. You can read more about the different techniques by clicking on the menu in the left side.
Comparison table
Assembly system
Speed
Scars
Res. digestion
Multiple part assembly
Ease of use
Standard BioBrick
Slow
Yes
Yes
No, two
low/medium
3 A
Slow
Yes
Yes
No, two
low/medium
Gibson
Medium-Fast
No
No
Yes
medium
Gateway
Slow-Medium
No
No
Yes
low
In fusion
Fast
Yes/No
No
Yes
high
Plug'n Play
Fast
Yes/No
No
Yes
high
Standard BioBrick Assembly
The Standard Assembly of BioBricks was first developed by Tom Knight, and has subsequently been modified by other scientist to overcome some of the hurdles of the Standard Assembly. Only two parts larger than 200 bp can be assembled by the Standard assembly in each cycle. Furthermore, the parts should preferably be either 500 bp smaller or larger from each other as well as the backbone. The system is based on one of the parts to be assembled in the destination vector from the beginning. The assembly can be performed with either suffix insertion (insertion of the added part behind the existing part), or a prefix insertion (insertion of the added part in front of the existing part).
The way it's done
The Standard assembly makes use of the restriction recognition sites of four restriction enzymes; EcoRI and XbaI should be located upstream of the BioBrick and SpeI and PstI downstream of the BioBrick. The BioBricks can not contain the four mentioned restriction recognition sites. These restriction recognition sites are used to cut the BioBrick to be inserted obtain sticky ends and open the second receiving plasmid also obtaining sticky ends.
The resulting insert and open vector must be purified so unwanted and unspecified parts can be removed. The inserted and cut plasmids are mixed and under the right conditions, making the sticky ends to form hybridization. Subsequently, a ligation is performed to re-ligate in order  to form a plasmid that can be transformed into E. coli cells. [1] .
Difference between Plug'n Play assembly and Standard Assembly
A major disadvantage of the Standard assembly is the need for restriction digestions, ligations and the need for site-directed mutagenesis if more restriction recognition sites are present on the plasmid. The limitation in only assembling two parts at the time, makes the Standard assembly much more time consuming. Furthermore, the scars made by the assembling make it impossible to create fusion proteins by the Standard assembly.
3A assembly
3A (3 antibiotic) is a method for assembling two BioBrick parts at the time. 3A assembly relies on three way ligation (between the two parts and the backbone vector), thereby it differs from Standard Assembly, which uses two way ligation between a part and a part + vector. The 3A assembly was designed to make the gel purification of the digested parts unnecessary. Furthermore, antibiotic selection is used to eliminate unwanted background. The assembled parts can either be in two plasmids or generated by PCR.
The way it's done
The process of assembly with 3A resembles the Standard Assembly method. The 3A assembly uses, just like the Standard Assembly, the restriction recognition sites of EcoRI, SpeI, Xbal and PstI for flanking the BioBricks and destination vector.
Digestion of the two parts and the destination plasmid are performed so sticky ends are compatible with the wanted assembly of the vector. The restriction digestion and ligation has to be executed in two separate steps. After transformation in E. coli cells, positive/negative selection is performed. [2] .
Difference between Plug 'n' Play assembly and 3A assembly
The 3A method is based on use of restriction enzyme digestion and ligation, which means that illegal restriction recognition sites need to be eliminated. The method also leaves a scar between the BioBricks. Another disadvantage of the 3A assembly is the requirement for plasmids to contain three different antibiotic markers. Furthermore, the 3A method can only combine two parts at a time.
Gibson Assembly
Gibson Assembly is an isothermal, single-reaction method for assembling multiple overlapping DNA fragments. The method was developed by Daniel G. Gibson at the J. Craig Venter Institute in 2009.
The assembly system employs 5´-T5 exonuclease, Phusion DNA polymerase, and Taq ligase. Gibson can be used to assemble both ssDNA and dsDNA fragments. This method makes it possible to join DNA molecules that are as large as 583kb and clone joined products in E. coli with a length up to 300kb. Among the advantages is that it takes the same amount of time to ligate N number of DNA fragments as for two DNA fragments [3].
The way it's done
The isothermal 5´-T5 exonuclease removes the bases from the 5'-end of double stranded DNA, leaving a recess in the DNA. The ssDNA overhang is used to assemble the DNA fragments.
The T5 exonuclease is inactivated during the incubation at 50ºC. Phusion polymerase and Taq ligase fills the gaps of the annealed complementary ssDNA overhangs and seals the nicks in the ends, leaving a joined DNA molecule ready for transformation [3] .
Difference between Plug'n Play assembly and Gibson Assembly
One of the disadvantages of the Gibson assembly is that the primers for the assembly are more expensive due to the 40 bp extra nucleotides the primers have to be flanked with. The Gibson assembly is not as specific as USER cloning, where there is nicked at a uracil instead of chewing back from the end point.
Gateway assembly
The Gateway® cloning technology is based on a site-specific recombination system of the lambda bacteriophage provided by Invitrogen.  The lambda phage can change between the lytic and the lysogenic life cycle by enzymes called clonases. The recombination occurs between phage and DNA of E. coli via specific recombination sequences denoted as att sites (4).  On the phage genome attP sites are found, and on the host bacterial genome attB sites are found. After recombination the att sites, hybrid sequences contain sequence from both the phage and the host att site and are then called attL and attR (5). Gateway cloning enables the assembly of multiple DNA fragments. Taking advantage of the site-specific clonase enzymes and the att sites, the problems with conventional cloning method are avoided, which use restriction enzymes and ligase (6).
The way it's done
In the Gateway technology four different recombination sites attB, attP, attL and attR are utilized (4). attB sites always recombine with attP sites in a reaction mediated by the BP clonase, and attL-sites recombine with attR-sites mediated by LR clonase. Furthermore, attB1 reacts only with attP1 and not attP2, thereby maintaining the orientation of the transferred DNA fragment and the reading frame (5). In the Gateway® cloning system the final plasmid is obtained through two steps of cloning.  Through PCR amplification with primers flanked with attB BioBricks for the first recombination, the BP reaction. First PCR fragments with the appropriate att sites and orientation has to be constructed as shown in the figure below.
The Gateway Assembly can assemble up to 4 DNA fragments into one vector. The Gateway assembly is executed in the same way whether you join two or four DNA fragments. Depending on the number of fragments specific att sites are attached, always starting with attB1 and ending with attB2 (4) .
Difference between Plug'n Play assembly and Gateway assembly
The gateway assembly relies on a kit from Invitrogen. Furthermore, different entry clones and destination vectors are often needed and requires that a big library is established first. Furthermore, the biggest difference between the two assembly systems are the speed. The Gateway assembly takes longer time, because it relies on two cloning and transformation steps, before the final plasmid is obtained. Thus, the Gateway Assembly is far more complex than Plug 'n’ Play.
In fusion
In-fusion assembly is a method for assembling of two or more parts, provided by Clontech. The assembly system can be semi-standardized by simple primer design rules, minimizing the time used on planning the assembly reactions.
The way it's done
The PCR fragments are assembled with the use of at least 15bp homology on both ends. The forward primer of the first PCR fragment must be homologues to the reverse primer of the second PCR fragment and so forth. The assembly system can be seen below. It works with either the up-stream or down-stream PCR amplification of the vector and gene.
Afterwards the PCR fragments can be fused into a pre-engineered vector containing an antibiotic resistance gene, by creation of single-stranded regions made by the In-fusion enzyme. [5] .
Difference between Plug'n Play and In-Fusion assembly
Just like the Plug 'n' Play reaction, the In-Fusion reaction is fast and has a high efficiency. However, the main disadvantage of the In-Fusion assembly system is that it is expensive. .
References
[1] http://partsregistry.org/Assembly:Standard_assembly (website accessed 21.09.2011).
[2] http://openwetware.org/wiki/Synthetic_Biology:BioBricks/3A_assembly (website accessed 21.09.2011).
[3] Gibson, D.D et. al., 2009. Enzymatic assembly of DNA molecules up to several hundred kilobases. Nature Methods, vol. 6, no. 5, pp. 343-47
[4] http://tools.invitrogen.com/downloads/gateway-multisite-seminar.html (website accessed 21.09.2011).
[5] Hartley, J.L., Temple, G.F. & Brasch, M.A., 2000. DNA Cloning Using In Vitro Site-Specific Recombination. Genome Research, vol. 10, no. 11, pp.1788-95.
[6] Sasaki, Y. et al., 2004. Evidence for high specificity and efficiency of multiple recombination signals in mixed DNA cloning by the Multisite Gateway system. Journal of Biotechnology, vol. 107, no. 3, pp.233-43.
[7] Sleight, S.C., Bartley, B,A., Lieviant, J,A., and Sauro, H,M.,2010. In-Fusion BioBrick assembly and re-engineering. Nucleic Acids Research, Vol. 38, No. 8, pp. 2624–36.
Retrieved from " http://2011.igem.org/Team:DTU-Denmark-2/Project/Other_assembly_systems "
Recent changes
What links here
Related changes
Special pages
My preferences
Printable version
Permanent link
Privacy policy
Disclaimers
