<HTML xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en" dir="ltr">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki  ltr ns-0 ns-subject page-Team_DTU-Denmark_Modeling"><DIV id="globalWrapper"><DIV id="top-section"><DIV id="p-logo"><A href="/Main_Page" title="Main Page">&quot;
	    </A></DIV><DIV id="menubar" class="left-menu noprint"><UL><LI class="selected"><A href="/Team:DTU-Denmark/Modeling">Page               </A></LI><LI class="new"><A href="/wiki/index.php?title=Talk:Team:DTU-Denmark/Modeling&amp;action=edit&amp;redlink=1">Discussion               </A></LI><LI><A href="/wiki/index.php?title=Team:DTU-Denmark/Modeling&amp;action=edit">View source               </A></LI><LI><A href="/wiki/index.php?title=Team:DTU-Denmark/Modeling&amp;action=history">History               </A></LI><LI style="color:#808080;cursor:default">teams</LI></UL></DIV><DIV class="right-menu noprint" id="menubar"><UL><LI id="pt-login"><A href="/wiki/index.php?title=Special:UserLogin&amp;returnto=Team:DTU-Denmark/Modeling" title="You are encouraged to log in; however, it is not mandatory [o]" accesskey="o">Log in</A></LI></UL></DIV><DIV id="search-controls" class="noprint"><FORM action="/Special:Search" id="searchform"> </FORM></DIV></DIV><DIV id="content"><H1 class="firstHeading">Team:DTU-Denmark/Modeling</H1><DIV id="bodyContent"><H3 id="siteSub" class="noprint">From 2011.igem.org</H3><DIV id="navigation"><DIV class="navigation"><DIV class="centering"><UL class="topnav" id="topnav"><LI class="home navitem1"><A href="/Team:DTU-Denmark">Home</A></LI><LI class="vision navitem2"><A href="/Team:DTU-Denmark/Vision" title="Test: A soft introduction to the idea and it's future applications.">Vision</A></LI><LI class="project navitem3"><A href="/Team:DTU-Denmark/Project">Project</A></LI><LI><A href="/Team:DTU-Denmark/Project">Overview</A></LI><LI><A href="/Team:DTU-Denmark/Bioinformatic">Bioinformatics</A></LI><LI><A href="/Team:DTU-Denmark/Project_testing_sRNA">Experiment: Testing sRNA</A></LI><LI><A href="/Team:DTU-Denmark/Project_improving_araBAD">Experiment: Improving araBAD</A></LI><LI><A href="/Team:DTU-Denmark/Modeling">Modeling</A></LI><LI><A href="/Team:DTU-Denmark/Data">Data page</A></LI><LI><A href="/Team:DTU-Denmark/Attributions">Attributions</A></LI></UL><LI class="technical_stuff navitem4"><A href="/Team:DTU-Denmark/Technical_stuff">Technical stuff</A></LI><LI><A href="/Team:DTU-Denmark/Parts">Parts</A></LI><LI><A href="/Team:DTU-Denmark/Notebook">Notebook</A></LI><LI><A href="/Team:DTU-Denmark/Protocols">Protocols</A></LI><LI><A href="/Team:DTU-Denmark/Technical_stuff_math">Math</A></LI><LI><A href="/Team:DTU-Denmark/Matlab">Matlab</A></LI><LI><A href="/Team:DTU-Denmark/Safety">Safety</A></LI><LI class="background navitem5"><A href="/Team:DTU-Denmark/Background">Background</A></LI><LI><A href="/Team:DTU-Denmark/Background_sRNA">Small RNA</A></LI><LI><A href="/Team:DTU-Denmark/Background_the_natural_system">The chitobiose system</A></LI><LI><A href="/Team:DTU-Denmark/Background_synthetic_biology">Synthetic biology</A></LI><LI class="team navitem6"><A href="/Team:DTU-Denmark/Team">Team</A></LI></DIV></DIV></DIV><DIV id="heading"><DIV class="centering"><H1>

Modeling

            </H1></DIV></DIV><DIV id="innercontent"><DIV class="centering"><DIV class="whitebox article"><TABLE id="toc" class="toc"><TBODY><TR><TD><DIV id="toctitle"><H2>Contents</H2></DIV><UL><LI class="toclevel-1 tocsection-1"><A href="#Motivation"><SPAN class="tocnumber">1</SPAN><SPAN class="toctext">Motivation</SPAN></A></LI><LI class="toclevel-1 tocsection-2"><A href="#General_kinetic_model"><SPAN class="tocnumber">2</SPAN><SPAN class="toctext">General kinetic model</SPAN></A></LI><LI class="toclevel-1 tocsection-3"><A href="#Model_1:_Catalytic_trap-RNA"><SPAN class="tocnumber">3</SPAN><SPAN class="toctext">Model 1: Catalytic trap-RNA</SPAN></A></LI><LI class="toclevel-1 tocsection-4"><A href="#Model_2:_Partly_stoichiometric_trap-RNA"><SPAN class="tocnumber">4</SPAN><SPAN class="toctext">Model 2: Partly stoichiometric trap-RNA</SPAN></A></LI><LI class="toclevel-1 tocsection-5"><A href="#Design_of_dynamic_range"><SPAN class="tocnumber">5</SPAN><SPAN class="toctext">Design of dynamic range</SPAN></A></LI><LI class="toclevel-1 tocsection-6"><A href="#Dynamics"><SPAN class="tocnumber">6</SPAN><SPAN class="toctext">Dynamics</SPAN></A></LI><LI class="toclevel-1 tocsection-7"><A href="#Parameters_and_proposed_experiments"><SPAN class="tocnumber">7</SPAN><SPAN class="toctext">Parameters and proposed experiments</SPAN></A></LI><LI class="toclevel-1 tocsection-8"><A href="#Estimating_the_role_of_Hfq"><SPAN class="tocnumber">8</SPAN><SPAN class="toctext">Estimating the role of Hfq</SPAN></A></LI><LI class="toclevel-1 tocsection-9"><A href="#Conclusion"><SPAN class="tocnumber">9</SPAN><SPAN class="toctext">Conclusion</SPAN></A></LI><LI class="toclevel-1 tocsection-10"><A href="#References"><SPAN class="tocnumber">10</SPAN><SPAN class="toctext">References</SPAN></A></LI></UL></TD></TR></TBODY></TABLE><H2><SPAN class="mw-headline" id="Motivation"> Motivation </SPAN></H2><P>A model is developed with the dual purpose of both developing hypotheses about the trap-RNA system and providing synthetic biologists the <B>framework</B> to characterize and incorporate the system into models of their designs. Several models for one-level sRNA-mRNA regulation exits in the literature<SPAN class="superscript"><A href="#References">[1]</A></SPAN><SPAN class="superscript"><A href="#References">[2]</A></SPAN>, but the two-level regulation has not previously been quantitatively characterised nor modelled.
</P><H2><SPAN class="mw-headline" id="General_kinetic_model"> General kinetic model </SPAN></H2><DIV class="thumb tright"><DIV class="thumbinner" style="width:402px;"><DIV class="thumbcaption"><B>Overview of kinetic model</B> indicating the catalytic or stoichiometric difference between model 1 and model 2. Blue is target mRNA, red is small RNA and green is trap-RNA</DIV></DIV></DIV><P>The modeling of the trap-RNA system is based on two general reaction schemes
\begin{align}
\color{blue} m + \color{red}s &amp;\mathop{\rightleftharpoons}_{k_{-1,s}}^{k_{1,s}} c_{ms} \mathop{\rightarrow}^{k_{2,s}} (1 - p_s) \color{red} s \\
\color{red} s + \color{green} r &amp;\mathop{\rightleftharpoons}_{k_{-1,r}}^{k_{1,r}} c_{sr} \mathop{\rightarrow}^{k_{2,r}} (1-p_r) \color{green} r
\end{align} 
</P><P>In the <B>first reaction</B> ${\color{blue} m}$RNA binds to a ${\color{red}s}$RNA forming a complex called $c_{ms}$. The RNAs of the duplex are then irreversibly degraded with stoichiometries defined by $p_s$, which denotes the probability that $s$ is co-degraded in the reaction. The majority of investigated small regulatory RNA (srRNA) acts stoichiometrically, they are degraded 1:1 with their target mRNA corresponding to $p_s = 1$. Interestingly, studies indicate that the trap-RNA system acts catalytically with $p_s = 0$<SPAN class="superscript"><A href="#References">[3]</A></SPAN>. In the <B>second reaction</B> t$\color{green} r$ap-RNA inhibits the activity of the sRNA but the mechanism is unknown. The general scheme allows two different hypotheses; trap-RNA mediates degradation of the sRNA either catalytically or stoichiometrically.
</P><P><I>In vivo</I> genes and their RNAs are constantly being expressed and degraded giving rise to finite lifetimes of RNA molecules. To model the trap-RNA system in context of living cells the expression of each RNA is described by a <B>production term</B> $\alpha_i$ and the turnover of each RNA is described by a <B>degradation term</B> of first order $\beta_i[RNA]_i$.
</P><P>\begin{equation}
	\label{proddeg}
	\mathop{\longrightarrow}^{production} [RNA]_i \mathop{\longrightarrow}^{degradation}
\end{equation}
</P><P>Eventually, the amount of RNA will settle into a steady-state where production equals degradation.
</P><P>Ordinary <B>differential equations</B> (ODEs) are set up by applying the law of mass action to the general reaction scheme, the production and background degradation. The ODEs are then simplified by assuming pseudo-steady state of the RNA complexes $c_{ms} $ and $ c_{sr}$. In the general reaction scheme it follows (See <A href="/Team:DTU-Denmark/Technical_stuff_math" title="Team:DTU-Denmark/Technical stuff math">ODEs</A>) that
</P><P>\begin{eqnarray}
	\label{system1}
	\frac{dm}{dt} &amp;=&amp; \alpha_m - \beta_m m - k_s ms
\\	\label{system2}
\frac{ds}{dt} &amp;=&amp; \alpha_s - \beta_s s - p_s k_s ms - k_r sr
\\	\label{system3}
\frac{dr}{dt} &amp;=&amp; \alpha_r - \beta_r r - p_r k_r sr
\end{eqnarray}
</P><P>where the kinetic constants of the general reaction scheme have been lumped into the overall kinetic constants $k_s = \frac{k_{1,s} k_{2,s}}{k_{-1,s} + k_{2,s}}$ and $k_r = \frac{k_{1,r} k_{2,r}}{k_{-1,r} + k_{2,r}}$. It is seen from the equations that setting either $p_s = 0$ or $p_r = 0$ simplifies the expression. Because $p_s = 0$ for the trap-RNA system whereas the value of $p_r$ is unknown, the general model is split up into <B>two models</B>; a catalytic and a partly stoichiometric trap-RNA mechanism corresponding to $p_r = 0$ or $0 &lt; p_r \leq 1$ respectively.
</P><H2><SPAN class="mw-headline" id="Model_1:_Catalytic_trap-RNA"> Model 1: Catalytic trap-RNA </SPAN></H2><P>Assuming $p_s = 0$ and $p_r = 0$ the <B>steady-state solution</B> is derivable and unique. It is furthermore stable because all the eigen values of the <I>Jacobian</I> have negative real parts (see <A href="/Team:DTU-Denmark/Technical_stuff_math#Steady-state_analysis" title="Team:DTU-Denmark/Technical stuff math"> Steady-state</A>). The steady-state of m is
</P><P>\begin{equation}
	\label{ss}
	m^* = \frac{\alpha_m (\beta_r \beta_s + \alpha_r k_r)}{\beta_r \alpha_s k_s + \beta_m \beta_r \beta_s + \beta_m \alpha_r k_r}
\end{equation}
</P><P>To eliminate the dependence on $\alpha_m$ the steady-state is scaled with respect to the maximum amount of expression $m^*_{max} = \frac{\alpha_m}{\beta_m}$. This maximum value of $m$ expression is achieved by not having any sRNA which is equivalent to $\alpha_s = 0$. A new measure of steady-state $\frac{m^*_{max}}{m^*}$ is interpreted as the <B>fold repression</B> caused by the system. To analyze how this fold repression depends on the variables and parameters, the steady-state solution is scaled and re-stated into its simpler form
</P><P>\begin{equation}
	\label{phi1}
	\phi =	\frac{m^*_{max}}{m^*} = 1 + \frac{\frac{k_s \alpha_s}{\beta_m \beta_s}}{1 + \frac{k_r \alpha_r}{\beta_s \beta_r}}
\end{equation}
</P><P>where $\phi$ is a measure of the fold repression.
</P><DIV class="center"><DIV class="thumb tnone"><DIV class="thumbinner" style="width:667px;"><DIV class="thumbcaption">3D <B>contour of model 1</B> showing $\frac{1}{\phi}$, target gene expression, plotted against induction of r and s. Blue is depicting low values of target mRNA corresponding to the <B>off state</B>. Red is marking high values of target mRNA corresponding to the <B>on state</B>. Parameters are estimated from literature.</DIV></DIV></DIV></DIV><H2><SPAN class="mw-headline" id="Model_2:_Partly_stoichiometric_trap-RNA"> Model 2: Partly stoichiometric trap-RNA </SPAN></H2><P>Assuming $p_s = 0$ and $0 &lt; p_r \leq 1$ the <B>steady-state solution</B> with respect to m is
\begin{equation}
	m^* = \frac{2}{\beta_m} \bigg( \frac{p_r \alpha_m \lambda_s}{\alpha_s p_r - \alpha_r -\lambda_r + 2 p_r \lambda_s + \sqrt{(\alpha_s p_r - \alpha_r - \lambda_r)^2 + 4 p_r \alpha_s \lambda_r}} \bigg)
\end{equation}
where $\lambda_r = \frac{\beta_s \beta_r}{k_r}$ and $\lambda_s = \frac{\beta_s \beta_m}{k_s}$. The fold repression is again defined removing the dependence on $\alpha_m$
\begin{equation}
	\label{phi2}
		\phi =	\frac{m^*_{max}}{m^*} = 1 + \frac{\alpha_s p_r - \alpha_r -\lambda_r + \sqrt{(\alpha_s p_r - \alpha_r - \lambda_r)^2 + 4 p_r \alpha_s \lambda_r}}{2 p_r \lambda_s}
\end{equation}
The fold repression for the two models  are reassuringly equivalent when $\alpha_r = 0$, which corresponds to ignoring the second reaction. This makes sense since the first reaction is identical for both dual degradation models.
</P><TABLE><TBODY><TR><TH> Parameter
</TH><TH> Meaning
</TH><TH> Unit
</TH></TR><TR><TD> $\alpha_m$
</TD><TD> Transcription rate of target mRNA
</TD><TD> [amount/time]
</TD></TR><TR><TD> $\alpha_s$
</TD><TD> Transcription rate of sRNA (s)
</TD><TD> [amount/time]
</TD></TR><TR><TD> $\alpha_r$
</TD><TD> Transcription rate of sRNA (r)
</TD><TD> [amount/time]
</TD></TR><TR><TD> $\beta_m$
</TD><TD> Degradation rate of free mRNA
</TD><TD> [1/time]
</TD></TR><TR><TD> $\beta_s$
</TD><TD> Degradation rate of free sRNA (s)
</TD><TD> [1/time]
</TD></TR><TR><TD> $\beta_r$
</TD><TD> Degradation rate of free sRNA (r)
</TD><TD> [1/time]
</TD></TR><TR><TD> $p_s$
</TD><TD> Probability that s is codegraded with m
</TD><TD> [-]
</TD></TR><TR><TD> $p_r$
</TD><TD> Probability that r is codegraded with s
</TD><TD> [-]
</TD></TR><TR><TD> $k_s$
</TD><TD> Kinetic constant of sRNA mediated degradation of mRNA
</TD><TD> [1/(time*amount)]
</TD></TR><TR><TD> $k_r$
</TD><TD> Kinetic constant of trap-RNA mediated degradation of sRNA
</TD><TD> [1/(time*amount)]
</TD></TR><TR><TD> $\lambda_s$
</TD><TD> Leakage rate of sRNA mediated degradation of mRNA
</TD><TD> [amount/time]
</TD></TR><TR><TD> $\lambda_r$
</TD><TD> Leakage rate of trap-RNA mediated degradation of sRNA
</TD><TD> [amount/time]
</TD></TR></TBODY></TABLE><H2><SPAN class="mw-headline" id="Design_of_dynamic_range"> Design of dynamic range </SPAN></H2><DIV class="thumb tright"><DIV class="thumbinner" style="width:402px;"><DIV class="thumbcaption"><B>Heatmap</B> of model 1 from projection of 3D contour with indicated dynamic ranges. Red arrow is the <B>direction of dynamic range</B> when changing induction of sRNA. Green arrow is the direction of the dynamic range when changing the induction of trap-RNA. The actual dynamic ranges are the target gene expression 'on' the arrows. Notice that the length of dynamic range is finite, restricting functionality to steady-state maps with desirable properties within the limits of biological plausible induction levels.</DIV></DIV></DIV><P>When only sRNA and target mRNA is present the steady-state of both model 1 and 2 is
</P><P>\begin{equation}
	\label{lin}
	\phi = 1 + \frac{ \alpha_s}{\lambda_s}
\end{equation}
</P><P>which describes the <B>baseline dynamic range</B> corresponding to a dynamic range when changing the induction of sRNA, $\alpha_s$. Visually the baseline dynamic range corresponds to starting at the lower left corner of a heatmap and proceeding to the right. 
</P><P>The effect of the parameter $\lambda_s = \frac{\beta_m \beta_s}{k_s}$ on the baseline dynamic range is valid for both models and determines the sensitivity to $\alpha_s$.
</P><P>Taking trap-RNA into account adds another component to the dynamic range. For model 1, the dynamic range of the fold repression when changing the induction of trap-RNA, $\alpha_r$, is described by the function
</P><P>\begin{equation}
	\phi(\alpha_r) = 1 + \frac{\lambda_s^{-1} \alpha_s}{1+\lambda_r^{-1} \alpha_r}
	\label{dyn}
\end{equation}
</P><P>The form of $\phi(\alpha_r)$ depends on the two effective parameter groups $\lambda_s^{-1} \alpha_s$ and $\lambda_r$. The parameter group $\lambda_s^{-1} \alpha_s$ governs the fold repression at $\alpha_r = 0$ - the starting point of the dynamic range. The other parameter group $\lambda_r$ governs the sensitivity with respect to $\alpha_r$. In a biological perspective both parameter groups are flexible because the underlying parameters are changeable. Especially, $\alpha_s$ is changeable by simply altering the induction of sRNA but also $k_r$ might be changeable by altering the binding affinity of sRNA and trap-RNA. Thus, the catalytic trap-RNA model suggests a <B>highly modular dynamic range</B>. Because induction is restricted to some biological possible levels it is important to design a gene silencing system to be within a functional dynamic range. When only sRNA and target mRNA is considered it is sufficient to obtain the desired fold repression of the target mRNA. For the full trap-RNA system the tradeoff for high fold repression at the <I>off state</I> is unwanted high fold repression at the <I>on state</I>. For the catalytic model the dynamic range when changing $\alpha_r$ is more complicated and investigated by comparing different heatmaps.
</P><DIV class="center"><DIV class="thumb tnone"><DIV class="thumbinner" style="width:667px;"><DIV class="thumbcaption">Heatmap comparison of stoichiometric model for a 10-fold decrease and increase of $\lambda_r$ and $\lambda_s$. The scaling of the axes are the same in all five subgraphs; the x-axes are $\alpha_s$, the y-axes are $\alpha_r$. $\lambda_r$ governs the steepness in the $\alpha_r$ direction. $\lambda_s$ governs the onset of repression in the $\lambda_s$ direction.</DIV></DIV></DIV></DIV><P>Another quality of the dynamic range curve is <B>steepness</B>. A steeper curve widens the dynamic range but a flatter curve makes it easier to fine-tune particular levels of fold repression.
</P><H2><SPAN class="mw-headline" id="Dynamics"> Dynamics </SPAN></H2><DIV class="thumb tright"><DIV class="thumbinner" style="width:402px;"><DIV class="thumbcaption">Dynamics simulation of trap-RNA system. The two arrows indicate induction events of sRNA $\alpha_s = 50$ and trap-RNA $\alpha_r = 200$. The steady-state levels of target mRNA are configurable by altering the induction levels.</DIV></DIV></DIV><P>The temporal behavior is investigated using Runge-Kutta algorithm for numerical solution of differential equations. The equations used are the model 1 ODEs on a <B>dimensionless form</B> to facilitate the interpretation (see <A href="/Team:DTU-Denmark/Technical_stuff_math#Dimensionless_analysis" title="Team:DTU-Denmark/Technical stuff math"> derivation</A>). The parameters are based on estimates from literature. The simulation indicate a fast response time on the order of minutes when silencing target gene and a longer response time when reactivating the target gene.
</P><P>The temporal behavior indicates the possibility of pulse experiments where any gene can be silenced or partly silenced upon induction and then subsequently reactivated.
</P><H2><SPAN class="mw-headline" id="Parameters_and_proposed_experiments"> Parameters and proposed experiments </SPAN></H2><P>In order to use the expressions for steady-state for empirical analysis the fold repression, $\phi$, must somehow be measurable. To arrive at some expression relating $\phi$ to empirical data, the steady-state of the expressed target gene is assumed to be proportional to the steady-state of the target mRNA. Using translational or transcriptional fusion of a reporter gene to the target gene the fold repression can be approximated by
</P><P>\begin{equation}
	\phi \approx \frac{[A_{max}]}{[A]}
\end{equation}
</P><P>where $[A]$ is some measure proportional to the amount of reporter gene and $[A_{max}]$ is some measure at maximum expression. The measures of reporter gene could be either fluorescence from GFP or absorbance of X-gal product due to LacZ activity.
</P><P>The parameters of $\lambda_s = \frac{\beta_m \beta_s}{k_s}$ and $\lambda_r = \frac{\beta_s \beta_r}{k_r}$, which arises in both steady-state solutions can be interpreted as <B>leakage rates</B>. In other words the ratio between the background degradation rate and the mediated specific degradation rate. The leakage rate $\lambda_s$ can be determined by fitting $\phi$ against $\alpha_s$. Similarly $\lambda_s$ and $\lambda_r$ can be determined by fitting $\phi$ to values of $\alpha_s$ and $\alpha_r$ using steady-state equations for either model 1 or model 2. The goodness of fit should provide evidence discerning the two alternative hypotheses of catalytic or partly stoichiometric trap-RNA.
</P><P>The production terms $\alpha_m$, $\alpha_s$ and $\alpha_r$ can be constant or variable depending on inducers. Inducible promoter activity can be approximated by a simple linear function or by the Hill binding equation
</P><P>\begin{equation}
	\alpha = \frac{\alpha_{max} [I]^n}{K_d + [I]^n}
\end{equation}
</P><P>where $[I]$ is the inducer concentration, $n$ is the Hill coefficient, $\alpha_{max}$ is the maximum production rate at full saturation and $K_d$ is the binding coefficient for the inducer-promoter complex.
</P><P>The background degradation rates can be modeled as first order decay processes and determined by fitting to
</P><P>\begin{equation}
	N(t) = N_0e^{-\beta t}
\end{equation}
</P><P>where $N(t)$ is the amount of RNA and intracellular production of RNA is stopped at $t=0$. Experiments indicate<SPAN class="superscript"><A href="#References">[3]</A></SPAN> that $\beta_s = \beta_m = 0.0257 min ^{-1}$.
</P><H2><SPAN class="mw-headline" id="Estimating_the_role_of_Hfq"> Estimating the role of Hfq </SPAN></H2><P>Hfq is necessary for the system to work and inherently related to the kinetic $k_s$ and $k_r$ defined by
</P><P>\begin{equation}
	k = \frac{k_1 k_2}{k_{-1} + k_2}
\end{equation}
</P><P>The dissociation constant is defined by $K_d = \frac{k_{-1}}{k_1}$. To relate $K_d$ to $k$ the on rate $k_1$ is assumed to be constant with respect to $K_d$ and any change in $K_d$ is assumed only to affect the off-rate $k_{-1}$. By inserting $k_{-1} = k_1 K_d$ it follows that
</P><P>\begin{equation}
	\frac{1}{k} = \frac{1}{k_1} + \frac{1}{k_2} K_d
\end{equation}
</P><P>Using experimentally determined values of $k$ and $K_d$ prediction from secondary RNA structure predictors, eg. RNA fold or Vienna package, it is possible to investigate the relationship.
</P><P>The role of Hfq in RNA-RNA hybridization has not been fully elucidated. A proposed mechanism is, that Hfq increases the on rate by some factor. If Hfq works by increasing the local concentration of RNA the factor would describe the increased probability of RNA collision caused by the presence of Hfq. Incorporating this idea we define the following altered on rate $k_{1} = \alpha_H k_{1}^{\circ}$ where $\alpha_H \geq 1$ and get
\begin{equation}
	\frac{1}{k} = \frac{1}{\alpha_H k_1^{\circ}} + \frac{1}{k_2} K_d
\end{equation}
An expression which could guide experiments aimed at investigating the role of Hfq in post-transcriptional regulation.
</P><H2><SPAN class="mw-headline" id="Conclusion"> Conclusion </SPAN></H2><P>The modeling of the trap-RNA system provides a <B>framework</B> for characterization and subsequent application in rational design or gene silencing experiments. The developed methods of characterization are general and applies to all potential target genes. The <B>fold repression</B> is independent on the amount of target mRNA, and hence system characterization is valid for any target mRNA expression level. This significantly simplifies applying the trap-RNA system because the trap-RNA system can be seen as an independent component simply lowering the expression by some factor. Temporal <B>simulation</B> of the model can be used to calculate the time necessary for steady-state to occur. The simulation indicates that gene silencing occurs fast, on the order of minutes, whereas reactivation is slower due to transcript build-up.
</P></DIV><DIV class="whitebox article"><H2><SPAN class="mw-headline" id="References"> References </SPAN></H2><P>[1] Levine, Erel, Zhongge Zhang, Thomas Kuhlman, and Terence Hwa. <I>Quantitative characteristics of gene regulation by small RNA.</I> PLoS biology. 5, no. 9 (2007).
</P><P>[2] Mitarai, Namiko, Julie-Anna M Benjamin, Sandeep Krishna, Szabolcs Semsey, Zsolt Csiszovszki, Eric Massé, and Kim Sneppen. <I>Dynamic features of gene expression control by small regulatory RNAs.</I> Proceedings of the National Academy of Sciences of the United States of America 106, no. 26 (2009): 10655-10659.
</P><P>[3] Overgaard, Martin, Jesper Johansen, Jakob Møller‐Jensen, and Poul Valentin‐Hansen. <I>Switching off small RNA regulation with trap‐mRNA.</I> Molecular Microbiology 73, no. 5 (September 2009): 790-800. <A href="http://onlinelibrary.wiley.com/doi/10.1111/j.1365-2958.2009.06807.x/abstract" class="external free" rel="nofollow">http://onlinelibrary.wiley.com/doi/10.1111/j.1365-2958.2009.06807.x/abstract</A>.
</P></DIV></DIV></DIV><DIV id="footer-panel"><DIV class="centering"><DIV class="footerBox"><H4>This project is part of iGEM</H4><P>Read more on the <A href="https://2011.igem.org/Main_Page">iGEM Main Page</A></P></DIV><DIV class="footerBox"><H4>Sponsors</H4><P>Thanks to:</P></DIV><DIV class="footerBox"><H4>How to customize an iGEM wiki?</H4><P>Learn all the tricks in our <A href="/Team:DTU-Denmark/How_to_customize_an_iGEM_wiki">wiki guide</A></P></DIV></DIV></DIV><DIV class="printfooter">
Retrieved from &quot;<A href="http://2011.igem.org/Team:DTU-Denmark/Modeling">http://2011.igem.org/Team:DTU-Denmark/Modeling</A>&quot;</DIV></DIV></DIV><DIV id="footer-box" class="noprint"><DIV id="footer"><UL id="f-list"><LI id="t-recentchanges"><A href="/Special:RecentChanges" title="Recent changes">Recent changes</A></LI><LI id="t-whatlinkshere"><A href="/Special:WhatLinksHere/Team:DTU-Denmark/Modeling" title="List of all wiki pages that link here [j]" accesskey="j">What links here</A></LI><LI id="t-recentchangeslinked"><A href="/Special:RecentChangesLinked/Team:DTU-Denmark/Modeling" title="Recent changes in pages linked from this page [k]" accesskey="k">Related changes</A></LI><LI id="t-specialpages"><A href="/Special:SpecialPages" title="List of all special pages [q]" accesskey="q">Special pages</A></LI><LI><A href="/Special:Preferences">My preferences</A></LI></UL></DIV><DIV id="footer"><UL id="f-list"><LI id="t-print"><A href="/wiki/index.php?title=Team:DTU-Denmark/Modeling&amp;printable=yes" title="Printable version of this page [p]" accesskey="p">Printable version</A></LI><LI id="t-permalink"><A href="/wiki/index.php?title=Team:DTU-Denmark/Modeling&amp;oldid=168436" title="Permanent link to this revision of the page">Permanent link</A></LI><LI id="privacy"><A href="/2011.igem.org:Privacy_policy" title="2011.igem.org:Privacy policy">Privacy policy</A></LI><LI id="disclaimer"><A href="/2011.igem.org:General_disclaimer" title="2011.igem.org:General disclaimer">Disclaimers</A></LI></UL></DIV></DIV></DIV></BODY></HTML>