"
Page
Discussion
View source
History
teams
Log in
 
Team:DTU-Denmark-2/Project/PlugnplayAssembly/customization
From 2011.igem.org
Home
About us
The Team
Supervisors
Gallery
The Project
Abstract
Introduction
The Plug 'n' Play
Assembly system
Customization
Other assembly systems
Achievements
Attributions
Results
Background
Characterization
Proof of concept
Mammalian Cells
Fungi
Data page
Submitted Biobricks
Collaboration
Safety
Safety
Safety rules
Notebook
Lab notes
Protocols
Our Partners
Sponsors
Advisors
Acknowledgements
The system is customizable!!
When do you need to customize the system?
Seamless assembly
Point mutations
Deletions
Adding short sequences
Example
Linker table
Tips & Tricks
References
Customization of the Plug 'n' Play with DNA assembly standard opens the door to a wide range of applications.
When do you need to customize the system?
When a desired biological part is not yet a part of the Plug'n'Play with DNA kit.
When seamless assembly is required (e.g. protein fusion).
When the introduction of mutations are required.
When the introduction of deletions are required.
When short sequences such as linkers, RBSs, signal peptides etc. are required.
A guide for the most common design requests is given below. The guide is based on work conducted by Morten H. H. Nørholm and Hansen et. al. Figures obtained from Genee et al. (in preparation).
Creating new parts
Creating new biological parts can be necessary, when they are not already included in the Plug 'n' Play kit and submitted to the Registry of Standard Biological Parts.
1. To create a new part, forward and reverse primers have to be designed for the DNA of interest. The free-ware netprimer is an excellent tool for finding suitable primers.
2. In accordance with the Plug’n’Play assembly system the category of the part i.e. promoter, GOI, TS, module, or marker cassette has to be determined.
3. Subsequently the linkers belonging to the category have to be identified. The appropriate linkers and Plug'n'Play tail for the part can be found in the linker table . The forward and reverse primer linkers are 8-9 bp long, and are placed at the 5’-end of the primers.
5. Order the primers from your favourite oligo company. Notice that the price can be a bit higher and delivery time a bit longer, when ordering primers with a uracil incorporated. We have had a good experience with the price, delivery time and quality of the primers ordered from Integrated DNA Technology.
6. After receiving the primers, you are ready to perform the PCR (see protocol Amplification of biobricks by PCR ). Be aware that the addition of the linker to the primer may increase the melting temperature of the primer.
7. Check the band size of the PCR product on an agarose-gel. The new BioBrick can be used directly or purified by agarose-gel purification.
8. The assembly of the new BioBricks should be performed according to the protocol USER cloning .
9. You should now have assembled one new plasmid or linear device.
Seamless assembly
Seamless assembly is often required when constructing fusion protein or if large genetic parts have to be assembled in pieces, e.g. when constructing  complex enzymes and synthases for fungi.
1. If you want to design a construct where all parts are seamlessly assembled go directly to step 2. Otherwise the standard linkers should be added to the parts that do not need to be fused seamless according to the linker table . For more details see the 'creating new parts' section.
2. To design the primers for the seamless assembly the software PHUSER can be used. This software was created by the DTU iGEM team from 2009.
3. The primers are ordered from your favourite company. Notice that the price can be a bit higher and delivery time a bit longer, when ordering primers with a uracil incorporated. We have had a good experience with the price, delivery time and quality of the primers ordered from Integrated DNA Technology.
4. To amplify the BioBricks the protocol Amplification of biobricks by PCR can be used.
5. Check the PCR reaction on an agarose-gel before assembly. The new BioBricks can be used directly or purified by agarose-gel purification.
6. The assembly of new BioBricks is executed according to the protocol USER cloning .
7. A scar free BioBrick device or plasmid should now have been assembled.
Point mutations
The introduction of point mutations can have different purposes, for instance the removal of undesired restriction site or for alteration of a proteins catalytic function. When introducing mutations it is only necessary for the primers to overlap in the complementary region (linker). Depending on whether the point mutation should be degenerate or not only one or both primers needs to carry the mutation. Degeneration means that more than one codon may code for the same amino acid. Thereby, a degenerate mutation can give different outcomes in use of tRNA's.
Introducing a point mutation with non-degenerate codons
Both primers have to carry the desired mutation to ensure that the codon and the resulting amino acid are generated by the mutation.
1. To introduce a point mutation in a vector, the mutation has to be defined first.
2. To design the primers the software PHUSER can be used. Remember to introduce the desired mutation into the primer. Two primers are used per mutation.
3. The primers are ordered from your favourite company. Notice that the price can be a bit higher and delivery time a bit longer, when ordering primers with a uracil incorporated. We have had a good experience with the price, delivery time and quality of the primers ordered from Integrated DNA Technology.
4. To amplify the BioBricks the protocol Amplification of biobricks by PCR can be used.
5. Check the PCR reaction on an agarose gel before assembly. The new BioBricks can be used directly or purified by agarose-gel purification if needed.
6. The assembly of new BioBricks should be performed according to the protocol USER cloning .
7. A BioBrick, device or plasmid with the desired point mutation should now have been assembled.
Introducing a degenerate point mutation, which can give a different codon usage
Only one primer has to carry the desired point mutation. The procedure of assembly is the same as for the non-degenerate point mutation above.
It is also possible to introduce several point mutations in different regions by customizing the Plug 'n' Play assembly standard. This only requires one round of PCR and one round of cloning. The difference is that the PCR fragments made for each mutation will have to be fused together meaning that the introduction of two mutations would require the fusion of three PCR fragments, and so on.
Deletions
With this guide a gene or part of a gene or any other desired sequence of a plasmid, device, or BioBrick can be deleted.
1. First the site of the deletion should be defined.
2. The primers need to be designed so that the fusion point will bridge two desired sections of the gene at the matching USER linkers.
3. The primers are ordered from your favourite company. Notice that the price can be a bit higher and delivery time a bit longer, when ordering primers with a uracil incorporated. We have had a good experience with the price, delivery time and quality of the primers ordered from Integrated DNA Technology.
4. To amplify the BioBricks the protocol Amplification of biobricks by PCR can be used.
5. Check the PCR reaction on an agarose gel before assembly. The new BioBricks can be used directly or purified by agarose-gel purification if needed.
6. The assembly of new BioBricks should be performed according to the protocol USER cloning .
7. A BioBrick, device or new plasmid with the deletion should now have been assembled.
Adding short sequences
Short sequences such as ribosomal binding sites, kozak sequences etc. can be added by incorporating them into the primer that will be used to amplify the part it should be fused with. Introducing a short sequence can also be of interest when two proteins are wished to be connected by a linker. The linker can also carry a single point mutation, be degenerate or non-degenerate, which means that only one or both primers needs to carry the mutation.  When creating a linker a short sequence of either random or known sequence is added by incorporating the sequence into the forward or reverse primers.
Introducing a linker
1. When introducing a linker in the connection with a gene of interest (GOI), the first step is to define the mutation.
2. Primers are designed for the defined mutation site. To introduce the desired mutation one nucleotide should be changed in the primer. Four primers are used per mutation.
3. The primers are ordered from your favourite company. Notice that the price can be a bit higher and delivery time a bit longer, when ordering primers with a uracil incorporated. We have had a good experience with the price, delivery time and quality of the primers ordered from Integrated DNA Technology.
4. To amplify the BioBricks the protocol Amplification of biobricks by PCR can be used.
5. Check the PCR reaction on an agarose gel before assembly. The new BioBricks can be used directly or purified by agarose-gel purification if needed.
6. The assembly of new BioBricks and parts should be performed according to the protocol USER cloning .
The figure below shows the process of creating a degenerate linker (illustrated with a loop). Primers can be designed by using the standard Plug’n’Play linkers.
Illustrated below the process of creating a non-degenerate linker.
Example
Synthetic biology is all about creativity, and sometimes the standard is just not enough, making it necessary to combine the above mentioned customizations. The figure below illustrates that "Plug 'n' Play with DNA" makes it possible to assemble a degenerate linker (red), a non-degenerate linker (yellow), introduce a non-degenerate point mutation (X) as well as a degenerate point mutation (N) into one plasmid in just ONE round of cloning.
Linker table
Tips & Tricks
Design primers så the Tms between forward and reverse primer are within 2-5°C.
Remember that the annealing temperature should normally be 3°C under the Tm. However, when the linker is added to the primer, the Tm is increased. We have experienced that the best results are obtain when the final annealing temperature is above 62°C, if the primer have a Tm around 59°C to begin with.
The X7 Phusion polymerase have in the PCR reaction a extension time of 1000bp per minut.
References
[1] Nørholm, M.H.H. A mutant Pfu DNA polymerase designed for advanced uracil-excision DNA engineering. BMC Biotechnol. 10, 21, 2010.
[2] Hansen, B. G.; Salomonsen, B.; Nielsen, M. T.; Nielsen, J. B.; Hansen, N. B.; Nielsen, K.F.; Regueira, T. B.; Nielsen, J.; Patil, K. R.; Mortensen, U. H.; “Versatile enzyme expression and Characterization system for Aspergillus, with the Penicillium brevicompactum Polyketide Synthase Gene from the Mycophenolic Acid Gene Cluster as a Test Case.” American Society for Microbiology, 2011, 3044-3051.
[3] Genee HJ, Bonde MT, Bagger FO, Olsen LR. PHUSER v2 (In preparation).
Retrieved from " http://2011.igem.org/Team:DTU-Denmark-2/Project/PlugnplayAssembly/customization "
Recent changes
What links here
Related changes
Special pages
My preferences
Printable version
Permanent link
Privacy policy
Disclaimers
