<HTML xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en" dir="ltr">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki  ltr ns-0 ns-subject page-Team_DTU-Denmark_Project_testing_sRNA"><DIV id="globalWrapper"><DIV id="top-section"><DIV id="p-logo"><A href="/Main_Page" title="Main Page">&quot;
	    </A></DIV><DIV id="menubar" class="left-menu noprint"><UL><LI class="selected"><A href="/Team:DTU-Denmark/Project_testing_sRNA">Page               </A></LI><LI class="new"><A href="/wiki/index.php?title=Talk:Team:DTU-Denmark/Project_testing_sRNA&amp;action=edit&amp;redlink=1">Discussion               </A></LI><LI><A href="/wiki/index.php?title=Team:DTU-Denmark/Project_testing_sRNA&amp;action=edit">View source               </A></LI><LI><A href="/wiki/index.php?title=Team:DTU-Denmark/Project_testing_sRNA&amp;action=history">History               </A></LI><LI style="color:#808080;cursor:default">teams</LI></UL></DIV><DIV class="right-menu noprint" id="menubar"><UL><LI id="pt-login"><A href="/wiki/index.php?title=Special:UserLogin&amp;returnto=Team:DTU-Denmark/Project_testing_sRNA" title="You are encouraged to log in; however, it is not mandatory [o]" accesskey="o">Log in</A></LI></UL></DIV><DIV id="search-controls" class="noprint"><FORM action="/Special:Search" id="searchform"> </FORM></DIV></DIV><DIV id="content"><H1 class="firstHeading">Team:DTU-Denmark/Project testing sRNA</H1><DIV id="bodyContent"><H3 id="siteSub" class="noprint">From 2011.igem.org</H3><DIV id="navigation"><DIV class="navigation"><DIV class="centering"><UL class="topnav" id="topnav"><LI class="home navitem1"><A href="/Team:DTU-Denmark">Home</A></LI><LI class="vision navitem2"><A href="/Team:DTU-Denmark/Vision" title="Test: A soft introduction to the idea and it's future applications.">Vision</A></LI><LI class="project navitem3"><A href="/Team:DTU-Denmark/Project">Project</A></LI><LI><A href="/Team:DTU-Denmark/Project">Overview</A></LI><LI><A href="/Team:DTU-Denmark/Bioinformatic">Bioinformatics</A></LI><LI><A href="/Team:DTU-Denmark/Project_testing_sRNA">Experiment: Testing sRNA</A></LI><LI><A href="/Team:DTU-Denmark/Project_improving_araBAD">Experiment: Improving araBAD</A></LI><LI><A href="/Team:DTU-Denmark/Modeling">Modeling</A></LI><LI><A href="/Team:DTU-Denmark/Data">Data page</A></LI><LI><A href="/Team:DTU-Denmark/Attributions">Attributions</A></LI></UL><LI class="technical_stuff navitem4"><A href="/Team:DTU-Denmark/Technical_stuff">Technical stuff</A></LI><LI><A href="/Team:DTU-Denmark/Parts">Parts</A></LI><LI><A href="/Team:DTU-Denmark/Notebook">Notebook</A></LI><LI><A href="/Team:DTU-Denmark/Protocols">Protocols</A></LI><LI><A href="/Team:DTU-Denmark/Technical_stuff_math">Math</A></LI><LI><A href="/Team:DTU-Denmark/Matlab">Matlab</A></LI><LI><A href="/Team:DTU-Denmark/Safety">Safety</A></LI><LI class="background navitem5"><A href="/Team:DTU-Denmark/Background">Background</A></LI><LI><A href="/Team:DTU-Denmark/Background_sRNA">Small RNA</A></LI><LI><A href="/Team:DTU-Denmark/Background_the_natural_system">The chitobiose system</A></LI><LI><A href="/Team:DTU-Denmark/Background_synthetic_biology">Synthetic biology</A></LI><LI class="team navitem6"><A href="/Team:DTU-Denmark/Team">Team</A></LI></DIV></DIV></DIV><DIV id="heading"><DIV class="centering"><H1>

Experiment: Testing sRNA

            </H1></DIV></DIV><DIV id="innercontent"><DIV class="centering"><DIV class="whitebox article"><TABLE id="toc" class="toc"><TBODY><TR><TD><DIV id="toctitle"><H2>Contents</H2></DIV><UL><LI class="toclevel-1 tocsection-1"><A href="#Motivation"><SPAN class="tocnumber">1</SPAN><SPAN class="toctext">Motivation</SPAN></A></LI><LI class="toclevel-1 tocsection-2"><A href="#Strain_construction"><SPAN class="tocnumber">2</SPAN><SPAN class="toctext">Strain construction</SPAN></A></LI><LI class="toclevel-1 tocsection-3"><A href="#Construction_of_plasmids"><SPAN class="tocnumber">3</SPAN><SPAN class="toctext">Construction of plasmids</SPAN></A></LI><LI class="toclevel-2 tocsection-4"><A href="#Verifying_constructs"><SPAN class="tocnumber">3.1</SPAN><SPAN class="toctext">Verifying constructs</SPAN></A></LI><LI class="toclevel-2 tocsection-5"><A href="#Results_and_Conclusions"><SPAN class="tocnumber">3.2</SPAN><SPAN class="toctext">Results and Conclusions</SPAN></A></LI></UL><LI class="toclevel-1 tocsection-6"><A href="#References"><SPAN class="tocnumber">4</SPAN><SPAN class="toctext">References</SPAN></A></LI></TD></TR></TBODY></TABLE><H2><SPAN class="mw-headline" id="Motivation"> Motivation </SPAN></H2><P>The biological experiments of the testing sRNA part is a proof of concept, showing that the <I>E. coli</I> sRNA <I>sroB</I>  and the intergenic sRNA (trap-RNA) in the <I>chbBCARG</I> gene can be used to control gene expression by targeting the <I>ybfM</I> Shine-Dalgarno, and that these sRNAs can be rationally designed to target other Shine-Dalgarno sequences. The purpose is two-fold to construct plasmids and strains with deletions.
</P><P><B>Strain construction</B> involves deleting the original genes from the chromosome of a <I>E. coli</I>  W3110 strain. Since we use the original genes, these need to be deleted from the chromosome to prevent them from interfering with our measurements.
</P><P><B>Construction of plasmids</B> necessary for testing our system involves taking the native system from <I>E. coli</I>, as well as a slightly modified system and putting them into plasmids that let us both control the expression of these components and measure the output of the system.
</P><H2><SPAN class="mw-headline" id="Strain_construction"> Strain construction </SPAN></H2><P>The strains used in testing our system were: <B>IG9</B> a <I>E. coli</I> based on the wild-type K-12<SPAN class="superscript"><A href="#References">[3]</A></SPAN> strain W3110 with the <I>lacZYA</I>, <I>chbBCARG</I> operons deleted, and <B>IG302</B> which is the K-12 <I>$\Delta$lacZYA</I>, <I>$\Delta$chiP</I> (alias <I>ybfM</I>), <I>$\Delta$chiX</I> (alias <I>sroB</I>, <I>micM</I>) strain constructed by Rasmussen et al.<SPAN class="superscript"><A href="#References">[6]</A></SPAN>. 
</P><P><I>chbBCARG</I>, <I>chiP</I> and <I>chiX</I> code for utilization of chitobiose (a sugar), chitobiose permease and the regulation of the chitobiose permease respectively.
</P><P>We used recombineering<SPAN class="superscript"><A href="#References">[1]</A></SPAN> to delete these genes. Schematically the procedure works in the following manner:
</P><OL><LI>Transform competent cells with Red and Cre helper plasmids.
</LI><LI>Make the cells competent and induce the Red recombination system.
</LI><LI>Transform with linear piece of DNA.
</LI><LI>Select for gain of resistance.
</LI><LI>Do colony PCR to check that resistance gene is inserted in the correct place.
</LI><LI>Move to medium inducing the Cre recombinase.
</LI><LI>Screen for loss of resistance.
</LI><LI>Verify the construct by colony PCR.
</LI><LI>Repeat step 2-7 as needed.
</LI><LI>Cure the helper plasmids.
</LI></OL><P>The Red helper plasmids encodes the $\lambda{}$ phage Red recombination system. This system allows recombination events between sequences with around 40 nucleotides of homology.
The cre helper plasmid encodes the Cre recombinase. This recombinase recognises 34bp <I>loxP</I> sequences and promotes recombination between them.
</P><P>An example of a linear piece of DNA that could be used with this method, as well as the simplified representation of the procedure, can be seen in figure:
</P><DIV class="thumb tright"><DIV class="thumbinner" style="width:402px;"><DIV class="thumbcaption">A schematic overview of the Recombineering procedure. A linear piece of DNA, carrying a resistance marker is transformed into a strain carrying two helper plasmids, the $\lambda{}$ phage recombination system mediates integration of the linear piece of DNA. The Cre recombinase mediates the excision of the resistance marker.</DIV></DIV></DIV><DIV class="thumb tright"><DIV class="thumbinner" style="width:402px;"><DIV class="thumbcaption">The Cre recombinase recognizes the lox-sites. The mutated sites lox-71 and lox-66 are recognized, but the resulting lox-site is not.</DIV></DIV></DIV><P>In case of two or more subsequent gene deletions using Red-Cre recombineering a problem of recombination between an old <I>loxP</I> site and newly introduced ones arises. Remember, that after each round of redswap one <I>loxP</I> site is left in the genome. As a result recombination using Cre recombinase in second or next redswap might not occur between loxP sites flanking a resistance gene. To circumvent this problem two mutated loxP sequences are used - <I>lox66</I> and <I>lox71</I><SPAN class="superscript"><A href="#References">[4]</A></SPAN>. In spite of the mutations these lox sites are recognized by Cre which can remove the intervening region. Finally, from <I>lox66</I> and <I>lox71</I> a new site is formed, <I>lox72</I>, which has a reduced affinity to Cre recombinase. Thus, more than one gene deletions can be subsequently conducted.
</P><H2><SPAN class="mw-headline" id="Construction_of_plasmids"> Construction of plasmids </SPAN></H2><P>In order to quantitatively describe the natural trap-RNA system the following four plasmids were constructed:
</P><UL><LI>P<SPAN class="subscript">chiP</SPAN>-lacZ plasmid
</LI></UL><DL><DD> To mimic and test repression of the chiP gene its upstream region of 599 bp was fused to reporter genes lacZ and inserted on the pSB1C3 plasmid. Expression from P<SPAN class="subscript">chiP</SPAN> promoter is costitutive however it can be induced to a higher level by chitobiose. LacZ codes for $\beta$-galactosidase and its activity was tested using $\beta$-Gal assays.
</DD></DL><UL><LI>P<SPAN class="subscript">chiP</SPAN>(mutated)-lacZ plasmid
</LI></UL><DL><DD> This plasmid is similar to the P<SUB>chiP</SUB>-lacZ plasmid, except the Shine-Dalgarno has been changed. This is expected to abolish or limit the regulatory effect of chiX.
</DD></DL><UL><LI>P<SPAN class="subscript">tet</SPAN>-chiX plasmid
</LI></UL><DIV class="thumb tright"><DIV class="thumbinner" style="width:302px;"><DIV class="thumbcaption">A schematic representation of the constructed plasmids, and the backbones we placed them in.</DIV></DIV></DIV><DL><DD> ChiX which is 84 bp long and regulates the wild-type chiP Shine-Dalgarno<SPAN class="superscript"><A href="#References">[2]</A></SPAN><SPAN class="superscript"><A href="#References">[5]</A></SPAN>, was inserted into pSB4K under the control of the inducible promoter P<SUB>tet</SUB>. Expression from P<SPAN class="subscript">tet</SPAN> promoter was induced using anhydrotetracycline.
</DD></DL><UL><LI>P<SPAN class="subscript">BAD</SPAN>-chiXR plasmid
</LI></UL><DL><DD>A 284 bp long introgenic region from chbBCARG operon known to regulate chiX<SPAN class="superscript"><A href="#References">[2]</A></SPAN><SPAN class="superscript"><A href="#References">[5]</A></SPAN> was inserted into pSB3T5 plasmid. Expression from that plasmid was controlled by regulatory element from <I>ara</I> operon with arabinose as inducer.
</DD></DL><P>Details on how the plasmids were constructed, including information on primers used, can be found in <A href="https://2011.igem.org/Team:DTU-Denmark/Notebook" class="external text" rel="nofollow">lab notebook</A>. 
</P><H4><SPAN class="mw-headline" id="Verifying_constructs"> Verifying constructs </SPAN></H4><P>The plasmids we constructed were verified in two ways. The P<SUB>BAD</SUB>-chiXR fusion and P<SUB>chiP</SUB>-lacZ fusion were tested by restriction analysis of the plasmid. And the P<SUB>tet</SUB>-chiX fusion was tested by PCR.
</P><P>The restriction of three plasmids tested revealed fragments of the expected sizes. These can be seen in the table below. The gel picture is also available.
</P><TABLE class="wikitable"><CAPTION>P<SUB>BAD</SUB>-chiXR
</CAPTION><TBODY><TR><TH> Restriction Enzymes
</TH><TH> Expected sizes
</TH></TR><TR><TD> EcoRI + BamHI
</TD><TD> 1150 + 2550
</TD></TR><TR><TD> EcoRI + NcoI
</TD><TD> 2930 + 770
</TD></TR></TBODY></TABLE><DIV class="center"><DIV class="thumb tnone"><DIV class="thumbinner" style="width:667px;"><DIV class="thumbcaption">A linearized version of the P<SUB>BAD</SUB>-chiXR construct inside the pSB1C3 plasmid. Restriction sites used, as well as expected fragment sizes are indicated.</DIV></DIV></DIV></DIV><DIV class="thumb tleft"><DIV class="thumbinner" style="width:372px;"><DIV class="thumbcaption">The gel picture of the P<SUB>BAD</SUB>-chiXR construct. The restriction enzymes used, and the sizes are indicated on the picture. EN = EcoRI + NcoI, EB = EcoRI + BamHI</DIV></DIV></DIV><DIV class="thumb tright"><DIV class="thumbinner" style="width:242px;"><DIV class="thumbcaption">The ladder used as supplied by [www.fermentas.com/templates/files/tiny_mce/coa_pdf/coa_sm0331.pdf Fermentas]</DIV></DIV></DIV><P>The PCR to test the P<SUB>tet</SUB>-chiX fusion was done on a number of colonies following a transformation. The primers used were CS149 (GGTTTGCCACCTGACGTCTAAGAA) and CS150 (CCAAATTACCGCCTTTGAGTG). These two primers anneal just outside the BioBrick and can be used to PCR any BioBrick in a BioBrick compatible plasmid. The expected size of the PCR product was 1358 bp. Two of the colonies tested contained bands of the correct size, while the rest were background. These two were purified and subsequently used.
</P><DIV class="thumb tleft"><DIV class="thumbinner" style="width:402px;"><DIV class="thumbcaption">Colony PCR of cells transformed with P<SUB>tet</SUB>-chiX inside pSB1C3, CS149 and CS150 primers were used. Expected band size is 1391 bp</DIV></DIV></DIV><P>A number of P<SUB>chiP</SUB>-lacZ fusions were verified using restriction analysis. The expected band sizes are shown below, as is a gel-picture of the restriction:
</P><TABLE class="wikitable"><CAPTION> P<SUB>chiP</SUB>-lacZ
</CAPTION><TBODY><TR><TH> Restriction Enzymes
</TH><TH> Expected sizes
</TH></TR><TR><TD> EcoRI + EcoRV + PstI
</TD><TD> 1743 + 2101 + 1136 + 898
</TD></TR></TBODY></TABLE><DIV class="center"><DIV class="thumb tnone"><DIV class="thumbinner" style="width:667px;"><DIV class="thumbcaption">A linearized version of the P<SUB>chiP</SUB>-lacZ construct inside the pSB1C3 plasmid. Restriction sites used, as well as expected fragment sizes are indicated.</DIV></DIV></DIV></DIV><DIV class="thumb tright"><DIV class="thumbinner" style="width:302px;"><DIV class="thumbcaption">A linearized version of the P<SUB>chiP</SUB>-lacZ construct inside the pSB1C3 plasmid. Restriction sites used, as well as expected fragment sizes are indicated.</DIV></DIV></DIV><H4><SPAN class="mw-headline" id="Results_and_Conclusions"> Results and Conclusions </SPAN></H4><P>Although we believe we obtained the correct plasmids, and they were subcloned into plasmids that let us test them, we were unable to do so due to time pressure. Only a few combinations with the P<SUB>chiP</SUB> plasmids were performed. These are summarized in the table below:
</P><TABLE class="wikitable"><TBODY><TR><TH> Strain
</TH><TH><A href="http://ecoliwiki.net/colipedia/index.php/NM522" class="external text" rel="nofollow">NM522</A></TH><TH><A href="https://2011.igem.org/Team:DTU-Denmark/Project_testing_sRNA#Strain_construction" class="external text" rel="nofollow">IG302</A></TH></TR><TR><TD> Genotype
</TD><TD> chiX +
</TD><TD> chiX -
</TD></TR><TR><TD><A href="http://partsregistry.org/wiki/index.php?title=Part:BBa_K564012" class="external text" rel="nofollow"><B>PchiP(original):lacZ, BBa_K564012</B></A></TD><TD> 467
</TD><TD> 4781
</TD></TR><TR><TD><A href="http://partsregistry.org/wiki/index.php?title=Part:BBa_K564013" class="external text" rel="nofollow"><B>PchiP(mutated):lacZ, BBa_K564013</B></A></TD><TD> 2353
</TD><TD> 2403
</TD></TR></TBODY></TABLE><P>We concluded that the P<SUB>chiP</SUB> is regulated by chiX, and that mutating the Shine-Dalgarno (SD) removes the regulatory effect of chiX on the P<SUB>chiP</SUB> SD. The lower baseline expression of the mutated promoter is most likely due to the slightly weaker SD.
</P></DIV><DIV class="whitebox article"><H2><SPAN class="mw-headline" id="References">References</SPAN></H2><P>[1] Datta, Simanti, Nina Costantino, and Donald L Court. “A set of recombineering plasmids for gram-negative bacteria.” Gene 379 (September 1, 2006): 109-15. <A href="http://www.ncbi.nlm.nih.gov/pubmed/16750601" class="external free" rel="nofollow">http://www.ncbi.nlm.nih.gov/pubmed/16750601</A>.
</P><P>[2] Figueroa-Bossi, Nara, Martina Valentini, Laurette Malleret, and Lionello Bossi. “Caught at its own game: regulatory small RNA inactivated by an inducible transcript mimicking its target.” Genes &amp; Development 23, no. 17 (2009): 2004 -2015. 
</P><P>[3] Hayashi, Koji, Naoki Morooka, Yoshihiro Yamamoto, Katsutoshi Fujita, Katsumi Isono, Sunju Choi, Eiichi Ohtsubo, et al. “Highly accurate genome sequences of Escherichia coli K-12 strains MG1655 and W3110.” Molecular Systems Biology 2 (2006): 2006.0007. <A href="http://www.ncbi.nlm.nih.gov/pubmed/16738553" class="external free" rel="nofollow">http://www.ncbi.nlm.nih.gov/pubmed/16738553</A>.
</P><P>[4] Lambert, Jolanda M, Roger S Bongers, and Michiel Kleerebezem. “Cre-lox-based system for multiple gene deletions and selectable-marker removal in Lactobacillus plantarum.” Applied and environmental microbiology 73, no. 4 (February 2007): 1126-35. <A href="http://www.pubmedcentral.nih.gov/articlerender.fcgi?artid=1828656&amp;tool=pmcentrez&amp;rendertype=abstract" class="external free" rel="nofollow">http://www.pubmedcentral.nih.gov/articlerender.fcgi?artid=1828656&amp;tool=pmcentrez&amp;rendertype=abstract</A>.
</P><P>[5] Overgaard, Martin, Jesper Johansen, Jakob Møller‐Jensen, and Poul Valentin‐Hansen. “Switching off small RNA regulation with trap‐mRNA.” Molecular Microbiology 73, no. 5 (September 2009): 790-800. 
</P><P>[6] Rasmussen, Anders Aamann, Jesper Johansen, Jesper S Nielsen, Martin Overgaard, Birgitte Kallipolitis, and Poul Valentin‐Hansen. “A conserved small RNA promotes silencing of the outer membrane protein YbfM.” Molecular Microbiology 72, no. 3 (May 2009): 566-577.
</P></DIV></DIV></DIV><DIV id="footer-panel"><DIV class="centering"><DIV class="footerBox"><H4>This project is part of iGEM</H4><P>Read more on the <A href="https://2011.igem.org/Main_Page">iGEM Main Page</A></P></DIV><DIV class="footerBox"><H4>Sponsors</H4><P>Thanks to:</P></DIV><DIV class="footerBox"><H4>How to customize an iGEM wiki?</H4><P>Learn all the tricks in our <A href="/Team:DTU-Denmark/How_to_customize_an_iGEM_wiki">wiki guide</A></P></DIV></DIV></DIV><DIV class="printfooter">
Retrieved from &quot;<A href="http://2011.igem.org/Team:DTU-Denmark/Project_testing_sRNA">http://2011.igem.org/Team:DTU-Denmark/Project_testing_sRNA</A>&quot;</DIV></DIV></DIV><DIV id="footer-box" class="noprint"><DIV id="footer"><UL id="f-list"><LI id="t-recentchanges"><A href="/Special:RecentChanges" title="Recent changes">Recent changes</A></LI><LI id="t-whatlinkshere"><A href="/Special:WhatLinksHere/Team:DTU-Denmark/Project_testing_sRNA" title="List of all wiki pages that link here [j]" accesskey="j">What links here</A></LI><LI id="t-recentchangeslinked"><A href="/Special:RecentChangesLinked/Team:DTU-Denmark/Project_testing_sRNA" title="Recent changes in pages linked from this page [k]" accesskey="k">Related changes</A></LI><LI id="t-specialpages"><A href="/Special:SpecialPages" title="List of all special pages [q]" accesskey="q">Special pages</A></LI><LI><A href="/Special:Preferences">My preferences</A></LI></UL></DIV><DIV id="footer"><UL id="f-list"><LI id="t-print"><A href="/wiki/index.php?title=Team:DTU-Denmark/Project_testing_sRNA&amp;printable=yes" title="Printable version of this page [p]" accesskey="p">Printable version</A></LI><LI id="t-permalink"><A href="/wiki/index.php?title=Team:DTU-Denmark/Project_testing_sRNA&amp;oldid=169234" title="Permanent link to this revision of the page">Permanent link</A></LI><LI id="privacy"><A href="/2011.igem.org:Privacy_policy" title="2011.igem.org:Privacy policy">Privacy policy</A></LI><LI id="disclaimer"><A href="/2011.igem.org:General_disclaimer" title="2011.igem.org:General disclaimer">Disclaimers</A></LI></UL></DIV></DIV></DIV></BODY></HTML>