"
Page
Discussion
View source
History
teams
Log in
 
Team:DTU-Denmark-2/Team/Protocols
From 2011.igem.org
Home
About us
The Team
Supervisors
Gallery
The Project
Abstract
Introduction
The Plug 'n' Play
Assembly system
Customization
Other assembly systems
Achievements
Attributions
Results
Background
Characterization
Proof of concept
Mammalian Cells
Fungi
Data page
Submitted Biobricks
Collaboration
Safety
Safety
Safety rules
Notebook
Lab notes
Protocols
Our Partners
Sponsors
Advisors
Acknowledgements
Protocols
1 Amplification of biobricks by PCR
1.1 List of primers for fungi and for mammalian cells
1.2 PCR
1.3 Purification of PCR Product
1.4 Gel electrophoresis
2 USER cloning
2.1 Tranformation in E. coli
2.2 Cultivation of transformed cells
2.3 Purification of plasmids for fungi
2.4 Purification of plasmides for mammalian cells
2.5 Restriction enzyme analysis
3 Fungi
3.1 Transformation in fungi
3.2 Production of conidiospores
3.3 Flourescence Microscopy
3.4 Fluorescence detection
3.5 Extraction of proteins
4 Assays
4.1 ß-galactosidase assay
4.2 Bradford assay
5 Mammalian cells
5.1 Cell culture and reagents
5.2 Cultivation of cells
5.3 Passing and maintenance of U2OS cells
5.4 Transferring cells to coverslips
5.5 Transfection of cells
5.6 Coverslips for microscopy
Amplification of biobricks by PCR
List of primers for fungi and for mammalian cells
Fungal primers
Primer name
Sequence
pgpd FW
ACGTCGCUATTCCCTTGTATCTCTACACACAGG
pgpd RV
ATCGCACUGCGGTAGTGATGTCTGCTCAA
pAlc FW
ACGTCGCUCTCCCCGATGACATACAGGAGG
pAlc RV
ATCGCACUTTTGAGGCGAGGTGATAGGATTG
DMKP-P6 FW
ACGTCGCUATCTCACTCCACTAGAATTCCTGTC
DMKP-P6 RV
ATCGCACUAAATGAGTCGAGAATGGCGG
GFP-module FW
AGTGCGAUATGGTGAGCAAGGGCGAG
GFP-module RV
ATCGGAAUTTACTTGTACAGCTCGTCCATGC
GFP-GOI FW
AGTGCGAUATGGTGAGCAAGGGCGAG
GFP-GOI RV
ATCGCTCUTTACTTGTACAGCTCGTCCATGC
GFP-TS FW
AGCGCTGGUATGGTGAGCAAGGGCGAG
GFP-TS RV
ATCGGAAUTTACTTGTACAGCTCGTCCATGC
GFP-PTS1_module RV
ATCGGAAU TTACAGCTTGGA CCTTGTACAGCTCGTCCATGCC
RFP-module FW
AGTGCGAUATGGCCTCCTCCGAGGAC
RFP-module RV
ATCGGAAUTTAGGCGCCGGTGGAGTG
RFP-GOI FW
AGTGCGAUATGGCCTCCTCCGAGGAC
RFP-GOI RV
ATCGCTCUTTAGGCGCCGGTGGAGTG
RFP-TS FW
AGCGCTGGUATGGCCTCCTCCGAGGAC
RFP-TS RV
ATCGGAAUTTACTTGTACAGCTCGTCCATGC
RFP-NLS_module RV
ATCGGAAU TTAGACCTTGCGCTTTTTCTTGGG GGCGCCGGTGGAGTGG
LacZ FW
AGTGCGAUATGACCATGATTACGGATTCACT
LacZ RV
ATCGGAAUTTATTTTTGACACCAGACCAACT
trpC FW
ATTCCGAUGATCCACTTAACGTTACTGAAATCA
trpC RV
ACGCAAGUGGGCGCTTACACAGTACACGAG
PyrG FW
ACTTGCGUCGTGGAGTTACCAGTGATTGACC
PyrG RV
AGCTTAAUCTTGCTAGATGACTGGTAGGAATCT
bleR FW
ACTTGCGUATGGCCAAGTTGACCAGTG
bleR RV
AGCTTAAUTCAGTCCTGCTCCTCGGCC
hyrR FW
ACTTGCGU GCTAGTGGAGGTCAACACATCAAT
hyrR RV
AGCTTAAUCGGTCGGCATCTACTCTATTCC
argB FW
ACTTGCGUCGCGGTTTTTTGGGGTAGT
argB RV
AGCTTAAUGCCACCTACAGCCATTGCGAA
PyrG-DR FW
ACTTGCGUTGGATAACCGTATTACCGCCT
PyrG-DR RV
AGCTTAAUTGCCAAGCTTAACGCGTACC
ptrA FW
ACTTGCGUGGAGATCGTCCGCCGATG
ptrA RV
AGCTTAAUCTAGAATGCCCCACCGTTACATAC
Amp-cas FW
AGCTTAAUTTACCAATGCTTAATCAGTGAGGC
Amp-cas RV
ACTTGCGUGACGTCAGGTGGCACTTTTCG
yA FW
AGTGCGAUATGTACCTCTCCACGGTCCTCT
yA RV
ATCGCTCUCTAAGAATCCCAAACATCAACCC
MTS FW
AGTGCGAUATGTTTACAGCGGCAGCTCG
MTS RV
ACCAGCGCUCTTGCGCCGCGGAGC
T1-motni FW
ATTCCGAUGGCTCCGAGGCTACTGGAGT
T1-motni FW
ACGCAAGUGCGACGTCTGATGCCAATAT
T2-motni FW
ATTCCGAUGCGGTTTTCAAGGATAACAGAT
T2-motni RV
ACGCAAGUCATGACCCCGGATAACTTTAAAA
T3 motni FW
ATTCCGAUAAGTCTTCCGTTACCCTTGCA
T3-motni RV
ACGCAAGUTGTAATTCCTACCTACCTACCTCTT
Mammalian primers
Primer name
Sequence
PGK FW
ACGTCGCUCCGGTAGGCGCCAACCG
PGK RV
ATCGCACUGGCTGCAGGTCGAAAGGCC
SV40 (+ori) FW
ACGTCGCUCTGTGGAATGTGTGTCAGTTAGG
SV40 (+ori) RV
ATCGCACUAGCTTTTTGCAAAAGCCTAGG
CMV FW
ACGTCGCUCGATGTACGGGCCAGATATAC
CMV-RV
ATCGCACUATTTCGATAAGCCAGTAAGCAGT
eGFP+K_GOI FW
AGTGCGAUCGCCACCATGGTGAGCAA
eGFP+K_GOI RV
ATCGCTCUTTACTTGTACAGCTCGTCCATGC
Mammalian backbone pU0020 FW
ATTAAGCUAGTGAGTCGAATAAGGGCGACA
Mammalian backbone pU0020 RV
AGCGACGUGAGTCGAATAAGGGCGACACC
eGFP_module FW
AGTGCGAUCGCCACCATGGTGAGCAA
eGFP_module RV
ATCGGAAUTTACTTGTACAGCTCGTCCATGCC
eGFP+K GOI FW
AGTGCGAUCGCCACCATGGTGAGCAA
eGFP+K GOI-stop RV
ACCAGCGCUCTTGTACAGCTCGTCCATGCC
eGFP_TS FW
AGAGCGAUCGCCACCATGGTGAGCAA
eGFP_TS RV
ATCGGAAUTTACTTGTACAGCTCGTCCATGCC
YFP_module FW
AGTGCGAUCGCCACCATGGTGAGCAA
YFP_module RV
ATCGGAAUTTATCTAGATCCGGTGGATCCC
YFP+K GOI FW
AGTGCGAUCGCCACCATGGTGAGCAA
YFP+K GOI RV
ACCAGCGCU TCTAGATCCGGTGGATCCCG
YFP_TS FW
AGCGCTGGUCGCCACCATGGTGAGCAA
YFP_TS RV
ATCGGAAUTTATCTAGATCCGGTGGATCCC
CFP_module FW
AGTGCGAUCGCCACCATGGTGAGCAA
CFP_module RV
ATCGGAAUTTATCTAGATCCGGTGGATCCC
CFP+K GOI FW
AGTGCGAUCGCCACCATGGTGAGCAA
CFP+K GOI-stop RV
ACCAGCGCUTCTAGATCCGGTGGATCCCG
CFP_TS FW
AGCGCTGGUCGCCACCATGGTGAGCAA
CFP_TS RV
ATCGGAAUTTATCTAGATCCGGTGGATCCC
mCherry module FW
AGTGCGAUCGCCACCATGGTGAGCAA
mCherry module RW
ATCGGAAUCTACTTGTACAGCTCGTCCATGC
mCherry+K GOI FW
AGTGCGAUCGCCACCATGGTGAGCAA
mCherry GOI-stop RV
ACCAGCGCUCTTGTACAGCTCGTCCATGCC
mCherry_TS FW
AGCGCTGGUCGCCACCATGGTGAGCAA
mCherry_TS RV
ATCGGAAUCTACTTGTACAGCTCGTCCATGC
BGH poly A FW
ATTCCGAUCTGTGCCTTCTAGTTGCCAGC
BGA poly A RV
ACGCAAGUCCATAGAGCCCACCGCATC
SV40 pA FW
ATTCCGAUAACTTGTTTATTGCAGCTTATAATGGTTAC
SV40 pA RV
ACGCAAGUCAGACATGATAAGATACATTGATGAGTTTG
Hygromycin FW
ACTTGCGUCCAGCAGGCAGAAGTATGCA
Hygromycin RV
AGCTTAAUCAGGCTTTACACTTTATGCTTCC
Neomycin FW
ACTTGCGUCTGTGGAATGTGTGTCAGTTAGG
Neomycin RV
AGCTTAAUCAGACATGATAAGATACATTGATGAGTTTG
Copenhagen primers
Primer name
Sequence
pIPTG FW
ACGTCGCUCAATACGCAAACCGCCTCTC
pIPTG RV
ATCGCACUTGTGTGAAATTGTTATCCGCTCA
CYP79A1 FW
AGTGCGAUaaagaggagaaaATGGCTCTGTTATTAGCAGTTTTT
CYP79A1 RV
ATCGGAAUTTAGATGGAGATGGACGGGTA
CYP79B1 FW
AGTGCGAUaaagaggagaaaATGTATTTACTTACAACGCTTCAAG
CYP79B1 RV
ATCGGAAUTACTTCACTGTAGGGTAAAGATGT
2C9 FW
AGTGCGAUaaagaggagaaaATGGCTCGACAATCTTCTGGA
2C9 RV
ATCGGAAUTTAATGGTGATGGTGATGGACAG
Terminator FW
ATTCCGAUCCAGGCATCAAATAAAACGAAA
Terminator RV
ACGCAAGUTATAAACGCAGAAAGGCCCAC
Amp-cas FW
AGCTTAAUTTACCAATGCTTAATCAGTGAGGC
Amp-cas RV
ACTTGCGUGACGTCAGGTGGCACTTTTCG
PCR
PCR mix
1 x PCR mix á 50 µl
5 x HF PCR buffer
10 µl
dNTP’s 2mM
5 µl
Primer forward 10 µM
5 µl
Primer reverse 10 µM
5 µl
Phusion DNA polymerase 5u/µl
0.5 µl
DNA template
1 µl
MilliQ water
23.5 µl
Procedure
The following PCR mix components are mixed together: HF Buffer, dNTP, Phusion DNA polymerase, and MilliQ water. The DNA template and the corresponding primers are added subsequently (see Materials).  Remember to multiply by the amount of PCR reactions.
50 µl PCR mix is added to each tube, and afterwards the DNA template and the corresponding primers are added to each PCR tubes. The solution is mixed well.
Often it can be advantageous to add 1 µl MgCl2 (50mM) to the reaction and reduce the volume of MilliQ water to 22.5 µl.
The following PCR program is applied, and the PCR samples are run.
PCR Programs
Temperature [°C]
Time [min]
Cycles
98
15:00
98
0:20
35
62
0:30
-
72
1:00
-
72
5:00
12
Store
For each PCR product a gel electrophoresis is made to make sure that the PCR product has the correct size.  7-10µl loading buffer is added to each 50µl PCR product and the samples are run in 2% agarose. The Gel electrophoresis were set on 75V for 30-60 minutes (according to the length of the product).
Purification of PCR Product
Materials
GFX PCR DNA and Gel band purification Kit from GE healthcare is used to purify the PCR product after gel electrophoresis.
Preparations
Ethanol is added to the Wash Buffer 1 and the label is marked. Store in airtight container.
Procedures - GE healthcare protocol
Sample Capture
A DNase-free 1,5 ml eppendorf tube is weighed, and the weight is noted.
By using long wavelength (365 nm) ultraviolet light and minimal exposure time, the sample in the gel is visualized. A clean scalpel is used to cut out an the sample from the agarose gel. The agarose gel band is placed into a DNase-free 1,5 ml eppendorf tube and weighed. The weight of the agarose band is calculated.
10 μl Capture buffer type 3 is added for each 10 mg of gel slice. Never use less than 300 μl Capture buffer type 3.
The Capture buffer type 3-sample mix is mixed by inversion and incubated at 60°C for 15–30 minutes until the agarose is completely dissolved. The sample is mixed by inversion every 3 minutes
For each purification a GFX MicroSpin column is placed into a Collection tube.
Sample Binding
The Capture buffer type 3-sample mix is centrifuged briefly to collect the liquid at the bottom of the DNase-free 1,5 ml microcentrifuge tube.
Up to 800 μl Capture buffer type 3- sample mix is transferred to the assembled GFX MicroSpin column.
The sample is incubated at room temperature for 1 minute.
The sample is centrifuged for 30 s at 16 000 × g. The flow-through is discarded.
Repeat the sample binding until all sample is loaded.
Wash and Dry
500 μl Wash buffer type 1 is added to the GFX MicroSpin column.
The sample is centrifuged at 16 000 × g for 30 seconds.
Discard the Collection tube and transfer the GFX MicroSpin column to a new DNase-free 1.5 ml eppendorf tube (supplied by user).
Elution
10–50 μl MilliQ water is added to the center of the membrane in the GFX MicroSpin column placed in a Collection tube. The sample is incubated at room temperature for 1 minute.
The GFX MicroSpin column placed in the Collection tube is centrifuged for 1 minute at 16 000 × g to recover the purified DNA.
The purified DNA is stored at -20°C.
Gel electrophoresis
Materials
Agarose
TAE Buffer (1L): 4.84 g Tris Base, 1.14 ml Glacial Acetic Acid, 2 ml 0.5M EDTA (pH 8.0), add water until volume is 1L
10X Loading Buffer
DNA ladder standard
Electrophoresis chamber
Gel casting tray and combs
Procedures
1.25 g Agarose powder is measured and added to a 500 ml flask
125 ml TAE Buffer is added to the flask.
The agarose is melted in a microwave or hot water bath until the solution becomes clear.
The solution is cooled down to 50-55°C. A trick is to swirl the flask lightly to cool evenly.
20 µl of cybrsafe is added in 4-5 drops on the casting tray. The agarose gel is added and mixed evenly. The gel is left to cure, which takes 30-45 min.
The gel is placed in a electrophoresis chamber. Make sure the TAE buffer in the chamber is 2-3mm above the agarose gel.
The gel is loaded with the samples, a negative sample, and the DNA ladder.
The gel is run at 70V. The time depends on the bp length, but 30-60 min are usually good for 1000-3000bp.
The gel is visualized in a -GEL EXPOSER!-.
USER cloning
Materials
Procedures
The USER mix components are mixed (Table in materials). The PCR product must be purified before used in USER cloning
2 µl of the USER mix is transferred to PCR tubes.
The PCR products is added in equal amounts of each to a total volume of 8 µl and incubated for 40 minutes at 37°C and for 30 min at 25°C.
USER mix
1 x USER mix á 10µl
USER enzyme
1 µl
NEB (10 x diluted)
0,5 µl
BSA
0,5 µl
PCR product(s)
8 µl
Tranformation in E. coli
The preparations for the transformation can preferably be done, while the USER cloning is incubating.
Materials
LB-plates with antibiotic resistance.
E. coli DHα5 cells"
Grigalsky spartula
Ethanol
USER reaction Bunsen burner
Procedures
LB- plates are taken out of the refrigerator and marked. Remember to use LB-plates with the right antibiotics.
50 µl competent E. coli DHα5 cells per USER reaction is taken from the -80C freezer and place on ice. Additionally, 1,5 ml tubes are placed on ice.
The USER reaction mix is added to the 50 µl competent E. coli cells. Mix well by pipetting.
The cells are kept on ice for 30 min.
The hot plate is set on 60C, and each transformation is heat shocked for 90 sec. The cells are put directly on ice for 2 min afterwards
Plating on LB plates.
With ampicillin resistence gene.
A Drigalski spartula is sterilized in 90% ethanol and flamed between each transformation.
The transformation mix is transferred to an LB-amp plate containing ampicillin and dispersed with the cooled drigalski.
The transformated cells are incubated over night at 37°C.
Next day; the plates are checked for visual colonies. These are used for cultivating.
Without ampicillin resistence gene
500 µl LB is added to the transformation mix and incubated for 30-60 min at RT.
The transformation mix is centrifuged for 1 min at 8000g.
The supernantant is removed except for approximately 50 µl.
The pellet is resuspended in the remaining LB.
The 50 µl of tranformatin mix is plated on the LB plate.
The transformated cells are incubated over night at 37°C.
Next day; the plates are checked for visual colonies. These are used for cultivating.
Cultivation of transformed cells
Material
Liquid LB with resistance marker
tooth pick
Procedure
A couple of colonies are chosen from the LB-plate.
Each colony are transferred to 5 ml LB + resistance marker with a pipet tip.
Incubate over night at 37°C in the shaking incubator.
Purification of plasmids for fungi
Materials
The GenElute Plasmid Miniprep Kit from Sigma-Aldrich is used to isolate our plasmids to use in fungi. QIAGENs EndoFree Plasmid Maxi kit is used to purify plasmids for use in mammalian cells.
Preparation
The agents are thoroughly mixed. If any reagent forms a precipitate, warm at 55–65 °C until the precipitate dissolves. Cool to room temperature before use.
The solution is resuspended. Spin the tube of the RNase A Solution briefly to collect the solution in the bottom of the tube. Add 13 μl (for 10 prep package), 78 μl (for 70 prep package) or 500 μl (for 350 prep package) of the RNase A Solution to the Resuspension Solution prior to initial use. Store at 4 °C.
Wash solution; Dilute with 95-100% ethanol prior to initial use.
Procedure - according to the manufactures protocol.
Harvest cells by centrifugation of 5 ml of an overnight recombinant E. coli culture. The optimal volume of culture to use depends upon the plasmid and culture density. For best yields, follow the instructions in the note below. Transfer the appropriate volume of the recombinant E. coli culture to a microcentrifuge tube and pellet cells at ≥12,000 3 g for 1 minute. Discard the supernatant.
Completely resuspend the bacterial pellet with 200 μl of the Resuspension Solution. Vortex or pipette up and down to thoroughly resuspend the cells until homogeneous. Incomplete resuspension will result in poor recovery.
Lyse the resuspended cells by adding 200 μl of the Lysis Solution. Immediately mix the contents by gentle inversion (6–8 times) until the mixture becomes clear and viscous. The lyse reaction must not exceed 5 min.
Precipitate the cell debris by adding 350 μl of the Neutralization/Binding Solution. Gently invert the tube 4–6 times. Pellet the cell debris by centrifuging at ≥12,000*3 g or maximum speed for 10 minutes. Cell debris, proteins, lipids, SDS, and chromosomal DNA should fall out of solution as a cloudy, viscous precipitate.
Prepare columns
Insert a GenElute Miniprep Binding Column into a provided microcentrifuge tube, if not already assembled. Add 500 μl of the Column Preparation Solution to each miniprep column and centrifuge at ≥12,000*3 g for 30 seconds to 1 minute. Discard the flow-through liquid.
Transfer the cleared lysate from step 3 to the column prepared in step 4 and centrifuge at ≥12,000*3 g for 30 seconds to 1 minute. Discard the flow-through liquid.
Add 750 μl of the diluted Wash Solution to the column. Centrifuge at ≥12,000*3 g for 30 seconds to 1 minute. The column wash step removes residual salt and other contaminants introduced during the column load. Discard the flow-through liquid and centrifuge again at maximum speed for 1 to 2 minutes without any additional Wash Solution to remove excess ethanol.
Transfer the column to a fresh collection tube. Add 100 μl of MilliQ water to the column. Centrifuge at ≥12,000*3 g for 1 minute.
The DNA is now present in the eluate and is ready for immediate use or storage at –20 °C.
Purification of plasmids for mammalian cells
Materials
EndoFree Plasmid Maxi kit from QIAGEN was used to purify plasmids for use in mammalian cells.
Procedure (according to manufactures protocol)
Pick a single colony from a freshly streaked selective plate and inoculate a starter culture of 2–5 ml LB medium containing the appropriate selective antibiotic. Incubate for ~8 h at 37°C with vigorous shaking (~300 rpm).
Dilute the starter culture 1/500 to 1/1000 into selective LB medium. For high-copy plasmids inoculate 100 ml medium, and for low-copy plasmids, inoculate 250 ml medium. Grow at 37°C for 12–16 h with vigorous shaking (~300 rpm).
Harvest the bacterial cells by centrifugation at 6000 x g for 15 min at 4°C. Resuspend the bacterial pellet in 10 ml Buffer P1
Add 10 ml Buffer P2, mix gently but thoroughly by inverting 4–6 times, and incubate at room temperature for 5 min. The lysis reaction time must not exceed 5 min.
Add 10 ml chilled Buffer P3 to the lysate, and mix immediately but gently by inverting 4–6 times. Proceed directly to next step. Do not incubate the lysate on ice.
Pour the lysate into the barrel of the QIAfilter Cartridge. Incubate at room temperature for 10 min. Do not insert the plunger!
Remove the cap from the QIAfilter Cartridge outlet nozzle. Gently insert the plunger into the QIAfilter Maxi Cartridge and filter the cell lysate into a 50 ml tube.
Add 2.5 ml Buffer ER to the filtered lysate, mix by inverting the tube approximately 10 times, and incubate on ice for 30 min.
Equilibrate a QIAGEN-tip 500 by applying 10 ml Buffer QBT, and allow the column to empty by gravity flow.
Apply the filtered lysate from step 9 to the QIAGEN-tip and allow it to enter the resin by gravity flow.
Wash the QIAGEN-tip with 2 x 30 ml Buffer QC.
Elute DNA with 15 ml Buffer QN.
Precipitate DNA by adding 10.5 ml (0.7 volumes) room-temperature isopropanol to the eluted DNA. Mix and centrifuge immediately at ≥15,000 x g for 30 min at 4°C. Carefully decant the supernatant.
Wash DNA pellet with 5 ml of endotoxin-free room-temperature 70% ethanol (add 40 ml of 96–100% ethanol to the endotoxin-free water supplied with the kit) and centrifuge at ≥15,000 x g for 10 min. Carefully decant the supernatant without disturbing the pellet.
Air-dry the pellet for 5–10 min, and redissolve the DNA in a suitable volume of endotoxin-free Buffer TE.
To determine the yield, DNA concentration should be determined by both UV spectrophotometry and quantitative analysis on an agarose gel.
Restriction enzyme analysis
Materials
Procedure
The components for restriction analysis of the fungi plasmid are mixed. The recipe for 1 restriction analysis below.
Ingredients
Volume: 10µl
Device
5 µl
Buffer 3
3 µl
Restriction enzyme
1,5 µl
H2O
5,5 µl
For fungal plasmides the restriction enzyme used: BgI II.
For mammalian plasmides the restriction enzyme used: ScaI.
The mixture is incubated for 1 hour and applied to gel electrophoresis
Fungi
Transformation in fungi
Genetic Transformation of filamentous fungi – protocol from Center for Microbial Biotechnology (CMB) at DTU, Author Nielsen, J. B.
Materials
Media
Minimal medium (MM) (1L):
50 mL D-glucose 20% w/V, 20 mL 50x mineral mix, 10 mL 1 M sodium nitrate, 20 g ager.
Transformation media(TM)(1L):
342.3 g Sucrose, 20 mL 50x mineral mix, 20 g agar.
Mineral Mix (1L):
26g KCL, 26g MgSO4·7H2O, 76g KH2PO4, 50 mL Trace element solution, MilliQ water to volume 1000 mL.
D-glucose 20% (0.5 L):
100g D-glucose and MilliQ water up to 500 ml.
Aspergillus protoplastationbuffer (APB):
Final conc. 1.1 M MgSO4 and 10 mM Na-phosphate buffer. pH is adjusted with 2 N NaOH to 5.8.
Aspergillus transformation buffer (ATB):
Final conc: 1.2 M Sorbitol; 50 mM CaCl2·2 H2O; 20 mM Tris; and 0.6 M KCl. pH is adjusted with 2 N HCl to 7.2.
PCT (200ml) - Final conc: 50% w/vol PEG 8000;
50 mM CaCl2; 20 mM Tris; and 0.6 M KCl. pH is adjusted with 2 N HCl to 7.5. Store at 4 °C.
Procedure
The host strain is grown as three-point stabs on Minimal medium plates with the require suppliants added. MM will be referred to as MM with the supplements included throughout the protocol.
Inoculation:
The conidia are harvested by adding 5 ml of MM and firmly rub with a sterile Grigalsky spatula. The conidial suspension is pipetted to a sterile 500 ml shake flask containing 100 ml MM. The cultures are incubated at 30 °C with 150 rpm of shaking over night (14-20 hours).
Mycelial harvest:
A funnel with a sterile Mira cloth (filter) is used to harvest mycelia. To remove residual glucose from mycelia the biomass are washed with Aspergillus protoplastationbuffer (APB).
The filtered biomass is transferred to a new Falcon tube with a sterile spoon.
Protoplastation:
Mycellium is resuspended in 10 ml filter-sterillized(0.45μm filters) APB containing 40 mg Glucanex/ml. The Glucanex is dissolved in APB with gentle magnetic stirring less than 100/min.
Mycelia with dissolved Glucanex are mixed at 30 °C with 150 rpm of shaking for 2-3 hours.
Protoplast solutions are diluted in APB adding up to 40 ml mark. An overlay of max. 5ml Aspergillus transformation buffer (ATB) diluted to ½x with sterile MilliQ-water is carefully placed on top of the APB.
Centrifuged at 13 min 3000 RCF in Sorvall centrifuge. Protoplates should be observed as a halo of with slurry in the interphase of the two liquids.
Withdraw of the protoplasts are done with pipette and placed in a Falcon tube.
ATB is added up to 40 ml mark. Centrifuge at 300 RCF in 13 min and supernatanted are discarded.
The protoplastes are resuspended in 1 ml ATB with a 5 ml pipette.
Genetic transformation:
Aliquotes of 50 μl are transferred to a 1.5 ml Eppendorf tube containgen 10 μl of DNA for transformation.
Protoplast and DNA are incubated at room temperature for at least 30 min.
Protoplat and DNA suspension are added to 1 ml PCT in a 15 ml tube and shake gently.
Incubate for 1-5 min at room temperature.
Dilute in 3 ml ATB. The tube is filled with molten transformation medium (TM) agar (temperature of 40-45°C) to apporeimately 12 ml. The tube is mix rapidly by inverting the tube twice.
Poure directly on pre-made TM plates and incubate at 37°C for 3-8 days.
Production of conidiospores
Materials
tooth stick
LB plates
Procedure
Gently touch the a mature colony with a tooth pick.
Hold the LB-plate upside down to avoid scattering of spores, and stab three different places.
Incubated plates at 37°C for approximately five days. After incubation spores were either harvested from the plates or they were stored in the fridge at 4°C.
Flourescence Microscopy
Materials
MM
Bleomycin
4-NQO
Procedure
Cultivation for fluorescence detection was performed by collecting the conidia from plates having three point stab colonies that have grown for 4-5 days. 5ml of MM with supplements were added to the plate and conidia were rubbed off the colonies with a sterile Drigalsky spatula.
Inoculate the conidial suspension in 5ml MM in a sterile 50ml Falcon.
Add bleomycin (10mg/mL) and 4-NQO (5 mg/mL). This will cause damage to the DNA inducing gene expression
Incubated at 37 °C with 150 rpm of shaking overnight.
Fluorescence detection
Materials
Coverslide + overlay cover.
Conidia Culture.
Procedure
Take 500µL of conidia culture (ref. till protocol for det) in a tube. This has to be done after 4, 24, 48 and 120 hours .
Centrifuge the tube at 16000g for 1min and 30 sec.
Discard the supernatant.
Resolute the pellet in 5µL Milli Q water and place on slide. Place the overlay cover
Detection microscopy was performed first with DIT filter and fluorescence filter with enlargement of 10x and 100x.
To check background fluorescence, comparison  of detection with yellow filter was made against detection by green and red filter.
Extraction of proteins
Materials
Z-buffer
AEBSF stock solution
Bradford Reagent
BSA
MilliQ water
Minimal media with aminoacids
Procedure
Inoculation of conidia in shaking bottle
The colonies from the three point stab are harvested by adding 2mL MilliQ water to the petri dish. The colonies are rubt with a digalski spartula.
500µl of the colony suspension is pipette into a 500mL shaking bottle with 100mL minimal media containing amino acids
Incubate in 48 hours at 37°C
Protein extract
2 mL culture are transferred from the shaking bottles to eppendorf tubes. Remember to do triple determination.
Centrifuge tubes in 1 min with 8000g
Remove the supernatant.
Transfer the mycelierne with a sterile spartula to FastPrep tubes.
250 µL Z-buffer is added to the FastPrep tubes in stink cabinet.
Add approximately 200µL of the small glass balls to the FastPrep tubes.
Add 12,5µL AEBSF stock solution.
Place the FastPrep tubes in the FastPrep machines. Let it run with max velocity  for 30 sec.
Tubes are kept on ice from now on. Add 250 µL Z-buffer and mix well.
The extract is transferred by a 1000 µL pipette. Place the pipet at the bottom of the tube and transfer it to a new eppendorf tube.
Centrifuge the eppendorp tubes in 15 min at 10000g to purify the protein extract.
Transfer the supernatant to a new eppendorp tube.
Assays
Materials
Z-buffer
AEBSF stock solution
Bradford Reagent
BSA
MilliQ water
Minimal media with aminoacids
Protein extract (from above)
ß-galactosidase assay
Procedure
Add 25 µL extract and 225 µL Z-buffer to a well in a microtiter plate. Three wells per extract for triple determination.
Mix enough ONPG stock solution so 50 µL can be added to each well.
Measure OD with the computer program Gen5; Shake the plate every 30 sec to mix the solution, rest for 10 min before measuring the OD.
Bradford assay
Procedure
Add 5 µL extract and 245 µL Bradford reagent to a well in a microtiter plate. Three wells per extract for triple determination.
Make a BSA standard with concentrations between 0,1-2,0mg/mL with. See table below:
Preparation of BSA standards
Standard
µL standard
µL dH2O
Concentration
Std 1
300 µL stock
1200µL
2 mg/ml
Std 2
200 µL stock
1000µL
1,67mg/ml
Std 3
700 µL Std 1
700µL
1 mg/ml
Std 4
700 µL Std 2
700µL
mg/ml
Std 5
700 µL Std 3
700µL
0,5 mg/ml
Std 6
700 µL Std 4
700µL
mg/ml
Std 7
700 µL Std 5
700µL
0.25mg/ml
Std 8
700 µL Std 7
700µL
0,125mg/ml
Std 9
-
700µL
0 mg/ml
Make triple determination.
Incubate at 37°C for 10 min.
Mammalian cells
Cell culture and reagents
Materials
cells: U2OS cells were kindly provided by The Danish Cancer Society.  U2OS cell line is derived from the bone tissue of a patient suffering from osteosarcoma. U2OS cells show epithelial adherent morphology.
Medium:
500 ml DMEM
50 ml Fetal Calf Serum (FCS)
5 ml Penicillin
5 ml Streptomycin
Procedures
Add FCS, penicillin, and streptomycin to a new flask of DMEM. Store at 5°C.
The cells are cultivated in Dulbecco’s Modified Eagle Medium (DMEM), supplemented with Penicillin, Streptomycin, and 10 % heat-inactivated foetal calf serum (FCS). Penicillin and Streptomycin are added to prevent any microbial growth. FCS is added to supply essential non-defined components, such as serum proteins and lipids. Supplemented DMEM medium is referred to as complete DMEM throughout the report. The cells are adherent and are kept in 75 cm2 culture flask until 80-100% confluency, where they are passed on to new culture flasks or cover slips.
Cultivation of cells
Materials
25 cm2 culture flask
Complete DMEM
Procedures
Place the appropriate amount of EDTA-trypsin and complete DMEM in the incubator (37°C), before handling the cells (about 1-2 hours before).
The U2OS cells are kept at -80°C until defrosted gently at 37°.
Exactly when defrosted, they are mixed with 12 ml complete DMEM in the pipette.
The cell suspension is transferred to a 15 ml vial and centrifuged at 280 g for 10 minutes. The supernatant is discarded.
The cell pellet is resuspended in 5 ml complete DMEM and transferred to a 25 cm2 culture flask.
The cells are kept at 37°C in a  5% CO2 incubator until the following day, where they are passed on to larger culture flasks.
Passing and maintenance of U2OS cells
Materials
75 cm2 culture flask:
50 ml vial
Complete DMEM medium
0.05 % EDTA-trypsin
Procedure
The appropriate amount of EDTA-trypsin and complete DMEM in the incubator (37°C), before handling the cells.
The filter of a 1 ml pipette is broken off, and the pipette is attached to the vacuum suction maschine. The medium is removed, and the pipette is put in a 50 ml vial for later use.
1 ml 0,05% EDTA-trypsin/PBS is added to wash the cells. Tilt the flask quickly, so the EDTA-trypsin covers the cells. Remove the liquid quickly.
1 ml 0.05% trypsin-EDTA/PBS is added and the flask is incubated for 3 min at 37°C and 5 % CO2.
9 ml complete medium is added to inactivate the EDTA-trypsin and to wash the cells of the surface. Make sure the cells are well resuspended, before you transfer them.
2,5 ml (depends on the concentration of the cells) cell suspension is transferred to a new 75 cm2 culture flask.
The cells are kept at 37°C in a 5% CO2 incubator until they are 80-100% confluent. This takes about 2-3 days. Then they are passed on to a new flask or plate.
Transferring cells to coverslips
Materials
75 cm2 culture flask:
50 ml vial
Complete DMEM medium
0.05 % EDTA-trypsin
6-well plate
Cover slips
Procedure
The filter of a 1 ml pipette is broken off, and the pipette is attached to the vacuum suction maschine. The medium is removed, and the pipette is put in a 50 ml vial for later use.
1,5 ml 0,05% EDTA-trypsin/PBS is added to wash the cells. Tilt the flask quickly, so the EDTA-trypsin covers the cells. Remove the liquid quickly.
1,5 ml 0.05% trypsin-EDTA/PBS is added and the flask is incubated for 3 min at 37°C and 5 % CO2.
9 ml complete medium is added to inactivate the EDTA-trypsin and to wash the cells of the surface. Make sure the cells are well resuspended, before you transfer them.
Calculate the concentration of cells you want in the wells
Cover slips are placed in the bottom of a 6-well plate (2 cover slips/well). 2 ml cell suspension is added gently to each well.
The plate is placed in the incubator at 37°C and 5% CO2 O/N.
Transfection of cells
Materials
Optimem
FuGENE transfection reagent
Plasmid DNA
Procedure
The LAF bench and gloves is disinfected with ethanol before and after working in the LAF bench.
46μl optimem per is transferred to a 1,5 ml eppendorf tube per tranfection.
3 μl of Fugene is mixed well with the optimem.
The tube is flicked gently to make sure that the solution is properly mixed. Incubation for 5 min at room temperature
1 μl of plasmid DNA is mixed well with the optimem/Fugene mix.
The tube is flicked gently to make sure the mixture is properly mixed.
If there is any liquid remaining on the side of the tube, the tube is centrifuged shortly.
The plasmid mixture is incubated for 15 min at 37°C.
The  dish with U-2OS cells from the incubator.
The 50 μl transfection mixture is added dropwise to the 6 cm2 dish containg the coverslips.
The mixture is gently mixed by rocking motion and the dish is placed back in the incubator.
Coverslips for microscopy
Materialer
PBS
MilliQ water
Formaldehyde
lint-free paper
Cover slides
Transparent nail polish
Procedure
A 10X PBS dilution is made from PBS and MilliQ water.
The coverslips are transferred to a 24-well plate (2 coverslip/well).
The coverslips are washed by adding 1 ml diluted PBS to the wells. The PBS is discarded. This is done twice
400 μl Formaldehyde is added to each well under the fume hood, and incubated for 12 min at RT. The liquid is transferred to the waste bin
The coverslips are washed 3 times with the diluted PBS.
The coverslips are dipped in MilliQ water before laid on lint-free paper for drying.
4 drops each of 4 μl Vectashield is placed on a glass slide.
When a coverslip is completely dry, it is placed on a drop of Vectashield on the glass slide. The procedure is repeated for each coverslip.
The coverslips are fixated with transparent nail polish.
Retrieved from " http://2011.igem.org/Team:DTU-Denmark-2/Team/Protocols "
Recent changes
What links here
Related changes
Special pages
My preferences
Printable version
Permanent link
Privacy policy
Disclaimers
