<HTML xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en" dir="ltr">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki  ltr ns-0 ns-subject page-Team_DTU-Denmark_Notebook"><DIV id="globalWrapper"><DIV id="top-section"><DIV id="p-logo"><A href="/Main_Page" title="Main Page">&quot;
	    </A></DIV><DIV id="menubar" class="left-menu noprint"><UL><LI class="selected"><A href="/Team:DTU-Denmark/Notebook">Page               </A></LI><LI class="new"><A href="/wiki/index.php?title=Talk:Team:DTU-Denmark/Notebook&amp;action=edit&amp;redlink=1">Discussion               </A></LI><LI><A href="/wiki/index.php?title=Team:DTU-Denmark/Notebook&amp;action=edit">View source               </A></LI><LI><A href="/wiki/index.php?title=Team:DTU-Denmark/Notebook&amp;action=history">History               </A></LI><LI style="color:#808080;cursor:default">teams</LI></UL></DIV><DIV class="right-menu noprint" id="menubar"><UL><LI id="pt-login"><A href="/wiki/index.php?title=Special:UserLogin&amp;returnto=Team:DTU-Denmark/Notebook" title="You are encouraged to log in; however, it is not mandatory [o]" accesskey="o">Log in</A></LI></UL></DIV><DIV id="search-controls" class="noprint"><FORM action="/Special:Search" id="searchform"> </FORM></DIV></DIV><DIV id="content"><H1 class="firstHeading">Team:DTU-Denmark/Notebook</H1><DIV id="bodyContent"><H3 id="siteSub" class="noprint">From 2011.igem.org</H3><DIV id="navigation"><DIV class="navigation"><DIV class="centering"><UL class="topnav" id="topnav"><LI class="home navitem1"><A href="/Team:DTU-Denmark">Home</A></LI><LI class="vision navitem2"><A href="/Team:DTU-Denmark/Vision" title="Test: A soft introduction to the idea and it's future applications.">Vision</A></LI><LI class="project navitem3"><A href="/Team:DTU-Denmark/Project">Project</A></LI><LI><A href="/Team:DTU-Denmark/Project">Overview</A></LI><LI><A href="/Team:DTU-Denmark/Bioinformatic">Bioinformatics</A></LI><LI><A href="/Team:DTU-Denmark/Project_testing_sRNA">Experiment: Testing sRNA</A></LI><LI><A href="/Team:DTU-Denmark/Project_improving_araBAD">Experiment: Improving araBAD</A></LI><LI><A href="/Team:DTU-Denmark/Modeling">Modeling</A></LI><LI><A href="/Team:DTU-Denmark/Data">Data page</A></LI><LI><A href="/Team:DTU-Denmark/Attributions">Attributions</A></LI></UL><LI class="technical_stuff navitem4"><A href="/Team:DTU-Denmark/Technical_stuff">Technical stuff</A></LI><LI><A href="/Team:DTU-Denmark/Parts">Parts</A></LI><LI><A href="/Team:DTU-Denmark/Notebook">Notebook</A></LI><LI><A href="/Team:DTU-Denmark/Protocols">Protocols</A></LI><LI><A href="/Team:DTU-Denmark/Technical_stuff_math">Math</A></LI><LI><A href="/Team:DTU-Denmark/Matlab">Matlab</A></LI><LI><A href="/Team:DTU-Denmark/Safety">Safety</A></LI><LI class="background navitem5"><A href="/Team:DTU-Denmark/Background">Background</A></LI><LI><A href="/Team:DTU-Denmark/Background_sRNA">Small RNA</A></LI><LI><A href="/Team:DTU-Denmark/Background_the_natural_system">The chitobiose system</A></LI><LI><A href="/Team:DTU-Denmark/Background_synthetic_biology">Synthetic biology</A></LI><LI class="team navitem6"><A href="/Team:DTU-Denmark/Team">Team</A></LI></DIV></DIV></DIV><DIV id="heading"><DIV class="centering"><H1>

Notebook

            </H1></DIV></DIV><DIV id="innercontent"><DIV class="centering"><DIV class="whitebox article"><TABLE id="toc" class="toc"><TBODY><TR><TD><DIV id="toctitle"><H2>Contents</H2></DIV><UL><LI class="toclevel-1 tocsection-1"><A href="#Constructed_plasmids"><SPAN class="tocnumber">1</SPAN><SPAN class="toctext">Constructed plasmids</SPAN></A></LI><LI class="toclevel-2 tocsection-2"><A href="#PChiP.2FPChiP.28mutated.29-lacZ_fusion"><SPAN class="tocnumber">1.1</SPAN><SPAN class="toctext">P<SUB>ChiP</SUB>/P<SUB>ChiP(mutated)</SUB>-lacZ fusion</SPAN></A></LI><LI class="toclevel-2 tocsection-3"><A href="#PTet-chiX_fusion"><SPAN class="tocnumber">1.2</SPAN><SPAN class="toctext">P<SUB>Tet</SUB>-chiX fusion</SPAN></A></LI><LI class="toclevel-2 tocsection-4"><A href="#PBad-chiXR_fusion"><SPAN class="tocnumber">1.3</SPAN><SPAN class="toctext">P<SUB>Bad</SUB>-chiXR fusion</SPAN></A></LI></UL><LI class="toclevel-1 tocsection-5"><A href="#Strain_construction"><SPAN class="tocnumber">2</SPAN><SPAN class="toctext">Strain construction</SPAN></A></LI><LI class="toclevel-1 tocsection-6"><A href="#Arabinose_promoter_improvement"><SPAN class="tocnumber">3</SPAN><SPAN class="toctext">Arabinose promoter improvement</SPAN></A></LI></TD></TR></TBODY></TABLE><H2><SPAN class="mw-headline" id="Constructed_plasmids"> Constructed plasmids </SPAN></H2><H3><SPAN class="mw-headline" id="PChiP.2FPChiP.28mutated.29-lacZ_fusion"> P<SUB>ChiP</SUB>/P<SUB>ChiP(mutated)</SUB>-lacZ fusion </SPAN></H3><P>The plasmids containing this DNA fragments were obtained by the following procedure:
</P><OL><LI><B>PCR</B>:
</LI><LI><B>ChiP:</B> using E. coli w3110 genomic DNA as a template and TAQ enzyme. Using upstream part of ChiP and two different downstream primers (one wild type and one with mutation). we obtained two types of ChiP: <B>ChiP original</B> and <B>ChiP mutated</B>, which has altered binding site for ChiX. 
</LI><LI><B>lacZ:</B>from BBa_I72005 biobrick using phusion enzyme. 
</LI><LI><B>pSB1C3</B></LI></OL><LI><B>Restriction digestions</B>: 
</LI><LI><B>ChiP original</B> and <B>ChiP mutated</B> with XbaI and NcoI, 
</LI><LI><B>lacZ</B>  with BspHI and PstI
</LI><LI><B>pSB1C3</B> with EcoRI and PstI <DIV class="thumb tright"><DIV class="thumbinner" style="width:302px;"><DIV class="thumbcaption">Fig.1. A linearized version of the P<SUB>chiP</SUB>-lacZ construct inside the pSB1C3 plasmid. Restriction sites used, as well as expected fragment sizes are indicated.</DIV></DIV></DIV></LI><LI><B>Ligations</B>:
</LI><LI><B>ChiP normal</B> with <B>lacZ</B> and <B>pSB1C3</B></LI><LI><B>ChiP mutated</B> with <B>lacZ</B> and <B>pSB1C3</B></LI><LI><B>Transformations</B> to <I>E. coli</I> NM522 cells.
</LI><LI><B>Plating the bacterial</B>and grow over night in 37℃ incubator
</LI><LI><B>Start of a liquid culture</B> of transformed strain.
</LI><LI><B>Mini prep the plasmid, do digestion analysis: </B>with EcoRI, PstI, EcoRV. <B>(fig.1)</B></LI><LI><B>Transformation</B> to NM522, IG9 and IG302 strain.
</LI><LI><B>Beta-Galactosidase Assay</B> Pick 3 transformants of chip original and chip mutated, and grow. Make betagal assays +/- ara.
</LI><H3><SPAN class="mw-headline" id="PTet-chiX_fusion"> P<SUB>Tet</SUB>-chiX fusion </SPAN></H3><P>The plasmids containing this DNA fragments were obtained by the following procedure:
</P><OL><LI><B>Start liquid culture of minigenes</B>: chiX, tetR
</LI><LI><B>PCR</B>:pSB4k5 (or any other non cam non pSB1 pSB6 plasmid)
</LI><LI><B>Purification of the PCR product</B></LI><LI><B>Mini prep minigenes</B></LI><LI><B>Restriction digestions</B>:
</LI><LI><B>chiX</B> with XbaI and PstI
</LI><LI><B>tetR</B> with EcoRI and SpeI
</LI><LI><B>pSB1C3</B> with EcoRI and PstI
</LI><LI><B>pSB4k5</B> with EcoRI and PstI
</LI></OL><LI><B>Ligations</B> of:<B>tetR</B> with <B>chiX</B> and <B>pSB1C3</B></LI><LI><B>Transformations</B> to <I>E. coli</I> NM522 cells.
</LI><LI><B>Plating the bacterial</B>and grow over night in 37℃ incubator
</LI><LI><B>Colony PCR</B>with CS149/150 on transformants. <B>gel analysis</B><B>(fig.2)</B></LI><LI><B>Start of a liquid culture</B> of correct ones.
</LI><LI><B>Mini prep the plasmid, do digestion analysis:</B>with EcoRI, PstI <DIV class="thumb tright"><DIV class="thumbinner" style="width:402px;"><DIV class="thumbcaption">Fig.2. Colony PCR of cells transformed with P<SUB>tet</SUB>-chiX inside pSB1C3, CS149 and CS150 primers were used. Expected band size is 1391 bp</DIV></DIV></DIV></LI><LI><B>Subclone Ptet-chiX part into pSB4k5</B></LI><LI><B>Transformation</B> to <I>E. coli</I> NM522 cells.
</LI><LI><B>Colony PCR</B> with CS149/150 on transformants.Pick 4 positive colonies and grow overnight
</LI><LI><B>Minipreps</B> the corect ones, <B>Transform</B> into IG302/pSB1C3 PchiP-lacZ
</LI><LI><B>Beta-Galactosidase Assay</B> Pick 2 transformants and grow. Make Beta-Galactosidase Assay +/- atc.
</LI><H3><SPAN class="mw-headline" id="PBad-chiXR_fusion"> P<SUB>Bad</SUB>-chiXR fusion </SPAN></H3><P>The plasmids containing this DNA fragments were obtained by the following procedure:
</P><OL><LI><B>Start liquid culture of minigenes</B>: chiXR
</LI><LI><B>PCR</B>:
</LI><LI>pSB3T5 (or any other non cam non pSB1 pSB6 plasmid)
</LI><LI>pBad 
</LI><LI>pSB1C3
</LI></OL><LI><B>Purification of the PCR product</B></LI><LI><B>Mini prep minigenes</B></LI><LI><B>Restriction digestions</B>:
</LI><LI><B>pBad</B> with SpeI and EcoRI
</LI><LI><B>chiXR</B> with XbaI and PstI
</LI><LI><B>pSB1C3</B> with EcoRI and PstI
</LI><LI><B>pSB4k5</B> with EcoRI and PstI
</LI><LI><B>Ligations</B> of:<B>pBad</B> with <B>chiXR</B> and <B>pSB1C3</B></LI><LI><B>Transformations</B> to <I>E. coli</I> NM522 cells.
</LI><LI><B>Plating the bacterial</B> and grow over night in 37℃ incubator
</LI><LI><B>Start liquid culture</B> of transformed strain.
</LI><LI><B>Mini prep the plasmid, do digestion analysis:</B>with EcoRI, PstI <B>(fig.3)</B><DIV class="thumb tright"><DIV class="thumbinner" style="width:372px;"><DIV class="thumbcaption">Fig.3. The gel picture of the P<SUB>BAD</SUB>-chiXR construct. The restriction enzymes used, and the sizes are indicated on the picture. EN = EcoRI + NcoI, EB = EcoRI + BamHI</DIV></DIV></DIV></LI><LI><B>Subclone PBad-chiXR part into pSB3T5</B></LI><LI><B>Transformation</B> to <I>E. coli</I> NM522 cells.
</LI><LI><B>Colony PCR</B> with CS149/150 on transformants.Pick 4 positive colonies and grow overnight
</LI><LI><B>Minipreps</B> the corect ones, <B>Transform</B> into NM522/pSB1C3PchiP-lacZ and IG09/pSB1C3PchiP-lacZ 
</LI><LI><B>Beta-Galactosidase Assay</B> Pick 2 transformants and grow. Make betagal assays +/- ara.
</LI><H2><SPAN class="mw-headline" id="Strain_construction"> Strain construction </SPAN></H2><P>The aim of this experimental section was to create strain with following gene deletetions:<I>lacZYA</I> and <I>chbBCARG</I> operon. 
</P><OL><LI> PCR of recombineering linear pieces of DNA using pSLD31 plasmid and RS233/RS234 primers pair (<I>lacZYA</I> deletion) and chb upstream/downstream primers pair. pSLD31, was provided by our supervisor Sebastién Lemire, and it contains a kanamycin resistance gene flanked by <I>lox66</I> and <I>lox71</I> sites. The upstream/downstream primers pair anneal at the <I>lox</I> sites and they contain overhangs providing homology region to the sequences flanking &quot;gene-to-be&quot; deleted.
</LI><LI> Transformation of the <I>E. coli</I> W3110 strain containing pSLD18 plasmid (provides Red recombineering system) with pfhc3220 plasmid (containig Cre recombinase).
</LI><LI> Once the strain containing both plasmids was constructed we were ready to proceed with the first round of recombineering. We followed the protocol describing this procedure to achieve first gene deletion - <I>$\Delta$chbBCARG</I>. First we screened for the gain of kanamycin resistance and did the colony PCR to confirm correct insert size. Then the Cre recombinase was activated and facilitated pop-out event of the resistance marker. We screened for loss of resistance and, again, confirmed the correct sizes with colony PCR. Colony PCRs were done using CSchb upstream/downstream primers pair. Correct strain was saved as IG7 and stored under -80<SUP>o</SUP>C.
</LI><LI> As the next step, the IG7 strain was transformed with linear piece of DNA designed for <I>lacZYA</I> deletion. Gain of kanamycin resistance was screened for and the expected size of the insert was confirmed with colony PCR. After that the loss of resistance was conducted by growing the cells in minimum medium arabinose (0.15%). We screened for the pop-out event and colony PCRed the strain to confirm the correct recombineering. Primers used were called CSlac upstream/downstream. Finally, the strain was named IG9 and saved under -80<SUP>o</SUP>C.
</LI></OL><P>Primers used:
</P><UL><LI>RS233 ACCAGTACGTTTTCCGCAGCTGTGGGTCAAAGAGGCATGACCTCCTTAGGAATTCTACCG
</LI><LI>RS234 GCTGGCACGACAGGTTTCCCGACTGGAAAGCGGGCAGTGACTAGCGGATCCTACCGTT
</LI><LI>chb upstream GGCCTGAGTTCTTAATTATCTTCGCGAATTATTTGCCCGATGTGTAGGCTGGAGCTGCTT
</LI><LI>chb downstream CCGGATGCAAGGTTCACGCCGCATCTGGCAAACATCCTCACATATGAATATCCTCCTTAG
</LI><LI>CSchb upstream TATCCGGCCTACAAAACTGTG
</LI><LI>CSchb downstream GCTGAATACGACACTGATCCAC
</LI><LI>CSlac upstream CGCTACAACGGGTAGCAA
</LI><LI>CSlac downstream TCGTTGGGCTGATGATCATAA
</LI></UL><H2><SPAN class="mw-headline" id="Arabinose_promoter_improvement"> Arabinose promoter improvement </SPAN></H2><P>Biobrick BBa_I746908 containing P<SUB>BAD</SUB>/araC promoter, BBa_I0500, fused with GFP was the scaffold for improving and testing the promoter.
</P><OL><LI> First, set of primers was used to PCR the p<SUB>BAD</SUB> promoter. 5 primers annealing to the promoter, SRS56-1, -2, -3, -4, -SPL, introduced either designed changes in the -35 and -10 region (first 4 primers) or random point mutations in the spacer region (-SPL primter). The SRS57 primer was used in pair with each of the these primers. 
</LI><LI> BBa_I746908 on the pSB1A2 plasmid was digested with BamHI and KpnI, while the PCR products were cut with BglII and KpnI.  
</LI><LI> Plasmid digestion was SAP treated in order to lower the chance of self-religating of the plasmid during the ligation step. 
</LI><LI> Now, the digested PCR products and the digested plasmid was ligated and transformed. 
</LI><LI> Then, after purifying the plasmids, the constructs could be tested for the correct insert by digesting with KpnI and BamHI. If the correct constructs were created (PCR products inserted into the plasmid) then restriction with BamHI couldn't take place. This restriction site was lost when fused with fragment digested with BglII. As a result only one band could be observed on the gel picture.
</LI><LI> Once the constructs were confirmed we analysed the GFP expression in BioLektor with 0.2% arabinose as inducer. 
</LI></OL><P>Primers used:
</P><UL><LI>SRS56-1 tagcagatcttacctgacgctttttatcgcaactctctaTAAtttctccatacccgtttttttgggctagctactagag
</LI><LI>SRS56-2 tagcagatcttacTtgacActttttatcgcaactctctactgtttctccatacccgtttttttgggctagctactagag
</LI><LI>SRS56-3 tagcagatcttacctgacActttttatcgcaactctctaTAAtttctccatacccgtttttttgggctagctactagag
</LI><LI>SRS56-4 tagcagatcttacctgacgctttttatcgcaactctctacAgtttctccatacccgtttttttgggctagctactagag
</LI><LI>SRS56SPL tagcagatctNNNctgacgNNNNNNNNNNNNNNNNNNtactgtNNNtccatacccgtttttttgggctagctactagag
</LI><LI>SRS57 ccaacccgggtaacgagcaaagcactgaacacca
</LI></UL></DIV></DIV></DIV><DIV id="footer-panel"><DIV class="centering"><DIV class="footerBox"><H4>This project is part of iGEM</H4><P>Read more on the <A href="https://2011.igem.org/Main_Page">iGEM Main Page</A></P></DIV><DIV class="footerBox"><H4>Sponsors</H4><P>Thanks to:</P></DIV><DIV class="footerBox"><H4>How to customize an iGEM wiki?</H4><P>Learn all the tricks in our <A href="/Team:DTU-Denmark/How_to_customize_an_iGEM_wiki">wiki guide</A></P></DIV></DIV></DIV><DIV class="printfooter">
Retrieved from &quot;<A href="http://2011.igem.org/Team:DTU-Denmark/Notebook">http://2011.igem.org/Team:DTU-Denmark/Notebook</A>&quot;</DIV></DIV></DIV><DIV id="footer-box" class="noprint"><DIV id="footer"><UL id="f-list"><LI id="t-recentchanges"><A href="/Special:RecentChanges" title="Recent changes">Recent changes</A></LI><LI id="t-whatlinkshere"><A href="/Special:WhatLinksHere/Team:DTU-Denmark/Notebook" title="List of all wiki pages that link here [j]" accesskey="j">What links here</A></LI><LI id="t-recentchangeslinked"><A href="/Special:RecentChangesLinked/Team:DTU-Denmark/Notebook" title="Recent changes in pages linked from this page [k]" accesskey="k">Related changes</A></LI><LI id="t-specialpages"><A href="/Special:SpecialPages" title="List of all special pages [q]" accesskey="q">Special pages</A></LI><LI><A href="/Special:Preferences">My preferences</A></LI></UL></DIV><DIV id="footer"><UL id="f-list"><LI id="t-print"><A href="/wiki/index.php?title=Team:DTU-Denmark/Notebook&amp;printable=yes" title="Printable version of this page [p]" accesskey="p">Printable version</A></LI><LI id="t-permalink"><A href="/wiki/index.php?title=Team:DTU-Denmark/Notebook&amp;oldid=169218" title="Permanent link to this revision of the page">Permanent link</A></LI><LI id="privacy"><A href="/2011.igem.org:Privacy_policy" title="2011.igem.org:Privacy policy">Privacy policy</A></LI><LI id="disclaimer"><A href="/2011.igem.org:General_disclaimer" title="2011.igem.org:General disclaimer">Disclaimers</A></LI></UL></DIV></DIV></DIV></BODY></HTML>