"
Page
Discussion
View source
History
teams
Log in
 
Team:DTU-Denmark-2/Project/PlugnplayAssembly
From 2011.igem.org
Home
About us
The Team
Supervisors
Gallery
The Project
Abstract
Introduction
The Plug 'n' Play
Assembly system
Customization
Other assembly systems
Achievements
Attributions
Results
Background
Characterization
Proof of concept
Mammalian Cells
Fungi
Data page
Submitted Biobricks
Collaboration
Safety
Safety
Safety rules
Notebook
Lab notes
Protocols
Our Partners
Sponsors
Advisors
Acknowledgements
Plug 'n' Play with DNA assembly standard
Making molecular biology easier
How it works
The procedure of assembly
Plug 'n' Play benefits
Improvements of BBF RFC 39
What you get
The design of the system
References
Making molecular biology easier
We imagine that iGEM should be about fast assembly of BioBricks, leaving more time for the actual project. It should be easy to combine any thinkable part, device or existing BioBrick. Unfortunately, classical cloning techniques can cause problems and even PCR can be cumbersome, if you have little or no laboratory experience.
Therefore, we introduce a simple and fast way of building new devices and vectors with ready to use PCR products. All that is needed is to select the favorite BioBricks and one of the ready to use backbone vectors, and mix it with USER enzyme. The assembled vector is ready for chemical transformation into competent E. coli cells only 70 minutes later.The 8-9 nucleotide overhangs (linkers) flanking the parts, can hybridize to complementary overhangs without the use of DNA-ligases.
No need to worry whether the destination vector has been fully linearized, no need to perform site-directed mutagenesis to remove unwished restriction recognition sites - just simple, fast, and easy.
It is possible to successfully assemble six biological parts in one reaction, reducing the plasmid construction time significantly. The upper limit of fragments that efficiently can be assembled has not yet been delineated (1).
In 2009 the first team from DTU participated in iGEM and introduced the USER fusion Assembly standard [BBF RFC 39] . This year we introduce a standardized version allowing easier use of the system. The Plug 'n' Play assembly standard can be found here (BBF RFC 80).
How it works
We have demonstrated that successful assembly of up to six biological parts in one reaction is possible e.g. (BBa_K678068) . The entire process of combining six BioBricks is illustrated in the figure below. The number of BioBricks can be adjusted as preferred and for instance mutations can easily be introduced by customizing the standard.
The procedure of assembly
Please refer to our Plug 'n' Play with DNA assembly standard BBF RFC 80 for a detailed protocol.
Pre-produced PCR products are selected from the Plug 'n' Play kit.
USER enzyme mix and buffer is added. This will remove the uracils, resulting in sticky ends on all BioBricks.  The BioBricks and the backbone plasmid have matching sticky ends, and will self-assemble.
Chemical transformation of E. coli. The time of the final incubation of the transformation mix can be reduced to 30 min when using ampicillin as antibiotic resistance marker and still results in a high number of transformants.     If a part is not included in the Plug 'n' Play kit a simple PCR amplification can be performed on the part intended for the assembly in accordance to the Customization .
Plug 'n' Play benefits
Improvements of BBF RFC 39
No need to use a USER cassette, thereby completely avoiding the use of restriction enzymes and increasing the flexibility of the system.
Standardized linkers
What you get
Rapid assembly of biological parts
A well documented system
Assembly of multiple biobricks in one step
No illegal restriction sites
100% restriction enzyme free
100% DNA ligase free
High fidelity ensured by PfuTurbo® 7x Hotstart DNA polymerase
Directionality of inserts are supported
Any vector can be made Plug’n’Play compatible
Suitable for large scale high throughput projects
Vectors for Plug 'n' Play have been prepared by PCR ensuring 100% linear fragments. This means that the occurrence of false positives after bacterial transformation can be completely avoided (2).
The USER compatible backbone vectors were made by PCR amplification with primers containing the standardized linkers 6 and 1.  For all the PCR reactions it is important to remove the template vector before utilizing it in the USER reaction to avoid inauspicious transformations and false positives. The template can be removed by DpnI treatment or by gel band purification(2).
Previous studies have demonstrated a high level of correct assembly and directionality of fragments (2,3) and although we did not make such analysis ourselves it was in general our perception that this was the case, as of the more than 25 different Plug 'n' Play assemblies performed we encountered no more than two false-positives.
It is well known that mutations can be introduced during PCR. To minimize mutations the x7 proof-reading polymerase (4) that are fully compatible with the Plug 'n' Play standard should be used. The x7 proof-reading polymerase helps lowering the risk of introducing mutations.
The design of the system
The system was designed based on the most common categories of biological part in the order; promotor, gene of interest (GOI), tail sequence (TS), terminator, marker cassette, and vector backbone. The standard marker cassette consists of a promoter, a gene, and a terminator. Short sequences are such as ribosomal binding sites (RBS) are meant to be integrated in the forward or reverse primer of the desired part when amplifying it by PCR. An example of this can be found on this page .
The system is dependent on the assembly of the custom-made linkers on each part. The parts with linkers are synthesized by PCR amplification of template DNA using primers designed with an upstream extension of 8-9 additional nucleotides with a single deoxyuridine residue directly upstream the part. In order to assemble multiple parts while having a flexible system the linkers have an adenine in the 5' end and a thymine in the 3' end. Furthermore, it is important that the linkers of the different parts are not identical to ensure the directionality and correct order of the biobricks.
Illustrated below, are the sequences of linkers between the parts, which are generated from the overhangs (linkers) on the PCR products.
The linkers are responsible for the flexibility of the system in that it for instance is possible use any promoter, since they will always have the same linkers. In the Plug 'n' Play system the coding sequence(s) can either consist of one gene of interest (GOI), called a module, or a GOI and a tail sequence, which could be any given DNA sequence of interest.
References
(1) Hansen, B.G.; Salomonsen, B.; Nielsen, M. T.; Nielsen, J. B.; Hansen, N. B.; Nielsen, K. F.; Regueira, T. B.; Nielsen, J.; Patil, K. R.; Mortensen, U. F.; “Versatile enzyme expression and Characterization system for Aspergillus, with the Penicillium brevicompactum Polyketide Synthase Gene from the Mycophenolic Acid Gene Cluster as a Test Case.” American Society for Microbiology, 2011, 3044-3051.
(2) Hansen, B. G.; Holm, D. K.; Nielsen M.T.; Mortensen, U.H. PCR based USER cloning for restriction enzyme and ligase-independent vector construction. Manuscript.
(3) Frandsen, R. J. N.; Andersson, J.A.; Kristensen, M. B.; Giese, H. Efficient four fragment cloning for the construction of vectors for targeted gene replacement in filamentous fungi.  BMC Molecular Biology 2008, 9:70.
(4) Nørholm, M. H. H. A mutant Pfu DNA polymerase designed for advanced uracil-excision DNA engineering. BMC Biotechnol. 10, 21 (2010).
Retrieved from " http://2011.igem.org/Team:DTU-Denmark-2/Project/PlugnplayAssembly "
Recent changes
What links here
Related changes
Special pages
My preferences
Printable version
Permanent link
Privacy policy
Disclaimers
