"
Page
Discussion
View source
History
teams
Log in
 
Team:DTU-Denmark/Protocols
From 2011.igem.org
Home
Vision
Project
Overview
Bioinformatics
Experiment: Testing sRNA
Experiment: Improving araBAD
Modeling
Data page
Attributions
Technical stuff
Parts
Notebook
Protocols
Math
Matlab
Safety
Background
Small RNA
The chitobiose system
Synthetic biology
Team
Protocols
Contents
1 β-gal assay
2 Gel preparation and electrophoresis
3 Ligation
4 Competent cells
5 PCR protocol
6 PCR program design
7 PCR product purification with NucleoSpin
8 Plasmid puriification
9 Recombineering
10 Restriction digestion
11 Transformation and selection
11.1 Part 1: Transformation by electroporation
11.2 Part 2: Selection of transformants
β-gal assay
This assay provides a way of measuring activity of β-Galactosidase which hydrolyzes β-galactosides into monosaccharides. One example of such reaction is hydrolysis of disaccharide lactose into glucose and galactose. Measurement β-Galactosidase activity is made using o-nitrophenyl-β-D-galactopyraniside (ONPG). Cleavage of ONPG results in release of yellow comound, o-nitrophenol, which absorbs 420 nm light. Activity of the enzyme is expressed in terms of increase of yellow color per minute.
Reagents:
Suited media
Z-buffer
2.5 mg/ml Fresh Lysozym
10% Triton X100
4mg/ml ONPG
0.5 M Na2CO3
Procedure: Day 1
Inoculate 10 ml suited media with 2-3 dilutions
Day 2
At OD450 =0.4-0.8 harvest 2 ml culture and resuspend in 2 ml Z-buffer.
Measure OD600.
In the following use 2 ml eppendorf tubes and make blind sample containing all the things except cells!
Pipet X µl cells and add Z-buffer to total volume of 0,832 ml.
Add 160 µl Fresh Lysozym solution and 8 µl 10% Triton X100.
Vortex for 5 sec.
Incubate for 5 min. at 30oC.
Add 100 µl 4 mg/ml ONPG.
Vortex for 5 sec. Note time.
Incubate on 30oC.
When the reaction turns yellow, add 1ml 0.5 M Na2CO3 to stop reaction. Note time.
Place the sample in the refrigerator while finishing all the samples.
Spin the samples for 2 min. at 15.000 rpm.
Measure OD420 and OD550, use the blind sample as reference. Measure both ODs on each sample. If OD550 is higher than 0.05 spin again and measure again.
Beta-gal acitivity = 1000 x (OD420 - 1.75 x OD550) / (T x V x OD600)
Gel preparation and electrophoresis
Gel preparation (1% gel) (100 ml of the buffer). Reagents:
100 ml of 1x TBE buffer .
10 μl of ethidium bomide (10 mg/ml).
1 g of agarose.
Assembled gel container.
Mix buffer with agarose and heat in microwave until the solution is clear
Add 10μl of ethidium bromide (10mg/ml).
Pour solution to the gel container and leave it to solidify (30-45 min).
Place gel container in the electrophoresis machine, remove combs and cover with the TBE buffer.
Gel electrophoresis:
2μl of DNA sample.
3 μl of distilled water.
1μl of 6x loading dye.
4.2 μl of Gene Ruler DNA ladder mix from Fermentas.
It is recommended to use 7V for each cm of the gel length and to run the gel for 45 min.
Ligation
Ligation mix (20 μl total):
2 μl 10x Buffer
vector+insert in molar ratio 1:5 (DNA concentration less than 50 ng)
1 μl ligase
fill up with water
Positive control (20 μl total):
2 μl 10x Buffer
vector (DNA concentration less than 50 ng)
1 μl ligase
fill up with water
Negative control (20 μl total):
2 μl 10x Buffer
vector (DNA concentration less than 50 ng)
fill up with water
Procedure:
Estimate insert and vector concentrations after gel electrophoresis.
Calculate amount of inser and vector to take for ligation.
Ligate – 1 hour at room temperature.
Transform competent cells; incubate; analyze growth on plates taking to account positive and negative control.
Competent cells
Day 1:
Inoculate 5 ml of LB from a colony or from a -80oC stock.
Day 2:
Dilute exponentially growing cells to OD600=0.05 and grow cells in a shaker to OD600=0.5-0.6 in pre-warmed LB. Remove the cells from the shaker and place on ice. Transfer liquid cultures to centrifuge tubes. Note: It’s very important to keep the cell on ice from now on.
Centrifuge at 6000 rpm for 10min; discard supernatant.
Gently resuspend in 5-7 ml of ice-cold 10% glycerol; consolidate to half the number of centrifuge tubes.
Fill up with 10% glycerol and centrifuge 10 min at 6 rpm.
Repeat step 3 and 4 (no consolidation) two to three times.
Resuspend in 5-7 ml of 10% glycerol and move to 15 ml or 50 ml tubes.
Centrifuge at 5000 rpm for 5 minutes, discard supernatant and resuspend gently in about 2 ml 10% glycerol pro L culture.
Flash freeze into Eppies placed in -80oC pure ethanol bath and store at -80oC. Make aliquots of 85µl, 135 µl and 175 µl for 2, 3 and 4 electroporations respectively.
Day 3:
Transform cells with 1 µl of 10 pg/µl of pUC19 (AmpR) to test efficiency of the competent cells.
Plate 10 µl and 100 µl on LB+Amp (100 µg/ml) plates.
Day 4:
Count colonies and calculate the transformation efficiency as colonies/µg of pUC DNA.
PCR protocol
Reagent
TAQ + PFU [μl]
Phusion [μl]
Enzyme
0.5
0.5
Forward primer
2.5
5
Reverse primer
2.5
5
dNTP
4
4
Template
1
1
Buffer
10
10
Water
79.5
74.5
PCR program design
Initial denaturation for 2 minutes at 95⁰C.
Denature for 1 minute at 95⁰C.
Anneal primers for 30 seconds at temperature ~5⁰C below melting temperature of primers.
Extend DNA at 72⁰C using each 1-2 minutes per kilobase of product, depending on polymerase used (see manufacturer’s instructions).
Repeat steps 2-4 for 25-30 cycles.
Final extension for 10 min at 72⁰C
PCR product purification with NucleoSpin
Mix 1 volume of sample with 2 volumes of NT buffer in an 1,5 ml Eppendorf tube.
Place a column into a 2 ml collection tube and load the sample.
Centrifuge at 11.000 g for 1 min.
Discard flow through and place the column back into the collection tube.
Add 600 µl NT3 buffer and centrifuge at 11.000 g for 1 min.
Discard flow through and place the column back into the collection tube.
Centrifuge at 11.000 g for 2 min to remove NT3 buffer. Discard flow through.
Place the column into a clean 1,5 ml Eppendorf tube.
Add 30 µl of water or NE buffer and incubate for 1 min to increase the yield of eluted DNA.
Centrifuge at 11.000 g for 1 min.
Plasmid puriification
This purification is based on the “Zyppy Plasmid Miniprep Kit”
Amounts of bacterial culture:  According to Zyppy the purification can be done on 600 ul cell culture, but our experience suggests that it is not enough for further processing/use of the DNA. We use 2-4 ml of cell culture.
Initial steps:
Add 1.5 ml of cell culture in LB medium to 2 ml eppendorf tube.
Spin at 15.000 g for 2 min.
Discard supernatant
Add 1.5 ml of cell culture (for a total of 3 ml cell culture)
Spin at 15.000 g for 2 min.
Remove as much supernatant as possible – pipette carefully. This is (the only) point of no return! To stop, freeze the pellet.
Add 600 µL of TE-buffer. Ensure that the pellet is completely suspended.
Add 100 µL 7x lysis buffer. Remember not to process more than 10 minipreps at a time.
Add 350 µL cold neutralization buffer. Mix gently and thoroughly (= all the way through ≠ violently)!
Spin at 15.000 g for 5 min.
Transfer the supernatant to the columns; be careful not to get some of the pellet! It’s better to leave some supernatant than to get some of the pellet. Several “lysis” can be poured together to up-concentrate.
Spin at 15.000 g for 30 sec.
Discard flow-through.
Add 200 µL endo-wash-buffer
Spin at 15.000 g for 30 sec.
Add 400 µL zyppy wash buffer
Spin at 15.000 g for 30 sec.
Transfer columns to clean 1.5 ml eppendorf tubes. Be careful when removing the tubes, the buffer may not touch the tip of the column! (if it happens, spin again).
Elute DNA in 30-100 ul of buffer of choice (TE/H2O/restriction buffer/Zyppy elution buffer). Add the buffer to the center of the column, but without touching the column material! If H2O, wait 5 min before proceeding to the final centrifugation step, as DNA is not easily suspended in water.
Spin at 15.000 g for 30 sec.
Check the purification by running a gel (at least until we get experienced with a high success rate).
Recombineering
The recombineering procedure is an efficient way of introducing multiple chromosomal gene deletions in E. coli in gradual manner. It exploits activity of two recombinases, Red and Cre. In the first step the Red recombinase inserts a linear DNA in place of a gene to be deleted based on sequence homology. Inserted DNA carries antibiotic resistance gene flanked with LoxP sites which can be selected for when growing bacteria on plates containing corresponding antibiotic. Then, Cre recombinase is activated and cuts out resistance gene from chromosome using the LoxP sites, which results in loss of resistance. Afterwards, the procedure can be repeated for subsequent gene deletions.
E. coli strain W3110, prior to be used with this protocol, was transformed with two helper plasmids: (1) pSLD18 containing the phage Lambda recombination system (also called Red) under temperature inducible promoter and erythromycin resistance gene, donated by our supervisor Sebastién Lemire; and (2) pHC3220 containing Cre recombinase under arabinose inducible promoter and ampicillin resistance gene, donated by Flemming G. Hansen. Both plasmids are temperature sensitive and temperature at which bacteria was grown is restricted to 30oC. Linear pieces of DNA where prepared (PCRed), that contained antibiotic resistance gene of choice, flanked by LoxP sites and homology regions to chromosomal regions flanking gene to be deleted.
Day 0 - Prepare overnight culture.
Pick wild-type E. coli colony from plate and whirl in 5 mL liquid LB + ery150 + amp100, grow at 30°C with aeration/shaking.
Day 1 – Gene knock-out
Prepare competent cells, (small batch):
Add 50 μL of ON culture to 5 mL LB, grow at 30°C for 2.5 hours (until exponential growth).
Bottles with cells are transferred to 42°C for EXACTLY 15 minutes, NO MORE.
The washing-buffer (10% w/w ultra pure glycerol in water) is put on ice
The cells are harvested for 5 minutes at 6000 rpm, 4°C in 2ml tubes. Repeat once for each tube to increase cell concentration.
The pellet is carefully re-suspended in 1 ml ice-cold glycerol 10%, and the cells are centrifuged for 5 minutes at 6000 rpm., 4°C. Discard the supernatant. This washing step is repeated three tims.
Resuspend in 50 µl cold 10% glycerol.
Transform cells with linear piece of DNA (see protocol for transformation).
Leave in recovery medium at 30oC for 2 hours with aeration/shaking.
Plate on plates containing appropriate antibiotic and grow overnight at 30oC.
Day 2 – Test transformation with PCR and remove resistance gene.
Do colony PCR for 4-8 colonies from yesterday transformations:
Pick a colony using a toothpick.
Whirl it in an eppendorf tube containing 100 μL TE.
Use 1 μL as template in PCR.
Move cells to minimal medium arabinose + selection. Use the eppendorf tubes from the colony PCR as a source of cells. The minimal medium is ABT medium with arabinose (0.15%) and ery150, amp100. Grow at 30°C.
At the end of the day restreak onto ery150 and/or amp100 plates, and grow at 30°C.
Day 3 – Select colonies that lost resistance.
Restreak onto LB and LB+kan50.
Day 4 – Check colonies by PCR.
Do colony PCR for 4-8 colonies from yesterday transformations:
Pick a colony using a toothpick.
Whirl it in an eppendorf tube containing 100 μL TE.
Use 1 μL as template in PCR.
Prepare ON culture for the next round of red-swap. Pick wild-type E. coli colony from plate and whirl in 5 mL liquid LB + ery150 + amp100, grow at 30°C with aeration/shaking.
Proceed with the subsequent gene deletion by starting the protocol from day 1, using a new PCR product.
After the last deletion the Red and Cre plasmids can be curated by growing cells at 37oC.
Restriction digestion
Reagents:
25 µL DNA
10 µL Buffer 10x
2.5µL of each restriction enzyme
Add  H2O to total volume of 100 µL
Mix everything. Leave for 2 hour at 37 C in thermo block/incubator - check the temperature with enzyme manufacturer.
Heat-deactivate enzyme – check the temperature with enzyme manufacturer.
Skip point 2 if proceeding immediately to DNA purification.
SAP treatment to prevent re-circularization of the plasmid:
Mix 50µL of the restricted DNA with 2.5µL SAP and 5µL SAP-buffer.
Put it in the incubator 37°C for 60 min.
15 minutes on heat block 65°C to deactivate enzyme
Transformation and selection
Part 1: Transformation by electroporation
Material:
ice
test-tubes
cuvettes
LB medium at room temperature
your strain
DNA
Procedure:
Get competent cells and store them on ice (the number on the lid indicate the number of transformation the amount corresponds to).
Pool the competent cells into one tube. This is done to ensure a homogen batch.
Put cuvettes for electroporation on ice.
Label as many glass tubes and cuvettes as samples and in addition to that, prepare some extra tubes. Add 1 ml recovery media (LB) to each. Check the LB media by shaking to ensure that it is sterile and there’s no bacteria growth.
Electorporation:
Use program EC2, and set the form to ‘time ms’, press the right blue square bottom to get started.
Add 2 µl of the ligation (DNA) to one side of the cuvette.
Add 40 µl of the competent cells to the same side of cuvette.
Mix by title gently, ensures that all liquid is at the bottom and that there are no bubbles. Dry the cuvette with paper towel.
Pipette 1 ml recovery media quickly.
Place the cuvette in the machine and press the pulse bottom, write down the time( usually is above 5 and below 5.8)
Immediately add 1 ml recovery media to the cuvette, pipette gently up and down for 1-2 times and try to get as much of the liquid retransferred to the tube where you took recovery media from.
Keep track of the time constant, it should preferentially be around 5.60. Mark the tube if the time constant is below 5.00 as the cells might be dead/weak. If the time-constant is 0.00 (ARG) the cells are dead! Try again, maybe with less ligation mix.
ARG can be due to different factors:
Too much DNA
Too many ions
Bubbles
Liquid on the outside of the cuvette (i.e. the cuvette was not dried properly)
Part 2: Selection of transformants
Material:
antibiotic plates
LB plates
Procedure:
You will be plating 100 ul on each plate (with antibiotic corresponding to the selection marker on the plasmid).
First, decide if you need to make a dilution series which depends on how well transformation goes. If it is high, then without diluting there will be an over-grow of bacteria on the plate and you can’t select a colony from there.
Choose your dilutions, e.g. from 100 to10-7 in order to get an appropriate number of bacteria for plating out. High efficiency transformation: use dilution -5, -6 or -7.
Plate 100 µL from your 10-X</sups> dilutions onto the antibiotic plates.
Negative control: Plate 100 µL from your undiluted bacteria culture (100 dilution) that was not transformed onto the selective plate.
Positive control: Plate 100 µL from your 10-x dilution onto a non-selective plate and spread.
Incubate your plates overnight at 37ºC (depends on the bacteria or experiment).
Place the rest of the LB-media with transformed cells on the bench overnight. If colonies are lacking the next day; plate again.
This project is part of iGEM
Read more on the iGEM Main Page
Sponsors
Thanks to:
How to customize an iGEM wiki?
Learn all the tricks in our wiki guide
Retrieved from " http://2011.igem.org/Team:DTU-Denmark/Protocols "
Recent changes
What links here
Related changes
Special pages
My preferences
Printable version
Permanent link
Privacy policy
Disclaimers
