Team:Gifu/Results
HOME
TEAM
Team Members
Collaborations
PROJECT
Description
Experiments
Protocol
InterLab
Results
Improve
Attributions
Calendar
Reference
PARTS
SAFETY
HUMAN PRACTICES
Human Practices
Education & Engagement
JUDGING FORM
Results
PCR
The result of PCR is shown in Fig.1~Fig.3. All PCR was performed by primers in table 1. All PCR were succeeded. We used PrimeSTAR Max premix for amplification.
Fig.1 PCR of  BBa_K1491033
Fig.2 PCR of  Seq.1
Fig.3 PCR of  Seq.2
Table.1 Primers
Cell-free expression
Here we show the result of expression. (Fig. 4~7) Seq.2 (monomer RFP, mRFP) was expressed well in PURE system and slightly expressed in myTXTL. Seq.1 (polymer RFP, pRFP) has no fluorescence. In myTXTL, T7 RNA polymerase (Takara) was supplemented before the use. For Seq.2, we added 50U. For Seq.1, we prepared two conditions, 50 U and 25 U.
Fig.4 After expression in PURE system (No UV)
P: Positive (DHFR gene), N: Negative (Water), tem.: Template DNA
mRFP tem. and mRFP PCR was slightly turned into red.
Fig.5 After expression in myTXTL (No UV)
P: Positive (eGFP), N: Negative (Water)
Positive (eGFP) shows green color.
Fig.6 Tubes of PURE system under UV
P: Positive (DHFR gene), N: Negative (Water), tem.: Template DNA
mRFP shows the clear fluorescence. mRFP tem. and mRFP PCR shows the same result. mRFP shown in the figure is mRFP tem.
Fig.7 Tubes of myTXTL under UV
P: Positive (eGFP), N: Negative (Water)
Positive (eGFP) turned into green.
SDSPAGE
We performed SDSPAGE to confirm whether pRFP was expressed or not. The result is shown below.
Fig.8 SDSPAGE
As a result of SDS-PAGE, both myTXTL and PURE system shows the intense expression of the protein. In PURE system, we could see the monomer RFP. With the eye, a slight expression of mRFP in myTXTL was confirmed but not in SDSPAGE. In both cell-free system, the long-chain protein and mRFP was not confirmed with Seq.1.
Discussion
3-dimensional structure of the enhancer, stem-loop structure may inhibit the function of ribozymes. Ribozymes need to maintain certain structures to catalyze the reaction. Chemically, it is possible that the enhancer changes the ribozymes' conformation. We need to make the same plasmid without the enhancer so that we can confirm whether the stem-loop structure effects on the ribozymes. We should extract circular RNA from the tube confirm if there is circular RNA.
Furthermore, in myTXTL, even we confirmed the large expression of eGFP, we could not see enough expression of RFP. This may be because our T7 RNA polymerase does not fit in myTXTL crude extract. We should change it into T7 RNA polymerase from Thermo at a conc. of 2 U/Î¼L. This is functional and confirmed by the company.
This time we did not use TEV protease site. Before performing the cut, we should consider whether no indispensable components for transcription and translation do not have the site. In addition, TEV protease can work in low temperature. To make monomer RFP, incubation temperature can be at low temperature. With this tip, the folding rate of the expressed protein would be much slower and the cutting rate may be effective.
Future work
Our final goal of the project is mass-production of cell-free system. Cell-free system is free system (Open system), so we can add any components from outside. It means "If amino acids may be running out, we can add them". If the system can be developed to industrial level, our idea is really interesting.
This year we focus on making monomer RFP from circular RNA. Circular RNA is a definitely amazing technique for cell-free system to cover its low ability of production of protein. We inserted TEV protease upstream the coding region. But when it comes to an operon, for example E.coli operon, they have a functional protein from one long mRNA. Technically, all protein of such mRNA has stop codons. Between each protein, they have spacer alignment. With the spacer region, even Ribosomes recognize the stop codons, it keeps binding to mRNA. This mechanism can be applied to circular RNA.
In addition to this, we tried to use PIE method to make circular RNA. However, the efficiency is estimated at 2.5% of all transcripted RNA. In cell-free system, plasmids can not be replicated and the efficiency of constructing the circular RNA is possibly quite small. Thus next year we would like to use in vitro circularization with RNA ligase. in vitro circularization has really high efficiency and we can use the RNA to quantify the speed of expression.
In conclusion, our future work is...
Focusing on the specific spacer, we would like to make it possible to have eternal production of protein, literally, iVEPOP in the near future.
,
