<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_Gifu_Description skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><DIV id="top_title"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:Gifu/Description</SPAN></H1></DIV><DIV id="HQ_page"><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><DIV class="igem_2018_team_menu displaying_menu"><A href="https://2018.igem.org/Team:Gifu"><DIV class="menu_item direct_link">
				HOME
			</DIV></A><DIV class="menu_item">
			TEAM
		</DIV><DIV class="submenu"><A href="https://2018.igem.org/Team:Gifu/Team"><DIV class="submenu_item">
					Team Members
				</DIV></A><A href="https://2018.igem.org/Team:Gifu/Collaborations"><DIV class="submenu_item">
					Collaborations
				</DIV></A></DIV><DIV class="menu_item">
			PROJECT
		</DIV><DIV class="submenu"><A href="https://2018.igem.org/Team:Gifu/Description"><DIV class="submenu_item">
					Description
				</DIV></A><A href="https://2018.igem.org/Team:Gifu/Experiments"><DIV class="submenu_item">
					Experiments
				</DIV></A><A href="https://2018.igem.org/Team:Gifu/Protocol"><DIV class="submenu_item">
					Protocol
				</DIV></A><A href="https://2018.igem.org/Team:Gifu/InterLab"><DIV class="submenu_item">
					InterLab
				</DIV></A><A href="https://2018.igem.org/Team:Gifu/Results"><DIV class="submenu_item">
					Results
				</DIV></A><A href="https://2018.igem.org/Team:Gifu/Improve"><DIV class="submenu_item">
					Improve
				</DIV></A><A href="https://2018.igem.org/Team:Gifu/Attributions"><DIV class="submenu_item">
					Attributions 
				</DIV></A><A href="https://2018.igem.org/Team:Gifu/Calendar"><DIV class="submenu_item">
                                        Calendar
                                </DIV></A><A href="https://2018.igem.org/Team:Gifu/Reference"><DIV class="submenu_item">
                                        Reference
                                </DIV></A></DIV><A href="https://2018.igem.org/Team:Gifu/Parts"><DIV class="menu_item direct_link">
				PARTS
			</DIV></A><A href="https://2018.igem.org/Team:Gifu/Safety"><DIV class="menu_item direct_link">
				SAFETY
			</DIV></A><DIV class="menu_item"> 
			HUMAN PRACTICES 
		</DIV><DIV class="submenu"><A href="https://2018.igem.org/Team:Gifu/Human_Practices"><DIV class="submenu_item">
					Human Practices
				</DIV></A><A href="https://2018.igem.org/Team:Gifu/Public_Engagement"><DIV class="submenu_item_ee">
					Education &amp; Engagement
				</DIV></A></DIV><A href="https://igem.org/2018_Judging_Form?team=Gifu"><DIV class="menu_item direct_link">
				JUDGING FORM
			</DIV></A></DIV><DIV class="igem_2018_team_content"><DIV class="igem_2018_team_column_wrapper"><DIV class="column full_size"><H1>Project description</H1><P>The largest benefit of the production of protein in cell-free system is that purifying produced protein is much easier than protein produced <I>in vivo</I> such as in <I>E.coli</I>. There are many applications of this technology, especially for the medical field. Also, it would be possible to use generate some types of enzymes for food procession. Under Japanese law on biodiversity, companies cannot use transformed bacteria and any other living things except for 8 kinds for any purposes. Thus, cell-free system can be a breakthrough technique for our country to develop food culture and industries. However, the production efficiency of cell-free system is not so high compared with that of <I>E.coli</I>. In the present situation, the technology is completely difficult to apply to the industrial level. What we are eager to achieve is making it possible to use circular RNA to generate functional protein in cell-free system and creating a platform of the technology. We have two goals this year. First, we clarify whether circular RNA can be generated in cell-free system using PIE method. In 2015, our team expressed the long-chain protein in <I>E.coli</I> and confirmed the technique is workable <I>in vitro</I>.  However, no one tries to use the technique in cell-free system. Thus our team used the technique in the system for the first time in the world. Second, we tried to create the appropriate linker with TEV protease cutting site. In 2015, we also found the expressed protein was aggregated. In our hypothesis, it is almost impossible to generate the functional protein from circular RNA. TEV protease is derived from a kind of viruses and has a specific cutting site. With the enzyme, we will cut the long-chain RFP into a single unit before the protein is folded firmly. </P><P>In 2015 we used lac promoter. This time we changed the promoter into T7 promoter because the T7 promoter is only a workable promoter sequence in cell-free system. Instead of <I>E.coli</I> K12 BL21, we will use PURE system and myTXTL as the cell-free system. This year is literally the first year for us to use cell-free system. Thus our goal is producing RFP for creating a platform for future work.</P><H1>Self-Splicing</H1><P>The group I intron in <I>td</I> gene of T4 phage has self-splicing mechanism. The self-splicing is a mechanism that circularizes the intron and connects exons. This is catalyzed by several base sequences of the ends of the introns as a ribozyme. We permuted exons and introns with the mechanism and attempted an exon circularization. Using this logic, we have already constructed mRNA circularization devices in 2014. </P><P>Here we explain the circularization mechanism of group I intron with <I>td</I> gene of T4 phage as an example. <I>Td</I> gene consists of an upstream exon, an upstream intron, an ORF, a downstream intron and a downstream exon.  This mechanism is divided into 3 steps. As the first step, a nucleophilic attack by a guanosine separates the upstream exon from the upstream intron and then the guanosine bonds to the 5’ end of the upstream intron. 
  As the second step, the downstream exon is separated from the downstream intron by a nucleophilic attack. The nucleophilic attack takes place by a hydroxy group at the 3’ end of the upstream exon. (Figure 1)</P><FIGURE><CENTER><FIELDSET><LEGEND><B>Fig.1 Self-splicing in T4 phage: the first and second step (Blue: intron, Orange: exon)</B></LEGEND></FIELDSET></CENTER></FIGURE><P>As the third step, the upstream intron bonds to the downstream intron by an attack on an adenine of the upstream intron. The attack takes place by a hydroxyl group of an end of the downstream intron. And then a circular intron is formed.(Figure 2)</P><FIGURE><CENTER><FIELDSET><LEGEND><B>Fig.2 Self-splicing in T4 phage: the third step (Blue: intron, Orange: exon)</B></LEGEND></FIELDSET></CENTER></FIGURE><H1>The permuted intron-exon method : PIE Method</H1><P>We can use the group I intron self-splicing mechanism in <I>td</I> gene of T4 phage to circularize mRNA. The group I intron self-splicing is a mechanism that circularizes an intron and connects exons. It occurs after transcription. The self-splicing is catalyzed by several base sequences of the ends of introns as a ribozyme. We permuted exons and introns with the mechanism and performed an exon circularization.</P><FIGURE><CENTER><FIELDSET><LEGEND><B>Fig.3 Method of the synthesis of the long chain protein </B></LEGEND></FIELDSET></CENTER></FIGURE><P>Two exons are connected with each other in the circularization system; Furthermore, an exon can theoretically be circularized by the system. (Fig.4) We have changed the order of exon and intron. Because of this change, mRNA from exon can be circularized. The intron part works like ribozymes.</P><FIGURE><CENTER><FIELDSET><LEGEND><B>Fig.4 An idea of mRNA circularization (Blue: intron, Orange: exon)</B></LEGEND></FIELDSET></CENTER></FIGURE><P>Based on some paper, Mr. Hasegawa, one of our advisors, made the figure below. According to this figure, we can see which part of intron the self-splicing starts. The total number of bases of circular RNA must be multiple of 3. For the production of the long-chain protein, circular RNA requires SD sequence (RBS) and coding. After one lap, SD sequence is also translated into amino acids sequence. </P><FIGURE><CENTER><FIELDSET><LEGEND><B>Fig.5 Self-splicing method</B></LEGEND></FIELDSET></CENTER></FIGURE></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></BODY></HTML>