<HTML xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en" dir="ltr">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki  ltr ns-0 ns-subject page-Team_Toronto_Modeling"><DIV id="globalWrapper"><DIV id="top-section"><DIV id="p-logo"><A href="/Main_Page" title="Main Page">&quot;
	    </A></DIV><DIV id="menubar" class="left-menu noprint"><UL><LI class="selected"><A href="/Team:Toronto/Modeling">Page               </A></LI><LI class="new"><A href="/wiki/index.php?title=Talk:Team:Toronto/Modeling&amp;action=edit&amp;redlink=1">Discussion               </A></LI><LI><A href="/wiki/index.php?title=Team:Toronto/Modeling&amp;action=edit">View source               </A></LI><LI><A href="/wiki/index.php?title=Team:Toronto/Modeling&amp;action=history">History               </A></LI><LI style="color:#808080;cursor:default">teams</LI></UL></DIV><DIV class="right-menu noprint" id="menubar"><UL><LI id="pt-login"><A href="/wiki/index.php?title=Special:UserLogin&amp;returnto=Team:Toronto/Modeling" title="You are encouraged to log in; however, it is not mandatory [o]" accesskey="o">Log in</A></LI></UL></DIV><DIV id="search-controls" class="noprint"><FORM action="/Special:Search" id="searchform"> </FORM></DIV></DIV><DIV id="content"><H1 class="firstHeading">Team:Toronto/Modeling</H1><DIV id="bodyContent"><H3 id="siteSub" class="noprint">From 2010.igem.org</H3><TABLE style="color:black;background-color:#FFFFFF;" cellpadding="2" cellspacing="0" border="1" width="100%" align="center"><TBODY><TR><TH align="center"><A href="/Team:Toronto" title="Team:Toronto">Home</A></TH><TH align="center"><A href="/Team:Toronto/Project" title="Team:Toronto/Project">Project</A></TH><TH align="center"><A href="/Team:Toronto/Design" title="Team:Toronto/Design">Design</A></TH><TH align="center"><A href="/Team:Toronto/Protocols" title="Team:Toronto/Protocols">Protocols</A></TH><TH align="center"><A href="/Team:Toronto/Notebook" title="Team:Toronto/Notebook">Notebook</A></TH><TH align="center"><A href="/Team:Toronto/Results" title="Team:Toronto/Results">Results</A></TH><TH align="center"><A href="/Team:Toronto/Parts" title="Team:Toronto/Parts">Parts Submitted to the Registry</A></TH><TH align="center"><STRONG class="selflink">Modeling</STRONG></TH><TH align="center"><A href="/Team:Toronto/Software" title="Team:Toronto/Software">Software Used</A></TH><TH align="center"><A href="/Team:Toronto/Human_Practices" title="Team:Toronto/Human Practices">Human Practices</A></TH><TH align="center"><A href="/Team:Toronto/Safety" title="Team:Toronto/Safety">Safety</A></TH><TH align="center"><A href="/Team:Toronto/Team" title="Team:Toronto/Team">Team</A></TH><TH align="center"><A href="https://igem.org/Team.cgi?year=2010&amp;team_name=Toronto" class="external text" rel="nofollow">Official Team Profile</A></TH></TR></TBODY></TABLE><TABLE id="toc" class="toc"><TBODY><TR><TD><DIV id="toctitle"><H2>Contents</H2></DIV><UL><LI class="toclevel-1"><A href="#Modeling"><SPAN class="tocnumber">1</SPAN><SPAN class="toctext">Modeling</SPAN></A></LI><LI class="toclevel-2"><A href="#In_silico_prediction_of_metabolic_response_to_catechol_degradation_in_E._coli"><SPAN class="tocnumber">1.1</SPAN><SPAN class="toctext">In silico prediction of metabolic response to catechol degradation in E. coli</SPAN></A></LI><LI class="toclevel-3"><A href="#E._coli_growth_as_a_function_of_glucose_and_catechol_uptake"><SPAN class="tocnumber">1.1.1</SPAN><SPAN class="toctext">E. coli growth as a function of glucose and catechol uptake</SPAN></A></LI><LI class="toclevel-3"><A href="#Metabolic_reaction_flux_variability_in_different_growth_conditions"><SPAN class="tocnumber">1.1.2</SPAN><SPAN class="toctext">Metabolic reaction flux variability in different growth conditions</SPAN></A></LI><LI class="toclevel-3"><A href="#Comparison_of_metabolic_flux_distribution_between_WT_and_OPT_growth_combinations"><SPAN class="tocnumber">1.1.3</SPAN><SPAN class="toctext">Comparison of metabolic flux distribution between WT and OPT growth combinations</SPAN></A></LI><LI class="toclevel-3"><A href="#Oxygen_limitation_and_increasing_catechol_uptake"><SPAN class="tocnumber">1.1.4</SPAN><SPAN class="toctext">Oxygen limitation and increasing catechol uptake</SPAN></A></LI></UL><LI class="toclevel-2"><A href="#Molecular_Simulations"><SPAN class="tocnumber">1.2</SPAN><SPAN class="toctext">Molecular Simulations</SPAN></A></LI><LI class="toclevel-3"><A href="#Results"><SPAN class="tocnumber">1.2.1</SPAN><SPAN class="toctext">Results</SPAN></A></LI><LI class="toclevel-3"><A href="#Future_Directions"><SPAN class="tocnumber">1.2.2</SPAN><SPAN class="toctext">Future Directions</SPAN></A></LI></TD></TR></TBODY></TABLE><H1><SPAN class="mw-headline" id="Modeling"><B>Modeling</B></SPAN></H1><H2><SPAN class="mw-headline" id="In_silico_prediction_of_metabolic_response_to_catechol_degradation_in_E._coli"><B>In silico prediction of metabolic response to catechol degradation in <I>E. coli</I></B></SPAN></H2><P>In the following section we will investigate in silico predictions of metabolic flux distribution in our catechol degrading <I>E. coli</I> strain through Flux Balance Analysis (FBA).  The analysis was performed using associated functions in the COBRA Toolbox.  A previously developed genome-scale reconstruction of <I>E. coli</I> (iJR904) was altered to include a catechol transport reaction and a reaction representing the breakdown of catechol via the pathway specified in our design.  
</P><P>The catechol pathway that is inserted into <I>E. coli</I> is shown below. In the model, the catechol degredation pathway was represented with the grouped reaction:
</P><P>Catechol + O2 + H2O + CoA --&gt; Acetyl-CoA + Succinate
</P><P>Since the enzymes EC: 1.13.11.1 and EC: 2.8.3.6 very likely catalyze irreversible reactions, the entire process was considered to be irreversible.  
</P><H3><SPAN class="mw-headline" id="E._coli_growth_as_a_function_of_glucose_and_catechol_uptake"><B><I>E. coli</I> growth as a function of glucose and catechol uptake</B></SPAN></H3><P>The inserted degradation pathway breaks down catechol into acetyl-CoA, which plays a central role in biochemical networks by serving as a connecting hub for many metabolic pathways.  This enables the native organism, <I>Pseudomonas</I> putida, to use catechol as an alternate energy source to a certain extent and may also result in similar capability for our engineered <I>E. coli</I> strain.  It is hypothesized that the combination of glucose and catechol that is taken up by the engineered strain may have a great affect on the flux distribution throughout the remaining metabolic network.  Figure 1 shows predicted <I>E. coli</I> growth at various combinations of glucose and catechol uptake.  Though experimentally determined transport rates of these energy sources are not yet available, we will do a preliminary analysis of the response to catechol uptake by investigating different points on the predicted metabolic landscape.  
</P><BLOCKQUOTE><B>Figure 1.</B>  Predicted growth of <I>E. coli</I> at various uptake rates of glucose and catechol.  The points specified on the surface are considered further in simulations.  Points are named according to the uptake rate of glucose and catechol (eg. Glc6/cat0 corresponds to a glucose uptake of 6 mmol/gDW/h and catechol uptake of zero).  Glc6/cat0 corresponds to a “wildtype” (WT) condition, while glc6/cat2.52 corresponds to the “optimal” (OPT) ratio predicted for by FBA to maximize biomass production.  Glc0/cat6 and glc6/cat20 represent additional hypothetical situtions where catechol replaces glucose as an energy source, and where both are used at their maximum rates, respectively.</BLOCKQUOTE><P>The predicted optimal uptake of catechol through FBA of the <I>E. coli</I> metabolic model is 2.52 mmol/gDW/h, resulting in a growth rate of 0.744 h-1.  This is a prediction of the optimal catechol input into the metabolic system in order to produce the maximum amount of biomass components, taking into account the steady-state balancing of all associated metabolic compounds and cofactors.  Though prediction of metabolic flux and microbial growth rate has been shown to be generally accurate through this method, experimental usage of catechol may be significantly higher or lower than the purely stoichiometric prediction.  This is because the input of catechol into the metabolic network <I>in vivo</I> may trigger metabolic regulation at the expense of the assumed cellular objective (biomass production).  Thus, the rates of catechol and glucose uptake are good candidates to be experimentally measured, and then set as constraints in model simulations.  Subsequently, FBA simulations can be used to study the remaining associated metabolic fluxes in our <I>E. coli</I> strain through stoichiometric balancing.   
</P><P>Furthermore, even at a given combination of glucose and catechol uptake rate, there is a degree of variability associated with the in silico prediction of internal metabolic fluxes.  FBA solutions are not unique, meaning that some reactions in the network can mathematically take more than one flux value for any given value of the network objective.  Thus, it will likely be useful to get additional information about in vivo metabolic activity through the measurement of additional transport fluxes and/or upregulation of enzymes through transcriptomics procedures.  This will serve to shrink the mathematical solution space (through the formation of additional constraints in FBA simulations) and narrow down the possible flux distributions towards those that are actually adapted by the <I>E. coli</I> in vivo.  However, in this study we will consider the changes in predicted metabolic reaction variability in order to gain insight into the metabolic response to different combinations of glucose and catechol uptake.  Specifically, we will investigate network variability at the four points indicated on Figure 1, each of which represent significantly different combinations of glucose and catechol uptake.  
</P><H3><SPAN class="mw-headline" id="Metabolic_reaction_flux_variability_in_different_growth_conditions"><B>Metabolic reaction flux variability in different growth conditions</B></SPAN></H3><P>Network reactions were classified into three categories based on their predicted possible flux range at optimal <I>E. coli</I> growth (Figure 2A); constant reactions (reaction with flux value greater than 1e-5, and that do not have range of variability greater than 1e-5), varied reactions (reactions with a flux value greater than 1e-5, and that have a variability range greater than 1e-5), and zero reactions (reactions with a flux value less than 1e-5).  In physiological terms, constant reactions represent areas of the metabolic network that must be active for optimal growth, varied reactions represent areas of the network that have flexibility in their activity, and zero reactions represent areas of the network that are essentially “turned off” for optimal growth.    
</P><BLOCKQUOTE><B>Figure 2.  A)</B> Reaction variability in <I>E. coli</I> at four different combinations of glucose and catechol uptake. <B>B)</B>  Comparison of zero reactions across all four simulated growth combinations.</BLOCKQUOTE><P>Comparison of the number of network reactions that fall into the three variability categories in different conditions reveals some trends in response to catechol uptake.  For one, we notice that all three cases that have at least some catechol transport into <I>E. coli</I> (OPT, glc0/cat6, and glc6/cat20), have less zero reactions than the WT optimal flux distribution (Figure 2A).  Furthermore, a four-way Venn diagram of the zero reactions in each case reveals that a large number are common to all cases (471) and a significant portion (82) are zero reactions in the WT case, but are active in all other cases with catechol uptake (Figure 2B).  The extreme case of glc6/cat20 results in the greatest number of unique zero reactions (19), but this is generally not observed among the cases with catechol uptake.  Taken together these results indicate that the transport of catechol into <I>E. coli</I> activates portions of the metabolic network that are predicted to have zero flux during growth on glucose.  However, the set of reactions that is activated does not show significant change at different combinations of glucose and catechol.  
</P><H3><SPAN class="mw-headline" id="Comparison_of_metabolic_flux_distribution_between_WT_and_OPT_growth_combinations"><B>Comparison of metabolic flux distribution between WT and OPT growth combinations</B></SPAN></H3><P>Though further analysis of the reaction variability data could be carried out for each of the hypothesized growth scenarios, here we will take a more detailed look at predicted change in metabolic reaction flux in the WT and OPT cases.  This represents the change in metabolic reaction activity when <I>E. coli</I> is growing on a combination of glucose with slight catechol supplementation, as compared to E. coli growih using solely glucose.  It was found that many of the reactions distributed into the three variability categories in the previous section had extremely minute fluxes.  Thus, in order to highlight only significant changes in the metabolic network as a result of catechol uptake, we changed our reaction classification definitions to consider a higher tolerance.  
</P><P>All reactions that have a predicted change in flux between the WT and OPT conditions are classified according to their change in flux variability status in Figure 3.  Many of the reactions go from one constant value to another (247/271).  Of the the remaining changing reactions, more are predicted to go from zero to either a constant or varied flux (15/271) as compared to a constant or varied flux to zero (7/271), which indicates that overall there is more activation of network reaction in the OPT case.  The magnitude of flux change across the metabolic network is mapped in Figure 4.  Most reactions have moderate change in flux value (yellow nodes), however a small percentage of reactions have a large change (green and red nodes).  As a pathway, the reactions of the TCA cycle have a greater flux in the OPT case as a result of the increased presence of acetyl-CoA from catechol degradation.  Reactions with the greatest positive change in flux and negative change in flux are listed in Table 1.  However it important to note that the distinction between positive/negative changes in flux does not necessarily correlate to increased/decreased metabolic flux.  Reactions that adopt a negative flux in the WT distribution (such as reversible reactions and transport reactions) and have a larger magnitude negative flux in the OPT distribution will be classified a negative change even though the flux through those metabolic processes has increased.  For example, the oxygen transport reaction EX_o2(e) has flux of -12.57 in the WT distribution, and a flux of -20 in the OPT distribution.  This indicates that a greater amount of oxygen is imported in the OPT case even though the reaction has a negative flux change. 
</P><BLOCKQUOTE><B>Figure 3.</B>  Breakdown of reactions predicted to have a change in flux in the WT and OPT optimal flux distributions.  Constant reactions are those have a flux variability less than 1e-2, varied reactions that have a range of fluxes greater than 1e-2, and zero reactions are those that have fluxes less than 1e-3.  Groups are named according to their status in the WT and OPT flux predictions, respectively.  For example, the &quot;zero – constant&quot; group represents reactions that are zero in the WT distribution and constant in the OPT distribution.</BLOCKQUOTE><BLOCKQUOTE><B>Figure 4.</B>  Flux change between the WT and OPT optimal distributions across the <I>E. coli</I> metabolic network.  Reactions are represented as nodes and grouped according to their defined pathway in the iJR904 model.  The change for variable reactions was calculated by considering the average point within its flux range.  Reactions with a significant flux change are shown in red and green while slight changes are shown in yellow.  Grey nodes represent reactions without a change in predicted metabolic flux.  Flux change and pathway for each reaction node can be obtained from the following <A href="/wiki/images/2/20/Utoronto_cytoscape.zip" class="internal" title="Utoronto cytoscape.zip"> cytoscape file‎</A>.</BLOCKQUOTE><BLOCKQUOTE><B>Table 1.</B> Top five reactions with the greatest positive and negative flux change in the WT and OPT optimal distributions.
<TABLE border="1" cellspacing="0" cellpadding="2"><TBODY><TR><TH scope="col"> RID
</TH><TH scope="col"> Reaction
</TH><TH scope="col"> Gene
</TH><TH scope="col"> WT flux
</TH><TH scope="col"> OPT flux
</TH><TH scope="col"> Difference
</TH></TR><TR><TH scope="row"> ATPS4r
</TH><TD> adp[c] + 4 h[e] + pi[c] &lt;=&gt; atp[c] + h2o[c] + 3 h[c]
</TD><TD> ( b3736  and  b3737  and  b3738 )  and  ( b3731  and  b3732  and  b3733  and  b3734  and  b3735 )  and  b3739
</TD><TD> 33.712
</TD><TD> 45.417
</TD><TD> 11.705
</TD></TR><TR><TH scope="row"> CYTBO3
</TH><TD> 2.5 h[c] + 0.5 o2[c] + q8h2[c]  -&gt;  h2o[c] + 2.5 h[e] + q8[c]
</TD><TD> ( b0429  and  b0430  and  b0431  and  b0432 )
</TD><TD> 25.128
</TD><TD> 34.95
</TD><TD> 9.822
</TD></TR><TR><TH scope="row"> EX_co2
</TH><TD> co2[e]  &lt;=&gt;</TD><TD> NA
</TD><TD> 13.649
</TD><TD> 20.217
</TD><TD> 6.568
</TD></TR><TR><TH scope="row"> NADH6
</TH><TD> 4.5 h[c] + nadh[c] + q8[c]  -&gt; 3.5 h[e] + nad[c] + q8h2[c]
</TD><TD> ( b2276  and  b2277  and  b2278  and  b2279  and  b2280  and  b2281  and  b2282  and  b2283  and  b2284  and  b2285  and  b2286  and  b2287  and  b2288 )
</TD><TD> 22.014
</TD><TD> 27.619
</TD><TD> 5.605
</TD></TR><TR><TH scope="row"> FUM
</TH><TD> fum[c] + h2o[c] &lt;=&gt; mal-L[c]
</TD><TD> ( b1612  or  b4122  or  b1611 )
</TD><TD> 3.283
</TD><TD> 7.833
</TD><TD> 4.55
</TD></TR><TR><TH scope="row"> ENO
</TH><TD> 2pg[c]  &lt;=&gt; h2o[c] + pep[c]
</TD><TD> b2779
</TD><TD> 8.198
</TD><TD> 7.273
</TD><TD> -0.925
</TD></TR><TR><TH scope="row"> GLUDy
</TH><TD> glu-L[c] + h2o[c] + nadp[c]  &lt;=&gt; akg[c] + h[c] + nadph[c] + nh4[c]
</TD><TD> b1761
</TD><TD> -4.487
</TD><TD> -6.195
</TD><TD> -1.708
</TD></TR><TR><TH scope="row"> EX_nh4
</TH><TD> nh4[e]  &lt;=&gt;</TD><TD> NA
</TD><TD> -5.76
</TD><TD> -7.953
</TD><TD> -2.193
</TD></TR><TR><TH scope="row"> EX_h
</TH><TD> h[e]  &lt;=&gt;</TD><TD> NA
</TD><TD> 4.928
</TD><TD> 1.812
</TD><TD> -3.116
</TD></TR><TR><TH scope="row"> EX_o2
</TH><TD> o2[e]  &lt;=&gt;</TD><TD> NA
</TD><TD> -12.568
</TD><TD> -20
</TD><TD> -7.432
</TD></TR></TBODY></TABLE></BLOCKQUOTE><H3><SPAN class="mw-headline" id="Oxygen_limitation_and_increasing_catechol_uptake"><B>Oxygen limitation and increasing catechol uptake</B></SPAN></H3><P>
It is important to note that the uptake of catechol in our engineered strain of E. coli is predicted to be under oxygen limitation.  For example, the OPT case of catechol degradation requires oxygen import at 20 mmol/gDW/h, which is the imposed maximum rate of oxygen transport.  This value of oxygen transport limitation has been derived previously, and is an approximation of the diffusion rate based on the maximum concentration gradient of oxygen that can be achieved between the extracellular solution and cytosol.  In vitro, sparging the experimental solution with oxygen gas may be required to ensure maximum oxygen available for transport to <I>E. coli</I>.  Furthermore, it may be possible to engineer the <I>E. coli</I> strain to increase catechol degradation through the incorporation of gene deletions.  Due to the oxygen limitation observed in the OPT case, a genome-wide scan of single gene deletions optimizing for the minimization of metabolic adjustment (MOMA) did not turn up any that were predicted to increase catechol uptake (data not shown).  However, algorithms to identify multiple gene deletions have been developed that can be applied to this problem.  For example, the OptKnock (Burgard et al, 2003) procedure introduces a bilevel optimization approach where a cellular objective (e.g. biomass production) and a bioengineering objective (e.g. catechol degradation) can be optimized simultaneously.  This method aims to identify gene deletions that will change the structure of the metabolic network so that the bioengineering objective becomes coupled to biomass production for the growing organism.  As an extension of this work, OptStrain (Burgard et al, 2004) considers the addition of reaction pathways to the metabolic system that would increase the bioengineering objective.  These procedures may be applied in future work to enhance the catechol degradation capability of the engineered <I>E. coli</I> strain.
</P><H2><SPAN class="mw-headline" id="Molecular_Simulations"><B>Molecular Simulations</B></SPAN></H2><P>In order to optimize the catechol degradation pathway such that there are proportional amounts of each of the multimeric enzyme complexes, monomer expression levels need to be adjusted appropriately.  This will prevent an over-abundance or deficit of a particular enzyme.  To determine the appropriate level of monomer expression for each enzyme, the multimerization equilibrium constant needs be estimated.
</P><P>
The equilibrium constant can be estimated using the formula: 
</P><P>
where the standard Gibbs free energy change of the reaction  . 
</P><P>
If we assume entropy change is negligible, the formula then becomes: 
</P><P>
The change in enthalpy for a reaction can be calculated using molecular simulations.

</P><H3><SPAN class="mw-headline" id="Results"><B>Results</B></SPAN></H3><P>Catechol 1,2-Dioxygenase (Dimer)
</P><P>&lt;p&gt;
The x-ray crystallography file for catechol 1,2-dioxygenase (2azq) only contains the monomer unit (Fig. 5).  The dimer configuration was obtained using ZDOCK in Accelrys Discovery Studio (Fig. 6).  The enthalpy change between the monomer and the dimer configuration was calculated using an augmented MM3 force field.  The calculated enthalpy for a monomer unit was -39,326 kJ/mol.  For a dimerized unit the enthalpy was -110,569 kJ/mol.  Hence, the enthalpy change for the entire reaction is -31,917 kJ/mol.  This yields K = 390,780 mol^-1.
</P><BLOCKQUOTE>
Fig. 5. Catechol 1,2-dioxygenase monomer unit.
</BLOCKQUOTE><BLOCKQUOTE>
Fig. 6.  Catechol 1,2-dioxygenase dimer as calculated by ZDOCK.
</BLOCKQUOTE><H3><SPAN class="mw-headline" id="Future_Directions"><B>Future Directions</B></SPAN></H3><P>For the design of the linked enzymes, determining the equilibrium constants will be important to determine the proportion of properly assembled structures to those that have multiple linkers tangled with each other.  The equilibrium constant will also be needed to determine the proportion of linked enzymes to ones that are not linked.
We will continue to characterize the enthalpies associated with the remaining enzymes using the MM3 force field, along with other force fields such as CHARMM for comparison.  For enzymes that do not have crystallography files associated with them, we will attempt to approximate their structures using homology modeling and other protein structure prediction methods.
</P><DIV class="printfooter">
Retrieved from &quot;<A href="http://2010.igem.org/Team:Toronto/Modeling">http://2010.igem.org/Team:Toronto/Modeling</A>&quot;</DIV></DIV></DIV><DIV id="footer-box" class="noprint"><DIV id="footer"><UL id="f-list"><LI id="t-recentchanges"><A href="/Special:RecentChanges" title="Recent changes">Recent changes</A></LI><LI id="t-whatlinkshere"><A href="/Special:WhatLinksHere/Team:Toronto/Modeling" title="List of all wiki pages that link here [j]" accesskey="j">What links here</A></LI><LI id="t-recentchangeslinked"><A href="/Special:RecentChangesLinked/Team:Toronto/Modeling" title="Recent changes in pages linked from this page [k]" accesskey="k">Related changes</A></LI><LI id="t-specialpages"><A href="/Special:SpecialPages" title="List of all special pages [q]" accesskey="q">Special pages</A></LI><LI><A href="/Special:Preferences">My preferences</A></LI></UL></DIV><DIV id="footer"><UL id="f-list"><LI id="t-print"><A href="/wiki/index.php?title=Team:Toronto/Modeling&amp;printable=yes" title="Printable version of this page [p]" accesskey="p">Printable version</A></LI><LI id="t-permalink"><A href="/wiki/index.php?title=Team:Toronto/Modeling&amp;oldid=209117" title="Permanent link to this revision of the page">Permanent link</A></LI><LI id="privacy"><A href="/2010.igem.org:Privacy_policy" title="2010.igem.org:Privacy policy">Privacy policy</A></LI><LI id="disclaimer"><A href="/2010.igem.org:General_disclaimer" title="2010.igem.org:General disclaimer">Disclaimers</A></LI></UL></DIV></DIV></DIV></BODY></HTML>