<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_UNebraska-Lincoln_modeling skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><DIV id="top_title"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:UNebraska-Lincoln/modeling</SPAN></H1></DIV><DIV id="HQ_page"><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><DIV id="page-wrapper" style="background-color: #9ECBDF;"><DIV id="NavBar"><DIV class="inner"><UL id="sdt_menu" class="sdt_menu"><LI style="background-color:#4c5c96;"><A href="https://2016.igem.org/Team:UNebraska-Lincoln"><SPAN class="sdt_wrap"><SPAN class="sdt_link">Home</SPAN></SPAN></A></LI><LI style="background-color:#4c5c96;"><A href="https://2016.igem.org/Team:UNebraska-Lincoln/Team"><SPAN class="sdt_wrap"><SPAN class="sdt_link">Team</SPAN></SPAN></A></LI><LI style="background-color:#4c5c96;"><A href="https://2016.igem.org/Team:UNebraska-Lincoln/The_drawing_board"><SPAN class="sdt_wrap"><SPAN class="sdt_link">Blueprints</SPAN></SPAN></A><DIV class="sdt_box"><A href="https://2016.igem.org/Team:UNebraska-Lincoln/background">Background</A><A href="https://2016.igem.org/Team:UNebraska-Lincoln/modeling">Modeling</A><A href="https://2016.igem.org/Team:UNebraska-Lincoln/design">Design</A></DIV></LI><LI style="background-color:#4c5c96;"><A href="https://2016.igem.org/Team:UNebraska-Lincoln/laboratory"><SPAN class="sdt_wrap"><SPAN class="sdt_link" style="">Laboratory</SPAN></SPAN></A><A href="https://2016.igem.org/Team:UNebraska-Lincoln/experiments">Experiments</A><A href="https://2016.igem.org/Team:UNebraska-Lincoln/lab_manual">Lab Notebook</A><A href="https://2016.igem.org/Team:UNebraska-Lincoln/safety_cases">Safety Cases</A></LI><LI style="background-color:#4c5c96;"><A href="https://2016.igem.org/Team:UNebraska-Lincoln/beyond_the_bench"><SPAN class="sdt_wrap"><SPAN class="sdt_link">Outreach</SPAN></SPAN></A><A href="https://2016.igem.org/Team:UNebraska-Lincoln/human_practices">Human Practices</A><A href="https://2016.igem.org/Team:UNebraska-Lincoln/Integrated_Practices">Integrated Practices</A><A href="https://2016.igem.org/Team:UNebraska-Lincoln/collaboration">Collaboration</A><A href="https://2016.igem.org/Team:UNebraska-Lincoln/industry">Industry</A></LI><LI style="background-color:#4c5c96;"><A href="https://2016.igem.org/Team:UNebraska-Lincoln"><SPAN class="sdt_wrap"><SPAN class="sdt_link">Thanks!</SPAN></SPAN></A><A href="https://2016.igem.org/Team:UNebraska-Lincoln/sponsors">Sponsors</A><A href="https://2016.igem.org/Team:UNebraska-Lincoln/attributions">Attributions</A></LI></UL></DIV></DIV><DIV class="drawing board" style=" background-color: #2e3141; padding-top:50px;"><SECTION><SECTION><DIV style="transform: scale(1);margin-left:150px; margin-right:150px"><H2 class="major">Computational Modeling</H2><H3><FONT color="white"><STRONG>Introduction:</STRONG></FONT></H3><P><FONT color="white"><SPAN style="font-weight: 400;">Our modeling efforts were mainly geared toward developing logic models to obtain general curve trends. The main focus of our project was to reduce nitrate ions to nitrite ions. For safety measures, a kill switch was designed and engineered for our microorganism. </SPAN></FONT></P><P><FONT color="white"><SPAN style="font-weight: 400;">In short, the system works like this: the nitrate ions activate the yeaR promoter and allow for the expression of the serA gene. The expression of serA leads to the synthesis of serine and allows the cell to live. When the cell is healthy, it expresses the napDABC gene, which is a nitrate reductase. Nitrate ions are reduced to nitrite ions and eventually, nitrate concentrations fall below the threshold to activate the yeaR promoter and the cell dies.</SPAN></FONT></P><P><FONT color="white"><SPAN style="font-weight: 400;">To model the logic of this system, iBioSim was primarily used.</SPAN></FONT></P><P><FONT color="white">Figure 1a. This is the model for both the nitrate reduction component and kill switch component of our project. This figure shows the components of the system and shows the relationshis between them. Nitrates regulate the expression of serA and are being reduced by napDABC.</FONT></P><P><FONT color="white">Figure 1b. This figure shows the general trends of the model design shown in figure 1a. Nitrate levels are being reduced in the present of nitrate ions.</FONT></P><P><FONT color="white">Figure 2a. This is model is designed to show the trends of GFP exxpression at varying nitrate concentrations. Three events are set up that change the nitrate levels at different times to test the logic of our kill switch design.</FONT></P><P><FONT color="white">Figure 2b. This simulation shows that expression of a reporter gene is directly related to nitrate concentrations. Nitrate concentrations and reporter gene expression are positively correlated.</FONT></P><P><FONT color="white">Figure 3a. This composite project model design takes into account the repression and induction of the nitrate sensitive promoter, PyeaR. In it's natural state, the yeaR promoter is inhibited by Nar and NsrR, but is induced in high nitrate concentrations.</FONT></P><P><FONT color="white">Figure 3b. This simulation shows the general trend lines of the model shown in figure 3a. This is what is generally expected. After serA is no longer expressed however, the NapDABC should be lowered signifying cell death.</FONT></P><P><FONT color="white"><SPAN style="font-weight: 400;">The following are the initial conditions and sequence of events for Figures 3a and 3b.</SPAN></FONT></P><H4><FONT color="white"><STRONG>Initial Conditions:</STRONG></FONT></H4><UL><FONT color="white"><LI style="font-weight: 400;"><SPAN style="font-weight: 400;"><FONT color="white">High nitrate levels</FONT></SPAN></LI><LI style="font-weight: 400;"><SPAN style="font-weight: 400;"><FONT color="white">Nar &amp; NsrR inhibit pYeaR</FONT></SPAN></LI></FONT></UL><H4><FONT color="white"><STRONG>Sequence of Events:</STRONG></FONT></H4><OL><LI style="font-weight: 400;"><SPAN style="font-weight: 400;"><FONT color="white">Nitric oxide sequesters NsrR</FONT></SPAN></LI><LI style="font-weight: 400;"><SPAN style="font-weight: 400;"><FONT color="white">Nitrite ions sequester Nar</FONT></SPAN></LI><LI style="font-weight: 400;"><SPAN style="font-weight: 400;"><FONT color="white">SerA is expressed</FONT></SPAN></LI><LI style="font-weight: 400;"><SPAN style="font-weight: 400;"><FONT color="white">SerA contributes to the health of the cell</FONT></SPAN></LI><LI style="font-weight: 400;"><SPAN style="font-weight: 400;"><FONT color="white">When the cell is healthy, NapDABC is expressed</FONT></SPAN></LI><LI style="font-weight: 400;"><SPAN style="font-weight: 400;"><FONT color="white">NapDABC reduces nitrate ions to nitrite</FONT></SPAN></LI><LI style="font-weight: 400;"><SPAN style="font-weight: 400;"><FONT color="white">The concentration of nitrate ions is less than Nar and PyeaR is repressed</FONT></SPAN></LI><LI style="font-weight: 400;"><SPAN style="font-weight: 400;"><FONT color="white">SerA is no longer expressed</FONT></SPAN></LI><LI style="font-weight: 400;"><SPAN style="font-weight: 400;"><FONT color="white">The concentration of SerA decreases (it is used by the cell)</FONT></SPAN></LI><LI style="font-weight: 400;"><SPAN style="font-weight: 400;"><FONT color="white">The cells health diminishes and the cell eventually dies</FONT></SPAN></LI><LI style="font-weight: 400;"><SPAN style="font-weight: 400;"><FONT color="white">NapDABC is no longer expressed</FONT></SPAN></LI><LI style="font-weight: 400;"><SPAN style="font-weight: 400;"><FONT color="white">Nitrate ions are no longer being reduced </FONT></SPAN></LI></OL><P><FONT color="white"> </FONT></P><P><FONT color="white"><SPAN style="font-weight: 400;">iBioSim proved effective for outlining the logic of our model and the main trend of its input-output behavior. However, we found some limitations in importing experimental data for curve fitting/parameter estimation. MATLAB SimBiology would be a great alternative worth exploring in the future. We plan to propose to our future iGEM teams to further expand this research to develop more complete and realistic computational models.</SPAN></FONT></P><P><FONT color="white"><SPAN style="font-weight: 400;">C. Myers, N. Barker, K. Jones, H. Kuwahara, C. Madsen, and N. Nguyen, iBioSim: a tool for the analysis and design of genetic circuits, in Bioinformatics, 25(21): 2848-2849, November 1, 2009.</SPAN></FONT></P></DIV></SECTION><SECTION id="footer"><DIV class="inner"><H2 class="major">How did this fit into our project?</H2></DIV></SECTION></SECTION></DIV></DIV></DIV></DIV></DIV></DIV></DIV></BODY></HTML>