"
Page
Discussion
View source
History
teams
Log in
 
Team:Cornell/testing/notebook/wetlab/pre
From 2012.igem.org
Home
Project
Overview
Background
Wet Lab
Dry Lab
Human Practices
Gallery
Safety
Notebook
Overview
Wet Lab
Dry Lab
Strain List
Protocols
Team
Overview
Bios
Profile
Sponsors
Outreach
Overview
Research Presentations
Panel Discussions
Public Awareness
2012: SAFE BET
Team Website
2013: Organofoam
2011: Biofactory
2010: OMG OMVs
2009: Cadmium Sensor
Wet Lab
Bootcamp
June
July
August
September
Wrap Up
Wet Lab Notebook - Bootcamp
Next Month>>
Bootcamp Overview
From June 12th-22nd, the DeLisa lab kindly hosted a synthetic biology ‘bootcamp’ for our team members. Dr. Didi Waraho assisted in the instruction of new team members, while Taylor Stevenson helped more experienced members get acquainted with the Gibson assembly method.
Didi’s group worked on the construction of our arsenic reporter plasmids (now referred to as p14k and p16k). This work involved PCR steps to append non-BioBrick cutsites (AscI and BamHI) to an existing mtrB BioBrick (BBa_K098994) and an arsenic-sensing region (BBa_J33201) in order to introduce modularity for easy promoter switching. By the end of the bootcamp, we confirmed via Sanger sequencing that both versions of our arsenic reporter had been successfully cloned into DH5a, electrocompetent E. coli cells. The next step would be to get the plasmids into our Shewanella strain lacking mtrB on the chromosome (JG700).
Taylor’s group had one goal: To get internal PstI and NotI cutsites out of our 10kb naphthalene-degrading (nah) operon while constructing the final plasmid to be transformed into Shewanella. By using mutagenic PCR primers that would introduce silent mutations to get rid of the non-BioBrick compatible internal cutsites, the group planned on ripping the nah operon apart, so to speak, and putting it back together again and into a pBMT-1 backbone via Gibson assembly. Using the NAH7 plasmid (containing nah operon) fromPseudomonas putida G7 (kindly provided by Dr. Gene Madsen), Taylor’s group completed all PCR steps necessary for Gibson assembly, incubated the PCR products with Gibson master mix, and transformed the Gibson products into DH5a—yielding three transformants. At the end of bootcamp, the group ran supercoiled plasmid (miniprepped from transformants) on a gel—as an initial screen for successful construction of our naphthalene-degrading plasmid. As shown below, the gel showed distinct bands for all three lanes. However, further confirmation was necessary, so the next step was to digest the Gibson products to check for correct fragment lengths, and to submit DNA for Sanger sequencing.
Next Month>>
Retrieved from " http://2012.igem.org/Team:Cornell/testing/notebook/wetlab/pre "
Recent changes
What links here
Related changes
Special pages
My preferences
Printable version
Permanent link
Privacy policy
Disclaimers
