<HTML xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en" dir="ltr">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki  ltr ns-0 ns-subject page-Team_Cornell_notebook_wetlab_august"><DIV id="globalWrapper"><DIV id="top-section"><DIV id="p-logo"><A href="/Main_Page" title="Main Page">&quot;
	    </A></DIV><DIV id="menubar" class="left-menu noprint"><UL><LI class="selected"><A href="/Team:Cornell/notebook/wetlab/august">Page               </A></LI><LI class="new"><A href="/wiki/index.php?title=Talk:Team:Cornell/notebook/wetlab/august&amp;action=edit&amp;redlink=1">Discussion               </A></LI><LI><A href="/wiki/index.php?title=Team:Cornell/notebook/wetlab/august&amp;action=edit">View source               </A></LI><LI><A href="/wiki/index.php?title=Team:Cornell/notebook/wetlab/august&amp;action=history">History               </A></LI><LI style="color:#808080;cursor:default">teams</LI></UL></DIV><DIV class="right-menu noprint" id="menubar"><UL><LI id="pt-login"><A href="/wiki/index.php?title=Special:UserLogin&amp;returnto=Team:Cornell/notebook/wetlab/august" title="You are encouraged to log in; however, it is not mandatory [o]" accesskey="o">Log in</A></LI></UL></DIV><DIV id="search-controls" class="noprint"><FORM action="/Special:Search" id="searchform"> </FORM></DIV></DIV><DIV id="content"><H1 class="firstHeading">Team:Cornell/notebook/wetlab/august</H1><DIV id="bodyContent"><H3 id="siteSub" class="noprint">From 2012.igem.org</H3><DIV class="row header off-canvas"><DIV class="twelve columns centered header no-pad" id="header-wrapper"><UL class="nav-bar"><LI class="pg-home"><A href="https://2012.igem.org/Team:Cornell">Home</A></LI><LI class="has-flyout pg-project pg-project_background pg-project_wetlab pg-project_drylab pg-project_hprac pg-project_gallery pg-project_safety"><A href="https://2012.igem.org/Team:Cornell/project">Project</A></LI><LI class="pg-project"><A href="https://2012.igem.org/Team:Cornell/project">Overview</A></LI><LI class="pg-project_background"><A href="https://2012.igem.org/Team:Cornell/project/background">Background</A></LI><LI class="pg-project_wetlab"><A href="https://2012.igem.org/Team:Cornell/project/wetlab">Wet Lab</A></LI><LI class="pg-project_drylab"><A href="https://2012.igem.org/Team:Cornell/project/drylab">Dry Lab</A></LI><LI class="pg-project_hprac"><A href="https://2012.igem.org/Team:Cornell/project/hprac">Human Practices</A></LI><LI class="pg-project_gallery"><A href="https://2012.igem.org/Team:Cornell/project/gallery">Gallery</A></LI><LI class="pg-project_safety"><A href="https://2012.igem.org/Team:Cornell/project/hprac/safety">Safety</A></LI></UL><LI class="has-flyout pg-notebook pg-notebook_wetlab pg-notebook_drylab pg-notebook_strainlist"><A href="https://2012.igem.org/Team:Cornell/notebook">Notebook</A></LI><LI class="pg-notebook"><A href="https://2012.igem.org/Team:Cornell/notebook">Overview</A></LI><LI class="pg-notebook_wetlab"><A href="https://2012.igem.org/Team:Cornell/notebook/wetlab">Wet Lab</A></LI><LI class="pg-notebook_drylab"><A href="https://2012.igem.org/Team:Cornell/notebook/drylab/june">Dry Lab</A></LI><LI class="pg-notebook_strainlist"><A href="https://2012.igem.org/Team:Cornell/notebook/strain_list" target="_blank">Strain List</A></LI><LI class="pg-protocols"><A href="https://2012.igem.org/Team:Cornell/protocols">Protocols</A></LI><LI class="has-flyout pg-team pg-team_bios pg-team_sponsors"><A href="https://2012.igem.org/Team:Cornell/team">Team</A></LI><LI class="pg-team"><A href="https://2012.igem.org/Team:Cornell/team">Overview</A></LI><LI class="pg-team_bios"><A href="https://2012.igem.org/Team:Cornell/team/bios">Bios</A></LI><LI><A href="https://igem.org/Team.cgi?year=2012&amp;team_name=Cornell" target="_blank">Profile</A></LI><LI class="pg-team_sponsors"><A href="https://2012.igem.org/Team:Cornell/team/sponsors">Sponsors</A></LI><LI class="has-flyout pg-outreach pg-outreach_presentations pg-outreach_panels pg-outreach_awareness"><A href="https://2012.igem.org/Team:Cornell/outreach">Outreach</A></LI><LI class="pg-outreach"><A href="https://2012.igem.org/Team:Cornell/outreach">Overview</A></LI><LI class="pg-outreach_presentations"><A href="https://2012.igem.org/Team:Cornell/outreach/presentations">Research Presentations</A></LI><LI class="pg-outreach_panels"><A href="https://2012.igem.org/Team:Cornell/outreach/panels">Panel Discussions</A></LI><LI class="pg-outreach_awareness"><A href="https://2012.igem.org/Team:Cornell/outreach/awareness">Public Awareness</A></LI><LI class="pg-badge"><A href="https://2013.igem.org/Team:Cornell" style="margin-top:-50px">2012: SAFE BET</A></LI><LI class="pg-badge_home"><A href="http://igem.engineering.cornell.edu">Team Website</A></LI><LI class="pg-badge_2013"><A href="https://2013.igem.org/Team:Cornell">2013: Organofoam</A></LI><LI class="pg-badge_2011"><A href="https://2011.igem.org/Team:Cornell">2011: Biofactory</A></LI><LI class="pg-badge_2010"><A href="https://2010.igem.org/Team:Cornell">2010: OMG OMVs</A></LI><LI class="pg-badge_2009"><A href="https://2009.igem.org/Team:Cornell" style="font-size:20px !important">2009: Cadmium Sensor</A></LI></DIV></DIV><DIV class="row"><DIV class="two columns"><UL class="side-nav ntbk-nav"><LI><H6>Wet Lab</H6></LI><LI><A href="https://2012.igem.org/Team:Cornell/notebook/wetlab">Overview</A></LI><LI><A href="https://2012.igem.org/Team:Cornell/notebook/wetlab/bootcamp">Bootcamp</A></LI><LI><A href="https://2012.igem.org/Team:Cornell/notebook/wetlab/june">June</A></LI><LI><A href="https://2012.igem.org/Team:Cornell/notebook/wetlab/july">July</A></LI><LI class="active"><A href="https://2012.igem.org/Team:Cornell/notebook/wetlab/august">August</A></LI><LI><A href="https://2012.igem.org/Team:Cornell/notebook/wetlab/september">September</A></LI><LI><A href="https://2012.igem.org/Team:Cornell/notebook/wetlab/wrap_up">Wrap Up</A></LI></UL></DIV><DIV class="ten columns team-bios-container"><DL class="tabs pill" style="display:block;float:right;width:390px;margin: -10px -20px 10px 0px;"><DD class="active"><A href="#pillTab1">Weekly Update</A></DD><DD><A href="#pillTab2">Daily Details</A></DD><DD><A href="#pillTab3">Both</A></DD></DL><DIV class="row"><DIV class="twelve columns"><H2 class="centered">Wet Lab - August</H2></DIV></DIV><UL class="tabs-content"><LI class="active" id="pillTab1Tab"><DIV class="row last-ele" style="margin-top:20px;"><DIV class="three columns"><A href="https://2012.igem.org/Team:Cornell/notebook/wetlab/july#pillTab1">&lt;&lt; Previous</A></DIV><DIV class="three columns offset-by-six"><A href="https://2012.igem.org/Team:Cornell/notebook/wetlab/september#pillTab1" style="display:block;float:right;">Next &gt;&gt;</A></DIV></DIV><DIV class="row"><DIV class="nine columns"><H3>August 1st - 4th</H3>
							It was confirmed that RFP was successfully inserted downstream of mtrB in our arsenic reporter parts in Shewanella. Site-directed mutagenesis of the nah operon failed. We are still trying to insert the nah operon into the mobility backbone (OriT) to prepare for conjugation into Shewanella later on. <A href="#" class="technical-desc" for="#technical-desc1" style="display:block;margin-top:20px;">Daily Details</A><DIV class="hide-me panel" style="background:white;margin-top: 20px;" id="technical-desc1"><H6>Daily Details:</H6><B> August 1st, Wednesday </B>
								In the morning, Dylan noted that 5 of the 7 transformations from Tuesday produced plates with colonies. Those that worked were three Anderson series constitutive promoters (with downstream mRFP) in pBBRBB and the versions of our two arsenic reporters with mRFP downstream of mtrB. In the evening, Swati and Dylan prepared overnight cultures from all transformants and JG700, both to set up conjugations to get plasmid into Shewanella and to screen isolated plasmids for successful ligation. We will use these constructs to characterize the activity of the arsenic-sensitive promoter in Shewanella with respect to known constitutive promoter strengths. The two transformations that failed were versions of the salicylate reporter.
								After staring at plates, Dylan ran the nah+p29, oriT+p17c, p31c, and nah operon (from p20) digestions on a gel, extracted the fragments of interest, and then dephosphorylated the backbone. The isolated digestion products were used later used by Steven, who set up two overnight ligations – one to put the nah operon, with a constitutive promoter, on a pSB3C5 backbone with an added oriT sequence, and one to put the nah operon into the MCS of pSB1C3 for subsequent site directed mutagenesis.
								While Dylan ran his gel, Caleb miniprepped from the JG700+SAL and S31 cultures. After quantification, SAL plasmid isolated from JG700 was submitted for sequencing in order to confirm successful conjugation.
								Meanwhile, Mark decided that he enjoyed making DH5a electrocompetent stocks on Tuesday so much that he had to make more WM3064 electrocompetent stocks today. Starting from subcultures that Dylan had prepared, he prepared the stocks, and was assisted by Danielle.
								<B>From left: ladder, oriT+p17c digest (E+X), nah + p29 digest (E+S), p31c digest (E+S), p20 PCR digest (E+S):</B><B> August 2nd, Thursday </B>
								Caleb and Dylan created conjugation plates of p14RFP, p16RFP, and p25-29BB as well as miniprepped from the same plasmids in WM3064 (see: strain list). Dylan, Mark, and Tina then desalted and transformed SAL2 and SALRFP into WM3064 as well as p31c_nah and oriT_nah_p17c into DH5a. Tina and Dylan then quantified the minipreps from earlier in the day. Dylan also digested pBBRBB and p33k with EcoRI &amp; PstI to put lac inducible mtrB into the pBBRBB backbone.
								<B> August 3rd, Friday </B>
								After running a digestion of p33k, the lac inducible mtrB part from the parts registry, we discovered that the part does not include mtrB! We should have seen a ~3.6kb band on our gel after digesting with EcoRI and PstI, but instead isolated a band ~1.5kb. After checking the sequencing on the parts registry we discovered this was because the part didn't include mtrB, so we won't be able to use it as a positive control for inducible mtrB.
								Sequencing of p37k-p41k from WM3064 came back good, meaning we can put these control parts, with RFP downstream of our various reporters, into Shewanella via conjugation. Hence, conjugation was carried out.
								<B>Digests of p11 and p33 (E+P):</B><B> August 4th, Saturday </B>
								This week, it was confirmed that RFP was successfully inserted downstream of mtrB in our arsenic reporter in Shewenella.
								Site-directed mutagenesis of the nah operon failed. We continued trying to insert the nah operon into the mobile backbone (OriT) to prepare for conjugation into Shewanella.
								</DIV></DIV></DIV><DIV class="row"><DIV class="nine columns"><H3>August 5th - 11th</H3>
							This week, we continued our site directed mutagenesis of nah_p31c and the addition of RFP to our SAL reporters. Sequencing of the arsenic reporter has been unsuccessful so far. We also decided that we need more quantitative data to characterize our parts.
							<A href="#" class="technical-desc" for="#technical-desc7" style="display:block;margin-top:20px;">Daily Details</A><DIV class="hide-me panel" style="background:white;margin-top: 20px;" id="technical-desc7"><H6>Daily Details:</H6><B>August 5th, Sunday </B>
								Swati yet again was saddled with a massive number of <A href="https://static.igem.org/mediawiki/2012/3/35/Ezna_Miniprep.pdf">minipreps </A>, since her magic fingers are able to cast yield-increasing spells on minipreps. She miniprepped: SAL2 from WM3064, p37-41k from JG700 (one arsenic reporter with cut sites flanking the RBS, another arsenic reporter without the cut sites, and three different Anderson series promoters with mRFP1 downstream on a pBBRBB backbone), oriT p29nah_p17c from DH5a, and nah_p31c from DH5a. And indeed her yields were impressive, massive, gargantuan! Good job Sorceress Swati.
								We will <A href="https://static.igem.org/mediawiki/2012/0/03/Sequencing_Preparation.pdf "> sequence </A> SAL2 and oriT p29nah_p17c to see if we should continue to conjugation, and sequence nah_p31c to see if we can start <A href=" https://static.igem.org/mediawiki/2012/a/a5/Site_Directed_Mutagenesis.pdf "> site-directed mutagenesis </A> to get a biobrick-compatible nah operon on the biobrick backbone. We will sequence or PCR to confirm p37k-p41k's conjugation into Shewanella.
								<B> August 6th, Monday </B>
								Caleb and Dylan amplified the two different arsenic promoters (p37k and p38k) and ran them on a gel alongside p14 and p16 (the arsenic reporters without mRFP1 downstream) to confirm successful insertion of RFP downstream of mtrB in our arsenic reporter parts in Shewenella. Band lengths appeared in expected places. Dylan is worried that we will not be able to use PCR to definitely confirm that p39-41k worked, so we submitted these for <A href="https://static.igem.org/mediawiki/2012/0/03/Sequencing_Preparation.pdf "> sequencing </A>. Unfortunately, sequencing failed, perhaps because random gunk from Shewanella added noise to the process.
								Tina did a PCR cleanup of the nah operon PCR. Dylan also ran a gel of nah operon PCRs to make sure that there was no mispriming. The gel looked good, so he submitted the PCRs for sequencing.
								<B>Successful insertion of RFP downstream of mtrB in arsenic reporter strains in Shewanella:</B><B> August 7th, Tuesday </B>
								JG1531 overnight culture didn't grow. We suspect that the plate Dylan picked from is dead. We'll have to go back to the glycerol stocks if we want to play with mtrE. Dylan set up a continuous flow M4 reactor in morning. Caleb started a liquid culture of MR-1 with which to inoculate the reactor. Checked sequencing results of nah stuff. oriT thing was bad. nah_p31c was good.
								Caleb and Dylan proceeded with <A href=" https://static.igem.org/mediawiki/2012/a/a5/Site_Directed_Mutagenesis.pdf "> site-directed mutagenesis </A> of nah_p31c, attempting to get rid of the first PstI cutsite in the nah operon. After digestion with DpnI, we purified using the E.Z.N.A. MicroElute kit and quantified. DNA was split into three directions: First, Danielle and Dylan set up another mutagenic PCR using the digestion product as template to get rid of the second internal cut site (we expect lower mutation efficiency because template DNA is not methylated). Second, we transformed DH5a with the mutated plasmid. Third, Danielle and Chie <A href="https://static.igem.org/mediawiki/2012/a/af/Double_Digest.pdf "> digested </A> both the purified – and hopefully mutated – plasmid and un-mutated plasmid with PstI. Dylan ran these digestions on a gel, along with supercoiled plasmid as a control. Unfortunately, the (hopefully) mutated plasmid never showed up on the gel.
								In the evening, we didn't observe growth in the MR-1 culture, so Dylan set up another culture just in case Shewanella was dead and not lazy.
								<B>Looking for mutagenesis – from left – ladder, unmutated nah, mutated nah, supercoil control:</B><B>August 8th, Wednesday </B>
								Caleb and Dylan performed another <A href="https://static.igem.org/mediawiki/2012/a/af/Double_Digest.pdf "> digest </A> to check if the second <A href="https://static.igem.org/mediawiki/2012/a/a5/Site_Directed_Mutagenesis.pdf "> site-directed mutagenesis </A> (that was supposed to get rid of the second PstI internal cut site within the nah operon) worked. Unfortunately, there wasn't enough DNA to see anything when visualized on a gel. They decided to proceed with electroporation into DH5a just in case - however, to use Caleb's terminology, cells exploded - there were probably too many salts in the solution, which causing arcing and a PBBHTTTZZZ! of cells all over the cuvette.
								Dylan re-digested p29 nah and oriT p17c to redo a ligation that would allow for conjugation into Shewanella later on. Dylan also re-digested p27 (Anderson series promoter with RFP downstream) and SAL to redo a ligation to create a plasmid with SAL-RFP.
								Dylan also submitted several samples for <A href="https://static.igem.org/mediawiki/2012/0/03/Sequencing_Preparation.pdf "> sequencing </A> to make sure that three Anderson promoters with RFP downstream in JG700 and SAL2 (the salicylate reporter without the BAMHI cut site) in WN3064 and JG700 were in the correct sequence.
								<B>Seeing if mutagenesis worked; nothing there:</B><B> August 9th, Thursday </B>
								Today Dylan chilled with his mom. Well, he tried to. :) Despite the eternal bonds that bind mother and son, plus the 3 billion miles she flew to see him, he still couldn't resist coming in to lab to open packages! And then got sucked into a long discussion of what had to be done for the rest of the day - scientific endeavors taking him away yet again from filial duties. Swati and Claire bonded over dry ice, and Caleb regaled us with tales of dry ice bombs gone awry.
								Swati and Caleb miniprepped the first mutagenesis of nah_p31c that had been transformed into DH5a yesterday. They then digested it with PstI-HF to see if the mutation was successful. Unfortunately the mutagenesis failed! We will start over from nah_p31c and lower the annealing temperature at 60degC. The Stratagene kit, which uses PFU ultra, asks for 60degC, but because we are using our own boot-leg protocol with Phusion we did the first try at 65degC, as Phusion usually calls for a higher annealing temperature than the theoretically calculated value. For this second try we will stick to the 60degC suggested by Stratagene and see if we get better results. Also, called NEB to find out if after PCR with Phusion, DpnI will still have activity in the following digestion step in Buffer 4, or if we need to clean up the PCR before digesting with DpnI - they said PCR clean up isn't needed.
								We also ran digestions for making the nah operon on a backbone with a mobility gene and the salicylate reporter (w/ BamHI cutsite) with RFP downstream. We cut our mobile backbone, OriT in p17c (pSB3C5), with EcoRI and XbaI, while cutting the nah operon (p29nah) with EcoRI and SpeI. The nah operon with mobility gene must be constructed so that we may conjugate into Shewy and start testing our salicylate reporter. The salicylate reporter and p27a (from the Anderson series), with RFP, were cut with SpeI and PstI. The salicylate/RFP part will be used for troubleshooting the salicylate reporter. Digestion products were run on a gel, extracted, and quantified. Swati then dephosphorylated backbones and ligated both parts.
								Sequencing for p39-41k didn't look good, and neither did the salicylate reporter w/BamHI cutsite miniprepped from JG700. The sequencing for salicylate reporter w/ BamHI in WM3064, however, looked good, suggesting that conjugation may not have been as efficient as we had hoped. After a pow-wow we decided not to sequence more colonies, as we are hoping some may be good, and more importantly that if we use qPCR to get quantitative characterization data, we won't need to use the RFP parts. The plates will stay in the fridge as a back-up plan.
								In other news: Claire cried because it was difficult to update the notebook with a week's worth of work. Mark's dedication to the notebook is laudable and impressive. Good job team for doing so much! My head can't even comprehend the magnitude of your endeavors.
								<B>Unsuccessful mutagenesis (test with Pst1 cutsite):</B><B>Digestion products from left – ladder, p27, p29nah, oriTp17c, SAL:</B><B> August 10th, Friday </B>
								Caleb miniprepped a plasmid with mtrE from JG1531, and just for kicks, miniprepped from the &quot;exploded cells&quot; from August 8th. Suprisingly, he ended up getting decent yields for both, showing that the electroporation worked despite arcing. Caleb then digested the nah operon of the miniprep with PSTI and NotI to check if the mutagenesis was a success. It was run on a gel alongside a PCR of the Anderson series promoters with RFP downstream and SAL2 from Shewanella (to check if SAL2 and the promoters were sucessfully conjugated after sequencing on Monday failed). Unfortunately, bands did not appear where we expected them to.
								Steven and Spencer performed a PCR to get mtrE out of the Gralnick (JG700) plasmid.
								<B>From left lane: ladder, p39 PCR, p40 PCR, p41 PCR, nah-p31c (P+N digestion), SAL2 PCR:</B><B> August 11th, Saturday </B>
								Spencer and Steven checked the nah operon PstI and NotI digest (because yesterday's gel ran weirdly) and their mtrE PCR from the previous day on a gel. Unfortunately, bands did not appear where we expected them to.
								<B>mtrE PCR product (left) and checking mutagenesis again (right):</B></DIV></DIV></DIV><DIV class="row"><DIV class="nine columns"><H3>August 12th - 18th</H3>
							This week, we:
							1)Performed the neccessary work for getting our reporters into the pSB1C3 backbone required for submission. This included the neccessary digestions of our reporters out and ligations into the submission plasmid. Confirmation of successful transformation and ligation will be performed in the following weeks.
							2)Confirmed successful transformation of Shewanella with our arsenic and salicylate reporters via colony PCR.
							3)Performed the neccessary digestions and ligations to have mRFP downstream of our reporter system so as to do an additional fluorescent testing of the reporters to serve a second form of confirmation of increased transcription in the presence of our toxins.

							<A href="#" class="technical-desc" for="#technical-desc2" style="display:block;margin-top:20px;">Daily Details</A><DIV class="hide-me panel" style="background:white;margin-top: 20px;" id="technical-desc2"><H6>Daily Details:</H6><B> August 12th, Sunday </B>
								Weekend overview: It looks like the mtrE primers are actually working to amplify the part out of JG1531. We'll be putting mtrE in p31c, both so that we may submit the novel part to the registry, and to begin site-directed mutagenesis.
								We'll use mtrE mutagenesis as something of a control for nah operon mutagenesis (to see if the large size of the plasmid and nah operon is the problem).
								It also looks like nah operon mutagenesis hasn't worked. We'll put this on hold, and continue with ligation of p29nah into oriT_p17c once desalting paper arrives.
								Also, we'll have to redo things to confirm p39-41, SAL2 in JG700. What was done over the weekend looks weird.
								<B> August 13th, Monday </B>
								Dylan ran four simultaneous Phusion PCRs with an annealing temperature of 67degC and an extension time of 45sec to amplify mtrE. The gel looked good, with no mispriming and one band ~2.2kb. Claire did PCR clean up of the product, which will eventually be cut and ligated into p31c (see: strain list).
								Because the results from the weekend's screenings were confusing, we also ran PCRs of p39-41k (miniprepped from JG700 strains) to confirm whether conjugation into Shewanella was successful. These plasmids contain mRFP under the control of constitutive promoters of varying strengths. After confirmation, we will use these parts to characterize our inducible promoters in Shewanella.
								Dylan also started a liquid culture of JG700 + SAL to be inoculated into a continuous flow reactor with M4 media. With this setup, we will begin characterizing our reporter strains, initially in response to salicylate.
								Dylan performed a Phusion PCR to amplify mtrE, and ran a gel to check for product. There was a single band so no mispriming was occurring.
								Claire performed a double digest of the p14k and p16k with XbaI and PstI HF. This is for the construction of our final biobricks to be submitted to the registry.
								<B>PCR amplification of mtrE:</B><B>p39 and p41 verification:</B><B> August 14th, Tuesday </B>
								Steven performed a double digest of mtrE (EcorI &amp; SpeI), Sal (SpeI &amp; PstI), Sal2 (SpeI &amp; PstI), and p26a (SpeI &amp; PstI).
								Steven performed a ligation of p29nah + oriTp17c, and also a ligation of p27 digest + Sal. Dylan transformed colonies with the ligation.
								<B> August 15th, Wednesday </B>
								Dylan ran a gel and Claire performed gel extraction of mRFP to put downstream of Sal reporters. They also checked to see if site-directed mutagenesis was successful for the nah-p31c. Found the digest of mRFP and p26 looked good, but the mutagenesis looked strange.
								Dylan performed a taq PCR to confirm the presence of Sal2 in Shewanella. However no band was present in the gel ran afterwards, so perhaps colony PCR is the next best step.
								Dylan also ran a gel of the p14k, p16k p31 digests (all cut with XbaI and PstI) for the set-up for forming the biobricks.
								Steven performed a colony Phusion PCR of four colonies containing the p4k plasmid to again test for the presence of mtrE. Spencer then did PCR clean up
								Spencer performed a double digest of p4kdlg and p27a cutting with EcoRI &amp; SpeI followed by running a gel and extracting the digests.
								Spencer picked colonies from the previous ligations of the Sal, Sal2, p31c biobrick plasmids for miniprepping on Thursday.
								This morning Dylan noted that we may be getting a detectable basal level of current - around 6mA - in a continuous flow setup. This would make it possible to bypass the addition of mtrE to the salicylate sensing part, and still be able to distinguish Shewanella not detecting salicylate from dead Shewanella. He then added salicylate to the reactor innoculated with our salicylate reporting strain, bringing the concentration to 10uM. Later in the day, he noted that the current had risen to around 9mA, which is promising: our strain may be working! Claire set up four hour digestions with XbaI and PstI to put our arsenic parts, p14 and p16, into the iGEM backbone for submission.
								Dylan also transformed a salicylate reporter with mRFP downstream into DH5alpha and WM3064, as well as the nah operon with the p29 promoter in a p17c backbone with an oriT. The second of these could be conjugated into Shewanella if transformed successfully. The first, the salicylate reporter with mRFP downstream, is a fluorescent version of the salicylate reporter. The fluorescent versions of our reporters is an alternative to qPCR to measure the relative expression level of mtrB: if we know what mRFP under the control of a constitutive promotor in Shewanella looks like, we can add arsenic or naphthalene/salicylate to our fluorescent reporter parts until reaching the same level of expression. Then, we can correlate this concentration of arsenic or salicylate to a certain strength of induced expression.
								In case the fluorescent SAL reporter was not successfully ligated, we are preparing more mRFP, SAL, and SAL2: - In the morning Claire also cleaned up the digestions of SAL, SAL2 and mtrE PCR. However, due to a silly mistake on her part involving wash buffer and absolute ethanol (absolutely missing, to be precise) we will redo the digestions in order to get more DNA for putting these reporters into the iGEM backbone, and to put mRFP downstream. She then set up four hour digestions with XbaI and PstI to put our arsenic parts, p14 and p16, into the iGEM backbone for submission. - We digested mRFP with SpeI and PstI to make the part with mRFP downstream of mtrB in the salicylate reporters. We have already successfully put mRFP downstream of mtrB in our arsenic reporters, but it did not seem to work in the arsenic reporter. The gel for mRFP looked as expected, with the band for the insert slightly shorter than 900bp. However, on the same gel the the mutagenesis trial of the nah operon, digested with PstI and NotI, showed only one band. If mutagenesis had not worked, we would expect more than two bands, and if it had worked we would still expect two bands. Therefore we are unsure how to interpret this result, which we have seen twice now, but are going to start over with site-directed mutagenesis of the nah operon. We will also visualize the unmutated nah operon, digested with PstI and NotI, on a gel to see if that looks like we expect it to.
								Finally, it should be noted with jubilance that Claire was reunited with her umbrella today! The joy in the lab at this event was palpable and will be remembered for years to come.
								<B>mRFP digest and additional (failed) attempt at confirming mutagenesis:</B><B>From left – ladder, failed SAL2, p14, p16, p31digestions:</B><B> August 16th, Thursday </B>
								Dylan performed a miniprep of the Sal, Sal2, and p31c reporters for us to do sequencing for confirmation of successful ligations. These will hopefully be the biobricks we send in.
								In the afternoon Dylan performed a dephosphorylation of the p31c (cut with XbaI, PstI) followed by a gel of the mtrE cut with EcoRI &amp; SpeI as well as mRFP cut with SpeI and PstI.
								A 4 hour digestion was also performed by Claire to get Sal parts into pSB1C3 for the SAL, SAL2, and p31 parts. Steven then ran a gel of the digest products.
								Steven also performed a colony Taq PCR of the Sal2 reporter to confirm its presence in Shewanella.
								Today, we continued work to get our engineered reporters into pSB1C3 for submission to the parts registry. In the morning, Dylan miniprepped both versions of our salicylate reporters, along with more pSB1C3 from overnight cultures of the corresponding DH5a strains. Following quantification, he set up digestions of each miniprep with XbaI and PstI. (We decided to cut with XbaI because we discovered an extra base pair between the NotI and XbaI cutsites in the normal BioBrick prefix, an artifact of a previous team's work). Caleb also dephosphorylated previously isolated pSB1C3 backbone to prevent self-ligation, while Steven gel extracted the salicylate reporter inserts and more pSB1C3 backbone (to be quantified Friday morning).
								<B>mtrE and mRFP digests:</B><B>Digestions of SAL, SAL2, and p31:</B><B> August 17th, Friday </B>
								Claire performed a ligation for the construction of the Sal_RFP, and Sal2_RFP plasmids. Transformation was then performed immediately after.
								</DIV></DIV></DIV><DIV class="row"><DIV class="nine columns"><H3>August 19th - 25th</H3>
							Conjugating nah into Shewy:
							Colony PCR was done to 1) make sure the nah operon was in the cells 2) troubleshoot the colony PCR protocol After imaging, we discovered that the colonies did contain the nah operon, and that our colony PCR protocol was good.
							<A href="#" class="technical-desc" for="#technical-desc3" style="display:block;margin-top:20px;">Daily Details</A><DIV class="hide-me panel" style="background:white;margin-top: 20px;" id="technical-desc3"><H6>Daily Details:</H6><B> August 19th, Sunday </B>
								In the morning, Dylan came to lab to look at the transformant plates from Saturday. There were colonies on all but one of the plates: It looks like either the ligation to put mtrE into p31c or the subsequent electroporation failed. It is of note that the competent DH5α cells that Dylan used for the mtrE electroporation were from an older stock – one which we'd previously determined to confer lower transformation efficiency. Because we used up all of the newer stock – with markedly better efficiency – we will make more electrocompetent DH5α in the coming week.
								Because all other transformations seemed to work, we grew up overnight cultures from single colonies on each plate. We will miniprep from these cultures and screen the minipreps for the correct insert. If confirmed, we will have successfully have put both versions of our arsenic reporters, as well as both versions of our salicylate reporters inside p31c (i.e., pSB1C3), as is required for registry submission. We will also have successfully put mRFP downstream of mtrB on SAL2, which will be used for characterization of the salicylate sensitive promoter in S. oneidensis.
								<B>Gel showing failure of nahoriT p17c:</B><B> August 22th, Wednesday </B>
								Conjugating nah into Shewy: Caleb, Tina, and a guest Louis miniprepped 5 plasmids containing nah operons with constituitive promoters from a WM3064 plate and miniprepped one plasmid that we previously confirmed as having the nah operon (without a promoter) to use as a positive control in the future.
								Claire performed miniprep of grown cultures of the Sal_RFP and Sal2_RFP to submit for sequencing.
								<B> August 24th, Friday </B>
								Jim performed the digestion of the nah operon+ p29a and the p31c backbone for ultimate ligation in the creation of the last biobrick. Ligation had to be redone, as sequencing results of last week's ligations looked great except for the nah operon biobrick.

								Conjugating nah into Shewy:
								We wanted to accomplish two things:
								•	Amplify the nah operon with a promoter using primers with appended cut sites.
								•	Optimize colony PCR for future Cornell iGEMers (past attempts have failed.)
								To accomplish both of these objectives, we did Phusion PCRs using plasmids containing nah operons with constituitive promoters as a template and a Phusion colony PCR using the same colonies we miniprepped the plasmid from. Because we expect the miniprep PCRs to work, doing the colony PCRs in parallel will allow us to determine whether our colony PCR technique is working or not. For our colony PCR purposes, we want to be able to screen for colonies while being certain we can go back and miniprep the same PCRed colony. Our colony PCR method is as follows:
								NOTE: We later determined there was a cataclysmic flaw in the following method, can you figure it out? (wink wink nudge nudge...)
								•	Label colonies of interest and correspondingly label microfuge tubes
								•	Add 50 uL ddH2O to each microfuge tube
								•	Using pipette tip, dip into colonies of interest then dip into labelled microfuge tubes and swirl around to release cells
								•	Use the the ddH2O+cell mixture as the DNA template for a PCR - instead of adding plain ddH2O to the PCR to bring it up to volume, add the ddH2O+cell mixture
								•	Confirm your PCR had desired results with a DNA gel, then use the ddH2O+cell mixture from the corresponding microfuge tube the PCR was done on to inoculate a culture or plate.
								<B> August 25th, Saturday </B>
								Jim performed an overnight ligation of the nah operon pSB1C3 for transformation on Sunday.
								Conjugating nah into Shewy:
								Caleb, Tina, and a guest Louis ran a gel of the miniprep and colony parallel Phusion PCRs. All of the miniprep PCRs worked, but only a couple colony PCRs worked. We believe this is due to not having added enough cells to the colony PCR tube because some of the colonies were very tiny.
								<B>Top: Positive control from colony, colony test 1, colony test 2, colony test 3, colony test 4, colony test 5, miniprep 5, Benchtop 1kb; Bottom: Benchtop 1kb, miniprep 1, miniprep 2, miniprep 3, miniprep 4, positive control miniprep:</B></DIV></DIV></DIV><DIV class="row last-ele"><DIV class="nine columns"><H3>August 26th - 31st</H3>
							Operation “get nah into Shewy continued this week.” We were not able to confirm through colony PCR that we have nah in WM3064. Team fluorescence was able to conjugate the SAL_mRFP constructs into Shewy. We continued site-directed mutagenesis of the nah operon but it has not been working like we expected.
							<A href="#" class="technical-desc" for="#technical-desc4" style="display:block;margin-top:20px;">Daily Details</A><DIV class="hide-me panel" style="background:white;margin-top: 20px;" id="technical-desc4"><H6>Daily Details:</H6><B> August 26th, Sunday </B>
								Conjugating nah into Shewy:
								Caleb miniprepped two plasmids - one contained the nah operon preceded with a constitutive promoter and the other is to become a backbone containing the origin of transfer required for conjugation and a gene conveying chloramphenicol resistance. The purified plasmids were digested to extract the nah operon and to cut open the backbone - these digests were then run on a gel. Tina gel extracted the backbone then accidentally threw away the rest of the gel that contained the nah operon fragments.
								
								Site-directed mutagenesis: Swati ran a mutagenic PCR of nah_p31c, then digested with DpnI to remove template DNA and column purified the resultant DNA. <B> August 27th, Monday </B>
								Conjugating nah into Shewy:
								Caleb gel extracted the nah operon containing fragments from yesterday - despite having been in the garbage overnight, the gel extraction worked! Caleb and Tina began a 24 hour 16 degrees Celsius ligation using molar rations of nah to backbone of 3:1 and 1:1.
								Fluorescence tests: Claire started a JG700 + SAL2 culture and tried to do a PCR of SAL2_mRFP with sequencing primers to see if the band produced would correspond to the length of SAL2 plus the length of mRFP. Unfortunately, due to a mishap involving p10s vs. p2s and not checking what volume her pipette was set to, she ended up with a 70uL total volume and PCR failed.
								
								Site-directed mutagenesis: Swati transformed the (hopefully) mutated DNA into DH5a. <B> August 28th, Tuesday </B>
								Conjugating nah into Shewy:
								Caleb and Tina de-salted the 3:1 and 1:1 ligations, then transformed them into a DAP-requiring conjugation strain WM3064 via electroporation. The electroporated cells were plated on DAP + Cm plates.
								Fluorescence tests: JG700 once again takes a long time to grow, so it wasn’t until the evening that Claire miniprepped SAL2 from JG700. Tomorrow we will run another PCR and see if we have SAL2_mRFP, and SAL2 in JG700.
								
								Site-directed mutagenesis: Swati's transformation did not work. After checking the PCR product and a PstI digest thereof on a gel, she discovered that the PCR was not working with high enough efficiency to produce a band on a gel. Upon consulting with Didi, our trusty post-doctoral adviser, she then restarted the mutagenic process with a PCR; this time, with a lower primer concentration and lower denaturation and annealing times. <B> August 29th, Wednesday </B>
								Conjugating nah into Shewy:
								Caleb checked the transformation plates after ~14 hours in a 37 degrees Celsius incubator and noticed colonies, but the colonies were barely visible indicating very slow growth of the cells.
								Fluorescence tests: Set up a Taq PCR of SAL_mRFP, SAL2_mRFP and SAL2, with an extension time of 4.5 minutes and annealing temperature of 68degC. SAL2_mRFP has a band corresponding only to the length of SAL2, without mRFP, so we need to redo this ligation. However, the band from SAL_mRFP was the correct length (a little more than 4kb), so this ligation was successful and we can conjugate into Shewenella. Finally, the JG700 miniprep is the correct length for SAL2 (~3.8kb), so SAL2 has been successfully conjugated into Shewenella!
								
								Site-directed mutagenesis: Swati digested her second mutagenic PCR with DpnI, column purified, and transformed into DH5a. <B> August 30th, Thursday </B>
								Conjugating nah into Shewy:
								Tina checked the growth of yesterday's colonies and tried doing a colony PCR of each colony and a PCR of a positive control (same product, same primes, but in a different purified plasmid). The primers were designed to sit on the backbone and face into the oriT region and the site where we wanted to insert the nah operon. A PCR amplicon indicating successful ligation of the nah operon into the backbone would be approximately 10-11kb, while a failed ligation would yield a ~700 bp amplicon. Caleb later ran a DNA gel of the colony PCRs and nothing but smears showed up in every lane of the gel except the positive control which didn't even have a smear (Caleb was so disappointed that he didn't post the picture). Since the positive control didn't show up, we concluded the PCR failed. Caleb started overnight cultures of the remaining cells+ddH2O used for the colony PCR.
								
								Site-directed mutagenesis: Alas, Swati's second attempt at mutagenesis was yet again fruitless. Concerned that the template may have been causing the problem, she performed a NotI digest and ran it on a gel alongside the latest mutagenic PCR. The template was fine; the PCR was, again, invisible. Swati grew extremely puzzled as to how her purified PCR product could have significant concentration upon quantification but yield no visible bands on a gel. She decided to try a third time, this time with longer denaturation and annealing times, but still the same concentration of primers.
								<B>From left lane: Ladder, SAL2_mRFP (1), SAL2_mRFP (2), SAL2:</B><B> August 31st, Friday </B>
								Conjugating nah into Shewy: Caleb noticed the overnight cultures didn't grow and concluded the cells must have died due to the cells being stored in an extremely hypotonic solution (ddH2O). He re-did colony PCRs of the colonies, a colony with a positive control, and a positive control of purified plasmid, but modified the colony PCR protocol. Instead of using the cells + ddH2O as template DNA, Caleb dipped the pipette tip into the colony, then dip and swirled it in the PCR rxn tube before dip and swirling into labeled microfuge tubes with 50 uL LB media. The microfuge tubes were stored in a 4 degrees Celsius fridge. A DNA gel of the colony PCRs showed that the ligation of nah into the backbone failed - as seen in the accompanying gel picture, our positive control worked (indicating the PCR conditions were fine), but the test colonies all only had 700 bp amplicons.

								Fluorescence tests: Claire found some old SAL2 and mRFP digested with PstI and SpeI already, so she decided to retry the ligation. She did a 40min ligation at room temp, and continued it overnight in the 4degC fridge, which the NEB rep said could increase efficience. However, since she forgot to dephosphorylate SAL2, we will never know if this room temp and overnight in the fridge combo would have been effective because it self ligated.
								
								Site-directed mutagenesis: Swat ran a DpnI digestion of half of the previous day's PCR product, in an attempt to rule out the possibility that DpnI was functioning incorrectly.

								<B>From left lane: Ladder, positive control miniprepped plasmid, colony 1, colony 2, colony 3, colony 4, colony 5, colony 6, colony 7, colony 8, positive control colony:</B></DIV></DIV></DIV><DIV class="row last-ele"><DIV class="three columns"><A href="https://2012.igem.org/Team:Cornell/notebook/wetlab/july#pillTab1">&lt;&lt; Previous</A></DIV><DIV class="three columns offset-by-six"><A href="https://2012.igem.org/Team:Cornell/notebook/wetlab/september#pillTab1" style="display:block;float:right;">Next &gt;&gt;</A></DIV></DIV></LI><LI id="pillTab2Tab"><DIV class="row last-ele" style="margin-top:20px;"><DIV class="three columns"><A href="https://2012.igem.org/Team:Cornell/notebook/wetlab/july#pillTab2">&lt;&lt; Previous</A></DIV><DIV class="three columns offset-by-six"><A href="https://2012.igem.org/Team:Cornell/notebook/wetlab/september#pillTab2" style="display:block;float:right;">Next &gt;&gt;</A></DIV></DIV><DIV class="row"><DIV class="nine columns"><H3>August 1st - 4th</H3><B> August 1st, Wednesday </B>
							In the morning, Dylan noted that 5 of the 7 transformations from Tuesday produced plates with colonies. Those that worked were three Anderson series constitutive promoters (with downstream mRFP) in pBBRBB and the versions of our two arsenic reporters with mRFP downstream of mtrB. In the evening, Swati and Dylan prepared overnight cultures from all transformants and JG700, both to set up conjugations to get plasmid into Shewanella and to screen isolated plasmids for successful ligation. We will use these constructs to characterize the activity of the arsenic-sensitive promoter in Shewanella with respect to known constitutive promoter strengths. The two transformations that failed were versions of the salicylate reporter.
							After staring at plates, Dylan ran the nah+p29, oriT+p17c, p31c, and nah operon (from p20) digestions on a gel, extracted the fragments of interest, and then dephosphorylated the backbone. The isolated digestion products were used later used by Steven, who set up two overnight ligations – one to put the nah operon, with a constitutive promoter, on a pSB3C5 backbone with an added oriT sequence, and one to put the nah operon into the MCS of pSB1C3 for subsequent site directed mutagenesis.
							While Dylan ran his gel, Caleb miniprepped from the JG700+SAL and S31 cultures. After quantification, SAL plasmid isolated from JG700 was submitted for sequencing in order to confirm successful conjugation.
							Meanwhile, Mark decided that he enjoyed making DH5a electrocompetent stocks on Tuesday so much that he had to make more WM3064 electrocompetent stocks today. Starting from subcultures that Dylan had prepared, he prepared the stocks, and was assisted by Danielle.
							<B>From left: ladder, oriT+p17c digest (E+X), nah + p29 digest (E+S), p31c digest (E+S), p20 PCR digest (E+S):</B><B> August 2nd, Thursday </B>
							Caleb and Dylan created conjugation plates of p14RFP, p16RFP, and p25-29BB as well as miniprepped from the same plasmids in WM3064 (see: strain list). Dylan, Mark, and Tina then desalted and transformed SAL2 and SALRFP into WM3064 as well as p31c_nah and oriT_nah_p17c into DH5a. Tina and Dylan then quantified the minipreps from earlier in the day. Dylan also digested pBBRBB and p33k with EcoRI &amp; PstI to put lac inducible mtrB into the pBBRBB backbone.
							<B> August 3rd, Friday </B>
							After running a digestion of p33k, the lac inducible mtrB part from the parts registry, we discovered that the part does not include mtrB! We should have seen a ~3.6kb band on our gel after digesting with EcoRI and PstI, but instead isolated a band ~1.5kb. After checking the sequencing on the parts registry we discovered this was because the part didn't include mtrB, so we won't be able to use it as a positive control for inducible mtrB.
							Sequencing of p37k-p41k from WM3064 came back good, meaning we can put these control parts, with RFP downstream of our various reporters, into Shewanella via conjugation. Hence, conjugation was carried out.
							<B>Digests of p11 and p33 (E+P):</B><B> August 4th, Saturday </B>
							This week, it was confirmed that RFP was successfully inserted downstream of mtrB in our arsenic reporter in Shewenella.
							Site-directed mutagenesis of the nah operon failed. We continued trying to insert the nah operon into the mobile backbone (OriT) to prepare for conjugation into Shewanella.
							</DIV></DIV><DIV class="row"><DIV class="nine columns"><H3>August 5th - 11th</H3><B>August 5th, Sunday </B>
							Swati yet again was saddled with a massive number of <A href="https://static.igem.org/mediawiki/2012/3/35/Ezna_Miniprep.pdf">minipreps </A>, since her magic fingers are able to cast yield-increasing spells on minipreps. She miniprepped: SAL2 from WM3064, p37-41k from JG700 (one arsenic reporter with cut sites flanking the RBS, another arsenic reporter without the cut sites, and three different Anderson series promoters with mRFP1 downstream on a pBBRBB backbone), oriT p29nah_p17c from DH5a, and nah_p31c from DH5a. And indeed her yields were impressive, massive, gargantuan! Good job Sorceress Swati.
							We will <A href="https://static.igem.org/mediawiki/2012/0/03/Sequencing_Preparation.pdf "> sequence </A> SAL2 and oriT p29nah_p17c to see if we should continue to conjugation, and sequence nah_p31c to see if we can start <A href=" https://static.igem.org/mediawiki/2012/a/a5/Site_Directed_Mutagenesis.pdf "> site-directed mutagenesis </A> to get a biobrick-compatible nah operon on the biobrick backbone. We will sequence or PCR to confirm p37k-p41k's conjugation into Shewanella.
							<B> August 6th, Monday </B>
							Caleb and Dylan amplified the two different arsenic promoters (p37k and p38k) and ran them on a gel alongside p14 and p16 (the arsenic reporters without mRFP1 downstream) to confirm successful insertion of RFP downstream of mtrB in our arsenic reporter parts in Shewenella. Band lengths appeared in expected places. Dylan is worried that we will not be able to use PCR to definitely confirm that p39-41k worked, so we submitted these for <A href="https://static.igem.org/mediawiki/2012/0/03/Sequencing_Preparation.pdf "> sequencing </A>. Unfortunately, sequencing failed, perhaps because random gunk from Shewanella added noise to the process.
							Tina did a PCR cleanup of the nah operon PCR. Dylan also ran a gel of nah operon PCRs to make sure that there was no mispriming. The gel looked good, so he submitted the PCRs for sequencing.
							<B>Successful insertion of RFP downstream of mtrB in arsenic reporter strains in Shewanella:</B><B> August 7th, Tuesday </B>
							JG1531 overnight culture didn't grow. We suspect that the plate Dylan picked from is dead. We'll have to go back to the glycerol stocks if we want to play with mtrE. Dylan set up a continuous flow M4 reactor in morning. Caleb started a liquid culture of MR-1 with which to inoculate the reactor. Checked sequencing results of nah stuff. oriT thing was bad. nah_p31c was good.
							Caleb and Dylan proceeded with <A href=" https://static.igem.org/mediawiki/2012/a/a5/Site_Directed_Mutagenesis.pdf "> site-directed mutagenesis </A> of nah_p31c, attempting to get rid of the first PstI cutsite in the nah operon. After digestion with DpnI, we purified using the E.Z.N.A. MicroElute kit and quantified. DNA was split into three directions: First, Danielle and Dylan set up another mutagenic PCR using the digestion product as template to get rid of the second internal cut site (we expect lower mutation efficiency because template DNA is not methylated). Second, we transformed DH5a with the mutated plasmid. Third, Danielle and Chie <A href="https://static.igem.org/mediawiki/2012/a/af/Double_Digest.pdf "> digested </A> both the purified – and hopefully mutated – plasmid and un-mutated plasmid with PstI. Dylan ran these digestions on a gel, along with supercoiled plasmid as a control. Unfortunately, the (hopefully) mutated plasmid never showed up on the gel.
							In the evening, we didn't observe growth in the MR-1 culture, so Dylan set up another culture just in case Shewanella was dead and not lazy.
							<B>Looking for mutagenesis – from left – ladder, unmutated nah, mutated nah, supercoil control:</B><B> August 8th, Wednesday </B>
							Caleb and Dylan performed another <A href="https://static.igem.org/mediawiki/2012/a/af/Double_Digest.pdf "> digest </A> to check if the second <A href="https://static.igem.org/mediawiki/2012/a/a5/Site_Directed_Mutagenesis.pdf "> site-directed mutagenesis </A> (that was supposed to get rid of the second PstI internal cut site within the nah operon) worked. Unfortunately, there wasn't enough DNA to see anything when visualized on a gel. They decided to proceed with electroporation into DH5a just in case - however, to use Caleb's terminology, cells exploded - there were probably too many salts in the solution, which causing arcing and a PBBHTTTZZZ! of cells all over the cuvette.
							Dylan re-digested p29 nah and oriT p17c to redo a ligation that would allow for conjugation into Shewanella later on. Dylan also re-digested p27 (Anderson series promoter with RFP downstream) and SAL to redo a ligation to create a plasmid with SAL-RFP.
							Dylan also submitted several samples for <A href="https://static.igem.org/mediawiki/2012/0/03/Sequencing_Preparation.pdf "> sequencing </A> to make sure that three Anderson promoters with RFP downstream in JG700 and SAL2 (the salicylate reporter without the BAMHI cut site) in WN3064 and JG700 were in the correct sequence.
							<B>Seeing if mutagenesis worked; nothing there:</B><B> August 9th, Thursday </B>
							Today Dylan chilled with his mom. Well, he tried to. :) Despite the eternal bonds that bind mother and son, plus the 3 billion miles she flew to see him, he still couldn't resist coming in to lab to open packages! And then got sucked into a long discussion of what had to be done for the rest of the day - scientific endeavors taking him away yet again from filial duties. Swati and Claire bonded over dry ice, and Caleb regaled us with tales of dry ice bombs gone awry.
							Swati and Caleb miniprepped the first mutagenesis of nah_p31c that had been transformed into DH5a yesterday. They then digested it with PstI-HF to see if the mutation was successful. Unfortunately the mutagenesis failed! We will start over from nah_p31c and lower the annealing temperature at 60degC. The Stratagene kit, which uses PFU ultra, asks for 60degC, but because we are using our own boot-leg protocol with Phusion we did the first try at 65degC, as Phusion usually calls for a higher annealing temperature than the theoretically calculated value. For this second try we will stick to the 60degC suggested by Stratagene and see if we get better results. Also, called NEB to find out if after PCR with Phusion, DpnI will still have activity in the following digestion step in Buffer 4, or if we need to clean up the PCR before digesting with DpnI - they said PCR clean up isn't needed.
							We also ran digestions for making the nah operon on a backbone with a mobility gene and the salicylate reporter (w/ BamHI cutsite) with RFP downstream. We cut our mobile backbone, OriT in p17c (pSB3C5), with EcoRI and XbaI, while cutting the nah operon (p29nah) with EcoRI and SpeI. The nah operon with mobility gene must be constructed so that we may conjugate into Shewy and start testing our salicylate reporter. The salicylate reporter and p27a (from the Anderson series), with RFP, were cut with SpeI and PstI. The salicylate/RFP part will be used for troubleshooting the salicylate reporter. Digestion products were run on a gel, extracted, and quantified. Swati then dephosphorylated backbones and ligated both parts.
							Sequencing for p39-41k didn't look good, and neither did the salicylate reporter w/BamHI cutsite miniprepped from JG700. The sequencing for salicylate reporter w/ BamHI in WM3064, however, looked good, suggesting that conjugation may not have been as efficient as we had hoped. After a pow-wow we decided not to sequence more colonies, as we are hoping some may be good, and more importantly that if we use qPCR to get quantitative characterization data, we won't need to use the RFP parts. The plates will stay in the fridge as a back-up plan.
							In other news: Claire cried because it was difficult to update the notebook with a week's worth of work. Mark's dedication to the notebook is laudable and impressive. Good job team for doing so much! My head can't even comprehend the magnitude of your endeavors.
							<B>Unsuccessful mutagenesis (test with Pst1 cutsite):</B><B>Digestion products from left – ladder, p27, p29nah, oriTp17c, SAL:</B><B> August 10th, Friday </B>
							Caleb miniprepped a plasmid with mtrE from JG1531, and just for kicks, miniprepped from the &quot;exploded cells&quot; from August 8th. Suprisingly, he ended up getting decent yields for both, showing that the electroporation worked despite arcing. Caleb then digested the nah operon of the miniprep with PSTI and NotI to check if the mutagenesis was a success. It was run on a gel alongside a PCR of the Anderson series promoters with RFP downstream and SAL2 from Shewanella (to check if SAL2 and the promoters were sucessfully conjugated after sequencing on Monday failed). Unfortunately, bands did not appear where we expected them to.
							Steven and Spencer performed a PCR to get mtrE out of the Gralnick (JG700) plasmid.
							<B>From left lane: ladder, p39 PCR, p40 PCR, p41 PCR, nah-p31c (P+N digestion), SAL2 PCR:</B><B> August 11th, Saturday </B>
							Spencer and Steven checked the nah operon PstI and NotI digest (because yesterday's gel ran weirdly) and their mtrE PCR from the previous day on a gel. Unfortunately, bands did not appear where we expected them to.
							<B>mtrE PCR product (left) and checking mutagenesis again (right):</B></DIV></DIV><DIV class="row"><DIV class="nine columns"><H3>August 12th - 18th</H3><B> August 12th, Sunday </B>
							Weekend overview: It looks like the mtrE primers are actually working to amplify the part out of JG1531. We'll be putting mtrE in p31c, both so that we may submit the novel part to the registry, and to begin site-directed mutagenesis.
							We'll use mtrE mutagenesis as something of a control for nah operon mutagenesis (to see if the large size of the plasmid and nah operon is the problem).
							It also looks like nah operon mutagenesis hasn't worked. We'll put this on hold, and continue with ligation of p29nah into oriT_p17c once desalting paper arrives.
							Also, we'll have to redo things to confirm p39-41, SAL2 in JG700. What was done over the weekend looks weird.
							<B> August 13th, Monday </B>
							Dylan ran four simultaneous Phusion PCRs with an annealing temperature of 67degC and an extension time of 45sec to amplify mtrE. The gel looked good, with no mispriming and one band ~2.2kb. Claire did PCR clean up of the product, which will eventually be cut and ligated into p31c (see: strain list).
							Because the results from the weekend's screenings were confusing, we also ran PCRs of p39-41k (miniprepped from JG700 strains) to confirm whether conjugation into Shewanella was successful. These plasmids contain mRFP under the control of constitutive promoters of varying strengths. After confirmation, we will use these parts to characterize our inducible promoters in Shewanella.
							Dylan also started a liquid culture of JG700 + SAL to be inoculated into a continuous flow reactor with M4 media. With this setup, we will begin characterizing our reporter strains, initially in response to salicylate.
							Dylan performed a Phusion PCR to amplify mtrE, and ran a gel to check for product. There was a single band so no mispriming was occurring.
							Claire performed a double digest of the p14k and p16k with XbaI and PstI HF. This is for the construction of our final biobricks to be submitted to the registry.
							<B>PCR amplification of mtrE:</B><B>p39 and p41 verification:</B><B> August 14th, Tuesday </B>
							Steven performed a double digest of mtrE (EcorI &amp; SpeI), Sal (SpeI &amp; PstI), Sal2 (SpeI &amp; PstI), and p26a (SpeI &amp; PstI).
							Steven performed a ligation of p29nah + oriTp17c, and also a ligation of p27 digest + Sal. Dylan transformed colonies with the ligation.
							<B> August 15th, Wednesday </B>
							Dylan ran a gel and Claire performed gel extraction of mRFP to put downstream of Sal reporters. They also checked to see if site-directed mutagenesis was successful for the nah-p31c. Found the digest of mRFP and p26 looked good, but the mutagenesis looked strange.
							Dylan performed a taq PCR to confirm the presence of Sal2 in Shewanella. However no band was present in the gel ran afterwards, so perhaps colony PCR is the next best step.
							Dylan also ran a gel of the p14k, p16k p31 digests (all cut with XbaI and PstI) for the set-up for forming the biobricks.
							Steven performed a colony Phusion PCR of four colonies containing the p4k plasmid to again test for the presence of mtrE. Spencer then did PCR clean up
							Spencer performed a double digest of p4kdlg and p27a cutting with EcoRI &amp; SpeI followed by running a gel and extracting the digests.
							Spencer picked colonies from the previous ligations of the Sal, Sal2, p31c biobrick plasmids for miniprepping on Thursday.
							This morning Dylan noted that we may be getting a detectable basal level of current - around 6mA - in a continuous flow setup. This would make it possible to bypass the addition of mtrE to the salicylate sensing part, and still be able to distinguish Shewanella not detecting salicylate from dead Shewanella. He then added salicylate to the reactor innoculated with our salicylate reporting strain, bringing the concentration to 10uM. Later in the day, he noted that the current had risen to around 9mA, which is promising: our strain may be working! Claire set up four hour digestions with XbaI and PstI to put our arsenic parts, p14 and p16, into the iGEM backbone for submission.
							Dylan also transformed a salicylate reporter with mRFP downstream into DH5alpha and WM3064, as well as the nah operon with the p29 promoter in a p17c backbone with an oriT. The second of these could be conjugated into Shewanella if transformed successfully. The first, the salicylate reporter with mRFP downstream, is a fluorescent version of the salicylate reporter. The fluorescent versions of our reporters is an alternative to qPCR to measure the relative expression level of mtrB: if we know what mRFP under the control of a constitutive promotor in Shewanella looks like, we can add arsenic or naphthalene/salicylate to our fluorescent reporter parts until reaching the same level of expression. Then, we can correlate this concentration of arsenic or salicylate to a certain strength of induced expression.
							In case the fluorescent SAL reporter was not successfully ligated, we are preparing more mRFP, SAL, and SAL2: - In the morning Claire also cleaned up the digestions of SAL, SAL2 and mtrE PCR. However, due to a silly mistake on her part involving wash buffer and absolute ethanol (absolutely missing, to be precise) we will redo the digestions in order to get more DNA for putting these reporters into the iGEM backbone, and to put mRFP downstream. She then set up four hour digestions with XbaI and PstI to put our arsenic parts, p14 and p16, into the iGEM backbone for submission. - We digested mRFP with SpeI and PstI to make the part with mRFP downstream of mtrB in the salicylate reporters. We have already successfully put mRFP downstream of mtrB in our arsenic reporters, but it did not seem to work in the arsenic reporter. The gel for mRFP looked as expected, with the band for the insert slightly shorter than 900bp. However, on the same gel the the mutagenesis trial of the nah operon, digested with PstI and NotI, showed only one band. If mutagenesis had not worked, we would expect more than two bands, and if it had worked we would still expect two bands. Therefore we are unsure how to interpret this result, which we have seen twice now, but are going to start over with site-directed mutagenesis of the nah operon. We will also visualize the unmutated nah operon, digested with PstI and NotI, on a gel to see if that looks like we expect it to.
							Finally, it should be noted with jubilance that Claire was reunited with her umbrella today! The joy in the lab at this event was palpable and will be remembered for years to come.
							<B>mRFP digest and additional (failed) attempt at confirming mutagenesis:</B><B>From left – ladder, failed SAL2, p14, p16, p31digestions:</B><B> August 16th, Thursday </B>
							Dylan performed a miniprep of the Sal, Sal2, and p31c reporters for us to do sequencing for confirmation of successful ligations. These will hopefully be the biobricks we send in.
							In the afternoon Dylan performed a dephosphorylation of the p31c (cut with XbaI, PstI) followed by a gel of the mtrE cut with EcoRI &amp; SpeI as well as mRFP cut with SpeI and PstI.
							A 4 hour digestion was also performed by Claire to get Sal parts into pSB1C3 for the SAL, SAL2, and p31 parts. Steven then ran a gel of the digest products.
							Steven also performed a colony Taq PCR of the Sal2 reporter to confirm its presence in Shewanella.
							Today, we continued work to get our engineered reporters into pSB1C3 for submission to the parts registry. In the morning, Dylan miniprepped both versions of our salicylate reporters, along with more pSB1C3 from overnight cultures of the corresponding DH5a strains. Following quantification, he set up digestions of each miniprep with XbaI and PstI. (We decided to cut with XbaI because we discovered an extra base pair between the NotI and XbaI cutsites in the normal BioBrick prefix, an artifact of a previous team's work). Caleb also dephosphorylated previously isolated pSB1C3 backbone to prevent self-ligation, while Steven gel extracted the salicylate reporter inserts and more pSB1C3 backbone (to be quantified Friday morning).
							<B>mtrE and mRFP digests:</B><B>Digestions of SAL, SAL2, and p31:</B><B> August 17th, Friday </B>
							Claire performed a ligation for the construction of the Sal_RFP, and Sal2_RFP plasmids. Transformation was then performed immediately after.
							</DIV></DIV><DIV class="row"><DIV class="nine columns"><H3>August 19th - 25th</H3><B> August 19th, Sunday </B>
							In the morning, Dylan came to lab to look at the transformant plates from Saturday. There were colonies on all but one of the plates: It looks like either the ligation to put mtrE into p31c or the subsequent electroporation failed. It is of note that the competent DH5α cells that Dylan used for the mtrE electroporation were from an older stock – one which we'd previously determined to confer lower transformation efficiency. Because we used up all of the newer stock – with markedly better efficiency – we will make more electrocompetent DH5α in the coming week.
							Because all other transformations seemed to work, we grew up overnight cultures from single colonies on each plate. We will miniprep from these cultures and screen the minipreps for the correct insert. If confirmed, we will have successfully have put both versions of our arsenic reporters, as well as both versions of our salicylate reporters inside p31c (i.e., pSB1C3), as is required for registry submission. We will also have successfully put mRFP downstream of mtrB on SAL2, which will be used for characterization of the salicylate sensitive promoter in S. oneidensis.
							<B>Gel showing failure of nahoriT p17c:</B><B> August 22th, Wednesday </B>
							Conjugating nah into Shewy: Caleb, Tina, and a guest Louis miniprepped 5 plasmids containing nah operons with constituitive promoters from a WM3064 plate and miniprepped one plasmid that we previously confirmed as having the nah operon (without a promoter) to use as a positive control in the future.
							Claire performed miniprep of grown cultures of the Sal_RFP and Sal2_RFP to submit for sequencing.
							<B> August 24th, Friday </B>
							Jim performed the digestion of the nah operon+ p29a and the p31c backbone for ultimate ligation in the creation of the last biobrick. Ligation had to be redone, as sequencing results of last week's ligations looked great except for the nah operon biobrick.

							Conjugating nah into Shewy:
							We wanted to accomplish two things:
							•	Amplify the nah operon with a promoter using primers with appended cut sites.
							•	Optimize colony PCR for future Cornell iGEMers (past attempts have failed.)
							To accomplish both of these objectives, we did Phusion PCRs using plasmids containing nah operons with constituitive promoters as a template and a Phusion colony PCR using the same colonies we miniprepped the plasmid from. Because we expect the miniprep PCRs to work, doing the colony PCRs in parallel will allow us to determine whether our colony PCR technique is working or not. For our colony PCR purposes, we want to be able to screen for colonies while being certain we can go back and miniprep the same PCRed colony. Our colony PCR method is as follows:
							NOTE: We later determined there was a cataclysmic flaw in the following method, can you figure it out? (wink wink nudge nudge...)
							•	Label colonies of interest and correspondingly label microfuge tubes
							•	Add 50 uL ddH2O to each microfuge tube
							•	Using pipette tip, dip into colonies of interest then dip into labelled microfuge tubes and swirl around to release cells
							•	Use the the ddH2O+cell mixture as the DNA template for a PCR - instead of adding plain ddH2O to the PCR to bring it up to volume, add the ddH2O+cell mixture
							•	Confirm your PCR had desired results with a DNA gel, then use the ddH2O+cell mixture from the corresponding microfuge tube the PCR was done on to inoculate a culture or plate.
							<B> August 25th, Saturday </B>
							Jim performed an overnight ligation of the nah operon pSB1C3 for transformation on Sunday.
							Conjugating nah into Shewy:
							Caleb, Tina, and a guest Louis ran a gel of the miniprep and colony parallel Phusion PCRs. All of the miniprep PCRs worked, but only a couple colony PCRs worked. We believe this is due to not having added enough cells to the colony PCR tube because some of the colonies were very tiny.
							<B>Top: Positive control from colony, colony test 1, colony test 2, colony test 3, colony test 4, colony test 5, miniprep 5, Benchtop 1kb; Bottom: Benchtop 1kb, miniprep 1, miniprep 2, miniprep 3, miniprep 4, positive control miniprep:</B></DIV></DIV><DIV class="row last-ele"><DIV class="nine columns"><H3>August 26th - 31st</H3><B> August 26th, Sunday </B>
							Conjugating nah into Shewy:
							Caleb miniprepped two plasmids - one contained the nah operon preceded with a constitutive promoter and the other is to become a backbone containing the origin of transfer required for conjugation and a gene conveying chloramphenicol resistance. The purified plasmids were digested to extract the nah operon and to cut open the backbone - these digests were then run on a gel. Tina gel extracted the backbone then accidentally threw away the rest of the gel that contained the nah operon fragments.
							
							Site-directed mutagenesis: Swati ran a mutagenic PCR of nah_p31c, then digested with DpnI to remove template DNA and column purified the resultant DNA. <B> August 27th, Monday </B>
							Conjugating nah into Shewy:
							Caleb gel extracted the nah operon containing fragments from yesterday - despite having been in the garbage overnight, the gel extraction worked! Caleb and Tina began a 24 hour 16 degrees Celsius ligation using molar rations of nah to backbone of 3:1 and 1:1.
							Fluorescence tests: Claire started a JG700 + SAL2 culture and tried to do a PCR of SAL2_mRFP with sequencing primers to see if the band produced would correspond to the length of SAL2 plus the length of mRFP. Unfortunately, due to a mishap involving p10s vs. p2s and not checking what volume her pipette was set to, she ended up with a 70uL total volume and PCR failed.
							
							Site-directed mutagenesis: Swati transformed the (hopefully) mutated DNA into DH5a. <B> August 28th, Tuesday </B>
							Conjugating nah into Shewy:
							Caleb and Tina de-salted the 3:1 and 1:1 ligations, then transformed them into a DAP-requiring conjugation strain WM3064 via electroporation. The electroporated cells were plated on DAP + Cm plates.
							Fluorescence tests: JG700 once again takes a long time to grow, so it wasn’t until the evening that Claire miniprepped SAL2 from JG700. Tomorrow we will run another PCR and see if we have SAL2_mRFP, and SAL2 in JG700.
							
							Site-directed mutagenesis: Swati's transformation did not work. After checking the PCR product and a PstI digest thereof on a gel, she discovered that the PCR was not working with high enough efficiency to produce a band on a gel. Upon consulting with Didi, our trusty post-doctoral adviser, she then restarted the mutagenic process with a PCR; this time, with a lower primer concentration and lower denaturation and annealing times. <B> August 29th, Wednesday </B>
							Conjugating nah into Shewy:
							Caleb checked the transformation plates after ~14 hours in a 37 degrees Celsius incubator and noticed colonies, but the colonies were barely visible indicating very slow growth of the cells.
							Fluorescence tests: Set up a Taq PCR of SAL_mRFP, SAL2_mRFP and SAL2, with an extension time of 4.5 minutes and annealing temperature of 68degC. SAL2_mRFP has a band corresponding only to the length of SAL2, without mRFP, so we need to redo this ligation. However, the band from SAL_mRFP was the correct length (a little more than 4kb), so this ligation was successful and we can conjugate into Shewenella. Finally, the JG700 miniprep is the correct length for SAL2 (~3.8kb), so SAL2 has been successfully conjugated into Shewenella!
							
							Site-directed mutagenesis: Swati digested her second mutagenic PCR with DpnI, column purified, and transformed into DH5a. <B> August 30th, Thursday </B>
							Conjugating nah into Shewy:
							Tina checked the growth of yesterday's colonies and tried doing a colony PCR of each colony and a PCR of a positive control (same product, same primes, but in a different purified plasmid). The primers were designed to sit on the backbone and face into the oriT region and the site where we wanted to insert the nah operon. A PCR amplicon indicating successful ligation of the nah operon into the backbone would be approximately 10-11kb, while a failed ligation would yield a ~700 bp amplicon. Caleb later ran a DNA gel of the colony PCRs and nothing but smears showed up in every lane of the gel except the positive control which didn't even have a smear (Caleb was so disappointed that he didn't post the picture). Since the positive control didn't show up, we concluded the PCR failed. Caleb started overnight cultures of the remaining cells+ddH2O used for the colony PCR.
							
							Site-directed mutagenesis: Alas, Swati's second attempt at mutagenesis was yet again fruitless. Concerned that the template may have been causing the problem, she performed a NotI digest and ran it on a gel alongside the latest mutagenic PCR. The template was fine; the PCR was, again, invisible. Swati grew extremely puzzled as to how her purified PCR product could have significant concentration upon quantification but yield no visible bands on a gel. She decided to try a third time, this time with longer denaturation and annealing times, but still the same concentration of primers.
							<B>From left lane: Ladder, SAL2_mRFP (1), SAL2_mRFP (2), SAL2:</B><B> August 31st, Friday </B>
							Conjugating nah into Shewy: Caleb noticed the overnight cultures didn't grow and concluded the cells must have died due to the cells being stored in an extremely hypotonic solution (ddH2O). He re-did colony PCRs of the colonies, a colony with a positive control, and a positive control of purified plasmid, but modified the colony PCR protocol. Instead of using the cells + ddH2O as template DNA, Caleb dipped the pipette tip into the colony, then dip and swirled it in the PCR rxn tube before dip and swirling into labeled microfuge tubes with 50 uL LB media. The microfuge tubes were stored in a 4 degrees Celsius fridge. A DNA gel of the colony PCRs showed that the ligation of nah into the backbone failed - as seen in the accompanying gel picture, our positive control worked (indicating the PCR conditions were fine), but the test colonies all only had 700 bp amplicons.

							Fluorescence tests: Claire found some old SAL2 and mRFP digested with PstI and SpeI already, so she decided to retry the ligation. She did a 40min ligation at room temp, and continued it overnight in the 4degC fridge, which the NEB rep said could increase efficience. However, since she forgot to dephosphorylate SAL2, we will never know if this room temp and overnight in the fridge combo would have been effective because it self ligated.
							
							Site-directed mutagenesis: Swat ran a DpnI digestion of half of the previous day's PCR product, in an attempt to rule out the possibility that DpnI was functioning incorrectly.
							<B>From left lane: Ladder, positive control miniprepped plasmid, colony 1, colony 2, colony 3, colony 4, colony 5, colony 6, colony 7, colony 8, positive control colony:</B></DIV></DIV><DIV class="row last-ele"><DIV class="three columns"><A href="https://2012.igem.org/Team:Cornell/notebook/wetlab/july#pillTab2">&lt;&lt; Previous</A></DIV><DIV class="three columns offset-by-six"><A href="https://2012.igem.org/Team:Cornell/notebook/wetlab/september#pillTab2" style="display:block;float:right;">Next &gt;&gt;</A></DIV></DIV></LI><LI id="pillTab3Tab"><DIV class="row last-ele" style="margin-top:20px;"><DIV class="three columns"><A href="https://2012.igem.org/Team:Cornell/notebook/wetlab/july#pillTab3">&lt;&lt; Previous</A></DIV><DIV class="three columns offset-by-six"><A href="https://2012.igem.org/Team:Cornell/notebook/wetlab/september#pillTab3" style="display:block;float:right;">Next &gt;&gt;</A></DIV></DIV><DIV class="row"><DIV class="nine columns"><H3>August 1st - 4th</H3>
							It was confirmed that RFP was successfully inserted downstream of mtrB in our arsenic reporter parts in Shewanella. Site-directed mutagenesis of the nah operon failed. We are still trying to insert the nah operon into the mobility backbone (OriT) to prepare for conjugation into Shewanella later on.
							<DIV class="panel" style="background:white;margin-top: 20px;"><H6>Daily Details:</H6><B> August 1st, Wednesday </B>
								In the morning, Dylan noted that 5 of the 7 transformations from Tuesday produced plates with colonies. Those that worked were three Anderson series constitutive promoters (with downstream mRFP) in pBBRBB and the versions of our two arsenic reporters with mRFP downstream of mtrB. In the evening, Swati and Dylan prepared overnight cultures from all transformants and JG700, both to set up conjugations to get plasmid into Shewanella and to screen isolated plasmids for successful ligation. We will use these constructs to characterize the activity of the arsenic-sensitive promoter in Shewanella with respect to known constitutive promoter strengths. The two transformations that failed were versions of the salicylate reporter.
								After staring at plates, Dylan ran the nah+p29, oriT+p17c, p31c, and nah operon (from p20) digestions on a gel, extracted the fragments of interest, and then dephosphorylated the backbone. The isolated digestion products were used later used by Steven, who set up two overnight ligations – one to put the nah operon, with a constitutive promoter, on a pSB3C5 backbone with an added oriT sequence, and one to put the nah operon into the MCS of pSB1C3 for subsequent site directed mutagenesis.
								While Dylan ran his gel, Caleb miniprepped from the JG700+SAL and S31 cultures. After quantification, SAL plasmid isolated from JG700 was submitted for sequencing in order to confirm successful conjugation.
								Meanwhile, Mark decided that he enjoyed making DH5a electrocompetent stocks on Tuesday so much that he had to make more WM3064 electrocompetent stocks today. Starting from subcultures that Dylan had prepared, he prepared the stocks, and was assisted by Danielle.
								<B>From left: ladder, oriT+p17c digest (E+X), nah + p29 digest (E+S), p31c digest (E+S), p20 PCR digest (E+S):</B><B> August 2nd, Thursday </B>
								Caleb and Dylan created conjugation plates of p14RFP, p16RFP, and p25-29BB as well as miniprepped from the same plasmids in WM3064 (see: strain list). Dylan, Mark, and Tina then desalted and transformed SAL2 and SALRFP into WM3064 as well as p31c_nah and oriT_nah_p17c into DH5a. Tina and Dylan then quantified the minipreps from earlier in the day. Dylan also digested pBBRBB and p33k with EcoRI &amp; PstI to put lac inducible mtrB into the pBBRBB backbone.
								<B> August 3rd, Friday </B>
								After running a digestion of p33k, the lac inducible mtrB part from the parts registry, we discovered that the part does not include mtrB! We should have seen a ~3.6kb band on our gel after digesting with EcoRI and PstI, but instead isolated a band ~1.5kb. After checking the sequencing on the parts registry we discovered this was because the part didn't include mtrB, so we won't be able to use it as a positive control for inducible mtrB.
								Sequencing of p37k-p41k from WM3064 came back good, meaning we can put these control parts, with RFP downstream of our various reporters, into Shewanella via conjugation. Hence, conjugation was carried out.
								<B>Digests of p11 and p33 (E+P):</B><B> August 4th, Saturday </B>
								This week, it was confirmed that RFP was successfully inserted downstream of mtrB in our arsenic reporter in Shewenella.
								Site-directed mutagenesis of the nah operon failed. We continued trying to insert the nah operon into the mobile backbone (OriT) to prepare for conjugation into Shewanella.
								</DIV></DIV></DIV><DIV class="row"><DIV class="nine columns"><H3>August 5th - 11th</H3>
							This week, we continued our site directed mutagenesis of nah_p31c and the addition of RFP to our SAL reporters. Sequencing of the arsenic reporter has been unsuccessful so far. We also decided that we need more quantitative data to characterize our parts.
							<DIV class="panel" style="background:white;margin-top: 20px;"><H6>Daily Details:</H6><B>August 5th, Sunday </B>
								Swati yet again was saddled with a massive number of <A href="https://static.igem.org/mediawiki/2012/3/35/Ezna_Miniprep.pdf">minipreps </A>, since her magic fingers are able to cast yield-increasing spells on minipreps. She miniprepped: SAL2 from WM3064, p37-41k from JG700 (one arsenic reporter with cut sites flanking the RBS, another arsenic reporter without the cut sites, and three different Anderson series promoters with mRFP1 downstream on a pBBRBB backbone), oriT p29nah_p17c from DH5a, and nah_p31c from DH5a. And indeed her yields were impressive, massive, gargantuan! Good job Sorceress Swati.
								We will <A href="https://static.igem.org/mediawiki/2012/0/03/Sequencing_Preparation.pdf "> sequence </A> SAL2 and oriT p29nah_p17c to see if we should continue to conjugation, and sequence nah_p31c to see if we can start <A href=" https://static.igem.org/mediawiki/2012/a/a5/Site_Directed_Mutagenesis.pdf "> site-directed mutagenesis </A> to get a biobrick-compatible nah operon on the biobrick backbone. We will sequence or PCR to confirm p37k-p41k's conjugation into Shewanella.
								<B> August 6th, Monday </B>
								Caleb and Dylan amplified the two different arsenic promoters (p37k and p38k) and ran them on a gel alongside p14 and p16 (the arsenic reporters without mRFP1 downstream) to confirm successful insertion of RFP downstream of mtrB in our arsenic reporter parts in Shewenella. Band lengths appeared in expected places. Dylan is worried that we will not be able to use PCR to definitely confirm that p39-41k worked, so we submitted these for <A href="https://static.igem.org/mediawiki/2012/0/03/Sequencing_Preparation.pdf "> sequencing </A>. Unfortunately, sequencing failed, perhaps because random gunk from Shewanella added noise to the process.
								Tina did a PCR cleanup of the nah operon PCR. Dylan also ran a gel of nah operon PCRs to make sure that there was no mispriming. The gel looked good, so he submitted the PCRs for sequencing.
								<B>Successful insertion of RFP downstream of mtrB in arsenic reporter strains in Shewanella:</B><B> August 7th, Tuesday </B>
								JG1531 overnight culture didn't grow. We suspect that the plate Dylan picked from is dead. We'll have to go back to the glycerol stocks if we want to play with mtrE. Dylan set up a continuous flow M4 reactor in morning. Caleb started a liquid culture of MR-1 with which to inoculate the reactor. Checked sequencing results of nah stuff. oriT thing was bad. nah_p31c was good.
								Caleb and Dylan proceeded with <A href=" https://static.igem.org/mediawiki/2012/a/a5/Site_Directed_Mutagenesis.pdf "> site-directed mutagenesis </A> of nah_p31c, attempting to get rid of the first PstI cutsite in the nah operon. After digestion with DpnI, we purified using the E.Z.N.A. MicroElute kit and quantified. DNA was split into three directions: First, Danielle and Dylan set up another mutagenic PCR using the digestion product as template to get rid of the second internal cut site (we expect lower mutation efficiency because template DNA is not methylated). Second, we transformed DH5a with the mutated plasmid. Third, Danielle and Chie <A href="https://static.igem.org/mediawiki/2012/a/af/Double_Digest.pdf "> digested </A> both the purified – and hopefully mutated – plasmid and un-mutated plasmid with PstI. Dylan ran these digestions on a gel, along with supercoiled plasmid as a control. Unfortunately, the (hopefully) mutated plasmid never showed up on the gel.
								In the evening, we didn't observe growth in the MR-1 culture, so Dylan set up another culture just in case Shewanella was dead and not lazy.
								<B>Looking for mutagenesis – from left – ladder, unmutated nah, mutated nah, supercoil control:</B><B> August 8th, Wednesday </B>
								Caleb and Dylan performed another <A href="https://static.igem.org/mediawiki/2012/a/af/Double_Digest.pdf "> digest </A> to check if the second <A href="https://static.igem.org/mediawiki/2012/a/a5/Site_Directed_Mutagenesis.pdf "> site-directed mutagenesis </A> (that was supposed to get rid of the second PstI internal cut site within the nah operon) worked. Unfortunately, there wasn't enough DNA to see anything when visualized on a gel. They decided to proceed with electroporation into DH5a just in case - however, to use Caleb's terminology, cells exploded - there were probably too many salts in the solution, which causing arcing and a PBBHTTTZZZ! of cells all over the cuvette.
								Dylan re-digested p29 nah and oriT p17c to redo a ligation that would allow for conjugation into Shewanella later on. Dylan also re-digested p27 (Anderson series promoter with RFP downstream) and SAL to redo a ligation to create a plasmid with SAL-RFP.
								Dylan also submitted several samples for <A href="https://static.igem.org/mediawiki/2012/0/03/Sequencing_Preparation.pdf "> sequencing </A> to make sure that three Anderson promoters with RFP downstream in JG700 and SAL2 (the salicylate reporter without the BAMHI cut site) in WN3064 and JG700 were in the correct sequence.
								<B>Seeing if mutagenesis worked; nothing there:</B><B> August 9th, Thursday </B>
								Today Dylan chilled with his mom. Well, he tried to. :) Despite the eternal bonds that bind mother and son, plus the 3 billion miles she flew to see him, he still couldn't resist coming in to lab to open packages! And then got sucked into a long discussion of what had to be done for the rest of the day - scientific endeavors taking him away yet again from filial duties. Swati and Claire bonded over dry ice, and Caleb regaled us with tales of dry ice bombs gone awry.
								Swati and Caleb miniprepped the first mutagenesis of nah_p31c that had been transformed into DH5a yesterday. They then digested it with PstI-HF to see if the mutation was successful. Unfortunately the mutagenesis failed! We will start over from nah_p31c and lower the annealing temperature at 60degC. The Stratagene kit, which uses PFU ultra, asks for 60degC, but because we are using our own boot-leg protocol with Phusion we did the first try at 65degC, as Phusion usually calls for a higher annealing temperature than the theoretically calculated value. For this second try we will stick to the 60degC suggested by Stratagene and see if we get better results. Also, called NEB to find out if after PCR with Phusion, DpnI will still have activity in the following digestion step in Buffer 4, or if we need to clean up the PCR before digesting with DpnI - they said PCR clean up isn't needed.
								We also ran digestions for making the nah operon on a backbone with a mobility gene and the salicylate reporter (w/ BamHI cutsite) with RFP downstream. We cut our mobile backbone, OriT in p17c (pSB3C5), with EcoRI and XbaI, while cutting the nah operon (p29nah) with EcoRI and SpeI. The nah operon with mobility gene must be constructed so that we may conjugate into Shewy and start testing our salicylate reporter. The salicylate reporter and p27a (from the Anderson series), with RFP, were cut with SpeI and PstI. The salicylate/RFP part will be used for troubleshooting the salicylate reporter. Digestion products were run on a gel, extracted, and quantified. Swati then dephosphorylated backbones and ligated both parts.
								Sequencing for p39-41k didn't look good, and neither did the salicylate reporter w/BamHI cutsite miniprepped from JG700. The sequencing for salicylate reporter w/ BamHI in WM3064, however, looked good, suggesting that conjugation may not have been as efficient as we had hoped. After a pow-wow we decided not to sequence more colonies, as we are hoping some may be good, and more importantly that if we use qPCR to get quantitative characterization data, we won't need to use the RFP parts. The plates will stay in the fridge as a back-up plan.
								In other news: Claire cried because it was difficult to update the notebook with a week's worth of work. Mark's dedication to the notebook is laudable and impressive. Good job team for doing so much! My head can't even comprehend the magnitude of your endeavors.
								<B>Unsuccessful mutagenesis (test with Pst1 cutsite):</B><B>Digestion products from left – ladder, p27, p29nah, oriTp17c, SAL:</B><B> August 10th, Friday </B>
								Caleb miniprepped a plasmid with mtrE from JG1531, and just for kicks, miniprepped from the &quot;exploded cells&quot; from August 8th. Suprisingly, he ended up getting decent yields for both, showing that the electroporation worked despite arcing. Caleb then digested the nah operon of the miniprep with PSTI and NotI to check if the mutagenesis was a success. It was run on a gel alongside a PCR of the Anderson series promoters with RFP downstream and SAL2 from Shewanella (to check if SAL2 and the promoters were sucessfully conjugated after sequencing on Monday failed). Unfortunately, bands did not appear where we expected them to.
								Steven and Spencer performed a PCR to get mtrE out of the Gralnick (JG700) plasmid.
								<B>From left lane: ladder, p39 PCR, p40 PCR, p41 PCR, nah-p31c (P+N digestion), SAL2 PCR:</B><B> August 11th, Saturday </B>
								Spencer and Steven checked the nah operon PstI and NotI digest (because yesterday's gel ran weirdly) and their mtrE PCR from the previous day on a gel. Unfortunately, bands did not appear where we expected them to.
								<B>mtrE PCR product (left) and checking mutagenesis again (right):</B></DIV></DIV></DIV><DIV class="row"><DIV class="nine columns"><H3>August 12th - 18th</H3>
							This week, we:
							1)Performed the neccessary work for getting our reporters into the pSB1C3 backbone required for submission. This included the neccessary digestions of our reporters out and ligations into the submission plasmid. Confirmation of successful transformation and ligation will be performed in the following weeks.
							2)Confirmed successful transformation of Shewanella with our arsenic and salicylate reporters via colony PCR.
							3)Performed the neccessary digestions and ligations to have mRFP downstream of our reporter system so as to do an additional fluorescent testing of the reporters to serve a second form of confirmation of increased transcription in the presence of our toxins.

							<DIV class="panel" style="background:white;margin-top: 20px;"><H6>Daily Details:</H6><B> August 12th, Sunday </B>
								Weekend overview: It looks like the mtrE primers are actually working to amplify the part out of JG1531. We'll be putting mtrE in p31c, both so that we may submit the novel part to the registry, and to begin site-directed mutagenesis.
								We'll use mtrE mutagenesis as something of a control for nah operon mutagenesis (to see if the large size of the plasmid and nah operon is the problem).
								It also looks like nah operon mutagenesis hasn't worked. We'll put this on hold, and continue with ligation of p29nah into oriT_p17c once desalting paper arrives.
								Also, we'll have to redo things to confirm p39-41, SAL2 in JG700. What was done over the weekend looks weird.
								<B> August 13th, Monday </B>
								Dylan ran four simultaneous Phusion PCRs with an annealing temperature of 67degC and an extension time of 45sec to amplify mtrE. The gel looked good, with no mispriming and one band ~2.2kb. Claire did PCR clean up of the product, which will eventually be cut and ligated into p31c (see: strain list).
								Because the results from the weekend's screenings were confusing, we also ran PCRs of p39-41k (miniprepped from JG700 strains) to confirm whether conjugation into Shewanella was successful. These plasmids contain mRFP under the control of constitutive promoters of varying strengths. After confirmation, we will use these parts to characterize our inducible promoters in Shewanella.
								Dylan also started a liquid culture of JG700 + SAL to be inoculated into a continuous flow reactor with M4 media. With this setup, we will begin characterizing our reporter strains, initially in response to salicylate.
								Dylan performed a Phusion PCR to amplify mtrE, and ran a gel to check for product. There was a single band so no mispriming was occurring.
								Claire performed a double digest of the p14k and p16k with XbaI and PstI HF. This is for the construction of our final biobricks to be submitted to the registry.
								<B>PCR amplification of mtrE:</B><B>p39 and p41 verification:</B><B> August 14th, Tuesday </B>
								Steven performed a double digest of mtrE (EcorI &amp; SpeI), Sal (SpeI &amp; PstI), Sal2 (SpeI &amp; PstI), and p26a (SpeI &amp; PstI).
								Steven performed a ligation of p29nah + oriTp17c, and also a ligation of p27 digest + Sal. Dylan transformed colonies with the ligation.
								<B> August 15th, Wednesday </B>
								Dylan ran a gel and Claire performed gel extraction of mRFP to put downstream of Sal reporters. They also checked to see if site-directed mutagenesis was successful for the nah-p31c. Found the digest of mRFP and p26 looked good, but the mutagenesis looked strange.
								Dylan performed a taq PCR to confirm the presence of Sal2 in Shewanella. However no band was present in the gel ran afterwards, so perhaps colony PCR is the next best step.
								Dylan also ran a gel of the p14k, p16k p31 digests (all cut with XbaI and PstI) for the set-up for forming the biobricks.
								Steven performed a colony Phusion PCR of four colonies containing the p4k plasmid to again test for the presence of mtrE. Spencer then did PCR clean up
								Spencer performed a double digest of p4kdlg and p27a cutting with EcoRI &amp; SpeI followed by running a gel and extracting the digests.
								Spencer picked colonies from the previous ligations of the Sal, Sal2, p31c biobrick plasmids for miniprepping on Thursday.
								This morning Dylan noted that we may be getting a detectable basal level of current - around 6mA - in a continuous flow setup. This would make it possible to bypass the addition of mtrE to the salicylate sensing part, and still be able to distinguish Shewanella not detecting salicylate from dead Shewanella. He then added salicylate to the reactor innoculated with our salicylate reporting strain, bringing the concentration to 10uM. Later in the day, he noted that the current had risen to around 9mA, which is promising: our strain may be working! Claire set up four hour digestions with XbaI and PstI to put our arsenic parts, p14 and p16, into the iGEM backbone for submission.
								Dylan also transformed a salicylate reporter with mRFP downstream into DH5alpha and WM3064, as well as the nah operon with the p29 promoter in a p17c backbone with an oriT. The second of these could be conjugated into Shewanella if transformed successfully. The first, the salicylate reporter with mRFP downstream, is a fluorescent version of the salicylate reporter. The fluorescent versions of our reporters is an alternative to qPCR to measure the relative expression level of mtrB: if we know what mRFP under the control of a constitutive promotor in Shewanella looks like, we can add arsenic or naphthalene/salicylate to our fluorescent reporter parts until reaching the same level of expression. Then, we can correlate this concentration of arsenic or salicylate to a certain strength of induced expression.
								In case the fluorescent SAL reporter was not successfully ligated, we are preparing more mRFP, SAL, and SAL2: - In the morning Claire also cleaned up the digestions of SAL, SAL2 and mtrE PCR. However, due to a silly mistake on her part involving wash buffer and absolute ethanol (absolutely missing, to be precise) we will redo the digestions in order to get more DNA for putting these reporters into the iGEM backbone, and to put mRFP downstream. She then set up four hour digestions with XbaI and PstI to put our arsenic parts, p14 and p16, into the iGEM backbone for submission. - We digested mRFP with SpeI and PstI to make the part with mRFP downstream of mtrB in the salicylate reporters. We have already successfully put mRFP downstream of mtrB in our arsenic reporters, but it did not seem to work in the arsenic reporter. The gel for mRFP looked as expected, with the band for the insert slightly shorter than 900bp. However, on the same gel the the mutagenesis trial of the nah operon, digested with PstI and NotI, showed only one band. If mutagenesis had not worked, we would expect more than two bands, and if it had worked we would still expect two bands. Therefore we are unsure how to interpret this result, which we have seen twice now, but are going to start over with site-directed mutagenesis of the nah operon. We will also visualize the unmutated nah operon, digested with PstI and NotI, on a gel to see if that looks like we expect it to.
								Finally, it should be noted with jubilance that Claire was reunited with her umbrella today! The joy in the lab at this event was palpable and will be remembered for years to come.
								<B>mRFP digest and additional (failed) attempt at confirming mutagenesis:</B><B>From left – ladder, failed SAL2, p14, p16, p31digestions:</B><B> August 16th, Thursday </B>
								Dylan performed a miniprep of the Sal, Sal2, and p31c reporters for us to do sequencing for confirmation of successful ligations. These will hopefully be the biobricks we send in.
								In the afternoon Dylan performed a dephosphorylation of the p31c (cut with XbaI, PstI) followed by a gel of the mtrE cut with EcoRI &amp; SpeI as well as mRFP cut with SpeI and PstI.
								A 4 hour digestion was also performed by Claire to get Sal parts into pSB1C3 for the SAL, SAL2, and p31 parts. Steven then ran a gel of the digest products.
								Steven also performed a colony Taq PCR of the Sal2 reporter to confirm its presence in Shewanella.
								Today, we continued work to get our engineered reporters into pSB1C3 for submission to the parts registry. In the morning, Dylan miniprepped both versions of our salicylate reporters, along with more pSB1C3 from overnight cultures of the corresponding DH5a strains. Following quantification, he set up digestions of each miniprep with XbaI and PstI. (We decided to cut with XbaI because we discovered an extra base pair between the NotI and XbaI cutsites in the normal BioBrick prefix, an artifact of a previous team's work). Caleb also dephosphorylated previously isolated pSB1C3 backbone to prevent self-ligation, while Steven gel extracted the salicylate reporter inserts and more pSB1C3 backbone (to be quantified Friday morning).
								<B>mtrE and mRFP digests:</B><B>Digestions of SAL, SAL2, and p31:</B><B> August 17th, Friday </B>
								Claire performed a ligation for the construction of the Sal_RFP, and Sal2_RFP plasmids. Transformation was then performed immediately after.
								</DIV></DIV></DIV><DIV class="row"><DIV class="nine columns"><H3>August 19th - 25th</H3>
							Conjugating nah into Shewy:
							Colony PCR was done to 1) make sure the nah operon was in the cells 2) troubleshoot the colony PCR protocol After imaging, we discovered that the colonies did contain the nah operon, and that our colony PCR protocol was good.
							<DIV class="panel" style="background:white;margin-top: 20px;"><H6>Daily Details:</H6><B> August 19th, Sunday </B>
								In the morning, Dylan came to lab to look at the transformant plates from Saturday. There were colonies on all but one of the plates: It looks like either the ligation to put mtrE into p31c or the subsequent electroporation failed. It is of note that the competent DH5α cells that Dylan used for the mtrE electroporation were from an older stock – one which we'd previously determined to confer lower transformation efficiency. Because we used up all of the newer stock – with markedly better efficiency – we will make more electrocompetent DH5α in the coming week.
								Because all other transformations seemed to work, we grew up overnight cultures from single colonies on each plate. We will miniprep from these cultures and screen the minipreps for the correct insert. If confirmed, we will have successfully have put both versions of our arsenic reporters, as well as both versions of our salicylate reporters inside p31c (i.e., pSB1C3), as is required for registry submission. We will also have successfully put mRFP downstream of mtrB on SAL2, which will be used for characterization of the salicylate sensitive promoter in S. oneidensis.
								<B>Gel showing failure of nahoriT p17c:</B><B> August 22th, Wednesday </B>
								Conjugating nah into Shewy: Caleb, Tina, and a guest Louis miniprepped 5 plasmids containing nah operons with constituitive promoters from a WM3064 plate and miniprepped one plasmid that we previously confirmed as having the nah operon (without a promoter) to use as a positive control in the future.
								Claire performed miniprep of grown cultures of the Sal_RFP and Sal2_RFP to submit for sequencing.
								<B> August 24th, Friday </B>
								Jim performed the digestion of the nah operon+ p29a and the p31c backbone for ultimate ligation in the creation of the last biobrick. Ligation had to be redone, as sequencing results of last week's ligations looked great except for the nah operon biobrick.

								Conjugating nah into Shewy:
								We wanted to accomplish two things:
								•	Amplify the nah operon with a promoter using primers with appended cut sites.
								•	Optimize colony PCR for future Cornell iGEMers (past attempts have failed.)
								To accomplish both of these objectives, we did Phusion PCRs using plasmids containing nah operons with constituitive promoters as a template and a Phusion colony PCR using the same colonies we miniprepped the plasmid from. Because we expect the miniprep PCRs to work, doing the colony PCRs in parallel will allow us to determine whether our colony PCR technique is working or not. For our colony PCR purposes, we want to be able to screen for colonies while being certain we can go back and miniprep the same PCRed colony. Our colony PCR method is as follows:
								NOTE: We later determined there was a cataclysmic flaw in the following method, can you figure it out? (wink wink nudge nudge...)
								•	Label colonies of interest and correspondingly label microfuge tubes
								•	Add 50 uL ddH2O to each microfuge tube
								•	Using pipette tip, dip into colonies of interest then dip into labelled microfuge tubes and swirl around to release cells
								•	Use the the ddH2O+cell mixture as the DNA template for a PCR - instead of adding plain ddH2O to the PCR to bring it up to volume, add the ddH2O+cell mixture
								•	Confirm your PCR had desired results with a DNA gel, then use the ddH2O+cell mixture from the corresponding microfuge tube the PCR was done on to inoculate a culture or plate.
								<B> August 25th, Saturday </B>
								Jim performed an overnight ligation of the nah operon pSB1C3 for transformation on Sunday.
								Conjugating nah into Shewy:
								Caleb, Tina, and a guest Louis ran a gel of the miniprep and colony parallel Phusion PCRs. All of the miniprep PCRs worked, but only a couple colony PCRs worked. We believe this is due to not having added enough cells to the colony PCR tube because some of the colonies were very tiny.
								<B>Top: Positive control from colony, colony test 1, colony test 2, colony test 3, colony test 4, colony test 5, miniprep 5, Benchtop 1kb; Bottom: Benchtop 1kb, miniprep 1, miniprep 2, miniprep 3, miniprep 4, positive control miniprep:</B></DIV></DIV></DIV><DIV class="row last-ele"><DIV class="nine columns"><H3>August 26th - 31st</H3>
							Operation “get nah into Shewy continued this week.” We were not able to confirm through colony PCR that we have nah in WM3064. Team fluorescence was able to conjugate the SAL_mRFP constructs into Shewy. We continued site-directed mutagenesis of the nah operon but it has not been working like we expected.
							<DIV class="panel" style="background:white;margin-top: 20px;"><H6>Daily Details:</H6><B> August 26th, Sunday </B>
								Conjugating nah into Shewy:
								Caleb miniprepped two plasmids - one contained the nah operon preceded with a constitutive promoter and the other is to become a backbone containing the origin of transfer required for conjugation and a gene conveying chloramphenicol resistance. The purified plasmids were digested to extract the nah operon and to cut open the backbone - these digests were then run on a gel. Tina gel extracted the backbone then accidentally threw away the rest of the gel that contained the nah operon fragments.
								
								Site-directed mutagenesis: Swati ran a mutagenic PCR of nah_p31c, then digested with DpnI to remove template DNA and column purified the resultant DNA. <B> August 27th, Monday </B>
								Conjugating nah into Shewy:
								Caleb gel extracted the nah operon containing fragments from yesterday - despite having been in the garbage overnight, the gel extraction worked! Caleb and Tina began a 24 hour 16 degrees Celsius ligation using molar rations of nah to backbone of 3:1 and 1:1.
								Fluorescence tests: Claire started a JG700 + SAL2 culture and tried to do a PCR of SAL2_mRFP with sequencing primers to see if the band produced would correspond to the length of SAL2 plus the length of mRFP. Unfortunately, due to a mishap involving p10s vs. p2s and not checking what volume her pipette was set to, she ended up with a 70uL total volume and PCR failed.
								
								Site-directed mutagenesis: Swati transformed the (hopefully) mutated DNA into DH5a. <B> August 28th, Tuesday </B>
								Conjugating nah into Shewy:
								Caleb and Tina de-salted the 3:1 and 1:1 ligations, then transformed them into a DAP-requiring conjugation strain WM3064 via electroporation. The electroporated cells were plated on DAP + Cm plates.
								Fluorescence tests: JG700 once again takes a long time to grow, so it wasn’t until the evening that Claire miniprepped SAL2 from JG700. Tomorrow we will run another PCR and see if we have SAL2_mRFP, and SAL2 in JG700.
								
								Site-directed mutagenesis: Swati's transformation did not work. After checking the PCR product and a PstI digest thereof on a gel, she discovered that the PCR was not working with high enough efficiency to produce a band on a gel. Upon consulting with Didi, our trusty post-doctoral adviser, she then restarted the mutagenic process with a PCR; this time, with a lower primer concentration and lower denaturation and annealing times. <B> August 29th, Wednesday </B>
								Conjugating nah into Shewy:
								Caleb checked the transformation plates after ~14 hours in a 37 degrees Celsius incubator and noticed colonies, but the colonies were barely visible indicating very slow growth of the cells.
								Fluorescence tests: Set up a Taq PCR of SAL_mRFP, SAL2_mRFP and SAL2, with an extension time of 4.5 minutes and annealing temperature of 68degC. SAL2_mRFP has a band corresponding only to the length of SAL2, without mRFP, so we need to redo this ligation. However, the band from SAL_mRFP was the correct length (a little more than 4kb), so this ligation was successful and we can conjugate into Shewenella. Finally, the JG700 miniprep is the correct length for SAL2 (~3.8kb), so SAL2 has been successfully conjugated into Shewenella!
								
								Site-directed mutagenesis: Swati digested her second mutagenic PCR with DpnI, column purified, and transformed into DH5a. <B> August 30th, Thursday </B>
								Conjugating nah into Shewy:
								Tina checked the growth of yesterday's colonies and tried doing a colony PCR of each colony and a PCR of a positive control (same product, same primes, but in a different purified plasmid). The primers were designed to sit on the backbone and face into the oriT region and the site where we wanted to insert the nah operon. A PCR amplicon indicating successful ligation of the nah operon into the backbone would be approximately 10-11kb, while a failed ligation would yield a ~700 bp amplicon. Caleb later ran a DNA gel of the colony PCRs and nothing but smears showed up in every lane of the gel except the positive control which didn't even have a smear (Caleb was so disappointed that he didn't post the picture). Since the positive control didn't show up, we concluded the PCR failed. Caleb started overnight cultures of the remaining cells+ddH2O used for the colony PCR.
								
								Site-directed mutagenesis: Alas, Swati's second attempt at mutagenesis was yet again fruitless. Concerned that the template may have been causing the problem, she performed a NotI digest and ran it on a gel alongside the latest mutagenic PCR. The template was fine; the PCR was, again, invisible. Swati grew extremely puzzled as to how her purified PCR product could have significant concentration upon quantification but yield no visible bands on a gel. She decided to try a third time, this time with longer denaturation and annealing times, but still the same concentration of primers.
								<B>From left lane: Ladder, SAL2_mRFP (1), SAL2_mRFP (2), SAL2:</B><B> August 31st, Friday </B>
								Conjugating nah into Shewy: Caleb noticed the overnight cultures didn't grow and concluded the cells must have died due to the cells being stored in an extremely hypotonic solution (ddH2O). He re-did colony PCRs of the colonies, a colony with a positive control, and a positive control of purified plasmid, but modified the colony PCR protocol. Instead of using the cells + ddH2O as template DNA, Caleb dipped the pipette tip into the colony, then dip and swirled it in the PCR rxn tube before dip and swirling into labeled microfuge tubes with 50 uL LB media. The microfuge tubes were stored in a 4 degrees Celsius fridge. A DNA gel of the colony PCRs showed that the ligation of nah into the backbone failed - as seen in the accompanying gel picture, our positive control worked (indicating the PCR conditions were fine), but the test colonies all only had 700 bp amplicons.

								Fluorescence tests: Claire found some old SAL2 and mRFP digested with PstI and SpeI already, so she decided to retry the ligation. She did a 40min ligation at room temp, and continued it overnight in the 4degC fridge, which the NEB rep said could increase efficience. However, since she forgot to dephosphorylate SAL2, we will never know if this room temp and overnight in the fridge combo would have been effective because it self ligated.
								
								Site-directed mutagenesis: Swat ran a DpnI digestion of half of the previous day's PCR product, in an attempt to rule out the possibility that DpnI was functioning incorrectly. <B>From left lane: Ladder, positive control miniprepped plasmid, colony 1, colony 2, colony 3, colony 4, colony 5, colony 6, colony 7, colony 8, positive control colony:</B></DIV></DIV></DIV><DIV class="row last-ele"><DIV class="three columns"><A href="https://2012.igem.org/Team:Cornell/notebook/wetlab/july#pillTab3">&lt;&lt; Previous</A></DIV><DIV class="three columns offset-by-six"><A href="https://2012.igem.org/Team:Cornell/notebook/wetlab/september#pillTab3" style="display:block;float:right;">Next &gt;&gt;</A></DIV></DIV></LI></UL></DIV><DIV class="printfooter">
Retrieved from &quot;<A href="http://2012.igem.org/Team:Cornell/notebook/wetlab/august">http://2012.igem.org/Team:Cornell/notebook/wetlab/august</A>&quot;</DIV></DIV></DIV><DIV id="footer-box" class="noprint"><DIV id="footer"><UL id="f-list"><LI id="t-recentchanges"><A href="/Special:RecentChanges" title="Recent changes">Recent changes</A></LI><LI id="t-whatlinkshere"><A href="/Special:WhatLinksHere/Team:Cornell/notebook/wetlab/august" title="List of all wiki pages that link here [j]" accesskey="j">What links here</A></LI><LI id="t-recentchangeslinked"><A href="/Special:RecentChangesLinked/Team:Cornell/notebook/wetlab/august" title="Recent changes in pages linked from this page [k]" accesskey="k">Related changes</A></LI><LI id="t-specialpages"><A href="/Special:SpecialPages" title="List of all special pages [q]" accesskey="q">Special pages</A></LI><LI><A href="/Special:Preferences">My preferences</A></LI></UL></DIV><DIV id="footer"><UL id="f-list"><LI id="t-print"><A href="/wiki/index.php?title=Team:Cornell/notebook/wetlab/august&amp;printable=yes" title="Printable version of this page [p]" accesskey="p">Printable version</A></LI><LI id="t-permalink"><A href="/wiki/index.php?title=Team:Cornell/notebook/wetlab/august&amp;oldid=291470" title="Permanent link to this revision of the page">Permanent link</A></LI><LI id="privacy"><A href="/2012.igem.org:Privacy_policy" title="2012.igem.org:Privacy policy">Privacy policy</A></LI><LI id="disclaimer"><A href="/2012.igem.org:General_disclaimer" title="2012.igem.org:General disclaimer">Disclaimers</A></LI></UL></DIV></DIV></DIV></DIV></BODY></HTML>