<HTML xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en" dir="ltr">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki  ltr ns-0 ns-subject page-Team_Cornell_testing_project_drylab_6"><DIV id="globalWrapper"><DIV id="top-section"><DIV id="p-logo"><A href="/Main_Page" title="Main Page">&quot;
	    </A></DIV><DIV id="menubar" class="left-menu noprint"><UL><LI class="selected"><A href="/Team:Cornell/testing/project/drylab/6">Page               </A></LI><LI class="new"><A href="/wiki/index.php?title=Talk:Team:Cornell/testing/project/drylab/6&amp;action=edit&amp;redlink=1">Discussion               </A></LI><LI><A href="/wiki/index.php?title=Team:Cornell/testing/project/drylab/6&amp;action=edit">View source               </A></LI><LI><A href="/wiki/index.php?title=Team:Cornell/testing/project/drylab/6&amp;action=history">History               </A></LI><LI style="color:#808080;cursor:default">teams</LI></UL></DIV><DIV class="right-menu noprint" id="menubar"><UL><LI id="pt-login"><A href="/wiki/index.php?title=Special:UserLogin&amp;returnto=Team:Cornell/testing/project/drylab/6" title="You are encouraged to log in; however, it is not mandatory [o]" accesskey="o">Log in</A></LI></UL></DIV><DIV id="search-controls" class="noprint"><FORM action="/Special:Search" id="searchform">Â </FORM></DIV></DIV><DIV id="content"><H1 class="firstHeading">Team:Cornell/testing/project/drylab/6</H1><DIV id="bodyContent"><H3 id="siteSub" class="noprint">From 2012.igem.org</H3><DIV class="row header off-canvas"><DIV class="twelve columns centered header no-pad" id="header-wrapper"><UL class="nav-bar"><LI class="pg-home"><A href="https://2012.igem.org/Team:Cornell">Home</A></LI><LI class="has-flyout pg-project pg-project_background pg-project_wetlab pg-project_drylab pg-project_hprac pg-project_gallery pg-project_safety"><A href="https://2012.igem.org/Team:Cornell/project">Project</A></LI><LI class="pg-project"><A href="https://2012.igem.org/Team:Cornell/project">Overview</A></LI><LI class="pg-project_background"><A href="https://2012.igem.org/Team:Cornell/project/background">Background</A></LI><LI class="pg-project_wetlab"><A href="https://2012.igem.org/Team:Cornell/project/wetlab">Wet Lab</A></LI><LI class="pg-project_drylab"><A href="https://2012.igem.org/Team:Cornell/project/drylab">Dry Lab</A></LI><LI class="pg-project_hprac"><A href="https://2012.igem.org/Team:Cornell/project/hprac">Human Practices</A></LI><LI class="pg-project_gallery"><A href="https://2012.igem.org/Team:Cornell/project/gallery">Gallery</A></LI><LI class="pg-project_safety"><A href="https://2012.igem.org/Team:Cornell/project/hprac/safety">Safety</A></LI></UL><LI class="has-flyout pg-notebook pg-notebook_wetlab pg-notebook_drylab pg-notebook_strainlist"><A href="https://2012.igem.org/Team:Cornell/notebook">Notebook</A></LI><LI class="pg-notebook"><A href="https://2012.igem.org/Team:Cornell/notebook">Overview</A></LI><LI class="pg-notebook_wetlab"><A href="https://2012.igem.org/Team:Cornell/notebook/wetlab">Wet Lab</A></LI><LI class="pg-notebook_drylab"><A href="https://2012.igem.org/Team:Cornell/notebook/drylab/june">Dry Lab</A></LI><LI class="pg-notebook_strainlist"><A href="https://2012.igem.org/Team:Cornell/notebook/strain_list" target="_blank">Strain List</A></LI><LI class="pg-protocols"><A href="https://2012.igem.org/Team:Cornell/protocols">Protocols</A></LI><LI class="has-flyout pg-team pg-team_bios pg-team_sponsors"><A href="https://2012.igem.org/Team:Cornell/team">Team</A></LI><LI class="pg-team"><A href="https://2012.igem.org/Team:Cornell/team">Overview</A></LI><LI class="pg-team_bios"><A href="https://2012.igem.org/Team:Cornell/team/bios">Bios</A></LI><LI><A href="https://igem.org/Team.cgi?year=2012&amp;team_name=Cornell" target="_blank">Profile</A></LI><LI class="pg-team_sponsors"><A href="https://2012.igem.org/Team:Cornell/team/sponsors">Sponsors</A></LI><LI class="has-flyout pg-outreach pg-outreach_presentations pg-outreach_panels pg-outreach_awareness"><A href="https://2012.igem.org/Team:Cornell/outreach">Outreach</A></LI><LI class="pg-outreach"><A href="https://2012.igem.org/Team:Cornell/outreach">Overview</A></LI><LI class="pg-outreach_presentations"><A href="https://2012.igem.org/Team:Cornell/outreach/presentations">Research Presentations</A></LI><LI class="pg-outreach_panels"><A href="https://2012.igem.org/Team:Cornell/outreach/panels">Panel Discussions</A></LI><LI class="pg-outreach_awareness"><A href="https://2012.igem.org/Team:Cornell/outreach/awareness">Public Awareness</A></LI><LI class="pg-badge"><A href="https://2013.igem.org/Team:Cornell" style="margin-top:-50px">2012: SAFE BET</A></LI><LI class="pg-badge_home"><A href="http://igem.engineering.cornell.edu">Team Website</A></LI><LI class="pg-badge_2013"><A href="https://2013.igem.org/Team:Cornell">2013: Organofoam</A></LI><LI class="pg-badge_2011"><A href="https://2011.igem.org/Team:Cornell">2011: Biofactory</A></LI><LI class="pg-badge_2010"><A href="https://2010.igem.org/Team:Cornell">2010: OMG OMVs</A></LI><LI class="pg-badge_2009"><A href="https://2009.igem.org/Team:Cornell" style="font-size:20px !important">2009: Cadmium Sensor</A></LI></DIV></DIV><DIV class="row"><DIV class="two columns"><UL class="side-nav"><LI><H6>Dry Lab</H6></LI><LI><A href="https://2012.igem.org/Team:Cornell/testing/project/drylab/1">Overview</A></LI><LI><A href="https://2012.igem.org/Team:Cornell/testing/project/drylab/2">Specifications</A></LI><LI><A href="https://2012.igem.org/Team:Cornell/testing/project/drylab/3">Design</A></LI><LI><A href="https://2012.igem.org/Team:Cornell/testing/project/drylab/4">Modeling</A></LI><LI class="active"><A href="https://2012.igem.org/Team:Cornell/testing/project/drylab/6">Animation</A></LI></UL></DIV><DIV class="ten columns team-bios-container"><DIV class="row" id="ugd-members"><DIV class="twelve columns"><H2 class="centered">Animation</H2></DIV></DIV><DIV class="row"><DIV class="nine columns"><H3>Protein Structures</H3>
				When making a 3D animation, it's important to define several features early on in the process to have an overview of where the animation is going and the style in which its actors will be represented. In this case, we wanted to depict the <A href="https://2012.igem.org/Team:Cornell/testing/project/wetlab/2">Mtr pathway</A> in Shewanella oneidensis, so one of the first things I decided was to have accurate protein models.<A href="http://www.rcsb.org/pdb/home/home.do">RCSB PDB</A> (Protein Data Bank) is an amazing resource that has virtually every published protein structure. Unfortunately, there is only one Mtr protein whose structure has been published, MtrF, whose role in the Mtr pathway is so peripheral that I chose not to represent it in the animation. This left me with two options: (1) find proteins that should be related, and use their structures, or (2) generate predicted structures. In the spirit of accuracy, I chose to generate predicted structures.
Using the gene sequences from <A href="http://www.ncbi.nlm.nih.gov">NCBI</A>, identified the open reading frame and translated the proteins with <A href="http://biologylabs.utah.edu/jorgensen/wayned/ape/">ApE</A>. I used two online tools for structure prediction: <A href="http://bioinf.cs.ucl.ac.uk/psipred/">PSIPRED Protein Structure Prediction Server</A>, which uses two neural networks to detect secondary structure by amino acid motifs, and <A href="http://swissmodel.expasy.org">SWISS-MODEL</A>, which has a tool that uses sequence alignments to find proteins with published structures that have regions likely to be homologous, then predicts structure based on the homologous region.
Unfortunately, a major shortcoming of SWISS-MODEL is that it only predicts structure in the homologous region; any amino acids outside of that have to be added back and folded by other means. For this purpose, I used <A href="http://pymol.org">PyMOL</A> to add back the missing residues, and the <A href="http://c4c.uwc4c.com/express_license_technologies/foldit">standalone version</A> of <A href="http://fold.it/portal/">Fold.It</A> to fold them. I used the secondary structure predictions from PSIPRED for these residues. For CymA, MtrA, and MtrC, this involved folding up to 50 residues at each end, which was simple, but for MtrB, believed to be a beta barrel protein, SWISS-MODEL only produced about half of the barrel, so I spent nearly a week <A href="https://static.igem.org/mediawiki/2012/2/25/CornellProteinFoldingIsHard.gif">manually folding</A> over 300 residues into the shape of the missing beta sheets.
				</DIV></DIV><DIV class="row"><DIV class="nine columns"><H3>Stylistic Decisions</H3>
Once I had the structures, the rest of the animation had to be determined. The second very important stylistic decision was to use a &quot;toon&quot; style of animation. While realistic lighting and shaders can look pretty awesome, they really don't convey information particularly well. If you've ever played a game where outlines or highlighting are used in some way to indicate an item or character of interest, it's very clear how much a simple step like that can enhance the accessibility of visual information. To this end, I decided to use ramp shaders, which color surfaces with sharp cutoffs, giving it an effect similar to cel-shading. However, features that cel shading would normally hide are still visible, such as bump maps, as they distort the boundaries between colors.
The third stylistic decision lay in how to represent election transfer. How does one really show events at the quantum level? They don't really have any set visual representation. The simplest way would be to show a small particle passing between electron carriers, but that would be pretty boring and difficult to see. Instead, I decided to represent electron transfer as lightning: it's bright, it's flashy, and it's dramatic. <A href="http://usa.autodesk.com/maya/">Autodesk Maya</A> also provides a very easy-to-use dynamic lightning tool, facilitating this decision greatly.
The final stylistic decision lay in how to show the differences between our system when induced and not induced. I opted for a side-by-side splitscreen in the interest of keeping the animation time-efficient, and to allow direct comparison of the two as they're occurring, rather than sequentially, which relies on the viewer remembering the events of the previous sequence. Furthermore, because most elements remain the same, showing them sequentially would be repetitive.
				</DIV></DIV><DIV class="row"><DIV class="nine columns"><H3>Composition</H3>
Our team has experience in both <A href="http://www.luxology.com/modo/">Luxology Modo</A> and Autodesk Maya. In my experience, Modo tends to make things look nicer with default settings, and has some really great real-time preview rendering options, but Maya has many tools built in for more advanced effects, and the <A href="http://www.molecularmovies.com/toolkit/">mMaya</A> toolkit provides a lot of features for importing pdb files (the standard for protein structures) and adjusting various parameters for modeling their backbone and surface.
As for developing the actual animation, the first step was to make a timeline. We are using the animation to go along with the presentation of the Mtr pathway, so it was important to have timings that allow for easy real-time narration. I went over the presentation with Dylan Webster, recording timings for each of the steps, and converted those to frames.
After the timeline was set, I built models for the cell, the electrode, and imported models for intermediate electron carriers from PyMOL. Then, I made curves on which to animate the various elements, and cameras to follow each one. The scene camera moves between these cameras, switching off at their common points. Finally, I added polishing touches, such as dynamically randomized offsets (giving the appearance of brownian motion), rippling of the cell membranes, addition of bump maps, and and instancing of lipid models along the membrane surface.
				</DIV></DIV><DIV class="row"><DIV class="twelve columns"><H3>The Result</H3></DIV></DIV></DIV></DIV><DIV class="printfooter">
Retrieved from &quot;<A href="http://2012.igem.org/Team:Cornell/testing/project/drylab/6">http://2012.igem.org/Team:Cornell/testing/project/drylab/6</A>&quot;</DIV></DIV></DIV><DIV id="footer-box" class="noprint"><DIV id="footer"><UL id="f-list"><LI id="t-recentchanges"><A href="/Special:RecentChanges" title="Recent changes">Recent changes</A></LI><LI id="t-whatlinkshere"><A href="/Special:WhatLinksHere/Team:Cornell/testing/project/drylab/6" title="List of all wiki pages that link here [j]" accesskey="j">What links here</A></LI><LI id="t-recentchangeslinked"><A href="/Special:RecentChangesLinked/Team:Cornell/testing/project/drylab/6" title="Recent changes in pages linked from this page [k]" accesskey="k">Related changes</A></LI><LI id="t-specialpages"><A href="/Special:SpecialPages" title="List of all special pages [q]" accesskey="q">Special pages</A></LI><LI><A href="/Special:Preferences">My preferences</A></LI></UL></DIV><DIV id="footer"><UL id="f-list"><LI id="t-print"><A href="/wiki/index.php?title=Team:Cornell/testing/project/drylab/6&amp;printable=yes" title="Printable version of this page [p]" accesskey="p">Printable version</A></LI><LI id="t-permalink"><A href="/wiki/index.php?title=Team:Cornell/testing/project/drylab/6&amp;oldid=268119" title="Permanent link to this revision of the page">Permanent link</A></LI><LI id="privacy"><A href="/2012.igem.org:Privacy_policy" title="2012.igem.org:Privacy policy">Privacy policy</A></LI><LI id="disclaimer"><A href="/2012.igem.org:General_disclaimer" title="2012.igem.org:General disclaimer">Disclaimers</A></LI></UL></DIV></DIV></DIV></BODY></HTML>