"
Page
Discussion
View source
History
teams
Log in
Â 
Team:Cornell/Notebook/Salicylate reporter/July15
From 2012.igem.org
Home
Team
Project
Parts
Modeling
Notebook
Protocols
Safety
Attributions
Contents
1 July 15th-21st
1.1 July 15th
1.2 July 16th
1.3 July 17th
The salicylate reporter is analogous to the arsenic reporter, but with a salicylate-sensitive promoter. That is, in the presence of salicylate it will produce MtrB, completing the Mtr pathway and allowing Shewanella to produce current in our biosensor. In combination with the nah operon, this reporter will be able to detect naphthalene levels. Back to salicylate reporter week view
July 15th-21st
July 15th
Dylan noticed that we had colonies on the plates re-streaked from the original plates of conjugated S. oneidensis. He started overnight cultures of S20 (See: strain list ) in kanamycin so that we can sequence and make glycerol stocks. He also started overnight cultures of w.t. S. oneidensis to innoculate into new reactors in Riley Robb, which will be used as positive controls.
July 16th
The overnight cultures of S. oneidensis need more time to grow - overnight cultures of these leisurely bacteria should be started at noon the previous day, instead of in the evening as with speedy E. coli.
Success!
Dylan ran another Phusion PCR of the entire nah operon, since we got a lot of mispriming the first time we ran it. The thought was that we had run the previous PCR at the optimal temperature for Vent but used Phusion, so Dylan set up the new PCR correctly, with a single annealing temperature of 66degC and a lengthened final extension time of 15 minutes to account for the size of the nah operon (~10kb). Caleb ran a gel of the PCR and visualized a single band ~9.5kb. PCR of the nah operon out of p20, P. putida (See: strain list ) was successful! Tina and Swati gel extracted and quantified, extracting two samples at 38.8 ng/uL and 27.2 ng/uL.
Spencer miniprepped p14, our arsenic reporter part (arsR + mtrB w/ BamHI cutsite), from S20 (See: strain list ) for sequencing, recording yields of 44.3ng/uL & 35.8ng/uL (colony 1) and 50.6ng/uL & 53.5ng/uL (colony 2).
Dylan and Tina set up transformations of p15 and p16 (See: strain list ), the arsenic reporter part without a BamHI cutsite, so that we can conjugate S. oneidensis in the next few days. They also set up a transformation of the miniprepped BBa_J01003 with the oriT mobility gene. Our new method of transformation into S. oneidensis, using conjugation rather than electroporation, requires that all our plasmids have the mobility gene. However, the plasmid we were going to use for the nah operon (pSB3C5) has no mobility gene. Possible approaches to take are to clone oriT from iGEM kit plates into pSB3C5, or to PCR the mobility gene out of one of our own plasmids and clone that into pSB3C5. We will try to get oriT from kit plates first.
July 17th
Shweta set up a Phusion PCR of p21 (See: strain list ) to try again to extract the salicylate-sensitive promoter. Sequencing of p21 showed that there was a stem loop sitting upstream of the ENX biobrick cutsites, which wasn't expected and is most likely why our primers are not effective. Before redesigning primers, we are going to try PCR with the standard iGEM forward sequencing primer and the same reverse primer that we designed for p21.
Success!
Retrieved from " http://2012.igem.org/Team:Cornell/Notebook/Salicylate_reporter/July15 "
Recent changes
What links here
Related changes
Special pages
My preferences
Printable version
Permanent link
Privacy policy
Disclaimers
