<HTML xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en" dir="ltr">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki  ltr ns-0 ns-subject page-Team_Cornell_notebook_wetlab_july"><DIV id="globalWrapper"><DIV id="top-section"><DIV id="p-logo"><A href="/Main_Page" title="Main Page">&quot;
	    </A></DIV><DIV id="menubar" class="left-menu noprint"><UL><LI class="selected"><A href="/Team:Cornell/notebook/wetlab/july">Page               </A></LI><LI class="new"><A href="/wiki/index.php?title=Talk:Team:Cornell/notebook/wetlab/july&amp;action=edit&amp;redlink=1">Discussion               </A></LI><LI><A href="/wiki/index.php?title=Team:Cornell/notebook/wetlab/july&amp;action=edit">View source               </A></LI><LI><A href="/wiki/index.php?title=Team:Cornell/notebook/wetlab/july&amp;action=history">History               </A></LI><LI style="color:#808080;cursor:default">teams</LI></UL></DIV><DIV class="right-menu noprint" id="menubar"><UL><LI id="pt-login"><A href="/wiki/index.php?title=Special:UserLogin&amp;returnto=Team:Cornell/notebook/wetlab/july" title="You are encouraged to log in; however, it is not mandatory [o]" accesskey="o">Log in</A></LI></UL></DIV><DIV id="search-controls" class="noprint"><FORM action="/Special:Search" id="searchform">Â </FORM></DIV></DIV><DIV id="content"><H1 class="firstHeading">Team:Cornell/notebook/wetlab/july</H1><DIV id="bodyContent"><H3 id="siteSub" class="noprint">From 2012.igem.org</H3><DIV class="row header off-canvas"><DIV class="twelve columns centered header no-pad" id="header-wrapper"><UL class="nav-bar"><LI class="pg-home"><A href="https://2012.igem.org/Team:Cornell">Home</A></LI><LI class="has-flyout pg-project pg-project_background pg-project_wetlab pg-project_drylab pg-project_hprac pg-project_gallery pg-project_safety"><A href="https://2012.igem.org/Team:Cornell/project">Project</A></LI><LI class="pg-project"><A href="https://2012.igem.org/Team:Cornell/project">Overview</A></LI><LI class="pg-project_background"><A href="https://2012.igem.org/Team:Cornell/project/background">Background</A></LI><LI class="pg-project_wetlab"><A href="https://2012.igem.org/Team:Cornell/project/wetlab">Wet Lab</A></LI><LI class="pg-project_drylab"><A href="https://2012.igem.org/Team:Cornell/project/drylab">Dry Lab</A></LI><LI class="pg-project_hprac"><A href="https://2012.igem.org/Team:Cornell/project/hprac">Human Practices</A></LI><LI class="pg-project_gallery"><A href="https://2012.igem.org/Team:Cornell/project/gallery">Gallery</A></LI><LI class="pg-project_safety"><A href="https://2012.igem.org/Team:Cornell/project/hprac/safety">Safety</A></LI></UL><LI class="has-flyout pg-notebook pg-notebook_wetlab pg-notebook_drylab pg-notebook_strainlist"><A href="https://2012.igem.org/Team:Cornell/notebook">Notebook</A></LI><LI class="pg-notebook"><A href="https://2012.igem.org/Team:Cornell/notebook">Overview</A></LI><LI class="pg-notebook_wetlab"><A href="https://2012.igem.org/Team:Cornell/notebook/wetlab">Wet Lab</A></LI><LI class="pg-notebook_drylab"><A href="https://2012.igem.org/Team:Cornell/notebook/drylab/june">Dry Lab</A></LI><LI class="pg-notebook_strainlist"><A href="https://2012.igem.org/Team:Cornell/notebook/strain_list" target="_blank">Strain List</A></LI><LI class="pg-protocols"><A href="https://2012.igem.org/Team:Cornell/protocols">Protocols</A></LI><LI class="has-flyout pg-team pg-team_bios pg-team_sponsors"><A href="https://2012.igem.org/Team:Cornell/team">Team</A></LI><LI class="pg-team"><A href="https://2012.igem.org/Team:Cornell/team">Overview</A></LI><LI class="pg-team_bios"><A href="https://2012.igem.org/Team:Cornell/team/bios">Bios</A></LI><LI><A href="https://igem.org/Team.cgi?year=2012&amp;team_name=Cornell" target="_blank">Profile</A></LI><LI class="pg-team_sponsors"><A href="https://2012.igem.org/Team:Cornell/team/sponsors">Sponsors</A></LI><LI class="has-flyout pg-outreach pg-outreach_presentations pg-outreach_panels pg-outreach_awareness"><A href="https://2012.igem.org/Team:Cornell/outreach">Outreach</A></LI><LI class="pg-outreach"><A href="https://2012.igem.org/Team:Cornell/outreach">Overview</A></LI><LI class="pg-outreach_presentations"><A href="https://2012.igem.org/Team:Cornell/outreach/presentations">Research Presentations</A></LI><LI class="pg-outreach_panels"><A href="https://2012.igem.org/Team:Cornell/outreach/panels">Panel Discussions</A></LI><LI class="pg-outreach_awareness"><A href="https://2012.igem.org/Team:Cornell/outreach/awareness">Public Awareness</A></LI><LI class="pg-badge"><A href="https://2013.igem.org/Team:Cornell" style="margin-top:-50px">2012: SAFE BET</A></LI><LI class="pg-badge_home"><A href="http://igem.engineering.cornell.edu">Team Website</A></LI><LI class="pg-badge_2013"><A href="https://2013.igem.org/Team:Cornell">2013: Organofoam</A></LI><LI class="pg-badge_2011"><A href="https://2011.igem.org/Team:Cornell">2011: Biofactory</A></LI><LI class="pg-badge_2010"><A href="https://2010.igem.org/Team:Cornell">2010: OMG OMVs</A></LI><LI class="pg-badge_2009"><A href="https://2009.igem.org/Team:Cornell" style="font-size:20px !important">2009: Cadmium Sensor</A></LI></DIV></DIV><DIV class="row"><DIV class="two columns"><UL class="side-nav ntbk-nav"><LI><H6>Wet Lab</H6></LI><LI><A href="https://2012.igem.org/Team:Cornell/notebook/wetlab">Overview</A></LI><LI><A href="https://2012.igem.org/Team:Cornell/notebook/wetlab/bootcamp">Bootcamp</A></LI><LI><A href="https://2012.igem.org/Team:Cornell/notebook/wetlab/june">June</A></LI><LI class="active"><A href="https://2012.igem.org/Team:Cornell/notebook/wetlab/july">July</A></LI><LI><A href="https://2012.igem.org/Team:Cornell/notebook/wetlab/august">August</A></LI><LI><A href="https://2012.igem.org/Team:Cornell/notebook/wetlab/september">September</A></LI><LI><A href="https://2012.igem.org/Team:Cornell/notebook/wetlab/wrap_up">Wrap Up</A></LI></UL></DIV><DIV class="ten columns team-bios-container"><DL class="tabs pill" style="display:block;float:right;width:390px;margin: -10px -20px 10px 0px;"><DD class="active"><A href="#pillTab1">Weekly Update</A></DD><DD><A href="#pillTab2">Daily Details</A></DD><DD><A href="#pillTab3">Both</A></DD></DL><DIV class="row"><DIV class="twelve columns"><H2 class="centered">Wet Lab - July</H2></DIV></DIV><UL class="tabs-content"><LI class="active" id="pillTab1Tab"><DIV class="row last-ele" style="margin-top:20px;"><DIV class="three columns"><A href="https://2012.igem.org/Team:Cornell/notebook/wetlab/june#pillTab1">&lt;&lt; Previous</A></DIV><DIV class="three columns offset-by-six"><A href="https://2012.igem.org/Team:Cornell/notebook/wetlab/august#pillTab1" style="display:block;float:right;">Next &gt;&gt;</A></DIV></DIV><DIV class="row"><DIV class="nine columns"><H3>July 1st-7th</H3>
							A lot of troubleshooting occurred this week. Once we figured out that SYBR Green was causing our gels to run strangely, we switched to staining with EtBr after running the gel. This approach gave us better results. We also continued to try electroporation protocols for transforming Shewanella, and continued to work on PCRing out the salicylate-sensing region. <A href="#" class="technical-desc" for="#technical-desc1" style="display:block;margin-top:20px;">Daily Details</A><DIV class="hide-me panel" style="background:white;margin-top: 20px;" id="technical-desc1"><H6>Daily Details:</H6><B>July 1st, Sunday</B>
								Dylan and Caleb set up two gels for <A href="https://static.igem.org/mediawiki/2012/6/6c/DNA_Gel_Electrophoresis.pdf">electrophoresis</A>. Caleb's was 1% agarose in BIO-RAD Mini-Sub Cell system for continuation of ladder test using SYBR Green, containing NEB 100 bp ladder, NEB 2-log ladder, Promega 1 kb ladder and run at 100V. Dylan's was 1% agarose in Owl box using ethidium bromide, containing the nah operon PCR product from previous night, and run at 55 V.
								Our plates of DH5a transformed with p21 ligated into pBBRBB with mtrB grew only one colony, possibly contamination. Dylan ran a colony PCR and got a smear, suggesting that the PCR of p21 or the ligation did not work.
								Because we learned that our SYBR Green was causing ladder to run strangely, Dylan decided to redo a <A href="https://static.igem.org/mediawiki/2012/3/33/Vent_PCR.pdf">Vent PCR</A> to amplify the salicylate reporter region out of p21.
								Also, a liquid culture of JG700 was prepared, as well as replating of p21, p22, JG700, JG1220, JG1537, JG1219. (See: <A href="https://2012.igem.org/Team:Cornell/notebook/strain_list">Strain List</A>)
<B>EtBr stained gel of nah Operon PCR product ran at 55V:</B><B>SYBR green stained gel of ladder experiment:</B><B>July 2nd, Monday</B>
								Today, Dylan and Caleb ran a <A href="https://static.igem.org/mediawiki/2012/6/6c/DNA_Gel_Electrophoresis.pdf">electrophoresis</A> of a <A href="https://static.igem.org/mediawiki/2012/3/33/Vent_PCR.pdf">Vent PCR</A> of p21 (the PCR product being the salicylate reporter) at 55 V. Additionally, Caleb decided to run a control gel at 100 V to determine whether higher voltage was a factor in our previous ladder problems (in addition to the SYBR Green stock). While we determined that the higher voltage did not cause our ladder issues, we did not see any bands from the p21 PCR (gel image not included).
								Dylan prepared electrocompetent cells using the <A href=" https://static.igem.org/mediawiki/2012/6/6f/MyersMyers_Shew-E-poration.pdf "> Myers and Myers </A> protocol. Modified protocol using two 5mL cultures @ 4000g for 10 min. Washed with 2 mL sorbitol, resuspended in 100 uL sorbitol. First electroporation ts = 2.32 ms, second ts = 2.02 ms. Added 1 uL of plasmid (575.5 ng) to each cuvette. First used .60 V, then 0.55 V, both with R = 400 ohms.
								Mark and Danielle started liquid cultures of S1, S9, S10, S11, S15, S16, S18, S27 (See: <A href="https://2012.igem.org/Team:Cornell/notebook/strain_list">Strain List</A>)
<B>EtBr stained ladders:</B><B>July 3rd, Tuesday</B>
								Caleb <A href="https://static.igem.org/mediawiki/2012/3/35/Ezna_Miniprep.pdf">miniprepped </A> S16 (p15), S22 (p21), S9 (p8), S10 (p9), S11 (p10), S18 (p17), and S15 (p14). (See: <A href="https://2012.igem.org/Team:Cornell/notebook/strain_list">Strain List</A>
								) Instead of a single EB elution, Caleb did two elutions, 30 ul each. The double 30 ul elution turned out to be effective at recovering a usable amount of DNA in the second elution, so we're including it on our miniprep protocol for future minipreps.
								The rerun of our Gibson sequencing failed again, so we tried a more roundabout method of determining whether our 3 Gibson transformants have complete nah operons or not. We digested them each with BamHI, expecting specific fragment lengths on a gel electrophoresis.
								When Dylan and Mark were attempting to determine what volume of ethidium bromide should be used in the gel, they came to the conclusion that better images may result from staining the gel in dilute ethidium bromide after running the gel, rather than including the stain in the gel. This experimental first staining used 200 mL of water conaining 20 uL of 10 mg/mL ethidium bromide, with gentle agitation for 1 hour. This stained the bands well, but also provided a large amount of background and took a very long time.
								We only saw one band on each, all of them at about 3.6kb.
								Because no transformations from our previous competent freezer stocks were successful, Dylan decided to make a starter culture of Shewanella strain JG 700 in preparation for making new competent stocks using the <A href="https://static.igem.org/mediawiki/2012/f/f7/PNNL_Electrocompetent_Shewanella.pdf
								"> PNNL Protocol </A>. Swati and Tina used this starter culture to complete the preparation.
<B>Three Gibson transformants and three ladders with Ethidium Bromide stain after running the gel:</B><B>Gel with SYBR green:</B><B>July 4th, Wednesday</B>
								In the morning, Dylan, Swati, and Danielle prepared p8, p9, p10, p14, and p15 for <A href="https://static.igem.org/mediawiki/2012/0/03/Sequencing_Preparation.pdf "> sequencing </A> (from the minipreps that Caleb preformed the previous day).
								The team took the rest of the day off and had a barbeque at Buttermilk Falls. There was an excess of food, spontaneous song, and fireflies.
								<B>July 5th, Thursday </B>
								Upon arriving to the lab, Dylan dropped off the sequencing tubes we'd set up on the morning of the 4th for analysis. He also decided to do a Colony PCR of the potential transformant with the salicylate reporter plasmid. However, when setting up this reaction, Dylan realized that we'd used the wrong sequencing primers (we'd used the standard VF2 BioBrick primer instead of a custom primer for the pBBRBB backbone). Consequently, Mark and Dylan resubmitted p14 and p15 for sequencing while the colony PCR was ongoing. After visualizing the PCR products using <A href="https://static.igem.org/mediawiki/2012/6/6c/DNA_Gel_Electrophoresis.pdf"> DNA gel electrophoresis</A>, Dylan concluded that the colony we'd screened did not have our plasmid of interest, and was the result of contamination.
								Similarly, Dylan noticed small, evenly spaced colonies on all of the JG700 transformant plates from the <A href=" https://static.igem.org/mediawiki/2012/6/6f/MyersMyers_Shew-E-poration.pdf "> Myers and Myers </A> procedure we'd undertaken on the 2nd of July. Because none of the colonies looked red (as they should have if they were replicating pSB3C5), Dylan concluded that the chloramphenicol on the plates had degraded, and the colonies were resultant from untransformed cells.
								Because the colony PCR of the potential salicylate reporter didn't yield good results, Dylan and Youjin set up another <A href=" https://static.igem.org/mediawiki/2012/2/26/Phusion_PCR.pdf "> Phusion PCR </A> to amplify the salicylate-sensing region from p21. Simultaneously, Mark made SOB media for recovery after future transformations, as we'd just received the ingredients for the broth.
								Because we needed to submit for sequencing twice (because we used the wrong sequencing primers), Swati et al. set up liquid cultures of p21, p8, p9, p10, p17, and p14 in order to have more DNA in the freezer. We like having DNA in the freezer. (See: <A href="https://2012.igem.org/Team:Cornell/notebook/strain_list">Strain List</A>)  
<B>Result of the colony PCR:</B><B>July 6th, Friday</B>
								Caleb <A href="https://static.igem.org/mediawiki/2012/3/35/Ezna_Miniprep.pdf">miniprepped </A> p21, p8, p9, p10, p17 from the liquid cultures set up the previous night. (See: <A href="https://2012.igem.org/Team:Cornell/notebook/strain_list">Strain List</A>
								) We couldn't miniprep p14, since â after checking notes from the previous night â we learned that the culture was made with the wrong antibiotic (chloramphenicol instead of kanamycin).
								Some sequencing results came back, however the Gibson results are still confusing.
								Also, a <A href="https://static.igem.org/mediawiki/2012/6/6c/DNA_Gel_Electrophoresis.pdf"> gel</A>  was run for the p21 PCR product.
<B>Gel showing the failure of the p21 PCR:</B></DIV></DIV></DIV><DIV class="row"><DIV class="nine columns"><H3>July 8th - 14th</H3>
							The focus of this week was to transfer our arsenic sensing plasmids to Shewanella strain JG700 via congugation with E. coli strain WM3064. After struggling to electroporate control plasmids, we consulted with Dr. Gralnick who reccomended proceeding with conjugation and kindly provided E. coli strain WM3064. This strain is auxotrophic for 2,6-Diaminopimelic Acid (DAP), which makes the strain useful for selecting against non-Shewanella conjugants.
							Another major decision this week was to give up on Gibson assembly - our sequencing results suggested that strange things had happened, and since we already had PCRed the full nah operon out of P. putida, we decided proceeding with Gibson was not an efficient use of time. We were also unsure as to why p21's PCR was repeatedly failing, and decided to submit the part from the registry for sequencing, to try to troubleshoot what was going wrong.
							<A href="#" class="technical-desc" for="#technical-desc7" style="display:block;margin-top:20px;">Daily Details</A><DIV class="hide-me panel" style="background:white;margin-top: 20px;" id="technical-desc7"><H6>Daily Details:</H6><B>July 9th, Monday </B>
								Dylan performed a <A href=" https://static.igem.org/mediawiki/2012/2/26/Phusion_PCR.pdf "> Phusion PCR </A> with a single annealing temperature of 59 degrees Celsius using primers 27 and 28 for the final time. In the future, new primers will be used. The purpose of this PCR was to amplify p21 (See: <A href="https://2012.igem.org/Team:Cornell/notebook/strain_list">Strain List</A>
								). The PCR product was analyzed using <A href="https://static.igem.org/mediawiki/2012/6/6c/DNA_Gel_Electrophoresis.pdf">agarose gel electrophoresis</A> to determine if the desired 1.127 kb fragment is present. We talked to Dr. Jeff Gralnick about electroporating Shewanella, and his recommendation was to try conjugation instead. To transform our plasmids into Shewanella with conjugation, we had asked Dr. Jeff Gralnick for WM3064 E. coli for which plates containing 2,6-Diaminopimelic Acid (DAP) are required. In preparation to receive this strain, the plates with DAP and DAP/kanamycin were prepared.
								<B>July 10th, Tuesday</B>
								Caleb <A href="https://static.igem.org/mediawiki/2012/3/35/Ezna_Miniprep.pdf">miniprepped </A>
								strains containing Gibson products and the arsenic promotors without BamHI cutsites: plasmids p8, p9, p10, p15, and p16 (See: <A href="https://2012.igem.org/Team:Cornell/notebook/strain_list">Strain List</A>
								). Also, our new freezer arrived. We cleaned and welcomed our new friend.
								The WM3064 came in and we plated from the agar stab. Dylan set up two 18 mL/hr continuous flow reactors using 10x diluted LB and innoculated one of the reactors with JG700 and one with wild type Shewanella oneidensis MR-1.

								<B>July 11th, Wednesday</B>
								Dylan started a WM3064 subculture in the morning before meeting Swati, Shweta, Caleb, and Danielle at the Boyce Thompson Institute for Plant Research to give a presentation both on our project and on the broader applications of synthetic biology to the energy industry to a group of high school teachers from New York state. After the presentation, Dylan and Danielle prepared plates containing DAP/chloramphenicol and DAP/ampicillin to grow transformed WM3064 on, while Mark continued preparing <A href="https://static.igem.org/mediawiki/2012/5/57/Making_Ecoli_Electrocompetent.pdf "> electrocompetent </A> stocks of WM3064 with the subculture that Dylan had started.
								We transformed things later that day.
								<B>July 12th, Thursday</B>
								Shweta prepared a glycerol stock of WM3064 for future use, and Dylan plated some remaining transformed WM3064 hoping to get more colonies of the successfully transformed cells.
								Dylan and Mark submitted p8, p9, and p10 for sequencing using two different primers for each plasmid.
								The previous night's p21 PCR seemed to have failed, even with the new primers. Based on the <A href="https://static.igem.org/mediawiki/2012/6/6c/DNA_Gel_Electrophoresis.pdf">gel </A>run by Claire, then analyzed by Dylan and Caleb.
<B>Gel showing failed p21 PCR product:</B><B>July 13th, Friday</B>
								Today began with Dylan and Caleb setting up conjugations between our transformed donor E. coli and S. oneidensis.
								The sequencing results for p8, p9 and p10 all came back as failed, so it was concluded that the Gibson assembly failed. There were five locations on the nah operon at which sequencing could begin, of which we considered all but the fourth. Sequencing at the first location resulted in a sequence that corresponded to the fifth sequencing location, with some sequence before it and all of the expected sequence after it. The fifth sequencing location produced the expected sequence. Then the middle two regions which we considered all resulted in failed sequencing. This has led us to the conclusion that segments that are required in the operon are missing, and thus the assembly failed. Dylan and Caleb began attempting to look into the possibility of this being a cause, and some potential solutions.
								In discussing why p21 PCR seems to continue failing, the idea of having p21 sequenced arose. Dylan prepared the sample and submitted it for sequencing. Meanwhile, Mark set up yet another <A href=" https://static.igem.org/mediawiki/2012/2/26/Phusion_PCR.pdf "> Phusion PCR</A> of p21, this time with a higher annealing temperature of 61 degrees Celsius and a decreased extension time of 18 seconds. The goal was to prevent the mispriming which seemed to have occurred in the previous day's attempt.
								Later that day, Dylan set up yet another PCR with a decreased annealing temperature of 55 degrees Celsius in order to empirically narrow-in on the optimal annealing temperature for the amplification of the salicylate sensing region from p21.
								<B>July 14th, Saturday</B>
								Swati ran a <A href="https://static.igem.org/mediawiki/2012/6/6c/DNA_Gel_Electrophoresis.pdf">gel </A>of 5uL of the p21 PCR that Mark set up yesterday. After staining, the DNA ladder was well visualized but there were no bands in the p21 lane, suggesting that the PCR did not work. In a fit of rage, Swati yelled at Maneesh for going to get a drink of water - yet another innocent victim of failed PCR. Then, after a full team meeting, we all came together for a dumpling party. Noodles, dumplings, and gnocchi of all kinds were eaten by the voracious team. In the spirit of true, unwavering scientific inquiry, we conducted a novel experiment whilst feasting on these delectable treats. The result: fried Dorito dumplings are delicious.
								</DIV></DIV></DIV><DIV class="row"><DIV class="nine columns"><H3>July 15th - 21st</H3>
							For this week, we began to conjugate our constructs into Shewanella and Dylan began seeding our reactors with WT Shewanella and with p14 to begin collecting data. Weâve also started work with the Anderson series of constitutive promoters for future work in anticipation of constructing control plasmids with constitutively produced MtrB.
							<A href="#" class="technical-desc" for="#technical-desc2" style="display:block;margin-top:20px;">Daily Details</A><DIV class="hide-me panel" style="background:white;margin-top: 20px;" id="technical-desc2"><H6>Daily Details:</H6><B>July 15th, Sunday </B>
								Dylan noticed that we had colonies on the plates re-streaked from the original plates of <A href=" https://static.igem.org/mediawiki/2012/0/0f/Shewanella_Conjugation.pdf
								">conjugated </A>
								S. oneidensis. He started overnight cultures of S20 (See: <A href="https://2012.igem.org/Team:Cornell/notebook/strain_list">Strain List</A>
								) in kanamycin so that we can <A href="https://static.igem.org/mediawiki/2012/0/03/Sequencing_Preparation.pdf "> sequence </A> and make glycerol stocks. He also started overnight cultures of w.t. S. oneidensis to inoculate into new reactors in Riley Robb, which will be used as positive controls.
								<B>July 16th, Monday</B>
								Dylan ran another <A href=" https://static.igem.org/mediawiki/2012/2/26/Phusion_PCR.pdf "> Phusion PCR </A> of the entire nah operon, since we got a lot of mispriming the first time we ran it; we believe this was because we had used the optimal annealing temperature for Vent Polymerase (55ï°C) with Phusion Polymerase. Dylan set up the new PCR correctly, with an annealing temperature of 66ï°C and a lengthened final extension time of 15 minutes to account for the size of the nah operon (~10kb). Caleb ran a gel of the PCR and visualized a single band ~9.5kb. PCR of the nah operon out of p20, P. putida (See: strain list) was successful! Tina and Swati gel extracted and quantified, extracting two samples at 38.8 ng/uL and 27.2 ng/uL.
								Spencer <A href="https://static.igem.org/mediawiki/2012/3/35/Ezna_Miniprep.pdf">miniprepped </A> p14, our arsenic reporter part (arsR + mtrB w/ BamHI cutsite), from S20 (See: <A href="https://2012.igem.org/Team:Cornell/notebook/strain_list">Strain List</A>
								) for sequencing, recording yields of 44.3ng/uL &amp; 35.8ng/uL (colony 1) and 50.6ng/uL &amp; 53.5ng/uL (colony 2) from quantification. Dylan and Tina set up transformations of p15 and p16 (See: strain list), the arsenic reporter parts without a BamHI cutsite, so conjugations with S. oneidensis can be done in the next few days. They also set up a transformation of the miniprepped BBa_J01003 with the oriT mobility gene. Because of our lack of success with electroporation, we are planning on trying <A href=" https://static.igem.org/mediawiki/2012/0/0f/Shewanella_Conjugation.pdf
								">conjugation </A> of our constructs into S. oneidensis. However, this requires that all our plasmids have the mobility gene. pSB3C5, the plasmid we are going to use for the nah operon has no mobility gene. Thus, we are going to try two things: to clone oriT from iGEM kit plates into pSB3C5, and to amplify the mobility gene out of one of our own plasmids and clone that into pSB3C5.
<B>Success!:</B><B>July 17th, Tuesday </B>
								Shweta set up a <A href=" https://static.igem.org/mediawiki/2012/2/26/Phusion_PCR.pdf "> Phusion PCR </A> of p21 (nahR and Psal) to try and extract the salicylate-sensitive promoter again. Sequencing of p21 showed a stem loop sitting upstream of the ENX biobrick cutsites, which was not expected and likely the reason for our unsuccessful PCRs. Before redesigning primers, we are going to try PCR with the standard iGEM forward sequencing primer and the same reverse primer that we designed for p21.
<B>Success!:</B><B>July 18th, Wednesday </B>
								Dylan set up p15k, p16k conjugation plates from overnight cultures of transformed WM3064 and JG700. Dylan let these plates incubate for 8 hours, and then streaked for single colonies on kan plates with E.coli and Shewanella controls. Caleb <A href="https://static.igem.org/mediawiki/2012/3/35/Ezna_Miniprep.pdf">miniprepped </A>
								p24a (BioBrick part with oriT) from overnight cultures. He then desalted an overnight ligation of the nah operon in pSB1C3 and transformed into DH5a. Dylan also trekked over to Riley Robb to set up a reactor for inoculation tomorrow. That night, Spencer set up overnight cultures of p25a, p28a, and p31c.
								<B>July 19th, Thursday </B>
								Only observed 2 possible colonies from transformants of yesterday's ligation (nah operon in pSB1C3). We restreaked from one of these colonies along with a DH5a control.
								We got single colonies on the JG700+p15 and JG700+p16 plates, and no growth on control plates. Thus, our tentative/optimistic conclusion is that the conjugation was successful. Dylan and Caleb made reference plates of the picked colonies, and grew up liquid cultures from same colonies for sequencing.
								Dylan and Caleb then <A href="https://static.igem.org/mediawiki/2012/3/35/Ezna_Miniprep.pdf">miniprepped </A> p25a, p28a, p31c from overnight cultures set up by Spencer and made glycerol stocks of S25, S28, and S31. That afternoon, Dylan also inoculated a reactor in Riley Robb with S20, our engineered arsenic reporter strain.
								We then set up a digestion of the p21 PCR product along with our miniprepped p14 with EcoRI and AscI. We purified the p21 PCR digestion with Omega Bio-Tek E.Z.N.A. MicroElute DNA Clean-Up Kit. We dephosphorylated p14 digestion with Antarctic Phosphatase, and ran the entire mixture on a gel. That night, we <A href=" https://static.igem.org/mediawiki/2012/4/46/EZNA_Gel_Extraction.pdf
								pdf "> gel extracted </A>the 6kb band with our Qiagen gel extraction kit and set up overnight cultures of a p26a, 027a, p29a, p30a (our Anderson series constitutive promoters), and p14k.
<B>Dephosphorylated p14 digestion:</B><B>July 20th, Friday </B>
								This morning Shweta and Tina <A href="https://static.igem.org/mediawiki/2012/3/35/Ezna_Miniprep.pdf">miniprepped </A>
								p26a, p27a, p29a, p30a (Anderson series constitutive promoters), and p14k from overnight cultures. Dylan set up a 30 minute room temperature ligation to connect the salicylate reporter with digested p21 PCR product and p14 (isolated yesterday, quantified today). Because Shewanella grows more slowly than E.coli, Claire did a second miniprep of p15k and p16k from JG700 in order to submit the plasmid for <A href="https://static.igem.org/mediawiki/2012/0/03/Sequencing_Preparation.pdf "> sequencing </A> to confirm that conjugation was successful. In anticipation of the failure of Dylan's ligation, Swati set up 3 <A href=" https://static.igem.org/mediawiki/2012/2/26/Phusion_PCR.pdf "> Phusion PCR</A> reactions in parallel, each with the same template (p21), and repeated the parameters that had proven successful (no mispriming) previously. Dylan also set up overnight cultures of the strain with the nah operon in pSB1C3, S18 (DH5a + pSB3C5/p17c), and WM3064+p14.
								</DIV></DIV></DIV><DIV class="row"><DIV class="nine columns"><H3>July 22nd - 28th</H3>
							The goal of this week was to transform DH5Î± with nah operon (p20 PCR + p25a, p27a, or p29a).
							<A href="#" class="technical-desc" for="#technical-desc3" style="display:block;margin-top:20px;">Daily Details</A><DIV class="hide-me panel" style="background:white;margin-top: 20px;" id="technical-desc3"><H6>Daily Details:</H6><B>July 22nd, Sunday </B>
								Danielle and Dylan <A href="https://static.igem.org/mediawiki/2012/a/af/Double_Digest.pdf ">digested </A> p14(Arsenic reporter) and p21(Salicylate sensing region) for ligation. Swati ran a <A href=" https://static.igem.org/mediawiki/2012/2/26/Phusion_PCR.pdf "> Phusion PCR </A> to amplify p21. Then Dylan ran a <A href="https://static.igem.org/mediawiki/2012/6/6c/DNA_Gel_Electrophoresis.pdf">gel electrophoresis</A> and did <A href=" https://static.igem.org/mediawiki/2012/4/46/EZNA_Gel_Extraction.pdf
								pdf "> gel extraction </A> to isolate p21 PCR digest and p14 backbone.
<B>PCR confirmation of p21:</B><B>Digest of p21 PCR an p14:</B><B>July 23rd, Monday </B>
								Dylan quantified the gel extractions, quite successfully with a newly invented protocol. Then he dephosphorylated p14, and submitted some DNA for <A href="https://static.igem.org/mediawiki/2012/0/03/Sequencing_Preparation.pdf "> sequencing </A>. At one point, the Magical Graduated Cylinder of Elmira (500 mL) fell from the sky and shattered in the sink. Some voodoo was clearly in the air. It appears that the ligation products from the day before were either not created or not successfully transformed, as the plates contained no colonies.
								Caleb and Tina prepared more kanamycin and chloramphenicol plates while Mark desalted Dylan's ligation product. Then Dylan, Mark, and Tina transformed DH5Î± with the ligation product, p33k, p14k, and p31c (See: <A href="https://2012.igem.org/Team:Cornell/notebook/strain_list">Strain List</A>).

								<B>July 24th, Tuesday </B>
								In the morning, Dylan did <A href=" https://static.igem.org/mediawiki/2012/2/26/Phusion_PCR.pdf "> Phusion PCR </A> of p20(nah operon). Then Caleb and Dylan did a <A href="https://static.igem.org/mediawiki/2012/a/af/Double_Digest.pdf "> double digestion </A> of p20 PCR and p17c for ligation. Dylan ran a DNA <A href="https://static.igem.org/mediawiki/2012/6/6c/DNA_Gel_Electrophoresis.pdf">gel electrophoresis</A> to confirm that the previous PCR was successful. Steven performed gel extraction to isolate p17c backbone fragment.
<B>Digest of p17 and PCR of p20:</B><B>July 25th, Wednesday </B>
								Mark set up two <A href="https://static.igem.org/mediawiki/2012/a/a7/T4_Ligase_Ligation.pdf ">ligations </A> of a p20 PCR product and p17c, one at standard concentrations and one at very high concentrations. (See: <A href="https://2012.igem.org/Team:Cornell/notebook/strain_list">Strain List</A>
								) p20 PCR was cut at Xbio1 and Spe1, which have compatible sticky ends. This means that p20 PCR can be ligated into backbone in two directions, only one of which is useful. These were desalted and transformed by Mark and Dylan, along with p33k. Meanwhile, Tina and Chie set up <A href="https://static.igem.org/mediawiki/2012/a/af/Double_Digest.pdf "> double digestions </A> of p24a, p20 PCR product, and <A href="https://static.igem.org/mediawiki/2012/f/fb/Single_Digest.pdf"> single digestions </A>
								of p25a, p27a, and p29a.
								Dylan ran two gels of the double and single digestions. The single digestions of the Anderson series promoters showed single bands, which ran slightly faster than the supercoiled DNA control. This may be because the supercoiled DNA was nicked. The gel of the oriT double digestion showed more long bands than expected â possibly a result of star activity â but had a band ~400bp which was interpreted to be the insert of interest.
<B>oriT insert / nah operon:</B><B>Constitutive promoters:</B><B>July 26th, Thursday </B>
								Caleb gel purified the digestion products of p24a, p20 PCR, p25a, p27a, and p29a. Danielle dephosphorylated products of p25a, p27a and p29a in preparation for <A href="https://static.igem.org/mediawiki/2012/a/a7/T4_Ligase_Ligation.pdf ">ligations </A>.
								<B>July 27th, Friday </B>
								In the morning, Claire <A href="https://static.igem.org/mediawiki/2012/3/35/Ezna_Miniprep.pdf">miniprepped </A> p11k, p17c, and p33k from the overnight cultures that Dylan had set up. Meanwhile, Dylan heat killed the ligase from the overnight <A href="https://static.igem.org/mediawiki/2012/a/a7/T4_Ligase_Ligation.pdf ">ligations </A>
								(p25a, p27a and p29a) (See: <A href="https://2012.igem.org/Team:Cornell/notebook/strain_list">Strain List</A>
								), while Caleb prepared to desalt the ligation mixtures before electroporation via drop dialysis. Shortly thereafter, Dylan and Caleb electroporated the four desalted mixtures into DH5Î±, allowing the cells to recover one hour in SOC before plating. If these transformations are successful, we will have cells that carry plasmids expressing the nah operon under the control of three different constitutive promoters with varying strength, as well as a cell line that carries a plasmid that confers chloramphenicol resistance, has a p15a origin of replication, and an origin of transfer. Eventually, the nah operon will be ligated as an insert into such a backbone.
								In the evening, Dylan and Tina started 30 mL overnight cultures of S24, S25, S27, S29, S15, and S17 to miniprep from the next morning. S24 carries a plasmid with an oriT, which will be <A href="https://static.igem.org/mediawiki/2012/3/35/Ezna_Miniprep.pdf">miniprepped </A> in case our previous ligation of an oriT into p17c failed. S25,27, and 29 all carry plasmids with Anderson series constitutive promoters of varying strength, which will be used to set up fluorescent controls to monitor mtrB expression levels. S15 and S17 carry our engineered arsenic reporter plasmids, which will be further modified to facilitate control studies in mtrB expression.
								After setting up the cultures, Dylan and Swati left Weill and headed over to Riley Robb where they assembled three new reactor setups in the Angenent lab. Two of these reactors will be run in continuous flow operation with a BioLogic potentiostat, while the other will be run in batch mode with a CH Instruments potentiostat. Tomorrow, one continuous flow and one batch reactor will be inoculated with wildtype Shewanella oneidensis MR-1, while the remaining continuous flow reactor will be inoculated with our engineered arsenic reporter strain, S20. The purpose of running the continuous flow reactors is to better define the maximum current output we can expect from our induced strains in response to analyte, and to repeat an experiment carried out on the CH potentiostat for basal activity from the uninduced arsenic reporter. Because we were unsure of the veracity of the data from our previous experiment using the CH potentiostat, we are running the batch reactor to see if the observed current response is what we'd expect from wildtype Shewanella.
								<B>July 28th, Saturday </B>
								Today, Swati did six <A href="https://static.igem.org/mediawiki/2012/3/35/Ezna_Miniprep.pdf">minipreps </A> in the morning and it took her four hours...and it paid off! The nanodrop told us that our highest yield was 812.6 ng/uL.
								While Swati was devoting her soul to the E.Z.N.A kit, Dylan continued to set up reactors in the Angenent Lab, making reference electrodes for the three reactors, and getting the pumps set up for the continuous flow reactors. After our weekly team meeting, he inoculated the reactor.
								Swati prepared overnight cultures and reference plates, while Danielle prepared kanamycin plates and did autoclave.
								In the evening, Dylan and Swati did <A href="https://static.igem.org/mediawiki/2012/a/af/Double_Digest.pdf "> digestions </A> for RFP controls and SAL2 reporter. Also, SAL reporter was transformed into WM3064.

								</DIV></DIV></DIV><DIV class="row last-ele"><DIV class="nine columns"><H3>July 29th - 31st</H3>
							SAL (the version of the salicylate reporter with the BamHI cutsite) was conjugated into JG700, and we created SAL2 (salicylate reporter without BamHI cutsite). We prepared constitutive controls and our reporters for fluorescence testing. We also continued cloning work to get the nah operon with a constitutive promoter ready to conjugate into JG700.
							<A href="#" class="technical-desc" for="#technical-desc4" style="display:block;margin-top:20px;">Daily Details</A><DIV class="hide-me panel" style="background:white;margin-top: 20px;" id="technical-desc4"><H6>Daily Details:</H6><B>July 29th, Sunday </B>
								Apparently, Swati has yet to get over her miniprep craze. While most of the wetlab hiked through Danby State Forest, Swati did more minipreps: p25a, oriT + PSB3C5, and nah operon + p29a. The cultures of S25 and nah p29 were bright magenta, while oriT in PSB3C5 was a pale tan. However, the last of these contained an unidentified crystalline substance, possibly a byproduct of producing the nah operon. Two other cell cultures also didn't grow, which was disheartening, as it meant there were less minipreps for Swati to do. Interspersed between the copious miniprepping were two gels to purify the digestions from the previous evening, all of which appeared to work! Dylan and Claire, returning from their nature trek, joined Swati in lab, finding Shweta and Eric along the way. While Dylan took over Swati's gel extraction, Swati, Shweta, and Claire discussed upcoming outreach events. As Ecologist Claire explained the feeding habits of Tomiculus troutus (AKA Tommy the trout), Eric began to feel left out and, in the process of showing off the website, discovered Dylan's odd love for typography. After spending roughly an hour accomplishing nothing, Dylan and Swati set up liquid cultures in preparation to put SAL into JG700. Dylan returned to his reactor-ridden lair for a short spell.
<B>07/29 gel:</B><B>July 30th, Monday </B>
								In the morning, Dylan set up conjugations between WM3064+SAL and JG700 while Caleb quantified the minipreps and gel extractions from Sunday. After quantification, he submitted nah+p29a and oriT+p17c for sequencing. Dylan and Mark dephosphorylated the digestions of SAL with SpeI &amp; PstI, p14k with SpeI &amp; PstI, p16k with SpeI &amp; PstI, p16k with EcoRI &amp; AscI, and p11k with EcoRI &amp; PstI. Tina and Danielle then set up ligations of mRFP+SAL, mRFP+p14, and mRFP+p16 to put mRFP downstream of the arsenic and salicylate reporters, in order to be able to characterize our induced promoter strengths by measuring fluorescence. We also ligated p16+SAL to make SAL2, the salicylate reporter without the BamHI cutsite. Finally, we ligated p25, p27, and p29 (the Anderson promoters with mRFP) into pBBRBB, in order to be able to put them into Shewanella.

								<B>July 31st, Tuesday </B>
								Mark made DH5Î± electrocompetent stocks. Dylan checked conjugation plates to put SAL into JG700. He then made overnight cultures of JG700+SAL and S31 (PSB1C3, as well as WM3064 to prepare competent stocks. Dylan autoclaved a reactor in Riley-Robb and set it up in batch with M4 media, and inoculated with wild-type S. oneidensis MR-1. The purpose was to repeat a previous batch experiment in order to be sure that it is something about feeding with LB that is decreasing current production. 
Sequencing results came back on nah+p29 and oriT+p17c and things looked good. Consequently, Dylan decided to digest nah+p29 with EcoRI and SpeI, and oriT+p17c with EcoRI and XbaI in order to eventually ligate the nah operon with a constitutive promoter into the p17c backbone, upstream from the oriT sequence (in order to allow the plasmid to be conjugated into Shewanella). Claire prepared these digestions, along with digestions of p31c and p20 PCR product (in order to get the nah operon into pSB1C3). 
Dylan and Caleb desalted the previous night's ligations and than transformed into WM3064, since we will want to get plasmid into Shewanella. 

								</DIV></DIV></DIV><DIV class="row last-ele"><DIV class="three columns"><A href="https://2012.igem.org/Team:Cornell/notebook/wetlab/june#pillTab1">&lt;&lt; Previous</A></DIV><DIV class="three columns offset-by-six"><A href="https://2012.igem.org/Team:Cornell/notebook/wetlab/august#pillTab1" style="display:block;float:right;">Next &gt;&gt;</A></DIV></DIV></LI><LI id="pillTab2Tab"><DIV class="row last-ele" style="margin-top:20px;"><DIV class="three columns"><A href="https://2012.igem.org/Team:Cornell/notebook/wetlab/june#pillTab2">&lt;&lt; Previous</A></DIV><DIV class="three columns offset-by-six"><A href="https://2012.igem.org/Team:Cornell/notebook/wetlab/august#pillTab2" style="display:block;float:right;">Next &gt;&gt;</A></DIV></DIV><DIV class="row"><DIV class="nine columns"><H3> July 1st â 7th </H3><B>July 1st, Sunday</B>
							Dylan and Caleb set up two gels for <A href="https://static.igem.org/mediawiki/2012/6/6c/DNA_Gel_Electrophoresis.pdf">electrophoresis</A>. Caleb's was 1% agarose in BIO-RAD Mini-Sub Cell system for continuation of ladder test using SYBR Green, containing NEB 100 bp ladder, NEB 2-log ladder, Promega 1 kb ladder and run at 100V. Dylan's was 1% agarose in Owl box using ethidium bromide, containing the nah operon PCR product from previous night, and run at 55 V.
							Our plates of DH5a transformed with p21 ligated into pBBRBB with mtrB grew only one colony, possibly contamination. Dylan ran a colony PCR and got a smear, suggesting that the PCR of p21 or the ligation did not work.
							Because we learned that our SYBR Green was causing ladder to run strangely, Dylan decided to redo a <A href="https://static.igem.org/mediawiki/2012/3/33/Vent_PCR.pdf">Vent PCR</A> to amplify the salicylate reporter region out of p21.
							Also, a liquid culture of JG700 was prepared, as well as replating of p21, p22, JG700, JG1220, JG1537, JG1219. (See: <A href="https://2012.igem.org/Team:Cornell/notebook/strain_list">Strain List</A>)
<B>EtBr stained gel of nah Operon PCR product ran at 55V:</B><B>SYBR green stained gel of ladder experiment:</B><B>July 2nd, Monday</B>
							Today, Dylan and Caleb ran a <A href="https://static.igem.org/mediawiki/2012/6/6c/DNA_Gel_Electrophoresis.pdf">electrophoresis</A> of a <A href="https://static.igem.org/mediawiki/2012/3/33/Vent_PCR.pdf">Vent PCR</A> of p21 (the PCR product being the salicylate reporter) at 55 V. Additionally, Caleb decided to run a control gel at 100 V to determine whether higher voltage was a factor in our previous ladder problems (in addition to the SYBR Green stock). While we determined that the higher voltage did not cause our ladder issues, we did not see any bands from the p21 PCR (gel image not included).
							Dylan prepared electrocompetent cells using the <A href=" https://static.igem.org/mediawiki/2012/6/6f/MyersMyers_Shew-E-poration.pdf "> Myers and Myers </A> protocol. Modified protocol using two 5mL cultures @ 4000g for 10 min. Washed with 2 mL sorbitol, resuspended in 100 uL sorbitol. First electroporation ts = 2.32 ms, second ts = 2.02 ms. Added 1 uL of plasmid (575.5 ng) to each cuvette. First used .60 V, then 0.55 V, both with R = 400 ohms.
							Mark and Danielle started liquid cultures of S1, S9, S10, S11, S15, S16, S18, S27 (See: <A href="https://2012.igem.org/Team:Cornell/notebook/strain_list">Strain List</A>)
<B>EtBr stained ladders:</B><B>July 3rd, Tuesday</B>
							Caleb <A href="https://static.igem.org/mediawiki/2012/3/35/Ezna_Miniprep.pdf">miniprepped </A> S16 (p15), S22 (p21), S9 (p8), S10 (p9), S11 (p10), S18 (p17), and S15 (p14). (See: <A href="https://2012.igem.org/Team:Cornell/notebook/strain_list">Strain List</A>
							) Instead of a single EB elution, Caleb did two elutions, 30 ul each. The double 30 ul elution turned out to be effective at recovering a usable amount of DNA in the second elution, so we're including it on our miniprep protocol for future minipreps.
							The rerun of our Gibson sequencing failed again, so we tried a more roundabout method of determining whether our 3 Gibson transformants have complete nah operons or not. We digested them each with BamHI, expecting specific fragment lengths on a gel electrophoresis.
							When Dylan and Mark were attempting to determine what volume of ethidium bromide should be used in the gel, they came to the conclusion that better images may result from staining the gel in dilute ethidium bromide after running the gel, rather than including the stain in the gel. This experimental first staining used 200 mL of water conaining 20 uL of 10 mg/mL ethidium bromide, with gentle agitation for 1 hour. This stained the bands well, but also provided a large amount of background and took a very long time.
							We only saw one band on each, all of them at about 3.6kb.
							Because no transformations from our previous competent freezer stocks were successful, Dylan decided to make a starter culture of Shewanella strain JG 700 in preparation for making new competent stocks using the <A href="https://static.igem.org/mediawiki/2012/f/f7/PNNL_Electrocompetent_Shewanella.pdf
							"> PNNL Protocol </A>. Swati and Tina used this starter culture to complete the preparation.
<B>Three Gibson transformants and three ladders with Ethidium Bromide stain after running the gel:</B><B>Gel with SYBR green:</B><B>July 4th, Wednesday</B>
							In the morning, Dylan, Swati, and Danielle prepared p8, p9, p10, p14, and p15 for <A href="https://static.igem.org/mediawiki/2012/0/03/Sequencing_Preparation.pdf "> sequencing </A> (from the minipreps that Caleb preformed the previous day).
							The team took the rest of the day off and had a barbeque at Buttermilk Falls. There was an excess of food, spontaneous song, and fireflies.
							<B>July 5th, Thursday </B>
							Upon arriving to the lab, Dylan dropped off the sequencing tubes we'd set up on the morning of the 4th for analysis. He also decided to do a Colony PCR of the potential transformant with the salicylate reporter plasmid. However, when setting up this reaction, Dylan realized that we'd used the wrong sequencing primers (we'd used the standard VF2 BioBrick primer instead of a custom primer for the pBBRBB backbone). Consequently, Mark and Dylan resubmitted p14 and p15 for sequencing while the colony PCR was ongoing. After visualizing the PCR products using <A href="https://static.igem.org/mediawiki/2012/6/6c/DNA_Gel_Electrophoresis.pdf"> DNA gel electrophoresis</A>, Dylan concluded that the colony we'd screened did not have our plasmid of interest, and was the result of contamination.
							Similarly, Dylan noticed small, evenly spaced colonies on all of the JG700 transformant plates from the <A href=" https://static.igem.org/mediawiki/2012/6/6f/MyersMyers_Shew-E-poration.pdf "> Myers and Myers </A> procedure we'd undertaken on the 2nd of July. Because none of the colonies looked red (as they should have if they were replicating pSB3C5), Dylan concluded that the chloramphenicol on the plates had degraded, and the colonies were resultant from untransformed cells.
							Because the colony PCR of the potential salicylate reporter didn't yield good results, Dylan and Youjin set up another <A href=" https://static.igem.org/mediawiki/2012/2/26/Phusion_PCR.pdf "> Phusion PCR </A> to amplify the salicylate-sensing region from p21. Simultaneously, Mark made SOB media for recovery after future transformations, as we'd just received the ingredients for the broth.
							Because we needed to submit for sequencing twice (because we used the wrong sequencing primers), Swati et al. set up liquid cultures of p21, p8, p9, p10, p17, and p14 in order to have more DNA in the freezer. We like having DNA in the freezer. (See: <A href="https://2012.igem.org/Team:Cornell/notebook/strain_list">Strain List</A>)  
<B>Result of the colony PCR:</B><B>July 6th, Friday</B>
							Caleb <A href="https://static.igem.org/mediawiki/2012/3/35/Ezna_Miniprep.pdf">miniprepped </A> p21, p8, p9, p10, p17 from the liquid cultures set up the previous night. (See: <A href="https://2012.igem.org/Team:Cornell/notebook/strain_list">Strain List</A>
							) We couldn't miniprep p14, since â after checking notes from the previous night â we learned that the culture was made with the wrong antibiotic (chloramphenicol instead of kanamycin).
							Some sequencing results came back, however the Gibson results are still confusing.
							Also, a <A href="https://static.igem.org/mediawiki/2012/6/6c/DNA_Gel_Electrophoresis.pdf"> gel</A> was run for the p21 PCR product.
<B>Gel showing the failure of the p21 PCR:</B></DIV></DIV><DIV class="row"><DIV class="nine columns"><H3>July 8th - 14th</H3><H6>Daily Details:</H6><B>July 9th, Monday </B>
							Dylan performed a <A href=" https://static.igem.org/mediawiki/2012/2/26/Phusion_PCR.pdf "> Phusion PCR </A> with a single annealing temperature of 59 degrees Celsius using primers 27 and 28 for the final time. In the future, new primers will be used. The purpose of this PCR was to amplify p21 (See: <A href="https://2012.igem.org/Team:Cornell/notebook/strain_list">Strain List</A>
							). The PCR product was analyzed using <A href="https://static.igem.org/mediawiki/2012/6/6c/DNA_Gel_Electrophoresis.pdf">agarose gel electrophoresis</A> to determine if the desired 1.127 kb fragment is present. We talked to Dr. Jeff Gralnick about electroporating Shewanella, and his recommendation was to try conjugation instead. To transform our plasmids into Shewanella with conjugation, we had asked Dr. Jeff Gralnick for WM3064 E. coli for which plates containing 2,6-Diaminopimelic Acid (DAP) are required. In preparation to receive this strain, the plates with DAP and DAP/kanamycin were prepared.
							<B>July 10th, Tuesday</B>
							Caleb <A href="https://static.igem.org/mediawiki/2012/3/35/Ezna_Miniprep.pdf">miniprepped </A>
							strains containing Gibson products and the arsenic promotors without BamHI cutsites: plasmids p8, p9, p10, p15, and p16 (See: <A href="https://2012.igem.org/Team:Cornell/notebook/strain_list">Strain List</A>
							). Also, our new freezer arrived. We cleaned and welcomed our new friend.
							The WM3064 came in and we plated from the agar stab. Dylan set up two 18 mL/hr continuous flow reactors using 10x diluted LB and innoculated one of the reactors with JG700 and one with wild type Shewanella oneidensis MR-1.

							<B>July 11th, Wednesday</B>
							Dylan started a WM3064 subculture in the morning before meeting Swati, Shweta, Caleb, and Danielle at the Boyce Thompson Institute for Plant Research to give a presentation both on our project and on the broader applications of synthetic biology to the energy industry to a group of high school teachers from New York state. After the presentation, Dylan and Danielle prepared plates containing DAP/chloramphenicol and DAP/ampicillin to grow transformed WM3064 on, while Mark continued preparing <A href="https://static.igem.org/mediawiki/2012/5/57/Making_Ecoli_Electrocompetent.pdf "> electrocompetent </A> stocks of WM3064 with the subculture that Dylan had started.
							We transformed things later that day.
							<B>July 12th, Thursday</B>
							Shweta prepared a glycerol stock of WM3064 for future use, and Dylan plated some remaining transformed WM3064 hoping to get more colonies of the successfully transformed cells.
							Dylan and Mark submitted p8, p9, and p10 for sequencing using two different primers for each plasmid.
							The previous night's p21 PCR seemed to have failed, even with the new primers. Based on the <A href="https://static.igem.org/mediawiki/2012/6/6c/DNA_Gel_Electrophoresis.pdf">gel </A>run by Claire, then analyzed by Dylan and Caleb.
<B>Gel showing failed p21 PCR product:</B><B>July 13th, Friday</B>
							Today began with Dylan and Caleb setting up conjugations between our transformed donor E. coli and S. oneidensis.
							The sequencing results for p8, p9 and p10 all came back as failed, so it was concluded that the Gibson assembly failed. There were five locations on the nah operon at which sequencing could begin, of which we considered all but the fourth. Sequencing at the first location resulted in a sequence that corresponded to the fifth sequencing location, with some sequence before it and all of the expected sequence after it. The fifth sequencing location produced the expected sequence. Then the middle two regions which we considered all resulted in failed sequencing. This has led us to the conclusion that segments that are required in the operon are missing, and thus the assembly failed. Dylan and Caleb began attempting to look into the possibility of this being a cause, and some potential solutions.
							In discussing why p21 PCR seems to continue failing, the idea of having p21 sequenced arose. Dylan prepared the sample and submitted it for sequencing. Meanwhile, Mark set up yet another <A href=" https://static.igem.org/mediawiki/2012/2/26/Phusion_PCR.pdf "> Phusion PCR</A> of p21, this time with a higher annealing temperature of 61 degrees Celsius and a decreased extension time of 18 seconds. The goal was to prevent the mispriming which seemed to have occurred in the previous day's attempt.
							Later that day, Dylan set up yet another PCR with a decreased annealing temperature of 55 degrees Celsius in order to empirically narrow-in on the optimal annealing temperature for the amplification of the salicylate sensing region from p21.
							<B>July 14th, Saturday</B>
							Swati ran a <A href="https://static.igem.org/mediawiki/2012/6/6c/DNA_Gel_Electrophoresis.pdf">gel </A>of 5uL of the p21 PCR that Mark set up yesterday. After staining, the DNA ladder was well visualized but there were no bands in the p21 lane, suggesting that the PCR did not work. In a fit of rage, Swati yelled at Maneesh for going to get a drink of water - yet another innocent victim of failed PCR. Then, after a full team meeting, we all came together for a dumpling party. Noodles, dumplings, and gnocchi of all kinds were eaten by the voracious team. In the spirit of true, unwavering scientific inquiry, we conducted a novel experiment whilst feasting on these delectable treats. The result: fried Dorito dumplings are delicious.
							</DIV></DIV><DIV class="row"><DIV class="nine columns"><H3>July 15th - 21st</H3><B>July 15th, Sunday </B>
							Dylan noticed that we had colonies on the plates re-streaked from the original plates of <A href=" https://static.igem.org/mediawiki/2012/0/0f/Shewanella_Conjugation.pdf
							">conjugated </A>
							S. oneidensis. He started overnight cultures of S20 (See: <A href="https://2012.igem.org/Team:Cornell/notebook/strain_list">Strain List</A>
							) in kanamycin so that we can <A href="https://static.igem.org/mediawiki/2012/0/03/Sequencing_Preparation.pdf "> sequence </A> and make glycerol stocks. He also started overnight cultures of w.t. S. oneidensis to inoculate into new reactors in Riley Robb, which will be used as positive controls.
							<B>July 16th, Monday</B>
							Dylan ran another <A href=" https://static.igem.org/mediawiki/2012/2/26/Phusion_PCR.pdf "> Phusion PCR </A> of the entire nah operon, since we got a lot of mispriming the first time we ran it; we believe this was because we had used the optimal annealing temperature for Vent Polymerase (55ï°C) with Phusion Polymerase. Dylan set up the new PCR correctly, with an annealing temperature of 66ï°C and a lengthened final extension time of 15 minutes to account for the size of the nah operon (~10kb). Caleb ran a gel of the PCR and visualized a single band ~9.5kb. PCR of the nah operon out of p20, P. putida (See: strain list) was successful! Tina and Swati gel extracted and quantified, extracting two samples at 38.8 ng/uL and 27.2 ng/uL.
							Spencer <A href="https://static.igem.org/mediawiki/2012/3/35/Ezna_Miniprep.pdf">miniprepped </A> p14, our arsenic reporter part (arsR + mtrB w/ BamHI cutsite), from S20 (See: <A href="https://2012.igem.org/Team:Cornell/notebook/strain_list">Strain List</A>
							) for sequencing, recording yields of 44.3ng/uL &amp; 35.8ng/uL (colony 1) and 50.6ng/uL &amp; 53.5ng/uL (colony 2) from quantification. Dylan and Tina set up transformations of p15 and p16 (See: strain list), the arsenic reporter parts without a BamHI cutsite, so conjugations with S. oneidensis can be done in the next few days. They also set up a transformation of the miniprepped BBa_J01003 with the oriT mobility gene. Because of our lack of success with electroporation, we are planning on trying <A href=" https://static.igem.org/mediawiki/2012/0/0f/Shewanella_Conjugation.pdf
							">conjugation </A> of our constructs into S. oneidensis. However, this requires that all our plasmids have the mobility gene. pSB3C5, the plasmid we are going to use for the nah operon has no mobility gene. Thus, we are going to try two things: to clone oriT from iGEM kit plates into pSB3C5, and to amplify the mobility gene out of one of our own plasmids and clone that into pSB3C5.
<B>Success!:</B><B>July 17th, Tuesday </B>
							Shweta set up a <A href=" https://static.igem.org/mediawiki/2012/2/26/Phusion_PCR.pdf "> Phusion PCR </A> of p21 (nahR and Psal) to try and extract the salicylate-sensitive promoter again. Sequencing of p21 showed a stem loop sitting upstream of the ENX biobrick cutsites, which was not expected and likely the reason for our unsuccessful PCRs. Before redesigning primers, we are going to try PCR with the standard iGEM forward sequencing primer and the same reverse primer that we designed for p21.
<B>Success!:</B><B>July 18th, Wednesday </B>
							Dylan set up p15k, p16k conjugation plates from overnight cultures of transformed WM3064 and JG700. Dylan let these plates incubate for 8 hours, and then streaked for single colonies on kan plates with E.coli and Shewanella controls. Caleb <A href="https://static.igem.org/mediawiki/2012/3/35/Ezna_Miniprep.pdf">miniprepped </A>
							p24a (BioBrick part with oriT) from overnight cultures. He then desalted an overnight ligation of the nah operon in pSB1C3 and transformed into DH5a. Dylan also trekked over to Riley Robb to set up a reactor for inoculation tomorrow. That night, Spencer set up overnight cultures of p25a, p28a, and p31c.
							<B>July 19th, Thursday </B>
							Only observed 2 possible colonies from transformants of yesterday's ligation (nah operon in pSB1C3). We restreaked from one of these colonies along with a DH5a control.
							We got single colonies on the JG700+p15 and JG700+p16 plates, and no growth on control plates. Thus, our tentative/optimistic conclusion is that the conjugation was successful. Dylan and Caleb made reference plates of the picked colonies, and grew up liquid cultures from same colonies for sequencing.
							Dylan and Caleb then <A href="https://static.igem.org/mediawiki/2012/3/35/Ezna_Miniprep.pdf">miniprepped </A> p25a, p28a, p31c from overnight cultures set up by Spencer and made glycerol stocks of S25, S28, and S31. That afternoon, Dylan also inoculated a reactor in Riley Robb with S20, our engineered arsenic reporter strain.
							We then set up a digestion of the p21 PCR product along with our miniprepped p14 with EcoRI and AscI. We purified the p21 PCR digestion with Omega Bio-Tek E.Z.N.A. MicroElute DNA Clean-Up Kit. We dephosphorylated p14 digestion with Antarctic Phosphatase, and ran the entire mixture on a gel. That night, we <A href=" https://static.igem.org/mediawiki/2012/4/46/EZNA_Gel_Extraction.pdf
							pdf "> gel extracted </A>the 6kb band with our Qiagen gel extraction kit and set up overnight cultures of a p26a, 027a, p29a, p30a (our Anderson series constitutive promoters), and p14k.
<B>Dephosphorylated p14 digestion:</B><B>July 20th, Friday </B>
							This morning Shweta and Tina <A href="https://static.igem.org/mediawiki/2012/3/35/Ezna_Miniprep.pdf">miniprepped </A>
							p26a, p27a, p29a, p30a (Anderson series constitutive promoters), and p14k from overnight cultures. Dylan set up a 30 minute room temperature ligation to connect the salicylate reporter with digested p21 PCR product and p14 (isolated yesterday, quantified today). Because Shewanella grows more slowly than E.coli, Claire did a second miniprep of p15k and p16k from JG700 in order to submit the plasmid for <A href="https://static.igem.org/mediawiki/2012/0/03/Sequencing_Preparation.pdf "> sequencing </A> to confirm that conjugation was successful. In anticipation of the failure of Dylan's ligation, Swati set up 3 <A href=" https://static.igem.org/mediawiki/2012/2/26/Phusion_PCR.pdf "> Phusion PCR</A> reactions in parallel, each with the same template (p21), and repeated the parameters that had proven successful (no mispriming) previously. Dylan also set up overnight cultures of the strain with the nah operon in pSB1C3, S18 (DH5a + pSB3C5/p17c), and WM3064+p14.
							</DIV></DIV><DIV class="row last-ele"><DIV class="nine columns"><H3>July 22nd - 28th</H3><B>July 22nd, Sunday </B>
							Danielle and Dylan <A href="https://static.igem.org/mediawiki/2012/a/af/Double_Digest.pdf ">digested </A> p14(Arsenic reporter) and p21(Salicylate sensing region) for ligation. Swati ran a <A href=" https://static.igem.org/mediawiki/2012/2/26/Phusion_PCR.pdf "> Phusion PCR </A> to amplify p21. Then Dylan ran a <A href="https://static.igem.org/mediawiki/2012/6/6c/DNA_Gel_Electrophoresis.pdf">gel electrophoresis</A> and did <A href=" https://static.igem.org/mediawiki/2012/4/46/EZNA_Gel_Extraction.pdf
							pdf "> gel extraction </A> to isolate p21 PCR digest and p14 backbone.
<B>PCR confirmation of p21:</B><B>Digest of p21 PCR an p14:</B><B>July 23rd, Monday </B>
							Dylan quantified the gel extractions, quite successfully with a newly invented protocol. Then he dephosphorylated p14, and submitted some DNA for <A href="https://static.igem.org/mediawiki/2012/0/03/Sequencing_Preparation.pdf "> sequencing </A>. At one point, the Magical Graduated Cylinder of Elmira (500 mL) fell from the sky and shattered in the sink. Some voodoo was clearly in the air. It appears that the ligation products from the day before were either not created or not successfully transformed, as the plates contained no colonies.
							Caleb and Tina prepared more kanamycin and chloramphenicol plates while Mark desalted Dylan's ligation product. Then Dylan, Mark, and Tina transformed DH5Î± with the ligation product, p33k, p14k, and p31c (See: <A href="https://2012.igem.org/Team:Cornell/notebook/strain_list">Strain List</A>).

							<B>July 24th, Tuesday </B>
							In the morning, Dylan did <A href=" https://static.igem.org/mediawiki/2012/2/26/Phusion_PCR.pdf "> Phusion PCR </A> of p20(nah operon). Then Caleb and Dylan did a <A href="https://static.igem.org/mediawiki/2012/a/af/Double_Digest.pdf "> double digestion </A> of p20 PCR and p17c for ligation. Dylan ran a DNA <A href="https://static.igem.org/mediawiki/2012/6/6c/DNA_Gel_Electrophoresis.pdf">gel electrophoresis</A> to confirm that the previous PCR was successful. Steven performed gel extraction to isolate p17c backbone fragment.
<B>Digest of p17 and PCR of p20:</B><B>July 25th, Wednesday </B>
							Mark set up two <A href="https://static.igem.org/mediawiki/2012/a/a7/T4_Ligase_Ligation.pdf ">ligations </A> of a p20 PCR product and p17c, one at standard concentrations and one at very high concentrations. (See: <A href="https://2012.igem.org/Team:Cornell/notebook/strain_list">Strain List</A>
							) p20 PCR was cut at Xbio1 and Spe1, which have compatible sticky ends. This means that p20 PCR can be ligated into backbone in two directions, only one of which is useful. These were desalted and transformed by Mark and Dylan, along with p33k. Meanwhile, Tina and Chie set up <A href="https://static.igem.org/mediawiki/2012/a/af/Double_Digest.pdf "> double digestions </A> of p24a, p20 PCR product, and <A href="https://static.igem.org/mediawiki/2012/f/fb/Single_Digest.pdf"> single digestions </A>
							of p25a, p27a, and p29a.
							Dylan ran two gels of the double and single digestions. The single digestions of the Anderson series promoters showed single bands, which ran slightly faster than the supercoiled DNA control. This may be because the supercoiled DNA was nicked. The gel of the oriT double digestion showed more long bands than expected â possibly a result of star activity â but had a band ~400bp which was interpreted to be the insert of interest.
<B>oriT insert / nah operon:</B><B>Constitutive promoters:</B><B>July 26th, Thursday </B>
							Caleb gel purified the digestion products of p24a, p20 PCR, p25a, p27a, and p29a. Danielle dephosphorylated products of p25a, p27a and p29a in preparation for <A href="https://static.igem.org/mediawiki/2012/a/a7/T4_Ligase_Ligation.pdf ">ligations </A>.
							<B>July 27th, Friday </B>
							In the morning, Claire <A href="https://static.igem.org/mediawiki/2012/3/35/Ezna_Miniprep.pdf">miniprepped </A> p11k, p17c, and p33k from the overnight cultures that Dylan had set up. Meanwhile, Dylan heat killed the ligase from the overnight <A href="https://static.igem.org/mediawiki/2012/a/a7/T4_Ligase_Ligation.pdf ">ligations </A>
							(p25a, p27a and p29a) (See: <A href="https://2012.igem.org/Team:Cornell/notebook/strain_list">Strain List</A>
							), while Caleb prepared to desalt the ligation mixtures before electroporation via drop dialysis. Shortly thereafter, Dylan and Caleb electroporated the four desalted mixtures into DH5Î±, allowing the cells to recover one hour in SOC before plating. If these transformations are successful, we will have cells that carry plasmids expressing the nah operon under the control of three different constitutive promoters with varying strength, as well as a cell line that carries a plasmid that confers chloramphenicol resistance, has a p15a origin of replication, and an origin of transfer. Eventually, the nah operon will be ligated as an insert into such a backbone.
							In the evening, Dylan and Tina started 30 mL overnight cultures of S24, S25, S27, S29, S15, and S17 to miniprep from the next morning. S24 carries a plasmid with an oriT, which will be <A href="https://static.igem.org/mediawiki/2012/3/35/Ezna_Miniprep.pdf">miniprepped </A> in case our previous ligation of an oriT into p17c failed. S25,27, and 29 all carry plasmids with Anderson series constitutive promoters of varying strength, which will be used to set up fluorescent controls to monitor mtrB expression levels. S15 and S17 carry our engineered arsenic reporter plasmids, which will be further modified to facilitate control studies in mtrB expression.
							After setting up the cultures, Dylan and Swati left Weill and headed over to Riley Robb where they assembled three new reactor setups in the Angenent lab. Two of these reactors will be run in continuous flow operation with a BioLogic potentiostat, while the other will be run in batch mode with a CH Instruments potentiostat. Tomorrow, one continuous flow and one batch reactor will be inoculated with wildtype Shewanella oneidensis MR-1, while the remaining continuous flow reactor will be inoculated with our engineered arsenic reporter strain, S20. The purpose of running the continuous flow reactors is to better define the maximum current output we can expect from our induced strains in response to analyte, and to repeat an experiment carried out on the CH potentiostat for basal activity from the uninduced arsenic reporter. Because we were unsure of the veracity of the data from our previous experiment using the CH potentiostat, we are running the batch reactor to see if the observed current response is what we'd expect from wildtype Shewanella.
							<B>July 28th, Saturday </B>
							Today, Swati did six <A href="https://static.igem.org/mediawiki/2012/3/35/Ezna_Miniprep.pdf">minipreps </A> in the morning and it took her four hours...and it paid off! The nanodrop told us that our highest yield was 812.6 ng/uL.
							While Swati was devoting her soul to the E.Z.N.A kit, Dylan continued to set up reactors in the Angenent Lab, making reference electrodes for the three reactors, and getting the pumps set up for the continuous flow reactors. After our weekly team meeting, he inoculated the reactor.
							Swati prepared overnight cultures and reference plates, while Danielle prepared kanamycin plates and did autoclave.
							In the evening, Dylan and Swati did <A href="https://static.igem.org/mediawiki/2012/a/af/Double_Digest.pdf "> digestions </A> for RFP controls and SAL2 reporter. Also, SAL reporter was transformed into WM3064.

							</DIV></DIV><DIV class="row last-ele"><DIV class="nine columns"><H3>July 29th - 31st</H3><B>July 29th, Sunday </B>
								Apparently, Swati has yet to get over her miniprep craze. While most of the wetlab hiked through Danby State Forest, Swati did more minipreps: p25a, oriT + PSB3C5, and nah operon + p29a. The cultures of S25 and nah p29 were bright magenta, while oriT in PSB3C5 was a pale tan. However, the last of these contained an unidentified crystalline substance, possibly a byproduct of producing the nah operon. Two other cell cultures also didn't grow, which was disheartening, as it meant there were less minipreps for Swati to do. Interspersed between the copious miniprepping were two gels to purify the digestions from the previous evening, all of which appeared to work! Dylan and Claire, returning from their nature trek, joined Swati in lab, finding Shweta and Eric along the way. While Dylan took over Swati's gel extraction, Swati, Shweta, and Claire discussed upcoming outreach events. As Ecologist Claire explained the feeding habits of Tomiculus troutus (AKA Tommy the trout), Eric began to feel left out and, in the process of showing off the website, discovered Dylan's odd love for typography. After spending roughly an hour accomplishing nothing, Dylan and Swati set up liquid cultures in preparation to put SAL into JG700. Dylan returned to his reactor-ridden lair for a short spell.
<B>07/29 gel:</B><B>July 30th, Monday </B>
								In the morning, Dylan set up conjugations between WM3064+SAL and JG700 while Caleb quantified the minipreps and gel extractions from Sunday. After quantification, he submitted nah+p29a and oriT+p17c for sequencing. Dylan and Mark dephosphorylated the digestions of SAL with SpeI &amp; PstI, p14k with SpeI &amp; PstI, p16k with SpeI &amp; PstI, p16k with EcoRI &amp; AscI, and p11k with EcoRI &amp; PstI. Tina and Danielle then set up ligations of mRFP+SAL, mRFP+p14, and mRFP+p16 to put mRFP downstream of the arsenic and salicylate reporters, in order to be able to characterize our induced promoter strengths by measuring fluorescence. We also ligated p16+SAL to make SAL2, the salicylate reporter without the BamHI cutsite. Finally, we ligated p25, p27, and p29 (the Anderson promoters with mRFP) into pBBRBB, in order to be able to put them into Shewanella.

								<B>July 31st, Tuesday </B>
								Mark made DH5Î± electrocompetent stocks. Dylan checked conjugation plates to put SAL into JG700. He then made overnight cultures of JG700+SAL and S31 (PSB1C3, as well as WM3064 to prepare competent stocks. Dylan autoclaved a reactor in Riley-Robb and set it up in batch with M4 media, and inoculated with wild-type S. oneidensis MR-1. The purpose was to repeat a previous batch experiment in order to be sure that it is something about feeding with LB that is decreasing current production. 
Sequencing results came back on nah+p29 and oriT+p17c and things looked good. Consequently, Dylan decided to digest nah+p29 with EcoRI and SpeI, and oriT+p17c with EcoRI and XbaI in order to eventually ligate the nah operon with a constitutive promoter into the p17c backbone, upstream from the oriT sequence (in order to allow the plasmid to be conjugated into Shewanella). Claire prepared these digestions, along with digestions of p31c and p20 PCR product (in order to get the nah operon into pSB1C3). 
Dylan and Caleb desalted the previous night's ligations and than transformed into WM3064, since we will want to get plasmid into Shewanella. 

								</DIV></DIV><DIV class="row last-ele"><DIV class="three columns"><A href="https://2012.igem.org/Team:Cornell/notebook/wetlab/june#pillTab2">&lt;&lt; Previous</A></DIV><DIV class="three columns offset-by-six"><A href="https://2012.igem.org/Team:Cornell/notebook/wetlab/august#pillTab2" style="display:block;float:right;">Next &gt;&gt;</A></DIV></DIV></LI><LI id="pillTab3Tab"><DIV class="row last-ele" style="margin-top:20px;"><DIV class="three columns"><A href="https://2012.igem.org/Team:Cornell/notebook/wetlab/june#pillTab3">&lt;&lt; Previous</A></DIV><DIV class="three columns offset-by-six"><A href="https://2012.igem.org/Team:Cornell/notebook/wetlab/august#pillTab3" style="display:block;float:right;">Next &gt;&gt;</A></DIV></DIV><DIV class="row"><DIV class="nine columns"><H3> July 1st â 7th </H3>
							A lot of troubleshooting occurred this week. Once we figured out that SYBR Green was causing our gels to run strangely, we switched to staining with EtBr after running the gel. This approach gave us better results. We also continued to try electroporation protocols for transforming Shewanella, and continued to work on PCRing out the salicylate-sensing region.
							<DIV class="panel" style="background:white;margin-top: 20px;"><H6>Daily Details:</H6><B>July 1st, Sunday</B>
								Dylan and Caleb set up two gels for <A href="https://static.igem.org/mediawiki/2012/6/6c/DNA_Gel_Electrophoresis.pdf">electrophoresis</A>. Caleb's was 1% agarose in BIO-RAD Mini-Sub Cell system for continuation of ladder test using SYBR Green, containing NEB 100 bp ladder, NEB 2-log ladder, Promega 1 kb ladder and run at 100V. Dylan's was 1% agarose in Owl box using ethidium bromide, containing the nah operon PCR product from previous night, and run at 55 V.
								Our plates of DH5a transformed with p21 ligated into pBBRBB with mtrB grew only one colony, possibly contamination. Dylan ran a colony PCR and got a smear, suggesting that the PCR of p21 or the ligation did not work.
								Because we learned that our SYBR Green was causing ladder to run strangely, Dylan decided to redo a <A href="https://static.igem.org/mediawiki/2012/3/33/Vent_PCR.pdf">Vent PCR</A> to amplify the salicylate reporter region out of p21.
								Also, a liquid culture of JG700 was prepared, as well as replating of p21, p22, JG700, JG1220, JG1537, JG1219. (See: <A href="https://2012.igem.org/Team:Cornell/notebook/strain_list">Strain List</A>)
<B>EtBr stained gel of nah Operon PCR product ran at 55V:</B><B>SYBR green stained gel of ladder experiment:</B><B>July 2nd, Monday</B>
								Today, Dylan and Caleb ran a <A href="https://static.igem.org/mediawiki/2012/6/6c/DNA_Gel_Electrophoresis.pdf">electrophoresis</A> of a <A href="https://static.igem.org/mediawiki/2012/3/33/Vent_PCR.pdf">Vent PCR</A> of p21 (the PCR product being the salicylate reporter) at 55 V. Additionally, Caleb decided to run a control gel at 100 V to determine whether higher voltage was a factor in our previous ladder problems (in addition to the SYBR Green stock). While we determined that the higher voltage did not cause our ladder issues, we did not see any bands from the p21 PCR (gel image not included).
								Dylan prepared electrocompetent cells using the <A href=" https://static.igem.org/mediawiki/2012/6/6f/MyersMyers_Shew-E-poration.pdf "> Myers and Myers </A> protocol. Modified protocol using two 5mL cultures @ 4000g for 10 min. Washed with 2 mL sorbitol, resuspended in 100 uL sorbitol. First electroporation ts = 2.32 ms, second ts = 2.02 ms. Added 1 uL of plasmid (575.5 ng) to each cuvette. First used .60 V, then 0.55 V, both with R = 400 ohms.
								Mark and Danielle started liquid cultures of S1, S9, S10, S11, S15, S16, S18, S27 (See: <A href="https://2012.igem.org/Team:Cornell/notebook/strain_list">Strain List</A>)
<B>EtBr stained ladders:</B><B>July 3rd, Tuesday</B>
								Caleb <A href="https://static.igem.org/mediawiki/2012/3/35/Ezna_Miniprep.pdf">miniprepped </A> S16 (p15), S22 (p21), S9 (p8), S10 (p9), S11 (p10), S18 (p17), and S15 (p14). (See: <A href="https://2012.igem.org/Team:Cornell/notebook/strain_list">Strain List</A>
								) Instead of a single EB elution, Caleb did two elutions, 30 ul each. The double 30 ul elution turned out to be effective at recovering a usable amount of DNA in the second elution, so we're including it on our miniprep protocol for future minipreps.
								The rerun of our Gibson sequencing failed again, so we tried a more roundabout method of determining whether our 3 Gibson transformants have complete nah operons or not. We digested them each with BamHI, expecting specific fragment lengths on a gel electrophoresis.
								When Dylan and Mark were attempting to determine what volume of ethidium bromide should be used in the gel, they came to the conclusion that better images may result from staining the gel in dilute ethidium bromide after running the gel, rather than including the stain in the gel. This experimental first staining used 200 mL of water conaining 20 uL of 10 mg/mL ethidium bromide, with gentle agitation for 1 hour. This stained the bands well, but also provided a large amount of background and took a very long time.
								We only saw one band on each, all of them at about 3.6kb.
								Because no transformations from our previous competent freezer stocks were successful, Dylan decided to make a starter culture of Shewanella strain JG 700 in preparation for making new competent stocks using the <A href="https://static.igem.org/mediawiki/2012/f/f7/PNNL_Electrocompetent_Shewanella.pdf
								"> PNNL Protocol </A>. Swati and Tina used this starter culture to complete the preparation.
<B>Three Gibson transformants and three ladders with Ethidium Bromide stain after running the gel:</B><B>Gel with SYBR green:</B><B>July 4th, Wednesday</B>
								In the morning, Dylan, Swati, and Danielle prepared p8, p9, p10, p14, and p15 for <A href="https://static.igem.org/mediawiki/2012/0/03/Sequencing_Preparation.pdf "> sequencing </A> (from the minipreps that Caleb preformed the previous day).
								The team took the rest of the day off and had a barbeque at Buttermilk Falls. There was an excess of food, spontaneous song, and fireflies.
								<B>July 5th, Thursday </B>
								Upon arriving to the lab, Dylan dropped off the sequencing tubes we'd set up on the morning of the 4th for analysis. He also decided to do a Colony PCR of the potential transformant with the salicylate reporter plasmid. However, when setting up this reaction, Dylan realized that we'd used the wrong sequencing primers (we'd used the standard VF2 BioBrick primer instead of a custom primer for the pBBRBB backbone). Consequently, Mark and Dylan resubmitted p14 and p15 for sequencing while the colony PCR was ongoing. After visualizing the PCR products using <A href="https://static.igem.org/mediawiki/2012/6/6c/DNA_Gel_Electrophoresis.pdf"> DNA gel electrophoresis</A>, Dylan concluded that the colony we'd screened did not have our plasmid of interest, and was the result of contamination.
								Similarly, Dylan noticed small, evenly spaced colonies on all of the JG700 transformant plates from the <A href=" https://static.igem.org/mediawiki/2012/6/6f/MyersMyers_Shew-E-poration.pdf "> Myers and Myers </A> procedure we'd undertaken on the 2nd of July. Because none of the colonies looked red (as they should have if they were replicating pSB3C5), Dylan concluded that the chloramphenicol on the plates had degraded, and the colonies were resultant from untransformed cells.
								Because the colony PCR of the potential salicylate reporter didn't yield good results, Dylan and Youjin set up another <A href=" https://static.igem.org/mediawiki/2012/2/26/Phusion_PCR.pdf "> Phusion PCR </A> to amplify the salicylate-sensing region from p21. Simultaneously, Mark made SOB media for recovery after future transformations, as we'd just received the ingredients for the broth.
								Because we needed to submit for sequencing twice (because we used the wrong sequencing primers), Swati et al. set up liquid cultures of p21, p8, p9, p10, p17, and p14 in order to have more DNA in the freezer. We like having DNA in the freezer. (See: <A href="https://2012.igem.org/Team:Cornell/notebook/strain_list">Strain List</A>)
<B>Result of the colony PCR:</B><B>July 6th, Friday</B>
								Caleb <A href="https://static.igem.org/mediawiki/2012/3/35/Ezna_Miniprep.pdf">miniprepped </A> p21, p8, p9, p10, p17 from the liquid cultures set up the previous night. (See: <A href="https://2012.igem.org/Team:Cornell/notebook/strain_list">Strain List</A>
								) We couldn't miniprep p14, since â after checking notes from the previous night â we learned that the culture was made with the wrong antibiotic (chloramphenicol instead of kanamycin).
								Some sequencing results came back, however the Gibson results are still confusing.
								Also, a <A href="https://static.igem.org/mediawiki/2012/6/6c/DNA_Gel_Electrophoresis.pdf"> gel</A>  was run for the p21 PCR product.
<B>Gel showing the failure of the p21 PCR:</B></DIV></DIV></DIV><DIV class="row"><DIV class="nine columns"><H3>July 8th - 14th</H3>
							The focus of this week was to transfer our arsenic sensing plasmids to Shewanella strain JG700 via congugation with E. coli strain WM3064. After struggling to electroporate control plasmids, we consulted with Dr. Gralnick who reccomended proceeding with conjugation and kindly provided E. coli strain WM3064. This strain is auxotrophic for 2,6-Diaminopimelic Acid (DAP), which makes the strain useful for selecting against non-Shewanella conjugants.
							Another major decision this week was to give up on Gibson assembly - our sequencing results suggested that strange things had happened, and since we already had PCRed the full nah operon out of P. putida, we decided proceeding with Gibson was not an efficient use of time. We were also unsure as to why p21's PCR was repeatedly failing, and decided to submit the part from the registry for sequencing, to try to troubleshoot what was going wrong.

							<DIV class="panel" style="background:white;margin-top: 20px;"><B>July 9th, Monday </B>
								Dylan performed a <A href=" https://static.igem.org/mediawiki/2012/2/26/Phusion_PCR.pdf "> Phusion PCR </A> with a single annealing temperature of 59 degrees Celsius using primers 27 and 28 for the final time. In the future, new primers will be used. The purpose of this PCR was to amplify p21 (See: <A href="https://2012.igem.org/Team:Cornell/notebook/strain_list">Strain List</A>
								). The PCR product was analyzed using <A href="https://static.igem.org/mediawiki/2012/6/6c/DNA_Gel_Electrophoresis.pdf">agarose gel electrophoresis</A> to determine if the desired 1.127 kb fragment is present. We talked to Dr. Jeff Gralnick about electroporating Shewanella, and his recommendation was to try conjugation instead. To transform our plasmids into Shewanella with conjugation, we had asked Dr. Jeff Gralnick for WM3064 E. coli for which plates containing 2,6-Diaminopimelic Acid (DAP) are required. In preparation to receive this strain, the plates with DAP and DAP/kanamycin were prepared.
								<B>July 10th, Tuesday</B>
								Caleb <A href="https://static.igem.org/mediawiki/2012/3/35/Ezna_Miniprep.pdf">miniprepped </A>
								strains containing Gibson products and the arsenic promotors without BamHI cutsites: plasmids p8, p9, p10, p15, and p16 (See: <A href="https://2012.igem.org/Team:Cornell/notebook/strain_list">Strain List</A>
								). Also, our new freezer arrived. We cleaned and welcomed our new friend.
								The WM3064 came in and we plated from the agar stab. Dylan set up two 18 mL/hr continuous flow reactors using 10x diluted LB and innoculated one of the reactors with JG700 and one with wild type Shewanella oneidensis MR-1.

								<B>July 11th, Wednesday</B>
								Dylan started a WM3064 subculture in the morning before meeting Swati, Shweta, Caleb, and Danielle at the Boyce Thompson Institute for Plant Research to give a presentation both on our project and on the broader applications of synthetic biology to the energy industry to a group of high school teachers from New York state. After the presentation, Dylan and Danielle prepared plates containing DAP/chloramphenicol and DAP/ampicillin to grow transformed WM3064 on, while Mark continued preparing <A href="https://static.igem.org/mediawiki/2012/5/57/Making_Ecoli_Electrocompetent.pdf "> electrocompetent </A> stocks of WM3064 with the subculture that Dylan had started.
								We transformed things later that day.
								<B>July 12th, Thursday</B>
								Shweta prepared a glycerol stock of WM3064 for future use, and Dylan plated some remaining transformed WM3064 hoping to get more colonies of the successfully transformed cells.
								Dylan and Mark submitted p8, p9, and p10 for sequencing using two different primers for each plasmid.
								The previous night's p21 PCR seemed to have failed, even with the new primers. Based on the <A href="https://static.igem.org/mediawiki/2012/6/6c/DNA_Gel_Electrophoresis.pdf">gel </A>run by Claire, then analyzed by Dylan and Caleb.
<B>Gel showing failed p21 PCR product:</B><B>July 13th, Friday</B>
								Today began with Dylan and Caleb setting up conjugations between our transformed donor E. coli and S. oneidensis.
								The sequencing results for p8, p9 and p10 all came back as failed, so it was concluded that the Gibson assembly failed. There were five locations on the nah operon at which sequencing could begin, of which we considered all but the fourth. Sequencing at the first location resulted in a sequence that corresponded to the fifth sequencing location, with some sequence before it and all of the expected sequence after it. The fifth sequencing location produced the expected sequence. Then the middle two regions which we considered all resulted in failed sequencing. This has led us to the conclusion that segments that are required in the operon are missing, and thus the assembly failed. Dylan and Caleb began attempting to look into the possibility of this being a cause, and some potential solutions.
								In discussing why p21 PCR seems to continue failing, the idea of having p21 sequenced arose. Dylan prepared the sample and submitted it for sequencing. Meanwhile, Mark set up yet another <A href=" https://static.igem.org/mediawiki/2012/2/26/Phusion_PCR.pdf "> Phusion PCR</A> of p21, this time with a higher annealing temperature of 61 degrees Celsius and a decreased extension time of 18 seconds. The goal was to prevent the mispriming which seemed to have occurred in the previous day's attempt.
								Later that day, Dylan set up yet another PCR with a decreased annealing temperature of 55 degrees Celsius in order to empirically narrow-in on the optimal annealing temperature for the amplification of the salicylate sensing region from p21.
								<B>July 14th, Saturday</B>
								Swati ran a <A href="https://static.igem.org/mediawiki/2012/6/6c/DNA_Gel_Electrophoresis.pdf">gel </A>of 5uL of the p21 PCR that Mark set up yesterday. After staining, the DNA ladder was well visualized but there were no bands in the p21 lane, suggesting that the PCR did not work. In a fit of rage, Swati yelled at Maneesh for going to get a drink of water - yet another innocent victim of failed PCR. Then, after a full team meeting, we all came together for a dumpling party. Noodles, dumplings, and gnocchi of all kinds were eaten by the voracious team. In the spirit of true, unwavering scientific inquiry, we conducted a novel experiment whilst feasting on these delectable treats. The result: fried Dorito dumplings are delicious.
								</DIV></DIV></DIV><DIV class="row"><DIV class="nine columns"><H3>July 15th - 21st</H3>
							For this week, we began to conjugate our constructs into Shewanella and Dylan began seeding our reactors with WT Shewanella and with p14 to begin collecting data. Weâve also started work with the Anderson series of constitutive promoters for future work in anticipation of constructing control plasmids with constitutively produced MtrB.
							<DIV class="panel" style="background:white;margin-top: 20px;"><H6>Daily Details:</H6><B>July 15th, Sunday </B>
								Dylan noticed that we had colonies on the plates re-streaked from the original plates of <A href=" https://static.igem.org/mediawiki/2012/0/0f/Shewanella_Conjugation.pdf
								">conjugated </A>
								S. oneidensis. He started overnight cultures of S20 (See: <A href="https://2012.igem.org/Team:Cornell/notebook/strain_list">Strain List</A>
								) in kanamycin so that we can <A href="https://static.igem.org/mediawiki/2012/0/03/Sequencing_Preparation.pdf "> sequence </A> and make glycerol stocks. He also started overnight cultures of w.t. S. oneidensis to inoculate into new reactors in Riley Robb, which will be used as positive controls.
								<B>July 16th, Monday</B>
								Dylan ran another <A href=" https://static.igem.org/mediawiki/2012/2/26/Phusion_PCR.pdf "> Phusion PCR </A> of the entire nah operon, since we got a lot of mispriming the first time we ran it; we believe this was because we had used the optimal annealing temperature for Vent Polymerase (55ï°C) with Phusion Polymerase. Dylan set up the new PCR correctly, with an annealing temperature of 66ï°C and a lengthened final extension time of 15 minutes to account for the size of the nah operon (~10kb). Caleb ran a gel of the PCR and visualized a single band ~9.5kb. PCR of the nah operon out of p20, P. putida (See: strain list) was successful! Tina and Swati gel extracted and quantified, extracting two samples at 38.8 ng/uL and 27.2 ng/uL.
								Spencer <A href="https://static.igem.org/mediawiki/2012/3/35/Ezna_Miniprep.pdf">miniprepped </A> p14, our arsenic reporter part (arsR + mtrB w/ BamHI cutsite), from S20 (See: <A href="https://2012.igem.org/Team:Cornell/notebook/strain_list">Strain List</A>
								) for sequencing, recording yields of 44.3ng/uL &amp; 35.8ng/uL (colony 1) and 50.6ng/uL &amp; 53.5ng/uL (colony 2) from quantification. Dylan and Tina set up transformations of p15 and p16 (See: strain list), the arsenic reporter parts without a BamHI cutsite, so conjugations with S. oneidensis can be done in the next few days. They also set up a transformation of the miniprepped BBa_J01003 with the oriT mobility gene. Because of our lack of success with electroporation, we are planning on trying <A href=" https://static.igem.org/mediawiki/2012/0/0f/Shewanella_Conjugation.pdf
								">conjugation </A> of our constructs into S. oneidensis. However, this requires that all our plasmids have the mobility gene. pSB3C5, the plasmid we are going to use for the nah operon has no mobility gene. Thus, we are going to try two things: to clone oriT from iGEM kit plates into pSB3C5, and to amplify the mobility gene out of one of our own plasmids and clone that into pSB3C5.
<B>Success!:</B><B>July 17th, Tuesday </B>
								Shweta set up a <A href=" https://static.igem.org/mediawiki/2012/2/26/Phusion_PCR.pdf "> Phusion PCR </A> of p21 (nahR and Psal) to try and extract the salicylate-sensitive promoter again. Sequencing of p21 showed a stem loop sitting upstream of the ENX biobrick cutsites, which was not expected and likely the reason for our unsuccessful PCRs. Before redesigning primers, we are going to try PCR with the standard iGEM forward sequencing primer and the same reverse primer that we designed for p21.
<B>Success!:</B><B>July 18th, Wednesday </B>
								Dylan set up p15k, p16k conjugation plates from overnight cultures of transformed WM3064 and JG700. Dylan let these plates incubate for 8 hours, and then streaked for single colonies on kan plates with E.coli and Shewanella controls. Caleb <A href="https://static.igem.org/mediawiki/2012/3/35/Ezna_Miniprep.pdf">miniprepped </A>
								p24a (BioBrick part with oriT) from overnight cultures. He then desalted an overnight ligation of the nah operon in pSB1C3 and transformed into DH5a. Dylan also trekked over to Riley Robb to set up a reactor for inoculation tomorrow. That night, Spencer set up overnight cultures of p25a, p28a, and p31c.
								<B>July 19th, Thursday </B>
								Only observed 2 possible colonies from transformants of yesterday's ligation (nah operon in pSB1C3). We restreaked from one of these colonies along with a DH5a control.
								We got single colonies on the JG700+p15 and JG700+p16 plates, and no growth on control plates. Thus, our tentative/optimistic conclusion is that the conjugation was successful. Dylan and Caleb made reference plates of the picked colonies, and grew up liquid cultures from same colonies for sequencing.
								Dylan and Caleb then <A href="https://static.igem.org/mediawiki/2012/3/35/Ezna_Miniprep.pdf">miniprepped </A> p25a, p28a, p31c from overnight cultures set up by Spencer and made glycerol stocks of S25, S28, and S31. That afternoon, Dylan also inoculated a reactor in Riley Robb with S20, our engineered arsenic reporter strain.
								We then set up a digestion of the p21 PCR product along with our miniprepped p14 with EcoRI and AscI. We purified the p21 PCR digestion with Omega Bio-Tek E.Z.N.A. MicroElute DNA Clean-Up Kit. We dephosphorylated p14 digestion with Antarctic Phosphatase, and ran the entire mixture on a gel. That night, we <A href=" https://static.igem.org/mediawiki/2012/4/46/EZNA_Gel_Extraction.pdf
								pdf "> gel extracted </A>the 6kb band with our Qiagen gel extraction kit and set up overnight cultures of a p26a, 027a, p29a, p30a (our Anderson series constitutive promoters), and p14k.
<B>Dephosphorylated p14 digestion:</B><B>July 20th, Friday </B>
								This morning Shweta and Tina <A href="https://static.igem.org/mediawiki/2012/3/35/Ezna_Miniprep.pdf">miniprepped </A>
								p26a, p27a, p29a, p30a (Anderson series constitutive promoters), and p14k from overnight cultures. Dylan set up a 30 minute room temperature ligation to connect the salicylate reporter with digested p21 PCR product and p14 (isolated yesterday, quantified today). Because Shewanella grows more slowly than E.coli, Claire did a second miniprep of p15k and p16k from JG700 in order to submit the plasmid for <A href="https://static.igem.org/mediawiki/2012/0/03/Sequencing_Preparation.pdf "> sequencing </A> to confirm that conjugation was successful. In anticipation of the failure of Dylan's ligation, Swati set up 3 <A href=" https://static.igem.org/mediawiki/2012/2/26/Phusion_PCR.pdf "> Phusion PCR</A> reactions in parallel, each with the same template (p21), and repeated the parameters that had proven successful (no mispriming) previously. Dylan also set up overnight cultures of the strain with the nah operon in pSB1C3, S18 (DH5a + pSB3C5/p17c), and WM3064+p14.
								</DIV></DIV></DIV><DIV class="row"><DIV class="nine columns"><H3>July 22nd - 28th</H3>
							The goal of this week was to transform DH5Î± with nah operon (p20 PCR + p25a, p27a, or p29a).
							<DIV class="panel" style="background:white;margin-top: 20px;"><H6>Daily Details:</H6><B>July 22nd, Sunday </B>
								Danielle and Dylan <A href="https://static.igem.org/mediawiki/2012/a/af/Double_Digest.pdf ">digested </A> p14(Arsenic reporter) and p21(Salicylate sensing region) for ligation. Swati ran a <A href=" https://static.igem.org/mediawiki/2012/2/26/Phusion_PCR.pdf "> Phusion PCR </A> to amplify p21. Then Dylan ran a <A href="https://static.igem.org/mediawiki/2012/6/6c/DNA_Gel_Electrophoresis.pdf">gel electrophoresis</A> and did <A href=" https://static.igem.org/mediawiki/2012/4/46/EZNA_Gel_Extraction.pdf
								pdf "> gel extraction </A> to isolate p21 PCR digest and p14 backbone.
<B>PCR confirmation of p21:</B><B>Digest of p21 PCR an p14:</B><B>July 23rd, Monday </B>
								Dylan quantified the gel extractions, quite successfully with a newly invented protocol. Then he dephosphorylated p14, and submitted some DNA for <A href="https://static.igem.org/mediawiki/2012/0/03/Sequencing_Preparation.pdf "> sequencing </A>. At one point, the Magical Graduated Cylinder of Elmira (500 mL) fell from the sky and shattered in the sink. Some voodoo was clearly in the air. It appears that the ligation products from the day before were either not created or not successfully transformed, as the plates contained no colonies.
								Caleb and Tina prepared more kanamycin and chloramphenicol plates while Mark desalted Dylan's ligation product. Then Dylan, Mark, and Tina transformed DH5Î± with the ligation product, p33k, p14k, and p31c (See: <A href="https://2012.igem.org/Team:Cornell/notebook/strain_list">Strain List</A>).

								<B>July 24th, Tuesday </B>
								In the morning, Dylan did <A href=" https://static.igem.org/mediawiki/2012/2/26/Phusion_PCR.pdf "> Phusion PCR </A> of p20(nah operon). Then Caleb and Dylan did a <A href="https://static.igem.org/mediawiki/2012/a/af/Double_Digest.pdf "> double digestion </A> of p20 PCR and p17c for ligation. Dylan ran a DNA <A href="https://static.igem.org/mediawiki/2012/6/6c/DNA_Gel_Electrophoresis.pdf">gel electrophoresis</A> to confirm that the previous PCR was successful. Steven performed gel extraction to isolate p17c backbone fragment.
<B>Digest of p17 and PCR of p20:</B><B>July 25th, Wednesday </B>
								Mark set up two <A href="https://static.igem.org/mediawiki/2012/a/a7/T4_Ligase_Ligation.pdf ">ligations </A> of a p20 PCR product and p17c, one at standard concentrations and one at very high concentrations. (See: <A href="https://2012.igem.org/Team:Cornell/notebook/strain_list">Strain List</A>
								) p20 PCR was cut at Xbio1 and Spe1, which have compatible sticky ends. This means that p20 PCR can be ligated into backbone in two directions, only one of which is useful. These were desalted and transformed by Mark and Dylan, along with p33k. Meanwhile, Tina and Chie set up <A href="https://static.igem.org/mediawiki/2012/a/af/Double_Digest.pdf "> double digestions </A> of p24a, p20 PCR product, and <A href="https://static.igem.org/mediawiki/2012/f/fb/Single_Digest.pdf"> single digestions </A>
								of p25a, p27a, and p29a.
								Dylan ran two gels of the double and single digestions. The single digestions of the Anderson series promoters showed single bands, which ran slightly faster than the supercoiled DNA control. This may be because the supercoiled DNA was nicked. The gel of the oriT double digestion showed more long bands than expected â possibly a result of star activity â but had a band ~400bp which was interpreted to be the insert of interest.
<B>oriT insert / nah operon:</B><B>Constitutive promoters:</B><B>July 26th, Thursday </B>
								Caleb gel purified the digestion products of p24a, p20 PCR, p25a, p27a, and p29a. Danielle dephosphorylated products of p25a, p27a and p29a in preparation for <A href="https://static.igem.org/mediawiki/2012/a/a7/T4_Ligase_Ligation.pdf ">ligations </A>.
								<B>July 27th, Friday </B>
								In the morning, Claire <A href="https://static.igem.org/mediawiki/2012/3/35/Ezna_Miniprep.pdf">miniprepped </A> p11k, p17c, and p33k from the overnight cultures that Dylan had set up. Meanwhile, Dylan heat killed the ligase from the overnight <A href="https://static.igem.org/mediawiki/2012/a/a7/T4_Ligase_Ligation.pdf ">ligations </A>
								(p25a, p27a and p29a) (See: <A href="https://2012.igem.org/Team:Cornell/notebook/strain_list">Strain List</A>
								), while Caleb prepared to desalt the ligation mixtures before electroporation via drop dialysis. Shortly thereafter, Dylan and Caleb electroporated the four desalted mixtures into DH5Î±, allowing the cells to recover one hour in SOC before plating. If these transformations are successful, we will have cells that carry plasmids expressing the nah operon under the control of three different constitutive promoters with varying strength, as well as a cell line that carries a plasmid that confers chloramphenicol resistance, has a p15a origin of replication, and an origin of transfer. Eventually, the nah operon will be ligated as an insert into such a backbone.
								In the evening, Dylan and Tina started 30 mL overnight cultures of S24, S25, S27, S29, S15, and S17 to miniprep from the next morning. S24 carries a plasmid with an oriT, which will be <A href="https://static.igem.org/mediawiki/2012/3/35/Ezna_Miniprep.pdf">miniprepped </A> in case our previous ligation of an oriT into p17c failed. S25,27, and 29 all carry plasmids with Anderson series constitutive promoters of varying strength, which will be used to set up fluorescent controls to monitor mtrB expression levels. S15 and S17 carry our engineered arsenic reporter plasmids, which will be further modified to facilitate control studies in mtrB expression.
								After setting up the cultures, Dylan and Swati left Weill and headed over to Riley Robb where they assembled three new reactor setups in the Angenent lab. Two of these reactors will be run in continuous flow operation with a BioLogic potentiostat, while the other will be run in batch mode with a CH Instruments potentiostat. Tomorrow, one continuous flow and one batch reactor will be inoculated with wildtype Shewanella oneidensis MR-1, while the remaining continuous flow reactor will be inoculated with our engineered arsenic reporter strain, S20. The purpose of running the continuous flow reactors is to better define the maximum current output we can expect from our induced strains in response to analyte, and to repeat an experiment carried out on the CH potentiostat for basal activity from the uninduced arsenic reporter. Because we were unsure of the veracity of the data from our previous experiment using the CH potentiostat, we are running the batch reactor to see if the observed current response is what we'd expect from wildtype Shewanella.
								<B>July 28th, Saturday </B>
								Today, Swati did six <A href="https://static.igem.org/mediawiki/2012/3/35/Ezna_Miniprep.pdf">minipreps </A> in the morning and it took her four hours...and it paid off! The nanodrop told us that our highest yield was 812.6 ng/uL.
								While Swati was devoting her soul to the E.Z.N.A kit, Dylan continued to set up reactors in the Angenent Lab, making reference electrodes for the three reactors, and getting the pumps set up for the continuous flow reactors. After our weekly team meeting, he inoculated the reactor.
								Swati prepared overnight cultures and reference plates, while Danielle prepared kanamycin plates and did autoclave.
								In the evening, Dylan and Swati did <A href="https://static.igem.org/mediawiki/2012/a/af/Double_Digest.pdf "> digestions </A> for RFP controls and SAL2 reporter. Also, SAL reporter was transformed into WM3064.

								</DIV></DIV></DIV><DIV class="row last-ele"><DIV class="nine columns"><H3>July 29th - 31st</H3>
							SAL (the version of the salicylate reporter with the BamHI cutsite) was conjugated into JG700, and we created SAL2 (salicylate reporter without BamHI cutsite). We prepared constitutive controls and our reporters for fluorescence testing. We also continued cloning work to get the nah operon with a constitutive promoter ready to conjugate into JG700.
							<DIV class="panel" style="background:white;margin-top: 20px;"><H6>Daily Details:</H6><B>July 29th, Sunday </B>
								Apparently, Swati has yet to get over her miniprep craze. While most of the wetlab hiked through Danby State Forest, Swati did more minipreps: p25a, oriT + PSB3C5, and nah operon + p29a. The cultures of S25 and nah p29 were bright magenta, while oriT in PSB3C5 was a pale tan. However, the last of these contained an unidentified crystalline substance, possibly a byproduct of producing the nah operon. Two other cell cultures also didn't grow, which was disheartening, as it meant there were less minipreps for Swati to do. Interspersed between the copious miniprepping were two gels to purify the digestions from the previous evening, all of which appeared to work! Dylan and Claire, returning from their nature trek, joined Swati in lab, finding Shweta and Eric along the way. While Dylan took over Swati's gel extraction, Swati, Shweta, and Claire discussed upcoming outreach events. As Ecologist Claire explained the feeding habits of Tomiculus troutus (AKA Tommy the trout), Eric began to feel left out and, in the process of showing off the website, discovered Dylan's odd love for typography. After spending roughly an hour accomplishing nothing, Dylan and Swati set up liquid cultures in preparation to put SAL into JG700. Dylan returned to his reactor-ridden lair for a short spell.
<B>07/29 gel:</B><B>July 30th, Monday </B>
								In the morning, Dylan set up conjugations between WM3064+SAL and JG700 while Caleb quantified the minipreps and gel extractions from Sunday. After quantification, he submitted nah+p29a and oriT+p17c for sequencing. Dylan and Mark dephosphorylated the digestions of SAL with SpeI &amp; PstI, p14k with SpeI &amp; PstI, p16k with SpeI &amp; PstI, p16k with EcoRI &amp; AscI, and p11k with EcoRI &amp; PstI. Tina and Danielle then set up ligations of mRFP+SAL, mRFP+p14, and mRFP+p16 to put mRFP downstream of the arsenic and salicylate reporters, in order to be able to characterize our induced promoter strengths by measuring fluorescence. We also ligated p16+SAL to make SAL2, the salicylate reporter without the BamHI cutsite. Finally, we ligated p25, p27, and p29 (the Anderson promoters with mRFP) into pBBRBB, in order to be able to put them into Shewanella.

								<B>July 31st, Tuesday </B>
								Mark made DH5Î± electrocompetent stocks. Dylan checked conjugation plates to put SAL into JG700. He then made overnight cultures of JG700+SAL and S31 (PSB1C3, as well as WM3064 to prepare competent stocks. Dylan autoclaved a reactor in Riley-Robb and set it up in batch with M4 media, and inoculated with wild-type S. oneidensis MR-1. The purpose was to repeat a previous batch experiment in order to be sure that it is something about feeding with LB that is decreasing current production. 
Sequencing results came back on nah+p29 and oriT+p17c and things looked good. Consequently, Dylan decided to digest nah+p29 with EcoRI and SpeI, and oriT+p17c with EcoRI and XbaI in order to eventually ligate the nah operon with a constitutive promoter into the p17c backbone, upstream from the oriT sequence (in order to allow the plasmid to be conjugated into Shewanella). Claire prepared these digestions, along with digestions of p31c and p20 PCR product (in order to get the nah operon into pSB1C3). 
Dylan and Caleb desalted the previous night's ligations and than transformed into WM3064, since we will want to get plasmid into Shewanella. 

								</DIV></DIV></DIV><DIV class="row last-ele"><DIV class="three columns"><A href="https://2012.igem.org/Team:Cornell/notebook/wetlab/june#pillTab3">&lt;&lt; Previous</A></DIV><DIV class="three columns offset-by-six"><A href="https://2012.igem.org/Team:Cornell/notebook/wetlab/august#pillTab3" style="display:block;float:right;">Next &gt;&gt;</A></DIV></DIV></LI></UL></DIV><DIV class="printfooter">
Retrieved from &quot;<A href="http://2012.igem.org/Team:Cornell/notebook/wetlab/july">http://2012.igem.org/Team:Cornell/notebook/wetlab/july</A>&quot;</DIV></DIV></DIV><DIV id="footer-box" class="noprint"><DIV id="footer"><UL id="f-list"><LI id="t-recentchanges"><A href="/Special:RecentChanges" title="Recent changes">Recent changes</A></LI><LI id="t-whatlinkshere"><A href="/Special:WhatLinksHere/Team:Cornell/notebook/wetlab/july" title="List of all wiki pages that link here [j]" accesskey="j">What links here</A></LI><LI id="t-recentchangeslinked"><A href="/Special:RecentChangesLinked/Team:Cornell/notebook/wetlab/july" title="Recent changes in pages linked from this page [k]" accesskey="k">Related changes</A></LI><LI id="t-specialpages"><A href="/Special:SpecialPages" title="List of all special pages [q]" accesskey="q">Special pages</A></LI><LI><A href="/Special:Preferences">My preferences</A></LI></UL></DIV><DIV id="footer"><UL id="f-list"><LI id="t-print"><A href="/wiki/index.php?title=Team:Cornell/notebook/wetlab/july&amp;printable=yes" title="Printable version of this page [p]" accesskey="p">Printable version</A></LI><LI id="t-permalink"><A href="/wiki/index.php?title=Team:Cornell/notebook/wetlab/july&amp;oldid=293513" title="Permanent link to this revision of the page">Permanent link</A></LI><LI id="privacy"><A href="/2012.igem.org:Privacy_policy" title="2012.igem.org:Privacy policy">Privacy policy</A></LI><LI id="disclaimer"><A href="/2012.igem.org:General_disclaimer" title="2012.igem.org:General disclaimer">Disclaimers</A></LI></UL></DIV></DIV></DIV></DIV></BODY></HTML>