<HTML xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en" dir="ltr">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki  ltr ns-0 ns-subject page-Team_Cornell_notebook_wetlab_september"><DIV id="globalWrapper"><DIV id="top-section"><DIV id="p-logo"><A href="/Main_Page" title="Main Page">&quot;
	    </A></DIV><DIV id="menubar" class="left-menu noprint"><UL><LI class="selected"><A href="/Team:Cornell/notebook/wetlab/september">Page               </A></LI><LI class="new"><A href="/wiki/index.php?title=Talk:Team:Cornell/notebook/wetlab/september&amp;action=edit&amp;redlink=1">Discussion               </A></LI><LI><A href="/wiki/index.php?title=Team:Cornell/notebook/wetlab/september&amp;action=edit">View source               </A></LI><LI><A href="/wiki/index.php?title=Team:Cornell/notebook/wetlab/september&amp;action=history">History               </A></LI><LI style="color:#808080;cursor:default">teams</LI></UL></DIV><DIV class="right-menu noprint" id="menubar"><UL><LI id="pt-login"><A href="/wiki/index.php?title=Special:UserLogin&amp;returnto=Team:Cornell/notebook/wetlab/september" title="You are encouraged to log in; however, it is not mandatory [o]" accesskey="o">Log in</A></LI></UL></DIV><DIV id="search-controls" class="noprint"><FORM action="/Special:Search" id="searchform"> </FORM></DIV></DIV><DIV id="content"><H1 class="firstHeading">Team:Cornell/notebook/wetlab/september</H1><DIV id="bodyContent"><H3 id="siteSub" class="noprint">From 2012.igem.org</H3><DIV class="row header off-canvas"><DIV class="twelve columns centered header no-pad" id="header-wrapper"><UL class="nav-bar"><LI class="pg-home"><A href="https://2012.igem.org/Team:Cornell">Home</A></LI><LI class="has-flyout pg-project pg-project_background pg-project_wetlab pg-project_drylab pg-project_hprac pg-project_gallery pg-project_safety"><A href="https://2012.igem.org/Team:Cornell/project">Project</A></LI><LI class="pg-project"><A href="https://2012.igem.org/Team:Cornell/project">Overview</A></LI><LI class="pg-project_background"><A href="https://2012.igem.org/Team:Cornell/project/background">Background</A></LI><LI class="pg-project_wetlab"><A href="https://2012.igem.org/Team:Cornell/project/wetlab">Wet Lab</A></LI><LI class="pg-project_drylab"><A href="https://2012.igem.org/Team:Cornell/project/drylab">Dry Lab</A></LI><LI class="pg-project_hprac"><A href="https://2012.igem.org/Team:Cornell/project/hprac">Human Practices</A></LI><LI class="pg-project_gallery"><A href="https://2012.igem.org/Team:Cornell/project/gallery">Gallery</A></LI><LI class="pg-project_safety"><A href="https://2012.igem.org/Team:Cornell/project/hprac/safety">Safety</A></LI></UL><LI class="has-flyout pg-notebook pg-notebook_wetlab pg-notebook_drylab pg-notebook_strainlist"><A href="https://2012.igem.org/Team:Cornell/notebook">Notebook</A></LI><LI class="pg-notebook"><A href="https://2012.igem.org/Team:Cornell/notebook">Overview</A></LI><LI class="pg-notebook_wetlab"><A href="https://2012.igem.org/Team:Cornell/notebook/wetlab">Wet Lab</A></LI><LI class="pg-notebook_drylab"><A href="https://2012.igem.org/Team:Cornell/notebook/drylab/june">Dry Lab</A></LI><LI class="pg-notebook_strainlist"><A href="https://2012.igem.org/Team:Cornell/notebook/strain_list" target="_blank">Strain List</A></LI><LI class="pg-protocols"><A href="https://2012.igem.org/Team:Cornell/protocols">Protocols</A></LI><LI class="has-flyout pg-team pg-team_bios pg-team_sponsors"><A href="https://2012.igem.org/Team:Cornell/team">Team</A></LI><LI class="pg-team"><A href="https://2012.igem.org/Team:Cornell/team">Overview</A></LI><LI class="pg-team_bios"><A href="https://2012.igem.org/Team:Cornell/team/bios">Bios</A></LI><LI><A href="https://igem.org/Team.cgi?year=2012&amp;team_name=Cornell" target="_blank">Profile</A></LI><LI class="pg-team_sponsors"><A href="https://2012.igem.org/Team:Cornell/team/sponsors">Sponsors</A></LI><LI class="has-flyout pg-outreach pg-outreach_presentations pg-outreach_panels pg-outreach_awareness"><A href="https://2012.igem.org/Team:Cornell/outreach">Outreach</A></LI><LI class="pg-outreach"><A href="https://2012.igem.org/Team:Cornell/outreach">Overview</A></LI><LI class="pg-outreach_presentations"><A href="https://2012.igem.org/Team:Cornell/outreach/presentations">Research Presentations</A></LI><LI class="pg-outreach_panels"><A href="https://2012.igem.org/Team:Cornell/outreach/panels">Panel Discussions</A></LI><LI class="pg-outreach_awareness"><A href="https://2012.igem.org/Team:Cornell/outreach/awareness">Public Awareness</A></LI><LI class="pg-badge"><A href="https://2013.igem.org/Team:Cornell" style="margin-top:-50px">2012: SAFE BET</A></LI><LI class="pg-badge_home"><A href="http://igem.engineering.cornell.edu">Team Website</A></LI><LI class="pg-badge_2013"><A href="https://2013.igem.org/Team:Cornell">2013: Organofoam</A></LI><LI class="pg-badge_2011"><A href="https://2011.igem.org/Team:Cornell">2011: Biofactory</A></LI><LI class="pg-badge_2010"><A href="https://2010.igem.org/Team:Cornell">2010: OMG OMVs</A></LI><LI class="pg-badge_2009"><A href="https://2009.igem.org/Team:Cornell" style="font-size:20px !important">2009: Cadmium Sensor</A></LI></DIV></DIV><DIV class="row"><DIV class="two columns"><UL class="side-nav ntbk-nav"><LI><H6>Wet Lab</H6></LI><LI><A href="https://2012.igem.org/Team:Cornell/notebook/wetlab">Overview</A></LI><LI><A href="https://2012.igem.org/Team:Cornell/notebook/wetlab/bootcamp">Bootcamp</A></LI><LI><A href="https://2012.igem.org/Team:Cornell/notebook/wetlab/june">June</A></LI><LI><A href="https://2012.igem.org/Team:Cornell/notebook/wetlab/july">July</A></LI><LI><A href="https://2012.igem.org/Team:Cornell/notebook/wetlab/august">August</A></LI><LI class="active"><A href="https://2012.igem.org/Team:Cornell/notebook/wetlab/september">September</A></LI><LI><A href="https://2012.igem.org/Team:Cornell/notebook/wetlab/wrap_up">Wrap Up</A></LI></UL></DIV><DIV class="ten columns team-bios-container"><DL class="tabs pill" style="display:block;float:right;width:390px;margin: -10px -20px 10px 0px;"><DD class="active"><A href="#pillTab1">Weekly Update</A></DD><DD><A href="#pillTab2">Daily Details</A></DD><DD><A href="#pillTab3">Both</A></DD></DL><DIV class="row"><DIV class="twelve columns"><H2 class="centered">Wet Lab - September</H2></DIV></DIV><UL class="tabs-content"><LI class="active" id="pillTab1Tab"><DIV class="row last-ele" style="margin-top:20px;"><DIV class="three columns"><A href="https://2012.igem.org/Team:Cornell/notebook/wetlab/august#pillTab1">&lt;&lt; Previous</A></DIV><DIV class="three columns offset-by-six"><A href="https://2012.igem.org/Team:Cornell/notebook/wetlab/wrap_up" style="display:block;float:right;">Next &gt;&gt;</A></DIV></DIV><DIV class="row"><DIV class="nine columns"><H3>September 1st-8th</H3>
							Conjugating nah into Shewy: The mysterious failure of past week's ligation lead to a redigestion (in case we run out of DNA) and religation of last week's digestions. Nah to oriT was ligated in ratios of 3:1 and 6:1 - when run on a gel, bands once more failed to appear... But wait! Epiphany! The team realized that we forgot to dephosphorylate the oriT backbone! After that silly mistake, 1:1, 3:1, and 6:1 ratios of nah:oriT were set up in a ligation mixture and electroporated into WM3064. Once again, slow growth and tiny colonies were observed - perhaps due to the strain of putting such a massive operon in a cell. 13 colonies were deemed large enough to try colony PCR, and 3 of them definitely showed bands that corresponded to the length of our construct. We set up liquid cultures for Miniprepping and once again observed slow growth. We tried subculturing - but still, general opaqueness. Miniprepping proceeded anyway, as we are a fairly optimistic bunch (and the cultures were by then over 24 hours old).
							
							Site-directed mutagenesis: After several more unsuccessful attempts, lengthy discussion among teammates, and consultation with several graduate advisors, the team decided to put this subproject on hold for the time being. Swati finished up with one last attempt, then joined Claire for fluorescence testing. That's all, folks! <A href="#" class="technical-desc" for="#technical-desc1" style="display:block;margin-top:20px;">Daily Details</A><DIV class="hide-me panel" style="background:white;margin-top: 20px;" id="technical-desc1"><H6>Daily Details:</H6><P><STRONG>September 1</STRONG></P><P><STRONG>Conjugating nah into Shewy:</STRONG>

									Due to the mysterious failings of our attempts to ligate the nah operon, Tina tried to start fresh by re-digesting the nah operon-containing plasmid and the oriT-containing backbone.
								</P><P><STRONG>Synthetic River Media:</STRONG>

									Mark and Chie created synthetic river media based on information on the mineral content of the Athabasca River, and started overnight cultures of numerous strains to test how sodium lactate supplemented river water would grow the cells.
								</P><P><STRONG>Site-directed mutagenesis:</STRONG> Swati transformed both her mutagenic PCR and her DpnI digest into DH5a after desalting on a membrane. She also ran both of the aforementioned samples on a gel without purifying, in order to check if either the digestion or purification steps were problematic. The gel was yet again blank, suggesting that the PCR itself was the problem.
								</P><P><STRONG>September 2nd</STRONG></P><P><STRONG>Conjugating nah into Shewy:</STRONG>

									Tina tried ligating an older digestion of the nah operon into the oriT backbone using a molar ratio or 3:1 and 6:1 for nah:backbone. She then de-salted and transformed the ligations into conjugation strain WM3064. Tina then ran a gel of the digested nah operon-containing plasmid and the oriT-containing backbone and extracted the nah operon and oriT backbone bands.
								</P><B>From left: Ladder, 5 ug backbone, 5 ug backbone, 5 ug nah operon, 5 ug nah operon, 5 ug nah operon, 5 ug nah operon, empty, empty, empty:</B><P><STRONG>Synthetic River Media:</STRONG>

									Mark set up a 96-well plate to test growth of the engineered strains in sodium lactate at varying concentrations, in synthetic river media.
								</P><P><STRONG>Site-directed mutagenesis:</STRONG>The third transformation attempt yielded no colonies on the plate of DpnI-digested DNA, but three colonies on the PCR only plate, suggesting that these were just excess template. Nonetheless, Swati set up cultures to miniprep the following day. She also realized that she had been using a template that was 10x more concentrated than it was supposed to be. She proceeded to set up a fourth attempt at a mutagenic PCR, this time with the correct template concentration, primers for mutating a different cutsite than she had previously been attempting, and a higher annealing temperature.
								</P><P><STRONG>September 3rd</STRONG></P><P><STRONG>Conjugating nah into Shewy:</STRONG>

									Tina extracted the nah operon digest and the oriT backbone from yesterday's gel. Tina and Dylan had an epiphany - we forgot to dephosphorylate the backbone! No wonder we kept seeing self-ligations! With a renewed spirit, Tina dephosphorylated the digested oriT backbone then started a sixteen hour 16 degrees Celsius ligation of the extracted digests. Tina tried nah to backbone ratios of 1:1, 3:1, and 6:1.
								</P><P><STRONG>Synthetic River Media:</STRONG>

									Mark returned to the lab to discover that the growth assay went wrong. Numerous issues were identified with the original protocol to explain the failed result, and the protocol was changed.
								</P><P><STRONG>Site-directed mutagenesis:</STRONG> Swati miniprepped the three suspicious PCR only colonies from the previous day, and digested with PstI to check for successful mutagenesis. She also did a DpnI digest and transformation of the PCR from the previous day. She ran the new PCR &amp; DpnI digest, along with the PstI-digested minipreps of the three earlier colonies, on a gel, but to no avail. The new things showed nothing (again), and the digested minipreps produced completely incorrect bands.
								</P><P><STRONG>September 4th</STRONG></P><P><STRONG>Conjugating nah into Shewy:</STRONG>

									Tina desalted yesterday's ligations then transformed them into our conjugation strain WM3064. WM3064 is E. coli that is auxotrophic for DAP (diaminopimelic acid epimerase).
								</P><P><STRONG>Site-directed mutagenesis:</STRONG> The latest transformation did not work. Thus concludes this subproject!
								</P><P><STRONG>September 5th</STRONG></P><P><STRONG>Conjugating nah into Shewy:</STRONG>

									Caleb and Tina checked the transformation plates - some colonies were visible on all three ligation ratio pltaes, but the colonies were so tiny we decided to wait another day before trying colony PCRs or starting liquid cultures.
								</P><P><STRONG>September 6th</STRONG></P><P><STRONG>Conjugating nah into Shewy:</STRONG>

									Caleb decided the transformation colonies were large enough to colony PCR and start liquid cultures from. 13 total colony PCRs were performed on colonies from all three ligation mixtures and one positive control was run in parallel to be sure the PCR conditions were correct. Tina ran a gel of the the colony PCRs and the positive control. As can be seen in the gel pictures, colonies 3, 4, and 8 definitely had the nah operon and many other colonies had matching middle bands. These matching middle bands likely indicate some sort of mispriming. Colony 5 was clearly the product of an oriT backbone self-ligation. Caleb and Tina started 15 mL liquid cultures of apparently nah containing colonies 3, 4, and 8 and non-nah-containing colony 5 to Miniprep from tomorrow.
								</P><B>From top left: Ladder, colony 1, colony 2, colony 3, positive control. From bottom left: Ladder, colony 4, colony 5, colony 6, colony 7, positive control:</B><B>From left: Ladder, colony 8, colony 9, colony 10, colony 11, colony 12, colony 13, positive control:</B><P><STRONG>Fluorescence tests:</STRONG> This week started late as Claire was getting over a massive cold. However, with new vim and vigor and her trusty labmate Swati, they set forth to decide the details of how they will run fluorescence tests. To get ready for this, they set up cultures of JG700 and E. coli constitutively producing mRFP under the control of Anderson series promoters.
								</P></DIV></DIV></DIV><DIV class="row"><DIV class="nine columns"><H3>September 9th-15th</H3>
							Conjugating nah into Shewy: Unfortunately, the miniprep yields from last week were all low except one shining colony - which didn't have the nah operon in it. We ran it to test our hypothesis that slow growth of E Coli was due to the added stress on the cell of having such a large operon in it - our data so far matches our hypothesis, as that colony grew faster in culture and the plate and also yielded substantially more DNA after miniprepping.
							
							Site-directed mutagenesis: Though we were unable to confirm the presence of the nah operon with sequencing (as the miniprep failed), we decided to proceed with conjugation into Shewanella just in case the transformation succeeded. We tried to grow one of the colonies with a very strong band from last week's gel in a liquid culture, and once again, observed slow growth. We conjugated into JG700 and JG700 with SAL, a plasmid with our salicylate reporter system. Liquid cultures were made from the conjugation - unfortunately, there was no growth. We streaked new plates with the old transformation in the hopes that something would finally grow!
							<A href="#" class="technical-desc" for="#technical-desc7" style="display:block;margin-top:20px;">Daily Details</A><DIV class="hide-me panel" style="background:white;margin-top: 20px;" id="technical-desc7"><H6>Daily Details:</H6><B>September 9th</B><B>Conjugating nah into Shewy:</B>
								Caleb checked the miniprep yields today - all but colony 5 had yields below 90 ug/uL. Colony 5 had a yield of over 500 ug/uL. Our hypothesis as to why the E. coli was growing so slowly was that the E. coli was stressed from having to express the many nah operon proteins. Evidence suuporting our hypothesis includes - colony 5 had a much larger colony, grew in liquid much faster, came out as lacking the nah operon, and yielded a lot more DNA from miniprepping than the colonies that supposedly had the nah operon.
								<B>Synthetic River Media:</B>
								Again, the result of the growth assay was failure. This time, it appeared that the only issue was blanking, as the negative controls were reading higher than the inoculated samples. After some discussion with Dr. Archer, a better blanking method was determined, as well as the addition of longer mixing to the protocol.
								<B>September 10th</B><B>Fluorescence tests: </B>
								We dephosphorylated SAL2 digested with SpeI and PstI for one hour, then heat killed at 65degC for 10min. We then ligated overnight with mRFP (also digested with SpeI and PstI) at 16degC in the thermocycler.
								However, JG700 and p41k didn’t grow, so we moved the plates to Riley Robb in the hopes that a more controlled incubator meant for Shewenella will help them thrive.
								<B>September 11th</B><B>Fluorescence tests:</B>
								Swati transformed 2uL of SAL2_mRFP ligation mixture into WM3064. We also made cultures of JG700, WM3064, and p39k, p40k, and p41k in both JG700 and WM3064. We are hoping to run a plate with both strains on it to get an idea of whether we can see mRFP constitutively produced in JG700 (which is naturally red and may have some background signal), and compare that fluorescence to constitutively produced mRFP in WM3064. Once we have this control data we hope to be able to correlate fluorescence in JG700 with promoter strength.
								<B>September 12th</B><B>Conjugating nah into Shewy:</B>
								Caleb and Tina started cultures of colony #3 from original transformation plate and cultures of JG700 + SAL and JG700. JG700 is Shewanella oneidensis with a ΔmtrB genotype. SAL refers to a plasmid with our reporter system that responds to the presence of salicylate by upregulating expression of mtrB.
								<B>Fluorescence tests:</B>
								Cultures of JG700 p39k and p41k didn’t grow. We made new cultures with giant swabs of cells from the plates to see if the plates are dead, and subcultured from all the other cultures started yesterday.
								There are also two colonies on our SAL2_mRFP transformation plate, so we made 25mL cultures from these to see if we ligated successfully.
								<B>September 13th</B><B>Conjugating nah into Shewy:</B>
								Caleb started liquid cultures of colonies #3 and #4 from the original transformation plate. Caleb also started a conjugation of JG700 + SAL with colony #3 and JG700 + colony #3. The conjugation was at room temperature on a DAP plate for 16.5 hours.
								<B>Fluorescence:</B>				Swati miniprepped the SAL2_mRFP cultures, and Claire set up a PCR to confirm whether SAL2_RFP was ligated successfully. Unfortunately it doesn’t look like the ligations worked, as neither band is ~4.5kb, which should be the length for SAL2 with mRFP. We think that the old digestions we are using may not be good, so we will start from scratch next time.
								<B>September 14th</B><B>Conjugating nah into Shewy:</B>
								Caleb checked the liquid cultures from yesterday and noted there was no growth, as expected, due to the nah operon stressing the cells. Caleb streaked new plates from the conjugation - the LB agar plates had kanamycin and chloramphenicol for the JG700 + SAL and colony #3 conjugation and just chloramphenicol for the JG700 and colony #3 conjugation. The plasmid in the SAL strain confers kanamycin resistance and the plasmid in colony #3 with the nah operon confers chloramphenicol resistance. The strain of E. coli in colony #3 was auxotrophic for DAP, so the E. coli couldn't grow on the newly streaked plates.
								<B>From left: Ladder, SAL2_mRFP (1), SAL2_mRFP (2), mut2 pst1, mut2 miniprep:</B><B>Synthetic River Media:</B>
								Cultures of only JG700 and Sal1 containing cells were started today, for another attempt at the growth assay.
								<B>September 15th</B><B>Running Reactors:</B>
								Because maximal current production did not increase, Dylan took down reactors with working electrodes poised at 0.35V with respect the the Ag/AgCl reference electrode in order to free up potentiostat channels so that we can begin characterizing our arsenic reporter strains.
							</DIV></DIV></DIV><DIV class="row"><DIV class="nine columns"><H3>September 16th-22nd</H3>
							Conjugating nah into Shewy: Some colonies were observed from last week's restreaking! A massive 15 colony liquid culture was set up, but to no avail. After five long days, we were forced to conclude that conjugation failed. Sadface. Good thing we had extra 3:1 nah to oriT ligation sitting around, which we transformed into both WM3064 and DH5a, the latter of which we hoped would grow faster. 16 colonies from the Sep 4th ligation were also restreaked. And lo! Every single restreak had colonies the next day, which warranted a massive 16 colony colony PCR (try saying that five times fast). Interestingly, the WM3064 plates grew, but the DH5a transformation failed, which we concluded was due to a bad electrocompetent cell stock (we noticed weird white precipitate in the cell mixture before).
							<A href="#" class="technical-desc" for="#technical-desc2" style="display:block;margin-top:20px;">Daily Details</A><DIV class="hide-me panel" style="background:white;margin-top: 20px;" id="technical-desc2"><H6>Daily Details:</H6><B>September 16th</B><B>Conjugating nah into Shewy:</B>
								Caleb checked the streaked conjugation plates - there were some colonies, but they were all on top of the initial streak. He and Tina started liquid cultures of all 15 colonies.
								<B>Synthetic River Media:</B>
								After finalizing the growth assay protocol, another experiment was set up, this time with only two strains (JG700 and Sal1) in triplicate with sodium lactate concentrations (varying from none to full M4 media concentrations) in synthetic Athabasca river water, plus positive LB controls and blanks. The assay was set to run at room temperature for 20 hours, taking data every 5 minutes with 30 seconds of mixing before each read. Blanking was done by group, for each media mixture.
								<B>Fluorescence tests:</B>
								Started liquid cultures of JG700, p25a, and p29a in order to re-conjugate these constitutively produced mRFP plasmids into Shewanella. p25a on a pBBRBB backbone in WM3064 corresponds to p39k in JG700, and p29a corresponds to p41k.
								<B>September 17th</B><B>Running Reactors:</B> After autoclaving a reactor setup and getting peristaltic pumps ready for a new continuous flow experiment, Dylan started a liquid culture of an arsenic reporter strain (JG700 + p14k) to be inoculated into reactors.
								<B>Conjugating nah into Shewy:</B>
								Caleb checked the liquid cultures started yesterday - no growth.
								<B>Synthetic River Media:</B>
								The growth assay was successful, with constant and low OD600 in the blank wells, and a standard growth curve in the LB wells. The data suggests that lowering the sodium lactate from M4 concentrations is impossible. Therefore, a solution containing 5% volume/volume of 40% sodium lactate weight/volume is what is necessary to maintain an OD600 of approximately 0.1 for an extended period of time. However, the supported OD600 may be higher in the final device at this concentration, due to the constant flow of new sodium lactate.
								<B>Fluorescence tests:</B>
								We redigested SAL2 and mRFP with SpeI and PstI-HF to try to clone mRFP into our salicylate reporter, so that we can do fluorescence experiments with the two salicylate parts. Swati also plated WM3064 with JG700 for conjugation of the constitutively produced mRFP.
								<B>September 18th</B><B>Conjugating nah into Shewy:</B>
								Caleb checked the liquid cultures started on the 16th - no growth.
								Fluorescence tests: We ran mRFP on a gel and extracted a 800bp band which was the appropriate link for mRFP. After enzyme purifying the SAL2 digestion and dephosphorylating, Claire set up an overnight ligation of SAL2 and mRFP.
								<B>September 19th</B><B>Conjugating nah into Shewy:</B>
								Caleb checked the liquid cultures started on the 16th - no growth.
								Fluorescence tests: Claire transformed the SAL2_mRFP ligation into WM3064, and set up cultures of p39k and p41k from the conjugation plate.
								<B>September 20th</B><B>Conjugating nah into Shewy:</B>
								Caleb checked the liquid cultures started on the 16th - no growth.
								Fluorescence test: Swati miniprepped cultures of p39k and p41k from the conjugation plates so that we can PCR to confirm if we have successfully conjugated constitutive mRFP into Shewanella.
								<B>September 21st</B><B>Conjugating nah into Shewy:</B>
								Caleb checked the liquid cultures started on the 16th - no growth. He concluded that the conjugation failed. Caleb and Tina electroporated the 3:1 (nah:oriT backbone) ligation of dephosphorylated and digested nah with the oriT backbone from September 3rd into both WM3064 and a DH5α strain. They then streaked 16 colonies from the September 4th transformation onto fresh plates.
								<B>September 22nd</B><B>Conjugating nah into Shewy:</B>
								Caleb and Tina noted the appearance of the re-streaked plates - every single streak had numerous colonies. They started liquid cultures and started an overnight colony PCR of all 16 of them. Included with the colony PCRs was a positive control that we knew would have the same amplicon from the nah operon if the colonies were the result of a successful ligation. The electroporated WM3064 plate grew with numerous isolated colonies; the DH5α transformation failed. We hypothesized the DH5α cells were at fault due to their appearance before electroporation - white precipitate was floating in the cell mixture.
								<B>Fluorescence tests:</B> Using samples from the 20th, Swati did a Phusion PCR to see if we successfully conjugated p39k and p41k into Shewanella. She also miniprepped WM3064 SAL2RFP and did PCR to see if that ligation was successful.
							</DIV></DIV></DIV><DIV class="row last-ele"><DIV class="nine columns"><H3>September 23rd-30th</H3>
							Conjugating nah into Shewy: A gel of last week's colony PCRs hinted that one of the colonies had the nah operon in it! Excitement! Liquid cultures were made, and there was just enough DNA after miniprepping to submit for sequencing. We will collectively hold our breaths and wait for positive results.
							
							This week, we were able to begin characterizing the current response to arsenic of S20 (JG700 + p14k; see strain list). Additionally, we submitted physical DNA for six BioBrick parts to the parts registry.
							<A href="#" class="technical-desc" for="#technical-desc3" style="display:block;margin-top:20px;">Daily Details</A><DIV class="hide-me panel" style="background:white;margin-top: 20px;" id="technical-desc3"><H6>Daily Details:</H6><B>September 23rd</B><B>Conjugating nah into Shewy:</B>
								Caleb and Tina ran a gel of the colony PCRs and positive control PCR. Swati also started cultures of our arsenic-sensitive strain, as well as control strains, for a plate testing our arsenic reporters.
								<B>From top-left: ladder, positive control, 8 test colonies. From bottom-left: ladder, 8 test colonies, and Swati's SAL2 RFP:</B><B>From left: ladder, empty, p39k PCR, p41k PCR, empty, SAL2_mRFP PCR:</B><B>September 24th</B><B>Conjugating nah into Shewy:</B>
								Caleb inoculated 30 mL LB+DAP+Cm with the 24 hour liquid culture of of colony N.
								<B>September 25th</B><B>Fluorescence tests:</B> Swati set up a conjugation of SAL2_mRFP into JG700. Also, sequencing came back good so once we have this part in Shewanella we can start testing our salicylate sensor. Jim and Swati also set up a plate to test our arsenic sensing parts at different concentrations of arsenic – they ran a plate with a blank LB column, five control columns (JG700, MR-1, p39k, p40k, and p41k), and three columns of each of our arsenic-sensitive strains. To each row they added a different concentration of arsenic, going from 0uM to 5uM arsenic. The final OD of 100uL in each well was 0.8, and the plate was left overnight in the plate reader.
								<B>Conjugating nah into Shewy:</B>
								Caleb miniprepped the 36 hour 30 mL culture of colony N. The size of the pellet after the first spin step was similar to the size of a ~5mL normal culture, consequently, the yield was only 96.7 ng/uL.
								<B>September 27th</B><B>Running Reactors:</B> For the first time, we added arsenic to our reactors. Specifically, once Dylan saw that the two reactors inoculated with an arsenic reporter strain (JG700+p14k) had been producing steady current for several retention times, he dosed the media vessel to a final concentration of 10 μM sodium arsenite.
								<B>Conjugating nah into Shewy:</B>
								Fortunately, there was enough DNA for Caleb to submit the miniprepped DNA from colony N for Sanger sequencing today. Tina started 3 x (1mL cultures of colony N).
								<B>Fluorescence tests:</B> Claire and Swati set up another control plate with constitutively produced mRFP in both WM3064 and JG700. Since using 100uL starting at and OD of 0.8 seemed to work well for the arsenic plate, we set up the control plate with the same parameters and left it in the plate reader overnight.
								<B>September 28th</B><B>Running Reactors:</B> Since several retention times passed without any current response to 10 μM sodium arsenite, Dylan increased the concentration in the media vessel, hitting our reactors with 100 μM arsenite.
								<B>Fluorescence tests:</B> We noticed that the arsenic data did not seem to have any signal above background, and realized that it may have been because the plate reader wasn’t set to read a clear-bottomed plate. We concluded that, since from the test done on the 27th we could see distinct difference between fluorescence levels of constitutively produced mRFP, our experimental procedure is good and the plate reader was just on the wrong setting. Additionally, since the bioreactors are not getting upregulation of arsenic up to 10uM, we decided next time to test higher concentrations of arsenic (0uM to 500uM)
							</DIV></DIV></DIV><DIV class="row last-ele"><DIV class="three columns"><A href="https://2012.igem.org/Team:Cornell/notebook/wetlab/august#pillTab1">&lt;&lt; Previous</A></DIV><DIV class="three columns offset-by-six"><A href="https://2012.igem.org/Team:Cornell/notebook/wetlab/wrap_up" style="display:block;float:right;">Next &gt;&gt;</A></DIV></DIV></LI><LI id="pillTab2Tab"><DIV class="row last-ele" style="margin-top:20px;"><DIV class="three columns"><A href="https://2012.igem.org/Team:Cornell/notebook/wetlab/august#pillTab2">&lt;&lt; Previous</A></DIV><DIV class="three columns offset-by-six"><A href="https://2012.igem.org/Team:Cornell/notebook/wetlab/wrap_up" style="display:block;float:right;">Next &gt;&gt;</A></DIV></DIV><DIV class="row"><DIV class="nine columns"><H3>September 1st-8th</H3><P><STRONG>September 1</STRONG></P><P><STRONG>Conjugating nah into Shewy:</STRONG>

								Due to the mysterious failings of our attempts to ligate the nah operon, Tina tried to start fresh by re-digesting the nah operon-containing plasmid and the oriT-containing backbone.
							</P><P><STRONG>Synthetic River Media:</STRONG>

								Mark and Chie created synthetic river media based on information on the mineral content of the Athabasca River, and started overnight cultures of numerous strains to test how sodium lactate supplemented river water would grow the cells.
							</P><P><STRONG>Site-directed mutagenesis:</STRONG> Swati transformed both her mutagenic PCR and her DpnI digest into DH5a after desalting on a membrane. She also ran both of the aforementioned samples on a gel without purifying, in order to check if either the digestion or purification steps were problematic. The gel was yet again blank, suggesting that the PCR itself was the problem.
							</P><P><STRONG>September 2nd</STRONG></P><P><STRONG>Conjugating nah into Shewy:</STRONG>

								Tina tried ligating an older digestion of the nah operon into the oriT backbone using a molar ratio or 3:1 and 6:1 for nah:backbone. She then de-salted and transformed the ligations into conjugation strain WM3064. Tina then ran a gel of the digested nah operon-containing plasmid and the oriT-containing backbone and extracted the nah operon and oriT backbone bands.
							</P><B>From left: Ladder, 5 ug backbone, 5 ug backbone, 5 ug nah operon, 5 ug nah operon, 5 ug nah operon, 5 ug nah operon, empty, empty, empty:</B><P><STRONG>Synthetic River Media:</STRONG>

								Mark set up a 96-well plate to test growth of the engineered strains in sodium lactate at varying concentrations, in synthetic river media.
							</P><P><STRONG>Site-directed mutagenesis:</STRONG>The third transformation attempt yielded no colonies on the plate of DpnI-digested DNA, but three colonies on the PCR only plate, suggesting that these were just excess template. Nonetheless, Swati set up cultures to miniprep the following day. She also realized that she had been using a template that was 10x more concentrated than it was supposed to be. She proceeded to set up a fourth attempt at a mutagenic PCR, this time with the correct template concentration, primers for mutating a different cutsite than she had previously been attempting, and a higher annealing temperature.
							</P><P><STRONG>September 3rd</STRONG></P><P><STRONG>Conjugating nah into Shewy:</STRONG>

								Tina extracted the nah operon digest and the oriT backbone from yesterday's gel. Tina and Dylan had an epiphany - we forgot to dephosphorylate the backbone! No wonder we kept seeing self-ligations! With a renewed spirit, Tina dephosphorylated the digested oriT backbone then started a sixteen hour 16 degrees Celsius ligation of the extracted digests. Tina tried nah to backbone ratios of 1:1, 3:1, and 6:1.
							</P><P><STRONG>Synthetic River Media:</STRONG>

								Mark returned to the lab to discover that the growth assay went wrong. Numerous issues were identified with the original protocol to explain the failed result, and the protocol was changed.
							</P><P><STRONG>Site-directed mutagenesis:</STRONG> Swati miniprepped the three suspicious PCR only colonies from the previous day, and digested with PstI to check for successful mutagenesis. She also did a DpnI digest and transformation of the PCR from the previous day. She ran the new PCR &amp; DpnI digest, along with the PstI-digested minipreps of the three earlier colonies, on a gel, but to no avail. The new things showed nothing (again), and the digested minipreps produced completely incorrect bands.
							</P><P><STRONG>September 4th</STRONG></P><P><STRONG>Conjugating nah into Shewy:</STRONG>

								Tina desalted yesterday's ligations then transformed them into our conjugation strain WM3064. WM3064 is E. coli that is auxotrophic for DAP (diaminopimelic acid epimerase).
							</P><P><STRONG>Site-directed mutagenesis:</STRONG> The latest transformation did not work. Thus concludes this subproject!
							</P><P><STRONG>September 5th</STRONG></P><P><STRONG>Conjugating nah into Shewy:</STRONG>

								Caleb and Tina checked the transformation plates - some colonies were visible on all three ligation ratio pltaes, but the colonies were so tiny we decided to wait another day before trying colony PCRs or starting liquid cultures.
							</P><P><STRONG>September 6th</STRONG></P><P><STRONG>Conjugating nah into Shewy:</STRONG>

								Caleb decided the transformation colonies were large enough to colony PCR and start liquid cultures from. 13 total colony PCRs were performed on colonies from all three ligation mixtures and one positive control was run in parallel to be sure the PCR conditions were correct. Tina ran a gel of the the colony PCRs and the positive control. As can be seen in the gel pictures, colonies 3, 4, and 8 definitely had the nah operon and many other colonies had matching middle bands. These matching middle bands likely indicate some sort of mispriming. Colony 5 was clearly the product of an oriT backbone self-ligation. Caleb and Tina started 15 mL liquid cultures of apparently nah containing colonies 3, 4, and 8 and non-nah-containing colony 5 to Miniprep from tomorrow.
							</P><B>From top left: Ladder, colony 1, colony 2, colony 3, positive control. From bottom left: Ladder, colony 4, colony 5, colony 6, colony 7, positive control:</B><B>From left: Ladder, colony 8, colony 9, colony 10, colony 11, colony 12, colony 13, positive control:</B><P><STRONG>Fluorescence tests:</STRONG> This week started late as Claire was getting over a massive cold. However, with new vim and vigor and her trusty labmate Swati, they set forth to decide the details of how they will run fluorescence tests. To get ready for this, they set up cultures of JG700 and E. coli constitutively producing mRFP under the control of Anderson series promoters.
							</P></DIV></DIV><DIV class="row"><DIV class="nine columns"><H3>September 9th-15th</H3><B>September 9th</B><B>Conjugating nah into Shewy:</B>
							Caleb checked the miniprep yields today - all but colony 5 had yields below 90 ug/uL. Colony 5 had a yield of over 500 ug/uL. Our hypothesis as to why the E. coli was growing so slowly was that the E. coli was stressed from having to express the many nah operon proteins. Evidence suuporting our hypothesis includes - colony 5 had a much larger colony, grew in liquid much faster, came out as lacking the nah operon, and yielded a lot more DNA from miniprepping than the colonies that supposedly had the nah operon.
				<B>Synthetic River Media:</B>
							Again, the result of the growth assay was failure. This time, it appeared that the only issue was blanking, as the negative controls were reading higher than the inoculated samples. After some discussion with Dr. Archer, a better blanking method was determined, as well as the addition of longer mixing to the protocol.
							<B>September 10th</B><B>Fluorescence tests: </B>
							We dephosphorylated SAL2 digested with SpeI and PstI for one hour, then heat killed at 65degC for 10min. We then ligated overnight with mRFP (also digested with SpeI and PstI) at 16degC in the thermocycler.
							However, JG700 and p41k didn’t grow, so we moved the plates to Riley Robb in the hopes that a more controlled incubator meant for Shewenella will help them thrive.
							<B>September 11th</B><B>Fluorescence tests:</B>
							Swati transformed 2uL of SAL2_mRFP ligation mixture into WM3064. We also made cultures of JG700, WM3064, and p39k, p40k, and p41k in both JG700 and WM3064. We are hoping to run a plate with both strains on it to get an idea of whether we can see mRFP constitutively produced in JG700 (which is naturally red and may have some background signal), and compare that fluorescence to constitutively produced mRFP in WM3064. Once we have this control data we hope to be able to correlate fluorescence in JG700 with promoter strength.
							<B>September 12th</B><B>Conjugating nah into Shewy:</B>
							Caleb and Tina started cultures of colony #3 from original transformation plate and cultures of JG700 + SAL and JG700. JG700 is Shewanella oneidensis with a ΔmtrB genotype. SAL refers to a plasmid with our reporter system that responds to the presence of salicylate by upregulating expression of mtrB.
							<B>Fluorescence tests:</B>
							Cultures of JG700 p39k and p41k didn’t grow. We made new cultures with giant swabs of cells from the plates to see if the plates are dead, and subcultured from all the other cultures started yesterday.
							There are also two colonies on our SAL2_mRFP transformation plate, so we made 25mL cultures from these to see if we ligated successfully.
							<B>September 13th</B><B>Conjugating nah into Shewy:</B>
							Caleb started liquid cultures of colonies #3 and #4 from the original transformation plate. Caleb also started a conjugation of JG700 + SAL with colony #3 and JG700 + colony #3. The conjugation was at room temperature on a DAP plate for 16.5 hours.
							<B>Fluorescence:</B>
							Swati miniprepped the SAL2_mRFP cultures, and Claire set up a PCR to confirm whether SAL2_RFP was ligated successfully. Unfortunately it doesn’t look like the ligations worked, as neither band is ~4.5kb, which should be the length for SAL2 with mRFP. We think that the old digestions we are using may not be good, so we will start from scratch next time.
							<B>September 14th</B><B>Conjugating nah into Shewy:</B>
							Caleb checked the liquid cultures from yesterday and noted there was no growth, as expected, due to the nah operon stressing the cells. Caleb streaked new plates from the conjugation - the LB agar plates had kanamycin and chloramphenicol for the JG700 + SAL and colony #3 conjugation and just chloramphenicol for the JG700 and colony #3 conjugation. The plasmid in the SAL strain confers kanamycin resistance and the plasmid in colony #3 with the nah operon confers chloramphenicol resistance. The strain of E. coli in colony #3 was auxotrophic for DAP, so the E. coli couldn't grow on the newly streaked plates.
							<B>From left: Ladder, SAL2_mRFP (1), SAL2_mRFP (2), mut2 pst1, mut2 miniprep:</B><B>Synthetic River Media:</B>
							Cultures of only JG700 and Sal1 containing cells were started today, for another attempt at the growth assay.
							<B>September 15th</B><B>Running Reactors:</B>
							Because maximal current production did not increase, Dylan took down reactors with working electrodes poised at 0.35V with respect the the Ag/AgCl reference electrode in order to free up potentiostat channels so that we can begin characterizing our arsenic reporter strains.
						</DIV></DIV><DIV class="row"><DIV class="nine columns"><H3>September 16th-22nd</H3><B>September 16th</B><B>Conjugating nah into Shewy:</B>
							Caleb checked the streaked conjugation plates - there were some colonies, but they were all on top of the initial streak. He and Tina started liquid cultures of all 15 colonies.
							<B>Synthetic River Media:</B>
							After finalizing the growth assay protocol, another experiment was set up, this time with only two strains (JG700 and Sal1) in triplicate with sodium lactate concentrations (varying from none to full M4 media concentrations) in synthetic Athabasca river water, plus positive LB controls and blanks. The assay was set to run at room temperature for 20 hours, taking data every 5 minutes with 30 seconds of mixing before each read. Blanking was done by group, for each media mixture.
							<B>Fluorescence tests:</B>
							Started liquid cultures of JG700, p25a, and p29a in order to re-conjugate these constitutively produced mRFP plasmids into Shewanella. p25a on a pBBRBB backbone in WM3064 corresponds to p39k in JG700, and p29a corresponds to p41k.
							<B>September 17th</B><B>Running Reactors:</B> After autoclaving a reactor setup and getting peristaltic pumps ready for a new continuous flow experiment, Dylan started a liquid culture of an arsenic reporter strain (JG700 + p14k) to be inoculated into reactors.
							<B>Conjugating nah into Shewy:</B>
							Caleb checked the liquid cultures started yesterday - no growth.
							<B>Synthetic River Media:</B>
							The growth assay was successful, with constant and low OD600 in the blank wells, and a standard growth curve in the LB wells. The data suggests that lowering the sodium lactate from M4 concentrations is impossible. Therefore, a solution containing 5% volume/volume of 40% sodium lactate weight/volume is what is necessary to maintain an OD600 of approximately 0.1 for an extended period of time. However, the supported OD600 may be higher in the final device at this concentration, due to the constant flow of new sodium lactate.
							<B>Fluorescence tests:</B>
							We redigested SAL2 and mRFP with SpeI and PstI-HF to try to clone mRFP into our salicylate reporter, so that we can do fluorescence experiments with the two salicylate parts. Swati also plated WM3064 with JG700 for conjugation of the constitutively produced mRFP.
							<B>September 18th</B><B>Conjugating nah into Shewy:</B>
							Caleb checked the liquid cultures started on the 16th - no growth.
							Fluorescence tests: We ran mRFP on a gel and extracted a 800bp band which was the appropriate link for mRFP. After enzyme purifying the SAL2 digestion and dephosphorylating, Claire set up an overnight ligation of SAL2 and mRFP.
							<B>September 19th</B><B>Conjugating nah into Shewy:</B>
							Caleb checked the liquid cultures started on the 16th - no growth.
							Fluorescence tests: Claire transformed the SAL2_mRFP ligation into WM3064, and set up cultures of p39k and p41k from the conjugation plate.
							<B>September 20th</B><B>Conjugating nah into Shewy:</B>
							Caleb checked the liquid cultures started on the 16th - no growth.
							Fluorescence test: Swati miniprepped cultures of p39k and p41k from the conjugation plates so that we can PCR to confirm if we have successfully conjugated constitutive mRFP into Shewanella.
							<B>September 21st</B><B>Conjugating nah into Shewy:</B>
							Caleb checked the liquid cultures started on the 16th - no growth. He concluded that the conjugation failed. Caleb and Tina electroporated the 3:1 (nah:oriT backbone) ligation of dephosphorylated and digested nah with the oriT backbone from September 3rd into both WM3064 and a DH5α strain. They then streaked 16 colonies from the September 4th transformation onto fresh plates.
							<B>September 22nd</B><B>Conjugating nah into Shewy:</B>
							Caleb and Tina noted the appearance of the re-streaked plates - every single streak had numerous colonies. They started liquid cultures and started an overnight colony PCR of all 16 of them. Included with the colony PCRs was a positive control that we knew would have the same amplicon from the nah operon if the colonies were the result of a successful ligation. The electroporated WM3064 plate grew with numerous isolated colonies; the DH5α transformation failed. We hypothesized the DH5α cells were at fault due to their appearance before electroporation - white precipitate was floating in the cell mixture.
							<B>Fluorescence tests:</B> Using samples from the 20th, Swati did a Phusion PCR to see if we successfully conjugated p39k and p41k into Shewanella. She also miniprepped WM3064 SAL2RFP and did PCR to see if that ligation was successful.
						</DIV></DIV><DIV class="row last-ele"><DIV class="nine columns"><H3>September 23rd-30th</H3><B>September 23rd</B><B>Conjugating nah into Shewy:</B>
							Caleb and Tina ran a gel of the colony PCRs and positive control PCR. Swati also started cultures of our arsenic-sensitive strain, as well as control strains, for a plate testing our arsenic reporters.
							<B>From top-left: ladder, positive control, 8 test colonies. From bottom-left: ladder, 8 test colonies, and Swati's SAL2 RFP:</B><B>From left: ladder, empty, p39k PCR, p41k PCR, empty, SAL2_mRFP PCR:</B><B>September 24th</B><B>Conjugating nah into Shewy:</B>
							Caleb inoculated 30 mL LB+DAP+Cm with the 24 hour liquid culture of of colony N.
							<B>September 25th</B><B>Fluorescence tests:</B> Swati set up a conjugation of SAL2_mRFP into JG700. Also, sequencing came back good so once we have this part in Shewanella we can start testing our salicylate sensor. Jim and Swati also set up a plate to test our arsenic sensing parts at different concentrations of arsenic – they ran a plate with a blank LB column, five control columns (JG700, MR-1, p39k, p40k, and p41k), and three columns of each of our arsenic-sensitive strains. To each row they added a different concentration of arsenic, going from 0uM to 5uM arsenic. The final OD of 100uL in each well was 0.8, and the plate was left overnight in the plate reader.
							<B>Conjugating nah into Shewy:</B>
							Caleb miniprepped the 36 hour 30 mL culture of colony N. The size of the pellet after the first spin step was similar to the size of a ~5mL normal culture, consequently, the yield was only 96.7 ng/uL.
							<B>September 27th</B><B>Running Reactors:</B> For the first time, we added arsenic to our reactors. Specifically, once Dylan saw that the two reactors inoculated with an arsenic reporter strain (JG700+p14k) had been producing steady current for several retention times, he dosed the media vessel to a final concentration of 10 μM sodium arsenite.
							<B>Conjugating nah into Shewy:</B>
							Fortunately, there was enough DNA for Caleb to submit the miniprepped DNA from colony N for Sanger sequencing today. Tina started 3 x (1mL cultures of colony N).
							<B>Fluorescence tests:</B> Claire and Swati set up another control plate with constitutively produced mRFP in both WM3064 and JG700. Since using 100uL starting at and OD of 0.8 seemed to work well for the arsenic plate, we set up the control plate with the same parameters and left it in the plate reader overnight.
							<B>September 28th</B><B>Running Reactors:</B> Since several retention times passed without any current response to 10 μM sodium arsenite, Dylan increased the concentration in the media vessel, hitting our reactors with 100 μM arsenite.
							<B>Fluorescence tests:</B> We noticed that the arsenic data did not seem to have any signal above background, and realized that it may have been because the plate reader wasn’t set to read a clear-bottomed plate. We concluded that, since from the test done on the 27th we could see distinct difference between fluorescence levels of constitutively produced mRFP, our experimental procedure is good and the plate reader was just on the wrong setting. Additionally, since the bioreactors are not getting upregulation of arsenic up to 10uM, we decided next time to test higher concentrations of arsenic (0uM to 500uM)
						</DIV></DIV><DIV class="row last-ele"><DIV class="three columns"><A href="https://2012.igem.org/Team:Cornell/notebook/wetlab/august#pillTab2">&lt;&lt; Previous</A></DIV><DIV class="three columns offset-by-six"><A href="https://2012.igem.org/Team:Cornell/notebook/wetlab/wrap_up" style="display:block;float:right;">Next &gt;&gt;</A></DIV></DIV></LI><LI id="pillTab3Tab"><DIV class="row last-ele" style="margin-top:20px;"><DIV class="three columns"><A href="https://2012.igem.org/Team:Cornell/notebook/wetlab/august#pillTab3">&lt;&lt; Previous</A></DIV><DIV class="three columns offset-by-six"><A href="https://2012.igem.org/Team:Cornell/notebook/wetlab/wrap_up" style="display:block;float:right;">Next &gt;&gt;</A></DIV></DIV><DIV class="row"><DIV class="nine columns"><H3>September 1st-8th</H3>
							Conjugating nah into Shewy: The mysterious failure of past week's ligation lead to a redigestion (in case we run out of DNA) and religation of last week's digestions. Nah to oriT was ligated in ratios of 3:1 and 6:1 - when run on a gel, bands once more failed to appear... But wait! Epiphany! The team realized that we forgot to dephosphorylate the oriT backbone! After that silly mistake, 1:1, 3:1, and 6:1 ratios of nah:oriT were set up in a ligation mixture and electroporated into WM3064. Once again, slow growth and tiny colonies were observed - perhaps due to the strain of putting such a massive operon in a cell. 13 colonies were deemed large enough to try colony PCR, and 3 of them definitely showed bands that corresponded to the length of our construct. We set up liquid cultures for Miniprepping and once again observed slow growth. We tried subculturing - but still, general opaqueness. Miniprepping proceeded anyway, as we are a fairly optimistic bunch (and the cultures were by then over 24 hours old).
							
							Site-directed mutagenesis: After several more unsuccessful attempts, lengthy discussion among teammates, and consultation with several graduate advisors, the team decided to put this subproject on hold for the time being. Swati finished up with one last attempt, then joined Claire for fluorescence testing. That's all, folks!
							<DIV class="panel" style="background:white;margin-top: 20px;"><H6>Daily Details:</H6><P><STRONG>September 1</STRONG></P><P><STRONG>Conjugating nah into Shewy:</STRONG>

									Due to the mysterious failings of our attempts to ligate the nah operon, Tina tried to start fresh by re-digesting the nah operon-containing plasmid and the oriT-containing backbone.
								</P><P><STRONG>Synthetic River Media:</STRONG>

									Mark and Chie created synthetic river media based on information on the mineral content of the Athabasca River, and started overnight cultures of numerous strains to test how sodium lactate supplemented river water would grow the cells.
								</P><P><STRONG>Site-directed mutagenesis:</STRONG> Swati transformed both her mutagenic PCR and her DpnI digest into DH5a after desalting on a membrane. She also ran both of the aforementioned samples on a gel without purifying, in order to check if either the digestion or purification steps were problematic. The gel was yet again blank, suggesting that the PCR itself was the problem.
								</P><P><STRONG>September 2nd</STRONG></P><P><STRONG>Conjugating nah into Shewy:</STRONG>

									Tina tried ligating an older digestion of the nah operon into the oriT backbone using a molar ratio or 3:1 and 6:1 for nah:backbone. She then de-salted and transformed the ligations into conjugation strain WM3064. Tina then ran a gel of the digested nah operon-containing plasmid and the oriT-containing backbone and extracted the nah operon and oriT backbone bands.
								</P><B>From left: Ladder, 5 ug backbone, 5 ug backbone, 5 ug nah operon, 5 ug nah operon, 5 ug nah operon, 5 ug nah operon, empty, empty, empty:</B><P><STRONG>Synthetic River Media:</STRONG>

									Mark set up a 96-well plate to test growth of the engineered strains in sodium lactate at varying concentrations, in synthetic river media.
								</P><P><STRONG>Site-directed mutagenesis:</STRONG>The third transformation attempt yielded no colonies on the plate of DpnI-digested DNA, but three colonies on the PCR only plate, suggesting that these were just excess template. Nonetheless, Swati set up cultures to miniprep the following day. She also realized that she had been using a template that was 10x more concentrated than it was supposed to be. She proceeded to set up a fourth attempt at a mutagenic PCR, this time with the correct template concentration, primers for mutating a different cutsite than she had previously been attempting, and a higher annealing temperature.
								</P><P><STRONG>September 3rd</STRONG></P><P><STRONG>Conjugating nah into Shewy:</STRONG>

									Tina extracted the nah operon digest and the oriT backbone from yesterday's gel. Tina and Dylan had an epiphany - we forgot to dephosphorylate the backbone! No wonder we kept seeing self-ligations! With a renewed spirit, Tina dephosphorylated the digested oriT backbone then started a sixteen hour 16 degrees Celsius ligation of the extracted digests. Tina tried nah to backbone ratios of 1:1, 3:1, and 6:1.
								</P><P><STRONG>Synthetic River Media:</STRONG>

									Mark returned to the lab to discover that the growth assay went wrong. Numerous issues were identified with the original protocol to explain the failed result, and the protocol was changed.
								</P><P><STRONG>Site-directed mutagenesis:</STRONG> Swati miniprepped the three suspicious PCR only colonies from the previous day, and digested with PstI to check for successful mutagenesis. She also did a DpnI digest and transformation of the PCR from the previous day. She ran the new PCR &amp; DpnI digest, along with the PstI-digested minipreps of the three earlier colonies, on a gel, but to no avail. The new things showed nothing (again), and the digested minipreps produced completely incorrect bands.
								</P><P><STRONG>September 4th</STRONG></P><P><STRONG>Conjugating nah into Shewy:</STRONG>

									Tina desalted yesterday's ligations then transformed them into our conjugation strain WM3064. WM3064 is E. coli that is auxotrophic for DAP (diaminopimelic acid epimerase).
								</P><P><STRONG>Site-directed mutagenesis:</STRONG> The latest transformation did not work. Thus concludes this subproject!
								</P><P><STRONG>September 5th</STRONG></P><P><STRONG>Conjugating nah into Shewy:</STRONG>

									Caleb and Tina checked the transformation plates - some colonies were visible on all three ligation ratio pltaes, but the colonies were so tiny we decided to wait another day before trying colony PCRs or starting liquid cultures.
								</P><P><STRONG>September 6th</STRONG></P><P><STRONG>Conjugating nah into Shewy:</STRONG>

									Caleb decided the transformation colonies were large enough to colony PCR and start liquid cultures from. 13 total colony PCRs were performed on colonies from all three ligation mixtures and one positive control was run in parallel to be sure the PCR conditions were correct. Tina ran a gel of the the colony PCRs and the positive control. As can be seen in the gel pictures, colonies 3, 4, and 8 definitely had the nah operon and many other colonies had matching middle bands. These matching middle bands likely indicate some sort of mispriming. Colony 5 was clearly the product of an oriT backbone self-ligation. Caleb and Tina started 15 mL liquid cultures of apparently nah containing colonies 3, 4, and 8 and non-nah-containing colony 5 to Miniprep from tomorrow.
								</P><B>From top left: Ladder, colony 1, colony 2, colony 3, positive control. From bottom left: Ladder, colony 4, colony 5, colony 6, colony 7, positive control:</B><B>From left: Ladder, colony 8, colony 9, colony 10, colony 11, colony 12, colony 13, positive control:</B><P><STRONG>Fluorescence tests:</STRONG> This week started late as Claire was getting over a massive cold. However, with new vim and vigor and her trusty labmate Swati, they set forth to decide the details of how they will run fluorescence tests. To get ready for this, they set up cultures of JG700 and E. coli constitutively producing mRFP under the control of Anderson series promoters.
								</P></DIV></DIV></DIV><DIV class="row"><DIV class="nine columns"><H3>September 9th-15th</H3>
							Conjugating nah into Shewy: Unfortunately, the miniprep yields from last week were all low except one shining colony - which didn't have the nah operon in it. We ran it to test our hypothesis that slow growth of E Coli was due to the added stress on the cell of having such a large operon in it - our data so far matches our hypothesis, as that colony grew faster in culture and the plate and also yielded substantially more DNA after miniprepping.
							
							Site-directed mutagenesis: Though we were unable to confirm the presence of the nah operon with sequencing (as the miniprep failed), we decided to proceed with conjugation into Shewanella just in case the transformation succeeded. We tried to grow one of the colonies with a very strong band from last week's gel in a liquid culture, and once again, observed slow growth. We conjugated into JG700 and JG700 with SAL, a plasmid with our salicylate reporter system. Liquid cultures were made from the conjugation - unfortunately, there was no growth. We streaked new plates with the old transformation in the hopes that something would finally grow!
							<DIV class="panel" style="background:white;margin-top: 20px;"><H6>Daily Details:</H6><B>September 9th</B><B>Conjugating nah into Shewy:</B>
								Caleb checked the miniprep yields today - all but colony 5 had yields below 90 ug/uL. Colony 5 had a yield of over 500 ug/uL. Our hypothesis as to why the E. coli was growing so slowly was that the E. coli was stressed from having to express the many nah operon proteins. Evidence suuporting our hypothesis includes - colony 5 had a much larger colony, grew in liquid much faster, came out as lacking the nah operon, and yielded a lot more DNA from miniprepping than the colonies that supposedly had the nah operon.
								<B>Synthetic River Media:</B>
								Again, the result of the growth assay was failure. This time, it appeared that the only issue was blanking, as the negative controls were reading higher than the inoculated samples. After some discussion with Dr. Archer, a better blanking method was determined, as well as the addition of longer mixing to the protocol.
								<B>September 10th</B><B>Fluorescence tests: </B>
								We dephosphorylated SAL2 digested with SpeI and PstI for one hour, then heat killed at 65degC for 10min. We then ligated overnight with mRFP (also digested with SpeI and PstI) at 16degC in the thermocycler.
								However, JG700 and p41k didn’t grow, so we moved the plates to Riley Robb in the hopes that a more controlled incubator meant for Shewenella will help them thrive.
								<B>September 11th</B><B>Fluorescence tests:</B>
								Swati transformed 2uL of SAL2_mRFP ligation mixture into WM3064. We also made cultures of JG700, WM3064, and p39k, p40k, and p41k in both JG700 and WM3064. We are hoping to run a plate with both strains on it to get an idea of whether we can see mRFP constitutively produced in JG700 (which is naturally red and may have some background signal), and compare that fluorescence to constitutively produced mRFP in WM3064. Once we have this control data we hope to be able to correlate fluorescence in JG700 with promoter strength.
								<B>September 12th</B><B>Conjugating nah into Shewy:</B>
								Caleb and Tina started cultures of colony #3 from original transformation plate and cultures of JG700 + SAL and JG700. JG700 is Shewanella oneidensis with a ΔmtrB genotype. SAL refers to a plasmid with our reporter system that responds to the presence of salicylate by upregulating expression of mtrB.
								<B>Fluorescence tests:</B>
								Cultures of JG700 p39k and p41k didn’t grow. We made new cultures with giant swabs of cells from the plates to see if the plates are dead, and subcultured from all the other cultures started yesterday.
								There are also two colonies on our SAL2_mRFP transformation plate, so we made 25mL cultures from these to see if we ligated successfully.
								<B>September 13th</B><B>Conjugating nah into Shewy:</B>
								Caleb started liquid cultures of colonies #3 and #4 from the original transformation plate. Caleb also started a conjugation of JG700 + SAL with colony #3 and JG700 + colony #3. The conjugation was at room temperature on a DAP plate for 16.5 hours.
								<B>Fluorescence:</B>
								Swati miniprepped the SAL2_mRFP cultures, and Claire set up a PCR to confirm whether SAL2_RFP was ligated successfully. Unfortunately it doesn’t look like the ligations worked, as neither band is ~4.5kb, which should be the length for SAL2 with mRFP. We think that the old digestions we are using may not be good, so we will start from scratch next time.
								<B>September 14th</B><B>Conjugating nah into Shewy:</B>
								Caleb checked the liquid cultures from yesterday and noted there was no growth, as expected, due to the nah operon stressing the cells. Caleb streaked new plates from the conjugation - the LB agar plates had kanamycin and chloramphenicol for the JG700 + SAL and colony #3 conjugation and just chloramphenicol for the JG700 and colony #3 conjugation. The plasmid in the SAL strain confers kanamycin resistance and the plasmid in colony #3 with the nah operon confers chloramphenicol resistance. The strain of E. coli in colony #3 was auxotrophic for DAP, so the E. coli couldn't grow on the newly streaked plates.
								<B>From left: Ladder, SAL2_mRFP (1), SAL2_mRFP (2), mut2 pst1, mut2 miniprep:</B><B>Synthetic River Media:</B>
								Cultures of only JG700 and Sal1 containing cells were started today, for another attempt at the growth assay.
								<B>September 15th</B><B>Running Reactors:</B>
								Because maximal current production did not increase, Dylan took down reactors with working electrodes poised at 0.35V with respect the the Ag/AgCl reference electrode in order to free up potentiostat channels so that we can begin characterizing our arsenic reporter strains.
							</DIV></DIV></DIV><DIV class="row"><DIV class="nine columns"><H3>September 16th-22nd</H3>
							Conjugating nah into Shewy: Some colonies were observed from last week's restreaking! A massive 15 colony liquid culture was set up, but to no avail. After five long days, we were forced to conclude that conjugation failed. Sadface. Good thing we had extra 3:1 nah to oriT ligation sitting around, which we transformed into both WM3064 and DH5a, the latter of which we hoped would grow faster. 16 colonies from the Sep 4th ligation were also restreaked. And lo! Every single restreak had colonies the next day, which warranted a massive 16 colony colony PCR (try saying that five times fast). Interestingly, the WM3064 plates grew, but the DH5a transformation failed, which we concluded was due to a bad electrocompetent cell stock (we noticed weird white precipitate in the cell mixture before).
							<DIV class="panel" style="background:white;margin-top: 20px;"><H6>Daily Details:</H6><B>September 16th</B><B>Conjugating nah into Shewy:</B>
								Caleb checked the streaked conjugation plates - there were some colonies, but they were all on top of the initial streak. He and Tina started liquid cultures of all 15 colonies.
								<B>Synthetic River Media:</B>
								After finalizing the growth assay protocol, another experiment was set up, this time with only two strains (JG700 and Sal1) in triplicate with sodium lactate concentrations (varying from none to full M4 media concentrations) in synthetic Athabasca river water, plus positive LB controls and blanks. The assay was set to run at room temperature for 20 hours, taking data every 5 minutes with 30 seconds of mixing before each read. Blanking was done by group, for each media mixture.
								<B>Fluorescence tests:</B>
								Started liquid cultures of JG700, p25a, and p29a in order to re-conjugate these constitutively produced mRFP plasmids into Shewanella. p25a on a pBBRBB backbone in WM3064 corresponds to p39k in JG700, and p29a corresponds to p41k.
								<B>September 17th</B><B>Running Reactors:</B> After autoclaving a reactor setup and getting peristaltic pumps ready for a new continuous flow experiment, Dylan started a liquid culture of an arsenic reporter strain (JG700 + p14k) to be inoculated into reactors.
								<B>Conjugating nah into Shewy:</B>
								Caleb checked the liquid cultures started yesterday - no growth.
								<B>Synthetic River Media:</B>
								The growth assay was successful, with constant and low OD600 in the blank wells, and a standard growth curve in the LB wells. The data suggests that lowering the sodium lactate from M4 concentrations is impossible. Therefore, a solution containing 5% volume/volume of 40% sodium lactate weight/volume is what is necessary to maintain an OD600 of approximately 0.1 for an extended period of time. However, the supported OD600 may be higher in the final device at this concentration, due to the constant flow of new sodium lactate.
								<B>Fluorescence tests:</B>
								We redigested SAL2 and mRFP with SpeI and PstI-HF to try to clone mRFP into our salicylate reporter, so that we can do fluorescence experiments with the two salicylate parts. Swati also plated WM3064 with JG700 for conjugation of the constitutively produced mRFP.
								<B>September 18th</B><B>Conjugating nah into Shewy:</B>
								Caleb checked the liquid cultures started on the 16th - no growth.
								Fluorescence tests: We ran mRFP on a gel and extracted a 800bp band which was the appropriate link for mRFP. After enzyme purifying the SAL2 digestion and dephosphorylating, Claire set up an overnight ligation of SAL2 and mRFP.
								<B>September 19th</B><B>Conjugating nah into Shewy:</B>
								Caleb checked the liquid cultures started on the 16th - no growth.
								Fluorescence tests: Claire transformed the SAL2_mRFP ligation into WM3064, and set up cultures of p39k and p41k from the conjugation plate.
								<B>September 20th</B><B>Conjugating nah into Shewy:</B>
								Caleb checked the liquid cultures started on the 16th - no growth.
								Fluorescence test: Swati miniprepped cultures of p39k and p41k from the conjugation plates so that we can PCR to confirm if we have successfully conjugated constitutive mRFP into Shewanella.
								<B>September 21st</B><B>Conjugating nah into Shewy:</B>
								Caleb checked the liquid cultures started on the 16th - no growth. He concluded that the conjugation failed. Caleb and Tina electroporated the 3:1 (nah:oriT backbone) ligation of dephosphorylated and digested nah with the oriT backbone from September 3rd into both WM3064 and a DH5α strain. They then streaked 16 colonies from the September 4th transformation onto fresh plates.
								<B>September 22nd</B><B>Conjugating nah into Shewy:</B>
								Caleb and Tina noted the appearance of the re-streaked plates - every single streak had numerous colonies. They started liquid cultures and started an overnight colony PCR of all 16 of them. Included with the colony PCRs was a positive control that we knew would have the same amplicon from the nah operon if the colonies were the result of a successful ligation. The electroporated WM3064 plate grew with numerous isolated colonies; the DH5α transformation failed. We hypothesized the DH5α cells were at fault due to their appearance before electroporation - white precipitate was floating in the cell mixture.
								<B>Fluorescence tests:</B> Using samples from the 20th, Swati did a Phusion PCR to see if we successfully conjugated p39k and p41k into Shewanella. She also miniprepped WM3064 SAL2RFP and did PCR to see if that ligation was successful.
							</DIV></DIV></DIV><DIV class="row last-ele"><DIV class="nine columns"><H3>September 23rd-30th</H3>
							Conjugating nah into Shewy: A gel of last week's colony PCRs hinted that one of the colonies had the nah operon in it! Excitement! Liquid cultures were made, and there was just enough DNA after miniprepping to submit for sequencing. We will collectively hold our breaths and wait for positive results.
							
							This week, we were able to begin characterizing the current response to arsenic of S20 (JG700 + p14k; see strain list). Additionally, we submitted physical DNA for six BioBrick parts to the parts registry.
							<DIV class="panel" style="background:white;margin-top: 20px;"><H6>Daily Details:</H6><B>September 23rd</B><B>Conjugating nah into Shewy:</B>
								Caleb and Tina ran a gel of the colony PCRs and positive control PCR. Swati also started cultures of our arsenic-sensitive strain, as well as control strains, for a plate testing our arsenic reporters.
								<B>From top-left: ladder, positive control, 8 test colonies. From bottom-left: ladder, 8 test colonies, and Swati's SAL2 RFP:</B><B>From left: ladder, empty, p39k PCR, p41k PCR, empty, SAL2_mRFP PCR:</B><B>September 24th</B><B>Conjugating nah into Shewy:</B>
								Caleb inoculated 30 mL LB+DAP+Cm with the 24 hour liquid culture of of colony N.
								<B>September 25th</B><B>Fluorescence tests:</B> Swati set up a conjugation of SAL2_mRFP into JG700. Also, sequencing came back good so once we have this part in Shewanella we can start testing our salicylate sensor. Jim and Swati also set up a plate to test our arsenic sensing parts at different concentrations of arsenic – they ran a plate with a blank LB column, five control columns (JG700, MR-1, p39k, p40k, and p41k), and three columns of each of our arsenic-sensitive strains. To each row they added a different concentration of arsenic, going from 0uM to 5uM arsenic. The final OD of 100uL in each well was 0.8, and the plate was left overnight in the plate reader.
								<B>Conjugating nah into Shewy:</B>
								Caleb miniprepped the 36 hour 30 mL culture of colony N. The size of the pellet after the first spin step was similar to the size of a ~5mL normal culture, consequently, the yield was only 96.7 ng/uL.
								<B>September 27th</B><B>Running Reactors:</B> For the first time, we added arsenic to our reactors. Specifically, once Dylan saw that the two reactors inoculated with an arsenic reporter strain (JG700+p14k) had been producing steady current for several retention times, he dosed the media vessel to a final concentration of 10 μM sodium arsenite.
								<B>Conjugating nah into Shewy:</B>
								Fortunately, there was enough DNA for Caleb to submit the miniprepped DNA from colony N for Sanger sequencing today. Tina started 3 x (1mL cultures of colony N).
								<B>Fluorescence tests:</B> Claire and Swati set up another control plate with constitutively produced mRFP in both WM3064 and JG700. Since using 100uL starting at and OD of 0.8 seemed to work well for the arsenic plate, we set up the control plate with the same parameters and left it in the plate reader overnight.
								<B>September 28th</B><B>Running Reactors:</B> Since several retention times passed without any current response to 10 μM sodium arsenite, Dylan increased the concentration in the media vessel, hitting our reactors with 100 μM arsenite.
								<B>Fluorescence tests:</B> We noticed that the arsenic data did not seem to have any signal above background, and realized that it may have been because the plate reader wasn’t set to read a clear-bottomed plate. We concluded that, since from the test done on the 27th we could see distinct difference between fluorescence levels of constitutively produced mRFP, our experimental procedure is good and the plate reader was just on the wrong setting. Additionally, since the bioreactors are not getting upregulation of arsenic up to 10uM, we decided next time to test higher concentrations of arsenic (0uM to 500uM)
							</DIV></DIV></DIV><DIV class="row last-ele"><DIV class="three columns"><A href="https://2012.igem.org/Team:Cornell/notebook/wetlab/august#pillTab3">&lt;&lt; Previous</A></DIV><DIV class="three columns offset-by-six"><A href="https://2012.igem.org/Team:Cornell/notebook/wetlab/wrap_up" style="display:block;float:right;">Next &gt;&gt;</A></DIV></DIV></LI></UL></DIV><DIV class="printfooter">
Retrieved from &quot;<A href="http://2012.igem.org/Team:Cornell/notebook/wetlab/september">http://2012.igem.org/Team:Cornell/notebook/wetlab/september</A>&quot;</DIV></DIV></DIV><DIV id="footer-box" class="noprint"><DIV id="footer"><UL id="f-list"><LI id="t-recentchanges"><A href="/Special:RecentChanges" title="Recent changes">Recent changes</A></LI><LI id="t-whatlinkshere"><A href="/Special:WhatLinksHere/Team:Cornell/notebook/wetlab/september" title="List of all wiki pages that link here [j]" accesskey="j">What links here</A></LI><LI id="t-recentchangeslinked"><A href="/Special:RecentChangesLinked/Team:Cornell/notebook/wetlab/september" title="Recent changes in pages linked from this page [k]" accesskey="k">Related changes</A></LI><LI id="t-specialpages"><A href="/Special:SpecialPages" title="List of all special pages [q]" accesskey="q">Special pages</A></LI><LI><A href="/Special:Preferences">My preferences</A></LI></UL></DIV><DIV id="footer"><UL id="f-list"><LI id="t-print"><A href="/wiki/index.php?title=Team:Cornell/notebook/wetlab/september&amp;printable=yes" title="Printable version of this page [p]" accesskey="p">Printable version</A></LI><LI id="t-permalink"><A href="/wiki/index.php?title=Team:Cornell/notebook/wetlab/september&amp;oldid=291940" title="Permanent link to this revision of the page">Permanent link</A></LI><LI id="privacy"><A href="/2012.igem.org:Privacy_policy" title="2012.igem.org:Privacy policy">Privacy policy</A></LI><LI id="disclaimer"><A href="/2012.igem.org:General_disclaimer" title="2012.igem.org:General disclaimer">Disclaimers</A></LI></UL></DIV></DIV></DIV></DIV></BODY></HTML>