"
Page
Discussion
View source
History
teams
Log in
Â 
Team:Cornell/project/wetlab/animation
From 2012.igem.org
Home
Project
Overview
Background
Wet Lab
Dry Lab
Human Practices
Gallery
Safety
Notebook
Overview
Wet Lab
Dry Lab
Strain List
Protocols
Team
Overview
Bios
Profile
Sponsors
Outreach
Overview
Research Presentations
Panel Discussions
Public Awareness
2012: SAFE BET
Team Website
2013: Organofoam
2011: Biofactory
2010: OMG OMVs
2009: Cadmium Sensor
Wet Lab
Overview
Chassis
DNA Assembly
Arsenic Reporter
Naphthalene Reporter
Testing & Results
BioBricks
Transcriptional Characterization
Current Response
MtrB Protein Expression
nah Operon Expression
Artificial River Media
Future Work
Animation
Animation
Protein Structures
When making a 3D animation, it's important to define several features early on in the process to have an overview of where the animation is going and the style in which its actors will be represented. In this case, we wanted to depict the Mtr pathway in Shewanella oneidensis, so one of the first things I decided was to have accurate protein models.
RCSB PDB (Protein Data Bank) is an amazing resource that has virtually every published protein structure. Unfortunately, there is only one Mtr protein whose structure has been published, MtrF, whose role in the Mtr pathway is so peripheral that I chose not to represent it in the animation. This left me with two options: (1) find proteins that should be related, and use their structures, or (2) generate predicted structures. In the spirit of accuracy, I chose to generate predicted structures.
Using the gene sequences from NCBI , identified the open reading frame and translated the proteins with ApE . I used two online tools for structure prediction: PSIPRED Protein Structure Prediction Server , which uses two neural networks to detect secondary structure by amino acid motifs, and SWISS-MODEL , which has a tool that uses sequence alignments to find proteins with published structures that have regions likely to be homologous, then predicts structure based on the homologous region.
Unfortunately, a major shortcoming of SWISS-MODEL is that it only predicts structure in the homologous region; any amino acids outside of that have to be added back and folded by other means. For this purpose, I used PyMOL to add back the missing residues, and the standalone version of Fold.It to fold them. I used the secondary structure predictions from PSIPRED for these residues. For CymA, MtrA, and MtrC, this involved folding up to 50 residues at each end, which was simple, but for MtrB, believed to be a beta barrel protein, SWISS-MODEL only produced about half of the barrel, so I spent nearly a week manually folding over 300 residues into the shape of the missing beta sheets.
Stylistic Decisions
Once I had the structures, the rest of the animation had to be determined. The second very important stylistic decision was to use a style of animation that emulates scanning electron microscope images. While realistic lighting and shaders can look pretty awesome, they really don't convey information particularly well. If you've ever played a game where outlines or highlighting are used in some way to indicate an item or character of interest, it's very clear how much a simple step like that can enhance the accessibility of visual information. Initially I considered a "toon" style, which meets these requirements very directly. However, after trying several different shaders, I found that SEM images actually have a similar effect, as the brightness of a surface is dependent on the angle relative to the camera; on surfaces that curve away from the camera, this causes edges to show up much brighter.
The third stylistic decision lay in how to represent election transfer. How does one really show events at the quantum level? They don't really have any set visual representation. The simplest way would be to show a small particle passing between electron carriers, but that would be pretty boring and difficult to see. Instead, I decided to represent electron transfer as lightning: it's bright, it's flashy, and it's dramatic. Autodesk Maya also provides a very easy-to-use dynamic lightning tool, facilitating this decision greatly.
The final stylistic decision lay in how to show the differences between our system when induced and not induced. I opted for a side-by-side splitscreen in the interest of keeping the animation time-efficient, and to allow direct comparison of the two as they're occurring, rather than sequentially, which relies on the viewer remembering the events of the previous sequence. Furthermore, because most elements remain the same, showing them sequentially would be repetitive.
Composition
Our team has experience in both Luxology Modo and Autodesk Maya. In my experience, Modo tends to make things look nicer with default settings, and has some really great real-time preview rendering options, but Maya has many tools built in for more advanced effects, and the mMaya toolkit provides a lot of features for importing pdb files (the standard for protein structures) and adjusting various parameters for modeling their backbone and surface.
As for developing the actual animation, the first step was to make a timeline. We are using the animation to go along with the presentation of the Mtr pathway, so it was important to have timings that allow for easy real-time narration. I went over the presentation with Dylan Webster , recording timings for each of the steps, and converted those to frames.
After the timeline was set, I built models for the cell, the electrode, and imported models for intermediate electron carriers from PyMOL. Then, I made curves on which to animate the various elements, and cameras to follow each one. The scene camera moves between these cameras, switching off at their common points. Finally, I added polishing touches, such as dynamically randomized offsets (giving the appearance of brownian motion), rippling of the cell membranes, addition of bump maps, and and instancing of lipid models along the membrane surface.
I hope you enjoy the result.
~ Rafael Lizarralde
The Result
Retrieved from " http://2012.igem.org/Team:Cornell/project/wetlab/animation "
Recent changes
What links here
Related changes
Special pages
My preferences
Printable version
Permanent link
Privacy policy
Disclaimers
