<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_UNIK_Copenhagen_Biosyntetic_Make_Up skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><DIV id="top_title"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:UNIK Copenhagen/Biosyntetic Make Up</SPAN></H1></DIV><DIV id="HQ_page"><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><DIV class="menu_wrapper"><DIV class="logo"><H1 style="font-family: aspace bold demo" style="font-weight: bold" style="font-size:30px"> COSMOCROPS</H1></DIV><DIV class="collapsable_menu_control"> MENU </DIV><UL id="accordion" class="accordion"><LI class="menu_item"><A href="https://2016.igem.org/Team:UNIK_Copenhagen">HOME </A></LI><LI class="menu_item">PROJECT
			</LI><LI><A href="https://2016.igem.org/Team:UNIK_Copenhagen/Description"> Overview of the Project </A></LI><LI><A href="https://2016.igem.org/Team:UNIK_Copenhagen/ConceptProof"> Proof of concept </A></LI><LI><A href="https://2016.igem.org/Team:UNIK_Copenhagen/Attributions"> Attributions</A></LI></UL><LI class="menu_item">TEAM 
			</LI><LI><A href="https://2016.igem.org/Team:UNIK_Copenhagen/Team"> People Involved </A></LI><LI class="menu_item">GREEN LAB
			</LI><LI><A href="https://2016.igem.org/Team:UNIK_Copenhagen/Green"> Green Lab Overview </A></LI><LI><A href="https://2016.igem.org/Team:UNIK_Copenhagen/Design"> Co-culture setup </A></LI><LI><A href="https://2016.igem.org/Team:UNIK_Copenhagen/Biosyntetic_Make_Up"> Biosynthetic make-up </A></LI><LI><A href="https://2016.igem.org/Team:UNIK_Copenhagen/Results"> Experiments and Results </A></LI><LI><A href="https://2016.igem.org/Team:UNIK_Copenhagen/Experiments"> Protocols </A></LI><LI class="menu_item">RED LAB
			</LI><LI><A href="https://2016.igem.org/Team:UNIK_Copenhagen/Red">Red Lab Overview </A></LI><LI><A href="https://2016.igem.org/Team:UNIK_Copenhagen/Experimental_Concept"> Experimental Concept </A></LI><LI><A href="https://2016.igem.org/Team:UNIK_Copenhagen/Scientific_Challenges"> Scientific Challenges </A></LI><LI><A href="https://2016.igem.org/Team:UNIK_Copenhagen/Achievements"> Achievements </A></LI><LI><A href="https://2016.igem.org/Team:UNIK_Copenhagen/Results"> Experiments and Results </A></LI><LI><A href="https://2016.igem.org/Team:UNIK_Copenhagen/Future_Perspective">  Future Perspective </A></LI><LI class="menu_item">OUTREACH
			</LI><LI><A href="https://2016.igem.org/Team:UNIK_Copenhagen/Human_Practices"> Human Practices </A></LI><LI><A href="https://2016.igem.org/Team:UNIK_Copenhagen/Collaborations"> Collaborations </A></LI><LI><A href="https://2016.igem.org/Team:UNIK_Copenhagen/Publications"> Publications </A></LI><LI><A href="https://2016.igem.org/Team:UNIK_Copenhagen/Calendar"> Calendar </A></LI><LI class="menu_item"><A href="https://2016.igem.org/Team:UNIK_Copenhagen/Funding">FUNDING </A></LI><LI class="menu_item"><A href="https://2016.igem.org/Team:UNIK_Copenhagen/Contact">CONTACT </A></LI></DIV><DIV class="content_wrapper"><H1 style="font-family: aspace bold demo" style="font-weight: bold" style="font-size:30px"> Biosynthetic Make-up </H1><H2 style="font-family: aspace bold demo" style="font-weight: bold" style="font-size:30px"> Introduction </H2><P><I>Bacillus subtilis</I> is a rod-shaped gram-positive bacterium that lives on soil and in our gastrointestinal tract. As already mentioned in the <A href="https://2016.igem.org/Team:UNIK_Copenhagen/Design">co-culture setup</A>, <I>B. subtilis</I>  is a part of our co-culture system that is set to be our producing organism of organic compounds. This could be the production of biological compounds such as vitamins, antibiotics or detergents. As a proof of concept, we decided to produce biodegradable plastic called Poly(Lactic Acid-co-3-HydroxyButyrate [P(LA-co-3HB)] that can be used as a raw material for 3D printing. This section would highlight the synthetic make-up and genetic engineering aspects of our project in <I>B. subtilis</I>. 
</P><H2 style="font-family: aspace bold demo" style="font-weight: bold" style="font-size:30px"> Why Bioplastic in Space?</H2><P>
conventional plastic is one of the world’s most versatile material that is used to make tools, plastic bags, bottles and lab equipments. On planet Earth, conventional plastic is known to be one of the factors promoting environmental pollution because it is non-degradable and can survive for several centuries. When colonizing Mars, it would be wise to choose a more sustainable and environment friendly lifestyle and decisions. In our project, we have chosen Poly(Lactic acid-co-3-HydroxyButyrate [P(LA-co-3HB)] production using approaches within synthetic biology. 
We have chosen P(LA-co-3HB) because,<OL><LI> It is a <B>biodegradable</B>, meaning that it is degraded by microorganisms and decompose to water, carbon dioxide and a soil based black material (Drumright RE <I>et. al.</I>,2000). </LI><LI> It has a <B>lifespan</B> of 6 months to 2 years on Earth, but it might be a bit longer in space due to less availability of microorganisms </LI><LI> It is a <B>renewable resource</B> which means that the bioplastic can be reused for another purpose and overcome plastic waste problem on Mars. </LI><LI> It is <B>thermoplastic</B> which means that it can can be warmed up and reused. </LI><LI> It can form <B>filamentous structure</B> and thereby can be used as a raw material for 3D printing. </LI><LI> P(LA-co-3HB) has similar properties as ABS which is used in making LEGO bricks (Grieser F, 2016) </LI><LI> has shown to have <B>low toxicity</B> to humans </LI></OL></P><P>
In regards to these properties, the P(LA-co-3HB) production using genetic engineered organisms that are easy to handle mass produce, lightweight and easy to mass produce is suitable for space flights and future martian colonizations. This concept can not only be used on Mars, but also on planet Earth. 
</P><P>
The biosynthesis of both pure homopolymer Poly Lactic &gt;cid (PLA) and heteropolymer P(LA-co-3HB) in <I>Escherichia coli</I> has previously been researched and optimized by Yamada <I>et. al.</I> (2010) and a similar work is demonstrated by iGEM team, <A href="https://2013.igem.org/Team:Yale/Project_Overview">Yale iGEM 2013</A>, who were inspired by the work done by Jung and coworkers (2009).
 
In Jung <I>et. al.</I> (2009), they use a synthetic two-step bioproduction of PLA and P(LA-co-3HB). In their first step, they introduce a synthetic pathway by incorporating four genes to synthesize P(LA-co-3HB). In the next step, they manage to synthesize pure PLA along with P(LA-co-3HB) with high fraction of lactic acid by enhancing production of lactic acid. The production of P(LA-co-3HB) is not been demonstrated before in <I>B. subtilis</I> and that is why our first goal is to make it synthesize P(LA-co-3HB) and thereby optimize it to produce pure PLA in the future, as it is done by the iGEM team from <A href="https://2016.igem.org/Team:Evry">Evry 2016</A>.
</P><H2 style="font-family: aspace bold demo" style="font-weight: bold" style="font-size:30px">The biochemical properties and one-step biosynthesis of P(LA-co-3HB)</H2><P>
P(LA-co-3HB) is an organic compound that is produced by hetero polymerization of lactyl-CoA and 3-hydroxybutyryl in bacterial cells. The fraction of lactic acid and 3-hydroxybutyryl identified in P(LA-co-3HB) varies and has shown to be dependent on amount of lactate and acetyl-CoA in the cell. The fraction of lactic acid in P(LA-co-3HB) influences the melting temperature of P(LA-co-3HB) (Yamada, <I>et. al.</I>, 2010). The melting temperature varies between 100-176 degrees on Earth. In the cells, P(LA-co-3HB) accumulates in a form of granular structures in the <I>E. coli</I> (Fig. 1A) where we expect somewhat similar when synthesized in <I>B. subtilis</I>.


When cells fermentate, they naturally produce lactate which is one of the substrate along with acetyl-CoA for P(LA-co-3HB) synthesis. Both of these derives from pyruvate as shown in Figure 1B. This step is naturally present in <I>B. subtilis</I> connected to sucrose catabolic pathway. As an inspiration from the work done in Yamada and coworkers (2010) and by iGEM team, Yale iGEM 2013 in <I>E. coli</I>, we introduce four heterologous genes for the synthesis of P(LA-co-3HB).

The four heterologous genes that are integrated are
<OL><LI> Propionate-CoA transferase (PCT): converts lactate into lactyl-CoA - <A href="http://parts.igem.org/Part:BBa_K1975001">BBa_K1975001</A></LI><LI>β-ketothiolase (PhaA): converts Acetyl-CoA to acetoacetyl-CoA - <A href="http://parts.igem.org/Part:BBa_K1975002&quot;">BBa_K1975002</A></LI><LI> NADPH-dependent acetyl-CoA reductase (PHaB): converts Acetoacetyl-CoA til (R)-3-hydroxybutyryl-CoA - <A href="http://parts.igem.org/Part:BBa_K1975003">BBa_K1975003</A></LI><LI> Lactate polymerizing enzyme (LPE): is a lactate polymerizing enzyme and is a mutated version of polyhydroxyalkanoate (PHA) synthase from <I>pseudomonas</I> species - <A href="http://parts.igem.org/Part:BBa_K1975000">BBa_K1975000</A></LI></OL></P><DIV class="center_content"><P><I>Figure 1:Figure 1: (B) Illustrates the biosynthetic pathway for P(LA-co-3HB) production B. subtilis. B. subtilis naturally produces lactate and Acetyl-CoA when fermentation of sucrose occurs. These two intermediates are used as substrates by the heterologous enzymes introduced to B. subtilis. (A) shows P(LA-co-3HB) accumulation in E. coli JLX10 strain forming granules like structure. Modified: Jung, YK. et.al (2009).</I></P></DIV><H2 style="font-family: aspace bold demo" style="font-weight: bold" style="font-size:30px">How we designed our gene constructs and assemble them in one vector</H2><P>
The gene constructs were assembled in pHT254 shuttle vector using Gibson Assembly. pHT254 shuttle vector harbours the origin of replication for <I>E. coli</I> and ampicillin as selection marker in <I>E. coli</I>, while chloramphenicol is used as selection marker in <I>B. subtilis</I>. pHT254 harbours an IPTG inducible promoter <I>Pgrac100</I> which is used.
</P><DIV class="center_content"><P><I>Figure 2:. Illustrates the cloning strategy of B. subtilis. (A) Schematic overview of the planned cloning. (B) Schematic overview of the Gibson Assembly. The colour region correspond to the homologous sites that were introduced with PCR on each gene.
</I></P></DIV>
PhaA and PhaB were ordered with fluorescence tags due to their small size, while PCT and LPE genes were bought separately along with Green Fluorescent Protein (GFP) and Yellow Fluorescent Protein (YFP) genes. PCT and YFP was then fused together in the laboratory to obtain the PCT-YFP construct. The same was tried for LPE and GFP, we were unfortunately unable in obtaining the LPE-GFP construct. In addition, the distinct restriction sites between and homologous sites were introduced by PCR between the genes and the fluorescence tag to remove the tag in the future if it is needed.
<TABLE><TBODY><TR><TD><P style="font-size:10px"><B>Validation</B></P></TD><TD><P style="font-size:10px"><B>Results</B></P></TD><TD><PSTYLE><B>Link</B></PSTYLE></TD></TR><TR><TD><P> 1) Is it possible to fuse LPE and GFP?</P></TD><TD><P>Negative</P></TD><TD><P><A href="https://2016.igem.org/Team:UNIK_Copenhagen/Results#experiment11">Go to experiment</A></P></TD></TR><TR><TD><P> 3) Is it possible to fuse PCT and YFP?</P></TD><TD><P>Positive</P></TD><TD><P><A href="https://2016.igem.org/Team:UNIK_Copenhagen/Results#experiment12">Go to experiment</A></P></TD></TR></TBODY></TABLE><P>
Instead, the gel-purified LPE and GFP which already contained the homologous sites was used as individual constructs along with gel-purified PCT-YFP, PhaA-CFP, PhaB-RFP-Term and the cut pHT254 vector from restriction digestion mix for the Gibson assembly as shown in Figure 2B. <I>E. coli</I> was transformed and the formed and the positive colonies with the vector harbouring all five constructs using colony-PCR.
</P><P style="font-size:10px"><B>Validation</B></P><P style="font-size:10px"><B>Results</B></P><PSTYLE><B>Link</B><P> 3) Was the pHT254 vector linearized?</P><P>Positive</P><P><A href="https://2016.igem.org/Team:UNIK_Copenhagen/Results#experiment0">Go to experiment</A></P><P> 4) Did Gibson Assembly work?</P><P>Negative</P><P><A href="https://2016.igem.org/Team:UNIK_Copenhagen/Results#experiment14">Go to experiment</A></P><P>
The Gibson Assembly was repeated three times, but unfortunately, the formed colonies did not show a positive result containing all the constructs. We realized that the Gibson mix we used in the first two transformations was expired but colony PCR the new Gibson mix showed the same result as previously. We hypothesized that the homologous sites were large and should be shortened using new primers or the amount/concentrations of the used constructs in the mixture could have been adjusted. 6 constructs including the plasmid is the limit for Gibson reaction and maybe did not go well. 
    

The next cloning approach was the conventional cloning using restriction digestion. New primers were designed for PhaA-CFP and PhaB-RFP-Term to introduce restriction sites. First, we tried to clone PhaA-CFP into the vector and transform <I>E. coli</I>. The acquired MiniPrep was then digested and a ligation was run with PhaB-RFP-Term. </P><P style="font-size:10px"><B>Validation</B></P><P style="font-size:10px"><B>Results</B></P><PSTYLE><B>Link</B><P> 5) Was PhaA-CFP successfully cloned in pHT254?</P><P>Positive</P><P><A href="https://2016.igem.org/Team:UNIK_Copenhagen/Results#experiment15">Go to experiment</A></P><P> 6) Was PhaB-RFP-Term successfully cloned into pHT254 already containing  PhaA-CFP?</P><P>Negative</P><P><A href="https://2016.igem.org/Team:UNIK_Copenhagen/Results#experiment16">Go to experiment</A></P><H2 style="font-family: aspace bold demo" style="font-weight: bold" style="font-size:30px">References</H2><P>
Drumright RE, Gruber PR, Henton DE. Polylactic Acid Technology. Advanced Materials. 2000; 12(32); 1481-1486.

Jung YK, Kim TY, Park SJ, L SY. Metabolic Engineering of Escherichia coli for the Production of Polylactic Acid and Its Copolymers.

Yamada M, Matsumoto K., Shimizu K., Uramoto S., Nakai T., Shozui F., Taguchi S. Adjustable Mutations in Lactate (LA)-Polymerizing Enzyme for the Microbial Production of LA-Based Polyesters with Tailor-Made Monomer Composition. Biomolecules.2010;11(3):815-819.</P></PSTYLE></PSTYLE></DIV></DIV></DIV></DIV></DIV></DIV></BODY></HTML>