<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_UNIK_Copenhagen_Results skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><DIV id="top_title"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:UNIK Copenhagen/Results</SPAN></H1></DIV><DIV id="HQ_page"><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><DIV class="menu_wrapper"><DIV class="logo"><H1 style="font-family: aspace bold demo" style="font-weight: bold" style="font-size:30px"> COSMOCROPS</H1></DIV><DIV class="collapsable_menu_control"> MENU </DIV><UL id="accordion" class="accordion"><LI class="menu_item"><A href="https://2016.igem.org/Team:UNIK_Copenhagen">HOME </A></LI><LI class="menu_item">PROJECT
			</LI><LI><A href="https://2016.igem.org/Team:UNIK_Copenhagen/Description"> Overview of the Project </A></LI><LI><A href="https://2016.igem.org/Team:UNIK_Copenhagen/ConceptProof"> Proof of concept </A></LI><LI><A href="https://2016.igem.org/Team:UNIK_Copenhagen/Attributions"> Attributions</A></LI></UL><LI class="menu_item">TEAM 
			</LI><LI><A href="https://2016.igem.org/Team:UNIK_Copenhagen/Team"> People Involved </A></LI><LI class="menu_item">GREEN LAB
			</LI><LI><A href="https://2016.igem.org/Team:UNIK_Copenhagen/Green"> Green Lab Overview </A></LI><LI><A href="https://2016.igem.org/Team:UNIK_Copenhagen/Design"> Co-culture setup </A></LI><LI><A href="https://2016.igem.org/Team:UNIK_Copenhagen/Biosyntetic_Make_Up"> Biosynthetic make-up </A></LI><LI><A href="https://2016.igem.org/Team:UNIK_Copenhagen/Results"> Experiments and Results </A></LI><LI><A href="https://2016.igem.org/Team:UNIK_Copenhagen/Experiments"> Protocols </A></LI><LI class="menu_item">RED LAB
			</LI><LI><A href="https://2016.igem.org/Team:UNIK_Copenhagen/Red">Red Lab Overview </A></LI><LI><A href="https://2016.igem.org/Team:UNIK_Copenhagen/Experimental_Concept"> Experimental Concept </A></LI><LI><A href="https://2016.igem.org/Team:UNIK_Copenhagen/Scientific_Challenges"> Scientific Challenges </A></LI><LI><A href="https://2016.igem.org/Team:UNIK_Copenhagen/Achievements"> Achievements </A></LI><LI><A href="https://2016.igem.org/Team:UNIK_Copenhagen/Results"> Experiments and Results </A></LI><LI><A href="https://2016.igem.org/Team:UNIK_Copenhagen/Future_Perspective">  Future Perspective </A></LI><LI class="menu_item">OUTREACH
			</LI><LI><A href="https://2016.igem.org/Team:UNIK_Copenhagen/Human_Practices"> Human Practices </A></LI><LI><A href="https://2016.igem.org/Team:UNIK_Copenhagen/Collaborations"> Collaborations </A></LI><LI><A href="https://2016.igem.org/Team:UNIK_Copenhagen/Publications"> Publications </A></LI><LI><A href="https://2016.igem.org/Team:UNIK_Copenhagen/Calendar"> Calendar </A></LI><LI class="menu_item"><A href="https://2016.igem.org/Team:UNIK_Copenhagen/Funding">FUNDING </A></LI><LI class="menu_item"><A href="https://2016.igem.org/Team:UNIK_Copenhagen/Contact">CONTACT </A></LI></DIV><DIV class="content_wrapper"><H1 style="font-family: aspace bold demo" style="font-weight: bold" style="font-size:30px">Experiments and Results</H1><H2 style="font-family: aspace bold demo" style="font-weight: bold" style="font-size:30px">Achievements</H2><OL><LI>Validation of a co-culture system using a re-purposed dialysis-bag from Spectrum Pore Labs. </LI><LI>Design and usage of a medium suitable for co-culturing <I>Synechococcus elongatus</I> and <I>Bacillus subtilis</I> together. </LI><LI>Characterization of cscB permease BioBrick <A href="http://parts.igem.org/Part:BBa_K656011">Bba_K656011</A>, a sucrose/proton symporter, in <I>S. elongatus</I> PCC7942. </LI><LI>Design of a BioBrick package for poly-lactic-co-3-hydroxybutyrate acid (PLA-co-3HB) production in wild type <I>B. subtilis 168</I>. </LI><LI>Design and validation of a β-lactamse, ampE. The BioBrick was experimentally validated in <I>Escherichia coli</I>. </LI><LI>Design and validation of the DNA helicase repA in <I>B. subtilis</I> 168.</LI><LI>Testing <I>S. elongatus</I> and <I>B. subtilis</I> at simulated extraterrestrial environments. </LI><P>Additionally, we planned to experimentally validate PLA-production by gas chromatography mass spectrometry. However, due to issues with assembling the construct with Gibson assembly, this was not achieved.</P></OL><P> </P><H2 style="font-family: aspace bold demo" style="font-weight: bold" style="font-size:30px">Growth Experiments - Validating our co-culture design and medium</H2><DIV class="center-content"><P><B> ---------------- </B></P></DIV><P><B>Question:</B> Was it possible to create an isolated compartment by using a dialysis membrane?</P><P><B>Experiments:</B>30 cm of dialysis tubing was filled with 1 mg/ml Dextran Blue solution dissolved in miliQ-water. The tubing was closed using clips from spectrum labs. The filled dialysis tube was carefully placed in a 1000 ml flask with sufficient miliQ-water covering the tube. If the tubing was leaky, Dextran Blue would be visible in the surrounding water.</P><P><B>Result:</B> Even after five days of incubation on a shaker at room temperature, no dextran blue was observed in the surrounding water (Fig. 1).</P><DIV class="center-content"><P><B> ---------------- </B></P></DIV><P><B>Question:</B> Was it possible to keep a subculture growing inside the isolated compartment without contaminating the surroundings? </P><P><B>Experiment:</B> Using Lysogeny-Broth (LB) medium with 10μg/ml chloramphenicol, Mock-transformed <I>B. subtilis</I> were inoculated into a dialysis tube filled with LB medium. The dialysis bag was closed using clips from Spectrum Labs, and carefully placed inside a 1000mL growth flask and incubated at 28°C overnight at 100rpm. 

</P><P><B>Result:</B> The first versions of our protocol for sterilization and inoculation of our dialysis bag caused wide-spread cross contamination between compartments (Fig. 2A). This also raised the question whether the contamination came from the act of inoculating or from other outside sources, which was investigated by omitting innoculation with and without antibiotics (Fig. 2B). This showed that contamination of the outside only happened in the innoculated sample, meaning that the contamination came from the act of inoculation.

It was therefore hypothesized that by cleaning the top of the dialysis bag with 70% ethanol after inoculation, it would be sterile. By using this improved protocol it was possible to isolate growth of Mock-transformed <I>B. subtilis</I> to the inner compartment (Fig. 3). This is seen by a turbid medium inside the dialysis after incubation.
</P><DIV class="center-content"><P><B> ---------------- </B></P></DIV><P><B>Question:</B> Can <I>S. elongatus</I> grow in BG-11 medium? </P><P><B>Experiment:</B><I>S. elongatus</I> was inoculated into BG-11 medium a 250mL growth flask. The culture was incubated for 4 days at 30°C in a light chamber at 250rpm. Optical density (OD) at 730nm was measured 5 times during this period. </P><P><B>Result:</B> Growth was relatively slow, but followed an almost linear trend (Fig. 4). This strongly indicates that <I>S. elongatus</I> can grow in BG-11 medium, which was expected.

</P><DIV class="center-content"><P><B> ---------------- </B></P></DIV><P><B>Question:</B> Can <I>B.subtilis</I> grow in BG-11 medium with and without sucrose supplement? </P><P><B>Experiment:</B> Unsupplemented, 10mM and 30mM sucrose supplemented BG-11 medium were set up in autoclaved 250mL baffled flasks. Wild-type <I>B.subtilis</I> 168 was inoculated from plates into a falcon tube containing some medium, which was then poured into the baffled flasks. No antibiotics were used.  The flasks were incubated at 28°C at 100rpm for 64 hours. Growth was observed by OD 600nm twice a day. </P><P><B>Result:</B> Poor growth was observed across all groups which implicates that a co-culture on BG-11 alone would ultimately fail (Fig. 5). This suggests that another medium composition is needed - maybe even a combination of different media. 

</P><DIV class="center-content"><P><B> ---------------- </B></P></DIV><P><B>Question:</B> Can <I>B.subtilis</I> grow in ATCC, CSE and C minimal media with and without 30 mM sucrose?  </P><P><B>Experiment:</B> Two experiments were made. First, <I>B. subtilis</I> was inoculated in 100mL ATCC medium that was supplemented with 30mM sucrose in a 250mL baffled flask. The cells were grown for a week at 28°C at 100rpm and OD at 600nm were measured twice a day, in the morning and in the evening. Also both C minimal medium and CSE minimal medium were tested, with and without supplemented 30mM sucrose under the same conditions. Second time, the experiment for ATCC had the same set-up. However, the cells were now grown for 22 days, and the medium was changed every fourth day. OD at 600nm was measured every second day. Contamination testing was carried out by plating out 250μL of the cell culture out on a LB plate without any antibiotics. The plate were dried and stored in a 30°C incubator overnight, and the plates were investigated.
 </P><P><B>Result:</B> To find the right minimal medium for <I>B. subtilis</I> three different minimal medium was tested. These are C minimal medium, CSE minimal medium and ATCC medium. After inoculation, the OD was measured to obtain the growth of the bacterium in the different media. CSE minimal medium supplemented with 30mM has the best growth of <I>B. subtilis</I> over a time range of 117 hours (Fig. 6). Already in the early time period this medium showed superior growth compared to the other two media tested. However, when we checked for contamination we discovered that the CSE minimal medium had been contaminated by another organism that was not <I>B. subtilis</I> (data not shown), meaning that the data for CSE minimal medium could not be trusted. This left C minimal medium and ATCC medium. 

</P><P>C minimal medium showed a constant level of <I>B. subtilis</I> growth until hour 117th. However, at this point, the medium began to be the same yellow colour as for CSE minimal medium. This lead to the ATCC medium that did not show a very high growth, but nevertheless, the cells actually grew in the medium. This was later confirmed to be <I>B. subtilis</I> by plating them out on LB plates. However, now a new problem arose - how could we get the <I>B. subtilis</I> to grow into a higher density which is needed to get bioproduction of a high-value compound such as P(LA-co-3HB). We hypothesized that by changing the medium once in awhile we would refresh the nutritions and <I>B. subtilis</I> will grow more. This was done in the second experiment we carried out with ATCC medium. The medium was changed every fourth day, and OD was measured every second day. This shows that the growth of <I>B. subtilis</I> indeed can be raised by changing the medium (Fig. 7). This is something we need to consider in the final design of the co-culture. How can the nutritions be refreshed? One way is to use different kind of waste products from astronauts or Mars settlers and add those to the medium instead of changing the medium.</P><DIV class="center-content"><P><B> ---------------- </B></P></DIV><P><B>Question:</B> Can <I>B. subtilis</I> grow under osmotic stress?</P><P><B>Experiment:</B><I>B. subtilis</I> was grown at different concentrations of supplemented NaCl (0 mM, 50 mM, 100mM, 150 mM, 200 mM and 500 mM) in LB medium. The samples were incubated at 28 degrees at 100rpm for 17 hours. Growth was measured with OD at 600nm over a time period.
</P><P><B>Result:</B> As stated by both Ducat <I>et. al.</I> and <A href="https://2011.igem.org/Team:Brown-Stanford/PowerCell/NutrientSecretion">Stanford-Brown 2011</A> the cscB transporter does not work without an osmotic stress to accumulate sucrose in the cytoplasm of the cyanobacterium. We therefore need to test the ability of <I>B. subtilis</I> to grow in high osmotic stress.
<I>B. subtilis</I> was inoculated in  LB medium supplemented with different concentrations of NaCl. The growth curve of <I>B. subtilis</I> was obtained by measuring OD at 600nm. The growth curve shows that <I>B. subtilis</I> is not affected by the salt concentrations at all (Fig. 8). Up to  200mM NaCl there is no difference between the high salt samples and the reference curve (without salt, blue diamond on Fig. 8). Only one sample showed severe growth impairment, which was the sample that was grown in 500mM NaCl (Organge curve on Fig. 8). These data indicates that <I>B. subtilis</I> would not have any problems in growing in the salt stress of our final co-culture setup. 
</P><DIV class="center-content"><P><B> ---------------- </B></P></DIV><P><B>Question:</B> Can <I>S. elongatus</I> grow under osmotic stress?</P><P><B>Experiment:</B><I>S. elongatus</I> was inoculated into BG-11 medium with supplemented NaCl (0mM, 150mM, 300mM). The cells were incubated in a light chamber at 30 degrees at 250rpm for 4 days. OD at 730nm was measured twice a day during the time period. 

</P><P><B>Result:</B> Inspired by the work of Ducat and coworkers, we use the cscB sucrose permease, which secretes sucrose out of the cell under osmotic pressure. The osmotic pressure is <A href="https://2016.igem.org/Team:UNIK_Copenhagen/Design">required</A> to secrete the sucrose but it might affect the growth of the <I>S. elongatus</I>. We therefore tested its ability to grow in different salt concentrations, and the growth was obtained by measuring OD at 730nm (Fig. 9).
As expected, <I>S. elongatus</I> grow best when no salt is present in its medium. This sample does not look like it has any trouble by growing, just as it is expected. However, when we then add salt to the medium <I>S. elongatus</I> will begin to be stressed. This can be seen by both the colour of the samples (Fig. 10) and the growth curves (Fig. 9). Surprisingly, the data for the <I>S. elongatus</I> grown in 300mM salt shows better growth than the sample with 150mM salt - at least up to hour 68 where after it begins to fall drastically (Green curve in Fig. 9). 

</P><P>This suggest that in long term <I>S. elongatus</I> would not be able to grow in 300mM NaCl without showing severe growth rates. <I>S. elongatus</I> grown in 150mM NaCl shows decreased growth, but the growth keeps increasing meaning that the cells survives but are under a lot of stress that makes them to grow slower than usual (Fig. 9 and 10)</P><DIV class="center-content"><P><B> ---------------- </B></P></DIV><P><B>Question:</B> Can <I>B. subtilis</I> grow in BG-11:ATCC mixed medium?</P><P><B>Experiment:</B><I>B. subtilis</I> was inoculated into 100mL of ATCC:BG11 minimal medium and grown for 22 days in a 28°C incubator under shaking at 100rpm. OD at 600nm was used to check for growth every second day in the period. Every fourth day the medium was changed by centrifuging the cells and remove the spent medium and add fresh. The cells were centrifuged for 10 minutes at 4500rpm. After 17 days, sucrose was added to a final concentration of 30mM.


</P><P><B>Result:</B> As already shown, <I>B. subtilis</I> cannot grow in BG-11 medium supplemented with 30mM sucrose. We therefore hypothesized that we have to have a mixture of the ATCC and the BG-11 media in the final co-culture. The ability of <I>B. subtilis</I> to grow in a combination of ATCC and BG-11 media was tested. Over a range of 22 days the OD was measured at 600nm to obtain a growth curve (Fig. 11). This shows limited growth of <I>B. subtilis</I> in the ATCC:BG-11 medium, even changing the medium every fourth day do not show any effect on the growth as it did for <I>B. subtilis</I> in ATCC medium (Fig. 7). 


</P><P>However, when the cells were centrifuged before the new medium was added it was noted that the pellet was red in the samples with the ATCC:BG-11 medium compared to cells grown in ATCC medium (Fig. 12) Also, this pellet was much easier to resuspend than the pellet grown in ATCC medium. This strongly indicates that <I>B. subtilis</I> will form spores when it is grown in the ATCC:BG-11 medium. 


To verify the spore formation in the ATCC:BG-11 medium, 30mM sucrose was added when the medium was changed after 17 days. After just a couple of days, it can be seen that the OD increases meaning that the cells begin to grow (Fig. 11). After the addition of the sucrose it was also noted that the red pellet began to decrease, while the cells got more dominated in the pellet after centrifugation.
These data taken together strongly indicates that <I>B. subtilis</I> cannot grow in ATCC:BG-11 medium due to carbon starvation. As soon sucrose is added, the formed spores will germinate and more cells will begin to form in the medium.
</P><DIV class="center-content"><P><B> ---------------- </B></P></DIV><P><B>Question:</B> Can <I>S. elongatus</I> grow in ATCC:BG-11 mixed medium? Can it grow during induction (+150 mM NaCl and 0.1 mM IPTG) as it should in our final co-culture? </P><P><B>Background:</B><I>S. elongatus</I> can grow in BG-11 but <I>B. subtilis</I> cannot, regardless of 30 mM sucrose supplementation. Therefore it was hypothesize that it might be able to grow on ATCC:BG-11 medium (later confirmed when the medium is supplemented with 30mM sucrose). Here, we test whether <I>S. elongatus</I> can grow in ATCC:BG-11 medium at different salt concentrations. 

</P><P><B>Experiment:</B><I>S. elongatus</I> was inoculated from BGH5 plates containing 10μg/ml chloramphenicol. In the series without antibiotics, <I>S. elongatus</I> was inoculated from plates without chloramphenicol. Experiments were done in 150mL baffled flasks in a total of 50mL ATCC:BG-11 medium with 10μg/ml chloramphenicol. The cultures were induced 3 days later, with 0.1mM IPTG and addition of different salt concentrations (0mM, 50mM, 100mM, 150mM, 200mM and 300mM) and incubated in a light chamber at 30°C while shaking at 250rpm. OD at 730 was measured twice a day for 5 days.

</P><P><B>Result:</B> CscB permease transformed <I>S. elongatus</I> grew at a slower rate compared to WT regardless of chloramphenicol addition, suggesting that rerouting carbon reflux through cscB, and not selective pressure, was the major inhibitor of growth rate (Fig. 13). This was confirmed by omitting IPTG at 300mM NaCl (we originally wanted to do this at 150mM  NaCl because sucrose export was higher at this concentration than 300mM, however, these samples were lost) (Fig. 14). We observe a full rescue from growth impairment, confirming the IPTG inducible cscB gene to be at fault. 

</P><P>Taken together, these data show that both the wild type <I>S. elongatus</I> and the cscB permease strain have the ability to grow in ATCC:BG-11 medium. However, the cscB gene is being induced too strongly. In the future, one might consider using a weaker promoter to sustain the culture during cscB induction. </P><DIV class="center-content"><P><B> ---------------- </B></P></DIV><P><B>Question:</B> Can <I>B. subtilis</I> grow in spent medium from cscB permase transformed <I>S. elongatus</I></P><P><B>Experiment:</B> The cscB containing <I>S. elongatus</I> strain was grown in a baffled flask with BG-11 medium containing 10μg/mL chloramphenicol for two days. When the samples began to be green - after a couple of days after inoculation, the samples were induced by adding 0.1mM IPTG to the <I>S. elongatus</I> along with varying salt concentrations (0mM, 100mM, 150mM, 200mM and 300mM NaCl). Then the samples were incubated for 7 days in a light chamber at 30°C, while shaking at 250rpm and the spent medium was collected by pellet the <I>S. elongatus</I>. The pelleting was carried out by centrifuging the samples in 50mL falcon tubes for 30 minutes at 4500rpm. The supernatant was then moved to new sterile 50mL falcon tubes and stored in fridge until usage. 
After collection of the spent medium it was combined with ATCC without sucrose in a ratio of 1:1. <I>B. subtilis</I> was then inoculated in the medium and incubated at 28°C while shaking at 100rpm for 14 days. OD at 600nm was measured every second day. From day six, the medium was removed and fresh spent medium with ATCC without sucrose in a ratio of 1:1 was added to the cells every second day.

</P><P><B>Result:</B> To test whether the <I>B. subtilis</I> could survive in spent medium from <I>S. elongatus</I>, <I>B. subtilis</I> was inoculated in spent medium. The spent medium was taken from <I>S. elongatus</I> that had grown in different environments (Fig. 15). The growth of <I>B. subtilis</I> in spent medium shows that <I>B. subtilis</I> grow best in the medium that arose from <I>S. elongatus</I> grown in 150mM NaCl. This correspond very well with the data obtained from the sucrose measurements. Also this shows that <I>S. elongatus</I> produce and secrete enough sucrose to sustain growth and development of <I>B. subtilis</I> in the co-culture. The fall the red curve has at the end suggests that the medium needs to be refreshed again - probably due to carbon starvation. However, it should be noted that the orange curves has the same growth conditions as the red, but shows a lot less growth of <I>B. subtilis</I> (Fig. 15). The reason for this can be that the <I>S. elongatus</I> showed less grow after induction compared to the cells from the red curve, this also means that there is less cells to secrete sucrose. 

</P><H2 style="font-family: aspace bold demo" style="font-weight: bold" style="font-size:30px">Package for PLA-production in <I> Bacillus subtilis</I></H2><DIV class="center-content"><P><B> ---------------- </B></P></DIV><P><B>Question:</B> Is it possible to fuse Lactate Polymerizing Enzyme (LPE) and Green Fluorescent Protein (GFP)?</P><P><B>Experiment:</B> The experimental set-up can be seen at our <A href="">protocol section</A>. Firstly, both LPE and GFP were amplified using PCR. The DNA template from IDT was mixed with the prepared MasterMix, and the PCR was started at the following conditions for LPE: 95°C for 3', 95°C for 30’’, 60°C for 30’’, 70°C for 1’ and 70°C for 10’, with the three middle step repeated 35 cycles. For GFP the PCR program used was 95°C for 3’, 95°C for 30’’, 55°C for 30’’, 70°C for 30’’ and 70°C for 10’, with the three middle step repeated 35 cycles. Both electrophoresis lasted for 1 hour at 115 volt. After amplification the integrity of the amplified DNA was checked on a 1% agarose gel, and the DNA was purified using PCR Purification kit from QIAGEN according to the manufacturer's protocol. After purification the DNA concentration was measured using a NanoDrop.
To fuse the two genes a fusion PCR was carried out. The purified gene products was used in a PCR along side with a prepared MasterMix containing GC buffer, see <A href="https://2016.igem.org/Team:UNIK_Copenhagen/Experiments">protocol section</A> for the exact composition of the MasterMix. The PCR program for fusing and amplifying the gene construct was a follows 95°C for 3’, 95°C for 45’, 65°C for 45’, 70°C for 2’ and 70°C for 10’. After amplification the PCR product was loaded on a gel for electrophoresis. The conditions for the electrophoresis was 1 hour at 115 volt.

</P><P><B>Result:</B> At first, the two genes LPE and GFP should be amplified to get the right overhangs. This is so, because the genes was ordered as the BioBrick standard. Therefore, primers were designed to incorporate a part of the vector to the 5’ end of LPE, a part of 5’ end of GFP to the 3’ end of LPE and a part of 5’ end of PCT to 3’ end of GFP. The reason for this is to make Gibson Assembly to insert all four genes into the vector pHT254. The PCR product showed that both genes were amplified, and gave a band as the correct sizes (Fig. 16A). Also, no unspecific binding was seen, meaning that PCR purification could be carried out for both of them.
</P><P>After the initial PCR, the two genes had to be fused to make one construct. The reason for this is because Gibson Assembly will have a higher success rate with as few constructs, meaning that as few constructs as possible is beneficial. 

The first time the fusion PCR was carried out did not show a band at the expected size of 2.4 kb (Fig. 17). Instead a band was seen around 1 kb. This could indicate that GFP along with a bit of LPE attached to its 5’ end. This could have been verified by sequencing - however, we choose not to, because the PCR protocol was tried to be optimized instead by looking into the PCR program, and especially the annealing temperature. These optimization steps did not work out as assumed, and no band at the expected size was seen after the PCR. 
</P><P>
These not successful approaches to get the fusion between LPE and GFP led us to look into the primer design. We found that the overhang between LPE and GFP might have been too big, and therefore new primers were designed to reduce the overhang, and the thereby enhance the chance for success for the fusion PCR. The PCR products after amplification can be seen in Figure 16B. After purification, a new fusion PCR was carried out (Fig. 18). 
</P><P>
As figure 18 shows, the fusion PCR did not work. Again, a band around 1 kb is seen instead of the expected size of 2.4 kb. The PCR reaction was then again tried to be optimized, however none of optimized PCR reactions showed the correct band - except for one. One of the fusion PCRs did work (Fig. 19). As it can be seen this is a very weak band that arose at the expected size. This piece was cut out, and a new PCR was done directly on it after advice from one of the lab technicians. However, this new PCR did not work, and the sample was lost.
</P><P>
New PCRs with the exact same conditions and recipe was done afterwards to try to obtain just a tiny bit of PCR product again, but all of the trials were unsuccessful, and no more fusion product for LPE and GFP was obtained.
Taken these data together show that LPE and GFP could not be fused together by the means of fusion PCR. Even though both the PCR programs was changed to try different conditions in regard to the annealing temperature, denature time and elongation time. Future experiments to get LPE and GFP to be fused is to make a touchdown PCR to see if starting at a high annealing temperature and then decrease it every cycle will make the primers more effective, or another approach is to try to redesign the primers to make a more suitable overhang for the fusion PCR. This is some of the approaches that will be carried out in the future to make the fusion of LPE and GFP possible.

</P><DIV class="center-content"><P><B> ---------------- </B></P></DIV><P><B>Question:</B> Is it possible to fuse Propionate-CoA Transferase (PCT) and Yellow Fluorescent Protein (YFP)?</P><P><B>Experiment:</B> The experimental set-up can be seen at our <A href="https://2016.igem.org/Team:UNIK_Copenhagen/Experiments">protocol section</A>. Firstly, both PCT and YFP were amplified using PCR. The DNA template from IDT was mixed with the prepared MasterMix, and the PCR was started at the following conditions for PCT: 95 degrees for 3’, 95 degrees for 30’’, 60 degrees for 30’’, 70 degrees for 1’ and 70 degrees for 10’, with the three middle step repeated 35 cycles. For YFP the PCR program used was 95 degrees for 3’, 95 degrees for 30’’, 56 degrees for 30’’, 68 degrees for 30’’ and 68 degrees for 10’, with the three middle step repeated 35 cycles. Both electrophoresis lasted for 1 hour at 115 volt. After amplification the integrity of the amplified DNA was checked on a 1% agarose gel, and the DNA was purified using PCR Purification kit from QIAGEN according to the manufacturer's protocol. After purification the DNA concentration was measured using a NanoDrop.
To fuse the two genes a fusion PCR was carried out. The purified gene products was used in a PCR along side with a prepared MasterMix containing GC buffer, see <A href="https://2016.igem.org/Team:UNIK_Copenhagen/Experiments">protocol section</A>  for the exact composition of the MasterMix. The PCR program for fusing and amplifying the gene construct was a follows 95°C for 3’, 95°C for 45’, 60°C for 35’, 72°C for 3’ and 72°C for 10’. After amplification the PCR product was loaded on a gel for electrophoresis. The conditions for the electrophoresis was 1 hour at 115 volt, and the obtained band was purified using Gel Extraction Purification kit from QIAGEN.

</P><P><B>Result:</B> As for LPE and GFP, both PCT and YFP were amplified by PCR using primers that would incorporate the right overhangs. For PCT the overhangs was the 3’ end of GFP on the 5’ end of PCT, the 5’ end of YFP on the 3’ end of PCT. The final PCR product can be seen in Figure 20.
</P><P>The PCR product was clearly seen at the expected sizes, meaning that the PCR was a success and it was possible to amplify both of the genes. PCR purification was carried out, and the purified products was used to make a fusion PCR to fuse PCT and YFP. Also here, the PCR protocol had to be updated to obtain a band at the correct size. The first few times, the fusion PCR did not work (Fig. 21A), so the protocol was improved in regard to annealing temperature and elongation time.
</P><P>
With the updated protocol it was able to fuse the two genes (Fig. 21B). This can be seen as a rather strong band at the expected size of 2.6 kb when the PCR product was loaded on a 1% agarose gel. AS it also can be noted a lot of unspecific binding was present, forcing us to made gel purification instead of the more efficiently PCR purification. This can be avoided by improve the PCR protocol further.
In conclusion the data strongly indicates that the two genes indeed can be fused together as expected. However, to get a more pure final product, the PCR protocol needs to be improved to avoid all the unspecific binding that can be seen in Figure 21B. Another way to improve the outcome of the PCR is to design new primers to make them more specific to the gene.
</P><DIV class="center-content"><P><B> ---------------- </B></P></DIV><P><B>Question:</B>Was the pHT254 vector linearized?</P><P><B>Experiment:</B> Vector pHT254 was digested with the restriction enzymes XbaI and XmaI to linearize it before the Gibson Assembly. The process for this can be seen in the protocol section.</P><P><B>Results:</B>To make the Gibson Assembly functional, the vector needs to be linearized. This is done by making a restriction digestion of the vector by using XbaI and XmaI (Fig. 22).</P><DIV class="Center-content"><P><I> Figure 22: Digestion analysis of pHT254 cut with the restriction enzymes XmaI and XbaI.</I></P></DIV><P>The digestion analysis shows we got a band between 7.5 kb and 8.0 kb that correspond to our vector. This band can both be seen when the vector was cut with one of or both of the restriction enzymes (Fig. 22). This means that the vector is being cut. No smaller fragment is being seen as such, which is expected because  the fragment that is cut out when both restriction enzymes are used are so small it cannot be visualized on the agarose gel. 
As it can be seen on Figure 22, a fragment around 1.0 kb is seen (at the very bottom of Fig. 22). This is a unidentified fragment, but it might be some uncut vector. When the vector is not cut it can be coiled so it mimics a smaller size when it is loaded on a gel. However, we hypothesize that the fragment that seen around 3.5 kb is the uncut fragment, because it has disappeared in the samples that have been cut with the restriction enzymes.
Taken together, these data shows that it was possible to digest the vector with the two restriction enzymes, thereby making it ready for Gibson Assembly.</P><P id="experiment14"><B>Journal reference:</B></P><P><B>Question:</B>Did Gibson Assembly work?
 </P><P><B>Experiment:</B>After all the genes were amplified so they all had homologous sites, as described in Biosynthetic Make-up (https://2016.igem.org/Team:UNIK_Copenhagen/Biosyntetic_Make_Up), Gibson Assembly was carried out according to the manufactures protocol. Each of the constructs (LPE, GFP, PCT-YFP, PhaA-CFP and PhaB-RFP-Terminator) were added in an Eppendorf tube, each with an amount of maximum 1pmol, and 2X Gibson MasterMix was added to the Eppendorf tube and the samples were incubated at 50 degrees for two hours. Then the samples were stored on ice upon transformation.
The transformation of E. coli was carried out as described in the protocol section (<A>https://2016.igem.org/Team:UNIK_Copenhagen/Experiments</A>). Chemical competent cells were thawed on ice and the DNA was added. Incubate on ice for 30 minutes, and heat shock the cells for 30 seconds at 42 degrees and let the cells rest on ice for two minutes. Then add recovery medium and incubate at 37 degrees for up to two hours and plate the cells out on LB plates containing 50μg/mL ampicillin. Store the plates at 37 degrees overnight and check the transformation efficiently the day after. 
Colony PCR is carried out on the formed colonies to screen for colonies that have the five construct and the vector backbone. The protocol for PCR can be found in the protocol section (<A>https://2016.igem.org/Team:UNIK_Copenhagen/Experiments</A>). 
</P><P><B>Result:</B> to test if the Gibson Assembly worked, a colony PCR was carried out on the grown colonies after transformation. The PCR was chosen to be done for GFP, because it gave a clear band when it was amplified (Fig. 16B). But also presence for YFP and PhaA were tested during the colony PCRs. 
Figure 23 shows the first colony PCR where presence for GFP was investigated. As it can be seen the forty colonies investigated, all but one were negative for GFP. This one colony was further investigated for the presence of YFP. However, this ended up being negative, meaning that it would not have had all the six fragments from the Gibson Assembly. Also, none of the forty colonies showed YFP. This further suggests that the Gibson did not work (Fig. 24). The PCR for YFP shows some bands in several of the colonies. However, these bands are only around 700 bp long, so about 200 bp smaller than the expected size for YFP. Because of this, we conclude that the primers can have some unspecific binding to the vector.
</P><DIV class="Center-content"><P><I> Figure 23: Colony PCR of GFP. Colon #39 show a positive band at the expected size for GFP. The ladder used was 1kb+ from Thermo Fisher Scientific©. The negative control is carried out with water as template. For the PCR conditions see the protocol section. The positive was taken from the synthetic gene from IDT.</I></P></DIV><DIV class="Center-content"><P><I> Figure 24: Colony PCR for YFP. The colonies here is the same as in Fig. 23. The ladder used was 1kb+ from Thermo Fisher Scientific©. The negative control is carried out with water as template. For the PCR conditions see the protocol section. The positive was taken from the synthetic gene from IDT.</I></P></DIV><DIV class="Center-content"><P><I> Figure 25: Colony PCR for PhaB for the second round of Gibson performed. Bands were observed for all colonies, however none of the bands correspond to the PhaB. The ladder used was 1kb+ from Thermo Fisher Scientific©. For the PCR conditions see the protocol section.</I></P></DIV><P>After several tries, Gibson still did not work out and we concluded that it was not possible to do the Gibson within the timeframe of this project. One possible way to have optimized it, was to design new primers to get better overhangs between the genes. Also, it might be worth looking into adding other tags than the fluorescence, like His or Myc tag. In that the constructs will be smaller and maybe easier to work with.
Another reason why it did not work out was that the Gibson MasterMix used expired one year ago. However, when a new one was bought, it did not work either, suggesting that one of the two suggesting above might have been the reason why it did not work out for us. Instead, we looked into doing conventional cloning on PhaA and PhaB.
</P><P id="experiment15"><B>Journal reference:</B></P><P><B>Question:</B>Was PHaA-CFP successfully cloned in pHT254? </P><P><B>Experiment:</B>PCR amplification was done on PHaA-CFP to introduce XbaI and BamHI restriction sites and the gel-purified amplicon was then digested with mentioned restriction enzymes. Plasmid was also linearized using XbaI and BamHI restriction enzyme. The enzymes were inactivated by incubating at 65 C for 20 min. A ligation reaction containing PHaA-CFP and linearized  pHT254 was carried out at 30 C at overnight. NEB10 <I>E. coli</I> cells were heat shock transformed with the ligation product, plated on 100µg/mL ampicillin and incubated at 37C.</P><P><B>Results:</B>Two small colonies of transformed <I>E. coli</I> with ligation product were detected after overnight incubation at 37C and were subjected to colony PCR to detect presence of PHaA-CFP. Both the colonies showed the presence of PHaA-CFP at an expected size around 1,9 kb. 
The positive colonies were inoculated and PCR was done on miniprep plasmid to verify the presence of PHaA-CFP. A clear band at 19 kb was detected corresponding to PHaA-CFP. Similar steps were carried out with the linearized plasmid which also gave colonies, but no band for PHaA-CFP at 1,9 kb was detected before and after the miniprep step. This concludes that our PHaA-CFP was successfully cloned into pHT254 vector using conventional cloning.</P><DIV class="Center-content"><P><I> Figure 26: Colony PCR for the detection of PHaA-CFP. The first two lane corresponds to the transformation with ligation mix containing PHaA-CFP and pHT254. Lane 3-5 is colony PCR on transformants transformed with ligation treated pHT254 vector. Arrows on the picture indicate PHaA-CFP.</I></P></DIV><DIV class="Center-content"><P><I>Figure 27: PCR for the detection of PHaA-CFP after miniprep purification of the vector from transformants. Lane 2 and 3 corresponds to ligation mix containing PHaA-CFP and pHT254. The arros indicate bands for PHaA-CFP at 1,9 KB. Lane 4 corresponds to miniprep on transformant transformed with ligation treated pHT254 vector only.
</I></P></DIV><P id="experiment16"><B>Journal reference:</B></P><P><B>Question:</B>Was PHaB-RFP-Term successfully cloned into pHT254 already containing  PHaA-CFP?
</P><P><B>Experiment:</B>PHaB-REFP-Term gene was PCR amplified to introduce BamHI XmaI restriction sites and was gel-purified. The purifed PHaB-REFP-Term and miniprep vector pHT254 containing PHaA-CFP was digested with BamHI and XmaI for 2 hours at 37C. An overnight  ligation reaction was carried at 16 C. NEB10 was transformed with ligation mixture, plated on 100µg/mL ampicillin and incubated at 37C. </P><P><B>Results:</B> No E.coli colonies were detected on the plate after transformation to analyse indicating that the cloning of PHaB-REFP-Term in pHT254 already containing  PHaA-CFP did not succeed (Fig. 28).</P><DIV class="Center-content"><P><I>Figure 28: E. coli transformed with PhaA-containing pHT254. No colonies were formed on the selective plate indicating that PhaB was not cloned into the PhaA-containing vector. 
</I></P></DIV><H2 style="font-family: aspace bold demo" style="font-weight: bold" style="font-size:30px">Characterization of previous BioBricks</H2><DIV class="center-content"><P><B> ---------------- </B></P></DIV><B>Journal reference:</B>22 july - 1 august<P><B>Question:</B> Does CscB-transformed PCC7942 export sucrose to the surrounding medium when induced by osmotic stress and IPTG? How much sucrose is exported?</P><P><B>Background:</B> Sucrose is a disaccharide composed of glucose and fructose that are linked together with alpha- (1,2) glycosidic linker. In photosynthetic organisms like cyanobacteria and plants, sucrose is naturally produced and stored by means of photosynthesis.</P><P><B>BioBrick characterization:</B> The iGEM team from Brown-Stanford 2011 utilized a cscB coding gene inspired from the research of Daniel Ducat and Pamela Silver at Harvard Medical School. This produced the BioBrick Bba_K656011 (<A>http://parts.igem.org/Part:BBa_K656011:Experience</A>). However, they were not able to perform sucrose measurements when their documentation ended, as described in their wiki-pages (<A>https://2011.igem.org/Team:Brown-Stanford/PowerCell/NutrientSecretion</A>). We  would therefore like to add sucrose measurements to their BioBrick pages. With the permission from the original authors, we borrowed a cscB-transformed PCC7942 S. elongatus and characterized growth and sucrose accumulation in the extracellular medium during 7 days of induction, at different osmotic pressures.</P><P><B>Experiment:</B></P><P>The S. elongatus PCC7942 was inoculated in 100 mL BG-11 media in a conical flask and incubated at 30 C under 200 rpm shaking and light bulb. After 3 days of growth, 1,5 mM of IPTG was added to induce the expression of sucrose export cscB gene for sucrose transport into the media.</P><P>The amount of sucrose secreted in the media was determined using Dionex- High-performance anion exchange chromatography - pulsed ampherometric detection (HPAEC-PAD) by using using CarboPac SA 10 (Thermo Scientific) column for fast separation of sugars.</P><P>HPAEC-PAD is an analytical device to detect and characterise simple to complex carbohydrates in the sample based on their electrocatalytic activity at high pH-level using NaOH. 200 µL of inoculated spinned media containing secreted sucrose was collected twice a day from the first day of induction for 7 days. The spined media was transferred to a HPAEC-PAD Vial (and septra) and the running program was consisted by a constant flow rate of 0,35 mL/min with the injection volume of 20 µL.</P><P>To validate the amount of sucrose secretion in different conditions, 17 different samples of induced S. elongatus PCC7942 were prepared with different osmolarity (0mM, 50mM, 100mM, 150mM, 200mM, 300mM). Furthermore, the amount of sucrose secretion was validated with and without selection pressure. Optical density was measured at 730nm wavelength to see a correlation between selected osmolarity and growth.</P><P>Parameters used:  salt 0-300 mM with/without the selection pressure. time, OD</P><P>received by Pamela Silvers lab Harvard medical School,</P><P>Samples of cyanobacteria supernatant were analyzed by HPAEC-PAD on a Dionex BioLC (Dionex, CA, USA), using CarboPac SA 10 (Thermo Scientific) column for fast separation of sugars. The running program was consisted by a constant flow rate of 0.35 ml/min and injection volume of 20 µL. Before injection of each sample, the column was washed with 0.5 M NaOH for 5 min, then equilibrated with water 10 min. The elution program consisted of an isocratic elution with water from 0 to 10 min, followed by a linear gradient up to 0.5 M NaOH from 10 to 15 min. Monosaccharides were detected using a pulsed amperometric detector (gold electrode) set on waveform A, according to manufacturer’s instructions, with a post-column addition of 1 M NaOH for detection. Monosaccharide and disaccharide standards were from Sigma and included D-Glc, D-Fru, and sucrose. For verification of the response factors, a standard calibration was performed before analysis of each batch of samples for measurement of sucrose content.</P><P><B>Results:</B> Although some samples were lost post 78 hours, sucrose was observed to rise steadily in the samples above 100 mM NaCl (Fig. 29) , which is also expected from the literature (Ducat et al 2012). However, a sudden and consistent drop in sucrose concentration was observed for the S5 series with 200 mM NaCl. We hypothesized that it was either a technical artifact or due to loss of live bacteria. Measuring OD confirmed that cell death could potentially explain the drop, since the OD for 200 mM series dropped throughout the experiment (Fig. 30). Such drop is likely due to rerouting of the carbon flux to the extracellular, however, we were only able to confirm that the drop in OD was dependent on the osmotic pressures, and that 50-150 mM were able to keep the cells alive (at least in figure 23, however, figure 25 tells a different story. They are likely both true and within a certain variance, we couldn’t measure). 
We were also interested in measuring OD and sucrose accumulation when CAM was omitted to test the stability of the cscB transformation (Fig. 31 and 32). However, due to sample loss it was difficult to compare them after seven days. Although it would seem, that antibiotics did not to seem to be crucial to keep the transformation stabile for seven days. In conclusion, cscB transformed PCC7942 was able to accumulate up to 1.4 mM in the extracellular medium, and omitting antibiotic did not seem to lower the overall sucrose exportation. </P><DIV class="center-content"><P><I>Figure 29: Sucrose accumulation with CAM: <I>S. elongatus</I> was induced at various concentrations of NaCl and 0.1 mM IPTG. Chloramphenicol(CAM) is an antibiotic, and was present at 10 ug/ml. Measurements span over 7 days with induction at time 0.
</I></P></DIV><DIV class="center-content"><P>Figure 30: OD measurements without CAM. S. elongatus was induced at various concentrations of NaCl and 0.1 mM IPTG. CAM was present at 10 ug/ml. Measurements span over 7 days with induction at time 0 hrs. 
</P></DIV><DIV class="center-content"><P><I>Figure 31: Sucrose accumulation without CAM. S. elongatus was induced at various concentrations of NaCl and 0.1 mM IPTG. CAM was omitted to test the stability of our setup without antibiotics. Measurements span over 7 days with induction at time 0 hrs. 
</I></P></DIV><DIV class="center-content"><P><I>Figure 32: OD measurements without CAM: S. elongatus was induced at various concentrations of NaCl and 0.1 mM IPTG. CAM was omitted to test the stability of our setup without antibiotics. Measurements span over 7 days with induction at time 0 hrs. 
</I></P></DIV><H2 style="font-family: aspace bold demo" style="font-weight: bold" style="font-size:30px">Validating our BioBricks</H2><P id="experiment17"><B>Journal reference:</B></P><P><B>Question:</B>Validation of AmpE in <I>E. coli</I>?

</P><P><B>Experiment:</B></P><P><B>The plasmid pTH254</B> harbours AmpE resistant gene. E.coli strain NEB10, DH5-alpha, DH5-beta is transformed with plasmid pHT254 using heat-shock transformation method, thereby plated on LB plate containing 100µg/mL ampicillin and incubated at 37 C. E. coli transformation with water was plated on 100 µg/mL ampicillin was used as a negative control.

</P><P><B>Results:</B> Colonies were formed when pHT254 transformed <I>E. coli</I> was plated on LB plate 100µg/mL ampicillin but no growth was detected on LB plate with 100µg/mL ampicillin when <I>E. coli</I> transformed with water only. This strongly indicates that <I>E.coli</I> needs AmpE gene in order to survive on ampicillin containing plates. Thereby, validating the newly added BioBrick <A href="http://parts.igem.org/Part:BBa_K1975008">BBa_K1975008</A>.
</P><DIV class="Center-content"><P><I>Figure 34: no colonies are formed on the plate with ampicillin if not they are transformed with the vector containing the AmpE gene. This means that the cells need this gene to survive on ampicillin.
</I></P></DIV><P id="experiment21"><B>Journal reference:</B> Future experiment</P><P><B>Question:</B> Is our composite biobrick able to produce plastic?</P><P><B>Experiment:</B> For instance scaling up the growth assays, and, very importantly, having <I>S. elongatus</I> and <I>B. subtilis</I> grow next to each other seperated by a membrane.</P><P><B>Result:</B></P><H2 style="font-family: aspace bold demo" style="font-weight: bold" style="font-size:30px">Testing <I>Synechococcus elongatus</I> and <I>Bacillus subtilis</I> at simulated extraterrestrial environments</H2><P><A>https://2016.igem.org/Team:UNIK_Copenhagen/Achievements</A></P></DIV></DIV></DIV></DIV></DIV></DIV></BODY></HTML>