"
Page
Discussion
View source
History
teams
Log in
 
Team:Berkeley/Project/GT
From 2013.igem.org
Project Blue Jeans
Introduction
Making Indican
Dyeing with Indican
Characterization of Indoxyl Production
Human Practices
Talks with Dyeing industry
Visit to a Scale-Up Facility
Engineering a Cost Effective System
Outreach
Methods
Protocols
Parts
Other Applications
Notebooks
iGEM
Judging
Safety
Team
Attributions
Making Indican
Why soluble indigo?
Biosynthetic production of indican
Testing homologous enzymes
Producing indican with OleD
Indigo plant cDNA
References
Solubilizing Indigo via Glycosylation
Why the need for a soluble indigo?
Indigo is so insoluble in water that it is not capable of dyeing clothes by itself.  Industrial dyeing gets around this problem by reducing indigo to leuco-indigo, a white soluble version of the dye. In our project, we have found a way to use indican (a natural occurring compound in indigo plants) to dye clothing . Indican is a soluble precursor to indigo, and serves as a biosynthetic alternative to leuco-indigo. Here we present the first instance of indican production using a recombinant enzyme.
BioSynthetic Production of Indican
Indigo producing plants have a naturally encoded pathway to produce indican. They utilize a glucosyl transferase (GT) to add a glucose molecule to the hydroxyl group of indoxyl. We intend to produce indican by co-expressing FMO and a GT in E. coli. Unfortunately, no sequence data is available for the glucosyl transferases that have activity on indoxyl. As part of our summer project, we embarked in a quest to find an indican producing GT. We followed two main tactics in our search; Testing homologous enzymes and screening plant cDNA libraries. Hard work resulted in discovery of three new GTs, and the demonstration of indican biosynthesis! This represents the first time that a protein sequence has been linked to the production of indican. Click here to jump to oleD, an indican producing GT.
Homologous Enzymes
Glucosyl transferases(GTs) are ubiquitous and have been shown to act on a vast variety of substrates. As part of our search for an indican producing GT we have tested enzymes shown in the literature to have activity on substrates that resemble indoxyl.
This summer we have cloned a variety of GTs that have activity on compounds like benzoate, and jasmonate. A table with accession numbers and native substrates of our GTs is shown below.
As a first experiment, we attempted to express all of our GTs in E.coli to check for solubility. We decided to test solubility by generating a C-terminus fusion of GTs with a yellow fluorescent protein (Venus). A sample parts plasmid is shown below.  Imaging under a fluorescent microscope showed that most of our GTs formed inclusion bodies (Represented by punctate fluorescence as opposed to diffuse fluorescence).
Figure: Representative pictures of fluorescently tagged GT expression in E.coli. (Left) AT1G05680-Venus fusion expressed in E.coli. Diffuse fluorescence inside the cell is representative of correctly expressed or soluble protein. (Right) AT1G05530-Venus fusion expressed in E.coli. Punctate fluorescence inside the cell is representative of incorrectly expressed or insoluble protein.
Producing indican with OleD
OleD is a bacterial glucosyltransferase that natively glucosylates oleandomycin. Most interestingly, OleD is known to have a broad substrate specificity which includes compounds that are similar to indoxyl. We cloned and purified OleD to test the enzyme invitro for indican production. The vector is shown below.
To test indican production, we set up the following experiment.
Reaction mixes were set up following the conditions found in Thorson et. al. 2010
Unfortunately, no indican production was observed. How did we make OleD work?
We came up with two possible solutions:
1. We can modify OleD’s substrate specificity to accommodate other compounds that are similar to indoxyl.
2. We can run the experiment under anaerobic conditions, thus preventing indoxyl from oxidizing into indigo.
A set of mutations identified by Gantt et. al. (2008) can impart a much broader substrate specificity to oleD than it already possesses. This set of mutations, called oleD-ASP, changes the A242V-S132F-P67T amino acid positions.  We cloned and purified oleD-ASP.
Here we present the first instance of indican production using a recombinant enzyme! This is a major accomplishment in the development of our new biological dyeing process. The results from our anaerobic in-vitro experimentation with OleD is shown below.
Absorbance data obtained from HPLC at 270nm. Reaction containing indoxyl, OleD, and donor molecule UDP-glucose peaks for indican and sodium complexed indican are observed along with the precursor indoxyl.
Absorbance data obtained from HPLC at 270nm. Control reaction containing indoxyl and OleD (No donnor molecule - UDP glucose).
Mass spectrometry data for the elution fraction at a retention time of 9.51 (Indican peak in HPLC data shown in previous figure). Peaks matching the mass of the precursor indoxyl (134), indican (296), and sodium complexed indican (318) were found. All unlabeled peaks did not correspond to expected ions.
Indigo plant cDNA
As part of our efforts to find an indican producing GT, we decided to search the transcriptome of indigo plants. This summer we have acquired 3 different indigo plants (Indigofera tinctoria, Indigofera suffruticosa, and Polygonum tinctorium seen in the picture below). In addition we collected a sample of Isatis tinctoria from the Berkeley botanical gardens.
Image: From left to right; Indigofera tinctoria, Indigofera suffruticosa, and Polygonum tinctorium. In the background, the beautiful San Francisco bay and the Golden Gate bridge.
Unfortunately, none of the indigo producing plants have had their genomes sequenced. To study them, we extracted their RNA from the leaf tissue and reverse transcribed it to generate cDNA libraries of all four plants. These cDNA libraries are being screened for indican producing GTs.
RNA work is tricky, so please take a look at our detailed protocols for RNA extraction and cDNA library generation from plants!
In order to screen cDNA, we have generated a multiple sequence alignment (MSA) of Glucosyl Transferases (See image below).
Figure: Multiple sequence alignment of beta glucosyl transferases from the taxonomic group “Core Eudicotyledons”
The MSA allowed us to find a very well conserved region known as the PSPG box. We have used this conserved region to create degenerate primers (see below) for the extraction of glucosyl transferases.
Our detailed protocols for degenerate primer generation can be found here .
Our screening efforts have started to give good results! We have extracted three GTs that have never been studied before. All GTs are being characterized and submitted to NCBI.
Here is a screen shot of our first NCBI submission!
References
Gantt , Richard W. "Probing the Aglycon Promiscuity of an Engineered Glycosyltransferase." 47.46 (2008): 8889–8892. Print.
.
Warzecha, Heribert. "Formation of the indigo precursor indican in genetically engineered tobacco plants and cell cultures." 1467.7652 (2007): 185–191. Print.
.
Links to Glucosyl Trasnferases. See table for more accession numbers: Accession Number Table
http://www.brenda-enzymes.org/literature/lit.php4?e=2.4.1.220&r=488950| BRENDA]
http://www.ncbi.nlm.nih.gov/protein/295854835?report=genbank&log$=prottop&blast_rank=3&RID=UA9TK3KH015
http://www.ncbi.nlm.nih.gov/nuccore/295854834?from=32&to=1462&sat=4&sat_key=63128784
http://www.ncbi.nlm.nih.gov/pmc/articles/PMC168234/pdf/624121.pdf
Retrieved from " http://2013.igem.org/Team:Berkeley/Project/GT "
Recent changes
What links here
Related changes
Special pages
My preferences
Printable version
Permanent link
Privacy policy
Disclaimers
