<HTML xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en" dir="ltr">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki  ltr ns-0 ns-subject page-Team_AMU-Poznan_Project"><DIV id="globalWrapper"><DIV id="top-section"><DIV id="p-logo"><A href="/Main_Page" title="Main Page">&quot;
	    </A></DIV><DIV id="menubar" class="left-menu noprint"><UL><LI class="selected"><A href="/Team:AMU-Poznan/Project">Page               </A></LI><LI class="new"><A href="/wiki/index.php?title=Talk:Team:AMU-Poznan/Project&amp;action=edit&amp;redlink=1">Discussion               </A></LI><LI><A href="/wiki/index.php?title=Team:AMU-Poznan/Project&amp;action=edit">View source               </A></LI><LI><A href="/wiki/index.php?title=Team:AMU-Poznan/Project&amp;action=history">History               </A></LI><LI style="color:white;cursor:default">teams</LI></UL></DIV><DIV class="right-menu noprint" id="menubar"><UL><LI id="pt-login"><A href="/wiki/index.php?title=Special:UserLogin&amp;returnto=Team:AMU-Poznan/Project" title="You are encouraged to log in; however, it is not mandatory [o]" accesskey="o">Log in</A></LI></UL></DIV><DIV id="search-controls" class="noprint"><FORM action="/Special:Search" id="searchform"> </FORM></DIV></DIV><DIV id="content"><H1 class="firstHeading">Team:AMU-Poznan/Project</H1><DIV id="bodyContent"><H3 id="siteSub" class="noprint">From 2014.igem.org</H3><P><TABLE id="menu" width="100%" cellspacing="0" height="135px"><TBODY><TR height="50px"><TD width="162px" align="center" onmouseover="this.bgColor='#FEE5AD'" onmouseout="this.bgColor='#e9eaed'" bgcolor="#e9eaed"><A href="https://2014.igem.org/Team:AMU-Poznan" style="text-decoration:none;color:#1C140D">HOME </A></TD><TD width="162px" align="center" onmouseover="this.bgColor='#FEE5AD'" onmouseout="this.bgColor='#e9eaed'" bgcolor="#e9eaed"><A href="https://2014.igem.org/Team:AMU-Poznan/Project" style="text-decoration:none;color:#1C140D">PROJECT</A></TD><TD width="162px" align="center" onmouseover="this.bgColor='#FEE5AD'" onmouseout="this.bgColor='#e9eaed'" bgcolor="#e9eaed"><A href="https://2014.igem.org/Team:AMU-Poznan/Parts" style="text-decoration:none;color:#1C140D">PARTS</A></TD><TD width="162px" align="center" onmouseover="this.bgColor='#FEE5AD'" onmouseout="this.bgColor='#e9eaed'" bgcolor="#e9eaed"><A href="https://2014.igem.org/Team:AMU-Poznan/Team" style="text-decoration:none;color:#1C140D">TEAM </A></TD><TD width="162px" align="center" onmouseover="this.bgColor='#FEE5AD'" onmouseout="this.bgColor='#e9eaed'" bgcolor="#e9eaed"><A href="https://igem.org/Team.cgi?year=2014&amp;team_name=AMU-Poznan" style="text-decoration:none;color:#1C140D">OFFICIAL TEAM PROFILE</A></TD><TD width="162px" align="center" onmouseover="this.bgColor='#FEE5AD'" onmouseout="this.bgColor='#e9eaed'" bgcolor="#e9eaed"><A href="https://2014.igem.org/Team:AMU-Poznan/Notebook" style="text-decoration:none;color:#1C140D">NOTEBOOK </A></TD><TD width="162px" align="center" onmouseover="this.bgColor='#FEE5AD'" onmouseout="this.bgColor='#e9eaed'" bgcolor="#e9eaed"><A href="https://2014.igem.org/wiki/index.php?title=Team:AMU-Poznan/Safety" style="text-decoration:none;color:#1C140D">SAFETY </A></TD><TD width="162px" align="center" onmouseover="this.bgColor='#FEE5AD'" onmouseout="this.bgColor='#e9eaed'" bgcolor="#e9eaed"><A href="https://2014.igem.org/Team:AMU-Poznan/Modeling" style="text-decoration:none;color:#1C140D">MODELING &amp; SURVEY </A></TD></TR></TBODY></TABLE></P><DIV class="home"><H1>sh-miR Designer v2.0</H1>
Sh-miR designer is a project we started during iGEM 2013. This year we would like to continue and expand functionality of the software. sh-miR designer v1.0 (link) is aimed to create sh-miR molecules based on siRNAs provided by the user. In sh-miR designer v2.0 only the mRNA number (from NCBI database), which expression should be decreased can be provided. Moreover, we expanded functionality of the software with off-target validation and check of immune motifs and also extended miRNA-shuttles database.
sh-miR Designer is a software aimed for fast and efficient design of effective RNA interference (RNAi) reagents - sh-miRs, also known as artificial miRNAs. sh-miRs are RNA particles whose structure is based on miRNA precursor pri-miRNA, but sequence interacting with transcript is changed depending on research purpose. Maintenance of structure of pri-miRNA is very important to enable cellular processing and therefore ensure functionality of artificial particles. sh-miRs delivered to cells on genetic vectors - plasmids or viral vectors - enter natural RNAi pathway and silence target mRNA. They can be used in genetic therapies and basic biomedical research.
We will provide two applications to access the software, one which require siRNA sequences and the second which require transcript accession number from NCBI database. Each user will receive an account with login (e-mail) and password where he/she would be able to check history of software usage.<H1>Biological background</H1>
RNA interference (RNAi) is a physiological process of posttranscriptional gene expression regulation caused by double stranded RNA molecules. This is one of the most willingly used experimental techniques. It is used in experiments designed to explore gene function, in creation cellular models of diseases or in genetic therapy design. Among RNAi technology reagents are siRNA (short interfering RNA) and vector reagents shRNA (short hairpin RNA) and sh-miR (sh- microRNA). RNAi reagents are integrated into endogenous RNAi pathway on different phases. sh-miR reagents are designed in the way to be similar to endogenous miRNA precursors (pri-miRNA), because it is important feature to be recognized by proteins involved in microRNA (miRNA) biogenesis.

shRNA or sh-miR reagents are introduced into cells with usage of genetic vectors – plasmids or viral vectors to ensure intracellular expression. They have huge therapeutic potential beacuse of stable reagent production in cells what enables one time therapeutic dosage. Usage of synthetic siRNA reagents requires often dosage, what makes therapy more difficult, especially in case of Central Nervous System Diseases. 

It was recently shown that RNAi reagents introduces into cells can trigger unwanted non-specific effects, e.g. can interact with different transcripts besides the desired one (so called off-target effect) or induce immunological effect. Because in processing of sh-miR reagents endogenous endonucleasis Drosha and Dicer are involved, in case of high expression of reagent they can saturate those proteins as well as Exportin-5 transport protein responsible for export of sh-miR from nucleus. The saturation can influence endogenous RNAi mechanism. It was shown that sh-miR reagents influence less endogenous RNAi pathway, because of lower level of produced double-stranded RNA (dsRNA), preserving efficiency of translation silencing. They also show longer effect than different types of RNAi reagents. Because of described features sh-miR reagents seems to be good choice when designing efficient and non-toxic RNAi reagents.
 
sh-miR Designer software was designed sh-miR Designer is a software aimed for fast and efficient design of effective RNA interference (RNAi) reagents - sh-miRs, also known as artificial miRNAs. sh-miRs are RNA particles whose structure is based on miRNA precursor pri-miRNA, but sequence interacting with transcript is changed depending on research purpose. Until now scientists have to use multiple unconnected software to design single sh-miR molecule. It is time consuming process, what limits number of designed molecules for biological testing. There is no such software, which includes all important features as molecule spatial structure. Maintenance of structure of pri-miRNA is very important to enable cellular processing and therefore ensure functionality of artificial particles. This interdisciplinary project includes computer sciences with biomedical science. sh-miRs delivered to cells on genetic vectors - plasmids or viral vectors - enter natural RNAi pathway and silence target mRNA. They can be used in genetic therapies and basic biomedical research. <H1>Standard Parts</H1>
We are creating a particle which can be inserted into plasmid built by Standard parts from Registry of Standard Biological Parts. The sh-miR molecules are inserted into plasmid backbone downstream chosen promoter. You can add any regulatory sequence you want or maybe a fragment coding fluorescent protein to see if the transfection was succesfull. The molecules are fully compatibile with Standard Biological Parts - what is important for users from iGEM especially :). 

picture from http://biosettia.com/
<H1>User Guide</H1><DIV align="left">
There are two suplementary versions of our software. First one (based on last year's software sh-miR v1.0) enable to design sh-miR molecules based on siRNA sequence and the second design sh-miR molecules when NCBI transcript number (NM_...) is provided. In the second version the user also can set GC content, maximum off-target and immunostimulatory properties. The user can also set which pri-miRNA should be used to design sh-miR (or all available in our miRNA database).

Please follow video where you can see how to use web version of the software
and short description how to use client version of the software
Both versions are well documented on github and readthedocs (see below)<H3>Usage via Web server
go to our website <A href="https://shmir.wmi.amu.edu.pl">shmir.wmi.amu.edu.pl</A>input form:results:
users' history:</H3><H3>Usage via client</H3>Fold sequence via mfold:

./shmir_client.py mfold ACTGAUUUGAC
Create sh-miR from one siRNA strand (active) or two siRNa strands separated by space.
First strand is active, both are in 5-3 orientation

./shmir_client.py from_sirna UUUGUAUUCGCCCUAGCGC CGCUAUGGCGAAUACAAACA
Create sh-miR from transcript name.
Optional parameters

--min_gc : Minimal &quot;GC&quot; content in strand -- default: 40
--max_gc : Maximal &quot;GC&quot; content in strand -- default: 60
--max_offtarget : Maximal offtarget in strand -- default: 10
--mirna_name : The name of miRNA backbone to use -- default: &quot;all&quot;
--stymulators : One of [&quot;yes&quot;, &quot;no&quot;, &quot;no_difference&quot;] -- default: &quot;no_difference&quot;
./shmir_client.py from_transcript NM_001618.3 --mirna_name miR-30a --min_gc 22</DIV><H1>Technology</H1><DIV align="left">The application was built with Python 2.7. It is divided into 3 parts:
RESTful API - the “heart” of our software - it performs all calculations. Our application is built on CentOS (Linux) operating system. To maintain better compatibility with our server we use Vagrant to create Virtual Machine and Puppet to manage infrastructure of our project. All you need to do is: type into console “vagrant up” and after a while everything is ready to work. This subproject is written with following technologies/frameworks/libraries: Flask, Flask-Cache, SqlAlchemy, PostgreSQL, Celery, Flower, Rabbitmq, Redis, BioPython, mfold (lightweight), uwsgi, sphinx, Python testing tools (mock, nose, unittest).
Console client - simple console application which communicates with API using requests library. It has 3 options: mfold (folding structure), from_sirna (create sh-miR from siRNA) and from_transcript (create sh-miR from transcript name).
Website - application written in Django framework. It enables you to use the API via graphical interface. You can also create your own account and save the history of your requests for later usage.
</DIV><H1>API documentation</H1>Here you can find documentation from code:

API: <A href="https://shmir-api.rtfd.org">shmir-api.rtfd.org</A>
Client: <A href="https://shmir-client.rtfd.org">shmir-client.rtfd.org</A>
Webpage: <A href="https://shmir-designer.rtfd.org">shmir-designer.rtfd.org</A><H1>API for developers</H1>
Here you can find api for developers: <A href="https://github.com/Nozdi/shmir">https://github.com/Nozdi/shmir</A><H1>Releases (re-use of 2013iGEM software)</H1><DIV align="left"><H3>sh-miR designer v1.0 </H3>
First version of the program presented on iGEM 2013 by our team. The user have to provide siRNA sequence (one or both strands). The miRNA database includes 5 miRNAs. <H3>sh-miR designer v1.1 </H3>
Strand discrimination functionality was added based on miRbase.  
extended miRNA database <H3>sh-miR designer v2.0 </H3>
siRNA prediction algorithm was added to functionality (the input is NCBI transcript number) 
user can choose GC content of siRNA molecule, maximal off-target, if he/she wants immunostimulatory sequences inside the molecule and to choose if he/she wants to include all or chosen scaffold 
extended miRNA database 
 
What we are working on: 
strand discrimination based on therodynamics of siRNA ends; 
preparing construct with flanking restriction enzymes 
providing plasmids with parts from standard parts </DIV><H1>Unit Tests</H1><DIV align="left">

In our application we are using unit tests to test alghoritms, functions and database.
We are testing inputs and outputs:<PRE>
    def test_validate_gc_content(self):
        is_in_range = validate_gc_content('ACGT', 40, 60)
        self.assertTrue(is_in_range)
Caught errors:
    def multiple_input(self, data, error):
        for strand in data:
            with self.assertRaises(errors.InputException) as err:
                validators.check_input(strand)
            self.assertEqual(error, str(err.exception))
Database models:
    def test_get_and_put_to_db(self):
        new_obj = self.TestModel(id=153)
        self.put_to_db(new_obj)
        objs = self.get_all(self.TestModel)
        self.assertEqual(len(objs), 1)</PRE>
We are using Mock library to test our functions which are using external API’s to create fake response from server and return them into our function:<PRE>
    @patch.object(ncbi_api.Entrez, 'esearch')
    @patch.object(ncbi_api.Entrez, 'efetch')
    def test_get_data(self, mock_efetch, mock_esearch):
        ids = [69]
        mock_read = Mock()
        mock_read.read.return_value = json.dumps({'esearchresult': {'idlist': ids}})
        mock_esearch.return_value = mock_read
        ncbi_api.get_data(self.GOOD_MRNA)
        mock_esearch.assert_called_with(
            term='NM_001128164.1', db=self.DATA_BASE, retmode='json'
        )
        mock_efetch.assert_called_once_with(
            db=self.DATA_BASE, id=ids, rettype='fasta', retmode='text'
        )</PRE></DIV><H1>Validation</H1><H3>old version shmiR v1.0 (average time)</H3>real	0m6.854s
user	0m0.063s
sys	0m0.027s<H3>new version shmiR v1.1 (average times)</H3>
real	0m3.271s
user	0m0.155s
sys	0m0.041s<H3>Designed sh-miR</H3>
for $ ./shmir_client.py from_transcript NM_000546.5

1: backbone: miR-31, score: 106, sequence: ACATGACGGAGGTTGTGAG
 pdf: ./results/transcript/e9e8b9a2-7e3b-4b24-b26a-633332a3f6b2/miR-31/7270b2d2-0aca-472b-853e-5a9631164d46
 result: CATAACAACGAAGAGGGATGGTATTGCTCCTGTAACTCGGAACTGGAGAGGACATGACGGAGGTTGTGAGCGTTGAACTGGGAACCCTCACAACCTCCGTCA
TGTCTCTCCAGTTCCGAGTTACAGGAGCAATACCATCCCTCTTCGTTGTTATGCTTTCCTGTCTGACAGCAGCTTGGCTACCTCCGTCCTGTTCCTCCTTGTCTT

2: backbone: miR-31, score: 106, sequence: ATGTTTTGTAGAGATGGGG
 pdf: ./results/transcript/e9e8b9a2-7e3b-4b24-b26a-633332a3f6b2/miR-31/56232916-2649-4eb4-9970-c0b0a08ce64e
 result: CATAACAACGAAGAGGGATGGTATTGCTCCTGTAACTCGGAACTGGAGAGGATGTTTTGTAGAGATGGGGCGTTGAACTGGGAACCCCCCATCTCTACA
AAACATCTCTCCAGTTCCGAGTTACAGGAGCAATACCATCCCTCTTCGTTGTTATGCTTTCCTGTCTGACAGCAGCTTGGCTACCTCCGTCCTGTTCCTCCTTGTCTT

3: backbone: miR-31, score: 106, sequence: ATTACATGGGGTCTAGAAC
 pdf: ./results/transcript/e9e8b9a2-7e3b-4b24-b26a-633332a3f6b2/miR-31/37cd84aa-362c-484d-8681-929043cc9c61
 result: CATAACAACGAAGAGGGATGGTATTGCTCCTGTAACTCGGAACTGGAGAGGATTACATGGGGTCTAGAACCGTTGAACTGGGAACCGTTCTAG
ACCCCATGTAATCTCTCCAGTTCCGAGTTACAGGAGCAATACCATCCCTCTTCGTTGTTATGCTTTCCTGTCTGACAGCAGCTTGGCTACCTCCGTCCTGTTCCTCCTTGTCTT

4: backbone: miR-31, score: 106, sequence: ACCATCATCACACTGGAAG
 pdf: ./results/transcript/e9e8b9a2-7e3b-4b24-b26a-633332a3f6b2/miR-31/50b763f6-2193-4743-96c0-61601880e574
 result: CATAACAACGAAGAGGGATGGTATTGCTCCTGTAACTCGGAACTGGAGAGGACCATCATCACACTGGAAGCGTTGAACTGGGAACCCTT
CCAGTGTGATGATGGTCTCTCCAGTTCCGAGTTACAGGAGCAATACCATCCCTCTTCGTTGTTATGCTTTCCTGTCTGACAGCAGCTTGGCTACCTCCGTCCTGTTCCTCCTTGTCTT

5: backbone: miR-31, score: 106, sequence: ATCTCACCCCATCCCACAC
 pdf: ./results/transcript/e9e8b9a2-7e3b-4b24-b26a-633332a3f6b2/miR-31/019ce915-0282-4d83-9538-93c36c790fa8
 result: CATAACAACGAAGAGGGATGGTATTGCTCCTGTAACTCGGAACTGGAGAGGATCTCACCCCATCCCACACCGTTGAAC
TGGGAACCGTGTGGGATGGGGTGAGATCTCTCCAGTTCCGAGTTACAGGAGCAATACCATCCCTCTTCGTTGTTATGCTTTCCTGTCTGACAGCAGCTTGGCTACCTCCGTCCTGTTCCTCCTTGTCTT
see example pdf with structure <A href="https://static.igem.org/mediawiki/2014/d/da/Sequence_1.pdf">link</A>
During sh-miR designer development, colleagues from befriended lab used our software, so we had constant feedback from potential users,
who by the way, have nothing to do with bioinformatics. So it's really hard to satisfy their strong need for easy interface.
We also had significant feedback from our survey (see Modeling tab), so we knew what we should emphasise on during develoment.<H1>Test cases</H1>1) ATXN3 - therapeutic sh-miR against SCA3 disease
transcript number: NM_001164782.1 
GC content: default 
Off-target: 10 
scaffold: ALL 
immunostimulatory: NO 
2) P54 - therapeutic sh-miR against cancer 
transcript number: NM_000546.5 
GC content: default 
Off-target: 10 
scaffold: ALL 
immunostimulatory: YES 
3) biological function analysis, PARP1, we have miR-30a in our lab 
transcript number: NM_001618.3 
GC content: default 
Off-target: 0 
scaffold: miR-30a 
immunostimulatory: no_difference (default) 
4) biological function, PARP1, we have miR-30a in our laboratory and have immune cells 
transcript number: NM_001618.3 
GC content: default 
Off-target: 0 
scaffold: miR-30a 
immunostimulatory: NO 
5) biological function, PARP1 and have immune cells 
transcript number: NM_001618.3 
GC content: default 
Off-target: 0 
scaffold: ALL 
immunostimulatory: NO<H1>References</H1><DIV align="left">
1. Boudreau RL, Monteys AM, Davidson BL. Minimizing variables among hairpin-based RNAi 
vectors reveals the potency of shRNAs. RNA. 2008 Sep;14(9):1834-44. Epub 2008 Aug 12.
2. Chang K, Elledge SJ, Hannon GJ. Lessons from Nature: microRNA-based shRNA libraries. Nat 
Methods. 2006 Sep;3(9):707-14.
3. Dow LE, Premsrirut PK, Zuber J, Fellmann C, McJunkin K, Miething C, Park Y, Dickins RA, 
Hannon GJ, Lowe SW. A pipeline for the generation of shRNA transgenic mice. Nat Protoc. 
2012 Feb 2;7(2):374-93. doi: 10.1038/nprot.2011.446.
4. Fahim M, Larkin PJ. Designing Effective amiRNA and Multimeric amiRNA Against Plant 
Viruses. Methods Mol Biol. 2013;942:357-77. doi: 10.1007/978-1-62703-119-6_19.
5. Fan ZD, Zhang L, Shi Z, Gan XB, Gao XY, Zhu GQ. Artificial microRNA interference targeting 
AT(1a) receptors in paraventricular nucleus attenuates hypertension in rats. Gene Ther. 2012 
Aug;19(8):810-7. doi: 10.1038/gt.2011.145. Epub 2011 Sep 29.
6. Fellmann C, Zuber J, McJunkin K, Chang K, Malone CD, Dickins RA, Xu Q, Hengartner MO, 
Elledge SJ, Hannon GJ, Lowe SW. Functional identification of optimized RNAi triggers using a 
massively parallel sensor assay. Mol Cell. 2011 Mar 18;41(6):733-46. Epub 2011 Feb 25.
7. Maczuga P, Lubelski J, van Logtenstein R, Borel F, Blits B, Fakkert E, Costessi A, Butler D, van
Deventer S, Petry H, Koornneef A, Konstantinova P. Embedding siRNA sequences targeting 
Apolipoprotein B100 in shRNA and miRNA scaffolds results in differential processing and in 
vivo efficacy. Mol Ther. 2012 Oct 23. doi: 10.1038/mt.2012.160. [Epub ahead of print]
8. Maczuga P, Koornneef A, Borel F, Petry H, van Deventer S, Ritsema T, Konstantinova P. 
Optimization and comparison of knockdown efficacy between polymerase II expressed 
shRNA and artificial miRNA targeting luciferase and Apolipoprotein B100. BMC
Biotechnol. 2012 Jul 24;12:42. doi: 10.1186/1472-6750-12-42.
9. Matveeva OV, Nazipova NN, Ogurtsov AY, Shabalina SA. Optimized models for design of 
efficient miR30-based shRNAs. Front Genet. 2012;3:163. Epub 2012 Aug 29.
10. McBride JL, Boudreau RL, Harper SQ, Staber PD, Monteys AM, Martins I, Gilmore BL, Burstein
H, Peluso RW, Polisky B, Carter BJ, Davidson BL. Artificial miRNAs mitigate shRNA-mediated 
toxicity in the brain: implications for the therapeutic development of RNAi. Proc Natl Acad Sci
U S A. 2008 Apr 15;105(15):5868-73. Epub 2008 Apr 8.
11. Schwab R, Ossowski S, Warthmann N, Weigel D. Directed gene silencing with artificial 
microRNAs. Methods Mol Biol. 2010;592:71-88.
12. Silva JM, Li MZ, Chang K, Ge W, Golding MC, Rickles RJ, Siolas D, Hu G, Paddison PJ, Schlabach
MR, Sheth N, Bradshaw J, Burchard J, Kulkarni A, Cavet G,Sachidanandam R, McCombie WR, Cleary MA, Elledge SJ, Hannon GJ. Second-generation shRNA libraries 
covering the mouse and human genomes. Nat Genet. 2005 Nov;37(11):1281-8. Epub 2005 
Oct 2.
13. Stegmeier F, Hu G, Rickles RJ, Hannon GJ, Elledge SJ. A lentiviral microRNA-based system for 
single-copy polymerase II-regulated RNA interference in mammalian cells. Proc Natl Acad Sci
U S A. 2005 Sep 13;102(37):13212-7. Epub 2005 Sep 1.
14. Wang X, Yang Y, Zhou J, Yu C, Cheng Y, Yan C, Chen J.Two-step method for constructing 
Arabidopsis artificial microRNA vectors. Biotechnol Lett. 2012 Jul;34(7):1343-9. Epub 2012 
Mar 22.
15. Warthmann N, Ossowski S, Schwab R, Weigel D. Artificial MicroRNAs for Specific Gene 
Silencing in Rice. Methods Mol Biol. 2013;956:131-49. doi: 10.1007/978-1-62703-194-3_11.
16. Zeng Y, Cullen BR. Efficient processing of primary microRNA hairpins by Drosha requires 
flanking nonstructured RNA sequences. J Biol Chem. 2005 Jul 29;280(30):27595-603. Epub 
2005 Jun 1.
17. Zeng Y, Wagner EJ, Cullen BR. Both natural and designed micro RNAs can inhibit the 
expression of cognate mRNAs when expressed in human cells. Mol Cell. 2002 Jun;9(6):1327-
33</DIV><H1>Judging Form (in <B>bold</B> what we did to achieve gold medal) </H1><DIV align="left">Bronze. The following 4 goals must be achieved:<UL><LI><B>Register the team, have a great summer, and have fun attending the Jamboree in Boston.</B></LI><LI>Create and share a description of the team's project via the iGEM wiki.</LI><LI>Present a Poster and Talk at the Regional Jamboree and World Championship Jamboree.<B> We will on Jamboree :)</B></LI><LI>Develop and make available, via The iGEM Software Registry, an open source software that supports Synthetic Biology based on Standard Parts.</LI></UL>
Silver: In addition to the Bronze Medal requirements, the following 4 goals must be achieved:<UL><LI><B>Demonstrate the relevance of your development for Synthetic Biology based on standard Parts.</B></LI><LI>Provide a comprehensive and well-designed User Guide. (Be creative! An instructional video may work as well.)</LI><LI>Provide detailed API documentation, preferably, automatically built from source code documentation (use tools like doxygen, yard, rdoc, naturaldocs, etc).</LI><LI>Demonstrate that you followed best practises in software development so that other developers can modify, use and reuse your code. Provide more than one realistic test case. Examples of best practices are automated unit testing and documentation of test coverage , bug tracking facilities, documentation of releases and changes between releases.</LI></UL>
Gold: In addition to the Bronze and Silver Medal requirements, two additional goals must be achieved: <B><UL><LI>Provide a convincing validation, testing the performance of the development -- experimentally (can be outsourced) or by other teams and users. Note, even if the algorithm or tool turns out not to work that well, the Gold requirement is fulfilled if the test is good and the analysis convincing. Validation may include: preferably experiments, informatics analysis (complexity, run time) of an algorithm, or user studies.</LI></UL></B>
And the second goal can be any one of the following:<UL><LI>Make your software interact / interface with the Registry.</LI><LI><B>Re-use and further develop previous iGEM software projects (or parts thereof) or use and/or improvement of existing synthetic biology tools or frameworks.</B></LI><LI>Develop a well documented library or API for other developers (rather than “only” a stand-alone app for end users.)</LI><LI>Support and use the SBOL and / or SBOLv standard.</LI><LI><B>iGEM projects involve important questions beyond the bench, for example relating to (but not limited to) ethics, sustainability, social justice, safety, security, or intellectual property rights. Describe an approach that your team used to address at least one of these questions. Evaluate your approach, including whether it allowed you to answer your question(s), how it influenced the team’s scientific project, and how it might be adapted for others to use (within and beyond iGEM). We encourage thoughtful and creative approaches, and those that draw on past Policy &amp; Practice (formerly Human Practices) activities.</B></LI></UL></DIV></DIV><DIV class="printfooter">
Retrieved from &quot;<A href="http://2014.igem.org/Team:AMU-Poznan/Project">http://2014.igem.org/Team:AMU-Poznan/Project</A>&quot;</DIV></DIV></DIV><DIV id="footer-box" class="noprint"><DIV id="footer"><UL id="f-list"><LI id="t-recentchanges"><A href="/Special:RecentChanges" title="Recent changes">Recent changes</A></LI><LI id="t-whatlinkshere"><A href="/Special:WhatLinksHere/Team:AMU-Poznan/Project" title="List of all wiki pages that link here [j]" accesskey="j">What links here</A></LI><LI id="t-recentchangeslinked"><A href="/Special:RecentChangesLinked/Team:AMU-Poznan/Project" title="Recent changes in pages linked from this page [k]" accesskey="k">Related changes</A></LI><LI id="t-specialpages"><A href="/Special:SpecialPages" title="List of all special pages [q]" accesskey="q">Special pages</A></LI><LI><A href="/Special:Preferences">My preferences</A></LI></UL></DIV><DIV id="footer"><UL id="f-list"><LI id="t-print"><A href="/wiki/index.php?title=Team:AMU-Poznan/Project&amp;printable=yes" title="Printable version of this page [p]" accesskey="p">Printable version</A></LI><LI id="t-permalink"><A href="/wiki/index.php?title=Team:AMU-Poznan/Project&amp;oldid=355234" title="Permanent link to this revision of the page">Permanent link</A></LI><LI id="privacy"><A href="/2014.igem.org:Privacy_policy" title="2014.igem.org:Privacy policy">Privacy policy</A></LI><LI id="disclaimer"><A href="/2014.igem.org:General_disclaimer" title="2014.igem.org:General disclaimer">Disclaimers</A></LI></UL></DIV></DIV></DIV></BODY></HTML>