"
Page
Discussion
View source
History
teams
Log in
Â 
Team:Tuebingen/Project/Plasmids
From 2014.igem.org
Home
Project
Overview
Motivation
Inteins
Enzymes
Team
About Us
Attributions
Notebook
Protocols
Lab Journal
Activities
SynBio-Day
School-Project
Aachen Conference
Results
Shipped Parts
Contact
Home
Project
Overview
Motivation
Plasmids
Immobilization Tags
Synthetic Peptides
Enzymes
Applications
Safety
Collaboration
Intellectual property
Team Heidelberg
Team Ui Oslo
Team
About Us
Acknowledgements
Attributions
Notebook
Protocols
Lab Journal
Activities
SynBioDay
School-Project
Aachen Meetup
Results
Shipped Parts
Modeling
Characterization
Contact
Plasmids
pETblue
Since a critical part of our project was the overexpression, chosing the correct vector was crucial. Of particular importance are of course the quality of the promotor and the RBS, as well as a His-Tag for purification. All these parts are available in the parts-registry, but even with the convenient cloning of BioBricks, we thought a more conventional approach would suit our needs better. We chose a pET-vector, which we could easily modify by ligating it with annealed oligo nucleotides. The vector pETBlue-1 already came equipped with a T7-promotor and a LacZ site (see figure 1).
Figure 1: Plasmid map of pETBlue-1 [1] . XbaI and XmaI were used to insert the annealed oligo nucleotides to create an easily accessible vector for overexpression.
Figure 2: Product of olio nucleotide annealing for insertion in pETBlue-1.
Using the oligo nucleotides depicted in figure 2, we were able to create a convenient vector for IPTG-inducible overexpression of genes in the RFC25 standard. The aligned oligo nucleotides had sticky ends to ligate with XbaI and EcoRI sites, creating a scar.
The DNA fragment from annealed oligos was to introduce:
very strong RBS
6xHis-Tag
Thromin-cleavage-site
NgoMIV-compatible restriction site for in-frame insertion
The last point proved to be somewhat tricky. Unfortunately, as figure 1 shows, pETBlue-1 is not optimal for RFC25 cloning, since it has both NgoMIV and AgeI restriction sites. We got around this problem by using a third enzyme with the same sticky ends: XmaI. This was to be the up-stream site for insertion.  Further modifications of the pET-vector were supposed to include the deletion of all NgoMIV and AgeI sites, as well as the changing of the introduced XmaI site to AgeI, so that the vector could be used with conventional RFC25 enzymes.
The AgeI site was successfully changed using site directed mutagenesis PCR. However, the difficulty we had with expression using the modified pETBlue1.2 led us to abandon this side project.
Heidelberg's pSBX1K3 and pSBX4K5
As part of our collaboration with Team Heidelberg we received two expression vectors they designed, pSBX1K3 and pSBX4K5 . We have used pSBX1K3 for test expressions of our constructs.
Figure 3: Plasmid map of pSBX1K3; high copy plasmid for overexpression.
Figure 4: Plasmid map of pSBX1K3; low copy plasmid for overexpression.
References
[1] http://www.addgene.org/vector-database/2657/
Retrieved from " http://2014.igem.org/Team:Tuebingen/Project/Plasmids "
Recent changes
What links here
Related changes
Special pages
My preferences
Printable version
Permanent link
Privacy policy
Disclaimers
