"
Page
Discussion
View source
History
teams
Log in
 
Team:Tuebingen/Notebook/Journal
From 2014.igem.org
Home
Project
Overview
Motivation
Inteins
Enzymes
Team
About Us
Attributions
Notebook
Protocols
Lab Journal
Activities
SynBio-Day
School-Project
Aachen Conference
Results
Shipped Parts
Contact
Home
Project
Overview
Motivation
Plasmids
Immobilization Tags
Synthetic Peptides
Enzymes
Applications
Safety
Collaboration
Intellectual property
Team Heidelberg
Team Ui Oslo
Team
About Us
Acknowledgements
Attributions
Notebook
Protocols
Lab Journal
Activities
SynBioDay
School-Project
Aachen Meetup
Results
Shipped Parts
Modeling
Characterization
Contact
Weekly Journal
Week 1 --- 04.08.14 - 08.08.14
In the first week we successfully transformed the E.coli strain XL-10-Gold with all our received parts. pUC57-aGal and pUC57-NAGA, containing two of our Enzymes came in a shipping from Genscript.  pRSH10.1-SNAP, containing the SNAP-Tag was a donation from a working group at our own institute. We also recieved pEX-A2-Intein, containing our N-split-intein from Eurofins. We minipreped the constructs and controlled them by restriction. We also transformed XL-10-Gold with pSB1C3- SpyCatcher , which we got from the registry, but got a lawn on the plate. It turned out that the used XL-10-Gold did already have a resistance against chloramphenicol, which is selected for in pSB1C3.  Besides that, we ran tests with several lab devices, including the gel documentation system where we also tested out our DNA ladder and our DNA-Stain-G, which we were not confident in and decided to switch to Midori Green instead.
Week 2 --- 11.08.14 - 15.08.14
In week two and three our 4th semester bachelor students attended a bachelor module, which we specifically organised with professors at the institute to improve our skills at cloning methods. We thought this was a important step, as our team mostly consists of 4th semester Bachelor students with little experience in longer projects.  The transformed constructs pUC57-aGal, pEX-A2-Intein, pRSH10.1-SNAP and pUC57-NAGA from last week were amplified in E.coli and purified, from where we subcloned them into pSB1C3, which we intended as our assembly vector aswell as out vector of choice for registry parts. We also tried to do three-part-assemblies in pSB1C3 with following inserts: SNAP+NAGA, Intein+NAGA, Intein+aGal and SNAP+aGal. Unfortunately we didn’t succeed in the transformation of E.coli BL21 DE3 with our ligated constructs. Possible reasons for that were thought to be improper concentrations of the parts for the ligations. Our last part, EABase-pEX-K4, arrived in the second week and was transformed into XL 10 Gold.
Week 3 --- 18.08.14 - 22.08.14
In the 3rd week we made a batch of competent BL21DE3. We received the BL21DE3 stock from a working group at our institute. Then we retried the ligation of several parts with pSB1C3, namely our three enzymes (NAGA,aGal and EABase) each with Intein (three-part-assemblies), aswell as our enzymes with of the three enzymes with pSB1C3-Spy(two-part-assembly) and all of our tags and enzymes with pSB1C3(two part assembly) except for Spy-Tag, which already was in pSB1C3. All the ligated constructs were transformed into E.coli BL21DE3. Although this time we used higher concentrations of our parts for the ligation, it didn’t work either.  In order for the SNAP-Tag to be used, we had to add the RFC25 prefix and suffix to it by overhang PCR.
Week 4 --- 25.08.14 - 29.08.14
After the  transformations into BL21DE3 failed multiple times we decided to swich to a different strain. We received a sample of competent NEB5α from a group at our institute from which we made a batch of our own competent cells. We tried to ligate our parts with pSB1C3 but this time we added extra ATP to the mix and used a decreasing temperature gradient for ligation. With the new competent NEB5α, we finally succeeded in transforming the ligated constructs pSB1C3-Intein, pSB1C3-NAGA, pSB1C3-Intein-NAGA, pSB1C3-aGal and pSB1C3-Intein-EABase. Furthermore we modified our chosen expression vector (pETBlue-1), by ligating it with annealed oligonucleotides.
Week 5 --- 01.09.14 - 05.09.14
Minipreparation and subsequent control restriction of the following constructs in pSB1C3 confirmed last week’s transformations: aGal, NAGA and EABase each with Spy and SNAP-NAGA, SNAP-EABAse. We also confirmed the modified pETBlue-1, named pETue1.1 by control restriction and transformed NEB5a with it. pETue1.1 was purified and used for ligation with our one-part and two-part-inserts. However, the transformation of the ligation constructs in NEB5α did not work. Apart from that, we aliquoted our pSB1C3-constructs, that had previously been confirmed by digestion and gel electrophoresis, to send them to GATC-biotech for sequencing. Results were in the next day, with confirmation of correct inserts. Although sequencing did not cover the whole insert, one could see that the parts were ligated correctly.
Week 6 --- 08.09.14 - 12.09.14
In the sixth week of lab work we decided to retry ligation into the pETue1.1 expression vector with all available constructs in pSB1C3. Ligations were then transformed into NEB5α for amplification. After miniprep, we checked the plasmid by control digestion and proceeded to retransform BL21DE3 with our constructs. We also managed to successfully construct pSB1C3-SNAP-EABase in NEB5α via three part ligation. It was later confirmed by digestion with EcoRI and PstI. Work on the synthetic peptides, which were to be used with our tags for immobilisation also began this week. The “invariable part”, which was the same for both C-Intein and Spy-Tag was synthesized manually. It contains a cystein residue for immobilisation with the SulfoLink and a Fluorescin labelled lysine residue.
Week 7 --- 15.09.14 - 19.09.14
On the weekend of the 13th and 14th five of our team members attended the iGEM meetup (hier n link)  in Aachen, where we presented our project and interchanged ideas with the other attending teams. An overnight expression test of our BL21DE3 with the pETue1.1[Link zum plasmid] constructs at 20°C did not yield any Protein. The functionality of our expression vector remained unconfirmed, as SDS-page did not show any overexpression bands. We also tried to purify protein with Ni-NTA columns, in case the expression was not very efficient in our cells. However nothing could be purified from the cell lysate. After this failed expression test we started an extensive error screening. Sequencing of pETue1.1 revealed that it in fact contained a deletion in the start codon, thereby inhibiting any expression. Constructs in pSB1C3 were also re-checked by sequencing of the insert from both sides using specifc RFC25-binding primers, aswell as a primers per enzyme, that  was specifically designed to bind inside of the constructs. They turned out correct, except for SNAP-aGAL and SNAP-EABase. Work on the synthetic peptides continued with construction of the C-Intein. This was performed using an Intavis AG ResPep System. The product was purified by HPLC and analysed with ESI. The fractions containing the product were lyophilised.
Week 8 --- 22.09.14 - 26.09.14
We began the 8th week by repicking colonies from the transformation of NEB5α with pETue1.1, since we suspected that we got unlucky before by picking a conoly with the start codon mutation. We managed to get a NEB5α colony with a perfect pETue1.1, which was confirmed by digestion with NsiI and full sequencing of our inserted annealed oligos. We decided to label this vector pETue1.2 in order to avoid cofusion, since we had already worked with the wrong vector (labeled 1.1) for some time. We ligated all available pSB1C3 constructs into the new pETue1.2 and then transformed NEB5α cells with the ligations. After mini-preping we confirmed the constructs and attempted retransformation of the vectors into BL21DE3. However transformation failed and no cultures grew on the plate over night.  Since we also planned on sending in our pETue1.2 expression vector to the registry, we had to remove AgeI cleavage sites from it by mutagenesis PCR. We transformed NEB5α with the product, which we had labelled pETue1.3. The success was later confirmed by a control digestion with AgeI. We also finished up the synthetic peptide by construction of the Spy-Tag, followed by HPLC purification and ESI mass spectrometer analysis. The fractions containing the product were lyophilised. We also did a blog entry on team Heidelbergs iGEM blog , presenting our project.
Week 9 --- 29.09.14 - 03.10.14
Previously we were in contact with Team Oslo, which whom we planned to collaborate. We were to receive PCR amplified parts, which we then wanted to ligate with pSB1C3 to send to the regisrty. Due to toll issues the parts arrived quite late though, leaving us with little time to complete Team Oslos request. Our efforts to do that are detailed here .  Last week’s plates of the failed transformation of Bl21DE3 with our pETue1.2 constructs strangely did show colonies after storage at room temperature over the weekend. We analysed them, in order to find out if it was just a contamination or actual BL21DE3 with pETue1.2-constructs. To our surprise the colonies did contain the correct constructs, as we analysed them by sequencing. We suspect that the plates did contain too much ampicillin, which delayed colony formation. A  test expression of these still did not yield any Protein though, even though the vector was sequenced and correct. Following up on our failed expression attempts we then tried to re construct our pETue in a last effort to try and make the pETue1.2 vector work.  Yet again NEB5α transformation worked, but retransformation with BL21DE3, showed no colonies on the plates. We also were in contact with Team Heidelberg, who had offered us a two way collaboration. We were to analyse one of their proteins by MALDI, to which they did not have access. In return they would provide us with their pSBX1K3 and pSBX4K5 expression vectors, which they had been successfully using.
Week 10 --- 06.10.14 - 10.10.14
As a last effort for our pETue1.2 vector, we tried overexpression again, this time varying IPTG concentrations. Although the cells grew normally no overexpression could be seen on the SDS-page. We also tried to add the fluorescein labelled, synthetic C-Intein to the lysate of Intein-aGAL overexpression, hoping to spot the protein band via fluorescence. Still there were no results. With very little time remaining at this point, we decided to focus our efforts on the vectors from team Heidelberg, which arrived this week. We agreed upon using the high copy plasmid pSBX1K3. Transformation of NEB5α with the vector did work very well, so we could be ready for construction of the final expression test next week. As part of our outreach, on the Friday of this week, we invited 40 pupils of a local high school's 12th grade to a self designed practical course, which was focused on giving them an impression of working in a synthetic biology laboratory. After an introductory seminar in which we introduced synthetic biology as well as discussing iGEM and our project, we divided them into groups of four for supervising and conducted a PCR, restriction digest and agarose gel electrophoresis with them, discussed the findings and explained the methods.
Week 11 --- 13.10.14 - 17.10.14
Since it was the final week a lot of our work was focused on completing the wiki. In the final week we successfully ligated eight out of our nine construct combinations into the pSBX1K3 vector from team Heidelberg. NEB5α transformation worked well and after miniprep all constructs could be confirmed by digestion. Retransformation of BL21DE3 worked out and a final test expression of our construct was conducted. However even with the vector from team Heidelberg no expression could be seen. Even after incubation with our synthetic peptides, which should be able to fluorescently label the product, no fluorescent band could be seen in SDS-page.
In week 11 we also received team Heidelbergs proteins (λ lysozyme, sho1, sgt2) and proceeded to analyse them using mass spectrometry (MALDI) at the Kalbacher lab in Tuebingen. We sent the results to team Heidelberg.
There is still a long way to go with this project, but we intend to continue our efforts to construct and optimize T-ECO in the following year.
Retrieved from " http://2014.igem.org/Team:Tuebingen/Notebook/Journal "
Recent changes
What links here
Related changes
Special pages
My preferences
Printable version
Permanent link
Privacy policy
Disclaimers
