Team:WPI Worcester/Description
Toggle navigation
Home
About
Team
Attributions
Collaboration
Sponsors
Project
Description
Design
Protocols
Results
Notebook
Parts
Parts
Basic Parts
Part Collection
Safety and Practices
Human Practices
Safety
Integrated Practices
Social Media
Engagement
Awards
Silver
Gold
Measurement
InterLab Data
There are a wide variety of diseases that affect humans on a day to day basis. Some of these diseases are caused by infections and some are caused by genetic mutations. When most people think of genetic diseases they typically think of whole missing genes, however, even just a single nucleotide change in the genome can result in devastating and, unfortunately, rather common genetic diseases. Worldwide, 1 in 7,000 males are affected by muscular dystrophy, 1 in 12 people are carriers for Cystic Fibrosis, and 1 in every 365 African Americans suffers from Sickle Cell Anemia. Each of these diseases can arise from a single nucleotide change.
While these diseases have devastating side effects, there are current genome treatments available. The most common treatment for genetic diseases is through the use of CRISPR technology. This technology works by utilizing the enzyme Cas9 to target a specific sequence in the genomic DNA and cut it. After this cut has been made, the DNA recognizes that something wrong has occurred and begins the repair process. Through this process, scientists are able to insert or delete parts of the cut DNA to, hopefully, repair the mutation in the genome.
In the last few years the CRISPR/Cas9 system has revolutionized what is possible in genome editing. Using this system any area of the genome can be targeted and edited.
However, CRISPR technology is not perfect due to many factors. One issue is that it relies on non-homologous recombination. Errors often occur when the DNA strands join back together because nucleotides can be inserted or deleted into the DNA. Another issue that is these edits occur on an organism's genome. Because of this the edits made using CRISPR are permanent, whether they are correct or not.
In May of 2016, a paper was published that investigated the ability to perform single base nucleotide edits in double stranded DNA without breaking any strands. The researchers did this with the editing enzyme APOBEC1, which performs Cytosine (C) to Uracil (U) edits. Uracil is not typically found in DNA, so an enzyme, Uracil DNA glycosylase, exists to remove it. In this case it would remove the U, identify the Guanine (G), and replace the U with a C, which will base pair with G.
Although the researchers were able to add an inhibitor to their system to resist this enzyme there were still some issues with this approach. There was only a 50% chance of their edited strand being chosen as the template in cellular mismatch repair and the desired edit occurring. Also, the researchers could only perform a C to U edit, so the capabilities were limited.
Based off of this, we started to think about editing the mRNA. But is targeting the RNA with CRISPR/Cas9 possible?
Targeting RNA
A paper published in 2014 showed that it is possible to target RNA with CRISPR through the use of a PAMmer. This is a short oligonucleotide that is separate from Cas9 but helps to guide the system there and activates nuclease to performs the cleavage. Through the use of a PAMmer, dCas9 was also shown to be able to target mRNA.
Advantages
There are many advantages to targeting mRNA. If any mistakes happen, they will not be permanent because the DNA is not being edited. The system will be regulated with doxycycline, and we can double the possible number of edits. Not only will we be able to use APOBEC1 and perform C to U edits, but we will also be able to use the editing enzyme ADAR, which performs A to I edits exclusively in RNA. Inosine (I) is a nucleotide that is common in brain pathways and is read by the ribosome as a G. By combining all of these concepts together, we developed a system to target single nucleotide changes in RNA. You can read more about the specifics of our system in our Design .
References
O'Connell, M.R., et al.(2014).Programmable RNA recognition and cleavage by CRISPR/Cas9.Nature.516:263-266.
Komor, A.C., et al.(2016).Programmable editing of a target base in genomic DNA without double-stranded DNA cleavage. Nature533:420-424.
Nelles DA et al. (2016).Programmable RNA Tracking in Live Cells with CRISPR/Cas9.Cell.165:488-496.
