Team:WPI Worcester/Design
Toggle navigation
Home
About
Team
Attributions
Collaboration
Sponsors
Project
Description
Design
Protocols
Results
Notebook
Parts
Parts
Basic Parts
Part Collection
Safety and Practices
Human Practices
Safety
Integrated Practices
Social Media
Engagement
Awards
Silver
Gold
Measurement
InterLab Data
With proper motivation and background for using the CRISPR/dCas9 and the editing enzymes APOBEC or ADAR to target and edit mRNA, designing effective plasmids and reporters was the first step in ensuring that we would be able to accurately show this proof of concept.
Reporters
We designed 2 reporters, one for each editing enzyme. Each reporter is designed to emulate certain point mutations that can occur and cause diseases. The APOBEC reporter imitates a mutation to a start codon and the ADAR reporter imitates a mutation that creates a premature termination codon (PTC). Both reporters used a green fluorescent protein (GFP) as the editing identifier. When expressed in the cell without presence of the editing complexes, the reporters should not produce GFP. Thus, the cells would not fluoresce. Once the editing complex binds and edits the reporters, however, GFP should then be translated and the cells will glow green.
APOBEC Reporter
For our APOBEC experiments we used the following reporter. The reporter is a GFP sequence with a 5’ untranslated region (UTR). The 5' UTR is a portion of a GAPDH sequence for the CRISPR/dCas9 to bind to. The only change we made to the GFP sequence was mutating the start codon (ATG) to an ACG codon. The reason for this being that when the ribosome would bind to the reporter’s mRNA, translation should not occur, since there is no start codon. Therefore, we should not see any fluorescence in the transfected cells.
Once the CRISPR/dCas9 and APOBEC fusion is added to the cells the CRISPR/dCas9 will be able to bind to the 5’ UTR of the reporter. APOBEC will then make its C to U edit on the ACG codon, making it an AUG codon, which is a start codon in mRNA. Now when the ribosome binds, the GFP should be translated and the cells will fluoresce.
ADAR Reporter
For our ADAR experiments we designed a slightly more complex reporter. The reporter is a beta globin sequence with a GFP sequence after. The reason for using beta globin is because it has an intron. When this intron is spliced out during transcription it will leave an exon junction. This is important because we put a PTC (UAG codon) before the exon junction. When the ribosome starts to translate the protein it will see that there is an exon junction after a stop codon. The ribosome recognizes that this should not occur and will mark the mRNA for degradation. The mRNA will be degraded and no protein will be translated. Resulting in no fluorescent cells.
Once the CRISPR/dCas9 and ADAR fusion is added to the cells the CRISPR/dCas9 will be able to bind to the beta globin before the PTC. ADAR will make its A to I edit on the PTC, changing the UAG codon to a UIG codon. I, inosine, is read as a G by the ribosome, so the new UGG sequence would produce a tryptophan instead of being a PTC. Without the PTC the ribosome should be able to translate the rest of the protein, including the GFP. With the GFP translated the cells should now fluoresce.
Plasmids
To accomplish our experiment we designed three different plasmids. The reason that we broke our system into three plasmids is because of the size of the components. The coding region for dCas9 on its own is about 4,000 base pairs. The components of each plasmid were cloned in E. coli cells and then miniprepped. All three plasmids were then transfected into our chassis, HEK293T cells.
dCas9 Plasmid
This is our first plasmid. It contains one of the editing enzymes, either ADAR1, ADAR2, or APOBEC1, fused to the N terminus of dCas9. The enzyme and dCas9 have 1-3 repeats of an XTEN linker to allow the enzyme to edit a certain distance away from dCas9. Downstream on the same plasmid is the rtTA gene and promoter.
One of the most common suggestions we received when conducting our integrated human practices was to design a tunable system. We don't want the enzymes to be editing at all time, especially if something was to go wrong. Regulation is the purpose of the rtTA gene in this plasmid. rtTA, which stands for reverse tetracycline trans activator, is constitutively expressed. It acts as a repressor binding to the tetracycline responsive element (TRE) promoter and preventing transcription of the editing enzyme and dCas9. However, when doxycycline is added to the system it will bind to the rtTA, removing it from the promoter and allowing transcription. Editing should not occur unless we introduce doxycycline into the cells.
mCherry and Guide RNA Plasmid
This plasmid contains our guide RNA and mCherry. The guide RNA is complementary to the target editing site and will help direct the dCas9 there. mCherry is a type of RFP and is constitutively expressed. This is our transfection marker and shows that the plasmids have successfully entered the cells.
Reporter Plasmid
Our GFP reporters will be constitutively expressed. They will be used as controls and to show successful editing.
