<HTML xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en" dir="ltr">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki  ltr ns-0 ns-subject page-Team_Osaka_SIGNAL"><DIV id="globalWrapper"><DIV id="top-section"><DIV id="p-logo"><A href="/Main_Page" title="Main Page">&quot;
	    </A></DIV><DIV id="menubar" class="left-menu"><UL><LI class="selected"><A href="/Team:Osaka/SIGNAL">Page               </A></LI><LI class="new"><A href="/wiki/index.php?title=Talk:Team:Osaka/SIGNAL&amp;action=edit&amp;redlink=1">Discussion               </A></LI><LI><A href="/wiki/index.php?title=Team:Osaka/SIGNAL&amp;action=edit">View source               </A></LI><LI><A href="/wiki/index.php?title=Team:Osaka/SIGNAL&amp;action=history">History               </A></LI><LI style="color:#808080;cursor:default">teams</LI></UL></DIV><DIV class="right-menu" id="menubar"><UL><LI id="pt-login"><A href="/wiki/index.php?title=Special:UserLogin&amp;returnto=Team:Osaka/SIGNAL" title="You are encouraged to log in; however, it is not mandatory [o]" accesskey="o">Log in</A></LI></UL></DIV><DIV id="search-controls"><FORM action="/Special:Search" id="searchform"> </FORM></DIV></DIV><DIV id="content"><H1 class="firstHeading">Team:Osaka/SIGNAL</H1><DIV id="bodyContent"><H3 id="siteSub">From 2009.igem.org</H3><P><TITLE>Home of iGEMOSAKA wiki</TITLE></P><DIV id="contents"><DIV id="right_contents" style="background-color: #FEF2EC"><H1 style="text-align: left">SIGNAL</H1><DIV id="CollapsiblePanel1" class="CollapsiblePanel"><DIV class="CollapsiblePanelTab" tabindex="0"><H2>Overview</H2></DIV><DIV class="CollapsiblePanelContent"><P>Bacteria expressing genes that code for color will form interesting patterns when spotted onto agar plates in appropriate locations and allowed to spread out and intermingle. But what if we could increase the complexity of the patterns formed by implementing intercellular signaling between different groups of bacteria?</P><P>We decided to use parts from the natural quorum-sensing mechanisms of various bacteria to implement our intracellular communication system. If it works, we can for example cause two colonies of bacteria to change color or stop moving as they approach each other, hopefully resulting in interesting patterns.</P><P>A brief overview of quorum-sensing: Bacteria, such as V. fisheri, coordinate their gene expression through a system in which each bacterial cell produces a limited amount of signaling molecules, called AHL, which diffuse through the medium and reach other bacteria in its vicinity. AHL molecules bind to receptor proteins which in turn bind to specific promoters that then up-regulate downstream transcription activity. When the bacteria reach a certain density, the amount of AHL in the environment (and thus in the cells) will be sufficient to trigger this increase in promoter activity, and the genes downstream of the promoter will be ‘switched’ on. Below is a schematic diagram depicting quorum-sensing. (Taken from Utsunomiya University).</P><P>Currently we are working on 2 distinct groups of parts: 'Senders' and 'Receivers'. 'Senders' code for enzymes that produce AHL signal molecules, which diffuse out of the cell, through the culture medium and into the receiving cell, where a receptor proteins encoded by the 'Receivers' bind the signals, forming a complex which in turn can bind to and up-regulate transcription from their specific promoter.</P></DIV></DIV><DIV id="CollapsiblePanel2" class="CollapsiblePanel"><DIV class="CollapsiblePanelTab" tabindex="0"><H2>Parts &amp; Devices</H2></DIV><DIV class="CollapsiblePanelContent"><P>We assembled the following devices using parts from the iGEM 2009 Spring Distribution:</P><H3>Senders:</H3>
Lux Sender - produces 3OC6HSL, includes a double terminator for easy insertion in front of or behind any other device/part<A href="http://partsregistry.org/Part:BBa_K204047">Las Sender</A> - same as above, but produces 3O-C12-HSL<A href="http://partsregistry.org/Part:BBa_K204040">Rhl Sender</A> - produces C4-HSL<A href="http://partsregistry.org/Part:BBa_K204025">Cin Sender</A> - produces 3OH-C14:1-HSL<H3>Receivers:</H3><A href="http://partsregistry.org/Part:BBa_K204051">Lux Receiver</A> - receives signal from Lux Sender transmitted in the form of (Lux-signaling system AHL), which then activates/upregulates transcription downstream of this device
Las Receiver - receives signal from Las Sender<A href="http://partsregistry.org/Part:BBa_K204031">Rhl Receiver</A> - receives signal from Rhl Sender<A href="http://partsregistry.org/Part:BBa_K204032">Cin Receiver</A> - receives signal from Cin Sender<H3>Test Constructs:</H3>
Lux Receiver with GFP attached downstream (&quot;<A href="http://partsregistry.org/Part:BBa_K204700">X1</A>&quot;) - a <A href="http://partsregistry.org/Part:BBa_E0240">GFP coding device</A> has been attached downstream of the Lux Receiver described above
Las Receiver with GFP attached downstream (&quot;X2&quot;)
Rhl Receiver with GFP attached downstream (&quot;<A href="http://partsregistry.org/Part:BBa_K204702">X3</A>&quot;)
Cin Receiver with GFP attached downstream (&quot;X4&quot;)<P>Of these parts, the receivers are thought to be the most generally useful as any protein coding region can theoretically be attached downstream of them to be triggered upon induction with the appropriate signal sender (or alternatively by addition of the corresponding AHL). Therefore, we tested those parts extensively, using both PCR sequencing to confirm their nucleotide sequences and fluorimetry measurements to check that the test constructs were functioning as planned.</P><P>Unfortunately, we did not have time to properly integrate our sensor modules with the color or motility modules to build the final devices that we initially devised. In future years hopefully we will be able to augment our sensors with further improvements as described under the 'Future Work' section below.</P></DIV></DIV><DIV id="CollapsiblePanel4" class="CollapsiblePanel"><DIV class="CollapsiblePanelTab" tabindex="0"><H2>Tests</H2></DIV><DIV class="CollapsiblePanelContent"><P>To test the effectiveness of our signaling system, we attached a <A href="http://partsregistry.org/Part:BBa_E0240">GFP coding unit</A> (rbs + protein coding region + terminator) behind each Receiver's AHL-activated promoter. (See ‘Test Constructs’ under Parts &amp; Devices above), which theoretically enables us to measure the amount of transcription upregulation that occurs in the presence of AHL.</P><P>We then tested our parts in two ways. First we tested the Receivers indirectly by adding chemically-derived AHL into culture solutions of the Test Constructs and looking for GFP fluorescence, which will indicate that transcription has been activated. Following that, we tested the Senders by using the above Receivers. We hoped to determine the amount of AHL produced by the Senders by comparing transcription activity induced by the Senders in relation to AHL-induced transcription activity. However this year we only managed to qualitatively determine that our Senders were functioning.</P><P>Details of the tests are given below. For detailed descriptions of the protocols followed during these tests, please refer to the “Protocols” section on this page.</P><H3>Sensor Experiment 1: Receivers vs AHL</H3><H4>Procedure:</H4>
1. Pre-culture cells transformed with parts to be tested in solution medium overnight.
2. Inoculate fresh medium with pre-cultured cells and culture till full growth.
3. Replace culture medium by ultracentrifuging and resuspending cells in fresh medium.
4. Add AHL to culture.
5. Measure OD and fluorescence at regular intervals.<H4>Results &amp; Discussion:</H4><P>A surprising lack of correspondence between AHL concentration and GFP fluorescence is noted. Simply adding chemically-derived AHL to culture solution may not be able to induce the receivers' promoter activities as a continual supply of AHL may need to be produced in order to offset the degradation and/or cellular metabolism of AHL molecules. The gradual and almost uniform increase in fluorescence for ALL samples may be simply due to increasing cell concentration which failed to be detected by OD600 measurements as at values of over 1.0 the linear relationship of OD with concentration is almost certainly lost.</P><H4>Comments:</H4><P>It was realized sometime after starting this experiment that in order to measure efficiency of promoter activation, the rate of change of GFP fluorescence would have been more useful than the absolute value of GFP fluorescence itself. Also, promoter activity might peak somewhere between the addition of AHL and the first subsequent measurement as response times of &lt;10min has been reported for similar devices. The solution would have been to take measurements on smaller time intervals, eg. every minute, and calculate rate of change of fluorescence from the measurements. Unfortunately we were not able to access instruments capable of automatically measuring fluorescence at smaller time intervals.</P><H3>Sensor Experiment 2: Senders vs Receivers</H3><H4>Procedure:</H4>
1. Pre-culture cells transformed with parts to be tested in solution medium overnight.
2. Pipette fresh medium into microcentrifuge tubes then inoculate with cells from overnight culture.
3. Culture overnight at 37 degrees Celsius in incubator.
4. Measure fluorescence.<H4>Results &amp; Discussion:</H4><P>The results were rather disappointing as it appeared that only Lux Receiver I (which was a part we borrowed from Chiba University) showed any appreciable fluorescence when mixed and cultured with the senders. The parts we constructed ourselves - Lux Receiver II, Rhl Receiver and Cin Receiver, did not show much response to the presence of senders, even though it can be argued that at least their fluorescence was marginally above that of the negative control (receiver-only culture).</P><P>The reason for the huge difference in fluorescence between our parts and the one received from Chiba University may be attributed to a faulty GFP coding device being attached behind the promoters, as it can be seen that even in the absence of a sender the Lux receiver should show leaky promotion and create a 'baseline' fluorescence which is absent from our own parts. A PCR sequence check has been done on the GFP coding device but results are still pending.</P><P>On the other hand, we were able to determine that our senders were working as co-culture with all senders resulted in noticeable increases in fluorescence for Lux Receiver I. The relatively high activation due to the Rhl sender-Lux Receiver I pairing may be attributed to cross-talk.</P><H4>Comments:</H4><P>3 samples were prepared for each set of conditions (sender-receiver pairing or AHL concentration-receiver pairing). This was done in order to obtain averages for more accurate results. However it was observed that the 3 measurement values for each set of condition often showed great deviation from each other. This can be attributed to both human error and instrumental error, both of which were likely amplified by the small volume of sample used.
Therefore, this experiment is not recommended for quantitative characterization of sensor parts. It does serve, to a limited extent, as a qualitative evaluation of which pairings work and which don't, which enabled us to determine which of our devices were actually able to function as planned.</P></DIV></DIV><DIV id="CollapsiblePanel9" class="CollapsiblePanel"><DIV class="CollapsiblePanelTab" tabindex="0"><H2>Methods</H2></DIV><DIV class="CollapsiblePanelContent"><H3>Sensor Experiment 1: Receivers vs AHL</H3><H4>Day 1 - Preparation</H4><P>1. From transformation plate, pick up a small quantity of cells from a colony using sterilized toothpick and streak on LA plate with the appropriate antibiotic resistance. Incubate at 37 degrees Celsius for 8+ hr.</P><P>2. From the streak plate, choose a single-cell colony and pick up cells using sterilized toothpick, then inoculate 5ml LB culture medium to which the appropriate antibiotic resistance (50mg/l) has been added. Incubate overnight at 37 degrees Celsius in a shaking incubator.</P><H4>Day 2 - Measurement</H4><P>1. Transfer 1ml of the overnight culture into 30ml fresh LB culture medium. Incubate in shaking incubator at 37 degrees.</P><P>2. Measure OD of culture every hour or so, until OD reaches fixed value (culture at full growth).</P><P>3. Transfer culture into 50ml ultracentrifuge tube and spin at 8000rpm for 5 minutes.</P><P>4. Resuspend pellet with 30ml fresh LB culture.</P><P>5. Incubate culture in shaking incubator at 30 degree Celcius for 10 minutes.</P><P>6. Transfer 1ml of culture into microcentrifuge tube and put on ice. Immediately add AHL to rest of culture to attain desired concentration.
(Note: Between the steps detailed below, samples should be kept on ice to minimize cellular activity.)</P><P>7. Obtain OD600 of 2x diluted sample by transferring 500ul of sample into cuvette, adding 500ul MiliQ water and measuring the resulting OD.</P><P>8. Ultracentrifuge remaining 500ul of sample at 13000rpm for 2min, discard supernatant, then resuspend pellet with 1ml MiliQ water.</P><P>9. Ultracentrifuge sample again at 13000rpm for 2min, discard supernatant and resuspend with precisely 1ml MiliQ water (this results in 2x dilution).</P><P>10. Measure GFP fluorescence with a fluorimeter at 488nm excitation, 513nm emission.</P><P>11. Every hour, sample 1ml of culture and repeat steps 7-10 until fluorescence reaches steady state.</P><H3>Sensor Experiment 2: Senders vs Receivers</H3><H4>Day 1 - Preparation</H4><P>1. From transformation plate, pick up a small quantity of cells from a colony using sterilized toothpick and streak on LA plate with the appropriate antibiotic resistance. Incubate at 37 degrees Celsius for 8+ hr.</P><P>2. From the streak plate, choose a single-cell colony and pick up cells using sterilized toothpick, then inoculate 5ml LB culture medium to which the appropriate antibiotic resistance (50mg/l) has been added. Incubate overnight at 37 degrees Celsius in a shaking incubator.</P><H4>Day 2 - Mix &amp; Culture</H4><P>1. Prepare microcentrifuge tubes (3 samples for each sender-receiver pair). Add 400ul of LB-Amp culture medium to each microcentrifuge tube.</P><P>2. Transfer 1ml of each overnight culture to separate microcentrifuge tubes and ultracentrifuge at 12000rpm for 1min. Discard supernatant and resuspend cells with 1ml LB-Amp. Repeat with remaining overnight culture if necessary to obtain the required amount of cell culture.</P><P>3. For sender-receiver test: Add 50ul each of sender and receiver cultures to a microcentrifuge tube prepared in step 1. For the negative control (receiver only), add 100ul of receiver culture only.</P><P>4. Incubate microcentrifuge tubes at 37 degrees Celsius overnight.</P><H4>Day 3 - Observation &amp; Measurement</H4><P>1. Wash cells by ultracentrifuging at 13000rpm for 2min, discarding supernatant and resuspending cells with MiliQ water. Repeat, and resuspend with precisely 1ml MiliQ water.</P><P>2. For direct observation, bring cells suspended in water to darkened room and shine with UV Black Light. GFP fluorescence should be observable if sufficiently strong promoter activation has been achieved.</P><P>3. Measure GFP fluorescence with a fluorimeter at 488nm excitation, 513nm emission.</P></DIV></DIV><DIV id="CollapsiblePanel5" class="CollapsiblePanel"><DIV class="CollapsiblePanelTab" tabindex="0"><H2>Model</H2></DIV><DIV class="CollapsiblePanelContent"><P>To make an effective simulation program we regarded the cell's movement as a diffusion. Therefore we applied the fick's second law of diffusion in modeling the cells movement along with the diffusion of autoinducers. Since the law of diffusion is a differential equation(strictly speaking a partial differential equation) the need for a numerical solution was inevitable. We used the finite difference method. We then applied it and converted it which gave us the equation (1) and (2), each representing the diffusion of the cell(or colony) and the autoinducer. These two equations were the bases of our program.
<CENTER> ...(1)
 ...(2)</CENTER></P><H3>Finite difference method-Expicit method</H3><P>As mentioned above we used the finite difference method (and also the explicit method). Finite difference methods are widely used numerical methods in solving differential equations, by considering the differential equations as a finite difference equations. We also used the explicit method which calculates the future state of a system by using the current state of the system. </P><P>Fick's second law of diffusion can be written as
<CENTER>・・・(3)</CENTER>
where C[normalized amount/μm<SUP>3</SUP>] is the concentration, t[s] is the time, x[μm] is the position, and D is the diffusion coefficient in dimensions of [μm<SUP>2</SUP>/ｓ].(Normalized　amount is an amount of cell divided by maximum.)</P><P>If we apply the finite difference method to the above equation, fick's second law of diffusion can be expressed in a finite difference equation written as
<CENTER>・・・(4)</CENTER>
where the concentration[C] of a substance at a time[t], and at a position [i,j] is represented as C<SUP>t</SUP><SUB>i,j</SUB>.</P><P>In this experiment, for convenience we let Δt=h and Δx=1 but for a more accurate result Δt and Δx can be adjusted.</P><P>By applying the explicit method to the above equation again we reach the following equation.
<CENTER>・・・(5)</CENTER>
Since the concentration of the substance can not be lower than zero, every term of the equation must be over zero. So we reach the following condition.
<CENTER>・・・(6)</CENTER></P><H3>Determination of the values of parameters</H3><P>For accurate results, the precise determination of the values of parameters used in the simulation was essential. We had to determine the values of the two diffusion coefficients(the cell and the autoinducer) along with the production rate of the autoinducers and the growth rate of the colony itself.</P><P>The growth rate of the colony was measured by experiment which took place in our lab[fig_model.1]. As a result the value of the colony's logarithmic　growth rate μ is 0.0024[s<SUP>-1</SUP>]. 
</P><H4>fig_model.1 growth curve</H4>
The diffusion coefficient of the cell is, by definition,
<CENTER>・・・(7)</CENTER>
where v<SUB>cell</SUB>, the average speed of a cell, is 0.02 mm/h(=20 μm/s) and T, the average random walk time, is 1s. So in conclusion the diffusion coefficient of the cell is 300 μm<SUP>2</SUP>/s <A href="#1">[2]</A>.<P>The diffusion coefficient of serine is known as 1000 μm<SUP>2</SUP>/sec. Since the diffusion coefficient is inversely proportional to the square root of the molecular weight by simple computation we were able to figure out the diffusion coefficients of the autoinducers we used. The autoinducer's diffusion coefficients is as follows<A href="#1">[1]</A>.</P><P text-indent="2em">D<SUB>C<SUB>4</SUB>HSL</SUB> = 784 μm<SUP>2</SUP>/s</P><P>D<SUB>3OC<SUB>6</SUB>HSL</SUB> = 702 μm<SUP>2</SUP>/s</P><P>D<SUB>3OC<SUB>12</SUB>HSL</SUB> = 607 μm<SUP>2</SUP>/s</P><H3>The production of autoinducers and the growth of E.coli</H3><P>Since the E.coli used in this experiment has itself an ability to produce autoinducers, a term which considers the cell's production of autoinducers must be contained in the equation. The added equation is shown as it follows. 
<CENTER>・・・(8)</CENTER>
where δ is the production rate of the individual cell. Because the autoinducer's amount, produced by the colony increases in proportion to the numbers of individual E.colis, a term of multiplication to the cell's density was added. </P><P>The growth rate of the colony is slightly more complex. Since the nutrient of the medium is limited, E.coli's density is finite. So we expressed the colony's grwoth rate by an sigmoid curve.
<CENTER>・・・(9)</CENTER></P></DIV></DIV><DIV id="CollapsiblePanel6" class="CollapsiblePanel"><DIV class="CollapsiblePanelTab" tabindex="0"><H2>Results of simulation</H2></DIV><DIV class="CollapsiblePanelContent"><H3>Triangle model</H3><P>This triangle model is one of the model we worked on. In this triangle model, three different cells send signal which inhibit the motility of other cells in a cyclic way. Fig_model.2 indicates this model. Red cells inhibit green cells. Green cells inhibit blue cells. And so on.
</P><H4>fig_model.2</H4><H3>Result of triangle model</H3><CENTER>         </CENTER><P>Fig_model.3 and fig_model.4 are simulation results indicating a pattern which cells spread on a petri dish surface. Fig_model.3 indicates our triangle regulation model. Fig_model.4 is the result which cell movements aren't regulated.</P><P>Please compare fig_model.3 with fig_model.4. There is slight difference. In triangle regulation model, the cells stops swimming when receive individual signal. Subsequently, cells advance to cells stopped by signal which they send. So, the pattern is like fig_model.3. In no regulation model, the border of different color cells is accurately straight line. This simulation result corresponds with experiments. (<A href="
https://2009.igem.org/Team:Osaka/WORKS">see works.</A>)</P><H3>3-D movie of three colonies</H3><P>We made the movie captured dynamic change of individual colony.</P></DIV></DIV><DIV id="CollapsiblePanel7" class="CollapsiblePanel"><DIV class="CollapsiblePanelTab" tabindex="0"><H2>Discussion</H2></DIV><DIV class="CollapsiblePanelContent"><H3>Adequacy of simulation and experiment</H3><P>We wanted to get the information we cannot directly know from some experiments. How much length is needed to switch on the transcription and stop cells movement? Oppositely, if we are able to know the length, can we decide the value of some parameter such as AHL production rate? We tried to answer the question using computer simulation. But we haven’t had enough experimental results yet. Though we use some value of parameter on papers, we could not correctly decide other parameters. Simulation is just ideal. We find the fact from experiment. But simulation sometimes becomes powerful tool for helping access the fact. If we were able to make program to fit the experimental results, this approach is significant.</P></DIV></DIV><DIV id="CollapsiblePanel3" class="CollapsiblePanel"><DIV class="CollapsiblePanelTab" tabindex="0"><H2>Future Work</H2></DIV><DIV class="CollapsiblePanelContent"><H3>Sensor Function on Agar Plate</H3><P>Once both Sender and Receiver function has been confirmed and characterized, we will attempt to characterize the function of the whole system in a way that relates to our intended usage. We will spot two colonies, one of Senders and one of Receivers, on a soft agar plate and determine the maximum distance that the Senders can successfully activate the Receivers. Of course, this will only work if the AHL molecules can effectively diffuse through agar medium, which is yet another thing that we have to find out.</P><H3>Amplification</H3><P>There is a possibility that the upregulation of transcription downstream of Receivers will be too weak, perhaps due to insufficient AHL produced by the Senders or poor diffusion of AHL through the soft agar medium. Therefore, we shall go back to the roots of natural quorum-sensing systems by introducing a positive feedback loop, in which weak detection of AHL will lead to transcription of genes that either enable production of even more AHL, leading to an escalating increase in AHL concentration, or more AHL receptors, leading to higher AHL sensitivity.</P><H3>Quenching</H3><P>There is also a possibility that the signals received may be too strong and render the system useless for producing color gradation. For example if the signal diffuses too efficiently the receiving cells may all uniformly change into the secondary color instead of showing a gradual change in color at the edges near the senders.</P><P>Therefore, we may consider implementing a signal-quenching system, perhaps using the aiiA system which involves the AHL-degrading aiiA enzyme. The aiiA gene can be made to express in limited quantity, such that the amount of intracellular AHL will be reduced but not completely erased. This causes only the receivers closer to senders, which receive a higher amount of AHL by diffusion, to be activated while the receivers further away receive too little AHL to pass the quenching threshold and are thus kept inactivated.</P><P>Extensive quantitative characterization of parts will be required to determine which of these future work ideas should be pursued and to what degree. As we have unfortunately ran out of time this year, these ideas will have to wait till the next iGEM Competition.</P></DIV></DIV><DIV id="CollapsiblePanel8" class="CollapsiblePanel"><DIV class="CollapsiblePanelTab" tabindex="0"><H2>Reference</H2></DIV><DIV class="CollapsiblePanelContent"><P class="ref"><A name="1" id="1" class="internal">[1]H.C.Berg, D.A.Brown, Chemtaxis in Esherichia coli analysed by three-dimensional traking, Nature 239 (5374) (1972)
500–504.
</A></P><P class="ref"><A name="2" id="2" class="internal">[2]H.C.Berg, Random Walks in Biology, Princeton University Press, 1983.
</A></P><P class="ref"><A name="3" id="3" class="internal">[3]W.CLAIBORNE FUQUA,Quorum Sensing in Bacteria:the LuxR-LuxI Family of Cell Density-Responsive Transcriptional Regulators, Journal of Bacteriology,Jan 1994
</A></P><P class="ref"><A name="4" id="4" class="internal">[4]Melissa　B.Miller and Bonnie L.Bassler,QUORUM SENSING IN BACTERIA,Annu.Rev.Microbiol.2001
</A></P></DIV></DIV></DIV></DIV><DIV class="printfooter">
Retrieved from &quot;<A href="http://2009.igem.org/Team:Osaka/SIGNAL">http://2009.igem.org/Team:Osaka/SIGNAL</A>&quot;</DIV></DIV></DIV><DIV id="footer-box"><DIV id="footer"><UL id="f-list"><LI id="t-recentchanges"><A href="/Special:RecentChanges" title="Recent changes">Recent changes</A></LI><LI id="t-whatlinkshere"><A href="/Special:WhatLinksHere/Team:Osaka/SIGNAL" title="List of all wiki pages that link here [j]" accesskey="j">What links here</A></LI><LI id="t-recentchangeslinked"><A href="/Special:RecentChangesLinked/Team:Osaka/SIGNAL" title="Recent changes in pages linked from this page [k]" accesskey="k">Related changes</A></LI><LI id="t-specialpages"><A href="/Special:SpecialPages" title="List of all special pages [q]" accesskey="q">Special pages</A></LI><LI><A href="/Special:Preferences">My preferences</A></LI></UL></DIV><DIV id="footer"><UL id="f-list"><LI id="t-print"><A href="/wiki/index.php?title=Team:Osaka/SIGNAL&amp;printable=yes" title="Printable version of this page [p]" accesskey="p">Printable version</A></LI><LI id="t-permalink"><A href="/wiki/index.php?title=Team:Osaka/SIGNAL&amp;oldid=169368" title="Permanent link to this revision of the page">Permanent link</A></LI><LI id="privacy"><A href="/2009.igem.org:Privacy_policy" title="2009.igem.org:Privacy policy">Privacy policy</A></LI><LI id="disclaimer"><A href="/2009.igem.org:General_disclaimer" title="2009.igem.org:General disclaimer">Disclaimers</A></LI></UL></DIV></DIV></DIV></BODY></HTML>