Team:Goettingen/Experiments
Toggle navigation
Home
Team
Team
Collaborations
Project
Description
Experiments
Proof of Concept
Results
Notebook
Parts
Parts
Basic Parts
Safety
Attributions
Human Practices
Human Practices
HP on the Road
HP on the Web
Engagement
Experiments
Overview of Experiments
1. Production Strains
Following strains were cultivated:
Escherichia coli DH5alpha: DSMZ
Raoultella planticola DSM 3069 (ATCC 33531): DSMZ
Shimwelia blattae DSM 4481 (ATCC 29907) : DSMZ
In order to ensure that the bacteria we are working with are really the ones we want to work with, a 16S rDNA PCR was performed.
As an antibiotic resistance will be used in our later experiments to select transformants, we checked for any native antibiotic resistance against ampicillin, kanamycin, tetracyclin and chloramphenicol in an LB agar plate growth test. It turned out that kanamycin was the only antibiotic, against which none of our strains showed any resistance. For this reason, we decided to use only vectors with kanamycin resistance for our later projects.
After this, we prepared electro competent cells from our strains. All of our strains were grown on LB medium.
2. BioBricks
Our BioBricks were designed using the software SnapGene. The original Genes for our B12 binding proteins, and the torA signal sequence were codon optimized for E. coli.
The DNA containing the Genes were synthesized and friendly provided by IDT Integrated DNA Technologies.
In case of torA-mutB, the length of the gene exceeded the maximum length of 2000 bp from IDT. For this reason, we designed two DNA parts, with a natural occurring HindIII restriction site at both ends. We intended to fuse these both genes by ligation. However, after having diffuculties to fuse the genes by ligation, we decided to fuse them by fusion PCR. In order to multiply the DNA for our cloning experiments, and to equip our genes with restriction sites, PCR was used.
For our purpose, we were searching for a vector with (a) kanamycin resistance and (b) araBAD operon. We thus decided to use the pBAD202 expression system. This vector, however, is equipped with a thioredoxin fusion protein which increases the rate of expression, but prohibits the physiological function of our protein. For this reason, we designed primers for an autarkic Synporter protein. In order to test the dependence of the expression system on the B12 export rate, we also decided to use different other vectors.
For vector ligation, we used a standart ligase reaction, for the pBAD202 expression, which works without ligation but topoisomerase reaction, we followed the manufacturer's protocol.
Fig. 1:Our Synporter genes torA-btuF and torA-glmS cloned into the pBAD202 plasmid, separated from the thioredoxin fusion protein.
3. Transformation and Cultivation
For our transformation experiments, we used electroporation for all our self prepared competent cells. For the heat competent E. coli Top10 cells provided with the pBAD202 expression system, we used the manufacturer's heat shock transformation protocol.
For vector multiplication, E. coli cells (Top10 for pBAD202 plasmids, DH5alpha for all other plasmids) were transformed with the constructed plasmids. After the transformation, the cells were cultivated on LB Medium. Transformants were selected, and the respective clones were picked, cultivated in LB medium, and plasmids were extracted.
The Plasmids were then introduced into the production strains Raoultella planticola ATCC 33531 and Shimwellia blattae ATCC 29907.
For the expression experiments, the all cells were cultivated and the expression induced in RM medium under aerobic conditions. As S. blattae and R. planticola are known to produce Vitamin B12 only under anaerobic conditions, the expression was induced under anerobic conditions in MM medium.
The induction was performed using different concentrations of l-arabinose and different induction time spans.
4. Expression Test
To test for the expression of our Synporter protein, the cell lysates of the expression cultures we submitted to an SDS-PAGE. From the SDS Gels, Western Blots were performed using anti-Poly-Histidine-tag antibodies.
In order to test the translocation of our Synporter proteins, the cells were fractionated into Periplasm, Spheroplasts, and Cytoplsma (gained by French Press cell disruption) as a control.
5. Vitamin B12 Assays
To test the physiological function of our Synporter protein, different assays for B12 detection were performed. For the assays, the periplasm and cytoplasm fractions of the cells were examined.
The periplasm fraction of the producing cells was gained by centrifugation.
5.1. Microbial Assay
The microbial B12 assay relies on the growth diameter of the E. coli DSM4261 strain on an agar plate, in dependence of the Vitamin B12 (or methionine) concentration in the medium.
For this purpose, agar plates with synthetic Guttman medium (free of B12 and amino acids, 1.5% agar-agar) were prepared. The E. coli detection cells were grown in liquid synthetic Guttman medium supplemented with selected amino acids (no methionine) and Vitmamin B12. Afterwards, the cells were washed with PBS buffer three times, in order to make the cells virtually free of Vitamin B12. The cells were resuspended in PBS buffer, and mixed with warm, liquid Guttman medium (free of B12 and amino acids, 0.8% agar-agar), and then poured on the prepared agar plates. For the assay, small paper platelets (usually used for antibiotic tests) were placed on the top agar layer, and soaked with differently concentrated calibration solutions of Vitamin B12. The plates were incubated overnight, and the diameters measured after 16 h. From the different diameters, dependent of the Vitamin B12 concentration, a calibration curve was calculated.
In order to determine the B12 content of our periplasm and cytoplasm fractions, the assay plates were prepared exactly as described above, and paper platelets were soaked with the respective solution. Comparing to the calibration curve, the diameter indicates the B12 amount.
5.1. Photometric Assay
In order to determine the Vitamin B12 concentration of a sample photometrically, the corrinoids in our samples needed to be converted into dicyano cobalamine. This was achieved by treating the samples with sodium cyanide or potassium cyanide. Other components like proteins and lipids which would distort the measurements, can easily be separated by precipitation through acidic and basic conditions as well as heating and subsequent centrifugation.
After measuring a spectrum of the supernatant from 350 to 650 nm, the Vitamin B12 concentration could be calculated with help of the extinction coefficients of dicyano cobalamine.
Protocols
Media
LB Medium
Component
Amount
Tryptone
10 g
NaCl
10 g
Yeast extract
5 g
optional:
Agar-agar
15 g
Add components to ddH2O and adjust volume to 1 L.
Autoclave for 20 min at 121°C.
M9 Minimal Medium
Component
Amount
Na2HPO4
10 g
KH2PO4
10 g
NH4Cl
5 g
NaCl
15 g
Add components to ddH2O, adjust the pH to 7.0 and bring the volume to 987 mL.
Autoclave for 20 min at 121°C.
Add the following
Component
Stock solution
Amount
Glucose
20%
10 mL
MgSO4 · 7 H2O
1 M
1 mL
Thiamine · HCl
30 mM
1 mL
CaCl2
0.13 M
1 mL
RM Medium
The dehydrated medium is distributed by Laboratories Conda (Cat. 1542) and was prepared according to protocol.
MM medium
Mineral medium with additon of yeast extract
Component
Amount
K2HPO4
14 g
KH2PO4
6 g
(NH4)2SO4
3 g
MgSO4 · 7 H2O
0.2 g
Yeast extract
0.2 g
CoCl2 (50 mM)
85 µL
SL-4
1 mL
Add components to ddH2O, adjust the pH to 7.5 and bring the volume to 1 L.
100 mM glycerine are added seperatly to the medium.
SL-4: Trace element solution (Pfenning and Lippert, 1966)
Component
Amount
EDTA-Na2
5 g
FeSO4 · 7 H2O
2 g
ZnSO4 · 7 H2O
0.1 g
MnCl2 · 4 H2O
0.03 g
H3BO4
0.3 g
CoCl2 · 6 H2O
0.2 g
CuCl2 · 2 H2O
0.01 g
NiCl2 · 6 H2O
0.02 g
NaMoO4 · 2 H2O
0.03 g
Add components to ddH2O, adjust the pH to 6.7 and bring the volume to 1 L.
The solution is stored at 4°C and protected from light.
Guttman Medium (B12 Assay)
Component
Amount
K2HPO4
14 g
KH2PO4
6 g
(NH4)2SO4
2 g
MgSO4 · 7 H2O
0.2 g
Na2citrate · 2 H2O
1 g
L-Asparagine
0.8 g
L-Arginine
0.2 g
L-Glutamic acid
0.2 g
Glycine
0.2 g
L-Tryptophan
0.2 g
L-Prolin
0.2 g
L-Histidine
0.2 g
Vitamin B12
40 µg
Add components to ddH2O, adjust the pH to 6.8-7.2 and bring the volume to 1 L.
50 mM glucose have to be added seperatly before the inoculation.
Cultivation and Transformation
Cultivation of Escherichia coli
For preperation of competent cells and cloning, the E. coli strains DH5α and TOP10 are grown in LB medium at 37°C.
For expression studies of constructs in pBAD202, E. coli TOP cells are grown in LB medium supplemented with 1 mg/mL Vitamin B12 and 50 µg/mL kanamycin.
Cultivation of Raoultella planticola
For preperation of competent cells and cloning, R. platicola is grown in LB medium at its temperature optimum of 30°C.
For expression studies of constructs in pBAD202, R. planticola is grown under anaerobic conditions in MM medium supplemented with 50 µg/mL kanamycin.
Cultivation of Shimwellia blattae
For preperation of competent cells and cloning, the S. blattae is grown in LB medium at 37°C.
For expression studies of constructs in pBAD202, S. blattae was grown under anaerobic conditions in MM medium supplemented with 50 µg/mL kanamycin.
Preparation of Electrocompetent Cells
Material to prepare
2x 5 mL LB
250 mL LB in 1 L Erlenmeyer flask with chicane
500 mL sterile Millipore-H2O at 4°C
50 mL sterile glycerol (10% (w/v)) at 4°C
30 sterile labeled Eppendorf cups at – 20°C or -80°C
1x sterile GS3 jar at 4°C
1x Aquatron at 30°C
1x Sorvall RC6 centrifuge at 4°C
1x Universal 320R centrifuge at 4°C
1 L liquid nitrogen
 
Day 1
Inoculate 2x 5 mL LB (if required with antibiotics) with your strain of interest, either from cryo culture or from agar plate.
Incubate overnight at 37°C and 150 rpm.
 
Day 2
Inoculate 250 mL LB (without antibiotics) with 2% (5 mL) preculture.
Incubate in the Aquatron at 30°C and 160 rpm until OD600 of 0.5-0.8.
Check the culture via microscope for contaminations.
All liquids and containers must be cooled on ice. The major task of cell preparation is the removal of salts. In case of some strains like pLys-strains, the pellets must be resuspended very carefully.
Let the cells (OD600 of 0.5-0.8) cool down in ice water for 10-20 min. All further steps are performed under cool conditions.
Decant the cells in sterile GS3 jar and centrifuge at 4°C and 5000 rpm (4230 g) for 5-10 min.
Remove supernatant (directly next to the centrifuge or the pellet might resolve) and resuspend the pellet in 250 mL sterile 4°C ddH2O.
Centrifuge the GS3 jar at 4°C and 5000 rpm (4230 g)for 5-10 min.
Remove supernatant (directly next to the centrifuge or the pellet might resolve) and resuspend the pellet in 250 mL sterile 4°C ddH2O.
Centrifuge the GS3 jar at 4°C and 5000 rpm (4230 g)for 5-10 min.
Remove supernatant (directly next to the centrifuge or the pellet might resolve) and resuspend the pellet in 10 mL sterile 4°C glycerol (10% (w/v)). Afterwards, transfer the culture into a 50 mL Falcon tube.
Centrifuge the Falcon tube at 4°C and 6200 rpm (4230 g) for 5-10 min.
Remove supernatant (directly next to the centrifuge or the pellet might resolve) and resuspend the pellet in 10 mL sterile 4°C glycerol (10% (w/v)).
Centrifuge the Falcon tube at 4°C and 6200 rpm (4230 g) for 5-10 min.
Remove supernatant (directly next to the centrifuge or the pellet might resolve) and resuspend the pellet in 500 µL sterile 4°C glycerol (10% (w/v)).
Aliquot the cells into the Eppendorf cups (40 µL per Eppendorf cup). While aliquoting, filled Eppendorf cups must be frozen directly in liquid nitrogen.
Store the filled Eppendorf cups in a cryobox at -80°C.
Transformation of Electrocompetent Cells by Electroporation
Mix 50-200 ng plasmid DNA with 40 µL of competent cells (always defrost on ice!). Transfer attempt into precooled electroporation cuvette. Incubate on ice for 10 min.
Important: DNA must be salt free! Use either directly in ddH2O eluted plasmids, or (e.g. directly after ligation) desalt the DNA for 30 min on a Millipore filter (MF Membrane Filters).
Electroporation: 2500 V for 2 mm cuvettes (1250 V for 1 mm cuvettes), 25 µF, 200 Ω, discharging time should be 3-5 msec. Contacts of the electroporation cuvette must be dry. Remove all air bubbles before electroporation.
Immediately add 960 µL liquid LB (or, if available, SOC medium for a higher transformation efficiency) into the cuvette and transfer the content into a 2 ml Eppendorf cup.
Incubate at 37°C and 150 rpm for 1 h. Fix tube in horizontal position with tape.
Plate 100 µL of 10-3 to 10-6 dilutions on plates with selection pressure
Incubate at 37°C overnight.
Transformation of Chemically Competent Cells by Heat-Shock
One Shot®TOP10 Competent cells (ThermoFisher) were transformed following the respective protocol for chemical transformation:
Add 3 µL of the TOPO®Cloning Reaction into a vial of competent E. coli cells and mix gently by inverting.
Incubate on ice for 5-30 min.
Heat-shock the cells for 30 s at 42°C without shaking.
Immediately transfer the tubes on ice.
Add 250 µL of S.O.C. medium (RT), cap the tube tightly and shake it horizontally (200 rpm) at 37°C for 1 h.
Spread 100-200 µL of different volumes or dilutions from each transformation on a prewarmed selective plate and incubate overnight at 37°C.
Pick colonies for analysis.
Plasmid preparation
Small scale preparation of plasmid DNA was performed using the NucleoSpin® Plasmid kit distributed by Macherey-Nagel (REF 740588) according to protocol.
PCR methods
PCR with Phusion DNA Polymerase
Phusion® High-Fidelity DNA Polymerase is used for all standard PCR amplifications.
Pipetting scheme
Component
20 µL reaction
50 µL reaction
Final concentration
Nuclease-free water
to 20 µL
to 50 µL
5X Phusion HF buffer
4 µL
10 µL
1X
10 mM dNTPs
0.4 µL
1 µL
200 µM
10 µM forward primer
1 µL
2.5 µL
0.5 µM
10 µM reverse primer
1 µL
2.5 µL
0.5 µM
Template DNA
variable
variable
1 pg - 10 ng
DMSO (optional)
(0.6 µL)
(1.5 µL)
3%
Phusion DNA Polymerase
0.2 µL
0.5 µL
0.02 U/µL
Mix all components, adding the DNA polymerase last and spin down.
Cycling
Step
Temperature
Time
Cycles
Initial Denaturation
98°C
30 s
1
Denaturation
98°C
10 s
25-35
Annealing
X°C
30 s
Extension
72°C
15-30 s/kb
Final Extension
72°C
10 min
1
4°C
hold
To optimize the annealing temperature for each primer pair, a temperature gradient can be run.
Fusion PCR
Fusion PCR was performed according to Harrison et al.
Pipetting scheme
Component
50 µL reaction
Final concentration
Nuclease-free water
to 50 µL
5X Phusion HF buffer
10 µL
1X
10 mM dNTPs
1 µL
200 µM
50 mM MgCl2
1.25 µL
1.25 mM
10 µM forward primer
1 µL
0.2 µM
10 µM reverse primer
1 µL
0.2 µM
Template DNA
variable
DMSO
1.5 µL
3%
Phusion DNA Polymerase
0.5 µL
0.02 U/µL
Mix all components, adding the DNA polymerase last and spin down.
Cycling
Step
Temperature
Time
Cycles
Initial Denaturation
98°C
1 min
1
Denaturation
98°C
10 s
4
Annealing
68 ± 4°C
30 s
Extension
72°C
40 s
Denaturation
98°C
10 s
27
Annealing
68 ± 4°C
30 s
Extension
72°C
50 s
Final Extension
72°C
10 min
1
4°C
hold
To optimize the annealing temperature for each primer pair, a temperature gradient can be run.
16S rDNA Colony PCR
16S rDNA PCR was performed to check the identity of the bacterial strains we were working with. For this, colonies were picked and resuspended in 50 µL ddH2O.
Pipetting scheme
Component
Amount
Nuclease-free water
to 50 µL
Optibuffer
5 µL
10 mM dNTPs
5 µL
5 mM forward primer (16S08)
4 µL
5 mM reverse primer (150UR)
4 µL
Template DNA
1 µL (suspension)
MgCl2
3 µL
BIO-X-ACT DNA Polymerase
0.5 µL
Mix all components, adding the DNA polymerase last and spin down.
Plasmid Construction
Digestion of DNA with Restriction Enzymes
Plasmid DNA and PCR products were digested according to FastDigest protocol (Thermo Scientific).
Component
Plasmid DNA
PCR product
Water, nuclease free
15 µL
17 µL
10X FastDigest or 10X FastDigest Green buffer
2 µL
2 µL
DNA
2 µL
(up to 1 µg)
10 µL
(~0.2 µg)
FastDigest enzyme
1 µL
1 µL
Mix gently and spin down.
Incubate at 37°C in a heat block or water thermostat for X min.
Inactivate the enzyme by heating for X min at X°C.
Comment: Incubation time and inactivation time and temperature depend on the applied FastDigest enzyme and were done according to the respective protocol.
If the FastDigest Green Buffer was used in the reaction, an aliquot of the reaction mixture can be loaded directly on an agarose gel.
Phosphorylation of DNA
MutB2 DNA was phosphorylated with T4 Polynucleotide Kinase (T4 PNK) according to protocol (NEB, #M0201).
Component
Amount
DNA
up to 300 pmol of 5' termini
T4 PNK Reaction Buffer (10X)
5 µL
ATP (10 mM)
5 µL
T4 PNK
1 µL (10 units)
Nuclease-free water
up to 50 µL
Prepare reaction mixture on ice.
Incubate at 37°C for 30 min.
Heat inactivate at 65°C for 20 min.
Ligation
Ligations were either done with T4 DNA ligase distributed by Invitrogen (Cat. No. 15224-017) or by Thermo Scientific (#EL0014).
Invitrogen
Prepare the following reaction mixture and spin briefly.
Component
Amount/concentration
5X ligase buffer
4 µL
vector DNA
3-30 fmol
insert DNA
9-90 fmol
T4 DNA ligase
0.1 U
ddH2O, autoclaved
to 20 µL
Incubate at 16°C over night.
Heat-inactivate at 65°C for 10 min.
Thermo Scientific
Prepare the following reaction mixture and spin briefly.
Component
Amount/concentration
10X ligase buffer
2 µL
vector DNA
20-100 ng
insert DNA
1:1 to 5:1 molar ratio over vector
T4 DNA ligase
1 U
ddH2O, autoclaved
to 20 µL
Incubate at RT for 1 h or at 16°C over night.
Heat-inactivate at 65°C for 10 min.
Blunt End Ligation
Blunt end ligations were done with T4 DNA ligase distributed by Thermo Scientific (#EL0014).
Prepare the following reaction mixture and spin briefly.
Component
Amount/concentration
10X ligase buffer
2 µL
vector DNA
20-100 ng
insert DNA
1:1 to 5:1 molar ratio over vector
50% PEG 4000 solution
2 µL
T4 DNA ligase
5 U
ddH2O, autoclaved
to 20 µL
Incubate at RT for 1 h.
Heat-inactivate at 65°C for 10 min.
pBAD Directional TOPO® Cloning
The D-TOPO® cloning reaction of pBAD202 was performed according to protocol (Thermo Scientific, K420201). A 0.5:1 to 2:1 molar ratio of PCR product to vector was used.
Component
Chemically competent cells
Electrocompetent cells
Fresh PCR product
0.5-4 µL
0.5-4 µL
Salt solution
1 µL
-
Dilute Salt Solution (1:4)
-
1 µL
ddH2O
add to a final volume of
5 µL
add to a final volume of
5 µL
TOPO® vector (pBAD202)
1 µL
1 µL
Final volume
6 µL
6 µL
Mix reaction gently and incubate for 5 min at RT.
Place the reaction on ice and proceed directly with the transformation of electrocompetent or chemically competent cells.
DNA Purification
DNA purification after PCR and restriction was performed using different kits
SureClean Plus distributed by Bioline (BIO-37047)
DNA Clean & Concentrator™-25 distributed by Zymo Research (D4033)
NucleoSpin® Gel and PCR Clean-up distributed by Macherey-Nagel (REF740609)
The procedures were done according to the respective protocol.
Analytic Tests
Agarose Gel Electrophoresis
For agarose gel electrophoresis, 1.0% agarose gels were prepared. For analytical purpose, the gels were run with the parameters U=90 V, I=400 mA, t=60 min. For preparative purpose, the gels were run with the parameters U=120 V, I=400 mA, t=35 min.Afterwards, the gels were stained in ethidium bromide solution, and then analyzed by UV light.
Gel extractions of DNA were performed using the NucleoSpin® Gel and PCR Clean-up kit distributed by Macherey-Nagel (REF740609).
Expression Test
In preperation for expression studies, the constructs in pBAD202 were transformed into E. coli TOP10, R. planticola or S. blattae cells. The respective cultivation conditions (medium, temperature) are listed in “Cultivation and Transformation”. The expression assay was done as described in the D-TOPO® protocol.
For each transformant, inoculate 2 mL of medium containing 50 µg/mL kanamycin with a single recombinant colony.
Grow overnight with shaking (225-250 rpm) to OD600 = 1-2.
The next day, label five tubes for each transformant and add 10 mL medium (50 µg/mL kanamycin) to each tube.
Inocculate each tube with 0.1 mL of the overnight culture.
Grow the cultures with vigorous shaking to an OD600= ~0.5.
Prepare four 10-fold serial dilutions of 20% arabinose with sterile water (e.g 2%, 0.2%, 0.02%, and 0.002%).
Remove a 1 mL aliquot of cells from each tube, centrifuge at maximum speed in a microcentrifuge for 30 s and aspirate the supernatant. Freeze the pellet at -20°C as a zero time point sample.
Add arabinose to the five 9 mL cultures according to the following table.
Tube
Stock Solution
Volume [mL]
Final concentration
1
0.002%
0.09
0.00002%
2
0.02%
0.09
0.0002%
3
0.2%
0.09
0.002%
4
2%
0.09
0.02%
5
20%
0.09
0.2%
Grow with shaking for 4 h.
Take 1 mL samples at 4 h and treat as described for the zero time point sample.
 
Analyze the samples by suspending the pellets in 80 µL 1X SDS-PAGE sample buffer.
Boil 5 min and centrifuge briefly.
Load 5-10 µL of each sample on an SDS-PAGE gel and electrophorese.
Cell Fractionation
Harvest 500 mL of cells from an induction experiment by centrifugation and wash three times with 0.9% NaCl.
Resuspend the pellet in 2 mL buffer P (pH 8.0) and incubate for 20-30 min on ice.
Component
Concentration
Tris-HCl
100 mM
Sucrose
20% (w/v)
EDTA
100 mM
Centrifuge for 5 min (14000 rpm, 4°C) and carefully remove the supernatant which contains the periplasm.
Resuspend the pellet in 2 mL buffer P and incubate for 20-30 min on ice.
After another centrifugation for 5 min (14000 rpm, 4°C) the supernatant with the spheroplasts can be carfully removed.
The samples can either be stored at -20°C or directly used for analysis.
SDS PAGE
Acrylamid Stocksolution (AA)
40% Acrylamide 4x cryst.- Mix 37.5:1 (Biomol 54535; stored at 4°C)
Resolving gel Stocksolution (RG)
Component
Amount
Tris-HCl
18.2 g
SDS
0.4 g
Add components to ddH2O, adjust the pH to 8.8 with HCl and bring the volume to 100 mL.
Store at 4°C.
Stacking gel Stocksolution (SG)
Component
Amount
Tris-HCl
6.1 g
SDS
0.4 g
Add components to ddH2O, adjust the pH to 6.8 with HCl and bring the volume to 100 mL.
Store at 4°C.
10x running buffer
Component
Amount
Tris-HCl
30.3 g
Glycine
144.1 g
SDS
10 g
Add components to ddH2O and bring the volume to 1 L. The pH should bea around 8.4.
Store at RT.
4x SDS Loading Dye
Component
Amount
Tris-HCl
0.4 g
SDS
1.2 g
Glycerine (87% w/v)
7.5 mL
β-Mercaptoethanol
2.5 mL
Bromphenol blue (2%)
0.5 mL
Add Tris-HCl and SDS to ddH2O and adjust the pH to 6.8.
Add the rest of the compnents and bring the volume to 50 mL
Store at -20°C.
Ammoniumpersulfat (APS)
10% APS (w/v) in ddH2O
Store at -20°C.
Pipetting schemes for one gel
Component
Resolving gel 10%
Stacking gel 4%
AA
1 mL
0.2 mL
RG
1 mL
-
SG
-
0.48 mL
ddH2O
2 mL
1.32 mL
TEMED
3 µL
2 µL
APS
30 µL
15 µL
Component
Resolving gel 15%
Stacking gel 4%
AA
1.5 mL
0.2 mL
RG
1 mL
-
SG
-
0.48 mL
ddH2O
1.5 mL
1.32 mL
TEMED
3 µL
2 µL
APS
30 µL
15 µL
Preperation of SDS-Gels
Glass plates, combs and spacers are cleaned with EtOH (70%) and assembled.
The resolving gel solution is prepared with TEMED and APS added last for polymerisation. The solution is poured between the assembled galss plates and covered with isopropanol.
After polymerisation, the isopropanol is removed and the stacking gel is prepared and poured on top of the resolving gel. The combs need to be inserted immediatly.
After polymerisation, the gels can either be put into the electrophoresis chamber with 1x running buffer or stored at 4°C in a bag with wet tissue. 
Preperation of samples
Samples are mixed with SDS loading dye (4:1), denaturated at 95°C for 5 min and breifly centrifuged.
Up to 15 µL of a sample are loaded into one well.
SDS-PAGE
After loading the samples and a protein marker onto the gel, it is run for 15 min with an electric current of 15 mA current (per gel). Afterwards, the gel is run with an electric current of 30 mA (per gel).
Staining with Coomassie-Brilliant-Blue
The following solutions have to be prepared.
Staining solution
Final concentration
Component
Amount
0.05%
Coomassie-Brilliant-Blue R 250
0.5 g
25%
Isopropanol
250 mL
10%
Acetic acid
100 mL
Dissolve the components in ddH2O and bring the volume to 1 L.
The solution can be used more than once and is stored at RT.
Fixation solution
10% acetic acid
 
Place the SDS-gel in a box with staining solution and heat in the microwave.
Incubate the gels on a rocker until protein bands become visible.
Wash the gel with water and discolour it in fixation solution for several hours or overnight.
Western Blot
For Western Blot an Anti-His antibody (Anti-His (C-term)/AP) was used.
The following solutions need to be prepared for the procedure.
10x TBS buffer
Final concentration
Component
Amount
0.1 M
Tris-HCl (MW 121.14)
6.057 g
1.4 M
NaCl (MW 58.44)
40.91 g
Dissolve the components in ddH2O, adjust the pH to 7.4 and bring the volume to 500 mL.
Store at RT.
1x Transfer buffer
Final concentration
Component
Amount
25 mM
Tris-HCl (MW 121.14)
3.029 g
192 mM
Glycine (MW 75.07)
14.41 g
Methanol
200 mL
Dissolve the components in ddH2O and bring the volume to 1 L. The pH should be between 8.1 and 8.4.
Store at RT.
10x TBS-Tween buffer
Final concentration
Component
Amount
0.1 M
Tris-HCl (MW 121.14)
6.057 g
1.4 M
NaCl (MW 58.44)
40.91 g
0.1%
Tween20
0.5 mL
Dissolve the components in ddH2O, adjust the pH to 7.4 and bring the volume to 500 mL.
Store at RT.
Blocking Solution
Dissolve 2.5 g skim milk in 50 mL 1x TBS buffer.
It has to be prepared freshly.
Phosphatase buffer
Final concentration
Component
Amount
0.1 M
Tris-HCl (MW 121.14)
6.057 g
0.1 M
NaCl (MW 58.44)
40.91 g
5 mM
MgCl2 x 6 H2O (MW 203.3)
0.51 g
Dissolve the components in ddH2O, adjust the pH to 7.4 and bring the volume to 500 mL.
Store at RT.
In preparation for the Western Blot, a SDS-PAGE with a Protein Ladder, which is compatible with Western Blot, has to be run.
Prepare 6 gel sized Whatman paper and gel sized blot membrane by soaking them in 1x Transfer buffer.
Set up the Western Blot. Place 3 moist Whatman papers on the anode (base of blotter), followed by the moist membrane, the unstained SDS-gel, 3 moist Whatman papers and the cathode (lid of blotter).
Blot at 200 mA (1 gel) and constant 10 V for 1 h.
Wash the membrane three times for 5 min with 1x TBS buffer on a rocker.
Block with 50 mL 1x TBS buffer with 5% skim milk for 1 h or over night on a rocker.
Wash three times for 15 min with 1x TBS buffer on a rocker.
Incubate with antibody solution (14 mL 1x TBS-Tween buffer + 200 µL 1x TBS buffer/5% skim milk + 7 µL antibody) for 2 h on a rocker.
Wash two times shortly with 1x TBS-Tween buffer.
Wash once with 1x TBS-Tween buffer for 15 min on a rocker.
Wash two times with 1x TBS-Tween buffer for 5 min on a rocker.
Wash two times with 1x TBS buffer for 10 min on a rocker.
Wash two times shortly in 1x phosphatase buffer.
Place the membrane in staining solution (10ml 1x phosphatase buffer + 66µl NBT + 33µl BCIP), stain in the dark until bands become visible.
Wash with H2O to stop the reaction.
Dry the membrane in Whatman paper and scan it.
Microbial B12 Plate Assay
The B12 Plate Assay allows the detection of corrinoids in cell extracts or supernatants. It is based on the fact, that E. coli DSM 4261 relies on its B12 dependent methionine synthase which is encoded by metH. This is caused by a point mutation in metE which encodes the B12 independent methionine synthase. In a medium without methionine, E. coli DSM 4261 therefore relies on the presence of B12.
Cultivation of E. coli DSM 4261
50 mL of Guttman medium are inoculated with a 5 mL preparatory overnight culture.
The cells are harvested at an OD600 of 0.2, washed three times with 20 mL 0.9% NaCl and afterwards resuspended in the same volume NaCl.
Preparation of samples
After induction, the cells are centrifuged down, and washed three times with 0.1 M sodium chloride solution.:
Component
Amount
K2HPO4
13.6 g
KH2PO4 · 3 H2O
22.8 g
Add components to ddH2O, adjust the pH tp 7.0 and bring the volume to 1 L.
Periplasm, cytoplasm and crude cell extracts from different expression studies are used as samples.
Assay
7 mL of the cell suspension, 50 mM and 0.2 g/L triphenyltetrazoliumchloride are added to 500 mL of the autoclaved mineral-agar (50°C).
Component
Amount
K2HPO4
7 g
KH2PO4
3 g
(NH4)2SO4
1 g
MgSO4 · 7 H2O
0.1 g
Na2citrate · 2 H2O
3 g
Agar Bacteriological
15 g
Add components to ddH2O and bring the volume to 1 L.
4 sterile filter papers (Ø 0.9 cm, Fa. Schleicher & Schuell, Dassel) are laid onto the plates and loaded with 10-15 µL of a sample or a Vitamin B12 calibration solution (0, 0.05, 0.25, 0.1, 5.0 μg/ml). Every sample or concentration is analyzed twice.
The plates are incubated at 37°C for 16-24 h.
If the samples contained Vitamin B12, the cells grow in a red zone around the filter paper. The size is proportional to the Vitamin 12 concentration and the reduction of triphenyltetrazoliumchloride causes the red staining.
To analyze the B12 concentration in the samples, the diameter of the red zone is measured and compared to the calibrations.
Photometric B12 Assay
To determine the corrinoid concentration in a sample, the corrinoids have to be converted into the soluble dicyano form and proteins and lipids have to be removed by precipitation and centrigugation.
Incubate 1 mL of your sample with 0.2% (w/v) NaCN or KCN for 30 min at RT in the dark.
Acidify the solution with pure acetic acid to a pH of 4.0.
Boil the samples at 100°C for 20 min.
Let the sample cool down and adjust the pH to 11.0 with KOH.
Add 4-5 mg NaCN or KCN.
Centriguge at 13,000 rpm for 30 min at RT.
Measure a spectrum of the supernatant from 350 to 650 nm.
The concentration of Vitamin B12 can be calculated with help of the molar extinction coefficient of dicyano cobalamine (Δε580-640 = 7.3 mM-1 · cm-1) by determining the differences in the absorption at 580 and 650 nm.
Alternatively, the concentration of Vitamin B12 can also be determined with the extinction coefficients of dicyano cobalamine at 580 nm (ε540 = 7800 mM-1 · cm-1) and 367 nm (Δε367 = 23800 mM-1 · cm-1) by calculating the average.
As a standard hydroxyl cobalamine and cyano cobalamine are used.
Bradford Assay
Bradford solution
Mix 800 mL ddH2O with 200 mL RotiQuant (5X) (Carl Roth).
Store at 4°C protected from light.
After preparing a new solution, the Bradford factor has to be determined by measuring a standard curve from protein standards with known concentrations.
Measuring protein sample
Mix 20 µL of the sample with 1mL Bradford solution in a photometer cuvette. Prepare triplicates.
Also prepare a blank by mixing 20 µL ddH2O with 1 mL Bradford solution in a photometer cuvette.
Incubate for 5 min at RT protected from light.
Measure the absorbance at 595 nm against the black. It should lie between 0.1 and 0.6, otherwise the protein sample has to be diluted.
Calculation of protein concentration
Contact
  Facebook
  Twitter
  E-Mail
  Homepage
  Address
iGEM Team Göttingen 2016
Institut für Mikrobiologie
und Genetik
AG Prof. Dr. Rolf Daniel
Grisebachstr. 8
37077 Göttingen
Navigation
Home
Team
Team
Collaborations
Project
Description
Experiments
Proof of Concept
Results
Notebook
Parts
Parts
Basic Parts
Safety
Attributions
Human Practices
Human Practices
HP on the Road
HP on the Web
Engagement
Sponsors
