<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_Goettingen_Description skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><DIV id="top_title"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:Goettingen/Description</SPAN></H1></DIV><DIV id="HQ_page"><DIV id="bodyContent"><NAV class="navbar navbar-inverse"><DIV class="container-fluid"><DIV class="navbar-header"><BUTTON type="button" class="navbar-toggle collapsed" data-toggle="collapse" data-target="#bs-example-navbar-collapse-1" aria-expanded="false"><SPAN class="sr-only">Toggle navigation</SPAN></BUTTON></DIV><DIV class="collapse navbar-collapse" id="bs-example-navbar-collapse-1"><UL class="nav navbar-nav"><LI><A href="https://2016.igem.org/Team:Goettingen">Home</A></LI><LI class="dropdown"><A href="#" class="dropdown-toggle" data-toggle="dropdown" role="button" aria-haspopup="true" aria-expanded="false">Team </A></LI><LI><A href=" https://2016.igem.org/Team:Goettingen/Team"> Team   </A></LI><LI><A href="https://2016.igem.org/Team:Goettingen/Collaborations"> Collaborations </A></LI></UL><LI class="dropdown"><A href="#" class="dropdown-toggle" data-toggle="dropdown" role="button" aria-haspopup="true" aria-expanded="false">Project </A></LI><LI><A href="https://2016.igem.org/Team:Goettingen/Description"> Description </A></LI><LI><A href="https://2016.igem.org/Team:Goettingen/Experiments"> Experiments </A></LI><LI><A href="https://2016.igem.org/Team:Goettingen/Proof"> Proof of Concept </A></LI><LI><A href="https://2016.igem.org/Team:Goettingen/Results"> Results </A></LI><LI><A href="https://2016.igem.org/Team:Goettingen/Notebook"> Notebook </A></LI><LI class="dropdown"><A href="#" class="dropdown-toggle" data-toggle="dropdown" role="button" aria-haspopup="true" aria-expanded="false">Parts </A></LI><LI><A href="https://2016.igem.org/Team:Goettingen/Parts">Parts </A></LI><LI><A href="https://2016.igem.org/Team:Goettingen/Basic_Part"> Basic Parts </A></LI><LI><A href="https://2016.igem.org/Team:Goettingen/Safety">Safety</A></LI><LI><A href="https://2016.igem.org/Team:Goettingen/Attributions">Attributions</A></LI><LI class="dropdown"><A href="#" class="dropdown-toggle" data-toggle="dropdown" role="button" aria-haspopup="true" aria-expanded="false">Human Practices </A></LI><LI><A href="https://2016.igem.org/Team:Goettingen/Human_Practices"> Human Practices </A></LI><LI><A href="https://2016.igem.org/Team:Goettingen/HP/Silver"> HP on the Road </A></LI><LI><A href="https://2016.igem.org/Team:Goettingen/HP/Gold"> HP on the Web </A></LI><LI><A href="https://2016.igem.org/Team:Goettingen/Engagement"> Engagement </A></LI></DIV></DIV></NAV><DIV id="pageContent"><DIV class="row"><DIV class="col-md-12"><DIV class="row"><DIV class="col-md-9"><DIV class="heading"><H1> Project Description </H1></DIV><DIV class="row"><DIV class="col-md-12"><DIV class="text"><H2>Background</H2><P>Vitamin B<SUB>12</SUB> is involved in metabolic functions in all organisms, and is therefore an essential nutrient. However, it can only be synthesized by some bacteria and archaea, thus, animals have to obtain it through their diet. Thereby, only animal products like meat and dairy contain B<SUB>12</SUB> in general. This B<SUB>12</SUB> was synthesized by microorganisms colonizing the gastrointestinal tract of those animals and accumulated in the animals’ tissues.</P>Vitamin B<SUB>12</SUB> is one of the most expensive biochemicals in the world, and its synthesis is extraordinarily complex. Since the chemical production of Vitamin B<SUB>12</SUB> requires 70 synthesis steps, it is far too technically challenging and expensive. Therefore, its production is facilitated by genetically engineered microorganisms. These are able to produce Vitamin B<SUB>12</SUB> in industrial amounts and achieve a high product quality. However, the produced Vitamin B<SUB>12</SUB> is harvested by cell lysis, which prevents a continuous production. The efficiency of production could be increased by exporting Vitamin B<SUB>12</SUB> outside the cells. To date, no natural cellular Vitamin B<SUB>12</SUB> exporter is known.</DIV><DIV class="text"><H2>Relevance</H2><P>An adult needs approximately 3.0 µg Vitamin B<SUB>12</SUB> per day which is essential for certain functions: it is involved in processes concerning synthesis of DNA, hormones and neurotransmitters and also acts in the formation of the nervous system and blood. Hence, a B<SUB>12</SUB> deficiency can cause diverse diseases like cancer, dementia, depression, pernicious anemia and polyneuropathy.</P><P>Nowadays, many people do not eat meat or animal products at all. Therefore, these people have a high risk to suffer from B<SUB>12</SUB> deficiency if their diet does not provide enough B<SUB>12</SUB>.</P><P>Vitamin B<SUB>12</SUB> is also used in the industry as it is needed in the biotechnical production of various organic substances. Furthermore, it is added to diverse daily products like toothpaste, fruit gum, non-diary milk or cleaning solution for contact lenses.</P></DIV><DIV class="text"><H2>Our Aim </H2><P>We intend to design, construct and introduce a synthetic Vitamin B<SUB>12</SUB> exporter (“Synporter”) into a production organism. Thereby, we aim for facilitated and higher yields in the industrial Vitamin B<SUB>12</SUB> production without requiring cell lysis.
The B<SUB>12</SUB> Synporter consists of a signal for a Twin Arginine Transporter (Tat) mediated export that is linked to a B<SUB>12</SUB>-binding domain. This construct will be expressed in <EM>R. planticola</EM> and <EM>S. blattae</EM>. We will not only test these different production organisms but also different B<SUB>12</SUB>-binding domains.</P><P>However, Vitamin B<SUB>12</SUB> can also be regarded as a prime example which is what we want to state with our work. This Synporter should work with basically any chemical which is desired to be exported out of a cell. We know from other iGEM Teams that they require bioactive compounds which are derived from nature, but are not synthesized by any organism (e.g. artificial amino acids). These are very expensive (&lt; 1000 € per gram). However, finding novel enzymes for the synthesis of artificial compounds like drugs, for instance using metagenomics, turned out to be very promising and successful. Once one found the right enzymes, one is able to produce these compounds in an organism. Using our Synporter approach with an exchanged binding protein domain, these components can then be exported efficiently, making the production significantly easier and cheaper.</P></DIV><DIV class="text"><H2> More Theory About Vitamin B<SUB>12</SUB></H2><P>Vitamin B<SUB>12</SUB> – or Cobalamin – is a very complex compound (see Fig. 1), consisting of a corrin-macrocycle with a centrally incorporated Cobalt-ion, carrying varying upper ligands, and further a dimethylbenzimidazole group attached to the lower side of the corrin-ring and the Cobalt-ion. Depending on what ligand is attached to the Cobalt-ion, different forms of Cobalamin can be distinguished. Natural occuring forms are methyl- (CH<SUB>3</SUB>-), hydroxy- (OH-), cyano- (CN-) and adenosylcobalamin (5’-deoxyadenosyl-) (also called coenzyme B<SUB>12</SUB>), while cyanocobalamin (which is Vitamin B<SUB>12</SUB> by definition) represents the industrial manufactured form. [1], [2], [3]</P><DIV class="caption"><P style="text-align:center;"><B>Fig. 1: The Structure of Cobalamin</B>. Modified from [2]</P></DIV><P>Only certain microorganisms are able to synthesize Vitamin B<SUB>12</SUB>. Since these microorganisms colonize animals’ gastrointestinal tracts where the B<SUB>12</SUB> accumulates, we humans uptake it via eating animal products like meat and dairy, with meat containing more B<SUB>12</SUB> in general. Thereby, the B<SUB>12</SUB> content varies between different types of animals and also between the diverse tissues. Ruminants’ entrails contain most B<SUB>12</SUB>. However, the type of cooking has further impact on the B<SUB>12</SUB> content. [4]</P><P>Also some vegetarian B<SUB>12</SUB> nourishments exist. But, again, the B<SUB>12</SUB> was synthesized by microorganisms living in symbiosis with these products, e.g. algae and edible mushrooms, or being part of the production process as in tempe or beer. Nevertheless, the B<SUB>12</SUB> content in plant-derived food is often biologically inactive in humans or only present in small quantities that an adult for example would have to drink several liters beer per day in order to cover the B<SUB>12</SUB> requirement. [5], [6]</P><P>An adult needs approximately 3.0 µg Vitamin B<SUB>12</SUB> per day [7]. Vitamin B<SUB>12</SUB> fulfills diverse functions in the human body. Thereby, only the two forms methylcobalamin (MeCbl) and adenosylcobalamin (AdoCbl) are biologically active. MeCbl functions together with the methionine synthase in the methylation of homocysteine, resulting in methionine. Thus, MeCbl contributes to the one-carbon metabolism and the formation of S-adenosylmethionine, thereby being part of the synthesis of diverse molecules like DNA, RNA, hormones, proteins, lipids or myelin. AdoCbl on the other hand interacts with the methylmalonyl-coenzyme A mutase in forming succinate out of propionate, by that contributing to the fat and protein metabolism, hemoglobin synthesis and gluconeogenesis. Consequently, a Vitamin B<SUB>12</SUB> deficiency can cause diverse disorders when important compounds cannot be synthesized and partially cytotoxic intermediates, like homocysteine, accumulate. Heart and vascular diseases, stroke, arteriosclerosis, megaloblastic anemia, depression, fatigue or neuropathy can amongst others result from a B<SUB>12</SUB> deficiency. [8], [9], [10] </P><P>Especially vegetarians and vegans as well as elderly people have a higher risk to lack Vitamin B<SUB>12</SUB>. Therefore, these people could need to compensate the deficiency by taking B<SUB>12</SUB> supplements.</P><P>Vitamin B<SUB>12</SUB> is also added to diverse everyday products like vegetal milk, fruit gum, contact lense solution, tooth paste, energy drink etc. Just take a closer look on the products’ ingredients you use in your daily routine.</P><H3>References</H3><OL class="ref" style="start:1;"><LI>L. Randaccio, S. Geremia, N. Demitri &amp; J. Wuerges (2010): ”Vitamin B<SUB>12</SUB>: Unique Metalorganic Compounds and the Most Complex Vitamins.” Molecules 15: 3228-3259</LI><LI>J.-H. Martens, H. Barg, M.J. Warren, D. Jahn (2001): Microbial production of vitamin B12. Appl Microbiol Biotechnol 58:275–285.</LI><LI>E. Raux, H. L. Schubert &amp; M. J. Warren (2000): “Biosynthesis of cobalamin (vitamin B<SUB>12</SUB>): a bacterial conundrum.” Cell Mol Life Sci 57(13-14):1880-93.</LI><LI>D. Gille &amp; A. Schmid (2015): ”Vitamin B<SUB>12</SUB> in meat and dairy products.” Nutr Rev 73(2): 106-115.</LI><LI>F. Watanabe, Y. Yabuta, Y. Tanioka &amp; T. Bito (2013):” Biologically Active Vitamin B<SUB>12</SUB> Compounds in Foods for Preventing Deficiency among Vegetarians and Elderly Subjects.” J Agr Food Chem 61: 6769-6775.</LI><LI>Jr Mayer O, J. Simon &amp; H. Rosolovà (2001): ”A population study of the influence of beer consumption on folate and homocysteine concentrations.” Eur J Clin Nutr 55(7): 605-609.</LI><LI>Deutsche Gesellschaft für Ernährung: Vitamin B<SUB>12</SUB> (Cobalamine). 
	Available online: https://www.dge.de/wissenschaft/referenzwerte/vitamin-B<SUB>12</SUB>/, 13.10.16 16:49.</LI><LI>N. F. Shuttle: “Mineral Nutrition of Livestock”. 4th Edition, 2010, CABI, London.
</LI><LI>National Institutes of Health: Vitamin B<SUB>12</SUB>.
Available online: https://ods.od.nih.gov/factsheets/VitaminB<SUB>12</SUB>-HealthProfessional/, 13.10.16 16:56.</LI><LI>J. F. Combs Jr: “The Vitamins: Fundamental Aspects in Nutrition and Health.” 3rd Edition, 2008, Elsevier Academic Press, Oxford, UK.</LI></OL></DIV><DIV class="text" style="margin-bottom:16px;"><H2>More Theory About our Synporter</H2><H3>The blueprint of our B<SUB>12</SUB> Synporter</H3><P>Our B<SUB>12</SUB> Synporter is built of a B<SUB>12</SUB>-binding protein domain linked to a signal for a Twin Arginine Transporter (Tat) mediated export. While the B<SUB>12</SUB>-binding domain (obviously) binds the vitamin, the export signal allows the protein-vitamin complex to leave the cytoplasm via the Tat system. In the following section, we will have a further look on these components.</P><H3>The Tat export pathway and its signal peptide</H3><P>In bacteria and archaea, proteins located outside the cytoplasm can reach their destination via either the Sec or the Tat (twin-arginine translocation) export pathway. While the Sec system translocates proteins in an unstructured state, the Tat apparatus has the unusual property of <B>transporting fully folded proteins</B> (Palmer and Berks, 2012). This system is very flexible in regard to the types of proteins that can be exported and the number of exported proteins highly differs between organisms. The <I>E. coli</I> Tat system is capable of transporting substrates up to 70 Å in diameter (Berks <I>et al.</I>., 2000). Many <B>exported proteins containing non-covalently bound cofactors</B> use this pathway, because the cofactor is held in place by the protein folding. The Tat pathway is only used by proteins containing certain types of cofactors that are classified as metal-sulphur clusters or nucleotide based cofactors, which include among others also cobalamins (Berks <I>et al.</I>., 2003). Therefore, our B<SUB>12</SUB>-bound Synporter should be able to be exported using this pathway.</P><P>Proteins are targeted to the Tat apparatus by <B>N-terminal signal peptides</B> that are normally cleaved by an externally facing signal peptidase (Lüke <I>et al.</I>., 2009). The overall architecture is similar to Sec signal peptides and includes a tripartite structure with a basic n region at the N terminus, a hydrophobic h region in the middle and a polar c region at the C terminus. The key element of a Tat signal peptide is the <B>highly conserved twin-arginine motif</B>, defined as SRRXFLK. The two arginines are almost always invariant, while the other residues occur with a frequency of &gt; 50 %. The amino acid at position X is usually polar (Palmer and Berks, 2012). For our Synporter we chose the <B>TorA signal peptide</B> from the trimethylamine-N-oxide reductase from <I>E. coli</I>, a well characterized protein exported via the Tat system. Using this signal peptide, it has already been achieved to export a heterologous protein normally transported by the Sec pathway through the Tat system (Christóbal <I>et al.</I>., 1999). To connect the export signal with a B<SUB>12</SUB>-binding protein domain we use the five amino acids following the signal peptide in trimethylamine-N-oxide reductase as a linker.</P><DIV class="caption"><P><B>Figure 1. Function of the Tat pathway.</B> In contrast to the Sec pathway which translocates unfolded proteins, the Tat pathway translocates already folded proteins through a cellular membrane. (Rao and Anné, 2011)</P></DIV><DIV class="caption"><P><B>Figure 2. Schematic view of the Tat signal peptide aligned with signal sequences from proteins exported via the tat pathway.</B></P><P>Residues that match the Tat consensus are shown in red, with the twin arginines underlined. (Palmer and Berks, 2012)</P></DIV><H3>The B<SUB>12</SUB> binding domain</H3><P>For the binding of B<SUB>12</SUB> to our Synporter, we use domains of proteins which are naturally capable of doing so. To increase our chances of success, we chose to test three different constructs.</P><P><B>MutB</B> is the α-subunit of the methylmalonyl coenzyme A mutase from the Gram-positive bacterium <I>Propionibacterium shermanii</I>. The enzyme family it belongs to uses adenosylcobalamin to create a substrate radical that then rearranges to exchange a hydrogen atom with a group attached to a neighbouring carbon atom. Methylmalonyl CoA mutase catalyses the interconversion between linear succinyl CoA and branched methylmalonyl CoA and is thus involved in the fermentation of pyruvate to propionate (Allen <I>et al.</I>.,1963). It is an αβ heterodimer and has a molecular weight of 150 kDa (80 kDa for the α chain and 70 kDa for the β chain). The active site as well as the cobalamin binding site are located in the α chain (Francalanci <I>et al.</I>., 1986). Crystal structures of the whole enzyme bound to cobalamin (PDB 1REQ; Mancia <I>et al.</I>., 1996) as well as bound to cobalamin and its substrate (PDB 4REQ; Manica and Evans, 1998) are available. Since a TorA-mutB fusion protein could already be produced in significant amounts and has been exported to the periplasm in <I>E. coli</I> BL21, <I>C. freundii</I> and <I>S. blattae</I> (Toeche-Mittler, 2002), MutB is our most promising candidate.</P><P><B>GlmS</B> is the B<SUB>12</SUB>-binding subunit of glutamate mutase (Glm) from <I>Clostridium cochlearium</I>. The mechanism by which the enzyme uses adenosylcobalamin is highly similar to methylmalonyl coenzyme A mutase. Glm catalyzes the reversible rearrangement of (2S)-glutamate to (2S,3S)-3-methylaspartate (Leutbecher <I>et al.</I>., 1992). This reaction is the first step in the fermentation of glutamate to acetate and butyrate (Buckel <I>et al.</I>., 1974). The assembly of the active enzyme, an α2β2 tetramer, is mediated by coenzyme B<SUB>12</SUB>. While GlmS as the smaller subunit (14.8 kDa) binds B<SUB>12</SUB>, the larger subunit GlmE harbors the substrate binding site (Zelder <I>et al.</I>., 1994). Crystal structures of the whole enzyme in complex with its substrate and adenosylcobalamin (PDB 1I9C; Gruber <I>et al.</I>., 2001) as well as an NMR structure of only GlmS (PDB 1B1A; Hoffmann <I>et al.</I>., 1999) are available.</P><P><B>BtuF</B> is the periplasmic binding protein for the Vitamin B<SUB>12</SUB> transporter BtuCD from <I>E. coli</I> (Cadieux <I>et al.</I>., 2002). While some bacteria and archaea are capable of its synthesis, <I>E. coli</I> belongs to the majority of prokaryotes that contain transport systems to import B<SUB>12</SUB> (Warren <I>et al.</I>., 2002). Its transmembrane transport is achieved by the Btu (B twelve uptake) system composed of BtuB, an outer membrane TonB-dependent transporter (Cadieux <I>et al.</I>., 1999), and the ABC transporter BtuCDF, which is located in the inner membrane. While BtuC and BtuD compose respectively the trans-membrane domain and the ABC (Bassford <I>et al.</I>., 1977), BtuF is the periplasmic binding protein. It has a size of 30.19 kDa and is composed of two globular domains, between which Vitamin B<SUB>12</SUB> is bound, linked by a rigid interdomain α-helix (Karpowich <I>et al.</I>., 2003). A crystal structure of the protein bound to B<SUB>12</SUB> is available (PDB1N4A; Karpowich <I>et al.</I>., 2003). For us it is highly interesting to see, if a protein that is naturally involved in B<SUB>12</SUB> import can be converted to mediate its export. If this was possible, the concept could potentially be applied to other proteins involved in the import of small molecules.</P><H3>References</H3><OL class="ref" style="start:10;"><LI>Allen S. H. P., Kellermeyer R. W., Stjernholm R., Wood H. G. 1964 Purification and properties of enzymes involved in the propionic acid fermentation. J. Bacteriol. 87:171-187</LI><LI>Bassford P. J., Jr., Kadner R. J. 1977 Genetic Analysis of Components Involved in Vitamin B<SUB>12</SUB> Uptake in Escherichia coli. J. Bacteriol. 132:796–805.</LI><LI>Berks B. C., Sargent F., Palmer T. 2000 The Tat protein export pathway. Mol Microbiol. 35(2):260-74.</LI><LI>Berks, B. C., Palmer, T., Sargent, F. 2003 The Tat protein translocation pathway and its role in microbial physiology. Adv. Microb. Physiol. 47:187–254.</LI><LI>Buckel, W. &amp; Barker, H.A. 1974 Two pathways of glutamate fermentation by anaerobic bacteria. J. Bacteriol. 117, 1248±1260.</LI><LI>Cadieux N, Bradbeer C, Reeger-Schneider E, Köster W, Mohanty AK, Wiener MC, Kadner RJ. 2002 Identification of the periplasmic cobalamin-binding protein BtuF of Escherichia coli. J Bacteriol. 184(3):706-17.</LI><LI>Cadieux N., Kadner R. J. 1999 Site-directed disulfide bonding reveals an interaction site between energy-coupling protein TonB and BtuB, the outer membrane cobalamin transporter. Proc. Natl. Acad. Sci. U. S. A. 96:10673–10678.</LI><LI>Christόbal S., de Gier J.-W., Nielsen H. and von Heijne G., 1999 Competition between Sec- and Tat- dependent protein translocation in Escherichia coli. EMBO J. Vol. 18, No. 11: 2982-2990.</LI><LI>Francalanci F., Davis N. K., Fuller J. Q., Murfitt D., Leadlay P. F., 1986 The subunit structure of methylmalonyl-CoA mutase from Propionibacterium shermanii. Biochem. J. 236:489-494</LI><LI>Gruber, K., Reitzer, R., Kratky, C. 2001 Radical Shuttling in a Protein: Ribose Pseudorotation Controls Alkyl-Radical Transfer in the Coenzyme B(12) Dependent Enzyme Glutamate Mutase. Angew.Chem.Int.Ed.Engl. 40: 3377-3380</LI><LI>Hoffmann B, Konrat R, Bothe H, Buckel W, Kräutler B. 1999 Structure and dynamics of the B<SUB>12</SUB>-binding subunit of glutamate mutase from <I>Clostridium cochlearium</I>. Eur. J. Biochem. 263(1):178-88.</LI><LI>Karpowich N. K., Huang H. H., Smith P. C., Hunt J. F. 2003 Crystal Structures of the BtuF Periplasmic-binding Protein for Vitamin B<SUB>12</SUB> Suggest a Functionally Important Reduction in Protein Mobility upon Ligand Binding. The Journal of Biological Chemistry 278, 8429-8434.</LI><LI>Leutbecher, U., Boecher, R., Linder, D. &amp; Buckel, W. 1992 Glutamate mutase from <I>Clostridium cochlearium</I>. Purification, cobamide content and stereospecific inhibitors. Eur. J. Biochem. 205, 759±765.</LI><LI>Lüke, I., Handford, J. I., Palmer, T. &amp; Sargent, F. 2009 Proteolytic processing of <I>Escherichia coli</I> twin-arginine signal peptides by LepB. Arch. Microbiol. 191:919–925.</LI><LI>Manica F., Evans P. R., 1998 Conformational changes on substrate binding to methylmalonyl CoA mutase and new insights into the free radical mechanism. Structure 6:711-720</LI><LI>Mancia, F., Keep, N.H., Nakagawa, A., Leadlay, P.F., McSweeney, S., Rasmussen, B., Bosecke, P., Diat, O., Evans, P.R. 1996 How coenzyme B<SUB>12</SUB> radicals are generated: the crystal structure of methylmalonyl-coenzyme A mutase at 2 A resolution. Structure 4:339-350</LI><LI>Palmer T., Berks B. C., 2012 The twin-arginine translocation (Tat) protein export pathway. Mature Reviews Microbiology 10:483-496</LI><LI>Rao S. and Anné J. 2011, Bacterial type I signal peptidases as antibiotic targets. Future Microbiology 11:1279-1296</LI><LI>Toeche-Mittler, C. 2002 Konstruktion eines bakteriellen Systems zum Export von Coenzym B<SUB>12</SUB>. Dissertation zum Erlangen des Doktorgrades der Mathematisch-Naturwissenschaftlichen Fakultäten der Georg-August-Universität zu Göttingen</LI><LI>Zelder, O., Beatrix, B., Leutbecher, U. &amp; Buckel, W. 1994 Characterization of the coenzyme-B<SUB>12</SUB>-dependent glutamate mutase from <I>Clostridium cochlearium</I> produced in Escherichia coli. Eur. J. Biochem. 226, 577±585.</LI><LI>Warren M. J., Raux E., Schubert H. L., Escalante-Semerena J. C. 2002 The biosynthesis of adenosylcobalamin (vitamin B<SUB>12</SUB>). Nat. Prod. Rep. 19:390–412.</LI></OL></DIV></DIV></DIV></DIV><DIV class="col-md-3"><DIV class="sidebar"><H3 style="text-align: center;">Contact</H3><DIV class="sidebaraddress"><UL><LI><A href="https://www.facebook.com/iGEM-Team-Goettingen-687557807970778">  Facebook</A></LI><LI><A href="https://twitter.com/iGEM_Goettingen">  Twitter</A></LI><LI><A href="mailto:igem2016@gwdg.de">  E-Mail</A></LI><LI><A href="http://igem2016.uni-goettingen.de/">  Homepage</A></LI><LI><A href="http://appmibio.uni-goettingen.de/">  Address</A></LI></UL></DIV><A href="http://appmibio.uni-goettingen.de/"><DIV class="sidebaraddress" style="padding-left:42px; margin-top:10px;"><FONT size="-1">
  iGEM Team Göttingen 2016
  Institut für Mikrobiologie 
  und Genetik
  AG Prof. Dr. Rolf Daniel
  Grisebachstr. 8
  37077 Göttingen</FONT></DIV></A><DIV class="sidebar_navigation"><H3 style="text-align: center;">Navigation</H3><DIV class="menu_wrapper"><UL id="accordion" class="accordion"><LI class="menu_item"><A href="https://2016.igem.org/Team:Goettingen">Home</A></LI><LI class="menu_item"> Team
  </LI><LI><A href=" https://2016.igem.org/Team:Goettingen/Team"> Team   </A></LI><LI><A href="https://2016.igem.org/Team:Goettingen/Collaborations">Collaborations</A></LI></UL><LI class="menu_item" style="background-color: #e7e7e7;"><B>Project</B></LI><LI><A style="background-color: #D8D8D8;" href="https://2016.igem.org/Team:Goettingen/Description"><B>Description</B></A></LI><LI><A href="https://2016.igem.org/Team:Goettingen/Experiments"> Experiments </A></LI><LI><A href="https://2016.igem.org/Team:Goettingen/Proof">Proof of Concept</A></LI><LI><A href="https://2016.igem.org/Team:Goettingen/Results"> Results </A></LI><LI><A href="https://2016.igem.org/Team:Goettingen/Notebook"> Notebook </A></LI><LI class="menu_item"> Parts
  </LI><LI><A href="https://2016.igem.org/Team:Goettingen/Parts">Parts</A></LI><LI><A href="https://2016.igem.org/Team:Goettingen/Basic_Part">Basic Parts </A></LI><LI class="menu_item"><A href="https://2016.igem.org/Team:Goettingen/Safety">Safety</A></LI><LI class="menu_item"><A href="https://2016.igem.org/Team:Goettingen/Attributions">Attributions</A></LI><LI class="menu_item"> Human Practices
  </LI><LI><A href="https://2016.igem.org/Team:Goettingen/Human_Practices">Human Practices</A></LI><LI><A href="https://2016.igem.org/Team:Goettingen/HP/Silver">HP on the Road</A></LI><LI><A href="https://2016.igem.org/Team:Goettingen/HP/Gold">HP in the Web</A></LI><LI><A href="https://2016.igem.org/Team:Goettingen/Engagement">Engagement</A></LI></DIV></DIV><DIV class="sidebar"><H3 style="text-align: center;">Sponsors</H3></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></BODY></HTML>