Team:Goettingen/Notebook
Toggle navigation
Home
Team
Team
Collaborations
Project
Description
Experiments
Proof of Concept
Results
Notebook
Parts
Parts
Basic Parts
Safety
Attributions
Human Practices
Human Practices
HP on the Road
HP on the Web
Engagement
Notebook
In the meaning of open science, we want you to be able to follow any single step we did in the lab. Come and see what we did during the whole year.
 
December
Wet Lab work has not yet started. It will start in April.
Dry Lab work has not yet started.
15/12/15:
Organization:
We had our first team meeting and started with introducing us to each other. To get an overview over our background, every team member shortly described their bachelor topics. We also had a discussion on time requirements decided that team members should only take one core module during the summer term. Furthermore, it was suggested to organize lab work in shifts. We also exchanged first ideas for possible projects and organized a meeting with Professor Stülke who wanted to present a possible project.
15/12/21:
Organization:
Professor Stülke and members of his lab shortly introduced a possible project from his department based on the idea of a minimal organism. After the meeting, the team members discussed the suggestion. We decided to ask other professors for different project ideas, before choosing a topic.
January
Wet Lab work has not yet started.  It will start in April.
Research:
on Vitamin B12 binding proteins, Vitamin B12 dependent reactions and organisms
Facebook:
Update of the Facebook page of last year’s iGEM team Göttingen by publishing our first team picture taken in the lab
Logo and Title:
collecting project titles
designing a logo for our project which will also be printed on our team t-shirts
16/01/08:
Organization:
Several team members introduced additional project ideas based on former projects and own ideas. Tobias S. agreed on asking Professor Daniel, who supervised the iGEM team from 2015, for a different project.
16/01/12:
Organization:
Tobias S. introduced the project proposed by Professor Daniel. The project would aim at constructing a synthetic Vitamin B12 exporter and testing it in different production organisms. We also discussed advantages of this project idea compared to others.
16/01/13:
Organization:
We decided on the project “Vitamin B12 exporter” proposed by Professor Daniel. All team members voted in favor of the project.
16/01/18:
Organization:
We had the first team meeting with our supervisors discussing a rough timeline with important deadlines.
We also agreed on collecting title ideas and searching literature for Vitamin B12 binding proteins, Vitamin B12 dependent reactions and production organisms for the next meeting.
Financials:
We started planning the project funding by sponsors and internal university funding.
16/01/21:
Organization:
After having collected many different project titles during a very funny meeting, the majority of our team voted for “B12 Synporter- Another brick in the wall”.
February
Wet Lab work has not yet started.  It will start in April.
Booklet:
Design of booklet
Writing texts on iGEM, our project idea, the team and sponsoring categories
16/02/01:
Financials:
We decided to design and print a booklet on our project to send it to possible sponsors. Amongst others, information on the iGEM competition, the background to our project, the team and sponsoring suggestions should be included. The topics were divided between team members. The organization team agreed on writing the email text and the text for a letter to address sponsors.
March
Wet Lab work has not yet started.  It will start in April.
Research:
On details about our chosen production organisms and Vitamin B12 binding proteins
possible Vitamin B12 assays
For Wet Lab:
ordering the production strains from the DMSZ
16/03/03:
Dry lab:
Together with the supervisors, we decided on the production organisms R. planticola and S. blattae to test the constructs for Vitamin B12 exporters. We did not yet choose different B12 binding proteins, which can be used in the exporter. Further literature research is needed for that. However, we already discussed possible experiments to test the B12 productivity in the production organisms in the end. Those include a photometric, a biological and an antibody assay. The strains for our production organisms have to be ordered from DSMZ. For a biological assay we also have to search for a B12 auxotrophic strain and a B12-free medium.
16/03/18:
Financials:
We registered the team for the IDT website to use their sponsoring for iGEM teams to synthesize our constructs.
16/03/29:
Dry lab:
After consulting our supervisors, we decided on three different constructs (GlmS, BtuF and MutB) to design our synthetic exporter.
April
Week 1: GH, TS, TK
Antibiotic resistance:
After having received our production strains S. blattae and R. planticola we could start lab work with testing them for natural antibiotic resistances. For this, they were grown in culture overnight and plated on LB-Agar plates supplemented with different antibiotics.
Week 2: TK, GH, LK, IF, MS, TS
Antibiotic resistance:
Due to inconsistent results in the week before, the test for antibiotic resistances had to be repeated. We additionally tested for resistance against chloramphenicol and used more different concentrations of the antibiotics and dilutions of the pre-cultures. The results showed, that all production strains have no antibiotic resistance for kanamycin. It can therefore be used as antibiotic resistance on our plasmids.
Bacterial identification:
To check the identification of our production strains, we performed a 16S rRNA PCR. All strains were correct, so glycerol stocks could be prepared.
Competent cells:
For the following experiments, we had to prepare electro competent cells of our productions strains and E. coli DH5α. This was done first for E. coli DH5α according to protocol.
Week 3: IF, NE, ME, MS, TK
Competent cells:
Electrocompetent cells for S. blattae and R. planticola were prepared.
 Week 4: IF, LK
Test transformation:
To check if our prepared electro competent cells were indeed electro competent, we transformed them with a simple vector containing a kanamycin resistance according to protocol and grew them on plates over night. Unfortunately, only E. coli DH5α showed resistance against kanamycin, suggesting that the transformation or the preparation of competent cells was not successful for the others.
Research:
rough outline of our project and the main experiments
on the TAT secretion system and export signals for it
on vector systems that can be induced by arabinose to easily express our vectors
For Wet Lab:
decision on pBAD202 D-TOPO as expression vector and ordering it from Thermo Fisher
design of the constructs and codon-optimization of the sequences for coli
Human practices:
collecting interesting and scientific facts about Vitamin B12 and introducing each team member with their favourite fact on Facebook
16/04/01:
Wet lab:
We planned the first steps, which needed to be taken in the lab. We have to grow the strains, make the cells electro competent, proof their identity via 16S PCR and check for possible antibiotic resistances.
Dry lab:
More research is needed on the TAT secretion system and possible export signals, as well as possible vector systems. It was decided that they should be inducible by arabinose, suggesting the pBAD vector system.
16/04/11:
Wet lab:
After discussing the first results from the lab concerning the antibiotic resistances, it was decided to repeat the experiments with chloramphenicol as additional antibiotic and using different concentrations.
For the following weaks, the team is devided into three groups, one testing antibiotic resistance again, one starting to prepare electro competent cells and the third team designing the constructs.
Organization:
We agreed on having a regular iGEM meeting with the supervisors every two weeks on Monday, 11 am. Furthermore, the organization team will start writing guide lines for lab work and Ines will organize a log book for signing in lab working hours.
16/04/25:
Wet lab:
The electro competence of our cells will be tested with a simple kanamycin vector.
To dissolve the synthesized constructs from IDT, we will use either a 10 mM Tris Buffer or an elution buffer from a purification Kit before making aliquots.
Dry Lab:
If we also want to use Pseudomonas denitrificans as a production organism, we need to search for the specific name and ask Ute, if the strain is available in the lab.
Tobias S. is in contact with Thermo Fisher for our order of the pBAD202 D-TOPO vector.
Financials:
The sponsoring by B. Braun was confirmed. We can also start sending letters to possible sponsors because the etiquettes for the envelopes are prepared.
Organization:
We might try to find a bachelor student, who wants to join our project after the “Bio & Brezeln” presentation. They would need to send a letter with a written application to the “Prüfungskommission” to get credits for participating.
Human practice:
The “Bio & Brezeln” presentation will be held by Tobias S., Tobias K. and Miriam.
Miriam will also contact the “Göttinger Tageblatt” and ask them to write an article about our participation in the iGEM competition.
Events:
We intend to participate in the meetup of German iGEM teams in Marburd. Tobias S. will organize the registration. Furthermore, we want to present our team at the “Göttinger Altstadtlauf” and the sports competition “DIES” from our university.
May
Week 1:
No wet lab work done.
Week 2: IF, LK, NE, TS, GH
Test transformation:
The previously prepared cells of our production strains were again tested for electro competence. The protocol was slightly changed using SOC medium instead of LB-medium and incubating the bacteria at their temperature optima (R. planticola: 30 °C, others: 37 °C). However, the cells again did not grow, showing that they were not electro competent.
Week 3: NE, TK, GH
Preparation:
For the following week, large amounts of media had to be prepared.
 Week 4: NE, GH, MS, AB
Competent cells:
The preparation of competent cells was repeated for S. blattae and R. planticola. The cells were again incubated at their respective temperature maxima.
For Wet Lab:
ordering the designed constructs for the Vitamin B12 synporter from IDT, who sponsor iGEM teams with synthesizing DNA sequences
design of primers, which can be used to amplify the constructs that were synthesized by IDT
Human practice
preparation of the “Bio & Brezeln” presentation which was held on the 26th of May
T-Shirt:
designing our team T-shirts, which we will wear for example at presentations and during the Giant Jamboree in Boston
Website:
developing a website based on WordPress, which gives information about our project, the idea behind it and the iGEM competition in general
16/05/09:
Wet lab:
The transformations need to be repeated, because they did not work. We might need to adjust the protocol.
Additionally, the competent cells need to be checked for electro competence.
Dry lab:
We will use snapgene to design our vectors and primers.
Human practice:
For the Master Information Day at our university we need to prepare a poster to present our project and the iGEM competition to future master students.
Furthermore, we need to work on the presentation for the “Bio & Brezeln” event at the 26th of May.
Financials:
We were sponsored by Zymo Research with Kits and T-Shirts. To thank them, we will take a group picture with the shirts and post it on our facebook profile.
Events:
We decided to participate in the meeting of the European iGEM teams in Paris (16/06/01-16/06/03) and intend to book the hotel rooms organized by the iGEM teams from Paris and bus tickets.
16/05/18:
Wet lab:
The vector from Thermo Fisher has arrived, so we could start with inserting our constructs. The work on electro competent cells, test transformations and ligation of the MutB construct should be finished by 6th of june.
Human practice:
We intend to present our project and iGEM at the “GZMB Sommerfest”. In return, we will offer our support in organizing the event.
Organization:
We discussed the final design for our team T-Shirts and possible offers for printing them.
16/05/30:
Dry lab:
Genis will help us to design the first primers and give us an introduction into the snapgene program.
Wet lab:
The newly prepared electro competent cells need to be tested. If necessary, we will repeat the preparation.
Human practice:
We need to continue working on our poster for the Master Information Day.
Organization:
For our website, we want to create a new figure of our “Synporter”.
June
Week 1: IF, LK, TK, GH, ME, AB
Test transformation:
The newly prepared electro competent cells were tested and compared with a test transformation of cells provided by our supervisor Genis. Only the S. blattae cells showed antibiotic resistance.
MutB construct:
After having received the synthesized constructs from IDT, we could start to prepare the constructs. MutB had to be ordered with a restriction site and synthesized in two fragments due to its length. Therefore, it had to be digested with HindIII, purified and ligated.
Week 2:
No wet lab work done.
Week 3: AB, ME
Competent cells:
The preparation of electro competent cells was repeated for R. planticola and test transformations with a simple kanamycin vector were successful.
MutB construct:
The ligation of the two fragments was repeated. Additionally, the synthesized MutB1 and MutB2 fragments were amplified by PCR, followed by gel electrophoresis and gel extraction to prepare a stock.
Week 4: AB, ME
MutB construct:
The amplified MutB fragments were digested with HindIII, purified and ligated. However, gel electrophoresis showed that none of the ligations worked yielding products of wrong sizes.
Human practices:
designing and writing texts for the poster, which was presented at the Master Information Day on the 26th of June
participating in the “Göttinger Altstadtlauf”
During June, no regular team meetings were held. Most of our team took part in a practical course of our master program. However, we kept informed about the progress in the lab.
July
Week 1: AB, ME
MutB construct:
Restriction, purification, ligation and gel electrophoresis had to be repeated, but the results were again negative.
GlmS and BtuF constructs:
The synthesized constructs were cloned into the pBAD202 D-TOPO vector.  In contrast to the suggested protocol, we used only half of the amount of the vector. The transformation was performed according to protocol into TOP10 E. coli. The resulting colonies were transferred into liquid culture and incubated over night. From the grown cultures, plates for storage were prepared and plasmid preparations were done.
Week 2: AB, ME
MutB construct:
Due to the difficulties of ligating the two fragments of MutB, we tried a different approach and prepared fragments with overhangs by PCR using newly ordered primers. After gel electrophoresis and purification, the fragments were directly ligated.
BtuF construct:
To check if BtuF was correctly inserted into pBAD202 it was digested with SmaI. The gel after electrophoresis did not show any bands.
Week 3: ME, TS, TK, MS
MutB construct:
To find out if the ligation was successful, a PCR to recover the fragments was done with the respective primers.
GlmS construct:
To check if GlmS was correctly inserted into pBAD202 it was digested with AccI. The gel after electrophoresis did not show any bands.
Week 4: TS, TK, MS, AB
MutB construct:
Due to problems with ligating the two MutB fragments with help of overhangs, we tried another approach. For this, the fragments were prepared with blunt ends via by PCR using respective primers. After gel electrophoresis and gel extraction the PCR products were then ligated via blunt-end ligation. However, the gel did again not show bands for products of the right size.
Week 5: TS, TK, MS, AB
MutB construct:
PCR, gel extraction and ligation of MutB blunt end fragments was repeated. To increase the amount of DNA, the product was amplified by PCR. Additionally, we tried to phosphorylate the MutB2 blunt end fragment before ligation.
For Wet Lab:
designing primers for MutB1 and MutB2 with blunt ends
Collaboration:
Patrick from the iGEM Team TU Darmstadt introduced their project to us and we discussed possible collaborations.
Human practices:
participating in the meetup of European iGEM teams in Paris presenting our poster and exchanging ideas with other iGEM teams
16/07/06:
Dry Lab:
Our binding proteins might only be partly loaded. We could check this quantitatively through Western blot using the epitope tag or the His-Tag.
Giant Jamboree:
We started looking for flights to and from Boston and discussed hotel options. Everyone has to do the ESTA registration to be allowed to travel to the USA.
Financials:
At the beginning of August we will check how much money is available for the Giant Jamboree.
Human practice:
We want to participate at the “GZMB Sommerfest” and will contact the responsible persons. Additionally, we will try to develop a concept for a school project and might organize interviews with experts.
Cooperation:
The iGEM team Leuven is working on an interesting project and we might be able to cooperate with them.
16/07/11:
Wet Lab work:
We reported the general laboratory progress to the team members who had not been working in the lab for some time and to our supervisors. The ligation of MutB was still not successful and we discussed further approaches including the introduction of blunt ends to the two MutB fragments.
We were also informed that the -80 °C freezer with our electrocompetent cells broke down during the weekend which means, that we have to check if our electrocompetent cells still work.
August
Week 1: TK, AB, MS, GH, ME
MutB construct:
The ligation from last week was purified and analyzed by PCR and gel electrophoresis. However, we again did not get a product with the right size.
GlmS and BtuF constructs:
Cloning of the two constructs into the pBAD202 according to the D-TOPO protocol was repeated, followed by transformation of TOP10 cells. After growing the cells, plasmid preparations were performed.
Week 2: ME, MS, GH, AB, TK
GlmS and BtuF constructs:
The plasmids from last week were digested with respective restriction enzymes and new liquid cultures for both constructs were prepared. But none of the prepared plasmids showed the right restriction pattern after digestion.
MutB construct:
MutB1 blunt end was prepared by PCR and gel extraction and amplified by PCR. Additionally, MutB1 and MutB2 were again generated with overhangs by PCR. The PCR was not successful, so we tried another approach by following the protocol for overhang generation used by Harrison et al.
Week 3: AB, TK, GH, MS, TS, LK
GlmS and BtuF constructs:
Cloning of GlmS and BtuF into pBAD202 and transformation into TOP10 cells had to be repeated. The transformation was then plated on plates with minimal M9 medium, but no colonies grew.
MutB construct:
The generation of MutB1 and MutB2 with overhangs and MutB1 with blunt end was repeated. Only MutB1+blunt end could be produced. Additionally, a repetition of the PCR of MutB2 with the phosphorylated MutB2 was done, but MutB2 could not be amplified.
Week 4: GH, LK, ME, TK, TS, IF, MS, AB
MutB constructs:
The MutB+blunt end DNA was amplified by PCR, but the control gel was showed negative results.
Preperation:
Our LB-Kan and M9-Kan plates were tested with a transformation of PLP03 in DH5α and compared with plates provided by the lab. After negative results, our plates had to be discarded but plasmid preparation could be performed from clones of the control plates. We prepared new plates for us.
GlmS and BtuF constructs:
The constructs could be successfully amplified with the newly designed primers and GlmS and BtuF in pBAD202 were transformed into TOP10. The transformation for GlmS was successful and plasmid preparation could be performed.
For Wet Lab:
designing primers for MutB fragments with overhangs
designing primers for GlmS and BtuF that exclude the thioredoxin sequence of the pBAD202 vector
controlling the restriction pattern for pBAD202, GlmS, BtuF and MutB
designing improved primers for MutB1 and MutB2
ordering new MutB constructs from IDT
Collaboration/Human practices:
designing a postcard for the Human practice, which was initiated by the iGEM Team Düsseldorf
presenting our project and iGEM during the GZMB symposium at our university
participating in the meetup of all German iGEM teams in Marburg introducing our project and exchanging ideas with other teams
16/08/08:
Events:
The meetup of the German iGEM teams in Marburg was a good experience and successful. We might cooperate with Bielefeld (on aptamers) and Darmstadt (on biosafety).
Wet lab:
Next steps are the transformation of BtuF and GlmS and sequencing afterwards to confirm the correct orientation. We then will continue with expression tests.
For MutB the new primers with overhangs have arrived for the new approach of ligating the two fragments.
Giant Jamboree:
The team members need to register for the Giant Jamboree in Boston until the 31st of August. Our planned travel time is from 16/10/25 to 16/11/02.
16/08/22:
Wet lab:
GlmS and BtuF have to amplified with the new primers and sequenced. In addition to that new plated need to be prepared.
Genis pointed out that the primers in MutB might overlap to much.
Giant Jamboree:
The registration fee has to be paid by the team members for now, but we might get an advance of travel costs. We also discussed options for flights and hotels and talked about organizational things.
Financials:
We will try to contact Sartorius again and ask them for sponsoring.
September
Week 1: AB, ME, MS, GH, LK
GlmS construct:
A control of the GlmS plasmids was done by digestion and the constructs were additionally sent in for sequencing. All had the right sequence.
BtuF construct:
Liquid cultures of BtuF colonies in LB and RM medium were prepared and plasmid preparations were performed. The plasmid from the RM medium could be digested with SmaI, but the gel showed only an empty vector.
MutB construct:
The MutB fragments were amplified with the improved primers by PCR and the PCR products were purified and digested with BamHI. The ligation afterwards was successful and a gel extraction was performed.
Competent cells:
New electro competent cells for E. coli DH5α were prepared.
Week 2: LK, GH, AB, TK, ME
MutB construct:
We tried to amplify the ligated MutB, but the amplification did not work. Therefore, the ligation was repeated. PCR proved the successful ligation. MutB was then amplified by PCR and purified by gel extraction.
GlmS construct:
GlmS was transformed into TOP10 and E. coli LMG194 cells. The TOP10 cells could afterwards be grown in RM medium for the following expression study which was done according to the pBAD202 D-TOPO manual. Additionally, the GlmS construct was modified excluding the thioredoxin sequence (GlmS w/o Thio) and amplified by PCR.
BtuF construct:
The transformation of TOP10 with BtuF was unsuccessful and had to be repeated. After DNA preparation, the product was digested with SmaI.
Test transformation:
The newly prepared E. coli DH5α cells were checked for electro competence and the preparation of the kanamycin vector PLP03 worked.
E. coli LMG194:
Electro competent cells of E. coli LMG194 were prepared and tested by transformation with kanamycin vector.
Week 3: GH, LK, IF, MS
MutB construct:
MutB in pET101 was transformed into TOP10, the DNA prepared and sent for sequencing.
GlmS construct:
A Western Blot of the expression study was done. Additionally, a Colony PCR of the transformed cells from last week was performed to check that the construct is not empty and afterward glycerol stocks could be prepared. The rest of the colonies were grown in LB medium with kanamycin and Vitamin B12.
BtuF construct:
BtuF in pET101 was transformed into TOP10 and the DNA prepared.
BtuF and GlmS w/o Thio constructs:
The constructs without thioredoxin were purified by gel extraction and transformed in pBAD202 into TOP10 and the DNA was prepared.
MutB w/o Thio construct:
The PCR for the preparation of MutB without thioredoxin was also repeated but again did not work.
MutB, BtuF and GlmS constructs:
To all three constructs a CACC(ATT) stop codon was added by PCR.
Week 4: GH, LK, IF, ME, TS, AB, MS
MutB, BtuF and GlmS constructs:
All constructs were amplified with new primers for the vector pBAD18. After gel extraction, double digestion and purification, GlmS and BtuF could be ligated in pBAD18. The constructs were then transformed into TOP10. For BtuF, no plasmids could be detected after plasmid preparation.
MutB construct:
pET101 with MutB was digested as a control and could be transformed into TOP10. After preparation, the DNA was sent in for sequencing.
MutB constructs from the lab:
Due to our difficulties with acquiring a correct MutB construct, we received old constructs from the lab. A plasmid preparation was performed for the samples PTM3A, PTM3B, PTML and PMB5.
BtuF construct:
pET101 with BtuF was digested as a control.
GlmS construct:
GlmS in pBAD202 was transformed into S. blattae and R. planticola. After plasmid preparation, only the plasmid from S. blattae could be digested and a glycerol stock prepared. For samples from R. planticola no plasmids could be detected.
BtuF and GlmS w/o Thio constructs:
pBAD202 with the respective constructs was digested as a control and three of the samples were analyzed through expression studies in E.coli TOP10. Expression of BtuF w/o Thio could be shown.
Western Blot that shows bands at the expected size of TorA-BtuF w/o Thio protein in E. coli Top10.
The samples were additionally sent in for sequencing.
GlmS w/o Thio in pBAD202 was transformed into S. blattae and R. planticola.
The constructs were also ligated into the pBAD18 vector, transformed into TOP10 and plated on LB plates with ampicillin and vitamin B12.
B12 Assay:
Guttman medium for the following B12 Assays was prepared.
Week 5: GH, LK, IF, ME, TS, AB, MS
Competent cells:
New electrocompetent cells of R. planticola and S. blattae were prepared.
BtuF and GlmS w/o Thio constructs:
Sequences of constructs in R. planticola were all positive.
BtuF in pBAD202 was transformed in R. planticola. The transformation was successful; colonies were picked the next day.
Expression studies were performed for BtuF in S. blattae and GlmS in S. blattae and R. planticola.
MutB constructs from the lab:
The constructs PTM6, PTM3A and PTM3B were transformed in S. blattae and DH5&alpha, PMB5 was transformed in R. planticola and DH5&alpha and colonies were picked the next day. From the resulting liquid cultures, glycerol stocks were prepared and plasmid preparations were performed.
MutB, GlmS and BtuF constructs:
The constructs were amplified with new primers to add restriction sites for cloning in the iGEM vector pSB1C3.
B12-Assay::
A test run of the assay was successfully performed using only defined concentrations of B12.
Successful expression of TorA-BtuF w/o Thio protein in E. coli Top10.
Western Blot that shows bands at the expected size of TorA-BtuF w/o Thio protein in E. coli Top10.
For Wet Lab:
designing primers to insert the iGEM prefix and suffix into our constructs
designing primers to add CACC(ATT) stop codon to our constructs
discussing the pBAD18 vector as a possible new vector, but it has an ampicillin resistance instead of a kanamycin resistence
designing primers to insert our constructs into the pBAD18 vector, which includes restriction sites for EcoRI and XbaI and also a His-Tag sequence at the N-terminus
Human practices:
developing a survey about synthetic biology and Vitamin B12 that was posted on facebook
designing a stop motion video about our project
participating in the Annual DECHEMA Conference in Aachen and giving a presentation
16/09/05:
Wet lab:
In the following weeks, lab work needs to be divided into different teams.
MutB and GlmS were successfully subcloned and can be transformed into the arabiniose negative E. coli strain for expression experiments.
It also needs to be tested if the thioredoxin fusion protein in the pBAD202 vector influences the expression of our Symporter protein.
Financials:
We talked about the travelling costs for Boston and are happy, that we get supported by Sartorius as new sponsor.
Human practice:
The postcards from the Human practice action initiated by the iGEM team Düsseldorf have to be distributed in public.
We also will present our project results in a colloquia at our institute at the beginning of the next semester.
16/09/12:
Wet lab:
We reported the general laboratory progress to our supervisors. The next steps are the expression studies of our constructs in TOP10 and analysing them by SDS-PAGE and Western Blot. However, for the proof of concept we will also need to do expression studies and western blot of our constructs in the production strains and Vitamin B12
Human practice:
We also discussed the participation at the biotechnology conference in Aachen to which we were invited.
16/09/21:
Wet lab:
The B12 detection tests have to be started. The strains will be cultivated according to protocol. For the tests minimal medium has to be prepared and the periplasmic space will be extracted by ultracentrifugation.
For GlmS the constructs without thioredoxin need to be checked by repeating the SDS-PAGE of the expression experiments.
Financials:
The travel expense accounting for Boston was done.
Human practice:
For the Wiki, texts about our human practices need to be written.
In addition to that, we might contact the schools again, organize a “lab work” event for students and arrange a BBQ for the new master students.
October
Week 1: GH, LK, IF, ME, TS, AB, MS
BtuF and GlmS w/o Thio constructs:
The expression studies for BtuF in S. blattae and GlmS in S. blattae and R. planticola were analyzed with SDS page and western blot. Expression of BtuF in S. blattae could be shown (see figure 4). An expression study of BtuF in R. planticola was performed. Analysis with SDS page and western blot showed a successful expression (see figure 4).
Plasmid preparations of the liquid cultures from the transformation of BtuF in R. planticola were performed.
A control digestion of BtuF in R. planticola was performed and successful for all tested clones.
Collaboration with Team TU Darmstadt:
Transformations of S. blattae with MiniColactive, MiniColmut and pSB1A3(empty vector) were performed. All transformations were successful, the colonies on each plate were counted.
GlmS and BtuF constructs:
GlmS and BtuF were cloned in pSB1C3 and transformed in E. coli TOP10. Plasmid preparations were performed and the plasmids were sequenced.
MutB constructs from the lab:
Expression studies were performed for PMB5 in R. planticola as well as for PTM3A, PTM3B and PTM6 in S. blattae. Analysis with SDS page and western blot showed expression of PMB5 in R. planticola.
B12-Assay:
R. planticola with Glms w/o Thio as well as with BtuF w/o Thio were cultured and induced with arabinose. After harvesting, fractions of cytoplasm, spheroplasm and periplasm were separated. Samples of the fractions were submitted to the B12 plate assay.
Western Blot of our synporter proteins. Both blots show the expression of BtuF and GlmS  in R. planticola, and BtuF in S. blattae
.
Week 2: IF, ME, LK, GH, AB, MS, TK
MutB constructs from the lab:
PMB-Amp was transformed in S. blattae and PMB-Kan was transformed in R. planticola. All MutB constructs were checked for restriction sites that may interfere with the ones present in iGEM prefix and suffix. It was concluded that there were too many to continue the work with these constructs.
B12-Assay:
The plates showed no growth of E. coliDSM4261 where the samples were added. Instead the area around the samples were lighter than the rest of the plates. This might be due to the presence of buffer P in the samples killing the E. coli cells. Therefore, in all following B12-assays, a desalting of the samples was suggested.
S. blattae with Glms w/o Thio as well as with BtuF w/o Thio were cultured anaerobically and induced with arabinose. Samples before and after induction were analyzed by western blot. Expression could be detected for BtuF and GlmS (see figure 4). After harvesting, fractions of cytoplasm, spheroplasm and periplasm were separated. Samples of the fractions were submitted to the B12 plate assay as well as to SDS page and western blot. BtuF could be detected in the spheroplasm and the periplasm (REFERENCE to WB 14.10.). In the plate assay, no B12 could be detected.
The empty vector pBAD202D/lacZ was transformed in E. coli TOP10 to be used as a negative control. Colonies were picked and plasmid preparations of the resulting liquid cultures were performed. Then a transformation of pBAD202D/lacZ in R. planticola and S. blattae was performed.
MutB construct:
A PCR to amplify the ligated MutB was performed, but no bands were visible on the analyzing agarose gel.
TorA-RFP construct:
Biobrick BBa_E1010 from the distribution kit (**highly** engineered mutant of red fluorescent protein from Discosoma striata) was transformed in E. coli TOP10. Colonies were picked and a plasmid preparation of the resulting liquid cultures was performed. TorA was amplified via PCR and restriction sites for cloning into pSB1C3 were added. The product was purified by agarose gel extraction. TorA and RFP in pSB1C3 were ligated into pSB1A3 according to the “3A Assembly” protocol from iGEM. The assembly product was transformed into E. coli TOP10.
GlmS and BtuF constructs:
GlmS and BtuF in pSB1C3 were prepared for shipment.
Week 3: GH, LK, MS, TS, AB, ME, IF, TS
TorA-RFP construct:
Colonies from the TOP 10 transformation were picked and the resulting liquid cultures were analyzed with fluorescence microscopy. No expression of RFP could be detected.
B12-Assay and expression test:
S. blattae and R. planticola with Glms w/o Thio as well as with BtuF w/o Thio were cultured anaerobically and induced with arabinose. Samples before and after induction were analyzed by western blot. After harvesting, fractions of cytoplasm, spheroplasm and periplasm were separated. Samples of the fractions were submitted to the B12 plate assay and the photometric assay as well as to SDS page and western blot. The same was done for aerobic cultures of S. blattae and R. planticola with Glms w/o Thio as well as with BtuF w/o Thio. These cultures were grown with B12 present in the medium.
Both Western Blots show the expression of BtuF and GlmS  in R. planticola, and BtuF in S. blattae
.
The plate assay gave no evaluable results. In the photometric assay, B12 could be detected in the cytoplasm and periplasm of R. planticola and S. blattae. Additionally, the western blots showed expression of BtuF in the periplasm of R. planticola and S. blattae.
Human practices:
developing an informative presentation about studying in general, synthetic biology, iGEM and our project for students of the Grotefend Gymnasium in Hann. Münden
16/10/11:
Giant Jamboree:
We met to plan our presentation for the Giant Jamboree. After swapping ideas, we came to an agreement and are now on our way to prepare the presentation.
Furthermore, we talked about the dress code for the day of our presentation and some further organizational stuff according our trip to Boston.
Wet Lab
Dry Lab  and Organisation
Team Meetings
The ones of us who participated in the respective work: AB , GH , IF , LK , ME , MS , NE , TK , TS .
Contact
  Facebook
  Twitter
  E-Mail
  Homepage
  Address
iGEM Team Göttingen 2016
Institut für Mikrobiologie
und Genetik
AG Prof. Dr. Rolf Daniel
Grisebachstr. 8
37077 Göttingen
Navigation
Home
Team
Team
Collaborations
Project
Description
Experiments
Proof of Concept
Results
Notebook
Parts
Parts
Basic Parts
Safety
Attributions
Human Practices
Human Practices
HP on the Road
HP on the Web
Engagement
Sponsors
