<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_Goettingen_Experiments skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><DIV id="top_title"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:Goettingen/Experiments</SPAN></H1></DIV><DIV id="HQ_page"><DIV id="bodyContent"><NAV class="navbar navbar-inverse"><DIV class="container-fluid"><DIV class="navbar-header"><BUTTON type="button" class="navbar-toggle collapsed" data-toggle="collapse" data-target="#bs-example-navbar-collapse-1" aria-expanded="false"><SPAN class="sr-only">Toggle navigation</SPAN></BUTTON></DIV><DIV class="collapse navbar-collapse" id="bs-example-navbar-collapse-1"><UL class="nav navbar-nav"><LI><A href="https://2016.igem.org/Team:Goettingen">Home</A></LI><LI class="dropdown"><A href="#" class="dropdown-toggle" data-toggle="dropdown" role="button" aria-haspopup="true" aria-expanded="false">Team </A></LI><LI><A href=" https://2016.igem.org/Team:Goettingen/Team"> Team   </A></LI><LI><A href="https://2016.igem.org/Team:Goettingen/Collaborations"> Collaborations </A></LI></UL><LI class="dropdown"><A href="#" class="dropdown-toggle" data-toggle="dropdown" role="button" aria-haspopup="true" aria-expanded="false">Project </A></LI><LI><A href="https://2016.igem.org/Team:Goettingen/Description"> Description </A></LI><LI><A href="https://2016.igem.org/Team:Goettingen/Experiments"> Experiments </A></LI><LI><A href="https://2016.igem.org/Team:Goettingen/Proof"> Proof of Concept </A></LI><LI><A href="https://2016.igem.org/Team:Goettingen/Results"> Results </A></LI><LI><A href="https://2016.igem.org/Team:Goettingen/Notebook"> Notebook </A></LI><LI class="dropdown"><A href="#" class="dropdown-toggle" data-toggle="dropdown" role="button" aria-haspopup="true" aria-expanded="false">Parts </A></LI><LI><A href="https://2016.igem.org/Team:Goettingen/Parts">Parts </A></LI><LI><A href="https://2016.igem.org/Team:Goettingen/Basic_Part"> Basic Parts </A></LI><LI><A href="https://2016.igem.org/Team:Goettingen/Safety">Safety</A></LI><LI><A href="https://2016.igem.org/Team:Goettingen/Attributions">Attributions</A></LI><LI class="dropdown"><A href="#" class="dropdown-toggle" data-toggle="dropdown" role="button" aria-haspopup="true" aria-expanded="false">Human Practices </A></LI><LI><A href="https://2016.igem.org/Team:Goettingen/Human_Practices"> Human Practices </A></LI><LI><A href="https://2016.igem.org/Team:Goettingen/HP/Silver"> HP on the Road </A></LI><LI><A href="https://2016.igem.org/Team:Goettingen/HP/Gold"> HP on the Web </A></LI><LI><A href="https://2016.igem.org/Team:Goettingen/Engagement"> Engagement </A></LI></DIV></DIV></NAV><DIV id="pageContent"><DIV class="row"><DIV class="col-md-12"><DIV class="row"><DIV class="col-md-9"><DIV class="heading"><H1> Experiments </H1></DIV><DIV class="row"><DIV class="col-md-12"><DIV class="text"><H2>Overview of Experiments</H2></DIV><DIV class="text" id="exp1"><H2>1. Production Strains</H2>

Following strains were cultivated:

<UL><LI><I>Escherichia coli</I> DH5alpha: <A href="https://www.dsmz.de/catalogues/details/culture/DSM-6897.html?tx_dsmzresources_pi5%5BreturnPid%5D=304"><B>DSMZ</B></A></LI><LI><I>Raoultella planticola</I> DSM 3069 (ATCC 33531): <A href="https://www.dsmz.de/catalogues/details/culture/DSM-3069.html?tx_dsmzresources_pi5%5BreturnPid%5D=304"><B>DSMZ</B></A></LI><LI><I>Shimwelia blattae</I> DSM 4481 (ATCC 29907) : <A href="https://www.dsmz.de/catalogues/details/culture/DSM-4481.html?tx_dsmzresources_pi5%5BreturnPid%5D=304"><B>DSMZ</B></A></LI></UL><P>In order to ensure that the bacteria we are working with are really the ones we want to work with, a 16S rDNA PCR was performed.</P><P>As an antibiotic resistance will be used in our later experiments to select transformants, we checked for any native antibiotic resistance against ampicillin, kanamycin, tetracyclin and chloramphenicol in an LB agar plate growth test. It turned out that kanamycin was the only antibiotic, against which none of our strains showed any resistance. For this reason, we decided to use only vectors with kanamycin resistance for our later projects.</P><P>After this, we prepared electro competent cells from our strains. All of our strains were grown on LB medium.</P></DIV><DIV class="text" id="exp2"><H2>2. BioBricks</H2><P>Our BioBricks were designed using the software SnapGene. The original Genes for our B<SUB>12</SUB> binding proteins, and the <I>torA</I> signal sequence were codon optimized for <I>E. coli</I>.</P><P>The DNA containing the Genes were synthesized and friendly provided by IDT Integrated DNA Technologies.</P><P>In case of <I>torA-mutB</I>, the length of the gene exceeded the maximum length of 2000 bp from IDT. For this reason, we designed two DNA parts, with a natural occurring <I>Hind</I>III restriction site at both ends. We intended to fuse these both genes by ligation. However, after having diffuculties to fuse the genes by ligation, we decided to fuse them by fusion PCR. In order to multiply the DNA for our cloning experiments, and to equip our genes with restriction sites, PCR was used.</P><P>For our purpose, we were searching for a vector with (a) kanamycin resistance and (b) araBAD operon. We thus decided to use the pBAD202 expression system. This vector, however, is equipped with a thioredoxin fusion protein which increases the rate of expression, but prohibits the physiological function of our protein. For this reason, we designed primers for an autarkic Synporter protein. In order to test the dependence of the expression system on the B<SUB>12</SUB> export rate, we also decided to use different other vectors.</P><P>For vector ligation, we used a standart ligase reaction, for the pBAD202 expression, which works without ligation but topoisomerase reaction, we followed the manufacturer's protocol.</P><DIV class="caption"><P><B>Fig. 1:</B>Our Synporter genes  <I>torA-btuF</I> and <I>torA-glmS</I> cloned into the pBAD202 plasmid, separated from the thioredoxin fusion protein.</P></DIV></DIV><DIV class="text" id="exp3"><H2>3. Transformation and Cultivation</H2><P>For our transformation experiments, we used electroporation for all our self prepared competent cells. For the heat competent <I>E. coli</I> Top10 cells provided with the pBAD202 expression system, we used the manufacturer's heat shock transformation protocol.</P><P>For vector multiplication,  <I>E. coli</I> cells (Top10 for pBAD202 plasmids, DH5alpha for all other plasmids) were transformed with the constructed plasmids. After the transformation, the cells were cultivated on LB Medium. Transformants were selected, and the respective clones were picked, cultivated in LB medium, and plasmids were extracted.</P><P>The Plasmids were then introduced into the production strains <I>Raoultella planticola</I> ATCC 33531 and <I>Shimwellia blattae</I> ATCC 29907.</P><P>For the expression experiments, the all cells were cultivated and the expression induced in RM medium under aerobic conditions. As <I>S. blattae</I> and <I>R. planticola</I> are known to produce Vitamin B<SUB>12</SUB> only under anaerobic conditions, the expression was induced under anerobic conditions in MM medium. </P><P>The induction was performed using different concentrations of <SPAN style="font-variant: small-caps;">l</SPAN>-arabinose and different induction time spans.</P></DIV><DIV class="text" id="exp4"><H2>4. Expression Test</H2><P>To test for the expression of our Synporter protein, the cell lysates of the expression cultures we submitted to an SDS-PAGE. From the SDS Gels, Western Blots were performed using anti-Poly-Histidine-tag antibodies.</P><P>In order to test the translocation of our Synporter proteins, the cells were fractionated into Periplasm, Spheroplasts, and Cytoplsma (gained by French Press cell disruption) as a control.


</P></DIV><DIV class="text" id="exp5"><H2>5. Vitamin B<SUB>12</SUB> Assays</H2><P>To test the physiological function of our Synporter protein, different assays for B<SUB>12</SUB> detection were performed. For the assays, the periplasm and cytoplasm fractions of the cells were examined.</P><P>The periplasm fraction of the producing cells was gained by centrifugation.</P><H3>5.1. Microbial Assay</H3><P>The microbial B<SUB>12</SUB> assay relies on the growth diameter of the <I>E. coli</I> DSM4261 strain on an agar plate, in dependence of the Vitamin B<SUB>12</SUB> (or methionine) concentration in the medium.</P><P>For this purpose, agar plates with synthetic Guttman medium (free of B<SUB>12</SUB> and amino acids, 1.5% agar-agar) were prepared.
The <I>E. coli</I> detection cells were grown in liquid synthetic Guttman medium supplemented with selected amino acids (no methionine) and Vitmamin B<SUB>12</SUB>. Afterwards, the cells were washed with PBS buffer three times, in order to make the cells virtually free of Vitamin B<SUB>12</SUB>. The cells were resuspended in PBS buffer, and mixed with warm, liquid Guttman medium (free of B<SUB>12</SUB> and amino acids, 0.8% agar-agar), and then poured on the prepared agar plates. For the assay, small paper platelets (usually used for antibiotic tests) were placed on the top agar layer, and soaked with differently concentrated calibration solutions of Vitamin B<SUB>12</SUB>. The plates were incubated overnight, and the diameters measured after 16 h. From the different diameters, dependent of the Vitamin B<SUB>12</SUB> concentration, a calibration curve was calculated.</P><P>In order to determine the B<SUB>12</SUB> content of our periplasm and cytoplasm fractions, the assay plates were prepared exactly as described above, and paper platelets were soaked with the respective solution. Comparing to the calibration curve, the diameter indicates the B<SUB>12</SUB> amount.</P><H3>5.1. Photometric Assay</H3><P>In order to determine the Vitamin B<SUB>12</SUB> concentration of a sample photometrically, the corrinoids in our samples needed to be converted into dicyano cobalamine. This was achieved by treating the samples with sodium cyanide or potassium cyanide. Other components like proteins and lipids which would distort the measurements, can easily be separated by precipitation through acidic and basic conditions as well as heating and subsequent centrifugation.</P><P>After measuring a spectrum of the supernatant from 350 to 650 nm, the Vitamin B<SUB>12</SUB> concentration could be calculated with help of the extinction coefficients of dicyano cobalamine.</P></DIV></DIV></DIV><DIV class="row"><DIV class="col-md-12"><DIV class="text"><H2>Protocols</H2><DIV class="panel panel-default"><A data-toggle="collapse" data-parent="#accordion" href="#1-0"><DIV class="panel-heading"><H4 class="panel-title">
                			Media
                		</H4></DIV></A><DIV id="1-0" class="panel-collapse collapse"><DIV class="panel-body"><DIV class="tab-content"><DIV id="1-0" class="tab-pane fade in active"><DIV class="panel panel-default"><A data-toggle="collapse" data-parent="#accordion" href="#1-1"><DIV class="panel-heading"><H5 class="panel-title">
                                            			LB Medium
                                            		</H5></DIV></A><DIV id="1-1" class="panel-collapse collapse"><DIV class="panel-body"><DIV class="tab-content"><DIV id="1-1" class="tab-pane fade in active"><DIV class="table-responsive"><TABLE class="table1" style="width:100%; max-width:400px"><TBODY><TR style="font-weight:bold;"><TH style="font-size:80%;">Component</TH><TH style="font-size:80%;">Amount</TH></TR><TR><TD>Tryptone</TD><TD>10 g</TD></TR><TR><TD>NaCl</TD><TD>10 g</TD></TR><TR><TD>Yeast extract</TD><TD>5 g</TD></TR><TR><TD colspan="3">optional:</TD></TR><TR><TD>Agar-agar</TD><TD>15 g</TD></TR></TBODY></TABLE></DIV><UL><LI>Add components to ddH<SUB>2</SUB>O and adjust volume to 1 L.</LI><LI>Autoclave for 20 min at 121°C.</LI></UL></DIV></DIV></DIV></DIV></DIV><DIV class="panel panel-default"><A data-toggle="collapse" data-parent="#accordion" href="#1-2"><DIV class="panel-heading"><H5 class="panel-title">
                                            			M9 Minimal Medium
                                            		</H5></DIV></A><DIV id="1-2" class="panel-collapse collapse"><DIV class="panel-body"><DIV class="tab-content"><DIV id="1-2" class="tab-pane fade in active"><DIV class="table-responsive"><TABLE class="table1" style="width:100%; max-width:400px"><TBODY><TR style="font-weight:bold;"><TH style="font-size:80%;">Component</TH><TH style="font-size:80%;">Amount</TH></TR><TR><TD>Na<SUB>2</SUB>HPO<SUB>4</SUB></TD><TD>10 g</TD></TR><TR><TD>KH<SUB>2</SUB>PO<SUB>4</SUB></TD><TD>10 g</TD></TR><TR><TD>NH<SUB>4</SUB>Cl</TD><TD>5 g</TD></TR><TR><TD>NaCl</TD><TD>15 g</TD></TR></TBODY></TABLE></DIV><UL><LI>Add components to ddH<SUB>2</SUB>O, adjust the pH to 7.0 and bring the volume to 987 mL.</LI><LI>Autoclave for 20 min at 121°C.</LI><LI>Add the following</LI></UL><DIV class="table-responsive"><TABLE class="table1" style="width:100%; max-width:400px"><TBODY><TR style="font-weight:bold;"><TH style="font-size:80%;">Component</TH><TH style="font-size:80%;">Stock solution</TH><TH style="font-size:80%;">Amount</TH></TR><TR><TD>Glucose</TD><TD>20%</TD><TD>10 mL</TD></TR><TR><TD>MgSO<SUB>4</SUB>· 7 H<SUB>2</SUB>O</TD><TD>1 M</TD><TD>1 mL</TD></TR><TR><TD>Thiamine · HCl</TD><TD>30 mM</TD><TD>1 mL</TD></TR><TR><TD>CaCl<SUB>2</SUB></TD><TD>0.13 M</TD><TD>1 mL</TD></TR></TBODY></TABLE></DIV></DIV></DIV></DIV></DIV></DIV><DIV class="panel panel-default"><A data-toggle="collapse" data-parent="#accordion" href="#1-3"><DIV class="panel-heading"><H5 class="panel-title">
                                            			RM Medium
                                            		</H5></DIV></A><DIV id="1-3" class="panel-collapse collapse"><DIV class="panel-body"><DIV class="tab-content"><DIV id="1-3" class="tab-pane fade in active"><P>The dehydrated medium is distributed by Laboratories Conda (Cat. 1542) and was prepared according to protocol.</P></DIV></DIV></DIV></DIV></DIV><DIV class="panel panel-default"><A data-toggle="collapse" data-parent="#accordion" href="#1-4"><DIV class="panel-heading"><H5 class="panel-title">
                                            			MM medium
                                            		</H5></DIV></A><DIV id="1-4" class="panel-collapse collapse"><DIV class="panel-body"><DIV class="tab-content"><DIV id="1-4" class="tab-pane fade in active"><P><U>Mineral medium with additon of yeast extract</U></P><DIV class="table-responsive"><TABLE class="table1" style="width:100%; max-width:400px"><TBODY><TR style="font-weight:bold;"><TH style="font-size:80%;">Component</TH><TH style="font-size:80%;">Amount</TH></TR><TR><TD>K<SUB>2</SUB>HPO<SUB>4</SUB></TD><TD>14 g</TD></TR><TR><TD>KH<SUB>2</SUB>PO<SUB>4</SUB></TD><TD>6 g</TD></TR><TR><TD>(NH<SUB>4</SUB>)<SUB>2</SUB>SO<SUB>4</SUB></TD><TD>3 g</TD></TR><TR><TD>MgSO<SUB>4</SUB>· 7 H<SUB>2</SUB>O</TD><TD>0.2 g</TD></TR><TR><TD>Yeast extract</TD><TD>0.2 g</TD></TR><TR><TD>CoCl<SUB>2</SUB> (50 mM)</TD><TD>85 µL</TD></TR><TR><TD>SL-4</TD><TD>1 mL</TD></TR></TBODY></TABLE></DIV><UL><LI>Add components to ddH<SUB>2</SUB>O, adjust the pH to 7.5 and bring the volume to 1 L.</LI><LI>100 mM glycerine are added seperatly to the medium.</LI></UL><P><U>SL-4: Trace element solution (Pfenning and Lippert, 1966)</U></P><DIV class="table-responsive"><TABLE class="table1" style="width:100%; max-width:400px"><TBODY><TR style="font-weight:bold;"><TH style="font-size:80%;">Component</TH><TH style="font-size:80%;">Amount</TH></TR><TR><TD>EDTA-Na<SUB>2</SUB></TD><TD>5 g</TD></TR><TR><TD>FeSO<SUB>4</SUB>· 7 H<SUB>2</SUB>O</TD><TD>2 g</TD></TR><TR><TD>ZnSO<SUB>4</SUB>· 7 H<SUB>2</SUB>O</TD><TD>0.1 g</TD></TR><TR><TD>MnCl<SUB>2</SUB>· 4 H<SUB>2</SUB>O</TD><TD>0.03 g</TD></TR><TR><TD>H<SUB>3</SUB>BO<SUB>4</SUB></TD><TD>0.3 g</TD></TR><TR><TD>CoCl<SUB>2</SUB>· 6 H<SUB>2</SUB>O</TD><TD>0.2 g</TD></TR><TR><TD>CuCl<SUB>2</SUB>· 2 H<SUB>2</SUB>O</TD><TD>0.01 g</TD></TR><TR><TD>NiCl<SUB>2</SUB>· 6 H<SUB>2</SUB>O</TD><TD>0.02 g</TD></TR><TR><TD>NaMoO<SUB>4</SUB>· 2 H<SUB>2</SUB>O</TD><TD>0.03 g</TD></TR></TBODY></TABLE></DIV><UL><LI>Add components to ddH<SUB>2</SUB>O, adjust the pH to 6.7 and bring the volume to 1 L.</LI><LI>The solution is stored at 4°C and protected from light.</LI></UL></DIV></DIV></DIV></DIV></DIV><DIV class="panel panel-default"><A data-toggle="collapse" data-parent="#accordion" href="#1-5"><DIV class="panel-heading"><H5 class="panel-title">
                                            			Guttman Medium (B<SUB>12</SUB> Assay)
                                            		</H5></DIV></A><DIV id="1-5" class="panel-collapse collapse"><DIV class="panel-body"><DIV class="tab-content"><DIV id="1-5" class="tab-pane fade in active"><DIV class="table-responsive"><TABLE class="table1" style="width:100%; max-width:400px"><TBODY><TR style="font-weight:bold;"><TH style="font-size:80%;">Component</TH><TH style="font-size:80%;">Amount</TH></TR><TR><TD>K<SUB>2</SUB>HPO<SUB>4</SUB></TD><TD>14 g</TD></TR><TR><TD>KH<SUB>2</SUB>PO<SUB>4</SUB></TD><TD>6 g</TD></TR><TR><TD>(NH<SUB>4</SUB>)<SUB>2</SUB>SO<SUB>4</SUB></TD><TD>2 g</TD></TR><TR><TD>MgSO<SUB>4</SUB>· 7 H<SUB>2</SUB>O</TD><TD>0.2 g</TD></TR><TR><TD>Na<SUB>2</SUB>citrate · 2 H<SUB>2</SUB>O</TD><TD>1 g</TD></TR><TR><TD>L-Asparagine</TD><TD>0.8 g</TD></TR><TR><TD>L-Arginine</TD><TD>0.2 g</TD></TR><TR><TD>L-Glutamic acid</TD><TD>0.2 g</TD></TR><TR><TD>Glycine</TD><TD>0.2 g</TD></TR><TR><TD>L-Tryptophan</TD><TD>0.2 g</TD></TR><TR><TD>L-Prolin</TD><TD>0.2 g</TD></TR><TR><TD>L-Histidine</TD><TD>0.2 g</TD></TR><TR><TD>Vitamin B<SUB>12</SUB></TD><TD>40 µg</TD></TR></TBODY></TABLE></DIV><UL><LI>Add components to ddH<SUB>2</SUB>O, adjust the pH to 6.8-7.2 and bring the volume to 1 L.</LI><LI>50 mM glucose have to be added seperatly before the inoculation.</LI></UL></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV><DIV class="panel panel-default"><A data-toggle="collapse" data-parent="#accordion" href="#2-0"><DIV class="panel-heading"><H4 class="panel-title">
                			Cultivation and Transformation
                		</H4></DIV></A><DIV id="2-0" class="panel-collapse collapse"><DIV class="panel-body"><DIV class="tab-content"><DIV id="2-0" class="tab-pane fade in active"><DIV class="panel panel-default"><A data-toggle="collapse" data-parent="#accordion" href="#2-1"><DIV class="panel-heading"><H5 class="panel-title">
                                            			Cultivation of <I>Escherichia coli</I></H5></DIV></A><DIV id="2-1" class="panel-collapse collapse"><DIV class="panel-body"><DIV class="tab-content"><DIV id="2-1" class="tab-pane fade in active"><P>For preperation of competent cells and cloning, the <EM>E. coli</EM> strains DH5α and TOP10 are grown in LB medium at 37°C.</P><P>For expression studies of constructs in pBAD202, <EM>E. coli </EM>TOP cells are grown in LB medium supplemented with 1 mg/mL Vitamin B<SUB>12</SUB> and 50 µg/mL kanamycin.</P></DIV></DIV></DIV></DIV></DIV><DIV class="panel panel-default"><A data-toggle="collapse" data-parent="#accordion" href="#2-2"><DIV class="panel-heading"><H5 class="panel-title">
                                            			Cultivation of <I>Raoultella planticola</I></H5></DIV></A><DIV id="2-2" class="panel-collapse collapse"><DIV class="panel-body"><DIV class="tab-content"><DIV id="2-2" class="tab-pane fade in active"><P>For preperation of competent cells and cloning, <EM>R. platicola</EM> is grown in LB medium at its temperature optimum of 30°C.</P><P>For expression studies of constructs in pBAD202, <EM>R. planticola</EM> is grown under anaerobic conditions in MM medium supplemented with 50 µg/mL kanamycin.</P></DIV></DIV></DIV></DIV></DIV><DIV class="panel panel-default"><A data-toggle="collapse" data-parent="#accordion" href="#2-3"><DIV class="panel-heading"><H5 class="panel-title">
                                            			Cultivation of <EM>Shimwellia blattae</EM></H5></DIV></A><DIV id="2-3" class="panel-collapse collapse"><DIV class="panel-body"><DIV class="tab-content"><DIV id="2-3" class="tab-pane fade in active"><P>For preperation of competent cells and cloning, the <EM>S. blattae</EM> is grown in LB medium at 37°C.</P><P>For expression studies of constructs in pBAD202, <EM>S. blattae</EM> was grown under anaerobic conditions in MM medium supplemented with 50 µg/mL kanamycin.</P></DIV></DIV></DIV></DIV></DIV><DIV class="panel panel-default"><A data-toggle="collapse" data-parent="#accordion" href="#2-4"><DIV class="panel-heading"><H5 class="panel-title">
                                            			Preparation of Electrocompetent Cells
                                            		</H5></DIV></A><DIV id="2-4" class="panel-collapse collapse"><DIV class="panel-body"><DIV class="tab-content"><DIV id="2-4" class="tab-pane fade in active"><P class="protocols_text"><STRONG>Material to prepare</STRONG></P><UL class="protocols_text"><LI>2x 5 mL LB</LI><LI>250 mL LB in 1 L Erlenmeyer flask with chicane</LI><LI>500 mL sterile Millipore-H<SUB>2</SUB>O at 4°C</LI><LI>50 mL sterile glycerol (10% (w/v)) at 4°C</LI><LI>30 sterile labeled Eppendorf cups at – 20°C or -80°C</LI><LI>1x sterile GS3 jar at 4°C</LI><LI>1x Aquatron at 30°C</LI><LI>1x Sorvall RC6 centrifuge at 4°C</LI><LI>1x Universal 320R centrifuge at 4°C</LI><LI>1 L liquid nitrogen</LI></UL><P> </P><P class="protocols_text"><STRONG>Day 1</STRONG></P><UL class="protocols_list"><LI class="protocols_list">Inoculate 2x 5 mL LB (if required with antibiotics) with your strain of interest, either from cryo culture or from agar plate.</LI><LI class="protocols_list">Incubate overnight at 37°C and 150 rpm.</LI></UL><P> </P><P class="protocols_text"><STRONG>Day 2</STRONG></P><UL class="protocols_text"><LI class="protocols_list">Inoculate 250 mL LB (without antibiotics) with 2% (5 mL) preculture.</LI><LI class="protocols_list">Incubate in the Aquatron at 30°C and 160 rpm until OD<SUB>600</SUB> of 0.5-0.8.</LI><LI class="protocols_list">Check the culture via microscope for contaminations.</LI></UL><P class="protocols_text1">All liquids and containers must be cooled on ice. The major task of cell preparation is the removal of salts. In case of some strains like pLys-strains, the pellets must be resuspended very carefully.</P><UL class="protocols_text"><LI class="protocols_list">Let the cells (OD<SUB>600</SUB> of 0.5-0.8) cool down in ice water for 10-20 min. All further steps are performed under cool conditions.</LI><LI class="protocols_list">Decant the cells in sterile GS3 jar and centrifuge at 4°C and 5000 rpm (4230 g) for 5-10 min.</LI><LI class="protocols_list">Remove supernatant (directly next to the centrifuge or the pellet might resolve) and resuspend the pellet in 250 mL sterile 4°C ddH<SUB>2</SUB>O.</LI><LI class="protocols_list">Centrifuge the GS3 jar at 4°C and 5000 rpm (4230 g)for 5-10 min.</LI><LI class="protocols_list">Remove supernatant (directly next to the centrifuge or the pellet might resolve) and resuspend the pellet in 250 mL sterile 4°C ddH<SUB>2</SUB>O.</LI><LI class="protocols_list">Centrifuge the GS3 jar at 4°C and 5000 rpm (4230 g)for 5-10 min.</LI><LI class="protocols_list">Remove supernatant (directly next to the centrifuge or the pellet might resolve) and resuspend the pellet in 10 mL sterile 4°C glycerol (10% (w/v)). Afterwards, transfer the culture into a 50 mL Falcon tube.</LI><LI class="protocols_list">Centrifuge the Falcon tube at 4°C and 6200 rpm (4230 g) for 5-10 min.</LI><LI class="protocols_list">Remove supernatant (directly next to the centrifuge or the pellet might resolve) and resuspend the pellet in 10 mL sterile 4°C glycerol (10% (w/v)).</LI><LI class="protocols_list">Centrifuge the Falcon tube at 4°C and 6200 rpm (4230 g) for 5-10 min.</LI><LI class="protocols_list">Remove supernatant (directly next to the centrifuge or the pellet might resolve) and resuspend the pellet in 500 µL sterile 4°C glycerol (10% (w/v)).</LI><LI class="protocols_list">Aliquot the cells into the Eppendorf cups (40 µL per Eppendorf cup). While aliquoting, filled Eppendorf cups must be frozen directly in liquid nitrogen.</LI><LI class="protocols_list">Store the filled Eppendorf cups in a cryobox at -80°C.</LI></UL></DIV></DIV></DIV></DIV></DIV><DIV class="panel panel-default"><A data-toggle="collapse" data-parent="#accordion" href="#2-5"><DIV class="panel-heading"><H5 class="panel-title">           			Transformation of Electrocompetent Cells by Electroporation
                                            		</H5></DIV></A><DIV id="2-5" class="panel-collapse collapse"><DIV class="panel-body"><DIV class="tab-content"><DIV id="2-5" class="tab-pane fade in active"><UL class="protocols_list"><LI class="protocols_list">Mix 50-200 ng plasmid DNA with 40 µL of competent cells (always defrost on ice!). Transfer attempt into precooled electroporation cuvette. Incubate on ice for 10 min.</LI><LI class="protocols_list">Important: <STRONG>DNA must be salt free!</STRONG> Use either directly in ddH<SUB>2</SUB>O eluted plasmids, or (e.g. directly after ligation) desalt the DNA for 30 min on a Millipore filter (MF Membrane Filters).</LI></UL><UL class="protocols_list"><LI class="protocols_list">Electroporation: 2500 V for 2 mm cuvettes (1250 V for 1 mm cuvettes), 25 µF, 200 Ω, discharging time should be 3-5 msec. Contacts of the electroporation cuvette must be dry. Remove all air bubbles before electroporation.</LI><LI class="protocols_list">Immediately add 960 µL liquid LB (or, if available, SOC medium for a higher transformation efficiency) into the cuvette and transfer the content into a 2 ml Eppendorf cup.</LI><LI class="protocols_list">Incubate at 37°C and 150 rpm for 1 h. Fix tube in horizontal position with tape.</LI><LI class="protocols_list">Plate 100 µL of 10<SUP>-3</SUP> to 10<SUP>-6</SUP> dilutions on plates with selection pressure</LI><LI class="protocols_list">Incubate at 37°C overnight.</LI></UL></DIV></DIV></DIV></DIV></DIV><DIV class="panel panel-default"><A data-toggle="collapse" data-parent="#accordion" href="#2-6"><DIV class="panel-heading"><H5 class="panel-title">
                                                                                			Transformation of Chemically Competent Cells by Heat-Shock
                                                                                		</H5></DIV></A><DIV id="2-6" class="panel-collapse collapse"><DIV class="panel-body"><DIV class="tab-content"><DIV id="2-6" class="tab-pane fade in active"><P>One Shot<SUP>®</SUP>TOP10 Competent cells (ThermoFisher) were transformed following the respective protocol for chemical transformation:</P><UL><LI>Add 3 µL of the TOPO<SUP>®</SUP>Cloning Reaction into a vial of competent <EM>E. coli</EM> cells and mix gently by inverting.</LI><LI>Incubate on ice for 5-30 min.</LI><LI>Heat-shock the cells for 30 s at 42°C without shaking.</LI><LI>Immediately transfer the tubes on ice.</LI><LI>Add 250 µL of S.O.C. medium (RT), cap the tube tightly and shake it horizontally (200 rpm) at 37°C for 1 h.</LI><LI>Spread 100-200 µL of different volumes or dilutions from each transformation on a prewarmed selective plate and incubate overnight at 37°C.</LI><LI>Pick colonies for analysis.</LI></UL></DIV></DIV></DIV></DIV></DIV><DIV class="panel panel-default"><A data-toggle="collapse" data-parent="#accordion" href="#2-7"><DIV class="panel-heading"><H5 class="panel-title">
                                            			Plasmid preparation
                                            		</H5></DIV></A><DIV id="2-7" class="panel-collapse collapse"><DIV class="panel-body"><DIV class="tab-content"><DIV id="2-7" class="tab-pane fade in active"><P>Small scale preparation of plasmid DNA was performed using the NucleoSpin<SUP>®</SUP> Plasmid kit distributed by Macherey-Nagel (REF 740588) according to protocol.</P></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV><DIV class="panel panel-default"><A data-toggle="collapse" data-parent="#accordion" href="#3-0"><DIV class="panel-heading"><H4 class="panel-title">
                			PCR methods
                		</H4></DIV></A><DIV id="3-0" class="panel-collapse collapse"><DIV class="panel-body"><DIV class="tab-content"><DIV id="3-0" class="tab-pane fade in active"><DIV class="panel panel-default"><A data-toggle="collapse" data-parent="#accordion" href="#3-1"><DIV class="panel-heading"><H5 class="panel-title">
                                            			PCR with Phusion DNA Polymerase
                                            		</H5></DIV></A><DIV id="3-1" class="panel-collapse collapse"><DIV class="panel-body"><DIV class="tab-content"><DIV id="3-1" class="tab-pane fade in active"><P>Phusion<SUP>®</SUP> High-Fidelity DNA Polymerase is used for all standard PCR amplifications.</P><P><U>Pipetting scheme</U></P><DIV class="table-responsive"><TABLE class="table1" style="width:100%; max-width:600px"><TBODY><TR style="font-weight:bold;"><TH style="font-size:80%;">Component</TH><TH style="font-size:80%;">20 µL reaction</TH><TH style="font-size:80%;">50 µL reaction</TH><TH style="font-size:80%;">Final concentration</TH></TR><TR><TD>Nuclease-free water</TD><TD>to 20 µL</TD><TD>to 50 µL </TD></TR><TR><TD>5X Phusion HF buffer</TD><TD>4 µL</TD><TD>10 µL</TD><TD>1X</TD></TR><TR><TD>10 mM dNTPs</TD><TD>0.4 µL</TD><TD>1 µL</TD><TD>200 µM</TD></TR><TR><TD>10 µM forward primer</TD><TD>1 µL</TD><TD>2.5 µL</TD><TD>0.5 µM</TD></TR><TR><TD>10 µM reverse primer</TD><TD>1 µL</TD><TD>2.5 µL</TD><TD>0.5 µM</TD></TR><TR><TD>Template DNA</TD><TD>variable</TD><TD>variable</TD><TD>1 pg - 10 ng</TD></TR><TR><TD>DMSO (optional)</TD><TD>(0.6 µL)</TD><TD>(1.5 µL)</TD><TD>3%</TD></TR><TR><TD>Phusion DNA Polymerase</TD><TD>0.2 µL</TD><TD>0.5 µL</TD><TD>0.02 U/µL</TD></TR></TBODY></TABLE></DIV><UL><LI>Mix all components, adding the DNA polymerase last and spin down.</LI></UL><P><U>Cycling</U></P><DIV class="table-responsive"><TABLE class="table1" style="width:100%; max-width:600px"><TBODY><TR style="font-weight:bold;"><TH style="font-size:80%;">Step</TH><TH style="font-size:80%;">Temperature</TH><TH style="font-size:80%;">Time</TH><TH style="font-size:80%;">Cycles</TH></TR><TR><TD>Initial Denaturation</TD><TD>98°C</TD><TD>30 s</TD><TD>1</TD></TR><TR><TD>Denaturation</TD><TD>98°C</TD><TD>10 s</TD><TD rowspan="3">25-35</TD></TR><TR><TD>Annealing</TD><TD>X°C</TD><TD>30 s</TD></TR><TR><TD>Extension</TD><TD>72°C</TD><TD>15-30 s/kb</TD></TR><TR><TD>Final Extension</TD><TD>72°C</TD><TD>10 min</TD><TD>1</TD></TR><TR><TD>4°C</TD><TD>hold</TD></TR></TBODY></TABLE></DIV><UL><LI>To optimize the annealing temperature for each primer pair, a temperature gradient can be run.</LI></UL></DIV></DIV></DIV></DIV></DIV><DIV class="panel panel-default"><A data-toggle="collapse" data-parent="#accordion" href="#3-2"><DIV class="panel-heading"><H5 class="panel-title">
                                            			Fusion PCR
                                            		</H5></DIV></A><DIV id="3-2" class="panel-collapse collapse"><DIV class="panel-body"><DIV class="tab-content"><DIV id="3-2" class="tab-pane fade in active"><P>Fusion PCR was performed according to Harrison <EM>et al.</EM></P><P><U>Pipetting scheme</U></P><DIV class="table-responsive"><TABLE class="table1" style="width:100%; max-width:600px"><TBODY><TR style="font-weight:bold;"><TH style="font-size:80%;">Component</TH><TH style="font-size:80%;">50 µL reaction</TH><TH style="font-size:80%;">Final concentration</TH></TR><TR><TD>Nuclease-free water</TD><TD>to 50 µL </TD></TR><TR><TD>5X Phusion HF buffer</TD><TD>10 µL</TD><TD>1X</TD></TR><TR><TD>10 mM dNTPs</TD><TD>1 µL</TD><TD>200 µM</TD></TR><TR><TD>50 mM MgCl<SUB>2</SUB></TD><TD>1.25 µL</TD><TD>1.25 mM</TD></TR><TR><TD>10 µM forward primer</TD><TD>1 µL</TD><TD>0.2 µM</TD></TR><TR><TD>10 µM reverse primer</TD><TD>1 µL</TD><TD>0.2 µM</TD></TR><TR><TD>Template DNA</TD><TD>variable</TD></TR><TR><TD>DMSO</TD><TD>1.5 µL</TD><TD>3%</TD></TR><TR><TD>Phusion DNA Polymerase</TD><TD>0.5 µL</TD><TD>0.02 U/µL</TD></TR></TBODY></TABLE></DIV><UL><LI>Mix all components, adding the DNA polymerase last and spin down.</LI></UL><P><U>Cycling</U></P><DIV class="table-responsive"><TABLE class="table1" style="width:100%; max-width:600px"><TBODY><TR style="font-weight:bold;"><TH style="font-size:80%;">Step</TH><TH style="font-size:80%;">Temperature</TH><TH style="font-size:80%;">Time</TH><TH style="font-size:80%;">Cycles</TH></TR><TR><TD>Initial Denaturation</TD><TD>98°C</TD><TD>1 min</TD><TD>1</TD></TR><TR><TD>Denaturation</TD><TD>98°C</TD><TD>10 s</TD><TD rowspan="3">4</TD></TR><TR><TD>Annealing</TD><TD>68 ± 4°C</TD><TD>30 s</TD></TR><TR><TD>Extension</TD><TD>72°C</TD><TD>40 s</TD></TR><TR><TD>Denaturation</TD><TD>98°C</TD><TD>10 s</TD><TD rowspan="3">27</TD></TR><TR><TD>Annealing</TD><TD>68 ± 4°C</TD><TD>30 s</TD></TR><TR><TD>Extension</TD><TD>72°C</TD><TD>50 s</TD></TR><TR><TD>Final Extension</TD><TD>72°C</TD><TD>10 min</TD><TD>1</TD></TR><TR><TD>4°C</TD><TD>hold</TD></TR></TBODY></TABLE></DIV><UL><LI>To optimize the annealing temperature for each primer pair, a temperature gradient can be run.</LI></UL></DIV></DIV></DIV></DIV></DIV><DIV class="panel panel-default"><A data-toggle="collapse" data-parent="#accordion" href="#3-3"><DIV class="panel-heading"><H5 class="panel-title">                                  			16S rDNA Colony PCR
                                            		</H5></DIV></A><DIV id="3-3" class="panel-collapse collapse"><DIV class="panel-body"><DIV class="tab-content"><DIV id="3-3" class="tab-pane fade in active"><P>16S rDNA PCR was performed to check the identity of the bacterial strains we were working with. For this, colonies were picked and resuspended in 50 µL ddH<SUB>2</SUB>O.</P><P><U>Pipetting scheme</U></P><DIV class="table-responsive"><TABLE class="table1" style="width:100%; max-width:600px"><TBODY><TR style="font-weight:bold;"><TH style="font-size:80%;">Component</TH><TH style="font-size:80%;">Amount</TH></TR><TR><TD>Nuclease-free water</TD><TD>to 50 µL </TD></TR><TR><TD>Optibuffer</TD><TD>5 µL</TD></TR><TR><TD>10 mM dNTPs</TD><TD>5 µL</TD></TR><TR><TD>5 mM forward primer (16S08)</TD><TD>4 µL</TD></TR><TR><TD>5 mM reverse primer (150UR)</TD><TD>4 µL</TD></TR><TR><TD>Template DNA</TD><TD>1 µL (suspension)</TD></TR><TR><TD>MgCl<SUB>2</SUB></TD><TD>3 µL</TD></TR><TR><TD>BIO-X-ACT DNA Polymerase</TD><TD>0.5 µL</TD></TR></TBODY></TABLE></DIV><UL><LI>Mix all components, adding the DNA polymerase last and spin down.</LI></UL></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV><DIV class="panel panel-default"><A data-toggle="collapse" data-parent="#accordion" href="#4-0"><DIV class="panel-heading"><H4 class="panel-title">
                			Plasmid Construction
                		</H4></DIV></A><DIV id="4-0" class="panel-collapse collapse"><DIV class="panel-body"><DIV class="tab-content"><DIV id="4-0" class="tab-pane fade in active"><DIV class="panel panel-default"><A data-toggle="collapse" data-parent="#accordion" href="#4-1"><DIV class="panel-heading"><H5 class="panel-title">
                                            			Digestion of DNA with Restriction Enzymes
                                            		</H5></DIV></A><DIV id="4-1" class="panel-collapse collapse"><DIV class="panel-body"><DIV class="tab-content"><DIV id="4-1" class="tab-pane fade in active"><P>Plasmid DNA and PCR products were digested according to FastDigest protocol (Thermo Scientific).</P><DIV class="table-responsive"><TABLE class="table1" style="width:100%; max-width:600px"><TBODY><TR style="font-weight:bold;"><TH style="font-size:80%;">Component</TH><TH style="font-size:80%;">Plasmid DNA</TH><TH style="font-size:80%;">PCR product</TH></TR><TR><TD>Water, nuclease free</TD><TD>15 µL</TD><TD>17 µL</TD></TR><TR><TD>10X FastDigest or 10X FastDigest Green buffer</TD><TD>2 µL</TD><TD>2 µL</TD></TR><TR><TD>DNA</TD><TD>2 µL (up to 1 µg)</TD><TD>10 µL (~0.2 µg)</TD></TR><TR><TD>FastDigest enzyme</TD><TD>1 µL</TD><TD>1 µL</TD></TR></TBODY></TABLE></DIV><UL><LI>Mix gently and spin down.</LI><LI>Incubate at 37°C in a heat block or water thermostat for X min.</LI><LI>Inactivate the enzyme by heating for X min at X°C.</LI></UL><P>Comment: Incubation time and inactivation time and temperature depend on the applied FastDigest enzyme and were done according to the respective protocol.</P><UL><LI>If the FastDigest Green Buffer was used in the reaction, an aliquot of the reaction mixture can be loaded directly on an agarose gel.</LI></UL></DIV></DIV></DIV></DIV></DIV><DIV class="panel panel-default"><A data-toggle="collapse" data-parent="#accordion" href="#4-2"><DIV class="panel-heading"><H5 class="panel-title">
                                            			Phosphorylation of DNA
                                            		</H5></DIV></A><DIV id="4-2" class="panel-collapse collapse"><DIV class="panel-body"><DIV class="tab-content"><DIV id="4-2" class="tab-pane fade in active"><P>MutB2 DNA was phosphorylated with T4 Polynucleotide Kinase (T4 PNK) according to protocol (NEB, #M0201).</P><DIV class="table-responsive"><TABLE class="table1" style="width:100%; max-width:500px"><TBODY><TR style="font-weight:bold;"><TH style="font-size:80%;">Component</TH><TH style="font-size:80%;">Amount</TH></TR><TR><TD>DNA</TD><TD>up to 300 pmol of 5' termini</TD></TR><TR><TD>T4 PNK Reaction Buffer (10X)</TD><TD>5 µL</TD></TR><TR><TD>ATP (10 mM)</TD><TD>5 µL</TD></TR><TR><TD>T4 PNK</TD><TD>1 µL (10 units)</TD></TR><TR><TD>Nuclease-free water</TD><TD>up to 50 µL</TD></TR></TBODY></TABLE></DIV><UL><LI>Prepare reaction mixture on ice.</LI><LI>Incubate at 37°C for 30 min.</LI><LI>Heat inactivate at 65°C for 20 min.</LI></UL></DIV></DIV></DIV></DIV></DIV><DIV class="panel panel-default"><A data-toggle="collapse" data-parent="#accordion" href="#4-3"><DIV class="panel-heading"><H5 class="panel-title">
                                            			Ligation
                                            		</H5></DIV></A><DIV id="4-3" class="panel-collapse collapse"><DIV class="panel-body"><DIV class="tab-content"><DIV id="4-3" class="tab-pane fade in active"><P>Ligations were either done with T4 DNA ligase distributed by Invitrogen (Cat. No. 15224-017) or by Thermo Scientific (#EL0014).</P><P><U>Invitrogen </U></P><UL><LI>Prepare the following reaction mixture and spin briefly.</LI></UL><DIV class="table-responsive"><TABLE class="table1" style="width:100%; max-width:500px"><TBODY><TR style="font-weight:bold;"><TH style="font-size:80%;">Component</TH><TH style="font-size:80%;">Amount/concentration</TH></TR><TR><TD>5X ligase buffer</TD><TD>4 µL</TD></TR><TR><TD>vector DNA</TD><TD>3-30 fmol</TD></TR><TR><TD>insert DNA</TD><TD>9-90 fmol</TD></TR><TR><TD>T4 DNA ligase</TD><TD>0.1 U</TD></TR><TR><TD>ddH<SUB>2</SUB>O, autoclaved</TD><TD>to 20 µL</TD></TR></TBODY></TABLE></DIV><UL><LI>Incubate at 16°C over night.</LI><LI>Heat-inactivate at 65°C for 10 min.</LI></UL><P><U>Thermo Scientific </U></P><UL><LI>Prepare the following reaction mixture and spin briefly.</LI></UL><DIV class="table-responsive"><TABLE class="table1" style="width:100%; max-width:500px"><TBODY><TR style="font-weight:bold;"><TH style="font-size:80%;">Component</TH><TH style="font-size:80%;">Amount/concentration</TH></TR><TR><TD>10X ligase buffer</TD><TD>2 µL</TD></TR><TR><TD>vector DNA</TD><TD>20-100 ng</TD></TR><TR><TD>insert DNA</TD><TD>1:1 to 5:1 molar ratio over vector</TD></TR><TR><TD>T4 DNA ligase</TD><TD>1 U</TD></TR><TR><TD>ddH<SUB>2</SUB>O, autoclaved</TD><TD>to 20 µL</TD></TR></TBODY></TABLE></DIV><UL><LI>Incubate at RT for 1 h or at 16°C over night.</LI><LI>Heat-inactivate at 65°C for 10 min.</LI></UL></DIV></DIV></DIV></DIV></DIV><DIV class="panel panel-default"><A data-toggle="collapse" data-parent="#accordion" href="#4-4"><DIV class="panel-heading"><H5 class="panel-title">
                                            			Blunt End Ligation
                                            		</H5></DIV></A><DIV id="4-4" class="panel-collapse collapse"><DIV class="panel-body"><DIV class="tab-content"><DIV id="4-4" class="tab-pane fade in active"><P>Blunt end ligations were done with T4 DNA ligase distributed by Thermo Scientific (#EL0014).</P><UL><LI>Prepare the following reaction mixture and spin briefly.</LI></UL><DIV class="table-responsive"><TABLE class="table1" style="width:100%; max-width:500px"><TBODY><TR style="font-weight:bold;"><TH style="font-size:80%;">Component</TH><TH style="font-size:80%;">Amount/concentration</TH></TR><TR><TD>10X ligase buffer</TD><TD>2 µL</TD></TR><TR><TD>vector DNA</TD><TD>20-100 ng</TD></TR><TR><TD>insert DNA</TD><TD>1:1 to 5:1 molar ratio over vector</TD></TR><TR><TD>50% PEG 4000 solution</TD><TD>2 µL</TD></TR><TR><TD>T4 DNA ligase</TD><TD>5 U</TD></TR><TR><TD>ddH<SUB>2</SUB>O, autoclaved</TD><TD>to 20 µL</TD></TR></TBODY></TABLE></DIV><UL><LI>Incubate at RT for 1 h.</LI><LI>Heat-inactivate at 65°C for 10 min.</LI></UL></DIV></DIV></DIV></DIV></DIV><DIV class="panel panel-default"><A data-toggle="collapse" data-parent="#accordion" href="#4-5"><DIV class="panel-heading"><H5 class="panel-title">
                                            			pBAD Directional TOPO<SUP>®</SUP> Cloning
                                            		</H5></DIV></A><DIV id="4-5" class="panel-collapse collapse"><DIV class="panel-body"><DIV class="tab-content"><DIV id="4-5" class="tab-pane fade in active"><P>The D-TOPO<SUP>®</SUP> cloning reaction of pBAD202 was performed according to protocol (Thermo Scientific, K420201). A 0.5:1 to 2:1 molar ratio of PCR product to vector was used. </P><DIV class="table-responsive"><TABLE class="table1" style="width:100%; max-width:600px"><TBODY><TR style="font-weight:bold;"><TH style="font-size:80%;">Component</TH><TH style="font-size:80%;">Chemically competent cells</TH><TH style="font-size:80%;">Electrocompetent cells</TH></TR><TR><TD>Fresh PCR product</TD><TD>0.5-4 µL</TD><TD>0.5-4 µL</TD></TR><TR><TD>Salt solution</TD><TD>1 µL</TD><TD>-</TD></TR><TR><TD>Dilute Salt Solution (1:4)</TD><TD>-</TD><TD>1 µL</TD></TR><TR><TD>ddH<SUB>2</SUB>O</TD><TD>add to a final volume of 5 µL</TD><TD>add to a final volume of 5 µL</TD></TR><TR><TD>TOPO<SUP>®</SUP> vector (pBAD202)</TD><TD>1 µL</TD><TD>1 µL</TD></TR><TR><TD>Final volume</TD><TD>6 µL</TD><TD>6 µL</TD></TR></TBODY></TABLE></DIV><UL><LI>Mix reaction gently and incubate for 5 min at RT.</LI><LI>Place the reaction on ice and proceed directly with the transformation of electrocompetent or chemically competent cells.</LI></UL></DIV></DIV></DIV></DIV></DIV><DIV class="panel panel-default"><A data-toggle="collapse" data-parent="#accordion" href="#4-6"><DIV class="panel-heading"><H5 class="panel-title">
                                            			DNA Purification
                                            		</H5></DIV></A><DIV id="4-6" class="panel-collapse collapse"><DIV class="panel-body"><DIV class="tab-content"><DIV id="4-6" class="tab-pane fade in active"><P>DNA purification after PCR and restriction was performed using different kits</P><UL><LI>SureClean Plus distributed by Bioline (BIO-37047)</LI><LI>DNA Clean &amp; Concentrator™-25 distributed by Zymo Research (D4033)</LI><LI>NucleoSpin<SUP>®</SUP> Gel and PCR Clean-up distributed by Macherey-Nagel (REF740609)</LI></UL><P>The procedures were done according to the respective protocol.</P></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV><DIV class="panel panel-default"><A data-toggle="collapse" data-parent="#accordion" href="#5-0"><DIV class="panel-heading"><H4 class="panel-title">
                			Analytic Tests
                		</H4></DIV></A><DIV id="5-0" class="panel-collapse collapse"><DIV class="panel-body"><DIV class="tab-content"><DIV id="5-0" class="tab-pane fade in active"><DIV class="panel panel-default"><A data-toggle="collapse" data-parent="#accordion" href="#5-1"><DIV class="panel-heading"><H5 class="panel-title">
                                            			Agarose Gel Electrophoresis
                                            		</H5></DIV></A><DIV id="5-1" class="panel-collapse collapse"><DIV class="panel-body"><DIV class="tab-content"><DIV id="5-1" class="tab-pane fade in active"><P>For agarose gel electrophoresis, 1.0% agarose gels were prepared. For analytical purpose, the gels were run with the parameters <I>U</I>=90 V, <I>I</I>=400 mA, <I>t</I>=60 min. For preparative purpose, the gels were run with the parameters <I>U</I>=120 V, <I>I</I>=400 mA, <I>t</I>=35 min.Afterwards, the gels were stained in ethidium bromide solution, and then analyzed by UV light.</P><P>Gel extractions of DNA were performed using the NucleoSpin<SUP>®</SUP> Gel and PCR Clean-up kit distributed by Macherey-Nagel (REF740609).</P></DIV></DIV></DIV></DIV></DIV><DIV class="panel panel-default"><A data-toggle="collapse" data-parent="#accordion" href="#5-2"><DIV class="panel-heading"><H5 class="panel-title">
                                            			Expression Test
                                            		</H5></DIV></A><DIV id="5-2" class="panel-collapse collapse"><DIV class="panel-body"><DIV class="tab-content"><DIV id="5-2" class="tab-pane fade in active"><P>In preperation for expression studies, the constructs in pBAD202 were transformed into <EM>E. coli</EM> TOP10, <EM>R. planticola</EM> or<EM> S. blattae</EM> cells. The respective cultivation conditions (medium, temperature) are listed in “Cultivation and Transformation”. The expression assay was done as described in the D-TOPO<SUP>®</SUP> protocol.</P><UL><LI>For each transformant, inoculate 2 mL of medium containing 50 µg/mL kanamycin with a single recombinant colony.</LI><LI>Grow overnight with shaking (225-250 rpm) to OD<SUB>600</SUB> = 1-2.</LI><LI>The next day, label five tubes for each transformant and add 10 mL medium (50 µg/mL kanamycin) to each tube.</LI><LI>Inocculate each tube with 0.1 mL of the overnight culture.</LI><LI>Grow the cultures with vigorous shaking to an OD<SUB>600</SUB>= ~0.5.</LI><LI>Prepare four 10-fold serial dilutions of 20% arabinose with sterile water (e.g 2%, 0.2%, 0.02%, and 0.002%).</LI><LI>Remove a 1 mL aliquot of cells from each tube, centrifuge at maximum speed in a microcentrifuge for 30 s and aspirate the supernatant. Freeze the pellet at -20°C as a zero time point sample.</LI><LI>Add arabinose to the five 9 mL cultures according to the following table.</LI></UL><DIV class="table-responsive"><TABLE class="table1" style="width:100%; max-width:400px"><TBODY><TR style="font-weight:bold;"><TH style="font-size:80%;">Tube</TH><TH style="font-size:80%;">Stock Solution</TH><TH style="font-size:80%;">Volume [mL]</TH><TH style="font-size:80%;">Final concentration</TH></TR><TR><TD>1</TD><TD>0.002%</TD><TD>0.09</TD><TD>0.00002%</TD></TR><TR><TD>2</TD><TD>0.02%</TD><TD>0.09</TD><TD>0.0002%</TD></TR><TR><TD>3</TD><TD>0.2%</TD><TD>0.09</TD><TD>0.002%</TD></TR><TR><TD>4</TD><TD>2%</TD><TD>0.09</TD><TD>0.02%</TD></TR><TR><TD>5</TD><TD>20%</TD><TD>0.09</TD><TD>0.2%</TD></TR></TBODY></TABLE></DIV><UL><LI>Grow with shaking for 4 h.</LI><LI>Take 1 mL samples at 4 h and treat as described for the zero time point sample.</LI></UL><P> </P><UL><LI>Analyze the samples by suspending the pellets in 80 µL 1X SDS-PAGE sample buffer.</LI><LI>Boil 5 min and centrifuge briefly.</LI><LI>Load 5-10 µL of each sample on an SDS-PAGE gel and electrophorese.</LI></UL></DIV></DIV></DIV></DIV></DIV><DIV class="panel panel-default"><A data-toggle="collapse" data-parent="#accordion" href="#5-3"><DIV class="panel-heading"><H5 class="panel-title">                                       			Cell Fractionation
                                            		</H5></DIV></A><DIV id="5-3" class="panel-collapse collapse"><DIV class="panel-body"><DIV class="tab-content"><DIV id="5-3" class="tab-pane fade in active"><UL><LI>Harvest 500 mL of cells from an induction experiment by centrifugation and wash three times with 0.9% NaCl.</LI><LI>Resuspend the pellet in 2 mL buffer P (pH 8.0) and incubate for 20-30 min on ice.</LI></UL><DIV class="table-responsive"><TABLE class="table1" style="width:100%; max-width:400px"><TBODY><TR style="font-weight:bold;"><TH style="font-size:80%;">Component</TH><TH style="font-size:80%;">Concentration</TH></TR><TR><TD>Tris-HCl</TD><TD>100 mM</TD></TR><TR><TD>Sucrose</TD><TD>20% (w/v)</TD></TR><TR><TD>EDTA</TD><TD>100 mM</TD></TR></TBODY></TABLE></DIV><UL><LI>Centrifuge for 5 min (14000 rpm, 4°C) and carefully remove the supernatant which contains the periplasm.</LI><LI>Resuspend the pellet in 2 mL buffer P and incubate for 20-30 min on ice.</LI><LI>After another centrifugation for 5 min (14000 rpm, 4°C) the supernatant with the spheroplasts can be carfully removed.</LI><LI>The samples can either be stored at -20°C or directly used for analysis.</LI></UL></DIV></DIV></DIV></DIV></DIV><DIV class="panel panel-default"><A data-toggle="collapse" data-parent="#accordion" href="#5-4"><DIV class="panel-heading"><H5 class="panel-title">
                                            			SDS PAGE
                                            		</H5></DIV></A><DIV id="5-4" class="panel-collapse collapse"><DIV class="panel-body"><DIV class="tab-content"><DIV id="5-4" class="tab-pane fade in active"><P><U>Acrylamid Stocksolution (AA)</U></P><UL><LI>40% Acrylamide 4x cryst.- Mix 37.5:1 (Biomol 54535; stored at 4°C)</LI></UL><P><U>Resolving gel Stocksolution (RG)</U></P><DIV class="table-responsive"><TABLE class="table1" style="width:100%; max-width:400px"><TBODY><TR style="font-weight:bold;"><TH style="font-size:80%;">Component</TH><TH style="font-size:80%;">Amount</TH></TR><TR><TD>Tris-HCl</TD><TD>18.2 g</TD></TR><TR><TD>SDS</TD><TD>0.4 g</TD></TR></TBODY></TABLE></DIV><UL><LI>Add components to ddH<SUB>2</SUB>O, adjust the pH to 8.8 with HCl and bring the volume to 100 mL.</LI><LI>Store at 4°C.</LI></UL><P><U>Stacking gel Stocksolution (SG)</U></P></DIV><DIV class="table-responsive"><TABLE class="table1" style="width:100%; max-width:400px"><TBODY><TR style="font-weight:bold;"><TH style="font-size:80%;">Component</TH><TH style="font-size:80%;">Amount</TH></TR><TR><TD>Tris-HCl</TD><TD>6.1 g</TD></TR><TR><TD>SDS</TD><TD>0.4 g</TD></TR></TBODY></TABLE></DIV><UL><LI>Add components to ddH<SUB>2</SUB>O, adjust the pH to 6.8 with HCl and bring the volume to 100 mL.</LI><LI>Store at 4°C.</LI></UL><P><U>10x running buffer</U></P><DIV class="table-responsive"><TABLE class="table1" style="width:100%; max-width:400px"><TBODY><TR style="font-weight:bold;"><TH style="font-size:80%;">Component</TH><TH style="font-size:80%;">Amount</TH></TR><TR><TD>Tris-HCl</TD><TD>30.3 g</TD></TR><TR><TD>Glycine</TD><TD>144.1 g</TD></TR><TR><TD>SDS</TD><TD>10 g</TD></TR></TBODY></TABLE></DIV><UL><LI>Add components to ddH<SUB>2</SUB>O and bring the volume to 1 L. The pH should bea around 8.4.</LI><LI>Store at RT.</LI></UL><P><U>4x SDS Loading Dye</U></P><DIV class="table-responsive"><TABLE class="table1" style="width:100%; max-width:400px"><TBODY><TR style="font-weight:bold;"><TH style="font-size:80%;">Component</TH><TH style="font-size:80%;">Amount</TH></TR><TR><TD>Tris-HCl</TD><TD>0.4 g</TD></TR><TR><TD>SDS</TD><TD>1.2 g</TD></TR><TR><TD>Glycerine (87% w/v)</TD><TD>7.5 mL</TD></TR><TR><TD>β-Mercaptoethanol</TD><TD>2.5 mL</TD></TR><TR><TD>Bromphenol blue (2%)</TD><TD>0.5 mL</TD></TR></TBODY></TABLE></DIV><UL><LI>Add Tris-HCl and SDS to ddH<SUB>2</SUB>O and adjust the pH to 6.8.</LI><LI>Add the rest of the compnents and bring the volume to 50 mL</LI><LI>Store at -20°C.</LI></UL><P><U>Ammoniumpersulfat (APS)</U></P><UL><LI>10% APS (w/v) in ddH<SUB>2</SUB>O</LI><LI>Store at -20°C.</LI></UL><P><U>Pipetting schemes for one gel</U></P><DIV class="table-responsive"><TABLE class="table1" style="width:100%; max-width:400px"><TBODY><TR style="font-weight:bold;"><TH style="font-size:80%;">Component</TH><TH style="font-size:80%;">Resolving gel 10%</TH><TH style="font-size:80%;">Stacking gel 4%</TH></TR><TR><TD>AA</TD><TD>1 mL</TD><TD>0.2 mL</TD></TR><TR><TD>RG</TD><TD>1 mL</TD><TD>-</TD></TR><TR><TD>SG</TD><TD>-</TD><TD>0.48 mL</TD></TR><TR><TD>ddH<SUB>2</SUB>O</TD><TD>2 mL</TD><TD>1.32 mL</TD></TR><TR><TD>TEMED</TD><TD>3 µL</TD><TD>2 µL</TD></TR><TR><TD>APS</TD><TD>30 µL</TD><TD>15 µL</TD></TR></TBODY></TABLE></DIV><DIV class="table-responsive"><TABLE class="table1" style="width:100%; max-width:400px"><TBODY><TR style="font-weight:bold;"><TH style="font-size:80%;">Component</TH><TH style="font-size:80%;">Resolving gel 15%</TH><TH style="font-size:80%;">Stacking gel 4%</TH></TR><TR><TD>AA</TD><TD>1.5 mL</TD><TD>0.2 mL</TD></TR><TR><TD>RG</TD><TD>1 mL</TD><TD>-</TD></TR><TR><TD>SG</TD><TD>-</TD><TD>0.48 mL</TD></TR><TR><TD>ddH<SUB>2</SUB>O</TD><TD>1.5 mL</TD><TD>1.32 mL</TD></TR><TR><TD>TEMED</TD><TD>3 µL</TD><TD>2 µL</TD></TR><TR><TD>APS</TD><TD>30 µL</TD><TD>15 µL</TD></TR></TBODY></TABLE></DIV><P><U>Preperation of SDS-Gels</U></P><UL><LI>Glass plates, combs and spacers are cleaned with EtOH (70%) and assembled.</LI><LI>The resolving gel solution is prepared with TEMED and APS added last for polymerisation. The solution is poured between the assembled galss plates and covered with isopropanol.</LI><LI>After polymerisation, the isopropanol is removed and the stacking gel is prepared and poured on top of the resolving gel. The combs need to be inserted immediatly.</LI><LI>After polymerisation, the gels can either be put into the electrophoresis chamber with 1x running buffer or stored at 4°C in a bag with wet tissue. </LI></UL><P><U>Preperation of samples</U></P><UL><LI>Samples are mixed with SDS loading dye (4:1), denaturated at 95°C for 5 min and breifly centrifuged.</LI><LI>Up to 15 µL of a sample are loaded into one well.</LI></UL><P><U>SDS-PAGE</U></P><UL><LI>After loading the samples and a protein marker onto the gel, it is run for 15 min with an electric current of 15 mA current (per gel). Afterwards, the gel is run with an electric current of 30 mA (per gel).</LI></UL><P><U>Staining with Coomassie-Brilliant-Blue</U></P><P>The following solutions have to be prepared.</P><P><U>Staining solution</U></P><DIV class="table-responsive"><TABLE class="table1" style="width:100%; max-width:500px"><TBODY><TR style="font-weight:bold;"><TH style="font-size:80%;">Final concentration</TH><TH style="font-size:80%;">Component</TH><TH style="font-size:80%;">Amount</TH></TR><TR><TD>0.05%</TD><TD>Coomassie-Brilliant-Blue R 250</TD><TD>0.5 g</TD></TR><TR><TD>25%</TD><TD>Isopropanol</TD><TD>250 mL</TD></TR><TR><TD>10%</TD><TD>Acetic acid</TD><TD>100 mL</TD></TR></TBODY></TABLE></DIV><UL><LI>Dissolve the components in ddH2O and bring the volume to 1 L.</LI><LI>The solution can be used more than once and is stored at RT.</LI></UL><P><U>Fixation solution</U></P><UL><LI>10% acetic acid</LI></UL><P> </P><UL><LI>Place the SDS-gel in a box with staining solution and heat in the microwave.</LI><LI>Incubate the gels on a rocker until protein bands become visible.</LI><LI>Wash the gel with water and discolour it in fixation solution for several hours or overnight.</LI></UL></DIV></DIV></DIV></DIV><DIV class="panel panel-default"><A data-toggle="collapse" data-parent="#accordion" href="#5-5"><DIV class="panel-heading"><H5 class="panel-title">
                                            			Western Blot
                                            		</H5></DIV></A><DIV id="5-5" class="panel-collapse collapse"><DIV class="panel-body"><DIV class="tab-content"><DIV id="5-5" class="tab-pane fade in active"><P>For Western Blot an Anti-His antibody (Anti-His (C-term)/AP) was used.</P><P>The following solutions need to be prepared for the procedure.</P><P><U>10x TBS buffer</U></P><DIV class="table-responsive"><TABLE class="table1" style="width:100%; max-width:400px"><TBODY><TR style="font-weight:bold;"><TH style="font-size:80%;">Final concentration</TH><TH style="font-size:80%;">Component</TH><TH style="font-size:80%;">Amount</TH></TR><TR><TD>0.1 M</TD><TD>Tris-HCl (MW 121.14)</TD><TD>6.057 g</TD></TR><TR><TD>1.4 M</TD><TD>NaCl (MW 58.44)</TD><TD>40.91 g</TD></TR></TBODY></TABLE></DIV><UL><LI>Dissolve the components in ddH<SUB>2</SUB>O, adjust the pH to 7.4 and bring the volume to 500 mL.</LI><LI>Store at RT.</LI></UL><P><U>1x Transfer buffer</U></P><DIV class="table-responsive"><TABLE class="table1" style="width:100%; max-width:400px"><TBODY><TR style="font-weight:bold;"><TH style="font-size:80%;">Final concentration</TH><TH style="font-size:80%;">Component</TH><TH style="font-size:80%;">Amount</TH></TR><TR><TD>25 mM</TD><TD>Tris-HCl (MW 121.14)</TD><TD>3.029 g</TD></TR><TR><TD>192 mM</TD><TD>Glycine (MW 75.07)</TD><TD>14.41 g</TD></TR><TR><TD>Methanol</TD><TD>200 mL</TD></TR></TBODY></TABLE></DIV><UL><LI>Dissolve the components in ddH<SUB>2</SUB>O and bring the volume to 1 L. The pH should be between 8.1 and 8.4.</LI><LI>Store at RT.</LI></UL><P><U>10x TBS-Tween buffer</U></P><DIV class="table-responsive"><TABLE class="table1" style="width:100%; max-width:400px"><TBODY><TR style="font-weight:bold;"><TH style="font-size:80%;">Final concentration</TH><TH style="font-size:80%;">Component</TH><TH style="font-size:80%;">Amount</TH></TR><TR><TD>0.1 M</TD><TD>Tris-HCl (MW 121.14)</TD><TD>6.057 g</TD></TR><TR><TD>1.4 M</TD><TD>NaCl (MW 58.44)</TD><TD>40.91 g</TD></TR><TR><TD>0.1%</TD><TD>Tween20</TD><TD>0.5 mL</TD></TR></TBODY></TABLE></DIV><UL><LI>Dissolve the components in ddH<SUB>2</SUB>O, adjust the pH to 7.4 and bring the volume to 500 mL.</LI><LI>Store at RT.</LI></UL><P><U>Blocking Solution</U></P><UL><LI>Dissolve 2.5 g skim milk in 50 mL 1x TBS buffer.</LI><LI>It has to be prepared freshly.</LI></UL><P><U>Phosphatase buffer</U></P><DIV class="table-responsive"><TABLE class="table1" style="width:100%; max-width:500px"><TBODY><TR style="font-weight:bold;"><TH style="font-size:80%;">Final concentration</TH><TH style="font-size:80%;">Component</TH><TH style="font-size:80%;">Amount</TH></TR><TR><TD>0.1 M</TD><TD>Tris-HCl (MW 121.14)</TD><TD>6.057 g</TD></TR><TR><TD>0.1 M</TD><TD>NaCl (MW 58.44)</TD><TD>40.91 g</TD></TR><TR><TD>5 mM</TD><TD>MgCl<SUB>2</SUB> x 6 H<SUB>2</SUB>O (MW 203.3)</TD><TD>0.51 g</TD></TR></TBODY></TABLE></DIV><UL><LI>Dissolve the components in ddH<SUB>2</SUB>O, adjust the pH to 7.4 and bring the volume to 500 mL.</LI><LI>Store at RT.</LI></UL><P>In preparation for the Western Blot, a SDS-PAGE with a Protein Ladder, which is compatible with Western Blot, has to be run.</P><UL><LI>Prepare 6 gel sized Whatman paper and gel sized blot membrane by soaking them in 1x Transfer buffer.</LI><LI>Set up the Western Blot. Place 3 moist Whatman papers on the anode (base of blotter), followed by the moist membrane, the unstained SDS-gel, 3 moist Whatman papers and the cathode (lid of blotter).</LI><LI>Blot at 200 mA (1 gel) and constant 10 V for 1 h.</LI><LI>Wash the membrane three times for 5 min with 1x TBS buffer on a rocker.</LI><LI>Block with 50 mL 1x TBS buffer with 5% skim milk for 1 h or over night on a rocker.</LI><LI>Wash three times for 15 min with 1x TBS buffer on a rocker.</LI><LI>Incubate with antibody solution (14 mL 1x TBS-Tween buffer + 200 µL 1x TBS buffer/5% skim milk + 7 µL antibody) for 2 h on a rocker.</LI><LI>Wash two times shortly with 1x TBS-Tween buffer.</LI><LI>Wash once with 1x TBS-Tween buffer for 15 min on a rocker.</LI><LI>Wash two times with 1x TBS-Tween buffer for 5 min on a rocker.</LI><LI>Wash two times with 1x TBS buffer for 10 min on a rocker.</LI><LI>Wash two times shortly in 1x phosphatase buffer.</LI><LI>Place the membrane in staining solution (10ml 1x phosphatase buffer + 66µl NBT + 33µl BCIP), stain in the dark until bands become visible.</LI><LI>Wash with H<SUB>2</SUB>O to stop the reaction.</LI><LI>Dry the membrane in Whatman paper and scan it.</LI></UL></DIV></DIV></DIV></DIV></DIV><DIV class="panel panel-default"><A data-toggle="collapse" data-parent="#accordion" href="#5-6"><DIV class="panel-heading"><H5 class="panel-title">
                                            			Microbial B<SUB>12</SUB> Plate Assay
                                            		</H5></DIV></A><DIV id="5-6" class="panel-collapse collapse"><DIV class="panel-body"><DIV class="tab-content"><DIV id="5-6" class="tab-pane fade in active"><P>The B<SUB>12</SUB> Plate Assay allows the detection of corrinoids in cell extracts or supernatants. It is based on the fact, that <EM>E. coli</EM> DSM 4261 relies on its B<SUB>12</SUB> dependent methionine synthase which is encoded by <EM>metH</EM>. This is caused by a point mutation in <EM>metE</EM> which encodes the B<SUB>12</SUB> independent methionine synthase. In a medium without methionine, <EM>E. coli</EM> DSM 4261 therefore relies on the presence of B<SUB>12</SUB>.</P><P><U>Cultivation of <EM>E. coli</EM> DSM 4261</U></P><UL><LI>50 mL of Guttman medium are inoculated with a 5 mL preparatory overnight culture.</LI><LI>The cells are harvested at an OD<SUB>600</SUB> of 0.2, washed three times with 20 mL 0.9% NaCl and afterwards resuspended in the same volume NaCl.</LI></UL><P><U>Preparation of samples</U></P><UL><LI>After induction, the cells are centrifuged down, and washed three times with 0.1 M sodium chloride solution.:</LI></UL><DIV class="table-responsive"><TABLE class="table1" style="width:100%; max-width:400px"><TBODY><TR style="font-weight:bold;"><TH style="font-size:80%;">Component</TH><TH style="font-size:80%;">Amount</TH></TR><TR><TD>K<SUB>2</SUB>HPO<SUB>4</SUB></TD><TD>13.6 g</TD></TR><TR><TD>KH<SUB>2</SUB>PO<SUB>4</SUB>· 3 H<SUB>2</SUB>O</TD><TD>22.8 g</TD></TR></TBODY></TABLE></DIV><UL><LI>Add components to ddH2O, adjust the pH tp 7.0 and bring the volume to 1 L.</LI><LI>Periplasm, cytoplasm and crude cell extracts from different expression studies are used as samples.</LI></UL><P><U>Assay</U></P><UL><LI>7 mL of the cell suspension, 50 mM and 0.2 g/L triphenyltetrazoliumchloride are added to 500 mL of the autoclaved mineral-agar (50°C).</LI></UL><DIV class="table-responsive"><TABLE class="table1" style="width:100%; max-width:400px"><TBODY><TR style="font-weight:bold;"><TH style="font-size:80%;">Component</TH><TH style="font-size:80%;">Amount</TH></TR><TR><TD>K<SUB>2</SUB>HPO<SUB>4</SUB></TD><TD>7 g</TD></TR><TR><TD>KH<SUB>2</SUB>PO<SUB>4</SUB></TD><TD>3 g</TD></TR><TR><TD>(NH<SUB>4</SUB>)<SUB>2</SUB>SO<SUB>4</SUB></TD><TD>1 g</TD></TR><TR><TD>MgSO<SUB>4</SUB>· 7 H<SUB>2</SUB>O</TD><TD>0.1 g</TD></TR><TR><TD>Na<SUB>2</SUB>citrate · 2 H<SUB>2</SUB>O</TD><TD>3 g</TD></TR><TR><TD>Agar Bacteriological</TD><TD>15 g</TD></TR></TBODY></TABLE></DIV><UL><LI>Add components to ddH<SUB>2</SUB>O and bring the volume to 1 L.</LI><LI>4 sterile filter papers (Ø 0.9 cm, Fa. Schleicher &amp; Schuell, Dassel) are laid onto the plates and loaded with 10-15 µL of a sample or a Vitamin B<SUB>12</SUB> calibration solution (0, 0.05, 0.25, 0.1, 5.0 μg/ml). Every sample or concentration is analyzed twice.</LI><LI>The plates are incubated at 37°C for 16-24 h.</LI></UL><P>If the samples contained Vitamin B<SUB>12</SUB>, the cells grow in a red zone around the filter paper. The size is proportional to the Vitamin <SUB>12</SUB> concentration and the reduction of triphenyltetrazoliumchloride causes the red staining.</P><UL><LI>To analyze the B<SUB>12</SUB> concentration in the samples, the diameter of the red zone is measured and compared to the calibrations.</LI></UL></DIV></DIV></DIV></DIV></DIV><DIV class="panel panel-default"><A data-toggle="collapse" data-parent="#accordion" href="#5-7"><DIV class="panel-heading"><H5 class="panel-title">
                                            			Photometric B<SUB>12</SUB> Assay
                                            		</H5></DIV></A><DIV id="5-7" class="panel-collapse collapse"><DIV class="panel-body"><DIV class="tab-content"><DIV id="5-7" class="tab-pane fade in active"><P>To determine the corrinoid concentration in a sample, the corrinoids have to be converted into the soluble dicyano form and proteins and lipids have to be removed by precipitation and centrigugation.</P><UL><LI>Incubate 1 mL of your sample with 0.2% (w/v) NaCN or KCN for 30 min at RT in the dark.</LI><LI>Acidify the solution with pure acetic acid to a pH of 4.0.</LI><LI>Boil the samples at 100°C for 20 min.</LI><LI>Let the sample cool down and adjust the pH to 11.0 with KOH.</LI><LI>Add 4-5 mg NaCN or KCN.</LI><LI>Centriguge at 13,000 rpm for 30 min at RT.</LI><LI>Measure a spectrum of the supernatant from 350 to 650 nm.</LI></UL><P>The concentration of Vitamin B<SUB>12</SUB> can be calculated with help of the molar extinction coefficient of dicyano cobalamine (Δε<SUB>580-640</SUB> = 7.3 mM<SUP>-1</SUP>· cm<SUP>-1</SUP>) by determining the differences in the absorption at 580 and 650 nm.  Alternatively, the concentration of Vitamin B<SUB>12</SUB> can also be determined with the extinction coefficients of dicyano cobalamine at 580 nm (ε<SUB>540</SUB> = 7800 mM<SUP>-1</SUP>· cm<SUP>-1</SUP>) and 367 nm (Δε<SUB>367</SUB> = 23800 mM<SUP>-1</SUP>· cm<SUP>-1</SUP>) by calculating the average.  As a standard hydroxyl cobalamine and cyano cobalamine are used.</P></DIV></DIV></DIV></DIV></DIV><DIV class="panel panel-default"><A data-toggle="collapse" data-parent="#accordion" href="#5-8"><DIV class="panel-heading"><H5 class="panel-title">                              			Bradford Assay
                                            		</H5></DIV></A><DIV id="5-8" class="panel-collapse collapse"><DIV class="panel-body"><DIV class="tab-content"><DIV id="5-8" class="tab-pane fade in active"><P><U>Bradford solution</U></P><UL><LI>Mix 800 mL ddH<SUB>2</SUB>O with 200 mL RotiQuant (5X) (Carl Roth).</LI><LI>Store at 4°C protected from light.</LI><LI>After preparing a new solution, the Bradford factor has to be determined by measuring a standard curve from protein standards with known concentrations.</LI></UL><P><U>Measuring protein sample</U></P><UL><LI>Mix 20 µL of the sample with 1mL Bradford solution in a photometer cuvette. Prepare triplicates.</LI><LI>Also prepare a blank by mixing 20 µL ddH<SUB>2</SUB>O with 1 mL Bradford solution in a photometer cuvette.</LI><LI>Incubate for 5 min at RT protected from light.</LI><LI>Measure the absorbance at 595 nm against the black. It should lie between 0.1 and 0.6, otherwise the protein sample has to be diluted.</LI></UL><P><U>Calculation of protein concentration</U></P></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV><DIV class="col-md-3"><DIV class="sidebar"><H3 style="text-align: center;">Contact</H3><DIV class="sidebaraddress"><UL><LI><A href="https://www.facebook.com/iGEM-Team-Goettingen-687557807970778">  Facebook</A></LI><LI><A href="https://twitter.com/iGEM_Goettingen">  Twitter</A></LI><LI><A href="mailto:igem2016@gwdg.de">  E-Mail</A></LI><LI><A href="http://igem2016.uni-goettingen.de/">  Homepage</A></LI><LI><A href="http://appmibio.uni-goettingen.de/">  Address</A></LI></UL></DIV><A href="http://appmibio.uni-goettingen.de/"><DIV class="sidebaraddress" style="padding-left:42px; margin-top:10px;"><FONT size="-1">
  iGEM Team Göttingen 2016
  Institut für Mikrobiologie 
  und Genetik
  AG Prof. Dr. Rolf Daniel
  Grisebachstr. 8
  37077 Göttingen</FONT></DIV></A><DIV class="sidebar_navigation"><H3 style="text-align: center;">Navigation</H3><DIV class="menu_wrapper"><UL id="accordion" class="accordion"><LI class="menu_item"><A href="https://2016.igem.org/Team:Goettingen">Home </A></LI><LI class="menu_item"> Team
  </LI><LI><A href=" https://2016.igem.org/Team:Goettingen/Team"> Team   </A></LI><LI><A href="https://2016.igem.org/Team:Goettingen/Collaborations">Collaborations</A></LI></UL><LI class="menu_item" style="background-color: #e7e7e7;"><B>Project</B></LI><LI><A href="https://2016.igem.org/Team:Goettingen/Description">Description</A></LI><LI><A style="background-color: #D8D8D8;" href="https://2016.igem.org/Team:Goettingen/Experiments"><B>Experiments</B></A></LI><LI><A href="https://2016.igem.org/Team:Goettingen/Proof">Proof of Concept</A></LI><LI><A href="https://2016.igem.org/Team:Goettingen/Results"> Results </A></LI><LI><A href="https://2016.igem.org/Team:Goettingen/Notebook"> Notebook </A></LI><LI class="menu_item"> Parts
  </LI><LI><A href="https://2016.igem.org/Team:Goettingen/Parts">Parts</A></LI><LI><A href="https://2016.igem.org/Team:Goettingen/Basic_Part">Basic Parts </A></LI><LI class="menu_item"><A href="https://2016.igem.org/Team:Goettingen/Safety">Safety</A></LI><LI class="menu_item"><A href="https://2016.igem.org/Team:Goettingen/Attributions">Attributions</A></LI><LI class="menu_item"> Human Practices
  </LI><LI><A href="https://2016.igem.org/Team:Goettingen/Human_Practices">Human Practices</A></LI><LI><A href="https://2016.igem.org/Team:Goettingen/HP/Silver">HP on the Road</A></LI><LI><A href="https://2016.igem.org/Team:Goettingen/HP/Gold">HP on the Web</A></LI><LI><A href="https://2016.igem.org/Team:Goettingen/Engagement">Engagement</A></LI></DIV></DIV><DIV class="sidebar"><H3 style="text-align: center;">Sponsors</H3></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></BODY></HTML>