<HTML xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en" dir="ltr">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki  ltr ns-0 ns-subject page-Team_UC_Berkeley_Tools_RelatedWork"><DIV id="globalWrapper"><DIV id="top-section"><DIV id="p-logo"><A href="/Main_Page" title="Main Page">&quot;
	    </A></DIV><DIV id="menubar" class="left-menu"><UL><LI class="selected"><A href="/Team:UC_Berkeley_Tools/RelatedWork">Page               </A></LI><LI><A href="/Talk:Team:UC_Berkeley_Tools/RelatedWork">Discussion               </A></LI><LI><A href="/wiki/index.php?title=Team:UC_Berkeley_Tools/RelatedWork&amp;action=edit">View source               </A></LI><LI><A href="/wiki/index.php?title=Team:UC_Berkeley_Tools/RelatedWork&amp;action=history">History               </A></LI><LI style="color:#808080;cursor:default">teams</LI></UL></DIV><DIV class="right-menu" id="menubar"><UL><LI id="pt-login"><A href="/wiki/index.php?title=Special:UserLogin&amp;returnto=Team:UC_Berkeley_Tools/RelatedWork" title="You are encouraged to log in; however, it is not mandatory [o]" accesskey="o">Log in</A></LI></UL></DIV><DIV id="search-controls"><FORM action="/Special:Search" id="searchform"> </FORM></DIV></DIV><DIV id="content"><H1 class="firstHeading">Team:UC Berkeley Tools/RelatedWork</H1><DIV id="bodyContent"><H3 id="siteSub">From 2008.igem.org</H3><DIV class="menu"><UL><LI><A title="Home" href="https://2008.igem.org/Team:UC_Berkeley_Tools" style="color: rgb(255,255,255)" class="menu">Home</A></LI></UL><UL><LI><A class="menu">Project</A></LI><LI><A title="Technical Information" href="https://2008.igem.org/Team:UC_Berkeley_Tools/Tutorial" style="color: rgb(0,0,0)">Clotho Tutorial</A></LI><LI><A title="Clotho Tutorial" href="https://2008.igem.org/Team:UC_Berkeley_Tools/Project" style="color: rgb(0,0,0)">Technical Description</A></LI><LI><A title="Clotho: Development and Impacts" href="https://2008.igem.org/Team:UC_Berkeley_Tools/Development" style="color: rgb(0,0,0)">Clotho: Development and Impacts</A></LI></UL><UL><LI><A class="menu" style="color: rgb(255,255,255)">The Team</A></LI><LI><A title="The Team" href="https://2008.igem.org/Team:UC_Berkeley_Tools/Team" style="color: rgb(0,0,0)">Who Are We?</A></LI><LI><A href="https://2008.igem.org/Team:UC_Berkeley_Tools/Notebook" style="color: rgb(0,0,0)"><SPAN class="drop"><SPAN>Notebooks</SPAN>
             &gt;&gt;
          </SPAN></A></LI><LI><A title="Doug's Notebook" href="https://2008.igem.org/Team:UC_Berkeley_Tools/Notebook/Doug's_Notebook" style="color: rgb(0,0,0)">Doug's Notebook</A></LI><LI><A title="Anne's Notebook" href="https://2008.igem.org/Team:UC_Berkeley_Tools/Notebook/Anne's_Notebook" style="color: rgb(0,0,0)">Anne's Notebook</A></LI><LI><A title="Matt's Notebook" href="https://2008.igem.org/Team:UC_Berkeley_Tools/Notebook/Matthew's_Notebook" style="color: rgb(0,0,0)">Matt's Notebook</A></LI><LI><A title="Nade's Notebook" href="https://2008.igem.org/Team:UC_Berkeley_Tools/Notebook/Nade's_Notebook" style="color: rgb(0,0,0)">Nade's Notebook</A></LI></UL><UL><LI><A class="menu" style="color: rgb(255,255,255)">Resources</A></LI><LI><A title="Downloads" href="https://2008.igem.org/Team:UC_Berkeley_Tools/Project/Downloads" class="menu" style="color: rgb(0,0,0)">Downloads</A></LI><LI><A title="Related Work" href="https://2008.igem.org/Team:UC_Berkeley_Tools/RelatedWork" style="color: rgb(0,0,0)">Related Work</A></LI><LI><A title="External Links" href="https://2008.igem.org/Team:UC_Berkeley_Tools/ExternalLinks" style="color: rgb(0,0,0)">External Links</A></LI></UL><UL><LI><A title="Download Now!" href="https://2008.igem.org/Team:UC_Berkeley_Tools/Project/Downloads" style="color: rgb(255,255,255)" class="menu">Get Clotho Now!</A></LI></UL></DIV><H2><SPAN class="mw-headline" id="Related_Work"><B>Related Work</B></SPAN></H2><P>This section will examine several tools which attempt to provide a design environment for synthetic biology. These can be divided into both academic and industrial offerings. 
</P><H3><SPAN class="mw-headline" id="Academic">Academic</SPAN></H3><H4><SPAN class="mw-headline" id="APE">APE</SPAN></H4><DIV class="thumb tright"><DIV class="thumbinner" style="width:250px;"><DIV class="thumbcaption">APE</DIV></DIV></DIV><P><A href="http://www.biology.utah.edu/jorgensen/wayned/ape/" class="external text" rel="nofollow">APE</A> (A Plasmid Editor) [6] is an editor which shows DNA sequence and translation information in a single text based viewer. It performs basic highlighting tasks for restriction sites (or user defined regions). It also shows translation, Tm, %GC, and ORF of selected DNA in real-time. It also reads DNA Strider, FASTA, Genbank and EMBL files. It has no database connectivity or notion of a larger synthetic biology community.
</P><H4><SPAN class="mw-headline" id="Athena">Athena</SPAN></H4><DIV class="thumb tright"><DIV class="thumbinner" style="width:302px;"><DIV class="thumbcaption">Athena</DIV></DIV></DIV><P>Athena is a tool developed at the University of Washington by Deepak Chandran, Frank Bergmann, Herbert Sauro. Here is their description from <A href="http://openwetware.org/wiki/Computational_Tools" class="external text" rel="nofollow">OpenWetWare's Computational Tool Site</A></P><P>Athena is a tool for building, simulating, and analyzing genetic circuits (as well as metabolic/signaling networks, such as SBML files). It provides a visual interface for building biological modules that can be saved and later connected together. The connection can be achieved using either the PoPS interface or by defining overlapping molecular species (similar to the concept of module in CellML and SBML). In addition to simulation, Athena supports a few other useful features: Database of Ecoli regulatory network from RegulonDB, Graphical view of part sequence, Automated derivation of transcription rate equations, Interface to all Systems Biology Workbench programs, Interface with R statistical language, and an easy plugin architecture.
</P><P>You can find out more about Athena <A href="http://www.codeplex.com/athena" class="external text" rel="nofollow">here</A>. 
</P><P><A href="http://www.washington.edu/staff/deepakc/downloads/InstallAthena.exe" class="external text" rel="nofollow">Download Athena</A></P><H4><SPAN class="mw-headline" id="BioJADE">BioJADE</SPAN></H4><DIV class="thumb tright"><DIV class="thumbinner" style="width:302px;"><DIV class="thumbcaption">BioJADE's Schematic Aspect</DIV></DIV></DIV><P>One of the early contributions in this area is <A href="http://web.mit.edu/jagoler/www/biojade/" class="external text" rel="nofollow">BioJADE</A> [1]. BioJADE is a design and simulation tool written in JAVA. It supports interaction with a number of part repositories including MIT's standard biological parts repository (XML interface) as well as relational database systems such as Oracle 9. It allows both the importing of devices from repositories as well as the submission of newly created parts to these same repositories. In terms of manipulating the design, BioJADE provides four <I>aspects</I>. These are <I>Schematic, DNA, Functional, and Icon</I>.  As shown, Schematic presents a traditional schematic editor view for the design of logic circuits. The DNA aspect illustrates the actual DNA nucleotide sequence of the design. Functional Network presents a closer view of the layout of the DNA. It is similar to the DNA aspect but allows the user to edit the sequence.  
The Icon aspect allows the designer to create icons for parts to be used in the schematic aspect. Input, output, and undefined terminals are specified either as a static number or as dynamic (added at runtime). 
Finally, BioJADE offers a simulation mode where the design can be simulated. It interacts with simulators like Stochastirator [3]  or Tabasco [4].
</P><H4><SPAN class="mw-headline" id="BioStudio">BioStudio</SPAN></H4><P>BioStudio is the work of researchers at Johns Hopkins University (Joel Bader, Jef Boeke, and Sarah Richardson). 
</P><P><I>This is taken from BioStudio press release material at <A href="http://research.microsoft.com/ur/us/fundingopps/RFPs/Synthetic_Biology_Awards_2006.aspx" class="external text" rel="nofollow">Computational Challenges in Synthetic Biology 2006 Awards</A></I></P><P>We propose BioStudio as a collaborative integrated design environment (IDE) providing editing and revision control for synthetic genomes. Windows SharePoint Services and SQL Server will provide back-end support. The server middle-layer will extend existing open-source visualization-only software (GBrowse) with editing functions, including a BioBricks palette, and will communicate with the back-end using .NET and FrontPage Remote Procedure Calls. Genome sequence and annotations will be stored in XML with an adapter to the Generic Feature Format (GFF) standard. Synthetic biology teams will connect with a web browser client. Development and testing will proceed in conjunction with an in-house project to engineer a synthetic yeast cell that encodes a combinatorial satisfiability experiment to identify a minimal genome. Our physical-level editor is complementary to logical-level design tools such as BioJADE. It provides a multiscale visualization metaphor that enables non-computer-scientist teams to collaborate on documents that are too large for &quot;Track changes&quot; and too critical or interdependent to be consigned to a Wiki. Software will be released under a BSD license.
</P><P>In addition to BioStudio there is an interactive web site called <A href="http://www.genedesign.org" class="external text" rel="nofollow">Gene Design</A>.
</P><H4><SPAN class="mw-headline" id="GenoCAD">GenoCAD</SPAN></H4><DIV class="thumb tleft"><DIV class="thumbinner" style="width:302px;"><DIV class="thumbcaption">GenoCAD Screen Capture</DIV></DIV></DIV><P><A href="http://synbio.vbi.vt.edu:25500/genocad/" class="external text" rel="nofollow">GenoCAD</A> [5] 
is a web based tool which guides you through the process of
creating new genetic constructs by combining DNA sequences known as
genetic parts or biological parts. Users can create their own account
on the web site to add their new genetic parts, customize the build-in
parts libraries for their own needs, and save their designs for future
reference. Upon completion of the design process, the design DNA
sequence can be saved as a text file for synthesis by a gene synthesis
company or simply for further analysis in sequence analysis
applications. GenoCAD also provides a construct validation feature (using a context free grammar (CFG) analysis)
that allows users to import the sequence of a design and check its
validity. GenoCAD relies on a linguistic model of the DNA sequence.
The site currently supports a single proof of concept grammar that has
been described in a paper published in Bioinformatics. Ongoing
development efforts aim at implementing support for multiple grammars
in GenoCAD.
</P><P>Strengths of this tool are primarily its attempt to validate the sequence and categorize parts via <I>production rules</I> based on 26 syntactic categories (i.e. gene coding region, terminator, stop codon, promoter, ribosome binding site, etc).
</P><P>For more information see: <A href="http://bioinformatics.oxfordjournals.org/cgi/reprint/btm446v2.pdf" class="external text" rel="nofollow">A syntactic model to design and verify synthetic genetic constructs derived from standard biological parts</A></P><H3><SPAN class="mw-headline" id="Industrial">Industrial</SPAN></H3><H4><SPAN class="mw-headline" id="Gene_Designer">Gene Designer</SPAN></H4><DIV class="thumb tleft"><DIV class="thumbinner" style="width:302px;"><DIV class="thumbcaption">Gene Designer</DIV></DIV></DIV><P><A href="http://www.dna20.com/tools/genedesigner.php" class="external text" rel="nofollow">Gene Designer</A> [2] by DNA 2.0 is a stand alone graphical software package for manipulating biological designs.  The software enables you to combine building blocks such as regulatory DNA elements, amino acid sequences, affinity tags and cloning features. Additionally you can optimize expression by codon optimizing proteins for any expression host. There are two design views for the user, Icon and Sequence. The icon view encapsulates the DNA sequence as a specific icon indicating what type of part it is. The sequence view is an annotated DNA sequence corresponding to the Icon view. Gene Designer supports a wide variety of computational manipulations to the design such as codon optimization and can identify various DNA regions. Designs can be imported/exported in FASTA format for use with other tools and flows. There is not currently support for external databases (but is forthcoming in version 2.0).
</P><P>Here is a list of features as given by DNA 2.0:
</P><UL><LI> Manage genetic elements, constructs and libraries of constructs via drag and drop
</LI><LI> Create/Edit/Split/Merge/Save genetic elements (AA and DNA sequences)
</LI><LI> Back-translate open reading frames using a codon bias table
</LI><LI> Add/Remove restriction sites or other motifs (locally or globally)
</LI><LI> Design sequencing primers for a construct, view sequencing trace files aligned construct [**]
</LI><LI> Perform /in silico/ plasmid cloning, show circular plasmid map [**]
</LI><LI> Annotate and Measure (TM, GC%, etc.) construct regions [**]
</LI><LI> Create, Download, Edit &amp; Merge codon bias tables [**]
</LI><LI> Support user specific operations via scripting environment [**]
</LI><LI> Import and Export various file formats Fasta, GenBank, etc. [**]
</LI><LI> E-commerce enabled
</LI></UL><H2><SPAN class="mw-headline" id="References">References</SPAN></H2><OL><LI>J. Goler. BioJADE: A Design and Simulation Tool for Synthetic Biological Systems. Master’s thesis, MIT, MIT Computer Science and Artificial Intelligence Laboratory, May 2004.
</LI><LI>A. Villalobos, J. E. Ness, C. Gustafsson, J. Minshull, and S. Govindarajan. Gene designer: a synthetic biology tool for constructing artificial dna segments. BMC Bioinformatics, 7:285, June 2006.
</LI><LI>D. T. Gillespie. A general method for numerically simulating the stochastic time evolution of coupled chemical reactions. Journal of Computational Physics, 22(4):403–434, December 1976.
</LI><LI>S. Kosuri, J. Kelly, and D. Endy. Tabasco: A single molecule, base-pair resolved gene expression simulator. BMC Bioinformatics, 8:480, December 2007.
</LI><LI>Y. Cai, B. Hartnett, C. Gustafsson, and J. Peccoud. A syntactic model to design and verify synthetic genetic constructs derived from standard biological parts. Bioinformatics, 23(20):2760–2767, October 2007.
</LI><LI>M. W. Davis. A Plasmid Editor. World Wide Web, <A href="http://www.biology.utah.edu/jorgensen/wayned/ape/" class="external free" rel="nofollow">http://www.biology.utah.edu/jorgensen/wayned/ape/</A>, 2008.
</LI></OL><P>[**]Partially implemented at present, planned for version 2 release
</P><DIV class="printfooter">
Retrieved from &quot;<A href="http://2008.igem.org/Team:UC_Berkeley_Tools/RelatedWork">http://2008.igem.org/Team:UC_Berkeley_Tools/RelatedWork</A>&quot;</DIV></DIV></DIV><DIV id="footer-box"><DIV id="footer"><UL id="f-list"><LI id="t-recentchanges"><A href="/Special:RecentChanges" title="Recent changes">Recent changes</A></LI><LI id="t-whatlinkshere"><A href="/Special:WhatLinksHere/Team:UC_Berkeley_Tools/RelatedWork" title="List of all wiki pages that link here [j]" accesskey="j">What links here</A></LI><LI id="t-recentchangeslinked"><A href="/Special:RecentChangesLinked/Team:UC_Berkeley_Tools/RelatedWork" title="Recent changes in pages linked from this page [k]" accesskey="k">Related changes</A></LI><LI id="t-specialpages"><A href="/Special:SpecialPages" title="List of all special pages [q]" accesskey="q">Special pages</A></LI><LI><A href="/Special:Preferences">My preferences</A></LI></UL></DIV><DIV id="footer"><UL id="f-list"><LI id="t-print"><A href="/wiki/index.php?title=Team:UC_Berkeley_Tools/RelatedWork&amp;printable=yes" title="Printable version of this page [p]" accesskey="p">Printable version</A></LI><LI id="t-permalink"><A href="/wiki/index.php?title=Team:UC_Berkeley_Tools/RelatedWork&amp;oldid=85685" title="Permanent link to this revision of the page">Permanent link</A></LI><LI id="privacy"><A href="/2008.igem.org:Privacy_policy" title="2008.igem.org:Privacy policy">Privacy policy</A></LI><LI id="disclaimer"><A href="/2008.igem.org:General_disclaimer" title="2008.igem.org:General disclaimer">Disclaimers</A></LI></UL></DIV></DIV></DIV></BODY></HTML>