<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_NU_Kazakhstan_IoP skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><DIV id="top_title"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:NU Kazakhstan/IoP</SPAN></H1></DIV><DIV id="HQ_page"><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><P><TITLE>Valet — A free HTML5 Template by FREEHTML5.CO</TITLE></P><DIV class="container"><NAV id="fh5co-menu-wrap" role="navigation"><UL class="sf-menu" id="fh5co-primary-menu"><LI><A href="https://2017.igem.org/Team:NU_Kazakhstan" class="active" style="color:rgb(255, 255, 255)">Home</A></LI><LI><A href="#" class="fh5co-sub-ddown" style="color:rgb(255, 255, 255)">Project</A></LI><LI><A href="https://2017.igem.org/Team:NU_Kazakhstan/Parts">Parts</A></LI><LI><A href="https://2017.igem.org/Team:NU_Kazakhstan/Results">Results</A></LI><LI><A href="https://2017.igem.org/Team:NU_Kazakhstan/Design">Design</A></LI><LI><A href="https://2017.igem.org/Team:NU_Kazakhstan/Model">Model</A></LI><LI><A href="https://2017.igem.org/Team:NU_Kazakhstan/IoP">Improvement of Parts</A></LI><LI><A href="https://2017.igem.org/Team:NU_Kazakhstan/FP">Future Plans</A></LI><LI><A href="https://2017.igem.org/Team:NU_Kazakhstan/References">References</A></LI></UL><LI><A href="#" class="fh5co-sub-ddown" style="color:rgb(255, 255, 255)">Team</A></LI><LI><A href="https://2017.igem.org/Team:NU_Kazakhstan/Team">Team</A></LI><LI><A href="https://2017.igem.org/Team:NU_Kazakhstan/Collaborations">Collaborations</A></LI><LI><A href="https://2017.igem.org/Team:NU_Kazakhstan/Attributions">Attributions</A></LI><LI><A href="#" class="fh5co-sub-ddown" style="color:rgb(255, 255, 255)">Notebook</A></LI><LI><A href="https://2017.igem.org/Team:NU_Kazakhstan/Protocols">Protocols</A></LI><LI><A href="https://2017.igem.org/Team:NU_Kazakhstan/Timeline">Calendar</A></LI><LI><A href="#" class="fh5co-sub-ddown" style="color:rgb(255, 255, 255)">Human Practices</A></LI><LI><A href="https://2017.igem.org/Team:NU_Kazakhstan/HP/Silver">Public Engagement</A></LI><LI><A href="https://2017.igem.org/Team:NU_Kazakhstan/HP/Gold_Integrated">Integrated HP</A></LI><LI><A href="https://2017.igem.org/Team:NU_Kazakhstan/InterLab" style="color:rgb(255, 255, 255)">Interlab</A></LI><LI><A href="https://2017.igem.org/Team:NU_Kazakhstan/Safety" style="color:rgb(255, 255, 255)">Safety</A></LI><LI><A href="https://2017.igem.org/Team:NU_Kazakhstan/Judging" style="color:rgb(255, 255, 255)">Judging Form</A></LI></NAV></DIV><DIV id="fh5co-main"><DIV class="container"><DIV class="row"><DIV class="col-md-8 col-md-push-4" id="fh5co-content"><DIV class="content-box animate-box" id="introduction"><H2>Improvement of the part BBa_K1184000</H2><P>Supernova (SN) is a monomeric variant of the KillerRed obtained by direct modification of the gene using random mutagenesis. It is used in CALI (chromophore-assisted light  inactivation) based interventions to kill living cells by the generation of ROS. As compared to its counterpart KillerRed, it has an important advantage of being a very good fusion partner and being much less toxic to cells when not stimulated by visible light [1]. </P><P> In order to improve the cytotoxicity of the SN, our team decided to use the existing part of the SN (BBa_K1184000) made by Carnegie Mellon university iGEM team and make it membrane-bound. Experimental evidence for the membrane-bound version of the KillerRed suggests that its efficiency and action speed increased significantly as compared to cytosolic version, presumably because of the lipid oxidation [2].  It was justifiable to propose that performing the same manipulation with SN will also improve its effectiveness. For this we codon-optimized SN for Chlamydomonas reinhardtii, added to the SN gene 3 additional domains: signal peptide, transmembrane domain and C-terminal domain from SGT1– the components of the membrane-localized glycosyltransferase enzyme found indigenously in Chlamydomonas reinhardtii[3].  The complete sequence for the modified SuperNova could be observed in the part section of our page. Several programs, including TMHMM server v2.0, that predict the location of the transmembrane domains and final location of a protein, were used in order to estimate the most probable position of SuperNova in the plasma membrane. One of the graphs can be observed below.</P><H4>Figure 1. TMHMM server v2.0</H4><P>Program SignaIP 4.1 was used in order to predict whether the constructed signal peptide will function properly and help the SN to be integrated into plasma membrane.</P><H4>Figure 2. SignalP 4.1 server. S- score predicts the probability of signal peptide to be functional, Y-score predicts the position at which signal peptide will be cleaved, C- score shows combined effect of S- and Y-scores.</H4><P> In order to account for the cytotoxicity of membrane-bound SuperNova and compare it to the effect of cytosolic SuperNova, we established a cell-viability assay which is based on the absorption of the erythrosine by dead algal cells [4]. Shortly, in this experiment the OD at 526 nm is measured to find out the concentration of unviable cells as a period of time, and in the end the cytosolic and membrane bound SN effects are compared to each other. It was hypothesized that the membrane-bound SN is much efficient at killing cells by inducing the generation of ROS. The protocol for the assay can be found in the protocol section of our page.</P><P>  Unfortunately, because our team was unable to find the erythrosine on time, we were unable to perform this experiment and prove that membrane-bound SN is more effective as compared to its cytosolic counterpart. Though, there is no reason to argue that it will not work, because even its very close predecessor having more complex folding and structure could function properly in the plasma membrane. Our team lays hopes on performing this test in the future and prove SN’s functionality experimentally.</P></DIV><DIV class="content-box animate-box" id="rp"><H2>References</H2><P>[1] Takemoto K, Matsuda T, Sakai N, et al. SuperNova, a monomeric photosensitizing fluorescent protein for chromophore-assisted light inactivation. Scientific Reports. 2013

[2] Maria E Bulina, Konstantin A Lukyanov, Olga V Britanova, Daria Onichtchouk, Sergey Lukyanov &amp; Dmitriy M Chudakov, “Chromophore-assisted light inactivation (CALI) using the phototoxic fluorescent protein KillerRed”, Nature Protocols Journal, -947 - 953 (2006); Published online: 3 August 2006 

[3] The information about the part SGT1 http://www.uniprot.org/uniprot/H3JU05

[4] Quentin Béchet, Ivan Feurgard, Benoit Guieysse, Filipa Lopes, “The colorimetric assay of viability for algae (CAVA): a fast and accurate technique”, Journal of Applied Phycology, Springer Verlag, 2015
</P></DIV></DIV><DIV class="col-md-4 col-md-pull-8 left-sidebar" id="fh5co-sidebar"><DIV class="sidebar-box animate-box"><H3 class="sidebar-heading">Improvement of Parts</H3><UL class="sidebar-links"><LI><A id="intr_sb" href="#">Improvement of the part BBa_K1184000</A></LI><LI><A id="rp_sb" href="#">References</A></LI></UL></DIV></DIV></DIV></DIV></DIV><FOOTER role="contentinfo" id="fh5co-footer"><DIV class="container"><DIV class="row"><DIV class="col-md-4 col-sm-6 footer-box"><H3 class="fh5co-footer-heading">School of Science and Technology</H3><UL class="fh5co-footer-links"><LI>Nazarbayev University</LI><LI>Astana, Kazakhstan</LI></UL></DIV><DIV class="col-md-4 col-sm-12 footer-box"><H3 class="fh5co-footer-heading">Get in touch</H3></DIV></DIV></DIV></FOOTER></DIV></DIV></DIV></DIV></DIV></BODY></HTML>