<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_NU_Kazakhstan_Model skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><DIV id="top_title"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:NU Kazakhstan/Model</SPAN></H1></DIV><DIV id="HQ_page"><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><P><TITLE>Valet — A free HTML5 Template by FREEHTML5.CO</TITLE></P><DIV class="container"><NAV id="fh5co-menu-wrap" role="navigation"><UL class="sf-menu" id="fh5co-primary-menu"><LI><A href="https://2017.igem.org/Team:NU_Kazakhstan" class="active" style="color:rgb(255, 255, 255)">Home</A></LI><LI><A href="#" class="fh5co-sub-ddown" style="color:rgb(255, 255, 255)">Project</A></LI><LI><A href="https://2017.igem.org/Team:NU_Kazakhstan/Parts">Parts</A></LI><LI><A href="https://2017.igem.org/Team:NU_Kazakhstan/Results">Results</A></LI><LI><A href="https://2017.igem.org/Team:NU_Kazakhstan/Design">Design</A></LI><LI><A href="https://2017.igem.org/Team:NU_Kazakhstan/Model">Model</A></LI><LI><A href="https://2017.igem.org/Team:NU_Kazakhstan/IoP">Improvement of Parts</A></LI><LI><A href="https://2017.igem.org/Team:NU_Kazakhstan/FP">Future Plans</A></LI><LI><A href="https://2017.igem.org/Team:NU_Kazakhstan/References">References</A></LI></UL><LI><A href="#" class="fh5co-sub-ddown" style="color:rgb(255, 255, 255)">Team</A></LI><LI><A href="https://2017.igem.org/Team:NU_Kazakhstan/Team">Team</A></LI><LI><A href="https://2017.igem.org/Team:NU_Kazakhstan/Collaborations">Collaborations</A></LI><LI><A href="https://2017.igem.org/Team:NU_Kazakhstan/Attributions">Attributions</A></LI><LI><A href="#" class="fh5co-sub-ddown" style="color:rgb(255, 255, 255)">Notebook</A></LI><LI><A href="https://2017.igem.org/Team:NU_Kazakhstan/Protocols">Protocols</A></LI><LI><A href="https://2017.igem.org/Team:NU_Kazakhstan/Timeline">Calendar</A></LI><LI><A href="#" class="fh5co-sub-ddown" style="color:rgb(255, 255, 255)">Human Practices</A></LI><LI><A href="https://2017.igem.org/Team:NU_Kazakhstan/HP/Silver">Public Engagement</A></LI><LI><A href="https://2017.igem.org/Team:NU_Kazakhstan/HP/Gold_Integrated">Integrated HP</A></LI><LI><A href="https://2017.igem.org/Team:NU_Kazakhstan/InterLab" style="color:rgb(255, 255, 255)">Interlab</A></LI><LI><A href="https://2017.igem.org/Team:NU_Kazakhstan/Safety" style="color:rgb(255, 255, 255)">Safety</A></LI><LI><A href="https://2017.igem.org/Team:NU_Kazakhstan/Judging" style="color:rgb(255, 255, 255)">Judging Form</A></LI></NAV></DIV><DIV id="fh5co-main"><DIV class="container"><DIV class="row"><DIV class="col-md-8 col-md-push-4" id="fh5co-content"><DIV class="content-box animate-box" id="introduction"><H2>Introduction</H2><P>Our team aimed to genetically modify <I>Chlamydomonas reinhardtii</I> to gain the ability to:
                                                               <OL><LI>Uptake Cr (VI) through Sulfate Permease transporter</LI><LI>Reduce toxic Cr (VI) to less toxic Cr (III) by Chromium Reductase</LI><LI>To bind Cr (III) by Chromodulin oligopeptide </LI></OL></P><P>Here we represent a model of transformed  Chlamydomonas reinhardtii (Figure 1) that expresses chromate reductase (ChrR) and Chromodulin, Cr (III) binding peptide under pHSP70, pRBCS2+intron as promoter and RBCS2 and 3’UTR as terminator. 
We decided to construct a model for our system in order to visualize the biochemical pathway clearly in a stepwise manner and identify the parts of the network that could be targeted practically to improve efficiency of the construct (Figure 1). We used TinkerCell software for this purpose. 
We had assumptions as to what might be the rate-limiting steps that should be improved to increase the output of our system. We have modelled both experimental and real-time conditions to test our system using COPASI software. We built a mathematical model of the pathway using the kinetic data available for the molecules and enzymes involved. The experimental conditions include those used by our team in laboratory. The real-time conditions include the maximum concentration values of chromates and sulfates in rivers and lakes measured in Kazakhstan (Table1). 
Our model will simulate the working model and help to identify the bottlenecks of our system by manipulating the variables. Finally, we can determine the optimal conditions and output rates of the system along with limiting factors that affect the efficiency of the system.

</P><H4>Figure 1. Molecular model of biochemical pathways in genetically modified Chlamydomonas reinhardtii. </H4>
This model represents the biochemical reactions involved in Chromium (VI) reduction, where:
ChrR- gene coding chromate reductase; cod 1- gene coding for Chromodulin; pro- promoter; ter- terminator

							</DIV><DIV class="content-box animate-box" id="idea"><H2>Mathematical Modeling</H2><P>Mathematical model of our system was constructed and tested using COPASI software. The data for concentrations and kinetic values were obtained and presented in Table 1 and 2. </P><H3>Reaction design</H3><P>In order to build a whole model of our system we have designed three main reactions:
<OL><LI>Sulfate permease (Competitive inhibition )
EC_Cr6+ → Cyt_Cr6+
</LI><LI>
Chromate reductase (irreversible Henri-Michaelis-Menten)
 Cyt_Cr6+ →  Cyt_Cr3+</LI><LI>
Chromodulin (irreversible mass action)
4Cyt_Cr3+ →  complex</LI></OL></P><P>All of the parameters are given according to experimental setup and literature:

Considering data that diameter of Chlamydomonas reinhardtii is 10μm, cytoplasmic volume is about 5.210-10mL.

Also the OD value of 0.3 was considered,  that was used for our experiments. 
Then formula  below was used to find cell concentration:
Cell concentration (cell/mL)=(OD750-0.088)/(9x10^(-8))[8]
Cell concentration (cell/mL)=(0.3-0.088)/(9x10^(-8))=2.36x10^6 cells/mL
If it is assumed that 2.36x10^6 cells occupy 1 mL of medium, the extracellular volume could be approximated to 4.24x10^(-7) mL.
</P><H4>Table 1. Conditions used for mathematical modeling in COPASI</H4><H4>Table 2. Kinetic values for COPASI</H4><H3>Michaelis-Menten kinetics</H3><P>COPASI uses hybrid algorithm that is able to simulate the model both deterministically and  stochastically. Our designed reactions were simulated deterministically in time course to see the changes in the reaction rates with different conditions. The first reaction in system is a simple competitive inhibition reaction, where sulfate acts as an inhibitor for chromium (VI) uptake by sulfate permease. Increased sulfate concentration inhibits chromium (VI) uptake, whereas the depletion of sulfate in media causes an increased chromium (VI) uptake. The second reaction is governed by Michaelis-Menten kinetics, where rate of enzymatic reaction related to the concentration of substrate. The last equation represent irreversible mass action reaction where the rate is directly proportional to the product of the activities or concentrations of the reactants. Overall the reaction rates are calculated using simple Michaelis-Menten equation: 
</P><P>Reaction rates of our designed reactions were derived using COPASI:</P><H3>Simulation</H3><H4>I. Experimental condition</H4><P>Experimental conditions were set up according to Table 1

</P><H5>Graph 1: Change of extracellular Chromium (VI) concentration, in red</H5><H5>Graph 2: Change of cytoplasmic Chromium (VI) &amp; Chromium (III), in blue and green respectively</H5><H5>Graph 3: Change of cytoplasmic Chromium(III)-Chromodulin complex, in pink</H5><H5>Data Interpretation</H5><P>Deterministic simulation of the system under experimental conditions clearly indicates a decrease of Extracellular Cr (VI) concentration as can be seen from Graph 1. At same time the concentration of cytoplasmic Cr (VI) increases (Graph 2, blue line). It is used up with the course of time as taken inside and become reduced to Chromium (III) by chromate reductase. Thus, the concentration of cytoplasmic Cr (VI) decreases as the concentration of Cr (III) increases. However, Cr (III) is simultaneously used up and form a complex, which concentration increases (Graph 3). The reaction finishes at 200 s when all the reagents are used up and the complex is formed with the concentration of 0.02 mmol/ml. </P><H4>II. Sulphate deprived condition</H4><H5>Graph 4: Change of extracellular Chromium (VI) concentration, in red</H5><H5>Graph 5: Change of cytoplasmic Chromium (VI) &amp; Chromium (III), in blue and green respectively</H5><H5>Graph 6: Change of cytoplasmic Chromium(III)-Chromodulin complex, in pink</H5><H5>Data Interpretation</H5><P>
This simulation is almost similar to the first simulation, except the media is sulfate deprived. It was stated that sulfate inhibits the uptake of Cr (VI) by sulfate permease. Here we want to see how sulfate affect the output of our construct. In experimental part we were using experimental concentration of sulfate as in TAP media, in which our algae were grown. In this part, sulfate concentration is set to zero. As it can be seen from the graphs it didn’t significantly affect the efficiency of the system. The complex formation is at the same level as in experimental part with sulfate in media. The complex formed with 0.02 mmol/ml concentration in 200 s. Therefore, we can assume that the TAP media sulfate concentrations did not affect the rate of the chromate uptake during the experimental procedures. This concentration can be neglected, however extremely high concentration of sulfates could reduce the Cr (VI) uptake rates by competitive inhibition according to rate law. </P><H4>III. Real-Life condition</H4><H5>Graph 7: Change of extracellular Chromium (VI) concentration, in red</H5><H5>Graph : Change of cytoplasmic Chromium (VI) &amp; Chromium (III), in blue and green respectively</H5><H5>Graph 9: Change of cytoplasmic Chromium(III)-Chromodulin complex, in pink</H5><H5>Data Interpretation</H5><P>The real-life model simulation uses maximum concentration of chromates and sulfates in rivers and lakes of Kazakhstan found from literature, which is 17 μM and 50 mg/dm3. Graph 7 shows a decrease in Extracellular Cr (VI), it is uptaken in 200 s. The reduction of Cr (VI) to Cr (III) occur in a slower manner according to Graph 8. The complex is started to form and at around 200 s it shows concentration of  0.035 mmol/ml. This indicates that the system will work much slower in a real-life situation due to the presence of sulfates in water. </P><H5>Conclusion</H5><P>Mathematical modeling is a powerful tool for biologists to see how designed system will perform during the experiment. It helps the team to determine the variables and change them in order to obtain optimum conditions for the experiment. Moreover, our model showed the simulation of the real-life situation, thus we can predict the output and the efficiency of our designed system in real-world application. This model also could help us in further research by manipulating different variables. However, there can be some difference between modelling output and experimental result. It might be originated from several entries such as oversimplification of the model and the fact that we cannot take into account all the intercalating activities inside the cell and environmental factors. 
</P></DIV><DIV class="content-box animate-box" id="des"><H2>3D protein structure modelling and domain prediction (membrane-bound SuperNova)</H2><P>
SuperNova protein was made membrane-bound by adding 4 sequences:


1) <U>Signal peptide from Peptidyl serine alpha-galactosyltransferase gene (SGT1) (1-26 aa)</U> [9].

It correctly predicted by SignalP 4.1 server [10] (predicts signal peptides) (Figure 2):


</P><P><CENTER>Figure 2. Results of SignalP 4.1 server.</CENTER></P><P>S score – probability of signal peptide
Y score – probability of signal peptidase site
C score – combined S and Y scores</P><P>2) <U>(GGGGS)2 flexible linker</U> (27-36 aa).

3) <U>Transmembrane domain and C-terminus from SGT1 protein</U> (37-74 aa). [9]

C-terminus was also added because it gave an optimal level of confidence in the presence of transmembrane domain both by TMHMM server 2.0 and Philius transmembrane prediction server.

Transmembrane domain and signal peptide were correctly predicted by TMHMM server v2.0 (predicts transmembrane domains) [11](Figure 3):</P><P><CENTER>Figure 3. Results of TMHMM server 2.0.</CENTER></P><P>4) <U>(EAAAK)2 rigid linker</U> (75-84 aa).

Total signal peptide, transmembrane domain and linkers were predicted by TMHMM server v2.0 and Philius transmembrane prediction server [12] with high confidence values: Type confidence = 0.91 and topology confidence = 0.93 (Figure 4):</P><P><CENTER>Figure 4. Results of Philius transmembrane prediction server.</CENTER></P><P><CENTER>Figure 5. 3D model of membrane bound SuperNova.</CENTER></P><P>The Modeller 9.0 software was used to develop a homology model of the Transmembrane domain and Rigid linker (Figure 5) [13]
The Supernova crystal structure coordinates (PDB ID:3WCK) were linked to this homology model. The Charmm-GUI membrane embedding code was used to embed this structure into a lipid bilayer composed of phospholipids and mono- and diacyl glycerols . [14] The distance from the chromophore to the lipid bilayer was measured to be 23 Angstroms well within the range of the radius of effectivity (whereas 3-4 nm is a half radius of action). [15]

</P></DIV><DIV class="content-box animate-box" id="fm"><H2>References:</H2><OL><LI>Biedlingmaier, S. and Schmidt, A. (1989). Sulfate Transport in Normal and S-Deprived Chlorella fusca. Zeitschrift für Naturforschung C, 44(5-6).</LI><LI>Pootakham, W., Gonzalez-Ballester, D., &amp; Grossman, A. R. (2010). Identification and Regulation of Plasma Membrane Sulfate Transporters in Chlamydomonas. Plant Physiology, 153(4), 1653–1668. http://doi.org/10.1104/pp.110.157875</LI><LI>Yildiz, F., Davies, J. and Grossman, A. (1994). Characterization of Sulfate Transport in Chlamydomonas reinhardtii during Sulfur-Limited and Sulfur-Sufficient Growth. Plant Physiology, 104(3), pp.981-987.</LI><LI>Chromate reductase. Uniprot http://www.uniprot.org/uniprot/P0AGE6</LI><LI>Мамырбаев А.А, 2012. Токсикология хрома и его соединений. 284 с. </LI><LI>Vincent JB. The biochemistry of chromium. J Nutr. 2000 Apr;130(4):715-8. Review.</LI><LI>Kazinmetr.kz. (2017). Руководящий документ. Массовая концентрация сульфатов в водах. МВИ гравиметрическим методом. РД 52.24.483-2005 / KZ.07.00.01939-2014 / уст-т методику изм. массовой конц. сульфатов в пробах вод суши и очищенных сточных вод. [online] Available at: https://kazinmetr.kz/bd/reestr/mvi/7032/ [Accessed 31 Oct. 2017].</LI><LI>GeneArt® Cryopreservation Kit for Algae (2014). Invitrogen. 
<A href="https://assets.thermofisher.com/TFS-Assets/LSG/manuals/geneart_cryopreservation_kit_man.pdf">https://assets.thermofisher.com/TFS-Assets/LSG/manuals/geneart_cryopreservation_kit_man.pdf</A></LI><LI>The UniProt Consortium. UniProt: the universal protein knowledgebase. Nucleic Acids Res. 45: D158-D169 (2017). http://www.uniprot.org/uniprot/H3JU05</LI><LI>Nielsen, H. (2017). Predicting Secretory Proteins with SignalP. Protein Function Prediction: Methods and Protocols, 59-73.</LI><LI>Krogh, A., Larsson, B., Von Heijne, G., &amp; Sonnhammer, E. L. (2001). Predicting transmembrane protein topology with a hidden Markov model: application to complete genomes. Journal of molecular biology, 305(3), 567-580.</LI><LI>Reynolds, S. M., Käll, L., Riffle, M. E., Bilmes, J. A., &amp; Noble, W. S. (2008). Transmembrane topology and signal peptide prediction using dynamic bayesian networks. PLoS computational biology, 4(11), e1000213.
</LI><LI>Martí-Renom, M. A., Stuart, A. C., Fiser, A., Sánchez, R., Melo, F., &amp; Šali, A. (2000). Comparative protein structure modeling of genes and genomes. Annual review of biophysics and biomolecular structure, 29(1), 291-325.</LI><LI>Vieler, A., Wilhelm, C., Goss, R., Süß, R., &amp; Schiller, J. (2007). The lipid composition of the unicellular green alga Chlamydomonas reinhardtii and the diatom Cyclotella meneghiniana investigated by MALDI-TOF MS and TLC. Chemistry and physics of lipids, 150(2), 143-155.</LI><LI>Takemoto, K., Matsuda, T., Sakai, N., Fu, D., Noda, M., Uchiyama, S., ... &amp; Ayabe, T. (2013). SuperNova, a monomeric photosensitizing fluorescent protein for chromophore-assisted light inactivation. Scientific reports, 3, 2629.</LI></OL></DIV></DIV><DIV class="col-md-4 col-md-pull-8 left-sidebar" id="fh5co-sidebar"><DIV class="sidebar-box animate-box"><H3 class="sidebar-heading">Modeling</H3><UL class="sidebar-links"><LI><A id="intr_sb" href="#">Introduction</A></LI><LI><A id="idea_sb" href="#">Mathematical Modeling</A></LI><LI><A id="des_sb" href="#">3D modelling and domain prediction of SuperNova</A></LI><LI><A id="fm_sb" href="#">References</A></LI></UL></DIV></DIV></DIV></DIV></DIV><FOOTER role="contentinfo" id="fh5co-footer"><DIV class="container"><DIV class="row"><DIV class="col-md-4 col-sm-6 footer-box"><H3 class="fh5co-footer-heading">School of Science and Technology</H3><UL class="fh5co-footer-links"><LI>Nazarbayev University</LI><LI>Astana, Kazakhstan</LI></UL></DIV><DIV class="col-md-4 col-sm-12 footer-box"><H3 class="fh5co-footer-heading">Get in touch</H3></DIV></DIV></DIV></FOOTER></DIV></DIV></DIV></DIV></DIV></BODY></HTML>