<HTML xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en" dir="ltr">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki  ltr ns-0 ns-subject page-Team_Harvard_Comm"><DIV id="globalWrapper"><DIV id="top-section"><DIV id="p-logo"><A href="/Main_Page" title="Main Page">&quot;
	    </A></DIV><DIV id="menubar" class="left-menu"><UL><LI class="selected"><A href="/Team:Harvard/Comm">Page               </A></LI><LI class="new"><A href="/wiki/index.php?title=Talk:Team:Harvard/Comm&amp;action=edit&amp;redlink=1">Discussion               </A></LI><LI><A href="/wiki/index.php?title=Team:Harvard/Comm&amp;action=edit">View source               </A></LI><LI><A href="/wiki/index.php?title=Team:Harvard/Comm&amp;action=history">History               </A></LI><LI style="color:#808080;cursor:default">teams</LI></UL></DIV><DIV class="right-menu" id="menubar"><UL><LI id="pt-login"><A href="/wiki/index.php?title=Special:UserLogin&amp;returnto=Team:Harvard/Comm" title="You are encouraged to log in; however, it is not mandatory [o]" accesskey="o">Log in</A></LI></UL></DIV><DIV id="search-controls"><FORM action="/Special:Search" id="searchform"> </FORM></DIV></DIV><DIV id="content"><H1 class="firstHeading">Team:Harvard/Comm</H1><DIV id="bodyContent"><H3 id="siteSub">From 2009.igem.org</H3><DIV style="width: 650px; height: 800px; left: 10px;"><TABLE width="650px" border="0" cellspacing="10" cellpadding="0"><TBODY><TR><TD><H2>Bacteria-to-Yeast Communication</H2>
/***********************************************************************************************************************/
<FONT size="+2"><P><B> PROJECT BACKGROUND</B></P></FONT><DIV class="center">,</DIV><FONT size="+1">Bacteria to yeast communication as a means to bridge a physically separated canonical lac operon using light </FONT><P><B>Project Overview: </B> Communication requires three components: a sender, a means of communication, and a receiver.  In the case of our bacteria-to-yeast optical signaling system, our signal senders are E. coli bacteria, expressing a red variant of luciferase from the firefly Photinus pyralis. Expression of this protein is controlled by IPTG induction. </P><P> The second component required is a means of communication. The means of communication in our system is the red bioluminescence produced by the E. coli. This optical signal is received by the yeast, which engineered to be light-sensitive via the expression of a two-hybrid system incorporating a phytochrome (PhyB)  from the plant Arabidopsis thaliana, and its interacting factor PIF3. Red light causes a change in conformation of the PhyB protein which allows it to interact with its interacting factor PIF3. </P><P> In a two hybrid system, one of these proteins of interest is fused to a Gal4 activation domain and one is fused to a Gal4 DNA binding domain. When these two domains are brought into close enough proximity via the interaction of the PhyB and PIF3, they result in activation of transcription of a gene of our choice under control of the Gal1 promoter, in this case, the lacZ gene. </P><P> The lacZ gene was chosen as the readout for our system because it is simple to assay for its expression, and because it endowed our system with a very interesting property: this system not only allows for interspecies optical communication, but enables us to optically bridge a physically separated canonical lac operon using light as a trans-acting factor. In other words, we have separated the induction portion and the expression portions of lac operon expression both spatially and temporally, using light as a means to bridge the separation over time and space. </P><P><B>In summary:</B> bacteria communicate the presence of IPTG in their culture to the yeast cells via red light. The red light is produced by luciferase in the bacteria. The yeast absorb the red light using a plant phytochrome. In response to this light, the bacteria produce beta-galactosidase, which can then be assayed for with an X-gal or ONPG assay. </P><DIV class="center">,</DIV><FONT size="+1"><P><B>What is the lac operon?</B></P></FONT><P> The lac operon is a set of genes in bacteria that code for the enzymes used to break down lactose, including the enzyme beta-galactosidase encoded by the lacZ gene. Bacteria are capable of using both glucose and lactose for energy. Because it takes more steps to break down lactose into usable energy, bacteria prefer to use glucose if it is available. However, in the absence of glucose, bacteria can use the enzyme beta-galactosidase to break down lactose into its usable components, glucose and galactose. </P><DIV class="center">,</DIV><P>The lac operon is particularly important to molecular biology because its regulation was the first complex genetic regulatory mechanism to be understood. In order to work efficiently, cells must only turn on genes when their transcription products are needed. Having genes being expressed all the time would be inefficient at best, and most likely extremely detrimental. Hence, most organisms have complicated mechanisms of gene regulation to ensure that genes are only expressed when they are needed. The lac operon has a two part regulatory system: in the absence of lactose, gene expression is turned off by the lac repressor. In the absence of glucose, it is turned on by the Catabolite Activator Protein (CAP). </P><P>In our experiments, we were primarily concerned with the lac repressor component of this system. When lactose is absent, the repressor is bound to the gene promoter, preventing transcription. It only stops repressing transcription when lactose (or a lactose mimic, like IPTG) binds to it and causes it to change conformation and unbind from the DNA. Transcription is then able to occur, resulting in transcription of the genes under control of the lac promoter. </P><P><B>How did we split the lac operon?</B> Typically both de-repression and transcription occur within the same cell, but we wanted to use the principles of synthetic biology to decouple these events and spatially separate them into two different cells.  Using IPTG induced bioluminescence produced by bacteria to signal to yeast and active the PhyB/PIF3 two hybrid system controlling beta galactosidase production, we were able to de-repress the lac operon in the bacteria, but produce the gene product beta-galactosidase in the yeast cell. Thus, we used light to bridge a physically separated lac operon. This can also be seen as a form of bacteria-to-yeast optical communication. </P><FONT size="+2"><P><B>THE SYSTEM</B></P></FONT><FONT size="+1"><P>System Input: Derepression of expression of red luciferase</P></FONT><P><B>IPTG Induction in bacteria:</B> IPTG, or Isopropyl β-D-1-thiogalactopyranoside, is a biological compound that mimics allolactose, the lactose derivative that turns on transcription of the lac operon. The lac operon is a set of genes in E. coli and many other bacteria that is used to metabolize lactose.   In molecular biology, IPTG is frequently used to induce gene transcription of your gene of choice. If you put your gene of interest under the control of the lac promoter by replacing the lac  genes with your desired gene, then the expression of that gene can be induced by the addition of IPTG. This allows for easy inducible control of gene transcription. In the case of this experiment, a red variant of firefly luciferase was under control of the lac promoter, so addition of IPTG would induce luciferase expression. </P><P>On a molecular level, IPTG induces gene transcription by binding to and inhibiting the function of a repressor of the lac operon—it stops the repressor from repressing transcription, thus allowing transcription of the genes. </P>
height=&quot;350&quot;/&gt;,<P><B>Luciferase function.</B> Luciferase is the enzyme used by fireflies to create bioluminescence. Unlike fluorescent proteins which emit light in response to excitation by incoming photons, luciferase creates light by oxidation of a substrate called luciferin. Production of bioluminescence by luciferase requires the presence of functional luciferase, luciferin substrate, and oxygen. An interesting side note about luciferase: fireflies are able to control their characteristic flashes of light not by altering luciferase or luciferin concentrations, but by altering the amount of oxygen available for the reaction.  Unmodified luciferase emits in the yellow-green spectrum. However, a number of variants of luciferase that emit in different spectra have been generated in labs, including green-shifted and red-shifted variants. In these experiments, we primarily use the red-shifted luciferase variant, referred to in our wiki as Red Luciferase. </P><DIV class="center">,</DIV><FONT size="+2"><P><B>Receiving the Red Light Signal </B></P></FONT><DIV class="center">,</DIV><P><B>PhyB:</B> PhyB is a light sensitive phytochome from Arabidopsis thaliana, which is involved in the growth and development of young seedling plants. PhyB is composed of a peptide and a covalently bound prosthetic group, PCB, which endows the protein with its photosensitive capabilities. PhyB has with two conformational states, Pr and Pfr, between which it can interconvert when struck by the right wavelength of light. While PhyB in the Pr state cannot bind to PIF3, PhyB in the Pfr state can bind to PIF3. In Arabidopsis, the PhyB/PIF3 complex translocates to the nucleus and binds DNA, resulting in transcriptional changes. The Pr state converts to Pfr when struck by red light, and the Pfr form reverts to the Pr form when struck by far red light.  In plantae, the PhyB/PIF3 system controls de-etiolation in seedlings.</P><P><B>PIF3:</B> PIF3 stands for Phytochrome Interacting Factor 3. As is evident from its name, the major function of PIF3 is that it interacts with PhyB. In its Pr form, PhyB is unable to interact with PIF3, but upon conversion to its Pfr form via red light, it becomes able to bind to PIF3. In plantae, this binding results in translocation to the nucleus and changes in gene transcription. When exposed to far-red light, the PhyB converts back to its Pr form, and can no longer bind PIF3. The molecular structures of PhyB and PIF3 are unknown, although they are both thought to be globular. </P><P><B>Is light from luciferase in the right spectrum to be absorbed by PhyB?</B> One of the primary requirements of the system is that the yeast be able to sense the light from the bacteria. In other words, the bacterial luciferase must emit at a wavelength that can be absorbed by the PhyB in order for the signal to be relayed to the yeast. Based on published emissions spectra for red luciferase and absorbance spectra for PhyB, it appeared that their spectra overlapped enough that light from the luciferase would be able to induce the conformational change in PhyB necessary to setting off the signaling cascade resulting in gene expression. </P><DIV class="center">,</DIV><P>(Right from Gambetta et al., Left from Branchini et al.)</P><P>To test this hypothesis, we added PCB extract to liquid bacterial culture of cells expressing red luciferase under IPTG induction. By eye, there was a very apparent decrease in red light emission in the culture with PCB (dissolved in DMSO) as opposed to the control with DMSO alone. Cultures had the same density. This implies that PCB is absorbing a significant fraction of the light emitted by the luciferase. Since PCB’s absorption spectrum is almost identical to the absorption spectrum of the holoprotein PhyB (PhyB bound to PCB), this is indirect evidence that PhyB can successfully absorb in the spectrum emitted by red luciferase. It will be necessary to quantify this decrease in luminescence to determine if it is statistically significant, but the results were striking by eye. </P><FONT size="+1"><P><B>The Output: Changes in transcription and completion of the lac operon</B></P></FONT><DIV class="center">,</DIV><P><B>What is a two hybrid system?</B> A yeast two-hybrid system is a system that uses protein binding to induce transcription of a gene of interest. One protein of interest is fused to the Gal4 activation domain, while another is fused to the Gal4 DNA binding domain. When the two proteins of interest are brought into close enough proximity, the complex is able to bind to DNA and recruit RNA polymerase to initiate transcription. </P><P><B>The specifics of the two-hybrid system.</B> When PhyB is fused to the GAL4 DNA binding domain (DBD) and PIF3 is fused to the GAL4 activation domain (AD), this creates a two-hybrid-system which can be used to induce transcription of genes under the control of the GAL1 promoter.  When the DBD- PhyB is excited by red light, it changes into the Pfr state and binds to the PIF3-AD, which then results in transcription of our desired gene.  Induction of transcription can be stopped by conversion of PhyB into the Pr state (unable to bind to PIF3-AD) by exposing it to far-red light. This system was initially developed by Shimizu-Sato et al. As a part of the iGEM project, we have successfully recapitulated this system using lacZ as our reporter gene. The red light can be from any source provided it is in the right range of wavelengths. </P><DIV class="center">,</DIV><P><B>The fusion proteins we made.</B> To conduct our experiments, it was necessary to create the requisite fusion proteins. Although the two-hybrid system has been published before by Shizimura-Sato et al, we were not able to obtain the constructs from them in a timely fashion. As a result, we had to clone the constructs from scratch.</P><P>In order to obtain the necessary PhyB and PIF3 genes, we did an RNA extraction from a Columbian Arabidopsis thaliana plant, a gift from the Pierce lab at Harvard University. We then did RT-PCR on the RNA extract to clone the PhyB and PIF3 genes. These genes were then inserted into plasmids containing a Gal4 DNA binding domain and a Gal4 DNA activation domain, respectively. The fusion proteins were made in the orientations indicated below. </P><DIV class="center">,</DIV><P>In these diagrams, PhyB is phytochrome B, NS is nuclear localization signal, DBD is DNA binding domain, ADH is the alcohol dehydrogenase promoter. The green circle is the start site of transcription, and the red circle is a terminator. Trp is the gene for tryptophan production, Leu is the gene for leucine production. These plasmids are non-integrating vectors for transformation into yeast, and both contain a 2 micron origin of replication, making them high copy number plasmids. </P><P>Selection for yeast colonies containing only the desired plasmids is done by growing the plasmids on medium deficient in particular amino acids, the genes for which have been knocked out in the lab strain of yeast.</P><P><B>Nuclear localization and binding the promoter.</B> In plants PhyB and PIF3 are cytoplasmic when not associated with each other. Upon binding, the complex is translocated to the nucleus, where it is able to bind to its promoter and induce transcription. As a part of our yeast-two-hybrid system, we added nuclear localization signals to the proteins. Thus, in the yeast, the PhyB-DNA Binding Domain and PIF3-Activation Domain fusion proteins are already located in the nucleus. It is important to note that PhyB/PIF3 have a DNA recognition sequence and inherent DNA binding and activation domains, since their function in plants is to alter gene expression. Nonetheless, rather than use the recognition sequence that is used in plants, we are using the Gal 4 binding and activation domains and the Gal1 promoter. It would be theoretically possible to use the PhyB/PIF3 recognition sequence that is used in nature and forgo the two-hybrid system altogether. However, because of how well characterized the function of two-hybrid-systems is, we elected to use it.</P><FONT size="+1"><P><B>Assaying for completion of the Operon</B></P></FONT><P><B>Beta-gal assay.<B> The ultimate output of our system is the production of beta-galactosidase, the major transcriptional product of the lac operon. </B></B></P></TD></TR></TBODY></TABLE></DIV>'
<DIV style="position: absolute; left: 630px; top: 240px; height: 800px; width: 300px; padding: 3em;"><TABLE width="300px" border="0" cellspacing="10" cellpadding="0"><TBODY><TR><TD><H2><A href="https://2009.igem.org/Team:Harvard/Team">Team</A></H2><H2><P>Projects</P></H2><UL><LI><A href="https://2009.igem.org/Team:Harvard/Comm">Bacteria-to-Yeast Communication</A></LI><LI><A href="https://2009.igem.org/Team:Harvard/Laser">Characterization</A></LI><LI><A href="https://2009.igem.org/Team:Harvard/PCB">PCB Biosynthesis in Yeast</A></LI><LI><A href="https://2009.igem.org/Team:Harvard/Split">Split Luciferase</A></LI><LI><A href="https://2009.igem.org/Team:Harvard/Blackboard">Other Possible Reporters</A></LI></UL><H2><P><A href="https://2009.igem.org/Team:Harvard/Parts">Parts</A></P></H2><H2><P><A href="https://2009.igem.org/Team:Harvard/References">References</A></P></H2><H2><P>Lab Notebook</P></H2><UL><LI><A href="https://2009.igem.org/Team:Harvard/Daily">Daily</A></LI><LI><A href="https://2009.igem.org/Team:Harvard/Protocols">Protocols</A></LI><LI><A href="https://2009.igem.org/Team:Harvard/Recipes">Recipes </A></LI></UL><H2><P><A href="https://2009.igem.org/Team:Harvard/Ethics">Ethics</A></P></H2><H2><P><A href="https://2009.igem.org/Team:Harvard/Safety">Safety</A></P></H2><H2><P><A href="https://2009.igem.org/Team:Harvard/Acknowledgements">Acknowledgements</A></P></H2></TD></TR></TBODY></TABLE></DIV><DIV class="printfooter">
Retrieved from &quot;<A href="http://2009.igem.org/Team:Harvard/Comm">http://2009.igem.org/Team:Harvard/Comm</A>&quot;</DIV></DIV></DIV><DIV id="footer-box"><DIV id="footer"><UL id="f-list"><LI id="t-recentchanges"><A href="/Special:RecentChanges" title="Recent changes">Recent changes</A></LI><LI id="t-whatlinkshere"><A href="/Special:WhatLinksHere/Team:Harvard/Comm" title="List of all wiki pages that link here [j]" accesskey="j">What links here</A></LI><LI id="t-recentchangeslinked"><A href="/Special:RecentChangesLinked/Team:Harvard/Comm" title="Recent changes in pages linked from this page [k]" accesskey="k">Related changes</A></LI><LI id="t-specialpages"><A href="/Special:SpecialPages" title="List of all special pages [q]" accesskey="q">Special pages</A></LI><LI><A href="/Special:Preferences">My preferences</A></LI></UL></DIV><DIV id="footer"><UL id="f-list"><LI id="t-print"><A href="/wiki/index.php?title=Team:Harvard/Comm&amp;printable=yes" title="Printable version of this page [p]" accesskey="p">Printable version</A></LI><LI id="t-permalink"><A href="/wiki/index.php?title=Team:Harvard/Comm&amp;oldid=169721" title="Permanent link to this revision of the page">Permanent link</A></LI><LI id="privacy"><A href="/2009.igem.org:Privacy_policy" title="2009.igem.org:Privacy policy">Privacy policy</A></LI><LI id="disclaimer"><A href="/2009.igem.org:General_disclaimer" title="2009.igem.org:General disclaimer">Disclaimers</A></LI></UL></DIV></DIV></DIV></BODY></HTML>