"
Page
Discussion
View source
History
teams
Log in
 
Team:Harvard/Daily
From 2009.igem.org
Daily Lab Notebook
/***********************************************************************************************************************/
Week 1: 6/10/09 - 6/12/09
First official meeting of the summer: finalizing the project details, setting up a tentative schedule.
We spent the first meeting narrowing in on the system we would use for our upcoming projects. After comparing the bacterial light sensing system with that of the modified yeast 2-hybrid system a consensus was reached to attempt the yeast system first. The yeast phytochrome based system seemed more sensitive and the bistable light inducible system would give us an added element of regulation, in that the system could be rapidly reset using IR wavelengths. This would be useful in a "bio-blackboard" configuration.
As a backup though, we should be prepared to revert to the bacterial system.
The potential problem for the yeast system is that phycocyanobilin (PCB) needs to be exogenously supplemented. It may need to be extracted from a suitable (algae?) source or the bacterial PCB biosyntheis pathway may need to be cloned into yeast, if possible. Also, if using yeast as a "sender" with red-luciferase, the luciferin substrate for luciferase also needs to be added externally. Uptake should occur if the media is made acidic.
David pointed out a more recent intein system that also utilizes the same Phytochrome/PIF3 domains that the modified yeast two-hybrid system uses.
We carefully studies schematics of the various light responsive genetic systems, including an attempt at adding an "inverter" component into the
,
from a few years ago, and the system created by
.
Week 2: 6/15/09 - 6/19/09
Lab Meeting Notes.
DB3.1 LB absorbance and scattering? What did the old IGEM team dilute in?  Intein people had a modification to the PCB extraction protocol—look that up.  PCR the genes we need out of a cDNA library.  Churchill lab has everything?—ask them for Arabadopsis cDNA library.  Arabadopsis 2 hybrid library Email Woody Hastings? Fluorescent dinoflagellates! Shake them to read by.
Assembly of low-promoter from oligos.  DTT is a reducing agent, ATP is energy, required for various enzynmes in the system to work.
First step is to take phosphorylate the resuspended oligos. The 5 prime ends need to be phosphorylated so that they can be ligated. Oligos normally come without the five prime phosphate.  Mix equimolar amounts of the four oligos and label oligo mix, 20 uL of each oligo.  Use PCR tube.
20 uL H20
6 ul oligo mix
4 ul 10x PNK buffer
4 uL DTT
4 ul ATP
2 ul PNK
Total:  40 uL…….keep on ice.
Samples put in thermocycler. See protocol for program, called “Assembly”.
Tuesday
For obtaining the PhyA, PhyB, and PIF3 genes, we have a number of methods we are pursuing.  1. Obtain cDNAs from another lab
a. Tuesday—The Mathews lab is willing to give us PhyA, but does not have PhyB or PIF3. They can help us with the construction of the PhyB and PIF3. We should ask them if they have a protocol they follow….
b. Wednesday--I have emailed the Mathews lab back, and will hopefully be meeting with Sarah Mathews later this week.
2. Obtain cDNAs from Biobricks registry
a. Tuesday--Pulled PhyB and PIF3 from the registry, as well as the PCB synthesis genes. The PCB synthesis genes look good according to registry sequence, but PhyB and PIF3 cite sequencing irregularities, so I am not hopeful.
b. Wednesday—Oliver compared the sequences from NCBI to the sequences given in the BioBricks registry for the PhyB and PIF3 and it does not look like they are correct, so that is not promising. We will continue with growing up the PCB biosynthetic enzymes though.
3. Obtain cDNAs from a cDNA library
a. Tuesday—No one seems to have a sample of the cDNA library we can have. Asked in the lab plant kid from 261r was in, he’s checking for us but is not hopeful. Purchase of a cDNA library is too expensive, $1200)
4. Obtain Arabidopsis plant and make a cDNA library, and from that clone out the genes of interest
a. Tuesday—The Pierce lab has kindly given us two lovely Arabidopsis plants, and we have ordered the necessary kits and mortar and pestle to do the RNA extraction. Tomorrow must design primers for sequencing and for cloning the genes out…these can be multipurpose.
b. Wednesday--
5. Obtain a library from TAIR. Arabidopsis.org
a. Wednesday—We are going to order libraries from the Arabidopsis people. They are very cheap, $5 a line, with $125 shipping fee total per order, so it’s fairly cheap. Shipping is combined for all orders.
b. 4 different libraries ordered.
Initial email to Mathews
Hi Dr. Mathews,
My name is Amrita Goyal, and I am a member of the Harvard iGEM team. We are looking for the cDNA for the PhyB, PhyA, or PIF3 components of the light-activated signalling system in plants. We were wondering if you had these genes available in a suitable vector that we could obtain. Our goal is to subclone these genes into a yeast-two-hybrid system. We would really appreciate it if you had a sample available that one of our teammates could pick up.
Also, if you have an extraction protocol for phycocyanobilin, or know someone who does, we would also appreciate that information. Thank you so much!
Sincerely,
Amrita Goyal and the Harvard iGEM team
Mathew’s Response to me
Hi Amrita,
Sorry for the delay in responding -- I've been doing jury duty.
We can help with a PHYA cDNA clone, in either pUC 18 or a bluescript vector.  We don't have PHYB or PIF clones, or an Arabidopsis cDNA library.  But depending on your timeframe, I could help with PHYB and PIF clones.
For a phycocyanobilin extraction protocol, I recommend you get in touch with someone in Clark Lagarias' lab (cc'd on this message).  Their web page is at:  http://www.mcb.ucdavis.edu/faculty-labs/lagarias/
We may have some phytochromobilin, but we have no phycocyanobilin.
Let me know when you'd like to pick up some PHYA clone, and I'd love to hear a bit more about your experiments.
Cheers,
Sarah
My response to Mathews
Hi Professor Mathews,
I hope that jury duty went well!
If we could get the PHYA cDNA in the pUC18 vector from you that would be fantastic. We would very much appreciate your help with making the PHYB and PIF3. We acquired a couple of Arabidopsis plants from a grad student in Naomi Pierce's lab yesterday and we were planning to do an RNA extraction and clone the PHYB and PIF3 out ourselves, if you think that is the best way to proceed.
If you think the phycochromobilin would work, we would also appreciate a sample of that. We did get Spirulina and are planning to attempt a PCB extraction soon.
What we are planning to do for our iGEM project is to create a cellular blackboard, using the Phy system in yeast. We want to use the PhyA and/or PhyB to turn on transcription of a green luciferase, so we can "write" on the blackboard using a red laser pointer. We would then "erase" the blackboard with far-red light to turn off luciferase transcription (although we will just have to wait for the luciferase to degrade then).
We were also thinking of engineering a system where we have two populations of yeast cells communicate with eachother using light. We would turn on production of a red luciferase in one population by shining a laser on it. The light produced by that population would then be used to turn on production of the luciferase in another population of cells.
If you have any thoughts or advice on either of these projects please let me know!
I can come by any time today or after lunch tomorrow, let me know when is most convenient for you! I actually have a couple of questions for you if you have a few minutes to talk in person in the next couple of days. Let me know what your schedule looks like!
Thanks,
Amrita
Response from Lagarias, jclagarias@ucdavis.edu
Colleagues,
I have been contacted for a related request by a Wash University iGEM Team. I told them that we would have to complete an MTA to send any plasmids (and we do not have PIF clones that we are at liberty to send; they were obtained from other labs). FYI, I have attached a protocol for PCB isolation.
Best.
Clark Lagarias
PREPARATION OF PHYCOCYANOBILIN Lagarias Lab Method
3E-phycocyanobilin isolation: Lagarias Lab Method (Terry, MJ, MD Maines, and JC Lagarias. 1993. Inactivation of Phytochrome-Chromophore and Phycobiliprotein-Chromophore Precursors by Rat Liver Biliverdin Reductase. J. Biol. Chem. 268(35):26099-26106): 3E-phycocyanobilin (PCB) was prepared from lyophilized Spirulina platensis (Sigma) using a method similar to that described by Beale and Cornejo (Beale & Cornejo (1991a) J. Biol. Chem. 266, 22328-22332; Beale & Cornejo (1991b) J. Biol. Chem. 266, 22333-22345).
1. Spirulina powder was rehydrated in deionized water (30 ml/g dry weight) for 10 min.
2. The slurry was centrifuged at 30,000 x g for 20 min and the deep blue supernatant was decanted.
3. Phycocyanin was precipitated from this supernatant with 1% (w/v) TCA by incubation for 1 h at 4oC in the dark and then collected by centrifugation at 30,000 x g for 20 min.
4. After washing with methanol (2 x 20 ml/g Spirulina powder), the blue pellet was resuspended in methanol (2 ml/g Spirulina powder) containing 1 mg/ml HgCl2.
5. Following incubation for 20 h at 42oC in darkness, the protein was removed by centrifugation at 10,000 x g for 10 min.
6. 2-Mercaptoethanol (1 ul/ml) was then added to precipitate the dissolved mercuric ion, which was also removed by centrifugation (30,000 x g for 10 min).
7. The crude bilin mixture was then diluted 10-fold with 0.1% (v/v) trifluoroacetic acid and applied to a C18 Sep-Pak cartridge (Waters-Millipore Corp., Milford, MA).  The Sep-Pak cartridge was sequentially washed with 0.1% (v/v) trifluoroacetic acid (2 x 3 ml) and acetonitrile/0.1% trifluoroacetic acid (20:80; 2 x 2 ml) followed by elution of the crude bilin mixture with 3 ml acetonitrile/0.1% trifluoroacetic acid (60:40) and drying in vacuo.
8. Spectral assay of the crude bilin mixture is performed at this point (use a 1/100 dilution) and a typical yield of 4 umol phycocyanobilin per 6 g dry weight Spirulina can be obtained.
9. This crude phycocyanobilin is quite pure but can be further purified by C18 reverse phase HPLC using a Varian 5000 liquid chromatograph and a Beckman Ultrasphere ODS column (4.6 x 150 mm; 5 um particle diameter).  The solvent system used was ethanol/acetone/water/acetic acid, 19:14:66:1 (v/v/v/v) with a flow rate of 1.5 ml/min, and the column eluate was monitored at 370 nm (Cornejo et al, 1992).
10. HPLC-purified 3E-phycocyanobilin was concentrated using a C18 Sep-Pak as described above, dried in vacuo and stored at -20oC.  Before use, phycocyanobilin was dissolved in dimethyl sulphoxide to a concentration of 1-1.5 mM.
11. An aliquot of each stock solution was diluted 200-fold into 2% HCl/methanol to estimate the bilin concentration spectrophotometrically.  The molar absorption coefficient of 47,900 M-1 cm-1 at 374 nm for 3E-phycocyanobilin (Cole, WJ, DJ Chapman, and HW Siegelman. 1967. The structure of phycocyanobilin. J. Am. Chem. Soc. 89:3643-3645) were used for these determinations.
1. HO1
a. BB—726 bp, 4425 bp.
2. PycA
a. BB—750 bp, 4425 bp.
3. PhyB
a. BB—1863 bp
b. TAIR—3996 bp
4. PIF3
a. BB—302 bp
b. TAIR—1790 bp
Truncated is better
There is a truncated PhyA
PhyB is main mediator of low fluence red light, conversion to Far form. W PhyA, main mediator of responses to far red light, in shaded environments. 2 modes—responses to prolonged exposure to far red light, and low fluence exposure if PhyA totally held in the dark. Still converts w red. Low fluence response does not matter on quality of light—any small exposure to visible light.
PhyA will bind PIF3, but in plantae, never. Getting phyA into nucleus FI1 needed for translocation to nucleus. Nuclear localization signal.
Lagarias—has been trying to get both phytochromobilin and the phytochrome to express concurrently in yeast, most people use the bacterial for heterologous expression. Clarke would be the person to ask to see hwat he has gotten to work. –email him he will respond.
They have primers for all Phy B and could design primers for just truncated section. Try just amplifying from genomic DNA in a single exon. Def try from genomic DNA.
Probbablyjust want to work with PhyB. In some ways could be advantageous s to work with Phy A—differs in how sensitive it is to light—once it sees light it degrades rapidly. Goes into proteosome degradation, and maybe a proteosome box is exposed? Maybe that’s just specific in Arabidopsis. It’s an E3 ligase. We need to look at specific structure of the protein. In plants PhyA builds to high levels in the dark and disappears in light—degradation and turning down. In yeast dunno how it would behave. –maybe working with a couple or three just to make sure one is robust would be good.
Def worth trying both PhyA and PhyB—espworking with full length. Don’t know if truncated has same sensitivity characteristics. PhyA is most sensitive.
You need to express the protein in the dark. The trick is the conditions under which you express the protein—use minimum minimum green light. Just put in green filters, 25 w bulb. If you can do that….The person collecting the data has a spectroradiometer and he had built a red and far-red filter to push them between the red and far red films in the cuvettes. She will see if they still have that instrument. The trick will be the conditions.
Mathews: Primer sequences to help get useful fragments—send messages to truncated clone people
You will never convert everything back from Pfr. 3 fctors effecting rate—it will dark revertat a specific rate, you can push it back with far red, and then temperature—revert faster if pulsed with far red.
We need the vectors for the yeast-two hybrid system, and the strains of yeast they used. We are particularly looking for the strains that have the UAS driven reporter if you have them.
Yeast two hybrid system
Strain missing Gal4 and Gal80. Gal4 has DNA binding domain and activation domain. Binds to several promoters, one is pGAL1. There is also a GAL2 and GAL4. The strain usually has these two deletions, and those are redulated under galactorse, truned on by it. In the absence of galalctose it is blocked by Gal80. Usually in one of the GAL promoters you have GAL4 deleted and GAL80 deleted. You usually have a reporter, and you can measure by adding OMPG or whatever substrate. The strains come in two mating types, A or alpha. They are haplopd, when opposite mating types meet they fuse.
PRS303—0 means it integrates. You put in your gene of interest and you cut this plasmid in the selectable marker and linearize the plasmid and it integrates into the yeast genome and there is enough homology so youhave a good copy of the leucine gene. Andyou can have it driven under whatever promoter.
PIF3—FL
PIF3—Partial
PhyB—Partial
We need full length Gal4—this will
PCB questions--
Do we need to HPLC it? Will it work without it?
Can we just insert the genes into the yeast and have it express the enzyme?
Resources www.addgene.org
www.atcc.org
www.sgd.org
www.arabidopsis.org
Week 3: 6/22/09 - 6/26/09
Discovery that there is a split luciferase, which works on the order of seconds. The concept would be to attach fragments to PIF3 and PhyB and then
Blackboard
Red light district
Signal transmission wire
C-terminus was tagged with phyB (phyB–YFPC). from Phee paper.
red mutant (S286N) luciferase . From wiki.
Renilla Luciferase
http://partsregistry.org/Part:BBa_J52008
Firefly Luciferase
http://partsregistry.org/Part:BBa_I712019
Use of a split-luciferase reporter to indicate interaction of PhyB and PIF3. I propose that for our system, we tag the PIF3 and PhyB with luciferase to form a split luciferase reporter system. When PhyB is stimulated with red photons, it undergoes a conformational change which gives it the ability to bind to PIF3 (phytochrome interacting factor 3). In order to reduce the amount of time between “writing” and “erasing” our theoretical blackboard, we had initially considered using a destabilized luciferase, but that did not solve the problem of  The theory behind a split-luciferase reporter. This technique was developed to help study protein-protein interactions in vitro, in vivo in cell culture, and in vivo in animals (like mice). The basis of a split luciferase reporter system is that when luciferase (like many other proteins, including GFP and ubiquitin) is cut in half, and each half fused to a protein of interest, the two halves lack enzymatic activity. However, when they are brought into close enough proximity to one another they reconstitute the functional enzyme. The split luciferin has been engineered to have minimal nonspecific affinity between the two halves, so it should only associate upon interaction of the two proteins the halves are fused to. Thus, upon photoactivation of PhyB, the binding of Pif3 and PhyB should facilitate the reconstitution of the functional luciferase, in the presence of the correct luciferin, result in a fluorescent signal.  Timing of luciferase reconstitution. Use of a split-luciferase system will allow us to turn on and off fluorescence in basically real time, without the delay of transcription to turn on the luciferase and degradation to turn it off. Studies have shown that association and dissociation occur within minutes. The split-luciferase reporter system has the distinct advantage over split-GFP systems that reconstitutiton of the luciferase is reversible, while reconstitution of the GFP fluorphore is irreversible. The GFP system is also disfavorable because GFP requires a high amount of excitation energy, and has high background, thus obscuring the signal (Cissell et al, 2009).  Luciferase species choice.  There are several options for which luciferase we can use, including firefly luciferase, Renilla luciferase, and Gaussia luciferase.  Firefly luciferase is from the firefly (beetle), Photinus pyralis, and is 61 kD. It uses luciferin in the presence of oxygen, ATP, and Mg2+ to produce bioluminescence in the range of 550-570 nm (greenish yellow) (http://www.promega.com/paguide/chap8.htm#title2).  When Luker et al examined the kinetics of cell lysates they found an increase in fluorescence with a t1/2 of less than one minute. They found it necessary to use cell lysates to examine kinetics of the interaction of the two halves because the presence of the cell membrane was preventing the rapamycin (the chemical they were using to induce association of the fused proteins) (Luker et al, 2004). Thus I assume that given that light will instantly permeate the cells, that we will see kinetics similar to those in the lysates. This paper does not mention dissociation kinetics.  Renilla luciferase is a 36 kD protein from the sea pansy (Renilla reniformis), which uses coelenterazine and oxygen to produce blue light, 480 nm (Promega).  The issue with use of Renilla luciferase in animals is that the blue light does not penetrate tissues well, and that the coelenterazine is transported by the multidrug resistance P-glycoprotein, which according to Luker et al, as well as others, can cause problems with delivery (Luker et el, 2004).  Gaussia luciferase is derived from the marine copepod Gaussia princeps. This 19.9 kDa protein catalyzes a reaction involving oxygen and coelenterazine to produce bioluminescence at a peak of 470 nm. At 185 amino acds this is the smallest of the luciferases (Verhegen et al, 2002).     Luciferase fragment choice. Firefly luciferase. Luker et al found an optimized pair of firefly luciferase fragments as X --Nf Luc 2-412, and CfLuc 398-550 – Y. This was reduced from the initial as X -- Luc 2-435, and Luc 21-550 – Y, a significant decrease in overlap which presumably served to reduce constitutive activity of the N terminal fragment. In the paper by Luker et al, they found that “FRB-NLuc  and CLuc-FKBP plus rapamycin produced a maximal bioluminescence of 2x 106 photon flux units per 1 x 104 cells in a 96-well format (7 x 104 photon flux units per ug of protein), 1,200-fold greater than untransfected cells or blank wells (Fig. 1E). By comparison, a control plasmid (pGL3) expressing intact luciferase produced 3-fold greater bioluminescence (6 x106 photon flux units 1 x 104 cells transfected with 33 ng of DNA). (Luker et al, 2004).” In their 2007 paper Paulmurugan identified (NFluc 398/CFluc 394) as the ideal pair, claiming it to be an improvement upon existing combinations. This combination has according to their paper near-zero background signal from self-complementation. They claim this to be a direct improvement upon the N416-C398 pair from the Luker paper. In terms of fragment orientation they recommend Nfluc-FRB/FKBP12-CFluc, but this is only with one protein pair. They mention reduction in enzymatic activity when you attach other proteins to the N terminus of the protein, but mention that Luker had success with a protein on the N terminus of the N terminal fragment. (Paulmurugan et al, 2007).   Luciferase DNA sources. Renilla and firefly are in the BioBricks database, and we have received Firefly luciferase in red and green variants from a source. The red luciferase contains a single point mutation, and the green luciferase contains three mutations which shift it greener.  Fusion protein construction. All combinations are as follows: Nfluc-PhyB,  Cfluc-PhyB,  PhyB-Nfluc,  PhyB-Cfluc,  Nfluc-PIF3,  Cfluc-PIF3,  PIF3-Nfluc,  PIF3-Cfluc.   Based on the suggestion that fusing things to the N terminal of luciferase reduces function, we should only attach proteins to the C terminal of the Nfluc. This leaves us with several possible pairs,  Nfluc-PhyB/C-fluc PIF3  Nfluc-PhyB/ PIF3-Cfluc  PhyB-Cfluc/ Nfluc-PIF3 Cfluc-PhyB/ Nfluc-PIF3.    Then the next question is do w e want to put the N or C terminus on PhyB or on PIF3? According to Phee, the cDNA of phyB or PAPP2C was subcloned into BiFC vector containing either the Nterminal region of YFP (yellow fluorescent protein) for YFPNE:: PhyB or the C-terminal region of YFP for YFP-CE::PAPP2C) (Phee et al, 2008). The Phee paper and Subramanian paper both put the protein fragment on the N terminus of PhyB, which leaves us with the following.  Nfluc-PhyB/C-fluc PIF3 Nfluc-PhyB/ PIF3-Cfluc Cfluc-PhyB/ Nfluc-PIF3  Based on the following: Subramanian used the following BRET pair: RLuc-PhyB:PiF3-YFP (Subramainan et al., 2006), we should probably follow the same orientations.  Nfluc-PhyB/ PIF3-Cfluc   Red-shift of reconstituted proteins. Another factor that must be taken into consideration is the red shift known to result from reconstitution of the protein.  In the paper by Cissell et al., they found that the emission max of the reconstituted Renilla luciferase was 495 nm, while native luciferase has a peak at 485 nm, a shift also seen with GFP. They describe this shift being a function of the change in proximity of amino acids around the chromophore as compared with native Rluc, and the  presence of the negatively charged DNA which in these experiments drove the association (Cissell et al, 2009). I assume this may also be an issue with the firefly luciferase.   Lack of yeast studies. Split luciferase complementation assays have primarily been done in mammalian cells lines (HEK293T notably), and I have not seen a paper where they have been done in yeast. I saw one paper in which it was done in Arabidopsis. I see no significant reason it cannot be used in yeast, given that the only requirements are that the two fragments are in the same cellular compartment, and that luciferase be able to function in its native state in yeast, which it can. Also, appropriate luciferins must be provided. This may pose a problem with the coelentarazine, because of the problems mentioned with multidrug resistance peptide P? Luciferin will also be an issue because you need low pH for it to be taken up, but we can examine the use of esterified luciferins.  Upon further examination the study of esterified luciferins was done in mammalian cells and in bacteria, but not in yeast. Further searches yielded no information about use of esterified luciferins in yeast (Craig et al, 1991). It appears that luciferase is not as commonly used in yeast as LacZ.   Conditions for luciferin uptake. We need to use D-luciferin. Addition of luciferin was done in the following manner: “D-luciferin (1 mM, 100 μl) in 0.1 M Na citrate buffer (pH 3.0 or 5.0) was pipetted into thewells containing induced cultures. The plate was briefly shaken and then immediately measuredusing a Victor multilabel counter (Perkin-Elmer Wallac, Turku, Finland) in the luminescence mode, using 1 s counting time. The light emission levels are expressed as RLU (relative light units = luminescence value given by the luminometer) and the normalized luminescence was calculated by dividing the RLU value of the induced culture by that of the blank solvent”  (Leskein et al, 2003).    Peroxisomal targeting of luciferases. Insect luciferases contain a peroxisomal targeting signal at their C terminus, Ser- Lys-Leu (Leskein et al, 2003).  In firefly luciferase this sequence is at the extreme C terminus, residues 548-550 (Gould et al, 1989). Leskein found that removal of the peroxisomal targeting signal resulted in better cell growth, which implies that peroxisomal targeting causes problems with cell growth. Also, use of the modified luciferase allows adequate luciferin uptake at pH 5 as opposed to pH 3 (Leskein et al, 2003). In that paper they had the luciferase under the control of a copper-inducible promoter. Modified luciferase was found to have 2 orders higher expression levels than wild type luciferase. This modification eliminates the need for centrifugation and resuspension of cells. They simply removed the peroxisomal targeting codons.    Positive control.  SNF1 and SNF4—they are in the same strain but if they are different sizes?  What is on drosophila library interacter?  Taheyen—ask her for maps.   Negative control.  Nfluc/Cfluc  Specifications for Primers Need to add appropriate restriction sites to the ends of the luciferases Need to remove peroxisomal targeting sequence Need to be in frame in pCETT
Tuesday June 23, 2009
PIF3 from biobricks matches first 300 bp of Arabidopsis PIF3 with the exception of one basepair. Changes glutamic acid to aspartic acid.
PhyB from biobricks matches first 300 bp of Arabidopsis PhyB.
PCB Subproject PCB extraction—crude→HPLC? Reconstitute pathway (HO1 and PcyA)—spec assay. Conditions?  PIF3 and PhyB Project PIF3—APB (100 aa), Full length PhyB—N Term (have sequences and primers)
Biobricks PhyB N
PIF3 APB
Library  PhyB N
PIF3 APB
PIF3 Full
Plant         PhyB N
PIF3 APB
PIF3 Full
July 1, 2009 (Weds)
We did the mRNA extraction from the Arabidopsis. Primers were diluted to 100 uM, which was followed by a subsequent 1-10 dilution to make a total of 100 uL of each primer at 10 uM. Final reactions were .6 uM, which means we added 3 uL of each primer per rxn.
Dilutions A. F.PIF3.SaII  242 uL
B. R. PIF3.partial.BsshII 214 uL
C. R.PIF3.FL.BsshII 208 uL
D. F.PIF3.BamHI.pACT2 240 uL
E. R.PIF3.partial.Xho1.pACT2 201 uL
F. R.PIF3.full.Xho1.pACT2 220 uL
G. F.PhyB.SpeI  170 uL
H. Nterm.BcII  213 uL
Reactions were mixed as follows: 27 uL RNAse free water 10 uL Buffer from kit 2 uL DNTPs 3 uL Forward primer 3 uL Reverse primer 2 uL RNA 2 uL Enzyme Total: 50 uL  Reactions were run according to the following  Dilution, PCR, gel, gel extraction, digest, PCR cleanup, ligation, transformation Gel, gel extract, digest, PCR cleanup, and then ready for ligations tomorrow.  Transformation  PCR Reactions for amplification of PIF3 and PhyB from plasmids from Florida.  5 ul 10x buffer 1 ul DNTPs 1 ul fwd primer (at 10 uM) 1 ul reverse primer (at 10 uM) 1 ul DNA (rxns 1, 3, 5 stock, 2, 4, 6, 10 fold dilution from stock) 1 uL VENT enzyme  40 uL ddH20 50 ul total volume
Week 4: 6/29/09 - 7/3/09
pGal promoter with luciferase (either red or green)—For the two hybrid system Into 17 and 18 plasmids from Murray lab (pFA6a + Gal1). Digest plasmids with BamHI and SalI (Buffer 3 BSA 37), and then ligate in the luciferase fragment, PCRed out with the following primers. Sites selected because absent in luciferase and easy to use in the Gal1 plasmids.  F.luciferase.BamHI   AAGCTTGGT TCCAAA atggaagacgccaaaaacataaag  Tm65 R.luciferase.SalI    AAG CTT GTC GAC AAA ttactttccgcccttcttggcc    Tm 71   Plasmid 1-BamHI/SalI works Plasmid 2-SpeI/HindIII Plasmid 3-BamHI/SalI works  For pPS293 Vector  F.luciferase.SpeI   AAGCTT ACT AGT  AAA atggaagacgccaaaaacataaag   R.luciferase.HindIII    AAG CTT AAG CTT AAA ttactttccgcccttcttggcc      For the pET-GFP-POS36 (David’s plasmid) F.Luciferase.NcoI AAGCTT CC A TGG aagacgccaaaaacataaag   R.Luciferase.XhoI AAG CTT CTC GAG ttattactttccgcccttcttggcc      For one of the random plasmids from Pam Silver’s lab. Except then we decided not to do this.   F.luciferase.BamHI   AAGCTTGGT TCCAAA atggaagacgccaaaaacataaag  Tm65 R.luciferase.HindIII    AAG CTT AAG CTT AAA ttactttccgcccttcttggcc      PCR  Reactions  Red    Green  5 ul Thermopol Buffer  5 ul Thermopol Buffer  1 ul dNTPs   1 ul dNTPs  1 ul F.luciferase.BamHI    1 ul F.luciferase.BamHI     1 ul R.luciferase.SalI      1 ul R.luciferase.SalI  1 ul Red DNA   1 ul Green DNA  1 uL VENT   1 ul VENT  40 uL H20   40 uL H20        Thermocycler conditions  5 minutes melting at 95  Per cycle: 30 seconds melting at 95 5 cycles at annealing temp 47, 30 cycles at annealing temp 52 Extension 1 min 48 sec at 72 degrees   5 minutes at 72 degrees   Gel. Lane 2 Red, Lane 3 green.  Both show fragments of approx 1650 bp, which is as expected for full length luciferase! This means we have successful amplification of the desired fragments with the appropriate restriction sites for insertion into the 17 pFA6a-TRP1-pGAL1 and 18 pFA6a-HIS3-pGAL1 plasmids from the Murray lab. Those restriction sites are the ones present downstream of the promoter in the MCS so ligation should work well!     Purification. PCR reactions were purified via column purification and eluted in 50 uL of water. Resulted in the following concentrations: Red- 398 ng/ul, green- 42 ng/ul, 17-154 ng/ul, 18-238 ng/ul.   Digests. Fragments and vectors were digested to create complementary sticky ends for ligation. Ligations were run at 37 degrees for an hour and a half.   Red   Green   17 pFA6a-TRP  18 pFA6a-HIS   1 ul BamHI  1 ul BamHI  1 ul BamHI  1 ul BamHI   1 ul SalI   1 ul SalI   1 ul SalI   1 ul SalI   1 ul Dpn1  1 ul Dpn1     6.5 uL Buffer 3  6.5 uL Buffer 3  3 uL Buffer 3  3 uL Buffer 3   6.5 uL BSA  6.5 uL BSA  3 uL BSA  3 uL BSA   50 uL DNA  50 uL DNA  10 uL DNA  10 uL DNA   Purification. Column purification  of the digest fragments resulted in the following concentrations: Red-34.8 ng/ul, Green-35.1 ng.ul, 17- 31.6 ng/ul, and 18-54.9 ng/ul   Ligations. Fragments from the above digests were ligated to create the complete reporter for the yeast two hybrid system. Ligations were done on 50 uL (A) Red+17  (B) Red+18  (C) Green+17  (D) Green+18  Control Red  Control 17 1.5 uL Red  1.5 uL Red  1.5 uL green  1.5 uL green  1.5 uL Red 1.6 uL 17  1.0 uL 18  1.6 uL 17  1.0 uL 18     1.6 uL 17 2 uL T4 buffer  2 uL T4 buffer  2 uL T4 buffer  2 uL T4 buffer  2 uL T4 buffer  2 uL T4 buffer    1 ul QuikLigase  1 ul QuikLigase  1 ul QuikLigase  1 ul QuikLigase  1 ul QuikLigase  1 ul QuikLigase  15 uL water  15 uL water  15 uL water  15 uL water  15 uL water  15 uL water  Transformation. 5 uL of ligation was transformed into TOP10 cells and allowed to grow for an hour following standard protocol. These were then plated onto amp plates, and grown overnight. A control of 17 digested backbone alone and red fragment alone was also carried out, which resulted in many colonies on the backbone control plate, far more than on the ligation plates. But colony PCR revealed these fewer colonies on the ligation plates to be correct. Oliver thought the strangeness with the control was due to incomplete cutting of the vector and efficient recircularization in the absence of insert, but that it was less efficient when the insert was present.   Colony PCR. Colony PCR was carried out to determine if vectors 17 and 18 ligated to the red and green luciferases actually contain the inserts.  Rxn Mix for 20 reactions: 20 uL  10x –MG buffer  4 uL  10 nM dNTP mix 6 uL  50 nM MgCl2 5 uL Primer Fwd F.luciferase.BamHI 5 uL Primer Rev R.luciferase.SalI 10 uL Taq 150 uL Water  10 uL aliquots were made, and ½ a culture added to each one, 5 from each of the 4 plates for a total of 20 colony PCRs. Thermocycler conditions were annealing 52 for 5 cycles then 55 for 25 cycles, extension time of 1 min 48 seconds because expected length of fragment is 1.65 kB (and I rounded up a little bit).   Gels of Colony PCR products Gel 1. Lanes 2-6 are A1-A5, lanes 7-11 are B1-B5 Gel2. Lanes 2-6 are C1-C5, lanes 7-11 are D1-D5     Colony Picking. Amy picked colonies to grow up for large scale prepping (this is an integrating vector and requires up to 10 ug per yeast transfection, but this will then create a stable line. But we need enough). Colonies picked were: A1, B3, C4, D5. Cultures grown with Amp, uL/mL, 100 mL cultures. Unfortunately nothing grew, we are repacking and culturing again tonight in the hopes that it was just because we left them on the counter for too long before putting them into the shaker. It was not likely that it was due to the Amp, because we used two different aliquots between the 4 cultures but none of the 4 grew.
Week 5: 7/6/09 - 7/10/09
Split Luciferase System Cloning   Primer Design for Split Luciferase 1. pFA6a primers (Nhe + SexAI) a. pFA6a forward (after citrine)— SexAI site+ sequence  5’-Filler + SexAI + Filler + plasmid sequence-3’ F. pFA6a.plasmid.SpL.SexAI AAG CTT ACC AGG TAA AAA ggcgcgccacttctaaataagc-3’ (41)  --overlap Tm57 total 67 F. pFA6a.SpL.SexAI.new TTA CCA GGT AAggcgcgccacttctaaataagc   overlap 57, total 65  b. pFA6a reverse (before citrine)—NheI site + sequence from start of citrine rev comp R.pFA6a.plasmid.SpL.Nhe AAG CTT GCT AGC  AAA tgttaattcagtaaattttcgatc 3’ (39) – Tm55 or 47. THIS PRIMER IS THE PROBLEM! TOTALLY.    R.pFA6a.SpL.Nhe.new CTT GCT AGC gttaattcagtaaattttcgatcttgggaag overlap 56 total 63 2. Nfluc (1-398) primers (Nhe + Kpn) a. nFluc forward—NheI site + sequence  F.Nfluc.SpL.NheI AAG CTT GCT AGC AAA atggaagacgccaaaaacataaag (39)—Tm52 b. nFluc reverse—KpnI site R.Nfluc.SpL.KpnI AAG CTT GGT ACC AAA cataatcataggtcctctgacac (38)—Tm 53  3. Cfluc (349-548) (Kpn + SexAI) a. cFluc forward—KpnI site  F.cFluc.SpL.KpnI AAG CTT GGT ACC AAA ggacctatgattatgtccggttatg (40)—Tm 56 b. cFluc reverse—incl Stop codon, SexAI (also removed last three amino acids)  c. R.Cfluc.SpL.SexAI AAG CTT ACC AGG TAA AAA ttactttccgcccttcttggcc (40)—Tm 55  4. PhyB (1-621) (Kpn + SexAI) a. PhyB forward—Kpn1 site  b. F.PhyB.SpL.KpnI AAG CTT GGT ACC AAA atggtttccggagtcggggg (35)—Tm 58 c. PhyB reverse—Phy B, stop codon, SexAI,  R.PhyB.SpL.SexAI AAG CTT ACC AGG TAA AAA ttaaagctggagcgagtgaatc (40)—Tm 51 or 47  R.PhyB.SpL.SexAI.correct AAG CTT ACC AGG TAA TTAAAGCTGGAGCGAGTGAATCGC 52/57 R.PhyB.pST39.BclI.correct AAG CTT TGA TCA TTAAAGCTGGAGCGAGTGAATCGC 52/57  5. PIF3 full  (Nhe + Kpn1)l a. PIF3 full forward—NheI site F.PIF3.SpL.NheI AAG CTT GCT AGC AAA atgcctctgtttgagcttttc (36)—Tm 50 or 57 b. PIF3 full reverse—Kpn1 site (remove stop codon!) R.PIF3.Full.SpL.KpnI AAG CTT GGT ACC AAA cgacgatccacaaaactgatc (36)—Tm 52 6. PIF 3 partial (Nhe + Kpn) a. PIF3 partial reverse—Kpn1 site R.PIF3.partial.SpL.KpnI AAG CTT GGT ACC AAA atgatgattcaaccatggaac (36)—Tm 49 or 55  Xma/Nhe digest for insert—Buffer 4  (PhyB nterm) Xma— CCC GGG Nhe—GCT AGC   F.Nfluc.SpL.XmaI AAG CTT CCC GGG AAA atggaagacgccaaaaacataaag R.PhyB.SpL.NheI AAG CTT GCT AGC TTA TTAAAGCTGGAGCGAGTGAATCGC   XmaI/EcoRI-HF digest for insert and for vector—Buffer 4.  Xma—CCC GGG EcoRI—GAA TTC      F.PIF3.SpL.XmaI AAG CTT CCC GGG AAA atgcctctgtttgagcttttc   R.Cfluc.SpL.EcoRI AAG CTT GAATTC  TTA ttactttccgcccttcttggcc  Checking frame for Nfluc+PhyB: agtgtcagaggacctatgattatgTTTGGTACCAAAatggtttccggagtcggggg. In frame! Checking frame for PIF3+Cfluc: gatcagttttgtggatcgtcgTTTGGTACCAAAggacctatgattatgtccggttatg  F.PIF3.pST39.XmaI TAG CTT CCC GGG  AAA atgcctctgtttgagcttttc 54, 67 F.PIF3.SpL.NheI AAG CTT GCT AGC AAA atgcctctgtttgagcttttc  54, 65 R.PIF3.Full.SpL.KpnI AAG CTT GGT ACC AAA cgacgatccacaaaactgatc 54, 65 R.PIF3.partial.SpL.KpnI AAG CTT GGT ACC AAA atgatgattcaaccatggaac 51, 53 49, 55  R.cFluc.pST39.MluI TAG CTT ACG CGT  AAA ttactttccgcccttcttggcc 60, 67 R.Cfluc.SpL.SexAI AAG CTT ACC AGG TAA AAA ttactttccgcccttcttggcc 60, 67 F.cFluc.SpL.KpnI AAG CTT GGT ACC AAA ggacctatgattatgtccggttatg 55, 65 52, 57   F.PIF3.pST39.XmaI TAG CTT CCC GGG  AAA atgcctctgtttgagcttttc 54, 67 R.cFluc.pST39.MluI TAG CTT ACG CGT  AAA ttactttccgcccttcttggcc 60, 67  F.PIF3.SpL.NheI AAG CTT GCT AGC AAA atgcctctgtttgagcttttc  54, 65 R.Cfluc.SpL.SexAI AAG CTT ACC AGG TAA AAA ttactttccgcccttcttggcc 60, 67   Split Luciferase PCR Reactions No. Product Frag. DNA Forward Reverse Digest Buffer Expected Size 1 His- vector pFA6a-Act-His F. pFA6a.plasmid.SpL.SexAI R.pFA6a.plasmid.SpL.Nhe SexAI/NheI/DpnI Buffer 4 + BSA @ 37, 1hr 5 kb 2 Ura- vector pFA6a-Act-Ura F. pFA6a.plasmid.SpL.SexAI R.pFA6a.plasmid.SpL.Nhe SexAI/NheI/DpnI Buffer 4 + BSA @ 37, 1hr 5 kb 3 Red Nfluc Red luciferase F.Nfluc.SpL.NheI R.Nfluc.SpL.KpnI NheI/KpnI/DpnI Buffer 1 + BSA @ 37, 1hr 1.2 kb 4 Green Nfluc Green luciferase F.Nfluc.SpL.NheI R.Nfluc.SpL.KpnI NheI/KpnI/DpnI Buffer 1 + BSA @ 37, 1hr 1.2 kb 5 R/G Cfluc  Red luciferase F.cFluc.SpL.KpnI R.Cfluc.SpL.SexAI KpnI/SexAI/DpnI Buffer 1 + BSA @ 37, 1hr 450 bp 6 PhyB Nterm PhyB florida  F.PhyB.SpL.KpnI R.PhyB.SpL.SexAI KpnI/SexAI/DpnI Buffer 1 + BSA @ 37, 1hr 1.8 kb 7 PIF3 Full PIF3 florida F.PIF3.SpL.NheI R.PIF3.Full.SpL.KpnI NheI/Kpn1/DpnI Buffer 1 + BSA @ 37, 1hr 1.6 kb 8 PIF3 Partial PIF3 florida F.PIF3.SpL.NheI R.PIF3.partial.SpL.KpnI NheI/Kpn1/DpnI Buffer 1 + BSA @ 37, 1hr 300 bp    Construction Goals HisVector/NheI/Nfluc Red/KpnI/PhyB/SexAI/Vector HisVector/NheI/Nfluc Green/KpnI/PhyB/SexAI/Vector UraVector/NheI/PIF3 full/KpnI/cFfluc/SexAI/Vector UraVector/NheI/PIF3 partial/KpnI/cFfluc/SexAI/Vector  His Vector+ Nfluc-PhyB  1. PCR Reactions a. In order to obtain vector backbone without the inserted citrine, PCR around the pFA6a-His vector (rxn 1). We could not just digest the citrine out because there is no upstream restriction site available. PCR with these primers will result in the addition of a NheI site on the end of the vector fragment near the promoter (to be ligated to the N terminus of the fusion protein), and a SexAI site on the end far from the promoter (to be ligated to the c terminus of the fusion protein) b. In order to obtain the Nfluc fragment (1-398), we will PCR out that fragment via rxn 2 for green luciferase and rxn 3 for red luciferase. The primers for these two variants of luciferase are identical; the color determining point mutations are all located in this N terminal fragment in the aa 240-250 range; they do not affect the primers. This will produce an Nfluc fragment framed by an NheI site on the N terminus and a Kpn1 site on the c terminus.  c. In order to obtain the PhyB fragment, we will PCR it out of the florida plasmids via rxn 6. This is just the N terminus of the PhyB, residues 1-621, chosen to reduce background, as in our other experiments. This will produce an PhyB fragment framed by an KpnI site on the N terminus and a SexAI site on the c terminus.  2. Post-PCR processing a. All fragments will be purified using a PCR cleanup column 3.  Restriction digests a. Vector backbone will be digested for 1 hr at 37 degrees with NheI and SexAI, Buffer 4 or 1 with BSA.  b. Nfluc fragment will be digested for 1 hr at 37 degrees with NheI and Kpn1, Buffer 1 with BSA.  c. PhyB fragment will be digested for 1 hr at 37 degrees with Kpn1 and SexAI, Buffer 1 with BSA 4. Purification a. All fragments will be purified using a PCR cleanup column 5. Ligation Method 1—3 piece ligation (side by side with method 2) a. All three fragments are added to the same ligation reaction (what fold excess of the two inserts? 3? 4?), incubated at 37 for 1 hour b. Ligations are transformed into XL10 gold (because need sensitivity) 6. Ligation Method 2—2 stage ligation (side by side with method 1) a. The Nfluc and PhyB fragments are ligated for one hour, followed by PCR with the F.Nfluc and R.PhyB primers in order to amplify the ligated fragment b. PCR product Nfluc-PhyB is digested with NheI and SexAI, followed by column purification c. Purified Nfluc-PhyB insert is ligated to vector backbone fragment for 1hour at 37 with DNA ligase d. Ligations are transformed into XL10 gold  Ura Vector + PIF3-Cfluc 1. PCR Reactions a. In order to obtain vector backbone without the inserted citrine, PCR around the pFA6a-Ura vector (rxn 1). We could not just digest the citrine out because there is no upstream restriction site available. PCR with these primers will result in the addition of a NheI site on the end of the vector fragment near the promoter (to be ligated to the N terminus of the fusion protein), and a SexAI site on the end far from the promoter (to be ligated to the c terminus of the fusion protein) b. In order to obtain the PIF3 fragment, we will PCR it out of the florida plasmids via rxn 7 for full length PIF3 and rxn 8 for partial PIF3 (aa 1-100). This is just the N terminus of the PhyB, residues 1-621, chosen to reduce background, as in our other experiments. This will produce an PhyB fragment framed by an NheI site on the N terminus and a KpnI site on the c terminus.  c. In order to obtain the cfluc fragment (394-547), we will PCR out that fragment via rxn 4 for either luciferase. The primers for these two variants of luciferase are identical; the color determining point mutations are all located in this N terminal fragment in the aa 240-250 range; they do not affect the primers. This will produce an Cfluc fragment framed by an Kpn1 site on the N terminus and a SexAI site on the c terminus. This will also serve to introduce a stop codon at the end of the cfluc and remove the SKL peroxisomal targeting sequence located on aa 548-550.  2. Post-PCR processing a. All fragments will be purified using a PCR cleanup column 3.  Restriction digests a. Vector backbone will be digested for 1 hr at 37 degrees with NheI and SexAI, Buffer 4 or 1 with BSA.  b. PIF3 fragment will be digested for 1 hr at 37 degrees with NheI and Kpn1, Buffer 1 with BSA.  c. Cfluc fragment will be digested for 1 hr at 37 degrees with Kpn1 and SexAI, Buffer 1 with BSA 4. Purification a. All fragments will be purified using a PCR cleanup column 5. Ligation Method 1—3 piece ligation (side by side with method 2) a. All three fragments are added to the same ligation reaction (what fold excess of the two inserts? 3? 4?), incubated at 37 for 1 hour b. Ligations are transformed into XL10 gold (because need sensitivity) 6. Ligation Method 2—2 stage ligation (side by side with method 1) a. The PIF3 and Cfluc fragments are ligated for one hour, followed by PCR with the F.PIF3 and R.Cfluc primers in order to amplify the ligated fragment. This will be done for both the partial and full length PIF3.  b. PCR product PIF3-Cfluc is digested with NheI and SexAI, followed by column purification. This will be done for both the partial and full length PIF3. c. Purified PIF3-Cfluc insert is ligated to vector backbone fragment for 1hour at 37 with DNA ligase d. Ligations are transformed into XL10 gold  Attempt 1 at assembly of split luciferase system in yeast vectors PCR Reaction Mixes Mastermix H2O  40.6 ul x5= 203ul  Buffer  5 ul x5= 25 ul dNTPs  0.4 ul x5= 2 ul PfuTurbo 1 ul x5= 5 ul Per Reaction Mix  47 uL mix per reaction Fwd Primer 1 ul Rev Primer 1 ul DNA  1 ul Total  50 uL per reaction No. Product Frag. DNA Forward Reverse 1 His- vector pFA6a-Act-His F. pFA6a.plasmid.SpL.SexAI R.pFA6a.plasmid.SpL.Nhe 2 Ura- vector pFA6a-Act-Ura F. pFA6a.plasmid.SpL.SexAI R.pFA6a.plasmid.SpL.Nhe 3 Red Nfluc Red luciferase F.Nfluc.SpL.NheI R.Nfluc.SpL.KpnI 4 Green Nfluc Green luciferase F.Nfluc.SpL.NheI R.Nfluc.SpL.KpnI 5 R/G Cfluc  Red luciferase F.cFluc.SpL.KpnI R.Cfluc.SpL.SexAI 6 PhyB Nterm PhyB florida  F.PhyB.SpL.KpnI R.PhyB.SpL.SexAI 7 PIF3 Full PIF3 florida F.PIF3.SpL.NheI R.PIF3.Full.SpL.KpnI 8 PIF3 Partial PIF3 florida F.PIF3.SpL.NheI R.PIF3.partial.SpL.KpnI    Thermocycler Conditions Ura and His Backbones 1 cycle of  95 for 2 minutes 5 cycles of  95 for 30 seconds   45 for 30 seconds   72 for 5 minutes 25 cycles of 95 for 30 seconds   52 for 30 seconds   72 for 5 minutes 1 cycle of  72 for 10 minutes nFluc Red, nFluc Green, cFluc, PhyB, PIF3 full and partial 1 cycle of  95 for 2 minutes 30 cycles of 95 for 30 seconds   52 for 30 seconds   72 for 5 minutes 1 cycle of  72 for 10 minutes  Check Gel All but PhyB appeared to work.     Digest Reaction Mixes Number Product Frag. Digest Buffer 1 His- vector SexAI/NheI/DpnI Buffer 4 + BSA @ 37, 1hr 2 Ura- vector SexAI/NheI/DpnI Buffer 4 + BSA @ 37, 1hr 3 Red Nfluc NheI/KpnI/DpnI Buffer 1 + BSA @ 37, 1hr 4 Green Nfluc NheI/KpnI/DpnI Buffer 1 + BSA @ 37, 1hr 5 R/G Cfluc  KpnI/SexAI/DpnI Buffer 1 + BSA @ 37, 1hr 6 PhyB Nterm KpnI/SexAI/DpnI Buffer 1 + BSA @ 37, 1hr 7 PIF3 Full NheI/Kpn1/DpnI Buffer 1 + BSA @ 37, 1hr 8 PIF3 Partial NheI/Kpn1/DpnI Buffer 1 + BSA @ 37, 1hr   Ligations and Transformations All ligations are 25 ng of vector and (a) are 1:3:3 vector:insert:insert, (b) are 1:1:1 vector:insert:insert  Ligation  Product Vector ng uL Insert 1 Ng Ul Insert 2 Ng ul Buffer Ligase 1A Red-Phy His 25 ng 10 ul RedN 18 ng 2 ul PhyB 27 ng  5 ul 1 ul 1B Red-Phy His 25 ng 10 ul RedN 6 ng 1.3 ul PhyB 9 ng  5 ul 1 ul 2A Gn-Phy—note: intended to be green but accidentally added red. His 25 ng 10 ul GreenN 18 ng 2 ul PhyB 27 ng  5 ul 1 ul 2B Gn-Phy note: intended to be green but accidentally added red. His 25 ng 10 ul GreenN 6 ng 2 ul PhyB 9 ng  5 ul 1 ul 3A PifFull-Cluc Ura 25 ng 16 ul cFluc 7.25 ng 0.35 ul PIF3 full 24 ng 4.8 ul 5 ul 1 ul 3B PifFull-Cluc Ura 25 ng 16 ul cFluc 2.25 ng 0.13 ul PIF3 full 8 ng 1.6 ul 5 ul 1 ul 4A PifPart-Cluc Ura 25 ng 16 ul cFluc 7.25 ng 0.35 ul PIF3 par 4.5 ng 1.6 ul 5 ul 1 ul 4B PifPart-Cluc Ura 25 ng 16 ul cFluc 2.25 ng 0.13 ul PIF3 par 1.5 ng 0.5 ul 5 ul 1 ul 6 Control His Control 6 ng 5 ul n/a n/a n/a n/a n/a n/a 5 ul 1 ul 7 Control Ura Control 10 ng 5 ul n/a n/a n/a n/a n/a n/a 5 ul 1 ul  None of the plates had any colonies on them after transformation into 25 ul of XL10 gold cells each followed by immediate plating onto ampicillin plates (no incubation). The next day after ligating overnight we tried transforming them into Top10 chemically competent cells, this time with the full hour incubation. There were still no colonies. This is probably attributable to not enough backbone DNA present. Will have to reassess cloning strategy.   PCR redo on PhyB  PCR Reactions 1. PhyB 221 (from Florida stock) 2. PhyB Dest 22 (from Florida stock) 3. PhyB 221 (our stock) 4. PhyB Dest 22 (our stock) Note: used primers at 100 ng/uL  H2O  40.6 ul x5= 203ul  Buffer  5 ul x5= 25 ul dNTPs  0.4 ul x5= 2 ul PrimerF 1 ul x5= 5 ul PrimerR 1 ul x5= 5 ul PfuTurbo 1 ul x5= 5 ul Total  49 uL per reaction    Thermocycler conditions for PhyB PhyB 221 original, PhyB Dest 22 original, PhyB 221 ours, PhyB Dest 22 ours 1 cycle of  95 for 2 minutes 30 cycles of 95 for 30 seconds   45 for 30 seconds   72 for 5 minutes 1 cycle of  72 for 10 minutes Check Gel for PhyB None of the 4 PCRs worked. Will need to reexamine the primer design. Reexamination found that I did design them incorrectly and they will have to be reordered. Damn. In the meantime I will focus on cloning of the PIF3-cFluc. Need to repeat the PCRs for the PIF3 and the cFluc    Redo of PCR for Ura and His Backbones  PCR Reactions  Mastermix 25 ul Buffer 5 ul DNTPs 15 ul QuickChange Solution 4 uL fwd primer 1A 4 ul reverse primer 1B 190 uL H20   49 uL mastermix reaction, plus 1ul of each of the following DNAs: 1) His, 2) His, 3)Ura, 4) Ura Thermocycler conditions 1 cycle of  95 for 2 minutes 5 cycles of  95 for 30 seconds   45 for 30 seconds   72 for 5 minutes 25 cycles of 95 for 30 seconds   52 for 30 seconds   72 for 5 minutes 1 cycle of  72 for 10 minutes  Check Gel Gel 1. Initial results of the 5 ul per well check gel were unclear—parent DNA band was visible, and there might have been a smeary band below it. Lane 1 is ladder, Lane 2 and 3 are His, Lane 4 and 5 are Ura, Lane 6 is ladder. 5 ul Ladder.   Gel 1     Gel 2                       Gel 2. To help determine if there was anything, the remainder of the reaction was subjected to a Dpn1 digest for 1 hr at 37 minutes and then 20 ul of the reaction were run out on a check gel. The parent band was gone, there was a ladder present, but no band indicative of product that we would expect around 5 kb. Lane 1 is ladder, Lane 2 and 3 are His, Lane 4 and 5 are Ura, Lane 6 is ladder. 5 ul 1 kb plus Ladder.   Troubleshooting the backbone PCR Should email the Murray lab people and ask them if they have a different vector that we can use for this, one that actually has a site upstream of the gene inserted into it. David looked at my primers and did not see a significant problem with them, Tms are a little on the low side but there is not much to do about that. I will try to redesign them and in the meantime we have ordered a new PlatnumPfx from Invitrogen which should be more processive and get better yield. Also should I consider increasing the extension time? That might help. It was a problem with the luciferase when I didn’t give it long enough.   PCR of PhyB and Luciferase Nterminus for insertion into the pFA6a backbones. This is being done because we need the inserts for the creation of the split luciferase system in yeast. These fragments will be digested with Kpn1, ligated to eachother, and then PCR amplified for insertion into the pFA6a backbones from the Murray lab.   Name DNA Fwd Primer Reverse Primer 221 PhyB SpL  221 from Mike stock F.PhyB.SpL.KpnI R.PhyB.SpL.SexAI.corrected Dest PhyB SpL Dest from Mike stock F.PhyB.SpL.KpnI R.PhyB.SpL.SexAI.corrected D155 PhyB SpL German with PhyB 2 F.PhyB.SpL.KpnI R.PhyB.SpL.SexAI.corrected Luciferase-Red Red Luciferase stock F.nFluc.SpL.NheI R.nFluc.SpL.KpnI Luciferase-Gn Gn luciferase stock F.nFluc.SpL.NheI R.nFluc.SpL.KpnI    Reaction Mixtures PhyB for SpL  PhyB for   Luciferase   Thermocycler conditions Phy B    Luciferase  Check Gel   Lane (1) 221 PhyB SpL, (2) 221 PhyB pST39, (3) Dest PhyB SpL (4) Dest PhyB pST39 (5) D154 PhyB SpL, (6)D154 PhyB pST39, (7) Red Nfluc pST39, (8) Red Nfluc SpL, (9) Green Nfluc SpL, (10) Green Nfluc pST39. All lanes have bands of the correct sizes, all DNAs column purified and speced.   Purification PhyB for SpL (used Dest)- 16.7 ng/ul, PhyB for pST39 (used Dest)—30 ng/ul, Red for SpL 49.9, Red for pST39 35.4 ng/ul, Green for SpL 49.6, Green for pST39 52.9.   Digestion  MasterMix for all 6 digests   Kpn1 8ul  Dpn1 8ul  Buffer 1 52 ul  BSA 52 ul   16 uL of mix per reaction , 50 uL of DNA per Rxn  Digested for 1 hr at 37 degrees.   Purification PhyB for SpL 19.5, PhyB pST39 32.9 ng/ul, Red SpL 55 ng/ul, Red pST39 22.1 ng/ul, Green SpL 48.0 ng/ul, Green pST39 41.7 ng/ul.   Ligation  PCR Amplification  Red-PhyB Spl (A) Green-PhyB-SpL (B) Red-PhyB pST39 ©  Green-PhyB pST39 (D)    Check gel   PCR amplification of the Backbones from Ura and His take 4 Tried PCR with old Platinum Pfx kit, annealing temps 52x5 cycles, 57x30 cycles, but according to Amy, no success. See her notes.   PCR amplification of the Backbones from Ura and His take 5 Tried PCR again with new Platinum Pfx kit, and using the thermal gradient cycler, ranging temps from 50-70 to optimize annealing temperature. Hopefully something will work.    Set 1: pJHK043, at the temperatures below. 30 ul reactions each.  Set 2: pJHK063, at the temperatures below. 30 ul reactions each.  Sample 1) 50, 2) 50.5, 3) 51.7, 4) 53.2, 5) 53.5, 6) 58.4, 7) 61.8, 8) 64.6, 9) 66.8, 10) 68.4, 11) 69.6 12) 70.0  Pooled the first 7 for each pJHK043 and pJHK063 and column purified them, concentration for pJHK043 288 ng/ul, pJHK063 343 ng/ul or something like that.  Inserts digested with SexAI and NheI Vectors digested with SexAI, NheI, and DpnI
Week 6: 7/13/09 - 7/17/09
Cloning into the Bacterial Vector, pST39 The plasmid contains 4 sets of restriction sites for insertion of genes: 1. XbaI/BamHI 2. EcoRI/HindIII 3. SacI/KpnI 4. BspEI/MluI We want to clone in 4 genes: Ho1, PcyA, nFluc-PhyB, PIF3-cFluc (partial and full). Which gene is to be put into which site was determined by comparing the list of sites (and complementary cutters) with the list of absent sites in the genes of interest. It was important to also look at complementary cutters because otherwise the cloning would not be possible (proper sites were not available) • PcyA can go into sites 1, 3, 4 • Ho1 can go into sites 1, 2, 3,4  • PhyB can go into sites 1, 4 • PIF3 can go into sites 1, 4 Based on this list, genes will be inserted as follows: • Site 1—nFluc-PhyB • Site 2—Ho1 • Site 3—PcyA • Site 4—PIF3-cFluc  PCRs—Ho1 and PcyA we have from the registry, sequenced and confirmed. We will PCR them out of the miniprepped stocks that we have in the lab. nFluc-PhyB, PIF3-cFluc will be PCRed out of the yeast vectors once those constructs are made.   Insertion of the PCB synthesis genes—will try to insert both genes simultaneously into different pST39 vectors, then work with the one that takes the insert to insert the second biosynthesis gene. Whichever one works first we go with  1. Ho1—Site 2:  a. PCR the insert out of existing Ho1 containing plasmids from the registry with F.Ho1.pST39.EcoRI and R.Ho1.pST39.HindIII primers, followed by gel purification of desired fragment (expected size 760 bp).  b. Cut the PCRed insert with EcoRI/HindIII (Buffer 2), followed by PCR cleanup column.  c. Cut the pST39 with EcoRI/HindIII (Buffer 2), followed by PCR cleanup column to remove enzymes. d. Ligate the vector fragment and the insert fragment using QuikLigase e. Transform the ligated plasmid into E coli, pick colonies, grow overnight, miniprep, diagnostic digest to test for insertion.  2. PcyA—Site 3:  a. PCR the insert out of existing Ho1 containing plasmids from the registry with F.PcyA.pST39.SacI and R.PcyA.pST39.KpnI primers, followed by gel purification of desired fragment (expected size 790 bp).  b. Cut the PCRed insert with SacI/KpnI (Buffer 1), followed by PCR cleanup column.  c. Cut the pST39 with SacI/KpnI (Buffer 1), followed by PCR cleanup column to remove enzymes. d. Ligate the vector fragment and the insert fragment using QuikLigase.  e. Transform the ligated plasmid into E coli, pick colonies, grow overnight, miniprep, diagnostic digest to test for insertion.  Insertion of the fusion proteins 3. nFluc-PhyB—Site 1: a. PCR the insert out of nFluc-PhyB fusion plasmids (yeast pFA6a) with F.nFluc.pST39.NheI and R.PhyB.pST39.BclI primers, followed by gel purification of desired fragment (expected size 1800 bp).  b. Cut the PCRed insert with NheI/BclI (Buffer 2, 37 for 1 hour, add BclI then 50 for 1 hour), followed by PCR cleanup column.  c. Cut the pST39-PCB with XbaI/BamHI (Buffer 3), followed by PCR cleanup column to remove enzymes. d. Ligate the vector fragment and the insert fragment using QuikLigase.  e. Transform the ligated plasmid into E coli, pick colonies, grow overnight, miniprep, diagnostic digest to test for insertion.  4. PIF3partial-cFluc—Site 4: a. PCR the insert out of PIF3partial-cFluc fusion plasmids (yeast pFA6a) with F.PIF3.pST39.XmaI and R.cFluc.pST39.MluI primers, followed by gel purification of desired fragment (expected size 800 bp).  b. Cut the PCRed insert with XmaI/MluI (Buffer 4), followed by PCR cleanup column.  c. Cut the pST39-PCB-PhyB with BspeI/MluI (Buffer 3), followed by PCR cleanup column to remove enzymes. d. Ligate the vector fragment and the insert fragment using QuikLigase.  e. Transform the ligated plasmid into E coli, pick colonies, grow overnight, miniprep, diagnostic digest to test for insertion.  5. PIF3full-cFluc—Site 4:  a. PCR the insert out of PIF3full-cFluc fusion plasmids (yeast pFA6a) with F.PIF3.pST39.XmaI and R.cFluc.pST39.MluI primers, followed by gel purification of desired fragment (expected size  2100 bp).  b. Cut the PCRed insert with XmaI/MluI (Buffer 4), followed by PCR cleanup column.  c. Cut the pST39-PCB-PhyB with BspeI/MluI (Buffer 3), followed by PCR cleanup column to remove enzymes. d. Ligate the vector fragment and the insert fragment using QuikLigase.  e. Transform the ligated plasmid into E coli, pick colonies, grow overnight, miniprep, diagnostic digest to test for insertion.    Designing Primers for Cloning into the pST39 vector for Bacterial expression Site 1: XbaI/BamHI: Nfluc-PhyB (NheI/BclI) F.nFluc.pST39.NheI AAG CTT GCT AGC AAA atggaagacgccaaaaacataaag  R.PhyB.pST39.BclI AAG CTT TGA TCA AAA ttaaagctggagcgagtgaatc  Site 2: EcoRI/HindIII: Ho1  F.Ho1.pST39.EcoRI TAG CTT GAA TTC AAA atgagtgtcaacttagcttccc 47/52  R.Ho1.pST39.HindIII TAG CTT AAG CTT AAA ttattagccttcggaggtggcg 57/52  Site 3: SacI/KpnI: PcyA  F.PcyA.pST39.SacI TAG CTT GAG CTC  AAA atggccgtcactgatttaagtttg 57  R.PcyA.pST39.KpnI TAG CTT GGT ACC  AAA ttattggataacatcaaataag 45/49  Site 4: BspE1/MluI: PIF3-Cfluc (xmaI/MluI)  F.PIF3.pST39.XmaI TAG CTT CCC GGG  AAA atgcctctgtttgagcttttc 54  R.cFluc.pST39.MluI TAG CTT ACG CGT  AAA ttactttccgcccttcttggcc 60   To Do Today 1. Do minipreps of pST39, and digest for ligations, column purification, ligation 2. Do PCRs of the PcyA and Ho1, followed by check gel, column purification, digest incl DpnI, column purification, ligation into pST39 3. Do PCRs of split luciferase plasmids, check gel, column purification, digest with KpnI and DpnI, column purification, ligation, PCR of fragment, check gel, column purification digest with SexAI and NheI, column purification, ligation to backbone 4. Do purifications of vector backbone, digest with NheI, SexAI and DpnI, and     PCRs to do 1. PcyA 2. Ho1 3. PIF3 full 4. PIF3 partial 5. cFluc   Need to make plasmid of 1. pST39 (do minipreps of cultures today) 2. Murray Lab plasmid 17 (pick colonies, do minipreps tomorrow) 3. Murray lab plasmid 18 (pick colonies, do minipreps tomorrow)   Insertion of Ho1 into pST39 PCR for amplification of PcyA and Ho1 PcyA         Ho1 5 ul Thermopol Buffer     5 ul Thermopol Buffer 1 ul DNTPs      1 ul DNTPs 1 ul fwd primer F.PcyA.pST39.SacI   1 ul fwd primer F.Ho1.pST39.EcoRI 1 ul reverse primer R.PcyA.pST39.KpnI   1 ul reverse primer R.Ho1.pST39.HindIII 1 ul DNA PcyA      1 ul DNA Ho1 40 ul H20      40 ul H20 1 ul VENT enzyme     1 ul VENT enzyme  Ho1 was amplified successfully (~700bp), PcyA showed an incorrect band between 1 and 1.65kb. This was the same band that Jen and Ivan saw, will repeat that PCR with a higher annealing temperature.   Digests of Ho1 and pST39 Ho1 (PCR product purified by Amy)   pST39 (DNA minpreped by Anu) 1 ul DPN1 1 ul EcoRI     1 ul EcoRI 1 ul HindIII     1 ul HindIII 6.5 uL Buffer 2     3 uL Buffer 2 6.5 ul BSA     3 ul BSA 50 uL DNA     22 uL DNA Total 66.5 uL      Total 30 uL  Digested for 1 hr at 37 degrees, followed by column purification and elution into 50 uL water.  Concentrations after purification: HO1: 15.1 ng/ul, pST39 11.5 ng/ul   Ligation Mix the following: Ligation       pST39 negative control   Ho1 negative control 50 ng pST39 → 4.4 uL of pST39 fragment   4.4 ul pST39 fragment 36 ng HO1 (for a 3:1 ratio) → 2.4 ul HO1 fragment     2.4 uL Ho1 fragment 10 ul water      15 uL water   15 uL water      Heat for 5 minutes at 45 degrees do denature any annealed ends, Cool on ice Add 2 uL ligase buffer and 0.5 uL ligase to each reactoin Incubate at 14 degrees C for 2-4 hours. (actually incubated for 2 hours)  Transformation Transformed into TOP10 cells.  Plates yielded numerous colonies.    Colony PCR Rxn Mix for 10 reactions: 10 uL  10x –MG buffer  2 uL  10 nM dNTP mix 3 uL  50 nM MgCl2 2.5 uL Primer Fwd F.luciferase.BamHI 2.5 uL Primer Rev R.luciferase.SalI 5 uL Taq 80 uL Water Then, 10 uL aliquots were made, and ½ a culture added to each one, for a total of 10 reactions.   Thermocycler conditions (1) 95 for 6 min. 2) 95 for 45 seconds. 3) 45 for 30 seconds. 4) 72 for 1 min. 5) Go to 2 4 more times. 6) 94 for 45 seconds. 7) 50 for 30 seconds. 8) 72 for 1 minute. 9) Go to 6 24 more times. 10) 72 for 10 min. 11) 4 for 24 hours, 12) End.    Gel Colony PCR indicated that almost all colonies contained the insert, HO1, which is approx 700 bp, as seen in the gel, wells 1-4. I have picked and grown up those colonies, and miniprepped them. Concentrations were low, but that is expected given that the plasmid is low copy number. Approx 10x lower in copy number expected, and concentrations were about 30 ng/ul, 10x lower than a 2 mL miniprep of a high copy number plasmid would typically yield. The last two lanes of this gel are PhyB redo pcrs, which once again did not work.        Insertion of PcyA into pST39 (containing HO1) to make pST39-PCB  Redo of PcyA PCR  PcyA          5 ul Thermopol Buffer      1 ul DNTPs       1 ul fwd primer F.PcyA.pST39.SacI    1 ul reverse primer R.PcyA.pST39.KpnI    1 ul DNA PcyA       40 ul H20       1ul VENT enzyme Thermocycler conditions (1) 95 for 5 min. 2) 95 for 30 seconds. 3) 47 for 30 seconds (increased from 42 last time, don’t remember). 4) 72 for 1 min. 5) Go to 2 4 more times. 6) 95 for 30 seocnds. 7) 53 for 30 seconds. 8) 72 for 1 minute. 9) Go to 6 24 more times. 10) 72 for 5 min. 11) 4 for 24 hours, 12) End.        Check Gel Results We did get the correct 750 bp band, to be ligated into the plasmid tomorrow! I don’t have an image for this, it was on an agarose gel Ivan ran, they just used one lane of it for me.   PcyA Purification Digest of PcyA and pST39-HO1 to make pST39-PCB  PcyA   pST39-Ho1  1 ul Sac1  1 ul Sac1  1 ul Kpn1  1 ul Kpn1  1 ul Dpn1  3 ul Buffer 1  3 uL Buffer 1  3 uL BSA  3 uL BSA Digest 1 hr at 37 (AMY) PCR cleanup column (AMY) Ligate PcyA into pST39-Ho1 to make pST39-PCB (AMY) Transform and plate onto Amp plates (AMY)  Colony PCR on 20 colonies from pST39-PCB plates   Mastermix   22 uL 10x buffer 4.1 uL DNTPS 6.2 uL MgCl2 5 uL F.PcyA 5 uL R.PcyA  1.5 uL Taq 170 uL water  10 uL per reaction x 20 colonies    Gels show a band at the bottom which is probably primers, and colonies 12, 14, 15, 16, 16, 17, 18, and 19 showed a band at ~700, which is the right size for PcyA! Success! Those colonies will then be picked, grown up, and diagnostic digested for PcyA and HO1.    Minipreps of Colonies: 12) 16.2 ng/ul, 14)17.1 ng/ul, 15)10.8 ng/ul, 16)14.5 ng.ul, 17)14.4 ng/ul, 18)15.0 ng/ul, 19) 15.1 ng/ul  Diagnostic Digests of pST39-PCB  Mix for HO1  Mix for PcyA  8 ul EcoRI-HF   8 ul SacI  8ul HindIII   8 uL KpnI  24 ul Buffer 2   24 uL Buffer 1  24 uL BSA   24 uL BSA  96 uL Water   96 uL water   20 ul mix per reaction, 10 uL DNA.   1-7 are digests of 12, 14, 15, 16, 17, 18, 19 for Ho1 (using EcoRI-HF and HindIII)    If the plasmids in fact contained the genes of interest, there should have been a 700 or 750 bp band in each. My hopes that the enzymes just didn’t cut properly were also dashed because the band that is there is about 2.9 kb, which is the size of the original plasmid sent by the Tan lab.   Insertion of PIF3-cFluc into pST39-PCB    Designing mutagenesis primers for pST39-PCB-PIF3 Need to mutagenize the Xba site located at nucleotide 175 in PIF3 sequence because it will cause significant problems with insertion of the PhyBNterm into the plasmid for construction of the vector.  Original: CACAAGCAAACTCTTCTAGAGCTAGAGAGATTGGAAATGG Mutagenesis primer F.PIF3.RemoveXbaI: CACAAGCAAACTCTTCGAGAGCTAGAGAGATTGGAAATGG         R.PIF3.RemoveXbaI: CCATTTCCAATCTCTCTAGCTCTCGAAGAGTTTGCTTGTG   Insertion of nFluc-PhyB into pST-PCB-PIF3-cFluc to make pST39-PCB-SpL aka pST39-iGEM09 PCR of PhyB and Luciferase Nterminus for insertion into the pST39 backbones. This is being done because we need the inserts for the creation of the split luciferase system in bacteria. These fragments will be digested with Kpn1, ligated to eachother, and then PCR amplified for insertion into the pST39-PCB backbones.     Creation of Strains for Testing Two Hybrid System • Yeast were transformed with 1ug of each of the two hybrid plasmids, PhyB in the German vector fused to the Gal4 Binding Domain, and PIF3 in the pACT2 vector, fused to the Gal4 activation domain. Both of those plasmids are high copy number with a 2 micron origin of replication. pACT2 has a Leu2 marker, and Trp1 in the D153dh-phyA FL.  o Y190 yeast cells. Grown on LEU- TRP- plates.  o Cells were grown for 16-20 hours in PCB under red light, and the negatives were in the 30 degree in the dark room.  Plates were coated with PCB at approx 50 uM (based on a rough approximation because it was a crude extract. 100x stock concentration, when Jen dried it as a powder we are under assumption it is pure PCB, so bc paper called for 24 umol, we used 50 uM because we have not HPLC purified it. Purified would probably work better, but our PCB is working). 20 uL extract in 480 uL DMSO, thickly coat a plate, let it dry in incubator for 10-20 min leaves a surface coating of PCB. Crude extract is working! In general everything preincubated in the dark, first with PCB, then subjected to light for another day. Minimum  16 hours in the dark for everything, minimum 16-20 hours. Then followed by 20-24 hours in red light. Plates put in about 5, at 5 developed.  o Developing—an x gal filter lift assay—Z buffer with BME supplemented. Xgal at 1mg/ml, a 50 mg/ml stock is made in DMF. 3 mL soaked on two whatman filter paprs, use nitrocellulose with rough side facing cells, when you see nitrocellulose moist, lift it off using forceps, and lift the colonies on. Let it float on aluminum boat for 20 seconds slowly freezing it, then 2 sec in liquid nitrogen. Let thaw 30 seconds to crack open yeast. Put it cell side up onto filter pads with zbuffer, and then 20 min to an hour you should see blue developments, which get more intense as time goes on. If you leave more than a day you get background blue. In a few days everything turns blue.  o We have repeated 3x the X gal assay to make sure the 2 hybrid system is working. Proof that the constructs work, proof that the PCB works. Those cells can then be transformed with HO endonuclease, or with Gal1-luciferase. Only grow them with PCB when you are going to do the light assay, so not in the liquid culture. Oliver could not get that to work. Need to find those two original constructs and hopefully we have some DNA left for additional transformations to make more functional yeast strains.  • These are PIF3 full length. The one that was partial did not work—we could try re-replica plating it but the PIF3 full was  the only one that  worked. There were 2 more PIF3 subclones that did not work.  • Ones that are turning really blue may have more copies of the plasmids, which risks that they might lose copies over time. We should make more stocks of the plasmids from those subclones that works.     Ivan: YipDCE1 (containing HO1 PcyA) integrate into ADE2 locus in yeast. Can we just add ferreoxin? We can grow them on glucose…we would rather adjust conditions than do genetic manipulation.   Characterization of the system Dose response Waiting for expression timescale PCB titration Sensitivity at different wavelengths Resolution  Other things to do Picking more strains Coculture  Blackboard:  Optical communication—Oliver and Amrita  Characterization: PCB, Time exposure, Wavelength—Oliver and Amrita and Gosia Red light district: HO Endonuclease—Anu and Neena PCB biosynthesis—Jen and Ivan  Knocking out Ura locus—Oliver  Cloning—Amy with Ivan and Amrita and Jen
Week 7: 7/20/09 - 7/24/09
Redoing the Split Luciferase Cloning into the TEF1 integrating vectors We are getting 3 different TEF1 integrating vectors from Addgene, p404TEF1, p405TEF1, and p406TEF1. We gave up on the previous cloning plan because the promoter was probably not strong enough, and because it was a stupid plan for cloning anyway.  p404TEF1 Trp, Amp resistant, Very strong promoter, integrating  http://www.addgene.org/pgvec1?f=c&cmd=findpl&identifier=15972 p405TEF1 Leu, Amp resistant, Very strong promoter, integrating  http://www.addgene.org/pgvec1?f=c&identifier=15968&atqx=p405TEF1&cmd=findpl p406 TEF1 Ura, Amp resistant, Very strong promoter, integrating  http://www.addgene.org/pgvec1?f=c&identifier=15976&atqx=p406TEF1&cmd=findpl  nFluc-PhyB (red or green) will be inserted into p405TEF1—Leu 1. PCR:  nFluc F.Nfluc.SpL.NheI (for both Red and Green) R.Nfluc.SpL.KpnI     PhyB F.PhyB.SpL.KpnI R.PhyB.pST39.BclI.correct   2. Digests  P405TEF1—XbaI (or SpeI)/BamHI  nFluc—NheI/KpnI (NheI ligates to XbaI) PhyB—BclI/KpnI (BclI ligates to BamHI) 3. Ligations—given the success of the three part ligation into the pST39 vector, I will do these as three part ligations; if those do not work we will PCR amplify the ligated insert  PIF3-cFluc will be inserted into p404TEF1—Trp 1. PCR PIF3 full  F.PIF3.SpL.XmaI  R.PIF3.Full.SpL.KpnI PIF3 partial F.PIF3.SpL.XmaI R.PIF3.Partial.SpL.KpnI cFluc   F.cFluc.SpL.KpnI R.Cfluc.SpL.EcoRI 2. Digests  P404TEF1—XmaI/EcoRI PIF3—XmaI/KpnI cFluc—KpnI/EcoRI 3. Ligations—three part ligations; if those do not work we will PCR amplify the ligated insert   PIF3 ligations appeared to yield colonies, but PhyB ligations did not. I think this is attributable to the difference in sizes. PIF3 full is 1600, PIF3 partial is 300, cFluc is 450, PhyB is 1800, nFluc is 1200, so the combination of the 1200+1600 into the 6 kb vector may be too much. This is based on the fact that the PIF3 partial with the smallest inserts has a lot of colonies but the PIF3 full has only a handful. There are a couple of things to try.  1. PCR amplify the ligated PhyB and nFluc inserts a. Ligate the PCR amplified fragments for 1hr at 16 degrees using T4 b. Ligate the PCR amplified fragments for 24 hours at 4 degrees 2. Redo the same three part ligation  a. Ligate 1 hr at 16 degrees using T4 b. Ligate 24 hours at 4 degrees 3. Redo the PCR, digests, and ligations with new fragments a. PCR b. Purify c. Digest d. CIP e. Purify f. Ligate     DBD-PhyB (Nterminal 1-621) PIF3-AD  (full or partial)
Week 8: 7/27/09 - 7/31/09
Bacteria to Yeast Communication Experiments  Your browser may not support display of this image.       Your browser may not support display of this image.  Per Oliver: " Using light as a trans-acting factor to optically bridge a physically separated canonical lac operon"...almost sounds like it belongs in a textbook.  And it just so happens that the P-gex6p2 plasmid is regulated by LacI (which is inhibited by the non-metabolizable allolactose analog IPTG) and our receiver output is encoded by LacZ (which cleaves the galactose analog X-gal or ONPG). Who says B-gal output is BORING?...only if we present it as, "yeah, so the cells turned blue when this gene turned on when it saw red light which came from some other cell that made luciferase....Now we have some historical context. Jacob and Monod would approve.”   Bacteria to Yeast Communication Experiment 1—Small bacterial petri dish sitting on large yeast petri dish.  Two mls of bacterial culture were grown overnight, then incubated with luciferase and placed in a sterile petri plate. That petri plate was placed on top of PIF3/PhyB yeast growing on Trp-/Leu- media. In order to test light-based communication between bacteria and luciferase, bacteria were transformed with luciferase plasmids under constitutive promoter, and expressed red lucifeerase. Cells grown overnight, spun down, resuspended in 4ml sodium citrate buffer with 1 mMolar luciferin. Yeast were grown from plate 2 that was apparently light sensitive grown on PCB plates overnight, and then small petri dish with resuspended luciferase bacteria placed on center of yeast plate. Sealed and put in incubator over night. Tomorrow will do filter lifts and assay for beta-gal expression. Hopefully will have circle of expressing. Cells are almost a lawn.  Your browser may not support display of this image.    Bacteria to Yeast Communication Experiment 2—separated by petri dish, increased number of bacterial cells  In this experiment the bacteria were placed in a small petri dish in the center of the plate contining yeast growing on solid agar. There is blue on the Bacterial light + PCB plate, but not on any of the negative controls. This is proof of principle that red light from bacteria can excite the system!!!!   Your browser may not support display of this image. Your browser may not support display of this image. Your browser may not support display of this image. Your browser may not support display of this image.    Bacteria to Yeast Communication in Solid Media Separated by Petri dish-- Experiment 3  Take 1: Bacteria have been IPTG inducing for 7 hours now. 10 mMol IPTG. This is from an overnight culture, 25 mL of 2xYT. Used frozen stock of cells, Red Luciferase in the PGEX IPTG inducible plasmid. At 1pm overnight bacterial culture was split and IPTG induction begun. Did 1 in 10 dilution (5mL cells into 50ml 2xYT with amp). One was IPTG induced with 10 mMol, .238 g for 100 mL culture. One culture was not IPTG induced. There was another culture that was not from frozen stock but was from plate. PDCE plasmid that was from an Ecoli transformation plate with amp resistance as a negative control, not expressing luciferase. That was into 70 mL YT with 70uL amp. That has been growing for 7+ hours as well. Overnight inoculations had been done at 11pm.  Yeast—two 2mL cultures growing, one with DMSO, 50 uMol (40uL of 100x PCB stock, which we assume to be 50uMol final concentration, we assume majority is Phycocyanobilin). That was inoculated last night at 1030pm. Still in the incubator at 30 degrees in the dark. They have been in there for 22 hours. 16 hours in PCB is recommended.  It is probably a very saturated culture, not growing well though. That is the problem.  Ideally we should split those cells and let them recover for a while…how crucial is that? We will do it tonight and again tomorrow. We have yeast that have been recovering for 2 hours (washed). We could throw them all together—Oliver was going to do a laser timecourse. For the laser timecourse,  Because it got too late we eneded up not doing this experiment on the night of Aug 4, we are going to actually do it on Aug 5.    Take 2: Bacterial cells were IPTG induced beginning at 1pm. Apparatus for suspensions of luciferase bacteria was constructed by crazygluing the lid of a small petri dish to the inside of the lid of a larger one, and a hole melted into the top to create a way to inject the cells into the pocket thus created. See images on camera. 2x 15 mL cultures of IPTG uinduced bacteria were spun down and resuspended in 5 mL ea. Sodium citrate buffer with 1 mmol luciferin. An addition 40 UL of 100 mMol luciferin was spiked in and seen to result in an increased luminescent signal by eye. 4 mL of bacteria were placed in the holding chamber for both PCB treated yeast and DMSO treated yeast. Two additional negative controls were also done, PCB treated yeast and DMSO treated yeast without bacteria. 50 ul of 100 mmol luciferin was spiked in at 1 hour, 2 hours, and 3 hours. Large mirrors were placed below the plates and small above, and the whole shebang wrapped in foil and placed on a shaking incubator at 30 degrees. By eye it was difficult to see luminescence at 1 or 2 hours, even though initial luminescence was extremely bright. Luminescence at 1 hour was clearer with coculture.  The result of this experiment was negative. There was no visible blue on any of the plates, more or less. This experiment will need to be repeated.  This is an image of the bacteria-yeast communication apparatus we designed so that the bacteria would not have to sit right on the yeast plate and therefore kill everything under it. There is one main hole for aeration and allowing us to spike in luciferase, and a smaller vent hole. The smaller dish is glued to the inside of the lid so that there is a 2 mm gap between it and the surface of the agar on which the yeast are growing.  Your browser may not support display of this image.    Bacteria and Yeast Coculture—Liquid—Experiment 1.  0.5 mL yeast and 2 ml bacteria in 5 umol luciferin. At the end of the timepoints spin those down atmax speed, pellet them put them in -80. Tomorrow will do an ONPG assay or something. 1. PCB yeast + glowing bacteria, 2. One with no PCB + glowing bacteria. 3. One with PCB yeast and nonglowing bacteria. Cells were spiked with 20 uL of 100 mMol luciferin at 1hr, 2hr, and then spun down and frozen at-80 degrees at 3 hours. Will be xgal and onpg assayed. As of 8/10 they are still in the freezer, this assay will probably have to be redone because of the age of the cultures.    Bacteria to Yeast Communication in Solid Media Separated by Petri dish-- Experiment 4. Changing speed of shaking; More aeration holes  Bacterial cells IPTG induced at 11:30 am, to grow for 5 hours, and be spun down at 3:30, for 4 hours of induction and assay begun at 4. 150 mL of medium inoculated with 10 mL of bacteria from overnight culture and 0.75 g IPTG added. Spiking with luciferase once an hour until 7. Dinner at 7. We made several changes in this attempt: we increased the shaking speed, and increased the number of aeration holes. This meant bacteria splashed out everywhere, and needed to be replenished at 11 pm (after 4 hours of IPTG induction). However, I also forgot to put the lid apparatuses on the yeast plates. Epic fail. Therefore there was no data from this experiment because we had a signal, but no receiver. However, the cells were kind of old so it’s ok. We will repeat Tuesday.  Your browser may not support display of this image.      Bacteria to Yeast Communication in Solid Media: Experiment 5—Using smaller dish, supplementing medium with Histidine, actually using the yeast plates, using fresh cultures.  Prepared small plates for assay using: 25 mL agar, 2.5 mL 10x dropout medium, 0.3 g nitrogen compounds mixed with 2 mL 40% glucose and filter sterilized, and bring to 50 mL with Trp-Leu- medium. Added ~8 mL to each small plate, making a total of 6 plates.   Lids going to be of equal size to plate.  IPTG induce 300 mL of culture for 5 hours because observed significan increase in bioluminescence when allowed to induce for 5 hours as opposed to 3 in failed Experiment 4.    Bacteria to Yeast Communication in Liquid Media: Experiment 1  Yeast placed in small test tube and sealed with sterile velvet stuffed in it. Small test tube placed in larger one containing bacteria+luciferin. Spike luciferin at regular intervals.    Note: Preparing Materials for Filter Lifts  Cut nitrocellulose and filters to the appropriate shapes. If using BioDot, clean apparatus, insert thick filter, and prewet with water. In case of biodot or plates, place a Whatman filter paper in a container: for a plate a petri dish, for biodot, a 200 uL tip box lid.  Preparation of Beta Gal solution: in 20 mL of Z buffer, add 54 uL of beta mercaptoethanol, and 1 mg/mL of Beta gal dissolved in DMF. I diluted the Beta Gal in DMF at 50 mg per 500 uL, and added then 200 uL to the solution.
Week 9: 8/3/09 - 8/7/09
Characterization of the Two Hybrid System  Laser Stimulation on Plates  Light was shone from a 660 nm laser on five points on the below plates for 10 seconds per point, with four points on the outside and a point in the center forming a cross. There is diffuse blue on the PCB+light plate and no blue on the control plates, indicating a positive result for light induced production of beta-gal.  Your browser may not support display of this image.      Qtip Assay  Take 1—Testing length of exposure of cells to light.
Week 10: 8/10/09 - 8/14/09
BioDot Characterization Assays  Biodot Assay I: Exposure time and Time to develop  Cells were exposed for sweep, 1 sec, and 10 sec, , in three replicates, one for each timepoint of ½  hour, 1 hr, 3 hours. I did this with oliver. After half an hour and 1 hours we loaded samples onto the biodot and  did X gal assay. We did not do third timepoint due to apparent extremely high background from PCB. This turned out to fade over time. The fourth column on each filter is a DMSO only control, shone for 10 seconds. These cells were taken from relatively new cultures during week 8. This assay worked to show that (1) there is more apparent signal at 1 hour than half an hour, and (2) the longer the exposure (up to 10 seconds) the bright the signal, and (3) the DMSO only control does not cause xgal assay to turn blue, thus making false positives  attributable to cell age.  Your browser may not support display of this image.      PCB Concentrations  2 plates. 2 rows equivalent, two different tubes, were 0 PCB, 0 DMSO. Cells grown overnight. 1 ML cells from culture, spun down at 8,000 rpm, pelleted and took out medium, washed, re-suspended in synthetic Trp-Leu-medium, and resuspended in 1ml. So 1 mL of culture from 2mL of culture originally incolutated at 10 pm last night. Took OD600 reading diluted at 1:10. DMSO makes the cells clump so you really have to vortex and shake them. Too much DMSO inhibits their growth, and PCB up to high concentrations does not seem to inhibit it. Need to multiply all number by 10. Not in triplicate, probably +- 0.05. Need to be consistant about shaking up REALLY well. You will be significantly off otherwise.        PCB DMSO Equivalents  0 uMol  .607 .586   5 .583 .518  10 .599 .533  25  .552 .452  50  .592 .432  100  .373 .316  125  .260 .175  The decrease in growth is apparently a function of DMSO concentration, not PCB, since it is seen with both.  Did a 1:10 dilution of everything. when you zapped with laser, each had 100 uL aliquots of cells. All numbers below .607, everything was brought up to .6 OD worth of cells, should be 1x10^7 cells per mL, and 100 uL of cells. Then the plates were staggered—on one plate you have 2 rows of cells with no PCB no DMSO. In second plate they were staggered. First had 5 uMol DMSO, 5 mMol PCB, etc. Each got 10 seconds. Now sitting in incubator, finished at 8pm, going to go till 11, then transfer onto filters.    Cells were loaded into the biodot as follows, and two replicates of this were made (for a total of 8 samples per condition between the two films). Cells were loaded onto the nitrocellulose three hours after exposure. Labelled photographs PCB Concentrations I, II, and III. 5 uM PCB  10 uM PCB  25 uM PCB  50 uM PCB  100 mM PCB  125 mM PCB 5 uM PCB  10 uM PCB  25 uM PCB  50 uM PCB  100 mM PCB  125 mM PCB 5 uM PCB  10 uM PCB  25 uM PCB  50 uM PCB  100 mM PCB  125 mM PCB 5 uM PCB  10 uM PCB  25 uM PCB  50 uM PCB  100 mM PCB  125 mM PCB 5 uM DMSO  10 uM DMSO  25 uM DMSO  50 uM DMSO  100 uM DMSO  125 uM DMSO 5 uM DMSO  10 uM DMSO  25 uM DMSO  50 uM DMSO  100 uM DMSO  125 uM DMSO 5 uM DMSO  10 uM DMSO  25 uM DMSO  50 uM DMSO  100 uM DMSO  125 uM DMSO 5 uM DMSO  10 uM DMSO  25 uM DMSO  50 uM DMSO  100 uM DMSO  125 uM DMSO   No light no DMSO control  No light no DMSO control No light no DMSO control  No light no DMSO control No light no DMSO control  No light no DMSO control No light no DMSO control  No light no DMSO control No light no DMSO control  No light no DMSO control No light no DMSO control  No light no DMSO control No light no DMSO control  No light no DMSO control No light no DMSO control  No light no DMSO control  Your browser may not support display of this image. Your browser may not support display of this image. Your browser may not support display of this image.    Conclusions: The experiment did not work because the cells are too old.    Laser Time-course  Times: 0.5sec, 1 sec, 5 sec, 10 sec, 30 sec, 1 min.  (DMSO, PCB, DMSO, PCB, etc). Using 50 mMol concentration. Overnight culture diluted to 1:10, diluted at 6pm, now it is 8:40. Load entire 96 well plate. Instead of a sweep we should do 0.5 sec. We will put 100 uL in each well. Take another OD reading of that sample. Going to be between about 0.6 and 0.8 OD600 for each well. They have been sitting since 6pm in media with no PCB. Still should be a lot of cells with active PCB…cytoplasm gets split evenly? Do they get diffused out? We only need a few complexes to initiate beta gal expression. Will be staggered like other plate, one row DMSO, one row PCB.  Cells were loaded into the biodot as follows, and two replicates of this were made (for a total of 8 samples per condition between the two films). Cells were loaded onto the nitrocellulose three hours after exposure.  Labelled photographs Timecourse I and II. 0.5 sec +PCB  1 sec + PCB  5 sec + PCB  10 sec + PCB  30 sec + PCB  1 min + PCB 0.5 sec +PCB  1 sec + PCB  5 sec + PCB  10 sec + PCB  30 sec + PCB  1 min + PCB 0.5 sec +PCB  1 sec + PCB  5 sec + PCB  10 sec + PCB  30 sec + PCB  1 min + PCB 0.5 sec +PCB  1 sec + PCB  5 sec + PCB  10 sec + PCB  30 sec + PCB  1 min + PCB No light +PCB     No light +PCB          No light +PCB     No light +PCB          No light +PCB     No light +PCB          No light +PCB     No light +PCB           Your browser may not support display of this image. Your browser may not support display of this image.  This assay did not work, all lanes turned blue, even the ones without light treatment or ones without PCB. We think there might be a mutation in the strain which results in constitutive expression of beta gal, or a contamination with some fungus or bacterium that does. If it is bacterial contamination this may pose a problem for the co-culture assay. Hopefully not though. This assay will be repeated with a lower density of cells, cultured from a frozen stock. Meanwhile I am looking for additional strains that will work.    Per Oliver:  “So regarding the observation that the Y190 pif3/Phyb strains have a high background of B-gal expression when taken from either later cultures or an older plate VS. from the earlier frozen glycerol stocks. I have a hypothesis: There shouldn't be a selective advantage to any cells that have a "leaky" or mutated gal1 promoter driving B-gal, quite the contrary since they're growing in glucose, so it would be a waste of resources to crank out b-gal. However, the other reporter gene in this strain, which is also regulated by a Gal1 promoter is the HIS3 gene, which if turned on all of the time, will give cells a selective advantage if they're on media which is depleted of supplemented histidine, i.e. old overnight cultures, old plates, etc. Since B-gal is also turned on, maybe whats happening is that we are over time inadvertantly selecting for mutation(s) in either PhyB or Pif3 (and remember, there are multiple copies of these plasmids within each cell) that enable both proteins to bind constitutively now sans light ( or hell, maybe even sans PCB).  If this is correct, then then any Gal1 promoter regulated gene would be constitutively on in such a strain. We can test this hypothesis, if we choose to later, to see if HIS3 is also up regulated and also if Pif3/PhyB are now bound sans light in a pull down assay, and whether those Pif3/PhyB plasmids have a particular point mutation that enable them to be constitutively bound. Also whether this mutation can crop up under other selective pressures, ie. growing the strain using galactose as the sole carbon source, for protracted periods of time.  So what does this mean practically, for the rest of the experiments, if correct? Well, we should make lots of FRESH overnight cultures directly from our frozen stocks,  making sure that the media has extra histidine supplemented, and freeze those down. Make more preps of the PhyB and Pif3 plasmids that are working. Consider putting those into another strain (Y187) that does not have a gal1 promoter driven marker that would offer a selective advantage when kept turned on. Keep all of this in mind when we're making the stable transformants, maybe we'll have to generate those first and then put the plasmids back in (either Y190 or Y187).  I haven't done any literature searches yet to see if this can likely occur in the context of a 2-hybrid system or if this effect has been already observed in other regulated selectable marker systems that offer a metabolic/nutritional advantage. But it makes sense to me if you consider that 5-FOA is used as a negative selection compound to screen for randomly arising URA3 point mutants.
Team
Projects
Bacteria-to-Yeast Communication
Characterization
PCB Biosynthesis in Yeast
Split Luciferase
Other Possible Reporters
Parts
References
Lab Notebook
Daily
Protocols
Recipes
Ethics
Safety
Acknowledgements
Retrieved from " http://2009.igem.org/Team:Harvard/Daily "
Recent changes
What links here
Related changes
Special pages
My preferences
Printable version
Permanent link
Privacy policy
Disclaimers
