<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_NUS_Singapore-A_Design skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><DIV id="top_title"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:NUS Singapore-A/Design</SPAN></H1></DIV><DIV id="HQ_page"><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><DIV class="nusgem_mail">CONNECT WITH US</DIV><DIV class="outer-border bg-primary"><DIV class="inner-border" style="background-image: none;"><DIV class="nav" id="myNav"><DIV class="dropdown"><BUTTON class="dropbtn"><A href="https://2018.igem.org/Team:NUS_Singapore-A" style="text-decoration: none; color: black;">HOME</A></BUTTON></DIV><DIV class="dropdown"><BUTTON class="dropbtn">PROJECT</BUTTON><DIV class="dropdown-content"><A href="https://2018.igem.org/Team:NUS_Singapore-A/Description">Description</A><A href="https://2018.igem.org/Team:NUS_Singapore-A/Design">Design</A><A href="https://2018.igem.org/Team:NUS_Singapore-A/Model">Modelling</A><A href="https://2018.igem.org/Team:NUS_Singapore-A/Results">Results</A><A href="https://2018.igem.org/Team:NUS_Singapore-A/Demonstrate">Demonstrate</A></DIV></DIV><DIV class="dropdown"><BUTTON class="dropbtn">LAB</BUTTON><DIV class="dropdown-content"><A href="https://2018.igem.org/Team:NUS_Singapore-A/Notebook">Notebook</A><A href="https://2018.igem.org/Team:NUS_Singapore-A/Safety">Safety</A><A href="https://2018.igem.org/Team:NUS_Singapore-A/InterLab">InterLab Study</A><A href="https://2018.igem.org/Team:NUS_Singapore-A/Measurement">Measurement</A></DIV></DIV><DIV class="dropdown"><BUTTON class="dropbtn">PARTS</BUTTON><DIV class="dropdown-content"><A href="https://2018.igem.org/Team:NUS_Singapore-A/Part_Collection">Parts Overview</A><A href="https://2018.igem.org/Team:NUS_Singapore-A/Composite_Part">Composite Parts</A><A href="https://2018.igem.org/Team:NUS_Singapore-A/Improve">Improved Parts</A></DIV></DIV><DIV class="dropdown"><BUTTON class="dropbtn">HARDWARE</BUTTON><DIV class="dropdown-content"><A href="https://2018.igem.org/Team:NUS_Singapore-A/Hardware">Overview</A><A href="https://2018.igem.org/Team:NUS_Singapore-A/Hardware/PDF-LA!">PDF-LA!</A><A href="https://2018.igem.org/Team:NUS_Singapore-A/Hardware/Sensor">2-in-1 Sensor</A><A href="https://2018.igem.org/Team:NUS_Singapore-A/Hardware/Pump">Peristaltic Pump</A><A href="https://2018.igem.org/Team:NUS_Singapore-A/Hardware/CoJar">Fermentation Chamber</A><A href="https://2018.igem.org/Team:NUS_Singapore-A/Hardware/Futuristic_Bioreactor">Futuristic Bioreactor</A></DIV></DIV><DIV class="dropdown"><BUTTON class="dropbtn">HUMAN PRACTICES</BUTTON><DIV class="dropdown-content"><A href="https://2018.igem.org/Team:NUS_Singapore-A/Human_Practices">Integrated Human Practices</A><A href="https://2018.igem.org/Team:NUS_Singapore-A/Public_Engagement">Education &amp; Public Engagement</A><A href="https://2018.igem.org/Team:NUS_Singapore-A/Collaborative_Report">Collaborative Report</A></DIV></DIV><DIV class="dropdown"><BUTTON class="dropbtn">TEAM</BUTTON><DIV class="dropdown-content"><A href="https://2018.igem.org/Team:NUS_Singapore-A/Team">Members</A><A href="https://2018.igem.org/Team:NUS_Singapore-A/Supervisors">Supervisors</A><A href="https://2018.igem.org/Team:NUS_Singapore-A/Sponsors">Sponsors</A><A href="https://2018.igem.org/Team:NUS_Singapore-A/Attributions">Attributions</A><A href="https://2018.igem.org/Team:NUS_Singapore-A/Collaborations">Collaborations</A><A href="https://2018.igem.org/Team:NUS_Singapore-A/Medals">Medal</A><A href="https://2018.igem.org/Team:NUS_Singapore-A/Gallery">Gallery</A><A href="https://2018.igem.org/Team:NUS_Singapore-A/Jamboree">Giant Jamboree!</A></DIV></DIV><A href="javascript:void(0);" class="icon" onclick="myFunction()">☰</A></DIV></DIV><BUTTON onclick="topFunction()" id="BacktoTopBtn" title="Go to top">Back to Top</BUTTON><DIV class="inner-border" style="border-top: none;"><DIV class="all-wrapper"><DIV class="content-wrapper"><H1> OUR PLATFORM DESIGN: AN OVERVIEW </H1><P>Not content with simple fermentation to produce our target compounds, we sought to address the gaps in current biomanufacturing and incorporated various features into <I>Coup Dy'état</I>. Our platform system was designed with three main principles in mind: commercial viability, eco-friendliness, and emerging technology adoption (<A href="https://2018.igem.org/Team:NUS_Singapore-A/Design#DesignFig01">Figure 1</A>).</P><FIGURE id="DesignFig01" class="figures2" style="max-width:90%;"><FIGCAPTION><B>Figure 1. Design principles of <SPAN><I class="no-italics">Coup Dy'état</I></SPAN></B>.</FIGCAPTION></FIGURE><P>In our project, we set out to demonstrate the possibility of converting waste to dyes in <I>E. coli</I> BL21 Star (DE3) (thereafter, BL21*) via dynamic and stress regulation, achieved using computer-assisted optogenetic control. The illustration below encapsulates how all the features are integrated together to complete the bioproduction process. </P><P>The various parts of our system are designed to be modular; they can be incorporated into another system and be expected to work. With the advent of more biomanufacturing technologies, additional features can be assimilated seamlessly. <I>Coup Dy'état</I> is thus a technological platform where microbial production of chemicals, drugs, or biomaterials can be optimized.</P><P>With <I>Coup Dy'état</I>, we hope to set the standard for biomanufacturing. </P><P>Click on each segment of the illustration to find out more about each feature of our system! Let's begin with xylose, the feedstock, and then down the biosynthetic pathway to luteolin, our target compound, before exploring the various elements of our technological platform! </P><BUTTON id="XYL" class="accordion"> XYLOSE AS FEEDSTOCK </BUTTON><DIV class="panel"><P>We are certain that our approach is much less harmful to the environment than the production of synthetic dyes. But what if we compared our project to traditional methods of obtaining natural dyes? While synthetic biology opens up new vistas for the bioproduction of heterologous compounds, we can only consider ourselves a superior alternative to plant cultivation and extraction if the process of acquiring the sugar feedstock for fermentation consumes significantly less land, water, and other resources <SUP>[1]</SUP>.
                </P><P>
But that’s not all. From sucrose derived from sugarcane in Brazil to glucose extracted from corn in the USA, the demand for sugar sources in biomanufacturing competes with demands for animal feeds and human consumption <SUP>[2]</SUP>. Additionally, sugarcane farming takes a substantial toll on the environment, fuelling deforestation, soil degradation, water pollution, and biodiversity loss <SUP>[3]</SUP>.
                </P><P>
Our team realized that we needed to ensure that our bioproduction method is itself eco-friendly and sustainable. We thus turned to lignocellulosic waste. It is the underutilized, non-edible portion of plants found in empty fruit bunches, the major biomass byproduct of the palm oil industry <SUP>[4]</SUP>. This is incinerated in abundant quantities in plantations all over Indonesia, the largest palm oil producer in the world, contributing to air pollution and haze <SUP>[4]</SUP> which plague her neighbours, including Singapore.
                </P><P>
All these <I>burning</I> issues can be alleviated if we can feed lignocellulosic waste to our bacteria. However, there is a major hurdle. Xylose and glucose are the major constituents of lignocellulose, and economically feasible bioconversion requires one to use both sugars at the same time. Unfortunately, this is hindered by carbon catabolite repression (CCR), a regulatory mechanism where expression of genes necessary for the utilization of secondary carbon sources is suppressed by the presence of a preferred substrate, which is often glucose (<A href="https://2018.igem.org/Team:NUS_Singapore-A/Design#DesignFig02">Figure 2a and b</A>) <SUP>[5]</SUP>.
                </P><FIGURE id="DesignFig02" class="figures2"><FIGCAPTION style="text-align:justify"><B>Figure 2. Relieving carbon catabolite repression of xylose.</B><B>a</B>, Xylose activates the xylose regulatory operon protein, XylR. However, the presence of glucose inhibits the activation of cAMP receptor protein, CRP. Without the activated CRP, XylR is unable to bind to its binding site at the operator, and RNA polymerase, RNAP, cannot bind to the promoter and transcription does not take place. 
                <B>b</B>, In the absence of glucose, CRP is activated and forms a complex with XylR. This complex binds to the bidirectional operator at the respective binding sites of the proteins, facilitating the binding of RNAP and transcription of the xylose metabolic genes, XylA and XylB, and the transport genes, XylF, XylG, and XylH. 
                <B>c</B>, The mutant XylR* is able to bind to the operator without the need to form a complex with CRP, hence transcription takes place even in the presence of glucose.</FIGCAPTION></FIGURE><P>
Recently, CCR has been shown to be released by performing two substitution mutations in the xylose operon regulatory protein (XylR) gene in <I>E. coli</I>, which creates a strain that is able to co-utilize xylose and glucose efficiently. This mutant, XylR*, has a higher affinity to its DNA binding site, and thus activates its promoter independently of the co-activator, the cAMP receptor protein (CRP) (<A href="https://2018.igem.org/Team:NUS_Singapore-A/Design#DesignFig02">Figure 2c</A>). As a result, expression of xylose metabolic and transport genes occurs even in the presence of glucose <SUP>[5]</SUP>!
                </P><FIGURE id="DesignFig03" class="figures2" style="max-width:40%;float:right;margin:15px"><FIGCAPTION style="text-align:justify"><B>Figure 3. The xylose-utilizing module.</B>
This construct consists of the coding sequence of the mutant XylR* placed under the control of the lactose-inducible P<SUB>T5</SUB> promoter. Hence, IPTG induction stimulates expression of XylR*. </FIGCAPTION></FIGURE><P>
This had huge implications for our project as converting waste into dyes is now viable. We thus constructed a plasmid which allows overexpression of XylR* upon induction by IPTG (<A href="https://2018.igem.org/Team:NUS_Singapore-A/Design#DesignFig03">Figure 3</A>)), and demonstrated its effectiveness in enhancing xylose utilization in <I>E. coli</I> BL21* by conducting growth experiments.
                </P><P>
We deviated from our initial plan of engineering a xylose-utilizing strain like what was accomplished in the study, it would be easier to transform this construct into our dye-producing bacteria, allowing xylose to serve as the feedstock. This also enables potential optimization of the expression levels of XylR* and balancing of metabolic flux, which may vary depending on the biosynthetic pathway or fermentation conditions.
                </P><BUTTON class="accordion-closer">CLOSE</BUTTON></DIV><BUTTON id="NAR" class="accordion"> DE NOVO BIOSYNTHESIS </BUTTON><DIV class="panel"><DIV class="panel-inside"><FIGURE id="DesignFig04" class="figures2" style="max-width:60%;float:right;margin:15px"><FIGCAPTION style="text-align:justify"><B>Figure 4. The flavonoid biosynthetic pathway.</B>
The various subgroups of flavonoids such as flavones, flavonols, anthocyanins, catechins, etc. can all be synthesized from naringenin, the intermediate platform compound.  
                    </FIGCAPTION></FIGURE><P>Along the targeted pathway, we took special interest in a key intermediate compound, naringenin. Naringenin, a flavanone, is extremely valuable due to its antioxidant, anti-inflammatory, antiviral, and anti-tumour properties <SUP>[6]</SUP>. More intriguingly, naringenin is also the intermediate platform compound for the biosynthesis of other flavonoids, many of which are potential candidates for textile dyes (<A href="https://2018.igem.org/Team:NUS_Singapore-A/Design#DesignFig04">Figure 4</A>).
                </P><P>
                    While commercial naringenin can be added as the substrate during the biomanufacturing process, we find it desirable to achieve de novo biosynthesis given its strategic location in the biosynthetic pathway, which provides us with the basis for the synthesis of many other high-value flavonoids. Besides reducing costs, this would eliminate the need to introduce expensive chemical precursors <SUP>[7]</SUP>, which is consistent with our team’s vision of involving less chemicals in the biomanufacturing process.
                </P><P>
Serendipitously, there was a recent breakthrough involving the production of naringenin from xylose via a synergistic co-culture system comprising <I>E. coli</I> and <I>S. cerevisiae</I><SUP>[6]</SUP>. Subsequent literature review also revealed the advantages of co-culture biosynthesis, with each independent strain devoted to the functional expression of different stages of the biosynthetic pathway, allowing for reduced metabolic burden, flexibility of employing suitable hosts, and adjustable biosynthetic strength <SUP>[8]</SUP>. Inspired by these revelations, we decided to adopt a co-culture strategy as well, especially considering the length of the biosynthetic pathway and how we intended to use xylose in our feedstock.
                </P><FIGURE id="DesignFig05" class="figures2" style="max-width:70%;float:right;margin:15px"><FIGCAPTION style="text-align:justify"><B>Figure 5. Recombinant expression of enzymes for de novo  naringenin synthesis in <I class="no-italics">E. coli.</I></B><B>a</B>, The biosynthetic pathway of naringenin starts from the native amino acid, tyrosine. The non-native enzymes PAL, 4CL, and OsPKS are cloned into <I>E. coli</I> to convert tyrosine to naringenin, while MCS is also overexpressed to produce sufficient amounts of malonyl-coA which is consumed in the process. Naringenin chalcone spontaneously cyclizes to naringenin.
                <B>b</B>, OsPKS and MCS are placed under a lactose-inducible promoter and their expression is induced by IPTG, while 4CL is constitutively produced. </FIGCAPTION></FIGURE><P>
Therefore, we sought to produce naringenin from tyrosine in a single <I>E. coli</I> strain, which could then be co-cultured with cells responsible for the synthesis of our final compound, thus completing the pathway. This process involves catalysis by four enzymes - phenylalanine ammonia lyase (PAL), 4-coumarate-CoA ligase (4CL), <I>Oryza sativa</I> polyketide synthase (OsPKS), and malonyl-CoA synthetase (MCS) <SUP>[9]</SUP> put together in a single plasmid (<A href="https://2018.igem.org/Team:NUS_Singapore-A/Design#DesignFig05">Figure 5a</A>).
                </P><P>
Currently, our de novo construct carries three of the enzymes necessary for naringenin production. While 4CL is placed under a constitutive promoter, OsPKS and MCS are strategically put under the control of a P<SUB>lac</SUB> promoter instead (<A href="https://2018.igem.org/Team:NUS_Singapore-A/Design#DesignFig05">Figure 5b</A>). This enables overexpression of the two more crucial enzymes involved in the latter stages of the pathway, thus helping to drive the reaction forward.
                </P><P>
By supplementing the medium with coumaric and malonic acid as substrates, we demonstrated the production of significant amounts of naringenin in <I>E. coli</I> BL21* cells carrying the de novo construct. The production of naringenin was verified using High Performance Liquid Chromatography (HPLC), while the expression of the enzymes was confirmed via SDS-PAGE.
                </P><P>
With the eventual complete construct, de novo biosynthesis of naringenin can be achieved with just the introduction of this plasmid. This design complements the use of xylose as feedstock to achieve a much more environmentally-friendly bioproduction of naringenin, and ultimately, dyes.
                </P></DIV><BUTTON class="accordion-closer">CLOSE</BUTTON></DIV><BUTTON id="LUT" class="accordion">LUTEOLIN</BUTTON><DIV class="panel"><P>
The choice of target compound was given careful consideration as its production is the raison d’être of <I>Coup Dy’état</I>, and the design of all the features in our system would revolve around its production. Our desire to renounce synthetic dyes prompted us to look for natural alternatives to serve as the first product of our system.
                </P><FIGURE id="DesignFig06" class="figures2" style="max-width:60%;float:right;margin:15px"><FIGCAPTION style="text-align:justify"><B>Figure 6. The leaky flavonoid biosynthetic pathway, and luteolin.</B>
The conversion of chrysanthemin, callistephin, and myrtillin from naringenin involves the same enzymes: F3H, DFR, ANS, and 3GT, while that of theaflavin also includes the former two enzymes. As can be seen, these enzymes act on multiple substrates, hence their promiscuity and the need to exert tight control. Luteolin can be synthesized from naringenin using two enzymes. F3’H converts naringenin to apigenin, another yellow flavone, and FNS synthesizes luteolin from apigenin. </FIGCAPTION></FIGURE><P>
We explored the production of various flavonoids, all derived from the intermediate platform compound, naringenin. Chrysanthemin (pink), callistephin (orange), and theaflavin (reddish-brown) were our original candidates as many enzymes are shared across their biosynthetic pathways. A range of colours could then theoretically be produced from a mixture of these compounds, whose relative concentrations could be varied by controlling the expression of key enzymes (<A href="https://2018.igem.org/Team:NUS_Singapore-A/Design#DesignFig06">Figure 6</A>). However, the promiscuity of the enzymes and inherent leakiness of the pathway necessitate extremely tight control of gene expression which was technically challenging to accomplish (see: <A href="https://2018.igem.org/Team:NUS_Singapore-A/Model"><I>Modelling</I></A>).
                </P><P>
The major turning point occurred when we interviewed designers and synthetic dye companies and learned that such colours were not commercially viable. Primary colours are preferred, as they could be mixed to produce the desired hue. Following this, we opted to focus our efforts on the synthesis of luteolin, a yellow flavone located downstream of naringenin too (<A href="https://2018.igem.org/Team:NUS_Singapore-A/Design#DesignFig06">Figure 6</A>).
                </P><P>
Notwithstanding the warmth and optimism that it evokes, the colour yellow has a murky history. The vivid Indian Yellow dye was derived from the urine of cows which had been fed only mango leaves and were thus kept in a perpetual state of near-starvation, yellow to be dubbed the “cruellest” colour <SUP>[10]</SUP>. Presently, the synthetic yellow compound PCB-11, which possesses a plethora of toxic effects, is found ubiquitously in textiles and other consumer products <SUP>[11]</SUP>. Luteolin, which is perhaps the oldest European dye, enjoys a much better reputation. It is deemed to possess the best light fastness amongst natural yellow dyes, and is still used to dye silk, wool, cotton, and leather <SUP>[12]</SUP>. A bright future for yellow seemed to be in the cards with luteolin. Our choice was also validated by our stakeholders (see: <A href="https://2018.igem.org/Team:NUS_Singapore-A/Human_Practices#iHP"><I>Integrated Human Practices</I></A>).
                </P><P>
Once we confirmed that we would be producing luteolin, we expressed the two enzymes required to produce it from naringenin in <I>E. coli</I>: flavone synthase (FNS) from <I>Petroselinum crispum</I> and flavonoid 3’-hydroxylase (F3’H) from <I>Gerbera hybrida</I> (<A href="https://2018.igem.org/Team:NUS_Singapore-A/Design#DesignFig06">Figure 6</A>) <SUP>[13] [14]</SUP>. <I>E. coli</I> BL21* was selected to be our chassis as it is an RNase knockout strain, which enhances protein expression and dye production <SUP>[15]</SUP>. It was also shown to be a suitable host for flavonoid synthesis <SUP>[16]</SUP>. The expression of the enzymes FNS and F3’H are optogenetically controlled (see: <A href="https://2018.igem.org/Team:NUS_Singapore-A/Design#BLS"><I>Blue Light Repressible System</I></A>).
                </P><P>
Through multiple rounds of biosynthesis, we were able to successfully demonstrate the production of luteolin via HPLC. The expression of our luteolin-producing constructs were also investigated through RT-qPCR. 
                </P><BUTTON class="accordion-closer">CLOSE</BUTTON></DIV><BUTTON id="BLS" class="accordion">BLUE LIGHT REPRESSIBLE SYSTEM</BUTTON><DIV class="panel"><DIV class="panel-inside"><FIGURE id="DesignFig07" class="figures2" style="max-width:60%;float:right;margin:15px"><FIGCAPTION style="text-align:justify"><B>Figure 7. Downsides of using chemical inducers in bioproduction, and corresponding advantages of using light.</B> Watch the animation to find out more! </FIGCAPTION></FIGURE><P>
Where induction of gene expression is necessary, chemical inducers are staples owing to their capability to trigger high expression levels <SUP>[16]</SUP>. However, the drawbacks are numerous: potential cellular toxicity, non-reversible induction, delayed diffusion, and high costs<SUP> [18] [19]</SUP>. All these problems are exacerbated in large-scale biomanufacturing (<A href="https://2018.igem.org/Team:NUS_Singapore-A/Design#DesignFig07">Figure 7</A>).
                </P><P>
Enter optogenetics. The allure of being able to manipulate cellular behaviours using light is causing optogenetics to make waves in the field of synthetic biology. Light-based induction offers many advantages: non-toxicity, reversibility, precise spatiotemporal control, tunability, and availability <SUP>[20] [21]</SUP> (<A href="https://2018.igem.org/Team:NUS_Singapore-A/Design#DesignFig01">Figure 7</A>). Hence, optogenetics offers a compelling alternative to traditional chemical induction, particularly in biomanufacturing, a view that has been corroborated by several experts in the field (see: <A href="https://2018.igem.org/Team:NUS_Singapore-A/Human_Practices#iHP"><I>Integrated Human Practices</I></A>).
                </P><P>
We adopted the EL222 blue light system, thus renouncing the use of chemical inducers. Originating from the marine bacterium <I>Erythrobacter litoralis</I> HTCC2594, EL222 is a photosensitive DNA binding protein, with a N-terminal light-oxygen-voltage (LOV) domain and a C-terminal helix-turn-helix (HTH) DNA binding domain. Irradiation by blue light of wavelength 450nm exposes the hitherto sequestered HTH, facilitating dimerization and subsequent DNA binding <SUP>[22]</SUP>.
                </P><P>
                    In <I>Coup Dy'état</I>, the repressible system was employed to facilitate dynamic gene regulation (see: <I><A href="https://2018.igem.org/Team:NUS_Singapore-A/Design#CMI">Cell-Machine Interface</A></I>). With energy conservation in mind, blue light is shone during the shorter growth phase, and switched off during the lengthier production phase to derepress the expression of enzymes. This is achieved by placing the DNA binding site of EL222 between the -35 and -10 hexamers of the consensus promoter in <I>E. coli</I>, creating the blue light repressible promoter P<SUB>BLrep</SUB><SUP>[22]</SUP>. As a result, EL222 acts as a repressor, blocking the binding of RNA polymerase and repress gene expression in the presence of blue light. In the dark, RNA polymerase can now bind, and gene expression takes place (<A href="https://2018.igem.org/Team:NUS_Singapore-A/Design#DesignFig08">Figure 8</A>).
                </P><FIGURE id="DesignFig08" class="figures2" style="max-width:70%;"><FIGCAPTION style="text-align:justify"><B>Figure 8. The EL222 blue light repressible system.</B>
EL222 is expressed constitutively. In the absence of blue light, the EL222 monomer is unable to bind to the promoter, allowing RNAP to transcribe the target gene, hence the ON state. When blue light is present, EL222 dimerizes and sits on its binding site in the promoter. This blocks the binding of RNAP and represses expression of the target gene, resulting in the OFF state.</FIGCAPTION></FIGURE><FIGURE id="DesignFig09" class="figures2" style="max-width:40%;float:right;margin:15px"><FIGCAPTION style="text-align:justify"><B>Figure 9. Gene circuit for characterization of EL222 blue light promoters.</B>
RFP, the reporter, is placed under the blue light inducible promoter, P<SUB>BLind</SUB>, and repressible promoter, P<SUB>BLrep</SUB>, respectively. Different degradation tags such as YbaQ are fused to the end of the RFP gene. </FIGCAPTION></FIGURE><P>            
To elucidate the strength, sensitivity, and leakiness of P<SUB>BLrep</SUB>, and its inducible counterpart, P<SUB>BLind</SUB>, we performed many rounds of characterization experiments. This was essential in providing us with insights for its subsequent integration into bioproduction to drive target gene expression. The characterization involves the use of RFP as a reporter placed under the control of the promoters. However, the output was not representative of the ON/OFF status of the promoter as a result of the accumulation of RFP, which has a long half-life. To resolve this, we introduced degradation tags of varying strengths so that the RFP produced would be targeted for quicker proteasomal degradation (<A href="https://2018.igem.org/Team:NUS_Singapore-A/Design#DesignFig09">Figure 9</A>). This improves on our previous model for promoter characterization.
                </P><FIGURE id="DesignFig10" class="figures2" style="max-width:40%;float:right;margin:15px"><FIGCAPTION style="text-align:justify"><B>Figure 10. Blue light repressible control of enzyme expression.</B>
Both enzymes, F3’H and FNS, are placed under the control of P<SUB>BLrep</SUB>. In the presence of blue light, expression of enzymes are repressed. Switching off blue light will enable enzyme expression. </FIGCAPTION></FIGURE><P>
Through our characterization experiments, we also found P<SUB>BLrep</SUB> to be better suited for our system due to its relatively lower leakiness, further justifying our choice of a repressible system. Hence, we replaced the chemically-inducible promoters, which were originally used to control the expression of our biosynthetic genes F3’H and FNS, with P<SUB>BLrep</SUB> (<A href="https://2018.igem.org/Team:NUS_Singapore-A/Design#DesignFig10">Figure 10</A>). To verify that our blue light repressible system works as intended, we demonstrated that cells that were exposed to blue light throughout the course of biosynthesis had a significantly lower yield.
                </P><P>
Our project thus built upon the landmark study on optogenetically-regulated bioproduction in yeast <SUP>[23]</SUP> and demonstrated that the use of this technology with bacteria shows promise. We envision this to be the future of biomanufacturing.
                </P></DIV><BUTTON class="accordion-closer">CLOSE</BUTTON></DIV><BUTTON id="SRM" class="accordion">  STRESS REPORTER  </BUTTON><DIV class="panel"><DIV class="panel-inside"><P>
Stress, and the cell’s response to it, is a universal phenomenon. In large industrial bioreactors, stress is exacerbated owing to the large variation in the local environment as compared to small bench-top fermenters or shake-flask cultures. This is the result of a longer global mixing time, causing significant heterogeneity in the medium. Besides hydrodynamic shear stress, cells are also subjected to stress induced by fluctuations in the level of oxygen, pH, substrate, and other parameters <SUP>[24]</SUP>. As Dr Nic Lindley said (see: <A href="https://2018.igem.org/Team:NUS_Singapore-A/Human_Practices#iHP"><I>Integrated Human Practices</I></A>):
                </P><DIV class="quote"> Take a sample point in a bioreactor and you will get an image, take a look of the whole bioreactor and you will get a film     - Dr Nic Lindley</DIV><P>
All these compound the difficulty in translating the biological potential of the cell as demonstrated in the laboratory into actual production. One type of stress, however, applies to bench-top experiments in the laboratory and large-scale processes in the industry, and across the bioproduction of any compounds. This opens up the possibility of investigating and regulating it in bioproduction.
                </P><P>
This particular stress is that induced by the expression of recombinant proteins. The depletion of finite cellular resources during the expression of synthetic constructs constitutes an unwanted burden, hampering the growth and expected performance of engineered cells in an unpredictable manner <SUP>[25]</SUP>. Stress regulation was accomplished by decreasing the expression of a synthetic construct in response to increasing burden. Regulated cells outperformed controls in protein yield <SUP>[26]</SUP>, a discovery with significant implications for the future of biomanufacturing.
 </P><P>
Therefore, we strove to improve our yield by incorporating a stress reporter module into our system, which studies the level of stress induced by our constructs and facilitates stress regulation. The module consists of the RFP gene placed under the control of a burden-responsive promoter P<SUB>htpG1</SUB> (<A href="https://2018.igem.org/Team:NUS_Singapore-A/Design#DesignFig11">Figure 11a</A>). This particular promoter showed high promise as a biosensor for burden, as it displayed the best ON/OFF characteristics compared to other promoters that were similarly upregulated upon the induction of synthetic constructs <SUP>[26]</SUP>.
                </P><FIGURE id="DesignFig11" class="figures2" style="max-width: 85%"><FIGCAPTION style="text-align:justify"><B>Figure 11. Experimental design for investigation of the level of stress induced by recombinant protein expression. </B><B>a</B>, The stress reporter module consists of the reporter, RFP, under the control of the burden-responsive promoter P<SUB>htpG1</SUB>. The level of fluorescence detected will correspond to the stress experienced by the cell, in this case, the basal stress level and that induced by the module itself.
                <B>b</B>, The stress reporter module was co-transformed with a synthetic construct in which GFP was constitutively expressed. The higher fluorescence level detected in b as compared to a would be due to stress induced by GFP expression.                                                                 
                <B>c</B>, As with b, the investigation of the stress induced by our constructs was done by co-transforming them with the stress reporter module. The de novo plasmid (see: (<A href="https://2018.igem.org/Team:NUS_Singapore-A/Design#DesignFig05">Figure 5</A>), <A href="https://2018.igem.org/Team:NUS_Singapore-A/Design#NAR"><I>De novo Biosynthesis</I></A>) is the construct illustrated here. </FIGCAPTION></FIGURE><P>
Using this part, we were able to quantify the level of stress induced by GFP production by measuring the fluorescence level, thus demonstrating proof of concept (<A href="https://2018.igem.org/Team:NUS_Singapore-A/Design#DesignFig11">Figure 11b</A>). Our constructs were subsequently tested and shown to impose a substantial metabolic burden (<A href="https://2018.igem.org/Team:NUS_Singapore-A/Design#DesignFig11">Figure 11c</A>). Characterization of the P<SUB>htpG1</SUB> promoter will hopefully pave the way for stress regulation and the ensuing optimization of the bioproduction process.
                </P></DIV><BUTTON class="accordion-closer">CLOSE</BUTTON></DIV><BUTTON id="CMI" class="accordion"> CELL-MACHINE INTERFACE  </BUTTON><DIV class="panel"><DIV class="panel-inside"><P>
Biomanufacturing involves the introduction of synthetic constructs often with the aim of maximizing flux towards heterologous pathways, placing tremendous burden on cells,  attenuating their growth and curtailing their production (see: <A href="https://2018.igem.org/Team:NUS_Singapore-A/Design#SRM"><I>Stress Reporter</I></A>). Cells’ natural propensity to grow and multiply is thus directly in conflict with synthetic biologists’ desire to maximize their production. 
                </P><P>
However, prioritization of both growth and production could conceivably be accomplished using dynamic control. This separates fermentation into two phases: a growth phase where cells are cultivated to high densities, followed by a production phase where expression of non-native pathways takes place <SUP>[27]</SUP>. Dynamic control can be achieved through these means - chemical induction, autoregulation, and computer-assisted feedback control. Compared to the other two, computer-assisted feedback control is tunable, versatile, modular - it is easier to incorporate additional modules e.g. nutrient- and substrate-sensing modules <SUP>[28]</SUP>, can perform more complex logic, and places less metabolic burden on the cell. These advantages that comes with giving control over to the user will only become more significant as our system scales up.
                </P><P>
Computer-assisted feedback control enjoys a particularly excellent synergy with optogenetics, as light sources are easily programmed to turn on/off. However, using light to control transcription remains a challenge in itself. Even state-of-the art methods are unable to simultaneously achieve precision and repeatability, nor can they withstand changing operating culture conditions <SUP>[29]</SUP>. Furthermore, based on our literature review, custom optogenetic tools have only been designed for well plates, which are small-volume, high-throughput systems. These tools are inadequate for our project as our objective is to scale up to a large-volume, low-throughput bioreactor. 
                </P><P>
Our first design focus was then to create an <I>in silico</I> automated feedback control system, a theoretically effective and feasible strategy for overcoming current challenges in light-controlled gene expression <SUP>[30]</SUP> as compared to typical open-loop feedback control systems for gene expression <SUP>[29]</SUP>. Our second design focus was to accommodate larger-volume containers for scaled-up cell cultivation. We thus produced solutions based on these design considerations.  
                </P><H2>Hardware</H2><P>
We designed two complementary hardware systems. The first, <I>PDF-LA!</I> fulfills users’ needs to characterize their optogenetic circuit in scaled-up systems. The second, <I>Light Wait</I> enables optogenetic biomanufacturing. See our <A href="https://2018.igem.org/Team:NUS_Singapore-A/Hardware">Hardware</A> pages for our design iterations, how we built our final designs, and how we validated these designs through experiments.
                </P></DIV><BUTTON class="accordion-closer">CLOSE</BUTTON></DIV><H2>References</H2><DIV class="reference">

[1] Goodacre, D. (2018). Faking Flavours. <I>Lateral Magazine, Physical Science</I> (28). Retrieved from http://www.lateralmag.com/articles/issue-28/faking-flavours. 
[2] Geddes, C. C., Nieves, I. U., &amp; Ingram, L. O. (2011). Advances in ethanol production. <I>Current opinion in biotechnology</I>, 22(3), 312-319.
[3] World Wildlife Fund. (2015). Sugarcane Farming's Toll on the Environment. <I>World Wildlife Magazine, Summer 2015</I>. Retrieved from https://www.worldwildlife.org/magazine/issues/summer-2015/articles/sugarcane-farming-s-toll-on-the-environment.
[4] Sudiyani, Y., Styarini, D., Triwahyuni, E., Sembiring, K. C., Aristiawan, Y., Abimanyu, H., &amp; Han, M. H. (2013). Utilization of biomass waste empty fruit bunch fiber of palm oil for bioethanol production using pilot–scale unit. <I>Energy Procedia</I>, 32, 31-38.
[5] Sievert, C., Nieves, L. M., Panyon, L. A., Loeffler, T., Morris, C., Cartwright, R. A., &amp; Wang, X. (2017). Experimental evolution reveals an effective avenue to release catabolite repression via mutations in XylR. <I>Proceedings of the National Academy of Sciences</I>, 114(28), 7349-7354.
[6] Zhang, W., Liu, H., Li, X., Liu, D., Dong, X. T., Li, F. F., ... &amp; Yuan, Y. J. (2017). Production of naringenin from D‐xylose with co‐culture of E. coli and S. cerevisiae. <I>Engineering in Life Sciences</I>, 17(9), 1021-1029.
[7] Marín, L., Gutiérrez-del-Río, I., Yagüe, P., Manteca, Á., Villar, C. J., &amp; Lombó, F. (2017). De novo biosynthesis of apigenin, luteolin, and eriodictyol in the actinomycete Streptomyces albus and production improvement by feeding and spore conditioning. <I>Frontiers in microbiology</I>, 8, 921.
[8] Ganesan, V., Li, Z., Wang, X., &amp; Zhang, H. (2017). Heterologous biosynthesis of natural product naringenin by co-culture engineering. <I>Synthetic and systems biotechnology</I>, 2(3), 236-242.
[9] Yeong, C. J. (2012). Discovery and development of novel anti-microbial therapeutics (Doctoral dissertation).
[10] Grovier, K. (2018). The murky history of the colour yellow. <I>BBC</I>. Retrieved from http://www.bbc.com/culture/story/20180906-did-animal-cruelty-create-indian-yellow.
[11]  Vorkamp, K. (2016). An overlooked environmental issue? A review of the inadvertent formation of PCB-11 and other PCB congeners and their occurrence in consumer products and in the environment. <I>Science of the Total Environment</I>, 541, 1463-1476.
[12] Singh, H. B., &amp; Kumar, A. B. (2014). <I>Handbook of natural dyes and pigments</I>. Woodhead Publishing India Pvt Limited.
[13] Marín, L., Gutiérrez-del-Río, I., Yagüe, P., Manteca, Á., Villar, C. J., &amp; Lombó, F. (2017). De novo biosynthesis of apigenin, luteolin, and eriodictyol in the actinomycete Streptomyces albus and production improvement by feeding and spore conditioning. <I>Frontiers in microbiology</I>, 8, 921.
[14] Zhu, S., Wu, J., Du, G., Zhou, J., &amp; Chen, J. (2014). Efficient synthesis of eriodictyol from L-tyrosine in Escherichia coli. <I>Applied and environmental microbiology</I>, AEM-03986.
[15] Lopez, P. J., Marchand, I., Joyce, S. A., &amp; Dreyfus, M. (1999). The C‐terminal half of RNase E, which organizes the Escherichia coli degradosome, participates in mRNA degradation but not rRNA processing in vivo. <I>Molecular microbiology</I>, 33(1), 188-199.
[16] Leonard, E., Yan, Y., Fowler, Z. L., Li, Z., Lim, C. G., Lim, K. H., &amp; Koffas, M. A. (2008). Strain improvement of recombinant Escherichia coli for efficient production of plant flavonoids. Molecular pharmaceutics, 5(2), 257-265.
[17] Zuo, J., &amp; Chua, N. (2000). Chemical-inducible systems for regulated expression of plant genes. <I>Current Opinion In Biotechnology</I>, 11(2), 146-151.
[18] Dvorak, P., Chrast, L., Nikel, P. I., Fedr, R., Soucek, K., Sedlackova, M., . . . Damborsky, J. (2015). Exacerbation of substrate toxicity by IPTG in escherichia coli BL21(DE3) carrying a synthetic metabolic pathway. <I>Microbial Cell Factories</I>, 14(1), 201-201.
[19] Politi, N., Pasotti, L., Zucca, S., Casanova, M., Micoli, G., Cusella De Angelis, Maria Gabriella, &amp; Magni, P. (2014). Half-life measurements of chemical inducers for recombinant gene expression. <I>Journal of Biological Engineering</I>, 8(1), 5-5.
[20] Tischer, D., &amp; Weiner, O. D. (2014). Illuminating cell signalling with optogenetic tools. <I>Nature Reviews. Molecular Cell Biology</I>, 15(8), 551-558. 
[21] Zhang, K., &amp; Cui, B. (2015). Optogenetic control of intracellular signaling pathways. <I>Trends in biotechnology</I>, 33(2), 92-100.
[22] Jayaraman, P., Devarajan, K., Chua, T. K., Zhang, H., Gunawan, E., &amp; Poh, C. L. (2016). Blue light-mediated transcriptional activation and repression of gene expression in bacteria. <I>Nucleic acids research</I>, 44(14), 6994-7005.
[23] Zhao, E. M., Zhang, Y., Mehl, J., Park, H., Lalwani, M. A., Toettcher, J. E., &amp; Avalos, J. L. (2018). Optogenetic regulation of engineered cellular metabolism for microbial chemical production. <I>Nature</I>, 555(7698), 683.
[24] Henzler, H. J., Schügerl, K., Kretzmer, G., Kieran, P. M., MacLoughlin, P. F., Malone, D. M., ... &amp; Yim, S. S. (Eds.). (2000). <I>Influence of stress on cell growth and product formation</I> (Vol. 67). Springer Science &amp; Business Media.
[25] Gasser, B., Saloheimo, M., Rinas, U., Dragosits, M., Rodríguez-Carmona, E., Baumann, K., ... &amp; Porro, D. (2008). Protein folding and conformational stress in microbial cells producing recombinant proteins: a host comparative overview. <I>Microbial cell factories</I>, 7(1), 11.
[26] Ceroni, F., Boo, A., Furini, S., Gorochowski, T. E., Borkowski, O., Ladak, Y. N., ... &amp; Ellis, T. (2018). Burden-driven feedback control of gene expression. <I>Nature methods</I>, 15(5), 387.
[27] Lalwani, M. A., Zhao, E. M., &amp; Avalos, J. L. (2018). Current and future modalities of dynamic control in metabolic engineering. <I>Current opinion in biotechnology</I>, 52, 56-65.
[28] Lo, T. M., Chng, S. H., Teo, W. S., Cho, H. S., &amp; Chang, M. W. (2016). A two-layer gene circuit for decoupling cell growth from metabolite production. <I>Cell systems</I>, 3(2), 133-143.
[29] Milias-Argeitis, A., Rullan, M., Aoki, S. K., Buchmann, P., &amp; Khammash, M. (2016). Automated optogenetic feedback control for precise and robust regulation of gene expression and cell growth. <I>Nature communications</I>, 7, 12546.
[30] Chen, S., Harrigan, P., Heineike, B., Stewart-Ornstein, J., &amp; El-Samad, H. (2013). Building robust functionality in synthetic circuits using engineered feedback regulation. <I>Current opinion in biotechnology</I>, 24(4), 790-796.</DIV></DIV></DIV></DIV><DIV class="footer_wrapper"><DIV class="big_black_thing "><DIV class="left_side"><H3 style="color:white;"><B>  HEAR OUR STORY </B></H3><P>We’re using synthetic biology to change the way the world thinks and functions.</P><P>We’re a little disruptive, dangerous and maybe a tad too bold. But the world needs some shaking up every now and then, and we think we’re just the right people for that.</P></DIV><DIV class="right_side "><DIV class="right-top"><H3 style="color:white;">Together we’re going to engineer biology and create something <I>awesome</I>.</H3></DIV><DIV class="right_bot"><SPAN><H3 style="color:white;">CONTACT US</H3><A href="mailto:nusigem@gmail.com" style="color:#ffffff; font-family: 'Lora', serif; font-size:16px;">nusigem@gmail.com</A><P>21 Lower Kent Ridge Rd, Singapore 119077
</P></SPAN></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></BODY></HTML>