Team:SCU China/Notebook
Toggle navigation
Home
Pangu
Team
Attributions
Member
Project
Description
Carbon Fixation
Nitrogen Fixation
Testing
Verification
Modeling
Notebook
Safety
Design
Parts
Tool and Method
Protocol
Software
Activities
Human Practice
Collaboration
Achievement
Notebook
Verification Part
2015.7．25
iGEM biobricks coded 2014K116N, 2014K118B were induct to microbe and induced to express but failed.
8.1
Parts named AM6 and LR were supposed to be connected to express proteins but we didn't succeed.
Connect cm1，2，7，9 with LR respectively.
8.10
Cm1 and cm9 has been connected with LR successfully.
The clone PCR of LR+cm1
8.20
Check the expression of Cm1+LR by metering the OD588/OD400
Explore the effluence to the promoter of time gradient and concentration gradient.
8.28
Time gradient and concentration gradient were designed to induce promoter and compare the results after centrifugation.
Testing Part
8.15-8.24
The optimal temperature for E1-PCR reaction was proposed by gradient PCR ranging from 50&#8451-65&#8451. The temperature gradually drops down from A to H.
The optimal temperature for E2-PCR reaction was also proposed by gradient PCR ranging from 50&#8451-65&#8451. The temperature gradually drops down from 1 to 8.
Result of E1-PCR and E2-PCR. The length of E1 and E2 are 1718 bp and 1878 bp respectively.
The optimal temperature for E2-PCR reaction was also proposed by gradient PCR ranging from 50&#8451-65&#8451. The temperature gradually drops down from left to right.
8.25-8.31  cas9 gel extraction
Cas9 was digested with X and P sites. Plasmids containing promoter parts was digested with S-P sites.
There is no positive clone.
9．1-9.4
The connection of A backbone /cas9/promoter has failed. There is no positive clone.
Check if the cas9 was add restriction sites. We supposed that cas9 has been add restriction sites from the result of electrophoresis.
9.5-9.14 cas9-PCR with new-designed primer cas91 and cas92
The product after gel extraction and X-P digestion was connected with promoter after S-P digestion. But there is no positive clone.
Quantify cas9 after gel extraction
Quantify cas9 after X-P digestion.
There is no positive clone.
9.14-9.17
Do electrotransformations instead of ligation.
Carbon Fixation Part
7.1-7.20
Checking reference
By Shengze xie, Yiwen Zhao
7.20-8.4
pre- experiment: How Ni2+ and anaerobic environment influence E.coli
By Yulong Tang, Zhilong Zhang, Zekun Lyu
8.5-9.1
synthesis the gene of 3S2X.Then run PCR to amplification,the picture below is the results of annealing temperature testing.
By Zhilong Zhang, Zekun Lyu
Assemble 3S2X expression vector.
By Yiwen Zhao, Haoyu Wang
Prepare expression vector of CODH/ACS.Assemble promoter, RBS, terminator and BamHI, HindIII.
By Yulong Tang, Haoyu Wang, Yiwen Zhao
9.2-9.17
Synthesis the six gene of our group. The gene we synthesis from IDT result as below.
PCR to add BamHI, HindIII onto both sides of the genes.
By Yulong Tang, Haoyu Wang
Assemble them onto expression vector above.(haven't done)
Assemble them onto pET28b. Expression.
By Haoyu Wang
Nitrogen Fixation Part
2015.7.28-2015.8.9, Nannan Xie, Zhisheng Liang, Shiji Zhao
We separated Azotobacteria from soil and tried to get nif cluster,but failed.
8.11-8.16,
We prepared the reporter plsmid for latter use.
We separated Paenibacillus from soil by boiled for 5min and 10min, trie2015.7.28-2015.8.9, Nannan Xie, Zhisheng Liang, Shiji Zhao
We separated Azotobacteria from soil and tried to get nif cluster,but failed.
8.11-8.16,
We prepared the reporter plsmid for latter use.
We separated Paenibacillus from soil by boiled for 5min and 10min, tried to get nif cluster,but failed.
Fig.1.(A)Electrophoresis of reporter plasmids.
(B)Failed PCR for attempt to get the cluster(10.5kb)(Lon EZTM polymerase was used).
8.17-8.20, Nan Wang, Nannan Xie
We analysed the genomes of Paenibacillus sp. WLY78, sliced it. After getting each single gene, we promoted their code sequences and sent to companies for synthesis.
ACCC strain we purchased were still not prepared, we had to be pushed due to limited time.
8.21-8.24
Waiting for materials. I did a PCR of hesA from separated bacterial, bands were clear, but that gene isn't a specific to species. Identify of that bacterial is not easy.
Fig.2.PCR of hesA in separated strain.
8.25-8.29
Pnif arrived, We put in on a pSB1C3 backbone and ligated it with different reporters.
8.30-9.4
Synthesized nifX,nifV,hesA arrived. We amplified these three gene and PCR to add overlap for Gibson assembly. Results of PCR weren't stable, so we did a gradient to meet the requirements.
Fig3.PCR of nifXV,hesA
Fig4.gradient PCR.
9.5-9.9
We amplified nifD,nifK,nifE and nifN, and PCR to add overlap for Gibson assembly.
9.8-9.16
We finally got the ACCC0430 strain though there were lots of difficulties during special days. It multiplied slowly, we tried to get the cluster, but nothing was detect from electrophoresis even primer dimer.
9.8-9.13
PCR of several genes always failed. We tried to find why.
We applied gel extraction on PCR products of nine genes, as primer dimmers are more than 100bp. With exonuclease I ,lamda exonuclease and DNA polymerase, we tried to do Gibson assembly, but no result was shown.
9.12-9.18.
We tried hard to put nifH on a pSB1C3 plasmid. As well, we ligated nifH on pET28b to see protein expression.
Specially, Haoyu Wang did the quantitive test of Pnif:
9/10: Transformation and culture (Plasmid from company, GFP, amilCP and Pd)
9/11: Plasmid extraction, cut and ligation (Plasmid from company, GFP, amilCP and Pd)
9/12:Transformation and culture
9/13:Monoclonal bacteria culture (Pnif + GFP, Pnif + amilCP, Pd + GFP)
9/14: Make glycerol stocked bacteria
9/15:Bacteria culture
9/16:Enzyme-labeled instrument test
9/17:Flow cytometry test
Fig5.Pnif+GFP, single bacteria under microscope.
Fig.6.Comparison of Pnif and Pd. Pd: PromoterBBa_J23100; Pnif: nif promoter.
Navigation
Top
Verification Part
Testing Part
Carbon Fixation Part
Nitrogen Fixation Part
