Team:SCU China/NitrogenFixation
Toggle navigation
Home
Pangu
Team
Attributions
Member
Project
Description
Carbon Fixation
Nitrogen Fixation
Testing
Verification
Modeling
Notebook
Safety
Design
Parts
Tool and Method
Protocol
Software
Activities
Human Practice
Collaboration
Achievement
NitrogenFixation
Introduction
Biological nitrogen fixation is the process in which azotobacters convert atmospheric nitrogen into ammonia with nitrogenases. The overall reaction for biological nitrogen fixation is: N2+8H++16 ATP+8e-→2NH3+H2+16ADP+16Pi. In common azotobacters, the generated ammonia is assimilated into glutamine through other metabolic pathways.
The nitrogen fixation genome contains a large number of modules, or clusters. However, the components of nif (nitrogen fixation) genes required to enable nitrogen fixation varies significantly among different nitrogen fixation species. It is noticeable that not all the modules are necessary for the growth of certain species.
In our project, the nitrogen fixation genes of diazotroph Paenibacillus sp. WLY78 was chosen. According to the research of prof. Liying Wang, the minimal nitrogen fixation genome consists of 9 necessary genes, which are arranged within a 10.5 kb region in the order nifB, nifH, nifD, nifK, nifE, nifN, nifX, hesA and nifV. In addition, all the gene clusters are under the control of a σ70-dependent promoter located upstream of nifB. [1]
We aimed to standardize all genes of this minimal nitrogen fixation genome, including the promoter, and express the genome in E.coli to make our nitrogen fixation E.pangu.
Fig. 1. Minimal nitrogen fixation genome of Paenibacillus sp. WLY78
The Functions of each gene[1][2][3]:
nifH: encoding dinitrogenase reductase, also called Fe protein. The protein is a homodimer bridged by an intersubunit [4Fe-4S] cluster that acts as an obligate electron donor to the MoFe protein.
nifD/nifK: α/β subunits of dinitrogenase, also called FeMo protein. nifD and nifK encode two metalloclusters: FeMo-co, a [Mo-7Fe-9S-C-homocitrate] cluster which serves as the active site of substrate binding and reduction and the P-cluster, a [8Fe-7S] cluster which shuttles electrons to FeMo-co.
nifE/nifN: encode FeMo-cofactor assembly protein.
nifB: essential for biosynthesis of the active-site nitrogenase cofactor and encodes a radical S-adenosylmethionine(SAM)-dependent enzyme that inserts the central carbon atom into the eight-Fe core of nifB cofactor(nifB-co).
nifV: homocitrate synthase, provides the organic moiety in all three nitrogenase cofactors.[2]
hesA: function has not so far been determined , but deletion analysis indicates that in addition to the core nif genes, hesA plays an important role in nitrogen fixation and is responsive to the availability of molybdenum. HesA is a member of the ThiF-MoeB-HesA family and contains an N-terminal nucleotide binding domain and a C-terminal MoeZ/MoeB-like domain. The ThiF-MoeB-HesA family engage in an ATP-dependent process that activates the C-terminus of partner ubiquitin-like proteins by forming an acyl adenylate complex that facilitates sulfur transfer.
nifX: encodes carrier proteins involved in the delivery of FeMoco to the MoFe protein , participate in FeMoco assembly in a chaperone-like function.[3]
The structure of this system is showed below (Figure 2).[4]
Fig. 2. Crystal structure of the Fe and MoFe protein components of Mo-dependent nitrogenase showing the nucleotides, metal clusters, and electron transfer pathways.(Left) Cartoon representation of MoFe protein (pdb code: 1M1N) with the α-subunits and the β-subunits and Fe protein (pdb code: 1FP6) with the γ-subunits. (Right) Structures of MgADP and the three metalloclusters of nitrogenase. The figure was generated using the computer program PyMol.
Results
Azotobacter Obtaining
The first problem we needed to address is how to obtain required Azotobacter. We tried to separate azotobacter from soil by ourselves, but we failed. Then we turned to obtain it by two ways: Synthesizing nif genes de novo and purchasing bacteria from Agricultural Culture Collection of China (ACCC).
Synthesized nif Genes
Here is the flowchart: Obtaining sequences → sequences optimization → synthesizing → standardization & gibson assembly → protein expression → confirmation of function.
According to the research by Wang L ,et al, the whole genome sequence of Paenibacillus sp. WLY78 is available. We also applied protein alignment on NCBI to find conservative areas and get the exact sequence of each gene, and then we optimized each sequence to make them more compatible with E.coli. Our nif genes and nif promoter were synthesized by SynbioTM and TsingkeTM. The standard prefix (5' GAATTCGCGGCCGCTTCTAGAG '3) and suffix (5' TACTAGTAGCGGCCGCTGCAG '3) were added to the sequences.
(1)Confirmation of nif promoter
We standardized the nif promoter of Paenibacillus sp. WLY78, and confirmed the reliability of this promoter by 3 ways. There are 4 reporter genes used in this part. The gene circuit (Fig. 3) and results are shown below:
Fig. 3. The gene circuit used in Pnif conformation.
Ⅰ Visible conformation
Pnif was linked to the upstream of 3 different reporter genes (RFP reporter: BBa_K1357010, YFP reporter: BBa_E0430, amilCP reporter: BBa_K1357009) to confirm the reliability of this promoter. The results are shown in Figure 4.
Fig. 4. Confirmation of nif promoter of Paenibacillus sp. WLY78. Left, Pnif+RFP reporter; Middle, Pnif+YFP reporter; Right, Pnif+amilCP reporter.
Ⅱ Flow cytometry test
We also tested Pnif expression with flow cytometry. A GFP biobrick part BBa_E0240 was linked downstream of this promoter, then we tested both Pnif sample and a blank control sample simultaneously. The GFP expression by Pnif was clearly shown in the result (Fig. 5).
Fig. 5. The result of flow cytometry test. All the data of blank control was shown in the left, and the data of CFP expression was shown in the right.
Ⅲ Quantitative test
For more accurate conformation, we tested the fluorescent with Multiskan Mk3 Reader. We compared the GFP (BBa_E0240) expression under constitutive promoter BBa_J23100 and Pnif, The result shows that the Pnif is a relatively weak promoter compared with BBa_J23100 for ordinary genes, and the expression will take approximately 5h to reached the plateau (Fig. 6).
Fig. 6. Comparison of the expression efficiency of Pnif and Pd.Pd: Promoter BBa_J23100; Pnif: nif promoter.
All the testing results confirmed the reliability of Pnif. The initiation of Pnif does not require inducement, but its expression efficiency is relatively weak. (The published data shows that the initiation efficiency of promoter BBa_J23100 is 67, and that of Pnif is 13) Thus it is a weak constitutive promoter and we think it is perfect to express harmful proteins which may slightly affect the host cells.
（2）Confirmation of single gene
We inserted each gene to pET-28b plasmid to see if the optimized genes will work. The following figure (Fig. 7) shows the result of colony PCR of nifH-pET-28b.
Fig.7. Electrophoresis of nifH-pET-28b (LXH) by colony PCR.The upper band of LXH is about 1300bp, as nifH is about 900bp and T7 promoters will add 160bp extra sequence, the result was acceptable. But the quantity is too low to be detected in digestion group. If aimed proteins were expressed successfully, we can detect it by SDS-PAGE. Gene expressions shall be verified respectively.
Gibson assembly
After conformation of each single gene, we will connect each gene to form the whole minimal nitrogen fixation genome. We designed our primers for Gibson assembly through NEBuilder, an online tool of NEB. Rendering is shown in figure 8, primers are shown in figure 9.
Fig.8. Fragment arrangement.The rendering of plasmid totally assembled with nine genes and a promoter.
Fig.9. Primers of Gibson Assembly.Totally we used 9 pairs for Gibson assembly.
Since we had no experience of Gibson assembly, first we tried to ligate two genes by overlapping, and we made it. Results of ligation of nifDK, nifKE and nifEN are shown in Figure 10.
Fig.10.Two genes ligation by overlapping.The upper banes are the right ligations of two genes, longer than 3000bp, and other bands are single gene fragments.
Then, we tried to arrange nine genes together in one reaction. The PCR program is as follows:
PCR program:
Step
Temperature
Time
Cycles
Initial Denaturation
98 °C
2min
1
Denaturation
98 °C
10s
35
Annealing
Average Tm
15s
Elongation
72 °C
5-15s/kb
Final Elongation
72 °C
8min
1
Final hold
4 °C
forever
Unfortunately, our first attempt failed. As we had no enough time to finish this, we hope we can finish this in our future work.
Purchased bacteria strain
We purchased azotobacter strain ACCC03043 from ACCC, which is the same species as Paenibacillus sp. WLY78. The strain was cultured on nitrogen-free medium under 30oC. Strain pattern is show in Figure 11. PCR and electrophoresis showed no result. Point mutant is needed because EcoRI and PstI restriction sites are in the original cluster sequence. As the time is not enough, we will finish this in our future work.
Fig.11.Strain pattern of ACCC03043.
Reference:
[1] Wang L, Liu Z, Zhao D, Liu X, et al. A Minimal Nitrogen Fixation Gene Cluster from Paenibacillus sp. WLY78 Enables Expression of Active Nitrogenase in Escherichia coli. PLoS Genet, 2013, 9:e1003865.
[2] Yang J, Xie X, et al. Reconstruction and minimal gene requirements for the alternative iron-only nitrogenase in Escherichia coli, PNAS, 2014, E3718-E3725.
[3] Hernandez JA, Igarashi RY, Soboh B, et al. nifX and nifEN exchange nifB cofactor and the VK-cluster, a newly isolated intermediate of the iron-molybdenum cofactor biosynthetic pathway. Mol Microbiol, 2007, 63:177-192.
[4]Georgiadis MM, Komiya H, Chakrabarti P, et al. Crystallographic structure of the nitrogenase iron protein from Azotobacter vinelandii. Science, 1992, 257: 1653-1659.
Navigation
Top
1. Introduction
2. Results
Reference
