Team:SCU China/Test
Toggle navigation
Home
Pangu
Team
Attributions
Member
Project
Description
Carbon Fixation
Nitrogen Fixation
Testing
Verification
Modeling
Notebook
Safety
Design
Parts
Tool and Method
Protocol
Software
Activities
Human Practice
Collaboration
Achievement
Testing
Introduction
The mean function of carbon fixation E. pangu is transforming inorganic substances into organic substances. Unfortunately, the process will definitely be affected by their natural ability of utilizing glucose to serve energy to E. pangu themselves. Thus, our project includes a part in which we will knockout the genes of glycolysis to verify the functions of carbon fixation E. pangu . The gene circuit for this part is shown in figure 1:
Fig. 1. Gene circuit of carbon fixation testing
Here we introduced a gene editing technique called CRISPR Cas9 to knock out glycolysis of E. pangu . CRISPR is widely used to edit genome based on the complexity of sgRNA and Cas9 protein. Unlike other gene edition technique, CRISPR can be designed in one dimension instead of stereoscopy. Thus, when targeting different sites, we just need to design corresponding sequences. After knocking out several key enzymes, we will be able to regulate the metabolism of E. pangu , so that the ability of carbon fixation of our engineered microbe can be qualified.
At the initial stage of bacterial growth, the energy requirement is relatively high. To ensure the normal growth at this stage, we added a regulated promoter before Cas9 protein, through which the glucose catabolism of our bacteria can be regulated. That is, we will not knock out the glycolysis system until the cell growth has reached stationary phase.
Result
1. pRhl Insight
Before we got started, we tested a standard biobrick BBa_K57503 (iGEM11_Northwestern) to find out the optimal condition for the initiation efficiency of promoter pRhl:
Fig. 2. Components of biobrick BBa_K57503
This part consists of a promoter pRhl and the gene of required binding protein, as well as a gene of GFP (E0040). We transformed this part into E.coli BL21 (TaKarRaTM), and added C4-HSL (Final Concentration 100uM) to induce the expression. After incubated in shaker (200rpm) at room temperature for 4h, we got the result as follows under UV light.
Fig. 3. pRhl testing
In figure 3, tube 1 is blank control without bacteria, and tube 2-7 are parallel test groups in which 100uM rhl was added. Obvious green florescence can be seen in each tube of test group. So we got the conclusion that promoter pRhl can work well under this condition.
2. Cas9 Protein
We obtained the plasmid Psuzm3a-Cas9 and primer pair 1 for Cas9 gene from Prof. Yizheng Zhang of college of life science, Sichuan University. Restriction sites Hind III, Xba I are added to forward primer and Spe I, Pst I are added to reverse primer. We didn’t add EcoR I site since there is such a site in the middle of sequence of Cas9. The primer sequence and PCR results (Fig. 4) are shown as below:
Primer pair 1: F  GTTTTCTAGAAAGCTTATGGATAAGAAATACT
                     R  CTTCTGCAGACTAGTATCAGTCACCTCCTAG
Fig. 4. PCR Result of Cas 9 protein
The result of electrophoresis shows we successfully got desired gene of Cas9 protein.
3.sgRNA
Since purchasing plasmids containing sgRNA is time consuming, we designed it by ourselves. The new biobricks(K1796203) is shown as below:
Fig. 5. Gene circuit of biobrick(K1796203)
The sequences of crRNA and tracrRNA came from the biobrick uniCAS RNAimer (BBa_K1150034), submitted by 2013 iGEM team Freiburg.
(1) Modifications on Promoter and Terminator
The constitutive promoter and the terminator are based on biobricks BBa_J23100 and BBa_B0012, respectively. To make these two parts more compatible with our sgRNA, we made some modifications on them. For the promoter, we changed -10 box into TATAAT, aiming at higher efficiency of transcription. What’s more, only the core domain of terminator B0012 which can form hairpin structure and oligo U were chosen. We have to make the part a short one to avoid undesired situations, for example, the sgRNA is too long to combine with Cas9 protein.
(2) Empty Part with BbsI Site Construction
According to the iGEM team of Freiburg University 2013 project, the sequence of crRNA and tracrRNA are shown as below:
crRNA: gttttagagctatgctgttttgaatggtcccaaaacgggt (repeat1) cttcgagaagac (cutting site) gttttagagctatgctgttttgaatggtcccaaaactttttctagcgc ( repeat2)
tracrRNA :ttggaaccattcaaaacagcatagcaagttaaaataaggctagtccgttatcaacttgaaaaag
tggcaccgagtcggtgc (core domain) ttttttttggc
To construct a universal part, we referred to the 2013 project of Freiburg iGEM team and added a BbsI restriction site to the sequence of crRNA by which any required target sequence can be inserted. sgRNA is the recombination of crRNA and tracrRNA. In reference of other articles, the sequence of sgRNA is shown as below:
sgRNA: ca
tttagagcta (part of repeat2) gaaa (a linker between crRNA and tracrRNA) tagcaagttaaaataaggctagtccgttatcaacttgaaaa
agtggca ccgagtcggtgc (main trunk of tracrRNA) tttttt (oligo U assist to terminate transcrption)
To standardize this sgRNA, we added prefix and suffix as well as modified promoter BBa_j23100 and terminator BBa_B0012.
The final sequence of biobrick part (K1796203) are shown as follows: gaattcgcggccgcttctagagttgacggctagctcagtcctaggtataatgctagcc
aaaacgggtcttcgagaagacgttttagagctagaaatagcaagttaaaataaggctagtccgttatcaactt
gaaaaagtggcaccgagtcggtgcttttttggctcaccttcgggtgggccttttactagtagcggccgctgcag
(3) Sequence Targeting
Considered that E. pangu will produce acetyl-CoA when taking CO2 as substrate, we chose pyruvate dehydrogenase (PDH) complex as knockout target in case that the natural process of acetyl-CoA producing will affect the reliability of our project. After PDH knockout, if the microbe are still alive, it is demonstrated that they have gained the ability to fix CO2 like autotrophs. Because without PDH, the normal metabolic pathway from pyruvate to acetyl-CoA is cut off, the prokaryotic cell can only utilize the acetyl-CoA comes from carbon fixation to finish citric acid cycle. However, it’s quite difficult to regulate metabolism in microbe since there are huge amounts of pathways related with each other. After searching for papers about metabolism, we found that we can not exactly foretell any influence of PDH knockout before hands on experiments.
In order to knock out PDH efficiently, we targeted two conserved domains of this complex: the 5’ terminal of subunit E1 (oligo1)(Fig. 6), and a crucial motif DHRXXDG of subunit E2 (oligo76) which plays a key role in constructing the complex. We also chose oligo64 randomly as control to appraise the efficiency of targeting. The target oligos are designed by 2013 iGEM team Freiburg.
Fig. 6. Structure of the pyruvate dehydrogenase complex E1 component  from Escherichia coli at 1.85 A resolution.
4. PDH Knockout
According to our project, we will detect if we have successfully knocked out the gene of subunits E1 and E2 by comparing the length of the PCR results of pre-knockout bacteria and after-knockout bacteria. First we found out the proper annealing temperature for E1 and E2 PCR (64 oC). The E1 and E2 PCR results of pre-knockout bacteria are shown as follows:
FFig. 7. E1 and E2 PCR results of pre-knockout bacteria. the length of E1 and E2 are 1718bp and 1878bp, respectively.
Unfortunately, the gene circuit for Cas9 expression was not constructed successfully. As a consequence, we didn’t get the PCR result of after-knockout bacteria. We hope we could finish this in our future work.
Navigation
Top
Introduction
Result
