<HTML xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en" dir="ltr">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki  ltr ns-0 ns-subject page-Team_Stanford_Research"><DIV id="globalWrapper"><DIV id="top-section"><DIV id="p-logo"><A href="/Main_Page" title="Main Page">&quot;
	    </A></DIV><DIV id="menubar" class="left-menu noprint"><UL><LI class="selected"><A href="/Team:Stanford/Research">Page               </A></LI><LI class="new"><A href="/wiki/index.php?title=Talk:Team:Stanford/Research&amp;action=edit&amp;redlink=1">Discussion               </A></LI><LI><A href="/wiki/index.php?title=Team:Stanford/Research&amp;action=edit">View source               </A></LI><LI><A href="/wiki/index.php?title=Team:Stanford/Research&amp;action=history">History               </A></LI><LI style="color:#808080;cursor:default">teams</LI></UL></DIV><DIV class="right-menu noprint" id="menubar"><UL><LI id="pt-login"><A href="/wiki/index.php?title=Special:UserLogin&amp;returnto=Team:Stanford/Research" title="You are encouraged to log in; however, it is not mandatory [o]" accesskey="o">Log in</A></LI></UL></DIV><DIV id="search-controls" class="noprint"><FORM action="/Special:Search" id="searchform">Â </FORM></DIV></DIV><DIV id="content"><H1 class="firstHeading">Team:Stanford/Research</H1><DIV id="bodyContent"><H3 id="siteSub" class="noprint">From 2010.igem.org</H3><P><CENTER><TABLE class="menu"><TBODY><TR><TD class="cell"><A class="menu" href="https://2010.igem.org/Team:Stanford">Home</A></TD><TD class="cell"><A class="menu" href="https://2010.igem.org/Team:Stanford/Research">Project</A></TD><TD class="cell"><A class="menu" href="https://2010.igem.org/Team:Stanford/Applications">Applications</A></TD><TD class="cell"><A class="menu" href="https://2010.igem.org/Team:Stanford/Research/Modeling">Modeling</A></TD><TD class="cell"><A class="menu" href="https://2010.igem.org/Team:Stanford/Parts">Parts</A></TD><TD class="cell"><A class="menu" href="https://2010.igem.org/IGEM:Stanford/2010/Team/Undergraduates">Team</A></TD><TD class="cell"><A class="menu" href="https://2010.igem.org/Team:Stanford/Notebook/Timelines">Notebook</A></TD><TD class="cell"><A class="menu" href="https://2010.igem.org/Team:Stanford/Twitter">Twitter</A></TD></TR></TBODY></TABLE></CENTER></P><CENTER><DIV style="width:850px;text-align:justify"><DIV class="contents"><TABLE id="toc" class="toc"><TBODY><TR><TD><DIV id="toctitle"><H2>Contents</H2></DIV><UL><LI class="toclevel-1 tocsection-1"><A href="#Why_Ratios.3F"><SPAN class="tocnumber">1</SPAN><SPAN class="toctext">Why Ratios?</SPAN></A></LI><LI class="toclevel-1 tocsection-2"><A href="#Our_Project:_Two_Designs_for_Ratio_Detection"><SPAN class="tocnumber">2</SPAN><SPAN class="toctext">Our Project: Two Designs for Ratio Detection</SPAN></A></LI><LI class="toclevel-1 tocsection-3"><A href="#Our_First_Sensor:_The_sRNA_Sensor"><SPAN class="tocnumber">3</SPAN><SPAN class="toctext">Our First Sensor: The sRNA Sensor</SPAN></A></LI><LI class="toclevel-2 tocsection-4"><A href="#Device_Properties"><SPAN class="tocnumber">3.1</SPAN><SPAN class="toctext">Device Properties</SPAN></A></LI><LI class="toclevel-2 tocsection-5"><A href="#Overview"><SPAN class="tocnumber">3.2</SPAN><SPAN class="toctext">Overview</SPAN></A></LI><LI class="toclevel-2 tocsection-6"><A href="#The_RSID:_Improving_sRNA_Interference"><SPAN class="tocnumber">3.3</SPAN><SPAN class="toctext">The RSID: Improving sRNA Interference</SPAN></A></LI><LI class="toclevel-2 tocsection-7"><A href="#Device_and_Parts_Level_Diagrams"><SPAN class="tocnumber">3.4</SPAN><SPAN class="toctext">Device and Parts Level Diagrams</SPAN></A></LI><LI class="toclevel-2 tocsection-8"><A href="#Flow_Cytometry_Data"><SPAN class="tocnumber">3.5</SPAN><SPAN class="toctext">Flow Cytometry Data</SPAN></A></LI></UL><LI class="toclevel-1 tocsection-9"><A href="#The_Second_Design:_Kinase_Phosphatase_System"><SPAN class="tocnumber">4</SPAN><SPAN class="toctext">The Second Design: Kinase Phosphatase System</SPAN></A></LI><LI class="toclevel-2 tocsection-10"><A href="#Device_Properties_2"><SPAN class="tocnumber">4.1</SPAN><SPAN class="toctext">Device Properties</SPAN></A></LI><LI class="toclevel-2 tocsection-11"><A href="#Overview_2"><SPAN class="tocnumber">4.2</SPAN><SPAN class="toctext">Overview</SPAN></A></LI><LI class="toclevel-2 tocsection-12"><A href="#Device_and_Parts_Level_Diagrams_2"><SPAN class="tocnumber">4.3</SPAN><SPAN class="toctext">Device and Parts Level Diagrams</SPAN></A></LI><LI class="toclevel-1 tocsection-13"><A href="#Ratios:_Better_than_Absolutes"><SPAN class="tocnumber">5</SPAN><SPAN class="toctext">Ratios: Better than Absolutes</SPAN></A></LI><LI class="toclevel-2 tocsection-14"><A href="#Project_Goals"><SPAN class="tocnumber">5.1</SPAN><SPAN class="toctext">Project Goals</SPAN></A></LI><LI class="toclevel-3 tocsection-15"><A href="#Digital_Sensor"><SPAN class="tocnumber">5.1.1</SPAN><SPAN class="toctext">Digital Sensor</SPAN></A></LI><LI class="toclevel-3 tocsection-16"><A href="#Analog_Sensor"><SPAN class="tocnumber">5.1.2</SPAN><SPAN class="toctext">Analog Sensor</SPAN></A></LI><LI class="toclevel-3 tocsection-17"><A href="#Both_Sensors"><SPAN class="tocnumber">5.1.3</SPAN><SPAN class="toctext">Both Sensors</SPAN></A></LI><LI class="toclevel-2 tocsection-18"><A href="#Important_Parameters"><SPAN class="tocnumber">5.2</SPAN><SPAN class="toctext">Important Parameters</SPAN></A></LI><LI class="toclevel-3 tocsection-19"><A href="#Digital_Sensor_2"><SPAN class="tocnumber">5.2.1</SPAN><SPAN class="toctext">Digital Sensor</SPAN></A></LI><LI class="toclevel-3 tocsection-20"><A href="#Analog_Sensor_2"><SPAN class="tocnumber">5.2.2</SPAN><SPAN class="toctext">Analog Sensor</SPAN></A></LI><LI class="toclevel-1 tocsection-21"><A href="#Biosafety"><SPAN class="tocnumber">6</SPAN><SPAN class="toctext">Biosafety</SPAN></A></LI><LI class="toclevel-1 tocsection-22"><A href="#Research"><SPAN class="tocnumber">7</SPAN><SPAN class="toctext">Research</SPAN></A></LI><LI class="toclevel-1 tocsection-23"><A href="#Medal_Requirements"><SPAN class="tocnumber">8</SPAN><SPAN class="toctext">Medal Requirements</SPAN></A></LI><LI class="toclevel-2 tocsection-24"><A href="#Bronze_Medal"><SPAN class="tocnumber">8.1</SPAN><SPAN class="toctext">Bronze Medal</SPAN></A></LI><LI class="toclevel-2 tocsection-25"><A href="#Silver_Medal"><SPAN class="tocnumber">8.2</SPAN><SPAN class="toctext">Silver Medal</SPAN></A></LI><LI class="toclevel-2 tocsection-26"><A href="#Gold_Medal"><SPAN class="tocnumber">8.3</SPAN><SPAN class="toctext">Gold Medal</SPAN></A></LI></TD></TR></TBODY></TABLE><H2><SPAN class="mw-headline" id="Why_Ratios.3F">Why Ratios?</SPAN></H2><P><B>Ratios rule the biological world, and controlling them will unlock a vast range of applications for synthetic biology.</B>In electrical and mechanical engineering, computers can be used to calculate a ratio from precise measurements, but biological engineering doesn't have that luxury. We need a hardwired device to compute and act on ratios independent of a controlling computer or human researcher.
</P><P>The Stanford team's ratio sensors use biological machinery to eliminate the human component from ratio computation. Our output system is more efficient than two individual sensors and, most importantly, can pass its results on to other parts and devices. A ratio sensor performs a calculation that a user doesn't need to/won't be able to.  A ratio sensor frees up a wavelength channel on the measurement device, and, more importantly for the future, allows the computed ratio to be passed on to another part. A useful ratio sensor that could be utilized in a broad regime of biological applications needs to be operable and accurate in micro-environments; the Stanford team's ratio sensor is engineered to function within E. Coli, which is fully operable over a range of environments.
</P><H2><SPAN class="mw-headline" id="Our_Project:_Two_Designs_for_Ratio_Detection">Our Project: Two Designs for Ratio Detection</SPAN></H2><P>Our team decided to pursue two different systems for detecting ratios. After considering the scenarios in which biological engineers would want to use ratio sensors, having two specialized sensors seemed to allow greater flexibility of usage than a single design.
</P><P>While both systems receive two input signals, the output they give is different, allowing them be applied in different situations. Here's a brief rundown of the two designs (you can see more by clicking through to the individual project pages). <B>Both sensors are: (1) <I>Modular</I>: input and output molecules can be changed without affecting the interior mechanism of the device; and (2) <I>Orthogonal</I>: device mechanisms are not found in E. coli, avoid crosstalk with host cell</B></P><H2><SPAN class="mw-headline" id="Our_First_Sensor:_The_sRNA_Sensor">Our First Sensor: The sRNA Sensor</SPAN></H2><H4><SPAN class="mw-headline" id="Device_Properties">Device Properties</SPAN></H4><P><B>Method</B>: Small RNA Interference
</P><P><B>Output</B>: One of two possible proteins, depending on whether the ratio of input chemicals lies above or below a predetermined threshold
</P><P><B>Useful</B>: For reporting on a situation with a boolean output: disease detection, preterm labor warning, cancer metastasis warning
</P><P><B>Data</B>: Initial data gathered using FACS on part K353002, along with a more detailed description of our sRNA interference design can be found below.
</P><H4><SPAN class="mw-headline" id="Overview">Overview</SPAN></H4><P>Our first sensor design is tuned to a <B>single ratio</B> of input chemicals and tells the user if the input chemicals are present in relative concentrations <B>below</B>, <B>at</B>, or <B>above</B> that ratio.
</P><P>L-arabinose binds to pBAD (BBa_I0500) which promotes the transcription of sRNA2, RSID1, and GFP. AHL binds to pLUX (BBa_F2620), which promotes the transcription of sRNA1, RSID2, and RFP. In the cytoplasm, each sRNA binds to its respective RSID before translation occurs. Ribosomes are prevented from binding to those mRNA's that are bound by sRNA's, meaning that only those mRNA's that remain unbound by sRNA's get translated into reporter protein. sRNA/mRNA dimers are eventually degraded by the cell's machinery. 
</P><P>If the concentration of L-arabinose is greater than that of AHL, more sRNA2 and GFP will be produced than sRNA1 and RFP. This means that each sRNA1 will be able to bind to a copy of RSID1, while there will not be enough sRNA2 to inhibit all of the RFP. An excess of L-arabinose therefore leads to an expression of GFP.
</P><P>To change the threshold ratio detected by the sensor, multiple copies of the genes encoding either the mRNA or the sRNA can be placed downstream of the promoters.
</P><P>By sensing only one ratio, we gain a sharp transition between no signal and a lot of signal.  This allows us to insulate our output from biological noise in all regimes of the input domain except for the small neighborhood surrounding the sharp transition.  
</P><P>By engineering readout redundancy into the sRNA sensor, we have insulated our sensor from detector-specific readout noise.  If there were only one color, we would register the transition from off -&gt; on whenever the signal exceeded the signal/noise ratio of our detector.  This means that we would actually read the off/on transition as occurring at different points in the input domain with different detectors!  But with two colors, (one transitioning from off -&gt; on, the other from on -&gt; off), we can set the detection threshold far above our detector's S/N threshold, and average the transition point in the two channels to achieve a detector-independent measurement of the specified ratio.
</P><P>You can read about our other sensor <A href="/Team:Stanford/Analog_Sensor" title="Team:Stanford/Analog Sensor">here</A> .
</P><H4><SPAN class="mw-headline" id="The_RSID:_Improving_sRNA_Interference">The RSID: Improving sRNA Interference</SPAN></H4><P>Our system uses sRNA interference, which is usually a non-modular operation. Because each sRNA must be complementary to the mRNA sequence of the protein it inactivates, changing outputs in a traditional sRNA interference system would require designing and inserting a new sRNA for each output. To overcome this design flaw and make our sensor more easily modified, we implemented a novel control system for sRNA interference: the RSID.
</P><P>The RSID is a synthetically-created DNA sequence that includes a ribosome binding site, and is inserted upstream of the output proteins. Each sRNA recognizes and binds to a specific RSID, covering the RBS contained within it and preventing the translation of the downstream output protein.
</P><P>Because the RSID's are independent from the output protein, new output proteins may be inserted without affecting the performance or design of the internal mechanism. This feature allows our system to be modified by future users to release any necessary output protein.
</P><P>In choosing the sequence of our RSID's and sRNA's, we were careful to avoid several possible pitfalls. Our sRNA's and RSID's are all non-complementary to anything in the host cell's genome in order to avoid accidentally inhibiting important cell processes. In addition, we screened our designs to make sure they had low levels of self-dimerization, which would limit their effectiveness.
</P><P>Our RSID's are valuable parts in their own right, as they can be applied to any other projects employing sRNA interference to yield similar modularity and precision targeting.
</P><H4><SPAN class="mw-headline" id="Device_and_Parts_Level_Diagrams">Device and Parts Level Diagrams</SPAN></H4><P>
(1) Input A activates Receiver and Reporter 1. (2) Input B activates Receiver and Reporter 2. (3) There is more input A, so Receiver and Reporter 1 will strongly repress Receiver and Reporter 2. (4) There is less input B, so the repression going the other way is weaker. (5) This leads to the production of output A, and  (6) little to no production of  output B. (7) The output of the device is either dominated by output A or output B. By seeing which is the case, we can infer whether input A or input B was present in greater amounts.
</P><P>
(1) Input A leads to the transcription of (2) GFP mRNA and (3) sRNA targeting RFP mRNA. This is how we implement both reporting and repressing outputs as a result of sengins input A. (4) Input B leads to two similar transcription events: (5) RFP mRNA and (6) sRNA targeting GFP mRNA. In this scenario, because there is more input A than input B, the GFP mRNA is translated whereas the RFP mRNA is silenced by its respective sRNA, thus leading to (7) a device output that is clearly more green than red, reflecting the relative levels of input A to input B.   </P><H4><SPAN class="mw-headline" id="Flow_Cytometry_Data">Flow Cytometry Data</SPAN></H4><P>Below is a graph of our flow cytometry data for part K353002 plotting GFP output for BW27783 cells:
</P><P>Both of these graphs plot the output of part K353002 in BW27783 cells induced with a series of concentrations of  L-arabinose, measured in molarity. The first is a histogram showing the distribution of expression levels at each concentration of L-arabinose. The second is the same data displayed as a cumulative distribution so that population medians are more apparent.
</P><P>
We measured a dose-response-curve of median output vs L-arabinose concentration in triplicate. This curve constitutes the basic characterization of part K353002, including measurement of the concentration of L-arabinose which gives half-maximal output, the steepness of transition from off to on, and the parts dynamic range.  Fitting a Hill curve provided the following values plus or minus 95% confidence intervals: </P><UL><LI>Half-max induction: 143 +- 36 uM
</LI><LI>Hill coefficient: 1.4 +- .6
</LI><LI>Fold Induction: 860
</LI></UL><H2><SPAN class="mw-headline" id="The_Second_Design:_Kinase_Phosphatase_System">The Second Design: Kinase Phosphatase System</SPAN></H2><H4><SPAN class="mw-headline" id="Device_Properties_2">Device Properties</SPAN></H4><P><B>Method</B>: Kinase/Phosphatase Regulation of a Transcription Factor
</P><P><B>Output</B>: One output protein whose concentration is linearly dependent on the ratio of the input chemicals
</P><P><B>Useful</B>: For acting in a situation requiring a graded response: drug delivery, metabolic flux control, detailed ratiometric reporting
</P><P><B>Data</B>: Please refer to the following pages for data:
</P><UL><LI><A href="http://partsregistry.org/Part:BBa_K353005" class="external free" rel="nofollow">http://partsregistry.org/Part:BBa_K353005</A></LI><LI><A href="http://partsregistry.org/Part:BBa_K353006" class="external free" rel="nofollow">http://partsregistry.org/Part:BBa_K353006</A></LI><LI><A href="http://partsregistry.org/Part:BBa_K353007" class="external free" rel="nofollow">http://partsregistry.org/Part:BBa_K353007</A></LI><LI><A href="http://partsregistry.org/Part:BBa_K353008" class="external free" rel="nofollow">http://partsregistry.org/Part:BBa_K353008</A></LI></UL><H4><SPAN class="mw-headline" id="Overview_2">Overview</SPAN></H4><P>Our second sensor design responds to a <B>range of ratios</B>, varying the <B>intensity of the output</B> to indicate the ratio of input chemicals.
</P><P>One of the input chemicals leads to the phosphorylation of a transcription factor, while the other leads to its de-phosphorylation. Only the phosphorylated version of the transcription factor can activate the promoter upstream of the output protein. Visit our <A href="/Team:Stanford/Research/Modeling" title="Team:Stanford/Research/Modeling"> modeling</A> page to read about how the ratio of the two inputs can be computed as a linear function of the output.
</P><P>uses a transcription factor regulated by a kinase/phosphotase pair. In this system, the phosphorylated form of the transcription factor causes transcription of a gene coding for our output protein. The production of the transcription factor is under the control of a constitutive promoter, which maintains a basal concentration. The kinase that acts on the transcription factor is under the control of a promoter positively regulated by A. A phosphotase is similarly controlled by input B. By testing the concentration of output protein in relation to various concentrations of input chemicals, we plan to create an algorithm that will allow us to work backwards from a given concentration of output protein to deduce the ratio of the original concentrations of input chemicals.
</P><P>Read about our <A href="/Team:Stanford/Digital_Sensor" title="Team:Stanford/Digital Sensor">other sensor</A> or return to our <STRONG class="selflink">project</STRONG> page.
</P><H4><SPAN class="mw-headline" id="Device_and_Parts_Level_Diagrams_2">Device and Parts Level Diagrams</SPAN></H4><P>Input A <B>(1)</B> activates receiver 1 <B>(3)</B>, which activates the reporter <B>(5)</B>, causing synthesis of the output protein <B>(6)</B>.
</P><P>Input B <B>(2)</B> activates receiver 2 <B>(4)</B>, which inactivates the reporter <B>(5)</B>, decreasing synthesis of the output protein <B>(6)</B>.
</P><P>Part K353007 is under the control of R0040, so there is always some basal level of transcription factor present within the cell. 
L-arabinose binds to I0500, causing transcription of K353005, which phosphorylates the transcription factor K353007, allowing it to bind and activate K353008 (reporter).
AHL binds to F2620, activating transcription of K353006, which removes the phosphate group from the the transcription factor (K353007), inactivating it and preventing it from binding K353008.
</P><H2><SPAN class="mw-headline" id="Ratios:_Better_than_Absolutes">Ratios: Better than Absolutes</SPAN></H2><P>Why did we go to the trouble of developing sensors to detect ratios when so many already exist to detect absolute concentrations? Because <B>ratios are better.</B></P><P>For many biological processes, absolute concentrations vary with time or environmental conditions. A sensor detecting the absolute concentration of a biologically significant molecule would respond to these fluctuations, creating systemic noise and false results. Our sensors can detect a &quot;normalized&quot; version of the molecule's concentration by comparing it to a different molecule whose regular cycles are similar.
</P><P>Our unified sensor is also better than simply combining two absolute sensors into a single cell and comparing the outputs. Besides requiring the cell to produce fewer proteins (freeing up cell energy for other processes), our approach significantly reduces output noise. Additionally, a two-output system requires a human observer to take readings and make a decision. Our system can be hooked up straight to an output protein, allowing a rapid in-vivo response.
</P><H3><SPAN class="mw-headline" id="Project_Goals">Project Goals</SPAN></H3><H4><SPAN class="mw-headline" id="Digital_Sensor">Digital Sensor</SPAN></H4><UL><LI>Accurately report whether the concentrations of input chemicals lie at, above, or below a given ratio
</LI><LI>Allow as little expression as possible at the detection ratio, and as much as possible at all other ratios
</LI></UL><H4><SPAN class="mw-headline" id="Analog_Sensor">Analog Sensor</SPAN></H4><UL><LI>Accurately report over a large range of input ratios
</LI><LI>Achieve a linear correlation between output protein concentration and detection ratio
</LI><LI>Have a low basal level of expression
</LI></UL><H4><SPAN class="mw-headline" id="Both_Sensors">Both Sensors</SPAN></H4><UL><LI>Allow future users to change output proteins and input promoters without disrupting the sensor's internal machinery
</LI><LI>Have both sensors be orthogonal to the host system to avoid cross-talk
</LI></UL><H3><SPAN class="mw-headline" id="Important_Parameters">Important Parameters</SPAN></H3><H4><SPAN class="mw-headline" id="Digital_Sensor_2">Digital Sensor</SPAN></H4><OL><LI>Range of possible ratios
</LI><LI>Background expression level
</LI><LI>Of both proteins at desired ratio
</LI><LI>Of either protein when it shouldn't be being expressed
</LI></OL><LI>Response curve
</LI><LI>Response time
</LI><H4><SPAN class="mw-headline" id="Analog_Sensor_2">Analog Sensor</SPAN></H4><OL><LI>Range of possible ratios
</LI><LI>Background expression level (no kinase-inducing chemical added)
</LI><LI>Phosphotase specificity
</LI><LI>Response curve
</LI><LI>Response time
</LI></OL><P>--&gt;</P><H2><SPAN class="mw-headline" id="Biosafety">Biosafety</SPAN></H2><P>1.  Would any of your project ideas raise safety issues in terms of researcher safety, public safety, or environmental safety?
</P><P><B>Answer:</B> There are no major safety risks regarding our general system design. In its current form, our device is only meant for ratiometric analysis of compounds within E. coli. As with any other biological hazard, typical laboratory precaution should be taken when handling cells resistant against antibiotics.
</P><P>
2. Is there a local biosafety group, committee, or review board at your institution?
</P><P><B>Answer:</B> There is a local board at Stanford University called Health and Safety at the Stanford University School of Medicine.
</P><P>
3. What does your local biosafety group think about your project?
</P><P><B>Answer:</B> We have not communicated with our local biosafety group regarding the design or safety of our project.  
</P><P>
4. Do any of the new BioBrick parts that you made this year raise any safety issues?
</P><P><B>Answer:</B> Although the kinases and phosphatases chosen for our project come from Mycobacterium tuberculosis and Streptomyces coelicolor A3(2), our team advisors and graduate mentors believe that these proteins do not pose any significant safety risks.
</P><H2><SPAN class="mw-headline" id="Research">Research</SPAN></H2><UL><LI><A href="/Team:Stanford/Research/Research_and_Development" title="Team:Stanford/Research/Research and Development">Research and Development</A></LI></UL><UL><LI><A href="/Team:Stanford/Research/Modeling" title="Team:Stanford/Research/Modeling">Modeling</A></LI></UL><H2><SPAN class="mw-headline" id="Medal_Requirements">Medal Requirements</SPAN></H2><H3><SPAN class="mw-headline" id="Bronze_Medal"> Bronze Medal </SPAN></H3><OL><LI>Register the team, have a great summer, and have fun attending the Jamboree. 
<DL><DD>Done!
</DD></DL></LI><LI>Successfully complete and submit a Project Summary form.
<DL><DD>Done!
</DD></DL></LI><LI>Create and share a Description of the team's project via the iGEM wiki
<DL><DD><A href="/Team:Stanford" title="Team:Stanford">Done!</A></DD></DL></LI><LI>Present a Poster and Talk at the iGEM Jamboree 
<DL><DD>Booked our plane tickets!
</DD></DL></LI><LI>Enter information detailing at least one new standard BioBrick Part or Device in the Registry of Parts
<DL><DD><A href="http://partsregistry.org/wiki/index.php?title=Part:BBa_K353002" class="external text" rel="nofollow">Done!</A></DD></DL></LI><LI>Entered information for each new part or device should at least include primary nucleic acid sequence, description of function, authorship, any relevant safety notes, and an acknowledgement of sources and references.  
<DL><DD><A href="http://partsregistry.org/wiki/index.php?title=Part:BBa_K353002" class="external text" rel="nofollow">Done!</A></DD></DL></LI><LI>Submit DNA for at least one new BioBrick Part or Device to the Registry of Parts.
<DL><DD>Done!
</DD></DL></LI></OL><H3><SPAN class="mw-headline" id="Silver_Medal"> Silver Medal </SPAN></H3><OL><LI>Demonstrate that at least one new BioBrick Part or Device of your own design and construction works as expected.
<DL><DD><A href="/Team:Stanford/Parts" title="Team:Stanford/Parts">Done!</A></DD></DL></LI><LI>Characterize the operation of at least one new BioBrick Part or Device and enter this information on the Parts or Device page via the Registry of Parts
<DL><DD><A href="/Team:Stanford/Parts" title="Team:Stanford/Parts">Done!</A></DD></DL></LI></OL><H3><SPAN class="mw-headline" id="Gold_Medal"> Gold Medal </SPAN></H3><OL><LI>Characterize or improve an existing BioBrick Part or Device and enter this information back on the Registry.
<DL><DD> When sequencing one of our ligations, we noticed an unexpected sequence in our results. After more investigation, we determined that that sequence came from the Distribution part <A href="http://partsregistry.org/wiki/index.php?title=Part:BBa_E1010" class="external text" rel="nofollow">BBa_E1010</A>, and that the part sequence listed on the Parts Registry was incomplete. We have notified the Registry and made a note of this discovery on the E1010 parts experience page.
</DD></DL></LI><LI>Outlined and detailed a new approach to an issue of Human Practice in synthetic biology as it relates to your project, such as safety, security, ethics, or ownership, sharing, and innovation. 
<DL><DD> With our Twitter project, we hope to help all iGEM teams by facilitating collaboration (i.e. sharing of data and inducing innovation) between teams. Read more about it <A href="/Team:Stanford/Twitter" title="Team:Stanford/Twitter"> here</A>!
</DD></DL></LI></OL></DIV></DIV></CENTER><DIV class="printfooter">
Retrieved from &quot;<A href="http://2010.igem.org/Team:Stanford/Research">http://2010.igem.org/Team:Stanford/Research</A>&quot;</DIV></DIV></DIV><DIV id="footer-box" class="noprint"><DIV id="footer"><UL id="f-list"><LI id="t-recentchanges"><A href="/Special:RecentChanges" title="Recent changes">Recent changes</A></LI><LI id="t-whatlinkshere"><A href="/Special:WhatLinksHere/Team:Stanford/Research" title="List of all wiki pages that link here [j]" accesskey="j">What links here</A></LI><LI id="t-recentchangeslinked"><A href="/Special:RecentChangesLinked/Team:Stanford/Research" title="Recent changes in pages linked from this page [k]" accesskey="k">Related changes</A></LI><LI id="t-specialpages"><A href="/Special:SpecialPages" title="List of all special pages [q]" accesskey="q">Special pages</A></LI><LI><A href="/Special:Preferences">My preferences</A></LI></UL></DIV><DIV id="footer"><UL id="f-list"><LI id="t-print"><A href="/wiki/index.php?title=Team:Stanford/Research&amp;printable=yes" title="Printable version of this page [p]" accesskey="p">Printable version</A></LI><LI id="t-permalink"><A href="/wiki/index.php?title=Team:Stanford/Research&amp;oldid=208274" title="Permanent link to this revision of the page">Permanent link</A></LI><LI id="privacy"><A href="/2010.igem.org:Privacy_policy" title="2010.igem.org:Privacy policy">Privacy policy</A></LI><LI id="disclaimer"><A href="/2010.igem.org:General_disclaimer" title="2010.igem.org:General disclaimer">Disclaimers</A></LI></UL></DIV></DIV></DIV></BODY></HTML>