<HTML xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en" dir="ltr">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki  ltr ns-0 ns-subject page-Team_York_Constructs"><DIV id="globalWrapper"><DIV id="top-section"><DIV id="p-logo"><A href="/Main_Page" title="Main Page">&quot;
	    </A></DIV><DIV id="menubar" class="left-menu noprint"><UL><LI class="selected"><A href="/Team:York/Constructs">Page               </A></LI><LI><A href="/Talk:Team:York/Constructs">Discussion               </A></LI><LI><A href="/wiki/index.php?title=Team:York/Constructs&amp;action=edit">View source               </A></LI><LI><A href="/wiki/index.php?title=Team:York/Constructs&amp;action=history">History               </A></LI><LI style="color:white;cursor:default">teams</LI></UL></DIV><DIV class="right-menu noprint" id="menubar"><UL><LI id="pt-login"><A href="/wiki/index.php?title=Special:UserLogin&amp;returnto=Team:York/Constructs" title="You are encouraged to log in; however, it is not mandatory [o]" accesskey="o">Log in</A></LI></UL></DIV><DIV id="search-controls" class="noprint"><FORM action="/Special:Search" id="searchform"> </FORM></DIV></DIV><DIV id="content"><H1 class="firstHeading">Team:York/Constructs</H1><DIV id="bodyContent"><H3 id="siteSub" class="noprint">From 2014.igem.org</H3><P><TITLE>Team York 2014</TITLE></P><DIV class="navbar navbar-inverse navbar-fixed-top"><DIV class="container"><DIV class="collapse navbar-collapse navHeaderCollapse"><UL class="nav navbar-nav navbar-right"><LI><A href="https://2014.igem.org/Team:York">Home</A></LI><LI class="dropdown active"><A href="#" class="dropdown-toggle" data-toggle="dropdown">Project
 </A></LI><LI><A href="https://2014.igem.org/Team:York/Project">The Challenge</A></LI><LI class="active"><A href="https://2014.igem.org/Team:York/Constructs">The Solution</A></LI><LI><A href="https://2014.igem.org/Team:York/Application">Future Applications</A></LI></UL><LI class="dropdown"><A href="#" class="dropdown-toggle" data-toggle="dropdown">Achievements
 </A></LI><LI><A href="https://2014.igem.org/Team:York/Judging">Judging Criteria</A></LI><LI><A href="https://2014.igem.org/Team:York/Results">Results</A></LI><LI><A href="https://2014.igem.org/Team:York/Parts">Parts</A></LI><LI class="dropdown"><A href="#" class="dropdown-toggle" data-toggle="dropdown">Policy And Practice
 </A></LI><LI><A href="https://2014.igem.org/Team:York/Environment">Environmental Impact</A></LI><LI><A href="https://2014.igem.org/Team:York/Surveys">Survey and GMOs</A></LI><LI><A href="https://2014.igem.org/Team:York/Outreach">Outreach</A></LI><LI><A href="https://2014.igem.org/Team:York/Collaborations">Collaborations</A></LI><LI class="dropdown"><A href="#" class="dropdown-toggle" data-toggle="dropdown">Notebook
 </A></LI><LI><A href="https://2014.igem.org/Team:York/Protocols">Protocols</A></LI><LI><A href="https://2014.igem.org/Team:York/Notebook">Notebook</A></LI><LI><A href="https://2014.igem.org/Team:York/Safety">Safety</A></LI><LI class="dropdown"><A href="#" class="dropdown-toggle" data-toggle="dropdown">Team
 </A></LI><LI><A href="https://2014.igem.org/Team:York/Team">Students and Instructors</A></LI><LI><A href="https://2014.igem.org/Team:York/Sponsors">Sponsors and Attributions</A></LI></DIV></DIV></DIV><DIV class="container"><DIV class="jumbotron"><H1>The Solution</H1><DIV class="row"><DIV class="col-lg-8"><H2>From Gene to Function</H2><P>Both cadmium and sulfates can create serious problems both to the environment and human health. Both can also be found in high concentrations in industrial output. This year, our project at iGEM York is focusing on increasing the uptake of sulfate in <I>E. coli</I> in order to chelate cadmium ions. The project has two main, interlinked approaches:<OL><LI>The increased uptake of sulfur using an exogenous sulfate transporter from <I>Bacillus</I>.
</LI><LI>The increased uptake and chelation of cadmium ions by metal binding proteins, to produce a potentially harvestable metal product.
The link between these two processes is phytochelatins; the sulfur rich metal binding proteins which will be used to chelate our cadmium ions.</LI></OL></P><P>Our system is activated by high cadmium concentration. Detection of cadmium ions by stress related proteins inside the cell (soxS and fur) result in the activation of the inducible promoter (pYodA). pYodA allows the expression of exogenous sulphate secondary transporter from Bacillus subtilis – CysP - and overexpression of endogenous cadmium transporters – MntH from NRAMP family - enhancing the response in a positive feedback (higher cadmium concentration-higher activation).</P><P>The sulphate is transported inside the cell and converted into cysteine in the cysteine biosynthesis pathway. Overproduction of Cysteine is achieved by a mutant enzyme in the pathway (CysE*) insensitive to negative feedback by the ultimate product: L-cysteine.</P><P>The cysteine produced in excess is utilized for phytochelatin production due to the expression of the Phytochelatin Synthase gene from S. pombe in our circuit. Therefore, phytochelatins will bind cadmium chelating it in the intracellular environment.</P><P>In order to enhance the phytochelatin production another BioBrick is present in the final circuit: GSH1*, the mutated version of the endogenous E. coli gene gamma-glutamylcysteine synthetase, responsible for the first step in Glutathione Biosynthesis Pathway (converting cysteine into γGlu-Cys). The wild-type enzyme is inhibited by its direct product γGlu-Cys, however, the mutated version is insensitive to the negative feedback and allows overproduction of γGlu-Cys and γGlu-Cys-Gly, the predecessors of phytochelatins in the Phytochelatin Biosynthesis Pathway.</P><H2>Final Construct</H2><UL class="nav nav-tabs" role="tablist" id="myTab"><LI class="active"><A href="#one" role="tab" data-toggle="tab">pYodA</A></LI><LI><A href="#two" role="tab" data-toggle="tab">NRAMP</A></LI><LI><A href="#three" role="tab" data-toggle="tab">CysP</A></LI><LI><A href="#four" role="tab" data-toggle="tab">SpPCS</A></LI><LI><A href="#five" role="tab" data-toggle="tab">Gsh1*</A></LI><LI><A href="#six" role="tab" data-toggle="tab">CysE*</A></LI></UL><DIV class="tab-content"><DIV class="tab-pane active" id="one" class="collapse"><H2>pYodA</H2><P><B>Name:</B> pYodA (ZinTp) <B>Organism:</B><I>Escherichia Coli</I><B>Normal Function:</B> pYodA is a cadmium-induced promoter that activates the yodA(zinT) gene, leading to the production of ZinT metal-binding protein.<B>Aim:</B> We are using pYodA in an alternative way; to regulate the expression of the CysP gene (sulfate transporter) and the NRAMP gene (Cadmium transporter). Due to using pYodA, the expression of these two genes will be regulated by the concentration of Cadmium in the extra cellular environment. The over-production of Cysteine will only occur when the concentration of Cadmium reaches the sensitivity threshold of pYodA. Due to cysteine production being both energetically demanding and toxic at high concentrations, we do not want these genes to be expressed constitutively. <B>Characterisation:</B> In order to characterise pYodA, we will couple it with a GFP gene. This will allow us to calculate the sensitivity threshold of pYodA and measure the amount of cadmium that can be chelated at various concentrations. <B>Literature:</B><OL><LI>http://mic.sgmjournals.org/content/148/12/3801.long</LI><LI>http://www.uniprot.org/uniprot/F4VWH2</LI><LI>http://sbkb.org/uid/F4VWH2/uniprot#structures</LI><LI>http://www.jbc.org/content/278/44/43728</LI><LI>http://biocyc.org/ECOLI/NEW-IMAGE?type=GENE&amp;object=G7061</LI></OL></P></DIV><DIV class="tab-pane" id="two" class="collapse"><H2>NRAMP</H2><P><B>Gene:</B> mntH<B>Organism:</B><I>Escherichia Coli</I><B>Protein:</B> NRAMP<B>Function:</B> NRAMP is a membrane divalent metal-ion transporter. In addition to iron and manganese, it also transports Cd ions into the cell.<B>Aim:</B> We want to use the endogenous transporter NRAMP to take up the cadmium from the environment. The cadmium will then activate pYoda and set off the phytochelatin synthesis process.<B>Expression:</B> We can run assays measuring the concentration of cadmium in the environment. If NRAMP is active, the concentration of cadmium should decrease.</P></DIV><DIV class="tab-pane" id="three" class="collapse"><H2>CysP</H2><P><B>Gene:</B> cysP (AKA YlnA) <B>Organism:</B><I>Bacillus Subtilis</I><B>Protein:</B> CysP <B>Function:</B> This gene encodes for CysP, a sulfate/thiosulfate ABC transporter found in the periplasmic space. <B>Aim:</B> We want to over-express this gene in E.coli. This would lead to increased sulfate uptake, which could be used to produce cysteine and eventually phytochelatins that would bind cadmium. <B>Expression:</B> We could run assays measuring the concentration of sulfate in the environment containing our bacteria. </P></DIV><DIV class="tab-pane" id="six" class="collapse"><H2>CysE*</H2><P><B>Original Gene:</B> cysE<B>Mutant Gene:</B> cysE*<B>Organism:</B><I>Escherichia Coli</I><B>Protein:</B> CysE(SAT)<B>Normal function:</B> To catalyse the acetylation of L-serine (the first step in cysteine biosynthesis)<B>Function:</B> Serine acetyltransferase (CysE) carries out the first step in cysteine biosynthesis; it  catalyses the acetylation of L-serine which generates O-acetyl-L-serine. Cysteine itself strongly inhibits the activity of serine acetyltransferase by binding to the serine-binding site. This inhibition depends on the protein's carboxy terminus, and has been localized to Met-256 specifically. Due to cysteine production being both energetically demanding and toxic at high concentrations, the cell does not want to produce cysteine constitutively.<B>Aim:</B> To over-produce cysteine.To fulfill this aim, we need to remove negative feedback from cysteine biosynthesis. To do this, we are using a mutant CysE gene (CysE*). Our mutant CysE gene will produce a protein that has a single amino acid substitution: Met-256 will be replaced by Ile by changing the corresponding AUG codon to AUC. This single amino-acid substitution will alter the three dimensional shape of the serine-binding site in our acetyltransferase. Changing the shape of the serine-binding site prevents cysteine from binding to it. Thus, our acetyltransferase will not be inhibited by cysteine. As a result, we will be able to over-produce cysteine in our cell.<B>Method:</B> We had the cysE* gene synthesised to produce the mutant CysE* protein.<B>Literature:</B><OL><LI>http://aem.asm.org/content/66/10/4497.full</LI><LI>Denk D., Bock A. <I>J. Gen. Microbiol</I>, 1987</LI></OL></P></DIV><DIV class="tab-pane" id="five" class="collapse"><H2>Gsh1*</H2><P><B>Original Gene:</B> gsh1/gshA<B>Organism:</B><I>Saccharomyces cerevisiae</I><B>Protein:</B> Glutamate-Cysteine Ligase (previously known as gamma-glutamylcysteine synthetase)<B>Function:</B> Gamma glutamylcysteine synthetase catalyzes the first step in glutathione (GSH) biosynthesis;
L-glutamate + L-cysteine + ATP -&gt; gamma-glutamyl cysteine + ADP + Pi
Expression is induced by oxidants, cadmium, and mercury. Protein abundance increases in response to DNA replication stress.<B>Aim:</B> We can use the overproduced cysteine to make gamma-glutamyl cysteine, which is the monomer that forms phytochelatins (n=10-20). In order to do this, we need glutamate-cysteine ligase to catalyse the reaction. We can over-express GSH1* using the pYodA promoter. <B>Literature:</B><OL><LI>http://biocyc.org/YEAST/NEW-IMAGE?type=GENE-IN-MAP-IN-PWY&amp;object=YJL101C</LI></OL></P></DIV><DIV class="tab-pane" id="four" class="collapse"><H2>spPCS</H2><P><B>Gene:</B> spPCS<B>Organism:</B><I>Schizosaccharomyces pombe</I><B>Function:</B> The phytochelatin synthase in S. pombe uses glutathione with a blocked thiol group  to synthesise phytochelatins.<B>Aim:</B> Over-expression of this gene, together with Gsh1*, in <I>E.coli</I> has been shown to increase phytochelatin production and lead to a 7.5-times-higher Cd accumulation. We want to use this gene to make our bacteria more efficient in  taking up cadmium.<B>Literature</B><OL><LI>http://www.ncbi.nlm.nih.gov/pmc/articles/PMC2075016/</LI></OL></P></DIV></DIV></DIV></DIV></DIV></DIV></DIV><DIV class="printfooter">
Retrieved from &quot;<A href="http://2014.igem.org/Team:York/Constructs">http://2014.igem.org/Team:York/Constructs</A>&quot;</DIV></DIV></DIV><DIV id="footer-box" class="noprint"><DIV id="footer"><UL id="f-list"><LI id="t-recentchanges"><A href="/Special:RecentChanges" title="Recent changes">Recent changes</A></LI><LI id="t-whatlinkshere"><A href="/Special:WhatLinksHere/Team:York/Constructs" title="List of all wiki pages that link here [j]" accesskey="j">What links here</A></LI><LI id="t-recentchangeslinked"><A href="/Special:RecentChangesLinked/Team:York/Constructs" title="Recent changes in pages linked from this page [k]" accesskey="k">Related changes</A></LI><LI id="t-specialpages"><A href="/Special:SpecialPages" title="List of all special pages [q]" accesskey="q">Special pages</A></LI><LI><A href="/Special:Preferences">My preferences</A></LI></UL></DIV><DIV id="footer"><UL id="f-list"><LI id="t-print"><A href="/wiki/index.php?title=Team:York/Constructs&amp;printable=yes" title="Printable version of this page [p]" accesskey="p">Printable version</A></LI><LI id="t-permalink"><A href="/wiki/index.php?title=Team:York/Constructs&amp;oldid=381365" title="Permanent link to this revision of the page">Permanent link</A></LI><LI id="privacy"><A href="/2014.igem.org:Privacy_policy" title="2014.igem.org:Privacy policy">Privacy policy</A></LI><LI id="disclaimer"><A href="/2014.igem.org:General_disclaimer" title="2014.igem.org:General disclaimer">Disclaimers</A></LI></UL></DIV></DIV></BODY></HTML>