"
Page
Discussion
View source
History
teams
Log in
 
Team:CityColSanFrancisco/Notebook/LabBooks/Bowen
From 2009.igem.org
07172009
 
07172009
Goals:    PCR amplification of OprB from genomic DNA:  P. aeurginosa.
                Digestion of both plasmids.
                Ligation of gene into plasmid for further alteration.
                Large liquid culture of R. palustris.
PCR reaction: 10 reactions, 25μL total volume each. Basic forward and reverse primers were used.  Homegrown Taq polymerase was used.  All tubes contained the same reaction.  Exp003 of thermocycler used.
Plasmid Digestion:  4 reactions were done according to the plasmid digestion protocol.  Enzymes used: EcoRI and SpeI.
Gel Electrophoresis: 1.0% gel was used to run both the PCR and Plasmid digestion.  Two separate gels were run. 
PCR Gel:  9μL of rxn w/ 2μL loading dye.  10μL was loaded into each of the lanes. 5μl of 100bp ladder was also loaded.
Please refer to Figure 1 – not available.
Lane 1 – Ladder
Lane 2 – tube 2
Lane 3 – tube 4
Lane 4 - tube 6
Lane 5 – tube 8
Lane 6 – Ladder
Results: No distinct banding was seen.  Although the ladder turned out quite well, no other valid banding is seen.  Gel was included as it was the first that I had run. 
Digestion Gel: 10μL of rxn w/ 2μL of loading dye.  10μL was loaded into was loaded into each of the lanes. 5μL of 100bp ladder was loaded.
Please refer to figure 2.
Lane 1 – Ladder
Lane 2 – BBa_digest 1
Lane 3 – BBa_digest 2
Lane 4 – pSB_digest 1
Lane 5 – pSB_digest 2
Lane 6 – Ladder
Results:  Success!!  BOTH plasmids were digested as planned.  Thank you iGEM for kits that work!! Although it seems that the banding is really close, the actual fragments of the digested plasmids are so similar that they would not be resolved on a 1.0% gel.  Perhaps on a 2.0% gel…
Conclusion:  Because the PCR was a flop, it is necessary to look at the primers and see if there is a way to alter them.  Cycling parameters are also looked at.  Possibly annealing temp and time should be altered to better suit our primers.  After plenty of thought, it was discovered that one of the primers was designed with the biobrick extension sites added to the wrong side of the priming region.  Primers were re-ordered – in the correct order, with the same priming region.
07202009
Cell Culture:
Liquid cultures of the E. coli containing plasmids were inoculated in liquid LB culture tubes w/ Morton closures.  Two of six cultures were placed in a shaker incubator (37°C) and two were left out at room temp.
Other cultures (all cell lines) were maintained.  Liquid cultures were made for frozen cell bank.  Plates were created so they could be parafilmed and placed in a refrigerator for potential use once summer break is over.   Old plates and tubes were disposed of via autoclave.
07212009
Frozen stocks were created from overnight cultures.
07242009
Planning day!  Last day before summer vacation begins.  Will resume work mid to late August.  Insert big frowny face here!
08262009
Media Prep: LB, YP, and FEM media.  1L each.  Autoclaved and stored.  This media will be used to start large scale cultures for the battery chambers.
                *The vitamin solution and trace elements DO NOT get autoclaved!!!
Cell Culture:  Liquid and plates were created of R. ferrireducins, Shewenella, and R. Palustris.
09042009
PCR for Point Mutagenesis:  The forward primer w/ biobrick extensions is paired with the reverse mutagenesis primer (combo 1/6).  The reverse primer w/ biobrick extensions is paired with the forward mutagenesis primer (combo 5/7).  Exp003 was used in the thermocycler.
09092009
 I was intending to run a gel, but for some reason, all of the PCR product had evaporated…I placed them in microfuge and pulsed them for quite some time.  However, no product was recovered.  I repeated the PCR protocol from 09042009.  Hopefully, there will be product for running a gel on 09112009.
 
09112009
Gel Electrophoresis:  1.0% agarose gel was used.  5μL product was combined with an equal part of loading dye.  All 10μl were loaded onto the gel.  6μL of 100bp ladder was also loaded.
                See Figure 3.
                Lane 1 - Blank
Lane 2 – Ladder
                Lane 3 – negative control
                Lane 4 – tube 1 – primer combo 5/7
                Lane 5 – tube 2 – primer combo 5/7
                Lane 6 – Blank
                Lane 7 - Ladder
                Lane 8 – tube 1 – primer combo 1/6
                Lane 9 – tube 2 – primer combo 1/6
                Lane 10 - Blank
Results:  Although there is a LOT of extraneous banding, there IS a band in each of the lanes that is very promising!  Lanes 4 and 5 have a bright lower band that corresponds to the size of the small product (OprB gene w/ single base change) and the third band from the top of lanes 8 and 9 correspond to the larger product from point mutagenesis.  This reaction needs to be optimized and then run on a prep gel for ban extraction and purification.  Then, the two pieces can be ligated via PCR with only the primers containing the biobrick extensions. 
09162009
Planning day!! We have attempted to set a schedule for the rest of the time that we have to work.  If all goes well, this schedule will be a good one.  However, capturing the “gene in a jar” is harder than it seems!  Our plasmids are digested and ready to go.  The gene of interest is not.
 
09182009
PCR:  Instead of optimizing the last reaction, it has been decided to combine the products of the last run and proceed with PCR in attempt to combine the fragments with the forward and reverse biobrick primers.  The ultimate goal is to capture the gene of interest with the restrictions sites added to the ends of the gene.  It will then get ligated into the plasmids and transformed into competent E. coli cells.  Once the transformation has (hopefully) occurred, the plasmids will be harvested for transformation into R. palustris.  Exp004 on the thermocycler was used.  This raised the annealing temperature and extended the elongation time.  Gels will be run next meeting. 
09212009
Gel Electrophoresis: of 09182009 PCR reaction.
                See Figure 4.
                Lane 1 – Blank
                Lane 2 – Ladder
                Lane 3 – negative control
                Lane 4 – tube 1
                Lane 5 – tube 2
                Lane 6 – Blank
Results:  Although faint bands are seen, there is no real product to work with.  In order to run a prep gel for band excision and purification, it is necessary to see a clear band of product.  Perhaps it is time to do a colony PCR.  This would use the genomic DNA from a colony directly.  Perhaps there were errors in purifying the DNA…we shall see!
 
 
 
 
 
 
 
 
09252009
More PCR:  Colony PCR of P. aeurignosa.  7 reactions, 25μL each.  See Excel spreadsheet for concentrations.  MgCl2 added, regular PCR buffer used. 
Cycling info:
95°C – 4 minutes
92°C – 30 seconds
50°C – 30 seconds
72°C – 90 seconds
72°C – 5 minutes
4°C – forever.
29 cycles were done (temps 2, 3, and 4). 
09282009
Gel Electrophoresis:  A 1.0% gel was run.  It yielded nothing.  Only the ladder showed up.  Including a picture and info seems futile.
10022009
PCR:  In order to optimize the reaction (and yield useable product) this reaction focused on optimizing the MgCl2 concentration in each reaction tube. 
The concentration gradient was:
0.5μM, 1.0μM, 2.0μM, 3.5μM, and 5.0μM – 6 reactions.
The theory behind this was that purchased genomic DNA was preserved in TE buffer.  It was a thought that the EDTA of the buffer was chelating all of the free magnesium ions in the reaction.  By working with a gradient, it may be possible to use a specific concentration in further reactions.  Exp003 was modified: the new annealing temp was 60°C, hoping to make the reaction more stringent (preventing non-specific binding). 
Two different primer sets were used.  “New” primers were developed without the biobrick extensions, as well as the mutagenesis primers.  The PCR reaction was done in four parts: Colony PCR w/ primers “1/6” and “5/7” for point mutagenesis, as well as with new primers to amplify only the gene of interest.  Genomic DNA was used with the same primers.  This created 2 reactions for each set.
 
10052009
Gel Electrophoresis:  6 gels were run…two of which yielded banding in promising places.  It seems that the mutagenesis primers are giving good results for the smaller fragment.  However, the other gels were not worth reporting. 
Results:
Figure 5: Banding seen is from Colony PCR and Primers 5/7.  It reveals the smaller fragment of the OprB gene.  The MgCl2 concentration that was optimal was 2.0μM
Figure 6: Banding here is similar to the banding in the previous gel.  However, the higher concentrations of MgCl2 (3.5 and 5.0μM) showed brighter banding. This is in congruence with the previous hypothesis that the DNA that was preserved in TE buffer needed additional MgCl2 to overcome the chelating agents found in the EDTA.
 
 
 
 
 
               
 
Retrieved from " http://2009.igem.org/Team:CityColSanFrancisco/Notebook/LabBooks/Bowen "
Recent changes
What links here
Related changes
Special pages
My preferences
Printable version
Permanent link
Privacy policy
Disclaimers
