Team:Concordia/Experiments/MF
iGEM Concordia Wiki
HOME
WEBSERIES
SAFETY
TEAM
MEET THE TEAM COLLABORATIONS ATTRIBUTIONS
PROJECT
DESCRIPTION DEMONSTRATE EXPERIMENTS PARTS NOTEBOOK
RESULTS
SYNTHESIS ATTACHMENT MICROFLUIDICS CDC28, FHUA-GBP, AND MELA IMPROVEMENT
HUMAN PRACTICES
PUBLIC ENGAGEMENT & EDUCATION INTEGRATED HUMAN PRACTICES
SPONSORS
SPONSOR LIST SPONSORSHIP PACKAGE
SPECIAL AWARDS
INTEGRATED HUMAN PRACTICES BASIC PART HARDWARE EDUCATION AND PUBLIC ENGAGEMENT
Back
Microfluidics Protocol
Master Fabrication
 
Clean 4’ Si wafer
 
-Rinse wafer with acetone
-Rinse with with water
-Rinse wafer with Isopropanol
-Rinse with water again
-Dry wafer with Nitrogen
-Heat wafer in 110-120C oven for 2-5 min OR use plasma asher if available
 
Coating step
-Place clean wafer into the spin coater and engage the vacuum to hold it in place
-Dispense a drop of HMDS spin onto the center of the wafer
-Spin at 3000rpm for 30 sec at acceleration 8(~660rpm/second)
-Dispense ~4ml of SU-8 2015 photoresist onto the center of the wafer
-Ramp to 500 rpm at 100 rpm/second acceleration(5 seconds)
-Ramp up rotational speed to 3000 rpm(300rpm/s) for 30 sec for 15 um thickness
-Clean spin coater
 
Soft Bake
-Prebake wafer on hot plate at 65C for 2 min
-Bake wafer on hot plate at 95C for 3 min
-Allow substrate to cool to room temperature
 
Exposure
-Align photomask on top of photoresist coated wafer with the mask aligner
-Exposure energy: (140 mj/cm2 for 5-6s)
 
Post Exposure Bake
-Bake wafer on hot plate at 65C for 1 min
-Bake wafer on hot plate at 95C for 4 min
-Allow substrate to cool for 20-30s
 
Development
-Submerge the wafer with  coated side up in SU-8 developer for 3 min (with agitation)
-Remove wafer with tweezers and rinse with Isopropanol
rinse with water
-Dry wafer gently with nitrogen gas (if you see white film on wafer, repeat development steps)
 
Hard Bake
-Bake wafer at 150C for 10 min
-Slowly cool substrate to room temperature
 
Storage
Store in petri dish away from sunlight at 4-21C
 
Chip Fabrication
 
-PDMS Mold
-Mix 50g of PDMS with 5 g of curing agent in a weighing boat
-Mix well and transfer the mixture to a 50ml falcon tube
-Centrifuge mixture at max rpm for 30s (make sure to balance centrifuge)
-Pour PDMS mixture into petri dish containing the master wafer
-Place Petri dish into vacuum desiccator until there are no more air bubbles (ideally overnight)
-Bake PDMS in a 65C oven for 2 hours (ideally overnight)
 
-Cut the border of the PDMS slab with a scalpel and peel it off the Wafer (gently)
-Punch holes into each fluid inlet and outlet
-Use a double sided scotch tape to remove ay residues or fingerprints for the PDMS mold
-Flush holes with Isopropanol
wash PDMS mold with Isopropanol, water and dry with nitrogen
-Wash glass microscope slides with Acetone,water,Isopropanol,water and dry with nitrogen
-Place PMDS slab and glass slide in plasma chamber channel side facing up(alternatively use corona discharge)
Gently press PDMS slab onto the glass substrate ensuring all parts are sealed to the glass
-Place the device in a 65C oven for 5-10 min
 
Flow 5ul pico glide through each device and wait 1 hour to coat the channels with a hydrophobic layer
Flow fluorinert through each device to flow out the pico glide
Insert proper tubing into the device and use syringe pump to flow in appropriate fluids
 
----------------------------------------------------------------------------------------------------------------------------
 
Write Notebook for Cloning(write protocols with dates)
-PCR part 1-7
-SOE parts 1 to 4 together and SOE parts 5-7 together
-homologous recombination  of SOE 1-4 and SOE 5-7 with pYES shuttle vector in saccharomyces cerevisiae
-Colony PCR the yeast that have undergone homologous recombination to check if SOE 1-4 and SOE 5-7 have been integrated into pYES properly
-Extract the plasmids from saccharomyces cerevisiae and transform them into E coli
-colony PCR the newly transformed E coli to check if plasmid integrated properly
-the 2 plasmids (FHUA → didnt work and MelA → did work)
 
PCR parts 1-7
July-09-2016
Purpose: The Polymerase Chain reaction (PCR) is a technique that we employed in the lab for DNA amplification. We used Phusion DNA polymerase in our setup  because of its extremely low error rates.
 
Protocol:
1. Reaction Setup: All the reaction components were assembled before the reactions were transferred to the thermocycler. A master mix was made composed of the following reagents:
Master mix:
195uL
autoclaved H2O
60uL
5X HF buffer
6uL
dNTPs (10 mM each)
 
-add 43.5ul of master mix to each PCR reaction
 
2. In 5 50ul PCR reaction in 200ul PCR tubes for: Gene(G) 1,2,4,6 and 7
G1: P1+P2
G2: P3+P4
G4: P7+P8
G6: P10+P11
G7: P12+P4
2.5 uL Primer F (10 uM)
2.5 uL Primer R (10 uM)
1 uL template (10 ng)
0.5 uL Phusion (add this last!!)
 
 
