<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_Concordia_Demonstrate_Recombinant skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><DIV id="top_title"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:Concordia/Demonstrate/Recombinant</SPAN></H1></DIV><DIV id="HQ_page"><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><P><TITLE>iGEM Concordia Wiki</TITLE></P><DIV style="background-color:#DCE1DE; min-width: 100px; max-width:2000px;"><HEADER><NAV><DIV class="dropdown"><BUTTON class="dropbtn"><A style="text-decoration:none; color:white;" href="https://2016.igem.org/Team:Concordia">HOME</A></BUTTON></DIV><DIV class="dropdown"><BUTTON class="dropbtn"><A style="text-decoration:none; color:white;" href="https://2016.igem.org/Team:Concordia/webseries">WEBSERIES</A></BUTTON></DIV><DIV class="dropdown"><BUTTON class="dropbtn"><A style="text-decoration:none; color:white;" href="https://2016.igem.org/Team:Concordia/Safety">SAFETY</A></BUTTON></DIV><DIV class="dropdown"><BUTTON class="dropbtn">TEAM</BUTTON><DIV class="dropdown-content"><A href="https://2016.igem.org/Team:Concordia/Team">MEET THE TEAM</A><A href="https://2016.igem.org/Team:Concordia/Collaborations">COLLABORATIONS</A><A href="https://2016.igem.org/Team:Concordia/Attributions">ATTRIBUTIONS</A></DIV></DIV><DIV class="dropdown"><BUTTON class="dropbtn">PROJECT</BUTTON><DIV class="dropdown-content"><A href="https://2016.igem.org/Team:Concordia/Description">DESCRIPTION</A><A href="https://2016.igem.org/Team:Concordia/Demonstrate">DEMONSTRATE</A><A href="https://2016.igem.org/Team:Concordia/Experiments">EXPERIMENTS</A><A href="https://2016.igem.org/Team:Concordia/Parts">PARTS</A><A href="https://2016.igem.org/Team:Concordia/Notebook">NOTEBOOK</A></DIV></DIV><DIV class="dropdown"><BUTTON class="dropbtn">RESULTS</BUTTON><DIV class="dropdown-content"><A href="https://2016.igem.org/Team:Concordia/Demonstrate/Synthesis_Results">SYNTHESIS</A><A href="https://2016.igem.org/Team:Concordia/Demonstrate/Attachment_Results">ATTACHMENT</A><A href="https://2016.igem.org/Team:Concordia/Demonstrate/Microfluidics_Results">MICROFLUIDICS</A><A href="https://2016.igem.org/Team:Concordia/Demonstrate/Recombinant">CDC28, FHUA-GBP, AND MELA IMPROVEMENT</A></DIV></DIV><DIV class="dropdown"><BUTTON class="dropbtn">HUMAN PRACTICES</BUTTON><DIV class="dropdown-content"><A href="https://2016.igem.org/Team:Concordia/HP/Silver">PUBLIC ENGAGEMENT &amp; EDUCATION</A><A href="https://2016.igem.org/Team:Concordia/HP/Gold">INTEGRATED HUMAN PRACTICES</A></DIV></DIV><DIV class="dropdown"><BUTTON class="dropbtn">SPONSORS</BUTTON><DIV class="dropdown-content"><A href="https://2016.igem.org/Team:Concordia/Sponsors">SPONSOR LIST</A><A href="https://2016.igem.org/Team:Concordia/SponsorshipPackage">SPONSORSHIP PACKAGE</A></DIV></DIV><DIV class="dropdown"><BUTTON class="dropbtn">SPECIAL AWARDS</BUTTON><DIV class="dropdown-content"><A href="https://2016.igem.org/Team:Concordia/Integrated_Practices">INTEGRATED HUMAN PRACTICES</A><A href="https://2016.igem.org/Team:Concordia/Basic_Part">BASIC PART</A><A href="https://2016.igem.org/Team:Concordia/Hardware">HARDWARE</A><A href="https://2016.igem.org/Team:Concordia/Engagement">EDUCATION AND PUBLIC ENGAGEMENT</A></DIV></DIV></NAV></HEADER></DIV><DIV style="background-color:#DCE1DE; overflow: hidden;"><DIV style="overflow:hidden;"><DIV id="" style="margin:3% 10%; padding:5%; border:solid 5px; border-color: black; background-color:#fffafa;"><H2 style="font-family: font-family:times new roman,times,serif;line-height:1.5;">Overview</H2><P style="font-family: font-family:times new roman,times,serif; line-height:1.5; font-size:150%">
As the name International Genetically Engineered Machine describes, iGEM is a competition that harnesses the use of molecular and genetic techniques for innovation and design of novel tools and devices for real-world applications. In order to harmonize the newly emerging field of nanotechnology with genetic engineering, iGEM Concordia’s 2016 team has been experimenting with effective methods of nanoparticle synthesis and attachment to different cell types, including  Escherichia coli bacteria and Saccharomyces cerevisiae yeast. In order to work toward nanoparticle synthesis and attachment, molecular cloning methods were utilized, and several BioBricks parts were improved in the process. iGEM Concordia has successfully contributed parts: BBa_K2045000 and BBa_K2045003 to the iGEM parts registry.
<B>BBa_K2045001 FhuA-GBP: Ferrichrome Iron outer membrane transporter-gold binding peptide</B>
In order to effectively bind gold nanoparticles to E.coli, Ferrichrome Iron outer membrane transporter was used as a binding intermediate between the E.coli cell and gold nanoparticles. This was done by improving iGEM BCCS-Bristol’s part, BBa_K259000, by fusing a gold binding peptide with their part which was FhuA.
We sought to synthesize and display the gold nanoparticles on the surface of E.coli cells. A gold binding peptide is a series of amino acids that have an affinity to gold, mostly sulfide based amino acids, since sulfide groups have a natural chemical affinity to gold. This property makes it a vital part for iGEM Concordia for effective attachment of gold nanoparticles onto the cell surface of E.coli. In order to accomplish this; gold nanoparticles were displayed by fusing the gold binding peptide with FhuA. FhuA is a ferrichrome Iron outer membrane transporter which we fused GBP with to create FhuA-GBP. The binding of gold nanoparticles occurred in cells expressing FhuA-GBP. These findings demonstrate the effectiveness of the proposed method in the synthesis and display of gold nanoparticles.This part has many future applications and will allow for the attachment of gold nanoparticles to many kinds of protein structures and will be a future key player in the emerging use of nanotechnology in synthetic biology.
<B>BBa_K2045002 CDC28: Cyclin-dependent kinase catalytic subunit</B>
In order to have effective cell battle with our microgladiators… It was important to ensure that cell division of our nanoparticle coated cells didn’t occur. Cell division would not only make it difficult for cell isolation in our microfluidic battle dome, but it will also compromise the integrity of the cell’s nanoweaponry; since the coating of nanoparticles will be halved as the cell membrane goes through cell division. 
The solution was CDC28, a gene that allows users to control the growth of their cells by steadily controlled external temperature. The cells will only be active and capable of cell division at 24oC and inactive, therefore incapable of cell division, at 37oC. By attaching nanoparticles to our yeast cells with CDC28, and keeping them at 37oC in the microfluidic battle system, we can ensure an effective battle between our micro-gladiators without compromising their attached nanoweapons. Since the presence of the wild type CDC28 gene in the yeast genome would mask the phenotype of the temperature sensitive mutant we decided to knock out the wild type version of the gene using CrispR. We designed 2 primers that would amplify the T.S. CDC28 with 3 silent mutations. CrispR gRNAs were then designed to bind to the wild type sequence of the gene but were unable to bind to the newly mutated version of the T.S. CDC28 due to the 3 silent mutations. The gRNAs and the mutated T.S. CDC28 were introduced to the wild type yeast and were selected for by antibiotic resistance. Future iGEM teams will be able to use our primer and gRNA designs in order to turn wild type yeast into cell cycle temperature sensitive yeast mutants.

1- PCR amplify temperature sensitive CDC28 gene with these primers
Forward primer: 5’ caaccgttaggagcAgaCatCgttaag  3’ 
Reverse primer: 5’ ctttactagActGtaCtgacagtgcagtagc 3’ 
(silent mutations in upper case and bolded)

2-Use newly amplified T.S. CDC28 as a donor  our 2 guide RNA constructs for CrispR chain reaction
gRNA1: gctgatattgttaagaagtt 
gRNA2: tataatgacagtgcagtagc




To characterize this biobrick, we grew an overnight culture of the temperature-sensitive CDC28 mutant and wild-type yeast cells. After diluting the cell solutions to an optical density of 0.1, both strains were grown in SC media at 24°C and 37°C in microplates. The OD at 600nm was measured every 20 minutes for 24 hours using a Tecan Sunrise machine to create growth curves for both the wild-type and the CDC28 mutant at the two temperatures. The results clearly indicate that at the nonpermissive temperature of 37°C there is a significant difference between the growth of the wild-type and mutant. The CDC28 mutant strain stays at a low OD while the wild-type displays a typical growth curve. At the permissive temperature of 24°C, there is little difference in the pattern of growth between the two cell strains. This confirms the ability of our biobrick part to inhibit growth at a specific temperature of 37°C.
</P><P style="font-family: font-family:times new roman,times,serif; line-height:1.5; font-size:150%"><B>BBa_K274001 MelA Cambridge</B>
iGEM Concordia 2016 aimed to synthesize nanoparticles for the overall goal of equipping cells with &quot;nano-weapons&quot; and having them fight in a microfluidics chip, in our project titled &quot;Combat Cells: League of Enhanced MicroGladiators.&quot; One of our methods of nanoparticle synthesis and attachment to cells is titled the recombinant method. Through the use of the protein MelA with its substrate L-DOPA, eumelanin is formed, which forms gold nanoparticles from gold ions in solution. 

IGEM Concordia 2016 aimed to improve the characterization of MelA by determining MelA's ability to use tyrosine as a substrate in place of L-DOPA. We cloned MelA's sequence into a plasmid, with MelA expression under the control of a constitutive promoter. This would allow us to determine if MelA had any cytotoxic effects. 

After transforming this plasmid into DH5alpha <I>E. coli</I> cells, we set up an experiment to test MelA's ability to use tyrosine as a substrate. We set up untransformed DH5alpha cells and just tyrosine wells as controls, alongside MelA-transformed DH5Alpha cells, all containing 2mM L-tyrosine. The wells contained 180 microlitres of cells at a starting OD600 of 0.5. We placed this plate into a Tecan-Sunrise, which measured the OD450 of the well contents at every 20 minutes for 2.5 days. The value of OD450 is directly related to the amount of melanin being formed in the wells. The following graph represents our obtained data.


Graphed for each of the wells containing DH5alpha cells (transformed with MelA and not transformed) is the average of three replicates. The error bars represent standard deviation. 

Tyrosine alone with no cells does not polymerize on its own, since there was no increase over time of OD450. Interestingly, there is a statistically significant difference in the OD450s when comparing the DH5alpha cells without and with MelA transformed in them. The data indicates that, in the presence of constitutive MelA production and 2mM L-tyrosine, more melanin in produced than wild-type <I>/E.coli</I>. This would indicate that 1) WT <I>E. coli</I> can produce melanin using L-tyrosine, and 2) MelA can use L-tyrosine to produce melanin, ever moreso than WT DH5Alpha. 

With this data, we intended to perform nanoparticle synthesis with L-tyrosine and MelA-transformed <I>E. coli</I> cells, and have these cells attach the gold nanoparticles to the surface of the E. coli cells using our FhuA-GBP fusion protein (BBa_K2045000), following the protocol created by Tsai et al [1].


[1] Tsai, Y.-J., Ouyang, C.-Y., Ma, S.-Y., Tsai, D.-Y., Tseng, H.-W., &amp; Yeh, Y.-C. (2014). Biosynthesis and display of diverse metal nanoparticles by recombinant Escherichia coli. RSC Adv., 4, 58717-58719. Royal Society of Chemistry. Retrieved from http://xlink.rsc.org/?DOI=C4RA12805B
</P></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></BODY></HTML>