Team:Concordia/Notebook/Recomb
iGEM Concordia Wiki
HOME
WEBSERIES
SAFETY
TEAM
MEET THE TEAM COLLABORATIONS ATTRIBUTIONS
PROJECT
DESCRIPTION DEMONSTRATE EXPERIMENTS PARTS NOTEBOOK
RESULTS
SYNTHESIS ATTACHMENT MICROFLUIDICS CDC28, FHUA-GBP, AND MELA IMPROVEMENT
HUMAN PRACTICES
PUBLIC ENGAGEMENT & EDUCATION INTEGRATED HUMAN PRACTICES
SPONSORS
SPONSOR LIST SPONSORSHIP PACKAGE
SPECIAL AWARDS
INTEGRATED HUMAN PRACTICES BASIC PART HARDWARE EDUCATION AND PUBLIC ENGAGEMENT
Plant Synthesis: Garlic
Plant Synthesis: Cabbage
Plant Synthesis: Aloe Vera
Chemical Synthesis: Martin
Chemical Synthesis: Turkevich
Cyborg Attachment
Nanoshell Attachment + Lyticase Test
Recombinant Method
Recombinant Notebook
PCR parts 1-7                                                
JULY 9
Introduction: The Polymerase Chain reaction (PCR) is a technique that we employed in the lab for DNA amplification. We used Phusion DNA polymerase in our setup  because of its extremely low error rates.
Purpose:The purpose of this experiment is to PCR amplify the Gblocks that amplify properly
 
Protocol:
Reaction Setup: All the reaction components were assembled before the reactions were transferred to the thermocycler. A master mix was made composed of the following reagents:
Master mix:
195uL
autoclaved H2O
60uL
5X HF buffer
6uL
dNTPs (10 mM each)
-add 43.5ul of master mix to each PCR reaction
2. In 5 50ul PCR reaction in 200ul PCR tubes for: Gene(G) 1,2,4,6 and 7
G1: P1+P2
G2: P3+P4
G4: P7+P8
   G6: P10+P11
 G7: P12+P4
32.5 uL
Autoclaved H2O
10uL
5x HF buffer
1uL
dNTPs (10 mM each)
2.5uL
Primer F (10 uM)
2.5uL
Primer R (10 uM)
1uL
template (10 ng)
0.5uL
Phusion
50uL
Total Volume
 
3. Insert the  5 PCR tubes into the Thermocycler.
Thermocycler Conditions for each Gene block
G1: P1+P2,G2: P3+P4
G4: P7+P8, G6: P10+P11
G7: P12+P4, FhuA-GBP
SOE parts 1 to 4 together and SOE parts 5-7                 
JULY 26
Introduction: Genetic recombination in which nucleotide sequences are exchanged between two similar or identical molecules of DNA. The SOE cassettes are inserted into the pYES and expressed S. cerevisiae.
Purpose: Insert SOE1-4 and SOE5-7 and express it yeast.
SOE1-4:MelA
pYES-MelA
32.5    uL  H2O
10 uL 5X GC buffer
1 uL dNTPs (10 mM each)
2.5 uL P1 (10 uM)
2.5 uL P4 (10 uM)
1 uL template G5
0.5 uL Phusion (add this last!!)
 
200 uL Final Volume
 
insert PCR tubes into thermocycler
 
PCR conditions: (ensure that the PCR lid is set to 108C to prevent evaporation)
 
98C 30 sec (initial denaturation)
98C 10 sec (denaturation) |
56 30 sec (annealing) |   x35
72C 1:20 minutes(extension) |
72 C 10 min (final extension)
12C HOLD
SOE5-7: FhuA
pYES BB_FHUA
pick colonies and resuspend in 50ul sterile water
 
Use the following reaction mix for each PCR:
-2 μl 10x Thermo polymerase buffer
-0.5 μl  dNTPs
-1.2 μl 10 μM P13 primer
-1.2 μl 10 μM P14 primer
-.2 μl Polymerase taq
-12.9 μl H2O
-2 μl template suspension
Master mix(7x)
-14 μl 10x Thermo polymerase buffer
-3.5 μl  dNTPs
-8.4 μl 10 μM P13 primer
-8.4 μl 10 μM P14 primer
-1.4 μl Polymerase taq
-90.3 μl H2O
-aliquot 18ul into each pcr tube and then add 2ul t  he corresponding template
PCR protocol
-95 C for 6 minutes (disrupt cells, separate DNA)
-Cycle 35 times:
-95 C for 30 s (melting)
-60 C for 30 s (annealing)
-72 C for 1:30 s (elongation)
-72 C for 10 minutes (final elongation)
-4 C forever
 
Colony PCR                                        
JULY 29
Introduction:  We employed a method for determining the presence or absence of abn insert DNA        
Purpose:the yeast that have undergone homologous recombination and to check if SOE 1-4 and SOE 5-7 have been integrated into pYES properly.
1.Use pipet tip to resuspend a plated colony in 15 μl sterile water.
 
2. Prepare reaction mixture
 
Reaction Mix
Use the following reaction mix for each PCR:
2 μl 10x Taq buffer
2 μl 10x dNTPs (10x = 2.5 mM each dNTP)
0.4 μl 40 μM FWD primer
0.4 μl 40 μM REV primer
0.2 μl Taq Polymerase
13 μl H2O
1.0 μl template suspension
 
PCR protocol
95 C for 6 minutes (disrupt cells, separate DNA)
Cycle 30 times:
95 C for 30 s (melting)
X C for 30 s (annealing)
72 C for X s (1min/kb) (elongation)
72 C for 10 minutes (final elongation)
4 C forever
 
Transformation of E.coli                                             
AUGUST 8
Introduction:“Competency” is the ability of a microorganism to uptake DNA from its environment. Some organisms are naturally competent. Competency can also be induced. One of the mysteries of molecular biology is how competency is induced… we know how do to it, but not quite how it works. There are two main ways to make organisms competent: high salt/heat, and electrical current. Salt/heat is easy and cheap: grow E. coli, mix with calcium salt, concentrate 20-fold. Once the cells are in calcium they are a bit fragile, so it’s important to resuspend gently.
Purpose:
Two flasks of 200 mL LB = 20 mL competent cells
Day 1
Pre-warm two 1L flasks – containing 200mL LB each – at 37°C overnight
NOTE: This isn’t necessary but decreases the time it takes for the E. coli to reach the right OD
Inoculate 5 mL DH5α in LB
For 400mL of inoculated LB, you will need:
200mL chilled 50mM CaCl2
20mL chilled 50mM CaCL2 + 15% glycerol
4 x 200mL pre-autoclaved, pre-chilled centrifuge bottles
NOTE: do NOT fill over half-full or they WILL leak!!!
 
Day 2
Inoculate DH5a 1:100 into both flasks (2mL into 200mL)
Grow to OD ~0.5 (between 0.45 and 0.55 is good)
Cool the temperature of the flasks as fast as possible: incubate the glassware in a slushy ice bath for 30 mins
Pour into 4 x 200mL pre-chilled centrifuge bottles. Balance the bottles
In pre-chilled centrifuge, spin @ 4000g for 10 mins
Decant supernatant
Resuspend each bottle in a small amount of cold 50mM CaCl2
Pool two bottles into one bottle, then complete each bottle to ~100mL 50mM CaCL2. Balance the bottles
Incubate on ice for 10 mins
In a pre-chilled centrifuge, spin @ 4000g for 10 mins
Decant supernatant; resuspend each bottle in small amount of cold 50mM CaCL2 + 15% glycerol
Pool two bottles into one 50mL Falcon tube, then fill to 20mL
Aliquot as desired
OPTIONAL: to increase transformation efficiency, snap-freeze in liquid nitrogen
Once cells are frozen, check the transformation efficiency of an aliquot
 
Colony PCR the newly transformed E coli         Aug-26-2016
 
Introduction:  We employed a method for determining the presence or absence of the transformed E.coli
1.Use pipet tip to resuspend a plated colony in 15 μl sterile water.
2. Prepare reaction mixture
 
Reaction Mix
Use the following reaction mix for each PCR:
2 μl 10x Taq buffer
2 μl 10x dNTPs (10x = 2.5 mM each dNTP)
0.4 μl 40 μM FWD primer
0.4 μl 40 μM REV primer
0.2 μl Taq Polymerase
13 μl H2O
1.0 μl template suspension
PCR protocol
95 C for 6 minutes (disrupt cells, separate DNA)
Cycle 30 times:
95 C for 30 s (melting)
X C for 30 s (annealing)
72 C for X s (1min/kb) (elongation)
72 C for 10 minutes (final elongation)
4 C forever
