"
Page
Discussion
View source
History
teams
Log in
 
Team:Macquarie Australia/Results
From 2013.igem.org
Project »
Project
Background
Parts Submitted
Achievements
Modeling
Photo Journal
Meet MQ »
The Team
Our Advisors
Our Instructors
Attributions
Official Profile
Sponsors
Human Practice »
Overview
Education
Safety
Open Day
Collaboration
Conference
Undergraduate Conference
Quarantine
Synthetic Society
Labwork »
Notebook
Protocols
Results and Characterisation
Links »
MQ Wiki Home
iGEM Home
Our University
Our Department
Our Twitter
Results and Characterisation
This page gives an overview of our results, which have provided large strides towards the production of Chlorophyll within E. coli. For more detail on our labwork and results, please see our Notebook .
BioBrick Construction             BioBrick Information             Characterisation
BioBrick Construction
Twelve BioBricks were successfully constructed, complying with all requirements set out by iGEM. An electrophoresis gel was run on EcoRI + PstI digests of all constructions, with bands confirming expected part sizes. Note that the part size in ChlD is roughly the same as the plasmid fragment, so only one band is visible in this lane.
BioBrick Information
All constructed BioBricks have been sent for sequencing to confirm fidelity with designs, and all sequences returned thus far have shown a 100% match with our gene design. Genes showing such a match have been submitted to iGEM via post, and we are currently awaiting sequences to be returned for our final three BioBricks. Click on gene names to be taken to the registry entry for that gene, with sequence information.
BioBricks Constructed
Gibson Composition
Sequence Confirmed
Submitted to iGEM
POR
2 fragments + vector
ChlG
2 fragments + vector
ChlP
2 fragments + vector
ChlI2
2 fragments + vector
YCF54
1 fragment + vector
CTH1
3 fragments + vector
ChlI1
2 fragments + vector
Gun4
2 fragments + vector
Plastocyanin
1 fragment + vector
ChlM
2 fragments + vector
pending...
DVR1
2 fragments + vector
pending...
ChlD
3 fragments + vector
pending...
Characterisation
Composite part creation
Composite part creation: attaching promoters to genes
We selected five of our genes for further characterisation: ChlD, ChlI1, ChlI2, Gun4, and Plastocyanin. A tac promoter BioBrick provided in the iGEM kit (part BBa_K864400) was digested with PstI and SpeI, gel cleaned, and used as a plasmid for ligation with XpeI and PstI digestions of these biobricks. These were ligated, transformed, and plated out, showing growth of hundreds of colonies per plate, with no growth on an unligated plasmid control (see plate image below).
Eight colonies were picked from each plate and grown up in LB+Chloramphenicol broth, and left to grow for 3 hours. 500ul of each broth was then pelleted, resuspended in 100ul water, and heated to 99 degrees for 5 minutes to lyse cells. PCR was then performed using lysed cell suspension as template, with BBVF and BBVR primers, with PCR products gelled (as shown below). Lanes with expected banding were used to identify successfully transformed colony cultures, which were then grown up in larger cultures for further investigation. Note that none of the eight ChlD colonies showed successful amplification - this may be due to the larger length of the ChlD gene.
Further investigation: ChlD
A simple assay is available to experimentally validate the function of our ChlD Biobrick. The basis of this test is to provide all proteins required to form the Magnesium Chelatase complex, except for ChlD. When the ChlD protein is present, Magnesium-protoporphyrin is formed from protoporphyrin already present in the E. coli. This formation can be observed via fluorescence, and we have performed this assay with the same setup as used in Zhou et al, 2010:
The final concentrations of assay components were 50 mM Tricine–NaOH (pH 8.0), 15 mM MgCl2, 2 mM dithiothreitol (DTT), 1 mM ATP (assay buffer), 2000 nM Proto, 200 nM CrChlI1 and 200nM CrChlI2, 500 nM CrChlH, 500 nM CrGUN4. ChlD Extract was added to CrI1 and CrI2 at 2x assay concentration in reaction buffer. Assay was started by adding 25 uL of Cr-ChlH and Cr-GUN4 at 2x assay concentration to 25 uL of the Cr-I1 Cr-I2 ChlD extract in assay buffer.
ChlD expressing E. coli cells were lysed with B-PER bacterial cell lysis reagent (Pierce) as per instructions. 2 ul of this extract gave the equivalent of 2.1ng of purified ChlD activity when added to the assay. The B-PER protein concentration was approximately 10mg/ml, indicating a low expression of the gene. Improvements of expression level may be obtained by using alternate promoters. The control used was a ChlI1 expressing cell extract at the same concentration (ie with ChlD activity).  Assay was monitored every 2 seconds for 100 seconds using fluorescence detection in a Pherastar plate reader with a 420nm excitation filter and a 590nm emission filter.
Graph 1 shows the fluorescence tracking for our control E. coli (without ChlD activity):
Graph 2 shows the fluorescence of two of our ChlD subcultures, with increasing fluorescence detected, indicating Magnesium-protoporphyrin-IX formation. Red line shows ChlD2 activity, green line shows ChlD4 activity:
References:
Zhou, S., Sawicki, A., Willows, R. D., & Luo, M. (2012). C-terminal residues of Oryza sativa GUN4 are required for the activation of the ChlH subunit of magnesium chelatase in chlorophyll synthesis. FEBS letters, 586(3), 205-210.
Further investigation: Plastocyanin
Our tac+plastocyanin composite biobrick theoretically expresses a copper containing protein. Colonies containing this biobrick should turn blue in the presence of inducer (IPTG) and copper. To test this, we plated this culture out on a regular LB plate, and a copper + IPTG LB plate. We also plated out a control plate with both Plasto and CHLI1 expressing colonies.
The first image below show plastocyanin expressing colonies growing blue on LB+copper+IPTG plates seen on the right, and white/yellow on regular LB plates shown on the left:
The second image shows a LB+copper+IPTG plate with plastocyanin colonies growing on one half, and non-expressing colonies (ChlI1) on the other half:
Retrieved from " http://2013.igem.org/Team:Macquarie_Australia/Results "
Recent changes
What links here
Related changes
Special pages
My preferences
Printable version
Permanent link
Privacy policy
Disclaimers
