"
Page
Discussion
View source
History
teams
Log in
 
Team:Macquarie Australia/Notebook2
From 2013.igem.org
Project »
Project
Background
Parts Submitted
Achievements
Modeling
Photo Journal
Meet MQ »
The Team
Our Advisors
Our Instructors
Attributions
Official Profile
Sponsors
Human Practice »
Overview
Education
Safety
Open Day
Collaboration
Conference
Undergraduate Conference
Quarantine
Synthetic Society
Labwork »
Notebook
Protocols
Results and Characterisation
Links »
MQ Wiki Home
iGEM Home
Our University
Our Department
Our Twitter
Notebook 2
This linearized second notebook page only exists as a backup to the original notebook, if a fault has occurred
Summer Break
We spent this week designing the sequences of the genes: ChlI1, CTH1, CHlM, POR, CHlD, ChlH, GUN4 and Plastocyanin that we will use for the project, including gBlocks and PCR products. The gBlocks were designed to eliminate unwanted restriction endonuclease sites, encode restriction enzyme sites EcoR1 (E), Xbal (X), SpeI (S) and PstI (P) at the ends of the gBlocks and possess 35bp overlaps between the gBlocks and PCR products that comprise each gene separately. Each gene sequence was ensured to have a bacterial ribosome binding site (shine-dalgarno sequence).
gBlocks:
1. ChlI1 - 1 gBlock + 2 PCR products
2. ChlH - 5 gBlocks + 3 PCR products
3. ChlD - 1 gBlock + 2 PCR products
4. GUN4 - 1 gBlock + 1 PCR product
5. ChlM - 1 gBlock + 1 PCR product
6. CTH1 - 2 gBlocks + 1 PCR products
7. POR - 1 gBlock + 1 PCR product
8. Plastocyanin - 1 gBlock
Short and long primers for the PCR products were designed. Long primers (only reverse primers) included restriction sites S and P, however the short primers were both forward and reverse primers.
Primers:
GUN4-F1   AAGCGCAACTGGGTTTACTTC
GUN4-R1S   ATACAACACCTGGCAGGATCA
GUN4-R1L   CAGCGGCGGCCTACTAGTATACAACACCTGGCAGG
ChlM-F1   AGACGGTGGACAAGGTGCT
CHLM-R1S   GTCCGAGTCCATCAGTCCTTAC
CHLM-R1L   CAGCGGCCGCCTACTAGTCCGAGTCCATCAGTCC
ChlD-F1   CCCGTGGAAGACCAAGATGC
ChlD-R1S   GTAACGCTCACATACCAACAC
ChlD-R1L   CAGCGGCCGCCTACTAGTAACGCTCACATACCAAC
Chll1-F1   AGCTGATGAGCGAGGAGGT
Chll1-R1S   TGCCGCTTACTCCATGCCG
Chll1-R1L   CAGCGGCCGCCTACTAGTGCCGCTGACTCCATG
CTH1-F1   CGGAGATGGTCAACGACTG
CTH-R1S   GCTCCAAAGCACTCTCAAAACT
CTHR1L   CAGCGGCCGCCTACTAGTGCTCCAAAGCACTCTC
ChlH-F1   GCTCATCTTCATTGAGGAGCTT
ChlH-R1   GCTGTTCACGAAGTTCTTG
ChlH-F2   GAGGATCTGATCCAGTCGGT
ChlH-R2   CTGGTCAACGTCAGCCAGGA
ChlH-F3   GAGATGTACCTGAAGCGCAAG
ChlH-R3S   ACTAGTGCTACCATGAAGGG
ChlH-R3L   CAGCGGCCGCCTACTAGTGCTACCATGAAGGG
POR-F1   ACTCAATCCTGCACCTGGAC
POR-R1S   CAGAGACGAACCGCAGGTAG
POR-R1L   CAGCGGCCGCCTACTAGTCAGAGACGAACCGCAG
Summer break week 2:
During week 2 we focused on making LB media, SOC media and gels for electrophoresis. PCR reactions were performed on all the genes.
The results we obtained showed that all the genes’ PCR products were successfully amplified, apart from ChlD (both long and short primers). From this we decided to use the short forward and long reverse primers for future PCR amplification of gene PCR products. The gene products were cut out from the gel and purified.
Nanodrop concentrations were taken for each gene product. For Gibson Assembly (required to join the gBlock and PCR products together) a concentration of 10ng/μl DNA was aimed to be obtained however, for smaller genes a smaller concentration could be adequate.
GUN4  - 15.0 ng/μL
ChlM  - 6.0 ng/μL
ChlI1 – 15.0 ng/μL
CTH1 – 17.0 ng/μL
ChlH1 - 12.0 ng/μL
ChlH2 – 8.0 ng/μL
ChlH3 – 4.0 ng/μL
POR short primer – 5.5 ng/μL
POR long primer – 5.4 ng/μL
To conclude this week’s wet lab work ChlH2, ChlH3 and POR were amplified again with PCR as the concentration of DNA was too low. ChlD was also re-amplified due to PCR failure.
Summer break week 3:
The gBlocks arrived for all the genes. ChlD, POR and ChlH2 and ChlH3 PCR products were run out on gels and DNA concentrations were observed using nanodrop. The results indicated the DNA concentrations were less than 6.0ng/μL thus, are not adequate enough for use in Gibson Assembly. E. coli Top10 cells were made competent using electroporation.
Summer break week 4:
The genes ChlH2, ChlH3, POR and CHlD were amplified again with PCR. However, the temperature of the PCR machine was set too low, thus PCR attempt was unsuccessful. Therefore, the PCR was redone using slightly modified primers (1/10 dilution of the ChlD forward primer).
GET PICTURE FOR THESE RESULTS – pg 18
This week, we discovered that our previous concentrations for GUN4, ChlM, ChlI1, CTH1 and ChlH1 may have been due to protein contamination. The nanodrop was being read at an absorbance of 260/230 rather than 260/280 thus the results are inconclusive. From these results we wanted to determine the gene intensity by comparing them against a control (T3 plasmid) using gel electrophoresis. The results indicated that all the genes had similar intensities to the control plasmid and vectors, except for ChlM, which was slightly lower.
The genes ChlI1, GUN4, ChlM and CTH1 had the same approximate DNA concentration as the control, confirming the DNA concentrations obtained by the nanodrop weren’t caused by protein contamination.
ChlH1 had a similar intensity to the control; however POR-L, ChlH2 and ChlH3-L possessed a much lower intensity (weaker bands).
We performed gel purification on GUN4, ChlI1, ChlM, CTH1, plastocyanin, ChlH2 and  ChlH3.
Nanodrop readings:
ChLH2: 11.0 ng/μL
ChLH3: 5.0 ng/μL
Summer break week 5:
The plastocyanin was digested with restriction enzymes (S and E) to check that the gene had been inserted. The results indicated that the gene was successfully inserted into the plasmid vector.
We tested the competent cells by measuring the absorbance of an inoculate of E. coli Top 10 cells in LB media. The Absorbance reading was recorded as 0. 687.
ChlD, ChlD (with plasmid), POR, ChlI1, ChlM, GUN4 and CTH1 digests (using S and E) and plasmids were amplified again with PCR. The results indicated that the plasmid digests for ChlI1 and CTH1 showed suggested gene insertion, thus could be sent off for gene sequencing. The circled section of POR was cut out, purified and a nanodrop indicated a concentration of 22.5 ng/μL of DNA (Fig. 5).
Following this gel electrophoresis, we performed a plasmid preparation and restriction digests (using S and E) for ChlM and GUN4. The results indicated that it was successfully digested, thus confirming the genes hae been inserted into the plasmid vectors.
Summer break week 6:
GUN4, ChlM, plastocyanin, CHlI1 and CTH1 were sent for sequencing. The method used was Big-Dye sequencing. GUN4 and plastocyanin came back as successful however, the others were incorrect. GUN4 and plastocyanin were placed into glycerol stock (-80oC storage).
Gibson assembly for ChlH and POR was completed; however our results indicated that it was unsuccessful. The promoters Tet and Lac, from the 2012 part kits, were tested and transformed into E. coli.
Objectives achieved: received gBlocks, designed sequences for genes and primers (except for ChlD) and successfully inserted GUN4 and plastocyanin into separate Biobrick parts.
Week 1
With the break between semesters over, the Macquarie iGEM team returned to classes; for many of us this was the first day of iGEM 2013. We had our first meeting and discussed our project. Dr. Louise Brown and Associate Professor Rob Willows were introduced to the team and we began to determine who would take on certain roles within the team.
We eagerly began our project, deciding to use Gibson Assembly to produce our optimised genes. We had decided to develop BioBricks necessary for the biosynthesis of chlorophyll within E.coli. To do this, we constructed a gene pathway, containing 12 genes to reach our goal of chlorophyll biosynthesis.
Objective: Redesign ChlD primers, Gibson assemble genes into BioBrick parts, transform BioBrick into E. coli, confirm insertion of genes, express individual BioBrick genes and form the composite BioBrick parts.
Making the genes
The genes POR, ChlM, CHlI1 and CTH1 were PCR amplified from gene template (plasmid vectors).  Only ChlM and CTH1 were successful (figure 1).  These were gel purified.
Work with ligating genes into BioBrick parts
Some work was done with BioBrick to get familiar with procedures.  ChlM and CTH1 were Gibson assembled into BioBrick vectors, which were subsequently PCR amplified (Figure 2).  These do not appear to have been successful, as the bands are not at the expected size. The bands in this gel are probably just empty vector.
Week 2
Gibson Assembly
Gibson assembly was set up for the following genes: ChlM, ChlI1, CTH1, POR, ChlG, ChlP, DVR1, and ChlI2.
These were then transformed using electroporation, incubated for 1 hour, then plated out (100µl and 300µl) and incubated overnight at 37°C. These plates were all unsuccessful except for ChlG. Transformation attempt 2
As all of the transformations were unsuccessful except for ChlG, the Gibson assembly products were retransformed using electroporation, incubated, and plated out onto agar with chloramphenicol. These were left to grow up over the weekend at 37°C. Growth was unsuccessful.
Work on ChlD
A new template for ChlD was needed as a change was made to the design of the gene (a gBlock was added to the end of the gene), thus new primers were also required. The primers were designed in the winter holidays and the primers arrived in week 2. ChlD was PCR amplified from a plasmid and run on a gel (figure 1 – 2 lanes on the right). This was successful and a band was cut from the gel and used as the template. An attempt to PCR amplify this gene using primers was made, however this was not successful (figure 1 – 2 lanes on the right).
Week 3
Plasmid prep for ChlG
Only ChlG grew colonies from week 2. White colonies were selected (red colonies also grew, but this red colour indicated that the gene had not been inserted into the plasmid). Thus, some colonies were picked, sub-cultured and incubated overnight at 37°C.  The plasmids were then extracted. The product was checked using PCR amplification and running the product on a gel. The PCR showed an expected result, suggesting a successful insertion into a BioBrick (figure 1). Strategies to improve transformation efficiency
Due to the low transformation efficiency from the attempt in week 2, we looked at the variables in the process to identify the problem. There were a number of issues that were tested over this week. We discovered that the SOC media used was old; therefore we retransformed using new SOC media. This didn’t seem to improve the results of the transformation. The electroporator was tested using different cells and was found to be successful. The antibiotic chloramphenicol was also suspected as having an effect on the growing colonies so plates of some of the genes were also grown up in ampicillin. These plates were also unsuccessful. Therefore, we suggested that the competency of the Top 10 E. coli cells was inadequate. The competencies of the E. coli cells were tested using a test kit (with a defined DNA concentration). The competency was found to be ok. However we did not have any luck with transforming any more genes this week. Visualisation of the Gibson Assembly products
In addition to checking the efficiency of the transformation process, the Gibson Assembly products were checked by PCR amplification and run on a gel to verify their size (figure 1). The sizes were as expected so the Gibson Assembly products were assumed to be successful. The only exception was for DVR1; the PCR of this gene failed. Work on ChlD
The PCR product for ChlD was also attempted from the template made in week 2 (figure 1), however this PCR failed. Therefore a new ChlD template was required. One theory for this was that the overall volume for Gibson Assembly was 37µl, which is far higher than recommended.
Work on ChlH
We planned to Gibson assemble the fragments of ChlH, and realized that the stock of gBlocks were very low so we ordered more.
Week 4
Work on ChlD
Due to the difficulty with the ChlD template, and the lack of amplification of the PCR fragment used in the previous Gibson attempt, we decided to use a new template to amplify this fragment from. The first attempt was to RT-PCR from a cDNA source, which unfortunately failed (see figure 1).
Issues with transformation
Due to issues with transforming the genes, a different strain of E. coli, DH5α– the recommended strain for Gibson Assembly, was tested in terms of its competency for Gibson Assembled genes. The DH5α cells were of low competency using normal plasmids yet with plasmids formed via Gibson Assembly, the competency was adequate.
The concentration levels of Top10 E.coli were also thought to be a potential problem for transformation. Consequently, higher concentrations of Top10 E.coli were made. This allowed for the efficiency of the two strains to be compared. The new Top10 cells gave higher colony counts using transformation efficiency test plasmids, however, still showed little to no colonies for Gibson transformations.
Gibson Assembly and transformations
A Gibson Assembly was redone for DVR1, as the first one did not grow colonies and the PCR check failed. The Gibson products for ChlP, ChlG, POR, ChlM, ChlI1 and CTH1 were transformed into Top10 E.coli. These were incubated for 1 hour at 37°C, and then spread onto plates with chloramphenicol (20µl and 200µl). These were incubated at 37° overnight. Of these transformations, colonies grew for ChlP, ChlG and CTH1. Colonies were picked and subcultured and incubated in LB broth for several hours at 37°C. A plasmid prep was done and the concentrations were taken using a nanodrop, and then were stored in LB broth at 4°C.
Testing new primers for ChlP, ChlG and CTH1
ChlP, ChlG and CTH1 genes were PCR amplified using primers to determine the success of the BioBrick formation within the Gibson Assembly. New primers that amplify from the BioBrick were used to check transformation efficiency. The following table shows the expected length of each gene with the old and new primers (new primers are BBF2 and BBVR which amplify from within the vector, which can verify successful insertion into the BioBrick).
This gel indicates that the old and new primers successfully amplified the genes as expected lengths were obtained. Thus, it can be assumed that the genes were successfully integrated into the BioBricks.
Transformations
ChlM, ChlI1, ChlI2, POR, YCF54 and DVR1 were transformed into DH5α E.coli cells using electroporation. These were incubated, then plated out (20µl and 200µl) and left to grow over the weekend.
Week 5
Transformations
No growth was found on the plates from week 4. It was discovered that the incubator used was faulty and there were temperature fluctuations with a peak of 42°C, which would have killed the cells. The incubator was then reset and tested over the next two days for further issues.
Another attempt at transformation
Consequently the Gibson Assembly products for DVR1, POR, YCF54 and ChlI2 were re-diluted (1/3) and were transformed again into DH5α cells. In addition to those genes, ChlD from pET plasmids were also transformed into Top10 E.coli for future use as a template for PCR products. These were all incubated for 1 hour at 37°C and then plated out (30µl and 300µl) onto agar with chloramphenicol, and kanamycin for ChlD. These were then incubated overnight at 37°C. Growth was successful for all plates except for ChlI2. Colonies were picked and sub-cultured overnight at 37°C. A plasmid prep was done for each colony.
The Gibson Assembly products of ChlM, ChlI1, ChlP and CTH1 were retransformed into DH5α E.coli cells, incubated for 1 hour, and plated out onto agar with chloramphenicol (20µ and 200µl). These were then incubated overnight at 37°C. Transformed colonies were previously obtained from CTH1, however the gel of the restriction digest for this BioBrick did not look right, and was therefore thought to be unsuccessful. A PCR test and restriction digest were performed to verify successful transformation; and as bands were in expected places these were expected to be successful.
Work on ChlD
The new ChlD template was used to PCR amplify the PCR fragment required for Gibson Assembly using primers. A PCR reaction was performed to amplify the required fragment for Gibson Assembly. This reattempted Gibson Assembly product was transformed into top10 E.coli using electroporation. These were incubated, plated out and incubated overnight. No growth was observed, however, this was using Top10 E.coli cells which had shown inefficient transformation in the past. We therefore decided to find a new source of DH5α cells.
Restriction digests
PCR and restriction digest tests were performed on ChlP, ChlM, ChlI1 and CTH1 in order to confirm the likely successful transformation of BioBricks into E.coli. Figure 1 shows a successful result for all of these genes.
Work on promoters
Two different promoters from the iGEM (tac and lac - parts BBa_K864400 and BBa_R0010) were tested this week to look at their transformation efficiency.  They were both transformed using electroporation, incubated at 37°C, then spread onto plates (20µl and 200µl) and left to grow overnight at 37°C. Growth was successful for both promoters. Tac was selected as the promoter to use for our genes as this is a more efficient promoter.
The tac promoter was inserted into the BioBricks in front of genes: ChlG, CTH1, DVR1, ChlP, Plastocyanin, YCF54 and GUN4 using restriction digests (X, E, P and S). These were then transformed into Top10 E.coli cells using electroporation, incubated for 1 hour at 37°C, spread onto plates and then incubated overnight at 37°C. This was not successful as there was no growth.
Sending off genes for sequencing
The gene sequences for ChlM, ChlI1, ChlG, ChlP, POR, CTH1, DVR1 and YCF54 were sent off for gene sequencing
Week 6
Work on the promoters
The tac promoter (from the iGEM kit BBa_K864400) was inserted into plasmids to make BioBricks using Gibson Assembly. These BioBricks were then transformed into Top10 E.coli using electroporation and incubated at 37°C, then spread onto plates (20µl and 200µl) and left to grow overnight at 37°C.  Growth was successful. Two colonies were picked and sub-cultured, and then incubated overnight at 37°C. The plasmids were extracted and stored in LB broth.
3A assembly
The promoters were then extracted from the plasmids using restriction digests and then transformed into Top10 E.coli cells using heat shock. These were plated out onto plates (containing Kanamycin) using both 20μL and 200μL amounts. There were some colonies on the plates, however these were not the expected colonies and there was also fungal contamination. Transformation was unsuccessful possibly due to the use of the heat shock method.
Restriction Digests
Restriction digests were done (using X, E, S, P) on the colonies for the genes YCF54, DVR1, CTH1, POR and ChlP to build composite parts. This was successful for all the genes except for POR, indicated by the presence of two bands in each lane (figure 1). The POR product was not as expected, thus is incorrect and possibly was not digested. It is possible that the digestion of POR was inhibited as the sample had the largest volume of plasmid. (That colony of POR – A1 - later turned out to be an incorrect sequence, which explains the strange result in this gel).
Work on ChlH
New gBlocks were received for ChlH, so a Gibson Assembly and transformation was attempted. This was not successful. However, this was not expected to work as the gene is big, so the volume required for Gibson Assembly was much higher than recommended.
To check the Gibson Assembly product of ChlH we tried to amplify different sections using different primers. A map of the ChlH primers can be seen below in figure 2. ChlD was also amplified to check the Gibson Assembly product (see figure 3). The gel results indicated that only ChlH (amplified with the BBF primers) was successful. The Gibson assembled ChlD fragments indicated faint bands at 2kb, which is the expected length suggesting a possible successful result. These bands were cut out and  gel purified. We attempted to amplify ChlD using this as a template, however this was not successful.
Gene sequences arrive
The gene sequences for ChlM, ChlI1, ChlG, ChlP, POR, CTH1, DVR1 and YCF54 were received and verified that each gene had successfully been inserted into the vector, thus forming a BioBrick. Note: The following genes had multiple colonies grown on the antibiotic infused agar plates so they were sequenced and sub-cultured separately to allow us to look at different samples in case one was different and unsuccessful.
ChlG – A & B
ChlI1 – A & B
ChlM – A, B & C
ChlP – A, B, C & D
CTH1 – A
DVR1 – A & B
POR – A & B
YCF54 – A & B
All the colonies that were sequenced successfully, were sub-cultured and the plasmids were prepped to give us a high concentration of BioBricks to work with. 500µl of culture of each BioBrick was stored in glycerol stocks.
Work on ChlI2
ChlI2 was grown, digested with restriction enzymes (X, E, S and P) and then PCR amplified to verify its size. As the size was as expected, the BioBrick with ChlI2 was sent off for sequencing.
Week 7
Work with ChlD
A design for the Gibson Assembly for ChlD was modified, with multiple PCR amplifications carried out to obtain a high concentration of the required fragment. This PCR was deemed to be successful as the bands were the expected size (figure 1 – 2 lanes on the right). This higher concentration fragment was used in a new Gibson assembly attempt, with a much lower volume than in the previous attempt. This lower volume was better as it allowed us to follow recommendations for Gibson Assembly procedure.
Work with promoters – Second attempt with 3A assembly
As the attempt last week to insert promoters into BioBricks with genes was not successful, 3A assembly was attempted a second time. Promoters and genes were transformed using heat shock. These were then plated out onto plates containing kanamycin, and incubated overnight at 37°C. No growth was observed. The suspected problem was the heat shock method; consequently a second attempt was made using electroporation. However this was also unsuccessful.
Bioinformatics
Each of the sequences for the genes ChlG, ChlI1, ChlM, ChlP, CTH1, DVR1, POR and YCF54 were compared to the template sequence for potential amino acid changes and addition or deletion of nucleotides, which would cause a frame shift. The following colonies yielded a sequence that matched the template and all were a 100% match in amino acid sequence, with two exceptions as noted.
ChlG – A
ChlI1 – A & B
ChlM – A, B & C (template design has one different amino acid from the gene on NCBI)
ChlP – B, C & D
CTH1 – A
DVR1 – A & B (A has a frame shift and B has some issues)
POR – A & B B (last amino acid s different)
YCF54 – A
All the colonies that were sequenced successfully were sub-cultured and the plasmids were prepped to give us high concentration BioBricks to work with. 500µl of culture of each BioBrick were stored in glycerol stocks.
Week 8
A Gibson Assembly was planned for ChlH. The different fragments of ChlH were checked to verify their size (figure 1). The ChlD Gibson Assembly was also checked using PCR, however the expected amplification was not achieved.
Work on ChlD
The Gibson product for ChlD was transformed into DH5alpha E. coli cells using electroporation, incubated at 37°C, and plated out onto agar containing chloramphenicol. These transformations were not successful.
The Gibson Assembly product of ChlD was retransformed into DH5α E. coli cells using heat shock. These were then incubated at 37°C for 1 hour in LB broth, plated out onto agar containing chloramphenicol, and then incubated overnight at 37°C. Growth was successful (figure 2).
Work on the ChlH fragments
A new strategy for ChlH was designed. It was decided to carry out sub-Gibson Assemblies to build up the BioBrick in steps. To test this approach,two fragments were Gibson Assembled; this was run on a gel and was of an expected size (figure 3). This Gibson product was PCR amplified using primers to amplify both the PCR product and gBlock together, and this was unsuccessful (figure 3). This was deemed unsuccessful due to the short length of our primers, and with the iGEM deadline fast approaching, we decided to halt construction on ChlH. We have ChlH available in a pET plasmid which we may use to test the pathway in the absence of a ChlH BioBrick.
Promoters – Problem shooting
As efforts to ligate the promoters and the genes together using restriction digests and transformation, set out on the iGEM website, were unsuccessful last week, a PCR was performed and a gel run to check if they were the expected size. The promoters (tac) and genes (noted below in figure 4. the first 7 lanes from the left), were digested (promoters with Spe1 and Pst1, and genes with Xbal and Pst1), then these were ligated together into a new vector and PCR amplified to check if the ligation was successful. This PCR was not successful; it is thought thattoo many primers ligated to vectors which had no gene or promoter in it, thus giving a product of about 150 base pairs from the vector, rather than the target promoter and gene. After some revision of the method and materials used, it was decided that the best strategy would be to change the strain of E. coli from Top10 to DH5alpha, which is the recommended strain for Gibson Assembly.
Checking the concentrations of the genes with confirmed sequences
The concentrations of the plasmids with confirmed sequences were checked and were found to have insufficient concentrations. The suspected problem so our plan forward is to use different E. coli. Restriction digests
Restriction digests were done for Plastocyanin, GUN4 and POR to in preparation for ligation of composite parts (See figure 5).
Building up the composite parts
The BioBricks with the genes and promoter tac were digested (figures 6 and 7). These bands were then cut out and gel purified. GUN4, POR and plastocyanin were ligated together, transformed into DH5α E. coli, incubated, plated out and left to grow overnight.
More DH5alpha E. coli cells were made competent in preparation for the building of composite parts as these are the cells that are recommended for optimum performance in Gibson Assembly.
Week 9
Work on ChlD
White colonies from the plates that were grown last week were picked, sub-cultured and incubated at 37°C overnight. A plasmid prep was done, and these were then stored in LB broth in the fridge.  A plasmid prep was also done directly from the plate as some of the colonies were large. The nanodrop concentrations were found to be within the expected range.
Working on characterization
As colonies of the potential composite BioBricks grew last week, 4 colonies were picked, sub-cultured and incubated. These were plasmid prepped and a PCR with appropriate primers was performed to verify if ligation and transformation was successful (figures 1 and 2). The genes with the ligated promoters were compared to the gene without the promoter.
Note: There are extra steps in this process compared to protocol on the iGEM website, including PCR cleanup, running a gel and gel extraction. These extra steps were included as there have been a lot of problems working with promoters, so these steps were to ensure that each step was successful.
From these gels the ligation appeared to have been successful for some colonies, however to be certain an SDS-PAGE was performed.
First attempt with SDS-PAGE gel
As the promoter tac was inserted in front of the genes GUN4, POR and plastocyanin, an SDS-PAGE gel was run to check the expression levels of the proteins. ITPG was used to induce expression of the promoter. These composite BioBricks were sub-cultured into two sets, and ITPG was added to one set. These were then incubated for 4 hours at 37°C. The proteins were denatured by boiling for 2 minutes then treating with SDS-PAGE sample buffer and shearing. The gel was run and then visualized using a coomassie stain. These gels did not yield any results as the gel was allowed to run for too long, so the expressed proteins were run off the gel.
Second attempt with SDS-PAGE gel
A second SDS-PAGE gel was run using the same prepared samples of proteins; however the gel was stopped earlier in order to allow for expression to be visualized. These gels were unsuccessful, and did not show the expected bands for any of the genes. It was decided to prepare the proteins again as the problem with the gels was expected to come from this stage.
Third attempt with SDS-PAGE gel
After the first two attempts failed, a third attempt was made from scratch, setting up GUN4, POR and plastocyanin again in two sets; one with ITPG and one without. The same procedure was followed, however this SDS-PAGE gel also failed. Consequently, it was decided to make new restriction digests with all of the genes.
A second attempt at restriction digests this week
A tac BioBrick was digested with P and S to be the left-hand part so that right-hand side genes could be added (including ChlI1, ChlI2, GUN4, ChlM, plastocyanin, POR, ChlD, ChlP, ChlG and YCF54) by digesting with X and P. These had a chloramphenicol backbone. They were transformed using electroporation, incubated and plated out (300µl and 30µl). Colonies grew for each gene. Four colonies were picked for each gene with a promoter to screen for successful ligation, sub-cultured and incubated at 37°C. Due to the looming deadline, we sub-cultured each sample after 2 hours, pelleted the E. coli that had grown, heated it to 99°C for 10 minutes to burst the cells and then PCR amplified each sample to check if the ligation was successful. The PCR results showed that all the genes and promoters had failed to ligate.
At the same time, an attempt to ligate composite parts was done (CTH1 + YCF54, ChlP + ChlG, ChlD + ChlI2 and ChlI1 + ChlM). These were run on a gel and this was successful, however, the DNA yield was too low to use so this attempt was abandoned.
Test of the plasmid backbone purity
As the gels had not yielded the gene and promoter products as expected, a test was done of the plasmid backbone we had used for ligation and it was found that it was not a pure cut plasmid. This indicates that there was some circular plasmid present which had grown a lot of background colonies with the promoter ligated but no gene. This background plasmid was present as it has the same antibiotic resistance. The band from this gel of the properly cut plasmid was cut out, gel purified and was then used as vector for future digests.
Note: There are extra steps in this process compared to protocol on the iGEM website, including PCR cleanup, running a gel and gel extraction. These extra steps were included as there have been a lot of problems working with promoters, so these steps were to ensure that each step was successful.
A third attempt at ligation with the properly cut plasmid
The ligated products of ChlI1, ChlI2, plastocyanin, ChlD and GUN4 in chloramphenicol and plastocyanin and GUN4 in kanamycin were diluted (1/3), and ligated these into plasmids with the tac promoter and transformed into DH5α E. coli cells, incubated and plated out (300µl and 30µl). A transformation of just the cut plasmid was also done in order to check for background. In addition to these, we also plated out plastocyanin onto a copper plate as a plastocyanin is expected to change blue in the presence of copper. These plates were left to incubate overnight.
Results of plates
No colonies grew on the background check plate and many colonies grew on all the other plates, indicating that the ligation of promoter and gene into the plasmid was successful. This was tested by picking 8 colonies from each gene with a promoter and performing a PCR screening. Plastocyanin, ChlI1 and ChlI2 were successfully ligated, as indicated by bands at the expected places (figure 3). These were transferred to 20ml of LB broth, incubated for 6 hours, and then 1mM of IPTG was added to induce expression of the genes. These were left shaking overnight at 30°C.
The plates with copper that grew plastocyanin grew some colonies that were faint blue, indicating successful function of protein (figure 4).
ChlI2
The sequence for ChlI2 came back and was confirmed as the correct sequence. This was then sub-cultured, and the plasmid was prepped to obtain a high concentration BioBrick, which was then stored in glycerol stocks.
WIKI FREEZE
Retrieved from " http://2013.igem.org/Team:Macquarie_Australia/Notebook2 "
Recent changes
What links here
Related changes
Special pages
My preferences
Printable version
Permanent link
Privacy policy
Disclaimers
