<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_Stony_Brook_Results skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:Stony Brook/Results</SPAN></H1><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><DIV id="menu"><DIV id="iGEM"><A href="https://2015.igem.org/Team:Stony_Brook"><P id="goof"><B>Home</B></P></A></DIV><DIV class="topstuff" id="topstuff5"><A>People</A><A href="https://2015.igem.org/Team:Stony_Brook/Team" id="project1">Team</A><A href="https://2015.igem.org/Team:Stony_Brook/Attributions" id="project2">Attributions</A></DIV><DIV class="topstuff"><A href="https://2015.igem.org/Team:Stony_Brook/Description">Project</A></DIV><DIV class="topstuff" id="topstuff1"><A href="https://2015.igem.org/Team:Stony_Brook/Parts">Parts</A></DIV><DIV class="topstuff"><A href="https://2015.igem.org/Team:Stony_Brook/Results">Results</A></DIV><DIV class="topstuff" id="topstuff1"><A href="https://2015.igem.org/Team:Stony_Brook/Modeling">Modeling</A></DIV><DIV class="topstuff"><A href="https://2015.igem.org/Team:Stony_Brook/Notebook">Notebook</A></DIV><DIV class="topstuff" id="topstuff5"><A>Outreach</A><A href="https://2015.igem.org/Team:Stony_Brook/Practices" id="outreach1">Practices</A><A href="https://2015.igem.org/Team:Stony_Brook/Collaborations" id="outreach2">Collaborations</A></DIV></DIV><DIV id="mainContainer"><DIV id="contentContainer"><DIV class="top"><H2>Results</H2></DIV><DIV id="leftscroll"><H1><B><A href="#text1">Tripeptide Production</A></B></H1></DIV><DIV id="middlescroll"><H1><B><A href="#text2">Melittin Expression</A></B></H1></DIV><DIV id="rightscroll"><H1><B><A href="#text3">Medal Requirements</A></B></H1></DIV><DIV id="colMask"><DIV id="colLeft"><DIV class="section" id="section1"><DIV class="contentImage" id="pic1"><H3 id="t49">Tripeptide Production</H3></DIV><DIV class="contentText" id="text1"><H1>Tripeptide Production</H1><UL><H5 id="cow">Peptide Production and Secretion</H5><UL><P>
The Stony Brook iGEM team was successful in building the construct coding for the sequence of repeating QSP tripeptides. This sequence is under the OmpC promoter, allowing it to be induced under conditions of high osmolarity. It was marked with the HlyA secretion tag.</P><P>After conducting a successful transformation, we wanted to induce the system in various osmotic conditions. The supernatant of the cells was then to be isolated and, through a combination of Liquid Chromatography and Mass Spectrometry, the production and secretion of the peptide sequence into the extracellular space was to be quantified. If the system were to function correctly, a direct relationship would be expected to be seen between the concentration of solute in the environment of the cell and the presence of the peptide string in the extracellular fluid.</P><P>Unfortunately, while plasmid sequencing confirmed that the designed biobrick was successfully constructed, we were unable to confirm the function of the biobrick due to time restrictions and equipment inaccessibility.</P></UL><H5 id="cow">Protease</H5><UL><P>
The Stony Brook iGEM team was successful in building the construct coding the sequence for proline iminopeptidase</P><P>After successful transformation, it was our goal to isolate large quantities of the protease to determine a kinetic baseline for its degradation of L-proline p-Nitroaniline, as monitored over time in time-lapse Infrared Spectrometry. The results of the experiment were to serve as a comparison to the activity of the enzyme when compared to the construct designed to express this protease on the membrane with the OmpA membrane display tag. Prolyl endopeptidase was the protease we originally wanted to test with cleaving our tripeptides. Unfortunately, the sequence was too large to synthesize so another protease, proline iminopeptidase, was chosen instead. While proline iminopeptidase only acts on an N-terminal proline and cannot successfully cleave the peptide sequence into tripeptides, we wanted to see if our system could successfully produce a proline protease in sufficient quantities and we believed that the results of this experiment would be representative of the production and action of prolyl endopeptidase.</P><P>Unfortunately, due to time constraints and equipment inaccessibility, we were unable to both obtain the baseline kinetics of the free proline iminopeptidase and failed to successfully clone the construct needed to display the protease on the membrane. 
</P></UL><DIV class="section" id="section2"><DIV class="contentImage" id="pic2"><H3 id="t49">Melittin Expression</H3></DIV><DIV class="contentText" id="text2"><H1>Melittin Expression</H1><UL><H5 id="cow">Melittin</H5><UL><P>
The 2014 Stony Brook iGEM team accomplished an astounding feat- they were the first iGEM team of any State University of New York school and they managed to fund and conduct a project to combat antibiotic resistant bacteria using a protein that causes pores to form in the membrane of bacterial cells. After isolating the melittin gene directly from honey bees, the team cloned the sequence into E. coli cells. A system was devised in which melittin could be produced in its in active form as a GST fusion protein with a TEV cut site placed between melittin and the GST tag. Once outside of the cell, the GST could be cleaved and the protein would begin forming pores in the adjacent cells, effectively circumventing the need for futile attempts to kill resistant bacteria with antibiotics.

 </P></UL><H5 id="cow">Improvement</H5><UL><P>
The 2015 Stony Brook iGEM team not only had an interest in infectious diseases, but chronic diseases as well. We just couldn’t leave melittin in the -20 and not improve on last year’s project! While we waited for plasmids to digest and cell cultures to grow for our project, Nephrotides, we started on improving the melittin BioBrick. Expression of melittin had been low and upon further investigation, it appeared that the melittin sequence in the part was not codon optimized for E. coli cells- understandable since the sequence came directed from honey bees. Once the dry lab portion of our team, a collection of genius Geneious users, worked through the melittin sequence to optimize it for expression in E. Coli cells we worked to express that protein and quantify the improvement. We attached the gene encoding for mCherry to the original melittin sequence as well as our codon optimized melittin optimus and compared the expression levels. We saw a great deal more coloration in the cells transformed with the codon optimized melittin mCherry sequence so we can infer that the codon optimization enhanced the expression of melittin.

</P></UL><DIV class="section" id="section3"><DIV class="contentImage" id="pic3"><H3 id="t49">Medal Requirements</H3></DIV><DIV class="contentText" id="text3"><H1>Medal Requirements</H1><UL><H5 id="cow">Bronze</H5><UL><LI>Register as an iGEM Team</LI><LI>Complete Judging Form</LI><LI>Document and Share our Progress on our Wiki page</LI><LI>Document our Parts on the Registry</LI><LI>Present a poster at the iGEM Conference</LI><LI>Document at least one new biobrick: Proline Iminopeptidase and QSP Peptides</LI></UL><H5 id="cow">Silver</H5><UL><LI>Experimentally Validate a new BioBrick and submit it to the registry: melittin mCherry</LI><LI>Work “Beyond the Bench” and participate in community outreach: the Stony Brook iGEM taught classes on synthetic biology at the DNA Learning Center at Cold Spring Harbor Laboratories and held debates on the ethics of synthetic biology at the New York Hall of Science and the Port Jefferson Maker Faire.</LI></UL><H5 id="cow">Gold</H5><UL><LI>Help another iGEM in a significant way: the Stony Brook iGEM team measured and categorized various ribosome binding sites in the organism Methanococcus maripaludis for the University of Georgia Atlantic iGEM team. </LI><LI>Improve an existing BioBrick: The existing melittin biobrick was optimized as Melittin Optimus</LI><LI>Incorporate Human Practices into the design of your project: After learning from our debates at the Hall of Science and the Maker Faire that the main factor in determining whether a patient would use our system was the approval of their doctor, we reached out to our local hospital. From there, we met with medical faculty to adjust the design of our project from one based on re-engineering the microbiome of the intestines to a system enclosed in a biologically compatible microcapsule.</LI></UL><DIV id="footer"><UL><LI><A href="https://2015.igem.org/Team:Stony_Brook">Home</A></LI><LI><A href="https://2015.igem.org/Team:Stony_Brook/Team" id="home1">Team</A></LI><LI><A href="https://2015.igem.org/Team:Stony_Brook/Description" id="project1">Project Description</A></LI><LI><A href="https://2015.igem.org/Team:Stony_Brook/Results" id="project3">Project Results</A></LI><LI><A href="https://2015.igem.org/Team:Stony_Brook/Parts" id="parts3">Team Parts</A></LI><LI><A href="https://2015.igem.org/Team:Stony_Brook/Modeling" id="home3">Modeling</A></LI><LI><A href="https://2015.igem.org/Team:Stony_Brook/Notebook" id="home2">Notebook</A></LI><LI><A href="https://2015.igem.org/Team:Stony_Brook/Collaborations" id="outreach1">Collaborations</A></LI><LI><A href="https://2015.igem.org/Team:Stony_Brook/Practices" id="outreach2">Human Practices</A></LI><LI><A href="https://2015.igem.org/Team:Stony_Brook/Attributions" id="home4">Attributions</A></LI></UL><A href="#" id="toTop">Back To Top</A></DIV></UL></DIV></DIV></UL></DIV></DIV></UL></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></BODY></HTML>