Team:SVCE Chennai/Project
Toggle navigation
Results Data and Chart
Gallery Click o Click
Practices Spread the word
Human Practices Collaboration Entrepreneurship
Notebook My Calendar
Safety Safety is my priority
Team My creators
Members Brainstorm Attributions
Project It's all about me
Project: Minicins
Table of Contents
Project Description
Plasmid Design
Protocols
Parts
Project Description
Minicells are achromosomal cells formed by aberrant cell division in living cells. Several mutant  strains of E. coli (P678-54, χ925) have been isolated that have been shown to harbor a mutation in   a particular set of genes. FtsZ, one of the proteins involved in cell division has been shown to be   one of those. Overexpression of this protein leads to septation at the poles, leading to the   formation of the so-called minicells. The non-replicative behaviour of the minicells gives rise to   a broad spectrum of applications. Our project focusses on engineering minicells to produce   certain novel peptides called bacteriocins. Bacteriocins  are antimicrobial peptides (AMPs)   produced by almost all bacteria having specific detrimental effect on the surrounding microbial   community. Hence engineering minicells to produce bacteriocins can have a wide range of   applications in fields like animal husbandry, human probiotics and strain selection.
Minicells will be engineered as probiotics to produce bacteriocins and can be used as a defence  against common infections and diseases in the gastro-intestinal tract. Because the minicells do   not have the ability to multiply, there is no risk of infection associated with children and   immune-compromised people. Pseudomonas aeruginosa has been selected as a model gastro-  intestinal pathogen. The appropriate bacteriocin has been chosen to be Thuricin S, produced by   Bacillus thuringiensis.
One of the most common issues in the field of Medicine and in general, Biology, the formation  of MRSA biofilms, can also be addressed with this approach. Studies have shown that a   bacteriocin, Lysostaphin, can penetrate the biofilm and act on Staphylococcus aureus. But, the   organism has already developed resistance towards lysostaphin. Building on that premise, we   will be engineering minicells to produce Bactofencin A, a novel cationic bacteriocin produced by   Staphylococcus simulans and test its functionality.
Aside from the above areas, there are other prospective areas where minicells producing   bacteriocins can be applied. Bacteriocins are highly specific with respect to their target organism   and will selectively kill the undesired organism to which it is directed against. This property can   be utilized for strain selection on petri-plates. In addition, Clostridium botulinum – specific   bacteriocin producing minicells can also be engineered to prevent botulism through packaged   foods.
Plasmid Design
Our project aims to neuter multidrug resistant bacterial infections with the help of bacteriocins in   a minicell chassis.
Firstly, the idea was to use two different plasmids – one for minicell division and the other for  bacteriocin production. Then, we realized that plasmids are carried to the minicells. This requires   the use of two different plasmids with two different promoters. Instead of going for two different   plasmids, we modulated our design to a plasmid with two different expression cassettes.
The two cassettes are under the regulation of different promoters for the expression of the   different phenotypes. Minicell phenotype, driven by the overexpression of the ftsZ gene, is under   the regulation of the IPTG-controlled lac promoter. Bacteriocin production is driven by the                arabinose operon.
In order to confirm the expression of ftsZ, GFP is included under the lac promoter control by the   introduction of an RBS after the ftsZ stop codon. Similarly, bacteriocin production is tagged to   RFP.
pSB1K3 was preferred to pUC vectors since minicell purification involves the addition of   penicillin and that requires penicillin – sensitivestrains. In order to drive very high expression of   ftsZ, we decided to use the T7 promoter. It requires T7 RNA polymerase, which also needs to be   cloned into the plasmid. ftsZ can be indirectly regulated by controlling the expression of T7                RNA polymerase.
A major drawback that can be seen in this design is that there are over 10 inserts and the total                insert size amounts to more than 8 kb.
The number of inserts and their size can be greatly reduced by judicious selection of the vector   and the host strain. In order reduce the inserts corresponding to the ftsZ regulation which   involves T7 RNA polymerase and promoter, BL21 was chosen as the host strain and pET 24a   was identified as the vector. BL21 strain has T7 RNA polymerase inserted into its genomic   DNA. pET 24a vector has a multiple cloning region preceded by an operator region consisting of                the lac operator and the T7 promoter.
Hence, a second construct was designed utilizing the pET 24a vector. The complete insert was to   be inserted between the NdeI and XhoI restriction sites. After assembly of the parts by standard   biobrick assembly, the complete insert will be PCR amplified with NdeI and XhoI sites flanking                the entire insert.
The sequences for both the cassettes were ordered as gBlocks® from IDT. The complete cassette   was divided into two gBlocks® sized 1664 bp and 1057 bp respectively. A SacI site was   introduced in the araC region of the pBAD promoter. The nucleotide in position 430 was   mutated from C to G for creating a unique SacI site. The mutation created no changes in the 3’-               5’ araC reading frame.
The gBlocks® received were PCR amplified. No amplification was detected for gBlock® 1. We   deduced that the gBlock® had denatured into several small fragments by agarose gel   electrophoresis. Hence, we set out for standard biobrick assembly to assemble ftsZ, T0 terminator   and pBAD/araC promoter. The block 1 was constructed by assembling every component. PCR   amplification was then done to introduce the NdeI and SacI restriction sites. The gblock® 2                containing the two test bacteriocin sequences were obtained from IDT.
The fragments were cloned one by one in the pET 24a vector.
Protocols
PLASMID ISOLATION
Spin the cell culture in a centrifuge to pellet the cells, empty the supernatant (media) into a waste collection container.
Resuspend pelleted bacterial cells in 250 µl Buffer P1 (kept at 4°C) and transfer to a microcentrifuge tube.  
Add 250μl Buffer P2 and gently invert the tube 4–6 times to mix. Do not vortex, as this will result in shearing of genomic DNA.
Add 350μl Buffer N3 and invert the tube immediately and gently 4–6 times.
Centrifuge for 10 min at 13,000 rpm (~17,900 x g) in a table-top microcentrifuge. A white pellet will form.
Apply the supernatants from step 4 to the QIAprep spin column by decanting or pipetting.
Centrifuge for 30–60 s. Discard the flow-through.
Wash the QIAprep spin column by adding 0.5ml Buffer PB and centrifuge for 30-60s. Discard the flow-through.
Wash QIAprep spin column by adding 0.75 ml Buffer PE and centrifuging for 30–60s. 
Spinning for 60 seconds produces good results.
Discard the flow-through, and centrifuge for an additional 1 min to remove residual wash buffer.
Place the QIAprep column in a clean 1.5 ml microcentrifuge tube. To elute DNA, add 50μl Buffer EB (10 mM Tris·Cl, pH 8.5) or water to the center of each QIAprep spin column, let stand for 1 min, and centrifuge for 1 min.
ELUTION OF DNA
1. Excise the DNA fragment from the gel with a clean, sharp scalpel. Weigh the gel slice and transfer to a 1.5 ml microfuge tube.
2. Add 400µl of binding buffer 2 for each 100mg of gel weight. Incubate at 50˚C to 60˚ for 10mins and shake occasionally until the agarose is completely     dissolved. For high concentration gels 700µl of binding buffer 2 per mg of agarose gel are added.
3. Add the above mixture to the EZ-10 column and let stand for two minutes. Centrifuge at 10,000 rpm for two minutes and discard the flow-through in the     tube.
4. Add 750 µl of wash solution and centrifuge at 10,000rpm for 1 minute. Discard the solution in the tube.
5. Repeat step 4. Centrifuge at 10,000rpm for an additional minute to remove any residual wash buffer.
6. Place the column in a clean 1.5 ml microfuge tube. Add 30-50 µl of elution buffer to the centre of the column and incubate at room temperature for 2minutes. Centrifuge at 10,000rpm for 2 minutes to elute the DNA.
7. Store purified dna at -20˚C.
COMPETENT CELL PREPARATION
Inoculate 250ml of  SOB  medium with 1 ml vial of seed stock and grow at 20°C to an OD 600nm of 0.3. Use the "cell culture" function on the Nanodrop to determine OD value. OD value = 600nm Abs reading x 10.
Fill an ice bucket halfway with ice. Use the ice to pre-chill as many flat bottom centrifuge bottles as needed.
Transfer the culture to the flat bottom centrifuge tubes. Weigh and balance the tubes using a scale
Centrifuge at 3000g at 4°C for 10 minutes in a flat bottom centrifuge bottle.
Decant supernatant into waste receptacle, bleach before pouring down the drain.
Gently resuspend in 80ml of ice cold CCMB80 buffer
SEED STOCK PREPARATION FOR COMPETENT CELL
1. Pick single colonies into 2 ml of SOB medium and shake overnight at 23°C.
2. Add glycerol to 15%.
3. Aliquot 1ml of sample to nanocryotubes.
4. Place tubes into zip lock bag, immerse bag into dry ice or ethanol bath for 5 minutes.
5. Place in -80 °C freezer.
TRANSFORMATION PROTOCOL
1. Thaw competent cells on ice. Do not refreeze unused thawed cells, as it will drastically reduce transformation efficiency.
2. Pipette 50μl of competent cells into 2ml tube. Tubes should be labeled, prechilled, and place in a floating tube rack for support. Keep all tubes on     ice. Don’t forget a 2ml tube for your control.
3. Pipette 1μl of resuspended DNA into 2ml tube.Gently pipette up and down a few times. Keep all tubes on ice.
4. Pipette 1μl of control DNA into 2ml tube.Gently pipette up and down a few times. Keep all tubes on ice.
5. Close 2ml tubes, incubate on ice for 30min. Tubes may be gently agitated/flicked to mix solution, but return to ice immediately.
6. Heat shock tubes at 42°C for 1 min.2ml tubes should be in a floating foam tube rack. Place in water bath to ensure the bottoms of the tubes are     submerged. Timing is critical.
7. Incubate on ice for 5min. Return transformation tubes to ice bucket.
8. Pipette 200μl SOC media to each transformation.
9. Incubate at 37°C for 2 hours, shaker or rotor is recommended.
10. Pipette each transformation on two petri plates for a 20μl and 200μl plating. Spread with sterilized spreader or glass beads immediately. This helps ensure that you will be able to pick out a single colony.
11. Incubate transformations overnight at 37°C.
12. Pick single colonies from transformations.Do a colony PCR to verify part size, make glycerol stocks, grow cell cultures and miniprep.
13. Count colonies for control transformation on the 20μl control plate and calculate competent cell efficiency. Competent cells should have an efficiency     of 1.5x10^8 to 6x10^8 cfu/μg DNA.
AGAROSE GEL ELECTROPHORESIS
1. Make the melted agarose solution. To make a 1% gel in the small gel tray, use 0.35 g of agarose in 35 ml of buffer.0.35 g / 35 ml = 0.01 or 1%. Put the agarose in a 250 ml erlenmyer flask. In a graduated cylinder, measure out the appropriate volume of 0.5 X TBE Buffer for the appropriate size gel tray and pour it into the flask. Microwave for 30 sec then swirl, and then microwave another 30 sec, or until it boils but before it tries to bubble out of the flask. Use a paper towel or flask holder to grab the flask (watch out, it’s hot!) and swirl it around well.
2.Tape up the open ends of the gel tray with lab labeling tape. Also, combs are chosen and kept into the slots on the tray.
3. Add Ethidium Bromide (EtBr is toxic, so avoid creating toxic vapor by adding EtBr to hot liquid) before pouring it into the tray.
4. The final concentration of EtBr in the gel should be about 0.3 - 0.5 ug/mL.
5. Cast the gel. Pour the slurry slowly into the taped gel tray. Let the tray stand on a level surface for about 20 minutes.
6. While waiting for the gel to set, thaw the sample.
7. 6 X loading dye/buffer
8. DNA ladder/molecular marker
9. DNA samples
10. Carefully lift up the combs from the hardened gel. Remove the tape from the sides and place the entire tray into the gel box/tank.
11. Place the gel tray containing the hardened gel into the box/tank so that the top of the gel (top = the end with the wells nearest to the edge) is at the     negative (black) electrode-end of the tank. Pour enough 0.5 X TBE into the tank such that the gel is completely submerged.
12. Mix the DNA with the loading dye and load the wells. About 1μl of dye is added for every of DNA.
13. After loading the wells, Ethidium Bromide is added to the positive end (red electrode, or end opposite the wells in the gel) of the gel tank.
14. Place the lid on the gel tank and be sure the electrode connections are secure and tight.
15. Turn on the power supply by flipping the switch. Voltage is set at 60V.
16. Gel is allowed to run for 20-30 minutes.
17. Gel is viewed under UV and picture is taken.
RESTRICTION DIGESTION
1. Add 250ng of DNA to be digested, and adjust with distilled water for a total volume of 16μl.
2. Add 2.5μl of NEBuffer 2.
3. Add 0.5μl of BSA.
4. Add 0.5μl of EcoRI.
5. Add 0.5μl of PstI.
6. There should be a total volume of 20μl. Mix well and spin down briefly.
7. Incubate the restriction digest at 37°C for 30min, and then 80°C for 20min to heat kill the enzymes.
8. Run a portion of the digest on a gel (8μl, 100ng), to check that both plasmid backbone and part length are accurate.
LIGATION
1. Add 2μl of digested plasmid backbone(25ng) to an eppendorf on ice.
2. Add equimolar amount of restricted DNA fragments to the eppendorf.
3. Add 1μl T4 DNA ligase buffer.
4. Add 0.5μl T4 DNA ligase enzyme and make up the volume to 10μl using double distilled water.
5.Incubate the reaction at 16°C for 30 minutes.
6.Heat kill the enzyme at 80°C for 20 minutes.
MINICELL PURIFICATION
1. Transform the plasmid into competent cell and plate them.
2. Transfer 2-3 transformed colonies to seven 5ml LB broth and incubate for 3hrs.
3. Add each 5ml culture to seven 250ml LB Kan+ broth .
4.Study growth kinetics with one broth and when in log phase add IPTG to 250ml LB Kan+ broth at various concentration required.
5.Transfer competent cell to one 250ml LB broth.
6.For every 2 hrs take 30ml of sample to study the minicell phenotypic expression.
7.Centrifuge at 2000xg for 10 mins and collect the supernatant.
8. Centrifuge at 1000xg for 10 mins and discard the supernatant.
9. To the pellet add 10ml LB broth and incubate for 40 minutes in shaker incubator.
10. Add ceftriazone to a final concentration of 100 μg/ml and incubate for 45 minutes.
11. Centrifuge at 400xg for 10minutes collect the supernatant.
12. Centrifuge the supernatant at 1000xg for 10 minutes.
13. Wash the pellet in saline buffer twice and suspend in buffer and plate them in LB Kan+ plate.
Functionality Assay:
1. Dip a sterile swab into the inoculum tube.
2. Rotate the swab against the side of the tube using firm pressure, to remove excess fluid. The swab should not be dripping wet.
3. Inoculate the dried surface of a MH agar plate by streaking the swab three times over the entire agar surface; rotate the plate approximately 60 degrees     each time to ensure an even distribution of the inoculum.
4. Rim the plate with the swab to pick up any excess liquid.
5. Discard the swab into an appropriate container.
6. Leave the plate to sit at room temperature at least 3 to 5 minutes, but no more than 15 minutes, for the surface of the agar plate to dry before     proceeding to the next step.
7. Place the appropriate antimicrobial-impregnated disks on the surface of the agar, using either forceps to dispense each antimicrobial disk one at a     time, or a multidisk dispenser to dispense multiple disks at one time.
8. Once all disks are in place, replace the lid, invert the plates, and place them in a 35°C air incubator for 16 to 18 hours.
PCR
Standard PCR reaction
Amount
Componen
Final Concentration
w µL
Water
5 µL
10x PCR Buffer (P2192, B5925 or P2317)
1x
1 µL
Deoxynucleotide Mix
w µL
Forward primer (typically 15-30 bases in length)
200 µM
x µL
Reverse primer                               (typically 15-30 bases in length)
0.1-0.5 µM
0.5 µL
Taq DNA Polymerase (D6677 or D5684)
0.1-0.5 µM
y µL
Template DNA (typically 10 ng)
0.05 units/µL
z µL
25 mM magnesium chloride (use only with buffer P2317)
200 pg/µL
50 µL
Total Reaction Volume
0.1-0.5 mM
BUFFER PREPARATION
1.CCMB80 buffer
- 10 mM KOAc pH 7.0 (10 ml of a 1M stock/L)
- 80 mM CaCl2.2H2O (11.8 g/L)
- 20 mM MnCl2.4H2O (4.0 g/L)
- 10 mM MgCl2.6H2O (2.0 g/L)
- 10% glycerol (100 ml/L)
- adjust pH DOWN to 6.4 with 0.1N HCl if necessary.
- adjusting pH up will precipitate manganese dioxide from Mn containing solutions.
- sterile filter and store at 4°C.
- slight dark precipitate appears not to affect its function.
2. SOC
- SOB
- 20 mM glucose(0.36g/100ml)
3.SOB medium(100ml)
- 0.5% (w/v) yeast extract
- 2% (w/v) tryptone
- 10 mM NaCl(0.0584g)
- 2.5 mM KCl(0.0186g)
- 20 mM MgSO4 (0.240g)
4. Buffer P1
- 50 mM Tris-HCl pH 8.0
- 10 mM EDTA
- 100 μg/ml RNaseA
Prep - Dissolve 6.06g Tris base, 3.72g EDTA-2H20 in 800ml dH2O . Adjust the pH to 8.0 with HCl. Adjust the volume to 1 liter with dH2O. Add 100mg RNase A per liter of P1.
5. Buffer P2
- 200 mM NaOH
- 1% SDS
- Dissolve 8.09g of NaOH pellets in 950ml dH2O, 50ml 20% SDS solution. The final volume should be 1 liter.
6. Buffer N3 (100ml)
- 4.2 M Gu-HCl (40.12g)
- 0.9 M potassium acetate(8.83g)
-  pH 4.8
7. Buffer PB
- 5 M Gu-HCl (47.76g for 100ml)
- 30% isopropanol
8. Buffer PE
- 10 mM Tris-HCl(0.095g/100ml) pH 7.5
- 80% ethanol
9. Buffer EB (10 mM Tris·Cl(0.095g/100ml), pH 8.5)
10. TBE buffer: 10X (1000ml)
- Tris 108g
- EDTA 9.3g
- Boric Acid 55g
- pH 8.3
11. TE buffer: 10X
For 100ml
- Tris-HCL 0.605g
- EDTA 0.372g
- pH 7.4
12. TAE Buffer: 1L 10X
- Tris 48.5g
- Glacial acetic acid 11.4ml
- 0.5M EDTA 20ml
- pH 8
13. Elution Buffer
- 0.5 M sodium acetate(ph 7)
- 1 mM EDTA(ph 8)
Parts
BBa_K1826000
BBa_K1826001
BBa_K1826002
BBa_K1826003
BBa_K1826004
Lets talk!
Sri Venkateswara College of Engineering, Chennai
