<HTML xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en" dir="ltr">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki  ltr ns-0 ns-subject page-Team_TU-Eindhoven_MRIProcessing"><DIV id="globalWrapper"><DIV id="top-section"><DIV id="p-logo"><A href="/Main_Page" title="Main Page">&quot;
	    </A></DIV><DIV id="menubar" class="left-menu noprint"><UL><LI class="selected"><A href="/Team:TU-Eindhoven/MRIProcessing">Page               </A></LI><LI class="new"><A href="/wiki/index.php?title=Talk:Team:TU-Eindhoven/MRIProcessing&amp;action=edit&amp;redlink=1">Discussion               </A></LI><LI><A href="/wiki/index.php?title=Team:TU-Eindhoven/MRIProcessing&amp;action=edit">View source               </A></LI><LI><A href="/wiki/index.php?title=Team:TU-Eindhoven/MRIProcessing&amp;action=history">History               </A></LI><LI style="color:white;cursor:default">teams</LI></UL></DIV><DIV class="right-menu noprint" id="menubar"><UL><LI id="pt-login"><A href="/wiki/index.php?title=Special:UserLogin&amp;returnto=Team:TU-Eindhoven/MRIProcessing" title="You are encouraged to log in; however, it is not mandatory [o]" accesskey="o">Log in</A></LI></UL></DIV><DIV id="search-controls" class="noprint"><FORM action="/Special:Search" id="searchform"> </FORM></DIV></DIV><DIV id="content"><H1 class="firstHeading">Team:TU-Eindhoven/MRIProcessing</H1><DIV id="bodyContent"><H3 id="siteSub" class="noprint">From 2013.igem.org</H3><DIV class="navwrapper"><DIV class="container"><DIV class="navbar"><DIV class="navbar-inner"><UL class="nav"><LI id="tue-menu-home"><A href="/Team:TU-Eindhoven">Home</A></LI><LI id="tue-menu-team"><A href="/Team:TU-Eindhoven/Team">Team</A></LI><LI id="tue-menu-project" class="dropdown"><A class="dropdown-toggle" data-toggle="dropdown" href="#">Project </A></LI><LI><A href="/Team:TU-Eindhoven/Project">Abstract</A></LI><LI><A href="/Team:TU-Eindhoven/Description">Project Description</A></LI><LI><A href="/Team:TU-Eindhoven/Background">Background</A></LI><LI><A href="/Team:TU-Eindhoven/Chasis">The Chassis</A></LI><LI><A href="/Team:TU-Eindhoven/Production">Potential Contrast Agents</A></LI><LI><A href="/Team:TU-Eindhoven/KillingMechanism">Kill Switch</A></LI><LI><A href="/Team:TU-Eindhoven/Applications">Applications</A></LI><LI class="nav-header">Results</LI><LI><A href="/Team:TU-Eindhoven/Results">Main Results</A></LI><LI><A href="/Team:TU-Eindhoven/Parts">Parts Submitted</A></LI><LI class="nav-header">Outlook</LI><LI><A href="/Team:TU-Eindhoven/FutureProspects">Future Steps</A></LI><LI class="nav-header">Others</LI><LI><A href="/Team:TU-Eindhoven/Attributions">Attributions</A></LI><LI><A href="/Team:TU-Eindhoven/Collaboration">Collaboration</A></LI><LI><A href="/Team:TU-Eindhoven/Achievements">Achievements</A></LI></UL><LI id="tue-menu-wetlab" class="dropdown"><A class="dropdown-toggle" data-toggle="dropdown" href="#">Wetlab </A></LI><LI><A href="/Team:TU-Eindhoven/Wetlab">Overview</A></LI><LI><A href="/Team:TU-Eindhoven/Preparation">Preparation</A></LI><LI class="nav-header">Lab Reports</LI><LI><A href="/Team:TU-Eindhoven/MRIProcessing">MRI Data Processing</A></LI><LI><A href="/Team:TU-Eindhoven/AnearobicTesting">Anaerobic FNR Testing</A></LI><LI><A href="/Team:TU-Eindhoven/CESTAgentTesting">CEST Agent Testing</A></LI><LI><A href="/Team:TU-Eindhoven/LabJournal">Lab Journal</A></LI><LI id="tue-menu-drylab" class="dropdown"><A class="dropdown-toggle" data-toggle="dropdown" href="#">Drylab </A></LI><LI><A href="/Team:TU-Eindhoven/Modeling">Summary</A></LI><LI><A href="/Team:TU-Eindhoven/ProteinSelection">Protein Selection</A></LI><LI class="nav-header">Contrast Mechanism</LI><LI><A href="/Team:TU-Eindhoven/StochasticModel">Decoy Sites</A></LI><LI><A href="/Team:TU-Eindhoven/ODEModel">FNR Dynamics</A></LI><LI><A href="/Team:TU-Eindhoven/FNRPromotor">FNR Promotor</A></LI><LI><A href="/Team:TU-Eindhoven/IntegralModel">Integral Model</A></LI><LI class="nav-header">Bacteria in the Body</LI><LI><A href="/Team:TU-Eindhoven/PBPK">Ganciclovir Distribution</A></LI><LI><A href="/Team:TU-Eindhoven/TimeKill">Time-kill Behavior</A></LI><LI id="tue-menu-safety" class="dropdown"><A class="dropdown-toggle" data-toggle="dropdown" href="#">Safety </A></LI><LI><A href="/Team:TU-Eindhoven/Safety">Overview</A></LI><LI><A href="/Team:TU-Eindhoven/LabSafety">Lab Safety</A></LI><LI><A href="/Team:TU-Eindhoven/PublicSafety">Public and Environmental Safety</A></LI><LI><A href="/Team:TU-Eindhoven/SafetyiGEM">Biosafety Within iGEM</A></LI><LI id="tue-menu-society" class="dropdown"><A class="dropdown-toggle" data-toggle="dropdown" href="#">Society </A></LI><LI><A href="/Team:TU-Eindhoven/Outreach">Outreach</A></LI><LI><A href="/Team:TU-Eindhoven/Society">Human Practice</A></LI><LI id="tue-menu-notebook"><A href="/Team:TU-Eindhoven/Notebook">Notebook</A></LI></DIV></DIV></DIV></DIV><DIV class="tue-content container"><TABLE id="toc" class="toc"><TBODY><TR><TD><DIV id="toctitle"><H2>Contents</H2></DIV><UL><LI class="toclevel-1 tocsection-1"><A href="#MRI_Data_Processing"><SPAN class="tocnumber">1</SPAN><SPAN class="toctext">MRI Data Processing</SPAN></A></LI><LI class="toclevel-2 tocsection-2"><A href="#The_Principle"><SPAN class="tocnumber">1.1</SPAN><SPAN class="toctext">The Principle</SPAN></A></LI><LI class="toclevel-2 tocsection-3"><A href="#Methods"><SPAN class="tocnumber">1.2</SPAN><SPAN class="toctext">Methods</SPAN></A></LI><LI class="toclevel-2 tocsection-4"><A href="#Processing"><SPAN class="tocnumber">1.3</SPAN><SPAN class="toctext">Processing</SPAN></A></LI><LI class="toclevel-2 tocsection-5"><A href="#Results"><SPAN class="tocnumber">1.4</SPAN><SPAN class="toctext">Results</SPAN></A></LI><LI class="toclevel-2 tocsection-6"><A href="#Conclusion"><SPAN class="tocnumber">1.5</SPAN><SPAN class="toctext">Conclusion</SPAN></A></LI><LI class="toclevel-2 tocsection-7"><A href="#References"><SPAN class="tocnumber">1.6</SPAN><SPAN class="toctext">References</SPAN></A></LI></UL></TD></TR></TBODY></TABLE><H1><SPAN class="mw-headline" id="MRI_Data_Processing">MRI Data Processing</SPAN></H1><P class="lead">Until recently, the most common way of generating MRI contrast was to use heavy metal based contrast agents. Examples of these are the commercially available Gadolinium(III) based Magnevist and the Iron Oxide based Resovist. A downside of using heavy metals is the toxicity of the agents in the human body<SPAN class="tue-reference"><SPAN class="tue-refid">GNSF</SPAN>T Grobner
and FC Prischl, Gadolinium and nephrogenic systemic fibrosis. <I>Kidney International</I><B>72</B>, 260–264 (2007)</SPAN><SPAN class="tue-reference"><SPAN class="tue-refid">Gsrf34</SPAN>Ihsan Ergün, Kenan Keven, et all., The safety of gadolinium in patients with stage 3 and 4 renal failure. <I>Nephrology Dialysis Transplantation</I><B>21</B>, 697-700 (2006)</SPAN><SPAN class="tue-reference"><SPAN class="tue-refid">Gint</SPAN>I. Buhaescu and H. Izzedine, Gadolinium-induced nephrotoxicity. <I>The International Journal of Clinical Practice</I><B>62</B>, 1113-1118 (2008)</SPAN>. Fortunately, a new type of MRI has been developed, which allows for the use of organic compounds as contrast agent.</P><H2><SPAN class="mw-headline" id="The_Principle">The Principle</SPAN></H2><DIV class="tue-float tue-float-right span4"><DIV class="tue-caption"><SPAN class="tue-caption-figure"><SPAN class="tue-refid">CESTExplaination</SPAN></SPAN> <SPAN class="tue-caption-text">a) and b) Solute protons are saturated at their specific resonance frequency in the proton spectrum, this saturation is then transferred to water. c) Measurement of normalized water saturation, the Z-spectrum. d) result of the asymmetry analysis of the Z-spectrum. <SPAN class="tue-refagain" id="tue-refagain-vanZijlCEST">vanZijlCEST</SPAN></SPAN></DIV></DIV>
This new type of MRI goes by the name Chemical Exchange Saturation Transfer MRI or CEST MRI<SPAN class="tue-reference"><SPAN class="tue-refid">vanZijlCEST</SPAN>van Zijl, Peter, and Nirbhay N. Yadav., Chemical exchange saturation transfer (CEST): what is in a name and what isn't?.. <I>Magnetic Resonance in Medicine</I><B>65.4</B>, 8927-948 (2011)</SPAN><SPAN class="tue-reference"><SPAN class="tue-refid">LiuCEST</SPAN>Liu, Guanshu, et al., Nuts and bolts of chemical exchange saturation transfer MRI.. <I>NMR in Biomedicine</I> ,  (2013)</SPAN><SPAN class="tue-reference"><SPAN class="tue-refid">GiladGene</SPAN>Gilad, Assaf A., et al., MRI reporter genes.. <I>Journal of Nuclear Medicine</I><B>49.12</B>, 1905-1908 (2008)</SPAN>. The principle behind this technique is based on compounds that contain pools of exchangeable protons that can be selectively saturated using radio frequency irradiation. Upon proton exchange with bulk water, these compounds can be indirectly visualized by measuring the bulk water. The amino acids Lysine, Arginine, Threonine and Serine contain these exchangeable protons and polypeptides containing those amino acids in abundance are therefore potential contrast agents <SPAN class="tue-reference"><SPAN class="tue-refid">GoffeneyCEST</SPAN>Goffeney, Nicholas, et al., Sensitive NMR detection of cationic-polymer-based gene delivery systems using saturation transfer via proton exchange.. <I>Journal of the American Chemical Society</I><B>123.35</B>, 8628-8629 (2001)</SPAN><SPAN class="tue-reference"><SPAN class="tue-refid">GiladCEST</SPAN>Gilad, Assaf A., et al., Artificial reporter gene providing MRI contrast based on proton exchange.. <I>Nature biotechnology</I><B>25.2</B>, 217-219 (2007)</SPAN>. Another advantage of CEST MRI is that different kinds of protons (e.g. amide and hydroxide protons) can be distinguished, which allows for 'multi-color' imaging when using multiple contrast agents simultaneously <SPAN class="tue-reference"><SPAN class="tue-refid">McMahonMulticolor</SPAN>McMahon, Michael T., et al., New 'multicolor' polypeptide diamagnetic chemical exchange saturation transfer (DIACEST) contrast agents for MRI.. <I>Magnetic Resonance in Medicine</I><B>60.4</B>, 803-812 (2008)</SPAN>.
<H2><SPAN class="mw-headline" id="Methods">Methods</SPAN></H2><P>This theory can be used to scan our library of potential contrast agents (<A href="/Team:TU-Eindhoven/Production" title="Team:TU-Eindhoven/Production">Protein Selection</A>). Hereto the proteins were (aerobically) over expressed in BL21 using a pET28a vector with a T7 promoter. The bacteria were spun down and fixed in PFA. The entire pellet (bacteria containing our proteins) was then measured in a 7 Tesla Bruker MRI machine. First, the correct water frequency was determined, the machine was shimmed, i.e. a homogeneous magnetic field was created. The first measurement was a T<SUB>2</SUB> weighted image for general orientation. Subsequently local shimming was performed on each of the separate pellets. For the final measurements, the saturation pulse was set to vary from $\pm -4 ppm$ to $\pm +4 ppm$ (relative to water), the measurements were averaged over 8 separate scans. Also a $S_{0}$ (without saturation pulse) image was taken.
</P><H2><SPAN class="mw-headline" id="Processing">Processing</SPAN></H2><P>To detect the CEST effect of our proteins a number of processing steps are required. First of all, the imaging should be taken at different frequencies of the saturation pulse. This way the signal at different chemical shifts can be measured. When bulk water is set to 0 ppm, the signal plot will be roughly symmetric around zero, since saturation pulses will excite the bulk water in a similar way when the absolute shift is equal. This also implies that when the signals with a negative shift from the bulk water are compared with the signals with a positive shift, asymmetry can be detected. The CEST effect is precisely what causes this asymmetry in the signal plot, due to the chemical shift of the CEST compounds being different from the chemical shift of water. Therefore, when the absolute shift from the bulk water of two saturation pulses is equal, this does not mean the absolute shift from the CEST compound is equal and the CEST protons will in one case hardly saturate at all  (depending on its shift from bulk water of course).
</P><DIV class="tue-float tue-float-left span8"><DIV>
$$\begin{aligned}
MTR_{ asym} &amp;= {{S_{w}^{-\Delta\omega_{sw}} - S_{w}^{-\Delta\omega_{sw}}}\over{S_0}} \\
PTE &amp;= {{2 \cdot [H_{2}O]}\over{[polypeptide]}} \cdot MTR_{asym}
\end{aligned}
$$

</DIV><DIV class="tue-caption"><SPAN class="tue-caption-figure"><SPAN class="tue-refid">MTRasymEquations</SPAN></SPAN> <SPAN class="tue-caption-text">Equations for analyzing the CEST MRI data.</SPAN></DIV></DIV><P> </P><P>Putting this principle into practice requires some processing steps. First of all, the asymmetric comparison should be conducted as shown in . Where the MTR<SUB>asym</SUB> is the asymmetry in the z-spectrum. Furthermore, $S_{0}$ and $S_{w}^{\pm\Delta\omega_{sw}}$ are, respectively, the water signals without saturation and with saturation at $\Delta\omega_{sw}$ from bulk water. The PTE is the Proton Transfer Enhancement, which gives an indication of the sensitivity of an agent, independent of the concentration. In this research, the concentration was not measured accurately, but the mass of cell pellet was aimed to be kept constant, so the MTR<SUB>asym</SUB> plots can still be compared quite well.
</P><P>To eliminate the background signal of other Lysine and Arginine rich proteins, the asymmetric plots of the potential agents were compared to the asymmetric plots of a control sample, where no protein was over-expressed. In practice this can be compared to using an MRI image of human tissue without contrast agent as control for an image of tissue with contrast agent. This comparison was done in two different ways, one option results in the absolute difference compared to the control sample. The other method is relatively comparing the samples ().
</P><DIV class="tue-float tue-float-left span12"><DIV>
$$\begin{aligned}
C_{abs} &amp;= MTR_{ asym}^{ agent} - MTR_{ asym}^{ control} \\
C_{rel} &amp;= {{MTR_{ asym}^{ agent}}\over{MTR_{ asym}^{ control}}} - 1
\end{aligned}
$$

</DIV><DIV class="tue-caption"><SPAN class="tue-caption-figure"><SPAN class="tue-refid">ContrastComparison</SPAN></SPAN> <SPAN class="tue-caption-text">Comparison of agent samples and the control sample.</SPAN></DIV></DIV><H2><SPAN class="mw-headline" id="Results">Results</SPAN></H2><P>When the previously described processing steps are applied to the MRI results of this research, the following graphs can be acquired.  shows the MTR<SUB>asym</SUB>, $C_{abs}$ (Absolute Difference) and $C_{rel}$ (Relative Difference) graphs for all potential contrast agents that were tested.
</P><DIV class="tue-float tue-float-left span12"><DIV class="tue-caption"><SPAN class="tue-caption-figure"><SPAN class="tue-refid">AllMRIGraphs</SPAN></SPAN> <SPAN class="tue-caption-text">The MTR<SUB>asym</SUB>, absolute contrast and relative contrast of all potential CEST contrast agents.</SPAN></DIV></DIV>
Lysine protons are predicted to show contrast at a shift of $\pm 3.7 ppm$ whereas arginine should produce contrast at $\pm 2.0 ppm$. Using this information it can be readily seen that <I>1PJN</I> (<A href="http://parts.igem.org/Part:BBa_K1123015" class="external text" rel="nofollow">BBa_K1123015</A>) and <I>Poly(KS)</I> (<A href="http://parts.igem.org/Part:BBa_K1123021" class="external text" rel="nofollow">BBa_K1123021</A>) both exhibit (Lysine-)CEST contrast. It should be noted that for more reliable results, more control samples should be taken into account. Furthermore the determination of the concentration of the agents should be carried out more precisely. However, due to the strange nature of the proteins, this has shown to be rather difficult in practice. But even using one control sample, quite clear results were achieved for the Lysine contrast.  shows the calculated contrasts (both absolute and relative) and the Lysine ratio of the agents, which gives an indication of how much contrast to expect. Here the agents with a high Lysine ratio express high Lysine contrast, which comes across with our expectations. However, the high contrast of Poly(TK) could be caused by other factors, since the contrast graphs in  do not show clear peaks at $\pm 3.7 ppm$. For Arginine contrast, it is not that simple. The peak is expected at $\pm 2.0 ppm$, which is considerably closer to the frequency of bulk water and is therefore more sensitive to noise. The same analysis as for Lysine is shown in , but here no clear correlation can be seen. However, looking at the graphs in  some peaks can be seen around 2.0 ppm, but no good conclusions can be drawn, since the Arginine protons can not be satisfactory distinguished from the rest of the protons.
<DIV class="tue-float tue-float-left span6"><DIV class="tue-caption"><SPAN class="tue-caption-figure"><SPAN class="tue-refid">LysineContrastPlot</SPAN></SPAN> <SPAN class="tue-caption-text">Lysine contrast of all tested contrast agents.</SPAN></DIV></DIV><DIV class="tue-float tue-float-right span6"><DIV class="tue-caption"><SPAN class="tue-caption-figure"><SPAN class="tue-refid">ArginineContrastPlot</SPAN></SPAN> <SPAN class="tue-caption-text">Arginine contrast of all tested contrast agents.</SPAN></DIV></DIV><P> </P><H2><SPAN class="mw-headline" id="Conclusion">Conclusion</SPAN></H2><P>All in all, the most promising candidate agent seems to be <I>1PJN</I> (<A href="http://parts.igem.org/Part:BBa_K1123015" class="external text" rel="nofollow">BBa_K1123015</A>). The lysine peak of this agent is clearly distinguishable from the rest of the signal. As a Arginine based compound, no clear results have been achieved and therefore no recommendation can be given, other than that the Arginine based agents are less suitable for generating contrast in general. It is therefore our recommendation to use lysine based agents, such as <I>1PJN</I>. Only for multicolor MRI or imaging lysine rich areas can Arginine agents be useful. Another conclusion that can be drawn from the MRI results, is that Poly(KS) has actually been expressed in the bacteria. This is contradictory to the other lab results, where no traces of Poly(KS) on SDS-Page gels were found. The most obvious explanation for these results, is that only a very small part of the Poly(KS) chain was created, which is to be expected due to the largely repetitive and non-optimal DNA of the construct.
</P><H2><SPAN class="mw-headline" id="References">References</SPAN></H2></DIV><DIV class="tue-gototop-sticky"><A href="#top" title="Go to the top of the page">Go to Top   </A></DIV><DIV class="printfooter">
Retrieved from &quot;<A href="http://2013.igem.org/Team:TU-Eindhoven/MRIProcessing">http://2013.igem.org/Team:TU-Eindhoven/MRIProcessing</A>&quot;</DIV></DIV></DIV><DIV id="footer-box" class="noprint"><DIV id="footer"><UL id="f-list"><LI id="t-recentchanges"><A href="/Special:RecentChanges" title="Recent changes">Recent changes</A></LI><LI id="t-whatlinkshere"><A href="/Special:WhatLinksHere/Team:TU-Eindhoven/MRIProcessing" title="List of all wiki pages that link here [j]" accesskey="j">What links here</A></LI><LI id="t-recentchangeslinked"><A href="/Special:RecentChangesLinked/Team:TU-Eindhoven/MRIProcessing" title="Recent changes in pages linked from this page [k]" accesskey="k">Related changes</A></LI><LI id="t-specialpages"><A href="/Special:SpecialPages" title="List of all special pages [q]" accesskey="q">Special pages</A></LI><LI><A href="/Special:Preferences">My preferences</A></LI></UL></DIV><DIV id="footer"><UL id="f-list"><LI id="t-print"><A href="/wiki/index.php?title=Team:TU-Eindhoven/MRIProcessing&amp;printable=yes" title="Printable version of this page [p]" accesskey="p">Printable version</A></LI><LI id="t-permalink"><A href="/wiki/index.php?title=Team:TU-Eindhoven/MRIProcessing&amp;oldid=331307" title="Permanent link to this revision of the page">Permanent link</A></LI><LI id="privacy"><A href="/2013.igem.org:Privacy_policy" title="2013.igem.org:Privacy policy">Privacy policy</A></LI><LI id="disclaimer"><A href="/2013.igem.org:General_disclaimer" title="2013.igem.org:General disclaimer">Disclaimers</A></LI></UL></DIV></DIV></DIV></BODY></HTML>