"
Page
Discussion
View source
History
teams
Log in
 
Team:TU-Eindhoven/CESTAgentTesting
From 2013.igem.org
Home
Team
Project
Abstract
Project Description
Background
The Chassis
Potential Contrast Agents
Kill Switch
Applications
Results
Main Results
Parts Submitted
Outlook
Future Steps
Others
Attributions
Collaboration
Achievements
Wetlab
Overview
Preparation
Lab Reports
MRI Data Processing
Anaerobic FNR Testing
CEST Agent Testing
Lab Journal
Drylab
Summary
Protein Selection
Contrast Mechanism
Decoy Sites
FNR Dynamics
FNR Promotor
Integral Model
Bacteria in the Body
Ganciclovir Distribution
Time-kill Behavior
Safety
Overview
Lab Safety
Public and Environmental Safety
Biosafety Within iGEM
Society
Outreach
Human Practice
Notebook
Contents
1 CEST Agent Testing
1.1 Abstract
1.2 Introduction
1.3 Methods and Results
1.3.1 Cloning
1.3.2 Protein Expression and Sampling for CEST Measurements
1.3.3 MRI Measurements
1.4 Discussion and Conclusions
CEST Agent Testing
Abstract
Until recently, the most common way of generating MRI contrast was to use heavy metal based contrast agents. Examples of these are the commercially available Gadolinium(III) based Magnevist and the Iron Oxide based Resovist. A downside of using heavy metals is the toxicity of the agents in the human bodyGNSFT Grobner and FC Prischl, Gadolinium and nephrogenic systemic fibrosis. Kidney International 72, 260–264 (2007)Gsrf34Ihsan Ergün, Kenan Keven, et all., The safety of gadolinium in patients with stage 3 and 4 renal failure. Nephrology Dialysis Transplantation 21, 697-700 (2006)GintI. Buhaescu and H. Izzedine, Gadolinium-induced nephrotoxicity. The International Journal of Clinical Practice 62, 1113-1118 (2008). Fortunately, a new type of MRI has been developed, which allows for the use of organic compounds as contrast agent. We chose several Arginine and Lysine rich proteins for testing them for CEST contrast. Hereto the proteins were expressed aerobically in BL21 bacteria. The bacterial cell pellets were fixed using PFA and then tested in a 7 Tesla Bruker MRI machine. From these experiments it was concluded that the most promising candidate for Lysine based CEST contrast agents is 1PJN, of which the lysine peak was clearly distinguishable from the rest of the signal. No clear results have been achieved for Arginine based CEST contrast agents.
Introduction
Initially, a total of 10 constructs with proteins were designed, all containing a high percentage of Lysine and/or Arginine, which have a chemical shifts that are suitable for CEST imaging. Four of them were naturally occurring proteins, native to either E. coli or to other organisms, in which case the protein sequence was optimized to match codon prevalence in E. coli. These naturally occurring proteins were 1G70, 1ETF, 1PJN Chain-1 and Human Protamine 1 (HPM1, optimized). The other five were non-natural polypeptides consisting of a two repeating dipeptides, namely poly(Arginine-Glycine) (P(RG)), poly(lysine-serine) (P(KS)), poly(arginine-serine) (P(RS))and poly(threonine-lysine) (P(TK)).  Like HPM1, these we E. coli optimized. Additionally, an EGFP construct would also be inserted and expressed as a positive control. The 1G70, 1ETF and 1PJN chains that were selected are quite short and would therefore be hard to detect in the lab, so they were repeated until a satisfactory size was reached.
symbolicpBRdesignpet Symbolic design of the pET28a construct
All the protein sequences were cut out of the ordered pUC57-simple vectors by NheI and XhoI restriction enzymes and ligated into pET28a vectors. As a result, all constructs shared the same lay-out, starting with prescribed iGEM restriction sites EcoRI, NotI and XbaI, followed by an T7 promoter site and ribosome binding site. Hereafter a poly-histidine tag (for protein purification), thrombin cleavage site and NheI restriction site (to isolate the protein sequence for ligation into a pET28a vector) are placed. Subsequently, there is the the variable part, the protein coding sequence. Then an XhoI restriction site and random spacer sequence follow, positioning the terminator at 30 base pairs from the stop codon. After the terminator the three prescribed iGEM end restriction sites, SpeI, NotI, PstI, plus a HindIII restriction site are positioned.
Methods and Results
Cloning
Once the pUC57-simple constructs were delivered, they were transformed into NB bacteria for amplification of the these vectors. When they were amplified, a digestion on the constructs as well as on the pET28a vector was performed using NheI and XhoI restriction enzymes to cut out just the protein sequence of the pUC57-simple construct. pET28a was chosen as vector for overexpression since it has a promoter that can be induced by IPTG which is helpful and the vector was readily available. Once the pUC57-simple constructs as well as pET28a vector were digested, a gel extraction was performed to single out the desired inserts.  Subsequently a ligation was performed to get the protein sequence from the pUC57-simple constructs into the pET28a vector. After ligation the constructs were plated with kanamycin antibiotics and grown overnight to create colonies. To check whether ligation had succeeded a colony PCR was performed. Primers used in the ligation of pET28a vectors were the T7 Forward and T7 Reverse (terminator) primers.
After completing the PCR protocol, an agarose gel electrophoresis was executed on the products and visualized using UV spectrometry to see whether the ligation was successful or not.
colpcrpet Colony PCR of the ligated constructs into pET28a
Except for poly(Arginine-Serine), all constructs yielded successfully ligated products for further processing. For each protein sequence, two of the ligation products were selected based on the intensity and the accuracy of the weights of the appearing bands.   Then the chosen constructs were transformed into NB and put into small culture tubes, again with kanamycin, and grown overnight. To retain the vectors, a miniprep procedure was performed and once the constructs were sequenced and seemed to be correct, they were transformed into BL21 bacteria for protein expression.
Protein Expression and Sampling for CEST Measurements
Large cultures with the BL21 bacteria, containing our constructs, were set in and were grown until the optical density had reached a value of circa 0.6. At that point in time, protein expression was induced by adding IPTG to induce the promoter and protein expression was then performed overnight at 37°C.
For protein retention a Bugbuster protocol was executed. Once pellets had formed at the end of the Bugbuster protocol, samples of the pellets were taken for CEST measurements. This was done by taking about 300-400 mg of the pellet (the weight should be equal for all samples to make comparison of the CEST measurements possible) and the pellets were placed into a PCR tube and the cells were fixated with 4% PFA (paraformaldehyde) and placed in the refrigerator at 4°C for at least two days.
SDSpellet 12% SDS gel analysis of pellets of the samples taken during aerobic expression
SDSsupernatant 12% SDS gel analysis of supernatant of the samples taken during aerobic expression
Since on and no protein expression is observed, Protamine was too small to be shown on the gel (8.9kDa) and for P(RG) nothing indicates that P(RG) is expressed (expected height at 17.6 kDa).
After expression of 1ETF and 1PJN, a Nickel-column purification was performed (for all proteins) to purify the proteins (since they have an His-tag). Samples were taken from each step (load, wash and elution) for further analysis (SDS gel analysis and Bradford assay), after rebuffering to TRIS buffer, since imidazole can cause protein denaturation.
SDSexpression1ETF1PJN 12% SDS gel analysis of 1ETF and 1PJN
SDSexpressionallproteins 12% SDS gel analysis: Supernatant and pellets after protein expression of several proteins
In there was an indication that the expressed proteins were caught in inclusion bodies. This could not be purified by the Nickel-column, so the proteins had to be expressed another time to perform a inclusion body exctraction protocol. The gel depicted in provided the information that all native proteins were caught in inclusion bodies. The non-natural proteins did not show any indications of expression. So we took 1ETF and 1G70 and performed an inclusion body extraction protocol.
SDSgel1ETF1PJNinclusionbody Inclusion body purification of 1ETF and 1PJN
In it can be seen that 1G70 is expressed. By this observation we know that some proteins are in inclusion bodies and thus will be in the cell pellets. So the samples used for the MRI measurements may contain the expressed proteins, even though there is no indication for that in the first SDS-PAGE gels.
MRI Measurements
The entire pellet (bacteria containing our proteins) was then measured in a 7 Tesla Bruker MRI machine. First, the correct water frequency was determined, the machine was shimmed, i.e. a homogeneous magnetic field was created. The first measurement was a T2 weighted image for general orientation. Subsequently local shimming was performed on each of the separate pellets. For the final measurements, the saturation pulse was set to vary from $\pm -4 ppm$ to $\pm +4 ppm$ (relative to water), the measurements were averaged over 8 separate scans. Also a $S_{0}$ (without saturation pulse) image was taken. For more information about the theory of CEST Imaging, see MRI Processing .
Discussion and Conclusions
All in all, the most promising candidate agent seems to be 1PJN ( BBa_K1123015 ). The Lysine peak of this agent is clearly distinguishable from the rest of the signal. As for Arginine based compounds no clear results have been achieved and therefore no recommendation can be given, other than that the Arginine based agents are less suitable for generating contrast in general. It is therefore our recommendation to use Lysine based agents, such as 1PJN. Only for multicolor MRI or imaging Lysine rich areas can Arginine agents be useful. Another conclusion that can be drawn from the MRI results, is that Poly(KS) has actually been expressed in the bacteria. This is contradictory to the other lab results, where no traces of Poly(KS) on SDS-Page gels were found. The most obvious explanation for these results, is that only a very small part of the Poly(KS) chain was created, which is to be expected due to the largely repetitive and non-optimal DNA of the construct.
Go to Top   
Retrieved from " http://2013.igem.org/Team:TU-Eindhoven/CESTAgentTesting "
Recent changes
What links here
Related changes
Special pages
My preferences
Printable version
Permanent link
Privacy policy
Disclaimers
