<HTML xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en" dir="ltr">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki  ltr ns-0 ns-subject page-Team_TU-Eindhoven_LabJournal"><DIV id="globalWrapper"><DIV id="top-section"><DIV id="p-logo"><A href="/Main_Page" title="Main Page">&quot;
	    </A></DIV><DIV id="menubar" class="left-menu noprint"><UL><LI class="selected"><A href="/Team:TU-Eindhoven/LabJournal">Page               </A></LI><LI class="new"><A href="/wiki/index.php?title=Talk:Team:TU-Eindhoven/LabJournal&amp;action=edit&amp;redlink=1">Discussion               </A></LI><LI><A href="/wiki/index.php?title=Team:TU-Eindhoven/LabJournal&amp;action=edit">View source               </A></LI><LI><A href="/wiki/index.php?title=Team:TU-Eindhoven/LabJournal&amp;action=history">History               </A></LI><LI style="color:white;cursor:default">teams</LI></UL></DIV><DIV class="right-menu noprint" id="menubar"><UL><LI id="pt-login"><A href="/wiki/index.php?title=Special:UserLogin&amp;returnto=Team:TU-Eindhoven/LabJournal" title="You are encouraged to log in; however, it is not mandatory [o]" accesskey="o">Log in</A></LI></UL></DIV><DIV id="search-controls" class="noprint"><FORM action="/Special:Search" id="searchform"> </FORM></DIV></DIV><DIV id="content"><H1 class="firstHeading">Team:TU-Eindhoven/LabJournal</H1><DIV id="bodyContent"><H3 id="siteSub" class="noprint">From 2013.igem.org</H3><DIV class="navwrapper"><DIV class="container"><DIV class="navbar"><DIV class="navbar-inner"><UL class="nav"><LI id="tue-menu-home"><A href="/Team:TU-Eindhoven">Home</A></LI><LI id="tue-menu-team"><A href="/Team:TU-Eindhoven/Team">Team</A></LI><LI id="tue-menu-project" class="dropdown"><A class="dropdown-toggle" data-toggle="dropdown" href="#">Project </A></LI><LI><A href="/Team:TU-Eindhoven/Project">Abstract</A></LI><LI><A href="/Team:TU-Eindhoven/Description">Project Description</A></LI><LI><A href="/Team:TU-Eindhoven/Background">Background</A></LI><LI><A href="/Team:TU-Eindhoven/Chasis">The Chassis</A></LI><LI><A href="/Team:TU-Eindhoven/Production">Potential Contrast Agents</A></LI><LI><A href="/Team:TU-Eindhoven/KillingMechanism">Kill Switch</A></LI><LI><A href="/Team:TU-Eindhoven/Applications">Applications</A></LI><LI class="nav-header">Results</LI><LI><A href="/Team:TU-Eindhoven/Results">Main Results</A></LI><LI><A href="/Team:TU-Eindhoven/Parts">Parts Submitted</A></LI><LI class="nav-header">Outlook</LI><LI><A href="/Team:TU-Eindhoven/FutureProspects">Future Steps</A></LI><LI class="nav-header">Others</LI><LI><A href="/Team:TU-Eindhoven/Attributions">Attributions</A></LI><LI><A href="/Team:TU-Eindhoven/Collaboration">Collaboration</A></LI><LI><A href="/Team:TU-Eindhoven/Achievements">Achievements</A></LI></UL><LI id="tue-menu-wetlab" class="dropdown"><A class="dropdown-toggle" data-toggle="dropdown" href="#">Wetlab </A></LI><LI><A href="/Team:TU-Eindhoven/Wetlab">Overview</A></LI><LI><A href="/Team:TU-Eindhoven/Preparation">Preparation</A></LI><LI class="nav-header">Lab Reports</LI><LI><A href="/Team:TU-Eindhoven/MRIProcessing">MRI Data Processing</A></LI><LI><A href="/Team:TU-Eindhoven/AnearobicTesting">Anaerobic FNR Testing</A></LI><LI><A href="/Team:TU-Eindhoven/CESTAgentTesting">CEST Agent Testing</A></LI><LI><A href="/Team:TU-Eindhoven/LabJournal">Lab Journal</A></LI><LI id="tue-menu-drylab" class="dropdown"><A class="dropdown-toggle" data-toggle="dropdown" href="#">Drylab </A></LI><LI><A href="/Team:TU-Eindhoven/Modeling">Summary</A></LI><LI><A href="/Team:TU-Eindhoven/ProteinSelection">Protein Selection</A></LI><LI class="nav-header">Contrast Mechanism</LI><LI><A href="/Team:TU-Eindhoven/StochasticModel">Decoy Sites</A></LI><LI><A href="/Team:TU-Eindhoven/ODEModel">FNR Dynamics</A></LI><LI><A href="/Team:TU-Eindhoven/FNRPromotor">FNR Promotor</A></LI><LI><A href="/Team:TU-Eindhoven/IntegralModel">Integral Model</A></LI><LI class="nav-header">Bacteria in the Body</LI><LI><A href="/Team:TU-Eindhoven/PBPK">Ganciclovir Distribution</A></LI><LI><A href="/Team:TU-Eindhoven/TimeKill">Time-kill Behavior</A></LI><LI id="tue-menu-safety" class="dropdown"><A class="dropdown-toggle" data-toggle="dropdown" href="#">Safety </A></LI><LI><A href="/Team:TU-Eindhoven/Safety">Overview</A></LI><LI><A href="/Team:TU-Eindhoven/LabSafety">Lab Safety</A></LI><LI><A href="/Team:TU-Eindhoven/PublicSafety">Public and Environmental Safety</A></LI><LI><A href="/Team:TU-Eindhoven/SafetyiGEM">Biosafety Within iGEM</A></LI><LI id="tue-menu-society" class="dropdown"><A class="dropdown-toggle" data-toggle="dropdown" href="#">Society </A></LI><LI><A href="/Team:TU-Eindhoven/Outreach">Outreach</A></LI><LI><A href="/Team:TU-Eindhoven/Society">Human Practice</A></LI><LI id="tue-menu-notebook"><A href="/Team:TU-Eindhoven/Notebook">Notebook</A></LI></DIV></DIV></DIV></DIV><DIV class="tue-content container"><P><UL class="nav nav-tabs" id="weekTabs"><LI><A href="#Introduction">Introduction </A></LI><LI><A href="#Week1">Week 1</A></LI><LI><A href="#Week2">Week 2</A></LI><LI><A href="#Week3">Week 3</A></LI><LI><A href="#Week4">Week 4</A></LI><LI><A href="#Week5">Week 5</A></LI><LI><A href="#Week6">Week 6</A></LI><LI><A href="#Week7">Week 7</A></LI><LI><A href="#Week8">Week 8</A></LI><LI><A href="#Week9">Week 9</A></LI><LI><A href="#Week10">Week 10</A></LI><LI><A href="#Week11">Week 11</A></LI><LI><A href="#Week12">Week 12</A></LI><LI><A href="#Week13">Week 13</A></LI></UL></P><DIV class="tab-content"><DIV class="tab-pane" id="Introduction"><DIV><DIV class="tue-timeline-top"><DIV class="tue-timeline-title"><H2>Introduction </H2></DIV></DIV><DIV class="tue-timeline-middle"><DIV class="tue-timeline-content">
As part of the iGEM competition, it is of course necessary for us to get our hands dirty and spend some time in the lab. In doing this we perform a multitude of different tasks, each of which is performed according to a very specific protocol where even the smallest adaptations can have drastic effects. 
<P>In the following pages of this lab journal we outline, in great detail, each of the steps we have performed during our time in the lab. We realise that the explanation of certain procedures could be considered as overkill, however we feel that a well kept and comprehensive lab journal is an invaluable asset when reviewing a project. Not only does the lab journal hold the results of all lab work completed, it can also reveal where and when certain procedures went wrong. 
</P><P>On top of this a well kept lab journal can also be used as reference material during future projects, or in the later stages of the iGEM competition. It holds a vast amount of information that can be used by yourself and others for many years to come. Finally it should also be noted that should other groups wish to replicate our results or improve upon our work, then it is also a necessity for these groups to have unlimited access to our work. 
</P><P>For these reasons, and more, we decided to write our lab journal in this way and we hope that others can appreciate this. 
</P><P>Enjoy!
</P><DIV class="row"><DIV class="tue-float tue-float-center span6"><DIV class="tue-caption"><SPAN class="tue-caption-figure"><SPAN class="tue-refid">FrontPage</SPAN></SPAN> </DIV></DIV></DIV></DIV></DIV></DIV></DIV><DIV class="tab-pane" id="Week1"><DIV><DIV class="tue-timeline-top"><DIV class="tue-timeline-title"><H2>Week 1</H2></DIV></DIV><DIV class="tue-timeline-middle"><DIV class="tue-timeline-leftbar"><DIV class="tue-timeline-node">24 July2013</DIV></DIV><DIV class="tue-timeline-content"><TABLE id="toc" class="toc"><TBODY><TR><TD><DIV id="toctitle"><H2>Contents</H2></DIV><UL><LI class="toclevel-1 tocsection-1"><A href="#Creating_agar_plates"><SPAN class="tocnumber">1</SPAN><SPAN class="toctext">Creating agar plates</SPAN></A></LI><LI class="toclevel-1 tocsection-2"><A href="#Transforming_the_DNA"><SPAN class="tocnumber">2</SPAN><SPAN class="toctext">Transforming the DNA</SPAN></A></LI><LI class="toclevel-1 tocsection-3"><A href="#Plating_the_bacteria"><SPAN class="tocnumber">3</SPAN><SPAN class="toctext">Plating the bacteria</SPAN></A></LI><LI class="toclevel-1 tocsection-4"><A href="#Making_LB_medium"><SPAN class="tocnumber">4</SPAN><SPAN class="toctext">Making LB medium</SPAN></A></LI><LI class="toclevel-1 tocsection-5"><A href="#Creating_small_cultures"><SPAN class="tocnumber">5</SPAN><SPAN class="toctext">Creating small cultures</SPAN></A></LI><LI class="toclevel-1 tocsection-6"><A href="#DNA_Retention_from_cultures"><SPAN class="tocnumber">6</SPAN><SPAN class="toctext">DNA Retention from cultures</SPAN></A></LI><LI class="toclevel-1 tocsection-7"><A href="#Nanodrop_Test"><SPAN class="tocnumber">7</SPAN><SPAN class="toctext">Nanodrop Test</SPAN></A></LI><LI class="toclevel-1 tocsection-8"><A href="#Transforming_DNA"><SPAN class="tocnumber">8</SPAN><SPAN class="toctext">Transforming DNA</SPAN></A></LI><LI class="toclevel-1 tocsection-9"><A href="#Plating_the_bacteria_2"><SPAN class="tocnumber">9</SPAN><SPAN class="toctext">Plating the bacteria</SPAN></A></LI><LI class="toclevel-1 tocsection-10"><A href="#Creating_the_Cultures"><SPAN class="tocnumber">10</SPAN><SPAN class="toctext">Creating the Cultures</SPAN></A></LI><LI class="toclevel-1 tocsection-11"><A href="#Retaining_the_DNA"><SPAN class="tocnumber">11</SPAN><SPAN class="toctext">Retaining the DNA</SPAN></A></LI><LI class="toclevel-1 tocsection-12"><A href="#Nanodrop_Test_2"><SPAN class="tocnumber">12</SPAN><SPAN class="toctext">Nanodrop Test</SPAN></A></LI><LI class="toclevel-1 tocsection-13"><A href="#Digestion"><SPAN class="tocnumber">13</SPAN><SPAN class="toctext">Digestion</SPAN></A></LI><LI class="toclevel-2 tocsection-14"><A href="#Construct_Digestion"><SPAN class="tocnumber">13.1</SPAN><SPAN class="toctext">Construct Digestion</SPAN></A></LI><LI class="toclevel-2 tocsection-15"><A href="#Vector_Digestion"><SPAN class="tocnumber">13.2</SPAN><SPAN class="toctext">Vector Digestion</SPAN></A></LI></UL><LI class="toclevel-1 tocsection-16"><A href="#PCR_Purification_for_the_Constructs"><SPAN class="tocnumber">14</SPAN><SPAN class="toctext">PCR Purification for the Constructs</SPAN></A></LI><LI class="toclevel-1 tocsection-17"><A href="#PCR_Puricfication_for_the_Vectors"><SPAN class="tocnumber">15</SPAN><SPAN class="toctext">PCR Puricfication for the Vectors</SPAN></A></LI><LI class="toclevel-1 tocsection-18"><A href="#Nanodrop_Results"><SPAN class="tocnumber">16</SPAN><SPAN class="toctext">Nanodrop Results</SPAN></A></LI><LI class="toclevel-1 tocsection-19"><A href="#Continuing_with_the_Purification"><SPAN class="tocnumber">17</SPAN><SPAN class="toctext">Continuing with the Purification</SPAN></A></LI><LI class="toclevel-2 tocsection-20"><A href="#Vector_Purification"><SPAN class="tocnumber">17.1</SPAN><SPAN class="toctext">Vector Purification</SPAN></A></LI><LI class="toclevel-2 tocsection-21"><A href="#Construct_Purification"><SPAN class="tocnumber">17.2</SPAN><SPAN class="toctext">Construct Purification</SPAN></A></LI><LI class="toclevel-1 tocsection-22"><A href="#Nanodrop_Results_2"><SPAN class="tocnumber">18</SPAN><SPAN class="toctext">Nanodrop Results</SPAN></A></LI><LI class="toclevel-1 tocsection-23"><A href="#Ligation"><SPAN class="tocnumber">19</SPAN><SPAN class="toctext">Ligation</SPAN></A></LI><LI class="toclevel-1 tocsection-24"><A href="#Preparing_Agar_Plates"><SPAN class="tocnumber">20</SPAN><SPAN class="toctext">Preparing Agar Plates</SPAN></A></LI><LI class="toclevel-1 tocsection-25"><A href="#Transformation"><SPAN class="tocnumber">21</SPAN><SPAN class="toctext">Transformation</SPAN></A></LI><LI class="toclevel-1 tocsection-26"><A href="#Plating"><SPAN class="tocnumber">22</SPAN><SPAN class="toctext">Plating</SPAN></A></LI><LI class="toclevel-1 tocsection-27"><A href="#Colony_Picking_and_PCR"><SPAN class="tocnumber">23</SPAN><SPAN class="toctext">Colony Picking and PCR</SPAN></A></LI><LI class="toclevel-1 tocsection-28"><A href="#Culturing_the_Ligation_Products"><SPAN class="tocnumber">24</SPAN><SPAN class="toctext">Culturing the Ligation Products</SPAN></A></LI><LI class="toclevel-1 tocsection-29"><A href="#Miniprep"><SPAN class="tocnumber">25</SPAN><SPAN class="toctext">Miniprep</SPAN></A></LI><LI class="toclevel-1 tocsection-30"><A href="#Nanodrop_Results_3"><SPAN class="tocnumber">26</SPAN><SPAN class="toctext">Nanodrop Results</SPAN></A></LI><LI class="toclevel-1 tocsection-31"><A href="#Vector_Sequencing"><SPAN class="tocnumber">27</SPAN><SPAN class="toctext">Vector Sequencing</SPAN></A></LI><LI class="toclevel-1 tocsection-32"><A href="#Colony_Picking_and_PCR_2"><SPAN class="tocnumber">28</SPAN><SPAN class="toctext">Colony Picking and PCR</SPAN></A></LI><LI class="toclevel-1 tocsection-33"><A href="#Transforming_and_Plating_New_Constructs"><SPAN class="tocnumber">29</SPAN><SPAN class="toctext">Transforming and Plating New Constructs</SPAN></A></LI><LI class="toclevel-1 tocsection-34"><A href="#Transforming_and_Plating_Old_Constructs"><SPAN class="tocnumber">30</SPAN><SPAN class="toctext">Transforming and Plating Old Constructs</SPAN></A></LI><LI class="toclevel-1 tocsection-35"><A href="#Creating_Small_Cultures_2"><SPAN class="tocnumber">31</SPAN><SPAN class="toctext">Creating Small Cultures</SPAN></A></LI><LI class="toclevel-1 tocsection-36"><A href="#Miniprepping_ligated_pBR322_vector_cultures"><SPAN class="tocnumber">32</SPAN><SPAN class="toctext">Miniprepping ligated pBR322 vector cultures</SPAN></A></LI><LI class="toclevel-1 tocsection-37"><A href="#Sequencing"><SPAN class="tocnumber">33</SPAN><SPAN class="toctext">Sequencing</SPAN></A></LI><LI class="toclevel-1 tocsection-38"><A href="#Culturing_New_Constructs"><SPAN class="tocnumber">34</SPAN><SPAN class="toctext">Culturing New Constructs</SPAN></A></LI><LI class="toclevel-1 tocsection-39"><A href="#Culturing_Old_Constructs"><SPAN class="tocnumber">35</SPAN><SPAN class="toctext">Culturing Old Constructs</SPAN></A></LI><LI class="toclevel-1 tocsection-40"><A href="#Miniprepping_New_Constructs"><SPAN class="tocnumber">36</SPAN><SPAN class="toctext">Miniprepping New Constructs</SPAN></A></LI><LI class="toclevel-1 tocsection-41"><A href="#Digestion_2"><SPAN class="tocnumber">37</SPAN><SPAN class="toctext">Digestion</SPAN></A></LI><LI class="toclevel-1 tocsection-42"><A href="#Testing_the_Expression"><SPAN class="tocnumber">38</SPAN><SPAN class="toctext">Testing the Expression</SPAN></A></LI><LI class="toclevel-1 tocsection-43"><A href="#Preparing_an_SDS-gel"><SPAN class="tocnumber">39</SPAN><SPAN class="toctext">Preparing an SDS-gel</SPAN></A></LI><LI class="toclevel-1 tocsection-44"><A href="#Running_the_SDS-Gel"><SPAN class="tocnumber">40</SPAN><SPAN class="toctext">Running the SDS-Gel</SPAN></A></LI><LI class="toclevel-1 tocsection-45"><A href="#Digestion_Purification"><SPAN class="tocnumber">41</SPAN><SPAN class="toctext">Digestion Purification</SPAN></A></LI><LI class="toclevel-2 tocsection-46"><A href="#Purifying_the_Vectors"><SPAN class="tocnumber">41.1</SPAN><SPAN class="toctext">Purifying the Vectors</SPAN></A></LI><LI class="toclevel-2 tocsection-47"><A href="#Purifying_the_Inserts"><SPAN class="tocnumber">41.2</SPAN><SPAN class="toctext">Purifying the Inserts</SPAN></A></LI><LI class="toclevel-1 tocsection-48"><A href="#Nanodrop_Results_4"><SPAN class="tocnumber">42</SPAN><SPAN class="toctext">Nanodrop Results</SPAN></A></LI><LI class="toclevel-1 tocsection-49"><A href="#Ligation_2"><SPAN class="tocnumber">43</SPAN><SPAN class="toctext">Ligation</SPAN></A></LI><LI class="toclevel-1 tocsection-50"><A href="#Transforming_the_Ligation_Products"><SPAN class="tocnumber">44</SPAN><SPAN class="toctext">Transforming the Ligation Products</SPAN></A></LI><LI class="toclevel-1 tocsection-51"><A href="#Plating_2"><SPAN class="tocnumber">45</SPAN><SPAN class="toctext">Plating</SPAN></A></LI><LI class="toclevel-1 tocsection-52"><A href="#Colony_Picking_and_PCR_3"><SPAN class="tocnumber">46</SPAN><SPAN class="toctext">Colony Picking and PCR</SPAN></A></LI><LI class="toclevel-1 tocsection-53"><A href="#Small_cultures"><SPAN class="tocnumber">47</SPAN><SPAN class="toctext">Small cultures</SPAN></A></LI><LI class="toclevel-1 tocsection-54"><A href="#Preparation_for_Protein_Expression"><SPAN class="tocnumber">48</SPAN><SPAN class="toctext">Preparation for Protein Expression</SPAN></A></LI><LI class="toclevel-1 tocsection-55"><A href="#Miniprepping_new_constructs_2"><SPAN class="tocnumber">49</SPAN><SPAN class="toctext">Miniprepping new constructs</SPAN></A></LI><LI class="toclevel-1 tocsection-56"><A href="#Sequencing_2"><SPAN class="tocnumber">50</SPAN><SPAN class="toctext">Sequencing</SPAN></A></LI><LI class="toclevel-1 tocsection-57"><A href="#Transformation_2"><SPAN class="tocnumber">51</SPAN><SPAN class="toctext">Transformation</SPAN></A></LI><LI class="toclevel-1 tocsection-58"><A href="#Plating_3"><SPAN class="tocnumber">52</SPAN><SPAN class="toctext">Plating</SPAN></A></LI><LI class="toclevel-1 tocsection-59"><A href="#Expression_Samples"><SPAN class="tocnumber">53</SPAN><SPAN class="toctext">Expression Samples</SPAN></A></LI><LI class="toclevel-2 tocsection-60"><A href="#Controls"><SPAN class="tocnumber">53.1</SPAN><SPAN class="toctext">Controls</SPAN></A></LI><LI class="toclevel-2 tocsection-61"><A href="#Expression_Samples_2"><SPAN class="tocnumber">53.2</SPAN><SPAN class="toctext">Expression Samples</SPAN></A></LI><LI class="toclevel-1 tocsection-62"><A href="#Buffer_Preparation"><SPAN class="tocnumber">54</SPAN><SPAN class="toctext">Buffer Preparation</SPAN></A></LI><LI class="toclevel-1 tocsection-63"><A href="#Protein_Purification"><SPAN class="tocnumber">55</SPAN><SPAN class="toctext">Protein Purification</SPAN></A></LI><LI class="toclevel-2 tocsection-64"><A href="#Protein_Retention"><SPAN class="tocnumber">55.1</SPAN><SPAN class="toctext">Protein Retention</SPAN></A></LI><LI class="toclevel-3 tocsection-65"><A href="#Preparation_for_CEST_measurements"><SPAN class="tocnumber">55.1.1</SPAN><SPAN class="toctext">Preparation for CEST measurements</SPAN></A></LI><LI class="toclevel-2 tocsection-66"><A href="#Return_to_Protein_Retention"><SPAN class="tocnumber">55.2</SPAN><SPAN class="toctext">Return to Protein Retention</SPAN></A></LI><LI class="toclevel-2 tocsection-67"><A href="#Nickel-Column_Purification"><SPAN class="tocnumber">55.3</SPAN><SPAN class="toctext">Nickel-Column Purification</SPAN></A></LI><LI class="toclevel-1 tocsection-68"><A href="#Concentration_Determination"><SPAN class="tocnumber">56</SPAN><SPAN class="toctext">Concentration Determination</SPAN></A></LI><LI class="toclevel-2 tocsection-69"><A href="#Making_the_Protein_Reagent"><SPAN class="tocnumber">56.1</SPAN><SPAN class="toctext">Making the Protein Reagent</SPAN></A></LI><LI class="toclevel-2 tocsection-70"><A href="#Making_the_BSA_standards"><SPAN class="tocnumber">56.2</SPAN><SPAN class="toctext">Making the BSA standards</SPAN></A></LI><LI class="toclevel-2 tocsection-71"><A href="#Performing_the_Bradford_Assay"><SPAN class="tocnumber">56.3</SPAN><SPAN class="toctext">Performing the Bradford Assay</SPAN></A></LI><LI class="toclevel-1 tocsection-72"><A href="#Rebuffering_the_Proteins"><SPAN class="tocnumber">57</SPAN><SPAN class="toctext">Rebuffering the Proteins</SPAN></A></LI><LI class="toclevel-1 tocsection-73"><A href="#SDS_Analysis"><SPAN class="tocnumber">58</SPAN><SPAN class="toctext">SDS Analysis</SPAN></A></LI><LI class="toclevel-1 tocsection-74"><A href="#Preparation_for_Anaerobic_Expression"><SPAN class="tocnumber">59</SPAN><SPAN class="toctext">Preparation for Anaerobic Expression</SPAN></A></LI><LI class="toclevel-2 tocsection-75"><A href="#Preparing_Culture_mediums"><SPAN class="tocnumber">59.1</SPAN><SPAN class="toctext">Preparing Culture mediums</SPAN></A></LI><LI class="toclevel-2 tocsection-76"><A href="#Preparing_Small_Cultures"><SPAN class="tocnumber">59.2</SPAN><SPAN class="toctext">Preparing Small Cultures</SPAN></A></LI><LI class="toclevel-1 tocsection-77"><A href="#MRI_Experiments"><SPAN class="tocnumber">60</SPAN><SPAN class="toctext">MRI Experiments</SPAN></A></LI><LI class="toclevel-1 tocsection-78"><A href="#Protein_Expression"><SPAN class="tocnumber">61</SPAN><SPAN class="toctext">Protein Expression</SPAN></A></LI><LI class="toclevel-1 tocsection-79"><A href="#Expression_Cultures"><SPAN class="tocnumber">62</SPAN><SPAN class="toctext">Expression Cultures</SPAN></A></LI><LI class="toclevel-1 tocsection-80"><A href="#Anaerobic_expression"><SPAN class="tocnumber">63</SPAN><SPAN class="toctext">Anaerobic expression</SPAN></A></LI><LI class="toclevel-1 tocsection-81"><A href="#Protein_Expression_2"><SPAN class="tocnumber">64</SPAN><SPAN class="toctext">Protein Expression</SPAN></A></LI><LI class="toclevel-1 tocsection-82"><A href="#Making_Small_Cultures"><SPAN class="tocnumber">65</SPAN><SPAN class="toctext">Making Small Cultures</SPAN></A></LI><LI class="toclevel-1 tocsection-83"><A href="#Preparing_the_Large_Culture_Vessel"><SPAN class="tocnumber">66</SPAN><SPAN class="toctext">Preparing the Large Culture Vessel</SPAN></A></LI><LI class="toclevel-1 tocsection-84"><A href="#Preparing_for_BioBricks"><SPAN class="tocnumber">67</SPAN><SPAN class="toctext">Preparing for BioBricks</SPAN></A></LI><LI class="toclevel-2 tocsection-85"><A href="#Digestion_3"><SPAN class="tocnumber">67.1</SPAN><SPAN class="toctext">Digestion</SPAN></A></LI><LI class="toclevel-2 tocsection-86"><A href="#PCR_Purification"><SPAN class="tocnumber">67.2</SPAN><SPAN class="toctext">PCR Purification</SPAN></A></LI><LI class="toclevel-2 tocsection-87"><A href="#Gel_Extraction"><SPAN class="tocnumber">67.3</SPAN><SPAN class="toctext">Gel Extraction</SPAN></A></LI><LI class="toclevel-2 tocsection-88"><A href="#Repeating_the_Digestion"><SPAN class="tocnumber">67.4</SPAN><SPAN class="toctext">Repeating the Digestion</SPAN></A></LI><LI class="toclevel-1 tocsection-89"><A href="#Protein_Purification_2"><SPAN class="tocnumber">68</SPAN><SPAN class="toctext">Protein Purification</SPAN></A></LI><LI class="toclevel-2 tocsection-90"><A href="#SDS_PAGE_Analysis"><SPAN class="tocnumber">68.1</SPAN><SPAN class="toctext">SDS PAGE Analysis</SPAN></A></LI><LI class="toclevel-1 tocsection-91"><A href="#Anaerobic_Expression_2"><SPAN class="tocnumber">69</SPAN><SPAN class="toctext">Anaerobic Expression</SPAN></A></LI><LI class="toclevel-1 tocsection-92"><A href="#Anaerobic_Expression_3"><SPAN class="tocnumber">70</SPAN><SPAN class="toctext">Anaerobic Expression</SPAN></A></LI><LI class="toclevel-1 tocsection-93"><A href="#Gel_Analysis"><SPAN class="tocnumber">71</SPAN><SPAN class="toctext">Gel Analysis</SPAN></A></LI><LI class="toclevel-1 tocsection-94"><A href="#Inclusion_Body_Extraction"><SPAN class="tocnumber">72</SPAN><SPAN class="toctext">Inclusion Body Extraction</SPAN></A></LI><LI class="toclevel-1 tocsection-95"><A href="#Nickel_Column_Purification"><SPAN class="tocnumber">73</SPAN><SPAN class="toctext">Nickel Column Purification</SPAN></A></LI><LI class="toclevel-1 tocsection-96"><A href="#SDS_Analysis_2"><SPAN class="tocnumber">74</SPAN><SPAN class="toctext">SDS Analysis</SPAN></A></LI><LI class="toclevel-1 tocsection-97"><A href="#Rebuffering_of_the_Proteins"><SPAN class="tocnumber">75</SPAN><SPAN class="toctext">Rebuffering of the Proteins</SPAN></A></LI><LI class="toclevel-1 tocsection-98"><A href="#Small_Cultures_2"><SPAN class="tocnumber">76</SPAN><SPAN class="toctext">Small Cultures</SPAN></A></LI><LI class="toclevel-1 tocsection-99"><A href="#Making_Agar_Plates"><SPAN class="tocnumber">77</SPAN><SPAN class="toctext">Making Agar Plates</SPAN></A></LI><LI class="toclevel-1 tocsection-100"><A href="#SDS_Analysis_3"><SPAN class="tocnumber">78</SPAN><SPAN class="toctext">SDS Analysis</SPAN></A></LI><LI class="toclevel-1 tocsection-101"><A href="#BugBuster_Protocol"><SPAN class="tocnumber">79</SPAN><SPAN class="toctext">BugBuster Protocol</SPAN></A></LI><LI class="toclevel-1 tocsection-102"><A href="#SDS_PAGE_Gel"><SPAN class="tocnumber">80</SPAN><SPAN class="toctext">SDS PAGE Gel</SPAN></A></LI><LI class="toclevel-1 tocsection-103"><A href="#Miniprepping_the_Small_Cultures"><SPAN class="tocnumber">81</SPAN><SPAN class="toctext">Miniprepping the Small Cultures</SPAN></A></LI><LI class="toclevel-1 tocsection-104"><A href="#Digesting_Samples"><SPAN class="tocnumber">82</SPAN><SPAN class="toctext">Digesting Samples</SPAN></A></LI><LI class="toclevel-1 tocsection-105"><A href="#Gel_Extraction_Purification"><SPAN class="tocnumber">83</SPAN><SPAN class="toctext">Gel Extraction Purification</SPAN></A></LI><LI class="toclevel-1 tocsection-106"><A href="#PCR_Purification_2"><SPAN class="tocnumber">84</SPAN><SPAN class="toctext">PCR Purification</SPAN></A></LI><LI class="toclevel-1 tocsection-107"><A href="#Nanodrop_Test_3"><SPAN class="tocnumber">85</SPAN><SPAN class="toctext">Nanodrop Test</SPAN></A></LI><LI class="toclevel-1 tocsection-108"><A href="#Ligation_3"><SPAN class="tocnumber">86</SPAN><SPAN class="toctext">Ligation</SPAN></A></LI><LI class="toclevel-1 tocsection-109"><A href="#Transformation_3"><SPAN class="tocnumber">87</SPAN><SPAN class="toctext">Transformation</SPAN></A></LI><LI class="toclevel-1 tocsection-110"><A href="#Plating_4"><SPAN class="tocnumber">88</SPAN><SPAN class="toctext">Plating</SPAN></A></LI><LI class="toclevel-1 tocsection-111"><A href="#Colony_Picking_and_PCR_4"><SPAN class="tocnumber">89</SPAN><SPAN class="toctext">Colony Picking and PCR</SPAN></A></LI><LI class="toclevel-1 tocsection-112"><A href="#Gel_analysis_2"><SPAN class="tocnumber">90</SPAN><SPAN class="toctext">Gel analysis</SPAN></A></LI><LI class="toclevel-1 tocsection-113"><A href="#Preparation_For_Protein_Expression_2"><SPAN class="tocnumber">91</SPAN><SPAN class="toctext">Preparation For Protein Expression</SPAN></A></LI><LI class="toclevel-1 tocsection-114"><A href="#Preparation_of_LB_Medium"><SPAN class="tocnumber">92</SPAN><SPAN class="toctext">Preparation of LB Medium</SPAN></A></LI><LI class="toclevel-1 tocsection-115"><A href="#Protein_Expression_3"><SPAN class="tocnumber">93</SPAN><SPAN class="toctext">Protein Expression</SPAN></A></LI><LI class="toclevel-1 tocsection-116"><A href="#Preparing_BioBricks"><SPAN class="tocnumber">94</SPAN><SPAN class="toctext">Preparing BioBricks</SPAN></A></LI><LI class="toclevel-1 tocsection-117"><A href="#Preparation_for_CEST_measurements_2"><SPAN class="tocnumber">95</SPAN><SPAN class="toctext">Preparation for CEST measurements</SPAN></A></LI><LI class="toclevel-1 tocsection-118"><A href="#Miniprepping"><SPAN class="tocnumber">96</SPAN><SPAN class="toctext">Miniprepping</SPAN></A></LI><LI class="toclevel-1 tocsection-119"><A href="#PCR"><SPAN class="tocnumber">97</SPAN><SPAN class="toctext">PCR</SPAN></A></LI><LI class="toclevel-1 tocsection-120"><A href="#PCR_Puricfication"><SPAN class="tocnumber">98</SPAN><SPAN class="toctext">PCR Puricfication</SPAN></A></LI><LI class="toclevel-1 tocsection-121"><A href="#Digestion_4"><SPAN class="tocnumber">99</SPAN><SPAN class="toctext">Digestion</SPAN></A></LI><LI class="toclevel-1 tocsection-122"><A href="#PCR_Purification_3"><SPAN class="tocnumber">100</SPAN><SPAN class="toctext">PCR Purification</SPAN></A></LI><LI class="toclevel-1 tocsection-123"><A href="#Gel_Extraction_2"><SPAN class="tocnumber">101</SPAN><SPAN class="toctext">Gel Extraction</SPAN></A></LI><LI class="toclevel-1 tocsection-124"><A href="#Re-digesting_1G70"><SPAN class="tocnumber">102</SPAN><SPAN class="toctext">Re-digesting  1G70</SPAN></A></LI><LI class="toclevel-1 tocsection-125"><A href="#Gel_Extraction_3"><SPAN class="tocnumber">103</SPAN><SPAN class="toctext">Gel Extraction</SPAN></A></LI><LI class="toclevel-1 tocsection-126"><A href="#Ligation_4"><SPAN class="tocnumber">104</SPAN><SPAN class="toctext">Ligation</SPAN></A></LI><LI class="toclevel-1 tocsection-127"><A href="#Transformation_and_Plating"><SPAN class="tocnumber">105</SPAN><SPAN class="toctext">Transformation and Plating</SPAN></A></LI><LI class="toclevel-1 tocsection-128"><A href="#Colony_Picking_and_PCR_5"><SPAN class="tocnumber">106</SPAN><SPAN class="toctext">Colony Picking and PCR</SPAN></A></LI><LI class="toclevel-1 tocsection-129"><A href="#Small_Cultures_3"><SPAN class="tocnumber">107</SPAN><SPAN class="toctext">Small Cultures</SPAN></A></LI><LI class="toclevel-1 tocsection-130"><A href="#Miniprep_2"><SPAN class="tocnumber">108</SPAN><SPAN class="toctext">Miniprep</SPAN></A></LI><LI class="toclevel-1 tocsection-131"><A href="#Preparing_the_Biobricks_for_shipment_to_Boston"><SPAN class="tocnumber">109</SPAN><SPAN class="toctext">Preparing the Biobricks for shipment to Boston</SPAN></A></LI><LI class="toclevel-1 tocsection-132"><A href="#Making_of_samples_for_sequencing"><SPAN class="tocnumber">110</SPAN><SPAN class="toctext">Making of samples for sequencing</SPAN></A></LI><LI class="toclevel-1 tocsection-133"><A href="#Prepare_BioBricks_for_Sending"><SPAN class="tocnumber">111</SPAN><SPAN class="toctext">Prepare BioBricks for Sending</SPAN></A></LI><LI class="toclevel-1 tocsection-134"><A href="#Culturing_Bacteria"><SPAN class="tocnumber">112</SPAN><SPAN class="toctext">Culturing Bacteria</SPAN></A></LI><LI class="toclevel-1 tocsection-135"><A href="#Prepare_Bacteria_for_Electroporation"><SPAN class="tocnumber">113</SPAN><SPAN class="toctext">Prepare Bacteria for Electroporation</SPAN></A></LI><LI class="toclevel-1 tocsection-136"><A href="#Electroporation"><SPAN class="tocnumber">114</SPAN><SPAN class="toctext">Electroporation</SPAN></A></LI><LI class="toclevel-1 tocsection-137"><A href="#Creating_small_samples"><SPAN class="tocnumber">115</SPAN><SPAN class="toctext">Creating small samples</SPAN></A></LI><LI class="toclevel-1 tocsection-138"><A href="#Preparing_LB_medium"><SPAN class="tocnumber">116</SPAN><SPAN class="toctext">Preparing LB medium</SPAN></A></LI><LI class="toclevel-1 tocsection-139"><A href="#Anaerobic_protein_expression"><SPAN class="tocnumber">117</SPAN><SPAN class="toctext">Anaerobic protein expression</SPAN></A></LI><LI class="toclevel-1 tocsection-140"><A href="#Protein_purification_3"><SPAN class="tocnumber">118</SPAN><SPAN class="toctext">Protein purification</SPAN></A></LI><LI class="toclevel-1 tocsection-141"><A href="#SDS_analysis_4"><SPAN class="tocnumber">119</SPAN><SPAN class="toctext">SDS analysis</SPAN></A></LI><LI class="toclevel-1 tocsection-142"><A href="#Protein_purification_4"><SPAN class="tocnumber">120</SPAN><SPAN class="toctext">Protein purification</SPAN></A></LI><LI class="toclevel-1 tocsection-143"><A href="#Fluorescence_analysis"><SPAN class="tocnumber">121</SPAN><SPAN class="toctext">Fluorescence analysis</SPAN></A></LI><LI class="toclevel-1 tocsection-144"><A href="#SDS_analysis_5"><SPAN class="tocnumber">122</SPAN><SPAN class="toctext">SDS analysis</SPAN></A></LI><LI class="toclevel-1 tocsection-145"><A href="#Preparing_the_anaerobic_vessel"><SPAN class="tocnumber">123</SPAN><SPAN class="toctext">Preparing the anaerobic vessel</SPAN></A></LI><LI class="toclevel-1 tocsection-146"><A href="#Anaerobic_Expression_4"><SPAN class="tocnumber">124</SPAN><SPAN class="toctext">Anaerobic Expression</SPAN></A></LI><LI class="toclevel-1 tocsection-147"><A href="#Sample_Storage"><SPAN class="tocnumber">125</SPAN><SPAN class="toctext">Sample Storage</SPAN></A></LI><LI class="toclevel-1 tocsection-148"><A href="#Preparing_the_Anaerobic_Vessel_2"><SPAN class="tocnumber">126</SPAN><SPAN class="toctext">Preparing the Anaerobic Vessel</SPAN></A></LI><LI class="toclevel-1 tocsection-149"><A href="#Preparing_for_Aerobic_Expression"><SPAN class="tocnumber">127</SPAN><SPAN class="toctext">Preparing for Aerobic Expression</SPAN></A></LI><LI class="toclevel-1 tocsection-150"><A href="#Semi-Anaerobic_Expression"><SPAN class="tocnumber">128</SPAN><SPAN class="toctext">Semi-Anaerobic Expression</SPAN></A></LI><LI class="toclevel-1 tocsection-151"><A href="#Sample_Analysis"><SPAN class="tocnumber">129</SPAN><SPAN class="toctext">Sample Analysis</SPAN></A></LI><LI class="toclevel-2 tocsection-152"><A href="#Purification"><SPAN class="tocnumber">129.1</SPAN><SPAN class="toctext">Purification</SPAN></A></LI><LI class="toclevel-2 tocsection-153"><A href="#Fluorescence_Measurements"><SPAN class="tocnumber">129.2</SPAN><SPAN class="toctext">Fluorescence Measurements</SPAN></A></LI><LI class="toclevel-2 tocsection-154"><A href="#SDS_Gel_and_Western_Blot_Preparation"><SPAN class="tocnumber">129.3</SPAN><SPAN class="toctext">SDS Gel and Western Blot Preparation</SPAN></A></LI><LI class="toclevel-1 tocsection-155"><A href="#Aerobic_Expression"><SPAN class="tocnumber">130</SPAN><SPAN class="toctext">Aerobic Expression</SPAN></A></LI><LI class="toclevel-1 tocsection-156"><A href="#Preparing_MRI_samples"><SPAN class="tocnumber">131</SPAN><SPAN class="toctext">Preparing MRI samples</SPAN></A></LI><LI class="toclevel-1 tocsection-157"><A href="#Sample_purification"><SPAN class="tocnumber">132</SPAN><SPAN class="toctext">Sample purification</SPAN></A></LI><LI class="toclevel-2 tocsection-158"><A href="#Purification_2"><SPAN class="tocnumber">132.1</SPAN><SPAN class="toctext">Purification</SPAN></A></LI><LI class="toclevel-2 tocsection-159"><A href="#Fluorescence_Measurements_2"><SPAN class="tocnumber">132.2</SPAN><SPAN class="toctext">Fluorescence Measurements</SPAN></A></LI><LI class="toclevel-2 tocsection-160"><A href="#SDS_Gel_and_Western_Blot_Preparation_2"><SPAN class="tocnumber">132.3</SPAN><SPAN class="toctext">SDS Gel and Western Blot Preparation</SPAN></A></LI><LI class="toclevel-1 tocsection-161"><A href="#EGFP_Protein_Analysis"><SPAN class="tocnumber">133</SPAN><SPAN class="toctext">EGFP Protein Analysis</SPAN></A></LI><LI class="toclevel-2 tocsection-162"><A href="#Running_the_SDS_Gels"><SPAN class="tocnumber">133.1</SPAN><SPAN class="toctext">Running the SDS Gels</SPAN></A></LI><LI class="toclevel-2 tocsection-163"><A href="#Coomaaise_Analysis"><SPAN class="tocnumber">133.2</SPAN><SPAN class="toctext">Coomaaise Analysis</SPAN></A></LI><LI class="toclevel-2 tocsection-164"><A href="#Western_Blot"><SPAN class="tocnumber">133.3</SPAN><SPAN class="toctext">Western Blot</SPAN></A></LI></TD></TR></TBODY></TABLE><H3><SPAN class="mw-headline" id="Creating_agar_plates">Creating agar plates</SPAN></H3><P>Before any real lab work could begin some supplies had to be created, including a selection of agar plates. We needed two different types of agar plates, some with ampicillin antibiotics and others with kanamycin antibiotics. To manage this we created 2 separate agar solutions, a 200mL mixture for the ampicillin plates and a 400mL mixture for the kanamycin plates. The protocol we followed for the creation of the agar solutions was as follows: 
</P><UL><LI>Mix together the following amounts to create 200mL agar solution:
</LI><LI>2g Peptones
</LI><LI>2g NaCl
</LI><LI>1g Yeast extract
</LI><LI>3g Agar
</LI><LI>Fill the container up to 200mL with demineralised water.
</LI></UL><LI>For a 400mL solution mix together:
</LI><LI>4g Peptones
</LI><LI>4g NaCl
</LI><LI>2g Yeast extract
</LI><LI>6g Agar
</LI><LI>Fill the container up to 400mL with demineralised water.
</LI><LI>Next the containers were autoclaved and allowed to cool, but not harden.
</LI><LI>The following steps were all performed in the vicinity of a blue flame to ensure a sterile working environment.
</LI><LI>Before pouring the plates the correct antibiotics were added. The concentration of the ampicillin antibiotics was 100ng/µL and that of the kanamycin antibiotics was 30ng/µL.
</LI><LI>The solutions were now ready to be poured into agar plates. From the solutions we made we were able to pour 8 ampicillin plates and after receiving a little extra (about 350mL) kanamycin agar solution we poured a total of 27 kanamycin plates.
</LI><LI>The plates were cooled on the bench and allowed to harden before being stored in a 4°C refrigerator.
</LI><H3><SPAN class="mw-headline" id="Transforming_the_DNA">Transforming the DNA</SPAN></H3><P>At this moment in time only one DNA vector (Protamine-1-Optimized) had arrived, but being impatient and rearing to go we decided to proceed with this one sample anyway which also allowed us a chance to get back into the rhythm of doing lab work. We needed to transform the 4µg of DNA into NB bacteria ready for plating and culturing. This all was done by completing the following steps:
</P><UL><LI>The first step was to dilute the 4µg of vector in 20µL of MilliQ water. This created a 200ng/µL solution. 
</LI><LI>Of the previously created solution 1µL was pipetted into  199µL of MilliQ water creating a 4000 times dilution (1ng/µL).
</LI><LI>Of this 1ng/µL solution 1µL was pipetted into 20µL of NB bacteria. The two dilutions could now be stored in the -20°C freezer.
</LI><LI>The NB/Vector solution was left on ice for a short while before being heat-shocked in a water-bath of 42°C for 30 seconds. 
</LI><LI>Now the NB bacteria were returned to ice for 2 minutes. 
</LI><LI>After allowing the bacteria to cool 80µL of SOC solution was added. Hereafter the NB bacteria were not returned to ice. Instead they were placed in a 37°C incubator for 60 minutes.
</LI></UL><H3><SPAN class="mw-headline" id="Plating_the_bacteria">Plating the bacteria</SPAN></H3><P>After incubating the bacteria for a hour they were ready to be plated so that we could create a number of cultures. This was done in the following fashion:
</P><UL><LI>The following steps were all performed in the vicinity of a blue flame to increase the sterility.
</LI><LI>The plate was opened and the entire bacterial solution (approx. 101µL) was pipetted onto an ampicillin agar plate. 
</LI><LI>To ensure even culture growth the solution was spread out over the plate using a sterile spreader. 
</LI><LI>The plate with its bacterial spread was then placed in a 37°C incubator and left there overnight to grow.
</LI></UL><H3><SPAN class="mw-headline" id="Making_LB_medium">Making LB medium</SPAN></H3><P>Another preparational step that was performed was the creation of LB medium which we will be using a lot in the coming weeks. The protocol for the making of LB medium follows that for the creation of agar solution without the addition of the agar. So to create 1L of LB medium we mixed the following:
</P><UL><LI>10g Peptones
</LI><LI>10g NaCl
</LI><LI>5g Yeast extract
</LI><LI>Fill the container up to 1L with demineralised water.
</LI><LI>The entire container was then placed in the autoclave and sterilized. 
</LI><LI>The LB medium was allowed to cool and has been stored for later use at room temperature.
</LI></UL></DIV></DIV><DIV class="tue-timeline-middle"><DIV class="tue-timeline-leftbar"><DIV class="tue-timeline-node">25 July2013</DIV></DIV><DIV class="tue-timeline-content"><H3><SPAN class="mw-headline" id="Creating_small_cultures">Creating small cultures</SPAN></H3><P>Today we continued with the Protamine-1-optimized sample. This sample had been plated onto an ampicillin agar plate and was allowed to grow overnight. The next step was to transfer cultures from this agar plate into small amounts of LB medium so we could obtain a slightly larger culture, essentially increasing the amount of viable DNA vectors we have. The transformation to LB went as follows:
</P><UL><LI>All following steps were performed in the vicinity of a blue flame increasing the sterility.
</LI><LI>Firstly three small falcon tubes were filled with 8mL of LB medium.
</LI><LI>To each of the falcon tubes 8µL of ampicillin antibiotics were added.
</LI><LI>Now the agar plate was opened and three free lying colonies were chosen for picking. Each of the chosen colonies was then picked by lightly scraping across it with a pipet point. Each pipet point and colony picking was then ejected into one of the falcon tubes.
</LI><LI>The falcon tubes were then placed into a rotating incubator set to 37°C and left there to culture overnight.
</LI></UL><P>
As there were no further preparational steps to perform and no other DNA vectors had arrived this concluded a rather short day in the lab. 
</P></DIV></DIV><DIV class="tue-timeline-middle"><DIV class="tue-timeline-leftbar"><DIV class="tue-timeline-node">26 July2013</DIV></DIV><DIV class="tue-timeline-content"><H3><SPAN class="mw-headline" id="DNA_Retention_from_cultures">DNA Retention from cultures</SPAN></H3><P>Previously (on the 25th July 2013) three small 8mL cultures had been placed in the incubator containing NB bacteria which housed the vector for the protamine-1-optimized sample. These cultures had grown overnight and it was our aim today to retain the DNA vectors from these three cultures, essentially having allowed us to multiple the amount of DNA we had available. To perform this DNA retention a miniprep protocol was followed as is described below:
</P><UL><LI>First the culture tubes were spun down in a centrifuge for 10 minutes at 3700rpm causing the bacterial cells with the DNA inside to form pellets isolating it from the growth medium. 
</LI><LI>The supernatant that had formed above the pellets was discarded and the pellets themselves were resuspended in 250µL of P1 buffer. The tubes were gently shaken by hand until the entire pellet had resuspended.
</LI><LI>The suspension was then transferred by means of pipetting into a smaller (1.5mL) eppendorf tube. 
</LI><LI>250µL of P2 buffer was then added to the eppendorf tube and the solution was inverted by hand until it had turned a clear blue. 
</LI><LI>Within 5 minutes of adding the P2 buffer 350µL of N3 buffer was added to the solution and the eppendorf tube was inverted again until the solution was clear again. 
</LI><LI>Once clear the eppendorf tube was placed in the centrifuge and spun for 10 minutes at 13000rpm seperating the bacterial cells from the DNA which remained suspended in the solution. The bacterial cells formed a pellet in the bottom of the eppendorf tube. (Unfortunately the pellet had not completely formed after 10 minutes so the eppendorf tube was spun for a further minute, again at 13000rpm.)
</LI><LI>As soon as the centrifuge had stopped and we could see that the pellet had properly formed, the supernatant was poured off into a special QIAcolumn. The pellet could be discarded in the bio-hazard waste. 
</LI><LI>The QIAcolumn was subsequently centrifuged for one minute at 13000rpm, during which the DNA vector bound to the column meaning the flow through could be discarded. 
</LI><LI>750µL of PE buffer was then added to the QIAcolumn before centrifuging it for another minute at 13000rpm. The flow through was discarded.
</LI><LI>To ensure that all the PE buffer had passed through the column it was then spun yet again for one minute at a speed of 13000rpm. Yet again the flow through could be discarded.
</LI><LI>The column itself was then placed above a sealable 1.5mL eppendorf tube so that the DNA could be contained and stored. To obtain the DNA from the column 50µL of MilliQ water was pipetted onto the very center membrane of the column after which it was spun down for one minute at 13000rpm. The flow through now sat in the 1.5mL eppendorf tube and contained the DNA vectors.
</LI></UL><H3><SPAN class="mw-headline" id="Nanodrop_Test">Nanodrop Test</SPAN></H3><P>To check how much DNA we had acquired by culturing the NB bacteria with our vector a small nanodrop test was performed. This would tell us how many ng of DNA could be found in each µL of our samples. 
The results have been given below:
</P><UL><LI>Tube 1 contained 145.5 ng/µL of DNA.
</LI><LI>Tube 2 contained 164.1 ng/µL of DNA.
</LI><LI>Tube 3 contained 153.9 ng/µL of DNA.
</LI></UL><P>
The remaining DNA was then placed in a -20°C freezer and stored for later use concluding the days lab work. 
</P></DIV></DIV></DIV></DIV><DIV class="tab-pane" id="Week2"><DIV><DIV class="tue-timeline-top"><DIV class="tue-timeline-title"><H2>Week 2</H2></DIV></DIV><DIV class="tue-timeline-middle"><DIV class="tue-timeline-leftbar"><DIV class="tue-timeline-node">31 July2013</DIV></DIV><DIV class="tue-timeline-content"><H3><SPAN class="mw-headline" id="Transforming_DNA">Transforming DNA</SPAN></H3><P>Today a further 4 constructs arrived (Poly(Arginine-Serine), Poly(Arginine-Glycine), 1PJN and 1ETF) so it became possible to continue with the lab work: increasing the amount of viable DNA for these samples much like we had done for the Protamine-1-optimized sample which we had recieved a week earlier. To increase the DNA we would transfer the DNA into Nova-Blue bacteria and culture these allowing the bacteria to replicate our own vector constructs. To isolate the DNA we would then move on to miniprepping the cultures. Firstly however the would need to be transformed into the bacteria. The steps taken to do this are listed below:
</P><UL><LI>Before transformation could be started the 4µg of DNA we had recieved would need to be diluted to a 1ng/µL concentration as follows:
</LI><LI>The first step was to dilute the 4µg of vector in 20µL of MilliQ water. This created a 200ng/µL solution. 
</LI><LI>Of the previously created solution 1µL was pipetted into 199µL of MilliQ water creating a 4000 times dilution (1ng/µL).
</LI></UL><LI>With this 1ng/µL solution the transformation could commense:
</LI><LI>Of this 1ng/µL solution 1µL was pipetted into 20µL of NB bacteria. The two dilutions could now be stored in the -20°C freezer.
</LI><LI>The NB/Vector solution was left on ice for a short while before being heat-shocked in a water-bath of 43.6°C for 30 seconds. (This should have been 42°C but the temperature gauge was thought to be broken at this time.)
</LI><LI>Now the NB bacteria were returned to ice for 2 minutes. 
</LI><LI>After allowing the bacteria to cool 80µL of SOC solution was added. Hereafter the NB bacteria were not returned to ice. Instead they were placed in a 37°C incubator for 60 minutes.
</LI><H3><SPAN class="mw-headline" id="Plating_the_bacteria_2">Plating the bacteria</SPAN></H3><P>After incubating the bacteria for a hour they were ready to be plated so that we could create a number of cultures. This was done in the following fashion:
</P><UL><LI>The following steps were all performed in the vicinity of a blue flame to increase the sterility.
</LI><LI>The plate was opened and the entire bacterial solution (approx. 101µL) was pipetted onto an ampicillin agar plate. 
</LI><LI>To ensure even culture growth the solution was spread out over the plate using a sterile spreader. 
</LI><LI>The plate with its bacterial spread was then placed in a 37°C incubator and left there overnight to grow.
</LI></UL></DIV></DIV><DIV class="tue-timeline-middle"><DIV class="tue-timeline-leftbar"><DIV class="tue-timeline-node">1 August2013</DIV></DIV><DIV class="tue-timeline-content"><H3><SPAN class="mw-headline" id="Creating_the_Cultures">Creating the Cultures</SPAN></H3><P>Looking ahead to the cloning we would like to perform later on and using the yields of DNA obtained from the Protamine-1-Optimized sample last week a small calculation was performed to see how much DNA we would need and therefore how many cultures would need to be created. As we were going to attempt to use gel extraction to purify the digestion products of these constructs we would need to culture approximately 20 times more DNA than that we would eventually need for the ligation. Based on the average 150ng/µL DNA yield from the Protamine-1-Optimized construct we determined we would need 12 8mL cultures per construct to obtain the correct amount of DNA. This meant that we would would be creating:
</P><UL><LI>12 cultures of Poly(Arginine-Glycine).
</LI><LI>12 cultures of Poly(Arginine-Serine).
</LI><LI>12 cultures of 1PJN.
</LI><LI>12 cultures of 1ETF.
</LI><LI>9 cultures of Protamine-1-Optimized to bring the total number of cultures up to 12. 
</LI></UL><P>Again looking ahead to the cloning we saw that we would also need to increase the amount of pBR322 and pET28a vectors we had. For each of these we would need to create 3 8mL cultures to be able to perform an equimolar ligation. This all brought the total number of cultures we would need to create up to 63. The creating of the cultures was then performed in the following manner:
</P><UL><LI>All the following steps were performed in the vicinity of a blue flame to increase the sterility.
</LI><LI>63 culture tubes were filled with 8mL of LB medium which had been prepared earlier.
</LI><LI>8µL of 1000x Ampicillin antibiotics were then added to 60 of the culture tubes. 8µL of 1000x stock Kanamycin antibiotics were then added to the remaining three culture tubes. This was done as the pET28a vectors were resilient to Kanamycin and not to Ampicillin which all the other constructs were.
</LI><LI>The culture plates for each of the constructs were then retrieved. Instead of, as we did with the previous Protamine-1-Optimized constructs, pipetting a single culture into each of the culture tubes we now to decided to pick a culture and dissolve it in 15µL of water. From this 15µL solution 5µL could be pipetted into a culture tube. This meant that we would not have to pick 12 cultures from the plate per construct (or 9 for the Protamine-1-Optimized construct) but only 4 (or 3 for the Protamine-1-Optimized construct).
</LI><LI>For the pET28a culture, an eppendorf tube containing a glycerol stock of pET28a vectors in NB bacteria was retrieved and from the -80°C freezer and once thawed, 5µL was pipetted into each of the 3 culture tubes.
</LI><LI>For the pBR322 culture we would first have to transform the DNA into NB bacteria in the following fashion:
</LI><LI>Firstly the pBR322 vector DNA would have to be diluted as follows:
</LI><LI>Add 450µL of MilliQ water to the DNA to create a 100ng/µL stock solution.
</LI><LI>Take 1µL of the 100ng/µL stock solution and add it to a further 99µL of MilliQ water creating a 1ng/µL stock solution.
</LI></UL><LI>Using this 1ng/µL stock solution the transformation could commense.
</LI><LI> 1µL of the stock solution was added to 20µL of NB bacteria.
</LI><LI>This solution was left on ice for 5 minutes before being placed in a 42°C water bath, heat shocking the bacteria.
</LI><LI>The bacterial solution was then placed on ice for a further 2 minutes before 80µL of SOC was added to the solution.
</LI><LI>Without placing on ice first the solution was then placed in a 37°C incubator for 60 minutes.
</LI><LI>After 60 minutes in the incubator the 20µL of the solution could be pipetted into each of the three pBR322 culture tubes. 
</LI><LI>60µL of SOC was subsequently added to the remainder of the bacterial solution so that it could be plated (on an Ampicillin agar plate) and in that way serve as a backup should the culture not grow.
</LI><LI>Now that all the culture tubes were complete they could be placed in a 37°C incubator overnight to grow. 
</LI></DIV></DIV><DIV class="tue-timeline-middle"><DIV class="tue-timeline-leftbar"><DIV class="tue-timeline-node">2 August2013</DIV></DIV><DIV class="tue-timeline-content"><H3><SPAN class="mw-headline" id="Retaining_the_DNA">Retaining the DNA</SPAN></H3><P>As there were 63 culture tubes it was decided to perform the miniprep protocol, used to retain the DNA from the cultures, on the cultures in two separate bulks. The first bulk would see the retention of DNA from the 1ETF, 1PJN and Poly(Aginine-Glycine) culture tubes. The second bulk would then deal with the Poly(Arginine-Serine), Protamine-1-Optimized, pBR322 and pET28a cultures. This did mean that the second bulk would have more time (several hours) to grow than the cultures of the first bulk. For both the bulks the same protocol was followed:
</P><UL><LI>Before beginning with the miniprep protocol 700µL of culture solution was added to 300µL of 50% Glycerol solution. This was done for one culture from each construct so that should something go wrong we would no longer have to worry about plating the desired constructs.
</LI><LI>To start the miniprep protocol the culture tubes were spun down in a centrifuge for 10 minutes at 3700rpm causing the bacterial cells with the DNA inside to form pellets isolating it from the growth medium. 
</LI><LI>The supernatant that formed above the pellets was discarded and the pellets themselves were resuspended in 250µL of P1 buffer. The tubes were gently shaken by hand until the entire pellet had resuspended.
</LI><LI>The suspension was then transferred by means of pipetting into a smaller (1.5mL) eppendorf tube. 
</LI><LI>250µL of P2 buffer was then added to the eppendorf tube and the solution was inverted by hand until it had turned a clear blue. 
</LI><LI>Within 5 minutes of adding the P2 buffer 350µL of N3 buffer was added to the solution and the eppendorf tube was inverted again until the solution was clear again. 
</LI><LI>Once clear the eppendorf tube was placed in the centrifuge and spun for 10 minutes at 13000rpm seperating the bacterial cells from the DNA which remained suspended in the solution. The bacterial cells formed a pellet in the bottom of the eppendorf tube. (Unfortunately the pellet had not completely formed after 10 minutes so the eppendorf tube was spun for a further minute, again at 13000rpm.)
</LI><LI>As soon as the centrifuge had stopped and we could see that the pellet had properly formed, the supernatant was poured off into a special QIAcolumn. The pellet could be discarded in the bio-hazard waste. 
</LI><LI>The QIAcolumn was subsequently centrifuged for one minute at 13000rpm, during which the DNA vector bound to the column meaning the flow through could be discarded. 
</LI><LI>750µL of PE buffer was then added to the QIAcolumn before centrifuging it for another minute at 13000rpm. The flow through was discarded.
</LI><LI>To ensure that all the PE buffer had passed through the column it was then spun yet again for one minute at a speed of 13000rpm. Yet again the flow through could be discarded.
</LI><LI>The column itself was then placed above a sealable 1.5mL eppendorf tube so that the DNA could be contained and stored. To obtain the DNA from the column 36µL of MilliQ water was pipetted onto the very centre membrane of the column after which it was spun down for one minute at 13000rpm. The flow through now sat in the 1.5mL eppendorf tube and contained the DNA vectors. (It would appear that for the 1ETf and 1PJN cultures 72µL of water was used instead of the desired 36µL meaning we had total volume for these tow samples which was twice as large as for the other cultures.)
</LI></UL><H3><SPAN class="mw-headline" id="Nanodrop_Test_2">Nanodrop Test</SPAN></H3><P>To determine how much DNA we had acquired for each of the constructs a nanodrop test could be performed. However before doing so all the DNA from a single construct was combined so that we had an indication of how much DNA we had in total. For the Protamine-1-Optimized sample this combination included the 3 50µL solutions we had obtained previously. The results of this nanodrop test are given below:
</P><UL><LI>432µL of Poly(Arginine-Glycine) with a concentration of 265.1ng/µL.
</LI><LI>432µL of Poly(Arginine-Serine) with a concentration of 262.3ng/µL.
</LI><LI>474µL of Protamine-1-Optimized with a concentration of 209.3ng/µL.
</LI><LI>864µL of 1ETF with a concentration of 225.1ng/µL.
</LI><LI>864µL of 1PJN with a concentration of 120.5ng/µL.
</LI><LI>108µL of pBR322 with a concentration of 108.4ng/µL.
</LI><LI>108µL of pET28a with a concentration of 62.1ng/µL.
</LI></UL></DIV></DIV></DIV></DIV><DIV class="tab-pane" id="Week3"><DIV><DIV class="tue-timeline-top"><DIV class="tue-timeline-title"><H2>Week 3</H2></DIV></DIV><DIV class="tue-timeline-middle"><DIV class="tue-timeline-leftbar"><DIV class="tue-timeline-node">5 August2013</DIV></DIV><DIV class="tue-timeline-content"><H3><SPAN class="mw-headline" id="Digestion">Digestion</SPAN></H3><P>Previously we had increased the amount of DNA we had to our disposal by culturing bacteria with containing our constructs and retaining the multiplied DNA using the miniprep method. Each of the five constructs we had designed and received (Protamine-1-Optimized, 1PJN, 1ETF, Poly(Arginine-Serine) and Poly(Arginine-Glycine)) had successfully been multiplied along with the two vectors (pET28a and pBR322) in which we wished to insert our constructs. To insert our constructs into the two vectors we would first need to digest all the constructs and the vectors using the correct restriction enzymes and then ligate the insert and the vector. Today was focussed around the digestion.
</P><H4><SPAN class="mw-headline" id="Construct_Digestion">Construct Digestion</SPAN></H4><P>As the our desired inserts for the ligation were in the pUC57-Simple vector we would need to perform digestion on all the vectors and then purify the product thereof to obtain the desired insert. However, due to the relatively large size of the inserts we could not do this using the standard PCR purification kit. This meant that purification would have to be done on basis of gel extraction. For this we would need a larger amount of DNA to start with to ensure a sufficient yield upon completion. It was decided that loading 10µg of DNA onto the gel for each construct would be adequate. Insert would be adequate. However for each construct two inserts would be created so each construct would have to provide 2 * 10µg of DNA for loading onto the gel. 
</P><P>For the digestion the following were added together in a PCR eppendorf tube:
</P><TABLE class="table table-striped"><TBODY><TR><TH align="center"> Construct
</TH><TH> 5µg DNA (µL)
</TH><TH> CutSmart Buffer (µL)
</TH><TH> Amount of each Enzyme (µL)
</TH><TH> MilliQ water (µL)
</TH></TR><TR><TD>Protamine-1-Optimized
</TD><TD>23.9
</TD><TD>5
</TD><TD>0.7
</TD><TD>20.4
</TD></TR><TR><TD>1ETF
</TD><TD>22.2
</TD><TD>5
</TD><TD>0.7
</TD><TD>22.1
</TD></TR><TR><TD>1PJN
</TD><TD>41.5
</TD><TD>5
</TD><TD>0.7
</TD><TD>2.8
</TD></TR><TR><TD>Poly(Arginine-Glycine)
</TD><TD>18.9
</TD><TD>5
</TD><TD>0.7
</TD><TD>25.4
</TD></TR><TR><TD>Poly(Arginine-Serine)
</TD><TD>19.1
</TD><TD>5
</TD><TD>0.7
</TD><TD>25.2
</TD></TR></TBODY></TABLE><P>For each of the 5 constructs the solutions in the above table were prepared with the exception of the enzymes a total of four times. This provided us with 20µg of DNA in total per construct spread out equally across 4 seperate PCR tubes. To two of these the 0.7µg of the NheI and 0.7µg of the XhoI restriction enzymes were added. The product of the restriction with these enzymes would cut out just the protein sequence from the construct allowing us to ligate that into the pET28a vector for aerobic expression. To the remaining two eppendorf tubes for each of the constructs 0.7µg of the EcoRI and 0.7µg of the HindIII restriction enzymes were added. This restriction would cut our entire construct from the pUC57-simple vector allowing us to ligate that into the pBR322 vector and test our construct anaerobically. 
</P><P>After adding the enzymes the solutions were quickly mixed and placed in the PCR apparatus.
</P><H4><SPAN class="mw-headline" id="Vector_Digestion">Vector Digestion</SPAN></H4><P>Rather more simple than the construct digestion the vectors too needed to be digested. As we would only require 500µg of DNA as a result of this digestion it was not necessary to use 5µg of DNA so it was decided only 2µg would be used per vector. The following solutions were made: 
</P><TABLE class="table table-striped"><TBODY><TR><TH align="center"> Vector
</TH><TH> 2µg DNA (µL)
</TH><TH> CutSmart Buffer (µL)
</TH><TH> Amount of each Enzyme (µL)
</TH><TH> MilliQ water (µL)
</TH></TR><TR><TD>pBR322
</TD><TD>32.2
</TD><TD>5
</TD><TD>0.7
</TD><TD>12.1
</TD></TR><TR><TD>pET28a
</TD><TD>18.5
</TD><TD>5
</TD><TD>0.7
</TD><TD>25.8
</TD></TR></TBODY></TABLE><P>0.7µL of the NheI and 0.7µL of the XhoI restriction enzymes were added to the pET28a vector solution and 0.7µL of the EcoRI and 0.7µL of the HindIII restriction enzymes were added to the pBR322 vector solution. These solutions were then placed in the same PCR apparatus as the construct solutions and the following PCR protocol was run for both the constructs and the vectors simultaneously:
</P><UL><LI>Reaction Step: 37°C for 60 minutes
</LI><LI>Enzyme Inactivation Step: 80°C for 20 minutes
</LI><LI>Storage Step: 4°C
</LI></UL><H3><SPAN class="mw-headline" id="PCR_Purification_for_the_Constructs">PCR Purification for the Constructs</SPAN></H3><P>Due to the large size of the insert produced by the digestion of the constructs a gel extraction would be used to select the appropriate sections of the digestion product of the constructs. To isolate the correct segments on the gel the following steps were taken:
</P><UL><LI>A 1% electrophoresis gel was created with 10 large lanes (each lane large enough for a 120µL sample).
</LI><LI>For each of the constructs the two PCR solutions containing the same enzymes were pipetted into a single tube creating a single 100µL solution.
</LI><LI>To each tube containing a 100µL solution 20µL of 6x stock solution loading dye was added bringing the total volume up to 120µL.
</LI><LI>Each of these 120µL samples were then brought over into one of the 10 available lanes.
</LI><LI>The gel was the &quot;run&quot; for 60 minutes with a voltage of 100V.
</LI><LI>After &quot;running&quot; the gel it was removed from its container and placed on a UV light. Here we were able to discern between the different bands. For each of the digestion results the smaller DNA segment was desired so the lowest band in each lane of the gel was cut from the gel and placed in an eppendorf tube. 
</LI><LI>The 10 eppendorf tubes containing the gel cut-outs were then placed in the -20°C freezer for storage until the real DNA extraction would take place. 
</LI></UL><DIV class="row"><DIV class="tue-float tue-float-left span6"><DIV class="tue-caption"><SPAN class="tue-caption-figure"><SPAN class="tue-refid">GelextractionLeft0508</SPAN></SPAN> <SPAN class="tue-caption-text">Gel extraction of digested pET28a constructs from left to right: 1: Protamine-1-Optimized, 2: 1ETF, 3: 1PJN, 4: P(RG), 5: P(RS)</SPAN></DIV></DIV><DIV class="tue-float tue-float-right span6"><DIV class="tue-caption"><SPAN class="tue-caption-figure"><SPAN class="tue-refid">GelextractionRight0508</SPAN></SPAN> <SPAN class="tue-caption-text">Gel extraction of digested pBR322 constructs from left to right: 1: Protamine-1-Optimized, 2: 1ETF, 3: 1PJN, 4: P(RG), 5: P(RS)</SPAN></DIV></DIV></DIV><H3><SPAN class="mw-headline" id="PCR_Puricfication_for_the_Vectors">PCR Puricfication for the Vectors</SPAN></H3><P>For the vectors a simple QIAquick PCR purification kit could be used to purify the digestion product as the cut out was smaller than 100 bases for each of the two vectors. The purification was carried out according to the following protocol:
</P><UL><LI>To the PCR product 5 volumes of PB buffer were added (In our case this was 250µL of PB buffer as we had 50µL of PRC product.)
</LI><LI>This solution was placed in a QIAquick spin column and spun for 60 seconds at 13000 rpm. The flow through could be discarded.
</LI><LI>Now 750µL of PE buffer could be added to the column before spinning it down once more for 60 seconds at 13000 rpm.
</LI><LI>Having discarded the flow through the column was spun for a further 60 seconds again at 13000 rpm to remove the last of the PE buffer.
</LI><LI>The column was then placed into a sealable eppendorf tube before adding 42µL of MilliQ water to the very centre membrane of the spin column and leaving it to stand for one minute.
</LI><LI>The column was now placed in the centrifuge and spun for 60 seconds at 13000 rpm producing a solution containing the desired DNA.
</LI></UL><H3><SPAN class="mw-headline" id="Nanodrop_Results">Nanodrop Results</SPAN></H3><P>To test how successful the entire process of restriction had been for the vectors the purification products were analysed using the nanodrop spectrometer. the results are given below:
</P><UL><LI>pBR322 had a concentration of 11.6ng/µL.
</LI><LI>pET28a had a concentration of 7.9ng/µL.
</LI></UL><P>Having eluded both of these samples in just 42µL of water meant that we did not have enough vector DNA for the ligation later on (we were aiming for 500ng). To solve this the entire restriction was performed again for the vectors. This meant remaking the solutions for the digestion and restarting the PCR. As it was late in the day, the solutions were prepared, the enzymes were added and the PCR was started. The PCR machine was set to return to 4°C to store the restriction product overnight and the PCR purification could then be performed the next day along with another nanodrop analysis, hopefully with better results.  
</P></DIV></DIV><DIV class="tue-timeline-middle"><DIV class="tue-timeline-leftbar"><DIV class="tue-timeline-node">6 August2013</DIV></DIV><DIV class="tue-timeline-content"><H3><SPAN class="mw-headline" id="Continuing_with_the_Purification">Continuing with the Purification</SPAN></H3><P>On August 5th digestion was performed on five of our ordered constructs, and on the two vectors in which we would like to place our constructs. At the end of the day the digestion products of the constructs were still within the gel cut outs which had been stored in a -20°C freezer. The constructs had been purified but the yield had been too low for ligation so the digestion was performed once more. Today then, before moving on to the ligation, the constructs would have to be extracted from the gel and the vectors would also have to be purified. 
</P><H4><SPAN class="mw-headline" id="Vector_Purification">Vector Purification</SPAN></H4><P>The purification of the vectors was performed in the same manner this time round as it was the previous time, following the protocol of the QIAquick PCR purification kit. To prevent falling into repetition this protocol will not be discussed here but can be found in the journal entry for August 5th. The results of this purification however, gotten through a nanodrop measurement are given below:
</P><UL><LI>pBR322 had a concentration of 15.3ng/µL.
</LI><LI>pET28a had a concentration of 10.7ng/µL.
</LI></UL><P>Needing these digested vectors for the ligation later on, we had hoped for a yield of at least 500ng of DNA per vector. Based on these results and an estimated volume of around 38µL this meant that we had achieved our goal for the pBR322 vector. For the pET28 factor however this was not the case. Fortunately we had not discarded the solutions from the previous pET28a digestion meaning that their combined volumes and concentrations would be sufficient for the ligation. From now on pET28a(1) will refer to the pET28a sample digested on August 5th with a concentration of 9.8ng/µL and pET28a(2) will refer to the pet28a sample digested today with a concentration of 10.7ng/µL. 
</P><H4><SPAN class="mw-headline" id="Construct_Purification">Construct Purification</SPAN></H4><P>Our constructs had been run on gels to separate the different products formed during the digestion. The desired bands had been cut out from these gels and had been stored in the -20°C freezer overnight. Now we were ready to perform the gel extraction, retrieving the DNA from these gel cut outs. This was done following the QIAquick gel extraction protocol which is explained here:
</P><UL><LI>First all the samples were weighed. A single QIAcolumn can be used for up to 400ng of gel, as all our cut-outs were almost double this they were disected into two almost equal parts and spread over two separate eppendorf tubes. The weight of the gel in each tube was noted.
</LI><LI>To each tube 3 volumes of QG buffer was added. A volume can be defined as the weight of the gel in mg transferred into µL. SO  a gel weighing 400mg would have a volume of 400µL, so 1200µL of QG would be added. 
</LI><LI>The eppendorf tubes containing the gel cut-outs and the QG buffer were placed in an incubator set at 50°C for 10 minutes, whilst vortexing the samples at regular (every 2 minutes) intervals until the gel had completely dissolved. 
</LI><LI>To these tubes 1 volume of isopropanol was added. As the addition of this isopropanol would not fit in a single eppendorf tube the solution in each of the tubes was split equally across two separate eppendorf tubes. To each of these separate eppendorf tubes half a volume of isopropanol was added. 
</LI><LI>One of the two tubes was then brought over into a QIAquick column and placed in the centrifuge where it was spun down for 60 seconds at 13000 rpm. After discarding the flow through the second of the two tubes was then brought over into the QIAquick column and spun down in the same fashion. Once more the flow through was discarded.
</LI><LI>Now 750µL of PE buffer was added to the column before centrifuging at 13000 rpm for 60 seconds. After discarding the flow through the same amount was added and once more and again centrifuged.
</LI><LI>Removing the flow through the column was then centrifuged again for 60 seconds at 13000 rpm removing the last of the PE buffer. 
</LI><LI>After bringing the column across into a sealable eppendorf tube 36µL of MilliQ water was pipetted onto the very centre membrane of the column which was then let to stand for 60 seconds. Subsequently the column was spun down at 13000 rpm for 60 seconds. The DNA is now in the flow through. 
</LI><LI>Remembering that we split the gel in half in the very first stage of this protocol, the two solutions containing the same DNA were combined so that a total volume of maximally 72µL of each insert was formed.
</LI></UL><H3><SPAN class="mw-headline" id="Nanodrop_Results_2">Nanodrop Results</SPAN></H3><P>To check the concentration of the inserts we had acquired a nanodrop test was performed using the spectrometer. The results are given below. Remember that for each construct, two different digestions were performed, one for ligation into pET28a and one for ligation into pBR322. 
</P><TABLE class="table table-striped"><TBODY><TR><TH align="center">Construct
</TH><TH>pET28a Digest (ng/µL)
</TH><TH>pBR322 Digest (ng/µL)
</TH></TR><TR><TD>Protamine
</TD><TD>2.6
</TD><TD>9.6
</TD></TR><TR><TD>1PJN
</TD><TD>5.0
</TD><TD>10.5
</TD></TR><TR><TD>1ETF
</TD><TD>7.0
</TD><TD>11.1
</TD></TR><TR><TD>Poly(Arginine-Glycine)
</TD><TD>11.8
</TD><TD>16.0
</TD></TR><TR><TD>Poly(Arginine-Serine)
</TD><TD>8.3
</TD><TD>13.4
</TD></TR></TBODY></TABLE><H3><SPAN class="mw-headline" id="Ligation">Ligation</SPAN></H3><P>Each of our five constructs (Protamine-1-Optimized, 1PJN, 1ETF, Poly(Arginine-Glycine) and Poly(Arginine-Serine)) restriction was performed in two ways. The first cutting at the NheI and XhoI restriction sites allowing us to place just the protein sequence into the pET28a vector. The second restriction cut at the EcoRI and HindIII sites, thus allowing us to ligate the entire construct (protein sequence along with promoters, terminators and His-tags) into the pBR322 vector.
</P><P>Before being able to prepare the ligation solution it was necessary to perform a number of calculations. For each vector we added to the solution we wished to add five times as many inserts. This would ensure that our inserts would be ligated into the vectors. To start with we calculated how many µL of our digested vectors would need to be added to the ligation solution to obtain 100ng of vector DNA. 
</P><TABLE class="table table-striped"><TBODY><TR><TH align="center"> Vector
</TH><TH> Concentration (ng/µL)
</TH><TH> 100g DNA (µL)
</TH><TH> Number of Bases in Vector
</TH></TR><TR><TD>pBR322
</TD><TD>15.3
</TD><TD>6.6
</TD><TD>4330
</TD></TR><TR><TD>pET28a(1)
</TD><TD>9.8
</TD><TD>10.2
</TD><TD>5296
</TD></TR><TR><TD>pET28a(2)
</TD><TD>10.7
</TD><TD>9.4
</TD><TD>5296
</TD></TR></TBODY></TABLE><P>By dividing 100ng by the number of bases in a single vector and multiplying that number by the number of bases in a single insert it can be determined how many ng of the insert is needed for molar equality (having an equal amount of vectors as there are inserts). 
</P><P>By then dividing the amount of DNA needed for molar equality (in ng) by the concentration of each insert the volume (in µL) of insert solution needed would be given. As we did not want molar equality but five times as many inserts as there are vectors this volume could be multiplied by five. 
</P><P>The calculations for the pBR322 inserts are given here:
</P><TABLE class="table table-striped"><TBODY><TR><TH align="center"> Construct
</TH><TH> Number of Bases
</TH><TH> Concentration (ng/µL)
</TH><TH> ng Needed for Molar Equality (ng)
</TH><TH> Volume Needed for Molar Equality (µL)
</TH><TH> Volume Needed for 5x Molar Equality (µL)
</TH></TR><TR><TD>Protamine-1-Optimized </TD><TD> 489 </TD><TD> 9.2 </TD><TD> 11.3 </TD><TD> 1.23 </TD><TD> 6.2
</TD></TR><TR><TD>1PJN </TD><TD> 726 </TD><TD> 10.5 </TD><TD> 16.8 </TD><TD> 1.6 </TD><TD> 8.0
</TD></TR><TR><TD>1ETF </TD><TD> 756 </TD><TD> 11.1 </TD><TD> 17.5 </TD><TD> 1.58 </TD><TD> 7.9
</TD></TR><TR><TD>Poly(Arginine-Glycine) </TD><TD> 771 </TD><TD> 16.0 </TD><TD> 17.8 </TD><TD> 1.11 </TD><TD> 5.6
</TD></TR><TR><TD>Poly(Arginine-Serine) </TD><TD> 771 </TD><TD> 13.4 </TD><TD> 17.8 </TD><TD> 1.32 </TD><TD> 6.6
</TD></TR></TBODY></TABLE><P>For the pET28a vector neither of the two digested samples had enough DNA to provide 100ng for ligation with each of the five constructs. Therefore it was decided to use the pET28a(1) digestion product for the Poly(Arginine-Glycine) and Poly(Arginine-Serine) inserts. The pET28a(2) digestion product would then be used for the remaining three ligations (with Protmaine-1-Optimized, 1PJN and 1ETF). 
</P><P>The calculations are given here for the pET28a(1) ligation:
</P><TABLE class="table table-striped"><TBODY><TR><TH align="center"> Construct
</TH><TH> Number of Bases
</TH><TH> Concentration (ng/µL)
</TH><TH> ng Needed for Molar Equality (ng)
</TH><TH> Volume Needed for Molar Equality (µL)
</TH><TH> Volume Needed for 5x Molar Equality (µL)
</TH></TR><TR><TD>Poly(Arginine-Glycine) </TD><TD> 447 </TD><TD> 11.8 </TD><TD>8.44 </TD><TD> 0.72 </TD><TD> 3.6
</TD></TR><TR><TD>Poly(Arginine-Serine) </TD><TD> 447 </TD><TD> 8.3 </TD><TD> 8.44 </TD><TD> 1.02 </TD><TD> 5.1
</TD></TR></TBODY></TABLE><P>And for the pET28a(2) ligation here:
</P><TABLE class="table table-striped"><TBODY><TR><TH align="center"> Construct
</TH><TH> Number of Bases
</TH><TH> Concentration (ng/µL)
</TH><TH> ng Needed for Molar Equality (ng)
</TH><TH> Volume Needed for Molar Equality (µL)
</TH><TH> Volume Needed for 5x Molar Equality (µL)
</TH></TR><TR><TD>Protamine-1-Optimized </TD><TD> 165 </TD><TD> 2.6 </TD><TD> 3.11 </TD><TD> 1.2 </TD><TD> 6.0
</TD></TR><TR><TD>1PJN </TD><TD> 402 </TD><TD> 5.0 </TD><TD> 7.6 </TD><TD> 1.52 </TD><TD> 7.6
</TD></TR><TR><TD>1ETF </TD><TD> 432 </TD><TD> 7.0 </TD><TD> 8.2 </TD><TD> 1.2 </TD><TD> 5.9
</TD></TR></TBODY></TABLE><P>Having determined the volumes of insert and vector we would need for the ligation reactions, the ligation solutions could be prepared. In addition to the vectors and inserts we would also need to add a buffer, MilliQ water and Ligase. All reaction solutions would end up equal to 20µL. The table below shows the amounts used to prepare the pBR322 vector ligation samples:
</P><TABLE class="table table-striped"><TBODY><TR><TH align="center"> Construct
</TH><TH> Amount of Vector pBR322 (µL)
</TH><TH> Amount of Insert (µL)
</TH><TH> 10x Buffer (µL)
</TH><TH> MilliQ Water (µL)
</TH><TH> Ligase (µL)
</TH></TR><TR><TD>Protamine-1-Optimized </TD><TD> 6.6 </TD><TD> 6.2 </TD><TD> 2 </TD><TD> 4.2 </TD><TD> 1
</TD></TR><TR><TD>1PJN </TD><TD> 6.6 </TD><TD> 8.0 </TD><TD> 2 </TD><TD> 2.4 </TD><TD> 1
</TD></TR><TR><TD>1ETF </TD><TD> 6.6 </TD><TD> 7.9 </TD><TD> 2 </TD><TD> 2.5 </TD><TD> 1
</TD></TR><TR><TD>Poly(Arginine-Glycine) </TD><TD> 6.6 </TD><TD> 5.6 </TD><TD> 2 </TD><TD> 4.8 </TD><TD> 1
</TD></TR><TR><TD>Poly(Arginine-Serine) </TD><TD> 6.6 </TD><TD> 6.6 </TD><TD> 2 </TD><TD> 3.8 </TD><TD> 1
</TD></TR></TBODY></TABLE><P>Below are give the solution quantities for the pET28a(1) and pET28a(2) vector ligations respectivly:
</P><TABLE class="table table-striped"><TBODY><TR><TH align="center"> Construct
</TH><TH> Amount of Vector pET28a(1) (µL)
</TH><TH> Amount of Insert (µL)
</TH><TH> 10x Buffer (µL)
</TH><TH> MilliQ Water (µL)
</TH><TH> Ligase (µL)
</TH></TR><TR><TD>Poly(Arginine-Glycine) </TD><TD> 10.2 </TD><TD> 3.6 </TD><TD> 2 </TD><TD> 3.1 </TD><TD> 1
</TD></TR><TR><TD>Poly(Arginine-Serine) </TD><TD> 10.2 </TD><TD> 5.1 </TD><TD> 2 </TD><TD> 1.7 </TD><TD> 1
</TD></TR></TBODY></TABLE><TABLE class="table table-striped"><TBODY><TR><TH align="center"> Construct
</TH><TH> Amount of Vector pET28a(2) (µL)
</TH><TH> Amount of Insert (µL)
</TH><TH> 10x Buffer (µL)
</TH><TH> MilliQ Water (µL)
</TH><TH> Ligase (µL)
</TH></TR><TR><TD>Protamine-1-Optimized </TD><TD> 9.4 </TD><TD> 6.0 </TD><TD> 2 </TD><TD> 1.6 </TD><TD> 1
</TD></TR><TR><TD>1PJN </TD><TD> 9.4 </TD><TD> 7.6 </TD><TD> 2 </TD><TD> 0.0 </TD><TD> 1
</TD></TR><TR><TD>1ETF </TD><TD> 9.4 </TD><TD> 5.9 </TD><TD> 2 </TD><TD> 1.7 </TD><TD> 1
</TD></TR></TBODY></TABLE><P>Having prepared the above solutions, and having added the Ligase only at the very last moment, the solutions were placed in the PCR apparatus ready for the ligation. The ligation protocol was set as follows:
</P><UL><LI>37°C for 120 minutes for the ligation reaction.
</LI><LI>65°C for 10 minutes for the Ligase inactivation.
</LI><LI>4°C FOREVER to store the ligation products until they would be used again. 
</LI></UL><P>The samples were left in the PCR apparatus overnight.
</P><H3><SPAN class="mw-headline" id="Preparing_Agar_Plates">Preparing Agar Plates</SPAN></H3><P>Before leaving, and in anticipation of the coming steps we decided to prepare a number of Ampicillin agar plates. The agar solution itself was made by mixing the following:
</P><UL><LI>5g Peptones
</LI><LI>5g NaCl
</LI><LI>2.5g Yeast Extract
</LI><LI>7.5g Agar
</LI><LI>500mL Demineralised Water
</LI></UL><P>By placing the above solution in the autoclave the agar was produced. Once cool 500µL of Ampicillin was added to the solution whilst being stirred. In the vicinity of a blue bundsen flame 25 Ampicilling agar plates were poured which were then left to harden before being placed in the 4°C refrigerator.
</P></DIV></DIV><DIV class="tue-timeline-middle"><DIV class="tue-timeline-leftbar"><DIV class="tue-timeline-node">7 August2013</DIV></DIV><DIV class="tue-timeline-content"><H3><SPAN class="mw-headline" id="Transformation">Transformation</SPAN></H3><P>Having left the ligation products in the PCR apparatus at 4°C overnight we were now able to transform these into NoveBlue bacteria ready to plate them. This plating will allow us to perform a ColonyPCR later on to check the efficiency of our ligation. It would also allow us to replicate, duplicate and sequence correctly ligated vectors and inserts. 
For the transformation we transformed all 10 ligation products (each of the five constructs in both the vectors) using the same transformation protocol:
</P><UL><LI>All steps were performed in the vicinity of a blue flame to increase sterility.
</LI><LI>1µL of each ligation product was pipetted into 20µL of NovaBlue bacteria.
</LI><LI>These were left to stand on ice for 5 minutes.
</LI><LI>Then to heat shock the samples they were placed in a water bath at 42°C for 30 seconds. 
</LI><LI>The samples were subsequently returned to ice for 2 minutes. 
</LI><LI>Then 80µL of SOC solution was added to each of the samples. After this they would not be allowed to return to ice. 
</LI><LI>The samples were then placed in an incubator set to 37°C for 60 minutes. 
</LI></UL><P>Apart from transforming the ligation products three controls were also prepared. This was done to check that the restriction of the vectors had been successful. To make the control samples made samples of 20µL consisting of the same amount of vector as used in the ligation solutions. The remaining volume (up to 20µL) was then filled with water. The transformation protocol of these controls follows that which was used for the ligation products. 
</P><H3><SPAN class="mw-headline" id="Plating">Plating</SPAN></H3><P>Once the transformations had been incubated for an hour they were ready for plating to allow colonies to grow on the plates. The samples which had been ligated into the pET28a vectors and the two pET28a vector controls were plated onto agar platescontianing Kanamycin antibiotics. The samples which had been ligated into pBR322 vectors and the pBR322 vector control were plated onto agar plates containing Ampicillin antibiotics.
For the plating the following protocol was used:
</P><UL><LI>In the vicinity of a blue bunsen flame the entire sample (101µL) was pipetted onto the centre of the agar plate. 
</LI><LI>A spreader which had been stored in 100% Ethanol was lit in the flame burning off the Ethanol and sterilizing the spreader.
</LI><LI>The sterile spreader was allowed to cool before being placed onto the agar plate and spreading out the bacterial sample equally. 
</LI></UL><P>The plates had now been spread with the transformations of the ligation products and were placed in the 37°C incubator overnight. Here they could grow and with luck, be used for ColonyPCR the next day. 
</P></DIV></DIV><DIV class="tue-timeline-middle"><DIV class="tue-timeline-leftbar"><DIV class="tue-timeline-node">8 August2013</DIV></DIV><DIV class="tue-timeline-content"><H3><SPAN class="mw-headline" id="Colony_Picking_and_PCR">Colony Picking and PCR</SPAN></H3><P>Having plated NovaBlue bacteria containing our own ligated vectors (Each of our five constructs: Protamine-1-Optimized, 1PJN, 1ETF, Poly(Arginine-Glycine) and Poly(Arginine-Serine), were ligated into both pBR322 and pET28a vectors) we could now pick a number of these samples, check the ligation products using PCR and culture those ligation products which had been successful. Due to the late delivery of the pBR322 primers it was decided to perform the colony PCR on just the pET28a ligations. The pBR322 agar plates would be stored at 4°C until the primers arrived. 
The following protocol was then used to perform the colony picking and PCR:
</P><UL><LI>All steps up to the PCR itself were performed in the vicinity of a blue bunsen flame for increased sterility.  
</LI><LI>From each agar plate (each plate containing one of the constructs ligated with the pET28a vector) 5 seperate colonies were picked and dissolved in 15µL of sterile water. We had also plated the two separately digested pET28a factors as controls and from these plates a single colony was also picked and dissolved in 15µL of sterile water. Photos of these plates are visible below:
</LI></UL><DIV class="row"><DIV class="tue-float tue-float-left span3"><DIV class="tue-caption"><SPAN class="tue-caption-figure"><SPAN class="tue-refid">pET28a_Protamine</SPAN></SPAN> <SPAN class="tue-caption-text">pET28a-Protamine</SPAN></DIV></DIV><DIV class="tue-float tue-float-left span3"><DIV class="tue-caption"><SPAN class="tue-caption-figure"><SPAN class="tue-refid">pET28a_1PJN1</SPAN></SPAN> <SPAN class="tue-caption-text">pET28a-1PJN</SPAN></DIV></DIV><DIV class="tue-float tue-float-left span3"><DIV class="tue-caption"><SPAN class="tue-caption-figure"><SPAN class="tue-refid">pET28a_1ETF</SPAN></SPAN> <SPAN class="tue-caption-text">pET28a-1ETF</SPAN></DIV></DIV><DIV class="tue-float tue-float-left span3"><DIV class="tue-caption"><SPAN class="tue-caption-figure"><SPAN class="tue-refid">pET28a_P(RG)</SPAN></SPAN> <SPAN class="tue-caption-text">pET28a-Poly(Arginine-Glycine)</SPAN></DIV></DIV><DIV class="tue-float tue-float-left span3"><DIV class="tue-caption"><SPAN class="tue-caption-figure"><SPAN class="tue-refid">pET28a_P(RS)</SPAN></SPAN> <SPAN class="tue-caption-text">pET28a-Poly(Arginine-Serine)</SPAN></DIV></DIV><DIV class="tue-float tue-float-left span3"><DIV class="tue-caption"><SPAN class="tue-caption-figure"><SPAN class="tue-refid">pET28a_1_Control</SPAN></SPAN> <SPAN class="tue-caption-text">pET28a(1) Control</SPAN></DIV></DIV><DIV class="tue-float tue-float-left span3"><DIV class="tue-caption"><SPAN class="tue-caption-figure"><SPAN class="tue-refid">pET28a_2_Control</SPAN></SPAN> <SPAN class="tue-caption-text">pET28a(2) Control</SPAN></DIV></DIV></DIV><UL><LI>After dissolving the pickings they were ready for PCR. Before starting the PCR protocol the following solutions had to be made:
</LI><LI>9µL of sterile water.
</LI><LI>1.25µL of 10µM T7 Fwd Primer.
</LI><LI>1.25µL of 10µM T7 Rw Primer.
</LI><LI>12.5µL of Kappa2G 2x solution.
</LI><LI>1µL of the picking solution.
</LI></UL><LI>Having mixed the above, adding the Kappa2G last, the PCR tubes were placed in the PCR and run with the following protocol:
</LI><LI>Initial Denaturation: 95°C for 3 minutes.
</LI><LI>The following steps should be performed in this order, repeating the process a total of 34 times.
</LI><LI>Denaturation: 95°C for 15 seconds.
</LI><LI>Annealing: 58°C for 15 seconds.
</LI><LI>Extension: 72°C for 20 seconds.
</LI><LI>Final Extension: 72°C for 10 minutes. 
</LI><LI> Storage: 4°C FOREVER.
</LI><LI>Having completed the PCR protocol 4µL of 6x Loading Dye was added to each of the PCR solutions. Subsequently 18µL of each was loaded onto a 1.5% agarose electrophoresis gel which was prepared by mixing the following:
</LI><LI>1.8g of Agarose.
</LI><LI>120mL of 1x TAE buffer
</LI><LI>12µL of CyberSafe. 
</LI><LI>The gel was then run at 100V for 75 minutes before placing the gel into a UV spectrometer. Here we could analyse the gel and decide which of the ligation products were successful. Of each of the five constructs two successfully ligated bacteria colonies were selected and these would then be grown in a larger culture. The gel result is shown below.
</LI><DIV class="row"><DIV class="tue-float tue-float-left span6"><DIV class="tue-caption"><SPAN class="tue-caption-figure"><SPAN class="tue-refid">pET28a_Protamine</SPAN></SPAN> <SPAN class="tue-caption-text">PCR Colony Gel for pET28a ligated constructs. Columns from left to right: 1 - 1Kb ladder, 2 - 0.1Kb ladder, 3-7 - Protamine-1-Optimized Sequences, 8-12 - 1PJN Sequences, 13-17 - 1ETF Sequences, 18-22 - Poly(Arginine-Glycine) Sequences, 23-27 - Poly(Arginine-Serine) Sequences, 28 - pET28a (1) Control, 29 - pET28a (2) Control, 30 - 0.1Kb ladder.</SPAN></DIV></DIV><DIV class="tue-float tue-float-right span6"><DIV class="tue-caption"><SPAN class="tue-caption-figure"><SPAN class="tue-refid">pBR322_Protamine</SPAN></SPAN> <SPAN class="tue-caption-text">PCR Colony Gel for pBR322 ligated constructs. Columns from left to right: 1 - 1Kb ladder, 2 - 0.1Kb ladder, 3-7 - Protamine-1-Optimized Sequences, 8-12 - 1PJN Sequences, 13-17 - 1ETF Sequences, 18-22 - Poly(Arginine-Glycine) Sequences, 23-27 - Poly(Arginine-Serine) Sequences, 28 - pBR322 (1) Control, 29 - pET28a (2) Control, 30 - 0.1Kb ladder.</SPAN></DIV></DIV></DIV><H3><SPAN class="mw-headline" id="Culturing_the_Ligation_Products">Culturing the Ligation Products</SPAN></H3><P>Having analyzed the gel and having chosen two successful ligations for each of the constructs these were then cultured. This was done in the following manner:
</P><UL><LI>All below steps were performed near a blue bunsen flame to increase sterility. 
</LI><LI>Culture tubes were filled with 8mL of LB medium.
</LI><LI>To each of the culture tubes 8µL of Kanamycin was added.
</LI><LI>5µL of the picked colony solution was pipetted into the culture medium.
</LI><LI>The culture tubes were then placed in the 37°C incubator overnight. 
</LI></UL></DIV></DIV><DIV class="tue-timeline-middle"><DIV class="tue-timeline-leftbar"><DIV class="tue-timeline-node">9 August2013</DIV></DIV><DIV class="tue-timeline-content"><H3><SPAN class="mw-headline" id="Miniprep">Miniprep</SPAN></H3><P>On August 8th a number of constructs were chosen to be cultured, multiplying the number of correctly ligated DNA vectors we had available to ourselves. To retain this DNA the miniprep protocol would be performed on each of the cultures:
</P><UL><LI>To start the miniprep protocol the culture tubes were spun down in a centrifuge for 10 minutes at 3700rpm causing the bacterial cells with the DNA inside to form pellets isolating it from the growth medium. 
</LI><LI>The supernatant that formed above the pellets was discarded and the pellets themselves were resuspended in 250µL of P1 buffer. The tubes were gently shaken by hand until the entire pellet had resuspended.
</LI><LI>The suspension was then transferred by means of pipetting into a smaller (1.5mL) eppendorf tube. 
</LI><LI>250µL of P2 buffer was then added to the eppendorf tube and the solution was inverted by hand until it had turned a clear blue. 
</LI><LI>Within 5 minutes of adding the P2 buffer 350µL of N3 buffer was added to the solution and the eppendorf tube was inverted again until the solution was clear again. 
</LI><LI>Once clear the eppendorf tube was placed in the centrifuge and spun for 10 minutes at 13000rpm seperating the bacterial cells from the DNA which remained suspended in the solution. The bacterial cells formed a pellet in the bottom of the eppendorf tube. (Unfortunately the pellet had not completely formed after 10 minutes so the eppendorf tube was spun for a further minute, again at 13000rpm.)
</LI><LI>As soon as the centrifuge had stopped and we could see that the pellet had properly formed, the supernatant was poured off into a special QIAcolumn. The pellet could be discarded in the bio-hazard waste. 
</LI><LI>The QIAcolumn was subsequently centrifuged for one minute at 13000rpm, during which the DNA vector bound to the column meaning the flow through could be discarded. 
</LI><LI>750µL of PE buffer was then added to the QIAcolumn before centrifuging it for another minute at 13000rpm. The flow through was discarded.
</LI><LI>To ensure that all the PE buffer had passed through the column it was then spun yet again for one minute at a speed of 13000rpm. Yet again the flow through could be discarded.
</LI><LI>The column itself was then placed above a sealable 1.5mL eppendorf tube so that the DNA could be contained and stored. To obtain the DNA from the column 42µL of MilliQ water was pipetted onto the very centre membrane of the column after which it was spun down for one minute at 13000 rpm. The flow through now sat in the 1.5mL eppendorf tube and contained the DNA vectors. 
</LI></UL><H3><SPAN class="mw-headline" id="Nanodrop_Results_3">Nanodrop Results</SPAN></H3><P>A useful step now, having acquired our ligated DNA vectors would be to send them off for sequencing to confirm that we had indeed ligated and cultured the correct vectors and inserts. Before being able to do this, we would need to determine the concentrations of each of the miniprep products.
</P><TABLE class="table table-striped"><TBODY><TR><TH align="center"> Vector
</TH><TH> Colony Picking Number
</TH><TH> Concentration (ng/µL)
</TH></TR><TR><TD>Protamine-1-Optimized </TD><TD> 1 </TD><TD> 60.0
</TD></TR><TR><TD>Protamine-1-Optimized </TD><TD> 2 </TD><TD> 65.9
</TD></TR><TR><TD>1PJN </TD><TD> 3 </TD><TD> 72.1
</TD></TR><TR><TD>1PJN </TD><TD> 4 </TD><TD> 66.6
</TD></TR><TR><TD>1ETF </TD><TD> 3 </TD><TD> 64.1
</TD></TR><TR><TD>1ETF </TD><TD> 4 </TD><TD> 72.8
</TD></TR><TR><TD>Poly(Arginine-Glycine) </TD><TD> 3 </TD><TD> 73.5
</TD></TR><TR><TD>Poly(Arginine-Glycine) </TD><TD> 5 </TD><TD> 117.8
</TD></TR><TR><TD>Poly(Arginine-Serine) </TD><TD> 2 </TD><TD> 206.7
</TD></TR><TR><TD>Poly(Arginine-Serine) </TD><TD> 5 </TD><TD> 57.8
</TD></TR></TBODY></TABLE><H3><SPAN class="mw-headline" id="Vector_Sequencing">Vector Sequencing</SPAN></H3><P>To sequence each of the vectors we sent off samples of our DNA to an external sequencing company. For this company to fully sequence each of our vectors we had to create two solutions for each vector we wished to sequence, one with the T7 FWD primer and one with the T7 RW primer, both with a concentration of 10µM. The second requirement was that we added between 400ng and 600ng of our DNA vectors. Unfortunately the maximal volume of the entire solution was 7µL, so for constructs with a concentration lower than 67ng/µL the minimal amount of DNA desired for sequencing was not reached. This led to the creation of the following solutions, with the same solution being made for both primers:
</P><TABLE class="table table-striped"><TBODY><TR><TH align="center"> Vector
</TH><TH> Colony Picking Number
</TH><TH> Volume Vector Solution (µL)
</TH><TH> Water (µL)
</TH><TH> Primer (µL)
</TH></TR><TR><TD>Protamine-1-Optimized </TD><TD> 1 </TD><TD> 6.0 </TD><TD> 0.0 </TD><TD> 1.0
</TD></TR><TR><TD>Protamine-1-Optimized </TD><TD> 2 </TD><TD> 6.0 </TD><TD> 0.0 </TD><TD> 1.0
</TD></TR><TR><TD>1PJN </TD><TD> 3 </TD><TD> 6.0 </TD><TD> 0.0 </TD><TD> 1.0
</TD></TR><TR><TD>1PJN </TD><TD> 4 </TD><TD> 6.0 </TD><TD> 0.0 </TD><TD> 1.0
</TD></TR><TR><TD>1ETF </TD><TD> 3 </TD><TD> 6.0 </TD><TD> 0.0 </TD><TD> 1.0
</TD></TR><TR><TD>1ETF </TD><TD> 4 </TD><TD> 6.0 </TD><TD> 0.0 </TD><TD> 1.0
</TD></TR><TR><TD>Poly(Arginine-Glycine) </TD><TD> 2 </TD><TD> 6.0 </TD><TD> 0.0 </TD><TD> 1.0
</TD></TR><TR><TD>Poly(Arginine-Glycine) </TD><TD> 5 </TD><TD> 4.0 </TD><TD> 2.0 </TD><TD> 1.0
</TD></TR><TR><TD>Poly(Arginine-Serine) </TD><TD> 3 </TD><TD> 3.0 </TD><TD> 3.0 </TD><TD> 1.0
</TD></TR><TR><TD>Poly(Arginine-Serine) </TD><TD> 5 </TD><TD> 6.0 </TD><TD> 0.0 </TD><TD> 1.0
</TD></TR></TBODY></TABLE><P>Having prepared these solutions in small PCR eppendorf tubes they were packaged and sent off for sequencing.
</P><H3><SPAN class="mw-headline" id="Colony_Picking_and_PCR_2">Colony Picking and PCR</SPAN></H3><P>On August 8th we had performed the colony picking and PCR for the constructs which had been ligated into the pET28a vector. As we did not have the primers for the pBR322 vector those colony plates were stored at 4°C. Today however the pBR322 primers had arrived and we could perform the colony pickings for the ligated pBR322 vectors.
It should be noted that the primers we used for the pBR322 vector had been designed ourselves with the following sequences:
</P><TABLE class="table table-striped"><TBODY><TR><TD><B>pBR322 FWD Primer</B></TD><TD> 5’- CCC GAA AAG TGC CAC CTG -3’
</TD></TR><TR><TD><B>pBR322 RW Primer</B></TD><TD> 5’- GT GCC GAG GAT GAC GATG -3’
</TD></TR></TBODY></TABLE><P>To then perform the colony PCR the following solutions had to be created:
</P><UL><LI>Much like for the pET28a ligated vectors we picked five colonies for each of the five constructs ligated into the pBR322 vector. These pickings were dissolved in 5µL of sterile water. Along with the five pickings per construct we also picked one colony from the control plate, which contained just the pBR322 vector. Photos of the agar plates are shown below:
</LI></UL><DIV class="row"><DIV class="tue-float tue-float-left span3"><DIV class="tue-caption"><SPAN class="tue-caption-figure"><SPAN class="tue-refid">pBR322_Protamine</SPAN></SPAN> <SPAN class="tue-caption-text">pBR322-Protamine</SPAN></DIV></DIV><DIV class="tue-float tue-float-left span3"><DIV class="tue-caption"><SPAN class="tue-caption-figure"><SPAN class="tue-refid">pBR322_1PJN1</SPAN></SPAN> <SPAN class="tue-caption-text">pBR322-1PJN</SPAN></DIV></DIV><DIV class="tue-float tue-float-left span3"><DIV class="tue-caption"><SPAN class="tue-caption-figure"><SPAN class="tue-refid">pBR322_1ETF</SPAN></SPAN> <SPAN class="tue-caption-text">pBR322-1ETF</SPAN></DIV></DIV><DIV class="tue-float tue-float-left span3"><DIV class="tue-caption"><SPAN class="tue-caption-figure"><SPAN class="tue-refid">pBR322_P(RG)</SPAN></SPAN> <SPAN class="tue-caption-text">pBR322-Poly(Arginine-Glycine)</SPAN></DIV></DIV><DIV class="tue-float tue-float-left span3"><DIV class="tue-caption"><SPAN class="tue-caption-figure"><SPAN class="tue-refid">pBR322_P(RS)</SPAN></SPAN> <SPAN class="tue-caption-text">pBR322-Poly(Arginine-Serine)</SPAN></DIV></DIV><DIV class="tue-float tue-float-left span3"><DIV class="tue-caption"><SPAN class="tue-caption-figure"><SPAN class="tue-refid">pBR322_Control</SPAN></SPAN> <SPAN class="tue-caption-text">pBR322 Control</SPAN></DIV></DIV></DIV><UL><LI>We could now go ahead and prepare our PCR solutions by mixing the following:
</LI><LI>9µL of sterile water.
</LI><LI>1.25µL of the FWD primer.
</LI><LI>1.25µL of the RW Primer.
</LI><LI>1µL of the pBR322 ligated vector colony picking solution.
</LI><LI>12.5µL of Kapa2G. 
</LI><LI>We made sure to only add the Kapa2G at the very end just before placing the solutions in the PCR. 
</LI></UL><P>Having made the above solutions they were brought across to the PCR apparatus which then performed the following PCR protocol:
</P><UL><LI>Initial Denaturation: 95°C for 3 minutes.
</LI><LI>The following steps should be performed in this order, repeating the process a total of 34 times.
</LI><LI>Denaturation: 95°C for 15 seconds.
</LI><LI>Annealing: 58°C for 15 seconds.
</LI><LI>Extension: 72°C for 20 seconds.
</LI></UL><LI>Final Extension: 72°C for 10 minutes. 
</LI><LI> Storage: 4°C FOREVER.
</LI><P>Having completed the PCR protocol 4µL of 6x Loading Dye was added to each of the PCR solutions. Subsequently 18µL of each was loaded onto a 1.5% agarose electrophoresis gel which was prepared by mixing the following:
</P><UL><LI>1.8g of Agarose.
</LI><LI>120mL of 1x TAE buffer
</LI><LI>12µL of CyberSafe. 
</LI></UL><P>The gel was then run at 100V for 75 minutes before placing the gel into a UV spectrometer. Here we could analyse the gel and decide which of the ligation products were successful. As it was a Friday afternoon there was no possibility to culture the correctly ligated vectors so these were stored in the 4°C refrigerator and left there until they could be cultured.  
</P></DIV></DIV></DIV></DIV><DIV class="tab-pane" id="Week4"><DIV><DIV class="tue-timeline-top"><DIV class="tue-timeline-title"><H2>Week 4</H2></DIV></DIV><DIV class="tue-timeline-middle"><DIV class="tue-timeline-leftbar"><DIV class="tue-timeline-node">12 August2013</DIV></DIV><DIV class="tue-timeline-content"><H3><SPAN class="mw-headline" id="Transforming_and_Plating_New_Constructs">Transforming and Plating New Constructs</SPAN></H3><P>Today we began work on four new constructs which had arrived late last week. The constructs that we had received were: EGFP, 1G70, Poly(Threonine-Lysine) and Poly(Lysine-Serine). For more information of these constructs please see the wetlab preparation page. 
The first step to be performed on each of the constructs was the preparing of two dilations. The first a 200ng/µL dilation made by adding 20µL of MilliQ water to the standard 4µg of DNA we recieved for each construct. From this dilution we then took 1µL and added it to another 199µL of MilliQ water forming a 1ng/µL solution which we could use for the transformation:
</P><UL><LI>All steps for the transformation were performed near a blue flame for sterility. 
</LI><LI>Start by adding 1µL of the 1ng/µL solution to 20µL of NB bacteria. 
</LI><LI>Place the bacteria on ice and let them stand for 5 minutes.
</LI><LI>Heat shock the bacteria by placing them in a water bath of 42°C for 30 seconds before returning them to ice for two minutes. 
</LI><LI>Now add 80µL of SOC to each sample and place the samples in a 37°C incubator for an hour. The samples should not (and were not) returned to ice. 
</LI></UL><P>This concludes the transformation stage of the constructs which are now enclosed within the NB bacteria. After incubation the bacteria can therefore be plated onto Ampicillin agar plates. 
</P><UL><LI>In the vicinity of a blue bunsen flame place pipet the entire solution of bacterial cells and SOC onto the centre of the plate. 
</LI><LI>Using a sterile spreader spread out the solution over the gel surface.
</LI><LI>The plates can now be incubated at 37°C for an overnight growth. 
</LI></UL><P>This concludes the work on the new constructs for today.
</P><H3><SPAN class="mw-headline" id="Transforming_and_Plating_Old_Constructs">Transforming and Plating Old Constructs</SPAN></H3><P>Last week a number of constructs were digested and ligated into pET28a. These new vectors were then multiplied and retained by performing a culture and miniprep respectively. These new vectors could now be transformed into BL21 bacteria so that expression could be induced using IPTG. Before embarking on the transformation however the following solutions had to be made. As you can see we no longer use all 10 ligated constructs to reduce the work load and will continue with one of each. The concentrations used were determined last week. These solutions will create 1ng/µL solutions of each of the ligated vector solutions. 
</P><TABLE class="table table-striped"><TBODY><TR><TH align="center"> Vector
</TH><TH> Colony Picking Number
</TH><TH> Concentration (ng/µL)
</TH><TH> Sterile Water (µL)
</TH><TH> Vector Solution (µL)
</TH></TR><TR><TD>Protamine-1-Optimized </TD><TD> 1 </TD><TD> 60.0 </TD><TD> 59.0 </TD><TD> 1.0
</TD></TR><TR><TD>1PJN </TD><TD> 3 </TD><TD> 72.1 </TD><TD> 71.1 </TD><TD> 1.0
</TD></TR><TR><TD>1ETF </TD><TD> 4 </TD><TD> 72.8 </TD><TD> 71.8 </TD><TD> 1.0
</TD></TR><TR><TD>Poly(Arginine-Glycine) </TD><TD> 3 </TD><TD> 73.5 </TD><TD> 72.5 </TD><TD> 1.0
</TD></TR><TR><TD>Poly(Arginine-Serine) </TD><TD> 5 </TD><TD> 57.8 </TD><TD> 56.8 </TD><TD> 1.0
</TD></TR></TBODY></TABLE><P>Having prepared the above solutions the transformation could be started:
</P><UL><LI>All steps for the transformation were performed near a blue flame for sterility. 
</LI><LI>Start by adding 1µL of the 1ng/µL solution to 20µL of BL21 bacteria. 
</LI><LI>Place the bacteria on ice and let them stand for 5 minutes.
</LI><LI>Heat shock the bacteria by placing them in a water bath of 42°C for 30 seconds before returning them to ice for two minutes. 
</LI><LI>Now add 80µL of SOC to each sample and place the samples in a 37°C incubator for an hour. The samples should not (and were not) returned to ice. 
</LI></UL><P>This concludes the transformation stage of the constructs which are now enclosed within the BL21 bacteria. After incubation the bacteria can therefore be plated onto Kanamycin agar plates. 
</P><UL><LI>In the vicinity of a blue bunsen flame place pipet the entire solution of bacterial cells and SOC onto the centre of the plate. 
</LI><LI>Using a sterile spreader spread out the solution over the gel surface.
</LI><LI>The plates can now be incubated at 37°C for an overnight growth. 
</LI></UL><P>This concludes the work on the ligated pET28a constructs for today.
</P><H3><SPAN class="mw-headline" id="Creating_Small_Cultures_2">Creating Small Cultures</SPAN></H3><P>As we were unable to produce small cultures of the old constructs ligated into the pBR322 vector last week this too would be performed today. Before beginning a new batch of LB medium was prepared: 
</P><UL><LI>The LB medium was created by mixing the following:
</LI><LI>10g Peptones
</LI><LI>10g NaCl
</LI><LI>5g Yeast Extract
</LI><LI>1L Demineralised Water
</LI></UL><LI>After adding together the above, the solution was autoclaved and left to cool. 
</LI><P>One the LB medium was ready for use the culture preparation could be performed:
</P><UL><LI>Using the electrophoresis gel from the previous week two ligated vectors were chosen to cultured for each of the five constructs. 
</LI><LI>This meant that 10 culture tubes would have to be filled with 8mL of LB medium each.
</LI><LI>To each tube a further 8µL of Ampicillin was added. 
</LI><LI>Finally 8µL of the chosen construct was added and the culture tubes were placed in the 37°C incubator for overnight growth. 
</LI></UL><P>This concluded not only the work on the pBR322 ligated constructs for today, but also the rest of the days work. 
</P></DIV></DIV><DIV class="tue-timeline-middle"><DIV class="tue-timeline-leftbar"><DIV class="tue-timeline-node">13 August2013</DIV></DIV><DIV class="tue-timeline-content"><H3><SPAN class="mw-headline" id="Miniprepping_ligated_pBR322_vector_cultures">Miniprepping ligated pBR322 vector cultures</SPAN></H3><P>On August 12th the pBR322 ligated constructs were placed in an incubator for overnight culture growth. Today we removed these cultured samples from the incubator, ready to retain the DNA. To do this the miniprep protocol would be used for each of the culture tubes. As a reminder, we had 10 culture tubes, 2 for each of the ligation constructs: Protamine-1-Optimized, 1PJN, 1ETF, Poly(Arginine-Glycine), Poly(Arginine-Serine) ligated with pBR322. The miniprep went as follows: 
</P><UL><LI>First the culture tubes were spun down in a centrifuge for 10 minutes at 3700G causing the bacterial cells with the DNA inside to form pellets isolating it from the growth medium. 
</LI><LI>The supernatant that had formed above the pellets was discarded and the pellets themselves were resuspended in 250µL of P1 buffer. The tubes were gently shaken by hand until the entire pellet had resuspended.
</LI><LI>The suspension was then transferred by means of pipetting into a smaller (1.5mL) eppendorf tube. 
</LI><LI>250µL of P2 buffer was then added to the eppendorf tube and the solution was inverted by hand until it had turned a clear blue. 
</LI><LI>Within 5 minutes of adding the P2 buffer 350µL of N3 buffer was added to the solution and the eppendorf tube was inverted again until the solution was clear again. 
</LI><LI>Once clear the eppendorf tube was placed in the centrifuge and spun for 10 minutes at 13000rpm seperating the bacterial cells from the DNA which remained suspended in the solution. The bacterial cells formed a pellet in the bottom of the eppendorf tube. (Unfortunately the pellet had not completely formed after 10 minutes so the eppendorf tube was spun for a further minute, again at 13000rpm.)
</LI><LI>As soon as the centrifuge had stopped and we could see that the pellet had properly formed, the supernatant was poured off into a special QIAcolumn. The pellet could be discarded in the bio-hazard waste. 
</LI><LI>The QIAcolumn was subsequently centrifuged for one minute at 13000rpm, during which the DNA vector bound to the column meaning the flow through could be discarded. 
</LI><LI>750µL of PE buffer was then added to the QIAcolumn before centrifuging it for another minute at 13000rpm. The flow through was discarded.
</LI><LI>To ensure that all the PE buffer had passed through the column it was then spun yet again for one minute at a speed of 13000rpm. Yet again the flow through could be discarded.
</LI><LI>The column itself was then placed above a sealable 1.5mL eppendorf tube so that the DNA could be contained and stored. To obtain the DNA from the column 42µL of MilliQ water was pipetted onto the very center membrane of the column after which it was spun down for one minute at 13000rpm. The flow through now sat in the 1.5mL eppendorf tube and contained the DNA vectors.
</LI></UL><P>The following concentrations were then measured for each of the DNA samples:
</P><TABLE class="table table-striped"><TBODY><TR><TH align="center"> Vector
</TH><TH> Colony Picking Number
</TH><TH> Concentration (ng/µL)
</TH></TR><TR><TD>Protamine-1-Optimized </TD><TD> 1 </TD><TD> 93.9
</TD></TR><TR><TD>Protamine-1-Optimized </TD><TD> 2 </TD><TD> 144.8
</TD></TR><TR><TD>1PJN </TD><TD> 1 </TD><TD> 90.5
</TD></TR><TR><TD>1PJN </TD><TD> 2 </TD><TD> 122.4
</TD></TR><TR><TD>1ETF </TD><TD> 3 </TD><TD> 86.4
</TD></TR><TR><TD>1ETF </TD><TD> 4 </TD><TD> 66.9
</TD></TR><TR><TD>Poly(Arginine-Glycine) </TD><TD> 1 </TD><TD> 71.4
</TD></TR><TR><TD>Poly(Arginine-Glycine) </TD><TD> 2 </TD><TD> 71.6
</TD></TR><TR><TD>Poly(Arginine-Serine) </TD><TD> 1 </TD><TD> 61.2
</TD></TR><TR><TD>Poly(Arginine-Serine) </TD><TD> 5 </TD><TD> 75.8
</TD></TR></TBODY></TABLE><P>We later saw that the elution for the Protamin-1-Optimized, picking 2, solution had not been successful as the volume of retained DNA was a mere 10µL. This meant that from now on only one of the Protamine samples would be available.
</P><H3><SPAN class="mw-headline" id="Sequencing">Sequencing</SPAN></H3><P>Having retained the DNA vectors we could send off samples for sequencing. Like the previous time, for each of the samples 2 sequencing solutions would have to be prepared. One would contain 1µL of our self-designed pBR322 FWD primer and the other would contain1µL of our self-designed pBR322 RW primer. 
The following solutions were therefore made: 
</P><TABLE class="table table-striped"><TBODY><TR><TH align="center"> Vector
</TH><TH> Colony Picking Number
</TH><TH> Volume Vector Solution (µL)
</TH><TH> Water (µL)
</TH><TH> Primer (µL)
</TH></TR><TR><TD>Protamine-1-Optimized </TD><TD> 1 </TD><TD> 6.0 </TD><TD> 0.0 </TD><TD> 1.0
</TD></TR><TR><TD>Protamine-1-Optimized </TD><TD> 2 </TD><TD> 4.0 </TD><TD> 2.0 </TD><TD> 1.0
</TD></TR><TR><TD>1PJN </TD><TD> 1 </TD><TD> 6.0 </TD><TD> 0.0 </TD><TD> 1.0
</TD></TR><TR><TD>1PJN </TD><TD> 2 </TD><TD> 5.0 </TD><TD> 1.0 </TD><TD> 1.0
</TD></TR><TR><TD>1ETF </TD><TD> 3 </TD><TD> 6.0 </TD><TD> 0.0 </TD><TD> 1.0
</TD></TR><TR><TD>1ETF </TD><TD> 4 </TD><TD> 6.0 </TD><TD> 0.0 </TD><TD> 1.0
</TD></TR><TR><TD>Poly(Arginine-Glycine) </TD><TD> 1 </TD><TD> 6.0 </TD><TD> 0.0 </TD><TD> 1.0
</TD></TR><TR><TD>Poly(Arginine-Glycine) </TD><TD> 2 </TD><TD> 6.0 </TD><TD> 0.0 </TD><TD> 1.0
</TD></TR><TR><TD>Poly(Arginine-Serine) </TD><TD> 1 </TD><TD> 6.0 </TD><TD> 0.0 </TD><TD> 1.0
</TD></TR><TR><TD>Poly(Arginine-Serine) </TD><TD> 5 </TD><TD> 6.0 </TD><TD> 0.0 </TD><TD> 1.0
</TD></TR></TBODY></TABLE><P>As mentioned there was only a small elution volume for the Protamine-1-Optimized, picking 2, DNA solution, so we could only prepare the FWD sequencing sample. For the reverse solution the DNA quantity was too low. 
</P><P>As we would only be needing the pBR322 constructs for anaerobic expression, which would only come after we had tested the CEST contrast of each construct, these DNA vectors were placed in the -20°C freezer and were to be left there until needed again in a later stage of this project.
</P><H3><SPAN class="mw-headline" id="Culturing_New_Constructs">Culturing New Constructs</SPAN></H3><P>On August 12th work began on four new constructs which had arrived: EGFP, 1G70, Poly(Threonine-Lysine) and Poly(Lysine-Serine). We had transformed each of the constructs into NovaBlue bacteria and had left these to grow overnight on Ampicillin agar plates. Today we would pick a number of grown colonies and culture them in small cultures. To do this the following steps were taken:
</P><UL><LI>All the following steps were performed near a blue flame for increased sterility.
</LI><LI>Fill 16 cultures tubes with 8mL of LB medium.
</LI><LI>Add 8µL of Ampicillin to each of the culture tubes.
</LI><LI>From each of the four culture plates 2 colonies were picked and dissolved in 15µL of sterile water. Pictures of the plates are given below. 
</LI></UL><DIV class="row"><DIV class="tue-float tue-float-left span3"><DIV class="tue-caption"><SPAN class="tue-caption-figure"><SPAN class="tue-refid">pUC57-Simple_EGFP</SPAN></SPAN> <SPAN class="tue-caption-text">pUC57-Simple-EGFP</SPAN></DIV></DIV><DIV class="tue-float tue-float-left span3"><DIV class="tue-caption"><SPAN class="tue-caption-figure"><SPAN class="tue-refid">pUC57-Simple_1G70</SPAN></SPAN> <SPAN class="tue-caption-text">pUC57-Simple-1G70</SPAN></DIV></DIV><DIV class="tue-float tue-float-left span3"><DIV class="tue-caption"><SPAN class="tue-caption-figure"><SPAN class="tue-refid">pUC57-Simple_P(TK)</SPAN></SPAN> <SPAN class="tue-caption-text">pUC57-Simple-Poly(Theronine-Lysine)</SPAN></DIV></DIV><DIV class="tue-float tue-float-left span3"><DIV class="tue-caption"><SPAN class="tue-caption-figure"><SPAN class="tue-refid">pUC57-Simple_P(KS)</SPAN></SPAN> <SPAN class="tue-caption-text">pUC57-Simple-Poly(Lysine-Serine)</SPAN></DIV></DIV></DIV><UL><LI>7.5µL of this bacterial picking solution was then pipetted into one of the culture tubes, until all tubes contained a portion of bacteria. Great care was taken in the labelling of the cultures so we could trace back which picking had been inserted into which tube.
</LI><LI>The culture tubes were then placed in the 37°C incubator and left for overnight growth.  
</LI></UL><P>It should be noted that four culture tubes were created per sample to ensure a large enough DNA amplification to be able to perform a restriction, gel extraction and ligation later on. Previously we had made 12 of these cultures per construct but it turned out that that was far too much (a miscalculation on our part), therefore we chose to create only four 8mL cultures per construct.
</P><H3><SPAN class="mw-headline" id="Culturing_Old_Constructs">Culturing Old Constructs</SPAN></H3><P>Apart from just transforming and plating the new constructs mentioned above, we had also transformed and plated the older pET28a ligated constructs on August 12th. These too had been left to grow overnight and we were now ready to continue work on these samples. Photos are given here:
</P><DIV class="row"><DIV class="tue-float tue-float-left span3"><DIV class="tue-caption"><SPAN class="tue-caption-figure"><SPAN class="tue-refid">pET28a_Protamine_BL21</SPAN></SPAN> <SPAN class="tue-caption-text">pET28a-Protamine-BL21</SPAN></DIV></DIV><DIV class="tue-float tue-float-left span3"><DIV class="tue-caption"><SPAN class="tue-caption-figure"><SPAN class="tue-refid">pET28a_1PJN_BL21</SPAN></SPAN> <SPAN class="tue-caption-text">pET28a-1PJN-BL21</SPAN></DIV></DIV><DIV class="tue-float tue-float-left span3"><DIV class="tue-caption"><SPAN class="tue-caption-figure"><SPAN class="tue-refid">pET28a_1ETF_BL21</SPAN></SPAN> <SPAN class="tue-caption-text">pET28a-1ETF-BL21</SPAN></DIV></DIV><DIV class="tue-float tue-float-left span3"><DIV class="tue-caption"><SPAN class="tue-caption-figure"><SPAN class="tue-refid">pET28a_P(RG)_BL21</SPAN></SPAN> <SPAN class="tue-caption-text">pET28a-Poly(Arginine-Glycine)-BL21</SPAN></DIV></DIV><DIV class="tue-float tue-float-left span3"><DIV class="tue-caption"><SPAN class="tue-caption-figure"><SPAN class="tue-refid">pET28a_P(RS)_BL21</SPAN></SPAN> <SPAN class="tue-caption-text">pET28a-Poly(Arginine-Serine)-BL21</SPAN></DIV></DIV></DIV>
As we wished to move onto the expression of our proteins we first wished to perform some tests to see how best to perform the expression (which environment we would need). Therefore it was decided that of the five constructs we had ligated with the pET28a factor and transformed into BL21 before plating, only two would be cultured further for the time being. Looking ahead we knew that we would like to perform expression on August 13th which meant growing the bacteria today, and adding IPTG tomorrow when the Optical Density (from now on: OD) was satisfactory. 
It was decided that a 100mL culture would be best suited to our desires as we would then be able to perform a large number of tests on the medium. The steps then for growing the bacteria in this 100mL culture was as follows:
<UL><LI>Firstly two 100mL culture mediums were created by mixing the following in two small Erlenmeyer flask: 
</LI><LI>1g Peptones
</LI><LI>1g NaCl
</LI><LI>0.5g Yeast Extract
</LI><LI>100mL Sterile Water
</LI></UL><LI>The above mentioned solutions were then placed in the autoclave to be sterilised. 
</LI><LI>After having cooled down 100µL of Kanamycin antibiotics were added to each of the flasks.
</LI><LI>Subsequently a single colony was picked from each of the following agar plates: Protamine-1-Optimized-pET28a and Poly(Arginine-Glycine)-pET28a. The two colonies were then transferred to one of the two flasks by ejecting the pipet point into the medium. 
</LI><LI>The culture flasks were then placed in the 37°C incubator and left there to grow overnight. 
</LI></DIV></DIV><DIV class="tue-timeline-middle"><DIV class="tue-timeline-leftbar"><DIV class="tue-timeline-node">14 August2013</DIV></DIV><DIV class="tue-timeline-content"><H3><SPAN class="mw-headline" id="Miniprepping_New_Constructs">Miniprepping New Constructs</SPAN></H3><P>On August 13th the four newest constructs (EGFP, 1G70, Poly(Threonine-Lysine) and Poly(Lysine-Serine)) were being cultured and were therefore placen in a 37°C incubator overnight. It was now time to retain the increased DNA using the miniprep protocol:
</P><UL><LI>First the culture tubes were spun down in a centrifuge for 10 minutes at 3700G causing the bacterial cells with the DNA inside to form pellets isolating it from the growth medium. 
</LI><LI>The supernatant that had formed above the pellets was discarded and the pellets themselves were resuspended in 250µL of P1 buffer. The tubes were gently shaken by hand until the entire pellet had resuspended.
</LI><LI>The suspension was then transferred by means of pipetting into a smaller (1.5mL) eppendorf tube. 
</LI><LI>250µL of P2 buffer was then added to the eppendorf tube and the solution was inverted by hand until it had turned a clear blue. 
</LI><LI>Within 5 minutes of adding the P2 buffer 350µL of N3 buffer was added to the solution and the eppendorf tube was inverted again until the solution was clear again. 
</LI><LI>Once clear the eppendorf tube was placed in the centrifuge and spun for 10 minutes at 13000rpm seperating the bacterial cells from the DNA which remained suspended in the solution. The bacterial cells formed a pellet in the bottom of the eppendorf tube. (Unfortunately the pellet had not completely formed after 10 minutes so the eppendorf tube was spun for a further minute, again at 13000rpm.)
</LI><LI>As soon as the centrifuge had stopped and we could see that the pellet had properly formed, the supernatant was poured off into a special QIAcolumn. The pellet could be discarded in the bio-hazard waste. 
</LI><LI>The QIAcolumn was subsequently centrifuged for one minute at 13000rpm, during which the DNA vector bound to the column meaning the flow through could be discarded. 
</LI><LI>750µL of PE buffer was then added to the QIAcolumn before centrifuging it for another minute at 13000rpm. The flow through was discarded.
</LI><LI>To ensure that all the PE buffer had passed through the column it was then spun yet again for one minute at a speed of 13000rpm. Yet again the flow through could be discarded.
</LI><LI>The column itself was then placed above a sealable 1.5mL eppendorf tube so that the DNA could be contained and stored. To obtain the DNA from the column 42µL of MilliQ water was pipetted onto the very center membrane of the column after which it was spun down for one minute at 13000rpm. The flow through now sat in the 1.5mL eppendorf tube and contained the DNA vectors.
</LI></UL><P>The following concentrations were then measured for each of the DNA samples:
</P><TABLE class="table table-striped"><TBODY><TR><TH align="center"> Vector
</TH><TH> Concentration (ng/µL)
</TH></TR><TR><TD>EGFP </TD><TD> 208.9
</TD></TR><TR><TD>1G70 </TD><TD> 165.6
</TD></TR><TR><TD>Poly(Threonine-Lysine) </TD><TD> 139.7
</TD></TR><TR><TD>Poly(Lysine-Serine) </TD><TD> 136.5
</TD></TR></TBODY></TABLE><H3><SPAN class="mw-headline" id="Digestion_2">Digestion</SPAN></H3><P>As we did for the previous batch of constructs we would now digest these newest constructs. After digestion we would then, as we did previously for the older constructs have to perform gel extraction to purify the digestion products of these constructs. This would mean digesting a total of 20µg of each construct. This would provide us with 10µg to digest for ligation into the pET28a vector and 10µg of to digest for ligation into the pBR322 vector. For the former digestion the XhoI and NheI restriction enzymes would be used. For the latter digestion we would use the EcoRI and HindIII restriction enxymes, to later allow us to ligate the restriction product into the pBR322 vector. 
As digestion is not optimal when 10µ of DNA are digested in a single solution each construct would be digested in four separate solutions each containing 5µg. Two of these solutions would contain the enzymes for ligation  into the pET28a vector and the remaining 2 solutions the enzymes for ligation in pBR322. 
The Complete solution for the digestions are given below:
</P><TABLE class="table table-striped"><TBODY><TR><TH align="center"> Construct
</TH><TH> Concentration (ng/µL)
</TH><TH> Volume for 5µg of DNA (µL)
</TH><TH> Cutsmart Buffer (µL)
</TH><TH> Enzyme (µL)
</TH><TH> Sterile Water (µL)
</TH></TR><TR><TD>EGFP </TD><TD> 208.9 </TD><TD> 23.9 </TD><TD> 5.0 </TD><TD> 0.7 </TD><TD> 19.7
</TD></TR><TR><TD>1G70 </TD><TD> 165.6 </TD><TD> 30.2 </TD><TD> 5.0 </TD><TD> 0.7 </TD><TD> 13.4
</TD></TR><TR><TD>Poly(Threonine-Lysine) </TD><TD> 139.7 </TD><TD> 35.8 </TD><TD> 5.0 </TD><TD> 0.7 </TD><TD> 7.8
</TD></TR><TR><TD>Poly(Lysine-Serine) </TD><TD> 136.5 </TD><TD> 36.6 </TD><TD> 5.0 </TD><TD> 0.7 </TD><TD> 7.0
</TD></TR></TBODY></TABLE><P>As mentioned two restriction enzymes were added to each solution. The given Enzyme volume is the amount that should be added of each of the two enzymes. 
It should also be noted that for the Poly(Threonine-Lysine) and Poly(Lysine-Serine) there was a slight deficit in the amount of DNA available for one of the solutions. This meant that one solution did not contain 5µg of DNA but slightly less (approx. 4.5µg).
</P><P>We also had to digest some more of both the vectors. For each vector we aimed to digest a total of 2µg. Therefore the following solutions were made:
</P><TABLE class="table table-striped"><TBODY><TR><TH align="center"> Vector
</TH><TH> Volume for 5µg of DNA (µL)
</TH><TH> Cutsmart Buffer (µL)
</TH><TH> Enzyme (µL)
</TH><TH> Sterile Water (µL)
</TH></TR><TR><TD>pET28a </TD><TD> 18.5 </TD><TD> 5.0 </TD><TD> 0.7 </TD><TD> 25.8
</TD></TR><TR><TD>pBR322 </TD><TD> 32.2 </TD><TD> 5.0 </TD><TD> 0.7 </TD><TD> 12.1
</TD></TR></TBODY></TABLE><P>Here we used the vector DNA that was prepared in the earlier stages (see August 2nd). 
</P><P>These solutions (Both for the vectors as well as the constructs were then placed in the PCR apparatus and set witht he following time schedule:
</P><UL><LI>Reaction step: 1 hour at 37°C
</LI><LI>Enzyme inactivation step: 20 minutes at 80°C
</LI><LI>Storage step: Forever at 4°C 
</LI></UL><P>The restriction products were then run and left at 4°C until they were needed again the next day.
</P><H3><SPAN class="mw-headline" id="Testing_the_Expression">Testing the Expression</SPAN></H3><P>On August 13th 2 cell cultures had been placed in the 37°C incubator to grow the bacteria overnight. It was now time to induce the expression. This would be done aerobically for the time being. 
It is important to test the protein expression on a small scale to start with so that you reduce risks when scaling up. It was decided that once we had induced expression by addition of IPTG that we would take samples of the expression medium every hour to get a good view of the expression curve. These samples would be collected today and analysed tomorrow. 
The following procedure was followed in order to obtain our samples:
</P><UL><LI>An Optical density measurement of the cell culture was taken to check the cell growth. An optical density above 0.6 would mean that expression induction would be possible. The following optical densities were found:
</LI><LI>Protamine culture: 2.140
</LI><LI>Poly(Arginine-Glycine) culture: 2.287
</LI></UL><LI>For this reason 50µL of IPTG was added to each of the cultures. After having placed each culture in the incubator for a further 15 minutes it occurred to us that an optical density above 2 might be far to high for proper expression as the medium would not contain large enough food supplies. The following emergency procedure was then performed with haste: 
</LI><LI>First three test sample tubes were made each with a different amount of new LB medium and a small amount of our existing cell culture:
</LI><LI>11mL LB and 1mL cell culture
</LI><LI>10mL LB and 2mL cell culture
</LI><LI>8mL LB and 4mL cell culture
</LI><LI>These three test tubes were then measured for their optical density. It was revealed that a dilution containing 10mL of LB medium and 2mL of our existing cell culture was optimal. Therefore 2 new culture tubes were filled with 10mL's of LB medium. To each we then added 2.5mL of one of the two cell cultures. We also added 12.5µg of Kanamycin antibiotics and an extra 5µL of IPTG to account for the dilution factor. The optical densities were then measured of these new culture tubes:
</LI><LI>Protamine cell culture: 0.50
</LI><LI>Poly(Arginine-Glycine) cell culture: 0.56
</LI><LI>These Optical densities were adequate and therefore a t=0 sample was collected by pipetting 1.5mL of the cell culture into an eppendorf tube which was subsequently placed in the 4°C refrigerator. 
</LI><LI>The new cell cultures were then placed in the 37°C incubator and 1.5mL samples of each were taken every hour from then on for a further 5 hours. Each sample was immediately placed in the 4°C refrigerator.
</LI><H3><SPAN class="mw-headline" id="Preparing_an_SDS-gel">Preparing an SDS-gel</SPAN></H3><P>Tomorrow we would like to run a number of samples on SDS gel to see whether the protein expression had been successful. In preparation of this we prepared two 12% SDS gels. 
To start with two solutions were prepared, a running and a stacking gel solution. The recipes for both are given below:
</P><TABLE class="table table-striped"><TBODY><TR><TH> 12% Running Gel (mL)
</TH><TH> Stacking Gel (mL)
</TH></TR><TR><TD>Sterile Water </TD><TD> 9.9 </TD><TD> 5.5
</TD></TR><TR><TD>30% Acrylamide Mix </TD><TD> 12 </TD><TD> 13
</TD></TR><TR><TD>1.5M Tris-HCl Buffer (pH 8.8) </TD><TD> 7.5 </TD><TD> 0
</TD></TR><TR><TD>1M Tris-HCl Buffer (pH 6.8) </TD><TD> 0 </TD><TD> 1
</TD></TR><TR><TD>10% SDS </TD><TD> 0.3 </TD><TD> 0.08
</TD></TR><TR><TD>10% Ammonium Persulfate </TD><TD> 0.3</TD><TD> 0.08
</TD></TR><TR><TD>TEMED </TD><TD> 0.012 </TD><TD> 0.008
</TD></TR></TBODY></TABLE><P>It should be remembered that the 10% Ammonium Persulfate and the TEMED should only be added once ready to pour, these will ensure the solutions harden into gels. 
</P><P>Having made the above solutions they were poured between too securely held glass plates. First the 12% running gel was poured and topped off with water to level it out. Once hard, the water could be poured off and the stacking gel could be added and allowed to hardened. Once hard the gels were wrapped in damp tissues and then aluminium foil. The entire package was then placed in the 4°C refrigerator overnight for use tomorrow.  
</P></DIV></DIV><DIV class="tue-timeline-middle"><DIV class="tue-timeline-leftbar"><DIV class="tue-timeline-node">15 August2013</DIV></DIV><DIV class="tue-timeline-content"><H3><SPAN class="mw-headline" id="Running_the_SDS-Gel">Running the SDS-Gel</SPAN></H3><P>On August 14th an SDS gel was prepared so that we would be able to run our expression samples today. Before being able to load the expression samples these first had to be prepared:
</P><UL><LI>We had taken 6 1.5mL samples for each of the two expression cultures (Protamine-1-Optimized and Poly(Arginine-Glycine)). For each of the samples we wished to isolate the bacteria containing our proteins from the LB medium. This was done by centrifuging the samples at 13400 rpm in a tabletop centrifuge for 8 minutes. 
</LI><LI>During the centrifuging a pellet had formed in the bottom of the eppendorf tube. This contained the bacteria and therefore also our proteins. The supernatant could therefore be discarded. 
</LI><LI>To isolate our proteins from the bacteria we would have to add both BugBuster and Benzonse. For BugBuster 2mL was added for every 100mL of culture we wished to re-suspend. For Benzonase we would have to add 1/1000th of the volume of the BugBuster that we added. As this would be a very small amount for each of the samples separately a master mix was made by mixing 500µL of Bugbuster with 0.5µL of Benzonase. 
</LI><LI>Of this master mix 30*micro;L was added to each of the pellets. These were then placed in a tabletop shaker for 25-30 minutes at 25°C to resuspend the pellet. 
</LI><LI>Once resuspended the solution was spun down again for 5 minutes at 13400 rpm in a tabletop centrifuge. Once more a pellet of bacteria formed only this time the proteins could be found in the supernatant. A portion of this supernatant would therefore be loaded onto the SDS gel. However it was also possible that the proteins we wanted to see remained in the inclusion bodies in the bacteria, therefore we would also have to run these on gel. 
</LI><LI>15µL of the supernatant was pipetted into a separate eppendorf tube ready for loading. The rest of the supernatant was then discarded. 
</LI><LI>100µL of pH 8.8 buffer was then added to the pellet which resuspended. Of this solution 15µL was also pipetted into a separate eppendorf tube also ready for gel loading. 
</LI><LI>For each of the two cultures we now had 12 samples. six of these were supernatant samples containing primarily proteins, the other six were solutions containing dissolved bacterial pellets. Each sample tube contained 15µL of solution. 
</LI><LI>TO be able to see the samples on the gel a loading dye would have to be added to each sample. The loading dye was prepared by combining the following:
</LI><LI>500µL of Loading Dye
</LI><LI>1µL of DTT
</LI></UL><LI>This created a 2x loading buffer that could be added to all the samples. 
</LI><LI>15µL of the loading dye solution was then added to each of the samples which were subsequently ready for loading. 
</LI><LI>11µL of each sample was loaded into the separate columns of the 12% SDS gel which was subsequently run with the following program:
</LI><LI>15 minutes at 90V for the stacking equality.
</LI><LI>40 minutes at 150V through the running gel. 
</LI><LI>After running the gels they were shortly washed with water three times before being placed in a water bucket on the shaking table for 15 minutes. 
</LI><LI>Hereafter the water was removed and the gels were paced in Coomassie Blue staining medium for two hours.
</LI><LI>The gels were then returned to water and left on the shaking table overnight.
</LI><DIV class="row"><DIV class="tue-float tue-float-left span6"><DIV class="tue-caption"><SPAN class="tue-caption-figure"><SPAN class="tue-refid">SDSGel508Pellet</SPAN></SPAN> <SPAN class="tue-caption-text">SDS gel with: 1: ladder, 2-7: Protamine pellet, 8-13: P(RG) pellet , 14: ladder</SPAN></DIV></DIV><DIV class="tue-float tue-float-right span6"><DIV class="tue-caption"><SPAN class="tue-caption-figure"><SPAN class="tue-refid">Supernatant</SPAN></SPAN> <SPAN class="tue-caption-text">SDS gel with: 1: ladder, 2-7: Protamine supernatant, 8: ladder, 9-14: P(RG) supernatant</SPAN></DIV></DIV></DIV><H3><SPAN class="mw-headline" id="Digestion_Purification">Digestion Purification</SPAN></H3><P>On August 14th the new construct were digested along with the vecotrs pBR322 and pET28a. The constructs would have to be purified by performing a gel extraction whereas the vectors could be purified simply by performing the PCR purification protocol. 
</P><H4><SPAN class="mw-headline" id="Purifying_the_Vectors">Purifying the Vectors</SPAN></H4><P>To purify the vectors after digestion, in preparation of the ligation the following steps were followed:
</P><UL><LI>To the PCR product 5 volumes of PB buffer were added (In our case this was 250µL of PB buffer as we had 50µL of PRC product.)
</LI><LI>This solution was placed in a QIAquick spin column and spun for 60 seconds at 13000 rpm. The flow through could be discarded.
</LI><LI>Now 750µL of PE buffer could be added to the column before spinning it down once more for 60 seconds at 13000 rpm.
</LI><LI>Having discarded the flow through the column was spun for a further 60 seconds again at 13000 rpm to remove the last of the PE buffer.
</LI><LI>The column was then placed into a sealable eppendorf tube before adding 42µL of MilliQ water to the very centre membrane of the spin column and leaving it to stand for one minute.
</LI><LI>The column was now placed in the centrifuge and spun for 60 seconds at 13000 rpm producing a solution containing the desired DNA.
</LI></UL><H4><SPAN class="mw-headline" id="Purifying_the_Inserts">Purifying the Inserts</SPAN></H4><P>The inserts too would need to be purified however as they were quite large this would not be possible using the PCR purification kit. This meant that we would have to load and extract the digestion products on an agarose gel. For each of the four constructs (EGFP, 1G70, Poly(Threonine-Lysine), Poly(Lysine-Serine)) we had four PCR products. Two of these had been digested for ligation into a pET28a vector and the other two solutions had been digested for ligation into pBR322. 
</P><UL><LI>The first step was the combination of the two solutions per construct which had been digested for ligation into the same vector. 
</LI><LI>After combination a total volume of 100µL had been attained. Therefore 20µL of 6x Loading Dye was added to each solution. 
</LI><LI>This entire solution was then loaded into a single column on a 1% agarose gel prepared by mixing the following:
</LI><LI>1.2g of Agarose
</LI><LI>120mL of TAE buffer
</LI><LI>12µL of CyberSafe
</LI></UL><LI>These gels were poured and let to harden before the samples were loaded into specially broadened columns. 
</LI><LI>Once fully loaded the gel was run at 100V for an hour.
</LI><DIV class="row"><DIV class="tue-float tue-float-left span6"><DIV class="tue-caption"><SPAN class="tue-caption-figure"><SPAN class="tue-refid">GelextractionLeft1508</SPAN></SPAN> <SPAN class="tue-caption-text">pBR322 constructs; EGFP, 1G70, P(TK) and P(KS)</SPAN></DIV></DIV><DIV class="tue-float tue-float-right span6"><DIV class="tue-caption"><SPAN class="tue-caption-figure"><SPAN class="tue-refid">GelextractionRight1508</SPAN></SPAN> <SPAN class="tue-caption-text">pET28a constructs; EGFP, 1G70, P(TK) and P(KS)</SPAN></DIV></DIV></DIV><P>After running the gel, gel extraction could be performed in the following manner:
</P><UL><LI>First the gels were placed on a blue light to light up the bands and the correct bands were cut out. (These were the lower bands as the inserts had a smaller base length than the construct they were restricted out of). 
</LI><LI>A gel extraction column can hold a maximum of 400mg of gel. The gel cut-outs all weighed more than this and therefore each of the bands were split into two halves. Each half was placed in a separate 2mL eppendorf tube and weighed. 
</LI><LI>To each tube 3 volumes of QG buffer was added. A volume can be defined as the weight of the gel in mg transferred into µL. So  a gel weighing 400mg would have a volume of 400µL, so 1200µL of QG would be added. 
</LI><LI>The eppendorf tubes containing the gel cut-outs and the QG buffer were placed in an incubator set at 50°C for 10 minutes, whilst vortexing the samples at regular (every 2 minutes) intervals until the gel had completely dissolved. 
</LI><LI>To these tubes 1 volume of isopropanol was added.  
</LI><LI>Each tube was then brought over into a QIAquick column and placed in the centrifuge where it was spun down for 60 seconds at 13000 rpm. The flow through was then discarded. If the volume of the gel, QG buffer and Isopropanol was larger than 800µL then this step was performed twice in the same column. 
</LI><LI>Now 750µL of PE buffer was added to the column before centrifuging at 13000 rpm for 60 seconds. After discarding the flow through the same amount was added and once more and again centrifuged.
</LI><LI>Removing the flow through the column was then centrifuged again for 60 seconds at 13000 rpm removing the last of the PE buffer. 
</LI><LI>After bringing the column across into a sealable eppendorf tube 36µL of MilliQ water was pipetted onto the very centre membrane of the column which was then let to stand for 60 seconds. Subsequently the column was spun down at 13000 rpm for 60 seconds. The DNA is now in the flow through. 
</LI><LI>Remembering that we split the gel in half in the very first stage of this protocol, the two solutions containing the same DNA were combined so that a total volume of maximally 72µL of each insert was formed.
</LI></UL><H3><SPAN class="mw-headline" id="Nanodrop_Results_4"> Nanodrop Results</SPAN></H3><P>For the ligation the concentrations of both the digested vectors and the digested inserts would have to be measured. The results can be seen in the tables below. The first table gives the concentration of both the vectors. The second gives the results for the digestion for ligation into pET28a and the third the results for ligation into pBR322
</P><TABLE class="table table-striped"><TBODY><TR><TH align="center"> Vector
</TH><TH> Concentration (ng/µL)
</TH></TR><TR><TD>pET28a </TD><TD> 13.2
</TD></TR><TR><TD>pBR322 </TD><TD> 5.2
</TD></TR></TBODY></TABLE><TABLE class="table table-striped"><TBODY><TR><TH align="center"> pET28a Digested Inserts
</TH><TH> Concentration (ng/µL)
</TH></TR><TR><TD>EGFP </TD><TD> 17.6
</TD></TR><TR><TD>1G70 </TD><TD> 8.8
</TD></TR><TR><TD>Poly(Threonine-Lysine) </TD><TD> 9.5
</TD></TR><TR><TD>Poly(Lysine-Serine) </TD><TD> 11.5
</TD></TR></TBODY></TABLE><TABLE class="table table-striped"><TBODY><TR><TH align="center"> pBR322 Digested Insert
</TH><TH> Concentration (ng/µL)
</TH></TR><TR><TD>EGFP </TD><TD> 9.6
</TD></TR><TR><TD>1G70 </TD><TD> 6.0
</TD></TR><TR><TD>Poly(Threonine-Lysine) </TD><TD> 3.9
</TD></TR><TR><TD>Poly(Lysine-Serine) </TD><TD> 3.2
</TD></TR></TBODY></TABLE><P>As we can see the digestion was unsuccessful for the pBR322 vector. Its concentration is too low to be of use for the ligation which we performed next. Luckily however we had an older pBR322 digested sample in the freezer with a concentration of 13.1ng/µL which we could use.
</P><H3><SPAN class="mw-headline" id="Ligation_2">Ligation</SPAN></H3><P>Having determined the concentrations of each of the constructs and the vectors we could start performing the calculations for the ligations. 
We wished to ligated 100ng of each of the vectors with each of the accordingly digested inserts. Ligation works best when there is more insert available than there is vector. Therefore we would first need to find out how many ng of insert will be needed for molar equality. This is done by dividing 100ng (the amount of vector we are using) by the number of bases in the vector. This amount would then be multiplied by the number of bases in the insert to reveal the mass in ng of insert we would need. 
Using this and with the concentrations of each insert at hand we could then determine the volume of DNA we would need to add to obtain a molar equality. Multiplying this volume by five would ensure a multitude of insert relative to the number of vectors in the ligation solutions. 
</P><P>Here the volume of each vector solution needed for 100ng of DNA:
</P><TABLE class="table table-striped"><TBODY><TR><TH align="center"> Vector
</TH><TH> Concentration (ng/µL)
</TH><TH> 100g DNA (µL)
</TH><TH> Number of Bases in Vector
</TH></TR><TR><TD>pBR322</TD><TD> 13.1 </TD><TD> 7.6 </TD><TD> 4330
</TD></TR><TR><TD>pET28a</TD><TD> 13.2 </TD><TD> 7.6 </TD><TD> 5296
</TD></TR></TBODY></TABLE><P>The following table gives the volumes needed for 5x molar equality of each of the inserts for the pET28a vector:
</P><TABLE class="table table-striped"><TBODY><TR><TH align="center"> Construct
</TH><TH> Number of Bases
</TH><TH> Concentration (ng/µL)
</TH><TH> ng Needed for Molar Equality (ng)
</TH><TH> Volume Needed for Molar Equality (µL)
</TH><TH> Volume Needed for 5x Molar Equality (µL)
</TH></TR><TR><TD>EGFP </TD><TD> 732 </TD><TD> 9.6 </TD><TD> 13.8 </TD><TD> 1.44 </TD><TD> 7.1
</TD></TR><TR><TD>1G70 </TD><TD> 417 </TD><TD> 6.0 </TD><TD> 12.7 </TD><TD> 2.12 </TD><TD> 10.6
</TD></TR><TR><TD>Poly(Threonine-Lysine) </TD><TD> 447 </TD><TD> 3.9 </TD><TD> 8.4 </TD><TD> 2.15 </TD><TD> 10.8
</TD></TR><TR><TD>Poly(Lysine-Serine) </TD><TD> 447 </TD><TD> 3.2 </TD><TD> 8.4 </TD><TD> 2.63 </TD><TD> 13.1
</TD></TR></TBODY></TABLE><P>
The following table gives the volumes needed for 5x molar equality of each of the inserts for the pBR322 vector: 
</P><TABLE class="table table-striped"><TBODY><TR><TH align="center"> Construct
</TH><TH> Number of Bases
</TH><TH> Concentration (ng/µL)
</TH><TH> ng Needed for Molar Equality (ng)
</TH><TH> Volume Needed for Molar Equality (µL)
</TH><TH> Volume Needed for 5x Molar Equality (µL)
</TH></TR><TR><TD>EGFP </TD><TD> 1056 </TD><TD> 9.6 </TD><TD> 13.8 </TD><TD> 1.44 </TD><TD> 7.1
</TD></TR><TR><TD>1G70 </TD><TD> 741 </TD><TD> 8.8 </TD><TD> 17.1 </TD><TD> 1.94 </TD><TD> 9.7
</TD></TR><TR><TD>Poly(Threonine-Lysine) </TD><TD> 771 </TD><TD> 9.5 </TD><TD> 17.8 </TD><TD> 1.87 </TD><TD> 9.4
</TD></TR><TR><TD>Poly(Lysine-Serine) </TD><TD> 771 </TD><TD> 11.5 </TD><TD> 17.8 </TD><TD> 1.55 </TD><TD> 7.7
</TD></TR></TBODY></TABLE><P>Using the above tables we now had all the information to be able to prepare our ligation solutions:
</P><TABLE class="table table-striped"><TBODY><TR><TH align="center"> pET28a Ligation Solutions
</TH><TH> Vector Volume (µL)
</TH><TH> Insert Volume (µL)
</TH><TH> 10x Buffer (µL)
</TH><TH> Sterile Water (µL)
</TH><TH> Ligase (µL)
</TH></TR><TR><TD>EGFP </TD><TD> 7.6 </TD><TD> 7.1 </TD><TD> 2.5 </TD><TD> 6.8 </TD><TD> 1
</TD></TR><TR><TD>1G70 </TD><TD> 7.6 </TD><TD> 10.6 </TD><TD> 2.5 </TD><TD> 3.3 </TD><TD> 1
</TD></TR><TR><TD>Poly(Threonine-Lysine) </TD><TD> 7.6 </TD><TD> 10.8 </TD><TD> 2.5 </TD><TD> 3.1 </TD><TD> 1
</TD></TR><TR><TD>Poly(Lysine-Serine) </TD><TD> 7.6 </TD><TD> 13.1 </TD><TD> 2.5 </TD><TD> 0.8 </TD><TD> 1
</TD></TR></TBODY></TABLE><TABLE class="table table-striped"><TBODY><TR><TH align="center"> pBR322 Ligation Solutions
</TH><TH> Vector Volume (µL)
</TH><TH> Insert Volume (µL)
</TH><TH> 10x Buffer (µL)
</TH><TH> Sterile Water (µL)
</TH><TH> Ligase (µL)
</TH></TR><TR><TD>EGFP </TD><TD> 7.6 </TD><TD> 6.1 </TD><TD> 2.5 </TD><TD> 7.0 </TD><TD> 1
</TD></TR><TR><TD>1G70 </TD><TD> 7.6 </TD><TD> 9.7 </TD><TD> 2.5 </TD><TD> 4.2 </TD><TD> 1
</TD></TR><TR><TD>Poly(Threonine-Lysine) </TD><TD> 7.6 </TD><TD> 9.4 </TD><TD> 2.5 </TD><TD> 4.5 </TD><TD> 1
</TD></TR><TR><TD>Poly(Lysine-Serine) </TD><TD> 7.6 </TD><TD> 7.7 </TD><TD> 2.5 </TD><TD> 7.2 </TD><TD> 1
</TD></TR></TBODY></TABLE><P>Having prepared the above 25µL ligation solutions they were placed in the PCR apparatus with the following program:
</P><UL><LI>Ligation: 2 hours at 25°C
</LI><LI>Denaturation: 10 minutes at 65°C
</LI><LI>Storage: FOREVER at 4°C
</LI></UL><P>The ligation products were then left in the PCR apparatus at 4°C overnight ready for transformation and plating the following day. 
</P></DIV></DIV><DIV class="tue-timeline-middle"><DIV class="tue-timeline-leftbar"><DIV class="tue-timeline-node">16 August2013</DIV></DIV><DIV class="tue-timeline-content"><H3><SPAN class="mw-headline" id="Transforming_the_Ligation_Products">Transforming the Ligation Products</SPAN></H3><P>After having ligated our four newest constructs on August 15th into both pET28a and pBR322 we were now able to transform these into NB ready for growth on an agar plate. 
</P><UL><LI>First 1µL of the ligation product was pipetted into 20µL of NB bacteria. These were then left to stand on ice for 5 minutes.
</LI><LI>The heat shock was performed by transferring the solutions to a 42°C water bath for 30 seconds before returning to ice for a further two minutes.
</LI><LI>Then 80µL of SOC solution was added to the bacteria where after they were not returned to ice. 
</LI><LI>The transformation solutions were then incubated for 60 minutes at 37°C. 
</LI></UL><H3><SPAN class="mw-headline" id="Plating_2">Plating</SPAN></H3><P>To grow cultures containing our ligation products we plated our transformation solutions. The constructs ligated into pET28a were plated on Kanamycin agar plates and those ligated into pBR322 were plated onto Ampicillin agar plates. 
This was done as follows:
</P><UL><LI>All steps were performed in the vicinity of a blue bunsen flame.
</LI><LI>Each transformation solution was pipetted in its entirety onto one of the plates.
</LI><LI>Using a sterile spreader the solution was spread out over the plate.
</LI><LI>The plates were then placed in a 37°C incubator overnight. As it was a Friday the incubator was set to 4°C for the remainder of the weekend after the plates had grown or 15 hours. 
</LI></UL></DIV></DIV></DIV></DIV><DIV class="tab-pane" id="Week5"><DIV><DIV class="tue-timeline-top"><DIV class="tue-timeline-title"><H2>Week 5</H2></DIV></DIV><DIV class="tue-timeline-middle"><DIV class="tue-timeline-leftbar"><DIV class="tue-timeline-node">19 August2013</DIV></DIV><DIV class="tue-timeline-content"><H3><SPAN class="mw-headline" id="Colony_Picking_and_PCR_3">Colony Picking and PCR</SPAN></H3><P>Previously, on August 16th, agar plates had been prepared by transforming the ligations of the four newest constructs (EGFP, 1G70, Poly(Threonine-Lysine) and Poly(Lysine-Serine)) in both the pET28a and pBR322 vectors into NovaBlue bacteria and plating this entirety onto either Ampicillin or Kanamycin agar plates, according to the vector (Ampicillin plates for pBR322 ligated vectors and Kanamycin plates for pET28a ligated vectors.)
Having grown these plates overnight, and storing them at 4°C for the remainder of the weekend we were now able to perform colony picking and PCR. The images of the plates are given here:
</P><DIV class="row"><DIV class="tue-float tue-float-left span3"><DIV class="tue-caption"><SPAN class="tue-caption-figure"><SPAN class="tue-refid">pET28a_EGFP</SPAN></SPAN> <SPAN class="tue-caption-text">pET28a-EGFP</SPAN></DIV></DIV><DIV class="tue-float tue-float-left span3"><DIV class="tue-caption"><SPAN class="tue-caption-figure"><SPAN class="tue-refid">pET28a_1G70</SPAN></SPAN> <SPAN class="tue-caption-text">pET28a-1G70</SPAN></DIV></DIV><DIV class="tue-float tue-float-left span3"><DIV class="tue-caption"><SPAN class="tue-caption-figure"><SPAN class="tue-refid">pET28a_P(TK)</SPAN></SPAN> <SPAN class="tue-caption-text">pET28a-Poly(Theronine-Lysine)</SPAN></DIV></DIV><DIV class="tue-float tue-float-left span3"><DIV class="tue-caption"><SPAN class="tue-caption-figure"><SPAN class="tue-refid">pET28a_P(KS)</SPAN></SPAN> <SPAN class="tue-caption-text">pET28a-Poly(Lysine-Serine)</SPAN></DIV></DIV><DIV class="tue-float tue-float-left span3"><DIV class="tue-caption"><SPAN class="tue-caption-figure"><SPAN class="tue-refid">pBR322_EGFP</SPAN></SPAN> <SPAN class="tue-caption-text">pBR322-EGFP</SPAN></DIV></DIV><DIV class="tue-float tue-float-left span3"><DIV class="tue-caption"><SPAN class="tue-caption-figure"><SPAN class="tue-refid">pBR322_1G70</SPAN></SPAN> <SPAN class="tue-caption-text">pBR322-1G70</SPAN></DIV></DIV><DIV class="tue-float tue-float-left span3"><DIV class="tue-caption"><SPAN class="tue-caption-figure"><SPAN class="tue-refid">pBR322_P(TK)</SPAN></SPAN> <SPAN class="tue-caption-text">pBR322-Poly(Theronine-Lysine)</SPAN></DIV></DIV><DIV class="tue-float tue-float-left span3"><DIV class="tue-caption"><SPAN class="tue-caption-figure"><SPAN class="tue-refid">pBR322_P(KS)</SPAN></SPAN> <SPAN class="tue-caption-text">pBR322-Poly(Lysine-Serine)</SPAN></DIV></DIV></DIV>
For the colony picking a 5 pickings were taken from each of the eight agar plates and dissolved into 15µL of sterile water. The 'epjes' into which these were placed were carefully labelled as they would be referenced to later on. For the colony PCR two master mixes were prepared. One contained the primers for the pET28a vector PCR and one contained the primers needed for the pBR322 vector PCR. The master mixes were otherwise equal and consisted of the following:
<UL><LI>216µL of sterile water.
</LI><LI>300µL of Kapa 2G.
</LI><LI>30µL of the according FWD primer.
</LI><LI>30µL of the according RW primer.
</LI></UL><P>Having prepared the two master mixes, 20 PCR eppendorf tubes per master mix were filled with 24µL of that master mix solution. To each of these prepared eppendorf tubes 1µL of the culture picking dilution was added, making care that the eppendorf tubes were labelled in the same manner as before and that the pBR322 ligated constructs were placed in the master mix containing the primers for that vector. The same was then done for the pET28a ligated constructs. 
Having prepared these samples they were placed in the PCR apparatus and set with the following protocol:
</P><UL><LI>Initial Denaturation: 95°C for 3 minutes.
</LI><LI>The following steps should be performed in this order, repeating the process a total of 34 times.
</LI><LI>Denaturation: 95°C for 15 seconds.
</LI><LI>Annealing: 58°C for 15 seconds.
</LI><LI>Extension: 72°C for 20 seconds.
</LI></UL><LI>Final Extension: 72°C for 10 minutes. 
</LI><LI> Storage: 4°C FOREVER.
</LI><P>Having completed the PCR protocol 4µL of 6x loading dye was added to each of the samples. From this solution now 18µL was loaded onto a 1.5% agarose gel which was prepared by mixing the following:
</P><UL><LI>1.8g of Agarose
</LI><LI>120mL of TAE buffer
</LI><LI>12µL of Cybersafe
</LI></UL><P>This gel was then run at 100V for 60 minutes.
</P><DIV class="row"><DIV class="tue-float tue-float-left span6"><DIV class="tue-caption"><SPAN class="tue-caption-figure"><SPAN class="tue-refid">GelExtractionpET28a1908</SPAN></SPAN> <SPAN class="tue-caption-text">Gel with pET28a constructs. 1: 1 kb ladder, 2: 100 bp ladder, 3-7: EGFP, 8-12: 1G70, 13-17: P(TK), 18-22: P(KS), 23: ladder.</SPAN></DIV></DIV><DIV class="tue-float tue-float-right span6"><DIV class="tue-caption"><SPAN class="tue-caption-figure"><SPAN class="tue-refid">GelExtractionpBR322a1908</SPAN></SPAN> <SPAN class="tue-caption-text">Gel with pBR322 constructs. 1: 1 kb ladder, 2: 100 bp ladder, 3-7: EGFP, 8-12: 1G70, 13-17: P(TK), 18-22: P(KS), 23: ladder.</SPAN></DIV></DIV></DIV><H3><SPAN class="mw-headline" id="Small_cultures">Small cultures</SPAN></H3><P>Having run the gel with the colony PCR products we were able to analyse this and see in which of our colony pickings the ligated had been successful. For each of the ligated constructs, two successfully ligated pickings were then chosen for cell culture allowing us to multiply the DNA we had. This was done by adding 5µL of the original cell picking dissolved in water to an 8mL LB culture medium. To this culture medium we then also added 8µL of Ampicillin or Kanamycin dependant on the vector of the construct (Ampicillin for pBR322 ligated structures and Kanamycin for pET28a ligated structurs). 
These small culture tubes were then placed in a 37°C incubator for overnight growth.
</P><H3><SPAN class="mw-headline" id="Preparation_for_Protein_Expression">Preparation for Protein Expression</SPAN></H3><P>Our previous test of protein expression had not been quite as successful as we had hoped as we had seen on August 16th. Therefore we were going to try again using two other constructs this time round namely: 1PJN and 1ETF both ligated into the pET28a vector. Each of these had already been transformed into BL21 bacteria and plated onto agar. 
We wished now to first grow these constructs on a small scale before transferring them to a larger culture for expression. To do this two culture tubes were prepared containing 10ml of LB and 10µL of Kanamycin. To each of these a pipet point was added with on the end a culture picking of one of the two constructs. These cultures were then placed in the 37°C incubator and left there for overnight growth. 
As we would be using the cell pellet formed after spinning down the expression product to measure the CEST contrast later this week, we would also need to provide a number of controls. One of these would be a cell pellet of a culture where no expression had taken place and a second would be the same only coated with poly-Lysines. To create the cell cultures for these controls 10mL of LB and 10µL of Kanamycin were mixed and to this solution a culture picking of 1ETF in BL21 bacteria was added. These too were cultured overnight at 37°C. 
</P><P>One further step was the preparation of two large cultures for the expression tomorrow. This was done by preparing 400mL of LB in two large erylmeyers. For the LB the following were mixed:
</P><UL><LI>4g of Peptones.
</LI><LI>4g of NaCl.
</LI><LI>2g of Yeast Extract.
</LI><LI>400mL of De-mineralised Water.
</LI></UL><P>The solution was then autoclaved and left overnight ready for expression. 
</P></DIV></DIV><DIV class="tue-timeline-middle"><DIV class="tue-timeline-leftbar"><DIV class="tue-timeline-node">20 August2013</DIV></DIV><DIV class="tue-timeline-content"><H3><SPAN class="mw-headline" id="Miniprepping_new_constructs_2">Miniprepping new constructs</SPAN></H3><P>On August 18th the four newest constructs (EGFP, 1G70, Poly(Threonine-Lysine) and Poly(Lysine-Serine)) had been ligated into both pBR322 and pET28a vectors. On August 19th a number of samples of these ligation products were taken and cultured in 8mL culture tubes. Today we would isolate the DNA vectors from the culture medium, send off DNA for sequencing and ultimately transform the DNA to BL21 bacteria which could be plated. 
To start off DNA would have to be retained which meant miniprepping the cultures, the protocol for which is given here:
</P><UL><LI>Firstly 700µL of each culture was pipetted into a storage tube along with 300µL of 50% Glycerol solution. The entire solution was then snap frozen in liquid nitrogen so that it could be stored in the -80°C freezer should we ever need to continue work with these vectors. 
</LI><LI>Now the culture tubes were spun down in a centrifuge for 10 minutes at 3700g causing the bacterial cells with the DNA inside to form pellets isolating it from the growth medium. 
</LI><LI>The supernatant that had formed above the pellets was discarded and the pellets themselves were resuspended in 250µL of P1 buffer. The tubes were gently shaken by hand until the entire pellet had resuspended.
</LI><LI>The suspension was then transferred by means of pipetting into a smaller (1.5mL) eppendorf tube. 
</LI><LI>250µL of P2 buffer was then added to the eppendorf tube and the solution was inverted by hand until it had turned a clear blue. 
</LI><LI>Within 5 minutes of adding the P2 buffer 350µL of N3 buffer was added to the solution and the eppendorf tube was inverted again until the solution was clear again. 
</LI><LI>Once clear the eppendorf tube was placed in the centrifuge and spun for 10 minutes at 13000rpm seperating the bacterial cells from the DNA which remained suspended in the solution. The bacterial cells formed a pellet in the bottom of the eppendorf tube. (Unfortunately the pellet had not completely formed after 10 minutes so the eppendorf tube was spun for a further minute, again at 13000rpm.)
</LI><LI>As soon as the centrifuge had stopped and we could see that the pellet had properly formed, the supernatant was poured off into a special QIAcolumn. The pellet could be discarded in the bio-hazard waste. 
</LI><LI>The QIAcolumn was subsequently centrifuged for one minute at 13000rpm, during which the DNA vector bound to the column meaning the flow through could be discarded. 
</LI><LI>750µL of PE buffer was then added to the QIAcolumn before centrifuging it for another minute at 13000rpm. The flow through was discarded.
</LI><LI>To ensure that all the PE buffer had passed through the column it was then spun yet again for one minute at a speed of 13000rpm. Yet again the flow through could be discarded.
</LI><LI>The column itself was then placed above a sealable 1.5mL eppendorf tube so that the DNA could be contained and stored. To obtain the DNA from the column 42µL of MilliQ water was pipetted onto the very center membrane of the column after which it was spun down for one minute at 13000rpm. The flow through now sat in the 1.5mL eppendorf tube and contained the DNA vectors.
</LI></UL><P>The following concentrations were then measured for each of the DNA samples:
</P><TABLE class="table table-striped"><TBODY><TR><TH align="center"> Construct
</TH><TH> Ligation Vector
</TH><TH> Colony Picking Number
</TH><TH> Concentration (ng/µL)
</TH></TR><TR><TD>EGFP </TD><TD> pET28a </TD><TD> 1 </TD><TD> 63.1
</TD></TR><TR><TD>EGFP </TD><TD> pET28a </TD><TD> 2 </TD><TD> 62.8
</TD></TR><TR><TD>EGFP </TD><TD> pBR322 </TD><TD> 1 </TD><TD> 71.0
</TD></TR><TR><TD>EGFP </TD><TD> pBR322 </TD><TD> 2 </TD><TD> 64.5
</TD></TR><TR><TD>1G70 </TD><TD> pET28a </TD><TD> 1 </TD><TD> 63.4
</TD></TR><TR><TD>1G70 </TD><TD> pET28a </TD><TD> 2 </TD><TD> 75.1
</TD></TR><TR><TD>1G70 </TD><TD> pBR322 </TD><TD> 3 </TD><TD> 72.2
</TD></TR><TR><TD>1G70 </TD><TD> pBR322 </TD><TD> 5 </TD><TD> 73.4
</TD></TR><TR><TD>Poly(Threonine-Lysine) </TD><TD> pET28a </TD><TD> 2 </TD><TD> 62.2
</TD></TR><TR><TD>Poly(Threonine-Lysine) </TD><TD> pET28a </TD><TD> 4 </TD><TD> 65.8
</TD></TR><TR><TD>Poly(Threonine-Lysine) </TD><TD> pBR322 </TD><TD> 2 </TD><TD> 49.5
</TD></TR><TR><TD>Poly(Threonine-Lysine) </TD><TD> pBR322 </TD><TD> 3 </TD><TD> 47.0
</TD></TR><TR><TD>Poly(Lysine-Serine) </TD><TD> pET28a </TD><TD> 1 </TD><TD> 61.9
</TD></TR><TR><TD>Poly(Lysine-Serine) </TD><TD> pET28a </TD><TD> 2 </TD><TD> 60.6
</TD></TR><TR><TD>Poly(Lysine-Serine) </TD><TD> pBR322 </TD><TD> 1 </TD><TD> 55.8
</TD></TR><TR><TD>Poly(Lysine-Serine) </TD><TD> pBR322 </TD><TD> 3 </TD><TD> 55.1
</TD></TR></TBODY></TABLE><H3><SPAN class="mw-headline" id="Sequencing_2">Sequencing</SPAN></H3><P>To check the sequences of our DNA vectors we again decided to send our vectors off to an external sequencing agency. For this they request a 7µL solution which contains 1µL of 10µM primer. They also request at least 400ng of DNA for successful sequencing. As the pBR322 ligated Poly(Threonine-Lysine) and Poly(Lysine-Serine) had such low concentrations these could not be sent off. Of the other constructs only a few would contain enough DNA but we would hope for the best. This meant that we would have to create two sample tubes per construct. One would contain the forward primer and one the reverse primer. We then added 6µL of each sample to the correct sample tube, being sure to label them well. Care should be taken to add the pET28a primers to the pET28a ligated constructs and the same of course for the pBR322 ligated constructs. 
</P><H3><SPAN class="mw-headline" id="Transformation_2">Transformation</SPAN></H3><P>From the above mentioned DNA samples we could now perform a transformation into BL21 bacteria. For this we would use the samples with the highest concentration from each pair of samples belonging to the same construct and ligation vector. (It should be noted that we only did this for the pET28a ligated constructs as these would all be expressed aerobically. We would not be bringing all constructs to expression anaerobically. However pBR322 ligated EGFP would be transformed now as that would be the first construct to get expressed anaerobically. This would mean 5 transformations would be taking place, the four pET28a ligated constructs with the highest concentrations and an EGFP ligated into pBR322.)
To perform a transformation the following steps were performed:
</P><UL><LI>From each of the chosen samples first a 1ng/µL solution had to be made by diluting a portion of the sample with sterile water. 
</LI><LI>1µL of each of the 1ng/µL dilutions would be pipetted into 20µL of BL21 bacteria which were left on ice for 5 minutes.
</LI><LI>The bacteria were then heat shocked by placing them in a water bath at 42°C for 30 seconds before returning them to ice for 2 minutes. 
</LI><LI>Hereafter, 80µL of SOC solution was added to each bacterial sample. The bacteria were now not returned to ice, rather they were incubated at 37°C for 60 minutes. 
</LI></UL><H3><SPAN class="mw-headline" id="Plating_3">Plating</SPAN></H3><P>The samples which had just be transformed and incubated could now be plated onto agar plates. All would be plated onto kanamycin agar plates apart from the pBR322 ligated EGFP which would be plated onto an ampicillin plate. 
The plating procedure is given below:
</P><UL><LI>In the vicinity of a blue flame pipet the entire bacterial solution onto the centre of the plate.
</LI><LI>Using a sterile spreader spread the bacteria out over the plate. 
</LI><LI>Incubate the plates in 37°C overnight. 
</LI></UL><P>This concludes the work on the four newest constructs for today.
</P><H3><SPAN class="mw-headline" id="Expression_Samples">Expression Samples</SPAN></H3><P>On August 19th four culture tubes had been placed in an incubator to grow overnight. Two contained pET28a ligated 1PJN and two contained pET28a ligated 1ETF. One of each these samples would be used for a control experiment when attempting to perform CEST experiments later on. The other two would now be brought across to the 2 large LB cultures (400mL) which had also been prepared on August 19th. 
</P><H4><SPAN class="mw-headline" id="Controls">Controls</SPAN></H4><P>The two control cultures were spun down at 3700rpm for 10 minutes forming a pellet of bacteria cells. The supernatant was poured off and the pellets were placed in a +4°C refrigerator for storage. 
</P><H4><SPAN class="mw-headline" id="Expression_Samples_2">Expression Samples</SPAN></H4><P>The two remaining cultures would now be used to express the proteins 1PJN and 1ETF in the larger culture mediums in the hope of obtaining a larger yield. This was attempted in the following manner:
</P><UL><LI>To start with 400µL of kanamycin was added to each of the 400mL culture mediums.
</LI><LI>Hereafter, the entire 10mL culture tube could be poured into one of the culture flasks.
</LI><LI>Now the culture mediums would be incubated at 37°C to ensure cell growth. Once there were enough cells IPTG could be added so that the bacteria would start producing our proteins. To follow the cell growth optical density measurements were performed:
</LI></UL><TABLE class="table table-striped"><TBODY><TR><TH align="center"> Time
</TH><TH> Absorbance 1PJN
</TH><TH> Absorbance 1ETF
</TH></TR><TR><TD>15:30 </TD><TD> 0.032 </TD><TD> 0.017
</TD></TR><TR><TD>16:45 </TD><TD> 0.155 </TD><TD> 0.071
</TD></TR><TR><TD>17.25 </TD><TD> 0.389 </TD><TD> 0.156
</TD></TR><TR><TD>17.50 </TD><TD> 0.678 </TD><TD> 0.320
</TD></TR><TR><TD>18.15 </TD><TD> - </TD><TD> 0.540
</TD></TR></TBODY></TABLE><UL><LI>Once the optical density reached a value of around 0.6 200µL of IPTG was added (for 1PJN at 17:50, for 1ETF at 18:15). This would ensure protein expression within our bacteria. The culture mediums were then kept left in the 37°C incubator overnight.
</LI></UL><H3><SPAN class="mw-headline" id="Buffer_Preparation">Buffer Preparation</SPAN></H3><P>To ensure that the protein purification could be performed in a fluid manner we decided to prepare a number some buffers for the Nickel-Column purification. We prepared three buffers: a Wash, Strip and Elution buffer using the following amounts: 
</P><TABLE class="table table-striped"><TBODY><TR><TH> Wash Buffer
</TH><TH> Elution Buffer
</TH><TH> Strip Buffer
</TH></TR><TR><TD>Trizma Base </TD><TD> 1.21 (g) </TD><TD> 0.484 (g) </TD><TD> 1.21 (g)
</TD></TR><TR><TD>NaCl </TD><TD> 14.6 (g) </TD><TD> 5.84 (g) </TD><TD> 14.6 (g)
</TD></TR><TR><TD>Imidazole </TD><TD> 1.02 (g) </TD><TD> 6.81 (g) </TD><TD> -
</TD></TR><TR><TD>EDTA </TD><TD> - </TD><TD> - </TD><TD> 18.6 (g)
</TD></TR><TR><TD>Demineralised Water </TD><TD> 500 (mL) </TD><TD> 200 (mL) </TD><TD> 500 (mL)
</TD></TR></TBODY></TABLE></DIV></DIV><DIV class="tue-timeline-middle"><DIV class="tue-timeline-leftbar"><DIV class="tue-timeline-node">21 August2013</DIV></DIV><DIV class="tue-timeline-content"><H3><SPAN class="mw-headline" id="Protein_Purification">Protein Purification</SPAN></H3><P>After having prepared two expression mediums on August 20th it was now time to retain the proteins from the cultures and perform a purification. To do this a lot of separate steps were performed which will each be handled below in chronological order:
</P><H4><SPAN class="mw-headline" id="Protein_Retention">Protein Retention</SPAN></H4><P>To retain the protein from the culture the BugBuster protocol was used which was doen as follows:
</P><UL><LI>Firstly the cultures were poured into separate 400mL flasks which would fit into a large centrifuge. The flasks were equally weighed to ensure that the centrifuge was balanced.
</LI><LI>The flasks with the culture medium were then spun down in the centrifuge at 10000G for 10 minutes forming a pellet containing our bacteria with our protein inside. The supernatant was therefore discarded so we were left with our pellet. 
</LI></UL><H5><SPAN class="mw-headline" id="Preparation_for_CEST_measurements">Preparation for CEST measurements</SPAN></H5><P>To be able to measure the CEST effects of our proteins we would have to bring them to the MRI scanner. Here they would be tested on their contrast. As our bacteria would house our proteins should they be used to find tumours this is what we wished to replicate in the scanner. This would mean taking a piece of our pellet and storing it separately. The following steps therefore were taken:
</P><UL><LI>Firstly from the two large pellets we had just obtained a small portion was placed in a PCR tube and spun down at 13400 rpm for 3 minutes to place the pellet at the bottom of the PCR tube. 
</LI><LI>We had also prepared two controls on August 20th. These were spun down to form pellets but having been left at 4°C they had dissolved again. However this solution was pipetted into a PCR tube and spun down forming a new pellet. The supernatant was then pipetted off again.
</LI><LI>To all four samples 250µL of 4% PFA (Paraformaldahyde) solution was added to fixate the cells pausing necrosis. 
</LI><LI>The four samples were then allowed to sit in the refrigerator at 4°C. 
</LI></UL><H4><SPAN class="mw-headline" id="Return_to_Protein_Retention">Return to Protein Retention</SPAN></H4><P>Continuing with the remainder of the large pellet and the BugBuster protocol the following steps were then performed on the two pellets:
</P><UL><LI>To each pellet 8mL of BugBuster was added along with 8µL Benzonaze. The pellets and solution were then placed on a shaking table in 4°C for 30 minutes untill the pellet had completely dissolved. 
</LI><LI>The solutions were then poured into a smaller flasks, weighed equally and returned to the heavy duty centrifuge where they were spun for 20 minutes at 16000G. 
</LI><LI>The supernatant now contained our protein (along with other native proteins) so it was poured off and stored separately. The pellet was kept for analysis later on in the 4°C refrigerator. 
</LI></UL><H4><SPAN class="mw-headline" id="Nickel-Column_Purification">Nickel-Column Purification</SPAN></H4><P>Now that we had obtained a solution containing our protein (hopefully) we could purify it using a Nickel-Column. Firstly two columns had to be prepared. To do this the following was done: 
</P><UL><LI>The columns were filled with demineralised water and 1mL of resin was added to each column. 
</LI><LI>The water was allowed to flow through but was refilled before the column was emptied. As the resin layer grew more resin was added to the column. 
</LI><LI>The above procedures were repeated until a 2mL thick resin layer had formed in the column. 
</LI><LI>Now a filter could be placed on top of the resin using a plunger.
</LI></UL><P>Having prepared a column it was time for the purification to begin:
</P><UL><LI>First the column was loaded with 8mL of Charge buffer containing 50mM NiSO4. The flow through of the column was stored separately due to the heavy metal content. 
</LI><LI>Once empty 16mL of Wash buffer was loaded and again the flow through was separately caught to reduce the risk of heavy metal contamination. 
</LI><LI>The entire protein sample could then be loaded onto the column. Of the flow through a single droplet was retrieved for future analysis. 
</LI><LI>The column was then washed once more with 16mL of wash buffer and again a single droplet was caught for analysis. 
</LI><LI>Now elution was performed by loading the column with 8mL of elution buffer. All the flow through droplets were retrieved by catching 0.5mL of flow through in separate eppendorf tubes. 
</LI><LI>The column was then stripped with 8mL of strip buffer, whereby the flowthrough was retrieved separately due to the Nickel content. Strip buffer was then added to the column before sealing it and placing it in the 4°C refrigerator. 
</LI></UL><P>Our purified protein could now be found in the 16 elution tubes we had collected (in reality we had collected 14 tubes from the 1PJN elution and 16 tubes from the 1ETF solution).
</P><H3><SPAN class="mw-headline" id="Concentration_Determination">Concentration Determination</SPAN></H3><P>Now we had our protein it was important to know what the concentration was. As we could not measure the 260/280 absorbance, a nanodrop measurement was not applicable. We had to think of a way to determine the concentration of heavily positive proteins which led us to the Bradford assay. The following steps were performed to achieve this:
</P><H4><SPAN class="mw-headline" id="Making_the_Protein_Reagent">Making the Protein Reagent</SPAN></H4><P>For the staining of our samples  we made a solution consisting of Coomassie Brilliant Blue G-250, 95% Ethanol and 85% Phosphoric Acid. 
</P><UL><LI>First 20mg of Coomassie Brilliant Blue G-250  was weighed and dissolved  in 10ml of 95% (w/v) Ethanol. 
</LI><LI>Next we added 20ml of (85% (w/v)  Phosphoric Acid and diluted the solution until a total volume of 200ml  was reached. 
</LI><LI>The solution was subsequently stirred for about an hour, to ensure that the Coomassie Brilliant Blue G-250 was completely dissolved. 
</LI></UL><H4><SPAN class="mw-headline" id="Making_the_BSA_standards">Making the BSA standards</SPAN></H4><P>To ensure we had reference dilutions to work with the following BSA (Bovine Serum Albumine) solutions were prepared. These would allow us to compare our protein concentrations with those of the standards, giving an indication of the concentration of our protein. 
</P><TABLE class="table table-striped"><TBODY><TR><TH align="center"> Samle #
</TH><TH> Standard Concentration (mg/mL)
</TH><TH> Standard Volume (µL)
</TH><TH> Dilent Volume (µL)
</TH><TH> Final Protein Concentration (µg/mL)
</TH></TR><TR><TD>1 </TD><TD> - </TD><TD> - </TD><TD> 500 </TD><TD> 0
</TD></TR><TR><TD>2 </TD><TD> 2 </TD><TD> 0.67 </TD><TD> 499.33 </TD><TD> 1.25
</TD></TR><TR><TD>3 </TD><TD> 2 </TD><TD> 1.25 </TD><TD> 498.75 </TD><TD> 2.5
</TD></TR><TR><TD>4 </TD><TD> 2 </TD><TD> 2.5 </TD><TD> 497.5 </TD><TD> 5.0
</TD></TR><TR><TD>5 </TD><TD> 2 </TD><TD> 3.33 </TD><TD> 496.67 </TD><TD> 10.0
</TD></TR><TR><TD>6 </TD><TD> 2 </TD><TD> 3.75 </TD><TD> 496.25 </TD><TD> 15.0
</TD></TR><TR><TD>7 </TD><TD> 2 </TD><TD> 5.0 </TD><TD> 495.0 </TD><TD> 20.0
</TD></TR><TR><TD>8 </TD><TD> 2 </TD><TD> 10 </TD><TD> 790.0 </TD><TD> 25.0
</TD></TR></TBODY></TABLE><P>In the above table the standard refers to the BSA solution and the Dilent used was a pH 8.0 TRIS buffer.
</P><H4><SPAN class="mw-headline" id="Performing_the_Bradford_Assay">Performing the Bradford Assay</SPAN></H4><P>To perform the Bradford assay a 96 wells plate was used and filled in the following manner: 
</P><UL><LI>100µL of each of the above standards was pipetted into one of the wells on a 96-well-plate. 
</LI><LI>To each standard 100µL of the Coomaissie dye was added for colouring. 
</LI><LI>We then loaded 50µL from each of the Nickel column elution flow through tubes. (We discarded the first 2 and last 2 samples from each protein as these would not accurately represent our proteins). 
</LI><LI>To this 50µL we then added 50µL of TRIS buffer (pH 8.0). 
</LI><LI>As with the standards 100µL of the Coomassie  Dye was then added.
</LI></UL><P>These dilutions were not very clear so it was decided to perform the test again but this time taking only 10µL from each of the samples and add 90µL of TRIS buffer. This allowed us to see clearly which of the sample tubes contained the highest concentration of our two proteins: 1PJN and 1ETF.
</P><P>The wells plate was then not placed in the spectrometer as we could clearly see which of the collection columns contained the highest protein concentration from each of the protein elutions. The five highest concentrations were then chosen for the next steps. The 96-Wells-Plate used for the assay is shown below:
</P><DIV class="row"><DIV class="tue-float tue-float-Centre span12"><DIV class="tue-caption"><SPAN class="tue-caption-figure"><SPAN class="tue-refid">Bradford_Assay</SPAN></SPAN> <SPAN class="tue-caption-text">Bradford Assay, Row C: Standards increasing from left to right. Row D: 10:90 dilutions of 1ETF elution products. Row E: 50:50 dilutions of 1PJN elution products. Row F: 10:90 dilutions of 1PJN elution products. The enclosed boxes on row D and F show the five chosen elution samples with the highest protein concentration (The darker the blue the higher the concentration).</SPAN></DIV></DIV></DIV><H3><SPAN class="mw-headline" id="Rebuffering_the_Proteins">Rebuffering the Proteins</SPAN></H3><P>As the proteins had been purified using a Nickel Column they were now suspended in a buffer containing Imidazole which can cause protein denaturation. This meant that the proteins had to be rebuffered into a more suitable buffer:
</P><UL><LI>First, from the five chosen elution samples 10µL was pipetted into a separate eppendorf tube for an SDS analysis later on. 
</LI><LI>Then the five chosen elution samples from each protein were combined in one large falcon tube. 
</LI><LI>The rebuffering was then performed using a PD-10 desalting column with the following protocol:
</LI><LI>First the column was opened and the buffer was poured off and discarded.
</LI><LI>Next 25mL of TRIS buffer (pH 8.0) was allowed to run through the column. The flow through was discarded.
</LI><LI>After this we were able to load our sample, during which the flow through was discarded. A maximum of 2.5mL of our sample could be loaded, which was the exact amount we had chosen to rebuffer. 
</LI><LI>Then 3.5mL of TRIS Buffer was run through the column for the elution, and the flow through was retained as it contained our protein. Retention was done in a number of eppendorf tubes each holding 1mL of the flow through.
</LI><LI>The proteins were then snap frozen in liquid nitrogen and stored in -80 degrees Celsius. 
</LI></UL></DIV></DIV><DIV class="tue-timeline-middle"><DIV class="tue-timeline-leftbar"><DIV class="tue-timeline-node">22 August2013</DIV></DIV><DIV class="tue-timeline-content"><H3><SPAN class="mw-headline" id="SDS_Analysis">SDS Analysis</SPAN></H3><P>Today we would analyse the results of the protein purification performed on August 21st. This would be done by running an SDS-PAGE gel. As we were not certain if the super positive proteins would like the conditions of a standard SDS PAGE gel we decided to also test our samples on a native gel. To further enhance this we would attempt to run this gel with the poles reversed, in theory forcing the protein down the gel. 
</P><UL><LI>To start with two 12% gels were prepared. The SDS PAGE gel followed the standard recipe whereas the native gel would not contain the 10% SDS in either the running or stacking gel solution. 
</LI><LI>Once the gels had hardened the samples could be prepared. We would like to load a sample from the loading, washing, the five chosen elution samples and a sample from the original pellet which we had also stored on August 20th. 
</LI><LI>For both proteins, for each of the 5 elution samples and for the loading and washing samples 2 PCR tubes were filled with 5µL of each sample. 
</LI><LI>To 1 of each pair of PCR tubes (containing the same sample) 5µL of standard 2x SDS loading dye with DTT (stock: 500µL 2x SDS loading dye, 1µL DTT) was added. 
</LI><LI>To the other PCR tube of the pair 5µL of SDS free loading dye was added. 
</LI><LI>Then a single eppendorf tube per protein was loaded with a small pellet scraping and 100µL of pH 8.0 TRIS buffer was added. 
</LI><LI>Of the pellet solution 2x 5µL was then pipetted in to separate PCR tubes and 5µL of the correct loading dye was added to each of the four tubes.
</LI><LI>Now all the samples containing the SDS loading dye were denatured for 5 minutes at 95°C.
</LI></UL><LI>Now the samples had been prepared the gels were placed in the tanks and surrounded with buffer. For the SDS gel 1x SDS buffer was added. For the SDS free gel an SDS free buffer was used.
</LI><LI>The gels were now run for 15 minutes at 90V and then for 40 minutes at 150V. 
</LI><LI>After the gels had run they were placed in water and allowed to sit for 30 minutes before being stained with 1x Coomaisse solution for two hours.
</LI><LI>The gels were then set in demineralised water and left on the shaking table overnight.
</LI><DIV class="row"><DIV class="tue-float tue-float-left span12"><DIV class="tue-caption"><SPAN class="tue-caption-figure"><SPAN class="tue-refid">SDSGel2208</SPAN></SPAN> <SPAN class="tue-caption-text">SDS gel of purification samples: 1: 1ETF eluted, 2: ladder, 3-6: 1ETF eluted, 7: 1ETF loaded, 8: 1ETF washed, 9-13: 1PJN eluted, 14: 1PJN loaded, 15: 1PJN washed</SPAN></DIV></DIV></DIV><H3><SPAN class="mw-headline" id="Preparation_for_Anaerobic_Expression">Preparation for Anaerobic Expression</SPAN></H3><P>It was finally time to work on the anaerobic expression. Over the past few weeks a lot of effort had gone into fixing up an old biofermentor making it usable once more. Now we had it up and running we were going to test it by attempting to express EGFP anaerobically. Firstly however a large culture solution had to be prepared and small cultures had to be grown for expression to be kick started. 
</P><H4><SPAN class="mw-headline" id="Preparing_Culture_mediums">Preparing Culture mediums</SPAN></H4><P>In the biofermentor a 4L LB solution was prepared before the fermentor was autoclaved. This solution was prepared in the vessel by mixing:
</P><UL><LI>40g Peptones
</LI><LI>40g NaCl
</LI><LI>20g Yeast Extract
</LI><LI>4L Demineralised Water
</LI></UL><P>The fermentor was then autoclaved. Along with it a smaller 400mL culture medium was autoclaved. This could be used to grow EGFP aerobically to check that no proteins would grow if oxygen was present. For this 400mL solution the following were mixed:
</P><UL><LI>4g Peptones
</LI><LI>4g NaCl
</LI><LI>2g Yeast Extract
</LI><LI>400mL Demineralised Water
</LI></UL><H4><SPAN class="mw-headline" id="Preparing_Small_Cultures">Preparing Small Cultures</SPAN></H4><P>To ensure that the bacteria did not have to grow from scratch in the two larger culture mediums smaller culture tubes were prepared. To do this we retrieved our pBR322 ligated EGFP agar plate from the refrigerator and followed the following protocol:
</P><UL><LI>Two eppendorf tubes were filled with 15µL of sterile water.
</LI><LI>The following steps were all performed in the vicinity of a blue flame.
</LI><LI>2 colonies were picked from the above mentioned agar plate and each was dissolved in one of the water samples. 
</LI><LI>Then 7 culture tubes were filled with 10mL of LB medium.
</LI><LI>To each culture tube a further 10µL of Ampicillin antibiotics was added.
</LI><LI>Then to 6 of the 7 tubes 5µL of the dissolved picking solutions was added. 
</LI><LI>To the 7th culture tube a single picking was taken and was ejected directly, including the pipet point, into the culture medium. 
</LI><LI>The culture tubes were then placed in the 37°C incubator and left to grow overnight, in preparation for tomorrows expression. 
</LI></UL><P>Six of the above prepared culture tubes would be used for the 4L anaerobic expression and the final tube would be used for the control. 
</P><H3><SPAN class="mw-headline" id="MRI_Experiments">MRI Experiments</SPAN></H3><P>On August 21st we had prepared a number of samples for testing in the MRI. These four pellets (3 1ETF and 1 1PJN) were stored in 4% PFA solution and left in the 4°C refrigerator overnight. Now they were transferred to the MRI laboratory for analysis of their CEST effect. Of the three 1ETF samples 1 had had protein expressed and 2 were controls without protein expression. The 1PJN sample had also endured protein expression. Of the two controls one had had 100µL of 0.01% Poly(Lysine) added to spike the sample. The MRI experiments were then performed using the following protocol:
</P><UL><LI>The four samples were placed and secured in the MRI and a standard scan was performed.
</LI><LI>The frequencies were then optimized for our proteins.
</LI><LI>We then shimmed the samples, which means we created a homogeneous magnetic field.  
</LI><LI>The T<SUB>2</SUB> was weighed for the orientation.
</LI><LI>We then performed local shimming on each of the separate pellets. 
</LI><LI>We then averaged our measurements over 8 separate scans, without inducing a saturation pulse. 
</LI></UL><P>Using these measurements large amounts of data were acquired which the modelling teams would analyse, giving us the results of our proteins. 
</P></DIV></DIV><DIV class="tue-timeline-middle"><DIV class="tue-timeline-leftbar"><DIV class="tue-timeline-node">23 August2013</DIV></DIV><DIV class="tue-timeline-content"><H3><SPAN class="mw-headline" id="Protein_Expression">Protein Expression</SPAN></H3><P>Today we would focus on the anaerobic expression of the pBR322 ligated EGFP. This should give the first indication of whether our promoter works or not. On August 22nd we had autoclaved two culture mediums, a 4L medium in the biofermentor to attempt anaerobic expression and a 400mL medium for an aerobic control experiment. Before beginning 4mL and 400µL of Ampicillin was added respectively to each of the above mentioned cultures. 
Before beginning with our experiments we had to calibrate the oxygen probe which was done in the following manner:
</P><UL><LI>Calibration at 100% by flooding culture vessel with air for 30 minutes (3L/minute, 300rpm agitation).
</LI><LI>When gas admission was increased to 10L/minute the dO<SUB>2</SUB> increased from 100% to 107.9%. 
</LI><LI>Calibration was repeated at 10L/minute air admission. 
</LI><LI>Once gas admission was removed the dO<SUB>2</SUB> stabilized at 92.0%. This value was then taken as the 100% calibration. 
</LI><LI>For the 0% calibration the Oxygen probe was disconnected. 
</LI></UL><P>Now we could begin with the protein expression. Multiple steps and measurements were taken at multiple time points. Brief descriptions of the steps have been given in the following table:
</P><TABLE class="table table-striped"><TBODY><TR><TH align="center"> Time
</TH><TH> Procedure
</TH></TR><TR><TD>09:02 </TD><TD> Single small  culture tube prepared on August 22nd was added to the culture medium and medium was placed in 37°C aerobic incubator.
</TD></TR><TR><TD>09:16 </TD><TD> First of 6 small culture tubes added to 4L anaerobic culture chamber. (pH = 7.78, dO<SUB>2</SUB> = 99.5, Temp = 36.7°C)
</TD></TR><TR><TD>09:28 </TD><TD> Last of 6 small culture tubes added to the 4L anaerobic culture chamber.
</TD></TR><TR><TD>10:03 </TD><TD> Aerobic culture sampled. O.D. = 0.080.
</TD></TR><TR><TD>10:11 </TD><TD> The dO<SUB>2</SUB> of the anaerobic chamber had fallen to 53.8. Therefore 100% air was implemented into the chamber at 3L/minute.
</TD></TR><TR><TD>10:24 </TD><TD> Following equilibrium had occured (pH = 7.75, dO<SUB>2</SUB> = 78.8, Temp = 36.9°C).
</TD></TR><TR><TD>10:27 </TD><TD> Anaerobic culture was sampled. O.D. = 0.084.
</TD></TR><TR><TD>10:37 </TD><TD> Anaerobic chamber conditions = (pH = 7.72, dO<SUB>2</SUB> = 71.6, Temp = 37.0°C)
</TD></TR><TR><TD>10:50 </TD><TD> Anaerobic chamber conditions = (pH = 7.67, dO<SUB>2</SUB> = 60.6, Temp = 37.0°C). The low Oxygen saturation meant we upped the gas induction to 5L/minute.
</TD></TR><TR><TD>11.35 </TD><TD> Anaerobic chamber conditions = (pH = 7.45, dO<SUB>2</SUB> = 36.3, Temp = 37.0°C). Low Oxygen saturation forced a gas induction of 10L/minute.
</TD></TR><TR><TD>11:42 </TD><TD> Anaerobic chamber conditions = (pH = 7.44, dO<SUB>2</SUB> = 73.7, Temp = 36.9°C). Higher Oxygen saturation resulted in a reduction of gas induction to 8L/minute.
</TD></TR><TR><TD>11:48 </TD><TD> Anaerobic culture was sampled. O.D. = 0.0511
</TD></TR><TR><TD>11.53 </TD><TD> Aerobic culture was sampled. O.D. = 1.087
</TD></TR><TR><TD>11:58 </TD><TD> Anaerobic chamber conditions = (pH = 7.25, dO<SUB>2</SUB> = 16.1, Temp = 37.0°C).
</TD></TR><TR><TD>12.04 </TD><TD> Anaerobic chamber conditions = (pH = 7.14, dO<SUB>2</SUB> = 30.3, Temp = 37.0°C). Agitation was reduced to 250rpm.
</TD></TR><TR><TD>12:15 </TD><TD> pH loop was induced. (Setpoint = 7.5, Deadband = 0.2, Multiplier = 50%)
</TD></TR><TR><TD>12:15 </TD><TD> Anaerobic chamber conditions = (pH = 6.97, dO<SUB>2</SUB> = 45.4, Temp = 37.1°C).
</TD></TR><TR><TD>12:18 </TD><TD> Anaerobic chamber conditions = (pH = 7.18, dO<SUB>2</SUB> = 57.4, Temp = 37.0°C).
</TD></TR><TR><TD>12:22 </TD><TD> dO<SUB>2</SUB> = 40.2. As the optical density was now presumable around 0.6 we could begin reducing the oxygen levels, hopefully inducing protein expression. (Gas = 4Gas)
</TD></TR><TR><TD>12:25 </TD><TD> Anaerobic chamber conditions = (pH = 7.84, dO<SUB>2</SUB> = 0.2, Temp = 37.1°C).
</TD></TR><TR><TD>12:25 </TD><TD> Sample of anaerobic chamber was taken (λ = 1).
</TD></TR><TR><TD>12:31 </TD><TD>  Anaerobic chamber conditions = (pH = 7.86, dO<SUB>2</SUB> = -0.7, Temp = 37.0°C). The 4Gas induction was set to 5L/minute.
</TD></TR><TR><TD>12:36 </TD><TD> Aerobic sample was taken. (µ = 1).
</TD></TR><TR><TD>12:42 </TD><TD> Anaerobic chamber conditions = (pH = 7.90, dO<SUB>2</SUB> = -0.4, Temp = 36.9°C). 0.1M H<SUB>2</SUB>SO<SUB>4</SUB> was connected to the acid inducer. pH multiplier was set to 20%.
</TD></TR><TR><TD>13:00 </TD><TD> Anaerobic chamber conditions = (pH = 8.02, dO<SUB>2</SUB> = 0.2, Temp = 37.0°C).
</TD></TR><TR><TD>13:02 </TD><TD> Anaerobic sample was taken. (λ = 2).
</TD></TR><TR><TD>13:14 </TD><TD> Anaerobic chamber conditions = (pH = 7.56, dO<SUB>2</SUB> = 1.2, Temp = 37.0°C).
</TD></TR><TR><TD>15:16 </TD><TD> Anaerobic chamber conditions = (pH = 7.45, dO<SUB>2</SUB> = -0.7, Temp = 37.0°C).
</TD></TR><TR><TD>15:22 </TD><TD> Anaerobic sample was taken. (λ = 3).
</TD></TR><TR><TD>15:25 </TD><TD> 4Gas was switched to 100% Nitrogen gas. Agitation was set to 100rpm to reduce foaming.
</TD></TR><TR><TD>17:03 </TD><TD> Anaerobic sample was taken. (λ = 4).
</TD></TR><TR><TD>18:05 </TD><TD> Anaerobic sample was taken. (λ = 5).
</TD></TR><TR><TD>18.07 </TD><TD> Heater was set to 20°C.
</TD></TR><TR><TD>18:24 </TD><TD> Gas admission was turned off. pH control was also turned off, pH = 7.64.
</TD></TR><TR><TD>18:38 </TD><TD> Temperature was 25.5°C.
</TD></TR></TBODY></TABLE><P>After the fermentor had been turned off all samples we poured into specialised centrifuge tubes and equally weighed. These were then centrifuged for 10 minutes at 16000g. The samples were then snap frozen and placed in the -30°C freezer.  and left for later analysis. 
</P></DIV></DIV></DIV></DIV><DIV class="tab-pane" id="Week6"><DIV><DIV class="tue-timeline-top"><DIV class="tue-timeline-title"><H2>Week 6</H2></DIV></DIV><DIV class="tue-timeline-middle"><DIV class="tue-timeline-leftbar"><DIV class="tue-timeline-node">26 August2013</DIV></DIV><DIV class="tue-timeline-content"><H3><SPAN class="mw-headline" id="Expression_Cultures">Expression Cultures</SPAN></H3><P>This coming, and final week of the summer we will be focussing primarily on the expression of our proteins and a second anaerobe expression due to a failed first attempt. The first thing we did therefore was chose a number of proteins which we wanted to express aerobically. In the end we choose the following: 1ETF, 1G70, Poly(Threonine-Lysine), Poly(Lysine-Serine), Poly(Arginine-Glycine) and EGFP as a control. Each of these proteins had already been ligated into the pET28a vector and transformed into BL21 bacteria. Therefore we already had agar plates with colonies on them of all the desired cultures. All we needed to do to prepare for expression tomorrow would be to grow these cultures in small culture tubes which was done in the following fashion:
</P><UL><LI>All steps were performed in the vicinity of a blue flame to increase sterility.
</LI><LI>6 small culture tubes were filled with 8mL of LB medium.
</LI><LI>8µL of Kanamycin antibiotics was added to each small culture. 
</LI><LI>One colony picking from each of the protein constructs was pipetted into the culture mixture.
</LI><LI>The culture tubes were then placed in an incubator overnight as 37°C.
</LI></UL><P>A secondary preparatory step for aerobic expression tomorrow was the making of 6 400mL culture mediums. This entailed making 6 erylmeyers containing 400mL of LB. The LB was prepared by mixing the following in each of the flasks:
</P><UL><LI>4g Peptones.
</LI><LI>4g NaCl.
</LI><LI>2g Yeast Extract.
</LI><LI>400mL De-mineralised Water. 
</LI></UL><P>After combining the above the flasks were left to stand and were autoclaved at the earliest convenience. This then concluded the preparation for the protein expression. 
</P><H3><SPAN class="mw-headline" id="Anaerobic_expression">Anaerobic expression</SPAN></H3><P>Previously on August 23rd an attempt had been made to express EGFP anaerobically. Unfortunately this had not yielded any results. After some research we discovered a possible cause, a mutation in the FNR coding gene within the BL21 bacteria. This would mean that FNR production would not occur whilst the proteins were being expressed in BL21. The first solution for this was not to use BL21 bacteria but another strain available to us in the lab: XL-1 Blue. This meant that we needed to transform and plate our pBR322 ligated EGFP into this new bacterial strain. To do this the following was done:
</P><UL><LI>All steps below are performed in the vicinity of a blue flame. 
</LI><LI>1µL of 1ng/µL ligated EGFP vector was added to 30µL of XL-1 Blue bacteria which were consequently left on ice for 30 minutes. 
</LI><LI>After ice incubation the bacteria were placed in a 42°C waterbath for 45 seconds before returning to ice for 2 minutes. 
</LI><LI>Hereafter 200µL of SOC solution was added to the bacteria and they were incubated at 37°C whilst shaking at 300rpm. 
</LI><LI>After incubation 110µL of the bacteria solution was pipetted on to an Ampicillin agar plate and spread with a sterile spreader. 
</LI><LI>The plate was then placed in a 37°C incubator for overnight growth. 
</LI></UL><P>This concludes the work on these bacteria for today. 
</P></DIV></DIV><DIV class="tue-timeline-middle"><DIV class="tue-timeline-leftbar"><DIV class="tue-timeline-node">27 August2013</DIV></DIV><DIV class="tue-timeline-content"><H3><SPAN class="mw-headline" id="Protein_Expression_2">Protein Expression</SPAN></H3><P>On August 26th small cultures were made in preparation for the protein expression which would be carried out today. The BL21 bacterial cells we had grown contained pET28a vectors ligated with the following constructs: EGFP, 1ETF, 1G70, Poly(Arginine-Glycine), Poly(Threonine-Lysine) and Poly(Lysine-Serine). The small cultures of each construct would now be transferred into the larger 400mL cultures which we had also prepared on August 26th:
</P><UL><LI>All the following steps were performed near a blue bunsen flame to ensure sterility.
</LI><LI>First 400µL of Kanamycin antibiotics were added to each of the large cultures.
</LI><LI>Next the small cultures were each poured into one of the large culture flasks. 
</LI><LI>The large cultures were then placed in the 37°C incubator to grow. At certain intervals the optical density of the cultures was measured by removing 1mL of the culture and placing it in a cuvette. This would then be placed in a spectrometer and the absorbance would be measured:
</LI></UL><TABLE class="table table-striped"><TBODY><TR><TH align="center"> Time
</TH><TH> Absorbance EGFP
</TH><TH> Absorbance 1ETF
</TH><TH> Absorbance 1G70
</TH><TH> Absorbance Poly(Agrinine-Glycine)
</TH><TH> Absorbance Poly(Threonine-Lysine)
</TH><TH> Absorbance Poly(Lysine-Serine)
</TH></TR><TR><TD>15:15 </TD><TD> 0.130 </TD><TD> 0.107 </TD><TD> 0.087 </TD><TD> 0.104 </TD><TD> 0.107 </TD><TD> 0.121
</TD></TR><TR><TD>16:25 </TD><TD> 0.682 * </TD><TD> 0.476 </TD><TD> 0.329 </TD><TD> 0.415 </TD><TD> 0.402 </TD><TD> 0.512
</TD></TR><TR><TD>16.45 </TD><TD> - </TD><TD> 0.718 * </TD><TD> 0.557 * </TD><TD> 0.614 * </TD><TD> 0.704 * </TD><TD> 0.773 *
</TD></TR></TBODY></TABLE><P>A * indicates that 200µL of 1M IPTG was added to the culture, inducing the protein expression. 
</P><UL><LI>The cultures were subsequently left in the 37°C incubator overnight for expression.
</LI></UL><H3><SPAN class="mw-headline" id="Making_Small_Cultures">Making Small Cultures</SPAN></H3><P>For the anaerobic expression which we wished to perform on August 28th we would need to produce a number of small cultures to kick start the bacteria growth. On August 26th we had plated pBR322 ligated EGFP in XL-1 Blue bacteria and we would be using these colonies to produce our small cultures:
</P><UL><LI>All steps were performed in close proximity to a blue bunsen flame.
</LI><LI>From the above mentioned plate 2 colonies were picked using a pipet. Each picking was then dissolved in 15µL of sterile water.
</LI><LI>6 culture tubes were then prepared with 8mL of LB solution and 8µL of Ampicillin. 
</LI><LI>To each of the 6 culture tubes 5µL of the picking solution was added.
</LI><LI>The small culture tubes were then placed in the 37°C incubator overnight for cell growth.
</LI></UL><H3><SPAN class="mw-headline" id="Preparing_the_Large_Culture_Vessel">Preparing the Large Culture Vessel</SPAN></H3><P>The large biofermentor in which we wished to perform the anaerobic protein expression was autoclaved so that it would be sterile for use tomorrow. Whilst autoclaving a 4L LB solution was prepared inside the vessel. The LB medium was made by mixing the following:
</P><UL><LI>40g Peptones.
</LI><LI>40g NaCl.
</LI><LI>20g Yeast Extract. 
</LI><LI>4000mL of Demineralised water. 
</LI></UL><P>The culture vessel was then autoclaved and sealed until needed. 
</P><H3><SPAN class="mw-headline" id="Preparing_for_BioBricks">Preparing for BioBricks</SPAN></H3><P>To make the biobricks a number of steps would need to be performed, the first of which would be the digestion of both our samples as well as the plasmid backbone. 
</P><H5><SPAN class="mw-headline" id="Digestion_3">Digestion</SPAN></H5><P>Both the backbone and the pUC57-simple constructs were digested with EcoRI-HF and PstI-HF. This was done in a PCR apparatus for 1 hour at 37°C. After that, the constructs were placed on ice, and 1µL of DpnI was added to the backbone. This procedure was executed for 5 minutes at 37°C and heat inactivation was achieved by heating to 80°C for 5 minutes.
The mixtures prepared for the digestion were made as follows:
</P><TABLE class="table table-striped"><TBODY><TR><TH align="center"> Construct
</TH><TH> 500ng DNA (µL)
</TH><TH> CutSmart Buffer (µL)
</TH><TH> Amount of each Enzyme (µL)
</TH><TH> MilliQ water (µL)
</TH></TR><TR><TD>pSB1C3 Linearised Bacbone
</TD><TD>20.0
</TD><TD>5
</TD><TD>0.2
</TD><TD>4.6
</TD></TR><TR><TD>Protamine-1-Optimized
</TD><TD>23.9
</TD><TD>5
</TD><TD>0.7
</TD><TD>19.7
</TD></TR><TR><TD>1ETF
</TD><TD>22.2
</TD><TD>5
</TD><TD>0.7
</TD><TD>21.6
</TD></TR><TR><TD>1PJN1
</TD><TD>41.5
</TD><TD>5
</TD><TD>0.7
</TD><TD>2.8
</TD></TR><TR><TD>Poly(Arginine-Glycine)
</TD><TD>18.9
</TD><TD>5
</TD><TD>0.7
</TD><TD>25.4
</TD></TR><TR><TD>Poly(Arginine-Serine)
</TD><TD>19.1
</TD><TD>5
</TD><TD>0.7
</TD><TD>25.2
</TD></TR></TBODY></TABLE><H5><SPAN class="mw-headline" id="PCR_Purification">PCR Purification</SPAN></H5><P>After having performed the digestion the linearised backbone needed to be purified. This was done using the PCR purification kit. Unfortunately the vectors and inserts were also purified using the PCR purification kit which was not our intention as we wished to purify these using the gel extraction method. 
The PCR purification protocol was performed as follows:
</P><UL><LI>To the PCR product 5 volumes of PB buffer were added (In our case this was 150µL of PB buffer as we had 30µL of PCR product.)
</LI><LI>This solution was placed in a QIAquick spin column and spun for 60 seconds at 13000 rpm. The flow through could be discarded.
</LI><LI>Now 750µL of PE buffer could be added to the column before spinning it down once more for 60 seconds at 13000 rpm.
</LI><LI>Having discarded the flow through the column was spun for a further 60 seconds again at 13000 rpm to remove the last of the PE buffer.
</LI><LI>The column was then placed into a sealable eppendorf tube before adding 42µL of MilliQ water to the very centre membrane of the spin column and leaving it to stand for one minute.
</LI><LI>The column was now placed in the centrifuge and spun for 60 seconds at 13000 rpm producing a solution containing the desired DNA.
</LI></UL><H5><SPAN class="mw-headline" id="Gel_Extraction">Gel Extraction</SPAN></H5><P>Having loaded the inserts onto the purification columns we were unsure if the Gel Purification would work, however an attempt was made. A 1.5% agarose gel was prepared. Unfortunately plain water was used instead of TAE buffer to prepare the gel. This meant that after having loaded and run the gel that not insert bands were to be seen. 
</P><P>As may have become clear the digestion and purification of our DNA in preparation for the production of BioBricks did not go quite as planned. Due to this we would have to redo todays experiments at a later date.
</P><H5><SPAN class="mw-headline" id="Repeating_the_Digestion">Repeating the Digestion</SPAN></H5><P>As a number of mistakes had been made, as mentioned above, it was decided to repeat the digestion we had performed this morning. The same solutions were made in the same way. Further more the same protocol was used. These 5 digested constructs and the digested backbone were then placed in the -30°C freezer and left there until they were needed again.  
</P></DIV></DIV><DIV class="tue-timeline-middle"><DIV class="tue-timeline-leftbar"><DIV class="tue-timeline-node">28 August2013</DIV></DIV><DIV class="tue-timeline-content"><H3><SPAN class="mw-headline" id="Protein_Purification_2">Protein Purification</SPAN></H3><P>On August 27th 6 cultures had been placed in the incubator for aerobic protein expression. These now needed to be purified so that we could retain and analyse the protein. the 6 constructs we had used were EGFP, 1ETF, 1G70, Poly(Arginine-Glycine), Poly(Threonine-Lysine) and Poly(Lysine-Serine):
</P><UL><LI>Each of the cultures was poured into a centrifuge flasks and all flasks were weighed evenly, accounting for the weight difference with water.
</LI><LI>The flasks were then centrifuged for 10 minutes at 10000g.
</LI><LI>Before moving on with the purification a side step was taken to ensure we could use these proteins to measure the CEST effect:
</LI><LI>The supernatant from each flask was discarded.
</LI><LI>From the pellet a small (30mg) sample was taken and placed in a small PCR tube.
</LI><LI>The pellet was spun down into the bottom of the tube. 
</LI><LI>Finally 250µL of 4% PFA solution was added to the pellet, fixing it for storage in the refrigerator at 4°C.
</LI></UL><LI>8mL of BugBuster was then added to the remainder of the pellet along with 8µL of Benzonase. 
</LI><LI>The pellet and solution were placed on the shaking table and left for 30 minutes. 
</LI><LI>The dissolved pellets were then poured across into smaller centrifuge flasks which were again weight equalized with water.
</LI><LI>The small flasks were then placed in the centrifuge and spun down for 20 minutes at 16000g. 
</LI><H4><SPAN class="mw-headline" id="SDS_PAGE_Analysis">SDS PAGE Analysis</SPAN></H4><P>After centrifuging we could now prepare samples for an SDS analysis. This analysis would hopefully reveal whether our protein was situated in the supernatant or the pellet, or if it had not been expressed at all. The following steps were taking in the gel analysis.
</P><UL><LI>15µL of each of the supernatants was pipetted into a separate PCR tube. The remaining supernatant was snap frozen and stored in -80°C
</LI><LI>From the pellet a small sample was taken and dissolved in 500µL of Tris buffer (pH 8.0). 15µL of this solution was then also pipetted into a small PCR tube. The rest of the pellet was then also snap frozen and stored in-80°C.
</LI><LI>To each of the 12 PCR tubes we now had, 15µL of loading dye could be added. This loading dye was retrieved from a master mix containing 500µL of 2x loading dye and 1µL of DTT. 
</LI><LI>The samples were then denatured at 95°C for 5 minutes.
</LI><LI>10µL of each sample was then loaded onto a 12% SDS PAGE gel and run. The running of the gel was done at 90V for 15 minutes and then at 150V for 40 minutes.
</LI><LI>After running the gel was then placed on a shaking table and washed with Coomaaise Blue overnight. 
</LI></UL><H3><SPAN class="mw-headline" id="Anaerobic_Expression_2">Anaerobic Expression</SPAN></H3><P>Having prepared 6 small cultures tubes with XL-1 Blue bacteria with therein pBR322 ligated EGFP and a large 4L culture medium we could now perform the anaerobic expression. The following table lists chronologically describes all the steps that were taken during the anaerobic expression:
</P><TABLE class="table table-striped"><TBODY><TR><TH align="center"> Time
</TH><TH> Procedure
</TH></TR><TR><TD>08:45 </TD><TD> Culture vessel was flooded with air. (10L/min)
</TD></TR><TR><TD>09:15 </TD><TD> Ampicillin was added to the LB medium inside the vessel (4mL of 100µg/mL).
</TD></TR><TR><TD>09:30 </TD><TD> The 6 EGFP culture tubes were inserted into the vessel using a syringe. The air flow was set to 3L/min and the Oxygen level was calibrated to 100%.
</TD></TR><TR><TD>09:40 </TD><TD> A sample was taken to measure the optical density, which was : 0.024. (pH = 9.99, dO<SUB>2</SUB> = 98.8, Temp = 36.9°C)
</TD></TR><TR><TD>10:00 </TD><TD> pH calibration probe was set at pH 7.0.
</TD></TR><TR><TD>10:20 </TD><TD> The agitation motor was slowed from 300rpm to 150rpm for foam control.
</TD></TR><TR><TD>10:55 </TD><TD> A sample was taken to measure the optical density, which was : 0.070. (pH = 6.8, dO<SUB>2</SUB> = 96.0, Temp = 37.0°C)
</TD></TR><TR><TD>11:15 </TD><TD> The air flow was increased from 3 to 4 L/min. (pH = 6.97, dO<SUB>2</SUB> = 93.0, Temp = 37.0°C).
</TD></TR><TR><TD>11:35 </TD><TD> A sample was taken to measure the optical density, which was : 0.181. (pH = 6.95, dO<SUB>2</SUB> = 92.4, Temp = 37.0°C)
</TD></TR><TR><TD>12:10 </TD><TD> A sample was taken to measure the optical density, which was : 0.384. The air flow was set to 8L/min. (pH = 6.88, dO<SUB>2</SUB> = 82.6, Temp = 37.0°C).
</TD></TR><TR><TD>12:30 </TD><TD> A sample was taken to measure the optical density, which was : 0.532. The air flow was set to 10L/min. (pH = 6.81, dO<SUB>2</SUB> = 47.8, Temp = 36.8°C)
</TD></TR><TR><TD>12:37 </TD><TD> The vessel was flooded with 100% Nitrogen.
</TD></TR><TR><TD>12:45 </TD><TD> The gas addition was changed to a self regulating 4-gas setting (mixing only air and Nitrogen). (pH = 6.82, dO<SUB>2</SUB> = -0.1, Temp = 37.0°C)
</TD></TR><TR><TD>13:09 </TD><TD> A sample was taken for later analysis. (pH = 6.75, dO<SUB>2</SUB> = -0.1, Temp = 37.0°C)
</TD></TR><TR><TD>14:25 </TD><TD> A sample was taken for later analysis. (pH = 6.34, dO<SUB>2</SUB> = -0.1, Temp = 37.0°C)
</TD></TR><TR><TD>15:30 </TD><TD> A sample was taken for later analysis. (pH = 6.57, dO<SUB>2</SUB> = 0.0, Temp = 37.0°C)
</TD></TR><TR><TD>16:00 </TD><TD> Agitation was changed to 200rpm to reduce foam build up.
</TD></TR><TR><TD>16:15 </TD><TD> pH regulation was activated in an attempt to raise the pH. A sample was taken for later analysis. (pH = 6.87, dO<SUB>2</SUB> = -0.2, Temp = 37.1°C)
</TD></TR><TR><TD>17:40 </TD><TD> A sample was taken for later analysis. (pH = 7.53, dO<SUB>2</SUB> = -0.2, Temp = 37.0°C)
</TD></TR><TR><TD>18:45 </TD><TD> A sample was taken for later analysis. (pH = 7.60, dO<SUB>2</SUB> = -0.1, Temp = 37.0°C)
</TD></TR><TR><TD>19:00 </TD><TD> The vessel was left on after we left the lab. The gas addition was set to 100% Nitrogen. The temperature would be set to 37°C overnight.
</TD></TR></TBODY></TABLE><P>The expression of EGFP would hopefully continue overnight in the anaerobic chamber. 
</P></DIV></DIV><DIV class="tue-timeline-middle"><DIV class="tue-timeline-leftbar"><DIV class="tue-timeline-node">29 August2013</DIV></DIV><DIV class="tue-timeline-content"><H3><SPAN class="mw-headline" id="Anaerobic_Expression_3">Anaerobic Expression</SPAN></H3><P>We started today where we had left off on August 28th. We had left the anaerobic chamber on overnight and were going to continue taking samples today to see how the expression would be affected. The following table shows the steps taken:
</P><TABLE class="table table-striped"><TBODY><TR><TH align="center"> Time
</TH><TH> Procedure
</TH></TR><TR><TD>08:20 </TD><TD> Gas addition changed from 4 gas system to 100% Nitrogen. A sample was taken for later analysis. (pH = 7.73, dO<SUB>2</SUB> = -0.3, Temp = 37.0°C)
</TD></TR><TR><TD>17:00 </TD><TD> A sample was taken for later analysis. The vessel was then turned off. (pH = 7.66, dO<SUB>2</SUB> = -0.2, Temp = 37.0°C)
</TD></TR></TBODY></TABLE><P>It should be noted that all the retained samples (for analysis) were stored in the 4°C refrigerator, both today as on August 28th. 
</P><H3><SPAN class="mw-headline" id="Gel_Analysis">Gel Analysis</SPAN></H3><P>On August 28th a number of aerobic protein expression samples from the proteins: EGFP, 1ETF, 1G70, Poly(Arginine-Glycine), Poly(Threonine-Lysine) and Poly(Lysine-Serine) were loaded onto a gel and run. This gel was now ready for analysis. 
</P><DIV class="row"><DIV class="tue-float tue-float-Centre span12"><DIV class="tue-caption"><SPAN class="tue-caption-figure"><SPAN class="tue-refid">SDS_SuperPellet_EGFP_1ETF_1G70</SPAN></SPAN> <SPAN class="tue-caption-text">SDS gel of expressed proteins from supernatant and pellet, respectively: 1: ladder, 2-3: EGFP, 4-5: 1ETF, 6-7: 1G70, 8-9: poly(Threonine-Lysine), 10-11: poly(Lysine-Serine), 12-13: poly(Arginine-Glycine), 14: ladder</SPAN></DIV></DIV></DIV><P>In the above picture it is clear that EGFP possesses protein both in the supernatant as in the pellet, whereas 1ETF and 1G70 show only proteins in the pellet. The Poly-chains however show no protein expression in either the supernatant or the pellet. As we would like to obtain the protein from the 1ETF and 1G70 we decided that we should perform an inclusion body extraction.
</P><H3><SPAN class="mw-headline" id="Inclusion_Body_Extraction">Inclusion Body Extraction</SPAN></H3><P>The pelleted samples of 1ETF and 1G70 were retained from the -80°C storage freezer and were treated in the following manner:
</P><UL><LI>First 5mL of inclusion body solubilization reagent was added to each pellet.
</LI><LI>The pellets were then placed on the shaking table for 45 minutes until the pellets had dissolved. 
</LI><LI>The samples were then weighed equally and centrifuged at 27000g for 15 minutes. 
</LI><LI>The supernatant was poured off into a falcon tube. The pellet itself was discarded. 
</LI><LI>15mL of Tris buffer (pH 8.0) was then added to dilute the supernatants.
</LI></UL><P>The supernatant now contained our protein but it had to be purified to remove other native proteins. 
</P><H3><SPAN class="mw-headline" id="Nickel_Column_Purification">Nickel Column Purification</SPAN></H3><P>The diluted supernatants could now be loaded onto the Nickel column and purified. The purification was carried out in the following fashion:
</P><UL><LI>Before beginning the strip buffer on the column was removed and the column was equalized with 15mL of sterile water. 
</LI><LI>The column was then loaded with 8mL of Charge buffer containing 50mM NiSO<SUB>4</SUB>. The flow through of the column was stored separately due to the heavy metal content. 
</LI><LI>Once empty 16mL of Wash buffer was loaded and again the flow through was separately caught to reduce the risk of heavy metal contamination. 
</LI><LI>The entire protein sample could then be loaded onto the column. Of the flow through a single droplet was retrieved for future analysis. 
</LI><LI>The column was then washed once more with 16mL of wash buffer and again a single droplet was caught for analysis. 
</LI><LI>Now elution was performed by loading the column with 8mL of elution buffer. All the flow through droplets were retrieved by catching 0.5mL of flow through in separate eppendorf tubes. 
</LI><LI>The column was then stripped with 8mL of strip buffer, whereby the flow through was retrieved separately due to the Nickel content. Strip buffer was then added to the column before sealing it and placing it in the 4°C refrigerator.
</LI></UL><H3><SPAN class="mw-headline" id="SDS_Analysis_2">SDS Analysis</SPAN></H3><P>Using 5 of 16 acquired elution samples per protein (1ETF and 1G70), along with the loading and washing samples we were now able to run an SDS PAGE gel to see if we had managed to purify our proteins. 
The 4th, 5th, 6th, 7th and 8th elution fractions were chosen for loading onto the gel. The gel preparation went as follows:
</P><UL><LI>Firstly 10µL of each of the above mentioned elution fractions was pipetted into a PCR tube. The same was done for the load and wash fractions. 
</LI><LI>To each PCR tube 10µL of SDS loading dye was added. The loading dye was prepared suing 2x SDS loading dye and DTT. 
</LI><LI>The samples were then denatured at 95°C for 5 minutes.
</LI><LI>10µL of each sample was then loaded onto the SDS gel which was subsequently run for 15 minutes at 90V and then for 40 minutes at 150V. 
</LI><LI>After running the gel was washed in water before being placed in Coomaaise Blue overnight (on a shaking table). 
</LI></UL><H3><SPAN class="mw-headline" id="Rebuffering_of_the_Proteins">Rebuffering of the Proteins</SPAN></H3><P>The 5 elution fractions we had chosen for the SDS analysis should have contained the highest concentration of protein, and would need to be rebuffered. This would be done using a PD-10 rebuffering column. and the following steps:
</P><UL><LI>First the column was opened and the storage buffer was poured off and discarded.
</LI><LI>Next 25mL of TRIS buffer (pH 8.0) was allowed to run through the column. The flow through was discarded.
</LI><LI>After this we were able to load our sample, during which the flow through was discarded. A maximum of 2.5mL of our sample could be loaded, which was the exact amount we had chosen to rebuffer. 
</LI><LI>Then 3.5mL of TRIS Buffer was run through the column for the elution, and the flow through was retained as it contained our protein. Retention was done in a number of eppendorf tubes each holding 1mL of the flow through.
</LI><LI>The proteins were then snap frozen in liquid nitrogen and stored in -80 degrees Celsius. 
</LI></UL><H3><SPAN class="mw-headline" id="Small_Cultures_2">Small Cultures</SPAN></H3><P>To be able to send in a number of BioBricks we would need to multiply the amount of pUC57-Simple vectors we had containing EGFP, 1G70, Poly(Threonine-Lysine) and Poly(Lysine-Serine). As we still had these vectors plated in NB bacteria it would be a simple case of making a small culture and then retaining the DNA using miniprep. The small cultures were prepared as follows:
</P><UL><LI>All steps were performed in the vicinity of a blue bunsen for increased sterility.
</LI><LI>For each of the 4 constructs we prepared 3 culture tubes with 8mL of LB and 8µL of Ampicillin antibiotics.
</LI><LI>A single NB colony containing the pUC57-simple vector was picked and dissolved in15µL of sterile water for each of the constructs.
</LI><LI>%µL of the above mentioned colony picking solution was then pipetted into each of the culture tubes.
</LI><LI>The culture tubes were then placed in the 37°C incubator overnight. 
</LI></UL><H3><SPAN class="mw-headline" id="Making_Agar_Plates">Making Agar Plates</SPAN></H3><P>As we would need to perform extra transformations to produce the desired BioBricks we would need to be in possession of agar plates containing Chloraphenicol antibiotics. The agar solution to pour these plates was made by mixing the following:
</P><UL><LI>3g Peptones.
</LI><LI>3G NaCl.
</LI><LI>1.5g Yeast Extract.
</LI><LI>4.5g Agar.
</LI><LI>300mL Demineralised Water.
</LI></UL><P>After autoclaving the above solution and allowing it to cool 300µg of Chloraphenicol was added to it. This solution was then poured out over 14 plate casings in the vicinity of a blue bunsen flame. 
</P></DIV></DIV><DIV class="tue-timeline-middle"><DIV class="tue-timeline-leftbar"><DIV class="tue-timeline-node">30 August2013</DIV></DIV><DIV class="tue-timeline-content"><H3><SPAN class="mw-headline" id="SDS_Analysis_3">SDS Analysis</SPAN></H3><P>The SDS PAGE gel which we had made on August 29th had been stained using Coomassie overnight and could now be imaged. On this imaged we saw that the 1G70 protein had indeed been purified perfectly, however the 1ETF sample had not. As it did not occur in the wash or load sample this led us to believe that the fault lay in the inclusion body purification. The most likely cause for this would be the insolubility of the protein.
</P><DIV class="row"><DIV class="tue-float tue-float-Centre span12"><DIV class="tue-caption"><SPAN class="tue-caption-figure"><SPAN class="tue-refid">SDS_PurInclusion_1ETF_1G70</SPAN></SPAN> <SPAN class="tue-caption-text">SDS gel of purified proteins: 1: ladder, 2: 1ETF loaded 3: 1ETF washed, 3-7: 1ETF eluted, 8: 1G70 loaded, 9: 1G70 washed, 10-14: 1G70 eluted</SPAN></DIV></DIV></DIV><H3><SPAN class="mw-headline" id="BugBuster_Protocol">BugBuster Protocol</SPAN></H3><P>The anaerobic samples we had taken on August 28th and 29th could now be checked to see if EGFP had been produced. We had taken 8 samples but only 6 of these would be analysed, (The 1st, 2nd, 4th, 6th, 7th and 8th). This was done using the following BugBuster protocol:
</P><UL><LI>First the sample were evenly weighed using water and spun down for 10 minutes at 10000g.
</LI><LI>The supernatant was then removed and 500µL of BugBuster was added to the pellet. Also 0.5µL Bensonaze was added.
</LI><LI>The pellet was then dissolved by leaving on a shaker for 30 minutes.
</LI><LI>The resuspended pellets were then pipetted into 1.5mL eppendorf tubes and centrifuged on a table top centrifuge for 5 minutes at 13400rpm.
</LI></UL><H3><SPAN class="mw-headline" id="SDS_PAGE_Gel">SDS PAGE Gel</SPAN></H3><P>Using the supernatant from the BugBustered samples we could now load and run an SDS PAGE gel. This was done as follows:
</P><UL><LI>From each of the 6 sample 10µL of supernatant was pipetted into a PCR tube.
</LI><LI>A loading dye mastermix was made using 100µL of 2x loading dye and 0.2µL of DTT. 
</LI><LI>&amp;10micro;L of the loading dye master mix was added to each of the PCR tubes.
</LI><LI>10µL of each sample was then loaded onto a 12% SDS PAGE gel which was subsequently run for 15 minutes at 90V and the for 40 minutes at 150V. 
</LI></UL><P>After running the gel it was washed with water before being placed in Coomaaise blue stainer for the weekend. 
</P><H3><SPAN class="mw-headline" id="Miniprepping_the_Small_Cultures">Miniprepping the Small Cultures</SPAN></H3><P>On August 29th small cultures had been made for the pUC57-simple constructs of EGFP, 1ETF, Poly(Threonine-Lysine) and Poly(Lysine-Serine). Each of these small cultures would now be miniprepped to retain the DNA using the following protocol:
</P><UL><LI>The culture tubes were spun down in a centrifuge for 10 minutes at 3700g causing the bacterial cells with the DNA inside to form pellets isolating it from the growth medium. 
</LI><LI>The supernatant that had formed above the pellets was discarded and the pellets themselves were resuspended in 250µL of P1 buffer. The tubes were gently shaken by hand until the entire pellet had resuspended.
</LI><LI>The suspension was then transferred by means of pipetting into a smaller (1.5mL) eppendorf tube. 
</LI><LI>250µL of P2 buffer was then added to the eppendorf tube and the solution was inverted by hand until it had turned a clear blue. 
</LI><LI>Within 5 minutes of adding the P2 buffer 350µL of N3 buffer was added to the solution and the eppendorf tube was inverted again until the solution was clear again. 
</LI><LI>Once clear the eppendorf tube was placed in the centrifuge and spun for 10 minutes at 13000rpm seperating the bacterial cells from the DNA which remained suspended in the solution. The bacterial cells formed a pellet in the bottom of the eppendorf tube. 
</LI><LI>As soon as the centrifuge had stopped and we could see that the pellet had properly formed, the supernatant was poured off into a special QIAcolumn. The pellet could be discarded in the bio-hazard waste. 
</LI><LI>The QIAcolumn was subsequently centrifuged for one minute at 13000rpm, during which the DNA vector bound to the column meaning the flow through could be discarded. 
</LI><LI>750µL of PE buffer was then added to the QIAcolumn before centrifuging it for another minute at 13000rpm. The flow through was discarded.
</LI><LI>To ensure that all the PE buffer had passed through the column it was then spun yet again for one minute at a speed of 13000rpm. Yet again the flow through could be discarded.
</LI><LI>The column itself was then placed above a sealable 1.5mL eppendorf tube so that the DNA could be contained and stored. To obtain the DNA from the column 36µL of MilliQ water was pipetted onto the very centre membrane of the column after which it was spun down for one minute at 13000rpm. The flow through now sat in the 1.5mL eppendorf tube and contained the DNA vectors.
</LI></UL><P>The following concentrations were then measured for each of the DNA samples:
</P><TABLE class="table table-striped"><TBODY><TR><TH align="center"> Construct
</TH><TH> Concentration (ng/µL)
</TH></TR><TR><TD>EGFP </TD><TD>  275.4
</TD></TR><TR><TD>1G70 </TD><TD>  243.6
</TD></TR><TR><TD>Poly(Threonine-Lysine) </TD><TD>  178.7
</TD></TR><TR><TD>Poly(Lysine-Serine) </TD><TD>  192.2
</TD></TR></TBODY></TABLE></DIV></DIV></DIV></DIV><DIV class="tab-pane" id="Week7"><DIV><DIV class="tue-timeline-top"><DIV class="tue-timeline-title"><H2>Week 7</H2></DIV></DIV><DIV class="tue-timeline-middle"><DIV class="tue-timeline-leftbar"><DIV class="tue-timeline-node">4 September2013</DIV></DIV><DIV class="tue-timeline-content"><H3><SPAN class="mw-headline" id="Digesting_Samples">Digesting Samples</SPAN></H3><P>Previously, on August 27th we had digested a number of our constructs (Protamine-1-Optimized, 1ETF, 1PJN, Poly(Arginine-Glycine) and Poly(Arginine-Serine)) along with the pSB1C3 vector. Before attempting to purify and ligate these samples we would first digest the remaining four constructs (EGFP, 1G70, Poly(Threonine-Lysine) and Poly(Lysine-Serine)). The digestion solutions for these digestions were made as follows and two digestion solutions were made for each of the constructs so that 10µg of DNA would be digested per construct:
</P><TABLE class="table table-striped"><TBODY><TR><TH align="center"> Construct
</TH><TH> 5µg DNA (µL)
</TH><TH> CutSmart Buffer (µL)
</TH><TH> Amount of each Enzyme (µL)
</TH><TH> MilliQ water (µL)
</TH></TR><TR><TD>EGFP
</TD><TD>18.2
</TD><TD>5
</TD><TD>0.7
</TD><TD>25.4
</TD></TR><TR><TD>1G70
</TD><TD>20.5
</TD><TD>5
</TD><TD>0.7
</TD><TD>23.1
</TD></TR><TR><TD>Poly(Threonine-Lysine)
</TD><TD>28.0
</TD><TD>5
</TD><TD>0.7
</TD><TD>15.6
</TD></TR><TR><TD>Poly(Lysine-Serine)
</TD><TD>26.0
</TD><TD>5
</TD><TD>0.7
</TD><TD>17.6
</TD></TR></TBODY></TABLE><H3><SPAN class="mw-headline" id="Gel_Extraction_Purification">Gel Extraction Purification</SPAN></H3><P>As we had now digested all 9 of the constructs we could now move on to purifying these samples. As the digested inserts were relatively large we could not use the conventional PCR purification columns we had to perform a gel extraction: 
</P><UL><LI> To allow for the loading of all nine samples two 1.5% gels were prepared by mixing the following for each gel:
</LI><LI>1.8g Agarose
</LI><LI>120mL TAE 
</LI><LI>12µL CyberSafe
</LI></UL><LI>When pouring the gels 4 columns were bound together to allow for the loading of the entire 100µL of sample we had for each of the construct digestions.
</LI><LI>The digestion samples were then pipetted into the gel and the gel was run for 60 minutes at 100V. 
</LI><LI>After running the gels they were placed atop a blue light allowing us to see the bands of DNA. For each of the constructs we then cut out the lower band and split this over two separate eppendorf tubes, each tube containing no more than 400mg of gel cut-out. 
</LI><LI> Per mg of gel cut-out, 3µL of Buffer QG was added to each of the eppendorf tubes. 
</LI><LI>The solution was then incubated for 10 minutes at 37°C, vortexing every couple of minutes until the gel was fully dissolved. 
</LI><LI>Thereafter 1µL of isopropanol per mg gel was added to the solution. 
</LI><LI>800µL of each sample was transferred to a QIAspin column and centrifuged for 60 seconds, the flow-through was subsequently discarded. This step was repeated until the entire sample had been loaded onto the column. 
</LI><LI>Afterwards, the columns were washed twice by adding 750 µL of PE buffer, centrifuging for 60 seconds, and removing the flow-through. 
</LI><LI>Hereafter a dry spin was performed for another 60 seconds. 
</LI><LI>The DNA was eluded from the column by adding 42 µL of sterile water to the centre of the column membrane. After letting the columns stand for 60 seconds, the columns were centrifuged once more and as a result, the DNA was in the flow-through. 
</LI><H3><SPAN class="mw-headline" id="PCR_Purification_2">PCR Purification</SPAN></H3><P>To prepare the backbone for ligation it too had to be purified after the digestion we performed on August 27th. This was done in the following manner using a standard PCR purification column:
</P><UL><LI>150µL of PB Buffer was added to the digested Backbone sample. This was loaded onto a QIA PCR purification column. 
</LI><LI>The column was centrifuged for 60 seconds and the flow-through was removed.
</LI><LI>750µL PE Buffer was added. After another 60 seconds of centrifuging, the flow-through was removed again and the sample was dry spun for another 60 seconds. 
</LI><LI>The backbone DNA was eluded by adding 42µL of sterile water to the centre membrane of the column and centrifuging for 60 seconds. 
</LI></UL><H3><SPAN class="mw-headline" id="Nanodrop_Test_3">Nanodrop Test</SPAN></H3><P>Before being able to perform ligation we would have to find out what concentrations of DNA we had. This was done for both the constructs and the backbone with the following results:
</P><TABLE class="table table-striped"><TBODY><TR><TH align="center"> Construct
</TH><TH> Concentration (ng/µL)
</TH></TR><TR><TD>pSB1C3 Vector </TD><TD> 8.0
</TD></TR><TR><TD>Protamine-1-Optimized </TD><TD> 2.1
</TD></TR><TR><TD>1ETF </TD><TD> 6.4
</TD></TR><TR><TD>1PJN </TD><TD> 3.1
</TD></TR><TR><TD>1G70 </TD><TD> 13.7
</TD></TR><TR><TD>EGFP </TD><TD> 21.9
</TD></TR><TR><TD>Poly(Arginine-Glycine) </TD><TD> 13.5
</TD></TR><TR><TD>Poly(Arginine-Serine) </TD><TD> 12.9
</TD></TR><TR><TD>Poly(Threonine-Lysine) </TD><TD> 14.0
</TD></TR><TR><TD>Poly(Lysine-Serine) </TD><TD> 13.2
</TD></TR></TBODY></TABLE><H3><SPAN class="mw-headline" id="Ligation_3">Ligation</SPAN></H3><P>To perform the ligation a number of calculations had to be performed in which we determined how many µL of each insert we would need to add to 32ng of backbone to ensure successful ligation. The method we used for these calculations has been explained extensively during the LogBook entry of August 15th and we will not be repeating these explanations here. The final ligation solutions we prepared are given below:
</P><TABLE class="table table-striped"><TBODY><TR><TH align="center"> Construct
</TH><TH> Volume Insert (µL)
</TH><TH> Volume Backbone (µL)
</TH><TH> 10x Buffer (µL)
</TH><TH> Sterile Water (µL)
</TH><TH> T4 Ligase (µL)
</TH></TR><TR><TD>Protamine-1-Optimized </TD><TD> 18.0 </TD><TD> 4 </TD><TD> 2.5 </TD><TD> 0 </TD><TD> 1
</TD></TR><TR><TD>1ETF </TD><TD> 8.8 </TD><TD> 4 </TD><TD> 2.5 </TD><TD> 8.7 </TD><TD> 1
</TD></TR><TR><TD>1PJN </TD><TD> 18.8 </TD><TD> 4 </TD><TD> 2.5 </TD><TD> 0 </TD><TD> 1
</TD></TR><TR><TD>1G70 </TD><TD> 4.2 </TD><TD> 4 </TD><TD> 2.5 </TD><TD> 13.3 </TD><TD> 1
</TD></TR><TR><TD>EGFP </TD><TD> 3.7 </TD><TD> 4 </TD><TD> 2.5 </TD><TD> 13.8 </TD><TD> 1
</TD></TR><TR><TD>Poly(Arginine-Glycine) </TD><TD> 4.5 </TD><TD> 4 </TD><TD> 2.5 </TD><TD> 13.0 </TD><TD> 1
</TD></TR><TR><TD>Poly(Arginine-Serine) </TD><TD> 4.6 </TD><TD> 4 </TD><TD> 2.5 </TD><TD> 12.9 </TD><TD> 1
</TD></TR><TR><TD>Poly(Threonine-Lysine) </TD><TD> 4.3 </TD><TD> 4 </TD><TD> 2.5 </TD><TD> 13.2 </TD><TD> 1
</TD></TR><TR><TD>Poly(Lysine-Serine) </TD><TD> 4.5 </TD><TD> 4 </TD><TD> 2.5 </TD><TD> 13.0 </TD><TD> 1
</TD></TR></TBODY></TABLE><P>The above ligation solutions were then placed in the PCR apparatus and set to run with the following program:
</P><UL><LI>25°C for 120 minutes.
</LI><LI>80°C for 10 minutes for enzyme deactivation.
</LI><LI>4°C FOREVER for overnight storage.
</LI></UL></DIV></DIV><DIV class="tue-timeline-middle"><DIV class="tue-timeline-leftbar"><DIV class="tue-timeline-node">5 September2013</DIV></DIV><DIV class="tue-timeline-content"><H3><SPAN class="mw-headline" id="Transformation_3">Transformation</SPAN></H3><P>Having performed a ligation on all nine constructs (Protamine-1-Optimzed, 1ETF, 1PJN, 1G70, EGFP, Poly(Arginine-Glycine), Poly(Arginine-Serine), Poly(Threonine-Lysine) and Poly(Lysine-Serine)) on August 5th, we could now transform these ligated vectors into NB bacteria to multiply the DNA available. The transformation was completed as follows:
</P><UL><LI>The ligation products were transferred to NB bacteria by adding 1µL of the ligation product to 20µL of NB bacteria (left on ice for 5 minutes). 
</LI><LI>Then the solution was heat shocked for 30 seconds at 42°C. 
</LI><LI>After heat shocking, the solution was returned to ice for 2 minutes. 
</LI><LI>Then 80µL of SOC solution was added and the bacteria were not be placed on ice again. 
</LI><LI>The solution was now incubated for 1 hour at 37°C.
</LI></UL><H3><SPAN class="mw-headline" id="Plating_4">Plating</SPAN></H3><P>After the transformation products had been incubated they could be plated on to agar plates for growth. These agar plates had been prepared on August 29th, containing chloramphenicol antibiotics, in the following manner:
</P><UL><LI>The following steps were performed in the vicinity of a blue bunsen flame for sterility.
</LI><LI>The entire transformation solution was pipetted onto the centre of the agar plate.
</LI><LI>Using a sterile spreader the bacteria were spread out across the plate.
</LI><LI>The agar plates were subsequently placed in the 37°C incubator overnight. 
</LI></UL></DIV></DIV><DIV class="tue-timeline-middle"><DIV class="tue-timeline-leftbar"><DIV class="tue-timeline-node">6 September2013</DIV></DIV><DIV class="tue-timeline-content"><H3><SPAN class="mw-headline" id="Colony_Picking_and_PCR_4">Colony Picking and PCR</SPAN></H3><P>Having plated our nine constructs ligated with the pSB1C3 vector transformed into NB we could now begin working on the colony picking and PCR to check whether or not the ligations had been successful. Before beginning we already noticed that on the agar plates for 1G70 and Poly(Lysine-Serine) no colonies had grown. These would therefore have to be digested, ligated and transformed again separately. The colony picking and PCR was resumed for the remaining 7 constructs as follows:
</P><DIV class="row"><DIV class="tue-float tue-float-left span3"><DIV class="tue-caption"><SPAN class="tue-caption-figure"><SPAN class="tue-refid">pSB1C3_Pro_Col</SPAN></SPAN> <SPAN class="tue-caption-text">pSB1C3 Protamine-1</SPAN></DIV></DIV><DIV class="tue-float tue-float-left span3"><DIV class="tue-caption"><SPAN class="tue-caption-figure"><SPAN class="tue-refid">pSB1C3_ETF_Col</SPAN></SPAN> <SPAN class="tue-caption-text">pSB1C3 ETF1</SPAN></DIV></DIV><DIV class="tue-float tue-float-left span3"><DIV class="tue-caption"><SPAN class="tue-caption-figure"><SPAN class="tue-refid">pSB1C3_1G70_Col</SPAN></SPAN> <SPAN class="tue-caption-text">pSB1C3 1G70</SPAN></DIV></DIV><DIV class="tue-float tue-float-left span3"><DIV class="tue-caption"><SPAN class="tue-caption-figure"><SPAN class="tue-refid">pSB1C3_PJN_Col</SPAN></SPAN> <SPAN class="tue-caption-text">pSB1C3 PJN1</SPAN></DIV></DIV><DIV class="tue-float tue-float-left span3"><DIV class="tue-caption"><SPAN class="tue-caption-figure"><SPAN class="tue-refid">pSB1C3_P(RG)</SPAN></SPAN> <SPAN class="tue-caption-text">pSB1C3 Poly(Arginine-Glycine)</SPAN></DIV></DIV><DIV class="tue-float tue-float-left span3"><DIV class="tue-caption"><SPAN class="tue-caption-figure"><SPAN class="tue-refid">pSB1C3_P(RS)</SPAN></SPAN> <SPAN class="tue-caption-text">pSB1C3 Poly(Arginine-Serine)</SPAN></DIV></DIV><DIV class="tue-float tue-float-left span3"><DIV class="tue-caption"><SPAN class="tue-caption-figure"><SPAN class="tue-refid">pSB1C3_P(TK)</SPAN></SPAN> <SPAN class="tue-caption-text">pSB1C3 Poly(Threonine-Lysine)</SPAN></DIV></DIV><DIV class="tue-float tue-float-left span3"><DIV class="tue-caption"><SPAN class="tue-caption-figure"><SPAN class="tue-refid">pSB1C3_P(KS)</SPAN></SPAN> <SPAN class="tue-caption-text">pSB1C3 Poly(Lysine-Serine)</SPAN></DIV></DIV></DIV><UL><LI>In the vicinity of the flame 4 colonies were picked from each of the agar plates and dissolved in 15µL of sterile water.
</LI><LI>Of these solutions we could then perform colony PCR. This was done by preparing the following sample for each of the solutions:
</LI></UL><TABLE class="table table-striped"><TBODY><TR><TH align="center"> Substance
</TH><TH> Volume (µL)
</TH></TR><TR><TD>Sterile Water </TD><TD> 9.0
</TD></TR><TR><TD>Kappa2G </TD><TD> 12.5
</TD></TR><TR><TD>Fwd Primer </TD><TD> 1.25
</TD></TR><TR><TD>Rwd Primer </TD><TD> 1.25
</TD></TR><TR><TD>Template DNA </TD><TD> 1.0
</TD></TR></TBODY></TABLE><UL><LI>The primers were those as provided by the iGEM help pages for the amplification of our ligation insert.
</LI><LI>Having prepared the above solutions we placed the eppendorf tubes in the PCR apparatus and ran the following protocol:
</LI><LI> Initial Denaturation: 95°C for 3 minutes.
</LI><LI> The folllowing steps were repeated 30 times:
</LI><LI>Denaturation: 95°C for 15 seconds.
</LI><LI>Annealing: 58°C for 15 seconds.
</LI><LI>Extension: 72°C for 45 seconds.
</LI></UL><LI>Final Extension: 72°C for 10 minutes.
</LI><H3><SPAN class="mw-headline" id="Gel_analysis_2">Gel analysis</SPAN></H3><P>To analyse the results of the clony PCR we prepared a 1% agarose gel by mixing the following:
</P><UL><LI>1.2g of Agarose.
</LI><LI>120mL of TAE.
</LI><LI>12µL of CyberSafe
</LI></UL><P>Having prepared the gel we then added 4µL of 6x Loading dye to each of the samples and loaded 13µL of each sample into one of the lanes of the gel. The gel was subsequently run for 60 minutes at 100V. 
</P><P>The results of the gel could show us which of the samples had been ligated correctly. We could then store those samples in the 4°C refrigerator so that we could culture these samples at a later date. 
</P></DIV></DIV></DIV></DIV><DIV class="tab-pane" id="Week8"><DIV><DIV class="tue-timeline-top"><DIV class="tue-timeline-title"><H2>Week 8</H2></DIV></DIV><DIV class="tue-timeline-middle"><DIV class="tue-timeline-leftbar"><DIV class="tue-timeline-node">9 September2013</DIV></DIV><DIV class="tue-timeline-content"><H3><SPAN class="mw-headline" id="Preparation_For_Protein_Expression_2">Preparation For Protein Expression</SPAN></H3><P>To ensure that all our native proteins would be tested in the MRI apparatus we needed to bring both the 1PJN and Protamine-1-Optimized proteins to expression aerobically and retain a section of the pellet for analysis. The two other native proteins (1ETF and 1G70) had already been expressed and are currently in storage in the 4°C refrigerator. To express the two proteins we wanted we first had to prepare a small culture for each of the proteins, allowing the bacterial cells to grow. We would also be preparing a control sample for the MRI experiments:
</P><UL><LI>All subsequent steps were performed in the vicinity of a blue bunsen flame for increased sterility.
</LI><LI>First three culture tubes were filled with 8mL of LB solution.
</LI><LI>To each of the culture tubes 8µL of Kanamycin antibiotics were added.
</LI><LI>For each of the proteins (1PJN and Protamine-1-Optimized) we were able to retrieve agar plates on which had been plated the relevant proteins within a pET28a vector inserted in NB bacteria. From each of the plates a picking was taken and the pipet point along with the picking was added to the culture tube. For the control a picking of 1PJN was added. 
</LI></UL><P>To further the preparations we also needed to prepare 3 large culture mediums. To make a single 400mL culture medium the following were mixed:
</P><UL><LI>4g Peptones
</LI><LI>4g NaCl
</LI><LI>2g Yeast Extract
</LI><LI>400mL Demineralised Water
</LI></UL><P>The above mixture was prepared for each of the three cultures and placed in the autoclave for sterilization.
</P><H3><SPAN class="mw-headline" id="Preparation_of_LB_Medium">Preparation of LB Medium</SPAN></H3><P>To ensure we would be able to prepare small cultures for the production of BioBricks we also needed to prepare an extra batch of LB culture medium. This was done by mixing the following:
</P><UL><LI>3g Peptones
</LI><LI>3g NaCl
</LI><LI>1.5g Yeast Extract
</LI><LI>300mL Demineralised Water
</LI></UL><P>The above mixture was subsequently sterilized by means of autoclaving.
</P></DIV></DIV><DIV class="tue-timeline-middle"><DIV class="tue-timeline-leftbar"><DIV class="tue-timeline-node">10 September2013</DIV></DIV><DIV class="tue-timeline-content"><H3><SPAN class="mw-headline" id="Protein_Expression_3">Protein Expression</SPAN></H3><P>On September 9th 3 small culture tubes had been prepared and had been left to grow in an incubator overnight. These grown samples could now be brought over to the large 400mL LB culture mediums, allowing for extra growth and ultimately expression. We had cultured 2 samples of 1PJN and 1 culture of Protamine-1-Optimized. This gave us the opportunity to prepare a number of control samples for the MRI experiments: 
</P><UL><LI>In the vicinity of a blue flame 400µL of Kanamycin antibiotics were added to each of the 400mL LB medium solutions.
</LI><LI>1 of the small 8mL culture tubes was then added to each of the 400mL LB medium solutions and they were placed in 37°C incubator. 
</LI><LI>The bacteria were allowed to grow at 37°C for approximately 120 minutes until the optical density had reached 0.6. Once this value had been reached IPTG (200µL) was added to the Protamine-1-Optimized culture and to one of the 1PJN cultures. 
</LI><LI>No IPTG was added to the second 1PJN culture as this would induce protein expression which we did not want. To act as a viable control we wanted to have bacterial cells which had grown under the same conditions but in which no protein expression had taken place. 
</LI><LI>The cultures were subsequently returned to the 37°C incubator and left overnight.
</LI></UL><H3><SPAN class="mw-headline" id="Preparing_BioBricks">Preparing BioBricks</SPAN></H3><P>On September 6th colony picking and PCR had been performed on all the constructs we had so that we would be able to make the biobricks we wanted to send to the iGEM headquarters. Using the results of this colony PCR we could now decide which of the colony pickings were adequate and multiply the vector DNA:
</P><UL><LI>For each of the 7 constructs (Protamine-1-Optimized, 1ETF, 1PJN, EGFP, Poly(Arginine-Glycine), Poly(Arginine-Serine), Poly(Threonine-Lysine)) two small cultures would be created. (The colony PCR or Ligation had failed for 1G70 and Poly(Lysine-Serine) so these would have to be digested and ligated a second time).
</LI><LI>Each of the following steps were performed in the vicinity of a blue bunsen flame. 
</LI><LI>As mentioned for each of the 7 successfully ligated constructs 2 culture tubes were filled with 8mL of the LB medium we had prepared on September 9th. 
</LI><LI>To each of the 14 culture tubes 8µL of Chloramphenicol antibiotics were added.
</LI><LI>To each of the 14 culture tubes we subsequently added 5µL of one of the successful colony picking samples. These pickings had been taken on September 9th and had been dissolved in sterile water. This solution had then been stored in the &amp;4deg;C refrigerator. 
</LI><LI>All 14 cultures were then placed in the 37°C incubator for overnight growth. 
</LI></UL></DIV></DIV><DIV class="tue-timeline-middle"><DIV class="tue-timeline-leftbar"><DIV class="tue-timeline-node">11 September2013</DIV></DIV><DIV class="tue-timeline-content"><H3><SPAN class="mw-headline" id="Preparation_for_CEST_measurements_2">Preparation for CEST measurements</SPAN></H3><P>Firstly the three cultures (1PJN, 1PJN control and Protamine-1-Optimized) were poured into separate 400mL flasks which would fit into a large centrifuge. The flasks were equally weighed to ensure that the centrifuge was well balanced.
The flasks with the culture medium were then spun down in the centrifuge at 10000G for 10 minutes forming a pellet containing our bacteria with our protein inside. The supernatant was therefore discarded so we were left with our pellet.
</P><P>To be able to measure the CEST effects of our proteins we would have to bring them to the MRI scanner. Here they will be tested on their contrast. As our bacteria would house our proteins should they be used to find tumours this is what we wished to replicate in the scanner. This would mean taking a piece of our pellet and storing it separately. The following steps therefore were taken:
</P><P>Firstly from the three large pellets we had just obtained a small portion was placed in a PCR tube. We made two samples PCR tubes with a small portion of pellet of the 1PJN control. The PCR tubes were then spun down at 13400 rpm for 3 minutes to place the pellet at the bottom of the PCR tube.
</P><P>To all four samples 250µL of 4% PFA (Paraformaldahyde) solution was added to fixate the cells pausing necrosis.
The four samples were then allowed to sit in the refrigerator at 4°C. To get the 4% PFA solution we diluted a 36% PFA stock 9 times.
</P><P>The rest of the pellets was kept in the large centrifuge flasks. We snap freezed these flasks and put them in the -80 freezer.
</P><H3><SPAN class="mw-headline" id="Miniprepping">Miniprepping</SPAN></H3><P>On September 9th thee constructs (Protamine-1-Optimized, 1ETF, 1PJN, EGFP, Poly(Arginine-Glycine), Poly(Arginine-Serine), Poly(Threonine-Lysine)) had been ligated into pSB1C3 vectors. On September 10th a number of samples of these ligation products were taken and cultured in 8mL culture tubes. Today we would isolate the DNA vectors from the culture medium, send off DNA for sequencing and for sending them to iGEM as BioBricks. To start off DNA would have to be retained which meant miniprepping the cultures, the protocol for which is given here:
</P><UL><LI>The culture tubes were spun down in a centrifuge for 10 minutes at 3700g causing the bacterial cells with the DNA inside to form pellets isolating it from the growth medium.
</LI><LI>The supernatant that had formed above the pellets was discarded and the pellets themselves were resuspended in 250µL of P1 buffer. The tubes were gently shaken by hand until the entire pellet had resuspended.
</LI><LI>The suspension was then transferred by means of pipetting into a smaller (1.5mL) eppendorf tube.
</LI><LI>250µL of P2 buffer was then added to the eppendorf tube and the solution was inverted by hand until it had turned a clear blue.
</LI><LI>Within 5 minutes of adding the P2 buffer 350µL of N3 buffer was added to the solution and the eppendorf tube was inverted again until the solution was clear again.
</LI><LI>Once clear the eppendorf tube was placed in the centrifuge and spun for 10 minutes at 13000rpm seperating the bacterial cells from the DNA which remained suspended in the solution. The bacterial cells formed a pellet in the bottom of the eppendorf tube. (Unfortunately the pellet had not completely formed after 10 minutes so the eppendorf tube was spun for a further minute, again at 13000rpm.)
</LI><LI>As soon as the centrifuge had stopped and we could see that the pellet had properly formed, the supernatant was poured off into a special QIAcolumn. The pellet could be discarded in the bio-hazard waste.
</LI><LI>The QIAcolumn was subsequently centrifuged for one minute at 13000rpm, during which the DNA vector bound to the column meaning the flow through could be discarded.
</LI></UL><P>750µL of PE buffer was then added to the QIAcolumn before centrifuging it for another minute at 13000rpm. The flow through was discarded.
</P><UL><LI>To ensure that all the PE buffer had passed through the column it was then spun yet again for one minute at a speed of 13000rpm. Yet again the flow through could be discarded.
</LI><LI>The column itself was then placed above a sealable 1.5mL eppendorf tube so that the DNA could be contained and stored. 
</LI><LI>To obtain the DNA from the column 42µL of MilliQ water was pipetted onto the very center membrane of the column after which it was spun down for one minute at 13000rpm. The flow through now sat in the 1.5mL eppendorf tube and contained the DNA vectors.
</LI><LI>The following concentrations were then measured for each of the DNA samples:
</LI></UL><TABLE class="table table-striped"><TBODY><TR><TH align="center"> Construct
</TH><TH> Ligation Vector
</TH><TH> Colony Picking Number
</TH><TH> Concentration (ng/µL)
</TH></TR><TR><TD>Poly(Arginine-Serine) </TD><TD> pSB1C3 </TD><TD> 2 </TD><TD> 85.4
</TD></TR><TR><TD>Poly(Arginine-Serine) </TD><TD> pSB1C3 </TD><TD> 3 </TD><TD> 54.3
</TD></TR><TR><TD>Poly(Arginine-Glycine) </TD><TD> pSB1C3 </TD><TD> 1 </TD><TD> 96.1
</TD></TR><TR><TD>Poly(Arginine-Glycine) </TD><TD> pSB1C3 </TD><TD> 2 </TD><TD> 36.0
</TD></TR><TR><TD>P(TK) </TD><TD> pSB1C3 </TD><TD> 1 </TD><TD> 47.8
</TD></TR><TR><TD>P(TK) </TD><TD> pSB1C3 </TD><TD> 2 </TD><TD> 64.4
</TD></TR><TR><TD>1ETF </TD><TD> pSB1C3 </TD><TD> 1 </TD><TD> 40.2
</TD></TR><TR><TD>1ETF </TD><TD> pSB1C3 </TD><TD> 2 </TD><TD> 31.6
</TD></TR><TR><TD>1PJN </TD><TD> pSB1C3 </TD><TD> 2 </TD><TD> 38.2
</TD></TR><TR><TD>1PJN </TD><TD> pSB1C3 </TD><TD> 3 </TD><TD> 40.9
</TD></TR><TR><TD>Protamine-1-Optimized </TD><TD> pSB1C3 </TD><TD> 1 </TD><TD> 40.8
</TD></TR><TR><TD>Protamine-1-Optimized </TD><TD> pSB1C3 </TD><TD> 2 </TD><TD> 48.6
</TD></TR><TR><TD>EGFP </TD><TD> pSB1C3 </TD><TD> 1 </TD><TD> 37.1
</TD></TR><TR><TD>EGFP </TD><TD> pSB1C3 </TD><TD> 2 </TD><TD> 54.0
</TD></TR></TBODY></TABLE><H3><SPAN class="mw-headline" id="PCR">PCR</SPAN></H3><P>A final BioBrick which we wished to create was a simple brick containing just the FNR promotor. To do this we would have to PCR this part out of the constructs we had designed using primers we had designed ourselves. As we were unsure of which restriction sites were available to us in the linear backbone we decided to perform two separate digestion with two different restriction enzymes. This also meant using two different primers. 
The PCR method was then used in conjunction with the construct DNA to extract the FNR promotor. To begin with the following solution was made:
</P><UL><LI> 32.7µL of Sterile Water
</LI><LI> 10µL of 5x Phusion Buffer
</LI><LI> 1µL of 10mM NTPs
</LI><LI> 2.5µL of the FW Primer
</LI><LI> 2.5µL of the RW Primer
</LI><LI>0.8µL of the Template DNA
</LI><LI> 0.5µL of Phusion DNA Polymerase
</LI></UL><P>For the FW primers we used a primer designed with an EcoRI restriction site attached to it. For the RW primers we used a primer with either a PstI or a SpeI restriction site embedded in it. 
For the template DNA we had chosen the 1ETF construct within a pUC57-Simple vector.
</P><P>The PCR reaction was then started and run as follows:
</P><UL><LI>Initial Denaturation: 30 seconds, 98°C
</LI><LI>The following steps were repeated 30 times:
</LI><LI> Denaturing: 10 seconds, 98°C
</LI><LI> Annealing: 15 seconds, 70deg;C
</LI><LI> Extension: 30 seconds, 72°C
</LI></UL><LI>Final Extension: 10 minutes; 72°C
</LI><P>After performing the PCR we added 1µL of DpnI to the solution for 5 minutes at 37°C with a heat inactivation step of 5 minutes at 80°C.
</P><H3><SPAN class="mw-headline" id="PCR_Puricfication">PCR Puricfication</SPAN></H3><P>For the inserts (FNR(SpeI), FNR(PstI)) a simple QIAquick PCR purification kit could be used to purify the digestion product as the cut out was smaller than 100 bases for each of the two vectors. The purification was carried out according to the following protocol:
</P><UL><LI>To the PCR product 5 volumes of PB buffer were added (In our case this was 250µL of PB buffer as we had 50µL of PRC product.)
</LI><LI>This solution was placed in a QIAquick spin column and spun for 60 seconds at 13000 rpm. The flow through could be discarded.
</LI><LI>Now 750µL of PE buffer could be added to the column before spinning it down once more for 60 seconds at 13000 rpm.
</LI><LI>Having discarded the flow through the column was spun for a further 60 seconds again at 13000 rpm to remove the last of the PE buffer.
</LI><LI>The column was then placed into a sealable eppendorf tube before adding 42µL of MilliQ water to the very centre membrane of the spin column and leaving it to stand for one minute.
</LI><LI>The column was now placed in the centrifuge and spun for 60 seconds at 13000 rpm producing a solution containing the desired DNA.
</LI></UL><P>The results of this PCR purification were as follows:
</P><TABLE class="table table-striped"><TBODY><TR><TH align="center"> Construct
</TH><TH>Concentration (ng/µL)
</TH></TR><TR><TD>FNR(PstI) </TD><TD> 18.7
</TD></TR><TR><TD>FNR(SpeI) </TD><TD> 25.8
</TD></TR></TBODY></TABLE><H3><SPAN class="mw-headline" id="Digestion_4">Digestion</SPAN></H3><P>Having amplified the FNR promoter we could now move on to digesting the ends of the PCR product. When doing this we would also digest the Backbone so that we could later ligate the promotor into the vector. We also needed to prepare two further samples. Previously we had not been successful in creating BioBricks for the 1G70 and the Poly(Lysine-Serine) construct. These were therefore being prepared once again and would also need digesting, which we would do now. Our Digestion solution was as follows:
</P><TABLE class="table table-striped"><TBODY><TR><TH align="center"> Construct
</TH><TH>Volume (µL)
</TH><TH>10x Cutsmart Buffer (µL)
</TH><TH>Enzyme (EcoRI-HF) (µL)
</TH><TH>Enzyme (PstI-HF) (µL)
</TH><TH>Enzyme (SpeI) (µL)
</TH><TH>Sterile Water (µL)
</TH></TR><TR><TD>pSB1C3 (80 ng) </TD><TD> 3.2 </TD><TD> 5 </TD><TD> 0.2 </TD><TD> 0.2 </TD><TD> - </TD><TD> 39.0
</TD></TR><TR><TD>pSB1C3 (140 ng) </TD><TD> 5.6 </TD><TD> 5 </TD><TD> 0.2 </TD><TD> - </TD><TD> 0.2 </TD><TD> 41.4
</TD></TR><TR><TD>FNR (PstI) </TD><TD> 26.7 </TD><TD> 5 </TD><TD> 0.2 </TD><TD> - </TD><TD> 18.9
</TD></TR><TR><TD>FNR (SpeI) </TD><TD> 19.4 </TD><TD> 5 </TD><TD> - </TD><TD> 0.2 </TD><TD> 25.2
</TD></TR><TR><TD>1G70 </TD><TD> 30.2 </TD><TD> 5 </TD><TD> 0.7 </TD><TD> 0.7 </TD><TD> - </TD><TD>  13.4
</TD></TR><TR><TD>Poly(Lysine-Serine) </TD><TD> 36.6 </TD><TD> 5 </TD><TD> 0.7 </TD><TD> 0.7 </TD><TD> - </TD><TD> 7.0
</TD></TR></TBODY></TABLE><P>Having prepared these Digestion solutions they were placed in the PCR apparatus and left to run for 60 minutes at 37°C before being placed in the 4°C refrigerator for storage. 
</P><H3><SPAN class="mw-headline" id="PCR_Purification_3">PCR Purification</SPAN></H3><P>Before ligation would be possible we would have to perform PCR purification on the digestion product of the FNR promotor. This was done as follows:
</P><UL><LI>To the PCR product 5 volumes of PB buffer were added (In our case this was 250µL of PB buffer as we had 50µL of PRC product.)
</LI><LI>This solution was placed in a QIAquick spin column and spun for 60 seconds at 13000 rpm. The flow through could be discarded.
</LI><LI>Now 750µL of PE buffer could be added to the column before spinning it down once more for 60 seconds at 13000 rpm.
</LI><LI>Having discarded the flow through the column was spun for a further 60 seconds again at 13000 rpm to remove the last of the PE buffer.
</LI><LI>The column was then placed into a sealable eppendorf tube before adding 42µL of MilliQ water to the very centre membrane of the spin column and leaving it to stand for one minute.
</LI><LI>The column was now placed in the centrifuge and spun for 60 seconds at 13000 rpm producing a solution containing the desired DNA.
</LI></UL><P>The results of the above purification have been given here:
</P><TABLE class="table table-striped"><TBODY><TR><TH align="center"> Construct
</TH><TH>Concentration (ng/µL)
</TH></TR><TR><TD>FNR (PstI) </TD><TD> 8.7
</TD></TR><TR><TD>FNR (SpeI) </TD><TD> 19.0
</TD></TR></TBODY></TABLE></DIV></DIV><DIV class="tue-timeline-middle"><DIV class="tue-timeline-leftbar"><DIV class="tue-timeline-node">12 September2013</DIV></DIV><DIV class="tue-timeline-content"><H3><SPAN class="mw-headline" id="Gel_Extraction_2">Gel Extraction</SPAN></H3><P>On September 11th two of our constructs (1G70 and Poly(Lysine-Serine)) had been digested once more so that we could perform a second attempt at the making of BioBricks for these parts. As the inserts which resulted from this digestion could not be separated from the vector using a standard PCR column gel extraction had to be performed:
</P><UL><LI>For both the constructs the digestion had been performed in two separate PCR tubes, each containing 50µL of PCR solution. These two solutions were mixed together to form a solution of 100µL and 20µL of 6x Loading Dye was added to each construct. 
</LI><LI>The entire 120µL were then pipetted into a 1.5% agarose gel which was made by mixing the following:
</LI><LI>1.8g of Agarose.
</LI><LI>120mL of TAE.
</LI><LI>12µL of Bromomethane (which was used instead of Cybersafe, which had run out).
</LI></UL><LI>After loading the sample the gel was run for 60 minutes at 100V.
</LI><P>After the gel had run we could then place the gel onto a blue light and cut out the desired bands for gel extraction using the following procedure:
</P><UL><LI>Upon placing the gel on the blue light we saw that there was very little DNA to be seen within the bands of the 1G70 construct. The bands were so faint that we did not attempt gel extraction here. The bands for the Poly(Lysine-Serine) construct appeared correct so these were cut out. 
</LI><LI>After cutting out the smallest band on the gel, the band was split in to two equal pieces, and both halves were placed into a 2mL eppendorf tube.
</LI><LI>The eppendorf tubes and the gel cut-outs were then weighed in preparation for the following steps.
</LI><LI>To each tube 3 volumes of QG buffer was added. A volume can be defined as the weight of the gel in mg transferred into µL. So  a gel weighing 400mg would have a volume of 400µL, so 1200µL of QG would be added. 
</LI><LI>The eppendorf tubes containing the gel cut-outs and the QG buffer were placed in an incubator set at 50°C for 10 minutes, whilst vortexing the samples at regular (every 2 minutes) intervals until the gel had completely dissolved. 
</LI><LI>To these tubes 1 volume of isopropanol was added.  
</LI><LI>Each tube was then brought over into a QIAquick column and placed in the centrifuge where it was spun down for 60 seconds at 13000 rpm. The flow through was then discarded. If the volume of the gel, QG buffer and Isopropanol was larger than 800µL then this step was performed twice in the same column. 
</LI><LI>Now 750µL of PE buffer was added to the column before centrifuging at 13000 rpm for 60 seconds. After discarding the flow through the same amount was added and once more and again centrifuged.
</LI><LI>Removing the flow through the column was then centrifuged again for 60 seconds at 13000 rpm removing the last of the PE buffer. 
</LI><LI>After bringing the column across into a sealable eppendorf tube 36µL of MilliQ water was pipetted onto the very centre membrane of the column which was then let to stand for 60 seconds. Subsequently the column was spun down at 13000 rpm for 60 seconds. The DNA is now in the flow through. 
</LI><LI>Remembering that we split the gel in half in the very first stage of this protocol, the two solutions containing the same DNA were combined so that a total volume of maximally 72µL of each insert was formed.
</LI></UL><P>The concentrations of DNA we aquired from this gel extraction is given below:
</P><TABLE class="table table-striped"><TBODY><TR><TH align="center"> Construct
</TH><TH>Concentration (ng/µL)
</TH></TR><TR><TD>Poly(Lysine-Serine) </TD><TD> 4.9
</TD></TR></TBODY></TABLE><H3><SPAN class="mw-headline" id="Re-digesting_1G70">Re-digesting  1G70</SPAN></H3><P>As the gel extraction had no been successful for the 1G70 construct we decided to re-digest this vector as soon as possible. The digestion solution was made as follows:
</P><TABLE class="table table-striped"><TBODY><TR><TH align="center"> Construct
</TH><TH>Volume (µL)
</TH><TH>10x Cutsmart Buffer (µL)
</TH><TH>Enzyme (µL)
</TH><TH>Sterile Water (µL)
</TH></TR><TR><TD>1G70 (5µg) </TD><TD> 20.5 </TD><TD> 5 </TD><TD> 0.7 </TD><TD> 23.1
</TD></TR><TR><TD>1G70 (2µg) </TD><TD> 8 </TD><TD> 5 </TD><TD> 0.7 </TD><TD> 35.6
</TD></TR></TBODY></TABLE><P>In the above table the enzymes used were EcoRI-HF and PstI-HF, and the volume mentioned is the volume of each enzyme added. 
</P><P>The above digestion solutions were subsequently placed in the PCR apparatus and run at 37°C of 60 minutes.
</P><H3><SPAN class="mw-headline" id="Gel_Extraction_3">Gel Extraction</SPAN></H3><P>A second gel was made today, identical to the one made this morning. Into this gel both of the digestion samples of 1G70 were pipetted, so that we had 1 lane containing 5µg of DNA and 1 lane containing 2µg of DNA. The gel was run for 60 minutes at 100V, again identical to the gel run earlier today. 
</P><P>We then moved on to the actual gel extraction again following the protocol we had followed this morning. We saw two separate bands appear in each of the columns which indicated that there could be a problem with the digestion of the 1G70 sample. As we were unsure of which band we wished to ligate it was decided to cut each band separately. To avoid confusion with the reader, we are labelling the bands as is seen in the understanding image:
</P><DIV class="row"><DIV class="tue-float tue-float-left span6"><DIV class="tue-caption"><SPAN class="tue-caption-figure"><SPAN class="tue-refid">1G70_Double_Bands</SPAN></SPAN> <SPAN class="tue-caption-text">Naming of different bands</SPAN></DIV></DIV></DIV>
Each of the four bands were then cut out of the gel and underwent the gel extraction protocol which has been described earlier today. The nanodrop results of these gel extractions is given below: 
<TABLE class="table table-striped"><TBODY><TR><TH align="center"> Construct
</TH><TH>Concentration (ng/µL)
</TH></TR><TR><TD>1G70 1.1 </TD><TD>  6.0
</TD></TR><TR><TD>1G70 1.2 </TD><TD> 5.0
</TD></TR><TR><TD>1G70 2.1 </TD><TD> 3.3
</TD></TR><TR><TD>1G70 2.2 </TD><TD> 4.7
</TD></TR></TBODY></TABLE><P>The samples were then stored in the -20°C freezer and left until they were needed again. 
</P></DIV></DIV><DIV class="tue-timeline-middle"><DIV class="tue-timeline-leftbar"><DIV class="tue-timeline-node">13 September2013</DIV></DIV><DIV class="tue-timeline-content"><H3><SPAN class="mw-headline" id="Ligation_4">Ligation</SPAN></H3><P>On September 11th we had digested and purified the FNR promoter in two different ways, we had also digested the 1G70 and Poly(Lysine-Serine) constructs which had been purified on September 12th. Now we could take all of these separate parts and ligate them into the pSB1C3 vector backbone. You may recall we had two performed two separate digestions on the FNR promoter, as we were not sure which restriction enzymes were available on the backbone. We had also arrived at two different purified versions of the 1G70 construct as this had produced two bands on the agarose gel. Of the Poly(Lysine-Serine) construct we had only one sample. 
For the ligation of these five samples the following ligation solutions were prepared:
</P><TABLE class="table table-striped"><TBODY><TR><TH align="center"> Construct
</TH><TH>5x Molar Equality of Insert DNA (µL)
</TH><TH>pSB1C3 BackBone (µL)
</TH><TH>10x Buffer (µL)
</TH><TH> T4 DNA Ligase (µL)
</TH><TH>Sterile Water (µL)
</TH></TR><TR><TD>FNR (PstI Digested) </TD><TD> 1.0 </TD><TD> 7.5 </TD><TD> 5 </TD><TD> 1.0 </TD><TD> 25.5
</TD></TR><TR><TD>FNR (SpeI Digested) </TD><TD> 1.0 </TD><TD> 30 </TD><TD> 5 </TD><TD> 1.0 </TD><TD> 13.0
</TD></TR><TR><TD>1G70 (1.1) </TD><TD> 6.0 </TD><TD> 7.5 </TD><TD> 5 </TD><TD> 1.0 </TD><TD> 30.5
</TD></TR><TR><TD>1G70 (1.2) </TD><TD> 7.0 </TD><TD> 7.5 </TD><TD> 5 </TD><TD> 1.0 </TD><TD> 29.5
</TD></TR><TR><TD>Poly(Lysine-Serine) </TD><TD> 8.0 </TD><TD> 5.0 </TD><TD> 5 </TD><TD> 1.0 </TD><TD> 28.5
</TD></TR></TBODY></TABLE><P>The ligation of the above solutions was then completed by placing the PCR tubes in the PCR apparatus and running the machine for 120minutes at 25°C and subsequently for 10 minutes at 80°C to inactivate the enzymes. 
</P><P>These ligated samples were then plaged in the 4°C  refrigerator and left there for storage. 
</P></DIV></DIV></DIV></DIV><DIV class="tab-pane" id="Week9"><DIV><DIV class="tue-timeline-top"><DIV class="tue-timeline-title"><H2>Week 9</H2></DIV></DIV><DIV class="tue-timeline-middle"><DIV class="tue-timeline-leftbar"><DIV class="tue-timeline-node">16 September2013</DIV></DIV><DIV class="tue-timeline-content"><H3><SPAN class="mw-headline" id="Transformation_and_Plating">Transformation and Plating</SPAN></H3><P>On September 13th the ligation of 5 different BioBrick constructs had been performed. This was done for two FNR promoter constructs, both digested using a different restriction enzyme, for two different bands of a purified 1G70 construct and for a digested Poly(Lysine-Serine) construct. These samples could now be plated so that we could analyse the colonies and judge whether or not the ligations had been performed successfully:
</P><UL><LI>Each of the following steps was performed in the vicinity of a blue bunsen flame for increased sterility.
</LI><LI>The entire ligation solution of each of the constructs was pipetted into 20µL of NB bacteria.
</LI><LI>The bacteria were then heat shocked for 30 seconds at 42°C before returning to ice for 2 minutes. 
</LI><LI>After having been left on ice for 2 minutes 80µL of SOC solution was added to each sample and the samples were incubated at 37°C for 60 minutes. 
</LI><LI>After incubation the entire solution was pipetted onto a clean agar plate (with chloramphenicol antibiotics).
</LI><LI>This solution was then spread out using a sterile spreader and the plate was left in the 37°C incubator for growth overnight. 
</LI></UL><P>This concluded the work for today. 
</P></DIV></DIV><DIV class="tue-timeline-middle"><DIV class="tue-timeline-leftbar"><DIV class="tue-timeline-node">17 September2013</DIV></DIV><DIV class="tue-timeline-content"><H3><SPAN class="mw-headline" id="Colony_Picking_and_PCR_5">Colony Picking and PCR</SPAN></H3><P>On September 16th we had plated a number of samples (FNR(PstI), FNR(SpeI), 1G70(1), 1G70(2) and Poly(Lysine-Serine)) onto chloramphenicol agar plates. These plates had been allowed to incubate at 37°C overnight so that we would be able to perform a colony picking and PCR today. This would allow us to check the results of the ligation and make a final choice as to which colonies we would like to continue with to send in to the iGEM registry. 
The colony picking and PCR were performed as follows:
</P><UL><LI>In the vicinity of a blue bunsen flame four colonies were picked from each of the agar plates and dissolved in 15µL of sterile water. (No colonies were seen on the agar plate which held the SpeI digested FNR sample, so we discarded this sample from this point onwards.)
</LI><LI>Of these solutions we could then perform colony PCR. This was done by preparing the following sample for each of the solutions:
</LI></UL><TABLE class="table table-striped"><TBODY><TR><TH align="center"> Substance
</TH><TH> Volume (µL)
</TH></TR><TR><TD>Sterile Water </TD><TD> 9.0
</TD></TR><TR><TD>Kappa2G </TD><TD> 12.5
</TD></TR><TR><TD>Fwd Primer </TD><TD> 1.25
</TD></TR><TR><TD>Rwd Primer </TD><TD> 1.25
</TD></TR><TR><TD>Template DNA </TD><TD> 1.0
</TD></TR></TBODY></TABLE><UL><LI>The primers were those as provided by the iGEM help pages for the amplification of our ligation insert.
</LI><LI>Having prepared the above solutions we placed the eppendorf tubes in the PCR apparatus and ran the following protocol:
</LI><LI> Initial Denaturation: 95°C for 3 minutes.
</LI><LI> The folllowing steps were repeated 30 times:
</LI><LI>Denaturation: 95°C for 15 seconds.
</LI><LI>Annealing: 58°C for 15 seconds.
</LI><LI>Extension: 72°C for 45 seconds.
</LI></UL><LI>Final Extension: 72°C for 10 minutes.
</LI><P>To check the ligation and PCR we would now have to run the PCR products on a 1% agarose gel which was made by mixing the following:
</P><UL><LI>1.2g Agarose.
</LI><LI>120mL TAE Buffer.
</LI><LI>12µL of SyberSafe.
</LI></UL><P>The samples we loaded were made by adding 4µL of 6x Loading dye to each of the PCR solutions of which then 12µL was loaded onto the gel. 
The gel was subsequently run for 60 minutes at 100V giving the understanding result:
</P><DIV class="row"><DIV class="tue-float tue-float-left span6"><DIV class="tue-caption"><SPAN class="tue-caption-figure"><SPAN class="tue-refid">1Colony_PCR_17_09_13</SPAN></SPAN> <SPAN class="tue-caption-text">Number of the columns from left to right: 1: 1Kbp ladder, 2: 100bp ladder, 3-6: FNR(PstI), 7-10: 1G70(1), 11-14: 1G70(2), 15-18: Poly(Lysine-Serine), 19: 100bp Ladder, 20: 1Kbp Ladder</SPAN></DIV></DIV></DIV><P>Looking at the gel we can conclude that all ligations were successful. We did however have to decide which 1G70 sample we would like to continue with. Based on the height of the bands we could conclude that the 1G70(1) samples came closest to the expected length so we would be discarding the 1G70(2) samples.
</P><H3><SPAN class="mw-headline" id="Small_Cultures_3">Small Cultures</SPAN></H3><P>Having confirmed which ligagtions had been successful we could now amplify those samples by creating a number of small cultures. For each of the three samples (FNR(Pst1), 1G70(1), Poly(Lysine-Serine) 2 colonies would be chosen to culture:
</P><UL><LI>FNR(Pst1):Colonies 1 and 2
</LI><LI>1G70(1): Colonies 3 and 4
</LI><LI>Poly(Lysine-Serine): Colonies 3 and 4
</LI></UL><P>The making of the small cultures was then performed as follows:
</P><UL><LI>All steps were performed near a blue bunsen flame.
</LI><LI>For each sample a small culture tube was filled with 8mL of LB medium.
</LI><LI>We then added 8µL of Chloramphenicol antibiotics to each culture tube.
</LI><LI>Finally 5µL of the correct colony picking solutions prepared earlier today was pipetted into one of the culture tubes.
</LI></UL><P>After preparing the cultures we could place them all in the 37°C incubator so they could grow and culture overnight, effectively multiplying the amount of DNA we had.
</P></DIV></DIV><DIV class="tue-timeline-middle"><DIV class="tue-timeline-leftbar"><DIV class="tue-timeline-node">18 September2013</DIV></DIV><DIV class="tue-timeline-content"><H3><SPAN class="mw-headline" id="Miniprep_2">Miniprep</SPAN></H3><P>The small cultures which were set in on Spetmeber 17th, now had to be miniprepped. This was done in the following manner:
</P><UL><LI> The culture tubes were centrifuged for 10 minutes at 3700g. 
</LI><LI> The supernatant above the pellets was discarded and 250µL of P1 buffer was added, to resuspend the pellets.
</LI><LI> The culture tubes had to be mixed so that the pellets are dissolved into the buffer. This mixture was then pipetted into a 1.5 mL eppendorf tube. 
</LI><LI> To each solution 250µL of P2 buffer was added, turning the solution blue. Inverting the tube several times ensured homogenisation of the solution. Within 5 minutes 350µL N3 buffer was added and the solution became clear by inverting the tube several times. 
</LI><LI> Then the tubes were centrifuged for 10 minutes at 13000rpm, again forming pellets. The supernatant was placed in the QIAprep spin column in a  number of steps.
</LI><LI> Then the columns were spun for 1 minute at 13400 rpm to bind the DNA to the column. 
</LI><LI> Then the flow through was discarded and 750 uL of PE buffer was added to wash the column. 
</LI><LI> Again the columns were spun for 1 minute at 13400 rpm, and the flow through was discarded. 
</LI><LI> Then a dry spinning was performed in the same manner. 
</LI><LI> Then the column was eluded with 50 uL of MilliQ and was spun for 1 minute at 13400 rpm. 
</LI></UL><P>The following concentrations were then measured by performing a nanodrop measurement. 
</P><TABLE class="table table-striped"><TBODY><TR><TH align="center"> Construct
</TH><TH> Concentration (ng/µL)
</TH></TR><TR><TD>pSB1C3-FNR 1 </TD><TD> 57.0
</TD></TR><TR><TD>pSB1C3-FNR 2 </TD><TD> 47.2
</TD></TR><TR><TD>pSB1C3-1G70 3 </TD><TD> 82.4
</TD></TR><TR><TD>pSB1C3-1G70 4 </TD><TD> 93.9
</TD></TR><TR><TD>pSB1C3-P(KS) 3 </TD><TD> 47.5
</TD></TR><TR><TD>pSB1C3-P(KS) 4 </TD><TD> 41.7
</TD></TR></TBODY></TABLE><H3><SPAN class="mw-headline" id="Preparing_the_Biobricks_for_shipment_to_Boston">Preparing the Biobricks for shipment to Boston </SPAN></H3><P>Before the samples could be shipped to Boston, we had to prepare the DNA. This meant that the Biobricks had to be in a 10µL solution of 25 ng/µL  in TE. 
A 10X TE stock was made by dissolving 1.21g of Trizma-base and 372mg of EDTA in 700mL of demineralized water. Then the pH was set to 8.0 by adding 5M HCl.  The total volume was then filled to 100mL. For each construct, the sample with the highest concentration after miniprep was taken to deliver the DNA for the Biobrick shipping. The samples were numbered from 1 to 10 in this order.
</P><TABLE class="table table-striped"><TBODY><TR><TH align="center"> Construct
</TH><TH> Concentration (ng/µL)
</TH><TH>µL needed for 250 ng
</TH><TH>10X TE Buffer (µL)
</TH><TH>Sterile water (µL)
</TH></TR><TR><TD>FNR </TD><TD> 57.0 </TD><TD> 4.4 </TD><TD> 1 </TD><TD> 4.6
</TD></TR><TR><TD>Protamine </TD><TD> 48.6 </TD><TD> 5.1 </TD><TD> 1 </TD><TD> 3.9
</TD></TR><TR><TD>1ETF </TD><TD> 40.2 </TD><TD> 6.2 </TD><TD> 1 </TD><TD> 2.8
</TD></TR><TR><TD>1PJN1 </TD><TD> 40.9 </TD><TD> 6.1 </TD><TD> 1 </TD><TD> 2.9
</TD></TR><TR><TD>1G70 </TD><TD> 93.9 </TD><TD> 2.7 </TD><TD> 1 </TD><TD> 6.3
</TD></TR><TR><TD>EGFP </TD><TD> 54.0 </TD><TD> 4.6 </TD><TD> 1 </TD><TD> 4.4
</TD></TR><TR><TD>P(RG) </TD><TD> 96.1 </TD><TD> 2.6 </TD><TD> 1 </TD><TD> 6.4
</TD></TR><TR><TD>P(RS) </TD><TD> 85.4 </TD><TD> 2.9 </TD><TD> 1 </TD><TD> 6.1
</TD></TR><TR><TD>P(TK) </TD><TD> 64.4 </TD><TD> 3.9 </TD><TD> 1 </TD><TD> 5.1
</TD></TR><TR><TD>P(KS) </TD><TD> 47.5 </TD><TD> 5.3 </TD><TD> 1 </TD><TD> 3.7
</TD></TR></TBODY></TABLE><H3><SPAN class="mw-headline" id="Making_of_samples_for_sequencing">Making of samples for sequencing</SPAN></H3><P>The Biobricks were also sent for sequencing. This time sequencing will be done by Baseclear. Therefore we had to put 20 µL of our DNA samples into a special tube. We also had to deliver them our primers for sequencing in different tubes. 3 µL of each primer (pSB1C3 colPCR FW &amp; RW, 100 µM) were taken and transferred to the special tube with 27 µL of MilliQ. 
</P></DIV></DIV></DIV></DIV><DIV class="tab-pane" id="Week10"><DIV><DIV class="tue-timeline-top"><DIV class="tue-timeline-title"><H2>Week 10</H2></DIV></DIV><DIV class="tue-timeline-middle"><DIV class="tue-timeline-leftbar"><DIV class="tue-timeline-node">23 September2013</DIV></DIV><DIV class="tue-timeline-content"><H3><SPAN class="mw-headline" id="Prepare_BioBricks_for_Sending">Prepare BioBricks for Sending</SPAN></H3><P>Due to an unfortunate clerical error, in which we played no part, we were forced to remake our BioBricks for sending to the iGEM registry. For this we would have to send the sample bricks as we had attempted to send previously. This was performed in the same way as it had been on September 18th:
</P><UL><LI>To 1µL of 10x TE stock solution we added 250ng/µL of each DNA construct.
</LI><LI>The solution was then filled up to 10µL with sterile water. 
</LI></UL><P>These samples, prepared in PCR tubes were subsequently labelled and sent off to the iGEM headquarters. 
</P><H3><SPAN class="mw-headline" id="Culturing_Bacteria">Culturing Bacteria</SPAN></H3><P>To make one last attempt at anaerobic expression of our proteins we had ordered a new strain of bacteria: MG 1655 E.coli. This strain was delivered to us via the mail. Upon receiving these bacteria we needed to create a small culture of these cells to increase their number.
This was done in the following manner:
</P><UL><LI>2 culture tubes were filled with 8mL of LB medium.
</LI><LI>To each culture tube we added a picking of the bacteria.
</LI><LI>The bacteria were then placed in the 37°C incubator and left there overnight to grow.
</LI></UL></DIV></DIV><DIV class="tue-timeline-middle"><DIV class="tue-timeline-leftbar"><DIV class="tue-timeline-node">24 September2013</DIV></DIV><DIV class="tue-timeline-content"><H3><SPAN class="mw-headline" id="Prepare_Bacteria_for_Electroporation">Prepare Bacteria for Electroporation</SPAN></H3><P>First glycerol stocks were made out of the small cultures that were set in yesterday. 
this was done by adding 700 µL of the culture sample to an eppendorf tube near a blue Bunsen flame. Then 300 µL of 50% glycerol was added and mixed in the tube again near the blue flame. Then the samples were snap frozen and placed in the -80°C fridge. 
</P><P>Then we were able to create the electrocompetent cells. 
</P><UL><LI>This was done by adding 11 mL of LB medium to new small  culture tubes and 1 mL sample of the same culture tube as were the glycerol stocks were made from
</LI><LI>The culture tubes were set in an incubator at 37°C to obtain an optical density between 0.3 and 0.4, since cells were needed in the mid-log phase 
</LI><LI>The optical densities measured were: 0.32 and 0.34 after aproximmately 50 minutes
</LI><LI>Then the culture tubes were chilled on ice for about 30 minutes 
</LI></UL><P>From now on, the cells and everything the cells came in contact with were held at 4°C. 
</P><UL><LI>The cells were spun down for 15 minutes at 2000g (3500 rpm) at 4°C
</LI><LI>Supernatant was removed and the cells were gently resuspended with 3mL ice-cold sterile water
</LI><LI>The cells were spun down again for 15 minutes at 2000g (3500 rpm) at 4°C
</LI><LI>Again the supernatant was removed and the cells were gently resuspended with 3 mL ice-cold sterile water
</LI><LI>The cells were spun down for 15 minutes at 2000g (3500 rpm) at 4°C
</LI><LI>The supernatant was removed and the cells were gently resuspended in 1 mL cold 50% glycerol
</LI><LI>The cells were again spun down for 15 minutes at 2000g (3500 rpm) at 4°C
</LI><LI>The supernatant was removed gently by using a pipet. The cells were resuspended with 60 µL 50% glycerol (which resulted in 120 µL of bacteria in total)
</LI><LI>The solutions were divided over 3 eppendorf tubes (40 µL each) and 2 tubes were snap frozen and stored in the -80°C fridge.
</LI></UL><H3><SPAN class="mw-headline" id="Electroporation">Electroporation</SPAN></H3><P>The electroporation was carried out using the Bio-Rad MicroPulser. 40 µL of bacterial (MG1655) solution was taken and 1 µL (pBR322-fnr-EGFP) of DNA was added. This solution was then transfered to a 0.1 cm electroporation  cuvette, which had been kept cool at 4 °C. The cuvette was put into the electroporation chamber of the MicroPulser. The machine was set to a standard E.Coli program for 0.1 cm cuvettes (Ec1). The pulse was given (1.78 kV, τ = 6 ms). After the pulse was delivered, SOC was immediately added to help the cells recover. The bacteria were then transfered to an Eppendorf tube and incubated at 37 °C for 1 hour. Afterwards the samples were plated on ampicilin plates and incubated overnight.
</P></DIV></DIV><DIV class="tue-timeline-middle"><DIV class="tue-timeline-leftbar"><DIV class="tue-timeline-node">25 September2013</DIV></DIV><DIV class="tue-timeline-content"><H3><SPAN class="mw-headline" id="Creating_small_samples">Creating small samples</SPAN></H3><P>On September 24th we had transformed our EGFP construct, in a pBR322 vector, into our newest bacteria strain, MG1655 E.coli. These bacteria had been left to grow overnight and were now ready to be cultured. This would subsequently allow us to perform anaerobic expression, making use of the smaller cultures. 
The small cultures were created in the following manner, in the vicinity of a blue bunsen flame for sterilities sake:
</P><UL><LI>First off 8 culture tubes were filled with 8mL of LB medium.
</LI><LI>Next, 8µL of ampicillin antibodies were added to each culture tubes.
</LI><LI>We then picked 4 separate colonies from the afore mentioned agar plate. Each of these colonies was dissolved in 15µL of sterile water.
</LI><LI>7µL of the dissolved picking solution was then pipetted into each of the culture tubes.
</LI><LI>The culture tubes were left in a 37°C incubator for overnight growth.
</LI></UL><P>This conclude the preparation of the bacterial samples.
</P><H3><SPAN class="mw-headline" id="Preparing_LB_medium">Preparing LB medium</SPAN></H3><P>To allow ourselves to preform an anaerobic culture we would need to once again make use of our BioFermentor. When doing so we would create a colony of 4L which means we needed to prepare the main vessel and fill it with 4L of LB medium. 
The LB medium was made by mixing the following:
</P><UL><LI>40g Peptones.
</LI><LI>40g NaCl.
</LI><LI>20g Yeast Extract.
</LI><LI>4L of demineralised water.
</LI></UL><P>After having prepared the LB medium it was poured into the vessel and the entire vessel and LB solution was placed in the autoclave for sterilization. 
</P><P>Click <A href="http://www.youtube.com/watch?v=eV1l-qkWYiM" class="external text" rel="nofollow">HERE </A> to view a video explaining the preparational procedures performed with the BioFermentor.
</P></DIV></DIV><DIV class="tue-timeline-middle"><DIV class="tue-timeline-leftbar"><DIV class="tue-timeline-node">26 September2013</DIV></DIV><DIV class="tue-timeline-content"><H3><SPAN class="mw-headline" id="Anaerobic_protein_expression">Anaerobic protein expression</SPAN></H3><P>On the 25th of September we prepared 8 small cultures tubes with MG1655 E.coli with therein pBR332 ligated EGFP and a large 4L culture medium. Having these we could now perform the anaerobic expression. One of the culture tubes was parafilmed and put back into an incubator to see if this would result in some anaerobic effect. The following table lists chronologically describes all the steps that were taken during the anaerobic expression:
</P><TABLE class="table table-striped"><TBODY><TR><TH align="center"> Time
</TH><TH> Procedure
</TH></TR><TR><TD>08:45 </TD><TD> Culture vessel was flooded with air. (10L/min)
</TD></TR><TR><TD>09:15 </TD><TD> Ampicillin was added to the LB medium inside the vessel (4mL of 100µg/mL).
</TD></TR><TR><TD>09:30 </TD><TD> The 7 EGFP culture tubes were inserted into the vessel using a syringe. The Oxygen level was calibrated to 100%.  And the pH calibration probe was set at pH 7.0.
</TD></TR><TR><TD>09:45 </TD><TD> A sample was taken to measure the optical density, which was : 0.102. (pH = 7.09, dO<SUB>2</SUB> = 96.0, Temp = 35.5°C)
</TD></TR><TR><TD>10:13 </TD><TD> A sample was taken to measure the optical density, which was : 0.121. (pH = 7.06, dO<SUB>2</SUB> = 98.5, Temp = 36.9°C)
</TD></TR><TR><TD>10:41 </TD><TD> A sample was taken to measure the optical density, which was : 0.250. (pH = 7.01, dO<SUB>2</SUB> = 88.7, Temp = 37.0°C) The agitation motor was slowed from 300rpm to 225 rpm for foam control.
</TD></TR><TR><TD>11:30 </TD><TD> A sample was taken to measure the optical density, which was : 0.765. (pH = 6,.86, dO<SUB>2</SUB> = 65.8, Temp = 37.0°C)
</TD></TR><TR><TD>11:32 </TD><TD> The vessel was flooded with 100% Nitrogen. And the agitation motor was slowed to 150 rpm because a lot of foam arose.
</TD></TR><TR><TD>11:35 </TD><TD> A sample labeled as sample 1 was taken for later analysis. (pH = 6.86, dO<SUB>2</SUB> = 2.2, Temp = 37.0°C)
</TD></TR><TR><TD>11:42 </TD><TD> The excessive growth of foam had stopped and the agitation motor was accelerated to 250 rpm again.
</TD></TR><TR><TD>12:45 </TD><TD> A sample labeled as sample 2 was taken but at the end of the day it was decided to not further analyse this sample because of practical reasons. (pH = 7, dO<SUB>2</SUB> = 0, Temp = 37.0°C)
</TD></TR><TR><TD>13:50 </TD><TD> A sample labeled as sample 3 was taken for later analysis. (pH = 6.33, dO<SUB>2</SUB> = 0.3, Temp = 37.0°C)
</TD></TR><TR><TD>14:50 </TD><TD> A sample labeled as sample 4 was taken for later analysis. (pH = 6.23, dO<SUB>2</SUB> = 0, Temp = 37.0°C)
</TD></TR><TR><TD>15:50 </TD><TD> A sample labeled as sample 5 was taken for later analysis. (pH = 6.26, dO<SUB>2</SUB> = 0, Temp = 37.0°C)
</TD></TR><TR><TD>16:50 </TD><TD> A sample labeled as sample 6 was taken for later analysis. (pH = 6.31, dO<SUB>2</SUB> = 0.2, Temp = 37.0°C)
</TD></TR><TR><TD>17:50 </TD><TD> A sample labeled as sample 7 was taken for later analysis. (pH = 6.35, dO<SUB>2</SUB> = 0, Temp = 37.0°C)
</TD></TR></TBODY></TABLE></DIV></DIV></DIV></DIV><DIV class="tab-pane" id="Week11"><DIV><DIV class="tue-timeline-top"><DIV class="tue-timeline-title"><H2>Week 11</H2></DIV></DIV><DIV class="tue-timeline-middle"><DIV class="tue-timeline-leftbar"><DIV class="tue-timeline-node">01 October2013</DIV></DIV><DIV class="tue-timeline-content"><H3><SPAN class="mw-headline" id="Protein_purification_3"> Protein purification </SPAN></H3><P>The XL1-Blue samples taken at the end of August from the biofermentor during anaerobic expression were now purified. 
The bugbuster samples were purified by a nickel spin column. 
</P><UL><LI>The solution was mixed and spun for 30 seconds at 100g. 
</LI><LI>Flow through was discarded and 600 µL wash buffer was added
</LI><LI>Solution was spun for 30 seconds at 100g. 
</LI><LI>Flow through was discarded and 600 µL of the sample was added and spun down for 30 seconds at 100g.   (this step was repeated until the bugbuster solutions were all on the column)
</LI><LI>Flow through was discarded and  600 µL wash buffer was added and spun down for 30 seconds at 100g. 
</LI><LI>Flow through was discarded and then 300 µL elution buffer was added and spun down for 30 seconds at 100g
</LI><LI>The flow through does now contain the purified proteins. 
</LI></UL><P>After purification the samples were loaded on a well for fluorescence analysis, which resulted in the correct EGFP emission spectra.
</P><H3><SPAN class="mw-headline" id="SDS_analysis_4"> SDS analysis </SPAN></H3><P>5 µL of each purified sample was taken (also three samples that were purified yesterday MG1655(1, 8 and 9))and mixed with 5 µL loading dye mastermix (250 µL 2X SDS dye and 0.5 µL DTT) and denaturated in 5 minutes at 95*C. 
These samples were loaded on the SDS gel that was made before (12% SDS gel). 
The SDS gel was made as follows:
</P><UL><LI>10 ml 12% gel
</LI><LI>3.3 ml water
</LI><LI>4 ml 30% acrylamide
</LI><LI>2.5 ml 1.5M TRIS
</LI><LI>0.1 ml 10% SDS
</LI><LI>0.1 ml 10% Ammonium Persulfate
</LI><LI>4µL TEMED
</LI></UL><P>The gel had ran for 15 minutes at 90V and 40 minutes at 150V. After running, the gel was washed three times for 5 minutes in fresh demineralized water and thereafter stained with Coomassie dye. The gel was stained overnight. 
</P><DIV class="row"><DIV class="tue-float tue-float-left span6"><DIV class="tue-caption"><SPAN class="tue-caption-figure"><SPAN class="tue-refid">{{{id}}}</SPAN></SPAN> <SPAN class="tue-caption-text">12% SDS gel with the bugbustered samples of the Mg1655 anaerobic expressions of EGFP (1-9 in the Biofermentor, 10&amp;11 in the incubator).</SPAN></DIV></DIV></DIV></DIV></DIV><DIV class="tue-timeline-middle"><DIV class="tue-timeline-leftbar"><DIV class="tue-timeline-node">2 October2013</DIV></DIV><DIV class="tue-timeline-content"><H3><SPAN class="mw-headline" id="Protein_purification_4"> Protein purification </SPAN></H3><P>The samples taken on Thursday 26th were taken from the biofermentor during anaerobic expression. They were spun down and the pellets were snap frozen. 
Now two other small culture tubes (that were parafilmed and stay at 37°C) had to be spun down. 
This was done at 10000g for 10 minutes. The supernatant was poured off. 
400 µL Bugbuster and 0.4 µL Benzonase was added to all 10 pellets. All samples were placed at a tabletop shaker until the pellets were completely dissolved. 
Then the samples were spun down at 16000g for 20 minutes.  The supernatant was poured into a 15 mL falcon tube, were the pellet was no longer needed.
</P><H3><SPAN class="mw-headline" id="Fluorescence_analysis"> Fluorescence analysis </SPAN></H3><P>From the bugbuster samples 200 µL was taken and placed in a well on a wellplate. 
The samples were measured in the same order as the samples were taken. The excitation and emission spectra were based on EGFP. There was a shift from the emission spectra to higher wavelengths then expected (530 instead of 510) and there was no specific peak. 
Therefore the bugbuster samples were purified by a nickel spin column. 
</P><UL><LI>The solution was mixed and spun for 30 seconds at 100g. 
</LI><LI>Flow through was discarded and 600 µL wash buffer was added
</LI><LI>Solution was spun for 30 seconds at 100g. 
</LI><LI>Flow through was discarded and 600 µL of the sample was added and spun down for 30 seconds at 100g.   (this step was repeated until the bugbuster solutions were all on the column)
</LI><LI>Flow through was discarded and  600 µL wash buffer was added and spun down for 30 seconds at 100g. 
</LI><LI>Flow through was discarded and then 300 µL elution buffer was added and spun down for 30 seconds at 100g
</LI><LI>The flow through does now contain the purified proteins. 
</LI></UL><P>After purification, the same procedure was performed, which resulted in the correct emission spectra. 
</P><H3><SPAN class="mw-headline" id="SDS_analysis_5"> SDS analysis </SPAN></H3><P>5 µL of each sample was taken and mixed with 5 µL loading dye mastermix (250 µL 2X SDS dye and 0.5 µL DTT) and denaturated in 5 minutes at 95*C. 
These samples were loaded on the SDS gel that was made before (12% SDS gel). 
The SDS gel was made as follows:
</P><UL><LI>10 ml 12% gel
</LI><LI>3.3 ml water
</LI><LI>4 ml 30% acrylamide
</LI><LI>2.5 ml 1.5M TRIS
</LI><LI>0.1 ml 10% SDS
</LI><LI>0.1 ml 10% Ammonium Persulfate
</LI><LI>4µL TEMED
</LI></UL><DIV class="row"><DIV class="tue-float tue-float-left span6"><DIV class="tue-caption"><SPAN class="tue-caption-figure"><SPAN class="tue-refid">{{{id}}}</SPAN></SPAN> <SPAN class="tue-caption-text">12% SDS Gel of the purified samples of anaerobic protein expression in XL1-Blue and MG1655</SPAN></DIV></DIV></DIV></DIV></DIV></DIV></DIV><DIV class="tab-pane" id="Week12"><DIV><DIV class="tue-timeline-top"><DIV class="tue-timeline-title"><H2>Week 12</H2></DIV></DIV><DIV class="tue-timeline-middle"><DIV class="tue-timeline-leftbar"><DIV class="tue-timeline-node">16 October2013</DIV></DIV><DIV class="tue-timeline-content"><H3><SPAN class="mw-headline" id="Preparing_the_anaerobic_vessel">Preparing the anaerobic vessel</SPAN></H3><P>To allow the team to produce some extra results in the transitional period between the regional and world jambories a number different experiments were to be performed. The first of these would be the repeating of the anaerobic growth of EGFP whilst keeping track of the optical density which would help us determine in what amounts the EGFP was being expressed. 
</P><P>Before initiating expression the vessel had to be filled with LB medium and autoclaved:
</P><UL><LI>The vessel was filled with 4L of LB medium which was prepared as follows:
</LI><LI>40g Peptones.
</LI><LI>40g NaCl.
</LI><LI>20g Yeast Extract.
</LI><LI>4L Demineralised Water.
</LI></UL><LI>After mixing the above into a clear solution the vessel was autoclaved overnight leaving the solution sterile for expression the next day.
</LI><P>We would also need to ensure that we could innocculate the vessel and therefore had to grow a number of small EGFP cultures:
</P><UL><LI>8 small culture tubes were each filled with:
</LI><LI>8mL LB medium.
</LI><LI>8µL Ampicillin Antibodies.
</LI><LI>An EGFP colony picking where EGFP was ligated into the pBR322 vector therefore containing the FNR promoter, within the XL-1-Blue strain. (We had this precise strain on an agar plate in the lab, left over from previous work performed in the summer.)
</LI></UL><P>As a control we would also make use of an aerobic growth to check whether no EGFP was being formed there. For this we would be able to innoculate the medium using one of the small cultures prepared above, but would also need to prepare and autoclave 400mL of LB medium, made using the same recipe as above, with all amounts decreased by factor 10. 
</P></DIV></DIV><DIV class="tue-timeline-middle"><DIV class="tue-timeline-leftbar"><DIV class="tue-timeline-node">17 October2013</DIV></DIV><DIV class="tue-timeline-content"><H3><SPAN class="mw-headline" id="Anaerobic_Expression_4">Anaerobic Expression</SPAN></H3><P>On October 17th the anaerobic vessel had been cleaned and filled with 4L of LB medium. On the same day 8 small cultures had been allowed to grow overnight in a 37°C incubator. We had also prepared a 400mL vessel of LB medium to allow for an aerobic control experiment. 
Before adding the small cultures to the LB medium the appropriate ampicillin antibodies were added to the medium. To the 4L vessel 4mL of ampicillin was added. To the 400mL medium 400µL of ampicillin was added. 
Hereafter 1 of the 8 small cultures was placed poured into the 400mL vessel which was subsequently placed in a 37°C incubator in contact with oxygen. 
The remaining 7 small culture tubes were then injected into the anaerobic vessel so that they could grow anaerobically. We would follow the growth of the bacterial cells in the fermentor and in the aerobic control by taking measurements of the optical density. We would also take samples from the anaerobic solution from the point at which we 
removed the oxygen from the vessel upto the point where we ended the expression. 
</P><P>Before removing oxygen from the vessel we would allow the bacteria to grow to an optical density of around 0.600. At this point in time the oxygen inlet would be closed and the vessel would be continually flooded with nitrogen. 
The below table shows the vessel environment over time:
</P><TABLE class="table table-striped"><TBODY><TR><TH align="center"> Time
</TH><TH> Procedure
</TH></TR><TR><TD>10:08 </TD><TD> Final small culture injected: (O.D.= 0.047, dO<SUB>2</SUB> = 97.8, Temp = 36.8°C, pH = 7.03)
</TD></TR><TR><TD>11:20 </TD><TD> Optical Density measurement: (O.D.= 0.136, dO<SUB>2</SUB> = 85.5, Temp = 37.0°C, pH = 6.98)
</TD></TR><TR><TD>12:15 </TD><TD> Optical Density measurement: (O.D.= 0.359, dO<SUB>2</SUB> = 55.4, Temp = 37.0°C, pH = 6.86)
</TD></TR><TR><TD>13:00 </TD><TD> Optical Density measurement: (O.D.= 0.737, dO<SUB>2</SUB> = 40.0, Temp = 37.0°C, pH = 6.83)
</TD></TR><TR><TD>13:00 </TD><TD> Oxygen removed, Nitrogen added. Sample taken: (O.D.= 0.737, dO<SUB>2</SUB> = 13.4, Temp = 37.0°C, pH = 6.83)
</TD></TR><TR><TD>14:05 </TD><TD> Sample taken: (O.D.= 0.920, dO<SUB>2</SUB> = -0.2, Temp = 37.1°C, pH = 6.82)
</TD></TR><TR><TD>15:06 </TD><TD> Sample taken: (O.D.= 1.006, dO<SUB>2</SUB> = -0.2, Temp = 37.0°C, pH = 6.82)
</TD></TR><TR><TD>16:10 </TD><TD> Sample taken: (O.D.= 0.978, dO<SUB>2</SUB> = -0.2, Temp = 37.0°C, pH = 6.81)
</TD></TR><TR><TD>17:15 </TD><TD> Sample taken: (O.D.= 0.978, dO<SUB>2</SUB> = -0.2, Temp = 37.0°C, pH = 6.80)
</TD></TR><TR><TD>18:15 </TD><TD> Sample taken: (O.D.= 0.956, dO<SUB>2</SUB> = -0.2, Temp = 37.0°C, pH = 6.82)
</TD></TR><TR><TD>*08:03 </TD><TD> Sample taken: (O.D.= 0.842, dO<SUB>2</SUB> = -0.3, Temp = 37.0°C, pH = 6.81)
</TD></TR><TR><TD>*12:30 </TD><TD> Sample taken: (O.D.= 0.774, dO<SUB>2</SUB> = -0.2, Temp = 37.0°C, pH = 6.82)
</TD></TR></TBODY></TABLE><P>The (*) represents that these samples were taken on October 18th and not October 17th like the rest of the samples. 
</P><P>We also followed the growth of the bacteria in aerobic conditions although we could only measure the Optical Density and we only took a sample at the very end of the growth period:
</P><TABLE class="table table-striped"><TBODY><TR><TH align="center"> Time
</TH><TH> Procedure
</TH></TR><TR><TD>13:00 </TD><TD> Sample taken: (O.D.= 0.927)
</TD></TR><TR><TD>14:10 </TD><TD> Sample taken: (O.D.= 1.524)
</TD></TR><TR><TD>15:10 </TD><TD> Sample taken: (O.D.= 1.999)
</TD></TR><TR><TD>16:20 </TD><TD> Sample taken: (O.D.= 2.265)
</TD></TR><TR><TD>17:20 </TD><TD> Sample taken: (O.D.= 2.363)
</TD></TR><TR><TD>18:23 </TD><TD> Sample taken: (O.D.= 2.483)
</TD></TR><TR><TD>*08:15 </TD><TD> Sample taken: (O.D.= 1.716)
</TD></TR><TR><TD>*12:30 </TD><TD> Sample taken: (O.D.=1.716)
</TD></TR></TBODY></TABLE></DIV></DIV><DIV class="tue-timeline-middle"><DIV class="tue-timeline-leftbar"><DIV class="tue-timeline-node">18 October2013</DIV></DIV><DIV class="tue-timeline-content"><H3><SPAN class="mw-headline" id="Sample_Storage">Sample Storage</SPAN></H3><P>During the anaerobic expression which was performed both today as well as on October 17th, a number of samples were taken. We obtained 7 samples from the anaerobic chamber and 2 samples were pre-cured from the aerobic culture medium at the end of the growth period. Each sample had a volume of approximately 25mL.
These samples were weighed equally against each other and were subsequently spun down at 16000rpm for 10 minutes. The formed supernatant was poured off and the pellet was snap frozen and placed in the -80°C freezer until the samples could be analysed further. 
</P></DIV></DIV></DIV></DIV><DIV class="tab-pane" id="Week13"><DIV><DIV class="tue-timeline-top"><DIV class="tue-timeline-title"><H2>Week 13</H2></DIV></DIV><DIV class="tue-timeline-middle"><DIV class="tue-timeline-leftbar"><DIV class="tue-timeline-node">21 October2013</DIV></DIV><DIV class="tue-timeline-content"><H3><SPAN class="mw-headline" id="Preparing_the_Anaerobic_Vessel_2">Preparing the Anaerobic Vessel</SPAN></H3><P>Having performed an anaerobic expression test on October 17th it was decided to repeat this expression only with an oxygen concentration not equal to zero, but still within the range of tumours. To do this the vessel needed to be prepared once more. 
Again 7 small cultures were prepared in the following manner:
</P><UL><LI>8mL of LB was added to each of 7 small culture tubes.
</LI><LI>8µL of Ampicillin antibodies was added to each culture tube.
</LI><LI>A single XL-1-Blue bacterial strain containing pBR322 ligated EGFP colony picking was then added to each tube. 
</LI></UL><P>The tubes were then placed in a 37°C incubator and left to grow overnight. 
</P><P>To ensure the vessel was clean and ready for use the vessel was cleaned using water, before refilling the vessel with 4L of LB medium:
</P><UL><LI>40g Peptones
</LI><LI>40g NaCl
</LI><LI>20g Yeast Extract
</LI><LI>4L Demineralised Water
</LI></UL><P>Once in the vessel the vessel was placed into the autoclave and autoclaved overnight ready for expression to take place the next day.
</P><H3><SPAN class="mw-headline" id="Preparing_for_Aerobic_Expression">Preparing for Aerobic Expression</SPAN></H3><P>To retest a number of our proteins in the MRI machine we would also need to express these proteins again. For this we created 3 small cultures, 1 for 1PJN (a Lysine based CEST protein), 1 for 1G70 (an Arginine based CEST protein) and 1 for the control (EGFP which would not be IPTG induced).
For each of these small cultures the following were prepared:
</P><UL><LI>8mL of LB medium was added to each culture tube
</LI><LI>8µL of Kanamycin was added to each culture tube.
</LI><LI>A bacterial colony picking was added to each culture tube. Each would consist of a BL21 bacterial strain containing a pET28a vector ligated with the appropriate protein. 
</LI></UL><P>For each of these small colonies a large culture would also need to be prepared so three vessels each containing 400mL of LB medium. Each of these was made by mixing:
</P><UL><LI>4g Peptones
</LI><LI>4g NaCl
</LI><LI>2g Yeast Extract
</LI><LI>400mL Demineralised Water.
</LI></UL><P>After mixing the above the solutions were autoclaved to ensure sterility. 
</P></DIV></DIV><DIV class="tue-timeline-middle"><DIV class="tue-timeline-leftbar"><DIV class="tue-timeline-node">22 October2013</DIV></DIV><DIV class="tue-timeline-content"><H3><SPAN class="mw-headline" id="Semi-Anaerobic_Expression">Semi-Anaerobic Expression</SPAN></H3><P>On October 21st the anaerobic vessel had been filled with LB medium and autoclaved overnight. On the same day 7 small cultures of EGFP in XL-1-Blue bacteria had been grown in a 37°C incubator. Today the small cultures would be injected into the large 4L vessel and the proteins would be expressed. Previously (October 16th/17th) a similar expression had been performed at 0% Oxygen Saturation. To aid in the modelling of our FNR promotor we would now perform the expression at 5% Oxygen Saturation.
</P><P>To being the vessel was flooded with oxygen to allow for a 100% measurement and 4mL of Ampicillin antibodies were added to the chamber, the temperature of which was set to 37°C. 
The 7 small cultures could then be injected into the vessel and the optical density was measured periodically until it reached a value around 0.6. Once here the oxygen concentration would be set to 5% which would be controlled by the fermentor itself using a gas mixture of Nitrogen and Air. During the expression (initiated by the low oxygen concentration) samples were taken approximately every hour as seen below:
</P><TABLE class="table table-striped"><TBODY><TR><TH align="center"> Time
</TH><TH> Procedure
</TH></TR><TR><TD>09:08 </TD><TD> Final small culture injected: (O.D.= 0.031, dO<SUB>2</SUB> = 89.1, Temp = 34.2°C, pH = 7.13)
</TD></TR><TR><TD>10:15 </TD><TD> Optical Density measurement: (O.D.= 0.061, dO<SUB>2</SUB> = 79.9, Temp = 37.0°C, pH = 7.05)
</TD></TR><TR><TD>11:13 </TD><TD> Optical Density measurement: (O.D.= 0.208, dO<SUB>2</SUB> = 52.9, Temp = 37.0°C, pH = 6.97)
</TD></TR><TR><TD>12:07 </TD><TD> Optical Density measurement: (O.D.= 0.338, dO<SUB>2</SUB> = 10.0, Temp = 37.0°C, pH = 6.82)
</TD></TR><TR><TD>13:23 </TD><TD> Set Oxygen concentration to 5%. Sample taken: (O.D.= 1.078, dO<SUB>2</SUB> = 55.0, Temp = 37.0°C, pH = 7.07)
</TD></TR><TR><TD>14:02 </TD><TD> Sample taken: (O.D.= 0.737, dO<SUB>2</SUB> = 6.6, Temp = 36.9°C, pH = 7.19)
</TD></TR><TR><TD>14:55 </TD><TD> Sample taken: (O.D.= 0.577, dO<SUB>2</SUB> = 4.7, Temp = 37.0°C, pH = 7.17)
</TD></TR><TR><TD>16:17 </TD><TD> Sample taken: (O.D.= 0.553, dO<SUB>2</SUB> = 4.9, Temp = 37.0°C, pH = 7.20)
</TD></TR><TR><TD>17:30 </TD><TD> Sample taken: (O.D.= 0.552, dO<SUB>2</SUB> = 4.7, Temp = 37.0°C, pH = 7.20)
</TD></TR><TR><TD>18:30 </TD><TD> Sample taken: (O.D.= 0.615, dO<SUB>2</SUB> = 4.7, Temp = 37.0°C, pH = 7.20)
</TD></TR><TR><TD>*09:30 </TD><TD> Sample taken: (O.D.= 2.354, dO<SUB>2</SUB> = 0.9, Temp = 37.0°C, pH = 7.14)
</TD></TR></TBODY></TABLE><P>The (*) shows samples which were taken on October 23rd and not October 22nd. 
</P><P>These samples were spun down in a centrifuge at 16000g for 10 minutes. These pellets were then snap frozen until they could be purified.
</P><H3><SPAN class="mw-headline" id="Sample_Analysis">Sample Analysis</SPAN></H3><P>On October 17th an anaerobic expression of EGFP had also been performed at 0% Oxygen saturation. The samples that were taken during this expression had been spun down into pellets which had been placed in the -80°C freezer for storage. Today these samples would be purified so that analysis of the EGFP expression would be possible. After purification the samples would be analysed making use of a fluorescence measurement as well as a western blot/SDS Page gel. 
</P><H4><SPAN class="mw-headline" id="Purification">Purification</SPAN></H4><P>For the actual purification of the samples the BugBuster protocol was followed. Each of the samples had an approximate volume of 25mL and had been spun down into a pellet. This pellet would form the basis of our purification efforts:
</P><UL><LI>400µL of BugBuster was added to each pellet.
</LI><LI>0.4µL of Benzonase was then added to each of the pellets.
</LI><LI>The pellets were placed on a shaking table and left for 30 minutes at room temperature until the entire pellet had dissolved into the solution.
</LI><LI>The solutions were then brought over into eppendorf tubes and spun down in a tabletop centrifuge for 20 minutes at maximum velocity. The supernatant now contains the desired EGFP protein so this was carefully transferred into a separate eppendorf tube.
</LI><LI>The protein solution was purified making use of a general healthcare minispin column kit. 
</LI><LI>The fluid atop the spin columns was poured off.
</LI><LI>Then the column was washed using standard wash buffer. (600µL was added to each column which was then spun down a 12000 rpm for 30 seconds.
</LI><LI>We then loaded 600µL of each sample solution onto a column and span this column down for 30 seconds. 
</LI><LI>Hereafter we performed the wash step a second time. 
</LI><LI>By now adding 300µL of elution buffer and spinning the column for 30 seconds at 12000rpm we retained a solution containing our purified EGFP proteins. 
</LI></UL><H4><SPAN class="mw-headline" id="Fluorescence_Measurements">Fluorescence Measurements</SPAN></H4><P>From the purified proteins we now pippetted 200µL of each sample into a well of a 96 wells plate. This plate was then left open so that the samples could mature and was subsequently placed in the fluorescence reader where the measurements were conducted. We recorded the specific preferences so that we would be able to perform the same measurement on other EGFP samples allowing for equal comparison. In the image below we can see the intensities of the EGFP samples:
</P><DIV class="row"><DIV class="tue-float tue-float-left span8"><DIV class="tue-caption"><SPAN class="tue-caption-figure"><SPAN class="tue-refid">EGFP Intensities</SPAN></SPAN> <SPAN class="tue-caption-text">EGFP intensity over time.</SPAN></DIV></DIV></DIV><H4><SPAN class="mw-headline" id="SDS_Gel_and_Western_Blot_Preparation">SDS Gel and Western Blot Preparation</SPAN></H4><P>Another analysis we would be performing on the samples would be an SDS gel and following that a Western blot. As this would be taking place at a later date, and considering that we had purified the samples using a buffer containing imidazole we would need to prepare the samples for use on an SDS gel and then freeze this sample. (If left untreated Imidazole will damage the proteins, and as the quantities we have are so small we cannot rebuffer the samples). 
</P><UL><LI>First a mastermix was created of 2x SDS loading dye and DTT consisting of 120µL of loading dye and 0.3µL of DTT.
</LI><LI>15µL of each samples was subsequently mixed with 15µL of the mastermix. 
</LI><LI>This entire sample could then be heated at 95°C to denature the proteins. 
</LI><LI>The samples were then snap frozen in liquid nitrogen and stored at -80°C. 
</LI></UL><P>When needed the samples can now be defrosted and immediately used for running on an SDS gel or a western blot.
</P><H3><SPAN class="mw-headline" id="Aerobic_Expression">Aerobic Expression</SPAN></H3><P>Aside from preparing the anaerobic vessels on October 21st, three cultures had also been prepared for aerobic expression. We would be attempting to express 1G70 and 1PJN alongside a control (EGFP which would not be IPTG induced.)
These samples were prepared on October 21st and were allowed to grow overnight. 
</P><P>The following procedures were then followed in the path to expression:
</P><UL><LI>400µL of Kanamycin antibodies was added to each of the three 400mL LB medium vessels.
</LI><LI>Hereafter the entire small cultures was poured into the appropriate vessel.
</LI><LI>The vessel were placed in the 37°C incubator and the optical densities were followed over time:
</LI></UL><TABLE class="table table-striped"><TBODY><TR><TH align="center"> Time
</TH><TH> 1PJN
</TH><TH> 1G70
</TH><TH> EGFP
</TH></TR><TR><TD>16:32 </TD><TD> 0.081 </TD><TD> 0.082 </TD><TD> 0.105
</TD></TR><TR><TD>17:35 </TD><TD> 0.391 </TD><TD> 0.359 </TD><TD> 0.442
</TD></TR><TR><TD>17:50 </TD><TD> 0.622* </TD><TD> 0.627* </TD><TD> 0.745
</TD></TR></TBODY></TABLE><UL><LI> A (*) shows that IPTG was added to the culture mediums.
</LI><LI>Once the optical density was around 0.6 200µL of IPTG (1M) was added to each of the vessels. 
</LI><LI>The vessels were then returned to the incubator and left there overnight for expression. 
</LI></UL></DIV></DIV><DIV class="tue-timeline-middle"><DIV class="tue-timeline-leftbar"><DIV class="tue-timeline-node">23 October2013</DIV></DIV><DIV class="tue-timeline-content"><H3><SPAN class="mw-headline" id="Preparing_MRI_samples">Preparing MRI samples</SPAN></H3><P>On October 22nd three proteins had been expressed aerobically under induction of IPTG. These proteins had been left to express overnight and would now need to be obtained so that MRI measurements could be performed. 
</P><UL><LI>Each of the three protein cultures was poured into a flask and the flasks were weighed equally against one another. 
</LI><LI>The flasks were the spun down for 10 minutes at 16000g. 
</LI><LI>The supernatant which had formed during the spinning was poured off leaving us with a bacterial pellet containing our proteins. 
</LI></UL><P>For the MRI measurements we wanted to image each of the three proteins simultaneously we would need to devise a way in which the proteins were all in the same place whilst remaining separate from one another. To achieve this a small 3.5mm diameter agar plate was made and onto this we spread a portion of each of the separate protein samples. These samples were not touching one another. 
For the case that the thickness of the samples was not large enough we also sent off a sample from each protein in a small PCR tube. 
</P><H3><SPAN class="mw-headline" id="Sample_purification">Sample purification</SPAN></H3><P>On October 22nd expression of EGFP had been performed at 5% oxygen saturation. Over the course of this expression a number of samples were taken and spun down forming pellets. To be able to analyse the EGFP expression we would have to first purify these pellets to obtain our pure protein. The purification would be performed making use of the BugBuster protocol, and the analysis would take place by performing fluorescence measurements alongside SDS gels and Western Blots. 
</P><H4><SPAN class="mw-headline" id="Purification_2">Purification</SPAN></H4><P>For the actual purification of the samples the BugBuster protocol was followed. Each of the samples had an approximate volume of 25mL and had been spun down into a pellet. This pellet would form the basis of our purification efforts:
</P><UL><LI>400µL of BugBuster was added to each pellet.
</LI><LI>0.4µL of Benzonase was then added to each of the pellets.
</LI><LI>The pellets were placed on a shaking table and left for 30 minutes at room temperature until the entire pellet had dissolved into the solution.
</LI><LI>The solutions were then brought over into eppendorf tubes and spun down in a tabletop centrifuge for 20 minutes at maximum velocity. The supernatant now contains the desired EGFP protein so this was carefully transferred into a separate eppendorf tube.
</LI><LI>The protein solution was purified making use of a general healthcare minispin column kit. 
</LI><LI>The fluid atop the spin columns was poured off.
</LI><LI>Then the column was washed using standard wash buffer. (600µL was added to each column which was then spun down a 12000 rpm for 30 seconds.
</LI><LI>We then loaded 600µL of each sample solution onto a column and span this column down for 30 seconds. 
</LI><LI>Hereafter we performed the wash step a second time. 
</LI><LI>By now adding 300µL of elution buffer and spinning the column for 30 seconds at 12000rpm we retained a solution containing our purified EGFP proteins. 
</LI></UL><H4><SPAN class="mw-headline" id="Fluorescence_Measurements_2">Fluorescence Measurements</SPAN></H4><P>From the purified proteins we now pippetted 200µL of each sample into a well of a 96 wells plate. This plate was then left open so that the samples could mature and was subsequently placed in the fluorescence reader where the measurements were conducted. Using the same preferences as we had used on October 22nd we would be able to accurately compare the results of both EGFP expression. The results of the fluorescence measurements are shown below:
</P><DIV class="row"><DIV class="tue-float tue-float-left span9"><DIV class="tue-caption"><SPAN class="tue-caption-figure"><SPAN class="tue-refid">EGFP intensities over time</SPAN></SPAN> <SPAN class="tue-caption-text">EGFP intensity over time</SPAN></DIV></DIV></DIV><H4><SPAN class="mw-headline" id="SDS_Gel_and_Western_Blot_Preparation_2">SDS Gel and Western Blot Preparation</SPAN></H4><P>Another analysis we would be performing on the samples would be an SDS gel and following that a Western blot. As this would be taking place at a later date, and considering that we had purified the samples using a buffer containing imidazole we would need to prepare the samples for use on an SDS gel and then freeze this sample. (If left untreated Imidazole will damage the proteins, and as the quantities we have are so small we cannot rebuffer the samples). 
</P><UL><LI>First a mastermix was created of 2x SDS loading dye and DTT consisting of 120µL of loading dye and 0.3µL of DTT.
</LI><LI>15µL of each samples was subsequently mixed with 15µL of the mastermix. 
</LI><LI>This entire sample could then be heated at 95°C to denature the proteins. 
</LI><LI>The samples were then snap frozen in liquid nitrogen and stored at -80°C. 
</LI></UL><P>When needed the samples can now be defrosted and immediately used for running on an SDS gel or a western blot.
</P></DIV></DIV><DIV class="tue-timeline-middle"><DIV class="tue-timeline-leftbar"><DIV class="tue-timeline-node">24 October2013</DIV></DIV><DIV class="tue-timeline-content"><H3><SPAN class="mw-headline" id="EGFP_Protein_Analysis">EGFP Protein Analysis</SPAN></H3><P>On both October 17th and October 22nd EGFP expression was performed and followed by the way of taking periodic samples of the culture medium. On respectively October 22nd and October 23rd these samples were prepared, ready to be used for the running of an SDS gel and a Western Blot. Using these samples we could now perform both the pre-mentioned techniques and gain an understanding of the EGFP expression under different oxygen concentrations. 
</P><H4><SPAN class="mw-headline" id="Running_the_SDS_Gels">Running the SDS Gels</SPAN></H4><P>After running an SDS gel the gels need to be stained with Coomaaise to make the bands visible on the gel. However this is not a specific staining method, which is why we wish to also make use of a Western Blot. However the two are not compatible, a gel which has been stained with Coomaaise is no longer usable for a Western Blot. Therefore we will need to load each of the samples onto 2 separate SDS gels, one for each of the imaging techniques we wish to use. 
</P><UL><LI> We made two separate 12% SDS gels and placed these in the appropriate buffer. 
</LI><LI> We then loaded 10µL of each sample into both of the gels.
</LI><LI> The gels were subsequently run in the following fashion:
</LI><LI> 15 minutes at 90V
</LI><LI> 45 minutes at 150V
</LI></UL><LI>After running the gels both gels were washed in de-mineralised water. 
</LI><H4><SPAN class="mw-headline" id="Coomaaise_Analysis">Coomaaise Analysis</SPAN></H4><P>After having run and washed the gels in water one of the gels was placed in 1x used coomaaise medium. The gel in solution was then placed on a rocking table and left to stain for approximately 2 hours. 
Hereafter the gel was washed with de-mineralissed water for a number of hours before the plate was imaged:
</P><DIV class="row"><DIV class="tue-float tue-float-left span9"><DIV class="tue-caption"><SPAN class="tue-caption-figure"><SPAN class="tue-refid">1G70_Double_Bands</SPAN></SPAN> <SPAN class="tue-caption-text">SDS gel showing the results of loading the purified EGFP samples onto a 12% SDS gel. The columns contain as follows: 1: Aerobic Control, 2-7: Samples taken during the anaerobic expression at 0% Oxygen saturation, 8-13: Samples taken during the anaerobic expression at 5% Oxygen saturation, 14: Control EGFP bound to CNA35 resulting in higher length (Control for Westernblot). The EGFP protein bands of interest are shown in red, the higher thicker bands above do not represent EGFP as later shown by the western blot, futher more the size indicated by these bands does not fit the EGFP profile.</SPAN></DIV></DIV></DIV><H4><SPAN class="mw-headline" id="Western_Blot">Western Blot</SPAN></H4><P>The other gel (which wasn't washed with coomaaise) would now under go Western Blotting. To do this the following steps were performed:
</P><UL><LI>The gels was placed in a clamp to allow for the transfer from the gel to a nitrocellulose membrane. 
</LI><LI>The clamp was placed in the following blot buffer:
</LI><LI> 1x SDS Blot Buffer
</LI><LI> 20% Methanol
</LI></UL><LI>The blot was then run for 60 minutes at 100V. 
</LI><LI>After blotting the membrane was placed into a 50mL falcon tube. (Make sure that the side you blotted onto is facing into the tube.)
</LI><LI>The tube was filled with 25mL of PBS/Tween with 5% Milk Powder and placed on a roller for 30 minutes.
</LI><LI>After 30 minutes the contents of the tube was removed and replaced with 5mL of PBS/Tween with 0.5% Milk Powder and a 1:2000 ratio of GFP specific antibodies. Containing this solution the tube was returned to the rollers and left for 60 minutes.
</LI><LI>Hereafter the contents was removed from the tube and the tube was washed 3 times for 5 minutes with PBS/Tween solution. 
</LI><LI>Now the tube could be filled with 5mL of PBS/Tween with 0.5% Milk Powder and a secondary antibody with the ratio 1:2000. The tube was then returned to the rollers for another hour. 
</LI><LI>The tube was once again washed 3 times for 5 minutes with PBS/Tween solution.
</LI><LI>The membrane could now be removed from the tube and spread out. Onto the membrane we then pipetted 2mL of ####### which made the bands visible.
</LI><P>An image of the final western blot can be seen below:
</P><DIV class="row"><DIV class="tue-float tue-float-left span9"><DIV class="tue-caption"><SPAN class="tue-caption-figure"><SPAN class="tue-refid">WesternBlot</SPAN></SPAN> <SPAN class="tue-caption-text">The Western blot clearly shows the bands which were marked on the SDS gel as being EGFP. We also no longer see the thick upper bands showing that these are in fact not EGFP. Futher the columns are given as follows: 1: Aerobic Control, 2-7: Samples taken during the anaerobic expression at 0% Oxygen saturation, 8-13: Samples taken during the anaerobic expression at 5% Oxygen saturation, 14: Control EGFP bound to CNA35 resulting in higher length.</SPAN></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV><DIV class="tue-gototop-sticky"><A href="#top" title="Go to the top of the page">Go to Top   </A></DIV><DIV class="printfooter">
Retrieved from &quot;<A href="http://2013.igem.org/Team:TU-Eindhoven/LabJournal">http://2013.igem.org/Team:TU-Eindhoven/LabJournal</A>&quot;</DIV></DIV></DIV><DIV id="footer-box" class="noprint"><DIV id="footer"><UL id="f-list"><LI id="t-recentchanges"><A href="/Special:RecentChanges" title="Recent changes">Recent changes</A></LI><LI id="t-whatlinkshere"><A href="/Special:WhatLinksHere/Team:TU-Eindhoven/LabJournal" title="List of all wiki pages that link here [j]" accesskey="j">What links here</A></LI><LI id="t-recentchangeslinked"><A href="/Special:RecentChangesLinked/Team:TU-Eindhoven/LabJournal" title="Recent changes in pages linked from this page [k]" accesskey="k">Related changes</A></LI><LI id="t-specialpages"><A href="/Special:SpecialPages" title="List of all special pages [q]" accesskey="q">Special pages</A></LI><LI><A href="/Special:Preferences">My preferences</A></LI></UL></DIV><DIV id="footer"><UL id="f-list"><LI id="t-print"><A href="/wiki/index.php?title=Team:TU-Eindhoven/LabJournal&amp;printable=yes" title="Printable version of this page [p]" accesskey="p">Printable version</A></LI><LI id="t-permalink"><A href="/wiki/index.php?title=Team:TU-Eindhoven/LabJournal&amp;oldid=330803" title="Permanent link to this revision of the page">Permanent link</A></LI><LI id="privacy"><A href="/2013.igem.org:Privacy_policy" title="2013.igem.org:Privacy policy">Privacy policy</A></LI><LI id="disclaimer"><A href="/2013.igem.org:General_disclaimer" title="2013.igem.org:General disclaimer">Disclaimers</A></LI></UL></DIV></DIV></DIV></BODY></HTML>