<HTML xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en" dir="ltr">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki  ltr ns-0 ns-subject page-Team_TU-Eindhoven_Modeling"><DIV id="globalWrapper"><DIV id="top-section"><DIV id="p-logo"><A href="/Main_Page" title="Main Page">&quot;
	    </A></DIV><DIV id="menubar" class="left-menu noprint"><UL><LI class="selected"><A href="/Team:TU-Eindhoven/Modeling">Page               </A></LI><LI class="new"><A href="/wiki/index.php?title=Talk:Team:TU-Eindhoven/Modeling&amp;action=edit&amp;redlink=1">Discussion               </A></LI><LI><A href="/wiki/index.php?title=Team:TU-Eindhoven/Modeling&amp;action=edit">View source               </A></LI><LI><A href="/wiki/index.php?title=Team:TU-Eindhoven/Modeling&amp;action=history">History               </A></LI><LI style="color:white;cursor:default">teams</LI></UL></DIV><DIV class="right-menu noprint" id="menubar"><UL><LI id="pt-login"><A href="/wiki/index.php?title=Special:UserLogin&amp;returnto=Team:TU-Eindhoven/Modeling" title="You are encouraged to log in; however, it is not mandatory [o]" accesskey="o">Log in</A></LI></UL></DIV><DIV id="search-controls" class="noprint"><FORM action="/Special:Search" id="searchform"> </FORM></DIV></DIV><DIV id="content"><H1 class="firstHeading">Team:TU-Eindhoven/Modeling</H1><DIV id="bodyContent"><H3 id="siteSub" class="noprint">From 2013.igem.org</H3><DIV class="navwrapper"><DIV class="container"><DIV class="navbar"><DIV class="navbar-inner"><UL class="nav"><LI id="tue-menu-home"><A href="/Team:TU-Eindhoven">Home</A></LI><LI id="tue-menu-team"><A href="/Team:TU-Eindhoven/Team">Team</A></LI><LI id="tue-menu-project" class="dropdown"><A class="dropdown-toggle" data-toggle="dropdown" href="#">Project </A></LI><LI><A href="/Team:TU-Eindhoven/Project">Abstract</A></LI><LI><A href="/Team:TU-Eindhoven/Description">Project Description</A></LI><LI><A href="/Team:TU-Eindhoven/Background">Background</A></LI><LI><A href="/Team:TU-Eindhoven/Chasis">The Chassis</A></LI><LI><A href="/Team:TU-Eindhoven/Production">Potential Contrast Agents</A></LI><LI><A href="/Team:TU-Eindhoven/KillingMechanism">Kill Switch</A></LI><LI><A href="/Team:TU-Eindhoven/Applications">Applications</A></LI><LI class="nav-header">Results</LI><LI><A href="/Team:TU-Eindhoven/Results">Main Results</A></LI><LI><A href="/Team:TU-Eindhoven/Parts">Parts Submitted</A></LI><LI class="nav-header">Outlook</LI><LI><A href="/Team:TU-Eindhoven/FutureProspects">Future Steps</A></LI><LI class="nav-header">Others</LI><LI><A href="/Team:TU-Eindhoven/Attributions">Attributions</A></LI><LI><A href="/Team:TU-Eindhoven/Collaboration">Collaboration</A></LI><LI><A href="/Team:TU-Eindhoven/Achievements">Achievements</A></LI></UL><LI id="tue-menu-wetlab" class="dropdown"><A class="dropdown-toggle" data-toggle="dropdown" href="#">Wetlab </A></LI><LI><A href="/Team:TU-Eindhoven/Wetlab">Overview</A></LI><LI><A href="/Team:TU-Eindhoven/Preparation">Preparation</A></LI><LI class="nav-header">Lab Reports</LI><LI><A href="/Team:TU-Eindhoven/MRIProcessing">MRI Data Processing</A></LI><LI><A href="/Team:TU-Eindhoven/AnearobicTesting">Anaerobic FNR Testing</A></LI><LI><A href="/Team:TU-Eindhoven/CESTAgentTesting">CEST Agent Testing</A></LI><LI><A href="/Team:TU-Eindhoven/LabJournal">Lab Journal</A></LI><LI id="tue-menu-drylab" class="dropdown"><A class="dropdown-toggle" data-toggle="dropdown" href="#">Drylab </A></LI><LI><A href="/Team:TU-Eindhoven/Modeling">Summary</A></LI><LI><A href="/Team:TU-Eindhoven/ProteinSelection">Protein Selection</A></LI><LI class="nav-header">Contrast Mechanism</LI><LI><A href="/Team:TU-Eindhoven/StochasticModel">Decoy Sites</A></LI><LI><A href="/Team:TU-Eindhoven/ODEModel">FNR Dynamics</A></LI><LI><A href="/Team:TU-Eindhoven/FNRPromotor">FNR Promotor</A></LI><LI><A href="/Team:TU-Eindhoven/IntegralModel">Integral Model</A></LI><LI class="nav-header">Bacteria in the Body</LI><LI><A href="/Team:TU-Eindhoven/PBPK">Ganciclovir Distribution</A></LI><LI><A href="/Team:TU-Eindhoven/TimeKill">Time-kill Behavior</A></LI><LI id="tue-menu-safety" class="dropdown"><A class="dropdown-toggle" data-toggle="dropdown" href="#">Safety </A></LI><LI><A href="/Team:TU-Eindhoven/Safety">Overview</A></LI><LI><A href="/Team:TU-Eindhoven/LabSafety">Lab Safety</A></LI><LI><A href="/Team:TU-Eindhoven/PublicSafety">Public and Environmental Safety</A></LI><LI><A href="/Team:TU-Eindhoven/SafetyiGEM">Biosafety Within iGEM</A></LI><LI id="tue-menu-society" class="dropdown"><A class="dropdown-toggle" data-toggle="dropdown" href="#">Society </A></LI><LI><A href="/Team:TU-Eindhoven/Outreach">Outreach</A></LI><LI><A href="/Team:TU-Eindhoven/Society">Human Practice</A></LI><LI id="tue-menu-notebook"><A href="/Team:TU-Eindhoven/Notebook">Notebook</A></LI></DIV></DIV></DIV></DIV><DIV class="tue-content container"><DIV class="row"><BLOCKQUOTE class="pull-right"><P>...no models can be 'correct'. The role of any model is to provide a framework for viewing known facts and to suggest experiments.</P><SMALL>Suresh Moolgavkar</SMALL></BLOCKQUOTE></DIV><H1><SPAN class="mw-headline" id="Modeling_Summary">Modeling Summary</SPAN></H1><P>The main objective of the modeling component of our project is to provide insight to the rest of team in what concerns the effect of altering the necessary conditions for the proper operation of each component of our device. Keeping this in mind we established a series of questions that if answered thoroughly would benefit our project. We categorized these questions in four main topics:
</P><DIV class="tue-float tue-float-left span12"><DIV class="tue-caption"><SPAN class="tue-caption-figure"><SPAN class="tue-refid">eColiModelsFigure</SPAN></SPAN> <SPAN class="tue-caption-text">General Models of Processes Happening Inside the E. Coli.</SPAN></DIV></DIV><P><I>Protein Selection</I></P><OL><LI> Which proteins that can be expressed in bacteria are best suited as CEST based markers? 
</LI></OL><P><I>Gene expression</I></P><OL><LI> How does the oxygen concentration affect the concentration of the transcription activator (FNR) in its active form?
</LI><LI> How does introducing decoy sites influence the gene expression rate?
</LI></OL><P><I>Tumor Imaging In Vivo</I></P><OL><LI> How do bacteria distribute within the human body?
</LI><LI> How long does it take for the bacteria to reach the target area?
</LI><LI> How do the bacteria react to the activation of the killing mechanism?
</LI><LI> What is the bacteria elimination rate?
</LI></OL><P>
Based on the previous question we developed seven models that can be divided into two categories, processes occurring inside the bacteria and the interaction of the bacteria with its surroundings.  presents a schematic diagram of the developed models related to the first category, while  depicts the model that accounts for the interaction between the bacteria and its surroundings when introduced into the human body for tumor imaging. Both figures will be further explained in the next paragraphs. The approaches used for the models range from molecular dynamics simulations to deterministic models and stochastic simulations as well as pharmacokinetics.
</P><P>
As previously mentioned,  aims to schematically depict all the models that are related to the diverse components of our device, e.g the proteins that will be produced by the device, the FNR promoter and decoy sites which are related to the gene expression, and the CEST protein production directly related to the CEST MRI contrast agent. 
</P><DIV class="tue-float tue-float-right span7"><DIV class="tue-caption"><SPAN class="tue-caption-figure"><SPAN class="tue-refid">PBPKFigure</SPAN></SPAN> <SPAN class="tue-caption-text">Model That Depicts The Administration Of The CEST MRI Contrast Producing Bacteria Into The Patient's Bloodstream.</SPAN></DIV></DIV><P>
To define which proteins are most suitable for the production of the MRI CEST contrast agent by the bacteria we used <A href="/Team:TU-Eindhoven/ProteinSelection" title="Team:TU-Eindhoven/ProteinSelection"> molecular dynamics simulations</A>. By using this approach we were able to determine the list of proteins that would be generated by our device, based on the the accessibility of the various exchangeable hydrogen atoms of the Arginine and Lysine amino acids of the proteins being considered. In what concerns the <A href="/Team:TU-Eindhoven/ODEModel" title="Team:TU-Eindhoven/ODEModel"> FNR promotor</A> and <A href="/Team:TU-Eindhoven/StochasticModel" title="Team:TU-Eindhoven/StochasticModel"> decoy sites</A>, both deterministic models and stochastic simulations were used to predict the concentration of transcription factor and influence of introducing decoy sites. Aside from the individual models made for these two components of the device, a merged deterministic model was created to analyze the complete gene expression process. A sensitivity analysis was used on the merged model to determine which elements of the model have the greatest influence on the outcome. Similarly, a PLE analysis was utilized on the decoy sites model. 
</P><P>
Furthermore, a deterministic model was developed for analyzing the protein production and degradation. Using this information it was possible to predict the average concentration of the CEST proteins produced by our device. These results where then compared to the experimental data.
</P><P> is related to the proposed applications of our contrast agent. This figure presents the complete process of injecting the bacteria into the bloodstream in order to reach the tumor region and start producing the MRI CEST contrast agent. When the contrast agent is no longer needed a prodrug is injected into the bloodstream to activate the killing mechanism that will induce apoptosis in the bacteria. The whole process is modeled using a <A href="/Team:TU-Eindhoven/PBPK" title="Team:TU-Eindhoven/PBPK"> pharmacokinetics approach</A>. By using this approach we aim at predicting the distribution of the bacteria in the body and model the kill switch that should be implemented when further developing this application.
</P><P><B>The source code of all models can be found <A href="/Team:TU-Eindhoven/Code:Models" title="Team:TU-Eindhoven/Code:Models"> here</A>.</B></P></DIV><DIV class="tue-gototop-sticky"><A href="#top" title="Go to the top of the page">Go to Top   </A></DIV><DIV class="printfooter">
Retrieved from &quot;<A href="http://2013.igem.org/Team:TU-Eindhoven/Modeling">http://2013.igem.org/Team:TU-Eindhoven/Modeling</A>&quot;</DIV></DIV></DIV><DIV id="footer-box" class="noprint"><DIV id="footer"><UL id="f-list"><LI id="t-recentchanges"><A href="/Special:RecentChanges" title="Recent changes">Recent changes</A></LI><LI id="t-whatlinkshere"><A href="/Special:WhatLinksHere/Team:TU-Eindhoven/Modeling" title="List of all wiki pages that link here [j]" accesskey="j">What links here</A></LI><LI id="t-recentchangeslinked"><A href="/Special:RecentChangesLinked/Team:TU-Eindhoven/Modeling" title="Recent changes in pages linked from this page [k]" accesskey="k">Related changes</A></LI><LI id="t-specialpages"><A href="/Special:SpecialPages" title="List of all special pages [q]" accesskey="q">Special pages</A></LI><LI><A href="/Special:Preferences">My preferences</A></LI></UL></DIV><DIV id="footer"><UL id="f-list"><LI id="t-print"><A href="/wiki/index.php?title=Team:TU-Eindhoven/Modeling&amp;printable=yes" title="Printable version of this page [p]" accesskey="p">Printable version</A></LI><LI id="t-permalink"><A href="/wiki/index.php?title=Team:TU-Eindhoven/Modeling&amp;oldid=331671" title="Permanent link to this revision of the page">Permanent link</A></LI><LI id="privacy"><A href="/2013.igem.org:Privacy_policy" title="2013.igem.org:Privacy policy">Privacy policy</A></LI><LI id="disclaimer"><A href="/2013.igem.org:General_disclaimer" title="2013.igem.org:General disclaimer">Disclaimers</A></LI></UL></DIV></DIV></DIV></BODY></HTML>