<HTML xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en" dir="ltr">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki  ltr ns-0 ns-subject page-Team_Utah_State_Results"><DIV id="globalWrapper"><DIV id="top-section"><DIV id="p-logo"><A href="/Main_Page" title="Main Page">&quot;
	    </A></DIV><DIV id="menubar" class="left-menu noprint"><UL><LI class="selected"><A href="/Team:Utah_State/Results">Page               </A></LI><LI class="new"><A href="/wiki/index.php?title=Talk:Team:Utah_State/Results&amp;action=edit&amp;redlink=1">Discussion               </A></LI><LI><A href="/wiki/index.php?title=Team:Utah_State/Results&amp;action=edit">View source               </A></LI><LI><A href="/wiki/index.php?title=Team:Utah_State/Results&amp;action=history">History               </A></LI><LI style="color:white;cursor:default">teams</LI></UL></DIV><DIV class="right-menu noprint" id="menubar"><UL><LI id="pt-login"><A href="/wiki/index.php?title=Special:UserLogin&amp;returnto=Team:Utah_State/Results" title="You are encouraged to log in; however, it is not mandatory [o]" accesskey="o">Log in</A></LI></UL></DIV><DIV id="search-controls" class="noprint"><FORM action="/Special:Search" id="searchform"> </FORM></DIV></DIV><DIV id="content"><H1 class="firstHeading">Team:Utah State/Results</H1><DIV id="bodyContent"><H3 id="siteSub" class="noprint">From 2013.igem.org</H3><DIV class="wrapOverall"><DIV style="display: inline-block; width: auto; height: 7050px;"><DIV style="position: absolute; top: 150px; display: inline-block; float: left;"><DIV style="display: inline-block; position: absolute; top: 160px; margin-left: 250px; width: 500px;"><DIV class="Header1">
                How our system works
                </DIV><DIV class="subHead1"><DIV class="Header1">
Data for our new favorite parts

                </DIV><DIV class="subHead1"><P class="textProfile3">
Main page: <B><A href="http://parts.igem.org/wiki/index.php?title=Part:BBa_K1162006"> (BBa_K1162006)</A></B> - LL-37 antimicrobial peptide. Demonstrated protein expression with GFP at C-terminal. Part was also used in a fusion with spider silk.


Main page: <B><A href="http://parts.igem.org/wiki/index.php?title=Part:BBa_K1162306"> (BBa_K1162306)</A></B> - LL-37 antimicrobial peptide fused with 8x spider silk protein, with 10x His-Tag. Part was able to produce antimicrobial spider silk from BioBricks.

Main page: <B><A href="http://parts.igem.org/wiki/index.php?title=Part:BBa_K1162009"> (BBa_K1162009)</A></B> - 10x-Histidine (10x-His) Tag with Met (ATG) For N-terminal purification. Demonstrated that this part functions correctly with the use of GFP (BBa_K208000) in part number: BBa_K1162013. 
</P></DIV><DIV class="Header1">
                Data for pre-existing parts
                </DIV><DIV class="subHead1"><P class="textProfile3">
Experience: <B><A href="http://parts.igem.org/Part:BBa_K208010:Experience#User_Reviews"> (BBa_K208010)</A></B> - Lac Promoter and RBS (Utah State iGEM 2009). This part was successfully able to express AMP-GFP-HT constructs and LL37-spider silk-HT (e.g. BBa_K1162306) constructs. Additionally this part was used successfully in BBa_K1162013. 

Experience: <B><A href="http://parts.igem.org/Part:BBa_K844000:Experience#User_Reviews"> (BBa_K844000)</A></B> - 10x Histidine (10x His)-Tag with double stop codon, was used as the purification tag for AMP production. It was successfully used in purification of LL37-spider silk-HT (e.g. BBa_K1162306). Also demonstrated purification of a new GFP part (BBa_K1162011) with this 10x His-Tag.
</P></DIV><DIV class="Header1">
                We've Also Characterized the Following Parts
                </DIV><DIV class="subHead1"><P class="textProfile3">
Main page: <B><A href="http://parts.igem.org/wiki/index.php?title=Part:BBa_K1162009"> (BBa_K1162009)</A></B> - 10x Histidine (10x His)-Tag with Methionine(atg) For N-terminal purification. Demonstrated that this part functions correctly with the use of GFP (BBa_K208000) in part number: BBa_K1162013.

Main page: <B><A href="http://parts.igem.org/wiki/index.php?title=Part:BBa_K1162011"> (BBa_K1162011)</A></B> - GFP with stop codon removed. Demonstrated that this part functions correctly with the addition of a C-terminal 10x His-Tag (BBa_K844000) for protein purification. 
</P></DIV></DIV><DIV class="Header1">
                Expression of AMPs in E. coli
                </DIV><DIV class="subHead1"><P class="textProfile3">
In looking at the registry we found the AMP LL37 which is commonly found in Humans. Unfortunately the previous iterations of LL37 were not suitable for our purposes (either lacking a start codon: BBa_K245114, or not RFC 23, BBa_K875009 was RFC10). We decided to construct our own LL37 with RFC 23 under consideration as well as adding a start codon.

All other AMPs were new to the registry (and all submitted). These AMPs were also designed under the same parameters set forth in the LL37 (addition of start codon, and RFC23). It should be mentioned that all AMPs were codon optimized for expression in E. coli.                   	

All AMPs were first cloned in front of a 10x His-Tag (BBa_K844000) and double terminator (BBa_B0015). A promoter and rbs was added to the construct to have a complete AMP production system. 

After growing cells and purifying protein (see protocols page), SDS-PAGE gels were run with the elution fraction of each AMP from the nickel column. After coomassie staining and de-staining it was found that it was too hard to see the if there was any AMP expression as the final AMP products were so small. 

The next step was to fuse the AMPs with GFP at the C-terminal. Since the AMPs were designed with RFC 23 under consideration creating a AMP-GFP fusion would not be an issue. Having GFP at the C-terminal means that if fluorescence is seen then AMPs were being expressed. In addition to fluorescence we wanted to try and purify AMP-GFP protein. This led to the creation/modification of an existing GFP to remove the stop codons (BBa_K1162011). The removal of the stop codon enabled the fusion of a C-terminal His-Tag to the end of the GFP. Protein purification was carried out using 0.2 mL Nickel columns. 
                	</P><P class="textProfile3">
The image above demonstrates that there is expression of LL37 and WAM in E c.oli as demonstrated by GFP fluorescence (GFP is at the C-terminal end). Unfortunately there were issues with binding to the Nickel column which will be troubleshooted in future work.</P></DIV><DIV class="Header1">
                Production of Antimicrobial Spider Silk in E. coli
                </DIV><DIV class="subHead1"><P class="textProfile3">
                	To demonstrate that antimicrobial spider silk could be manufactured in E. coli, the AMP LL-37 (BBa_K1162006) was fused to 8 repeats of spider silk (BBa_K844004. A lac inducible promoter system (BBa_K208010), 10x His Tag (BBa_K844000), and double terminator (BBa_B0015) were added to create the first antimicrobial spider silk generator with BioBricks.

The schematic below demonstrates the experimental process for validating and manufacturing of of AMP-spider silk fused proteins. As an example, the LL37-spider silk construct (BBa_K1162306) was chosen for expression studies (other AMP-spider silk constructs are listed on the BioBrick page). 
First 50 mL cultures were used to grow up bacteria containing LL37-spider silk. Protein was purified using a 0.2 mL nickel column and protein gel was run with: cell lysate, flow through, wash, and elution fractions. Gel from 50 mL culture did not show any evidence to suggest protein expression.
</P><P class="textProfile3">
Next we decided to scale up the process to a 500 mL culture, purify on a 3 mL nickel column and carry out the protein analysis again (similar to 50 mL culture). We hypothesized that maybe our expression levels were low and proceeded to a 10 L fermentation and purified the protein on a 5 mL nickel column. The results of the 10L fermentation are shown in the gel below. In the elution fraction of the gel image below, we can see a protein band at approximately 55-60 kDa that is the size of the LL37-silk protein. From this study we have demonstrated that antimicrobial spider silk can be produced in E. coli and scale-up. Since we also have additional AMP-spider silk constructs(see BioBricks page for a complete list), the next step would be to manufacture more of them using a similar approach to LL37-spider silk. The direct applications of LL37-spider silk could include the creation of biomedical sutures or films. Increase production of this potentially valuable product would need to take place in future work.</P></DIV><DIV class="Header1">
               N-terminal protein purification
                </DIV><DIV class="subHead1"><P class="textProfile3">
                	 To demonstrate that the N-terminal 10x His Tag (BBa_K1162009) functions correctly, it was cloned in front of Green Florescent Protein (GFP)with the lac promoter+rbs system (BBa_K208010)to give the complete construct BBa_K1162013. This construct was expressed in E. coli grown in 50mL LB media with the addition of chloramphenicol and induced with IPTG. After allowing to grow overnight the cells were spun down and protein was purified with a nickel spin column (see protein purification protocol on protocols page). Fractions from this nickel spin column procedure were saved and run on an SDS-PAGE gel (see below). Since this was a GFP purification procedure, the different fractions were dotted on parafilm and place on a UV gel box to visualize the protein (see below).
                	</P></DIV><DIV class="subHead1"><P class="textProfile3">
From the SDS-PAGE gel it can be seen that there is pure GFP (~26.9 kDa) in the elution fraction number 2 and 3. The wash fractions do not appear to have any GFP which indicates that the N terminal 10x His Tag is strongly bound to the Nickel Column during washing steps. Coupled with the GFP dot image it is clear that this method of purification works as desired and adds another purification system to the registry.

After demonstration that the N-terminal purification system functioned as desired, other constructs can be built to purify protein using this method. 

</P></DIV><DIV class="Header1">
                Modeling Structures of AMPs
                </DIV><DIV class="subHead1"><P class="textProfile3">
                	To visually monitor the structures of AMPs, amino acid sequences were entered into the Protein Homology/analogY Recognition Engine (PHYRE2,Protein structure prediction on the web: a case study using the Phyre server
Kelley LA and Sternberg MJE. Nature Protocols 4, 363 - 371 (2009),http://www.sbg.bio.ic.ac.uk/~phyre2/html/page.cgi?id=index). Output for each of the AMPs used by Utah State iGEM 2013 team is given below. Note: Grammistin (BBa_K1162003) was too small to model with this program. Interestingly when amino acid sequence for each AMP was entered into the program fused with 10x Histag, no change was seen the in the protein structure. 
                	</P></DIV><DIV style="display: inline-block; width:800px;">            </DIV><P class="textProfile5">EcAMP-1 (BBa_K1162001)        Spheniscin (BBa_K1162002)      Cg-Defh1 (BBa_K1162004)</P><DIV style="width:800px;">                  <P class="textProfile5">
Scygonadin (BBa_K1162005)                                LL37(BBa_K1162006)

WAM1(BBa_K1162007)structure

OHCATH (BBa_K1162008)
</P><DIV class="Header1">
                Flux balance analysis (FBA) 
                </DIV><DIV class="subHead1"><P class="textProfile3">
                	The E. coli JO1366 reconstruction (or E. coli 1366 reconstruction) was used in this study. Reactions for target production of Grammistin, LL37, and WAM1 were added to the reconstruction by using the amino acid composition of each protein and then adding the appropriate number of ATP for protein synthesis (4.3 ATP/ amino acid). Demand reactions were also added for modelling purposes to allow for end metabolites (in this case final AMP proteins) to accumulate. Since these amino acid sequences were so small it was decided to only model three of the AMPs.

Flux balance analysis (FBA) was performed to optimize the production of these three targets(Grammistin, LL37, and WAM). To have some initial results to compare later results to, initial theoretical analysis was performed by setting the lower bound of the biomass reaction to -0.1 mmol/gDW-hr, and optimizing for the target reaction (Grammistin, LL37, and WAM1).  This procedure is based off of the work of Feist, et. al, 2010.

The carbon source that was modeled was glucose as this is the most commonly use carbon substrate. Since E. coli was chosen as the chassis for this project, aerobic conditions were also selected in the model. Target production for all three AMPs was maximized as this would be the main objective in a real world situation. In addition to setting the carbon substrate as glucose, media supplements were also modeled (in the form of 38 exchange reactions) to see the affect that additional metabolites would have on the final yield. In order to accomplish this, the lower bounds of the exchange reaction was set to -10 mmol/DW-hr. 

The fluxes were converted into product yield (Y) by dividing by the substrate (S) consumption rate, the product yield from the additional supplement analysis was compared to the initial analysis product yield.  In many cases, the product yield increased with the addition of these metabolites. Below is a table demonstrating the increase in yield of specific AMPs with the addition of metabolites. From the table it is seen that the addition of D-Fructose increasing the yield of Grammistin from 4.30% to 5.39%. The addition of L-Arginine would increase the yields of LL37 and WAM from 1.75% and 1.78% to 2.16% and 2.19% respectively. The addition of L-Arginine to produce more LL37 and WAM is understandable as it is contained in both AMPs.
                	</P></DIV><DIV class="Header1">
               Optimizing production of AMPs with OptKnock
                </DIV><DIV class="subHead1"><P class="textProfile3">
                	OptKnock, a program that optimizes the flux through an objective reaction by systematically searching for reactions to “knock out” (by setting the upper and lower bounds of those reactions to zero) was performed for Grammistin(BBa_K1162003), LL37 (BBa_K1162006), and WAM1(BBa_K1162007). 
 Production envelope for Grammistin containing 3 knockouts

From carrying out OptKnock it was found that it was not able to find an optimal/maximal production rate for Grammistin(maximum production rate= 2.83 E-10 or zero). The other AMPs analyzed (LL37 and WAM) yielded similar results to Grammistin and hence additional AMPs were not modeled using OptKnock as each AMP took many hours to analyze. From the OptKnock data, knocking out genes may not be a good solution to optimization of AMPs in E. coli. One possible explanation for the OptKnock program not giving optimized production rate from knockouts is, that the AMPs are so small that their production may not have any affect on the overall metabolic system of  E. coli.
                	</P></DIV><DIV class="Header1">
               References
                </DIV><DIV class="subHead1"><P class="textProfile3">
Feist, A. M.; Zielinski, D. C.; Orth, J. D.; Schellenberger, J.; Herrgard, M. J.; Palsson, B. Ø., Model-driven evaluation of the production potential for growth-coupled products of Escherichia coli. Metabolic Engineering 2010, 12 (3), 173-186.
					</P></DIV><DIV class="Header1">
               Image URL
                </DIV><DIV class="subHead1"><P class="textProfile3">
http://www.gelifesciences.com/webapp/wcs/stores/servlet/productById/en/GELifeSciences/17524802
					</P></DIV></DIV></DIV></DIV><DIV class="printfooter">
Retrieved from &quot;<A href="http://2013.igem.org/Team:Utah_State/Results">http://2013.igem.org/Team:Utah_State/Results</A>&quot;</DIV></DIV></DIV><DIV id="footer-box" class="noprint"><DIV id="footer"><UL id="f-list"><LI id="t-recentchanges"><A href="/Special:RecentChanges" title="Recent changes">Recent changes</A></LI><LI id="t-whatlinkshere"><A href="/Special:WhatLinksHere/Team:Utah_State/Results" title="List of all wiki pages that link here [j]" accesskey="j">What links here</A></LI><LI id="t-recentchangeslinked"><A href="/Special:RecentChangesLinked/Team:Utah_State/Results" title="Recent changes in pages linked from this page [k]" accesskey="k">Related changes</A></LI><LI id="t-specialpages"><A href="/Special:SpecialPages" title="List of all special pages [q]" accesskey="q">Special pages</A></LI><LI><A href="/Special:Preferences">My preferences</A></LI></UL></DIV><DIV id="footer"><UL id="f-list"><LI id="t-print"><A href="/wiki/index.php?title=Team:Utah_State/Results&amp;printable=yes" title="Printable version of this page [p]" accesskey="p">Printable version</A></LI><LI id="t-permalink"><A href="/wiki/index.php?title=Team:Utah_State/Results&amp;oldid=362361" title="Permanent link to this revision of the page">Permanent link</A></LI><LI id="privacy"><A href="/2013.igem.org:Privacy_policy" title="2013.igem.org:Privacy policy">Privacy policy</A></LI><LI id="disclaimer"><A href="/2013.igem.org:General_disclaimer" title="2013.igem.org:General disclaimer">Disclaimers</A></LI></UL></DIV></DIV></DIV></DIV></DIV></BODY></HTML>