"
Page
Discussion
View source
History
teams
Log in
Â 
Team:Harvard/color/notebook
From 2010.igem.org
home
project
vectors
allergy
flavor
color
genetic fence
human practices
safety
parts
results
people
students
teaching fellows
advisors
acknowledgements
gallery
online
blog
twitter
openwetware
facebook
contact us
abstract lab notebook pigment metabolism engineering pathways targeting the petals parts & primers results references
notebook
Week 6
-
-
-
-
07-23-2010
Week 7
07-26-2010
07-27-2010
07-28-2010
07-29-2010
07-30-2010
Week 8
-
-
08-04-2010
08-05-2010
08-06-2010
07-23-2010 [ top ]
Received primers for LUT2 and BETA-OHASE 1 - began building knockdown constructs
PCR reaction on RS300 to insert recognition sites. Primers listed below
LUT2 rxn 1: A, CP4
LUT2 rxn 2: CP2, CP3
LUT2 rxn 3: CP1, B
BETA-OHASE 1 rxn 1: A, CP8
BETA-OHASE 1 rxn 2: CP6, CP7
BETA-OHASE 1 rxn 3: CP5, B
PCR purified completed reaction
07-26-2010 [ top ]
Ran 4 ul of PCR reaction on a 1% agarose gel
[click to enlarge]
Bands appeared to be the correct size, so started PCR stitching process
Ran two PCR reactions, one for each of LUT2 and BETA-OHASE 1
Template: Rxn 1, 2, and 3 of the appropriate construct
Template diluted to 1 ng/ul, used 1 ul of each template
Primers: A and B for both reactions
Size expected to be around 450 bp
PCR purified and ran 4 ul on a 1% agarose gel
[click to enlarge]
PCR was not clean, decided to re-do stitching reaction with undiluted template (concentrations around 50 ng/ul, used 1 ul of each
Ran 4 ul on 1% agarose gel
[click to enlarge]
Observed clear band at correct location for BETA-OHASE 1, but not for LUT2
07-27-2010 [ top ]
Redid PCR stitching for LUT2, ran 4 ul on a gel
[click to enlarge]
Observed clear bands, including one in the right location
Ran entire reaction on a gel (half in each lane), cut out appropriate band and gel purified
[click to enlarge]
Digested gel purified LUT2, PCR purified BETA-OHASE 1, and B21 (Biobrick part in V0120 - to serve as backbone) with EcoRI and PstI to prepare for ligation of constructs into biobrick backbone
[click to enlarge]
LUT2 had the correctly sized band, but BETA-OHASE 1 had two bands, one at 400 and one at 100.
Checked sequences of primers, and determined that primers for BETA-OHASE 1 contained an EcoRI site
Ordered new primers without biobrick restriction sites for BETA-OHASE 1. Checked other primers for restriction sites, and found them to be clean.
07-28-2010 [ top ]
Gel purified LUT2 and top band from B21 (backbone)
Ligated LUT2 and backbone. Transformed into TURBO e. coli and plated on LB + Amp.
07-29-2010 [ top ]
Obtained colonies for LUT2. Set up four cultures in LB + Amp.
08-04-2010 [ top ]
Received new primers for BETA-OHASE 1 - began building knockdown constructs for that and LYC
PCR reaction on RS300 to insert recognition sites. Primers listed below
LYC rxn 1: A, CP12
LYC rxn 2: CP10, CP11
LYC rxn 3: CP9, B
BETA-OHASE 1 rxn 1: A, CP16
BETA-OHASE 1 rxn 2: CP14, CP15
BETA-OHASE 1 rxn 3: CP13, B
08-05-2010 [ top ]
PCR Stitching of LYC and BETA-OHASE 1 amiRNA parts; Digestion and Ligation into biobrick backbone
Ran PCR reaction on a 1% agarose gel.
[click to enlarge]
Initial PCR was successful. Gel purified parts
Performed PCR Stitching for each of BETA-OHASE 1 and LYC
Template: Rxn 1, 2, and 3 of the appropriate construct
Used 1 ul of each template - concentrations were all around 25 ng/ul
Primers: A and B for both reactions
Size expected to be around 450 bp
PCR purified and ran 4 ul on a 1% agarose gel
[click to enlarge]
PCR was successful. Gel purified products
Digested products with EcoRI and PstI to ligate into V0120 backbone.
[click to enlarge]
Successful digestion - gel purified and ligated into backbone digested EcoRI/PstI last week
Transformed into TURBO E. coli and plated on LB + Amp
PCR Amplification of petal promoters
Ran PCR to amplify petal promoters
Template: Arabidopsis genomic DNA, 80ng
Primers: CP17/CP18 (At1G11850), CP19/CP20 (At1G70720 1.1kb), CP21/CP22 (At1G70720 1.5kb)
Protocol: Phusion
Ran completed reaction on 1% agarose gel
[click to enlarge]
PCR Failed. Retry with PFx
Template: Arabidopsis genomic DNA, 80ng
Primers: CP17/CP18 (At1G11850), CP19/CP20 (At1G70720 1.1kb), CP21/CP22 (At1G70720 1.5kb)
Protocol: PFx, extension time 2 min
Ran completed reaction on 1% agarose gel
[click to enlarge]
08-06-2010 [ top ]
Got colonies for C3 (LYC) but not for C2 (BETA-OHASE 1). Picked four colonies and set up cultures for C3
Redid digestion on C2, and digested PCR products C4-6 with EcoRI and PstI. Also digested B21 with EcoRI and PstI with TSAP to obtain backbone.
[click to enlarge]
C2 and B21 digests worked, C4 had a PstI site inside the part and C5/C6 had EcoRI sites
Gel purified C2 and B21 (top) bands
Redigested C4-C6 with XbaI and SpeI, as well as B21 with XbaI and SpeI with TSAP to obtain backbone
[click to enlarge]
Gel ran strangely - ladder didn't show. Cut out bands and gel purified, then ran second gel to check sizes again
Miniprepped cultures set up in the morning, and sent out for sequencing
Retrieved from " http://2010.igem.org/Team:Harvard/color/notebook "
Recent changes
What links here
Related changes
Special pages
My preferences
Printable version
Permanent link
Privacy policy
Disclaimers
