"
Page
Discussion
View source
History
teams
Log in
 
Team:Harvard/results
From 2010.igem.org
home
project
vectors
allergy
flavor
color
genetic fence
human practices
safety
parts
results
people
students
teaching fellows
advisors
acknowledgements
gallery
online
blog
twitter
openwetware
facebook
contact us
vectors flavor genetic fence parts future directions
results
vectors
We modified six plant vectors to be compatible with BioBrick Standard 21. We successfully cloned several parts into the expression series vectors V9 (BBa_K382002) and V10 (BBa_K382003), transformed the plasmids into Agrobacterium, and grew colonies that showed proper antibiotic resistance.
open series   click to enlarge
The open series vectors are designed for general insertion of a construct. We modified the vectors pORE O1 and pORE O2. pORE O1 confers plant resistance to glufosinate , and pORE O2 confers plant resistance to kanamycin , to be used in transformant selection.
reporter series   click to enlarge
The reporter series vectors contain a reporter on the trailing end of the multiple cloning site such that expression of the reporter follows that of the inserted construct. We modified the vectors pORE R1 and pORE R3. pORE R1 contains the gusA reporter, and pORE R3 the smgfp reporter. Both vectors confer plant resistance to kanamycin.
expression series   click to enlarge
The expression series vectors contain a promoter preceding the multiple cloning site such that the inserted construct can be easily expressed through activation of the contained promoter. We modified the vectors pORE E3 and pORE E4. Both vectors contain the ENTCUP2 promoter. V9 (pORE E3) confers plant resistance to glufosinate, and V10 (pORE E4) confers plant resistance to kanamycin
Source: Coutu, Catherine et al. "pORE: a modular binary vector series suited for both  monocot and dicot plant transformation." Transgenic Res (2007) 16:771–781.
flavor
The two flavors that are currently ready for transformation into plants are the "taste-inverter" miraculin (BBa_K382021) and the sweetener brazzein (BBa_K382020).  Given the long time-frame of plant transformation we used two different assays in E. Coli to confirm that our proteins could indeed be transcribed and translated. The results of those assays are shown here.
confirmation with 2xYFP tags
In order to confirm that the miraculin and brazzein are able to be expressed in E. Coli we attached a 2xYFP tag sequence to both the N- and C-terminus of each protein.  The proteins were placed under an IPTG-expressible promoter and used spectrophotometry to determine the level of YFP fluorescence against a baseline, untagged protein.  Figure 1 shows relative-fluorescence at times post induction.  In all circumstances the levels of YFP-fluorescence increased.
Figure 1   click to enlarge
Figure 1. Induced expression of YFP-tagged Miraculin and Brazzein in E. Coli Figure 1 (A) through (D) are normalized plots of miraculin and brazzein YFP-fused constructs expressed in E. Coli. 2xYFP tags were attached to either the N- or C- terminus to ensure that folding was not hindered.  In all cases relative YFP fluorescence had appreciably increased after 120 minutes as compared to the non-induced E. Coli
confirmation by western blot
A western blot assay was performed to check for E. Coli expression of miraculin and brazzein.  Proteins tagged at either the N- or C- terminus were placed under the control of an IPTG-inducible promoter.  In the miraculin assay, no protein expression was seen.  It is possible that the protein does not express well in E. Coli, or that the plant-specific codon optimization of the proteins resulted in reduced expressibility.  Brazzein, specifically C-terminus tagged brazzein was seen to be highly expressed in E. Coli.
Figure 2   click to enlarge
Figure 2. Western Blot of Miraculin and Brazzein Expression in E. Coli Proteins were tagged using a StrepII standard Antibody tag, attached to both N- and C- termini. Miraculin (A) does not appear to have been expressed in high enough quantities to be visualized. The expected protein weight is 25 kDa.  Brazzein (B) shows strong expression of a protein in the 10-15 kDa range.  Brazzein has an expected weight of 6.5 kDa, a discrepancy that we have attributed to inconsistencies in the gel.
genetic fence
induction of barnase (death gene) reduces cell growth
We characterized the activity of Barnase on an inducible plasmid constructed by UC Berkeley for iGEM 2007 (part I716408C). This contruct works by expressing background levels of Barstar with Barnase controlled by an arabinose inducible promoter such that it will overwhelm Barstar when induced. Higher levels of Barnase expression resulted in lower rates of growth in the bacteria, affirming the principle of Barnase-based growth control for the genetic fence, and confirming the results from Berkeley 2007. We characterized the growth repression of Barnase under a range of arabinose inducer concentrations.
Our results show that expression of Barnase is effective in reducing cell growth, suggesting that Barnase will enable the genetic fence to prevent growth of iGarden plants outside of their designated areas.
barnase growth control in E. Coli   [click to enlarge]
Parts transferred to the Agrobacterium shuttle chassis
We transformed 11 completed vectors into Agrobacterium and successfully isolated clones:
flavor parts
miraculin expression
brazzein expression
expression in arabidopsis
We are still waiting for the plants to grow to a size large enough that we can collect samples to verify expression, but we have selected for plants that have integrated the herbicide resistance marker along with the miraculin and brazzein expression constructs.
Miraculin:
Brazzein:
RNAi knockdown controls
amiRNA GFP knockdown: this vector will allow us to visualize RNAi knock-down of fluorescence
allergy parts for RNAi targeting of several panallergen homologs in Arabidopsis
LTP amiRNA
LTP hpRNA
Ger3 hpRNA
Bet v 1 hpRNA
expression in arabidopsis (Kan resistance)
LTP amiRNA
Ger3 hpRNA
color parts
LUT2 amiRNA: lycopene accumulation and red flowers
LYC amiRNA: lycopene accumulation and red flowers
Beta Ohase I amiRNA: beta carotene accumulation and orange flowers
expression in arabidopsis (Kan resistance)
Beta Ohase I
All 11 vectors were transformed into Arabidopsis, the expression chassis. A complete list of these parts and other parts built and submitted to the registry, please check out our parts page.
future directions
Because plants take a long time to grow, we were unfortunately unable to verify the function of our parts in Arabidopsis. As soon as we have a sufficient amount of plant tissue, we can confirm that the plants growing on selective plates are transformed via PCR. Alternatively, the GFP knockdown plants should be identifiable by their loss of fluorescence. Stay updated with our results after the Jamboree by checking out our
OpenWetWare page .
Retrieved from " http://2010.igem.org/Team:Harvard/results "
Recent changes
What links here
Related changes
Special pages
My preferences
Printable version
Permanent link
Privacy policy
Disclaimers
