<HTML xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en" dir="ltr">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki  ltr ns-0 ns-subject page-Team_Harvard_fences_notebook"><DIV id="globalWrapper"><DIV id="top-section"><DIV id="p-logo"><A href="/Main_Page" title="Main Page">&quot;
	    </A></DIV><DIV id="menubar" class="left-menu noprint"><UL><LI class="selected"><A href="/Team:Harvard/fences/notebook">Page               </A></LI><LI class="new"><A href="/wiki/index.php?title=Talk:Team:Harvard/fences/notebook&amp;action=edit&amp;redlink=1">Discussion               </A></LI><LI><A href="/wiki/index.php?title=Team:Harvard/fences/notebook&amp;action=edit">View source               </A></LI><LI><A href="/wiki/index.php?title=Team:Harvard/fences/notebook&amp;action=history">History               </A></LI><LI style="color:#808080;cursor:default">teams</LI></UL></DIV><DIV class="right-menu noprint" id="menubar"><UL><LI id="pt-login"><A href="/wiki/index.php?title=Special:UserLogin&amp;returnto=Team:Harvard/fences/notebook" title="You are encouraged to log in; however, it is not mandatory [o]" accesskey="o">Log in</A></LI></UL></DIV><DIV id="search-controls" class="noprint"><FORM action="/Special:Search" id="searchform"> </FORM></DIV></DIV><DIV id="content"><H1 class="firstHeading">Team:Harvard/fences/notebook</H1><DIV id="bodyContent"><H3 id="siteSub" class="noprint">From 2010.igem.org</H3><DIV id="allcontent"><DIV id="headerlinks"><UL id="nav"><LI><A class="bannertoplinks" href="https://2010.igem.org/Team:Harvard">home</A></LI><LI><A class="bannertoplinks" href="#">project</A></LI><LI><A class="bannerlinks" href="https://2010.igem.org/Team:Harvard/vectors">vectors</A></LI><LI><A class="bannerlinks" href="https://2010.igem.org/Team:Harvard/allergy">allergy</A></LI><LI><A class="bannerlinks" href="https://2010.igem.org/Team:Harvard/flavor">flavor</A></LI><LI><A class="bannerlinks" href="https://2010.igem.org/Team:Harvard/color">color</A></LI><LI><A class="bannerlinks" href="https://2010.igem.org/Team:Harvard/fences">genetic fence</A></LI><LI><A class="bannerlinks" href="https://2010.igem.org/Team:Harvard/human_practices">human practices</A></LI><LI><A class="bannerlinks" href="https://2010.igem.org/Team:Harvard/safety">safety</A></LI><LI><A class="bannerlinks" href="https://2010.igem.org/Team:Harvard/parts">parts</A></LI></UL><LI><A class="bannertoplinks" href="https://2010.igem.org/Team:Harvard/results">results</A></LI><LI><A class="bannertoplinks" href="#">people</A></LI><LI><A class="bannerlinks" href="https://2010.igem.org/Team:Harvard/students">students</A></LI><LI><A class="bannerlinks" href="https://2010.igem.org/Team:Harvard/tfs">teaching fellows</A></LI><LI><A class="bannerlinks" href="https://2010.igem.org/Team:Harvard/advisors">advisors</A></LI><LI><A class="bannerlinks" href="https://2010.igem.org/Team:Harvard/credits">acknowledgements</A></LI><LI><A class="bannerlinks" href="https://2010.igem.org/Team:Harvard/gallery">gallery</A></LI><LI><A class="bannertoplinks" href="#">online</A></LI><LI><A class="bannerlinks" href="http://harvardigem.org">blog</A></LI><LI><A class="bannerlinks" href="http://twitter.com/harvardigem">twitter</A></LI><LI><A class="bannerlinks" href="http://openwetware.org/wiki/IGEM:Harvard/2010">openwetware</A></LI><LI><A class="bannerlinks" href="http://www.facebook.com/pages/Cambridge-MA/Harvard-iGEM/269794024286?ref=ts">facebook</A></LI><LI><A class="bannerlinks" href="https://2010.igem.org/Team:Harvard/contact">contact us</A></LI></DIV><DIV id="sidebar"><A class="sidebarlinks" href="https://2010.igem.org/Team:Harvard/fences" style="color:white">genetic fence</A><A class="sidebarlinks" href="https://2010.igem.org/Team:Harvard/fences/notebook" style="color:white">lab notebook</A><A class="sidebarlinks" href="https://2010.igem.org/Team:Harvard/fences/design" style="color:white">design</A><A class="sidebarlinks" href="https://2010.igem.org/Team:Harvard/fences/safety" style="color:white">safety</A><A class="sidebarlinks" href="https://2010.igem.org/Team:Harvard/fences/parts" style="color:white">parts &amp; primers</A><A class="sidebarlinks" href="https://2010.igem.org/Team:Harvard/fences/results" style="color:white">results</A><A class="sidebarlinks" href="https://2010.igem.org/Team:Harvard/fences/references" style="color:white">references</A></DIV><H1><A name="notebook">notebook</A></H1><TABLE style="font-size:14px;text-align:center;width:550px;" border="0" cellspacing="6"><TBODY><TR><TD> Week 1  </TD><TD><A href="#06-14-2010">6-14-2010</A></TD><TD><A href="#06-15-2010">06-15-2010</A></TD><TD><A href="#06-16-2010">06-16-2010</A></TD><TD><A href="#06-17-2010">06-17-2010</A></TD><TD><A href="#06-18-2010">06-18-2010</A></TD></TR><TR><TD> Week 2  </TD><TD><A href="#06-21-2010">06-21-2010</A></TD><TD><A href="#06-22-2010">06-22-2010</A></TD><TD><A href="#06-23-2010">06-23-2010</A></TD><TD><A href="#06-24-2010">06-24-2010</A></TD><TD> -
</TD></TR><TR><TD> Week 3  </TD><TD><A href="#06-28-2010">06-28-2010</A></TD><TD><A href="#06-29-2010">06-29-2010</A></TD><TD><A href="#06-30-2010">06-30-2010</A></TD><TD><A href="#07-01-2010">07-01-2010</A></TD><TD><A href="#07-02-2010">07-02-2010</A></TD></TR><TR><TD> Week 4  </TD><TD> - </TD><TD><A href="#07-06-2010">07-06-2010</A></TD><TD><A href="#07-07-2010">07-07-2010</A></TD><TD><A href="#07-08-2010">07-08-2010</A></TD><TD><A href="#07-09-2010">07-09-2010</A></TD></TR><TR><TD> Week 5  </TD><TD><A href="#07-12-2010">07-12-2010</A></TD><TD><A href="#07-13-2010">07-13-2010</A></TD><TD><A href="#07-14-2010">07-14-2010</A></TD><TD><A href="#07-15-2010">07-15-2010</A></TD><TD><A href="#07-16-2010">07-16-2010</A></TD></TR><TR><TD> Week 6  </TD><TD> - </TD><TD> - </TD><TD><A href="#07-21-2010">07-21-2010</A></TD><TD><A href="#07-22-2010">07-22-2010</A></TD><TD><A href="#07-23-2010">07-23-2010</A></TD></TR><TR><TD> Week 7  </TD><TD> - </TD><TD><A href="#07-27-2010">07-27-2010</A></TD><TD><A href="#07-28-2010">07-28-2010</A></TD><TD><A href="#07-29-2010">07-29-2010</A></TD><TD><A href="#07-30-2010">07-30-2010</A></TD></TR><TR><TD> Week 8  </TD><TD><A href="#08-02-2010">08-02-2010</A></TD><TD><A href="#08-03-2010">08-03-2010</A></TD><TD><A href="#08-04-2010">08-04-2010</A></TD><TD><A href="#08-05-2010">08-05-2010</A></TD><TD><A href="#08-06-2010">08-06-2010</A></TD></TR><TR><TD> Week 9  </TD><TD><A href="#08-09-2010">08-09-2010</A></TD><TD><A href="#08-10-2010">08-10-2010</A></TD><TD><A href="#08-11-2010">08-11-2010</A></TD><TD><A href="#08-12-2010">08-12-2010</A></TD><TD><A href="#08-13-2010">08-13-2010</A></TD></TR></TBODY></TABLE><DIV id="entry"><SPAN id="date"><A name="06-14-2010" class="date">06-14-2010</A></SPAN> [<A class="top" href="#notebook">top</A>]</DIV><P><B>LacI Transformation</B></P><P>Performed bacterial transformation according to <A href="http://openwetware.org/wiki/Silver:_Bacterial_Transformation" class="external text" rel="nofollow">Silver Lab Bacterial Transformation</A>, however we used 1μL in step 3, and used entire tube of TOP10 competent cells instead of only 10-15μL.
</P><P>Transformed the following, each into its own tube of TOP10 E.Coli:
</P><P>1) LacI with a rapid degredation tail attached, biobrick BBa_C0012, located on plate 1, well 2O, plasmid pSB1A2
</P><P>2) LacI wildtype, biobrick BBa_I732100, located on plate 2, well 10E, plasmid pSB1A3
</P><P>Streaked 100μL each of 1 and 2 onto separate LB + Amp dishes, along with a control dish of 100μL untransformed TOP10 E.Coli cells.
</P><P>Plates incubated overnight, colonies observed in all plates except the control.
</P><P><TABLE><TBODY><TR><TD><DIV><A href="http://openwetware.org/images/6/65/Colonies_6-15-10.jpg" id="single_image"> [click to enlarge]</A></DIV></TD><TD><DIV><A href="http://openwetware.org/images/1/1c/Colonies-2_6-15-10.jpg" id="single_image"> [click to enlarge]</A></DIV></TD><TD><DIV><A href="http://openwetware.org/images/3/33/Colonies-3_6-15-10.jpg" id="single_image"> [click to enlarge]</A></DIV></TD></TR></TBODY></TABLE></P><DIV id="entry"><SPAN id="date"><A name="06-15-2010" class="date">06-15-2010</A></SPAN> [<A class="top" href="#notebook">top</A>]</DIV><P><B>Oligonucleotides for LacI</B>
the following sequences for LacI and a Nuclear Localization Signal (NLS) were ordered for PCR:
</P><P>LacIn.BB.Rev
</P><PRE> 5'-AAG GCT GCA GCG GCC GCT ACT AGT TCA AAC CTT TCT CTT CTT CTT AGG ATG AAC AAC AGA AGA CTG CCC GCT TTC CAG TCG GGA AA - 3'
</PRE><P>LacIn.BB.Fwd
</P><PRE>   5'- CCT TGA ATT CGC GGC CGC ATC TAG AAT GAA ACC AGT ACC GTT ATA CGA TGT C -3'
</PRE><P>NLS.BB.Rev
</P><PRE>   5'-GCG GCC GCT ACT AGT TCA AAC CTT TCT CTT CTTCTT AGG ATG AAC AAC AGA AGA -3'
</PRE><P>NLS.BB.Fwd 
</P><PRE>   5'-CTA GAT CTT CTG TTG TTC ATC CTA AGA AGA AGA GAA AGG TTT GAA CTA GTA GCG GCC GCT GCA -3'
</PRE><P><B>VP16 transcription activating domain bacterial transformation</B></P><P>Performed bacterial transformation according to <A href="/wiki/index.php?title=Silver:_Bacterial_Transformation&amp;action=edit&amp;redlink=1" class="new" title="Silver: Bacterial Transformation (page does not exist)">Silver:_Bacterial_Transformation</A>, however we used 1μL in step 3, and used entire tube of TOP10 competent cells instead of only 10-15μL.
</P><P>Transformed the following into its own tube of TOP10 E.Coli:
</P><P>VP16 Eukaryotic Activating Domain, biobrick BBa_K105001, located on plate 3, well 13C, plasmid pSB1A2.
</P><P>In the course of reconsitituting the the biobrick plasmid from its well, too little DH<SUB>2</SUB>O was added (1μL), so an additional 10μL were added to the well.
</P><P>The first tube of TOP10+biobrick plasmid was erroneously heatshocked before being chilled for 30 mins on ice, so a second tube of TOP10 cells was introduced, and the correct proceedure followed for the second tube. Additionally, the first tube was then allowed to chill and heat shock again, ultimately producing two tubes of transformed TOP10 cells.
</P><P>100μL of both tubes were streaked onto an LB + Amp dish, and incubated at 37°C on the plates overnight. Colonies the following morning, colonies were observed on both plates, although fewer on the plate containing the E.Coli which were erroneously heat-shocked too soon.
</P><P><B>LacI Miniprep Preparations</B>
Prepared 6 overnight cell cultures, consisting of three colonies each from the LacI wt and LacI+rapid degradation tail E.Coli cells transformed previously.
</P><P>Each culture consisted of 5mL of LB+Amp, and a scraping of 1 E.Coli colony, in a 100cc tube.
</P><P>These overnight cell cultures were set to shake at 37°C overnight.
</P><DIV id="entry"><SPAN id="date"><A name="06-16-2010" class="date">06-16-2010</A></SPAN> [<A class="top" href="#notebook">top</A>]</DIV><P><B>LacI Miniprep</B></P><P>Performed miniprep of overnight cell cultures of 3 colonies each of wildtype LacI transformed bacteria and LacI+rapid degradation tail bacteria.
</P><P>Minipreps performed according to pages 22-23 of the <A href="http://www.google.com/url?sa=t&amp;source=web&amp;cd=1&amp;ved=0CBIQFjAA&amp;url=http%3A%2F%2Fkirschner.med.harvard.edu%2Ffiles%2Fprotocols%2FQIAGEN_QIAprepMiniprepKit_EN.pdf&amp;ei=wOcbTOfxCIH-8AaIhI2ADA&amp;usg=AFQjCNENLjIQI2lUNMlpIqkcKrrq9buqLg&amp;sig2=GVeWspXGiA0bwGnulA18-Q" class="external text" rel="nofollow">QIAprep Miniprep Handbook</A>.
</P><P><B>LacI Nanodrop</B>
Blanked nanodrop sensor with 2μL EB buffer, then measured each of the 6 minipreped LacI plasmids.
</P><TABLE border="3" style="margin-left: 3em;"><TBODY><TR><TH> Plasmid </TH><TH> Quantity (ng/μL) </TH><TH> 260/280
</TH></TR><TR><TH> O2 #1
</TH><TD> 77.1 </TD><TD> 1.81
</TD></TR><TR><TH> O2 #2
</TH><TD> 90.1 </TD><TD> 1.9
</TD></TR><TR><TH> O2 #3
</TH><TD> 50.4 </TD><TD> 1.95
</TD></TR><TR><TH> E10 #1
</TH><TD> 99.6 </TD><TD> 1.93
</TD></TR><TR><TH> E10 #2
</TH><TD> 104.6 </TD><TD> 1.92
</TD></TR><TR><TH> E10 #3
</TH><TD> 58.6 </TD><TD> 1.98
</TD></TR></TBODY></TABLE><P><B>LacI Plasmid Digest</B>
We digested 15μL of each of the 6 samples with 1μLH<SUB>2</SUB>O, 1μL EcoR1, and 1μL Pst1 (high efficiency restriction enzymes).
</P><P>Ran an E-gel of the digest results.
</P><TABLE border="3" style="margin-left: 3em;"><TBODY><TR><TH> Lane # </TH><TH> 1 </TH><TH> 2 </TH><TH> 3 </TH><TH> 4 </TH><TH> 5 </TH><TH> 6 </TH><TH> 7 </TH><TH> 8 </TH><TH> 9 </TH><TH> 10
</TH></TR><TR><TH> O2 #1
</TH><TD>ladder </TD><TD> O2 #1 </TD><TD> O2 #2 </TD><TD> O2 #3 </TD><TD> E10 #1 </TD><TD> E10 #2 </TD><TD> E10 #3 </TD></TR></TBODY></TABLE><DIV><A href="http://openwetware.org/images/e/e4/LacI_digest_Gel_2010-06-16_16hr_34min.jpg" id="single_image"> [click to enlarge]</A></DIV><P><B>VP16 Colony Inoculation</B>
Prepared 3 10cc tubes each containing .5μL LB+amp and cells from one of the three colonies of the VP16 cell culture of the previous night (colonies from the second-tube plate, the cells in which were heatshocked in the correct order).
</P><P>Tubes were set to shake at 37°C overnight.
</P><P><B>Preparing Glycerol Stocks of LacI wt and LacI+LVA</B>
Prepared 2 cryo tubes, with the following labeling:
</P><P>Tube 1) contains BBa_C0012, from plate 1, well 2O
</P><P>On top: '#1'
</P><P>On side: '6-16-10 MP pSB1A2 LacI+LVA'
</P><P>Tube 2) contains BBa_I732100, from plate 2, well 10E
</P><P>On top: '#2'
</P><P>On side: '6-16-10 MP pSB1A3 LacI Wildtype'
</P><P>Each cryo tube contains .5mL of the respective overnight cell culture (O2 colony #1 and E10 colony #1), and .mL of 80% glycerol.
</P><P>Both Cryo tubes were placed in the box labeled 'Genetic Fence iGEM 2010' in the -80°C freezer.
</P><DIV id="entry"><SPAN id="date"><A name="06-17-2010" class="date">06-17-2010</A></SPAN> [<A class="top" href="#notebook">top</A>]</DIV><P><B>VP16 Miniprep</B></P><P>Following QIAgen Plasmid DNA Purification Protocol Using the QIAprep Spin Miniprep Kit and a Microcentrifuge the following plasmid was purified:
</P><UL><LI>VP16</LI></UL><P><B>Protocol</B></P><UL><LI>4ml from each overnight cell culture of VP16 (1,2&amp;3) were transferred into 5 ml conicals and spun for 6 minutes at 4400 rpm</LI><LI>remainder of overnight cell cultures were placed in the fridge for transfer to microcentrifuge tubes at a later time</LI><LI>excess LP+amp decanted, the pellet resuspended in 250μL P1</LI><LI>contents transferred from each 15ml conical to a new epindorf tube</LI><LI>250 μL P2 added to each tube, mixed by inverting gently 4-6 times</LI><LI>350 μL N3 added to each tube, mixed by inverting as before</LI><LI>tubes centrifuged for 10 minutes at 13,000 rpm</LI><LI>supernatants pipetted into new QIAprep sin columns</LI><LI>columns centrifuged for 30-60 seconds, flow through discarded</LI><LI>QIAprep columns washed with 0.5 ml buffer PB each and centrifuged for 30-60 seconds</LI><LI>columns washed with 0.75 ml buffer PE and again centrifuged for 30-60 seconds</LI><LI>flow through was discarded and tubes centrifuged for an additional minute</LI><LI>column was placed in a clean 1.5 ml microcentrifuge tube</LI><LI>50 μL buffer EB was added, tubes were let stand for 1 minute and centrifuged for one minute after</LI><LI>flow through kept and purity tested</LI></UL><P><B>Nanodrop Values for VP16</B></P><UL><LI>VP16 #1
</LI><LI>95.7 ng/μL
</LI><LI>260/280 = 1.92
</LI></UL><UL><LI>VP16 #2
</LI><LI>102.4 nm/μL
</LI><LI>260/280 = 1.92
</LI></UL><UL><LI>VP16 #3
</LI><LI>90.7 ng/μL
</LI><LI>260/280 = 1.92
</LI></UL><P><B>Digestion</B></P><P>Quantities per double digestion reaction
</P><UL><LI>10μL sample
</LI><LI>1μL EcoRI
</LI><LI>1μL PSD1
</LI><LI>2μL buffer
</LI><LI>6μL H20
</LI></UL><P>Single digest reaction
</P><UL><LI>10μL sample
</LI><LI>1μL EcoRI
</LI><LI>2μL buffer
</LI><LI>7μL H20
</LI></UL><P>ingredients combined and all tubes were put in 37°C bath for 10 minutes
</P><P>Digest with SPE1 and XBA1
</P><UL><LI>10μL sample
</LI><LI>1μL SPE1
</LI><LI>1μL XBA1
</LI><LI>2μL buffer
</LI><LI>6μL H20
</LI></UL><UL><LI>All 3 epindorfs of VP16 were combined for a total volume of 90 μL or 9μg
</LI><LI>1 unit enzyme cuts 1 μg DNA in 30-60 minutes
</LI></UL><P><B>Digestion Gel</B></P><P>Ingredients
</P><UL><LI>digested enzymes from above reaction
</LI><LI>1 eppendorf containing 2μL uncut VP16 and 18μL H2O
</LI><LI>kb ladder
</LI><LI>IMAGE TO BE ADDED + details
</LI></UL><P>Gel Recipe: 1 X TAE
</P><UL><LI>900 ml dH2O
</LI><LI>100 ml TAE 
</LI></UL><P>1 gel
</P><UL><LI>150 ml buffer TAE
</LI><LI>add ~2.8g agarose
</LI><LI>mix in beaker
</LI><LI>microwave until completely dissolved
</LI><LI>add 5 μL ethedium bromide per 100 ml gel after solution has cooled (no longer steaming)
</LI><LI>pour gel and let sit til solidified
</LI></UL><P>Procedure
</P><UL><LI>run products on gel for ~45 minutes
</LI><LI>cut out band containing DNA fragment
</LI><LI>follow extraction protocol in QIAquick spin handbook for QIAquick gel extraction kit protocol
</LI></UL><P>Nanodrop values for resulting DNA
</P><UL><LI>5.6ng/μL
</LI><LI>260/280 = 2.24
</LI></UL><P>Tube placed in -20 freezer labeled VP1 fragment XBA SPE1
</P><P>
Glycerol Stocks of VP16
</P><UL><LI>0.5 ml 80% glycerol stock combined with 0.5 ml overnight culture of colony #2 of V{16 TP{10 cells
</LI><LI>final product placed in the team fence box after mixing in the -80°C freezer
</LI></UL><P><B>Bacterial transformation of GAL4 DNA binding domain</B></P><P>BBa_K105007, plate 3 well 9I, psB1A2
</P><P>Procedure
</P><UL><LI>removed 1 tube TOP10 chemically competent E.Coli and placed on ice
</LI><LI>cleaned biobrick depository w/ ethanol
</LI><LI>pipetted 1 μL of plasmid into the TOP10 cell tube
</LI><LI>placed TOP10 cell tube on ice for 30 minutes
</LI><LI>heat shocked at 42°C for 30 seconds
</LI><LI>placed on ice for 2 minutes
</LI><LI>added 170 μL SOC medium
</LI><LI>streaked on LB + amp paltes, incubated at 37°C overnight
</LI></UL><DIV id="entry"><SPAN id="date"><A name="06-18-2010" class="date">06-18-2010</A></SPAN> [<A class="top" href="#notebook">top</A>]</DIV><P><B>LacI+NLS PCR</B></P><P>LacI+NLS primers arrived in mail
</P><P>spun at 14,000 rpm for 10 minutes
diluted stocks to 100 μM
</P><PRE>   LacIn.BB.Rev (1.7nM) diluted with 17μL DH<SUB>2</SUB>O
   LacIn.BB.Fwd (1.5nM) diluted with 1011μL DH<SUB>2</SUB>O
   NLS.BB.Rev (4.2nM) diluted with 42μL DH<SUB>2</SUB>O
   LacIn.BB.Fwd (101.1nM) diluted with 15μL DH<SUB>2</SUB>O
</PRE><P>PCR recipe
</P><UL><LI>20μL per reaction
</LI><LI>1μL template plasmid
</LI><LI>1μL R primer
</LI><LI>1μL F primer
</LI><LI>4μL HF buffer 5X
</LI><LI>2μL DNTPs
</LI><LI>0.5μL pFU polymerase
</LI><LI>rest H2O to make the volume per tube 20μL
</LI></UL><P>
used LACIN program in phusion PCR machine to perform PCR
</P><DIV id="entry"><SPAN id="date"><A name="06-21-2010" class="date">06-21-2010</A></SPAN> [<A class="top" href="#notebook">top</A>]</DIV><P><B>Barnase and Barstar</B></P><P>Barnase and Barstar Plasmids from ADDGENE arrived in mail
</P><P>pMT316 -
Barstar.
<A href="http://www.addgene.org/pgvec1?f=c&amp;identifier=8608&amp;atqx=pMT316&amp;cmd=findpl" class="external text" rel="nofollow">Info from ADDGENE</A></P><P>pMT413 -
Barnase, Barstar.
<A href="http://www.addgene.org/pgvec1?f=c&amp;identifier=8606&amp;atqx=pMT413&amp;cmd=findpl" class="external text" rel="nofollow">Info from ADDGENE</A></P><P>pMT1002 -
Barnase, Barstar.
<A href="http://www.addgene.org/pgvec1?f=c&amp;identifier=8621&amp;atqx=pMT413&amp;cmd=findpl" class="external text" rel="nofollow">Info from ADDGENE</A></P><P><B>Notes from lab meeting</B></P><P>Karmella suggested we try adding an LVA tail to the Cre we use for the crelox fence system.
</P><P><B>GAL4 DBD Innoculation</B></P><P>Pipetted 5mL of LB+Amp each into capped cc tubes, and mixed bacterial scrapings from 5 separate colonies of transformed GAL4 DNA binding domain E.Coli. Set to shake at 37 degrees C overnight.
</P><DIV id="entry"><SPAN id="date"><A name="06-22-2010" class="date">06-22-2010</A></SPAN> [<A class="top" href="#notebook">top</A>]</DIV><P><B>GAL4DBD Miniprep</B></P><DIV><A href="http://openwetware.org/images/1/12/062210_vector2.jpg" id="single_image"> [click to enlarge]</A></DIV><P>HindIII + PstI digest of miniprepped V9-V12 (#5,6) to check whether insert is present.]
</P><UL><LI>pipetted 4 ml from each overnight cell culture from colonies 1-5 into a 15 ml conical centrifuged at 4400 rpm for 6 minute
</LI><LI>remaining overnight cell cultures were placed in fridge
</LI><LI>decanted LB+amp, resuspended cells in 250 μL P1 buffer
</LI><LI>contents transferred to eppendorfs
</LI><LI>250 μL of P2 buffer was added per tube, the tubes were inverted 4-6 times
</LI><LI>350 μL N3 buffer added to each, tubes inverted 4-6 times
</LI><LI>centrifuged for 10 min at 13,000 rpm, supernatant collected ad transferred to QIAprep spin columns
</LI><LI>centrifuged for 30-60 seconds, flow through discarded
</LI><LI>0.5 ml buffer PB added and centrifuged for 30-60 seconds, flow through discarded, spun columns for an additional minute
</LI><LI>QIAprep columns put into new eppendorfs
</LI><LI>50 μL buffer EB added to columns, let stand for 1 minute
</LI><LI>centrifuged for 1 minute at 13,000 rpm
</LI><LI>tubes labeled &quot;GAL4&quot; plus the specific colony number
</LI></UL><P><B>PCR</B></P><P>PCR program for LacIN modified after Monday's unsuccessful O2 and E10 PCR (no product)
</P><P>1=95°C for 10:00
</P><P>2=95°C for 00:15
</P><P>3=50°C for 00:30
</P><P>4=72°C for 01:30
</P><P>5=steps 2-4 X 29
</P><P>6=72°C for 10:00
</P><P>7=4°C for ∞
</P><DIV><A href="http://openwetware.org/images/8/85/File.jpg" id="single_image"> [click to enlarge]</A></DIV>

PCR successful
<P>PCR for E10 LacI
</P><UL><LI>above PCR was repeated without O2 so as to finalize our E10 biobrick for use
</LI><LI>PCR was performed using the standard proportions of reagents
</LI><LI>The above settings on the PCR machine were used
</LI></UL><DIV><A href="http://openwetware.org/images/c/ce/Cutout622.jpg" id="single_image"> [click to enlarge]</A></DIV>

Gel cut-out
<P>To Make Agarose Gel
</P><UL><LI>150 mL TAE buffer combined with 1.5 g agarose in beaker
</LI><LI>solution microwaved until agarose disolved
</LI><LI>7.5 μL Ethidium Bromide added after beaker was no longer steaming
</LI><LI>solution poured into clean gel mold and gel was left to solidify
</LI><LI>E10 PCR product from the morning was consolidated into one tube, 
</LI><LI>final volume =90μL PCR product + 18μL 5X loading dye =110μL
</LI></UL><P>
DNA was extracted from gel by slicing out the bands and DNA was purified using the QIAquick Gel Extraction Kit, following protocol specific to using a centrifuge
</P><P>see <A href="http://www.qiagen.com/products/dnacleanup/gelpcrsicleanupsystems/qiaquickgelextractionkit.aspx#Tabs=t2" class="external text" rel="nofollow">QIAquick spin handbook</A></P><P><B>Innoculations</B></P><P>Colonies from the following cultures were innoculated and set to shake at 37°C overnight:
</P><UL><LI>PMT413
</LI><LI>PMT316
</LI><LI>firefly luciferase
</LI><LI>GFP (mut)
</LI><LI>cre recombinase
</LI><LI>lox66
</LI><LI>lox71
</LI></UL><DIV id="entry"><SPAN id="date"><A name="06-23-2010" class="date">06-23-2010</A></SPAN> [<A class="top" href="#notebook">top</A>]</DIV><P><B>Minipreps</B>
The following colonies were minipreped after passing the night in the shaking incubator (37°C)
</P><UL><LI> PMT413
</LI><LI> PMT316
</LI><LI> firefly luciferase
</LI><LI> GFP (mut)
</LI><LI> cre recombinase
</LI><LI> lox66
</LI><LI> lox71
</LI></UL><P><B>Nanodrops for above minipreps</B></P><P>Blanked nanodrop sensor with 2μL EB buffer, then measured each of the 6 minipreped LacI plasmids.
</P><P>If two numbers are given, separated by a slash, then two measurements were taken, both are presented.
</P><TABLE border="3" style="margin-left: 3em;"><TBODY><TR><TH> Plasmid </TH><TH> Quantity (ng/μL) </TH><TH> 260/280
</TH></TR><TR><TH> Lox 71 #1
</TH><TD> 41.4 </TD><TD> 1.82
</TD></TR><TR><TH> Lox 71 #2
</TH><TD> 59.6 </TD><TD> 1.89
</TD></TR><TR><TH> Lox 71 #3
</TH><TD> 17.4/80.2 </TD><TD> 1.98
</TD></TR><TR><TH> GFP #1
</TH><TD> 21.2 </TD><TD> 1.83
</TD></TR><TR><TH> GFP #2
</TH><TD> 110.0 </TD><TD> 1.90
</TD></TR><TR><TH> GFP #3
</TH><TD> 60.9/124 </TD><TD> 1.88
</TD></TR><TR><TH> pMT316 #1
</TH><TD> 11.6 </TD><TD> 1.76
</TD></TR><TR><TH> pMT316 #3
</TH><TD> 16.5 </TD><TD> 1.81
</TD></TR><TR><TH> Cre #1
</TH><TD> 84.9 </TD><TD> 1.93
</TD></TR><TR><TH> Cre #2
</TH><TD> 107.7 </TD><TD> 1.88
</TD></TR><TR><TH> Lox66 #2
</TH><TD> 51.3 </TD><TD> 1.87
</TD></TR><TR><TH> Firefly Luciferase #2
</TH><TD> 309.3 </TD><TD> 1.91
</TD></TR><TR><TH> Firefly Luciferase #3
</TH><TD> 325 </TD><TD> 1.9
</TD></TR><TR><TH> pMT413 #1
</TH><TD> 461.7 </TD><TD> 1.89
</TD></TR><TR><TH> pMT413 #2
</TH><TD> 362.2 </TD><TD> 1.91
</TD></TR><TR><TH> pMT413 #3
</TH><TD> 381.8 </TD><TD> 1.92
</TD></TR></TBODY></TABLE><DIV id="entry"><SPAN id="date"><A name="06-24-2010" class="date">06-24-2010</A></SPAN> [<A class="top" href="#notebook">top</A>]</DIV><P><B>Primers for Barstar, Barnase, and Gal4DBD arrive</B>
Primers:
</P><P>BB.Barstar.Fwd - 36.1 nM, diluted with 361μL H<SUB>2</SUB>O
</P><PRE>  5'- CCT TTC TAG AAT GAA AAA AGC AGT CAT TAA C -3'
</PRE><P>BB.Barstar.Rev - 30.0 nM, diluted with 300μL H<SUB>2</SUB>O
</P><PRE>  5'- AAG GCT GCA GCGGCC GCT ACT AGT TTA AGA AAG TAT GAT GGT GAT GTC -3'
</PRE><P>BB.Barnase.Fwd - 38.5 nM, diluted with 385μL H<SUB>2</SUB>O
</P><PRE>  5'- CCT TTC TAG AAT GAA AAA ACG ATT ATC ATG G -3'
</PRE><P>BB.Barnase.Rev - 13.2 nM, diluted with 132μL H<SUB>2</SUB>O
</P><PRE>  5'- AAG GCT GCA GCG GCC GCT ACT AGT TTA TCT GAT TTT TGT AAA GGT CTG -3'
</PRE><P>BB.Gal4DBD.Fwd - 38.5 nM, diluted with 385μL H<SUB>2</SUB>O
</P><PRE>  5'- CCT TTC TAG AAT GAA GCT ACT GTC TTC TAT -3'
</PRE><P>BB.Gal4DBD.Rev - 32.9 nM diluted with 329μL H<SUB>2</SUB>O
</P><PRE>  5'- AAG GCT GCA GCG GCC GCT ACT AGT CGA TAC AGT CAA CTG TCT TTG -3'
</PRE><P><B>Digestion</B></P><P>Our first digestion was unsuccessful except for the confirmation of the PMT316 fragment; this can be attributed to an incorrect combination of restriction enzymes.  The digestion was performed again in the afternoon using EcoRI + SpeI for the biobricked plasmids (lox71, GFP mut, lox66, cre recombinase, and firefly luciferase) and EcoRI + HindIII for PMT413.
</P><P>Gel Image, first digestion:
</P><DIV><A href="http://openwetware.org/images/d/da/Failgel624.jpg" id="single_image"> [click to enlarge]</A></DIV><P>PMT316 digests in lanes 8 and 9 show bands of approximately 439 bp. (above image)
</P><DIV><A href="http://openwetware.org/images/5/56/624secondattempt.jpg" id="single_image"> [click to enlarge]</A></DIV><P>
The length of the shortest bands were confirmed to be roughly the same length of the desired excerpt with the exception of Lox66 and Lox71 where it was determined that the inserts being only 34 bp had run off the gel, therefore the single band seen was the remaining plasmid.
</P><P><B>Yeast Growth Assay</B>
To assay the effect of Methoxyfenozide on yeast, three flasks containing 25μL DMSO (control), 2.5μL of 100mM Methoxyfenozide stock in 25mL DMSO to form the 1μM condition, and 25μL of 100mM Methoxyfenozide stock in 25mL DMSO to from the 10μM condition.
Each beaker also contained y190 yeast strains and YPD medium.
</P><P>To construct a growth curve of the yeast under the 0μM, 1μM and 10μM conditions, 1mL from each flask would be pipetted into a cuvette and measured in a spectrophotometer ever hour to assess yeast density.
</P><P>Unfortunately in this first attempt, no growth was observed after 4 hours, in the control and experimental groups, so the test was called off, to be repeated tomorrow.
</P><TABLE class="wikitable" border="1px solid black"><TBODY><TR><TH>Time
</TH><TH>0μM Methoxyfenozide
</TH><TH>1μM Methoxyfenozide
</TH><TH>10μM Methoxyfenozide
</TH></TR><TR><TD>11:00 AM
</TD><TD>.045
</TD><TD>.042
</TD><TD>.036
</TD></TR><TR><TD>12:00 PM
</TD><TD>.036
</TD><TD>.040
</TD><TD>.041
</TD></TR><TR><TD>3:00 PM
</TD><TD>.035
</TD><TD>.036
</TD><TD>.039
</TD></TR></TBODY></TABLE><DIV id="entry"><SPAN id="date"><A name="06-28-2010" class="date">06-28-2010</A></SPAN> [<A class="top" href="#notebook">top</A>]</DIV><P><B>B21 Innoculation</B>
Inoculated 3 colonies of B21 transformed E.Coli and set them to shake for 8 hours. We will maxiprep the B21 plasmid for use as a vector backbone.
</P><P><B>PCR of Gal4DBD and Barnase, attempt 2</B>
Ran 3 tubes of each for a total of 6.
7x Mastermix:
</P><UL><LI>14μL DNTP
</LI><LI>3.5μL Polymerase
</LI><LI>28μL 5x Buffer
</LI><LI>73.5μL DH<SUB>2</SUB>O
</LI></UL><P>Each tube contained:
</P><UL><LI>1μL Fwd primer
</LI><LI>1μL Rev Primer
</LI><LI>1μL Minipreped sample
</LI><LI>17μL Mastermix
</LI></UL><P>Reduced annealing temp to 50°C from 56°C which was the last attempt, ran otherwise identical LACIN program
</P><P><B>Gel of Gal4 DBD and Barnase from second PCR attempt</B>
Ran 1.2% E-gel of the 6 PCR product tubes.
</P><TABLE class="wikitable" border="1px solid black"><TBODY><TR><TH>1
</TH><TH>2
</TH><TH>3
</TH><TH>4
</TH><TH>5
</TH><TH>6
</TH><TH>7
</TH><TH>8
</TH><TH>9
</TH><TH>10
</TH><TH>11
</TH><TH>12
</TH></TR><TR><TD>Ladder
</TD><TD>Gal4 DBD 1
</TD><TD>Gal4 DBD 2
</TD><TD>Gal4 DBD 3
</TD><TD>Barnase 1
</TD><TD>Barnase 2
</TD><TD>Barnase 3
</TD><TD>Ladder
</TD></TR></TBODY></TABLE><DIV><A href="http://openwetware.org/images/7/71/Gal4_Barnase_PCR_2nd.jpg" id="single_image"> [click to enlarge]</A></DIV><P><B>QIAQuick Purification of PCR Product</B>
Because colonies 1 and 2 of Barstar worked properly on last week's gel, the PCR products in those two tubes were combined and purified using the Qiagen PCR purification protocol.
</P><UL><LI>as there was about 25μL of PCR product in the two together, 125μL of PB buffer was added (5x the concentration of the PCR product)
</LI><LI>pipetted the mix into a QIAquick spin column, spun for 30 seconds at 13,000 rpm
</LI><LI>discarded flow-through
</LI><LI>added .75mL PE buffer to the column, spun for 30 seconds
</LI><LI>discarded flow-through, and spun again for 1 minute
</LI><LI>moved the column to a fresh eppendorf tube
</LI><LI>added 50 μL EB buffer to the column, spun for 1 minute
</LI><LI>labeled the eppendorf tube 'Barstar PCR purified'
</LI></UL><DIV id="entry"><SPAN id="date"><A name="06-29-2010" class="date">06-29-2010</A></SPAN> [<A class="top" href="#notebook">top</A>]</DIV><P>B21 plasmid maxiprep
</P><P>colony inoculated overnight and medium was centrifuged to obtain a pellet.
</P><P>followed instructions as described in the QIAgen Plasmid Purification Handbook pg. 19-23. (2005 edition)
</P><DIV id="entry"><SPAN id="date"><A name="06-30-2010" class="date">06-30-2010</A></SPAN> [<A class="top" href="#notebook">top</A>]</DIV><P><B>pMT1002 Miniprep</B></P><UL><LI>4ml from each overnight cell culture of pMT1002 (1,2&amp;3) were transferred into 5 ml conicals and spun for 6 minutes at 4400 rpm</LI><LI>remainder of overnight cell cultures were placed in the fridge for transfer to microcentrifuge tubes at a later time</LI><LI>excess LP+amp decanted, the pellet resuspended in 250μL P1</LI><LI>contents transferred from each 15ml conical to a new epindorf tube</LI><LI>250 μL P2 added to each tube, mixed by inverting gently 4-6 times</LI><LI>350 μL N3 added to each tube, mixed by inverting as before</LI><LI>tubes centrifuged for 10 minutes at 13,000 rpm</LI><LI>supernatants pipetted into new QIAprep sin columns</LI><LI>columns centrifuged for 30-60 seconds, flow through discarded</LI><LI>QIAprep columns washed with 0.5 ml buffer PB each and centrifuged for 30-60 seconds</LI><LI>columns washed with 0.75 ml buffer PE and again centrifuged for 30-60 seconds</LI><LI>flow through was discarded and tubes centrifuged for an additional minute</LI><LI>column was placed in a clean 1.5 ml microcentrifuge tube</LI><LI>50 μL buffer EB was added, tubes were let stand for 1 minute and centrifuged for one minute after</LI><LI>flow through kept, eppendorfs labeled pMT1002 plus their colony number</LI></UL><P>Also see pages 22-23 of the <A href="http://www.google.com/url?sa=t&amp;source=web&amp;cd=1&amp;ved=0CBIQFjAA&amp;url=http%3A%2F%2Fkirschner.med.harvard.edu%2Ffiles%2Fprotocols%2FQIAGEN_QIAprepMiniprepKit_EN.pdf&amp;ei=wOcbTOfxCIH-8AaIhI2ADA&amp;usg=AFQjCNENLjIQI2lUNMlpIqkcKrrq9buqLg&amp;sig2=GVeWspXGiA0bwGnulA18-Q" class="external text" rel="nofollow">QIAprep Miniprep Handbook</A> for miniprep instructions.
<B>pMT1002 Miniprep Nanodrop values</B></P><TABLE border="3" style="margin-left: 3em;"><TBODY><TR><TH> Plasmid </TH><TH> Quantity (ng/μL) </TH><TH> 260/280
</TH></TR><TR><TH> pMT1002 #1
</TH><TD> 68.2 </TD><TD> 1.84
</TD></TR><TR><TH> pMT1002 #2
</TH><TD> 138.7 </TD><TD> 1.84
</TD></TR><TR><TH> pMT1002 #3
</TH><TD> 127.2 </TD><TD> 1.88
</TD></TR></TBODY></TABLE><P><B>PCR of miniprepped Barnase</B></P><P>20μL per tube
</P><P>In each tube:
</P><UL><LI>1μL Forward primer
</LI><LI>1μL Reverse primer
</LI><LI>0.5μL polymerase
</LI><LI>4μL 5X buffer
</LI><LI>2μL DNTPs
</LI><LI>1μL DNA sample
</LI><LI>10.5μL water
</LI></UL><P>3 samples were made for the Barnase
</P><P><B>B21 Plasmid Digest</B>
put 10x green FD buffer on ice to thaw.
</P><P>Tube A:
</P><UL><LI>1μL Xba1
</LI></UL><UL><LI>1μL Pst1
</LI></UL><UL><LI>2μL green 10x FD buffer
</LI></UL><UL><LI>3.4μL colony A plasmid (the amount used to get .5μg of plasmid)
</LI></UL><UL><LI>12.6μL H<SUB>2</SUB>O (to bring the total to 20μL)
</LI></UL><P>Tube B:
</P><UL><LI>1μL Xba1
</LI></UL><UL><LI>1μL Pst1
</LI></UL><UL><LI>2μL green 10x FD buffer
</LI></UL><UL><LI>1.7μL colony B plasmid (the amount used to get .5μg of plasmid)
</LI></UL><UL><LI>14.3μL H<SUB>2</SUB>O (to bring the total to 20μL)
</LI></UL><P>
Pst1 and Xba1 digest of B21 plasmid, lane 4 and 5 cut, 7 and 8 uncut
</P><DIV><A href="http://openwetware.org/images/f/f8/Pmt1002barnasePCRattempt1.jpg" id="single_image"> [click to enlarge]</A></DIV><P>
Second Attempt
</P><P>
Tube A:
</P><UL><LI>1μL Xba1
</LI></UL><UL><LI>1μL Pst1
</LI></UL><UL><LI>5μL green 10x FD buffer
</LI></UL><UL><LI>12.5μL colony A plasmid (the amount used to get .5μg of plasmid)
</LI></UL><UL><LI>30.5μL H<SUB>2</SUB>O (to bring the total to 50μL)
</LI></UL><P>Tube B:
</P><UL><LI>1μL Xba1
</LI></UL><UL><LI>1μL Pst1
</LI></UL><UL><LI>5μL green 10x FD buffer
</LI></UL><UL><LI>6.6μL colony B plasmid (the amount used to get .5μg of plasmid)
</LI></UL><UL><LI>36.4μL H<SUB>2</SUB>O (to bring the total to 50μL)
</LI></UL><P>Ran on a 1% agarose gel, 100 Volts for 30-40 mins.
</P><P>Lane 2 buffer, lane 8 and 9 Colony B, lanes 15 and 16 colony A
</P><DIV><A href="http://openwetware.org/images/4/44/B21_digest_6-30-10.jpg" id="single_image"> [click to enlarge]</A></DIV><P>Cut out the backbone (upper band, and placed into four eppendorf tubes, labeled 'B21 backbone' and with their respective weight before the gel was added. Placed all four in the iGEM team fence box in the -4°C freezer to be purified tomorrow.
</P><P><B>Other</B></P><UL><LI>Designed primer sequences for ACC synthase degradation site, linker sequence connecting Barnase to the degradation site.
</LI><LI>Designed primers to amplify germination/embryogenesis promoter and AtArp2 promoter.
</LI></UL><DIV id="entry"><SPAN id="date"><A name="07-01-2010" class="date">07-01-2010</A></SPAN> [<A class="top" href="#notebook">top</A>]</DIV><P>B21 Backbone Gel Extraction
</P><UL><LI>followed QIAquick gel extraction handbook except melting temperature was 42°C instead of 50°C
</LI><LI>4 gels to begin with, highest nanodrop value before consolidation = 1.4ng/μL
</LI><LI>The 4 tubes were consolidated into 2 and placed in the vacufuge for ≈30 minutes to increase DNA concentration
</LI><LI>The 2 tubes were consolidated into 1 for a final concentration of 18.6ng/μL
</LI></UL><P>PMT1002 Digestion
</P><UL><LI>2 reactions each containing the following:
</LI><LI>1μL EcoRI
</LI><LI>1μL HindIII
</LI><LI>2μL 10X digestion buffer
</LI><LI>1/2 ng of sample ≈7μL
</LI><LI>9μL dH2O
</LI></UL><P>only one band was visible in the gel for each reaction.  Julia forgot to put the ladder in so it was not possible to precisely determine the single band size, but judging from the position of the loading dye it seemed that the plasmid had only been cut once.
</P><P><B>Ligation</B></P><P>Ligation performed on Barstar, NLS, and LacIN
</P><UL><LI>3 times as much insert as vector per reaction, length of insert taken into consideration.
</LI><LI>Barstar: 0.4 kb = 18.5ng
</LI><LI>NLS: 0.06 kb = 3ng
</LI><LI>LacIN: 1.1 kb = 52 ng
</LI></UL><P>Ligation Recipe:
</P><UL><LI>10μL insert + backbone
</LI><LI>10μL ligation buffer (vortexed thoroughly)
</LI><LI>1μL ligase, mix gently
</LI></UL><P>let stand for 15 minutes at room temperature, then put on ice
use for 4μL of the product for transformation and freeze the rest
</P><P>LacIN
</P><UL><LI>3μL B21 vector
</LI><LI>7μL LacIN insert
</LI></UL><P>NLS
</P><UL><LI>3μL B21 vector
</LI><LI>1μL NLS insert
</LI><LI>6μL dH20
</LI></UL><P>Barstar
</P><UL><LI>3μL B21 vector
</LI><LI>1.7μL Barstar
</LI><LI>5.3μL dH20
</LI></UL><P>Transformation from ligation
</P><UL><LI>thawed 4 chemically competent TOP10 cell tubes on ice
</LI><LI>added 4μL of each ligase rxn to each incubated on ice for 30 minutes
</LI><LI>heat shock at 42°C for 45 seconds
</LI><LI>put on ice for 2 minutes
</LI><LI>added 170μL SOC medium to each tube set to shake at 37°C for 30 minutes
</LI><LI>pippetted each tube onto its own LB+amp plate, streak with beads 
</LI><LI>incubate at 37°C overnight
</LI></UL><DIV id="entry"><SPAN id="date"><A name="07-02-2010" class="date">07-02-2010</A></SPAN> [<A class="top" href="#notebook">top</A>]</DIV><P><B>pMT1002, pMT413 Digest</B></P><P>Digested with EcoR1 and HindIII
</P><P>Lane 2 KB ladder
</P><P>Lane 4 pMT1002
</P><P>Lane 6 pMT413
</P><DIV><A href="http://openwetware.org/images/e/eb/PMT1002_413ecoRHindIII7-2-1.jpg" id="single_image"> [click to enlarge]</A></DIV><P><B>Barnase PCR</B></P><P>Standard PCR measurements were used except that a new 50X dNTP mix was used in place of the 5X buffer used in all previous PCR reactions.  The amount of dH20 was adjusted for the new quantity of dNTP mix accordingly.
</P><DIV><A href="http://openwetware.org/images/b/b3/71barnasepcrweird.jpg" id="single_image"> [click to enlarge]</A></DIV><DIV id="entry"><SPAN id="date"><A name="07-06-2010" class="date">07-06-2010</A></SPAN> [<A class="top" href="#notebook">top</A>]</DIV><P>Inoculated 7 Barstar and 3 NLS colonies from Thursday's ligation.
</P><P>Performed PCR on LacIn (E10) following standard PCR measurements as previously described.
</P><P>Digestion of B21 backbone using Xba1 and Pst1
</P><P>Total volume per reaction: 50 μL
</P><UL><LI>2μL Xba1
</LI><LI>2μL Pst1
</LI><LI>10μL B21 plasmid (0.3μg/μL X 10μL = 3μg DNA)
</LI><LI>5μL 10X digestion buffer
</LI><LI>31μL H2O
</LI></UL><DIV><A href="http://openwetware.org/images/a/a4/76B21_digest_gel.pdf" id="single_image"> [click to enlarge]</A></DIV><P>Digestion successful, larger bands removed for gel extraction.
</P><P>Gel extraction performed according to QIAgen gel extraction protocol.
</P><DIV id="entry"><SPAN id="date"><A name="07-07-2010" class="date">07-07-2010</A></SPAN> [<A class="top" href="#notebook">top</A>]</DIV><P><B>Miniprep of Barastar and NLS Ligation products</B></P><P>Performed miniprep of overnight cell cultures of 7 colonies of Barstar, 3 colonies of NLS.
</P><P>Minipreps performed according to pages 22-23 of the <A href="http://www.google.com/url?sa=t&amp;source=web&amp;cd=1&amp;ved=0CBIQFjAA&amp;url=http%3A%2F%2Fkirschner.med.harvard.edu%2Ffiles%2Fprotocols%2FQIAGEN_QIAprepMiniprepKit_EN.pdf&amp;ei=wOcbTOfxCIH-8AaIhI2ADA&amp;usg=AFQjCNENLjIQI2lUNMlpIqkcKrrq9buqLg&amp;sig2=GVeWspXGiA0bwGnulA18-Q" class="external text" rel="nofollow">QIAprep Miniprep Handbook</A>.
</P><DIV><A href="http://openwetware.org/images/7/71/77LacINpcr.pdf" id="single_image"> [click to enlarge]</A></DIV><P><B>Digestion of Barstar and NLS Ligation products</B></P><P>7 reactions of Barstar and 3 reactions of NLS
</P><P>In each reaction:
</P><UL><LI>1μL Xba1
</LI><LI>1μL Pst1
</LI><LI>2.5μL of sample, Barstar or NLS Plasmid, respectively, obtained from miniprep (above)
</LI><LI>2μL Buffer
</LI><LI>13.5μL DH<SUB>2</SUB>O
</LI></UL><P>Ran on a 2% agarose E-gel
</P><P>Lane 1: Ladder
</P><P>Lane 2-8: Barstar
</P><P>Lane 9-11: NLS
</P><DIV><A href="http://openwetware.org/images/6/6b/77barstarnlsdigest1.pdf" id="single_image"> [click to enlarge]</A></DIV><P><B>PCR of LacIN</B>
3 reactions, each with:
</P><UL><LI>1μL Fwd LacIN Primer
</LI><LI>1μL Rev LacIN Primer
</LI><LI>.5μL Polymerase
</LI><LI>2μL DNTP
</LI><LI>4μL Buffer
</LI><LI>1μL E10 Plasmid (LacI)
</LI><LI>10.5μL DH<SUB>2</SUB>O
</LI></UL><P>Using program:
</P><P>1= 95°C for 10:10
</P><P>2= 95°C for :15
</P><P>3= 50°C for :30
</P><P>4= 72°C for 1:30
</P><P>5= goto 2, 29 times
</P><P>6=72°C for 10:10
</P><P>7=4°C forever
</P><P>Gel Image of LacIN PCR result:
</P><P><B>PCR of Barnase and Gal4DBD</B></P><P>5 reactions each of Barnase from pMT413, Barnase from pMT1002, and Gal4
</P><P>In pMT413 Barnase reactions:
</P><UL><LI>1μL Barnase2.Fwd Primer
</LI><LI>1μL Rev.Barnase Primer
</LI><LI>.5μL Polymerase
</LI><LI>2μL DNTP
</LI><LI>4μL Buffer
</LI><LI>2μL pMT413 plasmid
</LI><LI>10.5μL DH<SUB>2</SUB>O
</LI></UL><P>In pMT1002 Barnase Reactions:
</P><UL><LI>1μL Barnase2.Fwd Primer
</LI><LI>1μL Rev.Barnase Primer
</LI><LI>.5μL Polymerase
</LI><LI>2μL DNTP
</LI><LI>4μL Buffer
</LI><LI>2μL pMT1002 plasmid
</LI><LI>10.5μL DH<SUB>2</SUB>O
</LI></UL><P>In Gal4DBD reactions:
</P><UL><LI>1μL Gal4.Fwd Primer
</LI><LI>1μL Gal4.Rev Primer
</LI><LI>.5μL Polymerase
</LI><LI>2μL DNTP
</LI><LI>4μL Buffer</LI><LI>2μL Gal4DBD plasmid
</LI><LI>10.5μL DH<SUB>2</SUB>O
</LI></UL><DIV id="entry"><SPAN id="date"><A name="07-08-2010" class="date">07-08-2010</A></SPAN> [<A class="top" href="#notebook">top</A>]</DIV><DIV><A href="http://openwetware.org/images/b/b3/78PCRgelGalBarnaseLac.pdf" id="single_image"> [click to enlarge]</A></DIV><P>
After receiving the new primers we performed PCR on barnase vectors PMT413 and PMT1002, LacIN, as well as the GAL4 DNA binding domain.  The ladder is in lane 2, the GAL4 PCR products are in lanes 5 through 9, the barnase PMT413 lanes 10 through 14, and the LacIN PCR product is in the last 3 lanes.  After PCR was confirmed successful, we decided to gel purify the LacIN because of the multiple band sizes seen in the gel.  Below is the gel where LacIN was gel extracted, the chunk missing from the gel is where LacIN was identified to be.
</P><DIV><A href="http://openwetware.org/images/4/46/78gelextractskeleton.pdf" id="single_image"> [click to enlarge]</A></DIV><P>Gel extraction was performed according to the QIAgen gel extraction kit protocol using a microcentrifuge.
</P><P><B>Barstar Sequencing</B>
Submited plasmids from each of the 7 Barstar colonies for sequencing.
</P><DIV id="entry"><SPAN id="date"><A name="07-09-2010" class="date">07-09-2010</A></SPAN> [<A class="top" href="#notebook">top</A>]</DIV><P><B>Digest of yesterday's ligations</B>
LacIN - 65μL
</P><PRE> 1μL Xba1 
 1μL Pst1
 6.5μL FD buffer
 1μL DPN1
 50μL LacIN
 6.5μL DH<SUB>2</SUB>O
</PRE><P>Gal4DBD - 50μL
</P><PRE> 1μL Xba1
 1μL Pst1
 7μL Gal4
 1μL DPN1
 5μL FD Buffer
 36μL DH<SUB>2</SUB>O
</PRE><P>Barnase - 50μL
</P><PRE> 1μL Xba1
 1μL Pst1
 15μL Barnase
 1μL DPN1
 5μL FD buffer
 28μL DH<SUB>2</SUB>O
</PRE><P>Put in 37°C bath for 30 mins
</P><P>Following proceedure for PCR cleanup to clean up the digestion results
</P><UL><LI>added 5 volumes PB buffer to 1 volume digestion product
</LI><LI>325μL for LacIN, 250μL for Gal4DBD and Barnase
</LI><LI>pippetted each into a QIAquick column, spun for 30 secs
</LI><LI>discarded flow through, put columns back in same tube, added 750μL PE buffer
</LI><LI>spun for 30 secs, discared flow through
</LI><LI>Spun again for 1 min
</LI><LI>placed column in new eppendorf
</LI><LI>added 30μL EB buffer to elute, spun for 1 min
</LI></UL><P>Eppendorfs labeled &quot;LacIN digest cleanup 7/9,&quot; &quot;Gal4 digest cleanup 7/9,&quot; and &quot;Barnase digest cleanup 7/9&quot;
</P><P><B>7-10-10</B> (Saturday)
</P><P><B>Innoculating colonies from Friday's ligations</B></P><UL><LI>Barnase, Gal4, LacIN
</LI><LI>4 cc tubes each, 5mL LB+amp, set to shake at 37°C overnight
</LI></UL><P><B>7-11-10</B> (Sunday)
</P><P><B>Miniprep of Barnase, Gal4 and LacIN ligations</B></P><UL><LI> No cells found in Barnase colonies 1 and 2 --&gt; innoculations 3 and 4 renamed Barnase 1 and 2, replacing the cell-free cultures.
</LI></UL><UL><LI>pipetted 4 ml from each overnight cell culture (excepting the cell-free Barnase cultures) into a 15 ml conical *centrifuged at 4400 rpm for 6 minute
</LI><LI>remaining overnight cell cultures were placed in fridge
</LI><LI>decanted LB+amp, resuspended cells in 250 μL P1 buffer
</LI><LI>contents transferred to eppendorfs
</LI><LI>250 μL of P2 buffer was added per tube, the tubes were inverted 4-6 times
</LI><LI>350 μL N3 buffer added to each, tubes inverted 4-6 times
</LI><LI>centrifuged for 10 min at 13,000 rpm, supernatant collected ad transferred to QIAprep spin columns
</LI><LI>centrifuged for 30-60 seconds, flow through discarded
</LI><LI>0.5 ml buffer PB added and centrifuged for 30-60 seconds, flow through discarded, spun columns for an additional minute
</LI><LI>QIAprep columns put into new eppendorfs
</LI><LI>50 μL buffer EB added to columns, let stand for 1 minute
</LI><LI>centrifuged for 1 minute at 13,000 rpm
</LI></UL><DIV id="entry"><SPAN id="date"><A name="07-12-2010" class="date">07-12-2010</A></SPAN> [<A class="top" href="#notebook">top</A>]</DIV><P><B>Nanodrop of Barnase, Gal4 and LacIN Minipreps</B></P><TABLE border="3" style="margin-left: 3em;"><TBODY><TR><TH> Plasmid </TH><TH> Quantity (ng/μL) </TH><TH> 260/280
</TH></TR><TR><TH> Barnase 1
</TH><TD> 226.1 </TD></TR><TR><TH> Barnase 2
</TH><TD> 336.3 </TD><TD> 1.94
</TD></TR><TR><TH> Gal4 1
</TH><TD> 298.0 </TD><TD> 1.88
</TD></TR><TR><TH> Gal4 2
</TH><TD> 363.3 </TD><TD> 1.92
</TD></TR><TR><TH> Gal4 3
</TH><TD> 301.9 </TD><TD> 1.89
</TD></TR><TR><TH> LacIN 1
</TH><TD> 376.7 </TD><TD> 1.89
</TD></TR><TR><TH> LacIN 2
</TH><TD> 237.8 </TD><TD> 1.90
</TD></TR><TR><TH> LacIN 3
</TH><TD> 339.4 </TD><TD> 1.92
</TD></TR><TR><TH> LacIN 4
</TH><TD> 368.9 </TD><TD> 1.92
</TD></TR></TBODY></TABLE><P><B>Digestion of Ligation product minipreps</B>
Xba1 and Pst1
</P><P>Mastermix (10x)
</P><UL><LI> 10μL Xba1
</LI><LI> 10μL Pst1
</LI><LI> 20μL Buffer
</LI></UL><P>
Barnase 1
</P><PRE>  4μL Barnase 1
  12μL DH<SUB>2</SUB>O
  4μL Mastermix
</PRE><P>Barnase 2
</P><PRE>  3μL Barnase 2
  13μL DH<SUB>2</SUB>O
  4μL Mastermix
</PRE><P>Gal 1
</P><PRE>  3μL Gal 1
  13μL DH<SUB>2</SUB>O
  4μL Mastermix
</PRE><P>Gal 2
</P><PRE>  2.5μL Gal 2
  13.5μL DH<SUB>2</SUB>O
  4μL Mastermix
</PRE><P>Gal 3
</P><PRE>  3μL Gal 3
  13μL DH<SUB>2</SUB>O
  4μL Mastermix
</PRE><P>LacIN 1
</P><PRE>  2.5μL LacIN 1
  13.5μL DH<SUB>2</SUB>O
  4μL Mastermix
</PRE><P>LacIN 2
</P><PRE>  4μL LacIN 2
  12μL DH<SUB>2</SUB>O
  4μL Mastermix
</PRE><P>LacIN 3
</P><PRE>  2.5μL LacIN 1
  13.5μL DH<SUB>2</SUB>O
  4μL Mastermix
</PRE><P>LacIN 4
</P><PRE>  2.5μL LacIN 1
  13.5μL DH<SUB>2</SUB>O
  4μL Mastermix
</PRE><P>
Put tubes in 37°C heat bath for 30 mins, then put them on ice.
</P><P>Gel of Digestions:
</P><DIV><A href="http://openwetware.org/images/f/f9/07122010ve3ctordigestmirbraz2.jpg
" id="single_image"> [click to enlarge]</A></DIV><P><B>Sequencing</B></P><UL><LI>submitted Barnase 1 and 2, Gal4 1,2, and 3, and LacIN 1,2,3, and 4 for sequencing
</LI></UL><DIV id="entry"><SPAN id="date"><A name="07-13-2010" class="date">07-13-2010</A></SPAN> [<A class="top" href="#notebook">top</A>]</DIV><DIV><A href="http://openwetware.org/images/5/5a/713gelyeastbackbone.jpeg" id="single_image"> [click to enlarge]</A></DIV><DIV><A href="http://openwetware.org/images/6/6c/713gelyeastcutout.jpeg" id="single_image"> [click to enlarge]</A></DIV><P>The yeast backbones containing the PIF3 and PHYB inserts was isolated and purified.  The latter image shows where the appropriately sized bands were excised.
</P><P>Nanodrop values of final product are:
</P><P>PHYB 11ng/μL
PIF3 20ng/μL
</P><DIV id="entry"><SPAN id="date"><A name="07-14-2010" class="date">07-14-2010</A></SPAN> [<A class="top" href="#notebook">top</A>]</DIV><P><B>Phenol Chloroform DNA Extraction</B></P><UL><LI>resuspend DNA in approximately 200μL molecular grade pure water
</LI><LI>add 1 volume phenol chloroform
</LI><LI>when pipetting the phenol chloroform be sure to pipette from closer to the middle level of the liquid as the top two centimeters do not contain what we want
</LI></UL><LI>centrifuge at top speed for 5 minutes
</LI><LI>collect the aqueous top layer and transfer to a clean microcentrifuge tube
</LI><LI>the remaining liquid is toxic and must be carefully disposed of in the &quot;toxic waste&quot; bottle in the fume hood
</LI><LI>add 1 volume phenol chloroform
</LI><LI>centrifuge at top speed for 5 minutes
</LI><LI>collect the aqueous top layer and put into a fresh tube
</LI><LI>add three volumes 100% ethanol and 1/10 volume sodium acetate
</LI><LI>allow solution to incubate on ice for 15 minutes
</LI><LI>centrifuge at top speed for 10 minutes
</LI><LI>pipette off the aqueous solution leaving a pellet of DNA in the bottom of the tube
</LI><LI>wash in 500μL 70% ethanol
</LI><LI>mix tube by inverting
</LI><LI>centrifuge again for 10 minutes at top speed
</LI><LI>air dry pellet by leaving tube open on bench (not on ice) til dry
</LI><LI>after pellet has dried resuspend DNA in desired volume of H2O or buffer EB
</LI><LI>presence of DNA can be confirmed by running a small sample on a gel
</LI><P><B>NLS Ligations</B>
Ligation of NLS.Serine
</P><UL><LI>2.5μL Backbone
</LI><LI>1.5μL of 1/100 dilution (1.5ng NLS.Serine)
</LI><LI>6μL H<SUB>2</SUB>O
</LI><LI>10μL 2x quick ligase buffer
</LI><LI>1μL quick ligase
</LI></UL><P>B21 control
</P><UL><LI>2.5μL B21 Backbone
</LI><LI>7.5μL DH<SUB>2</SUB>O
</LI><LI>10μL 2x Quick Ligase Buffer
</LI><LI>1μL Quick Ligase
</LI></UL><P><B>Transformation</B></P><UL><LI>1μL NLS lig into TOP10 cells
</LI><LI>Chilled on ice for 30 mins
</LI><LI>Heat shocked at 42°C for 30 seconds
</LI><LI>Put on ice for 2 mins
</LI><LI>Added 170μL SOC medium
</LI><LI>Set to shake  in incubator for 30 mins
</LI><LI>Streaked on LB+amp plates, incubated overnight
</LI></UL><P><B>Yeast backbone gel purified nanodrops</B></P><TABLE border="1"><TBODY><TR><TH> Sample
</TH><TH> Concentraion in ng/μL
</TH><TH> 260/280
</TH></TR><TR><TH> 31 (1)
</TH><TD> 8.5 </TD><TD> 3.00
</TD></TR><TR><TH> 31(2)
</TH><TD> 10.2 </TD><TD>2.07
</TD></TR><TR><TH> 32(3)
</TH><TD> 11.1 </TD><TD>2.31
</TD></TR><TR><TH> 52(4)
</TH><TD> 28.5 </TD><TD>1.83
</TD></TR></TBODY></TABLE><DIV id="entry"><SPAN id="date"><A name="07-15-2010" class="date">07-15-2010</A></SPAN> [<A class="top" href="#notebook">top</A>]</DIV><P>Our previous attempt resulted in DNA with the following nanodrop values:
</P><UL><LI>12.1ng/μL 260/280: 0.9
</LI><LI>39.1ng/μL 260/280: 1.11
</LI></UL><P>It was believed that the other genomic DNA provided by the Matthews Lab before had similar values with regards to purity/contamination (also approaching 1)  Both team allergy and fence performed PCR multiple times unsuccessfully with these DNA samples.  It was thought that the low purity may be inhibiting the PCR.
</P><P>Today we used the QIAgen DNeasy Plant Mini Kit.  Liquid nitrogen was poured over the arabidopsis leaves in a mortar while they were ground with the pestle.  This produced a fine green powder of arabidopsis leaves from which genomic DNA was extracted starting with step seven of the QIAgen protocol following liquid nitrogen crushing.  The only change made to the protocol was that 40 μL of RNase 1 was added per reaction instead of the recommended 4 μL because the concentration of our supply was ten-fold lower.  
</P><P>Nanodrop values were as follows:
</P><UL><LI>6.7 ng/μL 260/280: 1.42
</LI><LI>5 ng/μL 260/280: 1.48
</LI></UL><DIV id="entry"><SPAN id="date"><A name="07-16-2010" class="date">07-16-2010</A></SPAN> [<A class="top" href="#notebook">top</A>]</DIV><P><B>PCR of Arabidopsis Genomic DNA, attempt 2</B>
PCR using the newly extracted DNA was unsuccessful.
</P><P>The degradation signal from the Austrians was successfully inoculated and colonies were found on the agar plates.  Individual colonies from the plates were inoculated into tubes of 5 ml LB+Amp and placed in the shaking incubator overnight. (37°C)
</P><P><B>Making Glyerol Stocks</B>
Combined .5 mL 80% glycerol with .5mL overnight cell culture of: Barnase 3 (number changed to Barnase 1 from miniprep onward), LacIN1, and Gal2.
</P><P><B>7-17-10</B> (Saturday)
</P><P><B>5xGal4 Promoter Annealing Product Ligation</B></P><UL><LI>Made a 1 in 1000 dilution of the annealing reaction
</LI></UL><TABLE border="1"><CAPTION><I>Nanodrop</I></CAPTION><TBODY><TR><TH> ng/μL </TH><TH> 260/280
</TH></TR><TR><TH> 5xGalpt 1/1000 dilution
</TH><TD align="center"> 17.6 </TD><TD> 2.25
</TD></TR></TBODY></TABLE><P>For a ligation reaction, we want:
</P><UL><LI>50ng of backbone
</LI><LI>A 1:3 ratio of backbone:insert
</LI><LI> 3(ng of backbone / basepairs of backbone) = (ng of insert / basepairs of insert)
</LI></UL><P>5xGalpt is 180 bases long, B21 backbone is 3.2kb.
So we want 8.4375 ng of plasmid
</P><P>To reach 8.4ng of 5xGalpt, made a 1 in 2000 dilution of the 5xGalpt Annealing product
</P><P>Ligation Reaction:
</P><UL><LI>1μL 1/2000 5xGalpt
</LI><LI>2.5μL B21 Backbone
</LI><LI>6.5μL DH<SUB>2</SUB>O
</LI><LI>1μL Quick Ligase
</LI><LI>10μL 2x Quick Ligase Buffer
</LI></UL><P>B21 Control Ligation:
</P><UL><LI>2.5μL B21 Backbone
</LI><LI>7.5μL DH<SUB>2</SUB>O
</LI><LI>1μL Quick Ligase
</LI><LI>10μL 2x Quick Ligase Buffer
</LI></UL><P>Incubated at room temp for 15 mins, put on ice and immediately transformed.
</P><P><B>Transformations of 5xGalpt, NLS.Serine anneal, and B21 Backbone control</B></P><UL><LI>Thawed 3 Turbo chemically competent cells on ice
</LI><LI>Pippeted 1μL of the ligation or annealing reaction into its own tube of turbo cells
</LI><LI>Chilled on ice for 30 mins
</LI><LI>Heatshocked in 42°C water on heat block for 30 seconds
</LI><LI>Chilled on ice for 2 mins
</LI><LI>Added 170μL SOC medium to each tube, set to shake at 37°C for 15 mins
</LI><LI>Streaked on LB + amp plates, and left in incubator overnight
</LI></UL><P><B>7-18-10</B> (Sunday)
</P><P><B>Re-doing 5xGalpt and NLS.Serine Annealing Reactions</B>
Set ~400ml of water to boil in a beaker on a heat block.
</P><P>Phosphorylating 5xGalpt
</P><UL><LI>3μL 100μM oligo1.5xgal4
</LI><LI>3μL 100μM oligo2.5xgal4
</LI><LI>3μL 100μM oligo3.5xgal4
</LI><LI>3μL 100μM oligo4.5xgal4
</LI><LI>3μL 10x PNK buffer
</LI><LI>2μL nM ATP
</LI><LI>2μL t4 PNK
</LI><LI>11μL DH<SUB>2</SUB>O
</LI></UL><P>put in 37°C waterbath for 90 mins
Then added 4μL 5M NaCl to halt the reaction
</P><P>To get rid of the NaCL so that I can ligate the overlapping oligos:
</P><UL><LI>added 3 volues of 100% ethanol (90μL), spun at top speed for 20 mins
</LI><LI>pippetted off and discarded the supernatant, added 100μL of 70% ethanol, broke up the pellet, and spun for 5 mins
</LI><LI>pippeted off supernatant, allowed to air dry.
</LI><LI>added 2μL T4 ligase buffer, 1μL T4 ligase, and 17μL DH<SUB>2</SUB>O, using this mix to resuspend the pellet
</LI><LI>set in PCR machine at 16°C for 12 hours, then 4°C until stopped.
</LI></UL><P><B>Annealing NLS.Serine (again)</B></P><UL><LI>3μL NLS.Serine.Fwd
</LI><LI>3μL NLS.Serine.Rev
</LI><LI>2μL 10x annealing buffer
</LI><LI>12μL DH<SUB>2</SUB>O
</LI></UL><P>Put in boiling water in beaker (see above), allowed to boil for 2 mins, then removed beaker from heat block, allowing the water to slowly cool with the annealing reaction tube still in it.
</P><DIV id="entry"><SPAN id="date"><A name="07-21-2010" class="date">07-21-2010</A></SPAN> [<A class="top" href="#notebook">top</A>]</DIV><P><B>Nanodroping 5xGal promoter</B></P><P>Nanodrop of ligated annealed oligos:
5200.8 ng/μL,   1.32 260/280
</P><P><B>Gel Extraction of 5xGal promoter</B></P><P>poured 2% agarose Gel for gel extraction, ran 2μL of the ligated annealing reaction in 13μL EB buffer, with two KB plus 1 ladders, one on each side.
</P><P>Gel extraction performed, into an eppendorf weighing .9563g
</P><DIV><A href="http://openwetware.org/images/9/91/5xGalprmt_gel_extract_07-21.jpg" id="single_image"> [click to enlarge]</A></DIV><P>Nanodrop for cleaned-up gel purified 5xGalpt negative - ran a 2% E-gel to check. There appears to be something at the .16kb size.
</P><DIV><A href="http://openwetware.org/images/a/a1/5xGalptGelPur.Check.7-21.jpg" id="single_image"> [click to enlarge]</A></DIV><DIV id="entry"><SPAN id="date"><A name="07-22-2010" class="date">07-22-2010</A></SPAN> [<A class="top" href="#notebook">top</A>]</DIV><P><B>Sequencing Results</B></P><P>Barstar samples 1, 2, 5, and 6 were confirmed to be correct.
</P><P>The LacI sequence was confirmed, however we were unable to confirm the presence of the NLS because of its position at the end of the gene where sequencing results are generally unreliable. (NNNNN)  Lac1R and Lac3R were poor matches because of what GENEWIZ calls &quot;Early Termination&quot;.
</P><P>Barnase had some serious issues. We are re-growing colonies following ligation and intend to submit some new PCR product as well as the original ADDGENE plasmid for sequencing in the near future.
</P><P>The Gal4 DNA binding domain samples 1 and 2 were perfect except for a SNP around 320 (in the case of GAL DBD 1) for our sequence and 223 of the biobricks sequence.  
</P><P>Gal4 DBD sample 3 was unsuccessful.
</P><P><B>Arabidopsis PCR</B></P><DIV><A href="http://openwetware.org/images/2/2e/722pcrexp2sucess.jpg" id="single_image"> [click to enlarge]</A></DIV><P>FINALLY! EXP2 PROMOTER AMPLIFIED!
</P><P>~1000kb
</P><DIV><A href="http://openwetware.org/images/3/37/722pcrexp2sucess2.jpg" id="single_image"> [click to enlarge]</A></DIV><P>Fragments were gel isolated and purified using the QIAgen Gel Purification Kit with the Gel Extraction protocol using a microcentrifuge.
</P><DIV id="entry"><SPAN id="date"><A name="07-23-2010" class="date">07-23-2010</A></SPAN> [<A class="top" href="#notebook">top</A>]</DIV><P>No colonies from last night's ligations.
</P><P><B>7-24-10</B> (Saturday)
</P><P>Successful ACC synthase transformation:
</P><DIV><A href="http://openwetware.org/images/5/58/725acc.jpg" id="single_image"> [click to enlarge]</A></DIV><P>PFX polymerase PCR
</P><UL><LI>1μL 10mM MgSO4
</LI><LI>2μL 10X pfx amplification buffer
</LI><LI>2μL 10X PCRx enhancer solution
</LI><LI>1μL F primer
</LI><LI>1μL R primer
</LI><LI>0.5μL pfx polymerase
</LI><LI>2μL DNTP
</LI></UL><P>9.5μL total
</P><P>DNA: 21ng/μL
</P><UL><LI>50 to 200μL DNA per reaction
</LI><LI>5 reactions
</LI><LI>4μL per reaction = 20μL sample total (MM)
</LI></UL><P>9.5μL + 4μL = 13.5μL
therefore 6.5μL H20 per reaction
</P><P>MM
</P><UL><LI>5μL 10nM MgSO4
</LI><LI>10μL 10X pfx amplification buffer
</LI><LI>10μL 10X PCRx enhancer solution
</LI><LI>5μL F primer
</LI><LI>5μL R primer
</LI><LI>2.5μL pfx polymerase
</LI><LI>10μL DNTP
</LI><LI>20μL genomic DNA
</LI><LI>32.5μL H20
</LI></UL><P>total volume:100μL
</P><P><B>Arabidopsis PCR</B></P><DIV><A href="http://openwetware.org/images/d/d5/726arapcregel.jpg-px240" id="single_image"> [click to enlarge]</A></DIV><P><B>Transformations</B></P><UL><LI>Thawed 2 turbo cells on ice to thaw
</LI><LI>Resuspended EcR, RXRHm, RXRLc tubes from Mr.Gene into 50μL DH<SUB>2</SUB>O (5μg into 50μL)
</LI><LI>Pippetted .5μL plasmid from each of the 5 synthesis products (EcR, RXRHm, RXRLc, ACC, and  ACT2LacOpt) into a labled eppendorf containing 20μL of turbo cells each
</LI><LI>Pippetted 2μL of the relavent ligation reaction from Friday (Strep.Barnase 2 ligation, Strep.Barnase 1, Gal4 DBD lig) into eppendorfs containing 20μL turbo cells (barn1strep, barn2strep and Gal4DBD)
</LI><LI>Chilled on ice for 15 mins
</LI><LI>Put in 42°C heat bath for 30 secs
</LI><LI>Put on ice for 2 mins
</LI><LI>Added 200μL SOC medium (to equal 10x the amount of cells)
</LI><LI>Set to shake at 37°C for 30 mins (longer for Act2pt because kan selective cultures need more time to become established before facing the harsher Kan selection medium
</LI><LI>Streaked 20μL on LB+amp plates
</LI><LI>Put in incubator to shake overnight
</LI><LI>After 45 mins to 1.5 hours, streaked Act2LacOpt on Kan+LB plate, put in incubator to shake overnight.
</LI></UL><P><B>Making LB+Kan Plates</B></P><UL><LI>made 4 LB+Kanamycin plates
</LI></UL><PRE>  15μL Kanamycin
  30μL DH<SUB>2</SUB>O to ease streaking
</PRE><P>Streaked this on LB agar plates with glass beads, and then turned the plate upside down and let dry until it was ready for bacterial sequencing, and then those same beads were reused.
</P><P><B>Transformation of pLAS and Terminator</B>
Resuspended the following from the registry with 10μL DH<SUB>2</SUB>O
</P><PRE>  p(LAS)TetO, on pSB2K3 (kan res.), from plate 2, sell 11N
  Terminator, on pSB1AK3 (kan and amp res.), from plate 1, well 4H
</PRE><DIV id="entry"><SPAN id="date"><A name="07-27-2010" class="date">07-27-2010</A></SPAN> [<A class="top" href="#notebook">top</A>]</DIV><P><B>Innoculations</B></P><UL><LI>RXRHuman, RXRLocust, Barn1strep, Barn2strep, ACC, and EcR
</LI></UL><P>(these are all amp resistant)
</P><UL><LI>6 colonies of each are inocculated, except for Barn1 and Barn2, of which 4 are innocculated each.
</LI><LI>5mL LB+amp and a smear of each colony from a relevant plate
</LI></UL><P>The same is done in LB+Kan for pLAS and Act2lacO promoter, which are Kanamycin resistant.
</P><P><B>Redoing Barnase-strep Tag Ligation</B>
poured a 2% agarose gel to gel-purify the barnase from  Barnase1 Xba Pst1 cleanup and Barnase2 Xba Pst1 cleanup, both from (7/23)
</P><UL><LI>Gel ran empty except for DB ladders
</LI></UL><P><B>Phosphatase Treating B15 Spe1/Pst1 Cleanup</B></P><UL><LI>to prevent religation of the whole vector
</LI></UL><UL><LI>35μL B15 Spe1 Pst1 cleanup from 7/23
</LI><LI>4μL 10x green FD buffer
</LI><LI>1μL Alkaline phosphatase
</LI></UL><DIV id="entry"><SPAN id="date"><A name="07-28-2010" class="date">07-28-2010</A></SPAN> [<A class="top" href="#notebook">top</A>]</DIV><P><B>Minipreps</B></P><P>Miniprepped and Nanodroped the following:
</P><TABLE border="3" style="margin-left: 3em;"><TBODY><TR><TH> Plasmid </TH><TH> Quantity (ng/μL) </TH><TH> 260/280
</TH></TR><TR><TH> Act2LacO 1
</TH><TD> 46.1 </TD><TD> 1.92
</TD></TR><TR><TH> RXRHm 4
</TH><TD> 23.8 </TD><TD> 2.15
</TD></TR><TR><TH> RXRHm 2
</TH><TD> 45.2 </TD><TD> 1.94
</TD></TR><TR><TH> Terminator 6
</TH><TD> 172.8 </TD><TD> 1.93
</TD></TR><TR><TH> Terminator 3
</TH><TD> 456.5 </TD><TD> 1.91
</TD></TR><TR><TH> RXRLc 2
</TH><TD> 42.7 </TD><TD> 1.76
</TD></TR><TR><TH> RXRLc 4
</TH><TD> 58.5 </TD><TD> 2.03
</TD></TR><TR><TH> RXRLc 5
</TH><TD> 45.1 </TD><TD> 2.06
</TD></TR><TR><TH> ACC 3
</TH><TD> 91.5 </TD><TD> 2.01
</TD></TR><TR><TH> Barn 2 Strep 3
</TH><TD> 41.0 </TD><TD> 2.12
</TD></TR><TR><TH> Barn 2 Strep 2
</TH><TD> 90.3 </TD><TD> 1.95
</TD></TR><TR><TH> Barn 1 Strep 2
</TH><TD> 67.1 </TD><TD> 1.99
</TD></TR><TR><TH> Barn 1 Strep 3
</TH><TD> 92.9 </TD><TD> 1.99
</TD></TR><TR><TH> EcR 5
</TH><TD> 115.3 </TD><TD> 1.95
</TD></TR><TR><TH> EcR 2
</TH><TD> 108.0 </TD><TD> 1.97
</TD></TR><TR><TH> EcR 4
</TH><TD> 79.4</TD><TD> 1.97
</TD></TR><TR><TH> EcR 3
</TH><TD> 111.6 </TD><TD> 1.96
</TD></TR><TR><TH> EcR 6
</TH><TD> 95.8 </TD><TD> 1.96
</TD></TR></TBODY></TABLE><P><B>Barnase Digest Gel Extraction</B>
Again blank on the first attempt, tried again:
</P><P>Success!
</P><TABLE border="1"><TBODY><TR><TH> 2 </TH><TH> 5 </TH><TH> 8 </TH><TH> 9 </TH><TH> 10 </TH><TH> 18
</TH></TR><TR><TH> DNA
</TH><TD> KB+ ladder </TD><TD> 15μL Barnase2 digest 1/5 dilution 7/23 </TD><TD> 20μL 1/3 dilution Barn2 digest cleanup 7/23 </TD><TD> 5μL 1/3 dilution Barn2 digest cleanup 7/23 </TD><TD> 20μL 1/3 dilution Barn2 digest cleanup 7/23 </TD><TD> KB+ ladder
</TD></TR></TBODY></TABLE><DIV><A href="http://openwetware.org/images/f/fe/Barnase_pstxba_gel_ext_7-28.jpg" id="single_image"> [click to enlarge]</A></DIV><DIV id="entry"><SPAN id="date"><A name="07-29-2010" class="date">07-29-2010</A></SPAN> [<A class="top" href="#notebook">top</A>]</DIV><P><B>Digest of Wednesday's minipreps</B></P><P><B>Gel Extraction of Barnase insert</B></P><UL><LI>followed gel extraction protocol in QIAquick Spin Handbook
</LI></UL><P>Nanodrops:
</P><TABLE border="3" style="margin-left: 3em;"><TBODY><TR><TH> Plasmid </TH><TH> Quantity (ng/μL) </TH><TH> 260/280
</TH></TR><TR><TH> Tube #1
</TH><TD> 9.7 </TD><TD> 1.79
</TD></TR><TR><TH> Tube #2
</TH><TD> 17.4 </TD><TD> 1.57
</TD></TR><TR><TH> Tube #3
</TH><TD> 2.7 </TD><TD> 1.24
</TD></TR></TBODY></TABLE><P><B>Samples 1-9</B></P><DIV><A href="hhttp://openwetware.org/images/b/b6/729_digest_1-9.jpg" id="single_image"> [click to enlarge]</A></DIV><UL><LI>Lane 1: ladder
</LI><LI>Lane 2: ladder
</LI><LI>Lane 3: ACC synthase 1
</LI><LI>Lane 4: ACC synthase 2
</LI><LI>Lane 5: Acc synthase 3
</LI><LI>Lane 6: Terminator 1
</LI><LI>Lane 7: Terminator 2
</LI><LI>Lane 8: Terminator 3
</LI><LI>Lane 9: Act2LacI 1
</LI><LI>Lane 10: Act2LacI 2
</LI><LI>Lane 11: Act2LacI 3
</LI><LI>Lane 12: ladder
</LI></UL><P>insert sizes:
</P><UL><LI>ACC synthase: 300 to 350
</LI><LI>Terminator: unknown
</LI><LI>Act2LacI: ~1200
</LI></UL><P><B>Samples 10-18</B></P><DIV><A href="http://openwetware.org/images/1/1a/729_digest_10-18.jpg" id="single_image"> [click to enlarge]</A></DIV><UL><LI>Lane 1: empty
</LI><LI>Lane 2: ladder
</LI><LI>Lane 3: Barn1-strep1
</LI><LI>Lane 4: Barn1-strep2
</LI><LI>Lane 5: Barn1-strep3
</LI><LI>Lane 6: Barn2-strep1
</LI><LI>Lane 7: Barn2-strep2
</LI><LI>Lane 8: Barn2-strep3
</LI><LI>Lane 9: EcR2
</LI><LI>Lane 10: EcR3
</LI><LI>Lane 11: EcR4
</LI><LI>Lane 12: ladder
</LI></UL><P>insert sizes
</P><UL><LI>Barn1-strep: &lt;400bp
</LI><LI>Barn2-strep: &lt;400bp
</LI><LI>Ecr: ~1000bp
</LI></UL><P><B>Samples 19-24</B></P><DIV><A href="http://openwetware.org/images/b/b8/729_digest_19-24.jpg" id="single_image"> [click to enlarge]</A></DIV><UL><LI>Lane 1: empty
</LI><LI>Lane 2: ladder
</LI><LI>Lane 3: empty
</LI><LI>Lane 4: RxR-Hm 1
</LI><LI>Lane 5: RxR-Hm 2
</LI><LI>Lane 6: RxR-Hm 3
</LI><LI>Lane 7: RxR-Lc 1
</LI><LI>Lane 8: RxR-Lc 2
</LI><LI>Lane 9: RxR-Lc 3
</LI><LI>Lane 10: empty
</LI><LI>Lane 11: empty
</LI><LI>Lane 12: ladder
</LI></UL><P>insert size
</P><UL><LI>RxR-Lc: 600-700bp
</LI><LI>RxR-Hm: 600-700bp
</LI></UL><P><B>Summary</B></P><UL><LI>We see one appropriately sized band (~350bp) from the ACC synthase miniprep #1, the others did not have any bands.  Therefore henceforth we will only use plasmid from miniprep #1 for our reactions.
</LI><LI>Terminator size unknown at present.
</LI><LI>For the Act2LacI promoter we saw in all three lanes the insert sized at ~1200bp.  All three minipreps should be fine to use for future reactions.
</LI><LI>The Barnase-strep lanes all displayed inserts of the correct size, ~350 to 400 bp.  All minipreps should be good for future reactions.
</LI><LI>The Ect. Receptor insert appears to be about the right size, &gt;1000bp, although the ladder is a bit hard to decipher, all minipreps should be good for future reactions. 
</LI><LI>The RxR-Hm did not show any evidence of the presence of an insert.
</LI><LI>The RxR-Lc did show evidence of the presence of an insert, although it may be a bit larger than the expected size (600-700bp).  Blurriness of the ladder makes it difficult to decipher.
</LI></UL><P><B>Nanodrop of Gal4DBD Minipreps (from the beginning of the summer</B></P><TABLE border="3" style="margin-left: 3em;"><TBODY><TR><TH> Plasmid </TH><TH> Quantity (ng/μL) </TH><TH> 260/280
</TH></TR><TR><TH> Gal4DBD #1
</TH><TD> 26.5 </TD><TD> 1.89
</TD></TR><TR><TH> Gal4DBD #3
</TH><TD> 16.7 </TD><TD> 1.85
</TD></TR><TR><TH> Gal4DBD #4
</TH><TD> 19.1 </TD><TD> 1.83
</TD></TR><TR><TH> Gal4DBD #5
</TH><TD> 82.2 </TD><TD> 2.16
</TD></TR></TBODY></TABLE><P><B>PCR of VP16 and Gal4DBD (preparing for yeast vector insertion)</B>
Gal4DBD
</P><UL><LI>2μL DNTPs
</LI><LI>.5μL phusion polymerase
</LI><LI>4μL 5x phusion buffer
</LI><LI>1μL 1/10 dilution of BB.Gal4DBD.Rev
</LI><LI>1μL 1/10 dilution of Bam.Gal4.Fwd
</LI><LI>1μL 1/2 dilution Gal4DBD #3 (made 1/2 dilution such that 1μL will be &lt;10 ng/μL
</LI><LI>μL 10.5μLDH<SUB>2</SUB>O
</LI></UL><P>4 tubes of each, 20μL each tube
</P><P>
VP16
</P><UL><LI>10.5μL DH<SUB>2</SUB>O
</LI><LI>2μL DNTPs
</LI><LI>.5μL phusion polymerase
</LI><LI>4μL 5xphusion buffer
</LI><LI>1μL 1/10 dilution of VP16.Hind.Fwd
</LI><LI>1μL 1/10 dilution of VP16.Rev
</LI><LI>1μL 1/10 dilution of VP16 #1
</LI></UL><P>
PCR program:
VP16 Gal4
</P><P>1= 98°C for 10:00
</P><P>2= 98°C for :15
</P><P>3= 50°C to 60°C for :30
</P><P>4= 72°C for 1:30
</P><P>5= goto 2, 29 times
</P><P>6= 72°C for 10:10
</P><P>7= 4°C forever
</P><DIV id="entry"><SPAN id="date"><A name="07-30-2010" class="date">07-30-2010</A></SPAN> [<A class="top" href="#notebook">top</A>]</DIV><P><B>PCR of VP16 and Gal4DBD</B></P><TABLE class="wikitable" border="1px solid black"><TBODY><TR><TH>1
</TH><TH>2
</TH><TH>3
</TH><TH>4
</TH><TH>5
</TH><TH>6
</TH><TH>7
</TH><TH>8
</TH><TH>9
</TH><TH>10
</TH><TH>11
</TH><TH>12
</TH></TR><TR><TD>KB+ Ladder
</TD><TD>Gal4 DBD 1
</TD><TD>Gal4 DBD 2
</TD><TD>Gal4 DBD 3
</TD><TD>Gal4 DBD 4
</TD><TD>VP16 1
</TD><TD>VP16 2
</TD><TD>VP16 3
</TD><TD>VP16 4
</TD><TD>KB+ Ladder
</TD></TR></TBODY></TABLE><DIV><A href="http://openwetware.org/images/4/4c/PCR_VP16_Gal4DBD_7-30.jpg" id="single_image"> [click to enlarge]</A></DIV><P><B>PCR again for VP16</B>
Using pfx polymerase
</P><UL><LI>1μL 1/10 dilution Hind.Fwd
</LI><LI>1μL 1/10 dilution VP16.Rev
</LI><LI>1μL DNTPs
</LI><LI>6μL enhancer buffer
</LI><LI>.5μL MgSO<SUB>4</SUB></LI><LI>.5μL PFX polymerase
</LI><LI>1μL 1/10 dilution VP16 #1
</LI><LI>5μL DH<SUB>2</SUB>O
</LI><LI>4μL amp(lification?) buffer
</LI></UL><P>Running PCR on two reactions. There was not enough PFX for both to get .5μL, so only one of them is likely to actually amplify.
</P><P><B>Second ACC + RxrHm digest gel</B></P><DIV><A href="http://openwetware.org/images/3/36/Sucesspcrkfkdigestacvtually.jpg" id="single_image"> [click to enlarge]</A></DIV><P>yes to ACC (left) no RxrHm
</P><DIV id="entry"><SPAN id="date"><A name="08-02-2010" class="date">08-02-2010</A></SPAN> [<A class="top" href="#notebook">top</A>]</DIV><P><B>2% E-gel of VP16 PCR product from Friday</B></P><TABLE class="wikitable" border="1px solid black"><TBODY><TR><TH>Lane
</TH></TR><TR><TD>1
</TD><TD>2
</TD><TD>3
</TD><TD>4
</TD><TD>5
</TD><TD>6
</TD><TD>7
</TD><TD>8
</TD><TD>9
</TD><TD>10
</TD><TD>11
</TD><TD>12
</TD></TR><TR><TD>KB+ Ladder
</TD><TD>VP16 PCR product 1
</TD><TD>VP16 PCR product 2
</TD><TD>KB+ Ladder
</TD></TR></TBODY></TABLE><DIV><A href="http://openwetware.org/images/e/ef/VP16_PCR.2_8-1.jpg" id="single_image"> [click to enlarge]</A></DIV><P><B>Gel extraction</B>
Gal4 spe bam 8/2 and EcR bam xba 8/2 gel extracted and purified.
</P><P>Gal4 on the left, EcR on the right
</P><DIV><A href="http://openwetware.org/images/f/fc/Gal4_Ecr_digest_gel_ex_8-2.jpg" id="single_image"> [click to enlarge]</A></DIV><P><B>Nanodrop of Gal4 and EcR Gel Extractions</B></P><TABLE border="3" style="margin-left: 3em;"><TBODY><TR><TH> Plasmid </TH><TH> Quantity (ng/μL) </TH><TH> 260/280
</TH></TR><TR><TH> Gal4 (1)
</TH><TD> 1.4 </TD><TD> -3.18
</TD></TR><TR><TH> Gal4 (2)
</TH><TD> 7.2 </TD><TD> 4.59
</TD></TR><TR><TH> EcR (1)
</TH><TD> 5.3 </TD><TD> 2.52
</TD></TR><TR><TH> EcR (2)
</TH><TD> 5.6 </TD><TD> 2.12
</TD></TR></TBODY></TABLE><P><B>VP16 PCR 2</B>
60μL reaction:
</P><UL><LI>1.5 μL pfx polymerase
</LI><LI>3μL DNTPs
</LI><LI>18μL enhancer buffer
</LI><LI>1.5μL MgSO<SUB>4</SUB></LI><LI>3μL 1/10 dilution VP16.Hind.Fwd
</LI><LI>3μL 1/10 dilution VP16.Rev
</LI><LI>3μL 1/10 dilution of VP16 (1)
</LI><LI>12μL amp buffer
</LI><LI>5μL DH<SUB>2</SUB>O
</LI></UL><P>using program:
</P><PRE>  1=94°C for 2:00
  2=94°C for :15
  3=68°C for :30
  4=Goto 2, 30 times
  5=4.0°C forever
  6=end
</PRE><P>No PCR product was visible.
</P><P><B>VP16 PCR.3</B>
60μL reaction:
</P><UL><LI>1.5 μL pfx polymerase
</LI><LI>3μL DNTPs
</LI><LI>18μL enhancer buffer
</LI><LI>1.5μL MgSO<SUB>4</SUB></LI><LI>3μL 1/10 dilution VP16.Hind.Fwd
</LI><LI>3μL 1/10 dilution VP16.Rev
</LI><LI>3μL 1/10 dilution of VP16 (1)
</LI><LI>12μL amp buffer
</LI><LI>5μL DH<SUB>2</SUB>O
</LI></UL><P>Using modified program:
</P><PRE>  1=94°C for 2:00
  2=94°C for :15
  3=60°C for :30
  4=70°C for :45
  5=Goto 2, 30 times
  6=4.0°C forever
  7=end
</PRE><P>(see tomorrow's entry for gel image)
</P><P><B>Ligations</B></P><PRE>  Gal4 into EcR
  Barnase into B15
  LTP into B11 (positive control from team allergy)
</PRE><P>Gal4
</P><UL><LI>3μL Gal4 spe bam gel pur. (2) 8/2
</LI><LI>9μL EcR xba bam gel pur. (2) 8/2
</LI><LI>2μL 10x T4 ligase buffer
</LI><LI>5μL DH<SUB>2</SUB>O
</LI><LI>1μL T4 ligase
</LI></UL><P>EcR control
</P><UL><LI>9μL EcR xba bam gel pur. (2) 8/2
</LI><LI>2μL 10x T4 ligase buffer
</LI><LI>8μL DH<SUB>2</SUB>O
</LI><LI>1μL T4 ligase
</LI></UL><P>Barnase
</P><UL><LI>1.5μL Barnase digest gel pur. (1) 7/29
</LI><LI>3μL B15 spe pst phosphorylated clean up
</LI><LI>2μL 10x T4 ligase buffer
</LI><LI>12.5μL DH<SUB>2</SUB>O
</LI><LI>1μL T4 ligase
</LI></UL><P>B15 control
</P><UL><LI>3μL B15 spe pst phosphorylated clean up
</LI><LI>2μL 10x T4 ligase buffer
</LI><LI>14μL DH<SUB>2</SUB>O
</LI><LI>1μL T4 ligase
</LI></UL><P>LTP
</P><UL><LI>2μL LTP
</LI><LI>1μL B11 backbone
</LI><LI>2μL 10x T4 ligase buffer
</LI><LI>16μL DH<SUB>2</SUB>O (should have been 14μL, the 16 was accidental)
</LI><LI>1μL T4 ligase
</LI></UL><P>B11 control
</P><UL><LI>1μL B11 backbone
</LI><LI>2μL 10x T4 ligase buffer
</LI><LI>16μL DH<SUB>2</SUB>O (should have been 14μL, the 16 was accidental)
</LI><LI>1μL T4 ligase
</LI></UL><P>Allowed tubes to sit at room temp for an hour, proceeded to transformation
</P><DIV id="entry"><SPAN id="date"><A name="08-03-2010" class="date">08-03-2010</A></SPAN> [<A class="top" href="#notebook">top</A>]</DIV><P>VP16 PCR from last night
</P><DIV><A href="http://openwetware.org/images/9/9f/83gelpcrvp16.jpg" id="single_image"> [click to enlarge]</A></DIV><P><B>Yesterday's transformations</B></P><DIV><A href="http://openwetware.org/images/b/b4/Gal4-EcR_lig_8-3.jpg" id="single_image"> [click to enlarge]</A></DIV><DIV><A href="http://openwetware.org/images/7/7b/EcR_neg_control_8-3.jpg" id="single_image"> [click to enlarge]</A></DIV><DIV><A href="http://openwetware.org/images/1/16/LTP_lig-1_8-3.jpg" id="single_image"> [click to enlarge]</A></DIV><DIV><A href="hhttp://openwetware.org/images/8/85/LTP_lig-2_8-3.jpg" id="single_image"> [click to enlarge]</A></DIV><DIV><A href="http://openwetware.org/images/3/38/B11_neg_cont_lig_8-3.jpg" id="single_image"> [click to enlarge]</A></DIV><DIV><A href="http://openwetware.org/images/7/71/Barn-strep_lig_8-3.jpg" id="single_image"> [click to enlarge]</A></DIV><DIV><A href="http://openwetware.org/images/7/71/Barn-strep_lig_8-3.jpg" id="single_image"> [click to enlarge]</A></DIV><DIV><A href="http://openwetware.org/images/e/e7/B15_cont_lig_8-3.jpg" id="single_image"> [click to enlarge]</A></DIV><P><B>Glycerol Stocks</B>
from 7-27-10
</P><P>keeping Act2lac0pt 1, ACC synthase 1, EcR 2, RXRLc 2
</P><P>added .5mL 80% glycerol to .5mL overnight cell culture of selected transformation, vortexed thoroughly, and placed in the -80°C freezer in team fence box.
</P><P><B>Annealing 35s min promoter</B></P><UL><LI>3μL 100μM 35sminpromt.Rev
</LI><LI>3μL 100μM 35sminpromt.Fwd
</LI><LI>3μL .5M NaCl
</LI><LI>3μL 10x PNK buffer
</LI><LI>18μL DH<SUB>2</SUB>O
</LI></UL><P>placed eppendorf in boiling water for 2 mins, then removed water from heat and allowed water and eppendorf to slowly cool together.
</P><DIV id="entry"><SPAN id="date"><A name="08-04-2010" class="date">08-04-2010</A></SPAN> [<A class="top" href="#notebook">top</A>]</DIV><P><B>PCR of VP16.3</B>
60μL reaction:
</P><UL><LI>1.5 μL pfx polymerase
</LI><LI>3μL DNTPs
</LI><LI>18μL enhancer buffer
</LI><LI>1.5μL MgSO<SUB>4</SUB></LI><LI>3μL 1/10 dilution VP16.Hind.Fwd
</LI><LI>3μL 1/10 dilution VP16.Rev
</LI><LI>3μL 1/10 dilution of VP16 (1)
</LI><LI>12μL amp buffer
</LI><LI>5μL DH<SUB>2</SUB>O
</LI></UL><P>Using program:
</P><PRE>  1=94°C for 2:00
  2=94°C for :15
  3=60/68°C for :30
  4=68°C for :10
  5=Goto 2, 30 times
  6=4.0°C forever
  7=end
</PRE><P><B>Digestion of Gal-EcR</B>
BamH1 and Nco1
12 digestions, one for each Gal-EcR miniprep
Each:
</P><UL><LI>1μL BamH1
</LI><LI>1μL Nco1
</LI><LI>1μL 10x FD green buffer
</LI><LI>2-7μL EcR-Gal miniprep
</LI><LI>5-0μL DH<SUB>2</SUB>O, to bring total volume to 10μL
</LI></UL><P>Ran a 1% gel of the digest
</P><TABLE class="wikitable" border="1px solid black"><TBODY><TR><TD>1
</TD><TD>2
</TD><TD>3
</TD><TD>4
</TD><TD>5
</TD><TD>6
</TD><TD>7
</TD><TD>8
</TD><TD>9
</TD><TD>10
</TD><TD>11
</TD><TD>12
</TD><TD>13
</TD><TD>14
</TD></TR><TR><TD>EcR Digest 1
</TD><TD>EcR Digest 2
</TD><TD>EcR Digest 3
</TD><TD>EcR Digest 5
</TD><TD>EcR Digest 6
</TD><TD>EcR Digest 7
</TD><TD>EcR Digest 8
</TD><TD>EcR Digest 9
</TD><TD>EcR Digest 10
</TD><TD>EcR Digest 11
</TD><TD>EcR Digest 12
</TD><TD>KB+ ladder
</TD></TR></TBODY></TABLE><P>[
</P><DIV><A href="http://openwetware.org/images/a/a3/EcR-Gal_lig_VP16.4_Pcr_8-04.jpg" id="single_image"> [click to enlarge]</A></DIV><P><B>Ligated EcR to Gal4 and Barnase to Strep</B></P><PRE><DIV id="entry"><SPAN id="date"><A name="08-05-2010" class="date">08-05-2010</A></SPAN> [<A class="top" href="#notebook">top</A>]</DIV></PRE><P><B>Colonies from yesterday's ligations</B></P><DIV><A href="http://openwetware.org/images/0/0c/84_colonies_1.jpg" id="single_image"> [click to enlarge]</A></DIV><DIV><A href="http://openwetware.org/images/1/1b/84_colonies_2.jpg" id="single_image"> [click to enlarge]</A></DIV><P><B>Gel extraction of EcR-Gal Bam Nco</B></P><UL><LI>applied 3x gel volume of QG buffer (in μL) to each respective tube
</LI><LI>placed in 42°C bath until the gel was melted
</LI><LI>added 1 gel volume of isopropanol to each, and vortexed
</LI><LI>applied contents of each tube to a QIAquick spin collumn
</LI><LI>spun for 1 min
</LI><LI>added 500μL QG buffer (to dissolve any remaining agarose)
</LI><LI>spun for 1 min
</LI><LI>added 750μL PE buffer
</LI><LI>spun for 1 min, discared flow-through
</LI><LI>spun for another minute
</LI><LI>placed QIAquick collumn in a fresh, labeled eppendorf tube, applied 50μL EB buffer to elute, and spun for 1 min
</LI></UL><P><B>Nanodrop of Gal4 and EcR Gel Extractions</B></P><TABLE border="3" style="margin-left: 3em;"><TBODY><TR><TH> Plasmid </TH><TH> Quantity (ng/μL) </TH><TH> 260/280
</TH></TR><TR><TH> Gal-EcR (1)
</TH><TD> 3.1 </TD><TD> 2.45
</TD></TR><TR><TH> Gal-EcR (2)
</TH><TD> 4.5 </TD><TD> 1.72
</TD></TR><TR><TH> Gal-EcR (3)
</TH><TD> 3.0 </TD><TD> 11.56
</TD></TR><TR><TH> Gal-EcR (4)
</TH><TD> 3.6 </TD><TD> 2.21
</TD></TR></TBODY></TABLE><P><B>Digestions</B>
VP16
</P><UL><LI>1μL xba
</LI><LI>1μL Pst
</LI><LI>5μL VP16(2) miniprep
</LI><LI>1μL 10x FD green buffer
</LI><LI>2μL DH<SUB>2</SUB>O
</LI></UL><P>5xGalUAS
</P><UL><LI>1μL xba
</LI><LI>1μL Pst
</LI><LI>2μL 5xGal4UAS (from Karmella)
</LI><LI>1μL 10x FD green buffer
</LI><LI>5μL DH<SUB>2</SUB>O
</LI></UL><P>PhyB
</P><UL><LI>1μL Bam
</LI><LI>1μL Nco
</LI><LI>16μL PhyB
</LI><LI>2μL 10x FD green buffer
</LI></UL><P><B>Sending EcR-Gal for sequencing</B></P><UL><LI>5μL of each primer, either fwd or rev, at 1/20 dilution
</LI><LI>10μL of sample and DH<SUB>2</SUB>O, such that sample is at 500ng total mass
</LI></UL><P>Sealed tubes with parafilm.
Submited Gal-EcR 1, 7, 10, and 12 (the one's with inserts on the digestion gel from 8/4)
</P><DIV id="entry"><SPAN id="date"><A name="08-06-2010" class="date">08-06-2010</A></SPAN> [<A class="top" href="#notebook">top</A>]</DIV><P><B>Digestions</B>
pACT2
</P><UL><LI>1μL Bam
</LI><LI>1μL Xho
</LI><LI>6μL pACT2 (tube with a green top)
</LI><LI>1μL FD Green buffer
</LI><LI>1μL DH<SUB>2</SUB>O
</LI></UL><P>RXRLc
</P><UL><LI>1μL Bam
</LI><LI>1μL Xho
</LI><LI>9μL RXRLc4
</LI><LI>2μL FD Green buffer
</LI><LI>7μL DH<SUB>2</SUB>O
</LI></UL><P>Barstar
</P><UL><LI>1μL Eco
</LI><LI>1μL Pst
</LI><LI>2μL Barstar 3
</LI><LI>1μL FD Green buffer
</LI><LI>5μL DH<SUB>2</SUB>O
</LI></UL><P>Barnase
</P><UL><LI>1μL Eco
</LI><LI>1μL Pst
</LI><LI>1.5μL Barnase 2
</LI><LI>1μL FD Green buffer
</LI><LI>5.5μL DH<SUB>2</SUB>O
</LI></UL><P>5xGalUAS
</P><UL><LI>1μL Spe
</LI><LI>1μL Pst
</LI><LI>2μL 5xGalUAS (from Karmella)
</LI><LI>1μL FD Green buffer
</LI><LI>5μL DH<SUB>2</SUB>O
</LI></UL><P>pENT Cup
</P><UL><LI>1μL Spe
</LI><LI>1μL Pst
</LI><LI>1μL pENT Cup
</LI><LI>1μL FD Green buffer
</LI><LI>6μL DH<SUB>2</SUB>O
</LI></UL><P>EcR-Gal
</P><UL><LI>1μL Xba
</LI><LI>1μL Pst
</LI><LI>2μL EcR-Gal 7
</LI><LI>1μL FD Green buffer
</LI><LI>5μL DH<SUB>2</SUB>O
</LI></UL><P>VP16
</P><UL><LI>1μL Spe
</LI><LI>1μL Pst
</LI><LI>5μL VP16(1)
</LI><LI>1μL FD Green buffer
</LI><LI>2μL DH<SUB>2</SUB>O
</LI></UL><P>RXRLc
</P><UL><LI>1μL Xba
</LI><LI>1μL Pst
</LI><LI>9μL RXRLc4
</LI><LI>2μL FD Green buffer
</LI><LI>7μL DH<SUB>2</SUB>O
</LI></UL><P>V24
</P><UL><LI>1μL Eco
</LI><LI>1μL Pst
</LI><LI>3μL V24
</LI><LI>1μL FD Green buffer
</LI><LI>4μL DH<SUB>2</SUB>O
</LI></UL><P>V26
</P><UL><LI>1μL Eco
</LI><LI>1μL Pst
</LI><LI>9μL V26
</LI><LI>12μL FD Green buffer
</LI><LI>7μL DH<SUB>2</SUB>O
</LI></UL><P>PhyB
</P><UL><LI>1μL Bam
</LI><LI>1μL Nco
</LI><LI>16μL PhyB
</LI><LI>2μL FD Green buffer
</LI></UL><DIV id="entry"><SPAN id="date"><A name="08-09-2010" class="date">08-09-2010</A></SPAN> [<A class="top" href="#notebook">top</A>]</DIV><DIV><A href="http://openwetware.org/images/0/07/88yescamvexp2.jpg" id="single_image"> [click to enlarge]</A></DIV><P>yes colonies for ligation of exp2 and transformation of camv
</P><P><B>Digests</B></P><DIV><A href="http://openwetware.org/images/c/ca/86_digests.jpg" id="single_image"> [click to enlarge]</A></DIV><UL><LI>ladder
</LI><LI>1. pACT2 (backbone)
</LI><LI>2. RxRLC (insert)
</LI><LI>3. Barstar (insert)
</LI><LI>4. Barnase (insert)
</LI><LI>5. 5XGalUAS (backbone)
</LI><LI>6. PENTcup (backbone)
</LI><LI>7. V26 (backbone)
</LI><LI>8. Ecr-Galv (insert)
</LI></UL><UL><LI>no insert for 2 (RxRLC) or 8 (Ecr-Galv)
</LI></UL><DIV><A href="http://openwetware.org/images/0/0b/88_other_digests.jpg" id="single_image"> [click to enlarge]</A></DIV><UL><LI>ladder
</LI><LI>RxRLC (Xba and Pst) (insert)
</LI><LI>RxRLC (Bam and Xho) (insert) NO INSERT HERE
</LI><LI>PhyB (backbone)
</LI><LI>V24 (backbone)
</LI><LI>VP16 (backbone)
</LI></UL><DIV id="entry"><SPAN id="date"><A name="08-10-2010" class="date">08-10-2010</A></SPAN> [<A class="top" href="#notebook">top</A>]</DIV><P>[Image: 810digestslater.jpg]]
</P><UL><LI>ladder
</LI><LI>PhyB (backbone)
</LI><LI>Barstar 4 (insert)
</LI><LI>V26 (backbone)
</LI><LI>GalDBD1 (insert)
</LI><LI>GalDBD5 (insert)
</LI><LI>B21 (backbone)
</LI></UL><P><B>Digests 8/10</B></P><UL><LI>PhyB with Bam and Nco
</LI><LI>Barstar with Pst and Eco
</LI><LI>V26 with Pst and Eco
</LI><LI>Gal4DBD with Eco and Spe (Gal4DBD minipreps 1 and 5)
</LI><LI>Arp2BB PCR product with Xba and Pst
</LI><LI>Arp PCR product with Xba and Pst
</LI><LI>B21 with Xba and Pst
</LI><LI>ACC synthase with Eco and Spe
</LI><LI>Barnase ligation product with Eco and Xba
</LI></UL><DIV id="entry"><SPAN id="date"><A name="08-11-2010" class="date">08-11-2010</A></SPAN> [<A class="top" href="#notebook">top</A>]</DIV><P><B>colonies from yesterday's transformations/ligations</B></P><DIV><A href="http://openwetware.org/images/2/21/810_colonies_1.jpg" id="single_image"> [click to enlarge]</A></DIV><DIV><A href="http://openwetware.org/images/5/51/810_colonies_6.jpg" id="single_image"> [click to enlarge]</A></DIV><DIV id="entry"><SPAN id="date"><A name="08-12-2010" class="date">08-12-2010</A></SPAN> [<A class="top" href="#notebook">top</A>]</DIV><DIV><A href="http://openwetware.org/images/8/8b/Arp2coloniesligb21.jpg" id="single_image"> [click to enlarge]</A></DIV><P>This picture is not the best picture but we did find colonies on the plate with the arp2/b21 transformation.
</P><DIV><A href="http://openwetware.org/images/e/ec/812_5xgaluas.jpg" id="single_image"> [click to enlarge]</A></DIV><P>transformation from karmella's stock
</P><DIV id="entry"><SPAN id="date"><A name="08-13-2010" class="date">08-13-2010</A></SPAN> [<A class="top" href="#notebook">top</A>]</DIV><P><B>Transformations from Yesterday's Ligations</B></P><DIV><A href="http://openwetware.org/images/8/80/VP16Gal4DBD_lig_8-13.jpg" id="single_image"> [click to enlarge]</A></DIV><DIV><A href="http://openwetware.org/images/f/f6/PENT_cont_lig_8-13.jpg" id="single_image"> [click to enlarge]</A></DIV><DIV><A href="http://openwetware.org/images/c/ce/Nost_Act2LacO_lig_8-13.jpg" id="single_image"> [click to enlarge]</A></DIV><DIV><A href="http://openwetware.org/images/7/71/Gal4spepst_cont_lig_8-13.jpg" id="single_image"> [click to enlarge]</A></DIV><DIV><A href="http://openwetware.org/images/4/40/Gal4_ecoxba_cont_lig_8-13.jpg" id="single_image"> [click to enlarge]</A></DIV><DIV><A href="http://openwetware.org/images/5/59/Act2Lac_cont_lig_8-13.jpg" id="single_image"> [click to enlarge]</A></DIV><DIV><A href="http://openwetware.org/images/6/6d/Nost_pENT_lig_8-13.jpg" id="single_image"> [click to enlarge]</A></DIV><DIV><A href="http://openwetware.org/images/2/2b/EcR-Gal_BB_lig_8-12.jpg" id="single_image"> [click to enlarge]</A></DIV><P>
Innoculated colonies from EcR-Gal, Nost-pENT, and VP16Gal4DBD. No colonies visible on 
Nost-Act2LacO.
</P><P>All Controls were clean or had very very few colonies (&lt;2).
</P><DIV class="printfooter">
Retrieved from &quot;<A href="http://2010.igem.org/Team:Harvard/fences/notebook">http://2010.igem.org/Team:Harvard/fences/notebook</A>&quot;</DIV></DIV></DIV><DIV id="footer-box" class="noprint"><DIV id="footer"><UL id="f-list"><LI id="t-recentchanges"><A href="/Special:RecentChanges" title="Recent changes">Recent changes</A></LI><LI id="t-whatlinkshere"><A href="/Special:WhatLinksHere/Team:Harvard/fences/notebook" title="List of all wiki pages that link here [j]" accesskey="j">What links here</A></LI><LI id="t-recentchangeslinked"><A href="/Special:RecentChangesLinked/Team:Harvard/fences/notebook" title="Recent changes in pages linked from this page [k]" accesskey="k">Related changes</A></LI><LI id="t-specialpages"><A href="/Special:SpecialPages" title="List of all special pages [q]" accesskey="q">Special pages</A></LI><LI><A href="/Special:Preferences">My preferences</A></LI></UL></DIV><DIV id="footer"><UL id="f-list"><LI id="t-print"><A href="/wiki/index.php?title=Team:Harvard/fences/notebook&amp;printable=yes" title="Printable version of this page [p]" accesskey="p">Printable version</A></LI><LI id="t-permalink"><A href="/wiki/index.php?title=Team:Harvard/fences/notebook&amp;oldid=171554" title="Permanent link to this revision of the page">Permanent link</A></LI><LI id="privacy"><A href="/2010.igem.org:Privacy_policy" title="2010.igem.org:Privacy policy">Privacy policy</A></LI><LI id="disclaimer"><A href="/2010.igem.org:General_disclaimer" title="2010.igem.org:General disclaimer">Disclaimers</A></LI></UL></DIV></DIV></DIV></DIV></BODY></HTML>