"
Page
Discussion
View source
History
teams
Log in
 
Team:Harvard/fences/notebook
From 2010.igem.org
home
project
vectors
allergy
flavor
color
genetic fence
human practices
safety
parts
results
people
students
teaching fellows
advisors
acknowledgements
gallery
online
blog
twitter
openwetware
facebook
contact us
genetic fence lab notebook design safety parts & primers results references
notebook
Week 1
6-14-2010
06-15-2010
06-16-2010
06-17-2010
06-18-2010
Week 2
06-21-2010
06-22-2010
06-23-2010
06-24-2010
-
Week 3
06-28-2010
06-29-2010
06-30-2010
07-01-2010
07-02-2010
Week 4
-
07-06-2010
07-07-2010
07-08-2010
07-09-2010
Week 5
07-12-2010
07-13-2010
07-14-2010
07-15-2010
07-16-2010
Week 6
-
-
07-21-2010
07-22-2010
07-23-2010
Week 7
-
07-27-2010
07-28-2010
07-29-2010
07-30-2010
Week 8
08-02-2010
08-03-2010
08-04-2010
08-05-2010
08-06-2010
Week 9
08-09-2010
08-10-2010
08-11-2010
08-12-2010
08-13-2010
06-14-2010 [ top ]
LacI Transformation
Performed bacterial transformation according to Silver Lab Bacterial Transformation , however we used 1μL in step 3, and used entire tube of TOP10 competent cells instead of only 10-15μL.
Transformed the following, each into its own tube of TOP10 E.Coli:
1) LacI with a rapid degredation tail attached, biobrick BBa_C0012, located on plate 1, well 2O, plasmid pSB1A2
2) LacI wildtype, biobrick BBa_I732100, located on plate 2, well 10E, plasmid pSB1A3
Streaked 100μL each of 1 and 2 onto separate LB + Amp dishes, along with a control dish of 100μL untransformed TOP10 E.Coli cells.
Plates incubated overnight, colonies observed in all plates except the control.
[click to enlarge]
[click to enlarge]
[click to enlarge]
06-15-2010 [ top ]
Oligonucleotides for LacI
the following sequences for LacI and a Nuclear Localization Signal (NLS) were ordered for PCR:
LacIn.BB.Rev
5'-AAG GCT GCA GCG GCC GCT ACT AGT TCA AAC CTT TCT CTT CTT CTT AGG ATG AAC AAC AGA AGA CTG CCC GCT TTC CAG TCG GGA AA - 3'
LacIn.BB.Fwd
5'- CCT TGA ATT CGC GGC CGC ATC TAG AAT GAA ACC AGT ACC GTT ATA CGA TGT C -3'
NLS.BB.Rev
5'-GCG GCC GCT ACT AGT TCA AAC CTT TCT CTT CTTCTT AGG ATG AAC AAC AGA AGA -3'
NLS.BB.Fwd
5'-CTA GAT CTT CTG TTG TTC ATC CTA AGA AGA AGA GAA AGG TTT GAA CTA GTA GCG GCC GCT GCA -3'
VP16 transcription activating domain bacterial transformation
Performed bacterial transformation according to Silver:_Bacterial_Transformation , however we used 1μL in step 3, and used entire tube of TOP10 competent cells instead of only 10-15μL.
Transformed the following into its own tube of TOP10 E.Coli:
VP16 Eukaryotic Activating Domain, biobrick BBa_K105001, located on plate 3, well 13C, plasmid pSB1A2.
In the course of reconsitituting the the biobrick plasmid from its well, too little DH2O was added (1μL), so an additional 10μL were added to the well.
The first tube of TOP10+biobrick plasmid was erroneously heatshocked before being chilled for 30 mins on ice, so a second tube of TOP10 cells was introduced, and the correct proceedure followed for the second tube. Additionally, the first tube was then allowed to chill and heat shock again, ultimately producing two tubes of transformed TOP10 cells.
100μL of both tubes were streaked onto an LB + Amp dish, and incubated at 37°C on the plates overnight. Colonies the following morning, colonies were observed on both plates, although fewer on the plate containing the E.Coli which were erroneously heat-shocked too soon.
LacI Miniprep Preparations
Prepared 6 overnight cell cultures, consisting of three colonies each from the LacI wt and LacI+rapid degradation tail E.Coli cells transformed previously.
Each culture consisted of 5mL of LB+Amp, and a scraping of 1 E.Coli colony, in a 100cc tube.
These overnight cell cultures were set to shake at 37°C overnight.
06-16-2010 [ top ]
LacI Miniprep
Performed miniprep of overnight cell cultures of 3 colonies each of wildtype LacI transformed bacteria and LacI+rapid degradation tail bacteria.
Minipreps performed according to pages 22-23 of the QIAprep Miniprep Handbook .
LacI Nanodrop Blanked nanodrop sensor with 2μL EB buffer, then measured each of the 6 minipreped LacI plasmids.
Plasmid
Quantity (ng/μL)
260/280
O2 #1
77.1
1.81
O2 #2
90.1
1.9
O2 #3
50.4
1.95
E10 #1
99.6
1.93
E10 #2
104.6
1.92
E10 #3
58.6
1.98
LacI Plasmid Digest
We digested 15μL of each of the 6 samples with 1μLH2O, 1μL EcoR1, and 1μL Pst1 (high efficiency restriction enzymes).
Ran an E-gel of the digest results.
Lane #
1
2
3
4
5
6
7
8
9
10
O2 #1
ladder
O2 #1
O2 #2
O2 #3
E10 #1
E10 #2
E10 #3
[click to enlarge]
VP16 Colony Inoculation
Prepared 3 10cc tubes each containing .5μL LB+amp and cells from one of the three colonies of the VP16 cell culture of the previous night (colonies from the second-tube plate, the cells in which were heatshocked in the correct order).
Tubes were set to shake at 37°C overnight.
Preparing Glycerol Stocks of LacI wt and LacI+LVA
Prepared 2 cryo tubes, with the following labeling:
Tube 1) contains BBa_C0012, from plate 1, well 2O
On top: '#1'
On side: '6-16-10 MP pSB1A2 LacI+LVA'
Tube 2) contains BBa_I732100, from plate 2, well 10E
On top: '#2'
On side: '6-16-10 MP pSB1A3 LacI Wildtype'
Each cryo tube contains .5mL of the respective overnight cell culture (O2 colony #1 and E10 colony #1), and .mL of 80% glycerol.
Both Cryo tubes were placed in the box labeled 'Genetic Fence iGEM 2010' in the -80°C freezer.
06-17-2010 [ top ]
VP16 Miniprep
Following QIAgen Plasmid DNA Purification Protocol Using the QIAprep Spin Miniprep Kit and a Microcentrifuge the following plasmid was purified:
VP16
Protocol
4ml from each overnight cell culture of VP16 (1,2&3) were transferred into 5 ml conicals and spun for 6 minutes at 4400 rpm
remainder of overnight cell cultures were placed in the fridge for transfer to microcentrifuge tubes at a later time
excess LP+amp decanted, the pellet resuspended in 250μL P1
contents transferred from each 15ml conical to a new epindorf tube
250 μL P2 added to each tube, mixed by inverting gently 4-6 times
350 μL N3 added to each tube, mixed by inverting as before
tubes centrifuged for 10 minutes at 13,000 rpm
supernatants pipetted into new QIAprep sin columns
columns centrifuged for 30-60 seconds, flow through discarded
QIAprep columns washed with 0.5 ml buffer PB each and centrifuged for 30-60 seconds
columns washed with 0.75 ml buffer PE and again centrifuged for 30-60 seconds
flow through was discarded and tubes centrifuged for an additional minute
column was placed in a clean 1.5 ml microcentrifuge tube
50 μL buffer EB was added, tubes were let stand for 1 minute and centrifuged for one minute after
flow through kept and purity tested
Nanodrop Values for VP16
VP16 #1
95.7 ng/μL
260/280 = 1.92
VP16 #2
102.4 nm/μL
260/280 = 1.92
VP16 #3
90.7 ng/μL
260/280 = 1.92
Digestion
Quantities per double digestion reaction
10μL sample
1μL EcoRI
1μL PSD1
2μL buffer
6μL H20
Single digest reaction
10μL sample
1μL EcoRI
2μL buffer
7μL H20
ingredients combined and all tubes were put in 37°C bath for 10 minutes
Digest with SPE1 and XBA1
10μL sample
1μL SPE1
1μL XBA1
2μL buffer
6μL H20
All 3 epindorfs of VP16 were combined for a total volume of 90 μL or 9μg
1 unit enzyme cuts 1 μg DNA in 30-60 minutes
Digestion Gel
Ingredients
digested enzymes from above reaction
1 eppendorf containing 2μL uncut VP16 and 18μL H2O
kb ladder
IMAGE TO BE ADDED + details
Gel Recipe: 1 X TAE
900 ml dH2O
100 ml TAE
1 gel
150 ml buffer TAE
add ~2.8g agarose
mix in beaker
microwave until completely dissolved
add 5 μL ethedium bromide per 100 ml gel after solution has cooled (no longer steaming)
pour gel and let sit til solidified
Procedure
run products on gel for ~45 minutes
cut out band containing DNA fragment
follow extraction protocol in QIAquick spin handbook for QIAquick gel extraction kit protocol
Nanodrop values for resulting DNA
5.6ng/μL
260/280 = 2.24
Tube placed in -20 freezer labeled VP1 fragment XBA SPE1
Glycerol Stocks of VP16
0.5 ml 80% glycerol stock combined with 0.5 ml overnight culture of colony #2 of V{16 TP{10 cells
final product placed in the team fence box after mixing in the -80°C freezer
Bacterial transformation of GAL4 DNA binding domain
BBa_K105007, plate 3 well 9I, psB1A2
Procedure
removed 1 tube TOP10 chemically competent E.Coli and placed on ice
cleaned biobrick depository w/ ethanol
pipetted 1 μL of plasmid into the TOP10 cell tube
placed TOP10 cell tube on ice for 30 minutes
heat shocked at 42°C for 30 seconds
placed on ice for 2 minutes
added 170 μL SOC medium
streaked on LB + amp paltes, incubated at 37°C overnight
06-18-2010 [ top ]
LacI+NLS PCR
LacI+NLS primers arrived in mail
spun at 14,000 rpm for 10 minutes diluted stocks to 100 μM
LacIn.BB.Rev (1.7nM) diluted with 17μL DH2O    LacIn.BB.Fwd (1.5nM) diluted with 1011μL DH2O    NLS.BB.Rev (4.2nM) diluted with 42μL DH2O    LacIn.BB.Fwd (101.1nM) diluted with 15μL DH2O
PCR recipe
20μL per reaction
1μL template plasmid
1μL R primer
1μL F primer
4μL HF buffer 5X
2μL DNTPs
0.5μL pFU polymerase
rest H2O to make the volume per tube 20μL
used LACIN program in phusion PCR machine to perform PCR
06-21-2010 [ top ]
Barnase and Barstar
Barnase and Barstar Plasmids from ADDGENE arrived in mail
pMT316 - Barstar. Info from ADDGENE
pMT413 - Barnase, Barstar. Info from ADDGENE
pMT1002 - Barnase, Barstar. Info from ADDGENE
Notes from lab meeting
Karmella suggested we try adding an LVA tail to the Cre we use for the crelox fence system.
GAL4 DBD Innoculation
Pipetted 5mL of LB+Amp each into capped cc tubes, and mixed bacterial scrapings from 5 separate colonies of transformed GAL4 DNA binding domain E.Coli. Set to shake at 37 degrees C overnight.
06-22-2010 [ top ]
GAL4DBD Miniprep
[click to enlarge]
HindIII + PstI digest of miniprepped V9-V12 (#5,6) to check whether insert is present.]
pipetted 4 ml from each overnight cell culture from colonies 1-5 into a 15 ml conical centrifuged at 4400 rpm for 6 minute
remaining overnight cell cultures were placed in fridge
decanted LB+amp, resuspended cells in 250 μL P1 buffer
contents transferred to eppendorfs
250 μL of P2 buffer was added per tube, the tubes were inverted 4-6 times
350 μL N3 buffer added to each, tubes inverted 4-6 times
centrifuged for 10 min at 13,000 rpm, supernatant collected ad transferred to QIAprep spin columns
centrifuged for 30-60 seconds, flow through discarded
0.5 ml buffer PB added and centrifuged for 30-60 seconds, flow through discarded, spun columns for an additional minute
QIAprep columns put into new eppendorfs
50 μL buffer EB added to columns, let stand for 1 minute
centrifuged for 1 minute at 13,000 rpm
tubes labeled "GAL4" plus the specific colony number
PCR
PCR program for LacIN modified after Monday's unsuccessful O2 and E10 PCR (no product)
1=95°C for 10:00
2=95°C for 00:15
3=50°C for 00:30
4=72°C for 01:30
5=steps 2-4 X 29
6=72°C for 10:00
7=4°C for ∞
[click to enlarge]
PCR successful
PCR for E10 LacI
above PCR was repeated without O2 so as to finalize our E10 biobrick for use
PCR was performed using the standard proportions of reagents
The above settings on the PCR machine were used
[click to enlarge]
Gel cut-out
To Make Agarose Gel
150 mL TAE buffer combined with 1.5 g agarose in beaker
solution microwaved until agarose disolved
7.5 μL Ethidium Bromide added after beaker was no longer steaming
solution poured into clean gel mold and gel was left to solidify
E10 PCR product from the morning was consolidated into one tube,
final volume =90μL PCR product + 18μL 5X loading dye =110μL
DNA was extracted from gel by slicing out the bands and DNA was purified using the QIAquick Gel Extraction Kit, following protocol specific to using a centrifuge
see QIAquick spin handbook
Innoculations
Colonies from the following cultures were innoculated and set to shake at 37°C overnight:
PMT413
PMT316
firefly luciferase
GFP (mut)
cre recombinase
lox66
lox71
06-23-2010 [ top ]
Minipreps
The following colonies were minipreped after passing the night in the shaking incubator (37°C)
PMT413
PMT316
firefly luciferase
GFP (mut)
cre recombinase
lox66
lox71
Nanodrops for above minipreps
Blanked nanodrop sensor with 2μL EB buffer, then measured each of the 6 minipreped LacI plasmids.
If two numbers are given, separated by a slash, then two measurements were taken, both are presented.
Plasmid
Quantity (ng/μL)
260/280
Lox 71 #1
41.4
1.82
Lox 71 #2
59.6
1.89
Lox 71 #3
17.4/80.2
1.98
GFP #1
21.2
1.83
GFP #2
110.0
1.90
GFP #3
60.9/124
1.88
pMT316 #1
11.6
1.76
pMT316 #3
16.5
1.81
Cre #1
84.9
1.93
Cre #2
107.7
1.88
Lox66 #2
51.3
1.87
Firefly Luciferase #2
309.3
1.91
Firefly Luciferase #3
325
1.9
pMT413 #1
461.7
1.89
pMT413 #2
362.2
1.91
pMT413 #3
381.8
1.92
06-24-2010 [ top ]
Primers for Barstar, Barnase, and Gal4DBD arrive Primers:
BB.Barstar.Fwd - 36.1 nM, diluted with 361μL H2O
5'- CCT TTC TAG AAT GAA AAA AGC AGT CAT TAA C -3'
BB.Barstar.Rev - 30.0 nM, diluted with 300μL H2O
5'- AAG GCT GCA GCGGCC GCT ACT AGT TTA AGA AAG TAT GAT GGT GAT GTC -3'
BB.Barnase.Fwd - 38.5 nM, diluted with 385μL H2O
5'- CCT TTC TAG AAT GAA AAA ACG ATT ATC ATG G -3'
BB.Barnase.Rev - 13.2 nM, diluted with 132μL H2O
5'- AAG GCT GCA GCG GCC GCT ACT AGT TTA TCT GAT TTT TGT AAA GGT CTG -3'
BB.Gal4DBD.Fwd - 38.5 nM, diluted with 385μL H2O
5'- CCT TTC TAG AAT GAA GCT ACT GTC TTC TAT -3'
BB.Gal4DBD.Rev - 32.9 nM diluted with 329μL H2O
5'- AAG GCT GCA GCG GCC GCT ACT AGT CGA TAC AGT CAA CTG TCT TTG -3'
Digestion
Our first digestion was unsuccessful except for the confirmation of the PMT316 fragment; this can be attributed to an incorrect combination of restriction enzymes.  The digestion was performed again in the afternoon using EcoRI + SpeI for the biobricked plasmids (lox71, GFP mut, lox66, cre recombinase, and firefly luciferase) and EcoRI + HindIII for PMT413.
Gel Image, first digestion:
[click to enlarge]
PMT316 digests in lanes 8 and 9 show bands of approximately 439 bp. (above image)
[click to enlarge]
The length of the shortest bands were confirmed to be roughly the same length of the desired excerpt with the exception of Lox66 and Lox71 where it was determined that the inserts being only 34 bp had run off the gel, therefore the single band seen was the remaining plasmid.
Yeast Growth Assay To assay the effect of Methoxyfenozide on yeast, three flasks containing 25μL DMSO (control), 2.5μL of 100mM Methoxyfenozide stock in 25mL DMSO to form the 1μM condition, and 25μL of 100mM Methoxyfenozide stock in 25mL DMSO to from the 10μM condition. Each beaker also contained y190 yeast strains and YPD medium.
To construct a growth curve of the yeast under the 0μM, 1μM and 10μM conditions, 1mL from each flask would be pipetted into a cuvette and measured in a spectrophotometer ever hour to assess yeast density.
Unfortunately in this first attempt, no growth was observed after 4 hours, in the control and experimental groups, so the test was called off, to be repeated tomorrow.
Time
0μM Methoxyfenozide
1μM Methoxyfenozide
10μM Methoxyfenozide
11:00 AM
.045
.042
.036
12:00 PM
.036
.040
.041
3:00 PM
.035
.036
.039
06-28-2010 [ top ]
B21 Innoculation Inoculated 3 colonies of B21 transformed E.Coli and set them to shake for 8 hours. We will maxiprep the B21 plasmid for use as a vector backbone.
PCR of Gal4DBD and Barnase, attempt 2 Ran 3 tubes of each for a total of 6. 7x Mastermix:
14μL DNTP
3.5μL Polymerase
28μL 5x Buffer
73.5μL DH2O
Each tube contained:
1μL Fwd primer
1μL Rev Primer
1μL Minipreped sample
17μL Mastermix
Reduced annealing temp to 50°C from 56°C which was the last attempt, ran otherwise identical LACIN program
Gel of Gal4 DBD and Barnase from second PCR attempt Ran 1.2% E-gel of the 6 PCR product tubes.
1
2
3
4
5
6
7
8
9
10
11
12
Ladder
Gal4 DBD 1
Gal4 DBD 2
Gal4 DBD 3
Barnase 1
Barnase 2
Barnase 3
Ladder
[click to enlarge]
QIAQuick Purification of PCR Product
Because colonies 1 and 2 of Barstar worked properly on last week's gel, the PCR products in those two tubes were combined and purified using the Qiagen PCR purification protocol.
as there was about 25μL of PCR product in the two together, 125μL of PB buffer was added (5x the concentration of the PCR product)
pipetted the mix into a QIAquick spin column, spun for 30 seconds at 13,000 rpm
discarded flow-through
added .75mL PE buffer to the column, spun for 30 seconds
discarded flow-through, and spun again for 1 minute
moved the column to a fresh eppendorf tube
added 50 μL EB buffer to the column, spun for 1 minute
labeled the eppendorf tube 'Barstar PCR purified'
06-29-2010 [ top ]
B21 plasmid maxiprep
colony inoculated overnight and medium was centrifuged to obtain a pellet.
followed instructions as described in the QIAgen Plasmid Purification Handbook pg. 19-23. (2005 edition)
06-30-2010 [ top ]
pMT1002 Miniprep
4ml from each overnight cell culture of pMT1002 (1,2&3) were transferred into 5 ml conicals and spun for 6 minutes at 4400 rpm
remainder of overnight cell cultures were placed in the fridge for transfer to microcentrifuge tubes at a later time
excess LP+amp decanted, the pellet resuspended in 250μL P1
contents transferred from each 15ml conical to a new epindorf tube
250 μL P2 added to each tube, mixed by inverting gently 4-6 times
350 μL N3 added to each tube, mixed by inverting as before
tubes centrifuged for 10 minutes at 13,000 rpm
supernatants pipetted into new QIAprep sin columns
columns centrifuged for 30-60 seconds, flow through discarded
QIAprep columns washed with 0.5 ml buffer PB each and centrifuged for 30-60 seconds
columns washed with 0.75 ml buffer PE and again centrifuged for 30-60 seconds
flow through was discarded and tubes centrifuged for an additional minute
column was placed in a clean 1.5 ml microcentrifuge tube
50 μL buffer EB was added, tubes were let stand for 1 minute and centrifuged for one minute after
flow through kept, eppendorfs labeled pMT1002 plus their colony number
Also see pages 22-23 of the QIAprep Miniprep Handbook for miniprep instructions. pMT1002 Miniprep Nanodrop values
Plasmid
Quantity (ng/μL)
260/280
pMT1002 #1
68.2
1.84
pMT1002 #2
138.7
1.84
pMT1002 #3
127.2
1.88
PCR of miniprepped Barnase
20μL per tube
In each tube:
1μL Forward primer
1μL Reverse primer
0.5μL polymerase
4μL 5X buffer
2μL DNTPs
1μL DNA sample
10.5μL water
3 samples were made for the Barnase
B21 Plasmid Digest put 10x green FD buffer on ice to thaw.
Tube A:
1μL Xba1
1μL Pst1
2μL green 10x FD buffer
3.4μL colony A plasmid (the amount used to get .5μg of plasmid)
12.6μL H2O (to bring the total to 20μL)
Tube B:
1μL Xba1
1μL Pst1
2μL green 10x FD buffer
1.7μL colony B plasmid (the amount used to get .5μg of plasmid)
14.3μL H2O (to bring the total to 20μL)
Pst1 and Xba1 digest of B21 plasmid, lane 4 and 5 cut, 7 and 8 uncut
[click to enlarge]
Second Attempt
Tube A:
1μL Xba1
1μL Pst1
5μL green 10x FD buffer
12.5μL colony A plasmid (the amount used to get .5μg of plasmid)
30.5μL H2O (to bring the total to 50μL)
Tube B:
1μL Xba1
1μL Pst1
5μL green 10x FD buffer
6.6μL colony B plasmid (the amount used to get .5μg of plasmid)
36.4μL H2O (to bring the total to 50μL)
Ran on a 1% agarose gel, 100 Volts for 30-40 mins.
Lane 2 buffer, lane 8 and 9 Colony B, lanes 15 and 16 colony A
[click to enlarge]
Cut out the backbone (upper band, and placed into four eppendorf tubes, labeled 'B21 backbone' and with their respective weight before the gel was added. Placed all four in the iGEM team fence box in the -4°C freezer to be purified tomorrow.
Other
Designed primer sequences for ACC synthase degradation site, linker sequence connecting Barnase to the degradation site.
Designed primers to amplify germination/embryogenesis promoter and AtArp2 promoter.
07-01-2010 [ top ]
B21 Backbone Gel Extraction
followed QIAquick gel extraction handbook except melting temperature was 42°C instead of 50°C
4 gels to begin with, highest nanodrop value before consolidation = 1.4ng/μL
The 4 tubes were consolidated into 2 and placed in the vacufuge for ≈30 minutes to increase DNA concentration
The 2 tubes were consolidated into 1 for a final concentration of 18.6ng/μL
PMT1002 Digestion
2 reactions each containing the following:
1μL EcoRI
1μL HindIII
2μL 10X digestion buffer
1/2 ng of sample ≈7μL
9μL dH2O
only one band was visible in the gel for each reaction.  Julia forgot to put the ladder in so it was not possible to precisely determine the single band size, but judging from the position of the loading dye it seemed that the plasmid had only been cut once.
Ligation
Ligation performed on Barstar, NLS, and LacIN
3 times as much insert as vector per reaction, length of insert taken into consideration.
Barstar: 0.4 kb = 18.5ng
NLS: 0.06 kb = 3ng
LacIN: 1.1 kb = 52 ng
Ligation Recipe:
10μL insert + backbone
10μL ligation buffer (vortexed thoroughly)
1μL ligase, mix gently
let stand for 15 minutes at room temperature, then put on ice use for 4μL of the product for transformation and freeze the rest
LacIN
3μL B21 vector
7μL LacIN insert
NLS
3μL B21 vector
1μL NLS insert
6μL dH20
Barstar
3μL B21 vector
1.7μL Barstar
5.3μL dH20
Transformation from ligation
thawed 4 chemically competent TOP10 cell tubes on ice
added 4μL of each ligase rxn to each incubated on ice for 30 minutes
heat shock at 42°C for 45 seconds
put on ice for 2 minutes
added 170μL SOC medium to each tube set to shake at 37°C for 30 minutes
pippetted each tube onto its own LB+amp plate, streak with beads
incubate at 37°C overnight
07-02-2010 [ top ]
pMT1002, pMT413 Digest
Digested with EcoR1 and HindIII
Lane 2 KB ladder
Lane 4 pMT1002
Lane 6 pMT413
[click to enlarge]
Barnase PCR
Standard PCR measurements were used except that a new 50X dNTP mix was used in place of the 5X buffer used in all previous PCR reactions.  The amount of dH20 was adjusted for the new quantity of dNTP mix accordingly.
[click to enlarge]
07-06-2010 [ top ]
Inoculated 7 Barstar and 3 NLS colonies from Thursday's ligation.
Performed PCR on LacIn (E10) following standard PCR measurements as previously described.
Digestion of B21 backbone using Xba1 and Pst1
Total volume per reaction: 50 μL
2μL Xba1
2μL Pst1
10μL B21 plasmid (0.3μg/μL X 10μL = 3μg DNA)
5μL 10X digestion buffer
31μL H2O
[click to enlarge]
Digestion successful, larger bands removed for gel extraction.
Gel extraction performed according to QIAgen gel extraction protocol.
07-07-2010 [ top ]
Miniprep of Barastar and NLS Ligation products
Performed miniprep of overnight cell cultures of 7 colonies of Barstar, 3 colonies of NLS.
Minipreps performed according to pages 22-23 of the QIAprep Miniprep Handbook .
[click to enlarge]
Digestion of Barstar and NLS Ligation products
7 reactions of Barstar and 3 reactions of NLS
In each reaction:
1μL Xba1
1μL Pst1
2.5μL of sample, Barstar or NLS Plasmid, respectively, obtained from miniprep (above)
2μL Buffer
13.5μL DH2O
Ran on a 2% agarose E-gel
Lane 1: Ladder
Lane 2-8: Barstar
Lane 9-11: NLS
[click to enlarge]
PCR of LacIN 3 reactions, each with:
1μL Fwd LacIN Primer
1μL Rev LacIN Primer
.5μL Polymerase
2μL DNTP
4μL Buffer
1μL E10 Plasmid (LacI)
10.5μL DH2O
Using program:
1= 95°C for 10:10
2= 95°C for :15
3= 50°C for :30
4= 72°C for 1:30
5= goto 2, 29 times
6=72°C for 10:10
7=4°C forever
Gel Image of LacIN PCR result:
PCR of Barnase and Gal4DBD
5 reactions each of Barnase from pMT413, Barnase from pMT1002, and Gal4
In pMT413 Barnase reactions:
1μL Barnase2.Fwd Primer
1μL Rev.Barnase Primer
.5μL Polymerase
2μL DNTP
4μL Buffer
2μL pMT413 plasmid
10.5μL DH2O
In pMT1002 Barnase Reactions:
1μL Barnase2.Fwd Primer
1μL Rev.Barnase Primer
.5μL Polymerase
2μL DNTP
4μL Buffer
2μL pMT1002 plasmid
10.5μL DH2O
In Gal4DBD reactions:
1μL Gal4.Fwd Primer
1μL Gal4.Rev Primer
.5μL Polymerase
2μL DNTP
4μL Buffer
2μL Gal4DBD plasmid
10.5μL DH2O
07-08-2010 [ top ]
[click to enlarge]
After receiving the new primers we performed PCR on barnase vectors PMT413 and PMT1002, LacIN, as well as the GAL4 DNA binding domain.  The ladder is in lane 2, the GAL4 PCR products are in lanes 5 through 9, the barnase PMT413 lanes 10 through 14, and the LacIN PCR product is in the last 3 lanes.  After PCR was confirmed successful, we decided to gel purify the LacIN because of the multiple band sizes seen in the gel.  Below is the gel where LacIN was gel extracted, the chunk missing from the gel is where LacIN was identified to be.
[click to enlarge]
Gel extraction was performed according to the QIAgen gel extraction kit protocol using a microcentrifuge.
Barstar Sequencing Submited plasmids from each of the 7 Barstar colonies for sequencing.
07-09-2010 [ top ]
Digest of yesterday's ligations LacIN - 65μL
1μL Xba1   1μL Pst1  6.5μL FD buffer  1μL DPN1  50μL LacIN  6.5μL DH2O
Gal4DBD - 50μL
1μL Xba1  1μL Pst1  7μL Gal4  1μL DPN1  5μL FD Buffer  36μL DH2O
Barnase - 50μL
1μL Xba1  1μL Pst1  15μL Barnase  1μL DPN1  5μL FD buffer  28μL DH2O
Put in 37°C bath for 30 mins
Following proceedure for PCR cleanup to clean up the digestion results
added 5 volumes PB buffer to 1 volume digestion product
325μL for LacIN, 250μL for Gal4DBD and Barnase
pippetted each into a QIAquick column, spun for 30 secs
discarded flow through, put columns back in same tube, added 750μL PE buffer
spun for 30 secs, discared flow through
Spun again for 1 min
placed column in new eppendorf
added 30μL EB buffer to elute, spun for 1 min
Eppendorfs labeled "LacIN digest cleanup 7/9," "Gal4 digest cleanup 7/9," and "Barnase digest cleanup 7/9"
7-10-10 (Saturday)
Innoculating colonies from Friday's ligations
Barnase, Gal4, LacIN
4 cc tubes each, 5mL LB+amp, set to shake at 37°C overnight
7-11-10 (Sunday)
Miniprep of Barnase, Gal4 and LacIN ligations
No cells found in Barnase colonies 1 and 2 --> innoculations 3 and 4 renamed Barnase 1 and 2, replacing the cell-free cultures.
pipetted 4 ml from each overnight cell culture (excepting the cell-free Barnase cultures) into a 15 ml conical *centrifuged at 4400 rpm for 6 minute
remaining overnight cell cultures were placed in fridge
decanted LB+amp, resuspended cells in 250 μL P1 buffer
contents transferred to eppendorfs
250 μL of P2 buffer was added per tube, the tubes were inverted 4-6 times
350 μL N3 buffer added to each, tubes inverted 4-6 times
centrifuged for 10 min at 13,000 rpm, supernatant collected ad transferred to QIAprep spin columns
centrifuged for 30-60 seconds, flow through discarded
0.5 ml buffer PB added and centrifuged for 30-60 seconds, flow through discarded, spun columns for an additional minute
QIAprep columns put into new eppendorfs
50 μL buffer EB added to columns, let stand for 1 minute
centrifuged for 1 minute at 13,000 rpm
07-12-2010 [ top ]
Nanodrop of Barnase, Gal4 and LacIN Minipreps
Plasmid
Quantity (ng/μL)
260/280
Barnase 1
226.1
Barnase 2
336.3
1.94
Gal4 1
298.0
1.88
Gal4 2
363.3
1.92
Gal4 3
301.9
1.89
LacIN 1
376.7
1.89
LacIN 2
237.8
1.90
LacIN 3
339.4
1.92
LacIN 4
368.9
1.92
Digestion of Ligation product minipreps Xba1 and Pst1
Mastermix (10x)
10μL Xba1
10μL Pst1
20μL Buffer
Barnase 1
4μL Barnase 1   12μL DH2O   4μL Mastermix
Barnase 2
3μL Barnase 2   13μL DH2O   4μL Mastermix
Gal 1
3μL Gal 1   13μL DH2O   4μL Mastermix
Gal 2
2.5μL Gal 2   13.5μL DH2O   4μL Mastermix
Gal 3
3μL Gal 3   13μL DH2O   4μL Mastermix
LacIN 1
2.5μL LacIN 1   13.5μL DH2O   4μL Mastermix
LacIN 2
4μL LacIN 2   12μL DH2O   4μL Mastermix
LacIN 3
2.5μL LacIN 1   13.5μL DH2O   4μL Mastermix
LacIN 4
2.5μL LacIN 1   13.5μL DH2O   4μL Mastermix
Put tubes in 37°C heat bath for 30 mins, then put them on ice.
Gel of Digestions:
[click to enlarge]
Sequencing
submitted Barnase 1 and 2, Gal4 1,2, and 3, and LacIN 1,2,3, and 4 for sequencing
07-13-2010 [ top ]
[click to enlarge]
[click to enlarge]
The yeast backbones containing the PIF3 and PHYB inserts was isolated and purified.  The latter image shows where the appropriately sized bands were excised.
Nanodrop values of final product are:
PHYB 11ng/μL PIF3 20ng/μL
07-14-2010 [ top ]
Phenol Chloroform DNA Extraction
resuspend DNA in approximately 200μL molecular grade pure water
add 1 volume phenol chloroform
when pipetting the phenol chloroform be sure to pipette from closer to the middle level of the liquid as the top two centimeters do not contain what we want
centrifuge at top speed for 5 minutes
collect the aqueous top layer and transfer to a clean microcentrifuge tube
the remaining liquid is toxic and must be carefully disposed of in the "toxic waste" bottle in the fume hood
add 1 volume phenol chloroform
centrifuge at top speed for 5 minutes
collect the aqueous top layer and put into a fresh tube
add three volumes 100% ethanol and 1/10 volume sodium acetate
allow solution to incubate on ice for 15 minutes
centrifuge at top speed for 10 minutes
pipette off the aqueous solution leaving a pellet of DNA in the bottom of the tube
wash in 500μL 70% ethanol
mix tube by inverting
centrifuge again for 10 minutes at top speed
air dry pellet by leaving tube open on bench (not on ice) til dry
after pellet has dried resuspend DNA in desired volume of H2O or buffer EB
presence of DNA can be confirmed by running a small sample on a gel
NLS Ligations Ligation of NLS.Serine
2.5μL Backbone
1.5μL of 1/100 dilution (1.5ng NLS.Serine)
6μL H2O
10μL 2x quick ligase buffer
1μL quick ligase
B21 control
2.5μL B21 Backbone
7.5μL DH2O
10μL 2x Quick Ligase Buffer
1μL Quick Ligase
Transformation
1μL NLS lig into TOP10 cells
Chilled on ice for 30 mins
Heat shocked at 42°C for 30 seconds
Put on ice for 2 mins
Added 170μL SOC medium
Set to shake  in incubator for 30 mins
Streaked on LB+amp plates, incubated overnight
Yeast backbone gel purified nanodrops
Sample
Concentraion in ng/μL
260/280
31 (1)
8.5
3.00
31(2)
10.2
2.07
32(3)
11.1
2.31
52(4)
28.5
1.83
07-15-2010 [ top ]
Our previous attempt resulted in DNA with the following nanodrop values:
12.1ng/μL 260/280: 0.9
39.1ng/μL 260/280: 1.11
It was believed that the other genomic DNA provided by the Matthews Lab before had similar values with regards to purity/contamination (also approaching 1)  Both team allergy and fence performed PCR multiple times unsuccessfully with these DNA samples.  It was thought that the low purity may be inhibiting the PCR.
Today we used the QIAgen DNeasy Plant Mini Kit.  Liquid nitrogen was poured over the arabidopsis leaves in a mortar while they were ground with the pestle.  This produced a fine green powder of arabidopsis leaves from which genomic DNA was extracted starting with step seven of the QIAgen protocol following liquid nitrogen crushing.  The only change made to the protocol was that 40 μL of RNase 1 was added per reaction instead of the recommended 4 μL because the concentration of our supply was ten-fold lower.
Nanodrop values were as follows:
6.7 ng/μL 260/280: 1.42
5 ng/μL 260/280: 1.48
07-16-2010 [ top ]
PCR of Arabidopsis Genomic DNA, attempt 2 PCR using the newly extracted DNA was unsuccessful.
The degradation signal from the Austrians was successfully inoculated and colonies were found on the agar plates.  Individual colonies from the plates were inoculated into tubes of 5 ml LB+Amp and placed in the shaking incubator overnight. (37°C)
Making Glyerol Stocks Combined .5 mL 80% glycerol with .5mL overnight cell culture of: Barnase 3 (number changed to Barnase 1 from miniprep onward), LacIN1, and Gal2.
7-17-10 (Saturday)
5xGal4 Promoter Annealing Product Ligation
Made a 1 in 1000 dilution of the annealing reaction
Nanodrop
ng/μL
260/280
5xGalpt 1/1000 dilution
17.6
2.25
For a ligation reaction, we want:
50ng of backbone
A 1:3 ratio of backbone:insert
3(ng of backbone / basepairs of backbone) = (ng of insert / basepairs of insert)
5xGalpt is 180 bases long, B21 backbone is 3.2kb. So we want 8.4375 ng of plasmid
To reach 8.4ng of 5xGalpt, made a 1 in 2000 dilution of the 5xGalpt Annealing product
Ligation Reaction:
1μL 1/2000 5xGalpt
2.5μL B21 Backbone
6.5μL DH2O
1μL Quick Ligase
10μL 2x Quick Ligase Buffer
B21 Control Ligation:
2.5μL B21 Backbone
7.5μL DH2O
1μL Quick Ligase
10μL 2x Quick Ligase Buffer
Incubated at room temp for 15 mins, put on ice and immediately transformed.
Transformations of 5xGalpt, NLS.Serine anneal, and B21 Backbone control
Thawed 3 Turbo chemically competent cells on ice
Pippeted 1μL of the ligation or annealing reaction into its own tube of turbo cells
Chilled on ice for 30 mins
Heatshocked in 42°C water on heat block for 30 seconds
Chilled on ice for 2 mins
Added 170μL SOC medium to each tube, set to shake at 37°C for 15 mins
Streaked on LB + amp plates, and left in incubator overnight
7-18-10 (Sunday)
Re-doing 5xGalpt and NLS.Serine Annealing Reactions Set ~400ml of water to boil in a beaker on a heat block.
Phosphorylating 5xGalpt
3μL 100μM oligo1.5xgal4
3μL 100μM oligo2.5xgal4
3μL 100μM oligo3.5xgal4
3μL 100μM oligo4.5xgal4
3μL 10x PNK buffer
2μL nM ATP
2μL t4 PNK
11μL DH2O
put in 37°C waterbath for 90 mins Then added 4μL 5M NaCl to halt the reaction
To get rid of the NaCL so that I can ligate the overlapping oligos:
added 3 volues of 100% ethanol (90μL), spun at top speed for 20 mins
pippetted off and discarded the supernatant, added 100μL of 70% ethanol, broke up the pellet, and spun for 5 mins
pippeted off supernatant, allowed to air dry.
added 2μL T4 ligase buffer, 1μL T4 ligase, and 17μL DH2O, using this mix to resuspend the pellet
set in PCR machine at 16°C for 12 hours, then 4°C until stopped.
Annealing NLS.Serine (again)
3μL NLS.Serine.Fwd
3μL NLS.Serine.Rev
2μL 10x annealing buffer
12μL DH2O
Put in boiling water in beaker (see above), allowed to boil for 2 mins, then removed beaker from heat block, allowing the water to slowly cool with the annealing reaction tube still in it.
07-21-2010 [ top ]
Nanodroping 5xGal promoter
Nanodrop of ligated annealed oligos: 5200.8 ng/μL,   1.32 260/280
Gel Extraction of 5xGal promoter
poured 2% agarose Gel for gel extraction, ran 2μL of the ligated annealing reaction in 13μL EB buffer, with two KB plus 1 ladders, one on each side.
Gel extraction performed, into an eppendorf weighing .9563g
[click to enlarge]
Nanodrop for cleaned-up gel purified 5xGalpt negative - ran a 2% E-gel to check. There appears to be something at the .16kb size.
[click to enlarge]
07-22-2010 [ top ]
Sequencing Results
Barstar samples 1, 2, 5, and 6 were confirmed to be correct.
The LacI sequence was confirmed, however we were unable to confirm the presence of the NLS because of its position at the end of the gene where sequencing results are generally unreliable. (NNNNN)  Lac1R and Lac3R were poor matches because of what GENEWIZ calls "Early Termination".
Barnase had some serious issues. We are re-growing colonies following ligation and intend to submit some new PCR product as well as the original ADDGENE plasmid for sequencing in the near future.
The Gal4 DNA binding domain samples 1 and 2 were perfect except for a SNP around 320 (in the case of GAL DBD 1) for our sequence and 223 of the biobricks sequence.
Gal4 DBD sample 3 was unsuccessful.
Arabidopsis PCR
[click to enlarge]
FINALLY! EXP2 PROMOTER AMPLIFIED!
~1000kb
[click to enlarge]
Fragments were gel isolated and purified using the QIAgen Gel Purification Kit with the Gel Extraction protocol using a microcentrifuge.
07-23-2010 [ top ]
No colonies from last night's ligations.
7-24-10 (Saturday)
Successful ACC synthase transformation:
[click to enlarge]
PFX polymerase PCR
1μL 10mM MgSO4
2μL 10X pfx amplification buffer
2μL 10X PCRx enhancer solution
1μL F primer
1μL R primer
0.5μL pfx polymerase
2μL DNTP
9.5μL total
DNA: 21ng/μL
50 to 200μL DNA per reaction
5 reactions
4μL per reaction = 20μL sample total (MM)
9.5μL + 4μL = 13.5μL therefore 6.5μL H20 per reaction
MM
5μL 10nM MgSO4
10μL 10X pfx amplification buffer
10μL 10X PCRx enhancer solution
5μL F primer
5μL R primer
2.5μL pfx polymerase
10μL DNTP
20μL genomic DNA
32.5μL H20
total volume:100μL
Arabidopsis PCR
[click to enlarge]
Transformations
Thawed 2 turbo cells on ice to thaw
Resuspended EcR, RXRHm, RXRLc tubes from Mr.Gene into 50μL DH2O (5μg into 50μL)
Pippetted .5μL plasmid from each of the 5 synthesis products (EcR, RXRHm, RXRLc, ACC, and  ACT2LacOpt) into a labled eppendorf containing 20μL of turbo cells each
Pippetted 2μL of the relavent ligation reaction from Friday (Strep.Barnase 2 ligation, Strep.Barnase 1, Gal4 DBD lig) into eppendorfs containing 20μL turbo cells (barn1strep, barn2strep and Gal4DBD)
Chilled on ice for 15 mins
Put in 42°C heat bath for 30 secs
Put on ice for 2 mins
Added 200μL SOC medium (to equal 10x the amount of cells)
Set to shake at 37°C for 30 mins (longer for Act2pt because kan selective cultures need more time to become established before facing the harsher Kan selection medium
Streaked 20μL on LB+amp plates
Put in incubator to shake overnight
After 45 mins to 1.5 hours, streaked Act2LacOpt on Kan+LB plate, put in incubator to shake overnight.
Making LB+Kan Plates
made 4 LB+Kanamycin plates
15μL Kanamycin   30μL DH2O to ease streaking
Streaked this on LB agar plates with glass beads, and then turned the plate upside down and let dry until it was ready for bacterial sequencing, and then those same beads were reused.
Transformation of pLAS and Terminator Resuspended the following from the registry with 10μL DH2O
p(LAS)TetO, on pSB2K3 (kan res.), from plate 2, sell 11N   Terminator, on pSB1AK3 (kan and amp res.), from plate 1, well 4H
07-27-2010 [ top ]
Innoculations
RXRHuman, RXRLocust, Barn1strep, Barn2strep, ACC, and EcR
(these are all amp resistant)
6 colonies of each are inocculated, except for Barn1 and Barn2, of which 4 are innocculated each.
5mL LB+amp and a smear of each colony from a relevant plate
The same is done in LB+Kan for pLAS and Act2lacO promoter, which are Kanamycin resistant.
Redoing Barnase-strep Tag Ligation poured a 2% agarose gel to gel-purify the barnase from  Barnase1 Xba Pst1 cleanup and Barnase2 Xba Pst1 cleanup, both from (7/23)
Gel ran empty except for DB ladders
Phosphatase Treating B15 Spe1/Pst1 Cleanup
to prevent religation of the whole vector
35μL B15 Spe1 Pst1 cleanup from 7/23
4μL 10x green FD buffer
1μL Alkaline phosphatase
07-28-2010 [ top ]
Minipreps
Miniprepped and Nanodroped the following:
Plasmid
Quantity (ng/μL)
260/280
Act2LacO 1
46.1
1.92
RXRHm 4
23.8
2.15
RXRHm 2
45.2
1.94
Terminator 6
172.8
1.93
Terminator 3
456.5
1.91
RXRLc 2
42.7
1.76
RXRLc 4
58.5
2.03
RXRLc 5
45.1
2.06
ACC 3
91.5
2.01
Barn 2 Strep 3
41.0
2.12
Barn 2 Strep 2
90.3
1.95
Barn 1 Strep 2
67.1
1.99
Barn 1 Strep 3
92.9
1.99
EcR 5
115.3
1.95
EcR 2
108.0
1.97
EcR 4
79.4
1.97
EcR 3
111.6
1.96
EcR 6
95.8
1.96
Barnase Digest Gel Extraction Again blank on the first attempt, tried again:
Success!
2
5
8
9
10
18
DNA
KB+ ladder
15μL Barnase2 digest 1/5 dilution 7/23
20μL 1/3 dilution Barn2 digest cleanup 7/23
5μL 1/3 dilution Barn2 digest cleanup 7/23
20μL 1/3 dilution Barn2 digest cleanup 7/23
KB+ ladder
[click to enlarge]
07-29-2010 [ top ]
Digest of Wednesday's minipreps
Gel Extraction of Barnase insert
followed gel extraction protocol in QIAquick Spin Handbook
Nanodrops:
Plasmid
Quantity (ng/μL)
260/280
Tube #1
9.7
1.79
Tube #2
17.4
1.57
Tube #3
2.7
1.24
Samples 1-9
[click to enlarge]
Lane 1: ladder
Lane 2: ladder
Lane 3: ACC synthase 1
Lane 4: ACC synthase 2
Lane 5: Acc synthase 3
Lane 6: Terminator 1
Lane 7: Terminator 2
Lane 8: Terminator 3
Lane 9: Act2LacI 1
Lane 10: Act2LacI 2
Lane 11: Act2LacI 3
Lane 12: ladder
insert sizes:
ACC synthase: 300 to 350
Terminator: unknown
Act2LacI: ~1200
Samples 10-18
[click to enlarge]
Lane 1: empty
Lane 2: ladder
Lane 3: Barn1-strep1
Lane 4: Barn1-strep2
Lane 5: Barn1-strep3
Lane 6: Barn2-strep1
Lane 7: Barn2-strep2
Lane 8: Barn2-strep3
Lane 9: EcR2
Lane 10: EcR3
Lane 11: EcR4
Lane 12: ladder
insert sizes
Barn1-strep: <400bp
Barn2-strep: <400bp
Ecr: ~1000bp
Samples 19-24
[click to enlarge]
Lane 1: empty
Lane 2: ladder
Lane 3: empty
Lane 4: RxR-Hm 1
Lane 5: RxR-Hm 2
Lane 6: RxR-Hm 3
Lane 7: RxR-Lc 1
Lane 8: RxR-Lc 2
Lane 9: RxR-Lc 3
Lane 10: empty
Lane 11: empty
Lane 12: ladder
insert size
RxR-Lc: 600-700bp
RxR-Hm: 600-700bp
Summary
We see one appropriately sized band (~350bp) from the ACC synthase miniprep #1, the others did not have any bands.  Therefore henceforth we will only use plasmid from miniprep #1 for our reactions.
Terminator size unknown at present.
For the Act2LacI promoter we saw in all three lanes the insert sized at ~1200bp.  All three minipreps should be fine to use for future reactions.
The Barnase-strep lanes all displayed inserts of the correct size, ~350 to 400 bp.  All minipreps should be good for future reactions.
The Ect. Receptor insert appears to be about the right size, >1000bp, although the ladder is a bit hard to decipher, all minipreps should be good for future reactions.
The RxR-Hm did not show any evidence of the presence of an insert.
The RxR-Lc did show evidence of the presence of an insert, although it may be a bit larger than the expected size (600-700bp).  Blurriness of the ladder makes it difficult to decipher.
Nanodrop of Gal4DBD Minipreps (from the beginning of the summer
Plasmid
Quantity (ng/μL)
260/280
Gal4DBD #1
26.5
1.89
Gal4DBD #3
16.7
1.85
Gal4DBD #4
19.1
1.83
Gal4DBD #5
82.2
2.16
PCR of VP16 and Gal4DBD (preparing for yeast vector insertion) Gal4DBD
2μL DNTPs
.5μL phusion polymerase
4μL 5x phusion buffer
1μL 1/10 dilution of BB.Gal4DBD.Rev
1μL 1/10 dilution of Bam.Gal4.Fwd
1μL 1/2 dilution Gal4DBD #3 (made 1/2 dilution such that 1μL will be <10 ng/μL
μL 10.5μLDH2O
4 tubes of each, 20μL each tube
VP16
10.5μL DH2O
2μL DNTPs
.5μL phusion polymerase
4μL 5xphusion buffer
1μL 1/10 dilution of VP16.Hind.Fwd
1μL 1/10 dilution of VP16.Rev
1μL 1/10 dilution of VP16 #1
PCR program: VP16 Gal4
1= 98°C for 10:00
2= 98°C for :15
3= 50°C to 60°C for :30
4= 72°C for 1:30
5= goto 2, 29 times
6= 72°C for 10:10
7= 4°C forever
07-30-2010 [ top ]
PCR of VP16 and Gal4DBD
1
2
3
4
5
6
7
8
9
10
11
12
KB+ Ladder
Gal4 DBD 1
Gal4 DBD 2
Gal4 DBD 3
Gal4 DBD 4
VP16 1
VP16 2
VP16 3
VP16 4
KB+ Ladder
[click to enlarge]
PCR again for VP16 Using pfx polymerase
1μL 1/10 dilution Hind.Fwd
1μL 1/10 dilution VP16.Rev
1μL DNTPs
6μL enhancer buffer
.5μL MgSO4
.5μL PFX polymerase
1μL 1/10 dilution VP16 #1
5μL DH2O
4μL amp(lification?) buffer
Running PCR on two reactions. There was not enough PFX for both to get .5μL, so only one of them is likely to actually amplify.
Second ACC + RxrHm digest gel
[click to enlarge]
yes to ACC (left) no RxrHm
08-02-2010 [ top ]
2% E-gel of VP16 PCR product from Friday
Lane
1
2
3
4
5
6
7
8
9
10
11
12
KB+ Ladder
VP16 PCR product 1
VP16 PCR product 2
KB+ Ladder
[click to enlarge]
Gel extraction Gal4 spe bam 8/2 and EcR bam xba 8/2 gel extracted and purified.
Gal4 on the left, EcR on the right
[click to enlarge]
Nanodrop of Gal4 and EcR Gel Extractions
Plasmid
Quantity (ng/μL)
260/280
Gal4 (1)
1.4
-3.18
Gal4 (2)
7.2
4.59
EcR (1)
5.3
2.52
EcR (2)
5.6
2.12
VP16 PCR 2 60μL reaction:
1.5 μL pfx polymerase
3μL DNTPs
18μL enhancer buffer
1.5μL MgSO4
3μL 1/10 dilution VP16.Hind.Fwd
3μL 1/10 dilution VP16.Rev
3μL 1/10 dilution of VP16 (1)
12μL amp buffer
5μL DH2O
using program:
1=94°C for 2:00   2=94°C for :15   3=68°C for :30   4=Goto 2, 30 times   5=4.0°C forever   6=end
No PCR product was visible.
VP16 PCR.3 60μL reaction:
1.5 μL pfx polymerase
3μL DNTPs
18μL enhancer buffer
1.5μL MgSO4
3μL 1/10 dilution VP16.Hind.Fwd
3μL 1/10 dilution VP16.Rev
3μL 1/10 dilution of VP16 (1)
12μL amp buffer
5μL DH2O
Using modified program:
1=94°C for 2:00   2=94°C for :15   3=60°C for :30   4=70°C for :45   5=Goto 2, 30 times   6=4.0°C forever   7=end
(see tomorrow's entry for gel image)
Ligations
Gal4 into EcR   Barnase into B15   LTP into B11 (positive control from team allergy)
Gal4
3μL Gal4 spe bam gel pur. (2) 8/2
9μL EcR xba bam gel pur. (2) 8/2
2μL 10x T4 ligase buffer
5μL DH2O
1μL T4 ligase
EcR control
9μL EcR xba bam gel pur. (2) 8/2
2μL 10x T4 ligase buffer
8μL DH2O
1μL T4 ligase
Barnase
1.5μL Barnase digest gel pur. (1) 7/29
3μL B15 spe pst phosphorylated clean up
2μL 10x T4 ligase buffer
12.5μL DH2O
1μL T4 ligase
B15 control
3μL B15 spe pst phosphorylated clean up
2μL 10x T4 ligase buffer
14μL DH2O
1μL T4 ligase
LTP
2μL LTP
1μL B11 backbone
2μL 10x T4 ligase buffer
16μL DH2O (should have been 14μL, the 16 was accidental)
1μL T4 ligase
B11 control
1μL B11 backbone
2μL 10x T4 ligase buffer
16μL DH2O (should have been 14μL, the 16 was accidental)
1μL T4 ligase
Allowed tubes to sit at room temp for an hour, proceeded to transformation
08-03-2010 [ top ]
VP16 PCR from last night
[click to enlarge]
Yesterday's transformations
[click to enlarge]
[click to enlarge]
[click to enlarge]
[click to enlarge]
[click to enlarge]
[click to enlarge]
[click to enlarge]
[click to enlarge]
Glycerol Stocks from 7-27-10
keeping Act2lac0pt 1, ACC synthase 1, EcR 2, RXRLc 2
added .5mL 80% glycerol to .5mL overnight cell culture of selected transformation, vortexed thoroughly, and placed in the -80°C freezer in team fence box.
Annealing 35s min promoter
3μL 100μM 35sminpromt.Rev
3μL 100μM 35sminpromt.Fwd
3μL .5M NaCl
3μL 10x PNK buffer
18μL DH2O
placed eppendorf in boiling water for 2 mins, then removed water from heat and allowed water and eppendorf to slowly cool together.
08-04-2010 [ top ]
PCR of VP16.3 60μL reaction:
1.5 μL pfx polymerase
3μL DNTPs
18μL enhancer buffer
1.5μL MgSO4
3μL 1/10 dilution VP16.Hind.Fwd
3μL 1/10 dilution VP16.Rev
3μL 1/10 dilution of VP16 (1)
12μL amp buffer
5μL DH2O
Using program:
1=94°C for 2:00   2=94°C for :15   3=60/68°C for :30   4=68°C for :10   5=Goto 2, 30 times   6=4.0°C forever   7=end
Digestion of Gal-EcR BamH1 and Nco1 12 digestions, one for each Gal-EcR miniprep Each:
1μL BamH1
1μL Nco1
1μL 10x FD green buffer
2-7μL EcR-Gal miniprep
5-0μL DH2O, to bring total volume to 10μL
Ran a 1% gel of the digest
1
2
3
4
5
6
7
8
9
10
11
12
13
14
EcR Digest 1
EcR Digest 2
EcR Digest 3
EcR Digest 5
EcR Digest 6
EcR Digest 7
EcR Digest 8
EcR Digest 9
EcR Digest 10
EcR Digest 11
EcR Digest 12
KB+ ladder
[
[click to enlarge]
Ligated EcR to Gal4 and Barnase to Strep
08-05-2010 [ top ]
Colonies from yesterday's ligations
[click to enlarge]
[click to enlarge]
Gel extraction of EcR-Gal Bam Nco
applied 3x gel volume of QG buffer (in μL) to each respective tube
placed in 42°C bath until the gel was melted
added 1 gel volume of isopropanol to each, and vortexed
applied contents of each tube to a QIAquick spin collumn
spun for 1 min
added 500μL QG buffer (to dissolve any remaining agarose)
spun for 1 min
added 750μL PE buffer
spun for 1 min, discared flow-through
spun for another minute
placed QIAquick collumn in a fresh, labeled eppendorf tube, applied 50μL EB buffer to elute, and spun for 1 min
Nanodrop of Gal4 and EcR Gel Extractions
Plasmid
Quantity (ng/μL)
260/280
Gal-EcR (1)
3.1
2.45
Gal-EcR (2)
4.5
1.72
Gal-EcR (3)
3.0
11.56
Gal-EcR (4)
3.6
2.21
Digestions VP16
1μL xba
1μL Pst
5μL VP16(2) miniprep
1μL 10x FD green buffer
2μL DH2O
5xGalUAS
1μL xba
1μL Pst
2μL 5xGal4UAS (from Karmella)
1μL 10x FD green buffer
5μL DH2O
PhyB
1μL Bam
1μL Nco
16μL PhyB
2μL 10x FD green buffer
Sending EcR-Gal for sequencing
5μL of each primer, either fwd or rev, at 1/20 dilution
10μL of sample and DH2O, such that sample is at 500ng total mass
Sealed tubes with parafilm. Submited Gal-EcR 1, 7, 10, and 12 (the one's with inserts on the digestion gel from 8/4)
08-06-2010 [ top ]
Digestions pACT2
1μL Bam
1μL Xho
6μL pACT2 (tube with a green top)
1μL FD Green buffer
1μL DH2O
RXRLc
1μL Bam
1μL Xho
9μL RXRLc4
2μL FD Green buffer
7μL DH2O
Barstar
1μL Eco
1μL Pst
2μL Barstar 3
1μL FD Green buffer
5μL DH2O
Barnase
1μL Eco
1μL Pst
1.5μL Barnase 2
1μL FD Green buffer
5.5μL DH2O
5xGalUAS
1μL Spe
1μL Pst
2μL 5xGalUAS (from Karmella)
1μL FD Green buffer
5μL DH2O
pENT Cup
1μL Spe
1μL Pst
1μL pENT Cup
1μL FD Green buffer
6μL DH2O
EcR-Gal
1μL Xba
1μL Pst
2μL EcR-Gal 7
1μL FD Green buffer
5μL DH2O
VP16
1μL Spe
1μL Pst
5μL VP16(1)
1μL FD Green buffer
2μL DH2O
RXRLc
1μL Xba
1μL Pst
9μL RXRLc4
2μL FD Green buffer
7μL DH2O
V24
1μL Eco
1μL Pst
3μL V24
1μL FD Green buffer
4μL DH2O
V26
1μL Eco
1μL Pst
9μL V26
12μL FD Green buffer
7μL DH2O
PhyB
1μL Bam
1μL Nco
16μL PhyB
2μL FD Green buffer
08-09-2010 [ top ]
[click to enlarge]
yes colonies for ligation of exp2 and transformation of camv
Digests
[click to enlarge]
ladder
1. pACT2 (backbone)
2. RxRLC (insert)
3. Barstar (insert)
4. Barnase (insert)
5. 5XGalUAS (backbone)
6. PENTcup (backbone)
7. V26 (backbone)
8. Ecr-Galv (insert)
no insert for 2 (RxRLC) or 8 (Ecr-Galv)
[click to enlarge]
ladder
RxRLC (Xba and Pst) (insert)
RxRLC (Bam and Xho) (insert) NO INSERT HERE
PhyB (backbone)
V24 (backbone)
VP16 (backbone)
08-10-2010 [ top ]
[Image: 810digestslater.jpg]]
ladder
PhyB (backbone)
Barstar 4 (insert)
V26 (backbone)
GalDBD1 (insert)
GalDBD5 (insert)
B21 (backbone)
Digests 8/10
PhyB with Bam and Nco
Barstar with Pst and Eco
V26 with Pst and Eco
Gal4DBD with Eco and Spe (Gal4DBD minipreps 1 and 5)
Arp2BB PCR product with Xba and Pst
Arp PCR product with Xba and Pst
B21 with Xba and Pst
ACC synthase with Eco and Spe
Barnase ligation product with Eco and Xba
08-11-2010 [ top ]
colonies from yesterday's transformations/ligations
[click to enlarge]
[click to enlarge]
08-12-2010 [ top ]
[click to enlarge]
This picture is not the best picture but we did find colonies on the plate with the arp2/b21 transformation.
[click to enlarge]
transformation from karmella's stock
08-13-2010 [ top ]
Transformations from Yesterday's Ligations
[click to enlarge]
[click to enlarge]
[click to enlarge]
[click to enlarge]
[click to enlarge]
[click to enlarge]
[click to enlarge]
[click to enlarge]
Innoculated colonies from EcR-Gal, Nost-pENT, and VP16Gal4DBD. No colonies visible on  Nost-Act2LacO.
All Controls were clean or had very very few colonies (<2).
Retrieved from " http://2010.igem.org/Team:Harvard/fences/notebook "
Recent changes
What links here
Related changes
Special pages
My preferences
Printable version
Permanent link
Privacy policy
Disclaimers
