<HTML xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en" dir="ltr">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki  ltr ns-0 ns-subject page-Team_Harvard_allergy_notebook"><DIV id="globalWrapper"><DIV id="top-section"><DIV id="p-logo"><A href="/Main_Page" title="Main Page">&quot;
	    </A></DIV><DIV id="menubar" class="left-menu noprint"><UL><LI class="selected"><A href="/Team:Harvard/allergy/notebook">Page               </A></LI><LI class="new"><A href="/wiki/index.php?title=Talk:Team:Harvard/allergy/notebook&amp;action=edit&amp;redlink=1">Discussion               </A></LI><LI><A href="/wiki/index.php?title=Team:Harvard/allergy/notebook&amp;action=edit">View source               </A></LI><LI><A href="/wiki/index.php?title=Team:Harvard/allergy/notebook&amp;action=history">History               </A></LI><LI style="color:#808080;cursor:default">teams</LI></UL></DIV><DIV class="right-menu noprint" id="menubar"><UL><LI id="pt-login"><A href="/wiki/index.php?title=Special:UserLogin&amp;returnto=Team:Harvard/allergy/notebook" title="You are encouraged to log in; however, it is not mandatory [o]" accesskey="o">Log in</A></LI></UL></DIV><DIV id="search-controls" class="noprint"><FORM action="/Special:Search" id="searchform"> </FORM></DIV></DIV><DIV id="content"><H1 class="firstHeading">Team:Harvard/allergy/notebook</H1><DIV id="bodyContent"><H3 id="siteSub" class="noprint">From 2010.igem.org</H3><DIV id="allcontent"><DIV id="headerlinks"><UL id="nav"><LI><A class="bannertoplinks" href="https://2010.igem.org/Team:Harvard">home</A></LI><LI><A class="bannertoplinks" href="#">project</A></LI><LI><A class="bannerlinks" href="https://2010.igem.org/Team:Harvard/vectors">vectors</A></LI><LI><A class="bannerlinks" href="https://2010.igem.org/Team:Harvard/allergy">allergy</A></LI><LI><A class="bannerlinks" href="https://2010.igem.org/Team:Harvard/flavor">flavor</A></LI><LI><A class="bannerlinks" href="https://2010.igem.org/Team:Harvard/color">color</A></LI><LI><A class="bannerlinks" href="https://2010.igem.org/Team:Harvard/fences">genetic fence</A></LI><LI><A class="bannerlinks" href="https://2010.igem.org/Team:Harvard/human_practices">human practices</A></LI><LI><A class="bannerlinks" href="https://2010.igem.org/Team:Harvard/safety">safety</A></LI><LI><A class="bannerlinks" href="https://2010.igem.org/Team:Harvard/parts">parts</A></LI></UL><LI><A class="bannertoplinks" href="https://2010.igem.org/Team:Harvard/results">results</A></LI><LI><A class="bannertoplinks" href="#">people</A></LI><LI><A class="bannerlinks" href="https://2010.igem.org/Team:Harvard/students">students</A></LI><LI><A class="bannerlinks" href="https://2010.igem.org/Team:Harvard/tfs">teaching fellows</A></LI><LI><A class="bannerlinks" href="https://2010.igem.org/Team:Harvard/advisors">advisors</A></LI><LI><A class="bannerlinks" href="https://2010.igem.org/Team:Harvard/credits">acknowledgements</A></LI><LI><A class="bannerlinks" href="https://2010.igem.org/Team:Harvard/gallery">gallery</A></LI><LI><A class="bannertoplinks" href="#">online</A></LI><LI><A class="bannerlinks" href="http://harvardigem.org">blog</A></LI><LI><A class="bannerlinks" href="http://twitter.com/harvardigem">twitter</A></LI><LI><A class="bannerlinks" href="http://openwetware.org/wiki/IGEM:Harvard/2010">openwetware</A></LI><LI><A class="bannerlinks" href="http://www.facebook.com/pages/Cambridge-MA/Harvard-iGEM/269794024286?ref=ts">facebook</A></LI><LI><A class="bannerlinks" href="https://2010.igem.org/Team:Harvard/contact">contact us</A></LI></DIV><DIV id="sidebar"><A class="sidebarlinks" href="https://2010.igem.org/Team:Harvard/allergy">allergy knockdown</A><A class="sidebarlinks" href="https://2010.igem.org/Team:Harvard/allergy/allergens">allergen targets</A><A class="sidebarlinks" href="https://2010.igem.org/Team:Harvard/allergy/notebook">lab notebook</A><A class="sidebarlinks" href="https://2010.igem.org/Team:Harvard/allergy/methods">methods</A><A class="sidebarlinks" href="https://2010.igem.org/Team:Harvard/allergy/parts">parts and primers</A><A class="sidebarlinks" href="https://2010.igem.org/Team:Harvard/allergy/references">references</A></DIV><H1><A name="notebook">notebook</A></H1><TABLE style="font-size:14px;text-align:center;width:550px;" border="0" cellspacing="6"><TBODY><TR><TD> Week 1  </TD><TD> -</TD><TD> - </TD><TD> - </TD><TD> - </TD><TD><A href="#06-18-2010">06-18-2010</A></TD></TR><TR><TD> Week 2  </TD><TD><A href="#06-21-2010">06-21-2010</A></TD><TD><A href="#06-22-2010">06-22-2010</A></TD><TD><A href="#06-23-2010">06-23-2010</A></TD><TD><A href="#06-24-2010">06-24-2010</A></TD><TD><A href="#06-25-2010">06-25-2010</A></TD></TR><TR><TD> Week 3  </TD><TD><A href="#06-28-2010">06-28-2010</A></TD><TD><A href="#06-29-2010">06-29-2010</A></TD><TD><A href="#06-30-2010">06-30-2010</A></TD><TD><A href="#07-01-2010">07-01-2010</A></TD><TD><A href="#07-02-2010">07-02-2010</A></TD></TR><TR><TD> Week 4  </TD><TD> - </TD><TD><A href="#07-06-2010">07-06-2010</A></TD><TD><A href="#07-07-2010">07-07-2010</A></TD><TD><A href="#07-08-2010">07-08-2010</A></TD><TD><A href="#07-09-2010">07-09-2010</A></TD></TR><TR><TD> Week 5  </TD><TD><A href="#07-12-2010">07-12-2010</A></TD><TD><A href="#07-13-2010">07-13-2010</A></TD><TD><A href="#07-14-2010">07-14-2010</A></TD><TD><A href="#07-15-2010">07-15-2010</A></TD><TD><A href="#07-16-2010">07-16-2010</A></TD></TR><TR><TD> Week 6  </TD><TD><A href="#07-19-2010">07-19-2010</A></TD><TD><A href="#07-20-2010">07-20-2010</A></TD><TD><A href="#07-21-2010">07-21-2010</A></TD><TD><A href="#07-22-2010">07-22-2010</A></TD><TD><A href="#07-23-2010">07-23-2010</A></TD></TR><TR><TD> Week 7  </TD><TD><A href="#07-26-2010">07-26-2010</A></TD><TD><A href="#07-27-2010">07-27-2010</A></TD><TD><A href="#07-28-2010">07-28-2010</A></TD><TD><A href="#07-29-2010">07-29-2010</A></TD><TD><A href="#07-30-2010">07-30-2010</A></TD></TR><TR><TD> Week 8  </TD><TD> - </TD><TD><A href="#08-03-2010">08-03-2010</A></TD><TD><A href="#08-04-2010">08-04-2010</A></TD><TD><A href="#08-05-2010">08-05-2010</A></TD><TD><A href="#08-06-2010">08-06-2010</A></TD></TR><TR><TD> Week 9  </TD><TD><A href="#08-09-2010">08-09-2010</A></TD><TD><A href="#08-10-2010">08-10-2010</A></TD><TD><A href="#08-11-2010">08-11-2010</A></TD><TD><A href="#08-12-2010">08-12-2010</A></TD><TD> -
</TD></TR></TBODY></TABLE><DIV id="entry"><SPAN id="date"><A name="06-18-2010" class="date">06-18-2010</A></SPAN> [<A class="top" href="#notebook">top</A>]</DIV><P><B>Progress</B></P><OL><LI> Identified allergens in arabidopsis, strawberries,tomatoes, and carrots and created primers for amplifying out these allergens (see primers and sequences)] 
</LI><LI> Extracted cDNA (rna extraction followed by rt-pcr) from strawberries and tried to PCR out our strawberry allergen (Fra a 1), but were unble ato do so
</LI><LI> Looked to find introns ~ 800 bp to use to create our hairpins since it was difficult to acquire pHannibal vector w/ the pdk intron
</LI></OL><P><B>Future Directions</B></P><OL><LI> Look towards ordering amiRNA for silencing
</LI><LI> Waiting on arabidopsis plants to grow to extract genomic DNA (mon)
</LI><LI> Look to find introns to use in creation of hairpin DNA 
</LI><LI> Cloning (digesting/ligating our sense/antisense/intron sequences necessary for hairpin creation into one vector to create biobricks)
</LI></OL><DIV id="entry"><SPAN id="date"><A name="06-21-2010" class="date">06-21-2010</A></SPAN> [<A class="top" href="#notebook">top</A>]</DIV><UL><LI> PCR amplification of gDNA from Arabadopsis thaliana for sense and antisense parts of LTP, Bet v 1, and Ger3. 
</LI><LI> Diagnostic Digest of PCR products
</LI></UL><P><B>Results</B></P><DIV><A href="https://static.igem.org/mediawiki/2010/1/1d/6.17.jpg" id="single_image"> [click to enlarge]</A></DIV><P>Lanes:
2. LTP sense
11. Ger 3 sense
12. Ger 3 antisense</P><DIV id="entry"><SPAN id="date"><A name="06-22-2010" class="date">06-22-2010</A></SPAN> [<A class="top" href="#notebook">top</A>]</DIV><UL><LI> Gel electrophoresis of PCR products
</LI><LI> Gel digest and purification of LTP sense, Ger sense and Ger antisense
</LI><LI> Digest with Xba and Pst
</LI><LI> Purification of parts
</LI></UL><P><B>Results</B>
Gel purification yielded 2.6, 7.8, and 7.9 ng/μL of LTP sense, Ger sense, and Ger antisense parts. We were able to obtain 1.5, 4.3 and 0.5 ng/μL of digested LTP sense, Ger 3 sense, and Ger 3 antisense, respectively, from the digest purification. 
</P><DIV id="entry"><SPAN id="date"><A name="06-23-2010" class="date">06-23-2010</A></SPAN> [<A class="top" href="#notebook">top</A>]</DIV><UL><LI> Digest and purification of vector V0120
</LI><LI> Ligation of LTP sense, Ger sense, and Ger antisense parts into V0120
</LI></UL><P><B>Results</B>
Ligations were plated, but no viable colonies grew.
</P><DIV id="entry"><SPAN id="date"><A name="06-24-2010" class="date">06-24-2010</A></SPAN> [<A class="top" href="#notebook">top</A>]</DIV><UL><LI> PCR of arabadopsis allergens for remaining parts
</LI><LI> PCR purification
</LI><LI> Diagnostic gel electrophoresis of PCR products
</LI></UL><P><B>Results</B></P><P>We successfully obtained the rest of the allergen sense and antisense parts. Nanodrop of the PCR products showed the following concentrations in ng/μL
</P><TABLE class="wikitable" border="1px solid black"><TBODY><TR><TH>Gene
</TH><TH>PCR Products
</TH><TH>Purified DNA
</TH></TR><TR><TD>LTP 1 sense
</TD><TD>220.9
</TD><TD>20.4
</TD></TR><TR><TD>LTP 1 antisense
</TD><TD>62.6
</TD><TD>12.2
</TD></TR><TR><TD>Bet v1.1 sense
</TD><TD>216.2
</TD><TD>15.4
</TD></TR><TR><TD>Bet v1.1 antisense
</TD><TD>228.1
</TD><TD>11.5
</TD></TR><TR><TD>Bet v1.2 sense
</TD><TD>205.5
</TD><TD>9.2
</TD></TR><TR><TD>Bet v1.2 antisense
</TD><TD>68.5
</TD><TD>7.7
</TD></TR><TR><TD>Ger 3 sense
</TD><TD>73.3
</TD><TD>11.2
</TD></TR><TR><TD>Ger 3 antisense
</TD><TD>21.0
</TD><TD>8.5
</TD></TR></TBODY></TABLE><P>Diagnostic gel electrophoresis of a small sample of the PCR products showed proper length bands (300 bps) for all the parts.
</P><DIV id="entry"><SPAN id="date"><A name="06-25-2010" class="date">06-25-2010</A></SPAN> [<A class="top" href="#notebook">top</A>]</DIV><UL><LI> Digest Arabadopsis allergen panel with Xba1 and Pst1
</LI><LI> Purify digested DNA
</LI><LI> Digest vector V0120 (from Team Vector)
</LI><LI> Purify digested V0120
</LI></UL><P><B>Results</B>
Stored digests at -20 degrees Celsius for ligations tomorrow.
</P><DIV id="entry"><SPAN id="date"><A name="06-26-2010" class="date">06-26-2010</A></SPAN> [<A class="top" href="#notebook">top</A>]</DIV><UL><LI> Create amiRNA
</LI></UL><OL><LI> PCR reaction of RS300 plasmid with primers containing RNAi sequence for Arabadopsis allergens (see methods for detailed protocol for working with RS300)
</LI><LI> Diagnostic gel electrophoresis 
</LI><LI> PCR purification to obtain parts
</LI></OL><UL><LI> Ligate parts together 
</LI></UL><OL><LI> Three in one sewing PCR primed with A and B
</LI><LI> Two in one sewing PCR primed with A and II, followed by another two in one PCR reaction primed with A and B.
</LI></OL><P><B>Results</B></P><P>9 reactions total for creating amiRNA parts. We used the following temperatures for the different primers and a 15 seconds extension time. 
</P><P>Annealing Temp: (LTP &amp; Bet)60C (GFP)69C
</P><P>The concentrations of parts 1, 2, and 3 of each allergen are:
</P><P>LTP 1: 5.4 ng/uL  2: 5.5 ng/uL  3: 1.2 ng/uL
GFP 1: 17 ng/uL  2: 6.6 ng/uL  3: 15.1 ng/uL
Bet 1:7.4 ng/uL  2: 29.6 ng/uL 3: 34.3 ng/uL
</P><P>Gel electrophoresis of the parts showed strong bands for each individual part, and correct bands for sewing ligations of all three allergens. 
</P><P>The image below show results sewing PCR reactions. Lanes are:
</P><P>1 (ladder); 2(LTP 1+2); 3 (Bet 1+2); 4(GFP 1+2); 5(Ladder); 6(LTP 1,2,3); 7(Be 1,2,3); 8(GFP 1,2,3); 9(LTP 1,2,3(using .5ul)); 10(GFP 1,2,3 (using .5 uL)). Lanes 2-4 should be ~a little less than 450 bp and 6-10 should be ~450bp 
</P><DIV><A href="https://static.igem.org/mediawiki/2010/f/ff/6.28.jpg" id="single_image"> [click to enlarge]</A></DIV><DIV id="entry"><SPAN id="date"><A name="06-29-2010" class="date">06-29-2010</A></SPAN> [<A class="top" href="#notebook">top</A>]</DIV><P>Today, we did not do work with hpRNA and focused our efforts on amiRNA. We obtained undigested V0120 with a death gene insert from Team Flavor, and we have inserts from previous PCR reactions that all need to be digested and ligated into V0120.
</P><P>Procedures:
</P><OL><LI>Digestion of Allergen Panel and V0120 with Xba and Pst
</LI><LI>Ligation of Allergen inserts into Vector Backbone
</LI><LI>Transformation of E. Coli 
</LI></OL><P><B>Results</B></P><P>Digestion of allergens yielded the following concentrations:
</P><P>Bet: 58.2 ng/uL; LTP (.5): 38.6 ng/uL; GFP (.5): 43.9 ng/uL; GFP: 29.1 ng/uL; LTP: 60.4 ng/uL; Backbone: 20.2 ng/uL
</P><DIV id="entry"><SPAN id="date"><A name="06-30-2010" class="date">06-30-2010</A></SPAN> [<A class="top" href="#notebook">top</A>]<UL><LI>Checked for transformants. No viable colonies except for Bet, but those two colonies could be just background.
</LI><LI>Grew up 2 tubes for minipreps at ~10am  (Should be ready ~ 4 pm)
</LI></UL><LI>Miniprepped w/ concetnrations of ~ 80 ng/uL and 108 ng/uL
</LI><UL><LI>Redid ligations and transformations (plated ~ 12 pm)
</LI><LI>Bet, LTP, GFP, LTP .5, GFP .5, Negative Control (just V0120 backbone)
</LI></UL><UL><LI>Transformed ligations into turbo cells and let it sit outside rather than in 37
</LI></UL><UL><LI>PCR of PAL3 and PME2 introns from arabidopsis genomic DNA
</LI></UL><DIV id="entry"><SPAN id="date"><A name="07-01-2010" class="date">07-01-10</A></SPAN> [<A class="top" href="#notebook">top</A>]<UL><LI><B>Ligations for full amiRNA and V0120 (w/o death gene)</B></LI></UL><P>note: .5 denotes multistep pcr reaction from yesterday in which .5 uL of part 3 of gfp/ltp were use in one of the three in one assemblies of parts 1,2,3 w/ primers A&amp;B
</P><P><I>Ligation Reactions</I></P><UL><LI> Used 50 ng of backbone and 150 ng of insert (ratio of insert to vector ~.14
</LI><LI> Did 1:10 dilutions for the digested inserts
</LI></UL><UL><LI><B>PCR of PME3 and PAL2</B></LI><LI> saw no bands from last night's pcr so we re ran it again.
</LI></UL><P>Annealing Temp: 69C
Extension Time: 30 sec
</P><P><I>2 PCR reactions (PME &amp; PAL)</I></P><UL><LI>Concentrations: PME: 250 ng/uL  PAL: 134 ng/uL
</LI></UL><LI>gel: lane 1:ladder; lane 2: Pal3; lane 3: Pme 2; lane 4: ladder
</LI><DIV><A href="https://static.igem.org/mediawiki/2010/0/07/PAL2_PME3.jpg" id="single_image"> [click to enlarge]</A></DIV><UL><LI><B>Sent amiRNA Bet for sequencing</B></LI></UL><UL><LI><B>Ligation of Antisense/sense allergens into V0120 (lacking death gene)</B></LI></UL><UL><LI> Used 34 ng of backbone and 10 ng of insert (ratio of insert to vector ~.09375)
</LI></UL><DIV id="entry"><SPAN id="date"><A name="07-02-2010" class="date">07-02-2010</A></SPAN> [<A class="top" href="#notebook">top</A>]<UL><LI> Check on sequences for Bet
</LI><LI> Digest Pme3 and Pal 2 introns/Ligate w/ V0120/Transform
</LI><LI>PCR purified digests
</LI><LI>Concentrations: Pme (ranges from 20-36 ng/uL); Pal (ranges from 20 to 28 ng/uL)
</LI></UL><LI>Ran backbone on gel and gel purified (Concentration: 3.1 ng/uL)
</LI><UL><LI>Ligations of Pme/Pal w/ V0120
</LI><LI>bbone 2 was potentially contaminated at a higher concentration 96.1 ng/uL
</LI></UL><UL><LI> Check on transformations of amiRNA (5 transformations) and antisense/sense allergens (8 transformations)
</LI><LI>There were colonies from the amiRNA transformations
</LI><LI>We grew up 24 colonies (3 for each GFP, GFP .5, LTP, LTP .5, BET) ~ 10 am to be ready ~3 pm for minipreps)
</LI><LI>The other 8 transformations did not show any colonies--we put it back in to see if we would get colonies (only 100 uL instead of 200 uL was plated)
</LI><LI>To do: Grow up colonies to miniprep, run on gel, sequence
</LI></UL><UL><LI>pHannibal and pKannibal arrived today (hannibal is amp resistant and kannibal is kan resistant)
</LI><LI>We transformed these plasmids
</LI></UL><DIV id="entry"><SPAN id="date"><A name="07-03-2010" class="date">07-03-2010</A></SPAN> [<A class="top" href="#notebook">top</A>]<P><B>Team Allergy</B>
Today, we isolated plasmid that contained antisense and sense sequences of the allergen panel and introns PME and PAL that we grew last week. 
</P><P><B>Procedures</B>
For amiRNA
</P><OL><LI>Nanodrop of miniprepped amiRNA plasmids
</LI><LI>Diagnostic digest of amiRNA plasmids
</LI><LI>E Gel of digested amiRNA
</LI><LI>Send plasmids to Genewiz for sequencing
</LI></OL><P>For ihpRNA
</P><OL><LI>Miniprep of plasmids containing introns for hpRNA
</LI><LI>Nanodrop of miniprepped intron plasmids
</LI><LI>Miniprep of plasmids containing Allergen Panel sense and antisense
</LI><LI>Nanodrop of minipreped allergen plasmids
</LI><LI>Diagnostic Digest of introns and allergens
</LI><LI>E Gel of digested parts
</LI><LI>Extract pKannibal and pHannibal vectors and transform into E. coli
</LI><LI>PCR of the PDK intron in pKannibal and pHannibal 
</LI></OL><P><B>Miniprep of turbo bacteria for plasmids containing sequences for amiRNA</B></P><P>Should obtain plasmids containing the complete sequence for amiRNA. The resulting concentration should be relatively high (&lt;100ng/μL) because all growing bacteria would have the plasmids inside. 
</P><P><B>Nanodrop of miniprepped amiRNA plasmids</B></P><P>We used 2μL of DNA per nanodrop. 
</P><TABLE border="1"><CAPTION><I>Concentrations in ng/μL</I></CAPTION><TBODY><TR><TH> Sample # --&gt;</TH><TH> One</TH><TH> Two</TH><TH> Three
</TH></TR><TR><TH> GFP
</TH><TD align="center"> 134.4</TD><TD align="center">167.6 </TD><TD align="center"> 207.2
</TD></TR><TR><TH> GFP 0.5
</TH><TD align="center">163.3 </TD><TD align="center">  106.9 </TD><TD align="center"> 230.9
</TD></TR><TR><TH> LTP
</TH><TD align="center"> 134.0</TD><TD align="center">  127.8 </TD><TD align="center">  165.5
</TD></TR><TR><TH> LTP 0.5
</TH><TD align="center"> 158.5 </TD><TD align="center">  137.2 </TD><TD align="center">  113.5
</TD></TR><TR><TH> Bet
</TH><TD align="center"> 129.4</TD><TD align="center">  78.9 </TD><TD align="center">  124.4
</TD></TR></TBODY></TABLE><P><B>Diagnostic digest of amiRNA plasmids</B>
Digested 200~500ng of plasmids with EcoR1 and Spe1. We will then run the digested vectors on an E Gel and look for DNA that is the length of the insert. If such DNA appears on the gel, then we will send it off to be sequenced.
</P><P><B>E Gel of digested amiRNA</B></P><DIV><A href="https://2010.igem.org/Image:Equencing_send_out_7-6.jpg" id="single_image"> [click to enlarge]</A></DIV><P>We had to run 21 samples, so we used two gels. The orders were:
</P><P>Gel #1
</P><OL><LI> 1kb Ladder
</LI><LI>GFP 1
</LI><LI>GFP 2
</LI><LI>GFP3
</LI><LI>GFP 0.5 1
</LI><LI>GFP 0.5 2
</LI><LI>GFP 0.5 3
</LI><LI>LTP 1
</LI><LI>LTP 2
</LI><LI>LTP 3
</LI><LI>LTP 0.5 1 <B>no visible band for neither plasmid nor insert</B></LI><LI>LTP 0.5 2
</LI></OL><P>Gel #2
</P><OL><LI> 1kb Ladder
</LI><LI>LTP 0.5 3
</LI><LI> Bet 1
</LI><LI>Bet 2
</LI><LI>Bet 3
</LI><LI>Neg Control for hpRNA (digested with Xba and Pst, and ligated without new inserts)  <B>no visible insert band</B></LI><LI>Neg Control for intron (digested with Eco and Spe, and ligated without new inserts)  
</LI><LI>PME Good <B>no visible insert band</B></LI><LI>PME Dubious <B>no visible insert band</B></LI><LI>PAL Good colony 1 <B>no visible insert band</B></LI><LI>PAL Good colony 2
</LI><LI> 1kb Ladder
</LI></OL><P><B>Miniprep of plasmids containing introns for hpRNA</B><B>Nanodrop of miniprepped intron plasmids</B></P><P>We used 2μL of DNA per nanodrop. 
</P><TABLE border="1"><CAPTION> Concentrations in ng/μL,  <I>cells are filled when applicable</I></CAPTION><TBODY><TR><TH> Insert Name </TH><TH> Good</TH><TH> Dubious</TH><TH> Colony 1</TH><TH> Colony 2</TH><TH> Colony 3
</TH></TR><TR><TH> PME
</TH><TD align="center"> 85.6</TD><TD align="center">66.8 </TD></TR><TR><TH> PAL
</TH><TD align="center"> 67.7 </TD><TD align="center">108.8 </TD></TR><TR><TH> Neg Control, Intron
</TH><TD align="center"> 74.0 </TD></TR><TR><TH> Neg Contron, hpRNA
</TH><TD align="center"> 63.5 </TD><TD align="center"> 30.1 </TD><TD align="center"> 37.3
</TD></TR></TBODY></TABLE><OL><LI>Miniprep of plasmids containing Allergen Panel sense and antisense (from overnight cultures)--still need to do preps of cultures grown today
</LI><LI>Nanodrop of minipreped allergen plasmids
</LI><LI>Diagnostic Digest of introns and allergens
</LI><LI>E Gel
</LI></OL><P><B>phannibal and pkannibal</B>
Extracted vectors from a piece of paper per instructions. Transformed vectors into E. coli. Miniprep and PCR to extract PDK intron. 
</P><UL><LI>pcr to get out pdk intron
</LI></UL><DIV id="entry"><SPAN id="date"><A name="07-07-2010" class="date">07-07-2010</A></SPAN> [<A class="top" href="#notebook">top</A>]<P>Yesterday, we ended by transforming pKannabil and pHannibal into E. coli and running a diagnostic digest of digested allergen panel. Yesterday's plates did not grow, and the diagnostic digest did not have good results. 
</P><UL><LI>Today, we will be redoing diagnostic digests of the allergen panel plasmids to see if we can see our desired bands (300 bp and 3200) and re-transforming pKannabil and pHannibal. 
</LI><LI>If the digested allergen panel runs correctly on a gel, then we will send the plasmids off for sequencing. 
</LI><LI>If nothing gel does not run correctly, then we will go back to re-digest V0120 with a death gene insert instead of YFP and ligate allergen inserts, transform, miniprep, and see if we get the correct insert. **If some run correctly and some did not, we will collect more samples from our culture plates for the allergens that did not work and redo the digests until we find a sample with the correct plasmid.
</LI></UL><LI> If the pKannabil and pHannibal plates grow, then we will miniprep to purify the plasmids containing the PDK intron, PCR the intron, and purify the intron.  
</LI><LI>Today, we are also running the PDK intron obtained yesterday to see if it is the correct length. 
</LI><P>First  thing tomorrow, we will culture E. coli containing plasmids for our allergen panel to obtain more plasmids. 
</P><P><B>Procedures</B></P><P>For pKannabil and pHannibal:
</P><OL><LI>Electronic Gel Electrophoresis of PDK introns (from PCR reaction yesterday, PCR off DNA from the eluted vectors)
</LI><LI>Re-grow E. coli containing pKannabil and pHannibal plasmids
</LI><LI>Culture colonies
</LI><LI>Miniprep colonies
</LI><LI>PCR for PDK intron
</LI><LI>Diagnostic Gel to check for correct length
</LI><LI>Repeat PCR of PDK intron using different Tm (gradient pcr)
</LI></OL><P>For Allergen panel
</P><OL><LI>Digest of Plasmids containing allergen inserts
</LI><LI>Diagnostic Gel
</LI><LI>Collect additional samples for allergens that did not work
</LI></OL><P><B>Results</B></P><P><B>Transformation of pHannibal pKannibal</B>
This transformation we performed slightly differently from the one yesterday. This protocol is nicknamed the &quot;quick and dirty transform.&quot; The transformation we started yesterday of pH and pK didn't work (no growth on either plate).
</P><P>Quick and Dirty Transform:
</P><OL><LI>Mix
</LI><LI>2ul 10x {pHannibal, pKannibal}
</LI><LI>20ul Turbo E. Coli
</LI><LI>220ul SOC Broth
</LI></OL><LI>Plate on {LB+(pHannibal-&gt;LB+Amp, pKannibal-&gt;LB+Kan), LB} Agar, grow
</LI><P><B> PCR of PDK intron</B></P><UL><LI><B>PCR of PME3 and PAL2</B></LI><LI>saw no bands from last night's pcr so we re ran it
</LI></UL><P>Annealing Temp: 69C
Extension Time: 30 sec
</P><P><I>Concentrations</I>: PDK:46.5 ng/uL &amp; 36.5 ng/uL
Didn't work. Our theory is that the Tm we used (the one IDT sent us) was too high. IDT assumes that the entire primer is going to anneal to a strand of DNA when they calculate their Tm, but since half of our primer won't anneal (the half that doesn't anneal was used to prepend/postpend the biobrick sequences) the Tm IDT calculates will be too high. Here are more correct calculations from Primer3:
</P><PRE>
OLIGO            start  len      tm     gc%   any    3' seq 

LEFT PRIMER       4229   20   51.11   30.00  8.00  3.00 CCAATTGGTAAGGAAATAAT
RIGHT PRIMER      5021   20   61.95   45.00  6.00  0.00 TTCGAACCCAATTTCCCAAC
</PRE><P>
See Gel Image Below:  ladder, lane 3 and 5 are pdk intron
</P><DIV><A href="https://static.igem.org/mediawiki/2010/c/c5/Pdkintron.jpg" id="single_image"> [click to enlarge]</A></DIV><P>Lanes:
3. pdk intron
5. pdk intron</P><P><I>Gradient PCR</I></P><P>Annealing Temps: 50,55,60,65
</P><P>Extension Time: 30 sec
</P><P>Ran 8 reactions, for samples from lane 3 and 5 for each of the four temperatures.
</P><P>Concentrations: PDK: 12.7 ng/uL
</P><P><B>Remaining steps</B></P><OL><LI>Culture colonies
</LI><LI>Miniprep colonies
</LI><LI>PCR for PDK intron
</LI><LI>Diagnostic Gel to check for correct length
</LI><LI>Repeat PCR of PDK intron using different Tm.
</LI></OL><P><B>Digest of Plasmids containing allergen inserts</B>
Digested with Xba and Pst using Fast Fermentas digest protocol
</P><P>
Out of the 24 lanes, 6 were successful. Of the 8 allergens, 3 were successful. Samples of LTP sense, Betv1 sense, and Ger sense had one band at around 3kbps and one band at 300bps. The other genes had one band at 3kbps and additional longer bands on the scale of twice or three times the length of the plasmid. 
</P><P>For the allergens that did not work, we will go back to our bacteria plates and sample 5 additional colonies for each allergen, grow up 10 mL overnight cultures, and miniprep so that we can redo the diagnostic digest. We are hoping that a larger sample size would allow us to find colonies with the correct plasmid. 
</P><P><B>Remaining Steps</B></P><OL><LI>Collect additional samples for allergens that did not work
</LI><LI>Culture them
</LI><LI>Miniprep
</LI><LI>Diagnostic Digest and Gel
</LI></OL><DIV id="entry"><SPAN id="date"><A name="07-08-2010" class="date">07-08-2010</A></SPAN> [<A class="top" href="#notebook">top</A>]<B>amiRNA</B><UL><LI>Sent out plasmids for sequencing w/ new primers 
</LI></UL><P><B>Allergen Panel</B></P><UL><LI>Sent out for sequencing this morning using new primers
</LI><LI>Minipreps of Allergen Panel
</LI><LI>Concentrations: 
</LI></UL><P><I>LTP 1,2,3</I>: 265.5 ng/uL, 507.1 ng/uL, 275.5 ng/uL; 
</P><P><I>GerA 1,2,3,4</I>: 283.1 ng/uL, 185.9 ng/uL, 182.4 ng/uL, 314.7 ng/uL; 
</P><P><I>Bet 2A 1,2,3,4</I>: 246.6 ng/uL, 251.4ng/uL, 550.3 ng/uL, 235.8 ng/uL; 
</P><P><I>Bet 2S 1,2,3,4,5,6</I>: 613 ng/uL, 296.8 ng/uL, 333.1 ng/uL, 467.3 ng/uL, 230.3 ng/uL, 129.4 ng/uL; 
</P><P><I>Bet 1A 1,2,3,4,5</I>: 505.8 ng/uL, 632.1 ng/uL, 561.7 ng/uL, 594.4 ng/uL, 609.2 ng/uL
</P><UL><LI>Diagnostic Digest 
</LI></UL><DIV><A href="https://static.igem.org/mediawiki/2010/b/b9/Allergen_Panel_after_40_min.jpg" id="single_image"> [click to enlarge]</A></DIV><P><B>Gradient PCR of PDK intron</B></P><DIV><A href="https://static.igem.org/mediawiki/2010/4/45/PHAN%2BpKAN_PDK_PCR.jpg" id="single_image"> [click to enlarge]</A></DIV><DIV><A href="https://static.igem.org/mediawiki/2010/5/5c/PHAN%2BpKAN_PDK_PCR2.jpg" id="single_image"> [click to enlarge]</A></DIV><P>Gel of gradient PCR of the 741bp PDK intron out of pHANNIBAL and pKANNIBAL. Lanes are (left to right, temperatures are annealing temperatures used in the gradient PCR)
1kb+ ladder, pHAN 50ºC, pKAN 50ºC, pHAN 55ºC, pKAN 55ºC, pHAN 60ºC, pKAN 60ºC, pHAN 65ºC, pKAN 65ºC.
</P><P>It appears that pHANNIBAL worked at 50, 55ºC but pKANNIBAL didn't. 
</P><P>Concentration: hannibal pdk 12.7 ng/uL kannibal pdk: 15.6 ng/uL
</P><P>Did a PCR of the PCR product 
</P><P>Annealing Temps: hannibal pdk (50 C); kannibal pdk (55 C)
</P><P>Extension Time: 30 sec
</P><P><I>12 reactions (6 hannibal pdk; 6 kannibal pdk)</I></P><UL><LI>Growing up 5 mL cultures of pHannibal and pKannibal for getting more DNA for pdk pcr
</LI><LI>Gel purifying/digesting/ligating/transforming pdk intron into e.coli
</LI></UL><DIV id="entry"><SPAN id="date"><A name="07-09-2010" class="date">07-09-2010</A></SPAN> [<A class="top" href="#notebook">top</A>]<UL><LI>PCRd pHAN.pdk, pKAN.pdk (they're the same 741bp pdk) using the values that worked in the gradient, got them in high ODs. Put them in box a2 rows 3-4.
</LI></UL><TABLE border="1"><CAPTION> Concentration of PCR products in ng/ul
</CAPTION><TBODY><TR><TH>Temp(ºC)
</TH><TH>Source
</TH><TH>Label
</TH><TH>Concentrations
</TH></TR><TR><TD>55 </TD><TD> H </TD><TD> HPDK55 </TD><TD> 64.0,190.9,66.4
</TD></TR><TR><TD>50 </TD><TD> H </TD><TD> HPDK50 </TD><TD> 119.3,125.5,92.4
</TD></TR><TR><TD>55 </TD><TD> K </TD><TD> KPDK55 </TD><TD> 84.3,97.3,106.5
</TD></TR><TR><TD>50 </TD><TD> K </TD><TD> KPDK50 </TD><TD> 52.5,93.5,71.0
</TD></TR></TBODY></TABLE><DIV id="entry"><SPAN id="date"><A name="07-12-2010" class="date">07-12-2010</A></SPAN> [<A class="top" href="#notebook">top</A>]<P>Today, we received sequences for the allergen , intron, and amiRNA inserts. Since 4 of the 8 allergen inserts were not correctly done (and were not set for sequencing) and we want to repeat the inserts that DID work, we will redo ligations for the entire panel.
</P><P>Procedures
</P><OL><LI>Check Sequencing Results 
</LI><LI>Redo Ligations for allergen Panel 
</LI></OL><P><B>Sequencing Results</B></P><UL><LI>amiRNA
</LI><LI>GFP #1 &amp; GFP.5 #3 worked
</LI><LI>Bet,LTP did not work
</LI></UL><LI>intron
</LI><LI>Pal(#2) matched
</LI><LI>ihpRNA
</LI><LI>LTP Sense 1,2,3 and Ger Sense 1 matched
</LI><LI>Bet sense did not match 
</LI><P>Full report (annotated sequences): <A href="/File:Report.zip" title="File:Report.zip">File:Report.zip</A></P><P>Redo Ligations w/ phosphatized backbone for entire allergen panel, 
</P><P>PME did not grow, we will not use it since we have good PDK and PAL introns.
</P><P>PAL and PDK will be digested and inserted into a plasmid containing a sense strand of an allergen. 
</P><P>Add PDK to amiRNA vectors if sequences match up
</P><DIV id="entry"><SPAN id="date"><A name="07-13-2010" class="date">07-13-2010</A></SPAN> [<A class="top" href="#notebook">top</A>]<P><B>Digestion</B></P><TABLE border="1"><CAPTION><I>Digestion Reactions for Backbones (plasmids containing sense)</I></CAPTION><TBODY><TR><TH> pLTPS1</TH><TH> pLTPS2</TH><TH> pLTPS3</TH><TH> GerS1
</TH></TR><TR><TH> DNA
</TH><TD align="center">10 </TD><TD align="center">  8</TD><TD align="center"> 7</TD><TD align="center"> 10</TD></TR><TR><TH> FD Buffer (10x)
</TH><TD align="center"> 2 </TD><TD align="center">  2 </TD><TD align="center">  2</TD><TD align="center">  2</TD></TR><TR><TH> diH<SUB>2</SUB>O
</TH><TD align="center"> 6 </TD><TD align="center">  8</TD><TD align="center">  9</TD><TD align="center">6</TD></TR><TR><TH> Spe
</TH><TD align="center"> 1 </TD><TD align="center">  1 </TD><TD align="center">  1</TD><TD align="center">  1</TD></TR><TR><TH> Pst
</TH><TD align="center"> 1 </TD><TD align="center">  1</TD><TD align="center"> 1</TD><TD align="center"> 1</TD></TR></TBODY></TABLE><UL><LI>add phosphatase to backbone (15 min)
</LI></UL><TABLE border="1"><CAPTION><I>Digestion Reactions for Inserts</I></CAPTION><TBODY><TR><TH> Pal2</TH><TH> PDK
</TH></TR><TR><TH> DNA (~ 1 ug)
</TH><TD align="center">9</TD></TR><TR><TH> FD Buffer (10x)
</TH><TD align="center"> 2 </TD><TD align="center">2
</TD></TR><TR><TH> diH<SUB>2</SUB>O
</TH><TD align="center"> 7 </TD></TR><TR><TH> Xba1
</TH><TD align="center"> 1</TD><TD align="center">1
</TD></TR><TR><TH> Pst1
</TH><TD align="center"> 1</TD><TD align="center">1
</TD></TR></TBODY></TABLE><UL><LI>gel purify backbones  LTPS: 8.7 ng/uL  Ger: 9.4 ng/uL
</LI><LI>pcr purify digested pdk: PDK1 14.4 ng/uL; PDK2 22.4 ng/uL; PDK3 11.4 ng/uL; PDK4 15.3 ng/uL
</LI></UL><P><B>Ligations</B></P><P><I>Ligations of PDK w/ V0120 (lacking YFP)</I></P><TABLE border="1"><CAPTION><I>Ligation Reactions</I></CAPTION><TBODY><TR><TH> Pdk1 </TH><TH> Pdk2</TH><TH> Pdk3</TH><TH> Pdk4
</TH></TR><TR><TH> DNA Insert
</TH><TD align="center"> 2.36 </TD><TD align="center"> 2.98</TD><TD align="center">1.52 </TD><TD align="center"> 2.22
</TD></TR><TR><TH> T4 DNA ligase buffer (10x)
</TH><TD align="center"> 2 </TD><TD align="center"> 2 </TD><TD align="center"> 2</TD><TD align="center"> 2
</TD></TR><TR><TH> diH<SUB>2</SUB>O
</TH><TD align="center">  12.64</TD><TD align="center">12.02  </TD><TD align="center">13.48 </TD><TD align="center"> 12.78
</TD></TR><TR><TH> T4 DNA ligase
</TH><TD align="center"> 1 </TD><TD align="center">  1 </TD><TD align="center">  1</TD><TD align="center">  1
</TD></TR><TR><TH> DNA Backbone (~50ng)
</TH><TD align="center"> 2uL </TD><TD align="center">2uL</TD><TD align="center">2uL</TD><TD align="center"> 2uL
</TD></TR></TBODY></TABLE><P><I>Ligations of PDK/Pal into V0120 + Sense Allergens (LTP &amp; GER)</I></P><TABLE border="1"><CAPTION><I>Ligation Reactions 3:1 excess</I></CAPTION><TBODY><TR><TH> PDK1 </TH><TH> PDK2 </TH><TH> PDK 3 </TH><TH> PDK 4
</TH></TR><TR><TH> DNA Insert (~36ng/ ~32 ng))
</TH><TD align="center"> 2.51; 2.27</TD><TD align="center">1.62;1.46 </TD><TD align="center"> 3.18; 2.86 </TD><TD align="center"> 2.37;2.13
</TD></TR><TR><TH> T4 DNA ligase buffer (10x)
</TH><TD align="center"> 2 </TD><TD align="center"> 2 </TD><TD align="center"> 2</TD><TD align="center"> 2
</TD></TR><TR><TH> diH<SUB>2</SUB>O
</TH><TD align="center">  8.49; 9.73</TD><TD align="center"> 9.38; 10.54 </TD><TD align="center"> 7.82;9.14 </TD><TD align="center"> 8.63;9.87
</TD></TR><TR><TH> T4 DNA ligase
</TH><TD align="center"> 1 </TD><TD align="center">  1 </TD><TD align="center">  1</TD><TD align="center">  1
</TD></TR><TR><TH> DNA Backbone (~50ng/(~47ng)
</TH><TD align="center"> LTP1S 6; GerS 5 </TD><TD align="center">LTP1S 6; GerS 5 </TD><TD align="center">LTP1S 6; GerS  5</TD><TD align="center">LTP1S 6; GerS 5 </TD></TR></TBODY></TABLE><DIV id="entry"><SPAN id="date"><A name="07-14-2010" class="date">07-14-2010</A></SPAN> [<A class="top" href="#notebook">top</A>]<UL><LI>Genomic DNA extraction
</LI><LI>PCR for allergen panel from genomic DNA 
</LI></UL><LI>Grew up cultures of transformations from yesterday (LTPS+intron; GerS+intron; PDK+V0120)--tubes weren't cloudy so we grew overnight cultures again
</LI><P><I>8 reactions (Ger S/A; BetS/A; Bet1.2 S/A; LTP S/A)</I></P><P>Annealing Temp: 65 C  Extension Temp: 30 sec
</P><TABLE class="wikitable" border="1px solid black"><TBODY><TR><TH>Name
</TH><TH>Amount (uL)
</TH></TR><TR><TD>10mM dNTPs
</TD><TD>1
</TD></TR><TR><TD>5x Phusion Buffer
</TD><TD>10
</TD></TR><TR><TD>Phusion Polymerase
</TD><TD>.5
</TD></TR><TR><TD>DNA
</TD><TD>1 uL
</TD></TR><TR><TD>AB primer mix (1x)
</TD><TD>2.5
</TD></TR><TR><TD>Water
</TD><TD>35
</TD></TR></TBODY></TABLE><P><I>Concentrations</I>: BetS: 109.1 ng/uL; BetA: 90.5 ng/uL; GerS: 69 ng/uL; GerA: 46 ng/uL; LTPS:36.7 ng/uL; LTPA:5.2 ng/uL; Bet1.2S: 77.4 ng/uL; Bet1.2A:66.2 ng/uL; PME: 64.2 ng/uL; PAL: 62.9 ng/uL; PDK: 35.7 ng/uL; PDK(2): 81.3 ng/uL; PDK(3): 21.5 ng/uL 
</P><DIV id="entry"><SPAN id="date"><A name="07-15-2010" class="date">07-15-2010</A></SPAN> [<A class="top" href="#notebook">top</A>]<P>Today, we did minipreps of intron and backbone. Would have also done minipreps of intron plus allergen part but none of those colonies grew.
</P><P>We also miniprepped Betv1.2 Sense, Betv1.1 Sense, Bet 1.2 Antisense, and Ger Sense of the plates that we grew on Monday. The rest of the allergen panel did not grow properly. What colonies that WERE on the plate grew probably because AMP was not spread properly on the plate. We deduced this because those colonies could not grow in liquid media. We are waiting on more LB amp plates so that we can redo ligations, which is probably the reason for our lack of colonies. 
</P><P>We will do diagnostic digests of all miniprepped DNA. 
</P><P><B>Results</B></P><UL><LI>Minipreps of PDK+V0120  (LTPS+ PDK and Ger_S + PDK did not grow)
</LI><LI>Made glycerol stocks  500uL of 80% glycerol + 500 uL cells 
</LI></UL><UL><LI>Concentrations of pPDK: 414.2 ng/uL; 478.8 ng/uL; 510.2 ng/uL; 119.2 ng/uL
</LI><LI>Tomorrow, sending PDK+V0120 for sequencing
</LI></UL><UL><LI>Looking at sequencing results for our other amiRNA parts
</LI><LI>Only GFP worked from our first sequencing order. This time, none of the artificial parts had the right sequence
</LI><LI>Repicking amiRNA colonies to miniprep (6 of each LTP, LTP 0.5, and Bet. We have correct GFP from our first sequencing order)
</LI></UL><UL><LI>Minipreps of Bet 1S (2 colonies); Bet 1.2 S (1 colony); Bet 1.2 A (1 colony); Ger S (3 colonies)
</LI><LI>Redoing Ligations &amp; Transformations of Sense/Antisense Parts once we have LB + amp plates
</LI><LI>Bet 1.1 S: 161 ng/uL &amp; 199.2 ng/uL; Bet 1.2 A: 199.4 ng/uL; Bet 1.2S: 162.5 ng/uL
</LI><LI>Concentrations of GerS: 347.7 ng/uL; 428.6 ng/uL 541.6 ng/uL
</LI></UL><UL><LI>Diagnostic Digest of (PDK+V0120; GERS; BetS; Bet 1.2S/A) 
</LI></UL><TABLE border="1"><CAPTION><I>Digestions</I></CAPTION><TBODY><TR><TH> pPDK (1-4)</TH><TH> pGERSPDK (1-3)</TH><TH> Bet1S (2); Bet2S/A</TH></TR><TR><TH> DNA
</TH><TD align="center">1</TD><TD align="center">  1</TD><TD align="center"> 1
</TD></TR><TR><TH> FD Buffer (10x)
</TH><TD align="center"> 2 </TD><TD align="center">  2 </TD><TD align="center">  2
</TD></TR><TR><TH> diH<SUB>2</SUB>O
</TH><TD align="center"> 15 </TD><TD align="center">  15</TD><TD align="center">  15
</TD></TR><TR><TH> Xba1
</TH><TD align="center"> 1 </TD><TD align="center">  1 </TD><TD align="center">  1
</TD></TR><TR><TH> Pst
</TH><TD align="center"> 1 </TD><TD align="center">  1</TD><TD align="center"> 1
</TD></TR></TBODY></TABLE><UL><LI>Ran on gel (Ladder, pdk, V0120 Backbone, Gers (1-3), Bet 1S (1,2), PDK (1-4), Bet 1.2 S, Bet 1.2 A
</LI></UL><DIV><A href="https://2010.igem.org/Image:2010-07-15_16hr_33min.jpg" id="single_image"> [click to enlarge]</A></DIV><P>To send for sequencing: ger s, Bet 1S, PDK 2
</P><UL><LI>Looking at PCR results for Allergen Panel from Genomic DNA that we extracted (did not work)
</LI></UL><P>Ladder, GFPS, GFPA (These first two should not work), LTPS, LTPA, BetS, BetA, Bet2S, Bet2A, GerS, GerA, PME, PAL, PDK
</P><DIV><A href="https://static.igem.org/mediawiki/2010/2/26/7.15.jpg" id="single_image"> [click to enlarge]</A></DIV><DIV id="entry"><SPAN id="date"><A name="07-16-2010" class="date">07-16-2010</A></SPAN> [<A class="top" href="#notebook">top</A>]<P>More arabadopsis DNA arrived today. We are going to obtain more hpRNA parts and 
</P><UL><LI> Did a second PCR/Purification of our allergen panel from Julia's arabidopsis genomic DNA. It worked! 
</LI></UL><P>GFP didn't show up (first 2 lanes). The plants for the extraction weren't the GFP variety so this is expected.
</P><DIV><A href="https://static.igem.org/mediawiki/2010/0/00/Genomic_dna_pcr_2.jpg" id="single_image"> [click to enlarge]</A></DIV><UL><LI>gel purifying PCR products
</LI><LI><I>Concentrations:</I> LTPS: LTPA: GERS: GERA: BETS: BETA: BET2S: BET2A: PME: PAL: 
</LI></UL><UL><LI>Digested Xba1 Pst1
</LI><LI>Sent BetS, PDK2, and GerS (3) for sequencing
</LI><LI>Miniprep amiRNA 
</LI></UL><P><B>Nanodrop of Miniprepped amiRNA</B><I>Concentrations</I></P><TABLE border="1"><CAPTION><I>Concentrations in ng/μL of colonies 1~6</I></CAPTION><TBODY><TR><TH> 1 </TH><TH> 2</TH><TH> 3 </TH><TH> 4 </TH><TH> 5 </TH><TH> 6
</TH></TR><TR><TH> LTP
</TH><TD align="center">320.7</TD><TD align="center">  359.4</TD><TD align="center"> 290.2</TD><TD align="center"> 238.5</TD><TD align="center"> N/A </TD><TD align="center">  212.4
</TD></TR><TR><TH> Bet
</TH><TD align="center">156.8</TD><TD align="center">  267.7</TD><TD align="center"> 340.0</TD><TD align="center"> 287.4</TD><TD align="center"> 278.9 </TD><TD align="center">  273.5
</TD></TR><TR><TH> LTP 0.5
</TH><TD align="center">296.4</TD><TD align="center">  268.8</TD><TD align="center"> 244.2</TD><TD align="center"> 212.0</TD><TD align="center"> 292.6</TD><TD align="center">  208.3
</TD></TR></TBODY></TABLE><P>Diagnostic Gel: Ladder, Bet1-6, LTP1-LTP4, Ladder;  Ladder, LTP6, LTP.5 (1-6), Ladder  
</P><DIV><A href="https://static.igem.org/mediawiki/2010/0/0e/AmiRNA_d_digest.jpg" id="single_image"> [click to enlarge]</A></DIV><DIV id="entry"><SPAN id="date"><A name="07-19-2010" class="date">07-19-2010</A></SPAN> [<A class="top" href="#notebook">top</A>]<UL><LI>Got back sequencing results of GerS, BetS, PDK
</LI><LI>All three samples of GerS and PDK sent in matched their corresponding sequences
</LI></UL><UL><LI>Ran PCR of Friday's digested B21. The gel suggested a partial digest: we saw two bands of backbone where we should have seen only one: 
</LI></UL><DIV><A href="https://static.igem.org/mediawiki/2010/2/22/Incomplete_backbone_digest.jpeg" id="single_image"> [click to enlarge]</A></DIV><UL><LI>Sending 6 samples of amiRNA (LTP.5), 5 samples of amiRNA (LTP), 6 samples of amiRNA (Bet)
</LI></UL><UL><LI>Second digest of B21 appears to have been more successful:
</LI></UL><DIV><A href="https://static.igem.org/mediawiki/2010/0/08/Backbone_digest_confirmatio.jpg" id="single_image"> [click to enlarge]</A></DIV><TABLE border="1"><CAPTION><I>Digestion Reactions</I></CAPTION><TBODY><TR><TH> V0120</TH></TR><TR><TH> DNA
</TH><TD align="center">~2.1 ugrams
</TD></TR><TR><TH> FD Buffer (10x)
</TH><TD align="center"> 2
</TD></TR><TR><TH> diH<SUB>2</SUB>O
</TH><TD align="center"> 20-rest
</TD></TR><TR><TH> Xba1
</TH><TD align="center"> 1
</TD></TR><TR><TH> EcoRI
</TH><TD align="center"> 1
</TD></TR></TBODY></TABLE><UL><LI>phosphatized backbone afterwards for 20 min after letting digestion run for an hour 
</LI><LI>concentrations
</LI><LI>backbone: (5 copies each ~ 35 ng/uL)
</LI><LI>Bet S/A:  ; Bet2S/A:  ; GerS/A:  ; LTPS/A:  ; PME:  ; Pal:  
</LI></UL><TABLE border="1"><CAPTION><I>Ligation Reactions</I></CAPTION><TBODY><TR><TH> BetS/A </TH><TH> Bet2S/A </TH><TH> LTPS/A </TH><TH> GerS/GerA </TH><TH> Pme</TH><TH>Pal
</TH></TR><TR><TH> diH<SUB>2</SUB>O
</TH><TD align="center"> 10-rest </TD><TD align="center"> 10-rest </TD><TD align="center"> 10-rest </TD><TD align="center"> 10-rest </TD><TD align="center"> 10-rest </TD><TD align="center"> 10-rest </TD></TR><TR><TH> T4 DNA ligase buffer (10x)
</TH><TD align="center"> 1 </TD><TD align="center"> 1 </TD><TD align="center"> 1</TD><TD align="center"> 1 </TD><TD align="center"> 1</TD><TD align="center"> 1
</TD></TR><TR><TH> DNA Insert
</TH><TD align="center"> 3 times excess</TD><TD align="center">  3 times excess </TD><TD align="center"> 3 times excess </TD><TD align="center"> 3 times excess </TD><TD align="center"> 3 times excess</TD><TD align="center"> 3 times excess
</TD></TR><TR><TH> DNA Backbone
</TH><TD align="center"> ~50ng </TD><TD align="center">  ~50ng </TD><TD align="center">  ~50ng </TD><TD align="center">  ~50ng </TD><TD align="center"> ~50ng</TD><TD align="center"> ~50ng
</TD></TR><TR><TH> T4 DNA ligase
</TH><TD align="center"> 1 </TD><TD align="center">  1</TD><TD align="center"> 1</TD><TD align="center"> 1</TD><TD align="center"> 1</TD><TD align="center"> 1
</TD></TR></TBODY></TABLE><UL><LI>Ligated and transformed
</LI><LI>allergen parts
</LI><LI>Pme/pal
</LI></UL><DIV id="entry"><SPAN id="date"><A name="07-20-2010" class="date">07-20-2010</A></SPAN> [<A class="top" href="#notebook">top</A>]<P>Today, we plated the transformations that we did yesterday. We are also expecting sequences for amiRNA that we ordered yesterday.
</P><P><B>Procedures</B></P><OL><LI>Plating transformants 
</LI><LI>Check sequences
</LI></OL><P><B>Results</B></P><P><B>Plating Transformations</B>
One plate of each allergen S and A and the introns PAL and PME. All but BetS and PME had colonies
</P><P><B>Sequencing Results for amiRNA</B></P><P>Not one matched the sequences that we wanted. The customization sequences that we wanted (our miRNA sequence+ a little bit of RS300) were not present in all of the LTP and Bet sequences sent in. Even the Biobrick ends were not present in the sequences except for GFP, from our earlier sequencing order. 
</P><P>What could have gone wrong? We suspect that the RS300 vector that was given to use already contains an interference sequence. The sequence that we used to design our primers are based on an RS300 vector without any inserted interference sequence. Christina has already sent an email to the lab that gave us the RS300 asking if the RS300 contains any interference, possibly GFP. That would explain our results. 
</P><P>It is possible that we will have to go back and redo PCR ~ transformation. These results are all still from the first time that we tried making amiRNA parts, just different colonies. 
</P><P><PRE>
&quot;c1&quot; and &quot;c2&quot; are the customization (mutagenesis) sites corresponding to the capitalized portions of the primers suggested by the WMD3 tool. 
&quot;c1&quot; corresponds to the capitalized region of what WMD3 calls primer3 and to the reverse compliment of the capitalized region of primer4. 
Similarly, &quot;c2&quot; corresponds to the capitalized region of primer1 and the reverse complement of the capitalized region of primer2. 
For instance, WMD3 gives us the following when asked to design primers for the BET sequence:
primer1:	gaTTGGTTATATATATCGCACCTtctctcttttgtattcc
primer2:	gaAGGTGCGATATATATAACCAAtcaaagagaatcaatga
primer3:	gaAGATGCGATATATTTAACCATtcacaggtcgtgatatg
primer4:	gaATGGTTAAATATATCGCATCTtctacatatatattcct

In the sequence itself, the 

File		c1 Seen			c2 Seen
----- First Sequencing Batch ------------
BETF-Fwd.seq	TTGGACTGAAGGGAGCTCCC	GAGAGCTTCCTTGAGTCCAT
BETR-Rev.seq	TTGGACTGAAGGGAGCTCCC	GAGAGCTTCCTTGAGTCCAT
----- Second Sequencing Batch -----------
bet_2_f-forw	TTGGACTGAAGGGAGCTCCC	GAGAGCTTCCTTGAGTCCAT
bet_3_f-forw	TTGGACTGAAGGGAGCTCCC	GAGAGCTTCCTTGAGTCCAT	
----- Third Sequencing Batch ------------
BET1F-f.seq	TTGGACTGAAGGGAGCTCCC	GAGAGCTTCCTTGAGTCCAT
BET1R-rev.seq	TTGGACTGAAGGGAGCTCCC	GAGAGCTTCCTTGAGTCCAT
BET2F-f.seq	TTGGACTGAAGGGAGCTCCC	GAGAGCTTCCTTGAGTCCAT
BET2R-rev.seq	TTGGACTGAAGGGAGCTCCC	GAGAGCTTCCTTGAGTCCAT
BET3F-f.seq	TTGGACTGAAGGGAGCTCCC	GAGAGCTTCCTTGAGTCCAT
BET3R-rev.seq	TTGGACTGAAGGGAGCTCCC	GAGAGCTTCCTTGAGTCCAT
BET4F-f.seq	TTGGACTGAAGGGAGCTCCC	GAGAGCTTCCTTGAGTCCAT
BET4R-rev.seq	TTGGACTGAAGGGAGCTCCC	GAGAGCTTCCTTGAGTCCAT
BET5F-f.seq	TTGGACTGAAGGGAGCTCCC	GAGAGCTTCCTTGAGTCCAT
BET5R-rev.seq	TTGGACTGAAGGGAGCTCCC	GAGAGCTTCCTTGAGTCCAT
BET6F-f.seq	TTGGACTGAAGGGAGCTCCC	GAGAGCTTCCTTGAGTCCAT
BET6R-rev.seq	TTGGACTGAAGGGAGCTCCC	GAGAGCTTCCTTGAGTCCAT
LTP1F-fwd.seq	TTGGACTGAAGGGAGCTCCC	GAGAGCTTCCTTGAGTCCAT
...
----- Theoretical Expectations ----------
RS300 Plasmid	TTGGACTGAAGGGAGCTCCC	GAGAGCTTCCTTGAGTCCAT
Bet.ape		AGATGCGATATATTTAACCAT	TTGGTTATATATATCGCACCT
GFP.ape		GTACAGCTCGCCGTCCACTAT	TTAGTGGTCGGCGAGCTGCAC
LTP.ape		GTAGTCCTATACAAAGTTAGT	TCTAACTATGTATAGGACCAC
RS300.ape	TATGGACTGAAGGGGACCCGT	ACAGGTCCCCTTCTGTCCATT
RS300 Backbone	TATGGACTGAAGGGGACCCGT	ACAGGTCCCCTTCTGTCCATT
</PRE></P><DIV id="entry"><SPAN id="date"><A name="07-21-2010" class="date">07-21-2010</A></SPAN> [<A class="top" href="#notebook">top</A>]<P>Yesterday, we plated transformants for the allergen panel and introns. Today, we will purify the plasmids from the transformants and send any successes in to Genewiz for sequencing.
</P><P><B>Procedures</B></P><OL><LI>Culture Colonies
</LI><LI>Miniprep of colonies
</LI><LI>Diagnostic Digest to check for inserts
</LI><LI>Send working colonies for sequencing
</LI></OL><P><B>Results</B></P><P><B>Diagnostic Digest</B></P><P>lader; ladder; GerS (1-3); Ger A (1-3); Bet2S (1-3); Bet2A (1-3); BetA (!-3); LTPS (1-3); LTPA (1-3); Pal (1-3); ladder; ladder
</P><P>(More contrasted image below):
</P><DIV id="entry"><SPAN id="date"><A name="07-22-2010" class="date">07-22-2010</A></SPAN> [<A class="top" href="#notebook">top</A>]<P>Today, we are expecting our sequencing results from Genewiz. Once those arrive, we will check the sequences for both the primers and the correct allergen sequence (around 300bps). If the sequences are correct, we will go back to our plasmids and cut the intron out of the plasmid and digest the plasmids containing the sense strands with S, P. 
</P><P><B>Procedures</B></P><OL><LI>Check Sequencing Results
</LI><LI>Digest Intron with X, P
</LI><LI>Digest Plasmids with Sense with S,P
</LI><LI>Gel Electrophoresis of Digested Plasmids (both Intron and Sense)
</LI><LI>Gel Extraction and Purification of Intron and Antisense
</LI><LI>PCR purification of Sense + Bbone 
</LI></OL><P><B>Results</B><B>Sequencing Results</B></P><UL><LI>Ger S (all 3 samples had correct sequence)
</LI><LI>Ger A (all 2 samples had correct sequence)
</LI><LI>Bet 2S (#3 did not work. #2 had correct sequence) 
</LI><LI>Bet 2A (all 3 samples had correct sequence)
</LI><LI>LTP S (all 3 samples had correct sequence)
</LI><LI>LTP A (all 2 samples had correct sequence)
</LI><LI>Pal (#3 did not contain plasmids. #1 and #2 had correct sequence)
</LI></UL><UL><LI>PME and Bet v1.1 S did not grow up from the plates, so they were not included in sequencing.
</LI></UL><P><B>Digest Intron with X, P</B>
Fast Fermentas protocol was used. 
</P><P><B>Digest Plasmids with S, P</B>
Fast Fermentas protocol was used.
</P><P><B>Gel electrophoresis of Digested Plasmids</B>
We PCR purified the products of the digestion (unnecessarily), losing some DNA. Elution was done with 50μL of EB. Tomorrow, we will run the purified PCR products on a gel and Gel Extract and Purify.
</P><DIV id="entry"><SPAN id="date"><A name="07-23-2010" class="date">07-23-2010</A></SPAN> [<A class="top" href="#notebook">top</A>]<P>Today, we will create our very first biobrick vector with both the sense and intron parts. We will use parts digested yesterday, purify them, and ligate them together.
</P><P><B>Procedures</B></P><OL><LI>Gel Electrophoresis of Intron and Antisense
</LI><LI>Gel purification of Intron and Antisense inserts
</LI><LI>Ligation of Intron insert and Sense + V0120
</LI><LI>Transform and plate 
</LI></OL><P><B>Results</B><B>Gel Electrophoresis</B></P><P>All the Antisense and intron lanes showed, though the last lane (PAL) was faint. 
</P><P><B>Gel Purification</B><B>Nanodrop of gel purification had concentrations of:</B></P><UL><LI>LTP: 1.8, 2.9
</LI><LI>Bet v1.1: 6.0, 3.2
</LI><LI>Bet v1.2: 1.8, 0.5, 1.3
</LI><LI>Ger 3: 0.8, -0.5
</LI><LI>Pal: 6.7, 0.7
</LI></UL><P>There is some cause for concern for the lower concentrations, since these small numbers are outside of the range of accuracy. 
</P><P><B>Ligation of intron insert into Sense + V0120</B>
PME was ligated into all backbones containing sense parts of the allergen panel (8 rxns), followed by PAL ligation into the same backbones. We plated two of each PAL ligation in case that there might be some increased chances of finding growth on plates if more than one plate is done for each ligation. We will check for colonies Saturday.
</P><DIV id="entry"><SPAN id="date"><A name="07-26-2010" class="date">07-26-2010</A></SPAN> [<A class="top" href="#notebook">top</A>]<P><B>Procedures</B></P><P>Last week, we finished ligation together Sense and Intron parts of the hpRNA and transformed them into E. coli. This week, we will extract the plasmids, add in the antisense part, and transform. 
</P><P>Procedures today...
</P><OL><LI>Culture colonies (done over the weekend)
</LI><LI>Miniprep of V0120 sense + PDK, PAL intron parts
</LI><LI>Diagnostic Digest 
</LI></OL><P>We also need to check which parts of the sequence on the team page corresponds to the sense/antisense parts of the gene. There may have been mistakes in the past.
</P><P><B>Results</B><B>Culture</B>
We cultured 38 colonies. All grew but one LTP+PAL and Ger+Pal grew. We labeled the cultures 1- 38 for simplicity sake. 
</P><P><B>Miniprep</B></P><P>Here are the concentrations of the miniprepped plasmid. 
</P><TABLE border="1"><CAPTION><B>Sample Number</B></CAPTION><TBODY><TR><TH> 1 </TH><TH> 2 </TH><TH> 3 </TH><TH> 4 </TH><TH> 5 </TH><TH> 6 </TH><TH> 7 </TH><TH> 8 </TH><TH> 9 </TH><TH> 10
</TH></TR><TR><TH> LTP+PDK
</TH><TD align="center"> 233.9 </TD><TD align="center"> 507.6 </TD><TD align="center"> 498.7 </TD><TD align="center"> 510.0 </TD><TD align="center"> 485.1 </TD><TD align="center"> 588.3 </TD></TR><TR><TH> LTP+PAL
</TH><TD align="center"> 471.8 </TD><TD align="center"> 586.7 </TD><TD align="center">500.2 </TD><TD align="center"> 449.4 </TD><TD align="center"> 492.7 </TD><TD align="center"> 430.0 </TD><TD align="center"> 521.8 </TD><TD align="center"> 605.8 </TD><TD align="center"> 499.1 </TD><TD align="center"> 299.9
</TD></TR><TR><TH>Bet v1+PDK
</TH><TD align="center"> 430.1</TD><TD align="center"> 519.3 </TD></TR><TR><TH> Bet v1+PAL
</TH><TD align="center"> 432.3 </TD><TD align="center"> 300.6 </TD><TD align="center"> 458.7 </TD><TD align="center"> 213.8 </TD></TR><TR><TH> Bet v2+PDK
</TH><TD align="center"> 270.3 </TD><TD align="center"> 223.3 </TD></TR><TR><TH> Bet v2+PAL
</TH><TD align="center"> 270.3 </TD><TD align="center"> 223.3 </TD></TR><TR><TH> Bet v2+PDK
</TH><TD align="center"> 270.3 </TD><TD align="center"> 223.3 </TD></TR><TR><TH> Ger3+PDK
</TH><TD align="center"> 440.4 </TD><TD align="center"> 205.2 </TD><TD align="center"> 342.7 </TD><TD align="center"> 290.2 </TD></TR><TR><TH> Ger3+PAL
</TH><TD align="center"> 335.4 </TD><TD align="center"> 297.8 </TD><TD align="center"> 474.3 </TD><TD align="center"> 396.1 </TD><TD align="center"> 354.7 </TD><TD align="center"> 366.2 </TD></TR></TBODY></TABLE><P><B>Diagnostic Digest</B></P><P>We digested the vectors with Spe and Pst1 and ligate with antisense parts (already digested and gel purified). We ran them on the E Gel too long so the lanes do not show anything conclusive, other than that there is digested DNA in the bottom half. 
</P><P>Result: </P><P><B>Sequence review</B> clarified which parts of the sequences on the team page corresponded to sense/antisense.
</P><DIV id="entry"><SPAN id="date">&lt;a name=&quot;07-27-2010&quot; class=&quot;date&quot;&gt;07-27-2010&lt;/a&gt;</SPAN> [&lt;a class=&quot;top&quot; href=&quot;#notebook&quot;&gt;top&lt;/a&gt;]<P><B>Procedures</B></P><P>Yesterday, we tried to run a diagnostic digest of the Sense + Intron hpRNA. The gel did not work properly, so we are redoing the gel today. We'll also go ahead and ligate antisense parts into all 38 minipreps that we HOPE contains correct Sense + Intron. Once the gel finishes running, we can go back and dispose of the ligations today that do not contain the correct S+I parts. 
</P><P>Today, we are also going to redo the ligation of Sense + Intron for PDK colonies because the gel that we are redoing today suggests that no PDK worked from last week's ligation. 
</P><P>Procedures
</P><OL><LI>Diagnostic Digest
</LI><LI>Diagnostic Gel
</LI><LI>Diagnostic Gel of PDK + Sense
</LI></OL><LI>Ligation of Antisense Part into 38 minipreps
</LI><P>Redoing ligations for Sense + PDK
</P><OL><LI>Digest of pLTP, pBet v1, pGer with S, P
</LI><LI>PCR purification
</LI><LI>Ligation of Sense+PDK
</LI><LI>Transform and Plate
</LI></OL><P><B>Results</B></P><P><B>Diagnostic Digest</B></P><P>38 digests of the sense+intron miniprepped plasmid DNA. The digest was done with Slow Digest Eco and Pst 
</P><P><B>Diagnostic Gel</B></P><P>From the digest, we can see that two LTPS+Pal worked (9,11), 3 Ger3S+Pal worked (36-38), 4 Bet2S+Pal Worked (25-28)
</P><P>Lanes are: 
1-6: LTP + PDK  (3200 &amp; 1200);
</P><P>7-16: LTP + PAL (3200 &amp; 540);
</P><P>17,18: Bet1 + PDK (3200 &amp; 1200);
</P><P>19-22: Bet1 + PAL (3200 &amp; 540);
</P><P>23,24: Bet2+PDK (3200 &amp; 1200);
</P><P>25-28: Bet2 + PAL (3200 &amp; 540);
</P><P>29-32: Ger3 + PDK (3200 &amp; 1200);
</P><P>33-38: Ger3 + PAL (3200 &amp; 540);
</P><P>Image: Ladder (every 6)
</P><P>Rerun of the lanes that were ligations of PDK + sense because they did not show on the gel above. We feel that there might be a chance that if the Sense+PDK assembly did not work, the 300bp evidence ran off the first gel. This second gel shows that most of the PDK+sense did NOT work. Only three worked -- minipreps 16, 30, and 31.  
</P><P><B>Ligated with antisense parts and transformed</B></P><P>Digested antisense parts with X, P and pSense+Intron with S, P. Digestion ran for 2 hours, with an additional 20 minutes to de-phosphatize the backbone. Ligations were done for colonies that showed correct Sense+Intron ligations. 
</P><TABLE border="1"><TBODY><TR><TH> 1-39
</TH></TR><TR><TH> DNA
</TH><TD align="center">~1.5 ug
</TD></TR><TR><TH> 10x BSA
</TH><TD align="center"> 3
</TD></TR><TR><TH> diH<SUB>2</SUB>O
</TH><TD align="center">Fill to 30
</TD></TR><TR><TH> Spe
</TH><TD align="center"> 1.5
</TD></TR><TR><TH> Pst1
</TH><TD align="center"> 1.5
</TD></TR><TR><TH>Neb Buffer 2 (10x)
</TH><TD align="center"> 3
</TD></TR></TBODY></TABLE><P><B>Transform and Plate</B></P><P>The standard Turbo Cell transformation protocol was followed. 
</P><P><B>Redo Ligations for PDK+ Sense</B></P><P>Since the gel showed that most of them did not work.
</P><P>Digestion of pLTP, pBet v1, and pGer 3 with S, P to prepare them as a back bone. Phosphatase was used. Did not digest pBet v2 because there was not enough.
</P><P>PCR purify the digested backbones containing Sense.
</P><P><I>Nanodrop</I></P><P>pLTP - 24.0 
pBet - 17.2
pGer - 42.8
</P><P>Ligate predigested PDK digested with X, P and ligate into purified backbone. Transform and Plate (6? plates total)
</P><DIV id="entry"><SPAN id="date"><A name="07-28-2010" class="date">07-28-2010</A></SPAN> [<A class="top" href="#notebook">top</A>]<P><B>Procedures</B></P><P>Yesterday, we plated E. coli containing complete hpRNA for Ger, Bet v2, and LTP. Bet v1 did not ligate properly in the earlier Sense+Intron ligation. 
</P><P>Today, we will extract the plasmids from the E. coli and prepare them for sequencing tomorrow. We will also do ligations with the false negatives (Sense + PDK ligate with Antisense). 
</P><OL><LI>Culture colonies 
</LI><LI>Miniprep
</LI><LI>Nanodrop 
</LI></OL><P>For our false negatives
</P><OL><LI>Ligate 
</LI><LI>Transform
</LI></OL><P><B>Results</B></P><P><B>Nanodrop</B></P><P>The format is: Miniprep # , Name of Allergen, concentration in ng/μL of colony 1, concentration in ng/μL of colony 2. 
All these working ligations are with the PAL intron
</P><OL><LI>9, LTP, 97.9, 94.1
</LI><LI>11, LTP, 77.9, 94.8
</LI><LI>25, Bet v2, 67.8, 97.8 (there was a mixup with colony 1 and LTP+PDK, but it is now fixed in our electronic notebooks)
</LI><LI>28, Bet v2, 73.2, 83.6
</LI><LI>36, Ger3, 76.1, 83.4
</LI><LI>37, Ger3, 70.0, 75.3
</LI><LI>38, Ger3, 90.4, 108.3
</LI></OL><P><B>Transformed more sense+pdk parts for our false negatives (16,30,31)</B></P><P>The parts already have been digested and purified, so today, all we had to do was ligate and transform.
</P><DIV id="entry"><SPAN id="date">&lt;a name=&quot;07-29-2010&quot; class=&quot;date&quot;&gt;07-29-2010&lt;/a&gt;</SPAN> [&lt;a class=&quot;top&quot; href=&quot;#notebook&quot;&gt;top&lt;/a&gt;]<UL><LI> Ran a gel for our second attempt at amiRNA stiching (stage 1): . It seems that the backbone is still present at high concentrations after PCR purification. Maybe this is why last time we had a high incidence of background? This time we're gel purifying.
</LI></UL><UL><LI>Dignostic Digest 
</LI></UL><P><B>Gel:</B>
ladder, 9, 9, 11, 11, 25, 25, 28, 28, ladder, 36, 36, 37, 37, 38, 38, ladder, LTPS+ PDK, LTPS+PDK, Bet2S+PDK, Bet2S+PDK, GerS+PDK, GerS+PDK, GerS+PDK, GerS+PDK
</P><P><B>Expected band lengths:</B></P><P><I>Complete Parts</I></P><P>9,11 (LTPS+Pal+LTPA): 840bp and 3200 bp
</P><P>25,28 (Bet2S+Pal+Bet2A): 840bp and 3200 bp
</P><P>36,37,38 (GerS+Pal+GerA): 840bp and 3200 bp
</P><P><I>Sense + Intron Parts</I></P><P>LTPS+ PDK: 1200 and 3200 bp
</P><P>Bet2S+ PDK: 1200 and 3200 bp
</P><P>GerS+PDK: 1200 and 3200 bp
</P><P>(9 c1, 25 c2, 36 c1 have a 540 and 3200 band as opposed to a 840 and 3200 band--prbly b/c sense+intron is there but antisense was not ligated in).  All intron+sense parts except for LTPS+PDK appear to have 300 bands as well indicating that PDK was not ligated in.  
</P><P>(a small mix up with two of the lids was made and its possible that LTPS+PDK (w/ an 840 and 3200 band is actually the complete Bet2 part so we're sending this in for sequening anyway).
</P><DIV id="entry"><SPAN id="date"><A name="07-30-2010" class="date">07-30-2010</A></SPAN> [<A class="top" href="#notebook">top</A>]<UL><LI>Sequencing Results:
</LI></UL><P>LTP ihpRNA parts that worked: 9c2, 11c1, 11c2
</P><P>Bet 2 ihpRNA parts that worked: 25c1,c2, 28c1,(LTPS+PDK is actually Bet 2 hpRNA)
</P><P>Ger ihpRNA parts that worked: 36c1,c2, 37c1,2, 38c1,c2
</P><DIV id="entry"><SPAN id="date"><A name="08-03-2010" class="date">08-03-2010</A></SPAN> [<A class="top" href="#notebook">top</A>]<UL><LI>Grew up cultures of completed ihpRNA constructs (Bet, LTP, Ger) in pORE expression vector 
</LI><LI>amiRNA PCR
</LI><LI> Will look at results of PCR tommorrow
</LI></UL><DIV id="entry"><SPAN id="date"><A name="08-04-2010" class="date">08-04-2010</A></SPAN> [<A class="top" href="#notebook">top</A>]<P><B>Tasks</B></P><UL><LI>amiRNA PCR appears to have worked at every Tm we tried:
</LI></UL><UL><LI>Digested V9/V10 to insert our constructs into
</LI><LI>Realized that we hadn't gel purified our ihpRNA inserts 
</LI><LI>Gel purification of inserts (entire ihpRNA parts)
</LI></UL><P>Ladder, 9, 11c1, 11c2, ladder, 25c1, 28c1, 28c2, 36c1, 36c2, ladder
</P><P><B>Results</B></P><UL><LI>Successfully gel purified our inserts and digested backbones that we will ligate into
</LI></UL><DIV id="entry"><SPAN id="date"><A name="08-05-2010" class="date">08-05-2010</A></SPAN> [<A class="top" href="#notebook">top</A>]<UL><LI> Gel extracted V9/V10 backbone
</LI></UL><P>Lanes: Ladder, V9, Ladder, V10
</P><P><I>Concentrations</I>: V9 (9.4 ng/uL; V10 (16.4 ng/uL)
</P><UL><LI>Ligated ihpRNA inserts into V9/V10 and transformed 
</LI><LI>For our ligations we only used ~ 2uL of backbone (around 18 and 32 ng of backbone)and used a 3x excess of insert
</LI></UL><LI> Verified that amiRNA stitching of Bet, LTP yielded the proper insert with a low level of background through PCR:
</LI><P>Lanes: 1-7: Bet, corresponding to 65.55 degrees C for stitching Tm (even spacing)
</P><P>Lanes: 8-10: LTP, corresponding to 65.55 degrees C during stitching annealing, even spacing
</P><UL><LI> Digested
</LI><LI> Bet,LTP inserts with X+P; B21 with X+P+phosphatase
</LI></UL><LI> Ligated, Transformed
</LI><DIV id="entry"><SPAN id="date"><A name="08-06-2010" class="date">08-06-2010</A></SPAN> [<A class="top" href="#notebook">top</A>]<P><I>amiRNA</I></P><UL><LI> amiRNA-transformed turbo cells didn't grow in YEB medium. There should be more LB+Amp available by the end of the day. In the meantime, we're digesting some more backbone (8x20ul X+P+B21)
</LI><LI>ligations of Bet/LTP into V0120 
</LI></UL><P><I>ihpRNA</I></P><UL><LI> no colonies on plates from transformations done yesterday after 16 hours (we let these plates grow for longer)
</LI><LI> redoing ligations of ihpRNA inserts in V9/V10
</LI><LI>Redigested and gel purified some backbone 
</LI></UL><UL><LI>Concentrations: V9 (5 ng/uL); V10 (6 ng/uL)
</LI><LI> 12 ligations each for V9 and V10
</LI></UL><DIV id="entry"><SPAN id="date"><A name="08-09-2010" class="date">08-09-2010</A></SPAN> [<A class="top" href="#notebook">top</A>]<UL><LI> DTLs of amiRNA insert were unsuccessful. Yesterday we concluded this was because turbo cells + amp resistance don't grow on YEB media (we were out of LB+amp media). Today we must adopt a new hypothesis as 0/16 colonies grew on LB+amp media. Working theory is that the backbone digestions pictured below have the digested backbone and inserts as the two bands (top to bottom) instead of the undigested &amp; digested backbone (in hindsight, I was trying to ligate into YFP*2, the original insert of B21). The confusion arose from the 1kb ladder, which at first appeared to match another ladder far better than 1kb (see Thursday's entry or thereabouts). This, combined with the fact that the 1kb ladder was hand-labeled, led me to suspect that it was in fact the other ladder (pR322 as pictured <A href="http://openwetware.org/wiki/Image:Ladders.jpg" class="external text" rel="nofollow">here</A>). This, in turn, supported the (theorized incorrect) conclusion that the top &amp; bottom bands <A href="http://openwetware.org/wiki/Image:Bbcut1.jpg" class="external text" rel="nofollow">here</A> and <A href="http://openwetware.org/wiki/Image:Bbcut2.jpg" class="external text" rel="nofollow">here</A>  were undigested &amp; digested backbone, respectively, and that the insert had diffused. I've DTL'd again, this time using the upper band (which didn't separate into digested &amp; undigested bands after running for 30m on a .7% agarose gel) for ligation. We'll see on Monday how it went.
</LI></UL><UL><LI>New Plan of action:
</LI></UL><OL><LI>Redigest V9/V10 (three digests of each backbone to use more DNA in ligations)
</LI><LI>Grew up V9/V10+ ihpRNA colonies
</LI><LI>Re pcrd amiRNA and digested to ligate into V9/V10
</LI><LI>Catalog parts to be ready for the registry
</LI></OL><DIV id="entry"><SPAN id="date"><A name="08-10-2010" class="date">08-10-2010</A></SPAN> [<A class="top" href="#notebook">top</A>]<P><B>ihpRNA</B></P><UL><LI>Only 2 colonies grew up (V10 9 (37c1) and V10 12 B(38 c2)) <I>ger ihprna constructs</I></LI><LI>growing up colonies again in 5 mL cultures w/less antibiotic (50 ng/uL of kan)
</LI><LI>concentrations: V10_9: 21.4 ng/uL V10_12B: 16.9 ng/uL
</LI></UL><LI>Diagnostic Digest (Should see ~ 7 kb band and 840 band)
</LI><UL><LI>Did not see exepcted bands (waiting on cultures grown up today to miniprep and hopefully transform into agrobacterium tommorow)
</LI></UL><P><B>amiRNA</B></P><UL><LI>Ligated into V9/V10 and transformed
</LI></UL><DIV id="entry"><SPAN id="date"><A name="08-11-2010" class="date">08-11-2010</A></SPAN> [<A class="top" href="#notebook">top</A>]<UL><LI> Minipreps of (V95, V91, V99, V99b, V1011, V107B, V1012, V109b)
</LI><LI>Concentrations (20-60 ng/uL range)
</LI></UL><LI>Diagnostic Digest (should see ~ 7 kb and 840 bp)
</LI><P>Ladder, V9 5 (Bet2), V91(LTP), V99 (Ger), V99B (Ger), V1011 (Ger), V107b(Ger), V1012 (Ger), V109B (Ger)
</P><P>V10 7B (Ger worked)
</P><DIV id="entry"><SPAN id="date"><A name="08-12-2010" class="date">08-12-2010</A></SPAN> [<A class="top" href="#notebook">top</A>]<P><B>ihpRNA</B></P><UL><LI>Minipreps of more V9/V10 colonies
</LI><LI>Diagnostic Digest of minipreps w/ Eco and Pst (Should see ~840 and 7000 bp)
</LI></UL><P>ladder, V92, V92b, V94, V94b, V95, V96b, V97, V97b, V98, V98b
</P><P>Lanes 2&amp;3: LTP (11c1); Lanes 4&amp;5: Bet2 (25c1); Lane 6: Bet2 (25c2); Lane 7: Bet2 (28c1); Lanes 8&amp;9: Ger (36c1); Lanes 10&amp;11 Ger (36c2)
</P><UL><LI>Digested V9/V10 backbone for ligations w/ Eco &amp; Pst (redoing ihpRNA ligations into V9/V10)
</LI></UL><P><B>amiRNA</B></P><UL><LI>Ligated into V9/V10 and transformed (mutagenesis sites confirmed through pcr using 20 bp sepecific primers)
</LI><LI>still need to be sequenced</LI></UL></DIV><DIV class="printfooter">
Retrieved from &quot;<A href="http://2010.igem.org/Team:Harvard/allergy/notebook">http://2010.igem.org/Team:Harvard/allergy/notebook</A>&quot;</DIV></DIV></DIV><DIV id="footer-box" class="noprint"><DIV id="footer"><UL id="f-list"><LI id="t-recentchanges"><A href="/Special:RecentChanges" title="Recent changes">Recent changes</A></LI><LI id="t-whatlinkshere"><A href="/Special:WhatLinksHere/Team:Harvard/allergy/notebook" title="List of all wiki pages that link here [j]" accesskey="j">What links here</A></LI><LI id="t-recentchangeslinked"><A href="/Special:RecentChangesLinked/Team:Harvard/allergy/notebook" title="Recent changes in pages linked from this page [k]" accesskey="k">Related changes</A></LI><LI id="t-specialpages"><A href="/Special:SpecialPages" title="List of all special pages [q]" accesskey="q">Special pages</A></LI><LI><A href="/Special:Preferences">My preferences</A></LI></UL></DIV><DIV id="footer"><UL id="f-list"><LI id="t-print"><A href="/wiki/index.php?title=Team:Harvard/allergy/notebook&amp;printable=yes" title="Printable version of this page [p]" accesskey="p">Printable version</A></LI><LI id="t-permalink"><A href="/wiki/index.php?title=Team:Harvard/allergy/notebook&amp;oldid=168947" title="Permanent link to this revision of the page">Permanent link</A></LI><LI id="privacy"><A href="/2010.igem.org:Privacy_policy" title="2010.igem.org:Privacy policy">Privacy policy</A></LI><LI id="disclaimer"><A href="/2010.igem.org:General_disclaimer" title="2010.igem.org:General disclaimer">Disclaimers</A></LI></UL></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></BODY></HTML>