"
Page
Discussion
View source
History
teams
Log in
 
Team:Harvard/allergy/methods
From 2010.igem.org
home
project
vectors
allergy
flavor
color
genetic fence
human practices
safety
parts
results
people
students
teaching fellows
advisors
acknowledgements
gallery
online
blog
twitter
openwetware
facebook
contact us
allergy knockdown allergen targets lab notebook methods parts and primers references
methods
Creating hypoallergenic plants is a complicated process. Many proteins that provoke allergic reactions are essential for the plant's survival, and plants frequently several isoforms of the allergen genes. Our ability to reduce and eliminate allergy-inducing proteins from a plant is constrained by what proteins the plants need for survival and our success in eliminating homologous versions of the offending protein.
When plants, or any organism, synthesize proteins, genomic DNA is transcribed into mRNA, which is then translated into a protein. In order to decrease or eliminate protein production, the genomic DNA coding for the mRNA can be removed, or transcription or translation can be stopped.
Targeted removing of genomic regions that code for particular proteins is difficult in plants, and is compounded by the existence of multiple isoforms of allergen genes. The preferred method of decreasing protein production in plants is through the process of RNA interference, where artificially introduced sequences of double stranded RNA interfere with the translation of the native mRNA with a complementary sequence.
RNAi
RNAi (RNA interference) is a process used to control expression of genes in living cells.  Since this process down-regulates gene expression by preventing the translation of specific proteins, RNAi is naturally used as a protection mechanism in cells against viruses.  In this process, the cell's machinery recognizes double stranded RNA sequences present in the cell. These sequences are then cut up into shorter fragment and mRNA transcripts that are complementary to these shorter sequences are then cleaved, thereby preveting translation of the proteins that would have come from these sequences. By introducing genes into the plant genome that code for synthetic double stranded RNA sequences complementary to the sequences of the various allergens that we would like to target, we hope to knockdown the expression of these allergens and their isoforms.
hpRNA
With RNAi, the problem of creating a hypoallergenic plant reduces to the problem of introducting short RNA strands into the cell, each with complementarity to allergen’s mRNA. One mechanism of flagging RNA for the RNA interference machinery is to create an RNA hairpin. The hairpin, expressed under a constitutive promoter, is made up of 300 base pairs of sequence that are identical to the targeted gene, a plant intron sequence, and 300 antisense base pairs complementary to the target gene. Upon transcription, this construct will form a hairpin: the targeting sequence and its reverse complement will anneal to each other, the intron will be spliced out, leaving behind a short loop sequence at the top of the hairpin. This structure is called a hpRNA, short for "hairpin RNA." The cell’s RNAi machinery will then process and incorporate part of one of the legs of the hairpin (targeting sequences) with which it will search for and destroy complementary RNA sequences.
Overview
Construction
ihpRNA creation broad overview   click to enlarge
ihpRNA creation detailed process   click to enlarge
We made ihpRNA constructs for our three allergens (Betv1, Ger, LTP). We used 3 sets of primers for each construct: 1)Forward/Reverse primers specific to the biobrick end and sense sequence for our allergen 2) Forward/Reverse primers specific to the biobrick end and antisense sequence for our allergen 3)Forward/Reverse primers specific to the biobrick end and our PDK intron found in the pHannibal Vector. After we had assembled these parts together using BioBrick assembly we could ligate the completed haripins into our agrobacterium expression vector .
amiRNA
Artificial microRNA activates a similar RNA interference mechanism but requires a much shorter input, only 21 base pairs. We used the Web MicroRNA Designer to create a constitutively expressed amiRNA construct that is processed by the plant cell to create a short hairpin RNA (shRNA) of 21 base pairs complementary to the target gene.
Overview
Construction
AmiRNA creation broad overview   click to enlarge
AmiRNA creation detailed process   click to enlarge
In order to create our amiRNA hairpin constructs we used the plasmid RS300 provided by Kirsten Bomblies and Detlef Weigel.  RS300 contains short sequences that would come together to form a hairpin and target a particular sequence in plants.  Since we wanted to target our sequences instead, we used a multi-step pcr process to replace the plasmid's endogenous miRNA sequences with our miRNA sequences.  We created four primers, two of which contained the sequences we wanted to insert.  These two primers would be used to amplify the region in between the plant's own miRNA sequence and add on "our" miRNA sequences to the ends of this region.  The other two primers were used to amplify the regions before and after the plant's endogenous miRNA sequences and add BioBrick prefix and suffix restriction enzymes for compatibility with our expression vectors.  These three pieces were then assembled together through pcr, such that our final construct would contain a biobrick end followed by a stretch of the original RS300, followed by the ~20 base-pairs that were unique to the allergen being targeted, followed by a stretch of RS300, followed by the complementary miRNA sequence of ~20 base pairs, finally ending with a stretch of RS300 and a biobrick end.
The efficiency of knockdown with each method can be significantly different for different targets and both will have to be tested to ensure hypoallergenicity of the resulting plant. For use in synthetic biology, hpRNA is more modular, able to incorporate any plant intron and any plant gene using only PCR and BioBrick assembly. amiRNA is less modular but does not require a PCR template from the plant gene being targeted as the specific miRNA regions can be specified with an oligonucleotide. However, the amiRNA primer designer cannot always find a suitable 20 base pair match in the gene of interest that will function as an effective shRNA and we were thus unable to create amiRNA constructs targeting Ger3 in Arabidopsis. Both methods will therefore likely be valuable for a customizable iGarden requiring efficient knockdown of a wide range of allergens.
Retrieved from " http://2010.igem.org/Team:Harvard/allergy/methods "
Recent changes
What links here
Related changes
Special pages
My preferences
Printable version
Permanent link
Privacy policy
Disclaimers
