<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_Stony_Brook_HP_Gold_Integrated skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><DIV id="top_title"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:Stony Brook/HP/Gold Integrated</SPAN></H1></DIV><DIV id="HQ_page"><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><P><TITLE>Stony Brook 2017</TITLE></P><DIV id="page-wrapper"><DIV id="header"><NAV id="nav"><UL><LI><A href="https://2017.igem.org/Team:Stony_Brook">Home</A></LI><LI><A href="#">Team</A></LI><LI><A href="https://2017.igem.org/Team:Stony_Brook/Team">Team</A></LI><LI><A href="https://2017.igem.org/Team:Stony_Brook/Collaborations">Collaborations</A></LI></UL><LI><A href="#">Project</A></LI><LI><A href="https://2017.igem.org/Team:Stony_Brook/Description">Description</A></LI><LI><A href="https://2017.igem.org/Team:Stony_Brook/Experiments">Experiments</A></LI><LI><A href="https://2017.igem.org/Team:Stony_Brook/Notebook">Notebook</A></LI><LI><A href="https://2017.igem.org/Team:Stony_Brook/InterLab">Interlab</A></LI><LI><A href="https://2017.igem.org/Team:Stony_Brook/Model">Modelling</A></LI><LI><A href="https://2017.igem.org/Team:Stony_Brook/Results">Results</A></LI><LI><A href="https://2017.igem.org/Team:Stony_Brook/Parts">Parts</A></LI><LI><A href="https://2017.igem.org/Team:Stony_Brook/Safety">Safety</A></LI><LI><A href="#">Human Practices</A></LI><LI><A href="https://2017.igem.org/Team:Stony_Brook/HP/Silver">Silver</A></LI><LI><A href="https://2017.igem.org/Team:Stony_Brook/HP/Gold_Integrated">Gold</A></LI><LI><A href="https://2017.igem.org/Team:Stony_Brook/Attributions">Attributions</A></LI></NAV></DIV><SECTION id="banner"><HEADER><H2>Integrated Human Practices</H2></HEADER></SECTION><HEADER><B>Our Initial Outlook</B></HEADER><P>When we sat down to brainstorm the types of antimicrobial peptides we could use to tackle MRSA, we researched various scientific literature and listed a variety of bacteriocins that were plausible candidates to potentially kill MRSA. Unfortunately, we came upon various hurdles throughout each step of our journey. We seeked out the expertise from professors and medical professionals to <B><I>analyze</I></B> the issues at hand and <B><I>integrate</I></B> their suggestions into our work.</P><HEADER><B>The First Challenge: Picking the Bacteriocins</B></HEADER><P>We met with Dr. Jarrod French, a researcher and assistant professor in the Department of Chemistry at Stony Brook University. During the conversation with Dr. French, we learned that several characteristics of our chosen bacteriocins were not feasible to experiment with and use for marketing in the future. In order to mass produce the bacteriocin as a product on the market, it would need to be expressed in <I>E. coli</I> and easily purified and stored. The key issues with our peptides and the implications we needed to consider were:</P><UL style="list-style-type:disc"><LI><FONT size="4">Some of our bacteriocins were too small. It would be favorable to pick bacteriocins that were  10 kDa or about 75 amino acids long, so that we could detect its presence on a gel. Bacteriocins are small in general, thus we needed to tag the peptides in order to increase their size.</FONT></LI><LI><FONT size="4">Some of our bacteriocins were cyclic and non-linear. It would be feasible for us to pick bacteriocin that were linear, so that the peptides are easier to tag, hence they could be spotted on a gel, indicating that they were being produced in the <I>E. coli</I>. This also would allow companies to manufacture our product and purify our bacteriocin efficiently, with a tag that is easily removable leaving a fully functional peptide behind.</FONT></LI><LI><FONT size="4">Some of our bacteriocins underwent post-translational modifications. Post-translational modifications would require more inserts of DNA that codes for the modifications, which is unfavorable for mass production of our bacteriocins using <I>E. coli</I>.</FONT></LI></UL><P>Through these alterations, we were lead to class IId bacteriocins. These are medium sized, linear peptides that require no post-translational modifications in order to kill the target bacteria. </P><HEADER><B>Second Challenge: Creating the Hybrid</B></HEADER><P> After taking the advice from Dr. French into consideration, we picked bacteriocins that were linear, medium-sized, and had no post-translational modifications. These bacteriocins were non-toxic to <I>E. coli</I>, hence we did not require additional immunity genes for our constructs. The next step was to create hybrids of these bacteriocins, but how do we know which of these bacteriocins could work with one another synergistically? We decided to make a phylogeny to understand and model how these peptides relate to one another. We decided to pick bacteriocins that were in the same group, class IId, as well bacteriocins that employed similar modes of action to penetrate the pathogenic strain. We met with Dr. Joshua Rest, a researcher and an associate professor in the Department of Ecology and Evolution at Stony Brook University. He advised us to not completely rule out other classes of bacteriocins and include a large number of bacteriocins from class IId, as well as non-class IId bacteriocins to our phylogenetic tree to confirm that the bacteriocins we picked are in fact more closely related to one another. He also explained the difference between character-based and distance-based models, and advised us to use a maximum likelihood method to construct our tree, with the inclusion of an outgroup to measure relative divergence times of each branch. </P><HEADER><B>Third Challenge: MRSA Testing</B></HEADER><P> Our team met with Dr. Sharon Nachman, a pediatric infectious diseases specialist at Stony Brook Hospital, to discuss our project idea and gain insight into how we could improve it. When we discussed our method of killing Methicillin-resistant <I>Staphylococcus aureus</I> (MRSA) using a hybrid of the bacteriocins Lacticin Z and Aureocin A53 (or Epidermicin NI01), she advised us to test against <I>Staphylococcus aureus</I> as well to demonstrate that our hybrids would be effective against more than one strain. Although the bacteriocins are believed to be specific in combatting only one type of bacteria, there could be possible exceptions that could allow the hybrids to tackle multiple strains. We took her suggestion and tested our hybrids and individual bacteriocins on the strain <I>Staphylococcus aureus</I> subsp. aureus Rosenbach, ATCC # 27661, in addition to the MRSA strain. Additionally, we discussed how the bacteriocins hybrids could be incorporated into certain treatment methods. Dr. Nachman informed us that instead of application on human skin, the project could have a real-life application by creating a wipe for foam wrestling mats or a spray for counters. She said that a lot more time and research is required to develop an ointment for the skin, because several factors need to be taken into consideration, such as how long will it take the ointment to penetrate the skin deep enough to tackle the pathogen and whether our bacteriocins can be toxic to beneficial biofilms. Hence, we needed to focus on creating a product for external cleaning and prevention of infection first. The vast majority of our project was developed theoretically, but this was one way to establish a direct use for it that would allow other people to benefit as well. </P><P>Once it came time to testing our bacteriocins against MRSA and <I>Staphylococcus aureus</I>, Dr. Sangeet Honey, a researcher and professor in the Department of Molecular Genetics and Microbiology at Stony Brook University, offered us his advice and supplies to help the process go as smoothly as possible. Without his dedication to our team, we would not have been able to complete our testing in a BSL 2 lab. Dr. Honey ensured the safety of everyone on Stony Brook’s campus and in our lab throughout the time we executed our experiments. He acted as a liaison between the members of our iGEM team and the administration of Stony Brook University and Stony Brook Hospital during the two months it took for us to get clearance to go ahead with our project. Dr. Honey edited our MRSA testing plan by reviewing what we were planning on doing and adding his expert opinion on how we could collect data that would be reviewed as a possible scientific discovery, while still maintaining the safety of all the members of our team in the process. We ultimately decided on the spot-on-lawn assay and the MIC assay to perform using our bacteriocins on MRSA and other strains of <I>S. Aureus</I>, because Dr. Honey felt that these would be the safest assays to complete in the short time frame we had for the testing. The spot-on-lawn assay had the potential to prove that our bacteriocins did have an effect on the growth of the bacteria we chose to test. The MIC assay was suggested to us by Dr. Honey, because it involved a relatively low amount of MRSA bacteria be used in the test and the results could be used for potential marketing of our product in the future. By knowing the minimum concentration of our peptide needed to kill 50% of the bacteria in our sample, this could be generalized to concentrations used for treatment of MRSA in human subjects. Dr. Honey helped us understand the importance of our project as well by explaining the long, costly procedure of producing a new antibiotic onto the market in today’s economy. The potential bacteriocins have in the medical field is vast, because of the growing issue of antibiotic resistant bacteria and the lack of new antibiotics being researched and released for clinical use. This solidified our reasoning for choosing this topic of research and with Dr. Honey’s help, we were able to go through the first necessary round of testing that could be built upon in the future. </P><DIV id="footer"><DIV class="container"><DIV class="row"><DIV class="12u"><DIV class="copyright"><UL class="menu"><LI>© 2017 Stony Brook iGEM</LI><LI>Design: <A href="http://html5up.net">HTML5 UP</A></LI></UL></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></BODY></HTML>