<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_UIUC_Illinois_Project skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><DIV id="top_title"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:UIUC Illinois/Project</SPAN></H1></DIV><DIV id="HQ_page"><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><DIV class="container"><DIV class="containers-fluid"><DIV class="topnav collapse navbar-collapse" id="bs-example-navbar-collapse-1"><A id="centered" href="https://2017.igem.org/Team:UIUC_Illinois">Home</A><A id="centered" href="https://2017.igem.org/Team:UIUC_Illinois/Project">Project</A><A id="centered" href="https://2017.igem.org/Team:UIUC_Illinois/Team">Team</A><A id="centered" href="https://2017.igem.org/Team:UIUC_Illinois/Parts">Parts</A><A id="centered" href="https://2017.igem.org/Team:UIUC_Illinois/Wet_lab">Wet lab</A><A id="centered" href="https://2017.igem.org/Team:UIUC_Illinois/Outreach">Outreach</A><A id="centered" href="https://2017.igem.org/Team:UIUC_Illinois/Collaboration">Collaboration</A><A id="centered" href="https://2017.igem.org/Team:UIUC_Illinois/Contact">Contact</A><A id="centered" href="javascript:void(0);" class="icon" onclick="myFunction()">☰</A></DIV></DIV></DIV><BUTTON onclick="topFunction()" class="topButton" id="myBtn" title="Go to top">Back to top</BUTTON><NAV id="sidenavbar"><UL style="list-style: none;"><LI><A href="#background" data-number="1"><SPAN class="cd-label">BACKGROUND</SPAN></A></LI><LI><A href="#description" data-number="1"><SPAN class="cd-label">DESCRIPTION</SPAN></A></LI><LI><A href="#safety" data-number="2"><SPAN class="cd-label">SAFETY</SPAN></A></LI><LI><A href="#results" data-number="3"><SPAN class="cd-label">RESULTS</SPAN></A></LI></UL></NAV><ASIDE id="fh5co-hero"><DIV class="fh5co-cta" style="background-image: url(https://static.igem.org/mediawiki/2017/0/07/DNA.jpg);"><DIV id="teamHeading" class="containers"><DIV class="col-md-10 col-md-offset-1 text-center"><H2 id="topHeader">Project</H2></DIV></DIV></DIV></ASIDE><DIV id="start" class="col-md-10 col-md-offset-1  fh5co-heading animate-box"><H1 id="homeH">Background</H1><P id="homep">
					Gibson Assembly is a revolutionary method for assembling multiple linear DNA fragments (original paper) by Dr. Daniel Gibson at the J. Craig Venter Institute in 2009. Multiple overlapping DNA fragments can be joined by a single reaction regardless of the fragment length, which adds to the versatility of the method. By adding the three different enzymes (T5 5’ exonuclease, Phusion DNA polymerase, and Taq DNA ligase), a fully ligated double-stranded DNA molecule is assembled. This method is proven to be efficient due to the ease of the reaction – needing only one tube of reaction – and the effectiveness of the reaction: no scars on the ligated DNA, non-selective compatibility of DNA fragments, and no specific restriction sites are needed.  
				</P><H> Hyperlink to original paper:</H><A href="http://www.nature.com/nmeth/journal/v6/n5/full/nmeth.1318.html">http://www.nature.com/nmeth/journal/v6/n5/full/nmeth.1318.html</A><OL><H style="text-align:center;"><B>Preparation before performing Gibson Assembly 
                                    </B></H><LI id="homep"><I>Primer Design</I>
When designing the primers, ensure that adjacent segments in the plasmid have identical sequences on the ends. The insert sequence and the vector sequence should be compatible with each other. These identical sequences can be created via PCR with primers containing a 5’ end identical to the adjacent segment and a 3’ end that would anneal to the target sequence. Once the primers have been designed, amplify the amount of primer DNA by PCR.  </LI><P>Traditional cloning relied on sticky ends created by restriction sites, which normally provide 4 bp overhangs. Gibson Assembly improves on this aspect by providing a larger overhang via the primers designed to amplify the DNA vector and DNA insert.  This technique generates a 20-30 bp overhang.
</P><LI id="homep">Check the purity and concentration of the PCR amplification and/or restriction digest. If the product is found to be impure, perform a gel purification to remove impurities.  </LI></OL><OL><H style="text-align:center;"><B>Preparation before performing Gibson Assembly: </B></H><P id="homep">
					For our method, we used NEB’s Gibson Assembly Master Mix. The Master Mix consists of three enzymes in a single buffer with the following functions:
                                      <UL>1. T5 5’ exonuclease, which chews back the 5’ end of the DNA to create a 3’ overhang so that the complementary strands could anneal to each other.</UL><UL>2.Phusion DNA Polymerase, which incorporates nucleotides to fill in the gaps in the annealed DNA fragment</UL><UL>3. Taq DNA ligase, which anneals the DNA fragments and removes the ‘nicks’ and ‘scars’.

</UL></P></OL><P id="homep">The method can simultaneously combine up to 15 DNA fragments based on sequence identity. It requires that the DNA fragments contain ~20-40 base pair overlap with adjacent DNA fragments. The appropriate amount of DNA, when combining 2-3 fragments in a Gibson Assembly reaction, is 0.02 – 0.5 pmol of total DNA. Cloning efficiency is best when 50 -100 ng of vector is used, with 2-3 equivalents of insert in a 20 µl reaction. Sample should then be incubated in a thermocycler at 50ºC for 15 minutes. After 15 minutes, the assembly should be complete. </P></DIV><DIV id="fh5co-services-section"><DIV id="start" class="col-md-10 col-md-offset-1  fh5co-heading animate-box"><H1 id="homeH">Experiment Description</H1><P id="homep">One of the biggest obstacles to scientific advancement and discovery is the exorbitantly high price for conducting simple experiments.  To put this in perspective, cloning 2-3 fragments of DNA using Gibson assembly costs <B id="fontp">$18.50</B>. Currently, the purchase of one Gibson Assembly Kit for ten reactions is <B id="fontp">$185.00</B>. For research that heavily depends on assembling many DNA fragments, the bill can rise very quickly.</P><P id="homep" v="">The aim of creating a homemade Gibson Assembly recipe was to drastically lower the prices of performing Gibson Assembly. The price could be lowered if the same outcome could be performed using unpurified enzymes. The unpurified enzymes would replace the purified enzymes commonly used in the Gibson Assembly master mix.  </P><P id="homep">The idea of our project can be summarized in one sentence “using Gibson assembly to make enzymes for Gibson assembly.”  </P><P id="homep">To expand on this statement, the Gibson assembly cloning method was used to create constructs that contained the genes for Pyrococcus furiosus DNA polymerase and Thermotoga maritima DNA ligase. These constructs were created so that when they are transformed into DH5α cells, the local cell machinery would be used for high expression of the genes and produce an abundant number of T. maritima DNA ligase and P. furiosus DNA polymerase enzymes.</P><P id="homep">Once colonies of the transformed cells grow, cell lysate will be made in buffer similar to the Gibson assembly buffer. Two tubes of cell lysate, one containing T. maritima DNA ligase and the other containing P. furiosus DNA polymerase, will be added in ratios relative to each other to the reaction tubes. The ratio that yields results will be determined empirically. The background T5 5’ exonuclease naturally present in the lysate would account for the T5 5’ exonuclease needed for the reaction.

</P><P id="homep">To verify that the isothermal reaction using unpurified enzyme worked, the allegedly assembled construct must be transformed into DH5α cells. The DNA can then be purified from the bacterial colonies and sent to the UIUC Core Sequencing facility for sequencing. If the returned sequence matches the known sequence of the insert, we can conclude that the isothermal reaction was successful.</P></DIV><DIV id="start" class="col-md-10 col-md-offset-1  fh5co-heading animate-box"><H1 id="homeH"> Safety Overview</H1><P>Our chassis organism was e. Coli. We did not work with other organisms to test our construct. We used E. coli strains BL21 and DH5-α.</P><P> Careful aseptic technique was used when handling cell cultures. Sharps were disposed of in a sharps container, and all other waste was deposited in a red bio-waste bag and later autoclaved.</P><P> To reduce risk, we used the ethidium bromide alternative, Midori Green, for preparing gels. All organisms handled were BSL1 by NIH standards.</P><P>As a rule, we did not work in the laboratory alone. Most work was done during regular business hours, and all weekend and evening wet lab work was done under the supervision of our advisors.</P><H1 id="homeH"> Safety Training</H1><P>All team members were required to attend safety training. Our team had two types of training. The first was an online training by the DRS (Division of Research Safety). This safety training included multiple quizzes and tutorials for General Laboratory Safety and Understanding BioSafety. Additionally, team members participated in a “synthetic biology bootcamp” at the beginning of the year to cover general laboratory procedures and safety guidelines. Other safety training such as autoclaving came at another time.</P><P>For all experiments, our lab required usage of gloves, safety glasses, lab coats, closed-toe shoes and long pants to minimize the amount of contact between us and the materials that are being used. Also, general aseptic protocols were adhered to when handling any lab techniques to prevent contamination. Our lab required us to go through a live training with a lab representative to learn proper handling and care of the autoclave.</P><H1 id="homeH">Personal Safety Checklist</H1><P> Laboratory coats, appropriate gloves, safety googles are worn. After use, gloves should be removed aseptically and hands will then be washed.</P><P> Personnel must wash their hands after handling infectious materials and animals, and before they leave the laboratory working areas.</P><P> Safety glasses, face shields (visors) or other protective devices must be worn when it is necessary to protect the eyes and face from splashes, impacting objects and sources of artificial ultraviolet radiation.</P><P> It is prohibited to wear protective laboratory clothing outside the laboratory, e.g. in canteens, coffee rooms, offices, libraries, staff rooms and toilets.</P><P>Open-toed footwear must not be worn in laboratories. </P><P> Eating, drinking, smoking, applying cosmetics, and handling contact lenses is prohibited in the laboratory working areas.</P><P> Storing human foods or drinks anywhere in the laboratory working areas is prohibited. </P><P> Protective laboratory clothing that has been used in the laboratory must not be stored in the same lockers or cupboards as street clothing.</P></DIV><DIV id="start" class="col-md-10 col-md-offset-1  fh5co-heading animate-box"><H1 id="homeH">Obtaining &amp; Confirming Gene Identity</H1><P>The first obstacle in starting an experiment such as ours is finding the genes that code for the proteins we need. In our case, we needed genes for DNA polymerase and DNA ligase.  We obtained strains of Escherichia coli from Dr. Chubiz’s Lab with the guidance and direction from Dr. Christopher Roa containing the plasmids for DNA polymerase from Pyrococcus furiosus and DNA ligase from Thermotoga maritima.</P><P>We grew cells with each type of plasmid on agar plates and purified DNA to send for sequencing. Based on the sequencing results and the BLAST results, we confirmed that DNA polymerase gene originated from Pyrococcus furiosus but was codon-optimized and and the DNA ligase gene originated form Thermotoga maritime.  The P. furiosus DNA polymerase contained a single-stranded binding protein that originated from Sulfolobus sulfataricus. We could not confirm the entire sequence of P. furiosus DNA polymerase other than the first 150bp and the last 150 bp. The codon-optimization quality of the P. furiosus DNA polymerase was the main issue in finding the complete sequence.  However, we were able to successfully confirm the complete sequence of T. maritime DNA ligase. </P><H1 id="homeH">Assembling Our Parts</H1><P>We thus proceeded to assemble the two genes into a pSB1C3 backbone, per iGEM requirements (see Fig. 1). </P><P>Figure 1. The BBa_J4500 plasmid is linearized and digested via restriction digest using SpeI and XbaI restriction enzymes. The P. furiosus DNA polymerase and T. maritime DNA ligase genes were amplified with Gibson assembly primers via PCR. The JOE backbone was also amplified using Gibson assembly primers via PCR. The beginning steps of Gibson assembly are depicted in the last step of the figure. This indicates the T5 5’ exonuclease chewing back the ends of the DNA fragments. </P></DIV><FOOTER id="fh5co-footer" role="contentinfo"><DIV class="containers"><DIV class="col-md-12 text-center"><P><SPAN>Made by UIUC_Illinois iGEM</SPAN></P></DIV></DIV></FOOTER></DIV></DIV></DIV></DIV></DIV></DIV></BODY></HTML>