<HTML xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en" dir="ltr">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki  ltr ns-0 ns-subject page-Team_Lyon-INSA_notebook"><DIV id="globalWrapper"><DIV id="top-section"><DIV id="p-logo"><A href="/Main_Page" title="Main Page">&quot;
	    </A></DIV><DIV id="menubar" class="left-menu noprint"><UL><LI class="selected"><A href="/Team:Lyon-INSA/notebook">Page               </A></LI><LI class="new"><A href="/wiki/index.php?title=Talk:Team:Lyon-INSA/notebook&amp;action=edit&amp;redlink=1">Discussion               </A></LI><LI><A href="/wiki/index.php?title=Team:Lyon-INSA/notebook&amp;action=edit">View source               </A></LI><LI><A href="/wiki/index.php?title=Team:Lyon-INSA/notebook&amp;action=history">History               </A></LI><LI style="color:#808080;cursor:default">teams</LI></UL></DIV><DIV class="right-menu noprint" id="menubar"><UL><LI id="pt-login"><A href="/wiki/index.php?title=Special:UserLogin&amp;returnto=Team:Lyon-INSA/notebook" title="You are encouraged to log in; however, it is not mandatory [o]" accesskey="o">Log in</A></LI></UL></DIV><DIV id="search-controls" class="noprint"><FORM action="/Special:Search" id="searchform"> </FORM></DIV></DIV><DIV id="content"><H1 class="firstHeading">Team:Lyon-INSA/notebook</H1><DIV id="bodyContent"><H3 id="siteSub" class="noprint">From 2012.igem.org</H3><DIV id="xml" style="display:none;"><XML id="xmldata" style="display:none;"><PROJECT><MONTH name="July" size="31"><JOUR nb="2"><TITRE>For all purposes</TITRE><DATE> Monday, July 2nd 2012</DATE><DESCRIPTION>Liquid culture (5 mL LB medium) of NM 522 cells and overnight incubation under agitation at 37°C for later transformation.</DESCRIPTION></JOUR><JOUR nb="3"><TITRE>For all purposes</TITRE><DATE> Tuesday, July 3rd 2012</DATE><DESCRIPTION><P>Dilution of 100 µL saturated culture in  5 mL LB medium. </P><P>Incubation time : 2 hours (until OD<SUB>600</SUB> = 0.3). </P>
Transformation of the NM522 strain (this experiment was repeated 3 times). 
<UL><LI>For the positive control the pSB1C3 plasmid was used;</LI><LI>For the transformation, the pBBA_I742123 was used (well A2, iGEM 2012 kit plate 5).</LI></UL>
The transformed bacteria were selected on chloramphenicol (Cm) plates.
</DESCRIPTION></JOUR><JOUR nb="4"><TITRE>For all purposes</TITRE><DATE> Wednesday, July 4th 2012</DATE><DESCRIPTION>
Transformation analysis :<UL><LI>Positive control : lots of colonies;</LI><LI>Negative control : one of the three negative controls was contaminated (the correspondent transformed colonies were not used). No colonies were observed on the two other negative control plates;</LI><LI>Test plate : between 1 and 8 were observed.</LI></UL>
4 liquid cultures (5mL LB medium + 50 µL chloramphenicol) and 4 streaked chloramphenicol plates were made using isolated colonies likely to contain transformed bacteria.

The antibiotic resistance to Ampicillin, Tetracyclin, Chloramphenicol and Kanamycin of the following strains was tested : BS 168, BS 168 MCherry, BS 168 GFP, BS 168 Lysostaphin, <I>Staphylococcus epidermidis</I>, BS <I>abrB</I>.
</DESCRIPTION></JOUR><JOUR nb="5"><TITRE>For all purposes</TITRE><DATE> Thursday, July 5th 2012</DATE><DESCRIPTION>
Antibiotics resistance tests :<UL><LI>Bs 168 : no resistance;</LI><LI>Bs 168 M cherry : no resistance;</LI><LI>Bs 168 GFP : no resistance;</LI><LI>Bs <I>abrB</I> : Cm resistant;</LI><LI>Bs 168 lysostaphin PWG100 : no resistance;</LI><LI><I>S. epidermidis</I> : Tet resistant.</LI></UL>
Extraction of 4 clones containing the pBBA_I742123 plasmid. Verification by gel electrophoresis (after <I>Eco</I>RI digestion)

</DESCRIPTION></JOUR><JOUR nb="6"><TITRE>For all purposes</TITRE><DATE> Friday, July 6th 2012</DATE><DESCRIPTION>

Phone conference with Romain Briandet.

Terminator was retrieved from plate 1 well 13D.
Long meeting.

</DESCRIPTION></JOUR><JOUR nb="9"><TITRE>For all purposes</TITRE><DATE> Monday, July 9th 2012</DATE><DESCRIPTION><UL><LI>pBBa_I742123 was put in storage (under the reference pBK1).</LI><LI>A liquid culture of NM522/pBK1 transformed cells was made so we could extract more plasmid DNA (50 mL LB medium + 500 µL Chloramphenicol + 500 µL liquid culture of transformed bacteria ).</LI><LI>The 3 genes coding for lysostaphin, dispersin and <I>lacI</I> repressor in pUC57 Ampicillin resistant plasmids were delivered.</LI><LI>Transformation of NM522 strain with pUC57-Lysostaphin, pUC57-Dispersin and pUC57-<I>sfp</I>.</LI><LI>200 µL of each transformed strains were spread on LB + Ampicillin plates.</LI><LI>Liquid cultures of the following strains were made : Bs 168 in LB, Bs <I>abrB</I> in LB + Chloramphenicol, <I>S. epidermidis</I> in LB + Tetracyclin</LI></UL></DESCRIPTION></JOUR><JOUR nb="10"><TITRE>For all purposes</TITRE><DATE> Tuesday, July 10th 2012</DATE><DESCRIPTION><UL><LI> The following parts were put in storage :</LI><UL><LI>Lysostaphin in pUC57 Amp resistant (pBK2);</LI><LI>Dispersin in pUC57 Amp resistant (pBK3);</LI><LI>Surfactin part 2 (RBS-<I>lacI</I>-terminator) in pUC57 Amp resistant (pBK4).</LI></UL><LI> and the following strains (in LB broth supplemented with required antibiotic) :</LI><UL><LI><I>S. epidermidis</I> = BK1 (Tet<SUP>R</SUP>);</LI><LI>Bs <I>abrB</I> = BK2 (Cm<SUP>R</SUP>);</LI><LI>Bs 168 = BK3.</LI></UL><LI>Ligation of Dispersin and Lysostaphin biobricks :</LI><UL><LI>digestion of pBK2 with the restriction enzymes <I>Eco</I>RI and <I>SpeI</I>;</LI><LI>digestion of pBK3 with the restriction enzymes <I>PstI</I> and <I>XbaI</I>;</LI><LI>digestion of pBK4 with the restriction enzymes <I>EcoRI</I> and <I>PstI</I>;</LI><LI>3A ligation of the digested parts;</LI><LI>electrophoresis control showed the expected fragment for lysostaphin and dispersin (2.1 kb). However, the digested backbone plasmid had 3 fragments instead of 2.</LI></UL><LI>Plasmid A2 extraction with a midiprep (pBK5) and digestion → 3 bands are observed : <STRONG>PROBLEM. Digestion is made again with E, P and E+P → There are 2 <I>PstI</I> sites !!! WRONG PLASMID</STRONG></LI><LI>Transformation of NM522 strain with the part BBa_K606061 Chloramphenicol resistant.</LI><LI>Transformation of NM522 strain with the part BBa_B0010 Ampicillin resistant.</LI><LI>Transformation of NM522 strain with the part BBa_K606040 Chloramphenicol resistant.</LI><LI>Design and order of the primers for the constitutive promoter (part BBa_K143012).</LI></UL></DESCRIPTION></JOUR><JOUR nb="11"><TITRE>For all purposes</TITRE><DATE> Wednesday, July 11th 2012</DATE><DESCRIPTION><UL><LI>Extraction of part BBa_K606061 (RBS) Chloramphenicol resistant from transformed NM522 strain.</LI><LI>Extraction of part BBa_B0010 (terminator) Ampicillin resistant from transformed NM522 strain. <I>(NB : the clones were pink which means that the part contains a RFP gene)</I></LI><LI>Extraction of part BBa_K606040 (pLacI) from transformed NM522 strain.</LI><LI>Extraction of  pUC57-Lysostaphin/pUC57-Dispersin/pUC57-<I>lacI</I> from transformed NM522 strains.</LI><LI>Transformation of NM522 strain with the part BBa_0010 Ampicillin resistant. The observed phenotype does not correspond to the description found in the registry (the color of the colonies is pink). We decided to make another transformation with the same part, only this time the 2011 kit was used.</LI></UL></DESCRIPTION></JOUR><JOUR nb="12"><TITRE>For all purposes</TITRE><DATE> Thursday, July 12th 2012</DATE><DESCRIPTION><UL><LI>PCR product electrophoresis of Promoter PCR : the product is not the good one.</LI><LI>Reception and storage of the primers (for the amplification of the BBa_K143012 part).</LI><LI>The transformed NM522 strain with  the part  BBa_0010 Ampicillin resistant from the 2011 iGEM kit shows the same phenotype as the part found in the 2012 kit.</LI><LI>Extractions with a miniprep kit are made :
     </LI><LI>4 clones containing the P<SUB>lac</SUB> promoter (1,2,3,4);</LI><LI>4 clones containing the <I>Bacillus subtilis</I> RBS (1,2,3,4);</LI><LI>3 clones containing theTerminator (1,2,3);</LI><LI>4 clones containing the gene for Dispersin;</LI><LI>4 clones containing the gene for Lysostaphin;</LI><LI>4 clones containing the gene for <I>lac</I>I.</LI></UL>
	Then a digestion is made on a 0.7% agarose gel.</DESCRIPTION></JOUR><JOUR nb="13"><TITRE>Kill</TITRE><DATE> Friday, July 13th 2012</DATE><DESCRIPTION><UL><LI>PCR of the constitutive promoter (part BBa_K143012) → the PCR did not work so we ran a new PCR with a more diluted promoter solution.</LI><LI>The following strains are put in storage:
     </LI><LI>NM522 containing <I>lacI</I>-pUC57;</LI><LI>NM522 containing RBS-pUC57;</LI><LI>NM522 containing dispersin-pUC57;</LI></UL></DESCRIPTION></JOUR><JOUR nb="16"><DATE> Monday, July 16th 2012</DATE><TITRE>Kill</TITRE><DESCRIPTION><UL><LI>PCR of constitutive promoter → it didn’t work. A new PCR is run with modifications of settings. The annealing temperature was modified from 50°C to 57°C, and 3 solutions with different concentration were tested, however the concentration did not affect the yield.</LI><LI>Purification of the PCR product.</LI><LI>Gel electrophoresis of lysostaphin.</LI></UL></DESCRIPTION><TITRE>For all purposes</TITRE><DESCRIPTION><UL><LI>Transformation of B0015 terminator.</LI><LI>Strains BK13 (Bs arbB), BK10 (Bs 168 GFP) and BK11 (Bs 168 mCherry) are put in storage.</LI></UL></DESCRIPTION></JOUR><JOUR nb="17"><DATE> Tuesday, July 17th 2012</DATE><TITRE>For all purposes</TITRE><DESCRIPTION><UL><LI>Long morning meeting to discuss results and to write some posters in a frenzied psychopath way in order to remember our tasks.</LI><LI>Ligation of promoter-RBS-<I>gfp</I> in plasmid.</LI><LI>Genomic DNA extraction of <I>B. subtilis</I> 168: buffers preparation.</LI><LI>Failure of B0015 transformation.</LI></UL></DESCRIPTION><TITRE>Kill</TITRE><DESCRIPTION><UL><LI>Cloning of the constitutive promoter in front of the dispersin part (the gene coding for Dispersin) and RBS/<I>gfp</I> (of <I>E. coli</I>) using a 3A ligation followed by transformation of the ligation in the NM522 strain;</LI><LI>Extraction of pUC57-lysostaphin. Gel electrophoresis showed a shaded DNA → RNase addition in the extraction kit buffer was forgotten.</LI></UL></DESCRIPTION><TITRE>Physiological tests : Testing the potential of S. epidermidis to form biofilms</TITRE><DESCRIPTION><UL><LI>Liquid culture of <I>S. epidermidis</I> in 5mL of TSB (conditions described in protocol “Tests on Staphylococcus aureus biofilms in 24 well plate” ).</LI><LI>A bacterial suspension of <I>S. epidermidis</I> with OD<SUB>600</SUB> close to 0.132 is prepared from a Petri dish containing <I>S. epidermidis</I>. A microtiter plate is inoculated with 2 mL of the suspension  diluted 1:100 with TSB supplemented with different concentrations of glucose in order to test the best conditions for biofilm formation. The plate is incubated during 24 hours at 37ºC.</LI></UL></DESCRIPTION></JOUR><JOUR nb="18"><DATE> Wednesday, July 18th 2012</DATE><TITRE>For all purposes</TITRE><DESCRIPTION><UL><LI>Extraction and digestion (E &amp; P) of Bs 168 strain GFP’s plasmid. Gel electrophoresis showed absence of non digested plasmid → extraction failure.</LI><LI>Genomic DNA extraction of <I>B. subtilis</I> 168 (Kit Genomic DNA from tissue).</LI><LI>Transformation of pBK5 in <I>B. subtilis</I><I>abrB</I> failed.</LI></UL></DESCRIPTION><TITRE>Kill</TITRE><DESCRIPTION><UL><LI>The transformation results of the 3A ligation ([dispersin+RBS/<I>gfp</I>+pSB1T3]) was unsuccessful, so it was repeated, this time with both a positive control (strain 39) and a negative one with NM522 culture ;</LI><LI>Addition of RNase to the miniprep from pUC57-lysostaphin clones. Gel electrophoresis : there is still a smear. However, the bands are as expected.</LI></UL></DESCRIPTION><TITRE>Physiological tests : Testing the potential of S. epidermidis to form biofilms</TITRE><DESCRIPTION><UL><LI>The microtiter plate prepared the day before is analyzed. Each well is stained with crystal violet and subsequently OD<SUB>600</SUB> is measured in order to quantify the adherence of the strain. We can conclude that the concentration of glucose has no impact on the adherence of <I>S. epidermidis</I>.</LI><LI>A microtiter plate is inoculated with 2mL of a suspension obtained from diluting the liquid culture 1:100  with TSB supplemented with different concentrations of glucose in order to test the best conditions for biofilm formation. The plate is incubated during 24 hours at 37ºC.</LI></UL></DESCRIPTION></JOUR><JOUR nb="19"><DATE> Thursday, July 19th 2012</DATE><TITRE>For all purposes</TITRE><DESCRIPTION><UL><LI>TSS tests are also made to be sure that all TSS solutions that we have are ok because some transformations did not work.</LI><LI>A 50µg/mL erythromycin solution is made in order to test the strains’ resistance;</LI><LI>Transformation of <I>yfp</I>, <I>gfp</I> and <I>cfp</I> (from iGEM plates) in NM522;</LI><LI>B0015 transformation in NM522.</LI></UL></DESCRIPTION><TITRE>Kill</TITRE><DESCRIPTION><UL><LI>The transformation of the 3A ligation ([dispersin+RBS/<I>gfp</I>+pSB1T3]) was successful, so 6 clones were tested on a plate containing LB broth supplemented with Tetracyclin.</LI><LI>The fluorescence test of the transformed bacteria containing [dispersin+RBS/<I>gfp</I>+pSB1T3]  confirm that the promoter is functional.</LI><LI>Ligation of the promoter in a Cm resistant iGEM plasmid was made in order to send it to the registry.</LI></UL></DESCRIPTION><TITRE>Physiological tests : Testing the potential of S. epidermidis to form biofilms</TITRE><DESCRIPTION>
The microtiter plate prepared the day before is analyzed. Each well is stained with crystal violet and subsequently OD<SUB>600</SUB> is measured in order to quantify the adherence of the strain. We can conclude that the concentration of glucose has no impact on the adherence of <I>S. epidermidis</I> and that the fact of cultivating the strain in broth or on solid medium has no impact on the biofilm's quality.
</DESCRIPTION></JOUR><JOUR nb="20"><DATE> Friday, July 20th 2012</DATE><TITRE>For all purposes</TITRE><DESCRIPTION><UL><LI>Transformation results : all TSS work (except maybe 1 and 2 with a smaller yield) and promoter+pSB1C3 OK → 6 clones are isolated on a LB+Cm plate.</LI><LI>Quantification and gel electrophoresis of <I>B. subtilis</I> genomic DNA.</LI><LI>Antibiotic testing of <I>B. thuringensis</I> 407 <I>gfp</I> strain and other strains in LB+Ery growth medium.</LI><LI>Failure of transforming pBK5 in <I>B. subtilis</I> 168 with the same protocol as previously. New task : find a new protocol.</LI></UL></DESCRIPTION><TITRE>Kill</TITRE><DESCRIPTION><UL><LI>Transformation of NM522 strain with the part BBa_K606030 (pBK6) and promoter+dispersin.</LI><LI>Ligation Promoter+Dispersin in pIG23 (from Cobalt Buster Lyon-INSA-ENS 2011 Team collection) and transformation.</LI><LI>Ligation Lysostaphin in pSB1C3 and transformation.</LI><LI>Isolation of 6 clones of the Terminator transformation.</LI><LI>Transformation results of fluorescent genes : ok, except <I>yfp</I>. </LI><LI>NM522 strain containing pUC57-lysostaphin is put in storage under the name of BK12.</LI></UL></DESCRIPTION></JOUR><JOUR nb="23"><DATE> Monday, July 23rd 2012</DATE><TITRE>For all purposes</TITRE><DESCRIPTION><UL><LI>Selected clones (NM522+pBK6) are red, meaning that P<SUB>veg</SUB> promoter and RBS from <I>B. subtilis</I> are recognized by <I>E. coli</I>’s ribosome. 4 clones are streaked in LB+Cm.</LI><LI>LB+Ery and TSB+Tet Petri plates are made.</LI><LI><I>yfp</I> transformation was successful.</LI><LI>GFP and CFP plasmids’ extraction showed a failure in ligation.</LI><LI><I>B. subtilis</I> 168 is put in storage (BK14) in TSB medium.</LI><LI>Extraction of B0015 terminator. Gel electrophoresis showed 4 good clones.</LI></UL></DESCRIPTION><TITRE>Kill</TITRE><DESCRIPTION><UL><LI>Transformation results</LI><UL><LI>Promoter+Dispersin in pIG23 : nothing on the plate;</LI><LI>Lysostaphin, negative control contains bacteria so the plate is put in junk.</LI></UL><LI>New digestions, ligations,transformations:</LI><UL><LI>Lysostaphin+Dispersin in pUC57;</LI><LI>Dispersin in pSB1C3;</LI><LI>Lysostaphin in pSB1C3;</LI><LI>Promoter+Dispersin in pIG23 (from Cobalt Buster Lyon-INSA-ENS 2011 Team collection).</LI></UL><LI>Streaking of 4 clones of the transformed strain NM522/pBK6 on a Cm + LB plate. All of them formed red colonies which shows that the constitutive promoter and the RBS specific to the <I>Bacillus subtilis</I> strain are recognized by the RNA polymerase of <I>E. coli</I>.</LI></UL></DESCRIPTION><TITRE>Stick</TITRE><DESCRIPTION>
PCR of xylR. Gel electrophoresis : the PCR didn’t work.
</DESCRIPTION></JOUR><JOUR nb="24"><DATE> Tuesday, July 24th 2012</DATE><TITRE>For all purposes</TITRE><DESCRIPTION><UL><LI>Plasmid extraction from Bacillus strain (BT407 GFP). We tested 2 methods in parallel. However, only the one using the extraction kit seems to work.</LI><LI>Extraction of <I>gfp</I>, <I>cfp</I>, <I>yfp</I> ;  test → OK : put in storage.</LI><LI>Extraction of the pHT315 GFP plasmid of <I>B. thuringiensis</I> 407 GFP to build a shuttle vector <I>B. subtilis</I> ↔ <I>E. coli</I>. Transformation in NM522 strain and selection on LB+Ampicillin medium : ok.</LI><LI><I>S. epidermidis</I> is put in storage in TSB medium under the reference BK15.</LI></UL></DESCRIPTION><TITRE>Kill</TITRE><DESCRIPTION><UL><LI>Transformation results:
       </LI><LI>Lysostaphin+Dispersin in pUC57 : the plate is covered of clones;</LI><LI>Promoter+Dispersin : nothing on the plate;</LI><LI>Dispersin in pSB1C3 : a lot of clones;</LI><LI>Lysostaphin in pSB1C3 : a lot of clones.</LI></UL>
       Selection of 4 clones of each lysostaphin+iGEM plasmid and dispersin+iGEM plasmid on Cm plate and in a liquid culture. Isolation of lysostaphin+dispersin (because there are too many clones!)<LI>Extraction of the plasmidic DNA [Promoter in pSB1C3].</LI><LI>Extraction of pBK6 from the transformed strain.</LI><LI>Extraction of the [Promoter+RBS+<I>gfp</I>] in pBK23 (Tet R) by miniprep and verification by electrophoresis after digestion by <I>Eco</I>RI and <I>Pst</I>I : gel ok. Congrats (pBK12) → 122,6 ng/µL.</LI><LI>Extraction of the pBK6 plasmid (P<SUB>veg</SUB>+RBS+RFP in pSB1C3). The electrophoresis confirms the plasmid’s structure.</LI><LI>Liquid culture of Bs168 pWG100 overnight in order to do the first lysostaphin tests on plates.</LI></DESCRIPTION><TITRE>Stick</TITRE><DESCRIPTION>
New PCR of xylR with some modifications in the protocol : the PCR didn’t work.

</DESCRIPTION></JOUR><JOUR nb="25"><DATE> Wednesday, July 25th 2012</DATE><TITRE>For all purposes</TITRE><DESCRIPTION><UL><LI>Strains are put in storage :</LI><UL><LI><I>S. epidermidis</I> in TSB+Tet (BK17);</LI><LI><I>B. thuringensis</I> 407 GFP in LB+Ery (BK16).</LI></UL><LI>Minipreps to extract the plasmidic DNA of the clones NM522 + pHT315 GFP and verification by electrophoresis : plasmid ok (digested by E, X, S, P and not digested).</LI><LI>Transformation attempt of Bs 168 by pBK5 with a new procedure : Failure → blaming the plasmid which seems not to be the expected plasmid... Try again with the shuttle vector pHT315 (GFP or not).</LI><LI>The plasmid extraction protocol from <I>B. subtilis</I> was tested once again, with a few modifications. The results were unsatisfying. After electrophoresis, only genomic DNA was observed.</LI></UL></DESCRIPTION><TITRE>Kill</TITRE><DESCRIPTION><UL><LI>Verification of the extracted plasmidic DNA of the clones transformed by [Promoter in pSB1C3] : none of the 4 clones is ok.</LI><LI>Selection of 4 clones transformed by [lysostaphin+dispersin] on LB+Amp plates and in liquid cultures.</LI><LI>Miniprep to extract [Lysostaphin+Cm] and [Dispersin+Cm] → Digestion.</LI><LI>3A ligation between : lysostaphin in pUC57 + Terminator in pSB1K3 + pSB1C3. Verification by electrophoresis : Lysostaphin, terminator and vector are ok but not the ligation (no DNA).</LI><LI>Lysostaphin tests (Bs 168 pWG100 supernatant) on antibiograms (diameter : 6mm) applied on LB+Tet plates having a biofilm already established and in development of <I>S. epidermidis.</I></LI></UL></DESCRIPTION><TITRE>Stick</TITRE><DESCRIPTION>
PCR on the <I>B. subtilis</I> 168 genome using universal primers of Phil Oger, and positive control with <I>Bulkhoderia cepacia</I> genome provided by Phil Oger. Goal : determine if the PCR problems come from the primers or the low concentration of genomic DNA of <I>B. subtilis</I> 168. 
</DESCRIPTION></JOUR><JOUR nb="26"><DATE> Thursday, July 26th 2012</DATE><TITRE>For all purposes</TITRE><DESCRIPTION><UL><LI>Transformation of NM522 strain with pHT304 and pHT315 (2 shuttle vectors for <I>B. subtilis</I> and <I>E. coli</I>).</LI><LI>gDNA extraction with 2 different protocols : the first for <I>Bacillus subtilis</I> 168 and the second for Gram- bacteria with some modifications. The first protocol gives a better yield.</LI><LI>PCR of rRNA 16s on B.s. 168 to test if PCR works in the bacteria without being lysed : it works.</LI><LI>NM522 + <I>gfp</I>, <I>cfp</I> and <I>yfp</I> in storage.</LI><LI>pHT315 GFP put in storage under the reference pBK18.</LI><LI>NM522 + pHT315 GFP strain put in storage under the reference BK21.</LI></UL></DESCRIPTION><TITRE>Kill</TITRE><DESCRIPTION><UL><LI>Other clones transformed with the Constitutive Promoter in pSB1C3 are selected in order to do other verification by extracting their plasmidic DNA (but they are not good again).</LI><LI>Extraction of the plasmidic DNA of the clones transformed with Lysostaphin+Dispersin in pBK2 and verification by electrophoresis gel. The clones are not right.</LI></UL></DESCRIPTION><TITRE>Surfactant</TITRE><DESCRIPTION>
3A assembly RBS-<I>gfp</I> (pBK7-pBK14) in pSB1K3 = L1. Transformation of L1 in NM522.
</DESCRIPTION></JOUR><JOUR nb="27"><DATE> Friday, July 27th 2012</DATE><TITRE>For all purposes</TITRE><DESCRIPTION>
Extraction of transformed clones (NM522/pHT304 and NM522/pHT315). The electrophoresis showed that the plasmids had approximately 7kb  and the restriction sites <I>Xba</I>I, <I>Eco</I>RI and <I>Pst</I>I as expected. However, there is a <I>Spe</I>I site which had not been mapped.
</DESCRIPTION><TITRE>Kill</TITRE><DESCRIPTION><UL><LI>Extraction of plasmidic DNA : Lysostaphin in pSB1C3 clones, Promoter in pSB1C3 clones.</LI><LI>Plasmidic DNA verification by digestion and electrophoresis : Lysostaphin clones okay and Promoter clones are not okay.</LI><LI>New 3A ligation with Lysostaphin/Terminator/pSB1C3 and transformation in NM522 strain.</LI><LI>Results of the lysostaphin tests on plates : no effect (regular biofilms)</LI></UL></DESCRIPTION><TITRE>Surfactant</TITRE><DESCRIPTION><UL><LI><I>sfp</I> (surfactin part 1) and <I>abrB</I> put in storage : pBK16 and pBK17.</LI><LI>Failure of transformation of L1 in NM522.</LI><LI>3A assembly of RBS-<I>cfp</I> (pBK7-pBK13) in pSB1C3 = L2. (!! the part2 construction already has a terminator)</LI><LI>3A assembly of <I>sfp</I>-part2(<I>lacI</I>)-terminator (pBK4-pBK10) in pSB1C3 = IV.</LI></UL></DESCRIPTION><TITRE>Stick</TITRE><DESCRIPTION><UL><LI>PCR of <I>xylR</I> from gDNA extracted yesterday. Test on gel : successful PCR !!</LI><LI>3A ligation of RBS (pBK7) and <I>xylR</I> (produced by PCR) in pSB1C3 ( ! the vector plasmid has the same resistance as the plasmid containing the RBS...).</LI></UL></DESCRIPTION></JOUR><JOUR nb="30"><DATE> Monday, July 30th 2012</DATE><TITRE>For all purposes</TITRE><DESCRIPTION><UL><LI>Meeting at 9 o’clock.</LI><LI><STRONG>It was also a fire day : a man’s hair and a lab bench on fire !!!! As the saying goes, ”everything comes in threes, if it's happened twice, it'll happen a third time ” (we’re still waiting for the third fire…).</STRONG></LI></UL></DESCRIPTION><TITRE>Kill</TITRE><DESCRIPTION><UL><LI>Digestion test and gel of pBK2 (promoter + lysostaphin) by <I>Eco</I>RI and <I>Spe</I>I, pBK3 (dispersin + terminator) by <I>Xba</I>I and <I>Pst</I>I and pSB1C3 by <I>Eco</I>RI and <I>Pst</I>I.</LI><LI>We got 3 clones for the transformation of NM522 bacteria with the ligation’s product Lysostaphin+Terminator-pSB1C3 : the plasmidic DNA of these 3 clones is extracted and tested by electrophoresis gel → the 3 clones aren’t right.</LI><LI>Lysostaphin test on plate : this time, by trying with biofilm in formation less dense (OD<SUB>600</SUB> = 0.5,  dilution the culture x100, 1000 and 10 000).</LI></UL></DESCRIPTION><TITRE>Surfactant</TITRE><DESCRIPTION><UL><LI>Transformation of L1, L2 and IV in NM522.</LI><LI>Transformation of <I>sfp</I> and <I>abrB</I> in NM522.</LI></UL></DESCRIPTION><TITRE>Stick</TITRE><DESCRIPTION>
Purification of <I>xylR</I>, produced by PCR.
</DESCRIPTION></JOUR><JOUR nb="31"><DATE> Tuesday, July 31st 2012</DATE><TITRE>Kill</TITRE><DESCRIPTION>
Results of the lysostaphin tests on plates : negative. There is no biofilm : too diluted but even though, the colonies do not seem to be affected by the presence of Bs 168 pWG100 supernatant → have the Bs 168 pWG100 lost their plasmid? (cultivated without selection pressure, that is without erythromycin).
</DESCRIPTION><TITRE>Surfactant</TITRE><DESCRIPTION><UL><LI>Given the fact that the previous transformation didn’t work, we made another transformation of NM522 strain with the <I>abrB</I> and <I><I>sfp</I></I> parts.</LI><LI>Results of the transformations of L1, L2 and IV : ok. Selection of some clones in order to do minipreps.</LI></UL></DESCRIPTION><TITRE>Stick</TITRE><DESCRIPTION>
Ligation of RBS and <I>xylR</I> and transformation in NM522 strain.
</DESCRIPTION></JOUR></MONTH><MONTH name="August" size="31"><JOUR nb="1"><TITRE>For all purposes</TITRE><DATE> Wednesday, August 1st 2012</DATE><DESCRIPTION><P>The transformation protocol for <I>Bacillus</I> was tested using the pHT315 GFP plasmid extracted from the <I>B. thuringiensis</I> BT407GFP strain.</P></DESCRIPTION><TITRE>Kill</TITRE><DESCRIPTION><UL><LI>Transformation results :
       </LI><LI>Lysostaphin+terminator : nothing;</LI><LI>Lysostaphin+dispersin : a lot of clones.</LI></UL>
       Cultures in LB broth of Lysostaphin+Dispersin are launched.<LI>Digestion of Promoter clones and lysostaphin clones followed by an electrophoresis : lysostaphin clones are okay, promoter clones are not okay.</LI><LI>pUC57 with Dispersin put in storage : pBK3.</LI><LI>Lysostaphin test returns ! This time, we used more <I>S. epidermidis</I> (OD<SUB>600</SUB> = 0.75 diluted 1000 times) and  a control for the stability of pWG100 plasmid (spreading of 200 µL on LB + Ery plates), after the liquid culture without selective pressure. Spreading on LB plates without Tetracycline (but addition of Tetracycline into the Bs 168 pWG100 supernatant).</LI></DESCRIPTION><TITRE>Surfactant</TITRE><DESCRIPTION><UL><LI>The strain NM522/pBK6 was put in storage under the reference BK22.</LI><LI>Transformations of the NM522 strain with the ordered constructions (<I>sfp</I> and <I>abrB</I> in pUC57 AmpR). The transformation was unsuccessful, so we did it again.</LI><LI>Quantification of <I>sfp</I> and <I>abrB</I> constructions provided by Genecust using the Nanodrop.</LI><LI>Miniprep of the L1, L2 and IV ligations : it didn’t work.</LI></UL></DESCRIPTION></JOUR><JOUR nb="2"><TITRE>For all purposes</TITRE><DATE> Thursday, August 2nd 2012</DATE><DESCRIPTION><UL><LI>The transformation of the <I>Bacillus</I> strain failed because of contaminated LB broth, so a new transformation was attempted.</LI><LI>pHT304 and pHT315 plasmids supernumerary site <I>Spe</I>I deletion attempt by digestion, filling-in, ligation and transformation in NM522.</LI></UL></DESCRIPTION><TITRE>Kill</TITRE><DESCRIPTION><UL><LI>Extraction of plasmidic DNA from the Lysostaphin + Dispersin clones in Cm iGEM plasmid (pSB1C3) and from the BK12 strain clones. Plasmidic DNA is checked by digestions. Lysostaphin+Dispersin clones digestion : clones not okay.</LI><LI>Ligations of :
       </LI><LI>Constitutive promoter in Cm iGEM plasmid;</LI><LI>Dispersin in Cm iGEM plasmid.</LI></UL>
       Ligation were verified by electrophoresis.<LI>Results of the Lysostaphin tests : some Bs 168 pWG100 contaminated the plate. However, we noticed a halo of 2-3 mm around the <I>Bacillus</I> colonies  where <I>S. epidermidis</I> didn’t grow due to lysostaphin activity.</LI><LI>New Lysostaphin test on LB+Tet plate with a negative control (10 µL of Ampicillin).</LI></DESCRIPTION><TITRE>Stick</TITRE><DESCRIPTION><UL><LI>Miniprep of 5 clones of the transformed NM522 strain with RBS-xylR and 2 clones of the NM522 transformed with the <I>abrB</I> construction. Given the fact that the NM522 strain used for the transformations was contaminated, the electrophoresis showed unexpected results.</LI><LI>New ligation of RBS and <I>xylR</I> in pSB1A3 and transformation in NM522 strain.</LI></UL></DESCRIPTION><TITRE>Surfactant</TITRE><DESCRIPTION>
Transformation of <I>sfp</I> and <I>abrB</I> in NM522 strain didn’t work.
</DESCRIPTION></JOUR><JOUR nb="3"><DATE> Friday, August 3rd 2012</DATE><TITRE>For all purposes</TITRE><DESCRIPTION><UL><LI>Meeting at 9 o’clock.</LI><LI>Purification of the NM522 strain (because of the problems on the negative control during the transformation) and resistance tests on the purified clones.</LI><LI>The results of <I>Bacillus</I> transformation are inconclusive : there are only a few clones on the plate. However, the negative control has a few colonies too. Despite this result, we decided to verify six clones and extract their DNA.</LI><LI>A lot of clones for the transformation of the pHT304 and pHT315 plasmids (without <I>Spe</I>I site) on LB+Amp plates → Purification of 12 clones each.</LI></UL></DESCRIPTION><TITRE>Kill</TITRE><DESCRIPTION><UL><LI>Velvet replication of Lysostaphin+Dispersin in pSB1C3 transformation plate on LB + Amp.</LI><LI>Antibiotic resistance checked for 5 Lysostaphin+Dispersin in iGEM plasmid clones (clones 1 to 5).</LI><LI>BK12 strain verification : spreading on LB + Amp plate.</LI><LI>Ligation of Promoter+Dispersin in Cm iGEM plasmid followed by transformation of the 3 ligations.</LI><LI>Ligation were verified by electrophoresis.</LI><LI>Transformation of Promoter+Dispersin ligation.</LI><LI>PCR to check the presence of the promoter in the Promoter in Cm iGEM plasmid clones.</LI><LI>Preliminary test of the liquid lysostaphin (10 µL of tetracycline to kill <I>Bacillus</I>, 500 µL of Bs 168 pWG100 supernatant, 500 µL of <I>S. epidermidis</I> with OD<SUB>600</SUB>=1.2 (the strains are cultivated overnight at 30°C). This test shows a possible effect of lysostaphin (decrease of OD<SUB>600</SUB> with the time, during 3h30). To be repeated with a negative control without Bs 168 pWG100.</LI></UL></DESCRIPTION><TITRE>Surfactant</TITRE><DESCRIPTION><UL><LI>Ligation (3A protocol) of pBK7, pBK13 and iGEM backbone pSB1K3 (=RBS+<I>cfp</I>+pSB1K3).</LI><LI>Ligation (3A protocol) of pBK7, pBK14 and iGEM backbone pSB1K3 (=RBS+<I>gfp</I>+pSB1K3).</LI><LI>Because of the difficulties to transform the bacteria with the plasmids containing <I>sfp</I> and <I>abrB</I>, we decided to do an electrophoresis directly on the constructions sent by Genecust. Result : the genes were not inserted in a pUC57 plasmid, although they should have been cloned.</LI><LI>Ligation of <I>abrB</I> and <I>sfp</I> in pSB1K3 and transformation in NM522.</LI></UL></DESCRIPTION><TITRE>Stick</TITRE><DESCRIPTION>
New ligation between RBS and <I>xylR</I> in pSB1A3 and transformation in NM522 strain.
</DESCRIPTION></JOUR><JOUR nb="4"><DATE> Saturday, August 4th 2012</DATE><TITRE>Kill</TITRE><DESCRIPTION><UL><LI>Transformation results :
       </LI><LI>The negative control is ok;</LI><LI>There are clones on every transformation plates (Promoter in pSB1C3, Dispersin in pSB1C3, Promoter+Dispersin in pSB1C3).</LI></UL>
       8 clones for each transformation are chosen and spread on LB+Cm and LB+Amp plates.</DESCRIPTION><TITRE>Surfactant</TITRE><DESCRIPTION><UL><LI>Transformations of <I>abrB</I> and <I>sfp</I> are a success !! :D</LI><LI>Liquid cultures of <I>abrB</I> and <I>sfp</I> clones are launched to do plasmid extractions.</LI></UL></DESCRIPTION><TITRE>Stick</TITRE><DESCRIPTION>
Sorting of the RBS + <I>xylR</I> clones to eliminate the clones containing two relegated plasmids.
</DESCRIPTION></JOUR><JOUR nb="5"><DATE> Sunday, August 5th 2012</DATE><TITRE>For all purposes</TITRE><DESCRIPTION>
Liquid culture of 6 NM522 clones with pHT304 S and 6 clones pHT315 S.
</DESCRIPTION><TITRE>Surfactant</TITRE><DESCRIPTION>
Plasmid extractions from 3 clones NM522/<I>abrB</I> and 5 clones NM522/sfp. The electrophoresis of the digested plasmids with EcoRI and PstI confirmed the presence of <I>sfp</I> construction (at 800 bp) and <I>abrB</I> construction  (at 500 bp).
</DESCRIPTION><TITRE>Stick</TITRE><DESCRIPTION>
Liquid cultures of RBS+<I>xylR</I> clones are launched to do plasmid extractions.
</DESCRIPTION></JOUR><JOUR nb="6"><DATE> Monday, August 6th 2012</DATE><TITRE>For all purposes</TITRE><DESCRIPTION><UL><LI>Plasmid extraction from 6 clones Bs 168 transformed by pH315 GFP : the electrophoresis of the digested plasmids showed that the clones had the right plasmid. The <I>Bacillus</I> transformation also worked, but the yield must be improved.</LI><LI>Miniprep and digestions by <I>Spe</I>I of the plasmid extracted from the 6 clones of NM522 with pHT304 S and NM522 with pHT315 S : Failure ! The <I>Spe</I>I site is still in the plasmids :’( </LI></UL></DESCRIPTION><TITRE>Kill</TITRE><DESCRIPTION><UL><LI>Selection of clones growing on LB+Cm plates and not on LB+Amp plates :
       </LI><LI>4 clones Dispersin in pSB1C3;</LI><LI>7 clones Promoter pSB1C3;</LI><LI>5 clones Promoter+Dispersin in pSB1C3.</LI></UL>
       Liquid cultures in LB+Cm are made with these clones. Then extraction of plasmidic DNA.<LI>8 clones Lysostaphin + Dispersin in pSB1C3 are spread on LB+Cm and LB+Amp to test their resistance.</LI><LI>Transformation of pSB1C3 and pSB1T3 in order to obtain a clone from the PCR matrix (red clone).</LI></DESCRIPTION><TITRE>Surfactant</TITRE><DESCRIPTION><UL><LI>Transformation of the following ligations, in NM522 strain :</LI><UL><LI>pBK7+pBK13+pSB1K3 (RBS+<I>cfp</I> in Kanamycin resistant backbone);</LI><LI>pBK7+pBK14+pSB1K3 (RBS+<I>gfp</I> in Kanamycin resistant backbone).</LI></UL><LI>PCR of RBS-<I>abrB</I> and P<SUB>xyl</SUB> and purification of these PCR products.</LI></UL></DESCRIPTION><TITRE>Stick</TITRE><DESCRIPTION><UL><LI>Purification of the <I>xylR</I> gene.</LI><LI>Miniprep of RBS-xylR and electrophoresis test → the ligation failed again...</LI></UL></DESCRIPTION></JOUR><JOUR nb="7"><DATE>Tuesday, August 7th 2012</DATE><TITRE>For all purposes</TITRE><DESCRIPTION><UL><LI>There is a little plasmid pHT304 S and pHT315 S post filling-in, new digestion by Spel enzyme to eliminate the plasmids which are not full on the SpeI site in order to increase the amount of plasmids without <I>Spe</I>I site.</LI><LI>Transformation in NM522 and spreading on LB+Amp plates.</LI></UL></DESCRIPTION><TITRE>Kill</TITRE><DESCRIPTION><UL><LI>8 new clones Lysostaphin+Dispersin in pSB1C3 are screened on Ampicillin. Ampicillin sensitive clones are put in liquid culture.</LI><LI>Verification of plasmids extracted the previous day and electrophoresis : Dispersin in pSB1C3 (clones not okay), Promoter+Dispersin in pSB1C3 (Clones not okay).</LI><LI>The transformation of pSB1C3 and pSB1T3 didn’t work.</LI><LI>New Lysostaphin liquid test with 3 measurements and a negative control (Bs 168 supernatant instead of Bs 168 pWG 100). This time, the cultures grew over 36 hours at 37°C (efficiency of the lysostaphin in stationary phase ?) → the test is positive, lysostaphin has a right effect on the <I>S. epidermidis</I> cultures.</LI></UL></DESCRIPTION><TITRE>Surfactant</TITRE><DESCRIPTION><UL><LI>3A ligation : [P<SUB>xyl</SUB>-RBS-<I>sfp</I>] + [RBS-<I>abrB</I>] + [psB1T3].</LI><LI>Transformation of the ligation [P<SUB>xyl</SUB>-RBS-<I>sfp</I>] + [RBS-<I>abrB</I>] + pSB1T3 in NM522 strain and spreading LB+Tet plates.</LI><LI>Results of the transformation by pBK7+pBK13+pSB1K3 and pBK7+pBK14+pSB1K3 : Failure! The negative control has a few colonies : the LB+Kan plate was contaminated and the used LB medium too! The transformation is done again...</LI><LI>Purification of P<SUB>xyl</SUB> produced by PCR.</LI></UL></DESCRIPTION></JOUR><JOUR nb="8"><DATE>Wednesday, August 8th 2012</DATE><TITRE>For all purposes</TITRE><DESCRIPTION><UL><LI>Result of the transformation of NM522 by pHT304 S and pHT315 S : 0 clone and 1 clone... this only clone is tested but the test is negative : the <I>Spe</I>I site is always here at 311 bp. The cultures of the transformation are concentrated and spread on LB plate : 0 clone for pHT304 S and 18 for pHT315 S but none of the clones is good. We must do the filling-in again...</LI><LI>Transformation of the pSB1C3 + RFP and pSB1T3 + RFP plasmids extracts of the iGEM plates to use them as tool of cloning after extraction.</LI></UL></DESCRIPTION><TITRE>Kill</TITRE><DESCRIPTION><UL><LI>Plasmidic extraction of Lysostaphin+Dispersin clones, then digestion and electrophoresis : clones not okay.</LI><LI>New ligations : 
       </LI><LI>Promoter+Dispersin in pSB1C3;</LI><LI>Lysostaphin+Dispersin in pSB1C3.</LI></UL>
       Then transformation in NM522.<LI>Plasmidic extraction of the BK12 strain : plasmid okay.</LI></DESCRIPTION><TITRE>Surfactant</TITRE><DESCRIPTION>
Results of the transformations :
      <UL><LI>Transformations of NM522 by pBK7+pBK13+pSB1K3 and pBK7+pBK14+pSB1K3 : ok.</LI><LI>Transformation by <I>sfp</I>-<I>abrB</I>-pSB1T3 : the negative control isn’t good, the transformation is done again...</LI></UL><LI>Miniprep of other clones transformed with the plasmid RBS + xylR : the electrophoresis shows that the ligation still isn’t good…</LI></DESCRIPTION></JOUR><JOUR nb="9"><DATE>Thursday, August 9th 2012</DATE><TITRE>For all purposes</TITRE><DESCRIPTION><UL><LI>Extraction of NM522/pHT315S. We expected a plasmid without a SpeI site. However, the selected clone contained a plasmid identical to pHT315 (the digestion by <I>Spe</I>I and <I>Eco</I>RI enzymes gives a fragment at 1000 bp). We decided to screen more clones (8 out of 17).</LI><LI>The transformation pSB1C3 + RFP and pSB1T3 + RFP is a success ! (red clones)</LI></UL></DESCRIPTION><TITRE>Kill</TITRE><DESCRIPTION>
Transformation results : 
<UL><LI>Positive control : okay;</LI><LI>Negative control : okay;</LI><LI>[Lysostaphin + Dispersin] : 8 clones;</LI><LI>[Promoter + Dispersin]  : more than 30 clones.</LI></UL>
Clones are screened on LB + Amp and LB + Cm plates.
</DESCRIPTION><TITRE>Surfactant</TITRE><DESCRIPTION><UL><LI>The transformation of NM522 strain by <I>sfp</I>-<I>abrB</I> didn’t work again : although the positive control contains many clones (plasmid A2) and the negative control contains nothing, the real transformation contains nothing either ! We decided to do the <I>sfp</I>-<I>abrB</I> ligation again...</LI><LI>Plasmid extraction of 4 clones NM522 transformed by pBK7+pBK13+pSB1K3 and pBK7+pBK14+pSB1K3 : the electrophoresis of the digested plasmids showed that 3 out of 4 clones had the good fragment.</LI><LI><STRONG>PROBLEM : WE RAN OUT OF LIGASE!!</STRONG></LI></UL></DESCRIPTION></JOUR><JOUR nb="10"><DATE>Friday, August 10th 2012</DATE><TITRE>For all purposes</TITRE><DESCRIPTION><UL><LI>Deletion of the <I>Spe</I>I site from pHT315 and pHT304 plasmids and filling-in using the <I>Pfu</I> polymerase. This time we used gel electrophoresis to verify the plasmids after each purification.</LI><LI>Extraction of the plasmid containing the RFP gene in the pSB1C3 iGEM backbone (midiprep).</LI></UL></DESCRIPTION><TITRE>Kill</TITRE><DESCRIPTION><UL><LI>New Lysostaphin tests are planned (on plates) : in this purpose, a culture of 60 mL of Bs 168 pWG100 is inoculated and then incubated overnight at 28°C.</LI><LI>None of the clones Lysostaphin+Dispersin grew in LB + Amp so LB cultures were inoculated.</LI><LI>Out of 29 clones, 18 Promoter+Dispersin clones are Ampicillin sensitive, LB cultures are launched with these clones.</LI><LI>Plasmidic extractions of these clones : clones not okay.</LI><LI>PCR to obtain pSB1C3 : the electrophoresis showed that there was no DNA, so another trial was done, but it was unsuccessful.</LI><LI>The strain NM522 with Constitutive Promoter + pSB1C3 was put in storage under the reference BK27.</LI></UL></DESCRIPTION><TITRE>Surfactant</TITRE><DESCRIPTION><UL><LI>Ligation of the <I>sfp</I> and <I>abrB</I> parts in an iGEM backbone.</LI><LI>The NM522 strain with <I>abrB</I>-pSB1K3 was put in storage under the reference BK25.</LI><LI>The NM522 strain with <I>sfp</I>-pSB1K3 was put in storage under the reference BK26.</LI></UL></DESCRIPTION></JOUR><JOUR nb="11"><DATE>Saturday, August 11th 2012</DATE><TITRE>For all purposes</TITRE><DESCRIPTION>
Gel electrophoresis showed that the elimination of the <I>Spe</I>I site was not successful.
</DESCRIPTION></JOUR><JOUR nb="12"><DATE>Sunday, August 12th 2012</DATE><TITRE>Kill</TITRE><DESCRIPTION>
Culture of 60 mL of Bs 168 pWG 100 at 28°C for the lysostaphin tests on plates.
</DESCRIPTION></JOUR><JOUR nb="13"><DATE>Monday, August 13th 2012</DATE><TITRE>For all purposes</TITRE><DESCRIPTION><UL><LI>We identified the reason why the deletion of the site <I>Spe</I>I from pHT315 and pHT304 plasmids and filling-in was not successful : the <I>Pfu</I> enzyme requires Mg<SUP>2+</SUP> and the buffer we used did not contain any, so we decided to give it another try, this time with the proper buffer.</LI><LI>The extracted plasmids containing the RFP gene in the pSB1C3 backbone were digested and verified by electrophoresis.</LI></UL></DESCRIPTION><TITRE>Kill</TITRE><DESCRIPTION><UL><LI>Lysostaphin tests : Flasks of 25mL of Bs 168 pWG 100 filtered supernatant were cultivated on Friday and frozen at -20°C on Sunday (2 flasks of each). The control flask is filled with 25mL of LB broth.  These flasks are then frozen at -80°C and freeze-dried overnight.</LI><LI>The gel electrophoresis of the digested plasmid [Lysostaphin + Dispersin] in pSB1C3 doesn’t correspond to the expected fragments. Similarly, the second electrophoresis shows that the clones with [Constitutive promoter + Dispersin] in pSB1C3 do not have the right plasmid : the digested fragments do not have  the right size.</LI></UL></DESCRIPTION><TITRE>Surfactant</TITRE><DESCRIPTION>
Transformation of the ligation [<I>sfp</I> + <I>abrB</I> + pSB1A3] in the NM522 strain.
</DESCRIPTION></JOUR><JOUR nb="14"><DATE>Tuesday, August 14th 2012</DATE><TITRE>For all purposes</TITRE><DESCRIPTION>
The gel electrophoresis of the digested plasmids pHT315 and pHT304 after the final purification shows promising results, so the two plasmids were transformed in the NM522 strain.
</DESCRIPTION><TITRE>Kill</TITRE><DESCRIPTION><UL><LI>Ligation of the Constitutive Promoter (extracted by PCR) in the pSB1C3 plasmid containing the RFP gene (purified by midiprep) in order to obtain the Constitutive Promoter in the pSB1C3.</LI><LI>Lysostaphin tests on plates return again ! The freeze-dried products are put in five times less water than at first. Volumes up to 100 µL of concentrated supernatant are applied on antibiograms which are used for tests on the <I>S. epidermidis</I> biofilm.</LI></UL></DESCRIPTION><TITRE>Surfactant</TITRE><DESCRIPTION>
The transformation of the ligation [<I>sfp</I>+<I>abrB</I>+pSB1A3] was successful, so 4 clones were put in liquid culture in order to be tested.
</DESCRIPTION><TITRE>Stick</TITRE><DESCRIPTION>
Standard ligation between pBK7 (=RBS in pSB1C3) and xylR (obtained by PCR) with 1µL of insert for 5µL of vector. The ligation was transformed into the NM522 strain. 
</DESCRIPTION></JOUR><JOUR nb="15"><DATE>Wednesday, August 15th 2012</DATE><TITRE>For all purposes</TITRE><DESCRIPTION>
The transformation of the pBK20 (pHT315) and pBK19 (pHT304) plasmids was successful. 12 clones per plasmid were tested.
</DESCRIPTION><TITRE>Kill</TITRE><DESCRIPTION><UL><LI>The lysostaphin test didn’t work : Valérie did the test again by applying first the supernatant on the paper disks before putting them on the plates, and by using TSM medium instead of LB broth.</LI><LI>Transformation of the ligation [pSB1C3 + constitutive promoter] in NM 522 strain.</LI></UL></DESCRIPTION><TITRE>Surfactant</TITRE><DESCRIPTION>
None of the 4 clones tested had the expected fragments, so we screened 6 others.
</DESCRIPTION><TITRE>Stick</TITRE><DESCRIPTION><UL><LI>Results of the transformation of NM522 by pBK7 and <I>xylR</I> : the negative control contains nothing, the positive control (transformed with pBK5) is full of bacteria and the plate with NM522 transformed by pBK7 + <I>xylR</I> contains a lot of clones too. 8 clones are selected to do liquid culture and extract their DNA.</LI><LI>Second standard ligation between pBK7 (=RBS in pSB1C3) and <I>xylR</I> (produced by PCR) with more insert (3µL of insert for 2µL of vector). Transformation into NM522 strains. </LI></UL></DESCRIPTION></JOUR><JOUR nb="16"><DATE>Thursday, August 16th 2012</DATE><TITRE>For all purposes</TITRE><DESCRIPTION><UL><LI>Miniprep of the clones having integrated the modified  shuttle vector (after filling-in). 4 clones seem to have the right plasmid.</LI><LI>The electrophoresis after digestion with <I>Eco</I>RI and <I>Spe</I>I showed that the <I>Spe</I>I site was eliminated. 4 clones were chosen for further tests. However, the DNA concentration was too low so the enzymes did not digest the plasmids.</LI></UL></DESCRIPTION><TITRE>Kill</TITRE><DESCRIPTION><UL><LI>Ligation of the Constitutive promoter (P<SUB>veg</SUB>) produced by PCR in an iGEM plasmid.</LI><LI>Result of the lysostaphin tests : didn’t work again. The Bs 168 pWG100 supernatant doesn’t contain any lysostaphin : the lysostaphin is probably degraded in one way or another (sensitivity to defrosting ?).</LI><LI>Result of the transformation with [pSB1C3 + promoter] : a lot of red clones, some white clones (20 - 30). Isolation of 6 white clones and plasmid extractions by miniprep.</LI></UL></DESCRIPTION><TITRE>Surfactant</TITRE><DESCRIPTION><UL><LI>Miniprep of 4 clones with the transformed 3A ligation (pBK22, gene <I>abrB</I> and pSB1A3 backbone). The electrophoresis did not turn out as expected for any of the 4 clones, so we decided to screen 6 others.</LI><LI>Ligation of P<SUB>xyl</SUB> produced by PCR in an iGEM plasmid.</LI></UL></DESCRIPTION><TITRE>Stick</TITRE><DESCRIPTION><UL><LI>Ligation of <I>xylR</I> produced by PCR in an iGEM plasmid (pSB1K3).</LI><LI>Results of the transformation of NM522 by [pBK7 + <I>xylR</I>] (for the second ligation) : the plate with bacteria transformed by [pBK7 + <I>xylR</I>] contains a lot of clones and the controls are standard. 14 clones are selected to do liquid culture and extract their DNA.</LI><LI>Plasmid extractions from the 8 clones transformed by [pBK7 + <I>xylR</I>]. The electrophoresis of the digested plasmids showed that the clones do not have the right plasmid : they have just the vector without <I>xylR</I>.</LI></UL></DESCRIPTION></JOUR><JOUR nb="17"><DATE>Friday, August 17th 2012</DATE><TITRE>For all purposes</TITRE><DESCRIPTION>
The plasmids (pBK19 and pBK20) from 2 clones were re-extracted using columns. This time, the digestion was successful. The tests showed that there was no <I>SpeI</I> site. However, the pHT315 plasmid also had no XbaI site. We digested the two plasmids with the XbaI enzyme and this time we incubated the digestion during one hour, instead of 10 minutes. We also tested the enzyme on another plasmid which had a <I>XbaI</I> site. The digestion of the control plasmid was partial.
</DESCRIPTION><TITRE>Surfactant</TITRE><DESCRIPTION>
Miniprep of 6 clones with the transformed 3A ligation (pBK22, gene <I>abrB</I> and pSB1A3 backbone). The gel verification showed that there were 3 clones having the expected profile. The plasmid was put in storage under the reference pBK29.
</DESCRIPTION><TITRE>Stick</TITRE><DESCRIPTION>
Plasmid extractions from the 14 clones transformed by [pBK7 + <I>xylR</I>] (with the second ligation). As for the first transformation, the electrophoresis of the digested plasmids showed that the clones do not have the right plasmid : they only have the vector without <I>xylR</I>.
</DESCRIPTION></JOUR><JOUR nb="20"><DATE>Monday, August 20th 2012</DATE><TITRE>For all purposes</TITRE><DESCRIPTION><UL><LI>pBK19 with no SpeI site was put in storage under the reference pBK25.</LI><LI>pBK20 with no SpeI site was put in storage under the reference pBK26.</LI></UL></DESCRIPTION><TITRE>Kill</TITRE><DESCRIPTION><UL><LI>Launch of 8 [Promoter+Dispersin] cultures.</LI><LI>Results of the transformation of NM522 by [pSB1T3 + P<SUB>xyl</SUB>] : the plate with bacteria transformed by [pSB1T3 + P<SUB>xyl</SUB>] contains few clones and the controls are standard. 3 clones are selected to do liquid cultures and extract their DNA : no plasmid.</LI><LI>Results of the transformation of NM522 by [pSB1T3 + RBS-<I>abrB</I>] : no clones.</LI></UL></DESCRIPTION><TITRE>Stick</TITRE><DESCRIPTION><UL><LI>Results of the transformation of NM522 by [pSB1K3 + <I>xylR</I>] : the plate with bacteria transformed by [pSB1K3 + <I>xylR</I>] contains a lot of clones and the controls are standard. 3 clones are selected to do liquid culture and extract their DNA : they have just the vector without <I>xylR</I>.</LI></UL></DESCRIPTION>A new PCR of <I>xylR</I> is made, in order to increase the stock.</JOUR></MONTH><JOUR nb="21"><DATE>Tuesday, August 21st 2012</DATE><TITRE>For all purposes</TITRE><DESCRIPTION><UL><LI>Meeting at 9 o’clock.</LI><LI>Midiprep of the modified pBK25 and pBK26 shuttle plasmids (without the SpeI site). The extracted DNA was verified by electrophoresis.</LI></UL></DESCRIPTION><TITRE>Kill</TITRE><DESCRIPTION><UL><LI>Standard ligation of pBK23 (= Constitutive Promoter + RBS + Lysostaphin in pSB1C3) with pBK25 (the shuttle vector <I>E. coli</I> - <I>B. subtilis</I>). Different ligations are made with different proportions of insert and vector. Transformation in NM522 strain.</LI><LI>Digestion of Lysostaphin in pSB1C3 and Dispersin in pUC57.</LI><LI>Verification of 8 [promoter-dispersin] clones : clones happen to be nonstandard.</LI></UL></DESCRIPTION><TITRE>Surfactant</TITRE><DESCRIPTION><UL><LI>Extraction of the plasmid containing the ligation [<I>sfp</I>+<I>abrB</I>+pSB1A3] from a liquid culture of transformed bacteria.</LI><LI>Results of the second transformation of NM522 by pSB1T3 and RBS-<I>abrB</I> : no clones.</LI></UL></DESCRIPTION><TITRE>Stick</TITRE><DESCRIPTION><UL><LI>Two standard ligations are done :</LI><UL><LI>pBK7-<I>xylR</I> (<I>xylR</I> cut in X and P sites);</LI><LI>pBK24-<I>xylR</I> (<I>xylR</I> cut in E and S sites).</LI></UL><LI>Transformation of pBK7-<I>xylR</I> into NM522 strain.</LI></UL></DESCRIPTION></JOUR><JOUR nb="22"><DATE>Wednesday, August 22nd 2012</DATE><TITRE>Kill</TITRE><DESCRIPTION><UL><LI>Result of the transformation of NM522 with [Constitutive Promoter + RBS + Lysostaphin] in pBK25 : all the controls are right, and there are a lot of clones on the different plates with the transformed bacteria. 14 clones are selected to do liquid cultures and extract their DNA.</LI></UL></DESCRIPTION>New trial of cloning : Digestion, Ligation, transformation to construct :<UL><LI>[Lysostaphin + Dispersin] in pSB1C3;</LI><LI>[Lysostaphin + Dispersin] in the shuttle vector (vector for <I>E. coli</I> and <I>B. subtilis,</I>).</LI></UL></JOUR><TITRE>Surfactant</TITRE><DESCRIPTION>
6 P<SUB>xyl</SUB> clones were tested, but none had integrated the plasmid.
</DESCRIPTION><TITRE>Stick</TITRE><DESCRIPTION><UL><LI>Verification of 6 [<I>xylR</I>-pSB1K3] clones : they only have the vector.</LI><LI>Transformation of [pBK24-<I>xylR</I>] into NM522 strain.</LI><LI>Results of [pBK7-<I>xylR</I>] transformation : lots of clones and no clones in the negative control. 12 clones are put in liquid culture for extraction.</LI></UL></DESCRIPTION><TITRE>Physiological tests</TITRE><DESCRIPTION><UL><LI>Liquid culture of <I>B. subtilis</I> 168 in 5mL of LB broth.</LI><LI>Liquid culture of <I>B. subtilis</I><I>abrB</I> in 5mL of LB broth.</LI></UL></DESCRIPTION></PROJECT><JOUR nb="23"><DATE>Thursday, August 23rd 2012</DATE><TITRE>For all purposes</TITRE><DESCRIPTION>
Cloning of the iGEM linker into the shuttle vector (pBK25 and pBK26) and transformation in the NM522 strain.
</DESCRIPTION><TITRE>Kill</TITRE><DESCRIPTION><UL><LI>Transformations results : All controls are okay.</LI><LI>Too many clones on [Lysostaphin + Dispersin] in shuttle vector transformation plate, so some clones are selected and spread and a new LB plate with the appropriate antibiotic.</LI><LI>12 liquid culture in LB are launched for the clones [Lysostaphin + Dispersin] in pSB1C3.</LI><LI>Miniprep of 14 clones transformed with the ligation product [Constitutive Promoter + RBS + Lysostaphin] in pBK25 and electrophoresis to analyze the extracted DNA : the transformation is successful for 3 clones =)</LI><UL><LI>The first clone [Constitutive Promoter + RBS + Lysostaphin] in pBK25 is put in storage under the reference pBK28 (1);</LI><LI>The second clone [Constitutive Promoter + RBS + Lysostaphin] in pBK25 is put in storage under the reference pBK28 (2);</LI><LI>The third clone [Constitutive Promoter + RBS + Lysostaphin] in pBK25 is put in storage under the reference pBK28 (3).</LI></UL></UL></DESCRIPTION><TITRE>Surfactant</TITRE><DESCRIPTION>
Transformation of [P<SUB>xyl</SUB>+pSB1T3] in NM522.
</DESCRIPTION><TITRE>Stick</TITRE><DESCRIPTION><UL><LI>Results of [pBK24-<I>xylR</I>] transformation : most of the clones are red, however a few of them are white, which is good. 7 white clones are put in liquid culture for extraction.</LI><LI>Extraction of pBK7-<I>xylR</I> from 12 clones, and then a gel electrophoresis is run: 2 clones seem interesting. We ran a second gel to verify these 2 clones, but it showed us that they were not good.</LI></UL></DESCRIPTION><TITRE>Physiological tests</TITRE><DESCRIPTION>
A microtiter plate is inoculated with <I>B. subtilis</I> 168 and <I>B. subtilis</I><I>abrB</I> in order to compare the adherence of each strain. (see Protocol 'Tests of Bacillus subtilis adherence in 24 wells plate').
</DESCRIPTION></JOUR><JOUR nb="24"><DATE>Friday, August 24th 2012</DATE><TITRE>For all purposes</TITRE><DESCRIPTION>
The transformation of the cloned shuttle vectors was successful, so 12 clones are put in liquid cultures for further tests.
</DESCRIPTION><TITRE>Surfactant</TITRE><DESCRIPTION>
3A ligation of pBK4 (pUC57 containing the <I>lacI</I> gene) and pBK29 (pSB1A3 containing the <I>sfp</I> and <I>abrB</I> genes) in pBK24 (pSB1C3 containing a RFP gene). The cloned fragment was transformed in the NM522 strain.
</DESCRIPTION><TITRE>Stick</TITRE><DESCRIPTION><UL><LI>After the bad results with <I>xylR</I>, we decided to cut pBK10 plasmid in <I>Sma</I>I site, and ligate with <I>xylR</I>, doing a blunt ligation. We also decided to ligate <I>xylR</I> with <I>xylR</I>, creating a big polymer, which will be like a “pre-ligation” molecule.</LI><LI>Extraction of pBK24-<I>xylR</I> from 7 clones. Gel electrophoresis is run → ONE CLONE IS GOOD!! Meaning that it’s the first time we manage to ligate <I>xylR</I> successfully.</LI></UL></DESCRIPTION><TITRE>Physiological tests</TITRE><DESCRIPTION>
The microtiter plate prepared the day before is analyzed. There is a significant difference between the adherence potential of the two strains. <I>B. subtilis</I><I>abrB</I> strain forms thin layer biofilms while <I>B. subtilis</I> 168 doesn't form any specific kind of biofilms.
</DESCRIPTION></JOUR><JOUR nb="25"><DATE>Saturday, August 25th 2012</DATE><TITRE>For all purposes</TITRE><DESCRIPTION>
Miniprep of 12 clones selected from transformed bacteria with the cloned shuttle vectors; we ran out of <I>Spe</I>I enzyme, so we could not digest the extracted plasmids.
</DESCRIPTION><TITRE>Stick</TITRE><DESCRIPTION>
6 white clones from the ligation pBK24-<I>xylR</I> are purified and put in liquid cultures.
</DESCRIPTION></JOUR><JOUR nb="26"><DATE>Sunday, August 26th 2012</DATE><TITRE>Kill</TITRE><DESCRIPTION>
Extraction from 12 clones of lysostaphin and dispersin in pBK26 (shuttle vector) ligation and gel electrophoresis. The gel didn’t show any good clone.
</DESCRIPTION><TITRE>Surfactant</TITRE><DESCRIPTION>
Miniprep of 6 clones (3A ligation: pBK24, pBK4, pBK29). The electrophoresis showed that none of the tested clones had integrated the right plasmid.
</DESCRIPTION><TITRE>Stick</TITRE><DESCRIPTION>
Extraction from the 6 white clones (containing <I>xylR</I>-pBK26). Gel electrophoresis is run, but it didn’t show any good clone. 
</DESCRIPTION></JOUR><JOUR nb="27"><DATE>Monday, August 27th 2012</DATE><TITRE>Kill</TITRE><DESCRIPTION><UL><LI>Transformation of Bs 168 strain with pBK28 (1) plasmid which contains the [Constitutive Promoter + RBS + Lysostaphin] in the shuttle vector.</LI><LI>Midiprep : Extraction of P<SUB>veg</SUB>-dispersin-pSB1C3 of clone 22.</LI><LI>BK33 strain was put in storage.</LI></UL></DESCRIPTION><TITRE>Surfactant</TITRE><DESCRIPTION><UL><LI>Given the fact that the digested plasmids extracted from transformed NM522 <I>E. coli</I> with the 3A ligation (pBK24, pBK4, pBK29) had unexpected fragments (and impossible to explain), the ligation was done again, this time changing the vector/insert ratio (1/1 and 1/3, as opposed to the first trial with a ⅙ ratio ).</LI><LI>Liquid cultures of two potentially good clones of the NM522 strain transformed with a 3A ligation (pBK22(promoter P<SUB>xyl</SUB>, RBS and <I>sfp</I> gene), <I>abrB</I> gene and pSB1A3 backbone) were made for miniprep (the two plasmids were extracted using the classic ABC protocol, with extraction solutions that we made, so the yield and the purity of the extractions were poor; that’s why we decided to extract the plasmids again, this time using an extraction kit).</LI></UL></DESCRIPTION><TITRE>Stick</TITRE><DESCRIPTION>
No transformation was done, because we ran out of LB broth. 
</DESCRIPTION></JOUR><JOUR nb="28"><DATE>Tuesday, August 28th 2012</DATE><TITRE>For all purposes</TITRE><DESCRIPTION>
Meeting at 1:30 pm.
</DESCRIPTION><TITRE>Kill</TITRE><DESCRIPTION><UL><LI>Ligation of [Constitutive promoter + Dispersin] in the shuttle vector pBK25.</LI><LI>Results of the transformation of BS 168 strain with the pBK28 (1) in the shuttle vector : the negative control is full of bacteria → Is the Bs 168 strain resistant to erythromycin ? Or is the erythromycin concentration too low ? 
We made another transformation and we spread the transformed bacteria on LB+Ery plates with different erythromycin concentration (from 5 µg/mL to 10 µg/mL).</LI></UL></DESCRIPTION><TITRE>Surfactant</TITRE><DESCRIPTION>
The transformations with the two ligations (3A ligation of pBK24, pBK4, pBK29, with two different insert/vector ratios ) was successful. 12 clones per transformation are put in liquid cultures and then streaked on LB+Cm plates and LB+Amp plates.
</DESCRIPTION><TITRE>Stick</TITRE><DESCRIPTION>
Transformation of <I>xylR</I>-pBK10 (blunt end ligation) in NM522.
</DESCRIPTION></JOUR><JOUR nb="29"><DATE>Wednesday, August 29th 2012</DATE><TITRE>Kill</TITRE><DESCRIPTION><UL><LI>Transformation of the previous ligation [promoter-dispersin-pBK25] in <I>E. coli</I> NM522.</LI><LI>Results of the second transformation of Bs 168 strain with pBK28 (1) in the shuttle vector : the negative control is still full of bacteria even though the erythromycin concentration was higher than the previous trial (0,5 µg/mL the first time, 5 and 10 µg/mL the second time).</LI><LI>We made (LB+Ery) plates with different erythromycin concentrations in order to make different tests.</LI></UL></DESCRIPTION><TITRE>Surfactant</TITRE><DESCRIPTION><UL><LI>Only one of the 24 screened colonies concerning the two ligations (3A ligation of pBK24, pBK4, pBK29, with two different insert/vector ratios) did not turn up to be red (LB+Cm plate) and did not grow on a LB+Amp plate. The clone was used to inoculate a liquid culture for an extraction.</LI><LI>Miniprep of the two clones containing the <I>sfp</I> and <I>abrB</I> genes. The gel electrophoresis did not turn out as expected, probably because the bacteria used for inoculating the liquid cultures were contaminated or did not come from the same colony. We decided to transform the low purity plasmids into the NM522 strain.</LI><LI>Given the fact that we were not very optimistic concerning the 3A ligation of pBK24, pBK4 and pBK29, we decided to do two more trials, this time using  two ligated genes (<I>sfp</I> and <I>abrB</I>) coming from clones other than the one containing pBK22. However, the miniprep of the bacteria which were supposed to contain the two plasmids was unsatisfying, so we decided to do the 3A ligation with the low purity plasmids, isolated with the classic protocol because we were running out of time and we were behind the schedule.</LI><LI>In parallel, we decided to do a standard ligation in order to transfer the <I>lacI</I> gene into a plasmid having an antibiotic resistance other than Ampicillin. This way, in case the 3A ligations fails, we would not be forced to do again a 3A ligation because the plasmids would have different antibiotic resistances.</LI></UL></DESCRIPTION><TITRE>Stick</TITRE><DESCRIPTION><UL><LI>Transformation of <I>xylR</I>-pBK24 in NM522. The transformation of <I>xylR</I>-pBK24 is done in order to see if <I>xylR</I> was successfully ligated or not. If the bacteria are red, then <I>xylR</I> was not ligated.</LI><LI>24 clones containing pBK10-<I>xylR</I> are patched in LB+Amp growth medium, and then incubated.</LI></UL></DESCRIPTION></JOUR><JOUR nb="30"><DATE>Thursday, August 30th 2012</DATE><TITRE>Kill</TITRE><DESCRIPTION><UL><LI>Transformation results : too many clones on the control digestion not ligated vector so the transformation can’t be used for the transformation [Lysostaphin+Dispersin] in pSB1C3.</LI><LI>12 clones are put in culture from the transformation of [Lysostaphin + Dispersin] in the shuttle vector treated by BamHI.</LI><LI>Preparation of tubes for sequencing, launch of miniprep and midiprep.</LI><LI>Standard ligation between the shuttle vector pBK26 (pHT315 modified) and the pBK23 plasmid (= Lysostaphin in pSB1C3). Transformation of the ligation product in NM522 strain.</LI></UL></DESCRIPTION><TITRE>Surfactant</TITRE><DESCRIPTION><UL><LI>All 5 transformations were successful. White colonies are selected (pBK24 is a plasmid containing RFP in the pSB1C3 backbone, so we can exclude the colonies having integrated the religated backbone) and tested on LB+Cm and LB+Amp plates.</LI><LI>The miniprep of the one clone that had the expected phenotype (obtained after the transformation of the 3A ligation of pBK24, pBK4, pBK29) was digested, but the gel electrophoresis did not turn out as expected.</LI><LI>The miniprep confirmed that the ligation containing <I>lacI</I> was successful.</LI></UL></DESCRIPTION><TITRE>Stick</TITRE><DESCRIPTION><UL><LI>The patched plate is used as a master for replica-printing. The replica plate contains LB+Kan growth medium. Results : only one clone didn’t grow in the replica plate, meaning that it might be a clone containing <I>xylR</I>-pBK10 ligation. The clone is put in liquid culture overnight.</LI><LI>Another test is run to confirm that pBK34 plasmid (<I>xylR</I>-pBK24) doesn’t contain xylR : we digested the plasmid using NdeI restriction enzyme. Since <I>xylR</I> contains a <I>Nde</I>I restriction site (pBK24 doesn’t contain it), if pBK34 is opened by the enzyme, it means that it contains <I>xylR</I>. Result : pBK34 doesn’t contain xylem.</LI></UL></DESCRIPTION></JOUR><JOUR nb="31"><DATE>Friday, August 31st 2012</DATE><TITRE>For all purposes</TITRE><DESCRIPTION>
We have finally received the enzymes, so we tested the minipreps containing the modified shuttle vectors (containing the iGEM linker). The results are promising : 4 plasmids are digested by <I>Spe</I>I, so the ligation was successful. The four plasmids are further tested for the other 3 iGEM sites. To make sure that the site SpeI is not the initial site that we eliminated by filling-in and that the plasmid used for transformation and ligation was not contaminated with the original plasmid, a double digestion was done (<I>Eco</I>RI and <I>Spe</I>I). There was no fragment at 1,000 bp, so the ligation was successful.
</DESCRIPTION><TITRE>Kill</TITRE><DESCRIPTION><UL><LI>Verification of [Lysostaphin-Dispersin] in the shuttle vector (pBK26) clones : clones are not good.</LI><LI>Verification of the miniprep for sequencing ([Promoter + Dispersin] in pSB1C3) : DNA is okay.</LI><LI>Result of the transformation of NM522 strain with the Lysostaphin in pBK26 :</LI><UL><LI>The negative control is ok;</LI><LI>The positive control is full of colonies;</LI><LI>The transformation is full of colonies : 24 clones are selected (on plates and in liquid culture) in order to extract and check their DNA.  </LI></UL></UL></DESCRIPTION><TITRE>Surfactant</TITRE><DESCRIPTION>
Miniprep of the saturated cultures with transformed bacteria containing the ligation of the <I>sfp</I> and <I>abrB</I> genes. The electrophoresis confirmed that this time the bacteria contained the right plasmids.
</DESCRIPTION><TITRE>Stick</TITRE><DESCRIPTION><UL><LI>Extraction of the good clone from the replica-printing. Then a electrophoresis gel is run : the blunt end ligation was successful.</LI><LI>The plasmid is digested by <I>Eco</I>RI, <I>Xba</I>I, <I>Spe</I>I and <I>Pst</I>I to find out if the problem comes from a bad site sequence. Result : the <I>Eco</I>RI restriction site is not good, therefore is not recognized by <I>Eco</I>RI enzyme.</LI></UL></DESCRIPTION></JOUR><MONTH name="September" size="30"><JOUR nb="1"><DATE>Saturday, September 1st 2012</DATE><TITRE>Kill</TITRE><DESCRIPTION><UL><LI>Digestion of Lysostaphin in the shuttle vector and dispersin in pUC57 and verification of digestion. Then the vector is treated with CALF protein and a ligation is made using different of insert concentration. Then the ligation is transformed with commercial cells (STLBÉ) according to the  manufacturer protocol and another trial is made with higher concentration.</LI><LI>Verification of midiprep : Midiprep of Lysostaphin in pSB1C3 is okay but not the one of Dispersin in pUC57.</LI><LI>Miniprep of 14 clones NM522 transformed with Lysostaphin in pBK26. The digestions and electrophoresis show that 7 clones are ok ! They are put in storage under the references pBK38 (1) to pBK38 (6).</LI><LI>Extraction of 14 clones transformed with Dispersin in pBK25. Gel electrophoresis showed no good clones.</LI></UL></DESCRIPTION><TITRE>Surfactant</TITRE><DESCRIPTION><UL><LI>One of the 12 screened clones transformed with the ligation of pBK22 and pBK39 has the expected fragments. Unfortunately, the plasmid is mixed with another one. To separate the two plasmids we decided to transform 1 µL of the miniprep into <I>E. coli</I> NM522 strain.</LI><LI>6 other clones with the transformed 3A ligation (done on August 29th) are screened.</LI><LI>Standard ligation of the <I>abrB</I> gene and the plasmid containing <I>lacI</I> in pSB1C3 backbone (pBK37) and transformation. In parallel, another ligation is attempted : [pBK37 + pBK29].</LI></UL></DESCRIPTION></JOUR><JOUR nb="2"><DATE>Sunday, September 2nd 2012</DATE><TITRE>For all purposes</TITRE><DESCRIPTION>
Liquid cultures containing the modified shuttle vectors with the iGEM linker are inoculated and incubated overnight.
</DESCRIPTION><TITRE>Kill</TITRE><DESCRIPTION><UL><LI>Shuttle vector containing lysostaphin is treated with an alkaline phosphatase, then a ligation with dispersin is realized,  followed by transformation in <I>E. coli</I> NM522 strain.</LI><LI>Dispersin is ligated with pBK26 and then digested by BamHI before transformation. BamHI digestion allows not to transform the circularized vectors which doesn’t contain the insert.</LI></UL></DESCRIPTION><TITRE>Surfactant</TITRE><DESCRIPTION><UL><LI>Plasmids from the 6 clones containing the 3A ligation are extracted and digested. Gel electrophoresis showed that one of the clones has the expected fragments. However, it was mixed with another plasmid. In order to separate them, we decided to transform the NM522 strain with 1µL of the miniprep.</LI><LI>12 clones for each ligation (done the day before) are screened.</LI></UL></DESCRIPTION></JOUR><JOUR nb="3"><DATE>Monday, September 3rd 2012</DATE><TITRE>For all purposes</TITRE><DESCRIPTION><UL><LI>Midiprep of the cloned shuttle vectors (containing the iGEM linker).</LI><LI>Miniprep of pBK26 shuttle vector is done in order to increase the stock.</LI></UL></DESCRIPTION><TITRE>Kill</TITRE><DESCRIPTION><UL><LI>No clones are obtained on the plate lysostaphin+dispersin when we followed the manufacturer protocol even on the positive control. But few colonies appeared on plates done with our own protocol.These clones are put in liquid cultures.</LI><LI>Tests are launched to verify manufacturer cells. And other tests are launched to verify that our plates contained the appropriate antibiotic.</LI><LI>A transformation is made with our cells of their positive control.</LI><LI>Transformations of Bs 168 :
</LI><LI>Trial n°1 : negative control with H<SUB>2</SUB>O, positive control with pHT315 GFP (pBK18), transformation with Lysostaphin in pBK25 (=pBK28).</LI><LI>Trial n°2 : negative control with H<SUB>2</SUB>O, positive control with pHT315 GFP (pBK18), transformation with Lysostaphin in pBK26 (=pBK38).</LI></UL>
The bacteria are spread on LB+Ery plates ([Ery]=1 µg/mL and [Ery]=10 µg/mL).
The transformation of NM522 with ligation Dispersin in pBK26 was successful.</DESCRIPTION><TITRE>Surfactant</TITRE><DESCRIPTION><UL><LI><I>abrB</I> gene obtained by PCR was cloned in pSB1T3 backbone.</LI><LI>6 clones transformed with the mixture of two plasmids are screened.</LI><LI>Miniprep of 12 clones per transformed ligation : only one had the expected fragments (ligation of the <I>lacI</I> and <I>abrB</I> genes). The clone is put in storage under the reference pBK39.</LI></UL></DESCRIPTION></JOUR><JOUR nb="4"><DATE>Tuesday, September 4th 2012</DATE><TITRE>For all purposes</TITRE><DESCRIPTION>
Multiple tubes containing LB medium and Ampicillin at different concentrations ranging from 0 to 1.3 mg/mL are inoculated with bacteria containing the shuttle vectors (pBK35 and pBK36).
</DESCRIPTION><TITRE>Kill</TITRE><DESCRIPTION><UL><LI>Lots of clones are obtained with our cells.So we have tried a transformation of commercial cells with bigger quantities of DNA.</LI><LI>Miniprep of clones obtained with STLB2 are made. Clones are not okay. A new transformation with the same ligation mix is made but with NM522.</LI><LI>Results of the transformations of Bs 168 : not coherent. However, the problem seems to be identified : it is the erythromycin solution. Indeed, the erythromycin must be diluted in ethanol and not in water and must be kept refrigerated at -20°C.</LI><LI>Standard ligations between :
      </LI><LI>P<SUB>lac</SUB> in pSB1C3 (pBK9) digested by <I>Spe</I>I and <I>Pst</I>I and [RBS-<I>gfp</I>] in pSB1T3 digested by <I>Xba</I>I and <I>Pst</I>I;</LI><LI>P<SUB>xyl</SUB> (amplified by PCR) digested by <I>Spe</I>I and <I>Eco</I>RI and [RBS-<I>gfp</I>] in pSB1T3 digested by <I>Xba</I>I and <I>Eco</I>RI.</LI></UL>
These two ligation products are transformed in NM522 strain.<LI>Acrylamide gels and samples are prepared for SDS-PAGE.</LI></DESCRIPTION><TITRE>Surfactant</TITRE><DESCRIPTION>
Gel electrophoresis showed that 5 out of 6 clones screened after the transformation with the miniprep containing two plasmids had the expected fragments. However, we had doubts and we decided to search for restriction sites in the sequences that the plasmid was supposed to contain to make sure we had the right construction. We found out that a digestion with <I>Eco</I>RV would give 5 fragments of different sizes, enough to confirm the presence of the expected genes. Unfortunately, the electrophoresis was disappointing.
</DESCRIPTION><TITRE>Stick</TITRE><DESCRIPTION>
A PCR is run in order to get RBS-<I>xylR</I>. The PCR failed (not enough DNA-polymerase).
</DESCRIPTION><TITRE>Biofilm</TITRE><DESCRIPTION><UL><LI>One plate with <I>S. epidermidis</I> to evaluate the adhesion.</LI><LI>One plate with <I>S. epidermidis</I> + supernatant of different cultures (lysostaphin, dispersin...).</LI><LI>Same plate as the one above + crystal violet.</LI></UL></DESCRIPTION></JOUR><JOUR nb="5"><DATE>Wednesday, September 5th 2012</DATE><TITRE>Kill</TITRE><DESCRIPTION>
Few clones are obtained with commercial cells, there are not very efficient. A new transformation is made with the same ligation mix but in NM522.
</DESCRIPTION><TITRE>Surfactant</TITRE><DESCRIPTION><UL><LI>Standard ligation of pBK22 (<I>sfp</I> gene) and pBK39 (<I>abrB</I>-<I>lacI</I>) and transformation into the <I>E. coli</I> NM522 strain.</LI><LI>A SDS-PAGE gel is run to analyze lysostaphin (in BK32 and BK35 strains) and dispersin (BK29 strain).</LI></UL></DESCRIPTION></JOUR><JOUR nb="6"><DATE>Thursday, September 6th 2012</DATE><TITRE>For all purposes</TITRE><DESCRIPTION>
Ampicillin resistance tests gave some surprising results : even at 1.3 mg/mL there is bacterial growth. However, the higher Ampicillin concentration is, the lower OD<SUB>600</SUB> of the liquid cultures is. Ampicillin resistance test of the two shuttle vectors did not show any significant difference between the two strains (BK37 and BK38). The result is not surprising, given the fact that both plasmids (pBK36 and pBK35) derive from the shuttle vectors pHT315 and pHT304 which have the same origin replication as pUC19. However, the test must be done again, this time with at least two samples for each Ampicillin concentration in order to be able to estimate an error.
</DESCRIPTION><TITRE>Kill</TITRE><DESCRIPTION><UL><LI>Clones obtained on lysostaphin-dispersin plates are put in liquid culture.</LI><LI>Transformation of Bs 168 : new trial ! To improve our protocol, we took periodic OD<SUB>600</SUB>600 measurements in order to stop the cell growth at the best stage (just between the exponential and the stationary phase). The Bs 168 strain is transformed by :
      </LI><LI>pBK 25 (= modified shuttle vector pHT304);</LI><LI>pBK 26 (= modified shuttle vector pHT315);</LI><LI>pBK 28 (= Lysostaphin in the  modified shuttle vector pHT304);</LI><LI>pBK 38 (= Lysostaphin in the modified shuttle vector pHT315);</LI><LI>Dispersin in pBK26.</LI></UL>
We spread 200 µL of each transformed cells on LB + Erythromycin ([Ery] = 16 µg/mL) plates.</DESCRIPTION><TITRE>Surfactant</TITRE><DESCRIPTION>
12 clones transformed with the ligation pBK22 and pBK39 are screened.
</DESCRIPTION><TITRE>Stick</TITRE><DESCRIPTION>
2 new PCR are done to have <I>xylR</I>, but they failed. 
</DESCRIPTION></JOUR><JOUR nb="7"><DATE>Friday, September 7th 2012</DATE><TITRE>Kill</TITRE><DESCRIPTION><UL><LI>A miniprep of potential lysostaphin-dispersin clones is made : no clones are okay.</LI><LI>Result of the transformation of Bs 168 : </LI><UL><LI>All the negative control plates are clear : the erythromycin concentration is high enough to do a selection;</LI><LI>Some plates with the transformed bacteria are empty but some have from 1 to 4 clones : these clones are put in liquid cultures in order to extract and analyze their DNA;</LI></UL><LI>Transformation of Bs 168 : new trial by electroporation. As the last transformation, the Bs 168 strain is transformed by pBK25, pBK26, pBK28, pBK38 and dispersin in pBK26.</LI><LI>A new SDS-PAGE is made using supernatants and pellets as samples. The samples were not concentrated enough, so the gels didn’t show any of the expected bands.</LI></UL></DESCRIPTION><TITRE>Surfactant</TITRE><DESCRIPTION>
All the screened clones had the religated vector, so we decided to use commercial competent cells from STLB2 strain to transform the ligation of pBK39 and pBK22 in order to diminish the number of clones so that the chances of finding the recombinant plasmid containing the 3 genes (<I>sfp</I>, <I>abrB</I> and <I>lacI</I>) would be higher.
</DESCRIPTION><TITRE>Stick</TITRE><DESCRIPTION>
A new RBS-<I>xylR</I> PCR is run, now with different concentrations of genomic DNA and a different annealing temperature. The PCR was successful.
</DESCRIPTION><TITRE>Physiological tests</TITRE><DESCRIPTION><UL><LI>Liquid culture of <I>S. epidermidis</I> is incubated without any agitation at 37ºC.</LI><LI>Liquid culture of <I>B. subtilis</I> containing the dispersin gene.</LI><LI>Liquid culture of <I>B. subtilis</I> containing the same plasmid as <I>B. subtilis</I> dispersin but without the dispersin gene.</LI><LI>Liquid culture of <I>B. subtilis</I> containing the lysostaphin gene.</LI><LI>Liquid culture of <I>B. subtilis</I> containing the same plasmid as <I>B. subtilis</I> lysostaphin but without the lysostaphin gene.</LI></UL></DESCRIPTION></JOUR><JOUR nb="8"><DATE>Saturday, September 8th 2012</DATE><TITRE>Kill</TITRE><DESCRIPTION>
Miniprep of the clones Bs 168 transformed by pBK26, pBK28 : we use the same protocol as DNA extraction for <I>E.coli</I> with addition of Lysozyme to the buffer A1. Electrophoresis to analyze extracted DNA : there is nothing on the gel → Maybe the DNA isn’t concentrated enough ?
New clones are put in liquid cultures in order to make other minipreps. 
</DESCRIPTION><TITRE>Surfactant</TITRE><DESCRIPTION>
16 clones containing the ligation of pBK22 and pBK39 transformed into STLB2 commercial competent cells are screened.
</DESCRIPTION><TITRE>Stick</TITRE><DESCRIPTION>
Ligation of P<SUB>lac</SUB>-RBS-<I>xylR</I> and then transformation in NM522.
</DESCRIPTION><TITRE>Physiological tests</TITRE><DESCRIPTION>
Each of the 24 wells of  two microtiter plates are inoculated with 2mL of a suspension obtained from diluting the liquid culture 100 times with TSB enriched with 1% glucose  The plate is incubated during 24 hours at 37ºC.
</DESCRIPTION></JOUR><JOUR nb="9"><DATE>Sunday, September 9th 2012</DATE><TITRE>Kill</TITRE><DESCRIPTION><UL><LI>Miniprep of the clones Bs 168 transformed by pBK25, pBK26, pBK28, and Dispersin in pBK26. Electrophoresis to analyze extracted DNA :</LI><UL><LI>Bs 168 clone transformed by pBK25 has the right plasmid !! =)</LI><LI>Bs 168 clone transformed by pBK28 has the right plasmid !! <STRONG>BACILLUS LYSOSTAPHIN IS HERE !! :D :D :D</STRONG></LI><LI>Bs 168 clones transformed by pBK26 and dispersin in pBK26 : nothing on the gel... </LI></UL><LI>Transformation of Bs 168 GFP with the five same plasmids as before. The transformed bacteria are spread on LB + Ery plates, with two different concentrations : [Ery]=10 µg/mL and [Ery]=15 µg/mL.</LI><LI>Given the fact that there was nothing on the miniprep for pBK26 and pBK28, we made a new electrophoresis after using the seeDNA method  to concentrate the DNA of the miniprep. We see very light stripes on the gel, but the results for pBK26 and dispersin in pBK26 have to be confirm again...</LI><LI>Thanks to the good results, we put in storage the transformed <I>Bacillus</I> under the reference :</LI><UL><LI>BK41 : Bs 168 + pBK25</LI><LI>BK42 : Bs 168 + pBK28</LI><LI>BK43 : Bs 168 + pBK26 (to confirm)</LI><LI>BK44 : Bs 168 + Dispersin in pBK26 (to confirm)</LI></UL></UL></DESCRIPTION><TITRE>Surfactant</TITRE><DESCRIPTION>
The electrophoresis of the 16 digested plasmids extracted did not turn out as expected : all clones are in fact the initial plasmid, pBK39!
</DESCRIPTION><TITRE>Physiological tests</TITRE><DESCRIPTION>
The supernatant from the two microtiter plates are decanted. The wells of a third microtiter plate are filled with the supernatant of the liquid culture which is supposed to contain the dispersin. The supernatants are diluted in order to test different concentrations of the supernatants. The wells of a fourth microtiter plate are filled with the supernatant of the liquid culture which is supposed to contain the lysostaphin.
</DESCRIPTION></JOUR><JOUR nb="10"><DATE>Monday, September 10th 2012</DATE><TITRE>For all purposes</TITRE><DESCRIPTION><UL><LI>4 transformations with the same amount of plasmid are done (pBK19,pBK20, pBK35 and pBK36) in order to characterize the transformation efficiency of the shuttle vectors.</LI><LI>Ampicillin resistance test was repeated, this time with 3 tubes per Ampicillin concentration for each plasmid.</LI></UL></DESCRIPTION><TITRE>Surfactant</TITRE><DESCRIPTION>
We decided to give another try and do the ligation once again, this time using 5 time less vector and a ⅛ ratio vector/insert. Given the fact that we are running out of time, we did two ligations hoping to construct the same biobrick: pBK22+pBK39 and pBK29+pBK37. Both ligations were transformed into the <I>E. coli</I> NM522 strain.
</DESCRIPTION><TITRE>Kill</TITRE><DESCRIPTION><UL><LI>OD<SUB>600</SUB> test in tanks to test lysostaphin effect on a <I>S. epidermidis</I> culture. Strains used for the test are <I>E. coli</I> strains which might produce lysostaphin. Supernatant is filtered and applied on the <I>S. epidermidis</I> culture.</LI><LI><I>S. epidermidis</I> cultures are launched with tetracycline or without tetracycline. Presence or absence of antibiotic doesn’t change test results. The only effect is that cultures grow more slowly with antibiotic.</LI><LI>According to the results there is an effect of lysostaphin but we observed an effect due to the initial OD<SUB>600</SUB> of cultures that were different, so we don’t know if the results can be used. A new protocol is tried out, culture are diluted to have the same OD<SUB>600</SUB>.</LI><LI>New SDS-PAGE 12% gel is done with just the supernatants. The samples preparation didn’t go well and they were not enough concentrated. Therefore the gel didn’t show anything.</LI></UL></DESCRIPTION><TITRE>Plac and Pxyl kinetics</TITRE><DESCRIPTION><UL><LI>Measurement of the NM522 + P<SUB>lac</SUB> kinetics (with Cm) during 12h in a 96 well plate with and without different IPTG concentration addition. (1 mM; 0.5mM and 0.1mM)</LI><LI>In the same 96 well plate NM522 + P<SUB>xyl</SUB> kinetics is also measured during 12h with and without different xylose concentration. (2%; 0.5% and 0.2%)</LI><LI>Results : kinetics decreases so there is a problem. But it can be explained by the fact that the promoters need to be activated by a sigma factor which is active in the exponential phase and 12h is not enough. So this experiment has to be done during 24h.</LI></UL></DESCRIPTION><TITRE>Physiological tests</TITRE><DESCRIPTION>
The microtiter plates prepared the day before are analyzed. Each well is stained with crystal violet and subsequently OD<SUB>600</SUB> is measured in order to quantify the adherence of the strain. We can see that the biofilm is not adherent enough and it is destroyed no matter what kind of supernatant is tested. Unfortunately, we cannot say what the effect of the dispersin or lysostaphin is using this kind of test.
</DESCRIPTION></JOUR><JOUR nb="11"><DATE>Tuesday, September 11th 2012</DATE><TITRE>Experiments realized in Massy</TITRE><DESCRIPTION><UL><LI>Seeding a 96 well plate with <I>S. aureus</I> to produce a biofilm. Addition of supernatant of Bs168+pBK25 or Bs168+pBK26 or Bs168+pBK28 or Bs168+disp in pBK26 and Romain Briandet’s strains (Bs168+pWG200) with different dilutions (straight, 1/10, 1/100).(plate 1)</LI><LI>Seeding a 96 well plate with <I>S. aureus</I> to produce a biofilm (plate 2).</LI></UL></DESCRIPTION><TITRE>For all purposes</TITRE><DESCRIPTION><UL><LI>Design of the primers required to sequence the modified sequences of the shuttle vectors (after elimination of SpeI site by filling-in and cloning of the iGEM linker).</LI><LI>Ampicillin resistance test was a complete failure : the negative control (NM522 strain grown in LB broth supplemented with Ampicillin at the usual selective concentration), so the Ampicillin that was used lost it’s biological activity.</LI></UL></DESCRIPTION><TITRE>Surfactant</TITRE><DESCRIPTION>
12 clones per transformation are screened.
</DESCRIPTION><TITRE>Kill</TITRE><DESCRIPTION><UL><LI>OD<SUB>600</SUB> test in tanks to test lysostaphin effect on a <I>S. epidermidis</I> culture. Strains used for the test are <I>E. coli</I> strains which might produce lysostaphin. Supernatant is filtered and applied on the <I>S. epidermidis</I> culture. <I>S. epidermidis</I> cultures are launched with tetracycline or without tetracycline. Presence or absence of antibiotic doesn’t change test results. The only effect is that cultures grow more slowly with antibiotic. According to the results there is an effect of lysostaphin but we observed an effect of dilution due to the protocol, so we don’t know if the results can be used. So a new protocol is worked out.</LI><LI>SDS-PAGE is done using pellets, to find a good concentration.</LI></UL></DESCRIPTION><TITRE>Physiological tests</TITRE><DESCRIPTION>
Liquid culture of <I>S. epidermidis</I> is incubated without any agitation at 37ºC.
</DESCRIPTION></JOUR><JOUR nb="12"><DATE>Wednesday, September 12th 2012</DATE><TITRE>Experiments realized in Massy</TITRE><DESCRIPTION><UL><LI>We remove the supernatant in the plate 2 made on Tuesday and added Bs168+dispersin in pBK26 and Bs168+pBK26 with different dilutions. Observation of the plate at the confocal laser scanning microscope after 5 hours of incubation.  Exploitation of the pictures with Imaris software. Dispersin seems to have an effect on the biofilm.</LI><LI>Reading of the plate 1 with a confocal laser scanning microscope (after an overnight incubation). Exploitation of the pictures with Imaris software.</LI></UL></DESCRIPTION><TITRE>For all purposes</TITRE><DESCRIPTION>
The transformed plates with the modified and unmodified shuttle vectors do not show any significant difference, so we decided to make another essay, this time with 3 transformation for each plasmid.
</DESCRIPTION><TITRE>Surfactant</TITRE><DESCRIPTION>
Unfortunately, the screened clones did not integrate the expected plasmid.
</DESCRIPTION><TITRE>Kill</TITRE><DESCRIPTION><UL><LI>OD<SUB>600</SUB> test in tanks to test lysostaphin effect on a <I>S. epidermidis</I> culture with the new protocol. Strains used for the test are <I>Bacillus subtilis</I> strains which may produce lysostaphin. Supernatant is filtered and applied on the <I>S. epidermidis</I> culture. There is a problem with the results : the control is not good.</LI><LI>Miniprep of one clone containing dispersin in pBK26 is done. Gel electrophoresis showed that the clone was good. The plasmid is then put in storage (pBK41).</LI></UL></DESCRIPTION><TITRE>Physiological tests</TITRE><DESCRIPTION><UL><LI>5 test tubes containing a lamella are inoculated with the culture prepared the day (<I>S. epidermidis</I>) before diluted 1:100 with TSB supplemented with 1% glucose.</LI><LI>Liquid culture of <I>B. subtilis</I> containing the dispersin gene.</LI><LI>Liquid culture of <I>B. subtilis</I> containing the same plasmid as <I>B. subtilis</I> dispersin but without the dispersin gene.</LI><LI>Liquid culture of <I>B. subtilis</I> containing the lysostaphin gene.</LI><LI>Liquid culture of <I>B. subtilis</I> containing the same plasmid as <I>B. subtilis</I> lysostaphin but without the lysostaphin gene.</LI></UL></DESCRIPTION></JOUR><JOUR nb="13"><DATE>Thursday, September 13th 2012</DATE><TITRE>Experiments realized in Massy</TITRE><DESCRIPTION><UL><LI>Seeding of 2 96 well plate with <I>S. aureus</I> to produce a biofilm. Addition of supernatant of Bs168+pBK25 or Bs168+pBK26 or Bs168+pBK28 or Bs168+dispersin in pBK26 and Romain Briandet's strain (Bs168+pWG200) with different dilutions (straight, 1/10, 1/100).(plate 1)</LI><LI>OD<SUB>600</SUB> measurements with filtered supernatants.</LI><LI>Mobility test of Bs168  on agar 0.25% plate.</LI></UL></DESCRIPTION><TITRE>For all purposes</TITRE><DESCRIPTION><UL><LI>The transformed plates with the modified and unmodified shuttle vectors do not show any undeniable difference, so we decided to make another essay, this time with 3 transformation for each plasmid.</LI><LI>Ampicillin resistance test is repeated (3 samples for each Ampicillin concentration).</LI></UL></DESCRIPTION><TITRE>Surfactant</TITRE><DESCRIPTION>
Another 12 clones per transformation were screened (from the transformation made on 10th September).
</DESCRIPTION><TITRE>Kill</TITRE><DESCRIPTION>
OD<SUB>600</SUB> test in tanks to test dispersin effect on a <I>S. epidermidis</I> culture. Strains used for the test are <I>Bacillus subtilis</I> strains which may produce dispersin. Supernatant is filtered and applied on the <I>S. epidermidis</I> culture. Dispersin has no effect on <I>S. epidermidis</I> culture according to the results.
</DESCRIPTION><TITRE>Physiological tests</TITRE><DESCRIPTION><UL><LI>One of the tubes and lamella prepared the day before containing <I>S. epidermidis</I> cultures is quantified using crystal violet staining and OD<SUB>600</SUB> is measured. <I>S. epidermidis</I> cells are quite adherent on glass surfaces.</LI><LI>The supernatant of the four other tubes are decanted and tubes are filled with the supernatants produced by the four liquid cultures of <I>B. subtilis</I> prepared the day before in order to test the effect of lysostaphin and dispersin.</LI></UL></DESCRIPTION></JOUR><JOUR nb="14"><DATE>Friday, September 14th 2012</DATE><TITRE>For all purposes</TITRE><DESCRIPTION><UL><LI>Transformation efficiency tests confirm our supposition : there is no significant difference between the transformation yields of the 4 plasmids (unmodified shuttle vectors, pBK19 and pBK20 and modified shuttle vectors, pBK35 and pBK36).
Moreover, OD<SUB>600</SUB> of the bacterial cultures in LB medium at various Ampicillin concentrations shows no significant difference between the BK37 and BK38 strains.</LI><LI>TG1 (<I>E. coli</I>) strain containing pJIM2241 shuttle vector is put in storage (BK47).</LI></UL></DESCRIPTION><TITRE>Kill</TITRE><DESCRIPTION><UL><LI>OD<SUB>600s</SUB> test in tanks to test lysostaphin effect on a <I>S. epidermidis</I> culture. The strains used for the test are <I>Bacillus subtilis</I> strains which may produce lysostaphin. Supernatant is filtered and applied on the <I>S. epidermidis</I> culture.</LI><LI>SDS-PAGE is done with some changes in the protocol. The gels showed a band that matches with the expected lysostaphin molecular weight.</LI></UL></DESCRIPTION><TITRE>Surfactant</TITRE><DESCRIPTION>
Gel electrophoresis of the extracted plasmids shows that there are 5 possible plasmids containing the ligated insert (P<SUB>xyl</SUB>-<I>sfp</I>-<I>abrB</I>-<I>lacI</I>-terminator). The five plasmids are digested by EcoRV and the electrophoresis confirms that, this time, the plasmids have the expected fragments.
</DESCRIPTION><TITRE>Physiological tests</TITRE><DESCRIPTION>
The four tubes and lamella prepared the day before are quantified using crystal violet staining and OD<SUB>600</SUB>is measured. The same result is obtained : <I>S. epidermidis</I> biofilms are not adherent enough and biofilms are destroyed when the growth medium is changed.
</DESCRIPTION></JOUR><JOUR nb="15"><DATE>Saturday, September 15th 2012</DATE><TITRE>For all purposes</TITRE><DESCRIPTION><I>Bacillus subtilis</I> transformation with the 2 shuttle vectors pBK35 and pBK36.
</DESCRIPTION><TITRE>Kill</TITRE><DESCRIPTION>
OD<SUB>600</SUB> test in 96 well plate to test impact of lysostaphin and dispersin on a culture of <I>S. epidermidis</I> (test during 12 hours).
</DESCRIPTION><TITRE>Physiological tests : Mobility of B. subtilis</TITRE><DESCRIPTION>
LB plates containing a 0.3% agarose gel are inoculated with 5 μL of a 5 hour liquid culture of <I>B. subtilis</I> containing the plasmids used in transformations.
</DESCRIPTION><TITRE>Surfactant</TITRE><DESCRIPTION>
Transformation of the ligations between the plasmid pBK42 containing the construction [P<SUB>xyl</SUB>-<I>sfp</I>-<I>arbB</I>-<I>lacI</I>] and the shuttle vectors pBK35 and pBK36 into the NM522 strain .
</DESCRIPTION></JOUR><JOUR nb="16"><DATE>Sunday, September 16th 2012</DATE><TITRE>For all purposes</TITRE><DESCRIPTION>
Screening of 12 clones per <I>Bacillus</I> transformation.
</DESCRIPTION><TITRE>Surfactant</TITRE><DESCRIPTION>
Screening of 15 clones transformed with the ligation pBK42+pBK35 and pBK42+pBK36
</DESCRIPTION><TITRE>Physiological tests : Mobility of B. subtilis</TITRE><DESCRIPTION>
The diameter of the disks formed by <I>B. subtilis</I> is measured. The fact of introducing the plasmids did not impact the swarming mobility of our bacteria.
</DESCRIPTION></JOUR><JOUR nb="17"><DATE>Monday, September 17th 2012</DATE><TITRE>For all purposes</TITRE><DESCRIPTION>
Erythromycin at different concentrations was spread on LB plates with in order to test the Erythromycin resistance of the <I>B. Subtilis</I> strain transformed with the shuttle vectors.
</DESCRIPTION><TITRE>Surfactant</TITRE><DESCRIPTION>
Unfortunately the screened clones (containing the ligations pBK42+pBK35 and pBK42+pBK36) did not have the right plasmid. Another ligation of the construction [P<SUB>xyl</SUB>-<I>sfp</I>-<I>arbB</I>-<I>lacI</I>] is attempted, this time after gel extraction and purification of the digested pBK42 plasmid with EcoRI and PstI. This way, the backbone is eliminated and the chances of ligating the construction instead of the plasmid are higher.
</DESCRIPTION></JOUR><JOUR nb="18"><DATE>Tuesday, September 18th 2012</DATE>
titre&gt;For all purposes</JOUR><DESCRIPTION>
Multiple tubes containing LB medium supplemented with Erythromycin at different concentrations are inoculated with the <I>B. subtilis</I> strains containing the shuttle vectors pBK35 and pBK36 in order to test the antibiotic resistance. For the same purpose LB + Ery plates are spotted with the same cultures at different dilution ratios.
</DESCRIPTION><TITRE>Surfactant</TITRE><DESCRIPTION>
30 clones containing the construction [P<SUB>xyl</SUB>-<I>sfp</I>-<I>arbB</I>-<I>lacI</I>] in the shuttle vector are screened.
</DESCRIPTION></MONTH><JOUR nb="19"><DATE>Wednesday, September 19th 2012</DATE><TITRE>For all purposes</TITRE><DESCRIPTION>
The strains that were supposed to contain the shuttle vectors did not grow, so we suspect that the clones are in fact mutants.
</DESCRIPTION><TITRE>Surfactant</TITRE><DESCRIPTION>
Miniprep of the transformed clones with the construction [P<SUB>xyl</SUB>-<I>sfp</I>-<I>arbB</I>-<I>lacI</I>]. The gel electrophoresis shows that only two of them have the expected fragments.
</DESCRIPTION></JOUR><JOUR nb="20"><DATE>Thursday, September 20th 2012</DATE><TITRE>For all purposes</TITRE><DESCRIPTION><I>Bacillus subtilis</I> transformation with the shuttle vectors pBK35 and pBK36.
</DESCRIPTION><TITRE>Surfactant</TITRE><DESCRIPTION>
Transformation of <I>B. subtilis</I> 168 and <I>B. subtilis abrB</I> strains with the construction [P<SUB>xyl</SUB>-<I>sfp</I>-<I>arbB</I>-<I>lacI</I>] in the pBK35 shuttle vector. 
</DESCRIPTION></JOUR><JOUR nb="21"><DATE>Friday, September 21st 2012</DATE><TITRE>Plac and Pxyl kinetics</TITRE><DESCRIPTION>
A 7-hour-long kinetics is made with P<SUB>lac</SUB> and P<SUB>xyl</SUB> to be sure to have measurement in the exponential phase. Only one IPTG and xylose concentration is used.
P<SUB>lac</SUB> = IPTG 0.5% and P<SUB>xyl</SUB> = xylose 0.5%.
Moreover, a 24-hour-long kinetics is also made in a 96 well plate. The conditions are the same as the one done for 12h except that bacteria are inoculated in a 1/100 dilution (instead of 1/50).
</DESCRIPTION><TITRE>Kill</TITRE><DESCRIPTION>
SDS-PAGE is done in order to detect Dispersin in <I>B. subtilis</I> and <I>E. coli</I>'s supernatants. The gels were not conclusive, and were not good due to an ammonium persulfate problem.
</DESCRIPTION></JOUR><JOUR nb="22"><DATE>Saturday, September 22th 2012</DATE><TITRE>For all purposes</TITRE><DESCRIPTION>
12 clones transformed with the shuttle vectors are screened.
</DESCRIPTION><TITRE>Kill</TITRE><DESCRIPTION>
A new SDS-PAGE is done for dispersin. A migration problem persists, due to ammonium persulfate. 
</DESCRIPTION><TITRE>Surfactant</TITRE><DESCRIPTION>
24 clones transformed with the construction [P<SUB>xyl</SUB>-<I>sfp</I>-<I>abrB</I>-<I>lacI</I>] are screened.
</DESCRIPTION></JOUR><JOUR nb="23"><DATE>Sunday, September 23th 2012</DATE><TITRE>For all purposes</TITRE><DESCRIPTION>
Multiple tubes containing  2 mL liquid LB medium are supplemented with Erythromycin at various concentrations in order to test the antibiotic resistance of the two shuttle vectors. Also, LB+Ery plates are spotted with liquid cultures with the <I>B. subtilis </I> strains containing the shuttle vectors.
</DESCRIPTION><TITRE>Kill</TITRE><DESCRIPTION>
A last SDS-PAGE is run. This time the problems were successfully solved. But the gels were not conclusive due to a low dispersin concentration.
</DESCRIPTION><TITRE>Surfactant</TITRE><DESCRIPTION>
6 of the 24 clones are put in liquid culture for further testing in LB medium supplemented with xylose at a concentration of 2%.
</DESCRIPTION></JOUR><JOUR nb="24"><DATE>Monday, September 24th 2012</DATE><TITRE>For all purposes</TITRE><DESCRIPTION>
Both strains grew in all tubes. However, there is a difference concerning OD<SUB>600</SUB> between the two strains. The test is repeated (with antibiotic concentrations ranging from 0 to 1.5 mg/mL) in order to make sure there is a notable difference between the two strains. </DESCRIPTION><TITRE>Surfactant</TITRE><DESCRIPTION><LI> In order to characterize the construction [P<SUB>xyl</SUB>-<I>sfp</I>-<I>arbB</I>-<I>lacI</I>] two tests are made. To confirm the presence of <I>sfp</I> gene, we made emulsions with the filtered supernatant of the transformed strains and sunflower oil. Concerning <I>abrB</I>  gene, a biofilm formation test was made in a 24-well microplate in order to compare the transformed strain to the wild-type strain. 
</LI></DESCRIPTION></JOUR><JOUR nb="25"><DATE>Tuesday, September 25th 2012</DATE><TITRE>For all purposes</TITRE><DESCRIPTION>
The results of the antibiotic resistance in liquid culture show that the <I>B. subtilis</I> strain containing pBK35 is less resistant than the one containing pBK36.</DESCRIPTION><TITRE>Surfactant</TITRE><DESCRIPTION>
The emulsion test and the biofilm formation test confirm the presence of <I>sfp</I> and <I>abrB</I> genes; 
</DESCRIPTION></JOUR><MONTH name="October" size="31"><JOUR nb="10"><DATE>Wednesday, October 10th 2012</DATE><TITRE>Surfactant</TITRE><DESCRIPTION><UL><LI>Ligation of the part pBK14 containing the <I>gfp</I> gene with the part pBK7 (containing a <I>B. subtilis</I> RBS).</LI><LI>Ligation of the part pBK13 containing the <I>rfp</I> gene with the part pBK7 (containing a <I>B. subtilis</I> RBS).</LI><LI>Transformation of the NM522 <I>E. coli</I> strain with the two ligations.</LI></UL></DESCRIPTION><TITRE>Stick</TITRE><DESCRIPTION>Ligation of the PCR product <I>xylR</I> gene with the part pBK9 (P<SUB><I>lac</I></SUB> gene)and NM522 transformation.
</DESCRIPTION></JOUR><JOUR nb="11"><DATE>Thursday, October 11th 2012</DATE><TITRE>Surfactant</TITRE><DESCRIPTION>
Screening of 15 clones per transformation (antibiotic resistance test).

</DESCRIPTION><TITRE>Stick</TITRE><DESCRIPTION>
Inoculation of 5 mL of LB medium with chloramphenicol with 1 colony of <I>Bacillus subtilis</I> 168 <I>ΔabrB</I> and 5 mL of LB medium with 1 colony of <I>Bacillus subtilis</I> 168. Incubation overnight at 37°C. (This is for the 48h positive <I>Bacillus subtilis</I> biofilm test plate)
</DESCRIPTION></JOUR><JOUR nb="12"><DATE>Friday, October 12th 2012</DATE><TITRE>Surfactant</TITRE><DESCRIPTION>
Liquid cultures are inoculated with colonies having the expected antibiotic phenotype.

To test the surfactant properties of the surfactin produced by the <I>B. subtilis</I> BK52 strain, a biofilm test was made in a 24-well plate. Liquid cultures are seeded with BK52 and BK49.

</DESCRIPTION><TITRE>Stick</TITRE><DESCRIPTION><UL><LI>(24h plate)The same inoculation as previously is prepared.</LI><LI>(48h plate)<I>Bacillus subtilis</I> 168 <I>ΔabrB</I> or <I>Bacillus subtilis</I> 168 are added in the 12 well plate with MgSO4  1mM and Glucose 0.1 %.</LI></UL>
The plate is incubated at 37°C for 48h

(For both) Inoculation of 5 mL of LB medium with 1 colony of <I>E coli</I> (<I>ompR++</I> GFP, curlis overproduction). Incubation overnight at 37°C. 
</DESCRIPTION></JOUR><JOUR nb="13"><DATE>Saturday, October 13th 2012</DATE><TITRE>Surfactant</TITRE><DESCRIPTION>
Miniprep of the screened clones. Unfortunately, the gel electrophoresis was disappointing and the extracted plasmids did not contain the ligated genes.

A 24-well plate is incubated with filtered supernatant of the saturated cultures with BK52 and BK49 at room temperature for 24 hours.


</DESCRIPTION><TITRE>Stick</TITRE><DESCRIPTION>
(24h plate)<I>Bacillus subtilis</I> 168 <I>ΔabrB</I> or <I>Bacillus subtilis</I> 168 are added in a 12-well plate with MgSO4  1mM and Glucose 0.1 %.
The plate is incubated at 37°C for 48h.
</DESCRIPTION></JOUR><JOUR nb="14"><DATE>Sunday, October 14th 2012</DATE><TITRE>Surfactant</TITRE><DESCRIPTION>
The wells are carefully rinsed with M63 medium two times after having previously discarded the supernatant. Then, each well is inoculated with a saturated adherent <I>E. coli</I> strain diluted 100 times in LB medium diluted 2 times.
Moreover, another 24-well plate assay was made using the same supernatant, but this time the plate was incubated only 4 hours before <I>E. coli</I> inoculation.

</DESCRIPTION><TITRE>Stick</TITRE><DESCRIPTION>
(For both) The supernatant is removed, wells are washed and LB is added.To see if <I>E coli</I> forms a biofilm over the <I>Bacillus subtilis</I> biofilm, the overnight culture of <I>E coli</I> is added into each well.
Plates are incubated for 36h at 30°C.
</DESCRIPTION></JOUR><JOUR nb="16"><DATE>Tuesday, October 16th 2012</DATE><TITRE>Surfactant</TITRE><DESCRIPTION>
Unfortunately, the observation of the plates under confocal microscope is not possible, so the experiment must be repeated. The only difference is that this time 12-well plates  with glass lamellae are used so the observation of the biofilm would be possible.

</DESCRIPTION><TITRE>Stick</TITRE><DESCRIPTION>
(For both) Now let's observe using the confocal microscope !
Results : It works ! Our positive <I>Bacillus subtilis</I> biofilm inhibits the stick of other bacteria !
</DESCRIPTION></JOUR><JOUR nb="17"><DATE>Wednesday, October 17th 2012</DATE><TITRE>Surfactant</TITRE><DESCRIPTION>
Liquid cultures were inoculated with the strains BK49 and BK52 and then incubated at 37C for 48 hours.
</DESCRIPTION></JOUR><JOUR nb="19"><DATE>Saturday, October 19th 2012</DATE><TITRE>Surfactant</TITRE><DESCRIPTION>
The supernatant from the plate is discarded. Each well is inoculated with an <I>E. coli</I> saturated culture diluted 50 times in LB medium diluted 2 times.


</DESCRIPTION></JOUR><JOUR nb="20"><DATE>Sunday, October 20th 2012</DATE><TITRE>Surfactant</TITRE><DESCRIPTION>
Ligation of pBK40 and pBK16 followed by transformation into the NM522 strain.

</DESCRIPTION></JOUR><JOUR nb="21"><DATE>Monday, October 21st 2012</DATE><TITRE>Surfactant</TITRE><DESCRIPTION>
The surfactin pre-treated glass lamellae are observed under the confocal microscope. The negative control has a thick biofilm whereas the treated lamella has a few isolated colonies. 

</DESCRIPTION></JOUR><JOUR nb="22"><DATE>Tuesday, October 22nd 2012</DATE><TITRE>Surfactant</TITRE><DESCRIPTION>
Screening of 12 clones.

</DESCRIPTION></JOUR><JOUR nb="23"><DATE>Wednesday, October 23rd 2012</DATE><TITRE>Surfactant</TITRE><DESCRIPTION>
Miniprep of the transformed clones. The gel electrophoresis confirmed the expected insert. (<I>abrB</I> gene in pSB1C3 backbone)

</DESCRIPTION></JOUR><JOUR nb="24"><DATE>Tuesday, October 24th 2012</DATE><TITRE>Surfactant</TITRE><DESCRIPTION>
The plasmids pBK42 and pBK46 were submitted to the Registry.

</DESCRIPTION></JOUR></MONTH></XML></DIV><DIV id="pageContent"><H1>The Notebook</H1><DIV id="monthSelection"><DIV class="month" id="July">July</DIV><DIV class="month" id="August">August</DIV><DIV class="month" id="September">September</DIV><DIV class="month" id="October">October</DIV></DIV><DIV id="descJour"><DIV id="jourPrecedent">Previous day</DIV><DIV id="jourSuivant">Next day</DIV><DIV id="leDiapo"><DIV id="titreDiapo">Watch us in action !</DIV></DIV></DIV></DIV><H1>Our collections</H1><DIV class="introduction contenuTexte" style="margin:20px;font-size:15px;"><DIV class="unProto" onclick="window.open('https://static.igem.org/mediawiki/2012/8/89/Tableau_souches.pdf', 'Strains collection'); return false;">Strains collection</DIV><DIV class="unProto" onclick="window.open('https://static.igem.org/mediawiki/2012/b/be/Tableaux_plasmide.pdf', 'Plasmids collection'); return false;">Plasmids collection</DIV></DIV></DIV></DIV><DIV class="printfooter">
Retrieved from &quot;<A href="http://2012.igem.org/Team:Lyon-INSA/notebook">http://2012.igem.org/Team:Lyon-INSA/notebook</A>&quot;</DIV></DIV><DIV id="footer-box" class="noprint"><DIV id="footer"><UL id="f-list"><LI id="t-recentchanges"><A href="/Special:RecentChanges" title="Recent changes">Recent changes</A></LI><LI id="t-whatlinkshere"><A href="/Special:WhatLinksHere/Team:Lyon-INSA/notebook" title="List of all wiki pages that link here [j]" accesskey="j">What links here</A></LI><LI id="t-recentchangeslinked"><A href="/Special:RecentChangesLinked/Team:Lyon-INSA/notebook" title="Recent changes in pages linked from this page [k]" accesskey="k">Related changes</A></LI><LI id="t-specialpages"><A href="/Special:SpecialPages" title="List of all special pages [q]" accesskey="q">Special pages</A></LI><LI><A href="/Special:Preferences">My preferences</A></LI></UL></DIV><DIV id="footer"><UL id="f-list"><LI id="t-print"><A href="/wiki/index.php?title=Team:Lyon-INSA/notebook&amp;printable=yes" title="Printable version of this page [p]" accesskey="p">Printable version</A></LI><LI id="t-permalink"><A href="/wiki/index.php?title=Team:Lyon-INSA/notebook&amp;oldid=292974" title="Permanent link to this revision of the page">Permanent link</A></LI><LI id="privacy"><A href="/2012.igem.org:Privacy_policy" title="2012.igem.org:Privacy policy">Privacy policy</A></LI><LI id="disclaimer"><A href="/2012.igem.org:General_disclaimer" title="2012.igem.org:General disclaimer">Disclaimers</A></LI></UL></DIV></DIV></BODY></HTML>