<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_ETH_Zurich_Notebook skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><DIV id="top_title"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:ETH Zurich/Notebook</SPAN></H1></DIV><DIV id="HQ_page"><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><DIV class="title_container"><SPAN id="title_text"><A href="https://2016.igem.org/Team:ETH_Zurich">PAVLOV'S COLI</A></SPAN></DIV><DIV class="menu_container" id="main_menu"><UL class="menu" id="menu"><LI class="menu_item"><A href="https://2016.igem.org/Team:ETH_Zurich">HOME</A></LI><LI class="menu_item"><A>TEAM</A></LI><LI><A href="https://2016.igem.org/Team:ETH_Zurich/Team">Team</A></LI><LI><A href="https://2016.igem.org/Team:ETH_Zurich/Collaborations">Collaborations</A></LI><LI><A href="https://2016.igem.org/Team:ETH_Zurich/Attributions">Attributions</A></LI></UL><LI class="menu_item"><A>PROJECT</A></LI><LI><A href="https://2016.igem.org/Team:ETH_Zurich/Description">Description</A></LI><LI><A href="https://2016.igem.org/Team:ETH_Zurich/Design">Design</A></LI><LI><A href="https://2016.igem.org/Team:ETH_Zurich/Proof">Proof of Concept</A></LI><LI><A href="https://2016.igem.org/Team:ETH_Zurich/Demonstrate">Demonstrate</A></LI><LI class="menu_item"><A>MODELING</A></LI><LI><A href="https://2016.igem.org/Team:ETH_Zurich/Model">Model</A></LI><LI><A href="https://2016.igem.org/Team:ETH_Zurich/Sensor_Module">Sensor Module</A></LI><LI><A href="https://2016.igem.org/Team:ETH_Zurich/Switch_Module">Switch Module</A></LI><LI><A href="https://2016.igem.org/Team:ETH_Zurich/Detector_Module">Reporter Module</A></LI><LI><A href="https://2016.igem.org/Team:ETH_Zurich/NO_Release">NO Release</A></LI><LI><A href="https://2016.igem.org/Team:ETH_Zurich/Parameters">Parameters</A></LI><LI class="menu_item"><A>LAB</A></LI><LI><A href="https://2016.igem.org/Team:ETH_Zurich/Experiments">Experiments</A></LI><LI><A href="https://2016.igem.org/Team:ETH_Zurich/LabResults">Lab Results</A></LI><LI><A href="https://2016.igem.org/Team:ETH_Zurich/InterLab">Interlab Study</A></LI><LI><A href="https://2016.igem.org/Team:ETH_Zurich/Safety">Safety</A></LI><LI><A href="https://2016.igem.org/Team:ETH_Zurich/Hardware">Hardware</A></LI><LI><A href="https://2016.igem.org/Team:ETH_Zurich/Notebook">Notebook</A></LI><LI class="menu_item"><A>PARTS</A></LI><LI><A href="https://2016.igem.org/Team:ETH_Zurich/Parts">Parts</A></LI><LI><A href="https://2016.igem.org/Team:ETH_Zurich/Basic_Part">Basic Parts</A></LI><LI><A href="https://2016.igem.org/Team:ETH_Zurich/Composite_Part">Composite Parts</A></LI><LI><A href="https://2016.igem.org/Team:ETH_Zurich/Part_Collection">Part Collection</A></LI><LI class="menu_item"><A>HUMAN PRACTICES</A></LI><LI><A href="https://2016.igem.org/Team:ETH_Zurich/Human_Practices">Human Practices</A></LI><LI><A href="https://2016.igem.org/Team:ETH_Zurich/HP/Silver">Silver</A></LI><LI><A href="https://2016.igem.org/Team:ETH_Zurich/HP/Gold">Gold</A></LI></DIV><DIV class="content_wrapper" id="main_wrapper"><UL class="menu" id="outline"><LI class="outline_item"><A href="#week1">Week 1</A></LI><LI class="outline_item"><A href="#week2">Week 2</A></LI><LI class="outline_item"><A href="#week3">Week 3</A></LI><LI class="outline_item"><A href="#week4">Week 4</A></LI><LI class="outline_item"><A href="#week5">Week 5</A></LI><LI class="outline_item"><A href="#week6">Week 6</A></LI><LI class="outline_item"><A href="#week7">Week 7</A></LI><LI class="outline_item"><A href="#week8">Week 8</A></LI><LI class="outline_item"><A href="#week9">Week 9</A></LI><LI class="outline_item"><A href="#week10">Week 10</A></LI><LI class="outline_item"><A href="#week11">Week 11</A></LI><LI class="outline_item"><A href="#week12">Week 12</A></LI><LI class="outline_item"><A href="#week13">Week 13</A></LI><LI class="outline_item"><A href="#week14">Week 14</A></LI><LI class="outline_item"><A href="#week15">Week 15</A></LI><LI class="outline_item"><A href="#week16">Week 16</A></LI><LI class="outline_item"><A href="#week16">Week 17</A></LI></UL><DIV class="sec page_title"><DIV><H1>NOTEBOOK</H1></DIV></DIV><DIV class="sec white" id="week1"><DIV><H1>JULY</H1></DIV><DIV><H2>WEEK 1 (27.6. – 5.7.)</H2><H3>Test 1A: Construction of pNorV and norR plasmids</H3><P>
We ordered gBlocks for:
<UL><LI>norR without forbidden restriction sites</LI><LI>two versions of pNorV: one with the native spacer after transcription start site and one without</LI></UL></P><H3>Test 1B: Construction of  promoters with esaboxes and esaR plasmid</H3><P>
We ordered gBlocks for promoters with esaboxes. E. coli colonies with plasmid with esaR from addgene arrived. 
                </P><H3>Test 3: Switch based on recombinases</H3><P>
We ordered gBlocks for 3 different codon optimized recombinases without forbidden restriction sites:
<UL><LI>bxb1</LI><LI>phiC31 and</LI><LI>tp901</LI></UL></P><H3>General</H3><P><UL><LI>We ordered first oligos</LI><LI>We prepared first TFB1 and TFB2 buffers for competent cells. The next day we prepared the first batch of competent TOP10 cells (80 transformations).</LI><LI>We did first transformations:
</LI><LI>interlab study plasmids</LI><LI>pSEVA backbone plasmids</LI><LI>plasmids from distribution kit to get J23118 promoter, terminator, prefix and suffix</LI><LI>Transformation of plasmids with fluorescent proteins we might use: sfGFP, mCherry, mNectarine, mTurqouise. </LI></UL><LI>Followed were first overnight cultures of transformations and first minipreps of the plasmids from transformations and addgene colonies.</LI><LI>We prepared first batch of ingredients for M9 media.</LI><LI>We poured first LB-agar plates with single resistances for carbenicillin, kanamycin and chloramphenicol.</LI></P></DIV></DIV><DIV class="sec light_grey" id="week2"><DIV><H2>WEEK 2 (6.7. – 12.7.)</H2><H3>Test 1A: Construction of pNorV and norR plasmids</H3><P>
We used two approaches to get norR and pNorV fragments: 
<UL><LI>PCR to extract genomic copy of norR and pNorV</LI><LI>PCR to multiply norR and pNorV from gblocks.</LI></UL>
Comment: Genomic PCR was not successful because we added too much bacterial culture. It was not repeated after we successfully multiplied norR from gBlock.
                </P><H3>Test 1B: Construction of promoters with esaboxes and esaR plasmid</H3><P>
We did PCR to get a fragment with esaR from addgene plasmid and added overhangs to it. 
</P><H3>Test 3: switch based on recombinases</H3><P>
We did site directed mutagenesis (PCR) to add ssRa tag to mNectarine and sfGFP.
</P><H3>Directed evolution: make EsaR specific to AHL present in the gut</H3><P><UL><LI>We prepared electro-competent DH5alpha -uptr cells.</LI><LI>We transformed respective cells with plasmid with uracil phosphoribosyltransferase (upp or uptr).</LI><LI>We performed Tecan plate reader experiment to test the response of cells with and without the plasmid towards 5-fluorouracil. Upp/gfp plasmid is under control of dmpR/phenol.
</LI><LI>5-FU had an inhibitory effect on growth with all concentrations. There was no growth difference between un-induced and induced state</LI></UL><LI>We attempted to construct  CAT-UPTR fusion protein:
</LI><LI>We did PCR to get fragments with CAT (chloramphenicol resitance) and UPTR.</LI><LI>We did Gibson asesembly of CAT and UPTR fragments to create fusion protein.</LI><LI>We attempted to construct hsvTK (herpes simplex virus thimidine kinase)-APH-Stop-GFP operon:
</LI><LI>We did PCR to create APH and hsvTK fragments and performed Gibson assembly. Colony PCR did not show any results. However, since the cells grew, they were Amp resistant.</LI></P><H3>General</H3><P><UL><LI>We repeated failed transformations.</LI><LI>We transformed additional pSEVA empty backbone plasmids.</LI><LI>We did PCR to linearize pSEVA backbones and to get fragments with prefix + J23118 and terminator + suffix for Gibson assemblies.</LI><LI>We did digestion to exchange oris on two plasmid backbones.</LI><LI>Following all successful transformations we did overnight culture and minipreps.</LI></UL></P></DIV></DIV><DIV class="sec white" id="week3"><DIV><H2>WEEK 3 (13.7. – 19.7.)</H2><H3>Test 1A: Construction of pNorV and norR plasmids</H3><P>
We performed a Gibson assembly to create a norR plasmid with J23118 promoter and T0 terminator.
</P><H3>Test 3: Switch based on recombinases</H3><P><UL><LI>We did PCR to create fragments with overhangs for Gibson assembly for parts containing recombinases phiC3, bxb1 and tp901, with attB site in front of recombinase and attP site after terminator. </LI><LI>We did transformations of sfGFP + ssRa and mNectarine + ssRa plasmids created a week before. Adding ssRa tag to sfGFP was not successful.</LI></UL></P><H3>Directed evolution</H3><P>
We did PCR of fragments for construction of hsvTK-GFP with overlaps to esaboxes. 
</P><H3>General</H3><P><UL><LI> We repeted all previously failed digestions to exchange resistances on empty backbone plasmids.</LI><LI> After transformations of plasmids created with GA or site directed mutagenesis, colonies were picked in the evening for overnight culture and miniprep was done the next day in the morning.</LI></UL></P></DIV></DIV><DIV class="sec light_grey" id="week4"><DIV><H2>WEEK 4 (20.7. – 26.7.)</H2><H3>Test 1A: Construction of pNorV and norR plasmids</H3><P><UL><LI>We did PCR to construct promoter library for norR</LI><LI>We did PCR to add overlaps to norR fragment and repeat Gibson assemblt for norR</LI><LI>We did Gibson assembly of pNorV + sfGFP plasmid. Followed we did PCR to create fragment with insert pNorV+sfGFP to put it in a different backbone.</LI></UL></P><H3>Test 1B: Construction of promoters with esaboxes and esaR plasmids</H3><P><UL><LI>We multiplied with PCR fragmets from gBlocks of promoters with esaboxes. </LI><LI>We created fragment with RBS+EsaR from addgene plasmid </LI></UL>
Comment: We had problem with constructing J23118+esabox plasmids: restriction did not yield the correct fragment size.
</P><H3>Test 3: Switch based on recombinases</H3><P><UL><LI>We did site directed mutagenesis (PCR) to add moderate ssRa tag to mNectarine.</LI><LI>-	We did PCR to create remaining fragments for Gibson assembly of our initial design for reporters for recombinases:
</LI><LI>fragments containing bxb1, tp901, phiC31</LI><LI>terminator</LI><LI>mNectarine</LI><LI>tetR</LI></UL>
Comment: At the end of the week all fragments for the reporter were created
</P></DIV></DIV><DIV class="sec white" id="week5"><DIV><H2>WEEK 5 (27.7. – 2.8.)</H2><H3>Test 1B: Construction of promoters with esaboxes and esaR plasmids</H3><P><UL><LI>We did PCR of fragments to construct esaR plasmid:
</LI><LI>empty backbone from norR plasmid to put esaR in place of norR</LI><LI>fragment with overlaps and RBS+EsaR</LI></UL><LI>We performed Gibson assembly to create esaR plasmid. </LI></P><H3>Test 2: AND gate promoter pNorV+esabox</H3><P>
We did restriction and ligation to create norR + esaR plasmid on medium copy plasmid
</P><H3>Test 3: Switch based on recombinases</H3><P><UL><LI>We did PCR to create fragment with RBS+TetR with overhangs and Gibson assembly to create tetR plasmid</LI><LI>We performed Gibson assembly of plasmid with mNectarine.</LI><LI>We did PCR to generate fragments with the according attB site in front of recombinases bxb1, phiC31 and tp901 and fragments with according attP sites after the terminator</LI></UL>
Comment: at the end of the week all fragments for the assembly of reporters are ready, except sfGFP with ssRa tag. 
</P><H3>General</H3><P><UL><LI>Ori exchange in plasmid did not work with restriction, we tried a new approach with two part Gibson assembly.</LI><LI>We were experiencing problem with gel cleanup, which often yielded impure fragments. Impurities from the fragment might inhibit Gibson assembly reactions. We tested gel extractions with different elution buffers: water, elution buffer from the kit and Tris-HCl. We decide to use Tris-HCl. </LI><LI>We 3D printed a stand with magnetic beads for purification of gel extractions with magnetic beads</LI></UL></P></DIV></DIV><DIV class="sec light_grey" id="week6"><DIV><H1>AUGUST</H1><H2>WEEK 6 (3.8. – 9.8.)</H2><H3>Test 1A: Construction of pNorV and norR plasmids</H3><P><UL><LI>We repeated Gibson assembly to create two versions of pNorV + sfGFP plasmids. </LI><LI>We did an ori exchange to create low copy plasmid with norR. </LI></UL></P><H3>Test 3: Switch based on recombinases</H3><P>
Results from model simulations show that the design where recombinase is not embedded between the recombinase recognition sites is better. We changed the original design into a three plasmid system and we redesigned plasmids and order oligos.
</P><H3>General</H3><P>
We prepared competent cells (160 transformations).
</P></DIV></DIV><DIV class="sec white" id="week7"><DIV><H2>WEEK 7 (10.8. – 16.8.)</H2><H3>Test 1A: Construction of pNorV and norR plasmids</H3><P><UL><LI>We did colony PCR and sent to sequencing pNorV+sfGFP plasmids</LI><LI>We did site directed mutagenesis to add moderate ssRa tag to sfGFP in a plasmid with pNorV+sfGFP. Sequencing results showed a missing terminator and ssRa tag. Addition of ssRa tag was not successful.</LI><LI>We did first double transformations of pNorV+sfGFP and norR plasmids with additional double transformations of pNorV + empty backbone as a control.</LI><LI>We did a plate reader experiment. Results were inconclusive due to dilution error. As a source of nitric oxide (NO) we used DETANO.</LI></UL></P><H3>Test 1B: Construction of promoters with esaboxes and esaR plasmids</H3><P>
We performed restriction and ligation to create a low copy plasmid version of esaR plasmid.
</P><H3>Test 3: Switch based on recombinases</H3><P><UL><LI>We received a plasmid with bxb1 recombinase from Bonnet lab. </LI><LI>We transform plasmid from the distribution kit with ptet promoter. </LI><LI>We did resistance exchange on pSEVA291 backbone.</LI><LI>We did PCR to create fragments without attP and attB sites next to recombinases and performed Gibson assembly of the new design for the recombinases. </LI><LI>We created a low copy plasmid version of tetR.</LI></UL></P><H3>General</H3><P>
We prepared competent cells and poured plates with double and triple resistances. We 3D print rack for sorting of white tips.
</P></DIV></DIV><DIV class="sec light_grey" id="week8"><DIV><H2>WEEK 8 (17.8. – 23.8.)</H2><H3>Test 1A: Construction of pNorV and norR plasmids</H3><P><UL><LI>We repeated addition of ssRa tag to pNorV+sfGFP with site directed mutagenesis. </LI><LI>Sequencing revealed deletion in norR gene. </LI><LI>We performed plate reader with low copy number plasmid versions of pNorV+sfGFP and norR plasmids:
</LI></UL></P><P><B>Figure 1:</B> PnorV dose response curve for a very low range of DETA/NO concentrations. Samples where norR was not present in the cell also got induced when the DETA/NO was added
			</P></DIV><DIV class="image_box full_size" style="max-width:500px;"><P><B>Figure 1:</B> Activation of pNorv promoter shows a very low induction in comparison to the positive control
			</P></DIV><LI>In the plate reader experiments we did a time and dose response curve for strains with pNorV and norR plasmid and with pNorV and empty backbone. Concentrations of NO from DETANO were based on a model developed by our modelers. Due to small amount of DETANO remaining in the lab we were only able to use concentrations in the lower range of the dose response. The difference between the fluorescence of positive control and pNorV plasmid strains additionally supports our assumption. Most importantly, get a dose response even when norR plasmid is not present in the cell. Results from this experiment show we get an induction of pNorV when the DETANO is present in the medium with or without norR plasmid. </LI>
Based on the plate reader experiment and on sequencing results which show deletion in norR we concluded the NorR from our plasmid is not responsible for the activation of transcription and the dose response we observe. We made a hypothesis that NorR from a genomic copy of norR is binding to the pNorV promoter on our plasmid. 
<H3>Test 1B: Construction of promoters with esaboxes and esaR plasmids</H3><P>
 We created a plasmid with KpnI site between J23118 promoter and sfGFP to enable addition of esaboxes with restriction and ligation. After that, we added esaboxes behind J23118 promoter using restriction and ligation. 
</P><H3>Test 3: Switch based on recombinases</H3><P><UL><LI>Colony PCR of Gibson assembly from the previous week did not show any positive results. We repeated Gibson assembly for plasmids with recombinases under tet promoter.</LI><LI>We put mNectarine on a medium copy plasmid</LI></UL></P></DIV></DIV><DIV class="sec white" id="week9"><DIV><H2>WEEK 9 (24.8. – 30.8.)</H2><H3>Test 2: AND gate promoter pNorV+esabox</H3><P><UL><LI>We did Gibson assembly of plasmid with esaR and norR on a low copy plasmid. </LI><LI>We performed a restriction and ligation to create hybrid promoters for the AND gate: We created 10 different combinations of promoters:
<DIV><TABLE><TBODY><TR><TH>Hybrid promoter </TH></TR><TR><TD> pNorV_native_spacer_esabox </TD></TR><TR><TD> pNorV_native_spacer_esabox_8bp_spacer_esabox </TD></TR><TR><TD> pNorV_native_spacer_esabox_8bp_spacer_esabox_8bp_spacer_esabox </TD></TR><TR><TD> pNorV_native_spacer_esabox_15bp_spacer_esabox </TD></TR><TR><TD> pNorV_native_spacer_esabox_15bp_spacer_esabox_15bp_spacer_esabox </TD></TR><TR><TD> pNorV_no_spacer_esabox </TD></TR><TR><TD> pNorV_no_spacer_esabox_8bp_spacer_esabox </TD></TR><TR><TD> pNorV_no_spacer_esabox_8bp_spacer_esabox_8bp_spacer_esabox </TD></TR><TR><TD> pNorV_no_spacer_esabox_15bp_spacer_esabox </TD></TR><TR><TD> pNorV_no_spacer_esabox_15bp_spacer_esabox_15bp_spacer_esabox </TD></TR></TBODY></TABLE></DIV></LI></UL></P><H3>Test 3: Switch based on recombinases</H3><P>
We did colony PCR and sequencing of plasmids with recombinases under tet promoter. We successfully created plasmids with bxb1 and tp901, both under tet inducible promoter ptet. We were not successful with creating plasmid containing phiC31. 
</P><H3>Test 4: AND gate promoter pNorV+LldO</H3><P>
We did site directed mutagenesis (PCR) to insert LldO1 site upstream of pNorV.
</P><H3>Interlab study</H3><P>
We transformed all interlab study test plasmids and did plate reader and FACS experiment. 
</P><H3>General</H3><P><UL><LI>We prepared stock of competent cells (160 transformations). </LI><LI>We prepared a stock of ingredients for M9 medium</LI><LI>We prepared LB-agar dry stock.</LI></UL></P></DIV></DIV><DIV class="sec light_grey" id="week10"><DIV><H1>SEPTEMBER</H1><H2>WEEK 10 (31.8. – 6.9.)</H2><H3>Test 1A: Construction of pNorV and norR plasmids</H3><P>
We did site directed mutagenesis to fix the mutation in norR from the gBlock.
</P><H3>Test 1B: Construction of promoters with esaboxes and esaR plasmids</H3><P><UL><LI>We attempted to replace J23118 promoter in esaR plasmid with J23114 promoter. Attempt was unsuccessful.</LI><LI>All versions of J23118+esabox promoters from previous attempt to construct them had mutations. We repeated restriction and ligation to insert esaboxes after J23118 promoter. Colony PCR and sequencing confirmed we got correct versions. </LI></UL></P><H3>Test 2: AND gate promoter pNorV+esabox</H3><P>
Sequencing showed we successfully build all 10 different AND gate promoters. 
</P><H3>Test 3: Switch based on recombinases</H3><P>
Recombinase plasmids and TetR plasmid were ready.
<UL><LI>We constructed mNectarine under ptet promoter.</LI><LI>We performed PCR to create fragments for Gibson assembly of reporter constructs for recombinases. Gibson assembly of reporter constructs failed. We introduced intermediate step in designing reporter plasmids by redesigning reporter plasmids for recombinases into simpler test plasmids before building the full reporter.</LI></UL></P><H3>Test 4: AND gate promoter pNorV+LldO</H3><P>
Previous attempt at site directed mutagenesis failed. We tried a second attempt to insert LldO1 site in front of pNorV promoter. We got several positive colony PCRs.
</P></DIV></DIV><DIV class="sec white" id="week11"><DIV><H2>WEEK 11 (7.9. – 13.9.)</H2><H3>Test 1A: Construction of pNorV and norR plasmids</H3><P>
We did a Gibson assembly with a gBlock corrected with site directed mutagenesis. Sequencing results show we still have mutation in norR. 
</P><H3>Test 1B: Construction of promoters with esaboxes and esaR plasmids</H3><P>
We performed plate reader experiments to test and compare all variations of the designed esabox promoters.
</P><DIV class="image_box full_size" style="max-width:500px;"><P><B>Figure 1:</B> Image shows a dose response for a promoter with only one esabox.
			</P></DIV><DIV class="image_box full_size" style="max-width:500px;"><P><B>Figure 1:</B> Image shows a dose response for a promoters with three esaboxes.
			</P></DIV><DIV class="image_box full_size" style="max-width:500px;"><P><B>Figure 1:</B> Image the effect of the size of the spacer (8bp or 15bp) in the promoters with 3 esaboxes.
			</P></DIV><H3>Test 2: AND gate promoter pNorV+esabox</H3><P>
Results from the plate reader experiments showed that the AND gate is not working yet. The two repressors have been shown to not be fully functional or not respond to the current dose responses. From previous experiments we made a hypothesis that the native norR on the chromosome might be enough for the activation of our pNorV promoter.
</P><DIV class="image_box full_size" style="max-width:500px;"><P><B>Figure 1:</B> AND gate is not functional.
			</P></DIV><H3>Test 3: Switch based on recombinases</H3><P>
Sequencing showed construction of mNectarine and mNectarine + ssRa tag under ptet promoter was successful. Construction of sfGFP under ptet promoter was also successful. We did not manage to create sfGFP+ssRa tag. 
</P><H3>Directed evolution</H3><P>
Directed evolution of esaR was put on hold until we built promoters with esaboxes.
<UL><LI>Double transformation of esaR plasmid and plasmid with esaboxes and sfGFP</LI><LI>Resistance exchange for plasmid with promoter+esaboxes to Tet resistance and for plasmid with esaR to Amp resistance.</LI></UL></P><H3>General</H3><P>
Plasmids with ampicilin resistance grow poorly on carbenicilin plates. Poor plates with halved dose of carbenicilin. 
</P></DIV></DIV><DIV class="sec light_grey" id="week12"><DIV><H2>WEEK 12 (14.9. – 20.9.)</H2><H3>Test 1A: Construction of pNorV and norR plasmids</H3><P>
We tried another attempt at fixing sequence in norR with site directed mutagenesis. We did not manage to fix the mutation. 
</P><H3>Test 1B: Construction of promoters with esaboxes and esaR plasmids</H3><P><UL><LI>Attempt to construct low copy plasmid with esaR under very weak promoter (J23102). </LI><LI>Plate reader experiments
</LI></UL></P><P><B>Figure 1:</B> Dose response of the promoter with one esabox with wider AHL concentration range with and without esaR plasmid
			</P></DIV><DIV class="image_box full_size" style="max-width:500px;"><P><B>Figure 1:</B> Dose response of the promoter with two esaboxes with wider AHL concentration range with and without esaR plasmid
			</P></DIV><DIV class="image_box full_size" style="max-width:500px;"><P><B>Figure 1:</B> Dose response of the promoter with three esaboxes with wider AHL concentration range with and without esaR plasmid
			</P></DIV><DIV class="image_box full_size" style="max-width:500px;"><P><B>Figure 1:</B> Dose response comparison of designs with different sizes of spacers for three esaboxes 
			</P></DIV><UL><LI>Plate reader experiments show that in comparison with the positive control, strains with esaR plasmid have lower fluorescence. This implies that esaR is functional and is repressing transcription of sfGFP. Induction with AHL did not significantly change fluorescence output. This implies esaR is not released from the promoter. We set the hypothesis that AHL we used was expired. In promoters with multiple esaboxes, the promoter with 8 base pair spacer is more effectively repressed in comparison to the promoter where there is 15 base pair spacer in between esaboxes. </LI></UL><H3>Test 2: AND gate promoter pNorV+esabox</H3><P>
Plate reader experiment:
</P><DIV class="image_box full_size" style="max-width:500px;"><P><B>Figure 1:</B> AND gate dose response
			</P></DIV><UL><LI>Increase in DETA/NO doses does not make any difference. We made a decision to design new AND gate variants with esaboxes added in different locations in pNorV. We additionallt decide to increase the promoter strength of esaR.</LI></UL><H3>Test 3: Switch based on recombinases</H3><P>
We did Gibson assembly of reporter construct for bxb1 and tp901 recombinases.
</P><H3>Test 4: AND gate promoter pNorV+LldO</H3><P>
We performed site directed mutagenesis to insert LldO2 site after pNorV promoter. Colony PCRs were successful. However, sequencing showed the two sites were not completely integrated.
</P><H3>Directed evolution</H3><P><UL><LI>We made plasmid with promoter and 3 esaboxes with 8bp spacers ready for insertion of dual selectors</LI></UL></P><P><B>Figure 1:</B> Testing of CAT-UPRT fusion with 5-FU
			</P></DIV>
We did a plate reader experiment to test CAT-UPRT fusion with 5-FU. DH5alpha cells were used as a negative control. They should not be responsive to 5-fluoro-uracil. <DIV class="image_box full_size" style="max-width:500px;"><P><B>Figure 1:</B>  DH5a response to kanamycin
			</P></DIV>
Experiments showed that plasmid-containing cells are much more sensitive to 5-FU than DH5a cells without the plasmid. Plasmid containing cells are completely Cm resistant. DH5a cells also grew, but much slower. Kanamycin did not inhibit the growth, which means that DH5 –upp cells have integrated Kan resistance instead of UPRT genes. 
<H3>General</H3><P>
We prepared isothermal reaction buffer solutions and  ITA/Gibson assembly mix.
</P></DIV></DIV><DIV class="sec white" id="week13"><DIV><H2>WEEK 13 (21.9. – 29.9.)</H2><H3>Test 1A: Construction of pNorV and norR plasmids</H3><P><UL><LI>plate reader experiment
</LI></UL></P><P><B>Figure 1:</B> Testing of the maximum activation of pNorV
			</P></DIV><DIV class="image_box full_size" style="max-width:500px;"><P><B>Figure 1:</B> Dose response of pNorV using endogenous copy of NorR
			</P></DIV><LI>We started with experiments to confirm the hypothesis that pNorV can be activated with genomic copy of norR. We prepared Keio competent cells of parent strain and of norR knockout strain (40 transformations) and transformed pNorV plasmid into Keio strains.</LI><H3>Test 1B: Construction of promoters with esaboxes and esaR plasmids</H3><DIV class="image_box full_size" style="max-width:500px;"><P><B>Figure 1:</B> EsaR can effectively repress gene expression.
			</P></DIV><H3>Test 2: AND gate promoter pNorV+esabox</H3><P><UL><LI>plate reader experiments
</LI></UL></P><P><B>Figure 1:</B> Using presence or absence of EsaR we can observe and gate behavior in the current design of the AND gate promoter. 
			</P></DIV><LI>We designed new AND gate combinations.</LI><H3>Test 3: Switch based on recombinases</H3><H4>plate reader experiments with reporters from 2014</H4><H4>testing mNectarine plasmid</H4><UL><LI>We did plate reader experiments to determine time and dose response points for mNectarine for FACS measurements.</LI><LI>We did FACS measurements of mNectarine with induction with aTc. 
</LI><LI>Measurements:
</LI><LI>Ptet_mNectarine + constitutive TetR (With/without ssrA tag</LI><LI>Positive control: Ptet_mNectarine + empty backbone (With/without ssrA tag)</LI><LI>Negative control: Empty backbone + constitutive TetR. </LI></UL><LI>Time points:
</LI><LI>{0, 40, 100, 172, 275} min after induction.</LI><LI> Doses:
</LI><LI>{0, 0.2, 2.0, 20.0, 200.0, 2000.0} ng/ul</LI>
Positive control: We saw cells without fluorescence in positive control. We made a hypothesis that those are dead cells because overnight culture reached stationary phase.

<H4>testing sfGFP plasmid</H4><UL><LI>We did plate reader experiments to determine time and dose response points for sfGFP for FACS measurements.</LI><LI>We did FACS measurements of sfGFP with induction with aTc. 
</LI><LI>Measurements:
</LI><LI>Ptet_SFgfp + constitutive TetR</LI><LI>Positive control: Ptet_sfGFP + empty backbone</LI><LI>Negative control: Empty backbone + constitutive TetR. </LI></UL><LI>Time points:
</LI><LI>{0, 45, 100, 155, 200} min after induction.</LI><LI> Doses:
</LI><LI>{0, 0.2, 2.0, 20.0, 200.0, 2000.0} ng/ul</LI>
Positive control had no fluorescence. Inactive (dead) cells present were in the measurements because overnight culture reached stationary phase and a portion of population of the cells was dead.
<H3>Test 5: switch based on Cpf1 system</H3><P><UL><LI>We created fragments for Gibson assembly and Gibson assembly of plasmid with mNectarine and gRNA recognition sites inside sfGFP gene. </LI><LI>PCR to amplify reporter fragments with homology flanks for chromosomal insertion into 3 different genes: LacZ, talB and yqgB. </LI></UL></P><H3>Directed evolution</H3><P><UL><LI>We performed random mutagenesis with Taq and manganese with esaR_Amp fusion fragment as a template. Following was Gibson assembly of mutated variants.</LI><LI>We did the first attempt at selection: After transformation of Gibson assembly of mutated variants of esaR, a portion was plated on plates and the rest was inoculated in LB. 5-FU was added to the culture after one hour. After discovering LB medium requires 10x more 5-FU than M9 medium, the attempt at selection was cancelled. </LI></UL></P><H3>Putting our plasmids into biobrick format</H3><P>
We did first attempt to create composite parts for biobricks based on iGEM protocol. Attempt was not successful. 
</P><H3>General</H3><P>
We prepared stock of competent cells (480 transformations).
</P></DIV></DIV><DIV class="sec light_grey" id="week14"><DIV><H1>OCTOBER</H1><H2>WEEK 14 (30.9. – 6.10.)</H2><H3>Test 1B: Construction of promoters with esaboxes and esaR plasmids</H3><P>
We created a plasmid with two copies of esaR to increase the amount of EsaR in the cells. 
</P><H3>Test 2: AND gate promoter pNorV+esabox</H3><P>
plate reader experiment
</P><H3>Test 3: Switch based on recombinases</H3><P>
plate reader and FACS results show the two versions of recombinases (bxb1 and tp901, both with moderate ssRa tag) we have either work too efficiently (p901) or not enough (bxb1). We designed, created fragments and did a Gibson assembly to create variations of recombinases with no ssRa tag and with stronger ssRa tag.
</P><DIV class="image_box full_size" style="max-width:500px;"><P><B>Figure 7:</B> FACS experiments show improved dynamics of bxb1 upon changing the RBS. In red we see the switch dynamics of bxb1 before the change of RBS. In blue we see the improved switch dynamics of bxb1.
		</P></DIV><H3>Test 4: AND gate promoter pNorV+LldO</H3><P>
Site directed mutagenesis did not work. We designed a gBlock with a hybrid promoter and a promoter with spacer instead of pNorV. We construct the AND gate with Gibson assembly. Colony PCRs and sequencing confirmed the correct insert.
</P><H3>Test 5: switch based on Cpf1 system</H3><P><UL><LI> We performed PCR of all fragments related to the test in order to construct:
</LI><LI>plasmid with gRNA in front of GFP</LI><LI>plasmid with cas9</LI><LI>plasmid with cpf1</LI><LI>plasmid with gRNA1 and gRNA2 for cas9</LI><LI>plasmid with mNectarine inside sfGFP and </LI><LI>oplasmid with mNectarine and sfGFP</LI></UL><LI>We did a Gibson assembly of plasmid with mNectarine inside sfGFP and plasmid with mNectarine and sfGFP. </LI></P><H3>Directed evolution</H3><P><UL><LI>We did a second attempt at selection: Transformation of mutated variants and incubation in SOC for 8 hours. Cells are transferred to M9 medium and prepared for plate reader experiment. 5-FU was added at OD600 = 0.1. When the OD reached 0.5 the cells were not able to grow further in the next 12 hours. Second attempt at selection cancelled. Hypothesis that 5-FU is more toxic to cells in stationary phase.</LI><LI>PCR for codon exchange from F to H in shvTK</LI></UL></P><H3>Putting our plasmids into biobrick format</H3><P>
We did a second attempt to create composite parts for biobricks based on iGEM protocol. Restriction yielded very poor results. 
</P><H3>General</H3></DIV></DIV><DIV class="sec white" id="week15"><DIV><H2>WEEK 15 (7.10. – 13.10.)</H2><H3>Test 1A: Construction of pNorV and norR plasmids</H3><DIV class="image_box full_size" style="max-width:500px;"><P><B>Figure 1:</B> Dose response of pNorV using endogenous copy of NorR
			</P></DIV>
plate reader experiment for dose response. 15 000 DETA/NO is toxic to cells
<H3>Test 2: AND gate promoter pNorV+esabox</H3><P>
Plate reader experiment to test new AND gate dose responses. We created our <A href="https://2016.igem.org/Team:ETH_Zurich/Part_Collection">Part Collection</A> from the old and new AND gate designs.
</P><DIV class="image_box full_size" style="max-width:500px;"><P><B>Figure 3:</B> Image shows examples of different hybrid promoters we constructed. The designed different in the location of the esaboxes (and whether the location caused steric or competitive inhibition of the pNorV promoter), the number of esaboxes and in the size of spacing between the esaboxes. The design shown up most left showed the best AND gate logic behavior.
			</P></DIV><DIV class="image_box full_size" style="max-width:500px;"><P><B>Figure 1:</B> Dose response for the newly designed AND gate
			</P></DIV><DIV class="image_box full_size" style="max-width:500px;"><P><B>Figure 1:</B> Dose response for the newly designed AND gate with and without EsaR
			</P></DIV><H3>Test 4: AND gate promoter pNorV+LldO</H3><P>
We did a resistance exchange on LldR plasmid from iGEM team ETH Zurich 2015 and a double transformation of hybrid promoters pNorV + lactate and LldR. 
</P><H3>Test 5: switch based on Cpf1 system</H3><P><UL><LI>We did a Gibson assembly of plasmids with gRNA, cas9 and cpf1. We picked the red colonies from the two plasmids where mNectarine is inside sfGFP gene and where it is separately on the same plasmid as sfGFP. Following were colony PCR, miniprep and sequencing of respective plasmids.</LI><LI>We did PCR to create fragments with mNectarine inside sfGFP with homology flanks to three different regions on chromosome:
</LI><LI>lacZ</LI><LI>talB and</LI><LI>yqgB. </LI></UL><LI>We did a PCR to create fragments with separate mNectarine and sfGFP genes with homology flanks to the same regions on the chromosome.</LI><LI>-	We attempted to insert fragments onto a chromosome: 
</LI><LI>preparation of electrocompetent EcNR1 cells</LI><LI>electroporation of respective fragments</LI><LI>FACS sorting for fluorescent cells and </LI><LI>o	inoculation of fluorescent cells in the LB medium overnight</LI></P><H3>Complete system</H3><P>
We created fragments, did Gibson assembly and colony PCR to create parts with bxb1 recombinase with different ssRa tags under AND gate promoter.
</P><H3>Putting our plasmids into biobrick format</H3><P>
Restriction overnight (8 hours) worked. This lead us to assume our enzymes are partially inactive and that the actual concentration of an active enzyme is much lower. We got new enzymes and repeated the restrictions. Restrictions were effective. We were finally able to start putting our biobrick parts into the shipping plasmid backbone. 
</P></DIV></DIV><DIV class="sec light_grey" id="week16"><DIV><H2>WEEK 16 (14.10. – 19.10.)</H2><H3>Test 4: AND gate promoter pNorV+LldO</H3><P>
We did plate reader experiment to test lactate + NO AND gate. The results show the potential for an AND gate behaviour after future optimization
</P><H3>Biobricks</H3><P>
We did PCR for basic parts, restriction and ligation of composite parts, transformations, minipreps and sequencing of all biobrick parts.
</P><H3>Complete system</H3><P>
We did triple transformations with the full system and did preliminary studies to test how all parts work together
</P></DIV></DIV><DIV class="sec white" id="week17"><DIV><H2>WEEK 17 (14.10. – 19.10.)</H2><H3>Test 4: AND gate promoter pNorV+LldO</H3><P>
We repeated the plate reader experiments. Results imply an AND gate logic of the hybrid promoter.
</P><DIV class="image_box full_size" style="max-width:500px;"><P><B>Figure 8:</B> The 2D dose response for the lactate and DETA/NO show that the hybrid promoter exhibits an AND logic behavior.
			</P></DIV><H3>Test 5: switch based on Cpf1 system</H3><P>
We run the preliminary experiments, where we tested whether the gRNA can effectively excise mNectarine gene inserted in the sfgfp gene. We observed the increase of GFP fluorescence upon induction of Cpf1
</P><DIV class="image_box full_size" style="max-width:500px;"><P><B>Figure 9:</B> The dose response show that Cpf1 can excise mNectarine from sfgfp. From left to right are increasingly higher doses of tetracyclin (0, 2000, 5000 ng/ml) and a positive control. When tetracycline induces expression of Cpf1, we observe green fluorescence.
		</P></DIV><H3>Complete system</H3><P>
We ran the full system with an AHL+NO AND gate together with the improved bxb1 recombinase. The cells contained three plasmids: the EsaR plasmid, the plasmid containing AND gate promoter with recombinase gene downstream and the plasmid containing sfgfp under promoter flanked with bxb1 recognition sites. In the last plasmid, the promoter was initially facing the opposite direction of sfgfp. 
</P><DIV class="image_box full_size" style="max-width:500px;"><P><B>Figure 10:</B> The output of the full system resembles an AND gate behavior upon induction.
			</P></DIV></DIV></DIV><DIV class="sec blue"><DIV><H2> Thanks to the sponsors that supported our project: </H2></DIV></DIV></DIV></BODY></HTML>