<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_ETH_Zurich_Switch_Module skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><DIV id="top_title"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:ETH Zurich/Switch Module</SPAN></H1></DIV><DIV id="HQ_page"><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><DIV class="title_container"><SPAN id="title_text"><A href="https://2016.igem.org/Team:ETH_Zurich">PAVLOV'S COLI</A></SPAN></DIV><DIV class="menu_container" id="main_menu"><UL class="menu" id="menu"><LI class="menu_item"><A href="https://2016.igem.org/Team:ETH_Zurich">HOME</A></LI><LI class="menu_item"><A>TEAM</A></LI><LI><A href="https://2016.igem.org/Team:ETH_Zurich/Team">Team</A></LI><LI><A href="https://2016.igem.org/Team:ETH_Zurich/Collaborations">Collaborations</A></LI><LI><A href="https://2016.igem.org/Team:ETH_Zurich/Attributions">Attributions</A></LI></UL><LI class="menu_item"><A>PROJECT</A></LI><LI><A href="https://2016.igem.org/Team:ETH_Zurich/Description">Description</A></LI><LI><A href="https://2016.igem.org/Team:ETH_Zurich/Design">Design</A></LI><LI><A href="https://2016.igem.org/Team:ETH_Zurich/Proof">Proof of Concept</A></LI><LI><A href="https://2016.igem.org/Team:ETH_Zurich/Demonstrate">Demonstrate</A></LI><LI class="menu_item"><A>MODELING</A></LI><LI><A href="https://2016.igem.org/Team:ETH_Zurich/Model">Model</A></LI><LI><A href="https://2016.igem.org/Team:ETH_Zurich/Sensor_Module">Sensor Module</A></LI><LI><A href="https://2016.igem.org/Team:ETH_Zurich/Switch_Module">Switch Module</A></LI><LI><A href="https://2016.igem.org/Team:ETH_Zurich/Detector_Module">Reporter Module</A></LI><LI><A href="https://2016.igem.org/Team:ETH_Zurich/NO_Release">NO Release</A></LI><LI><A href="https://2016.igem.org/Team:ETH_Zurich/Parameters">Parameters</A></LI><LI class="menu_item"><A>LAB</A></LI><LI><A href="https://2016.igem.org/Team:ETH_Zurich/Experiments">Experiments</A></LI><LI><A href="https://2016.igem.org/Team:ETH_Zurich/LabResults">Lab Results</A></LI><LI><A href="https://2016.igem.org/Team:ETH_Zurich/InterLab">Interlab Study</A></LI><LI><A href="https://2016.igem.org/Team:ETH_Zurich/Safety">Safety</A></LI><LI><A href="https://2016.igem.org/Team:ETH_Zurich/Hardware">Hardware</A></LI><LI><A href="https://2016.igem.org/Team:ETH_Zurich/Notebook">Notebook</A></LI><LI class="menu_item"><A>PARTS</A></LI><LI><A href="https://2016.igem.org/Team:ETH_Zurich/Parts">Parts</A></LI><LI><A href="https://2016.igem.org/Team:ETH_Zurich/Basic_Part">Basic Parts</A></LI><LI><A href="https://2016.igem.org/Team:ETH_Zurich/Composite_Part">Composite Parts</A></LI><LI><A href="https://2016.igem.org/Team:ETH_Zurich/Part_Collection">Part Collection</A></LI><LI class="menu_item"><A>HUMAN PRACTICES</A></LI><LI><A href="https://2016.igem.org/Team:ETH_Zurich/Human_Practices">Human Practices</A></LI><LI><A href="https://2016.igem.org/Team:ETH_Zurich/HP/Silver">Silver</A></LI><LI><A href="https://2016.igem.org/Team:ETH_Zurich/HP/Gold">Gold</A></LI></DIV><DIV class="content_wrapper" id="main_wrapper"><P><UL class="menu" id="outline"><LI class="outline_title"><A href="#globalWrapper">SWITCH MODULE</A></LI><LI class="outline_item"><A href="#overview">Overview</A></LI><LI class="outline_item"><A href="#goals">Goals</A></LI><LI class="outline_item"><A href="#model">Model</A></LI><LI class="outline_item"><A href="#results">Results</A></LI><LI class="outline_item"><A href="#references">References</A></LI></UL></P><DIV class="sec page_title" id="top"><DIV><H1>SWITCH MODULE</H1></DIV></DIV><DIV class="sec white" id="overview"><DIV><H2>OVERVIEW</H2><DIV class="image_box" style="max-width: 400px;"><P><B>Figure 1:</B> The memory element of our circuit: the switch module.</P></DIV><P>The switch is the memory element of our circuit. We designed and tested different variants which are all based on plasmid editing (integrase-based switches) or of the chromosome (CRISPR-based switches).</P><P>In general switches are binary elements. However, in biological systems switching doesn't happen at the same time on different plasmids and in different cells because of cell-to-cell variability. We exploit this property to obtain quantitative outputs by relating the measured fluorescence to the exposure time and concentration of the sensed molecules.</P><P>A key concern during the design of the system was the leakiness of the AND gate in the sensor module. We use the model to tune our switch in order to minimize the risk of false positives and optimize the kinetics for the expected exposure time. Moreover the model allowed us to identify a flaw in the original design of the circuit and validate the improved version.</P></DIV></DIV><DIV class="sec white three_columns paddingless"><DIV class="quicklinks"><DIV><UL><LI class="outline_item"><A href="#goals">GOALS</A></LI></UL></DIV><DIV><UL><LI class="outline_item"><A href="#model">MODEL</A></LI></UL></DIV><DIV style="padding-bottom:12px;"><UL><LI class="outline_item"><A href="#results">RESULTS</A></LI></UL></DIV></DIV></DIV><DIV class="sec light_grey title_only" id="goals"><DIV><H2>GOALS</H2></DIV></DIV><DIV class="sec light_grey two_columns content_only"><DIV><DIV><UL><LI>To provide a proof that the switch can work as expected.</LI><LI>To assist the design of the switch.</LI></UL></DIV><DIV><UL><LI>To optimize the parameters for the requirements of our application.</LI><LI>To model the kinetics of the integrase.</LI></UL></DIV></DIV></DIV><DIV class="sec white" id="model"><DIV><H2>MODEL</H2><DIV class="image_box"><P><B>Figure 2:</B> Integrase switch design, the same design applies to Bxb1, Tp901 and PhiC31. Click to enlarge.</P></DIV><P>The switch is based on the integrase family of recombinases. On this page we focus on the Bxb1 integrase, but the same model and analysis applies to Tp901 and PhiC31.</P><P>According to literature<SUP><A href="#cit1" class="cit">1</A></SUP>, the mechanics of the integrase appear to be well known. After activation of the AND gate in the sensor module, Bxb1 is expressed and dimerizes. In its dimerized form, Bxb1 can bind to the <I>attB</I> and <I>attR</I> binding sites placed around the P<SUB>out</SUB> promoter. When both sites are occupied, synapsis between the two can happen, enabling flipping. </P><P>The flipping process changes the direction of the reporter promoter and transforms the <I>attB</I> and <I>attP</I> sites in <I>attL</I> and <I>attR</I>. At this point the switch is in the ON state, Bxb1 dimers can still bind to the <I>attL</I> and <I>attR</I> sites, but wihtout flipping DNA again.</P><P>The following section describes the species and reactions involved.</P></DIV></DIV><DIV class="sec white two_columns content_only"><DIV><DIV><H4>REACTIONS</H4>
                    \begin{align*}
                    1) &amp;&amp; P_{hyb} &amp; \rightarrow P_{hyb} + mRNA_{int} \\
                    2) &amp;&amp; mRNA_{int} &amp; \rightarrow mRNA_{int} + Bxb1 \\
                    3) &amp;&amp; Bxb1 + Bxb1 &amp; \rightleftharpoons DBxb1 \\
                    4) &amp;&amp; S_0 + DBxb1 &amp; \rightleftharpoons S_1 \\
                    5) &amp;&amp; S_1 + DBxb1 &amp; \rightleftharpoons S_2 \\
                    6) &amp;&amp; S_2 &amp; \rightarrow P_{switched} \\
                    7) &amp;&amp; attLR_0 + DBxb1 &amp; \rightleftharpoons attLR_1 \\
                    8) &amp;&amp; mRNA_{int} &amp; \rightarrow \\
                    9) &amp;&amp; Bxb1 &amp; \rightarrow \\
                    10) &amp;&amp; DBxb1 &amp; \rightarrow \\
                    \end{align*}
                </DIV><DIV><H4>SPECIES</H4><TABLE><TBODY><TR><TH>Name </TH><TH>Description </TH></TR><TR><TD>$P_{hyb}$</TD><TD>Active AND gate hybrid promoter</TD></TR><TR><TD>$mRNA_{int}$</TD><TD>mRNA of the integrase</TD></TR><TR><TD>$Bxb1$</TD><TD>Integrase protein</TD></TR><TR><TD>$DBxb1$</TD><TD>Dimerized form of the integrase protein</TD></TR><TR><TD>$S_{0}$</TD><TD>Plasmid with free <I>attB</I> and <I>attP</I> sites.</TD></TR><TR><TD>$S_{1}$</TD><TD>Plasmid with DBxb1 bound to the <I>attB</I> or <I>attP</I> site.</TD></TR><TR><TD>$S_{2}$</TD><TD>Plasmid with DBxb1 bound to both the <I>attB</I> and <I>attP</I> sites.</TD></TR><TR><TD>$P_{switched}$</TD><TD>Plasmid with switched reporter promoter.</TD></TR><TR><TD>$attLR_0$</TD><TD>Free <I>attL</I> or <I>attR</I> site of a switched plasmid.</TD></TR><TR><TD>$attLR_1$</TD><TD><I>attL</I> or <I>attR</I> site of a switched plasmid with DBxb1 bound. Can be expressed as $attLR_1=2\cdot P_{switched}-attLR_0$.</TD></TR></TBODY></TABLE></DIV></DIV></DIV><DIV class="sec white two_columns content_only"><DIV><DIV><H3>STOCHASTIC REACTION RATES:</H3><P>
                        \begin{align*}
                        1) \quad &amp; k_{mRNAint} \cdot P_{hyb}^{ON} \\
                        2) \quad &amp; k_{Bxb1} \cdot mRNA_{int} \\
                        3) \quad &amp; k_{DBxb1} \cdot Bxb1 \cdot (Bxb1-1) \\
                                 &amp; k_{-DBxb1} \cdot DBxb1\\
                        4) \quad &amp; 2 \cdot k_{attBP} \cdot S_0 \cdot DBxb1 \\
                                 &amp; k_{-attBP} \cdot S_1 \\
                        5) \quad &amp; k_{attBP} \cdot S_1 \cdot DBxb1 \\
                                 &amp; 2 \cdot k_{-attBP} \cdot S_2 \\
                        6) \quad &amp; k_{flip} \cdot S_2 \\
                        7) \quad &amp; k_{attLR} \cdot attLR_0 \cdot DBxb1 \\
                                 &amp; k_{-attLR} \cdot attLR_1 \\
                        8) \quad &amp; d_{mRNAint} \cdot mRNA_{int} \\
                        9) \quad &amp; d_{Bxb1} \cdot Bxb1 \\
                        10) \quad &amp; d_{DBxb1} \cdot DBxb1 \\
                        \end{align*}
                    </P></DIV><DIV><H3>PARAMETERS</H3><TABLE><TBODY><TR><TH>Name </TH><TH>Description </TH></TR><TR><TD>$P_{hyb}^{ON}$</TD><TD>Fraction of the maximal activity of the hybrid AND gate promoter. This value is has been taken from simulation results of the <A href="https://2016.igem.org/Team:ETH_Zurich/Sensor_Module">sensor module</A>.</TD></TR><TR><TD>$k_{mRNAint}$</TD><TD>Integrase mRNA transcription rate.</TD></TR><TR><TD>$k_{Bxb1}$</TD><TD>Bxb1 translation rate.</TD></TR><TR><TD>$k_{DBxb1}$</TD><TD>Bxb1 dimerization rate.</TD></TR><TR><TD>$k_{-DBxb1}$</TD><TD>DBxb1 dissociation rate.</TD></TR><TR><TD>$k_{attBP}$</TD><TD>Binding rate of DBxb1 to an <I>attB</I> or <I>attP</I> site.</TD></TR><TR><TD>$k_{-attBP}$</TD><TD>Dissociation rate of DBxb1 from an <I>attB</I> or <I>attP</I> site.</TD></TR><TR><TD>$k_{attLR}$</TD><TD>Binding rate of DBxb1 to an <I>attL</I> or <I>attR</I> site.</TD></TR><TR><TD>$k_{-attLR}$</TD><TD>Dissociation rate of DBxb1 from an <I>attL</I> or <I>attR</I> site.</TD></TR><TR><TD>$k_{flip}$</TD><TD>Flipping rate of a plasmid in the $S_2$ state.</TD></TR></TBODY></TABLE></DIV></DIV></DIV><DIV class="sec light_grey" id="results"><DIV><H2>RESULTS</H2><H3>STOCHASTIC MODEL OF THE INTEGRASE</H3><P>Based on the known mechanism of integrases we developed a novel stochastic model for integrases capturing the mechanistic details of the switching process. <B>Figure 5</B> shows an example simulation of the switching process. We planned to use the stochastic model for estimating the parameters of the model using the <A href="https://2016.igem.org/Team:ETH_Zurich/Parameters">INSIGHT</A> tool. Unfortunately the flow cytometry data of the recombinase couldn't be collected in time for performing the estimation before the wiki freeze but we try our best to present it at the Jamboree.</P><DIV class="image_box full_size"><P><B>Figure 3:</B> Stochastic simulation of the switch. The plots show the evolution of the four possible states of a slipping cassette. In this case 1000 trajectories (colored) were simulated. The mean behavior of the population is shown in black.</P></DIV><A name="design_improvements"><H3>DESIGN IMPROVEMENTS</H3></A><P>We used the model to suggest important changes to the initial design of the system. In the initial version of the system, the integrase gene was placed <I>inside</I> the flipping cassette. This choice was taken with the intention to reduce the effect of the leakiness: on the plasmids where the cassette with the gene is flipped, the integrase protein is not expressed anymore. We expected cells where leakiness causes partial switching to express less integrase and thus reduce further switching.</P><P>However, simulations of the system showed that this approach indeed reduces switching under leakiness conditions, but it also makes more difficult to reach complete switching when the AND gate is fully active. We proposed a different design in which the integrase gene and the flipping cassette are placed on different plasmids and compared its behavior with the initial design (see <B>Figure 4</B>) </P><DIV class="image_box full_size"><P><B>Figure 4:</B> Comparison of the behavior of two designs of the switch in leakiness and activation conditions. The new design is clearly more suitable for our system.</P></DIV><P><B>Figure 4</B> shows the differences in the qualitative bahavior. The plots describe the percentage of flipped cassettes over 6 hours. The simulation shows that the new design is slighlty more sensible to leakiness (0.2% over 6 hours) but takes the percentage of switched cassettes from 80% to 100%. Since 20% is an important difference in the signal, we discarded the initial design and implemented the new one.</P><A name="tuning_switch"><H3 id="tuning">TUNING FOR THE TARGET EXPOSURE TIME</H3></A><P>In the context of IBD investigation, we have specific requirements for our system. Informally, the aim is to minimize the chance of false positives due to leakiness and to optimize the switching time for the expected exposure time.</P><P>In order to determine which parameters of the system can be tuned for our purpose, we perform a sensitivity analysis of the percentage of switched reporter promoters towards the main parameters. This allows us to indentify the parameters that have the greatest influence on the final state of the system.</P><DIV class="image_box full_size"><P><B>Figure 5:</B> Sensitivity of the promoter switching with respect to the main parameters. The integrase RBS strength (k_Bxb1) and the strength of the integrase degradation tag (d_Bxb1, d_DBxb1) are biologically tunable parameters and have significant influence on the response of the switch to activation.</P></DIV><DIV class="image_box full_size"><P><B>Figure 6:</B> Sensitivity of the promoter switching with respect to the main parameters in leakiness conditions. The integrase RBS strength (k_Bxb1) and the strength of the integrase degradation tag (d_Bxb1, d_DBxb1) are biologically tunable parameters and have significant influence on the response of the switch to leakiness.</P></DIV><P>Figure 5 and 6 shows which parameters have the greatest influence on the final state of the system in response to both activation of the AND gate and leakiness. We decide to tune the integrase RBS strength (k_Bxb1) and the strength of the integrase degradation tag (d_Bxb1, d_DBxb1), since they are sensitive and biologically tunable.</P><H4>REQUIREMENTS</H4><P>An interesting feature of our switch is the fact that it's based on multiple plasmid copies, and can thus contain quantitative information. </P><OL><LI><P><I>When no inflammation and candidate markers are detected, the switch must remain in the off-state for at least 4 days in order to ensure no false positive detection later in the lab. The off-state is defined as &lt;20% of the reporter promoters are switched (on average).</I></P><P>This is necessary to minimize the effect of leakiness from the AND gate. We aim for a stability over 4 days based on available estimates of stool transit times in the human body<SUP><A href="#cit2" class="cit">2</A></SUP>.</P></LI><LI><P><I>The system must turn from the on- to the off-state for a cumulative exposure time of 8 h. This means, if during the conditioning phase the system is exposed to both inflammation and candidate markers for in total 8 h, we have full activation. The full activation on-state is defined as &gt;80% of the reporter promoters are switched (on average).</I></P><P>The intensity and position of the inflamations in the gut vary a lot between different patients and types of IBDs. However, based on published clinical studies, an exposure time of 8 h seems to be a reasonable average for defining the on-state<SUP><A href="#cit2" class="cit">2</A>, <A href="#cit3" class="cit">3</A></SUP>.</P></LI><LI><P><I>The response of the system must be as quick as possible.</I></P><P>In some cases there are many short inflammated regions<SUP><A href="#cit3" class="cit">3</A></SUP>. In order to detect and store the information on all of them, a quick response is desirable.</P></LI></OL><H4>RESULTING SPECIFICATIONS</H4><P>We assume that our sensor is only slightly leaky (1%) and that Bxb1 degrades at the same rate in monomeric and dimerized form. We identified constraints on the ratio $r=\frac{k_{Bxb1}}{d_{Bxb1}}$ between RBS strength and degradation rate respecting the first two requirements as follows:</P><OL><LI>We simulate the system for 4 days without applying any input signal (only leakiness causes switching). The first requirement is satisfied for $r \geq 0.08$.</LI><LI>We simulate the system for 4 days, the system is affected by leakiness and is exposed to strong inflammation (the sensor promoter is completely on) for 8 hours. The second requirement is satisfied for $r \leq 1.4$.</LI></OL><DIV class="image_box full_size"><P><B>Figure 7:</B><B>Left:</B> Functional regions of the parameter space satisfying the first two requirements. <B>Right:</B> The intersection of the two regions defines a range of parameter ratios that allows the switch to function as required.</P></DIV><P><B>Figure 7</B> shows the contrains and the final feasible region. This gives us a linear region that defines the feasible ratios $r$ for our switch.</P><P>Assuming that after exposure to inflammation and candidate markers, the integrase mRNA concentration quickly reaches the steady state and that we can express the steady state of Bxb1 as follow:</P><P>\begin{align*}
                    [Bxb1]_{SS} =  \frac{k_{Bxb1}\cdot [mRNA_{int}]}{d_{Bxb1}}\cdot\left(1-e^{-d_{Bxb1}\cdot t}\right)      
                \end{align*}</P><P>The equation above is the analytical solution of the ODEs for simple production-degradation systems; it tells us that the degradation rate is the only factor determining how quickly the steady state is reached. Since at higher concentrations of Bxb1 we have faster switching, we can satisfy the third requirement by maximizing the degradation rate.</P><P>Given the above results, we informed the experimentalists that an optimal system should have maximal degradation rate under the constraint that:
                \begin{align*}
                    0.08 \leq \frac{k_{Bxb1}}{d_{Bxb1}} \leq 1.4    
                \end{align*}
                Moreover we suggested to choose, between the different AND gate designs, the one with the higher fold-change activation. Given the shape of the feasibel parameter space this property is preferable over absolute activation.</P></DIV></DIV><DIV class="sec white" id="references"><DIV><H2>REFERENCES</H2><UL><LI>[1] Singh, Shweta, Pallavi Ghosh, and Graham F. Hatfull. &quot;Attachment site selection and identity in Bxb1 serine integrase-mediated site-specific recombination.&quot; <I>PLoS Genet 9.5</I> (2013): e1003490.</LI><LI>[2] Friend, David R. &quot;New oral delivery systems for treatment of inflammatory bowel disease.&quot; Advanced drug delivery reviews 57.2 (2005): 247-265.</LI><LI>[3] Rutgeerts, Paul, Gaston Vantrappen, and Karel Geboes. &quot;Endoscopy in inflammatory bowel disease.&quot; Scandinavian Journal of Gastroenterology 24.sup170 (1989): 12-15.</LI></UL></DIV></DIV><DIV class="sec blue"><DIV><H2> Thanks to the sponsors that supported our project: </H2></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></BODY></HTML>