<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_ETH_Zurich_LabResults skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><DIV id="top_title"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:ETH Zurich/LabResults</SPAN></H1></DIV><DIV id="HQ_page"><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><DIV class="title_container"><SPAN id="title_text"><A href="https://2016.igem.org/Team:ETH_Zurich">PAVLOV'S COLI</A></SPAN></DIV><DIV class="menu_container" id="main_menu"><UL class="menu" id="menu"><LI class="menu_item"><A href="https://2016.igem.org/Team:ETH_Zurich">HOME</A></LI><LI class="menu_item"><A>TEAM</A></LI><LI><A href="https://2016.igem.org/Team:ETH_Zurich/Team">Team</A></LI><LI><A href="https://2016.igem.org/Team:ETH_Zurich/Collaborations">Collaborations</A></LI><LI><A href="https://2016.igem.org/Team:ETH_Zurich/Attributions">Attributions</A></LI></UL><LI class="menu_item"><A>PROJECT</A></LI><LI><A href="https://2016.igem.org/Team:ETH_Zurich/Description">Description</A></LI><LI><A href="https://2016.igem.org/Team:ETH_Zurich/Design">Design</A></LI><LI><A href="https://2016.igem.org/Team:ETH_Zurich/Proof">Proof of Concept</A></LI><LI><A href="https://2016.igem.org/Team:ETH_Zurich/Demonstrate">Demonstrate</A></LI><LI class="menu_item"><A>MODELING</A></LI><LI><A href="https://2016.igem.org/Team:ETH_Zurich/Model">Model</A></LI><LI><A href="https://2016.igem.org/Team:ETH_Zurich/Sensor_Module">Sensor Module</A></LI><LI><A href="https://2016.igem.org/Team:ETH_Zurich/Switch_Module">Switch Module</A></LI><LI><A href="https://2016.igem.org/Team:ETH_Zurich/Detector_Module">Reporter Module</A></LI><LI><A href="https://2016.igem.org/Team:ETH_Zurich/NO_Release">NO Release</A></LI><LI><A href="https://2016.igem.org/Team:ETH_Zurich/Parameters">Parameters</A></LI><LI class="menu_item"><A>LAB</A></LI><LI><A href="https://2016.igem.org/Team:ETH_Zurich/Experiments">Experiments</A></LI><LI><A href="https://2016.igem.org/Team:ETH_Zurich/LabResults">Lab Results</A></LI><LI><A href="https://2016.igem.org/Team:ETH_Zurich/InterLab">Interlab Study</A></LI><LI><A href="https://2016.igem.org/Team:ETH_Zurich/Safety">Safety</A></LI><LI><A href="https://2016.igem.org/Team:ETH_Zurich/Hardware">Hardware</A></LI><LI><A href="https://2016.igem.org/Team:ETH_Zurich/Notebook">Notebook</A></LI><LI class="menu_item"><A>PARTS</A></LI><LI><A href="https://2016.igem.org/Team:ETH_Zurich/Parts">Parts</A></LI><LI><A href="https://2016.igem.org/Team:ETH_Zurich/Basic_Part">Basic Parts</A></LI><LI><A href="https://2016.igem.org/Team:ETH_Zurich/Composite_Part">Composite Parts</A></LI><LI><A href="https://2016.igem.org/Team:ETH_Zurich/Part_Collection">Part Collection</A></LI><LI class="menu_item"><A>HUMAN PRACTICES</A></LI><LI><A href="https://2016.igem.org/Team:ETH_Zurich/Human_Practices">Human Practices</A></LI><LI><A href="https://2016.igem.org/Team:ETH_Zurich/HP/Silver">Silver</A></LI><LI><A href="https://2016.igem.org/Team:ETH_Zurich/HP/Gold">Gold</A></LI></DIV><DIV class="content_wrapper" id="main_wrapper"><UL class="menu" id="outline"><LI class="outline_item"><A href="#test1A">Test 1A</A></LI><LI class="outline_item"><A href="#test1B">Test 1B</A></LI><LI class="outline_item"><A href="#test2">Test 2</A></LI><LI class="outline_item"><A href="#test3">Test 3</A></LI><LI class="outline_item"><A href="#test4">Test 4</A></LI><LI class="outline_item"><A href="#test5">Test 5</A></LI><LI class="outline_item"><A href="#directed_evolution">Directed Evolution</A></LI><LI class="outline_item"><A href="#full_system">Full System</A></LI></UL><DIV class="sec page_title"><DIV><H1>LAB RESULTS</H1></DIV></DIV><DIV class="sec white" id="introduction"><DIV><P>
Our fellow iGEM colleagues will surely agree with us when we say that our precious bacteria are very stubborn and mischievous and that they don't always want to behave like we would want them to. It takes a group of very stubborn and cunning people to trick them into doing what we want. Last few months were a tug-of-war between experimentalists and Pavlov's coli. We definitely got pulled on several occasions. But we are a stubborn bunch of people, so let us present how we pulled back and got as far as we did.
</P><P>
This section provides insight into the wet lab part of our project and shows the results we got from the experiments in more detail. 
</P><P>
We divided our project into several tests. This enabled us a systematic approach to building our genetic circuit. That way we were easily able to find and troubleshoot the issues we encountered during our project.
</P><P>
Tests:
<UL><LI>Test 1A:
Construction and testing of the nitric oxide sensor</LI><LI>Test 1B:
Construction and testing of the AHL sensor</LI><LI>Test 2:
Construction and testing of the AHL+NO AND gate</LI><LI>Test 3:
Construction, testing and improving of the recombinases</LI><LI>Test 4:
Construction and testing of the lactate+NO AND gate
</LI><LI>Test 5:
Construction of the switch based on Cpf1.
</LI><LI>Directed evolution</LI></UL></P></DIV></DIV><DIV class="sec light_grey" id="test1A"><DIV><H2>Construction of nitric oxide sensor: NorV Promoter</H2><P>
Our initial design was focused on building a nitric oxide sensor which consists of a nitric oxide responsive promoter (pNorV) and the corresponding regulator NorR. We were happy that we were able to construct the plasmids very early on in the project. Unfortunately sequencing results for norR showed a relatively big deletion inside the norR gene. Nevertheless, we performed a preliminary experiment with our construct. To our surprise we saw a significant induction of pNorV promoter even with low doses of inducer DETA/NO. This lead us to hypothesize that a genomic version of norR is responsible for the activation our promoter. We proved the hypothesis by transforming and studying our system in a norR knockout E. coli strain (Keio NorR knockout strain).

</P><DIV class="image_box full_size" style="max-width:500px;"><P><B>Figure 1:</B> PnorV dose response curve for a very low range of DETA/NO concentrations. Samples where norR was not present in the cell also got induced when the DETA/NO was added
			</P></DIV><DIV class="image_box full_size" style="max-width:500px;"><P><B>Figure 2:</B> Figure shows that we are able to get an induction of pNorV in the wild type Keio strain where genomic copy of norR gene was present. There was no induction of pNorV in the norR knock-out strain
			</P></DIV><P>
We were able to build:
<UL><LI>functional pNorV promoter</LI></UL>
We were not able to build:
<UL><LI>norR regulator of pNorV</LI></UL>
However, we were able to prove that a genomic copy of norR is sufficient for an induction of pNorV
</P></DIV></DIV><DIV class="sec white" id="test1B"><DIV><H2>Construction of an AHL sensor: esabox/EsaR system</H2><P>
During our brainstorming sessions in the months before &quot;hitting the lab&quot; we put special focus on trying to find a system for an AHL sensor which would be based on a repressor rather than activator. We estimated this would allow us to build a better AND gate. We found the esabox/EsaR system. Esaboxes are binding sites for EsaR. We saw a potential to create many different combinations of hybrid promoters by placing esaboxes on different places and in different numbers to the promoter region. However, due to the strong secondary structure and repetitive sequences, esaboxes proved to be a challenge to create. We tried several different approaches and at the end we managed to construct five different combinations of synthetic promoters with esaboxes in different numbers and different spacings. We additionally managed to create more than 10 different AND gate promoters with esaboxes and pNorV. In total we created almost 20 different promoters just by varying the location and number of esaboxes in the promoter region. We were able to observe what is the effect of spacers and number of esaboxes on the behavior of the promoters.
</P><P>
When we started with testing phase of our esabox promoters we observed that while we can successfully repress transcription with our EsaR repressor, we cannot successfully release the repressor from the promoter. 
</P><P>
We were able to build:
<UL><LI>five different promoters with different combinations of esaboxes as roadblocks</LI></UL>
We were not able to build:
<UL><LI>EsaR repressor which can get released from promoter upon induction</LI></UL></P></DIV></DIV><DIV class="sec light_grey" id="test2"><DIV><H2>Construction of an AND gate</H2><P> We were able to build a library of over 10 different hybrid promoters for an AND gate. These AND gates build our <A href="https://2016.igem.org/Team:ETH_Zurich/Part_Collection">Part Collection</A>. We chose the AND gate design with <A href="http://parts.igem.org/wiki/index.php?title=Part:BBa_K2116011">the best AND gate</A> logic behavior and use it in the construction of the full circuit.
</P><DIV class="image_box full_size" style="max-width:500px;"><P><B>Figure 3:</B> Image shows examples of different hybrid promoters we constructed. The designed different in the location of the esaboxes (and whether the location caused steric or competitive inhibition of the pNorV promoter), the number of esaboxes and in the size of spacing between the esaboxes. The design shown up most left showed the best AND gate logic behavior.
			</P></DIV><DIV class="image_box full_size" style="max-width:500px;"><P><B>Figure 4:</B> The design with the best AND gate logic (up most left in the previous figure) was characterized further. Figure shows a 2D dose response of the promoter with respect to the doses NO and AHL.
			</P></DIV></DIV></DIV><DIV class="sec white" id="test3"><DIV><H2>Construction of a recombinase based switch</H2><P>
 Permanent memory is an important aspect of our project. The most obvious way of achieving permanent memory in cells is by changing their DNA. Recombinases provide an elegant way to create such uni-directional switch.They flip a sequence flanked between two recombinase recognition sites. However, it is important that we consider the dynamics of recombinases when we try to build such switch. Intuition often fails when it comes to estimating the dynamics of a design or a system. This is where a collaboration between theory and experiments plays an important role. In our project, the interaction between theory and experiments has been important since the beginning in the construction a recombinase based switch. In their quest to minimize the potential leakiness of our system,<A href="https://2016.igem.org/Team:ETH_Zurich/Switch_Module"> modelers discovered a better implementation of our idea</A>. That means that after we already spent a month in the lab designing and building our system,they provided us with important simulations of our system and offered a novel design which we immediately started to build. On the other hand, modelers needed rigorous experimental data to be able to characterize recombinases and their dynamics. This would again in turn help experimentalists create a better design. In our project we studied dynamics of two different recombinases, bxb1 and tp901 by creating reporters for the respective recombinases and perform rigorous experiment about their dynamics. We studied properties of three versions with different degradation tags for each of the recombinases. We contributed them to the iGEM depository (<A href="http://parts.igem.org/wiki/index.php?title=Part:BBa_K2116056">BBa_K2116056</A>, <A href="http://parts.igem.org/wiki/index.php?title=Part:BBa_K2116057">BBa_K2116057</A>, <A href="http://parts.igem.org/wiki/index.php?title=Part:BBa_K2116061">BBa_K2116061</A>, <A href="http://parts.igem.org/wiki/index.php?title=Part:BBa_K2116062">BBa_K2116062</A>, <A href="http://parts.igem.org/wiki/index.php?title=Part:BBa_K2116063">BBa_K2116063</A> and <A href="http://parts.igem.org/wiki/index.php?title=Part:BBa_K2116064">BBa_K2116064</A>). We were able to show the flipping rates for different recombinases and attempted to improve them.
</P><DIV class="image_box full_size" style="max-width:500px;"><P><B>Figure 5:</B> Image shows the flipping rate of the bxb1 before the improved RBS. We transformed a recombinase under Tet promoter together with a plasmid containing promoter J23118 flanked with bxb1 binding sites, flipped in the opposite direction of sfgfp. The positive control was a cell containing sfGFP with promoter J23118 facing the direction of the gene. After overnight exposure to the inducer tetracyclin, we observed a 7-fold change in the flourescence. However, in comparison to the positive control, we did not achieve to flip majority of promoters. 
			</P></DIV><DIV class="image_box full_size" style="max-width:500px;"><P><B>Figure 6:</B> Image shows the flipping rate of the bxb1 after improving the RBS. After overnight exposure to the inducer, we observed a 30-fold change in the flourescence. In comparison to the positive control, we did not achieve to flip majority of promoters, however, the level of flipped promoters was much higher after the change of RBS.
			</P></DIV><DIV class="image_box full_size" style="max-width:500px;"><P><B>Figure 7:</B> FACS experiments show improved dynamics of bxb1 upon changing the RBS. In red we see the switch dynamics of bxb1 before the change of RBS. In blue we see the improved switch dynamics of bxb1.
		</P></DIV><P>
We were able to construct reporters for recombinases with which we were also able to show the differences in dynamics of recombinases when different degradation tags were added.
</P></DIV></DIV><DIV class="sec light_grey" id="test4"><DIV><H2>Construction of the lactate + NO AND gate</H2><P>
Another important aspect of our idea is that by implementing associative learning in the circuit, we are  provided with the flexibility to detect several different markers. In the scope of our project we tried to demonstrate that there is potential to expand markers to other metabolites, which are considered important in IBD. One example of such metabolite is lactate, which is a product of microbial metabolism and which also gets utilized by many species of bacteria in the gut. We to demonstrate multiplexing, constructed a <A href="http://parts.igem.org/wiki/index.php?title=Part:BBa_K2116027">lactate + NO AND gate</A>.
</P><DIV class="image_box full_size" style="max-width:500px;"><P><B>Figure 8:</B> The 2D dose response for the lactate and DETA/NO show that the hybrid promoter exhibits an AND logic behavior.
			</P></DIV><P>
The hybrid promoter shows an AND gate behavior. However, due to the low fold change (1.5), the design needs to be improved in the future.
</P></DIV></DIV><DIV class="sec white" id="test5"><DIV><H2>Construction of the Cpf1 based switch </H2><P>
We created an alternative switch, which is not based on recombinases but on 
<A href="https://2016.igem.org/Team:ETH_Zurich/Description">Cpf1</A>. In the system, Cpf1 would replace the recombinase downstram of the AND gate promoter. In addition there is a sfgfp gene which has inserted mNectarine gene. Upon actvation of the AND gate, Cpf1 would excise mNectarine from the sfgfp. This would enable green fluorescence instead of red after induction. We tested whether Cpf1 is able to excise mNectarine from sfgfp. Cpf1 was put under tet inducible promoter. After induction, we were able to see a dose response, where we observe more green fluorescence if more tetracyclin was added to the media. 
</P><DIV class="image_box full_size" style="max-width:500px;"><P><B>Figure 9:</B> The dose response show that Cpf1 can excise mNectarine from sfgfp. From left to right are increasingly higher doses of tetracyclin (0, 2000, 5000 ng/ml) and a positive control. When tetracycline induces expression of Cpf1, we observe green fluorescence.
		</P></DIV></DIV></DIV><DIV class="sec light_grey" id="directed_evolution"><DIV><H2>Directed evolution</H2><P>
We used directed evolution to try and evolve EsaR to change specificity to a different AHL. Unfortunately esaR turned out to be a tough nut to crack for us this year. However, we were able to show that the dual selector system is a suitable selection tool if the repressor is strong enough. 
</P><H3>Dual Selector System: CAT-UPRT Fusion Protein </H3><H4>Assay:</H4><P>The response of the dual selector plasmid containing a CAT-UPRT fusion protein (CAT: chloramphenicol acetyltransferase, UPRT: uracil phosphoribosyltransferase) towards 5-fluorouracil was tested. The Keio-Collection strain JW2483-1 (δupp) was transformed with a plasmid constitutively expressing the fusion protein.
        The cells were grown overnight in M9 medium, diluted to OD=0.1 and transferred into a flat bottom 96-well plate. They were grown for another three hours and finally different concentrations of 5-fluorouracil and chloramphenicol were added (10 µl into 190 µl of bacterial culture, final concentration are shown.) 
        </P><H4>Summary:</H4><P><A href="http://parts.igem.org/wiki/index.php?title=Part:BBa_K2116053">BBa_K2116053</A>
Whereas cells expressing the fusion protein are resistant against chloramphenicol, the are especially sensitive towards 5-fluorouracil.
The cells lacking the plasmid as well as do not have a chromosomal copy of <I>upp</I> are less inhibited but are sensitive towards chloramphenicol.</P><H4>FACS-Plots:</H4><DIV class="image_box full_size"><P>Different kind of cells were analyzed by FACS:
<UL><LI>Left: non-transformed cells
</LI><LI>Middle: GFP-plasmid under control of EsaBox, co-transformed with EsaR plasmid
</LI><LI>Right: GFP-plasmid under control of EsaBox, no repressor
</LI></UL></P></DIV><P>FACS-plots show clearly that EsaR is leaky. Due to this, it is not suitable to undergo a negative selection process with 5-fluorouracil.
</P></DIV></DIV><DIV class="sec white" id="full_system"><DIV><H2>Full System</H2><P>
At the very end of the project, in the last week before the giant jamboree we were able to construct and test the full system. We transformed the cells with three plasmids:
<UL><LI>plasmid containing EsaR</LI><LI>plasmid with AHL + NO AND gate with bxb1 recombinase downstream</LI><LI>plasmid with J23118 promoter flanked with bxb1 recognition sites and sfgfp downstream</LI></UL></P><P> We induced the cells with AHL and NO. We observed that the output of our system resembled the AND gate logic. However, further experiments are required to improve the fold change and behavior of the circuit. 
</P><DIV class="image_box full_size" style="max-width:500px;"><P><B>Figure 10:</B> The output of the full system resembles an AND gate behavior upon induction.
			</P></DIV></DIV></DIV><DIV class="sec blue"><DIV><H2> Thanks to the sponsors that supported our project: </H2></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></BODY></HTML>