<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_ETH_Zurich_Sensor_Module skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><DIV id="top_title"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:ETH Zurich/Sensor Module</SPAN></H1></DIV><DIV id="HQ_page"><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><DIV class="title_container"><SPAN id="title_text"><A href="https://2016.igem.org/Team:ETH_Zurich">PAVLOV'S COLI</A></SPAN></DIV><DIV class="menu_container" id="main_menu"><UL class="menu" id="menu"><LI class="menu_item"><A href="https://2016.igem.org/Team:ETH_Zurich">HOME</A></LI><LI class="menu_item"><A>TEAM</A></LI><LI><A href="https://2016.igem.org/Team:ETH_Zurich/Team">Team</A></LI><LI><A href="https://2016.igem.org/Team:ETH_Zurich/Collaborations">Collaborations</A></LI><LI><A href="https://2016.igem.org/Team:ETH_Zurich/Attributions">Attributions</A></LI></UL><LI class="menu_item"><A>PROJECT</A></LI><LI><A href="https://2016.igem.org/Team:ETH_Zurich/Description">Description</A></LI><LI><A href="https://2016.igem.org/Team:ETH_Zurich/Design">Design</A></LI><LI><A href="https://2016.igem.org/Team:ETH_Zurich/Proof">Proof of Concept</A></LI><LI><A href="https://2016.igem.org/Team:ETH_Zurich/Demonstrate">Demonstrate</A></LI><LI class="menu_item"><A>MODELING</A></LI><LI><A href="https://2016.igem.org/Team:ETH_Zurich/Model">Model</A></LI><LI><A href="https://2016.igem.org/Team:ETH_Zurich/Sensor_Module">Sensor Module</A></LI><LI><A href="https://2016.igem.org/Team:ETH_Zurich/Switch_Module">Switch Module</A></LI><LI><A href="https://2016.igem.org/Team:ETH_Zurich/Detector_Module">Reporter Module</A></LI><LI><A href="https://2016.igem.org/Team:ETH_Zurich/NO_Release">NO Release</A></LI><LI><A href="https://2016.igem.org/Team:ETH_Zurich/Parameters">Parameters</A></LI><LI class="menu_item"><A>LAB</A></LI><LI><A href="https://2016.igem.org/Team:ETH_Zurich/Experiments">Experiments</A></LI><LI><A href="https://2016.igem.org/Team:ETH_Zurich/LabResults">Lab Results</A></LI><LI><A href="https://2016.igem.org/Team:ETH_Zurich/InterLab">Interlab Study</A></LI><LI><A href="https://2016.igem.org/Team:ETH_Zurich/Safety">Safety</A></LI><LI><A href="https://2016.igem.org/Team:ETH_Zurich/Hardware">Hardware</A></LI><LI><A href="https://2016.igem.org/Team:ETH_Zurich/Notebook">Notebook</A></LI><LI class="menu_item"><A>PARTS</A></LI><LI><A href="https://2016.igem.org/Team:ETH_Zurich/Parts">Parts</A></LI><LI><A href="https://2016.igem.org/Team:ETH_Zurich/Basic_Part">Basic Parts</A></LI><LI><A href="https://2016.igem.org/Team:ETH_Zurich/Composite_Part">Composite Parts</A></LI><LI><A href="https://2016.igem.org/Team:ETH_Zurich/Part_Collection">Part Collection</A></LI><LI class="menu_item"><A>HUMAN PRACTICES</A></LI><LI><A href="https://2016.igem.org/Team:ETH_Zurich/Human_Practices">Human Practices</A></LI><LI><A href="https://2016.igem.org/Team:ETH_Zurich/HP/Silver">Silver</A></LI><LI><A href="https://2016.igem.org/Team:ETH_Zurich/HP/Gold">Gold</A></LI></DIV><DIV class="content_wrapper" id="main_wrapper"><UL class="menu" id="outline"><LI class="outline_item"><A href="#andgate">And Gate</A></LI><LI class="outline_item"><A href="#nosensor">NO Sensor</A></LI><LI class="outline_item"><A href="#ahlsensor">AHL Sensor</A></LI><LI class="outline_item"><A href="#lactsensor">Lactate Sensor</A></LI></UL><DIV class="sec page_title"><DIV><H1>SENSOR MODULE</H1></DIV></DIV><DIV class="sec white" id="andgate"><DIV><H2>OVERVIEW</H2><P>
                Our idea was to identify bowel infection and its possible causes based on the intestine level of nitric oxyde (NO), which
                is infection specific, and of N-acyl homoserine-lactone (AHL), which is microbiota specific. Thus, the main goal was to detect the simultaneous presence of those two chemicals in an abnormal amount.
            </P><P>
                Additionally, we found out in our interview with Prof. Christophe Lacroix that lactate is also a molecule of interest in inflammatory bowel disease (IBD) research: lactate plays an important role in microbiome metabolism and recent studies suggest its presence in high amounts in certain cases of severe IBD.
            </P><P>
                For this reasons, we decided that two type of sensors are interesting to develop in order to investigate the causes of IBD: One sensor to associate the presence of both AHL and NO, and another sensor to associate lactate
                and NO.
            </P><P>summary
			</P><H3>SENSOR MODULE</H3><DIV class="image_box full_size"><P><B>Figure 2:</B> Two alternative designs for the sensor module. <B>Left:</B> A sensor module that associates the simultaneous presence of the inflammatory marker nitric oxide (NO) and the microbiotic marker AHL. <B>Right:</B> Association of NO with the microbiotic marker lactate.</P></DIV></DIV></DIV><DIV class="sec light_grey"><DIV><H3>GOALS</H3><UL><LI>To get an overall overview of the behavior and to compute a dose response curve.</LI><LI>To identify how the biological design may influence the the behavior of our system.</LI><LI>To identify sensitive parameters that can be tuned.</LI><LI>To compare alternative designs.</LI></UL></DIV></DIV><DIV class="sec white" id="nosensor"><DIV><DIV class="image_box" style="max-width: 500px;"><P><B>Figure 3:</B> NorR overview.</P></DIV><DIV class="sec white"><H2>NITRIC OXIDE SYSTEM</H2><P>
                    In the absence of NO, NorR is produced constitutively and binds to the PnorV promoter, which leads to a repression of the gene transcription.
                    When NO is present in the medium, it binds cooperatively to the hexameric form of NorR and activates the promoter.
                </P></DIV><DIV class="sec white"><H3>ASSUMPTIONS</H3><P> We assume here that the binding of NorR to the promoter PnorV does not affect the binding between NO and NorR. Thus, the reactions \begin{align*} NorR+NO&amp;\rightleftharpoons NorR_{NO}\\ \end{align*} and \begin{align*}
                    PnorV_{NorR}+NO&amp;\rightleftharpoons PnorV1\\ \end{align*} have the same reaction rate (PnorV1 is the complex consisting of PnorV, NO and NorR). Under this
                    assumption, the system of equation can be simplified as follows:
                </P></DIV></DIV></DIV><DIV class="sec white two_columns"><DIV><H4>REACTIONS</H4><DIV><P>NOrR SYSTEM:</P>

                \begin{align*} &amp;\rightarrow NorR\\ NO+NorR&amp;\rightleftharpoons NorR_{NO}\\ 2NorR_{NO}&amp;\rightleftharpoons DNorR_{NO2}\\
                2NorR &amp;\rightleftharpoons DNorR\\ DNorR+NO&amp;\rightleftharpoons DNorR_{NO1}\\ DNorR_{NO1}+NO&amp;\rightleftharpoons
                DNorR_{NO2}\\ DNorR_{NO2}+PnorV0&amp;\rightleftharpoons PnorV1\\ DNorR_{NO2}+PnorV1&amp;\rightleftharpoons
                PnorV2\\ DNorR_{NO2}+PnorV2&amp;\rightleftharpoons PnorV3\\ PnorV3&amp;\rightarrow mRNA_{Bxb1}\\ NorR&amp;\rightarrow\\
                DNorR&amp;\rightarrow \\ DNorR_{NO1}&amp;\rightarrow\\ DNorR_{NO2}&amp;\rightarrow\\ NorR_{NO}&amp;\rightarrow\\
                mRNA_{Bxb1}&amp;\rightarrow\\ \end{align*}

            </DIV><DIV><TABLE><TBODY><TR><TH>Species </TH><TH>Description </TH></TR><TR><TD>NO</TD><TD> Nitric Oxyde produced from DETA/NO reaction </TD></TR><TR><TD>NorR </TD><TD>NorR constitutively produced inside  <I>E. coli </I>  cells </TD></TR><TR><TD>NorR
                            <SUB>NO</SUB></TD><TD>NorR with one NO molecule bound</TD></TR><TR><TD>DNorR</TD><TD>NorR dimer, regulatory protein PnorV operon</TD></TR><TR><TD>DNorR
                            <SUB>NO1</SUB></TD><TD>NorR dimer with one NO molecule bound</TD></TR><TR><TD>DNorR
                            <SUB>NO2</SUB></TD><TD>NorR dimer with two NO molecules bound</TD></TR><TR><TD>PnorV
                            <SUB>i</SUB></TD><TD>PnorV promoter with i sites occupied by DNoR
                            <SUB>NO2</SUB></TD></TR><TR><TD>PnorV
                            <SUB>3</SUB></TD><TD>Active promoter, PnorV promoter with 3 sites occupied by DNoR
                            <SUB>NO2</SUB></TD></TR></TBODY></TABLE><DIV class="quicklinks"><DIV><DIV class="outline_item">For parameter values, please check <A href="https://2016.igem.org/Team:ETH_Zurich/Parameters">PARAMETERS</A></DIV></DIV></DIV></DIV></DIV></DIV><DIV class="sec white"><DIV><H3>RESULTS</H3><P>
               The sensor module must be able detect the different species with high specificity and a sensitivity that lies in the physiological concentration range. In this section we will 
explain how the we applied our model to provide useful insights for the biological implementation of the system. 
            </P></DIV></DIV><DIV class="sec white"><DIV><A name="NO_sensor_requirements"><H4>REQUIREMENTS</H4></A><P><UL><LI>NO sensor sensitivity range = [2 uM - 200 uM]</LI><LI>The system must be as fast as possible.</LI></UL></P><H5>KEY IDEA</H5><P>
Ideally, we need a dose response curve alignment for the sensor activation and the activation of the hybrid promoter. This ensures that the information on the inflammatory and candidate markers propagates to the switch and finally to the reporter. Based on flow cytometry data, the level of inflammation can 
then be inferred.
            </P><H5>PARAMETERS OF INTEREST</H5><P><UL><LI>Production rate NorR </LI><LI>Degradatioin rate of NorR</LI></UL></P><P>
            These parameters are tunable and will allow us to tune the kinetics and the steady-state concentration of NorR in the system.
            </P></DIV></DIV><DIV class="sec white"><DIV><H5>PARAMETER TUNING</H5><P> We aim for a medium promoter activation (20-60% activation) for NO concentrations around 2 uM and high promoter activation (&gt;70% activation) for NO concentrations around 200 uM. Since the NorR production and degradation rates are the tunable parameters, we investigated their impact on the promoter activation at [NO] = 2 uM and [NO] = 200 uM. The model shows that the ratio between the degradation and production rate needs to lie between 0.65 - 1.20 (see <B> Fig. 3 </B>) and that the degradation rate needs to be faster than 2.5 nM/min (see <B> Fig. 4 </B>).
			</P></DIV></DIV><DIV class="sec white two_columns"><DIV><DIV><DIV><DIV class="image_box full_size"><P><B>Figure 4:</B> This simulation was run with an input concentration of NO = 2 uM, the ideal lower limit of the sensor's dynamic. As explained before, we would like to achieve dose response alignment between the sensor and the hybrid promoter, in order to ensure optimal information transmission through the genetic circuit. In order to avoid false negative results during the conditioning phase, we aim for an activation between 20% and 60% at [NO] = 2 uM. To achieve this, the model suggests that we need to keep the ratio of the degradation and production rate between 0.65 - 1.20. </P></DIV></DIV></DIV><DIV><DIV><DIV class="image_box full_size"><P><B>Figure 5:</B> This simulation was run with an input concentration of NO = 200 uM, the ideal upper limit of the sensor's dynamic range. At this
					point we want a full activation of the sensor, which means at least 70% activation of the promoter. To achieve this, the model suggests that the NorR production rate needs to be faster than 2.5 nM/min. </P></DIV></DIV></DIV></DIV></DIV><DIV class="sec white"><DIV><DIV><H4>DOSE RESPONSE</H4><P>
				From the simulated dose response curves <B>Fig. 6</B>, one can roughly estimate the strength of the appropriate promoter to set the required limit of detection and dynamic range.
An easy way to visualize how the lower limit of detection is influenced by the production rate of NorR, we plotted the maximum of the derivative of the dose response curve, as a function of the production rate (fig. 6). 
This maximum corresponds to the rising of the sigmoid, therefore it corresponds to the limit of detection.
The correct production rate here would be, according to <B>Fig. 6</B> calibration curve, 3 nM. We plotted the concentration of NorR, for this production rate <B>(Fig. 7)</B>.
The concentration at steady state is around 140 nM, which roughly corresponds to the E. Coli's native NorR concentration.  
				</P></DIV><DIV class="image_box full_size"><P><B>Figure 6:</B> Dose response of the NO sensor for different NorR production rates. For each production rate, the degradation rate has been adjusted such that it meets the criteria identified in <B>Fig. 3</B> and <B>Fig 4</B>.</P></DIV></DIV></DIV><DIV class="sec white two_columns"><DIV><DIV><DIV><DIV class="image_box full_size"><P><B>Figure 7:</B>Plotting the concentration corresponding to the maximum of the derivative of the previous dose response curve
					we compute the limit of detection of the system as a function of the transcription rate, assuming a degradation rate respecting 
					the previous ratio constrain.</P></DIV></DIV></DIV><DIV><DIV><DIV class="image_box full_size"><P><B>Figure 8:</B>With regards to all the previous simulation it appears that a promoter strength of 3 nM for example is enough to see a 4 fold promoter activation under 
					200 uM of NO system stimulation. We wanted to determine which concentration of the NO species this promoter strength would represent. It appears that the NOrR concnetration
					remains quite low and similar to the concentration of native NorR in the E.Coli [1]. In order to make the circuit as easy to implement as possible. It was suggested to thus 
					only use the native NorR naturally present in the cell. This would simplify the circuit to reducing the amount of sequence to inject inside the plasmids.</P></DIV></DIV></DIV></DIV></DIV><DIV class="sec white"><DIV><P>In order to provide the biologists more accurate information for an efficient system tuning, we decided to estimate the parameters of the 
		real system. As the NO sensor already worked pretty well and shows a nice behaviour on the plate reader tests, we decided to fit the parameters of our model based on one of those plate reader 
		experiments. </P></DIV></DIV><DIV class="sec white"><DIV><H3>PARAMETER ESTIMATION</H3><P>In order to provide the biologists more accurate information for an efficient system tuning, we decided to estimate the parameters of the 
		real system. As the NO sensor already worked pretty well and shows a nice behaviour on the plate reader tests, we decided to fit the parameters of our model based on one of those plate reader 
		experiments. 
						</P></DIV></DIV><DIV class="sec white two_columns"><DIV><DIV><DIV><DIV class="image_box full_size"><P><B>Figure 9:</B>curve fitting for the NO sensor. Each curve correspond to a time response to a certain concentration of nitric oxide induction. we used MEIGO for the fitting. MEIGO uses metaheuristic and Bayesian methods to fit data to a system of differential equations. </P></DIV></DIV></DIV><DIV><DIV><DIV class="image_box full_size"><P><B>Figure 10:</B> Dose response of the NO sensor with the estimated parameters. As we can see the limit of detection is around 100 uM of NO.
					The dynamic range is between 100 uM and 10 mM of NO. We are too far from the desired range of detection [2 uM - 200 uM]. Therefore the system
					is not enough sensitive for our purpose. It may come from the facts we are using the E. Coli's native NorR, which might be too low. The model thus
					shows that we should add the constitutive promoter associated with the NorR gene to our circuit.</P></DIV></DIV></DIV></DIV></DIV><DIV class="sec white"><DIV><P>Therefore the system is not enough sensitive for our purpose. It may come from the facts we are using the E. Coli's native NorR, which might be too low. The model thus
					shows that we should add the constitutive promoter associated with the NorR gene to our circuit.
				</P></DIV></DIV><DIV class="sec light_grey" id="ahlsensor"><DIV><DIV class="image_box" style="max-width: 500px;"><P><B>Figure 11:</B> AHL Sensor overview</P></DIV><DIV><H2>AHL SENSOR</H2><P>
                    In the absence of AHL, EsaR is constitutively produced, dimerizes and bind as a dimer to the esaBox situated downstream the
                    promoter, preventing transcription as a roadblock. When a higher than normal amount of AHL is present
                    in the gut, it binds to the EsaR dimer, and free the promoter, allowing transcription. Later on, several
                    EsaBox can be added, in order to tune the sensor sensitivity.
                </P></DIV><DIV class="sec white"><H4>ASSUMPTION</H4><P>
                We assume the mass conservation for the Esar such that:
				\begin{align*}Esar_{tot} = Esar + 2 DEsar + 2 DEsar_{AHL} + 2 DEsar_{AHL2} + Pesar1_{AHL1}\\\end{align*}
				
                </P></DIV></DIV></DIV><DIV class="sec light_grey two_columns"><DIV><H4>REACTIONS</H4><DIV>EsaR Hybrid Promoter System:
                

                \begin{align*} &amp;\rightarrow EsaR\\ 2 EsaR &amp; \rightleftharpoons DEsaR\\ AHL+DEsaR &amp;\rightleftharpoons DEsaR_{AHL1}\\
                AHL+DEsaR_{AHL1}&amp;\rightleftharpoons DEsaR_{AHL2}\\ Pesar1+AHL&amp;\rightleftharpoons Pesar1_{AHL1}\\
                Pesar1_{AHL1}+AHL&amp;\rightleftharpoons Pfree +DEsaR_{AHL2}\\ Pfree &amp;\rightarrow mRNA_{GFP}\\ EsaR&amp;\rightarrow\\
                DEsaR&amp;\rightarrow \\ DEsaR_{AHL1}&amp;\rightarrow\\ DEsaR_{AHL2}&amp;\rightarrow\\ mRNA_{GFP} &amp;\rightarrow\\
                \end{align*}


            </DIV><DIV>Esar Reporter System:
                

                \begin{align*} &amp;\rightarrow EsaR\\ 2 EsaR &amp; \rightleftharpoons DEsaR\\ AHL+DEsaR&amp;\rightleftharpoons DEsaR_{AHL1}\\
                AHL+DEsaR_{AHL1} &amp; \rightleftharpoons DEsaR_{AHL2}\\ Pesar2+AHL&amp;\rightleftharpoons Pesar2_{AHL1}\\
                Pesar2_{AHL1}+AHL&amp;\rightleftharpoons Pout+DEsaR_{AHL2}\\ Pout &amp;\rightarrow mRNA_{GFP}\\ EsaR&amp;\rightarrow
                \\ DEsaR&amp;\rightarrow \\ DEsaR_{AHL1}&amp;\rightarrow\\ DEsaR_{AHL2}&amp;\rightarrow\\ mRNA_{GFP} &amp;\rightarrow\\
                \end{align*}



            </DIV><TABLE><TBODY><TR><TH>Species </TH><TH>Description </TH></TR><TR><TD>AHL</TD><TD> Acyl Homocerine Lactone introduced in the medium </TD></TR><TR><TD>EsaR </TD><TD>EsaR constitutively produced inside<I>E. coli </I> cells </TD></TR><TR><TD>DEsaR </TD><TD>Dimer of EsaR , regulatory protein binding to Esaboxes situated downstream the promoter</TD></TR><TR><TD>DEsaR
                        <SUB>AHL1</SUB></TD><TD>Dimer with one AHL bound to one of its site </TD></TR><TR><TD>DEsaR
                        <SUB>AHL2</SUB></TD><TD>Dimer with two AHL bound to one of its site </TD></TR><TR><TD>Pesar
                        <SUB>i</SUB></TD><TD>Pesar1 correspond to the hybrid promoter. Pesar1 is the reporter promoter. They are independant</TD></TR><TR><TD>Pfree Pout respectively</TD><TD>promoter freed from the road block constituted by the EsaR bound to the downstream esaboxes</TD></TR></TBODY></TABLE><DIV class="quicklinks"><DIV><DIV class="outline_item"><A href="https://2016.igem.org/Team:ETH_Zurich/Parameters">Paramters Values</A></DIV></DIV></DIV></DIV></DIV><DIV class="sec light_grey"><DIV><H3>RESULTS</H3><P>
               The sensor module must be able to finely sense the different species, and in the rigth amount of concentrations. In this section we will 
explain how the model was used to provide useful insights for the biological system parameters. 
            </P></DIV></DIV><DIV class="sec light_grey"><DIV><A name="AHL_sensor_requirements"><H5>REQUIREMENTS</H5></A><P>
               AHL sensor sensitivity range = [10 nM - 1 uM]
               Dynamic range : the system must be as fast as possible
            </P><H5>KEY IDEA</H5><P>
We want to make the sensitivity range of the sensor and the activation range of the hybrid promoter match, so it propagates information relative to the 
inflammatory and candidate species to the switch and thus to the reporter. Under FACS and fluorescence distribution analysis the level of inflammation could 
then be inferred
            </P><H5>PARAMETERS OF INTEREST</H5><P><UL><LI>Transcription rate of NorR</LI><LI>Translation rate of NorR (RBS concentration)</LI><LI>Degradatioin rate of NorR</LI></UL></P><P>
            those parameters will allow us to set with the kinetic and the steady-state concentration of NorR in the system.
            </P></DIV></DIV><DIV class="sec light_grey"><DIV><H5>SENSITIVITY ANALYSIS</H5><DIV><DIV class="image_box" style="max-width: 800px;"><P><B>Figure 12:</B>The activation of the promoter was simulated under a constant AHL simulation of 100 nM with varying translation and transcription rates of Esar.
					As we can see it seems that they have similar impact on the circuit behaviour</P></DIV></DIV><DIV><P>
As we can see on the graph below, translation and transcription have similar effect on promoter activation. Thus we decided to play with promoter strength 
rather than rbs level inside each cells.
Later on we decided to only play with the promoter strength (transcription rate), as a entire collection of biobrick promoters  is available, and thus spare to the lab to work on a rbs library
in order to modify the cell translation rate.
                </P></DIV></DIV></DIV><DIV class="sec light_grey two_columns"><DIV><DIV><DIV><DIV class="image_box full_size"><P><B>Figure 13:</B>We simulated the effect on the transcription and degradation rate on the AHL sensor promoter activity under 100 nM (ideal lower limit of detection)
					As before we want to match the detection and dynamic range of the sensor to propagate the level of inflammation through the genetic circuit. Here it implies that the 
					ratio (Kd) must stay between 0.1 nM and 0.5 nM</P></DIV></DIV></DIV><DIV><DIV><DIV class="image_box full_size"><P><B>Figure 14:</B>The same analysis performed on the AHL sensor system with the ideal upper detection limit 1 uM diplays the heatmap above. To guarantee
					a activation superior to 90% at the input level, we need a Kd &lt; 0.66 nM</P></DIV></DIV></DIV></DIV></DIV><DIV class="sec light_grey"><DIV><H5>DOSE RESPONSE</H5><P>
               We simulated the dose response curves to identify the proper Esar production rate in order to match the requirements of our sensor.
            </P></DIV><DIV class="image_box full_size"><P><B>Figure 15:</B>The same dose response analysis was performed on the AHL sensor to finely tune our system in order to make it behave as ideally as possible.
		a range of different Esar production rate were tested on the circuit while simulated the dose response, assuming the ration constrained respected.</P></DIV></DIV><DIV class="sec light_grey"><DIV><H5>PARAMETERS ESTIMATION</H5><P>
               Using the previous analysis, we now want to be able to give insights for the real system we have. The first step consists in estimated the actual parameters
			   of our circuit. Actually we can only play with a few parameters, as most of them are chemical reactions rate on which we do not have any impacts.
			   We used MEIGO, a optimisation tool, to infer the parameters.
            </P></DIV><DIV class="image_box full_size"><P><B>Figure 16:</B> We performed a parameters deterministic estimation using MEIGO. The Estimation was perform using facs datas. Meigo is an optimization toolbox that includes
metaheuristic methods and a Bayesian inference method for parameter estimation.		</P></DIV></DIV><DIV class="sec light_grey"><DIV><P> We have some trouble with the plate reader test during the first part of our lab project. In particular with the AHL sensor that presented an unexpected behaviour at some
concentration of AHL. Therefore, we decided to perform this parameter estimation in order to get a deeper understanding of the chemical mechanism, and to try to find an explanation
and give to the biologists some clues to improve the system. As before the number of parameters the biologists can play with is rather restricted. We thus decided to focus on 
the two following parameters : EsaR production and degradation rate, which are easily tunable.</P></DIV></DIV><DIV class="sec light_grey two_columns"><DIV><DIV><DIV><DIV class="image_box full_size"><P><B>Figure 17:</B> A plate reader experiment presenting &quot;unexpected behaviour&quot; </P></DIV></DIV></DIV><DIV><DIV><DIV class="image_box full_size"><P><B>Figure 18:</B> Simulation of the same plate reader experiment. As we can see, there is a &quot;fall&quot; of <B>GFP</B> activity for AHL concentrations
					of 10 and 100 nM. See the dose response plots below for more explanations</P></DIV></DIV></DIV></DIV></DIV><DIV class="sec light_grey"><DIV><P> The number of parameters the biologists can play with is rather restricted. We thus decided to focus on 
	the two following parameters : EsaR production and degradation rate, which are easily tunable. On the graphs below we simulated the influence of degradation and production rate
	of EsaR on the dose response.</P></DIV></DIV><DIV class="sec light_grey"><DIV><DIV class="image_box full_size"><P><B>Figure 21:</B> Influence of EsaR production rate on the dose response of the system. As stated above our ideal range of sensitivity to AHL is between 10 nM and 10 uM.
The &quot;bump&quot; at low concnetration must be attenuated in order to avoid false positive. The results above suggests the the bump is attenuated at low production rate, but then the lack of road block 
preventing the <B>GFP</B> transcription decrease the limit of detection, And the system became sensitive to 1 nM of AHL. Another solution would be to increase EsaR production rate, but as shown on the graph, 
activation disappear even at very high AHL concentration. On the rigth, we plotted the influence of EsaR degradation rate on the dose response of the system. Unlike Esar production rate, when Esar degradation rate decreases, the &quot;low-concentration bump&quot; decreases as well, but the activation at higher AHL concentration is not decreased.
Therefore, the model tends to suggest that decreasing EsaR degradation rate	could improve the circuit accuracy</P></DIV></DIV></DIV><DIV class="sec light_grey"><DIV><H3>EXPLANATION AND OUTLOOK</H3><P>This behaviour was totally unexpected as we never witnessed any information about it on litterature describing Esar and esaboxes systems. Therefore, our biologists spend quite along time trying to figure out the reason of this 
bumpy like dose response, thinking they did something wrong during experiment settings. However once we set a proper FACS analysis, we managed to estimate the parameters and plugged them in the model. Then, it appears that all our experoiment were totally fine, just displaying a counter-intuitive behaviour.
To explain it, one has to take a closer look to the equation: bothe AHL and Esar are involved in different equations : AHL presence shift forward the equilibrium of DEsaR_{AHL} production and \begin{align*} Pesar1_{AHL1}+AHL&amp;\rightarrow Pfree +DEsaR_{AHL2}\\\end{align*}. However an increase of EsaR concentration also increase the
production of DEsaR_{AHL} which shifts the last reaction on the opposite direction. Therefore there a particular range of AHL/EsaR ratio for which the activation is stronger than the repression, which explains the &quot;bumpy behavior&quot; for intermediate AHL concentrations.		
		</P></DIV></DIV><DIV class="sec white" id="lactsensor"><DIV><DIV class="image_box" style="max-width: 500px;"><P><B>Figure 22:</B> Lactate Sensor overview</P></DIV><DIV><H2>LACTATE SENSOR</H2><P>
                    The promoter if flanked of two LldR specific binding sites : O1 and O2. In the absence of of lactate, LldR and LldD are constitutively
                    produced. LldR then binds to O1 and O2 as a dimer, forms a DNA loop and preventing transcription. When
                    Lactate (Lac) is present, it binds to the LldR complex and free the promoter. LldD lowers the concentration
                    of Lactate inside the cell by catalyzing its transformation into pyruvate. The idea is to set a tunable
                    treshold to the Lactate sensor, as this species, just like AHL, is anyway always present in the gut,
                    and we only want to sense abnormal concentration.
                </P></DIV><DIV class="sec white"><H3>ASSUMPTIONS</H3><P>
                    LldR exists as a dimer in solution. 2 molecules of lactate bind to one LldR dimer (L2). Lldr dimer bind to the two operator
                    sites when no LldR is present. Lactate releases the binding of LldR dimer to the operators.
                </P></DIV></DIV></DIV><DIV class="sec white two_columns"><DIV><DIV>Reaction
                <P>Lactate system:</P>

                \begin{align*} &amp;\rightarrow LldD\\ &amp;\rightarrow LldR\\ LldD+Lac&amp;\rightleftharpoons Pyr+LldD\\ 2LldR&amp;\rightleftharpoons
                DLldR\\ DLldR+ G_on&amp;\rightleftharpoons G_off\\ DLldR + Lac&amp;\rightleftharpoons DLldR_{Lac1}\\ DLldR_{Lac1}+Lac&amp;\rightleftharpoons
                DLldR_{Lac2}\\ G_off + Lac&amp;\rightleftharpoons G_off_1\\ G_off_1 + Lac&amp;\rightleftharpoons G_on + DLldR_{Lac2}\\
                G_on&amp;\rightleftharpoons mRNA_{GFP}\\ LldD&amp;\rightarrow\\ LldR&amp;\rightarrow\\ DLldR&amp;\rightarrow\\
                DLldR_{Lac1}&amp;\rightarrow\\ DLldR_{Lac2}&amp;\rightarrow\\ \end{align*}
            </DIV></DIV><DIV><DIV><TABLE><TBODY><TR><TH>Species </TH><TH>Description </TH></TR><TR><TD>LldR</TD><TD> regulatory protein of the Lac system, acts as a repressor </TD></TR><TR><TD>DLldR</TD><TD> Dimer of LldR </TD></TR><TR><TD>Lac</TD><TD>Lactate introduced in the medium. Forms a complex with LldR, preventing it from repressing the Promoter.
                            Acts thus as an activator<I>E. coli </I> cells </TD></TR><TR><TD>Pyr
                            <SUB>NO</SUB></TD><TD>Pyruvate, inactive form of lactate<I>E. coli </I> cells </TD></TR><TR><TD>LldD</TD><TD>Regulatory protein, catalyse the oxydation of Lactate into Pyruvate</TD></TR><TR><TD>G_on
                            <SUB>NO1</SUB></TD><TD>Active promoter </TD></TR><TR><TD>G_off
                            <SUB>NO2</SUB></TD><TD>Promoter repressed by LldR binding </TD></TR><TR><TD>G_off_1
                            <SUB>NO2</SUB></TD><TD>Repressed promoter with 1 lactate molecule bound </TD></TR><TR><TD>DLldR_Lac1
                            <SUB>i</SUB></TD><TD>DLldR with one Lactate molecule bound
                            <SUB>NO2</SUB></TD></TR><TR><TD>DLldR_Lac2
                            <SUB>3</SUB></TD><TD>DLldR with two Lactate molecule bound</TD></TR></TBODY></TABLE><DIV class="quicklinks"><DIV><DIV class="outline_item"><A href="https://2016.igem.org/Team:ETH_Zurich/Parameters">Paramters Values</A></DIV></DIV></DIV></DIV></DIV></DIV><DIV class="sec white"><DIV><A name="lactate_sensor_requirements"><H4>REQUIREMENTS</H4></A><P>
               AHL sensor sensitivity range = [100 uM - 100 mM]
               Dynamic range : the system must be as fast as possible
            </P><H5>KEY IDEA</H5><P>
As before, we want to make the sensitivity range of the sensor and the activation range of the hybrid promoter match, so it propagates information relative to the 
inflammatory and candidate species to the switch and thus to the reporter. Under FACS and fluorescence distribution analysis the level of inflammation could 
then be inferred. ETH-Zurich previous team already worked on a Lactate sensor. Thus we  tried to improve their work and adapt the sensor sensitivity to our system.
It appears that their system is too sensitive for our purpose. As a consequence we decided to add to the plasmid LldR which is constitutively produced and, by inactivating
Lactate, artificially lower the concentration of active Lactate that would be able to bind the LLdR complex that inhibits gene transcription.
            </P><H5>PARAMETERS OF INTEREST</H5><P><UL><LI>transcription rate of LldD</LI><LI>transcription rate of LldR</LI><LI>Degradation rate of LldD</LI></UL></P><P>
            those parameters will allow us to set with the kinetic and the steady-state concentration of Lactate in the system. However in term of modularity,
			it turns out it is easier for the biologist to play with the constitutive promoters involved in LldR and LldD production.
            </P></DIV></DIV><DIV class="sec white"><DIV><H3>DOSE RESPONSE AND PARAMETERS TUNING</H3><H3><DIV class="sec white"><DIV><DIV class="image_box full_size"><P><B>Figure 15:</B> In the graph above, we plotted the dose response curve for a large range of LldR production rate. This shows that one can easily tune the system and adjust the limit of detection,
				by increasing or decreasing the production rate of the LldR protein.As for the LldR plot, the LldD production rate can modify the limit of detection and the range of sensitivity of the lactate sensor. Therefore for a more precise
				tuning, playing with the LldD production rate is a possible way to tune the circuit.</P></DIV></DIV></DIV><DIV class="sec white two_columns"><DIV><DIV><P>
			To quickly summarize the difficulties met with the lactate sensor: We based our model and design on the ETH_Zurich iGem team 2015. In their  project, they also use a lactate sensitive circuit, So we decided to
			use the datas they already have for our sensor. However,  last year sensor proved to be by far too sensitive for our purpose: they were able to sense up to only a few nM of Lactate in the system while we need tosense between 
			100 uM and 100 mM. Thus we needed to tune the circuit such that it meet our requirements. To do that we propose the following method: As the more accessible parameters biologists can play with are protein production and degradation rate,
			we decided to simulate the influence of those parameters on the limit of sensitivity and dynamic range of our system. It appears that aas LldR is a repressor protein to the promoter, increasing its production rate and/or decreasing its degradation rate 
			shift the limit of detection to the right, decreasing the sensitivity of our circuit. However the amount of available promoter is limited, and do is their strength. We came up with the idea of introducing the LldD specie in the design.
			LldD catalyse the degradation of lactate into <I>Pyruvate</I>. So its presence artificially decrease the concentration of Lactate available to  activate the reporter inside the cell.
			Based on parameters found on litterarure, we figured out that the more you increase its production rate the less sensitive is the sensor. There fore playing with the different constitutive promoter strength allows the biologists to finely tune the system and make it fit our sensitivity requirements.</P></DIV><DIV><DIV><DIV class="image_box full_size"><P><B>Figure 23:</B> Plotting the maximum of the derivative of the dose response curves for the range of LldD production rate, we are able to 
				define the limit of detection of the lactate sensor.</P></DIV></DIV></DIV></DIV></DIV><DIV class="sec light_grey"><DIV><DIV class="image_box" style="max-width: 500px;"><P><B>Figure 23:</B> Full AND Gate overview</P></DIV></DIV></DIV></H3><H2>Full AND Gate</H2><P>
                Now it is time to link the two previous modules together in order to create the full AND Gate. Ideally, we would like to
                keep the model as modular as possible. In a first part, our way to proceed in order to recreate the hybrid
                promoter behavior from the two simple PnorV+Esabox promoter will be described. Then we propose a second model
                which takes into account all the different states of the promoter under NO and AHL/lactate binding, that
                can be stochastically simulated.
            </P></DIV><DIV><P>We developed two models to simulate the and gate. The first one is based on the modular model and consider both systems (NO sensor part and AHL sensor part) as independent. It was developed in the idea to keep everything inter-operable, interchangeable and modular. However, this  require a strong assumption, namely that there is no influence of one of the module on the other one.
In order to keep the precision of our model, we developed a second model formed of 61 equation that takes into account all the interactions between in the AND gate.
Therefor, we have two models for the AND gate, the first one which allow modularity and gives you a quick but slightly approximated overview of the full AND gate circuit, and a Second model, more complex, and longer to implement and to simulate, but that keep the high precision in term of mass action equations, and which is interesting in term of tuning, but sacrifices modularity.</P></DIV></DIV><DIV class="sec light_grey"><DIV><DIV class="image_box full_size"><P><B>Figure 18:</B>We first run the full AND Gate simulatioin using the modular model where we consider that the AHL and NO systems are independent. Knowing that this is a big 
assumption, we also run the simulation using the full model we developed (see link below). As we can see, with ideal parameters, our system is capable of 
displaying a AND Gate like behaviour. On the right part of the figure, we plotted the simulation using the parameters estimated from the FACS and plate reader
of the independant modules. As we can see, the results is far less spectacular. The full-system simulation of the gate using the real parameters tends to show 
that there is few difference between activation and leakiness, and side activation appears at low AHL concentration (certainly side effect of the Esar module 
behaviour). As a conclusion, while the modular model provide flexibility and reasonably
consistent results, it still lacks precision due to the assumptions. On the opposite the full model provide a lot of information but is fastidious to use.</P></DIV></DIV></DIV><DIV class="sec light_grey"><DIV><H3>ADDITIONAL LINKS AND INFORMATIONS</H3><H4>FULL-MODEL</H4><P> For additional information about the full model, please check the link below.</P><DIV class="quicklinks"><DIV><DIV class="outline_item"><A href="https://static.igem.org/mediawiki/2016/e/ef/T--ETH_Zurich--FULL_MASS_ACTION_MODEL_FOR_AND_GATE_MODULE.pdf">Full Mass Action model for the AND Gate</A></DIV></DIV></DIV></DIV></DIV><DIV class="sec white"><DIV><H2>DETERMINISTIC OVER STOCHASTIC MODEL</H2><P>
                At the beginning of the project, we wanted to a model as precise as possible that would allow us to tune finely our system in order to get
				a response as close to the ideal one as possible. In addition, we also decided to go first for a full stochastic model. Thus FACS analysis and comparison
				as well as fitting would assure us a good parameter estimation. Moreover, we wished that looking at the reporter distribution would help us
				to get some additional information about the input of the circuit. As the system is thought as a detector, and its application (travelling through the gut
				which is seen as a black box to get activated eventually by the presence of both NO and specific AHL) is to express a reporter, we wished that by simply
				looking at he distribution of mean and variance of the reporter could give us enough information in order to reconstitute the input. Thus in a medical 
				application, a simple FACS analysis of the remaining E.Coli harvesting from the faeces, would be enough to determine the acuteness of the inflammation and
				a quantitative data on the microbiota disbalance.
				However, it appeared after several staochastic simulation that the fluctuation of the ratio of activated promoter, for one trajectory, was to fast to have any
				influence of the reporter protein expression, as binding and unbinding of protein and ligand to and from DNA strand are quick reaction compared to mRNA and protein
				production and folding. As shown on the graph below. Thus we decided to go for simple deterministic simulation for the AND gate characterization.
            </P></DIV><DIV><DIV class="image_box full_size"><P><B>Figure 19:</B>We compare the behavior of the model under deterministic and stochastic simulations. As we can see the mean behavior remains similar. The fluctuations around the mean value due to the stochasticity can be neglected at a promoter scale because protein-ligand binding-unbinding reaction is extremely fast compare to protein/mRNA production and folding. Thus choosing a deterministic approach here does not represent a loss of information neither for the kinetic nor for the concentration at steady state. </P></DIV></DIV></DIV><DIV class="sec light_grey"><DIV><H2>ADDITIONAL LINKS AND INFORMATIONS</H2><H3>MODEL-BRICK</H3><P> For additional information, please check our public Git repository. All our codes are available there. As the spirit of iGem is information and
		knowledge sharing, we tried to make our codes as easily usable as possible.	</P><DIV class="quicklinks"><DIV><DIV class="outline_item"><A href="https://bitbucket.org/mattiagollub/igem_2016">Our Git repository</A></DIV></DIV></DIV></DIV></DIV><DIV class="sec white"><DIV><H3>REFERENCES</H3><P>[1] Tucker, N. P. et al. &quot;Essential Roles Of Three Enhancer Sites In  54-Dependent Transcription By The Nitric Oxide Sensing Regulatory Protein Norr&quot;. Nucleic Acids Research 38.4 (2009): 1182-1194.</P></DIV></DIV><DIV class="sec blue"><DIV><H2> Thanks to the sponsors that supported our project: </H2></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></BODY></HTML>