<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_ETH_Zurich_Part_Collection skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><DIV id="top_title"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:ETH Zurich/Part Collection</SPAN></H1></DIV><DIV id="HQ_page"><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><DIV class="title_container"><SPAN id="title_text"><A href="https://2016.igem.org/Team:ETH_Zurich">PAVLOV'S COLI</A></SPAN></DIV><DIV class="menu_container" id="main_menu"><UL class="menu" id="menu"><LI class="menu_item"><A href="https://2016.igem.org/Team:ETH_Zurich">HOME</A></LI><LI class="menu_item"><A>TEAM</A></LI><LI><A href="https://2016.igem.org/Team:ETH_Zurich/Team">Team</A></LI><LI><A href="https://2016.igem.org/Team:ETH_Zurich/Collaborations">Collaborations</A></LI><LI><A href="https://2016.igem.org/Team:ETH_Zurich/Attributions">Attributions</A></LI></UL><LI class="menu_item"><A>PROJECT</A></LI><LI><A href="https://2016.igem.org/Team:ETH_Zurich/Description">Description</A></LI><LI><A href="https://2016.igem.org/Team:ETH_Zurich/Design">Design</A></LI><LI><A href="https://2016.igem.org/Team:ETH_Zurich/Proof">Proof of Concept</A></LI><LI><A href="https://2016.igem.org/Team:ETH_Zurich/Demonstrate">Demonstrate</A></LI><LI class="menu_item"><A>MODELING</A></LI><LI><A href="https://2016.igem.org/Team:ETH_Zurich/Model">Model</A></LI><LI><A href="https://2016.igem.org/Team:ETH_Zurich/Sensor_Module">Sensor Module</A></LI><LI><A href="https://2016.igem.org/Team:ETH_Zurich/Switch_Module">Switch Module</A></LI><LI><A href="https://2016.igem.org/Team:ETH_Zurich/Detector_Module">Reporter Module</A></LI><LI><A href="https://2016.igem.org/Team:ETH_Zurich/NO_Release">NO Release</A></LI><LI><A href="https://2016.igem.org/Team:ETH_Zurich/Parameters">Parameters</A></LI><LI class="menu_item"><A>LAB</A></LI><LI><A href="https://2016.igem.org/Team:ETH_Zurich/Experiments">Experiments</A></LI><LI><A href="https://2016.igem.org/Team:ETH_Zurich/LabResults">Lab Results</A></LI><LI><A href="https://2016.igem.org/Team:ETH_Zurich/InterLab">Interlab Study</A></LI><LI><A href="https://2016.igem.org/Team:ETH_Zurich/Safety">Safety</A></LI><LI><A href="https://2016.igem.org/Team:ETH_Zurich/Hardware">Hardware</A></LI><LI><A href="https://2016.igem.org/Team:ETH_Zurich/Notebook">Notebook</A></LI><LI class="menu_item"><A>PARTS</A></LI><LI><A href="https://2016.igem.org/Team:ETH_Zurich/Parts">Parts</A></LI><LI><A href="https://2016.igem.org/Team:ETH_Zurich/Basic_Part">Basic Parts</A></LI><LI><A href="https://2016.igem.org/Team:ETH_Zurich/Composite_Part">Composite Parts</A></LI><LI><A href="https://2016.igem.org/Team:ETH_Zurich/Part_Collection">Part Collection</A></LI><LI class="menu_item"><A>HUMAN PRACTICES</A></LI><LI><A href="https://2016.igem.org/Team:ETH_Zurich/Human_Practices">Human Practices</A></LI><LI><A href="https://2016.igem.org/Team:ETH_Zurich/HP/Silver">Silver</A></LI><LI><A href="https://2016.igem.org/Team:ETH_Zurich/HP/Gold">Gold</A></LI></DIV><DIV class="content_wrapper" id="main_wrapper"><DIV class="sec page_title"><DIV><H1>PART COLLECTION</H1></DIV></DIV><DIV class="sec white" id="overview"><DIV><H2>OVERVIEW</H2><P>Our associative learning circuit requires a sensor that can detect simultaneously occuring signals, in this case nitric
				oxide and AHL (Figure 1). Such a sensor with these specific inputs has not been described in the literature before. Thus
				we generated a collection of AND gates responsive to NO and AHL in order to find a design that best fits the requirements
				of our system. </P><P>We also wanted to demonstrate the flexibility of our system and created one more AND gate responsive to lactate and NO.
				The lactate portion of the AND gate was based on the work done by <A href="https://2015.igem.org/Team:ETH_Zurich/Part_Collection">ETH Zurich 2015 team.</A></P><DIV class="image_box" style="max-width: 400px;"><P><B>Figure 1:</B> The learning circuit requires simultaneous detection of nitric oxide and AHL. We designed a collection
					of AND gates for this purpose, using a combination of the NorV promoter and esaboxes.</P></DIV><H3>Components of the AHL-based AND gate</H3><P>NorV promoter <B>(PnorV)</B> is the native promoter controlling the nitric oxide reduction operon (norRVW) in <I>E. Coli</I>.
				<A href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10.1074%2Fjbc.M212462200">[1].</A> It's transcriptional regulator,
				NorR, can bind nitric oxide and activate gene expression. We used this promoter in combination with basal NorR production
				in <I>E.coli</I> as the Nitric Oxide (NO) sensor of our AND gate. The PnorV promoter can also be found among the biobricks
				we submitted to the registry <A href="http://parts.igem.org/Part:BBa_K2116002">Part:BBa_K2116002.</A></P><P>EsaR <A href="http://parts.igem.org/Part:BBa_K2116001" target="_blank">(Part:BBa_K2116001)</A> is a transcriptional regulator
				of the <I>Pantoea stewartii</I> quorum sensing system <A href="https://www.ncbi.nlm.nih.gov/pubmed/?term=The+Autoregulatory+Role+Of+Esar%2C+A+Quorum-Sensing+Regulator+In+Pantoea+Stewartii+Ssp.+Stewartii%3A+Evidence+For+A+Repressor+Function">[2].</A>				In the abscence of <A href="http://parts.igem.org/3OC6HSL">3OC6HSL</A> it can bind DNA and inhibit transcription. An
				<B>esabox</B> is a 18bp sequence where EsaR can bind. Unlike other quorum sensing regulators EsaR acts as a transcriptional
				repressor and not an activator. We took advantage of this property, and placed esaboxes either;</P><P>i) as roadblocks after transcription start site of PnorV, preventing the polymerase from advancing or;</P><P>ii) within the PnorV to establish competitive binding between NorR and EsaR or RNA polymerase and EsaR.</P><H3>Design Considerations</H3><H4>i) Roadblock AND Gates</H4><P>This collection of AND gates each have either one, two or three esaboxes placed as roadblock. Since more than one EsaR
				binding close to each other could create steric hindrance, we constructed variants where the spacing between the esaboxes
				is either 8bp or 15bp. </P><TABLE><TBODY><TR><TH>Design<SUP>*</SUP></TH><TH>Biobrick</TH><TH>Comments</TH></TR><TR><TD><A href="http://parts.igem.org/wiki/index.php?title=Part:BBa_K2116004">BBa_K2116004</A></TD><TD><P>This AND gate worked best, out of all roadblock AND gates we constructed</P></TD></TR><TR><TD><A href="http://parts.igem.org/wiki/index.php?title=Part:BBa_K2116005">BBa_K2116005</A></TD><TD><P>The two esaboxes are separated by an 8bp orthagonal spacer</P></TD></TR><TR><TD><A href="http://parts.igem.org/wiki/index.php?title=Part:BBa_K2116006">BBa_K2116006</A></TD><TD><P>The esaboxes are separated by 8bp long spacers</P></TD></TR><TR><TD><A href="http://parts.igem.org/wiki/index.php?title=Part:BBa_K2116007">BBa_K2116007</A></TD><TD><P>The esaboxes are separated by a 15bp long spacer</P></TD></TR><TR><TD><A href="http://parts.igem.org/wiki/index.php?title=Part:BBa_K2116008">BBa_K2116008</A></TD><TD><P>The esaboxes are separated by 15bp long spacers</P></TD></TR><TR><TD><A href="http://parts.igem.org/wiki/index.php?title=Part:BBa_K2116013">BBa_K2116013</A></TD><TD><P>The esaboxes are separated by a 8bp long spacer, and the native spacer of PnorV after the transcription start site has
						been removed from this design</P></TD></TR><TR><TD><A href="http://parts.igem.org/wiki/index.php?title=Part:BBa_K2116014">BBa_K2116014</A></TD><TD><P>The esaboxes are separated by 8bp long spacers, and the native spacer of PnorV after the transcription start site has
							been removed from this design</P></TD></TR></TBODY></TABLE><P><B><SUP>*</SUP>NorR = NorR binding site, Pol= RNA polymerase factor sigma 54 binding site, esabox= EsaR binding site</B></P><H4>ii) Competitive Binding and/or Roadblock AND Gates</H4><P>Here our aim was to prevent either NorR or the sigma54 factor of RNA polymerase from binding the promoter. There are 3
				NorR binding sites, and one sigma54 binding site on the NorV promoter (Figure 2). We placed esaboxes either right before
				the sigma54 binding site, as a replacement for part of the sigma54 binding site, or upstream of one of the NorR binding
				sites. For one of these designs, we also have a variant where an additional esabox is placed as a roadblock. This was
				designed as a solution to potential problems with leakiness of the AND gate.</P><TABLE><TBODY><TR><TH>Design<SUP>*</SUP></TH><TH>Biobrick</TH><TH>Comments</TH></TR><TR><TD><A href="http://parts.igem.org/wiki/index.php?title=Part:BBa_K2116011">BBa_K2116011</A></TD><TD><P>One esabox before NorR binding site 3, and one as a roadblock after the transcription start site. This is our <B>favourite AND gate</B></P></TD></TR><TR><TD><A href="http://parts.igem.org/wiki/index.php?title=Part:BBa_K2116038">BBa_K2116038</A></TD><TD>
					Unfortunately, placing an esabox before the polymerase binding site lead to a loss of function in the AND gate
					</TD></TR><TR><TD><A href="http://parts.igem.org/Part:BBa_K2116040">BBa_K2116040</A></TD><TD>
					Unfortunately, placing an esabox before the polymerase binding site lead to a loss of function in the AND gate
					</TD></TR></TBODY></TABLE><P><B><SUP>*</SUP>NorR = NorR binding site, Pol= RNA polymerase factor sigma 54 binding site, esabox= EsaR binding site</B></P><H3>Lactate-based AND gate</H3><P>The lactate and NO AND gate was designed to demonstrate the potential multiplexing power of our circuit. Unlike AHL, which is quorum sensing molecule, lactate is an intermediate metabolite of microbial metabolism. To detect lactate, we used the LldR repressor. In the absence of lactate LldR binds to the two LldR-binding sites and represses the transcription through looping of the DNA between the two binding sites. To get a promoter with an AND gate logic, we placed a pNorV promoter between the two LldR binding sites.</P><TABLE><TBODY><TR><TH>System Design</TH><TH>Biobrick</TH><TH>Comments</TH></TR><TR><TD><A href="http://parts.igem.org/wiki/index.php?title=Part:BBa_K2116027">BBa_K2116027</A></TD><TD>
Promoter displays an AND gate logic. The induction rate is 1.5. 
</TD></TR></TBODY></TABLE><H3>Model</H3><P>In order to match the experiments, we modelled independantly The EsaR interaction system, and the NorR systems. This way we could easily tuned the two facets of the AND Gate sensor module so that it matches the system sensing requirements. Later on, we combined those two independant modules together to model the complete AND Gate behavior.</P><P>PnorV is a looping promoter, which allows NorR to come in contact with the RNA polymerase. Transcription is initiated
				once NO binds and NorR is active. [3] The initial results showed that the AND gate was repressed by EsaR, but could not
				be fully induced. Our first hypothesis was that looping prevents EsaR from unbinding. However, further analysis of the EsaR system alone tend to show that two species $DEsaR_{AHL}$ and $AHL$ compete to disable/enable the promoter for transcription. Thus the system dose response does not display a monotonal behavior. On the contrary, the rate of freed promoter srongly depends on the $DEsaR_{AHL}$ / $AHL$ ratio, and displays an unexpected not so repressive behavior for a specific range of EsaR production rate. </P><P> Nevertheless, a careful analysis of the system based on estimated paramters from the data, demonstrate that this EsaR repression problem can be solved by changing the constitutive promoter preceding the EsaR gene.</P><P>For further details, do not hesitate to consults our
<A href="https://2016.igem.org/Team:ETH_Zurich/Sensor_Module#AHL_sensor_requirements">AHL sensor model page</A>.</P><H3>Conclusion</H3><P>After careful evaluation, we picked one variant to be the main AND gate of our system. After tuning the EsaR concentration, this AND gate demonstrated the
                best behaviour within the range of AHL and NO concentrations we wanted to detect. Below you can see a surface plot (Figure 6) generated through experimental data (black dots) and interpolation (surface). This describes the AND gate behaviour before tuning EsaR concentrations. Figure 7 shows that after tuning the AND gate responds to ranges indicated by the white dashes on the surface plot (Figure 6), which marks the desired area. </P><P>We have a collection of AND gates characterised experimentally and with a detailed model that could be used by future
				iGEM teams. </P><DIV class="image_box full_size"><P><B>Figure 6:</B> AND gate response. Fluorescence measured 6 hours after induction with a plate reader. Black dots indicate mean of three experimental technical replicates, and the surface is interpolated. White dashed area indicates application-relevant physiological concentrations of AHL and NO. </P></DIV><DIV class="image_box full_size"><P><B>Figure 7</B>: AND gate response before and after changing EsaR concentration in the system. Fluorescence measured 6h after induction with plate reader, error bars indicate S.D. of three technical replicates. </P></DIV><DIV><H4>References:</H4><P>[1] Gardner, A. M. &quot;Regulation Of The Nitric Oxide Reduction Operon (Norrvw) In Escherichia Coli. ROLE OF Norr AND Sigma
				54 IN THE NITRIC OXIDE STRESS RESPONSE&quot;. Journal of Biological Chemistry 278.12 (2003): 10081-10086.</P><P>[2] Minogue, Timothy D. et al. &quot;The Autoregulatory Role Of Esar, A Quorum-Sensing Regulator In Pantoea Stewartii Ssp.
				Stewartii: Evidence For A Repressor Function&quot;. Molecular Microbiology 44.6 (2002): 1625-1635. Web.</P></DIV></DIV></DIV><DIV class="sec blue"><DIV><H2> Thanks to the sponsors that supported our project: </H2></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></BODY></HTML>