<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_ETH_Zurich_Demonstrate skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><DIV id="top_title"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:ETH Zurich/Demonstrate</SPAN></H1></DIV><DIV id="HQ_page"><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><DIV class="title_container"><SPAN id="title_text"><A href="https://2016.igem.org/Team:ETH_Zurich">PAVLOV'S COLI</A></SPAN></DIV><DIV class="menu_container" id="main_menu"><UL class="menu" id="menu"><LI class="menu_item"><A href="https://2016.igem.org/Team:ETH_Zurich">HOME</A></LI><LI class="menu_item"><A>TEAM</A></LI><LI><A href="https://2016.igem.org/Team:ETH_Zurich/Team">Team</A></LI><LI><A href="https://2016.igem.org/Team:ETH_Zurich/Collaborations">Collaborations</A></LI><LI><A href="https://2016.igem.org/Team:ETH_Zurich/Attributions">Attributions</A></LI></UL><LI class="menu_item"><A>PROJECT</A></LI><LI><A href="https://2016.igem.org/Team:ETH_Zurich/Description">Description</A></LI><LI><A href="https://2016.igem.org/Team:ETH_Zurich/Design">Design</A></LI><LI><A href="https://2016.igem.org/Team:ETH_Zurich/Proof">Proof of Concept</A></LI><LI><A href="https://2016.igem.org/Team:ETH_Zurich/Demonstrate">Demonstrate</A></LI><LI class="menu_item"><A>MODELING</A></LI><LI><A href="https://2016.igem.org/Team:ETH_Zurich/Model">Model</A></LI><LI><A href="https://2016.igem.org/Team:ETH_Zurich/Sensor_Module">Sensor Module</A></LI><LI><A href="https://2016.igem.org/Team:ETH_Zurich/Switch_Module">Switch Module</A></LI><LI><A href="https://2016.igem.org/Team:ETH_Zurich/Detector_Module">Reporter Module</A></LI><LI><A href="https://2016.igem.org/Team:ETH_Zurich/NO_Release">NO Release</A></LI><LI><A href="https://2016.igem.org/Team:ETH_Zurich/Parameters">Parameters</A></LI><LI class="menu_item"><A>LAB</A></LI><LI><A href="https://2016.igem.org/Team:ETH_Zurich/Experiments">Experiments</A></LI><LI><A href="https://2016.igem.org/Team:ETH_Zurich/LabResults">Lab Results</A></LI><LI><A href="https://2016.igem.org/Team:ETH_Zurich/InterLab">Interlab Study</A></LI><LI><A href="https://2016.igem.org/Team:ETH_Zurich/Safety">Safety</A></LI><LI><A href="https://2016.igem.org/Team:ETH_Zurich/Hardware">Hardware</A></LI><LI><A href="https://2016.igem.org/Team:ETH_Zurich/Notebook">Notebook</A></LI><LI class="menu_item"><A>PARTS</A></LI><LI><A href="https://2016.igem.org/Team:ETH_Zurich/Parts">Parts</A></LI><LI><A href="https://2016.igem.org/Team:ETH_Zurich/Basic_Part">Basic Parts</A></LI><LI><A href="https://2016.igem.org/Team:ETH_Zurich/Composite_Part">Composite Parts</A></LI><LI><A href="https://2016.igem.org/Team:ETH_Zurich/Part_Collection">Part Collection</A></LI><LI class="menu_item"><A>HUMAN PRACTICES</A></LI><LI><A href="https://2016.igem.org/Team:ETH_Zurich/Human_Practices">Human Practices</A></LI><LI><A href="https://2016.igem.org/Team:ETH_Zurich/HP/Silver">Silver</A></LI><LI><A href="https://2016.igem.org/Team:ETH_Zurich/HP/Gold">Gold</A></LI></DIV><DIV class="content_wrapper" id="main_wrapper"><DIV class="sec white"><DIV><H2>SENSOR MODULE</H2><DIV class="image_box" style="max-width:300px;"><P><B>Figure 1:</B> AND gate design: Our AND gate operates through a nitric oxide sensing transcriptional activator (NorR)
				and an AHL sensitive repressor (EsaR).</P></DIV><P>Our associative learning system requires simultaneous detection of two signals. The first signal is nitric oxide, which
			serves as a marker for inflammation. The second signal can be a number of different microbiome markers inside the gut.
			We chose <A href="http://parts.igem.org/AHL"> AHL </A> as one example and constructed an AND gate that can detect NO and
			AHL. </P></DIV></DIV><DIV class="sec light_grey"><DIV><H3>CHARACTERIZATION</H3><H4>NITRIC OXIDE SENSOR: NorV PROMOTER</H4><P>NorV promoter(PnorV, <A href="http://parts.igem.org/Part:BBa_K2116002">BBa_K2116002</A>) controls NO dependent transcription
			in E.coli <A href="https://www.ncbi.nlm.nih.gov/pubmed/12529359">[1]</A>. Gene expression can only be activated when NO
			binds the PnorV transcriptional activator NorR.</P><P>Initially we wanted to see how much NorR we would require in order to sense a range of NO from 20-200μM, which is the range
			that is typically seen in IBD <A href="https://www.ncbi.nlm.nih.gov/pubmed/7996962">[2].</A> We <A href="https://2016.igem.org/Team:ETH_Zurich/Sensor_Module">modeled the system</A>			based on parameters found in the literature. Our model suggested that endogenous production of NorR in <I>E.coli</I><A href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2831303/"> [3]</A> is enough for our desired range of sensitivity. Thus
			after successfully cloning the NorV promoter, we tested its activity with the endogenous NorR.
			<A href="http://www.sigmaaldrich.com/catalog/product/sigma/d185?lang=en&amp;region=CH">DETA/NO</A> was used as a source of NO.
			Using our
			<A href="https://2016.igem.org/Team:ETH_Zurich/NO_Release">NO release model</A> for DETA/NO, we could show that the PnorV
			promoter is active within the range of 20-200μM of NO we would like to detect in the gut (Figure 2). </P><DIV class="image_box full_size"><P><B>Figure 2:</B> PnorV dose response curve for a range of DETA/NO concentrations that corresponds to 7-70μM of NO. Based
				on the insights we got from <A href="https://2016.igem.org/Team:ETH_Zurich/Sensor_Module#nosensor">our model</A>, we tested
				the promoter only in presence of endogenous NorR in <I>E.coli</I>. We can show that the activity range of the promoter
				is within the 20-200μM range that corresponds to inflammation in the gut
				<A href="https://www.ncbi.nlm.nih.gov/pubmed/7996962">[2]</A>. The dose response does not reach full saturation since too
				much nitric oxide effects growth of bacteria. Fluorescence measured with a plate reader, 6 hours after induction. Error
				bars represent S.D. from three technical replicates.
			</P></DIV><P>We also wanted to demonstrate that we are indeed using the endogenous NorR to activate our cloned promoter. For this purpose
			we used the
			<A href="http://cgsc.biology.yale.edu/KeioList.php">Keio norR KO strain</A> and compared it to its WT parent strain and
			proved that the endogenous NorR is activating our system.</P><DIV class="image_box full_size"><P><B>Figure 3:</B>A norR knock out <I>E.coli</I> strain was used to demonstrate that endogenous NorR activates PnorV in
				abscence of exogenously expressed protein. Induced with 5000μM DETA/NO and fluoresence measured with a plate reader 6
				hours after induction. Error bars represent S.D. from three technical replicates.</P></DIV></DIV></DIV><DIV class="sec white"><DIV><H4>AHL SENSOR: esaBox/EsaR SYSTEM</H4><DIV class="image_box" style="max-width:350px;"><P><B>Figure 4:</B> Mechanism of transcriptional repression by EsaR. In the abscence of 3OC6HSL EsaR dimerizes and binds
				the esabox sequence to repress transcription. </P></DIV><DIV class="image_box" style="max-width:200px;"><P><B>Figure 5:</B> Construct to test EsaR/esabox system. An esabox was placed after the constitutive promoter <A href="http://parts.igem.org/Part:BBa_J23118"> (BBa_J23118).</A>.</P></DIV><P>EsaR <A href="http://parts.igem.org/Part:BBa_K2116001">BBa_K2116001</A> is a transcriptional regulator of the <I>Pantoea stewartii</I>			quorum sensing system. <A hrefhttps:="">[4]</A>			In the abscence of <A href="http://parts.igem.org/3OC6HSL">3OC6HSL</A> it can bind DNA and inhibit transcription (Figure
			4). An esabox is a 18bp sequence where EsaR can bind. Unlike other quorum sensing regulators EsaR acts as a transcriptional
			repressor and not an activator.</P><P>In order to characterise repression efficiency of EsaR, we placed esaboxes after a constitutive promoter<A href="http://parts.igem.org/Part:BBa_J23118"> (BBa_J23118).</A>(Figure
			5) During dose response tests we discovered that EsaR system displays non-sigmoidal dose response. With the intermediate
			AHL concentrations we tested, there's a dip in the dose response curve (Figure 6). This corresponds to concentrations
			close to the saturation point of the dose response. Our <A href="https://2016.igem.org/Team:ETH_Zurich/Sensor_Module">AHL sensor model</A>			could later explain the reason behind this behaviour. </P><P>Based on this insight from the model, we changed the amount of EsaR we express inside the cell to reduce the dip in the
			dose response curve. Our strategy was to put two copies of EsaR under individual constitutive promoters(
			<A href="http://parts.igem.org/Part:BBa_J23118"> BBa_J23118</A>; expressing EsaR on a low copy plasmid instead of the initial
			medium copy plasmid; and expressing EsaR under a stronger constitutive promoter <A href="http://parts.igem.org/Part:BBa_J23102">
            (BBa_J23102).</A> In the end we obtained the best results with increasing EsaR expression through the stronger
			constitutive promoter <A href="http://parts.igem.org/Part:BBa_J23102">
            (BBa_J23102)</A>. Even though we cannot obtain fully sigmoidal behaviour, we managed to reduce the effect of
			this phenomenon on the EsaR dose response. (Compare Figure 6 and 7). Consequently we decided to use this construct for
			assembling our complete system. </P><DIV class="image_box full_size"><P><B>Figure 6:</B> Dose response curve for the EsaR/esabox test construct in Figure 5. At intermediate AHL (3OC6HSL) concentrations
				tested the dose response curve deviates from sigmoid behaviour. The reason behind this behaviour was later explained
				through our <A href="https://2016.igem.org/Team:ETH_Zurich/Sensor_Module">AHL sensor model</A>. Fluorescnece measured with
				plate reader 3 hours after induction. Error bars indicate S.D. of three technical replicates</P></DIV><DIV class="image_box full_size"><P><B>Figure 7:</B> Esar dose response for expression under strong constitutive promoter <A href="http://parts.igem.org/Part:BBa_J23118">
            (BBa_J23118).</A> The dose response behaviour is still not sigmoidal, however the effect of the dip seen in Figure
				5 is significantly attenuated by increasing EsaR expression. Fluorescence measured with plate reader 3 hours after induction.
				Error bars indicate S.D. of three technical replicates.</P></DIV></DIV></DIV><DIV class="sec light_grey"><DIV><H4>AND GATE</H4><P>We generated AND gates by placing esaboxes either; </P><P>i) as roadblocks after transcription start site of PnorV, preventing the polymerase from advancing or;</P><P>ii) within the PnorV to establish competitive binding between NorR and EsaR or RNA polymerase and EsaR.</P><P><A href="https://2016.igem.org/Team:ETH_Zurich/Part_Collection">(Check out our Part Collection for examples)</A></P><DIV class="image_box full_size"><P><B>Figure 8:</B> One example of our AND gates responsive to NO and AHL. This is the AND gate we eventually used for our
				complete system. Nomenclature: NRB-NorR binding site, sigma54B-RNA polymerase binding site.</P></DIV><P>Our complete system would express a recombinase under the control of this AND gate. From the <A href="https://2016.igem.org/Team:ETH_Zurich/Switch_Module#tuning"> model of the switch</A>			we know that low leakiness of the AND gate is crucial in order to ensure reliability of the stored information, thus the
			best AND gates are the ones that exhibit the highes fold-change activation. Following these indications, after characterizing
			our collection we decided to use the AND gate seen in Figure 8, since it showed the highest fold activation (Figure 9).</P><DIV class="image_box full_size"><P><B>Figure 9:</B> The AND gate we selected to use for our complete system displays 4-fold activation and low leakiness,
				fulfilling the criteria suggested by our <A href="https://2016.igem.org/Team:ETH_Zurich/Switch_Module#tuning">model for the switch</A>				Induction was with 5mM of <A href="https://2016.igem.org/Team:ETH_Zurich/Experiments">DETA/NO</A> Fluorescence measured
				with plate reader 3 hours after induction. Error bars indicate S.D. of three technical replicates.</P></DIV><P>An important requirement for our AND gate is that the second input should be easily exchangeable. This is where our learning
			circuit gets its flexibility from. This was an important motivation behind choosing an activator system for the NO part,
			and a repressor system for the AHL part of our AND gate. In order to demonstrate this flexibility, we exchanged the esaboxes
			with the lldO operators, allowing us to create an AND gate responsive to lactate and NO (Figure 10).</P><DIV class="image_box full_size"><P><B>Figure 10:</B> An alternative AND gate responsive to NO and lactate. Demonstrates the flexibility of our system. Induction
				was with 5mM DETA/NO and 10mM lactate. Fluorescence measured with plate reader 3 hours after induction. Error bars indicate
				S.D. of three technical replicates.</P></DIV></DIV></DIV><DIV class="sec white"><DIV><H2>SWITCH AND REPORTER MODULES</H2><P>The switch and the reporter together form the memory and read-out part of our associative learning system. Our reporter
			is a dual fluorescent construct, expressing the red fluorescent protein mNectarine when the system is OFF, and gfp when
			the memory has been triggered. This switching is induced by a recombinase that can flip a directional promoter flanked
			by attP and attB parts it recognises <A href="http://parts.igem.org/wiki/index.php?title=Part:BBa_K2116024">(BBa_K2116024).</A>			After flipping the promoter no longer faces mNectarine, but now faces the gfp.</P><P>We cloned two recombinases, bxb1<A href="http://parts.igem.org/wiki/index.php?title=Part:BBa_K2116026">(BBa_K2116026)</A>			and tp901<A href="http://parts.igem.org/wiki/index.php?title=Part:BBa_K2116035">(Bba_K2116035)</A> under the Ptet promoter.
			Since recombinase flipping is irreversible, our initial aim was to avoid leakiness as much as possible. Hence the recombinases
			were designed with weak RBS and fast degradation tags.</P><P> However our initial results, and our model eventually suggested that this was not the optimal design. Even when we removed
			the degradation tag the flipping was too weak.</P><P>We decided to use the model to <A href="https://2016.igem.org/Team:ETH_Zurich/Switch_Module#tuning">tune</A> the switch precisely.
			Performing a sensitivity analysis on the parameters of the switch, we identified integrase degradation rate and RBS strength
			as the key tunable parameters of the switch. Moreover we discovered that in order for the switch to work optimally, these
			parameters must have a specific ratio. Based on this input we designed a stronger RBS.</P><P>Under advice from the <A href="https://2016.igem.org/Team:ETH_Zurich/Detector_Module#results">model</A> we also decreased
			the repression of Ptet promoter by expressing TetR on a low copy plasmid rather than a medium one. These improvements
			resulted in significant improvement of the switch. We can now show that flipping at 8hours is about 3-fold more efficient compared to
			before (Figure 11.) In addition, the flipping kinetics satisfy the <A href="https://2016.igem.org/Team:ETH_Zurich/Switch_Module#tuning">requirements</A>			defined by the model almost perfectly.</P><DIV class="image_box full_size"><P><B>Figure 11:</B> Bxb1 flipping activity. When a directional promoter is flipped by bxb1, gfp is expressed. Here we show
				the distribution of cells expressing gfp under the control of bxb1 flipping. The weak RBS results in ~3-fold less efficient flipping
				over 8 hours, compared to the improved RBS which is stronger. Data was generated through flow cytometry analysis. The
				error bars indicate SEM. </P></DIV></DIV></DIV><DIV class="sec white"><DIV><H3>PROOF OF CONCEPT SIMULATION</H3><P>We started modeling the system already during the design phase, allowing us to provide a quick proof of concept of the
			proposed curcuits. Figure 12 shows the functioning of the full system: Pavlov's Coli is encpsulated in a pill and is ingested
			by the patient. During transit in the gut the system is exposed to the inflammation marker (NO) and the candidate marker
			(species-specific AHL). In presence of both markers the promoters of the switch start flipping, storing the association
			event on the DNA. After recovery of the pill, the system is exposed to the candidate marker (AHL) again and since AHL
			was associated with an inflammation marker, GFP is expressed again.</P><DIV class="image_box full_size"><P><B>Figure 12:</B> Simulation of the full system. Once NO and AHL are detected at the same time, our memory element start
				switching. After recovery of the pill, the system is exposed to AHL again and expresses GFP if NO and AHL were detected.</P></DIV></DIV></DIV><DIV class="sec light_grey"><DIV><H2> COMPLETE SYSTEM ASSEMBLY </H2><P>Once we showed functionality of all individual system components we assembled our three component system. There were a
			few issues with assembly:
			<UL><LI> In the initial design the reporter has an EsaR repressible promoter. In this test we omitted an esabox and used the constitutive
					promoter <A href="">Bba_j23118</A> instead for ease of observing the outcome. </LI><LI> Initially EsaR was expressed on a low copy plasmid, but this did not yield enough repression. However an ori exchange
					to the R6K ori was unsuccessful and due to time constraints we couldn't troubleshoot. In the end the plasmids bearing
					EsaR and the AND gate had incompatible oris, making our results hard to interpret.</LI></UL></P><P>The results demonstrate a potential for proper AND gate behaviour and simulatenous detection of NO and AHL. We believe that system activation in presence of only AHL is due to the problems caused by the incompatible oris. 
            </P><DIV class="image_box full_size"><P>Figure 13. Full system assembly results. DETA/NO= 5mM, AHL= 100nM. Fluorescence measured with plate reader 8 hours after induction. Error bars indicate
				S.D. of three technical replicates. </P></DIV></DIV></DIV><DIV class="sec white"><DIV><H2>CONCLUSION AND OUTLOOK</H2><UL><LI>Overall we have shown that the 3 main biobrick devices that make up our complete system work individually.
				</LI><LI> Our AND gate is not perfectly optimized yet, and requires tuning for the new RBS we have designed. We would ideally like
					to test more variants and find one with even higher fold-activation.
					</LI><LI> We have a functional recombinase, that is active within the time range we desire.
						</LI><LI> Our reporter construct works reliably.
							</LI><LI> We have assembled our full system, and tested it. Next step would be to complete the ori exchange and further optimize the whole system for application.
                                </LI><LI> There will also be work done on the encapsulation of our Pavlov's Coli, which will be tested in simulated gut environments.
		</LI></UL></DIV></DIV><DIV class="sec white"><DIV><H2>REFERENCES</H2><UL><LI> [1] Gardner, A. M. &quot;Regulation Of The Nitric Oxide Reduction Operon (Norrvw) In Escherichia Coli. ROLE OF Norr AND Sigma
				54 IN THE NITRIC OXIDE STRESS RESPONSE&quot;. Journal of Biological Chemistry 278.12 (2003): 10081-10086. Web. 16 Oct. 2016.
				</LI><LI> [2] Lundberg, J.O.N. et al. &quot;Greatly Increased Luminal Nitric Oxide In Ulcerative Colitis&quot;. The Lancet 344.8938 (1994):
					1673-1674.
					</LI><LI> [3] Tucker, N. P. et al. &quot;Essential Roles Of Three Enhancer Sites In  54-Dependent Transcription By The Nitric Oxide
						Sensing Regulatory Protein Norr&quot;. Nucleic Acids Research 38.4 (2009): 1182-1194.
						</LI><LI> [4] Minogue, Timothy D. et al. &quot;The Autoregulatory Role Of Esar, A Quorum-Sensing Regulator In Pantoea Stewartii Ssp.
							Stewartii: Evidence For A Repressor Function&quot;. Molecular Microbiology 44.6 (2002): 1625-1635.

		</LI></UL></DIV></DIV><DIV class="sec blue"><DIV><H2> Thanks to the sponsors that supported our project: </H2></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></BODY></HTML>