"
Page
Discussion
View source
History
teams
Log in
Â 
Team:Washington/Our Project
From 2014.igem.org
Home
UW 2014
UW 2013
UW 2012
UW 2011
UW 2010
UW 2009
UW 2008
iGEM Homepage
Our Project
Our System
Methods
Results
Future Plans
BioBricks
Outreach
Protocols
Safety
Safety
Project Considerations
Judging Form
Our Team
Our System
Natural Cell Processes
We sought to make use of a cell's natural methods of degrading misfolded proteins. If a protein is misfolded in a cell it is targeted by the E3 ligase which attaches a ubiquitin to it. This marks the protein for degradation by the proteasome. If a protein is unstable it is likely at very low levels within the cell as it is being degraded by this system.
Gal4-VP16
Our system relies on GFP being produced at different levels depending on the stability of a protein of interest. To do this our system puts the protein of interest in between Gal4 and VP16.
Fig 1. Our project utilizes the Gal4-VP16 transcriptional activator to test protein stability in terms of GFP output.
Together Gal4 and VP16 up-regulate the expression of a gene under certain promoters, in our case we used Gal1. Gal4 binds to the DNA and then VP16 recruits RNA polymerase to begin transcription of the gene under Gal1. Independently Gal4 simply binds to Gal1, and VP16 is not localized to the DNA. So if our gene located in between the two, degradation of the gene leads to no transcription of anything under Gal1.
Using a Degron to Exaggerate Differences in Stability
We sought to make a versatile system that could be used for proteins of various native stabilities. If a protein is stable enough to avoid ubiquitination but not stable enough for its engineered purpose our system would not be useful. To deal with this we used a degron.
A degron is an inherently unstable protein domain. By inserting this into the fusion protein produced by our plasmid we expect a protein which is just stable enough to avoid degredation will become unstable and be degraded. We expect a very stable protein to be able to overcome this source of instability and this will allow us to the measure differences in stability of more stable protein variants where without the degron they would give very similar measurements in our system.
Components of Our Plasmid
The fusion protein produced by expression of our plasmid is made up of the Gal4-VP16 transactivating complex with a protein of interest in between. Positioning of the degron is determined by the native stability of the protein of interest.
Fig 2. Potential Degron insert sites for our system.
There are 5 possible degron positions:
-Deg0: This construct contains only the Gal4-VP16 transcriptional activator complex with the protein of interest in between the two (shortened as Gal4-Protein-VP16).
-Deg1: This construct contains the Degron in front of our Gal4-Protein-VP16 complex.
-Deg2: This construct contains the Degron in between Gal4 and the protein in our Gal4-Protein-VP15 complex.
-Deg3: This construct contains the Degron in between the protein and VP16 in our Gal4-Protein-VP15 complex.
-Deg4: This construct contains the Degron at the end of our Gal4-Protein-VP16 complex.
Relative Stability Analyzed via Flow Cytometry
Fig 3. Expected GFP output based on our Degron constructs
Flow cytometry is a high throughput method of analyzing cells for various optical outputs, namely fluorescence. A flow cytometer is an analytical instrument in which cells that have been suspended in a solution are passed through a narrow channel in which fluorescence of individual cells can be measured.
By utilizing Flow Cytometry, we can measure the amount of GFP output within cells from each degron construct. Based on where the Degron is inserted, we expected a different level of fluorescence. As such, we expected to see the highest GFP production in our Deg0 construct, as it only contains the Gal4-Protein-VP16 complex with no Degron inserted, therefore we expect it to be the most stable. We expected that Deg2 and Deg3 would have a lower GFP production than Deg0 but higher than Deg1 and Deg4. This rationale was based on the fact that the Deg1 and Deg4 have the Degron exposed, making it more likely to be degraded by ubiquitination than in Deg2 and Deg3 which has the Degron buried inside the Gal4-Protein-VP16 complex.
Test Protein
The test protein that must be chosen in testing a novel and new system must be a protein that has been well studied and rigorously examined through other existing and well accepted protein stability testing methods.      Therefore,our team decided to use the protein known as BINDI.      BINDI and two of its less stable variants, BbpD04 and BbpD04.3 were studied and examined in "A Computationally Designed Inhibitor of an Epstein-Barr Viral Bcl-2 Protein Induces Apoptosis in Infected Cells" by Procko et al1.      We would like to acknowledge and thank Dr. Procko for giving us his genes to work with.
PyE1 a strain of Saccharomyces cerevisiae
We use a strain of Saccharomyces cerevisiae deveoloped in Stan Fields' lab at the University of Washington called PyE1.                    Its genome has been engineered to contain a gene from Green Fluorescent Protein (GFP) under a Gal1 promoter.                    When the Gal4 DNA-binding domain and the VP16 transcription activation domain are colocalized to the Gal1 promoter, expression of GFP is induced.      Therefore, using our test plasmids in PyE1 generates GFP relative to the level of Gal4/VP16 peptide in the cell.      The more stable the degron protein construct is, the more likely it is that more GFP will be expressed.     This relationship between stability and GFP forms the basis from which we will measure the relative protein stability of our degron constructs as well as the protein of interest degron construct.
1Procko, E, et al. "A Computationally Designed Inhibitor of an Epstein-Barr Viral Bcl-2 Protein Induces Apoptosis in Infected Cells" Cell 157 (2014): 1644-56.
Retrieved from " http://2014.igem.org/Team:Washington/Our_Project "
Recent changes
What links here
Related changes
Special pages
My preferences
Printable version
Permanent link
Privacy policy
Disclaimers
