<HTML xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en" dir="ltr">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki  ltr ns-0 ns-subject page-Team_Washington_Methods"><DIV id="globalWrapper"><DIV id="top-section"><DIV id="p-logo"><A href="/Main_Page" title="Main Page">&quot;
	    </A></DIV><DIV id="menubar" class="left-menu noprint"><UL><LI class="selected"><A href="/Team:Washington/Methods">Page               </A></LI><LI class="new"><A href="/wiki/index.php?title=Talk:Team:Washington/Methods&amp;action=edit&amp;redlink=1">Discussion               </A></LI><LI><A href="/wiki/index.php?title=Team:Washington/Methods&amp;action=edit">View source               </A></LI><LI><A href="/wiki/index.php?title=Team:Washington/Methods&amp;action=history">History               </A></LI><LI style="color:white;cursor:default">teams</LI></UL></DIV><DIV class="right-menu noprint" id="menubar"><UL><LI id="pt-login"><A href="/wiki/index.php?title=Special:UserLogin&amp;returnto=Team:Washington/Methods" title="You are encouraged to log in; however, it is not mandatory [o]" accesskey="o">Log in</A></LI></UL></DIV><DIV id="search-controls" class="noprint"><FORM action="/Special:Search" id="searchform"> </FORM></DIV></DIV><DIV id="content"><H1 class="firstHeading">Team:Washington/Methods</H1><DIV id="bodyContent"><H3 id="siteSub" class="noprint">From 2014.igem.org</H3><P><TABLE align="center"><TBODY><TR heigth="75px"><NAV id="igemuwmenu-wrap"><UL id="igemuwmenu"><LI><A href="https://2014.igem.org/Team:Washington">Home</A></LI><LI><A href="https://2014.igem.org/Team:Washington">UW 2014</A></LI><LI><A href="https://2013.igem.org/Team:Washington">UW 2013</A></LI><LI><A href="https://2012.igem.org/Team:Washington">UW 2012</A></LI><LI><A href="https://2011.igem.org/Team:Washington">UW 2011</A></LI><LI><A href="https://2010.igem.org/Team:Washington">UW 2010</A></LI><LI><A href="https://2009.igem.org/Team:Washington">UW 2009</A></LI><LI><A href="https://2008.igem.org/Team:University_of_Washington">UW 2008</A></LI><LI><A href="https://2013.igem.org/Main_Page">iGEM Homepage</A></LI></UL><LI><A href="https://2014.igem.org/Team:Washington/Our_Project">Our Project</A></LI><LI><A href="https://2014.igem.org/Team:Washington/Our_Project">Our System</A></LI><LI><A href="https://2014.igem.org/Team:Washington/Methods">Methods</A></LI><LI><A href="https://2014.igem.org/Team:Washington/Results">Results</A></LI><LI><A href="https://2014.igem.org/Team:Washington/Future Plans">Future Plans</A></LI><LI><A href="https://2014.igem.org/Team:Washington/BioBricks">BioBricks</A></LI><LI><A href="https://2014.igem.org/Team:Washington/Outreach">Outreach</A></LI><LI><A href="https://2014.igem.org/Team:Washington/Protocols">Protocols</A></LI><LI><A href="https://2014.igem.org/Team:Washington/Safety">Safety</A></LI><LI><A href="https://2014.igem.org/Team:Washington/Safety">Safety</A></LI><LI><A href="https://2014.igem.org/Team:Washington/ProjectConsiderations">Project Considerations</A></LI><LI><A href="https://2014.igem.org/Team:Washington/JudgingForm">Judging Form</A></LI><LI><A href="https://2014.igem.org/Team:Washington/Our_Team">Our Team</A></LI></NAV></TR></TBODY></TABLE></P><H1><CENTER>Method </CENTER></H1><P align="left">

		The essential process of our system involves cloning and manufacturing of a plasmid in <I> E. coli </I>. Once, the plasmids have been constructed and verified in <I> E. coli </I> they are transformed into <I>S. cerevisiae</I>. The plasmid constructs are then expressed. Following several days of growth the yeast cultures are passed through a Flow Cytometer	and the fluorescence of each cell is measured. Higher fluorescence is associated with higher expression of the protein of interest which in-turn is indicative of higher protein stability. 

	    </P><H2> Cloning in <I> Escherichia coli </I></H2><P align="left">  

		There are five possible degron constructs corresponding to five different positions the degron can take in our construct. Four of the five vectors for our protein of interest DNA code contains contain a degron (Deg1-4) as well as an EcoRI and Nhel111 restriction enzyme cutsite between Gal4 and VP16. Furthermore, the vector also contains a region that encodes Ampicillin resistance as well as autotrophic region that encodes for uracil synthesis in yeast strains. The DNA that encodes our protein of interest, i.e. the insert, is amplified to include both restriction endonuclease cutsites through a polymerase chain reaction. In a subsequent step the amplified fragment is then digested and then ligated with the appropriate vector and transformed into chemically competent<I> E.coli </I> either XL-1 Blue or XL10-Gold strains. Similarly, Deg0(no degron) vector has EcoRI along with a Hind111 cutsite. Using the different cutsites our DNA fragment is prepared using PCR and ligated into the Deg0 vector and then transformed. Once the transformation is complete, the cells are plated onto LB-agar plates supplimented with ampicillin in order to ensure that all <I> E.coli </I> colonies contain our recombinant plasmid. After being grown for a day, several colonies are swiped and added to an overnight culture. The overnight culture is grown overnight and their recombinant plasmid is harvested and sequenced (Sanger sequencing is used through Genewiz Inc.) the next morning.	If the plasmids are correct, we then proceed to create a glycerol stock of the cell culture as well as a miniprep stock of the plasmid in order to conduct further experimentation in yeast. </P><H2> Preparation and Passaging of <I> Saccharomyces cerevisiae </I></H2><P align="left"> 

		Once, plasmids of the five possible degron constructs have been cloned with our three test proteins and verified, they are subsequently transformed in PyE1 a strain of <I> S. cerevisiae </I> with the ability to produce green fluorescent proteins. Following the transformation, the cells are plated onto plates with on a  Selective Dropout (C-Uracil) media and incubated at 30<SUP>o</SUP>C for 2 days. The purpose of the dropout media is to ensure that only cells that contain our plasmid survive as the recombinant plasmid is encoded with a region that allows cells to produce uracil, an essential amino acid. Without the recombinant plasmid, the cell would be fatally deprived of uracil which has been &quot;knocked out&quot; of the plating media. After two days, three colonies are swiped from the plate and added to an overnight culture of 2-3mL Selective Dropout Media C-Uracil and 2% Glucose then incubated for another two days at 30<SUP>o</SUP>C. After another two days of incubation, a 20-50uL aliquot of each culture is &quot;passaged&quot; into another 3mL culture prepared in the same manner as before and incubated for the same duration and temperature as the previous culture. The passaging is done several times after each passage after the second passage, a glycerol stock is prepare from the culture and Flow Cytometry is also run on the culture. </P><P align="left"> 

		The purpose of passaging is to gradually remove excess copies of the plasmid constructs. Excess copies, exceeding one per cell will lead to multiple fold increase in the expression of the degron protein construct. As a result of this, GFP expression will also be increased thus reducing the viability and accuracy of the Flow Cytometry measurements conducted on each cell culture. This is problematic as GFP output will become related to the number of plasmids as well as the stability of the various degron constructs which will likely invalidate any results.

              </P><H2> Construct Stability Analysis with Flow Cytometry </H2><P> 

            After PyE1 cells have been passaged several times and each cell has roughly one copy of our degron construct containing plasmid, the cells are now ready to be run through a flow cytometer. On the morning of the day that the instrument is to be used, the optical density of cultures is read and an aliquot is taken from each culture and diluted to similar optical densities. This is to ensure that all cultures are in the same growth phase and will express our protein of interest degron construct equally so there will be no discrepancies in expression levels that could skew our fluorescence measurements. Once diluted, the cultures are incubated for roughly six hours (3-4 cell doublings). After the short incubation, approximately 200 microliters of each culture is pelleted, and resuspened in PBSF buffer and passed through the flow cytometer. 
    The instrument reads forward and side scatter allowing us to see cell debris or contaminants which can then be gated (excluded) from the actual fluorescent readings. Each construct containing culture is analyzed and both mean fluorescence of a culture as well as fluorescent culture’s population data is collected. Both of which can be used to analyze the relative stabilities of each construct and protein of interest. Higher GFP output is correlated with higher protein stability.

              </P><H2> Mutagenesis through Error Prone Polymerase Chain Reactions (E-PCRs) </H2><P>

		In order to validate our system as being capable of selecting more stable protein variants, we have to produce mutations in our protein of interest. Those mutations could potentially be beneficial and could carry a stabilizing affect on the protein of interest. Error-prone PCR utilizes DNA-polymerase's error-prone nature and further increases the likelyhood of mutations by manipulating the conditions in which, DNA-polymerase operates in, thereby causing the polymerase to create errors in DNA sequencing which in turn will create changes in the protein construct.	Once, the DNA coding of our protein has been changed we can express the mutations and analyze those stabilizing or de-stabilizing affects they have on our protein. The possible mutations can then be analyzed using the degron system and more stable mutations can be easily seen then sequenced.
				
	      </P><H2> Selecting Stable Variants through Fluorescence-Activated Cell Sorting (FACS) </H2><P>

		Once the DNA coding for a protein of interest has been mutated reassembled into our plasmid and transformed into PyE1, we can then analyze the fluorescent output and select cells
					that exhibit higher fluorescence. The nature of our system allows us to conclude that higher fluorescence output corresponds to higher stability
					of the protein of interest. Cells with higher fluorescence are then selected, cultured and the plasmid within them is sequenced.
				Fluorescence-Activated Cell Sorting (FACS) makes our system very high throughput and allows us to analyze a large number of cells and possible mutations.    

			
              </P><DIV class="printfooter">
Retrieved from &quot;<A href="http://2014.igem.org/Team:Washington/Methods">http://2014.igem.org/Team:Washington/Methods</A>&quot;</DIV></DIV></DIV><DIV id="footer-box" class="noprint"><DIV id="footer"><UL id="f-list"><LI id="t-recentchanges"><A href="/Special:RecentChanges" title="Recent changes">Recent changes</A></LI><LI id="t-whatlinkshere"><A href="/Special:WhatLinksHere/Team:Washington/Methods" title="List of all wiki pages that link here [j]" accesskey="j">What links here</A></LI><LI id="t-recentchangeslinked"><A href="/Special:RecentChangesLinked/Team:Washington/Methods" title="Recent changes in pages linked from this page [k]" accesskey="k">Related changes</A></LI><LI id="t-specialpages"><A href="/Special:SpecialPages" title="List of all special pages [q]" accesskey="q">Special pages</A></LI><LI><A href="/Special:Preferences">My preferences</A></LI></UL></DIV><DIV id="footer"><UL id="f-list"><LI id="t-print"><A href="/wiki/index.php?title=Team:Washington/Methods&amp;printable=yes" title="Printable version of this page [p]" accesskey="p">Printable version</A></LI><LI id="t-permalink"><A href="/wiki/index.php?title=Team:Washington/Methods&amp;oldid=395799" title="Permanent link to this revision of the page">Permanent link</A></LI><LI id="privacy"><A href="/2014.igem.org:Privacy_policy" title="2014.igem.org:Privacy policy">Privacy policy</A></LI><LI id="disclaimer"><A href="/2014.igem.org:General_disclaimer" title="2014.igem.org:General disclaimer">Disclaimers</A></LI></UL></DIV></DIV></DIV></BODY></HTML>