"
Page
Discussion
View source
History
teams
Log in
 
Team:UCLA/Notebook/Biobrick
From 2013.igem.org
HOME
PROJECT
TEAM
PARTS
MODELING
HUMAN PRACTICES
NOTEBOOK
SAFETY
ATTRIBUTIONS
JUDGING
Mtd Biobrick Construction
mRNA Display Library Generation
mRNA Display
Contents
1Making the Mtd Biobrick
1.1PCR to generate fragments of mtd
1.2Splicing Overlap-Extension (SOE) PCR to connect fragments 3 and 4
1.3Gibson Assembly of Fragments 1,2, and 3-4
1.4PCR amplification of mtd
Making the Mtd Biobrick
To create this biobrick, we utilized a combination of splicing overlap extension PCR and Gibson Assembly to extract a sequence from the BPP-1 phage's genomic DNA that would be free of the standard biobrick restriction enzyme sites. The protocols for those methods are listed below:
PCR to generate fragments of mtd
In this step, four separate fragments of the mtd gene are created via PCR. This step is necessary to eliminate illegal internal biobrick sites (EcoRI, PstI, NotI) and to add necessary biobrick sites to the 5' and 3' ends.
Fragment
Forward Primer
Reverse Primer
1
CCTTGAATTCGCGGCCGCATCTAGAATGAGTACCGCAGTCCAGTTCCG
GCCTGCGCTGCCGCGTTGCTTCC
2
GGAAGCAACGCGGCAGCGCAGGC
CCAGCGCCTGGAACTCGTTGTAGTTGGGCAGG
3
CCTGCCCAACTACAACGAGTTCCAGGCGCTGG
CCGATCCGCGGCCTCCAGTGTTGG
4
CCAACACTGGAGGCCGCGGATCGG
AAGGCTGCAGCGGCCGCTACTAGTCTCAAGAATCAGG
The 20 µL PCR mix for each fragment is as follows:
Reagent
Volume
mtd genomic template
1.0 µL (4.5 ng total)
10 µM forward primer
1.0 µL
10 µM reverse primer
1.0 µL
10 mM dNTPs
0.4 µL
Buffer HF
4.0 µL
Phusion Polymerase
0.2 µL
ddH2O
12.4 µL
The thermocycler program for each reaction is as follows:
# Cycles
Temperature (°C)
Time
1
98
0:30
30
98
Variable (70 for fragment 1, 69 for fragment 2, 69 for fragment 3, 72 for fragment 3)
72
0:10
0:20
0:15
1
72
5:00
1
4
--
The PCR product is then resolved on a 1% agarose gel and excised and column-extracted.
Splicing Overlap-Extension (SOE) PCR to connect fragments 3 and 4
The 20 µL reaction mix is as follows:
Reagent
Volume or Mass
Fragment 3
20 ng total
Fragment 4
20 ng total
10 µM forward primer
1.0 µL
10 µM reverse primer
1.0 µL
10 mM dNTPs
0.4 µL
Buffer HF
4.0 µL
Phusion Polymerase
0.2 µL
ddH2O
fill to 20 µL
The thermocycler program for this step is as follows:
# Cycles
Temperature (°C)
Time
1
98
0:30
30
98
72
72
0:10
0:20
0:15
1
72
5:00
1
4
--
Gibson Assembly of Fragments 1,2, and 3-4
The mix for the Gibson Assembly is as follows:
Reagent
Concentration or Volume
Fragment 1
0.2 pM
Fragment 2
0.2 pM
Fragment 3-4
0.2 pM
Assembly Master Mix (from New England Biolabs)
fill to 5.0 µL
The mix is then incubated at 50 °C for 1 hour.
PCR amplification of mtd
The mix for the amplification is as follows:
Reagent
Volume or Mass
mtd
20 ng
10 µM forward primer (CCTTGAATTCGCGGCCGCATCTAGAATGAGTACCGCAGTCCAGTTCCG )
0.6 µL
10 µM reverse primer (AAGGCTGCAGCGGCCGCTACTAGTCTCAAGAATCAGG )
0.6 µL
10 mM dNTPs
0.4 µL
2x xtreme buffer
10 µL
KOD xtreme hot start polymerase
0.4 µL
ddH2O
fill to 20 µL
The thermocycler program for the amplification is as follows:
# Cycles
Temperature (°C)
Time
1
94
2:00
35
98
70
68
0:10
0:30
1:20
1
10
--
Retrieved from " http://2013.igem.org/Team:UCLA/Notebook/Biobrick "
Recent changes
What links here
Related changes
Special pages
My preferences
Printable version
Permanent link
Privacy policy
Disclaimers
