"
Page
Discussion
View source
History
teams
Log in
 
Team:UCLA/Project/mrnaDisplay
From 2013.igem.org
HOME
PROJECT
TEAM
PARTS
MODELING
HUMAN PRACTICES
NOTEBOOK
SAFETY
ATTRIBUTIONS
JUDGING
Overview
Natural System
mRNA Display
Project Design
Contents
1 Historical background
2 What is it?
3 How does it work?
4 What does the Mtd have to do with mRNA display?
5 Other approaches for directed protein targeting?
6 What are the advantages of this route / why did we choose this route for our project?
7 Issues with using this route?
Historical background
Richard Roberts and Jack Szostak from Harvard developed this technique in 1997. They were able to generate and identify a myc epitope starting from a random mRNA library using this technique (Roberts).
What is it?
mRNA display is a technique used to select for proteins that bind to specific targets. It allows for the identification of these selected proteins because they are covalently attached to the DNA that codes for them (Roberts).
Overview of mRNA display (Liu)
How does it work?
A library of DNA sequence variants is generated using error prone PCR or PCR with variable primers.
The DNA library is transcribed in vitro to an RNA library.
Using a DNA splint, RNA strands are ligated to single stranded DNA oligonucleotides with puromycin at the 3' end
The RNA-puromycin fusions are then translated in vitro
The Puromycin mimics a tRNA and is incorporated into the A site of the ribosome. However  the puromycin has a non hydrolizable amide bond, therefore terminating translation and releasing  the polypeptide, which is still bound to the RNA template through the puromycin. This results in a library of proteins covalently bonded to the RNA that coded for them.
In order to create a more stable structure, the RNA is converted to a DNA-RNA duplex through in vitro reverse transcription.
The library is screened for the protein that best binds to the target molecule.
Identification of the protein will be simple, as the corresponding DNA strand is still attached to it. PCR of the DNA followed by sequencing yields the exact identity of the protein at the DNA level.
What does the Mtd have to do with mRNA display?
The Mtd protein is the scaffold for our mRNA display experiment. Most proteins are not capable of undergoing the large number of mutations necessary to create a large library while still maintaining their structure and function.  The Mtd is unique in that it has a structure that allows it to be mutagenized in certain sites and still remain stable.  The Mtd has evolved in the natural system of Bordetella bacteriophage to have precisely this property.  Therefore, it acts as an effective scaffold because the majority of the protein is able to remain constant and stable while a small localized region of each protein gets mutagenized (Yuan).
Other approaches for directed protein targeting?
Phage display:   An in vivo technique similar in principle to mRNA display in which bacteriophage are allowed to  replicate to create a diverse library with different binding affinities.  The DNA from the bacteriophage that corresponds to the protein that binds to the target can be isolated and then sequenced.
Ribosome display:  Another in vitro technique that fuses the RNA transcript to the protein that it codes for.  Instead of a puromycin linker between the transcript and the protein, it is the ribosome that links the protein and the template that codes for it (Smith).
What are the advantages of this route / why did we choose this route for our project?
One of the major advantages of the mRNA display approach is that it takes place completely in vitro.  This eliminates the limitations and complications that come with working in vivo and gives us more control over the entire process.  Also, using an in vitro method allows us to create a more diverse library.  Generating the library in vitro can potentially yield several orders of magnitude more protein variants than in vivo systems such as phage display.
With respect to ribosome display, which is also an in vitro method, mRNA display has an advantage when it comes to screening for protein binding.  The disadvantage of ribosome display is that the linker between the protein and the template DNA is the actual ribosome.  This is an issue because the ribosome is a very large protein and it has the potential to interact with the target protein, thereby throwing off the selection process. In contrast, the puromycin link in mRNA display is much smaller and is not likely to interfere in any way with the target protein (Lipovsek).
Issues with using this route?
What we are essentially doing with this project is taking the Mtd protein which comes from an in vivo system (the BPP-1 phage) and using it as the scaffold in our in vitro protein selection system.  We know that in that native system of the BPP-1 phage, a large library of phages are generated with unique variants of the Mtd protein on their tail fibers.  Out of this large library, some phage are able to effectively bind to a certain target using the Mtd protein.
A major concern is that the Mtd protein may not behave the same way in vitro as it does in the natural BPP-1 system.  Although we are theoretically able to generate more variants of Mtd than in the natural phage system, it is possible that isolating it from the phage could seriously affect its binding capabilities.  For example, the native phage has several tail fibers, each with a copy of the Mtd that could potentially bind to a target.  It may be that one isolated Mtd has a very low binding affinity and that it is necessary to have several in order to create a strong enough binding interaction with the target (Dai).
REFERENCES
Dai, W., et al. "Three-dimensional structure of tropism- switching Bordetella bacteriophage." PNAS. 107.9 (2010):4347-52. Print.
Lipovsek. D., A. Pluckthun. "In vitro protein evolution by ribosome display and mRNA display." Journal of Immunological Methods. 290 (2004):51–67. Print.
Liu, R., et al. "Optimized synthesis of RNA-protein fusions for in vitro protein selection". Meth Enzymol. 318 (2000):268–93. Print.
Roberts, R.W., J.W. Szostak. "RNA-peptide fusions for the in vitro selection of peptides and proteins." Proc. Natl. Acad. Sci. USA. 94.23 (1997):12297-302. Print
Smith, G., V. Petrenko. "Phage Display." Chem. Rev. 97 (1997):391-410. Print.
Yuan, T.Z., et al. "Protein Engineering with Biosynthesized Libraries from Bordetella bronchiseptica Bacteriophage." PLoS ONE. 8.2 (2013). Web.
Retrieved from " http://2013.igem.org/Team:UCLA/Project/mrnaDisplay "
Recent changes
What links here
Related changes
Special pages
My preferences
Printable version
Permanent link
Privacy policy
Disclaimers
