<HTML xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en" dir="ltr">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki  ltr ns-0 ns-subject page-Team_Tianjin_Modeling_Expression"><DIV id="globalWrapper"><DIV id="top-section"><DIV id="p-logo"><A href="/Main_Page" title="Main Page">&quot;
	    </A></DIV><DIV id="menubar" class="left-menu noprint"><UL><LI class="selected"><A href="/Team:Tianjin/Modeling/Expression">Page               </A></LI><LI class="new"><A href="/wiki/index.php?title=Talk:Team:Tianjin/Modeling/Expression&amp;action=edit&amp;redlink=1">Discussion               </A></LI><LI><A href="/wiki/index.php?title=Team:Tianjin/Modeling/Expression&amp;action=edit">View source               </A></LI><LI><A href="/wiki/index.php?title=Team:Tianjin/Modeling/Expression&amp;action=history">History               </A></LI><LI style="color:#808080;cursor:default">teams</LI></UL></DIV><DIV class="right-menu noprint" id="menubar"><UL><LI id="pt-login"><A href="/wiki/index.php?title=Special:UserLogin&amp;returnto=Team:Tianjin/Modeling/Expression" title="You are encouraged to log in; however, it is not mandatory [o]" accesskey="o">Log in</A></LI></UL></DIV><DIV id="search-controls" class="noprint"><FORM action="/Special:Search" id="searchform"> </FORM></DIV></DIV><DIV id="content"><H1 class="firstHeading">Team:Tianjin/Modeling/Expression</H1><DIV id="bodyContent"><H3 id="siteSub" class="noprint">From 2012.igem.org</H3><DIV id="wrap"><DIV id="secondwrap" style="border:8px solid #ffbe10"><DIV style="float:left;"><DIV id="secondpane"><P class="menu_head">Modeling Contents</P><DIV class="menu_body"><DIV class="menu_children"><A href="https://2012.igem.org/Team:Tianjin/Modeling">Modeling Home</A></DIV><DIV class="menu_children"><A href="https://2012.igem.org/Team:Tianjin/Modeling/HGTBass">HGTBass</A></DIV><DIV class="menu_children"><A href="https://2012.igem.org/Team:Tianjin/Modeling/Propagation">Optimum Beginning Time</A></DIV><DIV class="menu_children"><A href="https://2012.igem.org/Team:Tianjin/Modeling/Human">Questionnaire Modeling</A></DIV><DIV class="menu_children"><A href="https://2012.igem.org/Team:Tianjin/Modeling/Calculation">Calculation</A></DIV></DIV><P class="menu_head">In this page</P><DIV class="menu_body"><DIV class="menu_children"><A href="#Background">Background</A></DIV><DIV class="menu_children"><A href="#Modeling_Objective">Modeling Objective</A></DIV><DIV class="menu_children"><A href="#Model_Description_and_Design">Model Description and Design</A></DIV><DIV class="menu_children"><A href="#Verification_of_the_Orthogonality">Verification of the Orthogonality</A></DIV><DIV class="menu_children"><A href="#Construction_of_Multi-Orthogonal_Systems">Construction of Multi-Orthogonal Systems</A></DIV></DIV></DIV></DIV><DIV id="text-content"><CENTER><SPAN style="font-size:46px;font-family:Cambria;margin-top:10px">Protein Expression Prediction</SPAN></CENTER><H1><SPAN class="mw-headline" id="Background">Background</SPAN></H1><P>Within our chassis bacteria, the protein expression system is special because there are two channels to express the protein: the canonical channel exists within any wild type bacteria and the orthogonal channel created artificially with the help of orthogonal ribosomes and orthogonal mRNA whose protein expression is relatively independent of the canonical ones. There are two kinds of ribosome and mRNA: host ribosome (also known as normal ribosome, canonical ribosome or n-ribosome) orthogonal-ribosome (o-ribosome), normal-mRNA (n-mRNA) with canonical RBS sequence and orthogonal mRNA (o-mRNA) with mutated RBS sequence. The canonical mRNA can be translated with the help of canonical ribosome. On the other hand, o-ribosomes can also translate genes with altered Shine-Dalgarno (SD) sequences not recognized by host ribosomes.
</P><P>In the case of o-ribosomes, mutations are introduced into the ASD region such that they can base pair with complementary, noncanonical SD sequences. However, we are not familiar with the internal mechanism of how the two kinds of mRNA interact with the two kinds of ribosomes. In order to investigate the mechanism of how the four components interact with each other within our chassis bacteria, we simulate the whole process through modeling.
</P><H1><SPAN class="mw-headline" id="Modeling_Objective">Modeling Objective</SPAN></H1><P>As shown in <B>Figure 1</B>, there are two kinds of ribosome and two mRNA with two orthogonal RBS sequence. There can be four kinds of interactions between the four components: n-16S ~ n-RBS, n-16S ~ o-RBS, o-16S ~ o-RBS, o-16S ~ n-RBS. However, there are many questions which are unclear to us and thus need answering: How can the orthogonal mRNA express their encoded protein? How can the two proteins expressed system interact with each other? Since there are too much known about the mechanism of the protein expressed with coexistence of two protein expression system. Thus, our goal of modeling is to describe the whole process with the help of mathematical tool and to predict the result of the wet lab. Specifically, there are two goals: the verification of orthogonality and the prediction of protein expression level after introducing the orthogonal system by two steps.
</P><DIV class="center"><DIV class="thumb tnone"><DIV class="thumbinner" style="width:339px;"><DIV class="thumbcaption"><B>Figure 1</B>. Basic idea of our model: The four interactions among the four components.(from TJU iGEM Team 2012)</DIV></DIV></DIV></DIV><H1><SPAN class="mw-headline" id="Model_Description_and_Design">Model Description and Design</SPAN></H1><H3><SPAN class="mw-headline" id="How_is_protein_expressed_with_the_existence_of_an_orthogonal_system.3F">How is protein expressed with the existence of an orthogonal system?</SPAN></H3><P>In <B>Figure 1</B>, the solid lines indicate the strong combination of the n-16s – n-RBS and o-16s – o-RBS; and the dotted line stands for the combination of canonical sequence with orthogonal sequence. The translation processes are illustrated in <B>Figure 2</B>. Refer to <A href="#Background">Background</A> for details.
</P><DIV class="center"><DIV class="thumb tnone"><DIV class="thumbinner" style="width:502px;"><DIV class="thumbcaption"><B>Figure 2.</B> Translation process(from reference“Automated design of synthetic ribosome binding sites to control   protein expression”)</DIV></DIV></DIV></DIV><P>The strength of the interaction between SD and ASD sequence is thought to influence translational efficiency as mutations in either the SD or ASD sequence that weaken the interaction reduce the amount of protein made.
The mechanism of protein expression is primarily determined by the delta Gibbs free energy of the combination of SD sequence on ribosome and the ASD sequence on RBS of the mRNA. In most cases, translation initiation is the rate-limiting step. Its rate is determined by multiple molecular interactions, including the hybridization of the 16S rRNA to the RBS sequence, the binding of rRNA to the start codon, the distance between the 16S r RNA binding site and the start codon (called spacing) and the presence of RNA secondary structures that occlude either the 16S rRNA binding site or the standby site.
</P><P>Where, r stands for the translation initiation rate of the protein, ∆G<SUB>tot</SUB> means the total Gibbs free energy change of the SD and ASD sequence. ∆G<SUB>tot</SUB> is more negative when attractive interactions between ribosome and mRNA are present, and ∆G<SUB>tot</SUB> is more positive when mutually exclusive secondary structures are present. β is the apparent Boltzmann constant for the system, which converts thermodynamic free energies to temperature differences.
</P><P>The initiation rate is be proportional to the amount of protein expressed, as the equation (2) described
</P><P>Where, E is the amount of protein expressed. The proportionality factor k<SUB>2</SUB>  accounts for any ribosome-mRNA molecular interactions that are independent of mRNA sequence and any translation-independent parameters. k stands for all the factors exclusive of mRNA and ribosome amount.
</P><H3><SPAN class="mw-headline" id="How_to_calculate_.CE.94Gtot.3F">How to calculate ΔG<SUB>tot</SUB>?</SPAN></H3><P>As for how to calculate ΔG<SUB>tot</SUB>, we need to know how the SD and ASD sequence compliment with each other. The Watson-Crick base pairs and G:U wobbles (red lines) are shown in <B>Figure 3</B>.
</P><DIV class="center"><DIV class="thumb tnone"><DIV class="thumbinner" style="width:602px;"><DIV class="thumbcaption"><B>Figure 3.</B> Initial and final state of translation initiation process (from TJU iGEM Team 2012)</DIV></DIV></DIV></DIV><P>Given a specific mRNA sequence called the sub-sequence surrounding a start codon, ΔGtot is predicted according to an energy model (equation (3)), where the reference state is a fully unfolded sub-sequence with G=0.
</P><P>In such ΔG<SUB>mRNA:rRNA</SUB>  is the energy released when the last nine nucleotides (nt) of the E. coli 16S rRNA (3′-AUUCCUCCA-5′) hybridizes and co-folds to the mRNA sub-sequence (ΔG<SUB>mRNA:rRNA</SUB>&lt; 0). Intramolecular folding within the mRNA is allowed. All possible hybridizations between the mRNA and 16S rRNA are considered to find the highest affinity 16S rRNA binding site. The binding site minimizes the sum of the hybridization free energy ΔG<SUB>mRNA:rRNA</SUB>  and the penalty for nonoptimal spacing, ΔG<SUB>spacing</SUB>. Thus, the algorithm can identify the16S rRNA binding site regardless of its similarity to the consensus Shine-Dalgarno sequence.
</P><P>ΔG<SUB>start</SUB> is the energy released when the start codon hybridizes to the initiating tRNA anticodon loop (3′-UAC-5′).
</P><P>ΔG<SUB>standby</SUB> is the work required to unfold any secondary structures sequestering the standby site (ΔG<SUB>standby</SUB>&lt; 0) after the 30S complex assembly. We define the standby site as the four nucleotides upstream of the 16S rRNA binding site, which is its location in a previously studied mRNA.
</P><P>To calculate ΔG<SUB>mRNA:rRNA</SUB>, ΔG<SUB>start</SUB> ,ΔG<SUB>spacing</SUB>, ΔG<SUB>standby</SUB> and ΔG<SUB>mRNA</SUB>, we use the NUPACK suite of algorithms with the Mfold 3.0 RNA energy parameters. These free energy calculations do not have any additional fitting or training parameters and explicitly depend on the mRNA sequence. 
</P><P>In addition, the free energy terms are not orthogonal; changing a single nucleotide can potentially affect multiple energy terms. The relationship between the spacing and the ΔG<SUB>spacing</SUB> was empirically determined by measuring the protein expression level driven by synthetic RBSs of varying spacing and fitting a quantitative model to this data.
</P><H1><SPAN class="mw-headline" id="Verification_of_the_Orthogonality">Verification of the Orthogonality</SPAN></H1><P>Since the two protein expression systems are orthogonal, the orthogonality should be one of the most important factors in our modeling. So the first important task of the dry lab is to predict the orthogonality.
</P><P>If the two protein expression systems are orthogonal, the protein expressed through pathway 3 and 4 should be minute when compared with pathway 1 and 2. Therefore, we firstly calculate the individual protein expression level. 
</P><P>The individual ΔGtot is calculated according to the formula in previous section. Refer to the <A href="/Team:Tianjin/Modeling/Calculation" title="Team:Tianjin/Modeling/Calculation">Calculation page</A> for details.
</P><SPAN style="font-size:12px"><CENTER><B>Table 1.</B> RFP calculation result of ΔG (from TJU iGEM Team 2012)</CENTER></SPAN><P><CENTER><TABLE border="0" cellpadding="5" style="border:2px solid #805f08;"><TBODY><TR><TH style="background:#ffde87;">Pathway 1</TH><TH style="background:#ffde87;">Pathway 2</TH><TH style="background:#ffde87;">Pathway 3</TH><TH style="background:#ffde87;">Pathway 4</TH></TR><TR><TD style="background:#ffefc3;">  ΔG<SUB>tot</SUB>(kcal/mol)  </TD><TD style="background:#ffefc3;">  ΔG<SUB>tot</SUB>(kcal/mol)  </TD><TD style="background:#ffefc3;">  ΔG<SUB>tot</SUB>(kcal/mol)  </TD><TD style="background:#ffefc3;">  ΔG<SUB>tot</SUB>(kcal/mol)  </TD></TR><TR><TD>          -4.7</TD><TD>          -6.28</TD><TD>          3.73</TD><TD>          4.93</TD></TR></TBODY></TABLE></CENTER></P><P>According to the equation of protein expression level:
</P><P>And under the hypothesis that the k, m, and Rtot remain the same in the four pathways because they happen in the same cell, we can arrive at the following results:
</P><P>The results are shown in <B>Figure 4.</B> According to the results, we know that Pathway
</P><DIV class="center"><DIV class="thumb tnone"><DIV class="thumbinner" style="width:502px;"><DIV class="thumbcaption"><B>Figure 4.</B> Protein expression level(from TJU iGEM Team 2012)</DIV></DIV></DIV></DIV><P>E<SUB>1</SUB> and E<SUB>2</SUB> express considerable amount of protein, which indicates that the orthogonal system works just fine as works the canonical system. Furthermore, E<SUB>3</SUB> and E<SUB>4</SUB> is negligible compared to other two results. This show that the orthogonal ribosome cannot translate the canonical mRNA, and vice versa. Therefore, our system demonstrated great orthogonality, which goes very well with our experiment.
</P><H1><SPAN class="mw-headline" id="Construction_of_Multi-Orthogonal_Systems">Construction of Multi-Orthogonal Systems</SPAN></H1><P>After the design and construction of two orthogonal systems, the wet lab result verified the accuracy of the model. Then, we also created triple-orthogonal systems within bacteria. The prediction result is shown in <B>Figure 5</B>.
</P><DIV class="center"><DIV class="thumb tnone"><DIV class="thumbinner" style="width:450px;"><DIV class="thumbcaption"><B>Figure 5.</B> Prediciton result of triple-orthogonal system (from TJU iGEM Team 2012)</DIV></DIV></DIV></DIV><P>We can see from the figure that the protein expresion of diaganol line expression amount is significantly larger than that of others. 
</P><P>In the future we can also have the bold goals of the construction of multi-orhtoganol system theoritically feasible.
</P></DIV></DIV></DIV><DIV id="footer"><P>
E-mail：austinamens@gmail.com | Address:Building No.20, No.92 Weijin Road, Tianjin, China | Zip-code：300072
Copyright 2012 © Tianjin University iGEM Team
</P></DIV><DIV class="printfooter">
Retrieved from &quot;<A href="http://2012.igem.org/Team:Tianjin/Modeling/Expression">http://2012.igem.org/Team:Tianjin/Modeling/Expression</A>&quot;</DIV></DIV></DIV><DIV id="footer-box" class="noprint"><DIV id="footer"><UL id="f-list"><LI id="t-recentchanges"><A href="/Special:RecentChanges" title="Recent changes">Recent changes</A></LI><LI id="t-whatlinkshere"><A href="/Special:WhatLinksHere/Team:Tianjin/Modeling/Expression" title="List of all wiki pages that link here [j]" accesskey="j">What links here</A></LI><LI id="t-recentchangeslinked"><A href="/Special:RecentChangesLinked/Team:Tianjin/Modeling/Expression" title="Recent changes in pages linked from this page [k]" accesskey="k">Related changes</A></LI><LI id="t-specialpages"><A href="/Special:SpecialPages" title="List of all special pages [q]" accesskey="q">Special pages</A></LI><LI><A href="/Special:Preferences">My preferences</A></LI></UL></DIV><DIV id="footer"><UL id="f-list"><LI id="t-print"><A href="/wiki/index.php?title=Team:Tianjin/Modeling/Expression&amp;printable=yes" title="Printable version of this page [p]" accesskey="p">Printable version</A></LI><LI id="t-permalink"><A href="/wiki/index.php?title=Team:Tianjin/Modeling/Expression&amp;oldid=296178" title="Permanent link to this revision of the page">Permanent link</A></LI><LI id="privacy"><A href="/2012.igem.org:Privacy_policy" title="2012.igem.org:Privacy policy">Privacy policy</A></LI><LI id="disclaimer"><A href="/2012.igem.org:General_disclaimer" title="2012.igem.org:General disclaimer">Disclaimers</A></LI></UL></DIV></DIV></DIV></BODY></HTML>