<HTML xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en" dir="ltr">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki  ltr ns-0 ns-subject page-Team_Tianjin_Modeling_Calculation"><DIV id="globalWrapper"><DIV id="top-section"><DIV id="p-logo"><A href="/Main_Page" title="Main Page">&quot;
	    </A></DIV><DIV id="menubar" class="left-menu noprint"><UL><LI class="selected"><A href="/Team:Tianjin/Modeling/Calculation">Page               </A></LI><LI class="new"><A href="/wiki/index.php?title=Talk:Team:Tianjin/Modeling/Calculation&amp;action=edit&amp;redlink=1">Discussion               </A></LI><LI><A href="/wiki/index.php?title=Team:Tianjin/Modeling/Calculation&amp;action=edit">View source               </A></LI><LI><A href="/wiki/index.php?title=Team:Tianjin/Modeling/Calculation&amp;action=history">History               </A></LI><LI style="color:#808080;cursor:default">teams</LI></UL></DIV><DIV class="right-menu noprint" id="menubar"><UL><LI id="pt-login"><A href="/wiki/index.php?title=Special:UserLogin&amp;returnto=Team:Tianjin/Modeling/Calculation" title="You are encouraged to log in; however, it is not mandatory [o]" accesskey="o">Log in</A></LI></UL></DIV><DIV id="search-controls" class="noprint"><FORM action="/Special:Search" id="searchform"> </FORM></DIV></DIV><DIV id="content"><H1 class="firstHeading">Team:Tianjin/Modeling/Calculation</H1><DIV id="bodyContent"><H3 id="siteSub" class="noprint">From 2012.igem.org</H3><DIV id="wrap"><DIV id="secondwrap" style="border:8px solid #ffbe10"><DIV style="float:left;"><DIV id="secondpane"><P class="menu_head">Modeling Contents</P><DIV class="menu_body"><DIV class="menu_children"><A href="https://2012.igem.org/Team:Tianjin/Modeling">Modeling Home</A></DIV><DIV class="menu_children"><A href="https://2012.igem.org/Team:Tianjin/Modeling/Expression">Protein Expression</A></DIV><DIV class="menu_children"><A href="https://2012.igem.org/Team:Tianjin/Modeling/HGTBass">HGTBass</A></DIV><DIV class="menu_children"><A href="https://2012.igem.org/Team:Tianjin/Modeling/Propagation">Optimum Beginning Time</A></DIV><DIV class="menu_children"><A href="https://2012.igem.org/Team:Tianjin/Modeling/Human">Questionnaire Modeling</A></DIV></DIV><P class="menu_head"><A href="#Calculation_and_Derivation_of_the_Protein_Expression_Amount_Model_in_Three_States">Protein Expression Model</A></P><DIV class="menu_body"><DIV class="menu_children"><A href="#Overview">Overview</A></DIV><DIV class="menu_children"><A href="#Input_of_Our_Calculation">Input of Our Calculation</A></DIV><DIV class="menu_children"><A href="#Formula_Derivation">Formula Derivation</A></DIV><DIV class="menu_children"><A href="#The_Calculation">The Calculation</A></DIV><DIV class="menu_children"><A href="#Calculation_Results_of_.CE.94G">Modeling Based on Wet Lab and its Results</A></DIV><DIV class="menu_children"><A href="#Prediction_of_Protein_Expression_Amount">Prediction Of Protein Expression Amount</A></DIV><DIV class="menu_children"><A href="#Model_Extension">Model Extension</A></DIV></DIV><P class="menu_head"><A href="#Analytical_hierarchy_process_.28AHP.29">Analytical Hierarchy Process</A></P><DIV class="menu_body"><DIV class="menu_children"><A href="#What_is_AHP.3F">What is AHP?</A></DIV><DIV class="menu_children"><A href="#Is_there_any_application_in_real_life.3F">Application in Real Life</A></DIV><DIV class="menu_children"><A href="#The_AHP_procedure">The AHP Procedure</A></DIV><DIV class="menu_children"><A href="#Calculation_result_through_AHP_method">Calculation Result Through AHP Method</A></DIV><DIV class="menu_children"><A href="#References">References</A></DIV></DIV><P class="menu_head"><A href="#Principal_component_analysis_.28PCA.29">Principal Component Analysis</A></P><DIV class="menu_body"><DIV class="menu_children"><A href="#What_is_the_problem.3F">What is the problem?</A></DIV><DIV class="menu_children"><A href="#What_is_principal_component_analysis.3F">Principal Component Analysis</A></DIV><DIV class="menu_children"><A href="#How_to_use_PAC.3F">Usage of PAC</A></DIV><DIV class="menu_children"><A href="#References_2">References</A></DIV></DIV></DIV></DIV><DIV id="text-content"><CENTER><SPAN style="font-size:46px;font-family:Cambria;margin-top:10px;line-height:80%">Detail Calculation Method and Results</SPAN></CENTER><H1><SPAN class="mw-headline" id="Calculation_and_Derivation_of_the_Protein_Expression_Amount_Model_in_Three_States">Calculation and Derivation of the Protein Expression Amount Model in Three States</SPAN></H1><H2><SPAN class="mw-headline" id="Overview">Overview</SPAN></H2><P>Three problems came up while we start calculating: what sequences to input, which method to use and how cogent the result will be.
As for sequence, we input the both the SD and the protein coding sequence.
</P><P>Our goal is to calculate the total ΔG of each reaction and then predict the amount of protein expressed. There are several softwares dealing with DNA or RNA base-pairing progress, such as NUPACK, RBS-Calculator, and Vienna RNA etc. After comparison, we decide to use RBS-Calculator. 
</P><P>Due to the complication of translation progress and our lack of insight in this issue, the results of our modeling can’t be very precise. But at least it should have the precision of order of magnitude.
</P><H2><SPAN class="mw-headline" id="Input_of_Our_Calculation">Input of Our Calculation</SPAN></H2><H3><SPAN class="mw-headline" id="RBS_sequence">RBS sequence</SPAN></H3><UL><LI>RFP, normal RBS
</LI></UL><P>ATTTCACACATACTAGAGAAAGAGGAGAAATACTAGATGGCTTCCTCCGAAGACGTTATCAAAGAGTT
</P><P>CATGCGTT
</P><UL><LI>RFP, orthogonal RBS
</LI></UL><P>ATTTCACACATGTTCCGTACTAGATGGCTTCCTCCGAAGACGTTATCAAAGAGTTCATGCGTT
</P><UL><LI>GFP, normal RBS
</LI></UL><P>TACTAGAGAAAGAGGAGAAATACTAGATGCGTAAAGGAGAAGAACTTTTCACTGGAGTTGTCCCAAT
</P><P>TCTTGTT
</P><H3><SPAN class="mw-headline" id="16S_rRNA_sequence">16S rRNA sequence</SPAN></H3><UL><LI>normal 16S: ACCTCCTTA
</LI><LI>orthogonal 16S: ACGGAACTA
</LI></UL><H2><SPAN class="mw-headline" id="Formula_Derivation">Formula Derivation</SPAN></H2><H3><SPAN class="mw-headline" id="Basic_Idea">Basic Idea</SPAN></H3><P>For model design, please refer to 
<A href="/Team:Tianjin/Modeling#Our_Project" title="Team:Tianjin/Modeling">Design part</A>.
Data of the curve Er-time is obtained from E<SUB>c</SUB>=K∙E<SUB>r</SUB> experiments. The function of our model is to work out the proportion factor 'K'.
</P><H3><SPAN class="mw-headline" id="Basic_Assumption">Basic Assumption</SPAN></H3><OL><LI>The expression of RFP and GFP are independent.
</LI><LI>The expression of the two proteins are determined by the percentage of normal and orthogonal ribosomes rather than the number of the two ribosomes.
</LI><LI>The growth curve of bacteria do not change significantly after the transferred into orthogonal protein expression system.
</LI></OL><H3><SPAN class="mw-headline" id="Formula_Derivation_2">Formula Derivation</SPAN></H3><P>We start from Formula 1 and 2. In Formula 1, m stands for the number of mRNA transcript, R<SUB>tot</SUB> is the total number of ribosomes, β is the apparent Boltzmann constant, ∆G<SUB>tot</SUB> is the total change of Gibbs free energy, k is proportion factor. 
</P><P>Because the GFP and RFP coding sequence are on the same mRNA transcript, the values of m of GFP and RFP are always same. In different state and time, the total number of ribosomes varies. We assume that at the same time R<SUB>tot</SUB> of different state remain same. Calculation of DG<SUB>tot</SUB> is the main job of this model. The proportion factor 'k' represents all unknown factors. Here we assume that as for the same protein in deferent state 'k' varies little. 
</P><H2><SPAN class="mw-headline" id="The_Calculation">The Calculation</SPAN></H2><H3><SPAN class="mw-headline" id="Control_State_.28c.29">Control State (c)</SPAN></H3><DIV class="center"><DIV class="thumb tnone"><DIV class="thumbinner" style="width:502px;"><DIV class="thumbcaption"><B>Figure 1.</B> Diagram for control state</DIV></DIV></DIV></DIV><P>The two formulas above serve as denominators in following deduction. We get series of disjointed data of Function 3 and 4 through experiments. The amount of GFP and RFP can't be measured directly so we measured the fluorescence intensity of each protein. And because all the formula in this model are based on a singular cell, we must consider the influence caused by the number and growing condition of bacteria.
</P><H3><SPAN class="mw-headline" id="Experimental_state_1_.28E1.29">Experimental state 1 (E1)</SPAN></H3><DIV class="center"><DIV class="thumb tnone"><DIV class="thumbinner" style="width:502px;"><DIV class="thumbcaption"><B>Figure 2.</B> Diagram for experimental state 1</DIV></DIV></DIV></DIV><P>The expression of GFP is composed of two part, the expression of n-RBS::n-16S and of n-RBS::o-16S.
</P><P>There is no difference for the expression of GFP in Experimental state ONE from the control state, except for the distribution of ribosomes. For this reason, we presume that kc1,G and kr,G are nearly equal and thus Kc1,G equal to 1. The same thought is also shown in following derivation.
</P><P>We noticed that the proportion factors in equation 6 and 8 are equal, which is not a coincident.  This is because ΔG<SUB>3,G</SUB>-ΔG<SUB>1,G</SUB>≈ΔG<SUB>3,R</SUB>-ΔG<SUB>1,R</SUB>. The difference of ΔG<SUB>3,G</SUB> and ΔG<SUB>1,G</SUB>.
</P><H3><SPAN class="mw-headline" id="Experimental_state_2_.28E2.29">Experimental state 2 (E2)</SPAN></H3><DIV class="center"><DIV class="thumb tnone"><DIV class="thumbinner" style="width:502px;"><DIV class="thumbcaption"><B>Figure 3.</B> Diagram for experimental state 2</DIV></DIV></DIV></DIV><P>Note : Strictly speaking, the factor K should better be obtained from experimental data rather than assumed to be 1 for such simplification could lead to much deviation from real value.
</P><H3><SPAN class="mw-headline" id="The_Calculation_of_.CE.94Gtot">The Calculation of ΔG<SUB>tot</SUB></SPAN></H3><P>In most cases, the difference of delta G among the four pathways are mainly reflected by the ΔG<SUB>mRNA-rRNA</SUB>. There are two ways to calculate the ΔG<SUB>mRNA-rRNA</SUB>:
</P><OL><LI>With the help of RBS calculator;
</LI><LI>Use the method in literature Computational design of orthogonal ribosomes.
</LI></OL><P>There are some differences between the input and output of the two method.
</P><P>1st method:
</P><UL><LI>input:
</LI><LI>standby + RBS + Spacing + Start Codon + Protein-Coding,
</LI><LI>16S 3' last nine bases;
</LI></UL><LI>output: ΔG<SUB>mRNA:rRNA</SUB>, ΔG<SUB>start</SUB>, ΔG<SUB>spacing</SUB>, ΔG<SUB>standby</SUB>, ΔG<SUB>mRNA</SUB>;
</LI><LI>Software: RBS-Calculator;
</LI><LI>Strength: taking more factors into our consideration and are more accessible to real condition;
</LI><LI>Weakness: not sure of the input sequence.
</LI><P>2nd method:
</P><UL><LI>input:
</LI><LI>ASD sequence,
</LI><LI>SD sequence (each are 6 bases long);
</LI></UL><LI>Output: ΔG under different conditions;
</LI><LI>Software: RNA-Cofold in Vienna RNA web servers;
</LI><LI>Strength: only need to input the SD and ASD sequence which is very easy;
</LI><LI>Weakness: not as accurate as the first method.
</LI><P>Similarity of the two method：The core program of RBS-Calculator is based on Vienna RNA.
</P><P>There are also two points that should be focused on during our modeling:
</P><OL><LI>the specification of input;
</LI><LI>the analysis of errors if there are any in the output.
</LI></OL><P>As for analysis of errors, the most frequent errors are the Long-Range Paring which occurred when the head and the tile of the mRNA sequence complement with each other within the sequence itself. In such case, the ΔG<SUB>mRNA</SUB> result is not accurate. We usually use RNA-Fold to calculate the accurate ΔG<SUB>mRNA</SUB> to avoid such error.
</P><H2><SPAN class="mw-headline" id="Calculation_Results_of_.CE.94G">Calculation Results of ΔG</SPAN></H2><H2><SPAN class="mw-headline" id="Prediction_of_Protein_Expression_Amount">Prediction of Protein Expression Amount</SPAN></H2><P>We should, first of all, measure the relative protein expression amount (fluorescence intensity) and to obtain the parameters in the model through regression. After the obtaining the model parameters, we can predict the amount of protein expressed and to compare them with that of wet lab results. 
</P><P>With the calculation result from the previous relative expression formula, together with the wet lab result of fluoresce intensity of RFP which is mentioned in the notebook, we can have the exact value of percentage of normal ribosome <B>x</B>.
</P><P>
After the calculation, we can obtain that x=0.9737, y=0.0263.
</P><P>When determine the relative expression level, the GFP intensity should be the same for experiment state one and state two. However, there are some divergences between the GFP intensity of a single cell between the two states. We rationally assume the divergence in the total intensity is result from the number of bacteria (in other words, the OD value leads to the different GFP intensity). So the intensity of RFP for a single cell should also be adjusted.
</P><P>We can put the value of x and y into the formula of the relative protein expression ratio.  What is more, we also measured the Florence intensity curve of the control state. With the obtained value of the ratio we have predicted the curve of the experimental state one and the experimental state two. The result figure of our prediction process is shown in the following four figures.
</P><DIV class="center"><DIV class="thumb tnone"><DIV class="thumbinner" style="width:476px;"><DIV class="thumbcaption"><B>Figure 4.</B> Workflow of our modeling prediction</DIV></DIV></DIV></DIV><H2><SPAN class="mw-headline" id="Model_Extension">Model Extension</SPAN></H2><P>In the previous protein expression model, we just suppose the existence of the orthogonal expression system do not have any significant impact on our system. However, to make our system more accurate, we also need to take the factor of estimation.
</P><H1><SPAN class="mw-headline" id="Analytical_hierarchy_process_.28AHP.29">Analytical hierarchy process (AHP)</SPAN></H1><H2><SPAN class="mw-headline" id="What_is_AHP.3F">What is AHP?</SPAN></H2><P>The analytic hierarchy process (AHP) is a structured technique for organizing and analyz-ing complex decisions. Based on mathematics and psychology, it was developed by Thomas L. Saaty in the 1970s and has been extensively studied and refined since then.
</P><P>It has particular application in group decision making, and is used around the world in a wide variety of decision situations, in fields such as government, business, industry, healthcare, and education.
</P><P>Rather than prescribing a &quot;correct&quot; decision, the AHP helps decision makers find one that best suits their goal and their understanding of the problem. It provides a comprehensive and ra-tional framework for structuring a decision problem, for representing and quantifying its ele-ments, for relating those elements to overall goals, and for evaluating alternative solutions.
</P><H2><SPAN class="mw-headline" id="Is_there_any_application_in_real_life.3F">Is there any application in real life?</SPAN></H2><P>While it can be used by individuals working on straightforward decisions, the Analytic Hierarchy Process (AHP) is most useful where teams of people are working on complex problems, especially those with high stakes, involving human perceptions and judgments, whose reso-lutions have long-term repercussions. It has unique advantages when important elements of the decision are difficult to quantify or compare, or where communication among team members is impeded by their different specializations, terminologies, or perspectives.
</P><P>Decision situations to which the AHP can be applied include:
</P><UL><LI>Choice - The selection of one alternative from a given set of alternatives, usually where there are multiple decision criteria involved.
</LI><LI>Ranking - Putting a set of alternatives in order from most to least desirable
</LI><LI>Prioritization - Determining the relative merit of members of a set of alternatives, as opposed to selecting a single one or merely ranking them
</LI><LI>Resource allocation - Apportioning resources among a set of alternatives
</LI><LI>Benchmarking - Comparing the processes in one's own organization with those of other best-of-breed organizations
</LI><LI>Quality management - Dealing with the multidimensional aspects of quality and quality improvement
</LI><LI>Conflict resolution - Settling disputes between parties with apparently incompatible goals or positions
</LI></UL><H2><SPAN class="mw-headline" id="The_AHP_procedure">The AHP procedure</SPAN></H2><P>Users of the AHP first decompose their decision problem into a hierarchy of more easily comprehended sub-problems, each of which can be analyzed independently. The elements of the hierarchy can relate to any aspect of the decision problem—tangible or intangible, carefully measured or roughly estimated, well- or poorly-understood—anything at all that applies to the decision at hand.
</P><P>Once the hierarchy is built, the decision makers systematically evaluate its various elements by comparing them to one another two at a time, with respect to their impact on an element above them in the hierarchy. In making the comparisons, the decision makers can use concrete data about the elements, but they typically use their judgments about the elements' relative meaning and importance. It is the essence of the AHP that human judgments, and not just the underlying information, can be used in performing the evaluations.
</P><P>The AHP converts these evaluations to numerical values that can be processed and compared over the entire range of the problem. A numerical weight or priority is derived for each element of the hierarchy, allowing diverse and often incommensurable elements to be compared to one another in a rational and consistent way. This capability distinguishes the AHP from other decision making techniques.
</P><P>In the final step of the process, numerical priorities are calculated for each of the decision alternatives. These numbers represent the alternatives' relative ability to achieve the decision goal, so they allow a straightforward consideration of the various courses of action.
</P><P>Several firms supply computer software to assist in using the process.
</P><DIV class="center"><DIV class="thumb tnone"><DIV class="thumbinner" style="width:502px;"><DIV class="thumbcaption"><B>Figure 5.</B> An overall frame diagram</DIV></DIV></DIV></DIV><H2><SPAN class="mw-headline" id="Calculation_result_through_AHP_method">Calculation result through AHP method</SPAN></H2><H3><SPAN class="mw-headline" id="IGEM_Project_evaluation_model">IGEM Project evaluation model</SPAN></H3><DIV class="center"><DIV class="thumb tnone"><DIV class="thumbinner" style="width:467px;"><DIV class="thumbcaption"><B>Figure 6.</B> the performance of the three teams from the perspective of direct economic factor(From TJU iGEM Team 2012)</DIV></DIV></DIV></DIV><DIV class="center"><DIV class="thumb tnone"><DIV class="thumbinner" style="width:467px;"><DIV class="thumbcaption"><B>Figure 7.</B> the performance of the three teams from the perspective of overall economic factor(From TJU iGEM Team 2012)</DIV></DIV></DIV></DIV><DIV class="center"><DIV class="thumb tnone"><DIV class="thumbinner" style="width:466px;"><DIV class="thumbcaption"><B>Figure 8.</B> the overall performance of the three teams(From TJU iGEM Team 2012)</DIV></DIV></DIV></DIV><DIV class="center"><DIV class="thumb tnone"><DIV class="thumbinner" style="width:466px;"><DIV class="thumbcaption"><B>Figure 9.</B> the performance of the three teams from the perspective of social welfare factor(From TJU iGEM Team 2012)</DIV></DIV></DIV></DIV><DIV class="center"><DIV class="thumb tnone"><DIV class="thumbinner" style="width:466px;"><DIV class="thumbcaption"><B>Figure 10.</B> the performance of the three teams from the perspective of green and sustainability(From TJU iGEM Team 2012)</DIV></DIV></DIV></DIV><DIV class="center"><DIV class="thumb tnone"><DIV class="thumbinner" style="width:466px;"><DIV class="thumbcaption"><B>Figure 11.</B> the performance of the three teams from the perspective of commercialized degree(From TJU iGEM Team 2012)</DIV></DIV></DIV></DIV><DIV class="center"><DIV class="thumb tnone"><DIV class="thumbinner" style="width:466px;"><DIV class="thumbcaption"><B>Figure 12.</B> the performance of the three teams from the perspective of direct academic level(From TJU iGEM Team 2012)</DIV></DIV></DIV></DIV><DIV class="center"><DIV class="thumb tnone"><DIV class="thumbinner" style="width:466px;"><DIV class="thumbcaption"><B>Figure 13.</B> the performance of the three teams from the perspective of indirect economic factor(From TJU iGEM Team 2012)</DIV></DIV></DIV></DIV><H3><SPAN class="mw-headline" id="Determination_of_the_Optimum_propaganda_method">Determination of the Optimum propaganda method</SPAN></H3><DIV class="center"><DIV class="thumb tnone"><DIV class="thumbinner" style="width:466px;"><DIV class="thumbcaption"><B>Figure 14.</B></DIV></DIV></DIV></DIV><DIV class="center"><DIV class="thumb tnone"><DIV class="thumbinner" style="width:466px;"><DIV class="thumbcaption"><B>Figure 15.</B></DIV></DIV></DIV></DIV><DIV class="center"><DIV class="thumb tnone"><DIV class="thumbinner" style="width:466px;"><DIV class="thumbcaption"><B>Figure 16.</B></DIV></DIV></DIV></DIV><H3><SPAN class="mw-headline" id="Universal_Risk_evaluation_model_of_Environmental_problem">Universal Risk evaluation model of Environmental problem</SPAN></H3><DIV class="center"><DIV class="thumb tnone"><DIV class="thumbinner" style="width:466px;"><DIV class="thumbcaption"><B>Figure 17.</B> the performance of the four environmental problems from the perspective of prevention and cure(From TJU iGEM Team 2012)</DIV></DIV></DIV></DIV><DIV class="center"><DIV class="thumb tnone"><DIV class="thumbinner" style="width:466px;"><DIV class="thumbcaption"><B>Figure 18.</B> the performance of the four environmental problems from the perspective of environmental factor(From TJU iGEM Team 2012)</DIV></DIV></DIV></DIV><DIV class="center"><DIV class="thumb tnone"><DIV class="thumbinner" style="width:466px;"><DIV class="thumbcaption"><B>Figure 19.</B> the overall performance of the four environmental problems(From TJU iGEM Team 2012)</DIV></DIV></DIV></DIV><DIV class="center"><DIV class="thumb tnone"><DIV class="thumbinner" style="width:466px;"><DIV class="thumbcaption"><B>Figure 20.</B> the performance of the four environmental problems from the perspective of public life and health(From TJU iGEM Team 2012)</DIV></DIV></DIV></DIV><DIV class="center"><DIV class="thumb tnone"><DIV class="thumbinner" style="width:466px;"><DIV class="thumbcaption"><B>Figure 21.</B> the performance of the four environmental problems from the perspective of economic factor(From TJU iGEM Team 2012)</DIV></DIV></DIV></DIV><DIV class="center"><DIV class="thumb tnone"><DIV class="thumbinner" style="width:466px;"><DIV class="thumbcaption"><B>Figure 22.</B> the performance of the four environmental problems from the perspective of social factor(From TJU iGEM Team 2012)</DIV></DIV></DIV></DIV><H2><SPAN class="mw-headline" id="References">References</SPAN></H2><OL><LI>Saaty, Thomas L.; Peniwati, Kirti (2008). Group Decision Making: Drawing out and Rec-onciling Differences. Pittsburgh, Pennsylvania: RWS Publica-tions. <A href="/Special:BookSources/9781888603088" class="internal mw-magiclink-isbn">ISBN 978-1-888603-08-8</A>.
</LI><LI>Saaty, Thomas L. (2008-06). Relative Measurement and its Generalization in Decision Making: Why Pairwise Comparisons are Central in Mathematics for the Measurement of Intangible Factors - The Analytic Hierarchy/Network Process. RACSAM (Review of the Royal spanish Academy of Sciences, Series A, Mathematics) 102 (2): 251–318. Retrieved 2008-12-22.
</LI><LI>Bhushan, Navneet; Kanwal Rai (January 2004). Strategic Decision Making: Applying the Analytic Hierarchy Process. London: Springer-Verlag. <A href="/Special:BookSources/1852337567" class="internal mw-magiclink-isbn">ISBN 1-85233-756-7</A>.
</LI><LI>Forman, Ernest H.; Saul I. Gass (2001-07). The analytical hierarchy processan exposition. Operations Research 49 (4): 469–487. doi:10.1287/opre.49.4.469.11231.
</LI></OL><H1><SPAN class="mw-headline" id="Principal_component_analysis_.28PCA.29">Principal component analysis (PCA)</SPAN></H1><H2><SPAN class="mw-headline" id="What_is_the_problem.3F">What is the problem?</SPAN></H2><P>In assessing the main reasons of how the genetic pollution impact the world, we can analyze it from different perspectives, and the evaluation indexes are too numerous and complicated.
</P><P>To make our assessing task easier, we need to classify all these evaluation indexes into various independent types according to the inherent characteristics of the indexes themselves. This is also called dimensionality reduction which can simplify our problem into various types. Through classification, we can also evaluate our projects through different independent perspectives. What is more, this type of classification can also provide some basic knowledge to classes.
</P><P>After the classification we can not only analyze the problems from the main per-spective which avoid the trouble of assessing the same problem from too many indexes, we can also know the performance of one problem in different single prospects.
</P><P>As for how to accomplish this task, we need to use the principal component analysis (PCA) method.
</P><H2><SPAN class="mw-headline" id="What_is_principal_component_analysis.3F">What is principal component analysis?</SPAN></H2><P><B>Principal component analysis (PCA)</B> is a mathematical procedure that uses an orthogonal transformation to convert a set of observations of possibly correlated variables into a set of values of linearly uncorrelated variables called principal components. The number of principal components is less than or equal to the number of original variables. This transformation is defined in such a way that the first principal component has the largest possible variance (that is, accounts for as much of the variability in the data as possible), and each succeeding component in turn has the highest variance possible under the constraint that it be orthogonal to (i.e., uncorrelated with) the preceding components. Principal components are guaranteed to be independent only if the data set is jointly normally distributed. PCA is sensitive to the rela-tive scaling of the original variables. Depending on the field of application, it is also named the discrete Karhunen–Loève transform (KLT), the Hotelling transform or proper orthogonal de-composition (POD).
</P><P>PCA was invented in 1901 by Karl Pearson. Now it is mostly used as a tool in exploratory data analysis and for making predictive models. PCA can be done by eigenvalue decompo-sition of a data covariance (or correlation) matrix or singular value decomposition of a data matrix, usually after mean centering (and normalizing or using Z-scores) the data matrix for each attribute. The results of a PCA are usually discussed in terms of component scores, sometimes called factor scores (the transformed variable values corresponding to a particular data point), and loadings (the weight by which each standardized original variable should be multiplied to get the component score). 
</P><P>PCA is the simplest of the true eigenvector-based multivariate analyses. Often, its operation can be thought of as revealing the internal structure of the data in a way that best explains the variance in the data. If a multivariate dataset is visualized as a set of coordinates in a high-dimensional data space (1 axis per variable), PCA can supply the user with a low-er-dimensional picture, a &quot;shadow&quot; of this object when viewed from its (in some sense) most informative viewpoint. This is done by using only the first few principal components so that the dimensionality of the transformed data is reduced.
</P><P>PCA is closely related to factor analysis. Factor analysis typically incorporates more domain specific assumptions about the underlying structure and solves eigenvectors of a slightly different matrix.
</P><H2><SPAN class="mw-headline" id="How_to_use_PAC.3F">How to use PAC?</SPAN></H2><P>Principal component analysis (PCA) involves a mathematical procedure that transforms a number of (possibly) correlated variables into a (smaller) number of uncorrelated variables called principal components. The first principal component accounts for as much of the var-iability in the data as possible, and each succeeding component accounts for as much of the remaining variability as possible.
</P><H3><SPAN class="mw-headline" id="Determine_the_Objectives.3F">Determine the Objectives?</SPAN></H3><P>Generally, there are two objectives of the PAC listed in the following:
</P><UL><LI>To discover or to reduce the dimensionality of the data set;
</LI><LI>To identify new meaningful underlying variables.
</LI></UL><H3><SPAN class="mw-headline" id="How_to_start.3F">How to start?</SPAN></H3><P>We assume that the multi-dimensional data have been collected in a Table Of Real data matrix, in which the rows are associated with the cases and the columns with the variables.
Traditionally, principal component analysis is performed on the symmetric Covariance matrix or on the symmetric Correlation matrix. These matrices can be calculated from the data matrix. The covariance matrix contains scaled sums of squares and cross products. A correlation matrix is like a covariance matrix but first the variables, i.e. the columns, have been standardized. We will have to standardize the data first if the variances of variables differ much, or if the units of measurement of the variables differ. You can standardize the data in the Table Of Real by choosing Standardize columns.
To perform the analysis, we select the Tabel Of Real data matrix in the list of objects and choose To PCA. This results in a new PCA object in the list of objects.
We can now make a scree plot of the eigenvalues, Draw eigenvalues to get an indication of the importance of each eigenvalue. The exact contribution of each eigenvalue (or a range of ei-genvalues) to the &quot;explained variance&quot; can also be queried.
</P><H3><SPAN class="mw-headline" id="Determining_the_number_of_components">Determining the number of components</SPAN></H3><P>There are two methods to help you to choose the number of components. Both methods are based on relations between the eigenvalues.
</P><UL><LI>Plot the eigenvalues, Draw eigenvalues. If the points on the graph tend to level out (show an &quot;elbow&quot;), these eigenvalues are usually close enough to zero that they can be ignored;
</LI><LI>Limit the number of components to that number that accounts for a certain fraction of the total variance. For example, if you are satisfied with 95 of the total variance explained then use the number you get by the query Get number of components (VAF) no less than 0.95.
</LI></UL><H3><SPAN class="mw-headline" id="How_to_get_the_principal_components">How to get the principal components</SPAN></H3><P>Principal components are obtained by projecting the multivariate data vectors on the space spanned by the eigenvectors. This can be done in two ways:
</P><OL><LI>Directly from the Table Of Real without first forming a PCA object. You can then draw the Configuration or display its numbers. 
</LI><LI>Select a PCA and a Table Of Real object together and choose. In this way you project the Table Of Real onto the PCA's eigen space.
</LI></OL><H3><SPAN class="mw-headline" id="Mathematical_background_on_principal_component_analysis">Mathematical background on principal component analysis</SPAN></H3><P>The mathematical technique used in PCA is called eigen analysis: we solve for the eigenvalues and eigenvectors of a square symmetric matrix with sums of squares and cross products. The eigenvector associated with the largest eigenvalue has the same direction as the first principal component. The eigenvector associated with the second largest eigenvalue determines the direction of the second principal component. The sum of the eigenvalues equals the trace of the square matrix and the maximum number of eigenvectors equals the number of rows (or columns) of this matrix.
</P><H3><SPAN class="mw-headline" id="Algorithms">Algorithms</SPAN></H3><P>If our starting point happens to be a symmetric matrix like the covariance matrix, we solve for the eigenvalue and eigenvectors by first performing a Householder reduction to tridiagonal form, followed by the QL algorithm with implicit shifts.
</P><P>If, conversely, our starting point is the data matrix <B>A</B>, we do not have to form explicitly the matrix with sums of squares and cross products, <B>A′A</B>. Instead, we proceed by a numerically more stable method, and form the singular value decomposition of <B>A</B>, <B>U Σ V′</B>. The matrix <B>V</B> then contains the eigenvectors, and the squared diagonal elements of Σ contain the eigenvalues.
</P><H2><SPAN class="mw-headline" id="References_2">References</SPAN></H2><OL><LI>Pearson, K. (1901). &quot;On Lines and Planes of Closest Fit to Systems of Points in Space&quot; (PDF). Philosophical Magazine 2 (6): 559–572.
</LI><LI>Abdi. H., &amp; Williams, L.J. (2010). &quot;Principal component analysis.&quot;. Wiley Interdisciplinary Reviews: Computational Statistics, 2: 433–459.
</LI><LI>Shaw P.J.A. (2003) Multivariate statistics for the Environmental Sciences, Hod-der-Arnold. <A href="/Special:BookSources/0340807636" class="internal mw-magiclink-isbn">ISBN 0-340-80763-6</A>.[page needed]
</LI><LI><A href="http://www.fon.hum.uva.nl/praat/manual/Principal_component_analysis.html" class="external free" rel="nofollow">http://www.fon.hum.uva.nl/praat/manual/Principal_component_analysis.html</A></LI></OL></DIV></DIV></DIV><DIV id="footer"><P>
E-mail：austinamens@gmail.com | Address:Building No.20, No.92 Weijin Road, Tianjin, China | Zip-code：300072
Copyright 2012 © Tianjin University iGEM Team
</P></DIV><DIV class="printfooter">
Retrieved from &quot;<A href="http://2012.igem.org/Team:Tianjin/Modeling/Calculation">http://2012.igem.org/Team:Tianjin/Modeling/Calculation</A>&quot;</DIV></DIV></DIV><DIV id="footer-box" class="noprint"><DIV id="footer"><UL id="f-list"><LI id="t-recentchanges"><A href="/Special:RecentChanges" title="Recent changes">Recent changes</A></LI><LI id="t-whatlinkshere"><A href="/Special:WhatLinksHere/Team:Tianjin/Modeling/Calculation" title="List of all wiki pages that link here [j]" accesskey="j">What links here</A></LI><LI id="t-recentchangeslinked"><A href="/Special:RecentChangesLinked/Team:Tianjin/Modeling/Calculation" title="Recent changes in pages linked from this page [k]" accesskey="k">Related changes</A></LI><LI id="t-specialpages"><A href="/Special:SpecialPages" title="List of all special pages [q]" accesskey="q">Special pages</A></LI><LI><A href="/Special:Preferences">My preferences</A></LI></UL></DIV><DIV id="footer"><UL id="f-list"><LI id="t-print"><A href="/wiki/index.php?title=Team:Tianjin/Modeling/Calculation&amp;printable=yes" title="Printable version of this page [p]" accesskey="p">Printable version</A></LI><LI id="t-permalink"><A href="/wiki/index.php?title=Team:Tianjin/Modeling/Calculation&amp;oldid=238403" title="Permanent link to this revision of the page">Permanent link</A></LI><LI id="privacy"><A href="/2012.igem.org:Privacy_policy" title="2012.igem.org:Privacy policy">Privacy policy</A></LI><LI id="disclaimer"><A href="/2012.igem.org:General_disclaimer" title="2012.igem.org:General disclaimer">Disclaimers</A></LI></UL></DIV></DIV></DIV></BODY></HTML>