<HTML xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en" dir="ltr">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki  ltr ns-0 ns-subject page-Team_Tianjin_Project_Technology"><DIV id="globalWrapper"><DIV id="top-section"><DIV id="p-logo"><A href="/Main_Page" title="Main Page">&quot;
	    </A></DIV><DIV id="menubar" class="left-menu noprint"><UL><LI class="selected"><A href="/Team:Tianjin/Project/Technology">Page               </A></LI><LI class="new"><A href="/wiki/index.php?title=Talk:Team:Tianjin/Project/Technology&amp;action=edit&amp;redlink=1">Discussion               </A></LI><LI><A href="/wiki/index.php?title=Team:Tianjin/Project/Technology&amp;action=edit">View source               </A></LI><LI><A href="/wiki/index.php?title=Team:Tianjin/Project/Technology&amp;action=history">History               </A></LI><LI style="color:#808080;cursor:default">teams</LI></UL></DIV><DIV class="right-menu noprint" id="menubar"><UL><LI id="pt-login"><A href="/wiki/index.php?title=Special:UserLogin&amp;returnto=Team:Tianjin/Project/Technology" title="You are encouraged to log in; however, it is not mandatory [o]" accesskey="o">Log in</A></LI></UL></DIV><DIV id="search-controls" class="noprint"><FORM action="/Special:Search" id="searchform"> </FORM></DIV></DIV><DIV id="content"><H1 class="firstHeading">Team:Tianjin/Project/Technology</H1><DIV id="bodyContent"><H3 id="siteSub" class="noprint">From 2012.igem.org</H3><DIV id="wrap"><DIV id="secondwrap" style="border:8px solid #F76303"><DIV style="float:left;"><DIV id="secondpane"><P class="menu_head">Project Contents</P><DIV class="menu_body"><DIV class="menu_children"><A href="https://2012.igem.org/Team:Tianjin/Project">Project Home</A></DIV><DIV class="menu_children"><A href="https://2012.igem.org/Team:Tianjin/Project/OrthogonalSystem">Orthogonal System</A></DIV><DIV class="menu_children"><A href="https://2012.igem.org/Team:Tianjin/Project/Regulation">Logic Metabolism Regulation</A></DIV><DIV class="menu_children"><A href="https://2012.igem.org/Team:Tianjin/Project/Gene">Future Work</A></DIV><DIV class="menu_children"><A href="https://2012.igem.org/Team:Tianjin/Data">BioBrick</A></DIV></DIV><P class="menu_head">In this page</P><DIV class="menu_body"><DIV class="menu_children"><A href="#Replace_of_RBS_in_Plasmid">Replace of RBS in Plasmid</A></DIV><DIV class="menu_children"><A href="#Yeast_Assembler">Yeast Assembler</A></DIV><DIV class="menu_children"><A href="#MAGE">MAGE</A></DIV></DIV></DIV></DIV><DIV id="text-content"><CENTER><SPAN style="font-size:46px;font-family:Cambria;margin-top:10px;line-height:80%">Technology</SPAN></CENTER><H1><SPAN class="mw-headline" id="Replace_of_RBS_in_Plasmid">Replace of RBS in Plasmid</SPAN></H1><DIV class="center"><DIV class="thumb tnone"><DIV class="thumbinner" style="width:502px;"><DIV class="thumbcaption"><B>Figure 1.</B> The process of introducing the O-RBS to plasmid (from TJU iGEM Team 2012)</DIV></DIV></DIV></DIV><P>Here, how did we introduce the O-Lock, we just use two primers to build pcr. Those two primers have about 20bp overhands on their 5' end, which contain the O-Locks or other mutations. Then we directly transform those PCR products to E. coli by electroporation-mediated method. Then those PCR productions could cyclize and form the complete plasmids.
</P><H1><SPAN class="mw-headline" id="Yeast_Assembler">Yeast Assembler</SPAN></H1><H3><SPAN class="mw-headline" id="History">History</SPAN></H3><P>Yeast Assembler is based on in vivo homologous recombination in yeast. As for its high efficiency and ease to work with, in vivo homologous recombination in yeast has been widely used for gene cloning, plasmid construction and library creation. In the early of 2008, Zengyi Shao from University of lllinois at Urbana-Champaign, Urbana, used such a method to construct biochemical pathways. Such a method, for its high efficiency in assembling multiple genes, received great popularity since its appearance.
</P><H3><SPAN class="mw-headline" id="The_Difficulty_of_Large_Fragments_Assembly">The Difficulty of Large Fragments Assembly</SPAN></H3><P>The general digestion connection will leave scar and will be limited by the specific cleavage sequences.
</P><DIV class="center"><DIV class="thumb tnone"><DIV class="thumbinner" style="width:134px;"><DIV class="thumbcaption"><B>Figure 2.</B> Enzyme digestion (from the website of dnaQ</DIV></DIV></DIV></DIV><P>Long PCR fragment will suffer the decline of success rate and distortion, etc. 
</P><DIV class="center"><DIV class="thumb tnone"><DIV class="thumbinner" style="width:502px;"><DIV class="thumbcaption"><B>Figure 3.</B> PCR Recombinant &amp; PCR Machine (from the website of dnaQ</DIV></DIV></DIV></DIV><P>However, the construction of some large fragments cannot be avoided, so the development of a low-cost, simple operation, good fidelity, a little limiting factor large DNA fragment assembly method is particularly important.
</P><H3><SPAN class="mw-headline" id="Principles">Principles</SPAN></H3><P>One step assembly into a vector.
</P><DIV class="center"><DIV class="thumb tnone"><DIV class="thumbinner" style="width:428px;"><DIV class="thumbcaption"><B>Figure 4.</B> Principles of Yeast assembler (from &quot;DNA assembler, an in vivo genetic method for rapid construction of biochemical pathways&quot;)</DIV></DIV></DIV></DIV><P>When parts are transformed all parts into Yeast, homologous recombination occurs at the site “x”, and then all little parts are integrated into a vector.
</P><H3><SPAN class="mw-headline" id="Advantages_and_Disadvantages">Advantages and Disadvantages</SPAN></H3><P>Compared with other methods，the “Yeast assembler” are more efficient and useful for large gene assemble.
</P><CENTER><B>Table 1.</B> Advantages compared to the other two methods (from TJU iGEM Team 2012)</CENTER><P><CENTER><TABLE border="0" cellpadding="5" style="border:2px solid #7b3101;"><TBODY><TR><TH style="background:#fbb181;"> </TH><TH style="background:#fbb181;">Yeast Assembler</TH><TH style="background:#fbb181;">SLIC</TH><TH style="background:#fbb181;">Domino Method</TH></TR><TR><TD style="background:#fbb181;">Advantages</TD><TD style="background:#fdd8c0;">High efficient, Fit for large gene assemble rapid</TD><TD style="background:#fdd8c0;">Fit for many little parts into overall</TD><TD style="background:#fdd8c0;">Fit for two large gene parts integration</TD></TR><TR><TD style="background:#fbb181;">Disadvatages</TD><TD>Process is complex</TD><TD>Success rate is low, The size of recombinant is relatively small(&lt;8kb)</TD><TD>Require a great amount of time and labor. Process is sequential</TD></TR></TBODY></TABLE></CENTER></P><H3><SPAN class="mw-headline" id="Completely_Synthesizing_the_Genome_of_Mycoplasma_Genitalium_using_Yeast_Assembler">Completely Synthesizing the Genome of Mycoplasma Genitalium using Yeast Assembler</SPAN></H3><P>In 2008, Gibson from the J. Craig Venter Institute, published an article “one-step assembly in Yeast of 25 overlapping DNA fragments to form a complete synthetic Mycoplasma genitalium genome”. In the article, the author transformed 25 overlapping DNA fragments into Yeast, homologous recombination occurs, and then the whole genome is synthesized.
</P><DIV class="center"><DIV class="thumb tnone"><DIV class="thumbinner" style="width:407px;"><DIV class="thumbcaption"><B>Figure 5.</B> Synthetic Mycoplasma genitalium genome (from &quot;One-step assembly in yeast of 25 overlapping DNA fragments to form a complete synthetic <I>Mycoplasma genitalium</I> genome&quot;)</DIV></DIV></DIV></DIV><P>Construction of a synthetic M. genitalium genome in yeast. Yeast cells were transformed with 25 different overlapping A-series DNA segments (blue arrows; ~17 kb to35 kb each) composing the M. genitalium genome. To assemble these into a complete genome, a single yeast cell (tan) must take up at least one representative of the 25 different DNA fragments and incorporate them in the nucleus (yellow), where homologous recombination occurs. This assembled genome, called JCVI-1.1, is 590,011 bp, including the vector sequence (red triangle) shown internal to A86 – 89.
</P><H1><SPAN class="mw-headline" id="MAGE">MAGE</SPAN></H1><P>We thought that now that we could introduce O-Lock into a cell, could we introduce O-Lock to the whole system. Theoretically speaking, we could make it come true because there is a technology called MAGE (Multiplex Automated Genomic Engineering) which comes from Harvard University. We could use this technology to replace all the N-RBS to O-RBS of the genome. Then we could regulate the metabolic network more freely and accurately by many AND gates which come from the O-Locks.
</P><DIV class="center"><DIV class="thumb tnone"><DIV class="thumbinner" style="width:502px;"><DIV class="thumbcaption"><B>Figure 6.</B> The basic principle of MAGE (from TJU iGEM　Team 2012)</DIV></DIV></DIV></DIV></DIV></DIV></DIV><DIV id="footer"><P>
E-mail：austinamens@gmail.com | Address:Building No.20, No.92 Weijin Road, Tianjin, China | Zip-code：300072
Copyright 2012 © Tianjin University iGEM Team
</P></DIV><DIV class="printfooter">
Retrieved from &quot;<A href="http://2012.igem.org/Team:Tianjin/Project/Technology">http://2012.igem.org/Team:Tianjin/Project/Technology</A>&quot;</DIV></DIV></DIV><DIV id="footer-box" class="noprint"><DIV id="footer"><UL id="f-list"><LI id="t-recentchanges"><A href="/Special:RecentChanges" title="Recent changes">Recent changes</A></LI><LI id="t-whatlinkshere"><A href="/Special:WhatLinksHere/Team:Tianjin/Project/Technology" title="List of all wiki pages that link here [j]" accesskey="j">What links here</A></LI><LI id="t-recentchangeslinked"><A href="/Special:RecentChangesLinked/Team:Tianjin/Project/Technology" title="Recent changes in pages linked from this page [k]" accesskey="k">Related changes</A></LI><LI id="t-specialpages"><A href="/Special:SpecialPages" title="List of all special pages [q]" accesskey="q">Special pages</A></LI><LI><A href="/Special:Preferences">My preferences</A></LI></UL></DIV><DIV id="footer"><UL id="f-list"><LI id="t-print"><A href="/wiki/index.php?title=Team:Tianjin/Project/Technology&amp;printable=yes" title="Printable version of this page [p]" accesskey="p">Printable version</A></LI><LI id="t-permalink"><A href="/wiki/index.php?title=Team:Tianjin/Project/Technology&amp;oldid=294239" title="Permanent link to this revision of the page">Permanent link</A></LI><LI id="privacy"><A href="/2012.igem.org:Privacy_policy" title="2012.igem.org:Privacy policy">Privacy policy</A></LI><LI id="disclaimer"><A href="/2012.igem.org:General_disclaimer" title="2012.igem.org:General disclaimer">Disclaimers</A></LI></UL></DIV></DIV></DIV></BODY></HTML>