<HTML xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en" dir="ltr">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki  ltr ns-0 ns-subject page-Team_Tianjin_Modeling_Regulation"><DIV id="globalWrapper"><DIV id="top-section"><DIV id="p-logo"><A href="/Main_Page" title="Main Page">&quot;
	    </A></DIV><DIV id="menubar" class="left-menu noprint"><UL><LI class="selected"><A href="/Team:Tianjin/Modeling/Regulation">Page               </A></LI><LI class="new"><A href="/wiki/index.php?title=Talk:Team:Tianjin/Modeling/Regulation&amp;action=edit&amp;redlink=1">Discussion               </A></LI><LI><A href="/wiki/index.php?title=Team:Tianjin/Modeling/Regulation&amp;action=edit">View source               </A></LI><LI><A href="/wiki/index.php?title=Team:Tianjin/Modeling/Regulation&amp;action=history">History               </A></LI><LI style="color:#808080;cursor:default">teams</LI></UL></DIV><DIV class="right-menu noprint" id="menubar"><UL><LI id="pt-login"><A href="/wiki/index.php?title=Special:UserLogin&amp;returnto=Team:Tianjin/Modeling/Regulation" title="You are encouraged to log in; however, it is not mandatory [o]" accesskey="o">Log in</A></LI></UL></DIV><DIV id="search-controls" class="noprint"><FORM action="/Special:Search" id="searchform"> </FORM></DIV></DIV><DIV id="content"><H1 class="firstHeading">Team:Tianjin/Modeling/Regulation</H1><DIV id="bodyContent"><H3 id="siteSub" class="noprint">From 2012.igem.org</H3><DIV id="wrap"><DIV id="secondwrap" style="border:8px solid #F76303"><DIV style="float:left;"><DIV id="secondpane"><P class="menu_head">Project Contents</P><DIV class="menu_body"><DIV class="menu_children"><A href="https://2012.igem.org/Team:Tianjin/Project">Project Home</A></DIV><DIV class="menu_children"><A href="https://2012.igem.org/Team:Tianjin/Modeling/OrthogonalSystem">Orthogonal System</A></DIV><DIV class="menu_children"><A href="https://2012.igem.org/Team:Tianjin/Project/Gene">Safety Encryption</A></DIV></DIV><P class="menu_head">In this page</P><DIV class="menu_body"><DIV class="menu_children"><A href="#Background">Background</A></DIV><DIV class="menu_children"><A href="#Yeast Assembler">Yeast Assembler</A></DIV><DIV class="menu_children"><A href="#Synthesizing_the_Pathway_Needed_for_Synthesizing_Violacein">Synthesizing the Pathway Needed for Synthesizing Violacein</A></DIV><DIV class="menu_children"><A href="#Logic_Gate_Metabolic_Regulation">Logic Gate Metabolic Regulation</A></DIV></DIV></DIV></DIV><DIV id="text-content"><CENTER><SPAN style="font-size:46px;font-family:Cambria;margin-top:10px;line-height:80%">Logic Metabolism Regulation</SPAN></CENTER><H1><SPAN class="mw-headline" id="Background">Background</SPAN></H1><DIV class="thumb tright"><DIV class="thumbinner" style="width:222px;"><DIV class="thumbcaption"><B>Figure 1.</B> E. coli (from the website of dnaQ)</DIV></DIV></DIV><H3><SPAN class="mw-headline" id="Metabolic_Regulation">Metabolic Regulation</SPAN></H3><P>Cell metabolism is the foundation of cell growth, secretion, differentiation, etc. as well as the core process of the modern fermentation technology that can make large impact on the quality and output of product.
</P><P>However the modern metabolic regulation of strains has a large amount of areas for improvement. Take the most common E. coli as example. At present, most of the regulation means is to use a single promoter to construct specific operon gene cluster to achieve the regulation with the addition of certain inducers induction of a specific protein. But this induced expression is typically unidirectional, irreversible and it needs to build many complex operons’gene structure to construct multiple logical regulation, this will produce a plenty of limitation. 
</P><P>If we add our O-Key into the expression system, a novel regulation means will exist on the level of translation. Meanwhile, we can combine O-Key with the traditional transcription regulation to create logic gate regulation structure like “AND” gate. The logic gate working with normal close and normal open promoter can constitute multiple “AND/OR/NOR” logic gate control system which has more simple structure compared to traditional regulation.
</P><H3><SPAN class="mw-headline" id="The_Difficulty_of_Large_Fragments_Assembly">The Difficulty of Large Fragments Assembly</SPAN></H3><P>The general digestion connection will leave scar and will be limited by the specific cleavage sequences.
</P><DIV class="center"><DIV class="thumb tnone"><DIV class="thumbinner" style="width:134px;"><DIV class="thumbcaption"><B>Figure 2.</B> Enzyme digestion (from the website of dnaQ</DIV></DIV></DIV></DIV><P>Long PCR fragment will suffer the decline of success rate and distortion, etc. 
</P><DIV class="center"><DIV class="thumb tnone"><DIV class="thumbinner" style="width:502px;"><DIV class="thumbcaption"><B>Figure 3.</B> PCR Recombinant &amp; PCR Machine (from the website of dnaQ</DIV></DIV></DIV></DIV><P>However, the construction of some large fragments cannot be avoided, so the development of a low-cost, simple operation, good fidelity, a little limiting factor large DNA fragment assembly method is particularly important.
</P><H1><SPAN class="mw-headline" id="Yeast_Assembler">Yeast Assembler</SPAN></H1><H3><SPAN class="mw-headline" id="History">History</SPAN></H3><P>Yeast Assembler is based on in vivo homologous recombination in yeast. As for its high efficiency and ease to work with, in vivo homologous recombination in yeast has been widely used for gene cloning, plasmid construction and library creation. In the early of 2008, Zengyi Shao from University of lllinois at Urbana-Champaign, Urbana, used such a method to construct biochemical pathways. Such a method, for its high efficiency in assembling multiple genes, received great popularity since its appearance.
</P><H3><SPAN class="mw-headline" id="Principles">Principles</SPAN></H3><P>One step assembly into a vector.
</P><DIV class="center"><DIV class="thumb tnone"><DIV class="thumbinner" style="width:428px;"><DIV class="thumbcaption"><B>Figure 4.</B> Principles of Yeast assembler (from &quot;DNA assembler, an in vivo genetic method for rapid construction of biochemical pathways&quot;)</DIV></DIV></DIV></DIV><P>When parts are transformed all parts into Yeast, homologous recombination occurs at the site “x”, and then all little parts are integrated into a vector.
</P><H3><SPAN class="mw-headline" id="Advantages_and_Disadvantages">Advantages and Disadvantages</SPAN></H3><P>Compared with other methods，the “Yeast assembler” are more efficient and useful for large gene assemble.
</P><DIV class="center"><DIV class="thumb tnone"><DIV class="thumbinner" style="width:502px;"><DIV class="thumbcaption"><B>Figure 5.</B> Advantages and disadvantages of three assemble methods (from &quot;DNA assembler, an in vivo genetic method for rapid construction of biochemical pathways&quot;)</DIV></DIV></DIV></DIV><H3><SPAN class="mw-headline" id="Completely_Synthesizing_the_Genome_of_Mycoplasma_Genitalium_using_Yeast_Assembler">Completely Synthesizing the Genome of Mycoplasma Genitalium using Yeast Assembler</SPAN></H3><P>In 2008, Gibson from the J. Craig Venter Institute, published an article “one-step assembly in Yeast of 25 overlapping DNA fragments to form a complete synthetic Mycoplasma genitalium genome”. In the article, the author transformed 25 overlapping DNA fragments into Yeast, homologous recombination occurs, and then the whole genome is synthesized.
</P><DIV class="center"><DIV class="thumb tnone"><DIV class="thumbinner" style="width:407px;"><DIV class="thumbcaption"><B>Figure 6.</B> Synthetic Mycoplasma genitalium genome (from &quot;One-step assembly in yeast of 25 overlapping DNA fragments to form a complete synthetic Mycoplasma genitalium genome&quot;)</DIV></DIV></DIV></DIV><P>Construction of a synthetic M. genitalium genome in yeast. Yeast cells were transformed with 25 different overlapping A-series DNA segments (blue arrows; ~17 kb to35 kb each) composing the M. genitalium genome. To assemble these into a complete genome, a single yeast cell (tan) must take up at least one representative of the 25 different DNA fragments and incorporate them in the nucleus (yellow), where homologous recombination occurs. This assembled genome, called JCVI-1.1, is 590,011 bp, including the vector sequence (red triangle) shown internal to A86 – 89.
</P><H1><SPAN class="mw-headline" id="Synthesizing_the_Pathway_Needed_for_Synthesizing_Violacein">Synthesizing the Pathway Needed for Synthesizing Violacein</SPAN></H1><H3><SPAN class="mw-headline" id="Background_2">Background</SPAN></H3><P>In order to verify the abilities of orthogonal system to adjust metabolism, we chose the metabolic pathway of Violacein. The reason why we chose this one are based on the facts that the pathway is suitable to adjust, and has been deeply learned.
</P><P>Violacei, the major pigment produced by Chromobacterium violaceum, is a bactericide, a trypanocide, a tumoricide and in addition it has anti-viral activity.
</P><DIV class="center"><DIV class="thumb tnone"><DIV class="thumbinner" style="width:243px;"><DIV class="thumbcaption"><B>Figure 7.</B> Violacein's structure (from &quot;Production, extraction and purification of violacein: an antibiotic pigment produced by Chromobacterium violaceum&quot;)</DIV></DIV></DIV></DIV><P>The metabolic pathways to produce Violacein are revealed in the following picture.
</P><DIV class="center"><DIV class="thumb tnone"><DIV class="thumbinner" style="width:480px;"><DIV class="thumbcaption"><B>Figure 8.</B> Pathway of Violacein (from BBa_K274002)</DIV></DIV></DIV></DIV><P>From the pathway, we can know that except for the desire product, Violacein, the pathway will also produce side product, deoxyviolacein.
</P><H3><SPAN class="mw-headline" id="Principle">Principle</SPAN></H3><P>In order to verify the feasibility of the orthogonal system in adjusting metabolism, we mutated the RBS of the gene encoding VioD by MAGE, and assembled in Yeast Assembler. The VioD gene and the O-Key gene with pBad promoter were finally transformed into the cell. Because the cell itself doesn’t have the O-Key, the gene is stringently shut down. When we added Ala to induce pBad, the O-Key were produced to open the O-Lock, so the cell could produce violacin.
</P><H3><SPAN class="mw-headline" id="Experiment">Experiment</SPAN></H3><OL><LI>MAGE mutate the RBS of the gene coding VioD to o-RBS
</LI><LI>We construct different parts of Vio operons, and transformed into yeast for assembly(In this part, we also assembly the original genes without mutation)<DIV class="center"><DIV class="thumb tnone"><DIV class="thumbinner" style="width:502px;"><DIV class="thumbcaption"><B>Figure 9.</B> Assembler of Violacein (from TJU iGEM Team 2012)</DIV></DIV></DIV></DIV></LI><LI>Extracted the plasmid of Yeast and then transformed into our O-E.coli on LB plates.
</LI><LI>Verify through Cpcr and inculcated the right colony into liquid LB overnight.
</LI><LI>Extracted the plasmid of E.coli and verified through digestion of ligase.
</LI></OL><H3><SPAN class="mw-headline" id="Results_and_Analysis">Results and Analysis</SPAN></H3><DIV class="center"><DIV class="thumb tnone"><DIV class="thumbinner" style="width:502px;"><DIV class="thumbcaption"><B>Figure 10.</B> Process of experiments (from TJU iGEM Team 2012)</DIV></DIV></DIV></DIV><OL><LI>From the above picture, the process is finished smoothly and the result of Digested verification is right.<DIV class="center"><DIV class="thumb tnone"><DIV class="thumbinner" style="width:502px;"><DIV class="thumbcaption"><B>Figure 11.</B> Results of the normal pathway (from TJU iGEM Team 2012)</DIV></DIV></DIV></DIV></LI><LI>From this picture, we could find that the test tube with the normal pathway, the liquid is purple. In such condition, for PVA have the larger potential to turn into violacein, violacein stand for a large part and liquid is purple. <DIV class="center"><DIV class="thumb tnone"><DIV class="thumbinner" style="width:502px;"><DIV class="thumbcaption"><B>Figure 12.</B> Results of the mutated pathway without adding Ala (from TJU iGEM Team 2012)</DIV></DIV></DIV></DIV></LI><LI>In this picture, we could find that liquid is blue. As we didn’t add Ala into the liquid, the cells didn’t produce O-ribosome, then the vioD could not be expressed. The PVA could only become deoxyviolacein, which is blue. Therefore the liquid is blue. <DIV class="center"><DIV class="thumb tnone"><DIV class="thumbinner" style="width:502px;"><DIV class="thumbcaption"><B>Figure 13.</B> Results of the mutated pathway adding Ala (from TJU iGEM Team 2012)</DIV></DIV></DIV></DIV></LI><LI>In this picture, the liquid is lavender. For this tube, we added Ala, which could induce the production of O-ribosome, then PVA could turn into violacein, then the liquid is lavender.
</LI><LI>From the results of this experiment, we could find that the orthogonal system could work well when used to adjust metabolism, and the effect is satisfied. In the fourth part, we will further discuss the application of the orthogonal system in regulating the metabolism.
</LI></OL><H1><SPAN class="mw-headline" id="Logic_Gate_Metabolic_Regulation">Logic Gate Metabolic Regulation</SPAN></H1><P>Biological logic gate has been described as the most advanced “biological circuit” ever built. This year, we used the orthogonal system to achieve metabolic regulation by logic gate – the O-Key. Our technology is a translational regulation. It not only introduce a new “AND gate” regulation to cells, but also works perfect with transcriptional regulation, waste no sequence resources and has certain potential of encryption.
</P><DIV class="center"><DIV class="thumb tnone"><DIV class="thumbinner" style="width:405px;"><DIV class="thumbcaption"><B>Figure 14.</B> Theory figure of  logic gate metabolic regulation(from TJU iGEM Team 2012)</DIV></DIV></DIV></DIV><P>The core of our O-Key consists of two part: the o-ribosome and o-mRNA. Their roles in metabolic regulation can be described as key and lock. O-ribosome serves as key, while o-mRNA is the lock. We use the o-RBS to “lock” the target sequence, and make it decipherable only under o-ribosome. They altogether forms an AND gate.
</P><DIV class="center"><DIV class="thumb tnone"><DIV class="thumbinner" style="width:502px;"><DIV class="thumbcaption"><B>Figure 15.</B> The method of controling O-Key synthesis (from TJU iGEM Team 2012)</DIV></DIV></DIV></DIV><P>The regulation of o-ribosome can be achieved by four ways: constitutive promoter, chemical inducible systems, temperature-inducible systems, quorum sensing systems。We can chose different pathways according various conditions.
</P><DIV class="center"><DIV class="thumb tnone"><DIV class="thumbinner" style="width:502px;"><DIV class="thumbcaption"><B>Figure 16.</B> Orthoganol regulation of metabolism network (from KEGG)</DIV></DIV></DIV></DIV><P>We use modeling fitting to predict the key nodes in the metabolism network. MAGE can be used to mutate the RBS of target gene to o-RBS thus locking the target gene. Compared with the conventional way of regulating by overexpression and gene knockout, our O-Key avoids decreasing cell activity and slowing growth rate because of partial adjustment of regulation networks by altering multiple cell native control transcription simultaneously. This technology saves time and boost efficiency.
</P><DIV class="center"><DIV class="thumb tnone"><DIV class="thumbinner" style="width:502px;"><DIV class="thumbcaption"><B>Figure 17.</B> The difference between original regulation and orthogonal regulation (from TJU iGEM Team 2012)</DIV></DIV></DIV></DIV><P>We will elaborate on the function of O-Key in the following examples.
</P><DIV class="center"><DIV class="thumb tnone"><DIV class="thumbinner" style="width:502px;"><DIV class="thumbcaption"><B>Figure 18.</B> The first example of metabolism regulation (from TJU iGEM Team 2012)</DIV></DIV></DIV></DIV><P>This pathway has two branches. If we put an O-Lock between C and D, we can adjust the metabolic pathway according to the cell growth cycle. Say I is the primary metabolites, and the bacteria need it for growth, we can close the O-Lock to shut down pathway D. When the bacteria is in stationary phase and in sufficient quantity, we can open the O-Lock to turn on pathway D, and increase the secondary metabolites F by its competitive advantage.
</P><DIV class="center"><DIV class="thumb tnone"><DIV class="thumbinner" style="width:502px;"><DIV class="thumbcaption"><B>Figure 19.</B> The second example of metabolism regulation (from TJU iGEM Team 2012)</DIV></DIV></DIV></DIV><P>If the natural branch competitive advantage does not exist, or we need precise regulation of the metabolism of the two pathway, we can install different O-Key system on the two pathway, and use the O-Key to achieve accurate control.
</P><P>In a word, the O-Key system has two advantages. On the one hand, it can achieve the same function of other complex operons using much more concise sequence.
</P><DIV class="center"><DIV class="thumb tnone"><DIV class="thumbinner" style="width:479px;"><DIV class="thumbcaption"><B>Figure 20.</B> One example of logic metabolism regulation (from &quot;Synthesis of orthogonal transcription-translation networks&quot;)</DIV></DIV></DIV></DIV><P>Moreover, with the combination of different O-Key system we are able to construct more complicated logic networks using in a much simpler way. Currently, we have demonstrated that bacteria and DNA molecules can “copy” logic gate, and started constructing more complicated logic gates.
</P><P>We wish this research would result in a new generation biological processor, and they have the same importance in information processing as other electric devices.  Although there is still a long way before constructing a biological computer, we have the faith that our O-Key logic gate will be applied as the fundamental module in the computer.
</P><H1><SPAN class="mw-headline" id="References">References</SPAN></H1><P>1.  Computational design of orthogonal ribosomes Lon M. Chubiz and Christopher V. Rao
</P><P>2. Toward Engineering SyntheticMicrobialMetabolism George H.McArthur IV and Stephen S. Fong
</P><P>3. Automated design of synthetic ribosome binding sites to control protein expression Howard M Salis, Ethan A Mirsky &amp; Christopher A Voigt
</P><P>4. Synthesis of orthogonal transcriptiontranslation networks Wenlin An and Jason W. Chinl
</P><P>5. One-step assembly in yeast of 25 overlapping DNA fragments to form a complete synthetic Mycoplasma genitalium genome.full
</P><P>6. Daniel G. Gibsona,1, Gwynedd A. Bendersb, Kevin C. Axelroda, Jayshree Zaveri a, Mikkel A. Algirea, Monzia Moodiea,
Michael G. Montaguea, J. Craig Ventera, Hamilton O. Smithb, and Clyde A. Hutchison III
</P><P>7. Synthesis of DNA fragments in yeast by one-step assembly of overlapping oligonucleotides Daniel G. Gibson
</P><P>8. DNA assembler, an in vivo genetic method for rapid construction of biochemical pathways Zengyi Shao1, Hua Zhao1and Huimin Zhao
</P></DIV></DIV></DIV><DIV id="footer"><P>
E-mail：austinamens@gmail.com | Address:Building No.20, No.92 Weijin Road, Tianjin, China | Zip-code：300072
Copyright 2012 © Tianjin University iGEM Team
</P></DIV><DIV class="printfooter">
Retrieved from &quot;<A href="http://2012.igem.org/Team:Tianjin/Modeling/Regulation">http://2012.igem.org/Team:Tianjin/Modeling/Regulation</A>&quot;</DIV></DIV></DIV><DIV id="footer-box" class="noprint"><DIV id="footer"><UL id="f-list"><LI id="t-recentchanges"><A href="/Special:RecentChanges" title="Recent changes">Recent changes</A></LI><LI id="t-whatlinkshere"><A href="/Special:WhatLinksHere/Team:Tianjin/Modeling/Regulation" title="List of all wiki pages that link here [j]" accesskey="j">What links here</A></LI><LI id="t-recentchangeslinked"><A href="/Special:RecentChangesLinked/Team:Tianjin/Modeling/Regulation" title="Recent changes in pages linked from this page [k]" accesskey="k">Related changes</A></LI><LI id="t-specialpages"><A href="/Special:SpecialPages" title="List of all special pages [q]" accesskey="q">Special pages</A></LI><LI><A href="/Special:Preferences">My preferences</A></LI></UL></DIV><DIV id="footer"><UL id="f-list"><LI id="t-print"><A href="/wiki/index.php?title=Team:Tianjin/Modeling/Regulation&amp;printable=yes" title="Printable version of this page [p]" accesskey="p">Printable version</A></LI><LI id="t-permalink"><A href="/wiki/index.php?title=Team:Tianjin/Modeling/Regulation&amp;oldid=235539" title="Permanent link to this revision of the page">Permanent link</A></LI><LI id="privacy"><A href="/2012.igem.org:Privacy_policy" title="2012.igem.org:Privacy policy">Privacy policy</A></LI><LI id="disclaimer"><A href="/2012.igem.org:General_disclaimer" title="2012.igem.org:General disclaimer">Disclaimers</A></LI></UL></DIV></DIV></DIV></BODY></HTML>