"
Page
Discussion
View source
History
teams
Log in
 
Team:Tianjin/Project/Technology
From 2012.igem.org
Project Contents
Project Home
Orthogonal System
Logic Metabolism Regulation
Future Work
BioBrick
In this page
Replace of RBS in Plasmid
Yeast Assembler
MAGE
Technology
Replace of RBS in Plasmid
Figure 1. The process of introducing the O-RBS to plasmid (from TJU iGEM Team 2012)
Here, how did we introduce the O-Lock, we just use two primers to build pcr. Those two primers have about 20bp overhands on their 5' end, which contain the O-Locks or other mutations. Then we directly transform those PCR products to E. coli by electroporation-mediated method. Then those PCR productions could cyclize and form the complete plasmids.
Yeast Assembler
History
Yeast Assembler is based on in vivo homologous recombination in yeast. As for its high efficiency and ease to work with, in vivo homologous recombination in yeast has been widely used for gene cloning, plasmid construction and library creation. In the early of 2008, Zengyi Shao from University of lllinois at Urbana-Champaign, Urbana, used such a method to construct biochemical pathways. Such a method, for its high efficiency in assembling multiple genes, received great popularity since its appearance.
The Difficulty of Large Fragments Assembly
The general digestion connection will leave scar and will be limited by the specific cleavage sequences.
Figure 2. Enzyme digestion (from the website of dnaQ
Long PCR fragment will suffer the decline of success rate and distortion, etc.
Figure 3. PCR Recombinant & PCR Machine (from the website of dnaQ
However, the construction of some large fragments cannot be avoided, so the development of a low-cost, simple operation, good fidelity, a little limiting factor large DNA fragment assembly method is particularly important.
Principles
One step assembly into a vector.
Figure 4. Principles of Yeast assembler (from "DNA assembler, an in vivo genetic method for rapid construction of biochemical pathways")
When parts are transformed all parts into Yeast, homologous recombination occurs at the site “x”, and then all little parts are integrated into a vector.
Advantages and Disadvantages
Compared with other methods，the “Yeast assembler” are more efficient and useful for large gene assemble.
Table 1. Advantages compared to the other two methods (from TJU iGEM Team 2012)
 
Yeast Assembler
SLIC
Domino Method
Advantages
High efficient, Fit for large gene assemble rapid
Fit for many little parts into overall
Fit for two large gene parts integration
Disadvatages
Process is complex
Success rate is low, The size of recombinant is relatively small(<8kb)
Require a great amount of time and labor. Process is sequential
Completely Synthesizing the Genome of Mycoplasma Genitalium using Yeast Assembler
In 2008, Gibson from the J. Craig Venter Institute, published an article “one-step assembly in Yeast of 25 overlapping DNA fragments to form a complete synthetic Mycoplasma genitalium genome”. In the article, the author transformed 25 overlapping DNA fragments into Yeast, homologous recombination occurs, and then the whole genome is synthesized.
Figure 5. Synthetic Mycoplasma genitalium genome (from "One-step assembly in yeast of 25 overlapping DNA fragments to form a complete synthetic Mycoplasma genitalium genome")
Construction of a synthetic M. genitalium genome in yeast. Yeast cells were transformed with 25 different overlapping A-series DNA segments (blue arrows; ~17 kb to35 kb each) composing the M. genitalium genome. To assemble these into a complete genome, a single yeast cell (tan) must take up at least one representative of the 25 different DNA fragments and incorporate them in the nucleus (yellow), where homologous recombination occurs. This assembled genome, called JCVI-1.1, is 590,011 bp, including the vector sequence (red triangle) shown internal to A86 – 89.
MAGE
We thought that now that we could introduce O-Lock into a cell, could we introduce O-Lock to the whole system. Theoretically speaking, we could make it come true because there is a technology called MAGE (Multiplex Automated Genomic Engineering) which comes from Harvard University. We could use this technology to replace all the N-RBS to O-RBS of the genome. Then we could regulate the metabolic network more freely and accurately by many AND gates which come from the O-Locks.
Figure 6. The basic principle of MAGE (from TJU iGEM Team 2012)
E-mail：austinamens@gmail.com | Address:Building No.20, No.92 Weijin Road, Tianjin, China | Zip-code：300072
Copyright 2012 © Tianjin University iGEM Team
Retrieved from " http://2012.igem.org/Team:Tianjin/Project/Technology "
Recent changes
What links here
Related changes
Special pages
My preferences
Printable version
Permanent link
Privacy policy
Disclaimers
