"
Page
Discussion
View source
History
teams
Log in
 
Team:INSA-Lyon/Project/Stage3/Strategy/Designcurli
From 2010.igem.org
Home
About us Students | Supervisors | The INSA | Discover Lyon | Gallery
Droppy project Production | Uses | Regulation | Further direction | Notebook | Modelling | References
Human practice Ethics | Safety
Parts
Sponsors
Acknowledgements
Contact us
Droppy Project
> Production
> Uses
> Regulation
> Our Project
> Strategy
> Results
> Further Direction
> Notebook
> Modelling
> References
< back
Design of Curli and OmpR
We can find below some explanations about the design of the curli promoter and the ompR protein :
Design of Curli
Design of OmpR 234
Design of Curli promoter
One microbiology unit of our university works actively on the characterization and regulation of the curli promoter.  Thanks to their works, we had the complete sequence of the intergenic region between csgD and csgB. The curli promoter is localized in this region, and is regulated by the csgD protein.
We wanted to only synthesize the promoter with the site of regulation by csgD. After analyzing the sequence, we decided to begin our promoter 20bp before the csgD box, and to finish it on the +1bp of transcription. It corresponds to 70 bp. We added a non-CDS (non coding sequence) prefix and suffix to our sequence, to fit with iGEM standards.
The final length of the biobrick is 129 pb synthesized by Mr gene ( Curli part ).
Design of OmpR 234
Escherichia coli bacteria into biofilms contain many adherence structures, as curli structures composed by CsgB et CsgA proteins. Laboratory E. coli K12 strain hasn’t naturally got this phenotype. However, we had a mutant K12 MG1655 E. coli strain on which adherence phenotype had been observed. Indeed, there was a mutation in the genome of the bacteria, particularly in the ompR regulatory gene, at the position 234 (Complex Regulatory Network Controls Initial Adhesion and Biofilm Formation in Escherichia coli via Regulation of the csgD Gene, Claire Prigent-Combaret et al. J Bacteriol. 2001). The corresponding OmpR234 protein strongly activated the csgD promoter, and CsgD protein activated the CsgB et CsgA transcriptions. So, we intended to create an ompR234 coding sequence Biobrick by PCR on this bacteria genome. Chromosomic DNA PHL818 strain had been extracted with DNeasy Blood & Tissue QIAGEN kit.
Primers couple had been designed with iGEM extension, containing iGEM restriction enzymes sites: a forward primer with a perfect RBS (Ribosome Binding Site) and a reverse primer with a double transcription terminator.
Forward primer sequence: 5'-TTGAATTCGCGGCCGCTTCTAGAAGGAGGTATATAATGCAAGAGAACTACAAGATTC-3'
Reverse primer sequence: 5'-GTTTCTTCCTGCAGCGGCCGCTACTAGTATTATTATCATGCTTTAGAGCCGTCCG-3'
A PCR product about 790 pb was expected and the good band had been extracted on agarose gel with NucleoSpin Extract II, Macherey-Nagel kit. The final goal is the construction of the ompR234 BioBrick, cloned into pSB1C3, in order to ship the new part to iGEM HQ.
We aimed to make the following construction: constitutive promoter and ompR234 CDS (included a perfect RBS), cloned into a compatible iGEM plasmid backbone. Finally, we would like to do a double transformation with the OmpR234 regulator and also the curli promoter with β-glucosidase or GFP, as reporter genes. This intermediate step will help us to check the functionality of the both new parts. Indeed, we could measure the glucosidase activity or the GFP fluorescence.
Retrieved from " http://2010.igem.org/Team:INSA-Lyon/Project/Stage3/Strategy/Designcurli "
Recent changes
What links here
Related changes
Special pages
My preferences
Printable version
Permanent link
Privacy policy
Disclaimers
