<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_UiOslo_Norway_Design skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><DIV id="top_title"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:UiOslo Norway/Design</SPAN></H1></DIV><DIV id="HQ_page"><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><DIV class="navbar"><DIV class="dropdown"><BUTTON class="dropbtn">Team
         </BUTTON><DIV class="dropdown-content"><A href="https://2018.igem.org/Team:UiOslo_Norway/Team">Team members</A><A href="https://2018.igem.org/Team:UiOslo_Norway/Collaborations">Collaboration</A></DIV></DIV><DIV class="dropdown"><BUTTON class="dropbtn">Project
          </BUTTON><DIV class="dropdown-content"><A href="https://2018.igem.org/Team:UiOslo_Norway/Description">Description</A><A href="https://2018.igem.org/Team:UiOslo_Norway/Design">Design</A><A href="https://2018.igem.org/Team:UiOslo_Norway/Results">Results</A><A href="https://2018.igem.org/Team:UiOslo_Norway/Demonstrate">Demonstrate</A></DIV></DIV><DIV class="dropdown"><BUTTON class="dropbtn">Lab
          </BUTTON><DIV class="dropdown-content"><A href="https://2018.igem.org/Team:UiOslo_Norway/Experiments">Experiments</A><A href="https://2018.igem.org/Team:UiOslo_Norway/Notebook">Notebook</A><A href="https://2018.igem.org/Team:UiOslo_Norway/InterLab">Interlab</A></DIV></DIV><DIV class="dropdown"><BUTTON class="dropbtn">Parts
          </BUTTON><DIV class="dropdown-content"><A href="https://2018.igem.org/Team:UiOslo_Norway/Parts">Parts Overview</A><A href="https://2018.igem.org/Team:UiOslo_Norway/Basic_Part">Basic Parts</A></DIV></DIV><DIV class="dropdown"><BUTTON class="dropbtn">Human Practices
          </BUTTON><DIV class="dropdown-content"><A href="https://2018.igem.org/Team:UiOslo_Norway/Human_Practices">Human Practices</A><A href="https://2018.igem.org/Team:UiOslo_Norway/Integrated_Human_Practices">Integrated Human Practices</A><A href="https://2018.igem.org/Team:UiOslo_Norway/Public_Engagement">Education &amp; Engagement</A></DIV></DIV><DIV class="dropdown"><BUTTON class="dropbtn">Attributions
          </BUTTON><DIV class="dropdown-content"><A href="https://2018.igem.org/Team:UiOslo_Norway/Attributions">Attributions</A><A href="https://2018.igem.org/Team:UiOslo_Norway/Sponsors">Sponsors</A></DIV></DIV><DIV class="dropdown-right"><A href="https://igem.org/2018_Judging_Form?team=UiOslo_Norway">
        Judging Form
        </A></DIV><DIV class="dropdown-right"><A href="https://2018.igem.org/Team:UiOslo_Norway/Medal">
        Medal Criteria
        </A></DIV><DIV class="dropdown-right"><A href="https://2018.igem.org/Team:UiOslo_Norway/Safety">
        Safety
        </A></DIV></DIV><DIV class="container"><H1>Design</H1><H2>Brainstorming a project idea</H2><P>Deciding on a project was a difficult, albeit fun process.</P><DIV class="row"><DIV class="six columns"><UL><LI>Probiotic yoghurt</LI><LI>Enzymatic cleavage of antiobiotcs in the environment</LI><LI>Combat salmon-louse</LI></UL></DIV><DIV class="six columns"><UL><LI>Biofilm removal</LI><LI>Allergy test for mice and rats</LI><LI><STRONG>Home detection-kit for vaginal yeast infections</STRONG></LI></UL></DIV></DIV><P>After brainstorming, it was time to take restrictions such as time, economics, and feasibility into account. We wanted to create a home detection-kit for vulvovaginal <EM>Candida albicans</EM><EM>(C. albicans)</EM>. However, during the course of our project we decided against a consumer-based product, in favour of one operated by a health professional. The reasoning for this is elaborated on in our <A href="https://2018.igem.org/Team:UiOslo_Norway/Integrated_Human_Practices">Integrated Human Practices</A> page.</P><H2>Inspiration</H2><P>Since 30 % of the women who purchase over-the-counter antimycotics do not have an ongoing <EM>C. albicans</EM> infection, a fast detection kit for this infection could potentially aid in the fight against antimycotic resistance by helping people to treat only when they have an infection [A].</P><P>The first step in designing our project was figuring out how we wanted to detect <EM>C. albicans</EM>. We considered using either CRISPR/Cas9 or TALENS for specific detection of DNA. In the beginning we wanted to try both detection systems in parallel. However, after consulting literature one of our team members brought up that by using CRISPR/Cas9 as our detection system, we would be easily able to exchange the guide RNA to detect any pathogens with our system. We therefore decided to use CRISPR/Cas9 as our DNA detection system. </P><P>During our research into common fungal detection methods our supervisor pointed us to the UiOslo_Norway 2016 team, who played with the idea of linking CRISPR/Cas9 to a visible signal for DNA detection [B]. After further investigation we found that the Peking 2015 team employed a similar system using dCas9 and split luciferase [C]. Taking inspiration from these previous iGEM projects, we decided to use dCas9 coupled to split β-lactamase to detect specific <EM>C. albicans</EM> DNA sequences.  Since we are detecting <EM>C. albicans</EM>, we named our project Canditect.</P><H2>dCas9 with split β-lactamase</H2><P>When designing the dCas9-split-β-lactamase construct we used a dCas9 sequence modifed from iGEM Peking 2015. To this sequence we added one part of the split-β-lactamase (either N-terminal lactamase or C-terminal lactamase).</P><P>Galarneau et al, (2002) [1] demonstrated that splitting the β-lactamase into two fragments consisting of amino acids 26-196 and amino acids 198-290, does not interfere with the biological activity when the two split parts are re-connected. This means that the enzyme becomes a biologically active enzyme as soon as the two parts are in close proximity to each other.</P><P>The beta-lactamase cleaves the C-N bond in the beta-lactam ring of nitrocefin. The hydrolyzed product has a different color than that of nitrocefin, changing from yellow (580 nm) to red (482 nm) [6].</P><P style="text-align:center; margin-top:20px;"><STRONG>Figure 1:</STRONG> Hydrolysis of nitrocefin. Image from iGEM Calgary 2013</P><P>Nitrocefin is a cephalosporin with antibiotic activity, which is another important reason that our kit will be used at a doctors office or pharmacy, where it can be handled and disposed off in a safe manner. </P><P>To connect the dCas9 with the split β-lactamase, we inserted a linker sequence, consisting of three GGGGS repeats. The linker sequence was previously used by iGEM Peking 2015 [B]. We designed four different combinations of the construct: </P><DIV id="stuff"><P>Nlact connected to the C-terminus of dCas9</P><P>Nlact connected to the N-terminus of dCas9</P><P>Clact connected to the C-terminus of dCas9</P><P>Clact connected to the N-terminus of dCas9</P></DIV><P style="text-align:center;"><STRONG>Figure 2:</STRONG> Four different dCas9-constructs with split beta-lactamase.
  Orange: Overhang for vector backbone (Gibson cloning)</P><P>Unfortunately, IDT was not able to synthesize any of these four constructs due to their large size and complexity. Therefore, we decided to split each of the four constructs in two individual parts. Only then was IDT able to synthesize our sequences. </P><H2>Lytic enzymes</H2><P>In order for the dCas9/ split β-lactamase to detect a specific DNA sequence, the <EM>C. albicans</EM> DNA needs to be free in solution. During our literature search we found that the cell wall of <EM>C. albicans</EM> consists approximately of 80-90 % carbohydrate, with the three most prominent groups being β-glucan, chitin, and mannan. β-glucan and chitin are the components that form the rigidly and strength to the cell wall. β-glucans make up 47-60 % of the weight of the cell wall, and thus form the main structural component. In <EM>C. albicans</EM> the β-glucans are linked by either β-1,3 or β-1,6 bonds [2]. In addition, some bacteria may also produce beta-lactamase, making a selective lysis important. We therefore decided to use a β-1,3-glucanase and a mannanase to selectively lyse the yeast cell wall, and not that of bacteria.. Neither the glucanase nor the mannanase we needed were in the iGEM Parts Registry, so we decided to make these as new BioBricks. </P><P>The glucanase was synthesized by Twist Bioscience, and the sequence was codon optimized for <EM>Escherichia coli</EM> using their suggested optimization tool.</P><P>Twist graciously offered to sponsor the synthesis of the glucanase. Unfortunately, the mannanase could not be synthesized, so we continued using only the glucanase. </P><BUTTON class="collapsible">Glucanase sequence after codon optimization</BUTTON><DIV class="content"><H5>Glucanase sequence</H5><P>ATGGATCTCGCTCGCCAACGTTCCCCTCGCCCTACCCCTCGTCGCGGCGCGCGTCCCCGTCCAGGACGCCCACGCCGCCGCCCAGCGCCACGTTTGGGTGATACGGCACGTCGTGTGTTAGCAGGCGCAGTTGCCGCATTAACTGTTTCTGCGGGAGCCCTGGTTGCAGCGCCTGCGGCAACCGCTGCGCCCGGGGATATTATTTGGTCCGATGAATTTGATGGTGCTGCCGGTTCAGCACCTTCGTCGGCACGTTGGAATCATGAAACTGGTGCAGGTGGTTGGGGTAATGCAGAATTACAAAATTATACTACATCCCGTGCTAATAGCGCACTGGATGGCCAAGGTCATCTGGTGATTACTGCGCGCCGTGAAGCGGATGGCAGCTATACCAGCGCCCGTATGACCACGCAAGGTCGCTATCAACCCCAATATGGCCGTGTAGAAGCACGTATTCAAATTCCTCGTGGGCAAGGAATTTGGCCTGCATTTTGGATGCTGGGTGGAAATTTGCCGGGCGTTGGTTGGCCAACATCCGGGGAAATTGATGTGATGGAAAATGTAGGGTTTGAACCTCATCGTGTTCATGGTACTGTTCATGGCCCAGGCTATAGCGGCGGAGCGGGTATTACTGGTATGTATCAACATCCTCAAGGTTGGAGTTTTGCTGATGATTTTCATACTTTTGCGGTAGATTGGAAACCGGGCGGCATTACGTGGTTTGTGGATGGTCAACAATTTCATCGTGTAACCCGTGCTTCTGTGGGTGCGAATGCCTGGGTATTTGATCAACCCTTCTTTCTTATTTTGAATGTTGCAGTTGGTGGCCAATGGCCGGGTTATCCGGATGGAACCACTCAACTTCCGCAACAAATGAAAGTCGATTATGTACGTGTGTATGATAATGGTTCAGGTACTAGCAATCCCCCTAATCCGGGGGGGGGCCTGCCCACGGGCACCGGTACAATTCGTATTGCTAATGGTTATTGTCTGGATGTGCCTTGGGGCCAAACCCATGATGGTAATCGTGTTCAAATTGTTCCCTGTAATGGAAATGCGGCGCAATCATGGACTCGCGGCTCAGATGGTACGATTCGCGCCCTGGGTAAATGTTTGGATGTTTCGGGTTCTGGTACTGCCAATGGTACCGCCGTTCAATTGTGGACCTGTAATGGAACTAATGCCCAGAAATGGACGTATGATGCAGGCGCTAAAACCCTTCGTAATCCTCAAGCTGGCAAATGTCTTGATGCAGAAGGCGGGAATCCACTGTATGAAGGACAACGCTTTGTAATTTGGGATTGTCATGGAAGCACGAATCAACAATGGACTTTCTGA
    </P></DIV><H2>Guide RNA (gRNA)</H2><P>Another important part of our project was deciding which part of the <EM>C. albicans</EM> genome to target. After some research we found that Hyphae Wall Protein 1 (HWP1) was a unique target for <EM>C. albicans</EM> [3]. We performed a BLAST search of the HWP1 sequence against several bacterial and fungal species common in the vaginal flora (Table 1) [4, 5]).
   The dCas9 enzyme would be guided by a 20 nucleotide RNA sequence to its complementary DNA sequence in the <EM>C. albicans</EM> genome. We designed the gRNAs with the CRISPR function in the <A href="https://benchling.com/">Benchling software</A>. When inserting the DNA sequence of a specific protein the tool automatically designs the most effective gRNAs. Out of many choices we selected gRNA pairs consisting of two gRNAs with an interspace between 15 and 30 nucleotides. </P><P style="text-align:center;"><STRONG>Table 1:</STRONG> Common vaginal commensal organisms</P><TABLE class="tableizer-table"><THEAD><TR class="tableizer-firstrow"><TH>BACTERIA</TH><TH>YEAST</TH></TR></THEAD><TBODY><TR><TD>Lactobacillus iners</TD><TD>Cerevisiae krusei</TD></TR><TR><TD>Lactobacillus crispatus</TD><TD>Cerevisiae tropicalis</TD></TR><TR><TD>Gardnerella vaginalis</TD><TD>Cerevisiae glabrata</TD></TR><TR><TD>Atopobium vaginae</TD><TD>Cerevisiae parapsilosis</TD></TR><TR><TD>Bacterial Vaginosis-associated bacteria 1</TD><TD>Saccharomyces cerevisiae </TD></TR><TR><TD>Megasphaera 1</TD><TD> </TD></TR><TR><TD>Prevotella buccalis-like</TD><TD> </TD></TR><TR><TD>Prevotella spp.</TD></TR></TBODY></TABLE><P>Upon not finding any significant similarities between our target sequence (HWP1) and common microorganisms found in the vaginal flora, we decided to design guide RNAs targeting this protein. </P><P>After finding <EM>C. albicans</EM> specific DNA sequences, coding for one of the protein hyphal wall protein 1 (HWP1), which enables the yeast to go to its hyphae form. We made four different sgRNA sequences, that when combined in pairs would result in different spacer lengths ranging from 15 to 30 nucleotides, allowing us to find the optimal spacer length for the best color readout after the split β-lactamases recombine and cleave their substrate, nitrocefin, into a colored product [5]</P><H5>gRNA sequences</H5><OL><LI>CAAGGAACATCAGGTTGAGG</LI><LI>TCACAAGGAACATCAGGTTG</LI><LI>GGATTGTCACAAGGAACATC</LI><LI>GGTTGAGGTGGATTGTCGCA</LI></OL><H2>Canditect</H2><P>The last step of our iGEM journey is to put all the elements together in a kit. Taking inspiration from other similar test kits, we worked together with experienced 3D printer Arnab Sarkar to design a prototype of the Canditect kit. </P><P>After sending in rough sketches of our design, Sarkar converted the sketch into a 3D model and assisted us in printing and assembling the prototype. In the end, we were able to print a prototype test kit that met our standards. </P><H2>References</H2><H5>Web sources:</H5><UL><LI>A.	http://www.antibiotikaiallmennpraksis.no/index.php?action=showtopic&amp;topic=DrmsfZGV&amp;highlight=true</LI><LI>B.	https://2015.igem.org/Team:Peking</LI><LI>C.	https://2016.igem.org/Team:UiOslo_Norway/Description#description</LI></UL><H5>Literature</H5><OL><LI>Galarneau, A., et al., Beta-lactamase protein fragment complementation assays as in vivo and in vitro sensors of protein protein interactions. Nat Biotechnol, 2002. 20(6): p. 619-22</LI><LI>Chaffin, W.L., et al., Cell wall and secreted proteins of Candida albicans: identification, function, and expression. Microbiol Mol Biol Rev, 1998. 62(1): p. 130-80.</LI><LI>Mayer, F.L., D. Wilson, and B. Hube, Candida albicans pathogenicity mechanisms. Virulence, 2013. 4(2): p. 119-28.</LI><LI>Guo, R., et al., Increased diversity of fungal flora in the vagina of patients with recurrent vaginal candidiasis and allergic rhinitis. Microb Ecol, 2012. 64(4): p. 918-27.</LI><LI>Martin, D.H., The microbiota of the vagina and its influence on women's health and disease. Am J Med Sci, 2012. 343(1): p. 2-9.</LI><LI>O'Callaghan, C.H., et al., Novel method for detection of beta-lactamases by using a chromogenic cephalosporin substrate. Antimicrob Agents Chemother, 1972. 1(4): p. 283-8.</LI></OL></DIV></DIV></DIV></DIV></DIV></DIV></BODY></HTML>