<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_NAWI_Graz_Design skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><DIV id="top_title"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:NAWI Graz/Design</SPAN></H1></DIV><DIV id="HQ_page"><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><NAV class="navbar navbar-expand-sm navbar-light bg-light"><DIV class="container-fluid container"><DIV id="navbar" class="navbar-collapse"><UL class="nav navbar-nav"><LI class="nav-item dropdown"><A class="nav-link dropdown-toggle" data-toggle="dropdown" data-target="dropdown_target" href="https://2018.igem.org/Team:NAWI_Graz">
Project </A><DIV class="dropdown-menu" aria-labelledby="dropdown_target"><A class="dropdown-item" href="https://2018.igem.org/Team:NAWI_Graz">Home</A><A class="dropdown-item" href="https://2018.igem.org/Team:NAWI_Graz/Description">Description</A><A class="dropdown-item" href="https://2018.igem.org/Team:NAWI_Graz/Design">Design</A></DIV></LI><LI class="nav-item dropdown"><A class="nav-link dropdown-toggle" data-toggle="dropdown" data-target="dropdown_target" href="#">
Lab </A><DIV class="dropdown-menu" aria-labelledby="dropdown_target"><A class="dropdown-item" href="https://2018.igem.org/Team:NAWI_Graz/Protocols">Protocols</A><A class="dropdown-item" href="https://2018.igem.org/Team:NAWI_Graz/Notebook">Notebook</A><A class="dropdown-item" href="https://2018.igem.org/Team:NAWI_Graz/Improve">Improve</A><A class="dropdown-item" href="https://2018.igem.org/Team:NAWI_Graz/InterLab">InterLab</A></DIV></LI><LI class="nav-item dropdown"><A class="nav-link dropdown-toggle" data-toggle="dropdown" data-target="dropdown_target" href="#">
Human Practices </A><DIV class="dropdown-menu" aria-labelledby="dropdown_target"><A class="dropdown-item" href="https://2018.igem.org/Team:NAWI_Graz/Human_Practices">Human Practices</A><A class="dropdown-item" href="https://2018.igem.org/Team:NAWI_Graz/Public_Engagement">Public Engagement</A></DIV></LI><LI class="nav-item dropdown"><A class="nav-link dropdown-toggle" data-toggle="dropdown" data-target="dropdown_target" href="#">
Team </A><DIV class="dropdown-menu" aria-labelledby="dropdown_target"><A class="dropdown-item" href="https://2018.igem.org/Team:NAWI_Graz/Team">Team Members</A><A class="dropdown-item" href="https://2018.igem.org/Team:NAWI_Graz/Attributions">Attributions</A><A class="dropdown-item" href="https://2018.igem.org/Team:NAWI_Graz/Collaborations">Collaborations</A></DIV></LI></UL></DIV></DIV></NAV><DIV class="igem_2018_team_content"><DIV class="igem_2018_team_column_wrapper"><DIV class="column full_size"><H1>Transition of our project design through companies and industry professionals</H1><P style="font-size:80%;display:block;">
After deciding our projects topic we wanted to submit to the 2018 iGEM-competition, we were discussing several ways to tackle our approach.
</P><P style="font-size:80%;display:block;margin-top:7%;">
We started by talking to professionals at the local universities to get a better grasp on the current advancements in this field and to gather information on possible implementations of our idea. This led us to becoming aware of the very important differentiation of palm oil and palm kernel oil. The second is very important for the foods industry and much harder to substitute with other products.Since our decision to pick biosynthetic palm kernel oil as our project arose mainly from the fact that palm (kernel) oil is a hot topic in austria at the moment, where supermarkets actually started to advertise with palm oil free products in their foods, we decided to pick palm kernel oil as our target product.As a biobrick, we thought it fitting to use a mutation of the <I>tesA</I> gene as described in the paper: “Computational Redesign of Acyl-ACP Thioesterase with Improved Selectivity toward Medium-Chain-Length Fatty Acids” since we wanted to use this mutant for our fatty acid production.
</P><P style="font-size:80%;display:block;">
The first concept of the project was summarized in a basic flowchart (Fig. 1.1.1)
</P><P style="font-size:80%;display:block;margin-top:7%;">
To get going on the project, we immediately started to look for the best method for the esterification process and asked at the institute of biocatalysis for a lab space to conduct our preliminary tests. After a short deliberation period, we realized that chemical esterification of triglycerides is very hard and actually illegal, at least in th EU, if the product is supposed to be used in foods down the line. This development completely removed the first step of the diagram.
</P><P style="font-size:80%;display:block;">
As a result it was planned to use microorganism to do the esterification for us.  We thought about using yeast (<I>Saccharomyces cerevisiae</I>) for this matter because of its lipid droplets and inherent ability to form triacylglycerides. To realize this endeavour we consulted Dr. Klaus Natter who later on became our secondary PI. He provided us with a lab spot and valuable intel for the preliminary tests. With his consultation we decided to test the compatibility of some yeast strains with the main fatty acids palm kernel oil is comprised of, since they are both relatively short chained (C12 and C14) and our SI already knew that fatty acids (FAs) with a length of only C10 are toxic for yeast. The overhauled diagram can be seen in Figure 1.1.2.
</P><P style="font-size:80%;display:block;">
The results of the preliminary tests had meager success on all tested strains which led us to consider the first fork in the flowchart to be false. As next step we started making plans to realize the esterification process in <I>Escherichia coli</I>. First we wanted to implement the esterification process via plasmid, but since we are directly interfering with the cells metabolism we started researching implementation in the genom with CRISPR-Cas9 for more stable expression. At ACIB (Austrian Center of Industrial Biochemistry) we were shown the system they would use for the implementation of a project like ours. It is a two vector system with CRISPR-Cas9 and lambda-red recombineering proposed by Jiang et al. in the paper “Coupling the CRISPR/Cas9 System with Lambda Red Recombineering Enables Simplified Chromosomal Gene Replacement in <I>Escherichia coli</I>”. Naturally, since we had the opportunity to work with people experienced with this system and it promised a more stable implementation we chose it for our project. To get this system to work, we needed to design a polycistronic expression cassette that is compatible and brings all our wanted characteristics into the organism. 
</P><P style="font-size:80%;display:block;">
Since no one in our team had to design nor use a cassette like this in that special system before, the design process underwent several consultations by professionals at ACIB until a satisfactory result was reached. In this process we also realized that not only the <I>tesA</I> gene would be a fitting addition to the biobrick database, but also the <I>atfA</I> gene we used for esterification would make a helpful addition. When we checked for this particular gene we already found it in the database and we decided to submit an improved version with codon-optimization for <I>E. coli</I> and added a strong RBS for improved expression.
</P><P style="font-size:80%;display:block;">
We conducted several interviews with professionals from different industries to find out what role palm oil plays for their product. This brought us to the realization, that the final product we aspired to produce is only essential for the foods industry. All the other companies we interviewed deconstruct the triacylglycerides of the oil into fatty acids to use in their processes. Our conclusion was, that our product is only paying off for the foods industry since esterification is not necessary for most other industrial uses.
</P><P style="font-size:80%;display:block;">
We decided to try and improve yield of the wanted fatty acid for the wax industry (stearic acid) in yeast while we had to wait for reactions in our main project and biobrick implementations. We picked a yeast strain with a knocked out delta-desaturase gene and supplemented the necessary fatty acid for cell growth (oleic acid) to try and produce as much of stearic acid as possible.
</P><P style="font-size:80%;display:block;">
The final flowchart that our project actually followed can be seen in Figure 1.1.3.
</P><H1>
CRISPR-Cas9 and lambda red recombineering system as proposed by Jiang et. al. (Fig. 1.1.4)
</H1><P style="font-size:80%;display:block;"><UL><LI><UL>Choose strain and locus for genomic integration</UL></LI><LI><UL>Design editing fragment
</UL></LI><LI>500 bp homology up- and downstream of the introduced gap</LI><LI>design 20 bp target specific crRNA (ATUM)</LI></UL></P><LI><UL>construct target plasmid
</UL></LI><LI>cut pTargetF and editing fragment with BcuI/SalI</LI><LI>ligate pTargetF backbone with editing fragment  target plasmid</LI><LI>introduce target plasmid into electrocompetent <I>E. coli</I> K12 MG 1655 cells, e.g. Top10F’</LI><LI>plate onto LBspec plates (50 mg/L spectinomycin) and incubate o/n at 37 °C</LI><LI>isolate plasmids with Promega PureYieldTM Plasmid Miniprep System</LI><LI>sequence verify plasmid</LI><LI><UL>genome editing
</UL></LI><LI>prepare electrocompetent <I>E. coli</I> cells for expression, e.g. K12 MG 1655</LI><LI>transform the cells with pCas</LI><LI>prepare electrocompetent <I>E. coli</I> K12 MG 1655{pCas} recombineering cells</LI><LI>transform <I>E. coli</I> K12 MG 1655 {pCas} with target plasmid</LI><LI>plate onto LBkan, spec selective plates and incubate o/n at 28 °C</LI><LI><UL>confirm correct genomic integration
</UL></LI><LI>perform colony PCR of single colonies</LI><LI>primers bind within the editing fragment and within the locus (outside the homology hook)</LI><LI>sequence PCR product</LI><LI><UL>plasmid curing
</UL></LI><LI>prepare an overnight culture (ONC) of sequence verified clone with 0.5 mM IPTG and incubate o/n at 28°C</LI><LI>plate ONC (appropriate dilution) onto LBkan and LBkan,spec plates</LI><LI>perform colony PCR with primers that bind within the target plasmid</LI><LI>streak colonies that did not show a visible band after PCR onto LBkan, spec plates, check for loss of target plasmid (no growth)</LI><LI>prepare ONC of clone that did not grow on LBkan, spec and incubate o/n at 37°C</LI><LI>plate ONC (appropriate dilution) onto LB and LBkan plates</LI><LI>perform colony PCR with primers that bind within the target plasmid</LI><LI>streak colonies that did not show a visible band after PCR onto LBkan plates to check for loss of pCas9 (no growth)</LI><LI>use cured strain for further experiments</LI><H1>OE-PCR for gblock assembly</H1><P style="font-size:80%;display:block;">
To assemble the polycistronic expression cassette, OE-PCRs were planned parallel and in succession. The exact plan on how we wanted to go about the assembly can be seen in the figure 1.1.5 below.
</P><FIGURE><FIGCAPTION><SPAN style="font-size:70%;">Fig. 1.1.5: gblock assembly approach</SPAN></FIGCAPTION></FIGURE><H1>Design of the polycistronic expression cassette</H1><P style="font-size:80%;display:block;margin-top:10%;">
After the unsatisfying result from the pre-tests with yeast it was decided to use <I>E. coli</I> for our purposes. For this matter we consulted professionals at ACIB several times who are very experienced working with all kinds of <I>E. coli</I> strains and methods. As a result it was decided to use the paper: “Production of triacylglycerols in <I>Escherichia coli</I> by deletion of the diacylglycerol kinase gene and heterologous overexpression of <I>atfA</I> from <I>Acinetobacter baylyi</I> ADP1” by Helge Jans Janßen and Alexander Steinbüchel as guideline to make the strain capable of esterification of fatty acids into triglycerides (Fig.1.1.6). 
</P><P style="font-size:80%;display:block;">
After some consultation and consideration the following construct (Fig. 1.1.7) was planned to be created for implementation with a CRISPR-Cas9 and lambda red recombineering two vector system as Proposed by Jiang et. al..
</P><FIGURE><FIGCAPTION><SPAN style="font-size:70%;">Fig. 1.1.7: First suggested design of the TGA synthesis cassette - only including the essential genes and promotor + lacI + operator for regulation</SPAN></FIGCAPTION></FIGURE><P style="font-size:80%;display:block;">
Since neither of our team worked with Snapgene before, we had to familiarize ourselves with it while working on the cassette. The sequences of <I>lacI</I>, <I>lacP</I>, <I>tacP</I>, <I>lacO</I>, <I>fadD</I>, <I>plsB</I> and <I>pgpB</I> were retrieved from ecocyc (https://ecocyc.org/) and the sequence for <I>atfA</I> from NCBI (https://www.ncbi.nlm.nih.gov/). The RBS sequences were chosen with regards to their strength from the iGEM database.
</P><P style="font-size:80%;display:block;">
It was desired that a rearrangement of the genes is possible since the expression diminishes for each successive gene, giving the possibility to adjust it as needed. This led us to implementing restriction sites between all genes as seen in Fig. 1.1.8.
</P><FIGURE><FIGCAPTION><SPAN style="font-size:70%;">Fig. 1.1.8: TGA-Cassette with the added restriction sites for expression adjustment in the assembly stage - the chosen restriction enzymes where subject to repeated change during the cassette design process to achieve buffer compatibility, lower the price and avoid unwanted digest.</SPAN></FIGCAPTION></FIGURE><P style="font-size:80%;display:block;">
The restriction sites where picked with regards to price, availability and used buffer.
</P><P style="font-size:80%;display:block;">
This construct is supposed to replace the <I>dgkA</I> gene since it catalyzes the backreaction of an important intermediate to our final product. To ensure the integration works as intended, 500 base pairs upstream of the <I>dgkA</I> gene needed to be added to the 5’ end of the construct and  500 base pairs downstream of <I>dgkA</I> to the 3’ end (Fig. 1.1.9).
</P><FIGURE><FIGCAPTION><SPAN style="font-size:70%;">Fig.1.1.9: TGA-Cassette with added up- and downstream regions of the dgkA gene for CRISPR-Cas9 implementation - these regions were implemented  ensure that the dgkA surrounding sequences stay unaltered</SPAN></FIGCAPTION></FIGURE><P style="font-size:80%;display:block;">
So the CRISPR-Cas9 system finds the correct gene for replacement, we needed to add a guide-RNA (gRNA) sequence in front of the entire construct. This gRNA is assembled from a tracer-RNA and a crRNA that was calculated by ATUM (https://www.atum.bio/eCommerce/cas9/input) in alignment to the <I>dgkA</I> gene. Also  restriction sites for biobrick implementation at the very ends of the cassette were added (Fig. 1.2.1.)
</P><FIGURE><FIGCAPTION><SPAN style="font-size:70%;">Fig.1.2.1: TGA-cassette with added gRNA sequence for correct CRISPR-Cas9 insertion - the gRNA is an essential sequence that helps the CRISPR-Cas9 System to find the gene it is supposed to excise and subsequently add the cassette into the gap.</SPAN></FIGCAPTION></FIGURE><P style="font-size:80%;display:block;">
Silent mutations were introduced to avoid the appearance of our chosen restriction sites within the genes. After trying to add restriction sites for implementation in the chosen two vector system, the problem arose that the CRISPR-Cas 9 system and Biobrick system share the same restriction site but on opposite ends of the construct (CRISPR-Cas9 plasmid pTargetF: SpeI on 5’ end, Biobrick: SpeI on 3’ end). This forced us to use another restriction site in the pTargetF plasmid of the CRISPR-Cas9 system. As a result the base pairs between the two restriction sites on the plasmid were added to the cassette at the 5’ end (Fig. 1.2.2).
</P><FIGURE><FIGCAPTION><SPAN style="font-size:70%;">Fig.1.2.2: TGA-cassette with added base pairs from ptargetF, to circumvent the compatibility problems of the CRISPR-Cas9 System with the biobrick restriction sites - The restriction sites usually used for the CRISPR-Cas9 two vector system where incompatible with the biobrick system since both used the same Enzyme on opposite ends. This problem was circumvented by adding parts of the pTargetF vector from the CRISPR-Cas9 system to the cassette, enabling a cut at a different restriction site on the vector.</SPAN></FIGCAPTION></FIGURE><P style="font-size:80%;display:block;">
The cassette was then divided into sequences of about 900 bp with overlapping ends, so they could be synthezied through IDT and assembled via OE-PCR. Also the primers needed for colony- and OE-PCR were chosen so they can also be ordered through IDT. In this process, secondary binding sites for one primer were detected and it became apparent that <I>plsB</I> is partially present in the 500 bp upstream region of <I>dgkA</I> (in 3’ direction) and forced the change of the codons of our implemented <I>plsB</I> to avoid conflicts. First, silent mutation was done to all codons appearing in the 500bp up region to avoid the <I>plsB</I> primers binding there, but ultimately it was decided to codon optimize the whole plsB gene for e.coli via the IDT services (https://eu.idtdna.com/CodonOpt).
</P><P style="font-size:80%;display:block;">
The final version of the cassette can be seen in Fig. 1.2.3.
</P><FIGURE><FIGCAPTION><SPAN style="font-size:70%;">Fig.1.2.3: Finished TGA-cassette design with added gblock parts for synthesis via IDT and simplified  assembly - the gblocks were designed to be all roughly the same length with overlapping sequences  to simplify the OE-PCR process.</SPAN></FIGCAPTION></FIGURE></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></BODY></HTML>