<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_NAWI_Graz_Improve skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><DIV id="top_title"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:NAWI Graz/Improve</SPAN></H1></DIV><DIV id="HQ_page"><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><NAV class="navbar navbar-expand-sm navbar-light bg-light"><DIV class="container-fluid container"><DIV id="navbar" class="navbar-collapse"><UL class="nav navbar-nav"><LI class="nav-item dropdown"><A class="nav-link dropdown-toggle" data-toggle="dropdown" data-target="dropdown_target" href="https://2018.igem.org/Team:NAWI_Graz">
Project </A><DIV class="dropdown-menu" aria-labelledby="dropdown_target"><A class="dropdown-item" href="https://2018.igem.org/Team:NAWI_Graz">Home</A><A class="dropdown-item" href="https://2018.igem.org/Team:NAWI_Graz/Description">Description</A><A class="dropdown-item" href="https://2018.igem.org/Team:NAWI_Graz/Design">Design</A></DIV></LI><LI class="nav-item dropdown"><A class="nav-link dropdown-toggle" data-toggle="dropdown" data-target="dropdown_target" href="#">
Lab </A><DIV class="dropdown-menu" aria-labelledby="dropdown_target"><A class="dropdown-item" href="https://2018.igem.org/Team:NAWI_Graz/Protocols">Protocols</A><A class="dropdown-item" href="https://2018.igem.org/Team:NAWI_Graz/Notebook">Notebook</A><A class="dropdown-item" href="https://2018.igem.org/Team:NAWI_Graz/Improve">Improve</A><A class="dropdown-item" href="https://2018.igem.org/Team:NAWI_Graz/InterLab">InterLab</A></DIV></LI><LI class="nav-item dropdown"><A class="nav-link dropdown-toggle" data-toggle="dropdown" data-target="dropdown_target" href="#">
Human Practices </A><DIV class="dropdown-menu" aria-labelledby="dropdown_target"><A class="dropdown-item" href="https://2018.igem.org/Team:NAWI_Graz/Human_Practices">Human Practices</A><A class="dropdown-item" href="https://2018.igem.org/Team:NAWI_Graz/Public_Engagement">Public Engagement</A></DIV></LI><LI class="nav-item dropdown"><A class="nav-link dropdown-toggle" data-toggle="dropdown" data-target="dropdown_target" href="#">
Team </A><DIV class="dropdown-menu" aria-labelledby="dropdown_target"><A class="dropdown-item" href="https://2018.igem.org/Team:NAWI_Graz/Team">Team Members</A><A class="dropdown-item" href="https://2018.igem.org/Team:NAWI_Graz/Attributions">Attributions</A><A class="dropdown-item" href="https://2018.igem.org/Team:NAWI_Graz/Collaborations">Collaborations</A></DIV></LI></UL></DIV></DIV></NAV><DIV class="igem_2018_team_content"><DIV class="igem_2018_team_column_wrapper"><DIV class="column full_size"><H2>BioBricks</H2><H6>Why these parts?</H6><FIGURE><FIGCAPTION><SPAN style="font-size:70%;">Fig.1 Reaction catalyste by tesA (picture from <A href="http://parts.igem.org/Part:BBa_K654058">iGEM11_Utah_State</A>)</SPAN></FIGCAPTION></FIGURE><P style="font-size:80%;display:block;">
Since some of the genes used in the project had unique and useful features, they were chosen for submission to the iGEM biobrick parts database. They have the potential to improve and speed up projects of future iGEM teams that pick <I>E. coli</I> as their organism of choice.
</P><H6>BBa_K2850000:Acyl-ACP Thioesterase (<I>TesA</I>)</H6><P style="font-size:80%;display:block;">
For this matter it was decided to submit a mutated variant of the <I>tesA</I> gene of 3 codons (S122K, Y145K, and L146K) that should influence the fatty acid composition produced by the <I>E. coli</I> by shifting it from a C16/C18 heavy composition to a C12/14 favoured scenario. The mutations where proposed and tested in the paper: “Computational Redesign of Acyl-ACP Thioesterase with Improved Selectivity toward Medium-Chain-Length Fatty Acids” by Matthew J. Grisewood et. al.. The effects of this mutations fit perfectly for the project since the fatty acids produced were exactly the ones needed for the palm kernel oil alternative the work was aimed at. Since those fatty acids are harder to attain in high quantities than longer chain fatty acids this gene is a good addition to the iGEM biobrick database.
</P><H6>BBa_K2850001: Acyltransferase (<I>atfA</I>)</H6><P style="font-size:80%;display:block;">
The second Biobrick that was picked for submission was the <I>atfA</I> gene from <I>Acinetobacter bayliy</I> ADP1 since it gives <I>E. coli</I> the capabilities to assemble the produced fatty acids into triacylglycerides. We found a Biobrick part on the Database that already included this gene but only with added MoClo sites which were not needed for our purposes. It was decided to submit an improved version of this gene better fit for direct implementation into <I>E. coli</I>. To achieve this goal, the gene was codon optimized for <I>E. coli</I> and a strong RBS from the database (BBa_B0030) was added for a good expression rate. This biobrick is more specialized than the previous one since triacylglycerides are usually produced in other organisms with lipid droplets for storage, but short chain fatty acids like the C12 fatty acid lauric acid are unfavourable for the usually used organism yeast and therefore <I>E. coli</I> can offer a useful platform for those purposes. 
</P><H6>Improvement</H6><P style="font-size:80%;display:block;">
Our biobrick BBa_K2850001 (ester synthetase/acyl-CoA diacylglycerol acyltransferase) is an improvement of the BBa_K1705001 biobrick created by the MIT team in 2015. Since we did not know how to use the MoClo Fusion sites from this part, we decided to go another way to start our improvement. Therefore we looked up the protein sequence of <I>atfA</I> from <I>Acinetobacter bayliy</I> ADP1 in the NCBI database. The sequence was reverse translated and codon optimized for the use in <I>E. coli</I> and then synthesized by IDT. Since the biobrick of the MIT is not optimized for the use in <I>E. coli</I> our biobrick should have a much better translation rate and therefore a better expression. In addition our <I>atfA</I> was cleaned of restriction sites to make it compatible for any RFC biobrick system that is mentioned in iGEM registry. 
</P><H6>How did we construct the biobricks?</H6><P style="font-size:80%;display:block;">
As mention before the gblocks were prepared for further usage as proposed by IDTs website. As first step the gblocks and the transport vector were double restriction digested with XbaI and SpeI for later ligation into the completed Biobrick for submission. The added supplements were prepared as follows:
Final Volume: 50µL
</P><P style="font-size:80%;display:block;">
The restriction mix was kept at 37°C over night. The enzymes were then deactivated by placing the mix on a heat block at 80°C for 20min. For better time management chloramphenicol LB-agar-plates were prepared while the restriction was going.To prevent the transport vector from ligating with itself, it was treated with 2μL of alkaline phosphatase overnight and again inactivated with a heat block at 80°C for 20min.
</P><P style="font-size:80%;display:block;">
Implementation of the insert into the vector was done with a fast-link DNA ligation kit from epicenter. For this purpose, the instructions of the kit manufacturer were strictly followed. Of the resulting 15μL, 5μ were used for transformation into electrocompetent <I>E. coli</I> cells via electroporation (2500 V; 4.2ms). After 1h of regeneration time the electroporation mixes were diluted by a factor 1:10 and 1:100 and streaked out on previously prepared LB-agar-plates with chloramphenicol since the transport vector brings a chloramphenicol resistance marker with it. Multiple colonies of those plates were then grown in ONCs and the plasmid was isolated with a miniprep from promega. Then a PCR to amplify the insert with the following specs was prepared:
</P><P style="font-size:80%;display:block;">
To check the results of this PCR, an agarose gel electrophoresis was ran (Fig. 2.2). The findings showed that one of our biobricks (<I>tesA</I> mutated) was completed successfully. The second Biobrick however (<I>atfA</I> optimized) did not assemble correctly since it was not possible to amplify the insert with the PCR. To get this second biobrick, all steps since the ligation were repeated with leftover ligation mix yielding the wanted results.
</P><P style="font-size:80%;display:block;">
It is to note that simultaneously all biobrick assembly steps were ran again, to save time in case something went wrong with the assembly although ultimately not needed.
</P><P style="font-size:80%;display:block;">
The correct plasmids (about 250ng) were then filled into their respective wells on the submission plate and dried with a SpeedVac concentrator without its rotor.
</P><H6>Characterization</H6><P style="font-size:80%;display:block;">
For characterization of the biobricks some of the positive <I>tesA</I> and <I>atfA</I> clones were used to inoculate ONCs and later on for main cultures. After determination of the OD<SUB>600</SUB> the culture was split into equal parts and used for lipid extraction. For the extraction the method invented by Folch was used.
</P><P style="font-size:80%;display:block;">
For identification the formation of Triacylglycerides, the measurement of the fatty acid composition and for purification a thin layer chromatography (TLC) was performed.
</P><P style="font-size:80%;display:block;">
The fatty acid spots were scratched of the TLC plate and converted into fatty acid methyl ester. This methyl ester were analyzed by gas chromatography to identify the fatty acid composition.
</P><FIGURE><FIGCAPTION><SPAN style="font-size:70%;">Fig.2.3: GC-FID of the E.Coli strain with the mutated tesA in comparison to the wild type - the curves have been added into the same diagram with a slight shift on the x-axis for the tesA containing sample for comparison purposes</SPAN></FIGCAPTION></FIGURE><P style="font-size:80%;display:block;">
The addition of the mutated tesA gene via plasmid resulted in an upregulation of the tesA functions and therefore more fatty acids were produced in general. It is also a peak visible in the area were C14:0 is to be expected, giving us strong indication that the mutated tesA gene is fulfilling its purpose. For further evidence, the wild type tesA needs to be knocked out in a strain with the mutated variant present.
</P></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></BODY></HTML>