<HTML xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en" dir="ltr">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki  ltr ns-0 ns-subject page-Team_WLC-Milwaukee_Overview"><DIV id="globalWrapper"><DIV id="top-section"><DIV id="p-logo"><A href="/Main_Page" title="Main Page">&quot;
	    </A></DIV><DIV id="menubar" class="left-menu noprint"><UL><LI class="selected"><A href="/Team:WLC-Milwaukee/Overview">Page               </A></LI><LI class="new"><A href="/wiki/index.php?title=Talk:Team:WLC-Milwaukee/Overview&amp;action=edit&amp;redlink=1">Discussion               </A></LI><LI><A href="/wiki/index.php?title=Team:WLC-Milwaukee/Overview&amp;action=edit">View source               </A></LI><LI><A href="/wiki/index.php?title=Team:WLC-Milwaukee/Overview&amp;action=history">History               </A></LI><LI style="color:white;cursor:default">teams</LI></UL></DIV><DIV class="right-menu noprint" id="menubar"><UL><LI id="pt-login"><A href="/wiki/index.php?title=Special:UserLogin&amp;returnto=Team:WLC-Milwaukee/Overview" title="You are encouraged to log in; however, it is not mandatory [o]" accesskey="o">Log in</A></LI></UL></DIV><DIV id="search-controls" class="noprint"><FORM action="/Special:Search" id="searchform"> </FORM></DIV></DIV><DIV id="content"><H1 class="firstHeading">Team:WLC-Milwaukee/Overview</H1><DIV id="bodyContent"><H3 id="siteSub" class="noprint">From 2013.igem.org</H3><DIV id="navbar" width="965"><SPAN class="top"><UL><LI onmouseover="barcontract()"><A href="https://2013.igem.org/Team:WLC-Milwaukee">Home</A></LI><LI onmouseover=" barexpandProj()"><A href="javascript:barexpandProj()">Project</A><TABLE id="ProjBox" style="display:none"><TBODY><TR><TD><A href="https://2013.igem.org/Team:WLC-Milwaukee/Overview">Overview</A></TD></TR><TR><TD style:=""><A href="https://2013.igem.org/Team:WLC-Milwaukee/Modeling">Modeling</A></TD></TR><TR><TD><A href="https://2013.igem.org/Team:WLC-Milwaukee/Parts">Parts Submitted</A></TD></TR><TR><TD><A href="https://2013.igem.org/Team:WLC-Milwaukee/Future">Future Direction</A></TD></TR><TR><TD><A href="https://2013.igem.org/Team:WLC-Milwaukee/Notebook">Notebook</A></TD></TR></TBODY></TABLE></LI><LI onmouseover="pplexp()"><A href="#">People</A><TABLE width="100%" id="ppl" style="display:none"><TBODY><TR width="100%"><TD width="100%"><A href="https://2013.igem.org/Team:WLC-Milwaukee/Team">Team</A></TD></TR><TR><TD><A href="https://2013.igem.org/Team:WLC-Milwaukee/Mentors">Mentors</A></TD></TR><TR><TD><A href="https://2013.igem.org/Team:WLC-Milwaukee/Sponsors"> Our Sponsors</A></TD></TR></TBODY></TABLE></LI><LI onmouseover="barcontract()"><A href="https://2013.igem.org/Team:WLC-Milwaukee/Safety">Safety</A></LI><LI onmouseover="barexpandHum()"><A href="#">Human Outreach</A><TABLE id="humanBox" style="display:none" width="100%"><TBODY><TR><TD><A href="https://2013.igem.org/Team:WLC-Milwaukee/Camp">Summer Camp</A></TD></TR><TR><TD><A href="https://2013.igem.org/Team:WLC-Milwaukee/Survey">Science Survey</A></TD></TR></TBODY></TABLE></LI><LI onmouseover="barexpandHum2()"><A href="#">Human Pratices</A><TABLE id="humBx" style="display:none" width="100%"><TBODY><TR><TD><A href="https://2013.igem.org/Team:WLC-Milwaukee/Food">Food Economics</A></TD></TR><TR><TD><A href="https://2013.igem.org/Team:WLC-Milwaukee/MisInfo">Misinformation</A></TD></TR><TR><TD><A href="https://2013.igem.org/Team:WLC-Milwaukee/GMOPolitics">GMO Politics</A></TD></TR><TR><TD><A href="https://2013.igem.org/Team:WLC-Milwaukee/Culture">Culture Blowback</A></TD></TR><TR><TD><A href="https://2013.igem.org/Team:WLC-Milwaukee/Experts">The Expert</A></TD></TR></TBODY></TABLE></LI></UL></SPAN></DIV><H1>Our Project</H1><H2>Dr. Farley on Indigenous Peoples</H2><P>How will our project affect the sustainability and cultures of indigenous peoples?</P><P>Dr. Ned Farley: Sustainability is kind of the new topic, and when I say new it’s about ten years old but, it’s new for academics. It’s a new topic of discussion because anthropologists always look at the past more than the future, but what we have realized is that we cannot ignore one for the other. The past, the present, the future, they are all intertwined into culture, and how people use their culture, how they carry it, how they experience it. And so that, kind of in a fast nutshell there, really was a cause in sparking my interest in your proposal and the potential outcomes for that because, and I’ve even brought this up in my anthropology class, the idea of being able to increase rates of sustainability with little to no overhead or capital investment is exactly what humans have looked for since they started writing things down on paper. When we take a look at, for instance, domestication, the domestication of maize [eventually corn] allows people to live in urban environments in the southwest. It is a means of survival; it allows them to grow into large numbers and to flourish….. And so when you came to me yesterday, I think your concern was really great because even though it is not a direct laboratory concern, and that is really where your energies have been spent. You will transform the culture no matter what you do, no matter how hard you try not to, you are always going to affect it. What is awesome about your study is that they could create enough surplus where they could feed themselves. So, now it becomes indigenous again where I live off my herd, and sell it for cash. Right? And that’s powerful. If you can do that without disturbing the balance between biology and culture. You win, hands down. </P><H2>The Final Product</H2><P>This combinatory part is a self contained secretion system that combines a secretion and purification tag, a tripartite pump that is specific to the sec tag, a antinorovirus particle like antibody, Tse2 toxin mediated horizontal gene transfer mechanism. </P><H3>Tripartite SecI pump</H3>
    These are the three essential parts prtD, prtE, prtF, that make up the Type I secretion system from Erwinia chrysanthemi.This part is made of 3 genes: prtD, prtE, and prtF, that constitute a type I Erwinia chrysanthemi secretion system. The Pump is expressed from a strong constitutive promoter, BBa_k206000, and has the translational terminator BBa_B0014. In pSB1C3. Used to secrete proteins containing prtB C-terminal tag. The prtB C-terminal tag is built into the protein generator [BBa_K1175012]. Any protein of interest can be inserted into the protein generator and then secreted when used in conjunction with this secretion system.

<H3>Secretion and purification tag</H3>
    This part BBa_K1175012 A composite part formed between BBa_K206000 strong pBAD promoter, BBa_B0034 strong RBS, and BBa_K215001 a purification and secretion tag; a secretion system designed by Washington iGEM team 2009 BBa_K215001. This tag can be used to secrete proteins of interest by inserting them into a specific NheI cut site flanked by C terminus His tag and an N terminus Secretion tag. Both tags can be removed by a TEV protease. The secretion tag is also specific to the tripartite pump. 

<H3>Antinorovirus like particle antibody</H3>
    This construct BBa_K875004 is designed for the expression of an already described engineered antinorovirus (NoV) monoclonal antibody (mAb 54.6) in fusion with LPP-OmpA. The antibody is expressed in a single chain fragment variable (scFv) format containing light (VL) and heavy (VH) variable domains separeted by a flexible peptide linker. It has already been reported that the scFv 54.6 binds a native recombinant NoV particles (VLPs) and inhibits VLP interaction with cells. LPP-OmpA functions as a leader sequence and an anchor to display the scFv ot the bacterial surface.The construct consistes of T5 Lac Operator (Bba_K875002), ribosomal binding site, LPP-OmpA-scFv 54.6 antinorovirus, Histidine tag (6HIS), Terminator (B0015).
<H3>T5cumate repressed Tse2 mediated, antihorizontal transfer mechanism</H3>
    This construct BBa_K1175003 is a modification of the part designed by iGEM2012 Team Trieste it has a T5cumate inducible promoter suppressed by the CymR gene integrated genome into the chassis E.coli Nissle 1917. This part will not be suppressed by CymR in the bacterial cells that have taken up the plasmid.
<H2>Project Summary</H2><DIV class="WordSection1"><P class="MsoNormal" style="text-indent:.5in">Wisconsin Lutheran College’s 2013 iGEM team
has been hard at work to create a secretion mechanism that is safe and
customizable. Our way of approaching this is to make use of removable
purification and secretion tags that are separated by a single enzyme cut site
Nhel. This is the cut site that is the insertion point for any coding sequence
of interest. Then the induced gene can be secreted from the cytoplasm to the
extracellular space via a tripartite secI pump. In order to control a Tse2
toxin, a cymR-regulated T5cumate promoter is placed within the plasmid. This
will prevent horizontal gene transfer. Essentially, if the plasmid is ejected
from the bacteria and picked up by another, the gene producing the toxin will
not be suppressed and will kill the cell. The plasmid also encodes a chimeric
ompA-monoclonal antibody gene, which when expressed, inserts into the outer
membrane. The antibody then selectively binds norovirus within the gut to
hinder infection. This has great implications in the way of helping those with
viruses in developing nations. This plasmid construct was used to over-express
and secrete plant-based, polymer-degrading enzymes bglS, yesZ, and xynA. This
will allow us to break down  hemicellulose and pectin. These enzymes were
isolated from the cellulosome of Bacillus subtilis subtilis 168 to digest plant
material. We utilized Escherichia coli Nissle 1917 containing a
chromosome-integrated cymR gene to secrete these enzymes in the hope to
accentuate dietary uptake for humans and livestock in impoverished countries.</P><P class="MsoNormal"> </P><P class="MsoNormal">Implications:</P><P class="MsoNormal" style="text-indent:.5in">One of the great potential benefits of this
project is provide impoverished people with more access to food. This project
would allow a greater array of options for food sources, as well as increase
the nutritional value of the food that is already present. The mechanism that
this project provides will be extremely helpful to other iGEM teams. This
system can be used to secrete any other enzyme or molecule chosen by an iGEM
team or even general researchers.</P><P class="MsoNormal"> </P><P class="MsoNormal">Reception:</P><P class="MsoNormal" style="text-indent:.5in">The intent of this project is to create a
product that would aid developing countries, without necessarily changing their
cultures, or “westernizing” them. In these countries, nutrition is a major
concern. The lack of nutrition in developing countries has the added adverse
consequence of increasing chronic diseases. “Nutrition is something easily
changeable that has to power to prevent chronic disease, with scientific
evidence increasingly supporting the view that alterations in diet have strong
effects, both positive and negative, on health throughout life.” Nutrition is
the first step to overall health, and therefor, our product could improve the
general health of a developing country. Some sources even venture to say that
food production is considered an evolutionary advance. Culture is not static,
but dynamic. It is the aim of this project to enhance the cultures that it
touches.Countries also have varying concerns about GMOs, especially the UK and
India. Countries like the United States, Canada, and China would be more likely
to accept these products. This product would go through further testing to
ensure that it is safe for consumption in order to send to countries that could
highly benefit from this product. </P><P class="MsoNormal"> </P><P class="MsoNormal">Future:</P><P class="MsoNormal">Polysaccharides
are a convenient energy storage system for many organisms. In order to use the
energy, the organism must break down the polysaccharide chain into smaller
mono- and disaccharides. Cellulose is one such polysaccharide, an energy
storage form for plants. Glucose monomers link together in long rigid chains to
form cellulose. The plasmid breaks down plant products that are not cellulose,
but similar in structure (hemicellulose and pectin). We have three enzymes to
accomplish this: xynA, bglS, and yesZ. The endo-xylanase xynA cleaves xylan
polysaccharide chains to form shorter xylan chains. The enzyme yesZ has
beta-galactosidase (beta-galacturonidase) activity, cleaving exo-beta-D-galactans
to produce single galactose molecules from RG I Pectin. The endo-beta-glucanase
(bglS) will cleave internal 1,4 linkages adjacent to 1,3 linkages. A secretion
system has been designed to release the enzymes from the bacteria. In future
experiments, other enzyme-producing genes will be added to break down plant material
further. Cellulase genes are first in line, as cellulose is the largest
composition of cell wall. The 2008 Edinburgh team designed the genes required
to break down cellulose into glucose. These are an exocellulase, an
endocellulase, and a beta-glucosidase. Other enzymes required to fully degrade
plant material would be added as well.</P><P class="MsoNormal">Overall,
this project and the secretion system that it provides will be advantageous and
beneficial to future iGEM teams as well as developing countries. </P></DIV><DIV class="printfooter">
Retrieved from &quot;<A href="http://2013.igem.org/Team:WLC-Milwaukee/Overview">http://2013.igem.org/Team:WLC-Milwaukee/Overview</A>&quot;</DIV></DIV></DIV><DIV id="footer-box" class="noprint"><DIV id="footer"><UL id="f-list"><LI id="t-recentchanges"><A href="/Special:RecentChanges" title="Recent changes">Recent changes</A></LI><LI id="t-whatlinkshere"><A href="/Special:WhatLinksHere/Team:WLC-Milwaukee/Overview" title="List of all wiki pages that link here [j]" accesskey="j">What links here</A></LI><LI id="t-recentchangeslinked"><A href="/Special:RecentChangesLinked/Team:WLC-Milwaukee/Overview" title="Recent changes in pages linked from this page [k]" accesskey="k">Related changes</A></LI><LI id="t-specialpages"><A href="/Special:SpecialPages" title="List of all special pages [q]" accesskey="q">Special pages</A></LI><LI><A href="/Special:Preferences">My preferences</A></LI></UL></DIV><DIV id="footer"><UL id="f-list"><LI id="t-print"><A href="/wiki/index.php?title=Team:WLC-Milwaukee/Overview&amp;printable=yes" title="Printable version of this page [p]" accesskey="p">Printable version</A></LI><LI id="t-permalink"><A href="/wiki/index.php?title=Team:WLC-Milwaukee/Overview&amp;oldid=230751" title="Permanent link to this revision of the page">Permanent link</A></LI><LI id="privacy"><A href="/2013.igem.org:Privacy_policy" title="2013.igem.org:Privacy policy">Privacy policy</A></LI><LI id="disclaimer"><A href="/2013.igem.org:General_disclaimer" title="2013.igem.org:General disclaimer">Disclaimers</A></LI></UL></DIV></DIV></DIV></BODY></HTML>