Team:Kyoto/Discussion
Project
Description
Results
Demonstrate
Discussion
Basic Parts
Composite Parts
Practices
Human Practices
Integrated Practices
Public Engagement
Collaborations
Events
Further Consideration
Experiment
Notebook
Material&Methods
Team
Members
Attributions
Criteria
Achievements
HP/Silver
HP/Gold
Discussion
Discussion
1) The summary
2) The method of RNAi by feeding
3) Where is dsRNA in S. cerevisiae?  Do B. xylophilus suck up dsRNA?
4) Is feeding RNAi effective?
5) Targeting essential nematode genes
6) Future
1) The summary
Pine-wilt disease, which is spreading all over the world, is one of the major plant diseases causing severe economic damage. A significant amount of money and labor are required for the treatment of forests each year. We focused on the cause of the disease, a nematode called B. xylophilus, and started our “B. x. Busters” project designed to create a new genetically modified microorganism exterminating B. xylophilus. We searched for a carrier of RNAi by feeding and revealed that under laboratory conditions, B. xylophilus preys on S. cerevisiae by sucking up the yeast’s insides using a straw-like stylet. We developed a reporter distinguishing B. xylophilus which ate S. cerevisiae by recording green fluorescence in the digestive track of B. xylophilus in live microscopy. In addition, we constructed a plasmid for S. cerevisiae expressing dsRNA, characterized its expression, and observed B. xylophilus which preyed on S. cerevisiae expressing dsRNA. These results, combined with our integrated Human Practices, could bring about promising new methodology in the fight against pine-wilt disease.
2) The method of RNAi by feeding
First, by feeding RNAi we aimed to knockdown the AK1 gene of B. xylophilus, since a previous study claimed success in targeting this gene with soaking RNAi [1]. However, we couldn’t reproduce any fatal effect to B. xylophilus just by soaking in RNA solution. Instead, we chose to feed B. xylophilus with S. cerevisiae expressing dsRNA by the conditional Gal1 promoter (BBa_K517001). We spread S. cerevisiae cultured in galactose medium onto no-nutrient agar medium containing no carbon source and fed it to B. xylophilus. This low nutrient condition prevents miscellaneous germs from proliferating, but at the same time the S. cerevisiae also lack nutrients. It probably led to autophagy. Under these conditions, it’s possible that low dosages of dsRNA were given to B. xylophilus than we had initially anticipated. We recognize that we have plenty of room for improvement, optimizing the agar culture condition and preparing S. cerevisiae which maintaining dsRNA expression more stably for a longer period of time.
3) Where is dsRNA in S. cerevisiae?  Do B. xylophilus suck up dsRNA?
Our experiments confirmed SKI2Δ S. cerevisiae contains more dsRNA than wild type yeast. SKI complex is found in the cytoplasm [2],[3].In wild type yeast, SKI complex degrades dsRNA. Therefore, our results suggest that the dsRNA is partially in the cytoplasm. However, the results of Xenopus oocyte microinjection suggested that almost all the dsRNA is present in the nucleus. From our live imaging, it was not possible to see the yeast nucleus passed through the small diameter of the nematode’s stylet. If B. xylophilus doesn’t eat the yeast nucleus, the dsRNA will not reach the nematode’s body. Our further research confirmed that we could use the HIV Rev-RRE nuclear export pathway to help export dsRNA into the cytoplasm. Therefore, we predict that S. cerevisiae which has this function would improve dsRNA nuclear export to the cytoplasm, increasing the effective dose to predator B. xylophilus.
The eGFP feeding reporter we established provided us with ideas for optimizing RNAi by feeding. For instance, we could express a histone-eGFP fusion protein in S. cerevisiae and verify if the nucleus of S. cerevisiae is in fact eaten by B. xylophilus. Extracting DNA or RNA derived from S. cerevisiae will reveal whether B. xylophilus eats the yeast nucleus by identifying the nucleic acid of S. cerevisiae in the intestinal track.
4) Is feeding RNAi effective?
In C. elegans, since a dsRNA-specific bidirectional channel (SID-1) is expressed in the cell membrane [4], it is known that feeding RNAi occurs efficiently. Interestingly, human and Drosophila preserve proteins with similar functions [5]. As we were unable to reproduce published soaking RNAi results with B. xylophilus, we examined whether such channel proteins even existed in B. xylophilus using BLAST search.
When examining C. elegans SID-1 (GenBank: AF 478687.1) as a query, it seems that there is no highly homologous sequence in the B. xylophilus reference genome. As far as we consider it, the results we obtained seem to be reasonable, but we cannot exclude the possibility that the B. xylophilus draft genome is incomplete.
We also researched Meloidogyne incognita which is nematode with a stylet like B. xylophilus. Similar to B. xylophilus, we could not find SID-1 homologues in M. incognita. Interestingly, there is a report that remarkable effect was obtained in M. incognita with feeding RNAi using plants [6],[7]. We hypothesize that M. incognita has another RNAi enabling protein and RNAi reaction is occurring differently from C. elegans.
Finally, we considered that the mechanism of RNAi is relevant for selecting target genes. For example, if dsRNA cannot be transported from the intestinal tract to other cells of the body, it is necessary to select a gene expressed in the intestine as a target. RNAi targeting a gene such as a digestive enzyme or channel protein could result in starvation of the nematodes, reducing their fitness. As an alternative, it may be possible to engineer yeast that express a transporter to assist with dsRNA delivery to other tissues.
5) Targeting essential nematode genes
We tried to confirm whether RNAi by feeding has a fatal effect on B. xylophilus survival. However, we had no method to know the difference between life and death except for judging from the movement of B. xylophilus and, to make matters worse, dead B. xylophilus quickly dried-up and became difficult to detect. Therefore, it may be required to select a target gene (such as dpy [8]) which expressed an obvious change in phenotype without killing, to establish the effect of RNAi by feeding in the future.
6) Future
In the beginning, we intended to develop our “B. x. Busters” yeast as a biological pesticide, but we may need to solve many problems in advance, such as biosafety. As we described in our Human Practice ( https://2017.igem.org/Team:Kyoto/Discussion_HP ), a promising strategy, attempting to create genetically-engineered pine trees expressing RNAi, is proposed. Our yeast system should be ideal for screening target genes and effective RNAi. RNAi by feeding with S. cerevisiae with optimization should reduce the time required for growing plants and would help us to realize our plan earlier. Accordingly, we plan to further improve our “B. x. Busters” and establish more effective RNAi by feeding. In addition to SKI2, we should verify the effect that other RNA metabolism-related factors may have on dsRNA accumulation. We can also use the Rev-RRE system to promote nuclear export, and improve our eGFP maker. When it comes to E. coli, many kinds of safety systems for using genetically-engineered E. coli in the environment have already been considered. Developing such systems for S. cerevisiae may bring about the possibility of their use not only in laboratories, but also in our environment.
Reference
[1] X. rong Wang, X. Cheng, Y. dong Li, J. ai Zhang, Z. fen Zhang, and H. rong Wu, “Cloning arginine kinase gene and its RNAi in Bursaphelenchus xylophilus causing pine wilt disease,” Eur. J. Plant Pathol., vol. 134, no. 3, pp. 521–532, 2012.
[2] F. Halbach, P. Reichelt, M. Rode, and E. Conti, “The yeast ski complex: Crystal structure and rna channeling to the exosome complex,” Cell, 2013.
[3] K. Kalisiak et al., “A short splicing isoform of HBS1L links the cytoplasmic exosome and SKI complexes in humans,” Nucleic Acids Res., vol. 45, no. 4, 2017.
[4] J. D. Shih and C. P. Hunter, “SID-1 is a dsRNA-selective dsRNA-gated channel,” vol. 17(6), pp. 1057–1065, 2011.
[5] M. O. Elhassan, J. Christie, and M. S. Duxbury, “Homo sapiens Systemic RNA Interference-defective-1 Transmembrane Family Member 1 (SIDT1) Protein Mediates Contact-dependent Small RNA Transfer and MicroRNA-21-driven Chemoresistance ,” J. Biol. Chem., 2011.
[6] T. K. Dutta, P. K. Papolu, P. Banakar, D. Choudhary, A. Sirohi, and U. Rao, “Tomato transgenic plants expressing hairpin construct of a nematode protease gene conferred enhanced resistance to root-knot nematodes.,” Front. Microbiol., vol. 6, p. 260, 2015.
[7] B. C. Yadav, K. Veluthambi, and K. Subramaniam, “Host-generated double stranded RNA induces RNAi in plant-parasitic nematodes and protects the host from infection,” Mol. Biochem. Parasitol., vol. 148, pp. 219–222, 2006.
[8] A. P. Page and I. L. Johnstone, “The cuticle,” WormBook, vol. 1.138.1, 2007.
