<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_Kyoto_Results skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><DIV id="top_title"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:Kyoto/Results</SPAN></H1></DIV><DIV id="HQ_page"><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><DIV id="header"><HEADER><UL class="nv"><LI><A href="https://2017.igem.org/team:Kyoto/Project">Project</A></LI><LI><A href="https://2017.igem.org/team:Kyoto/Project">Description</A></LI><LI><A href="https://2017.igem.org/Team:Kyoto/Results">Results</A></LI><LI><A href="https://2017.igem.org/Team:Kyoto/Demonstrate">Demonstrate</A></LI><LI><A href="https://2017.igem.org/Team:Kyoto/Discussion">Discussion</A></LI><LI><A href="https://2017.igem.org/Team:Kyoto/Basic_Part">Basic Parts</A></LI><LI><A href="https://2017.igem.org/Team:Kyoto/Composite_Part">Composite Parts</A></LI></UL><LI><A href="https://2017.igem.org/Team:Kyoto/Human_Practices">Practices</A></LI><LI><A href="https://2017.igem.org/Team:Kyoto/Human_Practices">Human Practices</A></LI><LI><A href="https://2017.igem.org/Team:Kyoto/Integrated_Practices">Integrated Practices</A></LI><LI><A href="https://2017.igem.org/Team:Kyoto/Engagement">Public Engagement</A></LI><LI><A href="https://2017.igem.org/Team:Kyoto/Collaborations">Collaborations</A></LI><LI><A href="https://2017.igem.org/Team:Kyoto/Events">Events</A></LI><LI><A href="https://2017.igem.org/Team:Kyoto/Discussion_HP">Further Consideration</A></LI><LI><A href="https://2017.igem.org/Team:Kyoto/Notebook">Experiment</A></LI><LI><A href="https://2017.igem.org/Team:Kyoto/Notebook">Notebook</A></LI><LI><A href="https://2017.igem.org/Team:Kyoto/Experiments">Material&amp;Methods</A></LI><LI><A href="https://2017.igem.org/Team:Kyoto/Team">Team</A></LI><LI><A href="https://2017.igem.org/Team:Kyoto/Team">Members</A></LI><LI><A href="https://2017.igem.org/Team:Kyoto/Attributions">Attributions</A></LI><LI><A href="https://2017.igem.org/Team:Kyoto/Achievements">Criteria</A></LI><LI><A href="https://2017.igem.org/Team:Kyoto/Achievements">Achievements</A></LI><LI><A href="https://2017.igem.org/Team:Kyoto/HP/Silver">HP/Silver</A></LI><LI><A href="https://2017.igem.org/Team:Kyoto/HP/Gold_Integrated">HP/Gold</A></LI></HEADER></DIV><DIV id="BACKGROUND"><DIV id="wrapper"><H1>Results</H1><H5>Table of contents</H5><UL class="result"><LI><A href="#res1">1) Observe that <I>B. xylophilus</I> feeds on yeast</A></LI><LI><A href="#res2">2) Identify <I>B. xylophilus</I> which feeds yeast by fluorescence</A></LI><LI><A href="#res3">3) Choose dsRNA</A></LI><LI><A href="#res4">4) Conduct feeding RNAi in yeast</A></LI><LI><A href="#res5">5) Observe that <I>B. xylophilus</I> feeds yeast expressing dsRNA</A></LI><LI><A href="#res6">6) Improve transition of mRNA to cytosol</A></LI></UL><H5 id="res1">1) Observe that <I>B. xylophilus </I>feeds on yeast</H5><P>Dr. Y Takeuchi, a researcher of <I>B. xylophilus</I>, gave us a strain that was bred in the laboratory. These nematodes were cultivated with gray mold. It was also possible to grow nematodes on the plate growing gray mold.</P><P>After we washed nematodes cultured in mold with lactic acid to remove mold, we placed them with budding yeast on a new plate. At first we observed nematodes by microscope, but we could not observe them eating yeast. Although they tried to pierce yeast with their stylet, it seemed to be difficult for them to penetrate it well because the yeast was not fixed on the plate then moved, and also originally the yeast was too small.</P><P>So we thought it would be easier to pierce with a stylet if the yeast was bigger, and we modified the conditions and used diploid yeast instead of haploid yeast, which had been used thus far. Visually, diploid yeast is different from haploid with respect to shape and size, where diploid is bigger than haploid (fig 1-a). Therefore, we decided to continue observation with diploid yeast.</P><P class="caption"><B>Figure 1-a</B><I>S. cerevisiae</I> cells. 
Diploid (light blue) and haploid (orange) cells were mixed and photographed to compare their sizes. </P><P>We also improved the filming conditions. We stopped observing the nematodes on the medium directly, and changed to a method of sandwiching the medium and yeast between the slide glass and the cover glass and injecting nematodes from the side. By doing so, the yeast was fixed firmly on the medium and we could solve the problem that the yeast moved when nematodes were trying to pierce it with their stylet.</P><P>Shown in the movie below is the moment <I>B. xylophilus</I> preyed on budding yeast (Movie 1). This was filmed for the first time in the world. The needle-shaped structure found near the nematode's mouth is a stylet. The nematode pushed the stylet against the yeast and inserted it in the yeast. The yeast started to shrink rapidly sometime after the nematode pierced it, and in the end its shape almost disappeared. Yeast has a robust cell wall, but it seems to be crushed with a strong force.</P><P class="picture"><VIDEO controls="">Movie 1 The moment <I>B. xylophilus</I> prey on <I>S. cerevisiae</I>.
</VIDEO></P><P>The results of experiments on whether or not <I>B. xylophilus</I> that ate <I>S. cerevisiae</I> would grow was shown in the figure (Figure 1-b).</P><P class="caption"><B>Figure 1-b</B> Survival rate of <I>B. xylophilus</I> with different foods.
Diploid or haploid cells were grown in SD liquid culture and spread on agar plates. 100 nematodes were grown on each plate and the number of survivors were counted at the indicated time points. (n=3)
</P><P>In the case of haploid or no food, the survival rate decreases at Day 4, but when using diploids the survival rate increases. The large cell diploid yeast seems easier to ingest for <I>B. xylophilus</I>.</P><H5 id="res2">2) Identify <I>B. xylophilus</I> which ate yeast by fluorescence</H5><P>In order to confirm the effect of RNAi, we needed a marker to distinguish nematodes that preyed on yeast from the others within the experimental period. Cultured nematodes included many growth stages, so it was anticipated that they included some nematodes at the stages where they could not eat yeast. In order to distinguish only the individuals which ate yeast and identify the effect, we decided to repeat the feeding experiment using yeast which fluoresces with eGFP.</P><P>The results of observation of nematodes which ate yeast with a fluorescence microscope are shown in the figure (Figure 2-a~Figure 2-d).</P><P>The intestines of <I>B. xylophilus</I> were highlighted by green fluorescence penetrating through the center throughout the entire body. The mouth is on the left side and the anus is on the right side. eGFP seemed to be very stable in the intestine, and the entire intestine fluoresced uniformly. There was some fluorescence discontinued in the center, and this was correspondent with the part where <I>B. xylophilus</I> gonads crossed over the intestine.</P><P>From the above results, it was found that by expressing eGFP, it became possible to clearly distinguish nematodes that preyed on yeast from the others. By expressing dsRNA simultaneously with eGFP, it should be possible to measure the effect of dsRNA efficiently. As can be seen from this photograph, feeding nematodes on yeast expressing GFP also made it possible to clearly observe the structure of <I>B. xylophilus</I> intestine. This method seemed to be effective for closely observing abnormalities occurring in an intestine of <I>B. xylophilus</I>, like gastroscopy using barium in humans.</P><P class="caption"><B>Figure 2-a</B> eGFP expression cassette.<B>Figure 2-b</B> Yeast cells expressing eGFP.
eGFP expression cassette was cloned into a plasmid and introduced into wild type <I>S. cerevisiae</I> strain.
Images were recorded by fluorescence microscopy.<B>Figure 2-c</B> Nematodes with GFP signal.
Nematodes fed on Yeast were visualized by fluorescence microscopy. Fed on eGFP(+) yeast (left) and EGFP(-) yeast (right) are shown. Scale bar: 50μm<B>Figure 2-d</B> Nematodes with GFP signal 2.
(top) Nematode with GFP fluorescence was inspected by confocal microscopy. XY plane and Z plane (cross section) are shown, respectively.
(bottom) Negative control. Yeast cells with no eGFP plasmid were used. Scale bar: 25μm
</P><P>Among the nematodes being cultured, how many nematodes were thought to surely eat yeast? We collected nematodes on the 3rd and 7th day from the start of the culture by eGFP yeast and observed what proportion of them were fluorescent using a fluorescence microscope.
The results are shown in the figure (Figure 2-e). </P><P>In observation immediately after giving yeast, nematodes emitting fluorescence could not be observed, but about 30% of nematodes emitted fluorescence at Day 3. From this, it was found that at least 30% of nematodes at day 3 were ingesting yeast by this time (Figure 2-e). Interestingly, keeping the plate warm as it was continued observation, it was observed that the number of fluorescent nematodes was reduced to about half (15% of the total) in Day 7. (p = 0.001) Yeasts applied to nutrient-free water agar may die due to starvation or dryness by this time, and the supply of new GFP to nematodes has stopped, which may be the cause of this decrease. The fact that the proportion of fluorescent nematodes declines means that GFP in the intestine of nematodes is digested and degraded over time.</P><P class="caption"><B>Figure 2-e</B> Time course of the rate of eGFP(+) nematodes.
Nematodes were grown on eGFP(+) yeast and examined by fluorescence microscopy at the indicated time. (n=18) 
</P><H5 id="res3">3) Choose dsRNA</H5><P>In order to kill <I>B. xylophilus</I>, it is necessary to efficiently knock-down genes essential for growth. Examining the literature, we found a paper that succeeded in knocking down essential genes and reducing the survival rate by submerging <I>B. xylophilus</I> in high concentrations of dsRNA for a certain period of time (soaking RNAi) [1]. In this paper, the target was mRNA of arginine kinase AK1 which was an essential gene expressed in the intestines and RNAi of AK1 showed a fatal effect also in C. elegans. AK1 is an invertebrate-specific key enzyme of energy metabolism so it is often used as a target for development of invertebrate-specific inhibitors. It was a promising candidate for dsRNA expressed in yeast.
In order to select the most ideal target, we prepared dsRNA of several target mRNAs including AK1 in vitro and tried soaking RNAi. We obtained target sequence from a public database, designed oligos, and cloned genes by RT-PCR. At this time, we put the T7 promoter on both ends of the DNA so that dsRNA was synthesized by in vitro transcription. After transcription, association of dsRNA was induced by an annealing operation, and the template was removed with DNase. We confirmed the dsRNA finally obtained by electrophoresis (Figure 3-a).
</P><P class="caption"><B>Figure 3-a</B> In vitro synthesized dsRNAs for soaking RNAi experiments.
M DNA size marker, λSty I
1 dsAK-2 (692-bp)
2 dsEef-1g (528-bp)
3 dsAK-1 (449-bp)
4 dsAsb (559-bp)
5 ds14-3-3zeta (534-bp)
6 dsTropomyosin (532-bp)
7 ds14-3-3 protein (610-bp)
8 dsGFP (649-bp)
</P><P>We prepared these RNAs, adjusted to a concentration of 2 μg /μL, and tried soaking RNAi The results is shown in the figure (Figure 3-b). As is clearly shown, we could not see the phenotype due to the introduction of dsRNA which was inconsistent with the previously reported example. As a result of contacting several <I>B. xylophilus</I> researchers and gathering information, it turned out that even several Japanese researchers have attempted to reproduce <I>B. xylophilus</I> soaking RNAi, but no group was able to observe a clear effect. The reason may be that soaking RNAi of <I>B. xylophilus</I> contains technically unstable steps. Alternatively, since <I>B. xylophilus</I> used this time is derived from wild nematodes collected from the field, there may be a difference between the strain we used and nematodes in the publication where soaking RNAi was effective. </P><P>Although the effect of soaking RNAi was not observed, we decided to target the AK1 gene because there is already the report[1], and constructed the expression system of dsRNA in yeast.</P><P class="caption"><B>Figure 3-b</B> Effects of soaking RNAi
<I>B. xylophilus</I> were soaked into 2 mg/mL dsRNAs shown in Figure 3-a. After 4h incubation, nematodes were washed and incubated on M9 buffer plate (time=0h). Plates were examined for mortality of the nematodes up to 24h. The method of soaking RNAi was based on reference[1].
</P><H5 id="res4">4) Conduct feeding RNAi in yeast</H5><P>In order to express AK1-dsRNA, we placed inverted repeat derived from AK1 ORF downstream of the Gal1 promoter of the part (<A href="http://parts.igem.org/wiki/index.php/Part:BBa_K517000">BBa_K517000</A>), and inserted a small loop sequence of 67-nt between repeats. There was a report that this loop sequence was effective when <I>S. cerevisiae</I> expressed long dsRNA[2]. Since <I>S. cerevisiae</I> has no Dicer homolog, dsRNA is not processed into siRNA. However, overexpression of dsRNA may be toxic to <I>S. cerevisiae</I>, so we adopted the Gal1 conditional promoter. When <I>S.cerevisiae</I> is cultured in the presence of glucose, this promoter is inactive, and many mRNAs are expressed when the carbon source of the medium is replaced with galactose. At the same time, we also used the GPD promoter (<A href="http://parts.igem.org/Part:BBa_K517001">BBa_K517001</A>) which is a constitutive expression type promoter. dsGFP with a sequence specific to GFP and was designed as a negative control. Outline of construction is shown below (Figure 4-a).</P><P class="caption"><B>Figure 4-a</B> dsRNA expression vectors we used.
Left: Design of each dsRNA cassette. Plus strand and minus strand are tandemly transcribed as a single strand RNA. To enhance dsRNA formation, a short hairpin loop sequence was inserted in between.
Right: Our plasmids used in this study. Gal1 promoter or GPD promoter was fused to dsAK1 or dsGFP (negative control) cassettes. The cassetes were cloned in YEPlac195 plasmid (2-micron, high-copy number plasmid).
</P><P>We cultured plasmid-containing yeasts in several media, collected RNA, and quantified by qRT-PCR with the &quot;loop&quot; part as a target.Moreover, it is known that various viruses of dsRNA type exist in <I>S. cerevisiae</I>. As a factor closely related to the life cycle of such a virus, Ski gene group is known. Many of these are now revealing detailed functions. The Ski complex binds to the 3 'end of RNA and serves as a cofactor for RNA exosome, which is an exonuclease complex that degrades RNA in the 3-5 direction. By binding to the 3 'end to disband the higher-order structure of RNA, it makes the recognition of substrate by exosome efficient. Since the dsRNA virus is known to proliferate in the ski2Δ strain [3], it was hoped that the use of this strain would greatly increase the yield of the target dsRNA. 
</P><P>
The following is the result of qRT-PCR (Figure 4-c). First, expression of dsRNA was successfully detected when wild-type yeast into which Gal1 promoter-dsAK1 was introduced was induced by galactose. Almost the same values ​​are obtained even when the target of the primer set used for qRT-PCR is set to the loop portion or set within the AK1 gene. On the other hand, expression was suppressed as expected when replacing the medium with Glucose. From this, it was demonstrated that it is possible to conditionally induce long hairpin RNA expression using our plasmid.
Interestingly, the expression in Ski2Δ strain is higher than that in WT strain (p &lt;0.05). This indicates that in wild type yeast, Ski complex is degrading targeting foreign dsRNA in addition to RNA virus as expected. From these results, it was found that it is possible to raise the intracellular concentration of exogenous dsRNA by using yeast mutant strain.
In GPD promoter (<A href="http://parts.igem.org/Part:BBa_K517001">BBa_K517001</A>), dsRNA could not be expressed. This part is composed of only the 112 bp sequence near the center out of the TDH 3 promoter (588 bp). Strong expression was confirmed when the 588 bp full-length promoter (<A href="http://parts.igem.org/Part:BBa_K530008">BBa_K 530008</A>) was used for eGFP expression experiments (Figure 2-a, 2-b), so we believe that there is a high probability that this part is defective.
</P><P class="caption"><B>Figure 4-b</B> Quantification of dsRNA in <I>S. cerevisiae</I>
Dr. Makoto Kitabatake (Institute for frontier life and medical science, Kyoto University) performed qRT-PCR using purified yeast total RNAs. The indicated strains were grown in SD-Glucose or SD-Galactose and harvested at the mid to late log phase. Total RNAs were isolated by MasterPure Yeast RNA purification kit (Lucigen) and analyzed by SuperScript III Platinum SYBR qRT-PCR kit (Thermo) after DNase treatment. The primers used in this assay are shown. Quantification of dsRNA was normalized by 25S rRNA. (n=3)</P><H5 id="res5">5) Observe that <I>B. xylophilus </I>feeds on yeast expressing dsRNA</H5><P>We let <I>B. xylophilus</I> prey on the yeast prepared as described above and recorded the survival rate and behavior of nematodes as follows.</P><P>We counted the number of surviving nematodes which fed on dsRNA / eGFP expressing yeast every other day. We also confirmed the survival rate among nematodes that showed fluorescence of eGFP.</P><P class="caption"><B>Figure 5-a</B> Survival rate of <I>B. xylophilus</I> fed by yeast expressing dsAK1
Diploid yeast strain carrying the both dsAK1 and eGFP plasmids were grown in SD-Glucose (glu) or SD-Galactose (gal) medium and spread on new plates. ~100 nematodes were grown on each plate and examined by microscopy. (n=3) </P><P class="caption"><B>Figure 5-b</B> Mortality of GFP(+) nematodes
Nematodes fed by diploid yeast carrying dsAK1 and eGFP plasmids were examied by fluorescence microscopy. Gal, yeast culture was prepared by SD-Galactose medium (dsRNA induction). Glu, the same yeast strain was grown in SD-Glucose medium (dsRNA repression). (n=3) </P><P class="caption"><B>Figure 5-c</B> Survival rate of <I>B. xylophilus</I> fed by Ski2Δ strain
Feeding RNAi experiments were performed as in Figure 5-a. Ski2Δ strain instead of WT strain was used as prey. (n=1)
</P><P>From the above results, the number of nematodes did not decline predominantly even when using yeast whose expression of dsAK1 was confirmed (Figure 5-a). According to previous experiments, when the nematodes cultured by eGFP(+) yeast, the rate of eGFP(+) labbeled nematodes was only about 30% (Figure 2-d). For this reason, even if dsRNA taken in kills nematodes, since the proportion of nematodes ingesting a sufficient number of yeasts to obtain the effect is not so high, there is a possibility that the effect given by dsRNA has been diluted. We focused only on nematodes that fed yeast and confirmed the mortality of fluorescent nematodes to evaluate the effect of dsRNA (Figure 5-b).
Even in this case, we could not confirm the effect of dsRNA as expected.
Moreover, the survival rate of nematodes is lower when using yeast cultured in +glu SD medium which should suppress the expression of dsRNA.
The results were the same even when using ski2Δ yeast in which the expression level of dsRNA was increased (Figure 5-c).
This seemingly contradictory result will be discussed later in the discussion.

We thought that there might be some obstacle before dsRNA was taken up by nematodes and sought out the cause.</P><H5 id="res6">6) Improve transport of mRNA to cytosol</H5><P>As shown in the figure (Figure 6-a), the diameter of the stylet is very small, about a fraction of a single cell of yeast. For this reason, <I>B. xylophilus</I> seemed to draw out the cytoplasmic fraction, but large cellular componentns such as the nucleus may not be efficiently consumed by <I>B. xylophilus.</I></P><P class="caption"><B>Figure 6-a</B> Nematode’s stylet, diploid yeast, and haploid yeast. Scale bar : 10μm
</P><P>A number of studies have been done on the nuclear export of mRNA, and the basic mechanism has been elucidated. It is known that various RNAs such as mRNA, rRNA, tRNA, etc. are recognized by transporters specific to each type and pass through the nuclear pore complex[4]. However, since the dsRNA as prepared this time does not exist in nature, it is not known whether there is a transport factor that recognizes this RNA or whether it is efficiently transported out of nucleus.</P><P>In order to prepare remedies for this problem, we tried experiments utilizing the REV factor of HIV-1 RNA, which is known to have the function of improving the efficiency of nuclear export of RNA.</P><P>As shown in the figure, REV plays the role of carrying an unspliced RNA genome to the cytoplasm in the life cycle of HIV-1 (Figure 6-b). In the case of ordinary mRNA, there is a retention mechanism that prevent molecules retaining introns from transferring out of the nucleus, thus preventing the transport of immature mRNA. REV binds to a specific part (RRE: Rev responsive element) of the intron on the HIV-1 RNA genome and binds itself to the nuclear export factor CRM1, and overcomes such a retention mechanism and transports RNA to the cytoplasm[5]. Even if the dsRNA is not recognized as a nuclear export factor or even if it is retained in the nuclear retention factor, we thought that it is possible that the efficiency of nuclear export can be improved by inserting REV-RRE system, and provide these new parts to the iGEM community (<A href="http://parts.igem.org/Part:BBa_K2403000">BBa_K2403000</A>, 
<A href="http://parts.igem.org/Part:BBa_K2403002"> BBa_K2403002</A>)</P><P class="caption"><B>Figure 6-b</B> Rev protein induces nuclear export of RRE-containing RNAs.
</P><P>The results of microinjection of RI-labeled dsRNA with RRE into the nucleus of Xenopus oocytes are shown in the figure (Figure6c ~ Figure6f). Nuclear and cytoplasm were separated after a certain period of time following injection, RNA was recovered from each and analyzed . </P><P class="caption">
* Dr. Taniguchi, Institute for fronter life and medical science, Kyoto university, conducted an experiment on the process of treating RI on and after in vitro transcription.</P><P class="caption"><B>Figure 6-c</B> Outline of Xenopus oocyte microinjection</P><P class="caption"><B>Figure 6-d</B> RNAs produced by in vitro transcription
U6 and U6-RRE are used as controls.
</P><P class="caption"><B>Figure 6-e</B> Microinjection of dsGFP with or without RRE into Xenopus oocyte
Indicated RNAs with or without Rev protein were injected into Xenopus oocyte nucleus. The oocytes were dissected at the indicated time points (t=0, 60 min). The nuclear RNAs (N) and the cytoplasmic RNAs (C) were extracted and analyzed by PAGE. The left pannel shows dsGFP with RRE (GFP-RRE + GFPrev), the right pannel shows dsGFP without RRE (GFPfwd + GFPrev).</P><P class="caption"><B>Figure 6-f</B> Long exposure
</P><P>
As expected, when U6-RRE is injected together with buffer without Rev, U6-RRE remains in the nucleus, whereas U6 - RRE is injected with buffer containing Rev, U6-RRE was remarkably transported outside the nucleus. This result demonstrated that nuclear export of RNA is promoted depending on both the RRE sequence and the Rev protein in the case of U6 RNA originally staying in the nucleus. These effects indicate that these parts are promising as devices for efficiently transporting highly structured RNA, which is often used in synthetic biology, to the cytoplasm.(<A href=" http://parts.igem.org/Part:BBa_K2403000">BBa_K2403000</A><A href="http://parts.igem.org/Part:BBa_K2403002">BBa_K2403002</A>)
Unfortunately, transcription products of GFP-RRE were too thin to understand whether they responded to Rev. However, from the figure on the right of Figure 6-f, it was also found that dsRNA remained in the nucleus a lot. It is suggested that implementation of a system to promote nucleocytoplasmic transport is effective.
Compared to the signal at T = 0, since many signals are lost at 60 minutes after injection, there may be a mechanism for degrading dsRNA in cells. In addition to promoting transportation efficiency, there will be room for improvement to improve stability.
</P><P>This is the Result obtained in this project.
We would like to discuss Discussion on interpretation of Result and Future plan.</P><H6>Reference</H6><UL class="reference"><LI>[1]	X. rong Wang, X. Cheng, Y. dong Li, J. ai Zhang, Z. fen Zhang, and H. rong Wu, “Cloning arginine kinase gene and its RNAi in <I>Bursaphelenchus xylophilus</I> causing pine wilt disease,” Eur. J. Plant Pathol., vol. 134, no. 3, pp. 521–532, 2012.</LI><LI>[2]	A. Sigova, N. Rhind, and P. D. Zamore, “A single Argonaute protein mediates both transcriptional and posttranscriptional silencing in Schizosaccharomyces pombe,” genes Dev., 2004.</LI><LI>[3]	R. Esteban and R. B. Wickner, “A new non-mendelian genetic element of yeast that increases cytopathology produced by M1 double-stranded RNA in ski strains.,” Genetics, 1987.</LI><LI>[4]	M. T. B. Sloan, Katherine E, Pierre-Emmanuel Gleizes, “Nucleocytoplasmic Transport of RNAs and RNA–Protein Complexes,” J. Mol. Biol., vol. 428, no. 10, pp. 2040–2059, 2016.</LI><LI>[5]	V. W. Pollard and M. H. Malim, “the Hiv-1 Rev Protein,” Annu. Rev. Microbiol., vol. 52, no. 1, pp. 491–532, 1998.</LI></UL></DIV></DIV></DIV></DIV></DIV></DIV></DIV></BODY></HTML>