<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_Kyoto_Discussion skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><DIV id="top_title"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:Kyoto/Discussion</SPAN></H1></DIV><DIV id="HQ_page"><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><DIV id="header"><HEADER><UL class="nv"><LI><A href="https://2017.igem.org/team:Kyoto/Project">Project</A></LI><LI><A href="https://2017.igem.org/team:Kyoto/Project">Description</A></LI><LI><A href="https://2017.igem.org/Team:Kyoto/Results">Results</A></LI><LI><A href="https://2017.igem.org/Team:Kyoto/Demonstrate">Demonstrate</A></LI><LI><A href="https://2017.igem.org/Team:Kyoto/Discussion">Discussion</A></LI><LI><A href="https://2017.igem.org/Team:Kyoto/Basic_Part">Basic Parts</A></LI><LI><A href="https://2017.igem.org/Team:Kyoto/Composite_Part">Composite Parts</A></LI></UL><LI><A href="https://2017.igem.org/Team:Kyoto/Human_Practices">Practices</A></LI><LI><A href="https://2017.igem.org/Team:Kyoto/Human_Practices">Human Practices</A></LI><LI><A href="https://2017.igem.org/Team:Kyoto/Integrated_Practices">Integrated Practices</A></LI><LI><A href="https://2017.igem.org/Team:Kyoto/Engagement">Public Engagement</A></LI><LI><A href="https://2017.igem.org/Team:Kyoto/Collaborations">Collaborations</A></LI><LI><A href="https://2017.igem.org/Team:Kyoto/Events">Events</A></LI><LI><A href="https://2017.igem.org/Team:Kyoto/Discussion_HP">Further Consideration</A></LI><LI><A href="https://2017.igem.org/Team:Kyoto/Notebook">Experiment</A></LI><LI><A href="https://2017.igem.org/Team:Kyoto/Notebook">Notebook</A></LI><LI><A href="https://2017.igem.org/Team:Kyoto/Experiments">Material&amp;Methods</A></LI><LI><A href="https://2017.igem.org/Team:Kyoto/Team">Team</A></LI><LI><A href="https://2017.igem.org/Team:Kyoto/Team">Members</A></LI><LI><A href="https://2017.igem.org/Team:Kyoto/Attributions">Attributions</A></LI><LI><A href="https://2017.igem.org/Team:Kyoto/Achievements">Criteria</A></LI><LI><A href="https://2017.igem.org/Team:Kyoto/Achievements">Achievements</A></LI><LI><A href="https://2017.igem.org/Team:Kyoto/HP/Silver">HP/Silver</A></LI><LI><A href="https://2017.igem.org/Team:Kyoto/HP/Gold_Integrated">HP/Gold</A></LI></HEADER></DIV><DIV id="BACKGROUND"><DIV id="wrapper"><H1>Discussion</H1><H5>Discussion</H5><UL class="discussion"><LI><A href="#summary">1) The summary</A></LI><LI><A href="#method">2) The method of RNAi by feeding</A></LI><LI><A href="#dsRNA">3) Where is dsRNA in <I>S. cerevisiae</I>?  Do <I>B. xylophilus</I> suck up dsRNA?</A></LI><LI><A href="#is">4) Is feeding RNAi effective?</A></LI><LI><A href="#genes">5) Targeting essential nematode genes</A></LI><LI><A href="#future">6) Future</A></LI></UL><H5 id="summary">1) The summary</H5><P>Pine-wilt disease, which is spreading all over the world, is one of the major plant diseases causing severe economic damage. A significant amount of money and labor are required for the treatment of forests each year. We focused on the cause of the disease, a nematode called <I>B. xylophilus</I>, and started our “B. x. Busters” project designed to create a new genetically modified microorganism exterminating <I>B. xylophilus</I>. We searched for a carrier of RNAi by feeding and revealed that under laboratory conditions, <I>B. xylophilus</I> preys on <I>S. cerevisiae</I> by sucking up the yeast’s insides using a straw-like stylet. We developed a reporter distinguishing <I>B. xylophilus</I> which ate <I>S. cerevisiae</I> by recording green fluorescence in the digestive track of <I>B. xylophilus</I> in live microscopy. In addition, we constructed a plasmid for <I>S. cerevisiae</I> expressing dsRNA, characterized its expression, and observed <I>B. xylophilus</I> which preyed on <I>S. cerevisiae</I> expressing dsRNA. These results, combined with our integrated Human Practices, could bring about promising new methodology in the fight against pine-wilt disease.</P><H5 id="method">2) The method of RNAi by feeding</H5><P>First, by feeding RNAi we aimed to knockdown the AK1 gene of <I>B. xylophilus</I>, since a previous study claimed success in targeting this gene with soaking RNAi [1]. However, we couldn’t reproduce any fatal effect to <I>B. xylophilus</I> just by soaking in RNA solution. Instead, we chose to feed <I>B. xylophilus</I> with <I>S. cerevisiae</I> expressing dsRNA by the conditional Gal1 promoter (BBa_K517001). We spread <I>S. cerevisiae</I> cultured in galactose medium onto no-nutrient agar medium containing no carbon source and fed it to <I>B. xylophilus</I>. This low nutrient condition prevents miscellaneous germs from proliferating, but at the same time the <I>S. cerevisiae</I> also lack nutrients. It probably led to autophagy. Under these conditions, it’s possible that low dosages of dsRNA were given to <I>B. xylophilus</I> than we had initially anticipated. We recognize that we have plenty of room for improvement, optimizing the agar culture condition and preparing <I>S. cerevisiae</I> which maintaining dsRNA expression more stably for a longer period of time.</P><H5 id="dsRNA">3) Where is dsRNA in <I>S. cerevisiae</I>?  Do <I>B. xylophilus</I> suck up dsRNA?</H5><P> Our experiments confirmed SKI2Δ <I>S. cerevisiae</I> contains more dsRNA than wild type yeast. SKI complex is found in the cytoplasm [2],[3].In wild type yeast, SKI complex degrades dsRNA. Therefore, our results suggest that the dsRNA is partially in the cytoplasm. However, the results of Xenopus oocyte microinjection suggested that almost all the dsRNA is present in the nucleus. From our live imaging, it was not possible to see the yeast nucleus passed through the small diameter of the nematode’s stylet. If <I>B. xylophilus</I> doesn’t eat the yeast nucleus, the dsRNA will not reach the nematode’s body. Our further research confirmed that we could use the HIV Rev-RRE nuclear export pathway to help export dsRNA into the cytoplasm. Therefore, we predict that <I>S. cerevisiae</I> which has this function would improve dsRNA nuclear export to the cytoplasm, increasing the effective dose to predator <I>B. xylophilus</I>.</P><P>The eGFP feeding reporter we established provided us with ideas for optimizing RNAi by feeding. For instance, we could express a histone-eGFP fusion protein in <I>S. cerevisiae</I> and verify if the nucleus of <I>S. cerevisiae</I> is in fact eaten by <I>B. xylophilus</I>. Extracting DNA or RNA derived from <I>S. cerevisiae</I> will reveal whether <I>B. xylophilus</I> eats the yeast nucleus by identifying the nucleic acid of <I>S. cerevisiae</I> in the intestinal track.</P><H5 id="is">4) Is feeding RNAi effective?</H5><P>In <I>C. elegans</I>, since a dsRNA-specific bidirectional channel (SID-1) is expressed in the cell membrane [4], it is known that feeding RNAi occurs efficiently. Interestingly, human and Drosophila preserve proteins with similar functions [5]. As we were unable to reproduce published soaking RNAi results with <I>B. xylophilus</I>, we examined whether such channel proteins even existed in B. xylophilus using BLAST search.</P><P>When examining <I>C. elegans</I> SID-1 (GenBank: AF 478687.1) as a query, it seems that there is no highly homologous sequence in the <I>B. xylophilus</I> reference genome. As far as we consider it, the results we obtained seem to be reasonable, but we cannot exclude the possibility that the <I>B. xylophilus</I> draft genome is incomplete.</P><P>We also researched <I>Meloidogyne incognita</I> which is nematode with a stylet like <I>B. xylophilus</I>. Similar to <I>B. xylophilus</I>, we could not find SID-1 homologues in <I>M. incognita</I>. Interestingly, there is a report that remarkable effect was obtained in <I>M. incognita</I> with feeding RNAi using plants [6],[7]. We hypothesize that <I>M. incognita</I> has another RNAi enabling protein and RNAi reaction is occurring differently from <I>C. elegans</I>.</P><P>Finally, we considered that the mechanism of RNAi is relevant for selecting target genes. For example, if dsRNA cannot be transported from the intestinal tract to other cells of the body, it is necessary to select a gene expressed in the intestine as a target. RNAi targeting a gene such as a digestive enzyme or channel protein could result in starvation of the nematodes, reducing their fitness. As an alternative, it may be possible to engineer yeast that express a transporter to assist with dsRNA delivery to other tissues.</P><H5 id="genes">5) Targeting essential nematode genes</H5><P>We tried to confirm whether RNAi by feeding has a fatal effect on <I>B. xylophilus</I> survival. However, we had no method to know the difference between life and death except for judging from the movement of <I>B. xylophilus</I> and, to make matters worse, dead <I>B. xylophilus</I> quickly dried-up and became difficult to detect. Therefore, it may be required to select a target gene (such as dpy [8]) which expressed an obvious change in phenotype without killing, to establish the effect of RNAi by feeding in the future.</P><H5 id="future">6) Future</H5><P>In the beginning, we intended to develop our “B. x. Busters” yeast as a biological pesticide, but we may need to solve many problems in advance, such as biosafety. As we described in our Human Practice (<A href="https://2017.igem.org/Team:Kyoto/Discussion_HP">https://2017.igem.org/Team:Kyoto/Discussion_HP</A>), a promising strategy, attempting to create genetically-engineered pine trees expressing RNAi, is proposed. Our yeast system should be ideal for screening target genes and effective RNAi. RNAi by feeding with <I>S. cerevisiae</I> with optimization should reduce the time required for growing plants and would help us to realize our plan earlier. Accordingly, we plan to further improve our “B. x. Busters” and establish more effective RNAi by feeding. In addition to SKI2, we should verify the effect that other RNA metabolism-related factors may have on dsRNA accumulation. We can also use the Rev-RRE system to promote nuclear export, and improve our eGFP maker. When it comes to <I>E. coli</I>, many kinds of safety systems for using genetically-engineered <I>E. coli</I> in the environment have already been considered. Developing such systems for <I>S. cerevisiae</I> may bring about the possibility of their use not only in laboratories, but also in our environment.</P><H6>Reference</H6><UL class="reference"><LI>[1] X. rong Wang, X. Cheng, Y. dong Li, J. ai Zhang, Z. fen Zhang, and H. rong Wu, “Cloning arginine kinase gene and its RNAi in <I>Bursaphelenchus xylophilus</I> causing pine wilt disease,” Eur. J. Plant Pathol., vol. 134, no. 3, pp. 521–532, 2012.</LI><LI>[2] F. Halbach, P. Reichelt, M. Rode, and E. Conti, “The yeast ski complex: Crystal structure and rna channeling to the exosome complex,” Cell, 2013.</LI><LI>[3]	K. Kalisiak et al., “A short splicing isoform of HBS1L links the cytoplasmic exosome and SKI complexes in humans,” Nucleic Acids Res., vol. 45, no. 4, 2017.</LI><LI>[4]	J. D. Shih and C. P. Hunter, “SID-1 is a dsRNA-selective dsRNA-gated channel,” vol. 17(6), pp. 1057–1065, 2011.</LI><LI>[5]	M. O. Elhassan, J. Christie, and M. S. Duxbury, “Homo sapiens Systemic RNA Interference-defective-1 Transmembrane Family Member 1 (SIDT1) Protein Mediates Contact-dependent Small RNA Transfer and MicroRNA-21-driven Chemoresistance ,” J. Biol. Chem., 2011.</LI><LI>[6]	T. K. Dutta, P. K. Papolu, P. Banakar, D. Choudhary, A. Sirohi, and U. Rao, “Tomato transgenic plants expressing hairpin construct of a nematode protease gene conferred enhanced resistance to root-knot nematodes.,” Front. Microbiol., vol. 6, p. 260, 2015.</LI><LI>[7]	B. C. Yadav, K. Veluthambi, and K. Subramaniam, “Host-generated double stranded RNA induces RNAi in plant-parasitic nematodes and protects the host from infection,” Mol. Biochem. Parasitol., vol. 148, pp. 219–222, 2006.</LI><LI>[8]	A. P. Page and I. L. Johnstone, “The cuticle,” WormBook, vol. 1.138.1, 2007.</LI></UL></DIV></DIV></DIV></DIV></DIV></DIV></DIV></BODY></HTML>