<HTML xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en" dir="ltr">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki  ltr ns-0 ns-subject page-Team_UNITN-Trento_Project_CrustAway"><DIV id="globalWrapper"><DIV id="top-section"><DIV id="p-logo"><A href="/Main_Page" title="Main Page">&quot;
	    </A></DIV><DIV id="menubar" class="left-menu noprint"><UL><LI class="selected"><A href="/Team:UNITN-Trento/Project/CrustAway">Page               </A></LI><LI class="new"><A href="/wiki/index.php?title=Talk:Team:UNITN-Trento/Project/CrustAway&amp;action=edit&amp;redlink=1">Discussion               </A></LI><LI><A href="/wiki/index.php?title=Team:UNITN-Trento/Project/CrustAway&amp;action=edit">View source               </A></LI><LI><A href="/wiki/index.php?title=Team:UNITN-Trento/Project/CrustAway&amp;action=history">History               </A></LI><LI style="color:#808080;cursor:default">teams</LI></UL></DIV><DIV class="right-menu noprint" id="menubar"><UL><LI id="pt-login"><A href="/wiki/index.php?title=Special:UserLogin&amp;returnto=Team:UNITN-Trento/Project/CrustAway" title="You are encouraged to log in; however, it is not mandatory [o]" accesskey="o">Log in</A></LI></UL></DIV><DIV id="search-controls" class="noprint"><FORM action="/Special:Search" id="searchform"> </FORM></DIV></DIV><DIV id="content"><H1 class="firstHeading">Team:UNITN-Trento/Project/CrustAway</H1><DIV id="bodyContent"><H3 id="siteSub" class="noprint">From 2012.igem.org</H3><P><TITLE>Crust Away</TITLE></P><H2 style="font-family: 'Open Sans'; line-height: 40px; color:#ffae00; padding:0.5em; font-size:30pt !important; text-align: center; width: 900px; margin: 100px auto;">It appears you have JavaScript disabled. Our Wiki needs it for you to enjoy the full experience. Please enable Javascript and refresh the page.</H2><DIV id="header"><DIV id="topNav"><UL><LI><A href="https://2012.igem.org/Team:UNITN-Trento">Home</A></LI><LI class="folder activeLink"><A href="javascript:void(0);">Project</A></LI><LI><A href="https://2012.igem.org/Team:UNITN-Trento/Project/CrustAway">Crust Away</A></LI><LI><A href="https://2012.igem.org/Team:UNITN-Trento/Project/Terminators">Terminator 5</A></LI><LI><A href="https://2012.igem.org/Team:UNITN-Trento/Attributions">Attributions</A></LI></UL></DIV><LI><A href="https://2012.igem.org/Team:UNITN-Trento/DataPage">Data Page</A></LI><LI class="folder"><A href="javascript:void(0);">Parts</A></LI><LI><A href="https://2012.igem.org/Team:UNITN-Trento/Parts">Our Parts</A></LI><LI><A href="http://partsregistry.org/cgi/partsdb/pgroup.cgi?pgroup=iGEM2012&amp;group=UNITN-Trento">Registry Section</A></LI></DIV><LI><A href="https://2012.igem.org/Team:UNITN-Trento/Achievements">Judging</A></LI><LI class="folder"><A href="javascript:void(0);">Team</A></LI><LI><A href="https://2012.igem.org/Team:UNITN-Trento/Team">The Team</A></LI><LI><A href="https://2012.igem.org/Team:UNITN-Trento/Team/Supervisors">Super- &amp; Ad- visors</A></LI><LI><A href="https://2012.igem.org/Team:UNITN-Trento/Friends">Our Friends</A></LI><LI><A href="https://igem.org/Team.cgi?id=731">Team Profile</A></LI><LI><A href="https://2012.igem.org/Team:UNITN-Trento/Trento">Where We Are</A></LI><LI><A href="https://2012.igem.org/Team:UNITN-Trento/Press">Press</A></LI></DIV><LI class="folder"><A href="javascript:void(0);">Partners</A></LI><LI><A href="https://2012.igem.org/Team:UNITN-Trento/Sponsors">Sponsors</A></LI></DIV><LI class="folder"><A href="javascript:void(0);">Lab Notes</A></LI><LI><A href="https://2012.igem.org/Team:UNITN-Trento/Notebook">Notebook</A></LI><LI><A href="https://2012.igem.org/Team:UNITN-Trento/Meetings">Meetings</A></LI><LI><A href="https://2012.igem.org/Team:UNITN-Trento/Protocols">Protocols</A></LI><LI><A href="https://2012.igem.org/Team:UNITN-Trento/H2S">H<SUB>2</SUB>S</A></LI><LI><A href="https://2012.igem.org/Team:UNITN-Trento/Tools">Tools</A></LI></DIV><LI class="folder"><A href="javascript:void(0);">Safety</A></LI><LI><A href="https://2012.igem.org/Team:UNITN-Trento/Safety">Safety</A></LI><LI><A href="https://2012.igem.org/Team:UNITN-Trento/H2S">H<SUB>2</SUB>S</A></LI><LI><A href="https://2012.igem.org/Team:UNITN-Trento/Outreach">Outreach</A></LI><LI><A href="https://2012.igem.org/Team:UNITN-Trento/Art">Art&amp;Science</A></LI><DIV id="logo"><DIV class="wrapper"><H1><A href="https://2012.igem.org/Team:UNITN-Trento" style="font-family: 'Bree Serif';"><SPAN>i</SPAN>GEM Trento 2012</A></H1></DIV></DIV><DIV id="canvas" class="home"><DIV id="wrapper" style=""><SECTION id="main" style="width:100%; position: relative;"><DIV class="punch" style="margin: 0 auto;"><H2 style="text-align: center; font-size: 25px;">Crust Away</H2></DIV><DIV id="selector"><DIV id="intro" class="but now"><SPAN class="title">Introduction</SPAN></DIV><DIV id="method" class="but"><SPAN class="title">Our Method</SPAN></DIV><DIV id="results" class="but"><SPAN class="title">Results</SPAN></DIV><DIV id="application" class="but"><SPAN class="title">Application</SPAN></DIV><DIV class="but" id="intro" style="position: relative; left: 40px;"><SPAN class="title" style="color: black;">Terminators</SPAN></DIV></DIV><DIV class="content" style="padding-left: 20px ;padding-right: 20px ;"><DIV class="chapter intro visible"><H2 style="text-align: center;">INTRODUCTION</H2><P>Have you ever noticed that ugly black crust found on many precious monuments and statues? Here in Italy, we surely have it, and we’d like to find a way to return our monuments to their natural, beautiful state. </P><P>Our idea is to create a bioremediation kit to help every sculptor, restorer, marble cutter or anyone interested in removing that ugly and harmful black crust layer from their precious calcareous stones. Our target is the gypsum layer (i.e. CaSO<SUB>4</SUB>⋅2H<SUB>2</SUB>O) that is the main component of the black crust. A lot of chemical and mechanical approaches are already on the market, and some have also exploited in the past Sulfur Reducing Bacteria (SRB) to eliminate the trapping gypsum matrix (1, 2). </P><P>However all of these methods show some weak points:
							<OL><LI>Chemical and mechanical methods are too invasive and they risk to damage the underlying and precious marble surface; We have met with experts in the restoration field and learned about the <A href="https://2012.igem.org/Team:UNITN-Trento/Art">criteria to define a good restoration</A>.</LI><LI>Chemical methods have also health related risks. Workers, artists, restorers too often use chemicals to clean marble stones in unsafe conditions. We have met with one of the many local artists and reported our impressions on his methods to clean marble in the <A href="https://2012.igem.org/Team:UNITN-Trento/Art">Art &amp; Science</A> section of our Wiki. </LI><LI>Natural SRB, which have been used in a few cases, instead need anaerobic conditions and they constitutively express the enzymes required for sulphur reduction, thus not leaving the choice to the operator to control the rate and amount of sulphur reduced.</LI></OL>
							As a valid alternative we propose an engineered <I>E. coli</I> capable of reducing the sulphuric gypsum matrix that should have the following advantages:
							<OL><LI>Work in <B>AEROBIC</B> conditions: this is a breakthrough in the field of SRB!</LI><LI>Should be <B>CONTROLLED</B> and <B>MODULATED</B> as needed.</LI><LI>Should <B>NOT</B> be <B>INVASIVE</B> and work selectively against the gypsum matrix without touching the calcareous surface.</LI><LI>Should be <B>CHEAP</B> and <B>EASY TO APPLY</B>. There will be no more need of expensive chemical and trained workers that have to work many hours to clean our statues and monuments.</LI></OL></P><P>To summarize, with our biological system we want to mix the strengths of all the available methods and eliminate their weak points to propose an infallible and very safe method to clean precious marble pieces!</P><H3>How does the black crust form?</H3><P>One of the main factors in the formation of the black crust is the atmospheric pollution. The burning process of fossil fuels leads to an increase in the concentration of some acid gases in the atmosphere. In particular SO<SUB>2</SUB> when reacts with water induces the transformation of calcite (CaCO<SUB>3</SUB>, present in the stone substrate) into gypsum (CaSO<SUB>4</SUB>·2H<SUB>2</SUB>O), which precipitates with inclusions of carbon particulate matter.</P><P>The black crust is usually found on areas of the stone sheltered from rainfall, although still in presence of capillary water flowing through the pores. Calcium ions migrate to the surface of the stone during gypsum formation, leading to formation of cavities beneath the stone that weakens the stone’s integrity. The degradation process indeed affects both the stone conservation and appearance:
						<UL><LI>When the formed gypsum is washed away it takes some of the stone particles with it, causing at first loss of detail, but eventually leading, again, to a loss of structural integrity.</LI><LI>In areas sheltered from rainwater, though, gypsum crust remains. When combined with particulate matter from the atmosphere, gypsum creates the so called black crust.</LI></UL>
						The composition of the black crust is heterogeneous and varies accordingly to the environment. The crust found in big cities is rich in metals and carbonaceous particles, while the crust of stones found in a less urban environment is often characterized by the presence of microorganisms.
						Although pollution has actually diminished during the last decades, the black crust problem is not been solved yet. It is thereby clear how the black crust is a prominent issue in conservation and restoration, an issue that every remediator has to address.</P><DIV class="internalLink" style="text-align: center; width: 800px;" id="method"><A href="javascript:void(0);"><H3>READ ABOUT OUR METHOD</H3></A></DIV></DIV><DIV class="chapter method"><H2 style="text-align: center;">OUR METHOD</H2><P>Our method exploits an engineered <I>E. coli</I> to reduce aerobically the sulphuric gypsum layer in a controlled and modulated manner. In 2000 the Keasling group engineered bacteria to reduce sulphate aerobically and precipitate metals from water taking advantage of the production of hydrogen sulfide as one of the bioproducts of sulphur reduction (3). We took inspiration from this work and developed two new BioBrick devices directed to overproduce cysteine and then convert it to sulphide with the ultimate goal of dissolving the sulphate layer deposited on the blackened stones.</P><H3>Who are the players?</H3><UL><LI>CysE  is a Serine acetyltransferase that mediates conversion of L-serine to a precursor of L-cysteine in <I>E. coli</I>. More precisely we have used a mutant CysE (M256I) that has enhanced activity in that its enzymatic activity is less sensitive to feedback inhibition by cysteine (4).</LI></UL><UL><LI>CysDes  is a Cysteine Desulfhydrase, first isolated in <I>Treponema denticola</I>, an anaerobic organism involved in periodontal diseases. It is an aminotransferase that converts cysteine into pyruvate, ammonia, and hydrogen sulphide (5).</LI></UL><H3>How do we control the expression of our devices?</H3><P>The expression of CysE is controlled by an arabinose inducible cassette (our Part <A href="http://partsregistry.org/Part:BBa_K731201">BBa_K731201</A>). Our composite Device (<A href="http://partsregistry.org/Part:BBa_K731030">BBa_K731030</A>) is composed by araC–pBad, a strong RBS and the gene encoding M256I CysE (our Part <A href="http://partsregistry.org/Part:BBa_K731010">BBa_K731010</A>).
						The pBAD promoter is activated in the presence of L-arabinose. L-arabinose binds to the AraC protein and inactivates its inhibitory function, allowing the RNA polymerase to recognize the pBAD promoter and start the transcription of CysE.
						</P><P>CysDes was placed downstream of an IPTG inducible cassette (our Part <A href="http://partsregistry.org/Part:BBa_K731300">BBa_K731300</A>). The expression of CysDes is therefore controlled by a system composed of: LacI (repressor protein) and a lacIq promoter followed by a strong hybrid promoter (Ptac), a Lac Operator and a strong RBS. Addition of IPTG inactivates the LacI repressor, thus allowing the expression of our gene of interest CysDes. </P><P>The possibility of controlling simultaneously the expression of our two devices (i.e. by adding arabinose and/or IPTG) makes our system more <B>SAFE</B>, <B>CONTROLLABLE</B> and <B>MODULAR</B>.</P><H3>How do we test the expression of our enzymes?</H3><P>We used the Gibson assembly method to create two sfGFP-tagged devices that were useful reporter to verify the expression of our genes.</P><DIV class="internalLink" style="text-align: center; width: 800px;" id="results"><A href="javascript:void(0);"><H3>GO TO OUR RESULTS</H3></A></DIV></DIV><DIV class="chapter results"><H2 style="text-align: center;">RESULTS</H2><P>All the experiments herein reported were done transforming our parts into <I>E. coli</I> strain NEB10β grown in MOPS medium unless specified. In addition to the parts submitted to the Registry of Standard Parts we also have subcloned some of our parts into medium/low copy vectors with different antibiotic resistances, to transform the two plasmids simultaneously. For each experiment it will be specified the exact part used.
						</P><P>Which of our bacteria do you want to read about?</P><DIV id="cysE"><H2 style="margin-top: 30px;">CysE</H2><H3>Pinker is better!</H3><P>CysE is our keyplayer in the destruction of the black crust. It catalyses the production of O-acetyl-L-serine, a precursor for cysteine.  When cysteine is produced, <I>E. coli</I> needs more sulfate, resulting in the assimilation of the black crust gypsum layer. 
							Therefore production of cysteine means reduction of sulfate.</P><P>To assess if our devices producing CysE function in vivo we have measured the concentration of cysteine produced and secreted in the medium with ninhydrin, a small organic compound that reacts with different alpha-aminoacids at different pH (6). In the presence of cysteine at low pH the solution turns pink within a few minutes. The reaction between cysteine and ninhydrin at low pH also gives a characteristic UV-VIS spectrum with an absorbance maximum peak at 560 nm.</P><P class="caption"><I>Figure 1: NEB10β cells with an without Part <A href="http://partsregistry.org/Part:BBa_K731030">BBa_K7310130</A> (i.e. CysE) operated in the low copy vector pSB3C5 were grown in MOPS, supplemented with K2SO4 and glucose as carbon source, and induced with 5 mM Arabinose. After overnight induction a 0.5 mL aliquot was taken from each sample.  To each aliquot it was added 0.5 mL of glacial acetic acid and 0.5 mL of Ninhidrin reagent prepared as described in our protocol page. Panel A: NEB10β cells before induction (left) and after overnight induction (right). Panel B: Cells transformed with <A href="http://partsregistry.org/Part:BBa_K731030">BBA_K731030</A> before induction (left) and after overnight induction (right).</I></P><P>Cells transformed with CysE, grown in MOPS and induced with 5 mM arabinose were assayed with ninhydrin along with empty NEB10β cells, and left to grow overnight. Samples taken before induction served as controls.  The data show that when induced with arabinose cells expressing CysE produce a significant amount of cysteine that can be quantified by comparison with a calibration curve built with known concentration of cysteine. We estimated that after one night of induction in MOPS supplemented with glucose and K<SUB>2</SUB>SO<SUB>4</SUB> it was produced 0.025 mM cysteine. The experiment was repeated in triplicates. A light pink colour was observed  in a few cases with empty cells grown overnight, indicating that endogenous cysteine is produced in small amounts.</P><P class="caption"><I>Figure 2: Quantification of cysteine production. From left to right: NEB10β before induction, NEB10β after induction and overnight growth, cells transformed with CysE (<A href="http://partsregistry.org/Part:BBa_K731030">BBa_K731030</A>) before induction and after induction and overnight growth.</I></P><DIV class="trigger"><A class="trigger-cyse" href="javascript:void(0);">Read more about CysE</A></DIV><DIV class="details"><H3>MOPS with glucose or glycerol?</H3><P>After a deep analysis of CysE expression levels, toxicity, and production of cysteine we determined that the perfect medium to our purposes was MOPS supplemented with glucose.  Below it is illustrated the road that brought us to this conclusion.</P><P>To better control protein expression, we used two different recipes of the MOPS minimal medium: one with glucose as the carbon source (MOPS B), the other with glycerol (MOPS A).</P><P>Glucose is an inhibitor of the araC-pBAD promoter, while glycerol does not have a comparable inhibitory mechanism. We have tested the effect of these two different MOPS medium on cells growth and cysteine production with the intention of determining the best growth protocol for our future applications on the statues.</P><P>To this purpose, we have first analyzed the amount of protein produced in the two media by looking at fluorescence intensity using our sfGFP tagged reporter of CysE (<A href="http://partsregistry.org/wiki/index.php?title=Part:BBa_K731040">BBa_K731040</A>).</P><P class="caption"><I>Figure 3: Fluorescence intensity of CysE-sfGFP (<A href="http://partsregistry.org/Part:BBa_K731040">BBa_K731040</A>) after induction with arabinose (ara). MOPS supplemented with glycerol (A) is shown in blue, MOPS supplemented with glucose (B) is shown in yellow.</I></P><P>Protein levels obtained in the medium supplemented with glycerol were significantly higher than those obtained in MOPS supplemented with glucose because of the inhibitory mechanism of glucose.</P><P>In glycerol, thus, cells are producing a lot of CysE. To assess if and how CysE production is affecting cell growth, we perfomed serial dilutions at 8h after induction.</P><P>However, to our surprise cysteine production behaved differently. We tested cysteine production of cells samples uninduced and induced overnight in the two different media.</P><P class="caption"><I>Figure 4: Ninhydrin assay on NEB10β expressing Part <A href="http://partsregistry.org/Part:BBa_K731030">BBa_K731030</A>. A representative replicate is shown. From left to right samples are: CysE in MOPS A before induction, CysE in MOPS A not induced grown overnight, CysE in MOPS A induced grown overnight, CysE in MOPS B before induction, CysE in MOPS B not induced  grown overnight, CysE in MOPS B induced  grown overnight. MOPS supplemented with glycerol (A), MOPS supplemented with glucose (B).</I></P><P>Surprisingly cysteine production was higher in the uninduced sample grown in glycerol. In glucose the situation was the opposite and we had high cysteine production in the induced sample.  This effect was contrasting with protein expression levels that we observed.  We then hypothesized that high expression of CysE could have a toxic effect on the cells, which could explain why in glycerol we observed a lower cysteine concentration in the induced sample.</P><P>To address this problem we have analyzed the toxicity effect of CysE production by counting the number of colonies survived after 8 hours.</P><P class="caption"><I>Figure 5: Toxicity effect of CysE by serial dilution. NEB10β cells transformed with Part <A href="http://partsregistry.org/Part:BBa_K731030">BBa_K731030</A> were grown in MOPS supplemented with glycerol (60 mM, MOPS A) and glucose (30 mM, MOPS B). After 4 and 8 h of induction with arabinose a 500 μl aliquot of cells was taken to make serial dilution on LB agar plates. Cell number was calculated with the following formula: Cells/mL = (# colonies counted)*(dilution factor)/(mL of culture plated).</I></P><P>The serial dilution test confirmed our theory: when cells are induced in glycerol they show very high expression levels, but few cells can cope with this pressure. Even if each of these few cells is actually producing much CysE, the cell doesn’t have the capabilities to sustain such high cysteine production, which is not achieved in a single step reaction (i.e. a bottleneck is reached with other enzymes).
								When cells in glycerol are not induced, CysE production is decreased, allowing more cells to survive and producing higher amounts of cysteine.
								In glucose the situation is the opposite: we have many cells, but low expression due to the glucose inhibitory effect on the araCp-BAD promoter. All these cells are producing low amounts of CysE, leaving the cells more “healthy” and able to produce high cysteine concentrations.</P><P>To summarize we decided that MOPS supplemented with glucose was the best compromise between protein expression and cysteine production, which ultimately means more sulfate reduced!</P></DIV></DIV><DIV id="cysdes"><H2 style="margin-top: 30px;">CysDes</H2><H3>Green means H<SUB>2</SUB>S production!</H3><P>CysDes is the second player in our system. It transforms cysteine into ammonia, pyruvate and sulfidric acid (H<SUB>2</SUB>S).
							H<SUB>2</SUB>S production is the fundamental proof that intracellular cysteine being accumulated has been used by the enzyme and that our part operates correctly.</P><P>We assayed and quantified the production of H<SUB>2</SUB>S with a colorimetric assay that takes advantage of the production of methylene blue when M N,N-dimethyl-p-phenylenediamine sulfate reacts with produced H<SUB>2</SUB>S (7).</P><P class="caption"><I>Figure 6: The test was performed on a cell lysate obtained as described in our protocol page. Briefly, to each cell lysate it was added 0.1 mM cysteine and the samples were placed at 37°C for 1 hour. After the incubation at 37 °C 0.1 mL of a 0.02 M N,N-dimethyl-p-phenylenediamine sulfate solution and 0.1 mL of a 0.3 M FeCl3 solution were added to the lysate. The development of the color was immediate and it could be observed both by naked eye and by UV-VIS at 670 nm. The assay was performed in triplicates using <A href="http://partsregistry.org/Part:BBa_K731400">BBa_K731400</A> in the low copy vector pSB4K5. From left to right: CysDes -IPTG + cysteine, -IPTG, + IPTG + cysteine, + IPTG; Empty NEB10β cells with cysteine and - cysteine.</I></P><P>Upon induction with IPTG cells expressing CysDes produced H<SUB>2</SUB>S depending on the addition of cysteine. Uninduced cells still produced a small amount of H<SUB>2</SUB>S that can be explained by the small basal expression that was observed (see results on BBa_K731480). A small basal expression could in fact be enough to produce significant concentrations of the enzyme for catalytic activity. Empty cells instead did not show any production of H<SUB>2</SUB>S as expected.</P><P>H<SUB>2</SUB>S production is dependent on cysteine availability. We decided to assess the efficiency of the enzyme by quantifying more precisely the amount of H<SUB>2</SUB>S formed using a calibration curve made with different concentrations of Na<SUB>2</SUB>S. We estimated that the concentration of H<SUB>2</SUB>S produced in the presence of 0.1 mM of cysteine is about 0.1 mM after 2 hours from the induction.</P><P class="caption"><I>Figure 7: Quantification of H<SUB>2</SUB>S produced by Part <A href="http://partsregistry.org/Part:BBa_K731400">BBa_K731400</A>. Panel A: Intensity of Absorbance at 670 nm at different concentration of cysteine. Panel B: Concentration of H<SUB>2</SUB>S produced calculated based on a standard curve made with Na<SUB>2</SUB>S.</I></P><DIV class="trigger"><A href="javascript:void(0);">Read more about the enzymatic activity of CysDes</A></DIV><DIV class="details"><P>We assessed the production of H<SUB>2</SUB>S by several qualitative and more quantitative assays:</P><UL><LI>Gas Chromatography</LI><LI>Lead acetate strips</LI><LI>Copper precipitation</LI><LI>Triple sugar iron test</LI></UL><H3>Rotten eggs are dangerous?</H3><P>Up until now we used colorimetric assays and standard curves to prove and quantify the presence of H<SUB>2</SUB>S. However, hydrogen sulfide is a (toxic and poisonous) gas and as such it can be detected by Gas chromatography. To confirm the presence and estimate the concentration of H<SUB>2</SUB>S produced we asked the help of an expert in gas Chromatography in the physics department. Damiano Avi came one day to the lab with his portable gas chromatographer (MICROGC A3000 Agilent) equipped with a 50C Poraplot U column for the detection of H<SUB>2</SUB>S. Damiano keeps his H<SUB>2</SUB>S stock in 10 L champagne bottles, because they can hold really high pressure!</P><P class="caption"><I>Figure 8:  PANEL A: 50 mL of cells were grown in LB in a 250 mL sterile bottle with a modified screw cap that allows to connect the bottle directly to the instrument. PANEL B: After 4 hours of induction, with 0.1 mM IPTG, the bottle was attached to a portable gas chromatographer (MICROGC A3000 Agilent). PANEL C: H<SUB>2</SUB>S formation was qualitatively assessed by exposure to lead acetate test strips for few seconds. PANEL D: Gas Chromatography analysis of NEB10β cells with and without <A href="http://partsregistry.org/Part:BBa_K731480">BBa_K731480</A>. Measurements were taken 3 times at intervals of 2 minutes. A calibration curve was done with H<SUB>2</SUB>S.</I></P><P>We estimated that a sealed 50 mL culture of <I>E. coli</I> carrying our BBa_K731480 after 4 hours of induction accumulated between 20 and 30 ppm of H<SUB>2</SUB>S in a 250 mL bottle. This of course is the concentration of H<SUB>2</SUB>S accumulated in the sealed bottle. If the bottle were open, such a high value would not have been reached.  
									As you can read in our <A href="https://2012.igem.org/Team:UNITN-Trento/H2S">H<SUB>2</SUB>S section</A> these concentrations are not an immediate risk for human health. Nevertheless is absolutely not advisable to inhale (or smell) such quantities for a prolonged time, and for this reason we took all the possible precautions to work in the safest conditions (see more in our safety section).</P><H3>Measuring H<SUB>2</SUB>S production by an indirect method: free copper concentration in the solution
									</H3><P>In this second approach we exploited the formation of complexes between sulfide and metal ions. Briefly, adding a copper salt to the medium of an induced culture we observed a decrease of the free copper concentration as function of time, due to the precipitation of copper sulfide. The free copper concentration was measured after adding bathocuproinedisulfonic acid (BCS) (a chemical which turns orange when it reacts with Cu<SUP>2+</SUP> and measuring the absorbance by UV-VIS at 483nm. We tried to obtain an estimation of the free copper using a standard curve made with different concentrations of Cu<SUP>2+</SUP>. This test was done using only once using Part <A href="http://partsregistry.org/Part:BBa_K731400">BBa_K731400</A> and therefore quantitative data cannot be extrapolated surely.</P><P class="caption"><I>Figure 9: Copper precipitation by CysDes. Cells transformed with Part <A href="http://partsregistry.org/Part:BBa_K731400">BBa_K731400</A> (i.e. CysDes) were grown in LB in the presence of 2 mM Copper sulfate.To the supernatant it was added 1 µl of a 100 mM solution of Bathocuproinedisulfonic (BCS) reagent and 1 µl of a 1 M ascorbate. Absorbance was measured at 483 nm. Panel A: Uninduced cells, Panel B: Induced cells. Absorbance at 483 nm is shown in blue, optical density is shown in red. Read more details about this test in our protocol page.</I></P><H3>More metals to be precipitated: the TSI plates</H3><P>We learnt that sulfide reacting with metal ions produces a change in color. In this test we grew cells in triple sugar iron medium and observed the formation of black spots due to the precipitation of iron sulfide. The process was a bit slow, but after a week we observed clearly the “H<SUB>2</SUB>S” formation, suggesting that when lacking oxygen our CysDes producing bacteria do not work as efficiently, as in aerobic conditions.</P><P class="caption"><I>Figure 10: Cells transformed with <A href="http://partsregistry.org/Part:BBa_K731480">BBa_K731480</A> were induced with 0.1 mM IPTG and a 5 µl aliquot was placed inside the iron containing medium with a tip. Cells were left at 37°C for 1 week. After 48 hours the first black spots started appearing. The maximum production of FeS was reached after 1 week.</I></P></DIV><DIV><DIV class="trigger"><A href="javascript:void(0);">Read more about the expression of CysDes</A></DIV><DIV class="details"><P>CysDes expression was confirmed by fluorescence in two different MOPS media using Part <A href="http://partsregistry.org/Part:BBa_K731480">BBa_K731480</A>, which carries a sfGFP tag. Fluorescence intensity was measured to evaluate protein expression upon induction with IPTG. The expression levels observed were higher in MOPS when glycerol was used as the carbon source and improved when the cells were grown in the presence of cysteine. </P><P class="caption"><I>Figure 11: Cysteine effect on CysDes expression. NEB10β cells transformed with Part <A href="http://partsregistry.org/Part:BBa_K731480">BBa_K731480</A> were grown and induced with 0.1 mM IPTG in two different MOPS medium: MOPS A, 60 mM glycerol; MOPS B, 30 mM glucose. To assay the effect of cysteine the experiment was also performed in the presence and in the absence of 1 mM L-cysteine (cys). Fluorescence was measured after 4 and 8 hours of induction.</I></P><P>To analyze the possible toxicity effect of CysDes we have calculated the number of cells survived after 4 and 8 hours of induction by serial dilution.	</P><P class="caption"><I>Figure 12: Toxicity of CysDes. At 4 and 8 hours of induction a 500 µl sample was taken from the uninduced and the induced culture and used to make serial dilution . A 200 µl aliquot of each serial dilution was plated on LB agar and placed overnight at 37°C. The following day the number of colonies from each plate was counted. Cell number was calculated with the following formula: Cells/mL = (# colonies counted)*(dilution factor)/(mL of culture plated). Conditions used are MOPS with 60 mM glycerol (A) and MOPS with 30 mM glucose (B), both in the presence of 0.1 mM cysteine. Experiments were done with Part <A href="http://partsregistry.org/Part:BBa_K731400">BBa_K731400</A>.</I></P><P>We concluded that MOPS supplemented with glucose produced enough CysDes, in concentration sufficient to reduce the amount of cysteine produced by our other Part <A href="http://partsregistry.org/Part:BBa_K731030">BBa_K731030</A> (i.e. CysE).</P><P>At this point we are ready to work on our complete system.</P></DIV></DIV></DIV><DIV id="both"><H2 style="margin-top: 30px;">CysE + CysDes</H2><H3>Putting together the pathway: no more pink!</H3><P>CysE and CysDes sinergically worked to reduce sulfate aerobically in our engineered <I>E. coli</I>.</P><P class="caption"><I>Figure 13: Our engineered <I>E.coli</I> aerobically reduces sulfate and transform the produced cysteine into hydrogen sulphide. Panel A, B,  and C: NEB10b cells grown in MOPS supplemented with 30 mM glucose and 4 mM potassium sulfate were induced with 5 mM arabinose and/or 0.1 mM IPTG. The production of cysteine was measured with a colorimetric assay that uses ninhydrin, a small organic molecule that reacts with cysteine at low pH. Empty NEB10b cells do not produce cysteine (A), <A href="http://partsregistry.org/Part:BBa_K731030">BBa_K731030</A> when induced produce cysteine (B), cells co-transformed with <A href="http://partsregistry.org/Part:BBa_K731030">BBa_K731030</A> and <A href="http://partsregistry.org/Part:BBa_K731400">BBa_K731400</A> when co-induced do not show residual cysteine in the media, indicating that cysteine is being transformed into hydrogen sulphide (C).</I></P><P>When NEB10β cells were cotransformed with Parts <A href="http://partsregistry.org/Part:BBa_K731030">BBa_K731030</A> and <A href="http://partsregistry.org/Part:BBa_K731400">BBa_K731400</A> and induced both with arabinose and IPTG no cysteine was left in the media, as an indication that CysDes (i.e. <A href="http://partsregistry.org/Part:BBa_K731400">BBa_K731400</A>) is able to transform the accumulated cysteine into hydrogen sulfide. As a control, we tested the production of cysteine when CysDes was not induced. The experiment was done in duplicates.</P><P>This observation confirmed that our combined system is able to assimilate sulfate, reduce it and form H<SUB>2</SUB>S.</P><DIV class="trigger"><A href="javascript:void(0);">Read more about our data on CysE + CyDes</A></DIV><DIV class="details"><H3>Cotransformation</H3><P>In order to be able to co-express CysE and CysDes we subcloned Parts <A href="http://partsregistry.org/Part:BBa_K731030">BBa_K731030</A>, <A href="http://partsregistry.org/Part:BBa_K731400">BBa_K731400</A> and <A href="http://partsregistry.org/Part:BBa_K731480">BBa_K731480</A> in the low copy vectors pSB3C5 and pSB4K5. </P><P>We tested protein expression levels of CysE upon induction of CysDes by measuring fluorescence intensity of CysE-sfGFP (i.e. <A href="http://partsregistry.org/Part:BBa_K731040">BBa_K731040</A>) produced when CysDes (<A href="http://partsregistry.org/Part:BBa_K731400">BBa_K731400</A>) was uninduced or induced with IPTG.</P><P class="caption"><I>Figure 14: NEB10β cells cotransformed with Parts <A href="http://partsregistry.org/Part:BBa_K731040">BBa_K731040</A> and <A href="http://partsregistry.org/Part:BBa_K731400">BBa_K731400</A> were grown in LB and induced with 5 mM arabinose and different concentrations of IPTG. Time points were taken at 4 hours and 8 hours of expression to measure fluorescence intensity, as an indication of CysE expression levels.</I></P><P>The data show that CysE induction seems to be decreased significantly upon CysDes expression. For example, at 0.02 mM of IPTG the expression seems to be 3 fold less when compared with the levels of protein produced in the absence of IPTG.</P><P>To optimize CysE production we tried to induce the expression of the two proteins at different times. In the figure below it is shown the production of CysE, measured by fluorescence intensity, when CysDes was induced 4 hours later with 0.1 mM of IPTG. </P><P class="caption"><I>Figure 15: NEB10β cells co-transformed with Parts <A href="http://partsregistry.org/Part:BBa_K731040">BBa_K731040</A> and <A href="http://partsregistry.org/Part:BBa_K731400">BBa_K731400</A> were grown in LB and induced with 5 mM arabinose when it was reached an OD of 0.6 (time zero) and with 0.1 mM of IPTG 4 hours later. Time points were taken every hour to measure fluorescence intensity, as an indication of CysE expression levels.</I></P><P>The data showed that we reached a good compromise in CysE expression levels over time when delaying the induction of CysDes.</P><P>Based on our findings we decided that the conditions to be used on future applications on the statues would be:
									<UL><LI>MOPS supplemented with 30 mM of glucose</LI><LI>expression of CysE with 5 mM arabinose</LI><LI>expression of CysDes with 0.1 mM of IPTG after 4 hours from the induction with arabinose.</LI></UL></P><H3>Co-culture</H3><P>Unfortunately we were not able to look in the co-transformed cells at the expression of both proteins because we lacked fluorescent tagged parts with antibiotic compatibility.</P><P>Therefore we monitored CysE and CysDes expression levels over time in co-cultured NEB10β. </P><P>We established the following co-culture tests:</P><P>A) CysE-sfGFP + CysDes using Parts <A href="http://partsregistry.org/Part:BBa_K731040">BBa_K731040</A> and <A href="http://partsregistry.org/Part:BBa_K731400">BBa_K731400</A>.
								B) CysE + CysDes-sfGFP using Parts <A href="http://partsregistry.org/Part:BBa_K731030">BBa_K731030</A> and <A href="http://partsregistry.org/Part:BBa_K731480">BBa_K731480</A>.</P><P>For co-culture A we used the following growth conditions:</P><P>+5 mM arabinose, - 0.1 mM IPTG
								+5 mM arabinose, + 0.1 mM IPTG</P><P>For co-culture B we used the following growth conditions:</P><P>–5 mM arabinose, + 0.1 mM IPTG
								+5 mM arabinose, + 0.1 mM IPTG</P><P>and checked fluorescence intensity of the sfGFP tagged protein being used as reporter. For example, we were able to look at CysE expression levels when CysDes was uninduced or induced with IPTG (co-culture A) and viceversa (co-culture B).</P><P class="caption"><I>Figure 16: Top Panel CysE-sfGFP levels monitored by fluorescence intensity levels when CysDes is uninduced (+Arabinose -IPTG) and induced (+Arabinose +IPTG). Bottom Panel: CysDes-sfGFP monitored by fluorescence intensity levels when CysE was uninduced (-Arabinose +IPTG) and induced (+Arabinose +IPTG).</I></P><P>The data showed that induction of both CysE and CysDes had a synergistic effect on the expression levels of both enzymes. However, this is in contrast with what we saw with the co-transformed cells. This is possibly due to a much higher energetical request of the cell when the two enzymes are expressed simultaneously. This theory finds support on the fact the optical density levels recorded were significantly decreased when both genes were induced (about 30% less, data not shown).</P></DIV></DIV><DIV class="internalLink" style="text-align: center; width: 800px;" id="application"><A href="javascript:void(0);"><H3>FIND OUT HOW WE APPLY OUR BACTERIA TO THE STONES</H3></A></DIV></DIV><DIV class="chapter application"><H2 style="text-align: center;">APPLICATION</H2><DIV id="batterioni"><P>In this page you can find about our <A class="scroll-crustaway" href="javascript:void(0);">application results</A>, <A class="scroll-application" href="javascript:void(0);">application method</A> and our <A class="scroll-Crustonator" href="javascript:void(0);">DIY black crust</A>!</P></DIV><H3>Crust is definitely Away!</H3><P>Preliminary evidence show that our engineered <I>E. coli</I> was capable of removing the black crust.</P><P class="caption"><I>Figure 17: Black crust removal from Carrara marble. Panel A: before treatment. Panel B: after 3 cycles of treatment with NEB10β expressing Parts <A href="http://partsregistry.org/Part:BBa_K731030">BBa_K731030</A> and <A href="http://partsregistry.org/Part:BBa_K731400">BBa_K731400</A>. The black crust was reproduced in the laboratory with our Crustonator using a Carrara marble samples of the dimensions of 10 cm x 10 cm. The bacteria were applied on the surface with our Jelly-MOPS for 3 cycles of 12 hours each. After the 3 cycles the gel was removed and the stone was gently tapped with blotting paper and then rinsed with hot water. If you want to know more about the Crustonator and Jelly-MOPS you can find them at the end of this section.
								</I></P><P>We first tried our bacteria to clean a variety of marble samples. Although the results were very promising it was unclear to us if what we thought was black crust was actually black crust. Therefore, we decided to reproduce the black crust in the laboratory with an acid rain simulator (read more in the Crustonator section).</P><P>We applied to a black crust sample reproduced in the laboratory bacteria transformed with CysE and CysDes (Parts <A href="http://partsregistry.org/Part:BBa_K731030">BBa_K731030</A> and <A href="http://partsregistry.org/Part:BBa_K731400">BBa_K731400</A>), in addition to control cells that did not carry the enzymes.
						The treatment lasted for 3 cycles of 12 hours each.  Unfortunately it was difficult to apply the bacteria in one specific spot because the surface of the stone was not flat and the dimensions of the piece were too small to allow for a precise separate spotting. For this reason we currently do not have a good negative control.</P><P>Although the removal of the crust was not complete, it was possible to observe a significant change by eye both in the color and the appearance of the surface of the treated stone.  It should be kept in mind that the marble pieces exposed in the Crustonator showed a significant thick layer of gypsum crystals, probably because of the aggressive exposure to the acidic atmosphere.  With the black crust formed when marble is exposed to the environment,  the situation could be completely different.  For instance, the gypsum layer is usually less thick, making thus the removal quicker and more efficient.</P><P>At this point we had a much cleaner stone and a good feeling that our system was working. However, science is not based upon sensations. To have a less qualitatively analysis we asked the help of Flavio Deflorian, professor in material engineering, that visited our lab a few weeks earlier. Prof. Deflorian opened his lab to us and his staff supported us to collect Scanning Electron Microscopy (SEM) images, to analyze the absence of calcium sulfate crystals, as a confirmation that gypsum was removed by our bacteria.</P><P class="caption"><I>Figure 18: Removal of the black crust from a synthetically generated gypsum layer: Scanning Electron Microscopy analysis of the black crust before and after bacterial treatment. Untreated synthetic crust (A), SEM analysis of material scraped from untreated synthetic crust at 500x (B) and 1000x magnification (C). Synthetic crust treated with NEB10β cells expressing CysE and CysDes (Parts <A href="http://partsregistry.org/Part:BBa_K731201">BBa_K731030</A> and <A href="http://partsregistry.org/Part:BBa_K731400">BBa_K731400</A>) (D); SEM analysis of material scraped from treated synthetic crust at 500x (E) and 1000x magnification (F).
								</I></P><P>The SEM analysis showed changes in the crystal morphology of the stone surface. Before the treatment, calcium sulphate crystals were large (200-300 μm) and well structured, while after the application of our bacteria CaSO4 was still present on the surface but crystals were smaller (20-30 μm) and showed a loss in crystalline organization. We did not find any big and well structured crystal in the sample took after the treatment. Changes in light reflection were even noticeable by eye.</P><P>Our preliminary data demonstrate the feasibility of our method and the possibility of using engineered bacteria to clean the black crust from calcareous stones. However, more tests and controls should be performed to address the following points:</P><I><B>in vitro</B></I><UL style="margin-top: 5px;"><LI>Optimize the combined expression of CysE and CysDes.</LI><LI>Place CysDes behind a different promoter to avoid basal expression.</LI></UL><I><B>Applications on the statue</B></I><UL style="margin-top: 5px;"><LI>Optimize method of application</LI><LI>Have clear negative controls</LI><LI>Use different kinds of black crust and marble</LI></UL><P>We hope that another iGEM team will consider this as good starting point for a possible future project!</P><DIV class="trigger"><A href="javascript:void(0);">Read more about how to apply the bacteria on the stones</A></DIV><DIV class="details"><H3>Our application method.</H3><P>After characterizing our system in vitro we were ready to apply the bacteria on our collection of stone samples.</P><P>The first thing we needed to address is how to apply the bacteria on the stones. We needed a support that would allow for both bacterial survival and adhesion on the surface of the statue. 
								We developed a soft gel with a modified MOPS recipe and agar (jelly-MOPS), and set up a protocol that allowed the gel to stay in place, to remain wet and to keep bacteria alive through the night. This is all we needed, all the rest was left to the bacteria!</P><P>We created an agar matrix with separate wells on the stone surface to allow an easy application of the bacteria. This method simplified the procedure for multiple sequential applications in the same spot.</P><P class="caption"><I>Figure 19: Application steps on a stone sample. Preparation of a scaffold on the surface of the stone (Panel A).
										Then the matrix gel is poured and left to solidify directly on the surface. When solid, wells are cut out from the matrix (Panel B).
										Bacteria resuspended in Jelly-MOPS are placed in the wells (Panel C), covered with rice paper wetted in PBS. After three cycles of 12 hours, the gel and the matrix are removed (Panel D).</I></P><P>The bacteria were grown in MOPS with glucose and induced with arabinose +/- IPTG, depending on the parts being expressed. After 4 hours of induction the bacteria were pelleted and resuspended in jelly-MOPS and a 1-2 cm layer of bacterial gel was added inside each well.  The matrix was then covered with rice paper wet in PBS to protect and keep hydrated the bacterial gel.</P><P>The stone was subjected to 3 application cycles of 12 hours with freshly induced bacteria. At the end of the three applications both the gel and the matrix were removed, and the stone surface was gently tamped with blotting paper. Then hot water was poured gently on the stone to easily remove the remaining part of the gel.</P></DIV><DIV><DIV class="trigger"><A href="javascript:void(0);">Read more about how to apply the Crustonator</A></DIV><DIV class="details"><H3>An home made acid rain simulator.</H3><P>As we started collecting samples of dirty marble we realized that it was important to test our bacteria on an homogenous surface of black crust that was surely composed of a gyspsum layer.</P><P>Inspired by the work of a group of researchers at the University of Belfast an at the University of Oxford (M. Gomez-Heras, B.J. Smith and H.A. Viles ) we decided to build our own acid rain simulation chamber. For a detailed protocol on how to build your own Crustonator you can check our protocol section.</P><P>By using our labmade acid rain simulator we were able to successfully recreate the black crust on:</P><UL><LI>2 small white marble pieces from Carrara (10x10 cm)</LI><LI>2 small yellow marble pieces from Vicenza (10x10 cm)</LI></UL><P>The pieces were immersed partially in distilled water and subjected to three 72 hours cycles of exposure to a saturated atmosphere of sulfourus acid. At the beginning of each cycle a fine powder of charcoal was injected into the chamber with compressed air.</P><P>After the treatment the box was opened and we happily observed the formation of a gypsum layer in top of the stones and on the surface of the water. A small amount of the crust was scratched and looked under the microscope to confirm the presence of gypsum crystals.</P><P style="width: 600px; margin: 10px auto;"><I>Figure 20: The Crustonator: A small laboratory device was built to simulate the acid rain and pollutants on the stones. In a tightly sealed plexiglass box, connected to a separate chamber containg charcoal, were placed two marble pieces from Carrara and two yellow marble pieces from Vicenza (A). The calcareous stones were partially immersed in water (B) and subjected to a saturated atmosphere of SO<SUB>3</SUB> mixed with small particles of charcoal that were sprayed every 72 hours in the chamber (C). After 216 hours the box was opened and a black crust was observed on the top layer of the marble (D).</I></P><P>The blackened stones are ready to be cleaned by our bacteria!</P></DIV></DIV><DIV><H3>Project References</H3><OL><LI>F.Cappitelli, L.Toniolo, A.Sansonetti, D.Gulotta, G.Ranalli, E.Zanardini, C.Sorlini (2007)  Appl. Environ. Microbiol., 7317, 5671-5675.</LI><LI>F.Cappitelli, E.Zanardini, G.Ranalli, E.Mello, D.Daffonchio, C.Sorlini (2006) Improved Methodology for Bioremoval of Black Crusts on Historical Stone Artworks by Use of Sulfate-Reducing Bacteria, ) Appl. Environ. Microbiol., May, Vol.72, No.5, pp.3733–3737</LI><LI>C. L. Wang, P. D. Maratukulam, A. M. Lum, D. S. Clark, and J. D. Keasling.  (2000)  Metabolic engineering of an aerobic sulfate reduction pathway and its application to precipitation of cadmium on the cell surface.  Appl. Environ. Microbiol.  66:4497-4502.</LI><LI>D.Denk, A.Böck (1987) L-cysteine biosynthesis in Escherichia coli: nucleotide sequence and expression of the serine acetyltransferase (cysE) gene from the wild-type and cysteine-excreting mutant, J of Gen. Microbiol., 133, pp.515–525</LI><LI>L.Chu, A.Burgum, D.Kolodrubetz, S.C.Holt (1995) The 46-kilodalton-hemolysin gene from Treponema denticola encodes a novel hemolysin homologous to aminotransferases, Infection and Immunity, November 1995, Vol.63, No.11, pp.44–48</LI><LI>M.K.Gaitonde (1967) A spectrophotometric method for the direct determination of cysteine in the presence of other naturally occurring amino acids, Biochem. J., Vol.104, pp.627–633</LI><LI>Chu, L., J. L. Ebersole, G. P. Kurzban, and S. C. Holt. 1997.  Infect. Immun. 65:3231–3238.</LI><LI>M.Gomez-Heras, B.J.Smith, H.A.Viles, Proceedings, 11th International Congress on Deterioration and Conservation of Stone, Torun., Nicolaus Copernicus University Press, pp.105-112</LI></OL></DIV></DIV></DIV></SECTION></DIV></DIV><DIV id="footer"><DIV id="credits"><DIV id="mw"><UL><LI><A href="https://2012.igem.org">iGEM Main Page</A></LI><LI><A href="https://2012.igem.org/wiki/index.php?title=Team:UNITN-Trento/Project/CrustAway&amp;action=edit">Edit</A></LI><LI><A href="https://2012.igem.org/wiki/index.php?title=Team:UNITN-Trento/Project/CrustAway&amp;action=history">History</A></LI><LI><A href="javascript:void(0);">Teams</A></LI><LI><A href="https://2012.igem.org/Special:RecentChanges">Recent Changes</A></LI><LI><A href="https://2012.igem.org/Special:SpecialPages">Special Pages</A></LI><LI><A href="https://2012.igem.org/Special:Preferences">My Preferences</A></LI></UL></DIV><P>Design by Jason Fontana. Powered by <A href="http://www.mediawiki.org/wiki/MediaWiki">MediaWiki</A>. This website is under the <A href="http://creativecommons.org/licenses/by/3.0/">CC Attribution licence</A>.</P></DIV></DIV><DIV class="printfooter">
Retrieved from &quot;<A href="http://2012.igem.org/Team:UNITN-Trento/Project/CrustAway">http://2012.igem.org/Team:UNITN-Trento/Project/CrustAway</A>&quot;</DIV><DIV id="footer-box" class="noprint"><DIV id="footer"><UL id="f-list"><LI id="t-recentchanges"><A href="/Special:RecentChanges" title="Recent changes">Recent changes</A></LI><LI id="t-whatlinkshere"><A href="/Special:WhatLinksHere/Team:UNITN-Trento/Project/CrustAway" title="List of all wiki pages that link here [j]" accesskey="j">What links here</A></LI><LI id="t-recentchangeslinked"><A href="/Special:RecentChangesLinked/Team:UNITN-Trento/Project/CrustAway" title="Recent changes in pages linked from this page [k]" accesskey="k">Related changes</A></LI><LI id="t-specialpages"><A href="/Special:SpecialPages" title="List of all special pages [q]" accesskey="q">Special pages</A></LI><LI><A href="/Special:Preferences">My preferences</A></LI></UL></DIV><DIV id="footer"><UL id="f-list"><LI id="t-print"><A href="/wiki/index.php?title=Team:UNITN-Trento/Project/CrustAway&amp;printable=yes" title="Printable version of this page [p]" accesskey="p">Printable version</A></LI><LI id="t-permalink"><A href="/wiki/index.php?title=Team:UNITN-Trento/Project/CrustAway&amp;oldid=291243" title="Permanent link to this revision of the page">Permanent link</A></LI><LI id="privacy"><A href="/2012.igem.org:Privacy_policy" title="2012.igem.org:Privacy policy">Privacy policy</A></LI><LI id="disclaimer"><A href="/2012.igem.org:General_disclaimer" title="2012.igem.org:General disclaimer">Disclaimers</A></LI></UL></DIV></DIV></BODY></HTML>