<HTML xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en" dir="ltr">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki  ltr ns-0 ns-subject page-Team_UNITN-Trento_Notebook"><DIV id="globalWrapper"><DIV id="top-section"><DIV id="p-logo"><A href="/Main_Page" title="Main Page">&quot;
	    </A></DIV><DIV id="menubar" class="left-menu noprint"><UL><LI class="selected"><A href="/Team:UNITN-Trento/Notebook">Page               </A></LI><LI class="new"><A href="/wiki/index.php?title=Talk:Team:UNITN-Trento/Notebook&amp;action=edit&amp;redlink=1">Discussion               </A></LI><LI><A href="/wiki/index.php?title=Team:UNITN-Trento/Notebook&amp;action=edit">View source               </A></LI><LI><A href="/wiki/index.php?title=Team:UNITN-Trento/Notebook&amp;action=history">History               </A></LI><LI style="color:#808080;cursor:default">teams</LI></UL></DIV><DIV class="right-menu noprint" id="menubar"><UL><LI id="pt-login"><A href="/wiki/index.php?title=Special:UserLogin&amp;returnto=Team:UNITN-Trento/Notebook" title="You are encouraged to log in; however, it is not mandatory [o]" accesskey="o">Log in</A></LI></UL></DIV><DIV id="search-controls" class="noprint"><FORM action="/Special:Search" id="searchform"> </FORM></DIV></DIV><DIV id="content"><H1 class="firstHeading">Team:UNITN-Trento/Notebook</H1><DIV id="bodyContent"><H3 id="siteSub" class="noprint">From 2012.igem.org</H3><P><TITLE>Notebook</TITLE></P><H2 style="font-family: 'Open Sans'; line-height: 40px; color:#ffae00; padding:0.5em; font-size:30pt !important; text-align: center; width: 900px; margin: 100px auto;">It appears you have JavaScript disabled. Our Wiki needs it for you to enjoy the full experience. Please enable Javascript and refresh the page.</H2><DIV id="header"><DIV id="topNav"><UL><LI><A href="https://2012.igem.org/Team:UNITN-Trento">Home</A></LI><LI class="folder"><A href="javascript:void(0);">Project</A></LI><LI><A href="https://2012.igem.org/Team:UNITN-Trento/Project/CrustAway">Crust Away</A></LI><LI><A href="https://2012.igem.org/Team:UNITN-Trento/Project/Terminators">Terminator 5</A></LI><LI><A href="https://2012.igem.org/Team:UNITN-Trento/Attributions">Attributions</A></LI></UL></DIV><LI><A href="https://2012.igem.org/Team:UNITN-Trento/DataPage">Data Page</A></LI><LI class="folder"><A href="javascript:void(0);">Parts</A></LI><LI><A href="https://2012.igem.org/Team:UNITN-Trento/Parts">Our Parts</A></LI><LI><A href="http://partsregistry.org/cgi/partsdb/pgroup.cgi?pgroup=iGEM2012&amp;group=UNITN-Trento">Registry Section</A></LI></DIV><LI><A href="https://2012.igem.org/Team:UNITN-Trento/Achievements">Judging</A></LI><LI class="folder"><A href="javascript:void(0);">Team</A></LI><LI><A href="https://2012.igem.org/Team:UNITN-Trento/Team">The Team</A></LI><LI><A href="https://2012.igem.org/Team:UNITN-Trento/Team/Supervisors">Super- &amp; Ad- visors</A></LI><LI><A href="https://2012.igem.org/Team:UNITN-Trento/Friends">Our Friends</A></LI><LI><A href="https://igem.org/Team.cgi?id=731">Team Profile</A></LI><LI><A href="https://2012.igem.org/Team:UNITN-Trento/Trento">Where We Are</A></LI><LI><A href="https://2012.igem.org/Team:UNITN-Trento/Press">Press</A></LI></DIV><LI class="folder"><A href="javascript:void(0);">Partners</A></LI><LI><A href="https://2012.igem.org/Team:UNITN-Trento/Sponsors">Sponsors</A></LI></DIV><LI class="folder activeLink"><A href="javascript:void(0);">Lab Notes</A></LI><LI><A href="https://2012.igem.org/Team:UNITN-Trento/Notebook">Notebook</A></LI><LI><A href="https://2012.igem.org/Team:UNITN-Trento/Meetings">Meetings</A></LI><LI><A href="https://2012.igem.org/Team:UNITN-Trento/Protocols">Protocols</A></LI><LI><A href="https://2012.igem.org/Team:UNITN-Trento/Tools">Tools</A></LI></DIV><LI class="folder"><A href="javascript:void(0);">Safety</A></LI><LI><A href="https://2012.igem.org/Team:UNITN-Trento/Safety">Safety</A></LI><LI><A href="https://2012.igem.org/Team:UNITN-Trento/H2S">H<SUB>2</SUB>S</A></LI><LI><A href="https://2012.igem.org/Team:UNITN-Trento/Outreach">Outreach</A></LI><LI><A href="https://2012.igem.org/Team:UNITN-Trento/Art">Art&amp;Science</A></LI><DIV id="logo"><DIV class="wrapper"><H1><A href="https://2012.igem.org/Team:UNITN-Trento" style="font-family: 'Bree Serif';"><SPAN>i</SPAN>GEM Trento 2012</A></H1></DIV></DIV><DIV id="canvas" class="home"><DIV id="wrapper" style="padding: 20px 40px;"><DIV id="full"><DIV class="page"><H2 style="text-align: center;">Our Lab Book</H2><DIV class="faces" style="margin-bottom: 55px;"><H3 style="">Filter by:</H3><UL><LI id="jfdr"><A class="one" href="javascript:void(0);"><H3 class="bBlue">Black Crust CysE</H3></A></LI><LI id="atfg"><A class="two" href="javascript:void(0);"><H3 class="bGreen">Black Crust CysDes</H3></A></LI><LI id="ggad"><A class="three" href="javascript:void(0);"><H3 class="bOrange">Terminators</H3></A></LI></UL></DIV><DIV id="sticky" class="weekNav"><UL style="margin-left: 30px;"><LI>Jump to Week:</LI><LI><A class="w1" href="javascript:void(0);">1</A></LI><LI><A class="w2" href="javascript:void(0);">2</A></LI><LI><A class="w3" href="javascript:void(0);">3</A></LI><LI><A class="w4" href="javascript:void(0);">4</A></LI><LI><A class="w5" href="javascript:void(0);">5</A></LI><LI><A class="w6" href="javascript:void(0);">6</A></LI><LI><A class="w7" href="javascript:void(0);">7</A></LI><LI><A class="w8" href="javascript:void(0);">8</A></LI><LI><A class="w9" href="javascript:void(0);">9</A></LI><LI><A class="w10" href="javascript:void(0);">10</A></LI><LI><A class="w11" href="javascript:void(0);">11</A></LI><LI><A class="w12" href="javascript:void(0);">12</A></LI><LI><A class="w13" href="javascript:void(0);">13</A></LI><LI><A class="w14" href="javascript:void(0);">14</A></LI><LI><A class="w15" href="javascript:void(0);">15</A></LI><LI><A class="w16" href="javascript:void(0);">16</A></LI><LI><A class="w17" href="javascript:void(0);">17</A></LI><LI><A class="w18" href="javascript:void(0);">18</A></LI></UL><UL id="filter"><LI>Filter by:</LI><LI style="color: #50aae4;" id="jfdr-sticky"><A href="javascript:void(0);">Black Crust CysE</A></LI><LI style="color:#67f853;" id="atfg-sticky"><A href="javascript:void(0);">Black Crust CysDes</A></LI><LI id="ggad-sticky"><A href="javascript:void(0);">Terminators</A></LI></UL></DIV><H3 style="margin-top: 40px;">Week 1 (05/21 – 05/27)</H3><P>We start to meet in the lab. We have been assigned a big lab in Malga, the
					building that hosts all the teaching labs. It’s the biggest one.
					We’re starting to design the constructs and primers and prepare LB
					medium for culture. We also make a list of all we have at our disposal and
					what we need to order. </P><P class="jfdr hlBlue">And so it begins.
					I've been designing constructs and primers for the last few days, but finally it's time to enter the lab and get things started! I'm going to work on cloning and characterization of araC-pBAD, the promoting system we are going to use with CysE, the gene that will allow us to clean statues. I will extract the araC-pBAD promoter from part BBa_I0500 from the registry and then put a GFP under his control to study it.
					This is the first week of wet lab! We spent few days organizing spaces, equipment, supplies, making competent cells, buffers stocks, medium for coltures. We also started extracting the first parts from the distribution kit, but i had no luck with part BBa_I0500, containing araC-pBAD promoter. At first i thought i was terrible with the transformation protocol, but looking better in the registry i saw that the quality control of the part was horrible, so i planned on extracting it from another well, and also from the 2011 distribution kit that Sheref has in his lab.
					</P><H3>Week 2 (05/28 – 06/03)</H3><P class="jfdr hlBlue">During the weekend I and Guzz go statue hunting: we stop by a few
					marble/statues stores and ask for a few samples. We find kind people who
					let us take what we want, so we now have what we need to test the
					parts we will hopefully build.
					I continue to research on the statues project, while developing a Python
					script together with Guzz to detect illegal sites from sequences returning their position if they exist, or adding prefix and suffix instead.</P><P class="jfdr hlBlue">I still had problems on extracting the parts from the distribution kit, so i designed primers to PCR-extract it from a composite part.
					On day 10 i managed to transform a composite part, the one that i will use for the PCR-extraction. The sample was called “HOPE”, becouse it took so long for me to extract the part, and i was starting to get really disappointed. Getting the transformation done had a huge moral boost.
					This week has been the one of the first times: i had my first minipreps, (and i obviously messed them up!) and also my first PCR, where i extracted araC-pBAD both from the composite part and from the <I>E. coli</I> genome supplied by Sheref's Lab.
					</P><P class="atfg hlGreen">I purified many plasmids and PCR products, in particular LacI from pET-21 plasmid  and AraCpBAD from <I>E. coli</I> genome and from the Registry.</P><H3>Week 3 (06/04 – 06/10)</H3><P class="jfdr hlBlue">The primers to PCR-extract CysE from the <I>E. coli</I> genome arrive, so I can
					now start with it.
					My first electrophoresis, obviously, fails. I let it run too much. We now
					know that these gels run much faster than the one used during classes.
					In my next attempt I manage to see the product on the gel . I proceed with the restriction of CysE and pSB1C3 (the submission vector) with E and P, and then ligation and transformation in Nova Blue cells.
					The transformation succeeds, giving both red and white colonies. The
					screening of a white one confirms my first construct, and our first Part!</P><P class="jfdr hlBlue">The week started great, i got my first part ligated, transformed and confirmed by screening.
					I was going to CIBio, the research centre to learn hot to use FACS and to help Giacomo with the measurements, but we forgot controls!
					My araC-pBAD part and Jason's CysE are ready, so we started collaborating to put the two together and form our first awesome device! Jason took care of it while me and Giacomo tried again with FACS. Unfortunately, the machine was moved and broke during the transport. We were forced to start the characterization with the fluorimeter. After a couple of tries me and Jason were successful in creating the first device!
					</P><P class="atfg hlGreen">I selected a couple of possible terminators to test from the registry (B0010 and B0017), and I purified some of these colonies.
					I continued with the purification of many overnight cultures and PCR products, in particular LacI from pET–21 plasmid and araC-pBAD from <I>E. coli</I> genome and from the Registry.
					Giaco brought a delicious cake!
					We designed some primers, tried to extract some parts from registry, such as some standard vectors.
					Starting designing the terminator construct: we decided to use the Roberta Lentini’s plasmid RL0024, with the fluorescent proteins Cherry and Venus. This will allow us to calculate terminator’s efficiency as a ratio between the peak intensity of Venus and Cherry, avoiding problems of various variables that could influence raw measure.</P><DIV class="afterBookmark">To submit it we must mutate two illegal sites, XbaI and PstI, and insert prefix-suffix in the linker between the two proteins (take a look at our final construct).We also would like to test if the termination efficiency changes with a different promoter/ RNA polymerase; as the chosen vector already contains T7promoter, we decided to replace it with Ptac, thus making our second platform. 
					To take measurements we’ll use the <I>E. coli</I> strain BL21pLysS, an expression strain with both polymerases.
					A familiarizing week also with protocols and various tools.</DIV><H3>Week 4 (06/11 – 06/17)</H3><P class="jfdr hlBlue">I can now apply a mutagenesis PCR to CysE to obtain M256I CysE. The gel confirms that the PCR worked, which is then ligated into a digested pSB1C3 and transformed again in Nova Blue cells. I’m at the Univesity Olimpics when I have to take the inoculi from the thermoshaker, so Andrea kindly does it for me.
					The screening, again, confirms my second Part K731010 . At this point I have the building blocks to make my Composite Parts, using araC-pBAD as promoter, which is made by Ross.
					I and Ross then decide to start with araCpBad: we digest CysE with E and S, araCpBad with X and P and pSB1C3 with E and P. Ligation and transformation into Nova follow.</P><P class="jfdr hlBlue">I began focusing my attention of the characterization of araC-pBAD. I chose two different parts to do that, BBa_E0840 ad BBa_E0240. The two BioBricks are composed of a strong or medium RBS followed by the same gene, GFPmut3b. I will test promoter behaviour in presence or absence of glucose, differences in expression with the two different RBS and eventually differences when using a high copy or low copy plasmid. I waited for sequencing results to start amplifying all the parts I need. I also had time to have a BBQ at Giacomo's house for my birthday and to give an exam the next day!
					
					Me and Giacomo stayed in the lab all the weekend to advance the project. We did lots of things from PCR to mutate illegal sites, PCR to insert BB prefix and suffix on the plasmid we are creating, some transformations and a lot of screenings! We also had lots of fun on saturday evening, and the all night long partying m,ade us really sleepy and tired the next day.</P><P class="atfg hlGreen">I went on constructing my first BioBrick, which is K731300: it consists of LacI-LacIq into pSB1C3. Everything went right, and on 13th June I announce the birth of my first construct, after attending the endless run of a giant 2% agarose gel (I waited for about three hours…!) .
					In the rest of the week I kept on isolating and purifying some low-copy plasmids that could have been useful in the future.</P><P class="ggad hlOrange">Transformation of some Roberta’s plasmids: just Venus (RL009), just Cherry (RL007) and our template RL0024.
					I also tried my first mutagenesis, prefix-suffix insertion, but it failed…we got a multiple band gel, meaning that the primer didn’t annealed properly. After some attempts we decided to order another primer suffix-reverse, because we thought that one annealed aspecifically. This new primer was designed for a slightly different template: RL0030A, that’s identical to RL0024A except for a 30bp longer linker between the two proteins’ genes.
					Amplification of some plasmid backbones from registry. </P><H3>Week 5 (06/18 – 06/24)</H3><P class="jfdr hlBlue">The construct me and Ross made last week is purified but not yet quantified and screened; anyway we want
					to send it for sequencing before the Post Office closes (along with CysE
					and M256I CysE) or the samples will be stuck during the weekend (and we don’t
					want that!) We estimate a concentration of about and blindfoldingly
					prepare the samples. Cristina rushes and between traffic and parking is in
					front of the Post Office 5 minutes before closure. In the meantime, sadly,
					the construct is NOT confirmed by screening. Doh.</P><P class="jfdr hlBlue">This week i tried to do a lot of things: 
					
					- PCR to amplify parts i will use for cloning: FAILED.
					- Other PCR: FAILED.
					- Have a BBQ with all 3 years of biotechnology, professors and researchers without drinking too much and without staying up late: FAILED.
					After all this failures I decided to do a 3-assembly, but again I failed the minipreps of the parts I needed.
					 
					</P><P class="atfg hlGreen">I prepared all the samples for sequencing, and I continued purifying various plasmids and helped out other people in the lab. And I also studied for the exam of Developmental Biology.</P><P class="ggad hlOrange">This week Giaco and Ross went to CiBIO (research center) to try FACS for the first time: they took RL007, RL009, RL0024 in BL21, and got expected results.
					I failed another insertion PCR; also with the new primer on RL0030A the gel kept on showing multiple bands.
					This week Giaco and Ross had a honey weekend here in lab, all alone…îScience weekend has begunî [battle journal citation]. I don’t know exactly what happened, but anyway Giaco got the first mutation: the PstI site . </P><H3>Week 6 (06/25 – 07/01)</H3><P class="jfdr hlBlue">We start over, this time inserting CysE after araCpBad in pSB1C3, cutting
					the vector (which contains araCpBad) with S and P and then insert (CysE)
					with X and P. Last time we used the registry protocol; This time we used higher quantities of DNA. For the insert we used PCR products rather than a plasmid.
					The screening gives a variety of results. I realize we’re very unlucky, again: screening with E and P, which gives vector and insert bands, results in fragments of the same length. Cristina finds there’s a BamHI site near the end of araCpBad, so I can screen with BamHI and PstI to confirm the cloning . I also confirm by PCR using Prefix and Suffix primers, just to be sure . Both are good, meaning I have my third Part.
					This week I also begin to work with the mutated version of the composite
					part: araCpBad-M256I CysE. I try with a modified version of the 3A method, the one I didn’t manage to do well with Ross last time. This time, instead of simply digesting and ligating the biobrick , I PCR out CysE* and araCpBad and then digested them separately.Afterward I have ligated them into pSB1C3 in one shot.
					The screening of the transformation (with many many red colonies) is not
					good, again. The 3A assembly seems my nemesis.</P><P class="jfdr hlBlue">In the weekend I digested a whole miniprep for every part I needed use. Quantifications also were good, so Everything was ready for monday!
					Me Giaco and Guzz had and exam this week so we took a day off for studying, but at the end of the week I got the two main constructs done!</P><P class="atfg hlGreen">This week our gene synthesized by Genscript has arrived! I will be working all day long from now. From now on I will have to sleep in the lab.
					I immediately started to try to amplify the gene by PCR, and after 12 attempts I finally succeeded in it . In the weekend I purified and digested all the parts that I would have used to create my new parts, K731400 and K731450.</P><P class="atfg hlGreen">On the 27th June I will start working on the constructs assigned to me. Up until now, I've helped out others. 
					I will build and characterize our new lac operator-containing plasmids. I will then pass the information I gathered to Andrea and help her testing her parts.
					These are the constucts I will create:
					lacI_lacIq_pTAC_lacO in pSB1C3,
					lacI_lacIq_pTAC_lacO in pSB4K5,
					lacI_lacIq_pTAC_lacO_GFP in pSB1C3,
					lacI_lacIq_pTAC_lacO_GFP in pSB4K5.
					The gene we ordered is pTAC_lacO_RBS_CysDes into pUC57 and flanked by prefix and suffix. Since I need to extract the lac operator alone, along with Cristina I designed the primer lacoperator_rv, the reverse complement of lacO followed by the suffix reverse complement.
					Since the ordered gene needs to be amplified, I’ve transformed it into DH5α.
					I’ve had some troubles amplifying the 139bp construct, since the designed primer annealed with better affinity to the 30bp-long suffix. The results were two bands, one more intense corresponding to the whole ordered construct (unwanted), and one less intense of the desired length. After a couple of tries with different conditions and buffers, I’ve been able to selectively create and amplify the wanted construct . The key is extension time: shortening it to 10s prevented the amplification of the long construct.
					Numerous attempts of amplifying the ordered gene by both me and Andrea resulted in low yields; it is strange, since pUC57 is an extremely high-copy plasmid…</P><P class="ggad hlOrange">This week we tried to insert T7 in pSB1C3, but from the sequencing we found that it didn’t work.
					We also tried to extract the right band of the RL0024*(we designed with a star each mutation made on the construct)+prefix-suffix from the gel, but the attempt failed: the concentration of the final sample was almost nothing.
					Discovered that the prefix-fwd annealed also at the end of Venus (the two proteins have N-term homology), thus we ordered ANOTHER primer, this time the prefix-fwd one…
					When it arrived, I tried again this insertion, both with RL0024 and 30 and sent for sequencing .
					Giaco obtained the second mutation, the XbaI site . </P><H3>Week 7 (07/02 – 07/08)</H3><P class="jfdr hlBlue">What I can do now is transferring araCpBad-CysE from the submission
					vector (which is high copy) to a medium copy vector; it will be needed
					when I have to characterize it (it seems light years far now). We choose
					pSB3C5, digest it with E and P along with the PCR made to screen the
					construct in pSB1C3.
					I carry on with the ligation and transformation in DH5a, that the screening
					with both EcoRI and PstI and BamHI and PstI confirms . And another Part goes to the stock box.
					For M256I CysE, I decide to go on with 3 different approaches
					at the same time: inserting M256I CysE after araCpBad in its vector (this approach worked with the WT version), making new overnight cultures from the failed 3A Method of the previous week (you never know) and making a mutagenesis PCR on the confirmed WT Composite Part. Ironically, the one that doesn’t fail the screening is the second one (the one I made with the least hope). LOL.
					I make a PCR from the construct (in pSB1C3) with BioBrick primers  and digest with E and P, to insert it into pSB3C5 digested with the same
					enzymes.</P><P class="jfdr hlBlue">I also wanted to see if “HOPE” version of pBAD was functioning different from the <I>E. coli</I>'s genome one so I started making duplicates of the constructs with the other version of the promoter. I also used this week for defining the best conditions and protocol for the characterization.</P><P class="atfg hlGreen">I did not start the week in the best way, because I won the “King of the lab” title already from Monday… But even if I made some mistakes at the beginning of the week I managed to create my two new constructs by the end of the week! Here it is the gel that confirms that I won my battle:
					
					In the weekend I started the preparation of MOPS medium, that we are going to use in the next weeks to start to characterize our constructs; and I also began to make another new construct, K731600.
					</P><P class="atfg hlGreen">K731500 has been created. I use Andrea’s part K731300 and the lacOp part amplified by PCR from the gene we received from Genescript. It's hard to confirm it by electrophoresis since Andrea's and mine constructs differ by 0.15kbp.
					</P><P class="ggad hlOrange">Sequencing confirmed the correct mutation of both illegal sites.
					We remade the insertion PCR once again, using the new primer, but then we discovered that I made ANOTHER mistake: I forgot to exclude from the primer itself the EcoRI site that was in the linker between the two proteins… We had to order ANOTHER primer to be used in a insertion-deletion-point mutation PCR!!!
					We also tried to digest the construct with EcoRI and then ligate it, to see if maybe in this way we could eliminate the double EcoRI site.</P><H3>Week 8 (07/09 – 07/15)</H3><P class="jfdr hlBlue">To finish with the mutated construct I transform it into DH5a and confirm it
					by screening . I choose 3 colonies to send for sequencing, one of which has the vector band lower than the others (as a negative control).
					And now it’s time to start thinking about the characterization of the construct I
					made. Ross is currently testing the promoter expressing GFP, so his
					results will serve as a guideline for my experiments.
					Since I can’t actually see if the protein is expressed in real time, I use
					Gibson’s Assembly to attach a sfGFP at the end of M256I CysE directly in the
					medium copy vector. I’ll transfer it in the submission one afterwards, to let
					me test it first. The PCR fails.
					However, this doesn’t mean I can’t start to characterize: I transform araCpBad-CysE WT (I’ll call it 1025 from now on) in NEB10b, our chosen cell line for expression. I make my first real characterization experiment: I grow cells overnight in LB, dilute them 1:50 and, when they reach an O.D. of about 0.6, split and transfer them in minimal medium (MOPS) containing respectively Glycerol and Glucose as carbon source. At O.D. 0.8 I split them again, and induce
					one culture with arabinose (0.5mM). I take a sample from each culture before induction, after 4h of growth, and after 8h. Every hour after induction I measure the O.D, to get growth.</P><P class="jfdr hlBlue">More testing! I am getting myself used to MOPS medium, where the growth is slow, so I was trying to see if I could standardize a protocol for bacterial growth in the two versions of the medium we were using, the one with glycerol amd the one with glucose as carbon source.</P><P class="atfg hlGreen">This was the week of the medium: MOPS had to be ready as soon as possible to start with the characterization of our constructs. And this fun work was given to me! …I spent about three days weighting salts and filtering solutions drop by drop…
					Anyway, in the meantime I also prepared the Molecular Biology exam and then had the first failure with my new construct K731600: the ligation products did not survive to the transformation, neither one good colony grew up. So I prepared a new digestion: but only at the end of the week I realized I have used an “unreliable” plasmid: the final concentration resulted horrible, so I needed to start again from the beginning amplifying a new pSB1C3 and digest it again.
					This week will be remembered also as “the week of the first contamination”: plates of all of us started to have weird halos, and from now on the contamination started to be a fellow traveler of us.</P><P class="atfg hlGreen">I confirmed my construct (K731500) by screening in a 2% agarose gel. I'm waiting for the sequencing results. I've been working on K731510, I will have the results next week.</P><P class="ggad hlOrange">We played a little with the fluorimeter, measuring BL21 coltures with just Venus or Cherry, our template with both proteins and another construct that has them in the opposite order (RL0020). The coltures were inducted with IPTG 0.1mM and we analysed fluorescence at different times of induction. We also tried to left them O/N in the refrigerator and we found out that the fluorescence greatly increased.
					We repeated this kind of experiments several times, testing also RL0024 with the double mutation of the illegal sites.
					One day Mansy came to see how it was going on and to make fun of us a little (that is a kind of relaxing hobby for him), and we discussed together what could be the best parameters for the future measures: </P><DIV class="afterBookmark"><TABLE><THEAD><TR><TH style="text-align:center;">Excitation</TH><TH style="text-align:right;">Emission</TH></TR></THEAD><TBODY><TR><TD style="text-align:left;">Cherry</TD><TD style="text-align:center;">587nm</TD><TD style="text-align:right;">615nm</TD></TR><TR><TD style="text-align:left;">Venus</TD><TD style="text-align:center;">485nm</TD><TD style="text-align:right;">528nm</TD></TR></TBODY></TABLE></DIV><P class="ggad hlOrange">Then we began to test some others features of the protocol: we tried to sonicate the samples, and took an enormous amount of data at different induction times and different lapse times after sonication. We repeated each experiment several times and with different colonies from the transformation time. We were completely adsorbed by this work but organize and analyse all that data was far worse than take them…</P><DIV class="afterBookmark">We defined the final parameters for fluorescence measurements:
					* 2–3h induction 0.5mM IPTG
					* brief sonication
					* proteases inhibitor treatment
					* O/N in refrigerator
					Of course we also tried another PCR to insert prefix-suffix and of course it failed.</DIV><H3>Week 9 (07/16 – 07/22)</H3><P class="jfdr hlBlue">The characterization obviously continues. I make changes in the protocol to find the optimal method; I try, for instance, to grow directly in MOPS A (i.e. glycerol) overnight as Keasling does. After a few tries I find that the original approach (growing overnight in LB and then changing the medium before induction) works best for me (my cells).
					The sequencing results for araCpBad-M256I CysE (which, again, I’m gonna call 1035 from now on) arrived, and confirm one colony with an high confidence (I sent 3, but something went wrong with the primers so we only have half of the sequencing, I send those two back).
					I can now transform 1035 in NEB10b and characterize it too. This is the version that should work best on the statues, thanks to its mutation.
					And this is the time when the Tragedy, as we call it, occurs. In a moment of confusion with lots to do, I see Guzz is transforming. I give him my cells and miniprep of the 1035 confirmed colony. And what Cristina feared most happened: Guzz was transforming ligations. When we transform ligations we transform all the product, so we put cells in the ligation eppie. He does that with my (only) miniprep of 1035 confirmed colony. Which means that I have none of it now. Please note I’m not blaming Guzz for this, my part was my responsibility: my part, my mistake.
					I decide to transform it and hope the cells will survive from the DNA excess and let me miniprep my plasmid back. I am now to transform another colony in NEB10b, I choose one of the two that was half sequenced and look promising.
					In the shadow of all this, a change in the conditions of the Gibson’s PCR (2°C higher for the annealing step and a 30s longer initial denaturation) proves successful . Along with Andrea, who’s making the same thing with her CysDes, I use the Master Mix for Gibson’s Assembly and transform it in the good old DH5a. The screening, again, is kind of ambiguous. It seems, though, that the GFP is not there.
					Coming back to characterization, I have to say this phase of the project is very tiring. I’ve to stay until late hours to get to 8h of induction. I also add serial dilutions to the mess: at 4h and 8h I plate serial dilutions of the cultures to estimate the number of cells, as measuring O.D is not a direct method.
					Growth curves results are accumulating, I now have triplicates and have to
					find the time to analyze them.
					By the end of the week minipreps of the Tragedy are screened, confirmed
					and sent for sequencing. I try with one that looks good on the gel in
					NEB10b.</P><P class="jfdr hlBlue">I finally optimized the protocol for the characterization! First step of growth (until OD reaches 0.3-0.4) is done in LB broth, then the colture is splitted, spinned and resuspended in MOPS, half with glycerol as carbon source, the other half with glucose. Growth continues in the new medium until OD reach 0.6, then cultures are induced. This shortens considerably the time from the fresh inoculation and the induction, and bought me some more sleep too!
					This week I fully tested K731260: I tested the effect of  different concentrations of arabinose, and expression over time in LB broth and MOPS (both with glycerol and glucose as carbon source).</P><P class="atfg hlGreen">This was very tiring. I had too many things to do, and as they were not enough I had also to prepare the Biochemistry exam. But as it usually happens in situations when you are tired a lot of mistakes are made, starting from my usual unlucky construct K731600: this time the digestion did not work, and I had to transform twice before screening. I also started to prepare a new construct (K731660) with a different low-copy plasmid, pSB6A1, but the digestion did not work.
					Fortunately I was lucky with another new construct, K731480: it contains LacI-CysDes+sfGFP into pSB1C3 (PCR gel photo 1 and 2), and I constructed this BioBrick following the Gibson assembly protocol. I have screend 6 colonies, only one was correct and its overnight culture was fluorescent too! Anyway the screening seemed ok , we had just to wait for the sequencing results.
					
					During this week I also started to characterize my construct K731450, or better, I should have started to characterize this construct: as I said at the beginning I was very tired, and I forgot to add the antibiotic to the medium; so I threw away the culture, I will start again next week. After the Biochemistry exam of course, and after my first weekend of relax and freedom.</P><P class="atfg hlGreen">I've confirmed by screening the construct I created the previous week. Also, sequencing results of K731500 are OK.
					I’ve tried to create K731520 and K731530, but the screening is a total disaster.
					I tried to characterize anyway those constructs, by choosing and amplifying the most promising colonies. While I was able to measure IPTG-dependent levels of fluorescence for K731530, the other one did not seem to contain GFP gene. Just half a success.
					I’ve been told that the chosen approach doesn’t work. Others tried it even before I started my job and failed… if only I knew.
					We’ve also been in Trieste to meet the only other italian team. It has been fun, but I had to stay in the lab until 3AM after that.
					I will amplify K731530 by PCR and try to insert it into pSB1C3 to obtain K731520 – a safe approach…</P><P class="ggad hlOrange">This week we mutated again RL** with the new primer using a three steps Phusion program, ligated and transformed it. The screening digestion was good 
					and sent for sequencing some samples.
					Wednesday and Thursday I stayed at home to study for an exam and Giaco had a lot of fun without me: he screened the effective presence of the terminators B0010–14–15–17 in the parts extracted from registry. He found out that there was nothing in the parts B0010 and B0015 but the vector pSB1C3. He tried also to amplify them by PCR but the gel showed nothing once again. On the other hand, he confirmed B0014 and 17.
					He got through some difficulties but finally he made t7terminator in pSB1C3 . Last time, when I tried, seemed all positive but the sequencing revealed just empty plasmids, so we were quite concerned about it…cross fingers…</P><DIV class="afterBookmark">I came back to the lab just in time to check the sequencing results of our final construct Cherry-Venus with both mutations and the prefix-suffix…. They were all perfect except for a really unexplainable mutation right in the EcoRI site of the prefix (that was part of the primers) common to all the colonies sent!!!!! It was unbelievable!!!! How could it be? The chromatogram wasnít so good, because we had to use a primer that was quite far from the interested region…
					Immediately we set up a screening digestion using EcoRI and XhoI (that has a site after Venus) but the gel showed just the band of the entire vector! Noooooo!
					We were stunned and desperate, and we didnít believed that: we made another screening digestion in the same way and left it O/N.
					Even the morning after the gel presented us the deep nothing… We spent the rest of the weekend thinking if it was worth to go on.</DIV><H3>Week 10 (07/23 – 07/29)</H3><P class="jfdr hlBlue">One realization makes my Monday bright: in the chaos of the Tradegy we didn’t realize that the colony sent for sequencing was the negative control: the vector seemed shorter, and since we can’t see it on the sequencing, I decide to drop the Tragedy colony. Haha.
					Sequencing results are back to complete the ones we only had half, telling me the colony I’m using is good.
					I have now enough data from growth curves and serial dilutions, so it’s time to see if the proteins are actually working, beside being produced (as growth curves and dilutions suggest.. we can’t tell it for sure until we make a protein gel or have the Gibson’s construct to test).
					What I’m doing is a Ninhydrin test on the samples after 4h, 8h or overnight
					to see if there’s Cysteine in the medium, the final product of CysE activity.
					We find an old paper by Gaitonde et al. describing the protocol.  In this test ninhydrin is solubilized in a solution of glacial acetic acid and concentrated hydrochloride (it takes 20 minutes even at max speed vortex). You add the reagent to a cell sample + glacial acetic acid at 1:1:1 ratio. When heated at 90 degrees Celsius the color changes to a pink/purple in about 2 minutes; the more intense the more Cysteine there is.
					My first try is after 4h of induction, using 1025, 1035 and empty NEB10b. After 10 minutes only one sample is intensely pink.
					From our understanding of the pathway, only 1035 in MOPS A (with glycerol) should be induced. We make this assumption and joy. But, turns out it was 1025 in MOPS B (with Glucose). We begin to ponder, and come to the conclusion that our assumption was obviously wrong. The induction in glycerol or the absence of glucose alone provides the cell the ability to highly express the protein, which may impair growth (as growth curves suggest). The higher OD in glucose, in
					contrast, could be the cause of the pink color of cultures in it: the basal transcription or the one not favored by glucose presence might be sufficient to produce enough Cys. I should investigate more and retry the test on the cells left to grow overnight.
					The results the day after seem confirming our hypothesis: the lighter pink is the induced culture in 1035, the other follow.
					Another attempt to get the Gibson’s to work: I try to add DpnI to the PCR products already made last time, then purify and use the Master Mix and so on as before.</P><P class="jfdr hlBlue">We were planning to run a modified TSI assay, so I adapted the normal TSI recipe to our MOPS medium. It should allow us to see the characteristic black spots of the sulphate reducing bacteria! I also transformed a plasmid with superfolder GFP under the control of pBAD promoter, to see if there is some basal expression since I wasn't able to see any signal at the fluorimeter with the normal GFPmut3b. With maximum sensibility I still had no significant signal: I'm pretty confident that if there is basal expression, it's really low!
					Turned out that lots of transformations are contaminated, so we took every precaution throwing away our competent cells: me and Giacomo headed to the research centre to make some new ones.</P><P class="atfg hlGreen">Contamination continues to persecute us: this is the plates’ turn. And of course I threw away my ligation product of K731660 again; as it was not a priority I decided to stop trying to construct this part, maybe I will start from the beginning in the future if I have had time.
					A good news from the sequencing arrived: my construct K731480 with GFP is right! But I had the confirmation just looking at the plate, actually.
					Anyway also bad news came from sequencing results, and they involved K731600 of course… I sent two samples to sequence, one contained an unknown gene and the other had CysE and not CysDes as insert…! I can't explain how it could have happened. It remains the fact that this construct is cursed, and I had to start again from the beginning.
					This week I finally started to characterize my constructs, and I began by making growth curves, serial dilutions and fluorescence measurements of K731480. I did every measure twice, and I tried with two different carbon sources: glycerol (that is in MOPS A) and glucose (in MOPS B). I won’t show you every chart and data I have, you can find them into the file “Characterization of CysDes”.</P><P class="atfg hlGreen">… which almost failed.
					I screened twelve colonies instead of the usual ten. The 11th and 12th seemed successful.  Also, bacterial pellet was fluorescent at the transilluminator, so I’m pretty confident. While I wait for sequencing results, I will try to characterize the constructs. 
					K731530 has been fully characterized. In order to do so, I’ve done duplicates in a single day, for a total of 144 measurements. It has been harsh, but necessary.</P><P class="ggad hlOrange">On Monday our advisors came back and even Mansy didn't believed that sequencing. He suggested we set up another (yes, another one) screening digestion; we run the gel and….magically the bands appeared!! 
					We`ve got it, finally!!! We failed so many times making it but eventually we succeeded!
					Such a good birthday present!
					In the rest of the week we amplified the correct construct and screened several colonies from the T7terminator-pSB1C3 ligation; they seemed all positive , so we sent some for sequencing.
					Amplifying the right Cherry-Venus construct we had, ironically the bad luck kept to persecute us and that construct: I failed several times to miniprep the overnight culture, and when Giacomo had mercy and helped me, I spoiled also his ones!!!
					Anyway we had lot of fun even in failing, and this success put us in a good mood.</P><DIV class="afterBookmark">We also made the mutagenesis PCR to replace T7promoter with Ptac in the vector… We made it with different conditions: with both 10 and 1ng of template and using a three steps program with two different annealing temperatures, 62°C and 67°C.
					We often found that this was the better solution, but not this time: we got multiple band product (and the major wasn’t the right one). Thus we made another PCR, two steps: this time the gel was much better  and we transformedÖ and sent for sequencing.
					We also made two terminator’s insertions: T7teminator in the T7promoter plasmid  and B0010, extracted by PCR from Bba_E0840 (PCR gel photo), into the same vector .
					The more shocking thing of this week, anyway, is that we and all the team tided and cleaned up all the lab!</DIV><H3>Week 11 (07/30 – 08/05)</H3><P class="jfdr hlBlue">After a couple of qualitative tries of the ninhydrin assay, it’s time to go quantitative: after the reaction has finished I jump to the spectrophotometer and read the spectrum of the samples. Cysteine has a characteristic peak at 560nm. The results correspond to the intensity of the pink.
					However, repetitions of the test on different days are not so consistent, as
					the peaks of the same sample type are at different heights. The ratio and
					“order” is maintained, though.
					This is also true while building a standard curve for Cysteine, made by
					providing a known quantity of it in the samples: peaks have doubled heights the second time, confirming this test is not so reliable. It still lets me know that the protein is doing its work, luckily. Even if it’s not exactly what we expected.
					One time I “steal” 1mL from Andrea’s overnight culture. She’s in charge of
					CysDes, the second protein of our pathway. She’s also in the process of
					characterizing it, so she always has cultures ready too. I use her version of
					the protein in a vector providing Chloramphenicol resistance, in order to be able to inoculate cells along with mine, as all my construct are in Chloramphenicol too. I induce the mix at the proper OD with both arabinose and IPTG (she has the LacI promoter). The ninhydrin test gives me promising results: samples are of the same color as control, which probably means Cysteine is actively being degraded by CysDes. We need to try a more systematic analysis.
					Ross, who’s almost finished his characterization, can start to support me in
					my efforts. It’s kind of relieving, I was struggling to keep up with everything.
					He takes over my Gibson’s construct, completing the screening and
					confirming the construct. He’ll also transfer it in pSB1C3 and characterize it.</P><P class="jfdr hlBlue">I'm starting the characterization of CysE-sfGFP, starting directly with MOPS mediums, i'm using always the same concentrations of arabinose. For the first time, I managed to see some expression also in MOPS with glucose, but it's really low. Still, there is something, and this means that inhibition by glucose levels is not perfect. This is something we can exploit, growth is faster than MOPS with glycerol and there is some expression, of course it's lower, but in our particolar case this levels of expression could be enough. Jason is starting with Ninhidryn tests so probably can tell us something more.
					I didn't want to get caught unprepared when the application on statue began, so i started some tests on agar concentration that will allow us to set up a good protocol. I tested MOPS with different agar concentrations (1-6 g/l ) and 6g/l turned out to be quite perfect!
					First tests on statues finally! I tried different approach, but the next day gel was dried out. I felt like I was light years away from the proper application, but I was sure that I could find the way to maintain the gel wet for at least 12 hours!
					Me and Guzz did some tests on co-cultures, we want to see if there is some significant interference between CysE and CysDes transformed strains. There was. Looks like inducing both genes enhances expression of both genes. We are not so sure of what are we looking at, becouse we have no way to discriminate the contribution of one particular population respect to the other.</P><P class="atfg hlGreen">I dedicated this week to characterize my constructs K731400 and K731450 as I did for k731480 (without fluorimetric measurements, of course); and I also started AGAIN to construct the cursed BioBrick K731600. I will win this battle!
					But even this time something went wrong, because contamination was still waiting for us; and so I threw away the first ligation product. Then I made one mistake plating onto the wrong antibiotic, throwing away the second ligation product. This forced me to be in the lab at 5am the next Monday, to recover lost time and not to slow all the work of others, whose samples should have been sent in the morning of 08/06 together with mine. This time I was very confident, I felt it was the right time; but my up early revealed totally useless, as you will read.</P><P class="atfg hlGreen">K731520 has been fully characterized. I’ve been able to reduce the delay by doubling the work load on the last days, and with this my job is done.
					I’ve spent a lovely Sunday at Caldonazzo’s lake with my girlfriend. It has been my first free day in three weeks: up to saturday, it has been 23 consecutive days of work.</P><P class="ggad hlOrange">We tested the restriction enzyme activity digesting our terminator platform with XhoI and E, P, S, X.
					This time the digestion was made in a total volume of 20µl to avoid influence of glycerol concentration.
					Also decided to look for a better way to run gels of such little fragments: finally we tried with a 1.5X agarose gel, with a little more EtBr and TBE instead of TAE . Wow!
					We were also too confident: we amplified and made an insertion on a Ptac plasmid not yet sequenced… Friday the results arrived and was really puzzling: we took a mutated sample but 6/8 were correct…such a bad luck!
					We started right away with the correct colony and made all the remaining clonings:
					* pSB1C3 with B0010, B0015 
					* T7 terminator platform with B0015 
					* Ptac terminator platform with T7, B0010, B0015 
					During the whole cloning phase Roberta and Laura gave us a great help: also this time we had B0015 (registry 2011) from them, since with couldn’t extract it from the 2012 reg.</P><DIV class="afterBookmark">This week we also had a major happening: CONTAMINATION!
					All the competent cells began to grow on every antibiotics and the whole team’ plates contained carpets of cells!
					Sincerely, I think I had a bug part in this mess…
					We threw away antibiotics, plates, cells, LB, transformation buffer, glycerol…everything!
					It was really hard to eradicate it, but finally we won and get back to normal…
					(T7 and B0010 in platform were confirmed by sequencing)</DIV><H3>Week 12 (08/06 – 08/12)</H3><P class="jfdr hlBlue">My tests continue with even more likeable results. Our interpretation seems to be correct. Ross also confirms my protein is being expressed by fluorescence.
					It’s time to start to amplify the confirmed Parts, giving priority to those in the Submission vector. We set August 15th as the deadline to finish reamplifying.
					To avoid confusion and mistakes, I transform a Part a day, so I always have to do overnight cultures or minipreps of different constructs on any day of this this week. By Friday I have one left to do, which I leave for next week.
					To use a relevant expression, this week we set a milestone of our project: we start to put cells on marble samples. I put induced and not induced cells of 1025, 1035 and empty grown for 4h in both MOPS A and B on a marble slide and cover it with a gel matrix Ross optimized. We leave it overnight. I also immerse half of a marble piece in induced 1035 cultures.
					Truth being told, for me it’s a bad week. My Mac is sick. I need to go home to know what’s wrong, but I think the hard drive failed or the RAM got corrupted. In the meantime Sheref lends me an old Mac from his lab.
					Too bad I can’t install nor Lightroom nor Espresso, the software I need for
					photos and to build the website. I still have my iPad, so I can easily keep tracking my experiments and write this very journal.
					On Thursday I have a meeting with Cristina and Sheref to update them on
					my results and decide if I need to gather more data to put on the presentation for the Jamboree. It’s not the case, we decide. I should focus on experiments on statues from now on. We also make an outline of what will be my thesis.
					The same evening I jot down a more detailed outline with OmniOutliner
					and I am happy with the result. Next month is gonna be a tough ride.</P><P class="atfg hlGreen">My presence in the lab at 4.45am in the morning of Monday was totally useless, because even this time my attempt of construction of K731600 miserably failed. I cannot stand this part anymore.
					But I did not give up, and on Wednesday I started again from the beginning: it is a personal challenge, I will make that construct by the end of the summer. This time I made every possible screening during the cloning, and when it was time for screening on Sunday…. The gel after the digestion was illegible, maybe the agarose had some problems and the samples did not run as usual; so I had to digest again and wait until Monday and use a new type of agarose for screening. What’s wrong with this construct??
					In the meantime I did some tests to verify and quantify the good working of my gene CysDes, that produced H2S starting from cysteine. I tried the tests suggested by Keasling in his work (the methylene blue test) and a test suggested by the iGEM Yale team 2010 (the BCS test). I obtained good results. However I will have to repeat the test: the plate from which my coltures came from was a bit old, and the gene did not work at the maximum of the possibilities. For this reason I will repeat both the tests next week.
					Test suggested by Yale:					
					In this test you add CuSO4 in the colture medium. If your bacteria produce H2S the Cu binds to sulphide and precipitate forming covellite; you can measure how much Cu is remained in the solution adding BCS (bathocuproinedisulfonic acid), which immediately reacts with Cu and turns orange. Then you record your results measuring the absorbance of your samples at 483nm: less the Cu in the solution more your bacteria worked and produce H2S. In parallel you can measure the OD of your cultures and see how it changes in relation at the production of H2S.
					In my case I obtained a partial positive result: how I was expecting in the not induced culture there was no change in the concentration of Cu, meaning that H2S was not being produced; in the induced culture it is clear a decrease of the concentration of Cu between 3 and 4 hours, and it is positive. Actually the decrease maybe should be clearer throughout the whole time of incubation, and for this reason the next week I will try again the same experiment with a fresh culture of bacteria.
					Test suggested by Keasling:
					In this test you verify the production of H2S observing the turn of color of your solution upon addition of N,M-dimethyl-p-phenilendiamine sulphate: if there is production of H2S it turns immediately from yellow to green/blue.
					Even in this case my results were partially: one colture worked perfectly (CysDes-GFP induced), while the other colture (CysDes K) worked a bit worst: you can see a little difference in the color between the induced and not induced (the induced is little more green), but the induced one is too yellow: I think that this is always due to the too old plate. I will repeat the test next week with a freshly transformed plate.
					
					
					</P><P class="atfg hlGreen">I exposed the gathered data to the group.
					Starting from this week, I will work on Biosafety and Future Application sections. It’s not a light task, but it’s something new, so I’m happy about it. I don’t want to hear about cloning anymore. 
					I will also work on modeling.
					Along with Ross, I also started to characterize co-cultures.</P><P class="ggad hlOrange">Eventually I reach also this week, that’s the present one.
					On Tuesday we started to measure some terminators. We measure the fluorescence of T7-platform, B0010-platform and platform and compare all the data. We did a preparatory step to find the best way to measure also absolute values of peaks, as we want to see if the terminator’s presence influence Cherry expression.
					The results were amazing: the coli terminator seemed to work much better than the T7, and both terminators seemed to stabilize Cherry, in addition to reduce Venus. This behaviour wasn’t expected and we were really surprised and pleased of that.</P><P class="ggad hlOrange">Moreover we discovered that our results were quite strong: even for non equal OD coltures and different lapse times after sonication they seemed constant, although with a higher variability.
					Also the sequencing results arrived, and confirmed Ptac platform with t7 and B0010 terminators. Also pSB1C3 with the same two terminators was confirmed. No construct with B0015 was right, but luckily they are less important. </P><H3>Week 13 (08/13 – 08/15)</H3><P class="atfg hlGreen">
					This week was short, because from 16th to 19th we decided to have a break and go on holiday! So I worked only on Monday and Tuesday before cleaning the lab on Wednesday and taking the train to return to Romagna.
					In these two days I had time to fail again with my cursed construct: I screened 6 samples by digestion and also by PCR, but all the agarose gels I made were awful and it seemed that ligation did not work (again). Cristina wisely decided not to send anything to sequence before holiday, we were not sure about the goodness of my samples; it would be better to make new overnight cultures on Tuesday and screen them once returned to work in the lab on next week. And this is what I did, fortunately. 
					In the meantime on Monday our friend Damiano came back to the lab to help me in measuring the H2S concentration produced by my bacteria with his portable gas-chromatography. We had excellent results: empty cells did not produce H2S as expected while cells with CysDes produced a concentration of about 18ppm. This is a good result, because we demonstrated that my construct works but it does not produce a concentration too high of H2S that could be dangerous. Excellent!</P><P class="atfg hlGreen">I’ve characterized co-transformations.
					I’ve done a few tests on marble pieces.
					With Ross, I’ve started building an acid rain simulator which should be able to reproduce the process of black crust formation.</P><P class="ggad hlOrange">In this week we tested the terminators’ behaviour with Ptac. The results were quite surprising: although the terminator efficiency calculated with Venus’ raw peaks seems similar to that found with T7 promoter, the stabilizing effect on Cherry was considerably lower.
					This discover was really puzzling for us: it seemed that the stabilizing effect depends on the polymerase used and we had no idea how to explain this phenomena. Maybe this is due to the polymerase processivity or quickness??</P><DIV class="afterBookmark">Also Sheref was surprised with this result, and he proposed us to try an in vitro reaction in the absence of ribonucleases, to see if this effect is due to these enzymes.
					We planned to do that next week with the help of Fabio Chizzolini in the Mansy Lab.
					With a really freudian disattention we mistook the antibiotic in the colture for the second repeat of the Ptac measures: we started our holidays a little bit before!!! Yes, we are going on holiday for 4 days!!!
					We spent the last day talking about project’s logo and name.</DIV><H3>Week 14 (08/20 – 08/26)</H3><P class="atfg hlGreen">Come back to work. It was hard after seeing sea, sand and friends for just a couple of days, but an hard job was waiting for us. We had a long meeting at the beginning of the week to sum up all the things we still had to do, and returned home a bit shocked after realizing the amount of work that was still waiting for us! But on Monday we had already completed a long and debated discussion: we had the name of the project, CRUST AWAY! And now we only missed the logo, but Nadia was working hard to realize it for us.
					This was another important week for me, except for the name of the project: I completed my construct K731600! After screening the 6 new colonies I inoculated before holidays the PCR revealed the presence of my insert in the g colony !
					Actually even the i colony seemed to have my insert, so I sent for sequencing both the colonies for sure. On Friday morning I was at the post office to send the samples with a great great great hope in my pocket.
					Meantime I repeated the methylene blue test for three or four times to test new samples and collected good data to show. I definitively verify that this test does not work with the lysates of the co-coltures CysE+CysDes; one of the hypothesis is that the cysteine production by CysE is too low to be used by CysDes to produce detectable quantities of H2S. Anyway I repeated the test only with my coltures of CysDes and empty cells with different concentrations of cysteine, I obtained very good results.</P><P class="ggad hlOrange">This week we took another repeat of the Ptac measurements (in reality they were two but I’ve lost the data of the second one).
					Moreover, we did the first in vitro reaction, using the Ptac just-linker-construct. The reaction was done with PURExpress kit and a spectrophotometer different from the one always used. We took the fluorescence data for 6h, then the day after we took another measurement at the usual instrument. We were helped by Fabio Chizzolini and Silvia, from the Mansy lab, doing this in vitro reaction. </P><H3>Week 15 (08/27 – 09/02)</H3><P class="jfdr hlBlue">I made the last tests on cysteine production, now I can focus on the website. Cristina and the rest of the team help me in choosing the layouts, and give me all the pages ready to be uploaded.</P><P class="atfg hlGreen">I finally ended with the tests on CysDes, it is time to collect and organize the results. So I passed most of the time at the PC, because the Registry pages of my parts had to be ready as soon as possible to be a model for the other guys of the team. But among this mess and infinite charts I found also the time to go to the cemetery to pick up a huge piece of tombstone. Fortunately Giaco helped me to pick it up with his car, because I would have had some problems in transporting 100kg-marble stone on my poor motorcycle....</P><P style="height: 110px;">This was one of the weeks that I had more fun: we have built a crustonator! After following the useful advices of Damiano and reading careful the instructions on a paper by M. Gomez-Heras, the acid-rain simulator was finally completed and ready to start to produce a lot of black crust. For the first attempt we put in the box four white marble pieces, we will wait now for 9 days to observe the black crust formation. </P><P style="height: 90px;" class="ggad hlOrange">Giaco on 29th August had the last, huge exam before graduating: Physiology! It means that he had to worry about that, and I went on testing the terminators efficiency. This week I did in vitro measurements with the pure system in the Mansy lab.</P><H3>Week 16 (09/03 – 09/09)</H3><P style="height: 150px;">Time is running out, so we started to apply our fantastic bacteria on the statues! We had finally collected all the stone pieces we need, so we started with the applications. We are using Anna’s egg beater to make the gel! Ross had a great idea on how to apply the bacteria. We made a jelly pudding type layer of agar. We pored the gel on the marble and wait that it solidifies. With a syringe we made holes in the gel and applied in each hole a mixture of induced bacteria and agar-MOPS.</P><P class="atfg hlGreen">We thought about interviewing some experts of the field, so I contacted Flavio Deflorian, an engineer who works in the field of restoration and material chemistry. He was very kind to have lunch with us and come to our lab, where we interviewed him with some questions and he gave us a lot of precious advices and information.
					In the meantime I completed to upload my characterization results on the Registry pages, and started to worry about the two exams approaching next week.</P><P style="height: 90px;">We are getting ready for our graduation. We have only few weeks left to our defence. This is also a good opportunity to put together the project page of the website. Jason is still the web designer!</P><H3>Week 17 (09/10 – 09/16)</H3><P>The first marble pieces with synthetic black crust came out from the crustonator, and they were beautiful! This means that they needed a treatment, so we immediately spread on them our reducing bacteria. And we kept on with the applications on the other marble pieces of course, discovering that our system worked! We were so happy for that, we could start to think about a patent…</P><P class="atfg hlGreen">I finally finished all the exams, I felt so free! But a hard work is waiting for me in the lab, because a lot of pages have to be written and uploaded on the web site. So I passed most of the time at the pc, even if I found the time to go with Anna and Guzz and meet a nice marble cutter/artist and his brother who is a geologist. These were two very interesting meetings, from which we learned a lot of interesting things; and it was a beautiful occasion to see the place of work of an artist (what a mess!).</P><P>The degree is closer and closer, and panic is in the air! But we always needed Jason to go on with the site, so we kept on exploiting him. While Jason is adding this section about him and Giacomo he realizes they have been so overwhelmed they didn't even write these last few weeks of the Notebook, he thanks who kindly did this for them. He also remembers he forgot to add to the diary about the exams he did this summer.</P><H3>Week 18 and 19 (09/17 – 09/26)</H3><P>Amsterdam is too close and we have to do a lot of work! So we passed the last days stuck in front of our PCs and Macs without eating and sleeping. We had to find the time to finish the web site, draw the posters, finish the applications on statues, think about the activity with the Natural Sciences Museum and organize the Researcher’s night. And we could not forget the presentation and the speech… It was a very hard work, and at the end we were dead tired… But full of enthusiasm to fly to Amsterdam! Boston is waiting for us, guys!</P></DIV></DIV></DIV></DIV><DIV id="footer"><DIV id="credits"><DIV id="mw"><UL><LI><A href="https://2012.igem.org">iGEM Main Page</A></LI><LI><A href="https://2012.igem.org/wiki/index.php?title=Talk:Team:UNITN-Trento/Notebook&amp;action=edit&amp;redlink=1">Discussion</A></LI><LI><A href="https://2012.igem.org/wiki/index.php?title=Team:UNITN-Trento/Notebook&amp;action=edit">Edit</A></LI><LI><A href="https://2012.igem.org/wiki/index.php?title=Team:UNITN-Trento/Notebook&amp;action=history">History</A></LI><LI><A href="javascript:void(0);">Teams</A></LI><LI><A href="https://2012.igem.org/Special:RecentChanges">Recent Changes</A></LI><LI><A href="https://2012.igem.org/Special:RecentChangesLinked/Team:UNITN-Trento/Notebook">Related Changes</A></LI><LI><A href="https://2012.igem.org/Special:SpecialPages">Special Pages</A></LI><LI><A href="https://2012.igem.org/Special:Preferences">My Preferences</A></LI><LI><A href="https://2012.igem.org/2012.igem.org:Privacy_policy">Privacy policy</A></LI><LI><A href="https://2012.igem.org/2012.igem.org:General_disclaimer">Disclaimer</A></LI></UL></DIV><P>Design by Jason Fontana. Powered by <A href="http://www.mediawiki.org/wiki/MediaWiki">MediaWiki</A>. This website is under the <A href="http://creativecommons.org/licenses/by/3.0/">CC Attribution licence</A>.</P></DIV></DIV><DIV class="printfooter">
Retrieved from &quot;<A href="http://2012.igem.org/Team:UNITN-Trento/Notebook">http://2012.igem.org/Team:UNITN-Trento/Notebook</A>&quot;</DIV><DIV id="footer-box" class="noprint"><DIV id="footer"><UL id="f-list"><LI id="t-recentchanges"><A href="/Special:RecentChanges" title="Recent changes">Recent changes</A></LI><LI id="t-whatlinkshere"><A href="/Special:WhatLinksHere/Team:UNITN-Trento/Notebook" title="List of all wiki pages that link here [j]" accesskey="j">What links here</A></LI><LI id="t-recentchangeslinked"><A href="/Special:RecentChangesLinked/Team:UNITN-Trento/Notebook" title="Recent changes in pages linked from this page [k]" accesskey="k">Related changes</A></LI><LI id="t-specialpages"><A href="/Special:SpecialPages" title="List of all special pages [q]" accesskey="q">Special pages</A></LI><LI><A href="/Special:Preferences">My preferences</A></LI></UL></DIV><DIV id="footer"><UL id="f-list"><LI id="t-print"><A href="/wiki/index.php?title=Team:UNITN-Trento/Notebook&amp;printable=yes" title="Printable version of this page [p]" accesskey="p">Printable version</A></LI><LI id="t-permalink"><A href="/wiki/index.php?title=Team:UNITN-Trento/Notebook&amp;oldid=154086" title="Permanent link to this revision of the page">Permanent link</A></LI><LI id="privacy"><A href="/2012.igem.org:Privacy_policy" title="2012.igem.org:Privacy policy">Privacy policy</A></LI><LI id="disclaimer"><A href="/2012.igem.org:General_disclaimer" title="2012.igem.org:General disclaimer">Disclaimers</A></LI></UL></DIV></DIV></BODY></HTML>