<HTML xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en" dir="ltr">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki  ltr ns-0 ns-subject page-Team_UNITN-Trento_Journal"><DIV id="globalWrapper"><DIV id="top-section"><DIV id="p-logo"><A href="/Main_Page" title="Main Page">&quot;
	    </A></DIV><DIV id="menubar" class="left-menu noprint"><UL><LI class="selected"><A href="/Team:UNITN-Trento/Journal">Page               </A></LI><LI class="new"><A href="/wiki/index.php?title=Talk:Team:UNITN-Trento/Journal&amp;action=edit&amp;redlink=1">Discussion               </A></LI><LI><A href="/wiki/index.php?title=Team:UNITN-Trento/Journal&amp;action=edit">View source               </A></LI><LI><A href="/wiki/index.php?title=Team:UNITN-Trento/Journal&amp;action=history">History               </A></LI><LI style="color:#808080;cursor:default">teams</LI></UL></DIV><DIV class="right-menu noprint" id="menubar"><UL><LI id="pt-login"><A href="/wiki/index.php?title=Special:UserLogin&amp;returnto=Team:UNITN-Trento/Journal" title="You are encouraged to log in; however, it is not mandatory [o]" accesskey="o">Log in</A></LI></UL></DIV><DIV id="search-controls" class="noprint"><FORM action="/Special:Search" id="searchform"> </FORM></DIV></DIV><DIV id="content"><H1 class="firstHeading">Team:UNITN-Trento/Journal</H1><DIV id="bodyContent"><H3 id="siteSub" class="noprint">From 2012.igem.org</H3><P><TITLE>Journal</TITLE></P><DIV class="container_24"><DIV class="header"><H1><A href="https://2012.igem.org/Team:UNITN-Trento">iGEM Trento 2012</A></H1><DIV class="nav"><UL><LI><A href="https://2012.igem.org/Team:UNITN-Trento">Home</A></LI><LI><A href="https://2012.igem.org/Team:UNITN-Trento/Blog">Blog</A></LI><LI><A href="https://2012.igem.org/Team:UNITN-Trento/Notebook">Notebook</A></LI><LI>|</LI><LI><A href="https://2012.igem.org/Team:UNITN-Trento/Project">Project</A></LI><LI><A href="https://2012.igem.org/Team:UNITN-Trento/Team">Team</A></LI><LI><A href="https://2012.igem.org/Team:UNITN-Trento/Outreach">Outreach</A></LI><LI><A href="https://2012.igem.org/Team:UNITN-Trento/Partners">Partners</A></LI><LI>|</LI></UL></DIV></DIV><DIV class="phrase"><H2 style="width: 500px;"><SPAN>We've been diligently journaling our progress.</SPAN><SPAN>Have we?</SPAN></H2></DIV><DIV class="main clearfix"><DIV class="container blog"><ARTICLE><DIV class="line"><H3 class="posttitle">Day 1</H3><SPAN class="date">dd.mm</SPAN></DIV><P>We start to meet in the lab. We have been assigned a big lab in Malga, the building containing all the teaching labs. It's the biggest one.
				We're starting to design the constructs and primers and prepare LB medium for culture. We also make a list of all we have at our disposal and what we need to order.
				 We're also very lucky: some thieves broke into Ross' car and stole Guzz's and Giacomo's laptops and Jason's and Guzz's apartment keys! 
				 We're not gonna sleep sweet dreams.
				 Too bad they we're not Macs, we would have been able to retrieve them that way..
				</P></ARTICLE><ARTICLE><DIV class="line"><H3 class="posttitle">Day 2-3</H3><SPAN class="date">dd.mm</SPAN></DIV><P>We decided to put the lab part on hiatus until monday. We realized that the powder used on day 2 was LB-agar, and not just LB, so we had to throw it away. Guzz and Ross spent the rest of the time designing primers. Giaco looked into the registry for terminators, while Andrea and Anna started taking a look protocols.
				 </P></ARTICLE><ARTICLE><DIV class="line"><H3 class="posttitle">Day 4</H3><SPAN class="date">dd.mm</SPAN></DIV><P>Ross and Guzz woke up early today. From Schio, they came to Trento passing through CIBio in order to retrieve DH5α e NovaBlue destinied to be grown in iGEM lab. The team then prepared LB broth and Petri dishes, and grown cells had been treated with transformation buffer and glycerol. Meanwhile, the team (flanked by a marble/concrete putto) has been involved in a brief meeting and photosession.
				 </P></ARTICLE><ARTICLE><DIV class="line"><H3 class="posttitle">Day 5</H3><SPAN class="date">dd.mm</SPAN></DIV><P>We transformed DH5α and Nova with pSB1K3, pSB4K5, pSB4A5 and the BioBricks for AraC pBAD (on pSB2K3) and LacI (on pSB1A2).
				 </P></ARTICLE><ARTICLE><DIV class="line"><H3 class="posttitle">Day 6</H3><SPAN class="date">dd.mm</SPAN></DIV><P>We found that only one Petri of five (pSB1K3) had colonies. 
				 We cultured DH5 and Nova transformed with pSB1C3 and pSB1K3. 
				 We also developed a simple Phyton tool which should allow to detect illegal sites in input sequences, returning their position if they exist and adds prefix and suffix if they does not.
				 We're low on araC-pBAD (BBa_I0500) from the distribution kit: we're gonna need to extract it from other wells.
				 </P><P>We looked at the feasibility of the biofilm distruption project: we optimized NucB for Caolobacter, concluding it would need synthesis, and we probably cannot afford another one.
				 DspB could be obtained from another team (already optimized for Caolobacter). In E.coli we would need to optimize both genes and also furnish ABC transporter.
				 The project, in the end, is discarded.
				 </P><P>We're coming up with ideas to measure terminator efficiency and constructs designs.
				 </P><P>//schema
				 </P></ARTICLE><ARTICLE><DIV class="line"><H3 class="posttitle">Day 7</H3><SPAN class="date">dd.mm</SPAN></DIV><P>We have colonies in all the plates, meaning we can now have a stock of these plasmids: we inoculate 10 colonies  for each plasmid backbone. Tomorrow we'll be able to Miniprep them.
				 </P><P>Regarding CysE, we had designed primers to PCR extract the gene from the E. coli genome. Today they arrived, so we start with the extraction.
				 </P><P>The lacI construct is ready: we will transform BBa-Q04121, which contains lacI and a RBS and then insert a medium-strenght constitutive promoter (BBa-J23104).
				 </P><P>//img
				 </P><P>We're a bit confused about the araCpBAD part because of the overlaps between the regions: &quot;the Sheref&quot; in person will look into it.
				 </P><P>Finally we tranform lacI from BBa-C012, araCpBAD from BBa-I0500 (both of them had failed on day 5, so we increase the DNA quantity for the transformation), plus BBa-404121 (the one we talked about previously: lacI+RBS).
				 </P><P>The refidgerator stopped worked. Panic.
				 </P></ARTICLE><ARTICLE><DIV class="line"><H3 class="posttitle">Day 8</H3><SPAN class="date">dd.mm</SPAN></DIV><P>Everything is saved thanks to another fridge.
				 Only lacI from BBa-C012 went well, which was inoculated. 
				 We then realize it would be easier to PCR extract lacI with his promoter and RBS from pET21b, a plasmid Sheref is heavily using in his lab.
				 We design the primers with prefix and suffix so they cause an inversion of the gene: we don't want RNA pol to interfere with each other.
				 </P><P>//schema
				 </P><P>It's Giacomo's birthday and he didn't bring the cake. The cake is a lie.
				 </P><P>Moar tranformations using, this time, the Registry protocol: lacI, araCpBAD (the original and those from other wells), promoters BBa_J23100, BBa-B0017 and BBa-B0010.
				 </P><P>This time we're concentrating on the desing of the constructs fot the terminators characterization.
				 </P><P>// spiegazione e schema (RL024A)
				 </P><P>It's miniprepping time: pSB1K3 and pSB1C3: both gave low quantities.
				 </P></ARTICLE><ARTICLE><DIV class="line"><H3 class="posttitle">Day 9</H3><SPAN class="date">dd.mm</SPAN></DIV><P>Jason fais the CysE PCR product gel electrophoresis, we find that these gels run faster than expected.
			</P><P>Ross transfors pBAD/araC (BBa-10500, BBa-K113009), and a pBAD/araC-containing backbone (pSB1A10) into Nova and inoculates pSB1K3- and pSB1C3-transformed bacteria. He also Phails his gel electrophoresis with Phusion for CysE. 
			</P><P>Guzz transforms pET21b and mCherry/Venus (RL024A).
			</P></ARTICLE><ARTICLE><DIV class="line"><H3 class="posttitle">Day 10</H3><SPAN class="date">dd.mm</SPAN></DIV><P>Routine work has been done.
			</P><P>Ross inoculates &quot;HOPE&quot;.
			</P><P>Ross, Guzz and Cristina purify pSB1K3, pSB1C3 (two miniprep per plasmid type per person). Unfortunately, Guzz elutes his purification 10x, otherwise he would have won the miniprep competition - and even so, he did better than Ross. Maybe the referee is to blame: our spectrophotometer might be misfunctioning.
			</P><P>Jason manages to see CysE in his electrophoresis. He made the PCR with Taq. He proceeds with the restriction with EcoR1 and Pst1.
			</P><P>Andrea inoculates promoter- (J23100), terminator- (B0010, B0017) and araC/pBAD-containing cells.
			</P></ARTICLE><ARTICLE><DIV class="line"><H3 class="posttitle">Day 11</H3><SPAN class="date">dd.mm</SPAN></DIV><P>Andrea purifies again pSB1K3 (5 tubes) and pSB1C3 (1 tube), together with the two terminator-containing cells she had inoculated the day before (3 tubes of B0017 and 2 tubes of B0010). The results are not so good, but the spectrophotometer does not seem to be working, again.
			</P><P>Jason carries on with his programme, ligating CysE into pSB1C3 vector and then transforming it into NovaBlue competent cells.
			</P><P>Anna and Andrea aliquote new strains of competent cells in glycerol: ccdB, JM109, TB1, CAG and pLys5 (these are common names we use to recognize better the strains)
			</P></ARTICLE><ARTICLE><DIV class="line"><H3 class="posttitle">Day 12</H3><SPAN class="date">dd.mm</SPAN></DIV><P>Anna and Jason inoculate 3 colonies of pET21a plasmids, 3 of terminators RL0024 and 3 colonies of pSB1C3 with CysE prepared by Jason the day before.
			</P><P>Anna continues with Ross to inoculate colonies: 3 pSB1K3, 4 pSB1A10, 4 &quot;More HOPE&quot; and 250mL of pSB1C3 colonies.
			</P><P>Anna and Jason transform the plasmid containing terminators B0017 and B0010 following the protocol you can find on iGEM Registry website; then two low-copy plasmids (pSB4K5 and pSB4A5) in ccdB-resistant strain following the &quot;Mansy lab protocol&quot;.
			</P><P>In the meantime Giaco, Ross and Andrea make PCRs of three samples in duplicate, using both OneTaq and Phusion polymerases. They amplificate araCpBad promoter from the coli's genome, HOPE and LacI promoter from pET21 plasmid. Then do an electrophoresis gel as control: everything goes better than expected.
			</P></ARTICLE><ARTICLE><DIV class="line"><H3 class="posttitle">Day 13</H3><SPAN class="date">dd.mm</SPAN></DIV><P>Andrea purificates the PCR products of the day before (AraCpBad, HOPE and LacI) with the miniprep protocol. In the meantime Anna uses the same protocol to purify pET21a, RL024, CysE and pSB1K3, but the absorption results continue to be too low. We start to have serius doubts about the reliability of our spectrophotometer..
			</P><P>Anna inoculates the terminators R0017 and R0010; then transforms pSB4K5 plasmid in ccdB-resistant strain, RL007, RL009 and RL020 in Novablue and RL024 in BL21 cells.
			</P><P>Andrea makes some samples run on an electrophoresis gel: 7 CysE into pSB1C3 plasmid previously restricted, 1 empty plasmid pSB1C3 as control and the three PCR products AraCpBad from genome, HOPE and LacI. These last ones are very good and almost all the CysE genes are there too.
			</P><P>Jason minipreps and with Giacomo's and Guzz' help manages to quanitfy the inoculated colonies which has grown overnight. Two are red, meaning they don't contain the insert.
			He proceeds with an electrophoresis to confirm the reaction went as expected. The red colony and the good colonies give the expected bands.
			</P></ARTICLE><ARTICLE><DIV class="line"><H3 class="posttitle">Day 14</H3><SPAN class="date">dd.mm</SPAN></DIV><P>Andrea purificates the PCR products of the day before (AraCpBad from genome, HOPE and LacI) and the three empty plasmids pSB1C3 that will be the future hosts. This step is indeed followed by the ligation reactions and the transformation into DH5α competent cells.
			</P><P>Andrea transforms also two empty plasmids just arrived from CIBio: pSB4A5 and pSB4K5 into ccdB-resistant strain.
			</P><P>Anna inoculates RL007, RL009, RL020, RL024 and pSB1C3.
			</P><P>Jason and Guzz are ready to start with the mutgenesis of CysE. 
			The first one by Guzz with Phusion fails, but the second one by Jason goes well. The gel confirms the results. We also make a control PCR (with OneTaq) of CysE in pSB1C3 to check it's there, just to be sure.
			</P></ARTICLE><ARTICLE><DIV class="line"><H3 class="posttitle">Day 15</H3><SPAN class="date">dd.mm</SPAN></DIV><P>Giaco purifies B0017, B0010, RL007A, RL009A and RL024A with the MINIprep protocol, and has the usual problems with the spectrophotometer in the attempt of quantifying the DNA.
			</P><P>Andrea inoculates the ligation products of the day before (10 LacI, 10 AraCpBad and 10 HOPE).
			</P><P>Anna inoculates new DH5α and NovaBlue competent cells to refurnish our stock.
			</P><P>Jason proceeds with the ligation of the mutagenesis PCR products.
			Guzz and Jason transforms mutated CysE into Nova Blue.
			</P><P>Ross and Anna extract and transform 3 parts from the distribution kit: pSB4C5, pSB3C5 and pSB4K5.
			</P><P>Ross does an electrophoresis gel with the following samples: AraCpBad from genome, mutated CysE, HOPE, LacI digested, pSB1C3 digested.
			</P><P>Then Ross inoculates RL024A for FACS the following day.
			</P><P>In the meantime some primers for terminator are designed.
			</P></ARTICLE><ARTICLE><DIV class="line"><H3 class="posttitle">Day 16</H3><SPAN class="date">dd.mm</SPAN></DIV><P>Ross completes his programme and is ready for FACS. Everything goes well at the end of the day.
			</P><P>Guzz, Giaco and Andrea purifies LacI, AraCpBad and HOPE genes and obtain the usual weird values at the spectrophotometer. Why do we keep on trusting this instrument yet?!
			</P><P>Guzz prepares new DH5α and NovaBlue competent cells.
			</P><P>Andrea digests AraCpBad, LacI and HOPE inserted into pSB1C3 with EcoRI and PstI. Then makes a huge electrophoresis gel to control the occurred restriction (5 samples + 1 empty plasmid for each insert = 18 wells + marker!): she waitsd for about 3 hours, but the results are optimal! Very good news, the digestion occurred in all samples.
			</P><P>CysE* transformation went well, so colonies are inoculated.
			Jason is at Facoltiadi, so Andrea kindly comes on Saturday to remove the inocules from the thermoshaker, and stores them at -20°C.
			</P></ARTICLE><ARTICLE><DIV class="line"><H3 class="posttitle">Day 17</H3><SPAN class="date">dd.mm</SPAN></DIV><P>Guzz and Giaco go on purifying low-copy plasmids (pSB4C5, pSB4K5 and pSB3C5) and arguing with the spectrophotometer.
			</P><P>Guzz transforms B0014 and B0015 (from registry) in NovaBlue cells, RL007A, RL009A, RL020A and RL024A in pLysS.
			</P><P>Jason minpreps CysE*.
			Ross and Jason decide to start creating the first device: they digest CysE WT, AraCpBad and pSB4K5.
			</P><P>Guzz transforms plasmids with Venus, mCherry and Venus+mCherry in BL21 pLysS.
			</P></ARTICLE><ARTICLE><DIV class="line"><H3 class="posttitle">Day 18</H3><SPAN class="date">dd.mm</SPAN></DIV><P>Guzz inoculates B0014 and B0015 (into pSB1AK3).
			</P><P>Andrea transformes again low-copy plasmids pSB4C5, pSB3C5 and pSB4K5 to have more colonies stored. She uses both the DNA from the distribution kit and the DNA purified with MINIprep in the last days.
			</P><P>Anna phosphorilates primers to insert prefix and suffix in RL024A and then goes on with the insertion mutagenesis by PCR. Unfortunately the final electrophoresis gel reveals a fail.
			</P></ARTICLE><ARTICLE><DIV class="line"><H3 class="posttitle">Day 19</H3><SPAN class="date">dd.mm</SPAN></DIV><P>Giacomo inoculates RL0024A, RL007A, RL009A with ampicillin and pLysS with no antibiotic; these colonies will be ready for FACS.
			</P></ARTICLE><ARTICLE><DIV class="line"><H3 class="posttitle">Day 20</H3><SPAN class="date">dd.mm</SPAN></DIV><P>Anna tries for the second time to do the insertion mutagenesis failed last week, but it does not work again; there may be something wrong in the protocol.
			</P><P>Guzz purificates the terminators B0014 and B0015. In the meantime Jason purificates CysE*, and together transform the construct AraCpBAD-CysE (in pSB4K5) into Nova.
			</P><P>All of these things were happening in the lab while Giacomo and Ross were going at CiBIO and realized the FACS machine was broken: totally panic. Now we are without FACS, but praying that an expert will come soon and repair this precious machine.
			</P><P>Anyway RL0024A, RL007A, RL009A and pLysS were observed at the fluorometer and their aspect were as we expected.
			</P></ARTICLE><ARTICLE><DIV class="line"><H3 class="posttitle">Day 21</H3><SPAN class="date">dd.mm</SPAN></DIV><P>Anna tries again with the insertion, but this time she slightly modifies the protocol.
			</P><P>Ross inoculates BBa<EM>E0240 and BBa</EM>E0840 from registry (they will be useful to control the activity of our promoters), AraCpBAD-CysE and three low-copy plasmids (pSB3C5, pSB4C5, pSb4K5).
			</P><P>Guzz prepares DH10a competent cells and inoculates BL21LysS, Venus, mCherry and mCherry-Venus.
			</P><P>Andrea makes an electrophoresis to control the digestion of L2, L3 (LAcI colonies), A1, A2 (AraCpBAD colonies), H2, H3 (HOPE colonies) and D (CysE colonies). Everything works well, so we are ready for sequencing! The samples will be sent tomorrow.
			</P><P>Jason incoulates the successful transformation of araCpBad-CysE.
			</P></ARTICLE><ARTICLE><DIV class="line"><H3 class="posttitle">Day 22</H3><SPAN class="date">dd.mm</SPAN></DIV><P>Cristina wants to send two more samples for sequencing: two new constructs AraCpBAD-CysE. The office of &quot;Poste Italiane&quot; closes at 12.30, so we must work quickly!
			We purificates the sample, but there is no time for quantification: it is digested without knowing the exact concentration, but we can do a valutation of about 150ng/µL.
			The digestion goes on for about 40 minutes, and the control gel starts just in time to confirm the occurred digestion of samples G and H and to be in front of the office at 12.25! Now we can just hope.
			</P><P>In the meantime also the low-copy plasmids pSB3C5, pSB4C5, pSb4K5 were purified and quantified by Guzz, that by now is known as the &quot;quantifying man&quot;. Fortunately in the afternoon he manages to do something different from quantifying, because he induces FP cells with IPTG and does the analysis at the fluorometer.
			</P><P>Ross inoculates pLysS, mCherry-Venus, mCherry and Venus that also will be analysed at the fluorometer. He transformed BBa<EM>E0240 and BBa</EM>E0840 too (backbone is pSB1A2 for both).
			</P></ARTICLE><ARTICLE><DIV class="line"><H3 class="posttitle">Day 23</H3><SPAN class="date">dd.mm</SPAN></DIV><P>Guzz inoculates BL21 pLysS, Venus, mCherry and mCherry-Venus.
			</P><P>Cristina e Andrea designs primers to insert a Ypet reporter after the LacI construct and the LacI-CysDes construct (whose construction will start as soon as we receive the sythesized gene of CysDes).
			</P><P>Andrea makes also an outline of the poster which will present our team and our project during the CiBIO BBQ of next week. </P></ARTICLE></DIV></DIV><DIV class="footer"><DIV class="mw"><UL><LI><A href="https://2012.igem.org">iGEM Main Page</A></LI><LI><A href="https://2012.igem.org/wiki/index.php?title=Talk:Team:UNITN-Trento/Journal&amp;action=edit&amp;redlink=1">Discussion</A></LI><LI><A href="https://2012.igem.org/wiki/index.php?title=Team:UNITN-Trento/Journal&amp;action=edit">Edit</A></LI><LI><A href="https://2012.igem.org/wiki/index.php?title=Team:UNITN-Trento/Journal&amp;action=history">History</A></LI><LI><A href="#">Teams</A></LI><LI>|</LI><LI><A href="https://2012.igem.org/Special:RecentChanges">Recent Changes</A></LI><LI><A href="https://2012.igem.org/Special:RecentChangesLinked/Team:UNITN-Trento/Journal">Related Changes</A></LI><LI><A href="https://2012.igem.org/Special:SpecialPages">Special Pages</A></LI><LI><A href="https://2012.igem.org/Special:Preferences">My Preferences</A></LI><LI><A href="https://2012.igem.org/2012.igem.org:Privacy_policy">Privacy policy</A></LI><LI><A href="https://2012.igem.org/2012.igem.org:General_disclaimer">Disclaimer</A></LI></UL></DIV><P><SPAN class="toggle">Toggle controls</SPAN>  |  Design by Jason Fontana. Powered by <A href="http://www.mediawiki.org/wiki/MediaWiki">MediaWiki</A>. This website is under the <A href="http://creativecommons.org/licenses/by/3.0/">CC Attribution licence</A>.<SPAN style="left: 370px;
		position: relative;">Follow our daily journal on <A class="facebook" href="http://www.facebook.com/UniTNiGEM" target="_blank">Facebook</A>!</SPAN></P></DIV></DIV><DIV class="printfooter">
Retrieved from &quot;<A href="http://2012.igem.org/Team:UNITN-Trento/Journal">http://2012.igem.org/Team:UNITN-Trento/Journal</A>&quot;</DIV></DIV></DIV><DIV id="footer-box" class="noprint"><DIV id="footer"><UL id="f-list"><LI id="t-recentchanges"><A href="/Special:RecentChanges" title="Recent changes">Recent changes</A></LI><LI id="t-whatlinkshere"><A href="/Special:WhatLinksHere/Team:UNITN-Trento/Journal" title="List of all wiki pages that link here [j]" accesskey="j">What links here</A></LI><LI id="t-recentchangeslinked"><A href="/Special:RecentChangesLinked/Team:UNITN-Trento/Journal" title="Recent changes in pages linked from this page [k]" accesskey="k">Related changes</A></LI><LI id="t-specialpages"><A href="/Special:SpecialPages" title="List of all special pages [q]" accesskey="q">Special pages</A></LI><LI><A href="/Special:Preferences">My preferences</A></LI></UL></DIV><DIV id="footer"><UL id="f-list"><LI id="t-print"><A href="/wiki/index.php?title=Team:UNITN-Trento/Journal&amp;printable=yes" title="Printable version of this page [p]" accesskey="p">Printable version</A></LI><LI id="t-permalink"><A href="/wiki/index.php?title=Team:UNITN-Trento/Journal&amp;oldid=26571" title="Permanent link to this revision of the page">Permanent link</A></LI><LI id="privacy"><A href="/2012.igem.org:Privacy_policy" title="2012.igem.org:Privacy policy">Privacy policy</A></LI><LI id="disclaimer"><A href="/2012.igem.org:General_disclaimer" title="2012.igem.org:General disclaimer">Disclaimers</A></LI></UL></DIV></DIV></DIV></BODY></HTML>