<HTML xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en" dir="ltr">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki  ltr ns-0 ns-subject page-Team_UNITN-Trento_Protocols"><DIV id="globalWrapper"><DIV id="top-section"><DIV id="p-logo"><A href="/Main_Page" title="Main Page">&quot;
	    </A></DIV><DIV id="menubar" class="left-menu noprint"><UL><LI class="selected"><A href="/Team:UNITN-Trento/Protocols">Page               </A></LI><LI class="new"><A href="/wiki/index.php?title=Talk:Team:UNITN-Trento/Protocols&amp;action=edit&amp;redlink=1">Discussion               </A></LI><LI><A href="/wiki/index.php?title=Team:UNITN-Trento/Protocols&amp;action=edit">View source               </A></LI><LI><A href="/wiki/index.php?title=Team:UNITN-Trento/Protocols&amp;action=history">History               </A></LI><LI style="color:#808080;cursor:default">teams</LI></UL></DIV><DIV class="right-menu noprint" id="menubar"><UL><LI id="pt-login"><A href="/wiki/index.php?title=Special:UserLogin&amp;returnto=Team:UNITN-Trento/Protocols" title="You are encouraged to log in; however, it is not mandatory [o]" accesskey="o">Log in</A></LI></UL></DIV><DIV id="search-controls" class="noprint"><FORM action="/Special:Search" id="searchform"> </FORM></DIV></DIV><DIV id="content"><H1 class="firstHeading">Team:UNITN-Trento/Protocols</H1><DIV id="bodyContent"><H3 id="siteSub" class="noprint">From 2012.igem.org</H3><P><TITLE>Protocols</TITLE></P><H2 style="font-family: 'Open Sans'; line-height: 40px; color:#ffae00; padding:0.5em; font-size:30pt !important; text-align: center; width: 900px; margin: 100px auto;">It appears you have JavaScript disabled. Our Wiki needs it for you to enjoy the full experience. Please enable Javascript and refresh the page.</H2><DIV id="header"><DIV id="topNav"><UL><LI><A href="https://2012.igem.org/Team:UNITN-Trento">Home</A></LI><LI class="folder"><A href="javascript:void(0);">Project</A></LI><LI><A href="https://2012.igem.org/Team:UNITN-Trento/Project/CrustAway">Crust Away</A></LI><LI><A href="https://2012.igem.org/Team:UNITN-Trento/Project/Terminators">Terminator 5</A></LI><LI><A href="https://2012.igem.org/Team:UNITN-Trento/Attributions">Attributions</A></LI></UL></DIV><LI><A href="https://2012.igem.org/Team:UNITN-Trento/DataPage">Data Page</A></LI><LI class="folder"><A href="javascript:void(0);">Parts</A></LI><LI><A href="https://2012.igem.org/Team:UNITN-Trento/Parts">Our Parts</A></LI><LI><A href="http://partsregistry.org/cgi/partsdb/pgroup.cgi?pgroup=iGEM2012&amp;group=UNITN-Trento">Registry Section</A></LI></DIV><LI><A href="https://2012.igem.org/Team:UNITN-Trento/Achievements">Judging</A></LI><LI class="folder"><A href="javascript:void(0);">Team</A></LI><LI><A href="https://2012.igem.org/Team:UNITN-Trento/Team">The Team</A></LI><LI><A href="https://2012.igem.org/Team:UNITN-Trento/Team/Supervisors">Super- &amp; Ad- visors</A></LI><LI><A href="https://2012.igem.org/Team:UNITN-Trento/Friends">Our Friends</A></LI><LI><A href="https://igem.org/Team.cgi?id=731">Team Profile</A></LI><LI><A href="https://2012.igem.org/Team:UNITN-Trento/Trento">Where We Are</A></LI><LI><A href="https://2012.igem.org/Team:UNITN-Trento/Press">Press</A></LI></DIV><LI class="folder"><A href="javascript:void(0);">Partners</A></LI><LI><A href="https://2012.igem.org/Team:UNITN-Trento/Sponsors">Sponsors</A></LI></DIV><LI class="folder activeLink"><A href="javascript:void(0);">Lab Notes</A></LI><LI><A href="https://2012.igem.org/Team:UNITN-Trento/Notebook">Notebook</A></LI><LI><A href="https://2012.igem.org/Team:UNITN-Trento/Meetings">Meetings</A></LI><LI><A href="https://2012.igem.org/Team:UNITN-Trento/Protocols">Protocols</A></LI><LI><A href="https://2012.igem.org/Team:UNITN-Trento/Tools">Tools</A></LI></DIV><LI class="folder"><A href="javascript:void(0);">Safety</A></LI><LI><A href="https://2012.igem.org/Team:UNITN-Trento/Safety">Safety</A></LI><LI><A href="https://2012.igem.org/Team:UNITN-Trento/H2S">H<SUB>2</SUB>S</A></LI><LI><A href="https://2012.igem.org/Team:UNITN-Trento/Outreach">Outreach</A></LI><LI><A href="https://2012.igem.org/Team:UNITN-Trento/Art">Art&amp;Science</A></LI><DIV id="logo"><DIV class="wrapper"><H1><A href="https://2012.igem.org/Team:UNITN-Trento" style="font-family: 'Bree Serif';"><SPAN>i</SPAN>GEM Trento 2012</A></H1></DIV></DIV><DIV id="canvas" class="home"><DIV id="wrapper" style=""><SECTION id="main" style="width:100%; position: relative;"><DIV class="punch" style="margin: 0 auto;"><H2 style="text-align: center;">A summary of the Protocols we used</H2></DIV><DIV id="sticky" class="weekNav" style="display: none;"><UL style="margin-left: 50px;"><LI><A class="top" href="javascript:void(0);">Back to Top</A></LI></UL></DIV><DIV class="content" style="padding-left: 20px ;padding-right: 20px ;"><DIV class="summary"><DIV style="width: 63%; float: left;"><DIV style="width: 49%; float: left; margin-top: 0;"><H4>General</H4><UL style="margin-top: 0;"><LI>1. <A class="a" href="javascript:void(0);">Competent Cells</A></LI><LI>2. <A class="b" href="javascript:void(0);">Transformation</A></LI><LI>3. <A class="c" href="javascript:void(0);">PCR</A></LI><LI>4. <A class="d" href="javascript:void(0);">Cloning</A></LI><LI>5. <A class="e" href="javascript:void(0);">Gibson's Assembly</A></LI><LI>6. <A class="f" href="javascript:void(0);">Mutagenesis</A></LI></UL></DIV><DIV style="width: 48%; float: left; margin-top: 0;"><H4>Black Crust</H4><UL style="margin-top: 0;"><LI>7.    <A class="g" href="javascript:void(0);">MOPS Media</A></LI><LI>8.    <A class="h" href="javascript:void(0);">Cell Growth</A></LI><LI>9.    <A class="i" href="javascript:void(0);">Serial Dilutions</A></LI><LI>10. <A class="j" href="javascript:void(0);">Fluorescence Measurements</A></LI><LI>11. <A class="k" href="javascript:void(0);">Ninhydrin Assay</A></LI><LI>12. <A class="l" href="javascript:void(0);">Methylen Blue Assay</A></LI><LI>13. <A class="m" href="javascript:void(0);">TSI</A></LI><LI>14. <A class="n" href="javascript:void(0);">Copper Precipitation</A></LI><LI>15. <A class="o" href="javascript:void(0);">Crustanator</A></LI><LI>16. <A class="p" href="javascript:void(0);">Application on Statues</A></LI></UL></DIV></DIV><DIV style="width: 34%;float: left;"><H4>Terminators</H4><UL style="margin-top: 0;"><LI>17. <A class="q" href="javascript:void(0);">Fluorescence Measurements</A></LI><LI>18. <A class="r" href="javascript:void(0);">Ethanol Precipitation</A></LI><LI>19. <A class="s" href="javascript:void(0);">Cell Free Measurement</A></LI></UL></DIV></DIV><H3>1. COMPETENT CELLS</H3><UL><LI>Transformation Buffer: sterile 10 mM Tris-HCl, pH 7.0, 50 mM CaCl2</LI><LI>Grow a 50 mL culture in LB at 37 deg C from 1 colony.</LI><LI>When OD ~ 0.5, collect the cells in a sterile Falcon tube and chill on ice for 10 min.</LI><LI>Centrifuge at 5000 rpm for 10 min at 4 deg C. Discard supernatant.</LI><LI>Resuspend cells in 15 mL of transformation buffer.</LI><LI>Chill on ice for 15 min. Spin at 5000 rpm for 10 min at 4 deg C. Discard supernatant.</LI><LI>Resuspend cells in 4 mL of transformation buffer.</LI></UL><P>The cells are now ready to be transformed. They can be stored in this state at 4 deg C for under a week. </P><P>Alternatively, the competent cells can be aliquoted (200µL), adding glycerol to a final conc of 15% (v/v), and the cells stored at –80 deg C. </P><P>Every time you make new competent cells you should check for possible contaminations. Plate an aliquot of the new cells in LB plates + antibiotic (i.e. ampicillin, chloramphenicol, kanamycin). Strains such as DH5a, NEB10b, Novablue, should not grow in the presence of antibiotics.</P><H3>2. TRANSFORMATION</H3><UL><LI>Thaw home made CaCl2 competent cells on ice.</LI><LI>Add 1µL of DNA to 200µL of competent cells (concentration of DNA stock should be between 50–150 ng/µL).</LI><LI>Incubate on ice for 30 min.</LI><LI>Heat shock at 42 deg C for exactly 2 min.</LI><LI>Incubate on ice 1 min.</LI><LI>Add 800µL of LB (or SOC) and shake at 37 deg C for 1 h.</LI><LI>Plate the cells (use plates with the appropriate antibiotic according to your plasmid).</LI></UL><P>You can either plate a small amount (200µL) of the cells or more. </P><P>You should try a few conditions the first time and then choose the one that gives 30–300 separate colonies. </P><P>If few cells are expected: spin down the cells at 2500 rpm, discard supernatant &amp; resuspend in 150–200µL of LB and plate all the cells. </P><P>For ligation you should increase the amount of DNA to be transformed (see cloning protocol).</P><UL><LI>Incubate the plates O/N upside down at 37 deg C.</LI></UL><H3>3. PCR</H3><P>We have used a number of different Polymerases for amplification of our parts. </P><P>In each case we have followed the protocol suggested by the manufacturer. </P><P>Here is a list of the Polymerases that we have used.</P><P>New England Biolabs:
										- <A href="http://www.neb.com/nebecomm/products/protocol631.asp">Phusion</A>
										- <A href="http://www.neb.com/nebecomm/products/protocol811.asp">One Taq</A></P><P>Kapa Biosystems:
										- <A href="http://www.kapabiosystems.com/public/pdfs/kapa-hifi-pcr-kits/KAPA_HiFi_TDS.pdf">Kapa Hifi Polymerase</A></P><P>RBC:
										- Find link</P><H3>4. CLONING</H3><P>Most of our parts were cloned starting from a PCR amplified product, purified and digested, which was subsequently inserted into the destination vector.</P><P>Alternatively, we used a 2A assembly strategy, where the source parts and the destination plasmid where digested separately and subsequently purified before the ligation step. </P><P><A href="http://www.science.unitn.it/~igem/tools/Cloningtemplate.docx">You can find here our working sheet</A> that we used in the lab. We hope you find it helpful!</P><H4>STEP 1: DIGESTION</H4><P>For PCR products :</P><UL><LI>~ 3–4 µg of PCR product</LI><LI>2.5µL of restriction enzyme 1</LI><LI>2.5µL of restriction enzyme 2</LI><LI>10µL of buffer (i.e. NEB 4)</LI><LI>1µL of BSA 10X</LI><LI>xµL of H2O to reach 100µL</LI></UL><P>Incubate at 37C o/n. The day after add 1µL of DpnI at 37 deg C for 2 hours.
										Please note that PCR product must be purified before digestion.</P><P>For plasmids:</P><UL><LI>~2–3 µg of vector</LI><LI>1µL of BSA</LI><LI>1.5µL of enzyme 1</LI><LI>1.5µL of enzyme 2</LI><LI>5µL buffer</LI><LI>xµL of H2O up to 50µL</LI></UL><P>The day after add 1µL of phosphatase (CIP or SAP) to the vector and incubate for 2 hours at 37 deg C.</P><H4>STEP 2: PURIFICATION</H4><P>Purify the digested PCR product and digested vector with a kit.</P><P>Follow the kit’s protocol. </P><P>Subsequently, check the concentration by UV/VIS or electrophoresis for quantification.</P><H4>STEP 3: LIGATION</H4><P><A href="http://www.science.unitn.it/~igem/tools/IGEMTrento2012_ligationcalculator.xlsx">Find here our ligation calculator</A>! </P><P>Prepare your reaction and incubate at RT for 2 hours. Transform half of the reaction into 200µL of “homemade” competent cells (DH5a, NEB10b, Novablue or other appropriate strains) following a standard transformation protocol. Plate all the cells.</P><H4>STEP 4: SCREENING</H4><P>If your reaction worked the control plate should have none or few colonies. If you are working with psB1C3 or other RFP containing plasmid, you can screen colonies by looking at the color. </P><P>RED: no success.
										WHITE: ready to screen.</P><P>Grow 6 separate colonies, each in 10 mL of LB + antibiotic overnight at 37°C.
										Miniprep and quantify.</P><P>Digestion:</P><UL><LI>DNA ~1.0 µg</LI><LI>10X buffer 2</LI><LI>enzyme 1 1µL</LI><LI>enzyme 2 1µL</LI><LI>10X BSA 1</LI><LI>H2O up to 20µL</LI></UL><P>Incubate for 1.5 h at 37 deg C. </P><P>Run all the digested product on an agarose gel to screen colonies.</P><P>Correct constructs should have 2 bands, one corresponding to the vector and one corresponding to the insert. Send now for sequencing.</P><H3>5. GIBSON ASSEMBLY</H3><P>We have successfully used this method to build 2 of our favorite parts (xx and XX). We did not encounter any particular difficulties.</P><H4>STEP 1: PRIMER DESIGN</H4><P>To this purpose you may find useful the GIBTHON tool designed by Cambridge iGEM team 2010. XX</P><H4>STEP 2: PCR</H4><P>If you used the Phusion method with HF buffer, to PCR out your gene, ther ei sno need of purification.</P><P>Quantify each product preferably by gel.</P><H4>STEP 3: ASSEMBLY</H4><P>We used the NEB Gibson kit (E2611).</P><UL><LI>Master Mix 10µL</LI><LI>Fragment 1 xµL</LI><LI>Fragment 2 xµL</LI><LI>H2O up to 20µL</LI></UL><P>We optimized the quantities of DNA to be used:
										Big Fragments between 50–100ng.
										Small Fragment s 2–3 fold excess. </P><P>If needed liophilize and resuspend to reach the right concentration.</P><P>Mix together on ice the different fragments, add the mastermix and
										incubate at 50C for 1 hour. </P><P>Transform 10µL of the product into competent cells (i.e. DH5a or Novablue).</P><H4>STEP 4: SCREENING</H4><P>Proceed as indicated in STEP 4 of the cloning protocol.</P><H3>6. MUTAGENESIS (BACK TO BACK PCR)</H3><H4>STEP 1: PRIMER DESIGN</H4><P>Primers are designed back to back. The mutations should be placed in the middle of the forward primer.</P><P>For nucleotide change:</P><BLOCKQUOTE><P><U>ATCGCCTCCTGCAGCGGGGGC</U><I>GGCCTGGAGACG<B>GGC</B>AACAGCGAGGAAG</I></P></BLOCKQUOTE><P>Mutation is shown in bold.
										Primer forward will be the one in <I>italic</I>. Primer reverse will be the reverse complement of the <U>underlined</U> sequence.</P><P>For deletions:</P><BLOCKQUOTE><P><U>ATCGCCTCCTGCAGCGGGG</U><B>GCGGCCTGGAGACG</B><I>GGCAACAGCGAGGAAGGCTGG</I></P></BLOCKQUOTE><P>Part that you want to delete is in <B>bold</B>.
										Primer 1 will be the one in <I>italic</I> in the forward direction.
										Primer 2 will be the reverse complement of the <U>underlined</U> sequence. </P><P>For insertions:</P><BLOCKQUOTE><P><U>ATCGCCTCCTGCAGCGGGG<I><B>GCGGCCT</B></I></U><I><B>GGAGACG</B>GGCAACAGCGAGGAAGGCTGG</I></P></BLOCKQUOTE><P>Part to be inserted is shown in <B>bold</B>.
										Primer forward will be the one in <I>italic</I>.
										Primer reverse will be the reverse complement of the sequence <U>underlined</U>.</P><H4>STEP 2: PRIMER PHOSPHORYLATION</H4><P>Make primer stocks of 100µM.
										Set reaction with:</P><UL><LI>10x PNK buffer 5µL</LI><LI>10 mM ATP 5µL</LI><LI>100µM oligo 5µL</LI><LI>PNK 2µL</LI><LI>H2O 33µL</LI></UL><P>Leave at 37 deg C for 45 min, up to 1 hr.
										Deactivate PNK, by incubation for 20 min at 65 deg C </P><H3>STEP 3: PCR</H3><UL><LI>Oligo 1, phosphorylated (10µM) 2.5µL</LI><LI>Oligo 2, phosphorylated (10µM) 2.5µL</LI><LI>dNTP (10µM) 1µL</LI><LI>DNA template (50 pg- 1 ng) 1µL</LI><LI>DMSO (optional) 1.5µL</LI><LI>H2O 31.5µL</LI><LI>Phusion Polymerase 0.5µL</LI></UL><P>Phusion polymerase comes with 2 different buffers. Use as a default the HF buffer. If you have problems, try the GC buffer.</P><P>PCR Parameters:</P><OL><LI>30 sec @ 98 deg C</LI><LI>10 sec @ 98 deg C</LI><LI>30 sec @72 deg C</LI><LI>15–30 sec per kb of template @72 deg C
										Repeat cycles 2–4 for a total of 25 times</LI><LI>10 min @72 deg C</LI><LI>keep at 4 deg C</LI></OL><P>We optimized the concentration of DNA template to be used to 1 ng.
										<EM>Optional:</EM> Check to see if the reaction worked by running 10µL of the reaction on an agarose gel.</P><H4>STEP 4: LIGATION</H4><UL><LI>PCR product 3- 5µL</LI><LI>10x Ligase Buffer 2µL</LI><LI>Ligase 1µL</LI><LI>H2O up to 20µL</LI></UL><P>Incubate at room temp for 1 hour.
										Transform half of the ligation product into an aliquot of competent cells.
										Proceed with standard transformation protocol.</P><H4>STEP5: SCREENING</H4><P>Unfortunately for mutagenesis there is no good way to screen your correct colonies, unless you are making a big insertion/deletion. </P><P>Grow at least 6 different colonies in 10 mL LB + appropriate antibiotic O/N at 37C. </P><P>Miniprep and send for sequencing.</P><H3>7. MOPS MEDIA</H3><P><A href="http://www.science.unitn.it/~igem/tools/MOPScalculator.xlsx">Here is our MOPS calculator</A>! </P><H3>8. CELLS GROWTH</H3><UL><LI>Grow one colony in 5ml of LB+antibiotic O/N at 37°C.</LI><LI>In the morning make a 1:100 dilution in 30 mL of fresh LB+ antibiotic and grow the cells at 37C until you reach an 0.5 OD.</LI><LI>Once you have reached an OD of 0.5 split the culture in two parts and spin down the cells for 10min at 4100rpm.</LI><LI>Discard the supernatant and resuspend the each of the 2 cultures in 15ml of MOPS. One culture will be your control, the other will be the culture to be induced.</LI><LI>Grow the two cultures in the thermoshaker at 37°C until the OD is 0.7.</LI><LI>Induce one of the 2 cultures with 5mM arabinose or 0.1 mM IPTG (or other desired concentration).</LI><LI>Take an aliquot to measure optical density every hour for 8 hours.</LI></UL><H3>9. TOXICITY TEST BY SERIAL DILUTION</H3><UL><LI>Follow the same protocol as for the growth test.</LI><LI>After induction leave the cells at 37C for 8 hours (or desired amount of time).</LI><LI>After 8 hours take an aliquot (100µL) from the uninduced culture (sample A) and add it to 9.9 mL of fresh LB. Repeat the same procedure for the induced culture (sample B). These two samples will be used to make serial dilutions.</LI></UL><P>Sample A:
										Prepare 3 tubes with 4.5 mL of fresh LB and label each tube. Add 500µL of cells to 4.5 mL of LB and vortex. Subsequently, remove a 500µL aliquot from this newly made dilution and add it to a fresh tube of 4.5 mL LB. Repeat the process until a 10^5 fold dilution is reached.
										Plate 150µL of each sample on LB + antibiotic and incubate at 37C O/N.</P><P>Sample B:
										Repeat the same procedure used for sample A.</P><P>The day after, you can compare the number of colonies in each plate for the two different samples.
										To estimate the number of cells calculate the CFU/mL using this formula:</P><P>CFU/ml = (# colonies counted)*(dilution factor)/(mL of culture plated)</P><H3>10. FLUORESCENCE MEASUREMENTS TO TEST PROTEIN EXPRESSION</H3><UL><LI>Follow the same protocol used for growth characterization.</LI><LI>After induction leave the cells at 37C for 4 hours (or desired amount of time).</LI><LI>Take a 1.5 mL aliquot of the uninduced and of the induced culture every hour to measure fluorescence intensity.</LI><LI>Spin down the sample at 4000 RPM for 10 min. Discard the supernatant and resuspend in 1.5 mL of PBS. Place the sample in a cuvette and take a fluorescence spectrum using proper excitation wavelength (i.e. GFP 485 nm)</LI><LI>Leave the sample at 4C O/N to allow proper folding of GFP. The day after measure again the fluorescence.</LI></UL><H3>11. NINHYDRIN ASSAY</H3><P>To test for the presence of cysteine we used a modified version of the Gaitonde test (Biochem. J., 1967, 104, 627).</P><UL><LI>Prepare a Ninhydrin stock solution by dissolving 250mg of ninhydrin and 10ml of a solution composed by glacial acetic acid (60%) and HCl (40%). Vortex the reagent until complete solubilization. The stock solution should be made fresh each time.</LI><LI>Mix 500µl of ninhydrin reagent 500µl glacial acetic acid and 500µL of the culture sample to be tested.</LI><LI>Leave the solution in a 90°C bath for 10 minutes and wait for color development.</LI><LI>Measure absorbance with a UV/VIS spectrophotometer between 600nm and 400nm. Maximum absorbance peak for cysteine is at 560 nm.</LI></UL><P>A standard curve for quantification is built with cysteine in a concentration range between 0 and 0.5 mM. </P><H3>12. METHYLEN BLUE ASSAY FOR H2S DEVELOPMENT</H3><P>For this test we used the protocol used by the Keasling group in Appl. Environ. Microbiol., 2000, 4497–502.</P><P>Preparation of cell lysate:</P><UL><LI>Spin down 5 mL of cells at 17000g for 10min</LI><LI>Resuspend the cells in 660µl of a 50mM Tris-HCl solution (pH 7.5)</LI><LI>Add 840µl of lysis buffer (300 mM NaCl, 90 mM EDTA, 50 mM Tris-HCl , pH 7.5)</LI><LI>Sonicate the suspension 3 times for 10 seconds, waiting for 1 minute between each sonication step. Keep the cells on ice during the sonication.</LI><LI>Centrifuge the suspension for 20 min at 4°C.</LI></UL><P>Cysteine desulfhydrase activity assay:</P><UL><LI>Add 0.1mM Cysteine to the cell lysate</LI><LI>Incubate the mixture for 1h at 37°C</LI><LI>Prepare the following stock solutions:
										
										</LI><LI>Solution A: 0.02 M M N,N-dimethyl-p-phenylenediamine sulfate in 7.2 M HCl</LI><LI>Solution B: 0.3 M FeCl3 in 1.2 M HCl</LI></UL><LI>After the incubation period at 37C add 0.1ml of solution A and 0.1 mL of solution B to the cell lysate.</LI><LI>Vortex the mixture for about 1 minute.</LI><LI>Allow the color to develop: the solution turns immediately to green/blue if H2S is present.</LI><P>For quantification, take absorbance measurement with a UV-VIS spectrometer between 500 and 720 nm (maximum peak is at 670 nm).</P><H3>13. TSI</H3><P>We have used TSI from Sigma (cat no.44940 ) containing:
										<UL><LI>agar, 12 g/L</LI><LI>ferrous sulfate, 0.2 g/L</LI><LI>glucose, 1 g/L</LI><LI>lactose, 10 g/L</LI><LI>meat extract, 3 g/L</LI><LI>mixed peptone, 20 g/L</LI><LI>phenol red, 0.025 g/L</LI><LI>sodium chloride, 5 g/L</LI><LI>sodium thiosulfate, 0.3 g/L</LI><LI>sucrose, 10 g/L</LI><LI>yeast extract, 3 g/L</LI></UL>
										final pH  	 7.4±0.2 (25 °C)
										<UL><LI>Autoclave</LI><LI>When the medium reaches room temperature add antibiotic and IPTG (0.1 mM) and/or arabinose (5 mM).</LI><LI>Make separate slant tubes for each condition to be tested.</LI><LI>Once the gel is solid keep the tubes at 4 °C.</LI><LI>When you decide to use the slant, with a small tip place 5-10 ul of culture in the middle of the slant tube and incubate at 37 °C for 24 hours or more.</LI><UL>
										
										if H2S is developed the media will quickly blacken.
										</UL></UL></P><H3>14. COPPER PRECIPITATION</H3><P> Calibration curve:
										<UL><LI>Make 100mM solution of Bathocuproinedisulfonic (BCS) reagent.</LI><LI>Make serial dilutions of 1mM to 10pM CuSO4 in LB media. </LI><LI>Add 1µL of 1M ascorbate and 1µL of BCS to 100µL of each standard copper solution. </LI><LI>Measure absorbance of each copper solution at 483nm. </LI><LI>Use LB media without copper as blank. </LI></UL>
										Growth of the coltures:
										<UL><LI>Grow the bacteria in 20 mL of LB + antibiotic and with 2mM CuSO4 until reach an OD of 0.6. </LI><LI>Split the cells into two new sterile tubes and induce one of the two with 0.1 mM IPTG. </LI><LI>For each sample measure the OD every hour up to 5 hours. </LI><LI>Transfer all the media from the cuvettes to 1mL eppendorf tubes and centrifuge at 13,200 rpms for 2 minutes. Collect the supernatants. </LI><LI>Add 1µl of Bathocuproinedisulfonic to 100µl of supernatant and mix. </LI><LI>Repeat this step for each supernatant you collected. </LI><LI>Measure absorbance of supernatant at 470nm. </LI><LI>Use LB media without copper as blank.  </LI></UL></P><H3>15. CRUSTANATOR</H3><P>This protocol is a modified version of the protocol used by  Gomez, Smith and Viles at the University  of Oxford.</P><H4>Materials</H4><UL><LI>4x plexiglass panels of desired dimensions (2 for the long sides, 1 for the bottom and 1 for the top)</LI><LI>2x plexiglass panels for the short sides</LI><LI>5 m of plastic tube of 1 cm diameter</LI><LI>a glass jar with a metal cap</LI><LI>3 o-rings</LI><LI>silicon</LI><LI>rubber sheath</LI><LI>charcoal</LI></UL><H4>Reagents and glassware</H4><UL><LI>beaker</LI><LI>distilled water</LI><LI>100 mL of sulfurous acid (per treatment)</LI></UL><P>You will also need to connect the chamber to a compressed air source.</P><H4>Instructions to build the box</H4><UL><LI>Drill a hole in one of the panels and put an O-ring on it, this will be one of the sides. Keep one of the plexiglass panels aside and assemble the box with silicon. Make sure that the box is tightly sealed. </LI><LI>Once the box is assembled apply the sticky rubber sheath where the top of the box will be placed. The chamber must be hermetic at normal atmosphere pressure but should have the possibility to create a small opening when you inject compressed air. </LI><LI>Drill two holes in the metal cap of the jar, and place a O-ring in each hole. Gather some charcoal from the last barbecue, and crush it with a mortar and pestle. Pour the crushed ashes inside the jar, and screw the cap to close it.</LI><LI>Finally, cut the plastic tube in two pieces depending on your needs. Connect the first from the compressed air source to the jar and the second tubef rom the jar to the chamber.</LI></UL><H4>Black crust formation.</H4><UL><LI>work under fume hood and wear double gloves</LI><LI>Wet marble pieces with distilled water and place them into a beaker. It is preferable to leave the marble in water for 24 hours prior the experiment.</LI><LI>Fill the beaker until the marble pieces is submerged roughly by half.</LI><LI>Fill another beaker with sulfurous acid, and pour some on the surface of the marble piece too.</LI><LI>Place the beakers inside the camber, and cover the box with the top plexiglass panel. Place a weight on it to keep it down when you will inject air inside the chamber.</LI><LI>Inject compressed air for 2-3 seconds, you should be able to see ashes entering the chamber. </LI><LI>Repeat this step twice, every 72 hours. </LI><LI>At the ninth day open the box and collect the marble pieces. They should be covered in black crusts. A solid layer of gypsum and ashes may also form above water surface, and that can be used for tests too. Dry the marble samples.</LI></UL><H3>16. APPLICATION ON THE STATUES</H3><P> 
										
										Jelly MOPS:
										<UL><LI>Gel is freshly prepared prior to every application.</LI><LI>6 g/L agar is added to  MOPS medium and heated in the microwave until dissolved.</LI><LI>Cool the gel on ice.</LI><LI>Add antibiotic, arabinose and/or IPTG.</LI><LI>Whipp the gel using an egg beater or similar tool to incorporate air bubbles in the gel matrix.  </LI></UL>
										
										Bacteria preparation:
										<UL><LI>Grow bacteria in MOPS following our standard protocol.</LI><LI>After the bacteria are induced (i.e. arabinose and/or IPTG).</LI><LI>After 3 hours of induction, spin down the bacteria and discard the supernatant.</LI><LI>Resuspend the bacteria in freshly prepared Jelly MOPS and apply them on the statues.</LI><LI>Cover the gel with rice paper pre-rinsed in PBS.</LI><LI>After 12 hours remove the bacteria and gently clean the surface with a cotton swab.</LI><LI>If necessary apply freshly induced bacteria again.</LI><LI>After the last cycle of application, rinse the surface with warm water and gently pat it with tissue paper.</LI></UL></P><H2>TERMINATORS PROJECT</H2><H3>17. FLUORESCENCE MEASUREMENTS</H3><UL><LI>Take a 100µl aliquot of a frozen cell glycerol stock and add it to 10ml of LB with antibiotic.</LI><LI>When you reach an OD of 0.4 place the samples on ice.</LI><LI>Dilute the cells of each sample to an OD of 0.2 in 10 mL of LB. Make sure that you keep all the samples on ice during this step.</LI><LI>Place again the samples in the 37C thermoshaker simultaneously. From now on treat all the samples equally to guarantee less variability in the results.</LI><LI>Grow until 0.6 and induce with 0.5 mM IPTG.</LI><LI>After 3 hours keep the cells on ice.</LI><LI>Sonicate the cells 3 times for 10 seconds, with about 30 seconds lapse in between and 50% amplitude.</LI><LI>Centrifuge for 1min at 4000 RPM.</LI><LI>Mix 1ml of supernatant and 1ml of PBS 1X in a cuvette.</LI><LI>Leave the sample O/N in the refrigerator (4°C) to allow proper protein folding of fluorescent proteins.</LI></UL><P>The day after measures fluorescence intensities with a fluorimeter.</P><P>Parameters used are:</P><UL><LI>Voltage: 570V for the T7promoter constructs, 520V for TAC promoter.</LI><LI>Excitation wavelength for A206K Venus: 485nm.</LI><LI>Excitation wavelength for mCherry: 587nm.</LI><LI>Emission wavelength for A206K Venus: 528 nm.</LI><LI>Emission wavelength for mCherry: 615nm.</LI></UL><H3>18. ETHANOL PRECIPITATION</H3><UL><LI>Use between 80µl and 100µl of a DNA solution already purified with a miniprep kit (concentration &gt;100ng/µl).</LI><LI>Add 0.1 volume of 3M sodium acetate and 2.5 volumes of cold ethanol.</LI><LI>Mix well.</LI><LI>Incubate the sample on ice for 10 min.</LI><LI>Centrifuge for 30 minutes at maximum speed.</LI><LI>Decant or aspirate supernatant taking care not to dislodge the pellet.</LI><LI>Wash the pellet by adding enough volume of 70–80% ethanol to at least cover the pellet.</LI><LI>Dislodge the pellet by vortexing.</LI><LI>Finally, centrifuge for 30 minutes at maximum speed and decant or aspirate to remove the supernatant (as in previous step).</LI><LI>Dry the pellet with a speed-vac, desiccator or air dry.</LI><LI><EM>Optional</EM>: in the beginning you can add glycogen as a carrier, useful if you have low concentration of DNA. To do so, just add 1µL of 20mg/mL glycogen before first or second step.</LI></UL><P><EM>Note</EM>: shorter centrifugation times may decrease yields.</P><H3>19. CELL FREE MEASUREMENTS</H3><P>We have used PurExpress in vitro protein synthesis kit from NEB (E6800S), following the protocol suggested by the manufacturer. We have used 250ng of DNA template, previously purified by ethanol precipitation.</P><P><EM>Note</EM>: add DNA right before starting the measurements.</P><P>You nee to use a fluorimeter that is set up for kinetics measurements (i.e. PTI Quantamaster 40) and can acquire data using multiple excitation wavelengths simultaneously. </P><P>The parameters to take measures are: </P><P>| | Excitation | Emission |
										| A206K Venus | 485nm | 528nm |
										| mCherry | 587nm | 615nm | </P><P>Take a measurement every minute for 360 times.
										At the end of the measurement you can take also excitation and emission spectra of each protein using the same parameters. </P></DIV></SECTION></DIV></DIV><DIV id="footer"><DIV id="credits"><DIV id="mw"><UL><LI><A href="https://2012.igem.org">iGEM Main Page</A></LI><LI><A href="https://2012.igem.org/wiki/index.php?title=Team:UNITN-Trento/Protocols&amp;action=edit">Edit</A></LI><LI><A href="https://2012.igem.org/wiki/index.php?title=Team:UNITN-Trento/Protocols&amp;action=history">History</A></LI><LI><A href="javascript:void(0);">Teams</A></LI><LI><A href="https://2012.igem.org/Special:RecentChanges">Recent Changes</A></LI><LI><A href="https://2012.igem.org/Special:SpecialPages">Special Pages</A></LI><LI><A href="https://2012.igem.org/Special:Preferences">My Preferences</A></LI></UL></DIV><P>Design by Jason Fontana. Powered by <A href="http://www.mediawiki.org/wiki/MediaWiki">MediaWiki</A>. This website is under the <A href="http://creativecommons.org/licenses/by/3.0/">CC Attribution licence</A>.</P></DIV></DIV><DIV class="printfooter">
Retrieved from &quot;<A href="http://2012.igem.org/Team:UNITN-Trento/Protocols">http://2012.igem.org/Team:UNITN-Trento/Protocols</A>&quot;</DIV><DIV id="footer-box" class="noprint"><DIV id="footer"><UL id="f-list"><LI id="t-recentchanges"><A href="/Special:RecentChanges" title="Recent changes">Recent changes</A></LI><LI id="t-whatlinkshere"><A href="/Special:WhatLinksHere/Team:UNITN-Trento/Protocols" title="List of all wiki pages that link here [j]" accesskey="j">What links here</A></LI><LI id="t-recentchangeslinked"><A href="/Special:RecentChangesLinked/Team:UNITN-Trento/Protocols" title="Recent changes in pages linked from this page [k]" accesskey="k">Related changes</A></LI><LI id="t-specialpages"><A href="/Special:SpecialPages" title="List of all special pages [q]" accesskey="q">Special pages</A></LI><LI><A href="/Special:Preferences">My preferences</A></LI></UL></DIV><DIV id="footer"><UL id="f-list"><LI id="t-print"><A href="/wiki/index.php?title=Team:UNITN-Trento/Protocols&amp;printable=yes" title="Printable version of this page [p]" accesskey="p">Printable version</A></LI><LI id="t-permalink"><A href="/wiki/index.php?title=Team:UNITN-Trento/Protocols&amp;oldid=168588" title="Permanent link to this revision of the page">Permanent link</A></LI><LI id="privacy"><A href="/2012.igem.org:Privacy_policy" title="2012.igem.org:Privacy policy">Privacy policy</A></LI><LI id="disclaimer"><A href="/2012.igem.org:General_disclaimer" title="2012.igem.org:General disclaimer">Disclaimers</A></LI></UL></DIV></DIV></BODY></HTML>