<HTML xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en" dir="ltr">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki  ltr ns-0 ns-subject page-Team_Westminster_Protocols"><DIV id="globalWrapper"><DIV id="top-section"><DIV id="p-logo"><A href="/Main_Page" title="Main Page">&quot;
	    </A></DIV><DIV id="menubar" class="left-menu noprint"><UL><LI class="selected"><A href="/Team:Westminster/Protocols">Page               </A></LI><LI class="new"><A href="/wiki/index.php?title=Talk:Team:Westminster/Protocols&amp;action=edit&amp;redlink=1">Discussion               </A></LI><LI><A href="/wiki/index.php?title=Team:Westminster/Protocols&amp;action=edit">View source               </A></LI><LI><A href="/wiki/index.php?title=Team:Westminster/Protocols&amp;action=history">History               </A></LI><LI style="color:white;cursor:default">teams</LI></UL></DIV><DIV class="right-menu noprint" id="menubar"><UL><LI id="pt-login"><A href="/wiki/index.php?title=Special:UserLogin&amp;returnto=Team:Westminster/Protocols" title="You are encouraged to log in; however, it is not mandatory [o]" accesskey="o">Log in</A></LI></UL></DIV><DIV id="search-controls" class="noprint"><FORM action="/Special:Search" id="searchform"> </FORM></DIV></DIV><DIV id="content"><H1 class="firstHeading">Team:Westminster/Protocols</H1><DIV id="bodyContent"><H3 id="siteSub" class="noprint">From 2013.igem.org</H3><DIV class="navbar navbar-inverse navbar-fixed-top"><DIV class="container"><DIV class="navbar-collapse collapse"><UL class="nav navbar-nav"><LI class="dropdown"><A href="#" class="dropdown-toggle" data-toggle="dropdown">The Team </A></LI><LI><A href="https://igem.org/Team.cgi?year=2013&amp;team_name=Westminster">Official Team Page</A></LI><LI><A href="https://2013.igem.org/Team:Westminster/Team">Meet the Team Members</A></LI></UL><LI class="dropdown"><A href="#" class="dropdown-toggle" data-toggle="dropdown">Project </A></LI><LI><A href="https://2013.igem.org/Team:Westminster/Description">Project Description</A></LI><LI><A href="http://www.westminsterigem.org/2013/diary.html">Diary</A></LI><LI class="dropdown-header">Lab Work</LI><LI><A href="http://www.westminsterigem.org/2013/notebook.html">Notebook</A></LI><LI><A href="http://www.westminsterigem.org/2013/protocols.html">Protocols</A></LI><LI><A href="https://2013.igem.org/Team:Westminster/Safety">Safety</A></LI><LI><A href="https://2013.igem.org/Team:Westminster/Modelling">Modelling</A></LI><LI><A href="https://2013.igem.org/Team:Westminster/Parts">Parts</A></LI><LI><A href="http://www.westminsterigem.org/2013/outreach.html">Outreach</A></LI><LI><A href="https://2013.igem.org/Team:Westminster/Attributions">Attributions</A></LI></DIV></DIV></DIV><DIV id="wrap"><DIV class="container"><H1>PROTOCOLS</H1><DIV class="container"><H2>Making LB Agar plates</H2><OL start="1"><LI>Weigh 7.0g of LB agar.</LI><LI>Add 200ml of deionised water into a flask containing the weighed LB agar.</LI><LI>Mix well to dissolve the LB agar powder.</LI><LI>Autoclave the flask (remember to cover the flask with foil and sponge)</LI><LI>After autoclaving, store it in a clean dry space.</LI></OL><P>Re-heating agar to prepare plates.</P><OL start="1"><LI> Heat the agar in microwave till the LB agar has fully liquidised.</LI><LI>Let it cool.</LI><LI> Add required antibiotic in a sterile environment and pour the LB on plates.</LI><LI> </LI></OL><H2>Preparing competent cells –Top10 stock</H2><OL start="1"><LI>Pick single colony of cells from LB agar plate into 10 ml of LB media containing specific antibiotic for the cell type. Grow the culture overnight at 37 ˚C with shaking at 250rpm.</LI><LI>Inoculate 200ml of pre-warmed medium (no antibiotic or specific for the cell type) with 10ml of overnight cultures , and grow at 37 ˚C for 60 min, with vigorous shaking at 250rpm or until OD600 is 0.4-0.5</LI><LI>Put the flask on ice for 30min. At the same time chill sterile falcon tubes.</LI><LI>Aliquot culture into 50ml each 4 x 50ml chilled falcon tubes.</LI><LI>Harvest the cells by centrifugation for 7 minutes at 3500rpm, at 4 ˚C and discard the supernatant.</LI><LI>Re-suspend cells in each tube in 12.5ml of 0.1M MgCl2 </LI><LI>Centrifuge for 7 min at 3500rpm, at 4˚C and discard supernatant.</LI><LI>Re-suspend cells in each tube in 25ml of 0.1M CaCl2</LI><LI>Incubate cells on ice for 30 min.</LI><LI>Centrifuge for 7 min rpm, at 4˚C and discard supernatant.</LI><LI>Re-suspend cells in each tube in 700µl of 0.1M CaCl2 and 300µl of 50% glycerol to give a final volume of 1ml in each tube.</LI><LI>Aliquot 50µl into 1.5ml sterile micro centrifuge tubes on ice and store at -80˚C.</LI></OL><H2>Glycerol stock preparation</H2><H3>Materials</H3><P>-sterile 50% glycerol stock</P><P>300µl of 50% glycerol is added to 700µl overnight culture in an eppendorf.</P><P>Contents were mixed by inverting and immediately stored at -80˚C.</P><H2>50% Agarose gel preparation</H2><H2>Materials</H2><P>-0.5g of agarose weighed and added to 50ml of TAE buffer</P><P>• Add a few ml more of the 1X TAE since the buffer will evaporate in the microwave.</P><P>Dissolve the agarose in the TAE buffer in the microwave. Ensure all the agarose is completely dissolved. Allow to cool slightly (approx. 60C) and add 1µl of ethidium bromide. Pour gel into gel tray and place comb into position. Ensure there are no bubbles in the gel. Bubbles may be moved to the bottom of the gel using a sterile tip.  Allow gel to set completely. Run gel for approx. 1hour at 100volts.</P><P> </P><H1>Nanodrop procedure</H1><P>Add 1µl of RNAse free water as Blank</P><P>Add 1µl of sample to the stage and read.</P><P>Ideally want a concentration &gt;100ng/µl and purity (260/289nm) reading close to 1.0</P><H2>Preparation of Carbenicillin stock</H2><P>50mg/ml stock was prepared as follows:</P><P>1g of carbenicillin powder (Sigma product 50mg/ml) was weighed out and dissolved in 10 ml of sterile water. 1ml volumes were aliquotted to eppendorfs and marked with a red stripe and c.</P><H2>Preparation of carbenicillin Plates (100 µg /ml<SPAN style="font-size:11px">)</SPAN></H2><H3>Materials</H3><P>-petri dishes</P><P>-200ml autoclaved LB agar</P><P>50mg/ml<SPAN style="font-size:11px"> </SPAN>carbenicillin stock</P><P>Melt LB agar in microwave until completely molten. Allow to cool to approx 60C.</P><P>400µl of carbenicillin is added to the 200ml LB agar and approx 20ml poured into each plate.</P><P>Allow plates to cool and label the plates with red stripe on side to indicate the type of antibiotic.</P><P> </P><H1>Transformation</H1><H2>Materials</H2><P>- competent cells (1 vial per transformation and 1 for the control)</P><P>- antibiotic plates (2 per transformation)</P><P>- control plate without antibiotic</P><P>- control plasmid (PUC19)</P><P> </P><P>1. Remove plates from fridge and allow to warm and dry in 37C incubator.</P><P>2. Re-suspend plasmid DNA from iGEM plate by: (1) Pierce identified location on plate. (2) Add 10µl of sterile water and pipette up and down a few times. (3) Allow to stand for 5 mins.</P><P>3. Remove competent cells from -80C and leave on ice for 5 mins.  </P><P>4. Add 1µl of part DNA to competent cells and mix very gently.</P><P>5. Incubate preparation on ice for 5-20 mins.</P><P>6. Heat shock at 42C in a heat block (or water bath) for 45 seconds and immediately transfer to ice.</P><P>7. Incubate on ice for 5 mins - for chloramphenicol parts, incubate on ice for 20 mins.</P><P>8. Add 200µl of SOC or sterile LB broth and incubate at 37C in a shaking incubator for at least 1 hour.</P><P>9. Plate out 20µl and 230µl volumes to each LB antibiotic plate and spread using sterile beads.</P><P>10. Discard used beads into used beads waste, (beads are later cleaned and autoclaved).</P><P>11. Plate out control cells onto non-antibiotic plate to check viability of competent cells.</P><P>12. Incubate overnight at 37C and check for individual colonies</P><P> </P><H1>PCR Protocols</H1><P> </P><H2>Mutagenisis of the Chitinase genes</H2><P>In two separate tubes make up 50µl PCR reaction with DNA (plasmid prep of chiA), MgCl2 and dNTP and Pfu polymerase.</P><P>Tube A – ChiA-Fwd-Main and Chi-Rev-Muta</P><P>Tube B --  ChiA-Fwd-Muta and ChiA-Rev-Main</P><H3>Thermal cycling conditions</H3><P>95°C for 3 minutes followed by</P><P>30 cycles of</P><P>95°C for 30 seconds, 55 °C for 40 seconds, and 72 °C for</P><P>60 seconds (or 1min/kb), and a final step at 72 °C for 10 minutes.</P><P>After PCR – can run of gel to clean up – but may not be necessary. Set up a second PCR without any primers, but containing all other PCR components. Add 5µl from each reaction as DNA template and add 15µl of PCR mix (total volume of PCR =25µl.</P><P>Run PCR cycle as above for 10 cycles to generate mutated template.</P><P>After the 10 cycles –add a further 25µl of PCR mix – but this mix will contain the outer (flanking) primers to allow for amplification of the entire mutated sequence.</P><P>Run PCR as before for 30 cycles.</P><P>Do restriction digest – then run on gel and then gel purify.</P><P>Ligate to pSB1C3  backbone and transform.</P><P>Pick colonies and colony PCR and inoculate LB broth.</P><P>Select cloned colonies (from colony PCR) and grow those samples overnight in shaker.</P><P>Mini-prep and analyse by restriction analysis and send for sequencing</P><P> </P><H2>PCR MasterMix</H2><P>Make a master mix  X6 (6 reactions in total)</P><OL start="1"><LI>5x Q5 reaction 30µl</LI><LI>10mM dNTPs 3µl</LI><LI>DNA pol  1.5µl</LI><LI>dH2O 94.2µL.</LI></OL><P>For each reaction add:-   21.5µl master mix</P><P>                    1µl Template plasmid</P><P>                    1.25µl Forward primer</P><P>                    1.25µl reverse primer</P><H2>Chitin Azure Assay – the following was received from Prof. Frank Sargent regarding the chitin azure assay</H2><P>2.36g of Sodium Succinate was weighed out and dissolved in 200ml of water. The solution was pH’d with HCl to pH 6.09. Then 0.0337g of chitin azure was added to 50ml of Succinate assay buffer.</P><H3>Assay</H3><P>Grow culture in LB overnight. Samples are then centrifuged for 1 minute at 16000g and the supernatant collected and kept on ice. 200µl of supernatant is added to 400µl of chitin azure assay buffer and incubated at 37 oC for 72 h. The samples were then centrifuged at 16,000 g for 5 min and the A560 of the supernatant measured. Chitinolytic activity was measured as ∆A560 h-1 ml-1 per OD600 unit with respect to a blank incubated with just LB (not culture supernatant).</P></DIV><TABLE cellpadding="0" cellspacing="0"><TBODY><TR><TD><DIV class="container"><P> </P></DIV></TD><TD><DIV class="container"><P>Components</P></DIV></TD><TD><DIV class="container"><P>Quantity 1L</P></DIV></TD></TR><TR><TD><DIV class="container"><P> </P><P>Chitin Azure</P><P>Not sterilised</P><P>Buffer</P></DIV></TD><TD><DIV class="container"><P> </P><P>0.2% (w/v) chitin azure (Sigma)</P><P> </P><P>Succinate</P><P>Adjusted to pH 6 with NaOH</P><P> </P></DIV></TD><TD><DIV class="container"><P> </P><P>0.666 g</P><P> </P><P> </P><P>11.8 g</P></DIV></TD></TR></TBODY></TABLE><P> </P><P> </P><H3>References</H3><P>      Coulthurst, S. J., N. R. Williamson, et al. (2006). &quot;Metabolic and regulatory engineering of Serratia marcescens: mimicking phage-mediated horizontal acquisition of antibiotic biosynthesis and quorum-sensing capacities.&quot; Microbiology 152(Pt 7): 1899-1911.</P><DIV class="c25"><P> </P></DIV></DIV><DIV class="container"><FOOTER><CENTER><H5>Many thanks to our Sponsors</H5><P>by Westminster iGEM 2013</P></CENTER></FOOTER></DIV><DIV class="printfooter">
Retrieved from &quot;<A href="http://2013.igem.org/Team:Westminster/Protocols">http://2013.igem.org/Team:Westminster/Protocols</A>&quot;</DIV></DIV></DIV><DIV id="footer-box" class="noprint"><DIV id="footer"><UL id="f-list"><LI id="t-recentchanges"><A href="/Special:RecentChanges" title="Recent changes">Recent changes</A></LI><LI id="t-whatlinkshere"><A href="/Special:WhatLinksHere/Team:Westminster/Protocols" title="List of all wiki pages that link here [j]" accesskey="j">What links here</A></LI><LI id="t-recentchangeslinked"><A href="/Special:RecentChangesLinked/Team:Westminster/Protocols" title="Recent changes in pages linked from this page [k]" accesskey="k">Related changes</A></LI><LI id="t-specialpages"><A href="/Special:SpecialPages" title="List of all special pages [q]" accesskey="q">Special pages</A></LI><LI><A href="/Special:Preferences">My preferences</A></LI></UL></DIV><DIV id="footer"><UL id="f-list"><LI id="t-print"><A href="/wiki/index.php?title=Team:Westminster/Protocols&amp;printable=yes" title="Printable version of this page [p]" accesskey="p">Printable version</A></LI><LI id="t-permalink"><A href="/wiki/index.php?title=Team:Westminster/Protocols&amp;oldid=316908" title="Permanent link to this revision of the page">Permanent link</A></LI><LI id="privacy"><A href="/2013.igem.org:Privacy_policy" title="2013.igem.org:Privacy policy">Privacy policy</A></LI><LI id="disclaimer"><A href="/2013.igem.org:General_disclaimer" title="2013.igem.org:General disclaimer">Disclaimers</A></LI></UL></DIV></DIV></DIV></DIV></BODY></HTML>