<HTML xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en" dir="ltr">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki  ltr ns-0 ns-subject page-Team_Shenzhen_Project_YAO_Suicider"><DIV id="globalWrapper"><DIV id="top-section"><DIV id="p-logo"><A href="/Main_Page" title="Main Page">&quot;
	    </A></DIV><DIV id="menubar" class="left-menu noprint"><UL><LI class="selected"><A href="/Team:Shenzhen/Project/YAO.Suicider">Page               </A></LI><LI class="new"><A href="/wiki/index.php?title=Talk:Team:Shenzhen/Project/YAO.Suicider&amp;action=edit&amp;redlink=1">Discussion               </A></LI><LI><A href="/wiki/index.php?title=Team:Shenzhen/Project/YAO.Suicider&amp;action=edit">View source               </A></LI><LI><A href="/wiki/index.php?title=Team:Shenzhen/Project/YAO.Suicider&amp;action=history">History               </A></LI><LI style="color:#808080;cursor:default">teams</LI></UL></DIV><DIV class="right-menu noprint" id="menubar"><UL><LI id="pt-login"><A href="/wiki/index.php?title=Special:UserLogin&amp;returnto=Team:Shenzhen/Project/YAO.Suicider" title="You are encouraged to log in; however, it is not mandatory [o]" accesskey="o">Log in</A></LI></UL></DIV><DIV id="search-controls" class="noprint"><FORM action="/Special:Search" id="searchform"> </FORM></DIV></DIV><DIV id="content"><H1 class="firstHeading">Team:Shenzhen/Project/YAO.Suicider</H1><DIV id="bodyContent"><H3 id="siteSub" class="noprint">From 2012.igem.org</H3><DIV class="container"><DIV class="content"><UL class="ca-menu"><UL id="subs1"><LI id="ca-green1" class="ca-green"><A href="https://2012.igem.org/Team:Shenzhen/Team"><SPAN class="ca-icon">U</SPAN><DIV class="ca-content"><H2 class="ca-main">Team</H2></DIV></A></LI><DIV id="sub1" class="submenu"><LI class="ca-sub"><A href="https://2012.igem.org/Team:Shenzhen/Team">Consortium</A></LI><LI class="ca-sub"><A href="https://2012.igem.org/Team:Shenzhen/Team/BGI_College">BGI College</A></LI><LI class="ca-sub"><A href="https://2012.igem.org/Team:Shenzhen/Team/Instructors">Instructors</A></LI><LI class="ca-sub"><A href="https://2012.igem.org/Team:Shenzhen/Team/HUSTers">HUSTers</A></LI><LI class="ca-sub"><A href="https://2012.igem.org/Team:Shenzhen/Team/SCUTers">SCUTers</A></LI><LI class="ca-sub"><A href="https://2012.igem.org/Team:Shenzhen/Team/SCUers">SCUers</A></LI><LI class="ca-sub"><A href="https://2012.igem.org/Team:Shenzhen/Team/WHUers">WHUers</A></LI><LI class="ca-sub"><A href="https://2012.igem.org/Team:Shenzhen/Team/UESTCers">UESTCers</A></LI><LI class="ca-sub"><A href="https://2012.igem.org/Team:Shenzhen/Team/SEUers">SEUers</A></LI><LI class="ca-sub"><A href="https://2012.igem.org/Team:Shenzhen/Team/SCNUers">SCNUers</A></LI><LI class="ca-sub"><A href="https://2012.igem.org/Team:Shenzhen/Team/CUHKer">CUHKer</A></LI></DIV></UL><UL id="subs2"><LI id="ca-orange1" class="ca-orange"><A href="https://2012.igem.org/Team:Shenzhen/Project"><SPAN class="ca-icon">L</SPAN><DIV class="ca-content"><H2 class="ca-main">Project</H2></DIV></A></LI><DIV id="sub2" class="submenu"><LI class="ca-sub"><A href="https://2012.igem.org/Team:Shenzhen/Project">Stories</A></LI><LI class="ca-sub"><A href="https://2012.igem.org/Team:Shenzhen/Project/YAO.Genome">YAO.Genome</A></LI><LI class="ca-sub"><A href="https://2012.igem.org/Team:Shenzhen/Project/YAO.Channel">YAO.Channel</A></LI><LI class="ca-sub"><A href="https://2012.igem.org/Team:Shenzhen/Project/YAO.Sensor">YAO.Sensor</A></LI><LI class="ca-sub"><A href="https://2012.igem.org/Team:Shenzhen/Project/YAO.Suicider">YAO.Suicider</A></LI><LI class="ca-sub"><A href="https://2012.igem.org/Team:Shenzhen/Project/YAO.Factory">YAO.Factory</A></LI></DIV></UL><UL id="subs3"><LI id="ca-yellow1" class="ca-yellow"><A href="https://2012.igem.org/Team:Shenzhen/Result"><SPAN class="ca-icon">u</SPAN><DIV class="ca-content"><H2 class="ca-main">Result</H2></DIV></A></LI><DIV id="sub3" class="submenu"><LI class="ca-sub"><A href="https://2012.igem.org/Team:Shenzhen/Result">Summary</A></LI><LI class="ca-sub"><A href="https://2012.igem.org/Team:Shenzhen/Result/YAO.Genome">YAO.Genome</A></LI><LI class="ca-sub"><A href="https://2012.igem.org/Team:Shenzhen/Result/YAO.Channel">YAO.Channel</A></LI><LI class="ca-sub"><A href="https://2012.igem.org/Team:Shenzhen/Result/YAO.Sensor">YAO.Sensor</A></LI><LI class="ca-sub"><A href="https://2012.igem.org/Team:Shenzhen/Result/YAO.Suicider">YAO.Suicider</A></LI><LI class="ca-sub"><A href="https://2012.igem.org/Team:Shenzhen/Result/YAO.Factory">YAO.Factory</A></LI></DIV></UL><UL id="subs4"><LI id="ca-green2" class="ca-green"><A href="https://2012.igem.org/Team:Shenzhen/Biobricks"><SPAN class="ca-icon">S</SPAN><DIV class="ca-content"><H2 class="ca-main">BioBricks</H2></DIV></A></LI><DIV id="sub4" class="submenu"><LI class="ca-sub"><A href="https://2012.igem.org/Team:Shenzhen/Biobricks">Summary</A></LI><LI class="ca-sub"><A href="https://2012.igem.org/Team:Shenzhen/Biobricks/YAO.Genome">YAO.Genome</A></LI><LI class="ca-sub"><A href="https://2012.igem.org/Team:Shenzhen/Biobricks/YAO.Channel">YAO.Channel</A></LI><LI class="ca-sub"><A href="https://2012.igem.org/Team:Shenzhen/Biobricks/YAO.Sensor">YAO.Sensor</A></LI><LI class="ca-sub"><A href="https://2012.igem.org/Team:Shenzhen/Biobricks/YAO.Suicider">YAO.Suicider</A></LI><UL id="subs5"><LI id="ca-orange2" class="ca-orange"><A href="https://2012.igem.org/Team:Shenzhen/Notebook"><SPAN class="ca-icon">p</SPAN><DIV class="ca-content"><H2 class="ca-main">Notebook</H2></DIV></A></LI><DIV id="sub5" class="submenu"><LI class="ca-sub"><A href="https://2012.igem.org/Team:Shenzhen/Notebook">Team History</A></LI><LI class="ca-sub"><A href="https://2012.igem.org/Team:Shenzhen/Notebook/YAO.Genome">YAO.Genome</A></LI><LI class="ca-sub"><A href="https://2012.igem.org/Team:Shenzhen/Notebook/YAO.Channel">YAO.Channel</A></LI><LI class="ca-sub"><A href="https://2012.igem.org/Team:Shenzhen/Notebook/YAO.Sensor">YAO.Sensor</A></LI><LI class="ca-sub"><A href="https://2012.igem.org/Team:Shenzhen/Notebook/YAO.Suicider">YAO.Suicider</A></LI><LI class="ca-sub"><A href="https://2012.igem.org/Team:Shenzhen/Notebook/YAO.Factory">YAO.Factory</A></LI><UL id="subs6"><LI id="ca-green3" class="ca-green"><A href="https://2012.igem.org/Team:Shenzhen/Biosafety"><SPAN class="ca-icon">,</SPAN><DIV class="ca-content"><H2 class="ca-main">Biosafety</H2></DIV></A></LI></UL><UL id="subs7"><LI id="ca-yellow2" class="ca-yellow"><A href="https://2012.igem.org/Team:Shenzhen/Practices"><SPAN class="ca-icon">e</SPAN><DIV class="ca-content"><H2 class="ca-main">Practices</H2></DIV></A></LI><DIV id="sub7" class="submenu"><LI class="ca-sub"><A href="https://2012.igem.org/Team:Shenzhen/Practices">Summary</A></LI><LI class="ca-sub"><A href="https://2012.igem.org/Team:Shenzhen/Practices/Lectures">Lectures</A></LI><LI class="ca-sub"><A href="https://2012.igem.org/Team:Shenzhen/Practices/High_School">High School</A></LI><LI class="ca-sub"><A href="https://2012.igem.org/Team:Shenzhen/Practices/Teamwork">Teamwork</A></LI><UL id="subs8"><LI id="ca-orange3" class="ca-orange"><A href="https://2012.igem.org/Team:Shenzhen/Acknowledgment"><SPAN class="ca-icon">N</SPAN><DIV class="ca-content"><H2 class="ca-main">Acknow.</H2></DIV></A></LI><DIV id="sub8" class="submenu"><LI class="ca-sub"><A href="https://2012.igem.org/Team:Shenzhen/Acknowledgment">Summary</A></LI><LI class="ca-sub"><A href="https://2012.igem.org/Team:Shenzhen/Acknowledgment/Providers">Providers</A></LI><LI class="ca-sub"><A href="https://2012.igem.org/Team:Shenzhen/Acknowledgment/Advisors">Advisors</A></LI><LI class="ca-sub"><A href="https://2012.igem.org/Team:Shenzhen/Acknowledgment/Sponsors">Sponsors</A></LI><LI class="ca-sub"><A href="https://2012.igem.org/Team:Shenzhen/Acknowledgment/Home_Schools">Home Schools</A></LI></DIV><P><UL id="navigationMenu"><LI><A class="home" href="https://2012.igem.org/Team:Shenzhen/Project"><SPAN>Stories</SPAN></A></LI><LI><A class="genome" href="https://2012.igem.org/Team:Shenzhen/Project/YAO.Genome"><SPAN>YAO.Genome</SPAN></A></LI><LI><A class="channel" href="https://2012.igem.org/Team:Shenzhen/Project/YAO.Channel"><SPAN>YAO.Channel</SPAN></A></LI><LI><A class="sensor" href="https://2012.igem.org/Team:Shenzhen/Project/YAO.Sensor"><SPAN>YAO.Sensor</SPAN></A></LI><LI><A class="suicider" href="https://2012.igem.org/Team:Shenzhen/Project/YAO.Suicider"><SPAN>YAO.Suicider</SPAN></A></LI><LI><A class="houseyellow" href="https://2012.igem.org/Team:Shenzhen/Project/YAO.Factory"><SPAN>YAO.Factory</SPAN></A></LI></UL></P><DIV id="bodymain"><DIV class="context"><TABLE id="toc" class="toc"><TBODY><TR><TD><DIV id="toctitle"><H2>Contents</H2></DIV><UL><LI class="toclevel-1"><A href="#Introduction"><SPAN class="tocnumber">1</SPAN><SPAN class="toctext">Introduction</SPAN></A></LI><LI class="toclevel-1"><A href="#Amazing_Features:"><SPAN class="tocnumber">2</SPAN><SPAN class="toctext">Amazing Features:</SPAN></A></LI><LI class="toclevel-1"><A href="#Original_Ideas"><SPAN class="tocnumber">3</SPAN><SPAN class="toctext">Original Ideas</SPAN></A></LI><LI class="toclevel-1"><A href="#Final_Story"><SPAN class="tocnumber">4</SPAN><SPAN class="toctext">Final Story</SPAN></A></LI><LI class="toclevel-1"><A href="#Verification_Experiments_Design"><SPAN class="tocnumber">5</SPAN><SPAN class="toctext">Verification Experiments Design</SPAN></A></LI><LI class="toclevel-1"><A href="#References"><SPAN class="tocnumber">6</SPAN><SPAN class="toctext">References</SPAN></A></LI></UL></TD></TR></TBODY></TABLE><H5><SPAN class="mw-headline" id="Introduction">Introduction</SPAN></H5><UL><P>YAO is like a BOMBER and cell is like whoever plays with fire and get burst. Once the fuse is ignited, YAO sacrifices, and following cascade reaction will lead to the cell repression!</P></UL><UL><P>More than crippling of YAO by DNase I, we creatively couple it with apoptosis of yeast cell! 
</P></UL></DIV><DIV class="context"><H5><SPAN class="mw-headline" id="Amazing_Features:">Amazing Features:</SPAN></H5><UL><LI>1. Various Signaling?</LI></UL><UL><P>This module ships signals from nucleus to mitochondria directly or reversely in four distinct ways, including T7 RNAP/T7 promoter coupling, peptide signaling, RTG pathway and cyt C-mediated apoptosis !</P></UL><UL><LI>2. Biosafer?</LI></UL><UL><P>To guarantee biosafety, the engineered biofactory will die with its carrier cell ! </P></UL><UL><LI>3. Signal imitation? </LI></UL><UL><P>T7 RNAP/T7 promoter is employed to imitate the signal given to YAO instead of biosensor.</P></UL><UL><LI>4. Holin kill eukaryotic?</LI></UL><UL><P>We try to illustrate that holin can punch not only the cytomembrane of bacteria but also the outer membrane of mitochondria-like YAO and ER to release cytC.</P></UL><UL><LI>5. Special signaling from mitochondria to nucleus?</LI></UL><UL><P>RTG pathway tell the cell that mt “Emergency~ Emergency~”, so cell supply it some replenishment.</P></UL></DIV><DIV class="context"><H5><SPAN class="mw-headline" id="Original_Ideas">Original Ideas</SPAN></H5><UL><P>Originally, we try to construct such an artificial organelle YAO that YAO receives signal either from biosensor (internal) or from cytoplasm (external) or consequently trigger the host cell apoptosis. However, for YAO are not completed, we use mitochondrion as experiment target. And since we can’t rely on the availability of YAO.Sensor, the idea that Initial signal is given to YAO’s promoter is replaced by that nucleus produce T7 RNAP and transport it into mitochondrion to activate the T7 promoter inside it.</P></UL><UL><DIV class="figurep"><P>Figure 1. Original ideas.</P></DIV></UL></DIV><DIV class="context"><H5><SPAN class="mw-headline" id="Final_Story">Final Story</SPAN></H5><UL><P>The final story is consist of six parts:</P></UL><UL><LI>a. Signal Imitation</LI></UL><UL><P>After galactose is added to the medium, RNAP can bind to Gal promoter and start transcribe and produce T7 RNAP with signal peptide to mitochondrial matrix. Then T7 RNAP gets into mitochondrial matrix. </P></UL><UL><LI>b. Mitochondrion Dysfunction</LI></UL><UL><P>Once T7 promoter works, DNase is produced to fragmentize the mtDNA which can lead to Respiration difficulty.</P></UL><UL><LI>c. Retrograde Signaling from Mitochondrion</LI></UL><UL><P>Mitochondrial dysfunction can trigger the RTG pathway which is the only known signaling from mt to nucleus.</P></UL><UL><LI>d. Holin Production</LI></UL><UL><P>DLD3 promoter is the only stable downstream promoter relevant with RTG pathway. It can be binded and activated by RTG1/3p which is final production of RTG pathway. Once mitochondrial dysfunction exits, holin with DLD3promoter can be expressed.</P></UL><UL><LI>e. Holin Transformation and Destruction</LI></UL><UL><P>Once holin is produced, it can anchor on the outer membrane of mitochondria and ER, and form dimer with each other. Further more, they can form into a multidimer called dead draft if holin reachs a certain concentration. Dead drafts work like a large pore and release the apoptosis factor: cytochrome C, aif1 and calcium into cytoplasm.</P></UL><UL><LI>f. Apoptosis of Host</LI></UL><UL><P>Apoptosis factor cytochrome C and largely fleeing calcium can trigger the caspase-like pathway. Aif1 such as endo G can.get into nucleus and fragmentate DNA. Finally, host cell is suppressed or even die.</P></UL><UL><DIV class="figurep"><P>Figure 2. Final stories.</P></DIV></UL></DIV><DIV class="context"><H5><SPAN class="mw-headline" id="Verification_Experiments_Design">Verification Experiments Design</SPAN></H5><UL><LI>For T7 Imitation System</LI></UL><UL><DIV class="figurep"><P>Figure 3. Verify the function of T7 RNAP/Promoter system in cytoplasm. If it works, fluorescence of GFP can be detected in cytoplasm.</P></DIV></UL><UL><DIV class="figurep"><P>Figure 4. Verify the function of T7 RNAP/Promoter system in mitochondrion or unfinished YAO, basing on the availability of signal peptide from cytoplasm to mitochondrial matrix. If it works, fluorescence of mtGFP can be detected in mitochondrion.</P></DIV></UL><UL><LI>For Retrograde Signaling Part</LI></UL><UL><P>Because DLD3 is one of the target of RTG pathway of yeast, lowering the basal expression is required.</P></UL><UL><DIV class="figurep"><P>Figure 5. Verify the basal expression of gene downstream DLD3 promoter. Roughly judge the brightness of GFP, and consequently select a suitable degradation tag for gene downstream DLD3 promotor.</P></DIV></UL><UL><DIV class="figurep"><P>Figure 6. Verify the function of selected degradation tag by using UV as a imitation of DNase produced in mitochondrion that can lead to mitochondrion dysfunction which will drive the gene expression downstream DLD3 promoter through Retrograde signaling from mitochondrion to nucleus.</P></DIV></UL><UL><DIV class="figurep"><P>Figure 7. Verify that holing can make pores on the inner membrane of mitochondrion. This is the alterant of DNase causing mt dysfunction. And it can also produce a membrane without mtDNA.</P></DIV></UL><UL><LI>For Host Cell Suicide Part</LI></UL><UL><DIV class="figurep"><P>Figure 8. Verify the function of holing in yeast to prove that holing can act as bax/bak protein in human cell as a apoptosis activator. If it works, we can see that between the engineered and versus one, the growthrate curve (OD600) of engineered one tend to be lower than that of versus one.</P></DIV></UL></DIV><DIV class="context"><H5><SPAN class="mw-headline" id="References">References</SPAN></H5><UL><P>
1. Prindle, A. et al. A sensing array of radically coupled genetic ‘biopixels’. Nature. 2011;481:39-44.</P></UL><UL><P>2. Liu, Z. Butow, R. A. Mitochondrial retrograde signaling. Annu Rev Genet. 2006;40:159-85.</P></UL><UL><P>3. Shi, Y. Sun, J.Current advance in the topological structure and function of holin encoded by bacteriophage lambda. Wei Sheng Wu Xue Bao. 2012 ;52:141-5.</P></UL><UL><P>4. Agu, C. A et al. Bacteriophage-encoded toxins: the lambda-holin protein causes caspase-independent non-apoptotic cell death of eukaryotic cells. Cell Microbiol. 2007:1753. Lipton, S. A.</P></UL><UL><P>5. Bossy-Wetzel, E. Dueling activities of AIF in cell death versus survival: DNA binding and redox activity. Cell. 2002;111:147-50.</P></UL><UL><P>6. Buttner, S. Endonuclease G regulates budding yeast life and death. Mol Cell. 2007 ;25:233-46.</P></UL><UL><P>7. Spencer, S. L. Sorger, P. K. Measuring and modeling apoptosis in single cells. Cell. 2011;144:926-39.</P></UL><UL><P>8. Li, P. Nijhawan, D. Wang, X. Mitochondrial activation of apoptosis. Cell. 2004;116:S57-9.</P></UL><UL><P>9. Pinkham, J. L. Dudley, A. M. Mason, T. L. T7 RNA polymerase-dependent expression of COXII in yeast mitochondria. Mol Cell Biol. 1994 ;14:4643-52.</P></UL></DIV></DIV><DIV id="gallery"><DIV class="printfooter">
Retrieved from &quot;<A href="http://2012.igem.org/Team:Shenzhen/Project/YAO.Suicider">http://2012.igem.org/Team:Shenzhen/Project/YAO.Suicider</A>&quot;</DIV></DIV><DIV id="footer-box" class="noprint"><DIV id="footer"><UL id="f-list"><LI id="t-recentchanges"><A href="/Special:RecentChanges" title="Recent changes">Recent changes</A></LI><LI id="t-whatlinkshere"><A href="/Special:WhatLinksHere/Team:Shenzhen/Project/YAO.Suicider" title="List of all wiki pages that link here [j]" accesskey="j">What links here</A></LI><LI id="t-recentchangeslinked"><A href="/Special:RecentChangesLinked/Team:Shenzhen/Project/YAO.Suicider" title="Recent changes in pages linked from this page [k]" accesskey="k">Related changes</A></LI><LI id="t-specialpages"><A href="/Special:SpecialPages" title="List of all special pages [q]" accesskey="q">Special pages</A></LI><LI><A href="/Special:Preferences">My preferences</A></LI></UL></DIV><DIV id="footer"><UL id="f-list"><LI id="t-print"><A href="/wiki/index.php?title=Team:Shenzhen/Project/YAO.Suicider&amp;printable=yes" title="Printable version of this page [p]" accesskey="p">Printable version</A></LI><LI id="t-permalink"><A href="/wiki/index.php?title=Team:Shenzhen/Project/YAO.Suicider&amp;oldid=207469" title="Permanent link to this revision of the page">Permanent link</A></LI><LI id="privacy"><A href="/2012.igem.org:Privacy_policy" title="2012.igem.org:Privacy policy">Privacy policy</A></LI><LI id="disclaimer"><A href="/2012.igem.org:General_disclaimer" title="2012.igem.org:General disclaimer">Disclaimers</A></LI></UL></DIV></DIV></UL></DIV></UL></DIV></UL></DIV></UL></UL></DIV></DIV></DIV></DIV></DIV></BODY></HTML>