<HTML xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en" dir="ltr">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki  ltr ns-0 ns-subject page-Team_Shenzhen_Result_YAO_Suicider"><DIV id="globalWrapper"><DIV id="top-section"><DIV id="p-logo"><A href="/Main_Page" title="Main Page">&quot;
	    </A></DIV><DIV id="menubar" class="left-menu noprint"><UL><LI class="selected"><A href="/Team:Shenzhen/Result/YAO.Suicider">Page               </A></LI><LI class="new"><A href="/wiki/index.php?title=Talk:Team:Shenzhen/Result/YAO.Suicider&amp;action=edit&amp;redlink=1">Discussion               </A></LI><LI><A href="/wiki/index.php?title=Team:Shenzhen/Result/YAO.Suicider&amp;action=edit">View source               </A></LI><LI><A href="/wiki/index.php?title=Team:Shenzhen/Result/YAO.Suicider&amp;action=history">History               </A></LI><LI style="color:#808080;cursor:default">teams</LI></UL></DIV><DIV class="right-menu noprint" id="menubar"><UL><LI id="pt-login"><A href="/wiki/index.php?title=Special:UserLogin&amp;returnto=Team:Shenzhen/Result/YAO.Suicider" title="You are encouraged to log in; however, it is not mandatory [o]" accesskey="o">Log in</A></LI></UL></DIV><DIV id="search-controls" class="noprint"><FORM action="/Special:Search" id="searchform"> </FORM></DIV></DIV><DIV id="content"><H1 class="firstHeading">Team:Shenzhen/Result/YAO.Suicider</H1><DIV id="bodyContent"><H3 id="siteSub" class="noprint">From 2012.igem.org</H3><DIV class="container"><DIV class="content"><UL class="ca-menu"><UL id="subs1"><LI id="ca-green1" class="ca-green"><A href="https://2012.igem.org/Team:Shenzhen/Team"><SPAN class="ca-icon">U</SPAN><DIV class="ca-content"><H2 class="ca-main">Team</H2></DIV></A></LI><DIV id="sub1" class="submenu"><LI class="ca-sub"><A href="https://2012.igem.org/Team:Shenzhen/Team">Consortium</A></LI><LI class="ca-sub"><A href="https://2012.igem.org/Team:Shenzhen/Team/BGI_College">BGI College</A></LI><LI class="ca-sub"><A href="https://2012.igem.org/Team:Shenzhen/Team/Instructors">Instructors</A></LI><LI class="ca-sub"><A href="https://2012.igem.org/Team:Shenzhen/Team/HUSTers">HUSTers</A></LI><LI class="ca-sub"><A href="https://2012.igem.org/Team:Shenzhen/Team/SCUTers">SCUTers</A></LI><LI class="ca-sub"><A href="https://2012.igem.org/Team:Shenzhen/Team/SCUers">SCUers</A></LI><LI class="ca-sub"><A href="https://2012.igem.org/Team:Shenzhen/Team/WHUers">WHUers</A></LI><LI class="ca-sub"><A href="https://2012.igem.org/Team:Shenzhen/Team/UESTCers">UESTCers</A></LI><LI class="ca-sub"><A href="https://2012.igem.org/Team:Shenzhen/Team/SEUers">SEUers</A></LI><LI class="ca-sub"><A href="https://2012.igem.org/Team:Shenzhen/Team/SCNUers">SCNUers</A></LI><LI class="ca-sub"><A href="https://2012.igem.org/Team:Shenzhen/Team/CUHKer">CUHKer</A></LI></DIV></UL><UL id="subs2"><LI id="ca-orange1" class="ca-orange"><A href="https://2012.igem.org/Team:Shenzhen/Project"><SPAN class="ca-icon">L</SPAN><DIV class="ca-content"><H2 class="ca-main">Project</H2></DIV></A></LI><DIV id="sub2" class="submenu"><LI class="ca-sub"><A href="https://2012.igem.org/Team:Shenzhen/Project">Stories</A></LI><LI class="ca-sub"><A href="https://2012.igem.org/Team:Shenzhen/Project/YAO.Genome">YAO.Genome</A></LI><LI class="ca-sub"><A href="https://2012.igem.org/Team:Shenzhen/Project/YAO.Channel">YAO.Channel</A></LI><LI class="ca-sub"><A href="https://2012.igem.org/Team:Shenzhen/Project/YAO.Sensor">YAO.Sensor</A></LI><LI class="ca-sub"><A href="https://2012.igem.org/Team:Shenzhen/Project/YAO.Suicider">YAO.Suicider</A></LI><LI class="ca-sub"><A href="https://2012.igem.org/Team:Shenzhen/Project/YAO.Factory">YAO.Factory</A></LI></DIV></UL><UL id="subs3"><LI id="ca-yellow1" class="ca-yellow"><A href="https://2012.igem.org/Team:Shenzhen/Result"><SPAN class="ca-icon">u</SPAN><DIV class="ca-content"><H2 class="ca-main">Result</H2></DIV></A></LI><DIV id="sub3" class="submenu"><LI class="ca-sub"><A href="https://2012.igem.org/Team:Shenzhen/Result">Summary</A></LI><LI class="ca-sub"><A href="https://2012.igem.org/Team:Shenzhen/Result/YAO.Genome">YAO.Genome</A></LI><LI class="ca-sub"><A href="https://2012.igem.org/Team:Shenzhen/Result/YAO.Channel">YAO.Channel</A></LI><LI class="ca-sub"><A href="https://2012.igem.org/Team:Shenzhen/Result/YAO.Sensor">YAO.Sensor</A></LI><LI class="ca-sub"><A href="https://2012.igem.org/Team:Shenzhen/Result/YAO.Suicider">YAO.Suicider</A></LI><LI class="ca-sub"><A href="https://2012.igem.org/Team:Shenzhen/Result/YAO.Factory">YAO.Factory</A></LI></DIV></UL><UL id="subs4"><LI id="ca-green2" class="ca-green"><A href="https://2012.igem.org/Team:Shenzhen/Biobricks"><SPAN class="ca-icon">S</SPAN><DIV class="ca-content"><H2 class="ca-main">BioBricks</H2></DIV></A></LI><DIV id="sub4" class="submenu"><LI class="ca-sub"><A href="https://2012.igem.org/Team:Shenzhen/Biobricks">Summary</A></LI><LI class="ca-sub"><A href="https://2012.igem.org/Team:Shenzhen/Biobricks/YAO.Genome">YAO.Genome</A></LI><LI class="ca-sub"><A href="https://2012.igem.org/Team:Shenzhen/Biobricks/YAO.Channel">YAO.Channel</A></LI><LI class="ca-sub"><A href="https://2012.igem.org/Team:Shenzhen/Biobricks/YAO.Sensor">YAO.Sensor</A></LI><LI class="ca-sub"><A href="https://2012.igem.org/Team:Shenzhen/Biobricks/YAO.Suicider">YAO.Suicider</A></LI><UL id="subs5"><LI id="ca-orange2" class="ca-orange"><A href="https://2012.igem.org/Team:Shenzhen/Notebook"><SPAN class="ca-icon">p</SPAN><DIV class="ca-content"><H2 class="ca-main">Notebook</H2></DIV></A></LI><DIV id="sub5" class="submenu"><LI class="ca-sub"><A href="https://2012.igem.org/Team:Shenzhen/Notebook">Team History</A></LI><LI class="ca-sub"><A href="https://2012.igem.org/Team:Shenzhen/Notebook/YAO.Genome">YAO.Genome</A></LI><LI class="ca-sub"><A href="https://2012.igem.org/Team:Shenzhen/Notebook/YAO.Channel">YAO.Channel</A></LI><LI class="ca-sub"><A href="https://2012.igem.org/Team:Shenzhen/Notebook/YAO.Sensor">YAO.Sensor</A></LI><LI class="ca-sub"><A href="https://2012.igem.org/Team:Shenzhen/Notebook/YAO.Suicider">YAO.Suicider</A></LI><LI class="ca-sub"><A href="https://2012.igem.org/Team:Shenzhen/Notebook/YAO.Factory">YAO.Factory</A></LI><UL id="subs6"><LI id="ca-green3" class="ca-green"><A href="https://2012.igem.org/Team:Shenzhen/Biosafety"><SPAN class="ca-icon">,</SPAN><DIV class="ca-content"><H2 class="ca-main">Biosafety</H2></DIV></A></LI></UL><UL id="subs7"><LI id="ca-yellow2" class="ca-yellow"><A href="https://2012.igem.org/Team:Shenzhen/Practices"><SPAN class="ca-icon">e</SPAN><DIV class="ca-content"><H2 class="ca-main">Practices</H2></DIV></A></LI><DIV id="sub7" class="submenu"><LI class="ca-sub"><A href="https://2012.igem.org/Team:Shenzhen/Practices">Summary</A></LI><LI class="ca-sub"><A href="https://2012.igem.org/Team:Shenzhen/Practices/Lectures">Lectures</A></LI><LI class="ca-sub"><A href="https://2012.igem.org/Team:Shenzhen/Practices/High_School">High School</A></LI><LI class="ca-sub"><A href="https://2012.igem.org/Team:Shenzhen/Practices/Teamwork">Teamwork</A></LI><UL id="subs8"><LI id="ca-orange3" class="ca-orange"><A href="https://2012.igem.org/Team:Shenzhen/Acknowledgment"><SPAN class="ca-icon">N</SPAN><DIV class="ca-content"><H2 class="ca-main">Acknow.</H2></DIV></A></LI><DIV id="sub8" class="submenu"><LI class="ca-sub"><A href="https://2012.igem.org/Team:Shenzhen/Acknowledgment">Summary</A></LI><LI class="ca-sub"><A href="https://2012.igem.org/Team:Shenzhen/Acknowledgment/Providers">Providers</A></LI><LI class="ca-sub"><A href="https://2012.igem.org/Team:Shenzhen/Acknowledgment/Advisors">Advisors</A></LI><LI class="ca-sub"><A href="https://2012.igem.org/Team:Shenzhen/Acknowledgment/Sponsors">Sponsors</A></LI><LI class="ca-sub"><A href="https://2012.igem.org/Team:Shenzhen/Acknowledgment/Home_Schools">Home Schools</A></LI></DIV><P><UL id="navigationMenu"><LI><A class="home" href="https://2012.igem.org/Team:Shenzhen/Result"><SPAN>Summary</SPAN></A></LI><LI><A class="genome" href="https://2012.igem.org/Team:Shenzhen/Result/YAO.Genome"><SPAN>YAO.Genome</SPAN></A></LI><LI><A class="channel" href="https://2012.igem.org/Team:Shenzhen/Result/YAO.Channel"><SPAN>YAO.Channel</SPAN></A></LI><LI><A class="sensor" href="https://2012.igem.org/Team:Shenzhen/Result/YAO.Sensor"><SPAN>YAO.Sensor</SPAN></A></LI><LI><A class="suicider" href="https://2012.igem.org/Team:Shenzhen/Result/YAO.Suicider"><SPAN>YAO.Suicider</SPAN></A></LI><LI><A class="houseyellow" href="https://2012.igem.org/Team:Shenzhen/Result/YAO.Factory"><SPAN>YAO.Factory</SPAN></A></LI></UL></P><DIV id="bodymain"><DIV class="context"><TABLE id="toc" class="toc"><TBODY><TR><TD><DIV id="toctitle"><H2>Contents</H2></DIV><UL><LI class="toclevel-1"><A href="#Result_Summary"><SPAN class="tocnumber">1</SPAN><SPAN class="toctext">Result Summary</SPAN></A></LI><LI class="toclevel-1"><A href="#Preview_for_Whole_Experiments"><SPAN class="tocnumber">2</SPAN><SPAN class="toctext">Preview for Whole Experiments</SPAN></A></LI><LI class="toclevel-1"><A href="#Identification_for_Engineering_GAL1_Promoter_without_Downstream_ATG_Start_Codon"><SPAN class="tocnumber">3</SPAN><SPAN class="toctext">Identification for Engineering GAL1 Promoter without Downstream ATG Start Codon</SPAN></A></LI><LI class="toclevel-1"><A href="#Identification_for_Engineering_Lambda_Holin_into_BBa_Format"><SPAN class="tocnumber">4</SPAN><SPAN class="toctext">Identification for Engineering Lambda Holin into BBa Format</SPAN></A></LI><LI class="toclevel-1"><A href="#Identification_for_Bioparts"><SPAN class="tocnumber">5</SPAN><SPAN class="toctext">Identification for Bioparts</SPAN></A></LI><LI class="toclevel-1"><A href="#Protocol:_Point_mutation_to_GAL1_Promoter"><SPAN class="tocnumber">6</SPAN><SPAN class="toctext">Protocol: Point mutation to GAL1 Promoter</SPAN></A></LI><LI class="toclevel-1"><A href="#Protocol:_Test_of_T7_RNAP_Function_in_Nucleus"><SPAN class="tocnumber">7</SPAN><SPAN class="toctext">Protocol: Test of T7 RNAP Function in Nucleus</SPAN></A></LI><LI class="toclevel-1"><A href="#Protocol:_Holin_Adaptor_Engineering"><SPAN class="tocnumber">8</SPAN><SPAN class="toctext">Protocol: Holin Adaptor Engineering</SPAN></A></LI></UL></TD></TR></TBODY></TABLE><H5><SPAN class="mw-headline" id="Result_Summary">Result Summary</SPAN></H5><UL><P>In the experiment part of suicide, we have succefully engineered GAL1 promoter, lambda phage holin, and linked GAL1 promoter with yeast GFP, yeast RFP and lambda phage holin respectively.</P></UL></DIV><DIV class="context"><H5><SPAN class="mw-headline" id="Preview_for_Whole_Experiments">Preview for Whole Experiments</SPAN></H5><UL><P>    GAL1 promoter + kozak sequence + signal peptide to mitochondria + T7 RNAP + ADH terminator (backbone:YEP325)</P></UL><UL><P>    T7 promoter + mt RBS + DNase I + terminator for mitochondria (under experiment)</P></UL><UL><P>    DLD3 promoter + kozak sequence + Holin + degradation tag + ADH terminator (under experiment)</P></UL></DIV><DIV class="context"><H5><SPAN class="mw-headline" id="Identification_for_Engineering_GAL1_Promoter_without_Downstream_ATG_Start_Codon">Identification for Engineering GAL1 Promoter without Downstream ATG Start Codon</SPAN></H5><UL><P>0. This is based on BBa_J63006 and is without ATG start codon. We point-mutate the ATG to ACG. It is more convenient to use. There is ATG in GAL1 promoter RBS which can change read frame.</P></UL><UL><P>1. &gt;BBa_J63006 Part-only sequence (549 bp)</P></UL><UL><P>ccccattatcttagcctaaaaaaaccttctctttggaactttcagtaatacgcttaactgctcattgctatattgaagtacggattagaagccgccgagcgggtgacagccctccgaaggaagactctcctccgtgcgtcctcgtcttcaccggtcgcgttcctgaaacgcagatgtgcctcgcgccgcactgctccgaacaataaagattctacaatactagcttttatggttatgaagaggaaaaattggcagtaacctggccccacaaaccttcaaatgaacgaatcaaattaacaaccataggatgataatgcgattagttttttagccttatttctggggtaattaatcagcgaagcgatgatttttgatctattaacagatatataaatgcaaaaactgcataaccactttaactaatactttcaacattttcggtttgtattacttcttattcaaatgtaataaaagtatcaacaaaaaattgttaatatacctctatactttaacgtcaaggaggaaactagacccgccgccacc&lt;red&gt;atg&lt;/red&gt;gag</P></UL><UL><P>The backbone for it BBa_J63010 is for combinant protein.</P></UL><UL><P>2. &gt;Engineered GAL1(gz-GAL1)</P></UL><UL><P>ccccattatcttagcctaaaaaaaccttctctttggaactttcagtaatacgcttaactgctcattgctatattgaagtacggattagaagccgccgagcgggtgacagccctccgaaggaagactctcctccgtgcgtcctcgtcttcaccggtcgcgttcctgaaacgcagatgtgcctcgcgccgcactgctccgaacaataaagattctacaatactagcttttatggttatgaagaggaaaaattggcagtaacctggccccacaaaccttcaaatgaacgaatcaaattaacaaccataggatgataatgcgattagttttttagccttatttctggggtaattaatcagcgaagcgatgatttttgatctattaacagatatataaatgcaaaaactgcataaccactttaactaatactttcaacattttcggtttgtattacttcttattcaaatgtaataaaagtatcaacaaaaaattgttaatatacctctatactttaacgtcaaggaggaaactagacccgccgccacc&lt;red&gt;acg&lt;/red&gt;gag</P></UL><UL><P>The backbone for it is T vector, it can be linked to PSB1A3 or PSB1A2 in standard BioBrick format.</P></UL><UL><P>3. Primer for engineering:</P></UL><UL><P>PGALF1 5'-GTTTCTTCGAATTCGCGGCCGCTTCTAGAGCCCCATTATCTTAGCCTA</P><P>PGALR1 5'-GTTTCTTCCTGCAGCGGCCGCTACTAGTACTC&lt;red&gt;CGT&lt;/red&gt;GGTGGCGGCGGGTC</P></UL><UL><P>4. GAL 1 Promoter Construction Identification:</P></UL><UL><DIV class="figurep"><P>Figure 1. Sanger sequencing diagrams for GAL1 Promoter mutant. </P></DIV></UL><UL><DIV class="figurep"><P>Figure 2. Sequencing result comparison diagram. The cyan one means BBa standard prefix and suffix from standard part fabrication. The black one means GAL 1 promoter, and ATG start codon in the final is replaced by ACG.</P></DIV></UL><UL><P>As learn from the diagram, we have successfully constructed the GAL 1 promoter without ATG.</P></UL></DIV><DIV class="context"><H5><SPAN class="mw-headline" id="Identification_for_Engineering_Lambda_Holin_into_BBa_Format">Identification for Engineering Lambda Holin into BBa Format</SPAN></H5><UL><P>0. Due to that BBa_K112306 (lambda holin) is in BBb format and is inconvenient to use. We change it to BBa format. This is used to form pores on the outer membrane of mitochondria and ER to release apoptosis factor to repress yeast cells.</P></UL><UL><P>1. Origin holin and its adaptor:</P></UL><UL><P>GAATTCatgAGATCT</P></UL><UL><P>Atgccagaaaaacatgacctgttggccgccattctcgcggcaaaggaacaaggcatcggggcaatccttgcgtttgcaatggcgtaccttcgcggcagatataatggcggtgcgtttacaaaaacagtaatcgacgcaacgatgtgcgccattatcgcctggttcattcgtgaccttctcgacttcgccggactaagtagcaatctcgcttatataacgagcgtgtttatcggctacatcggtactgactcgattggttcgcttatcaaacgcttcgctgctaaaaaagccggagtagaagatggtagaaatcaataa</P></UL><UL><P>GGATCCtaaCTCGAG</P></UL><UL><P>The backbone for it BBa_J63010 is for combinant protein.</P></UL><UL><P>2. &gt;Engineered holin and its adaptor:</P></UL><UL><P>GTTTCTT C GAATTC GCGGCCGC T TCTAG ag</P></UL><UL><P>Atgccagaaaaacatgacctgttggccgccattctcgcggcaaaggaacaaggcatcggggcaatccttgcgtttgcaatggcgtaccttcgcggcagatataatggcggtgcgtttacaaaaacagtaatcgacgcaacgatgtgcgccattatcgcctggttcattcgtgaccttctcgacttcgccggactaagtagcaatctcgcttatataacgagcgtgtttatcggctacatcggtactgactcgattggttcgcttatcaaacgcttcgctgctaaaaaagccggagtagaagatggtagaaatcaataa</P></UL><UL><P>T ACTAGT A GCGGCCG CTGCAG G AAGAAAC</P></UL><UL><P>The backbone for it is T vector, it can be linked to PSB1A3 or PSB1A2 in standard BioBrick format.</P></UL><UL><P>3. Primer for engineering:</P></UL><UL><P>Holin-F GTTTCTTCGA ATTCGCGGCC GCTTCTAGAG ATGCCAGAAA AACATGACCT</P><P>Holin-R GTTTCTTCCT GCAGCGGCCG CTACTAGTAT TATTGATTTC TACCATCTT</P></UL><UL><P>4. Holin Construction Identification:</P></UL><UL><DIV class="figurep"><P>Figure 3. Sanger sequencing diagrams for Lambda Holin modification. </P></DIV></UL><UL><DIV class="figurep"><P>Figure 4. Sequencing result comparison diagram. The cyan one means BBa standard prefix and suffix from standard part fabrication. The black one means the coding gene of lambda phage holin.</P></DIV></UL><UL><P>As learn from the diagram, we have successfully constructed the Lambda holin in BBa format.</P></UL></DIV><DIV class="context"><H5><SPAN class="mw-headline" id="Identification_for_Bioparts">Identification for Bioparts</SPAN></H5><UL><LI>GAL1 promoter +E0030(GFP)</LI></UL><UL><DIV class="figurep"><P>Figure 5. Sequencing result comparison diagram for GAL1 promoter +E0030(GFP). </P></DIV></UL><UL><LI>GAL1 promoter +E1010(RFP)</LI></UL><UL><DIV class="figurep"><P>Figure 6. Sequencing result comparison diagram for GAL1 promoter +E1010(RFP).</P></DIV></UL><UL><LI>GAL1 promoter +lamda holin</LI></UL><UL><DIV class="figurep"><P>Figure 7. Sequencing result comparison diagram for GAL1 promoter +lamda holin.</P></DIV></UL></DIV><DIV class="context"><H5><SPAN class="mw-headline" id="Protocol:_Point_mutation_to_GAL1_Promoter">Protocol: Point mutation to GAL1 Promoter</SPAN></H5><UL><P>1. Plasmid extraction of GAL1 promoter (BBa-J63005)</P></UL><UL><P>2. Point mutation:</P><P>    Primers</P><P>        &gt; PGALF1   5'-GTTTCTTCGAATTCGCGGCCGCTTCTAGAGCCCCATTATCTTAGCCTA</P><P>        &gt; PGALR1   5'-GTTTCTTCCTGCAGCGGCCGCTACTAGTACTCCGTGGTGGCGGCGGGTC</P><P>    BBa-J63005    20ng</P><P>    Ex Taq    0.5ul</P><P>    10 X Ex Buffer    2ul</P><P>    Dntp Mix    1ul</P><P>    PGALF    0.5ul</P><P>    PGALR1    0.5ul</P><P>    ddH2O    20ul</P><P>    94℃    3min</P><P>    94℃    30s</P><P>    55℃    30s</P><P>    72℃    30s</P><P>    72℃    10min</P><P>    12℃    ∞</P><P>    30 cycles</P></UL><UL><P>3. Gel electrophoresis</P></UL><UL><DIV class="figurep"><P>Figure 8.  Channel 1: Marker100bp  Channel 2: PCR result </P></DIV></UL><UL><P>4. Gel recovery(QIAGEN)</P></UL><UL><P>5. Digestion:</P><P>    gz-GALproduction    17ul</P><P>    PSB1A2linear backbone    400ng</P><P>    EcoRI    0.5ul</P><P>    EcoRI    0.5ul</P><P>    PstI    0.5ul</P><P>    PstI    0.5ul</P><P>    10 x H Buffer    2ul</P><P>    10 x H Buffer    2ul</P><P>    ddH2O    20ul</P><P>    37oC X 1h, 80oC X  20min</P></UL><UL><P>6. Tranformation:</P><P>    As above.</P></UL><UL><P>7. Colony PCR:</P><P>    As above.</P><P>    Prefix</P><P>        5' GTTTCTT C GAATTC GCGGCCGC  T  TCTAGA  G   [part] 3'</P><P>        3' CAAAGAA G CTTAAG CGCCGGCG  A  AGATCT  C   [part] 5'</P><P>    Suffix</P><P>        5' [part] T ACTAGT  A  GCGGCCG CTGCAG G AAGAAAC   3'</P><P>        3' [part] A TGATCA  T  CGCCGGC GACGTC C TTCTTTG   5'</P></UL><UL><DIV class="figurep"><P>Figure 9.  Channel 1:maker(100-6000）  Channel 2-6:pcr result for sample</P></DIV></UL><UL><P>8. Sequencing</P></UL></DIV><DIV class="context"><H5><SPAN class="mw-headline" id="Protocol:_Test_of_T7_RNAP_Function_in_Nucleus">Protocol: Test of T7 RNAP Function in Nucleus</SPAN></H5><UL><LI>I. Construction of Vector </LI></UL><UL><P>1. Bacterial culture (BBa_I712074（T7 poromter，Amp resistant or Kana resistant），E0840（GFP generator，Amp resistant）</P></UL><UL><P>2. Plasmid extraction for BBa_I712074，E0840( Tiangen normal plasmid extraction kits</P></UL><UL><P>3. Measure the concentration of plasmid by NANODROP 2000(Thermo)</P></UL><UL><P>4. Digestion: (enzymes are all from Takara </P><P>    BBa_I712074    2-3ug</P><P>    PstI    2ul</P><P>    SpeI    2ul</P><P>    10 x H Buffer    5ul</P><P>    ddH2O    50ul</P><P>    E0840    2-3ug</P><P>    XbaI    2ul</P><P>    PstI    2ul</P><P>    10 x H Buffer    5ul</P><P>    ddH2O    50ul</P><P>    37oC    3h</P><P>    80 oC     20min</P></UL><UL><P>5. Electrophoresis: </P><P>    1% agarose gel, 120V, after 45min,EB dye. </P></UL><UL><P>6. Gel recovery of GAL(EcoRI/SpeI), Holin(XbaI/PstI) (QIAquick Gel recovery kits)</P></UL><UL><P>7. Measure the concentration of fragment of interest by NANODROP 2000(Thermo)</P></UL><UL><P>8. Linage: </P><P>    BBa_I712074 （(stI/SpeI)    2ul</P><P>    E0840 (PstI/XbaI)    6ul</P><P>    T4 ligase    1ul</P><P>    T4 ligase Buffer    1ul</P><P>    16 oC    3 hours</P></UL><UL><P>9. Transduction: </P><P>    1. UV sterilization half an hour before, get DH5a (competent yeast) out of fridge, thawing in water bath. </P><P>    2. 10μlinkage production with 50μl competent yeast cells, ice-bath for 30mins. </P><P>    3. 42oC ice-bath heat shock for 50s. </P><P>    4. Stewing on ice for 2mins. </P><P>    5. Add 500μl LB medium without antibiotic, 37℃ shaking(225rpm/min), recover it for culturing it for 1 hour. </P><P>    6. 3000rpm/min for 5mins, abandon the supermate, mixing the left. </P><P>    7. 60μl broth spread the Amp anti-LB solid plat, 37℃ inversed culture in incubation overnight. </P></UL><UL><P>10. Colony PCR: </P><P>    dNTPmix(2.5mM)    1μl</P><P>    PCR 10×buffer2μl</P><P>    VF    0.5μl</P><P>    VR    0.5μl</P><P>    Ex Taq    0.5μl</P><P>    H2O    15.5μl</P><P>    Primer</P><P>        VF:    TGCCACCTGACGTCTAAGAA</P><P>        VR:    ATTACCGCCTTTGAGTGAGC</P><P>    94℃    3min</P><P>    94℃    30s</P><P>    55℃    30s </P><P>    72℃    1min</P><P>    72℃    10min</P><P>    12℃    ∞</P><P>    30 cycles</P></UL><UL><P>11. 1%~1.5% agarose gel electrophoresis, &lt;130V, 40mins. </P></UL><UL><P>12. Sequencing: </P><P>    Pick the positive colony(colony PCR verification), marking lines and culture(Cmr resistant), sequencing. Name the right vector PSB1AK3-T7 poromter+GFP+T7 terminater</P></UL><UL><LI>II. Construction of Yeast Expression Vector</LI></UL><UL><P>1. Culture of PSB1AK3-T7 poromter+GFP+T7 terminater （Cmr resistant）gz-YEP352（AmpResistant, we have replaced the URA mark with trp mark）</P></UL><UL><P>2. Plasmid extraction: PSB1AK3-T7 poromter+GFP+T7 terminater，gz-YEP352</P></UL><UL><P>3. Digestion: </P><P>    Gz-YEP352    2ug </P><P>    XbaI    2ul</P><P>    PstI    2ul</P><P>    10 x H Buffer    5ul</P><P>    ddH2O        50ul</P><P>    PSB1AK3-T7 poromter+GFP+T7 terminater    2ug   </P><P>    XbaI    2ul </P><P>    Pst    2ul</P><P>    10 x H Buffer    5ul </P><P>    ddH2    50ul</P></UL><UL><P>4. Electrophoresis: </P><P>    1% agarose gel, 120V, after 45min,EB dye. </P></UL><UL><P>5. Gel recovery of gz-YEP352 (XbaI /PstI)PSB1AK3-T7 poromter+GFP+T7 terminater (XbaI / PstI) (Qiagen Gel recovery kits) </P></UL><UL><P>6. Lingake: </P><P>    Gz-YEP352 (XbaI /PstI)    100ng</P><P>    PSB1AK3-T7 poromter+GFP+T7 terminater    100ng</P><P>    T4 buffer    1ul</P><P>    T4 ligase    1ul</P><P>    ddH2O    10ul</P><P>    16oC    overnight</P></UL><UL><P>7. Transduction: </P><P>    1. UV sterilization half an hour before, get DH5a (competent yeast) out of fridge, thawing in water bath. </P><P>    2. 10μlinkage production with 50μl competent yeast cells, ice-bath for 30mins. </P><P>    3. 42oC ice-bath heat shock for 50s. </P><P>    4. Stewing on ice for 2mins. </P><P>    5. Add 500μl LB medium without antibiotic, 37℃ shaking(225rpm/min), recover it for culturing it for 1 hour. </P><P>    6. 3000rpm/min for 5mins, abandon the supermate, mixing the left. </P><P>    7. 60μl broth spread the Amp anti-LB solid plat, 37℃ inversed culture in incubation overnight. </P></UL><UL><P>8. Clony PCR:
&lt;p&gt;    dNTPmix(2.5mM)    1μl</P><P>    PCR 10×buffer    2μl</P><P>    M13_pUC_fwd_primer    0.5μl</P><P>    M13_reverse_primer    0.5μl</P><P>    Ex Taq    0.5μl</P><P>    H2O    15.5μl</P><P>    94℃    3min</P><P>    94℃    30s</P><P>    55℃    30s </P><P>    72℃    1.5min</P><P>    72℃    10min</P><P>    12℃    ∞</P><P>    30 cycles</P></UL><UL><P>9. 1.5% agarose gel electrophoresis, &lt;130V, 40mins. </P></UL><UL><P>10. Sequencing: </P><P>    Pick the positive colony(colony PCR verification), marking lines and culture(Cmr resistant), sequencing. Name the right vector:EP352-GAL1,Holin-ADH1</P></UL><UL><LI>III. Yeast Transformation Experiment:</LI></UL><UL><P>1. plasmid extraction. </P></UL><UL><P>2. Preparation  of Competent Yeast Cells - LiAc Method</P><P>    1.treak a YPDA agar plate with a small portion of AH109 or Y187. Incubate the plate upside down at 30°C until colonies appear(~3days). Yeast strains can be stored for up to 1 month at 4°C on YPDA medium in culture plates. </P><P>    2.Prepare 1.1X TE/LiAc Solution </P><P>    3.Prepare YPDA liquid medium (Yeast Protocols Handbook) </P><P>    4.Inoculate one colony(&lt;4 weeks old, 2–3mm in diameter) into 3ml of YPDA medium in a sterile, 15-ml   centrifuge tube. </P><P>    5.Incubate at 30°C with shaking    for   8hr. </P><P>    6.Transfer 5µl of the culture  to a 250-ml flask containing  50ml of YPDA. </P><P>    7.Incubate at 30°C with shaking at 230–250 rpm for   16–20hr.The OD600 should reach 0.15–0.3. </P><P>    8.Centrifuge the cells at 700 x g for 5min at room temperature. </P><P>    9.Discard the supernatant and resuspend the  cell  pellet in 100 ml of YPDA. </P><P>    10.Incubate at 30°C for 3–5hr (OD600= 0.4–0.5). </P><P>    11.Centrifuge the cells at 700 x g for 5min at room  temperature. </P><P>    12.Discard the supernatant and resuspend the cell pellet in 60ml of sterile, deionized H2O. </P><P>    13.Centrifuge the cells at 700 x g for 5min at room  temperature. </P><P>    14.Discard the supernatant and resuspend the cells in 3 ml of 1.1 X TE/LiAc Solution. </P><P>    15.Split the resuspension between two 1.5-ml microcentrifuge tubes (1.5 ml per tube). </P><P>    16.Centrifuge each tube at high speed for 15 sec. </P><P>    17.Discard the supernatant and resuspend each pellet in 600 µl of 1.1 X TE/LiAc Solution. </P></UL><UL><P>3.Transformation: </P><P>    1. Gz-YEP352-T7poromter+GFP+T7terminater 0.2ug </P><P>    Sperm DNA*(10 mg/ml) 0.1mg</P><P>    Sperm DNA  :when prepared, water-bath boiling for 20mins, then ice-bath, -20°C preservation. </P><P>    2. 100 ul dense medium of yeast competent cell by 1×TE/LiAc. Whirlpool oscillation. </P><P>    3. 600 ulPEG/LiAc, oscillation strongly, 30°C,200rpm, culture oscillating for 30mins. </P><P>    4. Add 70ul DMSO, mixing it slightly and 42oC heat shock by water-bath for 15mins, ice-bath for 1-2mins</P><P>    5. Centrifuging for 14,000rpm for 5sec, abandon the supermate, 0.1ml 1x TE suspense the cell precipitation, spread it to SD/ -Ura-trp solid plat, 30°C culture reversely for 3 days. </P></UL><UL><P>6. Holin function identification: </P><P>    Pick a SD/ -Ura-trp-cultured colony and inoculate it to 50ml YPDA medium, 30°C,220rpm culture reelingly to OD-0.1, transfer 20ml to 50ml sterile medium (alpha-galactose), versus is added by equivalent water. Measuring the OD value one time for one hour and drawing the curve. </P><P>    Result analyzation: </P><P>        If the bacteria in alpha-galactose show flurorescence, that shows T7 RNAP/T7 promoter can work. </P></UL></DIV><DIV class="context"><H5><SPAN class="mw-headline" id="Protocol:_Holin_Adaptor_Engineering">Protocol: Holin Adaptor Engineering</SPAN></H5><UL><P>1. Plasmid extraction of BBa-K112306（holin）</P></UL><UL><P>2. Point mutation:</P><P>    Primers</P><P>        &gt; Holin-F   GTTTCTTCGA ATTCGCGGCC GCTTCTAGAG ATGCCAGAAA AACATGACCT</P><P>        &gt; Holin-R   GTTTCTTCCT GCAGCGGCCG CTACTAGTAT TATTGATTTC TACCATCTT</P><P>    BBa-K112306    20ng</P><P>    Ex Taq    0.5ul</P><P>    10 X Ex Buffer    2ul</P><P>    Dntp Mix    1ul</P><P>    Holin-F    0.5ul</P><P>    Holin-R    0.5ul</P><P>    ddH2O    20ul</P><P>    94℃    3min</P><P>    94℃    30s</P><P>    55℃    30s</P><P>    72℃    30s</P><P>    72℃    10min</P><P>    12℃    ∞</P><P>    30 cycles</P></UL><UL><P>3. Gel electrophoresis</P></UL><UL><DIV class="figurep"><P>Figure 10.  Channel 1:marker(100BP)  Channel 2:holin</P></DIV></UL><UL><P>4. Gel recovery(QIAGEN)</P></UL><UL><P>5. T clone of holing(TAKARA T clone kits)</P></UL><UL><P>6. Colony PCR:</P><P>    Ex Taq    0.5ul</P><P>    10 X Ex Buffer    2ul</P><P>    Dntp Mix    1ul</P><P>    M13-47    0.5ul</P><P>    RV-M    0.5ul</P><P>    ddH2O    20ul</P><P>    94℃    3min</P><P>    94℃    30s</P><P>    55℃    30s</P><P>    72℃    30s</P><P>    72℃    10min</P><P>    12℃    ∞</P><P>    30 cycles</P></UL><UL><DIV class="figurep"><P>Figure 11.  Channel 1:maker（100bp）  Channel 3-6 ：5 strain contains holin</P></DIV></UL><UL><P>7. Digestion Verification:</P><P>    Plasmid T-holin    100ng</P><P>    XbaI    0.5ul</P><P>    PstI    0.5ul</P><P>    1 X M buffer    2ul</P><P>    ddH2O    20ul</P><P>    37oC 3 hours</P></UL><UL><DIV class="figurep"><P>Figure 12.  Channel 1 :maker100bp  channel 2: Holin digestion</P></DIV></UL><UL><P>8. Sequencing</P></UL></DIV></DIV><DIV id="gallery"><DIV class="printfooter">
Retrieved from &quot;<A href="http://2012.igem.org/Team:Shenzhen/Result/YAO.Suicider">http://2012.igem.org/Team:Shenzhen/Result/YAO.Suicider</A>&quot;</DIV></DIV><DIV id="footer-box" class="noprint"><DIV id="footer"><UL id="f-list"><LI id="t-recentchanges"><A href="/Special:RecentChanges" title="Recent changes">Recent changes</A></LI><LI id="t-whatlinkshere"><A href="/Special:WhatLinksHere/Team:Shenzhen/Result/YAO.Suicider" title="List of all wiki pages that link here [j]" accesskey="j">What links here</A></LI><LI id="t-recentchangeslinked"><A href="/Special:RecentChangesLinked/Team:Shenzhen/Result/YAO.Suicider" title="Recent changes in pages linked from this page [k]" accesskey="k">Related changes</A></LI><LI id="t-specialpages"><A href="/Special:SpecialPages" title="List of all special pages [q]" accesskey="q">Special pages</A></LI><LI><A href="/Special:Preferences">My preferences</A></LI></UL></DIV><DIV id="footer"><UL id="f-list"><LI id="t-print"><A href="/wiki/index.php?title=Team:Shenzhen/Result/YAO.Suicider&amp;printable=yes" title="Printable version of this page [p]" accesskey="p">Printable version</A></LI><LI id="t-permalink"><A href="/wiki/index.php?title=Team:Shenzhen/Result/YAO.Suicider&amp;oldid=207664" title="Permanent link to this revision of the page">Permanent link</A></LI><LI id="privacy"><A href="/2012.igem.org:Privacy_policy" title="2012.igem.org:Privacy policy">Privacy policy</A></LI><LI id="disclaimer"><A href="/2012.igem.org:General_disclaimer" title="2012.igem.org:General disclaimer">Disclaimers</A></LI></UL></DIV></DIV></UL></DIV></UL></DIV></UL></DIV></UL></UL></DIV></DIV></DIV></DIV></DIV></BODY></HTML>