<HTML xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en" dir="ltr">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki  ltr ns-0 ns-subject page-Team_Shenzhen_Result_YAO_Sensor"><DIV id="globalWrapper"><DIV id="top-section"><DIV id="p-logo"><A href="/Main_Page" title="Main Page">&quot;
	    </A></DIV><DIV id="menubar" class="left-menu noprint"><UL><LI class="selected"><A href="/Team:Shenzhen/Result/YAO.Sensor">Page               </A></LI><LI class="new"><A href="/wiki/index.php?title=Talk:Team:Shenzhen/Result/YAO.Sensor&amp;action=edit&amp;redlink=1">Discussion               </A></LI><LI><A href="/wiki/index.php?title=Team:Shenzhen/Result/YAO.Sensor&amp;action=edit">View source               </A></LI><LI><A href="/wiki/index.php?title=Team:Shenzhen/Result/YAO.Sensor&amp;action=history">History               </A></LI><LI style="color:#808080;cursor:default">teams</LI></UL></DIV><DIV class="right-menu noprint" id="menubar"><UL><LI id="pt-login"><A href="/wiki/index.php?title=Special:UserLogin&amp;returnto=Team:Shenzhen/Result/YAO.Sensor" title="You are encouraged to log in; however, it is not mandatory [o]" accesskey="o">Log in</A></LI></UL></DIV><DIV id="search-controls" class="noprint"><FORM action="/Special:Search" id="searchform"> </FORM></DIV></DIV><DIV id="content"><H1 class="firstHeading">Team:Shenzhen/Result/YAO.Sensor</H1><DIV id="bodyContent"><H3 id="siteSub" class="noprint">From 2012.igem.org</H3><DIV class="container"><DIV class="content"><UL class="ca-menu"><UL id="subs1"><LI id="ca-green1" class="ca-green"><A href="https://2012.igem.org/Team:Shenzhen/Team"><SPAN class="ca-icon">U</SPAN><DIV class="ca-content"><H2 class="ca-main">Team</H2></DIV></A></LI><DIV id="sub1" class="submenu"><LI class="ca-sub"><A href="https://2012.igem.org/Team:Shenzhen/Team">Consortium</A></LI><LI class="ca-sub"><A href="https://2012.igem.org/Team:Shenzhen/Team/BGI_College">BGI College</A></LI><LI class="ca-sub"><A href="https://2012.igem.org/Team:Shenzhen/Team/Instructors">Instructors</A></LI><LI class="ca-sub"><A href="https://2012.igem.org/Team:Shenzhen/Team/HUSTers">HUSTers</A></LI><LI class="ca-sub"><A href="https://2012.igem.org/Team:Shenzhen/Team/SCUTers">SCUTers</A></LI><LI class="ca-sub"><A href="https://2012.igem.org/Team:Shenzhen/Team/SCUers">SCUers</A></LI><LI class="ca-sub"><A href="https://2012.igem.org/Team:Shenzhen/Team/WHUers">WHUers</A></LI><LI class="ca-sub"><A href="https://2012.igem.org/Team:Shenzhen/Team/UESTCers">UESTCers</A></LI><LI class="ca-sub"><A href="https://2012.igem.org/Team:Shenzhen/Team/SEUers">SEUers</A></LI><LI class="ca-sub"><A href="https://2012.igem.org/Team:Shenzhen/Team/SCNUers">SCNUers</A></LI><LI class="ca-sub"><A href="https://2012.igem.org/Team:Shenzhen/Team/CUHKer">CUHKer</A></LI></DIV></UL><UL id="subs2"><LI id="ca-orange1" class="ca-orange"><A href="https://2012.igem.org/Team:Shenzhen/Project"><SPAN class="ca-icon">L</SPAN><DIV class="ca-content"><H2 class="ca-main">Project</H2></DIV></A></LI><DIV id="sub2" class="submenu"><LI class="ca-sub"><A href="https://2012.igem.org/Team:Shenzhen/Project">Stories</A></LI><LI class="ca-sub"><A href="https://2012.igem.org/Team:Shenzhen/Project/YAO.Genome">YAO.Genome</A></LI><LI class="ca-sub"><A href="https://2012.igem.org/Team:Shenzhen/Project/YAO.Channel">YAO.Channel</A></LI><LI class="ca-sub"><A href="https://2012.igem.org/Team:Shenzhen/Project/YAO.Sensor">YAO.Sensor</A></LI><LI class="ca-sub"><A href="https://2012.igem.org/Team:Shenzhen/Project/YAO.Suicider">YAO.Suicider</A></LI><LI class="ca-sub"><A href="https://2012.igem.org/Team:Shenzhen/Project/YAO.Factory">YAO.Factory</A></LI></DIV></UL><UL id="subs3"><LI id="ca-yellow1" class="ca-yellow"><A href="https://2012.igem.org/Team:Shenzhen/Result"><SPAN class="ca-icon">u</SPAN><DIV class="ca-content"><H2 class="ca-main">Result</H2></DIV></A></LI><DIV id="sub3" class="submenu"><LI class="ca-sub"><A href="https://2012.igem.org/Team:Shenzhen/Result">Summary</A></LI><LI class="ca-sub"><A href="https://2012.igem.org/Team:Shenzhen/Result/YAO.Genome">YAO.Genome</A></LI><LI class="ca-sub"><A href="https://2012.igem.org/Team:Shenzhen/Result/YAO.Channel">YAO.Channel</A></LI><LI class="ca-sub"><A href="https://2012.igem.org/Team:Shenzhen/Result/YAO.Sensor">YAO.Sensor</A></LI><LI class="ca-sub"><A href="https://2012.igem.org/Team:Shenzhen/Result/YAO.Suicider">YAO.Suicider</A></LI><LI class="ca-sub"><A href="https://2012.igem.org/Team:Shenzhen/Result/YAO.Factory">YAO.Factory</A></LI></DIV></UL><UL id="subs4"><LI id="ca-green2" class="ca-green"><A href="https://2012.igem.org/Team:Shenzhen/Biobricks"><SPAN class="ca-icon">S</SPAN><DIV class="ca-content"><H2 class="ca-main">BioBricks</H2></DIV></A></LI><DIV id="sub4" class="submenu"><LI class="ca-sub"><A href="https://2012.igem.org/Team:Shenzhen/Biobricks">Summary</A></LI><LI class="ca-sub"><A href="https://2012.igem.org/Team:Shenzhen/Biobricks/YAO.Genome">YAO.Genome</A></LI><LI class="ca-sub"><A href="https://2012.igem.org/Team:Shenzhen/Biobricks/YAO.Channel">YAO.Channel</A></LI><LI class="ca-sub"><A href="https://2012.igem.org/Team:Shenzhen/Biobricks/YAO.Sensor">YAO.Sensor</A></LI><LI class="ca-sub"><A href="https://2012.igem.org/Team:Shenzhen/Biobricks/YAO.Suicider">YAO.Suicider</A></LI><UL id="subs5"><LI id="ca-orange2" class="ca-orange"><A href="https://2012.igem.org/Team:Shenzhen/Notebook"><SPAN class="ca-icon">p</SPAN><DIV class="ca-content"><H2 class="ca-main">Notebook</H2></DIV></A></LI><DIV id="sub5" class="submenu"><LI class="ca-sub"><A href="https://2012.igem.org/Team:Shenzhen/Notebook">Team History</A></LI><LI class="ca-sub"><A href="https://2012.igem.org/Team:Shenzhen/Notebook/YAO.Genome">YAO.Genome</A></LI><LI class="ca-sub"><A href="https://2012.igem.org/Team:Shenzhen/Notebook/YAO.Channel">YAO.Channel</A></LI><LI class="ca-sub"><A href="https://2012.igem.org/Team:Shenzhen/Notebook/YAO.Sensor">YAO.Sensor</A></LI><LI class="ca-sub"><A href="https://2012.igem.org/Team:Shenzhen/Notebook/YAO.Suicider">YAO.Suicider</A></LI><LI class="ca-sub"><A href="https://2012.igem.org/Team:Shenzhen/Notebook/YAO.Factory">YAO.Factory</A></LI><UL id="subs6"><LI id="ca-green3" class="ca-green"><A href="https://2012.igem.org/Team:Shenzhen/Biosafety"><SPAN class="ca-icon">,</SPAN><DIV class="ca-content"><H2 class="ca-main">Biosafety</H2></DIV></A></LI></UL><UL id="subs7"><LI id="ca-yellow2" class="ca-yellow"><A href="https://2012.igem.org/Team:Shenzhen/Practices"><SPAN class="ca-icon">e</SPAN><DIV class="ca-content"><H2 class="ca-main">Practices</H2></DIV></A></LI><DIV id="sub7" class="submenu"><LI class="ca-sub"><A href="https://2012.igem.org/Team:Shenzhen/Practices">Summary</A></LI><LI class="ca-sub"><A href="https://2012.igem.org/Team:Shenzhen/Practices/Lectures">Lectures</A></LI><LI class="ca-sub"><A href="https://2012.igem.org/Team:Shenzhen/Practices/High_School">High School</A></LI><LI class="ca-sub"><A href="https://2012.igem.org/Team:Shenzhen/Practices/Teamwork">Teamwork</A></LI><UL id="subs8"><LI id="ca-orange3" class="ca-orange"><A href="https://2012.igem.org/Team:Shenzhen/Acknowledgment"><SPAN class="ca-icon">N</SPAN><DIV class="ca-content"><H2 class="ca-main">Acknow.</H2></DIV></A></LI><DIV id="sub8" class="submenu"><LI class="ca-sub"><A href="https://2012.igem.org/Team:Shenzhen/Acknowledgment">Summary</A></LI><LI class="ca-sub"><A href="https://2012.igem.org/Team:Shenzhen/Acknowledgment/Providers">Providers</A></LI><LI class="ca-sub"><A href="https://2012.igem.org/Team:Shenzhen/Acknowledgment/Advisors">Advisors</A></LI><LI class="ca-sub"><A href="https://2012.igem.org/Team:Shenzhen/Acknowledgment/Sponsors">Sponsors</A></LI><LI class="ca-sub"><A href="https://2012.igem.org/Team:Shenzhen/Acknowledgment/Home_Schools">Home Schools</A></LI></DIV><P><UL id="navigationMenu"><LI><A class="home" href="https://2012.igem.org/Team:Shenzhen/Result"><SPAN>Summary</SPAN></A></LI><LI><A class="genome" href="https://2012.igem.org/Team:Shenzhen/Result/YAO.Genome"><SPAN>YAO.Genome</SPAN></A></LI><LI><A class="channel" href="https://2012.igem.org/Team:Shenzhen/Result/YAO.Channel"><SPAN>YAO.Channel</SPAN></A></LI><LI><A class="sensor" href="https://2012.igem.org/Team:Shenzhen/Result/YAO.Sensor"><SPAN>YAO.Sensor</SPAN></A></LI><LI><A class="suicider" href="https://2012.igem.org/Team:Shenzhen/Result/YAO.Suicider"><SPAN>YAO.Suicider</SPAN></A></LI><LI><A class="houseyellow" href="https://2012.igem.org/Team:Shenzhen/Result/YAO.Factory"><SPAN>YAO.Factory</SPAN></A></LI></UL></P><DIV id="bodymain"><DIV class="context"><H5><SPAN class="mw-headline" id="Introduction_to_Modeling_Work">Introduction to Modeling Work</SPAN></H5><UL><P>    Based on the genetic design of YAO sensor, we modeled the sensor’s functions via the method of modular structured kinetics. In order to model the functions of YAO sensor, we divided the procession into 5 parts:</P><P>1. Transport of REX into mitochondrial matrix</P><P>2. Activation of REX as a transcription factor with NAD+ and NADH</P><P>3. Control of promoter activity by REX</P><P>4. Synthesis and degradation of mRNA</P><P>5. Synthesis, dilution and degradation of mtGFP</P><P>    Finally, the model’s response to dynamic state are tested.</P></UL><UL><P>    However, since many parameters for this model are not yet available, the model is not expected to make exact predictions but to give a direct understanding of the functions of the model.</P></UL></DIV><DIV class="context"><H5><SPAN class="mw-headline" id="Simulation_.26_Results">Simulation &amp; Results</SPAN></H5><UL><LI>I. Transportation of REX into Mitochondrial Matrix</LI></UL><UL><P>assume that the transcription of SP-REX is constant, SP-REX concentration in cytoplasm is constant, the probably for REX to be transported into each mitochondrion is equal, and mitochondria are of similar size. So with different number of mitochondria in yeast (Nm), the concentration of REX in each mitochondria (Ctf) might differ. Ksp is the transport coefficient of signal peptid which links the XR/Nm with Ctf. For different kinds of signal peptid, Ksp might be different.</P></UL><UL><P>  (1)</P><P><I>XSR: SP-REX concentration in cytoplasm. Nm: Number of mitochondria. Crex: Concentration of REX in Mitochondria. Ksp: Transport coefficient of signal peptid.</I></P></UL><UL><LI>II. Activation of REX as a Transcription Factor with NAD+ and NADH</LI></UL><UL><P>NAD+ and NADH can competitively bind to the same site of REX, when NADH binds to REX, REX will be repressed and cannot bind to ROP. Then the transcription function is repressed. </P></UL><UL><P><I>H stands for NADH, REXH stands for REX bidden with NADH.</I></P></UL><UL><P>At equilibrium:   (2)</P><P><I>N stands for NAD+, REXN stands for REX bidden with NAD+.</I></P></UL><UL><P>At equilibrium:   (3)</P></UL><UL><P>NAD+ and NADH are in a balance, so we assumed the total level of NAD+ and NADH in the mitochondria is constant.</P></UL><UL><P>REX, REXH and REXN are in a balance, so total level of REX, REXH and REXN equals to Crex.</P></UL><UL><P> (4)</P></UL><UL><P>REX which was not bidden with NADH can bind to ROP.</P></UL><UL><P> (5)</P></UL><UL><P>REX’ stands for active REX which can bind to ROP.</P></UL><UL><P>The fraction of active REX acting as repressor is Rf.</P></UL><UL><P> (6)</P></UL><UL><P>With (5), (4), (2) and (3) in (6), the fraction becomes</P></UL><UL><P> (7)</P></UL><UL><DIV class="figurep"><P>Figure 1. Rf responding to different NAD+/NADH ratio.</P><P>For plotting of this picture the following parameters were used: assuming K_H = 0,02 μM and K_N = 2 μM since REX NADH affinity &gt; REX NAD affinity [2]. </P></DIV></UL><UL><LI>III. Control of promoter activity by REX</LI></UL><UL><P>In yeast mitochondria, transcription initiation factor MTF1, combined with RNA polymerase Rpo41, bind to promoter to initiate transcription. It is MTF1 that discerns the promoter region, and the binding of MTF1 on promoter region is reversible. Also the binding of REX on ROP, a DNA region near promoter is also reversible.</P></UL><UL><P><I>P stands for promoter, RP stands for repressor, i.e. REX’, and PRP stands for promoter bidden with repressor.</I></P></UL><UL><P>At equilibrium:  (8)</P><P><I>P stands for promoter, IF stands for initiation factor i.e. MTF1, and PRP stands for promoter bidden with initiation factor.</I></P></UL><UL><P>At equilibrium:  (9)</P></UL><UL><P>Because most of promoters bidden with IF can started transcription, and few of promoters bidden by repressor or not bidden with any TF will transcript. So two coefficient α and β are assumed to reflect the relations.</P></UL><UL><P> (10)</P></UL><UL><P>The fraction of promoters that can be transcribed is Pf.</P></UL><UL><P>  (11)</P></UL><UL><P>With (10), (8) and (9) in (11), the fraction becomes</P><P>  (12)</P><P>And</P><P>  (13)</P></UL><UL><P>So the fraction will be:</P></UL><UL><P>  (14)</P><P><I>α=1，Kif=5μM, Krp=5μM, IF=1mM was assumed [5] in this equation.</I></P></UL><UL><DIV class="figurep"><P>Figure 2. Pf with different β and changing NAD+/NADH ratio. </P><P>X axis：NAD+/NADH ratio, range from 0 to 500, Y axis：β, range from 0 to 0.2, Z axis：Pf. </P></DIV></UL><UL><DIV class="figurep"><P>Figure 3. Pf with different Crex and changing NAD+/NADH ratio.</P><P>X axis：NAD+/NADH ratio, range from 0 to 500, Y axis：Crex , range from 0mM to 5mM, Z axis：Pf. </P></DIV></UL><UL><DIV class="figurep"><P>Figure 4. Pf with different NAD+/NADH ratio.</P><P>β=0.05, Crex= 2mM was used to plot this picture.</P></DIV></UL><UL><LI>IV. Synthesis and degradation of mRNA</LI></UL><UL><P>mRNA is produced from the transcription of the active promoter. Concentration of mitochondrial levels of mRNA (XmRNA) can be derived from the mRNA balance: </P></UL><UL><P>Here it is assumed that mRNA is degraded corresponding to a first order reaction and that mRNA is not exported from the cell. </P></UL><UL><P>The term μXmRNA represents the dilution due to growth. However mRNA is highly unstable and is thus degraded fast. This makes Kdm+μ≈Kdm, simplify the mRNA balance to: </P></UL><UL><P> (16)</P></UL><UL><P>It is assumed that when RNA polymerase is efficiently bound to the promoter, the promoter is active and mRNA is transcribed at constant maximum rate  . The binding of RNA polymerase is dependent on Pf. In the overall production rate of mRNA, the number of promoters (and genes) Np is also influencing: </P></UL><UL><P> (17)</P></UL><UL><P>With (16) and (17), the mRNA balance equation becomes</P><P> (18)</P></UL><UL><P>At equilibrium:  , so</P><P> (19)</P></UL><UL><DIV class="figurep"><P>Figure 5. XmRNA  with different NAD+/NADH ratio.</P><P>β=0.05, Vmax = 180 U/h (assuming 75 bases per second, and 1500 bases in gene), Np=10 (1 micron plasmid is used), Kdm=8.3 h-1 (half life = 5 min) was used to plot this picture.</P></DIV></UL><UL><LI>V. Synthesis, dilution and degradation of GFP</LI></UL><UL><P>The GFP protein balance:</P><P> (20)</P></UL><UL><P>It is assumed that GFP is degraded by first order rate. The dilution cannot be neglected since protein degradation is much slower than mRNA degradation. Furthermore it is assumed that GFP is not transported out of the cell. </P></UL><UL><P>The protein is synthesized by ribosomes. The number of ribosomes bound to one mRNA (Np) and the protein production rate of the ribosome (Kp) should be taken into account. The number of ribosomes in the cell, ATP and available charged amino acids are assumed to be non-limiting. Taking this into account, the protein synthesis rate can be described as </P><P> (21)</P><P>With (21) in (20),</P><P> (22)</P></UL><UL><P>At equilibrium:  , so</P><P> (23)</P><P><I>Nrib = 1. Kp = 288 U/h, assuming 40 amino acids per second. KdGFP = 1.386 h-1 equal to a half time of 30 min [3]. Growth rate = 0.1 h-1.</I></P></UL><UL><DIV class="figurep"><P>Figure 6. XGFP with different β and changing NAD+/NADH ratio.</P><P>X axis: NAD+/NADH ratio, range from 0 to 500, Y axis：β, range from 0 to 0.2, Z axis：XGFP unit.</P></DIV></UL><UL><DIV class="figurep"><P>Figure 7. XGFP with different Crex and changing NAD+/NADH ratio.</P><P>X axis: NAD+/NADH ratio, range from 0 to 500, Y axis：Crex , range from 0mM to 5mM, Z axis：XGFP unit.  </P></DIV></UL><UL><DIV class="figurep"><P>Figure 8. XGFP with different NAD+/NADH ratio.</P></DIV></UL><UL><LI>Response for Dynamic states</LI></UL><UL><P>Because the NAD+/NADH level might change in times, so we assume that the change is sinusoidal function as follows: </P><P>    NH=Asin(t/2)+B</P><P>    A=100, B=50, t=0~40 h. </P></UL><UL><P>The curve is shown in Figure 9. Then as a result the production of GFP might also changes as a response to the change in NAD+/NADH.</P></UL><UL><DIV class="figurep"><P>Figure 9. Dynamic change of NAD+/NADH</P></DIV></UL><UL><DIV class="figurep"><P>Figure 10. Response in GFP production</P></DIV></UL></DIV><DIV class="context"><H5><SPAN class="mw-headline" id="References">References</SPAN></H5><UL><P>[1] Heijnen J.J 2009. Modular structured kinetic models: protein production from genes.</P></UL><UL><P>[2] Brekasis, D., Paget, M.S., 2003. A novel sensor of NADH/NAD+ redox poise in Streptomyces coelicolor A3(2). EMBO J. 22, 4856–4865.</P></UL><UL><P>[3] Sasson, V., I. Shachrai, et al. 2012. &quot;Mode of regulation and the insulation of bacterial gene expression.&quot; Mol Cell 46(4): 399-407.</P></UL><UL><P>[4] Mateus and Avery, 2000. Destabilized green fluorescent protein for monitoring dynamic changes in yeast gene expression with flow cytometry. Yeast 2000; 16: 1313±1323. </P></UL><UL><P>[5] Deshpande, A. P. and S. S. Patel 2012. &quot;Mechanism of transcription initiation by the yeast mitochondrial RNA polymerase.&quot; Biochim Biophys Acta.</P></UL><UL><P>[6] Modelling of The Redoxilator by 2009 DTU-DENMARK iGEM team.</P></UL></DIV></DIV><DIV id="gallery"><DIV class="printfooter">
Retrieved from &quot;<A href="http://2012.igem.org/Team:Shenzhen/Result/YAO.Sensor">http://2012.igem.org/Team:Shenzhen/Result/YAO.Sensor</A>&quot;</DIV></DIV><DIV id="footer-box" class="noprint"><DIV id="footer"><UL id="f-list"><LI id="t-recentchanges"><A href="/Special:RecentChanges" title="Recent changes">Recent changes</A></LI><LI id="t-whatlinkshere"><A href="/Special:WhatLinksHere/Team:Shenzhen/Result/YAO.Sensor" title="List of all wiki pages that link here [j]" accesskey="j">What links here</A></LI><LI id="t-recentchangeslinked"><A href="/Special:RecentChangesLinked/Team:Shenzhen/Result/YAO.Sensor" title="Recent changes in pages linked from this page [k]" accesskey="k">Related changes</A></LI><LI id="t-specialpages"><A href="/Special:SpecialPages" title="List of all special pages [q]" accesskey="q">Special pages</A></LI><LI><A href="/Special:Preferences">My preferences</A></LI></UL></DIV><DIV id="footer"><UL id="f-list"><LI id="t-print"><A href="/wiki/index.php?title=Team:Shenzhen/Result/YAO.Sensor&amp;printable=yes" title="Printable version of this page [p]" accesskey="p">Printable version</A></LI><LI id="t-permalink"><A href="/wiki/index.php?title=Team:Shenzhen/Result/YAO.Sensor&amp;oldid=231529" title="Permanent link to this revision of the page">Permanent link</A></LI><LI id="privacy"><A href="/2012.igem.org:Privacy_policy" title="2012.igem.org:Privacy policy">Privacy policy</A></LI><LI id="disclaimer"><A href="/2012.igem.org:General_disclaimer" title="2012.igem.org:General disclaimer">Disclaimers</A></LI></UL></DIV></DIV></UL></DIV></UL></DIV></UL></DIV></UL></UL></DIV></DIV></DIV></DIV></DIV></BODY></HTML>