<HTML xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en" dir="ltr">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki  ltr ns-0 ns-subject page-Team_Shenzhen_Result_YAO_Channel"><DIV id="globalWrapper"><DIV id="top-section"><DIV id="p-logo"><A href="/Main_Page" title="Main Page">&quot;
	    </A></DIV><DIV id="menubar" class="left-menu noprint"><UL><LI class="selected"><A href="/Team:Shenzhen/Result/YAO.Channel">Page               </A></LI><LI class="new"><A href="/wiki/index.php?title=Talk:Team:Shenzhen/Result/YAO.Channel&amp;action=edit&amp;redlink=1">Discussion               </A></LI><LI><A href="/wiki/index.php?title=Team:Shenzhen/Result/YAO.Channel&amp;action=edit">View source               </A></LI><LI><A href="/wiki/index.php?title=Team:Shenzhen/Result/YAO.Channel&amp;action=history">History               </A></LI><LI style="color:#808080;cursor:default">teams</LI></UL></DIV><DIV class="right-menu noprint" id="menubar"><UL><LI id="pt-login"><A href="/wiki/index.php?title=Special:UserLogin&amp;returnto=Team:Shenzhen/Result/YAO.Channel" title="You are encouraged to log in; however, it is not mandatory [o]" accesskey="o">Log in</A></LI></UL></DIV><DIV id="search-controls" class="noprint"><FORM action="/Special:Search" id="searchform"> </FORM></DIV></DIV><DIV id="content"><H1 class="firstHeading">Team:Shenzhen/Result/YAO.Channel</H1><DIV id="bodyContent"><H3 id="siteSub" class="noprint">From 2012.igem.org</H3><DIV class="container"><DIV class="content"><UL class="ca-menu"><UL id="subs1"><LI id="ca-green1" class="ca-green"><A href="https://2012.igem.org/Team:Shenzhen/Team"><SPAN class="ca-icon">U</SPAN><DIV class="ca-content"><H2 class="ca-main">Team</H2></DIV></A></LI><DIV id="sub1" class="submenu"><LI class="ca-sub"><A href="https://2012.igem.org/Team:Shenzhen/Team">Consortium</A></LI><LI class="ca-sub"><A href="https://2012.igem.org/Team:Shenzhen/Team/BGI_College">BGI College</A></LI><LI class="ca-sub"><A href="https://2012.igem.org/Team:Shenzhen/Team/Instructors">Instructors</A></LI><LI class="ca-sub"><A href="https://2012.igem.org/Team:Shenzhen/Team/HUSTers">HUSTers</A></LI><LI class="ca-sub"><A href="https://2012.igem.org/Team:Shenzhen/Team/SCUTers">SCUTers</A></LI><LI class="ca-sub"><A href="https://2012.igem.org/Team:Shenzhen/Team/SCUers">SCUers</A></LI><LI class="ca-sub"><A href="https://2012.igem.org/Team:Shenzhen/Team/WHUers">WHUers</A></LI><LI class="ca-sub"><A href="https://2012.igem.org/Team:Shenzhen/Team/UESTCers">UESTCers</A></LI><LI class="ca-sub"><A href="https://2012.igem.org/Team:Shenzhen/Team/SEUers">SEUers</A></LI><LI class="ca-sub"><A href="https://2012.igem.org/Team:Shenzhen/Team/SCNUers">SCNUers</A></LI><LI class="ca-sub"><A href="https://2012.igem.org/Team:Shenzhen/Team/CUHKer">CUHKer</A></LI></DIV></UL><UL id="subs2"><LI id="ca-orange1" class="ca-orange"><A href="https://2012.igem.org/Team:Shenzhen/Project"><SPAN class="ca-icon">L</SPAN><DIV class="ca-content"><H2 class="ca-main">Project</H2></DIV></A></LI><DIV id="sub2" class="submenu"><LI class="ca-sub"><A href="https://2012.igem.org/Team:Shenzhen/Project">Stories</A></LI><LI class="ca-sub"><A href="https://2012.igem.org/Team:Shenzhen/Project/YAO.Genome">YAO.Genome</A></LI><LI class="ca-sub"><A href="https://2012.igem.org/Team:Shenzhen/Project/YAO.Channel">YAO.Channel</A></LI><LI class="ca-sub"><A href="https://2012.igem.org/Team:Shenzhen/Project/YAO.Sensor">YAO.Sensor</A></LI><LI class="ca-sub"><A href="https://2012.igem.org/Team:Shenzhen/Project/YAO.Suicider">YAO.Suicider</A></LI><LI class="ca-sub"><A href="https://2012.igem.org/Team:Shenzhen/Project/YAO.Factory">YAO.Factory</A></LI></DIV></UL><UL id="subs3"><LI id="ca-yellow1" class="ca-yellow"><A href="https://2012.igem.org/Team:Shenzhen/Result"><SPAN class="ca-icon">u</SPAN><DIV class="ca-content"><H2 class="ca-main">Result</H2></DIV></A></LI><DIV id="sub3" class="submenu"><LI class="ca-sub"><A href="https://2012.igem.org/Team:Shenzhen/Result">Summary</A></LI><LI class="ca-sub"><A href="https://2012.igem.org/Team:Shenzhen/Result/YAO.Genome">YAO.Genome</A></LI><LI class="ca-sub"><A href="https://2012.igem.org/Team:Shenzhen/Result/YAO.Channel">YAO.Channel</A></LI><LI class="ca-sub"><A href="https://2012.igem.org/Team:Shenzhen/Result/YAO.Sensor">YAO.Sensor</A></LI><LI class="ca-sub"><A href="https://2012.igem.org/Team:Shenzhen/Result/YAO.Suicider">YAO.Suicider</A></LI><LI class="ca-sub"><A href="https://2012.igem.org/Team:Shenzhen/Result/YAO.Factory">YAO.Factory</A></LI></DIV></UL><UL id="subs4"><LI id="ca-green2" class="ca-green"><A href="https://2012.igem.org/Team:Shenzhen/Biobricks"><SPAN class="ca-icon">S</SPAN><DIV class="ca-content"><H2 class="ca-main">BioBricks</H2></DIV></A></LI><DIV id="sub4" class="submenu"><LI class="ca-sub"><A href="https://2012.igem.org/Team:Shenzhen/Biobricks">Summary</A></LI><LI class="ca-sub"><A href="https://2012.igem.org/Team:Shenzhen/Biobricks/YAO.Genome">YAO.Genome</A></LI><LI class="ca-sub"><A href="https://2012.igem.org/Team:Shenzhen/Biobricks/YAO.Channel">YAO.Channel</A></LI><LI class="ca-sub"><A href="https://2012.igem.org/Team:Shenzhen/Biobricks/YAO.Sensor">YAO.Sensor</A></LI><LI class="ca-sub"><A href="https://2012.igem.org/Team:Shenzhen/Biobricks/YAO.Suicider">YAO.Suicider</A></LI><UL id="subs5"><LI id="ca-orange2" class="ca-orange"><A href="https://2012.igem.org/Team:Shenzhen/Notebook"><SPAN class="ca-icon">p</SPAN><DIV class="ca-content"><H2 class="ca-main">Notebook</H2></DIV></A></LI><DIV id="sub5" class="submenu"><LI class="ca-sub"><A href="https://2012.igem.org/Team:Shenzhen/Notebook">Team History</A></LI><LI class="ca-sub"><A href="https://2012.igem.org/Team:Shenzhen/Notebook/YAO.Genome">YAO.Genome</A></LI><LI class="ca-sub"><A href="https://2012.igem.org/Team:Shenzhen/Notebook/YAO.Channel">YAO.Channel</A></LI><LI class="ca-sub"><A href="https://2012.igem.org/Team:Shenzhen/Notebook/YAO.Sensor">YAO.Sensor</A></LI><LI class="ca-sub"><A href="https://2012.igem.org/Team:Shenzhen/Notebook/YAO.Suicider">YAO.Suicider</A></LI><LI class="ca-sub"><A href="https://2012.igem.org/Team:Shenzhen/Notebook/YAO.Factory">YAO.Factory</A></LI><UL id="subs6"><LI id="ca-green3" class="ca-green"><A href="https://2012.igem.org/Team:Shenzhen/Biosafety"><SPAN class="ca-icon">,</SPAN><DIV class="ca-content"><H2 class="ca-main">Biosafety</H2></DIV></A></LI></UL><UL id="subs7"><LI id="ca-yellow2" class="ca-yellow"><A href="https://2012.igem.org/Team:Shenzhen/Practices"><SPAN class="ca-icon">e</SPAN><DIV class="ca-content"><H2 class="ca-main">Practices</H2></DIV></A></LI><DIV id="sub7" class="submenu"><LI class="ca-sub"><A href="https://2012.igem.org/Team:Shenzhen/Practices">Summary</A></LI><LI class="ca-sub"><A href="https://2012.igem.org/Team:Shenzhen/Practices/Lectures">Lectures</A></LI><LI class="ca-sub"><A href="https://2012.igem.org/Team:Shenzhen/Practices/High_School">High School</A></LI><LI class="ca-sub"><A href="https://2012.igem.org/Team:Shenzhen/Practices/Teamwork">Teamwork</A></LI><UL id="subs8"><LI id="ca-orange3" class="ca-orange"><A href="https://2012.igem.org/Team:Shenzhen/Acknowledgment"><SPAN class="ca-icon">N</SPAN><DIV class="ca-content"><H2 class="ca-main">Acknow.</H2></DIV></A></LI><DIV id="sub8" class="submenu"><LI class="ca-sub"><A href="https://2012.igem.org/Team:Shenzhen/Acknowledgment">Summary</A></LI><LI class="ca-sub"><A href="https://2012.igem.org/Team:Shenzhen/Acknowledgment/Providers">Providers</A></LI><LI class="ca-sub"><A href="https://2012.igem.org/Team:Shenzhen/Acknowledgment/Advisors">Advisors</A></LI><LI class="ca-sub"><A href="https://2012.igem.org/Team:Shenzhen/Acknowledgment/Sponsors">Sponsors</A></LI><LI class="ca-sub"><A href="https://2012.igem.org/Team:Shenzhen/Acknowledgment/Home_Schools">Home Schools</A></LI></DIV><P><UL id="navigationMenu"><LI><A class="home" href="https://2012.igem.org/Team:Shenzhen/Result"><SPAN>Summary</SPAN></A></LI><LI><A class="genome" href="https://2012.igem.org/Team:Shenzhen/Result/YAO.Genome"><SPAN>YAO.Genome</SPAN></A></LI><LI><A class="channel" href="https://2012.igem.org/Team:Shenzhen/Result/YAO.Channel"><SPAN>YAO.Channel</SPAN></A></LI><LI><A class="sensor" href="https://2012.igem.org/Team:Shenzhen/Result/YAO.Sensor"><SPAN>YAO.Sensor</SPAN></A></LI><LI><A class="suicider" href="https://2012.igem.org/Team:Shenzhen/Result/YAO.Suicider"><SPAN>YAO.Suicider</SPAN></A></LI><LI><A class="houseyellow" href="https://2012.igem.org/Team:Shenzhen/Result/YAO.Factory"><SPAN>YAO.Factory</SPAN></A></LI></UL></P><DIV id="bodymain"><DIV class="context"><H5><SPAN class="mw-headline" id="Construction_of_a_Signal_Peptide_System">Construction of a Signal Peptide System</SPAN></H5><UL><P>    Signal peptides are the tools we use to assemble YAO. Channel-Toc and Tic proteins, and to guide the protein into YAO matrix through YAO. Channel. We have designed several Signal Peptide BioBricks (shown in <A href="http://partsregistry.org/cgi/partsdb/pgroup.cgi?pgroup=iGEM2012&amp;group=Shenzhen" class="external free" rel="nofollow">http://partsregistry.org/cgi/partsdb/pgroup.cgi?pgroup=iGEM2012&amp;group=Shenzhen</A>). </P></UL><UL><P>    In total, there are 6 types of signal peptide involved in our project: signal peptide (SP) of Zim17, SP of ferredoxi, SP of Tim21, SP of Tom40, SP of Tom70, SP of Tom22.</P></UL><UL><LI>1. SP of Zim17</LI><P>    Zim17 is a zinc finger protein located in mitochondrial matrix. The signal peptide of Zim17 is amino-terminal segments rich in positively charged residues that form two to three turns of a helix with amphipathic character, and it directs the proteins imported into mitochondrial matrix. The sequence can be found at <A href="http://www.uniprot.org/blast/?about=P42844" class="external free" rel="nofollow">http://www.uniprot.org/blast/?about=P42844</A>[1-47]. We synthesized the sequence and linked it with RFP &amp; YFP.</P><P>The document we mainly refer to is:</P><P>Burri L, Vascotto K, Fredersdorf S, Tiedt R, Hall MN, Lithgow T. Zim17, a novel zinc finger protein essential for protein import into mitochondria. J. Biol. Chem. 2004;279:50243-50249. </P></UL><UL><LI>2. SP of ferredoxi</LI><P>    The signal peptide of ferredoxi, as a chloroplast-target transit peptide, has an α helix at its N-terminus, followed by an unstructured C-terminal domain of ~19 amino acids. The sequence can be found at <A href="http://www.uniprot.org/blast/?about=P07839%5b1-32%5d" class="external free" rel="nofollow">http://www.uniprot.org/blast/?about=P07839%5b1-32%5d</A>. We synthesized the sequence and linked it with RFP &amp; YFP.</P><P>The document we mainly refer to is:</P><P>Bruce, B. D. (2000). &quot;Chloroplast transit peptides: structure, function and evolution.&quot; Trends in Cell Biology 10(10): 440-447. </P></UL><UL><LI>3. SP of Tim21</LI><P>    Tim21 is one component of the translocase of the inner mitochondrial membrane (TIM complex), and the signal peptide of Tim21 contains 239 amino acids with a N-terminally single transmembrane segment showing the characteristics of a positively charged mitochondrial presequence with a predicted cleavage site after residue 42. The sequence can be found at  <A href="http://www.uniprot.org/blast/?about=P53220" class="external free" rel="nofollow">http://www.uniprot.org/blast/?about=P53220</A>[1-70]. We synthesized the sequence and linked it with RFP &amp; YFP.</P><P>The document we mainly refer to is:</P><P>Chacinska, A., M. Lind, et al. (2005). &quot;Mitochondrial Presequence Translocase: Switching between TOM Tethering and Motor Recruitment Involves Tim21 and Tim17.&quot; Cell 120(6): 817-829.</P></UL><UL><LI>
4. SP of Tom40</LI><P>    
Tom40 is the core component of the translocase of the outer mitochondrial membrane (TOM complex), presenting an β-barrel structure. The signal peptide of Tom40 is a conserved β-signal motif, located near the C-terminus, The sequence can be found in the document BMC Genomics, 12, 79. We synthesized the sequence and linked it with RFP &amp; YFP.</P><P>
The document we mainly refer to is:</P><P>
Imai, K., Fujita, N., Gromiha, M.M. and Horton, P. (2011) Eukaryote-wide sequence analysis of mitochondrial beta-barrel outer membrane proteins. BMC Genomics, 12, 79.</P></UL><UL><LI>
5. SP of Tom70 </LI><P>    
Tom70 is also one component of TOM complex, but only contain a single transmembrane segment at their N terminus. The signal peptide of Tom70 is the transmembrane domain comprising a hydrophilic, positively charged segment. The sequence can be found in <A href="http://www.uniprot.org/blast/?about=P07213" class="external free" rel="nofollow">http://www.uniprot.org/blast/?about=P07213</A>[11-30]. We synthesized the sequence and linked it with RFP &amp; YFP.</P><P>
The document we mainly refer to is:</P><P>
Rapaport, D. (2003). &quot;Finding the right organelle.&quot; EMBO Rep 4(10): 948-952. </P></UL><UL><LI>
6. SP of Tom22</LI><P>    
Tom22 is similar with Tom70, but it’s a tail-anchored protein. The signal peptide of Tom22 is also similar with the one of Tom70: moderately hydrophobic, relatively short, with positive charges at the flanking regions. The sequence can be found in <A href="http://www.uniprot.org/blast/?about=P49334" class="external free" rel="nofollow">http://www.uniprot.org/blast/?about=P49334</A>[98-119].  We synthesized the sequence and linked it with RFP &amp; YFP.</P><P>
The document we mainly refer to is:</P><P>
Plant Physiology July 2000 vol. 123 no. 3 811-816.</P></UL></DIV><DIV class="context"><H5><SPAN class="mw-headline" id="Construction_of_Signal_Peptide_System">Construction of  Signal Peptide System</SPAN></H5><UL><P>Due to some inaccurate information in the iGEM website and our own mistakes, some errors accrued when we attempted to link the reporter gene with signal peptide.</P></UL><UL><P>The followings are the reason why the errors took place and the approach we applied to solved the problem.</P></UL><UL><LI>Causes</LI></UL><UL><P>The general adapter is:</P></UL><UL><P>    GAATTC GCGGCCGC T TCTAGA G[ATG ... TAA TAA] T ACTAGT A GCGGCCG CTGCAG</P></UL><UL><P>And general adapter of fusion protein is:</P></UL><UL><P>    GAATTC GCGGCCGC T TCTAGA  [ATG ... TAA TAA]  ACTAGT A GCGGCCG CTGCAG         </P></UL><UL><P>But our adapter of signal peptide is:</P></UL><UL><P>    GAATTC GCGGCCGC T TCTAG   [ATG ... TAA TAA] T ACTAGT A GCGGCCG CTGCAG</P></UL><UL><P>Bad news came when we attempted to link the General adapter with the General adapter fragment. The fusion protein is as following:</P></UL><UL><P>    GAATTC GCGGCCGC T TCTAGA G[part1] T ACTAGA G [part2]T ACTAGT A GCGGCCG CTGCAG</P></UL><UL><P>As we can see, there are 8 bases at the fused locus, and in this case, frame shift mutation will occur.</P></UL><UL><P>We also produce fusion protein using two General adapter, the result is:</P></UL><UL><P>    GAATTC GCGGCCGC T TCTAGA [part1]  ACTAGA  [part2] ACTAGT A GCGGCCG CTGCAG</P></UL><UL><P>There are 6 bases at the fused locus. No stop codon occurs, and it will not cause frame shift mutation.</P></UL><UL><P>If we use our adapter to link the adapter we designed, the locus is:</P></UL><UL><P>    GAATTC GCGGCCGC T TCTAG [part1] CTACTAGAG [part2] T ACTAGT A GCGGCCG CTGCAG</P></UL><UL><P>In this case, no frame shift mutation occurs, and there are a lot of GFPs available if we select this adapter for use.</P></UL><UL><P>However, when sequencing the fusion protein, we found the fusion locus had lost 2 bases thus causing frameshift mutation, and that the adapter of E1010 linked by backbone PSB1A3  became:</P></UL><UL><P>    GAATTC GCGGCCGC T TCTAG   [ATG ... TAA TAA] T ACTAGT A GCGGCCG CTGCAG</P></UL><UL><DIV class="figurep"><P>Figure 1. Sequence and mapping result.</P></DIV></UL><UL><LI>Solution</LI></UL><UL><P>To correct this error, we designed some primers to perform the point mutation.</P></UL><DIV class="figurep"><UL><P>Table 1. Primers we used in this experiment</P></UL><TABLE><TBODY><TR><TD><P>Common</P></TD><TD><P>5'-GTTTCTTCGAATTCGCGGCCGCTTCTAG</P></TD></TR><TR><TD><P>ZIM17-R</P></TD><TD><P>5'-GTTTCTTCCTGCAGCGGCCGCTATACTAGTAGATGTGCGATGAT</P></TD></TR><TR><TD><P>FER-R</P></TD><TD><P>5'-GTTTCTTCCTGCAGCGGCCGCTATACTAGTAGGGCCATGCAGCT</P></TD></TR><TR><TD><P>TIM21-R</P></TD><TD><P>5'-GTTTCTTCCTGCAGCGGCCGCTATACTAGTAGGGATTTTACTTG</P></TD></TR><TR><TD><P>TOM22-R</P></TD><TD><P>5'-GTTTCTTCCTGCAGCGGCCGCTATACTAGTAGCGGCAAGTATAG</P></TD></TR><TR><TD><P>TOM70-R</P></TD><TD><P>5'-GTTTCTTCCTGCAGCGGCCGCTATACTAGTAGGTTGTAATAATA</P></TD></TR><TR><TD><P>TOM20-R</P></TD><TD><P>5'-GTTTCTTCCTGCAGCGGCCGCTATACTAGTAGCCTGAATTGCGG</P></TD></TR><TR><TD><P>TOM40-R</P></TD><TD><P>5'-GTTTCTTCCTGCAGCGGCCGCTATACTAGTAGCAATTGAGGAAG</P></TD></TR></TBODY></TABLE></DIV><UL><P>COM-F is the forward primer of all signal peptides, and the other seven are the reverse primers of signal peptides. After our experimental adapters are as followings:</P></UL><UL><P>    GAATTC GCGGCCGC T TCTAG [part] CT ACTAG TA T A GCGGCCG CTGCAG</P></UL><UL><P>We add three bases in the adapter. If we link the signal peptide at N-terminal, the fused locus is:</P></UL><UL><P>    GAATTC GCGGCCGC T TCTAG [part1] CTACTAGAG [part2]T ACTAGT A GCGGCCG CTGCAG</P></UL><UL><P>    In this case, there are nine bases at the locus, and there aren’t stop codon or frame shift mutations, and our adapter can be used.</P></UL><UL><P>After such treatment, final adapter is:</P></UL><UL><P>    GAATTC GCGGCCGC T TCTAGA G [part] CT ACTAG TA T A GCGGCCG CTGCAG</P></UL><UL><P>If we add signal peptide at the N-terminal of adapter, fusion protein is:</P></UL><UL><P>    GAATTC GCGGCCGC T TCTAGAG [part1]  CTACTAGAG [part2] CT ACTAGT ATA GCGGCCG CTGCAG</P></UL><UL><LI>Protocol</LI></UL><UL><P>Step 1:Extraction plasmid</P></UL><UL><P>    Zim17, FER, TIM21, TOM22, TOM70, TOM20, TOM50</P></UL><UL><P>Step 2:Point mutation</P></UL><UL><P>    PCR system(take ZIM17 as example):</P></UL><UL><P>        ZIM17 plasmid template    0.2ul</P></UL><UL><P>        Ex Taq    0.5ul</P></UL><UL><P>        10 x Ex Taq Buffer    2.0ul</P></UL><UL><P>        dNTP Mix    1.0ul</P></UL><UL><P>        COM-Fv0.5ul</P></UL><UL><P>        ZIM17-R    0.5ul</P></UL><UL><P>        ddH2O    15.3ul</P></UL><UL><P>    Reaction condition:</P></UL><UL><P>        94℃    5min</P></UL><UL><P>        94℃v30s</P></UL><UL><P>        55&amp;#8451    30s    30 cycle</P></UL><UL><P>        72℃    30s</P></UL><UL><P>        72℃    10min</P></UL><UL><P>        12℃    ∞</P></UL><UL><P>Step 3:Add 4ul, 6 x loading Buffer, and Run agarose gel. The result are as followings:</P></UL><UL><DIV class="figurep"><P>Figure 2. Result for point mutation. Lane 1=100bp Marker  2=Tom70  4=Tom40  6=Tom20  8=sZIM!&amp; 10=FER  12=Tom22  13=Tim21</P></DIV></UL><UL><P>Step 4:gel extraction(use QIAquick Gel Extraction Kit)</P></UL><UL><P>Step 5:use NANODROP 2000 to measure concentration.</P></UL><UL><P>Step 6: enzyme digestion</P></UL><UL><P>    enzyme digestion system:</P></UL><UL><P>        target fragment    8ul</P></UL><UL><P>        ECoRI    0.5ul</P></UL><UL><P>        SpeI    0.5u</P></UL><UL><P>        10 x H Buffer     1ul</P></UL><UL><P>    Reaction condition:    37℃ x 3h + 70℃ x 30min</P></UL></DIV><DIV class="context"><H5><SPAN class="mw-headline" id="Signal_Peptides_Validation_Protocol">Signal Peptides Validation Protocol</SPAN></H5><UL><LI>Purpose</LI></UL><UL><P>Verify that the mitochondrial signal peptides can lead peptides into the mitochondria, or onto the mitochondrial membrane, while the chloroplast signal peptides can not.</P></UL><UL><LI>Materials</LI></UL><UL><P>GFP, Signal peptides (ZIM17, FER, TIM21, TOM22, TOM70, TOM20, TOM40),  yeast strain.</P></UL><UL><LI>Methods</LI></UL><UL><LI>I. First Vector Construction</LI></UL><UL><P>1. Synthesis of ZIM17, FER, TIM21, TOM22, TOM70, TOM20, TOM40.</P></UL><UL><P>2. The synthetic fragments are connected with PUC57 the carrier.  Mix the strain which containing the carrier and 4 ml of LB+AMP liquid medium in 15 ml centrifuge tube, shake at 37 ℃, 220 rpm overnight.</P></UL><UL><P>3. Mix other two strains( BBa-E0030, BBa-E1010 containing biobrick) and 4 ml of LB+AMP  liquid medium in 15 ml centrifuge tube, shake at 37 ℃, 220 rpm overnight.</P></UL><UL><P>4. Extract plasmid ( ZIM17, FER, TIM21, TOM22, TOM70, TOM20, TOM40, BBa-E0030, BBa-E1010).</P></UL><UL><P>5. Use NANODROP 2000 from  Thermo Co. to measure plasmid concentration.</P></UL><UL><P>6. Digest: </P><P>     ZIM17, FER, TIM21, TOM22, TOM70, TOM20, TOM40 2-3ug respectively.</P><P>    
ECoRI 2ul</P><P>    
SpeI 2ul</P><P>    
10 x H Buffer 5ul</P><P>    
ddH2O 50ul</P><P>    
BBa-E0030, BBa-E1010 2-3ug respectively</P><P>    
ECoRI 2ul</P><P>    
XbaI 2ul</P><P>    
10 x M Buffer 5ul</P><P>    
ddH2O 50ul</P><P>     - Reaction conditions：37℃ x 3h + 70℃ x 30min</P></UL><UL><P>7. Run a 1% agarose gel 120V 45 min, and EB staining.</P></UL><UL><P>8. Gel Extraction( QIAquick Gel Extraction Kit).</P></UL><UL><P>9. Measure the concentration of the recycle fragments.</P></UL><UL><P>10. Ligation:</P><P>    
(1) ZIM17(ECoRI/SpeI) 6ul (2) ZIM17(ECoRI/SpeI) 6ul</P><P>    
BBa-E0030(ECoRI/XbaI) 2ul  BBa-E1010(ECoRI/XbaI) 2ul</P><P>    
T4 ligase 1ul  T4 ligase 1ul</P><P>    
T4 ligase Buffer 1ul  T4 ligase Buffer 1ul</P><P>    
and the same to (3) ~ (14).</P><P>     - Ligation condition: Room temperature, 3h.</P></UL><UL><P>11. Transformation</P><P>     (1) UV sterilization half an hour earlier, and melt competent cells (DH5a) with ice bath.</P><P>    
(2) 10 μl ligation product + 50 μl of competent cells, with 30 min ice bath.</P><P>    
(3) 42 ℃ water bath heat shock 50s.</P><P>    
(4) cool down with ice for 2 min.</P><P>    
(5) Adding 500μl LB liquid medium without antibiotics, 37 ℃ shake (225 rpm) 1 h, recovery.</P><P>    
(6) 3000 rpm × 5 min, discard 400μl supernatant and percussion of the remaining liquid and mix.</P><P>    
(7) Overlay 60μl broth on solid LB medium with Kan resistance,  37 ℃, inverted overnight culture in incubator.</P></UL><UL><P>12. Validation positive clones</P><P>     dNTP mix (2.5mM) 1μl</P><P>    
PCR 10×buffer 2μl</P><P>    
VF 0.5μl</P><P>    
VR 0.5μl</P><P>    
Ex Taq 0.5μl</P><P>    
H2O 15.5μl</P><P>     
Primer sequence: </P><P>        
VF: TGCCACCTGACGTCTAAGAA</P><P>        
VR: ATTACCGCCTTTGAGTGAGC</P><P>    
94℃ 3 min</P><P>    
94℃ 30s</P><P>    
55℃ 30s 30 cycles</P><P>    
72℃ 1 min</P><P>    
72℃ 10 min</P><P>    
12℃ ∞</P><P>    
1% to 1.5% agarose gel electrophoresis, voltage &lt;130, time ≈ 40 min.</P><P>     - Result:</P></UL><UL><P>13. Sequencing</P><P>    
Pick single colony of PCR positive clones to sequencing</P></UL><UL><P>14. Named the correct vectors( for convenient description)</P><P>    
ZIM17-E0030, ZIM17-E1010, FER-E0030, FER-E1010, TIM2-E0030, TIM2-E1010, TOM22-E0030, TOM22-E1010, TOM70-E0030, TOM70-E1010, TOM20-E0030, TOM20-E1010, Tom40-E0030, Tom40-E1010</P></UL><UL><LI>II. Second Vector Construction</LI></UL><UL><P>1. Mix the strain which containing following carrier and 4 ml of LB+AMP liquid medium in 15 ml centrifuge tube, shake at 37 ℃, 220 rpm overnight.</P><P>    
ZIM17-E0030, ZIM17-E1010, FER-E0030, FER-E1010, TIM2-E0030, TIM2-E1010, TOM22-E0030, TOM22-E1010, TOM70-E0030, TOM70-E1010, TOM20-E0030, TOM20-E1010, Tom40-E0030, Tom40-E1010(kan resistant), T-gz-GAL1(Amp resistant).</P></UL><UL><P>2. Extract plasmid</P></UL><UL><P>3. Measure plasmid concentration</P></UL><UL><P>4. Digestion </P><P>    
ECoRI and XbaI:</P><P>    
ZIM17-E0030, ZIM17-E1010, FER-E0030, FER-E1010, TIM2-E0030, TIM2-E1010, TOM22-E0030, TOM22-E1010, TOM70-E0030, TOM70-E1010, TOM20-E0030, TOM20-E1010, Tom40-E0030, Tom40-E1010</P><P>    
ECoRI and SpeI:</P><P>    
T-gz-GAL1</P></UL><UL><P>5. Gel electrophoresis, recycle and measure plasmid concentration</P></UL><UL><P>6. Ligation </P><P>    
ZIM17-E0030, ZIM17-E1010, FER-E0030, FER-E1010, TIM2-E0030, TIM2-E1010, TOM22-E0030, TOM22-E1010, TOM70-E0030, TOM70-E1010, TOM20-E0030, TOM20-E1010, Tom40-E0030, Tom40-E1010 2ul </P><P>    
respectively ligating with T-gz-GAL1 6ul.</P></UL><UL><P>7. Transformation</P></UL><UL><P>8. PCR</P></UL><UL><P>9. Shake culture</P></UL><UL><P>10. Sequencing</P></UL><UL><P>11. Named the correct vectors( for convenient description) </P><P>    
GAL-ZIM17-E0030, GAL-ZIM17-E1010, GAL-FER-E0030, GAL-FER-E1010, GAL-TIM2-E0030, GAL-TIM2-E1010, GAL-TOM22-E0030, GAL-TOM22-E1010, GAL-TOM70-E0030, GAL-TOM70-E1010, GAL-TOM20-E0030, GALTOM20-E1010, GAL-Tom40-E0030, GAL-Tom40-E1010.</P></UL><UL><LI>III. Third Vector Construction</LI></UL><UL><P>1. Mix the strain which containing following carrier and 4 ml of LB+AMP liquid medium in 15 ml centrifuge tube, shake at 37 ℃, 220 rpm overnight.</P><P>    
GAL-ZIM17-E0030, GAL-ZIM17-E1010, GAL-FER-E0030, GAL-FER-E1010, GAL-TIM2-E0030, GAL-TIM2-E1010, GAL-TOM22-E0030, GAL-TOM22-E1010, GAL-TOM70-E0030, GAL-TOM70-E1010, GAL-TOM20-E0030, GAL-TOM20-E1010, GAL-Tom40-E0030, GAL-Tom40-E1010(kan resistant)</P><P>    
BBa-J63002(Amp resistant, yeast ADH terminator)</P></UL><UL><P>2. Extract plasmid</P></UL><UL><P>3. Measure plasmid concentration</P></UL><UL><P>4. Digestion SpeI and PstI with 10X H Buffer:</P><P>    
GAL-ZIM17-E0030, GAL-ZIM17-E1010, GAL-FER-E0030, GAL-FER-E1010, GAL-TIM2-E0030, GAL-TIM2-E1010, GAL-TOM22-E0030, GAL-TOM22-E1010, GAL-TOM70-E0030, GAL-TOM70-E1010, GAL-TOM20-E0030, GAL-TOM20-E1010, GAL-Tom40-E0030, GAL-Tom40-E1010</P><P>    
XbaI and PstI with 10X H Buffer: </P><P>    
BBa-J63002</P></UL><UL><P>5. Gel electrophoresis, recycle and measure plasmid concentration</P></UL><UL><P>6. Ligation   </P><P>    
GAL-ZIM17-E0030, GAL-ZIM17-E1010, GAL-FER-E0030, GAL-FER-E1010, GAL-TIM2-E0030, GAL-TIM2-E1010, GAL-TOM22-E0030, GAL-TOM22-E1010, GAL-TOM70-E0030, GAL-TOM70-E1010, GAL-TOM20-E0030, GAL-TOM20-E1010, GAL-Tom40-E0030, GAL-Tom40-E1010 2ul respectively ligating with BBa-J63002 6ul.</P></UL><UL><P>7. Transformation</P></UL><UL><P>8. PCR</P></UL><UL><P>9. Shake culture</P></UL><UL><P>10. Sequencing</P></UL><UL><P>11. Named the correct vectors( for convenient description) </P><P>    
GAL-ZIM17-E0030-J63002, GAL-ZIM17-E1010-J63002, GAL-FER-E0030-J63002, GAL-FER-E1010-J63002, GAL-TIM2-E0030-J63002, GAL-TIM2-E1010-J63002, GAL-TOM22-E0030-J63002, GAL-TOM22-E1010-J63002, GALTOM70-E0030-J63002, GAL-TOM70-E1010-J63002, GAL-TOM20-E0030-J63002, GALTOM20-E1010-J63002, GAL-Tom40-E0030-J63002, GAL-Tom40-E1010-J63002</P></UL><UL><LI>IV. Construction of yeast expression vector</LI></UL><UL><P>1. Mix the strain which containing following carrier and 4 ml of LB+AMP liquid medium in 15 ml centrifuge tube, shake at 37 ℃, 220 rpm overnight</P><P>    
GAL-ZIM17-E0030-J63002, GAL-ZIM17-E1010-J63002, GAL-FER-E0030-J63002, GAL-FER-E1010-J63002, GAL-TIM2-E0030-J63002, GAL-TIM2-E1010-J63002, GAL-TOM22-E0030-J63002, GAL-TOM22-E1010-J63002, GALTOM70-E0030-J63002, GAL-TOM70-E1010-J63002, GAL-TOM20-E0030-J63002, GALTOM20-E1010-J63002, GAL-Tom40-E0030-J63002, GAL-Tom40-E1010-J63002(kan resistant)</P><P>    
YEP352(Amp resistant, yeast ADH terminator)</P></UL><UL><P>2. Extract plasmid</P></UL><UL><P>3. Measure plasmid concentration</P></UL><UL><P>4. Digestion </P><P>    
XbaI and PstI:</P><P>    
GAL-ZIM17-E0030-J63002, GAL-ZIM17-E1010-J63002, GAL-FER-E0030-J63002, GAL-FER-E1010-J63002, GAL-TIM2-E0030-J63002, GAL-TIM2-E1010-J63002, GAL-TOM22-E0030-J63002, GAL-TOM22-E1010-J63002, GALTOM70-E0030-J63002, GAL-TOM70-E1010-J63002, GAL-TOM20-E0030-J63002, GAL-TOM20-E1010-J63002, GAL-Tom40-E0030-J63002, GAL-Tom40-E1010-J63002, YEP352</P></UL><UL><P>5. Gel electrophoresis, recycle and measure plasmid concentration</P></UL><UL><P>6. Ligation </P><P>    
GAL-ZIM17-E0030-J63002, GAL-ZIM17-E1010-J63002, GAL-FER-E0030-J63002, GAL-FER-E1010-J63002, GAL-TIM2-E0030-J63002, GAL-TIM2-E1010-J63002, GAL-TOM22-E0030-J63002, GAL-TOM22-E1010-J63002, GAL-TOM70-E0030-J63002, GAL-TOM70-E1010-J63002, GAL-TOM20-E0030-J63002, GAL-TOM20-E1010-J63002, GAL-Tom40-E0030-J63002, GAL-Tom40-E1010-J63002 6ul respectively ligating with YEP532 2ul.</P></UL><UL><P>7. Transformation</P></UL><UL><P>8. PCR</P></UL><UL><P>9. Shake culture</P></UL><UL><P>10. Sequencing</P></UL><UL><P>11. Named the correct vectors( for convenient description)</P><P>    
YEP-GAL-ZIM17-E0030-J63002, YEP-GAL-ZIM17-E1010-J63002, YEPGAL-FER-E0030-J63002, YEP-GAL-FER-E1010-J63002, YEP-GALTIM2-E0030-J63002, YEP-GAL-TIM2-E1010-J63002, YEP-GAL-TOM22-E0030-J63002, YEP-GAL-TOM22-E1010-J63002, YEP-GAL-TOM70-E0030-J63002, YEP-GAL-TOM70-E1010-J63002, YEP-GAL-TOM20-E0030-J63002, YEP-GAL-TOM20-E1010-J63002, YEPGAL-Tom40-E0030-J63002, YEP-GAL-Tom40-E1010-J63002</P></UL><UL><LI>V. Yeast transformation</LI></UL><UL><P>1. Extract plasmid</P><P>    
YEP-GAL-ZIM17-E0030-J63002, YEP-GAL-ZIM17-E1010-J63002, YEPGAL-FER-E0030-J63002, YEP-GAL-FER-E1010-J63002, YEP-GALTIM2-E0030-J63002, YEP-GAL-TIM2-1010-J63002, YEP-GAL-TOM22-E0030-J63002, </P><P>    
YEP-GAL-TOM22-E1010-J63002, YEP-GAL-TOM70-E0030-J63002, YEP-GAL-TOM70-E1010-J63002, YEP-GAL-TOM20-E0030-J63002, YEP-GAL-TOM20-E1010-J63002, YEPGAL-Tom40-E0030-J63002, YEP-GAL-Tom40-E1010-J63002</P></UL><UL><P>2. Preparation of yeast competent</P><P>    
(1) Streak a YPDA agar plate with a small portion of AH109 or Y187. Incubate the plate upside down at 30°C until colonies appear(~3days). Yeast strains can be  stored for up to 1 month at 4°C on YPDA medium in culture plates.</P><P>    
(2) Prepare 1.1X TE/LiAc Solution.</P><P>    
(3) Prepare YPDA liquid medium (Yeast  Protocols Handbook).</P><P>    
(4) Inoculate one colony(&lt;4 weeks old, 2–3mm in diameter) into 3ml of YPDA medium in a sterile, 15-ml centrifuge tube.</P><P>    
(5) Incubate at 30°C with shaking for 8hr.</P><P>    
(6) Transfer 5µl of the culture to a 250-ml flask containing 50ml of YPDA.</P><P>    
(7) Incubate at 30°C with shaking at 230–250 rpm for 16–20hr.The OD600 should reach 0.15–0.3.</P><P>    
(8) Centrifuge the cells at 700 x g for 5min at room temperature.</P><P>    
(9) Discard the supernatant and resuspend the cell pellet in 100 ml of YPDA.</P><P>    
(10) Incubate at 30°C for 3–5hr(OD600=0.4~0.5).</P><P>    
(11) Centrifuge the cells at 700 x g for 5min at room temperature.</P><P>    
(12) Discard the supernatant and resuspend the cell pellet in 60ml of sterile, deionized H2O.</P><P>    
(13) Centrifuge the cells at 700 x g for 5min at room temperature.</P><P>    
(14) Discard the supernatant and resuspend the cells in 3 ml of 1.1 X TE/LiAc Solution.</P><P>    
(15) Split the resuspension between two 1.5-ml microcentrifuge tubes (1.5 ml pertube).</P><P>    
(16) Centrifuge each tube at high speed for 15 sec.</P><P>    
(17) Discard the supernatant and resuspend each pellet in 600 µl of 1.1 X  TE/LiAc Solution.</P></UL><UL><P>3. Transformation</P><P>    
(1) 1.5ml centrifuge:</P><P>        
plasmid  0.2ug</P><P>        
Sperm DNA*(10 mg/ml) 0.1mg</P><P>        
Note: Sperm DNA:  When freshly prepared water bath boil for 20 minutes, and immediately inserted into the ice bath and stored at -20 °C</P><P>    
(2) Add 100 ul with 1 × TE / LiAc resuspended yeast competent cells  per tube, mixed with shaking;</P><P>    
(3) Each added 600ul PEG / LiAc, shaken vigorously (to improve the conversion efficiency), 30 °C, 200 rpm shaking culture 30 min; </P><P>    
(4) Each adding 70 ul DMSO, slowly inverted mix (not oscillating), 42 °C water bath thermal shock 15 min, insert into the ice bath to cool 1-2 min; </P><P>    
(5) 14,000rpm × 5 sec, discard supernatant, resuspend cells with 0.1 ml 1×TE, overlay on SD/-Ura solid medium, 30°C inverted culture 3 days in incubator.</P></UL><UL><P>4. Check out</P><P>    
Draw the colonies grown on the said plate  in the galactose-containing SD/-Ura solid plates, until the colonies grow, single colonies were picked, and microscopic examination under a fluorescence microscope and record the results.</P></UL></DIV></DIV><DIV id="gallery"><DIV class="printfooter">
Retrieved from &quot;<A href="http://2012.igem.org/Team:Shenzhen/Result/YAO.Channel">http://2012.igem.org/Team:Shenzhen/Result/YAO.Channel</A>&quot;</DIV></DIV><DIV id="footer-box" class="noprint"><DIV id="footer"><UL id="f-list"><LI id="t-recentchanges"><A href="/Special:RecentChanges" title="Recent changes">Recent changes</A></LI><LI id="t-whatlinkshere"><A href="/Special:WhatLinksHere/Team:Shenzhen/Result/YAO.Channel" title="List of all wiki pages that link here [j]" accesskey="j">What links here</A></LI><LI id="t-recentchangeslinked"><A href="/Special:RecentChangesLinked/Team:Shenzhen/Result/YAO.Channel" title="Recent changes in pages linked from this page [k]" accesskey="k">Related changes</A></LI><LI id="t-specialpages"><A href="/Special:SpecialPages" title="List of all special pages [q]" accesskey="q">Special pages</A></LI><LI><A href="/Special:Preferences">My preferences</A></LI></UL></DIV><DIV id="footer"><UL id="f-list"><LI id="t-print"><A href="/wiki/index.php?title=Team:Shenzhen/Result/YAO.Channel&amp;printable=yes" title="Printable version of this page [p]" accesskey="p">Printable version</A></LI><LI id="t-permalink"><A href="/wiki/index.php?title=Team:Shenzhen/Result/YAO.Channel&amp;oldid=188457" title="Permanent link to this revision of the page">Permanent link</A></LI><LI id="privacy"><A href="/2012.igem.org:Privacy_policy" title="2012.igem.org:Privacy policy">Privacy policy</A></LI><LI id="disclaimer"><A href="/2012.igem.org:General_disclaimer" title="2012.igem.org:General disclaimer">Disclaimers</A></LI></UL></DIV></DIV></UL></DIV></UL></DIV></UL></DIV></UL></UL></DIV></DIV></DIV></DIV></DIV></BODY></HTML>