<HTML xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en" dir="ltr">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki  ltr ns-0 ns-subject page-Team_Shenzhen_Notebook_YAO_Suicider"><DIV id="globalWrapper"><DIV id="top-section"><DIV id="p-logo"><A href="/Main_Page" title="Main Page">&quot;
	    </A></DIV><DIV id="menubar" class="left-menu noprint"><UL><LI class="selected"><A href="/Team:Shenzhen/Notebook/YAO.Suicider">Page               </A></LI><LI class="new"><A href="/wiki/index.php?title=Talk:Team:Shenzhen/Notebook/YAO.Suicider&amp;action=edit&amp;redlink=1">Discussion               </A></LI><LI><A href="/wiki/index.php?title=Team:Shenzhen/Notebook/YAO.Suicider&amp;action=edit">View source               </A></LI><LI><A href="/wiki/index.php?title=Team:Shenzhen/Notebook/YAO.Suicider&amp;action=history">History               </A></LI><LI style="color:#808080;cursor:default">teams</LI></UL></DIV><DIV class="right-menu noprint" id="menubar"><UL><LI id="pt-login"><A href="/wiki/index.php?title=Special:UserLogin&amp;returnto=Team:Shenzhen/Notebook/YAO.Suicider" title="You are encouraged to log in; however, it is not mandatory [o]" accesskey="o">Log in</A></LI></UL></DIV><DIV id="search-controls" class="noprint"><FORM action="/Special:Search" id="searchform"> </FORM></DIV></DIV><DIV id="content"><H1 class="firstHeading">Team:Shenzhen/Notebook/YAO.Suicider</H1><DIV id="bodyContent"><H3 id="siteSub" class="noprint">From 2012.igem.org</H3><DIV class="container"><DIV class="content"><UL class="ca-menu"><UL id="subs1"><LI id="ca-green1" class="ca-green"><A href="https://2012.igem.org/Team:Shenzhen/Team"><SPAN class="ca-icon">U</SPAN><DIV class="ca-content"><H2 class="ca-main">Team</H2></DIV></A></LI><DIV id="sub1" class="submenu"><LI class="ca-sub"><A href="https://2012.igem.org/Team:Shenzhen/Team">Consortium</A></LI><LI class="ca-sub"><A href="https://2012.igem.org/Team:Shenzhen/Team/BGI_College">BGI College</A></LI><LI class="ca-sub"><A href="https://2012.igem.org/Team:Shenzhen/Team/Instructors">Instructors</A></LI><LI class="ca-sub"><A href="https://2012.igem.org/Team:Shenzhen/Team/HUSTers">HUSTers</A></LI><LI class="ca-sub"><A href="https://2012.igem.org/Team:Shenzhen/Team/SCUTers">SCUTers</A></LI><LI class="ca-sub"><A href="https://2012.igem.org/Team:Shenzhen/Team/SCUers">SCUers</A></LI><LI class="ca-sub"><A href="https://2012.igem.org/Team:Shenzhen/Team/WHUers">WHUers</A></LI><LI class="ca-sub"><A href="https://2012.igem.org/Team:Shenzhen/Team/UESTCers">UESTCers</A></LI><LI class="ca-sub"><A href="https://2012.igem.org/Team:Shenzhen/Team/SEUers">SEUers</A></LI><LI class="ca-sub"><A href="https://2012.igem.org/Team:Shenzhen/Team/SCNUers">SCNUers</A></LI><LI class="ca-sub"><A href="https://2012.igem.org/Team:Shenzhen/Team/CUHKer">CUHKer</A></LI></DIV></UL><UL id="subs2"><LI id="ca-orange1" class="ca-orange"><A href="https://2012.igem.org/Team:Shenzhen/Project"><SPAN class="ca-icon">L</SPAN><DIV class="ca-content"><H2 class="ca-main">Project</H2></DIV></A></LI><DIV id="sub2" class="submenu"><LI class="ca-sub"><A href="https://2012.igem.org/Team:Shenzhen/Project">Stories</A></LI><LI class="ca-sub"><A href="https://2012.igem.org/Team:Shenzhen/Project/YAO.Genome">YAO.Genome</A></LI><LI class="ca-sub"><A href="https://2012.igem.org/Team:Shenzhen/Project/YAO.Channel">YAO.Channel</A></LI><LI class="ca-sub"><A href="https://2012.igem.org/Team:Shenzhen/Project/YAO.Sensor">YAO.Sensor</A></LI><LI class="ca-sub"><A href="https://2012.igem.org/Team:Shenzhen/Project/YAO.Suicider">YAO.Suicider</A></LI><LI class="ca-sub"><A href="https://2012.igem.org/Team:Shenzhen/Project/YAO.Factory">YAO.Factory</A></LI></DIV></UL><UL id="subs3"><LI id="ca-yellow1" class="ca-yellow"><A href="https://2012.igem.org/Team:Shenzhen/Result"><SPAN class="ca-icon">u</SPAN><DIV class="ca-content"><H2 class="ca-main">Result</H2></DIV></A></LI><DIV id="sub3" class="submenu"><LI class="ca-sub"><A href="https://2012.igem.org/Team:Shenzhen/Result">Summary</A></LI><LI class="ca-sub"><A href="https://2012.igem.org/Team:Shenzhen/Result/YAO.Genome">YAO.Genome</A></LI><LI class="ca-sub"><A href="https://2012.igem.org/Team:Shenzhen/Result/YAO.Channel">YAO.Channel</A></LI><LI class="ca-sub"><A href="https://2012.igem.org/Team:Shenzhen/Result/YAO.Sensor">YAO.Sensor</A></LI><LI class="ca-sub"><A href="https://2012.igem.org/Team:Shenzhen/Result/YAO.Suicider">YAO.Suicider</A></LI><LI class="ca-sub"><A href="https://2012.igem.org/Team:Shenzhen/Result/YAO.Factory">YAO.Factory</A></LI></DIV></UL><UL id="subs4"><LI id="ca-green2" class="ca-green"><A href="https://2012.igem.org/Team:Shenzhen/Biobricks"><SPAN class="ca-icon">S</SPAN><DIV class="ca-content"><H2 class="ca-main">BioBricks</H2></DIV></A></LI><DIV id="sub4" class="submenu"><LI class="ca-sub"><A href="https://2012.igem.org/Team:Shenzhen/Biobricks">Summary</A></LI><LI class="ca-sub"><A href="https://2012.igem.org/Team:Shenzhen/Biobricks/YAO.Genome">YAO.Genome</A></LI><LI class="ca-sub"><A href="https://2012.igem.org/Team:Shenzhen/Biobricks/YAO.Channel">YAO.Channel</A></LI><LI class="ca-sub"><A href="https://2012.igem.org/Team:Shenzhen/Biobricks/YAO.Sensor">YAO.Sensor</A></LI><LI class="ca-sub"><A href="https://2012.igem.org/Team:Shenzhen/Biobricks/YAO.Suicider">YAO.Suicider</A></LI><UL id="subs5"><LI id="ca-orange2" class="ca-orange"><A href="https://2012.igem.org/Team:Shenzhen/Notebook"><SPAN class="ca-icon">p</SPAN><DIV class="ca-content"><H2 class="ca-main">Notebook</H2></DIV></A></LI><DIV id="sub5" class="submenu"><LI class="ca-sub"><A href="https://2012.igem.org/Team:Shenzhen/Notebook">Team History</A></LI><LI class="ca-sub"><A href="https://2012.igem.org/Team:Shenzhen/Notebook/YAO.Genome">YAO.Genome</A></LI><LI class="ca-sub"><A href="https://2012.igem.org/Team:Shenzhen/Notebook/YAO.Channel">YAO.Channel</A></LI><LI class="ca-sub"><A href="https://2012.igem.org/Team:Shenzhen/Notebook/YAO.Sensor">YAO.Sensor</A></LI><LI class="ca-sub"><A href="https://2012.igem.org/Team:Shenzhen/Notebook/YAO.Suicider">YAO.Suicider</A></LI><LI class="ca-sub"><A href="https://2012.igem.org/Team:Shenzhen/Notebook/YAO.Factory">YAO.Factory</A></LI><UL id="subs6"><LI id="ca-green3" class="ca-green"><A href="https://2012.igem.org/Team:Shenzhen/Biosafety"><SPAN class="ca-icon">,</SPAN><DIV class="ca-content"><H2 class="ca-main">Biosafety</H2></DIV></A></LI></UL><UL id="subs7"><LI id="ca-yellow2" class="ca-yellow"><A href="https://2012.igem.org/Team:Shenzhen/Practices"><SPAN class="ca-icon">e</SPAN><DIV class="ca-content"><H2 class="ca-main">Practices</H2></DIV></A></LI><DIV id="sub7" class="submenu"><LI class="ca-sub"><A href="https://2012.igem.org/Team:Shenzhen/Practices">Summary</A></LI><LI class="ca-sub"><A href="https://2012.igem.org/Team:Shenzhen/Practices/Lectures">Lectures</A></LI><LI class="ca-sub"><A href="https://2012.igem.org/Team:Shenzhen/Practices/High_School">High School</A></LI><LI class="ca-sub"><A href="https://2012.igem.org/Team:Shenzhen/Practices/Teamwork">Teamwork</A></LI><UL id="subs8"><LI id="ca-orange3" class="ca-orange"><A href="https://2012.igem.org/Team:Shenzhen/Acknowledgment"><SPAN class="ca-icon">N</SPAN><DIV class="ca-content"><H2 class="ca-main">Acknow.</H2></DIV></A></LI><DIV id="sub8" class="submenu"><LI class="ca-sub"><A href="https://2012.igem.org/Team:Shenzhen/Acknowledgment">Summary</A></LI><LI class="ca-sub"><A href="https://2012.igem.org/Team:Shenzhen/Acknowledgment/Providers">Providers</A></LI><LI class="ca-sub"><A href="https://2012.igem.org/Team:Shenzhen/Acknowledgment/Advisors">Advisors</A></LI><LI class="ca-sub"><A href="https://2012.igem.org/Team:Shenzhen/Acknowledgment/Sponsors">Sponsors</A></LI><LI class="ca-sub"><A href="https://2012.igem.org/Team:Shenzhen/Acknowledgment/Home_Schools">Home Schools</A></LI></DIV><P><UL id="navigationMenu"><LI><A class="home" href="https://2012.igem.org/Team:Shenzhen/Notebook"><SPAN>Team History</SPAN></A></LI><LI><A class="genome" href="https://2012.igem.org/Team:Shenzhen/Notebook/YAO.Genome"><SPAN>YAO.Genome</SPAN></A></LI><LI><A class="channel" href="https://2012.igem.org/Team:Shenzhen/Notebook/YAO.Channel"><SPAN>YAO.Channel</SPAN></A></LI><LI><A class="sensor" href="https://2012.igem.org/Team:Shenzhen/Notebook/YAO.Sensor"><SPAN>YAO.Sensor</SPAN></A></LI><LI><A class="suicider" href="https://2012.igem.org/Team:Shenzhen/Notebook/YAO.Suicider"><SPAN>YAO.Suicider</SPAN></A></LI><LI><A class="houseyellow" href="https://2012.igem.org/Team:Shenzhen/Notebook/YAO.Factory"><SPAN>YAO.Factory</SPAN></A></LI></UL></P><DIV id="bodymain"><DIV class="context"><H5><SPAN class="mw-headline" id="Jounal">Jounal</SPAN></H5><UL><LI>April 14th – April 20th, 2012 
<P>Events:</P><P>Originally, we simply want our yeast cell die in four different ways,two ugly: various toxicity and membrane-dissolving, two graceful: apoptosis and autophagy.</P><P>For suicide, we put up with the idea that we let mitochondria to be the mediate between host cell death and mitochondrial dysfunction.</P><P>Firstly, we think of self-killer through apoptosis.</P><P>Therefore, we tried to learn about more knowledge of apoptosis in human cell and yeast cell, And figure out the difference between them by searching review and KEGG pathway.</P><P>We know that one kind of apoptosis pathway is mediated by cytochrome C. In human cytochrome C mediated apoptosis is mainly controlled by Bcl-2 and IAP family. Bcl-2 can separate into two groups: pro-apoptosis and anti apoptosis(anti-apoptosis is consisted of two classes: BH3only and be to activate BH3).</P><P>After we knew how cytochrome C activate the apoptosis, we tried to learn about how cytochrome C is fled away from outer mitochondrial membrane.</P><P>Contributors:</P><P>Yan Huang, Yang Deng, Xuefeng Luo, Senwei Tang, Jianhui Gong</P></LI></UL><UL><LI>April 21st – April 27th, 2012 
<P>Events:</P><P>We thought about that if any materials in the apoptosis pathway or its orthology or paralogy can flee out of outer mitochondrial membrane. We found that among all the human apoptosis-related factors in cytochrome C mediated apoptosis, CytC, ENDO, PKA, Cn and ATM have the paralogy in yeast. Yeast cell has 17 chromosomes, 16 in nucleus and 1 in mitochondrion, and all paralogy mentioned above are in nucleus. [1]</P><P>While in PE3-mediated pathway, the paralogies are ATM, CHK1, CHK2, p53R2, CytC. [2]</P><P>Contributors:</P><P>Ming Ni, Kang Kang, Yu Chen, Yang Deng, Jianhui Gong</P></LI></UL><UL><LI>April 28th – May 4th, 2012 
<P>Events:</P><P>The porin on OMM can allow small molecules (&lt;50000Dalton) in and out. Among the materials mentioned last week, cytC YJR048W has minimum aa(109), it can’t get through the pores.</P><P>Later we try to find out the paralogy to cytC YJR048W by ncbi-blast. The paralogy in Candida glabrata( another type of yeast) has aa (104). [3]We convert 50000Da to aa number, that is 54 (for 1aa=110 Da)</P><P>Contributors:</P><P>Yan Huang, Xuefeng Luo, Chenran Zhou, Xiaopeng Xu, Jianhui Gong</P></LI></UL><UL><LI>May 5th – May 11th, 2012 
<P>Events:</P><P>Try to find any peptide from mitochondrial matrix to intermembrane space or to outer membrane or even cytoplasm. But this fails. So based on the peptide sequence from mt matrix to inner membrane, we try to find out if there are any material can act like GFP when it show flurorescence, it can releast H2O2, or any material that can release H2O2, so that we can use the promoter can sense H2O2.</P><P>Contributors:</P><P>Qiwu XU Yu Chen, Kang Kang, Yan Huang , Senwei Tang, Jianhui Gong</P></LI></UL><UL><LI>May 12th – May 18th, 2012 
<P>Events:</P><P>Figure out the function of holing to e.coli and know that holing is like bax/bak in human apoptosis pathway and therefore infer that hoin can anchor on the outer membrane of ER and mitochondrion to release the calcium and protein inside.</P><P>Contributors:</P><P>Deng Yan, Senwei Tang, Jianhui Gong </P></LI></UL><UL><LI>May 19th – May 25th, 2012 
<P>Events:</P><P>Figure out that we can utilize the retrograde signaling to transfer the signal to nucleus but the premise is mt dysfunction? Maybe we can use DNase or RNase.</P><P>Contributors:</P><P>Kang Kang, Jianhui Gong</P></LI></UL><UL><LI>May 26th – June 1st, 2012 
<P>Events:</P><P>How to produce material in mt? Due to the similarity between mt and prokaryotic cell. E.coli Sigma 70 promoter? </P><P>Contributors:</P><P>Jianhui Gong</P></LI></UL><UL><LI>June 9th – June 15th, 2012 
<P>Events:</P><P>Brainstorming:</P><P>1.Mt suicide itself? </P><P>    Mt autophagy or toxicity or membrane desolving. </P><P>2.Mt mediated host cell death? </P><P>    Apoptosis</P><P>    2.1How to trigger apoptosis? Which way? CytC mediated? </P><P>        How CytC mediated apoptosis in human works? And how about in yeast cell? </P><P>        2.1.1The difference between these two? </P><P>        2.1.2Is there any paralogy can get through the inner and outer membrane of mt? </P><P>        2.1.3Is there any signal peptide from mt matrix to IM, OMM, cytoplasm? </P><P>        2.1.4There is retrograde signaling but the premise is the mt respiration hardship.</P><P>        2.1.5Respiration hardship can be imitated by mt dysfunction such as DNA fragmentation.</P><P>        2.1.6How to make DNA fragmentation? </P><P>            DNase? RNase? Phosphodiesterase? </P><P>        2.1.7Any repressor to TCA cycle? </P><P>    2.2How to sense the signal given to mt? </P><P>        2.2.1For the same reason that the inner membrane of mt is extremely close, and its cardiolipin resist to the polar material</P><P>        2.2.2Sense the signal from biosensor? Biosensor is not completed. </P><P>        2.2.3So we use T7 RNAP/T7 promoter to mimic the signal given to mt. </P><P>    2.3How to release CytC? </P><P>        2.3.1Bax/Bak? </P><P>        2.3.2Holin?? </P><P>    2.4How nucleus sense signal and produce holin? </P><P>        2.4.1Retrograde signaling RTG pathway from mt to nucleus. </P><P>Contributors:</P><P>Jianhui Gong</P></LI></UL><UL><LI>June 16th – June 22nd, 2012 
<P>Events:</P><P>Design the verification experiment for the whole story. </P></LI><LI><P>1. Originally, we try to construct such a artificial organell YAO that YAO receives signal either from biosensor (internal)or from cytoplasm (external) or consequently trigger the host cell apoptosis. However, for YAO are not completed, we use mitochondrion as experiment target. And since we can’t rely on the availability of YAO. sensor, the idea that Initial signal is given to YAO’s promoter is replaced by that nucleus produce T7 RNAP and transport it into mitochondrion to activate the T7 promoter inside it. </P></LI><LI><P>2. The final story is consist of six parts: </P></LI><LI><P>    a. Signal imitation: After galactose is added to the medium, RNAP can bind to Gal promoter and start transcribe and produce T7 RNAP with signal peptide to mitochondrial matrix. Then T7 RNAP get into mitochondrial matrix. </P></LI><LI><P>    b. Mitochondrion dysfunction: Once T7 promoter works, DNase is produced to fragmentate the mtDNA which can lead to Respiration difficulty. </P></LI><LI><P>    c. Retrograde signaling from mitochondrion: Mitochondrial dysfunction can trigger the RTG pathway which is the only known signaling from mt to nucleus. </P></LI><LI><P>    d. Holin production: DLD3 promoter is the only stable downstream promoter relevant with RTG pathway. It can be binded and activated by RTG1/3p which is final production of RTG pathway. Once mitochondrial dysfunction exits, holin with DLD3promoter can be expressed. </P></LI><LI><P>    e. Holin transformation and destruction: Once holin is produced, it can anchor on the outer membrane of mitochondria and ER, and form dimer with each other. Further more, they can form into a multidimer called dead draft if holin reachs a certain concentration. Dead drafts work like a large pore and release the apoptosis factor: cytochrome C, aif1 and calcium into cytoplasm. </P></LI><LI><P>    f. Apoptosis of host: Apoptosis factor cytochrome C and largely fleeing calcium can trigger the caspase-like pathway. Aif1 such as endo G can.get into nucleus and fragmentate DNA. Finally, host cell is suppressed or even die. </P></LI><LI><P>Contributors:</P><P>Qiwu Xu, Jianhui Gong</P></LI></UL><UL><LI>June 23rd – June 29th, 2012 
<P>Events:</P><P>For T7 imitation system: </P></LI><LI><P>    1. Verify the function of T7 RNAP/Promoter system in cytoplasm. If it works, fluorescence of GFP can be detected in cytoplasm. </P></LI><LI><P>    2. Verify the function of T7 RNAP/Promoter system in mitochondrion or unfinished YAO, basing on the availability of signal peptide from cytoplasm to mitochondrial matrix. If it works, fluorescence of mtGFP can be detected in mitochondrion. </P></LI><LI><P>For retrograde signaling part: </P></LI><LI><P>    3. Because DLD3 is one of the target of RTG pathway of yeast, lowering the basal expression is required. Verify the basal expression of gene downstream DLD3 promoter. Roughly judge the brightness of GFP, and consequently select a suitable degradation tag for gene downstream DLD3 promotor. </P></LI><LI><P>    4. Verify the function of selected degradation tag by using UV as a imitation of DNase produced in mitochondrion that can lead to mitochondrion dysfunction which will drive the gene expression downstream DLD3 promoter through Retrograde signaling from mitochondrion to nucleus. </P></LI><LI><P>    5. Verify that holing can make pores on the inner membrane of mitochondrion. This is the alterant of DNase causing mt dysfunction. And it can also produce a membrane without mtDNA. </P></LI><LI><P>For host cell suicide part: </P></LI><LI><P>    6. Verify the function of holing in yeast to prove that holing can act as bax/bak protein in human cell as a apoptosis activator. If it works, we can see that between the engineered and versus one, the growthrate curve (OD600) of engineered one tend to be lower than that of versus one. </P></LI><LI><P>Contributors:</P><P>Qiwu Xu, Yang Deng, Jianhui Gong </P></LI></UL></DIV><DIV class="context"><H5><SPAN class="mw-headline" id="References">References</SPAN></H5><UL><P>1. <A href="http://www.genome.jp/kegg-bin/show_pathway?map04210" class="external free" rel="nofollow">http://www.genome.jp/kegg-bin/show_pathway?map04210</A></P></UL><UL><P>2. <A href="http://www.genome.jp/kegg-bin/show_pathway?hsa04115" class="external free" rel="nofollow">http://www.genome.jp/kegg-bin/show_pathway?hsa04115</A></P></UL><UL><P>3. <A href="http://www.genome.jp/dbget-bin/www_bget?cgr:CAGL0I01408g" class="external free" rel="nofollow">http://www.genome.jp/dbget-bin/www_bget?cgr:CAGL0I01408g</A></P></UL></DIV></DIV><DIV id="gallery"><DIV class="printfooter">
Retrieved from &quot;<A href="http://2012.igem.org/Team:Shenzhen/Notebook/YAO.Suicider">http://2012.igem.org/Team:Shenzhen/Notebook/YAO.Suicider</A>&quot;</DIV></DIV><DIV id="footer-box" class="noprint"><DIV id="footer"><UL id="f-list"><LI id="t-recentchanges"><A href="/Special:RecentChanges" title="Recent changes">Recent changes</A></LI><LI id="t-whatlinkshere"><A href="/Special:WhatLinksHere/Team:Shenzhen/Notebook/YAO.Suicider" title="List of all wiki pages that link here [j]" accesskey="j">What links here</A></LI><LI id="t-recentchangeslinked"><A href="/Special:RecentChangesLinked/Team:Shenzhen/Notebook/YAO.Suicider" title="Recent changes in pages linked from this page [k]" accesskey="k">Related changes</A></LI><LI id="t-specialpages"><A href="/Special:SpecialPages" title="List of all special pages [q]" accesskey="q">Special pages</A></LI><LI><A href="/Special:Preferences">My preferences</A></LI></UL></DIV><DIV id="footer"><UL id="f-list"><LI id="t-print"><A href="/wiki/index.php?title=Team:Shenzhen/Notebook/YAO.Suicider&amp;printable=yes" title="Printable version of this page [p]" accesskey="p">Printable version</A></LI><LI id="t-permalink"><A href="/wiki/index.php?title=Team:Shenzhen/Notebook/YAO.Suicider&amp;oldid=194087" title="Permanent link to this revision of the page">Permanent link</A></LI><LI id="privacy"><A href="/2012.igem.org:Privacy_policy" title="2012.igem.org:Privacy policy">Privacy policy</A></LI><LI id="disclaimer"><A href="/2012.igem.org:General_disclaimer" title="2012.igem.org:General disclaimer">Disclaimers</A></LI></UL></DIV></DIV></UL></DIV></UL></DIV></UL></DIV></UL></UL></DIV></DIV></DIV></DIV></DIV></BODY></HTML>