Team:NTNU Trondheim/Parts
Home
Safety
Parts
Project
Description
Background
Results
Contribution
Demonstrate
Modeling
Lab Journal
Protocols
Human practices
Human practice
Public engagement
People
Team
Collaborations
Attributions
Sponsors
Awards
Achievements
Applied Design
Hardware
Parts
Overview of parts in this project:
Medal
Part name
Type
Description
Length (bp)
Bronze
BBa_K325909
Generator
Lux Operon (under pBAD)
7637
Silver
BBa_K2265000
Device
MP6 mutator (original)
4908
Our team improved the characterization of the BioBrick BBa_K325909 and designed and characterized the BioBrick BBa_K2265000 .                 Both are documented in the iGEM Registry.
Biobrick contribution: The MP6 mutator (original)
The MP6 mutator (original) biobrick is a plasmid with an operon under control of the promoter pBAD                 together with the transcription factor gene AraC, and six other genes that disrupt DNA replication:                 DnaQ926, Dam, SeqA, EmrR, Ugi and Cda1. We designed this biobrick to accelerate bacterial mutation rate,                 which could be useful for our evolution experiments.
The DnaQ926 is a dominant negative variant of the Escherichia coli DNA pol III proofreading domain. Dam has a                 strong mutator effect due to impaired mismatch repair. The SeqA protein negatively regulates the initiation                 of DNA replication at the origin of replication. A low expression of SeqA reduces the transcription of                 the MP6 plasmid in absence of arabinose. The EmrR gene codes for a protein that compromises intracellular                 dNTP pools. Ugi codes for a protein that inhibit Ung, a mutagenesis preventing enzyme, through mimicry                 of structural and electronic features of uracil-containing DNA. CDA1 codes for a cytidine deaminase                 that is reported to promote the mutation of both prokaryotic and eukaryotic genomic DNA.[1]
The biobrick is based on the MP6 plasmid from addgene (Addgene plasmid # 69669). The original plasmid                 was changed in order to make it a biobrick. The change done to the plasmid was removing a Spel restriction                 site by changing an A to a C (nucleotide number 4971 in the original MP6 plasmid). A biobrick prefix was                 also added in front of the araC gene and a biobrick suffix after the CDA1 gene.
  Source:
[1] Development of potent in vivo mutagenesis plasmids with broad mutational spectra. Badran AH, Liu DR.                  Nat Commun. 2015 Oct 7;6:8425. doi: 10.1038/ncomms9425. 10.1038/ncomms9425 PubMed 26443021
Characterization of the Lux Operon biobrick
We chose to improve this biobrick as we originally wanted to use the lux operon as a reporter for our                 arabinose-inducible system. In this way, we could test how the expression from the pBAD promoter was                 dependent on arabinose concentration.
By looking at previous characterisation results of the biobrick, we thought it would work reasonably to                 use DH5α competent E. coli cells and an incubation temperature of 37 °C. However, when testing with                 0.01M arabinose concentrations, no observable luminescence was detected at any time after induction.                 Therefore, we decided to characterise the biobrick by measuring the luminescence from the lux operon                 at different arabinose concentrations and temperatures. We measured the luminescence with a Tecan M200                 plate reader, using four different temperatures and six different arabinose concentrations. As the                 machine could only perform heating and not cooling, it was challenging to measure at temperatures                 below 29 °C.
The results are given on the biobrick parts page .
Email: igem-team@ntnu.no
