"
Page
Discussion
View source
History
teams
Log in
 
Team:HUST-China/Protocol/Part2
From 2013.igem.org
HOME
PROJECT
PROTOCOL
MODELLING
HUMAN PRACTICE
TEAM
Materials
Methods
The output evaluation of the recombination E.coli by HPLC
·Materials
Table 2-1: Bacterial strains and plasmids
Strains and vectors
Relevant genotype and characteristics
Originate
E.coli K12
strains
conserved in the lab
E.coli DH5 α
strains
conserved in the lab
E.coli BL21
strains
conserved in the lab
2118CA
vectors
iGEM package
Table 2-2: Primers used in this part
(* Midi Plasmid Kit , Midi Purification Kit and Pfu Taq produced by TIANGEN BIOTECH(BEIJING)CO.,LTD.LA Taq produced by TaKaRa Biotechnology (DaLian)Co. ,Ltd.)
Primer
Sequence（5’-3’）
rbs-sbm-f
CTTAACTTTAAGAAGGAGATATACATATGTCTAACGTGCAGGAGTGGCAACAGCT
std.sbm-f
CGAATTCGCGGCCGCTTCTAGAGTTAACTTTAAGAAGGAGATATACATATG
std.sbm-r
CCTGCAGCGGCCGCTACTAGTATTAATCATGATGCTGGCTTA
rbs-ygfG-f
CTTAACTTTAAGAAGGAGATATACATATGTCTTATCAGTATGTTAACGTTGTCACTATCA
std.ygfG-f
CGAATTCGCGGCCGCTTCTAGAGTTAACTTTAAGAAGGAGATATAC
std.ygfG-r
CCTGCAGCGGCCGCTACTAGTATTAATGACCAACGAAATTAG
rbs-ygfH-f
CTTAACTTTAAGAAGGAGATATACATATGGAAACTCAGTGGACAAGGATGACCGCCAA
std.ygfH-f
CGAATTCGCGGCCGCTTCTAGAGTTAACTTTAAGAAGGAGATATACATATGGAAACTCAG
std.ygfH-r
CCTGCAGCGGCCGCTACTAGTATTAACCCAGCATCGAGCC
rbs-ygfD-f
CTTAACTTTAAGAAGGAGATATACATATGATTAATGAAGCCACGCTGGCAG
std.ygfD-f
CGAATTCGCGGCCGCTTCTAGAGTTAACTTTAAGAAGGAGATATACAT
std.ygfD-r
CCTGCAGCGGCCGCTACTAGTATTAATCAAAATATTGCGTCT
VR
ATTACCGCCTTTGAGTGAGC
VF2
TGCCACCTGACGTCAAGAA
·Methods
Step 1 : PCR amplification with the primer(rbs-f/std-r) set as the template (E.coli K12 genome) will produce four fragments namely ygfG(786bp), ygfH(1479bp), ygfD(996bp) and Sbm(2145bp) with RBS and standard suffix added. And to follow we used PCR production as template and primer pair (std-f/std-r) to add standard prefix. The conditions of the reaction were listed in table 2-3 and fig 2-1 displayed the result testified by 1% agarose gel electrophoresis.
Table 2-3 The PCR reaction to harvest target genes
(*Item 2 to item 6 for 36 cycles)
Item
System components(50μl)
Conditions
1
Template  1.25μl
94℃5min
2
dNTPs 2.5μl
94℃30sec
3
10×LA PCR Buffer 5μl
58℃30sec
4
LA Tag 0.5μl
72℃(1min for ygfG, ygfD)
5
Primer-F(10μmol/L)2.5μl
72℃(1min30sec for ygfH)
6
Primer-R(10μmol/L)2.5μl
72℃(2min for sbm)
7
ddH2O up to 50μl
72℃ 10min
Fig 2-1 the four genes in the gel
1.Marker 5000  2.ygfG  3.ygfH    4.ygfD   5.Sbm(Marker 5000: 5000/3000/2000/ 1500 /1000/ 750 /500 /250/ 100)
Step 2 : After purification of the four genes, we digest them with restriction enzyme SpeI and XbaI;Digest vector 2118CA with SpeI
Table 2-4 Conditions for gene digestion
(*Item 2 to item 4 for 34 cycles)
System components(20μl)
Conditions
DNA        15.0μl
37℃  4h
SpeI enzyme  1.0μl
37℃  4h
XbaI enzyme  1.0μl
37℃  4h
ddH2O       1.0ul
37℃  4h
Table 2-5 Conditions for vector digestion
System components(30μl)
Conditions
DNA        25.0μl
37℃  4h
SpeI enzyme  2.0μl
37℃  4h
10*M buffer   2.0ul
37℃  4h
Fig 2-2 Gellectrophoresis of four genes and vector after digest
1.Maker 5000  2.Sbm  3.ygfG  4.ygfH 5.ygfD   6.2118CA  7.2118CA   8.negative control
After gel extraction, digested genes and vector were used for DNA ligation.
Table 2-6  Conditions for ligation
Item
System components(10μl)
Conditions
Solution I  5μl
16℃ 8h
DNA      4μl
16℃ 8h
2118CA   1μl
16℃ 8h
Step 3 : Then transfer the conjunctions into BL21 strain to pick the positive clone after being inoculated in the LB solid culture, 37℃ for one night. To ensure the conjunctions were correct, we confirmed the vectors by colonial PCR. Later the 2118CA holding ygfD \ygfH\ygfG\sbm were sent to sequence.
Table 2-7 The colonial PCR to find positive clone
(*Item 2 to item 6 for 35 cycles)
Item
System components(10μl)
Conditions
1
Template  1.0μl
94℃5min
2
Template  1.0μl
94℃30sec
3
Primer-R (std-r-gene)  (10μmol/L)0.5ul
57℃30sec
4
Primer-R (std-r-gene)  (10μmol/L)0.5ul
72℃(1min for ygfG, ygfD 1min30sec for ygfH 2min for Sbm)
5
Primer-R (std-r-gene)  (10μmol/L)0.5μl
72℃(1min30sec for ygfH)
6
Primer-R(10μmol/L)2.5μl
72℃(2min for sbm)
7
ddH2O  3.0μl
4℃ hold
Step 4 : Once the gene was confirmed to be correct without any lethal mutation, the positive clone will be saved for fermentation. At the same time, we did some works on the conditions of fermentation and HPLC.
The optimum time to measure the propionate：         1． Measure the optical absorption in 600nm just when the E.coli BL21 was incubated with LB broth. The O.D values 0.11 or so.
2． At the first three hours, transfer 1.5ml liquid out of the micro-culture medium once an hour to three 2ml EP tubes. Measure them in the same ultraviolet wavelength.
3． From the fourth hour, transfer 300ul liquid out of the medium once an hour to four 2ml EP tubes. Measure them in the same ultraviolet wavelength with 5 times diluted.
4． From the eighth hour, transfer 200ul liquid out of the medium twice an hour to five 2ml EP tubes. Measure them in the same ultraviolet wavelength with 10 times diluted.
5． Cease the measurement when the fermentation continues for 18 hours. Draw the line graph showing the relationship between the O.D600 values and the time. The recorded O.D600 value and the graph were listed in table 2-8 and fig 2-3.
Table 2-8 Changing O.D600 value recorded for 18h
Time/h
0
1
2
3
4
5
6
8
10
12
14
16
18
O.D600nm
0.11
0.18
0.44
0.92
1.7
2.32
2.69
2.99
3.47
3.4
4.21
4.23
4.08
Fig.2-3 Growth curve of BL21strain
The standard curve of propionate:         1． Dilute the 0.996g/ml propionate in AR pure into different concentration: 8000mg/L,4000 mg/L,2000 mg/L,1000 mg/L,500 mg/L ,250 mg/L ,100 mg/L.
2． Filter the standard samples with the 0.45um membrane, then outgas them with ultrasound for about 10mins.
3． Measure the standard propionate of seven gradients by HPLC, record the peak area and draw a linear graph as fig3-4.
Fig.2-4
Step 5 : The fermentation of the recombination E.coli BL21
1. Grow the transformed cells at 37 °C in 25ml culture bottle with 5ml liquid LB for about 4h, and then transfer the medium to 250ml triangular flask with 100ml liquid LB. In order to make sure the production won’t be influenced by the beginning bacteria density, we measured the opacity at 600nm and ensure that the value was not far away from 0.11. Then the flasks were placed in reeling table at 37 °C. The wild type was treated in the same way.
2. 24h later, we sampled 2ml cell-culture medium and harvested the supernatant by centrifugation (12000 rpm at 4 °C for 10min) in an Eppendorf rotor. Then the harvest was filtered with the 0.22um membrane later outgassed with ultrasound for about 10mins.
3. Measure all the samples by HPLC, record the peak area and calculate the concentration according to the standard linear graph. See whether there is a difference between the wild type and recombination one.
CopyRight © 2013.igem.org  All Rights Reserved!
Powered by the iGEM team comes from HuaZhong University of Science and Techology  HUST-China
Retrieved from " http://2013.igem.org/Team:HUST-China/Protocol/Part2 "
Recent changes
What links here
Related changes
Special pages
My preferences
Printable version
Permanent link
Privacy policy
Disclaimers
