<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_Waterloo_InDepthDesign_Crispr skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><DIV id="top_title"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:Waterloo/InDepthDesign/Crispr</SPAN></H1></DIV><DIV id="HQ_page"><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><DIV class="cd-auto-hide-header" ng-mouseleave="hider()" ng-controller="nav" ng-cloak=""><NAV class="cd-primary-nav"><A href="#cd-navigation" class="nav-trigger"><SPAN>
        Menu
      </SPAN></A><UL id="cd-navigation" class="mobile-menu"><LI onclick="$(this).toggleClass('show-secondary-nav');"><A href="#" ng-click="clickety=true" ng-mouseover="subnav='project'; ">PROJECT</A></LI><LI><A class="paragraph-medium" href="/Team:Waterloo/Description">DESCRIPTION</A></LI><LI><A class="paragraph-medium" href="/Team:Waterloo/Design">DESIGN</A></LI><LI><A class="paragraph-medium" href="/Team:Waterloo/Requirements">REQUIREMENTS</A></LI><LI><A class="paragraph-medium" href="/Team:Waterloo/Model">MODELLING</A></LI></UL><LI onclick="$(this).toggleClass('show-secondary-nav');"><A href="#" ng-click="clickety=true" ng-mouseover="subnav='design';">BLUEPRINT</A></LI><LI><A class="paragraph-medium" href="/Team:Waterloo/InDepthDesign/HowItWorks">HOW IT WORKS</A></LI><LI><A class="paragraph-medium" href="/Team:Waterloo/Software">PREMATURE STOP CODON</A></LI><LI><A class="paragraph-medium" href="/Team:Waterloo/InDepthDesign/PromoterSelection">PROMOTER SELECTION</A></LI><LI><A class="paragraph-medium" href="/Team:Waterloo/InDepthDesign/PlasmidLossModel">PLASMID LOSS</A></LI><LI><A class="paragraph-medium" href="/Team:Waterloo/NetworkModelling">NETWORK MODELLING</A></LI><LI><A class="paragraph-medium" href="/Team:Waterloo/Aggregation">AGGREGATION MODEL</A></LI><LI><A class="paragraph-medium" href="/Team:Waterloo/InDepthDesign/Crispr">CRISPR</A></LI><LI onclick="$(this).toggleClass('show-secondary-nav');"><A href="#" ng-click="clickety=true" ng-mouseover="subnav='results';">RESULTS</A></LI><LI><A class="paragraph-medium" href="/Team:Waterloo/Proof">PROOF OF CONCEPT</A></LI><LI><A class="paragraph-medium" href="/Team:Waterloo/InDepthDesign/PlasmidLossModel">PLASMID LOSS</A></LI><LI><A class="paragraph-medium" href="/Team:Waterloo/Results/PromoterAnalysis">PROMOTER ANALYSIS</A></LI><LI onclick="$(this).toggleClass('show-secondary-nav');"><A href="#" ng-click="clickety=true" ng-mouseover="subnav='lab'">LAB</A></LI><LI><A class="paragraph-medium" href="/Team:Waterloo/Parts">PARTS</A></LI><LI><A class="paragraph-medium" href="/Team:Waterloo/Experiments">PROTOCOLS</A></LI><LI><A class="paragraph-medium" href="/Team:Waterloo/Notebook">NOTEBOOK</A></LI><LI><A class="paragraph-medium" href="/Team:Waterloo/Safety">SAFETY</A></LI><LI onclick="$(this).toggleClass('show-secondary-nav');"><A href="#" ng-click="clickety=true" ng-mouseover="subnav='practices'">POLICY</A></LI><LI><A class="paragraph-medium" href="/Team:Waterloo/HP/Silver">SILVER</A></LI><LI><A class="paragraph-medium" href="/Team:Waterloo/HP/Gold">GOLD</A></LI><LI><A class="paragraph-medium" style="font-weight: bold; color: #cd5f94" href="/Team:Waterloo/Engagement">ENGAGEMENT</A></LI><LI><A class="paragraph-medium" style="color: #cd5f94" href="/Team:Waterloo/Engagement/GenderEquity">GENDER EQUITY</A></LI><LI><A class="paragraph-medium" style="color: #cd5f94" href="/Team:Waterloo/Engagement/Ethics">ETHICS</A></LI><LI><A class="paragraph-medium" style="color: #cd5f94" href="/Team:Waterloo/Engagement/Access">ACCESS</A></LI><LI><A class="paragraph-medium" style="font-weight:bold; color: #5f94cd;" href="/Team:Waterloo/Integrated_Practices">INTEGRATED PRACTICES</A></LI><LI><A class="paragraph-medium" style="color: #5f94cd;" href="/Team:Waterloo/Integrated_Practices/OpenScience">OPEN SCIENCE</A></LI><LI><A class="paragraph-medium" style="color: #5f94cd;" href="/Team:Waterloo/Integrated_Practices/Networks">NETWORKS</A></LI><LI onclick="$(this).toggleClass('show-secondary-nav');"><A href="#" ng-click="clickety=true" ng-mouseover="subnav='team';">TEAM</A></LI><LI><A class="paragraph-medium" href="/Team:Waterloo/Team">THE TEAM</A></LI><LI><A class="paragraph-medium" href="/Team:Waterloo/Attributions">ATTRIBUTIONS</A></LI><LI><A class="paragraph-medium" href="/Team:Waterloo/Sponsors">SPONSORS</A></LI><LI><A class="paragraph-medium" href="/Team:Waterloo/Collaborations">COLLABORATION</A></LI></NAV><NAV class="cd-secondary-nav" id="projecttab" ng-show="subnav=='project'"><UL><LI><A class="paragraph-medium" href="/Team:Waterloo/Description">DESCRIPTION</A></LI><LI><A class="paragraph-medium" href="/Team:Waterloo/Design">DESIGN</A></LI><LI><A class="paragraph-medium" href="/Team:Waterloo/Requirements">REQUIREMENTS</A></LI><LI><A class="paragraph-medium" href="/Team:Waterloo/Model">MODELLING</A></LI></UL></NAV><NAV class="cd-secondary-nav" id="designtab" ng-show="subnav == 'design'"><UL><LI><A class="paragraph-medium" href="/Team:Waterloo/InDepthDesign/HowItWorks">HOW IT WORKS</A></LI><LI><A class="paragraph-medium" href="/Team:Waterloo/Software">PREMATURE STOP CODON</A></LI><LI><A class="paragraph-medium" href="/Team:Waterloo/InDepthDesign/PromoterSelection">PROMOTER SELECTION</A></LI><LI><A class="paragraph-medium" href="/Team:Waterloo/InDepthDesign/PlasmidLossModel">PLASMID LOSS MODEL</A></LI><LI><A class="paragraph-medium" href="/Team:Waterloo/Aggregation">AGGREGATION MODEL</A></LI><LI><A class="paragraph-medium" href="/Team:Waterloo/InDepthDesign/Crispr">CRISPR</A></LI><LI><A class="paragraph-medium" href="/Team:Waterloo/NetworkModelling">NETWORK MODELLING</A></LI></UL></NAV><NAV class="cd-secondary-nav" id="resultstab" ng-show="subnav == 'results'"><UL><LI><A class="paragraph-medium" href="https://2016.igem.org/Team:Waterloo/Proof">PROOF OF CONCEPT</A></LI><LI><A class="paragraph-medium" href="/Team:Waterloo/Results/PlasmidLoss">PLASMID LOSS</A></LI><LI><A class="paragraph-medium" href="/Team:Waterloo/Results/PromoterAnalysis">PROMOTER ANALYSIS</A></LI></UL></NAV><NAV class="cd-secondary-nav" id="labtab" ng-show="subnav == 'lab'"><UL><LI><A class="paragraph-medium" href="/Team:Waterloo/Parts">PARTS</A></LI><LI><A class="paragraph-medium" href="/Team:Waterloo/Experiments">PROTOCOLS</A></LI><LI><A class="paragraph-medium" href="/Team:Waterloo/Notebook">NOTEBOOK</A></LI><LI><A class="paragraph-medium" href="/Team:Waterloo/Safety">SAFETY</A></LI></UL></NAV><NAV class="cd-secondary-nav" id="practices" ng-show="subnav == 'practices'"><UL><LI><A class="paragraph-medium" href="/Team:Waterloo/HP/Silver">SILVER</A></LI><LI><A class="paragraph-medium" href="/Team:Waterloo/HP/Gold">GOLD</A></LI><LI><A class="paragraph-medium" style="font-weight: bold; color: #cd5f94" href="/Team:Waterloo/Engagement">ENGAGEMENT</A></LI><LI><A class="paragraph-medium" style="color: #cd5f94" href="/Team:Waterloo/Engagement/GenderEquity">GENDER EQUITY</A></LI><LI><A class="paragraph-medium" style="color: #cd5f94" href="/Team:Waterloo/Engagement/Ethics">ETHICS</A></LI><LI><A class="paragraph-medium" style="color: #cd5f94" href="/Team:Waterloo/Engagement/Access">ACCESS</A></LI><LI><A class="paragraph-medium" style="font-weight:bold; color: #5f94cd;" href="/Team:Waterloo/Integrated_Practices">INTEGRATED PRACTICES</A></LI><LI><A class="paragraph-medium" style="color: #5f94cd;" href="/Team:Waterloo/Integrated_Practices/OpenScience">OPEN SCIENCE</A></LI><LI><A class="paragraph-medium" style="color: #5f94cd;" href="/Team:Waterloo/Integrated_Practices/Networks">NETWORKS</A></LI></UL></NAV><NAV class="cd-secondary-nav" id="teamtab" ng-show="subnav == 'team'"><UL><LI><A class="paragraph-medium" href="/Team:Waterloo/Team">THE TEAM</A></LI><LI><A class="paragraph-medium" href="/Team:Waterloo/Attributions">ATTRIBUTIONS</A></LI><LI><A class="paragraph-medium" href="/Team:Waterloo/Sponsors">SPONSORS</A></LI><LI><A class="paragraph-medium" href="/Team:Waterloo/Collaborations">COLLABORATION</A></LI></UL></NAV></DIV><DIV class="labsticker">
  Lab and Design
</DIV><DIV class="mathsticker">
  Math Modelling
</DIV><DIV class="container-fluid"><DIV class="bloodorangeisacolour"><DIV class="header-image" style="background-image: url(https://static.igem.org/mediawiki/2016/6/6c/T--Waterloo--office2.jpg);"><H3 class="header-text"><SPAN style="color: white; background-color: #000;opacity: 0.8;">CRISPRi</SPAN></H3></DIV></DIV></DIV><DIV class="wcontent-box"><DIV class="wcontent-title">
    Summary
  </DIV><DIV class="wcontent"><DIV class="wcontent-caption"><P>Figure 1: CRISPR action, demonstrating how Cas9 binds to sgRNA to cleave target DNA.</P></DIV><P>CRISPR (clustered regularly interspaced short palindromic repeats) is a system used for targeting specific sites of a genome (Cong, 2013; Jinek, 2012). The system is comprised of a Cas protein and an sgRNA which is a short guide RNA, usually about 20nt, which specifies the target site on the genome(Cong, 2013; Jinek, 2012). CRISPR is a very popular tool in genome editing because of its ability to cut specific sites, however it was originally discovered as an adaptive immune system in bacteria (Doudna, 2014). This function of the CRISPR system allows for a targeted knockout of a protein in question.
    </P></DIV></DIV><DIV class="wcontent-box"><DIV class="wcontent-title">
    Application for PriON to PriOFF
  </DIV><DIV class="wcontent"><P>Our system is a tool that allows the overexpression a protein of interest, which could be used to determine if the protein can slow the [PSI+] response or even cure it. It would also be useful to see if the opposite (a decrease in the expression of a protein of interest) could have similar effects. For instance, overexpressing hsp104 can cure the response by breaking down and refolding misfolded sup35, but knocking down hsp104 expression can also cure the response because the sup35 aggregates need hsp104 in order to replicate. The addition of a CRISPR interference system using dCas9 allows the targeting a protein of interest to cause this knockdown of expression. If we use our system to insert a premature stop codon in front of the CRISPR gene, then it will only be expressed during a [PSI+] state.</P></DIV></DIV><DIV class="wcontent-box"><DIV class="wcontent-title">
    dCas9
  </DIV><DIV class="wcontent"><DIV class="wcontent-caption"><P>Figure 2: CRISPRi knockdown action, demonstrating how dCas9 binds to sgRNA to prevent expression.</P></DIV><P>Most CRISPR systems use a Cas protein called Cas9 which makes a cut after binding to the DNA. Our system would use dCas9 which is a Cas protein that binds to a site and prevents transcription of that protein. This acts as a more reliable method for knocking out a protein, since cuts can sometimes be repaired effectively through non-homologous end joining. </P></DIV></DIV><DIV class="wcontent-box"><DIV class="wcontent-title">
    sgRNAs
  </DIV><DIV class="wcontent"><P>All sgRNAs must start with a certain sequence of nucleotides called a PAM site, or protospacer adjacent motif. This sequence is recognised by the Cas protein and is thus necessary for it to bind to the sgRNA. The sgRNA recognises the target DNA, thereby acting as an intermediate between the Cas protein and the target. The most common PAM site is NGG where N is any nucleotide.</P><P>The target DNA must be carefully chosen in order to avoid accidentally targeting genes important to the host organism, in this case <I>Saccharomyces cerevisiae</I>. CRISPR-Cas systems have been effectively employed in yeast hosts on a number of occasions, indicating that our intentions to do the same have a good chance of success (DiCarlo, 2013; 2015; Smith 2016).</P></DIV></DIV><DIV class="wcontent-box"><DIV class="wcontent-title">
   References
  </DIV><DIV class="wcontent"><P>Cong, L., Ran, F.A., Cox, D., Lin, S., Barretto, R., Habib, N., Hsu, P.D., Wu, X., Jiang, W., Marraffini, L.A., &amp; Zhang, F., 2013. Multiplex genome engineering using CRISPR/Cas systems. <I>Science</I> Feb 15;339(6121):819-23.</P><P>DiCarlo, J.E. et al., 2013. Genome engineering in Saccharomyces cerevisiae using CRISPR-Cas systems. <I>Nucleic acids research</I>, 41(7), pp.4336–43.</P><P>DiCarlo, J.E. et al., 2015. Safeguarding CRISPR-Cas9 gene drives in yeast. <I>Nature Biotechnology</I>, 33(12), pp.1250–1255.</P><P>Doudna, J.A. &amp; Charpentier, E., 2014. The new frontier of genome engineering with CRISPR-Cas9. <I>Science</I>, 346(6213), pp.1258096–1258096.</P><P> Jinek, M. et al., 2012. A Programmable Dual-RNA–Guided DNA Endonuclease in Adaptive Bacterial Immunity. <I>Science</I>, 337(6096), p.816 LP-821.</P><P>Smith, J.D. et al., 2016. Quantitative CRISPR interference screens in yeast identify chemical-genetic interactions and new rules for guide RNA design. <I>Genome Biology</I>, 17(1), p.45.</P></DIV></DIV><P class="footer-links" style="font-size:0.8em;text-align:center!important;"><A style="padding-right:0;font-size:1em;" href="http://igem.uwaterloo.ca">External Site</A>© 2016 Waterloo iGEM
    </P></DIV></DIV></DIV></DIV></DIV></BODY></HTML>