Team:Waterloo/Experiments
Menu
PROJECT
DESCRIPTION
DESIGN
REQUIREMENTS
MODELLING
BLUEPRINT
HOW IT WORKS
PREMATURE STOP CODON
PROMOTER SELECTION
PLASMID LOSS MODEL
NETWORK MODELLING
AGGREGATION MODEL
CRISPR
RESULTS
PROOF OF CONCEPT
PLASMID LOSS
PROMOTER ANALYSIS
LAB
PARTS
PROTOCOLS
NOTEBOOK
SAFETY
POLICY
SILVER
GOLD
ENGAGEMENT
GENDER EQUITY
ETHICS
ACCESS
INTEGRATED PRACTICES
OPEN SCIENCE
NETWORKS
TEAM
THE TEAM
ATTRIBUTIONS
SPONSORS
COLLABORATION
DESCRIPTION
DESIGN
REQUIREMENTS
MODELLING
HOW IT WORKS
PREMATURE STOP CODON
PROMOTER SELECTION
PLASMID LOSS MODEL
NETWORK MODELLING
AGGREGATION MODEL
CRISPR
PROOF OF CONCEPT
PLASMID LOSS
PROMOTER ANALYSIS
PARTS
PROTOCOLS
NOTEBOOK
SAFETY
SILVER
GOLD
ENGAGEMENT
GENDER EQUITY
ETHICS
ACCESS
INTEGRATED PRACTICES
OPEN SCIENCE
NETWORKS
THE TEAM
ATTRIBUTIONS
SPONSORS
COLLABORATION
Protocols
Escherichia coli
Agarose Gel
Agarose Gel Electrophoresis
Colony PCR
Column Purification
E. coli Transformation
Flow Cytometry
Gel Extraction
Inoculating Liquid Culture
Ligation
PCR Amplification
Plasmid Miniprep
Plating Cells
Restriction Enzyme Digest
Saccharomyces cerevisae
Fixation of Yeast Cells
Growth Curves
Yeast Transformation
Restriction Enzyme Digest
Restriction enzyme digest is a process in which restriction endonuclease identify a restriction site within the DNA sequence and cleave them, resulting in overhang or blunt ends. Type II restriction enzymes, such as EcoRI and PstI, allows you to cut at specific palindromic sites on multiple target DNA and attach them to one another by ligation of the complementary ends.
Calculation Guidelines
Total enzyme volume must be 10% of total reaction volume
Final buffer volume must be 10% total reaction volume
1000 ng DNA requires 1 uL of each enzyme
Ideal total volume is around 10 – 20 uL
Determine volume of DNA added with required amount and DNA concentration
Addition of ddH2O to make up reaction to desired volume
Example Restriction Digest Calculation:
A negative control is made with no restriction enzyme to test for any unexpected cleaving and contamination in the reagents used
Inoculating Liquid Culture
Asceptically pipette 5mL sterile liquid broth into a test tube. If there are no large pipette tips able to transfer 5mL of liquid broth, use a fresh falcon tube to measure.
Add appropriate antibiotic, if required, to desired concentration using a pipette. You will have to tilt the test tube so that the broth is close enough to the top of the tube that you can add the antibiotic to the broth. Dropping it in won’t work because you will typically be adding ~2.5mL and the drop will be too small to fall.
It is recommended that you make a “mastermix” with broth and antibiotics in it when you have many tubes that require the same antibiotic.
Plasmid Miniprep
This step is performed to isolate plasmid DNA from host cells. It is usually performed directly after growing cells in nutrient rich media and involves lysing the cells to extract the plasmid DNA.
Centrifuge inoculated broth at 13,000 rpm for 1 minute to separate medium from cell and discard supernatant.
Resuspend cells in 200 uL of resuspension buffer to digest RNA after the cells have been lysed.
Add 200 uL Lysis buffer to the cell mixture, invert the tube 5 times to mix and incubate at room temperature for 1 minute. The basicity of the lysis buffer causes the cells to break open and expose their contents.
Add 400 uL of Neutralization Buffer to renature the plasmid DNA and bring up the pH of the mixture. At this point, the neutralized solution should turn yellow and precipitate consisting of genomic DNA and cell debris should be visible.
Centrifuge at 13,000 rpm for 5 minutes to pellet down the precipitate and separate plasmid DNA from cell parts.
Transfer the supernatant to spin columns in 1.5 mL tubes.
Centrifuge 13,000 rpm for 1 minute and discard flow through.The negatively charged DNA will bind to the SiO2 in the column matrix.
Add 200 uL of Wash Buffer 1 to the spin column and centrifuge at 13,000 rpm for 1 minute.
Add 400 uL of Wash Buffer 2 to spin column and centrifuge at 13,000 rpm for 1 minute.  The DNA should be purified from contaminations in the previous steps.
Transfer spin column to a clean 1.5 mL microfuge tube and elute with 20 – 30 uL of preheated (65 oC) elution buffer. If the DNA is to be sent for sequencing, nuclease free water must be used to elute instead of elution buffer.
Allow elution buffer to incubate in spin column for 1 minute, and centrifuge to elute the DNA out of spin column. The plasmid DNA concentration and purity is now ready to be quantified in concentration through nanodrop.
Agarose Gel
Weigh out agarose to make 0.8 – 1.2 % gel concentration and pour into an Erlenmeyer flask. Depending on the size of the gel rig, typically 50 uL - 100 uL, ensure the correct amount of agarose in grams is measured on an analytical balance.
Measure out TAE to achieve the desired gel concentration and add it to the flask. If gel extraction is to be done after running the gel, add a pinch of guanosine.
Heat in microwave for 1 minute and swirl the mixture and heat again if necessary until all the agarose have been dissolved.
Once the agarose gel has cooled sufficiently, add gel red 1uL : 10 mL ratio of gel red to agarose gel.
Pour the molten gel into a casting tray and insert the comb to create wells, the gel will take 20 – 25 minutes to solidify.
When the gel is solidified, add TAE to completely submerge the gel.
The samples can now be loaded to the wells.
Agarose Gel Electrophoresis
This process allows the visualization of DNA by size and is used to confirm the success of a restriction digest process or to identify a specific segment of DNA after a PCR. Gel electrophoresis uses electric current to move the negatively charged DNA to move towards the positive electrode through the agarose gel matrix. Gel red is dissolved in the agar to allow for DNA visualization under UV light.
When running a diagnostic test, only a small amount of DNA (1 uL) is needed, while a gel extraction process requires all the DNA after the digest. A loading dye is added, typically 2 uL, to visually track the progress of the DNA within the gel and also to make the DNA denser so that it sinks into the well.
Calculation considerations:
DNA amount is usually 1-2 uL for a diagnostic test, while a gel extraction requires the full volume of the sample.
The loading dye is 6X, which means 1/6 of the total loading volume must consist of loading dye.
Water can be added to make up the rest of loading volume, typically 3 uL. A multiple of 6 uL volume is ideal to simplify amount of loading dye added.
Once calculation is done, add together the DNA, loading dye and water on a parafilm sheet and put identification on each sample.
Using a 10 uL pipette, transfer the samples into the wells of the agarose gel and record the sample position.
Apply the positive electrode at the opposite end of the wells and the negative electrode on the well’s end of the tray. The negatively charged DNA will travel to the positive electrode when carried by the current.
Apply 100 – 150 V to the gel and let the DNA travel for approximately 1 hour. The gel is now ready to be imaged under UV light.
Gel Extraction
Find the target DNA size to determine the band position that needs to be extracted from the gel and cut it out.
Using weighing by difference in a 15 mL Falcon tube, determine the weight of only the gel.
To dissolve the gel, add 3 mL of binding buffer for every gram of gel and heat in a 65 oC water bath.
Prepare the spin column in a 1.5mL tube and load the dissolved gel, incubating for 1 minute to let the DNA bind with the column matrix.
Centrifuge at 13,000 rpm for 1 minute and discard flow through. Yield may be increased if the flow through is reloaded into the column and centrifuged once more.
Add 500 uL of wash solution to purify DNA and centrifuge for 30 seconds. Washing step can be repeated for higher purity.
Transfer the spin column to a clean 1.5 mL tube and elute with 20 – 30 uL preheated (65 oC) elution buffer.
Incubate for 1 minute and centrifuge at 13,000 rpm for 1 minute. The DNA should now be ready for nanodrop to gain the concentration and purity.
Column Purification
This step allows you to cleanse DNA that has just completed a restriction digest to cleanse it from restriction enzymes so that it can be stored in a stable condition for an extended period.
Add 3 volumes of binding buffer to the total digest volume and then transfer the mixture to the spin column.
Incubate for 2 minutes to allow DNA to bind to the matrix.
Centrifuge at 13,000 rpm for 1 minute and discard the flow through.
The DNA is washed with 500 uL of wash solution and spin column is centrifuged once more for 30 seconds at 13,000 rpm.
Transfer the spin column to a clean 1.5mL tube and add  20 – 30 uL preheated (65 oC) elution buffer.
Incubate for 1 minute and centrifuge at 13,000 rpm for 1 minute to elute DNA out of the matrix. The DNA concentration and purity is now ready to be quantified by nanodrop.
Ligation
Ligation involves attaching two segments of DNA together and is key to inserting genes into an open cut in a plasmid. The process relies on restriction enzymes which will cut DNA at specific sites and produce DNA overhangs which are complementary to another segment of DNA.
The process requires vector DNA, insert DNA, Ligase, Buffer and ddH2O, all which has to be added in certain proportion. The ligation mixture is then allowed to react under room temperature for 4 – 24 hours.
Calculation consideration:
Base volume of vector and insert DNA from nanodrop concentration and add desired amount in nanograms (ng)
The insert : vector ratio should be 3:1 to 5:1 depending on size of insert and vector.
Ligase buffer must make up 10% of the total ligation reaction
For every ug of DNA in mixture, 1 uL of ligase must be added
Ideal final reaction volume is 10 – 20 uL
Example ligation recipe:
Two negative controls have to be set up for future ligation troubleshooting.
The first negative control contains no insert to test how well the Vector DNA ligates on itself.
The second negative control contains no ligase to test any ligation activity without the enzyme.
E. coli Transformation
This process results uptake of DNA into a competent cell and allowing the said cells to express these exogenous genes, a core aspect of synthetic biology.
The competent cells, previously frozen in -80 oC condition, is first thawed on ice.
In a 1.5 mL tubes, mix together 50 uL of competent cell and all the ligation mixture. If the DNA is from a miniprep, 1- 10 uL of the DNA is added depending on the concentration.
The cells and ligation mixture is allowed to incubate for 20 – 30 minutes.
The mixture is then heat shocked at 42 oC for 45 seconds and is put back on ice. This process temporarily turns cell membranes permeable and allow plasmid DNA to enter the competent cells.
The cells are incubated on ice for 5 minutes.
Add 750 uL of LB broth to the competent cells to allow for cell growth and incubate for 45 minutes to 1 hour in 37oC while agitating the mixture.
The cells are now ready to be plated in selective media. The plasmid DNA contains antibiotic resistance gene and will allow only the successfully transformed cells to survive in the selective media with the antibiotic.
A +ve control is made by using just the vector DNA to be transformed into the competent cell. The cells receiving the vector DNA should survive in the selective media should the transformation technique is performed correctly.
Plating Cells
After transformation, this step allows you to isolate cell colonies that have successfully taken in the plasmid DNA by allowing them to grow in a selective media.
Prepare agar plates containing the appropriate antibiotics appropriate to the transformed cell’s resistance.
Retrieve incubated cells and centrifuge 13,000 rpm for 1 minute to separate cells from LB broth.
Pour off supernatant and resuspend cells in remaining LB broth left on tube.
Draw the entire volume of cells with a pipette and apply to agar plate.
Use spreading technique to spread the cultures around the agar. Sterilize the cell spreader by dipping it to ethanol and burning the excess alcohol. Allow cell spreader to cool and spread the cells around the plate in a back and forth motion while rotating the plate, ensure cells are spread evenly.
Turn the plate upside down and incubate overnight in 37oC condition. The result should be individual cell colony growth on the agar surface containing the plasmid DNA in each cell.
PCR Amplification
Polymerase Chain Reaction takes a segment in a DNA template and amplifies it from a few copies in stock solution up to several order of magnitude by the assembly of primers and nucleotides along the template.
Each cycle of PCR includes a denaturation, annealing and elongation step. Denaturation of DNA templates produces a single strand DNA by raising the temperatures to break nucleotide bonds. Annealing step involves the association of single stranded DNA template with primers which attaches to the 3’ end on both template strands. Finally, elongation step allows the assembly of complementary nucleotides along the DNA template in a 5’ to 3’ direction. The process repeats itself from the denaturation step and more DNA strands is synthesized.
The reagents needed for PCR amplification is DNA template, forward primer, reverse primer, Buffer containing nucleotides and water to make up to volume.
PCR reagent example recipe:
Buffers used in the PCR reagent may vary from GoTaq, Q5 or Taq MM. However, the purpose is similar for each reagant in that buffers maintain ideal condition for PCR amplification and supplies the system with nucleotides in which elongation is possible.
PCR protocol (30 cycles)
Initial denaturation : 95oC  - 2 minutes
30 cycles
Denaturation : 95oC
Annealing: 60 – 70oC (primer Tm)
Elongation: 72 oC  - 1 minute/kb
Final extension : 72oC – 5 minutes
Hold: 4oC – indefinite
Validate success of PCR reaction with diagnostic gel step and column purify to cleanse DNA from primers
Annealing temperature is determined by the Tm of the primers currently in use. Designing forward and reverse primers must take this step to account in making both primer Tm as similar as possible.
Running multiple reaction PCR with differing primers of varying Tm can be accommodated using Temperature Gradient feature of PCR machine in applying different temperature to each PCR vessel.
-ve control with no template DNA is run alongside PCR to test for contaminations from primers or Taq polymerase when imaged in diagnostic gel.
Colony PCR
This PCR process is a useful tool to determine the presence of a desired gene in a cell colony. Unlike the standard PCR amplification step, this protocol’s purpose is solely to determine the presence of insert DNA in plasmids of transformed cells. The colony PCR process also ensures that the colony possessing the target gene is identified and plated in a separate agar plate for further sequencing or amplification.
Prepare as many microfuge tubes as the colonies you will colony PCR (ideally 16 – 20 tubes) and pipette 15 uL of nuclease free water into each PCR microfuge tube.
Draw up, using a 10 uL pipette tip, individual colonies from selective media.
Eject the cells from the pipette tip and aspirate until no visible clumps of cells remain. The process is repeated for every other individual colony in separate microfuge tubes until a desired sample amount is attained.
Prepare an agar plate with appropriate antibiotics applied and create a patch plate. In each patch spaces, pipette 10 uL of cells from the microfuge tubes and eject it directly within the patch.
Repeat the process for each of the microfuge tubes and record which patch plates corresponds to which microfuge tubes.
Similar to PCR amplification, the Polymerase Chain Reaction reagent requires DNA templates, forward and reverse primers, nucleotide containing buffers and nuclease free water to dilute the mixture.
Set up for example PCR reaction:
The PCR process is similar to PCR amplification in that a cell lysis process is included by heating the cells for a short period. The lysis process is followed with denaturation to make single stranded DNA templates, annealing step to attach primers to the single strands of DNA and finally an elongation step in which assembles complementary nucleotides extending from the primers and along the DNA templates.
PCR protocol (30 cycles)
Cell lysis and initial denaturation : 95 oC  - 10 minutes
30 cycles
Denaturation : 95 oC  - 30 seconds
Annealing: 55 oC (primer Tm) – 27 seconds
Elongation: 68 oC  - 1 minute/kb
Final extension : 72 oC – 5 minutes
Hold: 4 oC – indefinite
Annealing temperature is determined by the Tm of the primers currently in use. Designing forward and reverse primers must take this step to account in making both primer Tm as similar as possible.
-ve control is also included in which no template DNA is added to check for any potential nucleotide contamination in buffer or primers.
The PCR products is then run under a diagnostic gel to identify the colonies that possess the target genes. By the time the colonies in the patch plates have grown, you would have known which colonies possess the target gene and the colony is ready for sequencing.
Yeast Transformation
Modification of method of Schiestl, R. H. and Gietz, R. D. (1989) High efficiency transformation of intact yeast cells using single stranded nucleic acids as carrier. Curr. Genet. 16, 339-346
Inoculate strain to be transformed into 10 mL YPD and grow overnight at 30 oC and 230 rpm on the Glerum Lab shaker.
Normalize the culture to an OD600 of 0.1 in 100 mL of fresh YPD media
Can add more of the overnight culture then the calculation indicate
Grow cells at 30 °C with shaking at 230 rpm until the OD600 reaches 0.6 to 1
The OD can be allowed to go as high as 1.8
Centrifuge the cells at 2500g for 5 minutes in a sterile Falcon tube and remove the supernatant with a pipette
Resuspend the cells in 10 mL of TEL and centrifuge the cells again at 2500g for 5 minutes
If the full 100mL culture is needed, the pellets can be combined at this point
Resuspend the pellet (or combined pellets) in 500 uL of TEL. Volume can be modified if there are more or less cells, not exceeding 0.9 mL.
Aliquot the resuspended pellet into 100 uL volumes for the number of samples needed.
Add transforming DNA
Concentration should be between 500-5000 ng/uL
Never add more than 10 uL of DNA
Plasmid should be around 5-10 ug of DNA per tube
When transforming in a cassette for homologous recombination experiments; add as much DNA as can be containing in the 10 uL volume. This will require concentrating the DNA, most likely to a maximum amount of 5-7 ug of DNA following PCR and concentration via ethanol precipitation.
Add 5 uL of 10 mg/mL carrier salmon sperm or herring sperm (ssDNA)
Prior to the addition of the ssDNA, heat the sperm to 95°C for 5 minutes
Mix and incubate the transformed samples for 30 minutes at RT without shaking
Add 700 uL of PEG/TEL buffer and mix thoroughly by pipetting.
Transfer the mixed suspension to individual sterile Eppendorf tubes
Volume should be approximately 0.8 mL per tube.
Incubate the new tubes for 45 or more minutes at RT without shaking
Heat shock the tubes at 45°C for 10 minutes.
Centrifuge the samples in the microfuge for 1 minutes at 13,000 rpm.
Remove the supernatant with sterile aspiration needle
Resuspend the cells completely in 200 uL TE.
Centrifuge again for 1 minutes at 13, 000 rpm and remove the supernatant as indicated above.
Resuspend the cells in 200 uL TE
Spread the cells on selective medium (LB + Amp)
Growth Curves
Grow yeast with vector inside YPD media overnight
In the morning take the OD600
Normalize all samples to 0.1 in YPGal (and induce with the promoter if necessary)
Grow at 30 oC
Take the OD600 at timed intervals (such as every 3 hours)
Based on OD600, obtain 107 cells to use for yeast cell fixation and fluorometry
Fixation of Yeast Cells
Spin down sample containing 107 cells at 13,000 RPM for 1 minute and dispose of the supernate
Resuspend pellet in 1mL of dH2O
Spin down cells at 13,000 RPM for 1 minute
Dispose of the supernate
In the little volume of dH2O that is left after the disposure, vortex the pellet in it to resuspend the cells and to avoid clumping during fluorometry
Add 500 uL of 100% ethanol to fix the yeast cells to prevent further growth
Place fixed cells in 4oC fridge until needed for fluorometry
Flow Cytometry
Flow cytometry is a culture analysis / biomarker detection device that functions by allowing the cell culture to pass through a laser-based detector and quantify values such as cell shape and fluorescence. For this year’s project purposes, flow cytometer is used to monitor the presence of GFP and RFP expression in E.coli to test for CRISPR Cas9 sgRNA activity in RFP inhibition.
Initiate the flow cytometer and computer, specifically in that order and select the ISX software to begin your analysis. Once the program has opened, press “Startup” button and allow 45 minutes for the software to initialize.
To prepare the samples, add cold 1X formalin to cell culture in a 3:1 ratio. 1X PBS can also be used to substitute formalin. This process is preformed to halt the cell growth process for better comparison between the samples. A mixture of 50 uL of cells with 150 uL of cold 1X formalin should provide you enough cells.
Once the flow cytometer is ready, load the necessary template and graphs. Ideal graphs should include an aspect ratio to fluorescence graph. This graph will display the shapes of the cells passing through the detector, namely how spherical it is. A value of “0” indicates an ovoid shape, meaning it shows a cell undergoing mitosis. Meanwhile a value of “1” indicate a perfect sphere, which is the speed-beads used to break up cell clumps. For analysis purposes, an aspect ratio value of 0.2 – 0.9 should represent the cells to be analysed. A graph for fluorescence vs frequency should be made to quantify the numbers of each fluorescent intensity levels.
To analyse GFP and RFP, 432nm excitation laser and SSC should be on. Load the samples into the vial holder after pressing “Load”. The computer should briefly display the results of the analysis in real time, including the images of individual cell fluorescence and graph value.
When data is ready to be recorded, enter “20,000” as the cell count and set the destination of data storage. Press “Acquire” and wait until the analysis is complete. When finished, press “Okay” and “Return” to retrieve the analysed sample. The device will proceed to clean the vial holder for the next sample. Repeat all above steps for all the samples.
To end the analysis, press the “Shutdown” button and allow the device to sterilize, which will last for 45 minutes. After this period, the program should automatically shut down.
External Site © 2016 Waterloo iGEM
