Team:Waterloo/InDepthDesign/PromoterSelection
Menu
PROJECT
DESCRIPTION
DESIGN
REQUIREMENTS
MODELLING
BLUEPRINT
HOW IT WORKS
PREMATURE STOP CODON
PROMOTER SELECTION
PLASMID LOSS
NETWORK MODELLING
AGGREGATION MODEL
CRISPR
RESULTS
PROOF OF CONCEPT
PLASMID LOSS
PROMOTER ANALYSIS
LAB
PARTS
PROTOCOLS
NOTEBOOK
SAFETY
POLICY
SILVER
GOLD
ENGAGEMENT
GENDER EQUITY
ETHICS
ACCESS
INTEGRATED PRACTICES
OPEN SCIENCE
NETWORKS
TEAM
THE TEAM
ATTRIBUTIONS
SPONSORS
COLLABORATION
DESCRIPTION
DESIGN
REQUIREMENTS
MODELLING
HOW IT WORKS
PREMATURE STOP CODON
PROMOTER SELECTION
PLASMID LOSS MODEL
AGGREGATION MODEL
CRISPR
NETWORK MODELLING
PROOF OF CONCEPT
PLASMID LOSS
PROMOTER ANALYSIS
PARTS
PROTOCOLS
NOTEBOOK
SAFETY
SILVER
GOLD
ENGAGEMENT
GENDER EQUITY
ETHICS
ACCESS
INTEGRATED PRACTICES
OPEN SCIENCE
NETWORKS
THE TEAM
ATTRIBUTIONS
SPONSORS
COLLABORATION
Promoter Selection
Summary
It was necessary for us to choose promoters that expressed the constructs we were cloning into yeast cells. We wanted to characterize promoter expression with GFP before we decided which promoter to use in our system with the CFP-Hsp104 construct. Furthermore, we needed to induce a [PSI+] state by overexpression of Sup35 in the cell by inserting another copy of Sup35 into the system via a plasmid. Overall, we had two plasmids that we wanted to easily induce at any time.
We chose to characterize four main promoters that are commonly found in yeast: Gal1, Adh1, Met25, and Cup1. We cloned all of these into the pXP218 plasmid and then transformed them into yeast to test their strength with and without the presence of inducers. All of the promoters we picked are inducible except for Adh1, which acts constitutively. We made the promoter constructs by the traditional restriction digest cloning method after order the synthesized promoters with a GFP fluorescent marker and doing PCR amplification with appropriate primers.
Specifically, The promoters were added into the pXP218 plasmid between the restriction enzymes SphI and SalI, which are near the multiple cloning site in the plasmid.The criteria required for the plasmid were the following: there are no T7 promoter regions near the promoter insertion site, so that T7 does not give a false positive by increasing expression of the promoter being tested, it had to be able to replicate in yeast, be able to be constructed in Escherichia coli,  have a CEN/ARS or a 2 micron origin of replication, and have a bacterial origin of replication. Having a selective marker in the plasmid was also important an important factor so that we were able to select for transformants into yeast; some markers include LEU2, URA3, HIS3.
Pxp218 yeast expression vector showing the main features which include: the backbone of pUC18, a vector size of 6226, URA3 selectable marker, ampicillin bacterial resistance, temperature growth at 37 °C, with growth stains DH5alpha and a high copy number.
Why we chose our restriction enzymes
When selecting the section of the plasmid to insert the Gal1 promoter and GFP, it needed to be ensured that the restriction enzymes chosen for the plasmid were not located within the Gal1/GFP sequence. This process for ensuring that the selected enzyme was not in our insert was repeated for each promoter with the fused GFP. Furthermore, restriction enzymes that would be used to digest our vector backbone would need to be found at the ends of the insert. This could be done through PCR.The order for each construct was the promoter itself to initiate transcription, the kozak sequence to start translation, the stop codon to stop translation, GFP, ORF - a terminator to stop transcription, and two restriction enzyme sites to allow the vector to be cloned.To design the promoters, research was conducted regarding how much space would be required between the kozak sequence and each promoter as well as how much space is needed between all the elements. Our team used other plasmids with the same promoters we are testing and GFP as a template when designing the sequence between the promoter and GFP in our promoter to ensure the correct sequences were used. Each promoter insert that has been through PCR was ligated into the pSB1C3 and pXP218 plasmids. To ligate into the pSB1C3 plasmid specifically, we cut with E and P ligating into pXP218, and used SphI and SalI to cut. These restriction sites were created at the ends of our inserts by using primers that had the correct site.
For Gal1, the SphI and SalI restriction sites were used to create sticky ends when digested. This would allow the insert to properly ligate in the correct orientation into the PXP218 plasmid. GFP is required to be able to use fluorimetry on the plasmids to analyze fluorescence and to test our promotor. A small set of twelve nitrogenous bases between the GFP and Gal1 that was taken from other plasmids between a Gal1 and a various feature. Our team researched various plasmids containing Gal1 and took a sequence directly after the Gal1 promoter. The CYC1 terminator is used to end the insert while there is a biobrick prefix directly after the terminator. This is done to ensure it can be tested as an iGEM BioBrick. For Cup1 and Adh1, the same parts were used, except the actual promoter changed.
Our Promoters in Nature
Yeasts are an important expression host for the production of many recombinant proteins. The choice of the right promoter is a crucial point for efficient gene expression, as most regulations take place at the transcriptional level [1]. The promoters we have decided to use are MET25, GAL1, Cup1 and ADH1. All these promoters are found in Saccharomyces cerevisiae and therefore make excellent choices for our project which deals with prion aggregation in yeast.
The first promotor utilized in our project design was the MET25 promoter. The MET25 promoter is repressed when cells are grown in the presence of methionine [2]. In previous experiments, the MET25 promoter has been used for the production of human serum albumin (HSA) and HSA-fusion proteins in S. cerevisiae [3]. In media lacking methionine, the MET25 promoter yielded high expression levels of HSA [3]. Furthermore, the MET25 catalyzes the synthesis of homocysteine from O-acetylhomoserine [4].
Secondly, the GAL1 promoter was also used in our experiments. Expression of the GAL1 gene in S. cerevisiae is strongly repressed by growth on glucose. It is shown that two sites within the GAL1 promoter mediate glucose repression [5]. The promoter is tightly repressed by glucose and is strongly induced by galactose [6].
The use of the Cup1 was also essential to the use of our experiments. The CUP1 promoter is a yeast metallothionein gene promoter, and may be tightly modulated by copper [7]. It is inducible by adding copper to the medium [8].
The final crucial promoter to our experiment is alcohol dehydrogenase I (ADH1). The ADH1 promoter is a widely used constitutive yeast promoter [9]. On glucose, activity of the ADH1 promoter decreases during late exponential, ethanol production growth phase [10]. A full-length ADH1 promoter would be suitable for conditional expression of genes at high glucose concentrations [11].
References
Weinhandl, K., Winkler, M., Glieder, A., and Camattari, A. (2014). Carbon source dependent promoters in yeasts. Microbial Cell Factories, 13:5.
Mumberg, D., Müller, R., and Funk, M. (1994). Regulatable promoters of Saccharomyces cerevisiae: comparison of transcriptional activity and their use for heterologous expression. Nucleic Acids Research, 22:25. 5767-5768.
Solow, S. P., Sengbusch, J., and Laird, M. W. (2005). Heterologous protein production from the inducible MET25 promoter in Saccharomyces cerevisiae. Biotechnology Progress, 21:2. 617-620.
Thomas, D., Cherest, H., and Surdin-Kerjan, Y. (1989). Elements Involved in S-Adenosylmethionine-Mediated Regulation of the Saccharomyces cerevisiae MET25 Gene. Molecular and Cellular Biology, 9:8. 3292-3298.
Flick, J. S., and Johnston, M. (1990). Two Systems of Glucose Repression of the GAL] Promoter in Saccharomyces cerevisiae. Molecular and Cellular Biology, 10:9. 4747-4769.
Mumberg, D., Müller, R., and Funk, M. (1994). Regulatable promoters of Saccharomyces cerevisiae: comparison of transcriptional activity and their use for heterologous expression. Oxford University Press 22:25. 5767-5768.
Mascorro-Gallardo, J.O., Covarrubias, A. A., and Gaxiola, R. (1996). Construction of a CUP1 promoter-based vector to modulate gene expression in Saccharomyces cerevisiae. Gene, 172:1. 169-170.
Koller, A., Valesco, J., and Subramani, S. (2000). The CUP1 promoter of Saccharomyces cerevisiae is inducible by copper in Pichia pastoris. Yeast, 16. 651-656.
Weinhandl, K, Winkler, M., Glieder, A., Camattari, A. (2014). Carbon source dependent promoters in yeasts. Microbial Cell Factories, 13:5.
Ruohonen, L., Aalto, M. K., Keänen, S. (1995). Modifications to the ADH1 promoter of Saccharomyces cerevisiae for efficient production of heterologous proteins. Journal of biotechnology, 39:3. 193-20.
Partow, S., Siewers, V., Bjøm, S., Nielsen, J., Jérôme, M. (2010). Characterization of different promoters for designing a new expression vector in Saccharomyces cerevisiae. Yeast, 27. 955-964.
External Site © 2016 Waterloo iGEM
