<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_Waterloo_Description skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><DIV id="top_title"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:Waterloo/Description</SPAN></H1></DIV><DIV id="HQ_page"><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><DIV class="cd-auto-hide-header" ng-mouseleave="hider()" ng-controller="nav" ng-cloak=""><NAV class="cd-primary-nav"><A href="#cd-navigation" class="nav-trigger"><SPAN>
        Menu
      </SPAN></A><UL id="cd-navigation" class="mobile-menu"><LI onclick="$(this).toggleClass('show-secondary-nav');"><A href="#" ng-click="clickety=true" ng-mouseover="subnav='project'; ">PROJECT</A></LI><LI><A class="paragraph-medium" href="/Team:Waterloo/Description">DESCRIPTION</A></LI><LI><A class="paragraph-medium" href="/Team:Waterloo/Design">DESIGN</A></LI><LI><A class="paragraph-medium" href="/Team:Waterloo/Requirements">REQUIREMENTS</A></LI><LI><A class="paragraph-medium" href="/Team:Waterloo/Model">MODELLING</A></LI></UL><LI onclick="$(this).toggleClass('show-secondary-nav');"><A href="#" ng-click="clickety=true" ng-mouseover="subnav='design';">BLUEPRINT</A></LI><LI><A class="paragraph-medium" href="/Team:Waterloo/InDepthDesign/HowItWorks">HOW IT WORKS</A></LI><LI><A class="paragraph-medium" href="/Team:Waterloo/Software">PREMATURE STOP CODON</A></LI><LI><A class="paragraph-medium" href="/Team:Waterloo/InDepthDesign/PromoterSelection">PROMOTER SELECTION</A></LI><LI><A class="paragraph-medium" href="/Team:Waterloo/InDepthDesign/PlasmidLossModel">PLASMID LOSS</A></LI><LI><A class="paragraph-medium" href="/Team:Waterloo/NetworkModelling">NETWORK MODELLING</A></LI><LI><A class="paragraph-medium" href="/Team:Waterloo/Aggregation">AGGREGATION MODEL</A></LI><LI><A class="paragraph-medium" href="/Team:Waterloo/InDepthDesign/Crispr">CRISPR</A></LI><LI onclick="$(this).toggleClass('show-secondary-nav');"><A href="#" ng-click="clickety=true" ng-mouseover="subnav='results';">RESULTS</A></LI><LI><A class="paragraph-medium" href="/Team:Waterloo/Proof">PROOF OF CONCEPT</A></LI><LI><A class="paragraph-medium" href="/Team:Waterloo/InDepthDesign/PlasmidLossModel">PLASMID LOSS</A></LI><LI><A class="paragraph-medium" href="/Team:Waterloo/Results/PromoterAnalysis">PROMOTER ANALYSIS</A></LI><LI onclick="$(this).toggleClass('show-secondary-nav');"><A href="#" ng-click="clickety=true" ng-mouseover="subnav='lab'">LAB</A></LI><LI><A class="paragraph-medium" href="/Team:Waterloo/Parts">PARTS</A></LI><LI><A class="paragraph-medium" href="/Team:Waterloo/Experiments">PROTOCOLS</A></LI><LI><A class="paragraph-medium" href="/Team:Waterloo/Notebook">NOTEBOOK</A></LI><LI><A class="paragraph-medium" href="/Team:Waterloo/Safety">SAFETY</A></LI><LI onclick="$(this).toggleClass('show-secondary-nav');"><A href="#" ng-click="clickety=true" ng-mouseover="subnav='practices'">POLICY</A></LI><LI><A class="paragraph-medium" href="/Team:Waterloo/HP/Silver">SILVER</A></LI><LI><A class="paragraph-medium" href="/Team:Waterloo/HP/Gold">GOLD</A></LI><LI><A class="paragraph-medium" style="font-weight: bold; color: #cd5f94" href="/Team:Waterloo/Engagement">ENGAGEMENT</A></LI><LI><A class="paragraph-medium" style="color: #cd5f94" href="/Team:Waterloo/Engagement/GenderEquity">GENDER EQUITY</A></LI><LI><A class="paragraph-medium" style="color: #cd5f94" href="/Team:Waterloo/Engagement/Ethics">ETHICS</A></LI><LI><A class="paragraph-medium" style="color: #cd5f94" href="/Team:Waterloo/Engagement/Access">ACCESS</A></LI><LI><A class="paragraph-medium" style="font-weight:bold; color: #5f94cd;" href="/Team:Waterloo/Integrated_Practices">INTEGRATED PRACTICES</A></LI><LI><A class="paragraph-medium" style="color: #5f94cd;" href="/Team:Waterloo/Integrated_Practices/OpenScience">OPEN SCIENCE</A></LI><LI><A class="paragraph-medium" style="color: #5f94cd;" href="/Team:Waterloo/Integrated_Practices/Networks">NETWORKS</A></LI><LI onclick="$(this).toggleClass('show-secondary-nav');"><A href="#" ng-click="clickety=true" ng-mouseover="subnav='team';">TEAM</A></LI><LI><A class="paragraph-medium" href="/Team:Waterloo/Team">THE TEAM</A></LI><LI><A class="paragraph-medium" href="/Team:Waterloo/Attributions">ATTRIBUTIONS</A></LI><LI><A class="paragraph-medium" href="/Team:Waterloo/Sponsors">SPONSORS</A></LI><LI><A class="paragraph-medium" href="/Team:Waterloo/Collaborations">COLLABORATION</A></LI></NAV><NAV class="cd-secondary-nav" id="projecttab" ng-show="subnav=='project'"><UL><LI><A class="paragraph-medium" href="/Team:Waterloo/Description">DESCRIPTION</A></LI><LI><A class="paragraph-medium" href="/Team:Waterloo/Design">DESIGN</A></LI><LI><A class="paragraph-medium" href="/Team:Waterloo/Requirements">REQUIREMENTS</A></LI><LI><A class="paragraph-medium" href="/Team:Waterloo/Model">MODELLING</A></LI></UL></NAV><NAV class="cd-secondary-nav" id="designtab" ng-show="subnav == 'design'"><UL><LI><A class="paragraph-medium" href="/Team:Waterloo/InDepthDesign/HowItWorks">HOW IT WORKS</A></LI><LI><A class="paragraph-medium" href="/Team:Waterloo/Software">PREMATURE STOP CODON</A></LI><LI><A class="paragraph-medium" href="/Team:Waterloo/InDepthDesign/PromoterSelection">PROMOTER SELECTION</A></LI><LI><A class="paragraph-medium" href="/Team:Waterloo/InDepthDesign/PlasmidLossModel">PLASMID LOSS MODEL</A></LI><LI><A class="paragraph-medium" href="/Team:Waterloo/Aggregation">AGGREGATION MODEL</A></LI><LI><A class="paragraph-medium" href="/Team:Waterloo/InDepthDesign/Crispr">CRISPR</A></LI><LI><A class="paragraph-medium" href="/Team:Waterloo/NetworkModelling">NETWORK MODELLING</A></LI></UL></NAV><NAV class="cd-secondary-nav" id="resultstab" ng-show="subnav == 'results'"><UL><LI><A class="paragraph-medium" href="https://2016.igem.org/Team:Waterloo/Proof">PROOF OF CONCEPT</A></LI><LI><A class="paragraph-medium" href="/Team:Waterloo/Results/PlasmidLoss">PLASMID LOSS</A></LI><LI><A class="paragraph-medium" href="/Team:Waterloo/Results/PromoterAnalysis">PROMOTER ANALYSIS</A></LI></UL></NAV><NAV class="cd-secondary-nav" id="labtab" ng-show="subnav == 'lab'"><UL><LI><A class="paragraph-medium" href="/Team:Waterloo/Parts">PARTS</A></LI><LI><A class="paragraph-medium" href="/Team:Waterloo/Experiments">PROTOCOLS</A></LI><LI><A class="paragraph-medium" href="/Team:Waterloo/Notebook">NOTEBOOK</A></LI><LI><A class="paragraph-medium" href="/Team:Waterloo/Safety">SAFETY</A></LI></UL></NAV><NAV class="cd-secondary-nav" id="practices" ng-show="subnav == 'practices'"><UL><LI><A class="paragraph-medium" href="/Team:Waterloo/HP/Silver">SILVER</A></LI><LI><A class="paragraph-medium" href="/Team:Waterloo/HP/Gold">GOLD</A></LI><LI><A class="paragraph-medium" style="font-weight: bold; color: #cd5f94" href="/Team:Waterloo/Engagement">ENGAGEMENT</A></LI><LI><A class="paragraph-medium" style="color: #cd5f94" href="/Team:Waterloo/Engagement/GenderEquity">GENDER EQUITY</A></LI><LI><A class="paragraph-medium" style="color: #cd5f94" href="/Team:Waterloo/Engagement/Ethics">ETHICS</A></LI><LI><A class="paragraph-medium" style="color: #cd5f94" href="/Team:Waterloo/Engagement/Access">ACCESS</A></LI><LI><A class="paragraph-medium" style="font-weight:bold; color: #5f94cd;" href="/Team:Waterloo/Integrated_Practices">INTEGRATED PRACTICES</A></LI><LI><A class="paragraph-medium" style="color: #5f94cd;" href="/Team:Waterloo/Integrated_Practices/OpenScience">OPEN SCIENCE</A></LI><LI><A class="paragraph-medium" style="color: #5f94cd;" href="/Team:Waterloo/Integrated_Practices/Networks">NETWORKS</A></LI></UL></NAV><NAV class="cd-secondary-nav" id="teamtab" ng-show="subnav == 'team'"><UL><LI><A class="paragraph-medium" href="/Team:Waterloo/Team">THE TEAM</A></LI><LI><A class="paragraph-medium" href="/Team:Waterloo/Attributions">ATTRIBUTIONS</A></LI><LI><A class="paragraph-medium" href="/Team:Waterloo/Sponsors">SPONSORS</A></LI><LI><A class="paragraph-medium" href="/Team:Waterloo/Collaborations">COLLABORATION</A></LI></UL></NAV></DIV><TITLE>Waterloo iGEM</TITLE><DIV class="container-fluid"><DIV class="idkmakeityellow"><DIV class="header-image" style="background-image: url(https://static.igem.org/mediawiki/2016/4/42/T--Waterloo--plates_and_hockey_sticks.jpg);"><H3 class="header-text"><SPAN style="background-color: #FFFFFF;opacity: 0.8;">Description</SPAN></H3></DIV></DIV></DIV><DIV class="wcontent-box"><DIV class="wcontent-title">
      Motivation
    </DIV><DIV class="wcontent"><P>
        Human neurodegenerative diseases (NDDs) like Creutzfeldt-Jakob, Alzheimer’s, and Parkinson’s disease are associated with multiple metabolic complications in neurons. To name a known few, the metabolism of cholesterol, copper, iron, heme, NAD+, and some
        neurotransmitters have all been shown to be affected by prion proteins affiliated with these NDDs<SUP>1,2,3,4,5</SUP>. It is important to continue studying the metabolic implications of NDDs to further elucidate their pathological mechanisms. For diseases
        like Alzheimer’s, years of work have been put in, yet treatment and therapy are only beginning to progress more rapidly. We hope to contribute to this field by providing a synthetic biology approach that enables an alternative method for metabolic
        studies in neurons affected by prion proteins.
      </P><P>
        This year, we’ve designed a negative feedback loop using a novel regulatory element capable of overexpressing or repressing target proteins upon transitioning from a neutral state to a prion state. As a proof-of-concept, we use the Sup35 prion protein
        associated with the [<I>psi</I><SUP>-</SUP>] (healthy) / [<I>PSI</I><SUP>+</SUP>] (disease-like) state of <I>Saccharomyces cerevisiae</I> cells. We use read-through of a premature stop codon in fusions with Hsp104 or dCas9 to respectively overexpress or repress Hsp104. Both overexpression
        and repression of Hsp104, through different mechanisms outlined <A href="https://2016.igem.org/Team:Waterloo/InDepthDesign/HowItWorks">here</A>, cure the [<I>PSI</I><SUP>+</SUP>] state – from priON to OFF.
      </P><P>
        This system’s design consists of a three-step process: <I>trigger, signal, and control. </I></P></DIV></DIV><DIV class="wcontent-box"><DIV class="wcontent-title">
      Trigger
    </DIV><DIV class="wcontent"><P>
        Sup35’s translational terminator activity, which allows it to remove ribosomes from mRNAs when they’ve reached a stop codon, is reduced during the [<I>PSI</I><SUP>+</SUP>] state which we induce via introduction of an extra Sup35 gene fused to a green fluorescent protein (GFP)
        on a plasmid. Our specially designed fusion of cyan fluorescent protein (CFP) and a Sup35 disaggregase (Hsp104) has a premature stop codon positioned upstream of CFP’s fluorophore. The loss in translation terminator activity upon transitioning
        to [<I>PSI</I><SUP>+</SUP>] increases the likelihood of ribosomes translating the entire CFP::Hsp104 fusion protein due to read-through of the premature stop codon. This essentially triggers the increased production of CFP::Hsp104 in response to developing [<I>PSI</I><SUP>+</SUP>]state.
      </P><P>
        By inputting dCas9 in lieu of the Hsp104 ORF in the plasmid, we can use mathematical modeling to predict the ability of optimized sgRNAs to repress the expression of the native Hsp104 gene in the <I>S. cerevisiae</I> genome. This is based on the concept that
        overexpression of Hsp104 beyond an upper threshold cures the [<I>PSI</I><SUP>+</SUP>] state, and repression below a lower threshold prevents propagation of the Sup35 aggregates to daughter cells, thus curing future generations.
      </P></DIV></DIV><DIV class="wcontent-box"><DIV class="wcontent-title">
      Signal
    </DIV><DIV class="wcontent"><P>
        While the [<I>PSI</I><SUP>+</SUP>] state develops, the cell’s GFP signal increases due to Sup35::GFP overexpression. In response, the CFP signal also increases as it is triggered by the loss of translational terminator activity due to aggregated Sup35. With fluorimetry,
        we monitor the changes in the GFP and CFP signal shown in our Results Section. We also observe the changes between [<I>psi</I><SUP>-</SUP>] and [<I>PSI</I><SUP>+</SUP>] through other phenotypic outputs: the colour of the colonies, and the cellular granularity determined by flow
        cytometry.
      </P></DIV></DIV><DIV class="wcontent-box"><DIV class="wcontent-title">
      Control
    </DIV><DIV class="wcontent"><P>
        Within a cell in the [<I>PSI</I><SUP>+</SUP>] state the CFP::Hsp104 concentration increases beyond an upper threshold where it effectively disaggregates the Sup35 prions and aids in refolding them into their healthy functional form. This enables the premature stop codon in the CFP::Hsp104 to prevent full translation of the transcript into its functional form. Ultimately, triggering of the premature stop codon read-through can control the expression of a target protein and, in this case, allows for the [<I>PSI</I><SUP>+</SUP>] state to return to <I>psi</I><SUP>-</SUP> by Hsp104-mediated curing.
      </P></DIV></DIV><DIV class="wcontent-box"><DIV class="wcontent-title">
      Special Project: iGEM 2015 Networks Analysis
    </DIV><DIV class="wcontent"><P>
        It is with great enthusiasm that we present a study of the sociological network of iGEM team collaborations. By educating our community about the attributes that inherently contribute to the likelihood of Jamboree success, we will empower teams to increase competitiveness and improve the overall quality of their science.
      </P></DIV></DIV><DIV class="wcontent-box"><DIV class="wcontent-title">
      BioBrick Improvement: Documentation and characterization of <A href="http://parts.igem.org/Part:BBa_K1645998">BBa_K1645998</A> and <A href="http://parts.igem.org/Part:BBa_K1645999">BBa_K1645999</A></DIV><DIV class="wcontent"><P>
        Part of the <A href="https://2015.igem.org/Team:Waterloo/Parts">Waterloo iGEM 2015</A> project's work involved modifying an sgRNA such that its spacer was flanked by restriction sites to enable exchange of new spacers using basic molecular biology techniques. Adding these restriction sites required modification of the sgRNA scaffold, thus raising concern that the additions would disrupt the scaffold secondary structure and affect dCas9's performance. We created a BioBrick of an sgRNA, with no modifications, designed to target the lacI promoter of an RFP gene (<A href="http://parts.igem.org/Part:BBa_R0010">BBa_R0020</A>). This year, we brought this experiment back to complete its documentation and characterization with better flow cytometry experiments that produced more robust data. We successfully demonstrated that this BioBrick is able to repress RFP expression when compared to controls. We also demonstrated that the modified sgRNA allows dCas9 to repress RFP expression with very similar performance. 
      </P><P>
        Additionally, we used a reporter plasmid that contained a GFP gene cassette (<A href="http://parts.igem.org/Part:BBa_I20260">BBa_I20260</A>) and RFP gene casette (<A href="http://parts.igem.org/Part:BBa_J04450">BBa_204450</A>) for last year's flow cytometry experiments. These two gene cassettes were cloned together into pSB1C3, but the characterization of this BioBrick was never published last year. This year, we've successfully demonstrated that this BioBrick works by showing flow cytometry data indicative of GFP and RFP expression. 
      </P></DIV></DIV><DIV class="wcontent-box"><DIV class="wcontent-title">
      References
    </DIV><DIV class="wcontent"><P>
        [1] Cingaram PKR, Nyeste A, Dondapati DT, Fodor E, Welker E (2015) Prion Protein Does Not Confer Resistance to Hippocampus-Derived Zpl Cells against the Toxic Effects of Cu2+, Mn2+, Zn2+ and Co2+ Not Supporting a General Protective Role for PrP in Transition Metal Induced Toxicity. PLoS ONE 10(10): e0139219. <A href="http://doi:10.1371/journal.pone.0139219">http://doi:10.1371/journal.pone.0139219</A></P><P> 
        [2] Cui, H. L., Guo, B., Scicluna, B., Coleman, B. M., Lawson, V. A., Ellett, L., … Hill, A. F. (2014). Prion infection impairs cholesterol metabolism in neuronal cells. Journal of Biological Chemistry, 289(2), 789–802. <A href="http://doi.org/10.1074/jbc.M113.535807">http://doi.org/10.1074/jbc.M113.535807</A></P><P> 
        [3] Singh, N. (2014). The Role of Iron in Prion Disease and Other Neurodegenerative Diseases. PLoS Pathogens, 10(9), e1004335. <A href="http://doi.org/10.1371/journal.ppat.1004335">http://doi.org/10.1371/journal.ppat.1004335</A></P><P> 
        [4] Walsh, D. M., &amp; Selkoe, D. J. (2016). A critical appraisal of the pathogenic protein spread hypothesis of neurodegeneration. Nature Reviews. Neuroscience, 17(4), 251–60. <A href="http://doi.org/10.1038/nrn.2016.13">http://doi.org/10.1038/nrn.2016.13</A></P><P> 
        [5] Zhou, M., Ottenberg, G., Sferrazza, G. F., Hubbs, C., Fallahi, M., Rumbaugh, G., … Lasmezas, C. (2015). Neuronal death induced by misfolded prion protein is due to NAD+ depletion and can be relieved in vitro and in vivo by NAD+ replenishment. Brain : A Journal of Neurology, 138(Pt 4), 992–1008. <A href="http://doi.org/10.1093/brain/awv002">http://doi.org/10.1093/brain/awv002</A></P></DIV></DIV><P class="footer-links" style="font-size:0.8em;text-align:center!important;"><A style="padding-right:0;font-size:1em;" href="http://igem.uwaterloo.ca">External Site</A>© 2016 Waterloo iGEM
    </P></DIV></DIV></DIV></DIV></DIV></BODY></HTML>