<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_Waterloo_InDepthDesign_HowItWorks skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><DIV id="top_title"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:Waterloo/InDepthDesign/HowItWorks</SPAN></H1></DIV><DIV id="HQ_page"><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><DIV class="cd-auto-hide-header" ng-mouseleave="hider()" ng-controller="nav" ng-cloak=""><NAV class="cd-primary-nav"><A href="#cd-navigation" class="nav-trigger"><SPAN>
        Menu
      </SPAN></A><UL id="cd-navigation" class="mobile-menu"><LI onclick="$(this).toggleClass('show-secondary-nav');"><A href="#" ng-click="clickety=true" ng-mouseover="subnav='project'; ">PROJECT</A></LI><LI><A class="paragraph-medium" href="/Team:Waterloo/Description">DESCRIPTION</A></LI><LI><A class="paragraph-medium" href="/Team:Waterloo/Design">DESIGN</A></LI><LI><A class="paragraph-medium" href="/Team:Waterloo/Requirements">REQUIREMENTS</A></LI><LI><A class="paragraph-medium" href="/Team:Waterloo/Model">MODELLING</A></LI></UL><LI onclick="$(this).toggleClass('show-secondary-nav');"><A href="#" ng-click="clickety=true" ng-mouseover="subnav='design';">BLUEPRINT</A></LI><LI><A class="paragraph-medium" href="/Team:Waterloo/InDepthDesign/HowItWorks">HOW IT WORKS</A></LI><LI><A class="paragraph-medium" href="/Team:Waterloo/Software">PREMATURE STOP CODON</A></LI><LI><A class="paragraph-medium" href="/Team:Waterloo/InDepthDesign/PromoterSelection">PROMOTER SELECTION</A></LI><LI><A class="paragraph-medium" href="/Team:Waterloo/InDepthDesign/PlasmidLossModel">PLASMID LOSS</A></LI><LI><A class="paragraph-medium" href="/Team:Waterloo/NetworkModelling">NETWORK MODELLING</A></LI><LI><A class="paragraph-medium" href="/Team:Waterloo/Aggregation">AGGREGATION MODEL</A></LI><LI><A class="paragraph-medium" href="/Team:Waterloo/InDepthDesign/Crispr">CRISPR</A></LI><LI onclick="$(this).toggleClass('show-secondary-nav');"><A href="#" ng-click="clickety=true" ng-mouseover="subnav='results';">RESULTS</A></LI><LI><A class="paragraph-medium" href="/Team:Waterloo/Proof">PROOF OF CONCEPT</A></LI><LI><A class="paragraph-medium" href="/Team:Waterloo/InDepthDesign/PlasmidLossModel">PLASMID LOSS</A></LI><LI><A class="paragraph-medium" href="/Team:Waterloo/Results/PromoterAnalysis">PROMOTER ANALYSIS</A></LI><LI onclick="$(this).toggleClass('show-secondary-nav');"><A href="#" ng-click="clickety=true" ng-mouseover="subnav='lab'">LAB</A></LI><LI><A class="paragraph-medium" href="/Team:Waterloo/Parts">PARTS</A></LI><LI><A class="paragraph-medium" href="/Team:Waterloo/Experiments">PROTOCOLS</A></LI><LI><A class="paragraph-medium" href="/Team:Waterloo/Notebook">NOTEBOOK</A></LI><LI><A class="paragraph-medium" href="/Team:Waterloo/Safety">SAFETY</A></LI><LI onclick="$(this).toggleClass('show-secondary-nav');"><A href="#" ng-click="clickety=true" ng-mouseover="subnav='practices'">POLICY</A></LI><LI><A class="paragraph-medium" href="/Team:Waterloo/HP/Silver">SILVER</A></LI><LI><A class="paragraph-medium" href="/Team:Waterloo/HP/Gold">GOLD</A></LI><LI><A class="paragraph-medium" style="font-weight: bold; color: #cd5f94" href="/Team:Waterloo/Engagement">ENGAGEMENT</A></LI><LI><A class="paragraph-medium" style="color: #cd5f94" href="/Team:Waterloo/Engagement/GenderEquity">GENDER EQUITY</A></LI><LI><A class="paragraph-medium" style="color: #cd5f94" href="/Team:Waterloo/Engagement/Ethics">ETHICS</A></LI><LI><A class="paragraph-medium" style="color: #cd5f94" href="/Team:Waterloo/Engagement/Access">ACCESS</A></LI><LI><A class="paragraph-medium" style="font-weight:bold; color: #5f94cd;" href="/Team:Waterloo/Integrated_Practices">INTEGRATED PRACTICES</A></LI><LI><A class="paragraph-medium" style="color: #5f94cd;" href="/Team:Waterloo/Integrated_Practices/OpenScience">OPEN SCIENCE</A></LI><LI><A class="paragraph-medium" style="color: #5f94cd;" href="/Team:Waterloo/Integrated_Practices/Networks">NETWORKS</A></LI><LI onclick="$(this).toggleClass('show-secondary-nav');"><A href="#" ng-click="clickety=true" ng-mouseover="subnav='team';">TEAM</A></LI><LI><A class="paragraph-medium" href="/Team:Waterloo/Team">THE TEAM</A></LI><LI><A class="paragraph-medium" href="/Team:Waterloo/Attributions">ATTRIBUTIONS</A></LI><LI><A class="paragraph-medium" href="/Team:Waterloo/Sponsors">SPONSORS</A></LI><LI><A class="paragraph-medium" href="/Team:Waterloo/Collaborations">COLLABORATION</A></LI></NAV><NAV class="cd-secondary-nav" id="projecttab" ng-show="subnav=='project'"><UL><LI><A class="paragraph-medium" href="/Team:Waterloo/Description">DESCRIPTION</A></LI><LI><A class="paragraph-medium" href="/Team:Waterloo/Design">DESIGN</A></LI><LI><A class="paragraph-medium" href="/Team:Waterloo/Requirements">REQUIREMENTS</A></LI><LI><A class="paragraph-medium" href="/Team:Waterloo/Model">MODELLING</A></LI></UL></NAV><NAV class="cd-secondary-nav" id="designtab" ng-show="subnav == 'design'"><UL><LI><A class="paragraph-medium" href="/Team:Waterloo/InDepthDesign/HowItWorks">HOW IT WORKS</A></LI><LI><A class="paragraph-medium" href="/Team:Waterloo/Software">PREMATURE STOP CODON</A></LI><LI><A class="paragraph-medium" href="/Team:Waterloo/InDepthDesign/PromoterSelection">PROMOTER SELECTION</A></LI><LI><A class="paragraph-medium" href="/Team:Waterloo/InDepthDesign/PlasmidLossModel">PLASMID LOSS MODEL</A></LI><LI><A class="paragraph-medium" href="/Team:Waterloo/Aggregation">AGGREGATION MODEL</A></LI><LI><A class="paragraph-medium" href="/Team:Waterloo/InDepthDesign/Crispr">CRISPR</A></LI><LI><A class="paragraph-medium" href="/Team:Waterloo/NetworkModelling">NETWORK MODELLING</A></LI></UL></NAV><NAV class="cd-secondary-nav" id="resultstab" ng-show="subnav == 'results'"><UL><LI><A class="paragraph-medium" href="https://2016.igem.org/Team:Waterloo/Proof">PROOF OF CONCEPT</A></LI><LI><A class="paragraph-medium" href="/Team:Waterloo/Results/PlasmidLoss">PLASMID LOSS</A></LI><LI><A class="paragraph-medium" href="/Team:Waterloo/Results/PromoterAnalysis">PROMOTER ANALYSIS</A></LI></UL></NAV><NAV class="cd-secondary-nav" id="labtab" ng-show="subnav == 'lab'"><UL><LI><A class="paragraph-medium" href="/Team:Waterloo/Parts">PARTS</A></LI><LI><A class="paragraph-medium" href="/Team:Waterloo/Experiments">PROTOCOLS</A></LI><LI><A class="paragraph-medium" href="/Team:Waterloo/Notebook">NOTEBOOK</A></LI><LI><A class="paragraph-medium" href="/Team:Waterloo/Safety">SAFETY</A></LI></UL></NAV><NAV class="cd-secondary-nav" id="practices" ng-show="subnav == 'practices'"><UL><LI><A class="paragraph-medium" href="/Team:Waterloo/HP/Silver">SILVER</A></LI><LI><A class="paragraph-medium" href="/Team:Waterloo/HP/Gold">GOLD</A></LI><LI><A class="paragraph-medium" style="font-weight: bold; color: #cd5f94" href="/Team:Waterloo/Engagement">ENGAGEMENT</A></LI><LI><A class="paragraph-medium" style="color: #cd5f94" href="/Team:Waterloo/Engagement/GenderEquity">GENDER EQUITY</A></LI><LI><A class="paragraph-medium" style="color: #cd5f94" href="/Team:Waterloo/Engagement/Ethics">ETHICS</A></LI><LI><A class="paragraph-medium" style="color: #cd5f94" href="/Team:Waterloo/Engagement/Access">ACCESS</A></LI><LI><A class="paragraph-medium" style="font-weight:bold; color: #5f94cd;" href="/Team:Waterloo/Integrated_Practices">INTEGRATED PRACTICES</A></LI><LI><A class="paragraph-medium" style="color: #5f94cd;" href="/Team:Waterloo/Integrated_Practices/OpenScience">OPEN SCIENCE</A></LI><LI><A class="paragraph-medium" style="color: #5f94cd;" href="/Team:Waterloo/Integrated_Practices/Networks">NETWORKS</A></LI></UL></NAV><NAV class="cd-secondary-nav" id="teamtab" ng-show="subnav == 'team'"><UL><LI><A class="paragraph-medium" href="/Team:Waterloo/Team">THE TEAM</A></LI><LI><A class="paragraph-medium" href="/Team:Waterloo/Attributions">ATTRIBUTIONS</A></LI><LI><A class="paragraph-medium" href="/Team:Waterloo/Sponsors">SPONSORS</A></LI><LI><A class="paragraph-medium" href="/Team:Waterloo/Collaborations">COLLABORATION</A></LI></UL></NAV></DIV><NAV class="cd-vertical-nav"><UL style="list-style: none"><LI><A href="#psiyeast" class="active"><SPAN class="label">PSI<SUP>+</SUP></SPAN></A></LI><LI><A href="#strength"><SPAN class="label">Strength</SPAN></A></LI><LI><A href="#role"><SPAN class="label">Role</SPAN></A></LI><LI><A href="#cure"><SPAN class="label">Curing</SPAN></A></LI><LI><A href="#ref"><SPAN class="label">References</SPAN></A></LI></UL></NAV><DIV class="container-fluid"><DIV class="bloodorangeisacolour"><DIV class="header-image" style="background-image: url(https://static.igem.org/mediawiki/2016/f/f7/T--Waterloo--lab_microcentrifuge.jpg);"><H3 class="header-text"><SPAN style="background-color: #FFFFFF;opacity: 0.8;">How It Works</SPAN></H3></DIV></DIV></DIV><DIV class="wcontent-box"><DIV class="wcontent-title">
        The [<I>PSI<SUP>+</SUP></I>] Element in <I>Saccharomyces cerevisiae </I></DIV><DIV class="wcontent" id="psiyeast"><P>
        Sup35 is a protein in yeast that has a role in translation termination <SUP>[1]</SUP>. Specifically, it removes the ribosome from mRNA when it reaches a stop codon. This means that without Sup35, translation can continue after a stop codon in the mRNA sequence. In nature, Sup35 is typically found in a soluble form, but it can also take an irregular conformation leading to it aggregating and forming non-soluble fibers <SUP>[2]</SUP>, which form at higher rates when Sup35 is being overexpressed in the cell <SUP>[2,3]</SUP>.
        </P><P class="wcontent-caption"><I> Figure 1. </I> Sup35 mechanism. Sup35 will recognize the ribosome when it reaches the stop codon and the mRNA and helps physically remove the ribosome to release the primary amino acid sequence. 
</P><P class="wcontent-caption"><I> Figure 2. </I> Misfolded Sup35. Misfolded Sup35 will not be able to recognize the stop codon and the ribosome will read-through and the resulting Sup35 will be more prone to misfold. 
</P><P>
        The non-soluble fibers can be passed from mother to daughter cells, and misfolded Sup35 can cause soluble Sup35 to misfold and join the aggregate <SUP>[2]</SUP>. Thus, aggregates of misfolded Sup35 are deemed prions, which are non-Mendelian heritable elements <SUP>[2,3]</SUP>. A yeast cell containing aggregated Sup35 is said to have the [<I>PSI<SUP>+</SUP></I>] element <SUP>[3]</SUP>.
        </P><P>
        The ability of Sup35 to terminate translation is hindered, though not completely eliminated, when it aggregates <SUP>[2]</SUP>. This causes read-through of normal stop codons, but also nonsense suppression <SUP>[1,4]</SUP>. Nonsense suppression is the read through of premature stop codons, which can be created by mutations to a standard DNA sequence.
        </P></DIV></DIV><DIV class="wcontent-box"><DIV class="wcontent-title" id="strength">
        Weak and Strong [<I>PSI<SUP>+</SUP></I>] Variants
      </DIV><DIV class="wcontent"><P>
        There are different types of Sup35 aggregates, and the [<I>PSI<SUP>+</SUP></I>] state can also be defined as weak or strong depending on which type is present. The weak and strong states can both be induced in genetically identical strains, demonstrating that the difference in the states is not due to different alleles of the Sup35 gene, but the structure of the aggregates <SUP>[2]</SUP>. Yeast cells that have the weak [<I>PSI<SUP>+</SUP></I>] element contain a small number of structurally large prions, whereas yeast cells that have the strong [<I>PSI<SUP>+</SUP></I>] element contain a large number of small prions <SUP>[2]</SUP>. The result of this is that a greater fraction of Sup35 is aggregated in the strong variant. Interestingly, conversion between strong and weak [<I>PSI<SUP>+</SUP></I>] is rare <SUP>[2,3]</SUP>, suggesting a fundamental difference in the structure of the aggregates in strong and weak [<I>PSI<SUP>+</SUP></I>] strains.
        </P><P>
        A portion of Sup35 at the protein’s N-terminus is essential for prion formation <SUP>[3]</SUP>. This domain is called the prion domain, and is found at the core of any aggregate <SUP>[2]</SUP>. There is evidence suggesting this region is the only segment of Sup35 that changes its structure when Sup35 takes on its prion-forming conformation <SUP>[2]</SUP>. There are several theories as to how exactly the prion domains of Sup35 molecules interact to form aggregates, but the accepted structure suggests that prion domain interactions are stronger in the weak [<I>PSI<SUP>+</SUP></I>] variant <SUP>[2]</SUP>.
        </P><P class="wcontent-caption"><I> Figure 3. </I></P></DIV></DIV><DIV class="wcontent-box"><DIV class="wcontent-title" id="role">
        Our Role 
      </DIV><DIV class="wcontent"><P>
        We will take advantage of the Sup35 function by understanding how read through rates are affected when the yeast cell goes into a [<I>PSI<SUP>+</SUP></I>] state. Read through rates are higher in a [<I>PSI<SUP>+</SUP></I>] state because Sup35 has reduced functionality and often fails to remove the ribosome from mRNA.
        </P><P>
        We are interested in observing protein expression over time as a [<I>psi<SUP>-</SUP></I>] yeast cell becomes [<I>PSI<SUP>+</SUP></I>]. How we did this is by inserting a premature stop codon into a cyan fluorescent protein (CFP) N-terminal tag. Therefore when Sup35 is functioning normally only a small part of CFP will be translated and be immediately degraded by the proteasome. However, when Sup35 aggregates we will observe some degree of read-through and get expression of a protein of interest with an N-terminal CFP tag that can be observed by fluorimetry. 
        </P><P class="wcontent-caption"><I> Figure 4. </I> By inserting a premature stop codon into CFP, we will only get expression of CFP during a [<I>PSI<SUP>+</SUP></I>] state since Sup35 will not be completely functional and there will be read-through of stop codons. 
        </P></DIV></DIV><DIV class="wcontent-box"><DIV class="wcontent-title" id="cure">
        Curing [<I>PSI<SUP>+</SUP></I>] by Altering Hsp104 Expression
      </DIV><DIV class="wcontent"><P>
        Hsp104 is a heat shock protein that aids in the disaggregation of proteins in S. cerevisiae <SUP>[3]</SUP>. It is required for cells to maintain the [<I>PSI<SUP>+</SUP></I>] state; in the absence of Hsp104, the [<I>PSI<SUP>+</SUP></I>] element is lost within a few generations <SUP>[3]</SUP>. Interestingly, an abnormally large amount of Hsp104 in a yeast cell also eliminates the [<I>PSI<SUP>+</SUP></I>] element <SUP>[3]</SUP>. The exact effect of altering Hsp104 expression is different on weak and strong [<I>PSI<SUP>+</SUP></I>] variants, but generally the absence or excess of Hsp104 will cure the [<I>PSI<SUP>+</SUP></I>] state regardless of its strength <SUP>[2]</SUP>.
        </P><P>
        The leading theory explaining these effects is that Hsp104 has the capacity to shear apart aggregates of Sup35 by an unknown mechanism [2,3,5]. The theory begins by stating that the formation of new prions is reliant upon old prions breaking apart <SUP>[5]</SUP>. Without any Hsp104 in a yeast cell, prions cannot break up. Therefore no new prions can form and the [<I>PSI<SUP>+</SUP></I>] element disappears. Conversely, the presence of excess Hsp104 shears apart the prions to the point where no aggregates remain in the cell; the Sup35 then returns to its normal conformation and resumes its translation termination function [2,5].
        </P><P>
        To get rid of the strong [<I>PSI<SUP>+</SUP></I>] variant, we will insert an extra copy of Hsp104 with an N-terminal CFP tag into a yeast cell in a plasmid so the Hsp104 will only be expressed during a prion response. Once this overexpression cures the response, the system will turn off and extra Hsp104 will not be expressed anymore. Our system acts as a negative feedback loop since Hsp104 is regulated by the [<I>PSI<SUP>+</SUP></I>] state in the cell. This system was tested in the lab (see the <A href="https://2016.igem.org/Team:Waterloo/Proof">Proof of Concept</A> page). 
        </P><P>To get rid of the weak [<I>PSI<SUP>+</SUP></I>] variant, we will insert a copy of dCas9 with an N-terminal CFP tag into a yeast cell along with sgRNA targets so that the CRISPR system targets the promoter of Hsp104 in the yeast genome. During a prion response, dCas9 will be expressed and effectively knock-down Hsp104, causing disappearance of the weak [<I>PSI<SUP>+</SUP></I>] variant (see the <A href="https://2016.igem.org/Team:Waterloo/Proof">Proof of Concept</A> page). 
        </P></DIV></DIV><DIV class="wcontent-box"><DIV class="wcontent-title" id="cure">
        Alternative Prion Systems
      </DIV><DIV class="wcontent"><P><B>Introduction </B></P><P>
         During the course of our project, we examined prion states other than [<I>PSI<SUP>+</SUP></I>] which have the potential use in a system where the prion state controls the production of a protein of interest.  Many other yeast proteins have been shown to exist in both a soluble state as well as an infectious prion state.<SUP>[2]</SUP>  To evaluate prion systems that could be used in a project similar to ours, we examined proteins that both play a direct role in transcription or translation and can exist as a prion form.<SUP>[2]</SUP>  These include the prions/proteins: [<I>SWI<SUP>+</SUP></I>]/Swi1, [<I>OCT<SUP>+</SUP></I>]/Cyc8, [<I>MOT3</I>]/Mot3, and [<I>ISP<SUP>+</SUP></I>]/Sfp1.<SUP>[2]</SUP>  All of these systems have been studied in detail, and affect some aspect of transcriptional regulation.<SUP>[2]</SUP>  The prion/proteins [<I>RNQ<SUP>+</SUP></I>]/Rnq1 and [<I>URE3</I>]/Ure2, though they are both often used as models for prion systesm were not examined in detail as the native function of the proteins are not directly involved in protein production.<SUP>[2]</SUP>  From the literature it is clear that the prion systems of [<I>SWI<SUP>+</SUP></I>], [<I>OCT<SUP>+</SUP></I>], and [<I>MOT3</I>] warrant further investigation and that the [<I>ISP<SUP>+</SUP></I>] prion state is a poor option for a prion control system.<SUP>[6][7][8][9][10]</SUP></P><P><B>[<I>SWI<SUP>+</SUP></I>] Prion State of Swi1</B></P><P>
        Swi is a chromatin-remodeling complex that plays a role in transcriptional regulation.  It has already been used in a reporter construct called pLS7, in which a protein (in this case lacZ) is only produced when Swi is in its functional and native conformation and able to act as a transcriptional activator.<SUP>[7][8]</SUP>  A transcript is not produced, however, when Swi is in its prion form [<I>SWI<SUP>+</SUP></I>].<SUP>[2]</SUP>   The experiments that have been done with the [<I>SWI<SUP>+</SUP></I>] factor have focused on stopping the expression of a transcript upon prion formation.  This prion system has the opposite effect of the [<I>PSI<SUP>+</SUP></I>] system.  However, using the [<I>SWI<SUP>+</SUP></I>] system to regulate protein production has an added level of complexity.<SUP>[8]</SUP>  This is that Swi does not bind DNA with sequence specificity; its effect is mediated by interactions with other transcriptional regulators.<SUP>[2]</SUP></P><P>
Since the inactivation of Swi upon transition to the [<I>SWI<SUP>+</SUP></I>] state has already been used to decrease production of a protein of interest, this would be a very good system to use to make a system similar to our [<I>PSI<SUP>+</SUP></I>] system.
        </P><P><B>[<I>OCT<SUP>+</SUP></I>] Prion State of Cyc8</B></P><P>
        Cyc8 is a global transcriptional co-repressor protein which can propagate as the prion state [<I>OCT<SUP>+</SUP></I>].<SUP>[9]</SUP>  The transition to the [<I>OCT<SUP>+</SUP></I>] state inactivates Cyc8.<SUP>[9]</SUP>  One of the most important results of this is an increase in iso-2 cytochrome c levels within the yeast cell by the inactivation of the repressor complex Cyc8-Tup1.<SUP>[9]</SUP>   An increase in iso-2 cytochrome c due to repressor inactivation allows a yeast with a non-functioning isoform 1 of cytochrome c to grow on lactate.<SUP>[9]</SUP>  Without the inactivation of repressor protein Cyc8, isoform 2 of cytochrome c would not be produced in high enough levels to compensate for the loss of isoform 1 and the cell would not grow on lactate.<SUP>[9]</SUP>  Cyc8 also functions as a repressor for many genes, and its aggregation will increase the transcription of those genes.<SUP>[9]</SUP> This has been shown to cause defects in sporulation and mating, and results in slow growth.<SUP>[9]</SUP></P><P>
[<I>OCT<SUP>+</SUP></I>]/Cyc8 would be a good prion system to investigate further, as the native function of soluble Cyc8 is to act as a repressor protein and this function is greatly reduced upon prion formation.<SUP>[9]</SUP>  Using the [<I>OCT<SUP>+</SUP></I>]/Cyc8 system, however, would require an investigation into secondary effects on the cell, as Cyc8 is used as a repressor protein for much of the yeast genome.<SUP>[9]</SUP></P><P><B>[<I>MOT3</I>] Prion State of Mot3</B></P><P>
 Mot3 is a transcription factor that represses anaerobic genes.<SUP>[6]</SUP>  The aggregation of Mot3 inactivates the protein, making it unable to repress transcription of its target genes.<SUP>[6]</SUP>  One of the genes that Mot3 controls is [<I>DAN1</I>].<SUP>[6]</SUP>  A construct has been made in which the open reading frame of the [<I>DAN1</I>] gene was been replaced with the open reading frame of the [<I>URA3</I>] sequence (which is necessary to synthesize uracil).  The resulting construct was transformed into yeast with a non-functional genomic copy of [<I>URA3</I>].<SUP>[6]</SUP>  Only cells with Mot3 deactivated through mutation or prion aggregation were able to grow on media lacking uracil.<SUP>[6]</SUP>  It would be possible to again switch the [<I>DAN1</I>] open reading frame for another sequence to study the effects of increased transcription of a gene of interest on the [<I>MOT3</I>] prion state.       
          </P><P>
Since [<I>MOT3</I>]/Mot3 has already been used in a system similar to ours, in which protein expression is dependent upon prion state, so a system similar to ours can hopefully be made.<SUP>[6]</SUP></P><P><B>[<I>ISP<SUP>+</SUP></I>] Prion State of Sfp1</B></P><P>
Sfp1 positively controls the transcription of about 10% of the yeast's structural genes.<SUP>[10]</SUP>  Sfp1 can aggregate to the [<I>ISP<SUP>+</SUP></I>] prion state.  While in this state, its ability to control transcription is reduced though not eliminated.<SUP>[10]</SUP>  It contains zinc finger domains at the C terminus that remain functional while in the prion state.<SUP>[10]</SUP>  The effects of Sfp1 prion aggregation do not have the same effects as Sfp1 deletion.<SUP>[10]</SUP> For the other prion systems examined above, these effects are the same.<SUP>[10]</SUP></P><P>
Because of its incomplete inactivation upon prion aggregation, the [<I>ISP<SUP>+</SUP></I>] state is likely a poor candidate to create a system to study the effects of increased or decreased protein production upon prion aggregation.
          </P><P><B>Next Steps</B></P><P>
The prion states of [<I>SWI<SUP>+</SUP></I>], [<I>OCT<SUP>+</SUP></I>], and [<I>MOT3</I>] are good candidates to create a prion state dependent protein production system to complement our [<I>PSI<SUP>+</SUP></I>] dependent system.  This is because when the native function of the proteins associated with these states is inactivated by prion formation, the result is a direct and predictable change in protein production.<SUP>[6][7][9]</SUP>  Biobricking, or otherwise making these systems more accessible to researchers without a synthetic biology background will allow more complex experiments to be performed using yeast as a model organism for prion diseases.  [<I>ISP<SUP>+</SUP></I>]/Sfp1 is not a good system to use for a prion state dependent protein production system as the native function of Sfp1 is only partially inactivated upon prion formation.<SUP>[10]</SUP></P></DIV></DIV><DIV class="wcontent-box"><DIV class="wcontent-title">
      References
    </DIV><DIV class="wcontent" id="ref"><P>
        [1] Department of Genetics. (2014). Saccharomyces Genome Database.
        </P><P> 
        [2] Liebman, S. W., &amp; Chernoff, Y. O. (2012). Prions in yeast. Genetics. <A href="http://doi.org/10.1534/genetics.111.137760">http://doi.org/10.1534/genetics.111.137760</A></P><P> 
        [3] Chernoff, Y. O., Lindquist, S. L., Ono, B., Inge-Vechtomov, S. G., &amp; Liebman, S. W. (1995). Role of the chaperone protein Hsp104 in propagation of the yeast prion-like factor [psi+]. Science, 268(5212), 880-884. Retrieved from <A href="http://science.sciencemag.org/content/268/5212/880.abstract">http://science.sciencemag.org/content/268/5212/880.abstract</A></P><P> 
        [4] Tanaka, M., Chien, P., Naber, N., Cooke, R., &amp; Weissman, J. S. (2004). Conformational variations in an infectious protein determine prion strain differences. Nature, 428(March), 323–8. <A href="http://doi.org/10.1038/nature02392">http://doi.org/10.1038/nature02392</A></P><P> 
        [5] Park, Y. N., Zhao, X., Yim, Y. I., Todor, H., Ellerbrock, R., Reidy, M., ... &amp; Greene, L. E. (2014). Hsp104 overexpression cures Saccharomyces cerevisiae [PSI+] by causing dissolution of the prion seeds. Eukaryotic cell,13(5), 635-647. <A href="http://ec.asm.org/content/13/5/635.short">http://ec.asm.org/content/13/5/635.short</A></P><P> 
        [6] Alberti, S., Halfmann, R., King, O., Kapila, A., &amp; Lindquist, S. (2009). A Systematic Survey Identifies Prions and Illuminates Sequence Features of Prionogenic Proteins. Cell, 137, 146-158.
      </P><P> 
        [7] Du, Z., Park, K.-W., Yu, H., Fan, Q., &amp; Li, L. (2008). Newly identified prion linked to the chromatin-remodeling factor Swi in Saccharomyces cerevisiae. Nat Genet. 40(4), 460-465.
      </P><P> 
      [8] Du, Z., Crow, E. T., Kang, H. S., &amp; Li, L. (2010). Distinct Subregions of Swi1 Manifest Striking Differences in Prion transmission and SWI/SNF Function. Molecular and Cellular Biology. 30(19), 4644-4655.
      </P><P>
      [9] Patel, B. K., Gavin-Smyth, J., &amp; Liebman, S. W. (2009). The yeast global transcriptional co-repressor protein Cyc8 can propagate as a prion. Nature Cell Biology, 11(3), 344-349
      </P><P>
      [10]Rogoza, T., Goginashvili, A., Rodinova, S., Ivanov, M., Viktorovskaya, O., Rubel, A., Volkov, K., &amp; Mironova, L. (2010). Non-Mendelian determinant [ISP+] in yeast is a nuclear-residing form of the global transcriptional regulator Sfp1. PNAS, 107(23), 10573-10577.
     </P></DIV></DIV><P class="footer-links" style="font-size:0.8em;text-align:center!important;"><A style="padding-right:0;font-size:1em;" href="http://igem.uwaterloo.ca">External Site</A>© 2016 Waterloo iGEM
    </P></DIV></DIV></DIV></DIV></DIV></BODY></HTML>