Team:Waterloo/Parts
Menu
PROJECT
DESCRIPTION
DESIGN
REQUIREMENTS
MODELLING
BLUEPRINT
HOW IT WORKS
PREMATURE STOP CODON
PROMOTER SELECTION
PLASMID LOSS MODEL
NETWORK MODELLING
AGGREGATION MODEL
CRISPR
RESULTS
PROOF OF CONCEPT
PLASMID LOSS
PROMOTER ANALYSIS
LAB
PARTS
PROTOCOLS
NOTEBOOK
SAFETY
POLICY
SILVER
GOLD
ENGAGEMENT
GENDER EQUITY
ETHICS
ACCESS
INTEGRATED PRACTICES
OPEN SCIENCE
NETWORKS
TEAM
THE TEAM
ATTRIBUTIONS
SPONSORS
COLLABORATION
DESCRIPTION
DESIGN
REQUIREMENTS
MODELLING
HOW IT WORKS
PREMATURE STOP CODON
PROMOTER SELECTION
PLASMID LOSS MODEL
NETWORK MODELLING
AGGREGATION MODEL
CRISPR
PROOF OF CONCEPT
PLASMID LOSS
PROMOTER ANALYSIS
PARTS
PROTOCOLS
NOTEBOOK
SAFETY
SILVER
GOLD
ENGAGEMENT
GENDER EQUITY
ETHICS
ACCESS
INTEGRATED PRACTICES
OPEN SCIENCE
NETWORKS
THE TEAM
ATTRIBUTIONS
SPONSORS
COLLABORATION
Parts
Our System
Hsp104-NSC
Hsp104-SC1
Hsp104-SC2
Biobricks
ADH1-GFP
Gal1-GFP
Cup1-GFP
Gal1-CFP NSC
Gal1-CFP SC1
Gal1-CFP SC2
Promoter Testing
Hsp104-Gal1 (Gene Retention)
Hsp104-Cup1 (Gene Retention)
pXP218-ADH1
pXP218-Gal1
pXP218-Cup1
Parts Improvement
Dual Coloured Reporter Plasmid
sgRNA-RFP Targeting
Hsp104-NSC
This construct is a control to test the metabolic load of the Hsp104-CFP fusion protein and as a baseline against which we compare the change in the [PSI+]/[psi-] state. We synthesized a fragment to clone into the [Hsp-PRS315]. This plasmid will be referred to as the Hsp104 plasmid and is used in several of our experiments.
Figure 1. The Hsp104 plasmid.
Figure 2. The insertion of a Gal1,10 promoter and CFP fusion into the Hsp104 plasmid.
Hsp104-SC1
CFP was synthesized with a stop codon in the place of Tyr-39 (TAC -> TAG) and amplified with flanking ApaI and BamHI sites. The premature stop codon (before the CFP fluorophore) was expected to truncate the protein during normal transcription. Fluorimetry readings were then used to quantify the amount of read-through for the CFP-tagged Hsp104.
Figure 1. The insertion of a Gal1,10 promoter and CFP fusion containing a premature stop codon into the Hsp104 plasmid.
Figure 2. The exact location of the premature stop codon used to make Hsp104-SC1.
Hsp104-SC2
CFP was synthesized with a stop codon in the place of Val-22 (TCA -> TAG) and amplified with flanking ApaI and BamHI sites. The premature stop codon (before the CFP fluorophore) was expected to truncate the protein during normal transcription. Fluorimetry readings were then used to quantify the amount of read-through for the CFP-tagged Hsp104.
Figure 1. The insertion of a Gal1,10 promoter and CFP fusion containing a premature stop codon into the Hsp104 plasmid.
Figure 2. The exact location of the premature stop codon used to make Hsp104-SC1.
ADH1-GFP
This biobrick contains a fusion of the ADH1 promoter and GFP. ADH1 is a eukaryotic promoter; the presence of ethanol induces it. The fusion between it and GFP allows the strength of the promoter to be quantized in varying concentrations of ethanol using the fluorescence of the GFP.
Figure 1. The ADH1 and GFP fusion.
ADH1-GFP
This biobrick is a fusion of the Gal1 promoter and GFP protein. Gal1 is a eukaryotic promoter; the presence of galactose induces it. The fusion allows the strength of the promoter to be quantized in the presence of varying concentrations of galactose using the fluorescence of the GFP.
Figure 1. The Gal1 and GFP fusion.
Cup1-GFP
This biobrick is a fusion of the Cup1 promoter and GFP protein. Cup1 is a eukaryotic promoter; the presence of Cu2+ induces it. The fusion allows the strength of the promoter to be quantized in the presence of varying concentrations of copper ions using the fluorescence of the GFP.
Figure 1. The Cup1 and GFP fusion.
Gal1-CFP NSC
This biobrick is a fusion of the Gal1,10 promoter, Cyan Fluorescent Protein (CFP), and the chaperone protein Hsp104.
Figure 1. The Gal1, CFP, and Hsp104 fusion without a premature stop codon in GFP.
Gal1-CFP SC1
This biobrick is a fusion of the Gal1,10 promoter, Cyan Fluorescent Protein (CFP), and the chaperone protein Hsp104 with a premature stop codon introduced at position 1.
Figure 1. The Gal1, CFP, and Hsp104 fusion with a premature stop codon (SC1) in GFP.
Gal1-CFP SC2
This biobrick is a fusion of the Gal1,10 promoter, Cyan Fluorescent Protein (CFP), and the chaperone protein Hsp104 with a premature stop codon introduced at position 2.
Figure 1. The Gal1, CFP, and Hsp104 fusion with a premature stop codon (SC2) in GFP.
Hsp104-Gal1
The purpose of our gene retention experiments was to observe the retention of our plasmid by the yeast cells. We used GFP-tagged Hsp104 under the influence of different promoters to study the plasmid retention. A low copy number plasmid was used to more accurately study the partitioning of plasmids into daughter cells. It was also important that we use the same plasmid as our working prototype so that the findings from this experiment could be directly applied to our other data.
This construct is the same one used in the prototype (NSC); this experiment was a positive control to show the normal retention of our construct by cells.
Figure 1. The insertion of a Gal1,10 promoter and CFP fusion into the Hsp104 plasmid.
Hsp104-Cup1
The purpose of our gene retention experiments was to observe the retention of our plasmid by the yeast cells. We used GFP-tagged Hsp104 under the influence of different promoters to study the plasmid retention. A low copy number plasmid was used to more accurately study the partitioning of plasmids into daughter cells. It was also important that we use the same plasmid as our working prototype so that the findings from this experiment could be directly applied to our other data.
This construct is the same one used in the prototype (NSC) except with the Gal1,10 promoter replaced by a Cup1 promoter and CFP replaced by GFP; this experiment was a positive control to show the normal retention of our constructs with Cup1 promoters by cells.
Figure 1. The insertion of a Cup1 promoter and GFP fusion into the Hsp104 plasmid.
pXP218-ADH1
Compared to other model organisms, yeast is underrepresented in the iGEM registries. As our biobrick contribution, we characterized the ADH1 promoter. We also characterized the promoters Gal1 and Cup1 as controls against which we could compare our experiments. A high copy number plasmid was used in this case (compared to the low copy Hsp104 plasmid) to maximize the promoter activity for quantification.
ADH1 is a eukaryotic promoter; the presence of ethanol induces it. Using this promoter, we can increase the production of a desired molecule in the yeast cell by adding a small amount of ethanol.
Figure 1. The insertion of a ADH1 promoter and GFP fusion into the Hsp104 plasmid.
pXP218-Gal1
Compared to other model organisms, yeast is underrepresented in the iGEM registries. As our biobrick contribution, we characterized the ADH1 promoter. We also characterized the promoters Gal1 and Cup1 as controls against which we could compare our experiments. A high copy number plasmid was used in this case (compared to the low copy Hsp104 plasmid) to maximize the promoter activity for quantification.
Gal1 is a positively regulated promoter found on the Hsp104 plasmid. Characterization was important for us to understand the transcription rates in a normal cell in order to see the metabolic load of expression of additional Hsp104.
Figure 1. The insertion of a Gal1,10 promoter and GFP fusion into the Hsp104 plasmid.
pXP218-Cup1
Compared to other model organisms, yeast is underrepresented in the iGEM registries. As our biobrick contribution, we characterized the ADH1 promoter. We also characterized the promoters Gal1 and Cup1 as controls against which we could compare our experiments. A high copy number plasmid was used in this case (compared to the low copy Hsp104 plasmid) to maximize the promoter activity for quantification.
Cup 1 is a positively regulated promoter (induced with copper) found on the Sup35 plasmid [BB8]. It is less leaky than other inducible promoters which makes it useful for the study of Sup35, allowing us to control for the amount of Sup35 in the cell to see how the PSI+ state causes further misfolding of Sup35. Characterization of this promoter was therefore important for quantification of the prion response.
Figure 1. The insertion of a Cup1 promoter and GFP fusion into the Hsp104 plasmid.
Dual Coloured Reporter Plasmid
This part is a composite of an RFP gene and a GFP gene from the previously made BioBricks: BBa_I20260 and BBa_J04450, respectively. It can be used in CRISPRi experiments to demonstrate the effectiveness of sgRNA-Cas9 targeting. It can also be used for other gene expression suppression systems to provide quantitative feedback and data. It has been characterized using flow cytometry to demonstrate the expression of both RFP and GFP proteins. Further characterization of this part being used in a CRISPRi system can be found with the BBa_K1645998 BioBrick.
We performed a simple experiment to demonstrate that both genes in this part are properly expressed. We aim to compare their relative fluorescence to each other, and demonstrate their usability in a more complex system with CRISPRi (found in the characterization of BBa_K1645999).
To produce the data, we inoculated the appropriate E. coli culture into LB and grew it for 12hr at 37 degrees Celsius on a shaker at 200RPM. We diluted the culture four-fold into chilled formalin (1X PBS, 4% formaldehyde, 1.5% methanol). We used flow cytometry (Aminis ImageStream MKII) to run a sample and detected fluorescence using an excitation laser wavelength of 488nm at 200mW, as well as SSC at 1.5mW. After acquiring data from 20'000 cells in all channels, we performed analysis on the IDEAS Application v.6 software. This protocol is based off in-house protocols created by previous Waterloo iGEM members and revised over the years by advisors and experienced users.
Figure 1 shows that both RFP and GFP are fluorescing, though at different intensities. Overall, GFP's intensity data averages at 502 intensity units and RFP's intensity data averages at 139 intensity units. This means that GFP fluoresces approximately 3.5x more intensely than RFP. Figure 2 shows the frequency at which cells fluoresce at a particular intensity for GFP on the right and RFP on the left. In all, further experiments can provide more precise measurements of GFP and RFP fluorescence, but we present here adequate fluorescence data for other teams to understand the behavior of the Dual Colour Plasmid in a DH5α chassis.
Figure 1. A Comparison of GFP and RFP expression from the Dual Colour Plasmid
Figure 2. Frequency of GFP and RFP Intensity Measurements
In other characterization experiments detailed with BBa_K1645999, we realized the RFP's promoter (BBa_R0010) has a LacI binding site meaning that expression of our Dual Colour Plasmid in a chassis that produces the lac repressor will require induction with IPTG or lactose. For induction protocols with the this part, please refer to the details found with BBa_K1645999.
sgRNA-RFP Targeting
This part is an sgRNA from the CRISPRi system and is designed to provide (d)Cas9 the specificity to target the promoter of BBa_I20260. It can be used with Streptococcus pyogenes Cas9 and related variants. Here, we use it in conjunction with flow cytometry to demonstrate its ability to repress RFP expression. It has been characterized through numerous experiments presented in the next section.
We performed a series of experiments to demonstrate that this sgRNA when used with a dCas9 protein is able to repress RFP fluorescence when compared to controls.
To produce the data, we inoculated the appropriate E. coli strains into LB and grew it for 4hr to an OD600 of 0.4, followed by induction with IPTG at a final concentration of 1mM for 6hr. For negative controls, we did not add IPTG. Next, we diluted the culture four-fold into chilled formalin (1X PBS, 4% formaldehyde, 1.5% methanol). We used flow cytometry (Aminis ImageStream MKII) to run a sample and detected fluorescence using an excitation laser wavelength of 488nm at 200mW, as well as SSC at 1.5mW. After acquiring data from 20'000 cells in all channels, we performed analysis on the IDEAS Application v.6 software. For Figure 1 and 2, the protocol above was modified such that the cultures after induction were incubated for 9hr instead of 6hr. This protocol is based off in-house protocols created by previous Waterloo iGEM members and revised over the years by advisors and experienced users.
Figure 1. Preliminary Exploratory Comparison of RFP Expression with (a) and without (b) a Complete sgRNA-dCas9 pair
Figure 2. Comparison of (a) RFP with an sgRNA-dCas9 pair and (b) RFP with only dCas9 and no sgRNA
We began our experiments by first checking if our strains from last year were still viable. In Figure 1 (a) we show that the strain with an RFP and a dCas9 leads to expression of RFP, but in Figure 1 (b) we show that the strain with RFP and the complete dCas9-sgRNA pair resulted in repression of RFP fluorescence based on the spike in frequency of cells that have almost zero RFP fluorescence. With this raw data, we were confident moving on with the part characterization and decided to extend the induction time by 3hr in an attempt to get better expression and thus more robust data. Figure 2 provides similar information to Figure 1 where we see the complete sgRNA-dCas9 pair repressing RFP fluorescence. In Figure 2 (b), we see intensities up to 2000 intensity units, but in Figure (a) we see that virtually all cells show less than 500 intensity units - an approximately four-fold repression ability. Finally in Figure 3, we see that for (b) there is very little RFP fluorescence, but in (a) there is essentially no RFP fluorescence as well. This is indicative of IPTG having strong control over the expression of RFP in the E. coli cells. In summary, we demonstrate this part's ability to give dCas9 the specificity to target the promoter of BBa_I20260 and effectively repress RFP fluorescence, thus characterizing this part for the first time.
Figure 3. Comparison between (a) induced and (b) non-induced CRISPRi
External Site © 2016 Waterloo iGEM
