Team:Waterloo/Software
Menu
PROJECT
DESCRIPTION
DESIGN
REQUIREMENTS
MODELLING
BLUEPRINT
HOW IT WORKS
PREMATURE STOP CODON
PROMOTER SELECTION
PLASMID LOSS
NETWORK MODELLING
AGGREGATION MODEL
CRISPR
RESULTS
PROOF OF CONCEPT
PLASMID LOSS
PROMOTER ANALYSIS
LAB
PARTS
PROTOCOLS
NOTEBOOK
SAFETY
POLICY
SILVER
GOLD
ENGAGEMENT
GENDER EQUITY
ETHICS
ACCESS
INTEGRATED PRACTICES
OPEN SCIENCE
NETWORKS
TEAM
THE TEAM
ATTRIBUTIONS
SPONSORS
COLLABORATION
DESCRIPTION
DESIGN
REQUIREMENTS
MODELLING
HOW IT WORKS
PREMATURE STOP CODON
PROMOTER SELECTION
PLASMID LOSS MODEL
AGGREGATION MODEL
CRISPR
NETWORK MODELLING
PROOF OF CONCEPT
PLASMID LOSS
PROMOTER ANALYSIS
PARTS
PROTOCOLS
NOTEBOOK
SAFETY
SILVER
GOLD
ENGAGEMENT
GENDER EQUITY
ETHICS
ACCESS
INTEGRATED PRACTICES
OPEN SCIENCE
NETWORKS
THE TEAM
ATTRIBUTIONS
SPONSORS
COLLABORATION
Lab and Design
Math Modelling
STOP CODON
POSITION
AMINO ACID
FOLDING
REFERENCES
Premature Stop Codon
PREMATURE STOP CODON
UAG stop codon has a lower readthrough rate in [psi-] cells than in [psi+] cells relative to the other stop codons [1]. This feature of UAG in comparison to the other stop codons can be used to demonstrate the greatest increase in readthrough rates [1]. Altering TGG to TAG would have the smallest effect on the CFP sequence [2]. This is because UAG tends to be misread by “promiscuous” tRNATrp (GΨA) which can result in the insertion of a tryptophan [2].
UPSTREAM AND DOWNSTREAM
Figure 1: Image of the 3D structure of CFP.
The process of translational termination is significantly impacted by the nucleotides surrounding the termination codon [3]. The codon CAA in the +123 position is a known readthrough signal in yeast [4]. Since there are no such codons in CFP, a different codon CAG can be used to our advantage instead. CAG encodes for glutamine and G nuclotide can be mutated into a second A to create the desired readthrough signal. This could increase the readthrough rate from 0.88% [1] to 1.2% [1].
While there are known readthrough-stimulating contexts such as CAGCTA in the 3’ context, it was found that a 5’ context of GGG CAA resulted in a 0.65% stop codon readthrough in PSI+, and that a 3’ context of CAA GAA resulted in a 0.4% stop codon readthrough. When these two are combined, however, the readthrough spiked to 3%. [3].
The -1 and -2 codon positions don’t largely affect termination. When choosing surrounding codons, the +1235 and +1236 bases have been demonstrably important. Results have also strongly suggested that the +1236 CAAT and CAAC nucleotide contexts direct stop codon suppression [3]. Based on further findings, C1C4A6, A2A4G6, C1A5T6 and A2T4A6 can be suggested as generally good contexts for stop codon suppression [4].
Analysing the yeast genome reveals that AAA (lysine), ATA (isoleucine), and GTA (valine) often immediately precedes stop codons [3]. Due to limited amount of existing experimental data, we are unable to quantify the increase or decrease of stop codon suppression that would result from altering the -1 or -2 positions [3]. However, this method is considered to be much less impactful than the other method.
IMPACT OF AA CHANGES
Figure 2: Visualising the trade-off we had to make between gaining the highest readthrough rate possible and the perfect natural functionality of CFP.
The nucleotides CAA in the +123 positions after the stop codon is an important component for increasing readthrough efficiency. There are two sets of codons that make Gln, CAA and CAG. As mentioned previously, there are no CAA codons in the CFP sequence and CAG was not present in the sequence either before the molecule’s chromophore. Gln, or glutamine, is an amino acid with a polar, uncharged side chain. The amino acid with the most similar structure is Asn, or asparagine, whose side chain differs from that of Gln by only one carbon. The codons creating Asn are AAT and AAC. AAT is absent from the sequence of CFP, but there were two instances of AAC codons before the chromophore.
There is only one Tyr (tyrosine, a large hydrophobic amino acid) codon before the chromophore and it is followed by GGC, which encodes Gly (the smallest amino acid containing an almost non-existent side chain). It is in an area that has no important secondary structure, but changing this to CAA in order to encoding Gln may not be ideal since it a large polar amino acid going into the place of a small amino acid like Gly. With the objective to significantly increase readthrough rates this change can still be deemed risky. This is because, this area of interest is immediately followed by a helix structure while preceding a positively-charged β-sheet beginning. The fact that it would follow the replaced Tyr may be beneficial for the loss of the large amino acid should it be removed entirely from the sequence, but would likely have a negative impact on the native secondary structure should the stop codon be replaced by Tyr instead of ignored during readthrough.
Should the GGC codon be left as it is, the level of readthrough efficiency would be noticeably lower. Changing CAA to a similar codon, CAG, decreases the readthrough efficiency by more than half, and to further change it to a codon that doesn’t encode Gln, the efficiency decreases below 5% [3].
IMPACT ON FOLDING
Figure 3: Image of the secondary protein folding that occurs in CFP, taken from PDB.
PHYRE was run to estimate the effects of adding the stop codon to the sequence which can be accessed here. The simulation was run with an amino acid sequence that included Tyr instead of simply removing the codon, as would likely be the result of such a mutation [5].
REFERENCES
Keeling, K. M. (2004). Leaky termination at premature stop codons antagonizes nonsense-mediated mRNA decay in S. cerevisiae. RNA , 10(4), 691–703.
Beier, H. (2001). Misreading of termination codons in eukaryotes by natural nonsense suppressor tRNAs. Nucleic Acids Research, 29(23), 4767–4782.
Williams, I., Richardson, J., Starkey, A., & Stansfield, I. (2004). Genome-wide prediction of stop codon readthrough during translation in the yeast Saccharomyces cerevisiae. Nucleic Acids Research, 32(22), 6605–6616.
Namy, O., Olivier, N., Isabelle, H., & Jean-Pierre, R. (2001). Impact of the six nucleotides downstream of the stop codon on translation termination. EMBO Reports, 2(9), 787–793.
Jungreis, I., Lin, M. F., Spokony, R., Chan, C. S., Negre, N., Victorsen, A. Kellis, M. (2011). Evidence of abundant stop codon readthrough in Drosophila and other metazoa. Genome Research, 21(12), 2096–2113.
External Site © 2016 Waterloo iGEM
