<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_Waterloo_InDepthDesign_PlasmidLossModel skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><DIV id="top_title"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:Waterloo/InDepthDesign/PlasmidLossModel</SPAN></H1></DIV><DIV id="HQ_page"><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><DIV class="cd-auto-hide-header" ng-mouseleave="hider()" ng-controller="nav" ng-cloak=""><NAV class="cd-primary-nav"><A href="#cd-navigation" class="nav-trigger"><SPAN>
        Menu
      </SPAN></A><UL id="cd-navigation" class="mobile-menu"><LI onclick="$(this).toggleClass('show-secondary-nav');"><A href="#" ng-click="clickety=true" ng-mouseover="subnav='project'; ">PROJECT</A></LI><LI><A class="paragraph-medium" href="/Team:Waterloo/Description">DESCRIPTION</A></LI><LI><A class="paragraph-medium" href="/Team:Waterloo/Design">DESIGN</A></LI><LI><A class="paragraph-medium" href="/Team:Waterloo/Requirements">REQUIREMENTS</A></LI><LI><A class="paragraph-medium" href="/Team:Waterloo/Model">MODELLING</A></LI></UL><LI onclick="$(this).toggleClass('show-secondary-nav');"><A href="#" ng-click="clickety=true" ng-mouseover="subnav='design';">BLUEPRINT</A></LI><LI><A class="paragraph-medium" href="/Team:Waterloo/InDepthDesign/HowItWorks">HOW IT WORKS</A></LI><LI><A class="paragraph-medium" href="/Team:Waterloo/Software">PREMATURE STOP CODON</A></LI><LI><A class="paragraph-medium" href="/Team:Waterloo/InDepthDesign/PromoterSelection">PROMOTER SELECTION</A></LI><LI><A class="paragraph-medium" href="/Team:Waterloo/InDepthDesign/PlasmidLossModel">PLASMID LOSS</A></LI><LI><A class="paragraph-medium" href="/Team:Waterloo/NetworkModelling">NETWORK MODELLING</A></LI><LI><A class="paragraph-medium" href="/Team:Waterloo/Aggregation">AGGREGATION MODEL</A></LI><LI><A class="paragraph-medium" href="/Team:Waterloo/InDepthDesign/Crispr">CRISPR</A></LI><LI onclick="$(this).toggleClass('show-secondary-nav');"><A href="#" ng-click="clickety=true" ng-mouseover="subnav='results';">RESULTS</A></LI><LI><A class="paragraph-medium" href="/Team:Waterloo/Proof">PROOF OF CONCEPT</A></LI><LI><A class="paragraph-medium" href="/Team:Waterloo/InDepthDesign/PlasmidLossModel">PLASMID LOSS</A></LI><LI><A class="paragraph-medium" href="/Team:Waterloo/Results/PromoterAnalysis">PROMOTER ANALYSIS</A></LI><LI onclick="$(this).toggleClass('show-secondary-nav');"><A href="#" ng-click="clickety=true" ng-mouseover="subnav='lab'">LAB</A></LI><LI><A class="paragraph-medium" href="/Team:Waterloo/Parts">PARTS</A></LI><LI><A class="paragraph-medium" href="/Team:Waterloo/Experiments">PROTOCOLS</A></LI><LI><A class="paragraph-medium" href="/Team:Waterloo/Notebook">NOTEBOOK</A></LI><LI><A class="paragraph-medium" href="/Team:Waterloo/Safety">SAFETY</A></LI><LI onclick="$(this).toggleClass('show-secondary-nav');"><A href="#" ng-click="clickety=true" ng-mouseover="subnav='practices'">POLICY</A></LI><LI><A class="paragraph-medium" href="/Team:Waterloo/HP/Silver">SILVER</A></LI><LI><A class="paragraph-medium" href="/Team:Waterloo/HP/Gold">GOLD</A></LI><LI><A class="paragraph-medium" style="font-weight: bold; color: #cd5f94" href="/Team:Waterloo/Engagement">ENGAGEMENT</A></LI><LI><A class="paragraph-medium" style="color: #cd5f94" href="/Team:Waterloo/Engagement/GenderEquity">GENDER EQUITY</A></LI><LI><A class="paragraph-medium" style="color: #cd5f94" href="/Team:Waterloo/Engagement/Ethics">ETHICS</A></LI><LI><A class="paragraph-medium" style="color: #cd5f94" href="/Team:Waterloo/Engagement/Access">ACCESS</A></LI><LI><A class="paragraph-medium" style="font-weight:bold; color: #5f94cd;" href="/Team:Waterloo/Integrated_Practices">INTEGRATED PRACTICES</A></LI><LI><A class="paragraph-medium" style="color: #5f94cd;" href="/Team:Waterloo/Integrated_Practices/OpenScience">OPEN SCIENCE</A></LI><LI><A class="paragraph-medium" style="color: #5f94cd;" href="/Team:Waterloo/Integrated_Practices/Networks">NETWORKS</A></LI><LI onclick="$(this).toggleClass('show-secondary-nav');"><A href="#" ng-click="clickety=true" ng-mouseover="subnav='team';">TEAM</A></LI><LI><A class="paragraph-medium" href="/Team:Waterloo/Team">THE TEAM</A></LI><LI><A class="paragraph-medium" href="/Team:Waterloo/Attributions">ATTRIBUTIONS</A></LI><LI><A class="paragraph-medium" href="/Team:Waterloo/Sponsors">SPONSORS</A></LI><LI><A class="paragraph-medium" href="/Team:Waterloo/Collaborations">COLLABORATION</A></LI></NAV><NAV class="cd-secondary-nav" id="projecttab" ng-show="subnav=='project'"><UL><LI><A class="paragraph-medium" href="/Team:Waterloo/Description">DESCRIPTION</A></LI><LI><A class="paragraph-medium" href="/Team:Waterloo/Design">DESIGN</A></LI><LI><A class="paragraph-medium" href="/Team:Waterloo/Requirements">REQUIREMENTS</A></LI><LI><A class="paragraph-medium" href="/Team:Waterloo/Model">MODELLING</A></LI></UL></NAV><NAV class="cd-secondary-nav" id="designtab" ng-show="subnav == 'design'"><UL><LI><A class="paragraph-medium" href="/Team:Waterloo/InDepthDesign/HowItWorks">HOW IT WORKS</A></LI><LI><A class="paragraph-medium" href="/Team:Waterloo/Software">PREMATURE STOP CODON</A></LI><LI><A class="paragraph-medium" href="/Team:Waterloo/InDepthDesign/PromoterSelection">PROMOTER SELECTION</A></LI><LI><A class="paragraph-medium" href="/Team:Waterloo/InDepthDesign/PlasmidLossModel">PLASMID LOSS MODEL</A></LI><LI><A class="paragraph-medium" href="/Team:Waterloo/Aggregation">AGGREGATION MODEL</A></LI><LI><A class="paragraph-medium" href="/Team:Waterloo/InDepthDesign/Crispr">CRISPR</A></LI><LI><A class="paragraph-medium" href="/Team:Waterloo/NetworkModelling">NETWORK MODELLING</A></LI></UL></NAV><NAV class="cd-secondary-nav" id="resultstab" ng-show="subnav == 'results'"><UL><LI><A class="paragraph-medium" href="https://2016.igem.org/Team:Waterloo/Proof">PROOF OF CONCEPT</A></LI><LI><A class="paragraph-medium" href="/Team:Waterloo/Results/PlasmidLoss">PLASMID LOSS</A></LI><LI><A class="paragraph-medium" href="/Team:Waterloo/Results/PromoterAnalysis">PROMOTER ANALYSIS</A></LI></UL></NAV><NAV class="cd-secondary-nav" id="labtab" ng-show="subnav == 'lab'"><UL><LI><A class="paragraph-medium" href="/Team:Waterloo/Parts">PARTS</A></LI><LI><A class="paragraph-medium" href="/Team:Waterloo/Experiments">PROTOCOLS</A></LI><LI><A class="paragraph-medium" href="/Team:Waterloo/Notebook">NOTEBOOK</A></LI><LI><A class="paragraph-medium" href="/Team:Waterloo/Safety">SAFETY</A></LI></UL></NAV><NAV class="cd-secondary-nav" id="practices" ng-show="subnav == 'practices'"><UL><LI><A class="paragraph-medium" href="/Team:Waterloo/HP/Silver">SILVER</A></LI><LI><A class="paragraph-medium" href="/Team:Waterloo/HP/Gold">GOLD</A></LI><LI><A class="paragraph-medium" style="font-weight: bold; color: #cd5f94" href="/Team:Waterloo/Engagement">ENGAGEMENT</A></LI><LI><A class="paragraph-medium" style="color: #cd5f94" href="/Team:Waterloo/Engagement/GenderEquity">GENDER EQUITY</A></LI><LI><A class="paragraph-medium" style="color: #cd5f94" href="/Team:Waterloo/Engagement/Ethics">ETHICS</A></LI><LI><A class="paragraph-medium" style="color: #cd5f94" href="/Team:Waterloo/Engagement/Access">ACCESS</A></LI><LI><A class="paragraph-medium" style="font-weight:bold; color: #5f94cd;" href="/Team:Waterloo/Integrated_Practices">INTEGRATED PRACTICES</A></LI><LI><A class="paragraph-medium" style="color: #5f94cd;" href="/Team:Waterloo/Integrated_Practices/OpenScience">OPEN SCIENCE</A></LI><LI><A class="paragraph-medium" style="color: #5f94cd;" href="/Team:Waterloo/Integrated_Practices/Networks">NETWORKS</A></LI></UL></NAV><NAV class="cd-secondary-nav" id="teamtab" ng-show="subnav == 'team'"><UL><LI><A class="paragraph-medium" href="/Team:Waterloo/Team">THE TEAM</A></LI><LI><A class="paragraph-medium" href="/Team:Waterloo/Attributions">ATTRIBUTIONS</A></LI><LI><A class="paragraph-medium" href="/Team:Waterloo/Sponsors">SPONSORS</A></LI><LI><A class="paragraph-medium" href="/Team:Waterloo/Collaborations">COLLABORATION</A></LI></UL></NAV></DIV><DIV class="labsticker">
  Lab and Design
</DIV><DIV class="mathsticker">
  Math Modelling
</DIV><DIV class="container-fluid"><DIV class="bloodorangeisacolour"><DIV class="header-image" style="background-image: url(https://static.igem.org/mediawiki/2016/8/8b/T--Waterloo--lab_bench.jpg);"><H3 class="header-text"><SPAN style="color: white; background-color: #000;opacity: 0.8;">Plasmid Loss Model</SPAN></H3></DIV></DIV></DIV><DIV class="wcontent-box"><DIV class="wcontent-title">
   Background
  </DIV><DIV class="wcontent"><P>
Plasmids are small circular DNA strands found in cells that can replicate independently of the cell’s chromosome. They are an incredibly important in biotechnology. Researchers often insert synthetic plasmids into cells, called DNA vectors. The cells then use their molecular machinery to express genes of interest on that plasmid and to replicate it. These techniques are used in this and many projects.
    </P><P>
However, it has been observed that plasmid loss is a common phenomenon. Over time cells with plasmids may divide into plasmid-free and plasmid containing cells. This is a serious problem for researchers that hope to maximize the number of plasmid-carrying cells in their culturewhen selection mechanisms should not be applied. Hence here we explore a model to better understand how plasmid loss works with the intention of developing methods to limit plasmid loss in the future.
    </P></DIV></DIV><DIV class="wcontent-box"><DIV class="wcontent-title">
     Equation and Parameters
    </DIV><DIV class="wcontent"><P>The transformation and expression of plasmids in yeast systems load down cells. The generation of plasmid-free cells is the result of plasmid loss due to improper segregation at cell division or non-replication (Sardonini, 1987). The decline of plasmids and increase in plasmid-free cells results in loss of production of the gene product of interest, Hsp104, encoded on the plasmid. This can be explained by the two types of plasmid burden: replication-based and metabolic-based.</P><P>Plasmids used in yeast have a wide range of adjustable parameters that affect cell growth rates and plasmid copy number (Karim, 2013). These parameters include selection marker (reporter gene introduced to cell to indicate success of introduction of foreign DNA into cell), promotor, origin of replication, and strain ploidy (number of sets of chromosomes in a cell). </P><P>It has been concluded that promoter choice had a little effect on growth rate. However, copy number does have an effect on growth rate. High copy plasmids have a lower growth rate than low copy. This information guided the decision to continue lab experiments using the low copy number plasmid, which contained 2-5 copies per cell (Karim, 2013).</P><P>It was also concluded that the plasmid carrying strain fraction diminished with time depending on the probability of plasmid loss and the growth rates of each strain (Sardonini, 1987). This lead to the idea that a dependency exists between plasmid carrying cells and plasmid free cells. All of this information was put together in order to come up with a representation of plasmid loss.</P><P>Plasmid stability is often measured by measuring the number of plasmid-free cells over time in a culture of plasmid-containing cells. The gene retention team, however, was specifically interested in modelling the fraction of cells with plasmid. This would be done by further investigating the two equations below.</P><DIV class="wcontent-caption"><P>Equation 1: Rate of plasmid-free cell production(Lau, 2013)</P></DIV><DIV class="wcontent-caption"><P>Equation 2: Rate of plasmid loss (Sardonini, 1987)</P></DIV><P>Equation 1 models the rate of production of plasmid-free cells from plasmid containing cells. The equation was derived from that fact that cells increase by a factor 2-p each division. Tx represents the generation time of a plasmid containing cell. Equation 2 yields the rate of plasmid loss. The probability of plasmid loss, p, was retrieved from the Sardonini paper; in this paper is was assumed to be a constant value of 0.14, although plasmid loss probability could be a function of environmental conditions. Plasmid-carrying cells are represented by X+, and S (6.8 mg phosphate/L) represents the single limiting substrate used to limit the growth of plasmid-free cells. These two equations provide insight into the number of plasmid-free cells and the number of plasmid containing cells remaining due to plasmid loss. </P><P>Other measures of interest included measuring the growth rates of plasmid-free cells before and after segregated loss. The team aimed to gather this data from the lab team in order to be able to compare the rate of growth for the plasmid-free vs. plasmid-bearing cells.</P></DIV></DIV><DIV class="wcontent-box"><DIV class="wcontent-title">
     Assumptions
    </DIV><DIV class="wcontent"><P>Several assumptions had to be made in order for the model to hold true for Hsp104. It has been assumed that the model is applicable to the lab team’s parameters although it was not possible to verify the model with data from the lab team.</P><P>Some of the gene retention sub-team goals included measuring the actual rate of loss for the strain, promoter, and other parameters set by the lab team. Through the use of a fluorimeter, fluorescence of the green fluorescent protein (GFP) indicating the presence and expression of Hsp104 on the plasmids could have been detected.  This data would have helped to confirm the accuracy of the model and make any necessary adjustments.</P></DIV></DIV><DIV class="wcontent-box"><DIV class="wcontent-title">
     Lab Experiment Design
    </DIV><DIV class="wcontent"><P>This set of experiments was done to collect data to test the viability of the plasmid loss equations introduces in the plasmid loss equation and parameter section. The data is used to predict the total duration of the system’s effect on living systems and is an important consideration for feasibility in real-world application.  </P><P>Pxp218 plasmid is chosen to be the high copy number and Hsp104 to be low copy number plasmid. Sup35 plasmid was considered as a low copy number plasmid, but was replaced due to concerns of prion aggregation during testing. The plasmids will contain GFP reporter gene for a fluorescence test to indicate the plasmid DNA presence. For each of the plasmid type, a variant with the promoter absent to serve as the -ve control in ensuring there is no external source of fluorescence.</P><P>To begin the experiment, the cell host will be transformed with 5 different plasmids: Pxp218 no promoter, Pxp218 inducible promoter, Hsp104 no promoter, Hsp104 inducible promoter, and a negative control with no plasmids. The 5 different types of transformed cells will be separately grown on selective media plates for 12 hours and the colonies subject to observations on growth rate and fluorescence.</P><P>Growth rate: The purpose of this test is to determine whether  the additional metabolic load in expressing the plasmid genes have any significant effect on cell growth. Grow cells on nutrient broth and measure cell concentration with spectrophotometer to obtain the OD value for broth turbidity. The measurements are taken over the course of 48 hours with 2-hour time points in which readings are taken. </P><P>Fluorescence: The purpose of this analysis is to determine plasmid loss rate by detecting GFP expressions from cell culture with a flow cytometer. The amount of fluorescence is directly correlated with plasmid presence in the cells and so any changes in plasmid presence can be monitored over 48 hours, with 2 hour time-points in between.</P></DIV></DIV><DIV class="wcontent-box"><DIV class="wcontent-title">
     Results
    </DIV><DIV class="wcontent"><DIV class="wcontent-caption"><P>Figure 1: Theoretical cell concentration over time (Sardonini, 1987)</P></DIV><P>The experimental results are shown in Figure 1. The gray line represents growth of cells with no plasmid (X-), orange with a plasmid (X+), and blue with the plasmid expressing the protein. The equations are modeled with similar initial conditions in Figure 2. (Sardonini, 1987). Figure 2 shows no plasmid (X-) and plasmid (X+) theoretical graph curves. Both the theoretical graph experimental results graphs take on similar long-term behavior. They start with exponential growth then reaching a steady state, however, this is to be expected with cell growth. Beyond the steady state behavior, the math model and the experimental results diverge. In the math model, the steady state concentration of plasmid containing cells (X+) is greater than the concentration of cells without the plasmid (X-) whilst in the experimental results it is the opposite.</P><P>We hypothesize that the main reason that the model equation diverges from experimental results is the metabolic load a plasmid carrying cell faces. The cell with the plasmid must put energy into replicating the plasmid rather than replicating this own DNA. (Paulsson, J., &amp; Ehrenberg, M. (1998). This idea is further supported by the experimental growth curve of cells with plasmid and expressing the protein (blue). They too had a lower steady state value than the nonplasmid carrying the cell. This while having similar steady state values as the plasmid (X+) cell. </P><DIV class="wcontent-caption"><P>Figure 2: Modelled concentration of cells over time</P></DIV><P>A future model of cell gene retention must account for plasmid loss due to improper segregation at cell division and the metabolic load a cell with a plasmid experiences. Due to the poor mechanistic understanding of plasmid loss and plasmid, metabolic load future models with most likely have to be imperially based. </P></DIV></DIV><DIV class="wcontent-box"><DIV class="wcontent-title">
   References
  </DIV><DIV class="wcontent"><P>Ay, A., &amp; Arnosti, D. N. (2011). Mathematical modeling of gene expression: A guide for the perplexed biologist. Critical Reviews in Biochemistry and Molecular Biology, 46(2), 137-151. </P><P>Hjortso, M. A., &amp; Bailey, J. E. (1984). Plasmid stability in budding yeast populations: Steady-state growth with selection pressure. Biotechnol. Bioeng. Biotechnology and Bioengineering, 26(5), 528-536.</P><P>Karim, A. S., Curran, K. A., &amp; Alper, H. S. (2013). Characterization of plasmid burden and copy number in Saccharomyces cerevisiae for optimization of metabolic engineering applications. FEMS Yeast Research, 13(1), 107-116. </P><P>Lau, Billy T.c., Per Malkus, and Johan Paulsson. &quot;New Quantitative Methods for Measuring Plasmid Loss Rates Reveal Unexpected Stability.&quot; Plasmid 70.3 (2013): 353-61. </P><P>Paulsson, J., &amp; Ehrenberg, M. (1998). The trade-off between segregational stability and metabolic burden: A mathematical model of plasmid ColE1 replication control.  Journal of Molecular Biology, 279(1), 73-88. </P><P>Paulsson, J., &amp; Ehrenberg, M. (2001). Noise in a minimal regulatory network: Plasmid copy number control. Quarterly Reviews of Biophysics, 34(1), 1-59. </P><P>Romanos, M. A., Scorer, C. A., &amp; Clare, J. J. (1992). Foreign gene expression in yeast: A review. Yeast, 8(6), 423-488.</P><P>Sardonini, C. A., &amp; Dibiasio, D. (1987). A model for growth of Saccharomyces cerevisiae containing a recombinant plasmid in selective media. Biotechnol. Bioeng. Biotechnology and Bioengineering, 29(4), 469-475. </P></DIV></DIV><P class="footer-links" style="font-size:0.8em;text-align:center!important;"><A style="padding-right:0;font-size:1em;" href="http://igem.uwaterloo.ca">External Site</A>© 2016 Waterloo iGEM
    </P></DIV></DIV></DIV></DIV></DIV></BODY></HTML>