Team:Waterloo/InDepthDesign/Crispr
Menu
PROJECT
DESCRIPTION
DESIGN
REQUIREMENTS
MODELLING
BLUEPRINT
HOW IT WORKS
PREMATURE STOP CODON
PROMOTER SELECTION
PLASMID LOSS
NETWORK MODELLING
AGGREGATION MODEL
CRISPR
RESULTS
PROOF OF CONCEPT
PLASMID LOSS
PROMOTER ANALYSIS
LAB
PARTS
PROTOCOLS
NOTEBOOK
SAFETY
POLICY
SILVER
GOLD
ENGAGEMENT
GENDER EQUITY
ETHICS
ACCESS
INTEGRATED PRACTICES
OPEN SCIENCE
NETWORKS
TEAM
THE TEAM
ATTRIBUTIONS
SPONSORS
COLLABORATION
DESCRIPTION
DESIGN
REQUIREMENTS
MODELLING
HOW IT WORKS
PREMATURE STOP CODON
PROMOTER SELECTION
PLASMID LOSS MODEL
AGGREGATION MODEL
CRISPR
NETWORK MODELLING
PROOF OF CONCEPT
PLASMID LOSS
PROMOTER ANALYSIS
PARTS
PROTOCOLS
NOTEBOOK
SAFETY
SILVER
GOLD
ENGAGEMENT
GENDER EQUITY
ETHICS
ACCESS
INTEGRATED PRACTICES
OPEN SCIENCE
NETWORKS
THE TEAM
ATTRIBUTIONS
SPONSORS
COLLABORATION
Lab and Design
Math Modelling
CRISPRi
Summary
Figure 1: CRISPR action, demonstrating how Cas9 binds to sgRNA to cleave target DNA.
CRISPR (clustered regularly interspaced short palindromic repeats) is a system used for targeting specific sites of a genome (Cong, 2013; Jinek, 2012). The system is comprised of a Cas protein and an sgRNA which is a short guide RNA, usually about 20nt, which specifies the target site on the genome(Cong, 2013; Jinek, 2012). CRISPR is a very popular tool in genome editing because of its ability to cut specific sites, however it was originally discovered as an adaptive immune system in bacteria (Doudna, 2014). This function of the CRISPR system allows for a targeted knockout of a protein in question.
Application for PriON to PriOFF
Our system is a tool that allows the overexpression a protein of interest, which could be used to determine if the protein can slow the [PSI+] response or even cure it. It would also be useful to see if the opposite (a decrease in the expression of a protein of interest) could have similar effects. For instance, overexpressing hsp104 can cure the response by breaking down and refolding misfolded sup35, but knocking down hsp104 expression can also cure the response because the sup35 aggregates need hsp104 in order to replicate. The addition of a CRISPR interference system using dCas9 allows the targeting a protein of interest to cause this knockdown of expression. If we use our system to insert a premature stop codon in front of the CRISPR gene, then it will only be expressed during a [PSI+] state.
dCas9
Figure 2: CRISPRi knockdown action, demonstrating how dCas9 binds to sgRNA to prevent expression.
Most CRISPR systems use a Cas protein called Cas9 which makes a cut after binding to the DNA. Our system would use dCas9 which is a Cas protein that binds to a site and prevents transcription of that protein. This acts as a more reliable method for knocking out a protein, since cuts can sometimes be repaired effectively through non-homologous end joining.
sgRNAs
All sgRNAs must start with a certain sequence of nucleotides called a PAM site, or protospacer adjacent motif. This sequence is recognised by the Cas protein and is thus necessary for it to bind to the sgRNA. The sgRNA recognises the target DNA, thereby acting as an intermediate between the Cas protein and the target. The most common PAM site is NGG where N is any nucleotide.
The target DNA must be carefully chosen in order to avoid accidentally targeting genes important to the host organism, in this case Saccharomyces cerevisiae. CRISPR-Cas systems have been effectively employed in yeast hosts on a number of occasions, indicating that our intentions to do the same have a good chance of success (DiCarlo, 2013; 2015; Smith 2016).
References
Cong, L., Ran, F.A., Cox, D., Lin, S., Barretto, R., Habib, N., Hsu, P.D., Wu, X., Jiang, W., Marraffini, L.A., & Zhang, F., 2013. Multiplex genome engineering using CRISPR/Cas systems. Science Feb 15;339(6121):819-23.
DiCarlo, J.E. et al., 2013. Genome engineering in Saccharomyces cerevisiae using CRISPR-Cas systems. Nucleic acids research, 41(7), pp.4336–43.
DiCarlo, J.E. et al., 2015. Safeguarding CRISPR-Cas9 gene drives in yeast. Nature Biotechnology, 33(12), pp.1250–1255.
Doudna, J.A. & Charpentier, E., 2014. The new frontier of genome engineering with CRISPR-Cas9. Science, 346(6213), pp.1258096–1258096.
Jinek, M. et al., 2012. A Programmable Dual-RNA–Guided DNA Endonuclease in Adaptive Bacterial Immunity. Science, 337(6096), p.816 LP-821.
Smith, J.D. et al., 2016. Quantitative CRISPR interference screens in yeast identify chemical-genetic interactions and new rules for guide RNA design. Genome Biology, 17(1), p.45.
External Site © 2016 Waterloo iGEM
