"
Page
Discussion
View source
History
teams
Log in
 
Team:Purdue/Project/BCD
From 2013.igem.org
Menu
Home
Home
Accomplishments
Team
Official Profile
Student Bios
Adviser Bios
Purdue University
Project
Overview
Robustness
Bicistronic Design
Standardized Datasheets
Parts
Parts Submitted
Attributions
Notebook
Laboratory Notebook
Protocols
Safety
Human Practices
Overview and Project Impact
Outreach to Community
Outreach to High School
Biomaker Bench
Girl Scout Biotechnology Badge
Characterization Collaboration
Follow Us!
Sponsors
Sponsors
iGEM
Bicistronic Design
Improving and Regulating Protein Expression
Overview
Within the last few decades, DNA synthesis, sequencing and recombination technologies have significantly increased in sophistication and in turn practicality. However, the unpredictable variances in the behavior of standardized parts across setting changes remain an obstacle to time-efficient, rational gene network design. Vivek Mutalik and colleagues have in the past year designed bicistronic expression operating units (EOUs). Using these bicistronic designs (BCDs) Mutalik and others managed to express arbitrary genes within a twofold target window with ~93% reliability (Mutalik, 2012). The next best system at the time could only do the same with ~47% reliability (Salis, 2009). This summer, we attempted to make four existing BCD designs BioBrick compatible and modify them for assembly in an all-in-one-pot Golden Gate Cloning Reaction using the type IIs restriction enzyme BpiI.
Background
A bicistronic design works to standardize the mRNA secondary structure over the RBS-CDS junction. This in turn reduces variation in ribosome binding and expression across different RBS-CDS combinations (Mutalik, 2012). The above figure comes from the publication of Mutalik et al. listed in the references.
Golden gate cloning relies on type IIs restriction enzymes, such as BpiI, to assemble multiple fragments in one reaction. The unique feature of IIs REs which allows this is the separation of their recognition sites from their cut sites (Engler, 2009). The 2012 Freiburg iGEM team has an excellent page on Golden Gate assembly for those who wish to learn more. https://2012.igem.org/Team:Freiburg/Project/Golden
Design and Assembly
Assembly of Bicistronic Expression Operating Units<\p>
(Parts are not depicted to scale)
We originally planned to synthesize the BCDs with double terminators included as gBlocks. This would give them the exact same sequence after GGC as those sent to us by V. Mutalik. However, the hairpins of the BBa_B0015 sequence made gBlock synthesis unfeasible. Therefore, we combined the BCDs with BBa_B0015 and the pSB1C3 plasmid through standard assembly. The intent was that future users would start at step 2 of the above figure after synthesizing a compatible promoter and coding sequence of choice.
Existing promoters and coding sequences can be made compatible with this assembly by PCR primer overhangs or re-synthesis including the necessary BpiI flanking regions and sticky ends.
Promoter synthesis - 8bp-GAAGACCGTAAC-original sequence-GGGCATGTCTTC-8bp
Promoter F primer  - 5'8bp-GAAGACCGTAAC...
Promoter R primer  - 5'8bp-GAAGACATGCCC...
CDS synthesis      - 8bp-GAAGACCGA-original CDS-TTCGTCTTC-8bp
CDS F primer       - 5'8bp-GAAGACCGA...
CDS R primer       - 5'8bp-GAAGACGAA...
If not for time and budget constraints, we would have synthesized or PCR amplified many more (10-20) coding sequences with flanking BpiI sites for insertion into the BCDs. As well we would make monocistronic operating units with the the same promotors (Ptrc*) and coding sequences. This would have allowed us to calculate variances and compare BCD with MCD much the same way as did Mutalik et al. in their original paper.
Results and Conclusion
After we cloned in Golden Gate, we managed to successfully culture BCD constructs 7, 18, and 23 in liquid culture. After growing these up to a good OD, we spun them down and ran them through a fluorometer to measure the red fluorescence of mrfp. This indicates that the BpiI-GGC assembly scheme, our mfrp1 gene with synonymous DNA base substitutions and the Ptrc* promoter work at least in the context of a bicistronic expression operating unit.
The growth curve on the right contains three curves: control BL21 cells, BL21 cells with just pSB1C3, and BL21 cells with pSB1C3 with the prtc* promoter inserted. We believe that the strain of producing the antibiotic resistance delayed the pSB1C3 and prtc* cultures from entering exponential phase at the same time the control BL21 cells did. Because the control cells entered the phase quicker, they also reached the stationary phase faster. Overall, we determined that the prtc* promoter has no significant effect on the growth of BL21 cells that would be problematic for future assays.<\p>
In the graph shown, a plot of intensity vs wavelength is shown. The prtc* promoter was inserted into three separate bicistronic design constructs, which were low, medium, and high expression levels. The three plots represent the three BCD constructs, and the heights of the peaks align with the expected low, medium, and high expression levels. Although the intensity is quite low, the emission spectrum has a spike right where it should be for RFP. We believe that the low intensity of the fluorescence was due to an error in the dilution of the cell culture that we measured. The picture to the right shows the cells expressing RFP under blacklight.
Future Work
Our next immediate step were we to have more time would have been sequencing the fully assembled BCD EOUs to compare them to the original BCD EOUs we received from Vivek Mutalik. We would expect them to have identical sequences up until the BioBrick scar preceding our constructs' double terminators. Identical sequences should result in equal functionality assuming other variables, such as host strain held constant. However, more rigorous characterization of BCD function would require more than the one mrfp1 coding sequence we had in our budget to synthesize.
Although we could not submit our BpiI-GGC-compatible BCD EOUs to iGEM HQ by the competition deadline, we plan to more fully characterize them and send them to the Registry in the near future. Further work should be directed towards making more promoters and coding sequence BioBricks with the BpiI sites needed for cloning into our BCDs.
References
Mutalik, V., Endy, D., Guimaraes, J., Cambray, G., Lam, C., Juul, M., Tran, A., & Paull, M. (2013). Precise and reliable gene expression via standard transcription and translation initiation elements . Nature, 10(4),
Salis, H.M., Mirsky, E.A. & Voigt, C.A. Automated design of synthetic ribosome binding sites to control protein expression. Nat. Biotechnol. 27, 946–950 (2009).
Blog, Contact & More
Our Blog & Website: purduebiomakers.com
Twitter: @ purduebiomakers
Facebook: Purdue Biomakers Page
Email: purduebiomakers@gmail.com
Facebook
Tweets by @ PurdueBiomakers
Tweets by @PurdueBiomakers
Retrieved from " http://2013.igem.org/Team:Purdue/Project/BCD "
Recent changes
What links here
Related changes
Special pages
My preferences
Printable version
Permanent link
Privacy policy
Disclaimers
