Team:Tokyo Tech/AHL Assay/Rhl System Assay
3-2-3 Rhl system Assay
Contents
1. Introduction
2. Summary of the experiments
3. Results
4. Discussion
5. Materials and methods
   5-1. Construction
   5-2. Assay protocol
6. Reference
1. Introduction
As shown in the previous sections, Prhl( BBa_I14017 ) activity was too weak compared to Plux to operate the final genetic circuits. Similarly, simulation results suggested that the circuits did not work as we desired due to the same reason (read the Model bage ). We expected that this situation can be circumvented by using the previously improved Prhl ( BBa_K1529310 , BBa_K1529300 ) that was established by iGEM 2014 team Tokyo_Tech. However, these Prhl promoters were also found to be inappropriate for two reasons (written in the Discussion). Therefore, we improved the Prhl Promoter by our own team and succeeded to obtain novel and appropriate Prhl (Noticeable Mutant) ( BBa_K1949060 ), hereafter referred to as Prhl(NM).
2. Summary of the experiments
The purpose of the experiment in this section is to create a strong Prhl for our final genetic circuits.
This experiment consists of the three parts below.
Ⅰ. Improved Prhl by iGEM 2014 team Tokyo_Tech and characterization of rhlR
Ⅱ. Improvement of the wild type Prhl
Ⅲ. Comparison of the improved Prhl by iGEM 2014 team Tokyo_Tech to our original improved Prhl
-Plasmids
Ⅰ. Improved Prhl by iGEM 2014 team Tokyo_Tech and rhlR assay
A. Pcon ‐ rbs ‐ rhlR - LVA( BBa_C0071 ) (pSB6A1),
        Prhl(LR)( BBa_K1529310 ) ‐ rbs ‐ gfp (pSB3K3) (Fig. 3-2-3-2-1)
Fig. 3-2-3-2-1 Pcon ‐ rbs ‐ rhlR ‐ LVA (pSB6A1),
Prhl(LR) ‐ rbs ‐ gfp (pSB3K3)
B. Pcon ‐ rbs ‐ rhlR( BBa_C0171 ) (pSB6A1), Prhl(LR) ‐ rbs ‐ gfp (pSB3K3) (Fig. 3-2-3-2-2)
Fig. 3-2-3-2-2 Pcon ‐ rbs ‐ rhlR (pSB6A1),
Prhl(LR) ‐ rbs ‐ gfp (pSB3K3)
C. Pcon ‐ rbs ‐ rhlR ‐ LVA (pSB6A1), Prhl(RL)( BBa_K1529300 ) ‐ rbs ‐ gfp (pSB3K3) (Fig. 3-2-3-2-3)
Fig. 3-2-3-2-3 Pcon ‐ rbs ‐ rhlR ‐ LVA (pSB6A1),
Prhl(RL) ‐ rbs ‐ gfp (pSB3K3)
D. Pcon ‐ rbs ‐ rhlR (pSB6A1), Prhl(RL) ‐ rbs ‐ gfp (pSB3K3) (Fig. 3-2-3-2-4)
Fig. 3-2-3-2-4 Pcon ‐ rbs ‐ rhlR (pSB6A1),
Prhl(RL) ‐ rbs ‐ gfp (pSB3K3)
E. pSB6A1, pSB3K3 …Nagative control (Fig. 3-2-3-2-5)
Fig. 3-2-3-2-5 pSB6A1, pSB3K3
Ⅱ. Improvment the wild type Prhl
A. Prhl( BBa_R0071 ) ‐ rbs ‐ gfp (pSB3K3) (Fig. 3-2-3-2-6)
Fig. 3-2-3-2-6 Prhl ‐ rbs ‐ gfp (pSB3K3)
B. Pcon ‐ rbs ‐ rhlR - LVA (pSB6A1), Prhl(NM) ‐ rbs ‐ gfp( BBa_ K1949060 ) (pSB3K3) (Fig. 3-2-3-2-7)
Fig. 3-2-3-2-7 Pcon ‐ rbs ‐ rhlR ‐ LVA (pSB6A1),
Prhl(NM) ‐ rbs ‐ gfp (pSB3K3)
C. Pcon ‐ rbs ‐ rhlR - LVA (pSB6A1), Prhl(M) ‐ rbs ‐ gfp (pSB3K3) (Fig. 3-2-3-2-8)
Fig. 3-2-3-2-8 Pcon ‐ rbs ‐ rhlR ‐ LVA (pSB6A1),
Prhl(M) ‐ rbs ‐ gfp (pSB3K3)
D. Pcon ‐ rbs ‐ rhlR - LVA (pSB6A1), Prhl(WT) ‐ rbs ‐ gfp (pSB3K3) (Fig. 3-2-3-2-9)
Fig. 3-2-3-2-9 Pcon ‐ rbs ‐ rhlR ‐ LVA (pSB6A1),
Prhl(WT) ‐ rbs ‐ gfp (pSB3K3)
E. pSB6A1, pSB3K3 …Nagative control (Fig. 3-2-3-2-5)
Fig. 3-2-3-2-5 pSB6A1, pSB3K3
Ⅲ. Comparison the improved Prhl by iGEM 2014 Tokyo_Tech Team to our original improved Prhl
A. Pcon ‐ rbs ‐ rhlR - LVA (pSB6A1), Prhl(LR) ‐ rbs ‐ gfp (pSB3K3) (Fig. 3-2-3-2-1)
Fig. 3-2-3-2-1 Pcon ‐ rbs ‐ rhlR ‐ LVA (pSB6A1),
Prhl(LR) ‐ rbs ‐ gfp (pSB3K3)
B. Pcon ‐ rbs ‐ rhlR ‐ LVA (pSB6A1), Prhl(NM) ‐ rbs ‐ gfp (pSB3K3) (Fig. 3-2-3-2-7)
Fig. 3-2-3-2-7 Pcon ‐ rbs ‐ rhlR ‐ LVA (pSB6A1),
Prhl(NM) ‐ rbs ‐ gfp (pSB3K3)
C. pSB6A1, pSB3K3…Negative control (Fig. 3-2-3-2-5)
Fig. 3-2-3-2-5 pSB6A1, pSB3K3
3. Results
Ⅰ. Improved Prhl by iGEM 2014 team Tokyo_Tech and characterization of rhlR
We found that Prhl(RL) ( BBa_K1529300 ) activity was weak and the expression level from this promoter depended on the presence of LVA tag at the C-terminus of RhlR protein (Fig.3-2-3-3-1); LVA-tagged proteins are prone to be degraded by cellular proteases. Note that the LVA tag is one variant of the ssrA tag which is well-known as a protein degradation sequence in E. coli. Prhl(LR) ( BBa_K1529310 ) activity was strong but unexpectedly reacted with C12 (crosstalk) (Fig.3-2-3-3-4). Since crosstalk with C12 causes a serious problem in the circuits, this promoter could not be used.
Fig. 3-2-3-3-1 Effect of RhlR by tagging LVA
We measured Relative Fluorescence Units (RFU) of GFP / Turbidity with  Prhl （LR） and  LVA tag by using the plasimid-I-A,  Prhl(LR) and without LVA tag by using the plasmid-I-B, Prhl(RL) and LVA tag by using the plasmid-I-C, and Prhl(RL) and without LVA tag by using the plasmid-I-D. The vertical scale is calculated by subtracting the value in Negative control (plasmid-I-E)from the values that we got from each sample.
The colonies of transformants with a rhlR ( BBa_C0171 ) plasmid looked rough and growth was slow (Fig.3-2-3-3-2-left), while the colonies of transformants with a rhlR - LVA ( BBa_C0071 ) plasmid looked smooth and the growth was normal(Fig.3-2-3-3-2-right). However, the reason for this result is unclear.
Fig. 3-2-3-3-2 The colonies of transformants with a rhlR (left) or a rhlR - LVA right)
Ⅱ. Improvement of the wild type Prhl
By introducing various single point mutations into wild type Prhl ( BBa_R0071 ) by PCR, a library of Prhl mutants was generated. The target site for mutation was the -10 region of Prhl, because the sequence, TAAAAA, is far from the consensus sequence of E. coli -10 region, TATAAT. Then, stronger Prhl was screened from the library, and two Prhl mutants with stronger activity were obtained from 198 mutants tested (Fig.3-2-3-3-3).
Fig. 3-2-3-3-3 RFU of GFP / Turbidity of imoroved Prhl mutants
We measured RFU of GFP / Turbidity with  Prhl （NM) by using the plasimid-II-B,  Prhl(M) by using the plasmid-II-C, and Prhl(WT) using the plasmid-II-D. The vertical scale is calculated by subtracting the value in Negative control ( plasmid-II-E) from the values that we got from each sample.
The sequences of these two mutants are shown below. The small characters indicate the scar sequence and a single point mutation is colored with red.
Wild type Prhl ( BBa_R0071 )
TCCTGTGAAATCTGGCAGTTACCGTTAGCTTTCGAATTGGCTAAAAAGTGTTC tactagagAAAGAGGAGAAA
Prhl(NM) ( BBa_K1949060 )
TCCTGTGAAATCTGGCAGTTACCGTTAGCTTTCGAATTGGCTATAAAGTGTTC tactagagAAAGAGGAGAAA
Prhl(M)
TCCTGTGAAATCTGGCAGTTACCGTTAGCTTTCGAATTGGCTATAAAGTGTTC tactagtaAAAGAGGAGAAA
Ⅲ. Comparison of the improved Prhl by iGEM 2014 team Tokyo_Tech to our original improved Prhl
The activity of Prhl(NM) was compared to Prhl(LR) by observing expression of the reporter gene, (gap) (Fig.3-2-3-3-4). The reaction activity of Prhl(NM) to C4 was stronger than that of Prhl(LR), and Prhl(NM) did not react with C12 at all.
Fig. 3-2-3-3-4 Comparison of Prhl(NM) by Prhl(LR)
We measured RFU of GFP / Turbidity with  Prhl （LR) by using the plasimid-III-A and Prhl(NM) by using the plasmid-III-B. The vertical scale is calculated by subtracting the value in Negative control (plasmid-III-C) from the values that we got from each sample.
4. Discussion
Judging from the result of the experiment I, Prhl(RL) activity was unexpectedly weak and significant expression was obtained only when extremely high C4 concentration, which is employed by iGEM 2014 team Tokyo_Tech. Such feature of Prhl(RL) is inappropriate for our experiments, because such high level secretion of C4 is not desired from the Snow White-coli.
In the absence of C4, RhlR is known to repress Prhl and the repression is released when RhlR with bind C4(1). In our experiments, RhlR without LVA tag seemed to repress Prhl very tightly, and as a consequent, GFP could not be expressed very much under low C4 concentrations (Fig.3-2-3-3-1). On the other hand, under high C4 concentrations, it seemed that a lot of RhlR bound with C4, and the repression was successfully released and GFP was expressed greatly(Data not shown). RhlR with LVA tag tends to be degraded easily and its amount in cells should be low at steady-state level. Therefore, in our experiments, the repression by RhlR without LVA tag was not strong and GFP was expressed in high amounts.
Interestingly, RhlR without the LVA tag repressed growth of E. coli. The reason for this result is unclear, but high RhlR protein level might repress or activate E. coli gene unexpectedly.
Prhl activity became stronger when the sequence of -10 region was changed from “TAAAAA” to "TATAAA" which is known as that of the strongest promoters in E. coli. Prhl(M) had accidental two mutations in the scar sequence, and it is not suitable for a new part of BioBrick.
In our project, the combination of rhlR with LVA tag and Prhl(LR) might work well in our final genetic circuits. However, Prhl(LR) has lux box, and thus, the promoter also reacts with C12 (crosstalk) unintentionally. In our final circuits, we have to use C12 for Plux activation, and thus, usage of Prhl(LR) is inappropriate. Then, we here obtained the Prhl(NM) of which activity was stronger than Prhl(LR). When compared to Prhl(LR), Prhl(NM) showed little crosstalk with C12 and the signal-noise (SN) ratio was higher. The SN ratio of Prhl(LR) to C4 was small, possibly reflecting that leaky expression from Prhl(LR) was high.      As a conclusion, the combination of rhlR with LVA tag and Prhl(NM) is best for our final genetic circuits.
5. Materials and methods
5-1. Construction
-Strain
All the plasmids were prepared in XL1-Blue strain.
-Plasmids
Ⅰ. Improved Prhl by iGEM 2014 team Tokyo_Tech and rhlR assay
A. Pcon ‐ rbs ‐ rhlR ‐ LVA( BBa_C0071 ) (pSB6A1), Prhl(LR)( BBa_K1529310 ) ‐ rbs ‐ gfp(pSB3K3) (Fig. 3-2-3-5-1)
B. Pcon ‐ rbs ‐ rhlR( BBa_C0171 ) (pSB6A1), Prhl(LR) ‐ rbs ‐ gfp (pSB3K3) (Fig. 3-2-3-5-2)
C. Pcon ‐ rbs ‐ rhlR ‐ LVA (pSB6A1), Prhl(RL)( BBa_K1529300 ) ‐ rbs ‐ gfp (pSB3K3) (Fig. 3-2-3-5-3)
D. Pcon ‐ rbs ‐ rhlR (pSB6A1), Prhl(RL) ‐ rbs ‐ gfp (pSB3K3) (Fig. 3-2-3-5-4)
E. pSB6A1, pSB3K3 …Nagative control (Fig. 3-2-3-5-5)
Ⅱ. Improvment the wild type Prhl
A. Prhl( BBa_R0071 ) ‐ rbs ‐ gfp (pSB3K3) (Fig. 3-2-3-5-6)
B. Pcon ‐ rbs ‐ rhlR ‐ LVA (pSB6A1), Prhl(NM) ‐ rbs ‐ gfp( BBa_ K1949060 ) (pSB3K3) (Fig. 3-2-3-5-7)
C. Pcon ‐ rbs ‐ rhlR ‐ LVA (pSB6A1), Prhl(M) ‐ rbs ‐ gfp (pSB3K3) (Fig. 3-2-3-5-8)
D. Pcon ‐ rbs ‐ rhlR ‐ LVA (pSB6A1), Prhl(WT) ‐ rbs ‐ gfp (pSB3K3) (Fig. 3-2-3-5-9)
E. pSB6A1, pSB3K3 …Nagative control (Fig. 3-2-3-5-5)
Ⅲ. Comparison the improved Prhl by iGEM 2014 team Tokyo_Tech to our original improved Prhl
A. Pcon ‐ rbs ‐ rhlR ‐ LVA (pSB6A1), Prhl(LR) ‐ rbs ‐ gfp (pSB3K3) (Fig. 3-2-3-5-1)
B. Pcon ‐ rbs ‐ rhlR ‐ LVA (pSB6A1), Prhl(NM) ‐ rbs ‐ gfp (pSB3K3) (Fig. 3-2-3-5-7)
C. pSB6A1, pSB3K3…Negative control (Fig. 3-2-3-5-5)
-Medium
LB medium AK:
LB medium containing ampicillin (50 microg / mL) and kanamycin (50 microg / mL)
LB medium K:
LB medium containing kanamycin (50 microg / mL)
5-2. Assay protocol
The following experiments are performed at 37℃ unless otherwise stated.
I. Improved Prhl by iGEM 2014 team Tokyo_Tech and rhlR assay
1) Prepare overnight culture for each sample in 3mL LB medium AK with vigorous shaking.
2) Dilute the overnight cultures to 1 / 60 in fresh LB medium AK (1.2 mL).
3) Incubate the fresh cultures for 1 h with vigorous shaking.
4) Add C4 or DMSO to each 500 microL sample at the final concentration 10 microM or 100 microM.
5) Incubate the samples for 4 h with vigorous shaking.
6) Add 100 microL of the samples to each well of a plate reader.
7) Measure RFU of GFP at 490nm as an excitation wavelength, 525nm as an emission wavelength.
8) Measure the turbidity at 600 nm.
II. Improvement the wild type Prhl ( BBa_R0071 )
1) Introduce a point mutation to -10 region of Prhl( BBa_R0071 ) ‐ rbs ‐ gfp (pSB3K3) by inverse PCR using wild type Prhl( BBa_R0071 ) ‐ rbs ‐ gfp (pSB3K3) as a template and divergent primers. The sequences of the six forward premiers and of one reverse primer are shown below.
WT    5'-tcctgtgaaatctggcagttaccgttagctttcgaatt ggctaaaaagtgttctactagtagcg-3'
F-Mu1  5'-tcctgtgaaatctggcagttaccgttagctttcgaatt ggntaaaaagtgttctactagtagcg-3'
F-Mu2  5'-tcctgtgaaatctggcagttaccgttagctttcgaatt ggcnaaaaagtgttctactagtagcg-3'
F-Mu3  5'-tcctgtgaaatctggcagttaccgttagctttcgaatt ggctnaaaagtgttctactagtagcg-3'
F-Mu4  5'-tcctgtgaaatctggcagttaccgttagctttcgaatt ggctanaaagtgttctactagtagcg-3'
F-Mu5  5'-tcctgtgaaatctggcagttaccgttagctttcgaatt ggctaanaagtgttctactagtagcg-3'
F-Mu6  5'-tcctgtgaaatctggcagttaccgttagctttcgaatt ggctaaanagtgttctactagtagcg-3'
R       5'-aattcgaaagctaacggtaactgcca-3' (5' phosphorylated)
2) Prepare strains containing the pMu1~6 plasmids containing mutated Prhl ‐ rbs ‐ gfp (pSB3K3).
3) Streak the transformants on an agar plate containing kanamycin (50 microg / mL) and incubate overnight.
4) Pore the LB medium K at the extent of covering the surface of the agar, and collect the colonies with a spreader. Then, incubate them in 5 mL LB medium K for 16 h with vigorous shaking.
5) Extract the plasmid, and introduce Prhl(mut) ‐ rbs ‐ gfp (pSB3K3) and Pcon ‐ rbs ‐ rhlR -‐LVA to XL‐1 Blue.
6) Pore 200 microL LB medium AK containing C4 (6 microM) in one well of a 96-well plate and inoculate one colony to the well. Incubate the plate for 16 h.
7) Transfer 100 microL of the culture from the above samples to another plate that is suitable for fluorescence measurement. Measure the GFP RFU of GFP / Turbidity at 490 nm as an excitation wavelength and 525 nm as an emission wavelength as well as the turbidity at 600 nm.
8) Select mutants that have higher SN ratio than wild type Prhl ( BBa_R0071 ) and grow them in 3 mL LB medium AK overnight with vigorous shaking.
9) Dilute the overnight cultures to 1 / 60 in fresh LB medium AK (1.2 mL).
10) Incubate the fresh cultures for 1 h with vigorous shaking.
11) Add C4, C12 or DMSO to each 500 microL sample at the final concentration 10 microM.
12) Incubate the samples for 4 h with vigorous shaking.
13) Add 100 microL of the samples to each well of a plate reader.
14) Measure RFU of GFP at 490nm as an excitation wavelength, 525nm as an emission wavelength.
15) Measure the turbidity at 600 nm.
III. Comparison the improved Prhl promoter by iGEM 2014 team Tokyo_Tech to our original improved Prhl promoter
1) Prepare overnight cultures for each sample in 3mL LB medium AK with vigorous shaking.
2) Dilute the overnight cultures to 1 / 60 in fresh LB medium AK (1.2 mL).
3) Incubate the fresh cultures for 1 h with vigorous shaking.
4) Add C4, C12 or DMSO to each 500 microL sample at the final concentration 10 microM.
5) Incubate the samples for 4 h with vigorous shaking.
6) Add 100 microL of the samples to each well of a plate reader.
7) Measure RFU of GFP at 490nm as an excitation wavelength, 525nm as an emission wavelength.
8) Measure the turbidity at 600 nm.
6. Reference
(1) Gerardo Medina et al. (2003) Mechanism of Pseudomonas aeruginosa RhlR Transcriptional Regulation of the rhlAB Promoter. BACTERIOLOGY 185: 5976-5983
(2) John S. Chuang et al. (2009) Simpson’s Paradox in a Synthetic Microbial System.
Jump to AmiE assay page.
Jump to Home
