<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_Tokyo_Tech_Model skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><DIV id="top_title"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:Tokyo Tech/Model</SPAN></H1></DIV><DIV id="HQ_page"><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><DIV id="main_contents"><DIV id="page_header" class="container container_top"><H1 align="center">Model</H1></DIV><DIV id="contents" class="container"><DIV id="contents_header" class="container_header"><H2><SPAN>Contents</SPAN></H2></DIV><DIV id="contents_contents" class="container_contents"><DIV id="contents_menu"><H3 class="link"><A href="#overview">1. Overview</A></H3><H3 class="link"><A href="#story_simulation">2. Story simulation</A></H3><H3 class="link"><A href="#mathematical_model"><FONT size="2.7">   2-1. Mathematical model</FONT></A></H3><H3 class="link"><A href="#results"><FONT size="2.7">   2-2. Results</FONT></A></H3><H3 class="link"><A href="#fitting">3. Fitting</A></H3><H3 class="link"><A href="#population"><FONT size="2.7">   3-1. Population growth</FONT></A></H3><H3 class="link"><A href="#toxin"><FONT size="2.7">   3-2. Toxin-Antitoxin system</FONT></A></H3><H3 class="link"><A href="#promoters"><FONT size="2.7">   3-3. Promoters</FONT></A></H3><H3 class="link"><A href="#more"><FONT size="2.7">   3-4. More realistic model with mRNA</FONT></A></H3><H3 class="link"><A href="#analysis">4. Analysis</A></H3><H3 class="link"><A href="#prince_coli"><FONT size="2.7">   4-1. The Prince <SPAN style="font-style : italic">coli</SPAN> should be put in during the process</FONT></A></H3><H3 class="link"><A href="#prhl"><FONT size="2.7">   4-2. Prhl should be changed</FONT></A></H3><H3 class="link"><A href="#requirements"><FONT size="2.7">   4-3. Requirements</FONT></A></H3><H3 class="link"><A href="#production_ahl"><FONT size="2.7">   4-4. Production rate of C4HSL and 3OC12HSL by RhlI and LasI</FONT></A></H3><H3 class="link"><A href="#translation"><FONT size="2.7">   4-5. Translation rate of protein</FONT></A></H3><H3 class="link"><A href="#decomposition"><FONT size="2.7">   4-6. Decomposition rate of 3OC12HSL by AmiE</FONT></A></H3><H3 class="link"><A href="#degradation_amie"><FONT size="2.7">   4-6. Degradation rate of AmiE</FONT></A></H3><H3 class="link"><A href="#degradation_protein"><FONT size="2.7">   4-7. Degradation rate of RFP and GFP</FONT></A></H3><H3 class="link"><A href="#software">5. Software</A></H3><H3 class="link"><A href="#abstract"><FONT size="2.7">   5-1. Abstract</FONT></A></H3><H3 class="link"><A href="#key_achievements"><FONT size="2.7">   5-2. Key achievements</FONT></A></H3><H3 class="link"><A href="#work_flow"><FONT size="2.7">   5-3. Work flow</FONT></A></H3><H3 class="link"><A href="#demo"><FONT size="2.7">   5-4. Demonstration</FONT></A></H3><H3 class="link"><A href="#download"><FONT size="2.7">   5-5. Download</FONT></A></H3></DIV></DIV></DIV><DIV id="overview" class="container"><DIV id="overview_header" class="container_header"><H2><SPAN>1. Overview</SPAN></H2></DIV><DIV id="overview_contents" class="container_contents"><P class="normal_text">To recreate the story of ”the Snow White”, we have designed a cell-cell communication system with improved or characterized parts and collected data from comprehensive experiments. Furthermore, we constructed <A href="https://2016.igem.org/Team:Tokyo_Tech/Modeling_Details">the mathematical model</A> to simulate the behavior of the whole system and to confirm the feasibility of our story. This simulation successfully contributed to give <A href="https://2016.igem.org/Team:Tokyo_Tech/Model#prince_coli">the suggestions</A> to wet lab experiments. In addition, in order to help us utilize our Toxin-Antitoxin (TA) system, we developed a new software in Java for adjusting the number of ACA sequences, which MazF dimer recognizes and cleaves in mRNAs.</P></DIV></DIV><DIV id="story_simulation" class="container"><DIV id="story_simulation_header" class="container_header"><H2><SPAN>2. Story simulation</SPAN></H2></DIV><DIV id="story_simulation_contents" class="container_contents"><DIV id="mathematical_model"><DIV id="mathematical_model_header"><H3><SPAN>2-1. Mathematical model</SPAN></H3></DIV><P class="normal_text">In order to simulate our gene circuits, we developed an ordinary differential equation model.</P><P style="text-align:center;" class="normal_text">[<A href="https://2016.igem.org/Team:Tokyo_Tech/Modeling_Details">Model development</A>]</P><DIV onclick="obj=document.getElementById('open1').style; obj.display=(obj.display=='none')?'block':'none';"><A style="cursor:pointer;">▼ Differential equations</A></DIV><DIV id="open1" style="display:none;clear:both;"><H3>Snow White</H3>
							\begin{equation}
							\frac{d[mRNA_{RFP}]}{dt} = k - d[mRNA_{RFP}] - F_{DiMazF}(1-(1-f)^{f_{mRNA_{RFP}}})[mRNA_{RFP}][DiMazF]
							\end{equation}
							\begin{equation}
							\frac{d[mRNA_{RhlI}]}{dt} =  leak_{P_{lux}} + \frac{\kappa_{Lux}[C12]^{n_{Lux}}}{K_{mLux}^{n_{Lux}} + [C12]^{n_{Lux}}} - d[mRNA_{RhlI}] - F_{DiMazF}f[mRNA_{RhlI}][DiMazF]
							\end{equation}
							\begin{equation}
							\frac{d[RFP]}{dt} = \alpha [mRNA_{RFP}] - d_{RFP}[RFP]
							\end{equation}
							\begin{equation}
							\frac{d[RhlI]}{dt} = \alpha [mRNA_{RhlI}] - d_{RhlI}[RhlI]
							\end{equation}
							\begin{equation}
							\frac{d[C4]}{dt} = p_{C4}[RhlI]P_{Snow White} - d_{C4}[C4]
							\end{equation}
							\begin{eqnarray}
							\frac{d[mRNA_{MazF}]}{dt} &amp;=&amp;leak_{P_{lux}} + \frac{\kappa_{Lux}[C12]^{n_{Lux}}}{K_{mLux}^{n_{Lux}}+ [C12]^{n_{Lux}}} - d[mRNA_{MazF}]  \nonumber \\
							&amp;&amp;    \ \ \ \ \ \ \ \ \ \ \ \ -F_{DiMazF}(1-(1-f)^{f_{mRNA_{MazF}}})[mRNA_{MazF}][DiMazF]
							\end{eqnarray}
							\begin{equation}
							\frac{d[mRNA_{MazE}]}{dt} = k - d[mRNA_{MazE}] - F_{DiMazF}(1-(1-f)^{f_{mRNA_{MazE}}})[mRNA_{MazE}][DiMazF]
							\end{equation}
							\begin{equation}
							\frac{d[MazF]}{dt} = \alpha [mRNA_{MazF}] - 2k_{Di_{MazF}}[MazF] + 2k_{-Di_{MazF}}[DiMazF] - d_{MazF}[MazF]
							\end{equation}
							\begin{eqnarray}
							\frac{d[DiMazF]}{dt} &amp;=&amp; k_{Di_{MazF}}[MazF] - k_{-Di_{MazF}}[DiMazF] - 2k_{Hexa}[DiMazE][DiMazF]^2 \nonumber \\
							 &amp;&amp; \ \ \ \ \ \ \ \ \ \ \ \ \ \ \ \ \ \ \ \ \ +2k_{-Hexa}[MazHexamer] - d_{DiMazF}[DiMazF]
							\end{eqnarray}
							\begin{equation}
							\frac{d[MazE]}{dt} = \alpha [mRNA_{MazE}] - 2k_{Di_{MazE}}[MazE] + 2k_{-Di_{MazE}}[DiMazE] - d_{MazE}[MazE]
							\end{equation}
							\begin{eqnarray}
							\frac{d[DiMazE]}{dt} &amp;=&amp; k_{Di_{MazE}}[MazE] - k_{-Di_{MazE}}[DiMazE] - k_{Hexa}[DiMazE][DiMazF]^2 \nonumber \\
							 &amp;&amp;  \ \ \ \ \ \ \ \ \ + k_{-Hexa}[MazHexamer] - d_{DiMazE}[DiMazE]
							\end{eqnarray}
							\begin{equation}
							\frac{d[MazHexa]}{dt} = k_{Hexa}[DiMazE][DiMazF]^2 - k_{-Hexa}[MazHexa] - d_{Hexa}[MazHexa]
							\end{equation}
							\begin{equation}
							\frac{dP_{Snow White}}{dt} = g \frac{E_{DiMazF}}{E_{DiMazF}+[DiMazF]}\left(1- \frac{P_{Snow White}+P_{Queen}+P_{Prince}}{P_{max}} \right) P_{Snow White}
							\end{equation}
							<H3>Queen</H3>
							\begin{equation}
							\frac{d[mRNA_{GFP}]}{dt} = k - d[mRNA_{GFP}] - F_{DiMazF}(1-(1-f)^{f_{mRNA_{GFP}}})[mRNA_{GFP}][DiMazF]
							\end{equation}
							\begin{eqnarray}
							\frac{d[mRNA_{LasI}]}{dt} &amp;=&amp;  leak_{P_{rhl}} + \frac{\kappa_{Rhl}[C4]^{n_{Rhl}}}{K_{mRhl}^{n_{Rhl}} + [C4]^{n_{Rhl}}}  \nonumber \\
							&amp;&amp; \ \ \ \ - d[mRNA_{LasI}] - F_{DiMazF}(1-(1-f)^{f_{mRNA_{LasI}}})[mRNA_{LasI}][DiMazF]
							\end{eqnarray}
							\begin{equation}
							\frac{d[GFP]}{dt} = \alpha [mRNA_{GFP}] - d_{GFP}[GFP]
							\end{equation}
							\begin{equation}
							\frac{d[LasI]}{dt} = \alpha [mRNA_{LasI}] - d_{LasI}[LasI]
							\end{equation}
							\begin{equation}
							\frac{d[C12]}{dt} = p_{C12}[LasI]P_{Queen} - d_{C12}[C12] - D[C12][AmiE]
							\end{equation}
							\begin{eqnarray}
							\frac{d[mRNA_{MazF}]}{dt} &amp;=&amp;  leak_{P_{lux}} + \frac{\kappa_{Rhl}[C4]^{n_{Rhl}}}{K_{mRhl}^{n_{Rhl}} + [C4]^{n_{Rhl}}} \nonumber \\
							&amp;&amp; - d[mRNA_{MazF}]  - F_{DiMazF}(1-(1-f)^{f_{mRNA_{MazF}}})[mRNA_{MazF}][DiMazF]
							\end{eqnarray}
							\begin{equation}
							\frac{d[mRNA_{MazE}]}{dt} = k - d[mRNA_{MazE}] - F_{DiMazF}(1-(1-f)^{f_{mRNA_{MazE}}})[mRNA_{MazE}][DiMazF]
							\end{equation}
							\begin{equation}
							\frac{d[MazF]}{dt} = \alpha [mRNA_{MazF}] - 2k_{Di_{MazF}}[MazF] + 2k_{-Di_{MazF}}[DiMazF] - d_{MazF}[MazF]
							\end{equation}
							\begin{eqnarray}
							\frac{d[DiMazF]}{dt} &amp;=&amp; k_{Di_{MazF}}[MazF] - k_{-Di_{MazF}}[DiMazF] - 2k_{Hexa}[DiMazE][DiMazF]^2 \nonumber \\
							&amp;&amp; + 2k_{-Hexa}[MazHexamer] - d_{DiMazF}[DiMazF]
							\end{eqnarray}
							\begin{equation}
							\frac{d[MazE]}{dt} = \alpha [mRNA_{MazE}] - 2k_{Di_{MazE}}[MazE] + 2k_{-Di_{MazE}}[DiMazE] - d_{MazE}[MazE]
							\end{equation}
							\begin{eqnarray}
							\frac{d[DiMazE]}{dt} &amp;=&amp; k_{Di_{MazE}}[MazE] - k_{-Di_{MazE}}[DiMazE] - k_{Hexa}[DiMazE][DiMazF]^2 \nonumber \\
							&amp;&amp; + k_{-Hexa}[MazHexamer] - d_{DiMazE}[DiMazE]
							\end{eqnarray}
							\begin{equation}
							\frac{d[MazHexa]}{dt} = k_{Hexa}[DiMazE][DiMazF]^2 - k_{-Hexa}[MazHexa] - d_{Hexa}[MazHexa]
							\end{equation}
							\begin{equation}
							\frac{dP_{Queen}}{dt} = g \frac{E_{DiMazF}}{E_{DiMazF}+[DiMazF]}\left(1- \frac{P_{Snow White}+P_{Queen}+P_{Prince}}{P_{max}}\right) P_{Queen}\\
							\end{equation}
							<H3>Prince</H3>
							\begin{equation}
							\frac{d[mRNA_{AmiE}]}{dt} = leak_{P_{lux}} + \frac{\kappa_{Lux}[C12]^n}{K_{mLux}^n + [C12]^n} - d[mRNA_{AmiE}]
							\end{equation}
							\begin{equation}
							\frac{d[AmiE]}{dt} = \alpha [mRNA_{AmiE}]P_{Prince} - d_{AmiE}[AmiE]
							\end{equation}
							\begin{equation}
							\frac{dP_{Prince}}{dt} = g\left(1- \frac{P_{Snow White}+P_{Queen}+P_{Prince}}{P_{max}}\right) P_{Prince}
							\end{equation}
</DIV><DIV onclick="obj=document.getElementById('open2').style; obj.display=(obj.display=='none')?'block':'none';"><A style="cursor:pointer;">▼Parameters</A></DIV><DIV id="open2" style="display:none;clear:both;"><TABLE border="1" style="margin: auto;"><TBODY><TR><TD>Parameter </TD><TD> Description </TD></TR><TR><TD>$$g$$ </TD><TD> Growth rate of each cells</TD></TR><TR><TD>$$P_{max}$$ </TD><TD> Carrying capacity </TD></TR><TR><TD>$$E_{DiMazF}$$ </TD><TD> Effect of MazF dimer on growth rate</TD></TR><TR><TD>$$k$$ </TD><TD> Transcription rate of mRNA under \(P_{tet}\) </TD></TR><TR><TD>$$leak_{P_{lux}}$$ </TD><TD> Leakage of \(P_{lux}\) </TD></TR><TR><TD>$$leak_{P_{rhl}}$$ </TD><TD> Leakage of \(P_{rhl}\) </TD></TR><TR><TD>$$\kappa_{Lux}$$ </TD><TD> Maximum transcription rate of mRNA under \(P_{lux}\)</TD></TR><TR><TD>$$\kappa_{Rhl}$$ </TD><TD> Maximum transcription rate of mRNA under \(P_{rhl}\) </TD></TR><TR><TD>$$n_{Lux}$$ </TD><TD> Hill coefficient for \(P_{lux}\)</TD></TR><TR><TD>$$n_{Rhl}$$ </TD><TD> Hill coefficient for \(P_{rhl}\)</TD></TR><TR><TD>$$K_{mLux}$$ </TD><TD> Lumped paremeter for the Lux System</TD></TR><TR><TD>$$K_{mRhl}$$ </TD><TD> Lumped paremeter for the Rhl System</TD></TR><TR><TD>$$F_{DiMazF}$$ </TD><TD> Cutting rate at ACA sequences on mRNA by MazF dimer</TD></TR><TR><TD>$$f$$ </TD><TD> The probability of distinction of ACA sequencess in each mRNA</TD></TR><TR><TD>$$f_{mRNA_{RFP}}$$ </TD><TD> The number of ACA sequences in \(mRNA_{RFP}\)</TD></TR><TR><TD>$$f_{mRNA_{GFP}}$$ </TD><TD> The number of ACA sequences in \(mRNA_{GFP}\)</TD></TR><TR><TD>$$f_{mRNA_{RhlI}}$$ </TD><TD> The number of ACA sequences in \(mRNA_{RhlI}\)　</TD></TR><TR><TD>$$f_{mRNA_{LasI}}$$ </TD><TD> The number of ACA sequences in \(mRNA_{LasI}\)</TD></TR><TR><TD>$$f_{mRNA_{MazF}}$$ </TD><TD> The number of ACA sequences in \(mRNA_{MazF}\)　</TD></TR><TR><TD>$$f_{mRNA_{MazE}}$$ </TD><TD> The number of ACA sequences in \(mRNA_{MazE}\)　</TD></TR><TR><TD>$$\alpha$$ </TD><TD> Translation rate of Protein </TD></TR><TR><TD>$$k_{Di_{MazF}}$$ </TD><TD> Formation rate of MazF dimer </TD></TR><TR><TD>$$k_{-Di_{MazF}}$$ </TD><TD> Dissociation rate of MazF dimer </TD></TR><TR><TD>$$k_{Di_{MazE}}$$ </TD><TD> Formation rate of MazE dimer </TD></TR><TR><TD>$$k_{-Di_{MazE}}$$ </TD><TD> Dissociation rate of MazE dimer </TD></TR><TR><TD>$$k_{Hexa}$$ </TD><TD> Formation rate of Maz hexamer </TD></TR><TR><TD>$$k_{-Hexa}$$ </TD><TD> Dissociation rate of Maz hexamer</TD></TR><TR><TD>$$p_{C4}$$ </TD><TD> Production rate of C4HSL by RhlI</TD></TR><TR><TD>$$p_{C12}$$ </TD><TD> Production rate of 3OC12HSL by LuxI </TD></TR><TR><TD>$$D$$ </TD><TD> Decomposition rate of 3OC12HSL by AmiE </TD></TR><TR><TD>$$d$$ </TD><TD> Degradation rate of mRNA </TD></TR><TR><TD>$$d_{RFP}$$ </TD><TD> Degradation rate of RFP</TD></TR><TR><TD>$$d_{GFP}$$ </TD><TD> Degradation rate of GFP </TD></TR><TR><TD>$$d_{RhlI}$$ </TD><TD> Degradation rate of RhlI</TD></TR><TR><TD>$$d_{LasI}$$ </TD><TD> Degradation rate of LasI</TD></TR><TR><TD>$$d_{MazF}$$ </TD><TD> Degradation rate of MazF</TD></TR><TR><TD>$$d_{DiMazF}$$ </TD><TD> Degradation rate of MazF dimer</TD></TR><TR><TD>$$d_{MazE}$$ </TD><TD> Degradation rate of MazE </TD></TR><TR><TD>$$d_{DiMazE}$$ </TD><TD> Degradation rate of MazE dimer </TD></TR><TR><TD>$$d_{Hexa}$$ </TD><TD> Degradation rate of Maz Hexamer </TD></TR><TR><TD>$$d_{C4}$$ </TD><TD> Degradation rate of C4HSL </TD></TR><TR><TD>$$d_{C12}$$ </TD><TD> Degradation rate of 3OC12HSL </TD></TR><TR><TD>$$d_{AmiE}$$ </TD><TD> Degradation rate of AmiE </TD></TR></TBODY></TABLE></DIV></DIV><DIV id="results_contents" class="container_contents"><DIV id="results"><DIV id="results_header"><H3><SPAN>2-2. Results</SPAN></H3></DIV><P class="normal_text">We obtained and confirmed the desirable behavior of the whole system by modifying and improving parts. As described below, our simulation showed an appropriate transition of concentration of RFP and GFP for the story.</P><DIV align="center"><P class="caption" style="font-size: 16px; text-align: center;"><SPAN style="font-weight: bold;">Fig.5-2.2. Time-dependent change of the concentrations of fluorescent proteins</SPAN></P></DIV><P class="normal_text"> In the blue area of Fig.5-2-2, the concentration of fluorescent proteins start to increase. The concentration of RFP of Snow White <SPAN style="font-style : italic">coli</SPAN> exceeds  that of GFP of the Queen <SPAN style="font-style : italic">coli</SPAN>. 
					It is as if Snow White got fairer more and more.

					In the pink area of Fig.5-2-2, the concentration of C12 increase thanks to the appearance of C4. As a result, the MazF inside Snow White <SPAN style="font-style : italic">coli</SPAN> and the Queen <SPAN style="font-style : italic">coli</SPAN> start to suppress the increment of fluorescet proteins. 
					It is as if the Queen, influenced by the Mirror's answer, transforming into a Witch in order to give Snow White a poisoned apple.

					In the green area of Fig.5-2-2, the concentration of C12 more increases and the MazF inside Snow White <SPAN style="font-style : italic">coli</SPAN> more suppress the increment of GFP. So the concentration GFP exceeds that of RFP. 
					It looks as if Snow White bit the apple, sinking into unconsciousness soon.

					In the yellow area of Fig.5-2-2, the AmiE synthesized by the introduced Prince <SPAN style="font-style : italic">coli</SPAN> decomposes C12 so the MazF inside Snow White <SPAN style="font-style : italic">coli</SPAN> diminishes and the concentration of GFP resumes. 
					It looks as if the Prince lifted Snow White and she opened her eyes.</P></DIV></DIV></DIV></DIV><DIV id="fitting" class="container"><DIV id="fitting_header" class="container_header"><H2><SPAN>3. Fitting</SPAN></H2></DIV><DIV id="fitting_contents" class="container_contents"><DIV id="population"><DIV id="population_header"><H3><SPAN>3-1. Population growth</SPAN></H3></DIV><DIV id="population_contents"><P class="normal_text">First, we tried to model the growth curve of the system. When the number of <SPAN style="font-style : italic">E. coli</SPAN> approaches a certain value, the growth will stop. We defined this value in the culture as P<SUB>max</SUB>. Then the population growth equation for our system is described as follows:
					$$ \frac{dP}{dt} = g\left(1 - \frac{P}{P_{max}}\right)P$$
					where g is the population growth rate. 
					This equation can be analytically solved as:
					$$ P = \frac{P_{0} P_{max} e^{gt}}{P_{max} - P_{0} + P_{0} e^{gt}}$$
					where P <SUB> 0</SUB> is the population at t = 0. We used this equation to fit the experimental data.
					</P><DIV align="center"><P class="caption" style="font-size: 16px; text-align: center;"><SPAN style="font-weight: bold;">Fig.5-3-1. Modeled growth curve of <SPAN style="font-style : italic">E. coli</SPAN> fitted to experiment data</SPAN></P></DIV><P class="normal_text">Using the experimental data from the <A href="https://2016.igem.org/Team:Tokyo_Tech/Toxin_Assay/mazEF_System_Assay">Toxin assay</A> for this fitting, we estimated the following parameters:
					</P><P class="normal_text" style="text-align: center;">g = 0.0123
					and
					P<SUB>max</SUB> =3.3</P><P class="normal_text">respectively.
					These parameters can be used for Snow White <SPAN style="font-style : italic">coli</SPAN>, the Queen <SPAN style="font-style : italic">coli</SPAN> and the Prince <SPAN style="font-style : italic">coli</SPAN> in the same way.
					</P></DIV></DIV><DIV id="toxin"><DIV id="toxin_header"><H3><SPAN>3-2. Toxin-Antitoxin system</SPAN></H3></DIV><DIV id="toxin_contents"><DIV onclick="obj=document.getElementById('open3').style; obj.display=(obj.display=='none')?'block':'none';"><A style="cursor:pointer;">▼Read more</A></DIV><DIV id="open3" style="display:none;clear:both;"><P class="normal_text">Since there are only few researches that actually discussed on the kinetics of the TA system, we estimated the parameters for our TA system. We got the experimental data from <A href="https://2016.igem.org/Team:Tokyo_Tech/Toxin_Assay/mazEF_System_Assay">Toxin assay</A> for this fitting. The differential equations representing the TA system are described as follows:
						\begin{equation}
					 \frac{d[MazF]}{dt} = \alpha [mRNA_{MazF}] - 2k_{Di_{MazF}}[MazF] + 2k_{-Di_{MazF}}[DiMazF] - d_{MazF}[MazF] \nonumber
					\end{equation}
					
					 \begin{eqnarray}
					 \frac{d[DiMazF]}{dt} &amp;=&amp; k_{Di_{MazF}}[MazF] - k_{-Di_{MazF}}[DiMazF] - 2k_{Hexa}[DiMazE][DiMazF]^2 \nonumber \\
					 &amp;&amp; + 2k_{-Hexa}[MazHexamer] - d_{DiMazF}[DiMazF]
					\end{eqnarray}
					
					\begin{equation}
					 \frac{d[MazE]}{dt} = \alpha [mRNA_{MazE}] - 2k_{Di_{MazE}}[MazE] + 2k_{-Di_{MazE}}[DiMazE] - d_{MazE}[MazE] \nonumber
					 \end{equation}
					 
					 \begin{eqnarray}
					 \frac{d[DiMazE]}{dt} &amp;=&amp; k_{Di_{MazE}}[MazE] - k_{-Di_{MazE}}[DiMazE] - k_{Hexa}[DiMazE][DiMazF]^2  \nonumber \\
					 &amp;&amp; + k_{-Hexa}[MazHexamer] - d_{DiMazE}[DiMazE]
					 \end{eqnarray}
					 
					 \begin{equation}
					 \frac{d[MazHexa]}{dt} = k_{Hexa}[DiMazE][DiMazF]^2 - k_{-Hexa}[MazHexa] - d_{Hexa}[MazHexa]  \nonumber
					 \end{equation}
					 
					We used genetic algorithms to fit this data. We used ODs at 7 points and fit them to experimental ODs.
					</P><DIV align="center"><P class="caption" style="font-size: 16px; text-align: center;"><SPAN style="font-weight: bold;">Fig.5-3-2. Fitted growth curve of <SPAN style="font-style : italic">E.coli</SPAN> to the experimental data of TA system</SPAN></P></DIV><P class="normal_text">As a result, we obtained the following parameters.</P><TABLE border="1" style="margin: auto;"><TBODY><TR><TD>Parameter </TD><TD> Value </TD></TR><TR><TD> k<SUB>DiMazF</SUB></TD><TD> 6.82 nM<SUP>-1</SUP> min<SUP>-1</SUP></TD></TR><TR><TD> k<SUB>-DiMazF</SUB></TD><TD> 6.24 min<SUP>-1</SUP></TD></TR><TR><TD> k<SUB>DiMazE</SUB></TD><TD> 3.46 nM<SUP>-1</SUP> min<SUP>-1</SUP></TD></TR><TR><TD> k<SUB>-DiMazE</SUB></TD><TD> 7.25 min<SUP>-1</SUP></TD></TR><TR><TD> k<SUB>Hexa</SUB></TD><TD> 4.51nM<SUP>-2</SUP> min<SUP>-1</SUP></TD></TR><TR><TD> k<SUB>-Hexa</SUB></TD><TD> 4.05 min<SUP>-1</SUP></TD></TR><TR><TD> E<SUB>DiMazF</SUB></TD><TD> 0.46 nM<SUP>-1</SUP> min<SUP>-1</SUP></TD></TR><TR><TD> d<SUB>MazF</SUB></TD><TD> 0.7 min<SUP>-1</SUP></TD></TR><TR><TD> d<SUB>MazE</SUB></TD><TD> 0.55 min<SUP>-1</SUP></TD></TR><TR><TD> d<SUB>DiMazF</SUB></TD><TD> 0.17 min<SUP>-1</SUP></TD></TR><TR><TD> d<SUB>DiMazE</SUB></TD><TD> 0.416 min<SUP>-1</SUP></TD></TR><TR><TD> d<SUB>Hexa</SUB></TD><TD> 0.511 min<SUP>-1</SUP></TD></TR></TBODY></TABLE></DIV></DIV></DIV><DIV id="promoters"><DIV id="promoters_header"><H3><SPAN>3-3. Promoters</SPAN></H3></DIV><DIV id="promoters_contents"><DIV onclick="obj=document.getElementById('open4').style; obj.display=(obj.display=='none')?'block':'none';"><A style="cursor:pointer;">▼Read more</A></DIV><DIV id="open4" style="display:none;clear:both;"><P class="normal_text">We measured the relation between the AHL inputted and the fluorescence intensity.
					Changing the AHL concentration from 10<SUP>-4</SUP> to 10<SUP>4</SUP> by one power of 10 at a time, we measured the fluorescence intensity 4 hours after the insertion. We then performed the fitting with these data.
					In our experimental data we only measured the fluorescence intensity of GFP. However, we can only do the modulation for the concentration of GFP. Therefore, we needed the relationship between the concentration of GFP by modeling and the fluorescence intensity by experiment. We assumed that the fluorescence intensity is proportional to the concentration of GFP. We determined the parameter from experimental data.</P><P class="normal_text" style="text-align: center;">Fluorescence intensity = 31.8 [GFP]</P><P class="normal_text">From this parameter, we determined the parameters of the promoters.
					We defined the following set of differential equations.
					$$ \frac{d[GFP]}{dt} = leak_{Prhl} + \frac{\kappa_{Rhl}[C4]^{n_{Rhl}}}{K_{mRhl}^{n_{Rhl}} + [C4]^{n_{Rhl}}} $$
					The same holds true for Lux system.
					$$ \frac{d[GFP]}{dt} = leak_{Plux} + \frac{\kappa_{Lux}[C12]^{n_{Lux}}}{K_{mLux}^{n_{Lux}} + [C12]^{n_{Lux}}} $$
					We used MATLAB to fit the parameters to the experimental data and we fit them to the experimental concentrations.
					</P><DIV align="center"><P class="caption" style="font-size: 16px; text-align: center;"><SPAN style="font-weight: bold;">Fig.5-3-3-1. Fitted to the experimental data of Rhl system</SPAN></P></DIV><P class="normal_text">We obtained these parameters as follows:</P><P class="normal_text" style="text-align: center;">Leak<SUB>Prhl</SUB> = 0.86  
					                                                                             κ<SUB>Rhl</SUB>&amp;=&amp; 1.326
					                                                                             n<SUB>Rhl</SUB>&amp;=&amp; 5 
					                                                                             K<SUB>mRhl</SUB>&amp;=&amp; 1000 </P><DIV align="center"><P class="caption" style="font-size: 16px; text-align: center;"><SPAN style="font-weight: bold;">Fig.5-3-3-2. Fitted to the experimental data of Lux system</SPAN></P></DIV><P class="normal_text" style="text-align: center;">Leak<SUB>Plux</SUB> = 0.86  
					                                                                             κ<SUB>Lux</SUB> = 1.326 
					                                                                             n<SUB>Lux</SUB> = 5 
					                                                                             K<SUB>mLux</SUB> = 1000 </P></DIV></DIV></DIV><DIV id="more"><DIV id="more_header"><H3><SPAN>3-4. More realistic model with mRNA</SPAN></H3></DIV><DIV id="more_contents"><DIV onclick="obj=document.getElementById('open5').style; obj.display=(obj.display=='none')?'block':'none';"><A style="cursor:pointer;">▼Read more</A></DIV><DIV id="open5" style="display:none;clear:both;"><P class="normal_text">We defined the differential equations and estimated the parameters in the AHL - GFP system as described in 3-3. However, the translation from mRNA to protein was not considered in the model. To simulate the story of &quot;the Snow White,&quot; the translation is essential because the whole system includes the toxin-antitoxin system involving the cleavage of mRNA. We also had to take in account in our fitting that the AHL inputted at the beginning decreases with time.
					Then, we redefined the following set of differential equations.
					$$ \frac{d[mRNA_{GFP}]}{dt} = leak_{Prhl} + \frac{κ_{Rhl}[C4]^{n_{Rhl}}}{K_{mRhl}^{n_{Rhl}} + [C4]^{n_{Rhl}}} - d[mRNA_{GFP}]$$
					$$ \frac{d[GFP]}{dt} = α[mRNA_{GFP}] - d_{GFP}[GFP] $$
					$$ \frac{d[C4]}{dt} = - d_{C4}[C4] $$
					The same holds true for Lux system.
					$$ \frac{d[mRNA_{GFP}]}{dt} = leak_{Plux} + \frac{κ_{Lux}[C12]^{n_{Lux}}}{K_{mLux}^{n_{Lux}} + [C12]^{n_{Lux}}} - d[mRNA_{GFP}]$$
					$$ \frac{d[GFP]}{dt} = α[mRNA_{GFP}] - d_{GFP}[GFP] $$
					$$ \frac{d[C12]}{dt} = - d_{C12}[C12] $$
					As a result, we obtained these parameters as follows:</P><P class="normal_text" style="text-align: center;">Leak<SUB>Prhl</SUB> = 4.65  
					                                                                             κ<SUB>Rhl</SUB> = 14.95
					                                                                             n<SUB>Rhl</SUB> = 5 
					                                                                             K<SUB>mRhl</SUB> = 1000 
					                                                                             and                                    Leak<SUB>Plux</SUB> = 2.26  
					                                                                             κ<SUB>Lux</SUB> = 6.98 
					                                                                             n<SUB>Lux</SUB> = 0.76 
					                                                                             K<SUB>mLux</SUB> = 116.24 </P><P class="normal_text">Using these parameters we simulated the reproduction of the story of Snow White.</P></DIV></DIV></DIV></DIV></DIV><DIV id="analysis" class="container"><DIV id="analysis_header" class="container_header"><H2><SPAN>4. Analysis</SPAN></H2></DIV><DIV id="analysis_contents" class="container_contents"><DIV id="prince_coli"><DIV id="prince_coli_header"><H3><SPAN>4-1. The Prince <SPAN style="font-style : italic">coli</SPAN> should be put in during the process</SPAN></H3></DIV><DIV id="prince_coli_contents"><P class="normal_text">We run simulations in order to determine whether we would get a better behavior let we introduce the Prince <SPAN style="font-style : italic">coli</SPAN> at the beginning or halfway of the story.</P><DIV class="floating"><P class="caption" style="font-size: 16px; text-align: center;"><SPAN style="font-weight: bold;">Fig.5-4-1-1. Number of individuals when the Prince <SPAN style="font-style : italic">coli</SPAN> is introduced from the beginning</SPAN></P></DIV><DIV class="floating"><P class="caption" style="font-size: 16px; text-align: center;"><SPAN style="font-weight: bold;">Fig.5-4-1-2. AHL concentrations when the Prince <SPAN style="font-style : italic">coli</SPAN> is introduced from the beginning</SPAN></P></DIV><P class="normal_text">As a result, if we introduce the Prince <SPAN style="font-style : italic">coli</SPAN> from the beginning, the number of Prince <SPAN style="font-style : italic">coli</SPAN> increases too much (Fig.5-4-1-1) so the AmiE the Prince <SPAN style="font-style : italic">coli</SPAN> produces augments and the decomposition of C12 also occurs overly. So C12 is almost inexistent in the medium (Fig.5-4-1-2).</P><DIV class="floating"><P class="caption" style="font-size: 16px; text-align: center;"><SPAN style="font-weight: bold;">Fig.5-4-1-3. Number of individuals when the Prince <SPAN style="font-style : italic">coli</SPAN> is introduced at t = 700</SPAN></P></DIV><DIV class="floating"><P class="caption" style="font-size: 16px; text-align: center;"><SPAN style="font-weight: bold;">Fig.5-4-1-4. AHL concentrations when the Prince <SPAN style="font-style : italic">coli</SPAN> is introduced at t = 700</SPAN></P></DIV><P class="normal_text">On the other hand, if we introduce the Prince <SPAN style="font-style : italic">coli</SPAN> at t = 700, the number of Prince <SPAN style="font-style : italic">coli</SPAN> does not increment much (Fig.5-4-1-3), so C12 can exist until t = 700 and then decreases thanks to the augment of AmiE (Fig.5-4-1-4).In conclusion, if we introduce the Prince <SPAN style="font-style : italic">coli</SPAN> at t = 700, the circuit will behave accordingly.</P></DIV></DIV><DIV id="prhl"><DIV id="prhl_header"><H3><SPAN>4-2. Prhl should be changed</SPAN></H3></DIV><DIV id="prhl_contents"><P class="normal_text">In order to confirm the feasibility of the story with our gene circuit by the combination of the existing promoters, we performed some simulations based on the results of our assays.</P><DIV align="center"><P class="caption" style="font-size: 16px; text-align: center;"><SPAN style="font-weight: bold;">Fig.5-4-2. The intensity of Plux and Prhl promoters</SPAN></P><P class="normal_text">The diagram above shows that the intensity of the two promoters should be in the red region of the figure.The combination of promoters which we were originally going to use is shown in the graph by the green point. To move this point into the red region, we had to improve Prhl to raise its expression level.</P></DIV></DIV></DIV><DIV id="requirements"><DIV id="requirements_header"><H3><SPAN>4-3. Requirements</SPAN></H3></DIV><DIV id="requirements_contents"><DIV onclick="obj=document.getElementById('open6').style; obj.display=(obj.display=='none')?'block':'none';"><A style="cursor:pointer;">▼Read more</A></DIV><DIV id="open6" style="display:none;clear:both;"><P class="normal_text">We performed the sensitivity analysis descried in this section in order to examine which parameter dominates the story. We defined these requirements as the “successful Snow White story.”
					1)	At t = 150     concentration of RFP &gt; concentration of GFP
					2)	At t = 700      concentration  of RFP&lt; concentration of GFP
					3)	At t = 1500     concentration  of RFP &gt; concentration of GFP</P><P class="normal_text">We the began to analyze the graph that satisfies these requirements that we could say that recreates the story correctly. In the table bellow we show the range in which we modified each parameter. They were modified one step size at a time.</P><TABLE border="1" style="margin: auto;"><TBODY><TR><TD>Parameter </TD><TD> Range </TD><TD> Step size </TD></TR><TR><TD>D </TD><TD> $$ 0.0001 &lt; D &lt; 0.001 $$ </TD><TD>0.0001</TD></TR><TR><TD>p<SUB>C4</SUB></TD><TD> $$ 0.0001 &lt; p_{C4} &lt; 1 $$</TD><TD> 0.001 </TD></TR><TR><TD>p<SUB>C12</SUB></TD><TD> $$ 0.0001 &lt; p_{C12} &lt; 1 $$</TD><TD> 0.001 </TD></TR><TR><TD>α</TD><TD> $$ 0.01 &lt; α &lt; 0.2 $$</TD><TD> 0.01 </TD></TR><TR><TD>d<SUB>AmiE</SUB></TD><TD> $$ 0.001 &lt; d_{AmiE} &lt; 1 $$ </TD><TD> 0.001 </TD></TR></TBODY></TABLE><P class="normal_text">As a result, we obtained the following parameter ranges.</P><TABLE border="1" style="margin: auto;"><TBODY><TR><TD>Parameter </TD><TD> Value </TD></TR><TR><TD> D </TD><TD> $$ 0.0056 &lt; D &lt; 0.001 $$ </TD></TR><TR><TD> p<SUB>C4</SUB></TD><TD> $$ 0.0029 &lt; p_{C4} &lt; 0.778 $$ </TD></TR><TR><TD> p<SUB>C12</SUB></TD><TD> $$ 0.001 &lt; p_{C12} &lt; 0.217 $$ </TD></TR><TR><TD> α </TD><TD> $$ 0.01 &lt; α &lt; 0.16 $$ </TD></TR><TR><TD> d<SUB>AmiE</SUB></TD><TD> $$ 0.001 &lt; d_{AmiE} &lt; 1 $$ </TD></TR></TBODY></TABLE></DIV></DIV></DIV><DIV id="production_ahl"><DIV id="production_ahl_header"><H3><SPAN>4-4. Production rate of C4HSL and 3OC12HSL by RhlI and LasI</SPAN></H3></DIV><DIV id="production_ahl_contents"><DIV onclick="obj=document.getElementById('open7').style; obj.display=(obj.display=='none')?'block':'none';"><A style="cursor:pointer;">▼Read more</A></DIV><DIV id="open7" style="display:none;clear:both;"><P class="normal_text">The signaling molecule production rates by RhlI and LasI can be changed by modifying RhlI and LasI to make more or less signaling molecule in silico.</P><DIV align="center"><P class="caption" style="font-size: 16px; text-align: center;"><SPAN style="font-weight: bold;">Fig.5-4-4-1. Concentrations of GFP and RFP dependencies of production rate of C4 by RhlI</SPAN></P><P class="normal_text">Each line corresponds to the transition of the concentration of RFP and GFP with a certain production rate of C12. Red and green lines correspond to RFP and GFP, respectively. Blighter one indicates higher production rate of C12.
                   If the production rate of C4 is between 0.029 and 0.778, our system can recreate the story. If this parameter is too small, the production of LasI by the Queen <SPAN style="font-style : italic">coli</SPAN> is insufficiently inhibited by MazF so C12 increases greatly. As a result, The concentration of GFP overcomes the concentration of RFP and the story does not develop correctly. And if this parameter is too big, the production of LasI by the Queen <SPAN style="font-style : italic">coli</SPAN>  is overly inhibited by MazF so C12 does not increase.
                   As a result, the concentration of RFP is always greater than the GFP concentration and the story does not develop either.</P></DIV><DIV align="center"><P class="caption" style="font-size: 16px; text-align: center;"><SPAN style="font-weight: bold;">Fig.5-4-4-2. Concentrations of GFP and RFP dependencies of production rate of C12 by LasI</SPAN></P><P class="normal_text">Each line corresponds to the transition of the concentration of RFP/GFP with a certain production rate of C4. Red and green lines correspond to RFP and GFP, respectively. Blighter one indicates higher production rate of C4.
                    If production rate of C12 is between 0.001 and 0.217, our system can recreate the story.</P></DIV></DIV></DIV></DIV><DIV id="translation"><DIV id="translation_header"><H3><SPAN>4-5. Translation rate of protein</SPAN></H3></DIV><DIV id="translation_contents"><DIV onclick="obj=document.getElementById('open8').style; obj.display=(obj.display=='none')?'block':'none';"><A style="cursor:pointer;">▼Read more</A></DIV><DIV id="open8" style="display:none;clear:both;"><P class="normal_text">Translation rate affects the production of protein.</P><DIV align="center"><P class="caption" style="font-size: 16px; text-align: center;"><SPAN style="font-weight: bold;">Fig.5-4-5. Concentrations of GFP and RFP dependencies of translation rate of proteins</SPAN></P><P class="normal_text">Each line corresponds to the transition of the concentration of RFP and GFP with a certain translation rate. Red and green lines correspond to RFP and GFP, respectively. Blighter one indicates higher translation rate.
                   If translation rate of protein is between 0.01 and 0.16, our system can recreate the story.
</P></DIV></DIV></DIV></DIV><DIV id="decomposition"><DIV id="decomposition_header"><H3><SPAN>4-6. Decomposition rate of C12 by AmiE</SPAN></H3></DIV><DIV id="decomposition_contents"><DIV onclick="obj=document.getElementById('open9').style; obj.display=(obj.display=='none')?'block':'none';"><A style="cursor:pointer;">▼Read more</A></DIV><DIV id="open9" style="display:none;clear:both;"><P class="normal_text">AmiE decomposes C12. The concentration of C12 after input of the Prince <SPAN style="font-style : italic">coli</SPAN> is changed by AmiE. If the decomposition rate of C12 is too small, C12 does not decrease enough so the MazF inside Snow White <SPAN style="font-style : italic">coli</SPAN> continues being expressed and the concentration of RFP decreases.</P><DIV align="center"><P class="caption" style="font-size: 16px; text-align: center;"><SPAN style="font-weight: bold;">Fig.5-4-6. Concentrations of GFP and RFP dependencies of decomposition rate of C12 by AmiE</SPAN></P><P class="normal_text">Each line corresponds to the transition of the concentration of RFP and GFP with a certain decomposition rate of C12. Red and green lines correspond to RFP and GFP, respectively. Blighter one indicates higher value of decomposition rate of C12.
                   If degradation rate of C12 is higher than 0.00056, our system can recreate the story.</P></DIV></DIV></DIV></DIV><DIV id="degradation_amie"><DIV id="degradation_amie_header"><H3><SPAN>4-7. Degradation rate of AmiE</SPAN></H3></DIV><DIV id="degradation_amie_contents"><DIV onclick="obj=document.getElementById('open10').style; obj.display=(obj.display=='none')?'block':'none';"><A style="cursor:pointer;">▼Read more</A></DIV><DIV id="open10" style="display:none;clear:both;"><DIV align="center"><P class="caption" style="font-size: 16px; text-align: center;"><SPAN style="font-weight: bold;">Fig.5-4-7. Concentration of GFP and RFP dependencies of degradation rate of AmiE</SPAN></P><P class="normal_text">Each line corresponds to the transition of the concentration of RFP and GFP with a certain degradation rate of AmiE. Red and green lines correspond to RFP and GFP, respectively. Blighter one indicates higher degradation rate of AmiE.
                   Even if we modify the values of the parameters inside the defined range, the concentration of RFP overcomes the concentration of GFP. And even if the degradation rate of AmiE is small, the decomposition rate of C12 by AmiE is high enough so C12 decreases sufficiently.</P></DIV></DIV></DIV></DIV><DIV id="degradation_protein"><DIV id="degradation_protein"><H3><SPAN>4-8. Degradation rate of RFP and GFP</SPAN></H3></DIV><DIV id="degradation_protein_contents"><DIV onclick="obj=document.getElementById('open11').style; obj.display=(obj.display=='none')?'block':'none';"><A style="cursor:pointer;">▼Read more</A></DIV><DIV id="open11" style="display:none;clear:both;"><P class="normal_text">The degradation rate of RFP and GFP is key to the success the story of ‘&quot;Snow White&quot;.
					These parameters are closely related to the concentrations of GFP and RFP, so we conjectured that if they do not take appropriate values the story can not be correctly recreated.</P><DIV align="center"><P class="caption" style="font-size: 16px; text-align: center;"><SPAN style="font-weight: bold;">Fig.5-4-8 Relation of degradation rate of GFP and RFP</SPAN></P><P class="normal_text">The story is recreated only if the degradation rate of RFP and GFP are the same.</P></DIV></DIV></DIV></DIV></DIV></DIV><DIV id="software" class="container"><DIV id="software_header" class="container_header"><H2><SPAN>5. Software</SPAN></H2></DIV><DIV id="software_contents" class="container_contents"><DIV id="abstract"><DIV id="abstract_header"><H3><SPAN>5-1. Abstract</SPAN></H3></DIV><P class="normal_text">We developed a new software named ACADwarfs. This software helps to control the sensitivity of the protein to MazF by regulating the number of ACA sequences in the mRNA sequence. ACADwarfs can increase or decrease the number of ACA sequences on mRNA without changing the amino acid sequences that the mRNA specifies or frameshifts resulted from insertion of bases without considering. Then we improve the practicality of the characteristic of the <SPAN style="font-style : italic"> mazEF </SPAN> system. For example you can let protein A express constantly by eliminating ACA sequences of the sequence, while letting protein B stop being expressed, at the desirable timing, by expression of MazF.This software also evades the use of rare codons, so you don’t have to worry about them.</P><DIV id="key_achievements"><DIV id="key_achievements_header"><H3><SPAN>5-2. Key achievements</SPAN></H3></DIV><P class="normal_text">・Provided the tool regulating the number of ACA sequences</P><P class="normal_text">・Released under open-source license so everyone can use it</P><P class="normal_text">・Able to correspond to any base arrangements</P><P class="normal_text">・Rare codons are evaded</P><P class="normal_text">・Extend the application field of <SPAN style="font-style : italic">mazEF</SPAN> system</P><DIV id="work_flow"><DIV id="work_flow_header"><H3><SPAN>5-3. Work flow</SPAN></H3></DIV><DIV onclick="obj=document.getElementById('open12').style; obj.display=(obj.display=='none')?'block':'none';"><A style="cursor:pointer;">▼Read more</A></DIV><DIV id="open12" style="display:none;clear:both;"><OL type="1" class="list_parentheses"><LI><P class="normal_text"> Start ACADwarfs</P><P class="normal_text"> Download the zip file and open it.</P></LI><LI><P class="normal_text"> Put in the part sequence</P><P class="normal_text">Put the part sequence you want to regulate the effect of the <SPAN style="font-style : italic">mazEF</SPAN> system on in the upper window.</P></LI><LI><P class="normal_text"> Choose operation</P><P class="normal_text">If you want to magnify the effect of <SPAN style="font-style : italic">mazEF</SPAN> system, choose “increase ACA”.</P><P class="normal_text">If you want to lessen the effect of <SPAN style="font-style : italic">mazEF</SPAN> system, choose “decrease ACA”.</P></LI><LI><P class="normal_text">Get sequences</P><P class="normal_text">You can get the modified part sequences in the lower window. Modified codons will be written in capital letters and while the rest will be in small letters so you can easily locate the modified parts.And under this window you can see the number of ACA sequences on the sequence before the modification of the following &quot;previous number&quot; and the number of ACA sequences on the sequence after the modification of the following &quot;new number&quot;.</P></LI></OL></DIV><DIV id="demo"><DIV id="demo_header"><H3><SPAN>5-4. Demonstration</SPAN></H3></DIV><P class="normal_text">We created a demo to present the features of this software. Using this, we regulated the number of ACA sequences and control the sensitivity of the protein to MazF.</P><TABLE border="1" style="margin: auto;"><TBODY><TR><TD> Gene </TD><TD> Pre number of ACA sequences </TD><TD>Post number of ACA sequences </TD></TR><TR><TD> RFP</TD><TD> 10 </TD><TD> 30 </TD></TR><TR><TD> GFP </TD><TD> 23 </TD><TD> 39 </TD></TR><TR><TD> MazF </TD><TD> 2 </TD><TD> 1 </TD></TR><TR><TD> MazE </TD><TD> 2 </TD><TD> 1 </TD></TR></TBODY></TABLE><P class="normal_text">We increased the numbers of ACA sequences of RFP and GFP decreased the numbers of ACA sequences of MazF and MazE .</P><DIV align="center"><P class="caption" style="font-size: 16px; text-align: center;"><SPAN style="font-weight: bold;">Fig.5-4. Comparison between the results of simulations using original sequences and modified sequences</SPAN></P></DIV><P class="normal_text">We can see that the concentration of expressed MazF reacts more keenly after adjusting the ACA sequences than before doing so.</P><DIV id="download"><DIV id="download_header"><H3><SPAN>5-5. Download</SPAN></H3></DIV><P class="normal_text">To download click <A href="https://static.igem.org/mediawiki/2016/3/32/T--Tokyo_Tech--ACA_Dwarfs.zip">here</A>.</P><P class="normal_text">The code is available on <A href="https://github.com/Ryuta339/ACADwarfs">github</A>.</P></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></BODY></HTML>