Team:Tokyo Tech/Model
Model
Contents
1. Overview
2. Story simulation
   2-1. Mathematical model
   2-2. Results
3. Fitting
   3-1. Population growth
   3-2. Toxin-Antitoxin system
   3-3. Promoters
   3-4. More realistic model with mRNA
4. Analysis
   4-1. The Prince coli should be put in during the process
   4-2. Prhl should be changed
   4-3. Requirements
   4-4. Production rate of C4HSL and 3OC12HSL by RhlI and LasI
   4-5. Translation rate of protein
   4-6. Decomposition rate of 3OC12HSL by AmiE
   4-6. Degradation rate of AmiE
   4-7. Degradation rate of RFP and GFP
5. Software
   5-1. Abstract
   5-2. Key achievements
   5-3. Work flow
   5-4. Demonstration
   5-5. Download
1. Overview
To recreate the story of ”the Snow White”, we have designed a cell-cell communication system with improved or characterized parts and collected data from comprehensive experiments. Furthermore, we constructed the mathematical model to simulate the behavior of the whole system and to confirm the feasibility of our story. This simulation successfully contributed to give the suggestions to wet lab experiments. In addition, in order to help us utilize our Toxin-Antitoxin (TA) system, we developed a new software in Java for adjusting the number of ACA sequences, which MazF dimer recognizes and cleaves in mRNAs.
2. Story simulation
2-1. Mathematical model
In order to simulate our gene circuits, we developed an ordinary differential equation model.
[ Model development ]
▼ Differential equations
Snow White
\begin{equation}        \frac{d[mRNA_{RFP}]}{dt} = k - d[mRNA_{RFP}] - F_{DiMazF}(1-(1-f)^{f_{mRNA_{RFP}}})[mRNA_{RFP}][DiMazF]        \end{equation}        \begin{equation}        \frac{d[mRNA_{RhlI}]}{dt} =  leak_{P_{lux}} + \frac{\kappa_{Lux}[C12]^{n_{Lux}}}{K_{mLux}^{n_{Lux}} + [C12]^{n_{Lux}}} - d[mRNA_{RhlI}] - F_{DiMazF}f[mRNA_{RhlI}][DiMazF]        \end{equation}        \begin{equation}        \frac{d[RFP]}{dt} = \alpha [mRNA_{RFP}] - d_{RFP}[RFP]        \end{equation}        \begin{equation}        \frac{d[RhlI]}{dt} = \alpha [mRNA_{RhlI}] - d_{RhlI}[RhlI]        \end{equation}        \begin{equation}        \frac{d[C4]}{dt} = p_{C4}[RhlI]P_{Snow White} - d_{C4}[C4]        \end{equation}        \begin{eqnarray}        \frac{d[mRNA_{MazF}]}{dt} &=&leak_{P_{lux}} + \frac{\kappa_{Lux}[C12]^{n_{Lux}}}{K_{mLux}^{n_{Lux}}+ [C12]^{n_{Lux}}} - d[mRNA_{MazF}]  \nonumber \\ && \ \ \ \ \ \ \ \ \ \ \ \ -F_{DiMazF}(1-(1-f)^{f_{mRNA_{MazF}}})[mRNA_{MazF}][DiMazF]        \end{eqnarray}        \begin{equation}        \frac{d[mRNA_{MazE}]}{dt} = k - d[mRNA_{MazE}] - F_{DiMazF}(1-(1-f)^{f_{mRNA_{MazE}}})[mRNA_{MazE}][DiMazF]        \end{equation}        \begin{equation}        \frac{d[MazF]}{dt} = \alpha [mRNA_{MazF}] - 2k_{Di_{MazF}}[MazF] + 2k_{-Di_{MazF}}[DiMazF] - d_{MazF}[MazF]        \end{equation}        \begin{eqnarray}        \frac{d[DiMazF]}{dt} &=& k_{Di_{MazF}}[MazF] - k_{-Di_{MazF}}[DiMazF] - 2k_{Hexa}[DiMazE][DiMazF]^2 \nonumber \\ && \ \ \ \ \ \ \ \ \ \ \ \ \ \ \ \ \ \ \ \ \ +2k_{-Hexa}[MazHexamer] - d_{DiMazF}[DiMazF]        \end{eqnarray}        \begin{equation}        \frac{d[MazE]}{dt} = \alpha [mRNA_{MazE}] - 2k_{Di_{MazE}}[MazE] + 2k_{-Di_{MazE}}[DiMazE] - d_{MazE}[MazE]        \end{equation}        \begin{eqnarray}        \frac{d[DiMazE]}{dt} &=& k_{Di_{MazE}}[MazE] - k_{-Di_{MazE}}[DiMazE] - k_{Hexa}[DiMazE][DiMazF]^2 \nonumber \\ && \ \ \ \ \ \ \ \ \ + k_{-Hexa}[MazHexamer] - d_{DiMazE}[DiMazE]        \end{eqnarray}        \begin{equation}        \frac{d[MazHexa]}{dt} = k_{Hexa}[DiMazE][DiMazF]^2 - k_{-Hexa}[MazHexa] - d_{Hexa}[MazHexa]        \end{equation}        \begin{equation}        \frac{dP_{Snow White}}{dt} = g \frac{E_{DiMazF}}{E_{DiMazF}+[DiMazF]}\left(1- \frac{P_{Snow White}+P_{Queen}+P_{Prince}}{P_{max}} \right) P_{Snow White}        \end{equation}
Queen
\begin{equation}        \frac{d[mRNA_{GFP}]}{dt} = k - d[mRNA_{GFP}] - F_{DiMazF}(1-(1-f)^{f_{mRNA_{GFP}}})[mRNA_{GFP}][DiMazF]        \end{equation}        \begin{eqnarray}        \frac{d[mRNA_{LasI}]}{dt} &=& leak_{P_{rhl}} + \frac{\kappa_{Rhl}[C4]^{n_{Rhl}}}{K_{mRhl}^{n_{Rhl}} + [C4]^{n_{Rhl}}}  \nonumber \\ && \ \ \ \ - d[mRNA_{LasI}] - F_{DiMazF}(1-(1-f)^{f_{mRNA_{LasI}}})[mRNA_{LasI}][DiMazF]        \end{eqnarray}        \begin{equation}        \frac{d[GFP]}{dt} = \alpha [mRNA_{GFP}] - d_{GFP}[GFP]        \end{equation}        \begin{equation}        \frac{d[LasI]}{dt} = \alpha [mRNA_{LasI}] - d_{LasI}[LasI]        \end{equation}        \begin{equation} \frac{d[C12]}{dt} = p_{C12}[LasI]P_{Queen} - d_{C12}[C12] - D[C12][AmiE]        \end{equation}        \begin{eqnarray}        \frac{d[mRNA_{MazF}]}{dt} &=& leak_{P_{lux}} + \frac{\kappa_{Rhl}[C4]^{n_{Rhl}}}{K_{mRhl}^{n_{Rhl}} + [C4]^{n_{Rhl}}} \nonumber \\ && - d[mRNA_{MazF}]  - F_{DiMazF}(1-(1-f)^{f_{mRNA_{MazF}}})[mRNA_{MazF}][DiMazF]        \end{eqnarray}        \begin{equation}        \frac{d[mRNA_{MazE}]}{dt} = k - d[mRNA_{MazE}] - F_{DiMazF}(1-(1-f)^{f_{mRNA_{MazE}}})[mRNA_{MazE}][DiMazF]        \end{equation}        \begin{equation}        \frac{d[MazF]}{dt} = \alpha [mRNA_{MazF}] - 2k_{Di_{MazF}}[MazF] + 2k_{-Di_{MazF}}[DiMazF] - d_{MazF}[MazF]        \end{equation}        \begin{eqnarray}        \frac{d[DiMazF]}{dt} &=& k_{Di_{MazF}}[MazF] - k_{-Di_{MazF}}[DiMazF] - 2k_{Hexa}[DiMazE][DiMazF]^2 \nonumber \\ && + 2k_{-Hexa}[MazHexamer] - d_{DiMazF}[DiMazF]        \end{eqnarray}        \begin{equation}        \frac{d[MazE]}{dt} = \alpha [mRNA_{MazE}] - 2k_{Di_{MazE}}[MazE] + 2k_{-Di_{MazE}}[DiMazE] - d_{MazE}[MazE]        \end{equation}        \begin{eqnarray}        \frac{d[DiMazE]}{dt} &=& k_{Di_{MazE}}[MazE] - k_{-Di_{MazE}}[DiMazE] - k_{Hexa}[DiMazE][DiMazF]^2 \nonumber \\ && + k_{-Hexa}[MazHexamer] - d_{DiMazE}[DiMazE]        \end{eqnarray}        \begin{equation}        \frac{d[MazHexa]}{dt} = k_{Hexa}[DiMazE][DiMazF]^2 - k_{-Hexa}[MazHexa] - d_{Hexa}[MazHexa]        \end{equation}        \begin{equation}        \frac{dP_{Queen}}{dt} = g \frac{E_{DiMazF}}{E_{DiMazF}+[DiMazF]}\left(1- \frac{P_{Snow White}+P_{Queen}+P_{Prince}}{P_{max}}\right) P_{Queen}\\        \end{equation}
Prince
\begin{equation}        \frac{d[mRNA_{AmiE}]}{dt} = leak_{P_{lux}} + \frac{\kappa_{Lux}[C12]^n}{K_{mLux}^n + [C12]^n} - d[mRNA_{AmiE}]        \end{equation}        \begin{equation}        \frac{d[AmiE]}{dt} = \alpha [mRNA_{AmiE}]P_{Prince} - d_{AmiE}[AmiE]        \end{equation}        \begin{equation}        \frac{dP_{Prince}}{dt} = g\left(1- \frac{P_{Snow White}+P_{Queen}+P_{Prince}}{P_{max}}\right) P_{Prince}        \end{equation}
▼Parameters
Parameter
Description
$$g$$
Growth rate of each cells
$$P_{max}$$
Carrying capacity
$$E_{DiMazF}$$
Effect of MazF dimer on growth rate
$$k$$
Transcription rate of mRNA under \(P_{tet}\)
$$leak_{P_{lux}}$$
Leakage of \(P_{lux}\)
$$leak_{P_{rhl}}$$
Leakage of \(P_{rhl}\)
$$\kappa_{Lux}$$
Maximum transcription rate of mRNA under \(P_{lux}\)
$$\kappa_{Rhl}$$
Maximum transcription rate of mRNA under \(P_{rhl}\)
$$n_{Lux}$$
Hill coefficient for \(P_{lux}\)
$$n_{Rhl}$$
Hill coefficient for \(P_{rhl}\)
$$K_{mLux}$$
Lumped paremeter for the Lux System
$$K_{mRhl}$$
Lumped paremeter for the Rhl System
$$F_{DiMazF}$$
Cutting rate at ACA sequences on mRNA by MazF dimer
$$f$$
The probability of distinction of ACA sequencess in each mRNA
$$f_{mRNA_{RFP}}$$
The number of ACA sequences in \(mRNA_{RFP}\)
$$f_{mRNA_{GFP}}$$
The number of ACA sequences in \(mRNA_{GFP}\)
$$f_{mRNA_{RhlI}}$$
The number of ACA sequences in \(mRNA_{RhlI}\)
$$f_{mRNA_{LasI}}$$
The number of ACA sequences in \(mRNA_{LasI}\)
$$f_{mRNA_{MazF}}$$
The number of ACA sequences in \(mRNA_{MazF}\)
$$f_{mRNA_{MazE}}$$
The number of ACA sequences in \(mRNA_{MazE}\)
$$\alpha$$
Translation rate of Protein
$$k_{Di_{MazF}}$$
Formation rate of MazF dimer
$$k_{-Di_{MazF}}$$
Dissociation rate of MazF dimer
$$k_{Di_{MazE}}$$
Formation rate of MazE dimer
$$k_{-Di_{MazE}}$$
Dissociation rate of MazE dimer
$$k_{Hexa}$$
Formation rate of Maz hexamer
$$k_{-Hexa}$$
Dissociation rate of Maz hexamer
$$p_{C4}$$
Production rate of C4HSL by RhlI
$$p_{C12}$$
Production rate of 3OC12HSL by LuxI
$$D$$
Decomposition rate of 3OC12HSL by AmiE
$$d$$
Degradation rate of mRNA
$$d_{RFP}$$
Degradation rate of RFP
$$d_{GFP}$$
Degradation rate of GFP
$$d_{RhlI}$$
Degradation rate of RhlI
$$d_{LasI}$$
Degradation rate of LasI
$$d_{MazF}$$
Degradation rate of MazF
$$d_{DiMazF}$$
Degradation rate of MazF dimer
$$d_{MazE}$$
Degradation rate of MazE
$$d_{DiMazE}$$
Degradation rate of MazE dimer
$$d_{Hexa}$$
Degradation rate of Maz Hexamer
$$d_{C4}$$
Degradation rate of C4HSL
$$d_{C12}$$
Degradation rate of 3OC12HSL
$$d_{AmiE}$$
Degradation rate of AmiE
2-2. Results
We obtained and confirmed the desirable behavior of the whole system by modifying and improving parts. As described below, our simulation showed an appropriate transition of concentration of RFP and GFP for the story.
Fig.5-2.2. Time-dependent change of the concentrations of fluorescent proteins
In the blue area of Fig.5-2-2, the concentration of fluorescent proteins start to increase. The concentration of RFP of Snow White coli exceeds  that of GFP of the Queen coli.
It is as if Snow White got fairer more and more.
In the pink area of Fig.5-2-2, the concentration of C12 increase thanks to the appearance of C4. As a result, the MazF inside Snow White coli and the Queen coli start to suppress the increment of fluorescet proteins.
It is as if the Queen, influenced by the Mirror's answer, transforming into a Witch in order to give Snow White a poisoned apple.
In the green area of Fig.5-2-2, the concentration of C12 more increases and the MazF inside Snow White coli more suppress the increment of GFP. So the concentration GFP exceeds that of RFP.
It looks as if Snow White bit the apple, sinking into unconsciousness soon.
In the yellow area of Fig.5-2-2, the AmiE synthesized by the introduced Prince coli decomposes C12 so the MazF inside Snow White coli diminishes and the concentration of GFP resumes.
It looks as if the Prince lifted Snow White and she opened her eyes.
3. Fitting
3-1. Population growth
First, we tried to model the growth curve of the system. When the number of E. coli approaches a certain value, the growth will stop. We defined this value in the culture as Pmax. Then the population growth equation for our system is described as follows:
$$ \frac{dP}{dt} = g\left(1 - \frac{P}{P_{max}}\right)P$$
where g is the population growth rate.
This equation can be analytically solved as:
$$ P = \frac{P_{0} P_{max} e^{gt}}{P_{max} - P_{0} + P_{0} e^{gt}}$$
where P 0 is the population at t = 0. We used this equation to fit the experimental data.
Fig.5-3-1. Modeled growth curve of E. coli fitted to experiment data
Using the experimental data from the Toxin assay for this fitting, we estimated the following parameters:
g = 0.0123
and
Pmax =3.3
respectively.
These parameters can be used for Snow White coli, the Queen coli and the Prince coli in the same way.
3-2. Toxin-Antitoxin system
▼Read more
Since there are only few researches that actually discussed on the kinetics of the TA system, we estimated the parameters for our TA system. We got the experimental data from Toxin assay for this fitting. The differential equations representing the TA system are described as follows:       \begin{equation}       \frac{d[MazF]}{dt} = \alpha [mRNA_{MazF}] - 2k_{Di_{MazF}}[MazF] + 2k_{-Di_{MazF}}[DiMazF] - d_{MazF}[MazF] \nonumber      \end{equation}
\begin{eqnarray}       \frac{d[DiMazF]}{dt} &=& k_{Di_{MazF}}[MazF] - k_{-Di_{MazF}}[DiMazF] - 2k_{Hexa}[DiMazE][DiMazF]^2 \nonumber \\ && + 2k_{-Hexa}[MazHexamer] - d_{DiMazF}[DiMazF] \end{eqnarray}
\begin{equation}       \frac{d[MazE]}{dt} = \alpha [mRNA_{MazE}] - 2k_{Di_{MazE}}[MazE] + 2k_{-Di_{MazE}}[DiMazE] - d_{MazE}[MazE] \nonumber       \end{equation}
\begin{eqnarray}       \frac{d[DiMazE]}{dt} &=& k_{Di_{MazE}}[MazE] - k_{-Di_{MazE}}[DiMazE] - k_{Hexa}[DiMazE][DiMazF]^2  \nonumber \\ && + k_{-Hexa}[MazHexamer] - d_{DiMazE}[DiMazE]       \end{eqnarray}
\begin{equation}       \frac{d[MazHexa]}{dt} = k_{Hexa}[DiMazE][DiMazF]^2 - k_{-Hexa}[MazHexa] - d_{Hexa}[MazHexa]  \nonumber       \end{equation}
We used genetic algorithms to fit this data. We used ODs at 7 points and fit them to experimental ODs.
Fig.5-3-2. Fitted growth curve of E.coli to the experimental data of TA system
As a result, we obtained the following parameters.
Parameter
Value
kDiMazF
6.82 nM-1 min-1
k-DiMazF
6.24 min-1
kDiMazE
3.46 nM-1 min-1
k-DiMazE
7.25 min-1
kHexa
4.51nM-2 min-1
k-Hexa
4.05 min-1
EDiMazF
0.46 nM-1 min-1
dMazF
0.7 min-1
dMazE
0.55 min-1
dDiMazF
0.17 min-1
dDiMazE
0.416 min-1
dHexa
0.511 min-1
3-3. Promoters
▼Read more
We measured the relation between the AHL inputted and the fluorescence intensity.
Changing the AHL concentration from 10-4 to 104 by one power of 10 at a time, we measured the fluorescence intensity 4 hours after the insertion. We then performed the fitting with these data.
In our experimental data we only measured the fluorescence intensity of GFP. However, we can only do the modulation for the concentration of GFP. Therefore, we needed the relationship between the concentration of GFP by modeling and the fluorescence intensity by experiment. We assumed that the fluorescence intensity is proportional to the concentration of GFP. We determined the parameter from experimental data.
Fluorescence intensity = 31.8 [GFP]
From this parameter, we determined the parameters of the promoters.
We defined the following set of differential equations.
$$ \frac{d[GFP]}{dt} = leak_{Prhl} + \frac{\kappa_{Rhl}[C4]^{n_{Rhl}}}{K_{mRhl}^{n_{Rhl}} + [C4]^{n_{Rhl}}} $$
The same holds true for Lux system.
$$ \frac{d[GFP]}{dt} = leak_{Plux} + \frac{\kappa_{Lux}[C12]^{n_{Lux}}}{K_{mLux}^{n_{Lux}} + [C12]^{n_{Lux}}} $$      We used MATLAB to fit the parameters to the experimental data and we fit them to the experimental concentrations.
Fig.5-3-3-1. Fitted to the experimental data of Rhl system
We obtained these parameters as follows:
LeakPrhl = 0.86
κRhl &=& 1.326
nRhl &=& 5
KmRhl &=& 1000
Fig.5-3-3-2. Fitted to the experimental data of Lux system
LeakPlux = 0.86
κLux = 1.326
nLux = 5
KmLux = 1000
3-4. More realistic model with mRNA
▼Read more
We defined the differential equations and estimated the parameters in the AHL - GFP system as described in 3-3. However, the translation from mRNA to protein was not considered in the model. To simulate the story of "the Snow White," the translation is essential because the whole system includes the toxin-antitoxin system involving the cleavage of mRNA. We also had to take in account in our fitting that the AHL inputted at the beginning decreases with time.
Then, we redefined the following set of differential equations.
$$ \frac{d[mRNA_{GFP}]}{dt} = leak_{Prhl} + \frac{κ_{Rhl}[C4]^{n_{Rhl}}}{K_{mRhl}^{n_{Rhl}} + [C4]^{n_{Rhl}}} - d[mRNA_{GFP}]$$
$$ \frac{d[GFP]}{dt} = α[mRNA_{GFP}] - d_{GFP}[GFP] $$
$$ \frac{d[C4]}{dt} = - d_{C4}[C4] $$
The same holds true for Lux system.
$$ \frac{d[mRNA_{GFP}]}{dt} = leak_{Plux} + \frac{κ_{Lux}[C12]^{n_{Lux}}}{K_{mLux}^{n_{Lux}} + [C12]^{n_{Lux}}} - d[mRNA_{GFP}]$$
$$ \frac{d[GFP]}{dt} = α[mRNA_{GFP}] - d_{GFP}[GFP] $$
$$ \frac{d[C12]}{dt} = - d_{C12}[C12] $$
As a result, we obtained these parameters as follows:
LeakPrhl = 4.65
κRhl = 14.95
nRhl = 5
KmRhl = 1000
and
LeakPlux = 2.26
κLux = 6.98
nLux = 0.76
KmLux = 116.24
Using these parameters we simulated the reproduction of the story of Snow White.
4. Analysis
4-1. The Prince coli should be put in during the process
We run simulations in order to determine whether we would get a better behavior let we introduce the Prince coli at the beginning or halfway of the story.
Fig.5-4-1-1. Number of individuals when the Prince coli is introduced from the beginning
Fig.5-4-1-2. AHL concentrations when the Prince coli is introduced from the beginning
As a result, if we introduce the Prince coli from the beginning, the number of Prince coli increases too much (Fig.5-4-1-1) so the AmiE the Prince coli produces augments and the decomposition of C12 also occurs overly. So C12 is almost inexistent in the medium (Fig.5-4-1-2).
Fig.5-4-1-3. Number of individuals when the Prince coli is introduced at t = 700
Fig.5-4-1-4. AHL concentrations when the Prince coli is introduced at t = 700
On the other hand, if we introduce the Prince coli at t = 700, the number of Prince coli does not increment much (Fig.5-4-1-3), so C12 can exist until t = 700 and then decreases thanks to the augment of AmiE (Fig.5-4-1-4).
In conclusion, if we introduce the Prince coli at t = 700, the circuit will behave accordingly.
4-2. Prhl should be changed
In order to confirm the feasibility of the story with our gene circuit by the combination of the existing promoters, we performed some simulations based on the results of our assays.
Fig.5-4-2. The intensity of Plux and Prhl promoters
The diagram above shows that the intensity of the two promoters should be in the red region of the figure.The combination of promoters which we were originally going to use is shown in the graph by the green point. To move this point into the red region, we had to improve Prhl to raise its expression level.
4-3. Requirements
▼Read more
We performed the sensitivity analysis descried in this section in order to examine which parameter dominates the story. We defined these requirements as the “successful Snow White story.”
1) At t = 150
     concentration of RFP > concentration of GFP
2) At t = 700
      concentration  of RFP< concentration of GFP
3) At t = 1500
     concentration  of RFP > concentration of GFP
We the began to analyze the graph that satisfies these requirements that we could say that recreates the story correctly. In the table bellow we show the range in which we modified each parameter. They were modified one step size at a time.
Parameter
Range
Step size
D
$$ 0.0001 < D < 0.001 $$
0.0001
pC4
$$ 0.0001 < p_{C4} < 1 $$
0.001
pC12
$$ 0.0001 < p_{C12} < 1 $$
0.001
α
$$ 0.01 < α < 0.2 $$
0.01
dAmiE
$$ 0.001 < d_{AmiE} < 1 $$
0.001
As a result, we obtained the following parameter ranges.
Parameter
Value
D
$$ 0.0056 < D < 0.001 $$
pC4
$$ 0.0029 < p_{C4} < 0.778 $$
pC12
$$ 0.001 < p_{C12} < 0.217 $$
α
$$ 0.01 < α < 0.16 $$
dAmiE
$$ 0.001 < d_{AmiE} < 1 $$
4-4. Production rate of C4HSL and 3OC12HSL by RhlI and LasI
▼Read more
The signaling molecule production rates by RhlI and LasI can be changed by modifying RhlI and LasI to make more or less signaling molecule in silico.
Fig.5-4-4-1. Concentrations of GFP and RFP dependencies of production rate of C4 by RhlI
Each line corresponds to the transition of the concentration of RFP and GFP with a certain production rate of C12. Red and green lines correspond to RFP and GFP, respectively. Blighter one indicates higher production rate of C12.
If the production rate of C4 is between 0.029 and 0.778, our system can recreate the story. If this parameter is too small, the production of LasI by the Queen coli is insufficiently inhibited by MazF so C12 increases greatly. As a result, The concentration of GFP overcomes the concentration of RFP and the story does not develop correctly. And if this parameter is too big, the production of LasI by the Queen coli is overly inhibited by MazF so C12 does not increase.
As a result, the concentration of RFP is always greater than the GFP concentration and the story does not develop either.
Fig.5-4-4-2. Concentrations of GFP and RFP dependencies of production rate of C12 by LasI
Each line corresponds to the transition of the concentration of RFP/GFP with a certain production rate of C4. Red and green lines correspond to RFP and GFP, respectively. Blighter one indicates higher production rate of C4.
If production rate of C12 is between 0.001 and 0.217, our system can recreate the story.
4-5. Translation rate of protein
▼Read more
Translation rate affects the production of protein.
Fig.5-4-5. Concentrations of GFP and RFP dependencies of translation rate of proteins
Each line corresponds to the transition of the concentration of RFP and GFP with a certain translation rate. Red and green lines correspond to RFP and GFP, respectively. Blighter one indicates higher translation rate.
If translation rate of protein is between 0.01 and 0.16, our system can recreate the story.
4-6. Decomposition rate of C12 by AmiE
▼Read more
AmiE decomposes C12. The concentration of C12 after input of the Prince coli is changed by AmiE. If the decomposition rate of C12 is too small, C12 does not decrease enough so the MazF inside Snow White coli continues being expressed and the concentration of RFP decreases.
Fig.5-4-6. Concentrations of GFP and RFP dependencies of decomposition rate of C12 by AmiE
Each line corresponds to the transition of the concentration of RFP and GFP with a certain decomposition rate of C12. Red and green lines correspond to RFP and GFP, respectively. Blighter one indicates higher value of decomposition rate of C12.
If degradation rate of C12 is higher than 0.00056, our system can recreate the story.
4-7. Degradation rate of AmiE
▼Read more
Fig.5-4-7. Concentration of GFP and RFP dependencies of degradation rate of AmiE
Each line corresponds to the transition of the concentration of RFP and GFP with a certain degradation rate of AmiE. Red and green lines correspond to RFP and GFP, respectively. Blighter one indicates higher degradation rate of AmiE.
Even if we modify the values of the parameters inside the defined range, the concentration of RFP overcomes the concentration of GFP. And even if the degradation rate of AmiE is small, the decomposition rate of C12 by AmiE is high enough so C12 decreases sufficiently.
4-8. Degradation rate of RFP and GFP
▼Read more
The degradation rate of RFP and GFP is key to the success the story of ‘"Snow White".
These parameters are closely related to the concentrations of GFP and RFP, so we conjectured that if they do not take appropriate values the story can not be correctly recreated.
Fig.5-4-8 Relation of degradation rate of GFP and RFP
The story is recreated only if the degradation rate of RFP and GFP are the same.
5. Software
5-1. Abstract
We developed a new software named ACADwarfs. This software helps to control the sensitivity of the protein to MazF by regulating the number of ACA sequences in the mRNA sequence. ACADwarfs can increase or decrease the number of ACA sequences on mRNA without changing the amino acid sequences that the mRNA specifies or frameshifts resulted from insertion of bases without considering.
Then we improve the practicality of the characteristic of the mazEF system. For example you can let protein A express constantly by eliminating ACA sequences of the sequence, while letting protein B stop being expressed, at the desirable timing, by expression of MazF.
This software also evades the use of rare codons, so you don’t have to worry about them.
5-2. Key achievements
・Provided the tool regulating the number of ACA sequences
・Released under open-source license so everyone can use it
・Able to correspond to any base arrangements
・Rare codons are evaded
・Extend the application field of mazEF system
5-3. Work flow
▼Read more
Start ACADwarfs
Download the zip file and open it.
Put in the part sequence
Put the part sequence you want to regulate the effect of the mazEF system on in the upper window.
Choose operation
If you want to magnify the effect of mazEF system, choose “increase ACA”.
If you want to lessen the effect of mazEF system, choose “decrease ACA”.
Get sequences
You can get the modified part sequences in the lower window. Modified codons will be written in capital letters and while the rest will be in small letters so you can easily locate the modified parts.
And under this window you can see the number of ACA sequences on the sequence before the modification of the following "previous number" and the number of ACA sequences on the sequence after the modification of the following "new number".
5-4. Demonstration
We created a demo to present the features of this software. Using this, we regulated the number of ACA sequences and control the sensitivity of the protein to MazF.
Gene
Pre number of ACA sequences
Post number of ACA sequences
RFP
10
30
GFP
23
39
MazF
2
1
MazE
2
1
We increased the numbers of ACA sequences of RFP and GFP decreased the numbers of ACA sequences of MazF and MazE .
Fig.5-4. Comparison between the results of simulations using original sequences and modified sequences
We can see that the concentration of expressed MazF reacts more keenly after adjusting the ACA sequences than before doing so.
5-5. Download
To download click here .
The code is available on github .
