<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_BNDS_CHINA_Experiments skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><DIV id="top_title"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:BNDS CHINA/Experiments</SPAN></H1></DIV><DIV id="HQ_page"><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><P><HEADER><NAV><UL class="navbar"><LI class="navButton"><A href="https://2018.igem.org/Team:BNDS_CHINA">Home</A></LI><LI class="dropdown"><A class="dropdownButton">Team</A><DIV class="dropdownContent"><A href="https://2018.igem.org/Team:BNDS_CHINA/Team">Team Members</A><A href="https://2018.igem.org/Team:BNDS_CHINA/Collaborations">Collaborations</A></DIV></LI><LI class="dropdown"><A class="dropdownButton">Project</A><DIV class="dropdownContent"><A href="https://2018.igem.org/Team:BNDS_CHINA/Description">Description</A><A href="https://2018.igem.org/Team:BNDS_CHINA/Design">Design</A><A href="https://2018.igem.org/Team:BNDS_CHINA/Experiments">Experiments</A><A href="https://2018.igem.org/Team:BNDS_CHINA/Notebook">Notebook</A><A href="https://2018.igem.org/Team:BNDS_CHINA/InterLab">InterLab</A><A href="https://2018.igem.org/Team:BNDS_CHINA/Model">Model</A><A href="https://2018.igem.org/Team:BNDS_CHINA/Demonstrate">Demonstrate</A><A href="https://2018.igem.org/Team:BNDS_CHINA/Attributions">Attribution</A></DIV></LI><LI class="dropdown"><A class="dropdownButton">Parts</A><DIV class="dropdownContent"><A href="https://2018.igem.org/Team:BNDS_CHINA/Parts">Parts Overview</A></DIV></LI><LI class="navButton"><A href="https://2018.igem.org/Team:BNDS_CHINA/Safety">Safety</A></LI><LI class="dropdown"><A class="dropdownButton">Human Practices</A><DIV class="dropdownContent"><A href="https://2018.igem.org/Team:BNDS_CHINA/Human_Practices">Human Practices</A><A href="https://2018.igem.org/Team:BNDS_CHINA/Public_Engagement">Education &amp; Engagement</A></DIV></LI><LI class="dropdown"><A class="dropdownButton">Awards</A><DIV class="dropdownContent"><A href="https://2018.igem.org/Team:BNDS_CHINA/Model">Model</A></DIV></LI><LI class="navButton"><A href="https://igem.org/2018_Judging_Form?team=BNDS_CHINA">Judging Form</A></LI></UL></NAV></HEADER></P><H1>Experiments</H1><H2>I. Primer design</H2><P style="text-indent:2em">Primers are designed by software Snapgene and Oligo 7. The following criteria are met when the most suitable primers are picked out.</P><OL><LI>the Tm value of overlapping with the template should be no more than 55°C;</LI><LI>a hairpin with Tm value less than the annealing temperature could be included; </LI><LI>the absolute value of ΔG of the most stable 3’-dimer is less than 7 kcal/mol. </LI></OL><H2>II. PCR</H2><P style="text-indent:2em">Ingredients are mixed as follow:</P><DIV align="center"><B>Q5 Hot Start High-Fidelity 2X Master Mix System</B></DIV><TABLE align="center" width="80%" border="1"><TBODY><TR><TH>INGREDIENTS</TH><TH>25 µl REACTION</TH><TH>50 µl REACTION</TH><TH>FINAL CONCENTRATION</TH></TR><TR><TD>Forward Primer</TD><TD>1.25 µl</TD><TD>2.5 µl</TD><TD>0.5 µM</TD></TR><TR><TD>Reverse Primer</TD><TD>1.25 µl</TD><TD>2.5 µl</TD><TD>0.5 µM</TD></TR><TR><TD>Template DNA</TD><TD>Variable</TD><TD>Variable</TD><TD>1,000 ng or less</TD></TR><TR><TD>Q5 Hot Start High-Fidelity 2X Master Mix</TD><TD>12.5 µl</TD><TD>25 µl</TD><TD>1X</TD></TR><TR><TD>Nuclease-Free Water</TD><TD>To 25 µl</TD><TD>To 50 µl</TD></TR></TBODY></TABLE><P style="text-indent:2em">The program of PCR is set as:</P><DIV align="center"><B>Thermocycling Conditions for PCR</B></DIV><TABLE width="80%" border="1"><TBODY><TR><TH>STEP</TH><TH>TEMPERATURE</TH><TH>TIME</TH></TR><TR><TD>Initial Denaturation</TD><TD>98°C</TD><TD>30 seconds</TD></TR><TR><TD rowspan="3">35 Cycles</TD><TD>98°C</TD><TD>10 seconds</TD></TR><TR><TD>Tm-5°C</TD><TD>20 seconds</TD></TR><TR><TD>72°C</TD><TD>20-30 seconds/kb</TD></TR><TR><TD>Final Extension</TD><TD>72°C</TD><TD>2 minutes</TD></TR><TR><TD>Hold</TD><TD>16°C</TD><TD>∞</TD></TR></TBODY></TABLE><H2>III. DpnI digestion</H2><P style="text-indent:2em">1 µl DpnI is added into the PCR tube, then the PCR tube is placed into the 37°C incubator for 15-30 minutes.</P><H2>IV. Gel electrophoresis</H2><P style="text-indent:2em">Prepare the gel solution by adding 1 g agarose in to100 ml TAE 1X buffer.</P><P style="text-indent:2em">Heat until all agarose has dissolved completely. </P><P style="text-indent:2em">Add 10µl Gelstain into the cooled solution (at around 60°C), mix the solution evenly through shaking.</P><P style="text-indent:2em">Pour the solution into a cast and place the corresponding comb on top of the gel cast. </P><P style="text-indent:2em">Wait for the gel to solidify (about 30 minutes). </P><P style="text-indent:2em">Transfer the solid gel into the electrophoresis apparatus with TAE 1X buffer submerged.</P><P style="text-indent:2em">Add 10 µl DNA samples and 5 µl TransGene DNA Ladder into cells of the gel. </P><P style="text-indent:2em">Run the electrophoresis at 110 V for 30 minutes.</P><P style="text-indent:2em">Visualize the gel under UV light.</P><H2>V. DNA sequencing</H2><P style="text-indent:2em">When designing primers for sequencing, notice that sequencing primers are one-way with a Tm value at about 60-62°C. The starting point should be 100bp before the restriction site, and one primer can be responsible for sequencing 700 bp accurately. Repeat this process of 700 bp until another restriction site is reached. No additional nucleotides are required for sequencing.</P><H2>VI. Gel Extraction</H2><UL><LI>Objective: Purify DNA from the gel after PCR or after migration
    <TABLE align="center" width="80%" border="1"><TBODY><TR><TH>Volume</TH></TR><TR><TD>Cut DNA band (with gel)</TD><TD>n  (note that 100mg gel=100µl)</TD></TR><TR><TD>GSB buffer</TD><TD>≦3n</TD></TR><TR><TD>isopropanol</TD><TD>n </TD></TR><TR><TD>Nuclease free water (dd H2O)</TD><TD>40 µl</TD></TR></TBODY></TABLE></LI><LI>Weight the EP tube that is going to be used.</LI><LI>Cut the bands, as small as possible, from the gel and put them into separate EP tube. (the gel should be placed under ultraviolet light)</LI><LI>Weight the tube with gel again and calculate the mass for the gel</LI><LI>Add ≦ 3 volume of GSB for 1volume of gel (100 mg gel = 100 µl).</LI><LI>Heat with 55 °C until the gel is entirely dissolved. If the solution shows a color of purple, add sodium acetate (pH=5.2) until the solution shows the same color withe GSB (yellow).</LI><LI>Add 1 volume of isopropanol in order to obtain higher concentration of DNA solution.</LI><LI>Waite until the temperature of mixed solution drops to room temperature.</LI><LI>Put the mix into spin column, centrifuge at 10,000× g (10,000 rpm) for 1min. Discard the effluent.</LI><LI>Place the column into a clean EP tube.</LI><LI>Add 650 µL WB buffer, centrifuge at 10,000× g (10,000 rpm) for 1min. Discard the effluent.</LI><LI>Centrifuge again for 1-2 min in order to remove residual WB.</LI><LI>Place the column into another clean EP tube and heat them with 65 °C heat bath (lid open), until the remaining ethanol evaporate completely. Heat the nuclease free water at the same time.</LI><LI>To elute DNA add 40 µl nuclease free water to the center of the column and centrifuge for 1 min at 13000 rpm.</LI><LI>Use 1 µL of the product for Nanodrop testing concentration.
<P>Product is stored at 20°C.</P><H2>VII. Restriction enzyme double digestion</H2><UL><TABLE align="center" width="80%" border="1"><TBODY><TR><TH>Reagent</TH><TH>Quantity</TH><TH>Final concentration</TH></TR><TR><TD>DNA</TD><TD>0.5~2 µg</TD><TD>c≧1 µg/µl (volume ≦ 5 µl)</TD></TR><TR><TD>Enzyme1</TD><TD>1 µl</TD></TR><TR><TD>Enzyme2</TD><TD>1 µl</TD></TR><TR><TD>10× NEB Cutsmart Buffer</TD><TD>5 µl</TD><TD>1×</TD></TR><TR><TD>Ultrapure water</TD><TD>Add up to 50µl</TD></TR></TBODY></TABLE></UL></LI><LI>General reaction: final volume 50µl</LI><TABLE align="center" width="80%" border="1"><TBODY><TR><TH>Reagent</TH><TH>Quantity</TH><TH>Final concentration</TH></TR><TR><TD>DNA</TD><TD>0.2～1 µg (volume ≦ 5µl)</TD></TR><TR><TD>Enzyme1</TD><TD>0.5 µl</TD></TR><TR><TD>Enzyme2</TD><TD>0.5 µl</TD></TR><TR><TD>10 × NEB Cutsmart Buffer</TD><TD>2.5 µl</TD><TD>1×</TD></TR><TR><TD>Sterile water</TD><TD>Add up to 25 µl</TD></TR></TBODY></TABLE><LI>Small volume reaction: final volume 25 µl
    </LI><LI>Enzyme digestion was carried out 1 hour at 37 °C in incubator. (This time could be extended if it does not work well)</LI><LI>Heat inactivation: heat bath 80 °C for 20 min</LI><LI>This process can also be proceeded in PCR Thermocycle Instrument.</LI></UL><H2>VIII. Ligation</H2><UL><LI>For ligation we have system for both 20µl and 10µl</LI><TABLE align="center" width="80%" border="1"><TBODY><TR><TH colspan="2" align="left">Total volume: 20 µl</TH></TR><TR><TD>Insert</TD><TD>X µl </TD></TR><TR><TD>Backbone</TD><TD>Y µl</TD></TR><TR><TD>T4 DNA ligase</TD><TD>1 µl</TD></TR><TR><TD>10× T4 DNA ligase Buffer</TD><TD>2 µl</TD></TR><TR><TD>Sterile water</TD><TD>Add up to 20 µl</TD></TR></TBODY></TABLE><TABLE align="center" width="80%" border="1"><TBODY><TR><TH colspan="2" align="left">Total volume: 10µl</TH></TR><TR><TD>Insert</TD><TD>X µl </TD></TR><TR><TD>Backbone</TD><TD>Y µl</TD></TR><TR><TD>T4 DNA ligase</TD><TD>0.5 µl</TD></TR><TR><TD>10× T4 DNA ligase Buffer</TD><TD>2 µl</TD></TR><TR><TD>sterile water</TD><TD>Add up to 10 µl</TD></TR></TBODY></TABLE><LI>Note that the ratio of insert to backbone is approximately 3:1. (Total DNA volume should ≦10µl)</LI><LI>All the reactions above require enzyme to be added in the last step.</LI></UL><H2>IX. Gibson Assembly</H2><UL><LI>For Gibson assembly, we use either Gibson Assembly kit or bulk Gibson assembly. The former is not recommended for trial experiments since one kit can only do 10 reactions. It is more effective to use the kit with verified experiment or large amount of cloned gene fragments.</LI><TABLE align="center" width="80%" border="1"><TBODY><TR><TH>Reactant</TH><TH>quantity</TH></TR><TR><TH rowspan="5">Gibson Mix (every 1.2ml)Store at -20 °CSubpackge 15 µl per tubeFor Gibson Assembly kit, add 15 µl Gibson Mix directly (from the kit).</TH><TD>5× ISO buffer (from Wei lab)</TD><TD>320 µl</TD></TR><TR><TD>T5 exonuclease</TD><TD>0.64 µl</TD></TR><TR><TD>Phusion DNA polymerase </TD><TD>20 µl</TD></TR><TR><TD>Taq DNA ligase</TD><TD>16 µl</TD></TR><TR><TD>sterile water （dd H2O）</TD><TD>To 1.2 ml (i.e. in this case 843.4 µl)</TD></TR></TBODY></TABLE><TABLE align="center" width="80%" border="1"><TBODY><TR><TH>Reactant</TH><TH>quantity</TH></TR><TR><TH rowspan="2">DNATotal volume 5 µl</TH><TD>Insert DNA</TD><TD>X (for concentration)</TD></TR><TR><TD>Backbone (plasmid)</TD><TD>3X (for concentration)</TD></TR><TR><TD>Total volume 20 µl</TD></TR></TBODY></TABLE><LI>Reaction: at temperature 50 °C for 1 hour.</LI></UL><H2>X. Bacterial Transformation</H2><OL><LI>Take competent cell from -80°C and place it on ice. DO NOT interrupt.</LI><LI>Use separate EP tube for experiment group, negative control and positive control. Operate on ice.</LI><TABLE align="center" width="80%" border="1"><TBODY><TR><TD>Experiment group</TD><TD>50 µl competent cell + 5 µl ligation/Gibson product</TD></TR><TR><TD>Positive control</TD><TD>25 µl competent cell+ 1 µl vector</TD></TR><TR><TD>Negative control</TD><TD>25 µl competent cell+1µl dd H2O</TD></TR></TBODY></TABLE><LI>Waite 20 min (still on ice). Preheat the heat bath to 42°C </LI><LI>Heat Shock: Heat bath 42°C for 90s.</LI><LI>Replace immediately on ice for 2 min.</LI><LI>Add 150—200µl antibiotic-free LB. This should be done in the clean Bench.</LI><LI>Incubation: under 37 °C, 200—300 rpm for 30—40 min.</LI><LI>Spread the product (competent cell) on the corresponding agar plates. (the petri dishes should be marked).</LI></OL><H2>XI. Colony PCR</H2><P style="text-indent:2em">section system:
   Taq Mix: 12.5 μL
   Forward Primer/ Reverse Primer: 1 μL/1 μL
   Bacteria LB solution: 2 μL
   ddh2O: 10.5 μL
  </P><H2>XII. Plasmid extraction</H2><P style="text-indent:2em">After the sequencing results were confirmed with our constructed plasmid, we used the protocol of Tianprep Mini Plasmid Kit to extract the plasmid.</P><P style="text-indent:2em">The complete protocol can be found here:  <A href="www.tiangen.com/asset/imsupload/up0141604001433139285.pdf">www.tiangen.com/asset/imsupload/up0141604001433139285.pdf</A></P><H2>XIII. C4-HSL Induction</H2><P style="text-indent:2em">Using the similar protocol described (Saeidi2014), after overnight culturing of E. coli in a shaking incubator at 37 °C, the cultures were diluted to OD600nm of 0.002 and allowed to incubate further to OD0.5 under the same condition. Cultures were then transferred into a transparent, flat-bottom 96-well late in triplicate of 200 μl for induction with C4-HSL at varying molar concentrations (1.0E-3, 5.0E-4, 1.0E-4, 5.0E-5, 1.0E-5, 5.0E-6, 1.0E-6, 5.0E-7, 1.0E-7, 5.0E-8, 1.0E-8, 5.0E-9, 1.0E-9, 5.0E-10, 1.0E-10). The loading process is done by step dilution: continuously removing 20ul of solution and adding it into the next well. The plate was incubated at 37 °C with rapid shaking in a microplane reader and assayed for green fluorescence. Time-series fluorescence and OD600NM data were obtained at intervals of 10 min for a total run time of 3 hr. The result was zeroed with supplemented blank culture to remove the background fluorescence and OD600nm. The experimental results were used to build a mathematical model. The distribution of 96-well plate is shown below:</P><TABLE align="center" width="80%" border="1"><TBODY><TR><TD>1.0E-03</TD><TD>1.0E-04</TD><TD>1.0E-05</TD><TD>1E-06</TD><TD>1E-07</TD><TD>1E-08</TD><TD>1E-09</TD><TD>1E-10</TD><TD>5.0E-04</TD><TD>5.0E-05</TD><TD>5.0E-06</TD><TD>5.0E-07</TD></TR><TR><TD>5.0E-08</TD><TD>5.0E-09</TD><TD>5.0E-10</TD><TD>1.0E-03</TD><TD>1.0E-04</TD><TD>1.0E-05</TD><TD>1E-06</TD><TD>1E-07</TD><TD>1E-08</TD><TD>1E-09</TD><TD>1E-10</TD><TD>5.0E-04</TD></TR><TR><TD>5.0E-05</TD><TD>5.0E-06</TD><TD>5.0E-07</TD><TD>5.0E-08</TD><TD>5.0E-09</TD><TD>5.0E-10</TD><TD>1.0E-03</TD><TD>1.0E-04</TD><TD>1.0E-05</TD><TD>1E-06</TD><TD>1E-07</TD><TD>1E-08</TD></TR><TR><TD>1E-09</TD><TD>1E-10</TD><TD>5.0E-04</TD><TD>5.0E-05</TD><TD>5.0E-06</TD><TD>5.0E-07</TD><TD>5.0E-08</TD><TD>5.0E-09</TD><TD>5.0E-10</TD><TD>0</TD><TD>0</TD><TD>0</TD></TR></TBODY></TABLE><H2>XIV. Protein purification</H2><P style="text-indent:2em">Overnight culture of were centrifuged for 10 min with 4500RPM. Then, lysis buffer (0.2M Tris-HCL buffer, pH=7.4), (0.2M Tris-HCL buffer, pH=8.0), and (0.2M Tris-HCL buffer, pH=8.2) were added to F4, SCY, cPcAMP1 respectively and the cells were resuspended. Ultrasound cell disrupter is used to lyse our bacteria (180x for 5s, interval 10s, 180w). After centrifuged for an hour with 120,000 rpm, we suspended the supernatant on the nickel column for three times, which would allow proteins to suspend on the column but no other molecules due to the 6x His tag we added. The unbound proteins were washed with 10mM and 60mM imidazole. Then, a 20mM Tris-Cl solution was used to undergo competitive elution in order to elute our protein. After testing if the protein had been suspended on the nickel column, we ran SDS-PAGE to observe if there were correct protein bands for our protein.</P><H2>XV. SEM for amp effect preparation </H2><P style="text-indent:2em">Aeromonas hydrophila culture is diluted with PBS buffer to OD0.5, and the enzymes (SCY, AMP1 and F4) were diluted to 543ug/ml, 500ug/ml, and 551ug/ml respectively. Each antimicrobial peptide with volume of 500ml was mixed with 500ml of bacteria and centrifuged 5 min with 5000 rpm. The supernatant is removed, and the sample was stored at 4 C for 24hours after adding the stationary liquid.</P><P style="text-indent:2em">The ethanol with respective concentration of 50%, 60%, 80%, 90%, and 100% were used for elution. After a ten minutes interval, the samples were made on a small piece of cover glass and observed using an electron microscope.</P><H2>XVI. The preparation of lysogeny broth (LB)</H2><P style="text-indent:2em">For 1 liter of 1% lysogeny broth (LB), first measure out 10g tryptone, 5g yeast extract, and 10g NaCl (as well as 15g agar if solid medium is required for the experiment). Suspend the solids in deionized water so that the total volume of the mixture is 1 liter. Autoclave the solution at 121℃ for 20 minutes. The lysogeny broth (LB) is ready for use after cooling. If the liquid broth is prepared, preserve it in a 4℃ refrigerator; if the solid broth is prepare, preserve it in a 65℃ incubator to keep it from solidification. Antibiotics such as ampicillin and chloramphenicol must be added after autoclaving, with 1/1000 the volume of the lysogeny broth.</P><H2>XVII. The preparation of tryptic soy broth (TSB)</H2><P style="text-indent:2em">The powder of the tryptic soy broth is obtained from Huankai Microbial, Guangdong. When preparing 1 liter of the medium, first measure out 30g powder and suspend it in deionized water so that the total volume is 1 liter. Stir and heat the mixture until the powder solves in the water completely, and then autoclave the solution at 121℃ for 15 minutes. The tryptic soy broth is ready for use after cooling, or it can also be stored in a 4℃ refrigerator.</P><H2>XVIII. Sodium dodecyl sulfate–polyacrylamide gel electrophoresis (SDS-PAGE)</H2><P style="text-indent:2em">To prepare the separating gel (or low gel) with 12% acrylamide, first measure out all the substances according to the volume required for the electrophoresis (see the chart below) and put them into a beaker to make a solution.</P><TABLE align="center" width="80%" border="1"><TBODY><TR><TH>Vol.(ml)</TH><TH>5</TH><TH>10</TH><TH>15</TH><TH>20</TH></TR><TR><TH>ddH2O</TH><TD>1.0</TD><TD>2.0</TD><TD>3.0</TD><TD>4.0</TD></TR><TR><TH>30% acrylamide/
    bis-acrylamide</TH><TD>2.0</TD><TD>4.0</TD><TD>6.0</TD><TD>8.0</TD></TR><TR><TH>low buffer</TH><TD>1.9</TD><TD>3.8</TD><TD>5.7</TD><TD>7.6</TD></TR><TR><TH>10% Sodium dodecyl sulfate</TH><TD>0.05</TD><TD>0.10</TD><TD>0.15</TD><TD>0.20</TD></TR><TR><TH>10% ammonium persulfate</TH><TD>0.05</TD><TD>0.10</TD><TD>0.15</TD><TD>0.20</TD></TR><TR><TH>TEMED</TH><TD>0.002</TD><TD>0.004</TD><TD>0.006</TD><TD>0.008</TD></TR></TBODY></TABLE><P style="text-indent:2em">Set the casting frame on the stand, and then pour the solution into the gaps between the plates without creating bubbles. Fill isopropyl alcohol to press the separation gel for 20-30 minutes before the gel polymerizes. Next, prepare the solution of stacking gel according to the volume required (see the prescription below).</P><TABLE align="center" width="80%" border="1"><TBODY><TR><TH>Vol.(ml)</TH><TH>2</TH><TH>4</TH><TH>8</TH></TR><TR><TH>ddH2O</TH><TD>1.4</TD><TD>2.7</TD><TD>5.5</TD></TR><TR><TH>30% acrylamide/bis-acrylamide</TH><TD>0.33</TD><TD>0.67</TD><TD>1.3</TD></TR><TR><TH>high buffer</TH><TD>0.25</TD><TD>0.5</TD><TD>1.0</TD></TR><TR><TH>10% SDS</TH><TD>0.02</TD><TD>0.04</TD><TD>0.08</TD></TR><TR><TH>10% APS</TH><TD>0.02</TD><TD>0.04</TD><TD>0.08</TD></TR><TR><TH>TEMED</TH><TD>0.002</TD><TD>0.004</TD><TD>0.008</TD></TR></TBODY></TABLE><P style="text-indent:2em">After the isopropyl alcohol is removed, pour the solution of stacking gel on top of the solid separating gel and insert an appropriate comb. In 20-30 minutes, the gel will polymerize, which is when the gel should be carefully taken out without breaking. Either preserve the gel in water or transfer it immediately to the electrophoresis chamber.</P><P style="text-indent:2em">During the sample preparation part, add 5 μl loading buffer to each 20 μl sample with 5 replications. Heat the sample to 95℃ for 5 minutes, or to 70℃ for 10 minutes alternatively. Load 10 μl of both the prepared samples and the marker to their own wells separately, ensuring that the stacking gel is immersed in the electrophoresis buffer completely. 1 hour and 120V voltage is applied to for the proteins to migrate.</P><H2>XIX. Quorum quenching activity measurement assay</H2><P style="text-indent:2em">Colonies of sensor device I (DH5a), QQ device I (DH5a), QQ device II (DH5a), and the control group (E. coli (DH5a) transformed with empty RPG plasmid) are inoculated into 10 ml of supplemented LB ampicillin, respectively, for overnight culture in a shaking incubator at 37℃. When the cultures are incubated to OD600NM of 0.5, transfer 1 ml of each culture to a 1.5 ml eppendorf (with 27 replications in total for sensor device I (DH5a) and 9 replications each for other cultures), and add 10-4 M C4-HSL to the cultures containing either QQ device I (DH5a) or QQ device II (DH5a). The cultures are then incubated in a laboratory metal bath shaking at 37℃ and 1000 rpm for induction. After 4 hours, centrifuge all the cultures at 13000 rpm for 3 minutes to discard the supernatants of sensor device I (DH5a), and resuspend the precipitates with 1ml of supernatants of QQ device I (DH5a), QQ device II (DH5a), and the control group, respectively. The cultures are put back into the former laboratory metal bath for induction for 3 hours and thereafter transferred into a transparent, flat-bottom 96-well plate for the assay of green fluorescence in a microplate reader. The fluorescence (zeroed with the supplemented LB to remove the effect of background fluorescence) and OD600nm data are obtained for each type of quorum quenching enzyme. The one results in less fluorescence is considered to be more effective in the quorum quenching process.</P><H2>XX. Measurement of C4-HSL by HPLC/MS/MS</H2><OL><LI>Prepare the 1000 mg/L stock solutions by adding 1mg C4-HSL into 1 ml Methanol.</LI><LI>Dilute the 1000 mg/L stock solutions to 50μg/L，100μg/L and 300μg/L standard solutions using methanol as solvent.</LI><LI>Prepare A. hydrophila culture with TSB to reach OD 0.2. Centrifuge the liquid culture and collect the supernatant. Extract C4-HSL in the supernatant with same volume of ethyl acetate. Dry the C4-HSL solution and dissolve it with same volume of methanol.</LI><LI>Employ the Thermo TSQ VANTAGE Instrument for HPLC/MS/MS analysis with the following conditions:
   </LI><LI>Chromotographic column: SymmetryR C18 3.5μm,2.1*150 mm;</LI><LI>Mobile phase：methanol: water ( 1:1)</LI><LI>Flow rate: 0.3 mL/min</LI><LI>MS Scan type: SRM</LI><LI>Collision gas Pressure: 3.0 mTorr</LI><LI>Polarity: Positive</LI><LI>MS Acquire Time: 4 min</LI><LI>Parent Mass: 172.100</LI><LI>Product Mass: 102.000 (used for quantification) </LI><LI>Scan width: 0.002</LI></OL><H1>Reference</H1><P style="text-indent:2em">Saeidi, N., Wong, C. K., Lo, T., Nguyen, H. X., Ling, H., Leong, S. S., . . . Chang, M. W. (2014). 	Engineering microbes to sense and eradicate Pseudomonas aeruginosa, a human 	pathogen. Molecular Systems Biology, 7(1), 521-521. doi:10.1038/msb.2011.55</P></DIV></DIV></DIV></DIV></DIV></BODY></HTML>