<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_Cardiff_Wales_Description skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><DIV id="top_title"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:Cardiff Wales/Description</SPAN></H1></DIV><DIV id="HQ_page"><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><DIV class="menu_wrapper"><DIV class="collapsable_menu_control"> MENU ▤ </DIV><UL id="accordion" class="accordion"><LI class="menu_item"><A href="https://2016.igem.org/Team:Cardiff_Wales">HOME</A></LI><LI class="menu_item"> TEAM
          </LI><LI><A href=" https://2016.igem.org/Team:Cardiff_Wales/Team">TEAM</A></LI><LI><A href="https://2016.igem.org/Team:Cardiff_Wales/Collaborations"> Collaborations </A></LI></UL><LI class="menu_item"> PROJECT
          </LI><LI><A href="https://2016.igem.org/Team:Cardiff_Wales/Description">Cas-Find </A></LI><LI><A href="https://2016.igem.org/Team:Cardiff_Wales/mkeima">FUEL Project </A></LI><LI><A href="https://2016.igem.org/Team:Cardiff_Wales/Results"> Achievements </A></LI><LI><A href="https://2016.igem.org/Team:Cardiff_Wales/protocols">  Protocols </A></LI><LI class="menu_item"><A href="https://2016.igem.org/Team:Cardiff_Wales/Parts">PARTS</A></LI><LI class="menu_item"><A href="https://2016.igem.org/Team:Cardiff_Wales/Safety">SAFETY</A></LI><LI class="menu_item"><A href="https://2016.igem.org/Team:Cardiff_Wales/Attributions">ATTRIBUTIONS</A></LI><LI class="menu_item"> HUMAN PRACTICES
         </LI><LI><A href="https://2016.igem.org/Team:Cardiff_Wales/Human_Practices"> Human Practices </A></LI><LI><A href="https://2016.igem.org/Team:Cardiff_Wales/HP/Silver"> Silver </A></LI><LI><A href="https://2016.igem.org/Team:Cardiff_Wales/HP/Gold"> Gold </A></LI><LI><A href="https://2016.igem.org/Team:Cardiff_Wales/Integrated_Practices">Integrated Practices </A></LI><LI><A href="https://2016.igem.org/Team:Cardiff_Wales/Engagement"> Engagement </A></LI><LI class="menu_item"><A href="https://2016.igem.org/Team:Cardiff_Wales/Medal_Criteria"> AWARDS</A></LI><LI class="menu_item"><A href="https://2016.igem.org/Team:Cardiff_Wales/Notebook">NOTEBOOK</A></LI></DIV><DIV class="content_wrapper"><DIV><H2>'Cas-Find' is a novel bioluminescent system for point-of-care diagnostic testing.</H2></DIV><DIV id="parent"><DIV id="text"><P>Sexually transmitted infections (STI) represent an important issue for global Public health, due to their high-morbidity and prevalence. Laboratory-based tests for STI require specalised infrastructure, equipment and procedures for optimal performance <SUP><A href="">[1]</A></SUP>. As such the utilisation of these tests is generally expensive and time consuming, limiting the availability of results for immediate use in management decisions and potentially impacting on patient prognosis. In addition the majority of STI testing is conducted in resource-constrained environments, where such infrastructure and facilities are unavailable <SUP><A href="">[2]</A></SUP>.'Cas-Find' exploits CRISPR/Cas9 to potentially achieve point-of-care diagnosis alleviating some of the global health burden associated with these diseases.</P><P><B>CRISPR/Cas9 achieves sequence specific interrogation.</B>Most bacteria and archaea possess RNA-mediated adaptive defence mechanisms composed of clustered regularly interspaced short palindromic repeats (CRIPSR) and CRISPR-associated proteins (Cas). Initially exogenous DNA sequences are recognised and incorporated into the bacterial or archaeal genome at the CRISPR loci, transcription through which produces CRISPR RNA (crRNA). An additional trans-activating RNA (tracrRNA) is required for interference, and is partially complementary to this crRNA. In <I>Streptococcus pyogenes</I> this forms a crRNA:tracrRNA <SUP><A href="">[4]</A></SUP> duplex which recruits the endonuclease Cas9, and collectively these molecules achieve sequence specific DNA interrogation and cleavage. This is summarised in Figure 1, demonstrating the sequential acquisition of exogenous DNA, RNA processing and interference. In synthetic applications a single guide RNA (sgRNA) construct fulfills the role of the crRNA:tracrRNA duplex. This sgRNA consists of a 20 nucleotide sequence complementary to the target, a 42 nucleotide Cas9 binding RNA structure, and a 40 nucleotide transcription terminator <SUP><A href="">[5]</A></SUP>. 'Cas-Find' utilises the capability of CRISPR/Cas9 to achieve sequence specific interrogation.</P><P><B>The reconstitution of luciferase activity constitutes a positive signal.</B>Luciferases catalyse bioluminescent reactions using ATP in the presence of molecular oxygen (O<SUB>2</SUB>) and luciferin (LH<SUB>2</SUB>), with <I>in vivo</I> and <I>in vitro</I> applications in imaging and detection. Wild-type luciferases are highly sensitive to pH and are thermolabile, undergoing inactivation and bathochromic shift at 25<SUP>o</SUP>C <SUP><A href="">[7]</A></SUP>. We fused the C- and N- terminal fragments of a thermostable pH-tolerant <I>Photinus pyralis</I> luciferase mutant to a dCas9 isoform optimized for expression in <I>E. coli</I>. dCas9 lacks catalytic activity, and as such targeted sequences do not undergo cleavage. sgRNA constructs target these chimeric proteins to adjacent sequences, resulting in the reconstitution of luciferase activity and bioluminescence in the prescence of luciferin. The reconstitution of bioluminesce significantly greater than background activity constitutes the positive signal for pathogen detection. The mechanism of this 'Cas-Find' system is demonstrated in Fig 2.</P><P><B>'Cas-Find' could provide adaptable, specific point-of-care diagnosis.</B>All constructs were expressed under the T7 promoter and the control of the <I>lac</I> operator. dCas9 chimeric constructs were expressed in both pET16b and pCOLADuet expression vectors (see Fig 2). sgRNA constructs targeted to the 16S rRNA locus of <I>E. coli</I> were expressed in MSCS2 of pCOLADuet, in addition to the pSBC13 expression vector. The design of sgRNA constructs is discussed in more detail below. With further characterisation 'Cas-Find' thus presents a potential novel solution to STI diagnosis, especially where resources are constrained. The adaptable design of sgRNA constructs ensures that the 'Cas-Find' system can be designed to detect specific pathogens, and the requirement for the binding of two sgRNA constructs with a defined distance between them imparts high specificity. Bioluminescence can be detected using low cost and minimal equipment, and could be conducted rapidly at the point-of-care, potentially alleviating some of the global health burden associated with STI.</P></DIV><DIV style:flex-direction:column=""><DIV><P><B>Fig 1. <I>S.pyogenes</I> CRISPR/Cas9.</B></P></DIV><DIV><P><B>Fig 2. 'Cas-Find'</B></P></DIV><DIV><VIDEO width="400px" controls=""><SOURCE src="https://static.igem.org/mediawiki/2016/c/c6/T--Cardiff_Wales--vid1.mp4" type="video/mp4"> Your browser does not support HTML5 video.</SOURCE><P><B>Westminster Presentation.</B></P></VIDEO></DIV></DIV></DIV><H2>Proof of concept <I>in vitro</I> system targeted to the <I>Escherichia coli</I> 16S rRNA locus.</H2><DIV id="parent"><DIV style="order:2;" id="text"><P>Dr. Daniel Pass developed a program to assist in designing sgRNA constructs for non-standard genomic regions and species, to the specifications required for this system. Here, we targeted the  <I>E. coli</I> 16S rRNA locus in order to facilitate proof-of-concept testing of our in vitro system. The design of these constructs was achieved using a <A href="http://github.com/passdan/scriptdrop/blob/master/cas9_targeter.py">Python script</A> developed to find appropriate paired regions from a FASTA formatted genomic DNA region for paired target sequences using <A href="http://www.clontech.com/GB/Products/Genome_Editing/CRISPR_Cas9/Resources/Designing_sgRNA">guidelines</A> from Takara Bio USA alongside additional sources.The script initially identifies proto-spacer adjacent motif (PAM) sequences (5'-NGG-3') in this FASTA sequence in forward and reverse. The sgRNA sequence is complementary to the 20 nucleotides upstream of the PAM sequence, after accounting for other enhancement features. Viable pairs within a defined range of each other are selected and passed to BLASTn to test for simple alignment against a reference dataset. This would include the remainder of the species genome, and also multiple cross-reactive species. The output is a FASTA file of potential probe pairs, a table.txt file of the same information in a graphical representation, and the results of the blast search, to aid in choosing probes which do not demonstrate cross-reactivity </P><P>We designed one forward (F1) and three reverse (R1, R2 and R3) sgRNA constructs targeted to the <I>E. Coli</I> 16S rRNA locus using this program. Differing coexpression conditions could thus facilitate the characterization of background reconstitution at differing genomic distances. This builds on similar experimentation by <A href="https://2013.igem.org/Team:MIT/Venus">MIT</A> in 2013, and could contribute to future work with similar split reporter systems. This locus was selected due to its high copy number, promoting a strong signal during initial testing.</P></DIV><DIV style:flex-direction:column=""><DIV><P><B>Fig 3. sgRNA distance characterisation. </B></P></DIV></DIV></DIV><DIV style="width:100%;"><H2>Bibliography</H2><DIV><P><A name="1"><SUP>1</SUP></A> - Peeling, R.W., Holmes, K.K., Mabey, D. (2006) Rapid tests for sexually transmitted infections (STI's): the way forward.<I>Sexually Transmitted Infections</I>.<B>82:</B>1-6.</P></DIV><DIV><P><A name="2"><SUP>2</SUP></A> - Santrach, P.J. (2007) Current and Future Applications of Point of Care Testing. Mayo Clinic. [Online] Available at: http://wwwn.cdc.gov/cliac/pdf/addenda/cliac0207/addendumf.pdf [Accessed: 18 October 2016] </P></DIV><DIV><P><A name="3"><SUP>3</SUP></A> - Unemo M. (2013) Laboratory diagnosis of sexually transmitted infections, including human immunodeficiency virus. <EM>WHO.  </EM></P></DIV><DIV><P><A name="4"><SUP>4</SUP></A> - Jinek, M., Chylinsji, K., Fonfara, I., Hauer, M. Doudna, J., Charpentier, E. (2012) A Programmable Dual-RNA_Guided DNA Endonuclease in Adaptive Bacterial Immunitity. <I>Science.</I><B>337:</B>816-821.</P></DIV><DIV><P><A name="5"><SUP>5</SUP></A> - Larson, M., Gilbert, L.,Wang, X., Lim, W., Weissman, J., Qi, L. (2013)  	
CRISPR interference (CRISPRi) for sequence-specific control of gene expression. <I>Nature Protocols.</I><B>8:</B>2180-2196.</P></DIV><DIV><P><A name="6"><SUP>6</SUP></A> - Jathoul, A., Law, E., Gandelman, O., Pule, M., Tisi, L., Murray, J. (2012) Development of a pH-tolerant Thermostable <I>Photinus pyralis</I> Luciferase for Brighter <I>In Vivo</I> Imaging. <I>Bioluminescence - Recent Advances in Oceanic Measurements and Laboratory Applications.</I></P></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></BODY></HTML>