<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_Cardiff_Wales_mKeima skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><DIV id="top_title"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:Cardiff Wales/mKeima</SPAN></H1></DIV><DIV id="HQ_page"><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><DIV class="menu_wrapper"><DIV class="collapsable_menu_control"> MENU ▤ </DIV><UL id="accordion" class="accordion"><LI class="menu_item"><A href="https://2016.igem.org/Team:Cardiff_Wales">HOME</A></LI><LI class="menu_item"> TEAM
          </LI><LI><A href=" https://2016.igem.org/Team:Cardiff_Wales/Team">TEAM</A></LI><LI><A href="https://2016.igem.org/Team:Cardiff_Wales/Collaborations"> Collaborations </A></LI></UL><LI class="menu_item"> PROJECT
          </LI><LI><A href="https://2016.igem.org/Team:Cardiff_Wales/Description">Cas-Find </A></LI><LI><A href="https://2016.igem.org/Team:Cardiff_Wales/mkeima">FUEL Project </A></LI><LI><A href="https://2016.igem.org/Team:Cardiff_Wales/Results"> Achievements </A></LI><LI><A href="https://2016.igem.org/Team:Cardiff_Wales/protocols">  Protocols </A></LI><LI class="menu_item"><A href="https://2016.igem.org/Team:Cardiff_Wales/Parts">PARTS</A></LI><LI class="menu_item"><A href="https://2016.igem.org/Team:Cardiff_Wales/Safety">SAFETY</A></LI><LI class="menu_item"><A href="https://2016.igem.org/Team:Cardiff_Wales/Attributions">ATTRIBUTIONS</A></LI><LI class="menu_item"> HUMAN PRACTICES
         </LI><LI><A href="https://2016.igem.org/Team:Cardiff_Wales/Human_Practices"> Human Practices </A></LI><LI><A href="https://2016.igem.org/Team:Cardiff_Wales/HP/Silver"> Silver </A></LI><LI><A href="https://2016.igem.org/Team:Cardiff_Wales/HP/Gold"> Gold </A></LI><LI><A href="https://2016.igem.org/Team:Cardiff_Wales/Integrated_Practices">Integrated Practices </A></LI><LI><A href="https://2016.igem.org/Team:Cardiff_Wales/Engagement"> Engagement </A></LI><LI class="menu_item"><A href="https://2016.igem.org/Team:Cardiff_Wales/Medal_Criteria"> AWARDS</A></LI><LI class="menu_item"><A href="https://2016.igem.org/Team:Cardiff_Wales/Notebook">NOTEBOOK</A></LI></DIV><DIV class="content_wrapper"><H1>FUEL Project</H1><H1>Overview</H1><P>As a link to our use of luciferases in the <A href="https://2016.igem.org/Team:Cardiff_Wales/Cas-Find">Cas-Find</A> project we developed an interest in biological imaging. This was largely influenced by the research of our Secondary PI, Amit Jathoul in this general area. </P><P>One challenge in the field of biological imaging is in the use of fluorescent and bioluminescent proteins that emit light with longer wavelengths. The mKeima RFP variant has been previously used as a long stoke shifted fluorophore and was the subject of the <A href="https://2013.igem.org/Team:UCL_PG"> SPECTRA project of the 2013 UCL iGEM team </A>.
</P><P>Therefore as a side project we planned to alter the potential usage of the Lux Operon biobrick (Cambridge iGEM 2010 <A href="http://parts.igem.org/Part:BBa_K325909">BBa_K325909</A>) by inducing a red-shift in its bioluminescence via an interaction with the mKeima protein. 
</P><P>We hoped to show that blue light produced by <I>E.coli</I> expressing the native LUXoperon would excite the mKeima protein resulting in the emission of red light.  
</P><P>We aimed to analyse the interaction between LUXoperon and mKeima in two ways:
<LI><B>1. Co-expression:</B> Express the LUXoperon and mKeima in different bacterial cells and assess whether the <A href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4207116/">Fluorescence by Unbound Excitation from Luminescence (FUEL) reaction</A> was able to cause a change in the wavelength of the light output. The gene expression in both bacteria is controlled by the arabinose inducible promotor pBAD.</LI></P><P><LI>2. <B>Co-regulation:</B> Addition of a rbs-mKeima sequence to the 3’ end of the native <A href="http://parts.igem.org/Part:BBa_K325909">pBAD::LUXoperon</A>. We aimed to assess whether there was an alteration in wavelength of the light output in these bacteria compared to bacteria contained an unaltered LUXoperon. 
</LI></P><H2>Experimental Design</H2><P>The mKeima protein has been added to the registry by the UCL_PG team of 2013 (<A href="http://parts.igem.org/Part:BBa_K1135001">BBa_K1135001</A>). However as part of this side project we have submitted two unique parts to the registry that contain this protein:
</P><P><LI> 1. pBAD::LUXoperon:mKeima fusion (<A href="http://parts.igem.org/Part:BBa_K2060002">BB_K2060002</A>)</LI></P><P><LI> 2. pBAD::mKeima (<A href="http://parts.igem.org/Part:BBa_K2060001">BB_K2060001</A>).</LI></P><P>We aimed to grow these biobrick-containing bacteria in the presence of arabinose to induce expression of each construct and then measure the fluorescence and bioluminescence of the bacteria to assess:
</P><P><LI>1. Whether the proteins were correctly expressed by analysis of the appropriate spectra</LI></P><LI>2. Whether there is a change in light output when different bacterial cultures were mixed prior to analysis of appropriate spectra</LI><P>Initially we sub-cultured overnight bacterial cultures (1/10) and grew for 3hours prior to induction of gene expression by addition of arabinose at various concentrations for approximately 6hours.</P><P>Following induction we used a Cary XXXX to measure fluorescence and bioluminescence across appropriate spectra. The LUXoperon has been previously shown to <A href=" https://2012.igem.org/Team:Tokyo-NoKoGen/Project/lux_operon "> emit light at 488nm </A> whilst the emission wavelength of mKeima is 620nm. We also used a non-biobrick construct expressing pBAD::sfGFP as a control, for which  the emission wavelength is 510nm.

</P><H2>Results</H2><P>Firstly we grew biobrick-containing E.coli overnight and after sub-culturing (1/10) grew for 2hr before induction with mM concentrations of arabinose.
</P><P>Figure 1 shows that at these concentrations (5-20mM) of arabinose, bacteria containing the native LUXoperon showed a predicted bioluminescence spectra. However none of the other bacteria were bioluminescent. 
</P><P>Figure 2 shows that at these concentrations (5-20mM) of arabinose, bacteria containing sfGFP showed a predicted fluorescent spectra. However none of the other bacteria showed any fluorescence across the range of tested wavelengths.


</P><P><A href="http://www.pnas.org/content/suppl/2010/03/03/0914365107.DCSupplemental/pnas.0914365107_SI.pdf">Previous work</A> has demonstrated that mKeima expression under the pBAD promotor was best at lower concentrations of arabinose. Therefore we grew bacterial sub-cultures for 6hr in 100uM and 250uM Arabinose (Figure 3 and 4). Unfortunately this did also not show bioluminescence or fluorescence that was suggestive of mKeima expression. </P><P>Finally we attempted to stimulate expression of the mKeima-containing biobricks <A href="http://parts.igem.org/Part:BBa_K2060001">BB_K2060001</A> and <A href="http://parts.igem.org/Part:BBa_K2060002">BB_K2060002</A> by growing bacterial cultures overnight and spiking in arabinose at 10mM for 2hr before measurement. </P><P>Figure 5 demonstrates that the <A href="http://parts.igem.org/Part:BBa_K2060002">LUXoperon-mKeima </A> gave a bioluminescent blue-light output, indicating that the LUXoperon is working correctly. However figure 6 shows that there is no inducible fluorescence generated by these bacteria. This indicates that further work is necessary to stimulate effective expression of both the LUX components and mKeima from the same operon. </P><H2>Future plans</H2><P> iGEM headquarters recently published a paper demonstrating the challenges in expression of fluorescent proteins, 
<A href=" http://journals.plos.org/plosone/article?id=10.1371/journal.pone.0150182”&gt; http://dx.doi.org/10.1371/journal.pone.0150182 &lt;/a&gt;. We had clear difficulties expressing mKeima in bacteria containing either biobricks &lt;a href=" http:="">BB_K2060001</A> or <A href="http://parts.igem.org/Part:BBa_K2060002">BB_K2060002</A>. Sadly we ran out of time to conduct further analysis but might would recommend the following experiments:</P><LI>1. Induce expression at lower temperatures (18C or 22C) at varying concentrations of Arabinose (nM-mM range)</LI><LI>2. Addition of arabinose at different times in the bacterial growth cycle.</LI><LI>3. Vary timings post growth to allow for protein maturation</LI><P>We hope that future iGEM teams will take advantage of the availability of these biobricks as potentially very useful tool for the analysis of gene expression. With increased time the development of these tools will be of benefit for the entire iGEM community.</P><TITLE>Cardiff_Wales</TITLE><DIV id="footer"><A id="page_top" href="#">Top</A><DIV><A href="https://twitter.com/igem_cardiff" class="twitter-follow-button" data-show-count="false">Follow @igem_cardiff</A></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></BODY></HTML>