Team:Cardiff Wales/Results
MENU â–¤
HOME
TEAM
TEAM
Collaborations
PROJECT
Cas-Find
FUEL Project
Achievements
Protocols
PARTS
SAFETY
ATTRIBUTIONS
HUMAN PRACTICES
Human Practices
Silver
Gold
Integrated Practices
Engagement
AWARDS
NOTEBOOK
Results
Projects
Cas-Find
Cas-Found
IPTG inducible CRISPR-Cas9 guide RNA targeted to rRNA Reverse
Investigated viable systems for future investigation
Can't-Find
Use of vector (pET16B) with non-expected insert doomed 1/3 of our potential constructs
Failure to clone Cas-LucC/N & into E.Coli
As a result we have been unable to study the viablity of our proposed diagnostic tool
FUEL Project
Fuel Up
We have successfully cloned and sequenced our arabinose-inducible pBAD:mKeima and pBAD:Lux Operon:mKeima constructs
We have demonstrated that the LuxOperon portion of LUXoperon:mKeima of the fusion construct is functional
Running on Empty
We have been unable to show a red light output from the mKeima constructs
We have therefore been unable to observe any red shifting of the light output generated by the Lux Operon
The inaugural year of Cardiff iGEM has been an excellent learning experience. There is no doubt that next year we will be in a much stronger position to succeed in all aspects of the our project. We have already set the wheels in motion for a Cardiff_Wales iGEM 2017.
Therefore what did we specifically learn from this years iGEM project?
We would likely use the different pSB1C3 antibiotic resistance variants instead of the three distinct plasmids we chose to use (pSB1C3, pET16B, pCOLA-DUET). This would improve the control of copy number and help maintain uniform expression of our constructs. We were unaware of the different pSB1C3 variants due to our focus on cloning biobricks into the registry-compliant chloramphenicol resistant pSB1C3.
A number of our primers did not work as predicted and meant that we often couldn't inspect our transformed cultures without using direct sequencing. Next time we will set more time and resources to checking all our primers work appropriately, although within the time-scale of a ten-week project this can be challenging as by the time you realise there is a problem, the project has significantly progressed.
When we started our lab work, the whole team was involved. This has the effect of making the work a bit fractured and disjointed, not to mention complicating the management of samples. In discussion with other teams it became apparent that it was more routine to have only a few members designated to lab duties. We therefore decided to reduce the number of us in the lab, and split off into two groups with each one focusing on a project each. This is something we look to do from the start in 2017.
We didn't start our FUEL project until after half of our lab time was up, this really restricted the amount of time we could devote to it. Next time we would aim to split into two lab groups that start at the same together but work independently, which again would improve the management of lab work.
Collaborations
Provided Oxford with FLIM analysis of their copper chelators.
We have quantified ATP production from cells transformed with Washington's phosphoenolpyruvate kinase (PCK) and phosphoglycerate kinase (PGK) constructs
Unfortunately we were unable to practically collaborate with Saclay due to failures in our related dCas9. In future years we aim to increase our number of collaborations. We dedicated most of our lab time to cloning our own constructs, which meant there wasn't much time, or manpower left for collaborations.
