Team:Cardiff Wales/Parts
MENU â–¤
HOME
TEAM
TEAM
Collaborations
PROJECT
Cas-Find
FUEL Project
Achievements
Protocols
PARTS
SAFETY
ATTRIBUTIONS
HUMAN PRACTICES
Human Practices
Silver
Gold
Integrated Practices
Engagement
AWARDS
NOTEBOOK
Parts
PART NAME
PART NUMBER
IPTG inducible CRISPR-Cas9 guide RNA targeted to rRNA Reverse
BBa_K2060000
Arabinose inducible mKeima fluorophore
BBa_K2060001
Arabinose inducible LUXoperon::mKeima
BBa_K2060002
IPTG inducible CRISPR-Cas9 guide RNA targeted to rRNA Forward
In Preparation
dCas9-LUC N-terminal fragment
In Preparation
dCas9-LUC C-terminal fragment
In Preparation
Part Description
The submitted parts are described on the appropriate URL, please click on those links for further information.
Parts 'In preparation'
IPTG inducible CRISPR-Cas9 guide RNA targeted to rRNA Forward
This part was designed using the code generated by Dr Daniel Pass in order to interrogate the E.Coli 16S rRNA locus for appropriate Cas9 target sites.
The sequence is below has the Forward guide sequence and the General Scaffold sequences highlighted.
This was synthesised as an IDT gBlock as a single fragment and we aimed to directly clone this fragment into pSB1C3 using EcoRI and PstI. However we were unable to identify a clone with the correct sequence. Therefore this aspect of the project will continue during future related research.
GTCATCCTCTCAGACCAGCTGTTTTAGAGCTAGAAATAGCAAGTTAAAATAAGGCTAGTCCGTTATCAACTTGAAAAAGTGGCACCGAGTCGGTGCTTTTTT
dCas9-LucN-terminal and dCas9-LucC-terminal fragments
We took an E.coli codon-optimised version of Cas9 from the registry ( BBa_K1774001 ) and added the nucleotide changes to bring about the D10A and H840 amino acid alterations that remove nuclease activity. The alteration of these amino acid changes has been previously demonstrated by the MIT 2013 iGEM team to allow the dCas9 to be directed to an appropriate target sequence but to not cut at the sequence.
Following discussions with Secondary PI Dr Amit Jathoul we aimed to use the sequence of the modified Luciferase X11 enzyme that has been shown to provide a stronger light output when incubated with luciferin. We split the luciferase x11 sequence into N-terminal and C-terminal fragments and planned to attach these to the dCas9 sequence using a short linker sequence (agcagggctgaccccaagaaAaagaggaaggtAtgCggCggttctggGggAggttcAggt)
We synthesized the following IDT gBlocks:
dCas9: 1- 1368 bp
dCas9: 1369- 2739 bp
dCas9: 2738- 4101 bp
LucN: 1-1248 bp
LucC: 1249- 1641 bp
Our plan was to use NEB HiFi Assembly to generate clones that contained the entire dCas9 sequence fused to either LucN (dCas9-LucN) or LucC (dCas9-LucC). We would then use restriction enzyme digestions to introduce these into appropriate expression plasmids.
Unfortunately we were unable to generate appropriate clones using this strategy. This was most likely because of the difficulty in having a two-step process that did not allow analysis of the first step before the second was begun. The first step involved a challenging four-DNA-piece assembly into a single clone, which was then followed by a restriction digest using relatively large DNA fragments at low concentrations.
Later in the project we altered our cloning strategy to utilize a previously developed dCas9 biobrick ( BBa_K1218011 ). In this strategy we aimed to use NEB HiFi assembly to directly attach the LucN or LucC fragments to the dCas9 enzyme before moving these clones to appropriate expression vectors.
Unfortunately in our available time we were unable to make progress on this second strategy but the later development of this project will use this future cloning plan.
Cardiff_Wales
Top
Follow @igem_cardiff
