Team:UAlberta/Improve
Project
Description Design Results Demonstrate BeeLab InterLab
Parts
Overview Improve
Human Practices
Overview Integrated Human Practices Inclusivity Public Engagement
Notebook
Lab Notebook Protocols Safety
Team
Members Collaboration Attribution Awards Sponsors
Improve a Part / Project
Improved BioBrick Part: BBa_K2779912
Team UAlberta’s cloning work relied heavily on the use of fluorescent markers to assemble our desired constructs, either with fluorescence serving as the background or by incorporating fluorescence into our desired constructs. Additionally, as part of our characterization of our engineered pathway, we wanted to be able to express each enzyme and purify it for in vitro characterization and experimentation.
To accomplish this, Team UAlberta decided to use an araBAD-based expression system because:
It is inducible by L-arabinose and is tightly regulated by araC
It consists of a strong promoter  for increased expression
It is compatible with our preferred workhorse strain, DH10B, allowing us to perform our cloning and expression work in the same strain
It is more cost-effective since L-arabinose is more economical than IPTG.
When we initially looked through the repository, we found that while BBa_K1602055, which had GFP under the control of araC/pBAD, was the closest to our needs, there was no easy way for us to replace their inserted GFP with our desired inserts, and there was no way for us to collect the expressed enzymes.
Figure 1: A schematic of BBa_K1602055.
Figure 2: A schematic of our our submitted part, BBa_K2779912 which aims to improve upon BBa_K1602055.
As such, we are submitting this BioBrick (Figure 2), based out of Invitrogen’s pBAD/His expression vector, as an improvement from BBa_K1602055. The biggest improvement of this system from BBa_K1602055 is the inclusion of the 6xHis tag at the N-terminus of the protein of interest, allowing for easy purification. Invitrogen’s inclusion of the T7 tag gene leader also helps promote translation in E. coli, increasing efficiency.
When deciding how to insert GFP (and consequently, other genes of interest) to assemble our screening plasmid, we wanted to make our system as flexible as possible. While our main objectives for this construct are to enable the expression and purification of each of our enzymes and to facilitate the genes’ insertion into this system, we recognize that some users may be interested in retaining GFP and using it to tag their proteins of interest. Thus, we decided to flank the GFP with a unique restriction enzyme recognition site upstream while keeping the BioBrick suffix immediately downstream. This choice serves the dual purpose of facilitating the replacement of the GFP with any other gene of interest with the minimum number of additional restriction digest enzymes needed, and retaining the possibility of using this construct to fluorescently tag other proteins of interest at the C-terminus. Next, our choice of the unique restriction enzyme had to consider several factors. First, we wanted to eliminate the illegal restriction enzymes found in the MCS of pBAD, narrowing our options down to BamHI, XhoI, SacI, or BglII as they appear in the MCS. Out of these possibilities, we chose XhoI by process of elimination. First, BglII cannot be heat inactivated, and XhoI is more economical than SacI from both NEB and Thermo Fisher, narrowing our options down to BamHI and XhoI. Despite BamHI being more cost-effective, we decided to insert GFP using XhoI,  because this allows users to easily tag their gene of interest with GFP by ligating their gene of interest using the BamHI/XhoI restriction sites.
The last feature of our construct is the introduction of the A206K mutation in our variant of GFP, which has been shown to prevent dimerization of GFP at high concentrations thus making it truly monomeric. GFP’s tendency to dimerize without this mutation can be problematic for experiments involving GFP-tagged proteins as dimerization can lead to undesired aggregation of the proteins of interest. This aggregation can interfere with experiments, for example, by creating artifacts during imaging. Using a GFP variant carrying the A206K mutation should reduce, if not completely eliminate, these unwanted aggregation events.
A keynote regarding this construct (whether used to tag a protein of interest with GFP, or to replace GFP with a gene of interest) is that translation of the 6xHis tag occurs in the +2 frame. This means that to ensure proper fusion of the 6xHis tag to your protein of interest, users must include one spacer nucleotide between the restriction enzyme recognition site and the first codon of the gene of interest.
Figure 3: Final result of protein expression and purification using Ni-NTA. The GFP from our construct (right) can be successfully collected by and eluted from Ni-NTA, while BBa_K1602005 cannot.
Figure 4:Extracted mCherry prepared for the OLS Canmore iGEM Team
Our main objective for designing this construct is to have a screening plasmid in which we can easily clone our desired genes in for protein expression and purification. Thus, we carried out protein expression and purification experiments in parallel for BBa_K1602005 and our construct to demonstrate the functionality of our 6xHis tag. We successfully demonstrated that when compared to BBa_K1602005, our construct allows for increased expression and purification of GFP (Figure 3). While we did not directly use the GFP purified in this experiment, this design was used to express mCherry for OLS Canmore (Figure 4), as well as each of our individual enzymes for our pathway. We would also like to highlight the fact that tagging proteins of interest with fluorescent proteins like GFP facilitate expression and purification as there are visual cues that can indicate the quality of expression and purification.
Project
Description
Design
Results
Demonstrate
BeeLab
InterLab
Parts
Overview
Improve
Human Practices
Overview
Integrated Human Practices
Inclusivity
Public Engagement
Notebook
Lab Notebook
Protocols
Safety
Team
Members
Collaboration
Attribution
Awards
Sponsors
