<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_UAlberta_Design skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><DIV id="top_title"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:UAlberta/Design</SPAN></H1></DIV><DIV id="HQ_page"><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><DIV class="ua-navbar"><DIV class="container "><NAV class="navbar navbar-expand-lg navbar-dark"><DIV class="collapse navbar-collapse" id="navbarSupportedContent"><UL class="navbar-nav ml-auto"><LI class="nav-item dropdown"><A class="nav-link dropdown-toggle" href="https://2018.igem.org/Team:UAlberta" id="navbarDropdown" role="button" data-toggle="dropdown" aria-haspopup="true" aria-expanded="false">
                 Project
               </A><DIV class="dropdown-menu" aria-labelledby="navbarDropdown"><A class="dropdown-item" href="https://2018.igem.org/Team:UAlberta/Description">Description</A><A class="dropdown-item" href="https://2018.igem.org/Team:UAlberta/Design">Design</A><A class="dropdown-item" href="https://2018.igem.org/Team:UAlberta/Results">Results</A><A class="dropdown-item" href="https://2018.igem.org/Team:UAlberta/Demonstrate">Demonstrate</A><A class="dropdown-item" href="https://2018.igem.org/Team:UAlberta/BeeLab">BeeLab</A><A class="dropdown-item" href="https://2018.igem.org/Team:UAlberta/InterLab">InterLab</A></DIV></LI><LI class="nav-item dropdown"><A class="nav-link dropdown-toggle" href="#" id="navbarDropdown" role="button" data-toggle="dropdown" aria-haspopup="true" aria-expanded="false">
                 Parts
               </A><DIV class="dropdown-menu" aria-labelledby="navbarDropdown"><A class="dropdown-item" href="https://2018.igem.org/Team:UAlberta/Parts">Overview</A><A class="dropdown-item" href="https://2018.igem.org/Team:UAlberta/Improve">Improve</A></DIV></LI><LI class="nav-item dropdown"><A class="nav-link dropdown-toggle" href="#" id="navbarDropdown" role="button" data-toggle="dropdown" aria-haspopup="true" aria-expanded="false">
                 Human Practices
               </A><DIV class="dropdown-menu" aria-labelledby="navbarDropdown"><A class="dropdown-item" href="https://2018.igem.org/Team:UAlberta/Human_Practices">Overview</A><A class="dropdown-item" href="https://2018.igem.org/Team:UAlberta/IntegratedHP">Integrated Human Practices</A><A class="dropdown-item" href="https://2018.igem.org/Team:UAlberta/Inclusivity">Inclusivity</A><A class="dropdown-item" href="https://2018.igem.org/Team:UAlberta/Public_Engagement">Public Engagement</A></DIV></LI><LI class="nav-item dropdown"><A class="nav-link dropdown-toggle" href="#" id="navbarDropdown" role="button" data-toggle="dropdown" aria-haspopup="true" aria-expanded="false">
                 Notebook
               </A><DIV class="dropdown-menu" aria-labelledby="navbarDropdown"><A class="dropdown-item" href="https://2018.igem.org/Team:UAlberta/Notebook">Lab Notebook</A><A class="dropdown-item" href="https://2018.igem.org/Team:UAlberta/Protocols">Protocols</A><A class="dropdown-item" href="https://2018.igem.org/Team:UAlberta/Safety">Safety</A></DIV></LI><LI class="nav-item dropdown"><A class="nav-link dropdown-toggle" href="#" id="navbarDropdown" role="button" data-toggle="dropdown" aria-haspopup="true" aria-expanded="false">
                 Team
               </A><DIV class="dropdown-menu" aria-labelledby="navbarDropdown"><A class="dropdown-item" href="https://2018.igem.org/Team:UAlberta/Team">Members</A><A class="dropdown-item" href="https://2018.igem.org/Team:UAlberta/Collaborations">Collaboration</A><A class="dropdown-item" href="https://2018.igem.org/Team:UAlberta/Attributions">Attribution</A><A class="dropdown-item" href="https://2018.igem.org/Team:UAlberta/Awards">Awards</A><A class="dropdown-item" href="https://2018.igem.org/Team:UAlberta/Sponsors">Sponsors</A></DIV></LI></UL></DIV></NAV></DIV></DIV><DIV class="ua-research-content"><DIV class="container"><H1>Design</H1><DIV class="row"><DIV class="col-lg-6"><H2>Overview</H2><P>The ultimate goal we aimed to achieve when designing our constructs was to enable the overproduction of protoporphyrin IX (PPIX) in an <I> Escherichia coli</I> chassis. Previous literature has demonstrated that different intermediates in the heme biosynthetic pathway, including protoporphyrin IX, could be overproduced by increasing expression of intermediate enzymes [1][2]. These studies determined that overproduction of protoporphyrin IX (PPIX), our target intermediate in the biosynthesis pathway, could be produced at high yields (~80 μM) by overexpressing the genes <I> hemA, hemB, hemC, hemD¸ hemE,</I> and<I> hemF</I> in combination. Thus, Team UAlberta sought to design a unified system which packages the requisite <I>hemA</I>  through <I>hemF</I> genes for the express purpose of overproducing PPIX and aim to improve on previous approaches.</P></DIV><DIV class="col-lg-6"><H3>Heme Synthesis Pathway</H3><P>Heme and other metalloporphyrins are ubiquitous in nature as they are integral parts of many enzymes including various cytochromes and hemoglobin. There are two major routes which heme is produced which and it is determined by how D-aminolevulinic acid (ALA) is obtained [3]:</P><UL><LI>The Shemin-pathway in which ALA is produced in one reaction between glycine and succinyl-CoA.</LI><LI>The C5-pathway in which ALA is produced by two reactions involving a  glutamyl-tRNA precursor.</LI></UL><P>In most bacteria, including <I>E. coli</I>, the prevalent pathway is the C5-pathway, though, the intermediates and steps following the production of ALA are the same in both pathways. A schematic of the heme synthesis pathway is outlined in Figure 1.</P></DIV></DIV><DIV class-=""><DIV class="col-lg-12 align-self-center"><FIGURE class="figure"><FIGCAPTION class="figure-caption"><STRONG>Figure 1:</STRONG>The two main heme biosynthesis pathways where the first step in (1.i) the C5-pathway and (1.ii) the Shemin-pathway are explicitly denoted. The remaining steps (2-8) depict the steps found in the <I>E. coli</I> endogenous pathway and its associated genes. The first compound produced in this schematic is ALA and the last step depicts heme. Our molecule of interest is PPIX is the last intermediate produced before heme.
</FIGCAPTION></FIGURE></DIV></DIV><H3>Producing PPIX</H3><P>Studies have shown that a combination of <I>hemA, B, C, D, E, F</I> can push flux through the heme synthesis pathway beyond natural levels, and high titres of up to 80 uM of PPIX can be obtained from <I>E. coli. </I>[1][2]. However, in these studies, the initial two steps of the C5-pathway, mediated by two enzymes encoded by <I>hemA</I> and <I>hemL</I> of <I>E. coli</I> are bypassed by the introduction of the <I>hemA</I> gene from <I>Rhodobacter capsulatus.</I></P><P><I>hemA</I> from <I>R. capsulatus</I>, encodes for the aminolevulinic acid synthase of the Shemin-pathway, which <I>E. coli</I> was able to express in these previous studies. The reasons cited for the motivation behind using this alternate route in <I>E. coli</I> include reducing the number of steps needed to generate ALA, from two to one, and that high utilization of the glutamyl-tRNA precursor in the C5-pathway may but a load on metabolism as a component used in protein synthesis is being directly competed for.</P><P>With these previous results as a motivation, Team UAlberta also sought to overexpress these six genes in E. coli to fulfill our first objective for APIS: the engineering of a PPIX-producing strain. Thus, we present our design for APIS, which is a six-gene system optimized for inducible expression in <I>E. coli</I>!</P><H2>APIS: Gene Design<DIV class="row"><DIV class="col-lg-12 align-self-center"><FIGURE class="figure"><FIGCAPTION class="figure-caption"><STRONG>Figure 2:</STRONG> A schematic of the full gene design for APIS. Our design incorporates six genes in a non-linear fashion to increase cloning, transformation efficiency, and expression!</FIGCAPTION></FIGURE></DIV></DIV><P>This design required the insertion of <I>hemA</I> through <I>hemF</I> into a vector for transformation into the target <I>E. coli</I> strain. With this in mind, the pathway was engineered based on 4 key design considerations:</P><OL><LI>Minimizing insert size</LI><LI> Maximizing cloning efficiency</LI><LI>Optimized protein expression</LI><LI>Inducibility</LI></OL></H2><H3>Design Considerations</H3><P>First, the size of the construct was to be minimized. Given the size of the genes in question,  a linearly-assembled six-gene system would be prohibitively long, as large plasmids are more difficult for bacteria to accept and maintain. This linear approach also was utilized by the <A href="“https://2007.igem.org/BerkiGEM2007Present3”">2007 UC Berkely iGEM team</A>. However, minimizing the size would facilitate more efficient transformation and would minimize the energetic stress this system would place on the<I>E. coli</I> host.Reducing the overall construct size will allow us to fit our entire system into one plasmid unlike the previously cited designs which used two-plasmid systems [1][2].</P><P>Second, in order to accommodate the tight timeline of the project and mitigate the potential pitfalls in the assembly of such a complex system, checks had to be incorporated into the design to ensure that the assembly and cloning were proceeding efficiently and as intended.</P><P>Third, given that the success of the system is predicated on the continuous expression of all six genes in tandem, the construct was designed such that each gene would have similar expression levels as much as possible</P><P>Fourth, the system was to be inducible, such that the expression of all six genes could be controlled. This design consideration was made due to gaining further input from beekeepers we interviewed. Due to the honey contamination, any products that honeybees are treated with follow strict treatment schedules. Allowing for an inducible system would give more control of when PPIX is produced and options the route for large-scale fermenters for mass production of porphyrin.</P><P>Upon defining the above specifications, the novel gene design for APIS was completed. Given that IDT gBlocks have a maximum size of 3 kb, we separated the six genes into four blocks, with the two largest genes, <I>hemA</I> and <I>hemB</I>, placed on separate gBlocks. <I>hemC</I> and <I>hemD</I>, were paired on their own separate block, as were <I>hemE</I> and <I>hemF</I>. This separation ensured that with the additional regulatory and visual check components, a full gBlock would remain within IDT’s synthesis limit, and independently, be a functional operon. To enable gene expression, each set of genes were placed under the control of a T7 promoter and a strong ribosome site. We chose to use the T7 promoter primarily due to its small size (18 bp), but also because we can then introduce our system to BL21(DE3), a common <I>E. coli</I> strain used to overexpress proteins, where it will be inducible by IPTG.  We decided to position the operons so that they alternate between the top and bottom strands and to use bidirectional terminators—that is, terminators which terminate genes oriented on opposing strands - to keep the construct as compact as possible, thus achieving the minimal size specification whilst also providing a means for proper termination. </P><DIV class="row"><DIV class="col-lg-6 align-self-center"><FIGURE class="figure"><FIGCAPTION class="figure-caption"><STRONG>Figure 3:</STRONG> Schematics of each gBlock that we designed in SnapGene and synthesized through IDT. Only (A) BBa_K2779010 and (B) BBa_K2779020 have been submitted to the Registry.(C) and (D) were successfully used in the construction of the APIS device, though were not individually characterized.</FIGCAPTION></FIGURE></DIV><DIV class="col-lg-6"><P>To facilitate cloning, genes encoding fluorescent proteins were incorporated into each gBlock as a screening method. Each <I>hemX</I>-containing block would be followed by a RBS and a sequence encoding alternating a red or green fluorescent protein. The RBS and fluorescent protein sequence would be flanked by unique restriction enzymes that can then be used to digest this region out and allow for the ligation of the next gBlock. During assembly, the change in colour (or lack thereof) would indicate whether the construct was ligated properly at each step in the assembly of a unified construct. During stepwise ligation, the engineered strain should fluoresce from red (<I>hemA</I>), then green (<I>hemA+B</I>), back to red (<I>hemA+B+C+D</I>), and finally to colourless (<I>hemA+B+C+D+E+F</I>) in BL21(DE3) when the construct was successfully assembled. This design would facilitate the successful sequential insertion of each gBlock into a vector.</P></DIV></DIV><H2>References</H2><P>[1] M. T. Nielsen, Et al., “Assembly of Highly Standardized Gene Fragments for High-Level Production of Porphyrins in <I>E. coli</I>,” ACS Synthetic Biology, vol.4, no.3, 2015. doi: 10.1021/sb500055u</P><P>[2] S. J. Kwon, Et al., “High-Level Production of Porphyrins in Metabolically Engineered Escherichia coli: Systematic Extension of a Pathway Assembled from Overexpressed Genes Involved in Heme Biosynthesis,” Applied and Environmental Microbiology, vol.69, no.8, 2003. doi: 10.1128/AEM.69.8.4875-4883.2003</P><P>[3] G. Layer, J. Reichelt, D. Jahn, D. W. Heinz, “Structure and function of enzymes in heme biosynthesis,” Protein Science, vol.19, no.6, 2010. doi:10.1002/pro.405</P></DIV></DIV><FOOTER><DIV class="ua-footer"><DIV class="container"><DIV class="row"><DIV class="col-sm-12 col-md-2"><UL><H5><EM><B>Project</B></EM></H5><LI><A href="https://2018.igem.org/Team:UAlberta/Description">Description</A></LI><LI><A href="https://2018.igem.org/Team:UAlberta/Design">Design</A></LI><LI><A href="https://2018.igem.org/Team:UAlberta/Results">Results</A></LI><LI><A href="https://2018.igem.org/Team:UAlberta/Demonstrate">Demonstrate</A></LI><LI><A href="https://2018.igem.org/Team:UAlberta/BeeLab">BeeLab</A></LI><LI><A href="https://2018.igem.org/Team:UAlberta/InterLab">InterLab</A></LI></UL></DIV><DIV class="col-sm-12 col-md-2"><UL><H5><EM><B>Parts</B></EM></H5><LI><A href="https://2018.igem.org/Team:UAlberta/Parts">Overview</A></LI><LI><A href="https://2018.igem.org/Team:UAlberta/Improve">Improve</A></LI></UL></DIV><DIV class="col-sm-12 col-md-3"><UL><H5><EM><B>Human Practices</B></EM></H5><LI><A href="https://2018.igem.org/Team:UAlberta/Human_Practices">Overview</A></LI><LI><A href="https://2018.igem.org/Team:UAlberta/IntegratedHP">Integrated Human Practices</A></LI><LI><A href="https://2018.igem.org/Team:UAlberta/Inclusivity">Inclusivity</A></LI><LI><A href="https://2018.igem.org/Team:UAlberta/Public_Engagement">Public Engagement</A></LI></UL></DIV><DIV class="col-sm-12 col-md-2"><UL><H5><EM><B>Notebook</B></EM></H5><LI><A href="https://2018.igem.org/Team:UAlberta/Notebook">Lab Notebook</A></LI><LI><A href="https://2018.igem.org/Team:UAlberta/Protocols">Protocols</A></LI><LI><A href="https://2018.igem.org/Team:UAlberta/Safety">Safety</A></LI></UL></DIV><DIV class="col-sm-12 col-md-2"><UL><H5><EM><B>Team</B></EM></H5><LI><A href="https://2018.igem.org/Team:UAlberta/Team">Members</A></LI><LI><A href="https://2018.igem.org/Team:UAlberta/Collaborations">Collaboration</A></LI><LI><A href="https://2018.igem.org/Team:UAlberta/Attributions">Attribution</A></LI><LI><A href="https://2018.igem.org/Team:UAlberta/Awards">Awards</A></LI><LI><A href="https://2018.igem.org/Team:UAlberta/Sponsors">Sponsors</A></LI></UL></DIV></DIV></DIV></DIV></FOOTER></DIV></DIV></DIV></DIV></DIV></BODY></HTML>