<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_UAlberta_InterLab skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><DIV id="top_title"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:UAlberta/InterLab</SPAN></H1></DIV><DIV id="HQ_page"><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><DIV class="ua-navbar"><DIV class="container "><NAV class="navbar navbar-expand-lg navbar-dark"><DIV class="collapse navbar-collapse" id="navbarSupportedContent"><UL class="navbar-nav ml-auto"><LI class="nav-item dropdown"><A class="nav-link dropdown-toggle" href="https://2018.igem.org/Team:UAlberta" id="navbarDropdown" role="button" data-toggle="dropdown" aria-haspopup="true" aria-expanded="false">
                 Project
               </A><DIV class="dropdown-menu" aria-labelledby="navbarDropdown"><A class="dropdown-item" href="https://2018.igem.org/Team:UAlberta/Description">Description</A><A class="dropdown-item" href="https://2018.igem.org/Team:UAlberta/Design">Design</A><A class="dropdown-item" href="https://2018.igem.org/Team:UAlberta/Results">Results</A><A class="dropdown-item" href="https://2018.igem.org/Team:UAlberta/Demonstrate">Demonstrate</A><A class="dropdown-item" href="https://2018.igem.org/Team:UAlberta/BeeLab">BeeLab</A><A class="dropdown-item" href="https://2018.igem.org/Team:UAlberta/InterLab">InterLab</A></DIV></LI><LI class="nav-item dropdown"><A class="nav-link dropdown-toggle" href="#" id="navbarDropdown" role="button" data-toggle="dropdown" aria-haspopup="true" aria-expanded="false">
                 Parts
               </A><DIV class="dropdown-menu" aria-labelledby="navbarDropdown"><A class="dropdown-item" href="https://2018.igem.org/Team:UAlberta/Parts">Overview</A><A class="dropdown-item" href="https://2018.igem.org/Team:UAlberta/Improve">Improve</A></DIV></LI><LI class="nav-item dropdown"><A class="nav-link dropdown-toggle" href="#" id="navbarDropdown" role="button" data-toggle="dropdown" aria-haspopup="true" aria-expanded="false">
                 Human Practices
               </A><DIV class="dropdown-menu" aria-labelledby="navbarDropdown"><A class="dropdown-item" href="https://2018.igem.org/Team:UAlberta/Human_Practices">Overview</A><A class="dropdown-item" href="https://2018.igem.org/Team:UAlberta/IntegratedHP">Integrated Human Practices</A><A class="dropdown-item" href="https://2018.igem.org/Team:UAlberta/Inclusivity">Inclusivity</A><A class="dropdown-item" href="https://2018.igem.org/Team:UAlberta/Public_Engagement">Public Engagement</A></DIV></LI><LI class="nav-item dropdown"><A class="nav-link dropdown-toggle" href="#" id="navbarDropdown" role="button" data-toggle="dropdown" aria-haspopup="true" aria-expanded="false">
                 Notebook
               </A><DIV class="dropdown-menu" aria-labelledby="navbarDropdown"><A class="dropdown-item" href="https://2018.igem.org/Team:UAlberta/Notebook">Lab Notebook</A><A class="dropdown-item" href="https://2018.igem.org/Team:UAlberta/Protocols">Protocols</A><A class="dropdown-item" href="https://2018.igem.org/Team:UAlberta/Safety">Safety</A></DIV></LI><LI class="nav-item dropdown"><A class="nav-link dropdown-toggle" href="#" id="navbarDropdown" role="button" data-toggle="dropdown" aria-haspopup="true" aria-expanded="false">
                 Team
               </A><DIV class="dropdown-menu" aria-labelledby="navbarDropdown"><A class="dropdown-item" href="https://2018.igem.org/Team:UAlberta/Team">Members</A><A class="dropdown-item" href="https://2018.igem.org/Team:UAlberta/Collaborations">Collaboration</A><A class="dropdown-item" href="https://2018.igem.org/Team:UAlberta/Attributions">Attribution</A><A class="dropdown-item" href="https://2018.igem.org/Team:UAlberta/Awards">Awards</A><A class="dropdown-item" href="https://2018.igem.org/Team:UAlberta/Sponsors">Sponsors</A></DIV></LI></UL></DIV></NAV></DIV></DIV><DIV class="ua-research-content"><DIV class="container"><H1>Interlab</H1><H2>Overview</H2><P>To make biological entities easier to engineer, standardization and other engineering principles have been adapted for and applied to biological parts. Though, standardization must not only be implemented to parts themselves but also in standardizing the methods we use to test these parts so that direct comparisons of designs are possible.</P><P>Thus, the goal of iGEM InterLab studies is to develop standards and protocols for measuring the fluorescence response of fluorescent proteins, a quantity commonly used in evaluating the performance of biological systems. Fluorescent proteins, such as Green Fluorescent Protein (GFP), have become widespread tools as their fluorescence readout can be linked with parameters like gene expression and protein interactions. Although, the lack of a widely accepted standard protocol and variance in measurement instruments, like microplate readers, contributes to the difficulty in comparing fluorescence measurements between instruments and different instances of measurement.</P><P>For the 2018 InterLab Study, the variability in fluorescence measurements when assessing a population of cells was investigated and the utility of normalizing measurements to absolute cell count or colony forming units (CFU) was assessed <A href="#ref1">[1]</A>. Team UAlberta accomplished these objectives this by following the InterLab Study protocols to measure the fluorescence and cellular density of eight devices which were calibrated against established standards.</P><H2>Materials and Methods</H2><P>The calibration and measurement procedures were performed by the members of Team UAlberta as outlined in the 2018 InterLab Study Protocols <A href="#ref2">[2]</A>. The following protocols describe Team UAlberta’s methods when none was specified by the InterLab protocols and to provide information about the measurement instruments.</P><H3>Transforming InterLab Devices into DH5α</H3><P><EM>Escherichia coli</EM> DH5α cells were transformed with the eight InterLab GFP devices  (Negative control: BBa_R0040, Positive control: BBa_I20270, Test Device 1: BBa_J364000, Test Device 2: BBa_J364001, Test Device 3: BBa_J364002, Test Device 4: BBa_J364007, Test Device 5: BBa_J364008, Test Device 6: BBa_J364009) from the 2018 DNA Distribution Kit, Kit Plate 7 by electroporation in 2 mm cuvettes at 2 kV. The transformed cells were incubated in SOC media for a 1 hr recovery period and plated on LB+Chloramphenicol agar plates using glass beads. The plates were incubated at 37°C for 24 hr.</P><H3>Microplate Reader Measurements</H3><P>Absorbance at 600 nm and fluorescence with excitation at 485 nm and emission at 530 nm / 8 nm bandpass were measured using the Tecan Safire2 <A href="#ref3">[3]</A>. Thermo Scientific Nunc 96-well black microplates with clear flat bottoms were used to hold samples. Pathlength correction was disabled and reads were taken from the bottom of the samples.</P><H2>Results </H2><P>For each of the eight InterLab devices, data for two colonies of each eight InterLab GFP devices, with four replicates was obtained. Team UAlberta’s completed data collection document, provided by the iGEM Measurement Committee can be found <A href="”#”">here</A>. The following figures display the calibration curves generated in this study in both linear and log scales. The processed data for Replicate 1 for the each of the eight device’s Colony 1 and Colony 2 are also displayed.</P><H3>Calibration Curves</H3><DIV class="row"><DIV class="col-lg-12 align-self-center"><FIGURE class="figure"><FIGCAPTION class="figure-caption"><STRONG>Figure 1:</STRONG> Calibration curves relating microsphere concentration to its absolute absorbance at 600 nm (OD600) presented in (A) linear scale and (B) log scale. Absorbance at 600 nm (Abs600) will be converted into OD600 using a ratiometric conversion factor determined in the sheet <A href="”#”">OD600 reference point</A>. These curves exhibit slight deviation from the expected linear response.</FIGCAPTION></FIGURE></DIV></DIV><DIV class="row"><DIV class="col-lg-12 align-self-center"><FIGURE class="figure"><FIGCAPTION class="figure-caption"><STRONG>Figure 2:</STRONG> Calibration curves relating fluorescein concentration to fluorescence at 485 nm excitation and emission at 530 nm / 8 nm bandpass measured in fluorescence arbitrary units (AU) and presented in (A) linear scale and (B) logscale. These calibrations exhibit the expected linear response of fluorescence at these fluorescein concentrations.</FIGCAPTION></FIGURE></DIV></DIV><H3>InterLab Measurements</H3><DIV class="row"><DIV class="col-lg-12 align-self-center"><FIGURE class="figure"><FIGCAPTION class="figure-caption"><STRONG>Figure 3:</STRONG> The fluorescence response at 485 nm excitation and emission at 530 nm / 8 nm bandpass in AU of cultures expressing the eight InterLab devices at the time of inoculation and after 6 hours of incubation for (A) Colony 1, Replicate 1 and for (B) Colony 2, Replicate 2. The Abs600 of the corresponding cultures at the start and end of incubation for (C) Colony 1, Replicate 1 and for (D) Colony 2, Replicate 1.</FIGCAPTION></FIGURE></DIV></DIV><H2>Discussion</H2><H3>Calibration Curves</H3><P>The serial dilutions fluorescein dye resulted in a consistent and very linear relationship between fluorescein concentration and fluorescence response. This indicates that this particular standard is useful in generating robust calibration curves to be used in quantifying fluorescence from GFP. Team UAlberta is glad that this result corroborates the team’s positive experience in using fluorescein in the <A href="https://2017.igem.org/Team:UAlberta/InterLab">2017 InterLab Study</A>.</P><P>However, the standard curve generated by the microspheres was not linear as expected. The response at the lower bound concentrations of the microspheres was level but started to increase linearly near the upper concentrations of the curve. Ideally, this linear response should be seen throughout the range of concentrations. What this indicates is that pipetting error may have been introduced in the serial dilution, or that the settling characteristic to these microspheres contributed to higher than expected values. This suggests that these microspheres are not reliable to use for generating a OD600 standard curve as the absorbance of their dilutions can change with time due to settling.</P><H3>InterLab Device Measurements</H3><P>For both biological replicates, Colonies 1 and 2, the cultures expressing each of the eight InterLab devices did not grow similarly. Both replicates display the same pattern of growth where Test Devices 2, 3, and 6 grew to comparable density as the negative and positive controls. Meanwhile, Test Devices 1, 4, and 5 exhibited a marginal increase in Abs600, a sign of little growth. As Colonies 1 and 2 agree, these differences in culture density after the 6 hour incubation period can be attributed to the metabolic load the different devices cause on each culture.</P><P>These discrepancies can be rationalized with the design of the devices, as Test Device 1 has the strongest promoter and ribosome binding site (RBS), which should have the greatest metabolic load as its expression of GFP is driven the greatest. This explains the hindered growth of Test Device 1 cultures. However, this rationale cannot be directly extended to the rest of the devices, as all other devices, when considering strictly the promoter and RBS strength, should have lower expression than Test Device 1 and should be denser cultures. Though, this is not the case. What can be gleaned is that other factors, like unforeseen secondary structure effects caused by the different combinations of promoters and ribosome binding sites, create different expression dynamics and lead to unexpected effects on cell growth.</P><P>Likewise, the fluorescence response does not follow a general trend but is consistent between Colonies 1 and 2. This lack of a trend is likely due to the different cell densities where denser cultures have the potential of exhibiting greater fluorescence as there can be more GFP produced by the cells. This result highlights the limitations of microplate readers due to the variability inherent in measuring the fluorescence of a population of cells. Though, the positive and negative controls do display their expected behaviour and further corroborates the utility of these devices as comparisons.</P><H3>Normalization</H3><P>To address these discrepancies arising from the variability due to assessing a population of cells, the fluorescence response of the devices were normalized using equivalent fluorescein concentration with OD600, and molecules of equivalent fluorescein (MEFL) and number of particles. What this procedure aims to do is counteract the differences in cellular density of each sample in different ways. The first method aims to normalize fluorescence to the concentration of cells, approximated by OD600, while the second method aims to relate fluorescence to the equivalent number of fluorescein molecules per each cell, approximated by the number of microspheres. These methods are related but are still independent measures of normalized fluorescence. After the data was processed using the data collection spreadsheet provided, both methods result in the same pattern of normalized fluorescence for each device and for both Colonies 1 and 2. This is a strong indication that normalization according to the calibration curves is effective in reducing variability when assessing a population of cells!</P><P>The outcomes of Team UAlberta’s InterLab procedure can be rationalized and corroborate the idea that normalizing fluorescence to cellular density can reduce variability amongst measurements of a population of cells. However, our remaining replicates still need to be examined, along with the results of all the other InterLab participants! We look forward to seeing the official results!</P><H2>Conclusion</H2><P>Team UAlberta’s InterLab data has been successfully submitted and approved by the iGEM Measurement Committee. Our normalized fluorescence results are consistent for when equivalent fluorescein concentration is normalized to OD600 and when molecules of equivalent fluorescein are normalized to particle number. Thus LUDOX, fluorescein, and microsphere references seem to be robust calibration methods and that a concentration or discrete particle basis of normalization are valid approaches. Team UAlberta was glad to have participated in the 2018 InterLab Study, not only to fulfill iGEM Medal Requirements but to also contribute to standardized fluorescence measurement protocols!</P><H2>References</H2><P id="ref1"><A href="https://2018.igem.org/Measurement/InterLab">[1]</A> International Genetically Engineered Machine Foundation, “Fifth International InterLab Measurement Study,” <EM>International Genetically Engineered Machine Foundation</EM>, 2018. [Online]. Available: <A id="”ref1”" href="https://2018.igem.org/Measurement/InterLab">https://2018.igem.org/Measurement/InterLab</A> [Accessed: Oct. 15, 2018]</P><P id="ref2"><A href="https://static.igem.org/mediawiki/2018/0/09/2018_InterLab_Plate_Reader_Protocol.pdf">[2]</A><EM> iGEM 2018 InterLab Study Protocol</EM>, International Genetically Engineered Machine Foundation, 2018 [Online]. Available: <A href="https://static.igem.org/mediawiki/2018/0/09/2018_InterLab_Plate_Reader_Protocol.pdf">https://static.igem.org/mediawiki/2018/0/09/2018_InterLab_Plate_Reader_Protocol.pdf</A> [Accessed: Oct. 15, 2018]</P><P id="ref3"><A href="http://www.mtxlsi.com/tecansafire.htm">[3]</A> MTX Lab Systems, “Tecan Safire Monochromatic,” <EM>MTX Lab Systems</EM>, 2014 [Online]. Available: <A href="”http://www.mtxlsi.com/tecansafire.htm”">http://www.mtxlsi.com/tecansafire.htm</A> [Accessed: Oct. 15, 2018]</P></DIV></DIV><FOOTER><DIV class="ua-footer"><DIV class="container"><DIV class="row"><DIV class="col-sm-12 col-md-2"><UL><H5><EM><B>Project</B></EM></H5><LI><A href="https://2018.igem.org/Team:UAlberta/Description">Description</A></LI><LI><A href="https://2018.igem.org/Team:UAlberta/Design">Design</A></LI><LI><A href="https://2018.igem.org/Team:UAlberta/Results">Results</A></LI><LI><A href="https://2018.igem.org/Team:UAlberta/Demonstrate">Demonstrate</A></LI><LI><A href="https://2018.igem.org/Team:UAlberta/BeeLab">BeeLab</A></LI><LI><A href="https://2018.igem.org/Team:UAlberta/InterLab">InterLab</A></LI></UL></DIV><DIV class="col-sm-12 col-md-2"><UL><H5><EM><B>Parts</B></EM></H5><LI><A href="https://2018.igem.org/Team:UAlberta/Parts">Overview</A></LI><LI><A href="https://2018.igem.org/Team:UAlberta/Improve">Improve</A></LI></UL></DIV><DIV class="col-sm-12 col-md-3"><UL><H5><EM><B>Human Practices</B></EM></H5><LI><A href="https://2018.igem.org/Team:UAlberta/Human_Practices">Overview</A></LI><LI><A href="https://2018.igem.org/Team:UAlberta/IntegratedHP">Integrated Human Practices</A></LI><LI><A href="https://2018.igem.org/Team:UAlberta/Inclusivity">Inclusivity</A></LI><LI><A href="https://2018.igem.org/Team:UAlberta/Public_Engagement">Public Engagement</A></LI></UL></DIV><DIV class="col-sm-12 col-md-2"><UL><H5><EM><B>Notebook</B></EM></H5><LI><A href="https://2018.igem.org/Team:UAlberta/Notebook">Lab Notebook</A></LI><LI><A href="https://2018.igem.org/Team:UAlberta/Protocols">Protocols</A></LI><LI><A href="https://2018.igem.org/Team:UAlberta/Safety">Safety</A></LI></UL></DIV><DIV class="col-sm-12 col-md-2"><UL><H5><EM><B>Team</B></EM></H5><LI><A href="https://2018.igem.org/Team:UAlberta/Team">Members</A></LI><LI><A href="https://2018.igem.org/Team:UAlberta/Collaborations">Collaboration</A></LI><LI><A href="https://2018.igem.org/Team:UAlberta/Attributions">Attribution</A></LI><LI><A href="https://2018.igem.org/Team:UAlberta/Awards">Awards</A></LI><LI><A href="https://2018.igem.org/Team:UAlberta/Sponsors">Sponsors</A></LI></UL></DIV></DIV></DIV></DIV></FOOTER></DIV></DIV></DIV></DIV></DIV></BODY></HTML>