<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_UConn_Project skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><DIV id="top_title"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:UConn/Project</SPAN></H1></DIV><DIV id="HQ_page"><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><DIV class="headercontainer"><HEADER><NAV id="cssmenu"><UL><LI class="wide"><A class="item" href="https://2018.igem.org/Team:UConn">Home</A></LI><LI class="narrow"><A class="item" href="https://2018.igem.org/Team:UConn/Project">Project</A></LI><LI class="mobilenav"><A href="https://2018.igem.org/Team:UConn/Project#background" value="Reload Page" onclick="window.location.reload()">Background</A></LI><LI class="mobilenav"><A href="https://2018.igem.org/Team:UConn/Project#description" value="Reload Page" onclick="window.location.reload()">Description</A></LI><LI class="desktopnav"><A href="https://2018.igem.org/Team:UConn/Project#background">Background</A></LI><LI class="desktopnav"><A href="https://2018.igem.org/Team:UConn/Project#description">Description</A></LI></UL><LI class="narrow"><A class="item" href="https://2018.igem.org/Team:UConn/Notebook">Notebook</A></LI><LI class="mobilenav"><A href="https://2018.igem.org/Team:UConn/Notebook#interlab" value="Reload Page" onclick="window.location.reload()">Interlab</A></LI><LI class="mobilenav"><A href="https://2018.igem.org/Team:UConn/Notebook#labnotebook" value="Reload Page" onclick="window.location.reload()">Lab Notebook</A></LI><LI class="mobilenav"><A href="https://2018.igem.org/Team:UConn/Notebook#protocols" value="Reload Page" onclick="window.location.reload()">Protocols</A></LI><LI class="desktopnav"><A href="https://2018.igem.org/Team:UConn/Notebook#interlab">Interlab</A></LI><LI class="desktopnav"><A href="https://2018.igem.org/Team:UConn/Notebook#labnotebook">Lab Notebook</A></LI><LI class="desktopnav"><A href="https://2018.igem.org/Team:UConn/Notebook#protocols">Protocols</A></LI><LI class="narrow"><A class="item" href="https://2018.igem.org/Team:UConn/Parts">Parts</A></LI><LI class="mobilenav"><A href="https://2018.igem.org/Team:UConn/Parts#listofparts" value="Reload Page" onclick="window.location.reload()">List of Parts</A></LI><LI class="desktopnav"><A href="https://2018.igem.org/Team:UConn/Parts#listofparts">List of Parts</A></LI><LI class="narrow"><A class="item" href="https://2018.igem.org/Team:UConn/HumanPractices">Human Practices</A></LI><LI class="mobilenav"><A href="https://2018.igem.org/Team:UConn/HumanPractices#communityoutreach" value="Reload Page" onclick="window.location.reload()">Community Outreach</A></LI><LI class="mobilenav"><A href="https://2018.igem.org/Team:UConn/HumanPractices#collaborations" value="Reload Page" onclick="window.location.reload()">Collaborations</A></LI><LI class="desktopnav"><A href="https://2018.igem.org/Team:UConn/HumanPractices#communityoutreach">Community Outreach</A></LI><LI class="desktopnav"><A href="https://2018.igem.org/Team:UConn/HumanPractices#collaborations">Collaborations</A></LI><LI class="narrow"><A class="item" href="https://2018.igem.org/Team:UConn/Safety">Safety</A></LI><LI class="narrow"><A class="item" href="https://2018.igem.org/Team:UConn/Team">Team</A></LI><LI class="mobilenav"><A href="https://2018.igem.org/Team:UConn/Team#attributions" value="Reload Page" onclick="window.location.reload()">Attributions</A></LI><LI class="desktopnav"><A href="https://2018.igem.org/Team:UConn/Team#attributions">Attributions</A></LI></NAV></HEADER></DIV><DIV id="project" class="topsection"><DIV class="container"><H1>Project</H1></DIV></DIV><DIV id="projectsection1" class="section"><DIV class="container"><H2>Background</H2><H3>Background</H3><P>Transportation in America is primarily driven by petroleum. In 2017 alone, petroleum accounted for 92% of energy usage by transportation systems whereas biofuels were used for just 5%<SUP>1</SUP>. It’s no surprise; petroleum is much easier to obtain by tapping into the mass amounts of oil underground, and due to the amount of processing and testing biodiesel must undergo in order to be sold, petroleum ends up being the cheaper option. However, it comes at a cost to the environment. The drilling and burning of oil produces harmful byproducts, disturbs wildlife, and is ultimately unsustainable. 

In a world constantly looking for faster, cheaper methods of production, the answer is typically found in chemical synthesis. Biosynthesis methods simply cannot match the efficiency and cost of chemical synthesis, despite the environmental benefits they offer. These benefits fuel the question: is there a way to make microbial reductive pathways, such as biofuel production, more efficient and less expensive than current methods?</P><H3>Petrobrick</H3><P>A publication in <I>Science</I> in 2010 showed that E. coli can be genetically modified to produce short-chain alkanes that can imitate petroleum and other fuels, although yields are very low at about 0.3g alkanes per liter of output<SUP>2</SUP>. In 2011, the University of Washington iGEM team used this to create the Petrobrick, a biobrick used to create enzymes (acyl-ACP reductase and aldehyde decarbonylase) that can produce alkanes in E. coli. This work can be used as a baseline for our goal: to create a system that would produce more alkanes per unit of input than is currently possible.

Microbial Fuel Cells and Microbial Electrosynthesis
Microbial fuel cells (MFCs) are circuits consisting of two electrodes in two media chambers, one of which contains microbes. The chambers are connected by a semipermeable membrane and a wire with a load, creating a system. The microbes act as the electron donors, and the electrons travel from anode to cathode, creating a current that powers the load. Protons cross the semipermeable membrane to bind with electrons and oxygen, making water molecules. 

Conversely, in microbial electrosynthesis (MES) systems, also known as reverse MFCs, the microbes reside in the cathode chamber and the system uses an input of power other than microbes. At the cathode, electrons are taken up by microbes and this provides reduction power so that more reactants can be produced than normally would by the microbes alone, thus increasing the rate of reaction. MES systems have safer waste products, such as carbon dioxide and water.</P><H3>Sources</H3><P>
1. Energy Use for Transportation. (2018, May 23). Retrieved October 7, 2018, from
<A href="https://www.eia.gov/energyexplained/index.php?page=us_energy_transportation#tab1">https://www.eia.gov/energyexplained/index.php?page=us_energy_transportation#tab1</A>
2. Schirmer, A., Rude, M. A., Li, X., Popova, E., &amp; Del Cardayre, S. B. (2010). Microbial biosynthesis of alkanes. Science, 329(5991), 559-562. </P></DIV></DIV><DIV id="projectsection2" class="section"><DIV class="container"><H2>Description</H2><H3>Overview</H3><P>BASSET aims to engineer Escherichia coli to create a microbial electrosynthesis (MES) system that produces short-chained alkanes. The biosynthesis pathway produces alkanes from fatty acyl-ACPs, using energy derived from the MES module. Heterologous expression of pmt1231 from Prochlorococcus marinus and acr from Clostridium acetobutylicum, overexpression of the endogenous fadK and a mutant tesA, and deletion of endogenous fadE and fadR create a pathway to achieve this goal.
</P><P>To increase efficiency of the system, the engineered organism would accept electrons from an external source, such as off-peak solar energy, to produce reducing equivalents such as NADH or NADPH in the cell, which are then used in the reductive alkane production pathway. This functionality will be gained by heterologous expression of the MtrCAB pathway and the CymA cytochrome from Shewanella oneidensis, as well as phenazines (secreted extracellular electron mediators) produced by the phzA-H machinery from Pseudomonas aeruginosa. Overexpression of the endogenous cytochrome maturation gene, ccmA-H will also assist in this MES system.</P><H3>Alkane Biofuel Production</H3><P>The gene tesA is naturally found in E. coli but was mutated for our project to change the 109th amino acid from leucine to proline. The function of ‘tesA is to convert fatty acyl-ACPs, derived from the glucose energy source, into free fatty acids. The mutation serves to increase the output of shorter chain free fatty acids, which are more optimal for gasoline-like short chain alkanes, and reduce the output of longer chain free fatty acids.</P><P>The gene fadK naturally occurs in E. coli and was extracted from the genome for overexpression. It functions to convert free fatty acids to fatty acyl-CoA, which can then be used by FadE, endogenous to E. coli, to create 2-trans-enoyl-CoA or by Acr, originally from C. acetobutylicum, to continue the alkane pathway and produce fatty-aldehyde. Finally, Pmt1231, which comes from P. marinus, finishes the pathway by converting fatty-aldehyde into short chain alkanes. </P><H3>Microbial Electrosynthesis</H3><P>The microbial electrosynthesis module of our project consists of the genes mtrCAB, phzA-H, cymA, and ccmA-H, which will collectively create a system to transport electrons.</P><P class="clearfix">The phzA-H genes, otherwise known as the phenazine pathway, will create shuttles to carry electrons between the electrodes and bacteria to be used in the reductive pathway. The MtrCAB complex will be used to take electrons into the cell with the MtrB and MtrC proteins implanted in the outer membranes of cells and the MtrA protein carrying the electrons into the cells. The electrons then diffuse across the periplasmic space where they are taken up by CymA on the inner membrane. The ccmA-H genes are a set of cytochromes that serve to help mature c-type cytochromes such as the MtrCAB complex and CymA.


These genes have previously been used in microbial fuel cell (MFC) systems, in the bacterial organism, residing in a bottle with an electrode, supplies electrons at the anode side. Electrons are shuttled by the phenazines to the anode and flow through a circuit, creating a current that powers a load and ultimately brings the electrons to the cathode bottle. 
</P><P>Conversely, this pathway can be reversed so that, instead of the bacteria, an external power source provides electrons in place of a load using the current. Electrons flow to the bacteria in the cathode bottle, where they are used to reduce reactants of a reduction pathway, thus powering the pathway. The chosen genes are optimal for this use because both the mtrCAB + CymA and phenazine pathways are reversible; although originally designed to carry electrons out of cells, they are also capable of bringing electrons into cells which can reduce NADP+ into NADPH for use in the alkane synthesis pathway.</P><H3>Future Directions</H3><P>Currently, three out of the four parts for the alkane synthesis pathway have been created and those for microbial electrosynthesis are being troubleshooted. To continue our project, we will be implementing the microbial electrosynthesis system, and optimizing the alkane pathway.

Currently, alkane yields are relatively low due to restrictive genes in the pathway that optimize it for the use of the bacteria. In order to increase yield, we will knockout fadE and fadR using the lambda red system. The natural purpose of FadE in E. coli is to convert fatty-acyl, which is used in the alkane pathway, CoA to 2-trans-eonyl-CoA, which is not a part of the pathway, thus restricting the amount of input available for use by Acr and limiting the amount of alkanes produced. The gene fadR is also endogenous to E. coli and increases transcription of genes used in unsaturated fatty acid biosynthesis while decreasing the transcription of genes used in the alkane pathway. 

To negate the effects of FadE and FadR, they will be knocked out using the lambda red system. This system, which has been utilized by other iGEM teams has the ability to replace segments of genomic DNA with a specially designed linear segment (using homologous overlaps). Our general strategy will be to replace the initial segments of the fadE and fadR genes with resistance genes, select for the engineered bacteria, and then to excise this resistance using loxP sites we design into the linear segment. Thus, any controls that FadE and FadR might otherwise exert on the alkane production pathway will be silenced.
</P><H3>Sources</H3><P>1. Choi, Y. J., &amp; Lee, S. Y. (2013). Microbial production of short-chain alkanes. Nature, 502(7472), 571.
Das, D., Eser, B. E., Han, J., Sciore, A., &amp; Marsh, E. N. G. (2011). Oxygen‐Independent Decarbonylation of Aldehydes by Cyanobacterial Aldehyde Decarbonylase: A New Reaction of Diiron Enzymes. Angewandte Chemie International Edition, 50(31), 7148-7152.

2. Eser, B. E., Das, D., Han, J., Jones, P. R., &amp; Marsh, E. N. G. (2011). Oxygen-independent alkane formation by non-heme iron-dependent cyanobacterial aldehyde decarbonylase: investigation of kinetics and requirement for an external electron donor. Biochemistry, 50(49), 10743-10750.

3. Feng, J., Qian, Y., Wang, Z., Wang, X., Xu, S., Chen, K., &amp; Ouyang, P. (2018). Enhancing the performance of Escherichia coli-inoculated microbial fuel cells by introduction of the phenazine-1-carboxylic acid pathway. Journal of biotechnology, 275, 1-6.

4. Gescher, J. S., Cordova, C. D., &amp; Spormann, A. M. (2008). Dissimilatory iron reduction in Escherichia coli: identification of CymA of Shewanella oneidensis and NapC of E. coli as ferric reductases. Molecular microbiology, 68(3), 706-719.

5. Jensen, H. M., TerAvest, M. A., Kokish, M. G., &amp; Ajo-Franklin, C. M. (2016). CymA and exogenous flavins improve extracellular electron transfer and couple it to cell growth in Mtr-expressing Escherichia coli. ACS synthetic biology, 5(7), 679-688.

6. Jiménez‐Díaz, L., Caballero, A., Pérez‐Hernández, N., &amp; Segura, A. (2017). Microbial alkane production for jet fuel industry: motivation, state of the art and perspectives. Microbial biotechnology, 10(1), 103-124.
Campbell, J. W., Morgan‐Kiss, R. M., &amp; E Cronan, J. (2003). A new Escherichia coli metabolic competency: growth on fatty acids by a novel anaerobic β‐oxidation pathway. Molecular microbiology, 47(3), 793-805.

7. Lovley, D. R., &amp; Nevin, K. P. (2013). Electrobiocommodities: powering microbial production of fuels and commodity chemicals from carbon dioxide with electricity. Current opinion in biotechnology, 24(3), 385-390.

8. Rabaey, K., &amp; Rozendal, R. A. (2010). Microbial electrosynthesis—revisiting the electrical route for microbial production. Nature Reviews Microbiology, 8(10), 706.

9. Ross, D. E., Flynn, J. M., Baron, D. B., Gralnick, J. A., &amp; Bond, D. R. (2011). Towards electrosynthesis in Shewanella: energetics of reversing the Mtr pathway for reductive metabolism. PloS one, 6(2), e16649.

10. TerAvest, M. A., Zajdel, T. J., &amp; Ajo‐Franklin, C. M. (2014). The Mtr pathway of shewanella oneidensis MR‐1 couples substrate utilization to current production in escherichia coli. ChemElectroChem, 1(11), 1874-1879.</P></DIV></DIV><FOOTER><DIV id="footerhome"><A href="https://2018.igem.org/Team:UConn">Home</A></DIV><DIV id="bottomnav"><DIV id="bottomnavsection1" class="bottomnavsection"><DIV class="bottomnavitem"><A href="https://2018.igem.org/Team:UConn/Project">Project</A></DIV><UL><LI><A href="https://2018.igem.org/Team:UConn/Project#background">Background</A></LI><LI><A href="https://2018.igem.org/Team:UConn/Project#description">Description</A></LI></UL></DIV><DIV id="bottomnavsection2" class="bottomnavsection"><DIV class="bottomnavitem"><A href="https://2018.igem.org/Team:UConn/Notebook">Notebook</A></DIV><UL><LI><A href="https://2018.igem.org/Team:UConn/Notebook#interlab">Interlab</A></LI><LI><A href="https://2018.igem.org/Team:UConn/Notebook#labnotebook">Lab Notebook</A></LI><LI><A href="https://2018.igem.org/Team:UConn/Notebook#protocols">Protocols</A></LI></UL></DIV><DIV id="bottomnavsection3" class="bottomnavsection"><DIV class="bottomnavitem"><A href="https://2018.igem.org/Team:UConn/Parts">Parts</A></DIV><UL><LI><A href="https://2018.igem.org/Team:UConn/Parts#listofparts">List of Parts</A></LI></UL></DIV><DIV id="bottomnavsection4" class="bottomnavsection"><DIV class="bottomnavitem"><A href="https://2018.igem.org/Team:UConn/HumanPractices">Human Practices</A></DIV><UL><LI><A href="https://2018.igem.org/Team:UConn/HumanPractices#communityoutreach">Community Outreach</A></LI><LI><A href="https://2018.igem.org/Team:UConn/HumanPractices#collaborations">Collaborations</A></LI></UL></DIV><DIV id="bottomnavsection5" class="bottomnavsection"><DIV class="bottomnavitem"><A href="https://2018.igem.org/Team:UConn/Safety">Safety</A></DIV></DIV><DIV id="bottomnavsection6" class="bottomnavsection"><DIV class="bottomnavitem"><A href="https://2018.igem.org/Team:UConn/Team">Team</A></DIV><UL><LI><A href="https://2018.igem.org/Team:UConn/Notebook#attributions">Attributions</A></LI></UL></DIV></DIV><DIV id="bottomnavaddress"><P><A href="https://goo.gl/maps/yexLUgoBazx" target="blank">University of Connecticut (UConn), Storrs, Connecticut 06269, United States of America</A><A>Contact us at </A><A href="mailto:uconnigem@gmail.com">uconnigem@gmail.com</A></P></DIV></FOOTER></DIV></DIV></DIV></DIV></DIV></BODY></HTML>