Team:UConn/Notebook
Home
Project
Background
Description
Background
Description
Notebook
Interlab
Lab Notebook
Protocols
Interlab
Lab Notebook
Protocols
Parts
List of Parts
List of Parts
Human Practices
Community Outreach
Collaborations
Community Outreach
Collaborations
Safety
Team
Attributions
Attributions
Notebook
Interlab
The purpose of this year’s interlab is to address the high level of variability in fluorescence measurements. Typically, fluorescence is measured using the optical density (OD), or the absorbance of light by a sample when measured at 600nm using a plate reader,  nanodrop, or spectrophotometer. However, since this method can be inaccurate, Interlab seeks to improve the standard method of measuring fluorescence in populations of cells. Since the number of cells in a sample can vary, if the fluorescence is normalized to the cell count of the sample, then the average amount of fluorescence can be calculated, which is likely more accurate than that of a population of unknown size. Therefore, Interlab will investigate if normalizing fluorescence measurements to colony-forming units (CFUs) will yield more precise results than OD.
Each iGEM team was asked to normalize the absorbance to the number of cells in two ways: by comparing sample absorbance to absorbance of a sample of known concentration of beads similar in size to E. coli cells and by counting CFUs from each sample.
OD
For the first method, absorbance was measured for a sample and serial dilution of silicon beads. These beads, similar in size and shape to E. coli cells, are expected to scatter light in a similar manner, providing an analogous sample of known concentration. The first part of this process was to get a reference point for OD600 to easily interchange between Abs600 and OD600, accounting for differences in volume between wells. In this measurement, four equal samples LUDOX CL-X (a 45% colloidal silica suspension) were compared to samples of ddH2O, and the OD was measured using a plate reader. By finding the mean of the replicates, correcting by subtracting the mean of the water to get just the absorbance of the silica, and dividing by the OD600 from the provided Excel sheet, the conversion factor (OD600/Abs600) was calculated to be 3.500.
A particle standard curve was also developed using a serial dilution of monodisperse silica microspheres to plot the concentration versus absorbance, thus allowing the estimation of the number of cells based off the absorbance. The serial dilution was performed four times, and the Abs600 was measured to create the following standard curves:
Finally, a fluorescence standard curve was created to have a point of comparison between outputs of different plate readers, getting rid of the arbitrary units assigned by different instruments. Fluorescein is used as an easier, lower-cost alternative to GFP as it has similar excitation and emission properties. Four identical serial dilutions were completed for fluorescein, and the Abs600 was measured to create the following standard curves:
CFU
This part of the Interlab found the volume of a 0.1 OD600 culture and the number of cells in a known volume by growing a sample and counting the number of colonies that grew. For the “Starting Sample Preparation,” the OD600 of diluted overnight cultures of positive and negative controls were taken along with that of blank media. Based on the starting sample, overnights were diluted in triplicate, and the OD600 was taken and adjusted using the blank, yielding the absorbance. To find the number of cells in a given volume, a serial dilution was performed for each sample, and the last three dilutions were plated and grown overnight. Colonies were counted, and, with the understanding that one cell grows into one colony, the number of cells per unit of volume was found. To standardize across dilutions, we used the equation:
number of colonies x final dilution factor = CFU/mL
Dilution 3
Plate
Number of Colonies
CFU/mL 0.1 OD600 solution
1.1
185
14800000
1.2
28
2240000
1.3
79
6320000
2.1
Over 300
N/A
2.2
Over 300
N/A
2.3
Over 300
N/A
3.1
121
9680000
3.2
261
20880000
3.3
170
13600000
4.1
94
7520000
4.2
270
21600000
4.3
Over 300
N/A
Dilution 4
Plate
Number of Colonies
CFU/mL 0.1 OD600 solution
1.1
18
14400000
1.2
7
5600000
1.3
6
4800000
2.1
65
52000000
2.2
105
84000000
2.3
76
60800000
3.1
42
33600000
3.2
26
20800000
3.3
12
9600000
4.1
31
24800000
4.2
24
19200000
4.3
33
26400000
Dilution 5
Plate
Number of Colonies
CFU/mL 0.1 OD600 solution
1.1
1
8000000
1.2
0
0
1.3
1
8000000
2.1
1
8000000
2.2
2
16000000
2.3
8
64000000
3.1
1
8000000
3.2
3
24000000
3.3
1
8000000
4.1
0
0
4.2
5
40000000
4.3
3
24000000
Lab Notebook
Click on each section to view the week's notebook.
May
Week 2: 5/21/2018
May 22, 2018
Competent Cell Prep
Liz, Peter, Ryan, Shannon, Sricharan
We’re preparing competent cells (E. coli, XL1 Blue) for the summer.
Craig and Sricharan tried to do this in December, but it didn’t work since the shaker was at 200rpm instead of at 250rpm.
Today: prepping stock solutions, streaking plate
TfbII (50 mL)
Reagent
Concentration
Amount needed
Amount measured
MOPS
10mM
0.104g
0.1040g
CaCl2
75mM
0.417g
0.4160g
RbCl2
10mM
0.061g
0.0616g
Glycerol
15% (v/v)
7.5mL
7.5mL
Measured amounts of reagents in flask (next time, use beaker)
Added most of water (don’t top off, so you can adjust pH)
pH (w/stirring if possible) to 6.5, using 1M KOH dropwise - got to 6.52… (make sure to standardize with pH 7 and 4 buffers)
Topped off in graduated cylinder
0.45µm filter sterilized
No need to autoclave this solution
Stored in Tigger (4°C)
TfbI (200 mL)
Reagent
Concentration
Amount needed
Amount measured
Potassium Acetate
30mM
0.588g
0.589g
MnCl2
50mM
1.98g
1.99g
CaCl2
10mM
0.222g
0.224g
RbCl2
10mM
0.242g
0.242g
Glycerol
15% (v/v)
30mL
30mL
Measured amounts of reagents in flask (next time, use beaker)
Added most of water (don’t top off, so you can adjust pH)
pH (w/stirring if possible) to 5.8, using .2M acetic acid dropwise - got to 5.82… (make sure to standardize with pH 7 and 4 buffers)
Topped off in graduated cylinder
0.45µm filter sterilized
No need to autoclave this solution
Stored in Tigger (4°C)
ᴪa (500 mL)
Reagent
Concentration
Amount needed
Bacto yeast extract
5g/L
2.5g
Bacco tryptone
20g/L
10g
MgSO4
5g/L
2.5g
KOH
1M
Added dropwise until solution reached pH 7.6
Dry substances mixed in bottle (would be easier to use a beaker)
Water topped off to almost 500mL
Pour into flask to adjust pH using spinner
Standardized pH meter using 7 and 10 buffers, pain to do
Pour back into graduated cylinder to top off to 500 mL
Use sterile filter
Aliquot 100mL into two 300mL flasks with caps, remaining 300mL in bottle
Autoclave at liquid 30, 45 minutes dry
Removed from autoclave, allowed to cool, and transferred to 4℃ fridge (Tigger)
Also streaked Ψa plate
Sterile environment (with portable burner nearby)
Stabbed competent XL1 Blue cells (frozen stock) with a pipet tip
Gently spread frozen piece of cells onto plate with tip (be careful not to puncture agar)
Spread cells on plate with Pasteur spinner (not the plating beads, @Craig)
Put plate into incubator at 39°C (not 37°C, ugh)
May 22, 2018
Interlab Calibration
Liz, Peter, Ryan, Shannon, Sricharan
Calibration 1, OD600 Reference Point - LUDOX protocol:
100µL LUDOX CL-X in wells A1, B1, C1, D1
100µL ddH2O in wells A2, B2, C2, D2
Absorbances
1
2
A
0.045
0.024
B
0.041
0.025
C
0.042
0.025
D
0.042
0.025
Wavelength: 600nm
Optics: top
Temperature: 25.9°C
Calibration 2
Stock Solution (Vortex silica beads for 30 sec. before adding):
Reagent
Amount
Silica beads
96 µL
ddH2O
904 µL
Add 100 µL ddH2O to wells A1, B1, C1, D1, …, A12, B12, C12, D12
Add 200 µL stock solution to A1, B1, C1, and D1 (vortex 10s before adding)
Transfer 100 µL from A1 to A2, pipet to mix, transfer 100 µL from A2 to A3, pipet to mix, etc. until A11. Repeat for B, C, and D.
Cells A1, B1, C1, and D1 should be pure stock
Cells A12, B12, C12, and D12 should be pure ddH2O
Absorbance
1
2
3
4
5
6
7
8
9
10
11
12
A
1.075
0.545
0.343
0.152
0.087
0.055
0.051
0.032
0.028
0.031
0.028
0.025
B
1.199
0.666
0.306
0.161
0.086
0.057
0.04
0.033
0.028
0.026
0.026
0.025
C
1.191
0.651
0.346
0.176
0.091
0.03
0.051
0.043
0.031
0.028
0.027
0.025
D
1.325
0.587
0.299
0.168
0.088
0.057
0.039
0.032
0.027
0.026
0.025
0.025
Wavelength: 600nm
Optics: top
Temperature: 26°C
Calibration 3: Fluorescence Standard Curve
Fluorescein stock solution:
Reagent
Amount
Fluorescein
Provided in Kit (5 tubes)
1x PBS pH 7.4-7.6
10 mL
Spun down fluorescein in tube so the pellet was at the bottom
Made ~ 40 mL 1x PBS in falcon tube (originally used 10 x accidentally for first 3 attempts/ tubes)
Used 1 mL of 1x PBS to resuspend fluorescein in kit tube > makes 10x fluoresceiµLtocol calls for pH of 7.4-7.5. Problem? > email iGEM
Diluted 10 fluorescein stock with 900 µL of 1x PBS and 100 µL of 10x fluorescein stock to make 1x fluorescein solution (10 M)
Serial Dilutions:
Used clear flat-bottom 96 well plates
Put 200 µL of 1x fluorescein into column 1 rows A, B, C, D
Put 100 µL of 1x PBS in columns 2-12 for rows A, B, C, D
Column 12 for all rows (A, B, C, D) is control and only contains 100 µL 1x PBS
Transferred 100 µL of fluorescein stock from A1 to A2
Mixed A2 > pipetting up and down 3 times
Transferred 100 µL from A2 to A3 and mixed A3
Repeated until after mixing A11 > then 100 µL removed from A11 and put into liquid waste
Process described for row A repeated for all other rows (B, C, D)
Fluorescence
1
2
3
4
5
6
7
8
9
10
11
12
A
497
266
144
75
41
23
14
9
8
6
6
5
B
532
302
153
81
44
24
25
10
7
7
5
5
C
532
295
154
81
45
25
15
10
7
7
6
5
D
547
297
159
83
45
25
16
11
8
7
6
5
Excitation: 485 nm /20 nm
Emission: 528 nm /20 nm
Optics: top
Temperature: 25.8°C (room temp)
May 23, 2018
Competent Cell Prep
Sricharan
Took the Ψa plate out around 9am
It was densely packed with colonies (tan color), but some isolated ones appeared towards one edge of the plate
Kept plate at 4°C cold room during day
Overnight cultures prepared near portable burner (to the delight of Toni watching me try and refill the burner)
Picked 3 colonies with P200 tip
Each colony into a separate round bottomed tube with loose caps, filled with Ψb media
Tubes placed in shaker at 37°C and 250rpm around 4:30pm
May 24, 2018
Competent Cell Prep
Liz, Peter, Shannon, Sricharan
Took the overnight cultures out around 9:15am
They all are cloudy compared to the original media, so decent growth it seems
Will start subcultures around noon
Cultured in shaker for an hour and 22 minutes and we over grew both cultures by a little
Prepared another overnight culture
May 24, 2018
Competent Cell Prep
Liz, Peter, Shannon, Sricharan
Took the overnight cultures out around 9:00am
Growth in both cultures
Pre-warmed Ψb media before subculturing
Subcultured in Ψb media for 45 minutes (Flask A around 0.36 OD600; Flask B around 0.38 OD600)
Checked again at 55, then 58 minutes and each was around 0.48 OD600
Spun cells in pots in MSE 18 to isolate pellet and removed supernatant
Resuspended in TfbI and spun again and removed supernatant
Resuspended in TfbII, pooled A and B, and chilled on ice for 15 minutes
Aliquoted 200 µL of cell solution into autoclaved microcentrifuge tubes and snap-froze using liquid nitrogen
Put competent cells into -80 fridge on second floor
Week 3: 5/28/2018
May 29, 2018
Interlab Transformation
Shannon, Liz
Plasmid Re-suspension
Add 10 µL dH2O to wells 2D, 2B, 2F, 2H, 2J, 2L, 2N, and 2P on plate 7 from the interlab kit and pipette to resuspend the DNA in each well. Let sit for a few minutes. Transfer to 1.5 mL tubes.
Note: 2D and 2B were accidently added to the same 1.5 mL tube, so these two were re-done using 21B and 21D from kit plate 7 from the 2017 interlab kit, which have the same corresponding DNA.
May 29, 2018
Chloramphenicol Plate Preparation
Ryan, Peter, Sricharan
Preparing Chloramphenicol Stock
Made 10 mL stock of 35 mg/mL chloramphenicol in EtOH (this is 1000x as specified by iGEM)
Took from Rachel’s floor (we bought it though), 0.3496g made up to 10mL volume
Preparing Chloramphenicol LB Plates (24)
Collected ingredients for LB media and added to beaker
Q/s’d with dH20 to 400
Mixed in beaker for 20 min and Q/s’d to about 450 mL with spray bottle to get solid particles off of the sides of the beaker and into the solution
Tried pouring into grad cylinder to finish Q/s but still goupy
Poured ~ 50 mL of dH20 into grad. Cylinder to capture the goop stuck to the bottom after pouring as much as we could back into the beaker. Poured this 50 mL back into the beaker as well
Turned up temp and stir speed on hot plate and stirred for another 25 minute in beaker
Poured into bottle and stirred (stopping frequently to shake bottle) for ~ 30 minutes
Autoclaved on S03 setting for 30 minutes (autoclaving less than a liter of liquid)
Added 0.5 mL of chloramphenicol (1000x, 35 mg/mL) to solution
Plated LB + chloramphenicol onto 24 plates and put in cold room (-4 degrees C)
Rushed putting the plates away, so there might be some condensation on plates
May 30, 2018
Interlab Transformation
Shannon, Liz, Ryan
Thawed DH5-alpha competent cells on ice
Pipet 50 μL competent cells into empty 1.5 mL tubes
Pipet 1 μL resuspended DNA (from yesterday, see page above) into individual tubes (according to well number)  with competent cells
DNA from tubes: 21B, 21D, 2F, 2H, 2J, 2L, 2N, and 2P in box 1 from freezer
Pipet 1 μL control DNA (10pg/μL) into control tube with competent cells
Incubate tubes on ice for 30 min (0:00-30:23)
Heat shock at 42℃ for 45s (30:23-31:08)
Pipet 950 μL SOC media at room temperature into each tube
Incubate at 37℃ for 1 hr at 300 rpm
Plate 100 μL transformation (lower concentration)
Condensation got on agar, tried tapping plates to remove water but some remained
Spread water with spinners before putting transformation on plates
Unsure if plates N and P got switched
Spin down cells at 6800g for 3 min and discard 800 μL supernatant
Resuspend pellet in the remaining 100 μL and plate (higher concentration)
Amount of liquid remaining for resuspension varied from tube to tube
Incubate at 37℃ for 14-18 hours
Put into incubator ~6pm
Incubator is kept at 39℃
Checked after 15.5 hours, barely any growth
May 30, 2018
Competent Cell Transformation (Scott-Simanis, Test Kit)
Peter, Sricharan
2 vials of Scott-Simanis competent cells from 5/25/18 thawed at RT (200 μL in each tube)
After thawed, placed on ice for 10 minutes
At 8 minutes, aliquoted 50 μL (150 μL from each tube) into 6 separate tubes
At 10 minutes, added 1 μL of DNA from Competent Cell Test Kit
Duplicates of 10, 50, 100 pg/μL BBa_J04450, RFP construct
Chilled aliquots on ice 30 min
Heat shocked aliquots at 42 °C for 90 s
Incubated on ice for 90 s
Added 200μL Ψb media to each aliquot
Incubated at 37 °C, 300rpm for 1 hour
Plated onto LB plates with chloramphenicol (35 μg/mL) with spinners, under portable burner
Incubated at 39 °C incubator downstairs starting at 6pm
May 31, 2018
LB + Chloramphenicol Media Preparation (w/o agar)
Peter, Sricharan, Liz, Shannon, Ryan
Used for inoculation for O.C. & dilutions + control for transformation fluorescence readings
Preparing Chloramphenicol LB media (500 mL)
Collected and measured out ingredients for LB media and added to bottle
Q/s’d with dH20 to 400 mL and shook bottle to dissolve ingredients
Stirred in beaker for 15 min
Poured into grad cylinder to finish Q/s-ing to 500 mL
Poured back into bottle and stirred for ~ a minute
Autoclaved on S03 setting for 30 minutes (autoclaving less than a liter of liquid)
Added 0.5 mL of chloramphenicol (1000x, 35 mg/mL) to solution
May 31, 2018
Interlab Overnight Cultures
Peter, Sricharan, Liz, Shannon, Ryan
Condensation on plates = osmotic stress on cells…. Not a lot of growth
Checked plates at 9am - not a lot of growth
Took plates out at 12pm, at least a few colonies on most plates with concentrated transformations, except Wells 21D (2017), 21B (2017)
Made 2 OCs for each well with most well defined/isolated colonies from each well
Put into “new” old shaker in classroom (cleaned in morning with 70% ethanol)
220rpm, 37 °C
June 1, 2018
Interlab Fluorescence Measurements
Peter, Sricharan, Liz, Shannon, Ryan, Toni
Grabbed OCs @ 9:15am from shaker
Diluted 0.5mL of OCs 10x, put foil on diluted tubes
Took Abs600 readings on plate reader (100μL samples)
Samples on Plate
1
2
3
4
5
6
7
G
21D (‘17)
2F-1
2H-1
2J-1
2L-1
2N-1
2P-1
H
2F-2
2H-2
2J-2
2L-2
2N-2
2P-2
Since Abs=ebc, and we need to dilute to a target Abs600 of 0.02, and final volume of 12mL, Valiquot=0.24/Abs600mL. Also, Vmedia = 12mL-Valiquot
Need to put in values from data on computer
Abs600
1
2
3
4
5
6
7
G
H
Valiquot(mL)
1
2
3
4
5
6
7
G
7.7
4.6
2.3
2.3
1.7
2.8
2.1
H
1.8
2.2
2.3
2.0
3.1
2.4
Valiquot(mL)
1
2
3
4
5
6
7
G
4.3
7.4
9.7
9.7
10.3
9.2
9.9
H
10.2
9.8
9.7
10.0
8.9
9.6
@12pm Put diluted & absorbance-corrected cultures back in shaker
220rpm, 37 °C for 6 hours
Used 0.5mL aliquot of these cultures for t = 0h readings
Placed on 96-well plate as in protocol
Note we don’t have negative or positive controls… Need to redo next week
June 1, 2018
LB + Chloramphenicol Plates Preparation
Peter, Ryan
To be used for competent SS cell test & positive and negative controls for Interlab
Preparing Chloramphenicol LB plates (24)
Collected and measured out ingredients for LB media and added to bottle
Q/s’d with dH20 to 400 and shook bottle to dissolve ingredients
Stirred in beaker for 30 min
Poured into grad cylinder to finish Q/s to 500 mL
Poured back into bottle and stirred for ~ a minute
Autoclaved on S03 setting for 30 minutes (autoclaving less than a liter of liquid)
Added 0.5 mL of chloramphenicol (1000x, 35 mg/mL) to solution
Let plates cool for an hour on lab bench
Put them in the hood with the plates half open w/ agar side on the bottom to get rid of condensation (should have done this immediately to dry them)
June 1, 2018
Title
Names
June 1, 2018
Title
Sricharan, Liz
3
4
5
6
7
8
9
A (absorbance)
A (fluorescence)
B (absorbance)
B (fluorescence)
C (absorbance)
C (fluorescence)
D (absorbance)
D (fluorescence)
E (absorbance)
E
F(absorbance)
F (fluorescence)
G
G (fluorescence)
H
H (fluorescence)
June
Week 4: 6/4/2018
June 4, 2018
LB + Chloramphenicol Plate Prep
Ryan
See protocol from 6/1/18 for LB + Chloramphenicol Plate Prep
Notes:
Made 23 plates (whoops)
Burned serological pipette...
… and part of a plate
June 4, 2018
Interlab Transformation of 21B and 21D
Liz
See protocol from 5/29/18 transformation
Deviations/Notes
Plates were taken out of fridge and warmed to room temperature
No condensation on plates because they were allowed to mostly cool on the bench and then transferred to the hood with the lids partially off in the process of making the plates
Only doing DNA from wells 21B and 21D (from 2017 kit plate 7)
Colonies didn’t grow on plates last time (5/29)
For each sample, made 1 dilute plate and 2 concentrated plates with the hope of actually getting colonies this time
Forgot to prechill tubes before putting in competent cells and resuspended DNA
Centrifuged at 12:45 during 30 minute incubation on ice
Only ended up making 1 concentrated plate each for 21B and 21D
Put in incubator at 4:20pm
Times
0:00-30:00 - incubate on ice for 30 minutes
30:00-30:45 - heat shock at 42℃ for 45 s
30:45-35:45 - incubate for ice for 5 min
June 4, 2018
SS Competent Cell transformation Test (Day 1)
Peter
See protocol “Competent Cell Transformation (Scott-Simanis, Test Kit)” from 5/29/18 transformation test
Deviations/Notes
Plates were taken out of fridge and warmed to room temperature
No condensation on plates because they were allowed to mostly cool on the bench and then transferred to the hood with the lids partially off in the process of making the plates
Ψb taken out of fridge to warm to room temperature
Only made samples from tube 1 for 10 pg/ μL
Took two tubes of competent cells from -80 degrees celsius
Took ~ 7 minutes to add DNA (too long for cells to be warm during this step)
Centrifuged at 12:45 during 30 minute incubation on ice (should have done this before)
Re-plated 100 pg/ μL plate 1 (labeled new plate “1b”) because burned cells with pasteur pipette for plate 1a
Potentially killed cells for 10 pg/ μL on plate 1b
Put in incubator at ~ 4:20pm
June 5, 2018
SS Competent Cell transformation Test (Day 2)
Peter
Checked cells @ 9:15 am. Some growth on 100 pg/ μL plate 1a and no visible growth on others
Checked again @ 10:15 am and same growth situation
June 5, 2018
Checking Distribution Kit Plasmids
Sricharan, Liz, Peter, Shannon
Needed to run a gel for 2017 kit 7 wells 21B and 21D to check to see if the plasmid was there because the transformation yielded no colonies again
Used 2017 kit 7 wells 21B and 21D, 2017 kit 3 well 8P, and 2017 kit 2 well 6F
Originally thought we had to do a digest of plasmids before running gel so measured concentration of DNA in samples using nanodrop (measure 1μL)
Blank with water first
Results
Kit 7 well 21B: 134.4 ng/μL
Kit 7 well 21D: 109.0 ng/μL
Kit 3 well 8P: 114.7 ng/μL
Kit 2 well 6F: 131.7 ng/μL
Made 2 small agarose gels (1%)
25mL 1x TAE, 0.25g agarose warmed in microwave
2.5 µL SYBR Safe added
All wells in gel from 2017 Kits
Well
1
2
3
4
5
6
7
8
9
10
Item
2-Log Ladder
Kit 7, Well 21D
Kit 7, Well 21B
Kit 2, Well 6F
Kit 3, Well 8D
2-Log Ladder
Test Kit 100pg/µL
Test Kit 50pg/µL
Test Kit 10pg/µL
Linearized pSB1C3
Ran 52 minutes @ 50V on mini gel rig
No condensation (new TAE)
Image on transilluminator hard to see
Similar gel image - only clear lanes were ladder and linearized backbone
Diffuse spots for plasmids in wells 2-5, we should be fine
June 6, 2018
Redoing Interlab and Scott-Siamanis Transformations
Sricharan, Liz, Peter, Shannon
Redoing transformations from earlier this week (since they didn’t work)
Using water bath at 42°C instead of heat block for shocking cells
Keeping all tubes prechilled/on ice
Prewarming SOC media
Using incubator to shake cells <300rpm instead of thermomixer
This incubator thing is probably overkill, we’ll go back to thermomixer later
Got cells (2 tubes SS, 1 tube DH5-alpha) from -80°C
SS thawed off ice ~3min
DH5-alpha thawed on ice ~5min
Immediately portioned 50µL aliquots of cells into several 1.5mL tubes
Scott-Simanis cells - samples
Competent Cell Test Kit
100pg/µL
50pg/µL
10pg/µL
Well 2F, Plate 7, 2018 Kit
DH5-alpha cells - samples
Competent Cell Test Kit, 10pg/µL
pUC19 Control
Plate 7, 2018 Kit
Well 4B (0.2 µL)
Well 4B (1 µL)
Well 4D (0.2 µL)
Well 4D (1 µL)
Well 2F (1 µL)
For SS
Incubate 10 minute on ice, add DNA (1 µL), incubate on ice 30 min
For DH5-alpha
Add DNA immediately (1 µL or 0.2 µL if indicated on sample name), incubate on ice 30 min
Heat shock @ 42°C in water bath, 45 seconds (DH5-alpha) or 90s (SS)
Accidentally dropped DH5-alpha samples into water, held in water after
Incubate on ice 5 min (DH5-alpha) or 90s (SS)
Add (prewarmed) 950µL SOC media to each DH5-alpha tube, and 200µL Ψb into each SS tube
Incubate @ 37°C in shaker in classroom ~250rpm
Plated 100µL on LB+Cam plates
For DH5-alpha, followed iGEM protocol to concentrate solutions
Only plated for DH5-alpha controls after concentrated
Incubated at 39°C downstairs ~4:30pm
June 7, 2018
Interlab Positive and Negative Controls
Sricharan, Liz, Peter, Shannon
Took plates from yesterday out ~9:30am
Good growth on DH5-alpha plates (Interlab)
Except pUC19 control, which was aliquoted after comp cell tube was dropped into ethanol
No growth on most Scott-Simanis plates (not even Well 2F)
We know cells are not competent now - plates and protocol are fine
Will do overnight cultures for Interlab cells and for new batch of competent cells this afternoon
June 7, 2018
Psi Solutions
Liz, Peter
Psi A
Substance
Concentration
Protocol Amount
Actual Amount
Bacto yeast extract
5 g/L
0.625 g
0.6357 g
Bacco tryptone
20 g/L
2.5 g
2.5090 g
MgSO4
5 g/L
0.625 g
0.6261 g
Bacto agar
14 g/L
1.75 g
1.7512 g
Fill with dH2O to just under 250 mL
This was our mistake!! We used enough chemicals to make 125 mL but put in too much water, making the solution useless
Add 1M KOH dropwise until pH reached 7.63
Mixed in beaker using stir plate
Didn’t mix fast enough so agar clumped on sides and at bottom
Shook a lot in a bottle and poured back and forth to get out as much agar as possible
Autoclaved on liquid setting
Was about to pour plates but realized that we can’t use because too diluted
Substance
Concentration
Protocol Amount
Actual Amount
Bacto yeast extract
5 g/L
2.5 g
2.50 g
Bacco tryptone
20 g/L
10 g
10.0002 g
MgSO4
5 g/L
2.5 g
2.5026 g
Fill with dH2O to just under 500 mL
Mixed in beaker using stir plate
Adjusted pH to 7.60 by adding 1M KOH dropwise
Topped off to 500 mL
Put through a sterile filter and aliquoted 100mL each into 3 flasks and the rest into a bottle
Autoclaved on liquid setting
Store in fridge (4°C)
June 7, 2018
Chloramphenicol plate prep
Shannon and Toni
See protocol from 6/1 - made 1L
Mixture was goopy when transfering from bottle to graduated cylinder to Q/s to 500mL, so had to put back in bottle and shake/continue mixing on hot plate. Rinsed graduated cylinder with dH2O to get goop back into bottle (not enough to go past 500mL).
Week 5: 6/11/2018
June 11, 2018
Interlab CFU Day 1
Sricharan, Liz, Shannon
Circled three colonies (labeled a, b, c) from plates 4B 1𝜇L and 4D 1𝜇L (made 6/6/18)
Pipetted 5 mL LB broth into OC tubes
Six tubes: 4Ba, 4Bb, 4Bc, 4Da, 4Db, 4Dc
5mL in each tube
Used 200 1𝜇L pipet tip to take colony a from plate 4B and drop into tube 4Ba
Repeat with other circeled colonies into corresponding tubes
Put into shaker at about 37℃ and 220 rpm overnight
In: 3:30 pm
June 11, 2018
SS Competent Cells Trasformation Test Day 1
Peter, Deivydas, Ryan
See protocol “Competent Cell Transformation (Scott-Simanis, Test Kit)” from 5/29/18 transformation test
Deviations/Notes
Plates were taken out of fridge and warmed to room temperature
No condensation on plates because they were allowed to mostly cool on the bench and then transferred to the hood with the lids partially off in the process of making the plates
Ψb taken out of fridge to warm to room temperature
Made one sample/ plate for each concentration (10 pg/ μL, 50 pg/ μL, 100 pg/ μL) adn well 2F from interlan study as a control
Took one tube of competent cells from -80 degrees celsius
Let ice thaw for ~ one minute
Centrifuged during 30 min. To make sure is mixed (should not have done this- cells are fragile)
June 12, 2018
SS Competent Cells Trasformation Test Day 2
Peter, Deivydas, Ryan
Little to no growth - ask Rachel what we are doing wrong
Three colonies on 100 pg/ μL
June 12, 2018
Interlab CFU Day 2
Sricharan, Liz
Took overnight cultures of positive (4B) and negative (4D) controls
Following CFU protocol
C1V1 = C2V2→ V1 = C2V2/C1 = 0.1(1000mL)/8(1:8OD600-blank OD600)
Wells
1
2
F
1 (positive control)
2 (positive control)
G
3 (negative control)
4 (negative control)
H
Blank media
Blank media
OD600
1
2
F
0.237
0.122
G
0.250
0.208
H
0.033
0.034
Culture Volume (checked OD600 with plate reader)
V1: 61μL culture, 939μL media
V2: 141μL culture, 859μL media
V3: 58μL culture, 942μL media
V4: 72μL culture, 928μL media
Shannon is making more LB+Cam media for our dilutions
Plate notation: A.B.C
A=culture number (1,2=positive control, 3,4=negative control)
B=sample number
C=dilution number (3, 4, and 5 are plated)
Diluted according to protocol, plated cells in dilutions 3-5
Reused 15mL Falcon tubes for dilutions 1-3
Incubate at 39℃ downstairs overnight
In at ~4:15
June 12, 2018
SS Competent Cells Trasformation Test Day 1
Peter
***New Protocol*** - we were doing stuff wrong
Retrieved 3 vials of Scott-Simanis competent cells (XL1 Blue from 5/25/18, 6/8/18, and Rachel’s cells) from -80 and brought upstairs on ice (200 μL in each tube)
Thawed at RT for ~ one minute
NO ALIQUOTING AS PIPETTING SHREDS FRAGILE CELLS
Added 1μL of DNA immediately after thawed (BBa_J04450 100 pg/μL from 2018 CC test kit)
NO CENTRIFUGING TO MIX DNA IN
Chilled samples on ice for 30 minutes
Heat shocked samples WITH WATER BATH at 42 °C for 45 S
Incubated on ice for ~ 90 s
Added 800μL (4 times volume of samples) 2YT media to each sample
Incubated at 37 °C ON HEAT BLOCK (NO SHAKING) for 60 min
Plated out w/ flame two plates for each sample: 100 μL on plate 1 and 200 μL on plate 2
Incubated overnight @ 37 °C
June 13, 2018
SS Competent Cells Trasformation Test Day 2
Peter
Retrieved from the incubator @ 9:15 (~ 17 hours in the incubator)
FINALLY IT WORKED MOSTLY!!!
Colony Growth:
Rachel (Date?)
iGEM 5/25/18
iGEM 6/8/18
Plate 1 (100 μL)
~ 15 colonies
1 colony??
13 colonies
Plate 2 (200 μL)
69 colonies
3 colonies
~42 colonies
June 13, 2018
Counting Interlab Colonies
Liz, Sricharan, Shannon, Peter
Dilution 3
Plate
Number of Colonies
1.1
185
1.2
28
1.3
79
2.1
Over 300
2.2
Over 300
2.3
Over 300
3.1
121
3.2
261
3.3
170
4.1
94
4.2
270
4.3
Over 300
Dilution 4
Plate
Number of Colonies
1.1
18
1.2
7
1.3
6
2.1
65
2.2
105
2.3
Over 76
3.1
42
3.2
26
3.3
12
4.1
31
4.2
24
4.3
33
Dilution 5
Plate
Number of Colonies
1.1
1
1.2
0
1.3
1
2.1
1
2.2
2
2.3
8
3.1
1
3.2
3
3.3
1
4.1
0
4.2
5
4.3
3
Week 6: 6/18/2018
June 18, 2018
Transforming Parts from Registry
Sricharan
Resuspended some parts from 2018 Distribution Kit to transform tomorrow
Plate
Well
Part
Name (BBa_)
4
17D
Anderson Promoter (Strong)
J23100
4
19D
Anderson Promoter (Medium)
J23110
4
19J
Anderson Promoter (Weak)
J23114
1
17H
Ptac
K864400
5
17J
‘tesA
K1472601
5
7K
TUDelft mtrCAB
K1316012
6
21K
ADC (Petrobrick)
K590031
6
7K
AAR (Petrobrick)
K590032
Resuspending each well w/ 10μL of dH2O, let sit a few minutes, aliquoted into 1.5mL tubes.
June 19, 2018
2xYT Medium
Liz
Wanted to make 500mL for use for transformations/potentially overnight cultures (point is we don’t need a ton)
In a bottle measured:
450mL dH2O
8g tryptone (measured 8.00g)
5g yeast extract (measured 5.00)
2.5g NaCl (measured 2.50)
Shook bottle between adding each substance to dissolve
When dissolve adjusted pH to 7.0 with 5M NaOH
Added dH2O up to ~480mL before adjusting pH
Only had 1M NaOH
Had to use pH 4 calibration solution because start out acidic
Top off to 500mL
Autoclave for 20 min at 15 psi (1.05kg/cm3) on liquid cycle
Used 30 minute liquid setting
Put in cold room to cool before using for transformations
June 19, 2018
Making CAM and Amp plates
Peter
***See protocol from 6/1/18***
Notes/Deviations
After autoclaving, split 500 mL into two flasks of 250 mL each
Added 250 μL of chloramphenicol (35 mg/ mL stock) to one flask
Added 250 μL of Ampicillin (35 mg/ mL stock) to other flask
June 19, 2018
SS Competent Cells Trasformation Test Day 1
Peter, Shannon, Charan
Measured 0.5043g ampicillin (Na salt) from Rachel’s lab to make 100mg/mL stock solution (1000x) in 50% EtOH
***See protocol from 6/12 (week 5)***
Notes/Deviations
Retrieved 8 vials of second set of Scott-Simanis competent cells (XL1 Blue from 6/8/18) from -80 and brought upstairs on ice (200 μL in each tube)
DNA added: BBa_J23100, BBa_J23110, BBa_J23114, BBa_K864400, BBa_K1472601, BBa_K1316012, BBa_590031, BBa_590032
Heat shocked four at a time- tried to fit all 8 in at once but four hands wouldn’t fit in the water bath to hold them all so 4 were in briefly then taken out and put back on ice until the the first four were finished and this could have caused problems
Used 2YT media (richer than Ψb)
Did not originally add 800μL media to tube with BBa_590031 tube and did not realize until it was on heat block for ~10 minutes and then added so this could have caused problems
Plated out 200 μL of cells + media except for BBa_K1472601 and BBa_J23100 which we did two plates, one 200 μL and one 300 μL
Trouble keeping flame going (low on butane)
Put in 37 °C incubator at 4:20 p.m.
June 20, 2018
SS Competent Cells Trasformation Test Day 2
Peter
Notes
Observed growth on both the 200 μL and 300 μL plate for BBa_K1472601 and BBa_J23100, the BBa_J23110 plate, and the BBa_J23114 plate
~ 15 colonies (give or take 5) on average per plate
Will redo transformations of parts that were not successful (BBa_K864400, BBa_K1316012, BBa_590031, BBa_590032)
June 20, 2018
SS Competent Cells Trasformation Test Day 1 Repeat
Liz, Toni
Also resuspended BBa_B0015 from 2018, Plate 3, Well 3F (Cam resistance)
Retrieved 5 vials of SS competent cells from -80°C downstairs on ice
competent cells from 6/8/18
Let cells thaw at room temp for ~1 min
Added 1 μL DNA (K1316012, K590032, K864400, K590031, BBa_B0015) to each tube
Competent cell tube for K590031 was empty, had to go downstairs and get another tube so everything was done 6.5 minutes late
Chilled on ice for 30 minutes
K590031 started ice at 6:36 on stopwatch and went till 36:40
Heat shock in 42°C water bath for 45s (tubes held in water bath)
Added 800μL 2xYT media to each tube
Media prewarmed in warm room
Incubated at 37°C on heat block for 60 min
Heat block temporarily dipped to 35°C then went back up to 37°C
Flick tubes every 10 min
K1316012, K590032, K864488, BBa_B0015 in at 2:32, K590031 in at 2:35
Tubes removed from heat block and spun down at 6800g for 3 minutes ~800μL supernatant removed and discarded and cells resuspended in remaining media Cells plated and put in 39°C overnight (in at ~4ish)
Flame ran out of butane
June 21, 2018
SS Competent Cells Trasformation Test Day 1
Peter, Ryan
***See protocol from 6/12 (week 5)***
Notes/Deviations:
Retrieved 4 vials of second set of Scott-Simanis competent cells (XL1 Blue from 6/8/18) from -80 and brought upstairs on ice (200 μL in each tube)
DNA added: BBa_K864400, BBa_K1316012, BBa_590031, BBa_590032
Used 2YT media (richer than Ψb)
Plated out 200 μL of cells per plate
Used beads in hood instead of flame w/ pasteur pipette
June 22, 2018
Making Media for iGEM Competent Cells
Shannon and Ryan
Made 1L SOB and 1L CCMB80 according to the iGEM protocol.
SOB
Reagent
Amount (g)
Measured Amount (g)
Yeast extract
5
5.0006
Tryptone
20
20.0009
NaCl
0.584
0.5846
KCl
0.186
0.1886
MgSO4 * 6H2O
4.93
4.9333
CCMB80
Reagent
Amount (g)
Measured Amount (g)
KOAc
5mL of 1M stock (0.49g)
0.4938
CaCl2 anhyd.
4.44
4.4473
MnCl2 * 4H2O
2.0
2.0062
MgCl2 * 6H2O
1.0
1.0081
Glycerol (100%)
5mL
5mL
SOB was pH adjusted to pH 7.54
100mL SOB was aliquoted and 1.6078g agar was added and autoclaved, and then used to make 5 plates. The agar did not dissolve when stirred, but did dissolve in the autoclave. These plates were left in the hood for the weekend.
At first, we tried to make only 500mL of CCMB80, but we used 1M HCl, which decreased the pH much too rapidly, bringing it to around pH 5.8 after 1 drop. So, we doubled the amount and used 0.1M HCl. The final pH was 6.35.
June 22, 2018
Glycerol Stocks for Overnight Cultures
Liz
Made 40% glycerol stock (25mL) in blue falcon tube
10mL 100% glycerol
15mL DI water
Store extra in freezer (-20)
Put 0.75mL 40% glycerol into 10 microcentrifuge tubes (2 tubes per overnight culture, labeled as such)
Put 750 microliters of one culture into two different microcentrifuge tubes
Repeat for each culture, should have 2 microcentrifuge tubes per culture
Pipet up and down in culture to mix cells before transferring
Vortexed tubes to mix
Stored in -80 where DH5alpha cells used to be in a blue box
Week 7: 6/25/2018
June 25, 2018
Zyppy miniprep
Liz and Peter
Centrifuge tubes labeled:
*need to look this up*
Added 600 microliters culture to labeled 1.5 mL microcentrifuge tubes
Added 100 microliters 7x lysis buffer (blue) and mixed by inverting 4-6x
Turns opaque to clear blue when lysis is done
*time sensitive!!!* add 350 microliters neutralization buffer (yellow) to each tube
Neutralization buffer should be kept cold
Mixture turns yellow and forms yellow precipitate when neutralized
Invert 2-3x after color change
Centrifuged at 16,000rpm for 3 min then again at 11,000rpm for 1 minute
Pellets got disturbed and had to be repelleted)
Supernatant (~900 microliters) transferred into zymo-spin IIN column (don’t disturb cell pellet)
Put column into collection tube and centrifuge 15 s
Discarded flow-through and put column back into collection tube
Added 200 microliters Endo-wash buffer and centrifuged for 30s
Added 400 microliters zyppy wash buffer and centrifuged 1 min
Put column into clean 1.5mL tube and added 30mL zyppy elution buffer
Let sit 1 minute at room temp
Centrifuged 30s to elute plasmid DNA
June 25, 2018
Primer Rehydration
Liz and Peter
Spun down dry tubes
Added 1xTE buffer to make 100 micromolar solutions
primer
Amount TE added (microliters)
SB prep-3P-1
201
SB prep-2Ea
221
fadD_P1F
201
fadD_P1R
231
Vortexed
Stored in -20 degree freezer in primer box
June 25, 2018
SOB plates
Ryan
SOB plates failed/ dried up because they were left in the hood over the weekend (agar was thin and had a wavy texture, we felt it was better to make new plates).
100mL SOB was aliquoted and 1.6 g agar was added
The solution was heated and mixed, and then autoclaved for an hour (set at liquids 30, P03)
The agar did not completely dissolve when stirred, but did dissolve in the autoclave.
This was used to make 5 plates, which were of proper thickness
June 26, 2018
Making new competent JM109 cells
Liz and Peter
Streaking a SOB plate w/ Rachel’s JM109 cells (SOB plates specific to Top10 cells…)
Put in incubator at 1:45 p.m.
Results (6/27): good growth on plate
June 26, 2018
PCR of fadD
Liz, Peter, and Shannon
Thermocycler Settings
Cycle
Time
Temp. (C)
Initial denaturation
30s
98
35 Cycles
10s
98
30s
61
20s for fadD (should have been 40 s because 2kb) 50s for backbone
72
Final extension
10 min
72
Hold
indefinitely
4
Notes
For 61C during the 35 cycles, the time was found using NEBTm calculator
fadD was accidently put at 20s instead of 40s at 72C during the 35 cycles
We aren’t sure if fadD was held at 4C because it didn’t feel cold when opened
Add to PCR tube (for fadD):
Component
Amount (µL)
Nuclease-free water
12.2 (20 - sum of other components)
Phusion buffer (HF)
4
10mM dNTPs
0.4
10mM forward primer
1
10mM reverse primer
1
Template (E. coli) DNA
0.56
Phusion polymerase (add last)
0.2
DMSO
0.6
Notes
Made a 10mM stock solution for dNTPs (1µL of dATP, dTTP, dCTP, dGTP)
Diluted the forward and reverse primer to 10µM 100µM(V) = 10µM(1µL)
2x[0.1µL stock, 0.9µL H2O]
For template DNA, needed 40ng, stock was 40ng
40ng x 1µL/71ng = 0.56µL
Procedure repeated for backbone
4 backbones (PSB1A3, PSB1K3, PSB1T3, PSB1C3), put into 4 labeled PCR tubes
For primers use SB Prep-3P-1 and SBPrep - 2Ea
Annealing temp is 69C
Template DNA: 40ng x 1µL/25ng = 1.6 µL = 1.6 µL backbone DNA
Add to each tube:
Component
Amount (µL)
Nuclease free water
11.2
Backbone DNA
1.6
Phusion buffer (HF)
4
dNTPs
0.4
10mM forward primer
1
10mM reverse primer
1
DMSO
0.6
Phusion polymerase (add last)
0.2
Note: thermocycler did not feel cold when taking out, don’t know if this actually worked
June 27, 2018
PCR Purification
Liz and Shannon
Used protocol from PureLink PCR Purification Kit
Pipet 20µL PCR sample (PSB1T3, PSB1K3, PSB1C3, PSB1A3) into individual 1.5mL tubes
Add 80µL Binding Buffer 2 (B2) to each sample (4x volume) - pipette to mix
B2 is used for purifying PCR fragments
Load each solution into individual spin column collection tubes. Centrifuge at 10,000g for 1 min. Discard flow through and re-insert the column into the collection tube.
Add 650 µL wash buffer (W1) (ethanol had already been added). Centrifuge at 10,000g for 1 min. Discard flow-through and re-insert the column into the collection tube.
Re-centrifuge at max speed (16,100) for 3 min.
Using a clean collection tube, add 50µL Elution Buffer (E1) to column. Let sit for 1 min, then centrifuge at max speed for 2 min.
The purified PCR fragments are in the collection tube.
Products were nanodropped:
PSB1T3: 20.1ng/µL
PSB1K3: 7.8ng/µL
PSB1C3: 21.1ng/µL
PSB1A3: 23.3ng/µL
June 27, 2018
JM109 Overnight Culture
Shannon
SOB was not coming to room temp. Fast enough, so it was put in the warm room for about 10 min.
Used at about 21.1C
Put 5mL SOB into 2 round-bottom tubes and used a non-filtered pipette tip to inoculate the tubes with 2 colonies
Put in downstairs shaker overnight
Note: forgot to do this in the hood!
June 27, 2018
PCR of phz from Genomic DNA (1F-1R, 2F-3R, 4F-4R, 6F-6R)
Peter and Liz
Thermocycler Settings:
Cycle
Time
Temp. (C)
Initial denaturation
30s
98
35 Cycles
10s
98
30s
64.5
2F-3R: 54 s
1F-1R, 4F-4R, 6F-6R: 25s
72
Final extension
10 min
72
Hold
Indefinitely
4
Notes:
1F-1R: 0.5 kb
2F-3R: 1.8 kb
4F-4R: 0.9 kb
6F-6R: 1 kb
Add to PCR tubes:
Component
Amount (µL)
Nuclease-free water
11.97  (20 - sum of other components)
Phusion buffer (HF)
4
10mM dNTPs
0.4
10mM forward primer
1
10mM reverse primer
1
Template Genomic DNA (Pseudomonas aeruginosa)
0.83 (40 ng - stock is 48.3 ng/ µL)
Phusion polymerase (add last)
0.2
DMSO
0.6
Note: Started machine 2 on a 50 µL reaction for ~3 seconds and then stopped it and within 10 seconds started again for a 20 µL reaction
June 27, 2018
PCR purification and gel of backbones and phz
Liz
Made a gel for PCR products
25mL 1x TAE buffer and 0.25g agarose in flask and microwaved until dissolved
Let flask cool until it could be picked up
Added 2.5µL SYBR Safe
Poured into mold, put in comb, and let harden under ice bin to protect SYBR Safe from light
Purified PCR products (PureLink PCR Purification Kit)
Pipet 20µL PCR sample (1F/1R, 2F/3R, 4F/4R, 6F/6R) into individual 1.5mL tubes
Add 80µL Binding Buffer 2 (B2) to each sample (4x volume) - pipette to mix
B2 is used for purifying PCR fragments
Load each solution into individual spin column collection tubes
Centrifuge at 11,000g for 1 min and discard flow through
Add 650 µL wash buffer (W1) (ethanol had already been added)
Centrifuge at 11,000g for 1 min and discard flow-through
Centrifuge at max speed (16,100) for 3 min.
Add 50µL Elution Buffer (E1) to column and let sit for 1 min
Centrifuge at max speed (16,100) for 2 min.
The purified PCR fragments are in the collection tube
Nanodrop (blanked with elution buffer)
1F/1R: 5.5 ng/µL
2F/3R: 11.5 ng/µL
4F/4R: 6.4 ng/µL
6F/6R: 6.3 ng/µL
Readings were very inconsistent (can we trust the nanodrop?)
Gel electrophoresis
For each sample mixed 4µL water, 1µL loading dye, and 1µL PCR product
Filled rig with 1x TAE buffer then loaded
1
2
3
4
5
6
7
8
9
10
Ladder (2-log)
pSB1A3
pSB1C3
pSB1K3
pSB1T3
1F/1R
2F/3R
4F/4R
6F/6R
Ladder (2-log)
Filled rig with 1x TAE buffer then loaded
1
2
3
4
5
6
7
8
9
10
Ladder (2-log)
pSB1A3
pSB1C3
pSB1K3
pSB1T3
1F/1R
2F/3R
4F/4R
6F/6R
Ladder (2-log)
Gel did not come out
June 28, 2018
PCR troubleshooting
Liz and Peter
Redid PCR protocol for fadD and phz 2F/3R but used new dNTPs
Cycle
Time
Temp. (C)
fadD
phz 2F/3R
fadD
phz 2F/3R
30s
30s
98
98
10s
10s
98
98
40s
54s
72
72
10 min
10 min
72
72
infinite
infinite
4
4
Annealing temperatures from NEB annealing temperature calculator
Set to 20µL reaction
Labeled PCR tubes fadD and phz 2F/3R
Component
Amount (µL)
fadD
phz 2F/3R
Nuclease free water
12.2
11.97
Template DNA (40ng)
0.56*
0.83*
Phusion buffer (HF)
4
4
dNTP stock
0.4
0.4
10 mM forward primer
1
1
10mM reverse primer
1
1
DMSO
0.6
0.6
Phusion polymerase (add last)
0.2
0.2
Calculations
fadD: 40ng x 1µL/71ng = 0.56µL
phz 2F/3R: 40ng x 1µL/48.3ng = 0.83µL
New dNTP stock diluted (0.2µL each dNTP and 7.2µL dH2O) because old one had been thawed/refrozen too many times
Made a gel for PCR products
12.5mL 1x TAE buffer and ~0.125g agarose in flask and microwaved until dissolved
Can’t tell if gel is 1% because scale measures to nearest 0.05g
Let flask cool until it could be picked up
Added 1.75µL SYBR Safe
Poured into mold, put in comb, and let harden under ice bin to protect SYBR Safe from light
June 28, 2018
PCR troubleshooting purification/gel
Liz and Peter
Purified PCR products (PureLink PCR Purification Kit)
Pipet 20µL PCR sample (fadD and phz 2F/3R) into individual 1.5mL tubes
Add 80µL Binding Buffer 2 (B2) to each sample (4x volume) - pipette to mix
B2 is used for purifying PCR fragments
Load each solution into individual spin column collection tubes
Centrifuge at 11,000g for 1 min and discard flow through
Add 650 µL wash buffer (W1) (ethanol had already been added)
Centrifuge at 11,000g for 1 min and discard flow-through
Centrifuge at max speed (16,100) for 3 min
Add 50µL Elution Buffer (E1) to column and let sit for 1 min
Centrifuge at max speed (16,100) for 2 min
The purified PCR fragments are in the collection tube
Nanodrop (blanked with elution buffer)
fadD: 11.2 ng/µL
phz 2F/3R: 11.3 ng/µL
Gel electrophoresis
For each sample mixed 4µL water, 1µL loading dye, and 1µL PCR product
Topped off rig with 1x TAE buffer then loaded
Ran at 50V for 50 min
1
2
3
4
5
Ladder (2-log)
empty
phz 2F/3R
empty
fadD
June 29, 2018
Backbone and phz PCR
Liz and Peter
Parts to PCR (1 PCR tube per sample):
phz 1F/1R (510bp)
phz 4F/4R (946bp)
phz 6F/6R (997bp)
pSB1A3 (2155bp)
pSB1C3 (2070bp)
pSB1K3 (2204bp)
pSB1T3 (2461bp)
Set thermocycler:
Cycle
Time
Temp. (C)
phz
backbones
phz
backbones
30s
30s
98
98
10s
10s
98
98
30s
30s
64.5
69
25s
50s
72
72
10 min
10 min
72
72
infinite
infinite
4
4
Added to each tube:
Component
Amount (µL)
phz
backbones
Nuclease free water
11.97
11.2
Template DNA (40ng)
0.83*
1.6*
Phusion buffer (HF)
4
4
dNTP stock
0.4
0.4
10 mM forward primer
1
1
10mM reverse primer
1
1
DMSO
0.6
0.6
Phusion polymerase (add last)
0.2
0.2
Calculations
backbones: 40ng x 1µL/25ng = 0.56µL
phz: 40ng x 1µL/48.3ng = 0.83µL
Tubes kept on ice and vortexed before putting into thermocycler
Gel electrophoresis
For each sample mixed 4µL water, 1µL loading dye, and 1µL PCR product
Filled rig with 1x TAE buffer then loaded
1
2
3
4
5
6
7
8
9
10
Ladder (2-log)
phz 1F/1R
phz 4F/4R
phz 6F/6R
Ladder (2-log)
pSB1A3
pSB1C3
pSB1K3
pSB1T3
Ladder (2-log)
Gel didn’t get imaged correctly first time so redid gel on 7/2/2018:
1
2
3
4
5
6
7
8
9
Ladder (2-log)
pSB1A3
pSB1C3
pSB1K3
pSB1T3
phz 4F/4R
phz 6F/6R
phz 1F/1R
Ladder (2-log)
Ran at 50V for 30+20 min (time in between because didn’t know current stops when timer does)
July
Week 8: 7/2/2018
July 2, 2018
iGEM Competent Cell Prep, Day 1
Liz
Going to try with 1 tube JM109 cells and 1 tube NEB Gibson cells
Need to use all detergent-free glassware and rinsed plasticware
Autoclaved glassware filled ¾ of way with dH2O
Put 250mL SOB media into each of two 500mL flasks (labeled JM109 and NEB cells)
Measured/poured in hood because no antibiotics in SOB
Put media into warm room about 20 minutes before pouring, not sure if this is necessary
Put 1 tube thawed JM109 cells into JM109 flask and 1 tube thawed NEB Gibson cells into NEB flask and covered with foil
Thawed tubes in hand
Pipetted cells into media (not competent yet so shouldn’t kill cells)
Put ice in bottom of ice bucket and left flasks of cells in ice bucket overnight
Want OD600 of 0.3 (should be about 16 hrs at 20 degrees C)
Start at 5:00pm
July 3, 2018
iGEM Competent Cell Prep, Day 1
Liz, Shannon
Pre-chilled 2 flat-bottomed centrifuge bottles and preset centrifuge for 4 degrees C
Checked OD600 of overnights (want at 0.3):
Time
7:38
8:03
8:39
9:15
9:56
10:30
11:18
11:52
12:34
JM109
0.005
0.006
0.017
0.015
0.026
0.054
0.113
0.179
0.294
NEB cells
0.015
0.016
0.022
0.020
0.024
0.028
0.039
0.055
0.081
After first measurement, kept in warm room
Gave up on NEB cells (not growing)
Put equal amounts JM109 culture into two flat-bottomed centrifuge bottles
Centrifuge for 3000g at 4 degrees C for 10 min
Decant supernatent
Resuspend in 80mL ice cold CCMB80 buffer
40mL each tube
Resuspend in 20mL then pipet other 20mL against walls
Incubate on ice 20 min
Centrifuge at 3000g at 4 degrees C for 10 min
Decant supernatent
Resuspend pellet in 10mL ice cold CCMB80 (10mL is total, put 5mL in each bottle then combined bottles)
Was supposed to check OD of 800μL SOC and 200μL culture using spectrophotometer (blank with 800μL SOC and 200μLCCMB80) then adjust to 1.0-1.5 using CCMB80 but OD started at 0.532 so just skipped this step
Incubated on ice (was supposed to be 20 minutes but ended up being much longer because was trying to figure out the protocol)
Aliquotted 50μL per tube over liquid nitrogen (suspended over top to be cold but not too cold like flash freezing)
Made two batches because liquid nitrogen evaporated
Stored in -80 freezer
July 3, 2018
fadK PCR
Liz
Resuspended and diluted primers
1F: added 471μL to resuspend, to dilute to 10mM added 1μL resuspended primer and 9μL milliQ water
1R: added 339μL to resuspend, to dilute to 10mM added 1μL resuspended primer and 9μL milliQ water
Thermocycler Settings
Cycle
Time
Temperature (degrees Celsius)
Initial denaturation
30s
98
35 cycles
10s
98
30s
63
40s
72
Final extension
10 min
72
Hold
Infinite
4
Added to PCR tube:
Component
Amount (µL)
Nuclease-free water
12.2 (20 - sum of other components)
Phusion buffer (HF)
4
DMSO
0.6
10mM forward primer
1
10mM reverse primer
1
dNTPs
0.4
Template (E. coli) DNA
0.56
Phusion polymerase (add last)
0.2
July 5, 2018
fadK PCR purification and Nanodrops
Liz, Sricharan
Used protocol from 6/28 (1 PCR tube for fadK)
Ran 10 Nanodrops (blanked with elution buffer (Buffer E1) from Invitrogen Purif Kit) of PCR purif (4 phz parts, fadD, fadK, 4 linear plasmid backbones)
All ~10-15 ng/µL, good characteristic curves
Sample Name
Conc. (ng/µL)
260/280
260/230
phz1F-1R Purif
13.31
2.04
1.75
phz2F-3R Purif
11.83
1.89
1.53
phz4F-4R Purif
15.53
1.97
1.77
phz 6F-6R Purif
10.65
2.13
1.46
fadD Purif
8.03
1.79
2.49
fadK Purif
14.49
1.69
1.84
pSB1C3 Purif
15.01
2.01
1.72
pSB1T3 Purif
15.56
1.91
1.70
pSB1K3 Purif
12.88
1.94
1.99
pSB1A3 Purif
10.55
2.11
1.78
July 5, 2018
resuspension of phenazine gBlocks
Liz
Needed to resuspend phz4-5 and start of phz for gibson assembly
Centrifuged for 5s to make sure all of product is in bottom of tube
Added 1x TE buffer to get concentration to 10ng/μL
Phz4-5 (1000ng)
1000ng x 1μL/10ng = 100μL
Start of phz (500ng)
500ng x 1μL/10ng = 50μL
Vortexed
Incubated in hat block at 50 degrees C for 20 min
Vortexed and centrifuged
July 5, 2018
phz gibson PCR and fadD/fadK secondary PCR
Liz, Sricharan
Thermocycler Settings
Phz gibson
fad
Time
Temperature (℃)
Time
Temperature (℃)
Initial denaturation
30s
98
30s
98
35 cycles
10s
98
10s
98
30s
61
30s
72
60s
72
40s
72
Final extension
10 min
72
10 min
72
Hold
infinite
4
infinite
4
Add to PCR tubes:
Component
Amount (µL)
Phz gibson
fadD
fadK
Nuclease-free water
11.3 (20 - sum of other components)
7.8 (20 - sum of other components)
10.04 (20 - sum of other components)
Phusion buffer (HF)
4
4
4
DMSO
0.6
0.6
0.6
10mM forward primer
1 (1F)
1 (fadD bad start)
1 (fadD bad start)
10mM reverse primer
1 (1R)
1 (fadD bad end)
1 (fadD bad end)
dNTPs
0.4
0.4
0.4
Template (E. coli) DNA
1.5
5.0
2.76
Phusion polymerase (add last)
0.2
0.2
0.2
Gibson Assembly using the NEB kit … wanted 0.2pmol of DNA total (kit specifies 0.2-1pmol for 4-6 pieces)
Start of phz (gBlock)
0.05pmol x 50µL/1.950pmol = 1.3µL
phz 1F-1R Purif
0.05pmol x 157,303pg/1pmol x 1ng/1000pg x 1µL/13.3ng = 0.59µL
phz 4-5 (gBlock)
0.05pmol x 100µL/2.057pmol = 2.4µL
phz 2F-3R Purif
0.05pmol x 561,368pg/1pmol x 1ng/1000pg x 1µL/11.8ng = 2.4µL
For Gibson mix, mixing total volume of 6.69 µL fragments + 10 µL NEB Gibson Master Mix + 3.31 µL dH20
Above solution incubated at 50°C for 60 min then at 4°C for infinite hold.
Week 9: 7/9/2018
July 9, 2018
Resuspending G-Blocks
Shannon
Resuspended phz 1-2, cymA, mtr start, and tesA using TB
Centrifuge briefly to make sure all products were at the bottom of the tube
Note: Forgot to do this for phz 1-2 (ex. Phz 1-2 had 500ng in original tube, so 500ng x 1µL/10ng = 50µL)
G-Block
Amount of TB (µL)
Phz 1-2
50
cymA
100
Mtr start
25
tesA
100
Vortex briefly
Incubate at 50C for 20 min in heat block
Note: forgot to pre-heat heat block so they sat at room temp for a few minutes
Briefly vortex and centrifuge
Stored at -20C
July 9, 2018
Restriction Digest
Shannon
Used benchling protocol to digest fadD, tesA, cymA, psb1C3, and mtr start
All enzymes checked on nebcloner.neb.com/#!/redigest for incubation temperatures and “time-saver” (they can be incubated at 37C for 10 minutes rather than 1hr) and to verify that CutSmart could be used as the buffer
Add reagents in the order: water → buffer → DNA → enzymes
fadD/tesA/cymA digested with ecoRI and pstI
Component
Amount (µL)
ecoRI
1
pstI
1
CutSmart
2
DNA
5.8 µL for fadD
10 µL for tesA and cymA
dH2O
10.2 µL for fadD
6µL for tesA and cymA
Calculations:
For water: 20 µL - volume of all other components
DNA: wanted 100ng, so calculated volume ex. 100ng tesA x 1µL/10ng = 5.8µL
Psb1C3 digested with dpnI, ecoRI, and pstI
Component
Amount (µL)
dpnI
1
ecoRI
1
pstI
1
CutSmart
2
DNA
4
dH2O
13
Mtr start digested with ecoRI, bspHI
Component
Amount (µL)
ecoRI
1
bspHI
1
CutSmart
2
DNA
10
dH2O
6
Centrifuge briefly
Heat block at 37C for 10min
Heat block at 80C for 15min (heat-kill enzymes)
Store at 20C
July 9, 2018
fadK, phz gibson PCR (try 2), phz Beg PCR Assembly
Liz, Sricharan
No bands on gel for fadK and phz gibson 1 (7/15) so redid PCR
Notes/changes:
Did two tubes for phz (one uses HF buffer and other uses GC buffer)
Phz 35 cycles time 3 changed to 45s because doing PCR assembly at the same time
Also tried to use PCR assembly instead of Gibson for phz Beg
Just add phz 1F-1R, phz 4-5, and phz 2F-3R to PCR mix and they should assemble together in theory, Phusion does all the gaps
Need 40ng total template (later realized this should be less, 40ng is for genomic DNA)
Since construct is 300bp, this is ~0.02 pmol - using quantities from 7/5/2018
Followed PCR purification protocol from 6/28
Nanodropped pure products
pure_fadD: 17.24 ng/µL
pure_fadK: 3.57 ng/µL
pure_fadK_try2: 4.81 ng/µL
pure_phzgibson1: 17.04 ng/µL
Lane
1
2
3
4
5
Sample
Ladder
fadD PCR Purif
fadK PCR Purif
Phz Beg Gibson PCR
On a 1% agarose gel in TAE with SYBRSafe, as usual, run at 50V for 50min
July 10, 2018
Ligation of cymA, tesA, and fadD
Shannon and Toni
Did a ligation to insert cymA, tesA, and fadD, separately, into psB1C3 backbones
Using ligation protocol on benchling, calculated ng of insert dna needed via link
Sample
Insert DNA Length
Vector DNA Length
Vector DNA Mass
Insert DNA Mass
Ratio
tesA
800kb
2070kb
30ng
34.78ng
3:1
Fad D
1713kb
2070kb
30ng
49.65ng
2:1
cymA
773kb
2070kb
30ng
33.61ng
3:1
tesA
fadD
cymA
Added
4 µL
1.1 µL
4.3 µL
dH2O
2 µL
2 µL
2 µL
T4 buffer
6 µL
6 µL
6 µL
psB1C3 backbone
7 µL
9.9 µL
6.7 µL
DNA insert
1 µL
1 µL
1 µL
T4 ligase
Total volume of soln: 20ul; 100ng/20ul (20ng/ul) of stock vector/inserts
All reagents kept on ice and added in the order listed
Incubated at RT for 10 min
Incubated at 65C for 20 min
Placed immediately into -20C
July 10, 2018
PCR purification
Liz
Samples (4):
Phz gibson 1 product
With HF buffer
With GC buffer
fadK (secondary PCR)
Phz assembly (PCR assembly)
Followed protocol from 6/28
Nanodropped purified product
Phz gibson HF: 5.93 ng/µL
Phz gibson GC: 10.77 ng/µL
fadK: 9.28 ng/µL
Phz assembly: 20.11 ng/µL
Ran gel at 50V for 42 min
1
2
3
4
5
ladder
Phz gibson GC
Phz gibson HF
fadK
Phz PCA
July 10, 2018
MIT Competent Cells
Liz, Sricharan, Shannon, Toni
Using protocol given to us at NEGEM by Brian Teague of MIT
2 5mL OCs of JM109 were put in 2xYT the day before
Put 800µL JM109 overnight cultures into 400mL LB broth and put into shaker (37 degrees C, 250rpm) at 9:30am
Making transformation buffers (TfbI, 150mL; TfbII, 20mL):
TfbI Needed
TfbI Actual
TfbII Needed
TfbII Actual
RbCl
1.8g
1.802g
0.024g
0.0235g
MnCl2
1.5g
1.508g
-
-
CaCl2
0.166g
0.171g
0.166g
0.166g
KOAc
0.442g
0.448g
MOPS
-
-
0.042g
-
Glycerol
22.4mL
Measured w/grad cyl
3.0mL
Measured with serological pipet
Checked OD:
Time
11:51
13:15
OD600
0.156
>1
Failure - try again tomorrow; using TfbI and TfbII made today, which is not ideal but we’re also running out of RbCl
July 11, 2018
MIT Competent Cells
Liz, Sricharan, Peter
2 5mL OCs of JM109 were put in LB the day before
Put 800µL JM109 overnight cultures into 380mL LB broth and put into shaker (37 degrees C, 250rpm) at 7:30am
Want OD600 of 0.5-0.7
Checked OD:
Time
9:25
10:02
10:22
10:41
10:46
10:54
OD600
0.056
0.164
0.289
0.416
0.506
0.565
July 11, 2018
Transformation of Competent Cells
Peter and Shannon
Did a transformation (iGEM protocol) with GFP to test that the cells are competent
Pre-chill 3 1.5mL tubes of 50µL competent cells - kept on ice while labeling/pipetting
Add 10, 50, and 100 pg/µL of RFP construct to tubes
Incubate on ice for 30 min
Heat shock for 45 sec in 37C water bath
Incubate 5 min on ice
Add 950 pre-warmed 2YT to each tube
Incubate 1 hr at 37C on heat block
Note: forgot to do a control - will do tomorrow
July 12, 2018
PCR of FadK using GC buffer
Shannon
Thermocycler Settings
Phase
Time (sec)
Temp. (C)
Initial denaturation
30
98
35 cycles
10
98
30
72
40
72
Final extension
10
72
Hold
∞
4
Add to tubes:
Component
Amount (µL)
dH2O
10
GC buffer
4
DMSO
0.6
10mM forward primer (bad)
1
10mM reverse primer (bad)
1
dNTP mix
0.4
fadK purified PCR (template)
2.76
phusion
0.2
All components were kept on ice
July 12, 2018
iGEM Transformations
Liz, Sricharan
Tubes:
cmmA-H (biobrick, resuspended in 10µL milliQ water and stored in Registry Parts 2018 box in -20C)
RFP 25s HS
RFP 45s HS
RFP 60s HS
RFP 90s HS
Thawed competent cells on ice (~6 min)
Put 1µL DNA into each corresponding tube
Incubated on ice for 30 min
Heat shocked in 42C water
RFP 25s HS: 25s
RFP 45s HS, ccmA-H: 45s
RFP 60s HS: 60s
RFP 90s HS: 90s
Incubated on ice for 5 min
Added 450µL SOC media to each tube
Incubated in shaker (37C, 250rpm) for 1 hr
Spun down cells at 6800g for 3 min and discarded ~350-400mL
Resuspended cells and plated remaining 100µL in hood using pipet tip to spread
Incubated overnight at 37C (put in ~4:15)
Took plates out ~9:00 7/13 - it worked (only for ccmA-H)!!
July 12, 2018
Gibson of phz 1-2, 4F-4R, 6F-6R
Peter, Ryan
Set up the following reaction on ice:
phz 1-2 (gBlock): 0.045pmol x 50µL/1.180pmol = 1.91µL
phz 4F-4R Purif: 0.045pmol x 614,900pg/1pmol x 1ng/1000pg x 1µL/15.53ng = 1.78µL
phz 6F-6R Purif: 0.045pmol x 584,050pg/1pmol x 1ng/1000pg x 1µL/10.65ng = 2.74µL
For Gibson mix, mixing total volume of 6.43 µL fragments + 10 µL NEB Gibson Master Mix + 3.57 µL dH20
Above solution incubated at 50°C for 60 min then at 4°C for infinite hold.
July 12, 2018
PCR Cleanup of FadK
Shannon
Combine 80µL B2 buffer (4x the amount of PCR sample) to 20µL PCR sample
Transfer to collection tube, centrifuge at 10,000 rpm for 1 min. Discard flow-through. Re-insert column into collection tube.
Add 650µL wash buffer with ethanol. Centrifuge at 10,000 rpm for 1 min. Discard flow-through. Re-insert column into collection tube.
Centrifuge at max speed (16,100rpm) for 3 min. Discard flow-through. Insert column into a clean 1.7mL elution tube.
Add 50µL elution buffer to center of column.
Incubate at room temperature for 1 min.
Centrifuge at max speed for 2 min.
The elution tube now contains purified PCR product!
Store at -20C.
July 13, 2018
PCR of Gibson assembly from 7/12 (Phz 4F-4R, Phz 1-2, Phz 6F-6R)
Peter, Ryan
Thermocycler Settings
Phase
Time (sec)
Temp. (C)
Initial denaturation
30
98
35 Cycles
10
98
30
67
78
72
Final Extension
10
72
Hold
∞
4
Component
Amount (µL)
dH2O
10.15
GC buffer
4
DMSO
0.6
10mM forward primer (bad)
1
10mM reverse primer (bad)
1
dNTP mix
0.4
DNA Template from Gibson Assembly of
2.65
phusion
0.2
All components were kept on ice
July 13, 2018
Gel Electrophoresis of Purified FadK PCR
Shannon
Prepare gel
Combine 15 mL 1x TAE and 0.125g agarose (weigh between 0.10 and 0.15 because our scale isn’t that accurate) in a small erlenmeyer flask
Microwave until dissolved
Cover with foil, let cool until flask isn’t too hot to handle
Add 1.25µL Sybr Safe, swirl
Pour into a small mold, add comb
Let harden
July 13, 2018
LB Broth
Liz
500mL Broth
Component
Amount (g)
Measured (g)
tryptone
5
5.05
Yeast extract
2.5
2.50
NaCl
5
5.00
QSed to just under 500mL
Put in stir bar and stirred/heated for 15 min
QSed to 500mL in graduated cylinder
Autoclaved on liquids 30 then stored in 4C
July 13, 2018
PCR Cleanup of Phz(4F-6R Gibson)
Ryan and Peter
Combine 80µL B2 buffer (4x the amount of PCR sample) to 20µL PCR sample
Transfer to collection tube, centrifuge at 10,000 rpm for 1 min. Discard flow-through. Re-insert column into collection tube.
Add 650µL wash buffer with ethanol. Centrifuge at 10,000 rpm for 1 min. Discard flow-through. Re-insert column into collection tube.
Centrifuge at max speed (16,100rpm) for 3 min. Discard flow-through. Insert column into a clean 1.7mL elution tube.
Add 50µL elution buffer to center of column.
Incubate at room temperature for 1 min.
Centrifuge at max speed for 2 min.
The elution tube now contains purified PCR product!
Store at -20C.
Nanodroped @ 16.7 ng/µL
July 13, 2018
Gel of PCR’s for Phz 4F-6R, fadK Primary and Secondary PCR’s
Peter, Ryan
Wells
1
2
3
4
5
2 Log Ladder
PHz 4F-6R (Gibson) purif.
fadK Primary purif. (7/5/18)
fadK secondary purif. (7/9/18)
fadK secondary purif. (7/12/18)
-6 µL ladder
-2 µL purif. PCR
-3 µL MilliQ DH2O
-1 µL Loading Dye
-5 µL purif. PCR
-1 µL Loading Dye
-5 µL purif. PCR
-1 µL Loading Dye
-5 µL purif. PCR
-1 µL Loading Dye
Gibson worked… two visible bands, one a little less than 3 kb according to the ladder (full gibson should be 2.588 Kb) and one band that is probably both 4F-4R and 6F-6R that didn’t assemble are about both ~ 1 Kb
Secondary fadK most likely didn’t work because PCR used ~ 40 ng from Primary PCR so that is probably what is visible in this gel (also same length as primary PCR)
Week 10: 7/16/2018
July 16, 2018
Secondary PCR of fadD and fadK
Ryan, Shannon
Set thermocycler:
Phase
Time (sec)
Temp. (C)
Initial denaturation
30
98
35 cycles
10
98
30
67
40
72
Final extension
10 minutes
72
Hold
∞
4
Add to tubes:
Component
fadD Amount (µL)
fadK Amount (µL)
dH2O
12.6
12.5
Phusion buffer (HF)
4
4
DMSO
0.6
.6
10mM forward primer (bad)
1
1
10mM reverse primer (bad)
1
1
dNTP mix
0.4
.4
Template DNA
.2
.3
phusion
0.2
.2
All components were kept on ice
July 16, 2018
Restriction Digest of Secondary fadD and fadK PCRs
Liz
fadD and fadK:
Component
Amount
EcoRI (HF)
1µL
PstI (HF)
1µL
CutSmart
2µL
DNA template
100ng
fadD: 100ng x 1µL/14.3ng = 6.99µL
fadK: 100ng x 1µL/19.9ng = 5.02µL
Water
To 20µL
fadD: 9.00µL
fadK: 10.97µL
Order: water, buffer, DNA, enzymes
Spin down in tabletop centrifuge
Heat blocked at 37C for 25 min
Heat blocked at 65C for 15 min
Backbone:
Component
Amount
BpsHI
1µL
EcoRI (HF)
1µL
PstI (HF)
1µL
CutSmart
2µL
DNA template
100ng
fadD: 100ng x 1µL/14.3ng = 6.99µL
fadK: 100ng x 1µL/19.9ng = 5.02µL
Water
To 20µL
fadD: 9.00µL
fadK: 10.97µL
Order: water, buffer, DNA, enzymes
Spin down in tabletop centrifuge
Heat blocked at 37C for 25 min
Heat blocked at 80C for 15 min
This digest was wrong because DnpI should have been used for the backbone instead of BpsHI
July 16, 2018
Gel Electrophoresis of Purified FadK, FadD PCR
Ryan
Prepare gel
Combine 15 mL 1x TAE and 0.125g agarose (weigh between 0.10 and 0.15 because our scale isn’t that accurate) in a small erlenmeyer flask
Microwave until dissolved
Cover with foil, let cool until flask isn’t too hot to handle
Add 1.25µL Sybr Safe, swirl
Pour into a small mold, add comb
Let harden
Wells:
1
2
3
4
5
2 Log Ladder
fadD PCR2
fadD PCR0
fadK PCR2
fadK PCR0
-6 µL ladder
-1 µL purif. PCR
-4 µL MilliQ DH2O
-1 µL Loading Dye
-1 µL purif. PCR
-4 µL MilliQ DH2O
-1 µL Loading Dye
-1 µL purif. PCR
-4 µL MilliQ DH2O
-1 µL Loading Dye
-1 µL purif. PCR
-4 µL MilliQ DH2O
-1 µL Loading Dye
July 16, 2018
Transformation with RbCl JM109 Cells
Sricharan, Peter
Transforming:
BBa_J04450 (RFP), BBa_B0015 (DoubleTerminator), BBa_K590025 (PetroBrick), BBa_K1433005 (Lambda Red), BBa_I718008 (Inducible Cre-generator), BBa_K1316012 (Mtr + T7 Lac O.), TesA (from ligation), CymA (from ligation), BBa_K917006 (ccmA-H)
Grab 50μL aliquots of RbCl Uber-Competent Cells from -80° C freezer, let thaw on ice for ~3-4 minutes.
Add 1μL of plasmid from iGEM kit (or 2 μL of plasmid from ligation product) to the tube with cells.
Incubate cells on ice for 30 minutes.
Heat shock tubes at 42° C for 45s.
Incubate on ice again for 5 minutes.
Add 450μL SOC media to each tube (9 x volume) and incubate on the shaker at 37° C and 250rpm for 1 hour.
Spin down cells at ~6800rpm for 3 minutes, remove 370μL of supernatant, resuspend cell pellet in the remaining media, and pipet that onto the plate
Use P200 to spread cells across plate
Leave plates upside-down (with agar on top) in incubator (at 37° C) overnight for 16-18 hours.
Notes:
Added 2 μL pf DNA for TesA & CymA because from ligation reaction so not as much DNA as a prepared BioBrick
Heat-shocked ccmA-H (BBa_K917006) in 5 different tubes for 30, 45, 60, 75, and 90 seconds (one time per tube)
Added 900 μL of soc media for tube 4 because was 100 μL aliquot of cells
Used Amp plate for BBa_I718008 because plasmid has Amp resistance
Results (7/17/18):
Part/ Vector
Transformation Result
BBa_J04450 (RFP)
-No growth on plate
BBa_B0015 (Double Terminator)
-No observable cell pellet when spun down
-No growth on plate
BBa_K590025 (PetroBrick)
-Good growth (do O.C. on 7/17)
BBa_K1433005 (Lambda Red)
-Good growth (do O.C. on 7/17)
BBa_I718008 (Inducible Cre-generator)
-Good growth (do O.C. on 7/17)
BBa_K1316012 (Mtr + T7 Lac O.)
-No observable cell pellet when spun down
-No growth on plate
TesA (from ligation)
-No growth on plate (Ligation enzymes can be a problem)
CymA (from ligation)
-No growth on plate (Ligation enzymes can be a problem)
BBa_K917006 (ccmA-H)
-30 sec: works best
-45 sec: works well
-60 sec: works okay
-75 sec: no visible pellet but grew a few colonies
-90 sec: no visible pellet or growth on plate (probably heat-killed)
July 17, 2018
Ligation of fadD and fadK
Liz
*didn’t work because digested backbone wrong
fadD:
Insert DNA: 1689 bp
Vector length: 2070 bp
Vector mass: 25 ng
NEB Ligation Calculator: 2:1 → 40.80ng x 20L
July 17, 2018
Gel Electrophoresis of Purified FadK, FadD PCR
Ryan
Prepare gel
Combine 15 mL 1x TAE and 0.125g agarose (weigh between 0.10 and 0.15 because our scale isn’t that accurate) in a small erlenmeyer flask
Microwave until dissolved
Cover with foil, let cool until flask isn’t too hot to handle
Add 1.25µL Sybr Safe, swirl
Pour into a small mold, add comb
Let harden
Wells:
1
2
3
4
5
2 Log Ladder
fadD PCR2
fadD PCR0
fadK PCR2
fadK PCR0
-6 µL ladder
-1 µL purif. PCR
-4 µL MilliQ DH2O
-1 µL Loading Dye
-1 µL purif. PCR
-4 µL MilliQ DH2O
-1 µL Loading Dye
-1 µL purif. PCR
-4 µL MilliQ DH2O
-1 µL Loading Dye
-1 µL purif. PCR
-4 µL MilliQ DH2O
-1 µL Loading Dye
No bands visible on any part of the gel:
July 17, 2018
Gel Extraction of phz 4F-6R
Sricharan
Onto a 1% agarose gel with SYBR Safe, loaded (with 6X Purple Loading Dye from NEB)
1
2
3
4
5
2-log ladder
phz 1F-3R PCR (GC Buffer)
‘tesA(L109P) ligation
phz 4F-6R PCR
Same as Lane 4
6µL ladder
5µL item
1µL dye
1µL item
4µL dH2O
1µL dye
15µL item
3µL dH2O
Note, wells 2 and 3 had blue stain from tape (we tried to tape together these two wells to do gel extraction but it didn’t work very well).
Cut bands at 2.6kb on last two lanes out using tape (marked with arrows delineating top of gel, bottom of gel, and location of band to be excised) and wax paper + razor
Weighed cut bands: 0.0840g, used Zymoclean Gel DNA Recovery Kit to clean up DNA. Only deviation from protocol was centrifuged ~16K rpm to elute 30µL buffer from Invitrogen PCR Pure Kit.
Nanodrop Results:
Conc: 2.8 ng/µL
260/280: 1.23
260/230: 0.05
July 17, 2018
LB + Chloramphenicol Plates Preparation
Ryan
Preparing Chloramphenicol LB plates (24)
Collected and measured out ingredients for LB media and added to bottle
Q/s’d with dH20 to 400 and shook bottle to dissolve ingredients
Stirred in beaker for 30 min
Poured into grad cylinder to finish Q/s to 500 mL
Poured back into bottle and stirred for ~ a minute
Autoclaved on S03 setting for 30 minutes (autoclaving less than a liter of liquid)
Added 0.5 mL of chloramphenicol (1000x, 35 mg/mL) to solution
Put them in the hood with the plates half open w/ agar side on the bottom to get rid of condensation
**OOPS put in 35 microliters instead of .5 mL
July 17, 2018
Transformation with RbCl JM109 Cells
Liz, Peter
**See Protocol from 7/16/18**
Transforming failed transformations from 7/16:
BBa_J04450 (RFP), BBa_B0015 (DoubleTerminator), BBa_K1316012 (Mtr + T7 Lac O.), TesA (from ligation + now purified w/ PCR purif), CymA (from ligation + now purified w/ PCR purif), BBa_K917006 (ccmA-H)
***See protocol from 7/16***
Notes/ Deviations:
CymA and TesA have been purified w/ PCR purif.
Two tubes of cells/ transformation (to ensure one tube will have cells) so 0.5 μL of DNA added to each tube (only one tube for the blank)
Heat-shocked for 30 seconds based on results from transformations of ccmA-H
After resuspending, pooled secondary tube w/ primary tube if any cells (next time pool before spinning)
All except blank had one tube w/ obvious pellet after first spin
BBa_J04450 (RFP), BBa_B0015 (Double terminator), and CymA secondary tubes w/ no original pellet had slightly visible pellet after second spin. Were pooled with primary tube after resuspension
Not pellet for 6 so didn’t bother plating
Results (7/18/18):
Part/ Vector
Transformation Result
BBa_J04450 (RFP)
-No growth on plate
BBa_B0015 (Double Terminator)
-One colony (Do O.C. on 7/18)
BBa_K1316012 (Mtr + T7 Lac O.)
-No growth on plate
TesA (from ligation + now purified w/ PCR purif)
-No growth on plate
CymA (from ligation + now purified w/ PCR purif)
-One colony (Do O.C. on 7/18)
July 17, 2018
Overnight Cultures of successful transformations from 7/16/18
Peter, Shannon
Overnight Cultures:
BBa_K590025 (PetroBrick)
BBa_K1433005 (Lambda Red)
BBa_I718008 (Inducible Cre-generator)
BBa_K917006 (ccmA-H)
Results (7/18/18):
They worked: proceed to mini-prep and glycerol stocking
July 18, 2018
Nanodrops and a Gel: Miniprepped Plasmids, Gel Extraction PCR of 4F-6R, Gel Extraction of 1F-3R
Sricharan
Some OCs were grown - Shannon miniprepped and Liz made glycerol stocks; I Nanodropped
Item
Concentration (ng/µL)
260/280
260/230
I718008 MP
85.7
1.75
1.28
K590025 MP
51.7
1.73
0.85
K1433005 MP
53.9
1.74
1.37
K917006 MP
175.1
1.88
2.15
Phz 4F-6R GelExtr PCR
30.9
1.39
-0.82
Ran a gel:
Lane
1
2
3
4
5
6
Item
2-log ladder
Poked hole
K143305 MP
I718008 MP
K917006 MP
PCR of Gel Extr of phz 4F-6R
Amt (µL)
6
N/A
5+1 dye
5+1 dye
5+1 dye
2+3 dH2O+1 dye
7
8
9
10
11
12
2-log ladder
phz 1F-3R GC PCR
phz 1F-3R GC PCR
empty
K590025 MP
2-log ladder
6
15 + 3 dye
15 + 3 dye
5 + 1 dye
6
Note, items in lanes 3, 4, 5, 11 digested with PstI.
Dye is 6X Purple Loading Dye from NEB.
Liz gel extracted bands 8 and 9 at desired size (~3kb)
PCR of these bands:
To PCR tube on ice:
dH2O
11.8 µL
Phusion Buffer (GC)
4 µL
DMSO
0.6 µL
10µM forward primer (1F)
1 µL
10µM reverse primer (3R)
1 µL
10µM dNTPs
0.4 µL
Template DNA
1 µL
Phusion polymerase
0.2 µL
Cycle is based on Phusion polymerase protocol:
Time (s)
Temp (°C)
Initial Denaturation
30
98
35 Cycles
10
98
30
61
60
72
Final Extension
600
72
Hold
Infinite Hold
4
July 18, 2018
Miniprep of overnight cultures from 7/17/2018
Shannon
Used Zyppy Miniprep Kit and instructions from the manufacturer’s website
Overnight cultures of: K590025, K1433005, I718008, and K917006
Add 600µL of each bacterial culture to separate 1.5mL tubes
Add 100µL 7x lysis buffer, mix by inverting about 6 times
Solution should turn from opaque blue to clear blue
Add 350µL cold neutralization buffer (yellow), mix by inverting
Solution should turn yellow
Centrifuge at 13,000g for 3 min
Transfer supernatant (about 900µL) into column, place column in collection tube
Centrifuge for 15 seconds
Discard flow-through, put column back into the same collection tube
Add 200µL endo wash buffer, centrifuge 20 seconds
Add 400µL Zippy wash buffer to column. Centrifuge 1 min
Transfer the column to clean 1.5 mL tube, add 30µL elution buffer directly to column. Let stand for 1 min
Centrifuge 30 seconds to elute DNA
July 18, 2018
Transformation with RbCl JM109 Cells
Liz, Peter
**See Protocol from 7/16/18**
Transforming failed transformations from 7/17 + more:
BBa_J04450 (RFP), BBa_K1316012 (Mtr + T7 Lac O.) from 2017 and 2018 kit, TesA (from ligation + now purified w/ PCR purif), FadK (from ligation + now purified w/ PCR purif), FadD (from ligation + now purified w/ PCR purif), Blank
Notes/Deviations:
Two tubes of cells/ transformation (to ensure one tube will have cells) so 0.5 μL of DNA added to each tube (only one tube for the blank)
Heat-shocked for 30 seconds based on results from transformations of ccmA-H (done on 7/16)
Pooled like tubes (same vector) after 1 hour incubation
Results (7/19/18):
Part/ Vector
Transformation Result
BBa_J04450 (RFP)
1 colony (O.C. on 7/19)
BBa_K1316012 (Mtr + T7 Lac O.) 2017 kit
No growth
BBa_K1316012 (Mtr + T7 Lac O.) 2018 kit
No growth
TesA (from ligation + now purified w/ PCR purif)
1 colony (O.C. on 7/19)
FadK (from ligation + now purified w/ PCR purif)
1 colony (O.C. on 7/19)
FadK (from ligation + now purified w/ PCR purif)
No growth
FadD (from ligation + now purified w/ PCR purif)
No growth
Blank
No growth
July 18, 2018
LB + Chloramphenicol Plates Preparation
Ryan
Preparing Chloramphenicol LB plates (24)
Collected and measured out ingredients for LB media and added to bottle
Q/s’d with dH20 to 400 and shook bottle to dissolve ingredients
Stirred in beaker for 30 min
Poured into grad cylinder to finish Q/s to 500 mL
Poured back into bottle and stirred for ~ a minute
Autoclaved on S03 setting for 30 minutes (autoclaving less than a liter of liquid)
Added 0.5 mL of chloramphenicol (1000x, 35 mg/mL) to solution
Put them in the hood with the plates half open w/ agar side on the bottom to get rid of condensation
July 19, 2018
Miniprep and glycerol stocks of cymA ligation and B0015
Shannon
Used Zyppy Miniprep Kit and instructions from the manufacturer’s website
Add 600µL of each bacterial culture to separate 1.5mL tubes
Add 100µL 7x lysis buffer, mix by inverting
Add 350µL cold neutralization buffer (yellow), mix by inverting
Centrifuge at 13,000g for 3 min
Transfer supernatant (about 900µL) into column, place column in collection tube
Centrifuge for 15 seconds
Discard flow-through, put column back into the same collection tube
Add 200µL endo wash buffer, centrifuge 20 seconds
Add 400µL Zippy wash buffer to column. Centrifuge 1 min
Transfer the column to clean 1.5 mL tube, add 30µL elution buffer directly to column. Let stand for 1 min
Centrifuge 30 seconds to elute DNA
Made glycerol stocks by combining 0.75mL of 40% glycerol and 0.75mL overnight culture, stored at -80C
July 19, 2018
PCR Purification of fadD and fadK
Shannon
Using Lambda Biotech Column Pure PCR Cleanup Kit
Mix 20µL PCR reaction with 100µL binding buffer
Load onto spin column, centrifuge at full speed (16,100g) for 1 min
Discard flow-through
Add 700µL wash buffer, centrifuge 1 min
Discard flow-through, centrifuge again 1 min
Transfer column to clean 1.5mL tube
Add 30µL elution buffer, let sit 1 min, centrifuge 1 min to elute DNA
July 19, 2018
Restriction Digest of CymA and B0015
Peter
B0015 (Terminator):
Component
Amount
PstI (HF)
1µL
CutSmart
1µL (should have done 2µL)
DNA template
100ng x 1µL/53.8ng = 1.86µL
Water
16.14µL (To 20µL)
Order: water, buffer, DNA, enzymes
Spin down in tabletop centrifuge
Heat blocked at 37C for 10 min
Heat-killed on block at 80C for 15 min
CymA:
Component
Amount
PstI (HF)
1µL
CutSmart
1µL (should have done 2µL)
DNA template
5µL (20 ng/µL) -wanted a lot of it)
Water
13 µL (To 20µL)
Order: water, buffer, DNA, enzymes
Spin down in tabletop centrifuge
Heat blocked at 37C for 10 min
Heat-killed on block at 80C for 15 min
MtrCAB (2017 or 2018?):
Component
Amount
PstI (HF)
1µL
CutSmart
1µL (should have done 2µL)
DNA template
100ng x 1µL/112.5ng = 0.89µL
Water
17.11 µL (To 20µL)
Order: water, buffer, DNA, enzymes
Spin down in tabletop centrifuge
Heat blocked at 37C for 10 min
Heat-killed on block at 80C for 15 min
July 19, 2018
PCR Purification of phz 1F-3R Gel Extraction PCR and a Gel
Sricharan
PCR purified product from yesterday (phz 1F-3R gel extraction PCR)
Used Lambda Biotech Kit protocol, eluted with 30µL buffer
Agarose gel (1%, large 35mL)
Lane
1
2
3
4
5
6
7
8
9
10
Item
2-log ladder
phz 1F-3R Gel Extraction PCR
cymA MP
B0015 MP
K1316012 plasmid (from 2017 Kit)
fadD PCR2 with correct annealing temp
fadD PCR0 diluted to 40ng/30µL
fadK PCR2 with correct annealing temp
fadK PCR0 diluted to 40ng/30µL
2-log ladder
Amt (µL)
5
3+2 dH2O+1 dye
5+1 dye
5+1 dye
5+1 dye
5+1 dye
5+1 dye
5+1 dye
5+1 dye
6
Note that lanes 3-5 are digested with PstI
Seems that fadD PCR of the coding region (“PCR2”) is successful since it is significantly amplified past that of template.
Need to redo fadK PCR2
Also cymA and B0015 might be around the same size, but hard to tell here, need to retry the gel
July 19, 2018
Overnight Cultures of TesA and J04450 (RFP)
Peter
Overnight Cultures:
TesA
BBa_J04450 (RFP)
Results (7/20/18):
They worked: proceed to miniprep and glycerol stocking
July 20, 2018
PCR2 of FadK and PCR of ‘TesA(L109P) Basic Part from miniprep (7/18/18)
Peter, Sricharan
FadK PCR2:
Set thermocycler:
Phase
Time (sec)
Temp. (C)
Initial denaturation
30
98
35 cycles
10
98
30
72 (from NEB calculator)
48
72
Final extension
10 minutes
72
Hold
∞
4
Add to tubes in this order:
Component
Amount (µL)
dH2O
12.11
HF buffer
4
dNTP mix
0.4
10mM forward primer (bad)
1
10mM reverse primer (bad)
1
DMSO
0.6
Template DNA (FadK PCR0)
0.69 (10 ng)
phusion
0.2
All components were kept on ice
‘TesA (L109P):
Set thermocycler:
Phase
Time (sec)
Temp. (C)
Initial denaturation
30
98
35 cycles
10
98
30
72 (from NEB calculator)
26
72
Final extension
10 minutes
72
Hold
∞
4
Add to tubes in this order:
Component
Amount (µL)
dH2O
12.54
HF buffer
4
dNTP mix
0.4
10mM forward primer (bad)
1
10mM reverse primer (bad)
1
DMSO
0.6
Template DNA (‘TesA L109P from miniprep)
0.26 (10 ng)
phusion
0.2
All components were kept on ice
July 20, 2018
Gel of FadK PCR2, ‘TesA (L109P) basic part, and other stuff (see Sricharan’s notebook)
Peter, Ryan
Wells:
1
2
3
4
5
6
2 Log Ladder
fadK PCR2 HF
fadK PCR0 diluted to 0.69µL/30µL
fadK PCR2 GC
‘tesA(L109P) PCR Basic
K1316012 (mtrCAB) 2017 Kit Digest w/ PstI
7
8
9
10
11
12
2 Log Ladder
‘tesA(L109P) Miniprep Digest w/PstI
Kit 7, Well 2H 2018 Digest w/PstI
J04450 Miniprep Digest 3x
See lane 10
See Lane 1
Note that 6µL of ladder added to lanes 1, 7, 12; 5µL of item and 1µL of loading dye added to all other lanes
Results:
1
2
3
4
5
6
7
8
9
10
11
12
July 20, 2018
Restriction 5 things
Peter
Digest of J04450 (RFP), ‘TesA (L109P), FadD PCR2, Mtr 2016 (K1316012), Kit 7 Well 2H
J04450 (RFP):
Component
Amount
XBaI
1µL
SpeI
1µL
DpNI
1µL
CutSmart
1µL (should have done 2µL)
DNA template
10µL
dH2O
6µL (To 20µL)
Order: water, buffer, DNA, enzymes
Spin down in tabletop centrifuge
Heat blocked at 37C for 10 min
Heat-killed on block at 80C for 15 min
CymA:
Component
Amount
PstI (HF)
1µL
CutSmart
1µL (should have done 2µL)
DNA template
5µL (20 ng/µL) -wanted a lot of it)
Water
13 µL (To 20µL)
Order: water, buffer, DNA, enzymes
Spin down in tabletop centrifuge
Heat blocked at 37C for 10 min
Heat-killed on block at 80C for 15 min
MtrCAB (2017 or 2018?):
Component
Amount
PstI (HF)
1µL
CutSmart
1µL (should have done 2µL)
DNA template
100ng x 1µL/112.5ng = 0.89µL
Water
17.11 µL (To 20µL)
Order: water, buffer, DNA, enzymes
Spin down in tabletop centrifuge
Heat blocked at 37C for 10 min
Heat-killed on block at 80C for 15 min
Week 11: 7/23/2018
July 23, 2018
LB + Cam, LB + Kan Plates Preparation
Ryan
Preparing Chloramphenicol LB plates (18):
Collected and measured out ingredients for LB media and added to bottle
Q/s’d with dH20 to 350 mL and shook bottle to dissolve ingredients
Stirred in beaker for 30 min
Poured into grad cylinder to finish Q/s to 375 mL
Poured back into bottle and stirred for ~ a minute
Autoclaved on S03 setting for 30 minutes (autoclaving less than a liter of liquid)
Split solution in half such that 9 plates are for Cam and 9 plates are for Kan
Added 0.1875 mL of chloramphenicol (1000x, 35 mg/mL) to solution in one half, 0.1875 mL of kanamycin (1000x, 35 mg/mL) to other
Let plates cool for an hour on lab bench
July 23, 2018
Digest of J04450 and psb1C3
Peter and Shannon
Add reagents in order listed
Centrifuge briefly
Heat block at 37C (45 min for psb1C3, 10 min for the rest)
Heat block at 80C for 15 min to heat kill enzymes
psb1C3, I718008, K1433005, K1316012
Component
Amount (µL)
Water (milliQ)
psb1C3: 12
1718008: 14.83
K1433005: 14.14
K1316012: 15
CutSmart
2
DNA
psb1C3: 4
1718008: 1.17
K1433005: 1.86
K1316012:1
EcoRI
1
PstI
1
J04450
Component
Amount (µL)
Water (milliQ)
15.65
CutSmart
2
DNA
0.35
SpaI-HF
1
XbaI
1
July 23, 2018
Reviving Glycerol Stocks
Shannon
Put glycerol stocks on ice (try not to let them thaw too much)
Stab into the glycerol stock with a p20 pipette tip
Use pipette tip to t-streak on a place with the correct antibiotic
Incubate overnight at 37C
Revived: J23114, J23110, J23100, and cymA. Was going to revive K864400 but could not find it in the -80C freezer
Note: There were three tubes labelled J23110, and one tube that had the label wiped off. These may have been mislabelled. I believe two are J23110 and two are J23100. The two that have more color should be the stronger promoter, and the two with less color should be the weaker promoter. I labelled one plate “S” and one “W” for strong and weak.
Note: For the tube labelled J23110, I don’t know which of the three “J23110” tubes I used. I did this one before realizing the tubes may have been mislabelled.
July 23, 2018
Agarose Gel for Extraction and Checking stuff
Sricharan
Another 1% agarose gel
Liz gel extracted lanes 2 and 3 at pSB1C3 band (~2kb)
1
2
3
4
5
6
2-log ladder
J04450 Digest
J04450 Digest
K143305 Digest 7/23
I718008 Digest 7/23
K14316012 Digest 7/23
Note that 6µL of ladder to lane 1; 5µL item and 1µL dye to every other well. Lanes 2 - 5 are sourced from minipreps. Lane 6 is just DNA from the distribution kit we wanted to test.
July 23, 2018
Gel Extraction of J04450 3x Digest
Shannon
Excise DNA from gel, add to 1.5mL tube. 184mg were excised.
Add 3x volume ADB. 555mL were added.
Incubate at 45C for 5-10 min until dissolved
Transfer to spin column
Centrifuge 60 sec at 10,000g. Discard flow-through.
Add 200µL wash buffer, centrifuge 30 sec. Discard flow through. Repeat.
Add 10µL elution buffer directly onto column. Let sit for 1 min.
Place column in clean 1.5mL tube. Centrifuge 60 sec.
Store at -20C.
Note: the elution buffer was from the invitrogen kit.
July 23, 2018
Gel Extraction of J04450 3x Digest
Shannon
Excise DNA from gel, add to 1.5mL tube. 184mg were excised.
Add 3x volume ADB. 555mL were added.
Incubate at 45C for 5-10 min until dissolved
Transfer to spin column
Centrifuge 60 sec at 10,000g. Discard flow-through.
Add 200µL wash buffer, centrifuge 30 sec. Discard flow through. Repeat.
Add 10µL elution buffer directly onto column. Let sit for 1 min.
Place column in clean 1.5mL tube. Centrifuge 60 sec.
Store at -20C.
Note: the elution buffer was from the invitrogen kit.
July 23, 2018
J04450 Digest
Liz
Set heat blocks to 37C and 80C
Components (in order)
Amount (µL)
milliQ water
6
Cutsmart
2
DNA (J04450 miniprep SK 7/20, 287ng/µL)
10
SpeI-HF
1
XbaI
1
Incubated at 37C for 10 min
Heat shocked at 15 min
July 24, 2018
PCR of PSB1C3 Template from RFP mini-prep
Peter, Sricharan
Set thermocycler:
Phase
Time (sec)
Temp. (C)
Initial denaturation
30
98
35 cycles
10
98
30
71 (from NEB calculator)
60
72
Final extension
10 minutes
72
Hold
∞
4
*set to 50 µL reaction
Add to tubes in this order:
Component
Amount (µL)
dH2O
31.65
HF buffer
10
dNTP mix
1
10mM forward primer (bad)
2.5
10mM reverse primer (bad)
2.5
DMSO
1.5
Template DNA (FadK PCR0)
0.348 (5 ng - from 10x dilute of miniprep)
phusion
0.2
All components were kept on ice
Used lambda PCR cleanup kit
Nano-dropped @ 48.8 ng/µL
Did gel extract for linearized plasmid for Gibsons (no longer using for Gibsons)
July 24, 2018
Overnight Cultures
Shannon
Did overnight cultures of J23114, J23110, J23100, and cymA
Combine 5mL LB and 5µL antibiotic (ampicilin for J23114, J23110, and J23100 and chloramphenicol for cymA)
Put in shaker at 4:20pm
Note: “S” and “W” are either J23110 or J23100. “S” should be the stronger promoter, “w” should be the weaker promoter.
Note: The tube labelled “J23110” may be J23100. Check color against “S” and “W”?
July 24, 2018
Digest of psB1C3, acr 1, acr 3, fadD, fadK, B0015, cymA, pSB1K3, ccmA-H
Ryan and Liz
Add reagents in order listed
Centrifuge briefly
Heat block at 37C (45 min for psb1C3, 10 min for the rest)
Heat block at 80C for 15 min to heat kill enzymes
fadD, fadK, cymA, pSB1K3
Component
Amount (µL)
Water (milliQ)
fadD: 11.7
fadD: 9
cymA: 11
pSB1K3: 12
CutSmart
2
DNA
fadD: 4.3
fadK: 7
cymA: 5
pSB1K3: 4
EcoRI
1
Pstl
1
pSB1C3
Component
Amount (µL)
CutSmart
.5
DNA
4
EcoRI
.25
PstI
.25
Acr 1
Component
Amount (µL)
CutSmart
1.25
DNA
10
Pstl
1.25
Acr 3
Component
Amount (µL)
CutSmart
1.25
DNA
10
Pstl
1.25
B0015
Component
Amount (µL)
Water (milliQ)
14.1
CutSmart
2
DNA
1.86
Xbal
1
Pst1
1
ccmA-H
Component
Amount (µL)
Water (milliQ)
15.4
CutSmart
2
DNA
0.57
EcoRI
1
Spel
1
July 24, 2018
Gibson Reactons, Circular Version
Sricharan
Planning Gibson reactions to assemble acr. We realized the overlaps between our acr inserts and the pSB1C3 backbone isn’t great. We decided on a couple reactions to see if we can use these inserts
For 50ng pSB1C3, or 0.037pmol of backbone, using the NEB kit, we want about 2x excess of inserts. Using NEB calculator to figure out mass of insert required:
acr Synthesis 1
501bp
26ng
acr Synthesis 2
529bp
27.5ng
acr Synthesis 3
642bp
33.4ng
Gibson 1: Negative Control Reaction
Item
Amount (µL)
pSB1C3 from J04450 (XbaI and SpeI digested)
1.43
dH2O
9.41
NEB Gibson Master Mix
10.8
Gibson 2: acr Assembly
To have adequate overlap between acr inserts and pSB1C3, we’d need to digest something with pSB1C3 backbone with XbaI and SpeI (poorly designed inserts on our part). We did this to J04450 and gel extracted this band. However, this means that there’s a risk of a) contamination with J04450 plasmid and b) religation of the sticky ends of XbaI and SpeI. That’s why we’re running the negative control above and are considering reordering some gBlocks to make Gibsons go smoother. Other option would be to get new primers to amplify pSB1C3 with the overlaps designed into the primers.
Item
Concentration (ng/µL)
Amount (µL)
pSB1C3 from J04450 (XbaI and SpeI digested)
35
1.43
acr Synthesis 1 (EcoRI digested)
100/12.5
3.01
acr Synthesis 2
10
2.54
acr Synthesis 3 (PstI digested)
100/12.5
3.86
NEB Gibson Master Mix
10.8
Gibson 3: acr Assembly Alternate
Strategy - ligating pSB1C3 cut with EcoRI and PstI, acr Synthesis 1 cut with EcoRI, acr Synthesis 3 cut with PstI
Then this ligation is added to the Gibson reaction with acr Synthesis 2
Back calculated approx amts of each component required so that concentrations of digest/ligation is high as possible
For 70ng digested pSB1C3, or 0.052 pmol, a 1:1 excess of insert means for the ligation:
Item
Concentration (ng/µL)
Amount (µL)
pSB1C3 Linear Backbone (EcoRI and PstI digested)
20
3.5
acr Synthesis 1 (EcoRI digested)
100/12.5
2.12
acr Synthesis 3 (PstI digested)
100/12.5
2.12
T4 Ligase
100/12.5
0.46
T4 Ligation Buffer
0.91
(Coordinated on digest concentrations with Liz, used NEB calculator to obtain ng amounts of each insert)
Ligation was incubated at 10 min, RT; Heat killed at 65 °C for 20 min
Added 7µL of this ligation (0.04pmol, theoretical max of 78ng ligated product) to 3µL of acr Synthesis 2 (0.088pmol or 30ng)
July 25, 2018
Gibson Reactons, Circular Version, Day 2
Sricharan
Finished Gibson 2’ (same as 2 but redid since we were unsure about digests), 3 as described yesterday
Next day, got only 1 colony from all Gibsons, from Strategy 2 … later realized this was probably religated plasmid
July 25, 2018
Miniprep of J23114, “J23110”, “S”, “W”, and cymA
Shannon
“J23110” could be J23110 or J23100. “S” should be J23110, and “W” should be J23100.
Spin down 1.5mL of overnight culture, discard supernatant, add another 1.5mL, spin down, discard supernatant, and rehydrate the pellet in 600µL dH2O.
Spun at 16,100g for 1 min each time
Add 100µL lysis buffer, invert to mix
Add 200µL cold neutralization buffer (kept on ice), invert to mix
Centrifuge at 13,000 for 3 min
Transfer supernatant to column (about 900µL)
Place column in collection tube. Centrifuge 15 sec.
Discard flow-through, put back into collection tube
Add 200µL endo-wash buffer, centrifuge 30 sec.
Add 400µL wash buffer, centrifuge 1 min
Transfer column to clean 1.5mL tube. Add 30µL elution buffer. Let stand 1 min.
Centrifuge 30 sec to elute DNA
July 25, 2018
Transformation with RbCl JM109 Cells
Sricharan and Peter
**See Protocol from 7/16/18**
Transforming:
BBa_K1316012 (Mtr + T7 Lac O.) from 2016 kit w/ Strataclone Commercial Cells and our JM109 cells, FadD (from ligation + now purified w/ PCR purif), FadK (from ligation + now purified w/ PCR purif), BBa_J23100 (Strong Anderson Promoter), BBa_J23110 (Medium Anderson Promoter), BBa_K864400 (pTac promoter), ACR Gibson 2 (7/24), Gibson control (7/24), ccmA-H 3A (7/24), ACR Gibson 2’ (7/25), ACR Gibson 3 (7/25)
Notes/Deviations
Two tubes of cells/ transformation (to ensure one tube will have cells) so 0.5 μL of DNA added to each tube (Gibsons each get 1 μL per tube instead of 0.5, ligations should have maybe gotten 1 μL per tube instead of 0.5, 1 μL for both Mtr tubes of different cells)
Took DNA from diluted Gibson solutions
Heat-shocked for 30 seconds based on results from transformations of ccmA-H (done on 7/16)
Pooled like tubes (same vector) after 1 hour incubation
Amp plates for Anderson Promoters and Kanamycin plate for ccmA-H 3A
Results (7/26/18):
Part/ Vector
Transformation Result
BBa_K1316012 (Mtr + T7 Lac O.) 2016 kit Strataclone
1 colony (O.C. on 7/26)
BBa_K1316012 (Mtr + T7 Lac O.) 2018 kit Our JM109 Cells
No growth
FadD (from ligation + now purified w/ PCR purif)
No growth
FadK (from ligation + now purified w/ PCR purif)
1 colony (O.C. on 7/26)
BBa_J23100 (Anderson Strong)
Good growth (O.C. on 7/26)
BBa_J23110 (Anderson Medium)
Good growth (O.C. on 7/26)
BBa_K864400 (pTac)
No growth
ACR Gibson 2 (7/24)
1 colony - could be plasmid backbone ligated to itself (O.C. on 7/26)
Gibson Control (7/24)
1 colony - shows that plasmid backbone can ligate to itself  (O.C. on 7/26)
ccmA-H 3A (7/24)
Good growth (O.C. on 7/26)
ACR Gibson 2’ (7/25)
No growth
ACR Gibson 3 (7/25)
No growth? (Might not have carried out this transformation)
July 26, 2018
Resuspension of phz G-Blocks (end and 3) for Gibson
Peter
Centrifuge tube for 3-5 seconds on table-top centrifuge
Add TE to reach a final concentration of 10 ng/µL
Phz end 500ng / 50µL = 10 ng/µL
Phz 3 250ng/ 25µL = 10 ng/µL
Vortex briefly
Incubate @ 50 C for 20 minutes
Vortex and briefly centrifuge
Store in -20 C
July 26, 2018
Diagnostic Gel of B0015 and CymA and Gel Extract of PSB1C3
Peter and Liz
Wells:
1
2
3
4
5
2 Log Ladder
CymA
B0015
PSB1C3 Extract
PSB1C3 Extract
-6 µL ladder
-5 µL Double Digest
-1 µL Loading Dye
-5 µL Double Digest
-1 µL Loading Dye
-5 µL Digest
-1 µL Loading Dye
-5 µL Digest
-1 µL Loading Dye
Gel Extract (Zyppy gel extraction kit):
Extracted last 2 wells of gel above to get pSB1C3 (brightest band)
Other DNA in the band is RFP and ‘tesA (length of ‘tesA is questionable but we don’t really care about that since we just want backbone)
Pice of gel extracted: 0.1268 = 126.8mg → added 126.8 x 3 = 380.4µL ADP
Incubated at 42C in heat block until agarose dissolved
Put into spin column and centrifuged 1 min at 16,100g then pipetted out flow-through
Added 200µL wash buffer and centrifuged 30s at 16,100g (did this step 2x)
Added 6µL elution buffer (from invitrogen PCR purification kit), let sit for 1 min, then centrifuged 1 min at 16,100g
July 26, 2018
3A Assembly of cymA and Anderson promoters into psb1K3
Shannon
Need a 1:1 molarity of backbone to inserts, so use 8.768ng cymA and 0.397ng J23110
Dilute promoter solutions
We need 0.397ng of each. If undiluted, you would need to small an amount to pipette
Add 1µL of the J23100/J23110 digestion (concentration = 57.48 ng/µL and 57.81ng/µL) to 29µL dH2O (new concentration = 1.916ng/µL and 1.927ng/µL).
Add 1µL of the J23114 digestion (concentration = 469.18ng/µL) to 249µL dH2O (new concentration = 1.88ng/µL)
Component
Amount (µL)
Backbone (psb1K3)
5
cymA
1.75
Promoter
0.2
T4 ligase buffer
2.5
T4 enzyme
1.25
dH2O
14.3
Combine components in order listed
Ligate at 16C for 30min
Heat kill at 80 C for 20 min
Note: accidently put 1.2µL of J23100 into that reaction.
Note: I also forgot to dilute the promoters. Will have to re-do this tomorrow.
July 26, 2018
Some Planning
Sricharan
Planned out a Gibson for phz, but realized we don’t have enough of 1F-3R and 4F-6R fragments PCRed
Toni will run PCR of above two fragments again
July 26, 2018
Agarose Gel of Minipreps & PCR of Gibson 1F-3R and 4F-6R
Sricharan
Lane
1
2
3
4
5
6
Item
fadK MP Digest
ccmA-H + B0015 MP Digest
J23110 MP Digest
J23100 MP Digest
Phz Gibson 2 MP Digest
K1316012 MP Digest
7
8
9
10
11
12
Ladder (2-log)
Gibson Ctrl MP Digest
pSB1C3 PCR Digest
phz 1F-3R Gibson PCR
phz 4F-6R Gibson PCR
Nothing
All items digested w/PSTI except pSB1C3 (w/ EcoRI, PstI, DpnI - concentrations are off so might not work)
5µL item, 1µL loading dye, Dated 7/27/2018
July 26, 2018
Restriction Digests for phz Gibson and CymA 3A
Liz
Start of phz: (adjusted so no water for higher concentration)
Component
Amount (µL)
Water
0
CutSmart
1.25
DNA (rehydrated start of phz gBlock)
10 (100ng x 1µL/10ng = 10µL → 10/4 = 2.5µL enzymes/water)
EcoRI
1.25
End of phz: (adjusted so no water for higher concentration)
Component
Amount (µL)
Water
0
CutSmart
1.25
DNA (rehydrated start of phz gBlock)
10 (100ng x 1µL/10ng = 10µL → 10/4 = 2.5µL enzymes/water)
EcoRI
1.25
Medium Anderson Promoter (J23110):
Component
Amount (µL)
Water
14.27
CutSmart
2
DNA (J23110 miniprep)
1.73 (100ng x 1µL/57.81ng = 1.73µL)
EcoRI
1
Spel
1
cymA:
Component
Amount (µL)
Water
15.27
CutSmart
2
DNA (cymA miniprep)
0.83 (100ng x 1µL/120.6ng = 0.83µL)
XbaI
1
PstI
1
Strong Anderson Promoter (J23100):
Component
Amount (µL)
Water
14.26
CutSmart
2
DNA (J23100 miniprep)
1.74 (100ng x 1µL/57.48ng = 1.74µL)
EcoRI
1
Spel
1
Weak Anderson Promoter (J23114):
Component
Amount (µL)
Water
15.79
CutSmart
2
DNA (rehydrated start of phz gBlock)
0.213 (100ng x 1µL/469.18ng = 0.213µL)
EcoRI
1
Spel
1
RFP miniprep (J04450):
Component
Amount (µL)
Water
6
CutSmart
2
DNA
10
XbaI
1
Spel
1
For DNA used all of remaining miniprep and then another random miniprep (‘tesA?) to try to get to 10µL to maximize backbone
Heat blocked at 37C for 1 hr then 80C for 15 min (put in thermocycler at 5µL reaction)
July 26, 2018
Transformations
Liz
Used 2 tubes of RbCl JM109 cells for each sample
Gibson control (1µL DNA per tube)
acr gibson 2 (1µL DNA per tube)
acr gibson 2’ (1µL DNA per tube)
K864400 (0.5µL DNA per tube)
fadD ligation (1µL DNA per tube)
Followed transformation protocol (7/16/2018)
July 27, 2018
Restriction Digests
Shannon
FadK, J23110, J23100, gibson 2 transformation, gibson control
Component
Amount (µL)
Water (milliQ)
For fadK: 16.57
For J23110:16.26
For J23100: 16.80
For gibson 2 transformation: 16.56
For gibson control: 16.54
CutSmart
2
DNA
For fadK: 0.43
For J23110: 0.74
For J23100: 0.20
For gibson 2 transformation: 0.44
For gibson control: 0.46
Pstl
1
ccmA-H + B0015
Component
Amount (µL)
Water (milliQ)
14.97
CutSmart
2
DNA
1.03
PstI
1
XbaI
1
Stratoclone K1316012
Component
Amount (µL)
Water (milliQ)
15.79
CutSmart
2
DNA
0.21
PstI
1
BspHI
1
psb1K3
Component
Amount (µL)
Water (milliQ)
12
CutSmart
2
DNA
4
PstI
1
DpnI
1
psb1C3
Component
Amount (µL)
Water (milliQ)
24.4
CutSmart
9.76
DNA
10
EcoRI
4.88
PstI
4.88
DpnI
4.88
Add reagents in order listed
Centrifuge briefly
Heat block at 37C
10 min for all except psb1K3 and psb1C3. 45 min for those.
Heat block at 80C for 15 min.
Note: shouldn’t have increased the volume for the components in psb1C3. Did not need to scale.
August 2, 2018
Resuspending G-Blocks
Shannon
Resuspended phz start, phz end, acr1, acr3, pmt1231.1, pmt1231.2
500ng of each g-block was deliverd, so 50µL TE was added to each for a final concentration of 10ng/µL.
Centrifuge tubes for 3-5 sec on tabletop centrifuge
Add TE
Vortex briefly
Incubate at 50C for 20min
Briefly vortex and cetrifuge
Store at -20C
July 27, 2018
Making SOC Medium
Peter and Ryan
Bottle 1: Will be SOC Medium
Bottle 2: 2 M MgCl2
Bottle 3: 1 M Glucose
Add to Bottle 1:
Component
Amount
Water (dH2O)
950 mL
Bacto-tryptone
20 g
Bacto-yeast extract
5 g
NaCl
0.5 g
KCl
0.1863775 g
10 ml (0.01 L) of 250 mM solution (0.25 M solution)
0.01 L x 0.25mol/1L x 74.551g/1mol = 0.1863775g
Shake and used stir bar to dissolve components
pH to 7.0 w/ 5 N NaOH (~ 0.2 ml) -- didn’t have any so used 1 N NaOH
Adjust volume to 1 liter w/ dH2O (mark 1 liter point on bottle before adding components)
Autoclave with Bottle 2
Add to Bottle 2:
Component
Amount
Water (dH2O)
90mL
MgCl2
19 g
After components are dissolved, adjust volume to 100 mL w/ dH2O (mark 100 mL point on bottle before adding components)
Autoclave with Bottle 1
Add 5 mL to Bottle 1 AFTER AUTOCLAVING
Bottle 3:
Component
Amount
Water (dH2O)
90mL
Glucose
18g
After Glucose is dissolved, adjust volume to 100 mL w/ dH2O (mark 100 mL point on bottle before adding components)
Sterilize by filtration through 0.22-micron filter
Add 20 mL to Bottle 1 AFTER AUTOCLAVING
July 27, 2018
Minipreps
Liz
Miniprepped overnight cultures using zyppy miniprep kit
fadK ligation
ccmA-H + B0015 3A
J23110
J23100
Gibson acr 2 transformation
Gibson control
Strataclone K1316012
Used 3mL each overnight, spun down, and rehydrated in 600µL water
Eluted in 30µL elution buffer
Week 12: 7/30/2018
July 30, 2018
3 Digests and a Ligation (And a gel)
Sricharan and Liz
Digesting 13µL of mtrCAB (K1316012) with BspHI and PstI to be ligated to start and assembled to pSB1K3
Digesting 15µL of pSB1C3 (Peter’s PCR) and pSB1K3 (Linearized backbones kit) with EcoRI and PstI and DpnI (ligations in the future)
14µL ccmA-H + B0015 diagnostic digest with EcoRI to check on a gel
6µL of Mtr Start (gBlock) digested with EcoRI and BspHI to be ligated to rest of mtrCAB and assembled to
13µL of ccmA-H (K917006) also digested with EcoRI and SpeI for future 3A assembly
Using standard digestion protocol (all 20µL digests)
Ligation:
mtrCAB (TU Delft part) + synthesized mtr start (rehydrated gBlock) + pSB1K3
Component
Amount (µL)
water
3.24
T4 ligase buffer
0.67
Backbone (pSB1K3 from digestion above)
1.33 (25ng x 20µL/375ng = 1.33)
Digested mtr start (equimolar to backbone)
0.28 (1.418ng x 20µL/100ng = 0.28)
Digested mtrCAB (equimolar to backbone)
0.82 (58.87ng x 20µL/1437.36ng = 0.82)
T4 ligase
0.33
Scaled to 6.67µL reaction based on 25ng plasmid backbone
Lane
1
2
3
Item
2-log ladder
ccmA-H + B0015 cut with EcoRI (7/30), 5µL
pSB1C3 cut w/ EcoRI, PstI, DpnI (7/30), 1µL
July 30, 2018
Transformation with RbCl JM109 Cells
Sricharan and Peter
**See Protocol from 7/16/18**
Transforming:
FadD (from ligation + now purified w/ PCR purif), mtrCAB 3A (mtrCAB + New Start + PSBIC3), BBa_K864400 (pTac promoter)
Notes/Deviations:
Two tubes of cells/ transformation (to ensure one tube will have cells) so 0.5 μL of DNA added to each tube.
Ligation gets1 μL per tube instead of 0.5
Took DNA from diluted Gibson solutions
Heat-shocked for 30 seconds
Pooled like tubes (same vector) after 1 hour incubation
Kanamycin plate for mtrCAB 3A new start
Results (7/31/18):
Part/Vector
Transformation Result
FadD (from ligation + now purified w/ PCR purif)
No growth
mtrCAB 3A (mtrCAB + New Start + PSBIC3)
Good growth (O.C. on 7/31)
BBa_K864400 (pTac)
Good growth (O.C. on 7/31)
July 31, 2018
PCR of phz 4f-6R, ‘tesA (L109P), Basic cymA
Peter
Notes:
phz 4F-6R: 2.588 kb
Basic ‘tesA (L109P): 0.57 kb
Basic cymA: 0.595 kb
Set thermocycler:
Phase
Time (sec)
Temp. (C)
Initial denaturation
30
98
35 cycles
10
98
30
Phz: 67
Phz: 77.6s
‘tesA & CymA: 72 (from NEB calculator)
‘tesA & CymA: 17.3
72
Final Extension
10 minutes
72
Hold
∞
4
*set to 50 µL reaction for phz 4F-6R
*set to 20 µL reaction for ‘tesA and cymA
Phz 4F-6R Tube:
Component
Amount (µL)
dH2O
29
HF buffer
10
dNTP mix
1
10mM forward primer (bad)
2.5
10mM reverse primer (bad)
2.5
DMSO
1.5
Template DNA (Gibson of 4F-6R)
3 (10ng x 1µL/3.785ng = 2.642µL) -rounding up to 3 µL
phusion
0.5
‘tesA (L109P):
Component
Amount (µL)
dH2O
12.6
HF buffer
4
dNTP mix
0.4
10mM forward primer (bad)
1
10mM reverse primer (bad)
1
DMSO
0.6
Template DNA
0.211 (10ng x 1µL/47.5ng = 0.211µL) -from 4x dilute of miniprep
phusion
0.2
Cyma (Basic) Tube:
Component
Amount (µL)
dH2O
12.6
HF buffer
4
dNTP mix
0.4
10mM forward primer (bad)
1
10mM reverse primer (bad)
1
DMSO
0.6
Template DNA (FadK PCR0)
0.249 (10ng x 1µL/120.6ng = 0.249µL) -from 3x dilute of miniprep
phusion
0.2
All components were kept on ice
Used lambda PCR cleanup kit
August 1, 2018
Gel
Peter
Lane
1
2
3
4
5
6
7
8
Item
K86440 0(pTac)
Basic CymA (pcr purif.)
Basic ‘tesA (L109P)
2-log ladder
Failed Lane
Failed Lane
phz 4F-6R (pcr purif.)
mtrCAB new st. 3A
August 1, 2018
So Many Things
Sricharan
Miniprepping Peter’s OCs of MtrCAB (3A Assembled w/ new start) ->3 colonies picked, 3 OCs
Minipreps done with Zymo Plasmid MP Kit
BBa_K864400 (pTac) is 1 OC
250µL of each on ice, ready to add glycerol once we know which ones are good size… added glycerol
Lots of Digests (all in CutSmart buffer, all volumes in µL):
Sample
mtrCAB new st (1-3)
K864400 (4)
ccmA-H (5)
fadK (6)
cymA Basic (7)
‘tesA Basic (8)
dH20
0
0
11
11
0
0
Buffer
2
2
2
2
2
2
DNA
16
16
5
5
16
16
Enzyme
1 XbaI
1 PstI
1 EcoRI
1 SpeI
1 EcoRI
1 SpeI
1 XbaI
1 PstI
1 EcoRI
1 PstI
1 EcoRI
1 PstI
1-4 by PC, 5-8 by SK
3A Assemblies … Using NEB Ligation Calculator for amounts, 1:1:1 molar ratios
1) Anderson (J23110) + cymA + pSB1K3
Component
Mass (ng)
Volume (µL)
Notes
pSB1K3 (SK 7/30)
25
1.33
Digested 7/30
J23110 Digest (LCC 7/26)
33.84
6.77
2983 bp
cymA Digest (LCC 7/26)
32.25
6.45
2843 bp
T4 Ligase Buffer
-
2
-
T4 Ligase
-
1
-
dH2O
-
2.45
-
2) Anderson (J23110) + mtrCAB w/new st + pSB1C3
Component
Mass (ng)
Volume (µL)
Notes
pSB1C3 (SK 7/30)
25
0.68
Digested 7/30
J23110 Digest (LCC 7/26)
36.03
7.21
-
mtrCAB w/new st Digest (8/1)
89.23
2.85
> 388 bp
T4 Ligase Buffer
-
2
-
T4 Ligase
-
1
-
dH2O
-
6.26
-
3) ccmA-H (K917006) + B0015 + pSB1K3 … 10µL rxn
Component
Mass (ng)
Volume (µL)
Notes
pSB1K3 (SK 7/30)
25
1.33
2204 bp
B0015 Digest (LCC 7/24)
24.94
4.99
2199 bp
ccmA-H Digest (SK 8/1)
96.84
2.2
8537 bp
T4 Ligase Buffer
-
1
-
T4 Ligase
-
0.5
-
dH2O
-
0
-
pTac (K864400) + fadK + pSB1K3 … 10µL rxn
Component
Mass (ng)
Volume (µL)
Notes
pSB1K3 (SK 7/30)
25
1.33
2204 bp
pTac Digest (PC 8/1)
24.17
0.27
2131 bp
fadK Digest (SK 8/1)
43
0.73
3717 bp
T4 Ligase Buffer
-
1
-
T4 Ligase
-
0.5
-
dH2O
-
6.17
-
August 2, 2018
Planning Gibsons for Tomorrow
Sricharan
phz (2x insert excess)
Note for all the Gibsons we’re using new gBlocks that have revised short and end sequences designed to work well w/ pSB1C3 … using NEB calculator.
Fragments with identifying attributes and amounts listed below:
pSB1C3 (SK 7/30)
phz Start 7.27.2018
1F-3R (7/27)
phz3
4F-6R (8/1)
phz End 7.27.2018
50ng
gBlock, 10 ng/µL
24.6 ng/µL
gBlock, 10ng/µL
95.3 ng/µL
gBlock, 10ng/µL
20µL/375ng
426 bp
3072 bp
222 bp
2588 bp
277 bp
2.66µL
20.6ng
148 ng
10.7ng
125 ng
13.4 ng
0.037pmol
2.06µL
6.02µL
1.07µL
1.31 µL
1.34 µL
14.46µL total, add equivalent amt of NEB Gibson Master Mix
We’ll also do a ctrl w/ 50ng plasmid only w/ rest dH2O and NEB MMix
acr (2x insert excess)
Fragments with identifying attributes and amounts listed below:
pSB1C3 (SK 7/30)
acr1 7.27.2018
acr2
acr3 7.27.2018
25ng
gBlock, 10ng/µL
gBlock, 10ng/µL
gBlock, 10ng/µL
1.33µL
512 bp
529 bp
653 bp
12.4ng
12.8ng
15.8ng
1.24µL
1.28µL
1.58µL
5.43µL total, add equivalent amt of SGI Gibson MMix
pmt1231
Fragments with identifying attributes and amounts listed below:
pSB1C3 (SK 7/30)
pmt1231.1
pmt1231.2
25ng
gBlock, 10ng/µL
gBlock, 10ng/µL
1.33µL
532 bp
512 bp
12.9ng
12.4ng
1.29µL
1.24µL
3.84µL total, add 1.14µL dH2O and 5µL SGI Gibson MMix
Also doing control with 25ng plamsid only and rest dH2O and SGI MMix
(1.33µL pSB1C3, 3.67µL dH2O, 5µL SGI MMix)
August 2, 2018
Transformation (RbCl JM109) of 3A’s from 8/1
Peter
**See Protocol from 7/16/18**
Transforming:
CymA + J23110 + PSB1K3 (8/1 SK)
mtrCAB w/ new st. + J23110 + PSB1C3 (8/1 SK)
ccmA-H + B0015 + PSB1K3 (8/1 SK)
fadK + pTac + PSB1K3 (8/1 SK)
fadD ligation to PSB1C3 (7/24 LCC)
Notes/ Deviations:
Two tubes of cells/ transformation (to ensure one tube will have cells) so 0.5 μL of DNA added to each tube. Exceptions: 2 x 1μL for CymA and Mtr (from 20 μL 3A so more dilute than std 10 μL 3A); 2 x 1μL for fadD (hasn’t worked yet so adding a lot of DNA)
Heat-shocked for 30 seconds
Pooled like tubes (same vector) after 1 hour incubation
Plated with spinner and flame
Results (8/3/18):
Part/Vector
Transformation Result
CymA + J23110 + PSB1K3 (8/1 SK)
Good growth (O.C. on 8/3)
mtrCAB w/ new st. + J23110 + PSB1C3 (8/1 SK)
No growth
ccmA-H + B0015 + PSB1K3 (8/1 SK)
Good growth (O.C. on 8/3)
fadK + pTac + PSB1K3 (8/1 SK)
~32 colonies (O.C. on 8/3)
FadD (from ligation + now purified w/ PCR purif)
3 colonies (O.C. on 8/3)
August 3, 2018
Gibsons
Sricharan
Set up Gibsons as detailed 8/2/2018
Set @ 50°C 1 hr
4°C infinite hold
In thermal cycler (2 different cyclers for differing rxn volumes)
August
Week 13: 8/6/2018
August 2, 2018
Transformation (RbCl JM109) of 3A’s from 8/1
Peter
**See Protocol from 7/16/18**
Transforming:
mtrCAB w/ new st. + J23110 + PSB1C3 (8/1 SK)
Phz Gibson (NEB) w/ redesigned start and end (8/3 SK)
Acr Gibson (SGI) w/ redesigned start and end (8/3 SK)
Pmt1231 Gibson (SGI) w/ redesigned start and end (8/3 SK)
NEB PSB1C3 ctrl
SGI PSB1C3 ctrl
Notes/ Deviations:
Two tubes of cells/ transformation (to ensure one tube will have cells) so 0.5 μL of DNA added to each tube. Exceptions: 2 x 1μL for Mtr (from 20 μL 3A so more dilute than std 10 μL 3A)
Heat-shocked for 30 seconds
Pooled like tubes (same vector) after 1 hour incubation
CAM plates for all
Results (8/7/18):
Part/Vector
Transformation Result
mtrCAB w/ new st. + J23110 + PSB1C3 (8/1 SK)
No growth
Phz Gibson (NEB) w/ redesigned start and end (8/3 SK)
No growth
Acr Gibson (NEB) w/ redesigned start and end (8/3 SK)
No growth
Pmt1231 Gibson (SGI) w/ redesigned start and end (8/3 SK)
5-10 colonies (O.C. on 8/8)
NEB PSB1C3 ctrl
No growth
SGI PSB1C3 ctrl
No growth
August 7, 2018
Minipreps of 3As and fadD + Gel
Sricharan
Using Genscript Kit miniprepped:
tac+fadK 3A
cymA 3A (w/ med Anderson)
B0015+ccmA-H 3A
fadD
Nanodropped, results below:
Sample Name
Concentration (ng/μL)
260/280
260/230
tac+fadK
4.08
1.51
1.16
cymA 3A
41.13
1.58
0.69
B0015+ccmA-H
59.52
1.61
0.77
fadD
156.54
1.82
1.90
Diagnostic digests of these items, with
dH2O
0μL
CutSmart
2μL
DNA
17μL
EcoRI
1μL
August 7, 2018
Gel of a bunch of things
Liz
Made a 1% agarose gel
1
2
3
4
5
Ladder (6μL)
Tac + FadK EcoRI digest (5μL sample + 1μL digest)
cymA 3A EcorI digest (5μL sample + 1μL digest)
ccmA-H + B0015 3A EcoRI digest ((5μL sample + 1μL digest)
fadD EcoRI digest (5μL sample + 1μL digest)
Only lane 5 (fadD) had a band
August 8, 2018
Digests for 3A Assemblies
Liz, Peter
Double digests of pSB1A3 and J23114
pSB1A3:
cutsmart
2μL
DNA
15μL
EcoRI
1μL
PstI
1μL
DpnI
1μL
DNA from iGEM linearized plasmid backbone
In retrospect this is bad because need water in the digestion
J23114:
cutsmart
2μL
DNA (J23114 miniprep SD 7/25)
10μL
EcoRI
1μL
SpeI
1μL
water
6μL
Heat blocked at 37C for 15 min
Heat blocked at 80C for 15 min
Assembly part is in Peter’s notebook
August 8, 2018
3A Assemblies
Peter
***Using NEB Ligation Calculator for amounts, 1:1:1 molar ratios***
Protocol:
Add components in following order: water, buffer, DNA, enzymes
Let sit at room temp for 10 minutes
Heat block @ 65 C for 20 minutes
mtrCAB w/ new st. + Anderson (J23114) + pSB1C3
Component
Mass (ng)
Volume (µL)
Notes
pSB1C3
25
0.68
(7/30 SK)
J23114 Digest
36.03
0.21 (should have been 0.33)
(8/8 LCC)
mtrCAB w/ new st.
89.58
2.86
(8/1 SK)
T4 Ligase Buffer
-
2
-
T4 Ligase
-
1
-
dH2O
-
3.26
-
mtrCAB w/ new st. + pTac (K864400) + pSB1A3
Component
Mass (ng)
Volume (µL)
Notes
pSB1A3
25
1.25
(8/8 LCC)
pTac (K864400)
24.72
0.27
(SK 8/1)
mtrCAB w/ new st
2.74
2.86
(SK 8/1)
T4 Ligase Buffer
-
2
-
T4 Ligase
-
1
-
dH2O
-
2.74
-
August 8, 2018
Transformations of Gibsons and 3A Assemblies
Liz and Sricharan
1
2
3
4
5
6
7
NEB control gibson pSB1C3 (SK 8/3)
SGI control gibson pSB1C3 (SK 8/3)
Acr Gibson (SK 8/3)
Phz Gibson (SK 8/3)
mtrCAB with new start and J23114 pSB1C3 (PC 8/8)
mtrCAB with new start and pTac (K864400) in pSB1A3 (PC 8/8)
mtrCAB with new start and J23110 in pSB1C3 (SK 8/1)
2 tubes per sample, 1μL DNA in each tube
Followed transformation protocol (iGEM)
August 8, 2018
Colony PCR Checks of 3A Assemblies
Liz and Sricharan
1st Thing: Setting up PCR mixes … Fprimer = VF2, Rprimer = VR @ 10μM
Taq PCRs
Item
GoTaq (stuff < 3kb) … 100μL rxn
5x Buffer
20μL
10mM dNTPs
2μL
F Primer
5μL
R Primer
5μL
dH2O
62.5μL
GoTaq Polymerase
0.5μL
Colony Suspensionr
5μL
Temperature (°C)
Time
Initial Denaturation
95
5min
35 Cycles
95
45sec
65
45sec
72
2.5min
Final Extension
72
5min
Soak
4
Infinite
Phusion PCRs
Item
NEB Phusion (ccm) … 50μL
5x Buffer
10μL
10mM dNTPs
1μL
F Primer
2.5μL
R Primer
2.5μL
dH2O
31μL
GoTaq Polymerase
0.5μL
Colony Suspension
2.5μL
Temperature (°C)
Time
Initial Denaturation
98
5min
35 Cycles
98
10sec
63
30sec
72
2:10min
Final Extension
72
10min
Soak
4
Infinite
Picked 4 colonies of each 3A assembly (B0015+ccmA-H, tac+fadK, med Anderson+cymA)
Put tip in 20μL dH2O, pipetted up and down, spotted 3μL onto marked plate (Cam), incubated overnight … Later realized that these plasmids are kanamycin resistant
Put 0.5μL w/9.5μL of master mix above … ran cycler
Will run gel to check which ones are right
August 8, 2018
Colony PCR Checks of 3A Assemblies
Sricharan and Liz and Shannon
Lane
1
2
3
4
5
Item
2-log ladder
ccm colony 1
ccm colony 2
ccm colony 3
Phusion colony ctrl
Run @ 100V, 25min
No visible bands on this gel (?)
Lane
1
2
3
4
5
Item
2-log ladder
fadK colony 1
fadK colony 2
fadK colony 3
fadK colony 4
Lane
6
7
8
9
10
Item
cymA colony 1
cymA colony 2
cymA colony 3
cymA colony 4
GoTaq colony control
Run @ 100V, 25min
August 9, 2018
Transformations & OCs
Sricharan and Peter
Usual PC protocol
acr Gibson 8/3 (Strataclone, 1 tube, 1)
phz Gibson 8/3 (Strataclone, 1 tube 1)
weak Anderson + mtrCAB revised 8/8
acr Gibson 8/3 (RbCl cells)
phz Gibson 8/3 (RbCl cells)
pTac + mtrCAB revised 8/8
3, 4, 5, 6 - 2 tubes RbCl cells (50μL aliquots), 1μL DNA each tube
OCs of following in Cam:
tac+fadK 3A colony PCR (8/8)
pmt1231 Gibson (8/6)
med Anderson + mtrCAB w/ new start (8/8)
Week 14: 8/13/2018
August 13, 2018
Transformation of anderson, araC, and pBad/ araC promoters
Peter
**See Protocol from 7/16/18**
Transforming:
BBa_J23112 (amp)
BBa_J23103 (amp)
BBa_J23109 (amp)
BBa_J23117 (amp)
BBa_R0080 (inducible araC)
BBa_I0500 (inducible pBad/ araC)
Notes/Deviations:
Two tubes of cells/ transformation (to ensure one tube will have cells) so 0.5 μL of DNA added to each tube
Heat-shocked for 30 seconds
Pooled like tubes (same vector) after 1 hour incubation
August 13, 2018
OCs
Sricharan
OCs of pmt1231, acr, phz Gibsons - 2 colonies each
cymA, tac+fadK 3As - 3 colonies each
All OCs grew after 18 hours
August 14, 2018
Making Ampicillin Plates + Diagnostic Digests
Sricharan
From LCC’s MPs - Nanodropped - 3As are low concentration (~10-20ng/μL, Gibsons high concentration ~100ng/μL)
For digests:
Item
3A Assemblies
Gibsons
dH2O
0μL
12μL
CutSmart
2μL
2μL
DNA
17μL
5μL
EcoRI-HF
1μL
1μL
August 14, 2018
Lots of minipreps
Liz
Did 12 minipreps to digest and run on a gel
Pmt1231 (1 and 2)
Acr gibson (1 and 2)
Phz gibson
Tac and fadK 2A (1, 2, 3)
cymA 3A
Used GenScript kit and eluted in 30μL elution buffer
Minipreps were digested and run on a gel
Only pmt1231 had a logical band
Made a glycerol stock with 0.75mL 40% glycerol and 0.75mL pmt overnight culture
August 15, 2018
more minipreps
Liz
Did minipreps of overnight cultures
BBa_R0080
BBa_J23103
Used GenScript kit
Eluted in 30μL elution buffer
Nanodropped:
BBa_R0080: 113.97 ng/μL
BBa_J23103: 307.66 ng/μL
August 15, 2018
Colony PCR Gel
Sricharan and Liz
More colony PCRs using same protocol as some weeks before, this time using the correct antibiotic selection plate.
August 16, 2018
Colony PCR Gel
Sricharan and Liz
All samples had 5μL DNA and 1μL loading dye
Gel 1:
1
2
3
4
5
6
7
8
9
10
11
ladder
Mtr 1
Mtr 2
Mtr 3
Mtr 4
Mtr 5
Mtr 6
phz1
phz2
phz3
ladder
12
13
14
15
16
17
18
19
20
21
22
phz4
phz5
phz6
ccm1
ccm2
ccm3
ccm4
ccm5
ccm6
Kan phusion ctrl
ladder
Gel 2:
1
2
3
4
5
6
7
8
9
10
11
ladder
fadK1
fadK2
fadK3
fadK4
fadK5
fadK6
acr1
acr2
Kan ctrl gotaq
ladder
12
13
14
15
16
17
18
19
20
21
22
Cam ctrl gotaq
empty
cymA1
cymA2
cymA3
cymA4
cymA5
cymA6
Cam ctrl phusion
ladder
empty
August 16, 2018
phz Linear Gibson PCR
Liz
Thermocycler
time
Temperature (C)
Initial denaturation
30 s
98
35 cycles
10 s
98
20 s
64
2:10 (min:sec)
72
Final extension
10 min
72
hold
infinite
4
PCR tube
Phusion HF buffer
4μL
DMSO
0.6μL
10mM 1F primer
1μL
10mM 6F primer
1μL
dNTPs
0.4μL
Template DNA (used SK linear gibson)
1μL
phusion
0.2μL
Week 15: 8/20/2018
August 24, 2018
Restriction Digests for 3A Assemblies and 3A Assembly
Liz
cymA: cymA + J23110 (medium Anderson promoter) + pSB1K3
J23110
water
13.79μL
CutSmart
2μL
DNA (J23110 miniprep 7/27 LCC)
300ng = 2.21μL
EcoRI
1μL
SpeI
1μL
cymA
water
13.51μL
CutSmart
2μL
DNA (cymA miniprep SD 7/25)
300ng = 2.49μL
EcoRI
1μL
PstI
1μL
pSB1K3
water
4μL
CutSmart
2μL
DNA (J23110 miniprep 7/27 LCC)
300ng = 12μL
EcoRI
1μL
PstI
1μL
Put in heat block at 37C for 15 min
Heat killed at 80C for 15 min
Forgot that backbone also needs to be digested with DpnI so cooled, added 1μL DpnI to pSB1K3, heatblocked at 37C for 15 min, and heat killed at 80C for 15 min
Used digestions in 3A assembly:
cymA + J23110 + pSB1K3 (used open wetware protocol)
water
2.17μL
Digested pSB1K3
10ng = 0.7μL
Digested cymA
37.02ng = 2.45μL
Digested J23110
47.75ng = 3.18μL
T4 ligase buffer
1μL
T4 ligase
0.5μL
Thermocycler:
22.5C for 30 min
65C for 10 min
August 24, 2018
A lot of colony PCR fails and Linear Gibson
Sricharan
A new Linear Gibson of phz 1F-6R
5μL
NEB MMix
3.01μL
1F-3R (7/27)
0.53μL
phz3 (gBlock)
0.66μL
4F-6R (8/1)
0.80μL
dH2O
Running PCR on this Gibson (Phusion) using usual protocol, and using 1μL of template. Annealing temperature is 64 degrees C, extension time is 2min,30sec.
Week 16: 8/27/2018
August 27, 2018
cymA 3A gel
Sricharan and Peter
Changing some items to potentially have success in Gibson reactions:
Also redesigned inserts so that there are correct overlaps with pSB1C3
Using 0.05pmol backbone, equimolar inserts
pSB1C3 (SK 7/30)
acr1 7.27.2018
acr2
acr3 7.27.2018
3.57μL
gBlock (532bp)
gBlock (529bp)
gBlock (653bp)
67ng
16.57ng
17.12ng
21.14ng
2070bp
1.66μL
1.71μL
2.11μL
pSB1C3 (SK 7/30)
phz Start 7.27.2018
phz 1F-6R
phz End 7.27.2018
3.57μL
gBlock (426bp)
gBlock (5882bp)
gBlock (277bp)
67ng
13.79ng
190.4ng
8.97ng
2070bp
1.38μL
2.73μL
0.90μL
For actual reactions:
Cutting reaction volume in half (5μL Master mix, 5μL DNA + water)
August 27, 2018
phz 1F-6R Gel
Liz
Made a 1% agarose gel for linear phz 1F-6R gibson PCR
Purified linear 1F-6R gibson PCR (SK 8/24) using lambda kit
Nanodropped: 69.7 ng/μL
Gel:
1
2
3
4
ladder
Phz gibson pure PCR
pSB1C3 A
pSB1C3 B
5μL sample + 1μL dye
1μL dye+ 1μL sample + 4μL water
August 28, 2018
Linear Gibson PCR + Making more Backbone
Sricharan
Running Phusion PCR on phz 1F-6R (before the PCR was ~1kb, should have been ~6kb), pSB1C3
Usual protocol: for phz used 0.4μL sample in 20μL rxn, annealing at 64 degrees C, extension 2 minutes. Did PCR in HF and GC Buffers
For pSB1C3, used 0.2μL of Peter’s diluted J04450 MP (7/24) to 0.14ng/μL (100x). This was a 50μL rxn, annealing at 71 degrees C, 40s extension.
PCR purified on 8/28 by Sricharan
Nanodrop
phz HF
phz GC
pSB1C3 HF
Conc (ng/μL)
15.1
21.7
39.5
260/280
1.43
1.63
1.71
260/230
-0.27
-0.48
-0.49
Curves look like junk for all 3
Peter has gel on pg 49
August 28, 2018
Gel of Phz (1F-6R) pcr’s and pSB1C3
Peter
Lane
1
2
3
4
Item
2-log ladder
- 3 μL Phz (HF) pcr
-2 μL dH2O
-1 μL loading dye
Phz (GC) pcr
-2 μL dH2O
-1 μL loading dye
pSB1C3 (HF)
-2 μL dH2O
-1 μL loading dye
August 28, 2018
Transformation (RbCl JM109) of 3A’s from 8/1
Peter
**See Protocol from 7/16/18**
Transforming:
Acr circular gibson (8/27 SK)
CymA + J23110 + PSB1K3 3A (8/24 LCC)
Notes/Deviations
Two tubes of cells/ transformation (to ensure one tube will have cells) so 0.5 μL of DNA added to each tube. Exceptions: 2 x 1μL for Mtr (from 20 μL 3A so more dilute than std 10 μL 3A)
Heat-shocked for 30 seconds
Pooled like tubes (same vector) after 1 hour incubation
CAM for Acr
Kan for CymA
August 30, 2018
Overnight Cultures of cymA 3A Assemblies
Liz
Overnights of cymA 3A assemblies
Took 3 colonies from “RbCl transformation cymA + J23110 + pSB1K3 3A PC 8/28” plate
3A done by LCC 8/24
5mL LB broth + 5μL kanamycin
Put in 37C shaker overnight
August 31, 2018
Miniprep of Cultures Liz Picked up this Morning
Sricharan
MP of cymA + J23110 + pSB1K3 by LCC … Genscript Kit, eluted 30μL and glycerol stocks
September
Week 17: 9/3/2018
September 4, 2018
cymA 3A gel
Liz
Made 1% agarose gel
Nanodropped SK’s cymA minipreps 1-3 (8/31)
Miniprep 1: 21.05 ng/μL
Miniprep 2: 17.49 ng/μL
Miniprep 3: 29.24 ng/μL
Gel:
1
2
3
4
Ladder (6μL)
cymA miniprep 1
cymA miniprep 2
cymA miniprep 3
Samples loaded with 5μL sample and 1μL loading dye
Didn’t digest before running gel--ran again next day
September 5, 2018
cymA 3A gel 2.0
Liz
Digested cymA minipreps with EcoRI to run on gel
CutSmart
2μL
DNA (cymA minipreps 1-3 SK 8/31)
12μL
EcoRI
1μL
water
5μL
Heat blocked at 37C for 15 min
Heat killed at 65C for 15 min
Ran 1% agarose gel (same setup as previous day)
September 5, 2018
Gel extraction of phz
Peter
Notes:
Following Azady’s advice on getting greater concentration of pieces for gibson
Gel:
1
2
Ladder (6μL)
10 μL phz 4F-6R pure pcr (PC 7/13)
Extraction:
Bio basic kit
400 μL buffer * (0.048 g/100 mg) * (1000 mg/1 g) = 192 μL buffer used
September 7, 2018
phz 4F-6R PCRs - Aiming for high purity
Sricharan
Using
Original Gibson (7/12)
Old gel extract
PC new gel extract (9/5)
Usual Phusion PCR, 1μL template DNA, 67 degrees C annealing, 1:18 extension time. All put into 1.5mL tubes
Week 18: 9/10/2018
September 10, 2018
PCR purification, Nanodrop, Gel
Ryan
Started off by preparing a small gel, .125g agarose, 12.5 mL TAE, mixed and microwaved for 30 seconds. Added 1.25 uL SYBR Safe and poured into mold and let sit
PCR purification of Charan’s 3 PCR products from 9/3/18, phz 4F-6R
Volume of PCR product was 20 uL, so added 100 ul Binding Buffer, and loaded it into Spin column and centrifuged for 1 min, @10,000 rpm
Discard flow-through and add 700 uL Wash Buffer, centrifuge again (same settings)
Discard flow-through, centrifuge again
Transfer top of spin column into a new 1.5 mL tube for collection
Add 40 uL Elution Buffer and let sit for 1 min
Centrifuge again
I labelled the 3 PCR products 1, 2, 3 and ran a nanodrop
1 was found to be 22.8 ng/uL
2 was found to be 19.1 ng/uL
3 was found to be 16.5 ng/uL
Put gel into the electrophoresis machine and added 6 uL 2-log Ladder into lane 1
3 uL of PCR 1 + 2uL water + 1 uL loading dye into lane 2
3 uL of PCR 2 + 2uL water + 1 uL loading dye into lane 3
3 uL of PCR 3 + 2uL water + 1 uL loading dye into lane 4
And ran it for 30 minutes at 100V *23 minutes dye was near end of gel
September 10, 2018
Azady PCRs acr3 and pXB1C3
Liz
Acr
HF buffer
4uL
10mM dNTPs
0.4uL
F primer (Azady acr 3F)
1uL
R primer (Azady acr 3R)
1uL
Template DNA (acr 3 gBlock)
0.2uL
DMSO
0.6uL
phusion
0.2uL
Water
12.6uL
pSB1C3
HF buffer
4uL
10mM dNTPs
0.4uL
F primer (SB-prep-3P-1-mutated)
1uL
R primer (SB-prep-2Ea)
1uL
Template DNA (BBa_R0080 diluted 10x)
0.2uL
DMSO
0.6uL
phusion
0.2uL
Water
12.6uL
Thermocycler
Initial denaturation
98C
30s
35 cycles
98C
10s
67C
30s
72C
35s
Final extension
72C
10 min
hold
4C
infinite
September 13, 2018
Azady PCRs, acr 1 & 2
Sricharan
PCRing parts of acr to remove hairpins and amplify gBlocks, for better Gibson results
Resuspended primers in TE
spun down dry tubes
to 100µM in tubes, let sit 10 min
aliquot and dilute to 10µM
Usual Phusion HF Buffer
For acr 1, Primers: Azady acr 1F + Azady acr 1R, Anneal 68°C
For acr 2, Primers: Azady acr 2F + Azady acr 2R, Anneal 67°C
Usual cycle, with 15s extension time for both
Transferred into 1.5mL tubes
September 13, 2018
PCR Purifs + Gel of acr 1-3 + pSB1C3 mut
Sricharan
PCR purifying acr 1-3 gBlock PCRs & pSB1C3 mut (adapted for Gibson)
Lambda Kit, eluting with 30µL … Nanodrop and gel by PC
September 14, 2018
Gel of Acr g-block PSB1C3 pcr’s
Peter
Notes:
Following Azady’s advice to pcr the g-blocks and backbone before gibson
Backbone is mutated (using primers with one nucleotide difference) so can overlap with new acr 1 & 3 g-blocks and not form hairpin
Gel:
1
2
3
4
5
Ladder (6μL)
Acr 1 pcr purif. (9/14 SK)
-512 bp
Acr 2 pcr purif. (9/14 SK)
-529 bp
Acr 3 pcr purif. (9/14 SK)
-653 bp
Mutated pSB1C3 pcr purif. (SK 9/14)
Week 19: 9/17/2018
September 17, 2018
Azady PCR acr 1 and 2 (more DNA)
Liz
HF buffer
4uL
10mM dNTPs
0.4uL
F primer (Azady acr 1F/2F)
1uL
R primer (Azady acr 1R/2R)
1uL
Template DNA (acr 1, 2 gBlock)
2uL
DMSO
0.6uL
phusion
0.2uL
Water
10.8uL
Temperatures for thermocycler from Sricharan’s same PCR last week
September 18, 2018
Zymo DNA concentrator planning (acr)
Peter
Notes:
Concentrating fragments before gibson as suggested by Azady
Wanted 50 ng of acr 3 and calculated equimolar amounts of other fragments (acr 1 & 2)
Used 2:1 fragment to backbone ratio
Need to add at least 6 μL elution buffer
Added total of 10 μL of elution buffer to get 8.7 μL final product
Below is initial amounts added to column, see zymo kit for further details on the next steps and buffers to add
Centrifuging done at 15500 g
Component
Amount (μL)
Acr 3 pcr purif. (9/14 SK)
5.62
Acr 1 pcr purif. (9/14 SK)
3.25
Acr 2 pcr purif. (9/14 SK)
2.57
Mutated pSB1C3 pcr purif. (9/14 SK)
7.86
Binding Buffer
96.5 = 19.3 * 5 (total volume DNA mult. 5)
September 18, 2018
Gibson of acr (circular)
Peter
Component
Amount (μL)
Zymo pcr fragments product
8.7
SGI GA HiFi Master Mix
8.7
-run in thermo cycler @ 50 C for 1 hour
September 19, 2018
Transformation of acr Gibson (by PC)
Sricharan
We’ve now PCRed parts for acr Gibson and backbone (some mutations to remove secondary structure)
Peter concentrated DNA and ran Gibson yesterday
Note: (By accident) we forgot to restriction digest pSB1C3 mut
Will this affect product?
Also no - control with backbone … will do - control with water
Transformation of the usual RbCl2 cells with 1μL in each 50μL cells tube
Acr - transformed with 1μL acr Gibson each 50μL cells
Ctrl - transformed with 1μL dH2O each 50μL cells … 1 tube was forgotten so only transformed  tube
Plates labeled PC since Peter helped
September 19, 2018
Azady PCR: acr 3 and pSB1C3 (2nd time)
Sricharan
Acr 3: Phusion HF PCR with 67°C annealing, 15 sec extension, primers Azady acr 3F and azady acr 3R
pSB1C3: Phusion HF PCR with 66°C annealing, 40 sec extension, primers SB-prep-3P-1-mut and SB-prep-2-Ea
Week 20: 9/24/2018
September 24, 2018
Acr 1-3 PCR cleanup and gel
Liz
Made 1% agarose gel (small wells)
PCR purified acr 1-3 and pSB1C3
Acr 1-2 LCC 9/17
Acr 3 and pSB1C3 mut SK 9/21
Lambda kit
Eluted in 30μL
Nanodropped:
Acr 1: 12.68 ng/μL
Acr 2: 19.82 ng/μL
Acr 3 HF: 33.59 ng/μL
Acr 3 GC: 9.76 ng/μL
pSB1C3: 78.88 ng/μL
Gel:
1
2
3
4
5
6
7
8
ladder
Acr 1
Acr 2
Acr 3 HF
Acr 3 GC
pSB1C3
ladder
Acr 1
6μL
5μL sample, 1μL dye
6μL
1μL dye,
1.5μL water,
3.5μL sample
September 26, 2018
Transforming acr
Sricharan
Trying again to transform Gibson acr from 9/17
1 tube (50μL) Strataclone - 4μL acr Gibson (9/17)
2 tubes (100μL) RbCl cells - 8μL acr Gibson (9/17)
1 tube (50μL) RbCl cells - 4μL dH2O (ctrl)
Usual protocol for RbCl cells
For Strataclone
20 min on ice
45 sec heat shock
2 min on ice
250μL SOC - Incubate slightly >1hr
Plate 100μL or so
Tubes started 1 hr incubation at 2:45pm
End 3:45 pm
3 plates waiting on bench for Peter to plate - not in warm room since it seems humid there
September 27, 2018
acr OCs
Sricharan
Picked up transformations from yesterday
1 colony RbCl acr
Many (~20 colonies) Strataclone acr
OC’ed 1 RbCl and 6 Stratclone colonies
LB from Kate downstairs (says it was made 2013 on bottle?)
Accidentally forgot to add Cam so added after tips were submerged in tubes
In at 4:38pm, shaker downstairs
September 28, 2018
acr Miniprep/Diagnostic
Liz
Made 1% agarose gel
Minipreps of strataclone acr 1-6 and RbCl acr (SK 9/27)
Used genscript kit
Eluted in 30μL elution buffer
Single digest for diagnostic gel
CutSmart
2μL
water
7μL
Miniprep DNA (str acr 1-6 and RbCl acr)
10μL
EcoRI
1μL
Thermocycler:
37C for 15 minutes
65C for 15 min
4C hold
Gel:
1
2
3
4
5
6
7
8
ladder
Acr str 1
Acr str 2
Acr str 3
Acr str 4
Acr str 5
Acr str 6
RbCl acr
5μL sample, 1μL loading dye
Protocols
Note: Embedded PDFs not viewable in Safari. Click/tap "Download" to view them.
Download
3A Assembly
Download
Chloramphenicol LB Broth
Download
DNA Gel Extraction (EZ-10 Kit)
Download
DNA Ligation
Download
Enzyme DNA Digestion
Download
GenScript Miniprep
Download
Gibson Assembly
Download
Lambda PCR Purification Kit
Download
LB Agar Plates with Antibiotic
Download
PCR Protocol for Phusion® High-Fidelity DNA Polymerase
Download
PureLink PCR Purification Kit
Download
RbCl Competent Cell Transformation
Home
Project
Background
Description
Notebook
Interlab
Lab Notebook
Protocols
Parts
List of Parts
Human Practices
Community Outreach
Collaborations
Safety
Team
Attributions
University of Connecticut (UConn), Storrs, Connecticut 06269, United States of America
Contact us at uconnigem@gmail.com
