<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_UConn_Notebook skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><DIV id="top_title"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:UConn/Notebook</SPAN></H1></DIV><DIV id="HQ_page"><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><DIV class="headercontainer"><HEADER><NAV id="cssmenu"><UL><LI class="wide"><A class="item" href="https://2018.igem.org/Team:UConn">Home</A></LI><LI class="narrow"><A class="item" href="https://2018.igem.org/Team:UConn/Project">Project</A></LI><LI class="mobilenav"><A href="https://2018.igem.org/Team:UConn/Project#background" value="Reload Page" onclick="window.location.reload()">Background</A></LI><LI class="mobilenav"><A href="https://2018.igem.org/Team:UConn/Project#description" value="Reload Page" onclick="window.location.reload()">Description</A></LI><LI class="desktopnav"><A href="https://2018.igem.org/Team:UConn/Project#background">Background</A></LI><LI class="desktopnav"><A href="https://2018.igem.org/Team:UConn/Project#description">Description</A></LI></UL><LI class="narrow"><A class="item" href="https://2018.igem.org/Team:UConn/Notebook">Notebook</A></LI><LI class="mobilenav"><A href="https://2018.igem.org/Team:UConn/Notebook#interlab" value="Reload Page" onclick="window.location.reload()">Interlab</A></LI><LI class="mobilenav"><A href="https://2018.igem.org/Team:UConn/Notebook#labnotebook" value="Reload Page" onclick="window.location.reload()">Lab Notebook</A></LI><LI class="mobilenav"><A href="https://2018.igem.org/Team:UConn/Notebook#protocols" value="Reload Page" onclick="window.location.reload()">Protocols</A></LI><LI class="desktopnav"><A href="https://2018.igem.org/Team:UConn/Notebook#interlab">Interlab</A></LI><LI class="desktopnav"><A href="https://2018.igem.org/Team:UConn/Notebook#labnotebook">Lab Notebook</A></LI><LI class="desktopnav"><A href="https://2018.igem.org/Team:UConn/Notebook#protocols">Protocols</A></LI><LI class="narrow"><A class="item" href="https://2018.igem.org/Team:UConn/Parts">Parts</A></LI><LI class="mobilenav"><A href="https://2018.igem.org/Team:UConn/Parts#listofparts" value="Reload Page" onclick="window.location.reload()">List of Parts</A></LI><LI class="desktopnav"><A href="https://2018.igem.org/Team:UConn/Parts#listofparts">List of Parts</A></LI><LI class="narrow"><A class="item" href="https://2018.igem.org/Team:UConn/HumanPractices">Human Practices</A></LI><LI class="mobilenav"><A href="https://2018.igem.org/Team:UConn/HumanPractices#communityoutreach" value="Reload Page" onclick="window.location.reload()">Community Outreach</A></LI><LI class="mobilenav"><A href="https://2018.igem.org/Team:UConn/HumanPractices#collaborations" value="Reload Page" onclick="window.location.reload()">Collaborations</A></LI><LI class="desktopnav"><A href="https://2018.igem.org/Team:UConn/HumanPractices#communityoutreach">Community Outreach</A></LI><LI class="desktopnav"><A href="https://2018.igem.org/Team:UConn/HumanPractices#collaborations">Collaborations</A></LI><LI class="narrow"><A class="item" href="https://2018.igem.org/Team:UConn/Safety">Safety</A></LI><LI class="narrow"><A class="item" href="https://2018.igem.org/Team:UConn/Team">Team</A></LI><LI class="mobilenav"><A href="https://2018.igem.org/Team:UConn/Team#attributions" value="Reload Page" onclick="window.location.reload()">Attributions</A></LI><LI class="desktopnav"><A href="https://2018.igem.org/Team:UConn/Team#attributions">Attributions</A></LI></NAV></HEADER></DIV><DIV id="notebook" class="topsection"><DIV class="container"><H1>Notebook</H1></DIV></DIV><DIV id="notebooksection1" class="section"><DIV class="container"><H2>Interlab</H2><P>The purpose of this year‚Äôs interlab is to address the high level of variability in fluorescence measurements. Typically, fluorescence is measured using the optical density (OD), or the absorbance of light by a sample when measured at 600nm using a plate reader,  nanodrop, or spectrophotometer. However, since this method can be inaccurate, Interlab seeks to improve the standard method of measuring fluorescence in populations of cells. Since the number of cells in a sample can vary, if the fluorescence is normalized to the cell count of the sample, then the average amount of fluorescence can be calculated, which is likely more accurate than that of a population of unknown size. Therefore, Interlab will investigate if normalizing fluorescence measurements to colony-forming units (CFUs) will yield more precise results than OD.

Each iGEM team was asked to normalize the absorbance to the number of cells in two ways: by comparing sample absorbance to absorbance of a sample of known concentration of beads similar in size to E. coli cells and by counting CFUs from each sample.</P><H3>OD</H3><P>For the first method, absorbance was measured for a sample and serial dilution of silicon beads. These beads, similar in size and shape to E. coli cells, are expected to scatter light in a similar manner, providing an analogous sample of known concentration. The first part of this process was to get a reference point for OD<SUB>600</SUB> to easily interchange between Abs<SUB>600</SUB> and OD<SUB>600</SUB>, accounting for differences in volume between wells. In this measurement, four equal samples LUDOX CL-X (a 45% colloidal silica suspension) were compared to samples of ddH<SUB>2</SUB>O, and the OD was measured using a plate reader. By finding the mean of the replicates, correcting by subtracting the mean of the water to get just the absorbance of the silica, and dividing by the OD<SUB>600</SUB> from the provided Excel sheet, the conversion factor (OD<SUB>600</SUB>/Abs<SUB>600</SUB>) was calculated to be 3.500.

A particle standard curve was also developed using a serial dilution of monodisperse silica microspheres to plot the concentration versus absorbance, thus allowing the estimation of the number of cells based off the absorbance. The serial dilution was performed four times, and the Abs<SUB>600</SUB> was measured to create the following standard curves: 
</P><P class="clearfix">Finally, a fluorescence standard curve was created to have a point of comparison between outputs of different plate readers, getting rid of the arbitrary units assigned by different instruments. Fluorescein is used as an easier, lower-cost alternative to GFP as it has similar excitation and emission properties. Four identical serial dilutions were completed for fluorescein, and the Abs<SUB>600</SUB> was measured to create the following standard curves: 
</P><H3 class="clearfix">CFU</H3><P>This part of the Interlab found the volume of a 0.1 OD<SUB>600</SUB> culture and the number of cells in a known volume by growing a sample and counting the number of colonies that grew. For the ‚ÄúStarting Sample Preparation,‚Äù the OD<SUB>600</SUB> of diluted overnight cultures of positive and negative controls were taken along with that of blank media. Based on the starting sample, overnights were diluted in triplicate, and the OD<SUB>600</SUB> was taken and adjusted using the blank, yielding the absorbance. To find the number of cells in a given volume, a serial dilution was performed for each sample, and the last three dilutions were plated and grown overnight. Colonies were counted, and, with the understanding that one cell grows into one colony, the number of cells per unit of volume was found. To standardize across dilutions, we used the equation:
</P><DIV class="center"><P>number of colonies x final dilution factor = CFU/mL</P></DIV><H4>Dilution 3</H4><TABLE><TBODY><TR><TH>Plate</TH><TH>Number of Colonies</TH><TH>CFU/mL 0.1 OD<SUB>600</SUB> solution</TH></TR><TR><TD>1.1</TD><TD>185</TD><TD>14800000</TD></TR><TR><TD>1.2</TD><TD>28</TD><TD>2240000</TD></TR><TR><TD>1.3</TD><TD>79</TD><TD>6320000</TD></TR><TR><TD>2.1</TD><TD>Over 300</TD><TD>N/A</TD></TR><TR><TD>2.2</TD><TD>Over 300</TD><TD>N/A</TD></TR><TR><TD>2.3</TD><TD>Over 300</TD><TD>N/A</TD></TR><TR><TD>3.1</TD><TD>121</TD><TD>9680000</TD></TR><TR><TD>3.2</TD><TD>261</TD><TD>20880000</TD></TR><TR><TD>3.3</TD><TD>170</TD><TD>13600000</TD></TR><TR><TD>4.1</TD><TD>94</TD><TD>7520000</TD></TR><TR><TD>4.2</TD><TD>270</TD><TD>21600000</TD></TR><TR><TD>4.3</TD><TD>Over 300</TD><TD>N/A</TD></TR></TBODY></TABLE><H4>Dilution 4</H4><TABLE><TBODY><TR><TH>Plate</TH><TH>Number of Colonies</TH><TH>CFU/mL 0.1 OD<SUB>600</SUB> solution</TH></TR><TR><TD>1.1</TD><TD>18</TD><TD>14400000</TD></TR><TR><TD>1.2</TD><TD>7</TD><TD>5600000</TD></TR><TR><TD>1.3</TD><TD>6</TD><TD>4800000</TD></TR><TR><TD>2.1</TD><TD>65</TD><TD>52000000</TD></TR><TR><TD>2.2</TD><TD>105</TD><TD>84000000</TD></TR><TR><TD>2.3</TD><TD>76</TD><TD>60800000</TD></TR><TR><TD>3.1</TD><TD>42</TD><TD>33600000</TD></TR><TR><TD>3.2</TD><TD>26</TD><TD>20800000</TD></TR><TR><TD>3.3</TD><TD>12</TD><TD>9600000</TD></TR><TR><TD>4.1</TD><TD>31</TD><TD>24800000</TD></TR><TR><TD>4.2</TD><TD>24</TD><TD>19200000</TD></TR><TR><TD>4.3</TD><TD>33</TD><TD>26400000</TD></TR></TBODY></TABLE><H4>Dilution 5</H4><TABLE><TBODY><TR><TH>Plate</TH><TH>Number of Colonies</TH><TH>CFU/mL 0.1 OD<SUB>600</SUB> solution</TH></TR><TR><TD>1.1</TD><TD>1</TD><TD>8000000</TD></TR><TR><TD>1.2</TD><TD>0</TD><TD>0</TD></TR><TR><TD>1.3</TD><TD>1</TD><TD>8000000</TD></TR><TR><TD>2.1</TD><TD>1</TD><TD>8000000</TD></TR><TR><TD>2.2</TD><TD>2</TD><TD>16000000</TD></TR><TR><TD>2.3</TD><TD>8</TD><TD>64000000</TD></TR><TR><TD>3.1</TD><TD>1</TD><TD>8000000</TD></TR><TR><TD>3.2</TD><TD>3</TD><TD>24000000</TD></TR><TR><TD>3.3</TD><TD>1</TD><TD>8000000</TD></TR><TR><TD>4.1</TD><TD>0</TD><TD>0</TD></TR><TR><TD>4.2</TD><TD>5</TD><TD>40000000</TD></TR><TR><TD>4.3</TD><TD>3</TD><TD>24000000</TD></TR></TBODY></TABLE></DIV></DIV><DIV id="notebooksection2" class="section"><DIV class="container"><H2>Lab Notebook</H2><P>Click on each section to view the week's notebook.</P><H3>May</H3><BUTTON class="collapsible">Week 2: 5/21/2018</BUTTON><DIV class="content"><DIV class="notebookdate"><H4>May 22, 2018</H4><H4>Competent Cell Prep</H4><H4>Liz, Peter, Ryan, Shannon, Sricharan</H4></DIV><P>We‚Äôre preparing competent cells (E. coli, XL1 Blue) for the summer.

Craig and Sricharan tried to do this in December, but it didn‚Äôt work since the shaker was at 200rpm instead of at 250rpm.
Today: prepping stock solutions, streaking plate
</P><H4>TfbII (50 mL)</H4><TABLE><TBODY><TR><TH>Reagent</TH><TH>Concentration</TH><TH>Amount needed</TH><TH>Amount measured</TH></TR><TR><TD>MOPS</TD><TD>10mM</TD><TD>0.104g</TD><TD>0.1040g</TD></TR><TR><TD>CaCl<SUB>2</SUB></TD><TD>75mM</TD><TD>0.417g</TD><TD>0.4160g</TD></TR><TR><TD>RbCl<SUB>2</SUB></TD><TD>10mM</TD><TD>0.061g</TD><TD>0.0616g</TD></TR><TR><TD>Glycerol</TD><TD>15% (v/v)</TD><TD>7.5mL</TD><TD>7.5mL</TD></TR></TBODY></TABLE><UL><LI>Measured amounts of reagents in flask (next time, use beaker)</LI><LI>Added most of water (don‚Äôt top off, so you can adjust pH)</LI><LI>pH (w/stirring if possible) to 6.5, using 1M KOH dropwise - got to 6.52‚Ä¶ (make sure to standardize with pH 7 and 4 buffers)</LI><LI>Topped off in graduated cylinder</LI><LI>0.45¬µm filter sterilized</LI><LI>No need to autoclave this solution</LI><LI>Stored in Tigger (4¬∞C)</LI></UL><H4>TfbI (200 mL)</H4><TABLE><TBODY><TR><TH>Reagent</TH><TH>Concentration</TH><TH>Amount needed</TH><TH>Amount measured</TH></TR><TR><TD>Potassium Acetate</TD><TD>30mM</TD><TD>0.588g</TD><TD>0.589g</TD></TR><TR><TD>MnCl<SUB>2</SUB></TD><TD>50mM</TD><TD>1.98g</TD><TD>1.99g</TD></TR><TR><TD>CaCl<SUB>2</SUB></TD><TD>10mM</TD><TD>0.222g</TD><TD>0.224g</TD></TR><TR><TD>RbCl<SUB>2</SUB></TD><TD>10mM</TD><TD>0.242g</TD><TD>0.242g</TD></TR><TR><TD>Glycerol</TD><TD>15% (v/v)</TD><TD>30mL</TD><TD>30mL</TD></TR></TBODY></TABLE><UL><LI>Measured amounts of reagents in flask (next time, use beaker)</LI><LI>Added most of water (don‚Äôt top off, so you can adjust pH)</LI><LI>pH (w/stirring if possible) to 5.8, using .2M acetic acid dropwise - got to 5.82‚Ä¶ (make sure to standardize with pH 7 and 4 buffers)</LI><LI>Topped off in graduated cylinder</LI><LI>0.45¬µm filter sterilized</LI><LI>No need to autoclave this solution</LI><LI>Stored in Tigger (4¬∞C)</LI></UL><H4>·¥™a (500 mL)</H4><TABLE><TBODY><TR><TH>Reagent</TH><TH>Concentration</TH><TH>Amount needed</TH></TR><TR><TD>Bacto yeast extract</TD><TD>5g/L</TD><TD>2.5g</TD></TR><TR><TD>Bacco tryptone</TD><TD>20g/L</TD><TD>10g</TD></TR><TR><TD>MgSO<SUB>4</SUB></TD><TD>5g/L</TD><TD>2.5g</TD></TR><TR><TD>KOH</TD><TD>1M</TD><TD>Added dropwise until solution reached pH 7.6</TD></TR></TBODY></TABLE><UL><LI>Dry substances mixed in bottle (would be easier to use a beaker)</LI><LI>Water topped off to almost 500mL</LI><LI>Pour into flask to adjust pH using spinner</LI><UL><LI>Standardized pH meter using 7 and 10 buffers, pain to do</LI></UL><LI>Pour back into graduated cylinder to top off to 500 mL</LI><LI>Use sterile filter</LI><LI>Aliquot 100mL into two 300mL flasks with caps, remaining 300mL in bottle</LI><LI>Autoclave at liquid 30, 45 minutes dry</LI><LI>Removed from autoclave, allowed to cool, and transferred to 4‚ÑÉ fridge (Tigger)</LI></UL><P>Also streaked Œ®a plate</P><UL><LI>Sterile environment (with portable burner nearby)</LI><LI>Stabbed competent XL1 Blue cells (frozen stock) with a pipet tip</LI><LI>Gently spread frozen piece of cells onto plate with tip (be careful not to puncture agar)</LI><LI>Spread cells on plate with Pasteur spinner (not the plating beads, @Craig)</LI><LI>Put plate into incubator at 39¬∞C (not 37¬∞C, ugh)</LI></UL><DIV class="notebookdate"><H4>May 22, 2018</H4><H4>Interlab Calibration</H4><H4>Liz, Peter, Ryan, Shannon, Sricharan</H4></DIV><H4>Calibration 1, OD<SUB>600</SUB> Reference Point - LUDOX protocol:</H4><UL><LI>100¬µL LUDOX CL-X in wells A1, B1, C1, D1</LI><LI>100¬µL ddH<SUB>2</SUB>O in wells A2, B2, C2, D2</LI></UL><H4>Absorbances</H4><TABLE><TBODY><TR><TH>1</TH><TH>2</TH></TR><TR><TD>A</TD><TD>0.045</TD><TD>0.024</TD></TR><TR><TD>B</TD><TD>0.041</TD><TD>0.025</TD></TR><TR><TD>C</TD><TD>0.042</TD><TD>0.025</TD></TR><TR><TD>D</TD><TD>0.042</TD><TD>0.025</TD></TR></TBODY></TABLE><UL><LI>Wavelength: 600nm</LI><LI>Optics: top</LI><LI>Temperature: 25.9¬∞C</LI></UL><H4>Calibration 2</H4><P>Stock Solution (Vortex silica beads for 30 sec. before adding):</P><TABLE><TBODY><TR><TH>Reagent</TH><TH>Amount</TH></TR><TR><TD>Silica beads</TD><TD>96 ¬µL</TD></TR><TR><TD>ddH2O</TD><TD>904 ¬µL</TD></TR></TBODY></TABLE><UL><LI>Add 100 ¬µL ddH<SUB>2</SUB>O to wells A1, B1, C1, D1, ‚Ä¶, A12, B12, C12, D12</LI><LI>Add 200 ¬µL stock solution to A1, B1, C1, and D1 (vortex 10s before adding)</LI><LI>Transfer 100 ¬µL from A1 to A2, pipet to mix, transfer 100 ¬µL from A2 to A3, pipet to mix, etc. until A11. Repeat for B, C, and D.</LI><UL><LI>Cells A1, B1, C1, and D1 should be pure stock</LI><LI>Cells A12, B12, C12, and D12 should be pure ddH<SUB>2</SUB>O</LI></UL></UL><H4>Absorbance</H4><TABLE><TBODY><TR><TH>1</TH><TH>2</TH><TH>3</TH><TH>4</TH><TH>5</TH><TH>6</TH><TH>7</TH><TH>8</TH><TH>9</TH><TH>10</TH><TH>11</TH><TH>12</TH></TR><TR><TH>A</TH><TH>1.075</TH><TH>0.545</TH><TH>0.343</TH><TH>0.152</TH><TH>0.087</TH><TH>0.055</TH><TH>0.051</TH><TH>0.032</TH><TH>0.028</TH><TH>0.031</TH><TH>0.028</TH><TH>0.025</TH></TR><TR><TH>B</TH><TH>1.199</TH><TH>0.666</TH><TH>0.306</TH><TH>0.161</TH><TH>0.086</TH><TH>0.057</TH><TH>0.04</TH><TH>0.033</TH><TH>0.028</TH><TH>0.026</TH><TH>0.026</TH><TH>0.025</TH></TR><TR><TH>C</TH><TH>1.191</TH><TH>0.651</TH><TH>0.346</TH><TH>0.176</TH><TH>0.091</TH><TH>0.03</TH><TH>0.051</TH><TH>0.043</TH><TH>0.031</TH><TH>0.028</TH><TH>0.027</TH><TH>0.025</TH></TR><TR><TH>D</TH><TH>1.325</TH><TH>0.587</TH><TH>0.299</TH><TH>0.168</TH><TH>0.088</TH><TH>0.057</TH><TH>0.039</TH><TH>0.032</TH><TH>0.027</TH><TH>0.026</TH><TH>0.025</TH><TH>0.025</TH></TR></TBODY></TABLE><UL><LI>Wavelength: 600nm</LI><LI>Optics: top</LI><LI>Temperature: 26¬∞C</LI></UL><H4>Calibration 3: Fluorescence Standard Curve</H4><H4>Fluorescein stock solution:</H4><TABLE><TBODY><TR><TH>Reagent</TH><TH>Amount</TH></TR><TR><TH>Fluorescein</TH><TH>Provided in Kit (5 tubes)</TH></TR><TR><TH>1x PBS pH 7.4-7.6</TH><TH>10 mL</TH></TR></TBODY></TABLE><UL><LI>Spun down fluorescein in tube so the pellet was at the bottom</LI><LI>Made ~ 40 mL 1x PBS in falcon tube (originally used 10 x accidentally for first 3 attempts/ tubes)</LI><LI>Used 1 mL of 1x PBS to resuspend fluorescein in kit tube &gt; makes 10x fluorescei¬µLtocol calls for pH of 7.4-7.5. Problem? &gt; email iGEM</LI><LI>Diluted 10 fluorescein stock with 900 ¬µL of 1x PBS and 100 ¬µL of 10x fluorescein stock to make 1x fluorescein solution (10 M)</LI></UL><P>Serial Dilutions:</P><UL><LI>Used clear flat-bottom 96 well plates</LI><LI>Put 200 ¬µL of 1x fluorescein into column 1 rows A, B, C, D</LI><LI>Put 100 ¬µL of 1x PBS in columns 2-12 for rows A, B, C, D</LI><UL><LI>Column 12 for all rows (A, B, C, D) is control and only contains 100 ¬µL 1x PBS</LI></UL><LI>Transferred 100 ¬µL of fluorescein stock from A1 to A2</LI><LI>Mixed A2 &gt; pipetting up and down 3 times</LI><LI>Transferred 100 ¬µL from A2 to A3 and mixed A3</LI><LI>Repeated until after mixing A11 &gt; then 100 ¬µL removed from A11 and put into liquid waste</LI><LI>Process described for row A repeated for all other rows (B, C, D)</LI></UL><H4>Fluorescence</H4><TABLE><TBODY><TR><TH>1</TH><TH>2</TH><TH>3</TH><TH>4</TH><TH>5</TH><TH>6</TH><TH>7</TH><TH>8</TH><TH>9</TH><TH>10</TH><TH>11</TH><TH>12</TH></TR><TR><TH>A</TH><TH>497</TH><TH>266</TH><TH>144</TH><TH>75</TH><TH>41</TH><TH>23</TH><TH>14</TH><TH>9</TH><TH>8</TH><TH>6</TH><TH>6</TH><TH>5</TH></TR><TR><TH>B</TH><TH>532</TH><TH>302</TH><TH>153</TH><TH>81</TH><TH>44</TH><TH>24</TH><TH>25</TH><TH>10</TH><TH>7</TH><TH>7</TH><TH>5</TH><TH>5</TH></TR><TR><TH>C</TH><TH>532</TH><TH>295</TH><TH>154</TH><TH>81</TH><TH>45</TH><TH>25</TH><TH>15</TH><TH>10</TH><TH>7</TH><TH>7</TH><TH>6</TH><TH>5</TH></TR><TR><TH>D</TH><TH>547</TH><TH>297</TH><TH>159</TH><TH>83</TH><TH>45</TH><TH>25</TH><TH>16</TH><TH>11</TH><TH>8</TH><TH>7</TH><TH>6</TH><TH>5</TH></TR></TBODY></TABLE><UL><LI>Excitation: 485 nm /20 nm</LI><LI>Emission: 528 nm /20 nm</LI><LI>Optics: top</LI><LI>Temperature: 25.8¬∞C (room temp)</LI></UL><DIV class="notebookdate"><H4>May 23, 2018</H4><H4>Competent Cell Prep</H4><H4>Sricharan</H4></DIV><UL><LI>Took the Œ®a plate out around 9am</LI><UL><LI>It was densely packed with colonies (tan color), but some isolated ones appeared towards one edge of the plate</LI></UL><LI>Kept plate at 4¬∞C cold room during day</LI><LI>Overnight cultures prepared near portable burner (to the delight of Toni watching me try and refill the burner)</LI><UL><LI>Picked 3 colonies with P200 tip</LI><LI>Each colony into a separate round bottomed tube with loose caps, filled with Œ®b media</LI><LI>Tubes placed in shaker at 37¬∞C and 250rpm around 4:30pm</LI></UL></UL><DIV class="notebookdate"><H4>May 24, 2018</H4><H4>Competent Cell Prep</H4><H4>Liz, Peter, Shannon, Sricharan</H4></DIV><UL><LI>Took the overnight cultures out around 9:15am</LI><UL><LI>They all are cloudy compared to the original media, so decent growth it seems</LI><LI>Will start subcultures around noon</LI><LI>Cultured in shaker for an hour and 22 minutes and we over grew both cultures by a little</LI><LI>Prepared another overnight culture</LI></UL></UL><DIV class="notebookdate"><H4>May 24, 2018</H4><H4>Competent Cell Prep</H4><H4>Liz, Peter, Shannon, Sricharan</H4></DIV><UL><LI>Took the overnight cultures out around 9:00am</LI><UL><LI>Growth in both cultures</LI><LI>Pre-warmed Œ®b media before subculturing</LI><LI>Subcultured in Œ®b media for 45 minutes (Flask A around 0.36 OD600; Flask B around 0.38 OD600)</LI><LI>Checked again at 55, then 58 minutes and each was around 0.48 OD600</LI><LI>Spun cells in pots in MSE 18 to isolate pellet and removed supernatant</LI><LI>Resuspended in TfbI and spun again and removed supernatant </LI><LI>Resuspended in TfbII, pooled A and B, and chilled on ice for 15 minutes</LI><LI>Aliquoted 200 ¬µL of cell solution into autoclaved microcentrifuge tubes and snap-froze using liquid nitrogen</LI><LI>Put competent cells into -80 fridge on second floor</LI></UL></UL></DIV><BUTTON class="collapsible">Week 3: 5/28/2018</BUTTON><DIV class="content"><DIV class="notebookdate"><H4>May 29, 2018</H4><H4>Interlab Transformation</H4><H4>Shannon, Liz</H4></DIV><H4>Plasmid Re-suspension</H4><P>Add 10 ¬µL dH2O to wells 2D, 2B, 2F, 2H, 2J, 2L, 2N, and 2P on plate 7 from the interlab kit and pipette to resuspend the DNA in each well. Let sit for a few minutes. Transfer to 1.5 mL tubes.</P><UL><LI>Note: 2D and 2B were accidently added to the same 1.5 mL tube, so these two were re-done using 21B and 21D from kit plate 7 from the 2017 interlab kit, which have the same corresponding DNA.</LI></UL><DIV class="notebookdate"><H4>May 29, 2018</H4><H4>Chloramphenicol Plate Preparation</H4><H4>Ryan, Peter, Sricharan</H4></DIV><H4>Preparing Chloramphenicol Stock</H4><UL><LI>Made 10 mL stock of 35 mg/mL chloramphenicol in EtOH (this is 1000x as specified by iGEM)</LI><LI>Took from Rachel‚Äôs floor (we bought it though), 0.3496g made up to 10mL volume</LI></UL><H4>Preparing Chloramphenicol LB Plates (24)</H4><UL><LI>Collected ingredients for LB media and added to beaker</LI><LI>Q/s‚Äôd with dH<SUB>2</SUB>0 to 400</LI><LI>Mixed in beaker for 20 min and Q/s‚Äôd to about 450 mL with spray bottle to get solid particles off of the sides of the beaker and into the solution</LI><LI>Tried pouring into grad cylinder to finish Q/s but still goupy</LI><LI>Poured ~ 50 mL of dH20 into grad. Cylinder to capture the goop stuck to the bottom after pouring as much as we could back into the beaker. Poured this 50 mL back into the beaker as well</LI><LI>Turned up temp and stir speed on hot plate and stirred for another 25 minute in beaker</LI><LI>Poured into bottle and stirred (stopping frequently to shake bottle) for ~ 30 minutes</LI><LI>Autoclaved on S03 setting for 30 minutes (autoclaving less than a liter of liquid)</LI><LI>Added 0.5 mL of chloramphenicol (1000x, 35 mg/mL) to solution</LI><LI>Plated LB + chloramphenicol onto 24 plates and put in cold room (-4 degrees C)</LI><UL><LI>Rushed putting the plates away, so there might be some condensation on plates</LI></UL></UL><DIV class="notebookdate"><H4>May 30, 2018</H4><H4>Interlab Transformation</H4><H4>Shannon, Liz, Ryan</H4></DIV><UL><LI>Thawed DH5-alpha competent cells on ice</LI><LI>Pipet 50 ŒºL competent cells into empty 1.5 mL tubes</LI><LI>Pipet 1 ŒºL resuspended DNA (from yesterday, see page above) into individual tubes (according to well number)  with competent cells</LI><UL><LI>DNA from tubes: 21B, 21D, 2F, 2H, 2J, 2L, 2N, and 2P in box 1 from freezer</LI></UL><LI>Pipet 1 ŒºL control DNA (10pg/ŒºL) into control tube with competent cells</LI><LI>Incubate tubes on ice for 30 min (0:00-30:23)</LI><LI>Heat shock at 42‚ÑÉ for 45s (30:23-31:08)</LI><LI>Pipet 950 ŒºL SOC media at room temperature into each tube</LI><LI>Incubate at 37‚ÑÉ for 1 hr at 300 rpm</LI><LI>Plate 100 ŒºL transformation (lower concentration)</LI><UL><LI>Condensation got on agar, tried tapping plates to remove water but some remained</LI><LI>Spread water with spinners before putting transformation on plates</LI><LI>Unsure if plates N and P got switched</LI></UL><LI>Spin down cells at 6800g for 3 min and discard 800 ŒºL supernatant</LI><LI>Resuspend pellet in the remaining 100 ŒºL and plate (higher concentration)</LI><UL><LI>Amount of liquid remaining for resuspension varied from tube to tube</LI></UL><LI>Incubate at 37‚ÑÉ for 14-18 hours</LI><UL><LI>Put into incubator ~6pm</LI><LI>Incubator is kept at 39‚ÑÉ</LI><LI>Checked after 15.5 hours, barely any growth</LI></UL></UL><DIV class="notebookdate"><H4>May 30, 2018</H4><H4>Competent Cell Transformation (Scott-Simanis, Test Kit)</H4><H4>Peter, Sricharan</H4></DIV><UL><LI>2 vials of Scott-Simanis competent cells from 5/25/18 thawed at RT (200 ŒºL in each tube)</LI><LI>After thawed, placed on ice for 10 minutes</LI><UL><LI>At 8 minutes, aliquoted 50 ŒºL (150 ŒºL from each tube) into 6 separate tubes</LI><LI>At 10 minutes, added 1 ŒºL of DNA from Competent Cell Test Kit</LI><UL><LI>Duplicates of 10, 50, 100 pg/ŒºL BBa_J04450, RFP construct</LI></UL></UL><LI>Chilled aliquots on ice 30 min</LI><LI>Heat shocked aliquots at 42 ¬∞C for 90 s</LI><LI>Incubated on ice for 90 s</LI><LI>Added 200ŒºL Œ®b media to each aliquot</LI><LI>Incubated at 37 ¬∞C, 300rpm for 1 hour</LI><LI>Plated onto LB plates with chloramphenicol (35 Œºg/mL) with spinners, under portable burner</LI><LI>Incubated at 39 ¬∞C incubator downstairs starting at 6pm</LI></UL><DIV class="notebookdate"><H4>May 31, 2018</H4><H4>LB + Chloramphenicol Media Preparation (w/o agar)</H4><H4>Peter, Sricharan, Liz, Shannon, Ryan</H4></DIV><UL><LI>Used for inoculation for O.C. &amp; dilutions + control for transformation fluorescence readings</LI></UL><H4>Preparing Chloramphenicol LB media (500 mL)</H4><UL><LI>Collected and measured out ingredients for LB media and added to bottle</LI><LI>Q/s‚Äôd with dH<SUB>2</SUB>0 to 400 mL and shook bottle to dissolve ingredients</LI><LI>Stirred in beaker for 15 min</LI><LI>Poured into grad cylinder to finish Q/s-ing to 500 mL</LI><LI>Poured back into bottle and stirred for ~ a minute</LI><LI>Autoclaved on S03 setting for 30 minutes (autoclaving less than a liter of liquid)</LI><LI>Added 0.5 mL of chloramphenicol (1000x, 35 mg/mL) to solution</LI></UL><DIV class="notebookdate"><H4>May 31, 2018</H4><H4>Interlab Overnight Cultures</H4><H4>Peter, Sricharan, Liz, Shannon, Ryan</H4></DIV><UL><LI>Condensation on plates = osmotic stress on cells‚Ä¶. Not a lot of growth</LI><UL><LI>Checked plates at 9am - not a lot of growth</LI><LI>Took plates out at 12pm, at least a few colonies on most plates with concentrated transformations, except Wells 21D (2017), 21B (2017)</LI></UL><LI>Made 2 OCs for each well with most well defined/isolated colonies from each well</LI><UL><LI>Put into ‚Äúnew‚Äù old shaker in classroom (cleaned in morning with 70% ethanol)</LI><LI>220rpm, 37 ¬∞C</LI></UL></UL><DIV class="notebookdate"><H4>June 1, 2018</H4><H4>Interlab Fluorescence Measurements</H4><H4>Peter, Sricharan, Liz, Shannon, Ryan, Toni</H4></DIV><UL><LI>Grabbed OCs @ 9:15am from shaker</LI><LI>Diluted 0.5mL of OCs 10x, put foil on diluted tubes</LI><LI>Took Abs<SUB>600</SUB> readings on plate reader (100ŒºL samples)</LI></UL><TABLE><TBODY><TR><TH>Samples on Plate</TH><TH>1</TH><TH>2</TH><TH>3</TH><TH>4</TH><TH>5</TH><TH>6</TH><TH>7</TH></TR><TR><TD>G</TD><TD>21D (‚Äò17)</TD><TD>2F-1</TD><TD>2H-1</TD><TD>2J-1</TD><TD>2L-1</TD><TD>2N-1</TD><TD>2P-1</TD></TR><TR><TD>H</TD><TD>2F-2</TD><TD>2H-2</TD><TD>2J-2</TD><TD>2L-2</TD><TD>2N-2</TD><TD>2P-2</TD></TR></TBODY></TABLE><P>Since <SPAN class="italic">Abs=ebc</SPAN>, and we need to dilute to a target Abs<SUB>600</SUB> of 0.02, and final volume of 12mL, V<SPAN class="italic">aliquot</SPAN>=0.24/Abs<SUB>600</SUB>mL. Also, V<SUB class="italic">media</SUB> = 12mL-V<SUB class="italic">aliquot</SUB></P><P>Need to put in values from data on computer</P><TABLE><TBODY><TR><TH>Abs<SUB>600</SUB></TH><TH>1</TH><TH>2</TH><TH>3</TH><TH>4</TH><TH>5</TH><TH>6</TH><TH>7</TH></TR><TR><TD>G</TD></TR><TR><TD>H</TD></TR></TBODY></TABLE><TABLE><TBODY><TR><TH>V<SUB>aliquot</SUB>(mL)</TH><TH>1</TH><TH>2</TH><TH>3</TH><TH>4</TH><TH>5</TH><TH>6</TH><TH>7</TH></TR><TR><TD>G</TD><TD>7.7</TD><TD>4.6</TD><TD>2.3</TD><TD>2.3</TD><TD>1.7</TD><TD>2.8</TD><TD>2.1</TD></TR><TR><TD>H</TD><TD>1.8</TD><TD>2.2</TD><TD>2.3</TD><TD>2.0</TD><TD>3.1</TD><TD>2.4</TD></TR></TBODY></TABLE><TABLE><TBODY><TR><TH>V<SUB>aliquot</SUB>(mL)</TH><TH>1</TH><TH>2</TH><TH>3</TH><TH>4</TH><TH>5</TH><TH>6</TH><TH>7</TH></TR><TR><TD>G</TD><TD>4.3</TD><TD>7.4</TD><TD>9.7</TD><TD>9.7</TD><TD>10.3</TD><TD>9.2</TD><TD>9.9</TD></TR><TR><TD>H</TD><TD>10.2</TD><TD>9.8</TD><TD>9.7</TD><TD>10.0</TD><TD>8.9</TD><TD>9.6</TD></TR></TBODY></TABLE><UL><LI>@12pm Put diluted &amp; absorbance-corrected cultures back in shaker</LI><UL><LI>220rpm, 37 ¬∞C for 6 hours</LI><LI>Used 0.5mL aliquot of these cultures for t = 0h readings</LI><UL><LI>Placed on 96-well plate as in protocol</LI><LI>Note we don‚Äôt have negative or positive controls‚Ä¶ Need to redo next week</LI></UL></UL></UL><DIV class="notebookdate"><H4>June 1, 2018</H4><H4>LB + Chloramphenicol Plates Preparation</H4><H4>Peter, Ryan</H4></DIV><UL><LI>To be used for competent SS cell test &amp; positive and negative controls for Interlab</LI></UL><H4>Preparing Chloramphenicol LB plates (24)</H4><UL><LI>Collected and measured out ingredients for LB media and added to bottle</LI><LI>Q/s‚Äôd with dH<SUB>2</SUB>0 to 400 and shook bottle to dissolve ingredients</LI><LI>Stirred in beaker for 30 min</LI><LI>Poured into grad cylinder to finish Q/s to 500 mL</LI><LI>Poured back into bottle and stirred for ~ a minute</LI><LI>Autoclaved on S03 setting for 30 minutes (autoclaving less than a liter of liquid)</LI><LI>Added 0.5 mL of chloramphenicol (1000x, 35 mg/mL) to solution</LI><LI>Let plates cool for an hour on lab bench</LI><LI>Put them in the hood with the plates half open w/ agar side on the bottom to get rid of condensation (should have done this immediately to dry them)</LI></UL><DIV class="notebookdate"><H4>June 1, 2018</H4><H4>Title</H4><H4>Names</H4></DIV><DIV class="notebookdate"><H4>June 1, 2018</H4><H4>Title</H4><H4>Sricharan, Liz</H4></DIV><TABLE><TBODY><TR><TH>3</TH><TH>4</TH><TH>5</TH><TH>6</TH><TH>7</TH><TH>8</TH><TH>9</TH></TR><TR><TD>A (absorbance)</TD></TR><TR><TD>A (fluorescence)</TD></TR><TR><TD>B (absorbance)</TD></TR><TR><TD>B (fluorescence)</TD></TR><TR><TD>C (absorbance)</TD></TR><TR><TD>C (fluorescence)</TD></TR><TR><TD>D (absorbance)</TD></TR><TR><TD>D (fluorescence)</TD></TR><TR><TD>E (absorbance)</TD></TR><TR><TD>E</TD></TR><TR><TD>F(absorbance)</TD></TR><TR><TD>F (fluorescence)</TD></TR><TR><TD>G</TD></TR><TR><TD>G (fluorescence)</TD></TR><TR><TD>H</TD></TR><TR><TD>H (fluorescence)</TD></TR></TBODY></TABLE></DIV><H3>June</H3><BUTTON class="collapsible">Week 4: 6/4/2018</BUTTON><DIV class="content"><DIV class="notebookdate"><H4>June 4, 2018</H4><H4>LB + Chloramphenicol Plate Prep</H4><H4>Ryan</H4></DIV><P>See protocol from 6/1/18 for LB + Chloramphenicol Plate Prep</P><H4>Notes:</H4><UL><LI>Made 23 plates (whoops)</LI><LI>Burned serological pipette...</LI><LI>‚Ä¶ and part of a plate</LI></UL><DIV class="notebookdate"><H4>June 4, 2018</H4><H4>Interlab Transformation of 21B and 21D</H4><H4>Liz</H4></DIV><P>See protocol from 5/29/18 transformation</P><H4>Deviations/Notes</H4><UL><LI>Plates were taken out of fridge and warmed to room temperature</LI><UL><LI>No condensation on plates because they were allowed to mostly cool on the bench and then transferred to the hood with the lids partially off in the process of making the plates</LI></UL><LI>Only doing DNA from wells 21B and 21D (from 2017 kit plate 7)</LI><UL><LI>Colonies didn‚Äôt grow on plates last time (5/29)</LI><LI>For each sample, made 1 dilute plate and 2 concentrated plates with the hope of actually getting colonies this time</LI></UL><LI>Forgot to prechill tubes before putting in competent cells and resuspended DNA</LI><LI>Centrifuged at 12:45 during 30 minute incubation on ice</LI><LI>Only ended up making 1 concentrated plate each for 21B and 21D</LI><LI>Put in incubator at 4:20pm</LI></UL><H4>Times</H4><P>0:00-30:00 - incubate on ice for 30 minutes
30:00-30:45 - heat shock at 42‚ÑÉ for 45 s
30:45-35:45 - incubate for ice for 5 min</P><DIV class="notebookdate"><H4>June 4, 2018</H4><H4>SS Competent Cell transformation Test (Day 1)</H4><H4>Peter</H4></DIV><P>See protocol ‚ÄúCompetent Cell Transformation (Scott-Simanis, Test Kit)‚Äù from 5/29/18 transformation test</P><H4>Deviations/Notes</H4><UL><LI>Plates were taken out of fridge and warmed to room temperature</LI><UL><LI>No condensation on plates because they were allowed to mostly cool on the bench and then transferred to the hood with the lids partially off in the process of making the plates</LI></UL><LI>Œ®b taken out of fridge to warm to room temperature</LI><LI>Only made samples from tube 1 for 10 pg/ ŒºL</LI><LI>Took two tubes of competent cells from -80 degrees celsius</LI><LI>Took ~ 7 minutes to add DNA (too long for cells to be warm during this step)</LI><LI>Centrifuged at 12:45 during 30 minute incubation on ice (should have done this before)</LI><LI>Re-plated 100 pg/ ŒºL plate 1 (labeled new plate ‚Äú1b‚Äù) because burned cells with pasteur pipette for plate 1a</LI><LI>Potentially killed cells for 10 pg/ ŒºL on plate 1b</LI><LI>Put in incubator at ~ 4:20pm</LI></UL><DIV class="notebookdate"><H4>June 5, 2018</H4><H4>SS Competent Cell transformation Test (Day 2)</H4><H4>Peter</H4></DIV><UL><LI>Checked cells @ 9:15 am. Some growth on 100 pg/ ŒºL plate 1a and no visible growth on others</LI><LI>Checked again @ 10:15 am and same growth situation</LI></UL><DIV class="notebookdate"><H4>June 5, 2018</H4><H4>Checking Distribution Kit Plasmids</H4><H4>Sricharan, Liz, Peter, Shannon</H4></DIV><P>Needed to run a gel for 2017 kit 7 wells 21B and 21D to check to see if the plasmid was there because the transformation yielded no colonies again</P><UL><LI>Used 2017 kit 7 wells 21B and 21D, 2017 kit 3 well 8P, and 2017 kit 2 well 6F</LI><LI>Originally thought we had to do a digest of plasmids before running gel so measured concentration of DNA in samples using nanodrop (measure 1ŒºL)</LI><UL><LI>Blank with water first</LI></UL><LI>Results</LI><UL><LI>Kit 7 well 21B: 134.4 ng/ŒºL</LI><LI>Kit 7 well 21D: 109.0 ng/ŒºL</LI><LI>Kit 3 well 8P: 114.7 ng/ŒºL</LI><LI>Kit 2 well 6F: 131.7 ng/ŒºL</LI></UL><LI>Made 2 small agarose gels (1%)</LI><LI>25mL 1x TAE, 0.25g agarose warmed in microwave</LI><LI>2.5 ¬µL SYBR Safe added</LI></UL><P>All wells in gel from 2017 Kits</P><TABLE><TBODY><TR><TH>Well</TH><TH>1</TH><TH>2</TH><TH>3</TH><TH>4</TH><TH>5</TH><TH>6</TH><TH>7</TH><TH>8</TH><TH>9</TH><TH>10</TH></TR><TR><TD>Item</TD><TD>2-Log Ladder</TD><TD>Kit 7, Well 21D</TD><TD>Kit 7, Well 21B</TD><TD>Kit 2, Well 6F</TD><TD>Kit 3, Well 8D</TD><TD>2-Log Ladder</TD><TD>Test Kit 100pg/¬µL</TD><TD>Test Kit 50pg/¬µL</TD><TD>Test Kit 10pg/¬µL</TD><TD>Linearized pSB1C3</TD></TR></TBODY></TABLE><UL><LI>Ran 52 minutes @ 50V on mini gel rig</LI><UL><LI>No condensation (new TAE)</LI></UL><LI>Image on transilluminator hard to see</LI><LI>Similar gel image - only clear lanes were ladder and linearized backbone</LI><UL><LI>Diffuse spots for plasmids in wells 2-5, we should be fine</LI></UL></UL><DIV class="notebookdate"><H4>June 6, 2018</H4><H4>Redoing Interlab and Scott-Siamanis Transformations</H4><H4>Sricharan, Liz, Peter, Shannon</H4></DIV><UL><LI>Redoing transformations from earlier this week (since they didn‚Äôt work)</LI><UL><LI>Using water bath at 42¬∞C instead of heat block for shocking cells</LI><LI>Keeping all tubes prechilled/on ice</LI><LI>Prewarming SOC media</LI><LI>Using incubator to shake cells &lt;300rpm instead of thermomixer</LI><UL><LI>This incubator thing is probably overkill, we‚Äôll go back to thermomixer later</LI></UL></UL><LI>Got cells (2 tubes SS, 1 tube DH5-alpha) from -80¬∞C</LI><UL><LI>SS thawed off ice ~3min</LI><LI>DH5-alpha thawed on ice ~5min</LI></UL><LI>Immediately portioned 50¬µL aliquots of cells into several 1.5mL tubes</LI><UL><LI>Scott-Simanis cells - samples</LI><UL><LI>Competent Cell Test Kit</LI><UL><LI>100pg/¬µL</LI><LI>50pg/¬µL</LI><LI>10pg/¬µL</LI></UL><LI>Well 2F, Plate 7, 2018 Kit</LI></UL><LI>DH5-alpha cells - samples</LI><UL><LI>Competent Cell Test Kit, 10pg/¬µL</LI><LI>pUC19 Control</LI><LI>Plate 7, 2018 Kit</LI><UL><LI>Well 4B (0.2 ¬µL)</LI><LI>Well 4B (1 ¬µL)</LI><LI>Well 4D (0.2 ¬µL)</LI><LI>Well 4D (1 ¬µL)</LI><LI>Well 2F (1 ¬µL)</LI></UL></UL></UL><LI>For SS</LI><UL><LI>Incubate 10 minute on ice, add DNA (1 ¬µL), incubate on ice 30 min</LI></UL><LI>For DH5-alpha</LI><UL><LI>Add DNA immediately (1 ¬µL or 0.2 ¬µL if indicated on sample name), incubate on ice 30 min</LI></UL><LI>Heat shock @ 42¬∞C in water bath, 45 seconds (DH5-alpha) or 90s (SS)</LI><UL><LI>Accidentally dropped DH5-alpha samples into water, held in water after</LI></UL><LI>Incubate on ice 5 min (DH5-alpha) or 90s (SS)</LI><LI>Add (prewarmed) 950¬µL SOC media to each DH5-alpha tube, and 200¬µL Œ®b into each SS tube</LI><LI>Incubate @ 37¬∞C in shaker in classroom ~250rpm</LI><LI>Plated 100¬µL on LB+Cam plates</LI><UL><LI>For DH5-alpha, followed iGEM protocol to concentrate solutions</LI><LI>Only plated for DH5-alpha controls after concentrated</LI></UL><LI>Incubated at 39¬∞C downstairs ~4:30pm</LI></UL><DIV class="notebookdate"><H4>June 7, 2018</H4><H4>Interlab Positive and Negative Controls</H4><H4>Sricharan, Liz, Peter, Shannon</H4></DIV><UL><LI>Took plates from yesterday out ~9:30am</LI><UL><LI>Good growth on DH5-alpha plates (Interlab)</LI><UL><LI>Except pUC19 control, which was aliquoted after comp cell tube was dropped into ethanol</LI></UL><LI>No growth on most Scott-Simanis plates (not even Well 2F)</LI><UL><LI>We know cells are not competent now - plates and protocol are fine</LI></UL></UL><LI>Will do overnight cultures for Interlab cells and for new batch of competent cells this afternoon</LI></UL><DIV class="notebookdate"><H4>June 7, 2018</H4><H4>Psi Solutions</H4><H4>Liz, Peter</H4></DIV><H4>Psi A</H4><TABLE><TBODY><TR><TH>Substance</TH><TH>Concentration</TH><TH>Protocol Amount</TH><TH>Actual Amount</TH></TR><TR><TD>Bacto yeast extract</TD><TD>5 g/L</TD><TD>0.625 g</TD><TD>0.6357 g</TD></TR><TR><TD>Bacco tryptone</TD><TD>20 g/L</TD><TD>2.5 g</TD><TD>2.5090 g</TD></TR><TR><TD>MgSO<SUB>4</SUB></TD><TD>5 g/L</TD><TD>0.625 g</TD><TD>0.6261 g</TD></TR><TR><TD>Bacto agar</TD><TD>14 g/L</TD><TD>1.75 g</TD><TD>1.7512 g</TD></TR></TBODY></TABLE><UL><LI>Fill with dH<SUB>2</SUB>O to just under 250 mL</LI><UL><LI>This was our mistake!! We used enough chemicals to make 125 mL but put in too much water, making the solution useless</LI></UL><LI>Add 1M KOH dropwise until pH reached 7.63</LI><LI>Mixed in beaker using stir plate</LI><UL><LI>Didn‚Äôt mix fast enough so agar clumped on sides and at bottom</LI><LI>Shook a lot in a bottle and poured back and forth to get out as much agar as possible</LI></UL><LI>Autoclaved on liquid setting</LI><LI>Was about to pour plates but realized that we can‚Äôt use because too diluted</LI></UL><TABLE><TBODY><TR><TH>Substance</TH><TH>Concentration</TH><TH>Protocol Amount</TH><TH>Actual Amount</TH></TR><TR><TD>Bacto yeast extract</TD><TD>5 g/L</TD><TD>2.5 g</TD><TD>2.50 g</TD></TR><TR><TD>Bacco tryptone</TD><TD>20 g/L</TD><TD>10 g</TD><TD>10.0002 g</TD></TR><TR><TD>MgSO<SUB>4</SUB></TD><TD>5 g/L</TD><TD>2.5 g</TD><TD>2.5026 g</TD></TR></TBODY></TABLE><UL><LI>Fill with dH<SUB>2</SUB>O to just under 500 mL</LI><LI>Mixed in beaker using stir plate</LI><LI>Adjusted pH to 7.60 by adding 1M KOH dropwise</LI><LI>Topped off to 500 mL</LI><LI>Put through a sterile filter and aliquoted 100mL each into 3 flasks and the rest into a bottle</LI><LI>Autoclaved on liquid setting</LI><LI>Store in fridge (4¬∞C)</LI></UL><DIV class="notebookdate"><H4>June 7, 2018</H4><H4>Chloramphenicol plate prep</H4><H4>Shannon and Toni</H4></DIV><UL><LI>See protocol from 6/1 - made 1L</LI><LI>Mixture was goopy when transfering from bottle to graduated cylinder to Q/s to 500mL, so had to put back in bottle and shake/continue mixing on hot plate. Rinsed graduated cylinder with dH<SUB>2</SUB>O to get goop back into bottle (not enough to go past 500mL).</LI></UL></DIV><BUTTON class="collapsible">Week 5: 6/11/2018</BUTTON><DIV class="content"><DIV class="notebookdate"><H4>June 11, 2018</H4><H4>Interlab CFU Day 1</H4><H4>Sricharan, Liz, Shannon</H4></DIV><UL><LI>Circled three colonies (labeled a, b, c) from plates 4B 1ùúáL and 4D 1ùúáL (made 6/6/18)</LI><LI>Pipetted 5 mL LB broth into OC tubes</LI><UL><LI>Six tubes: 4Ba, 4Bb, 4Bc, 4Da, 4Db, 4Dc</LI><LI>5mL in each tube</LI></UL><LI>Used 200 1ùúáL pipet tip to take colony a from plate 4B and drop into tube 4Ba</LI><LI>Repeat with other circeled colonies into corresponding tubes</LI><LI>Put into shaker at about 37‚ÑÉ and 220 rpm overnight</LI><UL><LI>In: 3:30 pm</LI></UL></UL><DIV class="notebookdate"><H4>June 11, 2018</H4><H4>SS Competent Cells Trasformation Test Day 1</H4><H4> Peter, Deivydas, Ryan</H4></DIV><P>See protocol ‚ÄúCompetent Cell Transformation (Scott-Simanis, Test Kit)‚Äù from 5/29/18 transformation test</P><H4>Deviations/Notes</H4><UL><LI>Plates were taken out of fridge and warmed to room temperature</LI><UL><LI>No condensation on plates because they were allowed to mostly cool on the bench and then transferred to the hood with the lids partially off in the process of making the plates</LI></UL><LI>Œ®b taken out of fridge to warm to room temperature</LI><LI>Made one sample/ plate for each concentration (10 pg/ ŒºL, 50 pg/ ŒºL, 100 pg/ ŒºL) adn well 2F from interlan study as a control </LI><LI>Took one tube of competent cells from -80 degrees celsius</LI><LI>Let ice thaw for ~ one minute</LI><LI>Centrifuged during 30 min. To make sure is mixed (should not have done this- cells are fragile)</LI></UL><DIV class="notebookdate"><H4>June 12, 2018</H4><H4>SS Competent Cells Trasformation Test Day 2</H4><H4> Peter, Deivydas, Ryan</H4></DIV><UL><LI>Little to no growth - ask Rachel what we are doing wrong</LI><LI>Three colonies on 100 pg/ ŒºL</LI></UL><DIV class="notebookdate"><H4>June 12, 2018</H4><H4>Interlab CFU Day 2</H4><H4>Sricharan, Liz</H4></DIV><UL><LI>Took overnight cultures of positive (4B) and negative (4D) controls</LI><LI>Following CFU protocol</LI><UL><LI>C<SUB>1</SUB>V<SUB>1</SUB> = C<SUB>2</SUB>V<SUB>2</SUB>‚Üí V<SUB>1</SUB> = C<SUB>2</SUB>V<SUB>2</SUB>/C<SUB>1</SUB> = 0.1(1000mL)/8(1:8OD<SUB>600</SUB>-blank OD<SUB>600</SUB>)</LI></UL></UL><H4>Wells</H4><TABLE><TBODY><TR><TH>1</TH><TH>2</TH></TR><TR><TD>F</TD><TD>1 (positive control)</TD><TD>2 (positive control)</TD></TR><TR><TD>G</TD><TD>3 (negative control)</TD><TD>4 (negative control)</TD></TR><TR><TD>H</TD><TD>Blank media</TD><TD>Blank media</TD></TR></TBODY></TABLE><H4>OD<SUB>600</SUB></H4><TABLE><TBODY><TR><TH>1</TH><TH>2</TH></TR><TR><TD>F</TD><TD>0.237</TD><TD>0.122</TD></TR><TR><TD>G</TD><TD>0.250</TD><TD>0.208</TD></TR><TR><TD>H</TD><TD>0.033</TD><TD>0.034</TD></TR></TBODY></TABLE><H4>Culture Volume (checked OD<SUB>600</SUB> with plate reader)</H4><P>V<SUB>1</SUB>: 61ŒºL culture, 939ŒºL media
V<SUB>2</SUB>: 141ŒºL culture, 859ŒºL media
V<SUB>3</SUB>: 58ŒºL culture, 942ŒºL media
V<SUB>4</SUB>: 72ŒºL culture, 928ŒºL media</P><P>Shannon is making more LB+Cam media for our dilutions</P><H4>Plate notation: A.B.C</H4><P>A=culture number (1,2=positive control, 3,4=negative control)
  B=sample number
  C=dilution number (3, 4, and 5 are plated)</P><P>Diluted according to protocol, plated cells in dilutions 3-5</P><UL><LI>Reused 15mL Falcon tubes for dilutions 1-3</LI></UL><P>Incubate at 39‚ÑÉ downstairs overnight</P><UL><LI>In at ~4:15</LI></UL><DIV class="notebookdate"><H4>June 12, 2018</H4><H4>SS Competent Cells Trasformation Test Day 1</H4><H4>Peter</H4></DIV><H4>***New Protocol*** - we were doing stuff wrong</H4><UL><LI>Retrieved 3 vials of Scott-Simanis competent cells (XL1 Blue from 5/25/18, 6/8/18, and Rachel‚Äôs cells) from -80 and brought upstairs on ice (200 ŒºL in each tube) </LI><LI>Thawed at RT for ~ one minute</LI><LI>NO ALIQUOTING AS PIPETTING SHREDS FRAGILE CELLS</LI><LI>Added 1ŒºL of DNA immediately after thawed (BBa_J04450 100 pg/ŒºL from 2018 CC test kit)
</LI><LI>NO CENTRIFUGING TO MIX DNA IN </LI><LI>Chilled samples on ice for 30 minutes</LI><LI>Heat shocked samples WITH WATER BATH at 42 ¬∞C for 45 S</LI><LI>Incubated on ice for ~ 90 s</LI><LI>Added 800ŒºL (4 times volume of samples) 2YT media to each sample</LI><LI>Incubated at 37 ¬∞C ON HEAT BLOCK (NO SHAKING) for 60 min</LI><LI>Plated out w/ flame two plates for each sample: 100 ŒºL on plate 1 and 200 ŒºL on plate 2</LI><LI>Incubated overnight @ 37 ¬∞C</LI></UL><DIV class="notebookdate"><H4>June 13, 2018</H4><H4>SS Competent Cells Trasformation Test Day 2</H4><H4>Peter</H4></DIV><UL><LI>Retrieved from the incubator @ 9:15 (~ 17 hours in the incubator)</LI><LI>FINALLY IT WORKED MOSTLY!!!</LI></UL><H4>Colony Growth:</H4><TABLE><TBODY><TR><TH>Rachel (Date?)</TH><TH>iGEM 5/25/18</TH><TH>iGEM 6/8/18</TH></TR><TR><TD>Plate 1 (100 ŒºL)</TD><TD>~ 15 colonies</TD><TD>1 colony??</TD><TD>13 colonies</TD></TR><TR><TD>Plate 2 (200 ŒºL)</TD><TD>69 colonies</TD><TD>3 colonies</TD><TD>~42 colonies</TD></TR></TBODY></TABLE><DIV class="notebookdate"><H4>June 13, 2018</H4><H4>Counting Interlab Colonies</H4><H4>Liz, Sricharan, Shannon, Peter</H4></DIV><H4>Dilution 3</H4><TABLE><TBODY><TR><TH>Plate</TH><TH>Number of Colonies</TH></TR><TR><TD>1.1</TD><TD>185</TD></TR><TR><TD>1.2</TD><TD>28</TD></TR><TR><TD>1.3</TD><TD>79</TD></TR><TR><TD>2.1</TD><TD>Over 300</TD></TR><TR><TD>2.2</TD><TD>Over 300</TD></TR><TR><TD>2.3</TD><TD>Over 300</TD></TR><TR><TD>3.1</TD><TD>121</TD></TR><TR><TD>3.2</TD><TD>261</TD></TR><TR><TD>3.3</TD><TD>170</TD></TR><TR><TD>4.1</TD><TD>94</TD></TR><TR><TD>4.2</TD><TD>270</TD></TR><TR><TD>4.3</TD><TD>Over 300</TD></TR></TBODY></TABLE><H4>Dilution 4</H4><TABLE><TBODY><TR><TH>Plate</TH><TH>Number of Colonies</TH></TR><TR><TD>1.1</TD><TD>18</TD></TR><TR><TD>1.2</TD><TD>7</TD></TR><TR><TD>1.3</TD><TD>6</TD></TR><TR><TD>2.1</TD><TD>65</TD></TR><TR><TD>2.2</TD><TD>105</TD></TR><TR><TD>2.3</TD><TD>Over 76</TD></TR><TR><TD>3.1</TD><TD>42</TD></TR><TR><TD>3.2</TD><TD>26</TD></TR><TR><TD>3.3</TD><TD>12</TD></TR><TR><TD>4.1</TD><TD>31</TD></TR><TR><TD>4.2</TD><TD>24</TD></TR><TR><TD>4.3</TD><TD>33</TD></TR></TBODY></TABLE><H4>Dilution 5</H4><TABLE><TBODY><TR><TH>Plate</TH><TH>Number of Colonies</TH></TR><TR><TD>1.1</TD><TD>1</TD></TR><TR><TD>1.2</TD><TD>0</TD></TR><TR><TD>1.3</TD><TD>1</TD></TR><TR><TD>2.1</TD><TD>1</TD></TR><TR><TD>2.2</TD><TD>2</TD></TR><TR><TD>2.3</TD><TD>8</TD></TR><TR><TD>3.1</TD><TD>1</TD></TR><TR><TD>3.2</TD><TD>3</TD></TR><TR><TD>3.3</TD><TD>1</TD></TR><TR><TD>4.1</TD><TD>0</TD></TR><TR><TD>4.2</TD><TD>5</TD></TR><TR><TD>4.3</TD><TD>3</TD></TR></TBODY></TABLE></DIV><BUTTON class="collapsible">Week 6: 6/18/2018</BUTTON><DIV class="content"><DIV class="notebookdate"><H4>June 18, 2018</H4><H4>Transforming Parts from Registry</H4><H4>Sricharan</H4></DIV><P>Resuspended some parts from 2018 Distribution Kit to transform tomorrow</P><TABLE><TBODY><TR><TH>Plate</TH><TH>Well</TH><TH>Part</TH><TH>Name (BBa_)</TH></TR><TR><TD>4</TD><TD>17D</TD><TD>Anderson Promoter (Strong)</TD><TD>J23100</TD></TR><TR><TD>4</TD><TD>19D</TD><TD>Anderson Promoter (Medium)</TD><TD>J23110</TD></TR><TR><TD>4</TD><TD>19J</TD><TD>Anderson Promoter (Weak)</TD><TD>J23114</TD></TR><TR><TD>1</TD><TD>17H</TD><TD>Ptac</TD><TD>K864400</TD></TR><TR><TD>5</TD><TD>17J</TD><TD>‚ÄòtesA</TD><TD>K1472601</TD></TR><TR><TD>5</TD><TD>7K</TD><TD>TUDelft mtrCAB</TD><TD>K1316012</TD></TR><TR><TD>6</TD><TD>21K</TD><TD>ADC (Petrobrick)</TD><TD>K590031</TD></TR><TR><TD>6</TD><TD>7K</TD><TD>AAR (Petrobrick)</TD><TD>K590032</TD></TR></TBODY></TABLE><P>Resuspending each well w/ 10ŒºL of dH2O, let sit a few minutes, aliquoted into 1.5mL tubes.</P><DIV class="notebookdate"><H4>June 19, 2018</H4><H4>2xYT Medium</H4><H4>Liz</H4></DIV><P>Wanted to make 500mL for use for transformations/potentially overnight cultures (point is we don‚Äôt need a ton)

In a bottle measured: </P><UL><LI>450mL dH<SUB>2</SUB>O</LI><LI>8g tryptone (measured 8.00g)</LI><LI>5g yeast extract (measured 5.00)</LI><LI>2.5g NaCl (measured 2.50)</LI></UL><P>Shook bottle between adding each substance to dissolve

When dissolve adjusted pH to 7.0 with 5M NaOH</P><UL><LI>Added dH<SUB>2</SUB>O up to ~480mL before adjusting pH</LI><LI>Only had 1M NaOH</LI><LI>Had to use pH 4 calibration solution because start out acidic</LI></UL><P>Top off to 500mL</P><P>Autoclave for 20 min at 15 psi (1.05kg/cm<SUP>3</SUP>) on liquid cycle</P><UL><LI>Used 30 minute liquid setting</LI></UL><P>Put in cold room to cool before using for transformations</P><DIV class="notebookdate"><H4>June 19, 2018</H4><H4>Making CAM and Amp plates</H4><H4>Peter</H4></DIV><H4>***See protocol from 6/1/18***</H4><H4>Notes/Deviations</H4><UL><LI>After autoclaving, split 500 mL into two flasks of 250 mL each</LI><LI>Added 250 ŒºL of chloramphenicol (35 mg/ mL stock) to one flask</LI><LI>Added 250 ŒºL of Ampicillin (35 mg/ mL stock) to other flask</LI></UL><DIV class="notebookdate"><H4>June 19, 2018</H4><H4>SS Competent Cells Trasformation Test Day 1</H4><H4>Peter, Shannon, Charan</H4></DIV><P>Measured 0.5043g ampicillin (Na salt) from Rachel‚Äôs lab to make 100mg/mL stock solution (1000x) in 50% EtOH</P><H4>***See protocol from 6/12 (week 5)***</H4><H4>Notes/Deviations</H4><UL><LI>Retrieved 8 vials of second set of Scott-Simanis competent cells (XL1 Blue from 6/8/18) from -80 and brought upstairs on ice (200 ŒºL in each tube)</LI><LI>DNA added: BBa_J23100, BBa_J23110, BBa_J23114, BBa_K864400, BBa_K1472601, BBa_K1316012, BBa_590031, BBa_590032</LI><LI>Heat shocked four at a time- tried to fit all 8 in at once but four hands wouldn‚Äôt fit in the water bath to hold them all so 4 were in briefly then taken out and put back on ice until the the first four were finished and this could have caused problems</LI><LI>Used 2YT media (richer than Œ®b)</LI><LI>Did not originally add 800ŒºL media to tube with BBa_590031 tube and did not realize until it was on heat block for ~10 minutes and then added so this could have caused problems</LI><LI>Plated out 200 ŒºL of cells + media except for BBa_K1472601 and BBa_J23100 which we did two plates, one 200 ŒºL and one 300 ŒºL</LI><LI>Trouble keeping flame going (low on butane)</LI><LI>Put in 37 ¬∞C incubator at 4:20 p.m.</LI></UL><DIV class="notebookdate"><H4>June 20, 2018</H4><H4>SS Competent Cells Trasformation Test Day 2</H4><H4>Peter</H4></DIV><H4>Notes</H4><UL><LI>Observed growth on both the 200 ŒºL and 300 ŒºL plate for BBa_K1472601 and BBa_J23100, the BBa_J23110 plate, and the BBa_J23114 plate</LI><LI>~ 15 colonies (give or take 5) on average per plate</LI><LI>Will redo transformations of parts that were not successful (BBa_K864400, BBa_K1316012, BBa_590031, BBa_590032)</LI></UL><DIV class="notebookdate"><H4>June 20, 2018</H4><H4>SS Competent Cells Trasformation Test Day 1 Repeat</H4><H4>Liz, Toni</H4></DIV><P>Also resuspended BBa_B0015 from 2018, Plate 3, Well 3F (Cam resistance)</P><P>Retrieved 5 vials of SS competent cells from -80¬∞C downstairs on ice
</P><UL><LI>competent cells from 6/8/18</LI></UL><P>Let cells thaw at room temp for ~1 min</P><P>Added 1 ŒºL DNA (K1316012, K590032, K864400, K590031, BBa_B0015) to each tube
</P><UL><LI>Competent cell tube for K590031 was empty, had to go downstairs and get another tube so everything was done 6.5 minutes late</LI></UL><P>Chilled on ice for 30 minutes</P><UL><LI>K590031 started ice at 6:36 on stopwatch and went till 36:40</LI></UL><P>Heat shock in 42¬∞C water bath for 45s (tubes held in water bath)

Added 800ŒºL 2xYT media to each tube
</P><UL><LI>Media prewarmed in warm room</LI></UL><P>Incubated at 37¬∞C on heat block for 60 min</P><UL><LI>Heat block temporarily dipped to 35¬∞C then went back up to 37¬∞C</LI><LI>Flick tubes every 10 min</LI><LI>K1316012, K590032, K864488, BBa_B0015 in at 2:32, K590031 in at 2:35</LI></UL><P>Tubes removed from heat block and spun down at 6800g for 3 minutes
~800ŒºL supernatant removed and discarded and cells resuspended in remaining media
Cells plated and put in 39¬∞C overnight (in at ~4ish)
</P><UL><LI>Flame ran out of butane</LI></UL><DIV class="notebookdate"><H4>June 21, 2018</H4><H4>SS Competent Cells Trasformation Test Day 1</H4><H4>Peter, Ryan</H4></DIV><H4>***See protocol from 6/12 (week 5)***</H4><H4>Notes/Deviations:</H4><UL><LI>Retrieved 4 vials of second set of Scott-Simanis competent cells (XL1 Blue from 6/8/18) from -80 and brought upstairs on ice (200 ŒºL in each tube)</LI><LI>DNA added: BBa_K864400, BBa_K1316012, BBa_590031, BBa_590032</LI><LI>Used 2YT media (richer than Œ®b)</LI><LI>Plated out 200 ŒºL of cells per plate</LI><LI>Used beads in hood instead of flame w/ pasteur pipette</LI></UL><DIV class="notebookdate"><H4>June 22, 2018</H4><H4>Making Media for iGEM Competent Cells</H4><H4>Shannon and Ryan</H4></DIV><P>Made 1L SOB and 1L CCMB80 according to the iGEM protocol.</P><H4>SOB</H4><TABLE><TBODY><TR><TH>Reagent</TH><TH>Amount (g)</TH><TH>Measured Amount (g)</TH></TR><TR><TD>Yeast extract</TD><TD>5</TD><TD>5.0006</TD></TR><TR><TD>Tryptone</TD><TD>20</TD><TD>20.0009</TD></TR><TR><TD>NaCl</TD><TD>0.584</TD><TD>0.5846</TD></TR><TR><TD>KCl</TD><TD>0.186</TD><TD>0.1886</TD></TR><TR><TD>MgSO<SUB>4</SUB> * 6H<SUB>2</SUB>O</TD><TD>4.93</TD><TD>4.9333</TD></TR></TBODY></TABLE><H4>CCMB80</H4><TABLE><TBODY><TR><TH>Reagent</TH><TH>Amount (g)</TH><TH>Measured Amount (g)</TH></TR><TR><TD>KOAc</TD><TD>5mL of 1M stock (0.49g)</TD><TD>0.4938</TD></TR><TR><TD>CaCl<SUB>2</SUB> anhyd.</TD><TD>4.44</TD><TD>4.4473</TD></TR><TR><TD>MnCl<SUB>2</SUB> * 4H<SUB>2</SUB>O</TD><TD>2.0</TD><TD>2.0062</TD></TR><TR><TD>MgCl<SUB>2</SUB> * 6H<SUB>2</SUB>O</TD><TD>1.0</TD><TD>1.0081</TD></TR><TR><TD>Glycerol (100%)</TD><TD>5mL</TD><TD>5mL</TD></TR></TBODY></TABLE><UL><LI>SOB was pH adjusted to pH 7.54</LI><LI>100mL SOB was aliquoted and 1.6078g agar was added and autoclaved, and then used to make 5 plates. The agar did not dissolve when stirred, but did dissolve in the autoclave. These plates were left in the hood for the weekend.</LI><LI>At first, we tried to make only 500mL of CCMB80, but we used 1M HCl, which decreased the pH much too rapidly, bringing it to around pH 5.8 after 1 drop. So, we doubled the amount and used 0.1M HCl. The final pH was 6.35.</LI></UL><DIV class="notebookdate"><H4>June 22, 2018</H4><H4>Glycerol Stocks for Overnight Cultures</H4><H4>Liz</H4></DIV><P>Made 40% glycerol stock (25mL) in blue falcon tube</P><UL><LI>10mL 100% glycerol</LI><LI>15mL DI water</LI></UL><P>Store extra in freezer (-20)</P><P>Put 0.75mL 40% glycerol into 10 microcentrifuge tubes (2 tubes per overnight culture, labeled as such)</P><P>Put 750 microliters of one culture into two different microcentrifuge tubes</P><UL><LI>Repeat for each culture, should have 2 microcentrifuge tubes per culture</LI><LI>Pipet up and down in culture to mix cells before transferring</LI></UL><P>Vortexed tubes to mix</P><P>Stored in -80 where DH5alpha cells used to be in a blue box</P></DIV><BUTTON class="collapsible">Week 7: 6/25/2018</BUTTON><DIV class="content"><DIV class="notebookdate"><H4>June 25, 2018</H4><H4>Zyppy miniprep</H4><H4>Liz and Peter</H4></DIV><P>Centrifuge tubes labeled:

*need to look this up*
</P><P>Added 600 microliters culture to labeled 1.5 mL microcentrifuge tubes</P><P>Added 100 microliters 7x lysis buffer (blue) and mixed by inverting 4-6x</P><UL><LI>Turns opaque to clear blue when lysis is done</LI></UL><P>*time sensitive!!!* add 350 microliters neutralization buffer (yellow) to each tube</P><UL><LI>Neutralization buffer should be kept cold</LI><LI>Mixture turns yellow and forms yellow precipitate when neutralized</LI><LI>Invert 2-3x after color change</LI></UL><P>Centrifuged at 16,000rpm for 3 min then again at 11,000rpm for 1 minute</P><UL><LI>Pellets got disturbed and had to be repelleted)</LI></UL><P>Supernatant (~900 microliters) transferred into zymo-spin IIN column (don‚Äôt disturb cell pellet)</P><P>Put column into collection tube and centrifuge 15 s</P><P>Discarded flow-through and put column back into collection tube</P><P>Added 200 microliters Endo-wash buffer and centrifuged for 30s</P><P>Added 400 microliters zyppy wash buffer and centrifuged 1 min</P><P>Put column into clean 1.5mL tube and added 30mL zyppy elution buffer</P><UL><LI>Let sit 1 minute at room temp</LI></UL><P>Centrifuged 30s to elute plasmid DNA</P><DIV class="notebookdate"><H4>June 25, 2018</H4><H4>Primer Rehydration</H4><H4>Liz and Peter</H4></DIV><P>Spun down dry tubes</P><P>Added 1xTE buffer to make 100 micromolar solutions</P><TABLE><TBODY><TR><TH>primer</TH><TH>Amount TE added (microliters)</TH></TR><TR><TD>SB prep-3P-1</TD><TD>201</TD></TR><TR><TD>SB prep-2Ea</TD><TD>221</TD></TR><TR><TD>fadD_P1F</TD><TD>201</TD></TR><TR><TD>fadD_P1R</TD><TD>231</TD></TR></TBODY></TABLE><P>Vortexed</P><P>Stored in -20 degree freezer in primer box</P><DIV class="notebookdate"><H4>June 25, 2018</H4><H4>SOB plates</H4><H4>Ryan</H4></DIV><UL><LI>SOB plates failed/ dried up because they were left in the hood over the weekend (agar was thin and had a wavy texture, we felt it was better to make new plates).</LI><LI>100mL SOB was aliquoted and 1.6 g agar was added</LI><LI>The solution was heated and mixed, and then autoclaved for an hour (set at liquids 30, P03)</LI><LI>The agar did not completely dissolve when stirred, but did dissolve in the autoclave.</LI><LI>This was used to make 5 plates, which were of proper thickness</LI></UL><DIV class="notebookdate"><H4>June 26, 2018</H4><H4>Making new competent JM109 cells</H4><H4>Liz and Peter</H4></DIV><UL><LI>Streaking a SOB plate w/ Rachel‚Äôs JM109 cells (SOB plates specific to Top10 cells‚Ä¶)</LI><LI>Put in incubator at 1:45 p.m.</LI><LI>Results (6/27): good growth on plate</LI></UL><DIV class="notebookdate"><H4>June 26, 2018</H4><H4>PCR of fadD</H4><H4>Liz, Peter, and Shannon</H4></DIV><H4>Thermocycler Settings</H4><TABLE><TBODY><TR><TH>Cycle</TH><TH>Time</TH><TH>Temp. (C)</TH></TR><TR><TD>Initial denaturation</TD><TD>30s</TD><TD>98</TD></TR><TR><TD>35 Cycles</TD><TD>10s</TD><TD>98</TD></TR><TR><TD>30s</TD><TD>61</TD></TR><TR><TD>20s for fadD (should have been 40 s because 2kb)
50s for backbone</TD><TD>72</TD></TR><TR><TD>Final extension</TD><TD>10 min</TD><TD>72</TD></TR><TR><TD>Hold</TD><TD>indefinitely</TD><TD>4</TD></TR></TBODY></TABLE><H4>Notes</H4><UL><LI>For 61C during the 35 cycles, the time was found using NEBTm calculator</LI><LI>fadD was accidently put at 20s instead of 40s at 72C during the 35 cycles</LI><LI>We aren‚Äôt sure if fadD was held at 4C because it didn‚Äôt feel cold when opened</LI></UL><H4>Add to PCR tube (for fadD):</H4><TABLE><TBODY><TR><TH>Component</TH><TH>Amount (¬µL)</TH></TR><TR><TD>Nuclease-free water</TD><TD>12.2 (20 - sum of other components)</TD></TR><TR><TD>Phusion buffer (HF)</TD><TD>4</TD></TR><TR><TD>10mM dNTPs</TD><TD>0.4</TD></TR><TR><TD>10mM forward primer</TD><TD>1</TD></TR><TR><TD>10mM reverse primer</TD><TD>1</TD></TR><TR><TD>Template (E. coli) DNA</TD><TD>0.56</TD></TR><TR><TD>Phusion polymerase (add last)</TD><TD>0.2</TD></TR><TR><TD>DMSO</TD><TD>0.6</TD></TR></TBODY></TABLE><H4>Notes</H4><UL><LI>Made a 10mM stock solution for dNTPs (1¬µL of dATP, dTTP, dCTP, dGTP)</LI><LI>Diluted the forward and reverse primer to 10¬µM 100¬µM(V) = 10¬µM(1¬µL)
    2x[0.1¬µL stock, 0.9¬µL H2O]
  </LI><LI>For template DNA, needed 40ng, stock was 40ng
    40ng x 1¬µL/71ng = 0.56¬µL
  </LI></UL><P>Procedure repeated for backbone</P><UL><LI>4 backbones (PSB1A3, PSB1K3, PSB1T3, PSB1C3), put into 4 labeled PCR tubes</LI><LI>For primers use SB Prep-3P-1 and SBPrep - 2Ea</LI><LI>Annealing temp is 69C</LI><LI>Template DNA: 40ng x 1¬µL/25ng = 1.6 ¬µL = 1.6 ¬µL backbone DNA</LI></UL><H4>Add to each tube:</H4><TABLE><TBODY><TR><TH>Component</TH><TH>Amount (¬µL)</TH></TR><TR><TD>Nuclease free water</TD><TD>11.2</TD></TR><TR><TD>Backbone DNA</TD><TD>1.6</TD></TR><TR><TD>Phusion buffer (HF)</TD><TD>4</TD></TR><TR><TD>dNTPs</TD><TD>0.4</TD></TR><TR><TD>10mM forward primer</TD><TD>1</TD></TR><TR><TD>10mM reverse primer</TD><TD>1</TD></TR><TR><TD>DMSO</TD><TD>0.6</TD></TR><TR><TD>Phusion polymerase (add last)</TD><TD>0.2</TD></TR></TBODY></TABLE><P>Note: thermocycler did not feel cold when taking out, don‚Äôt know if this actually worked</P><DIV class="notebookdate"><H4>June 27, 2018</H4><H4>PCR Purification</H4><H4>Liz and Shannon</H4></DIV><P>Used protocol from PureLink PCR Purification Kit</P><UL><LI>Pipet 20¬µL PCR sample (PSB1T3, PSB1K3, PSB1C3, PSB1A3) into individual 1.5mL tubes</LI><LI>Add 80¬µL Binding Buffer 2 (B2) to each sample (4x volume) - pipette to mix</LI><UL><LI>B2 is used for purifying PCR fragments</LI></UL><LI>Load each solution into individual spin column collection tubes. Centrifuge at 10,000g for 1 min. Discard flow through and re-insert the column into the collection tube.</LI><LI>Add 650 ¬µL wash buffer (W1) (ethanol had already been added). Centrifuge at 10,000g for 1 min. Discard flow-through and re-insert the column into the collection tube.</LI><LI>Re-centrifuge at max speed (16,100) for 3 min.</LI><LI>Using a clean collection tube, add 50¬µL Elution Buffer (E1) to column. Let sit for 1 min, then centrifuge at max speed for 2 min.</LI><UL><LI>The purified PCR fragments are in the collection tube.</LI></UL></UL><P>Products were nanodropped:</P><UL><LI>PSB1T3: 20.1ng/¬µL</LI><LI>PSB1K3: 7.8ng/¬µL</LI><LI>PSB1C3: 21.1ng/¬µL</LI><LI>PSB1A3: 23.3ng/¬µL</LI></UL><DIV class="notebookdate"><H4>June 27, 2018</H4><H4>JM109 Overnight Culture</H4><H4>Shannon</H4></DIV><UL><LI>SOB was not coming to room temp. Fast enough, so it was put in the warm room for about 10 min.</LI><UL><LI>Used at about 21.1C</LI></UL><LI>Put 5mL SOB into 2 round-bottom tubes and used a non-filtered pipette tip to inoculate the tubes with 2 colonies</LI><LI>Put in downstairs shaker overnight</LI><LI>Note: forgot to do this in the hood!</LI></UL><DIV class="notebookdate"><H4>June 27, 2018</H4><H4>PCR of phz from Genomic DNA (1F-1R, 2F-3R, 4F-4R, 6F-6R)</H4><H4>Peter and Liz</H4></DIV><H4>Thermocycler Settings:</H4><TABLE><TBODY><TR><TH>Cycle</TH><TH>Time</TH><TH>Temp. (C)</TH></TR><TR><TD>Initial denaturation</TD><TD>30s</TD><TD>98</TD></TR><TR><TD>35 Cycles</TD><TD>10s</TD><TD>98</TD></TR><TR><TD>30s</TD><TD>64.5</TD></TR><TR><TD>2F-3R: 54 s
1F-1R, 4F-4R, 6F-6R: 25s</TD><TD>72</TD></TR><TR><TD>Final extension</TD><TD>10 min</TD><TD>72</TD></TR><TR><TD>Hold</TD><TD>Indefinitely</TD><TD>4</TD></TR></TBODY></TABLE><P>Notes:</P><UL><LI>1F-1R: 0.5 kb</LI><LI>2F-3R: 1.8 kb</LI><LI>4F-4R: 0.9 kb</LI><LI>6F-6R: 1 kb</LI></UL><H4>Add to PCR tubes:</H4><TABLE><TBODY><TR><TH>Component</TH><TH>Amount (¬µL)</TH></TR><TR><TD>Nuclease-free water</TD><TD>11.97  (20 - sum of other components)</TD></TR><TR><TD>Phusion buffer (HF)</TD><TD>4</TD></TR><TR><TD>10mM dNTPs</TD><TD>0.4</TD></TR><TR><TD>10mM forward primer</TD><TD>1</TD></TR><TR><TD>10mM reverse primer</TD><TD>1</TD></TR><TR><TD>Template Genomic DNA (Pseudomonas aeruginosa)</TD><TD>0.83 (40 ng - stock is 48.3 ng/ ¬µL)</TD></TR><TR><TD>Phusion polymerase (add last)</TD><TD>0.2</TD></TR><TR><TD>DMSO</TD><TD>0.6</TD></TR></TBODY></TABLE><P>Note: Started machine 2 on a 50 ¬µL reaction for ~3 seconds and then stopped it and within 10 seconds started again for a 20 ¬µL reaction</P><DIV class="notebookdate"><H4>June 27, 2018</H4><H4>PCR purification and gel of backbones and phz</H4><H4>Liz</H4></DIV><P>Made a gel for PCR products</P><UL><LI>25mL 1x TAE buffer and 0.25g agarose in flask and microwaved until dissolved</LI><LI>Let flask cool until it could be picked up</LI><LI>Added 2.5¬µL SYBR Safe</LI><LI>Poured into mold, put in comb, and let harden under ice bin to protect SYBR Safe from light</LI></UL><P>Purified PCR products (PureLink PCR Purification Kit)</P><UL><LI>Pipet 20¬µL PCR sample (1F/1R, 2F/3R, 4F/4R, 6F/6R) into individual 1.5mL tubes</LI><LI>Add 80¬µL Binding Buffer 2 (B2) to each sample (4x volume) - pipette to mix</LI><UL><LI>B2 is used for purifying PCR fragments</LI></UL><LI>Load each solution into individual spin column collection tubes</LI><LI>Centrifuge at 11,000g for 1 min and discard flow through</LI><LI>Add 650 ¬µL wash buffer (W1) (ethanol had already been added)</LI><LI>Centrifuge at 11,000g for 1 min and discard flow-through</LI><LI>Centrifuge at max speed (16,100) for 3 min.</LI><LI>Add 50¬µL Elution Buffer (E1) to column and let sit for 1 min</LI><LI>Centrifuge at max speed (16,100) for 2 min.</LI><UL><LI>The purified PCR fragments are in the collection tube</LI></UL></UL><P>Nanodrop (blanked with elution buffer)</P><UL><LI>1F/1R: 5.5 ng/¬µL</LI><LI>2F/3R: 11.5 ng/¬µL</LI><LI>4F/4R: 6.4 ng/¬µL</LI><LI>6F/6R: 6.3 ng/¬µL</LI><LI>Readings were very inconsistent (can we trust the nanodrop?)</LI></UL><P>Gel electrophoresis</P><UL><LI>For each sample mixed 4¬µL water, 1¬µL loading dye, and 1¬µL PCR product</LI><LI>Filled rig with 1x TAE buffer then loaded</LI></UL><TABLE><TBODY><TR><TH>1</TH><TH>2</TH><TH>3</TH><TH>4</TH><TH>5</TH><TH>6</TH><TH>7</TH><TH>8</TH><TH>9</TH><TH>10</TH></TR><TR><TD>Ladder (2-log)</TD><TD>pSB1A3</TD><TD>pSB1C3</TD><TD>pSB1K3</TD><TD>pSB1T3</TD><TD>1F/1R</TD><TD>2F/3R</TD><TD>4F/4R</TD><TD>6F/6R</TD><TD>Ladder (2-log)</TD></TR></TBODY></TABLE><UL><LI>Filled rig with 1x TAE buffer then loaded</LI></UL><TABLE><TBODY><TR><TH>1</TH><TH>2</TH><TH>3</TH><TH>4</TH><TH>5</TH><TH>6</TH><TH>7</TH><TH>8</TH><TH>9</TH><TH>10</TH></TR><TR><TD>Ladder (2-log)</TD><TD>pSB1A3</TD><TD>pSB1C3</TD><TD>pSB1K3</TD><TD>pSB1T3</TD><TD>1F/1R</TD><TD>2F/3R</TD><TD>4F/4R</TD><TD>6F/6R</TD><TD>Ladder (2-log)</TD></TR></TBODY></TABLE><P>Gel did not come out</P><DIV class="notebookdate"><H4>June 28, 2018</H4><H4>PCR troubleshooting</H4><H4>Liz and Peter</H4></DIV><P>Redid PCR protocol for fadD and phz 2F/3R but used new dNTPs</P><TABLE><TBODY><TR><TH>Cycle</TH><TH>Time</TH><TH>Temp. (C)</TH></TR><TR><TD>fadD</TD><TD>phz 2F/3R</TD><TD>fadD</TD><TD>phz 2F/3R</TD></TR><TR><TD>30s</TD><TD>30s</TD><TD>98</TD><TD>98</TD></TR><TR><TD>10s</TD><TD>10s</TD><TD>98</TD><TD>98</TD></TR><TR><TD>40s</TD><TD>54s</TD><TD>72</TD><TD>72</TD></TR><TR><TD>10 min</TD><TD>10 min</TD><TD>72</TD><TD>72</TD></TR><TR><TD>infinite</TD><TD>infinite</TD><TD>4</TD><TD>4</TD></TR></TBODY></TABLE><P>Annealing temperatures from NEB annealing temperature calculator</P><P>Set to 20¬µL reaction</P><P>Labeled PCR tubes fadD and phz 2F/3R</P><TABLE><TBODY><TR><TH>Component</TH><TH>Amount (¬µL)</TH></TR><TR><TD>fadD</TD><TD>phz 2F/3R</TD></TR><TR><TD>Nuclease free water</TD><TD>12.2</TD><TD>11.97</TD></TR><TR><TD>Template DNA (40ng)</TD><TD>0.56*</TD><TD>0.83*</TD></TR><TR><TD>Phusion buffer (HF)</TD><TD>4</TD><TD>4</TD></TR><TR><TD>dNTP stock</TD><TD>0.4</TD><TD>0.4</TD></TR><TR><TD>10 mM forward primer</TD><TD>1</TD><TD>1</TD></TR><TR><TD>10mM reverse primer</TD><TD>1</TD><TD>1</TD></TR><TR><TD>DMSO</TD><TD>0.6</TD><TD>0.6</TD></TR><TR><TD>Phusion polymerase (add last)</TD><TD>0.2</TD><TD>0.2</TD></TR></TBODY></TABLE><H4>Calculations</H4><UL><LI>fadD: 40ng x 1¬µL/71ng = 0.56¬µL</LI><LI>phz 2F/3R: 40ng x 1¬µL/48.3ng = 0.83¬µL</LI></UL><P>New dNTP stock diluted (0.2¬µL each dNTP and 7.2¬µL dH<SUB>2</SUB>O) because old one had been thawed/refrozen too many times</P><P>Made a gel for PCR products</P><UL><LI>12.5mL 1x TAE buffer and ~0.125g agarose in flask and microwaved until dissolved</LI><UL><LI>Can‚Äôt tell if gel is 1% because scale measures to nearest 0.05g</LI></UL><LI>Let flask cool until it could be picked up</LI><LI>Added 1.75¬µL SYBR Safe</LI><LI>Poured into mold, put in comb, and let harden under ice bin to protect SYBR Safe from light</LI></UL><DIV class="notebookdate"><H4>June 28, 2018</H4><H4>PCR troubleshooting purification/gel</H4><H4>Liz and Peter</H4></DIV><P>Purified PCR products (PureLink PCR Purification Kit)</P><UL><LI>Pipet 20¬µL PCR sample (fadD and phz 2F/3R) into individual 1.5mL tubes</LI><LI>Add 80¬µL Binding Buffer 2 (B2) to each sample (4x volume) - pipette to mix</LI><UL><LI>B2 is used for purifying PCR fragments</LI></UL><LI>Load each solution into individual spin column collection tubes</LI><LI>Centrifuge at 11,000g for 1 min and discard flow through</LI><LI>Add 650 ¬µL wash buffer (W1) (ethanol had already been added)</LI><LI>Centrifuge at 11,000g for 1 min and discard flow-through</LI><LI>Centrifuge at max speed (16,100) for 3 min</LI><LI>Add 50¬µL Elution Buffer (E1) to column and let sit for 1 min</LI><LI>Centrifuge at max speed (16,100) for 2 min</LI><UL><LI>The purified PCR fragments are in the collection tube</LI></UL></UL><P>Nanodrop (blanked with elution buffer)</P><UL><LI>fadD: 11.2 ng/¬µL</LI><LI>phz 2F/3R: 11.3 ng/¬µL</LI></UL><P>Gel electrophoresis</P><UL><LI>For each sample mixed 4¬µL water, 1¬µL loading dye, and 1¬µL PCR product</LI><LI>Topped off rig with 1x TAE buffer then loaded</LI><LI>Ran at 50V for 50 min</LI></UL><TABLE><TBODY><TR><TH>1</TH><TH>2</TH><TH>3</TH><TH>4</TH><TH>5</TH></TR><TR><TD>Ladder (2-log)</TD><TD>empty</TD><TD>phz 2F/3R</TD><TD>empty</TD><TD>fadD</TD></TR></TBODY></TABLE><DIV class="notebookdate"><H4>June 29, 2018</H4><H4>Backbone and phz PCR</H4><H4>Liz and Peter</H4></DIV><P>Parts to PCR (1 PCR tube per sample):</P><UL><LI>phz 1F/1R (510bp)</LI><LI>phz 4F/4R (946bp)</LI><LI>phz 6F/6R (997bp)</LI><LI>pSB1A3 (2155bp)</LI><LI>pSB1C3 (2070bp)</LI><LI>pSB1K3 (2204bp)</LI><LI>pSB1T3 (2461bp)</LI></UL><H4>Set thermocycler:</H4><TABLE><TBODY><TR><TH>Cycle</TH><TH>Time</TH><TH>Temp. (C)</TH></TR><TR><TD>phz</TD><TD>backbones</TD><TD>phz</TD><TD>backbones</TD></TR><TR><TD>30s</TD><TD>30s</TD><TD>98</TD><TD>98</TD></TR><TR><TD>10s</TD><TD>10s</TD><TD>98</TD><TD>98</TD></TR><TR><TD>30s</TD><TD>30s</TD><TD>64.5</TD><TD>69</TD></TR><TR><TD>25s</TD><TD>50s</TD><TD>72</TD><TD>72</TD></TR><TR><TD>10 min</TD><TD>10 min</TD><TD>72</TD><TD>72</TD></TR><TR><TD>infinite</TD><TD>infinite</TD><TD>4</TD><TD>4</TD></TR></TBODY></TABLE><H4>Added to each tube:</H4><TABLE><TBODY><TR><TH>Component</TH><TH>Amount (¬µL)</TH></TR><TR><TD>phz</TD><TD>backbones</TD></TR><TR><TD>Nuclease free water</TD><TD>11.97</TD><TD>11.2</TD></TR><TR><TD>Template DNA (40ng)</TD><TD>0.83*</TD><TD>1.6*</TD></TR><TR><TD>Phusion buffer (HF)</TD><TD>4</TD><TD>4</TD></TR><TR><TD>dNTP stock</TD><TD>0.4</TD><TD>0.4</TD></TR><TR><TD>10 mM forward primer</TD><TD>1</TD><TD>1</TD></TR><TR><TD>10mM reverse primer</TD><TD>1</TD><TD>1</TD></TR><TR><TD>DMSO</TD><TD>0.6</TD><TD>0.6</TD></TR><TR><TD>Phusion polymerase (add last)</TD><TD>0.2</TD><TD>0.2</TD></TR></TBODY></TABLE><H4>Calculations</H4><UL><LI>backbones: 40ng x 1¬µL/25ng = 0.56¬µL</LI><LI>phz: 40ng x 1¬µL/48.3ng = 0.83¬µL</LI></UL><P>Tubes kept on ice and vortexed before putting into thermocycler</P><P>Gel electrophoresis</P><UL><LI>For each sample mixed 4¬µL water, 1¬µL loading dye, and 1¬µL PCR product</LI><LI>Filled rig with 1x TAE buffer then loaded</LI></UL><TABLE><TBODY><TR><TH>1</TH><TH>2</TH><TH>3</TH><TH>4</TH><TH>5</TH><TH>6</TH><TH>7</TH><TH>8</TH><TH>9</TH><TH>10</TH></TR><TR><TD>Ladder (2-log)</TD><TD>phz 1F/1R</TD><TD>phz 4F/4R</TD><TD>phz 6F/6R</TD><TD>Ladder (2-log)</TD><TD>pSB1A3</TD><TD>pSB1C3</TD><TD>pSB1K3</TD><TD>pSB1T3</TD><TD>Ladder (2-log)</TD></TR></TBODY></TABLE><P>Gel didn‚Äôt get imaged correctly first time so redid gel on 7/2/2018:</P><TABLE><TBODY><TR><TH>1</TH><TH>2</TH><TH>3</TH><TH>4</TH><TH>5</TH><TH>6</TH><TH>7</TH><TH>8</TH><TH>9</TH></TR><TR><TD>Ladder (2-log)</TD><TD>pSB1A3</TD><TD>pSB1C3</TD><TD>pSB1K3</TD><TD>pSB1T3</TD><TD>phz 4F/4R</TD><TD>phz 6F/6R</TD><TD>phz 1F/1R</TD><TD>Ladder (2-log)</TD></TR></TBODY></TABLE><P>Ran at 50V for 30+20 min (time in between because didn‚Äôt know current stops when timer does)</P></DIV><H3>July</H3><BUTTON class="collapsible">Week 8: 7/2/2018</BUTTON><DIV class="content"><DIV class="notebookdate"><H4>July 2, 2018</H4><H4>iGEM Competent Cell Prep, Day 1</H4><H4>Liz</H4></DIV><P>Going to try with 1 tube JM109 cells and 1 tube NEB Gibson cells</P><UL><LI>Need to use all detergent-free glassware and rinsed plasticware</LI><UL><LI>Autoclaved glassware filled ¬æ of way with dH<SUB>2</SUB>O</LI></UL><LI>Put 250mL SOB media into each of two 500mL flasks (labeled JM109 and NEB cells)</LI><UL><LI>Measured/poured in hood because no antibiotics in SOB</LI><LI>Put media into warm room about 20 minutes before pouring, not sure if this is necessary</LI></UL><LI>Put 1 tube thawed JM109 cells into JM109 flask and 1 tube thawed NEB Gibson cells into NEB flask and covered with foil</LI><UL><LI>Thawed tubes in hand</LI><LI>Pipetted cells into media (not competent yet so shouldn‚Äôt kill cells)</LI></UL><LI>Put ice in bottom of ice bucket and left flasks of cells in ice bucket overnight</LI><UL><LI>Want OD<SUB>600</SUB> of 0.3 (should be about 16 hrs at 20 degrees C)</LI><LI>Start at 5:00pm</LI></UL></UL><DIV class="notebookdate"><H4>July 3, 2018</H4><H4>iGEM Competent Cell Prep, Day 1</H4><H4>Liz, Shannon</H4></DIV><P>Pre-chilled 2 flat-bottomed centrifuge bottles and preset centrifuge for 4 degrees C</P><P>Checked OD<SUB>600</SUB> of overnights (want at 0.3):</P><TABLE><TBODY><TR><TH>Time</TH><TH>7:38</TH><TH>8:03</TH><TH>8:39</TH><TH>9:15</TH><TH>9:56</TH><TH>10:30</TH><TH>11:18</TH><TH>11:52</TH><TH>12:34</TH></TR><TR><TD>JM109</TD><TD>0.005</TD><TD>0.006</TD><TD>0.017</TD><TD>0.015</TD><TD>0.026</TD><TD>0.054</TD><TD>0.113</TD><TD>0.179</TD><TD>0.294</TD></TR><TR><TD>NEB cells</TD><TD>0.015</TD><TD>0.016</TD><TD>0.022</TD><TD>0.020</TD><TD>0.024</TD><TD>0.028</TD><TD>0.039</TD><TD>0.055</TD><TD>0.081</TD></TR></TBODY></TABLE><UL><LI>After first measurement, kept in warm room</LI><LI>Gave up on NEB cells (not growing)</LI></UL><UL><LI>Put equal amounts JM109 culture into two flat-bottomed centrifuge bottles</LI><LI>Centrifuge for 3000g at 4 degrees C for 10 min</LI><LI>Decant supernatent</LI><LI>Resuspend in 80mL ice cold CCMB80 buffer</LI><UL><LI>40mL each tube</LI><LI>Resuspend in 20mL then pipet other 20mL against walls</LI></UL><LI>Incubate on ice 20 min</LI><LI>Centrifuge at 3000g at 4 degrees C for 10 min</LI><LI>Decant supernatent</LI><LI>Resuspend pellet in 10mL ice cold CCMB80 (10mL is total, put 5mL in each bottle then combined bottles)</LI><LI>Was supposed to check OD of 800ŒºL SOC and 200ŒºL culture using spectrophotometer (blank with 800ŒºL SOC and 200ŒºLCCMB80) then adjust to 1.0-1.5 using CCMB80 but OD started at 0.532 so just skipped this step</LI><LI>Incubated on ice (was supposed to be 20 minutes but ended up being much longer because was trying to figure out the protocol)</LI><LI>Aliquotted 50ŒºL per tube over liquid nitrogen (suspended over top to be cold but not too cold like flash freezing)</LI><UL><LI>Made two batches because liquid nitrogen evaporated</LI></UL><LI>Stored in -80 freezer</LI></UL><DIV class="notebookdate"><H4>July 3, 2018</H4><H4>fadK PCR</H4><H4>Liz</H4></DIV><P>Resuspended and diluted primers</P><UL><LI>1F: added 471ŒºL to resuspend, to dilute to 10mM added 1ŒºL resuspended primer and 9ŒºL milliQ water</LI><LI>1R: added 339ŒºL to resuspend, to dilute to 10mM added 1ŒºL resuspended primer and 9ŒºL milliQ water</LI></UL><H4>Thermocycler Settings</H4><TABLE><TBODY><TR><TH>Cycle</TH><TH>Time</TH><TH>Temperature (degrees Celsius)</TH></TR><TR><TD>Initial denaturation</TD><TD>30s</TD><TD>98</TD></TR><TR><TD>35 cycles</TD><TD>10s</TD><TD>98</TD></TR><TR><TD>30s</TD><TD>63</TD></TR><TR><TD>40s</TD><TD>72</TD></TR><TR><TD>Final extension</TD><TD>10 min</TD><TD>72</TD></TR><TR><TD>Hold</TD><TD>Infinite</TD><TD>4</TD></TR></TBODY></TABLE><H4>Added to PCR tube:</H4><TABLE><TBODY><TR><TH>Component</TH><TH>Amount (¬µL)</TH></TR><TR><TD>Nuclease-free water</TD><TD>12.2 (20 - sum of other components)</TD></TR><TR><TD>Phusion buffer (HF)</TD><TD>4</TD></TR><TR><TD>DMSO</TD><TD>0.6</TD></TR><TR><TD>10mM forward primer</TD><TD>1</TD></TR><TR><TD>10mM reverse primer</TD><TD>1</TD></TR><TR><TD>dNTPs</TD><TD>0.4</TD></TR><TR><TD>Template (E. coli) DNA</TD><TD>0.56</TD></TR><TR><TD>Phusion polymerase (add last)</TD><TD>0.2</TD></TR></TBODY></TABLE><DIV class="notebookdate"><H4>July 5, 2018</H4><H4>fadK PCR purification and Nanodrops</H4><H4>Liz, Sricharan</H4></DIV><P>Used protocol from 6/28 (1 PCR tube for fadK)</P><UL><LI>Ran 10 Nanodrops (blanked with elution buffer (Buffer E1) from Invitrogen Purif Kit) of PCR purif (4 phz parts, fadD, fadK, 4 linear plasmid backbones)</LI><LI>All ~10-15 ng/¬µL, good characteristic curves</LI></UL><TABLE><TBODY><TR><TH>Sample Name</TH><TH>Conc. (ng/¬µL)</TH><TH>260/280</TH><TH>260/230</TH></TR><TR><TD>phz1F-1R Purif</TD><TD>13.31</TD><TD>2.04</TD><TD>1.75</TD></TR><TR><TD>phz2F-3R Purif</TD><TD>11.83</TD><TD>1.89</TD><TD>1.53</TD></TR><TR><TD>phz4F-4R Purif</TD><TD>15.53</TD><TD>1.97</TD><TD>1.77</TD></TR><TR><TD>phz 6F-6R Purif</TD><TD>10.65</TD><TD>2.13</TD><TD>1.46</TD></TR><TR><TD>fadD Purif</TD><TD>8.03</TD><TD>1.79</TD><TD>2.49</TD></TR><TR><TD>fadK Purif</TD><TD>14.49</TD><TD>1.69</TD><TD>1.84</TD></TR><TR><TD>pSB1C3 Purif</TD><TD>15.01</TD><TD>2.01</TD><TD>1.72</TD></TR><TR><TD>pSB1T3 Purif</TD><TD>15.56</TD><TD>1.91</TD><TD>1.70</TD></TR><TR><TD>pSB1K3 Purif</TD><TD>12.88</TD><TD>1.94</TD><TD>1.99</TD></TR><TR><TD>pSB1A3 Purif</TD><TD>10.55</TD><TD>2.11</TD><TD>1.78</TD></TR></TBODY></TABLE><DIV class="notebookdate"><H4>July 5, 2018</H4><H4>resuspension of phenazine gBlocks</H4><H4>Liz</H4></DIV><P>Needed to resuspend phz4-5 and start of phz for gibson assembly</P><P>Centrifuged for 5s to make sure all of product is in bottom of tube</P><P>Added 1x TE buffer to get concentration to 10ng/ŒºL</P><UL><LI>Phz4-5 (1000ng)
1000ng x 1ŒºL/10ng = 100ŒºL</LI><LI>Start of phz (500ng)
500ng x 1ŒºL/10ng = 50ŒºL</LI></UL><P>Vortexed</P><P>Incubated in hat block at 50 degrees C for 20 min</P><P>Vortexed and centrifuged</P><DIV class="notebookdate"><H4>July 5, 2018</H4><H4>phz gibson PCR and fadD/fadK secondary PCR</H4><H4>Liz, Sricharan</H4></DIV><H4>Thermocycler Settings</H4><TABLE><TBODY><TR><TH>Phz gibson</TH><TH>fad</TH></TR><TR><TD>Time</TD><TD>Temperature (‚ÑÉ)</TD><TD>Time</TD><TD>Temperature (‚ÑÉ)</TD></TR><TR><TD>Initial denaturation</TD><TD>30s</TD><TD>98</TD><TD>30s</TD><TD>98</TD></TR><TR><TD>35 cycles</TD><TD>10s</TD><TD>98</TD><TD>10s</TD><TD>98</TD></TR><TR><TD>30s</TD><TD>61</TD><TD>30s</TD><TD>72</TD></TR><TR><TD>60s</TD><TD>72</TD><TD>40s</TD><TD>72</TD></TR><TR><TD>Final extension</TD><TD>10 min</TD><TD>72</TD><TD>10 min</TD><TD>72</TD></TR><TR><TD>Hold</TD><TD>infinite</TD><TD>4</TD><TD>infinite</TD><TD>4</TD></TR></TBODY></TABLE><H4>Add to PCR tubes:</H4><TABLE><TBODY><TR><TH>Component</TH><TH>Amount (¬µL)</TH></TR><TR><TD>Phz gibson</TD><TD>fadD</TD><TD>fadK</TD></TR><TR><TD>Nuclease-free water</TD><TD>11.3 (20 - sum of other components)</TD><TD>7.8 (20 - sum of other components)</TD><TD>10.04 (20 - sum of other components)</TD></TR><TR><TD>Phusion buffer (HF)</TD><TD>4</TD><TD>4</TD><TD>4</TD></TR><TR><TD>DMSO</TD><TD>0.6</TD><TD>0.6</TD><TD>0.6</TD></TR><TR><TD>10mM forward primer</TD><TD>1 (1F)</TD><TD>1 (fadD bad start)</TD><TD>1 (fadD bad start)</TD></TR><TR><TD>10mM reverse primer</TD><TD>1 (1R)</TD><TD>1 (fadD bad end)</TD><TD>1 (fadD bad end)</TD></TR><TR><TD>dNTPs</TD><TD>0.4</TD><TD>0.4</TD><TD>0.4</TD></TR><TR><TD>Template (E. coli) DNA</TD><TD>1.5</TD><TD>5.0</TD><TD>2.76</TD></TR><TR><TD>Phusion polymerase (add last)</TD><TD>0.2</TD><TD>0.2</TD><TD>0.2</TD></TR></TBODY></TABLE><P>Gibson Assembly using the NEB kit ‚Ä¶ wanted 0.2pmol of DNA total (kit specifies 0.2-1pmol for 4-6 pieces)</P><UL><LI>Start of phz (gBlock)</LI><UL><LI>0.05pmol x 50¬µL/1.950pmol = 1.3¬µL</LI></UL><LI>phz 1F-1R Purif</LI><UL><LI>0.05pmol x 157,303pg/1pmol x 1ng/1000pg x 1¬µL/13.3ng = 0.59¬µL</LI></UL><LI>phz 4-5 (gBlock)</LI><UL><LI>0.05pmol x 100¬µL/2.057pmol = 2.4¬µL</LI></UL><LI>phz 2F-3R Purif</LI><UL><LI>0.05pmol x 561,368pg/1pmol x 1ng/1000pg x 1¬µL/11.8ng = 2.4¬µL</LI></UL></UL><P>For Gibson mix, mixing total volume of 6.69 ¬µL fragments + 10 ¬µL NEB Gibson Master Mix + 3.31 ¬µL dH<SUB>2</SUB>0</P><P>Above solution incubated at 50¬∞C for 60 min then at 4¬∞C for infinite hold.</P></DIV><BUTTON class="collapsible">Week 9: 7/9/2018</BUTTON><DIV class="content"><DIV class="notebookdate"><H4>July 9, 2018</H4><H4>Resuspending G-Blocks</H4><H4>Shannon</H4></DIV><UL><LI>Resuspended phz 1-2, cymA, mtr start, and tesA using TB</LI><LI>Centrifuge briefly to make sure all products were at the bottom of the tube</LI><UL><LI>Note: Forgot to do this for phz 1-2 (ex. Phz 1-2 had 500ng in original tube, so 500ng x 1¬µL/10ng = 50¬µL)</LI></UL></UL><TABLE><TBODY><TR><TH>G-Block</TH><TH>Amount of TB (¬µL)</TH></TR><TR><TD>Phz 1-2</TD><TD>50</TD></TR><TR><TD>cymA</TD><TD>100</TD></TR><TR><TD>Mtr start</TD><TD>25</TD></TR><TR><TD>tesA</TD><TD>100</TD></TR></TBODY></TABLE><UL><LI>Vortex briefly</LI><LI>Incubate at 50C for 20 min in heat block</LI><UL><LI>Note: forgot to pre-heat heat block so they sat at room temp for a few minutes</LI></UL><LI>Briefly vortex and centrifuge</LI><LI>Stored at -20C</LI></UL><DIV class="notebookdate"><H4>July 9, 2018</H4><H4>Restriction Digest</H4><H4>Shannon</H4></DIV><UL><LI>Used benchling protocol to digest fadD, tesA, cymA, psb1C3, and mtr start</LI><LI>All enzymes checked on nebcloner.neb.com/#!/redigest for incubation temperatures and ‚Äútime-saver‚Äù (they can be incubated at 37C for 10 minutes rather than 1hr) and to verify that CutSmart could be used as the buffer</LI><LI>Add reagents in the order: water ‚Üí buffer ‚Üí DNA ‚Üí enzymes</LI><LI>fadD/tesA/cymA digested with ecoRI and pstI</LI></UL><TABLE><TBODY><TR><TH>Component</TH><TH>Amount (¬µL)</TH></TR><TR><TD>ecoRI</TD><TD>1</TD></TR><TR><TD>pstI</TD><TD>1</TD></TR><TR><TD>CutSmart</TD><TD>2</TD></TR><TR><TD>DNA</TD><TD>5.8 ¬µL for fadD</TD></TR><TR><TD>10 ¬µL for tesA and cymA</TD></TR><TR><TD>dH<SUB>2</SUB>O</TD><TD>10.2 ¬µL for fadD
6¬µL for tesA and cymA
</TD></TR></TBODY></TABLE><UL><LI>Calculations:</LI><UL><LI>For water: 20 ¬µL - volume of all other components</LI><LI>DNA: wanted 100ng, so calculated volume ex. 100ng tesA x 1¬µL/10ng = 5.8¬µL</LI></UL></UL><H4>Psb1C3 digested with dpnI, ecoRI, and pstI</H4><TABLE><TBODY><TR><TH>Component</TH><TH>Amount (¬µL)</TH></TR><TR><TD>dpnI</TD><TD>1</TD></TR><TR><TD>ecoRI</TD><TD>1</TD></TR><TR><TD>pstI</TD><TD>1</TD></TR><TR><TD>CutSmart</TD><TD>2</TD></TR><TR><TD>DNA</TD><TD>4</TD></TR><TR><TD>dH<SUB>2</SUB>O</TD><TD>13</TD></TR></TBODY></TABLE><H4>Mtr start digested with ecoRI, bspHI</H4><TABLE><TBODY><TR><TH>Component</TH><TH>Amount (¬µL)</TH></TR><TR><TD>ecoRI</TD><TD>1</TD></TR><TR><TD>bspHI</TD><TD>1</TD></TR><TR><TD>CutSmart</TD><TD>2</TD></TR><TR><TD>DNA</TD><TD>10</TD></TR><TR><TD>dH<SUB>2</SUB>O</TD><TD>6</TD></TR></TBODY></TABLE><UL><LI>Centrifuge briefly</LI><LI>Heat block at 37C for 10min</LI><LI>Heat block at 80C for 15min (heat-kill enzymes)</LI><LI>Store at 20C</LI></UL><DIV class="notebookdate"><H4>July 9, 2018</H4><H4>fadK, phz gibson PCR (try 2), phz Beg PCR Assembly</H4><H4>Liz, Sricharan</H4></DIV><P>No bands on gel for fadK and phz gibson 1 (7/15) so redid PCR</P><H4>Notes/changes:</H4><UL><LI>Did two tubes for phz (one uses HF buffer and other uses GC buffer)</LI><LI>Phz 35 cycles time 3 changed to 45s because doing PCR assembly at the same time</LI></UL><P>Also tried to use PCR assembly instead of Gibson for phz Beg</P><UL><LI>Just add phz 1F-1R, phz 4-5, and phz 2F-3R to PCR mix and they should assemble together in theory, Phusion does all the gaps</LI><LI>Need 40ng total template (later realized this should be less, 40ng is for genomic DNA)</LI><LI>Since construct is 300bp, this is ~0.02 pmol - using quantities from 7/5/2018</LI></UL><P>Followed PCR purification protocol from 6/28</P><P>Nanodropped pure products</P><UL><LI>pure_fadD: 17.24 ng/¬µL</LI><LI>pure_fadK: 3.57 ng/¬µL</LI><LI>pure_fadK_try2: 4.81 ng/¬µL</LI><LI>pure_phzgibson1: 17.04 ng/¬µL</LI></UL><TABLE><TBODY><TR><TH>Lane</TH><TH>1</TH><TH>2</TH><TH>3</TH><TH>4</TH><TH>5</TH></TR><TR><TH>Sample</TH><TD>Ladder</TD><TD>fadD PCR Purif</TD><TD>fadK PCR Purif</TD><TD>Phz Beg Gibson PCR</TD></TR></TBODY></TABLE><P>On a 1% agarose gel in TAE with SYBRSafe, as usual, run at 50V for 50min</P><DIV class="notebookdate"><H4>July 10, 2018</H4><H4>Ligation of cymA, tesA, and fadD</H4><H4>Shannon and Toni</H4></DIV><UL><LI>Did a ligation to insert cymA, tesA, and fadD, separately, into psB1C3 backbones</LI><LI>Using ligation protocol on benchling, calculated ng of insert dna needed via <A href="https://nebiocalculator.neb.com/#!/ligation">link</A></LI></UL><TABLE><TBODY><TR><TH>Sample</TH><TH>Insert DNA Length</TH><TH>Vector DNA Length</TH><TH>Vector DNA Mass</TH><TH>Insert DNA Mass</TH><TH>Ratio</TH></TR><TR><TH>tesA</TH><TD>800kb</TD><TD>2070kb</TD><TD>30ng</TD><TD>34.78ng</TD><TD>3:1</TD></TR><TR><TH>Fad D</TH><TD>1713kb</TD><TD>2070kb</TD><TD>30ng</TD><TD>49.65ng</TD><TD>2:1</TD></TR><TR><TH>cymA</TH><TD>773kb</TD><TD>2070kb</TD><TD>30ng</TD><TD>33.61ng</TD><TD>3:1</TD></TR></TBODY></TABLE><TABLE><TBODY><TR><TH>tesA</TH><TH>fadD</TH><TH>cymA</TH><TH>Added</TH></TR><TR><TD>4 ¬µL</TD><TD>1.1 ¬µL</TD><TD>4.3 ¬µL</TD><TD>dH<SUB>2</SUB>O</TD></TR><TR><TD>2 ¬µL</TD><TD>2 ¬µL</TD><TD>2 ¬µL</TD><TD>T4 buffer</TD></TR><TR><TD>6 ¬µL</TD><TD>6 ¬µL</TD><TD>6 ¬µL</TD><TD>psB1C3 backbone</TD></TR><TR><TD>7 ¬µL</TD><TD>9.9 ¬µL</TD><TD>6.7 ¬µL</TD><TD>DNA insert</TD></TR><TR><TD>1 ¬µL</TD><TD>1 ¬µL</TD><TD>1 ¬µL</TD><TD>T4 ligase</TD></TR></TBODY></TABLE><UL><LI>Total volume of soln: 20ul; 100ng/20ul (20ng/ul) of stock vector/inserts</LI><LI>All reagents kept on ice and added in the order listed</LI><LI>Incubated at RT for 10 min</LI><LI>Incubated at 65C for 20 min</LI><LI>Placed immediately into -20C</LI></UL><DIV class="notebookdate"><H4>July 10, 2018</H4><H4>PCR purification</H4><H4>Liz</H4></DIV><H4>Samples (4):</H4><UL><LI>Phz gibson 1 product</LI><UL><LI>With HF buffer</LI><LI>With GC buffer</LI></UL><LI>fadK (secondary PCR)</LI><LI>Phz assembly (PCR assembly)</LI></UL><P>Followed protocol from 6/28</P><P>Nanodropped purified product</P><UL><LI>Phz gibson HF: 5.93 ng/¬µL</LI><LI>Phz gibson GC: 10.77 ng/¬µL</LI><LI>fadK: 9.28 ng/¬µL</LI><LI>Phz assembly: 20.11 ng/¬µL</LI></UL><P>Ran gel at 50V for 42 min</P><TABLE><TBODY><TR><TH>1</TH><TH>2</TH><TH>3</TH><TH>4</TH><TH>5</TH></TR><TR><TD>ladder</TD><TD>Phz gibson GC</TD><TD>Phz gibson HF</TD><TD>fadK</TD><TD>Phz PCA</TD></TR></TBODY></TABLE><DIV class="notebookdate"><H4>July 10, 2018</H4><H4>MIT Competent Cells</H4><H4>Liz, Sricharan, Shannon, Toni</H4></DIV><P><A href="https://tanet.mit.edu/drupal7/node/839">Using protocol given to us at NEGEM by Brian Teague of MIT</A></P><P>2 5mL OCs of JM109 were put in 2xYT the day before</P><P>Put 800¬µL JM109 overnight cultures into 400mL LB broth and put into shaker (37 degrees C, 250rpm) at 9:30am</P><H4>Making transformation buffers (TfbI, 150mL; TfbII, 20mL):</H4><TABLE><TBODY><TR><TH>TfbI Needed</TH><TH>TfbI Actual</TH><TH>TfbII Needed</TH><TH>TfbII Actual</TH></TR><TR><TD>RbCl</TD><TD>1.8g</TD><TD>1.802g</TD><TD>0.024g</TD><TD>0.0235g</TD></TR><TR><TD>MnCl<SUB>2</SUB></TD><TD>1.5g</TD><TD>1.508g</TD><TD>-</TD><TD>-</TD></TR><TR><TD>CaCl<SUB>2</SUB></TD><TD>0.166g</TD><TD>0.171g</TD><TD>0.166g</TD><TD>0.166g</TD></TR><TR><TD>KOAc</TD><TD>0.442g</TD><TD>0.448g</TD></TR><TR><TD>MOPS</TD><TD>-</TD><TD>-</TD><TD>0.042g</TD><TD>-</TD></TR><TR><TD>Glycerol</TD><TD>22.4mL</TD><TD>Measured w/grad cyl</TD><TD>3.0mL</TD><TD>Measured with serological pipet</TD></TR></TBODY></TABLE><H4>Checked OD:</H4><TABLE><TBODY><TR><TH>Time</TH><TD>11:51</TD><TD>13:15</TD></TR><TR><TH>OD<SUB>600</SUB></TH><TD>0.156</TD><TD>&gt;1</TD></TR></TBODY></TABLE><P>Failure - try again tomorrow; using TfbI and TfbII made today, which is not ideal but we‚Äôre also running out of RbCl</P><DIV class="notebookdate"><H4>July 11, 2018</H4><H4>MIT Competent Cells</H4><H4>Liz, Sricharan, Peter</H4></DIV><UL><LI>2 5mL OCs of JM109 were put in LB the day before</LI><LI>Put 800¬µL JM109 overnight cultures into 380mL LB broth and put into shaker (37 degrees C, 250rpm) at 7:30am</LI><UL><LI>Want OD<SUB>600</SUB> of 0.5-0.7</LI></UL></UL><H4>Checked OD:</H4><TABLE><TBODY><TR><TH>Time</TH><TD>9:25</TD><TD>10:02</TD><TD>10:22</TD><TD>10:41</TD><TD>10:46</TD><TD>10:54</TD></TR><TR><TH>OD<SUB>600</SUB></TH><TD>0.056</TD><TD>0.164</TD><TD>0.289</TD><TD>0.416</TD><TD>0.506</TD><TD>0.565</TD></TR></TBODY></TABLE><DIV class="notebookdate"><H4>July 11, 2018</H4><H4>Transformation of Competent Cells</H4><H4>Peter and Shannon</H4></DIV><UL><LI>Did a transformation (iGEM protocol) with GFP to test that the cells are competent</LI><LI>Pre-chill 3 1.5mL tubes of 50¬µL competent cells - kept on ice while labeling/pipetting</LI><LI>Add 10, 50, and 100 pg/¬µL of RFP construct to tubes</LI><LI>Incubate on ice for 30 min</LI><LI>Heat shock for 45 sec in 37C water bath</LI><LI>Incubate 5 min on ice</LI><LI>Add 950 pre-warmed 2YT to each tube</LI><LI>Incubate 1 hr at 37C on heat block</LI><LI>Note: forgot to do a control - will do tomorrow</LI></UL><DIV class="notebookdate"><H4>July 12, 2018</H4><H4>PCR of FadK using GC buffer</H4><H4>Shannon</H4></DIV><H4>Thermocycler Settings</H4><TABLE><TBODY><TR><TH>Phase</TH><TH>Time (sec)</TH><TH>Temp. (C)</TH></TR><TR><TD>Initial denaturation</TD><TD>30</TD><TD>98</TD></TR><TR><TD>35 cycles</TD><TD>10</TD><TD>98</TD></TR><TR><TD>30</TD><TD>72</TD></TR><TR><TD>40</TD><TD>72</TD></TR><TR><TD>Final extension</TD><TD>10</TD><TD>72</TD></TR><TR><TD>Hold</TD><TD>‚àû</TD><TD>4</TD></TR></TBODY></TABLE><H4>Add to tubes:</H4><TABLE><TBODY><TR><TH>Component</TH><TH>Amount (¬µL)</TH></TR><TR><TD>dH<SUB>2</SUB>O</TD><TD>10</TD></TR><TR><TD>GC buffer</TD><TD>4</TD></TR><TR><TD>DMSO</TD><TD>0.6</TD></TR><TR><TD>10mM forward primer (bad)</TD><TD>1</TD></TR><TR><TD>10mM reverse primer (bad)</TD><TD>1</TD></TR><TR><TD>dNTP mix</TD><TD>0.4</TD></TR><TR><TD>fadK purified PCR (template)</TD><TD>2.76</TD></TR><TR><TD>phusion</TD><TD>0.2</TD></TR></TBODY></TABLE><UL><LI>All components were kept on ice</LI></UL><DIV class="notebookdate"><H4>July 12, 2018</H4><H4>iGEM Transformations</H4><H4>Liz, Sricharan</H4></DIV><P>Tubes:</P><UL><LI>cmmA-H (biobrick, resuspended in 10¬µL milliQ water and stored in Registry Parts 2018 box in -20C)</LI><LI>RFP 25s HS</LI><LI>RFP 45s HS</LI><LI>RFP 60s HS</LI><LI>RFP 90s HS</LI></UL><UL><LI>Thawed competent cells on ice (~6 min)</LI><LI>Put 1¬µL DNA into each corresponding tube</LI><LI>Incubated on ice for 30 min</LI></UL><P>Heat shocked in 42C water</P><UL><LI>RFP 25s HS: 25s</LI><LI>RFP 45s HS, ccmA-H: 45s</LI><LI>RFP 60s HS: 60s</LI><LI>RFP 90s HS: 90s</LI></UL><UL><LI>Incubated on ice for 5 min</LI><LI>Added 450¬µL SOC media to each tube</LI><LI>Incubated in shaker (37C, 250rpm) for 1 hr</LI><LI>Spun down cells at 6800g for 3 min and discarded ~350-400mL</LI><LI>Resuspended cells and plated remaining 100¬µL in hood using pipet tip to spread</LI><LI>Incubated overnight at 37C (put in ~4:15)</LI><LI>Took plates out ~9:00 7/13 - it worked (only for ccmA-H)!!</LI></UL><DIV class="notebookdate"><H4>July 12, 2018</H4><H4>Gibson of phz 1-2, 4F-4R, 6F-6R</H4><H4>Peter, Ryan</H4></DIV><P>Set up the following reaction on ice:</P><UL><LI>phz 1-2 (gBlock): 0.045pmol x 50¬µL/1.180pmol = 1.91¬µL</LI><LI>phz 4F-4R Purif: 0.045pmol x 614,900pg/1pmol x 1ng/1000pg x 1¬µL/15.53ng = 1.78¬µL</LI><LI>phz 6F-6R Purif: 0.045pmol x 584,050pg/1pmol x 1ng/1000pg x 1¬µL/10.65ng = 2.74¬µL</LI></UL><P>For Gibson mix, mixing total volume of 6.43 ¬µL fragments + 10 ¬µL NEB Gibson Master Mix + 3.57 ¬µL dH<SUB>2</SUB>0</P><P>Above solution incubated at 50¬∞C for 60 min then at 4¬∞C for infinite hold.</P><DIV class="notebookdate"><H4>July 12, 2018</H4><H4>PCR Cleanup of FadK</H4><H4>Shannon</H4></DIV><UL><LI>Combine 80¬µL B2 buffer (4x the amount of PCR sample) to 20¬µL PCR sample</LI><LI>Transfer to collection tube, centrifuge at 10,000 rpm for 1 min. Discard flow-through. Re-insert column into collection tube.</LI><LI>Add 650¬µL wash buffer with ethanol. Centrifuge at 10,000 rpm for 1 min. Discard flow-through. Re-insert column into collection tube.</LI><LI>Centrifuge at max speed (16,100rpm) for 3 min. Discard flow-through. Insert column into a <SPAN class="italic">clean</SPAN> 1.7mL elution tube.</LI><LI>Add 50¬µL elution buffer to center of column.</LI><LI>Incubate at room temperature for 1 min.</LI><LI>Centrifuge at max speed for 2 min.</LI><LI>The elution tube now contains purified PCR product!</LI><LI>Store at -20C.</LI></UL><DIV class="notebookdate"><H4>July 13, 2018</H4><H4>PCR of Gibson assembly from 7/12 (Phz 4F-4R, Phz 1-2, Phz 6F-6R)</H4><H4>Peter, Ryan</H4></DIV><H4>Thermocycler Settings</H4><TABLE><TBODY><TR><TH>Phase</TH><TH>Time (sec)</TH><TH>Temp. (C)</TH></TR><TR><TD>Initial denaturation</TD><TD>30</TD><TD>98</TD></TR><TR><TD>35 Cycles</TD><TD>10</TD><TD>98</TD></TR><TR><TD>30</TD><TD>67</TD></TR><TR><TD>78</TD><TD>72</TD></TR><TR><TD>Final Extension</TD><TD>10</TD><TD>72</TD></TR><TR><TD>Hold</TD><TD>‚àû</TD><TD>4</TD></TR></TBODY></TABLE><TABLE><TBODY><TR><TH>Component</TH><TH>Amount (¬µL)</TH></TR><TR><TD>dH<SUB>2</SUB>O</TD><TD>10.15</TD></TR><TR><TD>GC buffer</TD><TD>4</TD></TR><TR><TD>DMSO</TD><TD>0.6</TD></TR><TR><TD>10mM forward primer (bad)</TD><TD>1</TD></TR><TR><TD>10mM reverse primer (bad)</TD><TD>1</TD></TR><TR><TD>dNTP mix</TD><TD>0.4</TD></TR><TR><TD>DNA Template from Gibson Assembly of</TD><TD>2.65</TD></TR><TR><TD>phusion</TD><TD>0.2</TD></TR></TBODY></TABLE><UL><LI>All components were kept on ice</LI></UL><DIV class="notebookdate"><H4>July 13, 2018</H4><H4>Gel Electrophoresis of Purified FadK PCR</H4><H4>Shannon</H4></DIV><UL><LI>Prepare gel</LI><UL><LI>Combine 15 mL 1x TAE and 0.125g agarose (weigh between 0.10 and 0.15 because our scale isn‚Äôt that accurate) in a small erlenmeyer flask</LI><LI>Microwave until dissolved</LI><LI>Cover with foil, let cool until flask isn‚Äôt too hot to handle</LI><LI>Add 1.25¬µL Sybr Safe, swirl</LI><LI>Pour into a small mold, add comb</LI><LI>Let harden</LI></UL></UL><DIV class="notebookdate"><H4>July 13, 2018</H4><H4>LB Broth</H4><H4>Liz</H4></DIV><H4>500mL Broth</H4><TABLE><TBODY><TR><TH>Component</TH><TH>Amount (g)</TH><TH>Measured (g)</TH></TR><TR><TD>tryptone</TD><TD>5</TD><TD>5.05</TD></TR><TR><TD>Yeast extract</TD><TD>2.5</TD><TD>2.50</TD></TR><TR><TD>NaCl</TD><TD>5</TD><TD>5.00</TD></TR></TBODY></TABLE><UL><LI>QSed to just under 500mL</LI><LI>Put in stir bar and stirred/heated for 15 min</LI><LI>QSed to 500mL in graduated cylinder</LI><LI>Autoclaved on liquids 30 then stored in 4C</LI></UL><DIV class="notebookdate"><H4>July 13, 2018</H4><H4>PCR Cleanup of Phz(4F-6R Gibson)</H4><H4>Ryan and Peter</H4></DIV><UL><LI>Combine 80¬µL B2 buffer (4x the amount of PCR sample) to 20¬µL PCR sample</LI><LI>Transfer to collection tube, centrifuge at 10,000 rpm for 1 min. Discard flow-through. Re-insert column into collection tube.</LI><LI>Add 650¬µL wash buffer with ethanol. Centrifuge at 10,000 rpm for 1 min. Discard flow-through. Re-insert column into collection tube.</LI><LI>Centrifuge at max speed (16,100rpm) for 3 min. Discard flow-through. Insert column into a <SPAN class="italic">clean</SPAN> 1.7mL elution tube.</LI><LI>Add 50¬µL elution buffer to <SPAN class="italic">center</SPAN> of column.</LI><LI>Incubate at room temperature for 1 min.</LI><LI>Centrifuge at max speed for 2 min.</LI><LI>The elution tube now contains purified PCR product!</LI><LI>Store at -20C.</LI><LI>Nanodroped @ 16.7 ng/¬µL</LI></UL><DIV class="notebookdate"><H4>July 13, 2018</H4><H4>Gel of PCR‚Äôs for Phz 4F-6R, fadK Primary and Secondary PCR‚Äôs</H4><H4>Peter, Ryan</H4></DIV><H4>Wells</H4><TABLE><TBODY><TR><TH>1</TH><TH>2</TH><TH>3</TH><TH>4</TH><TH>5</TH></TR><TR><TD>2 Log Ladder</TD><TD>PHz 4F-6R (Gibson) purif.</TD><TD>fadK Primary purif. (7/5/18)</TD><TD>fadK secondary purif. (7/9/18)</TD><TD>fadK secondary purif. (7/12/18)</TD></TR><TR><TD>-6 ¬µL ladder</TD><TD>-2 ¬µL purif. PCR
-3 ¬µL MilliQ DH2O
-1 ¬µL Loading Dye
</TD><TD>-5 ¬µL purif. PCR
-1 ¬µL Loading Dye
</TD><TD>-5 ¬µL purif. PCR
-1 ¬µL Loading Dye
</TD><TD>-5 ¬µL purif. PCR
-1 ¬µL Loading Dye
</TD></TR></TBODY></TABLE><UL><LI>Gibson worked‚Ä¶ two visible bands, one a little less than 3 kb according to the ladder (full gibson should be 2.588 Kb) and one band that is probably both 4F-4R and 6F-6R that didn‚Äôt assemble are about both ~ 1 Kb</LI><LI>Secondary fadK most likely didn‚Äôt work because PCR used ~ 40 ng from Primary PCR so that is probably what is visible in this gel (also same length as primary PCR)</LI></UL></DIV><BUTTON class="collapsible">Week 10: 7/16/2018</BUTTON><DIV class="content"><DIV class="notebookdate"><H4>July 16, 2018</H4><H4>Secondary PCR of fadD and fadK</H4><H4>Ryan, Shannon</H4></DIV><H4>Set thermocycler:</H4><TABLE><TBODY><TR><TH>Phase</TH><TH>Time (sec)</TH><TH>Temp. (C)</TH></TR><TR><TH>Initial denaturation</TH><TD>30</TD><TD>98</TD></TR><TR><TH>35 cycles</TH><TD>10</TD><TD>98</TD></TR><TR><TD>30</TD><TD>67</TD></TR><TR><TD>40</TD><TD>72</TD></TR><TR><TH>Final extension</TH><TD>10 minutes</TD><TD>72</TD></TR><TR><TH>Hold</TH><TD>‚àû</TD><TD>4</TD></TR></TBODY></TABLE><H4>Add to tubes:</H4><TABLE><TBODY><TR><TH>Component</TH><TH>fadD Amount (¬µL)</TH><TH>fadK Amount (¬µL)</TH></TR><TR><TH>dH<SUB>2</SUB>O</TH><TD>12.6</TD><TD>12.5</TD></TR><TR><TH>Phusion buffer (HF)</TH><TD>4</TD><TD>4</TD></TR><TR><TH>DMSO</TH><TD>0.6</TD><TD>.6</TD></TR><TR><TH>10mM forward primer (bad)</TH><TD>1</TD><TD>1</TD></TR><TR><TH>10mM reverse primer (bad)</TH><TD>1</TD><TD>1</TD></TR><TR><TH>dNTP mix</TH><TD>0.4</TD><TD>.4</TD></TR><TR><TH>Template DNA</TH><TD>.2</TD><TD>.3</TD></TR><TR><TH>phusion</TH><TD>0.2</TD><TD>.2</TD></TR></TBODY></TABLE><UL><LI>All components were kept on ice</LI></UL><DIV class="notebookdate"><H4>July 16, 2018</H4><H4>Restriction Digest of Secondary fadD and fadK PCRs</H4><H4>Liz</H4></DIV><H4>fadD and fadK:</H4><TABLE><TBODY><TR><TH>Component</TH><TH>Amount</TH></TR><TR><TD>EcoRI (HF)</TD><TD>1¬µL</TD></TR><TR><TD>PstI (HF)</TD><TD>1¬µL</TD></TR><TR><TD>CutSmart</TD><TD>2¬µL</TD></TR><TR><TD>DNA template</TD><TD>100ng</TD></TR><TR><TD>fadD: 100ng x 1¬µL/14.3ng = 6.99¬µL</TD></TR><TR><TD>fadK: 100ng x 1¬µL/19.9ng = 5.02¬µL</TD></TR><TR><TD>Water </TD><TD>To 20¬µL</TD></TR><TR><TD>fadD: 9.00¬µL</TD></TR><TR><TD>fadK: 10.97¬µL</TD></TR></TBODY></TABLE><UL><LI>Order: water, buffer, DNA, enzymes</LI><LI>Spin down in tabletop centrifuge</LI><LI>Heat blocked at 37C for 25 min</LI><LI>Heat blocked at 65C for 15 min</LI></UL><H4>Backbone:</H4><TABLE><TBODY><TR><TH>Component</TH><TH>Amount</TH></TR><TR><TD>BpsHI</TD><TD>1¬µL</TD></TR><TR><TD>EcoRI (HF)</TD><TD>1¬µL</TD></TR><TR><TD>PstI (HF)</TD><TD>1¬µL</TD></TR><TR><TD>CutSmart</TD><TD>2¬µL</TD></TR><TR><TD>DNA template</TD><TD>100ng</TD></TR><TR><TD>fadD: 100ng x 1¬µL/14.3ng = 6.99¬µL</TD></TR><TR><TD>fadK: 100ng x 1¬µL/19.9ng = 5.02¬µL</TD></TR><TR><TD>Water</TD><TD>To 20¬µL</TD></TR><TR><TD>fadD: 9.00¬µL</TD></TR><TR><TD>fadK: 10.97¬µL</TD></TR></TBODY></TABLE><UL><LI>Order: water, buffer, DNA, enzymes</LI><LI>Spin down in tabletop centrifuge</LI><LI>Heat blocked at 37C for 25 min</LI><LI>Heat blocked at 80C for 15 min</LI><LI>This digest was wrong because DnpI should have been used for the backbone instead of BpsHI</LI></UL><DIV class="notebookdate"><H4>July 16, 2018</H4><H4>Gel Electrophoresis of Purified FadK, FadD PCR</H4><H4>Ryan</H4></DIV><P>Prepare gel</P><UL><LI>Combine 15 mL 1x TAE and 0.125g agarose (weigh between 0.10 and 0.15 because our scale isn‚Äôt that accurate) in a small erlenmeyer flask</LI><LI>Microwave until dissolved</LI><LI>Cover with foil, let cool until flask isn‚Äôt too hot to handle</LI><LI>Add 1.25¬µL Sybr Safe, swirl</LI><LI>Pour into a small mold, add comb</LI><LI>Let harden</LI></UL><H4>Wells:</H4><TABLE><TBODY><TR><TH>1</TH><TH>2</TH><TH>3</TH><TH>4</TH><TH>5</TH></TR><TR><TD>2 Log Ladder</TD><TD>fadD PCR2</TD><TD>fadD PCR0</TD><TD>fadK PCR2</TD><TD>fadK PCR0</TD></TR><TR><TD>-6 ¬µL ladder</TD><TD>-1 ¬µL purif. PCR
-4 ¬µL MilliQ DH2O
-1 ¬µL Loading Dye
</TD><TD>-1 ¬µL purif. PCR
-4 ¬µL MilliQ DH2O
-1 ¬µL Loading Dye
</TD><TD>-1 ¬µL purif. PCR
-4 ¬µL MilliQ DH2O
-1 ¬µL Loading Dye
</TD><TD>-1 ¬µL purif. PCR
-4 ¬µL MilliQ DH2O
-1 ¬µL Loading Dye
</TD></TR></TBODY></TABLE><DIV class="notebookdate"><H4>July 16, 2018</H4><H4>Transformation with RbCl JM109 Cells</H4><H4>Sricharan, Peter</H4></DIV><H4>Transforming:</H4><P>BBa_J04450 (RFP), BBa_B0015 (DoubleTerminator), BBa_K590025 (PetroBrick), BBa_K1433005 (Lambda Red), BBa_I718008 (Inducible Cre-generator), BBa_K1316012 (Mtr + T7 Lac O.), TesA (from ligation), CymA (from ligation), BBa_K917006 (ccmA-H)</P><UL><LI>Grab 50ŒºL aliquots of RbCl Uber-Competent Cells from -80¬∞ C freezer, let thaw on ice for ~3-4 minutes.</LI><LI>Add 1ŒºL of plasmid from iGEM kit (or 2 ŒºL of plasmid from ligation product) to the tube with cells.</LI><LI>Incubate cells on ice for 30 minutes.</LI><LI>Heat shock tubes at 42¬∞ C for 45s.</LI><LI>Incubate on ice again for 5 minutes.</LI><LI>Add 450ŒºL SOC media to each tube (9 x volume) and incubate on the shaker at 37¬∞ C and 250rpm for 1 hour.</LI><LI>Spin down cells at ~6800rpm for 3 minutes, remove 370ŒºL of supernatant, resuspend cell pellet in the remaining media, and pipet that onto the plate</LI><LI>Use P200 to spread cells across plate</LI><LI>Leave plates upside-down (with agar on top) in incubator (at 37¬∞ C) overnight for 16-18 hours.</LI></UL><H4>Notes:</H4><UL><LI>Added 2 ŒºL pf DNA for TesA &amp; CymA because from ligation reaction so not as much DNA as a prepared BioBrick</LI><LI>Heat-shocked ccmA-H (BBa_K917006) in 5 different tubes for 30, 45, 60, 75, and 90 seconds (one time per tube)</LI><LI>Added 900 ŒºL of soc media for tube 4 because was 100 ŒºL aliquot of cells</LI><LI>Used Amp plate for BBa_I718008 because plasmid has Amp resistance</LI></UL><H4>Results (7/17/18):</H4><TABLE><TBODY><TR><TH>Part/ Vector</TH><TH>Transformation Result</TH></TR><TR><TD>BBa_J04450 (RFP)</TD><TD>-No growth on plate</TD></TR><TR><TD>BBa_B0015 (Double Terminator)</TD><TD>-No observable cell pellet when spun down
-No growth on plate</TD></TR><TR><TD>BBa_K590025 (PetroBrick)</TD><TD>-Good growth (do O.C. on 7/17)</TD></TR><TR><TD>BBa_K1433005 (Lambda Red)</TD><TD>-Good growth (do O.C. on 7/17)</TD></TR><TR><TD>BBa_I718008 (Inducible Cre-generator)</TD><TD>-Good growth (do O.C. on 7/17)</TD></TR><TR><TD>BBa_K1316012 (Mtr + T7 Lac O.)</TD><TD>-No observable cell pellet when spun down
-No growth on plate</TD></TR><TR><TD>TesA (from ligation)</TD><TD>-No growth on plate (Ligation enzymes can be a problem)</TD></TR><TR><TD>CymA (from ligation)</TD><TD>-No growth on plate (Ligation enzymes can be a problem)</TD></TR><TR><TD>BBa_K917006 (ccmA-H)</TD><TD>-30 sec: works best
-45 sec: works well
-60 sec: works okay
-75 sec: no visible pellet but grew a few colonies
-90 sec: no visible pellet or growth on plate (probably heat-killed)
</TD></TR></TBODY></TABLE><DIV class="notebookdate"><H4>July 17, 2018</H4><H4>Ligation of fadD and fadK</H4><H4>Liz</H4></DIV><UL><LI>*didn‚Äôt work because digested backbone wrong</LI><LI>fadD:</LI><LI>Insert DNA: 1689 bp</LI><LI>Vector length: 2070 bp</LI><LI>Vector mass: 25 ng</LI><LI>NEB Ligation Calculator: 2:1 ‚Üí 40.80ng x 20L</LI></UL><DIV class="notebookdate"><H4>July 17, 2018</H4><H4>Gel Electrophoresis of Purified FadK, FadD PCR</H4><H4>Ryan</H4></DIV><P>Prepare gel</P><UL><LI>Combine 15 mL 1x TAE and 0.125g agarose (weigh between 0.10 and 0.15 because our scale isn‚Äôt that accurate) in a small erlenmeyer flask</LI><LI>Microwave until dissolved</LI><LI>Cover with foil, let cool until flask isn‚Äôt too hot to handle</LI><LI>Add 1.25¬µL Sybr Safe, swirl</LI><LI>Pour into a small mold, add comb</LI><LI>Let harden</LI></UL><H4>Wells:</H4><TABLE><TBODY><TR><TH>1</TH><TH>2</TH><TH>3</TH><TH>4</TH><TH>5</TH></TR><TR><TD>2 Log Ladder</TD><TD>fadD PCR2</TD><TD>fadD PCR0</TD><TD>fadK PCR2</TD><TD>fadK PCR0</TD></TR><TR><TD>-6 ¬µL ladder</TD><TD>-1 ¬µL purif. PCR
-4 ¬µL MilliQ DH<SUB>2</SUB>O
-1 ¬µL Loading Dye
</TD><TD>-1 ¬µL purif. PCR
-4 ¬µL MilliQ DH<SUB>2</SUB>O
-1 ¬µL Loading Dye
</TD><TD>-1 ¬µL purif. PCR
-4 ¬µL MilliQ DH<SUB>2</SUB>O
-1 ¬µL Loading Dye
</TD><TD>-1 ¬µL purif. PCR
-4 ¬µL MilliQ DH<SUB>2</SUB>O
-1 ¬µL Loading Dye
</TD></TR></TBODY></TABLE><UL><LI>No bands visible on any part of the gel:</LI></UL><DIV class="notebookdate"><H4>July 17, 2018</H4><H4>Gel Extraction of phz 4F-6R</H4><H4>Sricharan</H4></DIV><H4>Onto a 1% agarose gel with SYBR Safe, loaded (with 6X Purple Loading Dye from NEB)</H4><TABLE><TBODY><TR><TH>1</TH><TH>2</TH><TH>3</TH><TH>4</TH><TH>5</TH></TR><TR><TD>2-log ladder</TD><TD>phz 1F-3R PCR (GC Buffer)</TD><TD>‚ÄòtesA(L109P) ligation</TD><TD>phz 4F-6R PCR</TD><TD>Same as Lane 4</TD></TR><TR><TD>6¬µL ladder</TD><TD>5¬µL item
1¬µL dye
</TD><TD>1¬µL item
4¬µL dH<SUB>2</SUB>O
1¬µL dye
</TD><TD>15¬µL item
3¬µL dH<SUB>2</SUB>O</TD></TR></TBODY></TABLE><P>Note, wells 2 and 3 had blue stain from tape (we tried to tape together these two wells to do gel extraction but it didn‚Äôt work very well).</P><UL><LI>Cut bands at 2.6kb on last two lanes out using tape (marked with arrows delineating top of gel, bottom of gel, and location of band to be excised) and wax paper + razor</LI><LI>Weighed cut bands: 0.0840g, used Zymoclean Gel DNA Recovery Kit to clean up DNA. Only deviation from protocol was centrifuged ~16K rpm to elute 30¬µL buffer from Invitrogen PCR Pure Kit.</LI></UL><P>Nanodrop Results:</P><UL><LI>Conc: 2.8 ng/¬µL</LI><LI>260/280: 1.23</LI><LI>260/230: 0.05</LI></UL><DIV class="notebookdate"><H4>July 17, 2018</H4><H4>LB + Chloramphenicol Plates Preparation</H4><H4>Ryan</H4></DIV><P>Preparing Chloramphenicol LB plates (24)</P><UL><LI>Collected and measured out ingredients for LB media and added to bottle</LI><LI>Q/s‚Äôd with dH<SUB>2</SUB>0 to 400 and shook bottle to dissolve ingredients</LI><LI>Stirred in beaker for 30 min</LI><LI>Poured into grad cylinder to finish Q/s to 500 mL</LI><LI>Poured back into bottle and stirred for ~ a minute</LI><LI>Autoclaved on S03 setting for 30 minutes (autoclaving less than a liter of liquid)</LI><LI>Added 0.5 mL of chloramphenicol (1000x, 35 mg/mL) to solution</LI><LI>Put them in the hood with the plates half open w/ agar side on the bottom to get rid of condensation</LI></UL><P>**OOPS put in 35 microliters instead of .5 mL</P><DIV class="notebookdate"><H4>July 17, 2018</H4><H4>Transformation with RbCl JM109 Cells</H4><H4>Liz, Peter</H4></DIV><P>**See Protocol from 7/16/18**</P><H4>Transforming failed transformations from 7/16:</H4><P>BBa_J04450 (RFP), BBa_B0015 (DoubleTerminator), BBa_K1316012 (Mtr + T7 Lac O.), TesA (from ligation + now purified w/ PCR purif), CymA (from ligation + now purified w/ PCR purif), BBa_K917006 (ccmA-H)</P><P>***See protocol from 7/16***</P><H4>Notes/ Deviations:</H4><UL><LI>CymA and TesA have been purified w/ PCR purif.</LI><LI>Two tubes of cells/ transformation (to ensure one tube will have cells) so 0.5 ŒºL of DNA added to each tube (only one tube for the blank)</LI><LI>Heat-shocked for 30 seconds based on results from transformations of ccmA-H</LI><LI>After resuspending, pooled secondary tube w/ primary tube if any cells (next time pool before spinning)</LI><LI>All except blank had one tube w/ obvious pellet after first spin</LI><LI>BBa_J04450 (RFP), BBa_B0015 (Double terminator), and CymA secondary tubes w/ no original pellet had slightly visible pellet after second spin. Were pooled with primary tube after resuspension</LI><LI>Not pellet for 6 so didn‚Äôt bother plating</LI></UL><H4>Results (7/18/18):</H4><TABLE><TBODY><TR><TH>Part/ Vector</TH><TH>Transformation Result</TH></TR><TR><TD>BBa_J04450 (RFP)</TD><TD>-No growth on plate</TD></TR><TR><TD>BBa_B0015 (Double Terminator)</TD><TD>-One colony (Do O.C. on 7/18)</TD></TR><TR><TD>BBa_K1316012 (Mtr + T7 Lac O.)</TD><TD>-No growth on plate</TD></TR><TR><TD>TesA (from ligation + now purified w/ PCR purif)</TD><TD>-No growth on plate </TD></TR><TR><TD>CymA (from ligation + now purified w/ PCR purif)</TD><TD>-One colony (Do O.C. on 7/18)</TD></TR></TBODY></TABLE><DIV class="notebookdate"><H4>July 17, 2018</H4><H4>Overnight Cultures of successful transformations from 7/16/18</H4><H4>Peter, Shannon</H4></DIV><H4>Overnight Cultures:</H4><UL><LI>BBa_K590025 (PetroBrick)</LI><LI>BBa_K1433005 (Lambda Red)</LI><LI>BBa_I718008 (Inducible Cre-generator)</LI><LI>BBa_K917006 (ccmA-H)</LI></UL><H4>Results (7/18/18):</H4><UL><LI>They worked: proceed to mini-prep and glycerol stocking</LI></UL><DIV class="notebookdate"><H4>July 18, 2018</H4><H4>Nanodrops and a Gel: Miniprepped Plasmids, Gel Extraction PCR of 4F-6R, Gel Extraction of 1F-3R</H4><H4>Sricharan</H4></DIV><P>Some OCs were grown - Shannon miniprepped and Liz made glycerol stocks; I Nanodropped</P><TABLE><TBODY><TR><TH>Item</TH><TH>Concentration (ng/¬µL)</TH><TH>260/280</TH><TH>260/230</TH></TR><TR><TD>I718008 MP</TD><TD>85.7</TD><TD>1.75</TD><TD>1.28</TD></TR><TR><TD>K590025 MP</TD><TD>51.7</TD><TD>1.73</TD><TD>0.85</TD></TR><TR><TD>K1433005 MP</TD><TD>53.9</TD><TD>1.74</TD><TD>1.37</TD></TR><TR><TD>K917006 MP</TD><TD>175.1</TD><TD>1.88</TD><TD>2.15</TD></TR><TR><TD>Phz 4F-6R GelExtr PCR</TD><TD>30.9</TD><TD>1.39</TD><TD>-0.82</TD></TR></TBODY></TABLE><H4>Ran a gel:</H4><TABLE><TBODY><TR><TH>Lane</TH><TH>1</TH><TH>2</TH><TH>3</TH><TH>4</TH><TH>5</TH><TH>6</TH></TR><TR><TH>Item</TH><TD>2-log ladder</TD><TD>Poked hole</TD><TD>K143305 MP</TD><TD>I718008 MP</TD><TD>K917006 MP</TD><TD>PCR of Gel Extr of phz 4F-6R</TD></TR><TR><TH>Amt (¬µL)</TH><TD>6</TD><TD>N/A</TD><TD>5+1 dye</TD><TD>5+1 dye</TD><TD>5+1 dye</TD><TD>2+3 dH<SUB>2</SUB>O+1 dye</TD></TR></TBODY></TABLE><TABLE><TBODY><TR><TH>7</TH><TH>8</TH><TH>9</TH><TH>10</TH><TH>11</TH><TH>12</TH></TR><TR><TD>2-log ladder</TD><TD>phz 1F-3R GC PCR</TD><TD>phz 1F-3R GC PCR</TD><TD>empty</TD><TD>K590025 MP</TD><TD>2-log ladder</TD></TR><TR><TD>6</TD><TD>15 + 3 dye</TD><TD>15 + 3 dye</TD><TD>5 + 1 dye</TD><TD>6</TD></TR></TBODY></TABLE><UL><LI>Note, items in lanes 3, 4, 5, 11 digested with PstI.</LI><LI>Dye is 6X Purple Loading Dye from NEB.</LI><LI>Liz gel extracted bands 8 and 9 at desired size (~3kb)</LI></UL><H4>PCR of these bands:
To PCR tube on ice:
</H4><TABLE><TBODY><TR><TD>dH<SUB>2</SUB>O</TD><TD>11.8 ¬µL</TD></TR><TR><TD>Phusion Buffer (GC)</TD><TD>4 ¬µL</TD></TR><TR><TD>DMSO</TD><TD>0.6 ¬µL</TD></TR><TR><TD>10¬µM forward primer (1F)</TD><TD>1 ¬µL</TD></TR><TR><TD>10¬µM reverse primer (3R)</TD><TD>1 ¬µL</TD></TR><TR><TD>10¬µM dNTPs</TD><TD>0.4 ¬µL</TD></TR><TR><TD>Template DNA</TD><TD>1 ¬µL</TD></TR><TR><TD>Phusion polymerase</TD><TD>0.2 ¬µL</TD></TR></TBODY></TABLE><H4>Cycle is based on Phusion polymerase protocol:</H4><TABLE><TBODY><TR><TH>Time (s)</TH><TH>Temp (¬∞C)</TH></TR><TR><TH>Initial Denaturation</TH><TD>30</TD><TD>98</TD></TR><TR><TH>35 Cycles</TH><TD>10</TD><TD>98</TD></TR><TR><TD>30</TD><TD>61</TD></TR><TR><TD>60</TD><TD>72</TD></TR><TR><TH>Final Extension</TH><TD>600</TD><TD>72</TD></TR><TR><TH>Hold</TH><TD>Infinite Hold</TD><TD>4</TD></TR></TBODY></TABLE><DIV class="notebookdate"><H4>July 18, 2018</H4><H4>Miniprep of overnight cultures from 7/17/2018</H4><H4>Shannon</H4></DIV><UL><LI>Used Zyppy Miniprep Kit and instructions from the manufacturer‚Äôs website</LI><LI>Overnight cultures of: K590025, K1433005, I718008, and K917006</LI><LI>Add 600¬µL of each bacterial culture to separate 1.5mL tubes</LI><LI>Add 100¬µL 7x lysis buffer, mix by inverting about 6 times</LI><UL><LI>Solution should turn from opaque blue to clear blue</LI></UL><LI>Add 350¬µL cold neutralization buffer (yellow), mix by inverting</LI><UL><LI>Solution should turn yellow</LI></UL><LI>Centrifuge at 13,000g for 3 min</LI><LI>Transfer supernatant (about 900¬µL) into column, place column in collection tube</LI><LI>Centrifuge for 15 seconds</LI><LI>Discard flow-through, put column back into the same collection tube</LI><LI>Add 200¬µL endo wash buffer, centrifuge 20 seconds</LI><LI>Add 400¬µL Zippy wash buffer to column. Centrifuge 1 min</LI><LI>Transfer the column to clean 1.5 mL tube, add 30¬µL elution buffer directly to column. Let stand for 1 min</LI><LI>Centrifuge 30 seconds to elute DNA</LI></UL><DIV class="notebookdate"><H4>July 18, 2018</H4><H4>Transformation with RbCl JM109 Cells</H4><H4>Liz, Peter</H4></DIV><P>**See Protocol from 7/16/18**</P><H4>Transforming failed transformations from 7/17 + more:</H4><P>BBa_J04450 (RFP), BBa_K1316012 (Mtr + T7 Lac O.) from 2017 and 2018 kit, TesA (from ligation + now purified w/ PCR purif), FadK (from ligation + now purified w/ PCR purif), FadD (from ligation + now purified w/ PCR purif), Blank</P><H4>Notes/Deviations:</H4><UL><LI>Two tubes of cells/ transformation (to ensure one tube will have cells) so 0.5 ŒºL of DNA added to each tube (only one tube for the blank)</LI><LI>Heat-shocked for 30 seconds based on results from transformations of ccmA-H (done on 7/16)</LI><LI>Pooled like tubes (same vector) after 1 hour incubation</LI></UL><H4>Results (7/19/18):</H4><TABLE><TBODY><TR><TH>Part/ Vector</TH><TH>Transformation Result</TH></TR><TR><TD>BBa_J04450 (RFP)</TD><TD>1 colony (O.C. on 7/19)</TD></TR><TR><TD>BBa_K1316012 (Mtr + T7 Lac O.) 2017 kit</TD><TD>No growth</TD></TR><TR><TD>BBa_K1316012 (Mtr + T7 Lac O.) 2018 kit</TD><TD>No growth</TD></TR><TR><TD>TesA (from ligation + now purified w/ PCR purif)</TD><TD>1 colony (O.C. on 7/19)</TD></TR><TR><TD>FadK (from ligation + now purified w/ PCR purif)</TD><TD>1 colony (O.C. on 7/19)</TD></TR><TR><TD>FadK (from ligation + now purified w/ PCR purif)</TD><TD>No growth</TD></TR><TR><TD>FadD (from ligation + now purified w/ PCR purif)</TD><TD>No growth</TD></TR><TR><TD>Blank</TD><TD>No growth</TD></TR></TBODY></TABLE><DIV class="notebookdate"><H4>July 18, 2018</H4><H4>LB + Chloramphenicol Plates Preparation</H4><H4>Ryan</H4></DIV><P>Preparing Chloramphenicol LB plates (24)</P><UL><LI>Collected and measured out ingredients for LB media and added to bottle</LI><LI>Q/s‚Äôd with dH<SUB>2</SUB>0 to 400 and shook bottle to dissolve ingredients</LI><LI>Stirred in beaker for 30 min</LI><LI>Poured into grad cylinder to finish Q/s to 500 mL</LI><LI>Poured back into bottle and stirred for ~ a minute</LI><LI>Autoclaved on S03 setting for 30 minutes (autoclaving less than a liter of liquid)</LI><LI>Added 0.5 mL of chloramphenicol (1000x, 35 mg/mL) to solution</LI><LI>Put them in the hood with the plates half open w/ agar side on the bottom to get rid of condensation</LI></UL><DIV class="notebookdate"><H4>July 19, 2018</H4><H4>Miniprep and glycerol stocks of cymA ligation and B0015</H4><H4>Shannon</H4></DIV><UL><LI>Used Zyppy Miniprep Kit and instructions from the manufacturer‚Äôs website</LI><LI>Add 600¬µL of each bacterial culture to separate 1.5mL tubes</LI><LI>Add 100¬µL 7x lysis buffer, mix by inverting</LI><LI>Add 350¬µL cold neutralization buffer (yellow), mix by inverting</LI><LI>Centrifuge at 13,000g for 3 min</LI><LI>Transfer supernatant (about 900¬µL) into column, place column in collection tube</LI><LI>Centrifuge for 15 seconds</LI><LI>Discard flow-through, put column back into the same collection tube</LI><LI>Add 200¬µL endo wash buffer, centrifuge 20 seconds</LI><LI>Add 400¬µL Zippy wash buffer to column. Centrifuge 1 min</LI><LI>Transfer the column to clean 1.5 mL tube, add 30¬µL elution buffer directly to column. Let stand for 1 min</LI><LI>Centrifuge 30 seconds to elute DNA</LI><LI>Made glycerol stocks by combining 0.75mL of 40% glycerol and 0.75mL overnight culture, stored at -80C</LI></UL><DIV class="notebookdate"><H4>July 19, 2018</H4><H4>PCR Purification of fadD and fadK</H4><H4>Shannon</H4></DIV><UL><LI>Using Lambda Biotech Column Pure PCR Cleanup Kit</LI><LI>Mix 20¬µL PCR reaction with 100¬µL binding buffer</LI><LI>Load onto spin column, centrifuge at full speed (16,100g) for 1 min</LI><LI>Discard flow-through</LI><LI>Add 700¬µL wash buffer, centrifuge 1 min</LI><LI>Discard flow-through, centrifuge again 1 min</LI><LI>Transfer column to clean 1.5mL tube</LI><LI>Add 30¬µL elution buffer, let sit 1 min, centrifuge 1 min to elute DNA</LI></UL><DIV class="notebookdate"><H4>July 19, 2018</H4><H4>Restriction Digest of CymA and B0015</H4><H4>Peter</H4></DIV><H4>B0015 (Terminator):</H4><TABLE><TBODY><TR><TH>Component</TH><TH>Amount</TH></TR><TR><TD>PstI (HF)</TD><TD>1¬µL</TD></TR><TR><TD>CutSmart</TD><TD>1¬µL (should have done 2¬µL)</TD></TR><TR><TD>DNA template</TD><TD>100ng x 1¬µL/53.8ng = 1.86¬µL</TD></TR><TR><TD>Water</TD><TD>16.14¬µL (To 20¬µL)</TD></TR></TBODY></TABLE><UL><LI>Order: water, buffer, DNA, enzymes</LI><LI>Spin down in tabletop centrifuge</LI><LI>Heat blocked at 37C for 10 min</LI><LI>Heat-killed on block at 80C for 15 min</LI></UL><H4>CymA:</H4><TABLE><TBODY><TR><TH>Component</TH><TH>Amount</TH></TR><TR><TD>PstI (HF)</TD><TD>1¬µL</TD></TR><TR><TD>CutSmart</TD><TD>1¬µL (should have done 2¬µL)</TD></TR><TR><TD>DNA template</TD><TD>5¬µL (20 ng/¬µL) -wanted a lot of it)</TD></TR><TR><TD>Water</TD><TD>13 ¬µL (To 20¬µL)</TD></TR></TBODY></TABLE><UL><LI>Order: water, buffer, DNA, enzymes</LI><LI>Spin down in tabletop centrifuge</LI><LI>Heat blocked at 37C for 10 min</LI><LI>Heat-killed on block at 80C for 15 min</LI></UL><H4>MtrCAB (2017 or 2018?):</H4><TABLE><TBODY><TR><TH>Component</TH><TH>Amount</TH></TR><TR><TD>PstI (HF)</TD><TD>1¬µL</TD></TR><TR><TD>CutSmart</TD><TD>1¬µL (should have done 2¬µL)</TD></TR><TR><TD>DNA template</TD><TD>100ng x 1¬µL/112.5ng = 0.89¬µL</TD></TR><TR><TD>Water</TD><TD>17.11 ¬µL (To 20¬µL)</TD></TR></TBODY></TABLE><UL><LI>Order: water, buffer, DNA, enzymes</LI><LI>Spin down in tabletop centrifuge</LI><LI>Heat blocked at 37C for 10 min</LI><LI>Heat-killed on block at 80C for 15 min</LI></UL><DIV class="notebookdate"><H4>July 19, 2018</H4><H4>PCR Purification of phz 1F-3R Gel Extraction PCR and a Gel</H4><H4>Sricharan</H4></DIV><UL><LI>PCR purified product from yesterday (phz 1F-3R gel extraction PCR)</LI><LI>Used Lambda Biotech Kit protocol, eluted with 30¬µL buffer</LI></UL><H4>Agarose gel (1%, large 35mL)</H4><TABLE><TBODY><TR><TH>Lane</TH><TH>1</TH><TH>2</TH><TH>3</TH><TH>4</TH><TH>5</TH><TH>6</TH><TH>7</TH><TH>8</TH><TH>9</TH><TH>10</TH></TR><TR><TH>Item</TH><TD>2-log ladder</TD><TD>phz 1F-3R Gel Extraction PCR</TD><TD>cymA MP</TD><TD>B0015 MP</TD><TD>K1316012 plasmid (from 2017 Kit)</TD><TD>fadD PCR2 with correct annealing temp</TD><TD>fadD PCR0 diluted to 40ng/30¬µL</TD><TD>fadK PCR2 with correct annealing temp</TD><TD>fadK PCR0 diluted to 40ng/30¬µL</TD><TD>2-log ladder</TD></TR><TR><TH>Amt (¬µL)</TH><TD>5</TD><TD>3+2 dH2O+1 dye</TD><TD>5+1 dye</TD><TD>5+1 dye</TD><TD>5+1 dye</TD><TD>5+1 dye</TD><TD>5+1 dye</TD><TD>5+1 dye</TD><TD>5+1 dye</TD><TD>6</TD></TR></TBODY></TABLE><UL><LI>Note that lanes 3-5 are digested with PstI</LI></UL><UL><LI>Seems that fadD PCR of the coding region (‚ÄúPCR2‚Äù) is successful since it is significantly amplified past that of template.</LI><LI>Need to redo fadK PCR2</LI><LI>Also cymA and B0015 might be around the same size, but hard to tell here, need to retry the gel</LI></UL><DIV class="notebookdate"><H4>July 19, 2018</H4><H4>Overnight Cultures of TesA and J04450 (RFP)</H4><H4>Peter</H4></DIV><H4>Overnight Cultures:</H4><UL><LI>TesA</LI><LI>BBa_J04450 (RFP)</LI></UL><H4>Results (7/20/18):</H4><UL><LI>They worked: proceed to miniprep and glycerol stocking</LI></UL><DIV class="notebookdate"><H4>July 20, 2018</H4><H4>PCR2 of FadK and PCR of ‚ÄòTesA(L109P) Basic Part from miniprep (7/18/18)</H4><H4>Peter, Sricharan</H4></DIV><H4>FadK PCR2:</H4><P>Set thermocycler:</P><TABLE><TBODY><TR><TH>Phase</TH><TH>Time (sec)</TH><TH>Temp. (C)</TH></TR><TR><TH>Initial denaturation</TH><TD>30</TD><TD>98</TD></TR><TR><TH>35 cycles</TH><TD>10</TD><TD>98</TD></TR><TR><TD>30</TD><TD>72 (from NEB calculator)</TD></TR><TR><TD>48</TD><TD>72</TD></TR><TR><TH>Final extension</TH><TD>10 minutes</TD><TD>72</TD></TR><TR><TH>Hold</TH><TD>‚àû</TD><TD>4</TD></TR></TBODY></TABLE><H4>Add to tubes in this order:</H4><TABLE><TBODY><TR><TH>Component</TH><TH>Amount (¬µL)</TH></TR><TR><TD>dH<SUB>2</SUB>O</TD><TD>12.11</TD></TR><TR><TD>HF buffer</TD><TD>4</TD></TR><TR><TD>dNTP mix</TD><TD>0.4</TD></TR><TR><TD>10mM forward primer (bad)</TD><TD>1</TD></TR><TR><TD>10mM reverse primer (bad)</TD><TD>1</TD></TR><TR><TD>DMSO</TD><TD>0.6</TD></TR><TR><TD>Template DNA (FadK PCR0)</TD><TD>0.69 (10 ng)</TD></TR><TR><TD>phusion</TD><TD>0.2</TD></TR></TBODY></TABLE><UL><LI>All components were kept on ice</LI></UL><H4>‚ÄòTesA (L109P):</H4><P>Set thermocycler:</P><TABLE><TBODY><TR><TH>Phase</TH><TH>Time (sec)</TH><TH>Temp. (C)</TH></TR><TR><TH>Initial denaturation</TH><TD>30</TD><TD>98</TD></TR><TR><TH>35 cycles</TH><TD>10</TD><TD>98</TD></TR><TR><TD>30</TD><TD>72 (from NEB calculator)</TD></TR><TR><TD>26</TD><TD>72</TD></TR><TR><TH>Final extension</TH><TD>10 minutes</TD><TD>72</TD></TR><TR><TH>Hold</TH><TD>‚àû</TD><TD>4</TD></TR></TBODY></TABLE><H4>Add to tubes in this order:</H4><TABLE><TBODY><TR><TH>Component</TH><TH>Amount (¬µL)</TH></TR><TR><TD>dH<SUB>2</SUB>O</TD><TD>12.54</TD></TR><TR><TD>HF buffer</TD><TD>4</TD></TR><TR><TD>dNTP mix</TD><TD>0.4</TD></TR><TR><TD>10mM forward primer (bad)</TD><TD>1</TD></TR><TR><TD>10mM reverse primer (bad)</TD><TD>1</TD></TR><TR><TD>DMSO</TD><TD>0.6</TD></TR><TR><TD>Template DNA (‚ÄòTesA L109P from miniprep)</TD><TD>0.26 (10 ng)</TD></TR><TR><TD>phusion</TD><TD>0.2</TD></TR></TBODY></TABLE><UL><LI>All components were kept on ice</LI></UL><DIV class="notebookdate"><H4>July 20, 2018</H4><H4>Gel of FadK PCR2, ‚ÄòTesA (L109P) basic part, and other stuff (see Sricharan‚Äôs notebook)</H4><H4>Peter, Ryan</H4></DIV><H4>Wells:</H4><TABLE><TBODY><TR><TH>1</TH><TH>2</TH><TH>3</TH><TH>4</TH><TH>5</TH><TH>6</TH></TR><TR><TD>2 Log Ladder</TD><TD>fadK PCR2 HF</TD><TD>fadK PCR0 diluted to 0.69¬µL/30¬µL</TD><TD>fadK PCR2 GC</TD><TD>‚ÄòtesA(L109P) PCR Basic</TD><TD>K1316012 (mtrCAB) 2017 Kit Digest w/ PstI</TD></TR></TBODY></TABLE><TABLE><TBODY><TR><TH>7</TH><TH>8</TH><TH>9</TH><TH>10</TH><TH>11</TH><TH>12</TH></TR><TR><TD>2 Log Ladder</TD><TD>‚ÄòtesA(L109P) Miniprep Digest w/PstI</TD><TD>Kit 7, Well 2H 2018 Digest w/PstI</TD><TD>J04450 Miniprep Digest 3x</TD><TD>See lane 10</TD><TD>See Lane 1</TD></TR></TBODY></TABLE><UL><LI>Note that 6¬µL of ladder added to lanes 1, 7, 12; 5¬µL of item and 1¬µL of loading dye added to all other lanes</LI></UL><H4>Results:</H4><TABLE><TBODY><TR><TH>1</TH><TH>2</TH><TH>3</TH><TH>4</TH><TH>5</TH><TH>6</TH></TR></TBODY></TABLE><TABLE><TBODY><TR><TH>7</TH><TH>8</TH><TH>9</TH><TH>10</TH><TH>11</TH><TH>12</TH></TR></TBODY></TABLE><DIV class="notebookdate"><H4>July 20, 2018</H4><H4>Restriction 5 things</H4><H4>Peter</H4></DIV><P>Digest of J04450 (RFP), ‚ÄòTesA (L109P), FadD PCR2, Mtr 2016 (K1316012), Kit 7 Well 2H</P><H4>J04450 (RFP):</H4><TABLE><TBODY><TR><TH>Component</TH><TH>Amount</TH></TR><TR><TD>XBaI</TD><TD>1¬µL</TD></TR><TR><TD>SpeI</TD><TD>1¬µL</TD></TR><TR><TD>DpNI</TD><TD>1¬µL</TD></TR><TR><TD>CutSmart</TD><TD>1¬µL (should have done 2¬µL)</TD></TR><TR><TD>DNA template</TD><TD>10¬µL</TD></TR><TR><TD>dH<SUB>2</SUB>O</TD><TD>6¬µL (To 20¬µL)</TD></TR></TBODY></TABLE><UL><LI>Order: water, buffer, DNA, enzymes</LI><LI>Spin down in tabletop centrifuge</LI><LI>Heat blocked at 37C for 10 min</LI><LI>Heat-killed on block at 80C for 15 min</LI></UL><H4>CymA:</H4><TABLE><TBODY><TR><TH>Component</TH><TH>Amount</TH></TR><TR><TD>PstI (HF)</TD><TD>1¬µL</TD></TR><TR><TD>CutSmart</TD><TD>1¬µL (should have done 2¬µL)</TD></TR><TR><TD>DNA template</TD><TD>5¬µL (20 ng/¬µL) -wanted a lot of it)</TD></TR><TR><TD>Water</TD><TD>13 ¬µL (To 20¬µL)</TD></TR></TBODY></TABLE><UL><LI>Order: water, buffer, DNA, enzymes</LI><LI>Spin down in tabletop centrifuge</LI><LI>Heat blocked at 37C for 10 min</LI><LI>Heat-killed on block at 80C for 15 min</LI></UL><H4>MtrCAB (2017 or 2018?):</H4><TABLE><TBODY><TR><TH>Component</TH><TH>Amount</TH></TR><TR><TD>PstI (HF)</TD><TD>1¬µL</TD></TR><TR><TD>CutSmart</TD><TD>1¬µL (should have done 2¬µL)</TD></TR><TR><TD>DNA template</TD><TD>100ng x 1¬µL/112.5ng = 0.89¬µL</TD></TR><TR><TD>Water </TD><TD>17.11 ¬µL (To 20¬µL)</TD></TR></TBODY></TABLE><UL><LI>Order: water, buffer, DNA, enzymes</LI><LI>Spin down in tabletop centrifuge</LI><LI>Heat blocked at 37C for 10 min</LI><LI>Heat-killed on block at 80C for 15 min</LI></UL></DIV><BUTTON class="collapsible">Week 11: 7/23/2018</BUTTON><DIV class="content"><DIV class="notebookdate"><H4>July 23, 2018</H4><H4>LB + Cam, LB + Kan Plates Preparation</H4><H4>Ryan</H4></DIV><P>Preparing Chloramphenicol LB plates (18):</P><UL><LI>Collected and measured out ingredients for LB media and added to bottle</LI><LI>Q/s‚Äôd with dH20 to 350 mL and shook bottle to dissolve ingredients</LI><LI>Stirred in beaker for 30 min</LI><LI>Poured into grad cylinder to finish Q/s to 375 mL</LI><LI>Poured back into bottle and stirred for ~ a minute</LI><LI>Autoclaved on S03 setting for 30 minutes (autoclaving less than a liter of liquid)</LI><LI>Split solution in half such that 9 plates are for Cam and 9 plates are for Kan</LI><LI>Added 0.1875 mL of chloramphenicol (1000x, 35 mg/mL) to solution in one half, 0.1875 mL of kanamycin (1000x, 35 mg/mL) to other</LI><LI>Let plates cool for an hour on lab bench</LI></UL><DIV class="notebookdate"><H4>July 23, 2018</H4><H4>Digest of J04450 and psb1C3</H4><H4>Peter and Shannon</H4></DIV><UL><LI>Add reagents in order listed</LI><LI>Centrifuge briefly</LI><LI>Heat block at 37C (45 min for psb1C3, 10 min for the rest)</LI><LI>Heat block at 80C for 15 min to heat kill enzymes</LI></UL><H4>psb1C3, I718008, K1433005, K1316012</H4><TABLE><TBODY><TR><TH>Component</TH><TH>Amount (¬µL)</TH></TR><TR><TD>Water (milliQ)</TD><TD>psb1C3: 12</TD></TR><TR><TD>1718008: 14.83</TD></TR><TR><TD>K1433005: 14.14</TD></TR><TR><TD>K1316012: 15</TD></TR><TR><TD>CutSmart</TD><TD>2</TD></TR><TR><TD>DNA</TD><TD>psb1C3: 4</TD></TR><TR><TD>1718008: 1.17</TD></TR><TR><TD>K1433005: 1.86</TD></TR><TR><TD>K1316012:1</TD></TR><TR><TD>EcoRI</TD><TD>1</TD></TR><TR><TD>PstI</TD><TD>1</TD></TR></TBODY></TABLE><H4>J04450</H4><TABLE><TBODY><TR><TH>Component</TH><TH>Amount (¬µL)</TH></TR><TR><TD>Water (milliQ)</TD><TD>15.65</TD></TR><TR><TD>CutSmart</TD><TD>2</TD></TR><TR><TD>DNA</TD><TD>0.35</TD></TR><TR><TD>SpaI-HF</TD><TD>1</TD></TR><TR><TD>XbaI</TD><TD>1</TD></TR></TBODY></TABLE><DIV class="notebookdate"><H4>July 23, 2018</H4><H4>Reviving Glycerol Stocks</H4><H4>Shannon</H4></DIV><UL><LI>Put glycerol stocks on ice (try not to let them thaw too much)</LI><LI>Stab into the glycerol stock with a p20 pipette tip</LI><LI>Use pipette tip to t-streak on a place with the correct antibiotic</LI><LI>Incubate overnight at 37C</LI><LI>Revived: J23114, J23110, J23100, and cymA. Was going to revive K864400 but could not find it in the -80C freezer</LI><LI>Note: There were three tubes labelled J23110, and one tube that had the label wiped off. These may have been mislabelled. I believe two are J23110 and two are J23100. The two that have more color should be the stronger promoter, and the two with less color should be the weaker promoter. I labelled one plate ‚ÄúS‚Äù and one ‚ÄúW‚Äù for strong and weak.</LI><LI>Note: For the tube labelled J23110, I don‚Äôt know which of the three ‚ÄúJ23110‚Äù tubes I used. I did this one before realizing the tubes may have been mislabelled.</LI></UL><DIV class="notebookdate"><H4>July 23, 2018</H4><H4>Agarose Gel for Extraction and Checking stuff</H4><H4>Sricharan</H4></DIV><UL><LI>Another 1% agarose gel</LI><LI>Liz gel extracted lanes 2 and 3 at pSB1C3 band (~2kb)</LI></UL><TABLE><TBODY><TR><TH>1</TH><TH>2</TH><TH>3</TH><TH>4</TH><TH>5</TH><TH>6</TH></TR><TR><TD>2-log ladder</TD><TD>J04450 Digest</TD><TD>J04450 Digest</TD><TD>K143305 Digest 7/23</TD><TD>I718008 Digest 7/23</TD><TD>K14316012 Digest 7/23</TD></TR></TBODY></TABLE><UL><LI>Note that 6¬µL of ladder to lane 1; 5¬µL item and 1¬µL dye to every other well. Lanes 2 - 5 are sourced from minipreps. Lane 6 is just DNA from the distribution kit we wanted to test.</LI></UL><DIV class="notebookdate"><H4>July 23, 2018</H4><H4>Gel Extraction of J04450 3x Digest</H4><H4>Shannon</H4></DIV><UL><LI>Excise DNA from gel, add to 1.5mL tube. 184mg were excised.</LI><LI>Add 3x volume ADB. 555mL were added.</LI><LI>Incubate at 45C for 5-10 min until dissolved</LI><LI>Transfer to spin column</LI><LI>Centrifuge 60 sec at 10,000g. Discard flow-through.</LI><LI>Add 200¬µL wash buffer, centrifuge 30 sec. Discard flow through. Repeat.</LI><LI>Add 10¬µL elution buffer directly onto column. Let sit for 1 min.</LI><LI>Place column in clean 1.5mL tube. Centrifuge 60 sec.</LI><LI>Store at -20C.</LI><LI>Note: the elution buffer was from the invitrogen kit.</LI></UL><DIV class="notebookdate"><H4>July 23, 2018</H4><H4>Gel Extraction of J04450 3x Digest</H4><H4>Shannon</H4></DIV><UL><LI>Excise DNA from gel, add to 1.5mL tube. 184mg were excised.</LI><LI>Add 3x volume ADB. 555mL were added.</LI><LI>Incubate at 45C for 5-10 min until dissolved</LI><LI>Transfer to spin column</LI><LI>Centrifuge 60 sec at 10,000g. Discard flow-through.</LI><LI>Add 200¬µL wash buffer, centrifuge 30 sec. Discard flow through. Repeat.</LI><LI>Add 10¬µL elution buffer directly onto column. Let sit for 1 min.</LI><LI>Place column in clean 1.5mL tube. Centrifuge 60 sec.</LI><LI>Store at -20C.</LI><LI>Note: the elution buffer was from the invitrogen kit.</LI></UL><DIV class="notebookdate"><H4>July 23, 2018</H4><H4>J04450 Digest</H4><H4>Liz</H4></DIV><P>Set heat blocks to 37C and 80C</P><TABLE><TBODY><TR><TH>Components (in order)</TH><TH>Amount (¬µL)</TH></TR><TR><TD>milliQ water</TD><TD>6</TD></TR><TR><TD>Cutsmart</TD><TD>2</TD></TR><TR><TD>DNA (J04450 miniprep SK 7/20, 287ng/¬µL)</TD><TD>10</TD></TR><TR><TD>SpeI-HF</TD><TD>1</TD></TR><TR><TD>XbaI</TD><TD>1</TD></TR></TBODY></TABLE><UL><LI>Incubated at 37C for 10 min</LI><LI>Heat shocked at 15 min</LI></UL><DIV class="notebookdate"><H4>July 24, 2018</H4><H4>PCR of PSB1C3 Template from RFP mini-prep</H4><H4>Peter, Sricharan</H4></DIV><H4>Set thermocycler:</H4><TABLE><TBODY><TR><TH>Phase</TH><TH>Time (sec)</TH><TH>Temp. (C)</TH></TR><TR><TH>Initial denaturation</TH><TD>30</TD><TD>98</TD></TR><TR><TH>35 cycles</TH><TD>10</TD><TD>98</TD></TR><TR><TD>30</TD><TD>71 (from NEB calculator)</TD></TR><TR><TD>60</TD><TD>72</TD></TR><TR><TH>Final extension</TH><TD>10 minutes</TD><TD>72</TD></TR><TR><TH>Hold</TH><TD>‚àû</TD><TD>4</TD></TR></TBODY></TABLE><UL><LI>*set to 50 ¬µL reaction</LI></UL><H4>Add to tubes in this order:</H4><TABLE><TBODY><TR><TH>Component</TH><TH>Amount (¬µL)</TH></TR><TR><TD>dH<SUB>2</SUB>O</TD><TD>31.65</TD></TR><TR><TD>HF buffer</TD><TD>10</TD></TR><TR><TD>dNTP mix</TD><TD>1</TD></TR><TR><TD>10mM forward primer (bad)</TD><TD>2.5</TD></TR><TR><TD>10mM reverse primer (bad)</TD><TD>2.5</TD></TR><TR><TD>DMSO</TD><TD>1.5</TD></TR><TR><TD>Template DNA (FadK PCR0)</TD><TD>0.348 (5 ng - from 10x dilute of miniprep)</TD></TR><TR><TD>phusion</TD><TD>0.2</TD></TR></TBODY></TABLE><UL><LI>All components were kept on ice</LI><LI>Used lambda PCR cleanup kit</LI><LI>Nano-dropped @ 48.8 ng/¬µL</LI><LI>Did gel extract for linearized plasmid for Gibsons (no longer using for Gibsons)</LI></UL><DIV class="notebookdate"><H4>July 24, 2018</H4><H4>Overnight Cultures</H4><H4>Shannon</H4></DIV><UL><LI>Did overnight cultures of J23114, J23110, J23100, and cymA</LI><LI>Combine 5mL LB and 5¬µL antibiotic (ampicilin for J23114, J23110, and J23100 and chloramphenicol for cymA)</LI><LI>Put in shaker at 4:20pm</LI><LI>Note: ‚ÄúS‚Äù and ‚ÄúW‚Äù are either J23110 or J23100. ‚ÄúS‚Äù should be the stronger promoter, ‚Äúw‚Äù should be the weaker promoter.</LI><LI>Note: The tube labelled ‚ÄúJ23110‚Äù may be J23100. Check color against ‚ÄúS‚Äù and ‚ÄúW‚Äù?</LI></UL><DIV class="notebookdate"><H4>July 24, 2018</H4><H4>Digest of psB1C3, acr 1, acr 3, fadD, fadK, B0015, cymA, pSB1K3, ccmA-H</H4><H4>Ryan and Liz</H4></DIV><UL><LI>Add reagents in order listed</LI><LI>Centrifuge briefly</LI><LI>Heat block at 37C (45 min for psb1C3, 10 min for the rest)</LI><LI>Heat block at 80C for 15 min to heat kill enzymes</LI></UL><H4>fadD, fadK, cymA, pSB1K3</H4><TABLE><TBODY><TR><TH>Component</TH><TH>Amount (¬µL)</TH></TR><TR><TD>Water (milliQ)</TD><TD>fadD: 11.7</TD></TR><TR><TD>fadD: 9</TD></TR><TR><TD>cymA: 11</TD></TR><TR><TD>pSB1K3: 12</TD></TR><TR><TD>CutSmart</TD><TD>2</TD></TR><TR><TD>DNA</TD><TD>fadD: 4.3</TD></TR><TR><TD>fadK: 7</TD></TR><TR><TD>cymA: 5</TD></TR><TR><TD>pSB1K3: 4</TD></TR><TR><TD>EcoRI</TD><TD>1</TD></TR><TR><TD>Pstl</TD><TD>1</TD></TR></TBODY></TABLE><H4>pSB1C3</H4><TABLE><TBODY><TR><TH>Component</TH><TH>Amount (¬µL)</TH></TR><TR><TD>CutSmart</TD><TD>.5</TD></TR><TR><TD>DNA</TD><TD>4</TD></TR><TR><TD>EcoRI</TD><TD>.25</TD></TR><TR><TD>PstI</TD><TD>.25</TD></TR></TBODY></TABLE><H4>Acr 1</H4><TABLE><TBODY><TR><TH>Component</TH><TH>Amount (¬µL)</TH></TR><TR><TD>CutSmart</TD><TD>1.25</TD></TR><TR><TD>DNA</TD><TD>10</TD></TR><TR><TD>Pstl</TD><TD>1.25</TD></TR></TBODY></TABLE><H4>Acr 3</H4><TABLE><TBODY><TR><TH>Component</TH><TH>Amount (¬µL)</TH></TR><TR><TD>CutSmart</TD><TD>1.25</TD></TR><TR><TD>DNA</TD><TD>10</TD></TR><TR><TD>Pstl</TD><TD>1.25</TD></TR></TBODY></TABLE><H4>B0015</H4><TABLE><TBODY><TR><TH>Component</TH><TH>Amount (¬µL)</TH></TR><TR><TD>Water (milliQ)</TD><TD>14.1</TD></TR><TR><TD>CutSmart</TD><TD>2</TD></TR><TR><TD>DNA</TD><TD>1.86</TD></TR><TR><TD>Xbal</TD><TD>1</TD></TR><TR><TD>Pst1</TD><TD>1</TD></TR></TBODY></TABLE><H4>ccmA-H</H4><TABLE><TBODY><TR><TH>Component</TH><TH>Amount (¬µL)</TH></TR><TR><TD>Water (milliQ)</TD><TD>15.4</TD></TR><TR><TD>CutSmart</TD><TD>2</TD></TR><TR><TD>DNA</TD><TD>0.57</TD></TR><TR><TD>EcoRI</TD><TD>1</TD></TR><TR><TD>Spel</TD><TD>1</TD></TR></TBODY></TABLE><DIV class="notebookdate"><H4>July 24, 2018</H4><H4>Gibson Reactons, Circular Version</H4><H4>Sricharan</H4></DIV><P>Planning Gibson reactions to assemble acr. We realized the overlaps between our acr inserts and the pSB1C3 backbone isn‚Äôt great. We decided on a couple reactions to see if we can use these inserts</P><P>For 50ng pSB1C3, or 0.037pmol of backbone, using the NEB kit, we want about 2x excess of inserts. Using NEB calculator to figure out mass of insert required:</P><TABLE><TBODY><TR><TD>acr Synthesis 1</TD><TD>501bp</TD><TD>26ng</TD></TR><TR><TD>acr Synthesis 2</TD><TD>529bp</TD><TD>27.5ng</TD></TR><TR><TD>acr Synthesis 3</TD><TD>642bp</TD><TD>33.4ng</TD></TR></TBODY></TABLE><H4>Gibson 1: Negative Control Reaction</H4><TABLE><TBODY><TR><TH>Item</TH><TH>Amount (¬µL)</TH></TR><TR><TD>pSB1C3 from J04450 (XbaI and SpeI digested)</TD><TD>1.43</TD></TR><TR><TD>dH<SUB>2</SUB>O</TD><TD>9.41</TD></TR><TR><TD>NEB Gibson Master Mix</TD><TD>10.8</TD></TR></TBODY></TABLE><H4>Gibson 2: acr Assembly</H4><P>To have adequate overlap between acr inserts and pSB1C3, we‚Äôd need to digest something with pSB1C3 backbone with XbaI and SpeI (poorly designed inserts on our part). We did this to J04450 and gel extracted this band. However, this means that there‚Äôs a risk of a) contamination with J04450 plasmid and b) religation of the sticky ends of XbaI and SpeI. That‚Äôs why we‚Äôre running the negative control above and are considering reordering some gBlocks to make Gibsons go smoother. Other option would be to get new primers to amplify pSB1C3 with the overlaps designed into the primers.</P><TABLE><TBODY><TR><TH>Item</TH><TH>Concentration (ng/¬µL)</TH><TH>Amount (¬µL)</TH></TR><TR><TD>pSB1C3 from J04450 (XbaI and SpeI digested)</TD><TD>35</TD><TD>1.43</TD></TR><TR><TD>acr Synthesis 1 (EcoRI digested)</TD><TD>100/12.5</TD><TD>3.01</TD></TR><TR><TD>acr Synthesis 2</TD><TD>10</TD><TD>2.54</TD></TR><TR><TD>acr Synthesis 3 (PstI digested)</TD><TD>100/12.5</TD><TD>3.86</TD></TR><TR><TD>NEB Gibson Master Mix</TD><TD>10.8</TD></TR></TBODY></TABLE><H4>Gibson 3: acr Assembly Alternate</H4><UL><LI>Strategy - ligating pSB1C3 cut with EcoRI and PstI, acr Synthesis 1 cut with EcoRI, acr Synthesis 3 cut with PstI</LI><LI>Then this ligation is added to the Gibson reaction with acr Synthesis 2</LI><LI>Back calculated approx amts of each component required so that concentrations of digest/ligation is high as possible</LI></UL><P>For 70ng digested pSB1C3, or 0.052 pmol, a 1:1 excess of insert means for the ligation:</P><TABLE><TBODY><TR><TH>Item</TH><TH>Concentration (ng/¬µL)</TH><TH>Amount (¬µL)</TH></TR><TR><TD>pSB1C3 Linear Backbone (EcoRI and PstI digested)</TD><TD>20</TD><TD>3.5</TD></TR><TR><TD>acr Synthesis 1 (EcoRI digested)</TD><TD>100/12.5</TD><TD>2.12</TD></TR><TR><TD>acr Synthesis 3 (PstI digested)</TD><TD>100/12.5</TD><TD>2.12</TD></TR><TR><TD>T4 Ligase</TD><TD>100/12.5</TD><TD>0.46</TD></TR><TR><TD>T4 Ligation Buffer</TD><TD>0.91</TD></TR></TBODY></TABLE><UL><LI>(Coordinated on digest concentrations with Liz, used NEB calculator to obtain ng amounts of each insert)</LI><LI>Ligation was incubated at 10 min, RT; Heat killed at 65 ¬∞C for 20 min</LI><LI>Added 7¬µL of this ligation (0.04pmol, theoretical max of 78ng ligated product) to 3¬µL of acr Synthesis 2 (0.088pmol or 30ng)</LI></UL><DIV class="notebookdate"><H4>July 25, 2018</H4><H4>Gibson Reactons, Circular Version, Day 2</H4><H4>Sricharan</H4></DIV><UL><LI>Finished Gibson 2‚Äô (same as 2 but redid since we were unsure about digests), 3 as described yesterday</LI><LI>Next day, got only 1 colony from all Gibsons, from Strategy 2 ‚Ä¶ later realized this was probably religated plasmid</LI></UL><DIV class="notebookdate"><H4>July 25, 2018</H4><H4>Miniprep of J23114, ‚ÄúJ23110‚Äù, ‚ÄúS‚Äù, ‚ÄúW‚Äù, and cymA</H4><H4>Shannon</H4></DIV><UL><LI>‚ÄúJ23110‚Äù could be J23110 or J23100. ‚ÄúS‚Äù should be J23110, and ‚ÄúW‚Äù should be J23100.</LI><LI>Spin down 1.5mL of overnight culture, discard supernatant, add another 1.5mL, spin down, discard supernatant, and rehydrate the pellet in 600¬µL dH<SUB>2</SUB>O.</LI><UL><LI>Spun at 16,100g for 1 min each time</LI></UL><LI>Add 100¬µL lysis buffer, invert to mix</LI><LI>Add 200¬µL cold neutralization buffer (kept on ice), invert to mix</LI><LI>Centrifuge at 13,000 for 3 min</LI><LI>Transfer supernatant to column (about 900¬µL)</LI><LI>Place column in collection tube. Centrifuge 15 sec.</LI><LI>Discard flow-through, put back into collection tube</LI><LI>Add 200¬µL endo-wash buffer, centrifuge 30 sec.</LI><LI>Add 400¬µL wash buffer, centrifuge 1 min</LI><LI>Transfer column to clean 1.5mL tube. Add 30¬µL elution buffer. Let stand 1 min.</LI><LI>Centrifuge 30 sec to elute DNA</LI></UL><DIV class="notebookdate"><H4>July 25, 2018</H4><H4>Transformation with RbCl JM109 Cells</H4><H4>Sricharan and Peter</H4></DIV><P>**See Protocol from 7/16/18**</P><H4>Transforming:</H4><P>BBa_K1316012 (Mtr + T7 Lac O.) from 2016 kit w/ Strataclone Commercial Cells and our JM109 cells, FadD (from ligation + now purified w/ PCR purif), FadK (from ligation + now purified w/ PCR purif), BBa_J23100 (Strong Anderson Promoter), BBa_J23110 (Medium Anderson Promoter), BBa_K864400 (pTac promoter), ACR Gibson 2 (7/24), Gibson control (7/24), ccmA-H 3A (7/24), ACR Gibson 2‚Äô (7/25), ACR Gibson 3 (7/25)</P><H4>Notes/Deviations</H4><UL><LI>Two tubes of cells/ transformation (to ensure one tube will have cells) so 0.5 ŒºL of DNA added to each tube (Gibsons each get 1 ŒºL per tube instead of 0.5, ligations should have maybe gotten 1 ŒºL per tube instead of 0.5, 1 ŒºL for both Mtr tubes of different cells)</LI><LI>Took DNA from diluted Gibson solutions</LI><LI>Heat-shocked for 30 seconds based on results from transformations of ccmA-H (done on 7/16)</LI><LI>Pooled like tubes (same vector) after 1 hour incubation</LI><LI>Amp plates for Anderson Promoters and Kanamycin plate for ccmA-H 3A</LI></UL><H4>Results (7/26/18):</H4><TABLE><TBODY><TR><TH>Part/ Vector</TH><TH>Transformation Result</TH></TR><TR><TD>BBa_K1316012 (Mtr + T7 Lac O.) 2016 kit Strataclone</TD><TD>1 colony (O.C. on 7/26)</TD></TR><TR><TD>BBa_K1316012 (Mtr + T7 Lac O.) 2018 kit Our JM109 Cells</TD><TD>No growth</TD></TR><TR><TD>FadD (from ligation + now purified w/ PCR purif) </TD><TD>No growth</TD></TR><TR><TD>FadK (from ligation + now purified w/ PCR purif)</TD><TD>1 colony (O.C. on 7/26)</TD></TR><TR><TD>BBa_J23100 (Anderson Strong)</TD><TD>Good growth (O.C. on 7/26)</TD></TR><TR><TD>BBa_J23110 (Anderson Medium)</TD><TD>Good growth (O.C. on 7/26)</TD></TR><TR><TD>BBa_K864400 (pTac)</TD><TD>No growth</TD></TR><TR><TD>ACR Gibson 2 (7/24)</TD><TD>1 colony - could be plasmid backbone ligated to itself
(O.C. on 7/26)
</TD></TR><TR><TD>Gibson Control (7/24)</TD><TD>1 colony - shows that plasmid backbone can ligate to itself 
(O.C. on 7/26)
</TD></TR><TR><TD>ccmA-H 3A (7/24)</TD><TD>Good growth (O.C. on 7/26)</TD></TR><TR><TD>ACR Gibson 2‚Äô (7/25)</TD><TD>No growth</TD></TR><TR><TD>ACR Gibson 3 (7/25)</TD><TD>No growth? (Might not have carried out this transformation)</TD></TR></TBODY></TABLE><DIV class="notebookdate"><H4>July 26, 2018</H4><H4>Resuspension of phz G-Blocks (end and 3) for Gibson</H4><H4>Peter</H4></DIV><UL><LI>Centrifuge tube for 3-5 seconds on table-top centrifuge</LI><LI>Add TE to reach a final concentration of 10 ng/¬µL</LI><UL><LI>Phz end 500ng / 50¬µL = 10 ng/¬µL</LI><LI>Phz 3 250ng/ 25¬µL = 10 ng/¬µL</LI></UL><LI>Vortex briefly</LI><LI>Incubate @ 50 C for 20 minutes</LI><LI>Vortex and briefly centrifuge</LI><LI>Store in -20 C</LI></UL><DIV class="notebookdate"><H4>July 26, 2018</H4><H4>Diagnostic Gel of B0015 and CymA and Gel Extract of PSB1C3</H4><H4>Peter and Liz</H4></DIV><H4>Wells:</H4><TABLE><TBODY><TR><TH>1</TH><TH>2</TH><TH>3</TH><TH>4</TH><TH>5</TH></TR><TR><TD>2 Log Ladder</TD><TD>CymA</TD><TD>B0015</TD><TD>PSB1C3 Extract</TD><TD>PSB1C3 Extract</TD></TR><TR><TD>-6 ¬µL ladder</TD><TD>-5 ¬µL Double Digest
-1 ¬µL Loading Dye
</TD><TD>-5 ¬µL Double Digest
-1 ¬µL Loading Dye
</TD><TD>-5 ¬µL Digest
-1 ¬µL Loading Dye
</TD><TD>-5 ¬µL Digest 
-1 ¬µL Loading Dye
</TD></TR></TBODY></TABLE><P>Gel Extract (Zyppy gel extraction kit):</P><UL><LI>Extracted last 2 wells of gel above to get pSB1C3 (brightest band)</LI><UL><LI>Other DNA in the band is RFP and ‚ÄòtesA (length of ‚ÄòtesA is questionable but we don‚Äôt really care about that since we just want backbone)</LI></UL><LI>Pice of gel extracted: 0.1268 = 126.8mg ‚Üí added 126.8 x 3 = 380.4¬µL ADP</LI><LI>Incubated at 42C in heat block until agarose dissolved</LI><LI>Put into spin column and centrifuged 1 min at 16,100g then pipetted out flow-through</LI><LI>Added 200¬µL wash buffer and centrifuged 30s at 16,100g (did this step 2x)</LI><LI>Added 6¬µL elution buffer (from invitrogen PCR purification kit), let sit for 1 min, then centrifuged 1 min at 16,100g</LI></UL><DIV class="notebookdate"><H4>July 26, 2018</H4><H4>3A Assembly of cymA and Anderson promoters into psb1K3</H4><H4>Shannon</H4></DIV><UL><LI>Need a 1:1 molarity of backbone to inserts, so use 8.768ng cymA and 0.397ng J23110</LI><LI>Dilute promoter solutions</LI><UL><LI>We need 0.397ng of each. If undiluted, you would need to small an amount to pipette</LI><LI>Add 1¬µL of the J23100/J23110 digestion (concentration = 57.48 ng/¬µL and 57.81ng/¬µL) to 29¬µL dH2O (new concentration = 1.916ng/¬µL and 1.927ng/¬µL).</LI><LI>Add 1¬µL of the J23114 digestion (concentration = 469.18ng/¬µL) to 249¬µL dH2O (new concentration = 1.88ng/¬µL)</LI></UL></UL><TABLE><TBODY><TR><TH>Component</TH><TH>Amount (¬µL)</TH></TR><TR><TD>Backbone (psb1K3)</TD><TD>5</TD></TR><TR><TD>cymA</TD><TD>1.75</TD></TR><TR><TD>Promoter</TD><TD>0.2</TD></TR><TR><TD>T4 ligase buffer</TD><TD>2.5</TD></TR><TR><TD>T4 enzyme</TD><TD>1.25</TD></TR><TR><TD>dH<SUB>2</SUB>O</TD><TD>14.3</TD></TR></TBODY></TABLE><UL><LI>Combine components in order listed</LI><LI>Ligate at 16C for 30min</LI><LI>Heat kill at 80 C for 20 min</LI><LI>Note: accidently put 1.2¬µL of J23100 into that reaction.</LI><LI>Note: I also forgot to dilute the promoters. Will have to re-do this tomorrow.</LI></UL><DIV class="notebookdate"><H4>July 26, 2018</H4><H4>Some Planning</H4><H4>Sricharan</H4></DIV><UL><LI>Planned out a Gibson for phz, but realized we don‚Äôt have enough of 1F-3R and 4F-6R fragments PCRed</LI><LI>Toni will run PCR of above two fragments again</LI></UL><DIV class="notebookdate"><H4>July 26, 2018</H4><H4>Agarose Gel of Minipreps &amp; PCR of Gibson 1F-3R and 4F-6R</H4><H4>Sricharan</H4></DIV><TABLE><TBODY><TR><TH>Lane</TH><TH>1</TH><TH>2</TH><TH>3</TH><TH>4</TH><TH>5</TH><TH>6</TH></TR><TR><TH>Item</TH><TD>fadK MP Digest</TD><TD>ccmA-H + B0015 MP Digest</TD><TD>J23110 MP Digest</TD><TD>J23100 MP Digest</TD><TD>Phz Gibson 2 MP Digest</TD><TD>K1316012 MP Digest</TD></TR></TBODY></TABLE><TABLE><TBODY><TR><TH>7</TH><TH>8</TH><TH>9</TH><TH>10</TH><TH>11</TH><TH>12</TH></TR><TR><TD>Ladder (2-log)</TD><TD>Gibson Ctrl MP Digest</TD><TD>pSB1C3 PCR Digest</TD><TD>phz 1F-3R Gibson PCR</TD><TD>phz 4F-6R Gibson PCR</TD><TD>Nothing</TD></TR></TBODY></TABLE><UL><LI>All items digested w/PSTI except pSB1C3 (w/ EcoRI, PstI, DpnI - concentrations are off so might not work)</LI></UL><P>5¬µL item, 1¬µL loading dye, Dated 7/27/2018</P><DIV class="notebookdate"><H4>July 26, 2018</H4><H4>Restriction Digests for phz Gibson and CymA 3A</H4><H4>Liz</H4></DIV><H4>Start of phz: (adjusted so no water for higher concentration)</H4><TABLE><TBODY><TR><TH>Component</TH><TH>Amount (¬µL)</TH></TR><TR><TD>Water</TD><TD>0</TD></TR><TR><TD>CutSmart</TD><TD>1.25</TD></TR><TR><TD>DNA (rehydrated start of phz gBlock)</TD><TD>10 (100ng x 1¬µL/10ng = 10¬µL ‚Üí 10/4 = 2.5¬µL enzymes/water)</TD></TR><TR><TD>EcoRI</TD><TD>1.25</TD></TR></TBODY></TABLE><H4>End of phz: (adjusted so no water for higher concentration)</H4><TABLE><TBODY><TR><TH>Component</TH><TH>Amount (¬µL)</TH></TR><TR><TD>Water</TD><TD>0</TD></TR><TR><TD>CutSmart</TD><TD>1.25</TD></TR><TR><TD>DNA (rehydrated start of phz gBlock)</TD><TD>10 (100ng x 1¬µL/10ng = 10¬µL ‚Üí 10/4 = 2.5¬µL enzymes/water)</TD></TR><TR><TD>EcoRI</TD><TD>1.25</TD></TR></TBODY></TABLE><H4>Medium Anderson Promoter (J23110):</H4><TABLE><TBODY><TR><TH>Component</TH><TH>Amount (¬µL)</TH></TR><TR><TD>Water</TD><TD>14.27</TD></TR><TR><TD>CutSmart</TD><TD>2</TD></TR><TR><TD>DNA (J23110 miniprep)</TD><TD>1.73 (100ng x 1¬µL/57.81ng = 1.73¬µL)</TD></TR><TR><TD>EcoRI</TD><TD>1</TD></TR><TR><TD>Spel</TD><TD>1</TD></TR></TBODY></TABLE><H4>cymA:</H4><TABLE><TBODY><TR><TH>Component</TH><TH>Amount (¬µL)</TH></TR><TR><TD>Water</TD><TD>15.27</TD></TR><TR><TD>CutSmart</TD><TD>2</TD></TR><TR><TD>DNA (cymA miniprep)</TD><TD>0.83 (100ng x 1¬µL/120.6ng = 0.83¬µL)</TD></TR><TR><TD>XbaI</TD><TD>1</TD></TR><TR><TD>PstI</TD><TD>1</TD></TR></TBODY></TABLE><H4>Strong Anderson Promoter (J23100): </H4><TABLE><TBODY><TR><TH>Component</TH><TH>Amount (¬µL)</TH></TR><TR><TD>Water</TD><TD>14.26</TD></TR><TR><TD>CutSmart</TD><TD>2</TD></TR><TR><TD>DNA (J23100 miniprep)</TD><TD>1.74 (100ng x 1¬µL/57.48ng = 1.74¬µL)</TD></TR><TR><TD>EcoRI</TD><TD>1</TD></TR><TR><TD>Spel</TD><TD>1</TD></TR></TBODY></TABLE><H4>Weak Anderson Promoter (J23114):</H4><TABLE><TBODY><TR><TH>Component</TH><TH>Amount (¬µL)</TH></TR><TR><TD>Water</TD><TD>15.79</TD></TR><TR><TD>CutSmart</TD><TD>2</TD></TR><TR><TD>DNA (rehydrated start of phz gBlock)</TD><TD>0.213 (100ng x 1¬µL/469.18ng = 0.213¬µL)</TD></TR><TR><TD>EcoRI</TD><TD>1</TD></TR><TR><TD>Spel</TD><TD>1</TD></TR></TBODY></TABLE><H4>RFP miniprep (J04450): </H4><TABLE><TBODY><TR><TH>Component</TH><TH>Amount (¬µL)</TH></TR><TR><TD>Water</TD><TD>6</TD></TR><TR><TD>CutSmart</TD><TD>2</TD></TR><TR><TD>DNA</TD><TD>10</TD></TR><TR><TD>XbaI</TD><TD>1</TD></TR><TR><TD>Spel</TD><TD>1</TD></TR></TBODY></TABLE><UL><LI>For DNA used all of remaining miniprep and then another random miniprep (‚ÄòtesA?) to try to get to 10¬µL to maximize backbone</LI><LI>Heat blocked at 37C for 1 hr then 80C for 15 min (put in thermocycler at 5¬µL reaction)</LI></UL><DIV class="notebookdate"><H4>July 26, 2018</H4><H4>Transformations</H4><H4>Liz</H4></DIV><P>Used 2 tubes of RbCl JM109 cells for each sample</P><UL><LI>Gibson control (1¬µL DNA per tube)</LI><LI>acr gibson 2 (1¬µL DNA per tube)</LI><LI>acr gibson 2‚Äô (1¬µL DNA per tube)</LI><LI>K864400 (0.5¬µL DNA per tube)</LI><LI>fadD ligation (1¬µL DNA per tube)</LI></UL><P>Followed transformation protocol (7/16/2018)</P><DIV class="notebookdate"><H4>July 27, 2018</H4><H4>Restriction Digests</H4><H4>Shannon</H4></DIV><H4>FadK, J23110, J23100, gibson 2 transformation, gibson control</H4><TABLE><TBODY><TR><TH>Component</TH><TH>Amount (¬µL)</TH></TR><TR><TD>Water (milliQ)</TD><TD>For fadK: 16.57</TD></TR><TR><TD>For J23110:16.26</TD></TR><TR><TD>For J23100: 16.80</TD></TR><TR><TD>For gibson 2 transformation: 16.56</TD></TR><TR><TD>For gibson control: 16.54</TD></TR><TR><TD>CutSmart</TD><TD>2</TD></TR><TR><TD>DNA</TD><TD>For fadK: 0.43</TD></TR><TR><TD>For J23110: 0.74</TD></TR><TR><TD>For J23100: 0.20</TD></TR><TR><TD>For gibson 2 transformation: 0.44</TD></TR><TR><TD>For gibson control: 0.46</TD></TR><TR><TD>Pstl</TD><TD>1</TD></TR></TBODY></TABLE><H4>ccmA-H + B0015</H4><TABLE><TBODY><TR><TH>Component</TH><TH>Amount (¬µL)</TH></TR><TR><TD>Water (milliQ)</TD><TD>14.97</TD></TR><TR><TD>CutSmart</TD><TD>2</TD></TR><TR><TD>DNA</TD><TD>1.03</TD></TR><TR><TD>PstI</TD><TD>1</TD></TR><TR><TD>XbaI</TD><TD>1</TD></TR></TBODY></TABLE><H4>Stratoclone K1316012</H4><TABLE><TBODY><TR><TH>Component</TH><TH>Amount (¬µL)</TH></TR><TR><TD>Water (milliQ)</TD><TD>15.79</TD></TR><TR><TD>CutSmart</TD><TD>2</TD></TR><TR><TD>DNA</TD><TD>0.21</TD></TR><TR><TD>PstI</TD><TD>1</TD></TR><TR><TD>BspHI</TD><TD>1</TD></TR></TBODY></TABLE><H4>psb1K3</H4><TABLE><TBODY><TR><TH>Component</TH><TH>Amount (¬µL)</TH></TR><TR><TD>Water (milliQ)</TD><TD>12</TD></TR><TR><TD>CutSmart</TD><TD>2</TD></TR><TR><TD>DNA</TD><TD>4</TD></TR><TR><TD>PstI</TD><TD>1</TD></TR><TR><TD>DpnI</TD><TD>1</TD></TR></TBODY></TABLE><H4>psb1C3</H4><TABLE><TBODY><TR><TH>Component</TH><TH>Amount (¬µL)</TH></TR><TR><TD>Water (milliQ)</TD><TD>24.4</TD></TR><TR><TD>CutSmart</TD><TD>9.76</TD></TR><TR><TD>DNA</TD><TD>10</TD></TR><TR><TD>EcoRI</TD><TD>4.88</TD></TR><TR><TD>PstI</TD><TD>4.88</TD></TR><TR><TD>DpnI</TD><TD>4.88</TD></TR></TBODY></TABLE><UL><LI>Add reagents in order listed</LI><LI>Centrifuge briefly</LI><LI>Heat block at 37C</LI><UL><LI>10 min for all except psb1K3 and psb1C3. 45 min for those.</LI></UL><LI>Heat block at 80C for 15 min.</LI><LI>Note: shouldn‚Äôt have increased the volume for the components in psb1C3. Did not need to scale.</LI></UL><DIV class="notebookdate"><H4>August 2, 2018</H4><H4>Resuspending G-Blocks</H4><H4>Shannon</H4></DIV><UL><LI>Resuspended phz start, phz end, acr1, acr3, pmt1231.1, pmt1231.2</LI><LI>500ng of each g-block was deliverd, so 50¬µL TE was added to each for a final concentration of 10ng/¬µL.</LI><LI>Centrifuge tubes for 3-5 sec on tabletop centrifuge</LI><LI>Add TE</LI><LI>Vortex briefly</LI><LI>Incubate at 50C for 20min</LI><LI>Briefly vortex and cetrifuge</LI><LI>Store at -20C</LI></UL><DIV class="notebookdate"><H4>July 27, 2018</H4><H4>Making SOC Medium</H4><H4>Peter and Ryan</H4></DIV><UL><LI><SPAN class="bold">Bottle 1:</SPAN> Will be SOC Medium</LI><LI><SPAN class="bold">Bottle 2:</SPAN> 2 M MgCl<SUB>2</SUB></LI><LI><SPAN class="bold">Bottle 3:</SPAN> 1 M Glucose</LI></UL><H4>Add to Bottle 1:</H4><TABLE><TBODY><TR><TH>Component</TH><TH>Amount</TH></TR><TR><TD>Water (dH<SUB>2</SUB>O)</TD><TD>950 mL</TD></TR><TR><TD>Bacto-tryptone</TD><TD>20 g</TD></TR><TR><TD>Bacto-yeast extract</TD><TD>5 g</TD></TR><TR><TD>NaCl</TD><TD>0.5 g</TD></TR><TR><TD>KCl</TD><TD>0.1863775 g
    10 ml (0.01 L) of 250 mM solution (0.25 M solution)
0.01 L x 0.25mol/1L x 74.551g/1mol = 0.1863775g</TD></TR></TBODY></TABLE><UL><LI>Shake and used stir bar to dissolve components</LI><LI>pH to 7.0 w/ 5 N NaOH (~ 0.2 ml) -- didn‚Äôt have any so used 1 N NaOH</LI><LI>Adjust volume to 1 liter w/ dH<SUB>2</SUB>O (mark 1 liter point on bottle before adding components)</LI><LI>Autoclave with Bottle 2</LI></UL><H4>Add to Bottle 2:</H4><TABLE><TBODY><TR><TH>Component</TH><TH>Amount</TH></TR><TR><TD>Water (dH<SUB>2</SUB>O)</TD><TD>90mL</TD></TR><TR><TD>MgCl<SUB>2</SUB></TD><TD>19 g</TD></TR></TBODY></TABLE><UL><LI>After components are dissolved, adjust volume to 100 mL w/ dH<SUB>2</SUB>O (mark 100 mL point on bottle before adding components)</LI><LI>Autoclave with Bottle 1</LI><LI>Add 5 mL to Bottle 1 AFTER AUTOCLAVING</LI></UL><H4>Bottle 3:</H4><TABLE><TBODY><TR><TH>Component</TH><TH>Amount</TH></TR><TR><TD>Water (dH<SUB>2</SUB>O)</TD><TD>90mL</TD></TR><TR><TD>Glucose</TD><TD>18g</TD></TR></TBODY></TABLE><UL><LI>After Glucose is dissolved, adjust volume to 100 mL w/ dH2O (mark 100 mL point on bottle before adding components)</LI><LI>Sterilize by filtration through 0.22-micron filter</LI><LI>Add 20 mL to Bottle 1 AFTER AUTOCLAVING</LI></UL><DIV class="notebookdate"><H4>July 27, 2018</H4><H4>Minipreps</H4><H4>Liz</H4></DIV><P>Miniprepped overnight cultures using zyppy miniprep kit</P><UL><LI>fadK ligation</LI><LI>ccmA-H + B0015 3A</LI><LI>J23110</LI><LI>J23100</LI><LI>Gibson acr 2 transformation </LI><LI>Gibson control</LI><LI>Strataclone K1316012</LI></UL><UL><LI>Used 3mL each overnight, spun down, and rehydrated in 600¬µL water</LI><LI>Eluted in 30¬µL elution buffer</LI></UL></DIV><BUTTON class="collapsible">Week 12: 7/30/2018</BUTTON><DIV class="content"><DIV class="notebookdate"><H4>July 30, 2018</H4><H4>3 Digests and a Ligation (And a gel)</H4><H4>Sricharan and Liz</H4></DIV><UL><LI>Digesting 13¬µL of mtrCAB (K1316012) with BspHI and PstI to be ligated to start and assembled to pSB1K3</LI><LI>Digesting 15¬µL of pSB1C3 (Peter‚Äôs PCR) and pSB1K3 (Linearized backbones kit) with EcoRI and PstI and DpnI (ligations in the future)</LI><LI>14¬µL ccmA-H + B0015 diagnostic digest with EcoRI to check on a gel</LI><LI>6¬µL of Mtr Start (gBlock) digested with EcoRI and BspHI to be ligated to rest of mtrCAB and assembled to </LI><LI>13¬µL of ccmA-H (K917006) also digested with EcoRI and SpeI for future 3A assembly</LI><LI>Using standard digestion protocol (all 20¬µL digests)</LI></UL><H4>Ligation:
mtrCAB (TU Delft part) + synthesized mtr start (rehydrated gBlock) + pSB1K3
</H4><TABLE><TBODY><TR><TH>Component</TH><TH>Amount (¬µL)</TH></TR><TR><TD>water</TD><TD>3.24</TD></TR><TR><TD>T4 ligase buffer</TD><TD>0.67</TD></TR><TR><TD>Backbone (pSB1K3 from digestion above)</TD><TD>1.33 (25ng x 20¬µL/375ng = 1.33)</TD></TR><TR><TD>Digested mtr start (equimolar to backbone)</TD><TD>0.28 (1.418ng x 20¬µL/100ng = 0.28)</TD></TR><TR><TD>Digested mtrCAB (equimolar to backbone)</TD><TD>0.82 (58.87ng x 20¬µL/1437.36ng = 0.82)</TD></TR><TR><TD>T4 ligase</TD><TD>0.33</TD></TR></TBODY></TABLE><UL><LI>Scaled to 6.67¬µL reaction based on 25ng plasmid backbone</LI></UL><TABLE><TBODY><TR><TH>Lane</TH><TH>1</TH><TH>2</TH><TH>3</TH></TR><TR><TH>Item</TH><TD>2-log ladder</TD><TD>ccmA-H + B0015 cut with EcoRI (7/30), 5¬µL</TD><TD>pSB1C3 cut w/ EcoRI, PstI, DpnI (7/30), 1¬µL</TD></TR></TBODY></TABLE><DIV class="notebookdate"><H4>July 30, 2018</H4><H4>Transformation with RbCl JM109 Cells</H4><H4>Sricharan and Peter</H4></DIV><P>**See Protocol from 7/16/18**</P><H4>Transforming:</H4><P>FadD (from ligation + now purified w/ PCR purif), mtrCAB 3A (mtrCAB + New Start + PSBIC3), BBa_K864400 (pTac promoter)</P><H4>Notes/Deviations:</H4><UL><LI>Two tubes of cells/ transformation (to ensure one tube will have cells) so 0.5 ŒºL of DNA added to each tube.</LI><LI>Ligation gets1 ŒºL per tube instead of 0.5</LI><LI>Took DNA from diluted Gibson solutions</LI><LI>Heat-shocked for 30 seconds</LI><LI>Pooled like tubes (same vector) after 1 hour incubation</LI><LI>Kanamycin plate for mtrCAB 3A new start</LI></UL><H4>Results (7/31/18):</H4><TABLE><TBODY><TR><TH>Part/Vector</TH><TH>Transformation Result</TH></TR><TR><TD>FadD (from ligation + now purified w/ PCR purif)</TD><TD>No growth</TD></TR><TR><TD>mtrCAB 3A (mtrCAB + New Start + PSBIC3)</TD><TD>Good growth (O.C. on 7/31)</TD></TR><TR><TD>BBa_K864400 (pTac)</TD><TD>Good growth (O.C. on 7/31)</TD></TR></TBODY></TABLE><DIV class="notebookdate"><H4>July 31, 2018</H4><H4>PCR of phz 4f-6R, ‚ÄòtesA (L109P), Basic cymA</H4><H4>Peter</H4></DIV><H4>Notes:</H4><UL><LI>phz 4F-6R: 2.588 kb</LI><LI>Basic ‚ÄòtesA (L109P): 0.57 kb</LI><LI>Basic cymA: 0.595 kb</LI></UL><H4>Set thermocycler:</H4><TABLE><TBODY><TR><TH>Phase</TH><TH>Time (sec)</TH><TH>Temp. (C)</TH></TR><TR><TH>Initial denaturation</TH><TD>30</TD><TD>98</TD></TR><TR><TH>35 cycles</TH><TD>10</TD><TD>98</TD></TR><TR><TD>30</TD><TD>Phz: 67</TD></TR><TR><TD>Phz: 77.6s</TD><TD>‚ÄòtesA &amp; CymA: 72 (from NEB calculator)</TD></TR><TR><TD>‚ÄòtesA &amp; CymA: 17.3</TD><TD>72</TD></TR><TR><TH>Final Extension</TH><TD>10 minutes</TD><TD>72</TD></TR><TR><TH>Hold</TH><TD>‚àû</TD><TD>4</TD></TR></TBODY></TABLE><UL><LI>*set to 50 ¬µL reaction for phz 4F-6R</LI><LI>*set to 20 ¬µL reaction for ‚ÄòtesA and cymA</LI></UL><H4>Phz 4F-6R Tube:</H4><TABLE><TBODY><TR><TH>Component</TH><TH>Amount (¬µL)</TH></TR><TR><TD>dH<SUB>2</SUB>O</TD><TD>29</TD></TR><TR><TD>HF buffer</TD><TD>10</TD></TR><TR><TD>dNTP mix</TD><TD>1</TD></TR><TR><TD>10mM forward primer (bad)</TD><TD>2.5</TD></TR><TR><TD>10mM reverse primer (bad)</TD><TD>2.5</TD></TR><TR><TD>DMSO</TD><TD>1.5</TD></TR><TR><TD>Template DNA (Gibson of 4F-6R)</TD><TD>3 (10ng x 1¬µL/3.785ng = 2.642¬µL) -rounding up to 3 ¬µL</TD></TR><TR><TD>phusion</TD><TD>0.5</TD></TR></TBODY></TABLE><H4>‚ÄòtesA (L109P):</H4><TABLE><TBODY><TR><TH>Component</TH><TH>Amount (¬µL)</TH></TR><TR><TD>dH<SUB>2</SUB>O</TD><TD>12.6</TD></TR><TR><TD>HF buffer</TD><TD>4</TD></TR><TR><TD>dNTP mix</TD><TD>0.4</TD></TR><TR><TD>10mM forward primer (bad)</TD><TD>1</TD></TR><TR><TD>10mM reverse primer (bad)</TD><TD>1</TD></TR><TR><TD>DMSO</TD><TD>0.6</TD></TR><TR><TD>Template DNA</TD><TD>0.211 (10ng x 1¬µL/47.5ng = 0.211¬µL) -from 4x dilute of miniprep</TD></TR><TR><TD>phusion</TD><TD>0.2</TD></TR></TBODY></TABLE><H4>Cyma (Basic) Tube:</H4><TABLE><TBODY><TR><TH>Component</TH><TH>Amount (¬µL)</TH></TR><TR><TD>dH<SUB>2</SUB>O</TD><TD>12.6</TD></TR><TR><TD>HF buffer</TD><TD>4</TD></TR><TR><TD>dNTP mix</TD><TD>0.4</TD></TR><TR><TD>10mM forward primer (bad)</TD><TD>1</TD></TR><TR><TD>10mM reverse primer (bad)</TD><TD>1</TD></TR><TR><TD>DMSO</TD><TD>0.6</TD></TR><TR><TD>Template DNA (FadK PCR0)</TD><TD>0.249 (10ng x 1¬µL/120.6ng = 0.249¬µL) -from 3x dilute of miniprep</TD></TR><TR><TD>phusion</TD><TD>0.2</TD></TR></TBODY></TABLE><UL><LI>All components were kept on ice</LI><LI>Used lambda PCR cleanup kit</LI></UL><DIV class="notebookdate"><H4>August 1, 2018</H4><H4>Gel</H4><H4>Peter</H4></DIV><TABLE><TBODY><TR><TH>Lane</TH><TH>1</TH><TH>2</TH><TH>3</TH><TH>4</TH><TH>5</TH><TH>6</TH><TH>7</TH><TH>8</TH></TR><TR><TH>Item</TH><TD>K86440 0(pTac)</TD><TD>Basic CymA (pcr purif.)</TD><TD>Basic ‚ÄòtesA (L109P)</TD><TD>2-log ladder</TD><TD>Failed Lane</TD><TD>Failed Lane</TD><TD>phz 4F-6R (pcr purif.)</TD><TD>mtrCAB new st. 3A</TD></TR></TBODY></TABLE><DIV class="notebookdate"><H4>August 1, 2018</H4><H4>So Many Things</H4><H4>Sricharan</H4></DIV><UL><LI>Miniprepping Peter‚Äôs OCs of MtrCAB (3A Assembled w/ new start) -&gt;3 colonies picked, 3 OCs</LI><LI>Minipreps done with Zymo Plasmid MP Kit</LI><LI>BBa_K864400 (pTac) is 1 OC</LI></UL><UL><LI>250¬µL of each on ice, ready to add glycerol once we know which ones are good size‚Ä¶ added glycerol</LI></UL><H4>Lots of Digests (all in CutSmart buffer, all volumes in ¬µL):</H4><TABLE><TBODY><TR><TH>Sample</TH><TH>mtrCAB new st (1-3)</TH><TH>K864400 (4)</TH><TH>ccmA-H (5)</TH><TH>fadK (6)</TH><TH>cymA Basic (7)</TH><TH>‚ÄòtesA Basic (8)</TH></TR><TR><TH>dH<SUB>2</SUB>0</TH><TD>0</TD><TD>0</TD><TD>11</TD><TD>11</TD><TD>0</TD><TD>0</TD></TR><TR><TH>Buffer</TH><TD>2</TD><TD>2</TD><TD>2</TD><TD>2</TD><TD>2</TD><TD>2</TD></TR><TR><TH>DNA</TH><TD>16</TD><TD>16</TD><TD>5</TD><TD>5</TD><TD>16</TD><TD>16</TD></TR><TR><TH>Enzyme</TH><TD>1 XbaI
1 PstI</TD><TD>1 EcoRI
1 SpeI</TD><TD>1 EcoRI
1 SpeI</TD><TD>1 XbaI
1 PstI</TD><TD>1 EcoRI
1 PstI</TD><TD>1 EcoRI
1 PstI</TD></TR></TBODY></TABLE><UL><LI>1-4 by PC, 5-8 by SK</LI></UL><P>3A Assemblies ‚Ä¶ Using NEB Ligation Calculator for amounts, 1:1:1 molar ratios</P><H4>1) Anderson (J23110) + cymA + pSB1K3</H4><TABLE><TBODY><TR><TH>Component</TH><TH>Mass (ng)</TH><TH>Volume (¬µL)</TH><TH>Notes</TH></TR><TR><TD>pSB1K3 (SK 7/30)</TD><TD>25</TD><TD>1.33</TD><TD>Digested 7/30</TD></TR><TR><TD>J23110 Digest (LCC 7/26)</TD><TD>33.84</TD><TD>6.77</TD><TD>2983 bp</TD></TR><TR><TD>cymA Digest (LCC 7/26)</TD><TD>32.25</TD><TD>6.45</TD><TD>2843 bp</TD></TR><TR><TD>T4 Ligase Buffer</TD><TD>-</TD><TD>2</TD><TD>-</TD></TR><TR><TD>T4 Ligase</TD><TD>-</TD><TD>1</TD><TD>-</TD></TR><TR><TD>dH<SUB>2</SUB>O</TD><TD>-</TD><TD>2.45</TD><TD>-</TD></TR></TBODY></TABLE><H4>2) Anderson (J23110) + mtrCAB w/new st + pSB1C3</H4><TABLE><TBODY><TR><TH>Component</TH><TH>Mass (ng)</TH><TH>Volume (¬µL)</TH><TH>Notes</TH></TR><TR><TD>pSB1C3 (SK 7/30)</TD><TD>25</TD><TD>0.68</TD><TD>Digested 7/30</TD></TR><TR><TD>J23110 Digest (LCC 7/26)</TD><TD>36.03</TD><TD>7.21</TD><TD>-</TD></TR><TR><TD>mtrCAB w/new st Digest (8/1)</TD><TD>89.23</TD><TD>2.85</TD><TD>&gt; 388 bp</TD></TR><TR><TD>T4 Ligase Buffer</TD><TD>-</TD><TD>2</TD><TD>-</TD></TR><TR><TD>T4 Ligase</TD><TD>-</TD><TD>1</TD><TD>-</TD></TR><TR><TD>dH<SUB>2</SUB>O</TD><TD>-</TD><TD>6.26</TD><TD>-</TD></TR></TBODY></TABLE><H4>3) ccmA-H (K917006) + B0015 + pSB1K3 ‚Ä¶ 10¬µL rxn</H4><TABLE><TBODY><TR><TH>Component</TH><TH>Mass (ng)</TH><TH>Volume (¬µL)</TH><TH>Notes</TH></TR><TR><TD>pSB1K3 (SK 7/30)</TD><TD>25</TD><TD>1.33</TD><TD>2204 bp</TD></TR><TR><TD>B0015 Digest (LCC 7/24)</TD><TD>24.94</TD><TD>4.99</TD><TD>2199 bp</TD></TR><TR><TD>ccmA-H Digest (SK 8/1)</TD><TD>96.84</TD><TD>2.2</TD><TD>8537 bp</TD></TR><TR><TD>T4 Ligase Buffer</TD><TD>-</TD><TD>1</TD><TD>-</TD></TR><TR><TD>T4 Ligase</TD><TD>-</TD><TD>0.5</TD><TD>-</TD></TR><TR><TD>dH<SUB>2</SUB>O</TD><TD>-</TD><TD>0</TD><TD>-</TD></TR></TBODY></TABLE><H4>pTac (K864400) + fadK + pSB1K3 ‚Ä¶ 10¬µL rxn</H4><TABLE><TBODY><TR><TH>Component</TH><TH>Mass (ng)</TH><TH>Volume (¬µL)</TH><TH>Notes</TH></TR><TR><TD>pSB1K3 (SK 7/30)</TD><TD>25</TD><TD>1.33</TD><TD>2204 bp</TD></TR><TR><TD>pTac Digest (PC 8/1)</TD><TD>24.17</TD><TD>0.27</TD><TD>2131 bp</TD></TR><TR><TD>fadK Digest (SK 8/1)</TD><TD>43</TD><TD>0.73</TD><TD>3717 bp</TD></TR><TR><TD>T4 Ligase Buffer</TD><TD>-</TD><TD>1</TD><TD>-</TD></TR><TR><TD>T4 Ligase</TD><TD>-</TD><TD>0.5</TD><TD>-</TD></TR><TR><TD>dH<SUB>2</SUB>O</TD><TD>-</TD><TD>6.17</TD><TD>-</TD></TR></TBODY></TABLE><DIV class="notebookdate"><H4>August 2, 2018</H4><H4>Planning Gibsons for Tomorrow</H4><H4>Sricharan</H4></DIV><UL><LI>phz (2x insert excess)</LI><LI>Note for all the Gibsons we‚Äôre using new gBlocks that have revised short and end sequences designed to work well w/ pSB1C3 ‚Ä¶ using NEB calculator.</LI></UL><H4>Fragments with identifying attributes and amounts listed below:</H4><TABLE><TBODY><TR><TH>pSB1C3 (SK 7/30)</TH><TH>phz Start 7.27.2018</TH><TH>1F-3R (7/27)</TH><TH>phz3</TH><TH>4F-6R (8/1)</TH><TH>phz End 7.27.2018</TH></TR><TR><TD>50ng</TD><TD>gBlock, 10 ng/¬µL</TD><TD>24.6 ng/¬µL</TD><TD>gBlock, 10ng/¬µL</TD><TD>95.3 ng/¬µL</TD><TD>gBlock, 10ng/¬µL</TD></TR><TR><TD>20¬µL/375ng</TD><TD>426 bp</TD><TD>3072 bp</TD><TD>222 bp</TD><TD>2588 bp</TD><TD>277 bp</TD></TR><TR><TD>2.66¬µL</TD><TD>20.6ng</TD><TD>148 ng</TD><TD>10.7ng</TD><TD>125 ng</TD><TD>13.4 ng</TD></TR><TR><TD>0.037pmol</TD><TD>2.06¬µL</TD><TD>6.02¬µL</TD><TD>1.07¬µL</TD><TD>1.31 ¬µL</TD><TD>1.34 ¬µL</TD></TR></TBODY></TABLE><UL><LI>14.46¬µL total, add equivalent amt of NEB Gibson Master Mix</LI></UL><UL><LI>We‚Äôll also do a ctrl w/ 50ng plasmid only w/ rest dH2O and NEB MMix</LI><LI>acr (2x insert excess)</LI></UL><H4>Fragments with identifying attributes and amounts listed below:</H4><TABLE><TBODY><TR><TH>pSB1C3 (SK 7/30)</TH><TH>acr1 7.27.2018</TH><TH>acr2</TH><TH>acr3 7.27.2018</TH></TR><TR><TD>25ng</TD><TD>gBlock, 10ng/¬µL</TD><TD>gBlock, 10ng/¬µL</TD><TD>gBlock, 10ng/¬µL</TD></TR><TR><TD>1.33¬µL</TD><TD>512 bp</TD><TD>529 bp</TD><TD>653 bp</TD></TR><TR><TD>12.4ng</TD><TD>12.8ng</TD><TD>15.8ng</TD></TR><TR><TD>1.24¬µL</TD><TD>1.28¬µL</TD><TD>1.58¬µL</TD></TR></TBODY></TABLE><UL><LI>5.43¬µL total, add equivalent amt of SGI Gibson MMix</LI></UL><P>pmt1231</P><H4>Fragments with identifying attributes and amounts listed below:</H4><TABLE><TBODY><TR><TH>pSB1C3 (SK 7/30)</TH><TH>pmt1231.1</TH><TH>pmt1231.2</TH></TR><TR><TD>25ng</TD><TD>gBlock, 10ng/¬µL</TD><TD>gBlock, 10ng/¬µL</TD></TR><TR><TD>1.33¬µL</TD><TD>532 bp</TD><TD>512 bp</TD></TR><TR><TD>12.9ng</TD><TD>12.4ng</TD></TR><TR><TD>1.29¬µL</TD><TD>1.24¬µL</TD></TR></TBODY></TABLE><UL><LI>3.84¬µL total, add 1.14¬µL dH<SUB>2</SUB>O and 5¬µL SGI Gibson MMix</LI></UL><UL><LI>Also doing control with 25ng plamsid only and rest dH2O and SGI MMix</LI><LI>(1.33¬µL pSB1C3, 3.67¬µL dH<SUB>2</SUB>O, 5¬µL SGI MMix)</LI></UL><DIV class="notebookdate"><H4>August 2, 2018</H4><H4>Transformation (RbCl JM109) of 3A‚Äôs from 8/1</H4><H4>Peter</H4></DIV><P>**See Protocol from 7/16/18**</P><H4>Transforming:</H4><UL><LI>CymA + J23110 + PSB1K3 (8/1 SK)</LI><LI>mtrCAB w/ new st. + J23110 + PSB1C3 (8/1 SK)</LI><LI>ccmA-H + B0015 + PSB1K3 (8/1 SK)</LI><LI>fadK + pTac + PSB1K3 (8/1 SK)</LI><LI>fadD ligation to PSB1C3 (7/24 LCC)</LI></UL><H4>Notes/ Deviations:</H4><UL><LI>Two tubes of cells/ transformation (to ensure one tube will have cells) so 0.5 ŒºL of DNA added to each tube. Exceptions: 2 x 1ŒºL for CymA and Mtr (from 20 ŒºL 3A so more dilute than std 10 ŒºL 3A); 2 x 1ŒºL for fadD (hasn‚Äôt worked yet so adding a lot of DNA)</LI><LI>Heat-shocked for 30 seconds</LI><LI>Pooled like tubes (same vector) after 1 hour incubation</LI><LI>Plated with spinner and flame</LI></UL><H4>Results (8/3/18):</H4><TABLE><TBODY><TR><TH>Part/Vector</TH><TH>Transformation Result</TH></TR><TR><TD>CymA + J23110 + PSB1K3 (8/1 SK)</TD><TD>Good growth (O.C. on 8/3)</TD></TR><TR><TD>mtrCAB w/ new st. + J23110 + PSB1C3 (8/1 SK)</TD><TD>No growth</TD></TR><TR><TD>ccmA-H + B0015 + PSB1K3 (8/1 SK)</TD><TD>Good growth (O.C. on 8/3)</TD></TR><TR><TD>fadK + pTac + PSB1K3 (8/1 SK)</TD><TD>~32 colonies (O.C. on 8/3)</TD></TR><TR><TD>FadD (from ligation + now purified w/ PCR purif)</TD><TD>3 colonies (O.C. on 8/3)</TD></TR></TBODY></TABLE><DIV class="notebookdate"><H4>August 3, 2018</H4><H4>Gibsons</H4><H4>Sricharan</H4></DIV><UL><LI>Set up Gibsons as detailed 8/2/2018</LI><LI>Set @ 50¬∞C 1 hr</LI><UL><LI>4¬∞C infinite hold</LI></UL><LI>In thermal cycler (2 different cyclers for differing rxn volumes)</LI></UL></DIV><H3>August</H3><BUTTON class="collapsible">Week 13: 8/6/2018</BUTTON><DIV class="content"><DIV class="notebookdate"><H4>August 2, 2018</H4><H4>Transformation (RbCl JM109) of 3A‚Äôs from 8/1</H4><H4>Peter</H4></DIV><P>**See Protocol from 7/16/18**</P><H4>Transforming:</H4><UL><LI>mtrCAB w/ new st. + J23110 + PSB1C3 (8/1 SK)</LI><LI>Phz Gibson (NEB) w/ redesigned start and end (8/3 SK)</LI><LI>Acr Gibson (SGI) w/ redesigned start and end (8/3 SK)</LI><LI>Pmt1231 Gibson (SGI) w/ redesigned start and end (8/3 SK)</LI><LI>NEB PSB1C3 ctrl</LI><LI>SGI PSB1C3 ctrl</LI></UL><P>Notes/ Deviations:</P><UL><LI>Two tubes of cells/ transformation (to ensure one tube will have cells) so 0.5 ŒºL of DNA added to each tube. Exceptions: 2 x 1ŒºL for Mtr (from 20 ŒºL 3A so more dilute than std 10 ŒºL 3A)</LI><LI>Heat-shocked for 30 seconds</LI><LI>Pooled like tubes (same vector) after 1 hour incubation</LI><LI>CAM plates for all</LI></UL><H4>Results (8/7/18):</H4><TABLE><TBODY><TR><TH>Part/Vector</TH><TH>Transformation Result</TH></TR><TR><TD>mtrCAB w/ new st. + J23110 + PSB1C3 (8/1 SK)</TD><TD>No growth</TD></TR><TR><TD>Phz Gibson (NEB) w/ redesigned start and end (8/3 SK)</TD><TD>No growth</TD></TR><TR><TD>Acr Gibson (NEB) w/ redesigned start and end (8/3 SK)</TD><TD>No growth</TD></TR><TR><TD>Pmt1231 Gibson (SGI) w/ redesigned start and end (8/3 SK)</TD><TD>5-10 colonies (O.C. on 8/8)</TD></TR><TR><TD>NEB PSB1C3 ctrl</TD><TD>No growth</TD></TR><TR><TD>SGI PSB1C3 ctrl</TD><TD>No growth</TD></TR></TBODY></TABLE><DIV class="notebookdate"><H4>August 7, 2018</H4><H4>Minipreps of 3As and fadD + Gel</H4><H4>Sricharan</H4></DIV><P>Using Genscript Kit miniprepped:</P><UL><LI>tac+fadK 3A</LI><LI>cymA 3A (w/ med Anderson)</LI><LI>B0015+ccmA-H 3A</LI><LI>fadD</LI></UL><H4>Nanodropped, results below:</H4><TABLE><TBODY><TR><TH>Sample Name</TH><TH>Concentration (ng/ŒºL)</TH><TH>260/280</TH><TH>260/230</TH></TR><TR><TD>tac+fadK</TD><TD>4.08</TD><TD>1.51</TD><TD>1.16</TD></TR><TR><TD>cymA 3A</TD><TD>41.13</TD><TD>1.58</TD><TD>0.69</TD></TR><TR><TD>B0015+ccmA-H</TD><TD>59.52</TD><TD>1.61</TD><TD>0.77</TD></TR><TR><TD>fadD</TD><TD>156.54</TD><TD>1.82</TD><TD>1.90</TD></TR></TBODY></TABLE><H4>Diagnostic digests of these items, with</H4><TABLE><TBODY><TR><TD>dH<SUB>2</SUB>O</TD><TD>0ŒºL</TD></TR><TR><TD>CutSmart</TD><TD>2ŒºL</TD></TR><TR><TD>DNA</TD><TD>17ŒºL</TD></TR><TR><TD>EcoRI</TD><TD>1ŒºL</TD></TR></TBODY></TABLE><DIV class="notebookdate"><H4>August 7, 2018</H4><H4>Gel of a bunch of things</H4><H4>Liz</H4></DIV><P>Made a 1% agarose gel</P><TABLE><TBODY><TR><TH>1</TH><TH>2</TH><TH>3</TH><TH>4</TH><TH>5</TH></TR><TR><TD>Ladder (6ŒºL)</TD><TD>Tac + FadK EcoRI digest (5ŒºL sample + 1ŒºL digest)</TD><TD>cymA 3A EcorI digest (5ŒºL sample + 1ŒºL digest)</TD><TD>ccmA-H + B0015 3A EcoRI digest ((5ŒºL sample + 1ŒºL digest)</TD><TD>fadD EcoRI digest (5ŒºL sample + 1ŒºL digest)</TD></TR></TBODY></TABLE><P>Only lane 5 (fadD) had a band</P><DIV class="notebookdate"><H4>August 8, 2018</H4><H4>Digests for 3A Assemblies</H4><H4>Liz, Peter</H4></DIV><P>Double digests of pSB1A3 and J23114</P><H4>pSB1A3:</H4><TABLE><TBODY><TR><TD>cutsmart</TD><TD>2ŒºL</TD></TR><TR><TD>DNA</TD><TD>15ŒºL</TD></TR><TR><TD>EcoRI</TD><TD>1ŒºL</TD></TR><TR><TD>PstI</TD><TD>1ŒºL</TD></TR><TR><TD>DpnI</TD><TD>1ŒºL</TD></TR></TBODY></TABLE><P>DNA from iGEM linearized plasmid backbone</P><P>In retrospect this is bad because need water in the digestion</P><H4>J23114:</H4><TABLE><TBODY><TR><TD>cutsmart</TD><TD>2ŒºL</TD></TR><TR><TD>DNA (J23114 miniprep SD 7/25)</TD><TD>10ŒºL</TD></TR><TR><TD>EcoRI</TD><TD>1ŒºL</TD></TR><TR><TD>SpeI</TD><TD>1ŒºL</TD></TR><TR><TD>water</TD><TD>6ŒºL</TD></TR></TBODY></TABLE><UL><LI>Heat blocked at 37C for 15 min</LI><LI>Heat blocked at 80C for 15 min</LI><LI>Assembly part is in Peter‚Äôs notebook</LI></UL><DIV class="notebookdate"><H4>August 8, 2018</H4><H4>3A Assemblies</H4><H4>Peter</H4></DIV><P>***Using NEB Ligation Calculator for amounts, 1:1:1 molar ratios***</P><H4>Protocol:</H4><UL><LI>Add components in following order: water, buffer, DNA, enzymes</LI><LI>Let sit at room temp for 10 minutes</LI><LI>Heat block @ 65 C for 20 minutes</LI></UL><H4>mtrCAB w/ new st. + Anderson (J23114) + pSB1C3</H4><TABLE><TBODY><TR><TH>Component</TH><TH>Mass (ng)</TH><TH>Volume (¬µL)</TH><TH>Notes</TH></TR><TR><TD>pSB1C3</TD><TD>25</TD><TD>0.68</TD><TD>(7/30 SK)</TD></TR><TR><TD>J23114 Digest</TD><TD>36.03</TD><TD>0.21 (should have been 0.33)</TD><TD>(8/8 LCC)</TD></TR><TR><TD>mtrCAB w/ new st.</TD><TD>89.58</TD><TD>2.86</TD><TD>(8/1 SK)</TD></TR><TR><TD>T4 Ligase Buffer</TD><TD>-</TD><TD>2</TD><TD>-</TD></TR><TR><TD>T4 Ligase</TD><TD>-</TD><TD>1</TD><TD>-</TD></TR><TR><TD>dH<SUB>2</SUB>O</TD><TD>-</TD><TD>3.26</TD><TD>-</TD></TR></TBODY></TABLE><H4>mtrCAB w/ new st. + pTac (K864400) + pSB1A3</H4><TABLE><TBODY><TR><TH>Component</TH><TH>Mass (ng)</TH><TH>Volume (¬µL)</TH><TH>Notes</TH></TR><TR><TD>pSB1A3</TD><TD>25</TD><TD>1.25</TD><TD>(8/8 LCC)</TD></TR><TR><TD>pTac (K864400)</TD><TD>24.72</TD><TD>0.27</TD><TD>(SK 8/1)</TD></TR><TR><TD>mtrCAB w/ new st</TD><TD>2.74</TD><TD>2.86</TD><TD>(SK 8/1)</TD></TR><TR><TD>T4 Ligase Buffer</TD><TD>-</TD><TD>2</TD><TD>-</TD></TR><TR><TD>T4 Ligase</TD><TD>-</TD><TD>1</TD><TD>-</TD></TR><TR><TD>dH<SUB>2</SUB>O</TD><TD>-</TD><TD>2.74</TD><TD>-</TD></TR></TBODY></TABLE><DIV class="notebookdate"><H4>August 8, 2018</H4><H4>Transformations of Gibsons and 3A Assemblies</H4><H4>Liz and Sricharan</H4></DIV><TABLE><TBODY><TR><TH>1</TH><TH>2</TH><TH>3</TH><TH>4</TH><TH>5</TH><TH>6</TH><TH>7</TH></TR><TR><TD>NEB control gibson pSB1C3 (SK 8/3)</TD><TD>SGI control gibson pSB1C3 (SK 8/3)</TD><TD>Acr Gibson (SK 8/3)</TD><TD>Phz Gibson (SK 8/3)</TD><TD>mtrCAB with new start and J23114 pSB1C3 (PC 8/8)</TD><TD>mtrCAB with new start and pTac (K864400) in pSB1A3 (PC 8/8)</TD><TD>mtrCAB with new start and J23110 in pSB1C3 (SK 8/1)</TD></TR></TBODY></TABLE><UL><LI>2 tubes per sample, 1ŒºL DNA in each tube</LI><LI>Followed transformation protocol (iGEM)</LI></UL><DIV class="notebookdate"><H4>August 8, 2018</H4><H4>Colony PCR Checks of 3A Assemblies</H4><H4>Liz and Sricharan</H4></DIV><P>1st Thing: Setting up PCR mixes ‚Ä¶ Fprimer = VF2, Rprimer = VR @ 10ŒºM</P><H4>Taq PCRs</H4><TABLE><TBODY><TR><TH>Item</TH><TH>GoTaq (stuff &lt; 3kb) ‚Ä¶ 100ŒºL rxn</TH></TR><TR><TD>5x Buffer</TD><TD>20ŒºL</TD></TR><TR><TD>10mM dNTPs</TD><TD>2ŒºL</TD></TR><TR><TD>F Primer</TD><TD>5ŒºL</TD></TR><TR><TD>R Primer</TD><TD>5ŒºL</TD></TR><TR><TD>dH<SUB>2</SUB>O</TD><TD>62.5ŒºL</TD></TR><TR><TD>GoTaq Polymerase</TD><TD>0.5ŒºL</TD></TR><TR><TD>Colony Suspensionr</TD><TD>5ŒºL</TD></TR></TBODY></TABLE><TABLE><TBODY><TR><TH>Temperature (¬∞C)</TH><TH>Time</TH></TR><TR><TH>Initial Denaturation</TH><TD>95</TD><TD>5min</TD></TR><TR><TH>35 Cycles</TH><TD>95</TD><TD>45sec</TD></TR><TR><TD>65</TD><TD>45sec</TD></TR><TR><TD>72</TD><TD>2.5min</TD></TR><TR><TH>Final Extension</TH><TD>72</TD><TD>5min</TD></TR><TR><TH>Soak</TH><TD>4</TD><TD>Infinite</TD></TR></TBODY></TABLE><H4>Phusion PCRs</H4><TABLE><TBODY><TR><TH>Item</TH><TH>NEB Phusion (ccm) ‚Ä¶ 50ŒºL</TH></TR><TR><TD>5x Buffer</TD><TD>10ŒºL</TD></TR><TR><TD>10mM dNTPs</TD><TD>1ŒºL</TD></TR><TR><TD>F Primer</TD><TD>2.5ŒºL</TD></TR><TR><TD>R Primer</TD><TD>2.5ŒºL</TD></TR><TR><TD>dH<SUB>2</SUB>O</TD><TD>31ŒºL</TD></TR><TR><TD>GoTaq Polymerase</TD><TD>0.5ŒºL</TD></TR><TR><TD>Colony Suspension</TD><TD>2.5ŒºL</TD></TR></TBODY></TABLE><TABLE><TBODY><TR><TH>Temperature (¬∞C)</TH><TH>Time</TH></TR><TR><TH>Initial Denaturation</TH><TD>98</TD><TD>5min</TD></TR><TR><TH>35 Cycles</TH><TD>98</TD><TD>10sec</TD></TR><TR><TD>63</TD><TD>30sec</TD></TR><TR><TD>72</TD><TD>2:10min</TD></TR><TR><TH>Final Extension</TH><TD>72</TD><TD>10min</TD></TR><TR><TH>Soak</TH><TD>4</TD><TD>Infinite</TD></TR></TBODY></TABLE><P>Picked 4 colonies of each 3A assembly (B0015+ccmA-H, tac+fadK, med Anderson+cymA)</P><P>Put tip in 20ŒºL dH2O, pipetted up and down, spotted 3ŒºL onto marked plate (Cam), incubated overnight ‚Ä¶ Later realized that these plasmids are kanamycin resistant</P><P>Put 0.5ŒºL w/9.5ŒºL of master mix above ‚Ä¶ ran cycler</P><P>Will run gel to check which ones are right</P><DIV class="notebookdate"><H4>August 8, 2018</H4><H4>Colony PCR Checks of 3A Assemblies</H4><H4>Sricharan and Liz and Shannon</H4></DIV><TABLE><TBODY><TR><TH>Lane</TH><TH>1</TH><TH>2</TH><TH>3</TH><TH>4</TH><TH>5</TH></TR><TR><TH>Item</TH><TD>2-log ladder</TD><TD>ccm colony 1</TD><TD>ccm colony 2</TD><TD>ccm colony 3</TD><TD>Phusion colony ctrl</TD></TR></TBODY></TABLE><UL><LI>Run @ 100V, 25min</LI><LI>No visible bands on this gel (?)</LI></UL><TABLE><TBODY><TR><TH>Lane</TH><TH>1</TH><TH>2</TH><TH>3</TH><TH>4</TH><TH>5</TH></TR><TR><TH>Item</TH><TD>2-log ladder</TD><TD>fadK colony 1</TD><TD>fadK colony 2</TD><TD>fadK colony 3</TD><TD>fadK colony 4</TD></TR></TBODY></TABLE><TABLE><TBODY><TR><TH>Lane</TH><TH>6</TH><TH>7</TH><TH>8</TH><TH>9</TH><TH>10</TH></TR><TR><TH>Item</TH><TD>cymA colony 1</TD><TD>cymA colony 2</TD><TD>cymA colony 3</TD><TD>cymA colony 4</TD><TD>GoTaq colony control</TD></TR></TBODY></TABLE><P>Run @ 100V, 25min</P><DIV class="notebookdate"><H4>August 9, 2018</H4><H4>Transformations &amp; OCs</H4><H4>Sricharan and Peter</H4></DIV><P>Usual PC protocol</P><UL><LI>acr Gibson 8/3 (Strataclone, 1 tube, 1)</LI><LI>phz Gibson 8/3 (Strataclone, 1 tube 1)</LI><LI>weak Anderson + mtrCAB revised 8/8</LI><LI>acr Gibson 8/3 (RbCl cells)</LI><LI>phz Gibson 8/3 (RbCl cells)</LI><LI>pTac + mtrCAB revised 8/8</LI></UL><P>3, 4, 5, 6 - 2 tubes RbCl cells (50ŒºL aliquots), 1ŒºL DNA each tube</P><P>OCs of following in Cam:</P><UL><LI>tac+fadK 3A colony PCR (8/8)</LI><LI>pmt1231 Gibson (8/6)</LI><LI>med Anderson + mtrCAB w/ new start (8/8)</LI></UL></DIV><BUTTON class="collapsible">Week 14: 8/13/2018</BUTTON><DIV class="content"><DIV class="notebookdate"><H4>August 13, 2018</H4><H4>Transformation of anderson, araC, and pBad/ araC promoters</H4><H4>Peter</H4></DIV><P>**See Protocol from 7/16/18**</P><H4>Transforming:</H4><UL><LI>BBa_J23112 (amp)</LI><LI>BBa_J23103 (amp)</LI><LI>BBa_J23109 (amp)</LI><LI>BBa_J23117 (amp)</LI><LI>BBa_R0080 (inducible araC)</LI><LI>BBa_I0500 (inducible pBad/ araC)</LI></UL><H4>Notes/Deviations:</H4><UL><LI>Two tubes of cells/ transformation (to ensure one tube will have cells) so 0.5 ŒºL of DNA added to each tube</LI><LI>Heat-shocked for 30 seconds</LI><LI>Pooled like tubes (same vector) after 1 hour incubation</LI></UL><DIV class="notebookdate"><H4>August 13, 2018</H4><H4>OCs</H4><H4>Sricharan</H4></DIV><UL><LI>OCs of pmt1231, acr, phz Gibsons - 2 colonies each</LI><LI>cymA, tac+fadK 3As - 3 colonies each</LI><LI>All OCs grew after 18 hours</LI></UL><DIV class="notebookdate"><H4>August 14, 2018</H4><H4>Making Ampicillin Plates + Diagnostic Digests</H4><H4>Sricharan</H4></DIV><P>From LCC‚Äôs MPs - Nanodropped - 3As are low concentration (~10-20ng/ŒºL, Gibsons high concentration ~100ng/ŒºL)</P><H4>For digests:</H4><TABLE><TBODY><TR><TH>Item</TH><TH>3A Assemblies</TH><TH>Gibsons</TH></TR><TR><TD>dH<SUB>2</SUB>O</TD><TD>0ŒºL</TD><TD>12ŒºL</TD></TR><TR><TD>CutSmart</TD><TD>2ŒºL</TD><TD>2ŒºL</TD></TR><TR><TD>DNA</TD><TD>17ŒºL</TD><TD>5ŒºL</TD></TR><TR><TD>EcoRI-HF</TD><TD>1ŒºL</TD><TD>1ŒºL</TD></TR></TBODY></TABLE><DIV class="notebookdate"><H4>August 14, 2018</H4><H4>Lots of minipreps</H4><H4>Liz</H4></DIV><P>Did 12 minipreps to digest and run on a gel</P><UL><LI>Pmt1231 (1 and 2)</LI><LI>Acr gibson (1 and 2)</LI><LI>Phz gibson</LI><LI>Tac and fadK 2A (1, 2, 3)</LI><LI>cymA 3A</LI></UL><P>Used GenScript kit and eluted in 30ŒºL elution buffer</P><P>Minipreps were digested and run on a gel </P><P>Only pmt1231 had a logical band</P><UL><LI>Made a glycerol stock with 0.75mL 40% glycerol and 0.75mL pmt overnight culture</LI></UL><DIV class="notebookdate"><H4>August 15, 2018</H4><H4>more minipreps</H4><H4>Liz</H4></DIV><P>Did minipreps of overnight cultures</P><UL><LI>BBa_R0080</LI><LI>BBa_J23103</LI></UL><P>Used GenScript kit</P><UL><LI>Eluted in 30ŒºL elution buffer</LI></UL><P>Nanodropped:</P><UL><LI>BBa_R0080: 113.97 ng/ŒºL</LI><LI>BBa_J23103: 307.66 ng/ŒºL</LI></UL><DIV class="notebookdate"><H4>August 15, 2018</H4><H4>Colony PCR Gel</H4><H4>Sricharan and Liz</H4></DIV><P>More colony PCRs using same protocol as some weeks before, this time using the correct antibiotic selection plate.</P><DIV class="notebookdate"><H4>August 16, 2018</H4><H4>Colony PCR Gel</H4><H4>Sricharan and Liz</H4></DIV><P>All samples had 5ŒºL DNA and 1ŒºL loading dye</P><H4>Gel 1:</H4><TABLE><TBODY><TR><TH>1</TH><TH>2</TH><TH>3</TH><TH>4</TH><TH>5</TH><TH>6</TH><TH>7</TH><TH>8</TH><TH>9</TH><TH>10</TH><TH>11</TH></TR><TR><TD>ladder</TD><TD>Mtr 1</TD><TD>Mtr 2</TD><TD>Mtr 3</TD><TD>Mtr 4</TD><TD>Mtr 5</TD><TD>Mtr 6</TD><TD>phz1</TD><TD>phz2</TD><TD>phz3</TD><TD>ladder</TD></TR><TR><TH>12</TH><TH>13</TH><TH>14</TH><TH>15</TH><TH>16</TH><TH>17</TH><TH>18</TH><TH>19</TH><TH>20</TH><TH>21</TH><TH>22</TH></TR><TR><TD>phz4</TD><TD>phz5</TD><TD>phz6</TD><TD>ccm1</TD><TD>ccm2</TD><TD>ccm3</TD><TD>ccm4</TD><TD>ccm5</TD><TD>ccm6</TD><TD>Kan phusion ctrl</TD><TD>ladder</TD></TR></TBODY></TABLE><H4>Gel 2:</H4><TABLE><TBODY><TR><TH>1</TH><TH>2</TH><TH>3</TH><TH>4</TH><TH>5</TH><TH>6</TH><TH>7</TH><TH>8</TH><TH>9</TH><TH>10</TH><TH>11</TH></TR><TR><TD>ladder</TD><TD>fadK1</TD><TD>fadK2</TD><TD>fadK3</TD><TD>fadK4</TD><TD>fadK5</TD><TD>fadK6</TD><TD>acr1</TD><TD>acr2</TD><TD>Kan ctrl gotaq</TD><TD>ladder</TD></TR><TR><TH>12</TH><TH>13</TH><TH>14</TH><TH>15</TH><TH>16</TH><TH>17</TH><TH>18</TH><TH>19</TH><TH>20</TH><TH>21</TH><TH>22</TH></TR><TR><TD>Cam ctrl gotaq</TD><TD>empty</TD><TD>cymA1</TD><TD>cymA2</TD><TD>cymA3</TD><TD>cymA4</TD><TD>cymA5</TD><TD>cymA6</TD><TD>Cam ctrl phusion</TD><TD>ladder</TD><TD>empty</TD></TR></TBODY></TABLE><DIV class="notebookdate"><H4>August 16, 2018</H4><H4>phz Linear Gibson PCR</H4><H4>Liz</H4></DIV><H4>Thermocycler</H4><TABLE><TBODY><TR><TH>time</TH><TH>Temperature (C)</TH></TR><TR><TH>Initial denaturation</TH><TD>30 s</TD><TD>98</TD></TR><TR><TH>35 cycles</TH><TD>10 s</TD><TD>98</TD></TR><TR><TD>20 s</TD><TD>64</TD></TR><TR><TD>2:10 (min:sec)</TD><TD>72</TD></TR><TR><TH>Final extension</TH><TD>10 min</TD><TD>72</TD></TR><TR><TH>hold</TH><TD>infinite</TD><TD>4</TD></TR></TBODY></TABLE><H4>PCR tube</H4><TABLE><TBODY><TR><TD>Phusion HF buffer</TD><TD>4ŒºL</TD></TR><TR><TD>DMSO</TD><TD>0.6ŒºL</TD></TR><TR><TD>10mM 1F primer</TD><TD>1ŒºL</TD></TR><TR><TD>10mM 6F primer</TD><TD>1ŒºL</TD></TR><TR><TD>dNTPs</TD><TD>0.4ŒºL</TD></TR><TR><TD>Template DNA (used SK linear gibson)</TD><TD>1ŒºL</TD></TR><TR><TD>phusion</TD><TD>0.2ŒºL</TD></TR></TBODY></TABLE></DIV><BUTTON class="collapsible">Week 15: 8/20/2018</BUTTON><DIV class="content"><DIV class="notebookdate"><H4>August 24, 2018</H4><H4>Restriction Digests for 3A Assemblies and 3A Assembly</H4><H4>Liz</H4></DIV><P>cymA: cymA + J23110 (medium Anderson promoter) + pSB1K3</P><H4>J23110</H4><TABLE><TBODY><TR><TD>water</TD><TD>13.79ŒºL</TD></TR><TR><TD>CutSmart</TD><TD>2ŒºL</TD></TR><TR><TD>DNA (J23110 miniprep 7/27 LCC)</TD><TD>300ng = 2.21ŒºL</TD></TR><TR><TD>EcoRI</TD><TD>1ŒºL</TD></TR><TR><TD>SpeI</TD><TD>1ŒºL</TD></TR></TBODY></TABLE><H4>cymA</H4><TABLE><TBODY><TR><TD>water</TD><TD>13.51ŒºL</TD></TR><TR><TD>CutSmart</TD><TD>2ŒºL</TD></TR><TR><TD>DNA (cymA miniprep SD 7/25)</TD><TD>300ng = 2.49ŒºL</TD></TR><TR><TD>EcoRI</TD><TD>1ŒºL</TD></TR><TR><TD>PstI</TD><TD>1ŒºL</TD></TR></TBODY></TABLE><H4>pSB1K3</H4><TABLE><TBODY><TR><TD>water</TD><TD>4ŒºL</TD></TR><TR><TD>CutSmart</TD><TD>2ŒºL</TD></TR><TR><TD>DNA (J23110 miniprep 7/27 LCC)</TD><TD>300ng = 12ŒºL</TD></TR><TR><TD>EcoRI</TD><TD>1ŒºL</TD></TR><TR><TD>PstI</TD><TD>1ŒºL</TD></TR></TBODY></TABLE><UL><LI>Put in heat block at 37C for 15 min</LI><LI>Heat killed at 80C for 15 min</LI></UL><P>Forgot that backbone also needs to be digested with DpnI so cooled, added 1ŒºL DpnI to pSB1K3, heatblocked at 37C for 15 min, and heat killed at 80C for 15 min</P><H4>Used digestions in 3A assembly: 
cymA + J23110 + pSB1K3 (used open wetware protocol)
</H4><TABLE><TBODY><TR><TD>water</TD><TD>2.17ŒºL</TD></TR><TR><TD>Digested pSB1K3</TD><TD>10ng = 0.7ŒºL</TD></TR><TR><TD>Digested cymA</TD><TD>37.02ng = 2.45ŒºL</TD></TR><TR><TD>Digested J23110</TD><TD>47.75ng = 3.18ŒºL</TD></TR><TR><TD>T4 ligase buffer</TD><TD>1ŒºL</TD></TR><TR><TD>T4 ligase</TD><TD>0.5ŒºL</TD></TR></TBODY></TABLE><P>Thermocycler:</P><UL><LI>22.5C for 30 min</LI><LI>65C for 10 min</LI></UL><DIV class="notebookdate"><H4>August 24, 2018</H4><H4>A lot of colony PCR fails and Linear Gibson</H4><H4>Sricharan</H4></DIV><P>A new Linear Gibson of phz 1F-6R</P><TABLE><TBODY><TR><TD>5ŒºL</TD><TD>NEB MMix</TD></TR><TR><TD>3.01ŒºL</TD><TD>1F-3R (7/27)</TD></TR><TR><TD>0.53ŒºL</TD><TD>phz3 (gBlock)</TD></TR><TR><TD>0.66ŒºL</TD><TD>4F-6R (8/1)</TD></TR><TR><TD>0.80ŒºL</TD><TD>dH<SUB>2</SUB>O</TD></TR></TBODY></TABLE><P>Running PCR on this Gibson (Phusion) using usual protocol, and using 1ŒºL of template. Annealing temperature is 64 degrees C, extension time is 2min,30sec.</P></DIV><BUTTON class="collapsible">Week 16: 8/27/2018</BUTTON><DIV class="content"><DIV class="notebookdate"><H4>August 27, 2018</H4><H4>cymA 3A gel</H4><H4>Sricharan and Peter</H4></DIV><P>Changing some items to potentially have success in Gibson reactions:</P><P>Also redesigned inserts so that there are correct overlaps with pSB1C3</P><H4>Using 0.05pmol backbone, equimolar inserts</H4><TABLE><TBODY><TR><TH>pSB1C3 (SK 7/30)</TH><TH>acr1 7.27.2018</TH><TH>acr2</TH><TH>acr3 7.27.2018</TH></TR><TR><TD>3.57ŒºL</TD><TD>gBlock (532bp)</TD><TD>gBlock (529bp)</TD><TD>gBlock (653bp)</TD></TR><TR><TD>67ng</TD><TD>16.57ng</TD><TD>17.12ng</TD><TD>21.14ng</TD></TR><TR><TD>2070bp</TD><TD>1.66ŒºL</TD><TD>1.71ŒºL</TD><TD>2.11ŒºL</TD></TR></TBODY></TABLE><TABLE><TBODY><TR><TH>pSB1C3 (SK 7/30)</TH><TH>phz Start 7.27.2018</TH><TH>phz 1F-6R</TH><TH>phz End 7.27.2018</TH></TR><TR><TD>3.57ŒºL</TD><TD>gBlock (426bp)</TD><TD>gBlock (5882bp)</TD><TD>gBlock (277bp)</TD></TR><TR><TD>67ng</TD><TD>13.79ng</TD><TD>190.4ng</TD><TD>8.97ng</TD></TR><TR><TD>2070bp</TD><TD>1.38ŒºL</TD><TD>2.73ŒºL</TD><TD>0.90ŒºL</TD></TR></TBODY></TABLE><P>For actual reactions:</P><UL><LI>Cutting reaction volume in half (5ŒºL Master mix, 5ŒºL DNA + water)</LI></UL><DIV class="notebookdate"><H4>August 27, 2018</H4><H4>phz 1F-6R Gel</H4><H4>Liz</H4></DIV><P>Made a 1% agarose gel for linear phz 1F-6R gibson PCR</P><P>Purified linear 1F-6R gibson PCR (SK 8/24) using lambda kit</P><P>Nanodropped: 69.7 ng/ŒºL</P><H4>Gel:</H4><TABLE><TBODY><TR><TH>1</TH><TH>2</TH><TH>3</TH><TH>4</TH></TR><TR><TD>ladder</TD><TD>Phz gibson pure PCR</TD><TD>pSB1C3 A</TD><TD>pSB1C3 B</TD></TR><TR><TD>5ŒºL sample + 1ŒºL dye</TD><TD>1ŒºL dye+ 1ŒºL sample + 4ŒºL water</TD></TR></TBODY></TABLE><DIV class="notebookdate"><H4>August 28, 2018</H4><H4>Linear Gibson PCR + Making more Backbone</H4><H4>Sricharan</H4></DIV><UL><LI>Running Phusion PCR on phz 1F-6R (before the PCR was ~1kb, should have been ~6kb), pSB1C3</LI></UL><UL><LI>Usual protocol: for phz used 0.4ŒºL sample in 20ŒºL rxn, annealing at 64 degrees C, extension 2 minutes. Did PCR in HF and GC Buffers</LI><LI>For pSB1C3, used 0.2ŒºL of Peter‚Äôs diluted J04450 MP (7/24) to 0.14ng/ŒºL (100x). This was a 50ŒºL rxn, annealing at 71 degrees C, 40s extension.</LI></UL><UL><LI>PCR purified on 8/28 by Sricharan</LI></UL><H4>Nanodrop</H4><TABLE><TBODY><TR><TH>phz HF</TH><TH>phz GC</TH><TH>pSB1C3 HF</TH></TR><TR><TD>Conc (ng/ŒºL)</TD><TD>15.1</TD><TD>21.7</TD><TD>39.5</TD></TR><TR><TD>260/280</TD><TD>1.43</TD><TD>1.63</TD><TD>1.71</TD></TR><TR><TD>260/230</TD><TD>-0.27</TD><TD>-0.48</TD><TD>-0.49</TD></TR></TBODY></TABLE><UL><LI>Curves look like junk for all 3</LI><LI>Peter has gel on pg 49</LI></UL><DIV class="notebookdate"><H4>August 28, 2018</H4><H4>Gel of Phz (1F-6R) pcr‚Äôs and pSB1C3</H4><H4>Peter</H4></DIV><TABLE><TBODY><TR><TH>Lane</TH><TH>1</TH><TH>2</TH><TH>3</TH><TH>4</TH></TR><TR><TD>Item</TD><TD>2-log ladder</TD><TD>- 3 ŒºL Phz (HF) pcr
-2 ŒºL dH<SUB>2</SUB>O
-1 ŒºL loading dye</TD><TD>Phz (GC) pcr
-2 ŒºL dH<SUB>2</SUB>O
-1 ŒºL loading dye</TD><TD>pSB1C3 (HF)
-2 ŒºL dH<SUB>2</SUB>O
-1 ŒºL loading dye</TD></TR></TBODY></TABLE><DIV class="notebookdate"><H4>August 28, 2018</H4><H4>Transformation (RbCl JM109) of 3A‚Äôs from 8/1</H4><H4>Peter</H4></DIV><P>**See Protocol from 7/16/18**</P><H4>Transforming:</H4><UL><LI>Acr circular gibson (8/27 SK)</LI><LI>CymA + J23110 + PSB1K3 3A (8/24 LCC)</LI></UL><H4>Notes/Deviations</H4><UL><LI>Two tubes of cells/ transformation (to ensure one tube will have cells) so 0.5 ŒºL of DNA added to each tube. Exceptions: 2 x 1ŒºL for Mtr (from 20 ŒºL 3A so more dilute than std 10 ŒºL 3A)</LI><LI>Heat-shocked for 30 seconds</LI><LI>Pooled like tubes (same vector) after 1 hour incubation</LI><LI>CAM for Acr</LI><LI>Kan for CymA</LI></UL><DIV class="notebookdate"><H4>August 30, 2018</H4><H4>Overnight Cultures of cymA 3A Assemblies</H4><H4>Liz</H4></DIV><UL><LI>Overnights of cymA 3A assemblies</LI><LI>Took 3 colonies from ‚ÄúRbCl transformation cymA + J23110 + pSB1K3 3A PC 8/28‚Äù plate</LI><UL><LI>3A done by LCC 8/24</LI></UL><LI>5mL LB broth + 5ŒºL kanamycin</LI><LI>Put in 37C shaker overnight</LI></UL><DIV class="notebookdate"><H4>August 31, 2018</H4><H4>Miniprep of Cultures Liz Picked up this Morning</H4><H4>Sricharan</H4></DIV><P>MP of cymA + J23110 + pSB1K3 by LCC ‚Ä¶ Genscript Kit, eluted 30ŒºL and glycerol stocks</P></DIV><H3>September</H3><BUTTON class="collapsible">Week 17: 9/3/2018</BUTTON><DIV class="content"><DIV class="notebookdate"><H4>September 4, 2018</H4><H4>cymA 3A gel</H4><H4>Liz</H4></DIV><P>Made 1% agarose gel</P><P>Nanodropped SK‚Äôs cymA minipreps 1-3 (8/31)</P><UL><LI>Miniprep 1: 21.05 ng/ŒºL</LI><LI>Miniprep 2: 17.49 ng/ŒºL</LI><LI>Miniprep 3: 29.24 ng/ŒºL</LI></UL><H4>Gel:</H4><TABLE><TBODY><TR><TH>1</TH><TH>2</TH><TH>3</TH><TH>4</TH></TR><TR><TD>Ladder (6ŒºL)</TD><TD>cymA miniprep 1</TD><TD>cymA miniprep 2</TD><TD>cymA miniprep 3</TD></TR></TBODY></TABLE><P>Samples loaded with 5ŒºL sample and 1ŒºL loading dye</P><P>Didn‚Äôt digest before running gel--ran again next day</P><DIV class="notebookdate"><H4>September 5, 2018</H4><H4>cymA 3A gel 2.0</H4><H4>Liz</H4></DIV><P>Digested cymA minipreps with EcoRI to run on gel</P><TABLE><TBODY><TR><TD>CutSmart</TD><TD>2ŒºL</TD></TR><TR><TD>DNA (cymA minipreps 1-3 SK 8/31)</TD><TD>12ŒºL</TD></TR><TR><TD>EcoRI</TD><TD>1ŒºL</TD></TR><TR><TD>water</TD><TD>5ŒºL</TD></TR></TBODY></TABLE><P>Heat blocked at 37C for 15 min</P><P>Heat killed at 65C for 15 min</P><P>Ran 1% agarose gel (same setup as previous day)</P><DIV class="notebookdate"><H4>September 5, 2018</H4><H4>Gel extraction of phz</H4><H4>Peter</H4></DIV><H4>Notes:</H4><UL><LI>Following Azady‚Äôs advice on getting greater concentration of pieces for gibson</LI></UL><H4>Gel:</H4><TABLE><TBODY><TR><TH>1</TH><TH>2</TH></TR><TR><TD>Ladder (6ŒºL)</TD><TD>10 ŒºL phz 4F-6R pure pcr (PC 7/13)</TD></TR></TBODY></TABLE><P>Extraction:</P><UL><LI>Bio basic kit</LI><LI>400 ŒºL buffer * (0.048 g/100 mg) * (1000 mg/1 g) = 192 ŒºL buffer used</LI></UL><DIV class="notebookdate"><H4>September 7, 2018</H4><H4>phz 4F-6R PCRs - Aiming for high purity</H4><H4>Sricharan</H4></DIV><P>Using</P><UL><LI>Original Gibson (7/12)</LI><LI>Old gel extract</LI><LI>PC new gel extract (9/5)</LI></UL><P>Usual Phusion PCR, 1ŒºL template DNA, 67 degrees C annealing, 1:18 extension time. All put into 1.5mL tubes</P></DIV><BUTTON class="collapsible">Week 18: 9/10/2018</BUTTON><DIV class="content"><DIV class="notebookdate"><H4>September 10, 2018</H4><H4>PCR purification, Nanodrop, Gel</H4><H4>Ryan</H4></DIV><P>Started off by preparing a small gel, .125g agarose, 12.5 mL TAE, mixed and microwaved for 30 seconds. Added 1.25 uL SYBR Safe and poured into mold and let sit</P><P>PCR purification of Charan‚Äôs 3 PCR products from 9/3/18, phz 4F-6R</P><UL><LI>Volume of PCR product was 20 uL, so added 100 ul Binding Buffer, and loaded it into Spin column and centrifuged for 1 min, @10,000 rpm</LI><LI>Discard flow-through and add 700 uL Wash Buffer, centrifuge again (same settings)</LI><LI>Discard flow-through, centrifuge again</LI><LI>Transfer top of spin column into a new 1.5 mL tube for collection</LI><LI>Add 40 uL Elution Buffer and let sit for 1 min</LI><LI>Centrifuge again</LI></UL><P>I labelled the 3 PCR products 1, 2, 3 and ran a nanodrop</P><UL><LI>1 was found to be 22.8 ng/uL</LI><LI>2 was found to be 19.1 ng/uL</LI><LI>3 was found to be 16.5 ng/uL</LI></UL><P>Put gel into the electrophoresis machine and added 6 uL 2-log Ladder into lane 1</P><UL><LI>3 uL of PCR 1 + 2uL water + 1 uL loading dye into lane 2</LI><LI>3 uL of PCR 2 + 2uL water + 1 uL loading dye into lane 3</LI><LI>3 uL of PCR 3 + 2uL water + 1 uL loading dye into lane 4</LI></UL><P>And ran it for 30 minutes at 100V *23 minutes dye was near end of gel </P><DIV class="notebookdate"><H4>September 10, 2018</H4><H4>Azady PCRs acr3 and pXB1C3</H4><H4>Liz</H4></DIV><H4>Acr</H4><TABLE><TBODY><TR><TD>HF buffer</TD><TD>4uL</TD></TR><TR><TD>10mM dNTPs</TD><TD>0.4uL</TD></TR><TR><TD>F primer (Azady acr 3F)</TD><TD>1uL</TD></TR><TR><TD>R primer (Azady acr 3R)</TD><TD>1uL</TD></TR><TR><TD>Template DNA (acr 3 gBlock)</TD><TD>0.2uL</TD></TR><TR><TD>DMSO</TD><TD>0.6uL</TD></TR><TR><TD>phusion</TD><TD>0.2uL</TD></TR><TR><TD>Water</TD><TD>12.6uL</TD></TR></TBODY></TABLE><H4>pSB1C3</H4><TABLE><TBODY><TR><TD>HF buffer</TD><TD>4uL</TD></TR><TR><TD>10mM dNTPs</TD><TD>0.4uL</TD></TR><TR><TD>F primer (SB-prep-3P-1-mutated)</TD><TD>1uL</TD></TR><TR><TD>R primer (SB-prep-2Ea)</TD><TD>1uL</TD></TR><TR><TD>Template DNA (BBa_R0080 diluted 10x)</TD><TD>0.2uL</TD></TR><TR><TD>DMSO</TD><TD>0.6uL</TD></TR><TR><TD>phusion</TD><TD>0.2uL</TD></TR><TR><TD>Water</TD><TD>12.6uL</TD></TR></TBODY></TABLE><H4>Thermocycler</H4><TABLE><TBODY><TR><TD>Initial denaturation</TD><TD>98C</TD><TD>30s</TD></TR><TR><TD>35 cycles</TD><TD>98C</TD><TD>10s</TD></TR><TR><TD>67C</TD><TD>30s</TD></TR><TR><TD>72C</TD><TD>35s</TD></TR><TR><TD>Final extension</TD><TD>72C</TD><TD>10 min</TD></TR><TR><TD>hold</TD><TD>4C</TD><TD>infinite</TD></TR></TBODY></TABLE><DIV class="notebookdate"><H4>September 13, 2018</H4><H4>Azady PCRs, acr 1 &amp; 2</H4><H4>Sricharan</H4></DIV><P>PCRing parts of acr to remove hairpins and amplify gBlocks, for better Gibson results</P><P>Resuspended primers in TE</P><UL><LI>spun down dry tubes</LI><LI>to 100¬µM in tubes, let sit 10 min</LI><LI>aliquot and dilute to 10¬µM</LI></UL><UL><LI>Usual Phusion HF Buffer</LI><LI>For acr 1, Primers: Azady acr 1F + Azady acr 1R, Anneal 68¬∞C</LI><LI>For acr 2, Primers: Azady acr 2F + Azady acr 2R, Anneal 67¬∞C</LI><LI>Usual cycle, with 15s extension time for both</LI></UL><UL><LI>Transferred into 1.5mL tubes</LI></UL><DIV class="notebookdate"><H4>September 13, 2018</H4><H4>PCR Purifs + Gel of acr 1-3 + pSB1C3 mut</H4><H4>Sricharan</H4></DIV><UL><LI>PCR purifying acr 1-3 gBlock PCRs &amp; pSB1C3 mut (adapted for Gibson)</LI><LI>Lambda Kit, eluting with 30¬µL ‚Ä¶ Nanodrop and gel by PC</LI></UL><DIV class="notebookdate"><H4>September 14, 2018</H4><H4>Gel of Acr g-block PSB1C3 pcr‚Äôs</H4><H4>Peter</H4></DIV><H4>Notes:</H4><UL><LI>Following Azady‚Äôs advice to pcr the g-blocks and backbone before gibson</LI><LI>Backbone is mutated (using primers with one nucleotide difference) so can overlap with new acr 1 &amp; 3 g-blocks and not form hairpin</LI></UL><H4>Gel:</H4><TABLE><TBODY><TR><TH>1</TH><TH>2</TH><TH>3</TH><TH>4</TH><TH>5</TH></TR><TR><TD>Ladder (6ŒºL)</TD><TD>Acr 1 pcr purif. (9/14 SK)
-512 bp
</TD><TD>Acr 2 pcr purif. (9/14 SK)
-529 bp
</TD><TD>Acr 3 pcr purif. (9/14 SK)
-653 bp
</TD><TD>Mutated pSB1C3 pcr purif. (SK 9/14)</TD></TR></TBODY></TABLE></DIV><BUTTON class="collapsible">Week 19: 9/17/2018</BUTTON><DIV class="content"><DIV class="notebookdate"><H4>September 17, 2018</H4><H4>Azady PCR acr 1 and 2 (more DNA)</H4><H4>Liz</H4></DIV><TABLE><TBODY><TR><TD>HF buffer</TD><TD>4uL</TD></TR><TR><TD>10mM dNTPs</TD><TD>0.4uL</TD></TR><TR><TD>F primer (Azady acr 1F/2F)</TD><TD>1uL</TD></TR><TR><TD>R primer (Azady acr 1R/2R)</TD><TD>1uL</TD></TR><TR><TD>Template DNA (acr 1, 2 gBlock)</TD><TD>2uL</TD></TR><TR><TD>DMSO</TD><TD>0.6uL</TD></TR><TR><TD>phusion</TD><TD>0.2uL</TD></TR><TR><TD>Water</TD><TD>10.8uL</TD></TR></TBODY></TABLE><P>Temperatures for thermocycler from Sricharan‚Äôs same PCR last week</P><DIV class="notebookdate"><H4>September 18, 2018</H4><H4>Zymo DNA concentrator planning (acr)</H4><H4>Peter</H4></DIV><H4>Notes:</H4><UL><LI>Concentrating fragments before gibson as suggested by Azady</LI><LI>Wanted 50 ng of acr 3 and calculated equimolar amounts of other fragments (acr 1 &amp; 2)</LI><LI>Used 2:1 fragment to backbone ratio</LI><LI>Need to add at least 6 ŒºL elution buffer</LI><LI>Added total of 10 ŒºL of elution buffer to get 8.7 ŒºL final product</LI><LI>Below is initial amounts added to column, see zymo kit for further details on the next steps and buffers to add</LI><LI>Centrifuging done at 15500 g</LI></UL><TABLE><TBODY><TR><TH>Component</TH><TH>Amount (ŒºL)</TH></TR><TR><TD>Acr 3 pcr purif. (9/14 SK)</TD><TD>5.62</TD></TR><TR><TD>Acr 1 pcr purif. (9/14 SK)</TD><TD>3.25</TD></TR><TR><TD>Acr 2 pcr purif. (9/14 SK)</TD><TD>2.57</TD></TR><TR><TD>Mutated pSB1C3 pcr purif. (9/14 SK)</TD><TD>7.86</TD></TR><TR><TD>Binding Buffer</TD><TD>96.5 = 19.3 * 5 (total volume DNA mult. 5)</TD></TR></TBODY></TABLE><DIV class="notebookdate"><H4>September 18, 2018</H4><H4>Gibson of acr (circular)</H4><H4>Peter</H4></DIV><TABLE><TBODY><TR><TH>Component</TH><TH>Amount (ŒºL)</TH></TR><TR><TD>Zymo pcr fragments product</TD><TD>8.7</TD></TR><TR><TD>SGI GA HiFi Master Mix</TD><TD>8.7</TD></TR></TBODY></TABLE><UL><LI>-run in thermo cycler @ 50 C for 1 hour</LI></UL><DIV class="notebookdate"><H4>September 19, 2018</H4><H4>Transformation of acr Gibson (by PC)</H4><H4>Sricharan</H4></DIV><P>We‚Äôve now PCRed parts for acr Gibson and backbone (some mutations to remove secondary structure)</P><UL><LI>Peter concentrated DNA and ran Gibson yesterday</LI><LI>Note: (By accident) we forgot to restriction digest pSB1C3 mut</LI><LI>Will this affect product?</LI><LI>Also no - control with backbone ‚Ä¶ will do - control with water</LI></UL><UL><LI>Transformation of the usual RbCl2 cells with 1ŒºL in each 50ŒºL cells tube</LI></UL><UL><LI>Acr - transformed with 1ŒºL acr Gibson each 50ŒºL cells</LI><LI>Ctrl - transformed with 1ŒºL dH2O each 50ŒºL cells ‚Ä¶ 1 tube was forgotten so only transformed  tube</LI></UL><UL><LI>Plates labeled PC since Peter helped</LI></UL><DIV class="notebookdate"><H4>September 19, 2018</H4><H4>Azady PCR: acr 3 and pSB1C3 (2nd time)</H4><H4>Sricharan</H4></DIV><UL><LI>Acr 3: Phusion HF PCR with 67¬∞C annealing, 15 sec extension, primers Azady acr 3F and azady acr 3R</LI><LI>pSB1C3: Phusion HF PCR with 66¬∞C annealing, 40 sec extension, primers SB-prep-3P-1-mut and SB-prep-2-Ea</LI></UL></DIV><BUTTON class="collapsible">Week 20: 9/24/2018</BUTTON><DIV class="content"><DIV class="notebookdate"><H4>September 24, 2018</H4><H4>Acr 1-3 PCR cleanup and gel</H4><H4>Liz</H4></DIV><P>Made 1% agarose gel (small wells)</P><P>PCR purified acr 1-3 and pSB1C3</P><UL><LI>Acr 1-2 LCC 9/17</LI><LI>Acr 3 and pSB1C3 mut SK 9/21</LI><LI>Lambda kit</LI><LI>Eluted in 30ŒºL</LI></UL><P>Nanodropped:</P><UL><LI>Acr 1: 12.68 ng/ŒºL</LI><LI>Acr 2: 19.82 ng/ŒºL</LI><LI>Acr 3 HF: 33.59 ng/ŒºL</LI><LI>Acr 3 GC: 9.76 ng/ŒºL</LI><LI>pSB1C3: 78.88 ng/ŒºL</LI></UL><H4>Gel:</H4><TABLE><TBODY><TR><TH>1</TH><TH>2</TH><TH>3</TH><TH>4</TH><TH>5</TH><TH>6</TH><TH>7</TH><TH>8</TH></TR><TR><TD>ladder</TD><TD>Acr 1</TD><TD>Acr 2</TD><TD>Acr 3 HF</TD><TD>Acr 3 GC</TD><TD>pSB1C3</TD><TD>ladder</TD><TD>Acr 1</TD></TR><TR><TD>6ŒºL</TD><TD>5ŒºL sample, 1ŒºL dye</TD><TD>6ŒºL</TD><TD>1ŒºL dye,
    1.5ŒºL water, 
  3.5ŒºL sample</TD></TR></TBODY></TABLE><DIV class="notebookdate"><H4>September 26, 2018</H4><H4>Transforming acr</H4><H4>Sricharan</H4></DIV><P>Trying again to transform Gibson acr from 9/17</P><UL><LI>1 tube (50ŒºL) Strataclone - 4ŒºL acr Gibson (9/17)</LI><LI>2 tubes (100ŒºL) RbCl cells - 8ŒºL acr Gibson (9/17)</LI></UL><UL><LI>1 tube (50ŒºL) RbCl cells - 4ŒºL dH<SUB>2</SUB>O (ctrl)</LI></UL><UL><LI>Usual protocol for RbCl cells</LI><LI>For Strataclone</LI><LI>20 min on ice</LI><LI>45 sec heat shock</LI><LI>2 min on ice</LI><LI>250ŒºL SOC - Incubate slightly &gt;1hr</LI><LI>Plate 100ŒºL or so</LI></UL><UL><LI>Tubes started 1 hr incubation at 2:45pm</LI><LI>End 3:45 pm</LI></UL><P>3 plates waiting on bench for Peter to plate - not in warm room since it seems humid there</P><DIV class="notebookdate"><H4>September 27, 2018</H4><H4>acr OCs</H4><H4>Sricharan</H4></DIV><UL><LI>Picked up transformations from yesterday</LI><LI>1 colony RbCl acr</LI><LI>Many (~20 colonies) Strataclone acr</LI></UL><P>OC‚Äôed 1 RbCl and 6 Stratclone colonies</P><UL><LI>LB from Kate downstairs (says it was made 2013 on bottle?)</LI><LI>Accidentally forgot to add Cam so added after tips were submerged in tubes</LI><LI>In at 4:38pm, shaker downstairs</LI></UL><DIV class="notebookdate"><H4>September 28, 2018</H4><H4>acr Miniprep/Diagnostic</H4><H4>Liz</H4></DIV><P>Made 1% agarose gel</P><P>Minipreps of strataclone acr 1-6 and RbCl acr (SK 9/27)</P><UL><LI>Used genscript kit</LI><LI>Eluted in 30ŒºL elution buffer</LI></UL><H4>Single digest for diagnostic gel</H4><TABLE><TBODY><TR><TD>CutSmart</TD><TD>2ŒºL</TD></TR><TR><TD>water</TD><TD>7ŒºL</TD></TR><TR><TD>Miniprep DNA (str acr 1-6 and RbCl acr)</TD><TD>10ŒºL</TD></TR><TR><TD>EcoRI</TD><TD>1ŒºL</TD></TR></TBODY></TABLE><P>Thermocycler:</P><UL><LI>37C for 15 minutes</LI><LI>65C for 15 min</LI><LI>4C hold</LI></UL><H4>Gel:</H4><TABLE><TBODY><TR><TH>1</TH><TH>2</TH><TH>3</TH><TH>4</TH><TH>5</TH><TH>6</TH><TH>7</TH><TH>8</TH></TR><TR><TD>ladder</TD><TD>Acr str 1</TD><TD>Acr str 2</TD><TD>Acr str 3</TD><TD>Acr str 4</TD><TD>Acr str 5</TD><TD>Acr str 6</TD><TD>RbCl acr</TD></TR></TBODY></TABLE><P>5ŒºL sample, 1ŒºL loading dye</P></DIV></DIV></DIV><DIV id="notebooksection3" class="section"><DIV class="container"><H2>Protocols</H2><P class="clearfix"><SPAN class="bold">Note:</SPAN> Embedded PDFs not viewable in Safari. Click/tap <SPAN class="bold">&quot;Download&quot;</SPAN> to view them.</P><DIV class="embeddedpdf"><A href="https://static.igem.org/mediawiki/2018/f/f4/T--UConn--3AAssembly.pdf" download="">Download</A><BUTTON class="collapsible"><H3>3A Assembly</H3></BUTTON></DIV><DIV class="embeddedpdf"><A href="https://static.igem.org/mediawiki/2018/1/1d/T--UConn--Chloramphenicol-LB-Broth.pdf" download="">Download</A><BUTTON class="collapsible"><H3>Chloramphenicol LB Broth</H3></BUTTON></DIV><DIV class="embeddedpdf"><A href="https://static.igem.org/mediawiki/2018/f/f7/T--UConn--DNA-Gel-Extraction-%28EZ-10_Kit%29.pdf" download="">Download</A><BUTTON class="collapsible"><H3>DNA Gel Extraction (EZ-10 Kit)</H3></BUTTON></DIV><DIV class="embeddedpdf"><A href="https://static.igem.org/mediawiki/2018/a/a2/T--UConn--DNA-Ligation.pdf" download="">Download</A><BUTTON class="collapsible"><H3>DNA Ligation</H3></BUTTON></DIV><DIV class="embeddedpdf"><A href="https://static.igem.org/mediawiki/2018/5/5e/T--UConn--Enzyme-DNA-Digestion.pdf" download="">Download</A><BUTTON class="collapsible"><H3>Enzyme DNA Digestion</H3></BUTTON></DIV><DIV class="embeddedpdf"><A href="https://static.igem.org/mediawiki/2018/b/b3/T--UConn--GenScript-Miniprep.pdf" download="">Download</A><BUTTON class="collapsible"><H3>GenScript Miniprep</H3></BUTTON></DIV><DIV class="embeddedpdf"><A href="https://static.igem.org/mediawiki/2018/5/57/T--UConn--Gibson-Assembly.pdf" download="">Download</A><BUTTON class="collapsible"><H3>Gibson Assembly</H3></BUTTON></DIV><DIV class="embeddedpdf"><A href="https://static.igem.org/mediawiki/2018/5/5e/T--UConn--Lambda-PCR-Purification-Kit.pdf" download="">Download</A><BUTTON class="collapsible"><H3>Lambda PCR Purification Kit</H3></BUTTON></DIV><DIV class="embeddedpdf"><A href="https://static.igem.org/mediawiki/2018/9/99/T--UConn--LB-Agar-Plates-with-Antibiotic.pdf" download="">Download</A><BUTTON class="collapsible"><H3>LB Agar Plates with Antibiotic</H3></BUTTON></DIV><DIV class="embeddedpdf"><A href="https://static.igem.org/mediawiki/2018/8/8a/T--UConn--PCR-Protocol-for-Phusion-High-Fidelity-DNA-Polymerase.pdf" download="">Download</A><BUTTON class="collapsible"><H3>PCR Protocol for Phusion¬Æ High-Fidelity DNA Polymerase</H3></BUTTON></DIV><DIV class="embeddedpdf"><A href="https://static.igem.org/mediawiki/2018/c/c5/T--UConn--PureLink-PCR-Purification-Kit.pdf" download="">Download</A><BUTTON class="collapsible"><H3>PureLink PCR Purification Kit</H3></BUTTON></DIV><DIV class="embeddedpdf"><A href="https://static.igem.org/mediawiki/2018/5/53/T--UConn--RbCl-Competent-Cell-Transformation.pdf" download="">Download</A><BUTTON class="collapsible"><H3>RbCl Competent Cell Transformation</H3></BUTTON></DIV></DIV></DIV><FOOTER><DIV id="footerhome"><A href="https://2018.igem.org/Team:UConn">Home</A></DIV><DIV id="bottomnav"><DIV id="bottomnavsection1" class="bottomnavsection"><DIV class="bottomnavitem"><A href="https://2018.igem.org/Team:UConn/Project">Project</A></DIV><UL><LI><A href="https://2018.igem.org/Team:UConn/Project#background">Background</A></LI><LI><A href="https://2018.igem.org/Team:UConn/Project#description">Description</A></LI></UL></DIV><DIV id="bottomnavsection2" class="bottomnavsection"><DIV class="bottomnavitem"><A href="https://2018.igem.org/Team:UConn/Notebook">Notebook</A></DIV><UL><LI><A href="https://2018.igem.org/Team:UConn/Notebook#interlab">Interlab</A></LI><LI><A href="https://2018.igem.org/Team:UConn/Notebook#labnotebook">Lab Notebook</A></LI><LI><A href="https://2018.igem.org/Team:UConn/Notebook#protocols">Protocols</A></LI></UL></DIV><DIV id="bottomnavsection3" class="bottomnavsection"><DIV class="bottomnavitem"><A href="https://2018.igem.org/Team:UConn/Parts">Parts</A></DIV><UL><LI><A href="https://2018.igem.org/Team:UConn/Parts#listofparts">List of Parts</A></LI></UL></DIV><DIV id="bottomnavsection4" class="bottomnavsection"><DIV class="bottomnavitem"><A href="https://2018.igem.org/Team:UConn/HumanPractices">Human Practices</A></DIV><UL><LI><A href="https://2018.igem.org/Team:UConn/HumanPractices#communityoutreach">Community Outreach</A></LI><LI><A href="https://2018.igem.org/Team:UConn/HumanPractices#collaborations">Collaborations</A></LI></UL></DIV><DIV id="bottomnavsection5" class="bottomnavsection"><DIV class="bottomnavitem"><A href="https://2018.igem.org/Team:UConn/Safety">Safety</A></DIV></DIV><DIV id="bottomnavsection6" class="bottomnavsection"><DIV class="bottomnavitem"><A href="https://2018.igem.org/Team:UConn/Team">Team</A></DIV><UL><LI><A href="https://2018.igem.org/Team:UConn/Notebook#attributions">Attributions</A></LI></UL></DIV></DIV><DIV id="bottomnavaddress"><P><A href="https://goo.gl/maps/yexLUgoBazx" target="blank">University of Connecticut (UConn), Storrs, Connecticut 06269, United States of America</A><A>Contact us at </A><A href="mailto:uconnigem@gmail.com">uconnigem@gmail.com</A></P></DIV></FOOTER></DIV></DIV></DIV></DIV></DIV></BODY></HTML>