Team:UCLA/Notebook/Protein Cages/29 June 2015
iGEM UCLA
Home
Team
Meet the Team
Attributions
Collaborations
Projects
Programming Spider Silk
Functionalizing Silk
Honeybee Silk
Materials Processing
Protein Cages
Human Practices
Interlab Study
Parts
Team Parts
Basic Parts
Composite Parts
Part Collection
Notebook
Programming Spider Silk
Functionalizing Silk
Honeybee Silk
Materials Processing
Protein Cages
Interlab Study
Safety
Judging
Intro: Today I delved into the specifics of resuspending newly synthesized DNA once we receive it from IDT. Phil and I discussed primer design in more detail. We will order the cage gene flanked by the iGEM prefix and suffix, then use one round of PCR with primers overlapping the cage gene itself and including the NdeI and XhoI sites. Additionally, I designed amplification primers for the cage gene itself. Phil and I researched which sequencing primers to use for the pET22b vector.
Resuspending Newly-Synthesized DNA
Source: https://www.idtdna.com/pages/decoded/decoded-articles/core-concepts/decoded/2011/03/16/dna-oligonucleotide-resuspension-and-storage
lyophilized (freeze-dried)
spin down prior to opening
use TE buffer [Tris-EDTA pH 8.0 (nuclease free)]
store at -20C
conc notes: 100 uM stock is what IDT does, then dilutes a portion to work with
Q5 PCR protocol calls for 1 ng-1ug of genomic DNA and 1pg-1ng of plasmid/viral DNA (NEB)
Cage Amplification Primer Design: I used the PCquad 2.0 benchling file that Phil created by altering some of the optimized codons for preferred primer design. My primers amplify just the cage gene itself. They are 20 and 21 bp long, both with Tms very close to 55C. After analyzing them primers using IDT's OligoAnalyzer 3.1 ( https://www.idtdna.com/calc/analyzer ) I determined that the two primers formed a heterodimer with a deltaG of -6.21 kcal/mole. However, Phil and I decided that this was fine since anything greater than -9 kcal/mole should be alright. We tried to run heterodimer analyses of each primer against the cage gene itself, but the program timed out since the cage gene is so long. In any case, not much can be done about unfavorable annealing at this point. If anything weird happens we can simply gel purify the correct band.
Tyler Lee -- Wtleeiv 03:04, 30 June 2015 (CDT)
Phillip's notes:
Introduction:  Once again, the primers will be attempted to be design and optimized today, with the consideration of the secondary structures.
Since there was extensive secondary structure after analyzing the primers on IDT.  Some of the codons will be changed to avoid overlap.  Using the DNA codon table from https://en.wikipedia.org/wiki/DNA_codon_table .  The goal was to get a deltaG under 9kcal/mol for any homo/heterodimers, as suggested by IDT.
A new file was made on Benchling called PCquad Optimized 2.0.  This includes the previous version optimized to avoid iGEM RE sites, as well as an attempt to optimize for primer design.  The edits were first done on PCquad Optimized for pET22b.   Several changes were made in at the beginning and end of the protein cage sequence.  NCBI blast was used to verify that the amino acid sequence was the same as the original PCquad.   The new file to be used for our protein cage is “PCquad Optimized 2.0”
Notes on the primers for incorporation into pET22b. Forward:  TCAGaccatATGCCGTTcATTACCGTAGGAC Tm = 62.8 degrees for entire primer, Tm = 56.7 degrees for overlap with gene Hairpin with dG = -2.66 kcal/mol Self-dimer:  dG = -7.82kcal/mol due to NdeI site Reverse:  TTGACAGTctcgagCTGACGATGGCTTCTATGCTGAG Tm = 65.9 degrees for entire primer, Tm = 52.6 degrees for overlap with gene Hairpin: one with dG = -2.32 kcal/mol Self-dimer:  Highest at -9.96 kcal/mol due to XhoI site, several others between -4 to -6 kcal/mol Heterodimers:  several at about -6kcal/mol
The construct to order was updated and is named PCquad Optimized for iGEM 2.0 Tyler designed the amplifying primers for the PCquad gene. Vinson suggested that NEB Tm calculator be used for determining annealing temperature.   Here is a potential resource for PCR. https://www.qiagen.com/us/resources/molecular-biology-methods/dna/ The sequencing primer will be discussed tomorrow, but T7 universal sequencing primers are a candidate.
Concluding remarks:  After going over our primer with David tomorrow, the order for the construct and primers should be placed.  Sequencing primers must also be ordered, if they are not available.  The next step is to lay out the protocol for transformation and protein expression, as well as to design the primers for making the insertion mutants.
