Team:UCLA/Notebook/Recombinant Expression
iGEM UCLA
Home
Team
Meet the Team
Attributions
Collaborations
Projects
Programming Spider Silk
Functionalizing Silk
Honeybee Silk
Materials Processing
Protein Cages
Human Practices
Interlab Study
Parts
Team Parts
Basic Parts
Composite Parts
Part Collection
Notebook
Programming Spider Silk
Functionalizing Silk
Honeybee Silk
Materials Processing
Protein Cages
Interlab Study
Safety
Judging
Recombinant Silk Functionalization
mm/dd - mm/dd (most recent weeks on top)
mm/dd: Entry title/short description (most recent days on top)
Plain text for descriptions, etc.
Unordered lists for bullet points
For images, upload image onto wiki server. Then replace placeholder image url with your image's url.
Ordered lists for numbers
For tables, "tr" stands for "table row", and "td" means "table column."
Row 1, Column 1
Row 1, Column 2
Row 1, Column 3
Row 1, Column 4
Row 2, Column 1
Row 2, Column 2
Row 2, Column 3
Row 2, Column 4
Row 1, Column 1
Row 2, Column 2
Row 3, Column 3
Row 4, Column 4
For different types of text styling... bold, italics, underlined. For headers...
Headers
Smaller Header
You get the point
Feel free to google any other styling.
Keep all your html within the detail tags for the day/week you're putting your entry for.
To properly indent your html code, use: http://www.freeformatter.com/html-formatter.html
mm/dd: Sample Entry
Today we began cloning our GFP.
PCR'd off template
Ran gel
Restriction digest
Ligated into backbone
For our ligation, we made the following modifications:
Tried it with newly bought ligase
Left reaction overnight instead of 2 hrs
Vector to insert ratio was 1:5 instead of 1:3
PCR Reaction:
Component
Volume
5X Q5 Reaction Buffer
5
10 mM dNTPs
0.5
10 uM Forward (primer 3/7)
1.25
10 uM Reverse (primer 8)
1.25
Template (diluted to 1ng/uL)
0.5
Q5 High Fidelity DNA Polymerase
0.25
Nuclease Free Water
16.25
Gel: Lot of bands, all at correct sizes
Contents
1 Introduction
2 Goals
2.1 Goals
3 Achievements
3.1 Achievements
4 What to accomplish next
4.1 Julian
5 Raw lab notebook entries
Introduction
Silk fibers produced by the arachnid Nephila clavipes (Golden Silk-Orb Weavers) produce unique mechanical and immunogenic properties (Vepari et al., 2007). The highly tensile and elastic properties of these fibers, coupled with the non- immunogenic and non-allergenic phenomena observed with spider silk, make it a suitable candidate for a wide variety of biomedical applications (Schacht et al., 2014). In particular, studies into the use of the spider silk monomers MaSP1 and MaSP2 with fusions to select molecules or interactive binding domains have arisen to study the potential for spider silk as a seeding and drug delivery mechanism (Humenik et al., 2011).
However, large-scale production of spider silk has been hampered due to the poor sustainability of Golden Spider farming and subsequent silk collection procedures (Schiebel, 2004). Recent explorations into the use of recombinant silk proteins through bacterial chassis and microbiological vectors have shown tremendous promise, but are still hindered by a wide variety of issues, including (1) relatively low yield of recombinant protein relative to the amount of cell culture needed, (2) a complex and time-consuming process in constructing and troubleshooting modular forms of the silk-binding site fusions, (3) harsh chemical conditions during the purification of recombinant proteins that lead to potential denaturation of active peptides, (4) GC-rich hindrance of mRNA transcript stability and translational termination, and (5) poor scalability of recombinant silk production absent the use of large, relatively expressive bioreactors for bacterial fermentation (Winkler et al., 2000; Prince et al., 1995; Arcidiacono et al., 2002).
Studies investigating the potential for co-spinning, a process in which the terminal domains of the functional N. clavipes silk protein heavy chain are fused with genes encoding functional binding domains, and subsequently are combined with native silk proteins to form crystalline fibers, have shown some success (Sponner et al., 2005). In vivo co-spinning simulations have previously been conducted using the transgene in silkworms, producing functionally fluorescent N-terminal – GFP- C-terminal fusions that successfully incorporate into endogenous Bombyx more silkworm silk monomers (Kojima et. al., 2007). These N-Terminal and C-terminal regions are not only easier to clone due to their non-repetitive structures, but additionally allows for storage and assembly of endogenous ampullate spidroins (Askarieh et al., 2010). Composite silkworm-spider silk chimeras have shown improved mechanical properties, and may increase the ease in high-yield production due to the easier manipulation of silkworm silk monomers (Teule, et. al. 2012). Yet, this process still relies on the use of transgenic silkworms, which have longer developmental cycles, require larger storage and nutrient conditions, and difficulty to harvest relative to bacterial counterparts (Murphy et al., 2009).
An ex situ co-spinning process has been developed for the production of composite N. clavipes – B. more silk fibers with functional protein fusions in bacterial chassis (namely, laboratory strains of Escherichia coli ). This process results in the construction of a transgene that can be modularized using restriction enzymes to incorporate a wide range of functional peptide binding sites or other proteins, including avidin-biotin binding domains, RGD-cadherin motifs, albumin binding domains, or antibody affinity domains (Jansson et al., 2014). Specifically, this composite structure focuses on the ability to express functional sfGFP in silk fibers as a marker for successful co-spinning, and verifies proper NC-sfGFP expression using SDS-PAGE analysis.
A Standard Registry part encoding sfGFP (BBa_K515005) flanked by conserved N. clavipes N- and C- silk heavy chain terminal domains (NC-sfGFP) has been produced. This part is ligated in the standard registry pSB1C3 vector backbone, and transformed into electrocompetent E. coli cell cultures. Expression of the fusion protein is induced using IPTG, and extracts are purified using Immobilized Metal Affinity Chromatography (IMAC) using a Ni-NTA Sepharose resin bed designed to pull down polyhistidine tagged protein constructs.
Goals
Goals
Finish Purification of ABD containing construct
Create hydrogel with ABD and B.mori silk and see if it retains/binds albumin
Achievements
Achievements
Cloned the ABD construct and begun purification methods
What to accomplish next
Julian
Test albumin binding properties of the ABD silk construct
test hydrogel formation and albumin retention
Raw lab notebook entries
July
M
T
W
T
F
S
S
 
 
1
2
3
4
5
6
7
8
9
10
11
12
13
14
15
16
17
18
19
20
21
22
23
24
25
26
27
28
29
30
31
August
M
T
W
T
F
S
S
 
 
 
 
 
1
2
3
4
5
6
7
8
9
10
11
12
13
14
15
16
17
18
19
20
21
22
23
24
25
26
27
28
29
30
31
September
M
T
W
T
F
S
S
 
1
2
3
4
5
6
7
8
9
10
11
12
13
14
15
16
17
18
19
20
21
22
23
24
25
26
27
28
29
30
