Team:UCLA/Notebook/Protein Cages/25 June 2015
iGEM UCLA
Home
Team
Meet the Team
Attributions
Collaborations
Projects
Programming Spider Silk
Functionalizing Silk
Honeybee Silk
Materials Processing
Protein Cages
Human Practices
Interlab Study
Parts
Team Parts
Basic Parts
Composite Parts
Part Collection
Notebook
Programming Spider Silk
Functionalizing Silk
Honeybee Silk
Materials Processing
Protein Cages
Interlab Study
Safety
Judging
Intro: Today we designed the primer sequence for our codon optimized cage sequence from Yates' lab. I initially researched the iGEM specifics for the prefixes and suffix they require, so that we knew how to design primers for the sequence. Additionally, another set of primers with NdeI (prefix) and XhoI (suffix) sites will be designed for protein expression within the pET22b vector, since this plasmid was used by Dr. Yates.
We first spoke with Fasih to learn how to order gBlocks and to confirm our primer design methodology. This is what we learned:
1. select gBlocks gene fragments on the IDT (integrated DNA technologies) website
2. name the order
3. copy the gene from benchling into the sequence window
4. test the complexity to make sure its good to go
5. say no to all the questions asked by IDT
6. Notes: A. for payment, use oilgocard B. send to david attn. iGEM C. 1,000 ng is a good guaranteed yield
Remember to resuspend the gene at a concentration of 10 ng/microL.
Add a double stop codon.  Snap gene is a good place to look for plasmid sequences.
Tyler Lee -- Wtleeiv 19:27, 25 June 2015 (CDT)
Introduction: During our downtime with either lab protocols or orders to arrive, we decided to do a formal journal club within the protein cages group. Our first journal club will involve PCR(procedure,theory, application) and site directed mutagenesis as these will be heavily used in the project.
1.I was preparing for this journal club by finding papers and preparing a powerpoint presentation for this. First I looked up PCR on google then I got titles of some of the original papers on PCR. I typed these papers into google scholar which led me to the source of the papers(Science, Nature etc.)
2.I attended the meeting with David and Philip to go over our PCquad sequence and to decide what exactly to order.
Nithin
Introduction:  Today there will be a meeting with David to discuss the overview of our project.  There was a powerpoint created for the main points.  Fasih will then be assisting with ordering the parts.
First the optimized cage was verified to be the same amino acid sequence as the original.  This was done using Expasy and then was simply matched on Microsoft word.
PCcage original (from Dr. Yeates)
MPFITVGQENSTSIDLYYEDHGTGTPVVLIHGFPLSGHSWERQSAALLDAGARVITYDRR GFGQSSQPTTGYDYDTFAADLNTVLETLDLQDAVLVGFSMGTGEVARYVSSYGTARIAAV AFLASLEPFLLKTDDNPDGAAPQEFFDGIVAAVKADRYAFYTGFFNDFYNLDENLGTRIS EEAVRNSWNTAASGGFFAAAAAPTTWYTDFRADIPRIDVPALILHGTGDRTLPIENTARV FHKALPSAEYVEVEGAPHGLLWTHAEEVNTALLAFLAKAQEAQKQKLLTEVETYVLSIIP SGPLKAEIAQRLEDVFAGKNTDLEVLMEWLKTRPILSPLTKGILGFVFTLTVPSERGLQR RRFVQNALNGNGDPNNMDKAVKLYRKLKREITFHGAKEISLSYSAGALASCMGLIYNRMG AVTTEVAFGLVCATCEQIADSQHRSHRQLEHHHHHH
PCquad optmized MPFITVGQENSTSIDLYYEDHGTGTPVVLIHGFPLSGHSWERQSAALLDAGARVITYDRR GFGQSSQPTTGYDYDTFAADLNTVLETLDLQDAVLVGFSMGTGEVARYVSSYGTARIAAV AFLASLEPFLLKTDDNPDGAAPQEFFDGIVAAVKADRYAFYTGFFNDFYNLDENLGTRIS EEAVRNSWNTAASGGFFAAAAAPTTWYTDFRADIPRIDVPALILHGTGDRTLPIENTARV FHKALPSAEYVEVEGAPHGLLWTHAEEVNTALLAFLAKAQEAQKQKLLTEVETYVLSIIP SGPLKAEIAQRLEDVFAGKNTDLEVLMEWLKTRPILSPLTKGILGFVFTLTVPSERGLQR RRFVQNALNGNGDPNNMDKAVKLYRKLKREITFHGAKEISLSYSAGALASCMGLIYNRMG AVTTEVAFGLVCATCEQIADSQHRSHRQ
Both are the same amino acid sequence except for the XhoI site and the histag at the end of the original.
The steps to order the construct is the following.
Go to IDT Sign in go to gblocks gene fragments name copy paste sequence in test complexity add to order say no to everything for payment:  my account, oligo card, send to david, notify igem or david
Some suggestions from Fasih were the following.
Design the primers before ordering.   a) For XhoI and NdeI sites include handle of 6-8 basepairs b) igem prefix and suffix.  With two stop codons before the suffix.  This will go into psb1c3 perform digest and ligation on benchling Use snapegene for plasmids Need BL21 (DE3) with T7 polymerase for expression in pET22b
-Design of primers Primers were designed using Benchling For expression:   These are the NdeI and XhoI sites obtained from Benchling. NdeI:  5’-CATATG-3’, 3’-GTATAC-5’ XhoI:  5’-CTCGAG-3’, 3’GAGCTC-5’ The handle used was ACTGAC
For iGEM:  The iGEM prefix and suffix were used as stated previously.
Conclusions:  It seems as though the primer lengths may be too long, and that the melting temperatures are too high.  A second opinion may be needed tomorrow.  The order for just the PCquad optimized will be made in addition to primers to make it usable in the Pet22b vector, and for iGEM.  Once the strains from Yeate’s lab are obtained, the vector will be isolated and our optimized PCquad will be incorporated.  From there, protein expression will be attempted with Yeate’s strain as a control.
