<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_UCLA_Notebook_Protein_Cages_8_July_2015 skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:UCLA/Notebook/Protein Cages/8 July 2015</SPAN></H1><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><P><TITLE>iGEM UCLA</TITLE></P><DIV id="top_menu"><UL id="menu"><LI><A href="/Team:UCLA">Home</A></LI><LI><A href="/Team:UCLA/Team/Meet_the_Team">Team</A></LI><LI><A href="/Team:UCLA/Team/Meet_the_Team">Meet the Team</A></LI><LI><A href="/Team:UCLA/Attributions">Attributions</A></LI><LI><A href="/Team:UCLA/Collaborations">Collaborations</A></LI></UL><LI><A href="/Team:UCLA/Project">Projects</A></LI><LI><A href="/Team:UCLA/Project/Programming Spider Silk">Programming Spider Silk</A></LI><LI><A href="/Team:UCLA/Project/Functionalizing Silk">Functionalizing Silk</A></LI><LI><A href="/Team:UCLA/Project/Honeybee Silk">Honeybee Silk</A></LI><LI><A href="/Team:UCLA/Project/Protein Expression and Processing">Materials Processing</A></LI><LI><A href="/Team:UCLA/Project/Protein Cages">Protein Cages</A></LI><LI><A href="/Team:UCLA/Practices">Human Practices</A></LI><LI><A href="/Team:UCLA/Project/Interlab Study">Interlab Study</A></LI><LI><A href="/Team:UCLA/Parts">Parts</A></LI><LI><A href="/Team:UCLA/Parts">Team Parts</A></LI><LI><A href="/Team:UCLA/Basic_Part">Basic Parts</A></LI><LI><A href="/Team:UCLA/Composite_Parts">Composite Parts</A></LI><LI><A href="/Team:UCLA/Part_Collection">Part Collection</A></LI><LI><A href="/Team:UCLA/Notebook/Weekly Notes">Notebook</A></LI><LI><A href="/Team:UCLA/Notebook/Spider_Silk_Genetics">Programming Spider Silk</A></LI><LI><A href="/Team:UCLA/Notebook/Recombinant_Expression">Functionalizing Silk</A></LI><LI><A href="/Team:UCLA/Notebook/Honeybee_Silk">Honeybee Silk</A></LI><LI><A href="/Team:UCLA/Notebook/Materials">Materials Processing</A></LI><LI><A href="/Team:UCLA/Notebook/Protein_Cages">Protein Cages</A></LI><LI><A href="/Team:UCLA/Notebook/Interlab_Study">Interlab Study</A></LI><LI><A href="/Team:UCLA/Safety">Safety</A></LI><LI><A href="https://igem.org/Judging_Form.cgi?id=1763">Judging</A></LI></DIV><P>Phillip's notes:
</P><P>Introduction:  For today, colony PCR will be done in order to verify the Yeate construct.  In addition, ampicillin plates will be made in preparation for beginning protein expression in the next few days.  Much of it will be limited to what is possible to work with in the boelter lab..
</P><P>Resuspension of Oligos:
</P><P>
Resuspension, Dilution, Storage, and Other Tips
</P><P>After receiving an envelope containing their order of newly synthesized IDT oligonucleotides, many researchers find themselves asking the question “Now what should I do with these oligos?” At this point, several important decisions must be made regarding oligonucleotide resuspension, dilution, and storage, which may vary depending on the intended application for each oligo. Here, we provide recommendations for preparing oligos to use in experiments and guidelines for proper storage.
</P><P>
Resuspension
</P><P>Centrifugation and Resuspension. Most oligonucleotides synthesized and shipped from IDT are delivered dried down and often appear as a white flakey substance in the bottom of the tube. Dried DNA is quite stable and, typically, is easy to resuspend in an aqueous solution. The first step a researcher should take upon receipt of their oligos is to briefly centrifuge the tubes before opening them. This ensures that any dried DNA that may have become dislodged during shipping is brought down to the bottom of the tube. Next, the oligos should be resuspended in TE buffer (IDTE: 10 mM Tris, 0.1 mM EDTA, pH 8.0). Tris (tris(hydroxymethyl)aminomethane) acts as a buffer, helping to maintain a constant pH for the solution. EDTA (ethylenediaminetetraacetic acid) prevents nuclease digestion of the DNA. At higher concentrations, EDTA can inhibit enzymatic reactions, such as PCR, by chelating metals ions like Mg2+. However, the low concentration of EDTA present in IDTE buffer will not interfere with subsequent enzymatic reactions. TE buffer can be made with common lab reagents or is conveniently available for purchase from the IDT catalog.
Alternatively, nuclease-free water, pH 7.0, can be used for resuspending oligonucleotides, but it will not modulate pH over time as will TE buffer. Use of HPLC- or molecular biology–grade water is preferable, as water from a deionizing system (such as Millipore) can be acidic, with a pH as low as 5.0. Water treated with DEPC (diethylpyrocarbonate) is not recommended for resuspension of DNA oligonucleotides because it can degrade the DNA during long-term storage.
</P><P>
Concentration and Dilution. Investigators commonly call in for advice regarding the concentration at which their oligos should be resuspended. This is largely up to the discretion of the researcher, and is dependent on how the oligo will be used in future experiments. Oligonucleotides are stable when stored over a large range of concentrations; however, for optimal stability we recommend concentrations no less than 1 µM, and no greater than 10 mM. IDT offers a Resuspension Calculator as part of our SciTools® suite (www.idtdna.com/scitools), which is useful for calculating the volume of buffer to add to a dried down oligo to obtain a desired concentration. Resuspension calculations can be made using the yield information that is provided both on the oligo tubes themselves and on the IDT oligo product specification sheets provided with the order. For your convenience and utility, this yield information is presented in optical density (OD) units, mass (mg), and copy number (nmoles).
Our standard recommendation is to resuspend oligos to a 100 µM stock concentration, as this is a simple concentration to make and is highly versatile for subsequent dilution purposes. The volume of TE buffer required to achieve a 100 µM stock is easily determined by multiplying the number of nanomoles listed for a particular oligo by a factor of 10, and then resuspending the dry DNA in that same number of microliters of TE buffer. For example, if the oligo specification sheet states that 20.3 nmoles of oligo were delivered, add 203 µL TE buffer to obtain a 100 µM stock solution. This stock solution can be diluted as needed into appropriate working solutions.
</P><P>
Resuspension Difficulties. While most oligonucleotides will readily go into solution at concentrations &lt;10 µM, some oligos (with certain fluorophore modifications and some hydrophobic modifications such as cholesterol) can be more difficult to resuspend. For oligos that are harder to resuspend, and for which one might observe residual precipitate present following resuspension, we recommend that the oligo be heated at 55°C for 1–5 minutes, vortexed thoroughly, and then briefly centrifuged. Occasionally, residual by-products of the synthesis reactions remain in the tube and are visible after resuspension. These include Trityl (appears flakey) and CpG (which forms a pellet at the bottom of the tube). Neither of these reagents should negatively impact the performance of an oligonucleotide; however, both can be easily removed by passing the resuspended oligo over a Sephadex G50 column.
</P><P>
Storage
Optimal Conditions for Standard DNA Oligonucleotides. IDT has tested the stability of oligonucleotides following various storage conditions (Table 1). The results of these studies indicate that the most stable temperature at which to store a standard DNA oligonucleotide is  20°C. An oligo stored at –20°C is stable for at least 24 months when either dried down, or resuspended in TE buffer or nuclease-free water. Standard DNA oligos dried down or stored at 5°C in TE buffer or water were found to be stable for long periods; however, better stability is achieved by freezing if the oligos are to be stored for extended periods. Standard DNA oligos stored at 37°C are stable for at least 6 weeks in water, or 25 weeks dried down or in TE. Storage at higher temperatures is not recommended; however, accidents can happen, such as oligos being left on the bench top overnight or freezers failing at the weekend, and it is important to understand that the stability of the oligo should not be affected. Regardless of the storage temperature, TE buffer, rather than water or dried down, is optimal for long-term storage.
Storing Oligos with Fluorophores. Typically, fluorophore labeled oligos show a reduction in signal of about 20% after 24 months when stored at –20°C in TE. For oligos with a fluorophore modification, particularly the Cy and FAM dyes, researchers may wish to store these oligos in amber tubes or covered in foil to prevent photo bleaching which can occur over time with prolonged exposure to light.
</P><P>Resuspending your gBlocks™ Gene Fragment: 
</P><P>
1. Centrifuge the tube for 3−5 sec at a minimum of 3000 x g to pellet the material to the bottom of the tube. 
</P><P>2. Add 20 µL TE to the tube for a final concentration of 10 ng/µL. 
</P><P>3. Briefly vortex and centrifuge. 
</P><P>
Storing your gBlocks™ Gene Fragment: gBlocks Gene Fragments can be stored in TE at −20°C for up to 24 months. If gBlocks Gene Fragments will be stored for less than 1 month, they can be resuspended in nuclease-free water rather than TE.
</P><P>Procedures:  
</P><P>Today 15mL of TE buffer was made from 150uL of 1M Tris-HCl (pH8) and 3uL of 0.5M EDTA (pH8) and bringing the total volume up to 15mL in a falcon tube.  
</P><P>All primers were resuspended in TE buffer.  For the colony PCR primers, a 40uM stock was made for long term storage, and a 20uM working stock was made.  For primers for pET22b, sequencing, and amplifying, a 100uM stock was made for long term storage, as well as a 20uM working stock for the pET22b primers.
</P><P>Conclusions:  The bacterial plate with the Yeate’s strain was not found in the cold room.  Tomorrow, agar plates will be made in order to restreak the bacteria from the glycerol stocks.  Dr. Kosuri suggested working trying to work with the PSB1C3 sequence by adding a T7 promoter and RBS to the PCquad sequence.  A his-tag and stop codon would also have to be made for this purpose.  This may be done tomorrow, as well as making the plates for downstream PCR and protein expression.
</P></DIV></DIV></DIV></DIV></BODY></HTML>