Team:UCLA/Project/Interlab Study
iGEM UCLA
Home
Team
Meet the Team
Attributions
Collaborations
Projects
Programming Spider Silk
Functionalizing Silk
Honeybee Silk
Materials Processing
Protein Cages
Human Practices
Interlab Study
Parts
Team Parts
Basic Parts
Composite Parts
Part Collection
Notebook
Programming Spider Silk
Functionalizing Silk
Honeybee Silk
Materials Processing
Protein Cages
Interlab Study
Safety
Judging
SilkyColi: Reprogramming the physical and functional properties of synthetic silks
2015 InterLab Study
Introduction
The 2015 UCLA iGEM Team is proud to participate in the Second International InterLab Measurement Study in synthetic biology. As members of the synthetic biology community, we are committed to providing robust data for development of novel characterization methods in the rapidly growing biological design fields of synthetic biology. The purpose of the 2015 InterLab study is to "measure and characterize fluorescence data for three specific genetic devices" expressing GFPmut3b (SwissProt: P42212) from active iGEM teams participating around the world. By collecting fluorescence data from multiple teams in absolute units, variability in measurement and consistency of data collected from instrumentation following a uniform procedure can be determined within a high degree of accuracy.  This notebook will record all protocols, daily experiments, basic parameters and images, as well as the raw data used to prepare the Interlab Worksheet, Protocol, and Wiki page for submission at the 2015 Giant Jamboree.
Experimental Design
Three separate genetic "devices" were constructed using IDT gBlocks Gene Fragments synthesis, in addition to positive control BBa_I20270 (Constitutive Family Promoter J23151 inserted upstream of the promoter MeasKit) and negative control BBa_R0040 (pTetR - empty control plasmid). All gBlocks were designed on Benching to simulate the sequence of the BioBricks standard assembly product, for uniformity in measurement with teams that opted for RFC10 standard assembly. All devices were subcloned in the standard pSB1C3 (chloramphenicol resistance marker) backbone and transformed into BL21(DE3) Escherichia coli . As such, E. coli BL21(DE3) laboratory strains were used as the chassis for fluorescent measurement. Details as to the location of the registry pieces used to construct the devices are below:
Device #
Benchling Link
Spec Sheet & FASTA File
Promoter
GFP Generator
Final Device Backbone
Device #1
https://benchling.com/s/EnB0uD9g/edit
Specs and FASTA
BBa_J23101
BBa_I3504
(B0034-E0040-B0015)
pSB1C3
Device #2
https://benchling.com/s/hX68sjpy/edit
Specs and FASTA
BBa_J23106
BBa_I3504
pSB1C3
Device #3
https://benchling.com/s/8q493PdY/edit
Specs and FASTA
BBa_J23117
BBa_I3504
pSB1C3
Positive Control BBa_I20270
N/A
N/A
BBa_J23151
GFPmut3b Promoter MeasKit
(B0032-E0040-B0010-B0012)
pSB1C3
Negative Control BBa_R0040
N/A
N/A
TetR repressible promoter (BBa_R0040)
N/A
pSB1C3
Protocols
The following are a list of protocols designed for the InterLab Study.
Preparation of chemically competent E. coli BL21(DE3) cells using Zymo Mix & Go Transformation Kit and Buffer Set
Rapid isolation of plasmid DNA (pSB1C3) using the Promega PureYield MiniPrep System (Miniprep)
Double digestion of plasmid DNA using EcoRI and PstI restriction exonucleases (NEB EcoRI/PstI Digest)
Rapid ligation and subcloning of gBlocks double digests into pSB1C3 vector backbone using T4 ligase (NEB T4 Ligation)
Zymoclean Gel DNA recovery and purification of plasmid/digested DNA following gel electrophoresis (Gel Extraction)
Zymo DNA Clean and Concentrator-5 for rapid cleanup of PCR, Digestion, and Ligation products (PCR Cleanup)
Results
Persons responsible for conducting InterLab Study
Fasih Ahsan: Designed the Interlab protocols and methodology, cloned the devices and controls, measured, and processed the data.
Megan Satyadi: Assisted in transforming the device construct into the E. coli BL21(DE3) cell chassis, and growing the final overnight cultures of each device cell line in biological triplicates.
What chassis did you use?
E. coli BL21(DE3)
What Biosafety Level is your chassis?
BSL-1 - non-pathogenic bacterial strain
What PPE did you utilize during your experiments (from cloning through to measuring the devices)?
Individuals responsible for conducting Interlab study:
Fasih Ahsan – Designed the InterLab protocols and methodology, cloned the devices and controls, measured and processed the data.
Megan Satyadi – Assisted in transforming the device constructs into the E. coli BL21(DE3) cell chassis, and growing the final overnight cultures of each     device cell line in biological triplicates.
Chassis and Safety Information
What chassis did you use?
E. coli Bl21(DE3)
What Biosafety Level is your chassis?
BSL1
What PPE did you utilize during your experiments (from cloning through to measuring the devices)?
Long pants, ankle length socks, closed-toed shoes, nitrile blue gloves, safety goggles, flame-resistant blue lab coats.
Information about Cloning
What DNA assembly method did you use to create your devices?
DNA Synthesis
How did you validate the final devices?
DNA Sequencing
OTHER: Colony PCR
Please tell us about any challenges or problems you encountered during the cloning process.
We had difficulties performing standard BioBricks assembly of the final device constructs, namely in the initial transformation of the GFP reporter     construct BBa_I13504. We attempted to transform both the pSB1A2 and pSB1C3 variants, using both chemically competent and electrocompetent transformation     line protocols. We observed no colony growth from the DNA products derived from the Registry we were given, so we had to resort to ordering each device     sequence using IDT gBlocks gene synthesis.
Additionally, we have difficult with false positive clones in Devices 2 and 3, where the resulting overnighted cultures from selected colonies did not     grow. More colonies were picked and overgrown to alleviate this situation, but it did result in general delays throughout the process. Sequencing of the     derived plasmid constructs revealed proper Device sequence, so we verified cloning success.
Protocol – Growing Cells for Measurement
Step 1: Streak out an agar plate with your organism containing the device and any control organisms you used.
a. Streak out 1 plate per device and control.
b. Incubate plates overnight (18-20 hours or until individual colonies are clearly visible) at 37 C.
c. Note the time grown in hours below in “Other”
d. Other: 18 hours
Please provide the positive controls used.
An E. coli BL21(DE3) cell line containing GFP expression device BBa_I20270 in the pSB1C3 chloramphenicol resistance backbone.
Please provide the negative control(s) you used:
An E. coli BL21(DE) cell line containing pTetR device BBa_R0040 in the pSB1C3 chloramphenicol resistance backbone.
As a blank for the plate reader, aliquots of LB and LB supplemented with 35ug/uL of chloramphenicol were cultured in the 37 deg C shaking incubator for 18     hours.
What type of agar did you use for this step:
BD Difco LB Broth, Miller (Luria-Bertani) supplemented with Difco Bacto Agar and 34ug/mL working concentration of chloramphenicol powder dissolved in 100%     ethanol.
Step 2: Inoculate liquid culture with your experimental devices and controls:
a. Device 1
b. Device 2
c. Device 3
d. Positive Control
e. Negative Control
f. OTHER: LB + chloramphenicol
What type of vessel container did you use to grow your cells?
a. Test tube
Please provide any detailed information about your vessel/container below.
Each device was grown in biological triplicates using 13 x 100 mm Corning Pyrex Vista Test Tubes with fitted plastic aerating caps.
If you used a test tube, how were the test tubes oriented in the incubator.
At an angle.
What type of media did you use?
BD Difco LB Broth, Miller (Luria-Bertani) supplemented with 34ug.mL working concentration of chloramphenicol powder in 100% ethanol.
What volume did you use to grow your cells?
5mL
Did you set up biological replicates in triplicate?
Yes
Step 3: Incubate your liquid cultures.
a. Temperature at 37 C
b. Incubate for 16-18 hours.
c. Other: Shaking at 200 RPM.
Plate Reader Measurement
Did you measure your cells with a plate reader?
Yes
Step 1: Obtain initial OD600 measurement of your overnight cultures.
a. Set your instrument to read OD600.
b. Setup a 96-well plate with your cultures.
c. Take the measurement and record it.
Step 2: Dilute your samples to an OD600 of 0.5
a. We did not dilute samples to an OD600 of 0.5. The BioTek plate reader has an OD accuracy between 0-4, and the fluorescence values will be normalized to     OD600 value for each culture in triplicate.
Step 3: Measure your samples.
a. Set your instrument to measure GFP
b. Measure your cells.
Alternate Protocol or Additional Details:
The resulting fluorescence data from the plate reader was processed using the Imperial 2014 iGEM Team InterLab Study: https://2014.igem.org/Team:Imperial/InterLab_Study . Background absorbance and fluorescence     were subtracted from all samples assayed as a blanking control. The resulting absorbance values were then used to divide their respective resulting     fluorescent values to generate a fluorescence per OD unit value (FL/OD). The average FL/OD of the negative control culture with empty vector (R0040) was     subtracted from the FL/OD of the devices to remove background cell fluorescence, resulting in a FL/OD600 value. These values were then divided by the     conversion factor determined from the standard curve generated from a sodium fluorescein control assay to derived a final unit value as the equivalent     ng/mL fluorescein per OD600 unit.
Please rate your experience with filling in this InterLab Protocol Form.
Very easy to fill in, no problems/
Please let us know any other thoughts or comments you have about the InterLab study experience.
I would have preferred a much more difficult and/or rigorous extra credit assignment, actually! Otherwise, great experience and great way to teach data     analysis methodology for our new members.
Date of InterLab Study
Measurement was obtained on Friday, August 28th, 2015.
Equipment Information
What type of incubator did you use to grow your cells?
New Brunswick Scientific Innova 44 Incubator Shaker
If known, what was the incubator’s throw (shaking diameter)
The incubators orbit was 2.5 cm (1 in.).
What piece of equipment did you use to measure the devices?
BioTek Synergy H1 Hybrid Multi-Mode Plate Reader, with Gen5 statistical software.
When was this equipment last calibrated: January 2015
Who calibrated the equipment? : UCLA Department of Bioengineering.
What was the wavelength of light you used to excite the cells: 485 nm
What was the filter/channel you used to capture the light emission from the cells: 528 nm
What was the sampling frequency?
N/A
Protocol:
Did you fill in the InterLab Protocol provided by the Measurement committee?
Yes
Did you follow the InterLab protocol provided by the Measurement committee?
Yes
How did you determine the final dataset that you are reporting?
The resulting fluorescence data from the plate reader was processed using the Imperial 2014 iGEM Team InterLab Study design: https://2014.igem.org/Team:Imperial/InterLab_Study . Background absorbance and fluorescence     were subtracted from all samples assayed as a blanking control. The resulting absorbance values were then used to divide their respective resulting     fluorescent values to generate a fluorescence per OD unit value (FL/OD). The average FL/OD of the negative control culture with empty vector (R0040) was     subtracted from the FL/OD of the devices to remove background cell fluorescence, resulting in a FL/OD600 value. These values were then divided by the     conversion factor determined from the standard curve generated from a sodium fluorescein control assay to derived a final unit value as the equivalent     ng/mL fluorescein per OD600 unit.
Fluorescein isothiocyanate (FITC) standards were generated using serial dilution of a 0.01 mg/mL FITC stock in 1x Dulbecco’s Phosphate Buffered Saline     (1xDPBS, pH 7.4) diluted in MilliQ 18omega ultrapure water. 500, 375, 250, 125, 50 25, 10, 5, and 0ng/mL standards were generated from the stock using     serial dilutions. 200uL samples of each stock were loaded into a 96-well plate and assayed for fluorescence. Raw fluorescence data was blanked using the     background fluorescence of pure 1x DPBS, divided by the concentration of the standard to generate a FL per ng/mL unit, then averaged to generated a     conversion factor for fluorescence of samples on the plate reader to equivalent ng/mL of fluorescein.
Units reported:
Equivalent ng/mL fluorescein per OD600 unit
Constructs/BiologicalReplicates
Fluorescence (Equivalent ng/mL fluorescein per OD600 unit)
Mean of Biological Replicates
Standard Deviation of Biological Replicates
Positive Control - Replicate 1
6116.186937
5202.35468
854.78084
Positive Control - Replicate 2
4422.432086
Positive Control - Replicate 3
5068.445006
Negative Control - Replicate 1
-15.475064
3.29811493
25.2091355
Negative Control - Replicate 2
31.9506414
Negative Control - Replicate 3
-6.58123257
Device 1 - Replicate 1
41019.2474
58340.8108
15074.9127
Device 1 - Replicate 2
65509.757
Device 1 - Replicate 3
68493.4281
Device 2 - Replicate 1
34363.3144
44228.1659
8600.67012
Device 2 - Replicate 2
48168.0919
Device 2 - Replicate 3
50153.0914
Device 3 - Replicate 1
979.809346
736.966187
210.397513
Device 3 - Replicate 2
609.419774
Device 3 - Replicate 3
621.669441
Constructs/BioReplicate and Tech Replicate
Fluorescence (Equivalent ng/mL fluorescein per OD600 unit)
Mean of Technical Replicates
Standard Deviation of Technical Replicates
Positive Control Replicate 1
6116.186937
5034.07492
805.90703
Positive Control - 1-2
6002.302115
Positive Control - 1-3
5937.00772
Positive Control - Replicate 2
4422.432086
Positive Control - 2-2
4177.05199
Positive Control - 2-3
4300.65707
Positive Control - Replicate 3
5068.445006
Positive Control - 3-2
4243.84013
Positive Control - 3-3
5038.75122
Negative Control - Replicate 1
-15.475064
3.2341e-9
16.817082
Negative Control - 1-2
19.4853649
Negative Control - 1-3
-13.693965
Negative Control - Replicate 2
31.9506414
Negative Control - 2-2
-3.0827858
Negative Control - 2-3
-19.011612
Negative Control - Replicate 3
-6.58123257
Negative Control - 3-2
2.89126917
Negative Control 3-3
3.51738325
Device 1 - Replicate 1
41019.2474
63617.4118
8603.2703
Device 1 - 1-2
67188.7633
Device 1 - 1-3
6417.8843
Device 1 - Replicate 2
65509.757
Device 1 - 2-2
65431.4593
Device 1 - 2-3
67239.5673
Device 1 - Replicate 3
68493.4281
Device 1 - 3-2
66337.6542
Device 1 - 3-3
65380.1075
Device 2 - Replicate 1
34363.3144
46680.0032
4842.2311
Device 2 - 1-2
43517.0942
Device 2 - 1-3
49728.0673
Device 2 - Replicate 2
48168.0919
Device 2 - 2-2
46637.8242
Device 2 - 2-3
46698.9807
Device 2 - Replicate 3
50153.0914
Device 2 - 3-1
49590.0286
Device 3 - 3-2
46200.8032
Device 3 - Replicate 1
979.809346
715.689687
180.419777
Device 3- 1-2
984.765256
Device 3 - 1-3
1008.96117
Device 3 - Replicate 2
609.419774
Device 3 - 2-2
612.699404
Device 3 - 2-3
567.412791
Device 3 - Replicate 3
621.669441
Device 3 - 3-2
611.135666
Device 3 - 3-3
571.898632
