Team:UCLA/Notebook/Interlab Study
iGEM UCLA
Home
Team
Meet the Team
Attributions
Collaborations
Projects
Programming Spider Silk
Functionalizing Silk
Honeybee Silk
Materials Processing
Protein Cages
Human Practices
Interlab Study
Parts
Team Parts
Basic Parts
Composite Parts
Part Collection
Notebook
Programming Spider Silk
Functionalizing Silk
Honeybee Silk
Materials Processing
Protein Cages
Interlab Study
Safety
Judging
Contents
1 The 2015 UCLA iGEM Interlab/Measurements Study Notebook
2 Introduction
3 Experimental Design
4 Protocols
5 Where to go from here
6 Raw lab notebook entries
The 2015 UCLA iGEM Interlab/Measurements Study Notebook
Introduction
The 2015 UCLA iGEM Team is proud to participate in the Second International InterLab Measurement Study in synthetic biology.  As members of the synthetic biology community, we are committed to providing robust data for development of novel characterization methods in the rapidly growing biological design fields of synthetic biology.
The purpose of the 2015 InterLab study is to "measure and characterize fluorescence data for three specific genetic devices" expressing GFPmut3b (SwissProt: P42212) from active iGEM teams participating around the world.  By collecting fluorescence data from multiple teams in absolute units, variability in measurement and consistency of data collected from instrumentation following a uniform procedure can be determined within a high degree of accuracy.
This notebook will record all protocols, daily experiments, basic parameters and images, as well as the raw data used to prepare the Interlab Worksheet, Protocol, and Wiki page for submission at the 2015 Giant Jamboree.
Experimental Design
Three separate genetic "devices" were constructed using IDT gBlocks Gene Fragments synthesis, in addition to positive control BBa_I20270 (Constitutive Family Promoter J23151 inserted upstream of the promoter MeasKit) and negative control BBa_R0040 (pTetR - empty control plasmid). All gBlocks were designed on Benching to simulate the sequence of the BioBricks standard assembly product, for uniformity in measurement with teams that opted for RFC10 standard assembly.  All devices were subcloned in the standard pSB1C3 (chloramphenicol resistance marker) backbone and transformed into BL21(DE3) Escherichia coli . As such, E. coli BL21(DE3) laboratory strains were used as the chassis for fluorescent measurement.  Details as to the location of the registry pieces used to construct the devices are below:
Device #
Benchling Link
Spec Sheet & FASTA File
Promoter
GFP Generator
Final Device Backbone
Device #1
https://benchling.com/s/EnB0uD9g/edit
Specs and FASTA
BBa_J23101
BBa_I3504
(B0034-E0040-B0015)
pSB1C3
Device #2
https://benchling.com/s/hX68sjpy/edit
Specs and FASTA
BBa_J23106
BBa_I3504
pSB1C3
Device #3
https://benchling.com/s/8q493PdY/edit
Specs and FASTA
BBa_J23117
BBa_I3504
pSB1C3
Positive Control BBa_I20270
N/A
N/A
BBa_J23151
GFPmut3b Promoter MeasKit
(B0032-E0040-B0010-B0012)
pSB1C3
Negative Control BBa_R0040
N/A
N/A
TetR repressible promoter (BBa_R0040)
N/A
pSB1C3
All devices constructed using the IDT gBlocks gene fragment synthesis scheme.  Biobrick scar sites between the promoter and GFP generator sequence were preserved to simulate traditional Biobricks standard assembly.
Protocols
The following are a list of protocols designed for the InterLab Study.
Preparation of chemically competent E. coli BL21(DE3) cells using Zymo Mix & Go Transformation Kit and Buffer Set
Rapid isolation of plasmid DNA (pSB1C3) using the Promega PureYield MiniPrep System (Miniprep)
Double digestion of plasmid DNA using EcoRI and PstI restriction exonucleases (NEB EcoRI/PstI Digest)
Rapid ligation and subcloning of gBlocks double digests into pSB1C3 vector backbone using T4 ligase (NEB T4 Ligation)
Zymoclean Gel DNA recovery and purification of plasmid/digested DNA following gel electrophoresis (Gel Extraction)
Zymo DNA Clean and Concentrator-5 for rapid cleanup of PCR, Digestion, and Ligation products (PCR Cleanup)
Where to go from here
The following is the process for all 3 devices and the 2 controls (Promoter MeasKit and pTetR empty vector).
1. Order gBlocks gene fragments for all three devices
HERE
2. Design and order primer pair for amplification of all three devices.
HERE and HERE
3. Amplify all three devices, and PCR cleanup/gel extract each reaction.
HERE
Following is for preparation of the 3 devices for cloning into pSB1C3:
4. Digest devices using EcoRI and PstI
HERE
5. Run digests on gels and DNA clean and concentrate all digested pieces.
HERE
6. Ligate digested pieces in empty pSB1C3 vector backbone, and transform using chemical competency. </s> [1] and HERE
7. Pick transformants and prepare cultures for miniprep and glycerol stocking.
8. Digest miniprepped devices using EcoRI and PstI and run on a gel to verify size of the insert.
9. Sequence the 3 devices and compared alignments for predicted sequences to verify proper device implementation.
Following is the plan for fluorescence measurement and data analysis of the three devices, using + and - controls for baseline measurement.  [TBD - Fluorescence measurement and FACS analysis.]
Raw lab notebook entries
July
M
T
W
T
F
S
S
 
 
1
2
3
4
5
6
7
8
9
10
11
12
13
14
15
16
17
18
19
20
21
22
23
24
25
26
27
28
29
30
31
August
M
T
W
T
F
S
S
 
 
 
 
 
1
2
3
4
5
6
7
8
9
10
11
12
13
14
15
16
17
18
19
20
21
22
23
24
25
26
27
28
29
30
31
September
M
T
W
T
F
S
S
 
1
2
3
4
5
6
7
8
9
10
11
12
13
14
15
16
17
18
19
20
21
22
23
24
25
26
27
28
29
30
