<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_UCLA_Project_Programming_Spider_Silk skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:UCLA/Project/Programming Spider Silk</SPAN></H1><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><P><TITLE>iGEM UCLA</TITLE></P><DIV id="top_menu"><UL id="menu"><LI><A href="/Team:UCLA">Home</A></LI><LI><A href="/Team:UCLA/Team/Meet_the_Team">Team</A></LI><LI><A href="/Team:UCLA/Team/Meet_the_Team">Meet the Team</A></LI><LI><A href="/Team:UCLA/Attributions">Attributions</A></LI><LI><A href="/Team:UCLA/Collaborations">Collaborations</A></LI></UL><LI><A href="/Team:UCLA/Project">Projects</A></LI><LI><A href="/Team:UCLA/Project/Programming Spider Silk">Programming Spider Silk</A></LI><LI><A href="/Team:UCLA/Project/Functionalizing Silk">Functionalizing Silk</A></LI><LI><A href="/Team:UCLA/Project/Honeybee Silk">Honeybee Silk</A></LI><LI><A href="/Team:UCLA/Project/Protein Expression and Processing">Materials Processing</A></LI><LI><A href="/Team:UCLA/Project/Protein Cages">Protein Cages</A></LI><LI><A href="/Team:UCLA/Practices">Human Practices</A></LI><LI><A href="/Team:UCLA/Project/Interlab Study">Interlab Study</A></LI><LI><A href="/Team:UCLA/Parts">Parts</A></LI><LI><A href="/Team:UCLA/Parts">Team Parts</A></LI><LI><A href="/Team:UCLA/Basic_Part">Basic Parts</A></LI><LI><A href="/Team:UCLA/Composite_Parts">Composite Parts</A></LI><LI><A href="/Team:UCLA/Part_Collection">Part Collection</A></LI><LI><A href="/Team:UCLA/Notebook/Weekly Notes">Notebook</A></LI><LI><A href="/Team:UCLA/Notebook/Spider_Silk_Genetics">Programming Spider Silk</A></LI><LI><A href="/Team:UCLA/Notebook/Recombinant_Expression">Functionalizing Silk</A></LI><LI><A href="/Team:UCLA/Notebook/Honeybee_Silk">Honeybee Silk</A></LI><LI><A href="/Team:UCLA/Notebook/Materials">Materials Processing</A></LI><LI><A href="/Team:UCLA/Notebook/Protein_Cages">Protein Cages</A></LI><LI><A href="/Team:UCLA/Notebook/Interlab_Study">Interlab Study</A></LI><LI><A href="/Team:UCLA/Safety">Safety</A></LI><LI><A href="https://igem.org/Judging_Form.cgi?id=1763">Judging</A></LI></DIV><DIV class="page_header"><A href="#background"><DIV class="page_header_box" id="box1"><H1 align="middle" style="position:relative;top:0%;text-decoration:none;font-family:helvetica;font-size:150%;background-color:#0A64A4;">BACKGROUND</H1></DIV></A><A href="#methodology"><DIV class="page_header_box" id="box2"><H1 align="middle" style="position:relative;top:0%;text-decoration:none;font-family:helvetica;font-size:150%;background-color:#FFDE00;">METHODOLOGY</H1></DIV></A><A href="#results"><DIV class="page_header_box" id="box3"><H1 align="middle" style="position:relative;top:0%;text-decoration:none;font-family:helvetica;font-size:150%;background-color:#0A64A4;">RESULTS</H1></DIV></A><A href="#biobricks"><DIV class="page_header_box" id="box4"><H1 align="middle" style="position:relative;top:0%;text-decoration:none;font-family:helvetica;font-size:150%;background-color:#FFDE00;">BIOBRICKS</H1></DIV></A></DIV><DIV class="content_container"><DIV class="page_content" id="section1"><DIV class="content_subsection" id="background"><H1>Programming Spider Silk</H1><H2>Background</H2><H4>Abstract</H4><P>
A major obstacle in creating recombinant spider silk is the highly repetitive nature of the genes that encode it. Silk genes are comprised of a repetitive core region containing ~100 repeats of a spidroin gene which precludes the use of traditional cloning techniques due to non-specificity in primer binding. While other techniques such as head-to-tail assembly or concatemerization have been developed to facilitate spider silk cloning, none of these techniques can assemble silk genes in a quick and directed manner. We have adapted the use of Iterative Capped Assembly (ICA) as a technique to construct silk genes in a rapid and sequence specific manner. By creating a number of different constructs this summer, we show that ICA can greatly facilitate the engineering of recombinant spider-silk.
</P><H4>Introduction</H4><P>
Recombinant spider-silks with customizable properties present an appealing biomaterial that can be used in textiles, tissue scaffolds, and other unique applications. Spider silk is a proteinaceous fiber whose proteins consist of non-repetitive N and C terminal domains, and a highly repetitive central core that consists of up to 100 repeats of Spidroin1 (MaSp1) or Spidroin 2 (MaSp2). These spidroin repeats are directly responsible for the final properties and behavior of the spider silk. MaSp1 contributes strength to the fiber, while MaSp2 contributes elasticity. Importantly, the relative content of MaSp1 and MaSp2 monomers in silk proteins can dictate the fiber properties when spun. Engineering recombinant spider silk genes of varying lengths and spidroin content is nearly impossible using traditional cloning methods, due to the repetitive core. These repeats essentially forbid the use of primers to amplify said genes due to the possibility of non-specific priming. These obstacles have made engineering recombinant spider silk a difficult process.
</P><P>
Existing techniques seek to remove reliance of cloning on primer annealing (Tokareva et al, 2013). Generally, these techniques break the silk genetic into monomeric sequences, then assemble the monomers into the final construct. Head-to-tail cloning assembles gene constructs by ligating two plasmid halves together. Each half carries one of the silk monomers, and the resulting complete plasmid has been doubled. This technique can be used recursively to assemble increasingly large silk genes in a specified manner. Directional recursive ligation uses a similar tactic, where individual monomers are ligated one at a time into a receiving plasmid. Concatemerization is another technique where a pool of monomers are ligated in a single reaction, then cloned into plasmids. This particular technique is useful for creating a library of sizes and compositions.
</P><P>
These existing techniques are not ideal for engineering recombinant silks, because they require repeated and extensive cloning for large constructs, as in the case of head-to-tail assembly and directional recursive ligation, or do not offer any control over the length or genetic composition, as is the case for concatemerization. As it currently stands, there is no one technique that offers rapid and controllable assembly of recombinant spider silk genes.
</P><P>
Iterative Capped Assembly (ICA) is a cloning method that is used to sequentially assemble long,
repetitive DNA sequences. This technique was developed by Briggs et al. in 2012 as a method
to assemble Transcription Activator-Like Effector Nucleases (TALENs) which are sequence
specific DNA binding proteins that consist of multiple repetitive monomers. Each repeat
monomer is responsible for binding to a specific nucleotide in the target sequence. Due to the
repetitive nature of TALE genes, conventional PCR is unable to reliably amplify these sequences
due to non-specific primer binding.
</P><P>
Although ICA was developed using TALE construction as a model problem, this technique can
be used to construct long, repetitive DNA constructs in a directly controllable fashion. ICA assembles repetitive sequences one monomer unit at a time, while preventing the elongation of
incomplete nucleotide chains. The full length sequence is flanked by unique primer annealing
sites, which allows the PCR amplification of the final product. This entire process is performed
using a solid substrate, which greatly facilitates the construction of long sequences.
</P><P>
This summer, we adapt the use of Iterative Capped Assembly (ICA) as a technique for cloning recombinant spider silks in a time-efficient and specific manner that is unparalleled by existing methods. Our experience shows that ICA can be used as a method for the quick assembly of spider silk genes.
</P></DIV><DIV class="content_subsection" id="methodology"><H2>Methodology</H2><OL type="A"><LI>
Iterative capped assembly is similar to Golden Gate assembly, which uses unique sticky ends to assemble gene fragments in a specific order. Whereas Golden Gate is a one-pot reaction with all the pieces ligated simultaneously, ICA is a more “controlled” variation where pieces are assembled one at a time. ICA relies on using 3 different versions of the monomer to be assembled, each of which has different sticky ends such that the monomers must be assembled in an A-B-C fashion. This prevents monomers from self-ligating. Type IIS restriction enzymes (such as BsaI), which cleave outside of the recognition site are used to generate these.
</LI><LI>
 A basic biobrick for ICA consists of a gene monomer flanked by BsaI recognition and cleavage sites. While each of the core monomer is identical, the restriction sites are oriented such that digestion with BsaI yields distinct sticky ends for each of the three types of units. These pieces must be digested to reveal the sticky ends before assembly.
</LI><FIGURE align="middle"><FIGCAPTION style="width: 100%; margin: auto;" align="middle"><B>Fig 1:</B>Schematic of the three types of sticky ends we designed for ICA. Sticky end A is 5’-AGTT-3’. Sticky end B is 5’-TGTC-3’. Sticky end C is 5’-CGTG-3’. An assembled 3-mer construct AB+BC+CA is shown as an example of how these biobricks would be used. <I>Image from Cloning Guide by 2015 TU Eindhoven iGEM team.</I></FIGCAPTION></FIGURE><LI>
Accessory pieces required in ICA include the initiator, streptavidin coated beads, the terminator, and the capping oligos.
     </LI><LI>
     The initiator is a dsDNA fragment made by annealing two ssDNA oligos together. The initiator is designed such that one end is biotinylated, for conjugation to streptavidin coated beads. The other end has a sticky overhang and is designed to anneal to the forward sticky end of the ‘A-type’ monomer unit. This end is 5’-phosphorylated to enable ligation. The initiator also contains a primer binding site that can be used for PCR amplification, as well as other accessory sequences such as affinity tags and the biobrick prefix.
     </LI><LI>
     Streptavidin coated beads serve as a solid support for the elongating DNA chain during ICA. The biotinylated end of the initiator binds to streptavidin to anchor the nascent construct. The ability to physically separate the DNA from solution is needed due to repeated wash and ligation steps used during ICA.
     </LI><LI>
     The terminator is constructed similarly to the initiator, but lacks biotinylation. One end of the terminator is compatible to the reverse sticky end of the ‘C-type’ monomer unit. This end is 5’-phosphorylated to enable ligation. The terminator also contains a primer binding site that can be used for PCR amplification, as well as the biobrick suffix.
     </LI><LI>
      The capping oligos are comprised of a single 5’-phosphorylated ssDNA oligo that can form a stable stem loop structure with a unique sticky end. There are three distinct caps, each of which can bind to the A, B, or C sticky ends
     </LI></OL><FIGURE align="middle"><FIGCAPTION style="margin: auto; width: 100%;" align="middle"><B>Fig 2:</B>Schematic of examples of  initiator (a), terminator (b), and capping oligos (c), used in our ICA project. The cap shown has the B-type sticky end. <I>Image from Cloning Guide by 2015 TU Eindhoven iGEM team.</I></FIGCAPTION></FIGURE><LI>

In each extension step, the next sequential monomer (A, B, or C) is added onto the growing chain. Chains that failed to extend during the previous extension step are capped using a hair-pin oligo that prevents subsequent extension. These capped chains are still present in the mixture for the duration of ICA, but do not participate in any ligation event, and are not amplified in the final PCR. Each final construct is flanked by a biotinylated initiator oligo which allows immobilization onto streptavidin beads, and a terminator oligo. These two oligos provide primer annealing sites which can be used to amplify the sequence using conventional PCR.
</LI><LI>
A generalized workflow is demonstrated below:
     </LI><LI>
     The initiator, terminator, and capping oligos are prepared ahead of time by mixing the relevant oligos and ramping down from 95 C to form the working oligos.
     </LI><LI>
     Monomers of each type (A, B, C) are digested from plasmid with BsaI and purified prior to ICA. These are termed working monomers.
     </LI><LI>
     The initiator is attached to the streptavidin coated beads.
     </LI><LI>
     An ‘A-type’ monomer is ligated to the end of the initiator. Afterwards, any unreacted fragments, as well as the ligase are washed off.
     </LI><LI>
     Next, the ‘B-type’ monomer is ligated to the end of the growing chain. In this reaction mixture, an ‘A-type’ cap is also included, to terminate any chains that failed to extend in the previous ‘A-type’ ligation. Again, unreacted fragments are removed by washing.
     </LI><LI>
      Next, the ‘C-type’ monomer is ligated. This reaction mixture contains the ‘B-type’ cap.
     </LI><LI>
      Next the ‘A-type’ monomer is ligated. This time, the ‘C-type’ cap is also included in the mixture.
     </LI><LI>
     This proceeds in a repetitive fashion until the desired construct length is reached.
     </LI><LI>
     The final constructed is eluted off the beads. The eluate is used as a template for PCR to amplify the construct. Only complete constructs that contain the initiator and terminator are amplified. Capped constructs do not amplify.
     </LI><LI>
     The amplified construct can now be used for downstream cloning.
     </LI><FIGURE align="middle"><FIGCAPTION style="margin: auto; width: 100%;" align="middle"><B>Fig 3:</B>In ICA, the initiator of the growing DNA chains are immobilized on magnetic streptavidin beads to facilitate removal. The first monomer is ligated to the initiator. (Step A) Subsequent ligation steps incorporate capping oligos, which prevent the extension of incorrect chains: those that failed to ligate in the previous round. (Steps B, C) Ligation continues until the construct reaches the final desired length. The final step incorporates a terminator oligo. (Final Step) Only DNA constructs of the correct size can ligate to the terminator, as all incorrect constructs will have been capped. <I>Image from Cloning Guide by 2015 TU Eindhoven iGEM team.</I></FIGCAPTION></FIGURE><FIGURE align="middle"><FIGCAPTION style="margin: auto; width: 100%;" align="middle"><B>Fig 4:</B>Downstream Cloning after Iterative Capped Assembly. After elution from the beads, the ICA constructs are amplified using PCR primers that anneal to the initiator and the terminator. These primer binding sites are unique in the construct, and can be found nowhere else in the sequence. The only constructs that are amplified are those that have the initiator and terminator. All other constructs, while present, are excluded from amplification. After amplification, the construct can be cloned into a vector using traditional techniques.  <I>Image from Cloning Guide by 2015 TU Eindhoven iGEM team.</I></FIGCAPTION></FIGURE>

For detailed protocol on how to perform ICA, see <A href="https://2015.igem.org/Team:UCLA/Notebook/Spider_Silk_Genetics/Protocols">Spider Silk Genetics Protocols</A></DIV><DIV class="content_subsection" id="results"><H2>Results</H2><P>
Using ICA, we have generated 10 silk constructs. These include constructs of pure MaSp2 ranging from 3-15 mers, pure MaSp1 of 9 and 12-mers and 12-mers of MaSp1/2 hybrids in 3 different ratios. 
</P><FIGURE align="middle"><FIGCAPTION style="margin: auto; width: 100%;" align="middle"><B>Fig 5:</B>Gel image of constructs we created this summer using ICA. pSB1C3 Plasmids containing the sequence verified construct were digested using XbaI and PstI. Results were run on 1% TAE gel. The expected band size for the pSB1C3 is ~2070. Expected sizes for inserts fragments are indicated on the right hand side.
</FIGCAPTION></FIGURE><H4>Future Directions</H4><P>
While we were able to construct many sequences of varying length and composition using ICA, we were unable to explore its maximum potential for cloning repetitive genes. We have not established an upper limit on the number of monomers able to be assemble using ICA. In addition, we have not explored the extended use of ICA to create extremely large (greater than 50 monomer units)
</P><H4>Achievements</H4><OL><LI>
Successfully improved last year’s biobrick BBa_K1384000
     </LI><LI>
     Redesigned MaSp1 and MaSp2 monomers with modified sticky ends, and cloned these into biobricks.
     </LI></OL><LI>
Created a collection of parts to be used for Iterative Capped Assembly
</LI><LI>
Demonstrated that ICA is adaptable to silk using our designed sticky ends.
     </LI><LI>
     Optimized ICA for use with spider silk genes to enable fast, efficient assembly of repetitive constructs.
     </LI><LI>
Used ICA to assemble a variety of silk genetic constructs of different length and composition to examine their properties.
     </LI><LI>
     Used ICA to create 10 different silk constructs ranging from 3-mers to 15-mers, and constructs with MaSp1 and MaSp2 in ratios of [1:2], [1:1] and [2:1].
     </LI></DIV><DIV class="content_subsection" id="biobricks"><H2>List of Biobricks</H2><UL><LI><A href="http://parts.igem.org/Part:BBa_K1763002"> MaSp2 AB:               BBa_K1763002 </A></LI><LI><A href="http://parts.igem.org/Part:BBa_K1763003">MaSp2 BC:		BBa_K1763003 </A></LI><LI><A href="http://parts.igem.org/Part:BBa_K1763004">MaSp2 CA:		BBa_K1763004  </A></LI><LI><A href="http://parts.igem.org/Part:BBa_K1763009">MaSp2 SeqAB:		BBa_K1763009 </A></LI><LI><A href="http://parts.igem.org/Part:BBa_K1763010">MaSp1 AB:		BBa_K1763010 </A></LI><LI><A href="http://parts.igem.org/Part:BBa_K1763011">MaSp1 BC:		BBa_K1763011 </A></LI><LI><A href="http://parts.igem.org/Part:BBa_K1763012">MaSp1 CA:		BBa_K1763012 </A></LI><LI><A href="http://parts.igem.org/Part:BBa_K1763023">MaSp1 SeqAB2:	        BBa_K1763423 </A></LI><LI><A href="http://parts.igem.org/Part:BBa_K1763424">M2-3(1C3):		BBa_K1763424 </A></LI><LI><A href="http://parts.igem.org/Part:BBa_K1763425">M2-3(T7):		BBa_K1763425 </A></LI><LI><A href="http://parts.igem.org/Part:BBa_K1763426">M2-6(1C3):		BBa_K1763426 </A></LI><LI><A href="http://parts.igem.org/Part:BBa_K1763427">M2-6(T7):		BBa_K1763427 </A></LI><LI><A href="http://parts.igem.org/Part:BBa_K1763428">M2-9(1C3):		BBa_K1763428 </A></LI><LI><A href="http://parts.igem.org/Part:BBa_K1763429">M2-9(T7):		BBa_K1763429 </A></LI><LI><A href="http://parts.igem.org/Part:BBa_K1763430">M2-12(1C3):		BBa_K1763430 </A></LI><LI><A href="http://parts.igem.org/Part:BBa_K1763431">M2-12(T7):		BBa_K1763431 </A></LI><LI><A href="http://parts.igem.org/Part:BBa_K1763432">M2-15(1C3):		BBa_K1763432 </A></LI><LI><A href="http://parts.igem.org/Part:BBa_K1763433">M2-15(T7):		BBa_K1763433 </A></LI><LI><A href="http://parts.igem.org/Part:BBa_K1763434">M1-9(1C3):		BBa_K1763434 </A></LI><LI><A href="http://parts.igem.org/Part:BBa_K1763435">M1-9(T7):		BBa_K1763435 </A></LI><LI><A href="http://parts.igem.org/Part:BBa_K1763436">M1-12(1C3):		BBa_K1763436 </A></LI><LI><A href="http://parts.igem.org/Part:BBa_K1763437">M1-12(T7):		BBa_K1763437 </A></LI><LI><A href="http://parts.igem.org/Part:BBa_K1763438">M1/2[2:1]-12(1C3):	BBa_K1763438 </A></LI><LI><A href="http://parts.igem.org/Part:BBa_K1763439">M1/2[2:1]-12(T7):	BBa_K1763439 </A></LI><LI><A href="http://parts.igem.org/Part:BBa_K1763440">M1/2[1:1]-12(1C3):	BBa_K1763440 </A></LI><LI><A href="http://parts.igem.org/Part:BBa_K1763441">M1/2[1:1]-12(T7):	BBa_K1763441 </A></LI><LI><A href="http://parts.igem.org/Part:BBa_K1763442">M1/2[1:2]-12(1C3):	BBa_K1763442 </A></LI><LI><A href="http://parts.igem.org/Part:BBa_K1763443">M1/2[1:2]-12(T7):	BBa_K1763443 </A></LI></UL></DIV><DIV class="content_subsection" id="references"><H2>References</H2><P>
Briggs A., Rios X., Chari R., Luhan Y., Zhang F., Mali P., and Church G. Iterative capped assembly: rapid and scalable synthesis of repeat-module DNA such as TAL effectors from individual monomers. <I>Nucleic Acids Research.</I> 2012;<B>40</B>(15): e117
</P><P>
Hinman, M.B., Lewis, R. Isolation of a clone encoding a second dragline silk fibroin. Nephila clavipes dragline silk is a two-protein fiber. J. Biol. Chem. 1992;267: 19320–19324.
</P><P>
Tokareva O., Michalczechen-Lacerda V., Rech E., and Kaplan D. Recombinant DNA production of spider silk proteins. <I>Microbial Biotechnology.</I> 2013;<B>6</B>(6): 651-663
</P><P>
Xu, M., Lewis, R.V. Structure of a protein superfiber: spider dragline silk. PNAS;1990;87, 7120.
</P></DIV></DIV></DIV></DIV></DIV></DIV></DIV></BODY></HTML>