Team:UCLA/Project
iGEM UCLA
Home
Team
Meet the Team
Attributions
Collaborations
Projects
Programming Spider Silk
Functionalizing Silk
Honeybee Silk
Materials Processing
Protein Cages
Human Practices
Interlab Study
Parts
Team Parts
Basic Parts
Composite Parts
Part Collection
Notebook
Programming Spider Silk
Functionalizing Silk
Honeybee Silk
Materials Processing
Protein Cages
Interlab Study
Safety
Judging
HISTORICAL BACKGROUND
SILK BIOCHEMISTRY
APPLICATIONS
PROJECTS
Background
Historical Background
For centuries, humans have fascinated over silk. The earliest known production of the material dates back to the Yangshao period as early as 3000 B.C. Over the years, the process of sericulture, the growth of silkworms to rear silk, has evolved time and time again.  Silk as a material was at its height during the famed Silk Road period, where the lucrative Chinese silk trade dominated much of transcontinental trade. Long after, however, the idea behind silk processing still remains the same: growing silkworms, from which silk is extracted, cleaned and further processed.   For our project, we intend to take this a few steps further: instead of growing silkworms in factories, we want bacteria to be our microbial factories, producing silk proteins as it would any other.
Biochemistry of Silk
The mechanical properties of spider silk are a result of its primary amino acid sequence. Although the   amino acid sequences for various types of spider silk are well documented, it is still relatively unknown   how these amino acids aggregate and contribute to the strength and stability of silk. Silk proteins are   composed of repeats of short amino acid sequences (approximately 33 amino acids). These repeats can   be referred to as monomers. Each monomer consists of a glycine rich stretch, followed by an alanine   rich stretch. It is hypothesized that these hydrophobic alanines form beta sheet crystals with other silk   proteins, and contribute to the strength of the silk fibers. The glycine stretches are thought to form   alpha helices and contribute to the flexibility of the protein. This monomer sequence is repeated many   times in one protein, and allow for strong interactions with other silk proteins. These simple motifs   repeated over and over are the key to the formation of one of the strongest materials known to man.
Applications
Silk is a naturally strong and flexible material, and so it is a unique fiber that can be used in applications that require the normally contradicting qualities of tremendous strength and low weight. It could be woven into thick cables for heavy industrial use or into fabrics for ballistic protection.    Silk not only has amazing properties in its strength and elasticity, it is a highly versatile material that can exist in many different forms. For example, its ability to exist as a hydrogel combined with its natural biocompatibility and controlled rate of biodegradation make it a good material for use as an in vivo tissue scaffold. Silk scaffolds could also exist as mesh woven from fibers and used for skin grafts and bandages.
Projects
Customizing Silk
Modifying the genetic structure of silk can create a diverse new range of biomaterials.
With spider silk as our basis, we started to experiment with the genetic structure of silk. We rearranged portions of the repetitive DNA code to see how it affected the strength, elasticity, and spinning capabilities using Golden Gate Cloning and Iterative Capped Assembly. Click this text to learn about this portion of our project in-depth.
Functionalizing Fibers
By attaching various proteins, we can create silk with specific applications.
Glowing silk is an idea that appeals both to the scientific and artistic worlds. It is a proof of concept for the scientists, demonstrating how we are able to functionalize silk fibers, and an interesting new material to work with for the artists. This method can be applied to different kinds of proteins with different functions as well, but every idea has to start small. To see how we attached GFP to our silk constructs, click this text.
Processing Silk
Engineering our silk is only half the work, creating a tangible product is the rest.
After some stumbles in the beginning of summer when our makeshift rotary jet spinner failed to work, we decided to look into different methods of spinning synthetic fibers. We tested extrusion with a syringe pump, which gave us very thin, fragile fibers. Click this text to jump into the action and watch fibers take shape from solution.
Contents
1 Project Abstracts
1.1 Programming Spider Silk
1.2 Functionalizing Silk
1.3 Honey Bee Silk
1.4 Protein Expression and Processing
1.5 https://2015.igem.org/Team:UCLA/Project/Protein_Cages Protein Cages]
Project Abstracts
Programming Spider Silk
A major obstacle in creating recombinant spider silk is the highly repetitive nature of the genes that encode it. Silk genes are comprised of a repetitive core region containing ~100 repeats of a spidroin gene which precludes the use of traditional cloning techniques due to non-specificity in primer binding. While other techniques such as head-to-tail assembly or concatemerization have been developed to facilitate spider silk cloning, none of these techniques can assemble silk genes in a quick and directed manner. We have adapted the use of Iterative Capped Assembly (ICA) as a technique to construct silk genes in a rapid and sequence specific manner. By creating a number of different constructs this summer, we show that ICA can greatly facilitate the engineering of recombinant spider-silk.
Functionalizing Silk
Silk fibers possess the potential to be transformed into functional biomaterials that can be exploited in an array of biomedical applications, from aiding nanoscale drug delivery to simulating medical sutures. However, traditional methods of incorporating functional domains into fibers involve difficult, costly, and time-consuming processes. We propose an in vitro, co-spinning method to quickly and efficiently functionalize silk fibers. In essence, we spin a mixture of wild-type silk dope spiked with a small volume of functional domain. This functional domain which will bind to the native silk proteins when co-spun, thereby incorporating itself into the final synthetic fiber. To ensure proper binding of our functional domain, we created a co-spinning module. This module is a genetic construct consisting of our gene of interest flanked on either side by the N and C terminal domains of Bombyx mori (silkworm silk). When co-spun, the termini on our synthetic protein will bind to the respective termini in the native silk proteins, thereby functionalizing the fiber. Our goal is to develop, optimize and experimentally validate our co-spinning module, and assess its potential as a scalable and powerful tool to manufacture silk fibers with an array of functional capacities.
Honey Bee Silk
In addition to the more well known silks from spiders and silkworms, we decided to also explore silk from the honey bee Apis mellifera. We cloned the honey bee silk gene as well as several variants of it and submitted them as the first honey bee silk biobricks. To investigate its potential as a biomaterial, we expressed the silk protein and confirmed its presence using SDS PAGE.
Protein Expression and Processing
Following genetic design of our constructs, we must express and process them into functional materials. Here, we highlight the methods that we used to take our silks from DNA to proteins and ultimately to fibers and films.
https://2015.igem.org/Team:UCLA/Project/Protein_Cages Protein Cages]
The goal of our project is to modify a protein cage such that disassembly can be induced by thrombin protease. Our protein cage is a previously-created synthetic cage composed of 12 subunits that form a tetrahedron. By creating mutant protein cages with inserted thrombin cleavage sites, disassembly could be induced by treatment with thrombin protease. We designed 13 mutant protein cages and successfully cloned our top candidate based on selection criteria. We were then able to successfully express and affinity purify both the native protein cage and the mutant protein cage. However, complications with protein aggregation occurred, and upon assaying for protein cage assembly using dynamic light scattering, we found that our cages did not assemble at the expected size of 16nm. Future directions will be focused on improving expression conditions for proper cage assembly, as well as on functionalizing the interior/exterior.
