<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_UCLA_Notebook_Protein_Cages_25_June_2015 skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:UCLA/Notebook/Protein Cages/25 June 2015</SPAN></H1><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><P><TITLE>iGEM UCLA</TITLE></P><DIV id="top_menu"><UL id="menu"><LI><A href="/Team:UCLA">Home</A></LI><LI><A href="/Team:UCLA/Team/Meet_the_Team">Team</A></LI><LI><A href="/Team:UCLA/Team/Meet_the_Team">Meet the Team</A></LI><LI><A href="/Team:UCLA/Attributions">Attributions</A></LI><LI><A href="/Team:UCLA/Collaborations">Collaborations</A></LI></UL><LI><A href="/Team:UCLA/Project">Projects</A></LI><LI><A href="/Team:UCLA/Project/Programming Spider Silk">Programming Spider Silk</A></LI><LI><A href="/Team:UCLA/Project/Functionalizing Silk">Functionalizing Silk</A></LI><LI><A href="/Team:UCLA/Project/Honeybee Silk">Honeybee Silk</A></LI><LI><A href="/Team:UCLA/Project/Protein Expression and Processing">Materials Processing</A></LI><LI><A href="/Team:UCLA/Project/Protein Cages">Protein Cages</A></LI><LI><A href="/Team:UCLA/Practices">Human Practices</A></LI><LI><A href="/Team:UCLA/Project/Interlab Study">Interlab Study</A></LI><LI><A href="/Team:UCLA/Parts">Parts</A></LI><LI><A href="/Team:UCLA/Parts">Team Parts</A></LI><LI><A href="/Team:UCLA/Basic_Part">Basic Parts</A></LI><LI><A href="/Team:UCLA/Composite_Parts">Composite Parts</A></LI><LI><A href="/Team:UCLA/Part_Collection">Part Collection</A></LI><LI><A href="/Team:UCLA/Notebook/Weekly Notes">Notebook</A></LI><LI><A href="/Team:UCLA/Notebook/Spider_Silk_Genetics">Programming Spider Silk</A></LI><LI><A href="/Team:UCLA/Notebook/Recombinant_Expression">Functionalizing Silk</A></LI><LI><A href="/Team:UCLA/Notebook/Honeybee_Silk">Honeybee Silk</A></LI><LI><A href="/Team:UCLA/Notebook/Materials">Materials Processing</A></LI><LI><A href="/Team:UCLA/Notebook/Protein_Cages">Protein Cages</A></LI><LI><A href="/Team:UCLA/Notebook/Interlab_Study">Interlab Study</A></LI><LI><A href="/Team:UCLA/Safety">Safety</A></LI><LI><A href="https://igem.org/Judging_Form.cgi?id=1763">Judging</A></LI></DIV><P>Intro: Today we designed the primer sequence for our codon optimized cage sequence from Yates' lab. I initially researched the iGEM specifics for the prefixes and suffix they require, so that we knew how to design primers for the sequence. Additionally, another set of primers with NdeI (prefix) and XhoI (suffix) sites will be designed for protein expression within the pET22b vector, since this plasmid was used by Dr. Yates.
</P><P>We first spoke with Fasih to learn how to order gBlocks and to confirm our primer design methodology. This is what we learned:
</P><P>1. select gBlocks gene fragments on the IDT (integrated DNA technologies) website
</P><P>2. name the order
</P><P>3. copy the gene from benchling into the sequence window
</P><P>4. test the complexity to make sure its good to go
</P><P>5. say no to all the questions asked by IDT
</P><P>6. Notes:
A. for payment, use oilgocard
B. send to david attn. iGEM
C. 1,000 ng is a good guaranteed yield
</P><P>Remember to resuspend the gene at a concentration of 10 ng/microL. 
</P><P>Add a double stop codon. 
Snap gene is a good place to look for plasmid sequences.
</P><P>Tyler Lee
--<A href="/wiki/index.php?title=User:Wtleeiv&amp;action=edit&amp;redlink=1" class="new" title="User:Wtleeiv (page does not exist)">Wtleeiv</A> 19:27, 25 June 2015 (CDT)
</P><P>
Introduction:
During our downtime with either lab protocols or orders to arrive, we decided to do a formal journal club within the protein cages group. Our first journal club will involve PCR(procedure,theory, application) and site directed mutagenesis as these will be heavily used in the project. 
</P><P>1.I was preparing for this journal club by finding papers and preparing a powerpoint presentation for this.
First I looked up PCR on google then I got titles of some of the original papers on PCR. I typed these papers into google scholar which led me to the source of the papers(Science, Nature etc.) 
</P><P>2.I attended the meeting with David and Philip to go over our PCquad sequence and to decide what exactly to order. 
</P><P>Nithin
</P><P>
Introduction:  Today there will be a meeting with David to discuss the overview of our project.  There was a powerpoint created for the main points.  Fasih will then be assisting with ordering the parts.  
</P><P>First the optimized cage was verified to be the same amino acid sequence as the original.  This was done using Expasy and then was simply matched on Microsoft word.
</P><P>PCcage original (from Dr. Yeates)
</P><P>MPFITVGQENSTSIDLYYEDHGTGTPVVLIHGFPLSGHSWERQSAALLDAGARVITYDRR
GFGQSSQPTTGYDYDTFAADLNTVLETLDLQDAVLVGFSMGTGEVARYVSSYGTARIAAV
AFLASLEPFLLKTDDNPDGAAPQEFFDGIVAAVKADRYAFYTGFFNDFYNLDENLGTRIS
EEAVRNSWNTAASGGFFAAAAAPTTWYTDFRADIPRIDVPALILHGTGDRTLPIENTARV
FHKALPSAEYVEVEGAPHGLLWTHAEEVNTALLAFLAKAQEAQKQKLLTEVETYVLSIIP
SGPLKAEIAQRLEDVFAGKNTDLEVLMEWLKTRPILSPLTKGILGFVFTLTVPSERGLQR
RRFVQNALNGNGDPNNMDKAVKLYRKLKREITFHGAKEISLSYSAGALASCMGLIYNRMG
AVTTEVAFGLVCATCEQIADSQHRSHRQLEHHHHHH
</P><P>PCquad optmized
MPFITVGQENSTSIDLYYEDHGTGTPVVLIHGFPLSGHSWERQSAALLDAGARVITYDRR
GFGQSSQPTTGYDYDTFAADLNTVLETLDLQDAVLVGFSMGTGEVARYVSSYGTARIAAV
AFLASLEPFLLKTDDNPDGAAPQEFFDGIVAAVKADRYAFYTGFFNDFYNLDENLGTRIS
EEAVRNSWNTAASGGFFAAAAAPTTWYTDFRADIPRIDVPALILHGTGDRTLPIENTARV
FHKALPSAEYVEVEGAPHGLLWTHAEEVNTALLAFLAKAQEAQKQKLLTEVETYVLSIIP
SGPLKAEIAQRLEDVFAGKNTDLEVLMEWLKTRPILSPLTKGILGFVFTLTVPSERGLQR
RRFVQNALNGNGDPNNMDKAVKLYRKLKREITFHGAKEISLSYSAGALASCMGLIYNRMG
AVTTEVAFGLVCATCEQIADSQHRSHRQ</P><P>Both are the same amino acid sequence except for the XhoI site and the histag at the end of the original.
</P><P>The steps to order the construct is the following.
</P><P>Go to IDT
Sign in
go to gblocks gene fragments
name
copy paste sequence in
test complexity
add to order
say no to everything
for payment:  my account, oligo card,
send to david, notify igem or david
</P><P>Some suggestions from Fasih were the following.
</P><P>Design the primers before ordering.  
a)	For XhoI and NdeI sites include handle of 6-8 basepairs
b)	igem prefix and suffix.  With two stop codons before the suffix.  This will go into psb1c3
perform digest and ligation on benchling
Use snapegene for plasmids
Need BL21 (DE3) with T7 polymerase for expression in pET22b
</P><P>
-Design of primers
Primers were designed using Benchling
For expression:  
These are the NdeI and XhoI sites obtained from Benchling.
NdeI:  5’-CATATG-3’, 3’-GTATAC-5’
XhoI:  5’-CTCGAG-3’, 3’GAGCTC-5’
The handle used was ACTGAC
</P><P>For iGEM:  The iGEM prefix and suffix were used as stated previously.
</P><P>Conclusions:  It seems as though the primer lengths may be too long, and that the melting temperatures are too high.  A second opinion may be needed tomorrow.  The order for just the PCquad optimized will be made in addition to primers to make it usable in the Pet22b vector, and for iGEM.  Once the strains from Yeate’s lab are obtained, the vector will be isolated and our optimized PCquad will be incorporated.  From there, protein expression will be attempted with Yeate’s strain as a control.
</P></DIV></DIV></DIV></DIV></BODY></HTML>