<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_UCLA_Safety skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:UCLA/Safety</SPAN></H1><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><P><TITLE>iGEM UCLA</TITLE></P><DIV id="top_menu"><UL id="menu"><LI><A href="/Team:UCLA">Home</A></LI><LI><A href="/Team:UCLA/Team/Meet_the_Team">Team</A></LI><LI><A href="/Team:UCLA/Team/Meet_the_Team">Meet the Team</A></LI><LI><A href="/Team:UCLA/Attributions">Attributions</A></LI><LI><A href="/Team:UCLA/Collaborations">Collaborations</A></LI></UL><LI><A href="/Team:UCLA/Project">Projects</A></LI><LI><A href="/Team:UCLA/Project/Programming Spider Silk">Programming Spider Silk</A></LI><LI><A href="/Team:UCLA/Project/Functionalizing Silk">Functionalizing Silk</A></LI><LI><A href="/Team:UCLA/Project/Honeybee Silk">Honeybee Silk</A></LI><LI><A href="/Team:UCLA/Project/Protein Expression and Processing">Materials Processing</A></LI><LI><A href="/Team:UCLA/Project/Protein Cages">Protein Cages</A></LI><LI><A href="/Team:UCLA/Practices">Human Practices</A></LI><LI><A href="/Team:UCLA/Project/Interlab Study">Interlab Study</A></LI><LI><A href="/Team:UCLA/Parts">Parts</A></LI><LI><A href="/Team:UCLA/Parts">Team Parts</A></LI><LI><A href="/Team:UCLA/Basic_Part">Basic Parts</A></LI><LI><A href="/Team:UCLA/Composite_Parts">Composite Parts</A></LI><LI><A href="/Team:UCLA/Part_Collection">Part Collection</A></LI><LI><A href="/Team:UCLA/Notebook/Weekly Notes">Notebook</A></LI><LI><A href="/Team:UCLA/Notebook/Spider_Silk_Genetics">Programming Spider Silk</A></LI><LI><A href="/Team:UCLA/Notebook/Recombinant_Expression">Functionalizing Silk</A></LI><LI><A href="/Team:UCLA/Notebook/Honeybee_Silk">Honeybee Silk</A></LI><LI><A href="/Team:UCLA/Notebook/Materials">Materials Processing</A></LI><LI><A href="/Team:UCLA/Notebook/Protein_Cages">Protein Cages</A></LI><LI><A href="/Team:UCLA/Notebook/Interlab_Study">Interlab Study</A></LI><LI><A href="/Team:UCLA/Safety">Safety</A></LI><LI><A href="https://igem.org/Judging_Form.cgi?id=1763">Judging</A></LI></DIV><DIV style="position:relative;margin:auto;background-color:#113372;width:100%;height:200px;top:-25px;"><H1 style="position:relative;top:-20px;text-decoration:none;font-family: 'Nexa Light', sans-serif;color:white;" align="middle"><B>SilkyColi: Reprogramming the physical and functional properties of synthetic silks </B></H1></DIV><DIV class="content_container" style="top:300px;"><DIV class="page_content" id="section1"><DIV class="content_subsection" id="safety"><H1>Safety</H1><P>The <A href="https://2015.igem.org/Safety#Committee">iGEM Safety Committee</A> has approved the UCLA team to receive an exception to the <A href="https://2015.igem.org/Safety/Do_Not_Release">Policy Against Release of Wet-Lab Projects</A>. This exception permits the UCLA team to present samples of bacteria-derived silk at the 2015 Giant Jamboree, as long as the silk remains in a closed container. This exception was granted based on two tests performed in advance:

(1) a swab test that did not show any living transgenic bacteria remaining on a sample after it was processed
(2) a bioinformatic search against <A href="http://www.allergenonline.com/">AllergenOnline</A>, a database of allergenic proteins

Any questions may be directed to the iGEM Safety Commitee (email safety AT igem DOT org).</P><P> As a manufacturing iGEM team, we take safety very seriously, especially in the context of producing unique biomaterials for use in the environmental and biomedical fields.  As such, we focused on maintaining rigorous microbiological and allergenic safety of our synthesized silk products, especially in the context of potentially bringing our silk samples to the Jamboree. </P><H2>Streak-and-Swab Test</H2><P> In order to analyze whether our processing silk samples are free of transgenic bacterial contamination as a byproduct of genetic silk expression, we perform a swab and streak plating analysis of our resultant silk fibers to determine if the silk samples, when swabbed onto a plate, result in growth of bacterial colonies.  Our results, after a web with our sample samples containing proteins of the <I>Bombyx mori</I> fibroin heavy chain (FibH) and the co-spinning nodule containing FibNT, FibCt, and sfGFP proteins, suggests that the fibers a sterile (especially due to the post-processing wash in an 90% isopropanol coagulation bath to strengthen the fibers) (Figure 1, below). </P><H2>Silk Protein Bioinformatic Analysis for Allergen</H2><P>In order to determine if our resulting silk products can induce an allergenic response in individuals handling the silk, we conducted a bioinformatic analysis of all protein sequences in the potential silk structures to assay for putative allergenicity.  Fasta sequences of the amino acid structure of five silk proteins (AmelF3, MaSP1, MaSP2, <I>B. mori heavy chain silk fibroin (FibH)</I> and the co-spinning module were uploaded to <A href="http://allergenonline.com">AllergenOnline</A> and aligned against a large peer-reviewed database of known allergens.  The proteins were analyzed using a sliding 80-mer amino acid window, determining if epitope regions of the silk protein were markers for allergenicity.  Hits were scored on an identity threshold of 35% or higher, and verified for accuracy on a E-valu of 1e-7 or lower.  

Raw results for each protein sample can be found on the following <A href="https://static.igem.org/mediawiki/2015/8/8d/UCLAiGEMAllergenOnlineSilkResults.txt">list</A>.  Overall, the results indicate that these 5 proteins contain no sliding amino acid windows that match the thresholds required for allergenicity; thus indicate our samples are potential hypoallergenic.  This is consistent with previous published results and consensus materials analysis that indicate the poor immunogenicity of our products.  </P><H2>Training</H2><P><B>Briefly describe the topics that you learned about in your safety training.</B></P><P>Our safety training covered hazardous material storage and usage, biohazard usage practices, protective personal equipment, engineering and human safety controls, and in-depth properties of select hazards, such as peroxides and flammable compounds. The laboratory safety training requirements at UCLA can be found <A href="http://policy.ucop.edu/doc/3500598/LabSafetyTraining">here</A></P><P><B>In your country, what are the regulations that govern biosafety in research laboratories? Please give a link to these regulations, or briefly describe them if you cannot give a link.</B></P><P>In the USA, biosafety regulations are provided by the Center for Disease Control and Prevention (CDC) and the American Biological Safety Association (ABSA), and National Institute of Health (NIH)</P><H2>Project Risks</H2><P><B>Risks to the safety and health of team members, or other people working in the lab:</B></P><P><I>E. coli</I> K12 strain does not pose a major safety risk to humans. <I>Nephila clavipes</I> silk, which is the expressed protein product in our project, does not pose a safety risk either. Nevertheless, skin contact and potential ingestion should be eliminated or minimized.</P><P><B>Risks to the safety and health of the general public, particularly if any biological materials escaped from your lab:</B></P><P>Spider silk does not pose any safety or health risk to the environment if released. <I>E. coli</I> may have unpredictable impacts on the local ecosystem if released, but this is unlikely.</P><P><B>What measures are you taking to reduce these risks?</B></P><P>We chose a very safe chassis organism to use, K12 and Bl21(DE3) <I>E.coli</I>. We are also wearing proper Personal Protective Equipment, such as lab coats, gloves, and safety glasses when handling biological materials. Additionally, any surfaces that have been potentially contaminated by bacterial species are thoroughly cleaned using ethanol and/or bleach. </P><H2>Risks of Your Project in the Future</H2><P><B>What new risks might arise from your project's growth? Also, what risks might arise if the knowledge you generate or the methods you develop became widely available?</B></P><P>No safety risks would arise from the availability of our product or the knowledge of how to generate our product. There may be some impact to materials industries if the generation of genetically altered recombinant silk became commercially viable. For example, the natural silk industry may suffer if recombinant silk with altered properties became cheap. </P><P><B>Does your project currently include any design features to reduce risks? Or, if you did all the future work to make your project grow into a popular product, would you plan to design any new features to minimize risks?</B></P><P>We did not utilize any design features to mitigate risks. Expressing our protein causes the chassis bacteria to be metabolically disadvantaged, and so it would likely not survive in the wild.</P><H2>Standard Operating Procedures</H2><P>While the biological materials that we were working with did not present any hazards, a few of our solvents and chemicals did. Lithium bromide and hexafluoroisopropanol (HFIP) are used to solubilize natural and lyophilized silk, respectively. They were critical to our project, but due to their hazardous nature, we had to familiarize ourselves with how to safely and properly use them. We drafted the following Safe Operating Procedures for training and for reference:</P><A href="https://static.igem.org/mediawiki/2014/e/ed/HFIP_SOP.pdf">HFIP SOP </A><A href="https://static.igem.org/mediawiki/2014/1/1f/LiBr_SOP.pdf">Anhydrous Lithium Bromide SOP </A></DIV></DIV></DIV></DIV></DIV></DIV></DIV></BODY></HTML>