<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_UCLA_Project_Interlab_Study skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:UCLA/Project/Interlab Study</SPAN></H1><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><P><TITLE>iGEM UCLA</TITLE></P><DIV id="top_menu"><UL id="menu"><LI><A href="/Team:UCLA">Home</A></LI><LI><A href="/Team:UCLA/Team/Meet_the_Team">Team</A></LI><LI><A href="/Team:UCLA/Team/Meet_the_Team">Meet the Team</A></LI><LI><A href="/Team:UCLA/Attributions">Attributions</A></LI><LI><A href="/Team:UCLA/Collaborations">Collaborations</A></LI></UL><LI><A href="/Team:UCLA/Project">Projects</A></LI><LI><A href="/Team:UCLA/Project/Programming Spider Silk">Programming Spider Silk</A></LI><LI><A href="/Team:UCLA/Project/Functionalizing Silk">Functionalizing Silk</A></LI><LI><A href="/Team:UCLA/Project/Honeybee Silk">Honeybee Silk</A></LI><LI><A href="/Team:UCLA/Project/Protein Expression and Processing">Materials Processing</A></LI><LI><A href="/Team:UCLA/Project/Protein Cages">Protein Cages</A></LI><LI><A href="/Team:UCLA/Practices">Human Practices</A></LI><LI><A href="/Team:UCLA/Project/Interlab Study">Interlab Study</A></LI><LI><A href="/Team:UCLA/Parts">Parts</A></LI><LI><A href="/Team:UCLA/Parts">Team Parts</A></LI><LI><A href="/Team:UCLA/Basic_Part">Basic Parts</A></LI><LI><A href="/Team:UCLA/Composite_Parts">Composite Parts</A></LI><LI><A href="/Team:UCLA/Part_Collection">Part Collection</A></LI><LI><A href="/Team:UCLA/Notebook/Weekly Notes">Notebook</A></LI><LI><A href="/Team:UCLA/Notebook/Spider_Silk_Genetics">Programming Spider Silk</A></LI><LI><A href="/Team:UCLA/Notebook/Recombinant_Expression">Functionalizing Silk</A></LI><LI><A href="/Team:UCLA/Notebook/Honeybee_Silk">Honeybee Silk</A></LI><LI><A href="/Team:UCLA/Notebook/Materials">Materials Processing</A></LI><LI><A href="/Team:UCLA/Notebook/Protein_Cages">Protein Cages</A></LI><LI><A href="/Team:UCLA/Notebook/Interlab_Study">Interlab Study</A></LI><LI><A href="/Team:UCLA/Safety">Safety</A></LI><LI><A href="https://igem.org/Judging_Form.cgi?id=1763">Judging</A></LI></DIV><DIV style="position:relative;margin:auto;background-color:#113372;width:100%;height:200px;top:-25px;"><H1 style="position:relative;top:-20px;text-decoration:none;font-family: 'Nexa Light', sans-serif;color:white;" align="middle"><B>SilkyColi: Reprogramming the physical and functional properties of synthetic silks </B></H1></DIV><DIV class="content_container"><DIV class="page_content" id="section1"><DIV class="content_subsection" id="background"><H1>2015 InterLab Study</H1><H2>Introduction</H2><P>The 2015 UCLA iGEM Team is proud to participate in the Second International InterLab Measurement Study in synthetic biology. As members of the synthetic biology community, we are committed to providing robust data for development of novel characterization methods in the rapidly growing biological design fields of synthetic biology.
The purpose of the 2015 InterLab study is to &quot;measure and characterize fluorescence data for three specific genetic devices&quot; expressing GFPmut3b (SwissProt: P42212) from active iGEM teams participating around the world. By collecting fluorescence data from multiple teams in absolute units, variability in measurement and consistency of data collected from instrumentation following a uniform procedure can be determined within a high degree of accuracy.

This notebook will record all protocols, daily experiments, basic parameters and images, as well as the raw data used to prepare the Interlab Worksheet, Protocol, and Wiki page for submission at the 2015 Giant Jamboree.</P><H2> Experimental Design </H2><P> Three separate genetic &quot;devices&quot; were constructed using IDT gBlocks Gene Fragments synthesis, in addition to positive control BBa_I20270 (Constitutive Family Promoter J23151 inserted upstream of the promoter MeasKit) and negative control BBa_R0040 (pTetR - empty control plasmid). All gBlocks were designed on Benching to simulate the sequence of the BioBricks standard assembly product, for uniformity in measurement with teams that opted for RFC10 standard assembly. All devices were subcloned in the standard pSB1C3 (chloramphenicol resistance marker) backbone and transformed into BL21(DE3) Escherichia coli . As such, E. coli BL21(DE3) laboratory strains were used as the chassis for fluorescent measurement. Details as to the location of the registry pieces used to construct the devices are below: </P><TABLE class="wikitable"><TBODY><TR><TH> Device #
</TH><TH> Benchling Link
</TH><TH> Spec Sheet &amp; FASTA File
</TH><TH> Promoter
</TH><TH> GFP Generator
</TH><TH> Final Device Backbone
</TH></TR><TR><TD> Device #1
</TD><TD><A rel="nofollow" class="external free" href="https://benchling.com/s/EnB0uD9g/edit">https://benchling.com/s/EnB0uD9g/edit</A></TD><TD><A rel="nofollow" class="external text" href="https://drive.google.com/open?id=0BztPXrXqzHY8WngyWHNwUTdIRmc">Specs</A> and <A rel="nofollow" class="external text" href="https://drive.google.com/open?id=0BztPXrXqzHY8cWJYU3FRU0Fkb2c">FASTA</A></TD><TD><A rel="nofollow" class="external text" href="http://parts.igem.org/Part:BBa_J23101">BBa_J23101</A></TD><TD><A rel="nofollow" class="external text" href="http://parts.igem.org/Part:BBa_I3504">BBa_I3504</A><P>(B0034-E0040-B0015)
</P></TD><TD>pSB1C3
</TD></TR><TR><TD> Device #2
</TD><TD><A rel="nofollow" class="external free" href="https://benchling.com/s/hX68sjpy/edit">https://benchling.com/s/hX68sjpy/edit</A></TD><TD><A rel="nofollow" class="external text" href="https://drive.google.com/open?id=0BztPXrXqzHY8SW1PcXk5RERNeGs">Specs</A> and <A rel="nofollow" class="external text" href="https://drive.google.com/open?id=0BztPXrXqzHY8U0lfZFJ6bWgzTWc">FASTA</A></TD><TD><A rel="nofollow" class="external text" href="http://parts.igem.org/Part:BBa_J23106">BBa_J23106</A></TD><TD><A rel="nofollow" class="external text" href="http://parts.igem.org/Part:BBa_I3504">BBa_I3504</A></TD><TD>pSB1C3
</TD></TR><TR><TD> Device #3
</TD><TD><A rel="nofollow" class="external free" href="https://benchling.com/s/8q493PdY/edit">https://benchling.com/s/8q493PdY/edit</A></TD><TD><A rel="nofollow" class="external text" href="https://drive.google.com/open?id=0BztPXrXqzHY8cUFVcE1QSHFhZzA">Specs</A> and <A rel="nofollow" class="external text" href="https://drive.google.com/open?id=0BztPXrXqzHY8bUt2eDZOMjlSckk">FASTA</A></TD><TD><A rel="nofollow" class="external text" href="http://parts.igem.org/Part:BBa_J23117">BBa_J23117</A></TD><TD><A rel="nofollow" class="external text" href="http://parts.igem.org/Part:BBa_I3504">BBa_I3504</A></TD><TD>pSB1C3
</TD></TR><TR><TD> Positive Control <A rel="nofollow" class="external text" href="http://parts.igem.org/Part:BBa_I20270">BBa_I20270</A></TD><TD> N/A
</TD><TD> N/A
</TD><TD><A rel="nofollow" class="external text" href="http://parts.igem.org/Part:BBa_J23151">BBa_J23151</A></TD><TD> GFPmut3b Promoter MeasKit
<P>(B0032-E0040-B0010-B0012)
</P></TD><TD>pSB1C3
</TD></TR><TR><TD> Negative Control <A rel="nofollow" class="external text" href="http://parts.igem.org/Part:BBa_R0040">BBa_R0040</A></TD><TD> N/A
</TD><TD> N/A
</TD><TD> TetR repressible promoter (BBa_R0040)
</TD><TD> N/A
</TD><TD> pSB1C3
</TD></TR></TBODY></TABLE><H2><SPAN class="mw-headline" id="Protocols"><U> Protocols </U></SPAN></H2><P>The following are a list of protocols designed for the InterLab Study. 
</P><UL><LI> Preparation of chemically competent E. coli BL21(DE3) cells using <A rel="nofollow" class="external text" href="http://www.zymoresearch.com/downloads/dl/file/id/166/t3001i.pdf">Zymo Mix &amp; Go Transformation Kit and Buffer Set</A></LI><LI> Rapid isolation of plasmid DNA (pSB1C3) using the <A rel="nofollow" class="external text" href="https://www.promega.com/~/media/files/resources/protcards/pureyield%20plasmid%20miniprep%20system%20quick%20protocol.pdf">Promega PureYield MiniPrep System</A> (Miniprep)</LI><LI> Double digestion of plasmid DNA using EcoRI and PstI restriction exonucleases <A rel="nofollow" class="external text" href="http://nebcloner.neb.com/#!/protocol/re/double/EcoRI,PstI">(NEB EcoRI/PstI Digest)</A></LI><LI> Rapid ligation and subcloning of gBlocks double digests into pSB1C3 vector backbone using T4 ligase <A rel="nofollow" class="external text" href="https://www.neb.com/protocols/1/01/01/dna-ligation-with-t4-dna-ligase-m0202">(NEB T4 Ligation)</A></LI><LI> Zymoclean Gel DNA recovery and purification of plasmid/digested DNA following gel electrophoresis <A rel="nofollow" class="external text" href="http://www.zymoresearch.com/downloads/dl/file/id/34/d4001i.pdf">(Gel Extraction)</A></LI><LI> Zymo DNA Clean and Concentrator-5 for rapid cleanup of PCR, Digestion, and Ligation products <A rel="nofollow" class="external text" href="http://www.zymoresearch.com/downloads/dl/file/id/35/d4003i.pdf">(PCR Cleanup)</A></LI></UL><H2> Results </H2><H3> Persons responsible for conducting InterLab Study </H3><P> Fasih Ahsan: Designed the Interlab protocols and methodology, cloned the devices and controls, measured, and processed the data. </P><P> Megan Satyadi: Assisted in transforming the device construct into the <I>E. coli</I> BL21(DE3) cell chassis, and growing the final overnight cultures of each device cell line in biological triplicates.</P><H3> What chassis did you use? </H3><P><I>E. coli</I> BL21(DE3) </P><H3> What Biosafety Level is your chassis? </H3><P> BSL-1 - non-pathogenic bacterial strain</P><H3> What PPE did you utilize during your experiments (from cloning through to measuring the devices)?</H3><P><STRONG>Individuals responsible for conducting Interlab study: </STRONG></P><P>
    Fasih Ahsan – Designed the InterLab protocols and methodology, cloned the devices and controls, measured and processed the data.
</P><P>
    Megan Satyadi – Assisted in transforming the device constructs into the E. coli BL21(DE3) cell chassis, and growing the final overnight cultures of each
    device cell line in biological triplicates.
</P><P><STRONG><U>Chassis and Safety Information</U></STRONG></P><P><STRONG>What chassis did you use?</STRONG></P><P>
    E. coli Bl21(DE3)
</P><P><STRONG>What Biosafety Level is your chassis?</STRONG></P><P>
    BSL1
</P><P><STRONG>What PPE did you utilize during your experiments (from cloning through to measuring the devices)?</STRONG></P><P>
    Long pants, ankle length socks, closed-toed shoes, nitrile blue gloves, safety goggles, flame-resistant blue lab coats.
</P><P><STRONG><U>Information about Cloning</U></STRONG></P><P><STRONG>What DNA assembly method did you use to create your devices?</STRONG></P><P>
    DNA Synthesis
</P><P><STRONG>How did you validate the final devices?</STRONG></P><P>
    DNA Sequencing
</P><P>
    OTHER: Colony PCR
</P><P><STRONG>Please tell us about any challenges or problems you encountered during the cloning process.</STRONG></P><P>
    We had difficulties performing standard BioBricks assembly of the final device constructs, namely in the initial transformation of the GFP reporter
    construct BBa_I13504. We attempted to transform both the pSB1A2 and pSB1C3 variants, using both chemically competent and electrocompetent transformation
    line protocols. We observed no colony growth from the DNA products derived from the Registry we were given, so we had to resort to ordering each device
    sequence using IDT gBlocks gene synthesis.
</P><P>
    Additionally, we have difficult with false positive clones in Devices 2 and 3, where the resulting overnighted cultures from selected colonies did not
    grow. More colonies were picked and overgrown to alleviate this situation, but it did result in general delays throughout the process. Sequencing of the
    derived plasmid constructs revealed proper Device sequence, so we verified cloning success.
</P><P><STRONG><U>Protocol – Growing Cells for Measurement</U></STRONG></P><P>
    Step 1: Streak out an agar plate with your organism containing the device and any control organisms you used.
</P><P>
    a. Streak out 1 plate per device and control.
</P><P>
    b. Incubate plates overnight (18-20 hours or until individual colonies are clearly visible) at 37 C.
</P><P>
    c. Note the time grown in hours below in “Other”
</P><P>
    d. Other: 18 hours
</P><P><STRONG>Please provide the positive controls used. </STRONG></P><P>
    An E. coli BL21(DE3) cell line containing GFP expression device BBa_I20270 in the pSB1C3 chloramphenicol resistance backbone.
</P><P><STRONG>Please provide the negative control(s) you used:</STRONG></P><P>
    An E. coli BL21(DE) cell line containing pTetR device BBa_R0040 in the pSB1C3 chloramphenicol resistance backbone.
</P><P>
    As a blank for the plate reader, aliquots of LB and LB supplemented with 35ug/uL of chloramphenicol were cultured in the 37 deg C shaking incubator for 18
    hours.
</P><P><STRONG>What type of agar did you use for this step:</STRONG></P><P>
    BD Difco LB Broth, Miller (Luria-Bertani) supplemented with Difco Bacto Agar and 34ug/mL working concentration of chloramphenicol powder dissolved in 100%
    ethanol.
</P><P>
    Step 2: Inoculate liquid culture with your experimental devices and controls:
</P><P>
    a. Device 1
</P><P>
    b. Device 2
</P><P>
    c. Device 3
</P><P>
    d. Positive Control
</P><P>
    e. Negative Control
</P><P>
    f. OTHER: LB + chloramphenicol
</P><P>
    What type of vessel container did you use to grow your cells?
</P><P>
    a. Test tube
</P><P><STRONG>Please provide any detailed information about your vessel/container below.</STRONG></P><P>
    Each device was grown in biological triplicates using 13 x 100 mm Corning Pyrex Vista Test Tubes with fitted plastic aerating caps.
</P><P><STRONG>If you used a test tube, how were the test tubes oriented in the incubator.</STRONG></P><P>
    At an angle.
</P><P><STRONG>What type of media did you use?</STRONG></P><P>
    BD Difco LB Broth, Miller (Luria-Bertani) supplemented with 34ug.mL working concentration of chloramphenicol powder in 100% ethanol.
</P><P><STRONG>What volume did you use to grow your cells?</STRONG></P><P>
    5mL
</P><P><STRONG>Did you set up biological replicates in triplicate?</STRONG></P><P>
    Yes
</P><P>
    Step 3: Incubate your liquid cultures.
</P><P>
    a. Temperature at 37 C
</P><P>
    b. Incubate for 16-18 hours.
</P><P>
    c. Other: Shaking at 200 RPM.
</P><P>
    Plate Reader Measurement
</P><P><STRONG>Did you measure your cells with a plate reader?</STRONG></P><P>
    Yes
</P><P>
    Step 1: Obtain initial OD600 measurement of your overnight cultures.
</P><P>
    a. Set your instrument to read OD600.
</P><P>
    b. Setup a 96-well plate with your cultures.
</P><P>
    c. Take the measurement and record it.
</P><P>
    Step 2: Dilute your samples to an OD600 of 0.5
</P><P>
    a. We did not dilute samples to an OD600 of 0.5. The BioTek plate reader has an OD accuracy between 0-4, and the fluorescence values will be normalized to
    OD600 value for each culture in triplicate.
</P><P>
    Step 3: Measure your samples.
</P><P>
    a. Set your instrument to measure GFP
</P><P>
    b. Measure your cells.
</P><P><STRONG>Alternate Protocol or Additional Details:</STRONG></P><P>
The resulting fluorescence data from the plate reader was processed using the Imperial 2014 iGEM Team InterLab Study:    <A href="https://2014.igem.org/Team:Imperial/InterLab_Study">https://2014.igem.org/Team:Imperial/InterLab_Study</A>. Background absorbance and fluorescence
    were subtracted from all samples assayed as a blanking control. The resulting absorbance values were then used to divide their respective resulting
    fluorescent values to generate a fluorescence per OD unit value (FL/OD). The average FL/OD of the negative control culture with empty vector (R0040) was
    subtracted from the FL/OD of the devices to remove background cell fluorescence, resulting in a FL/OD600 value. These values were then divided by the
    conversion factor determined from the standard curve generated from a sodium fluorescein control assay to derived a final unit value as the equivalent
    ng/mL fluorescein per OD600 unit.
</P><P><STRONG>Please rate your experience with filling in this InterLab Protocol Form.</STRONG></P><P>
    Very easy to fill in, no problems/
</P><P><STRONG>Please let us know any other thoughts or comments you have about the InterLab study experience.</STRONG></P><P>
    I would have preferred a much more difficult and/or rigorous extra credit assignment, actually! Otherwise, great experience and great way to teach data
    analysis methodology for our new members.
</P><P><STRONG>Date of InterLab Study</STRONG></P><P>
    Measurement was obtained on Friday, August 28<SUP>th</SUP>, 2015.
</P><P><STRONG><U>Equipment Information</U></STRONG></P><P><STRONG>What type of incubator did you use to grow your cells?</STRONG></P><P>
    New Brunswick Scientific Innova 44 Incubator Shaker
</P><P><STRONG>If known, what was the incubator’s throw (shaking diameter)</STRONG></P><P>
    The incubators orbit was 2.5 cm (1 in.).
</P><P><STRONG>What piece of equipment did you use to measure the devices?</STRONG></P><P>
    BioTek Synergy H1 Hybrid Multi-Mode Plate Reader, with Gen5 statistical software.
</P><P><STRONG>When was this equipment last calibrated:</STRONG>
    January 2015
</P><P><STRONG>Who calibrated the equipment? : </STRONG>
    UCLA Department of Bioengineering.
</P><P><STRONG>What was the wavelength of light you used to excite the cells: </STRONG>
    485 nm
</P><P><STRONG>What was the filter/channel you used to capture the light emission from the cells:</STRONG>
    528 nm
</P><P><STRONG>What was the sampling frequency? </STRONG></P><P>
    N/A
</P><P><STRONG><U>Protocol:</U></STRONG></P><P><STRONG>Did you fill in the InterLab Protocol provided by the Measurement committee?</STRONG></P><P>
    Yes
</P><P><STRONG><U>Did you follow the InterLab protocol provided by the Measurement committee? </U></STRONG></P><P>
    Yes
</P><P><STRONG><U>How did you determine the final dataset that you are reporting?</U></STRONG></P><P>
The resulting fluorescence data from the plate reader was processed using the Imperial 2014 iGEM Team InterLab Study design:    <A href="https://2014.igem.org/Team:Imperial/InterLab_Study">https://2014.igem.org/Team:Imperial/InterLab_Study</A>. Background absorbance and fluorescence
    were subtracted from all samples assayed as a blanking control. The resulting absorbance values were then used to divide their respective resulting
    fluorescent values to generate a fluorescence per OD unit value (FL/OD). The average FL/OD of the negative control culture with empty vector (R0040) was
    subtracted from the FL/OD of the devices to remove background cell fluorescence, resulting in a FL/OD600 value. These values were then divided by the
    conversion factor determined from the standard curve generated from a sodium fluorescein control assay to derived a final unit value as the equivalent
    ng/mL fluorescein per OD600 unit.
</P><P>
    Fluorescein isothiocyanate (FITC) standards were generated using serial dilution of a 0.01 mg/mL FITC stock in 1x Dulbecco’s Phosphate Buffered Saline
    (1xDPBS, pH 7.4) diluted in MilliQ 18omega ultrapure water. 500, 375, 250, 125, 50 25, 10, 5, and 0ng/mL standards were generated from the stock using
    serial dilutions. 200uL samples of each stock were loaded into a 96-well plate and assayed for fluorescence. Raw fluorescence data was blanked using the
    background fluorescence of pure 1x DPBS, divided by the concentration of the standard to generate a FL per ng/mL unit, then averaged to generated a
    conversion factor for fluorescence of samples on the plate reader to equivalent ng/mL of fluorescein.
</P><P><STRONG><U>Units reported:</U></STRONG></P><P>
    Equivalent ng/mL fluorescein per OD600 unit
</P><DIV class="tg-wrap"><TABLE id="tg-4gSf8" class="tg"><TBODY><TR><TH class="tg-e3zv">Constructs/BiologicalReplicates</TH><TH class="tg-e3zv">Fluorescence (Equivalent ng/mL fluorescein per OD600 unit)</TH><TH class="tg-e3zv">Mean of Biological Replicates</TH><TH class="tg-e3zv">Standard Deviation of Biological Replicates</TH></TR><TR><TD class="tg-z2zr">Positive Control - Replicate 1</TD><TD class="tg-z2zr">6116.186937</TD><TD class="tg-z2zr">5202.35468</TD><TD class="tg-z2zr">854.78084</TD></TR><TR><TD class="tg-031e">Positive Control - Replicate 2</TD><TD class="tg-031e">4422.432086</TD></TR><TR><TD class="tg-z2zr">Positive Control - Replicate 3</TD><TD class="tg-z2zr">5068.445006</TD></TR><TR><TD class="tg-031e">Negative Control - Replicate 1</TD><TD class="tg-031e">-15.475064</TD><TD class="tg-031e">3.29811493</TD><TD class="tg-031e">25.2091355</TD></TR><TR><TD class="tg-z2zr">Negative Control - Replicate 2</TD><TD class="tg-z2zr">31.9506414</TD></TR><TR><TD class="tg-031e">Negative Control - Replicate 3</TD><TD class="tg-031e">-6.58123257</TD></TR><TR><TD class="tg-z2zr">Device 1 - Replicate 1</TD><TD class="tg-z2zr">41019.2474</TD><TD class="tg-z2zr">58340.8108</TD><TD class="tg-z2zr">15074.9127</TD></TR><TR><TD class="tg-031e">Device 1 - Replicate 2</TD><TD class="tg-031e">65509.757</TD></TR><TR><TD class="tg-z2zr">Device 1 - Replicate 3</TD><TD class="tg-z2zr">68493.4281</TD></TR><TR><TD class="tg-031e">Device 2 - Replicate 1</TD><TD class="tg-031e">34363.3144</TD><TD class="tg-031e">44228.1659</TD><TD class="tg-031e">8600.67012</TD></TR><TR><TD class="tg-z2zr">Device 2 - Replicate 2</TD><TD class="tg-z2zr">48168.0919</TD></TR><TR><TD class="tg-031e">Device 2 - Replicate 3</TD><TD class="tg-031e">50153.0914</TD></TR><TR><TD class="tg-z2zr">Device 3 - Replicate 1</TD><TD class="tg-z2zr">979.809346</TD><TD class="tg-z2zr">736.966187</TD><TD class="tg-z2zr">210.397513</TD></TR><TR><TD class="tg-031e">Device 3 - Replicate 2</TD><TD class="tg-031e">609.419774</TD></TR><TR><TD class="tg-z2zr">Device 3 - Replicate 3</TD><TD class="tg-z2zr">621.669441</TD></TR></TBODY></TABLE></DIV><DIV class="tg-wrap"><TABLE id="tg-ieadN" class="tg"><TBODY><TR><TH class="tg-e3zv">Constructs/BioReplicate and Tech Replicate</TH><TH class="tg-e3zv">Fluorescence (Equivalent ng/mL fluorescein per OD600 unit)</TH><TH class="tg-e3zv">Mean of Technical Replicates</TH><TH class="tg-e3zv">Standard Deviation of Technical Replicates</TH></TR><TR><TD class="tg-z2zr">Positive Control Replicate 1</TD><TD class="tg-z2zr">6116.186937</TD><TD class="tg-z2zr">5034.07492</TD><TD class="tg-z2zr">805.90703</TD></TR><TR><TD class="tg-031e">Positive Control - 1-2</TD><TD class="tg-031e">6002.302115</TD></TR><TR><TD class="tg-z2zr">Positive Control - 1-3</TD><TD class="tg-z2zr">5937.00772</TD></TR><TR><TD class="tg-031e">Positive Control - Replicate 2</TD><TD class="tg-031e">4422.432086</TD></TR><TR><TD class="tg-z2zr">Positive Control - 2-2</TD><TD class="tg-z2zr">4177.05199</TD></TR><TR><TD class="tg-031e">Positive Control - 2-3</TD><TD class="tg-031e">4300.65707</TD></TR><TR><TD class="tg-z2zr">Positive Control - Replicate 3</TD><TD class="tg-z2zr">5068.445006</TD></TR><TR><TD class="tg-031e">Positive Control - 3-2</TD><TD class="tg-031e">4243.84013</TD></TR><TR><TD class="tg-z2zr">Positive Control - 3-3</TD><TD class="tg-z2zr">5038.75122</TD></TR><TR><TD class="tg-031e">Negative Control - Replicate 1</TD><TD class="tg-031e">-15.475064</TD><TD class="tg-031e">3.2341e-9</TD><TD class="tg-031e">16.817082</TD></TR><TR><TD class="tg-z2zr">Negative Control - 1-2</TD><TD class="tg-z2zr">19.4853649</TD></TR><TR><TD class="tg-031e">Negative Control - 1-3</TD><TD class="tg-031e">-13.693965</TD></TR><TR><TD class="tg-z2zr">Negative Control - Replicate 2</TD><TD class="tg-z2zr">31.9506414</TD></TR><TR><TD class="tg-031e">Negative Control - 2-2</TD><TD class="tg-031e">-3.0827858</TD></TR><TR><TD class="tg-z2zr">Negative Control - 2-3</TD><TD class="tg-z2zr">-19.011612</TD></TR><TR><TD class="tg-031e">Negative Control - Replicate 3</TD><TD class="tg-031e">-6.58123257</TD></TR><TR><TD class="tg-z2zr">Negative Control - 3-2</TD><TD class="tg-z2zr">2.89126917</TD></TR><TR><TD class="tg-031e">Negative Control 3-3</TD><TD class="tg-031e">3.51738325</TD></TR><TR><TD class="tg-z2zr">Device 1 - Replicate 1</TD><TD class="tg-z2zr">41019.2474</TD><TD class="tg-z2zr">63617.4118</TD><TD class="tg-z2zr">8603.2703</TD></TR><TR><TD class="tg-031e">Device 1 - 1-2</TD><TD class="tg-031e">67188.7633</TD></TR><TR><TD class="tg-z2zr">Device 1 - 1-3</TD><TD class="tg-z2zr">6417.8843</TD></TR><TR><TD class="tg-031e">Device 1 - Replicate 2</TD><TD class="tg-031e">65509.757</TD></TR><TR><TD class="tg-z2zr">Device 1 - 2-2</TD><TD class="tg-z2zr">65431.4593</TD></TR><TR><TD class="tg-031e">Device 1 - 2-3</TD><TD class="tg-031e">67239.5673</TD></TR><TR><TD class="tg-z2zr">Device 1 - Replicate 3</TD><TD class="tg-z2zr">68493.4281</TD></TR><TR><TD class="tg-031e">Device 1 - 3-2</TD><TD class="tg-031e">66337.6542</TD></TR><TR><TD class="tg-z2zr">Device 1 - 3-3</TD><TD class="tg-z2zr">65380.1075</TD></TR><TR><TD class="tg-031e">Device 2 - Replicate 1</TD><TD class="tg-031e">34363.3144</TD><TD class="tg-031e">46680.0032</TD><TD class="tg-031e">4842.2311</TD></TR><TR><TD class="tg-z2zr">Device 2 - 1-2</TD><TD class="tg-z2zr">43517.0942</TD></TR><TR><TD class="tg-031e">Device 2 - 1-3</TD><TD class="tg-031e">49728.0673</TD></TR><TR><TD class="tg-z2zr">Device 2 - Replicate 2</TD><TD class="tg-z2zr">48168.0919</TD></TR><TR><TD class="tg-031e">Device 2 - 2-2</TD><TD class="tg-031e">46637.8242</TD></TR><TR><TD class="tg-z2zr">Device 2 - 2-3</TD><TD class="tg-z2zr">46698.9807</TD></TR><TR><TD class="tg-031e">Device 2 - Replicate 3</TD><TD class="tg-031e">50153.0914</TD></TR><TR><TD class="tg-z2zr">Device 2 - 3-1</TD><TD class="tg-z2zr">49590.0286</TD></TR><TR><TD class="tg-031e">Device 3 - 3-2</TD><TD class="tg-031e">46200.8032</TD></TR><TR><TD class="tg-z2zr">Device 3 - Replicate 1</TD><TD class="tg-z2zr">979.809346</TD><TD class="tg-z2zr">715.689687</TD><TD class="tg-z2zr">180.419777</TD></TR><TR><TD class="tg-031e">Device 3- 1-2</TD><TD class="tg-031e">984.765256</TD></TR><TR><TD class="tg-z2zr">Device 3 - 1-3</TD><TD class="tg-z2zr">1008.96117</TD></TR><TR><TD class="tg-031e">Device 3 - Replicate 2</TD><TD class="tg-031e">609.419774</TD></TR><TR><TD class="tg-z2zr">Device 3 - 2-2</TD><TD class="tg-z2zr">612.699404</TD></TR><TR><TD class="tg-031e">Device 3 - 2-3</TD><TD class="tg-031e">567.412791</TD></TR><TR><TD class="tg-z2zr">Device 3 - Replicate 3</TD><TD class="tg-z2zr">621.669441</TD></TR><TR><TD class="tg-031e">Device 3 - 3-2</TD><TD class="tg-031e">611.135666</TD></TR><TR><TD class="tg-z2zr">Device 3 - 3-3</TD><TD class="tg-z2zr">571.898632</TD></TR></TBODY></TABLE></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></BODY></HTML>