<HTML xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en" dir="ltr">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki  ltr ns-0 ns-subject page-Team_uOttawa_Project"><DIV id="globalWrapper"><DIV id="top-section"><DIV id="p-logo"><A href="/Main_Page" title="Main Page">&quot;
	    </A></DIV><DIV id="menubar" class="left-menu noprint"><UL><LI class="selected"><A href="/Team:uOttawa/Project">Page               </A></LI><LI class="new"><A href="/wiki/index.php?title=Talk:Team:uOttawa/Project&amp;action=edit&amp;redlink=1">Discussion               </A></LI><LI><A href="/wiki/index.php?title=Team:uOttawa/Project&amp;action=edit">View source               </A></LI><LI><A href="/wiki/index.php?title=Team:uOttawa/Project&amp;action=history">History               </A></LI><LI style="color:white;cursor:default">teams</LI></UL></DIV><DIV class="right-menu noprint" id="menubar"><UL><LI id="pt-login"><A href="/wiki/index.php?title=Special:UserLogin&amp;returnto=Team:uOttawa/Project" title="You are encouraged to log in; however, it is not mandatory [o]" accesskey="o">Log in</A></LI></UL></DIV><DIV id="search-controls" class="noprint"><FORM action="/Special:Search" id="searchform"> </FORM></DIV></DIV><DIV id="content"><H1 class="firstHeading">Team:uOttawa/Project</H1><DIV id="bodyContent"><H3 id="siteSub" class="noprint">From 2013.igem.org</H3><DIV id="mg-menubar"><A href="/Team:uOttawa">Home</A><A href="/Team:uOttawa/project">Wet Lab</A><A href="/Team:uOttawa/modeling">Modeling</A><A href="/Team:uOttawa/software">Software</A><A href="/Team:uOttawa/humanpractices">Human Practices</A><A href="/Team:uOttawa/team">Team</A><A href="/Team:uOttawa/parts">Parts</A><A href="/Team:uOttawa/notebook">Notebook</A><A href="/Team:uOttawa/safety">Safety</A><A href="/Team:uOttawa/attributions">Attributions</A></DIV><DIV class="row"><DIV class="container"><H1>Project Description</H1><H2>Background</H2><H3><DIV class="figure floatRight"><P class="caption"><EM>Figure 1.</EM> The type-I incoherent feedforward loop.</P></DIV>
The Type-I Incoherent Feedforward Loop and Fold-Change Detection
</H3><P>The type-I incoherent feedforward loop (I1-FFL) is a gene network in which protein X activates a gene Z while simultaneously activating the production of a repressor of gene Z, labeled Y.</P><P>For this year’s project, we are aiming to construct an I1-FFL that can detect fold-changes in the levels of protein X. In other words, the expression of Z would be reliant on the relative change in X as opposed to absolute values of X. For example, if X changes from an arbitrary concentration of 1 to an arbitrary concentration of 5, the level of expression of gene Z would be exactly the same as that resulting from a change from a concentration of 5 to a concentration of 25 of X (a fold-change by a factor of 5 in both cases).</P><H3>How Fold-Change Detection is Achieved</H3><P>This fold-change detection is enabled through the repressor Y. Take the case when activator X, repressor Y, and protein Z each have an  arbitrary concentration of 1 in the cell (1:1:1 ratio). If the concentration of X is doubled, the ratio between X and Y now becomes 2:1. X immediately activates Z, but since the repressor Y takes time to fold, it lags behind, and the concentration of Z spikes to 2 due to the activation. When Y folds into its active form and carries out its repressive activity, the concentration of Z returns to its initial concentration of 1. This is because at this point, X and Y return to a 1:1 ratio in the cell. In order for the concentration of Z to reach 2 once again, the X:Y ratio must first return to 2:1 - thus, X has to reach a concentration of 4 before the same output of Z is produced. In this way, fold-change detection is achieved.</P><DIV class="figure" style="margin-bottom:0px;"><P class="caption"><EM>Figure 2.</EM> The fold-change detection schematic using an incoherent feedforward loop. Concentrations of X, Y, and Z are shown in arbitrary units.</P></DIV></DIV></DIV><DIV class="row accent"><DIV class="container"><H2>Engineering our System</H2><H3>The Design of our Gene Network and the Interactions Between Components</H3><P>In our proposed design, rtTA will act as our activator (X), Lacl as our repressor (Y), and GFP as our reporter protein (Z). Each part is detectable via a different fluorescent protein. A visual representation of our pathway is shown in Figure 3.</P><DIV class="figure"><P class="caption"><EM>Figure 3.</EM> Network design for fold-change detection of varying levels of rtTA using a type-I incoherent feedforward loop. Red boxes indicate regions of the <I>S.cerevisiae</I> genome to be targeted, while black boxes signify selection markers. Orange arrows represent promoters. Green and blue boxes represent activators and repressors respectively, and purple boxes signify varying fluorescent proteins.</P></DIV><P>This network design allows for calibration of the network via tunable levels of β-estradiol, allowing for accurate control of the amount of rtTA(X) in the cell. This will allow us to ensure that fold-change detection works as theorized in our network.</P><P>Furthermore, activation by rtTA is mediated by varying levels of anhydrotetracycline (ATc), and LacI repression is mediated by varying levels of Isopropyl β-D-1-thiogalactopyranoside (IPTG). These methods of mediation allow for high tunability of the system, and we should be able to find a concentration range of ATc and IPTG at which the system will work as desired.</P><P>The system will then be characterized extensively. The fluorescent markers tagged to each component of the network will allow for simple temporal analysis using flow cytometry. This will allow for a high-throughput analysis of the activity of our network.</P></DIV></DIV><DIV class="row"><DIV class="container"><H2>The Potential Application</H2><H3>A Fold-Change Detection System for Toxic Molecules</H3><P>Once our system is tuned and is working, it can be modified to act as a fold-change detector for toxic molecules. By replacing the pGEV promoter in front of the rtTA gene with a promoter that is inducible by a toxic molecule, the amount of rtTA produced will be directly correlated to the concentration of the toxic molecule in the cellular environment.</P><DIV class="figure"><P class="caption"><EM>Figure 4.</EM> Modification of the network illustrated in Figure 3 to enable toxic molecule detection. The pGEV promoter will be exchanged for a promoter responsive to a toxic molecule.</P></DIV><P>This detector system is advantageous in that it detects fold-changes rather than changes in absolute values, which gives meaning to a signal in reference to the background signal level. This allows for the network to respond only to a signal that rises significantly above the background noise.</P></DIV></DIV><DIV class="row borderTop accent"><DIV class="container"><H2>References</H2><UL class="reference"><LI>Ellis, Tom, Xiao Wang, and James J Collins. &quot;Diversity-based, model-guided construction of synthetic gene networks with predicted functions&quot;. <I>Nature Biotechnology</I> 27.5 (2009): 465-471. Web. 30 May 2013.</LI><LI>Goentoro, Lea, Oren Shoval, Marc W. Kirschner, and Uri Alon. &quot;The Incoherent Feedforward Loop Can Provide Fold-Change Detection in Gene Regulation&quot;. <I>Molecular Cell</I> 36 (2009): 894-899. Print.</LI><LI>Mads Kærn and Ron Weiss. &quot;Synthetic gene regulatory systems&quot;. In: <I>Systems Modeling in Cellular Biology.</I> Z. Sallasi <I>et al.</I> (Eds.). MIT Press (2006).</LI><LI>Mangan, Shmoolik and Uri Alon. &quot;Structure and function of the feed-forward loop network motif&quot;. <I>Proceedings of the National Academy of Sciences</I> 100.21 (2003): 11980–11985. Web. 9 May 2013.</LI><LI>McIsaaca, R. Scott, Sanford J. Silvermana, Megan N. McCleana, Patrick A. Gibneya, Joanna Macinskasa, Mark J. Hickmana, Allegra A. Pettia, and David Botsteina. &quot;Fast-acting and nearly gratuitous induction of gene expression and protein depletion in <I>Saccharomyces cerevisiae</I>&quot;. <I>Molecular Biology of the Cell</I> 22 (2011): 4447-4459. Web. 7 May 2013.</LI></UL></DIV></DIV><DIV class="printfooter">
Retrieved from &quot;<A href="http://2013.igem.org/Team:uOttawa/Project">http://2013.igem.org/Team:uOttawa/Project</A>&quot;</DIV></DIV></DIV><DIV id="footer-box" class="noprint"><DIV id="footer"><UL id="f-list"><LI id="t-recentchanges"><A href="/Special:RecentChanges" title="Recent changes">Recent changes</A></LI><LI id="t-whatlinkshere"><A href="/Special:WhatLinksHere/Team:uOttawa/Project" title="List of all wiki pages that link here [j]" accesskey="j">What links here</A></LI><LI id="t-recentchangeslinked"><A href="/Special:RecentChangesLinked/Team:uOttawa/Project" title="Recent changes in pages linked from this page [k]" accesskey="k">Related changes</A></LI><LI id="t-specialpages"><A href="/Special:SpecialPages" title="List of all special pages [q]" accesskey="q">Special pages</A></LI><LI><A href="/Special:Preferences">My preferences</A></LI></UL></DIV><DIV id="footer"><UL id="f-list"><LI id="t-print"><A href="/wiki/index.php?title=Team:uOttawa/Project&amp;printable=yes" title="Printable version of this page [p]" accesskey="p">Printable version</A></LI><LI id="t-permalink"><A href="/wiki/index.php?title=Team:uOttawa/Project&amp;oldid=8479" title="Permanent link to this revision of the page">Permanent link</A></LI><LI id="privacy"><A href="/2013.igem.org:Privacy_policy" title="2013.igem.org:Privacy policy">Privacy policy</A></LI><LI id="disclaimer"><A href="/2013.igem.org:General_disclaimer" title="2013.igem.org:General disclaimer">Disclaimers</A></LI></UL></DIV></DIV></DIV></BODY></HTML>