"
Page
Discussion
View source
History
teams
Log in
 
Team:uOttawa/modeling
From 2013.igem.org
Home Wet Lab Modeling Software Human Practices Team Parts Notebook Safety Attributions
Overview Equations Parameters References
Modeling the Type-I Incoherent Feedforward Loop
The Feedforward loop can be broken down into 3 main steps
1.   Toxin will stimulate production of X (rtTA).
2.   X (rtTA) stimulates production of both Z (sfGFP) and Y (LacI).
3.   Sufficient repression from Y terminates Z (sfGFP) production.
You can take a better look at the system by clicking on the button below!
Click to add toxin!
Equations
Graph
X
1. Variation of X over variation of time
4. Variation of Y over variation of time
5. Variation of Z over variation of time
3. Amount of Active Y relative to amount of IPTG
2. Amount of Active X relative to amount of ATC
Y
Z
Click to show activation by X!
Click to show repression by Y!
Restart!
Equations involved in modeling the biological system
The mathematical model was developed using basic rate formulas of activation and repression as shown in the system of equations above. We then expanded them to include the different parameters required for our model.
These additions are as follows
IPTG concentration to calculate the amount of Y (LacI) repressing Z (sfGFP).
ATC concentration to calculate the amount of X (rtTA) available to bind Y promoter and Z promoter.
Transfer rates of proteins between the cytosol and the nuclei to calculate transfer rates by size of each protein.
Acknowledgements
The differential equations model for the system and implementation in MATLAB were done by Paul Reginato from the University of Waterloo iGEM team. This was only possible as a result of the great collaboration between the University of Waterloo iGEM team and the uOttawa iGEM teams learn more . Parameterization and analysis of the model to show pulse and fold change detection were done by Nicolas Tremblay from the University of Ottawa iGEM team with extensive guidance of Daniel Charlebois advisor for the uOttawa iGEM team.
Equations
The four rate equations for the production of mRNA over time of GEV (1.1), X (rtTA) (2.1), Y (LacI) (3.1) and Z (sfGFP) (4.1) in the nucleus are shown below.
Click for legend
1.1 Variation of GEV mRNA according to time.
MRP7 is the constitutive transcription rate that does not change. Replicates the role of a toxin in the system.
TrsGEV mRNA is the ratio of the amount of GEV mRNA that will be transfered to the cytosol to be translated into protein. This value is set to 1.
2.1 Variation of rtTA mRNA according to time.
pGEV is the fully induced transcription rate of this mRNA lenght.
KGEV is the affinity of the GEV to the rtTA promoter.
GEVBound is the Amount of activated GEV by ß-estradiol from equation 3.0.
n is the cooperativity of the molecule to the promoter or the Hill coefficient.
TrsrtTA mRNA is the ratio of the amount of rtTA mRNA that will be transfered to the cytosol to be translated into protein. This is set to 1.
rtTAmRNA is the amount of rtTA mRNA transcribed by the promoter.
leakage is the basic amount of rtTA mRNA that is produced per second without activation.
3.1 Variation of LacI mRNA according to time.
Values in equation 3 have the same role as in equation 2.0. Only the molecule's names have been substituded.
4.1 Variation of sfGFP mRNA according to time.
psfGFP is the fully induced transcription rate of this mRNA lenght.
KrtTA is the affinity of the rtTA bound to the psfGFP promoter
rtTABound is the Amount of activated rtTA by ATc
n is the cooperativity of the molecule to the promoter
LacIfree is the amount of LacI available to repress the activation sfGFP
KLacI is the affinity of the LacI to the sfGFP promoter
TrssfGFP mRNA transfer rate of the mRNA from the nucleus to the cytosol. This is set to 1.
sfGFPmRNA amount of mRNA transcribed in the nucleus.
leakage is the basic amount of GEV mRNA that is produced per second without activation.
The first sets of equations were used to describe the amount of mRNA transcribed. These equations also took into consideration that the mRNA would be transported out of the nuclei. In the model, we assumed that all of the mRNA molecules would be exported to the cytosol. The differential equations representing the variation in mRNA in the cytosol are represented below in equations 1.2, 2.2, 3.2 and 4.2.
Click for legend
2.1 Variation of GEV mRNA according to time.
TrsGEV mRNA is the ratio of the amount of GEV mRNA that was transfered to the cytosol. This value is set to 1.
GEVmRNA is the amount of mRNA transcribed in the nucleus.
DegGEV mRNA the average degradation rate of an mRNA molecule in a cell.
CytGEV mRNA amount of GEV mRNA in the cytosol.
2.2 Variation of rtTA mRNA according to time.
Values have same roles, but different names
3.2 Variation of LacI mRNA according to time.
Values have same roles, but different names
4.2 Variation of sfGFP mRNA according to time.
Values have same roles, but different names
Computing the variation and amounts of mRNA in the cytosol over time then allowed us to compute the variation of protein translated. The equations of translation shown below in equations 1.3, 2.3, 3.3, 4.3 represent the variation of protein in the cytosol over time. The translational rates were calculated based on the average translation rate of a ribosome (9.5 base pair per second 12). Here, we assume that the quantity of ribosome is not a limiting factor and that there are no translational regulations involved in the model. The degradation rates are taken from the average degradation rate of protein in yeast14.
Click for legend
1.3 Variation of GEV in the cytosol over time.
2.3 Variation of rtTA in the cytosol over time.
3.3 .Variation of LacI in the cytosol over time.
4.3 Variation of Z (sfGFP) in the cytosol over time.
Legend for variable that apply for all equations 4 equations
FullTrans the full translation rate of the mRNA into protein. The rate is calculated based on the size of protein and vary for each protein
Transf transfer rate of the protein from the cytosol back into the nuclues. Extrapolated from known data and varies based on protein size. Equation 4.4 does not have this component because sfGFP only needs to be observed and does not have an effect on transcription of other genes.
degprotein the average rate of degradation of protein. 3.5 molecules/hour in the cell.
The amounts of protein are represented by their nameCyt
Once the amount of protein calculated, we assumed that they would diffuse at an average rate through the nuclear pore to then affect the transcription of the other genes in the system. The calculations leading to the transfer rates are described in the parameter section under transf. Equations 1.4, 2.4 and 3.4 represent the variation of proteins present in the nucleus over time.
Click for legend
1.4 Variation of GEV in the nucleus over time.
2.4 Variation of rtTA in the nucleus over time.
3.4 .Variation of sfGFP in the nucleus over time.
Legend for variables that apply for all 4 equations
Transf transfer rate of the protein from the cytosol back into the nuclues. Extrapolated from known data and varies based on protein size.
The amounts of protein are represented by their nameCyt
degprotein the average rate of degradation of protein. 3.5 molecules/hour in the cell14.
The amounts of protein in the nuclues are represented by their nameNuc
To have better control on the feed-forward loop system, LacI was regulated with IPTG. Changing the amount of ATC regulated the amount of active rtTA. And the Amount of GEV available to activate production of rtTA is controlled with ß-estradiol. The following equations used to compute the amount of each compound are shown below. These numbers are imputed in the mRNA equations and vary accordingly with the amount of each transcription factors expressed in the ODE system.
Click for legend
LacIfree = the amount of LacI that is not bound to the IPTG. The LacI that is not bound to the IPTG is able to repress the expression of Z (sfGFP). Therefore the more IPTG present the more expression of Z is goign to be strong.
rtTAbound = the amount of rtTA that is available to activatite the Z (sfGFP) promoter and the Y (LacI) promoters. rtTA activity is regulated by ATC. The more ATC is present the stronger the expression of rtTA will be.
GEVbound = the amount of GEV that is available to activatite the X (rtTA) promoter. GEV activity is regulated by ß-estradiol. The more ß-estradiol present the stronger the expression of X (rtTA) will be.
K are the affinity constants of the different compounds.
Parameters
Some parameters were found in the literature. Others were derived from other known constants as explained below.
Transcription rate
The transcription rate was calculated using the known rate of 2kb per minute6 which translates to 33.3 nucleotides per second. This rate was then divided by the amount of nucleotides in one mRNA strand. The values are in the parameter table.
Translation rate
The translation rate was calculated using the known rate of 9.5 amino acid per second12. This rate was then divided by the amount of amino acids per protein. The values are in the parameter table.
Transfer rate of protein from the cytosol to the nucleus
The rate of transfer from the cytosol to the nucleus was extrapolated from the formula rate = 0.01*(5000-size of protein in Dalton)-50. This is the multiplied by 119 or the average number of nuclear pores in yeast7. The first equation was determined by assuming that transfer rate of protein 5000 Da takes 0 seconds (its size being insignificant ) and a 17000 Da protein takes 120 seconds 11 .
 
GEV
rtTA
LacI
sfGFP
IPTG
ß-estradiol
ATC
Translation rates (protein/min)
1.25
0.901
0.959
2.395
 
 
 
n (Hill coefficient)
1
2
2
-
2
1
1
K
10^-6 13
10^-6 13
10^-10 1
-
6.0*10^-6 3
10^-9 4
1.56*10^-9 13
Transcription rate (mRNA/min)
1.465
1.055
1.122
2.801
 
 
 
Transf
0.00249
0.00168
0.00181
 
 
 
 
Leakage
1.5   molecule/hour for one promoter
References
1. Kalisky, Tomer, Erez Dekel, and Uri Alon. "Cost–benefit Theory and Optimal Design of Gene Regulation Functions." Physical Biology 4.4 (2007): 229-45. Print. K of LacI to DNA.
2. Kalisky, Tomer, Erez Dekel, and Uri Alon. "Cost–benefit Theory and Optimal Design of Gene Regulation Functions." Physical Biology 4.4 (2007): 229-45. Print. K of IPTG to LacI.
3. Goodson, Kathy A. "LacI-DNA-IPTG Loops: Equilibria among Conformations by SingleMolecule FRET." The Journal of Physical Chemistry 117 (2013): 4713-722. Print. K of IPTG to LacI.
4. Orth, Peter, Dirk Schnappinger, Phaik-Eng Sum, George A. Ellestad, Wolfgang Hillen, Wolfram Saenger, and Winfried Hinrichs. "Crystal Structure of the Tet Repressor in Complex with a Novel Tetracycline, 9-(N, N-dimethylglycylamido)-6-demethyl-6-deoxy-tetracycline1." Journal of Molecular Biology 285.2 (1999): 455-61. Print.
5. Herskowitz, Ira. "Life Cycle of the Budding Yeast Saccharomyces Cerevisiae." Microbiological Reviews 53.4 (1988): 536-53. Print. Time division of yeast
6. Zenklusen, Daniel, Daniel R. Larson, and Robert H. Singer. "Single-RNA Counting Reveals Alternative Modes of Gene Expression in Yeast." Nature Structural & Molecular Biology 15.12 (2008): 1263-271. Print. Fully induced transcription
7. Maul, G. "Quantitative Determination of Nuclear Pore Complexes in Cycling Cells with Differing DNA Content." The Journal of Cell Biology 73.3 (1977): 748-60. Print. number of pores
8. "Team:PKU Beijing/Modeling/ODE." Main Page. N.p., 26 Sept. 2009. Web. 27 Sept. 2013. calculation of rate
9. Herskowitz, Ira. "Life Cycle of the Budding Yeast Saccharomyces Cerevisiae." Microbiological Reviews 53.4 (1988): 536-53. Print. Time division of yeast
10. Khmelinskii, Anton. "Tandem Fluorescent Protein Timers for in Vivo Analysis of Protein Dynamics." Nature Biotechnology 30.7 (2012): 708-14. Print. Half life of sfGFP
11. Alberts, Bruce. "The Transport of Molecules Between the Nucleus and the Cytosol." Molecular Biology of the Cell. 4th ed. New York: Garland Science, 2002. 669-78. Print. speed of pore
12. Shah, Premal. "Rate-Limiting Steps in Yeast Protein Translation." Cell 153.7 (2013): 1589-601. Print. Rate of translation
13. Approximated to adapt the model
14. Nevozhay, D., R. M. Adams, K. F. Murphy, K. Josic, and G. Balazsi. "Negative Autoregulation Linearizes the Dose-response and Suppresses the Heterogeneity of Gene Expression." Proceedings of the National Academy of Sciences 106.13 (2009): 5123-128. Print.
Retrieved from " http://2013.igem.org/Team:uOttawa/modeling "
Recent changes
What links here
Related changes
Special pages
My preferences
Printable version
Permanent link
Privacy policy
Disclaimers
