<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_IngenuityLab_Canada_Description skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><DIV id="top_title"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:IngenuityLab Canada/Description</SPAN></H1></DIV><DIV id="HQ_page"><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><P><TITLE>Ingenuity Lab - dNANO</TITLE></P><DIV class="il_header"><DIV class="wrapper"><DIV class="menu"><UL><A href="https://2016.igem.org/Team:IngenuityLab_Canada/dNANO">dNANO</A></UL><UL>Project
						<LI><A href="https://2016.igem.org/Team:IngenuityLab_Canada/Description">Description</A></LI><LI><A href="https://2016.igem.org/Team:IngenuityLab_Canada/Parts">Parts</A></LI><LI><A href="https://2016.igem.org/Team:IngenuityLab_Canada/Results">Results</A></LI><LI><A href="https://2016.igem.org/Team:IngenuityLab_Canada/Protocols">Protocols</A></LI><LI><A href="https://2016.igem.org/Team:IngenuityLab_Canada/Attributions">Attributions</A></LI></UL><UL>Practices
						<LI><A href="https://2016.igem.org/Team:IngenuityLab_Canada/Collaborations">Collaborations</A></LI><LI><A href="https://2016.igem.org/Team:IngenuityLab_Canada/Practices">Human Practices</A></LI></UL><UL>Team
						<LI><A href="https://2016.igem.org/Team:IngenuityLab_Canada/Members">Members</A></LI><LI><A href="https://2016.igem.org/Team:IngenuityLab_Canada/Sponsors">Sponsors</A></LI><LI><A href="https://2016.igem.org/Team:IngenuityLab_Canada/Blog">Blog</A></LI></UL><UL><A href="https://2016.igem.org/Team:IngenuityLab_Canada/Achievements">Achievements</A></UL><UL><A href="https://2016.igem.org/Team:IngenuityLab_Canada/Safety">Safety</A></UL></DIV></DIV></DIV><DIV class="contents"><DIV class="article"> <DIV class="wrapper-title"><DIV id="toc" class="toc"><DIV id="toctitle"><H2>Contents</H2></DIV><UL><LI class="toclevel-1"><A href="#Project_Description"><SPAN class="tocnumber">1</SPAN><SPAN class="toctext">Project Description</SPAN></A></LI><LI class="toclevel-1"><A href="#Project_Overview"><SPAN class="tocnumber">2</SPAN><SPAN class="toctext">Project Overview</SPAN></A></LI><LI class="toclevel-2"><A href="#Motivation:"><SPAN class="tocnumber">2.1</SPAN><SPAN class="toctext">Motivation:</SPAN></A></LI><LI class="toclevel-2"><A href="#Our_Aim:"><SPAN class="tocnumber">2.2</SPAN><SPAN class="toctext">Our Aim:</SPAN></A></LI><LI class="toclevel-2"><A href="#dNANO"><SPAN class="tocnumber">2.3</SPAN><SPAN class="toctext">dNANO</SPAN></A></LI><LI class="toclevel-3"><A href="#Self_Folding_Reaction:"><SPAN class="tocnumber">2.3.1</SPAN><SPAN class="toctext">Self Folding Reaction:</SPAN></A></LI></UL><LI class="toclevel-2"><A href="#Design:"><SPAN class="tocnumber">2.4</SPAN><SPAN class="toctext">Design:</SPAN></A></LI><LI class="toclevel-3"><A href="#Cadnano2"><SPAN class="tocnumber">2.4.1</SPAN><SPAN class="toctext">Cadnano2</SPAN></A></LI><LI class="toclevel-3"><A href="#DAEDALUS:_DNA_Origami_Sequence_Design_Algorithm_for_User-defined_Structures"><SPAN class="tocnumber">2.4.2</SPAN><SPAN class="toctext">DAEDALUS: DNA Origami Sequence Design Algorithm for User-defined Structures</SPAN></A></LI><LI class="toclevel-2"><A href="#Part_development:"><SPAN class="tocnumber">2.5</SPAN><SPAN class="toctext">Part development:</SPAN></A></LI><LI class="toclevel-1"><A href="#DNA_Origami_Overview"><SPAN class="tocnumber">3</SPAN><SPAN class="toctext">DNA Origami Overview</SPAN></A></LI><LI class="toclevel-2"><A href="#3-D_Design_.E2.80.93_caDNAno2"><SPAN class="tocnumber">3.1</SPAN><SPAN class="toctext">3-D Design – caDNAno2</SPAN></A></LI><LI class="toclevel-2"><A href="#3-D_Design_.E2.80.93_DAEDALUS"><SPAN class="tocnumber">3.2</SPAN><SPAN class="toctext">3-D Design – DAEDALUS</SPAN></A></LI><LI class="toclevel-1"><A href="#Photosystem_II_Overview"><SPAN class="tocnumber">4</SPAN><SPAN class="toctext">Photosystem II Overview</SPAN></A></LI><LI class="toclevel-2"><A href="#Photosystem_II_Purification:"><SPAN class="tocnumber">4.1</SPAN><SPAN class="toctext">Photosystem II Purification:</SPAN></A></LI><LI class="toclevel-2"><A href="#Oxygen_Evolution_and_Free_Electron_Generation_Test:"><SPAN class="tocnumber">4.2</SPAN><SPAN class="toctext">Oxygen Evolution and Free Electron Generation Test:</SPAN></A></LI><LI class="toclevel-2"><A href="#Proteoliposome:"><SPAN class="tocnumber">4.3</SPAN><SPAN class="toctext">Proteoliposome:</SPAN></A></LI><LI class="toclevel-2"><A href="#Attachment_to_Nanowire:"><SPAN class="tocnumber">4.4</SPAN><SPAN class="toctext">Attachment to Nanowire:</SPAN></A></LI></DIV><H1><SPAN class="mw-headline" id="Project_Description">Project Description</SPAN></H1></DIV><DIV class="buttons-tabs"><DIV id="pd_button_tab1"><EM>Project Overview</EM></DIV><DIV id="pd_button_tab2"><EM>DNA Origami Overview</EM></DIV><DIV id="pd_button_tab3"><EM>Photosystem II Overview</EM></DIV></DIV><DIV class="wrapper"><DIV id="pd_tab1"><H1><SPAN class="mw-headline" id="Project_Overview">Project Overview</SPAN></H1><H2><SPAN class="mw-headline" id="Motivation:">Motivation:</SPAN></H2><P>
	Although it is one of the most studied field of science, Nanotechnology is lagging in application to solve problems in all fields of life. Nanoparticles are huge risk to the health of common people because their lack of biocompatibility. It is also very expensive to manufacture nanotechnology products because of their small size and complex materials used. In 2017 alone, the budget for nanotechnology research is $1.7 billion dollars, but the outcome of this research seemed to be not high applicable in various fields, especially for scientist with limited amount funding.
					</P><H2><SPAN class="mw-headline" id="Our_Aim:">Our Aim:</SPAN></H2><P>Create a scaffold from material that can has following characteristics:</P><UL><LI>Biocompatible</LI><LI>Minimal Impact on health</LI><LI>Self-Assembling structure</LI><LI>Inexpensive to produce</LI><LI>Well characterized</LI></UL><H2><SPAN class="mw-headline" id="dNANO">dNANO</SPAN></H2><P>
	With dNANO, we have attempted to design and test DNA based Nano-scaffold that has application in wide range of field. Our choice of material, DNA fits the criteria that our aim established. The DNA is relatively non-toxic to the Human body and therefore it is Biocompatible. The synthetic DNA is inexpensive to produce and therefore to order to create massive amount, simple replication can create thousands of molecules. The self-assembling properties of the DNA is characterized and well understood since it is a polymer of four basic molecules.  
						</P><P>	We approached designing our DNA scaffold using advanced computer software such as CADnano2. The designing was done using M13mp18 DNA to keep the initial cost of the material low so that we abide by the rules set out by our aim with this project.  The scaffold is designed to be approximately 40 nm long, with an hollow cavity that can be design to place various inorganic material. Through the combination of organic material (DNA) and inorganic material such as gold, the product can have application in multiple fields.  
</P><P>	In our prototype, we used gold as the inorganic component to which are precipitated precisely in the hollow cavity of the scaffold and tested for conductance of electricity at nanoscale. To align multiple of our scaffold in a straight line, we used Teslaphoresis as the method, which again fits the criteria we set out to achieve in our aim for this project.  
</P><H3><SPAN class="mw-headline" id="Self_Folding_Reaction:">Self Folding Reaction:</SPAN></H3><H2><SPAN class="mw-headline" id="Design:">Design:</SPAN></H2><P>
					For this year’s project, we used two available softwares for designing our three-dimensional DNA origami nanostructures. The first is <STRONG>cadnano2</STRONG> supported under the open-source license by the Wyss Institute for Biologically Inspired Engineering at Harvard University. The second software we used is a recent one called, <STRONG>DAEDALUS</STRONG>. It is a free online resource developed and maintained by the Laboratory for Computational Biology and Biophysics at MIT, under the directorship of Professor Mark Bathe.
					</P><H3><SPAN class="mw-headline" id="Cadnano2">Cadnano2</SPAN></H3><P> </P><DIV class="article-cap"><STRONG>Figure 1:</STRONG> Screenshots of the design process using cadnano2.</DIV><P> </P><P> </P><DIV class="article-cap"><STRONG>Figure 2:</STRONG> Screenshot of the design process using cadnano2. This figure shows the “autostaples” function provided in the cadnano2 program. We had to make manual edits to the staples where appropriate.</DIV><P> </P><P> </P><DIV class="article-cap"><STRONG>Figure 3:</STRONG> This is the three-dimensional model of our finished DNA origami nanostructure that was generated using Maya 2015 (Autodesk. Inc.). Autodesk and cadnano2 worked together to provide a plugin between the two programs.</DIV><H3><SPAN class="mw-headline" id="DAEDALUS:_DNA_Origami_Sequence_Design_Algorithm_for_User-defined_Structures">DAEDALUS: <U>D</U>NA Orig<U>a</U>mi S<U>e</U>quence <U>D</U>esign <U>Al</U>gorithm for <U>U</U>ser-defined <U>S</U>tructures</SPAN></H3><P> </P><DIV class="article-cap"><STRONG>Figure 4:</STRONG> DAEDALUS workflow. (A) Unlike cadnano2, the user starts with a target geometry. In our case, an octahedron. (B) Next, the program provides a fully automatic sequence design followed by a three-dimensional view of your DNA-based nanoparticle. (C) This last image shows the scaffold strand in grey and the individual staples strand in blue.</DIV><H2><SPAN class="mw-headline" id="Part_development:">Part development:</SPAN></H2><P>We plan to submit the following 3 BioBrick parts for this year’s competition.</P><DIV class="article-cap"><STRONG>Figure 5:</STRONG> Schematic of the 3 parts that we will be submitting for the BioBrick requirement. Specifically, the first 2 parts for submission includes the CP47 subunit of Photosystem II fused to a 6x His-tag. BBa_K2127001, is unique due to the addition of TFBS (Transcription Factor Binding Site) that has been shown in literature to act as a super-promoter in native <I>Synechocystis sp.</I> PCC 6803. The last part for submission include the psbT subunit of Photosystem II as a standalone part.  All parts contain a lac promoter, lac operator, RBS, two-stop codons, and a terminator sequence. All parts are synthesized as shown by IDT DNA Inc.</DIV></DIV><DIV id="pd_tab2"><H1><SPAN class="mw-headline" id="DNA_Origami_Overview">DNA Origami Overview</SPAN></H1><P>
A string is one of the most versatile product mankind has ever manufactured. The DNA is similar to string in many ways such as it is flexible, it can be folded in a complex way to create structures. The process of folding of a single strand of DNA into a complex shape is known as DNA Origami and we wanted to utilize this feature to create a DNA scaffold. Our team at Ingenuity Lab got the inspiration from the amazing work of Dr. Seidel’s group at the Leipzig University to create an electrical nano-circuit.  
					</P><P>Our project uses the principles behind DNA Origami to design an organic/inorganic bionanoarchitecture that serves to function as DNA-based gold nanowire. There are vast potential applications using DNA nanotechnology. The project inspiration came from Dr. Seidel’s group at the Leipzig University. Following their work, we have designed a boxed structure with hollow using 214 short oligonucleotides (called staples) and a 7249nt scaffold DNA from M13mp18. Subsequently, this hollow cavity will then be seeded with a gold nanoparticle (AuNP). Seeding occurs because the inner cavity will contain polyA capture strands whereas the AuNP will have its complementary polyT strands. Once the AuNP is seeded, the DNA origami structure will be placed in gold ion solution, H[AuCl4], to allow the precipitation of gold within the hollow cavity. In other words, the DNA uses the gold nanoparticles as a reaction seed to mould and grow into the desired shape, pattern and complex components needed for programmable circuits.
</P><H2><SPAN class="mw-headline" id="3-D_Design_.E2.80.93_caDNAno2">3-D Design – caDNAno2</SPAN></H2><P>To design the DNA scaffold structure, we used caDNAno software to design the 3-D structure. This program is simple to use and it gave us the computational tools to design such complex structure with relative ease. Our design is different from the DNA scaffold that Dr. Seidel’s Group develop because our aim was to simply the process as much as possible. We design the scaffold that using M13mp18 DNA because of its low cost and our structure’s hollow cavity is 4 helices by 4 helices. A smaller hollow cavity allowed our structure to retain all the functions that allows DNA gold precipitation but have far lower numbers of staple DNA strand required. The helices at the center which serve to attach a Gold Nano particle functionalized with Poly (T) DNA strand, has two sites of binding, while Dr. Seidel’s group developed their design using four strands of attachment.  
</P><P> </P><DIV class="article-cap"><STRONG>Figure 1:</STRONG> Front view of the helices that form the DNA scaffold that is designed using caDNAno Software. The numbers indicate the helices number while helices 49 and 51 represent the strands which has Poly (A) protruding into the cavity which allows the DNA functionalized Gold-Nanoparticle can bind.</DIV><P> </P><P> </P><DIV class="article-cap"><STRONG>Figure 2:</STRONG> Screenshot of the design process using caDNAno2. This figure shows the “autostaples” function provided in the cadnano2 program. We had to make manual edits to the staples where appropriate.</DIV><P> </P><P> </P><DIV class="article-cap"><STRONG>Figure 3:</STRONG> Schematic showing gold nanoparticle functionalized with Poly(T) oligo strands being captured by Poly(A) oligo capture strands located on helices 49 and 51. The schematic also shows gold precipitation within the cavity of our structure; demonstrating the “moulding” ability of this design.</DIV><P> </P><P> </P><DIV class="article-cap"><STRONG>Figure 4:</STRONG> This is the three-dimensional model of our finished DNA origami nanostructure that was generated using Maya 2015 (Autodesk. Inc.). Autodesk and cadnano2 worked together to provide a plugin between the two programs.</DIV><P> </P><P>
Our design of the DNA based Gold Nano particle is likely biocompatible which allows it to be applicable in health care field. One field we tried to explore as a potential application of our circuit design is using Photosystem II to pass electrical current through the Gold nanowire. Since each scaffold is ~40 nm long, we aligned DNA based Gold nanowire using the Teslaphoresis technology developed at Rice University.  
					</P><H2><SPAN class="mw-headline" id="3-D_Design_.E2.80.93_DAEDALUS">3-D Design – DAEDALUS</SPAN></H2><P>
					DAEDALUS: <U>D</U>NA Orig<U>a</U>mi S<U>e</U>quence <U>D</U>esign <U>Al</U>gorithm for <U>U</U>ser-defined <U>S</U>tructures
					</P><P>					This free online resource is developed and maintained by the Laboratory for Computational Biology &amp; Biophysics at MIT, which is directed by Professor Mark Bathe.
</P><P>					Unlike Cadnano2, DAEDALUS uses a top-down approach to designing DNA 3D nanostructures. First the user generates a three-dimensional polygon structure. DAEDALUS, which utilizes MatLab to run its programming code, takes into account the number of edges and vertices. Calculates the length of the edges. Next, the user defines and uploads the scaffold DNA sequence and the number of base pairs that marks the length of each edge. Daedalus program will automatically generate the staples DNA sequences for folding the scaffold DNA. Figure 5 shows the finished product. Grey strands are the scaffold DNA (M13mp18) and blue strands show the DNA staples that binds to the scaffold.
</P><P>					Our DAEDALUS design starts with an octahedron. Our scaffold sequence is ssDNA circular M13mp18 and is 7249 base pairs long. We linearlized it using a 30nt oligonucleotide sequence that complements an unused region of the M13mp18 sequence. This creates a restriction endonuclease cut site, BsrBI, for that 30nt stretch. We then digest using BsrBI (NEB) to linearlize the circular ssDNA to a linear ssDNA. To fold this linear ssDNA into the three-dimensional octahedron nanoparticle, 36 unique DNA staples were added into the folding reaction mix.
</P><P> </P><DIV class="article-cap"><STRONG>Figure 5:</STRONG> DAEDALUS workflow. (A) Unlike cadnano2, the user starts with a target geometry. In our case, an octahedron. (B) Next, the program provides a fully automatic sequence design followed by a three-dimensional view of your DNA-based nanoparticle. (C) This last image shows the scaffold strand in grey and the individual staples strand in blue.</DIV><P> </P><P>
					Afterwards, DAEDALUS provides the option to generate molecular models of the DNA nanoparticle through Chimera ver 1.11.2. Chimera program is designed by the University of California – San Francisco. Daedalus will provide a .PDB file required by Chimera. UCSF Chimera is commonly used by bioinformaticians to generate molecular models of protein structures.
					</P><P> </P><DIV class="article-cap"><STRONG>Figure 6:</STRONG> UCSF Chimera ver 1.11.2 generated molecular model of our DNA Origami octahedron nanostructure.</DIV><P> </P><P><STRONG>References:</STRONG>
					Professor Ralf Seidel’s research group
					Helmi, Seham., Ziegler, Christoph., Kauert, Dominik., Seidel, Ralf. (2014) Shape-Controlled Synthesis of Gold Nanostructures Using DNA Origami Molds. ACS: Nano Letters. 14(11), 6693-6698. DOI: 10.1021/nl503441v
					Cadnano2:
					Douglas, Shawn M., Marblestone, Adam H., Teerapittayanon, Surat et al. (2009) Rapid prototyping of 3D DNA-origami shapes with caDNAno. Nucleic Acids Res. 37(15): 5001-5006. DOI: 10.1093/nar/qkp436
					Daedalus
					Veneziano, Remi., Ratanalert, Sakul., Zhang, Kaiming et al. (2016) Designer nanoscale DNA assemblies programmed from the top down. Science 352, aaf4388.
					DOI:  10.1126/science.aaf4388
					UCSF Chimera:
					Molecular graphics and analyses were performed with the UCSF Chimera package. Chimera is developed by the Resource for Biocomputing, Visualization, and Informatics at the University of California, San Francisco (supported by NIGMS P41-GM103311).
					UCSF Chimera--a visualization system for exploratory research and analysis. Pettersen EF, Goddard TD, Huang CC, Couch GS, Greenblatt DM, Meng EC, Ferrin TE. J Comput Chem. 2004 Oct;25(13):1605-12.
					</P></DIV><DIV id="pd_tab3"><H1><SPAN class="mw-headline" id="Photosystem_II_Overview">Photosystem II Overview</SPAN></H1><P>
Over 3 billion years of evolution has produced one of the most efficient system of machinery ever produced. The protein in Cyanobacteria, Photosystem II has efficiency of converting light into electrical energy with as high as 95% efficiency. We choose to test our nano-circuit if it can conduct electrical signal via attaching it to the Photosystem II protein.
					</P><P>The photosystem is a highly complex system with multiple subunit and being a membrane protein it needed high amount of attention during experiments to preserve its activity. Using a mutant strain of Synechocystis sp. PCC 6803 which has a His-tag on the CP47 subunit of the protein, we isolated the Photosystem II. To stabilize the protein during the course of purification, we used n-Dodecyl-β-D-Maltoside as the detergent to preserve the structure and activity of the protein.
</P><H2><SPAN class="mw-headline" id="Photosystem_II_Purification:">Photosystem II Purification:</SPAN></H2><P>Our team used the protocol developed by Dr. Bricker’s team as inspiration to optimize the protocol to successfully isolate the Photosystem II. Using Dr. Bricker’s team’s protocol, we modified and optimized each step of the protocol through rigorous Oxygen Evolution analysis. Oxygen Evolution tests allow us to quantify the activity of the Photosystem II at each step and during successive purifications we have noticed that our isolated protein efficiency has dramatically increased.
</P><P><I>Synechocystis sp</I>. PCC 6803 mutant, HT-3, was grown in 20L of BG-11 media supplemented with 2mM glucose and 50µM Kanamycin under 50µE m<SUP>-2</SUP>s<SUP>-1</SUP> at 30°C with air bubbling for 3 to 4 days. Upon reaching an optical density (OD) of 0.8-1.0 at A730, culture was harvested using a floor-model centrifuge model for 10 minutes at 8,000xg. The cell pellet was washed using Buffer A (50mM MES-NaOH, pH 6.0, 10mM MgCl2, 5mM CaCl2, 25% Glycerol) and combined cells were centrifuged at 12,000xg for 10 minutes. Using Buffer A with inhibitors to avoid proteolytic activity (1.0mM PMSF, 1 mM ε-amino caproic acid) and 50µg/ml DNAase, cells were resuspended and diluted to 1 mg mL<SUP>-1</SUP> of chlorophyll-a (chlA).
</P><P>The resuspended cells were broken by 2 passes through cell disrupter (Constant Systems, Ltd.) at 35k psi. Ultracentrifugation (175,000rcf for 30 minutes) was used to pellet the thylakoid membranes. The resulting pellet is brought to 1 mg mL<SUP>-1</SUP> chlA. Dodecyl ß-D-maltoside (DDM) was added to 0.8% (w/v) by dropwise addition of 20%(w/v) stock to solubilize the membranes. After 20 min incubation with gentle agitation non-solubilized material was removed by centrifugation; 100rcf to 22100rcf over 15 minutes.
</P><P>ÄKTA Pure 25 Fast Protein Liquid Chromatography (FPLC) was used for all chromatography steps. Fractions in 96 wells plate were pooled at the center 90% target peak at A673 containing PSII and concentrated using a 100 kD MWCO Amicon® Ultra centrifugal filter (EMD Millipore, Inc.). Samples were concentrated to 2-3 mg mL<SUP>-1</SUP> using Amicon® Ultra centrifugal filters, aliquoted, and stored at -80 °C.
</P><H2><SPAN class="mw-headline" id="Oxygen_Evolution_and_Free_Electron_Generation_Test:">Oxygen Evolution and Free Electron Generation Test:</SPAN></H2><P>
Throughout the process of purifying PSII, oxygen evolution assays were conducted in order to ensure oxygen was being evolved by the cyanobacteria from the start to the end of the purification. More importantly, this is also an indirect measurement of free electron generation. Assays were conducted using a Hansatech oxygen electrode and a 2800 µmol Hansatech LED1 red light. The sample chamber contained the buffer (pH 6.5), 300uM DCBQ, 1mM FeCy and the sample at a chlorophyll concentration of 10-20ug, similar to literature sample components1. DCBQ and FeCY serves as an electron acceptors and how the oxygen is measured is through the detection of oxygen levels through the electrode seated in the resin2. Oxygen levels are then plotted both a minute prior and after illumination of the light source, then quantified as the rates of the oxygen activity.
					</P><P>Looking at one node of this toolkit, to aid in the development of greener energy solutions, we strive to incorporate photosystem II from cyanobacteria as the battery of the toolkit.
</P><P>However, how will we incorporate the protein similar to a battery that runs electrons through cathode to anode? 
</P><P>To tackle this, we have conducted preliminary compatibility tests of connecting the protein via the Quinone channel. This shows promise as it can be incorporated in a circuit using menaquinone or DCBQ as these two substrates have shown to allow the protein to evolve the highest oxygen rates when compared to other quinone solutions.  However, in order to have an energy source for the circuit, the protein needs to be stabilized which will be discussed in the proteoliposome section.
</P><P> </P><DIV class="article-cap"><STRONG>Figure 1:</STRONG>  Oxygen Evolution Apparatus - Sample with chlorophyll concentration to 5-15µg (diluted using BG11 media or Assay Buffer) loaded into Borosilicate glass reaction vessel at center of apparatus with final concentrations of 300μM for 2,6 -DCBQ and 1mM Potassium Ferricyanide. The oxygen level is monitored by the electrode at the bottom of the resin and recorded for a minute prior to the illumination of the red LED light and a minute after, the rate of oxygen evolution then is interpolated. See Oxygen Evolution Protocols for details.</DIV><P> </P><DIV class="article-cap"><STRONG>Figure 2:</STRONG> Oxygen level and rate of oxygen evolution graph generated through oxygen assay apparatus. At time 0, illumination of a red LED light on the sample containing PSII causes the oxygen level to rise through which the electrode detects electrochemically, and in turn shows that the protein is active in generating electrons.</DIV><P> </P><DIV class="article-cap"><STRONG>Figure 3:</STRONG> Using 150μM to 500μM concentrations of various substrates including Menaquinonne, Decylubiquinone, DCBQ, etc., the effect of different substrates on the rate the oxygen evolution of PSII was investigated. This preliminary test reveals that DCBQ or Menaquinone allow for the highest rate of oxygen evolution which could be a potential molecule used to aid the conduction of electrons from PSII's quinone channel.</DIV><P><STRONG>References:</STRONG><SUP>1</SUP>Hillier, W. et al. (1993) Increases in peroxide formation by the Photosystem II oxygen evolving reactions upon removal of the extrinsic 16, 22 and 33 kDa proteins are reversed by CaCI 2 addition. Photosynthesis Research 38: 417-423.<SUP>2</SUP>Bricker, T. et al. (1998) Isolation of a highly active Photosystem II preparation from Synechocystis 6803 using a histidine-tagged mutant of CP 47. Biochimica et Biophysica Acta 1409.</P><H2><SPAN class="mw-headline" id="Proteoliposome:">Proteoliposome:</SPAN></H2><P>
					Proteoliposomes are small aqueous compartment surrounded by phospholipid bilayer with a protein reconstituted within the lipids. From Photosystem II (PSII) purification, the PSII protein is extracted from cyanobacterial native membrane by solubilizing detergent DDM concentration. As a result, PSII remains in detergent micelles. Because PSII is a membrane protein, it is not stable in a non-membrane environment. Thus, incorporating protein into liposome membrane will ensure to preserve activity of protein for further use, homogeneity of protein insertion in the lipid layer, control the orientation as it will ensure the transfer of electron onto the gold nano-wire, morphology and size of the reconstituted proteoliposomes. The homogeneity of size and distribution of liposome is controlled by extruding it through a polycarbonate filter with 0.4μm pore size.
					</P><P>					Escherichia coli Total Lipid Extract was dissolved in organic solvent, chloroform and brought down to a concentration of 20mg/ml. Thin film was created in the glass vial by evaporating the chloroform using dry nitrogen gas in a gas fume hood. Upon evaporation, the lipid film was rehydrate by suspending the film in buffer (5mM MES, 50mM KCl, 200μM Pyranine, 2mM MgCl2 pH 6.5) to a final concentration of 15.34mg/ml. The lipid film was vortexed and sonicated to emulsify in buffer, subjected to two freeze-thaw cycle, and then extruded through a 0.4μm nucleopore polycarbonate membrane 21 times to form liposomes (Avanti Polar Lipid, Inc.). The liposome was stored in -80°C until needed for use.
</P><P> </P><DIV class="article-cap"><STRONG>Figure 4:</STRONG> The reconstitution of PSII in the E.coli Total Lipid Extract membrane. Start with pre-extruded liposomes; Addition of detergent; Lipid and detergent incubation; Addition of PSII and incubation; Four addition of Biobeads™ at one hour intervals for detergent removal; Reconstituted PSII in the lipid membrane and Biobeads™ removed by filtration and centrifugation.</DIV><P> </P><P>
					Proteoliposomes were prepared by incubating buffer (5mM MES, 50mM KCl, 2mM MgCl2, 2mM CaCl2 pH 6.52), detergent at saturated (Rsat) and solubilized (Rsol) concentration and liposome solution overnight(~10hours). After addition of PSII to lipid-detergent solution, 30 minutes at 4°C with gentle rotation, BiobeadsTM were added three time at 1hour interval for detergent removal and lipid bilayer closure to allow the reconstitution of the protein (Figure 4). Biobeads were removed by SPN Column Protein Assay via centrifuge in bench-top micro-centrifuge at 1000g for 1 minutes. Proteoliposomes were desalt on FPLC using a 5mL Hi-trap column. Fractions of 1 ml in the center 90% of the target peak were collected. The reconstitution of PSII was tested with pH probe pyranine, electron acceptor, and ionophore (Figure 5).
					</P><P> </P><DIV class="article-cap"><STRONG>Figure 5:</STRONG> As proteoliposomes are exposed to light, a water molecule is split and H+ build up inside the liposome creating a gradient that allows for the detection of pH.  The orientation of the PSII is determined by alkalization or acidification of proteoliposomes as an indication of the location of water-splitting activity. The pH detecting Pyranine was incorporated inside the proteoliposomes, and the pH activity was testing my suspending the proteoliposomes in buffer (5mM MES, 50mM KCl, 2mM MgCl2, 2mM CaCl2 pH 6.52) that contained ionophore Valinomycin, electron acceptor DCBQ or DQ and Ferricyanide. The DPX quenches any pyranine outside the proteoliposomes in the buffer.</DIV><P> </P><P>
					FlexStation 3 Multi-Mode Microplate Reader was used to run the assays on Photosystem II (PSII) proteoliposomes. After adding 175ul of assay and 25ul of proteoliposomes sample, an initial reading is taken at λem = 410 nm and λexc= 460 nm. The reading ratio of 460/404 is used to determine the pH value for each sample. In this experiment, a ‘control’ sample that does not contain PSII was used to confirm that the proton gradient is only due to the protein splitting water activity in the liposome. In addition, the ‘control’ sample contained 3mM and 7mM CHAPS detergent labelled Rsat (lipid to detergent ratio~ 17.4) and Rsol (lipid to detergent ratio~ 7.5) respectively. Other controls in the experiment included Valinomycin, an ionophore that balances K+ ions down electrochemical gradient; Ferricyanide (FeCY), an electron acceptor impermeable to plasma membranes are in the assay outside the liposome; p-xylene-bis-pyridinium bromide (DPX) quenches the non-permeant pyranine pH indicator dye in the assay. 2,6-dichloro-p-benzoquinone (DCBQ) and decylubiquinone (DQ) were the different electron acceptors tested in the assays; DCBQ degrades after long exposure to light, whereas DQ is stable for much longer period of time. Changing substrate DCBQ and DQ, the expected result is to show pH value decreasing as proton gradient acidifies the internal compartment of proteoliposomes upon exposure to light. 
					</P><P>					Adding Carbonyl cyanide m-chlorophenyl hydrazone (CCCP) in the assay mixture causes break down of electrochemical gradient, and balance pH between assay and liposome. As a result, the pH of the sample increases. In this experiment, upon addition of CCCP after the 1020 seconds mark, the expected result is to see the pH going from an acidic state back to a state of equilibrium.
</P><P><STRONG>References:</STRONG>
					Rigaud, J. L., Levy, D. (2003) Reconstitution of membrane proteins into liposomes. Liposomes, PtB 372:65-86.
</P><H2><SPAN class="mw-headline" id="Attachment_to_Nanowire:">Attachment to Nanowire:</SPAN></H2><P>
					Our inspiration to attach the Photosystem II to the Gold Nano-wire is inspired from the work of the team of Dr. Yehezkeli. They developed a gold plate with immobilized Photosystem II protein via the Quinone Channel. In cyanobacteria, the Quinine channel is used to transport the electron from the Photosystem II to electron transport chain. We designed our assembling of the Protein with the nanowire using the same channel but with modified molecules. In order to find the best molecule that can harvest electron with highest efficiency, we tested Oxygen Evolution of the protein using multiple of Plastiquinone derivative molecules. A stack of this molecules would attach to our protein.
					</P></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></BODY></HTML>