<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_IngenuityLab_Canada_Parts skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><DIV id="top_title"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:IngenuityLab Canada/Parts</SPAN></H1></DIV><DIV id="HQ_page"><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><P><TITLE>Ingenuity Lab - dNANO</TITLE></P><DIV class="il_header"><DIV class="wrapper"><DIV class="menu"><UL><A href="https://2016.igem.org/Team:IngenuityLab_Canada/dNANO">dNANO</A></UL><UL>Project
						<LI><A href="https://2016.igem.org/Team:IngenuityLab_Canada/Description">Description</A></LI><LI><A href="https://2016.igem.org/Team:IngenuityLab_Canada/Parts">Parts</A></LI><LI><A href="https://2016.igem.org/Team:IngenuityLab_Canada/Results">Results</A></LI><LI><A href="https://2016.igem.org/Team:IngenuityLab_Canada/Protocols">Protocols</A></LI><LI><A href="https://2016.igem.org/Team:IngenuityLab_Canada/Attributions">Attributions</A></LI></UL><UL>Practices
						<LI><A href="https://2016.igem.org/Team:IngenuityLab_Canada/Collaborations">Collaborations</A></LI><LI><A href="https://2016.igem.org/Team:IngenuityLab_Canada/Practices">Human Practices</A></LI></UL><UL>Team
						<LI><A href="https://2016.igem.org/Team:IngenuityLab_Canada/Members">Members</A></LI><LI><A href="https://2016.igem.org/Team:IngenuityLab_Canada/Sponsors">Sponsors</A></LI><LI><A href="https://2016.igem.org/Team:IngenuityLab_Canada/Blog">Blog</A></LI></UL><UL><A href="https://2016.igem.org/Team:IngenuityLab_Canada/Achievements">Achievements</A></UL><UL><A href="https://2016.igem.org/Team:IngenuityLab_Canada/Safety">Safety</A></UL></DIV></DIV></DIV><DIV class="contents"><DIV class="article"> <DIV class="wrapper" id="home"><H1><SPAN class="mw-headline" id="Parts">Parts</SPAN></H1><H2><SPAN class="mw-headline" id="Part_Development:">Part Development:</SPAN></H2><P>We plan to submit the following 3 BioBrick parts for this year’s competition.</P><STRONG> BBa_K2127001 (Pcpc560): <A rel="nofollow" class="external free" href="http://parts.igem.org/Part:BBa_K2127001">http://parts.igem.org/Part:BBa_K2127001</A></STRONG><P>
Cyanobacteria is not optimal system to manipulate the protein expression mainly due of the lack of strong promoters. Pcpc560 is a Cyanobacterial super promoter discovered by Dr. Ma’s team that contains two predicted promoters 14 predicted transcription binding site.  Using the Pcpc560, Dr. Ma’s team demonstrated that functional protein were produced at 15% of the total protein production. Our team decided to test the effect protein production in E.Coli cells DH5α using the 14 predicted transcription binding site from Pcpc560. We believe that 14 sequential binding sites from Pcpc560 will allow us to increase the protein function production greatly using the E.Coli DH5α. We have constructed the 14 transcription binding followed by an strong RBS (BBa_B0030), followed by mutant HIS-Tag psbB. The super promoter can be an valuable asset to other iGEM teams for protein production and therefore we have decided to submit the Construct along with 14 Transcription binding site from Pcpc560.  
</P><STRONG> BBa_K2127002 (psbB): <A rel="nofollow" class="external free" href="http://parts.igem.org/Part:BBa_K2127002">http://parts.igem.org/Part:BBa_K2127002</A></STRONG><P>
The psbB gene codes for the CP-47 subunit of the Cyanobacterial Photosystem II in Synechocystis sp. PCC 6803. Using site-directed Mutagenesis Dr Frankel’s team developed a Synechocystis sp. PCC 6803 mutant containing a histidine tag at the C-Terminus of CP47 subunit. Using the His-Tag we isolated the Photosystem II complex and analyzed the activity using the electron evolution analyzer. Photosystem II purification with high activity highlighted that CP-47 with HIS-Tag at C-terminus can made into BioBricks. We are submitting psbB gene as a BioBrick to claim Silver Medal. </P><STRONG> BBa_K2127003 (psbT-like): <A rel="nofollow" class="external free" href="http://parts.igem.org/Part:BBa_K2127003">http://parts.igem.org/Part:BBa_K2127003</A></STRONG><P>
The psbT subunit of the Photosystem II in Synechocystis sp. PCC 6803 is a single membrane-spanning a-Helix of ~3.5 kDa with C-Terminus located on the stromal side of the Thylakoid Membrane.  Structural studies of psbT Studies have showing that removal of the psbT subunit reduces the rate of electron transfer between the QA and QB and less resistance to photo-damage. It is an integral part of the Cyanobacterial photosystem II complex and it plays an important role in the electron transfer. For iGEM 2016, our team decided to submit the psbT gene as a BioBrick due to Photosystem II being an integral part of our project. We are calming Bronze medal for this BioBrick.
</P><H2><SPAN class="mw-headline" id="Part_Improvement_.26_Contribution">Part Improvement &amp; Contribution</SPAN></H2><P><STRONG>psbC subunit (CP43) Please go to this part to view our contribution: <A rel="nofollow" class="external free" href="http://parts.igem.org/Part:BBa_K1640024">http://parts.igem.org/Part:BBa_K1640024</A></STRONG></P><P>As part of our gold medal requirement we further characterizes a pre-existing BioBrick in the registry. Part BBa_K1640024 was modeled using SwissModel to show the tertiary structure. The ribbon model was rendered using multiple templates of other highly characterized CP47 protein crystals found through Protein Data Bank. The sequences were aligned using the BLAST and the SwissModel script. Secondary structures were assigned based on homology and the alignment with the templates. Closer alignment allowed for better estimation of the structure. The secondary structures were then used to stack the image and develop the tertiary structure. Finally, energy minimization was done to render the most stable orientation and organization of the amino sequence in tertiary form. </P><P>The model allows for a better understanding of how the protein interacts within the complex since function is derived from structure. The secondary structures also allow for comparison and identification of key domains and motifs that are necessary for protein function. Likewise these regions are usually conserved and thus the can be used in the identification of unknown functions or the characterization of similar proteins in the organism as well as others.</P><P>UniProt was also used to identify Transmembrane regions, domains, and any motifs</P><P><STRONG>Tertiary Structure</STRONG></P></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></BODY></HTML>