"
Page
Discussion
View source
History
teams
Log in
 
Team:Bordeaux/Introduction
From 2012.igem.org
Biology - iGEM Bordeaux 2012
iGEMBordeaux 2012
team
project
modelling
biology
partners
contacts
Biology
Overview
Introduction
Material & Methods
BioBricks
Results & Conclusion
Note Book
iGEM - Bordeaux - Biology - Introduction
Our project idea comes from a fungus, which was found growing on some plates in the lab. This fungus is growing in concentric circles on plates, and keep growing with this pattern if put on a new plate (figure 1).
Figure 1: the mysterious fungus that inspired our project.
It reminds us of the patterns seen on butterflies’ wings or the ones on zebras and tigers. These kinds of patterns are due to extremely precise genetic regulation. Thus, we decide to try to make bacteria draw concentric circles on plates. This project is very interesting, because it involves displaying different phenotype with the same genetic background using a unicellular organism. Escherichia coli was picked up because we know the genetic background of the strain.
First of all, we had to decide what kind of circles we wanted. There were two options. We could use different colors, or empty circles using a toxin or a kind of apoptosis. We concluded that it should be easier to generate different colors. Three colors were selected: blue given by lacZ, green given by the GFP and red given by the mCherry.
As described in the literature [1], some bacteria can grow with specific patterns, depending on external factors. Some segments of the patterns acquire a specific function. To communicate between them, these bacteria use the quorum sensing. We decided to use the genes from this pathway in our own constructions.
We ended with a genetic map (fig. 2) counting four operons: one dedicated to the communication between cells, the three other dedicated to specific expression of the phenotype. Operons 1 to 3 expressions are regulated by quorum sensing genes [2, 3, 4] and genes involved in tetracycline [5]. LacZ, mCherry and GFP are reporters used to visualize our system working. The operon 4 contains the needed proteins to send and receive signal and is constitutively expressed. During the modeling phase of our project ( see the modeling part ), the cells were attributed states according to the operon expressed. For example, a cell expressing the operon 1 was attributed the state 1. As all our bacteria have the same DNA, we had to make sure that a cell in state 1 will not switch to state 2 by receiving the signal it send itself. Thus negative regulators had to be added [6].
Figure 2: Genetic map of the four operons. These operons create a precise network (fig. 3). Indeed, these regulating genes were selected on their ability to interact with each other in a positive way, or in a negative way [3, 4, 7].
Figure 3: successive actions of the operons. Bold pink arrows represent the signal sent to the neighbor cell. Black arrows represent the internal signals. The figures from A to C represent the successive reactions occurring in the cells when the system is started. C shows the full network.
The expected result is displayed on figure 4.
Figure 4 : Representation of what is expected to happen in the cell and on the dish.
References
[1] Ben-Jacob E., Cohen I., Gutnick D.L. Cooperative organization of bacterial colonies: from genotype to morphotype. (1998) Annual Review of Microbiology, 52: 779-806. [2] Liu C.,  Fu X.,Liu L., Ren X., Chau C.K.L., Li S.,Xiang L., Zeng H., Chen G., Tang L-H., Lenz P., Cui X., Huang W., Hwa T. and Huang J-D. Sequential establishment of stripe patterns in an expanding cell population. (2011) Science. 334: 238-241. [3] Melke P., Sahlin P., Levchenk A. and Jönsson H. A cell-based model for quorum sensing in heterogeneous bacterial colonies (2010) Plos Computational Biology. 6(6) [4] Zucca S., Pasotti L., Mazzini G., De Angelis M.G.C. and Magni P. Characterization of an inducible promoter in different DNA copy number conditions. (2011) BMC Bioinformatics. 13(Suppl 4):S11 [5] Zhu Z., Zheng T., Lee C.G., Homer R.J. and Elias J.A. Tetracycline-controlled transcriptional regulation systems: advances and application in transgenic animal modeling. (2002) Cell & Developmental Biology, 13: 121–128. [6] Garcia-Ojalvo J., Elowitz M.B., and Strogatz S.H. Modeling a synthetic multicellular clock: repressilators coupled by quorum sensing. (2004) Proceedings of the National Acadaemy of Sciences. 101 (30) : 10955–10960. [7] Ward J.P., King J.R., Koerber A.J., Croft J.M., Sockett R.E.and Williams P. Cell-signalling repression in bacterial quorumsensing. (2004) Mathematical Medicine and Biology. 21(3): 169-204.
Partners
AmbaFrance-us
LaBRI
Thermo Scientific
Project
Overview
Introduction
Biobricks
Others
Other
iGEM Bordeaux Team (c) 2012 |
You can contact us by email or phone : igem-bordeaux2-2012@googlegroups.com or +33(0)5 4000 8404
Retrieved from " http://2012.igem.org/Team:Bordeaux/Introduction "
Recent changes
What links here
Related changes
Special pages
My preferences
Printable version
Permanent link
Privacy policy
Disclaimers
