Team:LUBBOCK TTU/Design
Lubbock_TTU
      
Home
Team
Meet the Team
Collaborations
Project
Description
Design
Experiments
Notebook
Interlab
Contribution
Model
Results
Improve
Attributions
Parts
Our Parts
Safety
Human Practices
Silver HP
Integrated and Gold
2016 Wiki
Design—Build—TestHow We Planned for the Future
— Inspiration —
As is customary in any scientific endeavor, we first started our process by climbing onto the shoulders of giants. The giants in our case who served as our inspiration and gave us something to build on were the Johns Hopkins 2010 team and the Slovenia 2016 team. Slovenia created a system using ultrasound and mechanosensitive channels which showed us that a long term project, focusing on controlled gating using electromagnetic frequencies, could be feasible for the competition, though we chose to break ours into pieces or phases for each year. The 2010 Johns Hopkins team broke the necessary ground in the Calmodulin/Calcineurin pathway that we chose to work with. Their models and demonstrations of function paved the way for us to create a more complete model, and it showed that this pathway can be relied upon for the creation of larger systems.
— Pathway —
The Calmodulin/Calcineurin pathway is a eukaryotic pathway dependent on calcium(II) influx. As shown in the diagram below, Calcium(II) may enter though a channel, such as TRPV1 for reasons we will discuss in the future directions section. The pathway results in Crz1p acting as a transcription factor for various promoters. The promoter of interest is PMC1. The pathway begins in the cytosol, where calcium interacts with yeast calmodulin, which binds 3 calcium(II) ions before becoming active. Calmodulin then goes on to complex with calcineurin which activates its phosphatase activity. The newly formed complex dephosphorylates Crz1p, which then becomes active and is able to target the PMC1 promoter sequence. Transcription is then initiated.
It is important to note that PMC1 was chosen due to its relatively low affinity for the Crz1p transcription factor [1]. This means that calcium influx must be enough to exceed homeostasis conditions in order to generate a response. The normal Calcium(II) concentration within yeast cells range from 50nm-200nm [2] which was taken into account during BioBrick and experimental designs.
— BioBricks and Vectors —
The BioBricks themselves demonstrate the design-build-test struggle iGEM teams know so well. The PMC1 promoter sequence is new to the competition and does not lend well to creation through gBlock fragments. Synthesizing the PMC1 promoter on its own even with the use of custom gene synthesis would be troublesome, so we opted to only create the BioBricks that had fluorescent protein coding sequences attached, such as mNeonGreen and superfolder GFP (sfGFP). This gave more stable areas of the DNA downstream to help counteract the inherent difficulties of synthesizing the promoter sequence.
While looking at the diagrams above, you may have noticed the TspMI site added to the end of our synthesized sequence (but obviously not to our submitted BioBrick since it comes after the suffix). This was in preparation for any future characterization we may decide to do in the next phase of the Lubbock_TTU long-term project. This will be performed using a yeast centromere plasmid, pRG213MX, and a second 2µ plasmid, pRG223MX. In order to insert our fragments into these plasmids while maintaining the commercial primer sites, we can use TspMI and EcoRI. By creating our fragment in this way, we were able to cut off the extra portion before ligating the BioBrick into the shipping vector which allowed the single synthesis order to be used for all purposes. For this phase of our overall plan, we chose YEp352 (shown below) based on its compatibility with the BioBrick prefix and suffix. YEp352 is a high copy 2µ plasmid containing URA selection for expression in yeast. It contains other restriction sites that could allow for troubleshooting cloning protocols with different enzymes.
The specific fluorescent proteins were chosen based on maturation time and brightness. We found in literature review that the maturation time for sfGFP is just shy of 6 minutes even in an environment with high calcium(II) content [3]. mNeonGreen was chosen as a second reporter fluorescent protein due to its intensity upon maturation. The goal in picking these two proteins was to add parts to the registry that could be used with this pathway to provide rapid feedback to the synthetic biologist or a monitoring device. These reporter genes would be crucial in the characterization of parts that would be used to trigger this pathway. These two BioBricks were problematic during the build phase of the process. After optimization of the fluorescent gene sequence of the BioBrick using the IDT Codon Optimization Tool, we used the Genscript table to exchange codons contributing to illegal restriction sites for codons that were suboptimal. Since we were designing BioBricks with the PMC1 promoter, we could not fully optimize them as changing this sequence could have unforeseen consequences. This became a larger issue during gene synthesis since they contained sites that could not be modified where the GC content was too low.
— Experiments —
These characteristics were taken into account when we designed our experiments. The biggest implementation of the engineering design process was in this portion of our efforts. Our primers for PCR amplification were designed using the added sequence for TspMI, but initially failed. The protocol we designed and tested used a suggested annealing temperature from the manufacturer, but it was to high. We went back to the drawing board and redesigned our protocol. We used a higher volume reaction so we could run more on our gel, and we created a program with a gradient to look for the best temperature for annealing. These modifications allowed us to pinpoint the proper annealing temperature on the next rounds of building and testing. We returned to the design phase to make the necessary modifications, build, and then test once more before upscaling and moving on to the next step of our project. Our BioBricks are both nearly 2000 base pairs, and the shipping vector is 2070 base pairs. These sizes made the ligation step of cloning difficult for us. The original protocol we built for this used a roughly 1:1 ratio of vector to BioBrick. This proved ineffective after building and testing, so we returned to the design step. We decided on a higher ratio by increasing the concentration of our BioBrick, but halving the concentration of our vector. This new design was built and allowed us to obtain a higher ratio while maintaining the concentrations of our ligase and buffer. The test phase showed that the new protocol worked. We then took this and applied it to our expression vector, YEp352. After cloning, we attempted our yeast transformation. Our team did not have much experience with yeast so there was a steep learning curve. Our initial design of the protocol used one day old culture of BY4741 with a ccc1 knockout, but the transformation failed in the test phase. We examined the concentrations of our yeast using a Coulter Counter and found that the cultures were already in stationary phase, so we modified the protocol to only incubate for 12 hours in 5mL cultures. These were then split and given another two hours of incubation to ensure they were in the log phase of growth and that we had enough for the rest of the protocol. All of these protocols that were designed, built, and tested can be found in the protocols section of the wiki.
— References —
[1] Cunningham, K. W., & Fink, G. R. (1996). Calcineurin inhibits VCX1-dependent H+/Ca2+ exchange and induces Ca2+ ATPases in Saccharomyces cerevisiae. Molecular and Cellular Biology, 16(5), 2226–2237. [2] Cui, J., & Kaandorp, J. A. (2006). Mathematical modeling of calcium homeostasis in yeast cells. Cell Calcium, 39(4), 337-348. [3]"BioNumber Details Page." Maturation Time for SfGFP (super Folded GFP) - Budding Yeast Saccharomyces Ce - BNID 110546. BioNumbers, n.d. Web.
© 2017 Lubbock_TTU
