Team:Bristol/Experiments
BREATHE
Project
Background Description Parts Design Software
Lab
Protocols Experiments Results Notebook Safety Interlab
Modelling
Overview Single Cell Atmosphere Model
Human Practices
Human Practices Integrated Practices Public Engagement Big Bang Fair Pollution Bot
Team
Our Team Collaborations Sponsors Attributions Jamboree Secret page
Experiments
Experi-mental
The first few weeks of our wetlab work was spent completing the Interlab Study . After that, we spent much of our time in the lab using PCR to amplify our fragments and trying         to optimise this. We then attempted to use Gibson assembly and overlapping PCR to join them into full operons. At one point we got pink cells and thought that this might be our protein being expressed! Read all about it below...
PCR for Nap and Nrf fragments
The first stage of our wet lab work was to optimise the Nrf and Nap operons to be BioBrick compatible. This involved removing specific restriction sites, adding spacers, add most importantly adding the BioBrick prefix and suffix (more detail on our Design page). As these modifications would have taken significant time in the lab, we decided to get our constructs synthesised by IDT. After waiting a few weeks, we were informed         that the synthesis was not possible. Therefore, we split each operon into 4 fragments (made to be easier to synthesise), which we could assemble ourselves using Gibson         assembly to obtain the full constructs.
On receiving our Nap and Nrf fragments, we first had to amplify the fragments by PCR to have enough template for Gibson assembly. Unfortunately this was not as straight-forward as it first seemed. It took us a total of 1 month to achieve clean, purified         products of all of our fragments, apart from Nrf3 which still presents problems. Overall, three weeks of this stage was spent optimising the PCR for using Phusion polymerase (Fig. 1).
Fig. 1. Agarose gel of all amplified Nap and Nrf fragments and the pSB1K3 vector used for cloning.
Multiple Gibson assemblies of Nap and Nrf fragments were attempted with the Phusion-derived PCR products, however these consistently resulted in colonies with only re-circularised plasmids (as determined by colony PCR). With only a few weeks remaining,         we decided to move away from Gibson assembly, and attempt overlapping PCR instead to combine our fragments.
Overlapping PCR
We attempted overlapping PCR with all fragments in the reaction together, however this yielded no results. Therefore, we decided to use a simpler, sequential method, combining only two fragments at a time. We used a variant of overlapping PCR whereby         the first 10 cycles are run without any primers. This allows the homology regions of the fragments to anneal, leaving free 3’ ends for the polymerase to elongate, resulting in full dsDNA templates of the combined fragments. A further 25 cycles         are run with the addition of primers flanking the two fragments, thereby amplifying the joint fragments. We managed to assemble Nap1-DT and Nap2 using this method (Fig. 2), however due to non-specificity of our primers, many other incorrect assemblies         were present in the mix (Fig 2.A). Even after gel extracting our band of interest (Fig 2.B), and amplifying this with PCR, the nonspecific bands returned (Fig 2.C), preventing us from continuing with this method.
Fig. 2. A) Agarose gel of overlapping-PCR products. Combinations of fragments are indicated at the top of each lane. Differing annealing temperatures for duplicates of Nap12H and Nap12-DT are indicated. The band circled in blue is the expected size (~ 3.7 kbp) of assembled Nap1-DT (1.8 kbp) and Nap 2 (1.9 kbp). B) Agarose gel of the total remaining product of Nap12-DT overlapping PCR for gel extraction. Band to be excised is highlighted in blue. C) PCR of the product from gel extraction. Nap12-DT is highlighted in blue, and all other bands are nonspecific products.
Discovering Ultra
Our luck changed when we discovered an alternative polymerase to use in our PCR reactions. We discovered this when testing two polymerases, Ultra and Hi-Fi (PCR BIO), to try to obtain better quality PCR products (Fig. 3), as Phusion was giving impure         products. This test showed that Ultra was clearly much better suited to our templates, which would be expected as it has been engineered to have greater activity on difficult templates.
Fig. 3. Agarose gel of PCR products on Nrf2 (lanes 2-5) and Nap2 (lanes 6-9) testing Hi-Fi (lane 5) and Ultra (lane 9), with Phusion as a control.
We therefore decided to re-amplify all of our fragments using Ultra polymerase (Fig. 4), in order to obtain purer PCR products, and to improve our chances of Gibson assembly working. Amplification of Nrf3 however still remained poor.
Fig. 4. Agarose gel of the final purified PCR products of all fragments using Ultra polymerase.
Returning to Gibson assembly and investigating pink cells
Now with much purer fragments, we repeated our Gibson assembly experiments. Although none of our colony PCRs gave promising results, we obtained two Nap colonies which were bright pink in colour (shown right). We decided to investigate             this as it could potentially be a positive result, given that the Nap operon construct does not contain the LacI gene and could therefore be constitutively expressed, and with the operon containing FeS and Heme proteins, could produce coloured             proteins.
Western blotting for the V5-tagged NapA protein gave negative results (not shown), and Coomassie staining of a duplicate gel indicated that the main overexpressed protein was ~30 kDa (Fig. 5A), too small for NapA. Furthermore, restriction         digestion of the plasmid minipreps of these colonies showed an insert of ~1.3 kbp (Fig. 5B), smaller than the expected 6.5 kbp of the Nap operon. The mystery was answered once we got our sequencing results (Fig. 5C), which told us our plasmid         contained RFP. This was due to presence of the original BBa_J04450 pSB1K3 plasmid within our plasmid stock, which we did not digest with Dpn1, meaning there was always a small         chance this plasmid could be transformed into the E. coli yielding our pink cells.
Fig. 5. A) Coomassie stain of total protein from 2 pink colony cell extracts. B) Restriction digest of miniprep plasmid. C) Sequencing results indicated full RFP coding sequence within a pSB1K3 plasmid (VF2 and VR primers).
Fragment editing
As well as aiming to assemble our fragments into the full Nap and Nrf operons, we also aimed to submit as many of the individual genes as we could to the iGEM registry. This would allow future teams to use these genes individually, and would         also add important catalytic components of both Nrf and Nap to the registry. In order to do this, we designed new primers, which would isolate the complete open reading frames (ORFs) from the fragments we had, trimming off unnecessary flanking         sequences, while also adding any required prefix/suffixes. We designed such primers for Nap1, Nap4, Nrf2, Nrf3 and Nrf4 fragments (see our Design page for details),         and successfully obtained PCR products for all. Unfortunately, after ligating the new fragments into pSB1C3 and sending off for sequencing, all of the tested constructs had mutations. Most were severe deletions or insertions, however Nrf2, containing         NrfB, had only one His to Tyr mutation within the NrfB coding sequence (Fig. 6). We don’t yet know the structural or functional implications of the mutation, but submitted to the registry (see Parts ),         with the hope that a future team may still find use in the part, even if they have to edit this mutation for accuracy.
Fig. 6. Sequence alignment of edited NrfB pSB1C3 construct. H89Y mutation is labelled.
Cyclic voltammetry
We planned to conduct a series of experiments to assess the conditions under which ammonia could be oxidised within the context of our bacterial pod, specifically to find out the oxidation potential of ammonia in such conditions, and the range         of power outputs we could expect. The first of these experiments was to measure the oxidation/reduction potential of ammonia in an LB solution using cyclic voltammetry, with LB as a negative control. Latter experiments would be to set up an electrochemical         cell at the measured potential, and attempt to measure the production of current. Unfortunately, we could not measure oxidation of ammonia at any concentration using cyclic voltammetry. We suspect this could be due to incorrect electrode materials,         but with limited time we could not continue this line of investigation any further. See the protocols we used here .
Bristol BREATHE
47-49 St. Michael's Hill, Bristol
Quick Links
Project
Background
Description
Parts
Design
Software
Human Practices
Human Practices
Integrated Practices
Public Engagement
Big Bang Fair
Pollution Bot
Team
Team
Collaborations
Sponsors
Attributions
Jamboree
Sponsors
Bristol iGEM 2017 ©
