<HTML xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en" dir="ltr">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki  ltr ns-0 ns-subject page-Team_UChicago_Notebook"><DIV id="globalWrapper"><DIV id="top-section"><DIV id="p-logo"><A href="/Main_Page" title="Main Page">&quot;
	    </A></DIV><DIV id="menubar" class="left-menu noprint"><UL><LI class="selected"><A href="/Team:UChicago/Notebook">Page               </A></LI><LI class="new"><A href="/wiki/index.php?title=Talk:Team:UChicago/Notebook&amp;action=edit&amp;redlink=1">Discussion               </A></LI><LI><A href="/wiki/index.php?title=Team:UChicago/Notebook&amp;action=edit">View source               </A></LI><LI><A href="/wiki/index.php?title=Team:UChicago/Notebook&amp;action=history">History               </A></LI><LI style="color:white;cursor:default">teams</LI></UL></DIV><DIV class="right-menu noprint" id="menubar"><UL><LI id="pt-login"><A href="/wiki/index.php?title=Special:UserLogin&amp;returnto=Team:UChicago/Notebook" title="You are encouraged to log in; however, it is not mandatory [o]" accesskey="o">Log in</A></LI></UL></DIV><DIV id="search-controls" class="noprint"><FORM action="/Special:Search" id="searchform"> </FORM></DIV></DIV><DIV id="content"><H1 class="firstHeading">Team:UChicago/Notebook</H1><DIV id="bodyContent"><H3 id="siteSub" class="noprint">From 2014.igem.org</H3><DIV id="bg"><DIV id="fg"><DIV id="navbar"><UL><LI><A href="https://2014.igem.org/Team:UChicago">Home</A></LI><LI><A href="https://2014.igem.org/Team:UChicago/Team">Team</A></LI><LI><A href="https://igem.org/Team.cgi?year=2014&amp;team_name=UChicago">Official Team Profile</A></LI><LI><A href="https://2014.igem.org/Team:UChicago/Project">Project</A></LI><LI><A href="https://2014.igem.org/Team:UChicago/Parts">Parts</A></LI><LI><A href="https://2014.igem.org/Team:UChicago/Modeling">Modeling</A></LI><LI><A href="https://2014.igem.org/Team:UChicago/Notebook">Notebook</A></LI><LI><A href="/Team:UChicago/Outreach">Outreach</A></LI><LI><A href="https://2014.igem.org/Team:UChicago/Safety">Safety</A></LI><LI><A href="https://2014.igem.org/Team:UChicago/Attributions">Attributions</A></LI></UL></DIV><DIV id="notebookpage" class="maindiv"><TABLE id="toc" class="toc"><TBODY><TR><TD><DIV id="toctitle"><H2>Contents</H2></DIV><UL><LI class="toclevel-1"><A href="#Lab_Notebook"><SPAN class="tocnumber">1</SPAN><SPAN class="toctext">Lab Notebook</SPAN></A></LI><LI class="toclevel-2 tocsection-1"><A href="#Week_1_6.2F16_-_6.2F20"><SPAN class="tocnumber">1.1</SPAN><SPAN class="toctext">Week 1 6/16 - 6/20</SPAN></A></LI><LI class="toclevel-3 tocsection-2"><A href="#Monday"><SPAN class="tocnumber">1.1.1</SPAN><SPAN class="toctext">Monday</SPAN></A></LI><LI class="toclevel-3 tocsection-3"><A href="#Tuesday"><SPAN class="tocnumber">1.1.2</SPAN><SPAN class="toctext">Tuesday</SPAN></A></LI><LI class="toclevel-3 tocsection-4"><A href="#Wednesday"><SPAN class="tocnumber">1.1.3</SPAN><SPAN class="toctext">Wednesday</SPAN></A></LI><LI class="toclevel-3 tocsection-5"><A href="#Thursday"><SPAN class="tocnumber">1.1.4</SPAN><SPAN class="toctext">Thursday</SPAN></A></LI><LI class="toclevel-3 tocsection-6"><A href="#Friday"><SPAN class="tocnumber">1.1.5</SPAN><SPAN class="toctext">Friday</SPAN></A></LI></UL><LI class="toclevel-2 tocsection-7"><A href="#Week_2_6.2F23_-_6.2F27"><SPAN class="tocnumber">1.2</SPAN><SPAN class="toctext">Week 2 6/23 - 6/27</SPAN></A></LI><LI class="toclevel-3 tocsection-8"><A href="#Monday_2"><SPAN class="tocnumber">1.2.1</SPAN><SPAN class="toctext">Monday</SPAN></A></LI><LI class="toclevel-3 tocsection-9"><A href="#Tuesday_2"><SPAN class="tocnumber">1.2.2</SPAN><SPAN class="toctext">Tuesday</SPAN></A></LI><LI class="toclevel-3 tocsection-10"><A href="#Wednesday_2"><SPAN class="tocnumber">1.2.3</SPAN><SPAN class="toctext">Wednesday</SPAN></A></LI><LI class="toclevel-3 tocsection-11"><A href="#Thursday_2"><SPAN class="tocnumber">1.2.4</SPAN><SPAN class="toctext">Thursday</SPAN></A></LI><LI class="toclevel-2 tocsection-12"><A href="#Week_3_6.2F30_-_7.2F4"><SPAN class="tocnumber">1.3</SPAN><SPAN class="toctext">Week 3 6/30 - 7/4</SPAN></A></LI><LI class="toclevel-3 tocsection-13"><A href="#Monday_3"><SPAN class="tocnumber">1.3.1</SPAN><SPAN class="toctext">Monday</SPAN></A></LI><LI class="toclevel-3 tocsection-14"><A href="#Tuesday_3"><SPAN class="tocnumber">1.3.2</SPAN><SPAN class="toctext">Tuesday</SPAN></A></LI><LI class="toclevel-2 tocsection-15"><A href="#Week_4_7.2F7_-_7.2F11"><SPAN class="tocnumber">1.4</SPAN><SPAN class="toctext">Week 4 7/7 - 7/11</SPAN></A></LI><LI class="toclevel-3 tocsection-16"><A href="#Monday_4"><SPAN class="tocnumber">1.4.1</SPAN><SPAN class="toctext">Monday</SPAN></A></LI><LI class="toclevel-3 tocsection-17"><A href="#Tuesday_4"><SPAN class="tocnumber">1.4.2</SPAN><SPAN class="toctext">Tuesday</SPAN></A></LI><LI class="toclevel-3 tocsection-18"><A href="#Wednesday_3"><SPAN class="tocnumber">1.4.3</SPAN><SPAN class="toctext">Wednesday</SPAN></A></LI><LI class="toclevel-3 tocsection-19"><A href="#Thursday_3"><SPAN class="tocnumber">1.4.4</SPAN><SPAN class="toctext">Thursday</SPAN></A></LI><LI class="toclevel-3 tocsection-20"><A href="#Friday_2"><SPAN class="tocnumber">1.4.5</SPAN><SPAN class="toctext">Friday</SPAN></A></LI><LI class="toclevel-2 tocsection-21"><A href="#Week_5_7.2F14_-_7.2F18"><SPAN class="tocnumber">1.5</SPAN><SPAN class="toctext">Week 5 7/14 - 7/18</SPAN></A></LI><LI class="toclevel-3 tocsection-22"><A href="#Monday_5"><SPAN class="tocnumber">1.5.1</SPAN><SPAN class="toctext">Monday</SPAN></A></LI><LI class="toclevel-3 tocsection-23"><A href="#Tuesday_5"><SPAN class="tocnumber">1.5.2</SPAN><SPAN class="toctext">Tuesday</SPAN></A></LI><LI class="toclevel-3 tocsection-24"><A href="#Wednesday_4"><SPAN class="tocnumber">1.5.3</SPAN><SPAN class="toctext">Wednesday</SPAN></A></LI><LI class="toclevel-3 tocsection-25"><A href="#Thursday_4"><SPAN class="tocnumber">1.5.4</SPAN><SPAN class="toctext">Thursday</SPAN></A></LI><LI class="toclevel-3 tocsection-26"><A href="#Friday_3"><SPAN class="tocnumber">1.5.5</SPAN><SPAN class="toctext">Friday</SPAN></A></LI><LI class="toclevel-3 tocsection-27"><A href="#Saturday"><SPAN class="tocnumber">1.5.6</SPAN><SPAN class="toctext">Saturday</SPAN></A></LI><LI class="toclevel-2 tocsection-28"><A href="#Week_6_7.2F21_-_7.2F25"><SPAN class="tocnumber">1.6</SPAN><SPAN class="toctext">Week 6 7/21 - 7/25</SPAN></A></LI><LI class="toclevel-3 tocsection-29"><A href="#Monday_6"><SPAN class="tocnumber">1.6.1</SPAN><SPAN class="toctext">Monday</SPAN></A></LI><LI class="toclevel-3 tocsection-30"><A href="#Tuesday_6"><SPAN class="tocnumber">1.6.2</SPAN><SPAN class="toctext">Tuesday</SPAN></A></LI><LI class="toclevel-3 tocsection-31"><A href="#Wednesday_5"><SPAN class="tocnumber">1.6.3</SPAN><SPAN class="toctext">Wednesday</SPAN></A></LI><LI class="toclevel-3 tocsection-32"><A href="#Thursday_5"><SPAN class="tocnumber">1.6.4</SPAN><SPAN class="toctext">Thursday</SPAN></A></LI><LI class="toclevel-3 tocsection-33"><A href="#Friday_4"><SPAN class="tocnumber">1.6.5</SPAN><SPAN class="toctext">Friday</SPAN></A></LI><LI class="toclevel-2 tocsection-34"><A href="#Week_7_7.2F28_-_8.2F1"><SPAN class="tocnumber">1.7</SPAN><SPAN class="toctext">Week 7 7/28 - 8/1</SPAN></A></LI><LI class="toclevel-3 tocsection-35"><A href="#Monday_7"><SPAN class="tocnumber">1.7.1</SPAN><SPAN class="toctext">Monday</SPAN></A></LI><LI class="toclevel-3 tocsection-36"><A href="#Tuesday_7"><SPAN class="tocnumber">1.7.2</SPAN><SPAN class="toctext">Tuesday</SPAN></A></LI><LI class="toclevel-3 tocsection-37"><A href="#Wednesday_6"><SPAN class="tocnumber">1.7.3</SPAN><SPAN class="toctext">Wednesday</SPAN></A></LI><LI class="toclevel-3 tocsection-38"><A href="#Thursday_6"><SPAN class="tocnumber">1.7.4</SPAN><SPAN class="toctext">Thursday</SPAN></A></LI><LI class="toclevel-3 tocsection-39"><A href="#Friday_5"><SPAN class="tocnumber">1.7.5</SPAN><SPAN class="toctext">Friday</SPAN></A></LI><LI class="toclevel-2 tocsection-40"><A href="#Week_8_8.2F4_-_8.2F8"><SPAN class="tocnumber">1.8</SPAN><SPAN class="toctext">Week 8 8/4 - 8/8</SPAN></A></LI><LI class="toclevel-3 tocsection-41"><A href="#Monday_8"><SPAN class="tocnumber">1.8.1</SPAN><SPAN class="toctext">Monday</SPAN></A></LI><LI class="toclevel-3 tocsection-42"><A href="#Tuesday_8"><SPAN class="tocnumber">1.8.2</SPAN><SPAN class="toctext">Tuesday</SPAN></A></LI><LI class="toclevel-3 tocsection-43"><A href="#Wednesday_7"><SPAN class="tocnumber">1.8.3</SPAN><SPAN class="toctext">Wednesday</SPAN></A></LI><LI class="toclevel-3 tocsection-44"><A href="#Thursday_7"><SPAN class="tocnumber">1.8.4</SPAN><SPAN class="toctext">Thursday</SPAN></A></LI><LI class="toclevel-3 tocsection-45"><A href="#Friday_6"><SPAN class="tocnumber">1.8.5</SPAN><SPAN class="toctext">Friday</SPAN></A></LI><LI class="toclevel-2 tocsection-46"><A href="#Week_9_8.2F11_-_8.2F15"><SPAN class="tocnumber">1.9</SPAN><SPAN class="toctext">Week 9 8/11 - 8/15</SPAN></A></LI><LI class="toclevel-3 tocsection-47"><A href="#Monday_9"><SPAN class="tocnumber">1.9.1</SPAN><SPAN class="toctext">Monday</SPAN></A></LI><LI class="toclevel-3 tocsection-48"><A href="#Tuesday_9"><SPAN class="tocnumber">1.9.2</SPAN><SPAN class="toctext">Tuesday</SPAN></A></LI><LI class="toclevel-3 tocsection-49"><A href="#Wednesday_8"><SPAN class="tocnumber">1.9.3</SPAN><SPAN class="toctext">Wednesday</SPAN></A></LI><LI class="toclevel-3 tocsection-50"><A href="#Thursday_8"><SPAN class="tocnumber">1.9.4</SPAN><SPAN class="toctext">Thursday</SPAN></A></LI><LI class="toclevel-3 tocsection-51"><A href="#Friday_7"><SPAN class="tocnumber">1.9.5</SPAN><SPAN class="toctext">Friday</SPAN></A></LI><LI class="toclevel-2 tocsection-52"><A href="#Week_10_8.2F18_-_8.2F22"><SPAN class="tocnumber">1.10</SPAN><SPAN class="toctext">Week 10 8/18 - 8/22</SPAN></A></LI><LI class="toclevel-3 tocsection-53"><A href="#Tuesday_10"><SPAN class="tocnumber">1.10.1</SPAN><SPAN class="toctext">Tuesday</SPAN></A></LI><LI class="toclevel-3 tocsection-54"><A href="#Wednesday_9"><SPAN class="tocnumber">1.10.2</SPAN><SPAN class="toctext">Wednesday</SPAN></A></LI><LI class="toclevel-3 tocsection-55"><A href="#Thursday_9"><SPAN class="tocnumber">1.10.3</SPAN><SPAN class="toctext">Thursday</SPAN></A></LI><LI class="toclevel-3 tocsection-56"><A href="#Friday_8"><SPAN class="tocnumber">1.10.4</SPAN><SPAN class="toctext">Friday</SPAN></A></LI><LI class="toclevel-3 tocsection-57"><A href="#Sunday"><SPAN class="tocnumber">1.10.5</SPAN><SPAN class="toctext">Sunday</SPAN></A></LI><LI class="toclevel-2 tocsection-58"><A href="#Week_11_8.2F25_-_8.2F29"><SPAN class="tocnumber">1.11</SPAN><SPAN class="toctext">Week 11 8/25 - 8/29</SPAN></A></LI><LI class="toclevel-3 tocsection-59"><A href="#Monday_10"><SPAN class="tocnumber">1.11.1</SPAN><SPAN class="toctext">Monday</SPAN></A></LI><LI class="toclevel-3 tocsection-60"><A href="#Tuesday_11"><SPAN class="tocnumber">1.11.2</SPAN><SPAN class="toctext">Tuesday</SPAN></A></LI><LI class="toclevel-3 tocsection-61"><A href="#Wednesday_10"><SPAN class="tocnumber">1.11.3</SPAN><SPAN class="toctext">Wednesday</SPAN></A></LI><LI class="toclevel-3 tocsection-62"><A href="#Thursday_10"><SPAN class="tocnumber">1.11.4</SPAN><SPAN class="toctext">Thursday</SPAN></A></LI><LI class="toclevel-3 tocsection-63"><A href="#Saturday_2"><SPAN class="tocnumber">1.11.5</SPAN><SPAN class="toctext">Saturday</SPAN></A></LI><LI class="toclevel-2 tocsection-64"><A href="#Week_12_9.2F1_-_9.2F5"><SPAN class="tocnumber">1.12</SPAN><SPAN class="toctext">Week 12 9/1 - 9/5</SPAN></A></LI><LI class="toclevel-3 tocsection-65"><A href="#Monday--Labor_Day"><SPAN class="tocnumber">1.12.1</SPAN><SPAN class="toctext">Monday--Labor Day</SPAN></A></LI><LI class="toclevel-3 tocsection-66"><A href="#Tueday"><SPAN class="tocnumber">1.12.2</SPAN><SPAN class="toctext">Tueday</SPAN></A></LI><LI class="toclevel-3 tocsection-67"><A href="#Wednesday_11"><SPAN class="tocnumber">1.12.3</SPAN><SPAN class="toctext">Wednesday</SPAN></A></LI><LI class="toclevel-3 tocsection-68"><A href="#Thursday_11"><SPAN class="tocnumber">1.12.4</SPAN><SPAN class="toctext">Thursday</SPAN></A></LI><LI class="toclevel-3 tocsection-69"><A href="#Friday_9"><SPAN class="tocnumber">1.12.5</SPAN><SPAN class="toctext">Friday</SPAN></A></LI><LI class="toclevel-2 tocsection-70"><A href="#Week_13_9.2F8_-_9.2F12"><SPAN class="tocnumber">1.13</SPAN><SPAN class="toctext">Week 13 9/8 - 9/12</SPAN></A></LI><LI class="toclevel-3 tocsection-71"><A href="#Monday_11"><SPAN class="tocnumber">1.13.1</SPAN><SPAN class="toctext">Monday</SPAN></A></LI><LI class="toclevel-3 tocsection-72"><A href="#Tuesday_12"><SPAN class="tocnumber">1.13.2</SPAN><SPAN class="toctext">Tuesday</SPAN></A></LI><LI class="toclevel-3 tocsection-73"><A href="#Wednesday_12"><SPAN class="tocnumber">1.13.3</SPAN><SPAN class="toctext">Wednesday</SPAN></A></LI><LI class="toclevel-3 tocsection-74"><A href="#Thursday_12"><SPAN class="tocnumber">1.13.4</SPAN><SPAN class="toctext">Thursday</SPAN></A></LI><LI class="toclevel-3 tocsection-75"><A href="#Friday_10"><SPAN class="tocnumber">1.13.5</SPAN><SPAN class="toctext">Friday</SPAN></A></LI><LI class="toclevel-3 tocsection-76"><A href="#Saturday_3"><SPAN class="tocnumber">1.13.6</SPAN><SPAN class="toctext">Saturday</SPAN></A></LI><LI class="toclevel-2 tocsection-77"><A href="#Week_14_9.2F15_-_9.2F19"><SPAN class="tocnumber">1.14</SPAN><SPAN class="toctext">Week 14 9/15 - 9/19</SPAN></A></LI><LI class="toclevel-3 tocsection-78"><A href="#Sunday_2"><SPAN class="tocnumber">1.14.1</SPAN><SPAN class="toctext">Sunday</SPAN></A></LI><LI class="toclevel-3 tocsection-79"><A href="#Monday_12"><SPAN class="tocnumber">1.14.2</SPAN><SPAN class="toctext">Monday</SPAN></A></LI><LI class="toclevel-3 tocsection-80"><A href="#Tuesday_13"><SPAN class="tocnumber">1.14.3</SPAN><SPAN class="toctext">Tuesday</SPAN></A></LI><LI class="toclevel-3 tocsection-81"><A href="#Wednesday_13"><SPAN class="tocnumber">1.14.4</SPAN><SPAN class="toctext">Wednesday</SPAN></A></LI><LI class="toclevel-3 tocsection-82"><A href="#Thursday_13"><SPAN class="tocnumber">1.14.5</SPAN><SPAN class="toctext">Thursday</SPAN></A></LI><LI class="toclevel-3 tocsection-83"><A href="#Friday_11"><SPAN class="tocnumber">1.14.6</SPAN><SPAN class="toctext">Friday</SPAN></A></LI><LI class="toclevel-3 tocsection-84"><A href="#Saturday_4"><SPAN class="tocnumber">1.14.7</SPAN><SPAN class="toctext">Saturday</SPAN></A></LI><LI class="toclevel-2 tocsection-85"><A href="#Week_15_9.2F22_-_9.2F26"><SPAN class="tocnumber">1.15</SPAN><SPAN class="toctext">Week 15 9/22 - 9/26</SPAN></A></LI><LI class="toclevel-3 tocsection-86"><A href="#Sunday_3"><SPAN class="tocnumber">1.15.1</SPAN><SPAN class="toctext">Sunday</SPAN></A></LI><LI class="toclevel-3 tocsection-87"><A href="#Monday_13"><SPAN class="tocnumber">1.15.2</SPAN><SPAN class="toctext">Monday</SPAN></A></LI><LI class="toclevel-3 tocsection-88"><A href="#Tuesday_14"><SPAN class="tocnumber">1.15.3</SPAN><SPAN class="toctext">Tuesday</SPAN></A></LI><LI class="toclevel-3 tocsection-89"><A href="#Wednesday_14"><SPAN class="tocnumber">1.15.4</SPAN><SPAN class="toctext">Wednesday</SPAN></A></LI><LI class="toclevel-3 tocsection-90"><A href="#Thursday_14"><SPAN class="tocnumber">1.15.5</SPAN><SPAN class="toctext">Thursday</SPAN></A></LI><LI class="toclevel-3 tocsection-91"><A href="#Saturday_5"><SPAN class="tocnumber">1.15.6</SPAN><SPAN class="toctext">Saturday</SPAN></A></LI></TD></TR></TBODY></TABLE><H1><SPAN class="mw-headline" id="Lab_Notebook">Lab Notebook</SPAN></H1><P>See <A href="/wiki/images/a/a3/UChicago_LabNotebookSummary.pdf" class="internal" title="UChicago LabNotebookSummary.pdf">Media:UChicago_LabNotebookSummary.pdf</A> for the original document.
</P><H2><SPAN class="mw-headline" id="Week_1_6.2F16_-_6.2F20">Week 1 6/16 - 6/20</SPAN></H2><H3><SPAN class="mw-headline" id="Monday">Monday</SPAN></H3><P>First day of lab! We plated some DH5-alpha competent cells in preparation for gDNA extraction for our PCRs and future transformations. One person showed up (Kevin yayyyyy), then left. 
</P><H3><SPAN class="mw-headline" id="Tuesday">Tuesday</SPAN></H3><P>We attempted to make LB plates. We had fun watching it boiling over since we left it in the hot plate too long and it spilled everywhere. We did clean-up time afterwards! In the future, we will remember to to heat for only 1 min to dissolve everything except agar. Agar will dissolve during autoclave because science is cool. So exciting! :D  We then remade the LB for funsies :D We did a few other things to start things up like making SOB, CCMB80 competent cell buffer, and antibiotic plates.
</P><H3><SPAN class="mw-headline" id="Wednesday">Wednesday</SPAN></H3><P>MOAR BUFFERS! Qiagen buffers are expensive so we spent the day making bootleg Qiagen miniprep buffers (sorry Qiagen, you had <A href="/wiki/index.php?title=Team:UChicago/Donate&amp;action=edit&amp;redlink=1" class="new" title="Team:UChicago/Donate (page does not exist)">your chance</A>). Then, we contemplated life for the rest of the day. *take depressing pictures here*
</P><UL><LI>take pictures of buffers*
</LI></UL><H3><SPAN class="mw-headline" id="Thursday">Thursday</SPAN></H3><P>Made competent cells. Stuck hands in liquid nitrogen. Burnt our hands off (no pics because you need hands to use a camera, duh). EVEN MOAR BUFFERS
We also went to a biotech company exhibit and grabbed a lot of free samples! (and food of course)
We also extracted gDNA from DH5a to PCR out our mutator genes. We also learned not to use the free 1.5mL centrifuge tubes from last year, and to not smell our microcentrifuge anymore.
Stock strains MG1655 and delta-tyrR came in from the Yale E Coli Stock Center. Yay! So exciting! Kevin was so excited that he burnt his hair trying to plate them. XD
</P><P>We made competent cells (MY CRYSTAL BALL SAYS:  THEY WILL FAIL). Kevin BS’d the gDNA and got some somehow but also made the centrifuge stink like chloroform/phenol. Fun XD
</P><P>This company (name concealed) gave us some cool centrifuge tubes last year that warp when we spin them down. XD So we’re buying new ones. $$$ They’re wrong- money DOES grows on trees! AND we have lots of trees around here! Even better! We ran overnight PCRs, one with more and one with less cycles.
</P><H3><SPAN class="mw-headline" id="Friday">Friday</SPAN></H3><P>PCR-ed out 9/12 mutators. The lengths of almost all of them were verified to be correct!!! So much happy!!! Yes!!!! Yay!!!! The PCR with less cycles (bottom half of the gel) worked better so we’ll stick with that in the future. We also digested backbones that have the RBS we will use and ligate to all the mutators. What a good day!
</P><H2><SPAN class="mw-headline" id="Week_2_6.2F23_-_6.2F27">Week 2 6/23 - 6/27</SPAN></H2><H3><SPAN class="mw-headline" id="Monday_2">Monday</SPAN></H3><P>We tested our competent cells using the transformation efficiency kit. They didn’t work XD But that’s okay, we like making competent cells. It’s strangely exciting to pipette things lots of times :D
</P><P>Also, we fragment isolated the mutations and then did the restriction digests on the PCR products. The concentrations of the genes are fairly alright, about 20-50 ng/microL. We ran a gel of AroF to verify PCR of the gene, but unfortunately we did not get a band XD. That’s okay, life goes on even if your gels fail!!! XD IT’S summer time! Rainbows!
</P><P>We gel extracted the mutators that were digested last week. Fun. Got high contamination. But contamination is good because what doesn’t kill your experiments makes them stronger right ? Yes! Concentrations are listed below XD
</P><TABLE class="wikitable"><CAPTION>Mutator Concentrations
</CAPTION><TBODY><TR><TH>Sample</TH><TH>Conc (ng/microL)</TH><TH>260/280</TH><TH>260/230
</TH></TR><TR><TD>ParC</TD><TD>30.2</TD><TD>1.78</TD><TD>0.25
</TD></TR><TR><TD>DinB</TD><TD>55.8</TD><TD>1.70</TD><TD>0.42
</TD></TR><TR><TD>MutS</TD><TD>21.3</TD><TD>1.86</TD><TD>0.18
</TD></TR><TR><TD>Dam</TD><TD>37.0</TD><TD>1.68</TD><TD>0.18
</TD></TR><TR><TD>MutT</TD><TD>29.9</TD><TD>1.61</TD><TD>0.09
</TD></TR><TR><TD>Umu’C</TD><TD>24.5</TD><TD>1.80</TD><TD>0.05
</TD></TR><TR><TD>EmrR</TD><TD>36.8</TD><TD>1.71</TD><TD>0.08
</TD></TR><TR><TD>DnaE</TD><TD>31.1</TD><TD>1.53</TD><TD>0.32
</TD></TR></TBODY></TABLE><H3><SPAN class="mw-headline" id="Tuesday_2">Tuesday</SPAN></H3><P>NOTHING WORKED :)))))!!!!!!!! We drank our worries away XD XD XD. We drunkenly made glycerol stocks of the two strains from Yale that came last Thursday.
</P><UL><LI>hic* guuysh guysh we should *burrrrp* totally ligate a rBs to a vecter *hi*c 
</LI></UL><P>Alcohol is poison. (Delicious, delicious poison. And we are Mithridates.)
</P><P>Unfortunately, our cells weren’t competent when we tested them again :( We went over the protocol and made sure we’ll have everything down for next time XD. Or rather, fortunately! We get to make more! XD Soooo stoked!
We fragment isolated and gel purified our promoter plasmids. 
For our project planning, we met with our faculty advisors Mike and Jennifer. The meeting was super, super, super productive and educational! We could consider inserting a polylinker into our plasmids to reduce a lot of steps XD Polylinkers are fun!
</P><H3><SPAN class="mw-headline" id="Wednesday_2">Wednesday</SPAN></H3><P>We purified the 8 mutators that we PCR-ed out. As for AroF and the three mutators that didn’t work, we tried PCR with Phusion, but unfortunately it didn’t work again. Yay!
After digesting our RBS and ligating it with our previous cut promoter plasmids, we also did 3A assembly in parallel for RBS and promoter. Yay!
More amp and tet plates were made! Yayyy!
</P><UL><LI>picture of plates”
</LI></UL><H3><SPAN class="mw-headline" id="Thursday_2">Thursday</SPAN></H3><P>We had issues with SOC contamination today - We had to make a new batch for more transformations. We also tried transformation with the efficiency kit again, making sure we were following each step as it was written on our protocol.
Friday
Transformation of our 3A assembly colonies didn’t work, but the transformation of mcherry plasmids worked! This means that there’s nothing wrong with our transformation protocol. We will look into our digestion and ligation steps. We &lt;3 troubleshooting! All part of the scientific process XD!
Here’s a joke XD:
What do you call it when McDonald sells cherries?
mCherry! HAHAHA XD!
</P><P>We made more chloramphenicol plates again! Yay!
</P><H2><SPAN class="mw-headline" id="Week_3_6.2F30_-_7.2F4">Week 3 6/30 - 7/4</SPAN></H2><H3><SPAN class="mw-headline" id="Monday_3">Monday</SPAN></H3><P>We resuspended and transformed the promoter+RBS plasmids from the distribution kit. They turn red when resuspended! Such a pretty color!
</P><H3><SPAN class="mw-headline" id="Tuesday_3">Tuesday</SPAN></H3><P>A lot of our reagents came in today! WOOOOOOOOo!
Since we’re having problems with transformation right now, we are troubleshooting our procedures to see if there’s anything wrong with our technique or protocols. We followed the iGEM standard transformation protocol and transformed our promoter+RBS plasmids again. Let’s hope they work! 
</P><H2><SPAN class="mw-headline" id="Week_4_7.2F7_-_7.2F11">Week 4 7/7 - 7/11</SPAN></H2><H3><SPAN class="mw-headline" id="Monday_4">Monday</SPAN></H3><P>Made overnight cultures of DH5a and IPTG+RBS to make competent cells/miniprep tomorrow. Inoculated MG1655 for gDNA and generic promoter + RBS for miniprep. Transformed plasmids containing mCherry+LVA (on cam) and GFP(on amp) into DH5a cells from distribution kit (mcherry: 2014 plate 3 well 12F and 2013 plate 3 well 11F, GFP 2014 plate 3 well 10k). Made 7ml overnight culture of MG1655.
</P><H3><SPAN class="mw-headline" id="Tuesday_4">Tuesday</SPAN></H3><P>Transformants all showed no growth (positive RFP control showed some growth but not red). Attempted to make DH5a competent cells. Ran mutator PCR products on a gel:
</P><P>1: Ladder
2: Negative control
3: mutD
4: aroF
5: MutH
6: MutY 
7: MutL
8: DinB
paroF successfully amplified at .3kb. 
</P><H3><SPAN class="mw-headline" id="Wednesday_3">Wednesday</SPAN></H3><P>Colonies are now growing on 7/7’s mCherry-LVA transformants plate and the positive control colonies have turned red. Transformed newly made competent cells with RFP plasmids to determine competency. PCR was performed with phusion and taq on the remaining mutators.
</P><H3><SPAN class="mw-headline" id="Thursday_3">Thursday</SPAN></H3><P>Amplicons from yesterday were run on a gel:
</P><P>P denotes phusion, T denotes Taq
</P><OL><LI>Ladder
</LI><LI>P1 -- 1.1kb
</LI><LI>P2 -- 0.7kb
</LI><LI>P3 -- N/A
</LI><LI>P4 -- N/A
</LI><LI>Ladder
</LI><LI>T1 -- 0.7kb
</LI><LI>T2 -- 0.7kb
</LI><LI>T3 -- 1.1kb
</LI><LI>T4 -- 0.7kb
</LI><LI>T5 -- 1.5.b
</LI><LI>Neg
</LI></OL><P>no colonies for competency assay. Re-ran PCR on other mutators using taq and phusion.
</P><H3><SPAN class="mw-headline" id="Friday_2">Friday</SPAN></H3><P>3 colonies on test plate (cells are not very competent). Miniprepped GFP plasmid ( cells borrowed from a different lab).
Ran gel from yesterday’s PCR:
</P><P>Top gel: 1 ladder, 2 old primers mutD(taq), 3 new primers mutD(taq), 4 mutY old primers(taq), 5 mut Y new primers (phusion), 6 mutH old primers (taq), 7 mut H new (phusion), 8 mut L new (phusion), 9 negative control taq, 10 negative control phusion, 11 ladder
</P><P>Bottom gel: 1 ladder, 2 mutL old (taq), 3 mutL new (taq)
</P><H2><SPAN class="mw-headline" id="Week_5_7.2F14_-_7.2F18">Week 5 7/14 - 7/18</SPAN></H2><H3><SPAN class="mw-headline" id="Monday_5">Monday</SPAN></H3><P>We got the plasmids we ordered last week, including the 2 melA constructs (K193601 and K193602). We might not need them for a while, but always good to have them on hand. They came in agar stabs, so we restreaked them on plates. A bunch of our media was contaminated (again) so we had to remake all those. We also transformed our IPTG+GFP and our AroF+mCherry constructs, along with Cam, Kan, Tet, and Amp linearized backbones from the kit plate (other plasmids, cut in order to isolate the backbone).
</P><P>(I’m calling this transformation.jpg now.)
</P><H3><SPAN class="mw-headline" id="Tuesday_5">Tuesday</SPAN></H3><P>Yep. Transformation of our ligated parts and the backbones seems to have failed. We left the plates in the incubator for one more night because these cells have an annoying habit of only growing after two days. At least our stab streaks were good: all 7 of the plasmids, including the important melA constructs, grew successfully, and were made into overnight cultures.
Oh hey! After a few hours more there is some growth on the backbone plates, except for the Tet backbone. Transformation efficiencies are pretty terrible, as is par for the course here. Seriously need to fix that. 
We also redid our AroF+mCherry and our IPTG+GFP ligations with 3A assembly and the standard ligation procedure. Better work this time. We also miniprepped and fragment isolated more mCherry and GFP so that we have enough for the inevitable future redos. We’re also continuing work on determining the MIC for rifampicin for our fluctuation assays; this time growing them in liquid culture.
</P><H3><SPAN class="mw-headline" id="Wednesday_4">Wednesday</SPAN></H3><P>Made glycerol stocks of most of Monday’s new plasmids, and miniprepped K193601 and K193602. Ran a bunch of PCRs to make sure we had our promoters. Didn’t get IPTG+GFP, but at least we got a little bit of AroF+mCherry. We also checked our MIC liquid cultures, and it looks like the MIC we got was about 8ug/mL. We also transformed our ligation products into our competent cells and competent cells from another lab. The other lab’s cells will most assuredly have better transformation efficiencies than our cells.
</P><H3><SPAN class="mw-headline" id="Thursday_4">Thursday</SPAN></H3><P>Looks like our 3A fragments were cut incorrectly due to mislabeling somewhere along the line. Would probably explain why they failed this morning.
We can make our competent cells better, more competent. We have the technology. And buffer. Which we made today.
</P><H3><SPAN class="mw-headline" id="Friday_3">Friday</SPAN></H3><P>Plated some XL1-Blue cells to start making our competent cells. Hopefully these ones work better. Also tried ligating and transforming more mutators and promoters; hopefully we can do this without mislabeling things again.
(ipredictfailure.jpg)
</P><H3><SPAN class="mw-headline" id="Saturday">Saturday</SPAN></H3><P>Made a culture of XL1-Blue cells with 250mL LB medium, growing cells to an optimal OD of 0.6 to transform. Looks like this will be going overnight, as growth is slow. Yesterday’s transformation products finally worked without incident: making them into overnight cultures  so that we can have stocks in case of temporary incompetence.
</P><H2><SPAN class="mw-headline" id="Week_6_7.2F21_-_7.2F25">Week 6 7/21 - 7/25</SPAN></H2><H3><SPAN class="mw-headline" id="Monday_6">Monday</SPAN></H3><P>We are spending the week attempting to clone our mutators into our IPTG-inducible promoter+GFP plasmid. We used up some of the fragment isolated mutators we had, so we gel purified more from wild type. The mutators and IPTG+GFP were appropriately digested (mutators with XbaI and PstI and IPTG+GFP with SpeI and PstI). We also performed a cell competency test on our cells (which are probably not competent, knowing our luck).
</P><H3><SPAN class="mw-headline" id="Tuesday_6">Tuesday</SPAN></H3><P>At the suggestion of our advisors, we are now dephosphorylating our plasmids before cloning to decrease background (which was a problem with our previous digestions). This should help a lot yay!! We dephosphorylated IPTG+GFP/S+P and gel purified the dephosphorylated digest. We cloned the mutators into IPTG+GFP using a ratio of 3:1 insert:vector in a 10uL reaction with 25ug of vector. We also submitted IPTG+GFP, mCherry+AroF, promoter+TyrR for sequencing. (We also made a ton of cam plates so no one freaks out the next time they’re transforming something and realizes there are no plates). This was a super late day--at lab past midnight=fun, fun, fun :)
</P><H3><SPAN class="mw-headline" id="Wednesday_5">Wednesday</SPAN></H3><P>Our SOB is contaminated and an aliquot of SOC that was never used before contaminated. What is this madness?!?! More SOB was made. We also colony PCRed IPTG+GFP, mCherry+AroF, and promoter+TyrR in preparation for negative sequencing results (Always be prepared is one of our many mottos. We stole it from the Boy Scouts.). We also got Tyrosine!!!!
</P><H3><SPAN class="mw-headline" id="Thursday_5">Thursday</SPAN></H3><P>Well, our sequencing results came back, and it appears we have MAJOR issues with mislabelling. While we do have IPTG+GFP confirmed, aroF and TyrR are not in anything
The plasmid mcherry is not even in anything. What was labeled as aroF+mCherry actually contains the general promoter, and one labeled promoter+TyrR is actually IPTG but without GFP!!! Are our mCherry and promoter plasmids mislabeled?! Or was it just the ligation products? The struggle is five real. We made overnight cultures of all the ligation products that looked promising based on the amount of colonies (our colony PCR did not work and this will allow us to keep moving forward while troubleshooting colony PCR): 4x IGMD (IPTG---GFP---MutD), 4x IGMH (IPTG---GFP---MutH). 4x IGMT (IPTG---GFP---MutT). 4x IGDam/IGD (IPTG---GFP---Dam). 4x IGUC (IPTG---GFP---Umu’C), 4x IGER (IPTG---GFP---EmrR).
</P><H3><SPAN class="mw-headline" id="Friday_4">Friday</SPAN></H3><P>Miniprepped all overnight cultures that were made yesterday of ligation constructs. We then PCRed them and chose constructs to submit for sequencing.
</P><H2><SPAN class="mw-headline" id="Week_7_7.2F28_-_8.2F1">Week 7 7/28 - 8/1</SPAN></H2><H3><SPAN class="mw-headline" id="Monday_7">Monday</SPAN></H3><P>This week we’re continuing lots of cloning. We were planning to make competent cells, but messed up the OD (which should be about 0.6) :(. Sadness. PCR’d out the AroF promoter and the TyrR gene and fragment isolated them.
</P><H3><SPAN class="mw-headline" id="Tuesday_7">Tuesday</SPAN></H3><P>Re-inoculated new cells to retry making competent cells again. Hopefully they work this time. Did appropriate digestions on paroF, TyrR, the mCherry plasmid, and the IPTG-GFP mutator plasmid as we continue to clone all three different constructs (mutators, general promoter-TyrR, and paroF-mCherry). Did PCR’s using VF and VR2 primers to confirm the identity of the mCherry and constitutive promoter plasmids. However, mCherry looks possibly bad, so we ended up going ahead with cloning of only TyrR into the constitutive promoter plasmid. We got back sequencing results for three previous attempted ligations of Dam, EmrR, and MutT into IPTG-GFP, all of which look good. Yay, first real biobricks!
</P><H3><SPAN class="mw-headline" id="Wednesday_6">Wednesday</SPAN></H3><P>The cell line we’re using to make competent cells (XL1-Blu) that we’re trying to grow up is growing up faster than we expect, so we made a growth curve for how the OD changes over time. Did colony PCRs on attempted ligations of various mutators (Mut L, MutS, DinB, DnaE, MutY) into TyrR, but they all looked bad. Ligation fail.
</P><H3><SPAN class="mw-headline" id="Thursday_6">Thursday</SPAN></H3><P>We made competent cells today finally, and transformed some test plates to make sure they actually work. We redid the PCR to verify the mCherry plasmid, which again looked bad, so we’re just going to retransform mCherry just to be safe.
</P><H3><SPAN class="mw-headline" id="Friday_5">Friday</SPAN></H3><P>Success! Competent cells are actually competent after about 12 hours of growth. They look possibly pink, just as expected (since the plasmid they’re transformed with expresses RFP)
</P><H2><SPAN class="mw-headline" id="Week_8_8.2F4_-_8.2F8">Week 8 8/4 - 8/8</SPAN></H2><H3><SPAN class="mw-headline" id="Monday_8">Monday</SPAN></H3><P>Resuspended and transformed mcherry+lva from the distribution kit again. We've been using this a lot and looks like we’re running pretty low on this! Let’s hope the transformation will succeed and our glycerol stock will be fine! Also transformed the sequence verified IPTG+GFP plasmid and those with EmrR, Dam and MutT for later use in the mutation rate analysis.
</P><H3><SPAN class="mw-headline" id="Tuesday_8">Tuesday</SPAN></H3><P>Transformation yesterday seems ok! Since there are a few colonies on the negative control, we restreaked the transformed colonies just to be sure.
And finally, the long-awaited MIC determination test was done today! As a preliminary experiment, we plated saturated, 1:100 and 1:10000 dilutions of XL1-Blue cells onto cam plates with various rif concentrations.
</P><H3><SPAN class="mw-headline" id="Wednesday_7">Wednesday</SPAN></H3><P>The restreaked colonies grew! Promptly checked the colonies by colony PCR and made overnight cultures of them for miniprepping. 
</P><P>Only less than 5 colonies grew on the 5ug/ml rif plates, and none on the other plates! This is unexpected, as there should be as many colonies as the plain LB plates in the lower rif concentration plates. Looks like there’s gonna be more rif experiments! 
</P><H3><SPAN class="mw-headline" id="Thursday_7">Thursday</SPAN></H3><P>As we accidentally added an extra RBS overhang in our AroF primer, we PCRed out AroF again using our new primer without the overhang. The gel looks good! Let's miniprep it! 
</P><P>And what??? 4 out of 7 overnight cultures from yesterday didn’t grow? And we have to pick the colonies again? We really need mcherry+lva… :(
But the IPTG+GFP plasmids look promising. Miniprepped the plasmids and ran them on a gel. Look at this beautiful gel! Now let’s cut it up for gel extraction! 
</P><H3><SPAN class="mw-headline" id="Friday_6">Friday</SPAN></H3><P>What? Why didn’t the mcherry colonies grow again? :( Retransformed mcherry again. Ligated promoter and TyrR while waiting for transformation. 
For the mutator plasmids, the remaining mutators were digested and ligated into the IPTG+GFP plasmids. Transformation results will be ready tomorrow! The ligated MutT and EmrR plasmids from earlier this week were miniprepped and stored. 
</P><H2><SPAN class="mw-headline" id="Week_9_8.2F11_-_8.2F15">Week 9 8/11 - 8/15</SPAN></H2><H3><SPAN class="mw-headline" id="Monday_9">Monday</SPAN></H3><P>It looks like one of the tubes labelled as “promoter plasmid” was actually IPTG+GFP. Whoopsy-daisy. Also transformed general promoter from kit plate.
A lot of the mutator transformation plates worked. Woot! We can now use them for cloning and stuff. Did colony PCR for a bunch of them and then ran on gels. MutH, MutY, and UmuC all gave good results.
</P><H3><SPAN class="mw-headline" id="Tuesday_9">Tuesday</SPAN></H3><P>Made glycerol stocks of mCherry. Miniprepped mCherry+Iva, then restriction digested and gel purified it.
Ran colony PCR reactions for MutS, MutL, and DinB. Ran gels for DnaE, UmuC (redo), and ParC (redo). The redos were done because redundancy is useful in science. Redundancy is redundant. Redundancy is also redundant.
Made a bunch of variable Rifampicin plates and diluted MG1655 into 200ul aliquots for mic determination.
</P><P>Somewhere, in a deep, dark corner of the gel room, someone started making new TAE Buffer...but their mind was in physics mode instead of bio mode…*foreshadowing intensifies*
</P><H3><SPAN class="mw-headline" id="Wednesday_8">Wednesday</SPAN></H3><P>Did colony PCR on a boatload (okay, it would have to be a really small boat) of IPTG+GFP so that we’d have plenty for future gels. Ran gels for MutS, MutL, and DinB. MutS and DinB gave some good results, but the two MutL gels didn’t work (one showed nothing, the other disintegrated). And so the buffer problems began.
Miniprepped MutH, ParC, UmuC, DnaE, and MutY. Twice. The first time failed miserably, since it turns out that one of the buffers (P2) was improperly made and had been giving us screwy results for a while. Some members wanted to keep it for some reason, so we just gave it a choke collar label that said “BAD” on it. The second miniprep attempt was successful with a better-behaved buffer.
Made glycerol stocks of promoter. Restriction digested and gel purified promoter. Ligated mCherry+paroF and promoter+TyrR and transformed products.
Plated independent cultures of MG1655 onto restrictive media and made dilutions (only up to around 10^6, though, nothing like that 1/10^30 homeopathy BS)
</P><H3><SPAN class="mw-headline" id="Thursday_8">Thursday</SPAN></H3><P>Sent in a variety of mutators (MH, MY, UC, DE, and PC) for sequencing. Miniprepped DinB and MutS samples. Digested and ligated MutL and MutD with increased insert:vector ratio (now 5:1). Did Colony PCR on IGER3 and IG3 samples then ran on gels (used separate TAE buffer from the main stock, but the gels still failed anyways).
<STRIKE>Ran Colony PCR and gels for mCherry+paroF and Promoter+TyrR</STRIKE> Nevermind! The buffer is DEFINITELY bad, so there go the results for that! Time to do it all over again *eye twitches*.
Made more LB, rif, and dilution plates since the results from the previous ones were inconclusive.
</P><H3><SPAN class="mw-headline" id="Friday_7">Friday</SPAN></H3><P>Re-did the gels for mCherry+paroF and Promoter+TyrR.
Sent in DinB and MutS for sequencing. Transformed MutL, MutD, and IPTG+GFP into competent cells. It turns out that we didn’t need the IGER3 and IG3 overnight cultures that were made on Thursday, so they were thrown out.
Made new LB agar plates. Selective growth plates from the other day showed no growth, but the nonselective test plates showed reasonable amounts of growth.
</P><H2><SPAN class="mw-headline" id="Week_10_8.2F18_-_8.2F22">Week 10 8/18 - 8/22</SPAN></H2><P>Transformation results from 8/15
for IPTG+GFP+mutL (IGML),  IPTG+GFP+mutD (IGMD),  IPTG+GFP (IGIG), and negative control:
</P><P>Low transformation efficiency suggests that the transformation was unsuccessful but mutD appears to be successful from the following colony PCR gel:
</P><P>gel key:
Well 1: 4 µl of ladder
Well 2: 8 µl of IPTG+GFP
Well 3: Sample 1
Well 4: Sample 2
Well 5: Sample 3
Well 6: Sample 4
Well 7: Sample 5
Well 8: Sample 6
Well 9: Sample 7
Well 10: Sample 8
</P><P>mutD gel:
</P><P>mutL gel:
</P><P>Transformation of IPTG+GFP+mutL appears to be unsuccessful but samples 4-8 of mutD appear to have the mutD insert as they have higher base pair count.
</P><P>Quikchange of mutY and DinB was attempted but likely unsuccessful due to lack of any appropriate bands on the below gel and insufficient signal from sequencing
</P><P>1: Ladder
2: Neg control
3: MutY
4: DinB
Quikchange results transformed into XL1Blue cells. Overnight ligation of 10:1 TyrR:promoter(plasmid) attempted. Transformed ParC3, MutY7, DinB1, MutS8, MutH2, MutD5, and MutD8 into XL1Blue
Mutation rate assay:
Measured OD600 of XL1B wt overnight culture at 12:20: 1.617 (roughly 1.29x10^9 cells/mL) and made an inoculum with 5000 cells/mL in LB (with 1mM IPTG). Made 20 200ul aliquots of inoculum +IPTG and put into incubator at 1:16 pm.
</P><H3><SPAN class="mw-headline" id="Tuesday_10">Tuesday</SPAN></H3><P>Transformed mCherry+paroF, promoter+TyrR, mCherry+ligase, promoter+ligase, mCherry cut, promoter cut. No growth on mutY and DinB quikchange transformant plates. Attempted 2 more protocols for quikchange for mutY (2-step and 60oC annealing temperature). Both appeared to fail.
</P><P>1: ladder
2: 60C negative control
3: 60C MutY product
4: 2-Step negative control
5: 2-Step MutY product
</P><P>2 mutD-positive colonies from 8/18 miniprepped. mutation rate assay:
200ul cultures taken out of 37 degree shaker at 2:30 (~25 hrs), plated (16x) 200 ul on 80ug/mL rif plates as well as 200ul of 10^6 and 10^7 dilutions from 4 different overnight cultures on LB plates. The overnight culture of melA did not grow (the melA plasimid: BBa_K193602 has amp resistance rather than cam as listed on the registry). Inoculated overnight cultures at 5:00 in 5mL LB of XL1B, EmrR3, EmrR4, Dam).  Resuspended 2 likely rif resistant colonies and plated on 80ug/mL rif plates to check if rif-resistant bacteria could grow at such a high concentration of rif.
</P><H3><SPAN class="mw-headline" id="Wednesday_9">Wednesday</SPAN></H3><P>PCRed out and gel-purified TyrR, paroF. Yesterday’s cut plasmids w/o ligase transformants yielded colonies, suggesting that the digestion and dephosphorylaion failed. Re-digested TyrR, paroF, mCherry, promoter and dephosphorylated mCherry and Promoter containing plasmids. Made overnight cultures of yesterday’s mutator transformants (ParC3, MutY7, DinB1, MutS8, MutH2, MutD5, and MutD8) . Redigested Umu’C and mutL and ligated into IPTG+GFP plasmid. melA overnight culture does not look any darker to the naked eye; a melA overnight culture was made with 250ug/mL tyrosine and 1mM IPTG in LB. Mutation rate assay: Dam master inoculum made of ~5000 cfu/mL from yesterday’s overnight culture, appropriate concentration of cam and 1mM IPTG which was then aliquotted to 19 200ul cultures. In addition, a rif80 plate streaked with likely rif-resistant colony from 8/18 have near-lawn growth showing that 80ug/mL rif is not too high a concentration to completely hinder growth. No growth in EmrR3 culture -reinoculated. In addition, XL1Blue overnight cultures were inoculated.
</P><P>colony counts from XL1B wt test plates (200ul plated of the stated dilution):
</P><TABLE class="wikitable"><CAPTION>Colony Counts
</CAPTION><TBODY><TR><TH>culture</TH><TH>10^6 dil</TH><TH>10^7 dil
</TH></TR><TR><TD>1</TD><TD>61</TD><TD>8
</TD></TR><TR><TD>2</TD><TD>70</TD><TD>3
</TD></TR><TR><TD>3</TD><TD>63</TD><TD>1
</TD></TR><TR><TD>4</TD><TD>55</TD><TD>3
</TD></TR></TBODY></TABLE><P>This suggests a cfu/mL concentration of ~5X10^7 was plated on selective plates (a low amount).
</P><H3><SPAN class="mw-headline" id="Thursday_9">Thursday</SPAN></H3><P>Overnight cultures of ParC3, MutS, MutH2, DnaE, DinB1, MutY7, MutD5, and MutD8 miniprepped. Made additional overnight cultures of the above for glycerol stocks. mutY quikchange plates from 8/19 show a few colonies on the 2-step mutagenesis samples; these colonies have been used to inoculate overnight cultures. Ran 2-step quikchange protocol on DnaE, ParC, and mutH containing plasmids. Transformed yesterday’s IPTG+GFP+Umu’C/mutL ligation products as well as just IPTG+GFP into XL1Blue cells. paroF and TyrR were PCRed from genomic DNA and fragment isolated. paroF was digested and ligated into the mCherry plasmid, and TyrR into the promoter plasmid and the ligation products transfomed into XL1Blue. Mutation rate assay: 1 rif-resistant colony observed across 16 XL1Blue selective plates (plated on 8/19): likely a function of low plated cfu concentration. Significant growth on 2 rif80 plates platedwith  rif-resistant bacteria from 8/19 further confirming that rif concentration is not an issue. Plated yesterday’s Dam 200ul culture aliquots on selective media as well as non-selective test plates (10^6,10^7,10^8 dilutions for three cultures).  Made 20 mL culture aliquots of XL1Blue wt and put in 37 degree shaker (up from 200ul). Re-inoculated melA overnights for miniprep tomorrow (melA+tyrosine is not dark/barely grew and we want to sequence).
</P><H3><SPAN class="mw-headline" id="Friday_8">Friday</SPAN></H3><P>Made glycerol stocks of PC3, MS, MH2, DE, DB1, MY7, MD5, and MD8. Quikchange transformant plates from 8/21 show a few small colonies only on quikchanged mutH. Made colony suspensions of yesterday’s Umu’C and mutL ligation product transformants and ran on a gel after colony PCR: 
Order:
Well 1: Ladder
Well 2: IPTG+GFP plasmid
Well 3: Sample 1
Well 4: Sample 2
Well 5: Sample 3
Well 6: Sample 4
Well 7: Sample 5
Well 8: Sample 6
Well 9: Sample 7
Well 10: Sample 8
MutL gel:
</P><P>unsuccessful.
Umu’C gel:
</P><P>No amplicons at the correct band (greater than 1kb).
</P><P>Miniprepped mutY-quikchanged overnight cultures for sequencing. Quikchange transformants from yesterday still show only colonies(5) on the mutH plate. Transformed promoter+TyrR and aroF + mCherry ligation products yielded no colonies so re-ligated and retransformed. 
</P><H3><SPAN class="mw-headline" id="Sunday">Sunday</SPAN></H3><P>Mutation rate assay: no colonies on rif80 plates for Dam (16). Test plates showed inconsistent results (no colonies for culture 1 at any dilution, 1,0,4 colonies for culture 2 at 10^6 10^7, 10^8 dilutions, and 0, 2, 3 colonies for culture 3 at 10^6,10^7,10^8 dilutions). Plated XL1Blue 200ulx18 of the 2ml cultures on rif80 plates as well as 200ul of a 10^6 dilution on agar plates for 3 cultures.  
</P><P>Miniprepped melA but failed to achieve yield over 30ng/ul even with saturated 2x5ml cultures
</P><H2><SPAN class="mw-headline" id="Week_11_8.2F25_-_8.2F29">Week 11 8/25 - 8/29</SPAN></H2><H3><SPAN class="mw-headline" id="Monday_10">Monday</SPAN></H3><P>We finally got our AroF+mCherry construct ligated in! Ran a colony PCR to verify these, and looks good so far. Made overnight cultures for miniprep tomorrow.
We also Dpn1-digested last week’s ParC and DnaE quickchange products and transformed them in.
</P><P>Par for the course with this, really. We also ran a quickchange with ParC and DnaE again, this time under two different conditions: twice the amount of initial DNA and twice the amount of extension time (which is already too long). We transformed in the 2X DNA samples but the 2X time samples were not transformed until tomorrow. Also ran a colony PCR on last week’s MutL and UmuC ligation attempts: looks like half our UmuC colonies were successful, but all of MutL failed. 
</P><H3><SPAN class="mw-headline" id="Tuesday_11">Tuesday</SPAN></H3><P>Miniprepped the mCherry+AroF constructs while admiring the culture’s pinkish color. Sent that and a UmuC ligation attempt in for sequencing. We also ran another PCR of the TPC and TDE quickchange products (the ones with a doubled extension time), and while one of the TDE samples had promise, none of the TPC samples showed anything. Still went ahead with all the transformations anyway, because these things tend to be wonky at times. We also miniprepped mMutH for sequencing and MelA for verification.
</P><H3><SPAN class="mw-headline" id="Wednesday_10">Wednesday</SPAN></H3><P>Oh hey, a good sign! Monday’s mDnaE and mParC plates show a possible colony. Made overnight cultures of those and redid our MutS and MutY mutagenesis, with 2X DNA concentrations, since that seemed to have worked well. We transformed those in after Dpn1 digestion. Also sent in mMutH samples for sequencing and religated MutL, MutD, and DinB. We also ligated our promoter+TyrR construct and began a mutation rate assay of wild-type XL1-B cells.
</P><H3><SPAN class="mw-headline" id="Thursday_10">Thursday</SPAN></H3><P>Transformed in the constructs we ligated yesterday. Also miniprepped yesterday’s overnight cultures, but not much growth was seen. Miniprep yields were disappointingly low, so there go those samples (i cry evry tym). At least we had colonies grow on our rifampicin plates. Made Dam overnight cultures and new IPTG stock.
Friday
Really good results for our TyrR construct! Looks like we’re actually starting to make progress! Colony PCR doesn’t seem to be working for this, but hopefully sequencing on Monday will tell us if we got it for certain. No growth on yesterday’s ligation plates, however, but we’re giving the plates one extra day to grow. We seem to be having the same problems that we had a few weeks before. Maybe it’s just a fluke. Also miniprepped yesterday’s overnight cultures. Yields were somewhat low as usual, but looks like we at least have enough for sequencing. Redoing MutS, ParC mutagenesis with a 68C extension temperature.
</P><H3><SPAN class="mw-headline" id="Saturday_2">Saturday</SPAN></H3><P>Better results than usual! Apparently we had a colony on mMutY. We still need to get it sequenced, but maybe this one could work? Keeping our fingers crossed. We also miniprepped a bunch of mMutY samples.
</P><H2><SPAN class="mw-headline" id="Week_12_9.2F1_-_9.2F5">Week 12 9/1 - 9/5</SPAN></H2><H3><SPAN class="mw-headline" id="Monday--Labor_Day">Monday--Labor Day</SPAN></H3><P>Rikki was the only one who came in on a Federal Holiday. She ran colony PCR on Promoter+TyrR construct (but expected it not to work since for some reason it never amplifies the promoter well--she was right, you couldn’t see a band). She also made overnight cultures of the promoter+TyrR but only made two because she could not find more culture tubes. 
</P><H3><SPAN class="mw-headline" id="Tueday">Tueday</SPAN></H3><P>Miniprepped the two overnight cultures of Promoter+TyrR. We ran out of Promega buffers so we tried invitrogen, but the yields were too poor on one so only one sample was submitted. Guess this was a good thing because--spoiler--the results were negative. We also tried ethanol precipitation for miniprep yields which were too low to submit for sequencing and discovered experimentally that sometimes it works and sometimes it doesn’t. In addition to submitting Promoter+TyrR for sequencing, we also submitted mutagenized MutY and DnaE. DinB, MutD, MutL, UmuC, and IPTG+GFP were digested and the mutators were ligated to IPTG+GFP, this time as an overnight ligation at 4C (Promaga t4 DNA ligase calls for overnights to be done at 4C rather than 16C). We tried site-directed mutagenisis on ParC, MutS, MutY, and DnaE using turbo. Overnight cultures were made of the melanin plasmid, as well as IGDM and IGER (for rif assay).
</P><H3><SPAN class="mw-headline" id="Wednesday_11">Wednesday</SPAN></H3><P>Transformed ligation products of IPTG+GFP+ DinB, MutD, MutL, UmuC. Assuming failure for Quickchange, trying again but using Turbo, Turbo hot start, Phusion hot start, Q5, and Q5 hot start polymerases side-by-side. Hopefully one will work this time :/ Promoter+TyrR sequencing came back negative today, so 12 overnight cultures were made (since colony PCR doesn’t work well for this construct). Rikki found a random IPTG+GFP+MutD plate in the cold room that looked promising (a lot of colonies, some visibly green, some white). Colony PCR was run on four green colonies and four white colonies.
</P><H3><SPAN class="mw-headline" id="Thursday_11">Thursday</SPAN></H3><P>Miniprepped Promoter+TyrR overnights. Rikki meshed together a new protocol using invitrogen buffers but following a lot of Promega’s timing and the yield’s are great--like 500ng/uL). The promoter+TyrR was digested with XbaI and PstI and run on a gel alongside digested promoter:
</P><P>ladder, promoter, samples 1-5, sample 12
</P><P>ladder, promoter, samples 6-11
</P><P>It looks like some of the samples have TyrR!!! (1541 bp) Sample 10 probably didn’t cut right.
Submitting samples 1, 3, 12, 7 for sequencing tomorrow morning. Looks promising!!
</P><P>Colony PCRed MutD:
</P><P>IPTG+GFP was in the first lane, but must not have been added correctly.We compared the bp size to our IPTG+GFP plasmid and it lookes like 5-8 are just IPTG+GFP (they were picked from green colonies). Samples 1-4 (picked white colonies) appear to have MutD, so they will be miniprepped and sent in for sequencing.
Plated EmrR overnight cultures on rif plates and dilutions on control plates (following the rif protocol) and made overnights of EmrR and Dam for the next batch of rif assays.
Miniprepped more IPTG+GFP for future ligations.
</P><H3><SPAN class="mw-headline" id="Friday_9">Friday</SPAN></H3><P>Submitted promising Promoter+TyrR and IPTG+GFP+MutD for sequencing. These results should be good!!! Three of the four mutators that were Quikchanged have colonies on the plates! It actually looks like Turbo doesn’t work as well as Q5 of Phusion, which is probably why our results for Quikchange have been pretty bad so far. Overnight cultures were made of these colonies. The ligations of IPTG+GFP+ DinB, MutD, MutL, UmuC don’t appear to have worked, and we’re probably going to scrap anything we don’t get soon.
</P><H2><SPAN class="mw-headline" id="Week_13_9.2F8_-_9.2F12">Week 13 9/8 - 9/12</SPAN></H2><P>Summary
This was the week that several people (Rikki and Ryan B) were leaving lab and all our stuff suddenly started working last week, so it was an incredibly hectic week where we all stayed in lab until 2 or 3 in the morning every day. Summary of mutator progress by the end of this week (note to Derek/others: we should put this table up each week when available):
</P><TABLE class="wikitable"><CAPTION>Mutator Summary
</CAPTION><TBODY><TR><TH>Mutator</TH><TH>Colony PCR?</TH><TH>Seq Verified?</TH><TH>Quikchange?</TH><TH>Overall progress
</TH></TR><TR><TD>Dam</TD><TD>✓</TD><TD>✓</TD><TD>Not needed</TD><TD>Complete
</TD></TR><TR><TD>DinB</TD><TD>X</TD><TD>X</TD><TD>F13V</TD><TD>N/A (may be able to get MIT else retry)
</TD></TR><TR><TD>DnaE</TD><TD>✓</TD><TD>✓</TD><TD>E612K</TD><TD>Need sequencing primers to verify mutations! Ordered 9/9
</TD></TR><TR><TD>EmrR</TD><TD>✓</TD><TD>✓</TD><TD>Not needed</TD><TD>Complete
</TD></TR><TR><TD>MutD</TD><TD>✓</TD><TD>✓</TD><TD>L73W, A164V</TD><TD>1st mutation in progress
</TD></TR><TR><TD>MutH</TD><TD>✓</TD><TD>✓</TD><TD>E56A</TD><TD>Complete
</TD></TR><TR><TD>MutL</TD><TD>X</TD><TD>X</TD><TD>E32K</TD><TD>N/A
</TD></TR><TR><TD>MutS</TD><TD>✓</TD><TD>✓</TD><TD>G619D</TD><TD>Complete
</TD></TR><TR><TD>MutT</TD><TD>✓</TD><TD>✓</TD><TD>Can’t use :(</TD><TD>N/A
</TD></TR><TR><TD>MutY</TD><TD>✓</TD><TD>✓</TD><TD>V45A</TD><TD>Need sequencing primers to verify (start from bp 1)! ordered 9/10
</TD></TR><TR><TD>ParC</TD><TD>✓</TD><TD>✓</TD><TD>Y120H</TD><TD>Probably not doing due to the internal cut site. Eliot-Ryan can give it a shot though.
</TD></TR><TR><TD>UmuC</TD><TD>X (MIT is sending it out. will deal with getting primers for it once we get it 9/11) it requires UmuD’ which has the first 24 amino acids of UmuD removed)</TD><TD>X</TD><TD>Y11A</TD></TR></TBODY></TABLE><H3><SPAN class="mw-headline" id="Monday_11">Monday</SPAN></H3><P>We started PCRs to PCR out the mutators that have been successfully ligated (and Quikchanged when required) from the IPTG-GFP plasmid into the final ParoF-mCherry containing backbone we will actually carry out FREP in. For short, we call this ParoF-mCherry backbone ARM. We also continued various things, such as cloning DinB and MutL into IPTG-GFP, Quikchanging TyrR to make a mutated version of TyrR that has increased sensitivity and range of response to tyrosine, various already ligated things, and inoculating completely mutated Dam, EmrR, alongside wild-type XL1 Blu for the mutation rate assay. To test the promoter response to tyrosine repression, we also inoculated cells transformed with the AroF-TyrR construct into various concentrations of tyrosine (ranging from 0 to around 250 mg/L)
</P><H3><SPAN class="mw-headline" id="Tuesday_12">Tuesday</SPAN></H3><P>Designed and ordered more oligos that we needed. We continued the Quikchanges and ligations from yesterday. Sequencing results came back, which allowed us to confirm MutS mutagenesis working. However, we’ll probably have to jump ship on ParC unfortunately, because there was an internal cut site that we only just now discovered/bothered to check, which means we only ligated in half of ParC :/ Sadness. We continued trying to PCR out, fragment isolate, and ligate the mutators (Dam EmrR, and MutH) into the ARM construct that we can use for FREP. Measured various OD’s from yesterday’s inoculation for the mutation rate assay, plating a humongous number of plates also (literally hundreds of plates). We also got plate reader data for the assay for promoter response to tyrosine repression, though it looks kinda sketch for one of the strains (appearing to be induced rather than repressed).
</P><P>Happy birthday Ryan! ^This was at 2 AM on Wednesday morning since that’s how long we were in lab for. Stayed in lab till it was already Ryan’s birthday the next day, what a way to celebrate it XD
</P><P>When the last people finally left lab at 3AM wednesday morning, we were all incredibly happy as we walked out the door. :D
</P><H3><SPAN class="mw-headline" id="Wednesday_12">Wednesday</SPAN></H3><P>We needed to make more competent cells (which we inoculated last night), but sadly overgrew it, so we’re gonna reinoculate and try again tomorrow. We transformed ARM ligations from yesterday, PCR’d to get linearized amp backbone we’ll need for the final functional TyrR construct (right now it’s on a chloramp backbone, but we need it on amp since the mutator construct will already be on chloramp). We ran colony PCR on the MutL and DinB ligations, both of which luckily showed lots of colonies.
</P><H3><SPAN class="mw-headline" id="Thursday_12">Thursday</SPAN></H3><P>Sad farewells :( Rikki and Ryan B left last night, so we are down to two people manning the lab. Eliot will be back next week and Kevin will be leaving after Friday. We miniprepped MutS and mutated TyrR (the latter of which needs to be sent in for sequencing verification of the mutagenesis) from yesterday. We also collected data for the mutation rate assay: 
</P><TABLE class="wikitable"><CAPTION>Colony count of XLB and Dam
</CAPTION><TBODY><TR><TH>E</TH><TH>XLB</TH><TH>Dam
</TH></TR><TR><TD>1</TD><TD>0 colonies</TD><TD>5 colonies</TD><TD>0 colonies
</TD></TR><TR><TD>2</TD><TD>Near-lawn to lawn growth. Bad plate?</TD><TD>4 colonies</TD><TD>0 colonies
</TD></TR><TR><TD>3</TD><TD>0 colonies</TD><TD>7 colonies</TD><TD>0 colonies
</TD></TR><TR><TD>4</TD><TD>0 colonies</TD><TD>0 colonies</TD><TD>0 colonies
</TD></TR><TR><TD>5</TD><TD>0 colonies</TD><TD>4 colonies</TD><TD>0 colonies
</TD></TR><TR><TD>6</TD><TD>0 colonies</TD><TD>6 colonies</TD><TD>0 colonies
</TD></TR><TR><TD>7</TD><TD>0 colonies</TD><TD>10 colonies</TD><TD>0 colonies
</TD></TR><TR><TD>8</TD><TD>0 colonies</TD><TD>1 colonies</TD><TD>0 colonies
</TD></TR><TR><TD>9</TD><TD>N/A</TD><TD>8 colonies</TD><TD>0 colonies
</TD></TR><TR><TD>10</TD><TD>N/A</TD><TD>2 colonies</TD><TD>0 colonies
</TD></TR><TR><TD>11</TD><TD>N/A</TD><TD>11 colonies</TD><TD>1 colonies
</TD></TR><TR><TD>12</TD><TD>N/A</TD><TD>10 colonies</TD><TD>0 colonies
</TD></TR><TR><TD>13</TD><TD>N/A</TD><TD>0 colonies</TD><TD>0 colonies
</TD></TR><TR><TD>14</TD><TD>N/A</TD><TD>N/A</TD><TD>Bad plate? 1 cluster of colonies
</TD></TR><TR><TD>15</TD><TD>N/A</TD><TD>N/A</TD><TD>0 colonies
</TD></TR><TR><TD>16</TD><TD>N/A</TD><TD>N/A</TD><TD>0 colonies
</TD></TR></TBODY></TABLE><P>Looks like most of the mutator expressing plates have few colonies compared to wild type for whatever reason. Did colony PCR on the ARM and IPTG-GFP ligations from previous days, and found a few suitable colonies to send in (MutL specifically), though we need to do more colony PCR on some of the ligations (DinB, EmrR and DnaE specifically).
We also have been collecting lysate from cells transformed with the various mutators we’ve been ligating, and ran it on an SDS-PAGE gel to confirm that the IPTG-GFP plasmids are properly expressing the mutators. We have a pic of the gel on the next day.
Ladder
</P><OL><LI>XLIB/IPTG (9/9) (7)
</LI><LI>XLIB+IPTG (9/9) (1)
</LI><LI>XLIB+IPTG (9/9) (5)
</LI><LI>XLIB+IPTG (9/9) (6)
</LI><LI>XLIB+IPTG (9/4)
</LI><LI>EmrR+IPTG (9/9) (8)
</LI><LI>EmrR+IPTG (9/9) (10)
</LI><LI>EmrR+IPTG (9/4) (1)
</LI><LI>EmrR+IPTG (9/4) (2)
</LI><LI>EmrR+IPTG (9/4) (3)
</LI><LI>EmrR/IPTG (9/9) (3)
</LI><LI>EmrR/IPTG (9/9) (4)
</LI><LI>EmrR/IPTG (9/4) (1)
</LI><LI>EmrR/IPTG (9/4) (2)
</LI><LI>Dam+IPTG (9/9) (9)
</LI><LI>Dam/IPTG (9/9) (2)
</LI><LI>Dam/IPTG (9/9) (11)
</LI></OL><H3><SPAN class="mw-headline" id="Friday_10">Friday</SPAN></H3><P>We somehow lost our old mutated DnaE samples, so we went through the process of re-Quikchanging wild-type DnaE (alongside our newly ligated MutL). We also plan to ligate our sequence-verified, mutagenized MutS into ARM, so we set up the digests for that today, will need to ligate tomorrow. We ran more Colony PCRs of the ones that didn’t show any positives from before (see Thursday), and finally got more positives for EmrR and DamR, but still not for DinB (likely we’ll have to . We finished processing/imaging the protein gel we ran yesterday. A link here <A href="/File:UChicago_wk13gel1.tif" title="File:UChicago wk13gel1.tif">File:UChicago wk13gel1.tif</A>.
Note: the protein gel is inverted so lane labels as listed from yesterday are backwards. The two rows should be identical; they’re just done at different protein concentrations to see which one would image better. We got some plasmids from Walker lab at MIT also today, which we put on nonselective LB plates for recovery. We may need these plasmids for PolB and UmuD’C, but on second thoughts (as you’ll see later), we ended up not actually needing them. We checked Dam and EmrR plates from the mutation rate assay that were plated on Thursday for growth, but there were few/no colonies, so we’ll wait one more day and see what happens as time works its magic.
</P><H3><SPAN class="mw-headline" id="Saturday_3">Saturday</SPAN></H3><P>Ryan S came in for a bit today to do various household chores. Checked plates we restreaked yesterday from glycerol stock to repeat the mutation rate assay, restreaked Walker lab plasmids from yesterday on selective plates (cam for the PolB plasmid and amp for the UmuD’C plasmid). He checked Dam and EmrR plates from the mutation rate assay that were plated on Thursday once again for growth and recorded down counts. Transformed various Quikchanges and ligations from earlier in the week we had been stalling an transforming (we wanted to transform a whole bunch of things at once to make things more efficient). Finally, made overnight cultures of the colony PCRs from Friday that looked good for EmrR and DamR that had been ligated into the ARM construct.
</P><H2><SPAN class="mw-headline" id="Week_14_9.2F15_-_9.2F19">Week 14 9/15 - 9/19</SPAN></H2><P>Summary
</P><P>Running out of time/last final stretch: Everything’s coming together, but we still have a ton of work to do.
</P><H3><SPAN class="mw-headline" id="Sunday_2">Sunday</SPAN></H3><P>Ryan S counted colonies on the nonselective XL1B plates and checked all the plates from yesterday. None of the transformation have large colonies so they were put for an additional day. 
Miniprepped the overnight cultures of Dam and EmrR that actually grew. We have one of each! UmuD’C and PolB from the selective plates were innoculated and cultured in selective media. 
Made glycerol stocks of the 4 mutators we have, and performed mutation rate analysis. Made a mastermix in LB, rif and IPTG at the right cell dilution and aliquotted out for overnight cultures. They’re now in the incubator and ready for tomorrow’s plating! 
</P><H3><SPAN class="mw-headline" id="Monday_12">Monday</SPAN></H3><P>Checked all the transformation plates but none of them seems to have growth :(
Good news is, the overnight cultures of the mutators for mutation rate analysis (EmrR, MutS, MutH, Dam) have growth, so they were plated on LB plates and Rif plates when the right OD is reached. 
DinB and UmuD’C were digested and ligated onto IPTG+GFP. PolB and UmuD’C were miniprepped and glycerol stocks were made. The miniprep sample concentration was really high - they were around 500 ng/ul! 
</P><H3><SPAN class="mw-headline" id="Tuesday_13">Tuesday</SPAN></H3><P>UmuD’C and linearized amp backbone. Digested the UmuD’C, DinB and Dam and ligated them into IPTG+GFP with a 3:1 vector-insert ratio. AREmrR was sent in for sequencing and overnight cultures for ARDam was made for miniprepping. 
Transformation plates from yesterday and the day before looked bad. The nonselective plates showed a lot of growth while the selective plates don’t. Seems like we need to do ligations and quickchange again…
</P><P>Eliot then worked frantically to get something done! He quikchanged MutD, DnaE, MutL and TyrR. Then, he re-ligated UmuD’C and DinB into IPTG+GFP. 
Sequencing results: 
We have our mutated samples of mMutL, mMutY and mMutS and AroF+mCherry+mMutH! There was an extraneous mutation in TyrR that we used for all the ligations though. Fortunately, one of the templates (PT3) has a silent mutation. We could continue using this and make new quikchange templates of TyrR. 
</P><H3><SPAN class="mw-headline" id="Wednesday_13">Wednesday</SPAN></H3><P>Sent in ARMEmrR and ARMDam for sequencing! However, overnights of ARMDam appear to have no growth...Made overnights again for miniprep and glycerol stocks. WE FOUND A GLYCEROL STOCK FOR THE ARM BACKBONE!! Quickly plated them for miniprep. 
</P><H3><SPAN class="mw-headline" id="Thursday_13">Thursday</SPAN></H3><P>ARMDam overnights finally have growth! Glycerol stocks made and miniprepped. Checked the mutation rate assay plates but not all of the selective plates have growth. We’ll check them again tomorrow! Made overnight cultures of XL1-B and JW-1316-1 for competent cells and ARM backbone for miniprepping. Also digested and ligated mMutY and mMutS into ARM backbone with 5:1 ratio. We ran out of competent cells so we’ll transform the products once the competent cells are made! 
</P><H3><SPAN class="mw-headline" id="Friday_11">Friday</SPAN></H3><P>Miniprepped ARM backbone from yesterday’s overnight cultures. XL1-B and JW-1316-1 were cultured and competent cells were made. What a long day in the cold room! The cells were tested for their transformation efficiency using the iGEM kit. 
Quikchanged MutL, MutD, DnaE and TyrR. Ligations from yesterday and these quikchanged products were transformed into competent Xl1-B and plated. 
</P><H3><SPAN class="mw-headline" id="Saturday_4">Saturday</SPAN></H3><P>Transformation efficiency of our competent cells is approximately 2X10^6. Hmmm it’s a bit lower than what we want but we’ll just go with it. 
More overnight cultures of the mutators on IPTG+GFP were made for the mutation rate analysis!
</P><H2><SPAN class="mw-headline" id="Week_15_9.2F22_-_9.2F26">Week 15 9/22 - 9/26</SPAN></H2><P>Summary
</P><P>It’s the final countdown! And we still don’t (and probably won’t) have the final FREP stuff good to go! (*Panicking intensifies*). But don’t FRET over FREP! McGuire and Eliot will do their best to finish what they can and lament what they cannot.
</P><H3><SPAN class="mw-headline" id="Sunday_3">Sunday</SPAN></H3><P>Ryan continued the Mutation Rate Assay protocol, checking ODs and making according overnight cultures. He also autoclaved various items and transformed Q1-5, P1-5, ARMY, ARMS, and ARMS1 for Eliot.
</P><H3><SPAN class="mw-headline" id="Monday_13">Monday</SPAN></H3><P>Ryan checked ODs and plated cultures for the Mutation Rate Assay. He also started the final Mutation Rate Assay attempt, using XL1B with IPTG+GFP, Dam, IGER4, mMutH1, and mMutH2. McGuire also autoclaved a small boatload of micropipette tips.
All work on cloning/mutagenesis was stopped today. End results:
</P><TABLE class="wikitable"><CAPTION>Mutator Summary
</CAPTION><TBODY><TR><TH>Mutator</TH><TH>Colony PCR?</TH><TH>Seq Verified?</TH><TH>Quikchange?</TH><TH>Overall progress
</TH></TR><TR><TD>Dam</TD><TD>✓</TD><TD>✓</TD><TD>Not needed</TD><TD>Complete
</TD></TR><TR><TD><STRIKE>DinB</STRIKE></TD><TD><STRIKE>X</STRIKE></TD><TD><STRIKE>X</STRIKE></TD><TD><STRIKE>F13V</STRIKE></TD><TD><STRIKE>N/A (may be able to get MIT else retry)</STRIKE></TD></TR><TR><TD><STRIKE>DnaE</STRIKE></TD><TD><STRIKE>✓</STRIKE></TD><TD><STRIKE>✓</STRIKE></TD><TD><STRIKE>E612K</STRIKE></TD><TD><STRIKE>Canceled at mutagenesis</STRIKE></TD></TR><TR><TD>EmrR</TD><TD>✓</TD><TD>✓</TD><TD>Not needed</TD><TD>Complete
</TD></TR><TR><TD><STRIKE>MutD</STRIKE></TD><TD><STRIKE>✓</STRIKE></TD><TD><STRIKE>✓</STRIKE></TD><TD><STRIKE>L73W, A164V</STRIKE></TD><TD><STRIKE>Canceled at mutagenesis</STRIKE></TD></TR><TR><TD>MutH</TD><TD>✓</TD><TD>✓</TD><TD>E56A</TD><TD>Complete
</TD></TR><TR><TD><STRIKE>MutL</STRIKE></TD><TD><STRIKE>X</STRIKE></TD><TD><STRIKE>X</STRIKE></TD><TD><STRIKE>E32K</STRIKE></TD><TD><STRIKE>Canceled at mutagenesis</STRIKE></TD></TR><TR><TD><STRIKE>MutS</STRIKE></TD><TD><STRIKE>✓</STRIKE></TD><TD><STRIKE>✓</STRIKE></TD><TD><STRIKE>G619D</STRIKE></TD><TD><STRIKE>Canceled at ARM ligation</STRIKE></TD></TR><TR><TD><STRIKE>MutT</STRIKE></TD><TD><STRIKE>✓</STRIKE></TD><TD><STRIKE>✓</STRIKE></TD><TD><STRIKE>Can’t use :(</STRIKE></TD><TD><STRIKE>N/A</STRIKE></TD></TR><TR><TD>MutY</TD><TD>✓</TD><TD>✓</TD><TD>V45A</TD><TD>Complete
</TD></TR><TR><TD><STRIKE>ParC</STRIKE></TD><TD><STRIKE>✓</STRIKE></TD><TD><STRIKE>✓</STRIKE></TD><TD><STRIKE>Y120H</STRIKE></TD><TD><STRIKE>Probably not doing due to the internal cut site. Eliot-Ryan can give it a shot though.</STRIKE></TD></TR><TR><TD><STRIKE>UmuC</STRIKE></TD><TD><STRIKE>X (MIT is sending it out. will deal with getting primers for it once we get it 9/11) it requires UmuD’ which has the first 24 amino acids of UmuD removed)</STRIKE></TD><TD><STRIKE>X</STRIKE></TD><TD><STRIKE>Y11A</STRIKE></TD><TD><STRIKE>Cancelled at IG ligation, time constraints</STRIKE></TD></TR></TBODY></TABLE><P>Eliot started concentrating on the tyrosine assay. Since ligating TyrR into ARM would take too long and be subject to setbacks, he used only the ARM backbone with endogenous, wild-type TyrR in DH5a. Eliot transformed TyrR into DH5A cells today, growth in LB with TyrR to come on Tuesday.
</P><H3><SPAN class="mw-headline" id="Tuesday_14">Tuesday</SPAN></H3><P>Ryan continued the Mutation Rate Assay protocol, checking ODs and making according overnight cultures. Also checked the nonselective plates from Monday.
Eliot delayed transformation of the ARM backbone delayed since either he goofed and added DNA to the negative control or contamination or plates were bad or something. Ergo, Eliot decided to redo the transformation, making sure to grab a plate from a different batch this time.
</P><H3><SPAN class="mw-headline" id="Wednesday_14">Wednesday</SPAN></H3><P>Ryan checked ODs and plated cultures for the Mutation Rate Assay and made 40 more rif plates.
Eliot’s transformation went much better this time. Negative control looked weird: there might be some growth but the agar in this plate was weird, so it could have been the agar looking like that. ARM plate actually showed good growth, however, compared to the other plate. Eliot made the tyrosine cultures and added samples of 3 colonies to each. Needed to match the tyrosine concentrations, though. Eugene suggested starting from a controlled concentration and diluting it down, so trying that. Planning to pick 3 colonies, add them to different concentrations of tyrosine.
Concentrations: 0-500mg/L, in 50mg/L intervals (50, 100, 150…)
3 colonies picked, suspended in 35uL H2O each.
2uL suspended colonies put in 5uL LB/Tyr
</P><H3><SPAN class="mw-headline" id="Thursday_14">Thursday</SPAN></H3><P>Eliot took cultures at 12:05 PM, total incubation period is 16 hours. Successful growth occurred in most cultures, with the exception of 2 vials: should not affect results too much if data is consistent across all 3 colonies, as the vials correspond to different colonies and are not in the range of 200-300mg/L tyrosine, which appears to be the point at which AroF is repressed judging from the previous assay. He loaded approx. 200 µL culture into wells of 96-well plate. Rows A, B, and C corresponded to colonies 1, 2, and 3. Columns 1-12 were: 0=LB, no cells, 1, 0mg/uL tyrosine, 2=50mg/L tyr, 3= 100mg/L tyr, … 12= 500mg/L tyr. Measured 570-610mm.
Ryan checked nonselective plates for the Mutation Rate Assay. All fluoresced (yay)!
Friday
Ryan and Eliot were both busy, so no one went into the lab on this day of Fries.
</P><H3><SPAN class="mw-headline" id="Saturday_5">Saturday</SPAN></H3><P>Ryan stared across the barren wasteland lab, wondering if it was all worth it. Had all the blood, sweat, and tears been shed for this? This...end? It seemed that this experiment shall go out with a whimper rather than a bang...Ah, well.
Ryan checked the selective plates for the Mutation Rate Assay. All of these had fluorescing colonies as well.
McGuire headed to the Museum of Science and Industry to drink some blue lemonade for free whilst standing inside of a manmade mini-tornado.
</P></DIV><DIV class="printfooter">
Retrieved from &quot;<A href="http://2014.igem.org/Team:UChicago/Notebook">http://2014.igem.org/Team:UChicago/Notebook</A>&quot;</DIV></DIV></DIV><DIV id="footer-box" class="noprint"><DIV id="footer"><UL id="f-list"><LI id="t-recentchanges"><A href="/Special:RecentChanges" title="Recent changes">Recent changes</A></LI><LI id="t-whatlinkshere"><A href="/Special:WhatLinksHere/Team:UChicago/Notebook" title="List of all wiki pages that link here [j]" accesskey="j">What links here</A></LI><LI id="t-recentchangeslinked"><A href="/Special:RecentChangesLinked/Team:UChicago/Notebook" title="Recent changes in pages linked from this page [k]" accesskey="k">Related changes</A></LI><LI id="t-specialpages"><A href="/Special:SpecialPages" title="List of all special pages [q]" accesskey="q">Special pages</A></LI><LI><A href="/Special:Preferences">My preferences</A></LI></UL></DIV><DIV id="footer"><UL id="f-list"><LI id="t-print"><A href="/wiki/index.php?title=Team:UChicago/Notebook&amp;printable=yes" title="Printable version of this page [p]" accesskey="p">Printable version</A></LI><LI id="t-permalink"><A href="/wiki/index.php?title=Team:UChicago/Notebook&amp;oldid=397023" title="Permanent link to this revision of the page">Permanent link</A></LI><LI id="privacy"><A href="/2014.igem.org:Privacy_policy" title="2014.igem.org:Privacy policy">Privacy policy</A></LI><LI id="disclaimer"><A href="/2014.igem.org:General_disclaimer" title="2014.igem.org:General disclaimer">Disclaimers</A></LI></UL></DIV></DIV></DIV></DIV></DIV></BODY></HTML>