<HTML xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en" dir="ltr">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki  ltr ns-0 ns-subject page-Team_UChicago_Project"><DIV id="globalWrapper"><DIV id="top-section"><DIV id="p-logo"><A href="/Main_Page" title="Main Page">&quot;
	    </A></DIV><DIV id="menubar" class="left-menu noprint"><UL><LI class="selected"><A href="/Team:UChicago/Project">Page               </A></LI><LI class="new"><A href="/wiki/index.php?title=Talk:Team:UChicago/Project&amp;action=edit&amp;redlink=1">Discussion               </A></LI><LI><A href="/wiki/index.php?title=Team:UChicago/Project&amp;action=edit">View source               </A></LI><LI><A href="/wiki/index.php?title=Team:UChicago/Project&amp;action=history">History               </A></LI><LI style="color:white;cursor:default">teams</LI></UL></DIV><DIV class="right-menu noprint" id="menubar"><UL><LI id="pt-login"><A href="/wiki/index.php?title=Special:UserLogin&amp;returnto=Team:UChicago/Project" title="You are encouraged to log in; however, it is not mandatory [o]" accesskey="o">Log in</A></LI></UL></DIV><DIV id="search-controls" class="noprint"><FORM action="/Special:Search" id="searchform"> </FORM></DIV></DIV><DIV id="content"><H1 class="firstHeading">Team:UChicago/Project</H1><DIV id="bodyContent"><H3 id="siteSub" class="noprint">From 2014.igem.org</H3><DIV id="bg"><DIV id="fg"><DIV id="navbar"><UL><LI><A href="https://2014.igem.org/Team:UChicago">Home</A></LI><LI><A href="https://2014.igem.org/Team:UChicago/Team">Team</A></LI><LI><A href="https://igem.org/Team.cgi?year=2014&amp;team_name=UChicago">Official Team Profile</A></LI><LI><A href="https://2014.igem.org/Team:UChicago/Project">Project</A></LI><LI><A href="https://2014.igem.org/Team:UChicago/Parts">Parts</A></LI><LI><A href="https://2014.igem.org/Team:UChicago/Modeling">Modeling</A></LI><LI><A href="https://2014.igem.org/Team:UChicago/Notebook">Notebook</A></LI><LI><A href="/Team:UChicago/Outreach">Outreach</A></LI><LI><A href="https://2014.igem.org/Team:UChicago/Safety">Safety</A></LI><LI><A href="https://2014.igem.org/Team:UChicago/Attributions">Attributions</A></LI></UL></DIV><DIV id="projectinfopage" class="maindiv"><H1>Our Project</H1><DIV id="projectinfo"><H3>Introduction</H3><P>
Directed evolution is a fundamental technique in bioengineering organisms to express proteins of novel function and to mass produce industrially relevant biomolecules. Generally, directed evolution simulates an algorithmic process in which the entire sequence space is searched for an optimal genotype by increasing the natural mutation rate to artificially speed the process of selective evolution. Current methods of in vivo directed evolution typically rely on physical or chemical mutagens to accomplish stochastic genomic or plasmid mutagenesis. However, the extremely high and uncontrolled mutation rate often accumulates deleterious mutations nonspecific to the process of interest. This can result in a toxic effect to the organism, leading to suboptimal evolved levels of production.
</P><P>
A novel directed evolution system, termed feedback-regulated evolution of phenotype (FREP) (<A href="http://www.ncbi.nlm.nih.gov/pubmed/24131951">citation</A>), incorporates a dynamic mutation rate to overcome the existing problems of directed evolution by mimicking the plasticity of the mutation rate in natural evolution. This is achieved through dynamic control of a mutator element that is negatively regulated by the desired end product. In this feedback scheme, as more of the desired biomolecule is produced, the rate of mutation decreases and eventually approaches zero, allowing evolution and maintenance of a high level of production while minimizing the accumulation of toxic, nonspecific mutations.
</P><P>
Additionally, many directed evolution systems that incorporate mutator genes often rely on the strongest known mutator, MutD. However, the use of MutD alone in these systems is problematic because like all other individual mutator genes, it may only catalyze certain types of base pair substitutions reflective of its “mutational bias”, limiting the search of sequence space. Diversifying the mutator genes to eliminate mutational bias should increase the efficiency by which the sequence space can be searched. Ultimately, the addition of multiple mutators should lead to higher overall levels of evolved production. Despite a very few number of past studies to incorporate multiple mutators, these efforts have failed to show an increased efficiency of evolution. They fail to encompass a systematic screen of all mutators identified in literature, which would be required for determination of the ideal combination of mutators. In general, there is a surprising lack of studies on the use of different combinations of mutators in directed evolution.
</P><P>
The 2014 UChicago iGEM team has two main goals. The first goal is to implement and optimize FREP in E. coli by evolving mutants with elevated production of a given biomolecule, specifically evolving tyrosine production as a relatively simple pathway to demonstrate proof of concept. The mutator element will be controlled by the tyrosine-sensitive repressible promoter ParoF, suppressing the mutation rate as the tyrosine production of the cell increases. The mutator gene is linked to a fluorescent reporter gene which allows for convenient screening of desired phenotype, as bacteria with low levels of fluorescence should also have the highest levels of production due to the negative feedback loop.
The second goal is to enable a deeper exploration in the search space by combining multiple mutators with different biases to increase the initial mutation rate and speed the accumulation of beneficial mutations. We have identified previously characterized mutator genes that function in a diversity of pathways, including polymerases, proofreading enzymes, methylases and topoisomerases. We will characterize the mutation rates of these mutator genes in their active dominant negative forms individually as well as in various combinations, aiming to determine the ideal combination for directed evolution. We hope to introduce this ideal combination of mutators into the FREP system and demonstrate increased levels of evolved tyrosine production compared to the single mutator. If successful, our project should significantly improve the process of directed evolution of any biomolecule of industrial importance.
</P><H3>Mutator Constructs</H3><P>
In order to work towards our goal of finding the ideal multiple mutator gene combination for FREP, we first researched a variety of genes that when overexpressed, had been previously documented in literature to increase the mutation rate. Because of the manner in which FREP is specifically set up, dynamic control can only be achieved in a gene that increases mutation rate when overexpressed (as opposed to mutators that cause an increase in mutation rate from knockdown or full deletion for instance). Since most of the mutator genes we used had little effect from simple overexpression, we instead relied on mutation dominant negative versions of these genes that are documented to cause high increases in mutation rate when expressed in cells. These mutator genes are involved in a variety of pathways, and include polymerase subunits, DNA repair enzymes, and transposable elements. We also kept in mind the various mutational biases of the various mutators we were interested in. The table below lists all various types of mutators that we researched, as well as the point mutations needed to create the active dominant negative genes when necessary.; bolded mutators with an asterisk (*) are the actual mutators we ended up attempting to PCR from the wild-type E. Coli genome. (<A href="http://www.sciencedirect.com/science/article/pii/S0966842X98014243">citation</A>)
</P></DIV><H3>Mutators Studied</H3><TABLE class="mtable"><TBODY><TR><TH>Mutator gene</TH><TH>Mechanism (pathway)</TH><TH>Mutation rate (estimate)
If available, relative to wild type?</TH><TH>Bias</TH><TH>Point Mutations (when necessary)</TH><TH>References</TH></TR><TR><TD><STRONG>MutD*</STRONG></TD><TD>mutD (dnaQ) gene product controls
the editing capacity of pol III (a proofreading DNA exonuclease)

mutD5 mutation leads to a reduction in mismatch repair</TD><TD>10^3-10^4 errors per base pair per cell per generation
Editing efficiency:
WT 0.059
mutD 0.012
Mutation rate 10^5 higher than WT</TD><TD>Affects all bases, effect limited by temperature and richness of medium (works better in less rich media)
Mostly transversions</TD><TD>MutD51: L73W
This looks stronger than the real MutD5, which has a second suppressor mutation (below). We may want to do both though since it’s really not that much more work.
MutD52: A164V</TD><TD><A href="http://www.pnas.org/content/80/8/2189.full.pdf">[1]</A><A href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC401508/">[2]</A><A href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC285537/pdf/jbacter00575-0163.pdf">[3]</A><A href="http://download.springer.com/static/pdf/314/art%253A10.1007%252FBF02428026.pdf?auth66=1402462571_aee2a2895d3b41168f8a188700446022&amp;ext=.pdf">[4]</A></TD></TR><TR><TD>MutT</TD><TD>Nucleoside triphosphatase
Works by reducing the 8-oxo-dGTP level in the cell through hydrolysis to 8-oxo-dGMP</TD><TD>10^3-10^4 errors per base pair per cell per generation</TD><TD>AT to CG Transversions
mutT excludes A-G mismatches where the adenine resides on the template DNA strand.</TD><TD>None found</TD><TD><A href="https://www.wikigenes.org/e/gene/e/944824.html">Link</A> to various journal articles</TD></TR><TR><TD><STRONG>MutS*</STRONG></TD><TD>Primarily recognizes base mismatch sites, works with MutL, MutH to repair mismatch sites (MutS-1, at least). Seems to detect changes in adjacent base pairs
truncated form of the DNA mismatch repair protein</TD><TD>2.5 ± 2.0 × 10^–6 (?)</TD><TD>G619D</TD><TD><A href="http://onlinelibrary.wiley.com/doi/10.1046/j.1365-2958.2002.02855.x/full">[1]</A><A href="http://rstb.royalsocietypublishing.org/content/365/1544/1177.full.pdf">[2]</A><A href="http://jb.asm.org/content/176/17/5393.full.pdf">[3]</A></TD></TR><TR><TD>MutM</TD><TD>removes 8-oxoG residues from DNA by DNA glycosylase activity</TD><TD>&gt; 10^2</TD><TD>GC to TA Transversions
-combining mutM and MutY seems to increase mutation rate 25-75X compared to either mutator alone (see paper)</TD><TD>None found</TD><TD><A href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC49637/pdf/pnas01089-0369.pdf">[1]</A></TD></TR><TR><TD><STRONG>emrR*</STRONG></TD><TD>involved in multidrug resistance, transcription regulator</TD><TD>2.8 * 10^-4 //// 2.8 * 20^4</TD><TD>AT-&gt;GC preferred
transversions + frameshift (tested +1G)</TD><TD>NA (overexpression of wild-type gene sufficient)</TD><TD><A href="http://onlinelibrary.wiley.com/doi/10.1111/j.1365-2958.2004.04125.x/full">[1]</A></TD></TR><TR><TD><STRONG>Dam*</STRONG></TD><TD>involved in methylating DNA</TD><TD>6.4 * 10^-4 /// 6.4 * 10^4</TD><TD>Transversions</TD><TD>NA (overexpression of wild-type gene sufficient)</TD><TD><A href="http://onlinelibrary.wiley.com/doi/10.1111/j.1365-2958.2004.04125.x/full">[1]</A></TD></TR><TR><TD>bglY-mutant (bglY-galU region)</TD><TD>deletions, recessive, negative supercoiling?</TD><TD>2 to 10^2</TD><TD>Unknown</TD><TD>None found</TD><TD><A href="http://www.pnas.org/content/87/1/360.full.pdf">[1]</A></TD></TR><TR><TD><STRONG>MutL*</STRONG></TD><TD>Along with MutS and MutH, initiates repair cascade by recognizing base mismatches/indels of up to 4 bp (dam-directed mismatch repair)</TD><TD>2.75*10^-8 bp/generation
138 X wild type</TD><TD>Unknown</TD><TD>E32K</TD><TD><A href="http://www.pnas.org/content/109/41/E2774">[1]</A><A href="http://nar.oxfordjournals.org/content/24/13/2498.long">[2]</A><A href="http://www.jbc.org/content/275/13/9863.full">[3]</A></TD></TR><TR><TD><STRONG>DinB*</STRONG></TD><TD>Error-prone DNA polymerase DinB, mutagenesis
error-prone polymerase IV</TD><TD>NA</TD><TD>increase in frameshift and base substitution</TD><TD>F13V
SImple overexpression is also possibly mutagenic</TD><TD><A href="http://www.nature.com/nature/journal/v439/n7073/pdf/nature04318.pdf">[1]</A><A href="http://onlinelibrary.wiley.com/doi/10.1111/j.1365-2958.2004.04125.x/full">[2]</A></TD></TR><TR><TD><STRONG>Umu2D’C*</STRONG></TD><TD>DNA checkpoint effector, encodes pol V</TD><TD>NA</TD><TD>hypersensitivity to UV light damage in mutants</TD><TD>Y11A
Simple overexpression is also possibly mutagenic</TD><TD><A href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC2715726/">[1]</A></TD></TR><TR><TD><STRONG>MutH*</STRONG></TD><TD>Mismatch repair,nvolved in the same pathway as MutS and MutH</TD><TD>10^2-10^3</TD><TD>GC to AT, AT to GC, Frameshifts</TD><TD>MutH: E56A</TD><TD><A href="http://www.sci.sdsu.edu/~smaloy/MicrobialGenetics/topics/mutations/mutator-genes.html">[1]</A><A href="http://nar.oxfordjournals.org/content/early/2012/01/11/nar.gkr1298.full">[2]</A></TD></TR><TR><TD>IS transposable elements</TD><TD>More old-school transposons, generally exert effect by altering expression by insertion in promoter regions
-May also vary the number of IS elements introduced</TD><TD>~10X compared to wild type</TD><TD><A href="http://www.pubmedcentral.nih.gov/articlerender.fcgi?artid=3457772&amp;tool=pmcentrez&amp;rendertype=abstract">[1]</A></TD></TR><TR><TD>Mini transposons</TD><TD>Mini-Mu, Mini D-108, mini-Tn5, mini-Tn7 Could test 1-2 of these.</TD><TD>Variable</TD><TD>Can induce larger-scale chromosome rearrangements</TD><TD><A href="http://www.pubmedcentral.nih.gov/articlerender.fcgi?artid=3457772&amp;tool=pmcentrez&amp;rendertype=abstract">[1]</A></TD></TR><TR><TD><STRONG>DnaE*</STRONG></TD><TD>alpha subunit of DNA polymerase 3</TD><TD>E612K</TD><TD><A href="https://www.jstage.jst.go.jp/article/ggs/82/4/82_4_273/_pdf">[1]</A></TD></TR><TR><TD><STRONG>ParC*</STRONG></TD><TD>subunit of topIV</TD><TD>Y120H</TD><TD><A href="http://onlinelibrary.wiley.com/doi/10.1046/j.1365-2443.1996.d01-226.x/pdf">[1]</A></TD></TR><TR><TD><STRONG>MutY*</STRONG></TD><TD>G-A mismatch DNA repair</TD><TD>&gt; 10^2</TD><TD>GC to TA Transversions</TD><TD>V45A</TD><TD><A href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC2714593/pdf/1471-2091-10-19.pdf">[1]</A></TD></TR></TBODY></TABLE><P>Though we ran out of time and were never able to test actual combinations of mutators (we were only able to test individual mutators), it would make more sense to combine mutators with different biases (one could combine a mutator with a G to C bias with one with an A to T bias for example) as opposed to mutators with similar biases, which we would predict to allow greater coverage of the sequence space.
</P><H3>Cloning Strategy</H3><P>To engineer the mutators into the final FREP system, we divided up the project into several parts. First, the individual mutators were cloned into an IPTG-inducible promoter construct, and mutation rates of individual mutators were quantified in this construct, simultaneously allowing verification of their mutagenic activity. In a separate construct, the ParoF tyrosine-repressible promoter that normally controls the AroF-containing operon was cloned into the plasmid backbone. Finally, a mutated version of the TyrR transcription factor that regulates ParoF was to be created on a third plasmid; the mutations would alter and increase the range of tyrosine concentrations that TyrR was sensitive to.
</P><P>
All initial cloning of the mutators from the wild-type E. Coli genome and site-directed mutagenesis when necessary was done in the IPTG-inducible promoter construct. This design facilitated creating combinations of mutators in the ParoF construct, because we could design universal primers that allowed us to easily PCR out the mutagenized mutators once they had been created in the IPTG-inducible promoter construct, followed by sequential cloning of mutators in the desired order on the ParoF-containing construct.
</P><P>
To read about the specific details and functional units of each of the general constructs we used, click
<A href="/Team:UChicago/Parts">here</A>.
</P><P>
To read about the calculations and assays used to quantify mutation rate and verify the repressibility of the ParoF construct, click
<A href="/Team:UChicago/Modeling">here</A>.
</P><H3>Results</H3><P>We were able to isolate the mutator genes EmrR, Dam, mutD, mutL, DnaE, mutS, mutY, and mutH from E. coli genomic DNA and place them into a construct containing GFP under the control of an IPTG-inducible promoter. We then activated mutS,mutY, and mutH into their mutation-causing alleles via Quikchange mutagenesis (wild type Dam and EmrR can raise the mutation rate of E. coli when simply overexpressed)</P><P>
We were then able to quantify the mutation rate of XL1Blue (Agilent) bacteria containing the IPTG-&gt;GFP+EmrR plasmid and compare it to that of wild type Xl1Blue. Using a modified version of the Luria-Delbruck fluctuation analysis experimental setup (quantifing mutation rate through estimating number of mutational events through frequency of rifampicin-resistance) and online FALCOR tool (http://www.keshavsingh.org/protocols/FALCOR.html), we were able to estimate the mutation rate per cell per culture for both strains.</P><TABLE class="mtable"><TBODY><TR><TH>mact=(mobs*(1-z))/(z(ln(z))</TH></TR><TR><TH>rif-resistant colonies per parallel culture</TH><TH>avg#cells per culture</TH><TH>mutations(observed)</TH><TH>mutations (actual)</TH><TH>Mutation rate mutations/cell per culture)</TH></TR><TR><TD>wt</TD><TD>1,3,1,4,1,4,1,1,1,3,6,4,1,1,0,0</TD><TD>1.98E+09</TD><TD>1.217</TD><TD>5.617</TD><TD>2.84E-09</TD></TR><TR><TD>EmrR</TD><TD>1,2,3,1,1,1,1,0,0,0,0,0,0,0,0,0</TD><TD>5.95E+08</TD><TD>0.478</TD><TD>1.868</TD><TD>3.14E-09</TD></TR></TBODY></TABLE><P>The FALCOR tool uses a maximum likelihood function to generate an estimate for number of mutational events from the frequency of rif-resisitant colonies in parallel cultures. We had to correct the estimate of mutations m(observed) to m(actual) (using mact=(mobs*(1-z))/(z(ln(z)) with z being fraction of cultures plated) because we only plated 10% of each parallel culture onto the rifampicin selective media. The mutation rate is gained by dividing the actual number of mutations that occurred by the average number of cells per culture. Because the cell densities were different, this data is not wholly reliable as bacteria may have quorum sensing abilities that may impact endogenous mutation rate (http://www.nature.com/ncomms/2014/140429/ncomms4742/full/ncomms4742.html).
</P><P>We successfully created FREP-ready ARM constructs containing the mutagenizing proteins EmrR, Dam and mutH(E56A) and the reporter mCherry-LVA under the control of the tyrosine-sensitive promoter paroF und although we were not able to implement FREP due to time constraints.</P></DIV></DIV><DIV class="printfooter">
Retrieved from &quot;<A href="http://2014.igem.org/Team:UChicago/Project">http://2014.igem.org/Team:UChicago/Project</A>&quot;</DIV></DIV></DIV><DIV id="footer-box" class="noprint"><DIV id="footer"><UL id="f-list"><LI id="t-recentchanges"><A href="/Special:RecentChanges" title="Recent changes">Recent changes</A></LI><LI id="t-whatlinkshere"><A href="/Special:WhatLinksHere/Team:UChicago/Project" title="List of all wiki pages that link here [j]" accesskey="j">What links here</A></LI><LI id="t-recentchangeslinked"><A href="/Special:RecentChangesLinked/Team:UChicago/Project" title="Recent changes in pages linked from this page [k]" accesskey="k">Related changes</A></LI><LI id="t-specialpages"><A href="/Special:SpecialPages" title="List of all special pages [q]" accesskey="q">Special pages</A></LI><LI><A href="/Special:Preferences">My preferences</A></LI></UL></DIV><DIV id="footer"><UL id="f-list"><LI id="t-print"><A href="/wiki/index.php?title=Team:UChicago/Project&amp;printable=yes" title="Printable version of this page [p]" accesskey="p">Printable version</A></LI><LI id="t-permalink"><A href="/wiki/index.php?title=Team:UChicago/Project&amp;oldid=401624" title="Permanent link to this revision of the page">Permanent link</A></LI><LI id="privacy"><A href="/2014.igem.org:Privacy_policy" title="2014.igem.org:Privacy policy">Privacy policy</A></LI><LI id="disclaimer"><A href="/2014.igem.org:General_disclaimer" title="2014.igem.org:General disclaimer">Disclaimers</A></LI></UL></DIV></DIV></DIV></DIV></BODY></HTML>