"
Page
Discussion
View source
History
teams
Log in
 
Team:UChicago
From 2014.igem.org
Home
Team
Official Team Profile
Project
Parts
Modeling
Notebook
Outreach
Safety
Attributions
UChicago iGEM 2014
2014: A Sequence Space Odyssey: Diversifying Mutators to Optimize Directed Evolution
Project Abstract
Directed evolution is a fundamental technique in bioengineering organisms to express proteins of novel function and to mass produce industrially relevant biomolecules. Generally, directed evolution simulates an algorithmic process in which the entire sequence space is searched for an optimal genotype by increasing the natural mutation rate to artificially speed the process of selective evolution. Current methods of in vivo directed evolution typically rely on physical or chemical mutagens to accomplish stochastic genomic or plasmid mutagenesis. However, the extremely high and uncontrolled mutation rate often accumulates deleterious mutations nonspecific to the process of interest. This can result in a toxic effect to the organism, leading to suboptimal evolved levels of production.
A novel directed evolution system, termed feedback-regulated evolution of phenotype (FREP), incorporates a dynamic mutation rate to overcome the existing problems of directed evolution by mimicking the plasticity of the mutation rate in natural evolution. This is achieved through dynamic control of a mutator element that is negatively regulated by the desired end product. In this feedback scheme, as more of the desired biomolecule is produced, the rate of mutation decreases and eventually approaches zero, allowing evolution and maintenance of a high level of production while minimizing the accumulation of toxic, nonspecific mutations.
Additionally, many directed evolution systems that incorporate mutator genes often rely on the strongest known mutator, MutD. However, the use of MutD alone in these systems is problematic because like all other individual mutator genes, it may only catalyze certain types of base pair substitutions reflective of its “mutational bias”, limiting the search of sequence space. Diversifying the mutator genes to eliminate mutational bias should increase the efficiency by which the sequence space can be searched. Ultimately, the addition of multiple mutators should lead to higher overall levels of evolved production. Despite a very few number of past studies to incorporate multiple mutators, these efforts have failed to show an increased efficiency of evolution. They fail to encompass a systematic screen of all mutators identified in literature, which would be required for determination of the ideal combination of mutators. In general, there is a surprising lack of studies on the use of different combinations of mutators in directed evolution.
The 2014 UChicago iGEM team has two main goals. The first goal is to implement and optimize FREP in E. coli by evolving mutants with elevated production of a given biomolecule, specifically evolving tyrosine production as a relatively simple pathway to demonstrate proof of concept. The mutator element will be controlled by the tyrosine-sensitive repressible promoter ParoF, suppressing the mutation rate as the tyrosine production of the cell increases. The mutator gene is linked to a fluorescent reporter gene which allows for convenient screening of desired phenotype, as bacteria with low levels of fluorescence should also have the highest levels of production due to the negative feedback loop.
The second goal is to enable a deeper exploration in the search space by combining multiple mutators with different biases to increase the initial mutation rate and speed the accumulation of beneficial mutations. We have identified previously characterized mutator genes that function in a diversity of pathways, including polymerases, proofreading enzymes, methylases and topoisomerases. We will characterize the mutation rates of these mutator genes in their active dominant negative forms individually as well as in various combinations, aiming to determine the ideal combination for directed evolution. We hope to introduce this ideal combination of mutators into the FREP system and demonstrate increased levels of evolved tyrosine production compared to the single mutator. If successful, our project should significantly improve the process of directed evolution of any biomolecule of industrial importance.
Project Summary
What are mutators genes?
Mutators genes are defective copies of genes that can induce an elevated spontaneous mutation rate for all other genes in the organism. They increases genomic instability and hence are often employed as a tool for mutagenesis.
How do mutator genes work?
Mutator genes often confers defects in the DNA repair or replication pathways. For example, defects in polymerases, proofreading enzymes or recombinases can lead to inaccurate DNA replication, leading to significantly increased spontaneous mutation rate. Each mutator gene product has its mutational bias due to the inherent differences in function and nature of mutation. The mutT gene product, for instance, is a G-A mismatch repair mutator. The major type of mutation caused, hence its mutational bias, is AT to CG transversions.
Our Project
Keasling et. al has used the most potent mutator gene, mutD, in implementing FREP. However, the use of only one mutator can limit the search space due to low initial mutation rates and mutational biases. Our team has therefore chosen to investigate the following 11 well-characterized dominant negative mutators:
Mutator
Gene Function/Gene Product
Mutational Bias/Preferred Mutations
Dam
DNA methyltransferase that methylates A in -GATC-
Transversions
DinB
Error-prone DNA polymerase (pol IV)
Frameshift and base substitution
DnaE
Alpha subunit of DNA polymerase III holoenzyme
Transversions
EmrR
Transcriptional repressor of microcin biosynthesis genes
AT->GC transversions, frameshift, base substitutions
MutD
Epsilon subunit of DNA polymerase III
Affects all bases through Transitions, Transversions, Frameshifts
MutH
Part of the multimeric methyl-directed DNA mismatch repair protein (MutSHL)
Transitions, Frameshifts
MutL
Part of the multimeric methyl-directed DNA mismatch repair protein (MutSHL)
Transitions, Frameshifts
MutS
Part of the multimeric methyl-directed DNA mismatch repair protein (MutSHL)
Transitions, Frameshifts
MutT
Nucleoside triphosphatase
AT to CG Transversions
MutY
Adenine DNA glycosylase
GC to TA Transversions
ParC
Alpha subunit of DNA topoisomerase IV
Nicks DNA
UmuD’C
Subunits of DNA polumerase IV
Hypersensitivity to UV light damage
To further increase the search space, we have planned to use the following combinations of mutator genes:
Combinations
Reasoning
MutS + MutH + MutL
Belong to same multimeric protein
MutS + MutY + MutH
Mutants have been shown to increase mismatches between 8oxo-G and A
MutD + MutS + MutT
The E. coli strain XL1-Red contains mutations in these genes
MutY + EmrR
Different biases
Dam + MutH
Different biases
Dam + MutS
Different biases
MutD + MutH
Different biases
In our project, we only focused on point mutators to verify our hypothesis that a combination of mutators with different biases can increase the initial mutation rate and facilitate the search for the global optimum. If our hypothesis is proven correct, we will move onto more complex and larger-scale mutational mechanisms such as transposition and recombination to investigate their effects on the diversification of the search space.
Retrieved from " http://2014.igem.org/Team:UChicago "
Recent changes
What links here
Related changes
Special pages
My preferences
Printable version
Permanent link
Privacy policy
Disclaimers
