<HTML xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en" dir="ltr">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki  ltr ns-0 ns-subject page-Team_Peking_University_Project"><DIV id="globalWrapper"><DIV id="top-section"><DIV id="p-logo"><A href="/Main_Page" title="Main Page">&quot;
	    </A></DIV><DIV id="menubar" class="left-menu"><UL><LI class="selected"><A href="/Team:Peking_University/Project">Page               </A></LI><LI><A href="/Talk:Team:Peking_University/Project">Discussion               </A></LI><LI><A href="/wiki/index.php?title=Team:Peking_University/Project&amp;action=edit">View source               </A></LI><LI><A href="/wiki/index.php?title=Team:Peking_University/Project&amp;action=history">History               </A></LI><LI style="color:#808080;cursor:default">teams</LI></UL></DIV><DIV class="right-menu" id="menubar"><UL><LI id="pt-login"><A href="/wiki/index.php?title=Special:UserLogin&amp;returnto=Team:Peking_University/Project" title="You are encouraged to log in; however, it is not mandatory [o]" accesskey="o">Log in</A></LI></UL></DIV><DIV id="search-controls"><FORM action="/Special:Search" id="searchform"> </FORM></DIV></DIV><DIV id="content"><H1 class="firstHeading">Team:Peking University/Project</H1><DIV id="bodyContent"><H3 id="siteSub">From 2008.igem.org</H3><TABLE style="color:#ba0145;background-color:#dda303;" cellpadding="3" cellspacing="1" border="1" width="95%" align="center"><TBODY><TR><TH align="center"><A href="/Team:Peking_University" title="Team:Peking University">Home</A></TH><TH align="center"><A href="/Team:Peking_University/Team" title="Team:Peking University/Team">The Team</A></TH><TH align="center"><STRONG class="selflink">The Project</STRONG></TH><TH align="center"><A href="/Team:Peking_University/Parts" title="Team:Peking University/Parts">Parts</A></TH><TH align="center"><A href="/Team:Peking_University/Modeling" title="Team:Peking University/Modeling">Modeling</A></TH><TH align="center"><A href="/Team:Peking_University/Misc%26Fun" title="Team:Peking University/Misc&amp;Fun">Misc&amp;Fun</A></TH><TH align="center"><A href="http://www.openwetware.org/wiki/IGEM:Peking_University/2008" class="external text" rel="nofollow">Our OWW</A></TH><TH align="center"><A href="http://www.openwetware.org/wiki/IGEM:Peking_University/2008/Notebook" class="external text" rel="nofollow">Notebook</A></TH></TR></TBODY></TABLE><TABLE id="toc" class="toc"><TBODY><TR><TD><DIV id="toctitle"><H2>Contents</H2></DIV><UL><LI class="toclevel-1 tocsection-1"><A href="#Project_Abstract"><SPAN class="tocnumber">1</SPAN><SPAN class="toctext">Project Abstract</SPAN></A></LI><LI class="toclevel-1 tocsection-2"><A href="#Project_Background"><SPAN class="tocnumber">2</SPAN><SPAN class="toctext">Project Background</SPAN></A></LI><LI class="toclevel-1 tocsection-3"><A href="#Project_Details"><SPAN class="tocnumber">3</SPAN><SPAN class="toctext">Project Details</SPAN></A></LI><LI class="toclevel-2 tocsection-4"><A href="#Project_Proposal"><SPAN class="tocnumber">3.1</SPAN><SPAN class="toctext">Project Proposal</SPAN></A></LI><LI class="toclevel-2 tocsection-5"><A href="#The_Experiments"><SPAN class="tocnumber">3.2</SPAN><SPAN class="toctext">The Experiments</SPAN></A></LI><LI class="toclevel-3 tocsection-6"><A href="#Plasmids_Construction"><SPAN class="tocnumber">3.2.1</SPAN><SPAN class="toctext">Plasmids Construction</SPAN></A></LI><LI class="toclevel-3 tocsection-7"><A href="#Yeast_Strains_Preparation"><SPAN class="tocnumber">3.2.2</SPAN><SPAN class="toctext">Yeast Strains Preparation</SPAN></A></LI></UL><LI class="toclevel-1 tocsection-8"><A href="#Schematic_Overview_of_Results"><SPAN class="tocnumber">4</SPAN><SPAN class="toctext">Schematic Overview of Results</SPAN></A></LI><LI class="toclevel-1 tocsection-9"><A href="#The_Results"><SPAN class="tocnumber">5</SPAN><SPAN class="toctext">The Results</SPAN></A></LI></TD></TR></TBODY></TABLE><H2><SPAN class="mw-headline" id="Project_Abstract"><B>Project Abstract</B></SPAN></H2><P style="padding: 20px; text-align: justify"><B>A Genetic Circuit for Directed Evolution <I>in vivo</I></B>
Directed evolution is a powerful tool for answering scientific questions or constructing novel biological systems. Here we present a simple genetic circuit for in vivo directed evolution which comprises minimal elements for random mutation and artificial selection. We engineer yeast to generate the DNA mutator hAID, an essential protein in adaptive immunity, and target it specifically to a gene of interest. The target gene will be mutated and consequently promptly evolves. By linking the expression of hAID repressor LacI and favorite gene functionality, the mutation rate inversely correlates between the functionality of the desired gene and hAID. This circuit may be adopted for in vivo evolution in eukaryotic system on genetically encoded targets. It has a variety of potential applications in academic and industrial contexts, theoretically most inter-molecular interaction that involves proteins and RNAs. 
<B>Genetic Circuit</B></P><H2><SPAN class="mw-headline" id="Project_Background"><B>Project Background</B></SPAN></H2><P>Over millions of years, evolution has created finely tuned networks of proteins capable of producing incredible outputs because of their optimized interactions with each other.  The early efforts of synthetic biologists to recreate these networks artificially has given us a new respect for just how well-optimized natural systems really are; our efforts have in general appeared clumsy next to natural examples.
</P><P>However, natural evolution is a very slow process, governed by subtle selective pressures and long generation times.  Therefore, scientists have developed several strategies for directed evolution in the lab, breaking it down into mutation (various PCR based techniques, shuffling, mutator strains) and selection (display technologies, etc.)  These methods do cut down on time, and there have been some notable successes by researches using in-vitro directed evolution.  Nevertheless, running the necessary high-throughput screens and mutagenesis protocols requires a great deal of manpower, expertise, and time.
</P><P>It would seem that nature has beaten us to the punch once again, however, as a natural mechanism for rapid evolution already exists: the immune system's adaptive immunity.  It is well known that B-cells are capable of producing a large pool of diverse antibodies upon antigenic stimuli, a fact which underlies our own ability to survive infection.  Recent work has revealed that an enzyme called AID (activation induced cytidine deaminase) is the workhorse behind this immune capacity.  AID facilitates three processes crucial to antibody diversity: somatic hypermutation (SHM), class switch recombination (CSR), and gene conversion. 
</P><P>Briefly, AID converts cytidine to uracil by oxidizing the amino group to a carbonyl, resulting in a Watson-Crick mismatch.  DNA repair pathways then are activated the remove the mismatch, resulting in a changed coding sequence of the hypervariable region that AID is targeted to.  Because of its mutagenic properties, AID must be carefully targeted to avoid B-cell lymphocytoma. 
</P><P>AID has been shown to be functional in other cells besides human B-cells.  In 2005, Youri Pavlov's group expressed human AID in yeast, and found that AID maintained a high mutation rate even in this heterologous host.
</P><TABLE>||</TABLE><P>(Neuberger, M.S., et al., <I>Somatic hypermutation at A.T pairs: polymerase error versus dUTP incorporation.</I> Nat Rev Immunol, 2005. <B>5</B>(2): p. 171-8.<A href="http://www.ncbi.nlm.nih.gov/pubmed/15688043?ordinalpos=1&amp;itool=EntrezSystem2.PEntrez.Pubmed.Pubmed_ResultsPanel.Pubmed_RVMedline" class="external text" rel="nofollow">PubMed</A>)
</P><P>Inspired by this work, we hope to engineer a genetic circuit in yeast that will govern the directed evolution process from mutation to selection.  Compared to human B-cells, yeast is cheap and reproduces incredibly fast.  Moreover, yeast is a very well characterized eukaryotic model with many of the post-translational abilities found in human cells, allowing us to do in vivo analysis of protein function without having to extract and purify.  Our system allows for a wide variety of substrates, essentially any intermolecular interaction that involves proteins. 
</P><P><A href="http://www.openwetware.org/wiki/IGEM:Peking_University/2008/Related_Papers" class="external text" rel="nofollow">Learn More...</A></P><H2><SPAN class="mw-headline" id="Project_Details"><B>Project Details</B></SPAN></H2><H3><SPAN class="mw-headline" id="Project_Proposal"> Project Proposal </SPAN></H3><P>
Evolution could be dissociated into two parallel processes: random mutation, and natural selection. Whilst its role in life history is well understood and accepted, evolution is also evident across individual life process. In particular, adaptive immunity system adopts evolution strategy to produce antibodies for novel antigens. Artificial evolution method could be a powerful tool for answering scientific questions or engineering novel biological systems. Via systems biology approach, here we present a simple genetic circuit consisting functional elements for random mutation and artificial selection. This circuit may perform in vivo evolution on virtually any genetically encoded targets, with potential applications in academic and industrial contexts. 

The gene encoding the core element in adaptive immunity, hAID, is fused with the LexA-DBD domain with flexible linker. The fusion gene hAID-LexA is inserted into a yeast ESC expression cassette at 3’ of tandem LacO elements. The target gene (your favorite gene, yfg) is inserted into a yeast ESC expression cassette, with its 3’UTR containing tandem LexO elements. An inducible promoter response to yfg activity drives expression of His3, LacZ and LacI in cistrone spanned by IRES. These three plasmids together form a genetic circuit for in vivo evolution. 

As a proof-of-concept experiment, here we present one very simple example in yeast one hybrid: a mutated Gal4 gene is inserted in the target cassette. The selector cassette contains UAS to drive expression of His3, LacZ and LacI. All three plasmids are transfected or knock-in-ed into gal4-, his3- yeast strain with proper selection tags. Initially, we culture the yeast in complete YPD medium. Defect Gal4 product cannot bind to UAS, hence hAID-LexA is constitutively expressed, recruited to LexO sites of the target, and mutates the Gal4 conding sequence. Once Gal4 mutation is reversed, it binds to UAS to drive LacI expression, which represses hAID-LexA. We plate the yeast into Trp-, His-, Leu-, 3AT+, XGal+ plate and select for the large, blue colonies. Sequencing the colonies then gives us the activated Gal4 gene sequences. 

The power of this system could be best revealed in the above case: since selection is not required for initiation and attenuation of mutagenesis, there is virtually no need to perform selective pressure titration. 

The system could be readily adopted to in vivo evolution of any kind of protein-protein and protein-nucleic acid interaction: for this we simply adopt a yeast two hybrid-like approach, evolving a functional protein linked to Gal4 AD which could bind to the &quot;bait&quot; protein linked to Gal4 DBD. We could also use the yeast three hybrid system to study RNA-protein interaction, or use yeast one hybrid to evolve specific protein that binds to given promotor. 

Yeast shows several advantages in studying eukaryotic process, for it folds and modifies the eukaryotic protein accurately, whilst bacteria does not. Another advantage of yeast is that cell wall blocks intracellular communication between transmembrane proteins, and the selection happens completely within the cell, therefore blocking possible false positives. We plan to screen for extracellular activator of certain eukaryotic transmembrane protein, for example, novel peptidergic ligand for GPCR and antibody-antigen interaction.

We have been working on further improvements of the mutation system, by improving the enzyme to archieve directed, &quot;hotspot-less&quot;, and evenly distributed mutagenesis, by utilizing the overwhelming power of yeast genetics, and by harnessing novel protein chemistry.

//Last modified: ZY 20080704
</P><H3><SPAN class="mw-headline" id="The_Experiments"> The Experiments </SPAN></H3><H4><SPAN class="mw-headline" id="Plasmids_Construction">Plasmids Construction</SPAN></H4><P>To construct the genetic circuits above, we have used the pGREG series of vectors which were hosted in budding yeast <I>Saccharomyces cerevisiae</I> AH109. pGREG503, the expression plasmid with HIS3 auxotroph resistance was used to construct the UAS-lacI component. LacI was under the galactose inducible promoter GAL1. The promoters of pGREG504 and pGREG505 were replaced by pADH-lac and pACT respectively. pADH-lac is pADH based promoter which is sensitive to lacI, genes under this promoter will be inhibited by lacI. pACT is a constitutively expressing promoter derived from the promoter of actin in budding yeast. The construct hAID-linkers-lexA was under the pADH-lac promoter and the GAL4-lexO construct was engineered under the pACT promoter. Notably, our hAID sequence was yeast codon optimized kindly offered by Prof. Youri I Pavlov. A set of <A href="/wiki/images/d/da/Linkers.pdf" class="internal" title="Linkers.pdf">12 linkers</A> were synthesized to test the ability to maintain the activity of hAID and lexA DNA binding domain. Moreover, in the genome of the yeast strain AH109, it has GAL1-HIS3 and GAL1-lacZ constructs. PGREG504-hAID-linkers-lexA DBD was constructed via the following <A href="https://static.igem.org/mediawiki/2008/8/87/Construct.gif" class="external text" rel="nofollow">Figure</A>.
</P><P>To construct plasmid on pGREG503, we have used the versatile PCR-based homologous recombination system: the PCR product of lacI was directly co-transformed with pGREG503 and it was automatically recombined to the site flanking the 3’ of the promoter. Other constructs were accomplished with standard protocols.        
//Written by Zhou Zhou
</P><H4><SPAN class="mw-headline" id="Yeast_Strains_Preparation">Yeast Strains Preparation</SPAN></H4><P>Among more than 50 plasmids we have constructed, we have chosen the above 16 groups for the following yeast screening task. For the AID construct which is on pGREG504, we have pGREG-lacS (note: lacS is one type of pADH-lac promoters) as a negative control and those with lac1, another pADH-lac promoter. A set of four linkers were tested in order to mostly maintain the original activity of hAID. For the GAL4 constructs, many spot mutations were triggered in order to disrupt function of wildtype GAL4 but only a spot mutation at 77-position (C77T) was selected to carry out the further yeast screening process. GAL4(C77T) is not in the hot spot as predicted previously (See Modeling Part), high rate and successful muatation of this mutant to functional one will to a great extent suggest that in our system the hAID can generate novel, gain-of-function mutations in the targeted gene. Wildtype <I>gal4</I> in pGREG505 and plasmids without <I>gal4</I> gene were also used as positive and negative controls. 
</P><TABLE>||</TABLE><P>As the above table listed out, 16 strains of yeast were constructed via co-transformation with AID construct and GAL4 construct.
</P><H2><SPAN class="mw-headline" id="Schematic_Overview_of_Results"> Schematic Overview of Results</SPAN></H2><P><B>Crystal structure of APO2 which is supposed to have similar structure of hAID.</B><B>Schematic structure of hAID, showing the N-terminal as catalytic active domain and C-terminal as protein-protein interaction domain. We choosed to fuse the DNA binding domain to hAID C-terminal based on this structure-activity relationship.</B><I><B> Jayanta Chaudhuri &amp; Frederick W. Alt, Nature Reviews Immunology 4, 541-552 (July 2004)</B></I></P><P><B>In mammalian immune cells, hAID acts in cooperation with many trans-acting factors, likely to be DNA binding protein partners in the active SHM protein complex. Our approach harnesses an alternative strategy by fusion of DNA binding domain to directly target hAID onto a specific DNA loci. </B><I><B>Valerie H. Odegard &amp; David G. Schatz, Nature Reviews Immunology 6, 573-583 (August 2006)</B></I></P><P><B>The dysfunctional Gal4(C77T) is under the control of pAct. The fusion protein hAID-lexA DBD specifically binds to the lexO region flanking the 3' of Gal4(C77T). With the facilitation of the linker, hAID is able to target to Gal4(C77T). </B><B>The Gal4(C77T) has lost the of DNA binding activity, so it failed to induce the expression of lacI; when Gal4(C77T) is mutated until it has restored its original function, it activates the expression of lacI. </B><B>The expression of hAID is regulated by LacI. When lacI is expressed, expression of hAID is attenuated. </B><B>The triad relationship between the three components of our genetic circuit.</B><B>For yeast screening process, two different reporters are used. His3 is used to report the viability of yeast strain. Functional Gal4 is able to induce the expression of His3 which enables the yeast strain to survive on the His- medium. Another reporter gene lacZ will enable us to use blue and white screening based on X-Gal plate.</B></P><H2><SPAN class="mw-headline" id="The_Results"> The Results </SPAN></H2><P>
As a proof-of-concept experiment, we firstly tested whether overexpression hAID-LexA DBD fusion protein would enhance targetted mutagenesis efficiency without LacI repressor. The basic principle for our yeast screening is to test the viability of the 16 yeast strains. If our dysfunctional GAL4 is mutated to functional GAL4, this will lead to the further activation of HIS3 gene under the control of GAL1 promoter in yeast strain AH109. We choosed to use the Gal4(C77T) mutant because it is a null mutant but not of a hotspot of traditional hAID SHM, which means additional gain-of-function novel mutation is needed to revert the phenotype.
</P><P><B>Detailed view of P26 region of Gal4 in a Gal4-DNA complex structure. P26 is the outmost amino acid in the turning region of Gal4 helix DNA binding domain, and is essential for maintaining the structural motif needed to form coordination bond with Zn2+ ion (the yellow particle). Further more, the five-element ring structure of this proline has critical role to stablize the structure of Zinc finger domain. Replacing the proline to leucine drastically alters the structure and nullifies DNA binding function of Gal4. This structure-based theoretical prediction is consistent with our experiment result -- yeast strains containing Gal4(C77T) but without hAID or its fusion protein failed to grow on histidine free medium. </B></P><P>All strains were inoculated into liquid medium in which the concentration of histidine gradually reduced during the week’s long screening. Then, cells were plated on agar plates containing NO His to test their viability (<A href="/wiki/images/c/c1/Yeast_Screening_Model_Plate.JPG" class="internal" title="Yeast Screening Model Plate.JPG">See figure</A>). The culturing of the yeast and the plating used the following protocols. 
</P><P>
On the first day of screening, we have observed that the GAL4(C77T) is dysfunctional. AH109 cells transfected with UAS-Gal4(C77T) plated on the histidine free medium grew aberrantly and could not form dense lawn of cells (Fig.1A), whilst the positive control groups containing UAS-Gal4(wt), grew rapidly on histidine free medium(Fig.1B). 
</P><P>On the first day of screening, we did not observe our experiment groups, those strains containing the deficiency GAL4 construct and the hAID gene, survive on the histidine free plate. 
</P><P>On the second day of screening, we had luckily found that our strains with lacS-hAID and mutant GAL4 could grow on the selective medium(Fig.2).  
</P><P>Then we want to further test the activity of the linkers between hAID and lexA DNA Binding Domain. A set of four linkers were tested using the same methods on the second day of screening. Yeast strains with linker 4, 6, 9 (See Plasmids Construction Section) show high viability than those with linker 8. This result indicates that linker length is critical in determining the hAID-linker-LexA DBD protein function, and hence implicates that DNA binding mediated by LexA DBD is crucial -- otherwise linker length would not play a role. 
</P><P>We continued our screening for the third day, and the experiment group as well as the negative control group both grew well on selective medium which suggests that background mutation also contributed to the evolution process. 
</P><P>Some conclusions can be made through this round of screening. Under similar selection pressure, yeast strains with hAID-lexA DBD fusion protein evolve one day earlier than those without the mutator, suggesting a higher mutation rate in general. Specifically, the mutation rate to produce a revertant Gal4 phenotype is dependent on the linker length in between hAID and LexA DBD. This further suggests the mutagenesis efficiency of hAID-LexA DBD is dependent on LexA-mediated DNA binding. Our genetic circuits thus can increase the mutation rate of a target gene to a significant level as shown in the example of successful phenotypic reversion of GAL4(C77T). Though we have not yet been able to sequence all the mutants we obtained in the first round of screen, since C77T mutant is not within a traditional hAID SHM hotspot, it is unlikely that the mutation is a revertant. Hence, this finding suggests that the fusion protein hAID-LexA DBD enables novel gain-of-function mutagenesis in the target gene.
</P><DIV class="printfooter">
Retrieved from &quot;<A href="http://2008.igem.org/Team:Peking_University/Project">http://2008.igem.org/Team:Peking_University/Project</A>&quot;</DIV></DIV></DIV><DIV id="footer-box"><DIV id="footer"><UL id="f-list"><LI id="t-recentchanges"><A href="/Special:RecentChanges" title="Recent changes">Recent changes</A></LI><LI id="t-whatlinkshere"><A href="/Special:WhatLinksHere/Team:Peking_University/Project" title="List of all wiki pages that link here [j]" accesskey="j">What links here</A></LI><LI id="t-recentchangeslinked"><A href="/Special:RecentChangesLinked/Team:Peking_University/Project" title="Recent changes in pages linked from this page [k]" accesskey="k">Related changes</A></LI><LI id="t-specialpages"><A href="/Special:SpecialPages" title="List of all special pages [q]" accesskey="q">Special pages</A></LI><LI><A href="/Special:Preferences">My preferences</A></LI></UL></DIV><DIV id="footer"><UL id="f-list"><LI id="t-print"><A href="/wiki/index.php?title=Team:Peking_University/Project&amp;printable=yes" title="Printable version of this page [p]" accesskey="p">Printable version</A></LI><LI id="t-permalink"><A href="/wiki/index.php?title=Team:Peking_University/Project&amp;oldid=105288" title="Permanent link to this revision of the page">Permanent link</A></LI><LI id="privacy"><A href="/2008.igem.org:Privacy_policy" title="2008.igem.org:Privacy policy">Privacy policy</A></LI><LI id="disclaimer"><A href="/2008.igem.org:General_disclaimer" title="2008.igem.org:General disclaimer">Disclaimers</A></LI></UL></DIV></DIV></DIV></BODY></HTML>