<HTML xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en" dir="ltr">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki  ltr ns-0 ns-subject page-Team_NTNU_Trondheim_Notebook_July"><DIV id="globalWrapper"><DIV id="top-section"><DIV id="p-logo"><A href="/Main_Page" title="Main Page">&quot;
	    </A></DIV><DIV id="menubar" class="left-menu noprint"><UL><LI class="selected"><A href="/Team:NTNU_Trondheim/Notebook/July">Page               </A></LI><LI class="new"><A href="/wiki/index.php?title=Talk:Team:NTNU_Trondheim/Notebook/July&amp;action=edit&amp;redlink=1">Discussion               </A></LI><LI><A href="/wiki/index.php?title=Team:NTNU_Trondheim/Notebook/July&amp;action=edit">View source               </A></LI><LI><A href="/wiki/index.php?title=Team:NTNU_Trondheim/Notebook/July&amp;action=history">History               </A></LI><LI style="color:#808080;cursor:default">teams</LI></UL></DIV><DIV class="right-menu noprint" id="menubar"><UL><LI id="pt-login"><A href="/wiki/index.php?title=Special:UserLogin&amp;returnto=Team:NTNU_Trondheim/Notebook/July" title="You are encouraged to log in; however, it is not mandatory [o]" accesskey="o">Log in</A></LI></UL></DIV><DIV id="search-controls" class="noprint"><FORM action="/Special:Search" id="searchform">Â </FORM></DIV></DIV><DIV id="content"><H1 class="firstHeading">Team:NTNU Trondheim/Notebook/July</H1><DIV id="bodyContent"><H3 id="siteSub" class="noprint">From 2012.igem.org</H3></DIV></DIV></DIV><DIV class="header"><DIV class="container"><DIV class="heading visible-tablet visible-desktop"><A href="https://2012.igem.org/Team:NTNU_Trondheim">Search and Destroy</A></DIV><DIV class="heading heading-small visible-phone"><A href="https://2012.igem.org/Team:NTNU_Trondheim">Search and Destroy</A></DIV><DIV class="sub-heading visible-tablet visible-desktop" id="sub-heading1"><SPAN class="sub-heading-small">NTNU IS</SPAN><SPAN style="font-size: 30pt; font-style: italic;">B.A.C.K.</SPAN></DIV><DIV class="sub-heading visible-desktop" id="sub-heading2">
            B<SPAN class="sub-heading-small">acterial</SPAN> A<SPAN class="sub-heading-small">nti</SPAN>-C<SPAN class="sub-heading-small">ancer</SPAN>-K<SPAN class="sub-heading-small">amikaze</SPAN></DIV></DIV></DIV><DIV class="navbar navbar-below-header"><DIV class="navbar-inner"><DIV class="container"><DIV class="nav-collapse"><UL class="nav"><LI><A href="https://2012.igem.org/Team:NTNU_Trondheim"> Home</A></LI><LI class="dropdown"><A href="#" class="dropdown-toggle" data-toggle="dropdown">Project </A></LI><LI><A href="https://2012.igem.org/Team:NTNU_Trondheim/Project">Project description</A></LI><LI><A href="https://2012.igem.org/Team:NTNU_Trondheim/Model">Modelling</A></LI><LI><A href="https://2012.igem.org/Team:NTNU_Trondheim/Experiments_and_Results">Experiments and results</A></LI><LI><A href="https://2012.igem.org/Team:NTNU_Trondheim/Parts">BioBrick Parts</A></LI><LI><A href="https://2012.igem.org/Team:NTNU_Trondheim/Attributions">Attributions</A></LI></UL><LI class="dropdown"><A href="#" class="dropdown-toggle" data-toggle="dropdown"> Technical stuff </A></LI><LI><A href="https://2012.igem.org/Team:NTNU_Trondheim/Notebook">Notebook</A></LI><LI><A href="https://2012.igem.org/Team:NTNU_Trondheim/Protocols">Protocols</A></LI><LI><A href="https://2012.igem.org/Team:NTNU_Trondheim/Safety">Safety</A></LI><LI class="dropdown"><A href="#" class="dropdown-toggle" data-toggle="dropdown"> Team </A></LI><LI><A href="https://2012.igem.org/Team:NTNU_Trondheim/Team">Meet the team</A></LI><LI><A href="https://igem.org/Team.cgi?year=2012&amp;team_name=NTNU_Trondheim" target="_blank">Official team profile</A></LI><LI class="dropdown"><A href="#" class="dropdown-toggle" data-toggle="dropdown"> Outreach </A></LI><LI><A href="https://2012.igem.org/Team:NTNU_Trondheim/Collaboration">Collaboration</A></LI><LI><A href="https://2012.igem.org/Team:NTNU_Trondheim/Outreach">Human practices</A></LI><LI><A href="https://2012.igem.org/Team:NTNU_Trondheim/Press">Press coverage</A></LI><LI><A href="https://2012.igem.org/Team:NTNU_Trondheim/Achievements"> Achievements</A></LI><LI class="hidden-navbar"><A href="https://2012.igem.org/Team:NTNU_Trondheim/Matchmaker" target="_blank"> Matchmaker</A></LI></DIV></DIV></DIV></DIV><DIV class="container"><DIV class="page-header-top"><H1>Notebook <SMALL>July</SMALL></H1></DIV></DIV><DIV class="container main-container"><DIV class="row"><DIV class="span10 offset1"><DIV class="well well-small"><TABLE align="center" border="0" style="font-size: 12px;" cellpadding="15"><TBODY><TR valign="top"><TD><TABLE align="center" style="text-align:right;" cellpadding="3"><CAPTION><B><A href="https://2012.igem.org/Team:NTNU_Trondheim/Notebook/February" class="external text" rel="nofollow">February</A></B></CAPTION></TABLE></TD></TR><TR><TD> Mon
</TD><TD> Tue
</TD><TD> Wed
</TD><TD> Thu
</TD><TD> Fri
</TD><TD> Sat
</TD><TD> Sun
</TD></TR><TR><TD>1
</TD><TD>2
</TD><TD>3
</TD><TD>4
</TD><TD>5
</TD></TR><TR><TD>6
</TD><TD>7
</TD><TD>8
</TD><TD>9
</TD><TD>10
</TD><TD>11
</TD><TD>12
</TD></TR><TR><TD>13
</TD><TD>14
</TD><TD>15
</TD><TD>16
</TD><TD><A href="https://2012.igem.org/Notebook/February#Friday_17.02.12" class="external text" rel="nofollow">17</A></TD><TD>18
</TD><TD>19
</TD></TR><TR><TD>20
</TD><TD>21
</TD><TD>22
</TD><TD>23
</TD><TD>24
</TD><TD>25
</TD><TD>25
</TD></TR><TR><TD><A href="https://2012.igem.org/Notebook/February#Monday_27.02.12" class="external text" rel="nofollow">27</A></TD><TD>28
</TD><TD>29
</TD></TR></TBODY></TABLE><TABLE align="center" style="text-align:right;" cellpadding="3"><CAPTION><B><A href="https://2012.igem.org/Team:NTNU_Trondheim/Notebook/March" class="external text" rel="nofollow">March</A></B></CAPTION><TBODY><TR><TD> Mon
</TD><TD> Tue
</TD><TD> Wed
</TD><TD> Thu
</TD><TD> Fri
</TD><TD> Sat
</TD><TD> Sun
</TD></TR><TR><TD>1
</TD><TD>2
</TD><TD>3
</TD><TD>4
</TD></TR><TR><TD>5
</TD><TD>6
</TD><TD>7
</TD><TD>8
</TD><TD>9
</TD><TD>10
</TD><TD>11
</TD></TR><TR><TD>12
</TD><TD>13
</TD><TD>14
</TD><TD>15
</TD><TD><A href="https://2012.igem.org/Team:NTNU_Trondheim/Notebook/March#Friday_16.03.12" class="external text" rel="nofollow">16</A></TD><TD>17
</TD><TD>18
</TD></TR><TR><TD>19
</TD><TD>20
</TD><TD>21
</TD><TD>22
</TD><TD>23
</TD><TD>24
</TD><TD>25
</TD></TR><TR><TD>26
</TD><TD>27
</TD><TD>28
</TD><TD>29
</TD><TD><A href="https://2012.igem.org/Team:NTNU_Trondheim/Notebook/March#Friday_30.03.12" class="external text" rel="nofollow">30</A></TD></TR></TBODY></TABLE><TABLE align="center" style="text-align:right;" cellpadding="3"><CAPTION><B><A href="https://2012.igem.org/Team:NTNU_Trondheim/Notebook/April" class="external text" rel="nofollow">April</A></B></CAPTION><TBODY><TR><TD> Mon
</TD><TD> Tue
</TD><TD> Wed
</TD><TD> Thu
</TD><TD> Fri
</TD><TD> Sat
</TD><TD> Sun
</TD></TR><TR><TD>1
</TD></TR><TR><TD>2
</TD><TD>3
</TD><TD>4
</TD><TD>5
</TD><TD>6
</TD><TD>7
</TD><TD>8
</TD></TR><TR><TD>9
</TD><TD>10
</TD><TD>11
</TD><TD>12
</TD><TD><A href="https://2012.igem.org/Team:NTNU_Trondheim/Calendar/April#Friday_13.04.12" class="external text" rel="nofollow">13</A></TD><TD>14
</TD><TD>15
</TD></TR><TR><TD>16
</TD><TD>17
</TD><TD>18
</TD><TD>19
</TD><TD><A href="https://2012.igem.org/Team:NTNU_Trondheim/Calendar/April#Friday_20.04.12" class="external text" rel="nofollow">20</A></TD><TD>21
</TD><TD>22
</TD></TR><TR><TD>23
</TD><TD>24
</TD><TD>25
</TD><TD><A href="https://2012.igem.org/Team:NTNU_Trondheim/Calendar/April#Thursday_26.04.12" class="external text" rel="nofollow">26</A></TD><TD>27
</TD><TD>28
</TD><TD>29
</TD></TR><TR><TD>30
</TD></TR></TBODY></TABLE><TABLE align="center" style="text-align:right;" cellpadding="3"><CAPTION><B><A href="https://2012.igem.org/Team:NTNU_Trondheim/Notebook/May" class="external text" rel="nofollow">May</A></B></CAPTION><TBODY><TR><TD> Mon
</TD><TD> Tue
</TD><TD> Wed
</TD><TD> Thu
</TD><TD> Fri
</TD><TD> Sat
</TD><TD> Sun
</TD></TR><TR><TD>1
</TD><TD><A href="https://2012.igem.org/Team:NTNU_Trondheim/Notebook/May#Wednesday_02.05.12" class="external text" rel="nofollow">2</A></TD><TD><A href="https://2012.igem.org/Team:NTNU_Trondheim/Notebook/May#Thursday_03.05.12" class="external text" rel="nofollow">3</A></TD><TD>4
</TD><TD>5
</TD><TD>6
</TD></TR><TR><TD>7
</TD><TD>8
</TD><TD><A href="https://2012.igem.org/Team:NTNU_Trondheim/Notebook/May#Wednesday_09.05.12" class="external text" rel="nofollow">9</A></TD><TD>10
</TD><TD><A href="https://2012.igem.org/Team:NTNU_Trondheim/Notebook/May#Friday_11.05.12" class="external text" rel="nofollow">11</A></TD><TD>12
</TD><TD>13
</TD></TR><TR><TD>14
</TD><TD>15
</TD><TD>16
</TD><TD>17
</TD><TD><A href="https://2012.igem.org/Team:NTNU_Trondheim/Notebook/May#Friday_18.05.12" class="external text" rel="nofollow">18</A></TD><TD>19
</TD><TD>20
</TD></TR><TR><TD>21
</TD><TD>22
</TD><TD><A href="https://2012.igem.org/Team:NTNU_Trondheim/Notebook/May#Wednesday_23.05.12" class="external text" rel="nofollow">23</A></TD><TD>24
</TD><TD><A href="https://2012.igem.org/Team:NTNU_Trondheim/Notebook/May#Friday_25.05.12" class="external text" rel="nofollow">25</A></TD><TD>26
</TD><TD>27
</TD></TR><TR><TD>28
</TD><TD>29
</TD><TD>30
</TD><TD>31
</TD></TR></TBODY></TABLE><TABLE align="center" style="text-align:right;" cellpadding="3"><CAPTION><B><A href="https://2012.igem.org/Team:NTNU_Trondheim/Notebook/June" class="external text" rel="nofollow">June</A></B></CAPTION><TBODY><TR><TD> Mon
</TD><TD> Tue
</TD><TD> Wed
</TD><TD> Thu
</TD><TD> Fri
</TD><TD> Sat
</TD><TD> Sun
</TD></TR><TR><TD><A href="https://2012.igem.org/Team:NTNU_Trondheim/Notebook/June#Friday_01.06.12" class="external text" rel="nofollow">1</A></TD><TD>2
</TD><TD>3
</TD></TR><TR><TD>4
</TD><TD>5
</TD><TD>6
</TD><TD>7
</TD><TD>8
</TD><TD>9
</TD><TD>10
</TD></TR><TR><TD>11
</TD><TD>12
</TD><TD><A href="https://2012.igem.org/Team:NTNU_Trondheim/Notebook/June#Wednesday_13.06.12" class="external text" rel="nofollow">13</A></TD><TD>14
</TD><TD>15
</TD><TD>16
</TD><TD>17
</TD></TR><TR><TD><A href="https://2012.igem.org/Team:NTNU_Trondheim/Notebook/June#Monday_18.06.12" class="external text" rel="nofollow">18</A></TD><TD><A href="https://2012.igem.org/Team:NTNU_Trondheim/Notebook/June#Tuesday_19.06.12" class="external text" rel="nofollow">19</A></TD><TD><A href="https://2012.igem.org/Team:NTNU_Trondheim/Notebook/June#Wednesday_20.06.12" class="external text" rel="nofollow">20</A></TD><TD><A href="https://2012.igem.org/Team:NTNU_Trondheim/Notebook/June#Thursday_21.06.12" class="external text" rel="nofollow">21</A></TD><TD><A href="https://2012.igem.org/Team:NTNU_Trondheim/Notebook/June#Friday_22.06.12" class="external text" rel="nofollow">22</A></TD><TD><A href="https://2012.igem.org/Team:NTNU_Trondheim/Notebook/June#Saturday_23.06.12" class="external text" rel="nofollow">23</A></TD><TD><A href="https://2012.igem.org/Team:NTNU_Trondheim/Notebook/June#Sunday_24.06.12" class="external text" rel="nofollow">24</A></TD></TR><TR><TD><A href="https://2012.igem.org/Team:NTNU_Trondheim/Notebook/June#Monday_25.06.12" class="external text" rel="nofollow">25</A></TD><TD><A href="https://2012.igem.org/Team:NTNU_Trondheim/Notebook/June#Tuesday_26.06.12" class="external text" rel="nofollow">26</A></TD><TD><A href="https://2012.igem.org/Team:NTNU_Trondheim/Notebook/June#Wednesday_27.06.12" class="external text" rel="nofollow">27</A></TD><TD><A href="https://2012.igem.org/Team:NTNU_Trondheim/Notebook/June#Thursday_28.06.12" class="external text" rel="nofollow">28</A></TD><TD><A href="https://2012.igem.org/Team:NTNU_Trondheim/Notebook/June#Friday_29.06.12" class="external text" rel="nofollow">29</A></TD><TD><A href="https://2012.igem.org/Team:NTNU_Trondheim/Notebook/June#Saturday_30.06.12" class="external text" rel="nofollow">30</A></TD><TD><A href="https://2012.igem.org/Team:NTNU_Trondheim/Notebook/June#Sunday_31.06.12" class="external text" rel="nofollow">31</A></TD></TR></TBODY></TABLE><TABLE align="center" style="text-align:right;" cellpadding="3"><CAPTION><B><A href="https://2012.igem.org/Team:NTNU_Trondheim/Notebook/July" class="external text" rel="nofollow">July</A></B></CAPTION><TBODY><TR><TD> Mon
</TD><TD> Tue
</TD><TD> Wed
</TD><TD> Thu
</TD><TD> Fri
</TD><TD> Sat
</TD><TD> Sun
</TD></TR><TR><TD><A href="https://2012.igem.org/Team:NTNU_Trondheim/Notebook/July#Sunday_01.07.12" class="external text" rel="nofollow">1</A></TD></TR><TR><TD><A href="https://2012.igem.org/Team:NTNU_Trondheim/Notebook/July#Monday_02.07.12" class="external text" rel="nofollow">2</A></TD><TD><A href="https://2012.igem.org/Team:NTNU_Trondheim/Notebook/July#Tuesday_03.07.12" class="external text" rel="nofollow">3</A></TD><TD><A href="https://2012.igem.org/Team:NTNU_Trondheim/Notebook/July#Wednesday_04.07.12" class="external text" rel="nofollow">4</A></TD><TD><A href="https://2012.igem.org/Team:NTNU_Trondheim/Notebook/July#Thursday_05.07.12" class="external text" rel="nofollow">5</A></TD><TD><A href="https://2012.igem.org/Team:NTNU_Trondheim/Notebook/July#Friday_06.07.12" class="external text" rel="nofollow">6</A></TD><TD><A href="https://2012.igem.org/Team:NTNU_Trondheim/Notebook/July#Saturday_07.07.12" class="external text" rel="nofollow">7</A></TD><TD><A href="https://2012.igem.org/Team:NTNU_Trondheim/Notebook/July#Friday_08.07.12" class="external text" rel="nofollow">8</A></TD></TR><TR><TD><A href="https://2012.igem.org/Team:NTNU_Trondheim/Notebook/July#Monday_09.07.12" class="external text" rel="nofollow">9</A></TD><TD><A href="https://2012.igem.org/Team:NTNU_Trondheim/Notebook/July#Tuesday_10.07.12" class="external text" rel="nofollow">10</A></TD><TD><A href="https://2012.igem.org/Team:NTNU_Trondheim/Notebook/July#Wednesday_11.07.12" class="external text" rel="nofollow">11</A></TD><TD><A href="https://2012.igem.org/Team:NTNU_Trondheim/Notebook/July#Thursday_12.07.12" class="external text" rel="nofollow">12</A></TD><TD><A href="https://2012.igem.org/Team:NTNU_Trondheim/Notebook/July#Friday_13.07.12" class="external text" rel="nofollow">13</A></TD><TD><A href="https://2012.igem.org/Team:NTNU_Trondheim/Notebook/July#Saturday_14.07.12" class="external text" rel="nofollow">14</A></TD><TD><A href="https://2012.igem.org/Team:NTNU_Trondheim/Notebook/July#Sunday_15.07.12" class="external text" rel="nofollow">15</A></TD></TR><TR><TD><A href="https://2012.igem.org/Team:NTNU_Trondheim/Notebook/July#Monday_16.07.12" class="external text" rel="nofollow">16</A></TD><TD><A href="https://2012.igem.org/Team:NTNU_Trondheim/Notebook/July#Tuesday_17.07.12" class="external text" rel="nofollow">17</A></TD><TD><A href="https://2012.igem.org/Team:NTNU_Trondheim/Notebook/July#Wednesday_18.07.12" class="external text" rel="nofollow">18</A></TD><TD><A href="https://2012.igem.org/Team:NTNU_Trondheim/Notebook/July#Thursday_19.07.12" class="external text" rel="nofollow">19</A></TD><TD><A href="https://2012.igem.org/Team:NTNU_Trondheim/Notebook/July#Friday_20.07.12" class="external text" rel="nofollow">20</A></TD><TD><A href="https://2012.igem.org/Team:NTNU_Trondheim/Notebook/July#Saturday_21.07.12" class="external text" rel="nofollow">21</A></TD><TD><A href="https://2012.igem.org/Team:NTNU_Trondheim/Notebook/July#Sunday_22.07.12" class="external text" rel="nofollow">22</A></TD></TR><TR><TD><A href="https://2012.igem.org/Team:NTNU_Trondheim/Notebook/July#Monday_23.07.12" class="external text" rel="nofollow">23</A></TD><TD><A href="https://2012.igem.org/Team:NTNU_Trondheim/Notebook/July#Tuesday_24.07.12" class="external text" rel="nofollow">24</A></TD><TD><A href="https://2012.igem.org/Team:NTNU_Trondheim/Notebook/July#Wednesday_25.07.12" class="external text" rel="nofollow">25</A></TD><TD><A href="https://2012.igem.org/Team:NTNU_Trondheim/Notebook/July#Thursday_26.07.12" class="external text" rel="nofollow">26</A></TD><TD><A href="https://2012.igem.org/Team:NTNU_Trondheim/Notebook/July#Friday_27.07.12" class="external text" rel="nofollow">27</A></TD><TD><A href="https://2012.igem.org/Team:NTNU_Trondheim/Notebook/July#Saturday_28.07.12" class="external text" rel="nofollow">28</A></TD><TD><A href="https://2012.igem.org/Team:NTNU_Trondheim/Notebook/July#Sunday_29.07.12" class="external text" rel="nofollow">29</A></TD></TR><TR><TD><A href="https://2012.igem.org/Team:NTNU_Trondheim/Notebook/July#Monday_30.07.12" class="external text" rel="nofollow">30</A></TD><TD><A href="https://2012.igem.org/Team:NTNU_Trondheim/Notebook/July#Tuesday_31.07.12" class="external text" rel="nofollow">31</A></TD></TR></TBODY></TABLE><TABLE align="center" style="text-align:right;" cellpadding="3"><CAPTION><B><A href="https://2012.igem.org/Team:NTNU_Trondheim/Notebook/August" class="external text" rel="nofollow">August</A></B></CAPTION><TBODY><TR><TD> Mon
</TD><TD> Tue
</TD><TD> Wed
</TD><TD> Thu
</TD><TD> Fri
</TD><TD> Sat
</TD><TD> Sun
</TD></TR><TR><TD><A href="https://2012.igem.org/Team:NTNU_Trondheim/Notebook/August#Wednesday_01.08.12" class="external text" rel="nofollow">1</A></TD><TD><A href="https://2012.igem.org/Team:NTNU_Trondheim/Notebook/August#Thursday_02.08.12" class="external text" rel="nofollow">2</A></TD><TD><A href="https://2012.igem.org/Team:NTNU_Trondheim/Notebook/August#Friday_03.08.12" class="external text" rel="nofollow">3</A></TD><TD><A href="https://2012.igem.org/Team:NTNU_Trondheim/Notebook/August#Saturday_04.08.12" class="external text" rel="nofollow">4</A></TD><TD><A href="https://2012.igem.org/Team:NTNU_Trondheim/Notebook/August#Sunday_05.08.12" class="external text" rel="nofollow">5</A></TD></TR><TR><TD><A href="https://2012.igem.org/Team:NTNU_Trondheim/Notebook/August#Monday_06.08.12" class="external text" rel="nofollow">6</A></TD><TD><A href="https://2012.igem.org/Team:NTNU_Trondheim/Notebook/August#Tuesday_07.08.12" class="external text" rel="nofollow">7</A></TD><TD><A href="https://2012.igem.org/Team:NTNU_Trondheim/Notebook/August#Wednesday_08.08.12" class="external text" rel="nofollow">8</A></TD><TD><A href="https://2012.igem.org/Team:NTNU_Trondheim/Notebook/August#Thursday_09.08.12" class="external text" rel="nofollow">9</A></TD><TD><A href="https://2012.igem.org/Team:NTNU_Trondheim/Notebook/August#Friday_10.08.12" class="external text" rel="nofollow">10</A></TD><TD><A href="https://2012.igem.org/Team:NTNU_Trondheim/Notebook/August#Saturday_11.08.12" class="external text" rel="nofollow">11</A></TD><TD><A href="https://2012.igem.org/Team:NTNU_Trondheim/Notebook/August#Sunday_12.08.12" class="external text" rel="nofollow">12</A></TD></TR><TR><TD><A href="https://2012.igem.org/Team:NTNU_Trondheim/Notebook/August#Monday_13.08.12" class="external text" rel="nofollow">13</A></TD><TD><A href="https://2012.igem.org/Team:NTNU_Trondheim/Notebook/August#Tuesday_14.08.12" class="external text" rel="nofollow">14</A></TD><TD><A href="https://2012.igem.org/Team:NTNU_Trondheim/Notebook/August#Wednesday_15.08.12" class="external text" rel="nofollow">15</A></TD><TD><A href="https://2012.igem.org/Team:NTNU_Trondheim/Notebook/August#Thursday_16.08.12" class="external text" rel="nofollow">16</A></TD><TD><A href="https://2012.igem.org/Team:NTNU_Trondheim/Notebook/August#Friday_17.08.12" class="external text" rel="nofollow">17</A></TD><TD><A href="https://2012.igem.org/Team:NTNU_Trondheim/Notebook/August#Saturday_18.08.12" class="external text" rel="nofollow">18</A></TD><TD><A href="https://2012.igem.org/Team:NTNU_Trondheim/Notebook/August#Sunday_19.08.12" class="external text" rel="nofollow">19</A></TD></TR><TR><TD><A href="https://2012.igem.org/Team:NTNU_Trondheim/Notebook/August#Monday_20.08.12" class="external text" rel="nofollow">20</A></TD><TD><A href="https://2012.igem.org/Team:NTNU_Trondheim/Notebook/August#Tuesday_21.08.12" class="external text" rel="nofollow">21</A></TD><TD><A href="https://2012.igem.org/Team:NTNU_Trondheim/Notebook/August#Wednesday_22.08.12" class="external text" rel="nofollow">22</A></TD><TD><A href="https://2012.igem.org/Team:NTNU_Trondheim/Notebook/August#Thursday_23.08.12" class="external text" rel="nofollow">23</A></TD><TD><A href="https://2012.igem.org/Team:NTNU_Trondheim/Notebook/August#Friday_24.08.12" class="external text" rel="nofollow">24</A></TD><TD><A href="https://2012.igem.org/Team:NTNU_Trondheim/Notebook/August#Saturday_25.08.12" class="external text" rel="nofollow">25</A></TD><TD><A href="https://2012.igem.org/Team:NTNU_Trondheim/Notebook/August#Sunday_26.08.12" class="external text" rel="nofollow">26</A></TD></TR><TR><TD><A href="https://2012.igem.org/Team:NTNU_Trondheim/Notebook/August#Monday_.27.12" class="external text" rel="nofollow">27</A></TD><TD><A href="https://2012.igem.org/Team:NTNU_Trondheim/Notebook/August#Tuesday_28.08.12" class="external text" rel="nofollow">28</A></TD><TD><A href="https://2012.igem.org/Team:NTNU_Trondheim/Notebook/August#Wednesday_29.08.12" class="external text" rel="nofollow">29</A></TD><TD><A href="https://2012.igem.org/Team:NTNU_Trondheim/Notebook/August#Thursday_30.08.12" class="external text" rel="nofollow">30</A></TD><TD><A href="https://2012.igem.org/Team:NTNU_Trondheim/Notebook/August#Friday_31.08.12" class="external text" rel="nofollow">31</A></TD></TR></TBODY></TABLE><TABLE align="center" style="text-align:right;" cellpadding="3"><CAPTION><B><A href="https://2012.igem.org/Team:NTNU_Trondheim/Notebook/September" class="external text" rel="nofollow">September</A></B></CAPTION><TBODY><TR><TD> Mon
</TD><TD> Tue
</TD><TD> Wed
</TD><TD> Thu
</TD><TD> Fri
</TD><TD> Sat
</TD><TD> Sun
</TD></TR><TR><TD> 1
</TD><TD> 2
</TD></TR><TR><TD> 3
</TD><TD> 4
</TD><TD> 5
</TD><TD> 6
</TD><TD> 7
</TD><TD> 8
</TD><TD> 9
</TD></TR><TR><TD>10
</TD><TD>11
</TD><TD>12
</TD><TD>13
</TD><TD>14
</TD><TD>15
</TD><TD>16
</TD></TR><TR><TD>17
</TD><TD>18
</TD><TD>19
</TD><TD>20
</TD><TD>21
</TD><TD>22
</TD><TD>23
</TD></TR><TR><TD>24
</TD><TD>25
</TD><TD>26
</TD><TD>27
</TD><TD>28
</TD><TD>29
</TD><TD>30
</TD></TR></TBODY></TABLE></DIV></DIV></DIV><DIV class="row-fluid"><DIV class="span12"><H3><SPAN class="mw-headline" id="Tuesday_31.07.12">Tuesday 31.07.12</SPAN></H3><P>The restriction digest for colicin cut with E+P made yesterday was purified using the QIAquick PCR purification kit, and after purification, the following ligations were made:
</P><TABLE class="table table-bordered table-hover" style="margin: 1em auto 1em auto; width: auto;"><TBODY><TR><TH>Insert
</TH><TH>Backbone
</TH><TH>Volume of insert [Âµl]
</TH><TH>Volume of backbone [Âµl]
</TH><TH>Amount of T4 DNA ligase buffer [Âµl]
</TH><TH>Amount of T4 DNA ligase [Âµl]
</TH><TH>Amount of dH<SUB>2</SUB>O added [Âµl]
</TH></TR><TR><TD>Colicin cut with E+P
</TD><TD>pSB1C3 cut with E+P
</TD><TD>4.0
</TD><TD>2.0
</TD><TD>1.0
</TD><TD>0.5
</TD><TD>2.5
</TD></TR><TR><TD>Colicin cut with X+P
</TD><TD>K+RBS cut with S+P
</TD><TD>7.0
</TD><TD>5.0
</TD><TD>1.5
</TD><TD>0.75
</TD><TD>0.75
</TD></TR><TR><TD> -
</TD><TD>pSB1C3 cut with E+P
</TD><TD> -
</TD><TD>2.0
</TD><TD>1.0
</TD><TD>0.5
</TD><TD>6.5
</TD></TR><TR><TD> -
</TD><TD>K+RBS cut with S+P
</TD><TD> -
</TD><TD>5.0
</TD><TD>1.5
</TD><TD>0.75
</TD><TD>7.75
</TD></TR></TBODY></TABLE><P>Both the ligation and the religations were transformed to competent <I>E.coli</I> DH5Î± cells.
</P><P>Restriction digest was run on the Lysis device (&lt;partinfo&gt;BBa_K112808&lt;/partinfo&gt;) and the lld-promoter, as described in the <A href="/Team:NTNU_Trondheim/Protocols" title="Team:NTNU Trondheim/Protocols">protocol</A>. Lysis was cut with X and P, plld cut with S and P. Plld was purified using a PCR purification kit, gel electrophoresis was done with the lysis device. The concentration of the plld was 3,2 ng/Âµl.
</P><H3><SPAN class="mw-headline" id="Monday_30.07.12">Monday 30.07.12</SPAN></H3><P>Today, we started testing the Vgb (&lt;partinfo&gt;BBa_K561001&lt;/partinfo&gt;) and Vhb (&lt;partinfo&gt;BBa_K258005&lt;/partinfo&gt;) promoters. For both promoters, we have made test constructs consisting of promoter, RBS (&lt;partinfo&gt;BBa_B0034&lt;/partinfo&gt;), YFP (&lt;partinfo&gt;BBa_E0030&lt;/partinfo&gt;) and terminator (&lt;partinfo&gt;BBa_B0015&lt;/partinfo&gt;). The testing was carried out by using two parallel cultures of each promoter construct, and inducing one of the parallels by bubbling the culture with nitrogen gas to remove oxygen. Samples were collected every 10 minutes.
</P><P>Further, the constitutive promoter containing RBS (&lt;partinfo&gt;BBa_K081005&lt;/partinfo&gt;) was cut with S+P, and the purified colicin PCR product were cut with X+P, to be ligated tomorrow. Another colicin PCR product sample was cut with E+P, to be ligated into the already cut pSB1C3 tomorrow.
</P><P>We also test cutted P<SUB>lld</SUB> with BpmI and AflIII. This was performed because the number of religations for the pSB1C3 plasmid was higher than it should have been. One of the samples to be test cutted was taken from the miniprepped plasmids of one of the colonies on the religation plate, to be used as negative controll, and three parallel samples from the ligation plate were test cutted. The gel pictures are shown below:
</P><P>though the number of active promoters remains relatively low, the constant production of mRNA leads to a large production of LuxI.
</P><TABLE style="text-align:center; margin: 1em auto 1em auto; padding: 0 10px;" cellpadding="0px"><TBODY><TR><TD><DIV class="center"><DIV class="thumb tnone"><DIV class="thumbinner" style="width:452px;"><DIV class="thumbcaption">Religated pSB1C3, and three parallels of ligated P<SUB>lld</SUB> and pSB1C3 investigated by cutting with AflIII. AflIII cuts both inside the promoter region, and in the backbone, so if the promoter is present in the plasmid, the expected result for this test cut is two fragments; one on 277 bp, and one on 2136. All parallels from the ligation shows two bonds at the expected positions.</DIV></DIV></DIV></DIV></TD><TD><DIV class="thumb tright"><DIV class="thumbinner" style="width:452px;"><DIV class="thumbcaption">Religated pSB1C3, and three parallels of ligated P<SUB>lld</SUB> and pSB1C3 investigated by cutting with BpmI. BpmI also cuts both inside the promoter region and in the backbone, so if the promoter is present in the plasmid, the expected result for this test cut is two fragments; one on 898 bp, and one on 1515. Only one band is present on the gel.</DIV></DIV></DIV></TD></TR></TBODY></TABLE><P>The results from the test cutting were ambigous. The test cut performed with AflIII show the expected fragments, while the test cut performed with BpmI do not. It seems that the promoter is there in all thre parallels; if not it would have been impossible to yield two fragements in the AflIII test cut. But still, if the promoter sequence is as it should have been, also the test cut with BpmI should have yielded two fragments. It is, however, possible, that the BpmI restriction site inside the promoter is ok, but that the restriction site in the plasmid is damaged.
</P><H3><SPAN class="mw-headline" id="Sunday_29.07.12">Sunday 29.07.12</SPAN></H3><P>Made LB medium in preparation for experiment with the oxygen-sensitive promoters tomorrow.
Inspeccted the &lt;partinfo&gt;pSB1C3&lt;/partinfo&gt; religation plate in the incubator. Many colonies that were not visible at the time of the colony count yesterday has now appeared. The religation and the pllD plate now contains about the same number of colonies, casting doubt on the sucess of the transformation. Moved the plate to the fridge.
</P><P>Extracted plasmid DNA by miniprep from the pllD (unconfirmed) and &lt;partinfo&gt;pSB1C3&lt;/partinfo&gt; religation cultures inoculated yesterday, and the culture of &lt;partinfo&gt;BBa_E0040&lt;/partinfo&gt; (A. vic. GFP) inoculated 26/7. DNA concentrations were measured with NanoDrop, as follows (two parallel measurements were made for each sample):
</P><TABLE class="table table-bordered table-hover" style="margin: 1em auto 1em auto; width: auto;"><TBODY><TR><TH>Sample
</TH><TH>Concentration [ng/Âµl]
</TH></TR><TR><TD>pllD 1
</TD><TD>24,0/22,2
</TD></TR><TR><TD>pllD 2
</TD><TD>25,9/26,5
</TD></TR><TR><TD>pllD 3
</TD><TD>22,8/23,9
</TD></TR><TR><TD>&lt;partinfo&gt;pSB1C3&lt;/partinfo&gt; 1
</TD><TD>15,4/15,5
</TD></TR><TR><TD>&lt;partinfo&gt;pSB1C3&lt;/partinfo&gt; 2
</TD><TD>20,7/20,7
</TD></TR><TR><TD>&lt;partinfo&gt;BBa_E0040&lt;/partinfo&gt;</TD><TD>38,2/37,8
</TD></TR></TBODY></TABLE><P>DNA from all the above samples except GFP was digested with Xba1 and Spe1. The digested samples were analyzed by gel electrophoresis to find out whether the desired insert was obtained in any of the samples. 75 V was applied for 45 min, and the gel was imaged. After imaging, electrophoresis was continued at 40 V for a further 30 minutes before imaging again. The result of the second imaging is shown below.
</P><P>
Results of gel electrophoresis: All three samples from the pllD plate showed a short band identical between these samples, but it is difficult to determine the exact size - there seems to be an inconsistency in the migration of the 50 bp ladder on the left side of the gel and the 1 kb ladder on the right. Looking at the 50 bp ladder, the short band in the three first samples seems to be between the third and the fourth band from the bottom, representing 150 and 200 bp respectively. However, looking at the 1 kb ladder on the right, the short bands lie between the first and second ladder bands, equaling 250 and 500 bp respectively. The position of the short bands with respect to the 1 kb ladder appears to be consistent with the expected size of the pllD insert.
</P><P>Taken together, it seems possible that the short band represents the desired insert, but the samples should be analyzed again to get more data. The samples from the &lt;partinfo&gt;pSB1C3&lt;/partinfo&gt; religation plate does not show this band and is overall much fainter. 
</P><P>The top bands in the pllD wells are on heigth with the seventh band from the bottom, equaling 2500 bp, in the Generuler 1kb ladder on the right. This is approximately the size of the &lt;partinfo&gt;pSB1C3&lt;/partinfo&gt; plasmid and the pllD insert together. The two wells on the right containing DNA from the &lt;partinfo&gt;pSB1C3&lt;/partinfo&gt; religation plate show two faint bands each: One slightly above the top bands in the pllD wells, and one shorter band slightly above the 1000 bp reference band in the 1 kb ladder.
</P><H3><SPAN class="mw-headline" id="Saturday_28.07.12">Saturday 28.07.12</SPAN></H3><P>Inspected the plates with PllD in &lt;partinfo&gt;pSB1C3&lt;/partinfo&gt; and &lt;partinfo&gt;pSB1C3&lt;/partinfo&gt; religation transformants inoculated yesterday. Growth on both plates, but more on the PllD plate. No growth on negative control (non-transformed cells). Approximate colony cound:
Religation: 73
pllD: 240
</P><P>Assuming other factors are equal, based on these numbers a colony on the lldP plate should have about a 2/3 chance of containing the desired insert. Suggested course of action: Inoculate several colonies from the lddp plate, miniprep, perform a test restriction digest and gel electrophoresis to check size.  Use colonies from the religation plate for reference: Plasmids with the desired insert should be about 344 bp larger than the religated plasmid without insert.
</P><P>Inoculated 3 colonies from the PllD plate and 2 colonies from the &lt;partinfo&gt;pSB1C3&lt;/partinfo&gt; religation plate into 5 mL LB + Cm, and placed the cultures in 37 C shaking incubator. Placed the plates back in 37 C cabinet.
</P><P>Updated information about stored DNA samples and glycerol stocks on the wiki.
</P><P>Evening: Moved lldP plate from incubator to fridge.
</P><H3><SPAN class="mw-headline" id="Friday_27.07.12">Friday 27.07.12</SPAN></H3><P>The PCR batches run yesterday were investigated on gel electrophoresis, and the gel picture shows that colicin has been made!
</P><DIV class="center"><DIV class="thumb tnone"><DIV class="thumbinner" style="width:452px;"><DIV class="thumbcaption">PCR batches for P<SUB>lld</SUB>, LldR, and colicin investigated on gel. Apparently, no PCR product was obtained for P<SUB>lld</SUB> and LldR, but the colicin sample loaded in the third well shows a bond above 2000 bp. It is hard to imagine that this can be anything else than colicin.</DIV></DIV></DIV></DIV><P>The colicin PCR product were purified using the QIAquick PCR Purification Kit, and the concentration of product was measured to be 15.3 ng/Âµl.
</P><P>Linearized pSB1C3 plasmid was digested with EcoRI and PstI, according to the procedure for <A href="http://partsregistry.org/Help:Protocols/Linearized_Plasmid_Backbones" class="external text" rel="nofollow">digestion of linearized plasmids</A>, and ligated with the lld promoter made by PCR on wednesday. Religation was also performed, and both the ligated plasmid and the religated plasmid were transformed to competent <I>E.coli</I> DH5Î± cells.
</P><P>New primers were ordered for LldR, this time containing the biobrick prefix and suffix.
</P><P>Discussed experimental design for testing of oxygen-sensitive promoters with Martin. Made a plan for testing both promoters on Monday.
</P><P>Measured OD600 of samples from the cultures used to make glycerol stocks yesterday. The samples were taken yesterday and kept at 5 C until today. Results:
EYFP (RBS+ LVA- TERM)(&lt;partinfo&gt;BBa_E0430&lt;/partinfo&gt;): 1,00
pTet RFP (&lt;partinfo&gt;BBa_I13521&lt;/partinfo&gt;)Â : 2,14
ptet GFP (&lt;partinfo&gt;BBa_I13522&lt;/partinfo&gt;)Â : 0,86
</P><P>Measured YFP, GFP and RFP fluorescence from the following cultures (<A href="https://www.dropbox.com/s/ku0l7enk5c6trkz/Fluorescence%20measurements%2027.07.xlsx" class="external text" rel="nofollow">data</A>):
LB (22/7) (D1)
K + RBS + YFP + DTT (19/7) (D2)
ptet mRFP (23/7) (D3)
ptet mRFP (26/7) (D4)
ptet GFP (26/7) (D5)
EYFP (RBS+ LVA- TERM) (26/7) (D6)
</P><P>Measurements confirmed that ptet GFP and ptet MRFP produce the desired products and can be used as positive controls. Significant overlap was observed between GFP and YFP fluorescence.
</P><H3><SPAN class="mw-headline" id="Thursday_26.07.12">Thursday 26.07.12</SPAN></H3><P>The PCR-products for P<SUB>lld</SUB>, LldR, and colicin were run on gel:
</P><DIV class="center"><DIV class="thumb tnone"><DIV class="thumbinner" style="width:452px;"><DIV class="thumbcaption">The obtained PCR products from the three PCRs run wednesday. The first well after the ladder contains the PCR product for the P<SUB>lld</SUB> promoter, which in theory should have been 344 bp long. One PCR product was obtained, but even if it is closer to the lowest band on the ladder, being 250 bp long, it is assumed that the correct product was obtained. The second well after the ladder apparently contains no products, but should have contained LldR, being 777 bp long. The third well after the ladder should have contained colicin, being 2050 bp long. In this well, it is possible to see a weak band close to the 3000 band on the ladder, but whether or not this is the correct product is not certain.</DIV></DIV></DIV></DIV><P>
Since we are assuming that the P<SUB>lld</SUB> promoter is correct, this PCR product were purified using the QIAquick PCR purification kit. The concentration after purification was 16.5 ng/Âµl DNA. 
The biobrick were then cut with EcoRI and PstI, to be ligated with EcoRI and PstI digested pSB1C3 tomorrow.
</P><P>As no PCR product was made for LldR, and none of the expected product was made in the case of colicin, a new PCR were run, also including P<SUB>lld</SUB>, to investigate it on gel again, to see if the length of it relative to the ladder is more precise. For the case of the new PCR mixes, 50 Âµl were made instead of 20 Âµl, and 1 Âµl DMSO, which is known to increase the yield of desired product, was added. The PCR mixes were made according to the following procedure:
</P><UL><LI> 31.5 Âµl dH<SUB>2</SUB>O
</LI><LI> 10 Âµl 5x Phusion buffer
</LI><LI> 1 Âµl dNTPs
</LI><LI> 2.5 Âµl fwd primer (both primers having concentration 10 ÂµM)
</LI><LI> 2.5 Âµl rev primer
</LI><LI> 1 Âµl template DNA
</LI><LI> 1 Âµl DMSO
</LI><LI> 0.5 Âµl Finnzymes Phusion HF DNA polymerase
</LI></UL><P>The thermocyclers were programmed to use the same PCR programs as the ones used wednesday.
</P><P>
GFP (&lt;partinfo&gt;BBa_E0040&lt;/partinfo&gt;) were also transferred to liquid culture.
</P><P>Today, we also had a video conference with the <A href="https://2012.igem.org/Team:RHIT" class="external text" rel="nofollow">Rose Hulman Institute of Technology iGEM team</A>, discussing how we could collaborate with each other. It was really nice meeting you, and hopefully, we'll be seing you in Boston in November:-)
</P><P>Inspected plates with EYFP + LVA and ptet GFP inoculated 24/7, incubated overnight and kept in fridge since yesterday. Good growth. 
</P><P>Discarded K + RBS + RFP plate (17/7) found in incubator cabinet.
</P><P>Made glycerol stocks from the following cultures:
EYFP + LVA (&lt;partinfo&gt;BBa_E0430&lt;/partinfo&gt;) (26/7)
ptet GFP (&lt;partinfo&gt;BBa_I13522&lt;/partinfo&gt;) (26/7)
ptet mRFP (&lt;partinfo&gt;BBa_I13521&lt;/partinfo&gt;) (26/7)
</P><P>Re-inoculated following liquid cultures in new LB + amp medium (50 uL to 5 mL), followed by miniprep of the old culture. 
VGB + RBS + YFP + DTT (25/7) 
VHB + RBS + YFP + DTT (25/7)
EYFP + LVA (25/7)
ptet GFP (25/7)
</P><P>
Measured DNA concentrations (two parallels):
VGB + RBS + YFP + DTT: 23,2/22,6 
VHB + RBS + YFP + DTT: 15,7/15,5
EYFP + LVA: 19,5/20,1
ptet GFP: 23,2/22,3
</P><P>Ran gel electrophoresis of the latest PCR products (plld, LldR, colicin operon).
</P><H3><SPAN class="mw-headline" id="Wednesday_25.07.12">Wednesday 25.07.12</SPAN></H3><P>The pBad+RBS+YFP+DTT, the Vgb+RBS+YFP+DTT and the Vhb+RBS+YFP+DTT constructs were transferred to liquid culture. The primers we ordered two and a half weeks ago (!) also arrived this morning, so both the lld promoter, LldR and colicin were amplified using PCR. The following PCR mix was used for the three samples:
</P><UL><LI> 12.4 Âµl dH<SUB>2</SUB>O
</LI><LI> 4 Âµl 5x Phusion buffer
</LI><LI> 0.4 Âµl dNTPs
</LI><LI> 1 Âµl fwd primer (both primers having concentration 10 ÂµM)
</LI><LI> 1 Âµl rev primer
</LI><LI> 1 Âµl template DNA
</LI><LI> 0.2 Âµl Phusion HF DNA polymerase
</LI></UL><P>In the case of both the lld promoter and the LldR gene, the genome of <I>E.coli</I> K12 (strain ER2925) was used as template. For the colicin batch, the &lt;partinfo&gt;BBa_K150009&lt;/partinfo&gt; biobrick was used as template.
Two of the primer sets contained prefix and suffix, and for these, two different melting temperatures were determined. The reason for this is that in the first cycles, the only the primer parts able to bind to the template will be determining for the melting temperature, while in the later cycles, where the amplified DNA is dominating as template, also the prefix / suffix parts of the primers will be able to bind to the DNA, and hence the primers will obtain higher melting temperatures. For the two PCR mixes containing primers with prefix and suffix, a three step protocol was used in the first ten cycles, while a two step protocol was used in the last fifteen cycles. Phusion High Fidelity DNA Polymerase (Finnzymes). To calculate the melting temperature, we used Finnzymes' Tm calculator (<A href="http://www.finnzymes.fi/tm_determination.html" class="external free" rel="nofollow">http://www.finnzymes.fi/tm_determination.html</A>).
</P><TABLE class="table table-bordered table-hover" style="margin: 1em auto 1em auto; width: auto;"><TBODY><TR><TH>Primer
</TH><TH>Type
</TH><TH>Sequence
</TH><TH>Tm [Â°C]
</TH></TR><TR><TD>lld promoter fwd
</TD><TD>Forward
</TD><TD>cacattcctataggccgagtaaggt
</TD><TD>66.2
</TD></TR><TR><TD>lld promoter rev
</TD><TD>Reverse
</TD><TD>gcaggtctcctggagtccacgc
</TD><TD>73.8
</TD></TR><TR><TD>lldR fwd
</TD><TD>Forward
</TD><TD>atgattgttttacccagacgcctgt
</TD><TD>69.3
</TD></TR><TR><TD>lldR rev
</TD><TD>Reverse
</TD><TD>tcatgcgtttttctccctcgaat
</TD><TD>69.7
</TD></TR><TR><TD>Colicin fwd
</TD><TD>Forward
</TD><TD>atggaaaccgcggtagcgta
</TD><TD>68.7
</TD></TR><TR><TD>Colicin rev
</TD><TD>Reverse
</TD><TD>tgcgatggtccctccctgaa
</TD><TD>72.6
</TD></TR></TBODY></TABLE><P>The following PCR programs were used:
</P><TABLE style="text-align:center; margin: 1em auto 1em auto;"><TBODY><TR><TD><TABLE class="table table-bordered table-hover" style="margin: 1em auto 1em auto; width: auto;"><CAPTION> lld promoter
</CAPTION></TABLE></TD></TR><TR><TH> Step
</TH><TH> Action
</TH><TH> Temperature
</TH><TH> Duration
</TH></TR><TR><TD>1
</TD><TD>Heated lid:
</TD><TD>103Â°C
</TD></TR><TR><TD>2
</TD><TD>Initial denaturation;
</TD><TD>98Â°C
</TD><TD>30 s
</TD></TR><TR><TD>3
</TD><TD>Denaturation;
</TD><TD>98Â°C
</TD><TD>10 s
</TD></TR><TR><TD>4
</TD><TD>Annealing;
</TD><TD>69Â°C
</TD><TD>10 s
</TD></TR><TR><TD>5
</TD><TD>Elongation;
</TD><TD>72Â°C
</TD><TD>6 s
</TD></TR><TR><TD>6
</TD><TD>Go to step 3, repeat 10 x
</TD></TR><TR><TD>7
</TD><TD>Denaturation;
</TD><TD>98Â°C
</TD><TD>10 s
</TD></TR><TR><TD>8
</TD><TD>Elongation;
</TD><TD>72Â°C
</TD><TD>6 s
</TD></TR><TR><TD>9
</TD><TD>Go to step 7, repeat 15 x
</TD></TR><TR><TD>10
</TD><TD>Final elongation;
</TD><TD>72Â°C
</TD><TD>10 min
</TD></TR><TR><TD>11
</TD><TD>Hold
</TD><TD>4Â°C
</TD><TD>â
</TD></TR></TBODY></TABLE><TABLE class="table table-bordered table-hover" style="margin: 1em auto 1em auto; width: auto;"><CAPTION> Colicin
</CAPTION><TBODY><TR><TH> Step
</TH><TH> Action
</TH><TH> Temperature
</TH><TH> Duration
</TH></TR><TR><TD>1
</TD><TD>Heated lid:
</TD><TD>103Â°C
</TD></TR><TR><TD>2
</TD><TD>Initial denaturation;
</TD><TD>98Â°C
</TD><TD>30 s
</TD></TR><TR><TD>3
</TD><TD>Denaturation;
</TD><TD>98Â°C
</TD><TD>10 s
</TD></TR><TR><TD>4
</TD><TD>Annealing;
</TD><TD>70Â°C
</TD><TD>10 s
</TD></TR><TR><TD>5
</TD><TD>Elongation;
</TD><TD>72Â°C
</TD><TD>31 s
</TD></TR><TR><TD>6
</TD><TD>Go to step 3, repeat 10 x
</TD></TR><TR><TD>7
</TD><TD>Denaturation;
</TD><TD>98Â°C
</TD><TD>10 s
</TD></TR><TR><TD>8
</TD><TD>Elongation;
</TD><TD>72Â°C
</TD><TD>31 s
</TD></TR><TR><TD>9
</TD><TD>Go to step 7, repeat 15 x
</TD></TR><TR><TD>10
</TD><TD>Final elongation;
</TD><TD>72Â°C
</TD><TD>10 min
</TD></TR><TR><TD>11
</TD><TD>Hold
</TD><TD>4Â°C
</TD><TD>â
</TD></TR></TBODY></TABLE><TABLE class="table table-bordered table-hover" style="margin: 1em auto 1em auto; width: auto;"><CAPTION> lldR
</CAPTION><TBODY><TR><TH> Step
</TH><TH> Action
</TH><TH> Temperature
</TH><TH> Duration
</TH></TR><TR><TD>1
</TD><TD>Heated lid:
</TD><TD>103Â°C
</TD></TR><TR><TD>2
</TD><TD>Initial denaturation;
</TD><TD>98Â°C
</TD><TD>30 s
</TD></TR><TR><TD>3
</TD><TD>Denaturation;
</TD><TD>98Â°C
</TD><TD>10 s
</TD></TR><TR><TD>4
</TD><TD>Annealing;
</TD><TD>70Â°C
</TD><TD>10 s
</TD></TR><TR><TD>5
</TD><TD>Elongation;
</TD><TD>72Â°C
</TD><TD>24 s
</TD></TR><TR><TD>6
</TD><TD>Go to step 3, repeat 25 x
</TD></TR><TR><TD>7
</TD><TD>Final elongation;
</TD><TD>72Â°C
</TD><TD>10 min
</TD></TR><TR><TD>8
</TD><TD>Hold
</TD><TD>4Â°C
</TD><TD>â
</TD></TR></TBODY></TABLE><P>We also transformed the biobrick &lt;partinfo&gt;BBa_E0040&lt;/partinfo&gt;, which is a protein coding part coding for GFP.
</P><H3><SPAN class="mw-headline" id="Tuesday_24.07.12">Tuesday 24.07.12</SPAN></H3><P>Transformed the pBAD+RBS+YFP+DTT (&lt;partinfo&gt;Bba_K206000&lt;/partinfo&gt;, &lt;partinfo&gt;BBa_E0030&lt;/partinfo&gt; and &lt;partinfo&gt;BBa_B0015&lt;/partinfo&gt;), VGB+RBS+YFP+DTT (&lt;partinfo&gt;BBa_K561001&lt;/partinfo&gt;, &lt;partinfo&gt;BBa_B0034&lt;/partinfo&gt;, &lt;partinfo&gt;BBa_E0030&lt;/partinfo&gt; and &lt;partinfo&gt;BBa_B0015&lt;/partinfo&gt;), and VHB+RBS+YFP+DTT (&lt;partinfo&gt;BBa_K258005&lt;/partinfo&gt;, &lt;partinfo&gt;BBa_B0034&lt;/partinfo&gt;, &lt;partinfo&gt;BBa_E0030&lt;/partinfo&gt; and &lt;partinfo&gt;BBa_B0015&lt;/partinfo&gt;) constructs ligated yesterday to supercompetent <I>E.coli</I> Dh5Î± cells.
</P><P>We also started making our iGEM matchmaker, which will be useful for iGEM teams wanting collaboration with other teams. 
</P><P>Some of our primers arrived today, so tomorrow the PCR fun will start.
</P><P>Transformed the BioBrick parts &lt;partinfo&gt;BBa_E0430&lt;/partinfo&gt; and &lt;partinfo&gt;BBa_I13522&lt;/partinfo&gt; using the iGEM 2011 DNA distribution kit.
</P><H3><SPAN class="mw-headline" id="Monday_23.07.12">Monday 23.07.12</SPAN></H3><P>We continnued the assembly of the promoter test constructs. As the test gel run on friday showed the expected fragment, the YFP+DTT (&lt;partinfo&gt;BBa_E0030&lt;/partinfo&gt; and &lt;partinfo&gt;BBa_B0015&lt;/partinfo&gt;) samples were separated using gel electrophoresis. The inserts were cut out of the gel, as shown below, and purified using the QIAquick Gel Extraction Kit. 
</P><DIV class="center"><DIV class="thumb tnone"><DIV class="thumbinner" style="width:402px;"><DIV class="thumbcaption">The picture is showing two parallels of the YFP+DTT sample, containing an insert to be used in ligation with promoter+RBS constructs. The inserts, shown in red boxes, were cut out of the gel and purified.</DIV></DIV></DIV></DIV><P>The pBad+RBS (&lt;partinfo&gt;Bba_K206000&lt;/partinfo&gt;), Jen1+RBS (&lt;partinfo&gt;BBa_K284002&lt;/partinfo&gt; and &lt;partinfo&gt;BBa_B0034&lt;/partinfo&gt;), Vgb+RBS (&lt;partinfo&gt;BBa_K561001&lt;/partinfo&gt; and &lt;partinfo&gt;BBa_B0034&lt;/partinfo&gt;) and Vhb+RBS (&lt;partinfo&gt;BBa_K258005&lt;/partinfo&gt; and &lt;partinfo&gt;BBa_B0034&lt;/partinfo&gt;) backbones were purified using the QIAquick PCR Purification Kit. In all of the above it is the plasmid from RBS that's the backbone (&lt;partinfo&gt;psB1A2&lt;/partinfo&gt;). Concentrations of both inserts and backbones after purification are given below:
</P><TABLE class="table table-bordered table-hover" style="margin: 1em auto 1em auto; width: auto;"><TBODY><TR><TH>Biobrick
</TH><TH>Restriction enzymes used in digest
</TH><TH>Concentration [ng/Âµl]
</TH></TR><TR><TD>YFP+DTT
</TD><TD>XbaI + PstI
</TD><TD>4.1
</TD></TR><TR><TD>pBad+RBS
</TD><TD>SpeI + PstI
</TD><TD>0.5
</TD></TR><TR><TD>Vgb+RBS
</TD><TD>SpeI + PstI
</TD><TD>1.7
</TD></TR><TR><TD>Vhb+RBS
</TD><TD>SpeI + PstI
</TD><TD>2.4
</TD></TR><TR><TD>Jen1+RBS
</TD><TD>SpeI + PstI
</TD><TD>1.2
</TD></TR></TBODY></TABLE><P>VGB+RBS (&lt;partinfo&gt;BBa_K561001&lt;/partinfo&gt; and &lt;partinfo&gt;BBa_B0034&lt;/partinfo&gt;), VHB+RBS (&lt;partinfo&gt;BBa_K258005&lt;/partinfo&gt; and &lt;partinfo&gt;BBa_B0034&lt;/partinfo&gt;) and pBAD+RBS (&lt;partinfo&gt;Bba_K206000&lt;/partinfo&gt;) were all ligated with YFP+DTT (&lt;partinfo&gt;BBa_E0030&lt;/partinfo&gt; and &lt;partinfo&gt;BBa_B0015&lt;/partinfo&gt;).
</P><P>For the YFP + DTT fragment, DNA concentration measurements with NanoDrop after isolation with the quiagen kit were inconsistent. For the samples made from the two gel bands, values ranged from 3.7 to 14.8 and 4.0 to 24.4 ng/uL, respectively. Conclusion: Several measurements should always be made.
</P><P>Inspected mRFP and RBS plates from yesterday. Good growth on both. No growth on negative controlplate.
</P><P>Inoculated both mRFP and RBS from the plates into 5 mL LB + Amp (21/7), placed in 37 C shaking incubator
</P><H3><SPAN class="mw-headline" id="Sunday_22.07.12">Sunday 22.07.12</SPAN></H3><P>Attempted to measure GFP fluorescence in culture inoculated yesterday with negative result.
</P><P>Inoculated two new cultures of K + RBS and K + RBS + GFP + DTT by transferring 30 uL from the cultures inoculated yesterday to 3 mL LB + Amp (medium prepared yesterday and kept in refrigerator). Attempted to measure GFP fluorescence at aprox. 2 and 4 h inoculation (OD ~ 0.2 and ~ 1.2} with negative result.
</P><P>Is the problem with the construct itself/expression of GFP from the plasmid, or with plasmid stability? All cultures tested for GFP fluorescence so far have been &quot;old&quot; (probable stationary phase) or inoculated from &quot;old&quot; cultures. Reports indicate that B-lactamase in old culture media may degrade ampicillin at a high rate and lead to decreased plasmid stability, also when a new culture is inoculated using medium from an existing culture. To rule out this being the problem, a fresh culture should be inoculated from an agar plate and tested for fluorescence during log phase. Another possibility is to inoculate from an existing culture by spinning down the cells, washing the pellet with fresh medium (to remove B-lactamase), decant the washing medium and resuspend the cells in new growth medium.
</P><P>Transformed cells with the part &lt;partinfo&gt;BBa_I13521&lt;/partinfo&gt; (pTet mRFP) from the distribution kit and the part &lt;partinfo&gt;BBa_B0034&lt;/partinfo&gt; from miniprep DNA.
</P><H3><SPAN class="mw-headline" id="Saturday_21.07.12">Saturday 21.07.12</SPAN></H3><P>Remeasured GFP and RFP fluorescence using different wavelengths and new samples. Results still negative. Re-measured YFP fluorescence on original sample retained in 96-well plate. Results were similar to initial measurement, indicating that YFP is stable for several days. 
</P><P>Inoculated two 5 mL cultures of LB with K + RBS and K + RBS + GFP + DTT respectively, from agar plates.
</P><H3><SPAN class="mw-headline" id="Friday_20.07.12">Friday 20.07.12</SPAN></H3><P>Attempted to measure fluorescence of cultures inouculated yesterdat with transformants harboring GFP and RFP under constitutive promoter (&lt;partinfo&gt;BBa_K081005&lt;/partinfo&gt;). Results were negative - neither GFP or RFP seems to be expressed.
</P><P>As the GFP and RFP constructs failed, work was started on assembling the promoters Jen1, pBAD, VGB and RBS with YFP. Plasmid DNA of JEN1+ RBS, VGB + RBS, VHB + RBS and pBAD + RBS in pSB1A2 was digested with Spe1 and Pst1. Plasmid DNA of YFP + DTT in psB1AK3 was digested with XbaI and Pst1. For YFP + DTT, digestion was performed on two samples. 4 uL of each sample digested DNA was used for gel electrophoresis to confirm the result,, the rest was stored at - 20 C. Gel visualization showed the expected fragments.
</P><H3><SPAN class="mw-headline" id="Thursday_19.07.12">Thursday 19.07.12</SPAN></H3><P>Designed new primers for the DDT terminator in pSB1A3 and ordered all primers designed today and yesterday. Contacted GATC and asked them to send us barcode stickers necessary for sequencing. They were also asked if they would like to join us as sponsors, providing us with sequencing.
</P><P>Transferred transformants with RFP and GFP constructs to liquid culture.
</P><P>Measured fluorescence in liquid culture of cells transformed with YFP (&lt;partinfo&gt;BBa_E0030&lt;/partinfo&gt;) under a constitutive promoter (&lt;partinfo&gt;BBa_K081005&lt;/partinfo&gt;). Results indicated that YFP was expressed (<A href="https://www.dropbox.com/s/6npz5uux53o5qhy/Constitutive%20YFP%20fluorescence%2019.06.xlsx" class="external text" rel="nofollow">data</A>). Due to  the measurement of high fluorescence in the reference LB medium at the indicated maximum emission wavelength 527 nm, emission was measured centered at 544 nm.
</P><P>Researched the lysis device and luxR+HSL induced promoter. The activity of the lysis device &lt;partinfo&gt;BBa_K112808&lt;/partinfo&gt; under the control of &lt;partinfo&gt;BBa_R0062&lt;/partinfo&gt; with LuxR constitutively coexpressed under TetR (&lt;partinfo&gt;BBa_R0040&lt;/partinfo&gt; has been characterized by Pasotti et al. (<A href="http://www.jbioleng.org/content/5/1/8" class="external autonumber" rel="nofollow">[1]</A>)
</P><H3><SPAN class="mw-headline" id="Wednesday_18.07.12">Wednesday 18.07.12</SPAN></H3><P>Colonies from the petri dishes of VHB+RBS+RFP+DTT, VGB+RBS+GFP+DTT, Jen1+RBS+GFP+DTT and K+RBS+YFP+DTT were transferred to liquid medium. K+RBS+GFP+DTT and K+RBS+RFP+DTT didn't have any colonies, new petri dishes were made. The old and the new were incubated at 37C.
</P><P>Inspected DTT, YFP and D2 plates. All showed good growth. Moved the plates to the frigde and discarded the old DTT; YFP and D2 plates (dates, 24/6, 20/6 and 20/6, respectively).
</P><P>Inspectd RFP and VGB plates made yesterday. Both showed good growth. No growth on negative control plates. Discarded old VGB and RFP plates (dated 25/6 and 20/6). Prepared and stored glycerol stock of RFP and VGB.
</P><P>Made a new wiki design. Designed primers for amplification of the lldR gene from K12 and the DDT terminator in pSB1A3.
</P><H3><SPAN class="mw-headline" id="Tuesday_17.07.12">Tuesday 17.07.12</SPAN></H3><P>Transformed Vhb+RBS+RFP+DTT, Vgb+RBS+GFP+DTT, Jen1+RBS+GFP+DTT and K+RBS with YFP+DTT, GFP+DTT and RFP+DTT, a control was also made. Ampicillin petri dishes were used, and they are left to incubate for the night.
</P><P>Prepared and stored glycerol stocks from liquid cultures of DTT, CFP, YFP and &quot;D2&quot; per our protocol.
Measured OD on the cultures used. Result:
DTT: 1,42
CFP: 1,44
YFP: 1,41
D2: 1,35</P><P>Performed an experiment to assess effect of induction of lysis device on the number of colony forming units (CFU). Combined the two 3 mL lysis + pBAD cultures inoculated yesterday, mixed and pipetted out 2 mL to two each of two new tubes. Added 19 uL 0,1 M arabinose (solution made 10/7) to one (induced) and equal volume dH2O to the other (control). The cultures were then incubated at 37 C with shaking. Immediately before induction, and at 30 min, 1 h, 2h and 3h after induction, 1 uL culture was pipetted out from each culture, diluted in 1000 uL H2O, the dilution vortex mixed, and 1 uL of the dilution pipetted out into 1000 uL again for a 10^-6 dilution. For each culture and time point, two LA + Amp plates were inoculated with 20 uL and 200 uL of the 10^-6 dilution, respectively. Inoculated plates were incubated at 37 C.
</P><P>OD measurements were made immediately before induction, at 30 min, 1h and 3h post-induction. Results:
Pre-induction:
Induced: 1,72
Control: 1,81
</P><P>30 min:
Induced: 1,33
Control: 1,78
</P><P>1 h:
Induced: 1,02
Control: 1,79
</P><P>3 h:
Induced: 1,00/1,09 (remeasurement, new sample)
Control: 2,50/2,45 (remeasurement, new sample)
</P><H3><SPAN class="mw-headline" id="Monday_16.07.12">Monday 16.07.12</SPAN></H3><P>Miniprepped LuxR+DTT and LuxI+DTT by using the Wizard plus SV miniprep kit from Promega. Concentrations are given below:
</P><TABLE class="table table-bordered table-hover" style="margin: 1em auto 1em auto; width: auto;"><TBODY><TR><TH>Biobrick
</TH><TH>Concentration [ng/Âµl]
</TH></TR><TR><TD>LuxI+DTT
</TD><TD>37.6
</TD></TR><TR><TD>LuxR+DTT
</TD><TD>37.0
</TD></TR></TBODY></TABLE><P>
Vhb+RBS, Vgb+RBS, Jen1+RBS, and K+RBS (&lt;partinfo&gt;BBa_K081005&lt;/partinfo&gt;), was cut with S+P, and RFP+DTT, YFP+DTT, GFP+DTT and CFP+DTT was cut with X+P. The next step will be to ligate all promoters together with a fluorescent protein, in order to test them before putting them into the final construct.
</P><P>A gel electrophoresis was performed on Vhb+RBS, Vgb+RBS, Jen1+RBS and K+RBS as well as all the fluorescent proteins + double terminator. The fluorescent protein genes were cut out and a gel extraction was performed,, as in the protocol. The promoters were purified using the PCR Purification Kit, as described in the protocol. The concentrations after purification are as follows:
</P><TABLE class="table table-bordered table-hover" style="margin: 1em auto 1em auto; width: auto;"><TBODY><TR><TH>Construct
</TH><TH>Concentration [ng/Âµl]
</TH></TR><TR><TD>YFP+DTT
</TD><TD>4.9
</TD></TR><TR><TD>GFP+DTT
</TD><TD>10.6
</TD></TR><TR><TD>RFP+DTT
</TD><TD>32.1
</TD></TR><TR><TD>CFP+DTT
</TD><TD>5.3
</TD></TR><TR><TD>K+RBS
</TD><TD>8.2
</TD></TR><TR><TD>Vhb+RBS
</TD><TD>8.1
</TD></TR><TR><TD>Vgb+RBS
</TD><TD>9.0
</TD></TR><TR><TD>Jen1+RBS
</TD><TD>10.4
</TD></TR></TBODY></TABLE><P>K+RBS were ligated together with RFP+DTT, YFP+DTT and GFP+DTT. Jen1+RBS and VGB+RBS were ligated together with GFP+DTT, and VHB+RBS were ligated together with RFP+DTT.
</P><P>Inspected the plates made yesterday. All plates showed numerous (100++) colonies or bacterial lawn. No difference between plates inouclated with culture diluted in LB and diluted in H2O was immediately appearant. Conclusion: For making new plates from saturated E. coli cultures in LB, a volume of 20 uL from a 10 000 - 100 000 dilution in H2O is probably suitable. Discarded the plates inoculated with LB-diluted culture. Placed the plates inoculated with H2O diluted culture in the refrigerator.
</P><P>Inoculated two 3 mL LB+Amp aliquots with 30 uL pBAD + lysis culture, placed in 37 C shaking incubator.
</P><P>Prepared and stored glycerol stocks of LuxR+HSL, LuxR, Lysis, pBAD, GFP and pBAD+lysis.
</P><P>Made a new stock solution (2 mL) of chloramphenicol.
</P><P>Made new La+Amp plates (27).
</P><P>Plated out YFP, CFP, DTT and &quot;D2&quot; from the cultures made 13/7. Inoculated a liquid culture of RFP from the plate made on 20/6, and pf VGB from the plate made 25/6.
</P><H3><SPAN class="mw-headline" id="Sunday_15.07.12">Sunday 15.07.12</SPAN></H3><P>Removed liquid cultures of pBAD + lysis,pBAD, LuxR, LuxR+HSL regulated promoter, GFP lysis device and LuxR hybrid promoter from incubator. RFP showed no visible growth - may have used the wrong antibiotic (amp instead of kan). 
</P><P>Plated out LuxR+HSL, Lysis, pBAD, LuxR, GFP and pBAD + lysis, as follows: 1 uL of each culture was added to both of two eppendorf tubes containing 1000 uL LB medium and 1000 uL H2O, respectively. The dilutions were vortex mixed and 20 uL of each dilution plated out on plates containing LA + amp. The plates were incubated at 37 C, while the remainder of the LB dilutions were incubated at 37 C with shaking. All other liquid cultures were placed in the refrigerator.
</P><H3><SPAN class="mw-headline" id="Saturday_14.07.12">Saturday 14.07.12</SPAN></H3><P>Miniprepped LuxI+DTT and LuxR+DTT according to protocols. An unexpected white precipitate appeared in the column after eluting the DNA from the filter in the final centrifugation. This was discarded, and the OD of the supernatant was measured with NanoDrop. The resulting concentrations were much higher than expected, 293.8 ng/Âµl for LuxR+DTT and 264.9 ng/Âµl for LuxR+DTT. 
</P><P>Due to the unexpected results, new liquid cultures were made for the two samples and the miniprep procedure will be performed again on sunday or monday.
</P><H3><SPAN class="mw-headline" id="Friday_13.07.12">Friday 13.07.12</SPAN></H3><P>Isolated plasmid DNA from the fluorescense proteins RFP (&lt;partinfo&gt;Bba_E1010&lt;/partinfo&gt;), YFP(&lt;partinfo&gt;BBa_E0030&lt;/partinfo&gt;), GFP (&lt;partinfo&gt;BBa_K082003&lt;/partinfo&gt;) and CFP (&lt;partinfo&gt;BBa_E0020&lt;/partinfo&gt;) ligated with a double terminator DTT (&lt;partinfo&gt;BBa_B0015&lt;/partinfo&gt;) in addition to the constitutive promoter (&lt;partinfo&gt;BBa_K081005&lt;/partinfo&gt;). The concentrations of the isolated samples are given below:
</P><TABLE class="table table-bordered table-hover" style="margin: 1em auto 1em auto; width: auto;"><TBODY><TR><TH>Biobrick
</TH><TH>Concentration [ng/Âµl]
</TH></TR><TR><TD>RFP
</TD><TD>40,9
</TD></TR><TR><TD>YFP
</TD><TD>55,5
</TD></TR><TR><TD>GFP
</TD><TD>41,1
</TD></TR><TR><TD>CFP
</TD><TD>47,6
</TD></TR><TR><TD>Const. Promoter
</TD><TD>19,8
</TD></TR></TBODY></TABLE><P>Further work on the test construct in J5. Also, Clone was used to simulate cutting and ligating, yielding the sequence of plasmid &lt;partinfo&gt;pSB1A2&lt;/partinfo&gt; assembled with constitutive promoter and RBS (&lt;partinfo&gt;BBa_K081005&lt;/partinfo&gt;) using the standard BioBrick assembly method outlined at <A href="http://partsregistry.org/Help:Assembly" class="external free" rel="nofollow">http://partsregistry.org/Help:Assembly</A>.
</P><P>LuxI+DTT (&lt;partinfo&gt;Bba_C0061&lt;/partinfo&gt; and &lt;partinfo&gt;BBa_B0015&lt;/partinfo&gt;) and LuxR+DTT (&lt;partinfo&gt;BBa_C0062&lt;/partinfo&gt; and &lt;partinfo&gt;BBa_B0015&lt;/partinfo&gt;) was transferred to liquid medium for overnight incubation. Miniprepping will be done tomorrow.
</P><P>We had our weekly meeting with the advisors. A very quick summary can be found in the Dropbox folder.
</P><P>Inoculated new liquid cultures from existing agar plates of E. coli DH5a cells transformed with the following BioBrick parts/constructs, in preparation for making new plates:
</P><UL><LI>pBAD + lysis construct (&lt;partinfo&gt;Bba_K206000&lt;/partinfo&gt; and &lt;partinfo&gt;BBa_K112808&lt;/partinfo&gt;)</LI><LI>pBAD strong promoter (&lt;partinfo&gt;Bba_K206000&lt;/partinfo&gt;)
</LI><LI>LuxR (&lt;partinfo&gt;BBa_C0062&lt;/partinfo&gt;)</LI><LI>LuxR+HSL regulated promoter (&lt;partinfo&gt;BBa_R0062&lt;/partinfo&gt;)</LI><LI>GFP (&lt;partinfo&gt;BBa_K082003&lt;/partinfo&gt;)</LI><LI>YFP (&lt;partinfo&gt;BBa_E0030&lt;/partinfo&gt;)</LI><LI>CFP (&lt;partinfo&gt;BBa_E0020&lt;/partinfo&gt;)</LI><LI>Hybrid promoter: HSL-LuxR activated, P22 C2 repressed (&lt;partinfo&gt;BBa_K145150&lt;/partinfo&gt;)</LI><LI>T4 lysis device (&lt;partinfo&gt;BBa_K112808&lt;/partinfo&gt;)</LI></UL><H3><SPAN class="mw-headline" id="Thursday_12.07.12">Thursday 12.07.12</SPAN></H3><P>LuxI (&lt;partinfo&gt;Bba_C0061&lt;/partinfo&gt;) and LuxR (&lt;partinfo&gt;BBa_C0062&lt;/partinfo&gt;) were cut and ligated with the DTT terminator (&lt;partinfo&gt;BBa_B0015&lt;/partinfo&gt;). LuxI+DTT and LuxR+DTT were thereafter transformed.
</P><P>Colonies of the BioBricks RFP (&lt;partinfo&gt;Bba_E1010&lt;/partinfo&gt;), YFP(&lt;partinfo&gt;BBa_E0030&lt;/partinfo&gt;), GFP (&lt;partinfo&gt;BBa_K082003&lt;/partinfo&gt;), CFP (&lt;partinfo&gt;BBa_E0020&lt;/partinfo&gt;) and the constitutive promoter (&lt;partinfo&gt;BBa_K081005&lt;/partinfo&gt;) where transferred to liquid medium with amp resistance. 
</P><P>Restriction digestion were performed on Jen1+RBS (&lt;partinfo&gt;BBa_K284002&lt;/partinfo&gt; and &lt;partinfo&gt;BBa_B0034&lt;/partinfo&gt;), VGB+RBS (&lt;partinfo&gt;BBa_K561001&lt;/partinfo&gt; and &lt;partinfo&gt;BBa_B0034&lt;/partinfo&gt;), VHB+RBS (&lt;partinfo&gt;BBa_K258005&lt;/partinfo&gt; and &lt;partinfo&gt;BBa_B0034&lt;/partinfo&gt;) and LuxR (&lt;partinfo&gt;BBa_C0062&lt;/partinfo&gt;) with the enzymes SpeI and PstI. They are now ready to be ligated together with YFP+DTT and CFP+DTT, to be tested.
</P><H3><SPAN class="mw-headline" id="Wednesday_11.07.12">Wednesday 11.07.12</SPAN></H3><P>The BioBricks RFP (&lt;partinfo&gt;Bba_E1010&lt;/partinfo&gt;), YFP(&lt;partinfo&gt;BBa_E0030&lt;/partinfo&gt;), GFP (&lt;partinfo&gt;BBa_K082003&lt;/partinfo&gt;) and CFP (&lt;partinfo&gt;BBa_E0020&lt;/partinfo&gt;) where ligated together with the double terminator DTT (&lt;partinfo&gt;BBa_B0015&lt;/partinfo&gt;), and transformed. They are incubated overnight.
</P><P>Transformed a constitiutive promoter (&lt;partinfo&gt;BBa_K081005&lt;/partinfo&gt;) which also contains RBS so that we save a cloning step in our test constructs.
</P><P>An experiment was performed to assess the functioning of the Bacteriphage T4 lysis device (&lt;partinfo&gt;BBa_K112808&lt;/partinfo&gt;). The culture of pBAD (&lt;partinfo&gt;BBa_K206000&lt;/partinfo&gt;) + lysis transformed cells inoculated yesterday was divided into two aliquots of 25 mL each in capped 50 mL plastic tubes. After removing 0.5 mL from both cultures for measurement of OD pre-induction, one aliquot was induced with 240 uL 0.1 M arabinose for a final concentration of approximately 1 mM, and the same amount of distilled water was added to the other culture. After induction, both cultures were incubated at 37 C with shaking (200 rpm). OD 600 was measured with a PerkinElmer Lambda 35 spectrometer shortly before and at 0.5, 1, 2 and 3 h after induction. At 3 h, a precipitate had developed in both cultures, more being visible in the induced culture. At this point, clearing of the induced culture was plainly visible. The precipitate interferred with pipetting during the last sampling, and a small amount was transferred to the cuvettes when measuring OD of the induced sample. This may have affected the last measurements slightly.
</P><P>The results of the OD measurements are shown below.
</P><DIV class="center"><DIV class="thumb tnone"><DIV class="thumbinner" style="width:552px;"><DIV class="thumbcaption">Measurements of OD (600 nm) in <I>E. coli</I> culture after induction of lysis device under pBAD promoter using 1 ~mM arabinose.</DIV></DIV></DIV></DIV><P>The plates inoculated with lysis induced and control culture yesterday were inspected. The lysis induced plates appeared to show less growth, but the number of colonies was very high on all plates and it was not possible to get an accurate count.
</P><P>Designed the LuxR+HSL promoter testing circuit in J5. Designed and ordered primers for amplification of the desired genes from the colicin E1 operon in <I>E. coli</I>.
</P><H3><SPAN class="mw-headline" id="Tuesday_10.07.12">Tuesday 10.07.12</SPAN></H3><P>As the restriction cutting of RFP (&lt;partinfo&gt;Bba_E1010&lt;/partinfo&gt;), YFP (&lt;partinfo&gt;BBa_E0030&lt;/partinfo&gt;), GFP (&lt;partinfo&gt;BBa_K082003&lt;/partinfo&gt;) and CFP (&lt;partinfo&gt;BBa_E0020&lt;/partinfo&gt;) yesterday turned out to be a sucsess, the rest of the samples were run on gel, and the bands containing the fluorescent proteins were cut out and purified using the QIAquick Gel Extraction Kit. Concentrations are given below:
</P><TABLE class="table table-bordered table-hover" style="margin: 1em auto 1em auto; width: auto;"><TBODY><TR><TH>Biobrick
</TH><TH>Concentration [ng/Âµl]
</TH></TR><TR><TD>RFP
</TD><TD>2,8
</TD></TR><TR><TD>YFP
</TD><TD>2,7
</TD></TR><TR><TD>GFP
</TD><TD>2,0
</TD></TR><TR><TD>CFP
</TD><TD>1,4
</TD></TR></TBODY></TABLE><P>The two parallels of the terminator (&lt;partinfo&gt;BBa_B0015&lt;/partinfo&gt;) cut yesterday with E+X was purified using the QIAquick PCR purification kit. Concentrations are given below:
</P><TABLE class="table table-bordered table-hover" style="margin: 1em auto 1em auto; width: auto;"><TBODY><TR><TH>Parallel
</TH><TH>Concentration [ng/Âµl]
</TH></TR><TR><TD>A
</TD><TD>2.1
</TD></TR><TR><TD>B
</TD><TD>1.5
</TD></TR></TBODY></TABLE><P>Test cutting of the fluorescent protein parts (RFP (&lt;partinfo&gt;Bba_E1010&lt;/partinfo&gt;), YFP (&lt;partinfo&gt;BBa_E0030&lt;/partinfo&gt;), GFP (&lt;partinfo&gt;BBa_K082003&lt;/partinfo&gt;) and CFP (&lt;partinfo&gt;BBa_E0020&lt;/partinfo&gt;)) was performed. Results are shown below:
</P><DIV class="center"><DIV class="thumb tnone"><DIV class="thumbinner" style="width:502px;"><DIV class="thumbcaption">RFP (well 2), YFP (well 3), GFP (well 5) and CFP (well 6) investigated using gel electrophoresis. The lower bands of all samples containing the DNA sequences coding for fluorescent proteins, were cut out of the gel and purified.</DIV></DIV></DIV></DIV><P>We confirmed that the lysis device part (&lt;partinfo&gt;BBa_K112808&lt;/partinfo&gt;) we have extracted from the distribution kit is functional, by testing the construct with the pBAD strong promoter (&lt;partinfo&gt;Bba_K206000&lt;/partinfo&gt;) and lysis device assemmbled together. The device was activated by inducing the pBAD promoter with arabinose. Liquid culture of cells transformed with the construct was grown overnight, and 1 mM arabinose added to a 1 mL sample of the culture. After incubation at 37 C for 3 h, the induced sample was significantly clearer compared to a non-induced negative control sample, indicating lysis.
</P><DIV class="center"><DIV class="thumb tnone"><DIV class="thumbinner" style="width:502px;"><DIV class="thumbcaption">Results from testing of lysis device coupled to the arabinose induced promoter pBAD. Arabinose was added to the sample to the left. The sample to the right is the control sample.</DIV></DIV></DIV></DIV><P>The absorbance of the samples at 600 nm were measured with a Lambda 35 spectrometer, using LB medium as reference. The measured values for undiluted samples were:</P><UL><LI>Induced sample: 0,3584</LI><LI>Control sample: 1,6498
</LI></UL><P>Note that the measured absorbance of the control sample is outside of the range of 0.1-0.9 that is considered to not require dilution. Above this range, samples should be diluted to ensure that Beer's law applies. As measured, this absorbance is therefore probably too low.
</P><P>In order to make a more quantitative assesment of the extent of lysis, cells were plated out from both the induced and control samples, as follows: 2, 20 and 200 uL of each on LA + Amp plates, for a total of six plates.
</P><P>0.5 mL of culture inoculated yesterday (sample A) was transferred to 50 mL fresh LB + Amp in an sterile Erlenmeyer flask and incubated at 37 C with shaking.
</P><P>
The constructs dld+RBS (&lt;partinfo&gt;BBa_K284003&lt;/partinfo&gt; and &lt;partinfo&gt;BBa_B0034&lt;/partinfo&gt;), Jen1+RBS (&lt;partinfo&gt;BBa_K284002&lt;/partinfo&gt; and &lt;partinfo&gt;BBa_B0034&lt;/partinfo&gt;), VGB+RBS (&lt;partinfo&gt;BBa_B0034&lt;/partinfo&gt; and &lt;partinfo&gt;BBa_B0034&lt;/partinfo&gt;) and VHB+RBS (&lt;partinfo&gt;BBa_K258005&lt;/partinfo&gt; and &lt;partinfo&gt;BBa_B0034&lt;/partinfo&gt;) where all cut once with XbaI and SpeI, and once with an enzyme that has two seats where they digest. Which enzyme used for which construct can be found in the table below.
</P><TABLE class="table table-bordered table-hover" style="margin: 1em auto 1em auto; width: auto;"><TBODY><TR><TH>Construct
</TH><TH>Double digest enzyme
</TH></TR><TR><TD>dld+RBS
</TD><TD>Eco57I (AcuI)
</TD></TR><TR><TD>jen1+RBS
</TD><TD>ApaLI
</TD></TR><TR><TD>VGB+RBS
</TD><TD>ApaLI
</TD></TR><TR><TD>VHB+RBS
</TD><TD>ApaLI
</TD></TR></TBODY></TABLE><P>The Figure below shows the gel electrophoresis for the test cutting. Green circles indicate bands that were expected, red circles indicate unexpected bands or expected bands that were missing. The yellow circles indicate bands that might be correct, but not conclusive.
</P><DIV class="center"><DIV class="thumb tnone"><DIV class="thumbinner" style="width:402px;"><DIV class="thumbcaption">Dld+RBS (well 1+5), Jen1+RBS (well 2+6), VGB+RBS (well 3+7) and VHB+RBS (well 4+8) were test cut using X+S (well 1-4),Eco57I (well 5) and ApaLI (well 6-8). They were then studied using gel electrophoresis.</DIV></DIV></DIV></DIV><H3><SPAN class="mw-headline" id="Monday_09.07.12">Monday 09.07.12</SPAN></H3><P>Miniprepped pBAD+lysis, VHB+RBS, VGB+RBS, JEN1+RBS and DLD+RBS. Concentrations were measured using NanoDrop and were as follows:
</P><UL><LI> pBAD+lysis: 23.5 ng/ÂµL
</LI><LI> VHB+RBS: 48.3 ng/ÂµL
</LI><LI> VGB+RBS: 40.0 ng/ÂµL
</LI><LI> JEN1+RBS: 35.3 ng/ÂµL
</LI><LI> DLD+RBS: 24.1 ng/ÂµL
</LI></UL><P>All the segments mentioned above except pBAD+lysis were subsequently tested by digestion with restriction enzymes followed by gel electrophoresis. All plasmids were cut using XbaI and SpeI as well as one other enzyme having exactly two restriction sites on the plasmid. BciVI was used on VGB+RBS, ApaLI on JEN1+RBS and Eco57I on the remaining two plasmids, DLD+RBS and VHB+RBS.
</P><P>After cutting, gel electrophoresis was performed. After running for 45 minutes, the results were as follows:
</P><DIV class="center"><DIV class="thumb tnone"><DIV class="thumbinner" style="width:452px;"><DIV class="thumbcaption">Gel showing results from test cutting after 45 minutes. Green circles indicate bands that were expected, red circles indicate unexpected bands or expected bands that were missing. The yellow circle indicates a band that is probably not the one we expected.</DIV></DIV></DIV></DIV><P>Inoculated one colony of the pBAD + lysis transformants into two tubes with 10 mL LB + Amp each, using an inoculating needle.
</P><P>Performed restriction digestion with EcoRI and SpeI using the standard protocol on the following DNA samples:
</P><UL><LI> CFP (&lt;partinfo&gt;BBa_E0020&lt;/partinfo&gt;)
</LI><LI> YFP (&lt;partinfo&gt;BBa_E0030&lt;/partinfo&gt;)
</LI><LI> RFP (&lt;partinfo&gt;Bba_E1010&lt;/partinfo&gt;)
</LI><LI> GFP (+ LVA) (&lt;partinfo&gt;BBa_K082003&lt;/partinfo&gt;)
</LI></UL><P>DNA from the following samples were digested with EcoRI and XbaI:
</P><UL><LI> Double transcriptional terminator (DTT) &lt;partinfo&gt;BBa_B0015&lt;/partinfo&gt; [Amp] (Miniprep DNA from culture inoculated from colony on Amp plate]
</LI><LI> Double transcriptional terminator (DTT) &lt;partinfo&gt;BBa_B0015&lt;/partinfo&gt; [Kan] (Miniprep DNA from culture inoculated from colony on Kan plate) 
</LI></UL><P>After digestion, the results were visualized by gel electrophoresis, using 4 ÂµL from each digestion mixture. Based on the gel visualization, all the digestions appear to have been sucessful.
</P><DIV class="center"><DIV class="thumb tnone"><DIV class="thumbinner" style="width:302px;"><DIV class="thumbcaption">Gel showing results from test cutting of DTT.</DIV></DIV></DIV></DIV><P>We performed restriction digest <A href="https://2012.igem.org/Team:NTNU_Trondheim/Protocols" class="external autonumber" rel="nofollow">[2]</A> on a constitutive promoter &lt;partinfo&gt;BBa_J23119&lt;/partinfo&gt; an Lux I &lt;partinfo&gt;BBa_C0061&lt;/partinfo&gt;. The constitutive promoter was cut with the restriction enzymes SpeI and PstI. Lux I was cut with the restriction enzymes EcoRI and XbaI.
</P><P>The samples were run on gel, and the correct bands were cut out and extracted.Consentrations are given in the table below:
</P><TABLE class="table table-bordered table-hover" style="margin: 1em auto 1em auto; width: auto;"><TBODY><TR><TH>Parallel
</TH><TH>Concentration [ng/Âµl]
</TH></TR><TR><TD>BBa_J23119
</TD><TD>2,6
</TD></TR><TR><TD>BBa_C0061
</TD><TD>0,9
</TD></TR></TBODY></TABLE><H3><SPAN class="mw-headline" id="Saturday_07.07.12">Saturday 07.07.12</SPAN></H3><P>The ligated constructs of Jen1 (&lt;partinfo&gt;BBa_K284002&lt;/partinfo&gt;) + RBS (&lt;partinfo&gt;BBa_B0034&lt;/partinfo&gt;), Dld (&lt;partinfo&gt;BBa_K284003&lt;/partinfo&gt;) + RBS (&lt;partinfo&gt;BBa_B0034&lt;/partinfo&gt;), VHb (&lt;partinfo&gt;BBa_K258005&lt;/partinfo&gt;) + RBS (&lt;partinfo&gt;BBa_B0034&lt;/partinfo&gt;) VGB (&lt;partinfo&gt;BBa_K561001&lt;/partinfo&gt;)+ RBS (&lt;partinfo&gt;BBa_B0034&lt;/partinfo&gt;)and pBAD (&lt;partinfo&gt;BBa_K206000&lt;/partinfo&gt;) + lysis &lt;partinfo&gt;BBa_K112808&lt;/partinfo&gt; were transferred from plates to liquid media and placed in cabinet for incubation. 
</P><P>The the religation plate with only the backbone plasmid did also have som growth which is not what we want. We still decided to continue on with the promoter constructs. The religation of the pBad plasmid did not show any growth.
</P><H3><SPAN class="mw-headline" id="Friday_06.07.12">Friday 06.07.12</SPAN></H3><P>Transformed pBAD-lysis and pBAD religated.
</P><P>A gel electrophoresis was run on the BioBricks digested yesterday (Jen1 &lt;partinfo&gt;BBa_K284002&lt;/partinfo&gt;, Dld &lt;partinfo&gt;BBa_K284003&lt;/partinfo&gt; and VHb &lt;partinfo&gt;BBa_K258005&lt;/partinfo&gt;). The results were the same as the first gel electrophoresis, so no gel extraction was performed. The purified DNA from Thursday 05.07.12 was used further on.
</P><P>Jen1, Dld, Vgb, and Vhb cut with EcoRI and SpeI were ligated together with RBS cut with EcoRI and XbaI, and the ligated plasmids were transformed to competent Dh5Î± cells.
</P><H3><SPAN class="mw-headline" id="Thursday_05.07.12">Thursday 05.07.12</SPAN></H3><P>The gel electrophoresis of Jen1 (&lt;partinfo&gt;BBa_K284002&lt;/partinfo&gt;), Dld (&lt;partinfo&gt;BBa_K284003&lt;/partinfo&gt;) and VHb (&lt;partinfo&gt;BBa_K258005&lt;/partinfo&gt;) was analyzed. Only two bands per column were expected, insert and backbone, but there was at least one extra at each column. The length of the genes Jen1, Dld and VHb are 998 bp, 463 bp and 137 bp, respectively, so the bands closest to the given sizes were cut out of the gel and purified using a QIAquick Gel Extraction kit. The concentrations of the purified genes are listed below:
</P><UL><LI> Dld promoter: 2,7 ng/ÂµL
</LI><LI> Jen1 promoter: 1,6 ng/ÂµL
</LI><LI> VHb promoter: 4,1 ng/ÂµL
</LI></UL><P>Since there were so many bands per column in the gel electrophoresis for Jen1 (&lt;partinfo&gt;BBa_K284002&lt;/partinfo&gt;), Dld (&lt;partinfo&gt;BBa_K284003&lt;/partinfo&gt;) and VHb (&lt;partinfo&gt;BBa_K258005&lt;/partinfo&gt;), the procedure will be done one more time. They were cut, as described in the <A href="https://2012.igem.org/Team:NTNU_Trondheim/Protocols" class="external text" rel="nofollow">protocols</A>, but with half the concentration of DNA.
</P><P>Gel electrophoresis was also performed on the lysis device and pBAD promoter. The expected sizes of the DNA sequences were 2079 bp for the lysis plasmid, 1785 bp for the lysis device and about 2000 bp for the pBAD promoter with backbone. This corresponds well with the observed bands. Pictures of the gels obtained from the two electrophoresis runs are included below. On both gels, the far left and far right wells contain GeneRuler 1kb DNA ladder.
</P><TABLE style="text-align:center; margin: 1em auto 1em auto; padding: 0 10px;" cellpadding="10px"><TBODY><TR><TD><DIV class="center"><DIV class="thumb tnone"><DIV class="thumbinner" style="width:452px;"><DIV class="thumbcaption">Results from gel electrophoresis with DLD, JEN1 and VHB.</DIV></DIV></DIV></DIV></TD><TD><DIV class="center"><DIV class="thumb tnone"><DIV class="thumbinner" style="width:452px;"><DIV class="thumbcaption">Results from gel electrophoresis with lysis device and pBAD promoter.</DIV></DIV></DIV></DIV></TD></TR></TBODY></TABLE><P>The DNA segment containing the VGB promoter was extracted from another gel according to <A href="https://2012.igem.org/Team:NTNU_Trondheim/Protocols" class="external text" rel="nofollow">protocols</A>. The resulting DNA concentration was measured twice, once to 17.4 ng/ÂµL and once to 19.6 ng/ÂµL.
</P><H3><SPAN class="mw-headline" id="Wednesday_04.07.12">Wednesday 04.07.12</SPAN></H3><P>The BioBricks colicin (&lt;partinfo&gt;BBa_K150009&lt;/partinfo&gt;), dld-promoter (&lt;partinfo&gt;BBa_K284003&lt;/partinfo&gt;), jen1 (&lt;partinfo&gt;BBa_K284002&lt;/partinfo&gt;) and VHb-promoter (&lt;partinfo&gt;BBa_K258005&lt;/partinfo&gt;) were miniprepped as described in the <A href="/Team:NTNU_Trondheim/Protocols" title="Team:NTNU Trondheim/Protocols">protocol</A>. The concentration results are listed below:
</P><UL><LI> Colicin: 58,6 ng/ÂµL
</LI><LI> Dld-promoter: 63,8 ng/ÂµL
</LI><LI> Jen1-promoter: 102,1 ng/ÂµL
</LI><LI> VHb (oxygen-promoter): 44,9 ng/ÂµL
</LI></UL><P>The luxI BioBrick &lt;partinfo&gt;BBa_C0061&lt;/partinfo&gt; was transformed. Continued work on circuit assembly using J5.
</P><P>Restriction digestion was performed on dld-promoter (&lt;partinfo&gt;BBa_K284003&lt;/partinfo&gt;), jen1-promoter (&lt;partinfo&gt;BBa_K284002&lt;/partinfo&gt;) and VHb-promoter (&lt;partinfo&gt;BBa_K258005&lt;/partinfo&gt;) as described in the <A href="/Team:NTNU_Trondheim/Protocols" title="Team:NTNU Trondheim/Protocols">protocol</A>. All of them cut with E and S.
</P><P>Restriction digest were also performed for the vgb-promoter (E + S) and RBS (E + X), and gel electrophoresis was performed to verify that the correct DNA fragment were made during digestion. The RBS digestion sample was purified using a QIAquick PCR Purification kit. Using this kit will cause small DNA fragments, like the prefix fragment between the EcoRI and XbaI sites, to be lost. All of the vgb sample was run on gel, to separate the insert to be further used, from the backbone.
</P><P>Performed restriction digest on the pBAD (&lt;partinfo&gt;BBa_K206000&lt;/partinfo&gt;) and lysis &lt;partinfo&gt;BBa_K112808&lt;/partinfo&gt; parts, cutting pBAD with SpeI and PstI, and the lysis part with XbaI and PstI.
</P><H3><SPAN class="mw-headline" id="Tuesday_03.07.12">Tuesday 03.07.12</SPAN></H3><P>Wiki design updated. <A href="http://j5.jbei.org/" class="external text" rel="nofollow">J5</A> was used to determine how to assemble the genetic circuit. The total size of the complete plasmid containg all the parts of our system will probably be 8000-9000 bp ((depending on the size of the backbone), of which the BioBricks make up about 6500 bp.
</P><P>Also, it was discovered that the colicin BioBrick (&lt;partinfo&gt;BBa_K150009&lt;/partinfo&gt;) consists of a number of DNA segments, not only genes for the necessary colE1 proteins. Among these are the luxR gene and the LuxR-HSL promoter as well as a lysis-inducing component. This means that inserting the entire BioBrick into our circuit will be problematic, as the LuxR production and LuxR-HSL sensitivity of the construct will interfere with the oxygen promoter and the rest of the luxR components in our circuit. The problem can probably be circumvented if we amplify the genes coding for the two colE1 proteins and connect them to a constitutive promoter.
</P><P>Dld (&lt;partinfo&gt;BBa_K284003&lt;/partinfo&gt;), Jen1 (&lt;partinfo&gt;BBa_K284002&lt;/partinfo&gt;) and VHB (&lt;partinfo&gt;BBa_K258005&lt;/partinfo&gt;) were transferred to liquid medium.
</P><H3><SPAN class="mw-headline" id="Monday_02.07.12">Monday 02.07.12</SPAN></H3><P>Restriction digestion was performed on isolated DNA from the VGB and RBS biobricks (&lt;partinfo&gt;BBa_k561001&lt;/partinfo&gt; and &lt;partinfo&gt;BBa_B0034&lt;/partinfo&gt;) according to <A href="/Team:NTNU_Trondheim/Protocols" title="Team:NTNU Trondheim/Protocols">protocols</A>. Gel electrophoresis was used to determine the size of the resulting fragments. The RBS restriction digest was filtered using the QIAquick PCR purificat kit. The results for VGB were not as expected, and restriction digest of this part was therefore performed again, this time with addition of <A href="http://en.wikipedia.org/wiki/Calf-intestinal_alkaline_phosphatase" class="external text" rel="nofollow">CIP</A> to the digest solution at the end of the procedure.
</P><P>Dld (&lt;partinfo&gt;BBa_K284003&lt;/partinfo&gt;), Jen1 (&lt;partinfo&gt;BBa_K284002&lt;/partinfo&gt;) and VHB (&lt;partinfo&gt;BBa_K258005&lt;/partinfo&gt;) were transformed as described in <A href="/Team:NTNU_Trondheim/Protocols" title="Team:NTNU Trondheim/Protocols">protocols</A>.
</P><H3><SPAN class="mw-headline" id="Sunday_01.07.12">Sunday 01.07.12</SPAN></H3><P>Miniprepped all three tubes with pBAD inoculated yesterday. Tube 3 contained 10 mL culture and was divided into two tubes, so that all processed tubes contained ~5 mL culture. Results of DNA measurements were as follows (ng/uL):</P><UL><LI> pBAD B1: 24,0
</LI><LI> pBAD B2: 18,9
</LI><LI> pBAD B3-1: 11,4
</LI><LI> pBAD B3-2: 8,1
</LI></UL><P>Discarded B3-1 and B3-2, placed B1 and B2 in -20 C freezer.
</P><P>In order to make a new plate of Lysis 1, transferred 20 uL from one of the tubes with liquid culture inoculated yesterday to one LA + Amp plate, and 50 uL to another. Placed the plates in the 37 C incubator cabinet. Also transferred 50 uL as inoculate to the tube used as negative control yesterday and placed it in the shaking incubator. Then combined all three tubes into one and placed in refrigerator to limit cell death before miniprep tomorrow.
</P></DIV></DIV></DIV><FOOTER class="footer"><DIV class="container"><P>Designed and built with <A href="http://www.mediawiki.org/wiki/MediaWiki" target="_blank">MediaWiki</A>, <A href="http://jquery.com/" target="_blank">jQuery</A> and <A href="http://twitter.github.com/bootstrap/" target="_blank">Twitter Bootstrap</A>.</P><P>Icons by <A href="http://http://fortawesome.github.com/Font-Awesome/" target="_blank">Font Awesome</A>.</P><P><A href="mailto:igem.ntnu@gmail.com"> igem.ntnu@gmail.com</A></P></DIV></FOOTER><DIV class="printfooter">
Retrieved from &quot;<A href="http://2012.igem.org/Team:NTNU_Trondheim/Notebook/July">http://2012.igem.org/Team:NTNU_Trondheim/Notebook/July</A>&quot;</DIV><DIV id="footer-box" class="noprint"><DIV id="footer"><UL id="f-list"><LI id="t-recentchanges"><A href="/Special:RecentChanges" title="Recent changes">Recent changes</A></LI><LI id="t-whatlinkshere"><A href="/Special:WhatLinksHere/Team:NTNU_Trondheim/Notebook/July" title="List of all wiki pages that link here [j]" accesskey="j">What links here</A></LI><LI id="t-recentchangeslinked"><A href="/Special:RecentChangesLinked/Team:NTNU_Trondheim/Notebook/July" title="Recent changes in pages linked from this page [k]" accesskey="k">Related changes</A></LI><LI id="t-specialpages"><A href="/Special:SpecialPages" title="List of all special pages [q]" accesskey="q">Special pages</A></LI><LI><A href="/Special:Preferences">My preferences</A></LI></UL></DIV><DIV id="footer"><UL id="f-list"><LI id="t-print"><A href="/wiki/index.php?title=Team:NTNU_Trondheim/Notebook/July&amp;printable=yes" title="Printable version of this page [p]" accesskey="p">Printable version</A></LI><LI id="t-permalink"><A href="/wiki/index.php?title=Team:NTNU_Trondheim/Notebook/July&amp;oldid=176544" title="Permanent link to this revision of the page">Permanent link</A></LI><LI id="privacy"><A href="/2012.igem.org:Privacy_policy" title="2012.igem.org:Privacy policy">Privacy policy</A></LI><LI id="disclaimer"><A href="/2012.igem.org:General_disclaimer" title="2012.igem.org:General disclaimer">Disclaimers</A></LI></UL></DIV></DIV></BODY></HTML>