<HTML xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en" dir="ltr">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki  ltr ns-0 ns-subject page-Team_NTNU_Trondheim_Notebook_August"><DIV id="globalWrapper"><DIV id="top-section"><DIV id="p-logo"><A href="/Main_Page" title="Main Page">&quot;
	    </A></DIV><DIV id="menubar" class="left-menu noprint"><UL><LI class="selected"><A href="/Team:NTNU_Trondheim/Notebook/August">Page               </A></LI><LI class="new"><A href="/wiki/index.php?title=Talk:Team:NTNU_Trondheim/Notebook/August&amp;action=edit&amp;redlink=1">Discussion               </A></LI><LI><A href="/wiki/index.php?title=Team:NTNU_Trondheim/Notebook/August&amp;action=edit">View source               </A></LI><LI><A href="/wiki/index.php?title=Team:NTNU_Trondheim/Notebook/August&amp;action=history">History               </A></LI><LI style="color:#808080;cursor:default">teams</LI></UL></DIV><DIV class="right-menu noprint" id="menubar"><UL><LI id="pt-login"><A href="/wiki/index.php?title=Special:UserLogin&amp;returnto=Team:NTNU_Trondheim/Notebook/August" title="You are encouraged to log in; however, it is not mandatory [o]" accesskey="o">Log in</A></LI></UL></DIV><DIV id="search-controls" class="noprint"><FORM action="/Special:Search" id="searchform">Â </FORM></DIV></DIV><DIV id="content"><H1 class="firstHeading">Team:NTNU Trondheim/Notebook/August</H1><DIV id="bodyContent"><H3 id="siteSub" class="noprint">From 2012.igem.org</H3></DIV></DIV></DIV><DIV class="header"><DIV class="container"><DIV class="heading visible-tablet visible-desktop"><A href="https://2012.igem.org/Team:NTNU_Trondheim">Search and Destroy</A></DIV><DIV class="heading heading-small visible-phone"><A href="https://2012.igem.org/Team:NTNU_Trondheim">Search and Destroy</A></DIV><DIV class="sub-heading visible-tablet visible-desktop" id="sub-heading1"><SPAN class="sub-heading-small">NTNU IS</SPAN><SPAN style="font-size: 30pt; font-style: italic;">B.A.C.K.</SPAN></DIV><DIV class="sub-heading visible-desktop" id="sub-heading2">
            B<SPAN class="sub-heading-small">acterial</SPAN> A<SPAN class="sub-heading-small">nti</SPAN>-C<SPAN class="sub-heading-small">ancer</SPAN>-K<SPAN class="sub-heading-small">amikaze</SPAN></DIV></DIV></DIV><DIV class="navbar navbar-below-header"><DIV class="navbar-inner"><DIV class="container"><DIV class="nav-collapse"><UL class="nav"><LI><A href="https://2012.igem.org/Team:NTNU_Trondheim"> Home</A></LI><LI class="dropdown"><A href="#" class="dropdown-toggle" data-toggle="dropdown">Project </A></LI><LI><A href="https://2012.igem.org/Team:NTNU_Trondheim/Project">Project description</A></LI><LI><A href="https://2012.igem.org/Team:NTNU_Trondheim/Model">Modelling</A></LI><LI><A href="https://2012.igem.org/Team:NTNU_Trondheim/Experiments_and_Results">Experiments and results</A></LI><LI><A href="https://2012.igem.org/Team:NTNU_Trondheim/Parts">BioBrick Parts</A></LI><LI><A href="https://2012.igem.org/Team:NTNU_Trondheim/Attributions">Attributions</A></LI></UL><LI class="dropdown"><A href="#" class="dropdown-toggle" data-toggle="dropdown"> Technical stuff </A></LI><LI><A href="https://2012.igem.org/Team:NTNU_Trondheim/Notebook">Notebook</A></LI><LI><A href="https://2012.igem.org/Team:NTNU_Trondheim/Protocols">Protocols</A></LI><LI><A href="https://2012.igem.org/Team:NTNU_Trondheim/Safety">Safety</A></LI><LI class="dropdown"><A href="#" class="dropdown-toggle" data-toggle="dropdown"> Team </A></LI><LI><A href="https://2012.igem.org/Team:NTNU_Trondheim/Team">Meet the team</A></LI><LI><A href="https://igem.org/Team.cgi?year=2012&amp;team_name=NTNU_Trondheim" target="_blank">Official team profile</A></LI><LI class="dropdown"><A href="#" class="dropdown-toggle" data-toggle="dropdown"> Outreach </A></LI><LI><A href="https://2012.igem.org/Team:NTNU_Trondheim/Collaboration">Collaboration</A></LI><LI><A href="https://2012.igem.org/Team:NTNU_Trondheim/Outreach">Human practices</A></LI><LI><A href="https://2012.igem.org/Team:NTNU_Trondheim/Press">Press coverage</A></LI><LI><A href="https://2012.igem.org/Team:NTNU_Trondheim/Achievements"> Achievements</A></LI><LI class="hidden-navbar"><A href="https://2012.igem.org/Team:NTNU_Trondheim/Matchmaker" target="_blank"> Matchmaker</A></LI></DIV></DIV></DIV></DIV><DIV class="container"><DIV class="page-header-top"><H1>Notebook <SMALL>August</SMALL></H1></DIV></DIV><DIV class="container main-container"><DIV class="row"><DIV class="span10 offset1"><DIV class="well well-small"><TABLE align="center" border="0" style="font-size: 12px;" cellpadding="15"><TBODY><TR valign="top"><TD><TABLE align="center" style="text-align:right;" cellpadding="3"><CAPTION><B><A href="https://2012.igem.org/Team:NTNU_Trondheim/Notebook/February" class="external text" rel="nofollow">February</A></B></CAPTION></TABLE></TD></TR><TR><TD> Mon
</TD><TD> Tue
</TD><TD> Wed
</TD><TD> Thu
</TD><TD> Fri
</TD><TD> Sat
</TD><TD> Sun
</TD></TR><TR><TD>1
</TD><TD>2
</TD><TD>3
</TD><TD>4
</TD><TD>5
</TD></TR><TR><TD>6
</TD><TD>7
</TD><TD>8
</TD><TD>9
</TD><TD>10
</TD><TD>11
</TD><TD>12
</TD></TR><TR><TD>13
</TD><TD>14
</TD><TD>15
</TD><TD>16
</TD><TD><A href="https://2012.igem.org/Notebook/February#Friday_17.02.12" class="external text" rel="nofollow">17</A></TD><TD>18
</TD><TD>19
</TD></TR><TR><TD>20
</TD><TD>21
</TD><TD>22
</TD><TD>23
</TD><TD>24
</TD><TD>25
</TD><TD>25
</TD></TR><TR><TD><A href="https://2012.igem.org/Notebook/February#Monday_27.02.12" class="external text" rel="nofollow">27</A></TD><TD>28
</TD><TD>29
</TD></TR></TBODY></TABLE><TABLE align="center" style="text-align:right;" cellpadding="3"><CAPTION><B><A href="https://2012.igem.org/Team:NTNU_Trondheim/Notebook/March" class="external text" rel="nofollow">March</A></B></CAPTION><TBODY><TR><TD> Mon
</TD><TD> Tue
</TD><TD> Wed
</TD><TD> Thu
</TD><TD> Fri
</TD><TD> Sat
</TD><TD> Sun
</TD></TR><TR><TD>1
</TD><TD>2
</TD><TD>3
</TD><TD>4
</TD></TR><TR><TD>5
</TD><TD>6
</TD><TD>7
</TD><TD>8
</TD><TD>9
</TD><TD>10
</TD><TD>11
</TD></TR><TR><TD>12
</TD><TD>13
</TD><TD>14
</TD><TD>15
</TD><TD><A href="https://2012.igem.org/Team:NTNU_Trondheim/Notebook/March#Friday_16.03.12" class="external text" rel="nofollow">16</A></TD><TD>17
</TD><TD>18
</TD></TR><TR><TD>19
</TD><TD>20
</TD><TD>21
</TD><TD>22
</TD><TD>23
</TD><TD>24
</TD><TD>25
</TD></TR><TR><TD>26
</TD><TD>27
</TD><TD>28
</TD><TD>29
</TD><TD><A href="https://2012.igem.org/Team:NTNU_Trondheim/Notebook/March#Friday_30.03.12" class="external text" rel="nofollow">30</A></TD></TR></TBODY></TABLE><TABLE align="center" style="text-align:right;" cellpadding="3"><CAPTION><B><A href="https://2012.igem.org/Team:NTNU_Trondheim/Notebook/April" class="external text" rel="nofollow">April</A></B></CAPTION><TBODY><TR><TD> Mon
</TD><TD> Tue
</TD><TD> Wed
</TD><TD> Thu
</TD><TD> Fri
</TD><TD> Sat
</TD><TD> Sun
</TD></TR><TR><TD>1
</TD></TR><TR><TD>2
</TD><TD>3
</TD><TD>4
</TD><TD>5
</TD><TD>6
</TD><TD>7
</TD><TD>8
</TD></TR><TR><TD>9
</TD><TD>10
</TD><TD>11
</TD><TD>12
</TD><TD><A href="https://2012.igem.org/Team:NTNU_Trondheim/Calendar/April#Friday_13.04.12" class="external text" rel="nofollow">13</A></TD><TD>14
</TD><TD>15
</TD></TR><TR><TD>16
</TD><TD>17
</TD><TD>18
</TD><TD>19
</TD><TD><A href="https://2012.igem.org/Team:NTNU_Trondheim/Calendar/April#Friday_20.04.12" class="external text" rel="nofollow">20</A></TD><TD>21
</TD><TD>22
</TD></TR><TR><TD>23
</TD><TD>24
</TD><TD>25
</TD><TD><A href="https://2012.igem.org/Team:NTNU_Trondheim/Calendar/April#Thursday_26.04.12" class="external text" rel="nofollow">26</A></TD><TD>27
</TD><TD>28
</TD><TD>29
</TD></TR><TR><TD>30
</TD></TR></TBODY></TABLE><TABLE align="center" style="text-align:right;" cellpadding="3"><CAPTION><B><A href="https://2012.igem.org/Team:NTNU_Trondheim/Notebook/May" class="external text" rel="nofollow">May</A></B></CAPTION><TBODY><TR><TD> Mon
</TD><TD> Tue
</TD><TD> Wed
</TD><TD> Thu
</TD><TD> Fri
</TD><TD> Sat
</TD><TD> Sun
</TD></TR><TR><TD>1
</TD><TD><A href="https://2012.igem.org/Team:NTNU_Trondheim/Notebook/May#Wednesday_02.05.12" class="external text" rel="nofollow">2</A></TD><TD><A href="https://2012.igem.org/Team:NTNU_Trondheim/Notebook/May#Thursday_03.05.12" class="external text" rel="nofollow">3</A></TD><TD>4
</TD><TD>5
</TD><TD>6
</TD></TR><TR><TD>7
</TD><TD>8
</TD><TD><A href="https://2012.igem.org/Team:NTNU_Trondheim/Notebook/May#Wednesday_09.05.12" class="external text" rel="nofollow">9</A></TD><TD>10
</TD><TD><A href="https://2012.igem.org/Team:NTNU_Trondheim/Notebook/May#Friday_11.05.12" class="external text" rel="nofollow">11</A></TD><TD>12
</TD><TD>13
</TD></TR><TR><TD>14
</TD><TD>15
</TD><TD>16
</TD><TD>17
</TD><TD><A href="https://2012.igem.org/Team:NTNU_Trondheim/Notebook/May#Friday_18.05.12" class="external text" rel="nofollow">18</A></TD><TD>19
</TD><TD>20
</TD></TR><TR><TD>21
</TD><TD>22
</TD><TD><A href="https://2012.igem.org/Team:NTNU_Trondheim/Notebook/May#Wednesday_23.05.12" class="external text" rel="nofollow">23</A></TD><TD>24
</TD><TD><A href="https://2012.igem.org/Team:NTNU_Trondheim/Notebook/May#Friday_25.05.12" class="external text" rel="nofollow">25</A></TD><TD>26
</TD><TD>27
</TD></TR><TR><TD>28
</TD><TD>29
</TD><TD>30
</TD><TD>31
</TD></TR></TBODY></TABLE><TABLE align="center" style="text-align:right;" cellpadding="3"><CAPTION><B><A href="https://2012.igem.org/Team:NTNU_Trondheim/Notebook/June" class="external text" rel="nofollow">June</A></B></CAPTION><TBODY><TR><TD> Mon
</TD><TD> Tue
</TD><TD> Wed
</TD><TD> Thu
</TD><TD> Fri
</TD><TD> Sat
</TD><TD> Sun
</TD></TR><TR><TD><A href="https://2012.igem.org/Team:NTNU_Trondheim/Notebook/June#Friday_01.06.12" class="external text" rel="nofollow">1</A></TD><TD>2
</TD><TD>3
</TD></TR><TR><TD>4
</TD><TD>5
</TD><TD>6
</TD><TD>7
</TD><TD>8
</TD><TD>9
</TD><TD>10
</TD></TR><TR><TD>11
</TD><TD>12
</TD><TD><A href="https://2012.igem.org/Team:NTNU_Trondheim/Notebook/June#Wednesday_13.06.12" class="external text" rel="nofollow">13</A></TD><TD>14
</TD><TD>15
</TD><TD>16
</TD><TD>17
</TD></TR><TR><TD><A href="https://2012.igem.org/Team:NTNU_Trondheim/Notebook/June#Monday_18.06.12" class="external text" rel="nofollow">18</A></TD><TD><A href="https://2012.igem.org/Team:NTNU_Trondheim/Notebook/June#Tuesday_19.06.12" class="external text" rel="nofollow">19</A></TD><TD><A href="https://2012.igem.org/Team:NTNU_Trondheim/Notebook/June#Wednesday_20.06.12" class="external text" rel="nofollow">20</A></TD><TD><A href="https://2012.igem.org/Team:NTNU_Trondheim/Notebook/June#Thursday_21.06.12" class="external text" rel="nofollow">21</A></TD><TD><A href="https://2012.igem.org/Team:NTNU_Trondheim/Notebook/June#Friday_22.06.12" class="external text" rel="nofollow">22</A></TD><TD><A href="https://2012.igem.org/Team:NTNU_Trondheim/Notebook/June#Saturday_23.06.12" class="external text" rel="nofollow">23</A></TD><TD><A href="https://2012.igem.org/Team:NTNU_Trondheim/Notebook/June#Sunday_24.06.12" class="external text" rel="nofollow">24</A></TD></TR><TR><TD><A href="https://2012.igem.org/Team:NTNU_Trondheim/Notebook/June#Monday_25.06.12" class="external text" rel="nofollow">25</A></TD><TD><A href="https://2012.igem.org/Team:NTNU_Trondheim/Notebook/June#Tuesday_26.06.12" class="external text" rel="nofollow">26</A></TD><TD><A href="https://2012.igem.org/Team:NTNU_Trondheim/Notebook/June#Wednesday_27.06.12" class="external text" rel="nofollow">27</A></TD><TD><A href="https://2012.igem.org/Team:NTNU_Trondheim/Notebook/June#Thursday_28.06.12" class="external text" rel="nofollow">28</A></TD><TD><A href="https://2012.igem.org/Team:NTNU_Trondheim/Notebook/June#Friday_29.06.12" class="external text" rel="nofollow">29</A></TD><TD><A href="https://2012.igem.org/Team:NTNU_Trondheim/Notebook/June#Saturday_30.06.12" class="external text" rel="nofollow">30</A></TD><TD><A href="https://2012.igem.org/Team:NTNU_Trondheim/Notebook/June#Sunday_31.06.12" class="external text" rel="nofollow">31</A></TD></TR></TBODY></TABLE><TABLE align="center" style="text-align:right;" cellpadding="3"><CAPTION><B><A href="https://2012.igem.org/Team:NTNU_Trondheim/Notebook/July" class="external text" rel="nofollow">July</A></B></CAPTION><TBODY><TR><TD> Mon
</TD><TD> Tue
</TD><TD> Wed
</TD><TD> Thu
</TD><TD> Fri
</TD><TD> Sat
</TD><TD> Sun
</TD></TR><TR><TD><A href="https://2012.igem.org/Team:NTNU_Trondheim/Notebook/July#Sunday_01.07.12" class="external text" rel="nofollow">1</A></TD></TR><TR><TD><A href="https://2012.igem.org/Team:NTNU_Trondheim/Notebook/July#Monday_02.07.12" class="external text" rel="nofollow">2</A></TD><TD><A href="https://2012.igem.org/Team:NTNU_Trondheim/Notebook/July#Tuesday_03.07.12" class="external text" rel="nofollow">3</A></TD><TD><A href="https://2012.igem.org/Team:NTNU_Trondheim/Notebook/July#Wednesday_04.07.12" class="external text" rel="nofollow">4</A></TD><TD><A href="https://2012.igem.org/Team:NTNU_Trondheim/Notebook/July#Thursday_05.07.12" class="external text" rel="nofollow">5</A></TD><TD><A href="https://2012.igem.org/Team:NTNU_Trondheim/Notebook/July#Friday_06.07.12" class="external text" rel="nofollow">6</A></TD><TD><A href="https://2012.igem.org/Team:NTNU_Trondheim/Notebook/July#Saturday_07.07.12" class="external text" rel="nofollow">7</A></TD><TD><A href="https://2012.igem.org/Team:NTNU_Trondheim/Notebook/July#Friday_08.07.12" class="external text" rel="nofollow">8</A></TD></TR><TR><TD><A href="https://2012.igem.org/Team:NTNU_Trondheim/Notebook/July#Monday_09.07.12" class="external text" rel="nofollow">9</A></TD><TD><A href="https://2012.igem.org/Team:NTNU_Trondheim/Notebook/July#Tuesday_10.07.12" class="external text" rel="nofollow">10</A></TD><TD><A href="https://2012.igem.org/Team:NTNU_Trondheim/Notebook/July#Wednesday_11.07.12" class="external text" rel="nofollow">11</A></TD><TD><A href="https://2012.igem.org/Team:NTNU_Trondheim/Notebook/July#Thursday_12.07.12" class="external text" rel="nofollow">12</A></TD><TD><A href="https://2012.igem.org/Team:NTNU_Trondheim/Notebook/July#Friday_13.07.12" class="external text" rel="nofollow">13</A></TD><TD><A href="https://2012.igem.org/Team:NTNU_Trondheim/Notebook/July#Saturday_14.07.12" class="external text" rel="nofollow">14</A></TD><TD><A href="https://2012.igem.org/Team:NTNU_Trondheim/Notebook/July#Sunday_15.07.12" class="external text" rel="nofollow">15</A></TD></TR><TR><TD><A href="https://2012.igem.org/Team:NTNU_Trondheim/Notebook/July#Monday_16.07.12" class="external text" rel="nofollow">16</A></TD><TD><A href="https://2012.igem.org/Team:NTNU_Trondheim/Notebook/July#Tuesday_17.07.12" class="external text" rel="nofollow">17</A></TD><TD><A href="https://2012.igem.org/Team:NTNU_Trondheim/Notebook/July#Wednesday_18.07.12" class="external text" rel="nofollow">18</A></TD><TD><A href="https://2012.igem.org/Team:NTNU_Trondheim/Notebook/July#Thursday_19.07.12" class="external text" rel="nofollow">19</A></TD><TD><A href="https://2012.igem.org/Team:NTNU_Trondheim/Notebook/July#Friday_20.07.12" class="external text" rel="nofollow">20</A></TD><TD><A href="https://2012.igem.org/Team:NTNU_Trondheim/Notebook/July#Saturday_21.07.12" class="external text" rel="nofollow">21</A></TD><TD><A href="https://2012.igem.org/Team:NTNU_Trondheim/Notebook/July#Sunday_22.07.12" class="external text" rel="nofollow">22</A></TD></TR><TR><TD><A href="https://2012.igem.org/Team:NTNU_Trondheim/Notebook/July#Monday_23.07.12" class="external text" rel="nofollow">23</A></TD><TD><A href="https://2012.igem.org/Team:NTNU_Trondheim/Notebook/July#Tuesday_24.07.12" class="external text" rel="nofollow">24</A></TD><TD><A href="https://2012.igem.org/Team:NTNU_Trondheim/Notebook/July#Wednesday_25.07.12" class="external text" rel="nofollow">25</A></TD><TD><A href="https://2012.igem.org/Team:NTNU_Trondheim/Notebook/July#Thursday_26.07.12" class="external text" rel="nofollow">26</A></TD><TD><A href="https://2012.igem.org/Team:NTNU_Trondheim/Notebook/July#Friday_27.07.12" class="external text" rel="nofollow">27</A></TD><TD><A href="https://2012.igem.org/Team:NTNU_Trondheim/Notebook/July#Saturday_28.07.12" class="external text" rel="nofollow">28</A></TD><TD><A href="https://2012.igem.org/Team:NTNU_Trondheim/Notebook/July#Sunday_29.07.12" class="external text" rel="nofollow">29</A></TD></TR><TR><TD><A href="https://2012.igem.org/Team:NTNU_Trondheim/Notebook/July#Monday_30.07.12" class="external text" rel="nofollow">30</A></TD><TD><A href="https://2012.igem.org/Team:NTNU_Trondheim/Notebook/July#Tuesday_31.07.12" class="external text" rel="nofollow">31</A></TD></TR></TBODY></TABLE><TABLE align="center" style="text-align:right;" cellpadding="3"><CAPTION><B><A href="https://2012.igem.org/Team:NTNU_Trondheim/Notebook/August" class="external text" rel="nofollow">August</A></B></CAPTION><TBODY><TR><TD> Mon
</TD><TD> Tue
</TD><TD> Wed
</TD><TD> Thu
</TD><TD> Fri
</TD><TD> Sat
</TD><TD> Sun
</TD></TR><TR><TD><A href="https://2012.igem.org/Team:NTNU_Trondheim/Notebook/August#Wednesday_01.08.12" class="external text" rel="nofollow">1</A></TD><TD><A href="https://2012.igem.org/Team:NTNU_Trondheim/Notebook/August#Thursday_02.08.12" class="external text" rel="nofollow">2</A></TD><TD><A href="https://2012.igem.org/Team:NTNU_Trondheim/Notebook/August#Friday_03.08.12" class="external text" rel="nofollow">3</A></TD><TD><A href="https://2012.igem.org/Team:NTNU_Trondheim/Notebook/August#Saturday_04.08.12" class="external text" rel="nofollow">4</A></TD><TD><A href="https://2012.igem.org/Team:NTNU_Trondheim/Notebook/August#Sunday_05.08.12" class="external text" rel="nofollow">5</A></TD></TR><TR><TD><A href="https://2012.igem.org/Team:NTNU_Trondheim/Notebook/August#Monday_06.08.12" class="external text" rel="nofollow">6</A></TD><TD><A href="https://2012.igem.org/Team:NTNU_Trondheim/Notebook/August#Tuesday_07.08.12" class="external text" rel="nofollow">7</A></TD><TD><A href="https://2012.igem.org/Team:NTNU_Trondheim/Notebook/August#Wednesday_08.08.12" class="external text" rel="nofollow">8</A></TD><TD><A href="https://2012.igem.org/Team:NTNU_Trondheim/Notebook/August#Thursday_09.08.12" class="external text" rel="nofollow">9</A></TD><TD><A href="https://2012.igem.org/Team:NTNU_Trondheim/Notebook/August#Friday_10.08.12" class="external text" rel="nofollow">10</A></TD><TD><A href="https://2012.igem.org/Team:NTNU_Trondheim/Notebook/August#Saturday_11.08.12" class="external text" rel="nofollow">11</A></TD><TD><A href="https://2012.igem.org/Team:NTNU_Trondheim/Notebook/August#Sunday_12.08.12" class="external text" rel="nofollow">12</A></TD></TR><TR><TD><A href="https://2012.igem.org/Team:NTNU_Trondheim/Notebook/August#Monday_13.08.12" class="external text" rel="nofollow">13</A></TD><TD><A href="https://2012.igem.org/Team:NTNU_Trondheim/Notebook/August#Tuesday_14.08.12" class="external text" rel="nofollow">14</A></TD><TD><A href="https://2012.igem.org/Team:NTNU_Trondheim/Notebook/August#Wednesday_15.08.12" class="external text" rel="nofollow">15</A></TD><TD><A href="https://2012.igem.org/Team:NTNU_Trondheim/Notebook/August#Thursday_16.08.12" class="external text" rel="nofollow">16</A></TD><TD><A href="https://2012.igem.org/Team:NTNU_Trondheim/Notebook/August#Friday_17.08.12" class="external text" rel="nofollow">17</A></TD><TD><A href="https://2012.igem.org/Team:NTNU_Trondheim/Notebook/August#Saturday_18.08.12" class="external text" rel="nofollow">18</A></TD><TD><A href="https://2012.igem.org/Team:NTNU_Trondheim/Notebook/August#Sunday_19.08.12" class="external text" rel="nofollow">19</A></TD></TR><TR><TD><A href="https://2012.igem.org/Team:NTNU_Trondheim/Notebook/August#Monday_20.08.12" class="external text" rel="nofollow">20</A></TD><TD><A href="https://2012.igem.org/Team:NTNU_Trondheim/Notebook/August#Tuesday_21.08.12" class="external text" rel="nofollow">21</A></TD><TD><A href="https://2012.igem.org/Team:NTNU_Trondheim/Notebook/August#Wednesday_22.08.12" class="external text" rel="nofollow">22</A></TD><TD><A href="https://2012.igem.org/Team:NTNU_Trondheim/Notebook/August#Thursday_23.08.12" class="external text" rel="nofollow">23</A></TD><TD><A href="https://2012.igem.org/Team:NTNU_Trondheim/Notebook/August#Friday_24.08.12" class="external text" rel="nofollow">24</A></TD><TD><A href="https://2012.igem.org/Team:NTNU_Trondheim/Notebook/August#Saturday_25.08.12" class="external text" rel="nofollow">25</A></TD><TD><A href="https://2012.igem.org/Team:NTNU_Trondheim/Notebook/August#Sunday_26.08.12" class="external text" rel="nofollow">26</A></TD></TR><TR><TD><A href="https://2012.igem.org/Team:NTNU_Trondheim/Notebook/August#Monday_.27.12" class="external text" rel="nofollow">27</A></TD><TD><A href="https://2012.igem.org/Team:NTNU_Trondheim/Notebook/August#Tuesday_28.08.12" class="external text" rel="nofollow">28</A></TD><TD><A href="https://2012.igem.org/Team:NTNU_Trondheim/Notebook/August#Wednesday_29.08.12" class="external text" rel="nofollow">29</A></TD><TD><A href="https://2012.igem.org/Team:NTNU_Trondheim/Notebook/August#Thursday_30.08.12" class="external text" rel="nofollow">30</A></TD><TD><A href="https://2012.igem.org/Team:NTNU_Trondheim/Notebook/August#Friday_31.08.12" class="external text" rel="nofollow">31</A></TD></TR></TBODY></TABLE><TABLE align="center" style="text-align:right;" cellpadding="3"><CAPTION><B><A href="https://2012.igem.org/Team:NTNU_Trondheim/Notebook/September" class="external text" rel="nofollow">September</A></B></CAPTION><TBODY><TR><TD> Mon
</TD><TD> Tue
</TD><TD> Wed
</TD><TD> Thu
</TD><TD> Fri
</TD><TD> Sat
</TD><TD> Sun
</TD></TR><TR><TD> 1
</TD><TD> 2
</TD></TR><TR><TD> 3
</TD><TD> 4
</TD><TD> 5
</TD><TD> 6
</TD><TD> 7
</TD><TD> 8
</TD><TD> 9
</TD></TR><TR><TD>10
</TD><TD>11
</TD><TD>12
</TD><TD>13
</TD><TD>14
</TD><TD>15
</TD><TD>16
</TD></TR><TR><TD>17
</TD><TD>18
</TD><TD>19
</TD><TD>20
</TD><TD>21
</TD><TD>22
</TD><TD>23
</TD></TR><TR><TD>24
</TD><TD>25
</TD><TD>26
</TD><TD>27
</TD><TD>28
</TD><TD>29
</TD><TD>30
</TD></TR></TBODY></TABLE></DIV></DIV></DIV><DIV class="row-fluid"><DIV class="span12"><H3><SPAN class="mw-headline" id="Monday_27.08.12">Monday 27.08.12</SPAN></H3><P>Colicin expressed from the K+RBS+Colicin+DTT plasmid was tested on <I>E.coli</I> DH5Î± cells.
</P><P>Also, Plld w/ RBS from <I>E.coli</I> was cut with S+P, Plld w/RBS from <I>C.glutamicum</I> was cut with E+P, and linearized plasmid backbone pSB1A3 was cut with E+P. The samples will be purified and ligated tomorrow.
</P><P>
Results from the miniprep:
</P><TABLE class="table table-bordered table-hover" style="margin: 1em auto 1em auto; width: auto;"><TBODY><TR><TH>Gene
</TH><TH>Concentration [ng/Âµl]
</TH></TR><TR><TD>P<SUB>lld</SUB>CGr+YFP+DTT in pSB1A3 #1
</TD><TD>29,5
</TD></TR><TR><TD>P<SUB>lld</SUB>CGr+YFP+DTT in pSB1A3 #2
</TD><TD>23,7
</TD></TR><TR><TD>P<SUB>lld</SUB>CGr+YFP+DTT in pSB1A3 #3
</TD><TD>47,7
</TD></TR><TR><TD>P<SUB>lld</SUB>CGr+YFP+DTT in pSB1A3 #4
</TD><TD>25,2
</TD></TR><TR><TD>P<SUB>lld</SUB>CGr+YFP+DTT in pSB13 #1
</TD><TD>18,8
</TD></TR><TR><TD>P<SUB>lld</SUB>CGr+YFP+DTT in pSB1C3 #2
</TD><TD>21,0
</TD></TR><TR><TD>P<SUB>lld</SUB>CGr in pSB1C3 2.1
</TD><TD>24,8
</TD></TR><TR><TD>P<SUB>lld</SUB>CGr in pSB1C3 2.2
</TD><TD>20,7
</TD></TR><TR><TD>P<SUB>lld</SUB>CGr in pSB1C3 3.1
</TD><TD>21,3
</TD></TR><TR><TD>P<SUB>lld</SUB>CGr in pSB1C3 3.2
</TD><TD>18,4
</TD></TR></TBODY></TABLE><H3><SPAN class="mw-headline" id="Sunday_26.08.12">Sunday 26.08.12</SPAN></H3><P>RNA was isolated from the RBS(&lt;partinfo&gt;BBa_B0030&lt;/partinfo&gt;)+LacI(&lt;partinfo&gt;BBa_C0061&lt;/partinfo&gt;)+DTT(&lt;partinfo&gt;BBa_B0014&lt;/partinfo&gt;) and <I>E.coli</I> DH5Î± cells inoculated friday. RNA was isolated, and the DNAse reaction and the cDNA reaction were performed according to the protocol on the procedures page. The cDNA was frozen down to -80 Â°C.
</P><H3><SPAN class="mw-headline" id="Friday_24.08.12">Friday 24.08.12</SPAN></H3><P>Inoculated one cell culture of <I>E.coli</I> DH5Î± containing a plasmid consisting of RBS (&lt;partinfo&gt;BBa_B0030&lt;/partinfo&gt;) + LacI (&lt;partinfo&gt;BBa_C0061&lt;/partinfo&gt;) + DTT (&lt;partinfo&gt;BBa_B0014&lt;/partinfo&gt;), and one cell culture consisting of <I>E.coli</I> DH5Î± cells without plasmid. These colonies will be used for real time PCR.
</P><H3><SPAN class="mw-headline" id="Thursday_23.08.12">Thursday 23.08.12</SPAN></H3><P>BBa_K822001 in pSB1A3 was cut with Spe1 and Pst1, BBa_E0030 + terminator was cut with Xba1 and PSt1.
</P><P>BBa_E0030 + terminator was run on gel, and got the consentration 4,5 ng/uL. 
</P><P>BBa_K822001 in pSB1A3 was purified using a PCR purifying kit, and an concentration of 1,2 ng/uL was obtained.
</P><H3><SPAN class="mw-headline" id="Wednesday_22.08.12">Wednesday 22.08.12</SPAN></H3><P>P<SUB>lld</SUB> in pSB1A3 and pSB1C3 were miniprepped as described in the <A href="https://2012.igem.org/Team:NTNU_Trondheim/Protocols" class="external text" rel="nofollow">protocol</A>. The concentrations are listed below.
</P><TABLE class="table table-bordered table-hover" style="margin: 1em auto 1em auto; width: auto;"><TBODY><TR><TH>Gene
</TH><TH>Concentration [ng/Âµl]
</TH></TR><TR><TD>P<SUB>lld</SUB> in pSB1A3
</TD><TD>25,7
</TD></TR><TR><TD>P<SUB>lld</SUB> in pSB1C3
</TD><TD>39,5
</TD></TR></TBODY></TABLE><H3><SPAN class="mw-headline" id="Tuesday_21.08.12">Tuesday 21.08.12</SPAN></H3><P>The primers ordered came today, so PCR was performed on p<SUB>lld</SUB> with RBS (from <I>Corynebacterium glutamicum</I>), lldR from <I>E. coli</I> and <I>C. glutamicum</I>, and His-tagged lld from <I>C. glutamicum</I>. 
</P><P>The PCR mix and program are given below.
</P><TABLE class="table table-bordered table-hover" style="margin: 1em auto 1em auto; width: auto;"><TBODY><TR><TH>Substance
</TH><TH>Volume [Âµl]
</TH></TR><TR><TD>dH<SUB>2</SUB>O
</TD><TD>32,5
</TD></TR><TR><TD>5x-buffer
</TD><TD>10,0
</TD></TR><TR><TD>dNTPs
</TD><TD>1,0
</TD></TR><TR><TD>fwd primer
</TD><TD>2,5
</TD></TR><TR><TD>rev primer
</TD><TD>2,5
</TD></TR><TR><TD>template
</TD><TD>1,0
</TD></TR><TR><TD>Phusion polymerase
</TD><TD>0,5
</TD></TR></TBODY></TABLE><P>The addition of the Phusion polymerase is done immediately before starting the PCR program.
</P><TABLE><TBODY><TR><TH> Step
</TH><TH> Action
</TH><TH> Temperature [Â°C]
</TH><TH> Duration
</TH></TR><TR><TD>1
</TD><TD>Heated lid
</TD><TD>103
</TD></TR><TR><TD>2
</TD><TD>Initial denaturation
</TD><TD>98
</TD><TD>30 sek
</TD></TR><TR><TD>3
</TD><TD>Denaturation
</TD><TD>98
</TD><TD>10 sek
</TD></TR><TR><TD>4
</TD><TD>Annealing
</TD><TD>x<SUB>1</SUB></TD><TD>30 sek
</TD></TR><TR><TD>5
</TD><TD>Elongation
</TD><TD>72
</TD><TD>x<SUB>2</SUB></TD></TR><TR><TD>6
</TD><TD>Go to step 3, repeat 10 x
</TD></TR><TR><TD>7
</TD><TD>Denaturation
</TD><TD>98
</TD><TD>10 sek
</TD></TR><TR><TD>8
</TD><TD>Elongation
</TD><TD>72
</TD><TD>x<SUB>2</SUB></TD></TR><TR><TD>9
</TD><TD>Go to step 7, repeat 15 x, with 5 s extra each time
</TD></TR><TR><TD>10
</TD><TD>Final Elongation
</TD><TD>72
</TD><TD>5 min
</TD></TR><TR><TD>11
</TD><TD>Hold
</TD><TD>4
</TD><TD>â
</TD></TR></TBODY></TABLE><TABLE class="table table-bordered table-hover" style="margin: 1em auto 1em auto; width: auto;"><TBODY><TR><TH>Amplicon
</TH><TH>x<SUB>1</SUB> [Â°C]
</TH><TH>x<SUB>2</SUB> [s]
</TH></TR><TR><TD>His tagged lld, <I>C. glutamicum</I></TD><TD>62
</TD><TD>60
</TD></TR><TR><TD>P<SUB>lld</SUB>, <I>C. glutamicum</I></TD><TD>63
</TD><TD>5
</TD></TR><TR><TD>lldR, <I>E. coli</I></TD><TD> -
</TD><TD>26
</TD></TR><TR><TD>lldR, <I>C. glutamicum</I></TD><TD>66
</TD><TD>21
</TD></TR></TBODY></TABLE><H3><SPAN class="mw-headline" id="Monday_20.08.12">Monday 20.08.12</SPAN></H3><P>P<SUB>lld</SUB> in pSB1A3 was transformed into DH5Î± cells an plated out. Incubated at 37 Â°C.
The cells inoculated to liquid medium on saturday were miniprepped. The concentrations are given below, together with the volume necessary for obtaining 1000 ng of plasmid
</P><TABLE class="table table-bordered table-hover" style="margin: 1em auto 1em auto; width: auto;"><TBODY><TR><TH>Biobrick
</TH><TH>Concentration [ng/Âµl]
</TH><TH>Volume required to obtain 1000 ng plasmid
</TH></TR><TR><TD>Vgb+RBS+YFP+DTT
</TD><TD>27.1
</TD><TD>36.9
</TD></TR><TR><TD>Vgb+RBS+LuxR+DTT
</TD><TD>32.1
</TD><TD>31.2
</TD></TR><TR><TD>K+RBS+colicin
</TD><TD>28.4
</TD><TD>35.2
</TD></TR><TR><TD>&lt;partinfo&gt;BBa_K292006&lt;/partinfo&gt;</TD><TD>26.0
</TD><TD>38.5
</TD></TR><TR><TD>Colicin in pSB1C3
</TD><TD>35.3
</TD><TD>28.3
</TD></TR><TR><TD>YFP+DTT
</TD><TD>39.6
</TD><TD>25.3
</TD></TR><TR><TD>RBS*+LacI+DTT*
</TD><TD>33.0
</TD><TD>30.3
</TD></TR></TBODY></TABLE><P>The required volume for obtaining 1000 ng of plasmid were then pipetted out for each sample, and dried on a heat block prewarmed to 50 Â°C, to obtain an approximate volume of 15 Âµl.
The samples were then sent to sequencing.
</P><H3><SPAN class="mw-headline" id="Saturday_18.08.12">Saturday 18.08.12</SPAN></H3><P>P<SUB>lld</SUB> cut with E+P and &lt;partinfo&gt;pSB1A3&lt;/partinfo&gt; cut with E+P was ligated together to see if the P<SUB>lld</SUB> promoter will be more functional in an ampicillin plasmid.
In order to be able to send plasmids to sequencing on monday, Vgb+RBS+YFP+DTT, Vgb+RBS+LuxR+DTT, K+RBS+colicin, &lt;partinfo&gt;BBa_K292006&lt;/partinfo&gt;, colicin in pSB1C3, YFP+DTT and RBS*+LacI+DTT* was inoculated to liquid medium
</P><H3><SPAN class="mw-headline" id="Thursday_16.08.12">Thursday 16.08.12</SPAN></H3><P>P<SUB>lld</SUB>+lysis and pSB1A3 cut yesterday with E+P was investigated on gel. Two fragments were obtained for P<SUB>lld</SUB>+lysis, but the sizes of the fragments were not as expected. Since we are not sure which fragment is which, both of the fragments were cut out of the gel and purified using the QIAquick gel extraction kit. In case one of them is uncut plasmid, the unligated DNA from both bonds will be transformed tomorrow, and we will assume that if the cells transformed with DNA from one of the samples won't grow, that sample will contain digested plasmid. This sample will be ligated with pSB1A3, plated out on a ampicillin plate, and then, several cell cultures from this plate will be plated out on a chloramphenicol plate. The colonies that won't grow on chloramphenicol, is assumed to contain the correct plasmid. 
</P><P>Vgb+RBS religated, Vgb+RBS+LuxR+DTT #1, Vgb+RBS+LuxR+DTT #2, LuxI+DTT, K+RBS+LacI+DTT and his-tagged LldR was miniprepped using the Promega SV miniprep kit. The concentrations of isolated plasmid are given below:
</P><TABLE class="table table-bordered table-hover" style="margin: 1em auto 1em auto; width: auto;"><TBODY><TR><TH>Plasmid
</TH><TH>Concentration [ng/Âµl]
</TH></TR><TR><TD>Vgb+RBS religated
</TD><TD>27.3
</TD></TR><TR><TD>Vgb+RBS+LuxR+DTT #1
</TD><TD>26.2
</TD></TR><TR><TD>Vgb+RBS+LuxR+DTT #2
</TD><TD>31.7
</TD></TR><TR><TD>LuxI+DTT
</TD><TD>35.3
</TD></TR><TR><TD>K+RBS+LacI+DTT
</TD><TD>7.9
</TD></TR><TR><TD>his-LldR
</TD><TD>51.0
</TD></TR></TBODY></TABLE><P>Also, Vgb and Vhb was tested again, and Ove named two of the agar plates Mark and Doggie:-)
</P><H3><SPAN class="mw-headline" id="Wednesday_15.08.12">Wednesday 15.08.12</SPAN></H3><P>The high expression plasmid containing his-tagged LldR was transferred to liquid medium. 
</P><P>K+RBS+Colicin and the negative colicin control, LuxR+DTT was transferred to new liquid medium in order to be tested. 50 Âµl of the old cultures was transferred to 10 ml of LB containing ampicillin.
</P><P>Since we have read that chloramphenicol can be a problem for the lld promoter, we also decided to transfer this brick to a new backbone before further testing. Both P<SUB>lld</SUB> and P<SUB>lld</SUB>+RBS+lysis was cut with EcoRI and PstI. Linearized plasmid backbone pSB1A3 was also cut with EcoRI and PstI. Ligations will be performed tomorrow.
</P><H3><SPAN class="mw-headline" id="Tuesday_14.08.12">Tuesday 14.08.12</SPAN></H3><P>To check if the new restriction digest were ok, LuxI (&lt;partinfo&gt;Bba_C0061&lt;/partinfo&gt;) and LuxR+term (&lt;partinfo&gt;BBa_C0062&lt;/partinfo&gt;, &lt;partinfo&gt;BBa_B0015&lt;/partinfo&gt;) were tested with gel electrophoresis.
</P><DIV class="center"><DIV class="thumb tnone"><DIV class="thumbinner" style="width:302px;"><DIV class="thumbcaption">Picture of gel electrophoresis. LuxI and LuxR+Term cut with NotI, and LuxI and LuxR+term cut with E+S and X+P, respectively.</DIV></DIV></DIV></DIV><P>
Cut of the parts consisting the genes, LuxI (&lt;partinfo&gt;Bba_C0061&lt;/partinfo&gt;) and LuxR+term (&lt;partinfo&gt;BBa_C0062&lt;/partinfo&gt;, &lt;partinfo&gt;BBa_B0015&lt;/partinfo&gt;), and extracted the DNA from the gel. Concentrations are listed below.
</P><TABLE class="table table-bordered table-hover" style="margin: 1em auto 1em auto; width: auto;"><TBODY><TR><TH>BioBrick
</TH><TH>Enzymes
</TH><TH>Concentration [ng/Âµl]
</TH></TR><TR><TD>LuxI
</TD><TD>EcoRI+SpeI
</TD><TD>4.2
</TD></TR><TR><TD>LuxR+term
</TD><TD>XbaI+PstI
</TD><TD>5.6
</TD></TR></TBODY></TABLE><P>Ligation mixes were made according to <A href="https://2012.igem.org/Team:NTNU_Trondheim/Protocols" class="external text" rel="nofollow">protocol</A> with the following ingredients, as well as religation of the backbones:
</P><TABLE class="table table-bordered table-hover" style="margin: 1em auto 1em auto; width: auto;"><TBODY><TR><TH>Backbone
</TH><TH>Insert
</TH><TH>Plasmid
</TH></TR><TR><TD>K &lt;partinfo&gt;BBa_J23119&lt;/partinfo&gt;</TD><TD>RBS+LacI+term &lt;partinfo&gt;BBa_B0030&lt;/partinfo&gt;, &lt;partinfo&gt;Bba_C0012&lt;/partinfo&gt;, &lt;partinfo&gt;BBa_B0015&lt;/partinfo&gt;</TD><TD>&lt;partinfo&gt;psB1A2&lt;/partinfo&gt;</TD></TR><TR><TD>Vgb+RBS &lt;partinfo&gt;BBa_K561001&lt;/partinfo&gt;, &lt;partinfo&gt;BBa_B0034&lt;/partinfo&gt;</TD><TD>LuxR+term &lt;partinfo&gt;BBa_C0062&lt;/partinfo&gt;, &lt;partinfo&gt;BBa_B0015&lt;/partinfo&gt;</TD><TD>&lt;partinfo&gt;psB1A2&lt;/partinfo&gt;</TD></TR><TR><TD>term &lt;partinfo&gt;BBa_B0015&lt;/partinfo&gt;</TD><TD>LuxI &lt;partinfo&gt;BBa_C0061&lt;/partinfo&gt;</TD><TD>&lt;partinfo&gt;pSB1AK3&lt;/partinfo&gt;</TD></TR></TBODY></TABLE><P>These were then transformed in competent DH5É cells, as described in the <A href="https://2012.igem.org/Team:NTNU_Trondheim/Protocols" class="external text" rel="nofollow">protocol</A>. The petri dishes (all with Ampicillin) were left in the incubator over night.
</P><P>10 Âµl of the liquid cultures for K+RBS+colicin and LuxR+DTT was transferred to 5 ml of new liquid medium to be used in testing of the colicin biobrick. The rest of the samples were miniprepped using the Promega SV Miniprep kit. The concentrations of the samples are given below:
</P><TABLE class="table table-bordered table-hover" style="margin: 1em auto 1em auto; width: auto;"><TBODY><TR><TH>Biobrick
</TH><TH>Concentration [ng/Âµl]
</TH></TR><TR><TD>K+RBS+colicin
</TD><TD>31.1
</TD></TR><TR><TD>LuxR+DTT
</TD><TD>31.3
</TD></TR></TBODY></TABLE><P>We also recieved the high expression plasmid for his-tagged LldR today. This was transformed to competent <I>E.coli</I> K-12 ER2566 cells, as <I>E.coli</I> Dh5Î±, which are the cells we normally use in transformation, are unsuited when working with expression of proteins.
</P><H3><SPAN class="mw-headline" id="Monday_13.08.12">Monday 13.08.12</SPAN></H3><P>Transferred K+RBS+Colisin (&lt;partinfo&gt;BBa_K081005&lt;/partinfo&gt; and &lt;partinfo&gt;BBa_K150009&lt;/partinfo&gt;), P<SUB>LuxR+HSL</SUB>+RBS+Lysis (&lt;partinfo&gt;BBa_R0062&lt;/partinfo&gt;, &lt;partinfo&gt;BBa_B0034&lt;/partinfo&gt; and &lt;partinfo&gt;BBa_K112808&lt;/partinfo&gt;) to liquid media. 
</P><P>Did a restriction digest on the following biobricks, as described in the <A href="https://2012.igem.org/Team:NTNU_Trondheim/Protocols" class="external text" rel="nofollow">protocol</A>.
</P><TABLE class="table table-bordered table-hover" style="margin: 1em auto 1em auto; width: auto;"><TBODY><TR><TH>Biobricks
</TH><TH>BioBrick no.
</TH><TH>Enzymes
</TH></TR><TR><TD>Vgb+RBS
</TD><TD>&lt;partinfo&gt;BBa_K561001&lt;/partinfo&gt;, &lt;partinfo&gt;BBa_B0034&lt;/partinfo&gt;</TD><TD>SpeI+PstI, NotI
</TD></TR><TR><TD>LuxR+term
</TD><TD>&lt;partinfo&gt;BBa_C0062&lt;/partinfo&gt;, &lt;partinfo&gt;BBa_B0015&lt;/partinfo&gt;</TD><TD>XbaI+PstI, NotI
</TD></TR><TR><TD>RBS+LacI+term
</TD><TD>&lt;partinfo&gt;BBa_B0030&lt;/partinfo&gt;, &lt;partinfo&gt;Bba_C0012&lt;/partinfo&gt;, &lt;partinfo&gt;BBa_B0015&lt;/partinfo&gt;</TD><TD>XbaI+PstI, NotI
</TD></TR><TR><TD>LuxI
</TD><TD>&lt;partinfo&gt;Bba_C0061&lt;/partinfo&gt;</TD><TD>EcoRI+SpeI, NotI
</TD></TR><TR><TD>K
</TD><TD>&lt;partinfo&gt;BBa_J23119&lt;/partinfo&gt;</TD><TD>SpeI+PstI
</TD></TR></TBODY></TABLE><P>Did a gel electrophoresis on the mixes above (except for the konstitutive promoter &lt;partinfo&gt;BBa_J23119&lt;/partinfo&gt;), but only two of them where as expected. Did a new restriction digest on the two that didn't give the correct bands: LuxI (&lt;partinfo&gt;Bba_C0061&lt;/partinfo&gt;) and LuxR+term (&lt;partinfo&gt;BBa_C0062&lt;/partinfo&gt; and &lt;partinfo&gt;BBa_B0015&lt;/partinfo&gt;).
</P><DIV class="center"><DIV class="thumb tnone"><DIV class="thumbinner" style="width:302px;"><DIV class="thumbcaption">Picture of gel electrophoresis with test cutting with NotI (to the left) and regular cutting (to the right) of LuxI (EcoRI+SpeI), LuxR+term (X+P), RBS+lacI+term (XbaI+PstI) and Vgb+RBS (SpeI+PstI), respectively.</DIV></DIV></DIV></DIV><P>Extracted DNA from gel on the part RBS+LacI+term (&lt;partinfo&gt;BBa_B0030&lt;/partinfo&gt;, &lt;partinfo&gt;Bba_C0012&lt;/partinfo&gt; and &lt;partinfo&gt;BBa_B0015&lt;/partinfo&gt;) and did a PCR purification on Vgb+RBS (&lt;partinfo&gt;BBa_K561001&lt;/partinfo&gt; and &lt;partinfo&gt;BBa_B0034&lt;/partinfo&gt;) and the konstitutive promoter (K &lt;partinfo&gt;BBa_J23119&lt;/partinfo&gt;).
</P><P>The concentrations on the purified parts are as follows:
</P><TABLE class="table table-bordered table-hover" style="margin: 1em auto 1em auto; width: auto;"><TBODY><TR><TH>Biobricks
</TH><TH>BioBrick no.
</TH><TH>Concentration [ng/Âµl]
</TH></TR><TR><TD>K
</TD><TD>&lt;partinfo&gt;BBa_J23119&lt;/partinfo&gt;</TD><TD>3.0
</TD></TR><TR><TD>Vgb+RBS
</TD><TD>&lt;partinfo&gt;BBa_K561001&lt;/partinfo&gt;, &lt;partinfo&gt;BBa_B0034&lt;/partinfo&gt;</TD><TD>3.6
</TD></TR><TR><TD>RBS+LacI+term
</TD><TD>&lt;partinfo&gt;BBa_B0030&lt;/partinfo&gt;, &lt;partinfo&gt;Bba_C0012&lt;/partinfo&gt;, &lt;partinfo&gt;BBa_B0015&lt;/partinfo&gt;</TD><TD>4.7
</TD></TR></TBODY></TABLE><H3><SPAN class="mw-headline" id="Friday_10.08.12">Friday 10.08.12</SPAN></H3><P>Performed miniprep on the following constructs:
</P><OL><LI> VGB+RBS+YFP+term (&lt;partinfo&gt;BBa_K561001&lt;/partinfo&gt;, &lt;partinfo&gt;BBa_B0034&lt;/partinfo&gt;, &lt;partinfo&gt;BBa_E0030&lt;/partinfo&gt; and &lt;partinfo&gt;BBa_B0015&lt;/partinfo&gt;)
</LI><LI> VHB+RBS+YFP+term (&lt;partinfo&gt;BBa_K258005&lt;/partinfo&gt;, &lt;partinfo&gt;BBa_B0034&lt;/partinfo&gt;, &lt;partinfo&gt;BBa_E0030&lt;/partinfo&gt; and &lt;partinfo&gt;BBa_B0015&lt;/partinfo&gt;)
</LI><LI> LuxI+term (&lt;partinfo&gt;Bba_C0061&lt;/partinfo&gt;, &lt;partinfo&gt;BBa_B0015&lt;/partinfo&gt;)
</LI></OL><P>The concentrations were as follows:
</P><TABLE class="table table-bordered table-hover" style="margin: 1em auto 1em auto; width: auto;"><TBODY><TR><TH>Biobrick
</TH><TH>Concentration ng/uL
</TH></TR><TR><TD>VGB+RBS+YFP+term
</TD><TD>35,2
</TD></TR><TR><TD>VHB+RBS+YFP+term
</TD><TD>38,9
</TD></TR><TR><TD>LuxI+term
</TD><TD>48,2
</TD></TR></TBODY></TABLE><P>Transformed K+RBS+colicin.
</P><P>Test constructs for Vgb and Vhb (Vgb+RBS+YFP+DTT and Vhb+RBS+YFP+DTT) were transferred to liquid medium for testing on monday. 
</P><P>It was attempted to make LldR by PCR one more time. The following PCR mix was used:
</P><UL><LI> 32.5 Âµl dH<SUB>2</SUB>O
</LI><LI> 10 Âµl 5x Phusion DNA polymerase buffer
</LI><LI> 1 Âµl dNTPs
</LI><LI> 2.5 Âµl fwd primer previously diluted 1:10
</LI><LI> 2.5 Âµl rev primer previously diluted 1:10
</LI><LI> 1 Âµl template DNA (<I>E.coli</I> K-12 genome)
</LI><LI> 0.5 Âµl Phusion DNA polymerase
</LI></UL><P>The following primers were used:
</P><TABLE class="table table-bordered table-hover" style="margin: 1em auto 1em auto; width: auto;"><TBODY><TR><TH>Primer
</TH><TH>Sequence
</TH></TR><TR><TD>LldR fwd
</TD><TD>atgattgttttacccagacgcctgt
</TD></TR><TR><TD>LldR rev
</TD><TD>tcatgcgtttttctccctcgaat
</TD></TR></TBODY></TABLE><P>The PCR machine was programmed according to the following table:
</P><TABLE class="table table-bordered table-hover" style="margin: 1em auto 1em auto; width: auto;"><TBODY><TR><TH>Step
</TH><TH>Action
</TH><TH>Temperature
</TH><TH>Duration
</TH></TR><TR><TD>1
</TD><TD>Heated lid:
</TD><TD>103Â°C
</TD></TR><TR><TD>2
</TD><TD>Initial denaturation;
</TD><TD>98Â°C
</TD><TD>30 s
</TD></TR><TR><TD>3
</TD><TD>Denaturation;
</TD><TD>98Â°C
</TD><TD>10 s
</TD></TR><TR><TD>4
</TD><TD>Annealing;
</TD><TD>45Â°C
</TD><TD>10 s
</TD></TR><TR><TD>5
</TD><TD>Elongation;
</TD><TD>72Â°C
</TD><TD>24 s
</TD></TR><TR><TD>6
</TD><TD>Go to step 3, repeat 25 x
</TD></TR><TR><TD>7
</TD><TD>Final elongation;
</TD><TD>72Â°C
</TD><TD>5 min
</TD></TR><TR><TD>8
</TD><TD>Hold
</TD><TD>4Â°C
</TD><TD>â
</TD></TR></TBODY></TABLE><H3><SPAN class="mw-headline" id="Thursday_09.08.12">Thursday 09.08.12</SPAN></H3><P>Performed miniprep on RBS (&lt;partinfo&gt;BBa_B0030&lt;/partinfo&gt;) and plld+RBS. The concentration were as follows:
</P><TABLE class="table table-bordered table-hover" style="margin: 1em auto 1em auto; width: auto;"><TBODY><TR><TH>Biobrick
</TH><TH>Concentration ng/uL
</TH></TR><TR><TD>RBS
</TD><TD>41,3
</TD></TR><TR><TD>plld+RBS
</TD><TD>43,7
</TD></TR></TBODY></TABLE><P>Did a restriction digest as described in the <A href="https://2012.igem.org/Team:NTNU_Trondheim/Protocols" class="external text" rel="nofollow">protocol</A> on the following parts. On the pieces to be used as inserts, the number of base pairs are also listed.
</P><TABLE class="table table-bordered table-hover" style="margin: 1em auto 1em auto; width: auto;"><TBODY><TR><TH>Biobrick/Construct
</TH><TH>BioBrick
</TH><TH>Enzymes
</TH><TH>Buffer
</TH><TH>bp insert
</TH></TR><TR><TD>LuxI+term
</TD><TD>&lt;partinfo&gt;Bba_C0061&lt;/partinfo&gt;, &lt;partinfo&gt;BBa_B0015&lt;/partinfo&gt;</TD><TD>XbaI+PstI
</TD><TD>3
</TD><TD>747
</TD></TR><TR><TD>YFP+term
</TD><TD>&lt;partinfo&gt;BBa_E0030&lt;/partinfo&gt;, &lt;partinfo&gt;BBa_B0015&lt;/partinfo&gt;</TD><TD>XbaI+PstI
</TD><TD>3
</TD><TD>852
</TD></TR><TR><TD>Lysis
</TD><TD>&lt;partinfo&gt;BBa_K112808&lt;/partinfo&gt;</TD><TD>XbaI+PstI
</TD><TD>3
</TD><TD>1785
</TD></TR><TR><TD>LuxI
</TD><TD>&lt;partinfo&gt;Bba_C0061&lt;/partinfo&gt;</TD><TD>XbaI+PstI
</TD><TD>3
</TD><TD>618
</TD></TR><TR><TD>RBS*
</TD><TD>&lt;partinfo&gt;BBa_B0030&lt;/partinfo&gt;</TD><TD>SpeI+PstI
</TD><TD>1
</TD><TD> -
</TD></TR><TR><TD>P<SUB>luxR+HSL</SUB></TD><TD>&lt;partinfo&gt;BBa_R0062&lt;/partinfo&gt;</TD><TD>SpeI+PstI
</TD><TD>1
</TD><TD> -
</TD></TR><TR><TD>RBS
</TD><TD>&lt;partinfo&gt;BBa_B0034&lt;/partinfo&gt;</TD><TD>SpeI+PstI
</TD><TD>1
</TD><TD> -
</TD></TR><TR><TD>pSB1A3
</TD><TD>&lt;partinfo&gt;pSB1A3&lt;/partinfo&gt;</TD><TD>EcoRI+PstI+DpnI
</TD><TD>3
</TD><TD> -
</TD></TR><TR><TD>plld
</TD><TD> -
</TD><TD>EcoRI+SpeI
</TD><TD>4
</TD><TD>344
</TD></TR></TBODY></TABLE><P>The parts lysis &lt;partinfo&gt;BBa_K112808&lt;/partinfo&gt;, YFP+term (&lt;partinfo&gt;BBa_E0030&lt;/partinfo&gt;, &lt;partinfo&gt;BBa_B0015&lt;/partinfo&gt;), &lt;partinfo&gt;pSB1A3&lt;/partinfo&gt; and plld will be assembled using the 3A assembly, so made double cutting mix with these parts.
</P><P>Gel electrophoresis was done with luxI+term (&lt;partinfo&gt;Bba_C0061&lt;/partinfo&gt;, &lt;partinfo&gt;BBa_B0015&lt;/partinfo&gt;), lysis &lt;partinfo&gt;BBa_K112808&lt;/partinfo&gt;, YFP+term (&lt;partinfo&gt;BBa_E0030&lt;/partinfo&gt;, &lt;partinfo&gt;BBa_B0015&lt;/partinfo&gt;), LuxI &lt;partinfo&gt;Bba_C0061&lt;/partinfo&gt; and plld. The gel did not show either LuxI or LuxI+term, but cut out lysis, YFP+term and plld. Will do a gel extraction on those, and a PCR purification on &lt;partinfo&gt;pSB1A3&lt;/partinfo&gt;, RBS &lt;partinfo&gt;BBa_B0034&lt;/partinfo&gt;, RBS* &lt;partinfo&gt;BBa_B0030&lt;/partinfo&gt; and P<SUB>LuxR+HSL</SUB>&lt;partinfo&gt;BBa_R0062&lt;/partinfo&gt;. This will be done as described in <A href="https://2012.igem.org/Team:NTNU_Trondheim/Protocols" class="external text" rel="nofollow">protocols</A>.
</P><P>Ligated together the following, as described in <A href="https://2012.igem.org/Team:NTNU_Trondheim/Protocols" class="external text" rel="nofollow">protocols</A>:
</P><TABLE class="table table-bordered table-hover" style="margin: 1em auto 1em auto; width: auto;"><TBODY><TR><TH>Ligation #
</TH><TH>Backbone
</TH><TH>Insert
</TH></TR><TR><TD>1
</TD><TD>RBS* &lt;partinfo&gt;BBa_B0030&lt;/partinfo&gt;</TD><TD>LacI+term &lt;partinfo&gt;Bba_C0012&lt;/partinfo&gt;, &lt;partinfo&gt;BBa_B0015&lt;/partinfo&gt;</TD></TR><TR><TD>2
</TD><TD>RBS &lt;partinfo&gt;BBa_B0034&lt;/partinfo&gt;</TD><TD>LuxR+term &lt;partinfo&gt;BBa_C0062&lt;/partinfo&gt;, &lt;partinfo&gt;BBa_B0015&lt;/partinfo&gt;</TD></TR><TR><TD>3
</TD><TD>p<SUB>LuxR+HSL</SUB>&lt;partinfo&gt;BBa_R0062&lt;/partinfo&gt;</TD><TD>Lysis &lt;partinfo&gt;BBa_K112808&lt;/partinfo&gt;</TD></TR></TBODY></TABLE><P>3A assembly was used on the following parts, as described in the <A href="https://2011.igem.org/Team:Northwestern/Notebook/Protocols/3A_Assembly" class="external text" rel="nofollow">Protocol</A>, by Northwestern University:
</P><TABLE class="table table-bordered table-hover" style="margin: 1em auto 1em auto; width: auto;"><TBODY><TR><TH>Ligation #
</TH><TH>Backbone
</TH><TH>Insert 1
</TH><TH>Insert 2
</TH></TR><TR><TD>1
</TD><TD>&lt;partinfo&gt;pSB1A3&lt;/partinfo&gt;</TD><TD>plld
</TD><TD>YFP+term&lt;partinfo&gt;BBa_E0030&lt;/partinfo&gt;, &lt;partinfo&gt;BBa_B0015&lt;/partinfo&gt;</TD></TR><TR><TD>2
</TD><TD>&lt;partinfo&gt;pSB1A3&lt;/partinfo&gt;</TD><TD>plld
</TD><TD>Lysis &lt;partinfo&gt;BBa_K112808&lt;/partinfo&gt;</TD></TR></TBODY></TABLE><P>Since the restriction digest didn't give anything for LuxI &lt;partinfo&gt;Bba_C0061&lt;/partinfo&gt; and LuxI+term (&lt;partinfo&gt;Bba_C0061&lt;/partinfo&gt;, &lt;partinfo&gt;BBa_B0015&lt;/partinfo&gt;), will a colony from LuxI+term (&lt;partinfo&gt;Bba_C0061&lt;/partinfo&gt;, &lt;partinfo&gt;BBa_B0015&lt;/partinfo&gt;) be transferred to liquid media and LuxI &lt;partinfo&gt;Bba_C0061&lt;/partinfo&gt; will be transformed again. 
</P><P>The following was also transformed: RBS*+LacI+term (&lt;partinfo&gt;BBa_B0030&lt;/partinfo&gt;, &lt;partinfo&gt;Bba_C0012&lt;/partinfo&gt; and0 &lt;partinfo&gt;BBa_B0015&lt;/partinfo&gt;), RBS+LuxR+term (&lt;partinfo&gt;BBa_B0034&lt;/partinfo&gt;, &lt;partinfo&gt;BBa_C0062&lt;/partinfo&gt; and &lt;partinfo&gt;BBa_B0015&lt;/partinfo&gt;), p<SUB>LuxR+HSL</SUB>+Lysis (&lt;partinfo&gt;BBa_R0062&lt;/partinfo&gt; and &lt;partinfo&gt;BBa_K112808&lt;/partinfo&gt;), &lt;partinfo&gt;pSB1A3&lt;/partinfo&gt;+plld+YFP+term (&lt;partinfo&gt;BBa_E0030&lt;/partinfo&gt; and &lt;partinfo&gt;BBa_B0015&lt;/partinfo&gt;) and &lt;partinfo&gt;pSB1A3&lt;/partinfo&gt;+plld+Lysis (&lt;partinfo&gt;BBa_K112808&lt;/partinfo&gt;). Did religations of the backbones for all of the above. Transferred 200 Âµl to petri dishes and inoculated in 37C over night.
</P><H3><SPAN class="mw-headline" id="Wednesday_08.08.12">Wednesday 08.08.12</SPAN></H3><P>Tranformed Vgb+RBS+YFP+term (&lt;partinfo&gt;BBa_K561001&lt;/partinfo&gt;, &lt;partinfo&gt;BBa_B0034&lt;/partinfo&gt;, &lt;partinfo&gt;BBa_E0030&lt;/partinfo&gt; and &lt;partinfo&gt;BBa_B0015&lt;/partinfo&gt;) and Vhb+RBS+YFP+term (&lt;partinfo&gt;BBa_K258005&lt;/partinfo&gt;, &lt;partinfo&gt;BBa_B0034&lt;/partinfo&gt;, &lt;partinfo&gt;BBa_E0030&lt;/partinfo&gt; and &lt;partinfo&gt;BBa_B0015&lt;/partinfo&gt;) in competent DH5É cells as described in the <A href="https://2012.igem.org/Team:NTNU_Trondheim/Protocols" class="external text" rel="nofollow">protocol</A>. The plasmid in both of the constructs is &lt;partinfo&gt;psB1A2&lt;/partinfo&gt;. They were left in the incubator over night.
</P><P>Inspected the petri dishes with plld+RBS (lactate induced promoter and &lt;partinfo&gt;BBa_B0030&lt;/partinfo&gt;) and the religation of RBS &lt;partinfo&gt;BBa_B0030&lt;/partinfo&gt;, had many more colonies on the plld+RBS than the religation, which is good. Transferred a colony of the plld+RBS (lactate promoter and &lt;partinfo&gt;BBa_B0030&lt;/partinfo&gt;) and a colony of RBS &lt;partinfo&gt;BBa_B0030&lt;/partinfo&gt; to liquid media, with ampicillin resistance.
</P><P>LacI+term (&lt;partinfo&gt;BBa_C0012&lt;/partinfo&gt;) + (&lt;partinfo&gt;BBa_B0014&lt;/partinfo&gt;)was miniprepped and concentration was 216,1 ng/uL.
</P><P>Cut LacI+term (&lt;partinfo&gt;BBa_C0012&lt;/partinfo&gt;) and (&lt;partinfo&gt;BBa_B0014&lt;/partinfo&gt;), LuxI+term (&lt;partinfo&gt;Bba_C0061&lt;/partinfo&gt; and &lt;partinfo&gt;BBa_B0015&lt;/partinfo&gt;) and LuxR+term (&lt;partinfo&gt;BBa_C0062&lt;/partinfo&gt; and &lt;partinfo&gt;BBa_B0015&lt;/partinfo&gt;) with XbaI and PstI, so that they can be used as inserts. RBS(&lt;partinfo&gt;BBa_B0034&lt;/partinfo&gt;) was cut with Spe1 and Pst1 and will be used further as a backbone.Used the <A href="https://2012.igem.org/Team:NTNU_Trondheim/Protocols" class="external text" rel="nofollow">protocol</A> with NEBuffer 2 and 3.
</P><P>Did a gel electrophoresis with LuxI+term (&lt;partinfo&gt;Bba_C0061&lt;/partinfo&gt; and &lt;partinfo&gt;BBa_B0015&lt;/partinfo&gt;) and LuxR+term (&lt;partinfo&gt;BBa_C0062&lt;/partinfo&gt; and &lt;partinfo&gt;BBa_B0015&lt;/partinfo&gt;).
</P><P>Purified RBS and LuxR+term. Resulting concentrations were 4.8 and 11.2 ng/Âµl, respectively. LacI+ term was also purified from gel using the gel extracion kit for Qiagen, and the concentration was 1,5 ng/uL.
</P><H3><SPAN class="mw-headline" id="Tuesday_07.08.12">Tuesday 07.08.12</SPAN></H3><P>Inspected the petri dishes from yesterday, LacI &lt;partinfo&gt;BBa_C0012&lt;/partinfo&gt; + terminator &lt;partinfo&gt;BBa_B0015&lt;/partinfo&gt; showed colonies, the religation of the terminator &lt;partinfo&gt;BBa_B0015&lt;/partinfo&gt; did not. Transferred a LacI &lt;partinfo&gt;BBa_C0012&lt;/partinfo&gt; + terminator &lt;partinfo&gt;BBa_B0015&lt;/partinfo&gt; colony to LB with ampicillin and inoculated at 37C with shaking.
</P><P>Did a gel electrophoresis on plld, cut it out and extracted it, according to <A href="https://2012.igem.org/Team:NTNU_Trondheim/Protocols" class="external text" rel="nofollow">protocols</A>. The concentration after gel extraction was: c<SUB>plld</SUB> = 10,6 ng/Âµl.
</P><DIV class="center"><DIV class="thumb tnone"><DIV class="thumbinner" style="width:402px;"><DIV class="thumbcaption">Picture of gel electrophoresis with plld. It was found at approximately 340 bp, as expected (344 bp).</DIV></DIV></DIV></DIV><P>Ligation was performed with plld as insert and RBS &lt;partinfo&gt;BBa_B0034&lt;/partinfo&gt; as backbone. That means that the construct has a &lt;partinfo&gt;psB1A2&lt;/partinfo&gt; plasmid. Also did a religation of the RBS &lt;partinfo&gt;BBa_B0034&lt;/partinfo&gt; without any insert.
</P><P>Transformed plld and RBS &lt;partinfo&gt;BBa_B0034&lt;/partinfo&gt; in competent DH5É cells, as well as the religation of RBS &lt;partinfo&gt;BBa_B0034&lt;/partinfo&gt;. They were transferred to petri dishes with ampicillin resistance and left in the incubator over night.
</P><P>Miniprepped VHB+RBS+lysis (&lt;partinfo&gt;BBa_K258005&lt;/partinfo&gt;, &lt;partinfo&gt;BBa_B0034&lt;/partinfo&gt; and &lt;partinfo&gt;BBa_K112808&lt;/partinfo&gt;), VGB+RBS+lysis (&lt;partinfo&gt;BBa_K561001&lt;/partinfo&gt;, &lt;partinfo&gt;BBa_B0034&lt;/partinfo&gt; and &lt;partinfo&gt;BBa_K112808&lt;/partinfo&gt;), pllD+lysis (lactate promoter and &lt;partinfo&gt;BBa_K112808&lt;/partinfo&gt;) and three parallels of pBad+YFP (&lt;partinfo&gt;Bba_K206000&lt;/partinfo&gt; and &lt;partinfo&gt;BBa_E0030&lt;/partinfo&gt;)
</P><TABLE class="table table-bordered table-hover" style="margin: 1em auto 1em auto; width: auto;"><TBODY><TR><TH>Sample
</TH><TH>Concentration [ng/Âµl]
</TH></TR><TR><TD>VHB+RBS+lysis
</TD><TD>38,2
</TD></TR><TR><TD>VGB+RBS+lysis
</TD><TD>50,5
</TD></TR><TR><TD>pllD+lysis
</TD><TD>40,2
</TD></TR><TR><TD>pBad+YFP #1
</TD><TD>46,6
</TD></TR><TR><TD>pBad+YFP #2
</TD><TD>40,5
</TD></TR><TR><TD>pBad+YFP #3
</TD><TD>38,9
</TD></TR></TBODY></TABLE><H3><SPAN class="mw-headline" id="Monday_06.08.12">Monday 06.08.12</SPAN></H3><P>In order to start making the construct for the biobrick we are going to improve, the parts needed for the construct were cut using the restriction digest protocol <A href="https://2012.igem.org/Team:NTNU_Trondheim/Protocols" class="external autonumber" rel="nofollow">[1]</A>.
</P><P>RBS &lt;partinfo&gt;BBa_B0030&lt;/partinfo&gt; will be used as a backbone and was cut with EcoRI and XbaI. LacI &lt;partinfo&gt;BBa_C0012&lt;/partinfo&gt; will be an insert, and was cut with EcoRI and SpeI. The terminator &lt;partinfo&gt;BBa_B0014&lt;/partinfo&gt; will be used as a backbone and was cut with EcoRI and XbaI. In addition we will test the construct using a constitutive promoter &lt;partinfo&gt;BBa_J23119&lt;/partinfo&gt; which was cut using EcoRI and SpeI.
</P><P>The inserts LacI &lt;partinfo&gt;BBa_C0012&lt;/partinfo&gt; and constitutive promoter &lt;partinfo&gt;BBa_J23119&lt;/partinfo&gt; were run on gel after the restriction digest.  The problem with the constitutive promoter is that it is only 35 b.p long, and is therefore difficult to use as an insert. We tried to stop the gel early to see if we could detect the constitutive promoter, but we could not see anything at all. We therefore ran the gel further and cut the LacI &lt;partinfo&gt;BBa_C0012&lt;/partinfo&gt; out of the gel.
</P><DIV class="center"><DIV class="thumb tnone"><DIV class="thumbinner" style="width:402px;"><DIV class="thumbcaption">Gel picture of the two inserts ran on gel. Used two different ladders. To the left we used a 1kb ladder to identify the Lac I insert, and to the right we used a 50 kb ladder to identify the constitutive promoter. Expected the LacI band to be approximately 1100 bp which seems to be about right. The constitutive promoter was expected to be 35 bp, but was not detected on the gel at all.</DIV></DIV></DIV></DIV><P>Used the PCR purification kit and protocol from www.qiagen.com <A href="http://www.google.no/url?sa=t&amp;rct=j&amp;q=&amp;esrc=s&amp;source=web&amp;cd=2&amp;ved=0CFUQFjAB&amp;url=http%3A%2F%2Fwww.qiagen.com%2FHB%2FQIAquickGelExtractionKit_EN&amp;ei=RhYhUPiCA4iL4gS64oD4CA&amp;usg=AFQjCNFwed-RxSRRgTFSJgtn7Fl5qisiWw&amp;sig2=-pxyzkbLpVKt4DLlcSzQcw" class="external autonumber" rel="nofollow">[2]</A>, on RBS &lt;partinfo&gt;BBa_B0030&lt;/partinfo&gt;, pBad &lt;partinfo&gt;Bba_K206000&lt;/partinfo&gt; and terminator &lt;partinfo&gt;BBa_B0014&lt;/partinfo&gt;. Did a gel extraction using the Gel Extraction kit and <A href="https://2012.igem.org/Team:NTNU_Trondheim/Protocols" class="external text" rel="nofollow">Protocol</A> on LacI &lt;partinfo&gt;BBa_C0012&lt;/partinfo&gt;.
</P><P>Did a ligation with LacI &lt;partinfo&gt;BBa_C0012&lt;/partinfo&gt; and double terminator &lt;partinfo&gt;BBa_B0015&lt;/partinfo&gt;, where the double terminator &lt;partinfo&gt;BBa_B0015&lt;/partinfo&gt; was used as a backbone. That means, the construct has a &lt;partinfo&gt;psB1AK3&lt;/partinfo&gt; backbone and resistance Ampicillin and Kanamycin. A religation of the backbone &lt;partinfo&gt;BBa_B0015&lt;/partinfo&gt; was also performed.
</P><P>The ligated LacI &lt;partinfo&gt;BBa_C0012&lt;/partinfo&gt; and double terminator &lt;partinfo&gt;BBa_B0015&lt;/partinfo&gt; was transformed in competent DH5É cells, along with the religation of the backbone &lt;partinfo&gt;BBa_B0015&lt;/partinfo&gt;. The petri dishes are left to inoculate through the night. 
</P><P>A new batch of LA-medium was made and used to make ampicillin petri dishes.
</P><P>A restriction digest was done on the lld promoter, as described in the <A href="https://2012.igem.org/Team:NTNU_Trondheim/Protocols" class="external text" rel="nofollow">protocols</A>. It was cut with EcoRI and SpeI, and will be an insert with RBS &lt;partinfo&gt;BBa_B0030&lt;/partinfo&gt; as backbone. This will then make the plasmid &lt;partinfo&gt;pSB1A2&lt;/partinfo&gt; the backbone.
</P><H3><SPAN class="mw-headline" id="Sunday_05.08.12">Sunday 05.08.12</SPAN></H3><P>Plasmid DNA was isolated from pllD-1B and &lt;partinfo&gt;BBa_B0014&lt;/partinfo&gt; cultures were isolated by miniprep. DNA concentrations were measured as 41,2/42,5 and 53,3/52,8 ng/uL respectively (two measurements per sample).
</P><P>Inspected agar plates in incubator inoculated on 3/8:
VGB+RBS religation: 1 colony
VHB+RBS+lysis: Good growth
VHB+RBS religation: ~ 20 colonies
VHB+RBS + lysis: Good growth
pBAD (w/ backbone), religated: 40+ colonies
pBAD +YFP +DTT: 7 colonies
</P><P>Comments: Judging from the growth on the religation plates, colonies with non-insert plasmids is a possibility on all plates. pBAD is especially worrisome, as the number of colonies is lower on the insert ligation plate than on the backbone religation plate. The lysis part used in the above constructs already has an RBS, so the RBS part is redundant.
</P><P>Induced one 5 mL culture (induction culture) pllD + lysis (3/8) with 210 uL ~ 1M lactate and addded 210 uL dH2O to another (control culture). Sampled 200 uL from both before induction, for OD measurements.
</P><P>Inoculated 5 5 mL LB + Amp cultures with colonies from the following agar plates:
VGB+RBS+Lysis (3/8)
VHB+RBS+Lysis (3/8)
pBAD + YFP +DTT (3/8) (3 colonies)
</P><P>Inoculated 3 5 mL LB + Cm cultures from pllD + lysis agar plate.
</P><H3><SPAN class="mw-headline" id="Friday_03.08.12">Friday 03.08.12</SPAN></H3><P>Isolated plasmid DNA from the cultures of RBS* (&lt;partinfo&gt;BBa_B0030&lt;/partinfo&gt;) and LacI (&lt;partinfo&gt;BBa_C0012&lt;/partinfo&gt;) inoculated yesterday.
</P><P>Ligated pBad (&lt;partinfo&gt;Bba_K206000&lt;/partinfo&gt;) together with YFP+DTT (&lt;partinfo&gt;BBa_E0030&lt;/partinfo&gt; and &lt;partinfo&gt;BBa_B0015&lt;/partinfo&gt;), and VGB+RBS (&lt;partinfo&gt;BBa_K561001&lt;/partinfo&gt; and &lt;partinfo&gt;BBa_B0034&lt;/partinfo&gt;) and VHB+RBS (&lt;partinfo&gt;BBa_K258005&lt;/partinfo&gt; and &lt;partinfo&gt;BBa_B0034&lt;/partinfo&gt;) with Lysis (&lt;partinfo&gt;BBa_K124017&lt;/partinfo&gt;). These were then transformed into competent E. coli DH5Î± cells and plated out on petri dishes with Ampicillin.
</P><P>Terminator &lt;partinfo&gt;BBa_B0014&lt;/partinfo&gt; and re-transformed pllD in &lt;partinfo&gt;pSB1C3&lt;/partinfo&gt; (&quot;pllD 1-B&quot;) were transferred to liquid culture.
</P><P>Ran gel on the PCR-products from yesterday.
</P><P>Inoculated 2 x 5 mL, 1 x 10 mL and 1 x 25 mL LB + Cm with plld + lysis.
</P><H3><SPAN class="mw-headline" id="Thursday_02.08.12">Thursday 02.08.12</SPAN></H3><P>5 Âµl of the PCR product from yesterday was run on gel, but it did not give any results. We looked at the primers and thought we had done something wrong, but we did not. Will try to do the PCR one more time, but with a lower annealing temperature (55Â°C).
</P><P>The constructs containing plld ligated with the lysis device(&lt;partinfo&gt;BBa_K112808&lt;/partinfo&gt;), and plld in plasmid &lt;partinfo&gt;PSB1C3&lt;/partinfo&gt; were miniprepped. The concentration were measured using nano drop.
</P><TABLE class="table table-bordered table-hover" style="margin: 1em auto 1em auto; width: auto;"><TBODY><TR><TH>Sample
</TH><TH>Concentration [ng/Âµl]
</TH></TR><TR><TD>pllD + lysis
</TD><TD>45,5
</TD></TR><TR><TD>pllD in PSB1C3
</TD><TD>42,3
</TD></TR></TBODY></TABLE><P>Inoculated 1 colony from each of three plates containing LacI (&lt;partinfo&gt;BBa_C0012&lt;/partinfo&gt;), RBS* &lt;partinfo&gt;BBa_B0030&lt;/partinfo&gt; and pllD + lysis (all inoculated 1/8), into 5 mL LB + Amp (BBa_B0030, LacI)/ LB + Cm (pllD + lysis). 
</P><P>Transformed the part &lt;partinfo&gt;BBa_B0014&lt;/partinfo&gt; from the distribution kit, and re-transformed pllD in pSB1C3 with DNA taken from the sample shipped to the RHIT team yesterday.
</P><P>Reviewed the data from the oxygen-promoter experiment on monday. In conclusion, it appears that oxygen levels in the induced (anaerobic) cultures were not as low as desired, as growth rates were very similar between the aerobic and anaerobic cultures, while one would expect anaerobic cultures to grow slower. After correcting for OD, the fluorescence in the cultures at the end of the experiment was lower than the fluorescence measured from a control culture with no expected production of fluorescent protein. For this reason, a full analysis of the rest of the data was not performed.
</P><H3><SPAN class="mw-headline" id="Wednesday_01.08.12">Wednesday 01.08.12</SPAN></H3><P>Gel purification was performed on the gel piece from yesterday, the lysis device cut with X+P. The concentration is 10,1 ng/Âµl. The lysis device and plld were ligated together, plld (with &lt;partinfo&gt;pSB1C3&lt;/partinfo&gt;) as backbone.
</P><P>The ligated lysis and plld were transformed into competent <I>E.coli</I> DH5Î± cells. This was also done for the biobricks lacI repressor from E. coli (&lt;partinfo&gt;BBa_C0012&lt;/partinfo&gt;) and RBS (&lt;partinfo&gt;BBa_B0030&lt;/partinfo&gt;).
</P><P>The petri dishes from yesterday showed colonies on the dishes with the pSB1C3+Colisin construct (&lt;partinfo&gt;pSB1C3&lt;/partinfo&gt; and &lt;partinfo&gt;BBa_K150009&lt;/partinfo&gt;) and the K+RBS+Colisin construct (&lt;partinfo&gt;BBa_K081005&lt;/partinfo&gt; and &lt;partinfo&gt;BBa_K150009&lt;/partinfo&gt;). One colony from each petri dish were transferred to liquid medium.
</P><P>Sent a sample of the E. coli pllD promoter in &lt;partinfo&gt;pSB1C3&lt;/partinfo&gt; backbone to the iGEM team at RHIT.
</P><P>PCR was performed on pBad (&lt;partinfo&gt;Bba_K206000&lt;/partinfo&gt;) and DTT (&lt;partinfo&gt;BBa_B0015&lt;/partinfo&gt;), the primers used, PCR-mix and programmed times are listed below.
</P><TABLE class="table table-bordered table-hover" style="margin: 1em auto 1em auto; width: auto;"><TBODY><TR><TH>Primer
</TH><TH>Type
</TH><TH>Sequence
</TH><TH>Tm [Â°C]
</TH></TR><TR><TD>DTT fwd
</TD><TD>Forward
</TD><TD>CCAGGCATCAAATAAAACGAAAGGCTCAGTCGAAAG
</TD><TD>72
</TD></TR><TR><TD>DTT rev
</TD><TD>Reverse
</TD><TD>CTCTAGAAGCGGCCGCGAATTCCAGAAATC
</TD><TD>72
</TD></TR><TR><TD>pBad scar fwd
</TD><TD>Forward
</TD><TD>TACTAGAGTACTAGTAGCGGCCGCTGCAGT
</TD><TD>72
</TD></TR><TR><TD>pBad fwd
</TD><TD>Forward
</TD><TD>TACTAGTAGCGGCCGCTGCAGTC
</TD><TD>70
</TD></TR><TR><TD>pBad rev
</TD><TD>Reverse
</TD><TD>GCTAGCCCAAAAAAACGGTATGGAGAAACAGTAGAGAG
</TD><TD>72
</TD></TR></TBODY></TABLE><P>Mix 1:
</P><UL><LI> 1 Âµl dNTP
</LI><LI> 2,5 Âµl 1:10 fwd primer
</LI><LI> 2,5 Âµl 1:10 rev primer
</LI><LI> 1 Âµl template DNA
</LI><LI> 18 Âµl dH<SUB>2</SUB>O
</LI></UL><P>Mix 2:
</P><UL><LI> 19,25 Âµl d<SUB>2</SUB>O
</LI><LI> 5 Âµl buffer with MgCl<SUB>2</SUB></LI><LI> 0,75 Âµl Expand High Fidelity enzyme
</LI></UL><P>This PCR mix was found here: <A href="http://francois.schweisguth.free.fr/protocols/High_fildelity_roche.pdf" class="external free" rel="nofollow">http://francois.schweisguth.free.fr/protocols/High_fildelity_roche.pdf</A></P><P>Thermal timetables:
</P><TABLE><TBODY><TR><TD style="padding-right:7px;"><TABLE class="table table-bordered table-hover" style="margin: 1em auto 1em auto; width: auto;"><CAPTION>pBAD
</CAPTION></TABLE></TD></TR><TR><TH> Step
</TH><TH> Action
</TH><TH> Temperature
</TH><TH> Duration
</TH></TR><TR><TD>1
</TD><TD>Heated lid:
</TD><TD>103Â°C
</TD></TR><TR><TD>2
</TD><TD>Initial denaturation;
</TD><TD>94Â°C
</TD><TD>2 min
</TD></TR><TR><TD>3
</TD><TD>Denaturation;
</TD><TD>94Â°C
</TD><TD>15 s
</TD></TR><TR><TD>4
</TD><TD>Annealing;
</TD><TD>66Â°C
</TD><TD>30 s
</TD></TR><TR><TD>5
</TD><TD>Elongation;
</TD><TD>72Â°C
</TD><TD>2 min
</TD></TR><TR><TD>6
</TD><TD>Go to step 3, repeat 10 x
</TD></TR><TR><TD>7
</TD><TD>Denaturation;
</TD><TD>98Â°C
</TD><TD>10 s
</TD></TR><TR><TD>8
</TD><TD>Elongation;
</TD><TD>72Â°C
</TD><TD>31 s
</TD></TR><TR><TD>9
</TD><TD>Go to step 7, repeat 15 x, with 5 s extra each time.
</TD></TR><TR><TD>10
</TD><TD>Final Elongation;
</TD><TD>72Â°C
</TD><TD>7 min
</TD></TR><TR><TD>11
</TD><TD>Hold
</TD><TD>4Â°C
</TD><TD>â
</TD></TR></TBODY></TABLE><TABLE class="table table-bordered table-hover" style="margin: 1em auto 1em auto; width: auto;"><CAPTION> DTT
</CAPTION><TBODY><TR><TH> Step
</TH><TH> Action
</TH><TH> Temperature
</TH><TH> Duration
</TH></TR><TR><TD>1
</TD><TD>Heated lid:
</TD><TD>103Â°C
</TD></TR><TR><TD>2
</TD><TD>Initial denaturation;
</TD><TD>94Â°C
</TD><TD>2 min
</TD></TR><TR><TD>3
</TD><TD>Denaturation;
</TD><TD>94Â°C
</TD><TD>15 s
</TD></TR><TR><TD>4
</TD><TD>Annealing;
</TD><TD>66Â°C
</TD><TD>30 s
</TD></TR><TR><TD>5
</TD><TD>Elongation;
</TD><TD>72Â°C
</TD><TD>3 min
</TD></TR><TR><TD>6
</TD><TD>Go to step 3, repeat 10 x
</TD></TR><TR><TD>7
</TD><TD>Denaturation;
</TD><TD>98Â°C
</TD><TD>10 s
</TD></TR><TR><TD>8
</TD><TD>Elongation;
</TD><TD>72Â°C
</TD><TD>31 s
</TD></TR><TR><TD>9
</TD><TD>Go to step 7, repeat 15 x, with 5 s extra each time.
</TD></TR><TR><TD>10
</TD><TD>Final Elongation;
</TD><TD>72Â°C
</TD><TD>7 min
</TD></TR><TR><TD>11
</TD><TD>Hold
</TD><TD>4Â°C
</TD><TD>â
</TD></TR></TBODY></TABLE></DIV></DIV></DIV><FOOTER class="footer"><DIV class="container"><P>Designed and built with <A href="http://www.mediawiki.org/wiki/MediaWiki" target="_blank">MediaWiki</A>, <A href="http://jquery.com/" target="_blank">jQuery</A> and <A href="http://twitter.github.com/bootstrap/" target="_blank">Twitter Bootstrap</A>.</P><P>Icons by <A href="http://http://fortawesome.github.com/Font-Awesome/" target="_blank">Font Awesome</A>.</P><P><A href="mailto:igem.ntnu@gmail.com"> igem.ntnu@gmail.com</A></P></DIV></FOOTER><DIV class="printfooter">
Retrieved from &quot;<A href="http://2012.igem.org/Team:NTNU_Trondheim/Notebook/August">http://2012.igem.org/Team:NTNU_Trondheim/Notebook/August</A>&quot;</DIV><DIV id="footer-box" class="noprint"><DIV id="footer"><UL id="f-list"><LI id="t-recentchanges"><A href="/Special:RecentChanges" title="Recent changes">Recent changes</A></LI><LI id="t-whatlinkshere"><A href="/Special:WhatLinksHere/Team:NTNU_Trondheim/Notebook/August" title="List of all wiki pages that link here [j]" accesskey="j">What links here</A></LI><LI id="t-recentchangeslinked"><A href="/Special:RecentChangesLinked/Team:NTNU_Trondheim/Notebook/August" title="Recent changes in pages linked from this page [k]" accesskey="k">Related changes</A></LI><LI id="t-specialpages"><A href="/Special:SpecialPages" title="List of all special pages [q]" accesskey="q">Special pages</A></LI><LI><A href="/Special:Preferences">My preferences</A></LI></UL></DIV><DIV id="footer"><UL id="f-list"><LI id="t-print"><A href="/wiki/index.php?title=Team:NTNU_Trondheim/Notebook/August&amp;printable=yes" title="Printable version of this page [p]" accesskey="p">Printable version</A></LI><LI id="t-permalink"><A href="/wiki/index.php?title=Team:NTNU_Trondheim/Notebook/August&amp;oldid=219703" title="Permanent link to this revision of the page">Permanent link</A></LI><LI id="privacy"><A href="/2012.igem.org:Privacy_policy" title="2012.igem.org:Privacy policy">Privacy policy</A></LI><LI id="disclaimer"><A href="/2012.igem.org:General_disclaimer" title="2012.igem.org:General disclaimer">Disclaimers</A></LI></UL></DIV></DIV></BODY></HTML>