<HTML xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en" dir="ltr">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki  ltr ns-0 ns-subject page-Team_NTNU_Trondheim_Logg"><DIV id="globalWrapper"><DIV id="top-section"><DIV id="p-logo"><A href="/Main_Page" title="Main Page">&quot;
	    </A></DIV><DIV id="menubar" class="left-menu noprint"><UL><LI class="selected"><A href="/Team:NTNU_Trondheim/Logg">Page               </A></LI><LI class="new"><A href="/wiki/index.php?title=Talk:Team:NTNU_Trondheim/Logg&amp;action=edit&amp;redlink=1">Discussion               </A></LI><LI><A href="/wiki/index.php?title=Team:NTNU_Trondheim/Logg&amp;action=edit">View source               </A></LI><LI><A href="/wiki/index.php?title=Team:NTNU_Trondheim/Logg&amp;action=history">History               </A></LI><LI style="color:#808080;cursor:default">teams</LI></UL></DIV><DIV class="right-menu noprint" id="menubar"><UL><LI id="pt-login"><A href="/wiki/index.php?title=Special:UserLogin&amp;returnto=Team:NTNU_Trondheim/Logg" title="You are encouraged to log in; however, it is not mandatory [o]" accesskey="o">Log in</A></LI></UL></DIV><DIV id="search-controls" class="noprint"><FORM action="/Special:Search" id="searchform"> </FORM></DIV></DIV><DIV id="content"><H1 class="firstHeading">Team:NTNU Trondheim/Logg</H1><DIV id="bodyContent"><H3 id="siteSub" class="noprint">From 2012.igem.org</H3><DIV id="main-menu"><UL class="sf-menu"><SPAN id="menu-item1"><LI class="current"><A class="sf-with-ul" href="#a">Project</A></LI></SPAN><SPAN id="menu-item2"><LI><A href="https://2012.igem.org/Team:NTNU_Trondheim/Notebook">Notebook</A></LI></SPAN><SPAN id="menu-item3"><LI><A class="sf-with-ul" href="#">Human practice</A></LI></SPAN><SPAN id="menu-item4"><LI><A href="#">Extras</A></LI></SPAN><SPAN id="menu-item5"><LI><A href="#">Team</A></LI></SPAN></UL></DIV><TABLE id="toc" class="toc"><TBODY><TR><TD><DIV id="toctitle"><H2>Contents</H2></DIV><UL><LI class="toclevel-1 tocsection-1"><A href="#Friday_06.07.12"><SPAN class="tocnumber">1</SPAN><SPAN class="toctext">Friday 06.07.12</SPAN></A></LI><LI class="toclevel-1 tocsection-2"><A href="#Thursday_05.07.12"><SPAN class="tocnumber">2</SPAN><SPAN class="toctext">Thursday 05.07.12</SPAN></A></LI><LI class="toclevel-1 tocsection-3"><A href="#Wednesday_04.07.12"><SPAN class="tocnumber">3</SPAN><SPAN class="toctext">Wednesday 04.07.12</SPAN></A></LI><LI class="toclevel-1 tocsection-4"><A href="#Tuesday_03.07.12"><SPAN class="tocnumber">4</SPAN><SPAN class="toctext">Tuesday 03.07.12</SPAN></A></LI><LI class="toclevel-1 tocsection-5"><A href="#Monday_02.07.12"><SPAN class="tocnumber">5</SPAN><SPAN class="toctext">Monday 02.07.12</SPAN></A></LI><LI class="toclevel-1 tocsection-6"><A href="#Sunday_1.07.12"><SPAN class="tocnumber">6</SPAN><SPAN class="toctext">Sunday 1.07.12</SPAN></A></LI><LI class="toclevel-1 tocsection-7"><A href="#Saturday_30.06.12"><SPAN class="tocnumber">7</SPAN><SPAN class="toctext">Saturday 30.06.12</SPAN></A></LI><LI class="toclevel-1 tocsection-8"><A href="#Friday_29.06.12"><SPAN class="tocnumber">8</SPAN><SPAN class="toctext">Friday 29.06.12</SPAN></A></LI><LI class="toclevel-1 tocsection-9"><A href="#Thursday_28.06.12"><SPAN class="tocnumber">9</SPAN><SPAN class="toctext">Thursday 28.06.12</SPAN></A></LI><LI class="toclevel-1 tocsection-10"><A href="#Wednesday_27.06.12"><SPAN class="tocnumber">10</SPAN><SPAN class="toctext">Wednesday 27.06.12</SPAN></A></LI><LI class="toclevel-1 tocsection-11"><A href="#Tuesday_26.06.12"><SPAN class="tocnumber">11</SPAN><SPAN class="toctext">Tuesday 26.06.12</SPAN></A></LI><LI class="toclevel-1 tocsection-12"><A href="#Monday_25.06.12"><SPAN class="tocnumber">12</SPAN><SPAN class="toctext">Monday 25.06.12</SPAN></A></LI><LI class="toclevel-1 tocsection-13"><A href="#Sunday_24.6"><SPAN class="tocnumber">13</SPAN><SPAN class="toctext">Sunday 24.6</SPAN></A></LI><LI class="toclevel-1 tocsection-14"><A href="#Saturday_23.06.12"><SPAN class="tocnumber">14</SPAN><SPAN class="toctext">Saturday 23.06.12</SPAN></A></LI><LI class="toclevel-1 tocsection-15"><A href="#Friday_22.06.12"><SPAN class="tocnumber">15</SPAN><SPAN class="toctext">Friday 22.06.12</SPAN></A></LI><LI class="toclevel-1 tocsection-16"><A href="#Thursday_21.06.12"><SPAN class="tocnumber">16</SPAN><SPAN class="toctext">Thursday 21.06.12</SPAN></A></LI><LI class="toclevel-1 tocsection-17"><A href="#Wednesday_20.06.12"><SPAN class="tocnumber">17</SPAN><SPAN class="toctext">Wednesday 20.06.12</SPAN></A></LI><LI class="toclevel-1 tocsection-18"><A href="#Tuesday_19.06.12"><SPAN class="tocnumber">18</SPAN><SPAN class="toctext">Tuesday 19.06.12</SPAN></A></LI><LI class="toclevel-1 tocsection-19"><A href="#Monday_18.06.12"><SPAN class="tocnumber">19</SPAN><SPAN class="toctext">Monday 18.06.12</SPAN></A></LI><LI class="toclevel-1 tocsection-20"><A href="#Wednesday_13.06.12"><SPAN class="tocnumber">20</SPAN><SPAN class="toctext">Wednesday 13.06.12</SPAN></A></LI><LI class="toclevel-1 tocsection-21"><A href="#Friday_01.06.12"><SPAN class="tocnumber">21</SPAN><SPAN class="toctext">Friday 01.06.12</SPAN></A></LI><LI class="toclevel-1 tocsection-22"><A href="#Friday_25.05.12"><SPAN class="tocnumber">22</SPAN><SPAN class="toctext">Friday 25.05.12</SPAN></A></LI><LI class="toclevel-1 tocsection-23"><A href="#Wednesday_23.05.12"><SPAN class="tocnumber">23</SPAN><SPAN class="toctext">Wednesday 23.05.12</SPAN></A></LI><LI class="toclevel-1 tocsection-24"><A href="#Friday_18.05.12"><SPAN class="tocnumber">24</SPAN><SPAN class="toctext">Friday 18.05.12</SPAN></A></LI><LI class="toclevel-1 tocsection-25"><A href="#Friday_11.05.12"><SPAN class="tocnumber">25</SPAN><SPAN class="toctext">Friday 11.05.12</SPAN></A></LI><LI class="toclevel-1 tocsection-26"><A href="#Wednesday_09.05.12"><SPAN class="tocnumber">26</SPAN><SPAN class="toctext">Wednesday 09.05.12</SPAN></A></LI><LI class="toclevel-1 tocsection-27"><A href="#Thursday_03.05.12"><SPAN class="tocnumber">27</SPAN><SPAN class="toctext">Thursday 03.05.12</SPAN></A></LI><LI class="toclevel-1 tocsection-28"><A href="#Wednesday_02.05.12"><SPAN class="tocnumber">28</SPAN><SPAN class="toctext">Wednesday 02.05.12</SPAN></A></LI><LI class="toclevel-1 tocsection-29"><A href="#Thursday_26.04.12"><SPAN class="tocnumber">29</SPAN><SPAN class="toctext">Thursday 26.04.12</SPAN></A></LI><LI class="toclevel-1 tocsection-30"><A href="#Friday_20.04.12"><SPAN class="tocnumber">30</SPAN><SPAN class="toctext">Friday 20.04.12</SPAN></A></LI><LI class="toclevel-1 tocsection-31"><A href="#Friday_13.04.12"><SPAN class="tocnumber">31</SPAN><SPAN class="toctext">Friday 13.04.12</SPAN></A></LI><LI class="toclevel-1 tocsection-32"><A href="#Friday_30.03.12"><SPAN class="tocnumber">32</SPAN><SPAN class="toctext">Friday 30.03.12</SPAN></A></LI><LI class="toclevel-1 tocsection-33"><A href="#Friday_16.03.12"><SPAN class="tocnumber">33</SPAN><SPAN class="toctext">Friday 16.03.12</SPAN></A></LI><LI class="toclevel-1 tocsection-34"><A href="#Monday_27.02.12"><SPAN class="tocnumber">34</SPAN><SPAN class="toctext">Monday 27.02.12</SPAN></A></LI><LI class="toclevel-1 tocsection-35"><A href="#Friday_17.02.12"><SPAN class="tocnumber">35</SPAN><SPAN class="toctext">Friday 17.02.12</SPAN></A></LI></UL></TD></TR></TBODY></TABLE><H3><SPAN class="mw-headline" id="Friday_06.07.12">Friday 06.07.12</SPAN></H3><P>Made a stock solution of 100 mM arabinose in H2O. The final concentration is slightly lower (by 10-15 %?) than 100 mM, as addition of 10 mL dH20 to the dry arabinose results in a final volume in excess of 10 mL. Sterilized the solution by filtering through a 0.2 um filter. Placed the stock solution in the refrigerator door shelf.
</P><P>I suggest that we do a clean-up in the incubator and a review of all the cell cultures/agar plates that we have some day soon. --<A href="/wiki/index.php?title=User:Jarlemag&amp;action=edit&amp;redlink=1" class="new" title="User:Jarlemag (page does not exist)">Jarlemag</A> 10:52, 6 July 2012 (CDT)
</P><H3><SPAN class="mw-headline" id="Thursday_05.07.12">Thursday 05.07.12</SPAN></H3><P>Did a clean-up of the 37 C incubator cabinet. Discarded the following plates:
&quot;Kan neg control 29.06&quot;
&quot;Neg ctrl 29.06&quot; (chloramphenicol)
&quot;LuxI 28.06&quot; (Amp)
&quot;LuxI Kloramfenikol 29.06&quot;
&quot;LuxI 29.06.12&quot; (Kan)
&quot;LI Amp BBa_K092400 28.06&quot;
&quot;Neg kontroll 28.06.12&quot; (Amp)
&quot;pBAD neg ctrl 29.6&quot; (Amp)
</P><P>Of the listed plates, only the following showed any growth:
</P><P>&quot;LuxI Kloramfenikol 29.06&quot; (~15 colonies)
&quot;LuxI 28.06&quot; (Amp) (2 colonies)
&quot;Neg ctrl 29.06&quot; (chloramphenicol) (3 colonies)
</P><P>It is reasonable to assume to attribute this late (and except for LuxI on Amp, in any case unexpected) growth to the break-down of the antibiotics after ~1 week at 37 C. --<A href="/wiki/index.php?title=User:Jarlemag&amp;action=edit&amp;redlink=1" class="new" title="User:Jarlemag (page does not exist)">Jarlemag</A> 10:02, 5 July 2012 (CDT)
</P><H3><SPAN class="mw-headline" id="Wednesday_04.07.12">Wednesday 04.07.12</SPAN></H3><P>Performed restriction digest on the pBAD (&lt;partinfo&gt;BBa_K206000&lt;/partinfo&gt;) and lysis &lt;partinfo&gt;BBa_K112808&lt;/partinfo&gt; parts, cutting pBAD with SpeI and PstI, and the lysis part with XbaI and PstI. For pBAD, used  10 uL from isolated DNA sample &quot;pBAD B1&quot; (remeasured DNA concentration: 25 ng/uL). For the lysis part, used 2,3 uL from &quot;lysis 1 DNA 24/6&quot; (remeasured DNA concentration: 110 ng/uL). Incubated the restriction digest on waterbath at 37 C for aproximately 1.5 hour (protocol is 1h). No post-digestion CIP treatment or enzyme inactivation was performed.
</P><P>Discarded the &quot;Lysis 1&quot; liquid culture inoculated on 1/7. Discarded 1 contaminated Erlen-Meyer bottle of LB.
</P><H3><SPAN class="mw-headline" id="Tuesday_03.07.12">Tuesday 03.07.12</SPAN></H3><P>Got started with our test constructs, and began by cutting an oxygenregulated promotor, VGB <A href="http://partsregistry.org/Part:BBa_K561001" class="external autonumber" rel="nofollow">[1]</A>, and RBS <A href="http://partsregistry.org/Part:BBa_B0034" class="external autonumber" rel="nofollow">[2]</A>. We used the RBS-part as a backbone, and the VGB-promotor as the insert.
</P><P>We tried running the VGB-promotor and RBS on gel, but the separation was poor, and we only got bands from the backbone. We then tried to cut the plasmids again, and added CIP to both of the solutions.
</P><P>The solutions were then ligated <A href="http://partsregistry.org/Help:Protocols/Ligation" class="external autonumber" rel="nofollow">[3]</A>, transformed and plated out.
</P><H3><SPAN class="mw-headline" id="Monday_02.07.12">Monday 02.07.12</SPAN></H3><P>Restriction digestion was performed on isolated DNA from the VGB and RBS biobricks (&lt;partinfo&gt;BBa_k561001&lt;/partinfo&gt; and &lt;partinfo&gt;BBa_B0034&lt;/partinfo&gt;) according to <A href="/Team:NTNU_Trondheim/Protocols" title="Team:NTNU Trondheim/Protocols">protocol</A>. Gel electrophoresis was used to determine the size of the resulting fragments. The RBS restriction digest was filtered using the QIAquick PCR purification kit to remove the smallest fragment, which is expected to pass through the filter while the desired backbone is retained. The results for VGB were not as expected, and restriction digest of this part was therefore performed again, this time with addition of <A href="http://en.wikipedia.org/wiki/Calf-intestinal_alkaline_phosphatase" class="external text" rel="nofollow">CIP</A> to the digest solution at the end of the procedure.
</P><P>Both lysis 1 plates inoculated yesterday showed extensive growth (bacterial lawn). Moved them to the refrigerator.
</P><H3><SPAN class="mw-headline" id="Sunday_1.07.12">Sunday 1.07.12</SPAN></H3><P>Miniprepped all three tubes with pBAD inoculated yesterday. Tube 3 contained 10 mL culture and was divided into two tubes, so that all processed tubes contained ~5 mL culture. Results of DNA measurements were as follows (ng/uL):
pBAD B1: 24,0
pBAD B2: 18,9
pBAD B3-1: 11,4
pBAD B3-2: 8,1
</P><P>Discarded B3-1 and B3-2, placed B1 and B2 in -20 C freezer. --<A href="/wiki/index.php?title=User:Jarlemag&amp;action=edit&amp;redlink=1" class="new" title="User:Jarlemag (page does not exist)">Jarlemag</A> 11:04, 1 July 2012 (CDT)
</P><P>In order to make a new plate of Lysis 1, transferred 20 uL from one of the tubes with liquid culture inoculated yesterday to one LA + Amp plate, and 50 uL to another. Placed the plates in the 37 C incubator cabinet. Also transferred 50 uL as inoculate to the tube used as negative control yesterday and placed it in the shaking incubator. Then combined all three tubes into one and placed in refrigerator to limit cell death before miniprep tomorrow.
--<A href="/wiki/index.php?title=User:Jarlemag&amp;action=edit&amp;redlink=1" class="new" title="User:Jarlemag (page does not exist)">Jarlemag</A> 11:37, 1 July 2012 (CDT)
</P><H3><SPAN class="mw-headline" id="Saturday_30.06.12">Saturday 30.06.12</SPAN></H3><P>Removed pBAD plates from 37 C incubator. Good growth on all transformed plates, Negative control showed no growth. Inoculated three colonies from plate B in liquid LB + Amp. Discarded the &quot;mixed&quot; plate.  Placed plates A and B in refrigerator. Also inoculated three colonies from the lysis biobrick &lt;partinfo&gt;BBa_K112808&lt;/partinfo&gt; using the plate from 22/6. Used an inoculating needle for all inoculations today. --<A href="/wiki/index.php?title=User:Jarlemag&amp;action=edit&amp;redlink=1" class="new" title="User:Jarlemag (page does not exist)">Jarlemag</A> 11:03, 1 July 2012 (CDT)
</P><H3><SPAN class="mw-headline" id="Friday_29.06.12">Friday 29.06.12</SPAN></H3><P>Researched the ArcA protein and its binding sequence(s) to investigate whether the lldPRD operon promoter can be modified to eliminate repression by ArcA-P in the anaerobic state. Located a putative ArcA-P binding site in the promoter sequence: The sequence GTTAACTAAATGTTA is the reverse complement of the minus strand of the 85 bp promoter sequence from position 38 to 52. This sequence is identified by <A href="http://onlinelibrary.wiley.com/doi/10.1046/j.1365-2958.1999.01347.x/full#t1" class="external text" rel="nofollow">McGuire et al (1999)</A> (see <A href="http://arep.med.harvard.edu/ecoli_matrices/dat/arcA.dat" class="external text" rel="nofollow">this</A> list). In the GenBank entry this part of the sequence reads TAACATTTAGTTAAC
</P><P>Favorov et al. have made a newer suggestion for a general motif for ArcA binding sites - see <A href="http://bioinformatics.oxfordjournals.org/content/21/10/2240.full#sec-5" class="external text" rel="nofollow">this article</A> and the computational result <A href="http://favorov.bioinfolab.net/SeSiMCMC/examples/arcA/result.html" class="external text" rel="nofollow">here</A>.  According to Favorov, the crucial features of the site is a direct repeat, as shown below:
</P><P>atacaTAACatttagtTAACcattc	
</P><P>Extracted the pBAD strong promoter biobrick &lt;partinfo&gt;BBa_K206000&lt;/partinfo&gt; and transformed it in two samples. Plated out on three Amp plates as follows: 200 uL from sample A, 200 uL from sample B, 20 uL each from sample A and B.
</P><P>--<A href="/wiki/index.php?title=User:Jarlemag&amp;action=edit&amp;redlink=1" class="new" title="User:Jarlemag (page does not exist)">Jarlemag</A> 12:52, 29 June 2012 (CDT)
</P><H3><SPAN class="mw-headline" id="Thursday_28.06.12">Thursday 28.06.12</SPAN></H3><P>Researched the part &lt;partinfo&gt;BBa_K112805&lt;/partinfo&gt; (subpart of Bba_K112808) containing the T4 Bacteriophage lysis t gene (locus T4p254 - <A href="http://www.ncbi.nlm.nih.gov/nuccore/29366675?report=gbwithparts&amp;from=160221&amp;to=160877" class="external text" rel="nofollow">GenBank entry</A>) encoding holin (<A href="http://www.genome.jp/dbget-bin/www_bget?uniprot:VLYS_BPT4" class="external text" rel="nofollow">Uniprot entry</A>) to determine if it must be assembled together with an RBS. The part sequence appears to have a RBS at bases 7-12 (AGGAG, which is the RBS <A href="http://www.scielo.br/scielo.php?pid=S1415-47572004000400028&amp;script=sci_arttext" class="external text" rel="nofollow">consensus sequence</A>) while the sequence found in GenBank (atg...) starts at base 17 of the part sequence. The recorded part sequence has a single base change, A-&gt;C, compared to the GenBank sequence, at position 70 (position 54 of the GenBank sequence). Otherwise, the part sequence is identical to the cited GenBank sequence, with the addition of the 16 preceding bases in the GenBank T4 complete genome sequence. For a review of the T4 genome, see <A href="http://mmbr.asm.org/content/67/1/86.long" class="external free" rel="nofollow">http://mmbr.asm.org/content/67/1/86.long</A> --<A href="/wiki/index.php?title=User:Jarlemag&amp;action=edit&amp;redlink=1" class="new" title="User:Jarlemag (page does not exist)">Jarlemag</A> 05:34, 28 June 2012 (CDT)
</P><P>
Found a reference indicating that the lldPRD operon is inhibited during anaerobic conditions (<A href="http://jb.asm.org/content/190/8/2997.full.pdf+html" class="external free" rel="nofollow">http://jb.asm.org/content/190/8/2997.full.pdf+html</A>):
</P><P><I>the lldPRD operon is proposed to be under the control of the global regulator ArcA, which binds to positions -14 to 3 with respect to the transcriptional start site, corresponding to the proposed P2 promoter, and represses the expression of this operon under anaerobic conditions.</I></P><P>The cited reference is <A href="http://jb.asm.org/content/178/21/6238.full.pdf+html" class="external free" rel="nofollow">http://jb.asm.org/content/178/21/6238.full.pdf+html</A> (in this article the operon is called lctPRD)
</P><BLOCKQUOTE><I>Of all the operons known to be regulated by the Arc system,</I>
the lctPRD operon shows the widest range of response, with
90- to 100-fold anaerobic repression apparent (23, 26<I>)</I></BLOCKQUOTE><P>Of all the operons known to be regulated by the Arc system,
the lctPRD operon shows the widest range of response, with
90- to 100-fold anaerobic repression apparent (23, 26)
</P><P>May this affect the usability of the promoter? --<A href="/wiki/index.php?title=User:Jarlemag&amp;action=edit&amp;redlink=1" class="new" title="User:Jarlemag (page does not exist)">Jarlemag</A> 07:38, 28 June 2012 (CDT)
If so, perhaps it could be possible to modify the sequence so that ArcA does not bind.
</P><H3><SPAN class="mw-headline" id="Wednesday_27.06.12">Wednesday 27.06.12</SPAN></H3><P>Found the sequence for the lldp lactate operon at <A href="http://www.ncbi.nlm.nih.gov/nuccore/L13970" class="external free" rel="nofollow">http://www.ncbi.nlm.nih.gov/nuccore/L13970</A></P><P>Performed isolation of plasmid DNA from RBS (&lt;partinfo&gt;BBa_B0034&lt;/partinfo&gt;,  C78, C79 and &lt;partinfo&gt;BBa_K561001&lt;/partinfo&gt; (VGF promoter). The yield from RBS was low today as well, so the spin column was centrifuged one more time together with the VGB-promoter. 
</P><P>Results from the nanodrop:
</P><P>RBS: 30,2 ng/uL
C78: 63,4 ng/uL
C79: 70,5 ng/uL
VGB: 22,9 ng/uL
</P><P>
Researched E. coli strains and their properties. Must DNA methylation be taken into account? Conclusion: No, not in the <A href="http://tools.ietf.org/html/rfc10" class="external mw-magiclink-rfc">RFC 10</A> standard.
</P><P>Some useful information on common laboratory strain mutations <A href="http://books.google.no/books?id=OWXM4Q3-ieEC&amp;pg=PA65&amp;lpg=PA65&amp;dq=lacZYA-argF&amp;source=bl&amp;ots=xqv8JrjVYZ&amp;sig=P3MxMnbSDNh18yW-ek7kOGvmNec&amp;hl=no&amp;sa=X&amp;ei=jPvqT5KsM-Tm4QSN3IT3Ag&amp;ved=0CFsQ6AEwAg#v=onepage&amp;q=lacZYA-argF&amp;f=false" class="external text" rel="nofollow">here</A>. --<A href="/wiki/index.php?title=User:Jarlemag&amp;action=edit&amp;redlink=1" class="new" title="User:Jarlemag (page does not exist)">Jarlemag</A> 08:11, 27 June 2012 (CDT)
</P><H3><SPAN class="mw-headline" id="Tuesday_26.06.12">Tuesday 26.06.12</SPAN></H3><P>Colonies from C78, C79 and &lt;partinfo&gt;BBa_K561001&lt;/partinfo&gt; (VGF promoter) were transferred from agar plates to liquid medium. There were two plates including the negative control that had not been incubated correctly (L1 and L2). They had not been placed with their lid facing downwards. These plates did not have any growth. The transformed cells from yesterday (L1 and L2) were then plated out onto new agar plates in hope to get colonies for further use.
</P><P>Performed isolation of plasmid DNA from RBS (&lt;partinfo&gt;BBa_B0034&lt;/partinfo&gt;, DTT (Short for &quot;Double transcriptional terminator&quot;, &lt;partinfo&gt;BBa_B0015&lt;/partinfo&gt;) grown on ampicillin and DTT grown on kanamycin. The yield was quite poor, especially for RBS. We then decided to run the spin columns one more time to see if we could get a higher yeild the second time. Since we are going to use a lot of RBS, we made a liquid culture right away, so we can perform a new DNA isolation on RBS tomorrow.
</P><P>Results from the nanodrop:
</P><P>RBS concentration: 16,8 ng/uL
DTT + amp concentration: 35,8 ng/uL
DTT + kan concentration: 26,5 ng/uL
</P><H3><SPAN class="mw-headline" id="Monday_25.06.12">Monday 25.06.12</SPAN></H3><P>Made chloramphenicol stock solution. Weighted out 0,135 g and dissolved in 4 mL 100 % ethanol to a concentration of 34 mg/mL. In media, a concentration of 24 ug/mL should be used. For this, 0,7 mL stock solution per liter medium is needed.
</P><P>For todays transformations, used 0.1 mL supercompetent cells as there were no 0.5 mL tubes left.
</P><P>Checked RBS and DTT plates from yesterday. No growth on negative control plate. 20++ on other three (RBS on La+Amp, DTT on La+Amp and La+ Kan).
</P><P>For negative control when making liquid cultures, placed a clean toothpick in each of two tubes with LB + Amp and LA+ Kan respectively.
</P><P>Placed the isolated DNA from the LuxR+HSL miniprep in the -20 C freezer. 
</P><H3><SPAN class="mw-headline" id="Sunday_24.6">Sunday 24.6</SPAN></H3><P>Tube with culture inoculated from the 20 uL &lt;partinfo&gt;BBa_k112808&lt;/partinfo&gt; showed growth, despite the fact that no colonies were visible on the plate, and the &quot;colony&quot; that was scratched for inoculation turned out to be a water bubble below the agar.
</P><P>After miniprep of this culture, DNA concentration of two samples was measured with NanoDrop as 111 and 50 ng/uL.
</P><P>Miniprepped culture of LuxR &amp; HSL regulated promoter (&lt;partinfo&gt;BBa_R0062&lt;/partinfo&gt;).  Measured concentration of isolated DNA as 15,7 ng/uL.
</P><P>Extracted bricks &lt;partinfo&gt;BBa_B0034&lt;/partinfo&gt; (abbreviated &quot;RBS&quot;, Plate 1, well 2M) and BBa_B0015 (abbreviated &quot;DTT&quot;, plate 1, well 23L) from the distribution kit for transformation. Stored the remainder of the resuspended BBa_B0034 DNA in the -20 C freeze. There was not enough DNA left over of BBa_B0015 to store. --<A href="/wiki/index.php?title=User:Jarlemag&amp;action=edit&amp;redlink=1" class="new" title="User:Jarlemag (page does not exist)">Jarlemag</A> 07:03, 5 July 2012 (CDT)
</P><H3><SPAN class="mw-headline" id="Saturday_23.06.12">Saturday 23.06.12</SPAN></H3><P>Of two LA + Amp plates with &lt;partinfo&gt;BBaK112808&lt;/partinfo&gt; transformants (20 and 200 uL) incubated since yesterday, only the 200 uL plate showed growth (10+ colonies). Negative control plate (untransformed cells) showed no growth. One colony was transferred to liquid culture.
</P><P>On the 20 uL plate, a water bubble below the agar was mistaken for a colony, and a liquid culture was inoculated with a toothpick after scratching the plate. The mistake was realized, but the tube was still left to incubate, as it was thought it could act as a (weak) negative control.
</P><H3><SPAN class="mw-headline" id="Friday_22.06.12">Friday 22.06.12</SPAN></H3><P>The BBa_R0062 transformant from yesterday yielded several colonies on its plate, and one colony was inoculated into liquid medium and incubated at 37 C with shaking. After adding antibiotic to the liquid medium in the growth tube, some of the medium was spilled, so the growth volume was about 2 mL.
</P><P>Isolation of plasmid DNA from all 7 initial transformants (see 20.06.12) was performed with the Promega Wizard Plus SV Minipreps DNA Purification System A1460.
</P><P>The biobrick &lt;partinfo&gt;BBa_K112808&lt;/partinfo&gt; (Enterobacteria phage T4 Lysis Device - no promoter) was extracted from the distribution kit and transformed into <I>E. coli</I>.
</P><H3><SPAN class="mw-headline" id="Thursday_21.06.12">Thursday 21.06.12</SPAN></H3><P>We started the day with a lecture on genetic circuit modelling by Ph.d student and iGEM team instructor Marius Eidsaa, followed by a discussion of the options available in designing our system.
</P><P>All 7 transformants from yesterday yielded 1 or more colonies. Negative controls (untransformed cells) plated out on Kanamycin and Ampicillin plates showed no growth, indicating that the antibiotics were effective in selecting transformants for growth. 1 colony from each of the transformants was inoculated in liquid LB medium and incubated at 37 C with shaking.
</P><P>The biobrick <A href="http://partsregistry.org/Part:BBa_R0062" class="external text" rel="nofollow">BBa_R0062</A> (Promoter (luxR &amp; HSL regulated -- lux pR)) was extracted from the distribution kit and transformed using our modified version of the official iGEM transformation protocol (see <A href="/Team:NTNU_Trondheim/Protocols" title="Team:NTNU Trondheim/Protocols">Protocols</A>).
</P><H3><SPAN class="mw-headline" id="Wednesday_20.06.12">Wednesday 20.06.12</SPAN></H3><P>We started the day with an introduction to computer modeling of biological sytems, and use of the Cain chemical kinetics simulation program. We then researched and discussed various biobricks, and transformed several biobricks from the iGEM DNA distribution plates into E coli.
</P><P>The following biobricks were extracted from the iGEM 2012 DNA distribution kit and transformed:
</P><P><A href="http://partsregistry.org/partsdb/get_part.cgi?part=BBa_K091146" class="external text" rel="nofollow">BBa_K091146</A><A href="http://partsregistry.org/Part:BBa_K145150" class="external text" rel="nofollow">BBa_K145150</A><A href="http://partsregistry.org/partsdb/get_part.cgi?part=BBa_J23119" class="external text" rel="nofollow">BBa_J23119</A><A href="http://partsregistry.org/Part:BBa_E1010" class="external text" rel="nofollow">BBa_E1010</A><A href="http://partsregistry.org/Part:BBa_E0030" class="external text" rel="nofollow">BBa_E0030</A><A href="http://partsregistry.org/Part:BBa_E0020" class="external text" rel="nofollow">BBa_E0020</A><A href="http://partsregistry.org/Part:BBa_K082003" class="external text" rel="nofollow">BBa_K082003</A></P><P>For transformation, the iGEM HQ <A href="http://partsregistry.org/Help:Protocols/Transformation" class="external text" rel="nofollow">transformation protocol</A> was followed, with the exception of the use of LB media in place of SOC and heat shock for 45 s instead of 60 s.
</P><P>After media had been added to all tubes and the tubes placed on ice, it was discovered that the lid of the tube with Bba_E1010 was not closed tightly. The lid was then closed, and we believe that no media leaked out. 
</P><P>As this was the first time several of the team members performed the procedure, longer time than prescribed was spent on some parts of the protocol. This does not include the heat shock treatment, which is most time-critical.
</P><H3><SPAN class="mw-headline" id="Tuesday_19.06.12">Tuesday 19.06.12</SPAN></H3><P>We started the day preparing lab equipment. We sterilized pipette tips, toothpicks, water and LA and LB medium. We also prepared stock solutions of ampicillin and kanamycin, and made petri dishes containing LA + Amp (100 ug/mL) and LA + Kan (100 ug/mL). Equipment and solutions were autoclaved at 120 C for 20 minutes.
</P><P>We prepared LB medium (1 L) and LA medium (2 L). The media were prepared by adding 1 L MQ water to the solid contents of each batch, giving a total volume slightly in excess of 1 L. We discussed whether we should instead add a smaller amount of water, dissolve the nutrients and then add water to an exact total volume of 1 L, but concluded that this would probably not be necessary.
</P><P>After mixing, each batch was divided into two bottles for autoclaving. For the LA media, we had to retry this a couple of times, as the first times we were too slow: Some of the agar content solidified in the first bottle before pouring into the second, giving a lower agar content in the second bottle. We solved this problem by pouring all the liquid back into one bottle, reheating under stirring to redisolve the agar and then pouring the liquid quickly from one bottle to the other before the agar had time to solidify.
</P><P>Stock solutions of Ampicillin and Kanamycin (both 100 mg/mL) were prepared from dry powders. After preparation and between use, the solutions were stored at -20 C.
</P><P>We forgot to add a magnetic stirrer to one of the LA bottles before autoclaving, and thus hadd to add a magnet (flame sterilized) to the bottle afterwards, slightly increasing the chance of contamination.
</P><P>After autoclaving, LA media bottles were left to cool. After reaching a temperature where they could be comfortably handled, the desired antibiotic was added using sterile technique. (500 uL stock solution to each bottle containing aproximiately 0.5 L, to a final antibiotic concentration of 100 ug/mL). The medium was then stirred and poured into petri dishes. After pouring, the petri dished were left to cool, and after solidifying enough for safe transport, placed in cold storage.
</P><P>During addition of antibiotics to the media, two bottles of LA medium were confused, causing some doubt if Ampicillin had been added to one of the bottles or not. We believe that the problem was resolved correctly and that Ampicillin was added to both bottles. Still, if contamination becomes a problem with the ampicillin plates produced today, the question of whether all of them actually contain the antibiotic must be considered.
</P><P>The above problem highlighted a challenge when working as a team in the lab and reinforced the importance of good labeling. We instituted a color coding system for antibiotics: All media and plates containing ampicillin should be marked with a green bar, while Kanmycin is indicated by a red bar.
</P><H3><SPAN class="mw-headline" id="Monday_18.06.12">Monday 18.06.12</SPAN></H3><P>The team visited the lab and was given a EHS run-through by Merethe Christensen. We also risk evaluated the project and handed in the risk assessment, so now we are ready to start working in the lab:-)
</P><H3><SPAN class="mw-headline" id="Wednesday_13.06.12">Wednesday 13.06.12</SPAN></H3><P>Today, we tried to come up with a preliminary genetic circuit. We decided that using HGF as a signal molecule could be difficult, since we don't even know if a protein this size could penetrate the outer membrane and the peptidoglycan layer of <B>E.coli</B>. But we have found out that cancer cells excrete more lactate than healthy cells, so we decided to go for lactate, which is a small molecule. If the construct works, it could be modified to respond to other signal molecules.
</P><P>Preliminary genetic construct:
</P><H3><SPAN class="mw-headline" id="Friday_01.06.12">Friday 01.06.12</SPAN></H3><P>We talked about what we had found out since last week. Rolf and Jarle had since last week investigated different ways of making a cell lyse, and they found out that genes for lysis are already in this years iGEM kit.
Ove and Nina had been investigating toxins, but didn't find as much as they hoped for, but they have however found some toxins. Colisin E1 should be possible to use. They have also sent an email to Pål Fallnes, who Marit Otterlei suggested we could try to get in touch with. Apparently, he has been working quite a lot with expression of toxins in bacteria.
Gunvor and Eirin have since last week been trying to find a suitable signal molecule we could detect. Marit Otterlei suggested that we could use HGF, so Eirin and Gunvor did some reasearch around this growth factor. They found out that HGF is the only known lignad to the receptor c-Met, which is a tyrosin kinase. The idea so far is to use this, find out what signaling pathways this receptor is connected to, and find the endpoint of the signaling pathway. We assume that the endpoint is a growth factor possibly regulating a promoter, and if we find such a promoter, this could be set to control the lysis genes. The problem is that we don't know how signals are transducted to bacteria from the exterior, and this proved to be hard to find any information about. But we know that proteins resembling tyrosin kinases called BY kinases exists in bacteria, and also that proteins from eucaryotic cells have sometimes been working in bacteria when introns are removed by using mRNA and reverse transcriptase.
</P><P>Both in the case of toxins we can express, and a signal molecule we could get our cells to respond to; we need more time. So we decided that Ove and Nina will continue looking for toxins for another week, and Eirin and Gunvor will investigate signal molecules more in depth.
</P><P>Rahmi have found some possibly useful biobricks, and will be sending references to the biobricks in question to the team by email. Rahmi had also baked a cake for today's meeting:-)
</P><P>In the case of sponsors, many of the companies we have asked request a budget. Gunvor will try to get hold of this from Eivind, and she will also talk to Merethe Christensen, who is an engineer at dept. of biotechnology, to arrange a safety excursion in the lab, which is necessary before we're allowed to work there.
</P><P>We also elected Nina as our photo chief:-)
</P><H3><SPAN class="mw-headline" id="Friday_25.05.12">Friday 25.05.12</SPAN></H3><P>We started continuing our dicussion on which toxins we should make our cells produce. If we manage to make the cells lyse only in the presence of the signaling molecule we chose to go with, it shouldn't mather what the cells produce, but if the system turns out to be leaky, it will be a problem if the toxins we prodce is too toxic. That way we would also kill healthy cells. Also, another problem with too toxic molecules is that we will need a special lab to work with them. So we decided to go for the happy mean.
We also discussed the posiibility of making the intracellular concentration of toxins a checkpoint for lysis, so that even if a signal molecule is present, or the cells is experiencing an oxygen deficient environment, they would still not lyse until the concentration of anti cancer molecules is high enough.
</P><P>We now have three different modules involved in our project; a production part, a detection part and a lysis part. We decided to split in three groups and investigate these modules in depth;
</P><UL><LI> Production - Nina and Ove
</LI></UL><DL><DD>Nina and Ove will investigate further what kind of molecules we could make our cells produce, and since we got the names of some people working with toxins on the meeting with prof. Otterlei, we decided to contact them to hear if they can help us. We will also be contacting Jay Bradner.
</DD></DL><UL><LI> Detection - Eirin and Gunvor
</LI></UL><DL><DD>Eirin and Gunvor will look at possible signaling molecules we could use for detection of cancer cells. So far, we have decided to investigate HGF, the growth factor prof. Otterlei suggested, and VEGF.
</DD></DL><UL><LI> Lysis - Jarle and Rolf
</LI></UL><DL><DD>Jarle and Rolf will look at different ways to make a cell lyse, and they will also investigate the suicide switch used by the Tokyo team. 
</DD></DL><P>We also decided that we need a PR chief. Rolf volunteered! 
</P><H3><SPAN class="mw-headline" id="Wednesday_23.05.12">Wednesday 23.05.12</SPAN></H3><P>We had our first meeting with our new advisor, Marit Otterlei, and we discussed which signal molecule we could use to detect the cancer cells in addition to using the O<SUB>2</SUB> promoter to detect oxygen deficient areas.
Prof. Otterlei suggested that we could use the Hepatocyte Growth Factor (abbreviated HGF), which is a growth factor regulating cell growth, cell motility and morphogenesis, that has the ability to bind to a tyrosine kinase.
Prof. Otterlei also told us that many toxins can be produced by cells, so we decided to look more at making the cells produce toxins in addition to the ones we've already talked about.
</P><H3><SPAN class="mw-headline" id="Friday_18.05.12">Friday 18.05.12</SPAN></H3><P>We began this weeks meeting with going through the to-do-list from last meeting.
</P><UL><LI> Eirin:
</LI></UL><DL><DD>Has concacted Christopher Anderson by email to ask how his system works, but apparently he was unnwilling to give away too many details, as his work is not yet published.
</DD><DD>But we also came to the decision that specially because his work is still not published, we can't really be sure what his system looks like, and it is not our fault if it turns out that our systems actually work the same way, therefore, we shouldn't care too much that he is working on a similar project. But still; Rahmi and Eirin will try to get some more details out of him.
</DD></DL><UL><LI> Rolf and Jarle:
</LI></UL><DL><DD>Have been looking at possible sponsors. So far; we know that we will get a discount at VWR, so if we need something they sell, we should order it from them. Also, we have gotten a semi-positive response from Sigma-Aldrich.
</DD><DD>Before next meeting, Rolf will talk to more companies.
</DD></DL><UL><LI> Gunvor:
</LI></UL><DL><DD>Has since the last iGEM-meeting had a meeting with Gaute Brede, who is a postdoc working in the cell biology group at institute of cancer research and molecular medicine. I tried to ask him about signal molecules we could use, and his idea was to use ATP or adenosine, as cancer cells secrete more ATP than a normal cell. This is a signal molecule that attracts the attention of the immune system, so to avoid an imune reaction, the cancer cells also secrete enzymes called ectonucleotidases, which job is to degrade ATP to adenosine, which does not attract the immune system. Also healthy cells secrete ATP, but in smaller amounts than cancer cells. And healthy cells does not secrete CD39 and CD73, which is the ectonucleotidases responsible for degradation of ATP. So in summary; since cancer cells has these ectonucleotidases, they have lower levels of ATP in their surroundings compared to healthy cells, and higher levels of adenosine in their surroundings compared to healthy cells. Also, it seems like most types of cancer cells secrete CD39 and CD873.
</DD></DL><P>We discussed the possibility of using adenosine as a signal molecule, but Rhami was sceptical, as he thinks adenosine and ATP are taken up by the cells through passive transport. So that means we might have to look for another signal molecule. Since we have few ideas ourself, we decided to have a meeting early next week instead, and try to get our new advisor Marit Ottelei to attend the meeting, since she might have some ideas.
</P><P>In the time leading up to the next meeting, Rolf will continue looking for sponsors, and the ones who has time should start looking at different ways to lyse a cell.
</P><H3><SPAN class="mw-headline" id="Friday_11.05.12">Friday 11.05.12</SPAN></H3><P>We had our weekly meeting. We started by going through what we have done since last week:
</P><UL><LI> Eirin:
</LI></UL><DL><DD>Since last week, Eirin have been reading Christopher Andersons article. It seems like several parts of his project is similar to what we are planing now. But we still don't know if the cells Anderson used lysed by a stimulus given by the cancer cells. We have to find out if he did this or not, so before the next meeting, Eirin vil send an email to ask him.
</DD></DL><UL><LI> Ove:
</LI></UL><DL><DD>Ove will be looking more at making a calender function, so we can see a calendar on top of the page and be able to click different dates. Gunvor suggested that he could take a look at the code the NTNU team used to generate a calendar last year. 
</DD></DL><UL><LI> Nina:
</LI></UL><DL><DD>Nina has found several new potential anti cancer drugs that can be syntheszed by cells, but she will be looking for more potential drugs in the time leading up to next meeting.
</DD></DL><UL><LI> Gunvor:
</LI></UL><DL><DD>Has asked around at Institute of Physics for cancer cells we could use in some of the experiments we are planning. She talked to the engineer responsible for the cell lab, and she said that they would like to provide us with cancer cells, but it would be an advantage if we could get some bottles of cancer cells from someone who is already using them. The problem is, she is going to leave for summer vacation in the middle of june, and if she's not here, we have to start our own cell line, which means that one of us will have to spend some hours a day in the cell lab attending to the cancer cells for about two weeks, which is the time it takes from the cells have been thawed to they are ready to be used in experiments.
</DD><DD>Since this solution doesn't seem optimal, Gunvor instead talked to a postdoc at her research group who has collaborations to researchers in the hospital. She has planned a meeting with one of her collaborators next wednesday, and Gunvor will follow her there to talk to them.
</DD><DD>She have also sent Marit Otterlei (professor at Institute of cancer research and molecular medicine) an email, and contacted Terje Espevik, who works with confocal microscopy at the hospital, about how we can characterize the O<SUB>2</SUB>-promoter using fluorescence microscopy.
</DD></DL><P>We also decided that we should start looking for sponsors. We will submit an application for funds to Programme of Bioinformatics (PBI), and Rolf and Jarle is going to contact VWR, Sigma-Aldrich, and Fisher Scientific.
</P><P>Eivind reminded us that it is important to come up with an idea for what our genetic construct will actually look like as soon as possible, so we can start the modelling. 
We decided that we will decide on an idea for a genetic circuit on next meeting, which will be on friday 18.05, at 13:30.
</P><P>Have a nice weekend:-)
</P><H3><SPAN class="mw-headline" id="Wednesday_09.05.12">Wednesday 09.05.12</SPAN></H3><P>I have been playing around with a wiki design scheme today which can be found <A href="/Team:NTNU_Trondheim/Test" title="Team:NTNU Trondheim/Test">here</A>. I hope we can discuss the wiki design a little bit this friday. Also I have been trying to make a calendar solution, but I haven't found any easier or more user-friendly way to implent this than to simply use the default wiki setup. So, at least for now, I think we should just keep using this site the way it is and add updates the way Gunvor did below (and I am doing now) ;) When you have added a new post to a day, you can click the button &quot;Your signature with timestamp&quot; in the editing menu to add your username along with the current time and date.
</P><P>Hope you are all having a nice day and get to enjoy some sun!
</P><P>--<A href="/wiki/index.php?title=User:Oyas&amp;action=edit&amp;redlink=1" class="new" title="User:Oyas (page does not exist)">Oyas</A> 09:45, 9 May 2012 (CDT)
</P><H3><SPAN class="mw-headline" id="Thursday_03.05.12">Thursday 03.05.12</SPAN></H3><P>We had a meeting, and we discussed several things we would like to look more into before we start planning what our genetic circuit will actually look like. 
</P><P>Here is a list of what we decided to do, and who will do it:
</P><UL><LI> Check Christopher Anderson's article to see how similar his project was to ours - Eirin
</LI><LI> Make a calendar on our wiki page - Ove
</LI><LI> Look deeper into what kind of anti cancer drugs we could make our cells synthetize - Nina
</LI><LI> Check if we have access to cancer cells, and find out more about molecules that are secreted form cancer cells in larger amounts than by normal cells - Gunvor
</LI><LI> Find out more about the O<SUB>2</SUB>-sensitive promoter - Rahmi
</LI><LI> Rolf was also going to do something, but I have forgotten what he was going to do...
</LI></UL><P>- We also decided to have our next meeting at 13:30 next friday, and we decided to eat lunch together on wednesdays
</P><P>- Rahmi came with a suggestion for characterization of the O<SUB>2</SUB>-sensitive promoter:
</P><DL><DD>We could make the promoter control the expression of a fluorescent protein, then transform the system to cells, grow the cells on a soft medium they could migrate into, let them grow for a while, and then investigate the medium for coloured cells.
</DD><DD>Suggestion from Gunvor: We could use TIRF (Total Internal Reflection Fluorescence) microscope, which allows us to investigate the fluorescence in a certain layer of the medium.
</DD></DL><H3><SPAN class="mw-headline" id="Wednesday_02.05.12">Wednesday 02.05.12</SPAN></H3><P>Gunvor and Rahmi held an introductory lecture to cloning techniques. Gunvor held a crash course in molecular biology, the biobrick concept, and the most common molecular biology techniques, while Rahmi covered more advanced cloning techniques like SLIC and Gibson. If anyone wants more information on for example SLIC and Gibson, google j5 assembly;-)
</P><H3><SPAN class="mw-headline" id="Thursday_26.04.12">Thursday 26.04.12</SPAN></H3><P>Today, all the groups from last time started with giving an overview of possible project within their topic. Then we discussed for quite a long time, and in the end we decided we wanted to work with cancer. But we also decided to keep the biosynthesis of fatty acids as a side project. The electricity project turned out to be quite hard to complete in only two months, so we decided to drop it. 
</P><P>Our final project idea is then to make bacteria, for example E.coli cells, produce anti cancer drugs; preferably as many different molecules as possible. We have read about both enzymes and endpoints of metabolic pathways that are disadvantageous for cancer cells. 
Our engineered cells should also be able to respond to a signal molecule secreted in larger amounts from cancer cells than from normal cells, for example a signaling molecule that promotes angiogenesis.
When the cells detect the signal molecule from the cancer cells, they should lyse, releasing the anti cancer molecule close to the tumour. 
Another approach to reach the tumour could be to take advantage of the fact that the environment inside tumours normally is oxygen deficient. And as E.coli cells naturally migrate towards areas low in oxygen, a possible solution is also to activate the lysis gene when the cells are in an are with little oxygen. 
Rahmi told us about a promoter that gets activated by low concentration of O<SUB>2</SUB>. This could regulate the lysis genes.
</P><P>For the main project, we have to make the cells lyse at a certain concentration of an outer stimuli. For our side project it could be interresting to do the oposite; making the cells lyse when they have produced a certain amount of fatty acids.
</P><H3><SPAN class="mw-headline" id="Friday_20.04.12">Friday 20.04.12</SPAN></H3><P>Today, we decided on the overall top three projects for the team. We added the points all the team members have given to the different projects, and the list then becomes as follows:
</P><TABLE border="1"><TBODY><TR><TH>Project
</TH><TH>Number of points
</TH></TR><TR><TD>Cancer (search and destroy)
</TD><TD>14 points
</TD></TR><TR><TD>Biosynthesis of fatty acids
</TD><TD>10 points
</TD></TR><TR><TD>Bacteria as electrical switch
</TD><TD> 6 points
</TD></TR><TR><TD>Water purification - hormones
</TD><TD> 3 points
</TD></TR><TR><TD>Water purification - sucralose
</TD><TD> 1 point
</TD></TR><TR><TD>Oil spill removing bacteria
</TD><TD> 1 point
</TD></TR><TR><TD>Biomining
</TD><TD> 1 point
</TD></TR></TBODY></TABLE><P>We decided to look deeper into the overall top three projects. We formed three groups of two and two, and decided to look at the following things for the next meeting:
</P><UL><LI> Look at earlier projects on the topic
</LI><LI> Come up with some specific project ideas
</LI><LI> Look at the feasibility of the proposed projects
</LI><LI> Look at which biobricks we would need for each of the proposed projects, and see if they are present in the registry, or if we would have to make them ourselves. 
</LI></UL><P>The groups:
</P><UL><LI> Eirin and Jarle - Biosynthesis of fatty acids
</LI><LI> Rolf and Ove - Bacteria as electrical switch
</LI><LI> Nina and Gunvor - Cancer (search and destroy)
</LI></UL><P>We also elected Ove as wiki chief:-)
</P><H3><SPAN class="mw-headline" id="Friday_13.04.12">Friday 13.04.12</SPAN></H3><P>Since we have quite a few project ideas to choose between now, we decided that for next meeting, all team members should pick their three favourite projects. We decided to give our favourite project three points, the project on second place two points, and our third favourite project one point. Before the next meeting we will add all the points together, and se which projects we should look at more in depth. 
At this meeting, we also discussed activities for Researchers Night. So far we have thought about making a construction kit to make the attendants on RN understand how we can combine different biobricks.
We also discussed bringing pipettes, letting the attendants make alginate beads with different colours that they could bring home, and bringing a microscope and coloured cells (harmless, of course).
</P><H3><SPAN class="mw-headline" id="Friday_30.03.12">Friday 30.03.12</SPAN></H3><P>We kept discussing possible projects, and we now have several projects that would be interresting to work with. Here are the topics we have discussed so far:
</P><UL><LI> Fatty acid synthesis in E.coli (EDA and DHA production)
</LI><LI> Using cells in water purification
</LI><LI> Using bacteria to remove hormones from wastewater
</LI><LI> Using bacteria to remove sucralose from wastewater
</LI></UL><LI> Be able to control pili production in order to control electron transfer between cells
</LI><LI> Biosensor detecting cancer cells, starting synthesis of chemoterapeutic drugs when cancer cells are detected
</LI><LI> Biosensor detecting harmful bacteria
</LI><LI> Making bacteria destroy oil spill
</LI><LI> Digitalize a signal from an environmental stimulus
</LI><LI> Biomining (using microorganisms to extract metals
</LI><H3><SPAN class="mw-headline" id="Friday_16.03.12">Friday 16.03.12</SPAN></H3><P>We had a meeting, where we started discussing ideas for this year's iGEM project. We also decided to eat lunch together once a week, to get to know one another.
For the outreach part of the project, Gunvor suggested that we could collaborate with Studentersamfundet to make a &quot;lørdagsmøte&quot; about synthetic biology.
</P><H3><SPAN class="mw-headline" id="Monday_27.02.12">Monday 27.02.12</SPAN></H3><P>We had our first meeting, and the team members met each other for the first time!
</P><H3><SPAN class="mw-headline" id="Friday_17.02.12">Friday 17.02.12</SPAN></H3><P>We recieved emails from Eivind letting us know that we are the ones that have been selected to represent NTNU in iGEM 2012. Everybody is happy!
</P><DIV class="printfooter">
Retrieved from &quot;<A href="http://2012.igem.org/Team:NTNU_Trondheim/Logg">http://2012.igem.org/Team:NTNU_Trondheim/Logg</A>&quot;</DIV></DIV></DIV><DIV id="footer-box" class="noprint"><DIV id="footer"><UL id="f-list"><LI id="t-recentchanges"><A href="/Special:RecentChanges" title="Recent changes">Recent changes</A></LI><LI id="t-whatlinkshere"><A href="/Special:WhatLinksHere/Team:NTNU_Trondheim/Logg" title="List of all wiki pages that link here [j]" accesskey="j">What links here</A></LI><LI id="t-recentchangeslinked"><A href="/Special:RecentChangesLinked/Team:NTNU_Trondheim/Logg" title="Recent changes in pages linked from this page [k]" accesskey="k">Related changes</A></LI><LI id="t-specialpages"><A href="/Special:SpecialPages" title="List of all special pages [q]" accesskey="q">Special pages</A></LI><LI><A href="/Special:Preferences">My preferences</A></LI></UL></DIV><DIV id="footer"><UL id="f-list"><LI id="t-print"><A href="/wiki/index.php?title=Team:NTNU_Trondheim/Logg&amp;printable=yes" title="Printable version of this page [p]" accesskey="p">Printable version</A></LI><LI id="t-permalink"><A href="/wiki/index.php?title=Team:NTNU_Trondheim/Logg&amp;oldid=26134" title="Permanent link to this revision of the page">Permanent link</A></LI><LI id="privacy"><A href="/2012.igem.org:Privacy_policy" title="2012.igem.org:Privacy policy">Privacy policy</A></LI><LI id="disclaimer"><A href="/2012.igem.org:General_disclaimer" title="2012.igem.org:General disclaimer">Disclaimers</A></LI></UL></DIV></DIV></DIV></BODY></HTML>