"
Page
Discussion
View source
History
teams
Log in
 
Team:NTNU Trondheim/Notebook/August
From 2012.igem.org
Search and Destroy
Search and Destroy
NTNU IS B.A.C.K.
Bacterial Anti-Cancer-Kamikaze
Home
Project
Project description
Modelling
Experiments and results
BioBrick Parts
Attributions
Technical stuff
Notebook
Protocols
Safety
Team
Meet the team
Official team profile
Outreach
Collaboration
Human practices
Press coverage
Achievements
Matchmaker
Notebook
August
February
Mon
Tue
Wed
Thu
Fri
Sat
Sun
1
2
3
4
5
6
7
8
9
10
11
12
13
14
15
16
17
18
19
20
21
22
23
24
25
25
27
28
29
March
Mon
Tue
Wed
Thu
Fri
Sat
Sun
1
2
3
4
5
6
7
8
9
10
11
12
13
14
15
16
17
18
19
20
21
22
23
24
25
26
27
28
29
30
April
Mon
Tue
Wed
Thu
Fri
Sat
Sun
1
2
3
4
5
6
7
8
9
10
11
12
13
14
15
16
17
18
19
20
21
22
23
24
25
26
27
28
29
30
May
Mon
Tue
Wed
Thu
Fri
Sat
Sun
1
2
3
4
5
6
7
8
9
10
11
12
13
14
15
16
17
18
19
20
21
22
23
24
25
26
27
28
29
30
31
June
Mon
Tue
Wed
Thu
Fri
Sat
Sun
1
2
3
4
5
6
7
8
9
10
11
12
13
14
15
16
17
18
19
20
21
22
23
24
25
26
27
28
29
30
31
July
Mon
Tue
Wed
Thu
Fri
Sat
Sun
1
2
3
4
5
6
7
8
9
10
11
12
13
14
15
16
17
18
19
20
21
22
23
24
25
26
27
28
29
30
31
August
Mon
Tue
Wed
Thu
Fri
Sat
Sun
1
2
3
4
5
6
7
8
9
10
11
12
13
14
15
16
17
18
19
20
21
22
23
24
25
26
27
28
29
30
31
September
Mon
Tue
Wed
Thu
Fri
Sat
Sun
1
2
3
4
5
6
7
8
9
10
11
12
13
14
15
16
17
18
19
20
21
22
23
24
25
26
27
28
29
30
Monday 27.08.12
Colicin expressed from the K+RBS+Colicin+DTT plasmid was tested on E.coli DH5α cells.
Also, Plld w/ RBS from E.coli was cut with S+P, Plld w/RBS from C.glutamicum was cut with E+P, and linearized plasmid backbone pSB1A3 was cut with E+P. The samples will be purified and ligated tomorrow.
Results from the miniprep:
Gene
Concentration [ng/µl]
PlldCGr+YFP+DTT in pSB1A3 #1
29,5
PlldCGr+YFP+DTT in pSB1A3 #2
23,7
PlldCGr+YFP+DTT in pSB1A3 #3
47,7
PlldCGr+YFP+DTT in pSB1A3 #4
25,2
PlldCGr+YFP+DTT in pSB13 #1
18,8
PlldCGr+YFP+DTT in pSB1C3 #2
21,0
PlldCGr in pSB1C3 2.1
24,8
PlldCGr in pSB1C3 2.2
20,7
PlldCGr in pSB1C3 3.1
21,3
PlldCGr in pSB1C3 3.2
18,4
Sunday 26.08.12
RNA was isolated from the RBS(<partinfo>BBa_B0030</partinfo>)+LacI(<partinfo>BBa_C0061</partinfo>)+DTT(<partinfo>BBa_B0014</partinfo>) and E.coli DH5α cells inoculated friday. RNA was isolated, and the DNAse reaction and the cDNA reaction were performed according to the protocol on the procedures page. The cDNA was frozen down to -80 °C.
Friday 24.08.12
Inoculated one cell culture of E.coli DH5α containing a plasmid consisting of RBS (<partinfo>BBa_B0030</partinfo>) + LacI (<partinfo>BBa_C0061</partinfo>) + DTT (<partinfo>BBa_B0014</partinfo>), and one cell culture consisting of E.coli DH5α cells without plasmid. These colonies will be used for real time PCR.
Thursday 23.08.12
BBa_K822001 in pSB1A3 was cut with Spe1 and Pst1, BBa_E0030 + terminator was cut with Xba1 and PSt1.
BBa_E0030 + terminator was run on gel, and got the consentration 4,5 ng/uL.
BBa_K822001 in pSB1A3 was purified using a PCR purifying kit, and an concentration of 1,2 ng/uL was obtained.
Wednesday 22.08.12
Plld in pSB1A3 and pSB1C3 were miniprepped as described in the protocol . The concentrations are listed below.
Gene
Concentration [ng/µl]
Plld in pSB1A3
25,7
Plld in pSB1C3
39,5
Tuesday 21.08.12
The primers ordered came today, so PCR was performed on plld with RBS (from Corynebacterium glutamicum), lldR from E. coli and C. glutamicum, and His-tagged lld from C. glutamicum.
The PCR mix and program are given below.
Substance
Volume [µl]
dH2O
32,5
5x-buffer
10,0
dNTPs
1,0
fwd primer
2,5
rev primer
2,5
template
1,0
Phusion polymerase
0,5
The addition of the Phusion polymerase is done immediately before starting the PCR program.
Step
Action
Temperature [°C]
Duration
1
Heated lid
103
2
Initial denaturation
98
30 sek
3
Denaturation
98
10 sek
4
Annealing
x1
30 sek
5
Elongation
72
x2
6
Go to step 3, repeat 10 x
7
Denaturation
98
10 sek
8
Elongation
72
x2
9
Go to step 7, repeat 15 x, with 5 s extra each time
10
Final Elongation
72
5 min
11
Hold
4
∞
Amplicon
x1 [°C]
x2 [s]
His tagged lld, C. glutamicum
62
60
Plld, C. glutamicum
63
5
lldR, E. coli
-
26
lldR, C. glutamicum
66
21
Monday 20.08.12
Plld in pSB1A3 was transformed into DH5α cells an plated out. Incubated at 37 °C. The cells inoculated to liquid medium on saturday were miniprepped. The concentrations are given below, together with the volume necessary for obtaining 1000 ng of plasmid
Biobrick
Concentration [ng/µl]
Volume required to obtain 1000 ng plasmid
Vgb+RBS+YFP+DTT
27.1
36.9
Vgb+RBS+LuxR+DTT
32.1
31.2
K+RBS+colicin
28.4
35.2
<partinfo>BBa_K292006</partinfo>
26.0
38.5
Colicin in pSB1C3
35.3
28.3
YFP+DTT
39.6
25.3
RBS*+LacI+DTT*
33.0
30.3
The required volume for obtaining 1000 ng of plasmid were then pipetted out for each sample, and dried on a heat block prewarmed to 50 °C, to obtain an approximate volume of 15 µl. The samples were then sent to sequencing.
Saturday 18.08.12
Plld cut with E+P and <partinfo>pSB1A3</partinfo> cut with E+P was ligated together to see if the Plld promoter will be more functional in an ampicillin plasmid. In order to be able to send plasmids to sequencing on monday, Vgb+RBS+YFP+DTT, Vgb+RBS+LuxR+DTT, K+RBS+colicin, <partinfo>BBa_K292006</partinfo>, colicin in pSB1C3, YFP+DTT and RBS*+LacI+DTT* was inoculated to liquid medium
Thursday 16.08.12
Plld+lysis and pSB1A3 cut yesterday with E+P was investigated on gel. Two fragments were obtained for Plld+lysis, but the sizes of the fragments were not as expected. Since we are not sure which fragment is which, both of the fragments were cut out of the gel and purified using the QIAquick gel extraction kit. In case one of them is uncut plasmid, the unligated DNA from both bonds will be transformed tomorrow, and we will assume that if the cells transformed with DNA from one of the samples won't grow, that sample will contain digested plasmid. This sample will be ligated with pSB1A3, plated out on a ampicillin plate, and then, several cell cultures from this plate will be plated out on a chloramphenicol plate. The colonies that won't grow on chloramphenicol, is assumed to contain the correct plasmid.
Vgb+RBS religated, Vgb+RBS+LuxR+DTT #1, Vgb+RBS+LuxR+DTT #2, LuxI+DTT, K+RBS+LacI+DTT and his-tagged LldR was miniprepped using the Promega SV miniprep kit. The concentrations of isolated plasmid are given below:
Plasmid
Concentration [ng/µl]
Vgb+RBS religated
27.3
Vgb+RBS+LuxR+DTT #1
26.2
Vgb+RBS+LuxR+DTT #2
31.7
LuxI+DTT
35.3
K+RBS+LacI+DTT
7.9
his-LldR
51.0
Also, Vgb and Vhb was tested again, and Ove named two of the agar plates Mark and Doggie:-)
Wednesday 15.08.12
The high expression plasmid containing his-tagged LldR was transferred to liquid medium.
K+RBS+Colicin and the negative colicin control, LuxR+DTT was transferred to new liquid medium in order to be tested. 50 µl of the old cultures was transferred to 10 ml of LB containing ampicillin.
Since we have read that chloramphenicol can be a problem for the lld promoter, we also decided to transfer this brick to a new backbone before further testing. Both Plld and Plld+RBS+lysis was cut with EcoRI and PstI. Linearized plasmid backbone pSB1A3 was also cut with EcoRI and PstI. Ligations will be performed tomorrow.
Tuesday 14.08.12
To check if the new restriction digest were ok, LuxI (<partinfo>Bba_C0061</partinfo>) and LuxR+term (<partinfo>BBa_C0062</partinfo>, <partinfo>BBa_B0015</partinfo>) were tested with gel electrophoresis.
Picture of gel electrophoresis. LuxI and LuxR+Term cut with NotI, and LuxI and LuxR+term cut with E+S and X+P, respectively.
Cut of the parts consisting the genes, LuxI (<partinfo>Bba_C0061</partinfo>) and LuxR+term (<partinfo>BBa_C0062</partinfo>, <partinfo>BBa_B0015</partinfo>), and extracted the DNA from the gel. Concentrations are listed below.
BioBrick
Enzymes
Concentration [ng/µl]
LuxI
EcoRI+SpeI
4.2
LuxR+term
XbaI+PstI
5.6
Ligation mixes were made according to protocol with the following ingredients, as well as religation of the backbones:
Backbone
Insert
Plasmid
K <partinfo>BBa_J23119</partinfo>
RBS+LacI+term <partinfo>BBa_B0030</partinfo>, <partinfo>Bba_C0012</partinfo>, <partinfo>BBa_B0015</partinfo>
<partinfo>psB1A2</partinfo>
Vgb+RBS <partinfo>BBa_K561001</partinfo>, <partinfo>BBa_B0034</partinfo>
LuxR+term <partinfo>BBa_C0062</partinfo>, <partinfo>BBa_B0015</partinfo>
<partinfo>psB1A2</partinfo>
term <partinfo>BBa_B0015</partinfo>
LuxI <partinfo>BBa_C0061</partinfo>
<partinfo>pSB1AK3</partinfo>
These were then transformed in competent DH5ɑ cells, as described in the protocol . The petri dishes (all with Ampicillin) were left in the incubator over night.
10 µl of the liquid cultures for K+RBS+colicin and LuxR+DTT was transferred to 5 ml of new liquid medium to be used in testing of the colicin biobrick. The rest of the samples were miniprepped using the Promega SV Miniprep kit. The concentrations of the samples are given below:
Biobrick
Concentration [ng/µl]
K+RBS+colicin
31.1
LuxR+DTT
31.3
We also recieved the high expression plasmid for his-tagged LldR today. This was transformed to competent E.coli K-12 ER2566 cells, as E.coli Dh5α, which are the cells we normally use in transformation, are unsuited when working with expression of proteins.
Monday 13.08.12
Transferred K+RBS+Colisin (<partinfo>BBa_K081005</partinfo> and <partinfo>BBa_K150009</partinfo>), PLuxR+HSL+RBS+Lysis (<partinfo>BBa_R0062</partinfo>, <partinfo>BBa_B0034</partinfo> and <partinfo>BBa_K112808</partinfo>) to liquid media.
Did a restriction digest on the following biobricks, as described in the protocol .
Biobricks
BioBrick no.
Enzymes
Vgb+RBS
<partinfo>BBa_K561001</partinfo>, <partinfo>BBa_B0034</partinfo>
SpeI+PstI, NotI
LuxR+term
<partinfo>BBa_C0062</partinfo>, <partinfo>BBa_B0015</partinfo>
XbaI+PstI, NotI
RBS+LacI+term
<partinfo>BBa_B0030</partinfo>, <partinfo>Bba_C0012</partinfo>, <partinfo>BBa_B0015</partinfo>
XbaI+PstI, NotI
LuxI
<partinfo>Bba_C0061</partinfo>
EcoRI+SpeI, NotI
K
<partinfo>BBa_J23119</partinfo>
SpeI+PstI
Did a gel electrophoresis on the mixes above (except for the konstitutive promoter <partinfo>BBa_J23119</partinfo>), but only two of them where as expected. Did a new restriction digest on the two that didn't give the correct bands: LuxI (<partinfo>Bba_C0061</partinfo>) and LuxR+term (<partinfo>BBa_C0062</partinfo> and <partinfo>BBa_B0015</partinfo>).
Picture of gel electrophoresis with test cutting with NotI (to the left) and regular cutting (to the right) of LuxI (EcoRI+SpeI), LuxR+term (X+P), RBS+lacI+term (XbaI+PstI) and Vgb+RBS (SpeI+PstI), respectively.
Extracted DNA from gel on the part RBS+LacI+term (<partinfo>BBa_B0030</partinfo>, <partinfo>Bba_C0012</partinfo> and <partinfo>BBa_B0015</partinfo>) and did a PCR purification on Vgb+RBS (<partinfo>BBa_K561001</partinfo> and <partinfo>BBa_B0034</partinfo>) and the konstitutive promoter (K <partinfo>BBa_J23119</partinfo>).
The concentrations on the purified parts are as follows:
Biobricks
BioBrick no.
Concentration [ng/µl]
K
<partinfo>BBa_J23119</partinfo>
3.0
Vgb+RBS
<partinfo>BBa_K561001</partinfo>, <partinfo>BBa_B0034</partinfo>
3.6
RBS+LacI+term
<partinfo>BBa_B0030</partinfo>, <partinfo>Bba_C0012</partinfo>, <partinfo>BBa_B0015</partinfo>
4.7
Friday 10.08.12
Performed miniprep on the following constructs:
VGB+RBS+YFP+term (<partinfo>BBa_K561001</partinfo>, <partinfo>BBa_B0034</partinfo>, <partinfo>BBa_E0030</partinfo> and <partinfo>BBa_B0015</partinfo>)
VHB+RBS+YFP+term (<partinfo>BBa_K258005</partinfo>, <partinfo>BBa_B0034</partinfo>, <partinfo>BBa_E0030</partinfo> and <partinfo>BBa_B0015</partinfo>)
LuxI+term (<partinfo>Bba_C0061</partinfo>, <partinfo>BBa_B0015</partinfo>)
The concentrations were as follows:
Biobrick
Concentration ng/uL
VGB+RBS+YFP+term
35,2
VHB+RBS+YFP+term
38,9
LuxI+term
48,2
Transformed K+RBS+colicin.
Test constructs for Vgb and Vhb (Vgb+RBS+YFP+DTT and Vhb+RBS+YFP+DTT) were transferred to liquid medium for testing on monday.
It was attempted to make LldR by PCR one more time. The following PCR mix was used:
32.5 µl dH2O
10 µl 5x Phusion DNA polymerase buffer
1 µl dNTPs
2.5 µl fwd primer previously diluted 1:10
2.5 µl rev primer previously diluted 1:10
1 µl template DNA (E.coli K-12 genome)
0.5 µl Phusion DNA polymerase
The following primers were used:
Primer
Sequence
LldR fwd
atgattgttttacccagacgcctgt
LldR rev
tcatgcgtttttctccctcgaat
The PCR machine was programmed according to the following table:
Step
Action
Temperature
Duration
1
Heated lid:
103°C
2
Initial denaturation;
98°C
30 s
3
Denaturation;
98°C
10 s
4
Annealing;
45°C
10 s
5
Elongation;
72°C
24 s
6
Go to step 3, repeat 25 x
7
Final elongation;
72°C
5 min
8
Hold
4°C
∞
Thursday 09.08.12
Performed miniprep on RBS (<partinfo>BBa_B0030</partinfo>) and plld+RBS. The concentration were as follows:
Biobrick
Concentration ng/uL
RBS
41,3
plld+RBS
43,7
Did a restriction digest as described in the protocol on the following parts. On the pieces to be used as inserts, the number of base pairs are also listed.
Biobrick/Construct
BioBrick
Enzymes
Buffer
bp insert
LuxI+term
<partinfo>Bba_C0061</partinfo>, <partinfo>BBa_B0015</partinfo>
XbaI+PstI
3
747
YFP+term
<partinfo>BBa_E0030</partinfo>, <partinfo>BBa_B0015</partinfo>
XbaI+PstI
3
852
Lysis
<partinfo>BBa_K112808</partinfo>
XbaI+PstI
3
1785
LuxI
<partinfo>Bba_C0061</partinfo>
XbaI+PstI
3
618
RBS*
<partinfo>BBa_B0030</partinfo>
SpeI+PstI
1
-
PluxR+HSL
<partinfo>BBa_R0062</partinfo>
SpeI+PstI
1
-
RBS
<partinfo>BBa_B0034</partinfo>
SpeI+PstI
1
-
pSB1A3
<partinfo>pSB1A3</partinfo>
EcoRI+PstI+DpnI
3
-
plld
-
EcoRI+SpeI
4
344
The parts lysis <partinfo>BBa_K112808</partinfo>, YFP+term (<partinfo>BBa_E0030</partinfo>, <partinfo>BBa_B0015</partinfo>), <partinfo>pSB1A3</partinfo> and plld will be assembled using the 3A assembly, so made double cutting mix with these parts.
Gel electrophoresis was done with luxI+term (<partinfo>Bba_C0061</partinfo>, <partinfo>BBa_B0015</partinfo>), lysis <partinfo>BBa_K112808</partinfo>, YFP+term (<partinfo>BBa_E0030</partinfo>, <partinfo>BBa_B0015</partinfo>), LuxI <partinfo>Bba_C0061</partinfo> and plld. The gel did not show either LuxI or LuxI+term, but cut out lysis, YFP+term and plld. Will do a gel extraction on those, and a PCR purification on <partinfo>pSB1A3</partinfo>, RBS <partinfo>BBa_B0034</partinfo>, RBS* <partinfo>BBa_B0030</partinfo> and PLuxR+HSL <partinfo>BBa_R0062</partinfo>. This will be done as described in protocols .
Ligated together the following, as described in protocols :
Ligation #
Backbone
Insert
1
RBS* <partinfo>BBa_B0030</partinfo>
LacI+term <partinfo>Bba_C0012</partinfo>, <partinfo>BBa_B0015</partinfo>
2
RBS <partinfo>BBa_B0034</partinfo>
LuxR+term <partinfo>BBa_C0062</partinfo>, <partinfo>BBa_B0015</partinfo>
3
pLuxR+HSL<partinfo>BBa_R0062</partinfo>
Lysis <partinfo>BBa_K112808</partinfo>
3A assembly was used on the following parts, as described in the Protocol , by Northwestern University:
Ligation #
Backbone
Insert 1
Insert 2
1
<partinfo>pSB1A3</partinfo>
plld
YFP+term<partinfo>BBa_E0030</partinfo>, <partinfo>BBa_B0015</partinfo>
2
<partinfo>pSB1A3</partinfo>
plld
Lysis <partinfo>BBa_K112808</partinfo>
Since the restriction digest didn't give anything for LuxI <partinfo>Bba_C0061</partinfo> and LuxI+term (<partinfo>Bba_C0061</partinfo>, <partinfo>BBa_B0015</partinfo>), will a colony from LuxI+term (<partinfo>Bba_C0061</partinfo>, <partinfo>BBa_B0015</partinfo>) be transferred to liquid media and LuxI <partinfo>Bba_C0061</partinfo> will be transformed again.
The following was also transformed: RBS*+LacI+term (<partinfo>BBa_B0030</partinfo>, <partinfo>Bba_C0012</partinfo> and0 <partinfo>BBa_B0015</partinfo>), RBS+LuxR+term (<partinfo>BBa_B0034</partinfo>, <partinfo>BBa_C0062</partinfo> and <partinfo>BBa_B0015</partinfo>), pLuxR+HSL+Lysis (<partinfo>BBa_R0062</partinfo> and <partinfo>BBa_K112808</partinfo>), <partinfo>pSB1A3</partinfo>+plld+YFP+term (<partinfo>BBa_E0030</partinfo> and <partinfo>BBa_B0015</partinfo>) and <partinfo>pSB1A3</partinfo>+plld+Lysis (<partinfo>BBa_K112808</partinfo>). Did religations of the backbones for all of the above. Transferred 200 µl to petri dishes and inoculated in 37C over night.
Wednesday 08.08.12
Tranformed Vgb+RBS+YFP+term (<partinfo>BBa_K561001</partinfo>, <partinfo>BBa_B0034</partinfo>, <partinfo>BBa_E0030</partinfo> and <partinfo>BBa_B0015</partinfo>) and Vhb+RBS+YFP+term (<partinfo>BBa_K258005</partinfo>, <partinfo>BBa_B0034</partinfo>, <partinfo>BBa_E0030</partinfo> and <partinfo>BBa_B0015</partinfo>) in competent DH5ɑ cells as described in the protocol . The plasmid in both of the constructs is <partinfo>psB1A2</partinfo>. They were left in the incubator over night.
Inspected the petri dishes with plld+RBS (lactate induced promoter and <partinfo>BBa_B0030</partinfo>) and the religation of RBS <partinfo>BBa_B0030</partinfo>, had many more colonies on the plld+RBS than the religation, which is good. Transferred a colony of the plld+RBS (lactate promoter and <partinfo>BBa_B0030</partinfo>) and a colony of RBS <partinfo>BBa_B0030</partinfo> to liquid media, with ampicillin resistance.
LacI+term (<partinfo>BBa_C0012</partinfo>) + (<partinfo>BBa_B0014</partinfo>)was miniprepped and concentration was 216,1 ng/uL.
Cut LacI+term (<partinfo>BBa_C0012</partinfo>) and (<partinfo>BBa_B0014</partinfo>), LuxI+term (<partinfo>Bba_C0061</partinfo> and <partinfo>BBa_B0015</partinfo>) and LuxR+term (<partinfo>BBa_C0062</partinfo> and <partinfo>BBa_B0015</partinfo>) with XbaI and PstI, so that they can be used as inserts. RBS(<partinfo>BBa_B0034</partinfo>) was cut with Spe1 and Pst1 and will be used further as a backbone.Used the protocol with NEBuffer 2 and 3.
Did a gel electrophoresis with LuxI+term (<partinfo>Bba_C0061</partinfo> and <partinfo>BBa_B0015</partinfo>) and LuxR+term (<partinfo>BBa_C0062</partinfo> and <partinfo>BBa_B0015</partinfo>).
Purified RBS and LuxR+term. Resulting concentrations were 4.8 and 11.2 ng/µl, respectively. LacI+ term was also purified from gel using the gel extracion kit for Qiagen, and the concentration was 1,5 ng/uL.
Tuesday 07.08.12
Inspected the petri dishes from yesterday, LacI <partinfo>BBa_C0012</partinfo> + terminator <partinfo>BBa_B0015</partinfo> showed colonies, the religation of the terminator <partinfo>BBa_B0015</partinfo> did not. Transferred a LacI <partinfo>BBa_C0012</partinfo> + terminator <partinfo>BBa_B0015</partinfo> colony to LB with ampicillin and inoculated at 37C with shaking.
Did a gel electrophoresis on plld, cut it out and extracted it, according to protocols . The concentration after gel extraction was: cplld = 10,6 ng/µl.
Picture of gel electrophoresis with plld. It was found at approximately 340 bp, as expected (344 bp).
Ligation was performed with plld as insert and RBS <partinfo>BBa_B0034</partinfo> as backbone. That means that the construct has a <partinfo>psB1A2</partinfo> plasmid. Also did a religation of the RBS <partinfo>BBa_B0034</partinfo> without any insert.
Transformed plld and RBS <partinfo>BBa_B0034</partinfo> in competent DH5ɑ cells, as well as the religation of RBS <partinfo>BBa_B0034</partinfo>. They were transferred to petri dishes with ampicillin resistance and left in the incubator over night.
Miniprepped VHB+RBS+lysis (<partinfo>BBa_K258005</partinfo>, <partinfo>BBa_B0034</partinfo> and <partinfo>BBa_K112808</partinfo>), VGB+RBS+lysis (<partinfo>BBa_K561001</partinfo>, <partinfo>BBa_B0034</partinfo> and <partinfo>BBa_K112808</partinfo>), pllD+lysis (lactate promoter and <partinfo>BBa_K112808</partinfo>) and three parallels of pBad+YFP (<partinfo>Bba_K206000</partinfo> and <partinfo>BBa_E0030</partinfo>)
Sample
Concentration [ng/µl]
VHB+RBS+lysis
38,2
VGB+RBS+lysis
50,5
pllD+lysis
40,2
pBad+YFP #1
46,6
pBad+YFP #2
40,5
pBad+YFP #3
38,9
Monday 06.08.12
In order to start making the construct for the biobrick we are going to improve, the parts needed for the construct were cut using the restriction digest protocol [1] .
RBS <partinfo>BBa_B0030</partinfo> will be used as a backbone and was cut with EcoRI and XbaI. LacI <partinfo>BBa_C0012</partinfo> will be an insert, and was cut with EcoRI and SpeI. The terminator <partinfo>BBa_B0014</partinfo> will be used as a backbone and was cut with EcoRI and XbaI. In addition we will test the construct using a constitutive promoter <partinfo>BBa_J23119</partinfo> which was cut using EcoRI and SpeI.
The inserts LacI <partinfo>BBa_C0012</partinfo> and constitutive promoter <partinfo>BBa_J23119</partinfo> were run on gel after the restriction digest.  The problem with the constitutive promoter is that it is only 35 b.p long, and is therefore difficult to use as an insert. We tried to stop the gel early to see if we could detect the constitutive promoter, but we could not see anything at all. We therefore ran the gel further and cut the LacI <partinfo>BBa_C0012</partinfo> out of the gel.
Gel picture of the two inserts ran on gel. Used two different ladders. To the left we used a 1kb ladder to identify the Lac I insert, and to the right we used a 50 kb ladder to identify the constitutive promoter. Expected the LacI band to be approximately 1100 bp which seems to be about right. The constitutive promoter was expected to be 35 bp, but was not detected on the gel at all.
Used the PCR purification kit and protocol from www.qiagen.com [2] , on RBS <partinfo>BBa_B0030</partinfo>, pBad <partinfo>Bba_K206000</partinfo> and terminator <partinfo>BBa_B0014</partinfo>. Did a gel extraction using the Gel Extraction kit and Protocol on LacI <partinfo>BBa_C0012</partinfo>.
Did a ligation with LacI <partinfo>BBa_C0012</partinfo> and double terminator <partinfo>BBa_B0015</partinfo>, where the double terminator <partinfo>BBa_B0015</partinfo> was used as a backbone. That means, the construct has a <partinfo>psB1AK3</partinfo> backbone and resistance Ampicillin and Kanamycin. A religation of the backbone <partinfo>BBa_B0015</partinfo> was also performed.
The ligated LacI <partinfo>BBa_C0012</partinfo> and double terminator <partinfo>BBa_B0015</partinfo> was transformed in competent DH5ɑ cells, along with the religation of the backbone <partinfo>BBa_B0015</partinfo>. The petri dishes are left to inoculate through the night.
A new batch of LA-medium was made and used to make ampicillin petri dishes.
A restriction digest was done on the lld promoter, as described in the protocols . It was cut with EcoRI and SpeI, and will be an insert with RBS <partinfo>BBa_B0030</partinfo> as backbone. This will then make the plasmid <partinfo>pSB1A2</partinfo> the backbone.
Sunday 05.08.12
Plasmid DNA was isolated from pllD-1B and <partinfo>BBa_B0014</partinfo> cultures were isolated by miniprep. DNA concentrations were measured as 41,2/42,5 and 53,3/52,8 ng/uL respectively (two measurements per sample).
Inspected agar plates in incubator inoculated on 3/8: VGB+RBS religation: 1 colony VHB+RBS+lysis: Good growth VHB+RBS religation: ~ 20 colonies VHB+RBS + lysis: Good growth pBAD (w/ backbone), religated: 40+ colonies pBAD +YFP +DTT: 7 colonies
Comments: Judging from the growth on the religation plates, colonies with non-insert plasmids is a possibility on all plates. pBAD is especially worrisome, as the number of colonies is lower on the insert ligation plate than on the backbone religation plate. The lysis part used in the above constructs already has an RBS, so the RBS part is redundant.
Induced one 5 mL culture (induction culture) pllD + lysis (3/8) with 210 uL ~ 1M lactate and addded 210 uL dH2O to another (control culture). Sampled 200 uL from both before induction, for OD measurements.
Inoculated 5 5 mL LB + Amp cultures with colonies from the following agar plates: VGB+RBS+Lysis (3/8) VHB+RBS+Lysis (3/8) pBAD + YFP +DTT (3/8) (3 colonies)
Inoculated 3 5 mL LB + Cm cultures from pllD + lysis agar plate.
Friday 03.08.12
Isolated plasmid DNA from the cultures of RBS* (<partinfo>BBa_B0030</partinfo>) and LacI (<partinfo>BBa_C0012</partinfo>) inoculated yesterday.
Ligated pBad (<partinfo>Bba_K206000</partinfo>) together with YFP+DTT (<partinfo>BBa_E0030</partinfo> and <partinfo>BBa_B0015</partinfo>), and VGB+RBS (<partinfo>BBa_K561001</partinfo> and <partinfo>BBa_B0034</partinfo>) and VHB+RBS (<partinfo>BBa_K258005</partinfo> and <partinfo>BBa_B0034</partinfo>) with Lysis (<partinfo>BBa_K124017</partinfo>). These were then transformed into competent E. coli DH5α cells and plated out on petri dishes with Ampicillin.
Terminator <partinfo>BBa_B0014</partinfo> and re-transformed pllD in <partinfo>pSB1C3</partinfo> ("pllD 1-B") were transferred to liquid culture.
Ran gel on the PCR-products from yesterday.
Inoculated 2 x 5 mL, 1 x 10 mL and 1 x 25 mL LB + Cm with plld + lysis.
Thursday 02.08.12
5 µl of the PCR product from yesterday was run on gel, but it did not give any results. We looked at the primers and thought we had done something wrong, but we did not. Will try to do the PCR one more time, but with a lower annealing temperature (55°C).
The constructs containing plld ligated with the lysis device(<partinfo>BBa_K112808</partinfo>), and plld in plasmid <partinfo>PSB1C3</partinfo> were miniprepped. The concentration were measured using nano drop.
Sample
Concentration [ng/µl]
pllD + lysis
45,5
pllD in PSB1C3
42,3
Inoculated 1 colony from each of three plates containing LacI (<partinfo>BBa_C0012</partinfo>), RBS* <partinfo>BBa_B0030</partinfo> and pllD + lysis (all inoculated 1/8), into 5 mL LB + Amp (BBa_B0030, LacI)/ LB + Cm (pllD + lysis).
Transformed the part <partinfo>BBa_B0014</partinfo> from the distribution kit, and re-transformed pllD in pSB1C3 with DNA taken from the sample shipped to the RHIT team yesterday.
Reviewed the data from the oxygen-promoter experiment on monday. In conclusion, it appears that oxygen levels in the induced (anaerobic) cultures were not as low as desired, as growth rates were very similar between the aerobic and anaerobic cultures, while one would expect anaerobic cultures to grow slower. After correcting for OD, the fluorescence in the cultures at the end of the experiment was lower than the fluorescence measured from a control culture with no expected production of fluorescent protein. For this reason, a full analysis of the rest of the data was not performed.
Wednesday 01.08.12
Gel purification was performed on the gel piece from yesterday, the lysis device cut with X+P. The concentration is 10,1 ng/µl. The lysis device and plld were ligated together, plld (with <partinfo>pSB1C3</partinfo>) as backbone.
The ligated lysis and plld were transformed into competent E.coli DH5α cells. This was also done for the biobricks lacI repressor from E. coli (<partinfo>BBa_C0012</partinfo>) and RBS (<partinfo>BBa_B0030</partinfo>).
The petri dishes from yesterday showed colonies on the dishes with the pSB1C3+Colisin construct (<partinfo>pSB1C3</partinfo> and <partinfo>BBa_K150009</partinfo>) and the K+RBS+Colisin construct (<partinfo>BBa_K081005</partinfo> and <partinfo>BBa_K150009</partinfo>). One colony from each petri dish were transferred to liquid medium.
Sent a sample of the E. coli pllD promoter in <partinfo>pSB1C3</partinfo> backbone to the iGEM team at RHIT.
PCR was performed on pBad (<partinfo>Bba_K206000</partinfo>) and DTT (<partinfo>BBa_B0015</partinfo>), the primers used, PCR-mix and programmed times are listed below.
Primer
Type
Sequence
Tm [°C]
DTT fwd
Forward
CCAGGCATCAAATAAAACGAAAGGCTCAGTCGAAAG
72
DTT rev
Reverse
CTCTAGAAGCGGCCGCGAATTCCAGAAATC
72
pBad scar fwd
Forward
TACTAGAGTACTAGTAGCGGCCGCTGCAGT
72
pBad fwd
Forward
TACTAGTAGCGGCCGCTGCAGTC
70
pBad rev
Reverse
GCTAGCCCAAAAAAACGGTATGGAGAAACAGTAGAGAG
72
Mix 1:
1 µl dNTP
2,5 µl 1:10 fwd primer
2,5 µl 1:10 rev primer
1 µl template DNA
18 µl dH2O
Mix 2:
19,25 µl d2O
5 µl buffer with MgCl2
0,75 µl Expand High Fidelity enzyme
This PCR mix was found here: http://francois.schweisguth.free.fr/protocols/High_fildelity_roche.pdf
Thermal timetables:
pBAD
Step
Action
Temperature
Duration
1
Heated lid:
103°C
2
Initial denaturation;
94°C
2 min
3
Denaturation;
94°C
15 s
4
Annealing;
66°C
30 s
5
Elongation;
72°C
2 min
6
Go to step 3, repeat 10 x
7
Denaturation;
98°C
10 s
8
Elongation;
72°C
31 s
9
Go to step 7, repeat 15 x, with 5 s extra each time.
10
Final Elongation;
72°C
7 min
11
Hold
4°C
∞
DTT
Step
Action
Temperature
Duration
1
Heated lid:
103°C
2
Initial denaturation;
94°C
2 min
3
Denaturation;
94°C
15 s
4
Annealing;
66°C
30 s
5
Elongation;
72°C
3 min
6
Go to step 3, repeat 10 x
7
Denaturation;
98°C
10 s
8
Elongation;
72°C
31 s
9
Go to step 7, repeat 15 x, with 5 s extra each time.
10
Final Elongation;
72°C
7 min
11
Hold
4°C
∞
Designed and built with MediaWiki , jQuery and Twitter Bootstrap .
Icons by Font Awesome .
igem.ntnu@gmail.com
Retrieved from " http://2012.igem.org/Team:NTNU_Trondheim/Notebook/August "
Recent changes
What links here
Related changes
Special pages
My preferences
Printable version
Permanent link
Privacy policy
Disclaimers
