<HTML xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en" dir="ltr">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki  ltr ns-0 ns-subject page-Team_Ciencias-UNAM_Project"><DIV id="globalWrapper"><DIV id="top-section"><DIV id="p-logo"><A href="/Main_Page" title="Main Page">&quot;
	    </A></DIV><DIV id="menubar" class="left-menu noprint"><UL><LI class="selected"><A href="/Team:Ciencias-UNAM/Project">Page               </A></LI><LI class="new"><A href="/wiki/index.php?title=Talk:Team:Ciencias-UNAM/Project&amp;action=edit&amp;redlink=1">Discussion               </A></LI><LI><A href="/wiki/index.php?title=Team:Ciencias-UNAM/Project&amp;action=edit">View source               </A></LI><LI><A href="/wiki/index.php?title=Team:Ciencias-UNAM/Project&amp;action=history">History               </A></LI><LI style="color:white;cursor:default">teams</LI></UL></DIV><DIV class="right-menu noprint" id="menubar"><UL><LI id="pt-login"><A href="/wiki/index.php?title=Special:UserLogin&amp;returnto=Team:Ciencias-UNAM/Project" title="You are encouraged to log in; however, it is not mandatory [o]" accesskey="o">Log in</A></LI></UL></DIV><DIV id="search-controls" class="noprint"><FORM action="/Special:Search" id="searchform"> </FORM></DIV></DIV><DIV id="content"><H1 class="firstHeading">Team:Ciencias-UNAM/Project</H1><DIV id="bodyContent"><H3 id="siteSub" class="noprint">From 2013.igem.org</H3><P><TITLE>Project</TITLE></P><DIV id="main_container"><DIV id="u2container" style="position:absolute; left:160px; top:130px; width:940px; height:60px;; overflow:visible"><DIV id="u3container" style="position:absolute; left:2px; top:2px; width:941px; height:61px; overflow:visible"><DIV id="u4" class="u4_container"><DIV id="u5" class="u5"><DIV id="u5_rtf"><P style="text-align:center;"><SPAN style="font-family:Museo Slab;font-size:16px;font-weight:normal;font-style:normal;text-decoration:none;color:#4B4F54;">HOME</SPAN></P></DIV></DIV></DIV><DIV id="u6" class="u6_container"><DIV id="u7" class="u7"><DIV id="u7_rtf"><P style="text-align:center;"><SPAN style="font-family:Museo Slab;font-size:16px;font-weight:normal;font-style:normal;text-decoration:none;color:#4B4F54;">PROJECT</SPAN></P></DIV></DIV></DIV><DIV id="u8" class="u8_container"><DIV id="u9" class="u9"><DIV id="u9_rtf"><P style="text-align:center;"><SPAN style="font-family:Museo Slab;font-size:16px;font-weight:normal;font-style:normal;text-decoration:none;color:#4B4F54;">HUMAN PRACTICES </SPAN></P></DIV></DIV></DIV><DIV id="u10" class="u10_container"><DIV id="u11" class="u11"><DIV id="u11_rtf"><P style="text-align:center;"><SPAN style="font-family:Museo Slab;font-size:16px;font-weight:normal;font-style:normal;text-decoration:none;color:#4B4F54;">TEAM</SPAN></P></DIV></DIV></DIV><DIV id="u12" class="u12_container"><DIV id="u13" class="u13"><DIV id="u13_rtf"><P style="text-align:center;"><SPAN style="font-family:Museo Slab;font-size:16px;font-weight:normal;font-style:normal;text-decoration:none;color:#4B4F54;">SAFETY</SPAN></P></DIV></DIV></DIV></DIV></DIV><DIV id="sm0u2" style="position:absolute; left:160px; top:130px; width:940px; height:60px; visibility:hidden; width:1px; height:1px;"><DIV id="u14container" style="position:absolute; left:152px; top:58px; width:150px; height:184px;; overflow:visible"><DIV id="u15container" style="position:absolute; left:2px; top:2px; width:151px; height:185px; overflow:visible"><DIV id="u16" class="u16_container"><DIV id="u17" class="u17"><DIV id="u17_rtf"><P style="text-align:center;"><SPAN style="font-family:Museo Slab;font-size:16px;font-weight:normal;font-style:normal;text-decoration:none;color:#4B4F54;">Overview</SPAN></P></DIV></DIV></DIV><DIV id="u18" class="u18_container"><DIV id="u19" class="u19"><DIV id="u19_rtf"><P style="text-align:center;"><SPAN style="font-family:Museo Slab;font-size:16px;font-weight:normal;font-style:normal;text-decoration:none;color:#4B4F54;">General Design</SPAN></P></DIV></DIV></DIV><DIV id="u20" class="u20_container"><DIV id="u21" class="u21"><DIV id="u21_rtf"><P style="text-align:center;"><SPAN style="font-family:Museo Slab;font-size:16px;font-weight:normal;font-style:normal;text-decoration:none;color:#4B4F54;">Wet Lab</SPAN></P></DIV></DIV></DIV><DIV id="u22" class="u22_container"><DIV id="u23" class="u23"><DIV id="u23_rtf"><P style="text-align:center;"><SPAN style="font-family:Museo Slab;font-size:16px;font-weight:normal;font-style:normal;text-decoration:none;color:#4B4F54;">Dry Lab</SPAN></P></DIV></DIV></DIV><DIV id="u24" class="u24_container"><DIV id="u25" class="u25"><DIV id="u25_rtf"><P style="text-align:center;"><SPAN style="font-family:Museo Slab;font-size:16px;font-weight:normal;font-style:normal;text-decoration:none;color:#4B4F54;">Conclusions</SPAN></P></DIV></DIV></DIV><DIV id="u26" class="u26_container"><DIV id="u27" class="u27"><DIV id="u27_rtf"><P style="text-align:center;"><SPAN style="font-family:Museo Slab;font-size:16px;font-weight:normal;font-style:normal;text-decoration:none;color:#4B4F54;">References</SPAN></P></DIV></DIV></DIV></DIV></DIV></DIV><DIV id="sm1u2" style="position:absolute; left:160px; top:130px; width:940px; height:60px; visibility:hidden; width:1px; height:1px;"><DIV id="u28container" style="position:absolute; left:552px; top:58px; width:151px; height:64px;; overflow:visible"><DIV id="u29container" style="position:absolute; left:2px; top:2px; width:152px; height:65px; overflow:visible"><DIV id="u30" class="u30_container"><DIV id="u31" class="u31"><DIV id="u31_rtf"><P style="text-align:center;"><SPAN style="font-family:Museo Slab;font-size:16px;font-weight:normal;font-style:normal;text-decoration:none;color:#4B4F54;">Bios</SPAN></P></DIV></DIV></DIV><DIV id="u32" class="u32_container"><DIV id="u33" class="u33"><DIV id="u33_rtf"><P style="text-align:center;"><SPAN style="font-family:Museo Slab;font-size:16px;font-weight:normal;font-style:normal;text-decoration:none;color:#4B4F54;">Sponsors</SPAN></P></DIV></DIV></DIV></DIV></DIV></DIV><DIV id="sm2u2" style="position:absolute; left:160px; top:130px; width:940px; height:60px; visibility:hidden; width:1px; height:1px;"><DIV id="u34container" style="position:absolute; left:302px; top:58px; width:178px; height:94px;; overflow:visible"><DIV id="u35container" style="position:absolute; left:2px; top:2px; width:179px; height:95px; overflow:visible"><DIV id="u36" class="u36_container"><DIV id="u37" class="u37"><DIV id="u37_rtf"><P style="text-align:center;"><SPAN style="font-family:Museo Slab;font-size:16px;font-weight:normal;font-style:normal;text-decoration:none;color:#4B4F54;">Overview</SPAN></P></DIV></DIV></DIV><DIV id="u38" class="u38_container"><DIV id="u39" class="u39"><DIV id="u39_rtf"><P style="text-align:center;"><SPAN style="font-family:Museo Slab;font-size:16px;font-weight:normal;font-style:normal;text-decoration:none;color:#4B4F54;">Journalism</SPAN></P></DIV></DIV></DIV><DIV id="u40" class="u40_container"><DIV id="u41" class="u41"><DIV id="u41_rtf"><P style="text-align:center;"><SPAN style="font-family:Museo Slab;font-size:16px;font-weight:normal;font-style:normal;text-decoration:none;color:#4B4F54;">Interviews</SPAN></P></DIV></DIV></DIV></DIV></DIV></DIV><DIV id="u46" class="u46"><DIV id="u46_rtf"><P style="text-align:left;"><SPAN style="font-family:Museo Slab;font-size:14px;font-weight:normal;font-style:normal;text-decoration:none;color:#666666;">Home &gt; Project &gt; Overview</SPAN></P></DIV></DIV><DIV id="u50" class="u50"><DIV id="u50_rtf"><SPAN style="font-family:Museo Slab;font-size:16px;font-weight:normal;font-style:normal;text-decoration:none;color:#666666;">
ABSTRACT
</SPAN><SPAN style="font-family:Arial;font-size:14px;font-weight:normal;font-style:normal;text-decoration:none;color:#666666;text-decoration:none;"> </SPAN><P style="text-align:justify;"><SPAN style="font-family:Arial;font-size:14px;font-weight:normal;font-style:normal;text-decoration:none;color:#666666;">
The human peptide LL-37 is a cationic peptide with antimicrobial activity against both Gram-positive and Gram-negative microorganisms. It has been shown to protect against gastritis caused by <I>Helicobacter pylori</I> infection. Most of the current synthetic expression systems for LL-37 depend on the construction of soluble fusion partners to avoid cytotoxic effects of the antimicrobial peptide in the <I>E. coli</I> host strain. However, the fusion systems require additional cleavage steps using enzymatic or chemical methods, which makes them impossible to express an active LL-37 peptide <I>in vivo</I>. In order to create a resistant host that can export LL-37 to the media, we intend to overexpress the <I>E. coli</I> marRAB operon, which activates the AcrAB-TolC efflux pump, a mechanism that has been related with resistance to this and similar antimicrobial peptides by expulsion. Our aim is to create a system in which <I>E. coli</I> expels LL-37 only when <I>H. pylori</I> and other pathogenic bacteria are present. In order to do this, we are using the LsrA promoter, which allows transcription in presence of AI-2, a molecule produced by bacteria to communicate via quorum-sensing.</SPAN></P></DIV></DIV><DIV id="u53" class="u53"><DIV id="u53_rtf"><H1><SPAN style="font-family:Museo Slab;font-size:25px;font-weight:normal;font-style:normal;text-decoration:none;color:#666666;">THEORETICAL FRAMEWORK</SPAN></H1><SPAN style="font-family:Museo Slab;font-size:16px;font-weight:normal;font-style:normal;text-decoration:none;color:#666666;">CONTENTS<SPAN style="font-family:Museo Slab;font-size:16px;font-weight:normal;font-style:normal;text-decoration:none;color:#666666;"><A href="#TP">1. The Problem</A></SPAN><SPAN style="font-family:Museo Slab;font-size:14px;font-weight:normal;font-style:normal;text-decoration:none;color:#666666;"><BLOCKQUOTE><A href="#TP.EA">1.1 Etiologic Agent</A><A href="#TP.E">1.2 Epidemiology</A><A href="#TP.PP">1.3 Pathology and Pathogenesis</A><A href="#TP.DP">1.4 Diagnosis and Prescription</A><A href="#TP.T">1.5 Treatment: <I>H. pylori</I> Infections</A><A href="#TP.R">1.6 Resistance</A></BLOCKQUOTE></SPAN><SPAN style="font-family:Museo Slab;font-size:16px;font-weight:normal;font-style:normal;text-decoration:none;color:#666666;"><A href="#TS">2. The Solution</A></SPAN><BLOCKQUOTE><SPAN style="font-family:Museo Slab;font-size:14px;font-weight:normal;font-style:normal;text-decoration:none;color:#666666;"><A href="#TS.QS">2.1 Using <I>quorum sensing</I> to regulate gene expression</A><A href="#TS.AP">2.2 Antimicrobial Peptide</A><A href="#TS.R">2.3 Resistance</A></SPAN></BLOCKQUOTE></SPAN><H2><SPAN style="font-family:Museo Slab;font-size:16px;font-weight:normal;font-style:normal;text-decoration:none;color:#666666;">1. The Problem
</SPAN></H2><P align="justify"><SPAN style="font-family:Arial;font-size:13px;font-weight:normal;font-style:normal;text-decoration:none;color:#666666;"><I>Helicobacter pylori</I> colonizes the stomach of 50% of the world's human population throughout their lifetimes. Colonization with this organism is the main risk factor for peptic ulceration as well as for gastric adenocarcinoma and gastric MALT (mucosa-associated lymphoid tissue) lymphoma. Treatment for <I>H. pylori</I> has revolutionized the management of peptic ulcer disease, providing a permanent cure in most cases. Such treatment also represents first-line therapy for patients with low-grade gastric MALT lymphoma. Treatment of <I>H. pylori</I> is of no benefit in the treatment of gastric adenocarcinoma, but prevention of <I>H. pylori</I> colonization could potentially prevent gastric malignancy and peptic ulceration [1,2]

</SPAN></P><P style="text-align:left;"><SPAN style="font-family:Arial;font-size:13px;font-weight:normal;font-style:normal;text-decoration:none;color:#666666;text-decoration:none;"> </SPAN></P><P style="text-align:left;"><SPAN style="font-family:Museo Slab;font-size:16px;font-weight:normal;font-style:normal;text-decoration:none;color:#666666;">1.1 Etiologic Agent</SPAN><SPAN style="font-family:Arial;font-size:9px;font-weight:normal;font-style:normal;text-decoration:none;color:#666666;"><A href="#TOP">       [Back to top]</A></SPAN></P><SPAN style="font-family:Arial;font-size:13px;font-weight:normal;font-style:normal;text-decoration:none;color:#666666;"><P align="justify"><I>H. pylori</I> is a gram-negative bacillus that has naturally colonized humans for at least 50,000 years—and probably throughout human evolution. It lives in gastric mucus, with a small proportion of the bacteria adherent to the mucosa and possibly a very small number of the organisms entering cells or penetrating the mucosa; its distribution is never systemic. Its spiral shape and flagella render <I>H. pylori</I> motile in the mucus environment. The organism has several acid-resistance mechanisms, most notably a highly expressed urease that catalyzes urea hydrolysis to produce buffering ammonia. <I>H. pylori</I> is microaerophilic (requiring low levels of oxygen), is slow-growing, and requires complex growth media <I>in vitro</I>. Publication of several complete genomic sequences of <I>H. pylori</I> since 1997 has led to significant advances in the understanding of the organism's biology [3].
A very small proportion of gastric <I>Helicobacter</I> infections are due to species other than <I>H. pylori</I>, possibly acquired as zoonoses. Whether these non-<I>pylori</I> gastric helicobacters cause disease remains controversial. In immunocompromised hosts, several nongastric (intestinal) <I>Helicobacter</I> species can cause disease with clinical features resembling those of <I>Campylobacter</I> infections. </P></SPAN><P style="text-align:left;"><SPAN style="font-family:Arial;font-size:13px;font-weight:normal;font-style:normal;text-decoration:none;color:#666666;text-decoration:none;"> </SPAN></P><P style="text-align:left;"><SPAN style="font-family:Museo Slab;font-size:16px;font-weight:normal;font-style:normal;text-decoration:none;color:#666666;">1.2 Epidemiology</SPAN><SPAN style="font-family:Arial;font-size:9px;font-weight:normal;font-style:normal;text-decoration:none;color:#666666;"><A href="#TOP">       [Back to top]</A></SPAN></P><SPAN style="font-family:Arial;font-size:13px;font-weight:normal;font-style:normal;text-decoration:none;color:#666666;"><P align="justify">
The prevalence of <I>H. pylori</I> among adults is 30% in the United States and other developed countries as opposed to &gt;80% in most developing countries. In the United States, prevalence varies with age: 50% of 60-year-old persons, 20% of 30-year-old persons, and &lt;10% of children are colonized.<I>H. pylori</I> is usually acquired in childhood. The age association is due mostly to a birth-cohort effect whereby current 60-year-olds were more commonly colonized as children than are current children. Spontaneous acquisition or loss of <I>H. pylori</I> in adulthood is uncommon. Other strong risk factors for <I>H. pylori</I> colonization are markers of crowding and maternal colonization. The low incidence among children in developed countries at present is due, at least in part, to decreased maternal colonization and increased use of antibiotics.
Humans are the only important reservoir of <I>H. pylori</I>. Children may acquire the organism from their parents (more often from the mother) or from other children. Whether transmission takes place more often by the fecal-oral or the oral-oral route is unknown, but <I>H. pylori</I> is easily cultured from vomitus and gastroesophageal refluxate and is less easily cultured from stool.
In Mexico, seroprevalence studies were positive in 20% of children under 1 year, 50% at 10 years and 80% in those over 25 years [11, 12].
</P></SPAN><P style="text-align:left;"><SPAN style="font-family:Arial;font-size:13px;font-weight:normal;font-style:normal;text-decoration:none;color:#666666;text-decoration:none;"> </SPAN></P><P style="text-align:left;"><SPAN style="font-family:Museo Slab;font-size:16px;font-weight:normal;font-style:normal;text-decoration:none;color:#666666;">1.3 Pathology and Pathogenesis</SPAN><SPAN style="font-family:Arial;font-size:9px;font-weight:normal;font-style:normal;text-decoration:none;color:#666666;"><A href="#TOP">       [Back to top]</A></SPAN></P><SPAN style="font-family:Arial;font-size:13px;font-weight:normal;font-style:normal;text-decoration:none;color:#666666;"><P align="justify"><I>H. pylori</I> colonization induces a tissue response in the stomach, <I>chronic superficial gastritis</I>, which includes infiltration of the mucosa by both mononuclear and polymorphonuclear cells. (The term <I>gastritis</I> should be used specifically to describe histologic features; it has also been used to describe endoscopic appearances and even symptoms, which do not correlate with microscopic findings or even with the presence of <I>H. pylori</I>.) Although <I>H. pylori</I> is capable of numerous adaptations that prevent excessive stimulation of the immune system, colonization is accompanied by a considerable persistent immune response, including the production of both local and systemic antibodies as well as cell-mediated responses. However, these responses are ineffective in clearing the bacterium. This inefficient clearing appears to be due in part to <I>H. pylori</I>'s downregulation of the immune system, which fosters its own persistence. 
The pattern of gastric inflammation is associated with disease risk: antral-predominant gastritis is most closely linked with duodenal ulceration, whereas pangastritis is linked with gastric ulceration and adenocarcinoma. This difference probably explains why patients with duodenal ulceration are not at high risk of developing gastric adenocarcinoma later in life, despite being colonized by <I>H. pylori</I>. 
How gastric colonization causes duodenal ulceration is now becoming clearer. <I>H. pylori</I>–induced inflammation diminishes the number of somatostatin-producing D cells. Since somatostatin inhibits gastrin release, gastrin levels are higher than in <I>H. pylori</I>–negative persons, and these higher levels lead to increased meal-stimulated acid secretion in the gastric corpus, which is only mildly inflamed in antral-predominant gastritis. How this increases duodenal ulcer risk remains controversial, but the increased acid secretion may contribute to the formation of the potentially protective gastric metaplasia found in the duodenum of duodenal ulcer patients. Gastric metaplasia in the duodenum may become colonized by <I>H. pylori</I> and subsequently inflamed and ulcerated. 
The pathogenesis of gastric ulceration and that of gastric adenocarcinoma are less well understood, although both conditions arise in association with pan- or corpus-predominant gastritis. The hormonal changes described above still occur, but the inflammation in the gastric corpus means that it produces less acid (hypochlorhydria) despite hypergastrinemia. Gastric ulcers usually occur at the junction of antral and corpus-type mucosa, and this region is particularly inflamed. Gastric cancer probably stems from progressive DNA damage and the survival of abnormal epithelial cell clones. The DNA damage is thought to be due principally to reactive oxygen and nitrogen species arising from inflammatory cells and perhaps in relation to other bacteria that survive in a hypochlorhydric stomach. Longitudinal analyses of gastric biopsy specimens taken years apart from the same patient show that the common intestinal type of gastric adenocarcinoma follows stepwise changes from simple gastritis to gastric atrophy, intestinal metaplasia, and dysplasia. A second, <I>diffuse</I> type of gastric adenocarcinoma may arise directly from chronic gastritis alone.
</P></SPAN><P style="text-align:left;"><SPAN style="font-family:Arial;font-size:13px;font-weight:normal;font-style:normal;text-decoration:none;color:#666666;text-decoration:none;"> </SPAN></P><P style="text-align:left;"><SPAN style="font-family:Museo Slab;font-size:16px;font-weight:normal;font-style:normal;text-decoration:none;color:#666666;">1.4 Diagnosis and Prescription
</SPAN><SPAN style="font-family:Arial;font-size:9px;font-weight:normal;font-style:normal;text-decoration:none;color:#666666;"><A href="#TOP">       [Back to top]</A></SPAN></P><SPAN style="font-family:Arial;font-size:13px;font-weight:normal;font-style:normal;text-decoration:none;color:#666666;"><P align="justify">

Tests for the presence of <I>H. pylori</I> can be divided into two groups: invasive tests, which require upper gastrointestinal endoscopy and are based on the analysis of gastric biopsy specimens, and noninvasive tests. Endoscopy often is not performed in the initial management of young dyspeptic patients without &quot;alarm&quot; symptoms but is commonly used to exclude malignancy in older patients. If endoscopy is performed, the most convenient biopsy-based test is the biopsy urease test, in which one large or two small antral biopsy specimens are placed into a gel containing urea and an indicator. The presence of <I>H. pylori</I> urease leads to a pH alteration and therefore to a color change, which often occurs within minutes but can require up to 24 h. Histologic examination of biopsy specimens for <I>H. pylori</I> also is accurate, provided that a special stain (e.g., a modified Giemsa or silver stain) permitting optimal visualization of the organism is used. If biopsy specimens are obtained from both antrum and corpus, histologic study yields additional information, including the degree and pattern of inflammation, atrophy, metaplasia, and dysplasia. Microbiologic culture is most specific but may be insensitive because of difficulty with <I>H. pylori</I> isolation. Once the organism is cultured, its identity as <I>H. pylori</I> can be confirmed by its typical appearance on Gram's stain and its positive reactions in oxidase, catalase, and urease tests. Moreover, the organism's susceptibility to antibiotics can be determined, and this information can be clinically useful in difficult cases. The occasional biopsy specimens containing the less common non-<I>pylori</I> gastric helicobacters give only weakly positive results in the biopsy urease test. Positive identification of these bacteria requires visualization of the characteristic long, tight spirals in histologic sections [4,5].
<TABLE style="font-family:Arial;font-size:13px;font-weight:normal;font-style:normal;text-decoration:none;color:#666666;" border="1"><TBODY><TR><TH colspan="3">Tests Commonly Used to Detect <I>Helicobacter pylori</I></TH></TR><TR><TD align="center"><B>Test</B></TD><TD align="center"><B>Advantages</B></TD><TD align="center"><B>Disadvantages</B></TD></TR><TR><TD align="center" colspan="3"><B>Invasive (Based on Endoscopic Biopsy)<B> </B></B></TD></TR><TR class="tg-even"><TD align="center">Biopsy urease test</TD><TD align="center">Quick, simple</TD><TD align="center">Some commercial tests not fully sensitive before 24 h</TD></TR><TR><TD align="center">Histology</TD><TD align="center">May give additional histologic information</TD><TD align="center">Sensitivity dependent on experience and use of special stains</TD></TR><TR class="tg-even"><TD align="center">Culture</TD><TD align="center">Permits determination of antibiotic susceptibility</TD><TD align="center">Sensitivity dependent on experience</TD></TR><TR><TD align="center" colspan="3"><B>Noninvasive</B></TD></TR><TR class="tg-even"><TD align="center">Serology</TD><TD align="center">Inexpensive and convenient; not affected by recent antibiotics or proton pump inhibitors to the same extent as breath and stool tests</TD><TD align="center">Cannot be used for early follow-up after treatment; some commercial kits inaccurate, and all less accurate than breath test</TD></TR><TR><TD align="center"><SUP>13</SUP>C urea breath test</TD><TD align="center">Inexpensive and simpler than endoscopy; useful for follow-up after treatment</TD><TD align="center">Requires fasting; not as convenient as blood or stool tests</TD></TR><TR class="tg-even"><TD align="center">Stool antigen test</TD><TD align="center">Inexpensive and convenient; useful for follow-up after treatment; may be useful in children</TD><TD align="center">May be disliked by people from some cultures; may be slightly less accurate than urea breath test, particularly when used to assess treatment success</TD></TR></TBODY></TABLE></P><P align="justify">

Noninvasive <I>H. pylori</I> testing is the norm if gastric cancer does not need to be excluded by endoscopy. The most consistently accurate test is the urea breath test. In this simple test, the patient drinks a solution of urea labeled with the nonradioactive isotope <SUP>13</SUP>C and then blows into a tube. If <I>H. pylori</I> urease is present, the urea is hydrolyzed and labeled carbon dioxide is detected in breath samples. The stool antigen test, another simple assay, is more convenient and potentially less expensive than the urea breath test but has been slightly less accurate in some comparative studies. The simplest tests for ascertaining <I>H. pylori</I> status are serologic assays measuring specific IgG levels in serum by enzyme-linked immunosorbent assay or immunoblot. The best of these tests are as accurate as other diagnostic methods, but many commercial tests—especially rapid office tests—do not perform well.


</P></SPAN><P style="text-align:left;"><SPAN style="font-family:Arial;font-size:13px;font-weight:normal;font-style:normal;text-decoration:none;color:#666666;text-decoration:none;"> </SPAN></P><P style="text-align:left;"><SPAN style="font-family:Museo Slab;font-size:16px;font-weight:normal;font-style:normal;text-decoration:none;color:#666666;">1.5 Treatment: <I>H. pylori </I>  Infections
</SPAN><SPAN style="font-family:Arial;font-size:9px;font-weight:normal;font-style:normal;text-decoration:none;color:#666666;"><A href="#TOP">       [Back to top]</A></SPAN></P><SPAN style="font-family:Arial;font-size:13px;font-weight:normal;font-style:normal;text-decoration:none;color:#666666;"><P align="justify">

The most clear-cut indications for treatment are <I>H. pylori</I>–related duodenal or gastric ulceration or low-grade gastric B cell lymphoma. <I>H. pylori</I> should be eradicated in patients with documented ulcer disease, whether or not the ulcers are currently active, to reduce the likelihood of relapse. Many guidelines now recommend <I>H. pylori</I> eradication in uninvestigated simple dyspepsia following noninvasive diagnosis; others also recommend treatment in functional dyspepsia, in case the patient is one of the perhaps 7% (beyond placebo effects) to benefit from such treatment. Individuals with a strong family history of gastric cancer should be treated to eradicate <I>H. pylori</I> in the hope that their cancer risk will be reduced. Currently, widespread community screening for and treatment of <I>H. pylori</I> as primary prophylaxis for gastric cancer and peptic ulcers are not recommended, mainly because it is unclear whether treatment for <I>H. pylori</I> reduces the risk of cancer to that in persons who have never acquired the organism. The largest randomized controlled study to date (performed in China) showed no cancer risk reduction during the 7 years of follow-up, although a post hoc subgroup analysis documented improvement in the group of participants who did not already have gastric atrophy or intestinal metaplasia. Other studies have found a reduced cancer risk after treatment, but the size of this effect in different populations remains unclear, and the results of further large-scale prospective interventional studies must be awaited. Other reasons for not treating <I>H. pylori</I> in asymptomatic populations at present include:
<OL><LI>The adverse side effects of the multiple-antibiotic regimens used. (which are common and can be severe in rare cases)</LI><LI>Antibiotic resistance, which may arise in <I>H. pylori</I> or other incidentally carried bacteria</LI><LI>The anxiety that may arise in otherwise healthy people, especially if treatment is unsuccessful.</LI><LI> The apparent existence of a subset of people who will develop GERD symptoms after treatment, although on average H. pylori treatment does not affect GERD symptoms or severity.</LI></OL></P><P align="justify">
Although <I>H. pylori</I> is susceptible to a wide range of antibiotics in vitro, monotherapy is not usually successful, probably because of inadequate antibiotic delivery to the colonization niche. Failure of monotherapy has prompted the development of multidrug regimens, the most successful of which are triple and quadruple combinations. Initially these regimens produced <I>H. pylori</I> eradication rates of &gt;90% in many trials; in recent years, however, resistance to key antibiotics has become more common, a trend leading to <I>H. pylori</I> eradication rates of only 75–80% for the most commonly used regimens. Current regimens consist of a PPI or ranitidine bismuth citrate and two or three antimicrobial agents given for 7–14 days. Research on optimizing drug combinations to increase efficacy continues, and it is likely that guidelines will change as the field develops and as countries increasingly individualize treatment to suit local antibiotic resistance patterns and economic needs [6,7].
An increasing number of infected individuals are found to harbor antibiotic-resistant bacteria. This results in initial treatment failure and requires additional rounds of antibiotic therapy or alternative strategies, such as a quadruple therapy.[8,9] 
<TABLE style="font-family:Arial;font-size:13px;font-weight:normal;font-style:normal;text-decoration:none;color:#666666;" border="1"><TBODY><TR><TH colspan="5">Recommended Treatment Regimens for <I>Helicobacter pylori</I></TH></TR><TR class="tg-even"><TD align="center"><B>Regimen (Duration)</B></TD><TD align="center"><B>Drug 1</B></TD><TD align="center"><B>Drug 2</B></TD><TD align="center"><B>Drug 3</B></TD><TD align="center"><B>Drug 4</B></TD></TR><TR><TD align="center"><B>Regimen 1: OCM (7–14 days)<SUP>a</SUP></B></TD><TD align="center">Omeprazoleb (20 mg bid) </TD><TD align="center">Clarithromycin (500 mg bid)</TD><TD align="center">Metronidazole (500 mg bid)</TD><TD align="center">—</TD></TR><TR class="tg-even"><TD align="center"><B>Regimen 2: OCA (7–14 days)<SUP>a</SUP></B></TD><TD align="center">Omeprazoleb (20 mg bid) </TD><TD align="center">Clarithromycin (500 mg bid)</TD><TD align="center">Amoxicillin (1 g bid)</TD><TD align="center">—</TD></TR><TR><TD align="center"><B>Regimen 3: OBTM (14 days)<SUP>c</SUP></B></TD><TD align="center">Omeprazoleb (20 mg bid) </TD><TD align="center">Bismuth subsalicylate (2 tabs qid)</TD><TD align="center">Tetracycline HCl (500 mg qid)</TD><TD align="center">Metronidazole (500 mg tid)</TD></TR><TR class="tg-even"><TD align="center" rowspan="2"><B>Regimen 4<SUP>d</SUP>: sequential (5 days + 5 days)</B></TD><TD align="center">Omeprazoleb (20 mg bid) </TD><TD align="center">Amoxicillin (1 g bid)</TD><TD align="center">—</TD><TD align="center">—</TD></TR><TR><TD align="center">Omeprazoleb (20 mg bid) </TD><TD align="center">Clarithromycin (500 mg bid)</TD><TD align="center">Tinidazole (500 mg bid)</TD><TD align="center">—</TD></TR><TR class="tg-even"><TD align="center"><B>Regimen 5<SUP>e</SUP>: OAL (10 days)</B></TD><TD align="center">Omeprazoleb (20 mg bid) </TD><TD align="center">Amoxicillin (1 g bid)</TD><TD align="center">Levofloxacin (500 mg qid)</TD><TD align="center">—</TD></TR></TBODY></TABLE></P></SPAN><SPAN style="font-family:Arial;font-size:10px;font-weight:normal;font-style:normal;text-decoration:none;color:#666666;"><P align="justify"><UL><LI><SUP>a</SUP> Meta-analyses show that a 14-day course of therapy is slightly superior to a 7-day course. However, in populations where 7-day treatment is known to have very high success rates, this shorter course is still often used.
</LI><LI><SUP>b</SUP> Omeprazole may be replaced with any proton pump inhibitor at an equivalent dosage or, in regimens 1 and 2, with ranitidine bismuth citrate (400 mg).
</LI><LI><SUP>c</SUP> Data supporting this regimen come mainly from Europe and are based on the use of bismuth subcitrate and metronidazole (400 mg tid). This is the most commonly used second-line regimen.
</LI><LI><SUP>d</SUP>Data supporting this regimen come from Europe. Although the two 5-day courses of different drugs have usually been given sequentially, recent evidence suggests no added benefit from this approach. Thus 10 days of the four drugs combined may be as good and may aid compliance.
</LI><LI><SUP>e</SUP> Data supporting this second- or third-line regimen come from Europe. This regimen may be less effective where rates of quinolone use are high. Theoretically, it may also be wise to avoid it in populations where Clostridium difficile infection is common after broad-spectrum antibiotic use.
</LI></UL></P></SPAN><SPAN style="font-family:Arial;font-size:13px;font-weight:normal;font-style:normal;text-decoration:none;color:#666666;"><P align="justify">
The standard first-line therapy is a one week triple therapy consisting of proton pump inhibitors such as omeprazole and the antibiotics clarithromycin and amoxicillin [6,7]. Variations of the triple therapy have been developed over the years, such as using a different proton pump inhibitor, as with pantoprazole or rabeprazole [7].

The two most important factors in successful <I>H. pylori</I> treatment are the patient's close compliance with the regimen and the use of drugs to which the patient's strain of <I>H. pylori</I> has not acquired resistance. Treatment failure following minor lapses in compliance is common and often leads to acquired resistance to metronidazole or clarithromycin. To stress the importance of compliance, written instructions should be given to the patient, and minor side effects of the regimen should be explained. Resistance to clarithromycin and, to a lesser extent, to metronidazole is of growing concern. Clarithromycin resistance is less prevalent but, if present, usually results in treatment failure. Strains of <I>H. pylori</I> that are apparently resistant to metronidazole are more common but still may be cleared by metronidazole-containing regimens, which have only slightly reduced efficacy. Assessment of antibiotic susceptibilities before treatment would be optimal but is not usually undertaken because endoscopy and mucosal biopsy are necessary to obtain <I>H. pylori</I> for culture and because most microbiology laboratories are inexperienced in <I>H. pylori</I> culture. In the absence of susceptibility information, a history of the patient's (even distant) antibiotic use for other conditions should be obtained; use of the agent should then be avoided if possible, particularly in the case of clarithromycin (e.g., previous use for upper respiratory infection). If initial <I>H. pylori</I> treatment fails, one of two strategies may be used. The more common approach is empirical re-treatment with another drug regimen, usually quadruple therapy. The second approach is endoscopy, biopsy, and culture plus treatment based on documented antibiotic sensitivities. If re-treatment fails, susceptibility testing should ideally be performed, although empirical third-line therapies are often used.
Clearance of non-<I>pylori</I> gastric helicobacters can follow the use of bismuth compounds alone or of triple-drug regimens. However, in the absence of trials, it is unclear whether this outcome represents successful treatment or natural clearance of the bacterium

</P></SPAN><P style="text-align:left;"><SPAN style="font-family:Arial;font-size:13px;font-weight:normal;font-style:normal;text-decoration:none;color:#666666;text-decoration:none;"> </SPAN></P><P style="text-align:left;"><SPAN style="font-family:Museo Slab;font-size:16px;font-weight:normal;font-style:normal;text-decoration:none;color:#666666;">1.6 Resistance

</SPAN><SPAN style="font-family:Arial;font-size:9px;font-weight:normal;font-style:normal;text-decoration:none;color:#666666;"><A href="#TOP">       [Back to top]</A></SPAN></P><SPAN style="font-family:Arial;font-size:13px;font-weight:normal;font-style:normal;text-decoration:none;color:#666666;"><P align="justify">

Because of the emergence of antibiotic resistance and adverse drug reactions, such as diarrhea, eradication rates with this triple therapy are falling. The worldwide appearance of drug resistance to <I>H. pylori</I> has led to a search for new therapeutic agents that may help to control <I>H. pylori</I> infection and its associated morbidities [13].

</P><H2><SPAN style="font-family:Museo Slab;font-size:16px;font-weight:normal;font-style:normal;text-decoration:none;color:#666666;">2. The Soultion
</SPAN></H2><P style="text-align:left;"><SPAN style="font-family:Arial;font-size:13px;font-weight:normal;font-style:normal;text-decoration:none;color:#666666;text-decoration:none;"> </SPAN></P><P style="text-align:left;"><SPAN style="font-family:Museo Slab;font-size:16px;font-weight:normal;font-style:normal;text-decoration:none;color:#666666;">2.1 Using <I>quorum sensing</I> to regulate gene expression

</SPAN><SPAN style="font-family:Arial;font-size:9px;font-weight:normal;font-style:normal;text-decoration:none;color:#666666;"><A href="#TOP">       [Back to top]</A></SPAN></P><SPAN style="font-family:Arial;font-size:13px;font-weight:normal;font-style:normal;text-decoration:none;color:#666666;"><P align="justify">

Quorum sensing or the regulation of gene expression in response to cell population density is a process that bacteria use to co-ordinate the gene expression of the community. Presumably, the ability to control behaviour on a collective scale enables bacteria to behave like multi- cellular organisms. <I>Quorum sensing</I> involves the pro- duction of extracellular signalling molecules called autoinducers. As a population of autoinducer-producing bacteria grows, the external concentration of autoinducer increases. When a threshold autoinducer concentration is reached, the bacteria detect the autoinducer and initiate a signal transduction cascade that culminates in a change in the behaviour of the population [1,2].
Autoinducer-2 (AI-2) is a quorum-sensing signal produced by the LuxS protein that accumulates in the bacterial environment in a density- dependent manner. LuxS is the AI-2 synthase and that AI-2 is produced from S-adenosylmethionine in three enzymatic steps. The substrate for LuxS is S-ribosylhomo-cysteine, which is cleaved to form two products, one of which is homocysteine, and the other is AI-2. In this report, we also provide evidence that the biosynthetic pathway and biochemical intermediates in AI-2 biosynthesis are identical in <I>Escherichia coli, Salmonella typhimurium, V. harveyi, Vibrio cholerae</I> and <I>Enterococcus faecalis</I> [4]. In 2011, Rader et al. showed that Helicobacter pylori moves in response to environmental chemical cues using AI-2 which accumulates in the bacterial environment in a density-dependent manner [3]. 
The regulatory network for AI-2 uptake is comprised of two important components, <I>lsrR</I> and <I>lsrK</I>, both of which are located adjacent, but divergently transcribed from the <I>lsr</I> operon (Figure 1). LsrR is the repressor of the <I>lsr</I> operon and itself. LsrK is a kinase responsible for converting AI-2 to phospho-AI-2, which is required for relieving LsrR repression. Functions of LsrR and LsrK can be partially revealed using <I>lsrK</I> and <I>lsrR</I> mutants. In <I>lsrR</I> mutants, the LsrACDB transporter is produced, and extracellular AI-2 is continuously imported into the cell (cytoplasmic AI-2). In <I>lsrK</I> mutants, the expression of <I>lsr</I> is repressed, and AI-2 remains in the supernatant (extracellular AI-2) [5-10]. Since LsrR contains a helix-turn-helix (HTH) DNA-binding domain, it was hypothesized that LsrR represses the expression of <I>lsr</I> operon and itself by binding to their promoter regions [6, 9, 11]. However, no evidence of their direct binding has been shown yet. It has also been postulated that phospho-AI-2 binds to LsrR and inactivates it to derepress the transcription of <I>lsr</I> [5, 6]. 
In our system, LL-37, the antimicrobial peptide, will be under the <I>lsr</I> promoter, so it will only be produced when a pathogenic bacteria, such as <I>Helicobacter pylori</I>  is present, producing AI-2. </P></SPAN><SPAN style="font-family:Arial;font-size:13px;font-weight:normal;font-style:normal;text-decoration:none;color:#666666;"><P align="justify"><B>Figure 1</B> Model for regulation, transportation, and modification of AI-2 by the Lsr proteins in <I>E. coli</I>. AI-2 is synthesized by LuxS and accumulates extracellularly. The AI-2 uptake repressor LsrR represses the <I>lsr</I> operon (comprised of <I>lsrACDBFG</I>) and the <I>lsrRK</I>. Basal expression of the LsrACDB transporter allows some AI-2 to enter the cytoplasm, where it is phosphorylated by LsrK. Phospho-AI-2 has been reported to bind to LsrR and relieve its repression effect on the <I>lsr</I> transporter genes, thus stimulating additional AI-2 uptake.
</P></SPAN><P style="text-align:left;"><SPAN style="font-family:Arial;font-size:13px;font-weight:normal;font-style:normal;text-decoration:none;color:#666666;text-decoration:none;"> </SPAN></P><P style="text-align:left;"><SPAN style="font-family:Museo Slab;font-size:16px;font-weight:normal;font-style:normal;text-decoration:none;color:#666666;">
2.2 Antimicrobial peptide
</SPAN><SPAN style="font-family:Arial;font-size:9px;font-weight:normal;font-style:normal;text-decoration:none;color:#666666;"><A href="#TOP">       [Back to top]</A></SPAN></P><SPAN style="font-family:Arial;font-size:13px;font-weight:normal;font-style:normal;text-decoration:none;color:#666666;"><P align="justify">

Cathelicidin, an antimicrobial peptide of the innate immune system, has been shown to modulate microbial growth, wound healing and inflammation. It has been shown that genetic ablation of cathelicidin-related antimicrobial peptides (CRAMP) in mice significantly increased the susceptibility of <I>H. pylori</I> colonization and the associated gastritis as compared with the wild-type control. Furthermore, replenishment with exogenous CRAMP, delivered via a bioengineered CRAMP-secreting strain of <I>Lactococcus lactis</I>, reduced <I>H. pylori</I> density in the stomach as well as the associated inflammatory cell infiltration and cytokine production [1].
 The relationship between <I>H. pylori</I> and cathelicidin was first  addressed by Hase et al. <I>H. pylori</I> infection upregulated the  expression of human cathelicidin (LL-37), in gastric secretion and epithelium. Moreover, they found that LL-37 alone or together with another host defense peptide known as human b-defensin-1 could effectively kill several strains of <I>H. pylori</I>, including SD4,  SD14 and Sydney Strain 1 (SS1) in vitro [2]. The effects of cathelicidin on <I>E. coli</I> included the ability to permeabilize both cell membranes, as could be observed by the increase of β-galactosidase activity in extracellular space in time [3]. Cationic antimicrobial peptides (CAMPs) such as LL-37 utilize the negative charge of the bacterial cell membranes to collect on, and form hydrophilic channels through, the outer and inner membranes of the bacterial cells, causing osmotic damage to the bacterium [4]. CAMPs also affect bacterial cytoplasmic proteins [5].
Since LL-37 is the only human antimicrobial peptide in the cathelicidin family, and shows a broad spectrum of antimicrobial activity at physiologic or elevated salt concentrations, there is a significant interest in developing this peptide for pharmaceutical applications [6]. Hong <I>et al.</I> produced recombinant LL-37 peptide, attaching the initiating amino acid, methionine, to the N-terminal region of recombinant LL-37.  This LL-37 crude extract (with methionine as the initiating residue) from <I>P. pastoris</I> showed an antimicrobial activity against <I>Micrococcus luteus</I>. Theirs successful expression of human LL-37 indicates that the system may be applicable to the expression of human defensins without resorting to fusion protein constructions [7].
Most of the current expression systems for LL-37 depend on the construction of soluble fusion partners, such as thioredoxin and glutathione S-transferase, to avoid cytotoxic effects of the antimicrobial peptide in the <I>E. coli</I> host strain [8]. However, the fusion systems require additional cleavage steps using enzymatic (thrombin or Factor Xa) or chemical (CNBr, formic acid) methods [7].
In laboratory assay conditions, this peptide inhibits the growth of a variety of Gram-negative (<I>Pseudomonas aeruginosa, Salmonella typhimurium, Escherichia coli</I>) and Gram-positive (<I>Staphylococcus aureus, Staphylococcus epidermis, Listeria monocytogenes, and vancomycin-resistant enterococci</I>) species in the micromolar and sub-micromolar range of peptide concentration [9].
The major problem of the conventional antibiotics being the tendency of the target microorganism to acquire resistance and the side effects of some antibiotics further burdening the organism make the scientific research is focused on looking for alternatives with negative traits such as these eliminated. One of the most hopeful candidates, which fulfill the requirements of the modern medicine, appears to be antimicrobial peptides of human origin. The cationic cathelicidin (LL-37) is one of the better known and researched antimicrobial peptides showing high antimicrobial activity against both Gram-positive and Gram-negative microorganisms. The critical point to make the production of recombinant LL- 37 interesting for the pharmaceutical industry is the cost-efficiency of the expression system used. In this work, we tried to establish an expression system based on <I>E. coli</I> capable of producing sufficient amounts of the antimicrobial peptide, and subsequent protein purification methods applicable on industrial scale [3].</P></SPAN><SPAN style="font-family:Arial;font-size:13px;font-weight:normal;font-style:normal;text-decoration:none;color:#666666;text-decoration:none;"> </SPAN><P style="text-align:left;"><SPAN style="font-family:Museo Slab;font-size:16px;font-weight:normal;font-style:normal;text-decoration:none;color:#666666;">
2.3 Resistance
</SPAN><SPAN style="font-family:Arial;font-size:9px;font-weight:normal;font-style:normal;text-decoration:none;color:#666666;"><A href="#TOP">       [Back to top]</A></SPAN></P><SPAN style="font-family:Arial;font-size:13px;font-weight:normal;font-style:normal;text-decoration:none;color:#666666;"><P align="justify">

Transcriptional regulators are an important component of intrinsic antibiotic resistance and are frequently the sites of spontaneous mutation in antibiotic-resistant strains isolated from the clinic or environment [1]. Antimicrobial peptides are found in locations and cells that interact with E. coli during colonization of the human host. Warner <I>et al.</I> discovered that overexpression of <I>marA</I> decreases susceptibility to CAMPs via upregulation of the AcrAB-TolC efflux pump and possibly other TolC-dependent RND efflux pumps [2].
The AcrB and AcrA proteins respectively make up the inner membrane and periplasm-spanning regions of the tripartite <I>E. coli</I> RND efflux pump AcrAB-TolC, which acts to expel a wide variety of substrates including dyes, bile salts, organic solvents, and structurally dissimilar antibiotics (<B>Fig. 1</B>) [3]. The TolC protein component is located in the bacterial outer membrane and also pairs with subunits of other membrane pumps [4]. CAMPs have a molecular mass of 3–4 kDa, much larger than that of the chemicals and dyes that are known substrates of the AcrAB-TolC efflux pump. Still, TolC can act as a portal for such large substrates as haemolysin and colicins.
Warner <I>et al.</I> discovered that while a deletion of the <I>marA</I> gene had only a slight impact on CAMP susceptibility, overexpression of MarA from plasmid pSMarAB produced an antibiotic-resistant <I>E. coli</I> strain that was also sevenfold more resistant to host CAMPs [2]. </P></SPAN></SPAN></DIV></DIV><DIV id="u54" class="u54"><DIV id="u54_rtf"><P style="text-align:left;"><SPAN style="font-family:Museo Slab;font-size:16px;font-weight:normal;font-style:normal;text-decoration:none;color:#666666;">PROJECT MENU</SPAN></P></DIV></DIV><DIV id="u55" class="u55"><DIV id="u55_rtf"><P style="text-align:left;"><SPAN style="font-family:Arial;font-size:14px;font-weight:bold;font-style:normal;text-decoration:none;color:#929EA0;">Overview</SPAN></P></DIV></DIV><DIV id="u56" class="u56"><DIV id="u56_rtf"><P style="text-align:left;"><SPAN style="font-family:Arial;font-size:14px;font-weight:bold;font-style:normal;text-decoration:none;color:#929EA0;">General Design</SPAN></P></DIV></DIV><DIV id="u57" class="u57"><DIV id="u57_rtf"><P style="text-align:left;"><SPAN style="font-family:Arial;font-size:14px;font-weight:bold;font-style:normal;text-decoration:none;color:#929EA0;">Wet Lab</SPAN></P></DIV></DIV><DIV id="u58" class="u58"><DIV id="u58_rtf"><P style="text-align:left;"><SPAN style="font-family:Arial;font-size:14px;font-weight:normal;font-style:normal;text-decoration:none;color:#929EA0;">BioBricks</SPAN></P></DIV></DIV><DIV id="u59" class="u59"><DIV id="u59_rtf"><P style="text-align:left;"><SPAN style="font-family:Arial;font-size:14px;font-weight:normal;font-style:normal;text-decoration:none;color:#929EA0;">Results</SPAN></P></DIV></DIV><DIV id="u60" class="u60"><DIV id="u60_rtf"><P style="text-align:left;"><SPAN style="font-family:Arial;font-size:14px;font-weight:normal;font-style:normal;text-decoration:none;color:#929EA0;">Characterization</SPAN></P></DIV></DIV><DIV id="u61" class="u61"><DIV id="u61_rtf"><P style="text-align:left;"><SPAN style="font-family:Arial;font-size:14px;font-weight:normal;font-style:normal;text-decoration:none;color:#929EA0;">Notebook</SPAN></P></DIV></DIV><DIV id="u62" class="u62"><DIV id="u62_rtf"><P style="text-align:left;"><SPAN style="font-family:Arial;font-size:14px;font-weight:bold;font-style:normal;text-decoration:none;color:#929EA0;">Dry Lab</SPAN></P></DIV></DIV><DIV id="u63" class="u63"><DIV id="u63_rtf"><P style="text-align:left;"><SPAN style="font-family:Arial;font-size:14px;font-weight:normal;font-style:normal;text-decoration:none;color:#929EA0;">Modelling</SPAN></P></DIV></DIV><DIV id="u64" class="u64"><DIV id="u64_rtf"><P style="text-align:left;"><SPAN style="font-family:Arial;font-size:14px;font-weight:bold;font-style:normal;text-decoration:none;color:#929EA0;">Conclusions </SPAN></P></DIV></DIV><DIV id="u65" class="u65"><DIV id="u65_rtf"><P style="text-align:left;"><SPAN style="font-family:Arial;font-size:14px;font-weight:normal;font-style:normal;text-decoration:none;color:#929EA0;">Future Work</SPAN></P></DIV></DIV><DIV id="u66" class="u66"><DIV id="u66_rtf"><P style="text-align:left;"><SPAN style="font-family:Arial;font-size:14px;font-weight:bold;font-style:normal;text-decoration:none;color:#929EA0;">References</SPAN></P></DIV></DIV></DIV></DIV><DIV class="printfooter">
Retrieved from &quot;<A href="http://2013.igem.org/Team:Ciencias-UNAM/Project">http://2013.igem.org/Team:Ciencias-UNAM/Project</A>&quot;</DIV></DIV></DIV><DIV id="footer-box" class="noprint"><DIV id="footer"><UL id="f-list"><LI id="t-recentchanges"><A href="/Special:RecentChanges" title="Recent changes">Recent changes</A></LI><LI id="t-whatlinkshere"><A href="/Special:WhatLinksHere/Team:Ciencias-UNAM/Project" title="List of all wiki pages that link here [j]" accesskey="j">What links here</A></LI><LI id="t-recentchangeslinked"><A href="/Special:RecentChangesLinked/Team:Ciencias-UNAM/Project" title="Recent changes in pages linked from this page [k]" accesskey="k">Related changes</A></LI><LI id="t-specialpages"><A href="/Special:SpecialPages" title="List of all special pages [q]" accesskey="q">Special pages</A></LI><LI><A href="/Special:Preferences">My preferences</A></LI></UL></DIV><DIV id="footer"><UL id="f-list"><LI id="t-print"><A href="/wiki/index.php?title=Team:Ciencias-UNAM/Project&amp;printable=yes" title="Printable version of this page [p]" accesskey="p">Printable version</A></LI><LI id="t-permalink"><A href="/wiki/index.php?title=Team:Ciencias-UNAM/Project&amp;oldid=217032" title="Permanent link to this revision of the page">Permanent link</A></LI><LI id="privacy"><A href="/2013.igem.org:Privacy_policy" title="2013.igem.org:Privacy policy">Privacy policy</A></LI><LI id="disclaimer"><A href="/2013.igem.org:General_disclaimer" title="2013.igem.org:General disclaimer">Disclaimers</A></LI></UL></DIV></DIV></BODY></HTML>