"
Page
Discussion
View source
History
teams
Log in
 
Team:Groningen/Notebook/Biobricks
From 2014.igem.org
7 – 13 Juli
Primers were designed for making BioBricks out of the separate genes from  the nisin operon. For this, the sequence of the transposon Tn5307 was used, a transposon that contains the nisin operon, and of  which the sequence is known.1 This sequence is close to the sequence of the transposon Tn5276, the transposon which will form the template for making the new BioBricks. The primers for making a BioBrick out of PNisI with a RBS were based on a sequence found in a research paper that documented the identification of this promoter.2
It was decided to combine the genes NisR and NisK as one BioBrick and NisF, NisE and NisG as one BioBrick, because these genes are related in function or work together.
Primer
Sequence
NisA forward
GTTTCTTCGAATTCGCGGCCGCTTCTAGATGAGTACAAAAGATTTTAA
NisA reverse
GTTTCTTCCTGCAGCGGCCGCTACTAGTATTATTATTTGCTTACGTGAA
NisB forward
GTTTCTTCGAATTCGCGGCCGCTTCTAGATGATAAAAAGTTCATTTAA
NisB reverse
GTTTCTTCCTGCAGCGGCCGCTACTAGTATTATCATTTCATGTATTCTT
NisT forward
GTTTCTTCGAATTCGCGGCCGCTTCTAGATGGATGAAGTGAAAGAATTTACATCA
NisT reverse
GTTTCTTCCTGCAGCGGCCGCTACTAGTATTATTATTCATCATTATCCT
NisC forward
GTTTCTTCGAATTCGCGGCCGCTTCTAGATGAATAAAAAAAATATAAA
NisC reverse
GTTTCTTCCTGCAGCGGCCGCTACTAGTATTATCATTTCCTCTTCCCTC
NisI forward
GTTTCTTCGAATTCGCGGCCGCTTCTAGATGAGAAGATATTTAATACT
NisI reverse
GTTTCTTCCTGCAGCGGCCGCTACTAGTATTACTAATTTCCTACCTTCG
NisP forward
GTTTCTTCGAATTCGCGGCCGCTTCTAGATGAAAAAAATACTAGGTTT
NisP reverse
GTTTCTTCCTGCAGCGGCCGCTACTAGTATTATCAATTTTTAGTCTTTC
NisRK forward
GTTTCTTCGAATTCGCGGCCGCTTCTAGGTGTATAAAATTTTAATAGT
NisRK reverse
GTTTCTTCCTGCAGCGGCCGCTACTAGTATTATTACTTTTTTATTTTTA
NisFEG forward
GTTTCTTCGAATTCGCGGCCGCTTCTAGATGCAGGTAAAAATTCAAAA
NisFEG reverse
GTTTCTTCCTGCAGCGGCCGCTACTAGTATTATTATCTAATCTTTTTTT
PNisI+RBS forward
GTTTCTTCGAATTCGCGGCCGCTTCTAGAGAGCACTGGATAATGACTATT
PNisI+RBS reverse
TACTAGTAGCGGCCGCTGCAGGAAGAAACTTCCTCTTCCCTCCTTTCAA
There were also primer designed for making a BioBrick out of the sfGFP(Bs). This sfGFP was originally optimized for Bacillus subtilis. When it was tested in Lactococcus lactis, the sfGFP(Bs) was shown to perform really well in L. lactis as well.3
Primer
Sequence
sfGFP(Bs) forward
GTTTCTTCGAATTCGCGGCCGCTTCTAGATGTCAAAAGGAGAAGAGCT
sfGFP(Bs) reverse
GTTTCTTCCTGCAGCGGCCGCTACTAGTATTATTACTTATAAAGCTCAT
14 - 20 Juli
A collection of constitutive promoters was also desirable for the toolbox. We decided to use the CP promoter collection and test it in Lactococcus lactis, that was BioBricked by the Uppsala iGEM team of 2013. The CP promoters are registered as BBa_K1033219 (CP1), BBa_K1033220 (CP8), BBa_K1033221 (CP11), BBa_K1033222 (CP29), BBa_K1033223 (CP30), BBa_K1033224 (CP41) and BBa_K1033225 (CP44). These parts were ordered from iGEM HQ, except for CP8 as for this promoter the sequencing was inconsistent. The promoters were sent by iGEM HQ using Escherichia coli containing pSB1C3 with the promoter as insert. E. coli was grown and the plasmids with the promoter were isolated.
21 - 27 Juli
BioBricking the genes NisT, NisI, NisP and the combined genes NisF,  NisE and NisG gave problems with illegal restriction sites.
The illegal restrictionsite in NisI could be removed using the reverse primer as listed above. This primer contains a mutation needed to remove the illegal restriction site.
The other parts gave more problems, as these all contained multiple restriction sites. Therefore, it was decided to use Gibson assembly with primers containing mutations to remove the restriction site. By using Gibson assembly, the genes could also be constructed in pSB1C3 in one go.
The primers in the table below are denoted with the position of the  illegal restriction site that they should remove. Illegal EcoRI sites  are abbreviated with an E, XbaI sites with an X, SpeI site with an S and  PstI sites with a P.
Gene
Primer
Sequence
NisT
Prefix forward
GTTTCTTCGAATTCGCGGCCGCTTCTAGATGGATGAAGTGAAAGAATTTACATCA
S980 reverse
TTGTTCCATAAACAAACTTGTATTATAAATGATGTAAATA
S980 forward
TATTTACATCATTTATAATACAAGTTTGTTTATGGAACAA
S1210 reverse
CCATAGTTGGTTGATATAATCCTGAAATTATCTTTACTTGTGTAC
S1210 forward
GTACACAAGTAAAGATAATTTCAGGATTATATCAACCAACTATGG
Suffix reverse
GTTTCTTCCTGCAGCGGCCGCTACTAGTATTATTATTCATCATTATCCT
Suffix forward
ATGAATAATAATACTAGTAGCGGCCGCTGCAGGAAGAAACAAAAGGGCAA
Prefix reverse
CACTTCATCCATCTAGAAGCGGCCGCGAATTCGAAGAAACATCCTTAGCG
NisP
Prefix forward
GTTTCTTCGAATTCGCGGCCGCTTCTAGGTGAAAAAAATACTAGGTTT
P225 reverse
AGTCCTTGTTGTCGATTTTACTGCTGGTGACTGCGCCCCT
P225 forward
AGGGGCGCAGTCACCAGCAGTAAAATCGACAACAAGGACT
E347 reverse
AACGCTATCTCTCTTACTAAATTCAGAACTAACTTGAGTT
E347 forward
AACTCAAGTTAGTTCTGAATTTAGTAAGAGAGATAGCGTT
E1657 reverse
CAATACTCTATTCCCATTAACTTCTGGACTATTCATTAGA
E1657 forward
TCTAATGAATAGTCCAGAAGTTAATGGGAATAGAGTATTG
Suffix reverse
GTTTCTTCCTGCAGCGGCCGCTACTAGTATTATCAATTTTTAGTCTTTC
Suffix forward
AAATTGATAATACTAGTAGCGGCCGCTGCAGGAAGAAACAAAAGGGCAA
Prefix reverse
CCTAGTATTTTTTTCATCTAGAAGCGGCCGCGAATTCGAAGAAACATCCTTAGCG
NisFEG
Prefix forward
GTTTCTTCGAATTCGCGGCCGCTTCTAGATGCAGGTAAAAATTCAAAA
P186 reverse
TCCAGTATCAGCAGAAATTAAACCAAACAAAATTTTCATC
P186 forward
GATGAAAATTTTGTTTGGTTTAATTTCTGCTGATACTGGA
E336/X349 reverse
CCTGTTTCTGCCAAACCAATCACTTCTAGTGTTTCATGTATTCTCTTA
E336/X349 forward
TAAGAGAATACATGAAACACTAGAAGTGATTGGTTTGGCAGAAACAGG
Suffix reverse
GTTTCTTCCTGCAGCGGCCGCTACTAGTATTATTATCTAATCTTTTTTT
Suffix forward
TAGATAATAATACTAGTAGCGGCCGCTGCAGGAAGAAACAAAAGGGCAA
Prefix reverse
TTTTACCTGCATCTAGAAGCGGCCGCGAATTCGAAGAAACATCCTTAGCG
28 Juli - 3 August
The CP promoters in pSB1C3, that were isolated in the week of 14 - 20 Juli, were tested on their insert size. For this, a PCR was done on the plasmid using the primers VF2 (BBa_G00100) and VR (BBa_G00101) were used. The insert size corresponded to the expected size.
A touchdown PCR was done on the genes of the nisin operon and the  sfGFP(Bs) gene. In this PCR the annealing temperature dropped from 55 °C to  45 °C, lowering the temperature with 1 &degC each cycle. This  unfortunately resulted in a PCR program of just 10 cycles, instead of the intended 30. The remaining steps of the program were finished the next morning. Only the genes NisA, PNisI and sfGFP(Bs) were amplified this way, see figure 1. Therefore, the PCR was repeated, this time with a complete cycle. No additional genes were amplified this way.
Another attempt was made for amplification of the remaining genes. This time, the most ideal annealing temperature for each gene was used by using a gradient PCR and placing the tubes at the optimal temperature. This still did not amplify the remaining genes. Also, a PCR with a general annealing temperature of 50 &degC was done. This also did not amplify the remaining genes.
The genes that were amplified in the first PCR (NisA, PNisI and sfGFP(Bs)) were purified using the GeneJET PCR Purification Kit from Thermo Scientific. The purified products were digested with the enzymes EcoRI  and PstI, using 2 &mul of the product. These purified and restricted products were loaded on gel, see figure 2. It was then discovered that the genes were barely visible on gel. The restricted genes were purified with the GeneJET PCR Purification Kit and the concentration was measured with the NanoDrop 1000. The clean, restricted products were too low in  concentration to be suitable for ligation.
4 - 10 August
Because the PCR products of NisA, PNisI and sfGFP(Bs) were lost during the purification and restriction in the week of 28 Juli - 3 August, the PCR was repeated for this genes, together with the genes that could not be amplified yet. This time a PCR was used that did not lower in  temperature, like the touchdown PCR, but that increased in temperature  with each step. This way it was hoped to get over the huge gap between the annealing temperature of the primer in the first cycle and the annealing temperature of the primer when the flap of the primer can also  anneal to the first PCR products. The temperature was set to increase from 40 &degC to 60 &degC in 20 cycles. Then, an additional 20 cycles were done at 65 &degC. The PCR was performed under standard conditions as was done before, together with a series of PCR that contained GC buffer (supplied with the Phusion DNA polymerase by Thermo Scientific), and a series of PCR with GC buffer and 1.5% DMSO. This time, with the help of Lisa, the PCR was finally successful for all genes.
11 - 17 August
The PCR on the BioBrick of the combined genes NisR and NisK was repeated because the yield of this product was very low after the PCR in the week of 4 - 10 August. The mixture of the previous PCR reaction was used as the template. The PCR was performed under standard conditions, with an annealing temperature of 64 &degC and 1 &mul, 2&mul and 3 &mul template. No product was obtained with this PCR. So the PCR was once again repeated, this time using diluted template. The mixture was diluted 50x, 300x, 1500x and 15000x. The PCR was repeated using 1 &mul and 2 &mul of each of the dilutions. This way, product was obtained for all reactions.
The PCR products of the genes NisA, NisB, NisC, NisRK, PNisI and sfGFP(Bs) were purified using the GeneJET purification kit. Then 1 &mug of each purified product was restricted with EcoRI and PstI, together with 1 &mug of the pSB1C3 plasmid. The restriction enzymes were then inactivated by heating the samples at 80 &degC for 20 minutes. Ligated 6 &mul of the restricted PCR products to 2 &mul restricted pSB1C3. Inactivated the ligase by incubating at 65 &degC for 10 minutes. Mixed 5 &mul of the ligation mixture with 25 &mul electrocompetent Escherichia coli DH5&alfa.
18 - 31 August
References
1. Trmčić, A. et al. (2011) Complete nisin A gene cluster from Lactococcus  lactis M78 (HM219853) – obtaining the nucleic acid sequence and  comparing it to other published nisin sequences. Genes Genom. 33: 217-221
2. Li, H. and O´Sullivan, D.J. (2006) Identification of a nisI Promoter  within the nisABCTIP Operon That May Enable Establishment of Nisin  Immunity Prior to Induction of the Operon via Signal Transduction.  J. Bacteriol. 188: 8496-8503
3. Overkamp, W. et al. (2013) Benchmarking Various Green Fluorescent  Protein Variants in Bacillus subtilis, Streptococcus pneumoniae, and Lactococcus lactis for Live Cell Imaging. Appl. Environ. Microbiol. 79: 6481-6490
Notebook / Biobricks
Home
Project
Overview
Detection
Secretion
The bandage
Modeling
Why modeling?
Modeling of the bandage
Modeling of L. lactis
Modeling experiments
Policy and practice
Overview
Future perspectives
Martini Hospital
Sharing our idea
Media Coverage
Survey
Education
Synthetic biology
Lactococcus lactis
Burn wounds
Antibiotic resistance
Team
Team members
Instructors and supervisors
Photogallery
iGEM team profile
Parts
Biobricks
iGEM parts page
Characterization
Notebook
Detection
Secretion
The bandage
Biobricks
Protocols
Safety
Safety
Art and design
Art
Awards
Badges
Checklist
Acknowledgements
Acknowledgements
Retrieved from " http://2014.igem.org/Team:Groningen/Notebook/Biobricks "
Recent changes
What links here
Related changes
Special pages
My preferences
Printable version
Permanent link
Privacy policy
Disclaimers
