<HTML xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en" dir="ltr">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki  ltr ns-0 ns-subject page-Team_Groningen_Template_MODULE_Notebook_toolbox"><DIV id="globalWrapper"><DIV id="top-section"><DIV id="p-logo"><A href="/Main_Page" title="Main Page">&quot;
	    </A></DIV><DIV id="menubar" class="left-menu noprint"><UL><LI class="selected"><A href="/Team:Groningen/Template/MODULE/Notebook/toolbox">Page               </A></LI><LI class="new"><A href="/wiki/index.php?title=Talk:Team:Groningen/Template/MODULE/Notebook/toolbox&amp;action=edit&amp;redlink=1">Discussion               </A></LI><LI><A href="/wiki/index.php?title=Team:Groningen/Template/MODULE/Notebook/toolbox&amp;action=edit">View source               </A></LI><LI><A href="/wiki/index.php?title=Team:Groningen/Template/MODULE/Notebook/toolbox&amp;action=history">History               </A></LI><LI style="color:white;cursor:default">teams</LI></UL></DIV><DIV class="right-menu noprint" id="menubar"><UL><LI id="pt-login"><A href="/wiki/index.php?title=Special:UserLogin&amp;returnto=Team:Groningen/Template/MODULE/Notebook/toolbox" title="You are encouraged to log in; however, it is not mandatory [o]" accesskey="o">Log in</A></LI></UL></DIV><DIV id="search-controls" class="noprint"><FORM action="/Special:Search" id="searchform"> </FORM></DIV></DIV><DIV id="content"><H1 class="firstHeading">Team:Groningen/Template/MODULE/Notebook/toolbox</H1><DIV id="bodyContent"><H3 id="siteSub" class="noprint">From 2014.igem.org</H3><DIV class="entry gridcell grid"><DIV class="module breadcrummodule gridcell"><DIV class="content"><DIV class="wrapper"><SPAN class="tail">Notebook</SPAN><SPAN class="seperator">&gt;</SPAN><SPAN class="head">Toolbox</SPAN><DIV class="hspacer"> </DIV></DIV><DIV class="hspacer"> </DIV></DIV><DIV class="hspacer"> </DIV></DIV><DIV class="module contentmodule gridcell"><DIV class="content"><DIV class="wrapper"><DIV class="title">

7 July - 13 July

</DIV><DIV class="hspacer"> </DIV><DIV class="text">
Primers were designed for making BioBricks out of the separate genes from 
the nisin operon. For this, the sequence of the transposon
Tn5307 was used, a transposon that contains the nisin operon, and of 
which the sequence is known.<SUP>1</SUP> This sequence is close to the sequence
of the transposon Tn5276, the transposon which will form the template
for making the new BioBricks. The primers for making a BioBrick out of
PNisI with a RBS were based on a sequence found in a research paper that
documented the identification of this promoter.<SUP>2</SUP> The list of
primersequences can be found in the first table at the bottom of this page.
It was decided to combine the genes NisR
and NisK as one BioBrick and NisF, NisE and NisG as one BioBrick, because
these genes are related in function or work together.
</DIV><DIV class="hspacer"> </DIV><DIV class="text">
There were also primers designed for making a BioBrick out of the sfGFP(Bs), the primersequences can be found at second table at the bottom of this page. This sfGFP
was originally optimized for <I>Bacillus subtilis</I>. When it was
tested in <I>Lactococcus lactis</I>, the sfGFP(Bs) was shown to perform
really well in <I>L. lactis</I> as well.<SUP>3</SUP></DIV><DIV class="hspacer"> </DIV><DIV class="hspacer"> </DIV></DIV><DIV class="hspacer"> </DIV></DIV><DIV class="hspacer"> </DIV></DIV><DIV class="module contentmodule gridcell"><DIV class="content"><DIV class="wrapper"><DIV class="title">
14 July - 20 July

</DIV><DIV class="hspacer"> </DIV><DIV class="text">
A collection of constitutive promoters was also desirable for the toolbox.
We decided to use the CP promoter collection and test it in 
<I>Lactococcus lactis</I>, that was BioBricked by
the Uppsala iGEM team of 2013. The CP promoters are registered as <A href="http://parts.igem.org/Part:BBa_K1033219"> CP1</A>, <A href="http://parts.igem.org/Part:BBa_K1033220">CP8</A>, <A href="http://parts.igem.org/Part:BBa_K1033221">CP11</A>, <A href="http://parts.igem.org/Part:BBa_K1033222">CP29</A>,
<A href="http://parts.igem.org/Part:BBa_K1033223">CP30</A>, <A href="http://parts.igem.org/Part:BBa_K1033224">CP41</A> and <A href="http://parts.igem.org/Part:BBa_K1033225">CP44</A>. These parts were
ordered from iGEM HQ, except for CP8 as for this promoter the sequencing
was inconsistent. The promoters were sent by iGEM HQ using 
<I>Escherichia coli</I> containing pSB1C3 with the promoter as insert.
<I>E. coli</I> was grown and the plasmids with the promoter were
isolated.
</DIV><DIV class="hspacer"> </DIV><DIV class="hspacer"> </DIV></DIV><DIV class="hspacer"> </DIV></DIV><DIV class="hspacer"> </DIV></DIV><DIV class="module contentmodule gridcell"><DIV class="content"><DIV class="wrapper"><DIV class="title">

21 July - 27 July

</DIV><DIV class="hspacer"> </DIV><DIV class="text">
BioBricking the genes NisT, NisI, NisP and the combined genes NisF, 
NisE and NisG gave problems with illegal restriction sites.
The illegal restrictionsite in NisI could be removed using the reverse
primer as listed above. This primer contains a mutation needed to
remove the illegal restriction site.
The other parts gave more problems, as these all contained multiple
restriction sites. Therefore, it was decided to use Gibson assembly
with primers containing mutations to remove the restriction site.
By using Gibson assembly, the genes could also be constructed in
pSB1C3 in one go. The primersequences can be found at the third table at the bottom of this page. 
</DIV><DIV class="hspacer"> </DIV><DIV class="hspacer"> </DIV></DIV><DIV class="hspacer"> </DIV></DIV><DIV class="hspacer"> </DIV></DIV><DIV class="module contentmodule gridcell"><DIV class="content"><DIV class="wrapper"><DIV class="title">

28 July - 3 August

</DIV><DIV class="hspacer"> </DIV><DIV class="text">
The CP promoters in pSB1C3, that were isolated in the week
of 14 - 20 July, were tested on their insert size. For this, a PCR
was done on the plasmid using the primers <A href="http://parts.igem.org/Part:BBa_G00100">VF2</A> and <A href="http://parts.igem.org/Part:BBa_G00101">VR</A>.
 The insert size corresponded to the expected size.
</DIV><DIV class="hspacer"> </DIV><DIV class="text">
A touchdown PCR was done on the genes of the nisin operon and the 
sfGFP(Bs) gene. In this PCR the annealing temperature dropped from <SPAN>55 °C</SPAN> to 
45 °C, lowering the temperature with 1 °C each cycle. This 
unfortunately resulted in a PCR program of just 10 cycles, instead of
the intended 30. The remaining steps of the program were finished the
next morning. Only the genes NisA, PNisI and sfGFP(Bs) were amplified
this way, see figure 1. Therefore, the PCR 									
was repeated, this time with a complete cycle. No additional genes
were amplified this way.
</DIV><DIV class="hspacer"> </DIV><DIV class="figure"><DIV class="hspacer"> </DIV><DIV class="caption"><SPAN class="title">Figure 1:</SPAN>

Amplification of PNisI with RBS, NisA and sfGFP(Bs), with an added prefix and suffix.

</DIV><DIV class="hspacer"> </DIV></DIV><DIV class="hspacer"> </DIV><DIV class="text">
Another attempt was made for amplification of the remaining genes.
This time, the most ideal annealing temperature for each gene was
used by using a gradient PCR and placing the tubes at the optimal
temperature. This still did not amplify the remaining genes. Also, a
PCR with a general annealing temperature of 50 °C was done. This also
did not amplify the remaining genes.
</DIV><DIV class="hspacer"> </DIV><DIV class="text">
The genes that were amplified in the first PCR (NisA, PNisI and sfGFP(Bs))
were purified using the GeneJET PCR Purification Kit from Thermo
Scientific. The purified products were digested with the enzymes EcoRI 
and PstI, using 2 μl of the product. These purified and restricted
products were loaded on gel, see figure 2. It was then discovered that the genes were 
barely visible on gel. The restricted genes were purified with the
GeneJET PCR Purification Kit and the concentration was measured with the
NanoDrop 1000. The clean, restricted products were too low in 
concentration to be suitable for ligation.
</DIV><DIV class="hspacer"> </DIV><DIV class="figure"><DIV class="hspacer"> </DIV><DIV class="caption"><SPAN class="title">Figure 2:</SPAN>

At the left: purified NisA, PNisI and sfGFP(Bs). At the right: NisA and sfGFP(Bs) after restriction with EcoRI and PstI.

</DIV><DIV class="hspacer"> </DIV></DIV><DIV class="hspacer"> </DIV><DIV class="hspacer"> </DIV></DIV><DIV class="hspacer"> </DIV></DIV><DIV class="hspacer"> </DIV></DIV><DIV class="module contentmodule gridcell"><DIV class="content"><DIV class="wrapper"><DIV class="title">

4 August - 10 August

</DIV><DIV class="hspacer"> </DIV><DIV class="text">
Because the PCR products of NisA, PNisI and sfGFP(Bs) were lost during
the purification and restriction in the week of 28 July - 3 August, the
PCR was repeated for this genes, together with the genes that could not
be amplified yet. This time a PCR was used that did not lower in 
temperature, like the touchdown PCR, but that increased in temperature 
with each step. This way it was hoped to get over the huge gap between
the annealing temperature of the primer in the first cycle and the
annealing temperature of the primer when the flap of the primer can also 
anneal to the first PCR products. The temperature was set to increase from
40 °C to 60 °C in 20 cycles. Then, an additional 20 cycles were
done at 65 °C. The PCR was performed under standard conditions as
was done before, together with a series of PCR that contained GC buffer
(supplied with the Phusion DNA polymerase by Thermo Scientific), and
a series of PCR with GC buffer and 1.5% DMSO. This time, with the help
of Lisa, the PCR was finally successful for all genes, see figure 3. 							
</DIV><DIV class="hspacer"> </DIV><DIV class="figure"><DIV class="hspacer"> </DIV><DIV class="caption"><SPAN class="title">Figure 3:</SPAN>

PCR products of all the genes of the nisin operon with BioBrick prefix and suffix.

</DIV><DIV class="hspacer"> </DIV></DIV><DIV class="hspacer"> </DIV><DIV class="hspacer"> </DIV></DIV><DIV class="hspacer"> </DIV></DIV><DIV class="hspacer"> </DIV></DIV></DIV><DIV class="entry grid gridcell last"><DIV class="module imagemodule gridcell"><DIV class="content"><DIV class="wrapper"><DIV class="image"><DIV class="hspacer"> </DIV></DIV><DIV class="hspacer"> </DIV><DIV class="hspacer"> </DIV><DIV class="text"><P>The toolbox is supposed to make it possible to use <I>Lactococcus lactis</I> as a chassis for iGEM constructs in the future. For the toolbox, some basic BioBricks are constructed that form the basis of larger constructs.
</P></DIV><DIV class="hspacer"> </DIV></DIV><DIV class="hspacer"> </DIV></DIV><DIV class="hspacer"> </DIV></DIV><DIV class="hspacer"> </DIV><DIV class="module contentmodule gridcell"><DIV class="content"><DIV class="wrapper"><DIV class="title">

11 August - 17 August

</DIV><DIV class="hspacer"> </DIV><DIV class="text">
The PCR on the BioBrick of the combined genes NisR and NisK was repeated
because the yield of this product was very low after the PCR in the week
of 4 - 10 August. The mixture of the previous PCR reaction was used as
the template. The PCR was performed under standard conditions, with an
annealing temperature of 64 °C and 1 μl, 2μl and 3 μl template.
No product was obtained with this PCR. So the PCR was once again repeated,
this time using diluted template. The mixture was diluted 50x, 300x,
1500x and 15000x. The PCR was repeated using 1 μl and 2 μl of each
of the dilutions. This way, product was obtained for all reactions.			
</DIV><DIV class="hspacer"> </DIV><DIV class="text">
The PCR products of the genes NisA, NisB, NisC, NisRK, PNisI and sfGFP(Bs)
were purified using the GeneJET purification kit. Then 1 μg of each
purified product was restricted with EcoRI and PstI, together with 1 μg
of the pSB1C3 plasmid. The restriction enzymes were then inactivated by
heating the samples at 80 °C for 20 minutes. Ligated 6 μl of the
restricted PCR products to 2 μl restricted pSB1C3. Inactivated the
ligase by incubating at 65 °C for 10 minutes. Mixed 5 μl of the
ligation mixture with 25 μl electrocompetent <I>Escherichia coli</I> 
DH5α. The transformants were grown on LB with 10 μg/ml 
chloramphenicol, see figure 4. The white colonies were tested on insertsize with a
colony PCR with the <A href="http://parts.igem.org/Part:BBa_G00100">VF2</A> and <A href="http://parts.igem.org/Part:BBa_G00101">VR</A> primers.			

For every gene transformants were found that contained pSB1C3 with a
correctly sized insert, except for NisB and NisRK.                                      
</DIV><DIV class="hspacer"> </DIV><DIV class="figure"><DIV class="hspacer"> </DIV><DIV class="caption"><SPAN class="title">Figure 4:</SPAN><I>E. coli</I> containing pSB1C3 with a mRFP construct (pink colonies) and with another insert than mRFP, possibly a correct BioBrick (white colonies).

</DIV><DIV class="hspacer"> </DIV></DIV><DIV class="hspacer"> </DIV><DIV class="text">
The <I>E. coli</I> that contained the pSB1C3 plasmid with a correctly 
sized insert were grown as a liquid culture and the plasmid was isolated
using the GeneJET Plasmid Miniprep Kit.
</DIV><DIV class="hspacer"> </DIV><DIV class="hspacer"> </DIV></DIV><DIV class="hspacer"> </DIV></DIV><DIV class="hspacer"> </DIV></DIV><DIV class="module contentmodule gridcell"><DIV class="content"><DIV class="wrapper"><DIV class="title">
18 August - 31 August

</DIV><DIV class="hspacer"> </DIV><DIV class="text">
The plasmids containing the new BioBricks NisA, NisC, PNisI and sfGFP(Bs) 
that were isolated in the week of 11 - 17 August were concentrated
with a SpeedVac and then sent for sequencing with the primers <A href="http://parts.igem.org/Part:BBa_G00100">VF2</A> 
 and <A href="http://parts.igem.org/Part:BBa_G00101">VR</A>. The sequence of each BioBrick was
confirmed.
</DIV><DIV class="hspacer"> </DIV><DIV class="hspacer"> </DIV></DIV><DIV class="hspacer"> </DIV></DIV><DIV class="hspacer"> </DIV></DIV><DIV class="module contentmodule gridcell"><DIV class="content"><DIV class="wrapper"><DIV class="title">

1 September - 7 September

</DIV><DIV class="hspacer"> </DIV><DIV class="text">
Colony PCR on more transformants containing pSB1C3 with NisB and NisRK
was repeated, using <A href="http://parts.igem.org/Part:BBa_G00100">VF2</A> and
<A href="http://parts.igem.org/Part:BBa_G00101">VR</A>. This time, a 
correct insert size wasbfound for a NisRK transformant.
Because there was still no correct insert size found for a NisB 
transformant, the remaining NisB colonies were tested for a correct insert size.
There were no positive transformants found.
</DIV><DIV class="hspacer"> </DIV><DIV class="hspacer"> </DIV></DIV><DIV class="hspacer"> </DIV></DIV><DIV class="hspacer"> </DIV></DIV><DIV class="module contentmodule gridcell"><DIV class="content"><DIV class="wrapper"><DIV class="title">

8 September - 14 September

</DIV><DIV class="hspacer"> </DIV><DIV class="text">
A PCR was done on all the genes that contained illegal restriction sites:
NisI, NisT, NisP and NisFEG. This should generate multiple PCR fragments
for each gene, that will be assembled again using Gibson assembly.
The PCR was split into two seperate reactions because of the large
differences between the size of the different fragments. Fragments
were sorted on &lt;750 kb and &gt;750 kb. The experiment was not continued
because of time limits.
</DIV><DIV class="hspacer"> </DIV><DIV class="hspacer"> </DIV></DIV><DIV class="hspacer"> </DIV></DIV><DIV class="hspacer"> </DIV></DIV><DIV class="module contentmodule gridcell"><DIV class="content"><DIV class="wrapper"><DIV class="title">

14 September - 21 September

</DIV><DIV class="hspacer"> </DIV><DIV class="text">
Because of the constant low concentrations of the plasmid pSB1C3 with the
new BioBricks, it was decided to transform the isolated pSB1C3 with the
new BioBricks (NisA, NisC, NisRK, PNisI and sfGFP(Bs)) again in 				<I>Escherichia coli</I> DH5α. 
The pSB1C3 plasmids containing the new BioBricks were again isolated.
For each transformation, two seperate cultures were grown in LB with 10 μg/ml
chloramphenicol. A miniprep was done on the cultures and the insert size
was checked with colony PCR, using <A href="http://parts.igem.org/Part:BBa_G00100">VF2</A> and <A href="http://parts.igem.org/Part:BBa_G00101">VR</A> as
primers. In addition, a restriction analysis was done using EcoRI and PstI.
Cultures that showed a correct insert size were sent for sequencing.
</DIV><DIV class="hspacer"> </DIV><DIV class="hspacer"> </DIV></DIV><DIV class="hspacer"> </DIV></DIV><DIV class="hspacer"> </DIV></DIV><DIV class="module contentmodule gridcell"><DIV class="content"><DIV class="wrapper"><DIV class="title">

22 September - 28 September

</DIV><DIV class="hspacer"> </DIV><DIV class="text">
The sequences of NisA, NisC, NisRK and sfGFP(Bs) were analyzed. The sequences of NisA and sfGFP(Bs) were completely correct. The sequence of NisC showed that a mutation occured at residue 185, where an AAT was changed to an AAA, changing an asparagine to a lysine. The changed amino acid was on the outside of the protein, so may not have an effect on the function of NisC. The sequence of NisRK showed two mutations. At residue 199 of NisR a GAA was changed to a GAG, not changing the amino acid at this position. At residue 300 of NisK an ACA was changed to an ATA, changing a threonine to an isoleucine. The PNisI with RBS showed a large mutation. At position 508, 18 basepairs were missing. Therefore it was decided to not send PNisI to iGEM HQ anymore.
</DIV><DIV class="hspacer"> </DIV><DIV class="text">
A PCR was done to generate fragments for assembling genes that had
illegal restriction sites. Every fragment was succesfully amplified.
The fragments were purified using the GeneJET PCR clean up kit.
Not all fragments have a sufficient concentration or purity, so only the fragments for
the combined BioBrick of NisF, NisE and NisG was made using Gibson
assembly. The transformants that contained pSB1C3 with NisFEG were
tested using colony PCR. No positive transformants were found.
</DIV><DIV class="hspacer"> </DIV><DIV class="hspacer"> </DIV></DIV><DIV class="hspacer"> </DIV></DIV><DIV class="hspacer"> </DIV></DIV><DIV class="module contentmodule gridcell"><DIV class="content"><DIV class="wrapper"><DIV class="title">

29 September - 5 October

</DIV><DIV class="hspacer"> </DIV><DIV class="text">
Completed BioBricks were sent to iGEM HQ in a pSB1C3 backbone: <A href="http://parts.igem.org/Part:BBa_K1365000">NisA</A>, <A href="http://parts.igem.org/Part:BBa_K1365003">NisC</A>, 
<A href="http://parts.igem.org/Part:BBa_K1365006">NisRK</A> and <A href="http://parts.igem.org/Part:BBa_K1365020">sfGFP(Bs)</A>.
</DIV><DIV class="hspacer"> </DIV><DIV class="hspacer"> </DIV></DIV><DIV class="hspacer"> </DIV></DIV><DIV class="hspacer"> </DIV></DIV><DIV class="module referencemodule gridcell"><DIV class="content"><DIV class="wrapper"><DIV class="title">References</DIV><DIV class="hspacer"> </DIV><DIV class="reference">
1. Trmčić, A. et al. (2011) Complete nisin A gene cluster from <I>Lactococcus 
lactis</I> M78 (HM219853) – obtaining the nucleic acid sequence and 
comparing it to other published nisin sequences. <I>Genes Genom.</I> 33: 217-221
</DIV><DIV class="hspacer"> </DIV><DIV class="reference">
2. Li, H. and O´Sullivan, D.J. (2006) Identification of a NisI promoter 
within the NisABCTIP operon that may enable establishment of nisin 
immunity prior to induction of the operon via signal transduction. 
<I>J. Bacteriol.</I> 188: 8496-8503
</DIV><DIV class="hspacer"> </DIV><DIV class="reference">
3. Overkamp, W. et al. (2013) Benchmarking various green fluorescent 
protein variants in <I>Bacillus subtilis</I>, <I>Streptococcus pneumoniae</I>, and 
<I>Lactococcus lactis</I> for live cell imaging. <I>Appl. Environ. Microbiol.</I> 79:
6481-6490
</DIV><DIV class="hspacer"> </DIV></DIV><DIV class="hspacer"> </DIV></DIV><DIV class="hspacer"> </DIV></DIV></DIV><DIV class="entry gridcell grid spanall"><DIV class="module tablemodule gridcell"><DIV class="content"><DIV class="wrapper"><DIV class="title">

Primers for making BioBricks out of the nisin operon

</DIV><DIV class="hspacer"> </DIV><TABLE class="table"><TBODY><TR><TH>Primer</TH><TH>Sequence</TH></TR><TR><TD>NisA forward</TD><TD>GTTTCTTCGAATTCGCGGCCGCTTCTAGATGAGTACAAAAGATTTTAA</TD></TR><TR><TD>NisA reverse</TD><TD>GTTTCTTCCTGCAGCGGCCGCTACTAGTATTATTATTTGCTTACGTGAA</TD></TR><TR><TD>NisB forward</TD><TD>GTTTCTTCGAATTCGCGGCCGCTTCTAGATGATAAAAAGTTCATTTAA</TD></TR><TR><TD>NisB reverse</TD><TD>GTTTCTTCCTGCAGCGGCCGCTACTAGTATTATCATTTCATGTATTCTT</TD></TR><TR><TD>NisT forward</TD><TD>GTTTCTTCGAATTCGCGGCCGCTTCTAGATGGATGAAGTGAAAGAATTTACATCA</TD></TR><TR><TD>NisT reverse</TD><TD>GTTTCTTCCTGCAGCGGCCGCTACTAGTATTATTATTCATCATTATCCT</TD></TR><TR><TD>NisC forward</TD><TD>GTTTCTTCGAATTCGCGGCCGCTTCTAGATGAATAAAAAAAATATAAA</TD></TR><TR><TD>NisC reverse</TD><TD>GTTTCTTCCTGCAGCGGCCGCTACTAGTATTATCATTTCCTCTTCCCTC</TD></TR><TR><TD>NisI forward</TD><TD>GTTTCTTCGAATTCGCGGCCGCTTCTAGATGAGAAGATATTTAATACT</TD></TR><TR><TD>NisI reverse</TD><TD>GTTTCTTCCTGCAGCGGCCGCTACTAGTATTACTAATTTCCTACCTTCG</TD></TR><TR><TD>NisP forward</TD><TD>GTTTCTTCGAATTCGCGGCCGCTTCTAGATGAAAAAAATACTAGGTTT</TD></TR><TR><TD>NisP reverse</TD><TD>GTTTCTTCCTGCAGCGGCCGCTACTAGTATTATCAATTTTTAGTCTTTC</TD></TR><TR><TD>NisRK forward</TD><TD>GTTTCTTCGAATTCGCGGCCGCTTCTAGGTGTATAAAATTTTAATAGT</TD></TR><TR><TD>NisRK reverse</TD><TD>GTTTCTTCCTGCAGCGGCCGCTACTAGTATTATTACTTTTTTATTTTTA</TD></TR><TR><TD>NisFEG forward</TD><TD>GTTTCTTCGAATTCGCGGCCGCTTCTAGATGCAGGTAAAAATTCAAAA</TD></TR><TR><TD>NisFEG reverse</TD><TD>GTTTCTTCCTGCAGCGGCCGCTACTAGTATTATTATCTAATCTTTTTTT</TD></TR><TR><TD>PNisI+RBS forward</TD><TD>GTTTCTTCGAATTCGCGGCCGCTTCTAGAGAGCACTGGATAATGACTATT</TD></TR><TR><TD>PNisI+RBS reverse</TD><TD>TACTAGTAGCGGCCGCTGCAGGAAGAAACTTCCTCTTCCCTCCTTTCAA</TD></TR></TBODY></TABLE><DIV class="hspacer"> </DIV><DIV class="hspacer"> </DIV></DIV><DIV class="hspacer"> </DIV></DIV><DIV class="hspacer"> </DIV></DIV></DIV><DIV class="entry gridcell grid spanall"><DIV class="module tablemodule gridcell"><DIV class="content"><DIV class="wrapper"><DIV class="title">

Primers for making the new sfGFP(Bs) BioBrick

</DIV><DIV class="hspacer"> </DIV><TABLE class="table"><TBODY><TR><TH>Primer</TH><TH>Sequence</TH></TR><TR><TD>sfGFP(Bs) forward</TD><TD>GTTTCTTCGAATTCGCGGCCGCTTCTAGATGTCAAAAGGAGAAGAGCT</TD></TR><TR><TD>sfGFP(Bs) reverse</TD><TD>GTTTCTTCCTGCAGCGGCCGCTACTAGTATTATTACTTATAAAGCTCAT</TD></TR></TBODY></TABLE><DIV class="hspacer"> </DIV><DIV class="hspacer"> </DIV></DIV><DIV class="hspacer"> </DIV></DIV><DIV class="hspacer"> </DIV></DIV></DIV><DIV class="entry gridcell grid spanall"><DIV class="module tablemodule gridcell"><DIV class="content"><DIV class="wrapper"><DIV class="title">

Primers for making BioBricks out of genes with illegal restriction sites

</DIV><DIV class="hspacer"> </DIV><DIV class="caption">
The primers are denoted with the position of the 
illegal restriction site that they should remove. Illegal EcoRI sites 
are abbreviated with an E, XbaI sites with an X, SpeI site with an S and 
PstI sites with a P.
</DIV><DIV class="hspacer"> </DIV><TABLE class="table"><TBODY><TR><TH>Gene</TH><TH>Primer</TH><TH>Sequence</TH></TR><TR><TD>NisT</TD><TD>Prefix forward</TD><TD>GTTTCTTCGAATTCGCGGCCGCTTCTAGATGGATGAAGTGAAAGAATTTACATCA</TD></TR><TR><TD> </TD><TD>S980 reverse</TD><TD>TTGTTCCATAAACAAACTTGTATTATAAATGATGTAAATA</TD></TR><TR><TD> </TD><TD>S980 forward</TD><TD>TATTTACATCATTTATAATACAAGTTTGTTTATGGAACAA</TD></TR><TR><TD> </TD><TD>S1210 reverse</TD><TD>CCATAGTTGGTTGATATAATCCTGAAATTATCTTTACTTGTGTAC</TD></TR><TR><TD> </TD><TD>S1210 forward</TD><TD>GTACACAAGTAAAGATAATTTCAGGATTATATCAACCAACTATGG</TD></TR><TR><TD> </TD><TD>Suffix reverse</TD><TD>GTTTCTTCCTGCAGCGGCCGCTACTAGTATTATTATTCATCATTATCCT</TD></TR><TR><TD> </TD><TD>Suffix forward</TD><TD>ATGAATAATAATACTAGTAGCGGCCGCTGCAGGAAGAAACAAAAGGGCAA</TD></TR><TR><TD> </TD><TD>Prefix reverse</TD><TD>CACTTCATCCATCTAGAAGCGGCCGCGAATTCGAAGAAACATCCTTAGCG</TD></TR><TR><TD>NisP</TD><TD>Prefix forward</TD><TD>GTTTCTTCGAATTCGCGGCCGCTTCTAGGTGAAAAAAATACTAGGTTT</TD></TR><TR><TD> </TD><TD>P225 reverse</TD><TD>AGTCCTTGTTGTCGATTTTACTGCTGGTGACTGCGCCCCT</TD></TR><TR><TD> </TD><TD>P225 forward</TD><TD>AGGGGCGCAGTCACCAGCAGTAAAATCGACAACAAGGACT</TD></TR><TR><TD> </TD><TD>E347 reverse</TD><TD>AACGCTATCTCTCTTACTAAATTCAGAACTAACTTGAGTT</TD></TR><TR><TD> </TD><TD>E347 forward</TD><TD>AACTCAAGTTAGTTCTGAATTTAGTAAGAGAGATAGCGTT</TD></TR><TR><TD> </TD><TD>E1657 reverse</TD><TD>CAATACTCTATTCCCATTAACTTCTGGACTATTCATTAGA</TD></TR><TR><TD> </TD><TD>E1657 forward</TD><TD>TCTAATGAATAGTCCAGAAGTTAATGGGAATAGAGTATTG</TD></TR><TR><TD> </TD><TD>Suffix reverse</TD><TD>GTTTCTTCCTGCAGCGGCCGCTACTAGTATTATCAATTTTTAGTCTTTC</TD></TR><TR><TD> </TD><TD>Suffix forward</TD><TD>AAATTGATAATACTAGTAGCGGCCGCTGCAGGAAGAAACAAAAGGGCAA</TD></TR><TR><TD> </TD><TD>Prefix reverse</TD><TD>CCTAGTATTTTTTTCATCTAGAAGCGGCCGCGAATTCGAAGAAACATCCTTAGCG</TD></TR><TR><TD>NisFEG</TD><TD>Prefix forward</TD><TD>GTTTCTTCGAATTCGCGGCCGCTTCTAGATGCAGGTAAAAATTCAAAA</TD></TR><TR><TD> </TD><TD>P186 reverse</TD><TD>TCCAGTATCAGCAGAAATTAAACCAAACAAAATTTTCATC</TD></TR><TR><TD> </TD><TD>P186 forward</TD><TD>GATGAAAATTTTGTTTGGTTTAATTTCTGCTGATACTGGA</TD></TR><TR><TD> </TD><TD>E336/X349 reverse</TD><TD>CCTGTTTCTGCCAAACCAATCACTTCTAGTGTTTCATGTATTCTCTTA</TD></TR><TR><TD> </TD><TD>E336/X349 forward</TD><TD>TAAGAGAATACATGAAACACTAGAAGTGATTGGTTTGGCAGAAACAGG</TD></TR><TR><TD> </TD><TD>Suffix reverse</TD><TD>GTTTCTTCCTGCAGCGGCCGCTACTAGTATTATTATCTAATCTTTTTTT</TD></TR><TR><TD> </TD><TD>Suffix forward</TD><TD>TAGATAATAATACTAGTAGCGGCCGCTGCAGGAAGAAACAAAAGGGCAA</TD></TR><TR><TD> </TD><TD>Prefix reverse</TD><TD>TTTTACCTGCATCTAGAAGCGGCCGCGAATTCGAAGAAACATCCTTAGCG</TD></TR></TBODY></TABLE><DIV class="hspacer"> </DIV><DIV class="hspacer"> </DIV></DIV><DIV class="hspacer"> </DIV></DIV><DIV class="hspacer"> </DIV></DIV></DIV><DIV class="printfooter">
Retrieved from &quot;<A href="http://2014.igem.org/Team:Groningen/Template/MODULE/Notebook/toolbox">http://2014.igem.org/Team:Groningen/Template/MODULE/Notebook/toolbox</A>&quot;</DIV></DIV></DIV><DIV id="footer-box" class="noprint"><DIV id="footer"><UL id="f-list"><LI id="t-recentchanges"><A href="/Special:RecentChanges" title="Recent changes">Recent changes</A></LI><LI id="t-whatlinkshere"><A href="/Special:WhatLinksHere/Team:Groningen/Template/MODULE/Notebook/toolbox" title="List of all wiki pages that link here [j]" accesskey="j">What links here</A></LI><LI id="t-recentchangeslinked"><A href="/Special:RecentChangesLinked/Team:Groningen/Template/MODULE/Notebook/toolbox" title="Recent changes in pages linked from this page [k]" accesskey="k">Related changes</A></LI><LI id="t-specialpages"><A href="/Special:SpecialPages" title="List of all special pages [q]" accesskey="q">Special pages</A></LI><LI><A href="/Special:Preferences">My preferences</A></LI></UL></DIV><DIV id="footer"><UL id="f-list"><LI id="t-print"><A href="/wiki/index.php?title=Team:Groningen/Template/MODULE/Notebook/toolbox&amp;printable=yes" title="Printable version of this page [p]" accesskey="p">Printable version</A></LI><LI id="t-permalink"><A href="/wiki/index.php?title=Team:Groningen/Template/MODULE/Notebook/toolbox&amp;oldid=334904" title="Permanent link to this revision of the page">Permanent link</A></LI><LI id="privacy"><A href="/2014.igem.org:Privacy_policy" title="2014.igem.org:Privacy policy">Privacy policy</A></LI><LI id="disclaimer"><A href="/2014.igem.org:General_disclaimer" title="2014.igem.org:General disclaimer">Disclaimers</A></LI></UL></DIV></DIV></DIV></BODY></HTML>