<HTML xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en" dir="ltr">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki  ltr ns-0 ns-subject page-Team_Groningen_Notebook_Biobricks"><DIV id="globalWrapper"><DIV id="top-section"><DIV id="p-logo"><A href="/Main_Page" title="Main Page">&quot;
	    </A></DIV><DIV id="menubar" class="left-menu noprint"><UL><LI class="selected"><A href="/Team:Groningen/Notebook/Biobricks">Page               </A></LI><LI class="new"><A href="/wiki/index.php?title=Talk:Team:Groningen/Notebook/Biobricks&amp;action=edit&amp;redlink=1">Discussion               </A></LI><LI><A href="/wiki/index.php?title=Team:Groningen/Notebook/Biobricks&amp;action=edit">View source               </A></LI><LI><A href="/wiki/index.php?title=Team:Groningen/Notebook/Biobricks&amp;action=history">History               </A></LI><LI style="color:white;cursor:default">teams</LI></UL></DIV><DIV class="right-menu noprint" id="menubar"><UL><LI id="pt-login"><A href="/wiki/index.php?title=Special:UserLogin&amp;returnto=Team:Groningen/Notebook/Biobricks" title="You are encouraged to log in; however, it is not mandatory [o]" accesskey="o">Log in</A></LI></UL></DIV><DIV id="search-controls" class="noprint"><FORM action="/Special:Search" id="searchform"> </FORM></DIV></DIV><DIV id="content"><H1 class="firstHeading">Team:Groningen/Notebook/Biobricks</H1><DIV id="bodyContent"><H3 id="siteSub" class="noprint">From 2014.igem.org</H3><DIV id="ourentirewebsite"><DIV id="scrollablecontent"><DIV id="contentbox"><SPAN id="content_home" class="visibilitytoggle content_home"><B>7 – 13 Juli</B>
Primers were designed for making BioBricks out of the separate genes from 
the nisin operon. For this, the sequence of the transposon
Tn5307 was used, a transposon that contains the nisin operon, and of 
which the sequence is known.<SUP>1</SUP> This sequence is close to the sequence
of the transposon Tn5276, the transposon which will form the template
for making the new BioBricks. The primers for making a BioBrick out of
PNisI with a RBS were based on a sequence found in a research paper that
documented the identification of this promoter.<SUP>2</SUP>
It was decided to combine the genes NisR
and NisK as one BioBrick and NisF, NisE and NisG as one BioBrick, because
these genes are related in function or work together. 
<TABLE border="1" style="width:90%"><TBODY><TR><TH width="25%">Primer</TH><TH>Sequence</TH></TR><TR><TD>NisA forward</TD><TD>GTTTCTTCGAATTCGCGGCCGCTTCTAGATGAGTACAAAAGATTTTAA</TD></TR><TR><TD>NisA reverse</TD><TD>GTTTCTTCCTGCAGCGGCCGCTACTAGTATTATTATTTGCTTACGTGAA</TD></TR><TR><TD>NisB forward</TD><TD>GTTTCTTCGAATTCGCGGCCGCTTCTAGATGATAAAAAGTTCATTTAA</TD></TR><TR><TD>NisB reverse</TD><TD>GTTTCTTCCTGCAGCGGCCGCTACTAGTATTATCATTTCATGTATTCTT</TD></TR><TR><TD>NisT forward</TD><TD>GTTTCTTCGAATTCGCGGCCGCTTCTAGATGGATGAAGTGAAAGAATTTACATCA</TD></TR><TR><TD>NisT reverse</TD><TD>GTTTCTTCCTGCAGCGGCCGCTACTAGTATTATTATTCATCATTATCCT</TD></TR><TR><TD>NisC forward</TD><TD>GTTTCTTCGAATTCGCGGCCGCTTCTAGATGAATAAAAAAAATATAAA</TD></TR><TR><TD>NisC reverse</TD><TD>GTTTCTTCCTGCAGCGGCCGCTACTAGTATTATCATTTCCTCTTCCCTC</TD></TR><TR><TD>NisI forward</TD><TD>GTTTCTTCGAATTCGCGGCCGCTTCTAGATGAGAAGATATTTAATACT</TD></TR><TR><TD>NisI reverse</TD><TD>GTTTCTTCCTGCAGCGGCCGCTACTAGTATTACTAATTTCCTACCTTCG</TD></TR><TR><TD>NisP forward</TD><TD>GTTTCTTCGAATTCGCGGCCGCTTCTAGATGAAAAAAATACTAGGTTT</TD></TR><TR><TD>NisP reverse</TD><TD>GTTTCTTCCTGCAGCGGCCGCTACTAGTATTATCAATTTTTAGTCTTTC</TD></TR><TR><TD>NisRK forward</TD><TD>GTTTCTTCGAATTCGCGGCCGCTTCTAGGTGTATAAAATTTTAATAGT</TD></TR><TR><TD>NisRK reverse</TD><TD>GTTTCTTCCTGCAGCGGCCGCTACTAGTATTATTACTTTTTTATTTTTA</TD></TR><TR><TD>NisFEG forward</TD><TD>GTTTCTTCGAATTCGCGGCCGCTTCTAGATGCAGGTAAAAATTCAAAA</TD></TR><TR><TD>NisFEG reverse</TD><TD>GTTTCTTCCTGCAGCGGCCGCTACTAGTATTATTATCTAATCTTTTTTT</TD></TR><TR><TD>PNisI+RBS forward</TD><TD>GTTTCTTCGAATTCGCGGCCGCTTCTAGAGAGCACTGGATAATGACTATT</TD></TR><TR><TD>PNisI+RBS reverse</TD><TD>TACTAGTAGCGGCCGCTGCAGGAAGAAACTTCCTCTTCCCTCCTTTCAA</TD></TR></TBODY></TABLE>
There were also primer designed for making a BioBrick out of the sfGFP(Bs). This sfGFP
was originally optimized for <I>Bacillus subtilis</I>. When it was
tested in <I>Lactococcus lactis</I>, the sfGFP(Bs) was shown to perform
really well in <I>L. lactis</I> as well.<SUP>3</SUP><TABLE border="1" style="width:90%"><TBODY><TR><TH width="25%">Primer</TH><TH>Sequence</TH></TR><TR><TD>sfGFP(Bs) forward</TD><TD>GTTTCTTCGAATTCGCGGCCGCTTCTAGATGTCAAAAGGAGAAGAGCT</TD></TR><TR><TD>sfGFP(Bs) reverse</TD><TD>GTTTCTTCCTGCAGCGGCCGCTACTAGTATTATTACTTATAAAGCTCAT</TD></TR></TBODY></TABLE><B>14 - 20 Juli</B>
A collection of constitutive promoters was also desirable for the toolbox.
We decided to use the CP promoter collection and test it in 
<I>Lactococcus lactis</I>, that was BioBricked by
the Uppsala iGEM team of 2013. The CP promoters are registered as
BBa_K1033219 (CP1), BBa_K1033220 (CP8), BBa_K1033221 (CP11), BBa_K1033222 (CP29),
BBa_K1033223 (CP30), BBa_K1033224 (CP41) and BBa_K1033225 (CP44). These parts were
ordered from iGEM HQ, except for CP8 as for this promoter the sequencing
was inconsistent. The promoters were sent by iGEM HQ using 
<I>Escherichia coli</I> containing pSB1C3 with the promoter as insert.
<I>E. coli</I> was grown and the plasmids with the promoter were
isolated.

<B>21 - 27 Juli</B>
BioBricking the genes NisT, NisI, NisP and the combined genes NisF, 
NisE and NisG gave problems with illegal restriction sites.
The illegal restrictionsite in NisI could be removed using the reverse
primer as listed above. This primer contains a mutation needed to
remove the illegal restriction site.
The other parts gave more problems, as these all contained multiple
restriction sites. Therefore, it was decided to use Gibson assembly
with primers containing mutations to remove the restriction site.
By using Gibson assembly, the genes could also be constructed in
pSB1C3 in one go.
The primers in the table below are denoted with the position of the 
illegal restriction site that they should remove. Illegal EcoRI sites 
are abbreviated with an E, XbaI sites with an X, SpeI site with an S and 
PstI sites with a P.
<TABLE border="1" style="width:90%"><TBODY><TR><TH width="25%">Gene</TH><TH>Primer</TH><TH>Sequence</TH></TR><TR><TD rowspan="8">NisT</TD><TD>Prefix forward</TD><TD>GTTTCTTCGAATTCGCGGCCGCTTCTAGATGGATGAAGTGAAAGAATTTACATCA</TD></TR><TR><TD>S980 reverse</TD><TD>TTGTTCCATAAACAAACTTGTATTATAAATGATGTAAATA</TD></TR><TR><TD>S980 forward</TD><TD>TATTTACATCATTTATAATACAAGTTTGTTTATGGAACAA</TD></TR><TR><TD>S1210 reverse</TD><TD>CCATAGTTGGTTGATATAATCCTGAAATTATCTTTACTTGTGTAC</TD></TR><TR><TD>S1210 forward</TD><TD>GTACACAAGTAAAGATAATTTCAGGATTATATCAACCAACTATGG</TD></TR><TR><TD>Suffix reverse</TD><TD>GTTTCTTCCTGCAGCGGCCGCTACTAGTATTATTATTCATCATTATCCT</TD></TR><TR><TD>Suffix forward</TD><TD>ATGAATAATAATACTAGTAGCGGCCGCTGCAGGAAGAAACAAAAGGGCAA</TD></TR><TR><TD>Prefix reverse</TD><TD>CACTTCATCCATCTAGAAGCGGCCGCGAATTCGAAGAAACATCCTTAGCG</TD></TR><TR><TD rowspan="10">NisP</TD><TD>Prefix forward</TD><TD>GTTTCTTCGAATTCGCGGCCGCTTCTAGGTGAAAAAAATACTAGGTTT</TD></TR><TR><TD>P225 reverse</TD><TD>AGTCCTTGTTGTCGATTTTACTGCTGGTGACTGCGCCCCT</TD></TR><TR><TD>P225 forward</TD><TD>AGGGGCGCAGTCACCAGCAGTAAAATCGACAACAAGGACT</TD></TR><TR><TD>E347 reverse</TD><TD>AACGCTATCTCTCTTACTAAATTCAGAACTAACTTGAGTT</TD></TR><TR><TD>E347 forward</TD><TD>AACTCAAGTTAGTTCTGAATTTAGTAAGAGAGATAGCGTT</TD></TR><TR><TD>E1657 reverse</TD><TD>CAATACTCTATTCCCATTAACTTCTGGACTATTCATTAGA</TD></TR><TR><TD>E1657 forward</TD><TD>TCTAATGAATAGTCCAGAAGTTAATGGGAATAGAGTATTG</TD></TR><TR><TD>Suffix reverse</TD><TD>GTTTCTTCCTGCAGCGGCCGCTACTAGTATTATCAATTTTTAGTCTTTC</TD></TR><TR><TD>Suffix forward</TD><TD>AAATTGATAATACTAGTAGCGGCCGCTGCAGGAAGAAACAAAAGGGCAA</TD></TR><TR><TD>Prefix reverse</TD><TD>CCTAGTATTTTTTTCATCTAGAAGCGGCCGCGAATTCGAAGAAACATCCTTAGCG</TD></TR><TR><TD rowspan="8">NisFEG</TD><TD>Prefix forward</TD><TD>GTTTCTTCGAATTCGCGGCCGCTTCTAGATGCAGGTAAAAATTCAAAA</TD></TR><TR><TD>P186 reverse</TD><TD>TCCAGTATCAGCAGAAATTAAACCAAACAAAATTTTCATC</TD></TR><TR><TD>P186 forward</TD><TD>GATGAAAATTTTGTTTGGTTTAATTTCTGCTGATACTGGA</TD></TR><TR><TD>E336/X349 reverse</TD><TD>CCTGTTTCTGCCAAACCAATCACTTCTAGTGTTTCATGTATTCTCTTA</TD></TR><TR><TD>E336/X349 forward</TD><TD>TAAGAGAATACATGAAACACTAGAAGTGATTGGTTTGGCAGAAACAGG</TD></TR><TR><TD>Suffix reverse</TD><TD>GTTTCTTCCTGCAGCGGCCGCTACTAGTATTATTATCTAATCTTTTTTT</TD></TR><TR><TD>Suffix forward</TD><TD>TAGATAATAATACTAGTAGCGGCCGCTGCAGGAAGAAACAAAAGGGCAA</TD></TR><TR><TD>Prefix reverse</TD><TD>TTTTACCTGCATCTAGAAGCGGCCGCGAATTCGAAGAAACATCCTTAGCG</TD></TR></TBODY></TABLE><B>28 Juli - 3 August</B>
The CP promoters in pSB1C3, that were isolated in the week
of 14 - 20 Juli, were tested on their insert size. For this, a PCR
was done on the plasmid using the primers VF2 (BBa_G00100) and VR
(BBa_G00101) were used. The insert size corresponded to the expected
size.

A touchdown PCR was done on the genes of the nisin operon and the 
sfGFP(Bs) gene. In this PCR the annealing temperature dropped from 55 °C to 
45 °C, lowering the temperature with 1 &amp;degC each cycle. This 
unfortunately resulted in a PCR program of just 10 cycles, instead of
the intended 30. The remaining steps of the program were finished the
next morning. Only the genes NisA, PNisI and sfGFP(Bs) were amplified
this way, see figure 1. Therefore, the PCR 									
was repeated, this time with a complete cycle. No additional genes
were amplified this way.

Another attempt was made for amplification of the remaining genes.
This time, the most ideal annealing temperature for each gene was
used by using a gradient PCR and placing the tubes at the optimal
temperature. This still did not amplify the remaining genes. Also, a
PCR with a general annealing temperature of 50 &amp;degC was done. This also
did not amplify the remaining genes.

The genes that were amplified in the first PCR (NisA, PNisI and sfGFP(Bs))
were purified using the GeneJET PCR Purification Kit from Thermo
Scientific. The purified products were digested with the enzymes EcoRI 
and PstI, using 2 &amp;mul of the product. These purified and restricted
products were loaded on gel, see figure 2. It was then discovered that the genes were 
barely visible on gel. The restricted genes were purified with the
GeneJET PCR Purification Kit and the concentration was measured with the
NanoDrop 1000. The clean, restricted products were too low in 
concentration to be suitable for ligation.

<B>4 - 10 August</B>
Because the PCR products of NisA, PNisI and sfGFP(Bs) were lost during
the purification and restriction in the week of 28 Juli - 3 August, the
PCR was repeated for this genes, together with the genes that could not
be amplified yet. This time a PCR was used that did not lower in 
temperature, like the touchdown PCR, but that increased in temperature 
with each step. This way it was hoped to get over the huge gap between
the annealing temperature of the primer in the first cycle and the
annealing temperature of the primer when the flap of the primer can also 
anneal to the first PCR products. The temperature was set to increase from
40 &amp;degC to 60 &amp;degC in 20 cycles. Then, an additional 20 cycles were
done at 65 &amp;degC. The PCR was performed under standard conditions as
was done before, together with a series of PCR that contained GC buffer
(supplied with the Phusion DNA polymerase by Thermo Scientific), and
a series of PCR with GC buffer and 1.5% DMSO. This time, with the help
of Lisa, the PCR was finally successful for all genes. 							<B>11 - 17 August</B>
The PCR on the BioBrick of the combined genes NisR and NisK was repeated
because the yield of this product was very low after the PCR in the week
of 4 - 10 August. The mixture of the previous PCR reaction was used as
the template. The PCR was performed under standard conditions, with an
annealing temperature of 64 &amp;degC and 1 &amp;mul, 2&amp;mul and 3 &amp;mul template.
No product was obtained with this PCR. So the PCR was once again repeated,
this time using diluted template. The mixture was diluted 50x, 300x,
1500x and 15000x. The PCR was repeated using 1 &amp;mul and 2 &amp;mul of each
of the dilutions. This way, product was obtained for all reactions.			
The PCR products of the genes NisA, NisB, NisC, NisRK, PNisI and sfGFP(Bs)
were purified using the GeneJET purification kit. Then 1 &amp;mug of each
purified product was restricted with EcoRI and PstI, together with 1 &amp;mug
of the pSB1C3 plasmid. The restriction enzymes were then inactivated by
heating the samples at 80 &amp;degC for 20 minutes. Ligated 6 &amp;mul of the
restricted PCR products to 2 &amp;mul restricted pSB1C3. Inactivated the
ligase by incubating at 65 &amp;degC for 10 minutes. Mixed 5 &amp;mul of the
ligation mixture with 25 &amp;mul electrocompetent <I>Escherichia coli</I> 
DH5&amp;alfa.

<B>18 - 31 August</B><B>References</B>
1. Trmčić, A. et al. (2011) Complete nisin A gene cluster from <I>Lactococcus 
lactis</I> M78 (HM219853) – obtaining the nucleic acid sequence and 
comparing it to other published nisin sequences. Genes Genom. 33: 217-221 
2. Li, H. and O´Sullivan, D.J. (2006) Identification of a nisI Promoter 
within the nisABCTIP Operon That May Enable Establishment of Nisin 
Immunity Prior to Induction of the Operon via Signal Transduction. 
J. Bacteriol. 188: 8496-8503 
3. Overkamp, W. et al. (2013) Benchmarking Various Green Fluorescent 
Protein Variants in <I>Bacillus subtilis</I>, <I>Streptococcus pneumoniae</I>, and 
<I>Lactococcus lactis</I> for Live Cell Imaging. Appl. Environ. Microbiol. 79:
6481-6490 </SPAN></DIV></DIV><DIV id="noscroll"><DIV id="titlebox"><H1> Notebook / Biobricks</H1></DIV><DIV id="navigationmenubox"><FONT face="verdana"><DIV id="navigationmenu"><DIV class="menu-item"><SPAN id="menu_home"><A href="https://2014.igem.org/Team:Groningen">Home</A></SPAN></DIV><DIV class="menu-item"><SPAN id="menu_project">Project</SPAN><DIV class="sub-menu"><DIV id="menu_project_overview" class="sub-menu-item"><A href="https://2014.igem.org/Team:Groningen/Project/Overview">Overview</A></DIV><DIV id="menu_project_detection" class="sub-menu-item"><A href="https://2014.igem.org/Team:Groningen/Project/Detection">Detection</A></DIV><DIV id="menu_project_secretion" class="sub-menu-item"><A href="https://2014.igem.org/Team:Groningen/Project/Secretion">Secretion</A></DIV><DIV id="menu_project_bandage" class="sub-menu-item"><A href="https://2014.igem.org/Team:Groningen/Project/Bandage">The bandage</A></DIV></DIV></DIV><DIV class="menu-item"><SPAN id="menu_modelling">Modeling</SPAN><DIV class="sub-menu"><DIV id="menu_modelling_model1" class="sub-menu-item"><A href="https://2014.igem.org/Team:Groningen/Modelling/Whymodeling?">Why modeling?</A></DIV><DIV id="menu_modelling_model2" class="sub-menu-item"><A href="https://2014.igem.org/Team:Groningen/Modelling/Overview">Modeling of the bandage</A></DIV><DIV id="menu_modelling_model3" class="sub-menu-item"><A href="https://2014.igem.org/Team:Groningen/Modelling/Modelingofthebandage">Modeling of L. lactis</A></DIV><DIV id="menu_modelling_model4" class="sub-menu-item"><A href="https://2014.igem.org/Team:Groningen/Modelling/Modelingofourbacteria">Modeling experiments</A></DIV></DIV></DIV><DIV class="menu-item"><SPAN id="menu_policyandpractice">Policy and practice</SPAN><DIV class="sub-menu"><DIV id="menu_policyandpractice_overview" class="sub-menu-item"><A href="https://2014.igem.org/Team:Groningen/PP/Overview">Overview</A></DIV><DIV id="menu_policyandpractice_futureperspectives" class="sub-menu-item"><A href="https://2014.igem.org/Team:Groningen/PP/Futureperspectives">Future perspectives</A></DIV><DIV id="menu_policyandpractice_martinihospital" class="sub-menu-item"><A href="https://2014.igem.org/Team:Groningen/PP/Martini">Martini Hospital</A></DIV><DIV id="menu_policyandpractice_events" class="sub-menu-item"><A href="https://2014.igem.org/Team:Groningen/PP/Sharingourideas">Sharing our idea</A></DIV><DIV id="menu_policyandpractice_newscoverage" class="sub-menu-item"><A href="https://2014.igem.org/Team:Groningen/PP/Media">Media Coverage</A></DIV><DIV id="menu_policyandpractice_survey" class="sub-menu-item"><A href="https://2014.igem.org/Team:Groningen/PP/Survey">Survey</A></DIV></DIV></DIV><DIV class="menu-item"><SPAN id="menu_education"> Education</SPAN><DIV class="sub-menu"><DIV id="menu_education_syntheticbiology" class="sub-menu-item"><A href="https://2014.igem.org/Team:Groningen/Education/SyntheticBiology">Synthetic biology</A></DIV><DIV id="menu_education_lactococcuslactis" class="sub-menu-item"><A href="https://2014.igem.org/Team:Groningen/Education/Lactococcus">Lactococcus lactis</A></DIV><DIV id="menu_education_burnwounds" class="sub-menu-item"><A href="https://2014.igem.org/Team:Groningen/Education/Burnwounds">Burn wounds</A></DIV><DIV id="menu_education_antibioticresistance" class="sub-menu-item"><A href="https://2014.igem.org/Team:Groningen/Education/Resistance">Antibiotic resistance</A></DIV></DIV></DIV><DIV class="menu-item"><SPAN id="menu_team">Team</SPAN><DIV class="sub-menu"><DIV id="menu_team_teammembers" class="sub-menu-item"><A href="https://2014.igem.org/Team:Groningen/Team/Members">Team members</A></DIV><DIV id="menu_team_instructors_and_supervisors" class="sub-menu-item"><A href="https://2014.igem.org/Team:Groningen/Team/Supervisors">Instructors and supervisors</A></DIV><DIV id="menu_team_photogallery" class="sub-menu-item"><A href="https://2014.igem.org/Team:Groningen/Team/Gallery">Photogallery</A></DIV><DIV class="sub-menu-item"><A href="https://igem.org/Team.cgi?year=2014&amp;team_name=Groningen">iGEM team profile</A></DIV></DIV></DIV><DIV class="menu-item"><SPAN id="menu_parts">Parts</SPAN><DIV class="sub-menu"><DIV id="menu_parts_biobricks" class="sub-menu-item"><A href="https://2014.igem.org/Team:Groningen/Parts/Biobricks">Biobricks</A></DIV><DIV class="sub-menu-item">
          iGEM parts page
        </DIV><DIV id="menu_parts_characterization" class="sub-menu-item"><A href="https://2014.igem.org/Team:Groningen/Parts/Characterization">Characterization</A></DIV></DIV></DIV><DIV class="menu-item"><SPAN id="menu_notebook">Notebook</SPAN><DIV class="sub-menu"><DIV id="menu_notebook_detection" class="sub-menu-item"><A href="https://2014.igem.org/Team:Groningen/Notebook/Detection">Detection</A></DIV><DIV id="menu_notebook_secretion" class="sub-menu-item"><A href="https://2014.igem.org/Team:Groningen/Notebook/Secretion"> Secretion</A></DIV><DIV id="menu_notebook_bandage" class="sub-menu-item"><A href="https://2014.igem.org/Team:Groningen/Notebook/Bandage">The bandage</A></DIV><DIV id="menu_notebook_biobricks" class="sub-menu-item"><A href="https://2014.igem.org/Team:Groningen/Notebook/Biobricks">Biobricks</A></DIV><DIV id="menu_notebook_protocols" class="sub-menu-item"><A href="https://2014.igem.org/Team:Groningen/Notebook/Protocols">Protocols</A></DIV></DIV></DIV><DIV class="menu-item"><SPAN id="menu_safety">Safety</SPAN><DIV class="sub-menu"><DIV id="menu_safety_safety" class="sub-menu-item"><A href="https://2014.igem.org/Team:Groningen/Safety/Safety">Safety</A></DIV></DIV></DIV><DIV class="menu-item"><SPAN id="menu_artanddesign">Art and design</SPAN><DIV class="sub-menu"><DIV id="menu_artanddesign_art" class="sub-menu-item"><A href="https://2014.igem.org/Team:Groningen/Art/Art">Art</A></DIV></DIV></DIV><DIV class="menu-item"><SPAN id="menu_awards">Awards</SPAN><DIV class="sub-menu"><DIV id="menu_awards_awards" class="sub-menu-item"><A href="https://2014.igem.org/Team:Groningen/Awards/Badges">Badges</A></DIV><DIV id="menu_awards_checklist" class="sub-menu-item"><A href="https://2014.igem.org/Team:Groningen/Awards/Checklist">Checklist</A></DIV></DIV></DIV><DIV class="menu-item"><SPAN id="menu_acknowledgements">Acknowledgements</SPAN><DIV class="sub-menu"><DIV id="menu_acknowledgements_acknowledgements" class="sub-menu-item"><A href="https://2014.igem.org/Team:Groningen/Acknowledgements/Acknowledgements">Acknowledgements</A></DIV></DIV></DIV></DIV></FONT></DIV></DIV></DIV></DIV><DIV class="printfooter">
Retrieved from &quot;<A href="http://2014.igem.org/Team:Groningen/Notebook/Biobricks">http://2014.igem.org/Team:Groningen/Notebook/Biobricks</A>&quot;</DIV></DIV></DIV><DIV id="footer-box" class="noprint"><DIV id="footer"><UL id="f-list"><LI id="t-recentchanges"><A href="/Special:RecentChanges" title="Recent changes">Recent changes</A></LI><LI id="t-whatlinkshere"><A href="/Special:WhatLinksHere/Team:Groningen/Notebook/Biobricks" title="List of all wiki pages that link here [j]" accesskey="j">What links here</A></LI><LI id="t-recentchangeslinked"><A href="/Special:RecentChangesLinked/Team:Groningen/Notebook/Biobricks" title="Recent changes in pages linked from this page [k]" accesskey="k">Related changes</A></LI><LI id="t-specialpages"><A href="/Special:SpecialPages" title="List of all special pages [q]" accesskey="q">Special pages</A></LI><LI><A href="/Special:Preferences">My preferences</A></LI></UL></DIV><DIV id="footer"><UL id="f-list"><LI id="t-print"><A href="/wiki/index.php?title=Team:Groningen/Notebook/Biobricks&amp;printable=yes" title="Printable version of this page [p]" accesskey="p">Printable version</A></LI><LI id="t-permalink"><A href="/wiki/index.php?title=Team:Groningen/Notebook/Biobricks&amp;oldid=244943" title="Permanent link to this revision of the page">Permanent link</A></LI><LI id="privacy"><A href="/2014.igem.org:Privacy_policy" title="2014.igem.org:Privacy policy">Privacy policy</A></LI><LI id="disclaimer"><A href="/2014.igem.org:General_disclaimer" title="2014.igem.org:General disclaimer">Disclaimers</A></LI></UL></DIV></DIV></BODY></HTML>