"
Page
Discussion
View source
History
teams
Log in
 
Team:Groningen/Template/MODULE/Notebook/toolbox
From 2014.igem.org
Notebook > Toolbox
 
 
 
7 July - 13 July
 
Primers were designed for making BioBricks out of the separate genes from  the nisin operon. For this, the sequence of the transposon Tn5307 was used, a transposon that contains the nisin operon, and of  which the sequence is known.1 This sequence is close to the sequence of the transposon Tn5276, the transposon which will form the template for making the new BioBricks. The primers for making a BioBrick out of PNisI with a RBS were based on a sequence found in a research paper that documented the identification of this promoter.2 The list of primersequences can be found in the first table at the bottom of this page. It was decided to combine the genes NisR and NisK as one BioBrick and NisF, NisE and NisG as one BioBrick, because these genes are related in function or work together.
 
There were also primers designed for making a BioBrick out of the sfGFP(Bs), the primersequences can be found at second table at the bottom of this page. This sfGFP was originally optimized for Bacillus subtilis. When it was tested in Lactococcus lactis, the sfGFP(Bs) was shown to perform really well in L. lactis as well.3
 
 
 
 
14 July - 20 July
 
A collection of constitutive promoters was also desirable for the toolbox. We decided to use the CP promoter collection and test it in Lactococcus lactis, that was BioBricked by the Uppsala iGEM team of 2013. The CP promoters are registered as CP1 , CP8 , CP11 , CP29 , CP30 , CP41 and CP44 . These parts were ordered from iGEM HQ, except for CP8 as for this promoter the sequencing was inconsistent. The promoters were sent by iGEM HQ using Escherichia coli containing pSB1C3 with the promoter as insert. E. coli was grown and the plasmids with the promoter were isolated.
 
 
 
 
21 July - 27 July
 
BioBricking the genes NisT, NisI, NisP and the combined genes NisF,  NisE and NisG gave problems with illegal restriction sites.
The illegal restrictionsite in NisI could be removed using the reverse primer as listed above. This primer contains a mutation needed to remove the illegal restriction site.
The other parts gave more problems, as these all contained multiple restriction sites. Therefore, it was decided to use Gibson assembly with primers containing mutations to remove the restriction site. By using Gibson assembly, the genes could also be constructed in pSB1C3 in one go. The primersequences can be found at the third table at the bottom of this page.
 
 
 
 
28 July - 3 August
 
The CP promoters in pSB1C3, that were isolated in the week of 14 - 20 July, were tested on their insert size. For this, a PCR was done on the plasmid using the primers VF2 and VR .  The insert size corresponded to the expected size.
 
A touchdown PCR was done on the genes of the nisin operon and the  sfGFP(Bs) gene. In this PCR the annealing temperature dropped from 55 °C to  45 °C, lowering the temperature with 1 °C each cycle. This  unfortunately resulted in a PCR program of just 10 cycles, instead of the intended 30. The remaining steps of the program were finished the next morning. Only the genes NisA, PNisI and sfGFP(Bs) were amplified this way, see figure 1. Therefore, the PCR           was repeated, this time with a complete cycle. No additional genes were amplified this way.
 
 
Figure 1: Amplification of PNisI with RBS, NisA and sfGFP(Bs), with an added prefix and suffix.
 
 
Another attempt was made for amplification of the remaining genes. This time, the most ideal annealing temperature for each gene was used by using a gradient PCR and placing the tubes at the optimal temperature. This still did not amplify the remaining genes. Also, a PCR with a general annealing temperature of 50 °C was done. This also did not amplify the remaining genes.
 
The genes that were amplified in the first PCR (NisA, PNisI and sfGFP(Bs)) were purified using the GeneJET PCR Purification Kit from Thermo Scientific. The purified products were digested with the enzymes EcoRI  and PstI, using 2 μl of the product. These purified and restricted products were loaded on gel, see figure 2. It was then discovered that the genes were  barely visible on gel. The restricted genes were purified with the GeneJET PCR Purification Kit and the concentration was measured with the NanoDrop 1000. The clean, restricted products were too low in  concentration to be suitable for ligation.
 
 
Figure 2: At the left: purified NisA, PNisI and sfGFP(Bs). At the right: NisA and sfGFP(Bs) after restriction with EcoRI and PstI.
 
 
 
 
 
4 August - 10 August
 
Because the PCR products of NisA, PNisI and sfGFP(Bs) were lost during the purification and restriction in the week of 28 July - 3 August, the PCR was repeated for this genes, together with the genes that could not be amplified yet. This time a PCR was used that did not lower in  temperature, like the touchdown PCR, but that increased in temperature  with each step. This way it was hoped to get over the huge gap between the annealing temperature of the primer in the first cycle and the annealing temperature of the primer when the flap of the primer can also  anneal to the first PCR products. The temperature was set to increase from 40 °C to 60 °C in 20 cycles. Then, an additional 20 cycles were done at 65 °C. The PCR was performed under standard conditions as was done before, together with a series of PCR that contained GC buffer (supplied with the Phusion DNA polymerase by Thermo Scientific), and a series of PCR with GC buffer and 1.5% DMSO. This time, with the help of Lisa, the PCR was finally successful for all genes, see figure 3.
 
 
Figure 3: PCR products of all the genes of the nisin operon with BioBrick prefix and suffix.
 
 
 
 
 
 
 
 
The toolbox is supposed to make it possible to use Lactococcus lactis as a chassis for iGEM constructs in the future. For the toolbox, some basic BioBricks are constructed that form the basis of larger constructs.
 
 
 
 
11 August - 17 August
 
The PCR on the BioBrick of the combined genes NisR and NisK was repeated because the yield of this product was very low after the PCR in the week of 4 - 10 August. The mixture of the previous PCR reaction was used as the template. The PCR was performed under standard conditions, with an annealing temperature of 64 °C and 1 μl, 2μl and 3 μl template. No product was obtained with this PCR. So the PCR was once again repeated, this time using diluted template. The mixture was diluted 50x, 300x, 1500x and 15000x. The PCR was repeated using 1 μl and 2 μl of each of the dilutions. This way, product was obtained for all reactions.
 
The PCR products of the genes NisA, NisB, NisC, NisRK, PNisI and sfGFP(Bs) were purified using the GeneJET purification kit. Then 1 μg of each purified product was restricted with EcoRI and PstI, together with 1 μg of the pSB1C3 plasmid. The restriction enzymes were then inactivated by heating the samples at 80 °C for 20 minutes. Ligated 6 μl of the restricted PCR products to 2 μl restricted pSB1C3. Inactivated the ligase by incubating at 65 °C for 10 minutes. Mixed 5 μl of the ligation mixture with 25 μl electrocompetent Escherichia coli DH5α. The transformants were grown on LB with 10 μg/ml  chloramphenicol, see figure 4. The white colonies were tested on insertsize with a colony PCR with the VF2 and VR primers. For every gene transformants were found that contained pSB1C3 with a correctly sized insert, except for NisB and NisRK.
 
 
Figure 4: E. coli containing pSB1C3 with a mRFP construct (pink colonies) and with another insert than mRFP, possibly a correct BioBrick (white colonies).
 
 
The E. coli that contained the pSB1C3 plasmid with a correctly  sized insert were grown as a liquid culture and the plasmid was isolated using the GeneJET Plasmid Miniprep Kit.
 
 
 
 
18 August - 31 August
 
The plasmids containing the new BioBricks NisA, NisC, PNisI and sfGFP(Bs)  that were isolated in the week of 11 - 17 August were concentrated with a SpeedVac and then sent for sequencing with the primers VF2 and VR . The sequence of each BioBrick was confirmed.
 
 
 
 
1 September - 7 September
 
Colony PCR on more transformants containing pSB1C3 with NisB and NisRK was repeated, using VF2 and VR . This time, a  correct insert size wasbfound for a NisRK transformant.
Because there was still no correct insert size found for a NisB  transformant, the remaining NisB colonies were tested for a correct insert size. There were no positive transformants found.
 
 
 
 
8 September - 14 September
 
A PCR was done on all the genes that contained illegal restriction sites: NisI, NisT, NisP and NisFEG. This should generate multiple PCR fragments for each gene, that will be assembled again using Gibson assembly. The PCR was split into two seperate reactions because of the large differences between the size of the different fragments. Fragments were sorted on <750 kb and >750 kb. The experiment was not continued because of time limits.
 
 
 
 
14 September - 21 September
 
Because of the constant low concentrations of the plasmid pSB1C3 with the new BioBricks, it was decided to transform the isolated pSB1C3 with the new BioBricks (NisA, NisC, NisRK, PNisI and sfGFP(Bs)) again in Escherichia coli DH5α.  The pSB1C3 plasmids containing the new BioBricks were again isolated. For each transformation, two seperate cultures were grown in LB with 10 μg/ml chloramphenicol. A miniprep was done on the cultures and the insert size was checked with colony PCR, using VF2 and VR as primers. In addition, a restriction analysis was done using EcoRI and PstI. Cultures that showed a correct insert size were sent for sequencing.
 
 
 
 
22 September - 28 September
 
The sequences of NisA, NisC, NisRK and sfGFP(Bs) were analyzed. The sequences of NisA and sfGFP(Bs) were completely correct. The sequence of NisC showed that a mutation occured at residue 185, where an AAT was changed to an AAA, changing an asparagine to a lysine. The changed amino acid was on the outside of the protein, so may not have an effect on the function of NisC. The sequence of NisRK showed two mutations. At residue 199 of NisR a GAA was changed to a GAG, not changing the amino acid at this position. At residue 300 of NisK an ACA was changed to an ATA, changing a threonine to an isoleucine. The PNisI with RBS showed a large mutation. At position 508, 18 basepairs were missing. Therefore it was decided to not send PNisI to iGEM HQ anymore.
 
A PCR was done to generate fragments for assembling genes that had illegal restriction sites. Every fragment was succesfully amplified. The fragments were purified using the GeneJET PCR clean up kit. Not all fragments have a sufficient concentration or purity, so only the fragments for the combined BioBrick of NisF, NisE and NisG was made using Gibson assembly. The transformants that contained pSB1C3 with NisFEG were tested using colony PCR. No positive transformants were found.
 
 
 
 
29 September - 5 October
 
Completed BioBricks were sent to iGEM HQ in a pSB1C3 backbone: NisA , NisC , NisRK and sfGFP(Bs) .
 
 
 
 
References
 
1. Trmčić, A. et al. (2011) Complete nisin A gene cluster from Lactococcus  lactis M78 (HM219853) – obtaining the nucleic acid sequence and  comparing it to other published nisin sequences. Genes Genom. 33: 217-221
 
2. Li, H. and O´Sullivan, D.J. (2006) Identification of a NisI promoter  within the NisABCTIP operon that may enable establishment of nisin  immunity prior to induction of the operon via signal transduction. J. Bacteriol. 188: 8496-8503
 
3. Overkamp, W. et al. (2013) Benchmarking various green fluorescent  protein variants in Bacillus subtilis, Streptococcus pneumoniae, and Lactococcus lactis for live cell imaging. Appl. Environ. Microbiol. 79: 6481-6490
 
 
 
Primers for making BioBricks out of the nisin operon
 
Primer
Sequence
NisA forward
GTTTCTTCGAATTCGCGGCCGCTTCTAGATGAGTACAAAAGATTTTAA
NisA reverse
GTTTCTTCCTGCAGCGGCCGCTACTAGTATTATTATTTGCTTACGTGAA
NisB forward
GTTTCTTCGAATTCGCGGCCGCTTCTAGATGATAAAAAGTTCATTTAA
NisB reverse
GTTTCTTCCTGCAGCGGCCGCTACTAGTATTATCATTTCATGTATTCTT
NisT forward
GTTTCTTCGAATTCGCGGCCGCTTCTAGATGGATGAAGTGAAAGAATTTACATCA
NisT reverse
GTTTCTTCCTGCAGCGGCCGCTACTAGTATTATTATTCATCATTATCCT
NisC forward
GTTTCTTCGAATTCGCGGCCGCTTCTAGATGAATAAAAAAAATATAAA
NisC reverse
GTTTCTTCCTGCAGCGGCCGCTACTAGTATTATCATTTCCTCTTCCCTC
NisI forward
GTTTCTTCGAATTCGCGGCCGCTTCTAGATGAGAAGATATTTAATACT
NisI reverse
GTTTCTTCCTGCAGCGGCCGCTACTAGTATTACTAATTTCCTACCTTCG
NisP forward
GTTTCTTCGAATTCGCGGCCGCTTCTAGATGAAAAAAATACTAGGTTT
NisP reverse
GTTTCTTCCTGCAGCGGCCGCTACTAGTATTATCAATTTTTAGTCTTTC
NisRK forward
GTTTCTTCGAATTCGCGGCCGCTTCTAGGTGTATAAAATTTTAATAGT
NisRK reverse
GTTTCTTCCTGCAGCGGCCGCTACTAGTATTATTACTTTTTTATTTTTA
NisFEG forward
GTTTCTTCGAATTCGCGGCCGCTTCTAGATGCAGGTAAAAATTCAAAA
NisFEG reverse
GTTTCTTCCTGCAGCGGCCGCTACTAGTATTATTATCTAATCTTTTTTT
PNisI+RBS forward
GTTTCTTCGAATTCGCGGCCGCTTCTAGAGAGCACTGGATAATGACTATT
PNisI+RBS reverse
TACTAGTAGCGGCCGCTGCAGGAAGAAACTTCCTCTTCCCTCCTTTCAA
 
 
 
 
Primers for making the new sfGFP(Bs) BioBrick
 
Primer
Sequence
sfGFP(Bs) forward
GTTTCTTCGAATTCGCGGCCGCTTCTAGATGTCAAAAGGAGAAGAGCT
sfGFP(Bs) reverse
GTTTCTTCCTGCAGCGGCCGCTACTAGTATTATTACTTATAAAGCTCAT
 
 
 
 
Primers for making BioBricks out of genes with illegal restriction sites
 
The primers are denoted with the position of the  illegal restriction site that they should remove. Illegal EcoRI sites  are abbreviated with an E, XbaI sites with an X, SpeI site with an S and  PstI sites with a P.
 
Gene
Primer
Sequence
NisT
Prefix forward
GTTTCTTCGAATTCGCGGCCGCTTCTAGATGGATGAAGTGAAAGAATTTACATCA
 
S980 reverse
TTGTTCCATAAACAAACTTGTATTATAAATGATGTAAATA
 
S980 forward
TATTTACATCATTTATAATACAAGTTTGTTTATGGAACAA
 
S1210 reverse
CCATAGTTGGTTGATATAATCCTGAAATTATCTTTACTTGTGTAC
 
S1210 forward
GTACACAAGTAAAGATAATTTCAGGATTATATCAACCAACTATGG
 
Suffix reverse
GTTTCTTCCTGCAGCGGCCGCTACTAGTATTATTATTCATCATTATCCT
 
Suffix forward
ATGAATAATAATACTAGTAGCGGCCGCTGCAGGAAGAAACAAAAGGGCAA
 
Prefix reverse
CACTTCATCCATCTAGAAGCGGCCGCGAATTCGAAGAAACATCCTTAGCG
NisP
Prefix forward
GTTTCTTCGAATTCGCGGCCGCTTCTAGGTGAAAAAAATACTAGGTTT
 
P225 reverse
AGTCCTTGTTGTCGATTTTACTGCTGGTGACTGCGCCCCT
 
P225 forward
AGGGGCGCAGTCACCAGCAGTAAAATCGACAACAAGGACT
 
E347 reverse
AACGCTATCTCTCTTACTAAATTCAGAACTAACTTGAGTT
 
E347 forward
AACTCAAGTTAGTTCTGAATTTAGTAAGAGAGATAGCGTT
 
E1657 reverse
CAATACTCTATTCCCATTAACTTCTGGACTATTCATTAGA
 
E1657 forward
TCTAATGAATAGTCCAGAAGTTAATGGGAATAGAGTATTG
 
Suffix reverse
GTTTCTTCCTGCAGCGGCCGCTACTAGTATTATCAATTTTTAGTCTTTC
 
Suffix forward
AAATTGATAATACTAGTAGCGGCCGCTGCAGGAAGAAACAAAAGGGCAA
 
Prefix reverse
CCTAGTATTTTTTTCATCTAGAAGCGGCCGCGAATTCGAAGAAACATCCTTAGCG
NisFEG
Prefix forward
GTTTCTTCGAATTCGCGGCCGCTTCTAGATGCAGGTAAAAATTCAAAA
 
P186 reverse
TCCAGTATCAGCAGAAATTAAACCAAACAAAATTTTCATC
 
P186 forward
GATGAAAATTTTGTTTGGTTTAATTTCTGCTGATACTGGA
 
E336/X349 reverse
CCTGTTTCTGCCAAACCAATCACTTCTAGTGTTTCATGTATTCTCTTA
 
E336/X349 forward
TAAGAGAATACATGAAACACTAGAAGTGATTGGTTTGGCAGAAACAGG
 
Suffix reverse
GTTTCTTCCTGCAGCGGCCGCTACTAGTATTATTATCTAATCTTTTTTT
 
Suffix forward
TAGATAATAATACTAGTAGCGGCCGCTGCAGGAAGAAACAAAAGGGCAA
 
Prefix reverse
TTTTACCTGCATCTAGAAGCGGCCGCGAATTCGAAGAAACATCCTTAGCG
 
 
 
 
Retrieved from " http://2014.igem.org/Team:Groningen/Template/MODULE/Notebook/toolbox "
Recent changes
What links here
Related changes
Special pages
My preferences
Printable version
Permanent link
Privacy policy
Disclaimers
