"
Page
Discussion
View source
History
teams
Log in
 
Team:Groningen/Template/MODULE/Notebook/Secretion
From 2014.igem.org
Notebook > Secretion
 
 
 
July 21 - July 25
 
In silico preparation of primers for the Gibson assembly between signal sequence USP45 and modified version of aiiA ( BBa_C0060 ).
 
In silico production of synthetic gene ssUSP45DspB ( BBa_K1365111 )
 
 
 
 
July 23 - August 3
 
Primer design for the synthetic gene
 
 
 
 
August 6 - August 8
 
Obtaining all the BioBricks needed for the secretion system to destroy P. aeruginosa and S. aureus
 
Performing three transformation to E. coli, one with promoter P2 ( BBa_I746104 ), one with lasI ( BBa_K081009 ), and one with aiiA ( BBa_C0060 )
The tranformated E. coli was divided in several concentrations over LB agar plates with Chloramphenicol
Three glycerol stocks of E. coli DH5α containing the RBS ( BBa_ B0034 ), the double terminator ( BBa_ B0015 ) and the promoter LasR ( BBa_ R0079 ), were inoculated in LB medium, and grown overnight
The plasmids were isolated out of the overnight cultures, and checked on gel
 
 
 
 
 
August 11 - August 15
 
Assembling the promoters P2 ( BBa_I746103 ) and pLas ( BBa_K091117 ) with RBS and the gene coding for GFP ( BBa_ E0040 ) with double terminator ( BBa_ B0015 )
 
The Biobricks pLAS, P2, GFP, RBS and Double terminator were assembled using three-A assembly, the constructs were ligated, and transformed to E. coli DH5α
The transformations were plated on LB agar with kanamycin
All the colourless colonies were plated to fresh plates again, and grown overnight
The construct was checked by colony PCR, ending up with the following constructs: Ps with RBS, pLas with RBS, and the gene coding for GFP with double terminator
Another colony PCR was performed with Phusion polymerase this time, in order to produce a more clean product for a second assembly
The primers and the synthetic gene ssUSP45DspB ( BBa_K1365111 ) arrived
The mrfp insert was cut out of pSB1C3, run on gel, and the linearized vector was purified out of the gel
ssUSP45DspB was ligated with pSB1C3, and transformed to E. coli DH5α
Few colonies were visible after growing the transformation mixture of LB agar plates with chloramphenicol
After digestion, the constructs of the few colonies were checked on gel, only one colony showed the right size for the insert
 
 
 
 
 
August 25 - August 29
 
Obtaining constructs with different promoters and the gene coding for GFP for promoter analysis in L. lactis
An overnight culture of E. coli DH5α with the P2 with RBS construct, pLas with RBS construct, and the gene coding for GFP with the double terminator construct was made
The constructs P2 with RBS, pLas with RBS, the gene coding for GFP with the double terminator and ssUSP45DspB were checked on gel, only P2 with RBS and pLas with RBS had the correct size
The same experiment with the ssUSP45DspB was performed again, but again no positive results were obtained
PCR was performed with the synthetic ssUSP45DspB gene, resulting in ssUSP45 and ssUSP45DspB without the His-tag
 
 
 
 
 
September 1 - September 5
 
Obtaining ssUSP45DspB ( BBa_K1365111 ) as a BioBrick in the pSB1C3 vector and ssUSP45Aiia ( BBa_K1365169 ) as a BioBrick in pSB1C3 vector
 
Two PCR reactions directly on the products were performed
The products were ligated with pSB1C3, and transformed to E. coli DH5α
Another PCR was done with the amplificated PCR product of ssUSPdspB, one with the signal sequence of USP45, and one with the ssUSP45DspB without His-tag
These products were also ligated with pSB1C3, and transformed to E. coli
Afterwards, Gibson assembly had been done with modified Aiia and the signal sequence of USP45 making ssUSP45aiiA, to enhance the chances of successfully ligating the Gibson product into pSB1C3, PCR was done on the final Gibson product
The PCR product was checked on gel, giving positive results for the ssUSP45aiiA, then it was ligated with pSB1C3, and transformed to E. coli
Several colonies were visible on the plated transformation products
Colony PCR was done on them with the regular pSB1C3 test primers, which showed no results
The colonies were grown overnight, and the plasmids were isolated in the next week
 
 
 
 
 
 
 
 
 
 
 
September 8 - September 12
 
Analysis of the constructed Biobricks
 
PCR was performed directly on the ssUSP45DspB product
The plamids containing  the ssUSP45aiiA was isolated our of E. coli DH5α and checked on gel, which showed some positive results for the contructs
New primer were designed, to give it another try
the first BioBricks were send for sequencing
 
 
 
 
 
September 15 - September 19
 
Obtaining the anti-Staphylococcus aureus system (p2s) and the anti-Pseudomonas aeruginosa system(pLASs)
 
Both systems were assembled by using Gibson assembly
Downstream of the pLas a RBS is attached together with the promoter P2 and dspB with tail PCR
A Gibson tail was attached with PCR to the P2 with RBS, dspB with RBS, aiiA and the double terminator
ssUSP45DspB, and nisA was added and incubated for 90 minutes at 50 °C
The Gibson fragment was amplified with gradient PCR
The construct was digested, checked on gel, and send for sequencing
Only 1200 base pairs of the construct could be found on gel
 
 
 
 
 
September 29 - October 5
 
Obtaining the anti-Pseudomonas aeruginosa system with Three-A assembly
 
Extra amounts of dspB, ssUSP45DspB with His-tag, aiiA, ssUSP45aiiA with His-tag, pLAS, the RBS, the double terminator, pSB2k3 vector, pSB1C3 vector and pSB1A3 vector were miniprepped to be used for the assembly
 
aiiA, dspB, ssUSP45DspB with His-tag and ssUSP45aiiA with His-tag were digested with XbaI and PstI, and the RBS was cut with EcoRI and SpeI
 
These were eventually ligated for 3 hours into pSB2K3 and transformed into E. coli NEB cells
 
Name assembly
Upstream part
Downstream part
Backbone
A1’
RBS
aiiA
pSB2K3
A2’
RBS
dspB
pSB2K3
A3’
RBS
dspB+HIS
pSB2K3
A4’
RBS
aiiA+HIS
pSB2K3
 
After growing them for 1 day, colony PCR was done on the correct clones, these were inoculated for plasmid extraction
 
Then A1’, A4’ was digested with XbaI and PstI. A2’ and A3’ was cut with EcoRI and SpeI. Then ligated
 
Name assembly
Upstream part
Downstream part
Backbone
A1”
RBS + aiiA
Double terminator
pSB1A3
A2”
pLAS
RBS + dspB
pSB1A3
A3”
pLAS
RBS + dspB+HIS
pSB1A3
A4”
RBS + aiiA+HIS
Double terminator
pSB1A3
 
After doing growing analyzing and miniprepping the correct clones, A1” and A3”was cut with EcoRI and SpeI. A2” and A4” was cut with XbaI and PstI and ligated afterwards
 
Name assembly
Upstream part
Downstream part
Backbone
A1”’
pLAS+RBS+dspB
RBS+aiiA+Dterm
pSB1C3
A2”’
pLAS+RBS+dspB+HIS>
RBS+aiiA+HIS+Dterm
pSB1C3
 
After transformation, the plasmids were isolated, and confirmed by sequencing
 
 
 
 
October 7 - October 12
 
Fusing the secretion and detection system of Pseudomonas aeruginosa
 
The detection system was cut with EcoRI & SpeI, whilst the secretion system was cut with XbaI & PstI, and they were ligated into pSB2K3
 
 
 
 
 
October 13 - October 16
 
Placing the Pseudomonas Aeruginosa system in Lactococcus lactis NZ9800
 
The Pseudomonas aeruginosa system (detection system fused with the secretion system) and pIL253 vector was digested with EcoRI & PstI
The Pseudomonas aeruginosa system was ligated with the pIL253 vector, and transformed to L. lactis NZ9800
 
 
 
 
 
Retrieved from " http://2014.igem.org/Team:Groningen/Template/MODULE/Notebook/Secretion "
Recent changes
What links here
Related changes
Special pages
My preferences
Printable version
Permanent link
Privacy policy
Disclaimers
