"
Page
Discussion
View source
History
teams
Log in
 
Team:Groningen/Notebook/Secretion
From 2014.igem.org
Week 1 (21-25 july)
In silico preparation of primers for the Gibson assembly between signal sequence USP45 and modified version of Aiia. In silico production of synthetic gene ssUSP45DspB.  General preparation of lab necessities.
week 2 (28 july-3 august)
designing primers for the synthetic gene.
Week 3 (6 – 8 august):
goal: obtain all the biobricks necessary for the secreting systems of P. aeruginosa and S. aureus.
E. coli was chemically transformed with 3 biobricks  in order to obtain them:
BBa_I746104: P2.
BBa_K081009: LasI (for the Detection part).
BBa_C0060: aiiA.
After which 1, 10, 20, 50% was inoculated on Chloramphenicol LB-agar plates.
The RBS, Double Terminators, and the promotor LAS biobricks were already transformed and isolated.
The clones were inoculated in liquid media were they would be prepared for miniprep.
The O/N culture was miniprepped and checked on gel, giving a positive result.
week 4 (  11 – 17 august):
goal: assemble the promotors with RBS and gfp (BBa_E0040) with double terminator.
Biobricks pLAS, P2, gfp, RBS and Double terminator were assembled accordingly to their place in the construct with 3A assembly, ligated, chemically transformed and inoculated on kanamycine agar plates.
The O/N grown colorless clones were picked and grown on their own individual plate. Afterwards, colony PCR on all the colonies showed a positive result for assembled: P2+RBS, pLAS+RBS, GFP+Dterm.
After the colony PCR, another one was done with phusion DNA polymerase, so that the product could be used for a second assembly with (P2+RBS)+(GFP+Dterm) and (pLAS+RBS)+(GFP+Dterm) corresponding in only a few clones which gave negative results after colony PCR.
Therefore, another assembly with the constructs should be done.
Primers came in and were prepared accordingly.
The synthetic gene ssUSP45dspB, which arrived from IDT, was prepared according to the protocol made by IDT.  1 µl of pSB1C3 was cut with EcoRI & PstI. The digestion mix ran on a agarose gel, afterwhich gel purification of the linearized backbone occurred. 70ng of vector was used to ligate ssUSP45dspB.
The yield of transformants was very low after transforming the ligation mixture and growing the bacteria.  Only one colony contained the ssUSP45dspB according to gel.
Week 5 (25 – 29 august):
goal: obtaining a construct with the promotors and GFP for promotor analysis in L. lactis.
All the clones containing the constructs with p2+rbs, pLAS+rbs and gfp+Dtermwere grown again to be miniprepped. After this, these products, including ssUSP45dspB ran on a gel showing that p2+rbs and pLAS+rbs had the correct size, but ssUSP45dspB and gfp+Dterm didn’t had the correct size.
Again, the gblock of ssUSP45dspB was cloned with the digestion mix, and again rendering negative results.
After a couple of failures with the small amount of gblocks which has been given, PCR was done on 1µl of the digestion mixture. The products which were produced were: ssUSP45 and ssUSP45dspB without his-tag.
week 6: (1 – 5 september)
goal: obtain ssUSP45dspB in pSB1C3 biobrick and ssUSP45aiiA in pSB1C3 biobrick.
Because of the very low amount of gblocks given, a decision was made to do a PCR directly on the product itself, therefor multiplying it exponentially.
2 series of PCR ran:
1. Amplification of gblock in 50µl reaction.
2. amplification of product: 10*50 µl reaction.
The products were ligated into 70ng of pSB1C3 and transformed into E. coli . Another PCR was done with the amplificated PCR product of ssUSPdspB, these products were:
1. the signal sequence of USP45
2. the gblock without HIS-tag.
These products were also ligated into pSB1C3 and transformed into E. coli.
20 nucleotide overhangs were created on aiiA, making it ready for Gibson assembly
Afterwards, Gibson assembly had been done with modified aiiA and the signal sequence of USP45 making ssUSP45aiiA. to enhance the chances of successfully ligating the Gibson product into pSB1C3, PCR was done on the final gibson product.
The PCR product was checked on gel, giving positive results for presence of ssUSP45aiiA, then it got ligated into pSB1C3 and transformed Into E. coli as well.
Results of the transformed E. coli gave a yield of 40% of possible clones.
Colony PCR was done on them with the regular pSB1C3 test primers, but no product was seen. Therefor 40 possible clones were grown O/N and miniprepped the next week.
Week 7 (8 – 12 september)
Goal: analysis of the possible made Biobricks.
All the O/N cultures were miniprepped. The possible plasmids were cut with EcoRI and SpeI. After running on gel, the result showed us that 20 of the 40 clones contained all the biobricks, though with a low yield of plasmids.
A new strategy of assembling has been prepared and primers for this strategy have been made.
All the Biobricks were send for sequencing.
Notebook / Secretion
Home
Project
Overview
Detection
Secretion
The bandage
Modeling
Why modeling?
Modeling of the bandage
Modeling of L. lactis
Modeling experiments
Policy and practice
Overview
Future perspectives
Martini Hospital
Sharing our idea
Media Coverage
Survey
Education
Synthetic biology
Lactococcus lactis
Burn wounds
Antibiotic resistance
Team
Team members
Instructors and supervisors
Photogallery
iGEM team profile
Parts
Biobricks
iGEM parts page
Characterization
Notebook
Detection
Secretion
The bandage
Biobricks
Protocols
Safety
Safety
Art and design
Art
Awards
Badges
Checklist
Acknowledgements
Acknowledgements
Retrieved from " http://2014.igem.org/Team:Groningen/Notebook/Secretion "
Recent changes
What links here
Related changes
Special pages
My preferences
Printable version
Permanent link
Privacy policy
Disclaimers
