Team:Athens/Experiments
Team
Members Collaborations Attributions
Project
Description MERS-CoV Notebook Design Results Safety
Wet Lab
Experiments InterLab Parts
Dry Lab
Toehold Design Kinetic Modeling 3D Simulations Temperature Simulations
Human Practices
Overview Education & Public Engagement Integrated Human Practices
Product
Applied Design
Experiments
• Trehalase
• Gibson Assembly
• Toehold Switch Testing
• Protocols
Experiments
Trehalase
The first step of our experimental process was the isolation of our reporter gene, trehalase (TreA) from E.coli BL21 genomic DNA. We chose to clone TreA in pET-15b vector, since pET systems are reliable for recombinant protein expression in E. coli. After cloning TreA successfully, we used the whole gene to PCR amplify the two split fragments (TreA-a & TreA-b). The expression and subsequent purification of TreA was a prerequisite in order to gain insight on the trehalase assay and optimize its conditions. The 6xHIS-tagged protein was purified with nickel columns, in native and denaturing conditions. The denaturing agent used was guanidine.
The evaluation of the enzymatic activity of the purified protein followed, using a glucose detection assay. Trehalase hydrolyses the disaccharide trehalose to glucose, which, in our case was detected by two different methods DNS and Benedict’s test, in order to assess which one was more sensitive and ensure a result in variable conditions. In the first instance, we assayed trehalase activity at pH 7.5, which resulted in no enzymatic activity. That is because the catalytic activity of trehalase is optimized at a slightly acidic pH. Thus, we continued our experiments fixing the trehalase assay pH at 5.9. We incubated purified trehalase with trehalose for 15min at 37oC and then added either DNS or Benedict’s reagent to the reaction.
Both methods are based on color change. The DNS method produces a change from light to dark orange, while the Benedict’s test changes from clear blue to cloudy red (blue-green-yellow-orange-red). As is evident from the above image, the DNS method gave better results, with obvious darkening in the color of all the protein samples’ reactions. Concluding, we decided to perform our toehold switch testings with the trehalase assay at pH 5.7 and using DNS for glucose detection.
Gibson Assembly
The cloning vector chosen for our trigger RNAs and for our toehold switches was pGEM-3zf+. The main reason for this decision was the fact that this vector contains a T7 promoter, which would facilitate the in-vitro transcription of our parts. After each vector linearization, we performed treatment with the restriction enzyme DpnI, in order to destroy the plasmid template before setting up the NEBuilder HiFi DNA assembly reaction.
As mentioned above, the sequences of TreA used were isolated from the genome of E. coli BL21. The toehold switches that contain the corresponding zipper sequence, along with the triggers were all synthesized as double-stranded DNA fragments (IDT) .
Trigger RNAs
Our trigger primers were 30 nt long, with vector overlaps of 25-30 nt on both ends. Thus, preparation for the assembly only demanded linearization of the vector.
Toehold Switches with TreA-a
In this case, there were three fragments to be assembled: the vector, the toehold switch (from TBS up to zipper-a) and TreA-a. Therefore, we needed to perform two PCR reactions as preparation for the assembly; one for the linearization of the vector and one for the attachment (ligation) of overlapping primers at the ends of TreA-a. Having overlaps on the ends of all the fragments to be assembled, we could perform the cloning.
Toehold Switches with TreA
For the toehold switches with whole trehalase as reporter (TreA), we used as template vectors the assembled vectors with TreA-a which were produced in the previous step for each toehold switch. At the linearization step, the primers were designed so that the amplified part would be the whole vector apart from the zipper-a and TreA-a sequence. This way, this part would be deleted and we could replace it with TreA in the assembly process, constructing the corresponding toehold switches with TreA as reporter.
Zipper-b and TreA-b
For this assembly, the fragments we worked with were zipper-b, TreA-b and pET-15b vector. We followed the same process steps as in the assembly of the switches. The only alteration was the design of a ds-DNA region on the 5’ end of zipper-b overlapping the vector, which was used instead of overlapping primers.
Cloning Evaluation
After each assembly, E. coli DH5a chemically competent cells were transformed with the produced parts. Initially, following the protocol of NEBuilder HiFi DNA Assembly Cloning Kit, we used the assembly product mixture for the transformation, which resulted in no colonies. The first alternative we tried was gel extracting the product (via Monarch® DNA Gel Extraction Kit, NEB), in order to transform the cells solely with the assembled vector. This too resulted in no colonies. We also tried performing DNA purification to the assembly product mixture (via Monarch® PCR & DNA Cleanup Kit, NEB), getting the same result. Attempting to solve this problem, we performed the same transformations using XL1-Blue competent cells as well, which again resulted in no colonies. In the above cases we could detect the correct products following electrophoresis of the assembly reaction.
Therefore, we decided to concentrate the ligated DNA by ethanol precipitation before transformation. This method was successful for some assembly reactions in both DH5a and XL1-Blue strains, although the growth rate of the forming colonies was relatively small.
After screening these colonies by extracting plasmid DNA with small scale preparation (mini preps) and the appropriate enzymatic digests, we obtained the assembled sequences for Toehold Switch 16 and zipper-b/ TreA-b. We were unable to obtain the assembled sequences of the toehold switches with the whole trehalase, the triggers, and other toeholds with the split trehalase. One possible reason for not being able to obtain the switch with the whole trehalase is the fact that the insert was relatively large, resulting in low yield in the assembly process. In our effort to solve this issue, we increased the assembly time (4 hours instead of 50 minutes), which nevertheless did not work. Regarding the triggers, we tested the very few colonies we obtained but they contained only the empty vector. These transformations should be repeated in order to get more colonies for screening.
Toehold Switch Testing
Recapping, for each toehold switch testing, the corresponding trigger RNA represents a sequence of the viral RNA, which would be amplified during the sample pretreatment. In the in-vitro transcription & translation system, the toehold switches are transcribed by default, but their translation is only allowed upon trigger binding, under the condition that the toehold switch - trigger RNA pair tested is successful.
Thus, adding the transcribed trigger RNA in the transcription & translation system, was expected to activate TreA-a / zipper-a expression. Sampling from this reaction and mixing this sample with TreA-b / zipper-b, would allow zipper interactions and consequently TreA reassembly. The TreA-b / zipper-b complex would be expressed using the same kit, in order to avoid any buffer incompatibilities.
Trehalase Cell-Free expression
Since we did not have enough time to solve our transformation problem and did not obtain all the parts needed for the toehold switch testing, we decided to get a rough estimate of the time needed for a diagnosis by expressing TreA in our cell-free system. Therefore, we set up a PURExpress® reaction using 1 μg of TreA expression plasmid and carried out coupled transcription / translation for 3.5 hours to ensure sufficient TreA protein production. We tested the resulting reaction for the presence of functional TreA by incubating a small quantity of the transcription/translation reaction with trehalose and then adding DNS reagent. As shown in the photo below, appropriate colour formation indicated that the pH of the reaction was suitable for the TreA to work.
Left: Blank without trehalose. Right: Blank with trehalose
We then tried to determine the minimum time demanded for sufficient TreA production in order to reach the DNS glucose threshold. Thus, we took samples from the reaction at different time points and performed the same process as before.
We concluded that about 1 hour of TreA expression was demanded to produce sufficient TreA, in order to get a visible color change in the DNS assay.
Commercial Glucose Meter Results
The commercial glucose meter has a lower sensitivity, since it is intended for the quantification of glucose in human blood samples, on which the levels are significantly higher compared to the threshold of the DNS assay. Thus, performing the same experiments as for the DNS assay, we first attempted to prove that the time frame of 3.5 hours of reaction could give sufficient amount of TreA, in order to be subsequently quantified by the glucose meter.
The results confirmed that the commercial glucose meter is a suitable signal indicator and most importantly, that our proposed diagnostic methodology is valid. On the same logic as in the DNS assay experiments, we defined the minimum translation time needed in order to get enough TreA and a detectable signal from the commercial glucose test.
The samples taken at t=0, 45min, 1hr, 1:15hr, 1:30hr resulted in this indication, which implies that the glucose concentration was too low to be quantified.
The first measurement received was at the time frame of 1:45 hours post TreA expression.
❮ ❯
iGEM Part Construction
We chose XbaI and SpeI restriction enzymes that were included in the prefix and suffix of the pSB1C3 vector. We incorporated these enzymes in our overlapping primers so that all parts could be cloned directly into the pSB1C3 vector. We digested parts and vector with the enzymes and then we dephosphorylated pSB1C3 with Calf Intestine Alkaline Phosphatase (CIAP) to prevent vector religation since XbaI and SpeI create compatible overhangs. We carried out several ligations with T4 ligase with various ratios of vector to insert. After several unsuccessful transformations, we obtained red colonies but one day after the part submission deadline. Therefore, we were unable to send our part in the pSB1C3 vector.
Protocols
All Protocols were carried out following the manufacturer’s instructions.
Minipreps were carried out using the Monarch® Plasmid Miniprep Kit | NEB.
Maxiprepswere carried out using the QIAGEN® Plasmid Maxi Kit.
Gel extractionswere carried out using the Monarch® DNA Gel Extraction Kit | NEB.
PCR purificationswere carried out using the Monarch® PCR & DNA Cleanup Kit (5 μg) | Purification Kits | NEB.
Assemblies were carried out using the NEBuilder® HiFi DNA Assembly Cloning Kit | NEB.
The Protino® Ni-TED and Ni-IDA Packed Columns - MACHEREY-NAGEL were used for the purification of His-tag proteins.
The PURExpress® In Vitro Protein Synthesis Kit | NEB was used for in vitro transcription| translation.
Preparation of competent cells
Pick a single colony from a fresh LB-plate. Grow overnight at 37oC (10 ml in a universal).
Transfer 5 ml of overnight cells into 100 ml of LB in a 500 ml flask. Grow with shaking at 37oC for 2-3 hours, until the OD600 reaches 0.4- 0.6.
Chill the cells on ice for 5 min. Transfer to chilled Falcon tubes and spin at 4oC at about 3K for 5 mins.
Pour away the supernatant. Resuspend gently in 25 ml of cold 0.1 M CaCl2. Leave it on ice for 2 min.
Gently spin for 10 mins at 3000 rpm at 4oC.
Resuspend in cold 0.1 M CaCl2 (10 ml) and leave for at least 1 hour on ice (1-4 hours).
Add 10 ml of 40% sterile glycerol and mix gently.
Transfer 1 ml into chilled Eppendorf tubes.
Freeze at -80oC.
Transformation
Prepare LB+ antibiotic agar plates.
Take out the competent cells from the freezer and place on ice (30 min- 1 hour).
Aliquot into 200 μl aliquots in Eppendorf tubes.
Add 1-5 μl of ligation/ DNA to each tube. Mix gently. Leave on ice for 40 min.
Heat shock cells for 90 sec at 42oC. Place on ice for 2 min.
Add 800 μl of LB medium prewarmed to 37oC.
Grow gently shaking for 1 hour (37oC).
After incubation for 1 hour at 37oC, 100 μl of culture can be spread on the plate and the rest of the transformation can be centrifuged for 3 min at 13000 rpm. Subsequently, supernatant is discarded and the pellet is resuspended in 100 μl of LB. This solution is spread on the plate.
The plates are left to dry for 10- 15 min. After drying they are incubated at 37oC in inverted position.
LB Agar preparation
Measure 500 ml of ddH2O and transfer in glass bottle.
Weigh 15.25g of Luria Agar powder and add to water.
Autoclave the bottle containing the solution.
When the solution has cooled down, add antibiotic.
Pour the LB Agar solution into plates.
LB Broth preparation
Measure 500 ml of ddH2O and transfer in glass bottle.
Weigh 5 g tryptone, 2.5 g yeast, 2.5g NaCl and add to water.
Autoclave the bottle containing the solution.
Glycerol stock preparation
Pour 500 ul of glycerol 40% in 1.5 or 2 ml tube.
Add 500 ul of culture in LB.
Store at -80oC.
Overnight inoculums
Add 5 ml of LB broth to glass tubes.
Add antibiotic (volume depends on antibiotic’s concentration in order to reach the correct final concentration).
Pick single colonies from plate using a pipette with a tip on and inoculate the LB.
Agarose gel
Agarose gel % and volume were chosen according to the needs of each electrophoresis performed. For an 1% agarose gel (100 ml):
Measure 100 ml of 1X TBE and transfer to a Pyrex glass bottle.
Weigh 1g of agarose and add to bottle.
Heat mixture in the microwave until agarose is totally dissolved.
Carefully add 7 ul of EtBr.
Pour mixture into cast.
Agarose gel electrophoresis
Place agarose gel previously prepared in agarose gel electrophoresis device.
Fill the apparatus with TBE 1X.
Load appropriate ladder and the samples to the wells of the gel.
Place the lid on the device, connect the electrodes and run the gel.
Turn off power and disconnect electrodes.
Assess bands under UV light.
Polyacrylamide gel electrophoresis
For a 10% SDS PAGE:
For a 5 ml stacking gel: 3.4 ml H2O, 1.25 ml 0.5 M Tris-HCl (pH 6.8), 0.05 ml 10% (w/v) SDS, 0.83 ml 30% Acrylamide, 50 ul 10% (w/v) AP, 5 ul TEMED
For a 10 ml separating gel: 4 ml H2O, 3.3 ml 30% Acrylamide, 0.63 ml 1M Tris (pH= 8.8), 0.05 ml 10% (w/v) SDS, 0.05 ml 10% (w/v) AP, 5 ul TEMED
Make the separating gel.
Set the cast.
Carefully pipette the gel in the gap between the glass plates of the cast.
Make the stacking gel.
When separating gel is ready, carefully pipette in the stacking gel on top.
Place comb.
When gel is ready, take out comb, set the glass frames in the running device and pour in the running buffer.
Prepare and load samples under reducing conditions.
Connect the electrodes and run the electrophoresis.
Turn of power and disconnect the electrodes.
Assess bands by staining with Coomassie blue and de-staining the gel.
Enzymatic reactions
Digests were carried out at 50 ul using 5 units of enzyme, x μl of DNA (depending on each digest), 1x digest buffer, x ul WFI (depending on each digest). Digests were placed in the thermocycler at 37oC for 3 hours or overnight, depending on each digest. Heat inactivation of the enzymes was performed for 20 minutes at 65 or 80oC, depending on each digest.
Ligations were performed following the T4 DNA Ligase NEB protocol, at different ratios of vector/insert, always following the manufacturer’s instructions.
Contact Info:
igemathens2018@gmail.com
