<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_Athens_Model skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><DIV id="top_title"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:Athens/Model</SPAN></H1></DIV><DIV id="HQ_page"><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><NAV><UL id="navigation-list"><LI class="dropdownItem"><A class="dropdownButton" href="javascript:void(0)">
                    Team
                </A><DIV class="dropdownList"><A href="https://2018.igem.org/Team:Athens/Team">Members</A><A href="https://2018.igem.org/Team:Athens/Collaborations">Collaborations</A><A href="https://2018.igem.org/Team:Athens/Attributions">Attributions</A></DIV></LI><LI class="dropdownItem"><A class="dropdownButton" href="javascript:void(0)">
                    Project
                </A><DIV class="dropdownList"><A href="https://2018.igem.org/Team:Athens/Description">Description</A><A href="https://2018.igem.org/Team:Athens/MersCov">MERS-CoV</A><A href="https://2018.igem.org/Team:Athens/Notebook">Notebook</A><A href="https://2018.igem.org/Team:Athens/Design">Design</A><A href="https://2018.igem.org/Team:Athens/Results">Results</A><A href="https://2018.igem.org/Team:Athens/Safety">Safety</A></DIV></LI><LI class="dropdownItem"><A class="dropdownButton" href="javascript:void(0)">
                    Wet Lab
                </A><DIV class="dropdownList"><A href="https://2018.igem.org/Team:Athens/Experiments">Experiments</A><A href="https://2018.igem.org/Team:Athens/InterLab">InterLab</A><A href="https://2018.igem.org/Team:Athens/Parts">Parts</A></DIV></LI><LI class="dropdownItem"><A class="dropdownButton" href="javascript:void(0)">
                    Dry Lab
                </A><DIV class="dropdownList"><A href="https://2018.igem.org/Team:Athens/Model">Toehold Design</A><A href="https://2018.igem.org/Team:Athens/Model#kinetic">Kinetic Modeling</A><A href="https://2018.igem.org/Team:Athens/Model#3d_simulations">3D Simulations</A><A href="https://2018.igem.org/Team:Athens/Model#temp_simulations">Temperature Simulations</A></DIV></LI><LI class="dropdownItem"><A class="dropdownButton" href="javascript:void(0)">
                    Human Practices
                </A><DIV class="dropdownList"><A href="https://2018.igem.org/Team:Athens/HP_Overview">Overview</A><A href="https://2018.igem.org/Team:Athens/Public_Engagement">Education &amp; Public Engagement</A><A href="https://2018.igem.org/Team:Athens/Human_Practices">Integrated Human Practices</A></DIV></LI><LI class="dropdownItem"><A class="dropdownButton" href="javascript:void(0)">
                    Product
                </A><DIV class="dropdownList"><A href="https://2018.igem.org/Team:Athens/Applied_Design">Applied Design</A></DIV></LI></UL></NAV><TITLE>Modelling</TITLE><DIV id="Athens-body"><ARTICLE><DIV id="column"><DIV class="menu"><DIV id="menu-center"><UL><LI><A href="#toehold_design"><SPAN class="bullet-minimenu">•</SPAN> Toehold Design
                                </A></LI><LI><A href="#kinetic"><SPAN class="bullet-minimenu">•</SPAN> Kinetic Modeling
                                </A></LI><LI><A href="#3d_simulations"><SPAN class="bullet-minimenu">•</SPAN> 3D Simulations
                                </A></LI><LI><A href="#temp_simulations"><SPAN class="bullet-minimenu">•</SPAN> Temperature Simulations
                                </A></LI><LI><A href="#references"><SPAN class="bullet-minimenu">•</SPAN> References
                                </A></LI></UL></DIV></DIV><H1>Modeling</H1><P>The demanding-accuracy-character of the proposed diagnostic project led to the development of models for its full extent, revolving around four main axes: design, analysis, prediction, validation. Firstly, in order to determine the trigger sequence and design the toehold switches, a computational tool was created. Scanning through conserved and unique regions of the MERS-CoV genome and taking into consideration specific thermodynamic criteria, a neural network chose the 4 most promising sequences. Secondly, a kinetics study was performed using appropriate equations, giving insight on the reaction times of the system. Thirdly, the newly proposed reporter system’s (split reporter protein interaction via coiled coils) behaviour had to be predicted via Molecular Dynamics Simulations, and compared to laboratory results. Finally, the development of a final product, a kit that implements all the processes, had to be meticulously studied. Through PDEs, optimum times were calculated ensuring optimum conditions for every step.</P><DIV id="toehold_design" class="athensSection"><H2>Toehold Design</H2><H3>Toehold Parts</H3><UL class="appliedUL"><LI><DIV><U>TBS</U>: Trigger Binding Site, a 30 nucleotides long sequence, reverse complementary to the target RNA. Target RNA will be attracted and finally bind to it.
                            </DIV></LI><LI><DIV><U>RBS</U>: Ribosome Binding Site, a 12 nucleotides long sequence, responsible for the recruitment of a ribosome during the initiation of protein translation.
                            </DIV></LI><LI><DIV><U>Linker</U>: A 21 nucleotides long sequence, connecting the reporter protein to rest of the toehold, ensuring minimum interaction between the two.
                            </DIV></LI></UL><H3>Target RNA search</H3><P>
                        The whole Toehold Switch mechanism aims at the recognition of a specific RNA sequence, the target RNA. This 30nt long sequence is part of the MERS-CoV genome. Every target RNA corresponds to a single Toehold. All the possible pairs are created and tested based on thermodynamic criteria. The first target RNA corresponds to the first 30 nucleotides, the second to nucleotides 2-31 etc. That means that practically for a genome of length N, there are N possible switches (pair of target and corresponding toehold). MERS-CoV has a length of about 30.000 nucleotides, rendering this not a negligible computational problem.
                    </P><DIV><H4>Conserved Regions</H4><FIGURE><FIGCAPTION><B>Figure 1:</B> Red lines, correspond to chosen target sequences.
                            </FIGCAPTION></FIGURE><P>
                            Different strains of viruses may present significant differences on various regions of their RNA sequence. Nevertheless, conserved regions exist and in fact are the ones targeted in classic diagnostic molecular methods, such as PCR. Consequently, search for target RNA is limited to conserved sequences and not the whole MERS-CoV genome reducing the number of switches tested to about 1000 from 30.000. The computational load is significantly decreased but so does the possibility of finding an admissible sequence. The conserved regions are determined through bibliography<SUP>[1]</SUP>.
                        </P></DIV><H4>Toehold specificity to MERS-CoV</H4><P>
                        Due to the relatively large size of the target RNA, the possibility of the exact same sequence to exist in other viruses or humans is relative small. Therefore, BLAST search is conducted only at the set of the most promising targets for time efficiency, confirming the presence of the sequence only at the MERS-CoV genome.
                    </P><H3>Thermodynamic Evaluation</H3><P>
                        Every switch needs thermodynamic evaluation to determine its ability to stay closed in absence of target and to open when the target is present. After the evaluation, every switch is scored and the 4 most promising where chosen for in vitro testing.
                    </P><H4>MFE</H4><P>
                        MFE(Minimum Free Energy) criterion, ensures the unavailability of the RBS in absence of the target RNA. Highly negative values of MFE imply great stability of the toehold thus zero expression of the reporter protein. On the contrary, values near zero indicate expression leakage, an undesirable phenomenon where reporter protein is expressed in absence of the target, resulting in false positive diagnosis.
                    </P><H4>ΔMFE</H4><P>ΔMFE criterion, ensures the binding of the target to the TBS. Two possible states exist for the system containing the target and the toehold, them binding together to form a duplex or them existing separately. ΔMFE will determine the most favorable state, in terms of thermodynamic stability. \( \Delta MFE = MFE_{target} + MFE_{toehold} – MFE_{duplex}\), the higher its value the more possible it is for the target to bind to the TBS.
                    </P><H4>ΔG</H4><P>ΔG criterion, ensures the binding of the ribosome to the RBS. Even after the initial unfolding of the switch with the binding of the target to the TBS, the hairpin at the top of the switch may remain closed, detering the binding of the ribosome. High values of ΔG (close to zero, as ΔG is a negative quantity) suggest easy unfolding of the upper hairpin and thus, RBS availability.
                    </P><H4>Switch-Switch/Target-Target</H4><P>
                        During actual implementation, the sample will contain a large number of switches. Pairs of triggers-triggers and switches-switches will be created, decreasing, possibly in a significant manner, the expression of the reporter protein. The more negative the MFE of these pairs is, the more possible this unfavorable state is.
                    </P><H3>Scoring Method</H3><P>
                        After implementation of the thermodynamic criteria, every toehold corresponds to a set of 5 energies. Those 5-dimensional data can’t give a definitive answer as to which switch is preferable, because no switch will have the most desirable values for every criterion. A score method needs to be introduced, in order to quantify the suitability of every switch. A neural network was trained to accomplish this task.
                    </P><H4>Neural Network Training</H4><P>
                        The training of a neural network requires a big amount of data. Experimental data on toehold switches’ performance are available online from Green<SUP>[2]</SUP>. Using these data, a neural network was trained with the Levenberg-Marquardt backpropagation method to correlate a set of 5 energies to a score. One layer with 5 neurons is used.
                    </P><H4>Neural Network Efficacy</H4><P>
                        The performance of the NN on the training set is more than sufficient. Nevertheless, tests must be conducted to ensure that the performance is satisfying on other data sets too. For this reason 30% of the available data weren’t used for training but as independent test, to ensure that the NN is not overfitted.
                    </P></DIV><DIV id="kinetic" class="athensSection"><H2>Kinetic Modeling</H2><P>
                        In order to gain a better understanding of the RNA-induced glucose production occurring in the cell-free system in use, we developed a simple kinetic model. In that way, we can use our insight on the detection mechanism to assist our suggested kit design. We also tried to incorporate the toehold in-silico thermodynamic evaluation to the kinetic performance of the riboregulator. We formulated a set of ODEs describing our biochemical reactions and solved them with MATLAB.
                    </P><H3>Our Aim</H3><P>
                        By applying the kinetic model that we developed, we wanted to better understand the in vitro detection mechanism itself and the course of our wet lab experiments, as well as assist the kit design process.
                    </P><H3>Brief Model Description</H3><P>
                        In our system, the trigger RNA sequence concentration [Trigger] works as the input, while the glucose levels [Glucose] as the output. The model accounts for the DNA circuit transcription (TX) to toehold-regulated trehalose-encoding mRNA, the mRNA binding to the trigger sequence, the trehalase translation (TL), the trehalose enzymatic hydrolysis towards glucose and the RNA denaturation. As suggested in the existing literature on PURE TX-TL system modeling, we assign Michaelis-Menten kinetics to the TX and TL reactions occurring, in order to take into account the observed saturation in the reaction rate<SUP>[3]</SUP>.
                    </P><P>
                        Since trehalase is reported to have an optimum pH of 5,5 and shows lower to no activity in neutral pH, we decided to split our study in two parts; during the first part the trigger is added to the cell-free system where toehold transcription, toehold-trigger binding and trehalase translation take place, while in the second part the produced trehalase is activated by lowering the pH and the glucose production occurs. Schematically, the reactions taking place are:
                    </P><H5>1<SUP>st</SUP> Part</H5><DIV style="font-size:16px">
                        \[Toehold\_DNA \overset{TX}{\longrightarrow} Toehold + Toehold\_DNA~~~~~~~(1)\] \[Activated \overset{K_d}{\longleftrightarrow} Toehold + Trigger~~~~~~~~~~~~~~~~~~~~~~~~~~(2)\] \[Activated \overset{TL1}{\longrightarrow} TreA + Activated~~~~~~~~~~~~~~~~~~~~~~~~~~~~~(3)\] \[Toehold \overset{TL2}{\longrightarrow} TreA + Toehold~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~(4)\] \[Toehold \longrightarrow \varnothing~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~(5)\] \[Trigger \longrightarrow \varnothing~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~(6)\]
                    </DIV><H5>2<SUP>nd</SUP> Part</H5><DIV style="font-size:16px">
                        \[TreA + Trehalose \overset{M.M}{\longrightarrow} TreA + 2 \cdot Glucose ~~~~~~~~~~~~~~~~(7)\]
                    </DIV><H4>Transcription reactions (1)</H4><DIV style="font-size:16px">
                        \[r_{TX} = \frac{k_1^{TL}\cdot[Toehold\_DNA]}{k_m^{TL} + [Toehold\_DNA]}\]
                    </DIV><P>
                        The circuit DNA TX follows Michaelis-Menten kinetics. The constants’ values used are found in literature<SUP>[3, 4]</SUP>. The suggested \( k_1^{TL}\) value for GFP encoding DNA transcription is 18,2nM/min<SUP>[4]</SUP>. Since the trehalase encoding gene is approximately 2,2 –fold bigger in size than the one encoding GFP, we assumed that the TX speed is accordingly slower. The \( K_m^{TX}\) value used
                        <SUP>[3]</SUP>, is calculated in a PURE system for GFP transcription. Using the same system for our experiments and assuming that there is no strict correlation between the gene type and the transcription enzymes affinity, we can use the same value for TreA-gene transcription.
                    </P><H4>Translation reactions (3) and (4)</H4><DIV style="font-size:16px">
                        \[r_{TX} = \frac{k_1^{TL}\cdot[Toehold]}{k_{m_1}^{TL} + [Toehold]}\] \[r_{TL1} = \frac{k_2^{TL}\cdot[Activated]}{k_{m_2}^{TL} + [Activated]}\]
                    </DIV><P>
                        Similarly, the RNA translation step follows Michaelis-Menten Kinetics. We assume that the activated dimer’s translation constant \( k_2^{TL}\) is 2.2-fold smaller than the corresponding constant for the GFP mRNA<SUP>2</SUP>. The toehold TL constant is initially estimated to be \( k_1^{TL} \approx \frac{k_2^{TL}}{40}\). We take the assumption that the TL Michaelis-Menten constant corresponds to the ribosome binding affinity to the mRNA molecule, thus is independent to the protein type. The constant also has the same value for the activated dimer and the toehold<SUP>[3]</SUP>: \( k_{m_1}^{TL} = k_{m_2}^{TL} = k_m^{TL} \)
                    </P><H4>Activation reaction (2)</H4><P>We suppose that the toehold- trigger activation reaction quickly reaches the equilibrium state<SUP>[3]</SUP>:</P><DIV style="font-size:16px">
                        \[K_d = \frac{[Toehold]\cdot[Trigger]}{Activated}\]
                    </DIV><P>
                        Where \( K_d \) is the activated dimer’s dissociation constant. Its value is thermodynamically estimated a priori:
                    </P><DIV style="font-size:16px">
                        \[\Delta G^o = - RTlnK_d\]
                    </DIV><P>
                        Where the \(\Delta G^o\) value, corresponds to the difference in the Minimum Free Energy values of the molecules in the two states and in 25 <SUP>o</SUP>C<SUP>[5]</SUP>:
                    </P><DIV style="font-size:16px">
                        \[\Delta G^o = (MFE_{toehold} + MFE_{trigger}) - MFE_{activated}\]
                    </DIV><P>
                        Initially, we believed that such an assumption could provide us a reliable value for the dissociation constant. Dr Estevez-Torres, suggested that we revise that assumption, because despite its simplicity it might be proven insufficient. For that reason, we compared values thermodynamically derived for toehold/trigger pairs that have been tested in PURE TX-TL systems by Senoussi <I>et.al</I>. with the experimentally derived values<SUP>[3]</SUP>. Since the differences were substantial, we decided to initiate a correction multiplication factor that links the thermodynamic estimates to reasonable kinetic constants:
                    </P><DIV style="font-size:16px">
                        \[K_d^* = \alpha \cdot K_d\]
                    </DIV><P>
                        The RNA molecules simultaneously co-exist in many different possible conformations. This explains why the Minimum Free Energy value cannot be used alone for the equilibrium constant estimation.
                    </P><H4>Denaturation/Deactivation reactions (5) and (6)</H4><P>
                        We assume that only the RNA species are susceptible to denaturation during the detection test duration. The mRNA follows a first order denaturation kinetics<SUP>[4]</SUP>.
                    </P><DIV style="font-size:16px">
                        \[r_d = - k_d \cdot [mRNA]\]
                    </DIV><H4>Trehalose Hydrolysis (7)</H4><P>
                        Trehalose enzymatic hydrolysis to glucose follows Michaelis-Menten kinetics. In order to perform our calculations, we used the Michaelis-Menten kinetic constants, as calculated by Gibson <I>et.al</I><SUP>[6]</SUP>:
                    </P><DIV style="font-size:16px">
                        \[r_{M.M} = 2 \cdot \frac{k_{cat}\cdot TreA \cdot [Trehalose]}{K_m+[Trehalose]}\]
                    </DIV><TABLE id="kinetics-constants" style="margin:0 auto"><CAPTION>The constants used in the model</CAPTION><TBODY><TR><TH>Constant</TH><TH>Value</TH></TR><TR><TD>\( k_1^{TX}\)</TD><TD>0.138 nM/s</TD></TR><TR><TD>\( K_m^{TX}\)</TD><TD>4.2 nM</TD></TR><TR><TD>\( k_1^{TL}\)</TD><TD>0.003 nM/s</TD></TR><TR><TD>\( k_2^{TL}\)</TD><TD>0.122 nM/s</TD></TR><TR><TD>\( k_m^{TL}\)</TD><TD>265 nM</TD></TR><TR><TD>\( R \)</TD><TD>1.985∙10<SUP>-3</SUP> kcal∙mol<SUP>-1</SUP>∙K<SUP>-1</SUP></TD></TR><TR><TD>\( T \)</TD><TD>310 K</TD></TR><TR><TD>\( a \)</TD><TD>0.11</TD></TR><TR><TD>\( k_d \)</TD><TD>7.8∙10<SUP>-4</SUP> min<SUP>-1</SUP></TD></TR><TR><TD>\( K_{cat} \)</TD><TD> 199<SUP>-1</SUP></TD></TR><TR><TD>\( K_m \)</TD><TD>0.41 nM</TD></TR></TBODY></TABLE><H3>The in vitro mechanism</H3><P>
                        For the in vitro toehold testing we assume that the trigger RNA and the toehold DNA sequences are simultaneously added to the system (time = 0 min). In the following diagrams we present the cell free translation-transcription reactions’ propagation for different Trigger concentrations. After a time course of 60 minutes, we assume an instant pH drop from 7 to 5,5 whereas the trehalose hydrolysis occurs .
                    </P><FIGURE><FIGCAPTION><B>Figure 2:</B> Activated dimer’s concentration for different trigger concentrations over a time period of 60 minutes; From top to bottom the concentration curves correspond to initial trigger concentrations of 10<SUP>-5,00</SUP>M, 10<SUP>-6,00</SUP>M, 10<SUP>-7,00</SUP>M. The initial Toehold DNA concentration is [Toehold_DNA] = 3,00∙10<SUP>-9,00</SUP>M.
    </FIGCAPTION></FIGURE><FIGURE><FIGCAPTION><B>Figure 3:</B> Final trehalase levels over a time period of 60 minutes for different Trigger concentrations.
                        </FIGCAPTION></FIGURE><FIGURE><FIGCAPTION><B>Figure 4:</B> Trehalase concentration for three different trigger concentrations. From top to bottom the concentration curves correspond to initial trigger concentrations of 10<SUP>-5,00</SUP>M, 10<SUP>-6,00</SUP>M, 10<SUP>-7,00</SUP>M.
                        </FIGCAPTION></FIGURE><FIGURE><FIGCAPTION><B>Figure 5:</B> Glucose concentration over a time period of 5 minutes after the pH drop to 5,5. The curves correspond to different concentrations of Trigger RNA in the sample. From left to right: 10<SUP>-5,00</SUP>M, 10<SUP>-6,00</SUP>M, 10
                            <SUP>-7,00</SUP>M, 0 M. The Glucose levels in trigger absence are due to transcription of the Toehold molecule and are reported as expression leakage. The samples that contain the RNA trigger sequence, are predicted to surpass glucometer typical threshold levels, whereas they exceed the levels of glucose in blood in less than a minute. The sample without trigger is predicted to reach the reported glucometer threshold after 10 minutes.
                        </FIGCAPTION></FIGURE><FIGURE><FIGCAPTION><B>Figure 6:</B>Trehalose concentration over a time period of 5 minutes after the pH drop to 5,5. The curves correspond to different concentrations of Trigger RNA in the sample. From bottom to up: 10
                            <SUP>-5,00</SUP>M, 10<SUP>-6,00</SUP>M, 10<SUP>-7,00</SUP>M.
                        </FIGCAPTION></FIGURE><H3>Kit Design</H3><P>
                        The kinetic modeling of the kit set as a primary goal the definition of the time spans of each one of the test steps. Since we did not have enough data to model the kinetics of the Recombinase Polymerase Amplification (RPA), our study focused on the post-amplification steps. We assumed that after the amplification the expected Trigger sequence concentration may vary between 10 nM - 10 μΜ. The amplification step is set to 30 minutes, as to ensure that the input trigger concentration for the cell-free reactions will lie between the above-mentioned range.
                    </P><P>
                        After a series of meetings
                        (Integrated HP) we concluded on setting the 1 hour as the upper limit for the whole detection process, since lengthier applications may be proven unsuitable for PoC applications.
                    </P><P>
                        As Professor Keith Pardee suggested, we run the amplification and the DNA-circuit-to-toehold transcription in different compartments because they are shown to be incompatible. In order to save time, we run these reactions in parallel, supposing that they instantly mix after 30 minutes. The initial Toehold-DNA concentration in all the cases is assumed to be 3 nM.
                    </P><P>
                        Both the TX-TL reaction and the Enzymatic Hydrolysis have to take place in a total of 30 minutes. In order to decide the division between the two steps, we expressed the final glucose levels as a function of the TX-TL step, which produces the hydrolytic enzyme- Trehalase.
                    </P><FIGURE><FIGCAPTION><B>Figure 7:</B>Trehalase production during a TX-TL 20 min step in absence of the trigger sequence.
                        </FIGCAPTION></FIGURE><FIGURE><FIGCAPTION><B>Figure 8:</B> Glucose production during the trehalose hydrolysis preceding step corresponding to trigger absence.
                        </FIGCAPTION></FIGURE><P>
                        During this approach, we set the value of 4 mmol/L glucose as the viral detection threshold, since this value corresponds to typical fasting levels in blood for a healthy adult, targeting to such a value, we ensure that it will be displayed by the glucometer regardless its type and detection range.
                    </P><FIGURE><FIGCAPTION><B>Figure 9:</B> Glucose levels as a function of the TX-TL step duration. Total test time is set to 1 hour.
         </FIGCAPTION></FIGURE><P>
                        In order to reach final glucose levels above the aforementioned glucose limit, the duration of the TX-TL step should lie between 8,8 min - 21 min. While our first pick was the 15 min duration, where the glucose levels reach a pick, we chose to set the TX-TL reaction in 20 minutes, in order to take into account possible negative feedback because of the mixing with the amplification reactants. It is worth mentioning, that the use of round time fractions for each step is essential in order to facilitate the kit operation.
                    </P><P>
                        With that being said, The TX-TL step is set in 20 minutes and the enzymatic reaction step in 10 minutes:
                    </P><FIGURE><FIGCAPTION><B>Figure 10:</B>Trehalase production during the designed TX-TL step. The initial trigger concentration is set to [Trigger] = 10 nM.
                        </FIGCAPTION></FIGURE><FIGURE><FIGCAPTION><B>Figure 11:</B>Glucose production during the enzymatic step. The final glucose concentration exceeds the target threshold-value.
                        </FIGCAPTION></FIGURE></DIV><DIV id="3d_simulations" class="athensSection"><H2>3D Simulations</H2><H3>Protein Complex Simulation</H3><P>
                        Our main goal for performing the Molecular Dynamics simulation of the <I>Split Trehalase Reporter System</I> was to develop a model in order to observe the protein interaction in a solution and, thus, have an insight on the system’s behaviour.
                    </P><DIV><U style="text-align: center; font-size: 16.65px">Homology Modeling</U><P>
                            In order to simulate how the two parts of trehalase assemble, when each of them is merged with half a <B>leucine zipper</B>, we first needed to create a model for both parts. Trehalase’s three-dimensional structure, has been <B>crystallographically</B> determined and the leucine zipper is quite a common <B>structural motif</B> in proteins, so it was easy to create models using HHpred and MODELLER<SUP>[7]</SUP>.
                        </P><P>
                            HHpred is a fast server for <B>remote protein homology detection</B> and
                            <B>structure prediction</B> and implements pairwise comparison of profile hidden Markov models (HMMs)
                            <SUP>[8]</SUP>. HHpred’s results can be easily forwarded to MODELLER, which is, as the name would suggest, used for <B>
                            homology or comparative
                            modeling
                        </B> of protein three-dimensional structures
                            <SUP>[9]</SUP>.
                        </P><P>
                            Below, you can find listed the sequences that were used for the creation of the models, the PDB structures that HHpred proposed and we selected for the creation of the models and the respective info and statistics:
                        </P><OL class="numberedAth-list" style="list-style-type:decimal"><LI><DIV><B>TreA-a part</B> (translated by the toehold switch), contains: ATG + Toehold Linker + Zipper + Zipper Linker + TreA-a (see Design section for more information.
                                </DIV></LI><LI><DIV>
                                            HHpred selected results:
                                            <FIGURE><FIGCAPTION><B>Figure 12</B>: On the top, the sequence of the junk amino acids and the leucine zipper part are depicted in grey, while the trehalase part is depicted in blue. Below, the PDB structures are depicted in red or orange. The length of each line indicates the part of the sequence that each structure covers on the model. In the middle of each line, the structure’s PDB ID is indicated with white letters.
                                                </FIGCAPTION></FIGURE></DIV></LI><LI><DIV>
                                            Info and statistics:
                                            <TABLE style="margin: 0 auto;"><TBODY><TR><TH>PDB ID</TH><TH>Probability(%)</TH><TH>e-value</TH></TR><TR><TD><A href="http://www.rcsb.org/pdb/explore/explore.do?structureId=2JG0">2JG0_A</A></TD><TD>99.49</TD><TD>1.6e-16</TD></TR><TR><TD><A href="https://www.rcsb.org/structure/5N6N">5N6N_C</A></TD><TD>97.8</TD><TD>2e-7</TD></TR><TR><TD><A href="https://www.rcsb.org/structure/4DZN">4DZN_B</A></TD><TD>97.24</TD><TD>0.000016</TD></TR><TR><TD><A href="https://www.rcsb.org/structure/4IFF">4IFF_C</A></TD><TD>91.13</TD><TD>0.69</TD></TR></TBODY></TABLE></DIV></LI><FIGURE><FIGCAPTION><B>Figure 13</B>: Predicted structure of TreA-a part along with its zipper, the long purple a-helical part. Visualization enabled via the VMD package.
                                    </FIGCAPTION></FIGURE><LI><DIV><B>TreA-b part</B> contains: Zipper + Zipper Linker + TreA-b (see Design section for more information)
                                </DIV></LI><LI><DIV>
                                            HHpred selected results:
                                            <FIGURE><FIGCAPTION><B>Figure 14</B>: On the top, the input sequence is depicted in blue. Below, the PDB structures are depicted in red. The length of each line indicates the part of the sequence that each structure covers on the model. In the middle of each line, the structure’s PDB ID is indicated with white letters.
                                                </FIGCAPTION></FIGURE></DIV></LI><LI><DIV>
                                            Info and statistics:
                                            <TABLE style="margin: 0 auto;"><TBODY><TR><TH>PDB ID</TH><TH>Probability(%)</TH><TH>e-value</TH></TR><TR><TD><A href="https://www.rcsb.org/structure/2JG0">2JG0</A></TD><TD>100</TD><TD>1.3e-66</TD></TR><TR><TD><A>2OKX_B</A></TD><TD>99.96</TD><TD>4.9e-31</TD></TR></TBODY></TABLE></DIV></LI></OL><FIGURE><FIGCAPTION><B>Figure 15</B>: Predicted structure of TreA-b part along with its zipper, the purple a-helical part observed on the right. Visualization enabled via the VMD package
                            </FIGCAPTION></FIGURE></DIV><DIV><U style="text-align: center; font-size: 16.65px">Molecular Dynamics Simulations</U><P>
                            A molecular dynamics (MD) simulation for the aforementioned complex of the Split Trehalase Reporter System was performed using the <I>GROMACS</I> packageup
                            <SUP>[10][11]</SUP>, taking into account the MD simulations performed by Aronsson, <I>et al. 2012</I><SUP>[12]</SUP>. The <B>stages</B> of the simulation, observations as well as comments are mentioned in the “Information &amp; Comments” section below.
                        </P><P><B>Note 1</B>: The NAMD package could also be used, if one is more familiar with it. However, GROMACS is preferred for protein simulations, as it displays better performance.
                        </P><P><B>Note 2</B>: Because of the size of the system and the amount of time needed to produce promising results, the whole process run on the national supercomputer, <B>ARIS</B> of the <B>Greek Research &amp; Technology Network (GRNET)</B>.
                        </P><P><U>Information &amp; Comments:</U></P><OL class="numberedAth-list" style="list-style-type:decimal"><LI><DIV>
                                    In order to facilitate the process, the two protein parts were brought to an 8-12Å distance from each other, with appropriate orientation. This was mainly done so that the size of the water box (discussed below) would be manageable. This means that the solvated system would contain a reasonable number of <B>atoms</B> that the computer could process in a relatively rational amount of time. At the same time, it should be mentioned that the position and orientation of the proteins could, but it is not necessary, aid their interaction.
                                </DIV></LI><LI><DIV><B>Force Field</B>: The <I>CHARMM36</I><SUP>[13][14]</SUP> force field was implemented for the peptide and ions parameters.
                                </DIV></LI><LI><DIV><B>Water box, Solvation, Charge</B>: A <I>rhombic dodecahedron</I> box was chosen, with a periodic distance equal to the one of a cubic box but with <B>~71%</B> its volume, ensuring minimal number of atoms of the solvated system. The three-point <I>TIP3P</I><SUP>[15]</SUP> water model was used. The total charge of the system was set to <B>neutral</B> by adding Natrium ions.
                                </DIV></LI><LI><DIV><B>Other Details</B>: <I>Periodic boundary</I> conditions (pbc = xyz) were implemented on all stages. Regarding the electrostatics, the <I>particle-mesh-Ewald</I> method was chosen, setting the grid length (
                                    <I>fourierspacing</I>) equal to <B>0.16nm</B>, with a <B>1.2nm</B> cut-off. A cut-off of <B>1nm</B> was included for the Van der Waals interactions.
                                </DIV></LI><LI><DIV><B>Minimization</B>: A <I>steepest descent algorithm</I> was used for energy minimization, with a maximum force tolerance (<I>emtol</I>) of 1000.0kJ mol<SUP>-1</SUP> nm<SUP>-1</SUP>. As shown on the diagram below, the system’s energy is minimized in less than <B>1.5ns</B>.
                                </DIV><FIGURE><FIGCAPTION><B>Figure 16</B>: Potential energy vs. Time for the duration of the Energy Minimization process. Plot obtained via XMGrace.
                                    </FIGCAPTION></FIGURE></LI><LI><DIV><B>Temperature Equilibration (NVT)</B>: The temperature of the peptides and solvent was kept constant at
                                    <B>310K</B>, at the temperature our real-life system is expected to work, with <B>V-rescale</B> as the chosen thermostat. Each group of molecules (<I>Protein, Water_and_ions</I>) was coupled independently. The temperature time constant was <B>0.1ps</B>. <B>Position restraints</B> were implemented for the protein, while the
                                    <B>LINCS constraints algorithm</B> was used for <I>h-bonds constraints</I>.The integration of the equations of motion was performed using a <I>lead frog algorithm (md)</I> with a time step of 2fs, for a total time of <B>10ns</B>.
                                </DIV><FIGURE><FIGCAPTION><B>Figure 17</B>: Temperature vs. Time for the duration of the NVT Equilibration process. Plot obtained via XMGrace.
                                    </FIGCAPTION></FIGURE></LI><LI><DIV><B>Pressure Equilibration (NPT)</B>: Temperature coupling was on during this stage, using V-rescale thermostat, as described above. For pressure coupling, the <B>Berendsen barostat</B> was chosen, which is best for the equilibration stages. The reference pressure was
                                    <B>1.0bar</B>, the pressure time constant was
                                    <B>2ps</B>, while the compressibility was
                                    <B>4.5e-5</B> (appropriate value for T=310K or T=300K and p=1.0bar).
                                    <B>Position restraints</B> were implemented for the protein, while the
                                    <B>LINCS constraints algorithm</B> was used for h-bonds constraints. The integration of the equations of motion was performed using a lead frog algorithm (md) with a time step of
                                    <B>2fs</B>, for a total time of
                                    <B>50ns</B>. <I><U>Suggestion</U></I>: A suggestion our team received was to pressure equilibrate in
                                    <B>two steps</B>, one with position restraints and one without, and even reducing the total NPT simulation time down to 10ns. The suggestion was tested for reference on a different system, showing good results.
                                </DIV><FIGURE><FIGCAPTION><B>Figure 18</B>:Pressure vs. Time for the duration of the NVT Equilibration process. The fluctuations of pressure are to be expected, with the running average fluctuates to a less degree. Plot obtained via XMGrace.
                                    </FIGCAPTION></FIGURE><FIGURE><FIGCAPTION><B>Figure 19</B>: Density vs. Time for the duration of the NVT Equilibration process. The density converges nicely, with little fluctuations that are to be expected. Plot obtained via XMGrace.
                                    </FIGCAPTION></FIGURE></LI><LI><DIV><B>Production Run</B> Temperature and Pressure coupling were on during this stage, with a V-rescale thermostat and a
                                    <B>Parrinello-Rahman</B> barostat. The system was set free of any position restraints. The
                                    <B>LINCS constraints algorithm</B> was used for h-bonds constraints. The integration of the equations of motion was performed using a lead frog algorithm (md) with a time step of
                                    <B>2fs</B>, for a total time of <B>200ns.</B></DIV></LI><LI><DIV><B>Analysis</B>: Below the trajectory of the protein complex is showed, for a total duration of 100ns (the last 100ns of the simulation).
                                </DIV></LI></OL><FIGURE><FIGCAPTION><B>Animation 1</B>: Split trehalase system simulation, trajectory analysis of the last 100ns of the Production Run. Towards the end of the 100ns, the protein molecules approach each other in the center of the box, a promising result for their in between interactions.
                            </FIGCAPTION></FIGURE><P>
                            One of the first thing one notices in the movie above is that the protein molecules drift away from each other and exit the water box. However, this is not entirely true. As mentioned, Periodic Boundary Conditions (pbc) were implemented in the system. That means that when a protein “exits” from the right, its periodic image will “enter” from the left, <B>as protein molecules are free to diffuse in the assigned waterbox</B>. For example:
                        </P><DIV><FIGURE style="display: inline-block; vertical-align: middle;"><FIGCAPTION><B>Figure 20</B>: (a) Split trehalase complex, frame 83 of the trajectory analysis, dodecahedron box, pbc: off, (b) Split trehalase complex, frame 83 of the trajectory analysis, dodecahedron box, pbc: on. The diffusion of the protein molecules play a major role in their orientation inside the water box and in their in between interaction. Visualization enabled via the VMD package.
                                </FIGCAPTION></FIGURE></DIV><P>
                            So, by enabling periodic boundary conditions on the visualization software, one can observe that the proteins actually do approach each other. However, what we have not observed yet, is the leucine zippers interaction.
                        </P><FIGURE><FIGCAPTION><B>Figure 21</B>: Split trehalase complex, frame 90 of the trajectory analysis, dodecahedron box. The protein molecules have approached each other in the center of the waterbox.
                            </FIGCAPTION></FIGURE></DIV><H3>Observations</H3><P>
                        As expected, the proteins move around in the water box. Thus, their interaction relies mostly on diffusion terms, on their structure, their polarity and size. In order to observe a significant result, the Production Run stage of the simulation has to run for a <B>significant amount of time</B>, where a <B>500ns total duration</B> is estimated to produce better results. However, for the total time of the simulation, one can observe that the two molecules do approach each other, then drift apart, then approach again with different orientations, indicating the expected interaction.
                    </P><H3>Future Plans</H3><UL class="appliedUL"><LI><DIV>
                                In order to better observe the interaction of the two leucine zippers and the protein parts, a <I>Steered Molecular Dynamics Simulation</I> could be performed. Through the Steered Molecular Dynamics Simulation, we could also obtain a better estimate of the Gibbs Free Energy and subsequently the equilibrium constant Keq.
                            </DIV></LI><LI><DIV>
                                Using the <B>Protein-Protein Docking</B> approach prior to the MD Simulation, via
                                <I>FRODOCK</I><SUP>[16]</SUP> or <I>HADDOCK</I><SUP>[17]</SUP> servers, a comparison to the method described above can be made. This way, one can determine whether docking is a promising preparatory method for a protein complex prior to the MD Simulation.
                            </DIV></LI><LI><DIV>
                                Due to software limitations, <B>RNA 3D Simulations</B> are very difficult to perform. It is our team’s desire to develop a <I>3D model for our RNA toehold switches</I>, when the necessary softwares become available.
                            </DIV></LI></UL></DIV><DIV id="temp_simulations" class="athensSection"><H2>Temperature Simulations</H2><FIGURE><FIGCAPTION><B>Figure 22</B>: Half symmetric part of the geometry, with visable mesh.
                        </FIGCAPTION></FIGURE><P>
                        For the steps of viral RNA amplification and transcription-translation of the cell-free system, a constant temperature of 37<SUP>o</SUP>C needs to be achieved. Positive Temperature Coefficient (PTC) thermistors are going to be used for that purpose. Simulations need to be conducted in order to determine the time needed for the kit temperature to reach steady state conditions. A 3D model was formulated, and time dependent simulations revealed the requested time
                    </P><FIGURE><FIGCAPTION><B>Animation 2</B>: A simulation of temperature distribution through time.
                        </FIGCAPTION></FIGURE><P>
                        Temperature is monitored in the center of the large paper disc. The results shown on the image below are used to calculate the steady-state time. It is obvious that the temperature profile is steady after 10-15 minutes, thus 10 minutes are deemed enough as a waiting time before the start of step 3 (RNA amplification). Furthermore, using the same diagram we investigate the solution dependency on the used mesh. The blue line (Original Mesh, 18.000 elements) and the red circles (Finer Mesh, 45.000 elements) completely overlap, so we deem that the normal mesh is sufficient enough and more practical, thus it is going to be used for the further calculations needed.
                    </P><FIGURE><FIGCAPTION><B>Figure 23</B>: Temperature profile for the first 30 minutes. Mesh convergence investigation.
                        </FIGCAPTION></FIGURE><P>
                        Heater position is of great importance for the development of the temperature profile. A parametric study was conducted to obtain the most favorable, in terms of fast achievement of steady-state conditions, position relevant to the initially suggested. All the results presented used this optimum position.
                    </P><FIGURE><FIGCAPTION><B>Figure 24</B>: Parametric investigation for determination of optimum heater position.
                        </FIGCAPTION></FIGURE><P>
                        Humidity of the paper may play a significant role at the heat transfer inside the paper and consequently at the time needed for reaching steady-state conditions. Humidity affects the heat transfer coefficient (k), but the correlation of the two can’t be described adequately due to the bizarre nature of the liquid. Instead we conducted a parametric investigation with respect to the heat transfer coefficient directly. The results obtained clearly indicate that the 10 minutes calculated in the beginning are enough no matter the value of the heat transfer coefficient.
                    </P><FIGURE><FIGCAPTION><B>Figure 25</B>: Parametric investigation with respect to the heat transfer coefficient, for determination of the steady-state time needed.
                        </FIGCAPTION></FIGURE><P>
                        Based on the above investigation, a time of 10 minutes is needed after step 2 in order to achieve a constant temperature of 37<SUP>o</SUP>C, required in step 3 (RNA amplification). Step 4 (transcription-translation of the cell-free system) also requires a constant temperature of 37<SUP>o</SUP>C, but this time the temperature is already achieved so zero lag time is required.
                    </P></DIV><DIV id="references" class="athensSection"><H2>References</H2><DIV style="text-align:justify">
                        [1]Corman VM, Müller MA, Costabel U, Timm J, Binger T, Meyer B, Kreher P, Lattwein E, Eschbach-Bludau M, Nitsche A, Bleicker T, Landt O, Schweiger B, Drexler JF,Osterhaus AD, Haagmans BL, Dittmer U, Bonin F, Wolff T &amp; Drosten C., “Assays for laboratory confirmation of novel human coronavirus (hCoV-EMC) infections”, EuroSurveill,<B>17</B>(49):pii: 20334, (2012).
                         [2]Green AA, Silver PA, Collins JJ &amp; Yin P., “Toehold switches: De-novo-designed regulators of gene expression”, <I>Cell</I>,
                        <B>159</B>(4):925–39, (2014).
                         [3]Senoussi, A. et al. Quantitative characterization of translational riboregulators using an in vitro transcription-translation system. bioRxiv 290403 (2018). doi:10.1101/290403.
                         [4]Stögbauer, T., Windhager, L., Zimmer, R. &amp; Rädler, J. O. Experiment and mathematical modeling of gene expression dynamics in a cell-free system. Integr. Biol. <B>4</B>, 494 (2012).
                         [5]Zadeh, J. N. et al. NUPACK: Analysis and design of nucleic acid systems. J. Comput. Chem. <B>32</B>, 170–173 (2011).
                         [6]Gibson, R. P. et al. Molecular Basis for Trehalase Inhibition Revealed by the Structure of Trehalase in Complex with Potent Inhibitors. Angew. Chemie <B>119</B>, 4193–4197 (2007).
                         [7] Zimmermann, L., Stephens, A., Nam, S.-Z., Rau, D., Kübler, J., Lozajic, M., … Alva, V. (2018). A Completely Reimplemented MPI Bioinformatics Toolkit with a New HHpred Server at its Core. Journal of Molecular Biology, 430(15), 2237–2243. doi:10.1016/j.jmb.2017.12.007.
                         [8] Soding, J., Biegert, A., &amp; Lupas, A. N. (2005). The HHpred interactive server for protein homology detection and structure prediction. Nucleic Acids Research, 33(Web Server), W244–W248. doi:10.1093/nar/gki408.
                         [9] Webb, B. and Sali, A. 2016. Comparative protein structure modeling using MODELLER. Curr. Protoc. Protein Sci. 86:2.9.1-2.9.37. doi: 10.1002/cpps.20
                         [10] Hess, B., Kutzner, C., van der Spoel, D., Lindahl, E. GROMACS 4: Algorithms for Highly Efficient, Load-Balanced, and Scalable Molecular Simulation. J. Chem. Theory Comput. 4(3):435–447, 2008.
                         [11] M.J. Abraham, D. van der Spoel, E. Lindahl, B. Hess, and the GROMACS development team, GROMACS User Manual version 2018.3, www.gromacs.org (2018)
                         [12] Aronsson, C., et.al. Self-sorting heterodimeric coiled coil peptides with defined and tuneable self-assembly properties. Scientific Reports, <B>5</B>. doi:10.1038/srep14063 (2015).
                         [13] Huang, J., Rauscher, S., Nawrocki, G., Ran, T., Feig, M., de Groot, B. L., … MacKerell, A. D.. CHARMM36m: an improved force field for folded and intrinsically disordered proteins. Nature Methods, 14(1), 71–73. doi:10.1038/nmeth.4067 (2016).
                         [14] Best, R. B., Zhu, X., Shim, J., Lopes, P. E. M., Mittal, J., Feig, M., &amp; MacKerell, A. D.. Optimization of the Additive CHARMM All-Atom Protein Force Field Targeting Improved Sampling of the Backbone ϕ, ψ and Side-Chain χ1 and χ2 Dihedral Angles. Journal of Chemical Theory and Computation, 8(9), 3257–3273. doi:10.1021/ct300400x (2012).
                         [15] Jorgensen, W. L., Chandrasekhar, J., Madura, J. D., Impey, R. W., &amp; Klein, M. L. . Comparison of simple potential functions for simulating liquid water. The Journal of Chemical Physics, 79(2), 926–935. doi:10.1063/1.445869 (1983).
                         [16] E. Ramírez-Aportela, J.R. López-Blanco, and P. Chacón (2016). FRODOCK 2.0: Fast Protein-Protein docking server. Bioinformatics, 32(15), 2386-2388.
                         [17] Cyril Dominguez, Rolf Boelens and Alexandre M.J.J. Bonvin. HADDOCK: a protein-protein docking approach based on biochemical and/or biophysical information.J. Am. Chem. Soc. <B>125</B>, 1731-1737 (2003).
                        </DIV></DIV></DIV></ARTICLE><DIV id="Athens-footer"><DIV id="column"><H3>Contact Info:</H3><ADDRESS id="athEmail"><A href="mailto:igemathens2018@gmail.com">igemathens2018@gmail.com</A></ADDRESS></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></BODY></HTML>