"
Page
Discussion
View source
History
teams
Log in
 
Team:Frankfurt/Notebook/Protocol
From 2013.igem.org
Team: iGEM Frankfurt - 2012.igem.org
Home
Team
Who we are
Official Team Profile
Contact us
Project
Description
Methods
Results
Biobricks
Perspectives
Organisms
Notebook
Protocols
Labwork
Safety
Human Practices
Night of Science
Attributions
Contents
1Methods and Protocols
1.1Plasmid Preparation
1.1.1Plasmid Preparation from E.coli (Mini Preparation)
1.1.2Plasmid Preparation from Saccharomyces cerevisia
1.2Transformation
1.2.1Yeast Transformation
1.2.2Yeast transformation and gap repair (example)
1.2.3General procedure of a transformation via gap repair
1.2.4E.coli Transformation
1.3PCR
1.4Restriction Digestion for Linearization of a Plasmid
1.5Culture Media
1.6Agar Plate
1.6.1LBampicilline-Agar
1.6.2LBchloramphenicol-Agar
1.6.3SCD-Agar
1.6.4YEPDG418-Agar
1.7Gel Electrophoresis
1.8Sample preperation for GC
1.9Kits
Methods and Protocols
Plasmid Preparation
Plasmid Preparation from E.coli (Mini Preparation)
transfer 1,5 ml of an overnight culture in a reaction tube
centrifuge at 8000 rpm at RT for 5 min
resuspend the pellet in 100 µl solution 1
addition of 200 µl solution 2
mix till the solution is clear
incubate at RT for 5 min
addition of 150 µl cold solution 3, mix till protein clumps are build
incubate 10 min on ice, centrifuge 15 min with 10000 rpm
transfer the supernatant in a clean reaction tube
fill the reaction tube with 96 % EtOH, mix and let it precipitate at -20 °C for 10 min
centrifuge 10 min at 10000 rpm
wash the pellet with 70 % EtOH
dry the pellet and resuspend it with 30 µl water or TE buffer
Solution 1
50 mM glucose
10 mM EDTA
25 mM Tris-HCl pH8
Solution 2
0,2 M NaOH
1 % SDS
Solution 3
3 M KaAc pH 5,5
Plasmid Preparation from Saccharomyces cerevisia
overnight culture in 5 ml
centrifuge 2 ml of the cells for 1-2 min
wash the cells  with water
resuspend the pellet in 400 µl buffer 1 with RNase
addition of 400 µl buffer 2 and mix carefully
addition of 2/3 volume of glass beads
cell destruction: vibrax the cellsin a 2 ml reaction tube for 5 min at 4°C
transfer 500 µl of the supernatant in a clean reaction tube
addition of 250 µl buffer 3, mix, incubate it for 10 min on ice
centrifuge at 10000 rpm, 15 min
transfer the supernatant in a clean reaction tube, fill with isopropanol and mix
centrifuge at 13000 rpm, 30 min
wash the pellet with cold 70 % EtOH and let it dry
resuspend the DNA in 30 µl water or TE buffer
P1
50 mM Tris/HCl pH 8
10 mM EDTA
100 µg/ml RNase A
P2
0,2 M NaOH
1 % SDS
P3
3 M KAc
Transformation
Yeast Transformation
Inoculate the synthetic complete medium (SC) with the strain
Incubate with shaking overnight at 30°C
Harvest the cells at a OD600 0,5-0,6 by centrifugation (3000x g, 5 min, RT)
Wash the cells with 0,5 vol of sterile water (resuspend by shaking, centrifugate with 3000x g, 5 min, RT)
Resuspend the cells in 0,01 vol of sterile water, transfer the suspension to a reaction tube and pellet the cells (3000x g, 5 min, RT)
Resuspend the pellet in 0,01 vol of sterile filtrated FCC (frozen competent cell) solution
Aliquot 50 µl of the solution into the reaction tubes
Store the cells at -80°C for at least one night (up to one year)
Mastermix for the FCC transformation mixture:
Substance
Volume [µl]
PEG 3350 (50% (w/v))
260
LiAcetat 1.0 M
36
Single-stranded carrier DNA (10 mg/ml)
10
Total volume
306
Prepare DNA aliquots: Solute enough DNA (e.g. 100 ng plasmid) in 54 µl of water
Unfreeze the cells in a 37°C block for 15-30 sec
Centrifuge the solution at 13000x g for 2 minutes
Remove the supernatant
Add 306 µl of FCC transformation mixture to the cells
Add 54 µl of the DNA to the solution and vortex shortly
When all the reaction tubes are prepared, vortex the samples well until all pellets are completely resoluted
Incubate the samples for 40 minutes at 42°C in a heating block
Centrifuge the cells at 13000x g for 30 sec and pour off the supernatant
Resuspend the cells in sterile water by vortexing
Spread onto the appropriate medium
Let the cells grow at 30°C
Yeast transformation and gap repair (example)
DNA Fragment
Size[bp]
Concentration[ng/µl]
Equimolar Quantities°[ng]
Used Volume[µl]
HMG-CoA
1600
224
222
1
tHXT7
360
51,6
50
1
pPFK1
600
54
83
1,5
ERG20
1100
139
152
1
tPFK2
360
47,4
50
1
pPGK1
600
49
83
1,7
GGPPS
900
14,5
125
8,6
5520
765
+ 19,8 µl linear p426 + 18,4 µl water
° determination of 50 ng for 360 bp
use of a vector-insert relation of 1:1
samples: linear p426 with 7 inserts
positive control (p426)
negative control (linear p426)
plating the samples on SCD-ura and incubation at 30° C
General procedure of a transformation via gap repair
Amplification of the required DNA fragment with homologue ends of the plasmid (in our case homologue ends of the gene/promoter/terminator beside the DNA fragment)   via PCR and an agarose gel to review the correct size
Linearization of the plasmid and an agarose gel for review
Yeast transformation with the PCR products and the linear plasmid (homologue recombination)
Selection on an agar plate without the metabolite, which is on the plasmid for yeast selection (in our case it was uracil on p426 and histidin on p423)
Inoculation of clones and isolation of the plasmids from yeast
Transformation of a plasmid in E.coli and selection on LB medium with ampicillin (the plasmids have an ampicillin resistance)
Isolation of the plasmids from E.coli
Diagnostic restriction of the plasmid in order to find the correct plasmid with all inserts
Transformation of the correct plasmid in yeast to do further experiments
E.coli Transformation
a) Production of competent cells
1. Inoculate 400 ml LB medium (pre-warmed to 37°C) in a 1 l flask with 100 µl of a fresh over night culture
2. Harvest the cells at a OD600 0,60-0,65 (in a room with 37°C, sterile)
3. Aliquot the solution in 8 x 50 ml falcons (sterile, pre-cooled on ice)
4. Incubate for 30 min on ice
5. Centrifuge for 12,5 min with 4000x g, 4°C
6. Resuspend the pellets in each 10 ml sterile water (4°C)
7. Centrifuge for 10 min with 4000x g, 4°C
8. Repeat steps 6 and 7 two times
9. Resuspend the cells in 10 ml 10% glycerine (+millipor, sterile, 4°C)
10. Centrifuge for 15 min with 4000x g, 4°C
11. Add 800 µl of 10% glycerine (sterile, 4°C) to the cells
12. Aliquot 40 µl of the solution in each reaction tube (pre-cooled) and freeze them in liquid nitrogen
13. Store them up to 6 months at -70°C
b) Transformation (on ice)
1. Add DNA (e.g. 1 µl of extracted yeast plasmids) to the frozen cells
2. Unfreeze the cells on ice
3. Fill the cells into cuvettes for electroporation
4.  Electroporation (for 2 mm cuvettes: voltage I = 2,5 kV; resistance R = 2000 ohm; electrical current A = 25  µF; for 1 mm cuvettes: I = 2,2 kV; resistance R = 2000 ohm; electrical current A = 25 µF)
5. Fill the cells with 1 ml of SOC medium back in a reaction tube
6. Shake them for 1 hour at 37°C
7. Centrifuge for 1 min 8000x g
8. Decant the supernatant
9. Spread onto the appropriate medium
PCR
PCR protocol
Components
Final Concentration
Volume of Example PCR
Polymerase-buffer (e.g. for Phusion Polymerase)
1x
10 µl (5x)
dNTPs
200 µM
5 µl (2 mM)
Primer fw/rev
respectively 0.2 µM
respectively 1 µl (10 µM]
Template
0.2 ng/µl
1 µl (10 ng/µl)
DNA-Polymerase (e.g. Phusion Polymerase)
0.02 U/µl
0.5 µl (2 U/µl)
Water
add to 50 µl solution
31 µl
Magnesiumchlorid
10 mM
0.5 µl (1M)
Total Volume
50 µl
50 µl
Add all components with the right concentrations to a PCR tube. Pay attention to defrost phusion-polymerase on ice and add it at the very end. Put the PCR tube, containing a 50 µl reaction mixture, into a PCR cycler. Make the following settings for running the PCR programm:
PCR programm for one PCR run
Steps
Temperature
Time
First Denaturation
98 °C
30 sec
Denaturation
98 °C
10 sec
Annealing
Depending on Primer
20-30 sec
Polymerization
72 °C
1 min/kbp for Tay-Polymerase; 20 sec/kbp for Phusion Polymerase
Final Polymerization
72 °C
5 min
Let the PCR-Programm run for about 35 cycles.
Restriction Digestion for Linearization of a Plasmid
Digest the choosen plasmid with one suitable restriction enzyme.
Example for digestion reaction:
Componen
Concentration
Shuffle Plasmid
1 µl (3-5 µg/µl)
Restriction Enzyme
respectively 1µl
Suitable Buffer
5 µl (10x)
Water
add to 50 µl
Total Volume
50 µl
Incubate the digestion mix 2 hours at 37 °C. AFter that purify the solution with e.g. a PCR purification kit.
Culture Media
Full Medium (YEPD) for Yeast
Yeast Extract
1 % (weight/volume)
Pepton
2 % (w/v)
Glucose
2 % (w/v)
Synthetic Complete Medium (SC) for Yeast
Yeast Nitrogen Base
0.17 % (w/v)
Ammoniumsulfate
0.5 % (w/v)
Glucose
2 % (w/v)
Amino Acid Mix°
50 ml/l
Histidin**
0.25 mM
Tryptophan**
0.19 mM
Leucin**
0.35 mM
Uracil**
0.44 mM
pH has to be regulated with KOH to pH=6.3
° contains no His, Leu, Trp and Uracil
** addition of this components depents on the respective selection medium
SOC-Medium for Regeneration of transformed Escherichia coli`s after Electroporation
Trypton
2 % (w/v)
Yeast Extract
0.5 % (w/v)
NaCl
10 mM
KCl
2,5 mM
MgCl2
10 mM
MgSO4
10 mM
Glucose
20 mM
pH has to be regulated to pH=6.8-7.0
Full Medium (LB) for E.coli
Yeast Extract
0.5 % (w/v)
Trypton
1 % (w/v)
NaCl
0.5 % (w/v)
pH has to be regulated with NaOH to pH=7.5
Every cluture medium has to be autoclaved to be sterile.
Agar Plate
LBampicilline-Agar
Add 2 % agar to LB-medium. After autoclaving and cooling-down to 60 °C steril ampicillin is added. Plates were poured.
LBchloramphenicol-Agar
Add 2 % agar to LB-medium. After autoclaving and cooling-down to 60 °C ethanolic chlorampenicol solvation is added.
SCD-Agar
Add 2 % agar to SCD-medium. After autoclaving and cooling-down steril amino acid solution is added. Dependent on the respective selective medium Histidin (0.25 mM), Trypthophan (0.19 mM), Uracil (0.44 mM) or Leucin (0.35 mM) are added. Plates were poured.
YEPDG418-Agar
Add 2 % agar to YEPD-medium. After autoclaving and cooling-down sterile G418 (final concentration 2g/l) is added. Plates were poured.
Gel Electrophoresis
Agarose Gel (1x)
TAE puffer
1x
Agarose
1 % (w/v)
Solve agarose in TAE by boiling it. After cooling-down to 55-60 °C gel is poured.
TAE Puffer (50x) for Gel Electrophoresis
EDTA
18,6 g
Tris
242g
Glacial Acetic Acid
57,2 ml
Purified Water
1000ml
pH has to be regulated with glacial acetic acid to pH=8.
Gels were run with a tension of 80-140 V.
Sample preperation for GC
GC-sample preparation of S. cerevisae GC/MS – according Huang et. al 2011 - 2.5 sample preparation
1. Wash pellet of 200ml Yeast-Solution with 1x PBS
2. Weigh pellet
3. Add the same volume of Buffer pH 8.0 (50mM Tris, 1% DMSO, 50mM NaCl, 0.5mM MgCl2, pH with KOH)and Vortex for 1 minute.
4. Chill glass beads on ice, add 1-3g of chilled glass beads per gram of wet cell weight. Vortex 3 - 5 times for 1 minute, each time keeping the cells on ice for 1 minute.
5. Add 3U Pyrophosphatase and incubate at 60 °C for 120 minutes.
6. For hexene extraction: Mix the cell lysate with 1ml Hexene in glass vials.
7. Vortex for 3 minutes and centrifuge for 20 min with 3000x g.
8. Remove the hexene phase, concentrate and add corresponding amount (80µl solution and 20µl MSTFA).Incubate at 60°C for 30 minutes.
9. Resuspend the cells in 10 ml 10% glycerine (+millipor, sterile, 4°C)
10. Centrifuge for 15 min with 4000x g, 4°C
Kits
PCR Purification Kit from Qiagen
Gel Extraction Kit from Qiagen
Midi Plasmid Preparation Kit from Qiagen
iGEM-Team Frankfurt 2013
Home | Project | Organisms | Labwork | Human Practice | Attributions | Contact Us
Retrieved from " http://2013.igem.org/Team:Frankfurt/Notebook/Protocol "
Recent changes
What links here
Related changes
Special pages
My preferences
Printable version
Permanent link
Privacy policy
Disclaimers
