<HTML xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en" dir="ltr">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki  ltr ns-0 ns-subject page-Team_Newcastle_Notebook_protocols"><DIV id="globalWrapper"><DIV id="top-section"><DIV id="p-logo"><A href="/Main_Page" title="Main Page">&quot;
	    </A></DIV><DIV id="menubar" class="left-menu noprint"><UL><LI class="selected"><A href="/Team:Newcastle/Notebook/protocols">Page               </A></LI><LI class="new"><A href="/wiki/index.php?title=Talk:Team:Newcastle/Notebook/protocols&amp;action=edit&amp;redlink=1">Discussion               </A></LI><LI><A href="/wiki/index.php?title=Team:Newcastle/Notebook/protocols&amp;action=edit">View source               </A></LI><LI><A href="/wiki/index.php?title=Team:Newcastle/Notebook/protocols&amp;action=history">History               </A></LI><LI style="color:white;cursor:default">teams</LI></UL></DIV><DIV class="right-menu noprint" id="menubar"><UL><LI id="pt-login"><A href="/wiki/index.php?title=Special:UserLogin&amp;returnto=Team:Newcastle/Notebook/protocols" title="You are encouraged to log in; however, it is not mandatory [o]" accesskey="o">Log in</A></LI></UL></DIV><DIV id="search-controls" class="noprint"><FORM action="/Special:Search" id="searchform"> </FORM></DIV></DIV><DIV id="content"><H1 class="firstHeading">Team:Newcastle/Notebook/protocols</H1><DIV id="bodyContent"><H3 id="siteSub" class="noprint">From 2013.igem.org</H3><DIV><DIV id="pop_up"><DIV id="pop_up_underlay"> </DIV><DIV id="pop_up_display"><DIV id="pop_up_close">X</DIV><DIV id="pop_up_content"> </DIV></DIV></DIV><DIV class="menu" class="grid_12"><UL class="top_menu" style="margin-left:10px"><LI><A href="https://2013.igem.org/Team:Newcastle">Home</A></LI><LI><A href="#" onclick="return false;">Team</A></LI><LI><A href="https://igem.org/Team.cgi?year=2013&amp;team_name=Newcastle" target="_blank">Official Profile</A></LI><LI><A href="/Team:Newcastle/Team">Individual Profiles</A></LI></UL><LI><A href="#" onclick="return false;">Project</A></LI><LI><A href="https://2013.igem.org/Team:Newcastle/Project">Overview</A></LI><LI><A href="https://2013.igem.org/Team:Newcastle/Project/L_forms">L-forms</A></LI><LI><A href="https://2013.igem.org/Team:Newcastle/Project/shuffling_endosymbiosis">Genome Shuffling</A></LI><LI><A href="https://2013.igem.org/Team:Newcastle/Project/plants">L-forms in Plants</A></LI><LI><A href="https://2013.igem.org/Team:Newcastle/Project/shape_shifting">Shape Shifting</A></LI><LI><A href="#" onclick="return false;">Parts</A></LI><LI><A href="https://2013.igem.org/Team:Newcastle/Parts/l_form_switch">L-form Switch</A></LI><LI><A href="https://2013.igem.org/Team:Newcastle/Parts/HBsu-fp">HBsu-xFP</A></LI><LI><A href="https://2013.igem.org/Team:Newcastle/Parts/e-coli_swich">E-coli Switch</A></LI><LI><A href="#" onclick="return false;">Models</A></LI><LI><A href="https://2013.igem.org/Team:Newcastle/Modelling/Introduction">Introduction</A></LI><LI><A href="https://2013.igem.org/Team:Newcastle/Modelling/BioNetGen">BioNetGen</A></LI><LI><A href="https://2013.igem.org/Team:Newcastle/Modelling/CellShapeModel">Cell Shape Model</A></LI><LI><A href="https://2013.igem.org/Team:Newcastle/Modelling/Cell_Fusion">Cell Fusion</A></LI><LI><A href="https://2013.igem.org/Team:Newcastle/Modelling/HBsu_Fusion_Protein">HBsu-xFP</A></LI><LI><A href="https://2013.igem.org/Team:Newcastle/Modelling/L-form_Switch">L-form Switch</A></LI><LI><A href="#" onclick="return false;">Notebook</A></LI><LI><A href="https://2013.igem.org/Team:Newcastle/Notebook/achievements">Achievements</A></LI><LI><A href="https://2013.igem.org/Team:Newcastle/Notebook/timeline">Timeline</A></LI><LI><A href="https://2013.igem.org/Team:Newcastle/Notebook/calendar">Calendar</A></LI><LI><A href="https://2013.igem.org/Team:Newcastle/Notebook/protocols">Protocols</A></LI><LI><A href="https://2013.igem.org/Team:Newcastle/Notebook/agenda">Meetings</A></LI><LI><A href="#" onclick="return false;">Implications</A></LI><LI><A href="https://2013.igem.org/Team:Newcastle/Outreach/Overview">Overview</A></LI><LI><A href="https://2013.igem.org/Team:Newcastle/Outreach/BioGame">BioGame</A></LI><LI><A href="https://2013.igem.org/Team:Newcastle/Outreach/uk_meet">YSB 1.0</A></LI><LI><A href="https://2013.igem.org/Team:Newcastle/Outreach/Workshop">Modelling Workshop</A></LI><LI><A href="https://2013.igem.org/Team:Newcastle/Outreach/Architecture">Architecture Workshop</A></LI><LI><A href="https://2013.igem.org/Team:Newcastle/HP/Safety">Safety</A></LI><LI><A href="https://2013.igem.org/Team:Newcastle/HP/Synthetic_Bio">Synthetic Biology</A></LI><LI><A href="https://2013.igem.org/Team:Newcastle/HP/Ethics">Ethics</A></LI><LI><A href="https://2013.igem.org/Team:Newcastle/HP/Law">Law</A></LI><LI><A href="#" onclick="return false;">Attributions</A></LI><LI><A href="https://2013.igem.org/Team:Newcastle/Collaborations">Collaborations</A></LI><LI><A href="https://2013.igem.org/Team:Newcastle/Attributions">Acknowledgments</A></LI><LI><A href="https://2013.igem.org/Team:Newcastle/Sponsors">Sponsors</A></LI><LI><A href="https://2013.igem.org/Team:Newcastle/Architecture">Architecture</A></LI><LI><A href="https://2013.igem.org/Main_Page">iGEM</A></LI></DIV><TABLE id="toc" class="toc"><TBODY><TR><TD><DIV id="toctitle"><H2>Contents</H2></DIV><UL><LI class="toclevel-1 tocsection-1"><A href="#Protocols"><SPAN class="tocnumber">1</SPAN><SPAN class="toctext">Protocols</SPAN></A></LI><LI class="toclevel-2 tocsection-2"><A href="#Introduction"><SPAN class="tocnumber">1.1</SPAN><SPAN class="toctext">Introduction</SPAN></A></LI><LI class="toclevel-2 tocsection-3"><A href="#L-form_Media_Components"><SPAN class="tocnumber">1.2</SPAN><SPAN class="toctext">L-form Media Components</SPAN></A></LI><LI class="toclevel-3 tocsection-4"><A href="#2.C3.97MSM"><SPAN class="tocnumber">1.2.1</SPAN><SPAN class="toctext">2×MSM</SPAN></A></LI><LI class="toclevel-3 tocsection-5"><A href="#2.C3.97Nutrient_Broth_.28NB.2C_Oxoid.29"><SPAN class="tocnumber">1.2.2</SPAN><SPAN class="toctext">2×Nutrient Broth (NB,  Oxoid)</SPAN></A></LI><LI class="toclevel-3 tocsection-6"><A href="#2.C3.97Nutrient_Agar_.28NA.2C_Oxoid.29"><SPAN class="tocnumber">1.2.3</SPAN><SPAN class="toctext">2×Nutrient Agar (NA, Oxoid)</SPAN></A></LI><LI class="toclevel-3 tocsection-7"><A href="#Stock_PenG"><SPAN class="tocnumber">1.2.4</SPAN><SPAN class="toctext">Stock PenG</SPAN></A></LI></UL><LI class="toclevel-2 tocsection-8"><A href="#L-form_Growth_Media"><SPAN class="tocnumber">1.3</SPAN><SPAN class="toctext">L-form Growth Media</SPAN></A></LI><LI class="toclevel-3 tocsection-9"><A href="#Solid_Media"><SPAN class="tocnumber">1.3.1</SPAN><SPAN class="toctext">Solid Media</SPAN></A></LI><LI class="toclevel-3 tocsection-10"><A href="#Liquid_media"><SPAN class="tocnumber">1.3.2</SPAN><SPAN class="toctext">Liquid media</SPAN></A></LI><LI class="toclevel-2 tocsection-11"><A href="#Creating_L-forms"><SPAN class="tocnumber">1.4</SPAN><SPAN class="toctext">Creating L-forms</SPAN></A></LI><LI class="toclevel-3 tocsection-12"><A href="#Method"><SPAN class="tocnumber">1.4.1</SPAN><SPAN class="toctext">Method</SPAN></A></LI><LI class="toclevel-2 tocsection-13"><A href="#Protoplasting_to_generate_L-forms"><SPAN class="tocnumber">1.5</SPAN><SPAN class="toctext">Protoplasting to generate L-forms</SPAN></A></LI><LI class="toclevel-3 tocsection-14"><A href="#Method_2"><SPAN class="tocnumber">1.5.1</SPAN><SPAN class="toctext">Method</SPAN></A></LI><LI class="toclevel-2 tocsection-15"><A href="#Regeneration_of_Rod_cells_from_L-form"><SPAN class="tocnumber">1.6</SPAN><SPAN class="toctext">Regeneration of Rod cells from L-form</SPAN></A></LI><LI class="toclevel-3 tocsection-16"><A href="#Protoplast_regeneration_media"><SPAN class="tocnumber">1.6.1</SPAN><SPAN class="toctext">Protoplast regeneration media</SPAN></A></LI><LI class="toclevel-3 tocsection-17"><A href="#L-form_growth_media_2"><SPAN class="tocnumber">1.6.2</SPAN><SPAN class="toctext">L-form growth media</SPAN></A></LI><LI class="toclevel-2 tocsection-18"><A href="#Streak_plate"><SPAN class="tocnumber">1.7</SPAN><SPAN class="toctext">Streak plate</SPAN></A></LI><LI class="toclevel-3 tocsection-19"><A href="#From_glycerol_stocks"><SPAN class="tocnumber">1.7.1</SPAN><SPAN class="toctext">From glycerol stocks</SPAN></A></LI><LI class="toclevel-3 tocsection-20"><A href="#From_liquid_culture"><SPAN class="tocnumber">1.7.2</SPAN><SPAN class="toctext">From liquid culture</SPAN></A></LI><LI class="toclevel-2 tocsection-21"><A href="#Liquid_culture"><SPAN class="tocnumber">1.8</SPAN><SPAN class="toctext">Liquid culture</SPAN></A></LI><LI class="toclevel-2 tocsection-22"><A href="#Glycerol_stocks"><SPAN class="tocnumber">1.9</SPAN><SPAN class="toctext">Glycerol stocks</SPAN></A></LI><LI class="toclevel-3 tocsection-23"><A href="#Making_glycerol_stocks"><SPAN class="tocnumber">1.9.1</SPAN><SPAN class="toctext">Making glycerol stocks</SPAN></A></LI><LI class="toclevel-2 tocsection-24"><A href="#Chromosomal_DNA_Extraction"><SPAN class="tocnumber">1.10</SPAN><SPAN class="toctext">Chromosomal DNA Extraction</SPAN></A></LI><LI class="toclevel-2 tocsection-25"><A href="#Nanodrop_quantification_of_DNA_extracts"><SPAN class="tocnumber">1.11</SPAN><SPAN class="toctext">Nanodrop quantification of DNA extracts</SPAN></A></LI><LI class="toclevel-2 tocsection-26"><A href="#PCR"><SPAN class="tocnumber">1.12</SPAN><SPAN class="toctext">PCR</SPAN></A></LI><LI class="toclevel-2 tocsection-27"><A href="#QIAquick_Gel_Extraction_Protocol"><SPAN class="tocnumber">1.13</SPAN><SPAN class="toctext">QIAquick Gel Extraction Protocol</SPAN></A></LI><LI class="toclevel-3 tocsection-28"><A href="#Notes_before_starting"><SPAN class="tocnumber">1.13.1</SPAN><SPAN class="toctext">Notes before starting</SPAN></A></LI><LI class="toclevel-3 tocsection-29"><A href="#Method_3"><SPAN class="tocnumber">1.13.2</SPAN><SPAN class="toctext">Method</SPAN></A></LI><LI class="toclevel-2 tocsection-30"><A href="#QIAprep_Spin_Miniprep_Kit"><SPAN class="tocnumber">1.14</SPAN><SPAN class="toctext">QIAprep Spin Miniprep Kit</SPAN></A></LI><LI class="toclevel-2 tocsection-31"><A href="#Escherichia_coli_Competent_Cell_Preparation"><SPAN class="tocnumber">1.15</SPAN><SPAN class="toctext">Escherichia coli Competent Cell Preparation</SPAN></A></LI><LI class="toclevel-3 tocsection-32"><A href="#Method_4"><SPAN class="tocnumber">1.15.1</SPAN><SPAN class="toctext">Method</SPAN></A></LI><LI class="toclevel-2 tocsection-33"><A href="#Escherichia_coli_Transformation"><SPAN class="tocnumber">1.16</SPAN><SPAN class="toctext">Escherichia coli  Transformation</SPAN></A></LI><LI class="toclevel-2 tocsection-34"><A href="#Bacillus_subtilis_168_Competent_cell_Prep_and_Transformation"><SPAN class="tocnumber">1.17</SPAN><SPAN class="toctext">Bacillus subtilis 168 Competent cell Prep and Transformation</SPAN></A></LI><LI class="toclevel-3 tocsection-35"><A href="#SMM_medium"><SPAN class="tocnumber">1.17.1</SPAN><SPAN class="toctext">SMM medium</SPAN></A></LI><LI class="toclevel-3 tocsection-36"><A href="#MM_competence_medium"><SPAN class="tocnumber">1.17.2</SPAN><SPAN class="toctext">MM competence medium</SPAN></A></LI><LI class="toclevel-3 tocsection-37"><A href="#Starvation_medium"><SPAN class="tocnumber">1.17.3</SPAN><SPAN class="toctext">Starvation medium</SPAN></A></LI><LI class="toclevel-3 tocsection-38"><A href="#Methods"><SPAN class="tocnumber">1.17.4</SPAN><SPAN class="toctext">Methods</SPAN></A></LI><LI class="toclevel-2 tocsection-39"><A href="#Inoculating_L-forms_into_Chinese_Cabbage"><SPAN class="tocnumber">1.18</SPAN><SPAN class="toctext">Inoculating L-forms into Chinese Cabbage</SPAN></A></LI><LI class="toclevel-2 tocsection-40"><A href="#Visualisation_of_L-forms_in_plants"><SPAN class="tocnumber">1.19</SPAN><SPAN class="toctext">Visualisation of L-forms in plants</SPAN></A></LI><LI class="toclevel-2 tocsection-41"><A href="#Making_Starch_Agar_Plates"><SPAN class="tocnumber">1.20</SPAN><SPAN class="toctext">Making Starch Agar Plates</SPAN></A></LI></TD></TR></TBODY></TABLE><H1><SPAN class="mw-headline" id="Protocols">Protocols</SPAN></H1><H2><SPAN class="mw-headline" id="Introduction">Introduction</SPAN></H2><P>Over the course of our iGEM project we used many different laboratory procedures. In this page we have kept a detailed record of our protocols for our own use and so people exploring our wiki can understand how we carried out our experiments and gained our results. Please click on a protocol name on the tab on the right side of the page to view it. 
</P><H2><SPAN class="mw-headline" id="L-form_Media_Components">L-form Media Components</SPAN></H2><H3><SPAN class="mw-headline" id="2.C3.97MSM">2×MSM</SPAN></H3><P>Two times concentrated stock of 1 M sucrose, 40 mM Maleic acid, and 40 mM Magnesium Chloride.
</P><UL><LI>342 g Sucrose.
</LI><LI>4.65 g Maleic acid.
</LI><LI>8.12 g Magnesium chloride hexahydrate.
</LI></UL><OL><LI>Add d-H2O to a final volume of 1 litre.
</LI><LI>Adjust to pH 7 with NaOH.
</LI><LI>Aliquot into 500 ml bottles.
</LI><LI>Autoclave on a short cycle (15 minutes at 121°C) as prolonged heating caramelizes the sugar giving a dark brown solution that the bacteria don’t grow in.
</LI></OL><H3><SPAN class="mw-headline" id="2.C3.97Nutrient_Broth_.28NB.2C_Oxoid.29">2×Nutrient Broth (NB,  Oxoid)</SPAN></H3><P>Add 2.6 g of NB to 100 ml of demin-H2O then autoclave.
</P><H3><SPAN class="mw-headline" id="2.C3.97Nutrient_Agar_.28NA.2C_Oxoid.29">2×Nutrient Agar (NA, Oxoid)</SPAN></H3><P>Add 5.6 g of NA to 100 ml of d-H2O then autoclave.
</P><H3><SPAN class="mw-headline" id="Stock_PenG">Stock PenG</SPAN></H3><P>200 mg/ml of Penicillin G (benzylpenicillin or PenG, Sigma).
</P><P>Add 2 g of PenG to 10 ml final volume of d-H2O and filter sterilise.  Aliquot and freeze at -20°C.
</P><H2><SPAN class="mw-headline" id="L-form_Growth_Media">L-form Growth Media</SPAN></H2><H3><SPAN class="mw-headline" id="Solid_Media">Solid Media</SPAN></H3><OL><LI>First make 2x Nutrient agar (Oxoid, NA), 2x MSM and 200 mg/ml PenG.
</LI><LI>Allow the nutrient agar to cool to about 80-90°C.
</LI><LI>Add an equal volume of MSM at room temperature.
</LI><LI>Mix thoroughly.
</LI><LI>Add 200 µl of 200 mg/ml PenG and pour. 
</LI><LI>Allow the agar to set and cool to room temperature.
</LI><LI>Air dry for no more than 5 minutes.  
</LI></OL><P>Notes:
These last two steps are critical as evaporation of water increases the concentration of sucrose: this should be avoided as L-forms are highly sensitive to changes in the osmolarity of the media.
</P><H3><SPAN class="mw-headline" id="Liquid_media">Liquid media</SPAN></H3><OL><LI>Mix equal volumes of 2× MSM and 2× Nutrient Broth (Oxoid, NB) and add 200µl of (200 mg/ml) PenG if required.
</LI><LI>Add 14-20 ml of media to a 250 ml Erlenmeyer flask.  Or 8-10 ml into 100 ml flask. 
</LI><LI>Inoculate the media with 5-10 µl of cells from an L-form colony/liquid broth with a pipette. 
</LI><LI>Gently mix the media to break up the cells.
</LI><LI>Incubate at 30°C without shaking. 
</LI></OL><P>Notes:
A low media volume to flask volume ratio allows a thin layer of media which helps maximise diffusion of oxygen. Shaking is not necessary for growth, and if done should be very gentle.
</P><UL><LI><A href="https://2013.igem.org/wiki/index.php?title=Team:Newcastle/Notebook/protocols" class="external text" rel="nofollow">Return to top</A></LI></UL><H2><SPAN class="mw-headline" id="Creating_L-forms">Creating L-forms</SPAN></H2><P>After <A href="https://2013.igem.org/Team:Newcastle/Notebook/protocols#Bacillus_subtilis_168_Competent_cell_Prep_and_Transformation" class="external text" rel="nofollow">transforming</A> your <I>Bacillus subtilis</I> with the <A href="https://2013.igem.org/Team:Newcastle/Parts/l_form_switch" class="external text" rel="nofollow">switch brick</A>, you will need to plate them out onto LB (1.5% agar) with 5 µg/ml of chloramphenicol (Cm) and 0.5-0.8% xylose. 
</P><H3><SPAN class="mw-headline" id="Method">Method</SPAN></H3><OL><LI>Pick a colony of rod cell <I>B. subtilis</I> with switch brick transformed into them, and inoculate the 50/50 NB/MSM media.
</LI><LI>Another culture should be grown with colonies inoculated into NB/MSM that has been supplemented with 0.5-0.8% xylose + 5 µg/ml Cm to act as control.
</LI><LI>In non xylose-supplemented NB/MSM, the rod cells will transition into L-form state. In order to keep this culure pure you need to add PenG 200 µg/ml in addition to Cm.
</LI></OL><P>Notes: 
The best way to maintain L-form culture is to plate them out as rod cells into LB (1.5% agar) with 5µg/ml of Chloramphenicol and 0.5-0.8% xylose plates before converting to L-form. 
</P><H2><SPAN class="mw-headline" id="Protoplasting_to_generate_L-forms">Protoplasting to generate L-forms</SPAN></H2><P>This protocol can be followed once the <I>B. subtilis</I> is transformed with the <A href="https://2013.igem.org/Team:Newcastle/Parts/l_form_switch" class="external text" rel="nofollow">switch brick</A> and plated out onto LB agar + xylose-supplemented plate. By using this protocol, which is similar to protocols for generating protoplasts, it is possible to generate pure L-form colonies in a shorter time-frame then letting cells lose their cell wall naturally. 
</P><H3><SPAN class="mw-headline" id="Method_2">Method</SPAN></H3><OL><LI>Take the cells from the xylose-supplemented plate and inoculate into 4ml LB + xylose (0.8%) and incubate at 37°C for 2-3 hours.
</LI><LI>Dilute the cells by adding 6ml LB + 0.8% xylose media.
</LI><LI>Check optical density (OD, 600 nm) of cells every 1 hour until between 0.3-0.6 (mid exponential stage).
</LI><LI>Spin down at ~5,500rpm for 4 minutes and discard supernatant to remove the xylose.
</LI><LI>Wash once with LB (10 ml).
</LI><LI>Pellet the cells at 4,500rpm for 15 minutes and discard supernatant.
</LI><LI>Resuspend pellet with Lysis solution (50/50 NB/MSM (4 ml) + PenG (200 µg/ml) + Lysozyme (4 mg/ml)).
</LI><LI>Incubate for 1-1.30 hours at 37°C for the lysozyme to work.
</LI><LI>Check samples under a microscope during the incubation period until the overwhelming majority of cells are protoplasts.
</LI><LI>Spin down and re-suspend with NB/MSM  (10-15 ml) + chloramphenicol (5 µg/ml) + 200µl of 200 mg/ml PenG.
</LI><LI>Incubate at 30°C for at least 2 days.
</LI></OL><H2><SPAN class="mw-headline" id="Regeneration_of_Rod_cells_from_L-form">Regeneration of Rod cells from L-form</SPAN></H2><P>Rod regeneratiom can be done either in solid or liquid media. It can be performed using either of the following media: -
</P><H3><SPAN class="mw-headline" id="Protoplast_regeneration_media">Protoplast regeneration media</SPAN></H3><UL><LI>(2X)DM3.
</LI><LI>0.5 M sodium succinate (pH 7.3).
</LI><LI>0.5 % Casaminoacids.
</LI><LI>0.5 % Yeast extract.
</LI><LI>0.5 % Glucose.
</LI></UL><UL><LI>Mixed with (2X)MSM in 1:1 ratio .
</LI><LI>Xylose is added to 1% final concentration.
</LI></UL><H3><SPAN class="mw-headline" id="L-form_growth_media_2">L-form growth media</SPAN></H3><P>Media for L-form growth (NA/MSM) supplemented with (0.5-0.8%) xylose can also be used.
</P><UL><LI><A href="https://2013.igem.org/wiki/index.php?title=Team:Newcastle/Notebook/protocols" class="external text" rel="nofollow">Return to top</A></LI></UL><H2><SPAN class="mw-headline" id="Streak_plate">Streak plate</SPAN></H2><H3><SPAN class="mw-headline" id="From_glycerol_stocks">From glycerol stocks</SPAN></H3><OL><LI>Glycerol stock must be kept in an ice bucket.
</LI><LI>Working close to lit Bunsen burner.
</LI><LI>Flame a loop until red hot then dip into glycerol stock in cryo-vial.
</LI><LI>Streak plate by: -
</LI><LI>Dragging loop across agar 5-10 times.
</LI><LI>Flame loop, then dip in agar (away from streak) to cool.
</LI><LI>Streak 4 times at a 90° angle from first streak.
</LI><LI>Streak again 4 times at 90° from second set of streaks.
</LI><LI>Streak again 4 times at 90° from third set of streaks to complete a square on the agar plate.
</LI><LI>Finally drag loop in a squiggle motion across the centre of the agar plate.
</LI><LI>The streak plate must then be left overnight at 37°C for colonies to grow.
</LI><LI>LB agar plates will usually be used unless alternative conditions are required.
</LI></OL><H3><SPAN class="mw-headline" id="From_liquid_culture">From liquid culture</SPAN></H3><P>Protocol is similar to that of creating streak plates from glycerol stocks. However, an overnight culture of the strain to be streaked is grown and the loop is dipped into this culture rather than a cryo-vial of the glycerol stock.
</P><H2><SPAN class="mw-headline" id="Liquid_culture">Liquid culture</SPAN></H2><P>Working close to lit Bunsen burner: -
</P><OL><LI>Liquid cultures can be grown in plastic universal tubes if only a small volume is needed (e.g. 5 ml).
</LI><LI>Transfer a colony from a plate of colonies using a flame-sterilised loop into liquid LB media in tube.
</LI><LI>Cultures should be made from only one colony.
</LI><LI>Leave the plastic universals on rotating plate (aerator) at 37°C overnight.
</LI></OL><H2><SPAN class="mw-headline" id="Glycerol_stocks">Glycerol stocks</SPAN></H2><H3><SPAN class="mw-headline" id="Making_glycerol_stocks">Making glycerol stocks</SPAN></H3><P>NB: Only 50% glycerol solution is necessary for preserving cells by freezing at -80°C.
</P><OL><LI>1 volume of 100% glycerol should be mixed with 1 volume of Mili-Q filtered water (i.e. 1 ml glycerol for 1 ml water).
</LI><LI>Vortex the solution briefly to mix.
</LI><LI>Filter the solution using a sterile syringe filter, then pipette 500 µl of the 50% glycerol solution into cryo-vials.
</LI><LI>Pipette 500 µl of the LB cell culture to be preserved into the cryo-vials.
</LI><LI>Mix by inverting the tubes.
</LI><LI>Glycerol stocks can then be stored in deep freeze at -80°C.
</LI></OL><UL><LI><A href="https://2013.igem.org/wiki/index.php?title=Team:Newcastle/Notebook/protocols" class="external text" rel="nofollow">Return to top</A></LI></UL><H2><SPAN class="mw-headline" id="Chromosomal_DNA_Extraction">Chromosomal DNA Extraction</SPAN></H2><OL><LI>Add 1.75 ml of bacterial cell culture to a 2 ml tube.
</LI><LI>Spin tubes at 20,000 x g (max speed of centrifuge may be 16,000 x g, if so spin at this speed) for 5 minutes in centrifuge and then discard the liquid supernatant.
</LI><LI>Add 180 µl of enzymatic lysis buffer (36 µl lysozyme and 144 µl buffer) to the tube and vortex for 10-20 seconds.
</LI><LI>Incubate at 37°C for 20 minutes.
</LI><LI>Add 25 µl proteinase K.
</LI><LI>Add 200 µl Buffer AL.
</LI><LI>Vortex briefly.
</LI><LI>Incubate at 56°C for 5 minutes.
</LI><LI>Add 200 µl ethanol to the tube and vortex briefly.
</LI><LI>Transfer entire contents (~600 µl) to a spin column in 2 ml collection tube using a pipette.
</LI><LI>Centrifuge column at 10,000 x g for 1 minute.
</LI><LI>Discard flow-through and replace spin column in a 2 ml collection tube.
</LI><LI>Add 500 µl Buffer AW1 to the column and centrifuge at 10,000 x g for 1 minute.
</LI><LI>Discard flow through and replace spin column in a 2 ml collection tube.
</LI><LI>Add 500 µl Buffer AW2 to the column and centrifuge at 16,000 x g for 3 minutes.
</LI><LI>Carefully remove tubes from centrifuge without allowing the flow-through to contact the column (if this happens, spin tube in centrifuge again for 1 minute at 16,000 x g) and transfer column to a 1.5 ml tube.
</LI><LI>Add 200 µl Buffer AE to the column and leave to stand at room temperature for 1 minute.
</LI><LI>Centrifuge at 10,000 x g for 1 minute and then discard the column.  Store the DNA in the collection tube at 4°C.
</LI></OL><P>NB: Steps 17 and 18 can be carried out in repeat using smaller volumes of Buffer AE in order to try and increase DNA yield.  Adding 30 µl Buffer AE to the column and leaving to stand at room temperature for 5 minutes before centrifuging (and repeating 4-6 times) produces a high DNA yield.
</P><H2><SPAN class="mw-headline" id="Nanodrop_quantification_of_DNA_extracts">Nanodrop quantification of DNA extracts</SPAN></H2><P>Nanodrop procedure: -
</P><OL><LI>Log into computer.
</LI><LI>Open Nanodrop 1000.
</LI><LI>Click on ‘Nucleic Acids’ button.
</LI><LI>Default DNA setting will be DNA-50 – this is correct.
</LI><LI>Wipe pedestal and top with tissue paper before starting.
</LI><LI>Initialise by adding 3 µl H2O to pedestal as instructed by the program.
</LI><LI>Blank with 3 µl elution Buffer (EB) (click ‘Blank’).
</LI><LI>Add 3 µl sample and click ‘Measure’.
</LI><LI>Wipe pedestal with tissue paper between measurements.
</LI><LI>Re-blank with water when finished to clean instrument, dry with tissue and log off.
</LI></OL><H2><SPAN class="mw-headline" id="PCR">PCR</SPAN></H2><OL><LI>Add the following reagents (quantities are for a 50µl reaction) to a PCR tube in the listed order:
</LI><LI>27.5 µl H2O.
</LI><LI>10.0 µl 5x Buffer HF.
</LI><LI>1.0 µl dNTP (200 µM).
</LI><LI>5.0 µl Reverse primer.
</LI><LI>5.0 µl Forward primer.
</LI><LI>1.0 µl Template DNA.
</LI><LI>0.5 µl Phusion Polymerase (must be added last)
</LI></OL><LI>Mix these reagents by inverting the tube and then centrifuge for a couple of seconds for all of the reagents to collect at the bottom of the tube.&lt;/li&gt;</LI><LI>Set conditions of PCR on PCR machine:
</LI><LI>98°C	30 seconds		initial denaturation
</LI><LI>98°C	10 seconds		denaturation
</LI><LI>x°C	        (10-20 seconds)	anneal
</LI><LI>72°C	30 seconds/kb		extension
</LI><LI>72°C	5-10 minutes		final extension
</LI><LI>4°C	        hold
</LI><LI>Store the PCR products at 4°C.
</LI><P>NB: The temperature of x°C at the annealing step differs depending on primers used.
</P><P>NB: The denaturation, anneal and extension steps are repeated cyclically for 30 cycles.
</P><P>NB: Procedure for PCR differs between DNA polymerases.  This procedure is for Phusion DNA polymerase.
</P><UL><LI><A href="https://2013.igem.org/wiki/index.php?title=Team:Newcastle/Notebook/protocols" class="external text" rel="nofollow">Return to top</A></LI></UL><H2><SPAN class="mw-headline" id="QIAquick_Gel_Extraction_Protocol">QIAquick Gel Extraction Protocol</SPAN></H2><H3><SPAN class="mw-headline" id="Notes_before_starting">Notes before starting</SPAN></H3><UL><LI>The yellow colour of Buffer QG indicates a pH &lt;7.5.
</LI><LI>Add ethanol (96-100%) to Buffer PE before use (see bottle label for volume).
</LI><LI>Isopropanol (100%) and a heating block or water bath at 50°C are required.
</LI><LI>All centrifugation steps are carried out at 17,900 x g (13,000 rpm) in a conventional table-top microcentrifuge.
</LI></UL><H3><SPAN class="mw-headline" id="Method_3">Method</SPAN></H3><OL><LI>Excise the DNA fragment from the agarose gel with a clean, sharp scalpel.
</LI><LI>Weigh the gel slice in a colourless tube. Add 3 volumes of Buffer QG to 1 volume of gel (100 mg ~ 100 μl). <I>If the colour of the mixture is orange or violet, add 10 μl of 3 M sodium acetate, pH 5.0, and mix. The colour of the mixture will turn to yellow.</I></LI><LI>Incubate at 50°C for 10 min (or until the gel slice has completely dissolved). To help dissolve gel, mix by vortexing the tube every 2–3 min during the incubation. <I>For &gt;2% gels, increase incubation time.</I></LI><LI>After the gel slice has dissolved completely, check that the colour of the mixture is yellow (similar to Buffer QG without dissolved agarose). <I>If the colour of the mixture is orange or violet, add 10 μl of 3 M sodium acetate, pH 5.0, and mix. The colour of the mixture will turn to yellow.</I></LI><LI>Add 1 gel volume of isopropanol to the sample and mix.
</LI><LI>Place a QIAquick spin column in a provided 2ml collection tube.
</LI></OL><H2><SPAN class="mw-headline" id="QIAprep_Spin_Miniprep_Kit">QIAprep Spin Miniprep Kit</SPAN></H2><OL><LI>Resuspend pelleted bacterial cells in 250 µl Buffer P1 and transfer to a microcentrifuge tube.
</LI><LI>Add 250 µl Buffer P2and mix by inverting the tube 4-6 times.
</LI><LI>Add 350 µl Buffer N3 and mix immediately by inverting 4-6 times.
</LI><LI>Centrifuge for 10 min at 13,000 rpm in a table top microcentrifuge.
</LI><LI>Apply the supernatant to the QIAprep spin column by pipetting.
</LI><LI>Centrifuge for 30-60s. Discard the flow-through.
</LI><LI>Wash the QIAprep spin column by adding 0.5 ml Buffer PB and centrifuging for 30-60s. Discard the flow-through.
</LI><LI>Wash QIAprep spin column by adding 0.75 ml Buffer PE and centrifuging for 30-60s.
</LI><LI>Discard the flow-through, and centrifuge for an additional min to remove residual  wash buffer.
</LI><LI>To elute the DNA, put the QIAprep column in a clean 1.5 ml microcentrifuge tube and add 50 µl Buffer EB to the centre of each QIAprep spin column. Let it stand for 1 min and centrifuge for 1 min.
</LI></OL><UL><LI><A href="https://2013.igem.org/wiki/index.php?title=Team:Newcastle/Notebook/protocols" class="external text" rel="nofollow">Return to top</A></LI></UL><H2><SPAN class="mw-headline" id="Escherichia_coli_Competent_Cell_Preparation"><I>Escherichia coli</I> Competent Cell Preparation</SPAN></H2><P>Before starting this procedure, you need to prepare the following media:
</P><UL><LI>TFBI:
</LI><LI>30 mM KAc
</LI><LI>10 mM CaCl2
</LI><LI>100 mM KCl
</LI><LI>15% (v/v) glycerol
</LI></UL><P>Dissolve and autoclave in 900 ml d.H2O, then add 100 ml autoclaved 500 mM MnCl2 to make up the volume.
</P><P>i.e. Per litre: 2.94 g KAc, 1,4 g CaCl2.2H2O, 7.64 g KCl, Per 100 ml: 9.89 MnCl2.4H2O.
</P><UL><LI>TFBII:
</LI><LI>75mM CaCl2
</LI><LI>10mM KCl
</LI><LI>15% glycerol
</LI></UL><P>Dissolve in 900 ml H2O and autoclave. Make up volume by adding 100 ml of autoclaved 100vmM Na-MOPS, pH 7.0.
</P><P>i.e. Per litre: 11.03 g CaCl2.2H2O, 0.75 g KCl, Per 100 ml: 10.47 g MOPS, pH with NaOH.
</P><H3><SPAN class="mw-headline" id="Method_4">Method</SPAN></H3><OL><LI>Inoculate 300 ml of LB broth with 1/20 volume of an overnight culture of the desired strain (i.e. 15 ml O/N for 300 ml).
</LI><LI>Grow the cells at 37°C (with shaking) to an OD (600) nm of 0.6.
</LI><LI>Chill the cells on ice and harvest by centrifugation at 4°C for 10 min.
</LI><LI>Re-suspend the pellet in 100 ml of ice cold TFBI.
</LI><LI>Re-centrifuge the cells as before and re-suspend in 20 ml of ice cold TFBII.
</LI><LI>Aliquot 200 µl volume of the cell suspension into cooled, sterile microfuge tubes, and flash freeze in an EtOH/dry-ice bath.
</LI><LI>Store the cells at -80°C until required.
</LI></OL><H2><SPAN class="mw-headline" id="Escherichia_coli_Transformation"><I>Escherichia coli</I>  Transformation</SPAN></H2><OL><LI> Thaw a 200 µl aliquot of the desired strain of <I>E.coli</I> and add the transforming DNA. Incubate on ice for up to 45 mins.
</LI><LI> Heat-shock the cells in a water bath (42°C) for 120 secs, and place on ice again for 3-4 mins.
</LI><LI> Add 1 ml of LB broth and incubate the cells at 37°C for 1 - 1.5hr.
</LI><LI> Spin down the cells in a microfuge (~20secs) and remove the supernatant.
</LI><LI> Re-suspend the cells pellet in ~200 µl of broth or d.H2O and plate out on LB (agar at 1.5%), containing the appropriate selection markers.
</LI></OL><H2><SPAN class="mw-headline" id="Bacillus_subtilis_168_Competent_cell_Prep_and_Transformation"><I>Bacillus subtilis</I> 168 Competent cell Prep and Transformation</SPAN></H2><P>Before starting this procedure, you need to prepare the following media: -
</P><H3><SPAN class="mw-headline" id="SMM_medium">SMM medium</SPAN></H3><P>For 1 litre you need: -
</P><UL><LI>2.0 g ammonium sulphate
</LI><LI>14.0 g dipotassium hydrogen phosphate
</LI><LI>6.0 g potassium dihydrogen phosphate
</LI><LI>1.0 g sodium citrate dehydrate (trisodium citrate)
</LI><LI>100 µl 1M magnesium sulphate
</LI></UL><P>Make it up to 1 litre, split into 5X 200 ml bottles and autoclave.
</P><H3><SPAN class="mw-headline" id="MM_competence_medium">MM competence medium</SPAN></H3><P>For 10 ml you need: -
</P><UL><LI>10 ml SMM media
</LI><LI>125 µl solution E (40% glucose)
</LI><LI>100 µl Tryptophan solution
</LI><LI>60 µl solution F (MgSO4)
</LI><LI>10 µl casamino acids
</LI><LI>5 µl Fe-NH4-citrate
</LI></UL><H3><SPAN class="mw-headline" id="Starvation_medium">Starvation medium</SPAN></H3><P>For 10 ml you need: -
</P><UL><LI>10 ml SMM media
</LI><LI>125 µl solution E (40% glucose)
</LI><LI>60 µl solution F (MgSO4)
</LI></UL><H3><SPAN class="mw-headline" id="Methods">Methods</SPAN></H3><OL><LI>Inoculate <I>B. subtilis</I> W168 into a 50 ml falcon tube containing 10 ml of MM competence medium.
</LI><LI>Incubate overnight in a shaking incubator at 37°C.
</LI><LI> Inoculate 1 ml of the overnight culture into a 125 ml flask containing 10 ml of MM competence medium.
</LI><LI> Incubate the flask at 37°C until cells reach the transition from exponentialphase to the stationary phase between OD (600nm) of 0.4-0.45 (after ~3hrs).
</LI><LI>Warm up the starvation medium to 37°C.
</LI><LI>Add 10 ml of starvation medium (pre-warmed) into each flask and incubate for a further 1.5hrs at 37°C (may need to pre-induce with selection antibiotic: Ery 0.002 µg/ml, Cm 0.05 µg/ml).
</LI><LI>Concentrate  cells 10x by centrifuging 4500 rpm for 10 minutes at room temperature.
</LI><LI>Remove 90% of the supernatant and dissolve the cels in 2 ml that remain.
</LI><LI>Transfer 50 µl of the medium containing <I>B. subtilis</I> into 2 ml tubes, and add appropriate amount of DNA.
</LI><LI>Incubate the samples for 1hr at 37°C in the shaking incubator. The tubes have to be aerated, therefore incubate the tubes on their side.
</LI><LI>Plate the mix onto agar plates containing the appropriate antibiotic markers and incubate plates overnight at 37°C.
</LI></OL><UL><LI><A href="https://2013.igem.org/wiki/index.php?title=Team:Newcastle/Notebook/protocols" class="external text" rel="nofollow">Return to top</A></LI></UL><H2><SPAN class="mw-headline" id="Inoculating_L-forms_into_Chinese_Cabbage">Inoculating L-forms into Chinese Cabbage</SPAN></H2><OL><LI>Rinse seeds for 2 minutes in 70% (v/v) ethanol.
</LI><LI>Soak seeds for 10 minutes in 20% (v/v) Milton’s sterilising fluid.
</LI><LI>Wash seeds thoroughly five times in sterile distilled water and leave in final wash for fifteen minutes.
</LI><LI>Place seeds in (9 cm) Petri dishes (20-25 seeds per dish) containing Murashige and Skoog (M and S) basal medium, solidified with 0.8% (w/v) agar no.1 (Oxoid, UK) and incubate at 25°C in the dark until radicals just appear (this should take 21-24hr for Chinese Cabbage).
</LI><LI>Use spectrophotometry to produce a bacterial suspension containing approximately 10<SUP>7</SUP> CFU ml-1 (OD600: approx. 0.7).
</LI><LI>Select seeds with radicals 1-2mm in length and soak in the bacterial suspension (20 seeds per 10 ml) for 3 hours at 25°C, gently shaking the seeds by hand every 30 minutes. Treat some seedlings with 5% (w/v) mannitol instead of  L-forms to act as a control.
</LI><LI>Wash seeds ten times in distilled water to lyse any extracellular L-forms.
</LI><LI>Replant plants on M and S agar plates. 
</LI><LI>Incubate seeds in sunlight for between 24 hours and 7 days at 25°C.
</LI><LI>The plants are ready to be viewed using microscopy.
</LI></OL><P>References: -
</P><P><A href="http://www.ncbi.nlm.nih.gov/pubmed/12859751" class="external text" rel="nofollow">Tsomlexoglou, E., Daulagala, P.W.H.K.P., Gooday, G.W., Glover, L.A., Seddon, B. and Allan, E.J. (2003) 'Molecular detection and β-glucuronidase expression of gus-marked <I>Bacillus subtilis</I> L-form bacteria in developing Chinese cabbage seedlings', Journal of Applied Microbiology, 95(2), pp. 218-224.</A></P><H2><SPAN class="mw-headline" id="Visualisation_of_L-forms_in_plants">Visualisation of L-forms in plants </SPAN></H2><P>For the visualisation of L-forms in plants, both of the following solutions must be produced prior to commencing the procedure.
</P><UL><LI>Gus staining solution
</LI><LI>0.5 mg ml-1 5-bromo-4-chloro-3-indolyl-ß-D-glucuronide, X-gluc (Sigma)
</LI><LI>100 mM sodium phosphate buffer (pH 7•0)
</LI><LI>1% Triton X-100
</LI><LI>1%N,N' dimethylformamide (DMF) 
</LI><LI>10 mM EDTA
</LI></UL><UL><LI>Fixing solution
</LI><LI>5% formaldehyde (v/v)
</LI><LI>5% (v/v) acetic acid 
</LI><LI>20% (v/v) ethanol
</LI></UL><OL><LI>Harvest batches of five seedlings at daily intervals for seven days after treatment.
</LI><LI>Vacuum-infiltrate intact seedlings in a vacuum oven for 20 minutes with gus staining solution.
</LI><LI>Cover the infiltrated seedlings and incubate in staining solution at 37°C until blue colour appears (approximatly 2 hours).
</LI><LI>Wash seedlings twice with 50% (v/v) glycerol to remove gus staining solution and re-suspend in 50% (v/v) glycerol.
</LI><LI>When blue colour appears, fix in fixing solution.
</LI><LI>Store in 100% ethanol.
</LI></OL><P>References: -
</P><P><A href="http://www.ncbi.nlm.nih.gov/pubmed/12859751" class="external text" rel="nofollow">Tsomlexoglou, E., Daulagala, P.W.H.K.P., Gooday, G.W., Glover, L.A., Seddon, B. and Allan, E.J. (2003) 'Molecular detection and β-glucuronidase expression of gus-marked <I>Bacillus subtilis</I> L-form bacteria in developing Chinese cabbage seedlings', Journal of Applied Microbiology, 95(2), pp. 218-224.</A></P><H2><SPAN class="mw-headline" id="Making_Starch_Agar_Plates">Making Starch Agar Plates</SPAN></H2><OL><LI>Add 1 g starch powder (to 100 ml LB).
</LI><LI>Add 1 g bacterial agar powder.
</LI><LI>Mix by shaking gently and microwave until solute has completely dissolved.
</LI><LI>Pour into pre-labelled petri-dishes (100 ml for 4 plate).
</LI><LI>Remove any bubbles by quickly moving the flame of a Bunsen burner over the poured agar.
</LI><LI>Leave to set 30-45 minutes.
</LI><LI>Put the plates in a fan-cupboard for 10 minutes with their lids partially off to dry out any condensation before plating.
</LI></OL><UL><LI><A href="https://2013.igem.org/wiki/index.php?title=Team:Newcastle/Notebook/protocols" class="external text" rel="nofollow">Return to top</A></LI></UL><DIV class="printfooter">
Retrieved from &quot;<A href="http://2013.igem.org/Team:Newcastle/Notebook/protocols">http://2013.igem.org/Team:Newcastle/Notebook/protocols</A>&quot;</DIV></DIV></DIV><DIV id="footer-box" class="noprint"><DIV id="footer"><UL id="f-list"><LI id="t-recentchanges"><A href="/Special:RecentChanges" title="Recent changes">Recent changes</A></LI><LI id="t-whatlinkshere"><A href="/Special:WhatLinksHere/Team:Newcastle/Notebook/protocols" title="List of all wiki pages that link here [j]" accesskey="j">What links here</A></LI><LI id="t-recentchangeslinked"><A href="/Special:RecentChangesLinked/Team:Newcastle/Notebook/protocols" title="Recent changes in pages linked from this page [k]" accesskey="k">Related changes</A></LI><LI id="t-specialpages"><A href="/Special:SpecialPages" title="List of all special pages [q]" accesskey="q">Special pages</A></LI><LI><A href="/Special:Preferences">My preferences</A></LI></UL></DIV><DIV id="footer"><UL id="f-list"><LI id="t-print"><A href="/wiki/index.php?title=Team:Newcastle/Notebook/protocols&amp;printable=yes" title="Printable version of this page [p]" accesskey="p">Printable version</A></LI><LI id="t-permalink"><A href="/wiki/index.php?title=Team:Newcastle/Notebook/protocols&amp;oldid=303240" title="Permanent link to this revision of the page">Permanent link</A></LI><LI id="privacy"><A href="/2013.igem.org:Privacy_policy" title="2013.igem.org:Privacy policy">Privacy policy</A></LI><LI id="disclaimer"><A href="/2013.igem.org:General_disclaimer" title="2013.igem.org:General disclaimer">Disclaimers</A></LI></UL></DIV></DIV></DIV></DIV></BODY></HTML>