<HTML xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en" dir="ltr">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki  ltr ns-0 ns-subject page-Team_Newcastle_Project_L_forms"><DIV id="globalWrapper"><DIV id="top-section"><DIV id="p-logo"><A href="/Main_Page" title="Main Page">&quot;
	    </A></DIV><DIV id="menubar" class="left-menu noprint"><UL><LI class="selected"><A href="/Team:Newcastle/Project/L_forms">Page               </A></LI><LI class="new"><A href="/wiki/index.php?title=Talk:Team:Newcastle/Project/L_forms&amp;action=edit&amp;redlink=1">Discussion               </A></LI><LI><A href="/wiki/index.php?title=Team:Newcastle/Project/L_forms&amp;action=edit">View source               </A></LI><LI><A href="/wiki/index.php?title=Team:Newcastle/Project/L_forms&amp;action=history">History               </A></LI><LI style="color:white;cursor:default">teams</LI></UL></DIV><DIV class="right-menu noprint" id="menubar"><UL><LI id="pt-login"><A href="/wiki/index.php?title=Special:UserLogin&amp;returnto=Team:Newcastle/Project/L_forms" title="You are encouraged to log in; however, it is not mandatory [o]" accesskey="o">Log in</A></LI></UL></DIV><DIV id="search-controls" class="noprint"><FORM action="/Special:Search" id="searchform"> </FORM></DIV></DIV><DIV id="content"><H1 class="firstHeading">Team:Newcastle/Project/L forms</H1><DIV id="bodyContent"><H3 id="siteSub" class="noprint">From 2013.igem.org</H3><DIV><DIV id="pop_up"><DIV id="pop_up_underlay"> </DIV><DIV id="pop_up_display"><DIV id="pop_up_close">X</DIV><DIV id="pop_up_content"> </DIV></DIV></DIV><DIV class="menu" class="grid_12"><UL class="top_menu" style="margin-left:10px"><LI><A href="https://2013.igem.org/Team:Newcastle">Home</A></LI><LI><A href="#" onclick="return false;">Team</A></LI><LI><A href="https://igem.org/Team.cgi?year=2013&amp;team_name=Newcastle" target="_blank">Official Profile</A></LI><LI><A href="/Team:Newcastle/Team">Individual Profiles</A></LI></UL><LI><A href="#" onclick="return false;">Project</A></LI><LI><A href="https://2013.igem.org/Team:Newcastle/Project">Overview</A></LI><LI><A href="https://2013.igem.org/Team:Newcastle/Project/L_forms">L-forms</A></LI><LI><A href="https://2013.igem.org/Team:Newcastle/Project/shuffling_endosymbiosis">Genome Shuffling</A></LI><LI><A href="https://2013.igem.org/Team:Newcastle/Project/plants">L-forms in Plants</A></LI><LI><A href="https://2013.igem.org/Team:Newcastle/Project/shape_shifting">Shape Shifting</A></LI><LI><A href="#" onclick="return false;">Parts</A></LI><LI><A href="https://2013.igem.org/Team:Newcastle/Parts/l_form_switch">L-form Switch</A></LI><LI><A href="https://2013.igem.org/Team:Newcastle/Parts/HBsu-fp">HBsu-xFP</A></LI><LI><A href="https://2013.igem.org/Team:Newcastle/Parts/e-coli_swich">E-coli Switch</A></LI><LI><A href="#" onclick="return false;">Models</A></LI><LI><A href="https://2013.igem.org/Team:Newcastle/Modelling/Introduction">Introduction</A></LI><LI><A href="https://2013.igem.org/Team:Newcastle/Modelling/BioNetGen">BioNetGen</A></LI><LI><A href="https://2013.igem.org/Team:Newcastle/Modelling/CellShapeModel">Cell Shape Model</A></LI><LI><A href="https://2013.igem.org/Team:Newcastle/Modelling/Cell_Fusion">Cell Fusion</A></LI><LI><A href="https://2013.igem.org/Team:Newcastle/Modelling/HBsu_Fusion_Protein">HBsu-xFP</A></LI><LI><A href="https://2013.igem.org/Team:Newcastle/Modelling/L-form_Switch">L-form Switch</A></LI><LI><A href="#" onclick="return false;">Notebook</A></LI><LI><A href="https://2013.igem.org/Team:Newcastle/Notebook/achievements">Achievements</A></LI><LI><A href="https://2013.igem.org/Team:Newcastle/Notebook/timeline">Timeline</A></LI><LI><A href="https://2013.igem.org/Team:Newcastle/Notebook/calendar">Calendar</A></LI><LI><A href="https://2013.igem.org/Team:Newcastle/Notebook/protocols">Protocols</A></LI><LI><A href="https://2013.igem.org/Team:Newcastle/Notebook/agenda">Meetings</A></LI><LI><A href="#" onclick="return false;">Implications</A></LI><LI><A href="https://2013.igem.org/Team:Newcastle/Outreach/Overview">Overview</A></LI><LI><A href="https://2013.igem.org/Team:Newcastle/Outreach/BioGame">BioGame</A></LI><LI><A href="https://2013.igem.org/Team:Newcastle/Outreach/uk_meet">YSB 1.0</A></LI><LI><A href="https://2013.igem.org/Team:Newcastle/Outreach/Workshop">Modelling Workshop</A></LI><LI><A href="https://2013.igem.org/Team:Newcastle/Outreach/Architecture">Architecture Workshop</A></LI><LI><A href="https://2013.igem.org/Team:Newcastle/HP/Safety">Safety</A></LI><LI><A href="https://2013.igem.org/Team:Newcastle/HP/Synthetic_Bio">Synthetic Biology</A></LI><LI><A href="https://2013.igem.org/Team:Newcastle/HP/Ethics">Ethics</A></LI><LI><A href="https://2013.igem.org/Team:Newcastle/HP/Law">Law</A></LI><LI><A href="#" onclick="return false;">Attributions</A></LI><LI><A href="https://2013.igem.org/Team:Newcastle/Collaborations">Collaborations</A></LI><LI><A href="https://2013.igem.org/Team:Newcastle/Attributions">Acknowledgments</A></LI><LI><A href="https://2013.igem.org/Team:Newcastle/Sponsors">Sponsors</A></LI><LI><A href="https://2013.igem.org/Team:Newcastle/Architecture">Architecture</A></LI><LI><A href="https://2013.igem.org/Main_Page">iGEM</A></LI></DIV><TABLE id="toc" class="toc"><TBODY><TR><TD><DIV id="toctitle"><H2>Contents</H2></DIV><UL><LI class="toclevel-1 tocsection-1"><A href="#L-forms"><SPAN class="tocnumber">1</SPAN><SPAN class="toctext">L-forms</SPAN></A></LI><LI class="toclevel-2 tocsection-2"><A href="#Background"><SPAN class="tocnumber">1.1</SPAN><SPAN class="toctext">Background</SPAN></A></LI><LI class="toclevel-3 tocsection-3"><A href="#L-form_division"><SPAN class="tocnumber">1.1.1</SPAN><SPAN class="toctext">L-form division</SPAN></A></LI></UL><LI class="toclevel-2 tocsection-4"><A href="#Our_Primary_Objective_and_Results"><SPAN class="tocnumber">1.2</SPAN><SPAN class="toctext">Our Primary Objective and Results</SPAN></A></LI><LI class="toclevel-2 tocsection-5"><A href="#Characterisation_of_Growth_and_Division"><SPAN class="tocnumber">1.3</SPAN><SPAN class="toctext">Characterisation of Growth and Division</SPAN></A></LI><LI class="toclevel-3 tocsection-6"><A href="#Growth_rate_of_L-forms"><SPAN class="tocnumber">1.3.1</SPAN><SPAN class="toctext">Growth rate of L-forms</SPAN></A></LI><LI class="toclevel-2 tocsection-7"><A href="#Characterisation_of_Osmotic_Instability"><SPAN class="tocnumber">1.4</SPAN><SPAN class="toctext">Characterisation of Osmotic Instability</SPAN></A></LI><LI class="toclevel-2 tocsection-8"><A href="#Uses_of_L-forms_in_Synthetic_Biology"><SPAN class="tocnumber">1.5</SPAN><SPAN class="toctext">Uses of L-forms in Synthetic Biology</SPAN></A></LI><LI class="toclevel-3 tocsection-9"><A href="#Easier_export_of_cellular_products"><SPAN class="tocnumber">1.5.1</SPAN><SPAN class="toctext">Easier export of cellular products</SPAN></A></LI><LI class="toclevel-3 tocsection-10"><A href="#Gene_fusion.2C_shuffling_and_directed_evolution"><SPAN class="tocnumber">1.5.2</SPAN><SPAN class="toctext">Gene fusion, shuffling and directed evolution</SPAN></A></LI><LI class="toclevel-3 tocsection-11"><A href="#Antibiotic_advances"><SPAN class="tocnumber">1.5.3</SPAN><SPAN class="toctext">Antibiotic advances</SPAN></A></LI><LI class="toclevel-3 tocsection-12"><A href="#Helping_plants"><SPAN class="tocnumber">1.5.4</SPAN><SPAN class="toctext">Helping plants</SPAN></A></LI><LI class="toclevel-2 tocsection-13"><A href="#References"><SPAN class="tocnumber">1.6</SPAN><SPAN class="toctext">References</SPAN></A></LI></TD></TR></TBODY></TABLE><H1><SPAN class="mw-headline" id="L-forms">L-forms</SPAN></H1><H2><SPAN class="mw-headline" id="Background">Background</SPAN></H2><P>We propose the use of L-form bacteria as a chassis for synthetic biology.  L-form strains are derivatives of common cell-walled bacterial strains, however, L-forms are cell wall deficient.  Many modern bacteria have the capacity to switch into L-form state, though specifically we are investigating L-forms in the model Gram-positive bacteria <I>Bacillus subtilis</I>. Unlike protoplasts, which are cells which have chemically treated to remove their cell wall, L-forms are still able to propagate and grow, like their cell-walled counterparts.  This growth and division does not occur in the same way as walled cells - L-forms have been described to undergo <A href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3603455/" class="external text" rel="nofollow">membrane blebbing, tubulation and vesiculation</A>.
</P><P><I>Timeline showing points of interest to our project in the history of L-form research.</I></P><P>As a result of their cell wall-deficiency L-forms are restricted by osmolarity much more than walled bacteria.  L-forms require incubation for many generations in specific osmotically balanced conditions with enzymes and antibiotics that prevent invasion by walled contaminant cells.  However, these specific osmotic demands of L-forms also conveniently work as a kill-switch.  Any bacteria that escape suitable conditions will no longer be in a maintained and osmotically balanced environment and will consequently not survive.
</P><P>For conventional cell-walled bacterial strains to switch into wall-deficient L-form state, the synthesis of the peptidoglycan cell wall must be disrupted.  In <I>B. subtilis</I> this disruption can be achieved through controlling the expression of <I>murE</I>.  The <I>murE</I> gene is responsible for the synthesis of a number of enzymes that are involved in the synthesis of a peptidoglycan precursor.  When <I>murE</I> expression is down-regulated, this has a cascade effect leading to the down-regulation of peptidoglycan synthesis.  It has been found that a mutation that spontaneously occurs in L-form <I>B. subtilis</I> cells is also necessary for the <A href="http://www.ncbi.nlm.nih.gov/pubmed/19212404" class="external text" rel="nofollow">survival of stable L-forms</A>.  After selection for wall-deficient cells,  survivors exhibit a mutation within the <I>yqiD</I> gene, which is similar to the <I>Escherichia coli</I> gene <I>ispA</I>.  This mutation allows <A href="http://www.ncbi.nlm.nih.gov/pubmed/23452849" class="external text" rel="nofollow">for stabilisation of L-forms that are undergoing shape modulation due to an excess of cell membrane</A>.
</P><H3><SPAN class="mw-headline" id="L-form_division">L-form division</SPAN></H3><P>The following videos and figures display the various methods through which L-forms actually divide.  As mentioned above, as L-forms have lost their cell wall they divide independently of FtsZ-ring - a structure that most walled bacteria rely upon for division.  Consequently, growth and division undertaken by L-forms occurs differently than growth and division in cell walled bacteria.
</P><P>Video 1 shows the increase in size of L-form cells over a period of time.  This highlights the difference between L-forms and protoplasts (which neither grow or divide) and suggests the L-forms shown are taking up and metabolising materials.  The L-form cells in this video are tagged with HBsu-GFP to make them more visible.  The L-form cell positioned towards the bottom right of the video is stationary throughout - growth of this cell in particular is clear throughout the duration of the clip.
</P><P><I>Video 1. Showing L-forms with HBsu-GFP tag growing in size from around &lt;2µm up to 6µm, suggesting they take up and metabolise cellular materials.</I>Video taken from <A href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3603455/" class="external text" rel="nofollow">Errington J. (2013)</A></P><P>Video 2 and Figure 3 show division of L-form cells via blebbing.  Surface blebs (initially small bulges) are visible on the membrane of the cell as division begins (430 and 590 minutes in Figure 3, indicated by arrowheads).  These small surface blebs resolve into progeny once fully extruded and split away from the parent cell (indicated by arrows in Figure 3).
</P><P><I>Video 2. Showing larger size L-forms (3.5µm - 6µm) dividing through forming transient bulges around the cell membrane, followed by a series of small bodies erupting from the membrane forming apparent progeny.</I>Video taken from <A href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3603455/" class="external text" rel="nofollow">Errington J. (2013)</A></P><P><I>Figure 3. As shown in Video 2, this figure shows larger size L-forms (3.5µm - 6µm) dividing through forming transient bulges around the cell membrane, followed by a series of small bodies erupting from the membrane forming apparent progeny.  This form of division is known as blebbing.  Each window shows the cell at a different time-frame during incubation.  Time is given in minutes.  Scale bar 5µm.</I>Images taken from <A href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3603455/" class="external text" rel="nofollow">Errington J. (2013)</A></P><P>Video 3 and Figure 4 show division of L-forms via tubulation.  The cell forms a protrusion (from 430 minutes onwards in Figure 4, indicated by arrowheads) over a period of time.  This protrusion extends before splitting into six resolved cells (visible at the end of Video 3 and at 560 minutes in Figure 4).  Another cell becomes visible over the course of this division (marked with an asterisk in Figure 4).  This second cell was not the result of division of the first cell, simply a second L-form cell that had moved into the field of vision.
</P><P><I>Video 3. Showing the common division mechanism that L-forms around 2um in diameter undergo. Initially a series of disruptions can be seen on the membranes, which is characterised as a blunt protrusion. This protrusion retracts and re-emerges before being resolved into multiple progeny.</I>Video taken from <A href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3603455/" class="external text" rel="nofollow">Errington J. (2013)</A></P><P><I>Figure 4. As shown in Video 3 this figure shows the common division mechanism that L-forms around 2µm in diameter undergo. Initially a series of disruptions can be seen on the membranes, which is characterised as a blunt protrusion. This protrusion retracts and re-emerges before being resolved into multiple progeny.  This form of division is known as tubulation.  Each window shows the cell at a different time-frame in incubation.  Time is given in minutes.  Scale bar 5µm.</I> Images taken from <A href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3603455/" class="external text" rel="nofollow">Errington J. (2013)</A></P><P>Video 4 and Figure 5 show division of L-forms via vesiculation.  The cell (indicated by arrow in Figure 5 at 10 minutes) divides (after 160 minutes in Figure 5) to produce two cells (indicated by arrows at 220 minutes in Figure 5), which both undergo a second division (after 350 minutes in Figure 5).  This second division produces both symmetrical progeny (indicated by arrows at 430 minutes in Figure 5) and asymmetrical progeny (indicated by arrowheads at 430 minutes in Figure 5).  Another cell visible during this period (marked with an asterisk in Figure 5) does not undergo division during the course of observation.
</P><P><I>Video 4. An often-used method of division by L-forms with a diameter less than 2µm. They undergo a similar process as larger L-forms, the only difference is that they usually gave rise to progeny of equal sizes.</I>Video taken from <A href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3603455/" class="external text" rel="nofollow">Errington J. (2013)</A></P><P><I>Figure 5.  As shown in Video 4, this figure shows an often-used method of division by L-forms &lt;2µm. They undergo a similar process as larger L-forms, the only difference is that they usually gave rise to progeny of equal sizes.  This form of division is known as vesiculation.  Each window shows the cell at a different time-frame in incubation.  Time is given in minutes. Scale bar 5µm.</I>Images taken from <A href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3603455/" class="external text" rel="nofollow">Errington J. (2013)</A></P><P>In Figure 6, division events can be seen in greater detail, as viewed under a transmission electron microscope.  This figure shows two cells joined by a thin strand (A), presumably after a division event.  Also shown is the division of an L-form cell into two cells at a stage of resolution (B); here the two resultant cells can clearly be distinguished.  Finally, an L-form cell that appears to be blebbing can be seen (C).  Blebbing events are visible at the surface of the membrane.  Smaller L-forms are also visible around the larger L-form cell, these may be the product of earlier blebbing events.
</P><P><I>Figure 6. <B>a-c,</B> Transmission electron micrographs of dividing L-forms. <B>a</B>, showing two cells connected by a thin strand (arrow, and inset at 3 times magnification). <B>b</B>, a cell at late stage of resolution. <B>c</B>, a cell with surface blebs and possible progeny (arrows). Scale bars, 500 nm.</I> Images taken from <A href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3603455/" class="external text" rel="nofollow">Errington J. (2013)</A></P><P>The <I>B. subtilis</I> strain <A href="http://www.ncbi.nlm.nih.gov/pubmed/23452849" class="external text" rel="nofollow">LR2</A> - an L-form derivative of <I>B. subtilis</I> 168 created by the Errington research group - contains a xylose-controlled promoter <I>PxylR</I> upstream of the <I>murE</I> gene on its chromosome (<A href="http://www.ncbi.nlm.nih.gov/pubmed/22122227" class="external text" rel="nofollow">Leaver M et al. 2012</A>).  This allows for xylose-mediated control over <I>murE</I> expression.  This strain also contains a chloramphenicol acetyl-transferase (<I>cat</I>) gene close to the locus of the <I>PxylR</I> promoter.  This gene confers resistance to chloramphenicol and allows for selection for <I>B. subtilis</I> LR2 when in rod form.
</P><H2><SPAN class="mw-headline" id="Our_Primary_Objective_and_Results">Our Primary Objective and Results</SPAN></H2><P>The primary objective of our project was to create a BioBrick that enables the conversion of cell-walled <I>B. subtilis</I> cells into L-form cells - that are cell wall deficient, yet still grow and divide.  It was important that L-forms should be able to be maintained in wall-less state once conversion from walled form was successfully completed.  The BioBrick also had to facilitate the reversion of L-forms back to walled cells when desired.
</P><P>Fulfilment of this defined criteria ensured the provision of a BioBrick that would effectively operate as a switch, enabling the cell wall to be switched both on and off at the whim of the synthetic biologist.
</P><P>We sequenced the region of containing the PxylR promoter and the cat gene from the Errington LR2 strain. We used this sequence to inform the model-based computational design of our own, synthetic, refactored, switch biobrick, both for <I>B. subtilis</I> (<A href="http://parts.igem.org/Part:BBa_K1185000" class="external text" rel="nofollow">BBa_K1185000</A>) and <I>E. coli</I> (<A href="http://parts.igem.org/Part:BBa_K1185000" class="external text" rel="nofollow">BBa_K1185005</A>). We had the <I>B. subtilis</I> design synthesised by DNA2.0 and integrated it onto the <I>B. subtilis</I> chromosome. 
</P><P>The L-form switch BioBrick  for <I>B. subtilis</I> that was fully characterised and a design for <I>E. coli</I> that was designed left for another team to characterise. 
</P><P>We also investigated a number of L-form applications to demonstrate their utility to the Synthetic Biology community.
</P><H2><SPAN class="mw-headline" id="Characterisation_of_Growth_and_Division">Characterisation of Growth and Division</SPAN></H2><P>With the removal of the cell wall, growth and division occurs differently than the same process in cell walled bacteria.  L-forms have been described to undertake membrane blebbing, tubulation and vesiculation as forms of division.  Proliferation of <I>B. subtilis</I> in L-form state occurs <A href="http://www.ncbi.nlm.nih.gov/pubmed/22122227" class="external text" rel="nofollow">entirely independently of the FtsZ-ring</A>, which is necessary for division in almost all walled bacteria. Because of this intriguing process of growth and division, we investigated the rate of growth of populations of <I>B. subtilis</I> L-forms over a period of time.  To achieve this, the optical density of L-form cell cultures was taken at 600nm, as an indicator of population growth (higher optical density occurs with higher population size). Growth curves from this data were produced.  Growth curves were also produced for <I>B. subtilis</I> BSB1 and 168 rod cells that maintained a cell wall for comparison.
</P><H3><SPAN class="mw-headline" id="Growth_rate_of_L-forms">Growth rate of L-forms</SPAN></H3><P>Table 1 summarises the change in optical density at 600nm (as a result of increasing number of cells) of cellular cultures of rod-formed <I>B. subtilis</I> str. BSB1 and <I>B. subtilis</I> str. 168 at 30°C.  Figure 1 shows the growth curves plotted from this data.  A slow increase in the optical density (and thus number of cells in the culture) of both cultures is noticeable in the early stages of observation.  This corresponds to the expected lag phase of a growth curve.  After roughly 100 minutes the growth rate increases, which can be seen as a steepening of the curve in Figure 1.  The cell cultures are in their exponential phases of growth here.  As both cultures approach 500 minutes of incubation, the growth rate no longer appears steady as the cell populations approach the stationary phase.
</P><P><I>Table 1. This table shows the change in optical density at 600nm, with time, as B.subtilis str. BSB1 and 168, in their rod form, are left to grow at 30°C.</I></P><P><I>Figure 1. This graph shows the change in optical density at 600nm, with time, as B.subtilis str. BSB1 and 168, in their rod form, are left to grow at 30°C.</I></P><P>Table 2 summarises the change in optical density at 600nm (as a result of increasing number of cells) of cell cultures of <I>B. subtilis</I> L-forms.  Figure 2 shows the growth curve plotted from this data.  As with the cell culture of the walled <I>B. subtilis</I> an expected growth curve is produced - with lag, exponential and stationary phases visible on the curve in Figure 2.  However, there is a great difference in time-frame.  The optical density of <I>B. subtilis</I> rod cell cultures was measured over hundreds of minutes.  These cultures approached stationary phase at around 500 minutes (just over 8 hours).  The L-form cell cultures required incubation for around 140 hours before approaching stationary phase.  Also noticeable is the much lower optical densities of the L-form cell culture compared to the rod cell culture, throughout the observed time period.
</P><P><I>Table 2. This table shows the change in optical density at 600nm absorbance, with time, as L-forms of B.subtilis are left to grow at 30°C.</I></P><P><I>Figure 2. This graph shows the change in optical density at 600nm absorbance, with time, as L-forms of B.subtilis are left to grow at 30°C.</I></P><H2><SPAN class="mw-headline" id="Characterisation_of_Osmotic_Instability">Characterisation of Osmotic Instability</SPAN></H2><P>One way that L-forms differ from their cell walled counterparts is in their inability to survive in a wide range of osmotic pressures. This adds biosecurity that any L-forms that escape into the environment will not survive. Such biosecurity provides one reason why genetically modified L-forms could be used in the agriculture industry as an alternative to other engineered bacteria. We conducted experiments to investigate what range of osmotic pressures are suitable for our <I>B. subtilis</I> L-forms. This included growing L-forms in 50/50 NB/MSM media with a variety of sucrose concentrations. A soil sample was also taken from outside the Centre for Bacterial Cell Biology at Newcastle University, from which water was extracted. This was used to test the ability of L-forms to survive in soil water, in the event that they escape from a laboratory or a host plant. 
</P><P>Table 3 summarises the changes in optical density, and thus number of L-form cells in culture, relative to sucrose concentration (osmotic pressure). The rate of decrease in L-form numbers (%) (measured through optical density) in one hour drops as the sucrose concentration increases. In media with 0M sucrose the rate of decrease is over 90%. This drops to less than 50% in media containing 0.4M sucrose. In soil water the rate of decrease is over 95% highlighting the unlikelihood of L-forms surviving if they escaped the environment - this decrease in number of L-forms is actually higher than that observed in the laboratory NB/MSM media that was supplemented with 0M sucrose. The rate of decrease in the 0.5M sucrose media is -1.28 (2.d.p.) as rather than the L-forms bursting in large numbers, the L-forms are actually able to proliferate in the osmotically balanced conditions.
</P><TABLE class="data"><TBODY><TR><TH>Sucrose Concentration (M)</TH><TH>Time (mins)</TH><TH>O.D (600)</TH><TH>Rate of Decrease (%)</TH></TR><TR class="odd"><TD class="data">0</TD><TD class="data">0.640</TD></TR><TR class="odd"><TD class="data">10</TD><TD class="data">0.126</TD></TR><TR class="odd"><TD class="data">20</TD><TD class="data">0.114</TD></TR><TR class="odd"><TD class="data">0 (positive control)</TD><TD class="data">30</TD><TD class="data">0.106</TD><TD class="data">90.31</TD></TR><TR class="odd"><TD class="data">40</TD><TD class="data">0.100</TD></TR><TR class="odd"><TD class="data">50</TD><TD class="data">0.080</TD></TR><TR class="odd"><TD class="data">60</TD><TD class="data">0.062</TD></TR><TR class="even"><TD class="data">0</TD><TD class="data">0.691</TD></TR><TR class="even"><TD class="data">10</TD><TD class="data">0.123</TD></TR><TR class="even"><TD class="data">20</TD><TD class="data">0.100</TD></TR><TR class="even"><TD class="data">0.1</TD><TD class="data">30</TD><TD class="data">0.091</TD><TD class="data">88.71</TD></TR><TR class="even"><TD class="data">40</TD><TD class="data">0.083</TD></TR><TR class="even"><TD class="data">50</TD><TD class="data">0.080</TD></TR><TR class="even"><TD class="data">60</TD><TD class="data">0.078</TD></TR><TR class="odd"><TD class="data">0</TD><TD class="data">1.705</TD></TR><TR class="odd"><TD class="data">10</TD><TD class="data">0.676</TD></TR><TR class="odd"><TD class="data">20</TD><TD class="data">0.605</TD></TR><TR class="odd"><TD class="data">0.2</TD><TD class="data">30</TD><TD class="data">0.559</TD><TD class="data">74.25</TD></TR><TR class="odd"><TD class="data">40</TD><TD class="data">0.523</TD></TR><TR class="odd"><TD class="data">50</TD><TD class="data">0.491</TD></TR><TR class="odd"><TD class="data">60</TD><TD class="data">0.439</TD></TR><TR class="even"><TD class="data">0</TD><TD class="data">1.604</TD></TR><TR class="even"><TD class="data">10</TD><TD class="data">0.828</TD></TR><TR class="even"><TD class="data">20</TD><TD class="data">0.772</TD></TR><TR class="even"><TD class="data">0.3</TD><TD class="data">30</TD><TD class="data">0.729</TD><TD class="data">59.16</TD></TR><TR class="even"><TD class="data">40</TD><TD class="data">0.685</TD></TR><TR class="even"><TD class="data">50</TD><TD class="data">0.672</TD></TR><TR class="even"><TD class="data">60</TD><TD class="data">0.655</TD></TR><TR class="odd"><TD class="data">0</TD><TD class="data">1.546</TD></TR><TR class="odd"><TD class="data">10</TD><TD class="data">0.984</TD></TR><TR class="odd"><TD class="data">20</TD><TD class="data">0.891</TD></TR><TR class="odd"><TD class="data">0.4</TD><TD class="data">30</TD><TD class="data">0.865</TD><TD class="data">49.55</TD></TR><TR class="odd"><TD class="data">40</TD><TD class="data">0.847</TD></TR><TR class="odd"><TD class="data">50</TD><TD class="data">0.810</TD></TR><TR class="odd"><TD class="data">60</TD><TD class="data">0.780</TD></TR><TR class="even"><TD class="data">0</TD><TD class="data">1.768</TD></TR><TR class="even"><TD class="data">10</TD><TD class="data">0.222</TD></TR><TR class="even"><TD class="data">20</TD><TD class="data">0.190</TD></TR><TR class="even"><TD class="data">Soil</TD><TD class="data">30</TD><TD class="data">0.150</TD><TD class="data">95.93</TD></TR><TR class="even"><TD class="data">40</TD><TD class="data">0.128</TD></TR><TR class="even"><TD class="data">50</TD><TD class="data">0.096</TD></TR><TR class="even"><TD class="data">60</TD><TD class="data">0.072</TD></TR><TR class="odd"><TD class="data">0</TD><TD class="data">1.569</TD></TR><TR class="odd"><TD class="data">10</TD><TD class="data">1.569</TD></TR><TR class="odd"><TD class="data">20</TD><TD class="data">1.569</TD></TR><TR class="odd"><TD class="data">0.5 (negative control)</TD><TD class="data">30</TD><TD class="data">1.583</TD><TD class="data">-1.28</TD></TR><TR class="odd"><TD class="data">40</TD><TD class="data">1.584</TD></TR><TR class="odd"><TD class="data">50</TD><TD class="data">1.584</TD></TR><TR class="odd"><TD class="data">60</TD><TD class="data">1.589</TD></TR></TBODY></TABLE><P><I>Table 3. Change in the optical density of L-forms in media with different sucrose concentrations and also soil water, over time. This highlights how a change in osmotic pressure can affect the propensity of L-forms to survive.</I></P><P>The results are also summarised in Figure 7, which shows the number of cells plummeting in all the media except the one containing 0.5M sucrose. This highlights that L-forms require an environment with a high osmolarity to survive. The L-forms were also unable to survive in the soil water sample as expected. 
</P><P><I>Figure 7. Change in the optical density of L-forms in media with different sucrose concentrations and also soil water, over time. This highlights how a change in osmotic pressure can affect the propensity of L-forms to survive.</I></P><P>These results are highlighted further in Figure 8.  Here microscopy of samples taken from each of the cultures can be seen.  Clear and intact L-forms are only visible in the sample taken from the culture grown in 0.5M sucrose media.  Fluorescence microscopy (image on the right for each sample) shows that there is cell material present in each of the other samples.  However, as can be seen in the phase contrast image (left image for each sample), cell material appears to be debris and the remnants of disrupted and burst L-forms rather than the clear and intact, circular L-form cell images visible in the sample taken from the culture grown in 0.5M sucrose media.
</P><P><TABLE border="0"><TBODY><TR><TD>Figure 8. Microscopy of L-forms after being left in NB/MSM media with varying sucrose concentrations for 60 minutes. There is also an image of L-forms that have been left in NB/MSM media + soil water for 60 minutes.</TD></TR></TBODY></TABLE></P><P>To further highlight the osmotic instability of L-forms, Video 5 shows fluorescing L-forms appearing to disappear in a microfluidics apparatus. These L-forms are actually bursting due to the fact that soil water is flooding the microfluidics chip that they are contained in. This displays the effects of non-osmotically balanced media on L-forms in an environment where the L-forms can be held in position to be viewed.  That the L-form cells burst when the soil water is added (which supports our findings outlined above) gives backing to the concept of biosecurity and an in-built kill-switch preventing escape of L-forms.  As suggested, should L-form cells escape a laboratory or escape a plant in which they are supported, the non-osmotically balanced environment (soil water in this demonstration) in which they find themselves will lead to lysing and bursting of the cell. The lack of xylose in the environment will ensure that the cells will be in L-form state.
</P><P><I>Video 5. Showing L-forms with HBsu-sfGFP and L-forms with HBsu-RFP tag inside microfluidics chamber lysing upon the introduction of soil water.</I></P><H2><SPAN class="mw-headline" id="Uses_of_L-forms_in_Synthetic_Biology">Uses of L-forms in Synthetic Biology</SPAN></H2><P>The applications and research potential afforded to Synthetic Biology through employment of L-forms are numerous.  All of these areas benefit from the removal of the cell wall from cells - as occurs when cells adopt the L-form state.  A few are listed below though, as is the case in Synthetic Biology as a whole, the number of applications that L-form employment could be used towards is limited only by imagination. The purpose of our project, and in particular the development of an L-form switch BioBrick, is to more easily facilitate the exploration of these applications and the research that L-forms provide.
</P><H4><SPAN class="mw-headline" id="Easier_export_of_cellular_products">Easier export of cellular products</SPAN></H4><P>L-forms, characteristically, have no cell wall. Having eliminated one of its main barriers, it is now easier to move things in and out of the cell. <I>B. subtilis</I> is a popularly used organism in biotechnology, in part because of its ability to export proteinaceous products. Bacteria in L-form state secrete more proteins than their cell-walled counterparts. This means that proteins that cannot usually be secreted due to the barrier of the cell wall can now be secreted and that proteins can be secreted at increased rates.  These characteristics may make L-forms useful candidates for the industrial synthesis of cellular products.
</P><H4><SPAN class="mw-headline" id="Gene_fusion.2C_shuffling_and_directed_evolution">Gene fusion, shuffling and directed evolution</SPAN></H4><P>Bacteria in L-form state are able to fuse together. When they do this, their genomes can recombine or “shuffle”.  Resultant &quot;offspring&quot; cells may contain a mixture of genes from these two fusant cells - in effect the fusants will reproduce sexually.  Some of the bacteria, which result from genome shuffling will show more useful characteristics than the original bacteria. By selecting improved bacteria and forcing them to recombine their genomes, even more useful bacteria can result. The use of iterative mutation and selection termed &quot;directed evolution&quot;.  Via this method of directed evolution new useful strains or potentially even new species may be created.  We explored this area of L-form application in our project through attempting to show the fusion of two L-forms, and more specifically the recombination of their DNA.
</P><H4><SPAN class="mw-headline" id="Antibiotic_advances">Antibiotic advances</SPAN></H4><P>Cell wall-deficient L-forms also provide a great research tool for antibiotic discovery and progression.  It is suggested that bacteria are able to adopt L-form state to avoid threats to their cell wall.  As many antibiotics target the cell wall, the adoption of L-form state provides bacteria resistance to many antibiotics.  Research undertaken with L-forms can allow for greater in-depth understanding of the mechanisms through which antibiotic resistance can be achieved.  Further knowledge in this area may give antibiotic development a ‘leg-up’ in the arms race against antibiotic-resistant pathogens.  Furthermore, through trialling against L-forms, novel antibiotics, which work through non-cell wall-targeted means, may be discovered.
</P><H4><SPAN class="mw-headline" id="Helping_plants">Helping plants</SPAN></H4><P><A href="http://www.ncbi.nlm.nih.gov/pubmed/11849491" class="external text" rel="nofollow">L-forms have been shown</A> to form symbiotic relationships with some plants. Such symbioses consist of L-form bacteria helping protect their plant hosts from fungal infections; the plants provide a suitable environment for the L-forms to grow. This symbiosis could be manipulated to confer further benefits to plants. If L-forms could be engineered to deliver useful substances to plants then agriculture could really benefit.  This area of L-form application was explored in our project - we endeavoured to visually prove the existence of L-form <I>B. subtilis</I> within plants.
</P><H2><SPAN class="mw-headline" id="References">References</SPAN></H2><P><A href="http://www.ncbi.nlm.nih.gov/pubmed/22122227" class="external text" rel="nofollow">Domínguez-Cuevas P, Mercier R, Leaver M, Kawai Y, Errington J. (2012) The rod to L-form transition of Bacillus subtilis is limited by a requirement for the protoplast to escape from the cell wall sacculus. <I>Molecular Microbiology</I>, <B>83</B>, 52-66.</A></P><P><A href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3603455/" class="external text" rel="nofollow">Errington J. (2013) L-form bacteria, cell walls and the origins of life. <I>Open Biology</I>,  <B>3</B>, 120143.</A></P><P><A href="http://www.ncbi.nlm.nih.gov/pubmed/19212404" class="external text" rel="nofollow">Leaver M., Dominguez-CuevasP., Coxhead J.M., Daniel R.A. and Errington J. (2009) Life without a wall or division machine in Bacillus subtilis.  <I>Nature</I>, <B>457</B>, 849-853.</A></P><P><A href="http://www.ncbi.nlm.nih.gov/pubmed/23452849" class="external text" rel="nofollow">Mercier R., Kawai Y. and Errington J. (2013) Excess membrane synthesis drives a primitive mode of cell proliferation. <I>Cell</I>, <B>152</B>, 997-1007.</A></P><P><A href="http://www.ncbi.nlm.nih.gov/pubmed/11849491" class="external text" rel="nofollow">Walker R., Ferguson CM., Booth NA and Allan EJ.(2002) The symbiosis of Bacillus subtilis L-forms with Chinese cabbage seedlings inhibits conidial germination of <I>Botrytis cinerea</I>. <I>Letters in Applied Microbiology</I>. <B>34</B>, 42-45.</A></P><DIV class="printfooter">
Retrieved from &quot;<A href="http://2013.igem.org/Team:Newcastle/Project/L_forms">http://2013.igem.org/Team:Newcastle/Project/L_forms</A>&quot;</DIV></DIV></DIV><DIV id="footer-box" class="noprint"><DIV id="footer"><UL id="f-list"><LI id="t-recentchanges"><A href="/Special:RecentChanges" title="Recent changes">Recent changes</A></LI><LI id="t-whatlinkshere"><A href="/Special:WhatLinksHere/Team:Newcastle/Project/L_forms" title="List of all wiki pages that link here [j]" accesskey="j">What links here</A></LI><LI id="t-recentchangeslinked"><A href="/Special:RecentChangesLinked/Team:Newcastle/Project/L_forms" title="Recent changes in pages linked from this page [k]" accesskey="k">Related changes</A></LI><LI id="t-specialpages"><A href="/Special:SpecialPages" title="List of all special pages [q]" accesskey="q">Special pages</A></LI><LI><A href="/Special:Preferences">My preferences</A></LI></UL></DIV><DIV id="footer"><UL id="f-list"><LI id="t-print"><A href="/wiki/index.php?title=Team:Newcastle/Project/L_forms&amp;printable=yes" title="Printable version of this page [p]" accesskey="p">Printable version</A></LI><LI id="t-permalink"><A href="/wiki/index.php?title=Team:Newcastle/Project/L_forms&amp;oldid=354158" title="Permanent link to this revision of the page">Permanent link</A></LI><LI id="privacy"><A href="/2013.igem.org:Privacy_policy" title="2013.igem.org:Privacy policy">Privacy policy</A></LI><LI id="disclaimer"><A href="/2013.igem.org:General_disclaimer" title="2013.igem.org:General disclaimer">Disclaimers</A></LI></UL></DIV></DIV></DIV></DIV></BODY></HTML>