<HTML xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en" dir="ltr">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki  ltr ns-0 ns-subject page-Team_Newcastle_Modelling_HBsu_Fusion_Protein"><DIV id="globalWrapper"><DIV id="top-section"><DIV id="p-logo"><A href="/Main_Page" title="Main Page">&quot;
	    </A></DIV><DIV id="menubar" class="left-menu noprint"><UL><LI class="selected"><A href="/Team:Newcastle/Modelling/HBsu_Fusion_Protein">Page               </A></LI><LI class="new"><A href="/wiki/index.php?title=Talk:Team:Newcastle/Modelling/HBsu_Fusion_Protein&amp;action=edit&amp;redlink=1">Discussion               </A></LI><LI><A href="/wiki/index.php?title=Team:Newcastle/Modelling/HBsu_Fusion_Protein&amp;action=edit">View source               </A></LI><LI><A href="/wiki/index.php?title=Team:Newcastle/Modelling/HBsu_Fusion_Protein&amp;action=history">History               </A></LI><LI style="color:white;cursor:default">teams</LI></UL></DIV><DIV class="right-menu noprint" id="menubar"><UL><LI id="pt-login"><A href="/wiki/index.php?title=Special:UserLogin&amp;returnto=Team:Newcastle/Modelling/HBsu_Fusion_Protein" title="You are encouraged to log in; however, it is not mandatory [o]" accesskey="o">Log in</A></LI></UL></DIV><DIV id="search-controls" class="noprint"><FORM action="/Special:Search" id="searchform"> </FORM></DIV></DIV><DIV id="content"><H1 class="firstHeading">Team:Newcastle/Modelling/HBsu Fusion Protein</H1><DIV id="bodyContent"><H3 id="siteSub" class="noprint">From 2013.igem.org</H3><DIV><DIV id="pop_up"><DIV id="pop_up_underlay"> </DIV><DIV id="pop_up_display"><DIV id="pop_up_close">X</DIV><DIV id="pop_up_content"> </DIV></DIV></DIV><DIV class="menu" class="grid_12"><UL class="top_menu" style="margin-left:10px"><LI><A href="https://2013.igem.org/Team:Newcastle">Home</A></LI><LI><A href="#" onclick="return false;">Team</A></LI><LI><A href="https://igem.org/Team.cgi?year=2013&amp;team_name=Newcastle" target="_blank">Official Profile</A></LI><LI><A href="/Team:Newcastle/Team">Individual Profiles</A></LI></UL><LI><A href="#" onclick="return false;">Project</A></LI><LI><A href="https://2013.igem.org/Team:Newcastle/Project">Overview</A></LI><LI><A href="https://2013.igem.org/Team:Newcastle/Project/L_forms">L-forms</A></LI><LI><A href="https://2013.igem.org/Team:Newcastle/Project/shuffling_endosymbiosis">Genome Shuffling</A></LI><LI><A href="https://2013.igem.org/Team:Newcastle/Project/plants">L-forms in Plants</A></LI><LI><A href="https://2013.igem.org/Team:Newcastle/Project/shape_shifting">Shape Shifting</A></LI><LI><A href="#" onclick="return false;">Parts</A></LI><LI><A href="https://2013.igem.org/Team:Newcastle/Parts/l_form_switch">L-form Switch</A></LI><LI><A href="https://2013.igem.org/Team:Newcastle/Parts/HBsu-fp">HBsu-xFP</A></LI><LI><A href="https://2013.igem.org/Team:Newcastle/Parts/e-coli_swich">E-coli Switch</A></LI><LI><A href="#" onclick="return false;">Models</A></LI><LI><A href="https://2013.igem.org/Team:Newcastle/Modelling/Introduction">Introduction</A></LI><LI><A href="https://2013.igem.org/Team:Newcastle/Modelling/BioNetGen">BioNetGen</A></LI><LI><A href="https://2013.igem.org/Team:Newcastle/Modelling/CellShapeModel">Cell Shape Model</A></LI><LI><A href="https://2013.igem.org/Team:Newcastle/Modelling/Cell_Fusion">Cell Fusion</A></LI><LI><A href="https://2013.igem.org/Team:Newcastle/Modelling/HBsu_Fusion_Protein">HBsu-xFP</A></LI><LI><A href="https://2013.igem.org/Team:Newcastle/Modelling/L-form_Switch">L-form Switch</A></LI><LI><A href="#" onclick="return false;">Notebook</A></LI><LI><A href="https://2013.igem.org/Team:Newcastle/Notebook/achievements">Achievements</A></LI><LI><A href="https://2013.igem.org/Team:Newcastle/Notebook/timeline">Timeline</A></LI><LI><A href="https://2013.igem.org/Team:Newcastle/Notebook/calendar">Calendar</A></LI><LI><A href="https://2013.igem.org/Team:Newcastle/Notebook/protocols">Protocols</A></LI><LI><A href="https://2013.igem.org/Team:Newcastle/Notebook/agenda">Meetings</A></LI><LI><A href="#" onclick="return false;">Implications</A></LI><LI><A href="https://2013.igem.org/Team:Newcastle/Outreach/Overview">Overview</A></LI><LI><A href="https://2013.igem.org/Team:Newcastle/Outreach/BioGame">BioGame</A></LI><LI><A href="https://2013.igem.org/Team:Newcastle/Outreach/uk_meet">YSB 1.0</A></LI><LI><A href="https://2013.igem.org/Team:Newcastle/Outreach/Workshop">Modelling Workshop</A></LI><LI><A href="https://2013.igem.org/Team:Newcastle/Outreach/Architecture">Architecture Workshop</A></LI><LI><A href="https://2013.igem.org/Team:Newcastle/HP/Safety">Safety</A></LI><LI><A href="https://2013.igem.org/Team:Newcastle/HP/Synthetic_Bio">Synthetic Biology</A></LI><LI><A href="https://2013.igem.org/Team:Newcastle/HP/Ethics">Ethics</A></LI><LI><A href="https://2013.igem.org/Team:Newcastle/HP/Law">Law</A></LI><LI><A href="#" onclick="return false;">Attributions</A></LI><LI><A href="https://2013.igem.org/Team:Newcastle/Collaborations">Collaborations</A></LI><LI><A href="https://2013.igem.org/Team:Newcastle/Attributions">Acknowledgments</A></LI><LI><A href="https://2013.igem.org/Team:Newcastle/Sponsors">Sponsors</A></LI><LI><A href="https://2013.igem.org/Team:Newcastle/Architecture">Architecture</A></LI><LI><A href="https://2013.igem.org/Main_Page">iGEM</A></LI></DIV><TABLE id="toc" class="toc"><TBODY><TR><TD><DIV id="toctitle"><H2>Contents</H2></DIV><UL><LI class="toclevel-1 tocsection-1"><A href="#HBsu-xFP"><SPAN class="tocnumber">1</SPAN><SPAN class="toctext">HBsu-xFP</SPAN></A></LI><LI class="toclevel-2 tocsection-2"><A href="#The_Model"><SPAN class="tocnumber">1.1</SPAN><SPAN class="toctext">The Model</SPAN></A></LI><LI class="toclevel-2 tocsection-3"><A href="#Methods"><SPAN class="tocnumber">1.2</SPAN><SPAN class="toctext">Methods</SPAN></A></LI><LI class="toclevel-2 tocsection-4"><A href="#Results"><SPAN class="tocnumber">1.3</SPAN><SPAN class="toctext">Results</SPAN></A></LI><LI class="toclevel-3 tocsection-5"><A href="#0mM_IPTG"><SPAN class="tocnumber">1.3.1</SPAN><SPAN class="toctext">0mM IPTG</SPAN></A></LI><LI class="toclevel-3 tocsection-6"><A href="#0.05mM_IPTG"><SPAN class="tocnumber">1.3.2</SPAN><SPAN class="toctext">0.05mM IPTG</SPAN></A></LI><LI class="toclevel-3 tocsection-7"><A href="#0.1mM_IPTG"><SPAN class="tocnumber">1.3.3</SPAN><SPAN class="toctext">0.1mM IPTG</SPAN></A></LI><LI class="toclevel-3 tocsection-8"><A href="#0.2mM_IPTG"><SPAN class="tocnumber">1.3.4</SPAN><SPAN class="toctext">0.2mM IPTG</SPAN></A></LI><LI class="toclevel-3 tocsection-9"><A href="#0.4mM_IPTG"><SPAN class="tocnumber">1.3.5</SPAN><SPAN class="toctext">0.4mM IPTG</SPAN></A></LI><LI class="toclevel-3 tocsection-10"><A href="#0.8mM_IPTG"><SPAN class="tocnumber">1.3.6</SPAN><SPAN class="toctext">0.8mM IPTG</SPAN></A></LI></UL><LI class="toclevel-2 tocsection-11"><A href="#References"><SPAN class="tocnumber">1.4</SPAN><SPAN class="toctext">References</SPAN></A></LI></TD></TR></TBODY></TABLE><H1><SPAN class="mw-headline" id="HBsu-xFP">HBsu-xFP</SPAN></H1><P>The BioBrick we created to visualize genome shuffling is under the control of an IPTG-inducible promoter. Normally a repressor protein is bound to it, preventing gene transcription. However IPTG can binds to the repressor, changing its shape and removing it from the promoter, allowing expression and production of HBsu-xFP.
Using BioNetGen we attempted to model this system, and see if we could predict the time it took for gene expression to start, given initial concentrations of IPTG. As we had no access to Western Blotting or other techniques to determine protein production experimentally, we also attempted to link fluorescence to the average molecule numbers our model predicted.</P><H3><SPAN class="mw-headline" id="The_Model">The Model</SPAN></H3><P>The model can be split into a few sections. The main part is the binding of 2 IPTG molecules to the protein repressing the BioBrick promoter, the repressor leaving the DNA, allowing gene expression.
</P><PRE>IPTG(p) + DNA(prot!0,promoter~0).Rep(dna!0,a) &lt;-&gt; IPTG(p) + 
DNA(prot!0,promoter~0).Rep(dna!0,a!1).IPTG(p!1)
</PRE><P>This represents the bonding of both IPTG molecules to the repressor
</P><PRE>IPTG(p!1).Rep(a!1,a!2,dna!3).IPTG(p!2).DNA(prot!3,promoter~0) -&gt; 
IPTG(p!1).Rep(a!1,a!2,dna).IPTG(p!2)+ DNA(prot,promoter~1)
</PRE><P>Here the repressor leaves the operator region upstream of the BioBrick. Note that DNA(promoter~0) has changed to DNA(promoter~1) signifying that the DNA can now be transcribed.
</P><P>Other repressor protein not bound to the DNA will also be attacked by IPTG, preventing them from binding to the now free DNA. This is represented by the reaction rule:
</P><PRE>IPTG(p) + Rep(dna,a) &lt;-&gt; IPTG(p!1).Rep(a!1,dna)
</PRE><P>Free repressor can still bind to the DNA however:
</P><PRE>DNA(prot,promoter~1) + Rep(dna,a,a) -&gt; 
DNA(prot!0,promoter~0).Rep(dna!0,a,a)
</PRE><P>The next stage is the transcription and translation of the HBsu-xFP gene:
</P><PRE>DNA(prot,promoter~1) -&gt; DNA(prot,promoter~1) + hbsu_mRNA()
hbsu_mRNA() -&gt; hbsu_mRNA() + HBsu()
</PRE><P>The protein will however slowly degrade:
</P><PRE>HBsu()  -&gt; 0 
</PRE><P>Before binding to the repressor, IPTG must first diffuse into the cell:
</P><PRE>IPTG_outside(a) -&gt; IPTG(p)
</PRE><P>Finally the repressor is constantly being produced and degraded. The lac repressor is a dimer-dimer. Thus the mRNA codes for a monomer, which dimerizes, and dimerizes again:
</P><PRE>0 -&gt; Rep_mRNA()
Rep_mRNA() -&gt; Rep_mono()
Rep_mono() + Rep_mono()  &lt;-&gt; Rep_dimer()
Rep_dimer() + Rep_dimer() &lt;-&gt; Rep(dna,a,a)
Rep_mRNA()  -&gt;0 
</PRE><P>Parameters can be found in the .txt file, and are based on work on the lac Operon by <A href="http://www.ncbi.nlm.nih.gov/pubmed/19186128" class="external text" rel="nofollow">Stamatakis and Mantzaris</A>.
The network diagram for this model is shown below. 
</P><P><A href="https://static.igem.org/mediawiki/2013/b/bb/Hbsu_model.txt" target="_blank">The BioNetGen file</A> (in .txt format, if you'd like to try it out just change the extension to .bngl)</P><H3><SPAN class="mw-headline" id="Methods">Methods</SPAN></H3><P>The model was simulated stochastically 25 times at 6 different initial concentrations of IPTG: 0mM, 0.05mM, 0.1mM 0.2mM, 0.4mM and 0.8mM, as these were the concentrations we used experimentally. The results for at each concentration were aggregated and averaged using R and graphed against the measured intensity of the HBsu-sfGFP/RFP fluorescence at each concentration. </P><P>This fluorescence was calculated using <A href="http://rsbweb.nih.gov/ij/" class="external text" rel="nofollow">imageJ</A>. Initially we used the line selection tool to drew across the cell and measure its mean fluorescence, with mean background fluorescence then measured as a control. To get a Corrected Total Cell Fluorescence (CTCF) for each cell, the mean background fluorescence was deducted from the mean fluorescence of the cell. This was done for 20 cells at each IPTG concentration. 
This showed that our model predicted the time taken for IPTG to bind to the repressor and allow expression of the BioBrick with reasonable accuracy. Fluorescence was measured experimentally at 0,10,30,60,120 and 240 minutes. Expression of the BioBrick began roughly when expected, although the model was less reliable at lower concentrations of IPTG, often predicting no expression within 240 minutes at 0.05mM IPTG. The stochasticity also increased at lower IPTG concentrations (until no expression was predicted), though this is to be expected. 
</P><H3><SPAN class="mw-headline" id="Results">Results</SPAN></H3><H4><SPAN class="mw-headline" id="0mM_IPTG">0mM IPTG</SPAN></H4><P>Even with no IPTG fluorescence was recorded. This could be due to small amounts of transcription despite the presence of the repressor- which does happen. However negative fluorescence results are recorded. This shows that our method of measuring fluorescence wasn't perfectly accurate. This is an area for further improvement.
</P><H4><SPAN class="mw-headline" id="0.05mM_IPTG">0.05mM IPTG</SPAN></H4><P>sfGFP fluorescence (and thus protein production) began at some point between the recorded 100 minutes and 240 minutes. The model predicts gene expression began around 205 minutes after exposure to IPTG, which lies between these values. However RFP fluorescence is negligible. The model predicts fewer than 200 molecules of the protein produced, so this isn’t too suprising as the RFP signal is weaker than the sfGFP  
</P><H4><SPAN class="mw-headline" id="0.1mM_IPTG">0.1mM IPTG</SPAN></H4><P>sfGFP fluorescence began between 60-120 minutes, and the model predicted gene expression after around 100 minutes. From the fluorescence data it might be expected for gene expression to occur a little earlier, but is a decent approximation. Predicted molecule number correlates well with RFP fluorescence at this concentration.  
</P><H4><SPAN class="mw-headline" id="0.2mM_IPTG">0.2mM IPTG</SPAN></H4><P>At 0.2mM the predicted time after exposure to IPTG until expression correlates with both RFP and GFP fluorescence. 
</P><H4><SPAN class="mw-headline" id="0.4mM_IPTG">0.4mM IPTG</SPAN></H4><P>The model accurately predicts the time of expression. Of note however is the increase in fluorescence dropping off after 120 minutes, whilst the model predicts molecule number continuing to increase. This could be because:
</P><UL><LI> The model should have the HBsu-xFP degrade more quickly. The half-life of HBsu is 18-24 hours, however our new protein could be lower.
</LI><LI> The cell may be controlling the gene expression via some form of negative feedback pathway. This may be as
</LI></UL><P>fluorescence proteins are large and  
</P><UL><LI> The protein could be toxic at higher concentrations
</LI><LI> The relationship between the protein and fluorescence is not linear
</LI></UL><P>We did not have the time or resources to test these hypotheses, but ideally tests would be done and if necessary the model tweaked.    
</P><H4><SPAN class="mw-headline" id="0.8mM_IPTG">0.8mM IPTG</SPAN></H4><P>Once again the point of time at which expression began was well predicted by the model. However after 120 minutes fluorescence actually decreased! This could just be due to the intrinsic variabilty of the fluorescent measurements, or could be due to one of the possibilities considered above
</P><P>Fluorescence was then plotted against molecule number for each non-0 molecule number result. This showed that there is a positive correlation between molecule number and log(fluorescence). The first graph below is for HBsu-GFP, and underneath that HBsu-RFP. 
</P><H3><SPAN class="mw-headline" id="References">References</SPAN></H3><P><A href="http://www.ncbi.nlm.nih.gov/pubmed/19186128" class="external text" rel="nofollow">Stamatakis M., Mantzaris N.V., 2009. Comparison of deterministic and stochastic models of the lac operon genetic network. Biophysical Journal, Nov; 97(9):2651</A></P><P><A href="http://www.ncbi.nlm.nih.gov/pubmed/10224127" class="external text" rel="nofollow">Kouji, N., Shou-ichi, Y., Takao, Y. &amp; Kunio , Y., 1999. Bacillus subtilis Histone-like Protein, HBsu, Is an Integral Component of a SRP-like Particle That Can Bind the Alu Domain of Small Cytoplasmic RNA. The Journal of Biological Chemistry , Volume 274, pp. 13569-13576.</A></P><P><A href="http://www.ncbi.nlm.nih.gov/pubmed/1902464" class="external text" rel="nofollow">Micka, B., Groch, N., Heinemann, U. &amp; Marahiel , M., 1991. Molecular cloning, nucleotide sequence, and characterization of the Bacillus subtilis gene encoding the DNA binding protein HBsu. Journal of Bacteriology, May;173(10), pp. 3191 -3198.</A></P><DIV class="printfooter">
Retrieved from &quot;<A href="http://2013.igem.org/Team:Newcastle/Modelling/HBsu_Fusion_Protein">http://2013.igem.org/Team:Newcastle/Modelling/HBsu_Fusion_Protein</A>&quot;</DIV></DIV></DIV><DIV id="footer-box" class="noprint"><DIV id="footer"><UL id="f-list"><LI id="t-recentchanges"><A href="/Special:RecentChanges" title="Recent changes">Recent changes</A></LI><LI id="t-whatlinkshere"><A href="/Special:WhatLinksHere/Team:Newcastle/Modelling/HBsu_Fusion_Protein" title="List of all wiki pages that link here [j]" accesskey="j">What links here</A></LI><LI id="t-recentchangeslinked"><A href="/Special:RecentChangesLinked/Team:Newcastle/Modelling/HBsu_Fusion_Protein" title="Recent changes in pages linked from this page [k]" accesskey="k">Related changes</A></LI><LI id="t-specialpages"><A href="/Special:SpecialPages" title="List of all special pages [q]" accesskey="q">Special pages</A></LI><LI><A href="/Special:Preferences">My preferences</A></LI></UL></DIV><DIV id="footer"><UL id="f-list"><LI id="t-print"><A href="/wiki/index.php?title=Team:Newcastle/Modelling/HBsu_Fusion_Protein&amp;printable=yes" title="Printable version of this page [p]" accesskey="p">Printable version</A></LI><LI id="t-permalink"><A href="/wiki/index.php?title=Team:Newcastle/Modelling/HBsu_Fusion_Protein&amp;oldid=309764" title="Permanent link to this revision of the page">Permanent link</A></LI><LI id="privacy"><A href="/2013.igem.org:Privacy_policy" title="2013.igem.org:Privacy policy">Privacy policy</A></LI><LI id="disclaimer"><A href="/2013.igem.org:General_disclaimer" title="2013.igem.org:General disclaimer">Disclaimers</A></LI></UL></DIV></DIV></DIV></DIV></BODY></HTML>