<HTML xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en" dir="ltr">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki  ltr ns-0 ns-subject page-Team_Newcastle_Project_plants"><DIV id="globalWrapper"><DIV id="top-section"><DIV id="p-logo"><A href="/Main_Page" title="Main Page">&quot;
	    </A></DIV><DIV id="menubar" class="left-menu noprint"><UL><LI class="selected"><A href="/Team:Newcastle/Project/plants">Page               </A></LI><LI class="new"><A href="/wiki/index.php?title=Talk:Team:Newcastle/Project/plants&amp;action=edit&amp;redlink=1">Discussion               </A></LI><LI><A href="/wiki/index.php?title=Team:Newcastle/Project/plants&amp;action=edit">View source               </A></LI><LI><A href="/wiki/index.php?title=Team:Newcastle/Project/plants&amp;action=history">History               </A></LI><LI style="color:white;cursor:default">teams</LI></UL></DIV><DIV class="right-menu noprint" id="menubar"><UL><LI id="pt-login"><A href="/wiki/index.php?title=Special:UserLogin&amp;returnto=Team:Newcastle/Project/plants" title="You are encouraged to log in; however, it is not mandatory [o]" accesskey="o">Log in</A></LI></UL></DIV><DIV id="search-controls" class="noprint"><FORM action="/Special:Search" id="searchform"> </FORM></DIV></DIV><DIV id="content"><H1 class="firstHeading">Team:Newcastle/Project/plants</H1><DIV id="bodyContent"><H3 id="siteSub" class="noprint">From 2013.igem.org</H3><DIV><DIV id="pop_up"><DIV id="pop_up_underlay"> </DIV><DIV id="pop_up_display"><DIV id="pop_up_close">X</DIV><DIV id="pop_up_content"> </DIV></DIV></DIV><DIV class="menu" class="grid_12"><UL class="top_menu" style="margin-left:10px"><LI><A href="https://2013.igem.org/Team:Newcastle">Home</A></LI><LI><A href="#" onclick="return false;">Team</A></LI><LI><A href="https://igem.org/Team.cgi?year=2013&amp;team_name=Newcastle" target="_blank">Official Profile</A></LI><LI><A href="/Team:Newcastle/Team">Individual Profiles</A></LI></UL><LI><A href="#" onclick="return false;">Project</A></LI><LI><A href="https://2013.igem.org/Team:Newcastle/Project">Overview</A></LI><LI><A href="https://2013.igem.org/Team:Newcastle/Project/L_forms">L-forms</A></LI><LI><A href="https://2013.igem.org/Team:Newcastle/Project/shuffling_endosymbiosis">Genome Shuffling</A></LI><LI><A href="https://2013.igem.org/Team:Newcastle/Project/plants">L-forms in Plants</A></LI><LI><A href="https://2013.igem.org/Team:Newcastle/Project/shape_shifting">Shape Shifting</A></LI><LI><A href="#" onclick="return false;">Parts</A></LI><LI><A href="https://2013.igem.org/Team:Newcastle/Parts/l_form_switch">L-form Switch</A></LI><LI><A href="https://2013.igem.org/Team:Newcastle/Parts/HBsu-fp">HBsu-xFP</A></LI><LI><A href="https://2013.igem.org/Team:Newcastle/Parts/e-coli_swich">E-coli Switch</A></LI><LI><A href="#" onclick="return false;">Models</A></LI><LI><A href="https://2013.igem.org/Team:Newcastle/Modelling/Introduction">Introduction</A></LI><LI><A href="https://2013.igem.org/Team:Newcastle/Modelling/BioNetGen">BioNetGen</A></LI><LI><A href="https://2013.igem.org/Team:Newcastle/Modelling/CellShapeModel">Cell Shape Model</A></LI><LI><A href="https://2013.igem.org/Team:Newcastle/Modelling/Cell_Fusion">Cell Fusion</A></LI><LI><A href="https://2013.igem.org/Team:Newcastle/Modelling/HBsu_Fusion_Protein">HBsu-xFP</A></LI><LI><A href="https://2013.igem.org/Team:Newcastle/Modelling/L-form_Switch">L-form Switch</A></LI><LI><A href="#" onclick="return false;">Notebook</A></LI><LI><A href="https://2013.igem.org/Team:Newcastle/Notebook/achievements">Achievements</A></LI><LI><A href="https://2013.igem.org/Team:Newcastle/Notebook/timeline">Timeline</A></LI><LI><A href="https://2013.igem.org/Team:Newcastle/Notebook/calendar">Calendar</A></LI><LI><A href="https://2013.igem.org/Team:Newcastle/Notebook/protocols">Protocols</A></LI><LI><A href="https://2013.igem.org/Team:Newcastle/Notebook/agenda">Meetings</A></LI><LI><A href="#" onclick="return false;">Implications</A></LI><LI><A href="https://2013.igem.org/Team:Newcastle/Outreach/Overview">Overview</A></LI><LI><A href="https://2013.igem.org/Team:Newcastle/Outreach/BioGame">BioGame</A></LI><LI><A href="https://2013.igem.org/Team:Newcastle/Outreach/uk_meet">YSB 1.0</A></LI><LI><A href="https://2013.igem.org/Team:Newcastle/Outreach/Workshop">Modelling Workshop</A></LI><LI><A href="https://2013.igem.org/Team:Newcastle/Outreach/Architecture">Architecture Workshop</A></LI><LI><A href="https://2013.igem.org/Team:Newcastle/HP/Safety">Safety</A></LI><LI><A href="https://2013.igem.org/Team:Newcastle/HP/Synthetic_Bio">Synthetic Biology</A></LI><LI><A href="https://2013.igem.org/Team:Newcastle/HP/Ethics">Ethics</A></LI><LI><A href="https://2013.igem.org/Team:Newcastle/HP/Law">Law</A></LI><LI><A href="#" onclick="return false;">Attributions</A></LI><LI><A href="https://2013.igem.org/Team:Newcastle/Collaborations">Collaborations</A></LI><LI><A href="https://2013.igem.org/Team:Newcastle/Attributions">Acknowledgments</A></LI><LI><A href="https://2013.igem.org/Team:Newcastle/Sponsors">Sponsors</A></LI><LI><A href="https://2013.igem.org/Team:Newcastle/Architecture">Architecture</A></LI><LI><A href="https://2013.igem.org/Main_Page">iGEM</A></LI></DIV><TABLE id="toc" class="toc"><TBODY><TR><TD><DIV id="toctitle"><H2>Contents</H2></DIV><UL><LI class="toclevel-1 tocsection-1"><A href="#L-forms_in_Plants"><SPAN class="tocnumber">1</SPAN><SPAN class="toctext">L-forms in Plants</SPAN></A></LI><LI class="toclevel-2 tocsection-2"><A href="#Purpose"><SPAN class="tocnumber">1.1</SPAN><SPAN class="toctext">Purpose</SPAN></A></LI><LI class="toclevel-2 tocsection-3"><A href="#Our_Objectives"><SPAN class="tocnumber">1.2</SPAN><SPAN class="toctext">Our Objectives</SPAN></A></LI><LI class="toclevel-2 tocsection-4"><A href="#Methods"><SPAN class="tocnumber">1.3</SPAN><SPAN class="toctext">Methods</SPAN></A></LI><LI class="toclevel-3 tocsection-5"><A href="#Inoculating_L-forms_into_Chinese_Cabbage"><SPAN class="tocnumber">1.3.1</SPAN><SPAN class="toctext">Inoculating L-forms into Chinese Cabbage</SPAN></A></LI></UL><LI class="toclevel-2 tocsection-6"><A href="#Alternative_Methods_Considered"><SPAN class="tocnumber">1.4</SPAN><SPAN class="toctext">Alternative Methods Considered</SPAN></A></LI><LI class="toclevel-3 tocsection-7"><A href="#.CE.B2-glucuronidase_assay"><SPAN class="tocnumber">1.4.1</SPAN><SPAN class="toctext">β-glucuronidase assay</SPAN></A></LI><LI class="toclevel-3 tocsection-8"><A href="#Enzyme-linked_immunosorbent_assay_.28ELISA.29"><SPAN class="tocnumber">1.4.2</SPAN><SPAN class="toctext">Enzyme-linked immunosorbent assay (ELISA)</SPAN></A></LI><LI class="toclevel-3 tocsection-9"><A href="#DAPI_Staining"><SPAN class="tocnumber">1.4.3</SPAN><SPAN class="toctext">DAPI Staining</SPAN></A></LI><LI class="toclevel-3 tocsection-10"><A href="#PCR"><SPAN class="tocnumber">1.4.4</SPAN><SPAN class="toctext">PCR</SPAN></A></LI><LI class="toclevel-3 tocsection-11"><A href="#Alternative_Mounting"><SPAN class="tocnumber">1.4.5</SPAN><SPAN class="toctext">Alternative Mounting</SPAN></A></LI><LI class="toclevel-3 tocsection-12"><A href="#Propium_Iodide"><SPAN class="tocnumber">1.4.6</SPAN><SPAN class="toctext">Propium Iodide</SPAN></A></LI><LI class="toclevel-3 tocsection-13"><A href="#FM_1-43"><SPAN class="tocnumber">1.4.7</SPAN><SPAN class="toctext">FM 1-43</SPAN></A></LI><LI class="toclevel-2 tocsection-14"><A href="#Results"><SPAN class="tocnumber">1.5</SPAN><SPAN class="toctext">Results</SPAN></A></LI><LI class="toclevel-2 tocsection-15"><A href="#References"><SPAN class="tocnumber">1.6</SPAN><SPAN class="toctext">References</SPAN></A></LI></TD></TR></TBODY></TABLE><H1><SPAN class="mw-headline" id="L-forms_in_Plants">L-forms in Plants</SPAN></H1><P>In  nature L-forms and plants can exist in a symbiotic relationship as plants provide an osmotically suitable environment. L-forms can confer benefits to their host including <A href="http://www.ncbi.nlm.nih.gov/pubmed/11849491" class="external text" rel="nofollow">inhibition of fungal pathogenesis</A>. This lead us to consider whether it would be possible to create an artificial symbiotic relationship between plants and L-forms. We washed <I>Brassica pekinensis</I> (Chinese cabbage) seedlings in a solution of green fluorescent protein (GFP) labelled L-forms, allowing the plants to take up the bacteria. We then viewed our L-forms inside the plant using light and confocal microscopy.
</P><P>In the future, L-forms could be engineered to supply nutrients or other useful compounds, such as pesticides, to plants. This could potentially increase crop yield and create more nutritious plants.  
</P><P>The use of L-forms decreases the chance of genetic pollution as they lyse if they escape from their host plant into the environment. <A href="https://2013.igem.org/Team:Newcastle/Project/L_forms#Characterisation_of_osmotic_instability" class="external text" rel="nofollow">L-forms are osmotically sensitive</A>. This makes them much less likely to survive in the environment than cell walled bacteria.
</P><H2><SPAN class="mw-headline" id="Purpose">Purpose</SPAN></H2><P>As the world’s population continues growing there is an ever greater need to improve crop yield. There are already worldwide food shortages and things will only get worse. Furthermore climate change will reduce the area of suitable agricultural land and the effectiveness of many commonly used crops, therefore it is necessary to maintain and increase the productivity of agricultural land wherever possible.
</P><P>We propose that L-forms could be modified to produce benefits to plants - producing nutrients or protection against pathogens. 
<A href="http://www.ncbi.nlm.nih.gov/pubmed/11069643" class="external text" rel="nofollow">Research has already shown</A> prevention of fungal pathogenesis in strawberry plants which have been exposed to <I>B. subtilis</I> L-forms. The mechanism by which this works is not yet understood, but could we harness the anti-fungal nature of these micro-organisms? It could be possible to take nitrogen fixing bacteria such as Rhizobia, turn them into L-forms and inoculate them into plants to produce non-leguminous nitrogen fixing crops. These sorts of ideas could have a big impact on agriculture, preventing crop spoilage and increasing crop yield. As the world's population rises so will the demand for food. Producing enough food is not a new problem for humans, yet this novel form of bacteria may be part of the solution.
</P><P>Using L-forms may appear an unnecessary step- why not just modify the plant cells? First, bacteria are significantly easier to manipulate and genetically modify than plant cells, which are very complex. There are also bacterial genes, beneficial to plants, which would not be functional if transformed into a GM plant.  Furthermore, there are ethical boundaries associated with modifying plants. GM plants could well outcompete native species, spreading away from farmland and into the wild, becoming weeds and reducing biodiversity. They could also crossbreed with other strains.
</P><P>Using non-L-form bacteria also has multiple potential pitfalls. Kill switches are never 100% safe as there is always the risk of mutations that turn off these genes, resulting in bacteria able to proliferate freely. L-forms however are incredibly osmotically sensitive, unable to survive outside of plants. Therefore they could be introduced into the environment - if they do leave the plants, they will very quickly pop due to water entering through osmosis, and a lack of a cell wall to counteract the resulting increased internal pressure. We have shown this experimentally using water extracted from soil.
</P><P>Inoculating plants with L-forms is therefore safer than inoculating with regular bacteria and also safer than directly modifying plants.
</P><H2><SPAN class="mw-headline" id="Our_Objectives">Our Objectives</SPAN></H2><P>Our primary objective was to grow L-forms inside Chinese cabbage, to illustrate that it would be possible to create an artificial symbiotic relationship. However in the process of doing this we also produced a <A href="https://2013.igem.org/Team:Newcastle/Notebook/protocols#Inoculating_L-forms_into_Chinese_Cabbage" class="external text" rel="nofollow">protocol for the inoculation of L-forms into Chinese cabbage</A> that can be used by future iGEM teams working with L-forms. We also wanted to <A href="https://2013.igem.org/Team:Newcastle/Project/L_forms#Characterisation_of_Osmotic_Instability" class="external text" rel="nofollow">characterize the osmotic instability of L-forms</A>, as this instability makes L-forms a more attractive proposition to inoculate into crops than cell-walled bacteria. Finally we wished to <A href="https://2013.igem.org/Team:Newcastle/HP/Law#Implications_for_BareCillus" class="external text" rel="nofollow">investigate the laws governing the release of genetically modified organisms</A>.
</P><P><I>Figure 1. Chinese cabbage seedlings inoculated with L-forms growing on a Murishage and Skoog plate.</I></P><H2><SPAN class="mw-headline" id="Methods">Methods</SPAN></H2><P>In order to prove that L-forms can grow in plants we soaked germinating Chinese cabbage seedlings in a solution of GFP-labelled L-forms. We then replanted the seedlings and viewed our L-forms via light microscopy before using confocal microscopy to generate better quality images. Our protocol is modified from <A href="http://www.ncbi.nlm.nih.gov/pubmed/12859751" class="external text" rel="nofollow">Tsomlexoglou <I>et al.</I> (2003)</A>: -
</P><H4><SPAN class="mw-headline" id="Inoculating_L-forms_into_Chinese_Cabbage">Inoculating L-forms into Chinese Cabbage</SPAN></H4><OL><LI>Rinse seeds for 2 minutes in 70% (v/v) ethanol.
</LI><LI>Soak seeds for 10 minutes in 20% (v/v) Milton’s sterilizing fluid.
</LI><LI>Wash seeds thoroughly five times in sterile distilled water and leave in final wash for fifteen minutes.
</LI><LI>Place seeds in (9cm) Petri dishes (20-25 seeds per dish) containing <A href="http://www.sigmaaldrich.com/content/dam/sigma-aldrich/docs/Sigma/Product_Information_Sheet/1/m5519pis.pdf" class="external text" rel="nofollow">Murashige and Skoog (M and S) basal medium</A>, solidified with 0.8% (w/v) agar no.1 (Oxoid, UK) and incubate at 25°C in the dark until radicals just appear (this should take 21-24hr for Chinese cabbage).
</LI><LI>Use spectrophotometry to produce a L-form suspension containing approximately 10<SUP>7</SUP> CFU ml-1 (OD600: approx. 0.7).
</LI><LI>Select seeds with radicals 1-2mm in length and soak in the bacterial suspension (20 seeds per 10ml) for 3 hours at 25°C, gently shaking the seeds by hand every 30 minutes. Treat some seedlings with 5% (w/v) mannitol instead of  L-forms to act as a control.
</LI><LI>Wash seeds ten times in distilled water.
</LI><LI>Replant plants on M and S agar plates. 
</LI><LI>Incubate seeds in sunlight for between 24 hours and 7 days at 25°C.
</LI><LI>The plants are ready to be viewed using microscopy.
</LI></OL><P>The washing of seeds in ethanol and Milton’s sterilizing fluid is carried out to kill any bacteria residing on the surface of the seed. M and S medium is a growth media designed for the cultivation of plants. Washing the seeds in distilled water after soaking them in bacterial solution lyses any L-forms not inside the plant. We carried out our microscopy 4 days after the seeds were soaked in L-forms, to co-inside with when <A href="http://www.ncbi.nlm.nih.gov/pubmed/12859751" class="external text" rel="nofollow">Tsomlexoglou <I>et al.</I> (2003)</A> carried out PCR to prove the presence of L-forms in their Chinese cabbage.
</P><H2><SPAN class="mw-headline" id="Alternative_Methods_Considered">Alternative Methods Considered</SPAN></H2><P>Before deciding to use GFP-labelled L-forms, we considered a few alternative ways to detect L-forms in plants:
</P><H4><SPAN class="mw-headline" id=".CE.B2-glucuronidase_assay">β-glucuronidase assay</SPAN></H4><P>In order to prove that L-forms can grow in plants, we considered transforming <I>B. subtillis</I> strain LR2 with the <I>gusA</I> reporter gene coding for β-glucuronidase. We would have removed <I>gusA</I> BioBrick from its plasmid backbone and attach it to another plasmid containing <I>aprE</I> for homologous recombination and kanamycin resistance for selection. We would then have transformed <I>B. subtillis</I> with the new transformation vector. The bacteria that have taken up the plasmid can would have been selected for via kanamycin resistance. A transformation vector would have been used rather than a normal plasmid so that <I>gusA</I> would integrate through homologous recombination into the bacteria’s chromosome, securing the expressing of the <I>gusA</I>.
</P><P>The β-glucuronidase produced by the L-forms in plants would cleave the colourless substrate 5-bromo-4-chloro-3-indolyl-β-D-glucuronide (X-Gluc) to produce a blue product allowing the L-form distribution in plant tissue to be observed. We considered this method after reading a paper by <A href="http://www.ncbi.nlm.nih.gov/pubmed/12859751" class="external text" rel="nofollow">Tsomlexoglou <I>et al.</I> (2003)</A>.
</P><H4><SPAN class="mw-headline" id="Enzyme-linked_immunosorbent_assay_.28ELISA.29">Enzyme-linked immunosorbent assay (ELISA)</SPAN></H4><P>We considered producing an ELISA to detect L-forms. We could have generated antibodies against an L-form specific antigen. Alternatively we considered introducing a novel gene into L-forms to produce a novel antigen to generate antibodies against. However this would have involved using animals to generate poly-clonal antibodies. We decided this raised ethical issues and was too time consuming to complete in our ten week placement. We considered this method after reading a paper by <A href="http://www.ncbi.nlm.nih.gov/pubmed/11069643" class="external text" rel="nofollow">Ferguson <I>et al.</I> (2003)</A>.
</P><H4><SPAN class="mw-headline" id="DAPI_Staining">DAPI Staining</SPAN></H4><P><A href="http://probes.invitrogen.com/media/pis/mp01306.pdf" class="external text" rel="nofollow">4',6-diamidino-2-phenylindole (DAPI)</A> can be used to stain both bacterial and plant DNA fluorescent blue. For this reason we considered using DAPI to counter-stain the plant genome and inoculating the seedlings with GFP-labelled L-forms so we can see both the plant nucleus and our L-forms. Alternatively we could have stained GFP-labelled L-forms with DAPI so that when viewing L-forms with confocal microscopy we could check if they fluoresce blue as well as green. This would add further evidence that we had successfully inoculated L-forms into Chinese cabbage.
</P><H4><SPAN class="mw-headline" id="PCR">PCR</SPAN></H4><P><A href="http://www.ncbi.nlm.nih.gov/pubmed/12859751" class="external text" rel="nofollow">Tsomlexoglou <I>et al.</I> (2003)</A> used the polymerase chain reaction (PCR) to detect L-forms containing the <I>gusA</I> gene in four day old Chinese cabbage. Primers specific to the <I>gusA</I> gene were used as this gene was specific to the genetically modified L-forms. We considered using primers specific to our GFP-labelled L-forms to carry out PCR. We chose not to carry out this method due to time constraints and we wanted a more aesthetic detection method.
</P><H4><SPAN class="mw-headline" id="Alternative_Mounting">Alternative Mounting</SPAN></H4><P>Some difficulties faced while imaging the plants by confocal microscopy were:
</P><P>1)There are deep layers of highly refractile cell wall and aqueous cytosol in the plant cell wall
</P><P>2)The plant had auto-fluorescence and light scattering constituents
</P><P>To address these difficulties, two methods were considered:
</P><P>1)To fix and clear the tissue with a high-refractive index mounting medium
</P><P>2)To directly image living tissue using suitably corrected microscope optics
</P><P>This second method was looked into at greater depth as it would be difficult to effectively clear plant wholemounts without causing artifacts or risking the loss of GFP fluorescence. Following the second method, we mounted our seedlings in water under glass cover slips. This method of direct visualization of GFP fluorescence in living tissue solved problems to do with fixation or staining artifacts. The images obtained from this method give good clarity <A href="http://link.springer.com/protocol/10.1385%2F1-59259-722-X%3A241#" class="external text" rel="nofollow">(Haseloff <I>et al.</I>, 1999)</A>.
</P><P>Besides methods of imaging, alternative staining methods were also considered to enhance the images of L-forms in plants. As auto-fluorescent chloroplasts are normally present in the upper parts of the plant, certain red fluorescent dyes can provide useful counter fluorescence for GFP. An example of this would be propium iodide and FM 1-43. Although, these staining methods were not used due to the time constraint of this project, if given more time, images that illustrated in detail the localization of the L-forms may have been obtained.
</P><H4><SPAN class="mw-headline" id="Propium_Iodide">Propium Iodide</SPAN></H4><P>Propium iodide is a red fluorescent stain that can be detected using a filter set suitable for Texas Red fluorescence. It can be applied to live seedlings in water to specifically label root cell walls, forming an outline of the cell. Staining the plant cell wall will provide an extremely good contrast, revealing the localization of the bacteria <A href="http://link.springer.com/protocol/10.1385%2F1-59259-722-X%3A241#" class="external text" rel="nofollow">(Haseloff <I>et al.</I>, 1999)</A>.
</P><H4><SPAN class="mw-headline" id="FM_1-43">FM 1-43</SPAN></H4><P>FM1-43 is an orange stain that stains the plasma membrane in roots and shoot tissues. The imaging that has been conducted was mainly of the base of the shoot, making this dye a good stain to provide the contrast between the plasma membrane and the fluorescence emitting from the GFP-labelled L-forms <A href="http://link.springer.com/protocol/10.1385%2F1-59259-722-X%3A241#" class="external text" rel="nofollow">(Haseloff <I>et al.</I>, 1999)</A>.
</P><H2><SPAN class="mw-headline" id="Results">Results</SPAN></H2><P>We checked to see if the bacteria had found their way into the seedlings four days after inoculating our Chinese cabbage with L-forms. We checked our L-form-washed plants and our mannitol-washed (negative control) plants using confocal microscopy to see if there was any green fluorescence of the corresponding size and shape to be L-forms. We were able to generate images of our L-form-washed seedlings showing green fluorescent blobs ranging in size from approximately 2 to 6 microns as seen in Figure 2.  The fluorescent shapes are of the correct size and appearance to be L-forms. In our negative control we could find no such green fluorescence, only images showing plant cells as shown in Figure 3. These results show that the Chinese cabbage that we treated contained our GFP-labelled L-forms.
</P><P><I>Figure 2. Confocal microscopy of L-form washed plants. The image on the left shows a brightfield image of plant cells, with an image of the green fluorescence overlaid on top. The green blobs ranging in size from approximately 2 to 6 microns are L-forms. The image on the right shows just the brightfield image of plant cells.</I></P><P><I>Figure 3. Confocal microscopy of negative control plants. The left hand image shows a brightfield image of plant cells, with an image of the green fluorescence overlaid on top.  There are no L-forms present in the image, the green fluorescence is sparse and not of the correct appearance to be L-forms. The image on the right shows just the brightfield image of plant cells.</I></P><P>These results show that we were successful in producing plants containing genetically engineered L-forms. In the future other synthetic biologists could adopt this delivery method to inoculate Chinese cabbage with L-forms engineered to provide substances to the plant including nitrates, anti-fungals or plant hormones.
</P><H2><SPAN class="mw-headline" id="References">References</SPAN></H2><P><A href="http://www.ncbi.nlm.nih.gov/pubmed/1905284" class="external text" rel="nofollow">Allan, E.J. (1991) 'Induction and cultivation of a stable L-form of Bacillus subtilis', J Appl Bacteriol, 70(4), pp. 339-43.</A></P><P><A href="http://www.ncbi.nlm.nih.gov/pubmed/8486565" class="external text" rel="nofollow">Allan, E.J., Amijee, F., Tyson, R.H., Strang, J.A., Innes, C.M. and Paton, A.M. (1993) 'Growth and physiological characteristics of Bacillus subtilis L-forms', J Appl Bacteriol, 74(5), pp. 588-94.</A></P><P><A href="http://onlinelibrary.wiley.com/doi/10.1111/j.1365-2672.1984.tb01375.x/abstract" class="external text" rel="nofollow">Aloysius, S. and Paton, A.M. (1984) 'Artificially Induced Symbioic Assoiations of L-form Bacteria and Plants', Journal of Applied Bacteriology, 56(3), pp. 465-477.</A></P><P><A href="http://www.ncbi.nlm.nih.gov/pubmed/11069643" class="external text" rel="nofollow">Ferguson, C.M.J., Booth, N.A. and Allan, E.J. (2000) An ELISA for the detection of Bacillus subtilis L-form bacteria confirms their symbiosis in strawberry. <I>Letters in Applied Microbiology</I>. 31(5), p. 390-394.</A></P><P><A href="http://link.springer.com/protocol/10.1385%2F1-59259-722-X%3A241#" class="external text" rel="nofollow">Haseloff, J., Dormand, E.-L. and Brand, A. (1999) 'Live Imaging with Green Fluorescent Protein', in Paddock, S. (ed.) Confocal Microscopy Methods and Protocols. Humana Press,  pp. 241-259.</A></P><P><A href="http://probes.invitrogen.com/media/pis/mp01306.pdf" class="external text" rel="nofollow">INVITROGEN. (2006) <I>DAPI Nucleic Acid Stain</I>.  (online) Available from: http://probes.invitrogen.com/media/pis/mp01306.pdf (Accessed 28 August 2013)</A></P><P><A href="http://onlinelibrary.wiley.com/doi/10.1111/j.1365-2672.1987.tb04856.x/abstract" class="external text" rel="nofollow">Paton, A.M. (1987) 'L-forms: evolution or revolution?', J Appl Bacteriol, 63(5), pp. 365-71.</A></P><P><A href="http://onlinelibrary.wiley.com/doi/10.1111/j.1365-2672.1991.tb04663.x/abstract" class="external text" rel="nofollow">Paton, A.M. and Innes, C.M.J. (1991) 'Methods for the establishment of intracellular associations of L-forms with higher plants', Journal of Applied Bacteriology, 71(1), pp. 59-64.</A></P><P><A href="http://www.ncbi.nlm.nih.gov/pubmed/12859751" class="external text" rel="nofollow">Tsomlexoglou, E., Daulagala, P.W.H.K.P., Gooday, G.W., Glover, L.A., Seddon, B. and Allan, E.J. (2003) Molecular detection and β-glucuronidase expression of gus-marked <I>Bacillus subtilis</I> L-form bacteria in developing Chinese cabbage seedlings. <I>Journal of Applied Microbiology</I>. 95(2), p. 218-224.</A></P><P><A href="http://www.ncbi.nlm.nih.gov/pubmed/11849491" class="external text" rel="nofollow">Walker, R. , Ferguson, C.M.J. , Booth, N.A. and Allan, E.J. (2002) The symbiosis of Bacillus subtilis L-forms with Chinese cabbage seedlings inhibits conidial germination of Botrytis cinerea. <I>Letters in Applied Microbiology</I>. (34) p.42–45.</A></P><P><A href="http://onlinelibrary.wiley.com/doi/10.1111/j.1472-765X.1996.tb01157.x/abstract" class="external text" rel="nofollow">Waterhouse, R.N., Buhariwalla, H., Bourn, D., Rattray, E.J. and Glover, L.A. (1996) 'CCD detection of lux-marked Pseudomonas syringae pv. phaseolicola L-forms associated with Chinese cabbage and the resulting disease protection against Xanthomonas campestris', Letters in Applied Microbiology, 22(4), pp. 262-266.</A></P><DIV class="printfooter">
Retrieved from &quot;<A href="http://2013.igem.org/Team:Newcastle/Project/plants">http://2013.igem.org/Team:Newcastle/Project/plants</A>&quot;</DIV></DIV></DIV><DIV id="footer-box" class="noprint"><DIV id="footer"><UL id="f-list"><LI id="t-recentchanges"><A href="/Special:RecentChanges" title="Recent changes">Recent changes</A></LI><LI id="t-whatlinkshere"><A href="/Special:WhatLinksHere/Team:Newcastle/Project/plants" title="List of all wiki pages that link here [j]" accesskey="j">What links here</A></LI><LI id="t-recentchangeslinked"><A href="/Special:RecentChangesLinked/Team:Newcastle/Project/plants" title="Recent changes in pages linked from this page [k]" accesskey="k">Related changes</A></LI><LI id="t-specialpages"><A href="/Special:SpecialPages" title="List of all special pages [q]" accesskey="q">Special pages</A></LI><LI><A href="/Special:Preferences">My preferences</A></LI></UL></DIV><DIV id="footer"><UL id="f-list"><LI id="t-print"><A href="/wiki/index.php?title=Team:Newcastle/Project/plants&amp;printable=yes" title="Printable version of this page [p]" accesskey="p">Printable version</A></LI><LI id="t-permalink"><A href="/wiki/index.php?title=Team:Newcastle/Project/plants&amp;oldid=349813" title="Permanent link to this revision of the page">Permanent link</A></LI><LI id="privacy"><A href="/2013.igem.org:Privacy_policy" title="2013.igem.org:Privacy policy">Privacy policy</A></LI><LI id="disclaimer"><A href="/2013.igem.org:General_disclaimer" title="2013.igem.org:General disclaimer">Disclaimers</A></LI></UL></DIV></DIV></DIV></DIV></BODY></HTML>