<HTML xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en" dir="ltr">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki  ltr ns-0 ns-subject page-Team_Newcastle_HP_Ethics"><DIV id="globalWrapper"><DIV id="top-section"><DIV id="p-logo"><A href="/Main_Page" title="Main Page">&quot;
	    </A></DIV><DIV id="menubar" class="left-menu noprint"><UL><LI class="selected"><A href="/Team:Newcastle/HP/Ethics">Page               </A></LI><LI class="new"><A href="/wiki/index.php?title=Talk:Team:Newcastle/HP/Ethics&amp;action=edit&amp;redlink=1">Discussion               </A></LI><LI><A href="/wiki/index.php?title=Team:Newcastle/HP/Ethics&amp;action=edit">View source               </A></LI><LI><A href="/wiki/index.php?title=Team:Newcastle/HP/Ethics&amp;action=history">History               </A></LI><LI style="color:white;cursor:default">teams</LI></UL></DIV><DIV class="right-menu noprint" id="menubar"><UL><LI id="pt-login"><A href="/wiki/index.php?title=Special:UserLogin&amp;returnto=Team:Newcastle/HP/Ethics" title="You are encouraged to log in; however, it is not mandatory [o]" accesskey="o">Log in</A></LI></UL></DIV><DIV id="search-controls" class="noprint"><FORM action="/Special:Search" id="searchform"> </FORM></DIV></DIV><DIV id="content"><H1 class="firstHeading">Team:Newcastle/HP/Ethics</H1><DIV id="bodyContent"><H3 id="siteSub" class="noprint">From 2013.igem.org</H3><DIV><DIV id="pop_up"><DIV id="pop_up_underlay"> </DIV><DIV id="pop_up_display"><DIV id="pop_up_close">X</DIV><DIV id="pop_up_content"> </DIV></DIV></DIV><DIV class="menu" class="grid_12"><UL class="top_menu" style="margin-left:10px"><LI><A href="https://2013.igem.org/Team:Newcastle">Home</A></LI><LI><A href="#" onclick="return false;">Team</A></LI><LI><A href="https://igem.org/Team.cgi?year=2013&amp;team_name=Newcastle" target="_blank">Official Profile</A></LI><LI><A href="/Team:Newcastle/Team">Individual Profiles</A></LI></UL><LI><A href="#" onclick="return false;">Project</A></LI><LI><A href="https://2013.igem.org/Team:Newcastle/Project">Overview</A></LI><LI><A href="https://2013.igem.org/Team:Newcastle/Project/L_forms">L-forms</A></LI><LI><A href="https://2013.igem.org/Team:Newcastle/Project/shuffling_endosymbiosis">Genome Shuffling</A></LI><LI><A href="https://2013.igem.org/Team:Newcastle/Project/plants">L-forms in Plants</A></LI><LI><A href="https://2013.igem.org/Team:Newcastle/Project/shape_shifting">Shape Shifting</A></LI><LI><A href="#" onclick="return false;">Parts</A></LI><LI><A href="https://2013.igem.org/Team:Newcastle/Parts/l_form_switch">L-form Switch</A></LI><LI><A href="https://2013.igem.org/Team:Newcastle/Parts/HBsu-fp">HBsu-xFP</A></LI><LI><A href="https://2013.igem.org/Team:Newcastle/Parts/e-coli_swich">E-coli Switch</A></LI><LI><A href="#" onclick="return false;">Models</A></LI><LI><A href="https://2013.igem.org/Team:Newcastle/Modelling/Introduction">Introduction</A></LI><LI><A href="https://2013.igem.org/Team:Newcastle/Modelling/BioNetGen">BioNetGen</A></LI><LI><A href="https://2013.igem.org/Team:Newcastle/Modelling/CellShapeModel">Cell Shape Model</A></LI><LI><A href="https://2013.igem.org/Team:Newcastle/Modelling/Cell_Fusion">Cell Fusion</A></LI><LI><A href="https://2013.igem.org/Team:Newcastle/Modelling/HBsu_Fusion_Protein">HBsu-xFP</A></LI><LI><A href="https://2013.igem.org/Team:Newcastle/Modelling/L-form_Switch">L-form Switch</A></LI><LI><A href="#" onclick="return false;">Notebook</A></LI><LI><A href="https://2013.igem.org/Team:Newcastle/Notebook/achievements">Achievements</A></LI><LI><A href="https://2013.igem.org/Team:Newcastle/Notebook/timeline">Timeline</A></LI><LI><A href="https://2013.igem.org/Team:Newcastle/Notebook/calendar">Calendar</A></LI><LI><A href="https://2013.igem.org/Team:Newcastle/Notebook/protocols">Protocols</A></LI><LI><A href="https://2013.igem.org/Team:Newcastle/Notebook/agenda">Meetings</A></LI><LI><A href="#" onclick="return false;">Implications</A></LI><LI><A href="https://2013.igem.org/Team:Newcastle/Outreach/Overview">Overview</A></LI><LI><A href="https://2013.igem.org/Team:Newcastle/Outreach/BioGame">BioGame</A></LI><LI><A href="https://2013.igem.org/Team:Newcastle/Outreach/uk_meet">YSB 1.0</A></LI><LI><A href="https://2013.igem.org/Team:Newcastle/Outreach/Workshop">Modelling Workshop</A></LI><LI><A href="https://2013.igem.org/Team:Newcastle/Outreach/Architecture">Architecture Workshop</A></LI><LI><A href="https://2013.igem.org/Team:Newcastle/HP/Safety">Safety</A></LI><LI><A href="https://2013.igem.org/Team:Newcastle/HP/Synthetic_Bio">Synthetic Biology</A></LI><LI><A href="https://2013.igem.org/Team:Newcastle/HP/Ethics">Ethics</A></LI><LI><A href="https://2013.igem.org/Team:Newcastle/HP/Law">Law</A></LI><LI><A href="#" onclick="return false;">Attributions</A></LI><LI><A href="https://2013.igem.org/Team:Newcastle/Collaborations">Collaborations</A></LI><LI><A href="https://2013.igem.org/Team:Newcastle/Attributions">Acknowledgments</A></LI><LI><A href="https://2013.igem.org/Team:Newcastle/Sponsors">Sponsors</A></LI><LI><A href="https://2013.igem.org/Team:Newcastle/Architecture">Architecture</A></LI><LI><A href="https://2013.igem.org/Main_Page">iGEM</A></LI></DIV><TABLE id="toc" class="toc"><TBODY><TR><TD><DIV id="toctitle"><H2>Contents</H2></DIV><UL><LI class="toclevel-1 tocsection-1"><A href="#Ethics"><SPAN class="tocnumber">1</SPAN><SPAN class="toctext">Ethics</SPAN></A></LI><LI class="toclevel-2 tocsection-2"><A href="#Prevention_of_escape"><SPAN class="tocnumber">1.1</SPAN><SPAN class="toctext">Prevention of escape</SPAN></A></LI><LI class="toclevel-2 tocsection-3"><A href="#Antibiotic_discovery"><SPAN class="tocnumber">1.2</SPAN><SPAN class="toctext">Antibiotic discovery</SPAN></A></LI><LI class="toclevel-2 tocsection-4"><A href="#Environmental_Safety-_Plants"><SPAN class="tocnumber">1.3</SPAN><SPAN class="toctext">Environmental Safety- Plants</SPAN></A></LI><LI class="toclevel-2 tocsection-5"><A href="#Environmental_Safety-_Genome_Shuffling"><SPAN class="tocnumber">1.4</SPAN><SPAN class="toctext">Environmental Safety- Genome Shuffling</SPAN></A></LI><LI class="toclevel-2 tocsection-6"><A href="#Environmental_Safety-_BioBricks"><SPAN class="tocnumber">1.5</SPAN><SPAN class="toctext">Environmental Safety- BioBricks</SPAN></A></LI><LI class="toclevel-3 tocsection-7"><A href="#L-form_switch_BioBrick"><SPAN class="tocnumber">1.5.1</SPAN><SPAN class="toctext">L-form switch BioBrick</SPAN></A></LI><LI class="toclevel-3 tocsection-8"><A href="#HBSU-xFP"><SPAN class="tocnumber">1.5.2</SPAN><SPAN class="toctext">HBSU-xFP</SPAN></A></LI></UL><LI class="toclevel-2 tocsection-9"><A href="#L-forms_and_Evolutionary_Theory"><SPAN class="tocnumber">1.6</SPAN><SPAN class="toctext">L-forms and Evolutionary Theory</SPAN></A></LI><LI class="toclevel-3 tocsection-10"><A href="#Relation_to_Religion"><SPAN class="tocnumber">1.6.1</SPAN><SPAN class="toctext">Relation to Religion</SPAN></A></LI><LI class="toclevel-2 tocsection-11"><A href="#References"><SPAN class="tocnumber">1.7</SPAN><SPAN class="toctext">References</SPAN></A></LI></TD></TR></TBODY></TABLE><H1><SPAN class="mw-headline" id="Ethics">Ethics</SPAN></H1><H2><SPAN class="mw-headline" id="Prevention_of_escape">Prevention of escape</SPAN></H2><P>Though possessing the potential to aid many, synthetic biology comes under close scrutiny due to the use of genetic manipulation – an area the public is generally wary of.  One area of concern is the effect that genetically modified organisms may have on the health of the public.  Public health is fundamentally important to the development of all work in the area of synthetic biology.  Relative to our project, this has been a consideration taken throughout the entirety of our work.
</P><P>The bacteria that are genetically modified with our L-form switch BioBrick have specific properties that prevent their escape.  L-forms are only able to survive in osmotically suitable conditions.  If L-forms escaped from a laboratory setting, they would be unlikely to survive.  We have shown that <I>B. subtilis</I> L-forms <A href="https://2013.igem.org/Team:Newcastle/Project/L_forms#Characterisation_of_Osmotic_Instability" class="external text" rel="nofollow">burst</A> when in contact with water extracted from soil.  The BioBrick enables cells to remain in rod-form; however, this requires the presence of xylose.  Therefore osmotically balanced conditions or xylose are required for the survival of cells genetically modified with our BioBrick.  These requirements prevent the escape of such bacteria, paramount in ensuring public health and safety is maintained.
</P><H2><SPAN class="mw-headline" id="Antibiotic_discovery">Antibiotic discovery</SPAN></H2><P>The research potential for L-forms includes areas that may be of great importance to public health, specifically in the area of antibiotic discovery.  Pathogens are continually evolving resistance to commonly used antibiotics.  This forces an ‘arms-race’ between the pathogens and those developing the antibiotics.  More resistant pathogens require different or improved antibiotics for treatment.  Crucially, a falling number of antibiotics are able to effectively and reliably treat and destroy pathogens.  Antibiotic discovery is a pressing issue to alleviate this concern.  Research using L-forms may be able to help address this.
</P><P>It has been postulated that bacteria are able to adopt L-form state to avoid threats to their cell wall.  As many antibiotics target the cell wall, the adoption of L-form state provides bacteria resistance to many antibiotics.  It is important that mechanisms of antibiotic resistance can be learnt about in the greatest depth of detail possible so that these resistance mechanisms can be circumvented.  Such research may also provide developments through sourcing novel antibiotics that work via non-cell wall-targeted means.
</P><H2><SPAN class="mw-headline" id="Environmental_Safety-_Plants">Environmental Safety- Plants</SPAN></H2><P> Our team is suggesting the use of L-forms instead of normal bacteria or genetically modifying plants to provide benefits to plants. This is because we believe that using L-forms will be safer and have fewer ethical issues, as discussed <A href="https://2013.igem.org/Team:Newcastle/Project/plants#Why_L-forms" class="external text" rel="nofollow">here</A>. However there are still safety and ethical issues to consider. </P><P>Our BioBrick for turning <I>B. subtilis</I> cell walls on and off could mutate, the cells regaining their walls regardless of the presence of xylose. However, it would be reasonably simple to produce a similar mutant, in which part of or the entire <I>murE</I> operon has been removed. As our BioBrick controls the expression of this operon the effect should be the same. It would not be feasible for random mutations to mask this, as functions of multiple genes essential to peptidoglycan synthesis would have to be replaced.</P><P>The <I>murE</I> operon contains the gene <I>Spovd</I>. Although involved in peptidoglycan synthesis, the product is involved in sporulation. Could our BioBrick have a further effect, inhibiting the ability of <I>B. subtilis</I> to sporulate? We are currently considering methods of exploring this. Sporulated bacteria are very difficult to destroy, able to survive for long periods of time in near-extreme conditions. Our L-forms burst in the harsh conditions which induce sporulation.</P><P>Establishing whether our bacteria can survive in the wild is only the first step when considering the safety of releasing plants containing our L-forms into the environment. Could DNA released by burst L-forms be incorporated by other bacteria? If so, what effects could this have? What would happen to animals that ingest plants containing our L-forms?</P><P>Although there are naturally occurring L-forms that appear to survive (and indeed subsist) in the human body, our L-forms do not survive in the pH and osmotic conditions found in the stomach and digestive tract. It should also be stressed that our L-forms should be harmless, as the <I>B. subtilis</I> we have modified is. However there are ways in which cell wall free bacteria could be harmful, if they were able to survive harsh osmotic conditions. It has been speculated that L-forms are difficult for the immune system to detect, having lost many of the antigens found on the cell wall. Furthermore, many popular antibiotics, including penicillin, work by targeting the cell wall. L-forms are therefore resistant to these, which would make them quite difficult to kill if not for their inherent fragility.</P><H2><SPAN class="mw-headline" id="Environmental_Safety-_Genome_Shuffling">Environmental Safety- Genome Shuffling</SPAN></H2><P>The fusion of two cells of the same species causes genetic recombination to occur. This technique has started to be explored in bacteria as a method of directed evolution, as discussed <A href="https://2013.igem.org/Team:Newcastle/Project/shuffling_endosymbiosis#Existing_Work" class="external text" rel="nofollow">here</A>.</P><P>Genome fusion is interesting as it causes recombination over the whole genome, rather than a specific gene, and the recombination occurs randomly across random segments of the genome. This means that we don’t necessarily know, or need to know, the genes responsible for a chosen process to ‘improve’ said process. Furthermore, we won’t know why the improvements have occurred without further, optional, study. Is this important? 
Could we unknowingly be creating potentially harmful bacteria? </P><P>Yes. One could assay already harmful bacteria for increased pathogenesis, for ability to survive against immune system components. Research has even been done in a similar area where viruses were evolved to reproduce as quickly as possible (though the viruses quickly lost the ability to infect cells). Thankfully there are incredibly stringent rules for obtaining and working with pathogenic bacteria.</P><P>But could we accidently produce dangerous bacteria? Admittedly many of the traits desirable in ‘useful’ bacteria- sturdiness, rapid replication- are also those which cause increased pathogenesis. 
It should be noted that we are proposing the fusion of bacteria of the same species. Whilst the base composition of our ‘progeny’ may be unique, the genes will be same. There will always risks, a protein gaining a novel product, a gene switched off, but these are not risks that are unique to our project- they apply to any kind of breeding. However, as the many de novo negative mutations that arise even from human reproduction, we do need to be careful.</P><P>There are risks with not understanding how or why our experiments have modified the bacteria on a genetic level. If adverse effects are expressed de novo by our bacteria, it may be difficult to understand what has caused them- as it could be due to modifications in any number of the organism’s genes- our process can modify all of them!</P><H2><SPAN class="mw-headline" id="Environmental_Safety-_BioBricks">Environmental Safety- BioBricks</SPAN></H2><P>To convert <I>B. subtilis</I> 168 to L-forms that express HBsu-xFP we transformed them with different BioBricks, using plasmids as vectors. The BioBrick could theoretically pass from these cells into other bacteria. Could the genes be expressed, and would this be dangerous?
</P><P>First, our L-forms also currently contain antibiotic resistance markers. These are necessary as they allow us to grow the L-forms up without risk of contamination. However, if released into the wild these genes could pass into other, pathogenic bacteria, reducing the effectiveness of various antibiotics in medicine. Antibiotic resistance is already a major problem facing healthcare! Therefore not only is releasing bacteria containing antibiotic resistance conferring genes dangerous, it is also illegal. If we aimed to release our L-forms into the environment, we would first need to remove these markers. 
</P><P>As we are using <I>B. subtilis</I> as a chassis our BioBricks can not be expressed unless integrated into the bacterial genome. This requires the genome to have an area of homology with the plasmid, so homologous recombination can take place and the BioBrick move from the plasmid onto the genome. If one of these vectors did get into a bacteria with no region of homology, it may be able to replicate in the cell but would have no gene expression, and so would be harmless. Human, or any animals cells thankfully do not have any of the homology regions we used and so there is no risk of us taking up this DNA. Even if we did, the chance of this happening would be miniscule.
</P><H3><SPAN class="mw-headline" id="L-form_switch_BioBrick">L-form switch BioBrick</SPAN></H3><P>The L-forms switch BioBrick is on a pSB1C3 integration plasmid, which has homology to <I>pbpb</I> and <I>murE</I>. Therefore the BioBrick would be able to integrate into the genome of any organisms that contains these regions. A NCBI Nucleotide Blast of the <I>murE</I> homology region gave no species other than <I>B. subtilis</I> that contained regions with over 85% homology (although a number of bacteria have 100% homology for <I>pbpb</I>, both regions need high homolgy for integration). This suggests that this BioBrick could only be integrated into this organism. 
</P><P>If the BioBrick is expressed, it will cause the bacteria to lose it's cell wall. As mentioned in <A href="https://2013.igem.org/Team:Newcastle/HP/Ethics#Environmental_Safety-_Plants" class="external text" rel="nofollow">Environmental Safety- Plants</A> this should merely cause the bacteria to quickly lyse, and shouldn't have any negative effects.
</P><P>The BioBrick also contains a cat gene, which confers bacteriostatic chloramphenicol resistance. As discussed above, this would give any organism that takes up and integrates the BioBrick resistance to chloramephenicol. This should not be too troublesome as on <I>B. subtilis</I> can take up the BioBrick, and this organism is harmless so there shouldn't be a need to kill it with antibiotics.  
</P><H3><SPAN class="mw-headline" id="HBSU-xFP">HBSU-xFP</SPAN></H3><P>The HBSU-xFP BioBrick was submitted to the iGEM repository on a pSB1C3 plasmid. It is not under the control of a promoter, so cannot be expressed even if it were integrated into a bacterial genome. This should also not be possible as the plasmid does not have a region for integration. The plasmid does have a chloramphenicokl resistance marker on it however.
</P><P>However we initially had the HBSU-xFP BioBrick on a pMUTIN4 plasmid. This plasmid has the BioBrick under the control of an IPTG inducable promoter, and also contains a lacI gene. Therefore even if the plasmid integrates the Biobrick will only be expressed if the transformed bacteria is exposed to IPTG (or lactose if the bacteria produces beta-Galactosidase). If the BioBrick is expressed, it could affect sporulation as HBsu is involved with making the DNA more compact before sporulation. As our BioBrick will lead to higher levers of HBsu in the cell, it may make the bacteria sporulate more easily, though this is purely conjecture. The BioBrick is also very large- ~10000bp so it may disrupt local genes.    
</P><P>The pMUTIN4 plasmid does have a single integration site with homology to <I>amyE</I>, so could integrate into the genome of any organism that contains this gene. It also has two bactericidal antibiotic resistance markers: <I>ery</I>, which is expressed in <I>B. subtilis</I> and <I>amp</I>, expressed in <I>E. coli</I>. In <I>E. coli</I> the the plasmid will not integrate but this is not necessary for <I>amp</I> expression.  
</P><H2><SPAN class="mw-headline" id="L-forms_and_Evolutionary_Theory">L-forms and Evolutionary Theory</SPAN></H2><P>L-forms are currently used as a model for primitive cells. Nearly all modern bacteria possess a cell wall, and those which do not, such as the Tenericutes, appear to have evolved from cell walled organisms. However it is reasonable to assume that cell wall-less cells preceded cell walled organisms. Possession of a cell wall affords cells with a host of benefits, it offers protection from physical and chemical damage and importantly osmotic instability, allowing cells to occupy a more diverse range of environments. It also protects cells from unwanted horizontal gene transfer.  L-forms give clues as to what bacterial cells were like before cell walls became the norm.
</P><P>Simple lipid vesicles are a popular candidate for studying early life. However, using these as models of primitive cells abstracts out the internal composition of cells. As put by <A href="http://www.ncbi.nlm.nih.gov/pubmed/23108858" class="external text" rel="nofollow">Briers et al. 2012</A> “[research into] L-forms is probing the last universal common ancestor while [research into] giant vesicles is probing the first living cellular systems”. L-forms provide hints as to the division mechanics of ancient bacteria.
</P><P>Lipid vesicles are able to divide through physiochemical reactions. Division of L-forms appears to be strikingly similar to that of lipid vesicles and neither requires the ubiquitous protein FtsZ. It is thought that the way in which L-forms divide could give us an insight into primitive cells’ mode of division, which has still been retained and used when necessary e.g. when cell wall synthesis is compromised. 
</P><P>Since, so far, no quantitative measurements have been made on the division mechanics of L-forms, it is unknown whether the modes of division of lipid vesicles and L-forms are actually the same or just look similar. This project allows <I>B. subtilis</I> L-forms to be made easily which makes future exploration of this more likely.
Cell wall-less bacteria would likely have a greater rate of evolution: 
</P><P>-&gt; L-forms have been observed undergoing spontaneous fusion events which lead to recombined genomes or cells with multiple genomes and 
</P><P>-&gt; there would be an increased horizontal gene transfer due to vesicles budding off from a parent cell, which would carry genetic material through environment.
</P><H3><SPAN class="mw-headline" id="Relation_to_Religion">Relation to Religion</SPAN></H3><P>The position of many modern religions, with respects to evolution, is something of an unknown quantity.  Though numerous religions have been viewed as rejecting theories of evolution (more so in the case of people of more fundamental religious faith), in recent years this association has softened.  In fact religions that choose to address this issue now often support theories of evolution to some degree - perhaps on the back of such overwhelming evidence as is provided by the scientific community.
</P><P>L-forms do not specifically address the issues of a religion-evolution debate head on.  More so, the postulation of L-form-like organisms as an early wall-less ancestor to modern organisms provides another link in the chain tracing modern life back to its earliest origins.
</P><H2><SPAN class="mw-headline" id="References">References</SPAN></H2><P><A href="http://www.ncbi.nlm.nih.gov/pubmed/23108858" class="external text" rel="nofollow">Briers Y, Walde P, Schuppler M and Loessner M (2012) How did bacterial ancestors reproduce? Lessons from L-form cells and giant lipid vesicles Multiplication similarities between lipid vesicles and L-form bacteria. <I>Bioessays</I>, <B>34</B>, 1078–1084</A></P><P><A href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3603455/" class="external text" rel="nofollow">Errington J. (2013) L-form bacteria, cell walls and the origins of life. <I>Open Biology</I>,  <B>3</B>, 120143.</A></P><P><A href="http://www.ncbi.nlm.nih.gov/pubmed/19212404" class="external text" rel="nofollow">Leaver M., Dominguez-CuevasP, Coxhead J.M., Daniel R.A. and Errington J. (2009) Life without a wall or division machine in Bacillus subtilis.  <I>Nature</I>, <B>457</B>, 849-853.</A></P><DIV class="printfooter">
Retrieved from &quot;<A href="http://2013.igem.org/Team:Newcastle/HP/Ethics">http://2013.igem.org/Team:Newcastle/HP/Ethics</A>&quot;</DIV></DIV></DIV><DIV id="footer-box" class="noprint"><DIV id="footer"><UL id="f-list"><LI id="t-recentchanges"><A href="/Special:RecentChanges" title="Recent changes">Recent changes</A></LI><LI id="t-whatlinkshere"><A href="/Special:WhatLinksHere/Team:Newcastle/HP/Ethics" title="List of all wiki pages that link here [j]" accesskey="j">What links here</A></LI><LI id="t-recentchangeslinked"><A href="/Special:RecentChangesLinked/Team:Newcastle/HP/Ethics" title="Recent changes in pages linked from this page [k]" accesskey="k">Related changes</A></LI><LI id="t-specialpages"><A href="/Special:SpecialPages" title="List of all special pages [q]" accesskey="q">Special pages</A></LI><LI><A href="/Special:Preferences">My preferences</A></LI></UL></DIV><DIV id="footer"><UL id="f-list"><LI id="t-print"><A href="/wiki/index.php?title=Team:Newcastle/HP/Ethics&amp;printable=yes" title="Printable version of this page [p]" accesskey="p">Printable version</A></LI><LI id="t-permalink"><A href="/wiki/index.php?title=Team:Newcastle/HP/Ethics&amp;oldid=269620" title="Permanent link to this revision of the page">Permanent link</A></LI><LI id="privacy"><A href="/2013.igem.org:Privacy_policy" title="2013.igem.org:Privacy policy">Privacy policy</A></LI><LI id="disclaimer"><A href="/2013.igem.org:General_disclaimer" title="2013.igem.org:General disclaimer">Disclaimers</A></LI></UL></DIV></DIV></DIV></DIV></BODY></HTML>