<HTML xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en" dir="ltr">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki  ltr ns-0 ns-subject page-Team_Newcastle_Modelling_CellShapeModel"><DIV id="globalWrapper"><DIV id="top-section"><DIV id="p-logo"><A href="/Main_Page" title="Main Page">&quot;
	    </A></DIV><DIV id="menubar" class="left-menu noprint"><UL><LI class="selected"><A href="/Team:Newcastle/Modelling/CellShapeModel">Page               </A></LI><LI class="new"><A href="/wiki/index.php?title=Talk:Team:Newcastle/Modelling/CellShapeModel&amp;action=edit&amp;redlink=1">Discussion               </A></LI><LI><A href="/wiki/index.php?title=Team:Newcastle/Modelling/CellShapeModel&amp;action=edit">View source               </A></LI><LI><A href="/wiki/index.php?title=Team:Newcastle/Modelling/CellShapeModel&amp;action=history">History               </A></LI><LI style="color:white;cursor:default">teams</LI></UL></DIV><DIV class="right-menu noprint" id="menubar"><UL><LI id="pt-login"><A href="/wiki/index.php?title=Special:UserLogin&amp;returnto=Team:Newcastle/Modelling/CellShapeModel" title="You are encouraged to log in; however, it is not mandatory [o]" accesskey="o">Log in</A></LI></UL></DIV><DIV id="search-controls" class="noprint"><FORM action="/Special:Search" id="searchform"> </FORM></DIV></DIV><DIV id="content"><H1 class="firstHeading">Team:Newcastle/Modelling/CellShapeModel</H1><DIV id="bodyContent"><H3 id="siteSub" class="noprint">From 2013.igem.org</H3><DIV><DIV id="pop_up"><DIV id="pop_up_underlay"> </DIV><DIV id="pop_up_display"><DIV id="pop_up_close">X</DIV><DIV id="pop_up_content"> </DIV></DIV></DIV><DIV class="menu" class="grid_12"><UL class="top_menu" style="margin-left:10px"><LI><A href="https://2013.igem.org/Team:Newcastle">Home</A></LI><LI><A href="#" onclick="return false;">Team</A></LI><LI><A href="https://igem.org/Team.cgi?year=2013&amp;team_name=Newcastle" target="_blank">Official Profile</A></LI><LI><A href="/Team:Newcastle/Team">Individual Profiles</A></LI></UL><LI><A href="#" onclick="return false;">Project</A></LI><LI><A href="https://2013.igem.org/Team:Newcastle/Project">Overview</A></LI><LI><A href="https://2013.igem.org/Team:Newcastle/Project/L_forms">L-forms</A></LI><LI><A href="https://2013.igem.org/Team:Newcastle/Project/shuffling_endosymbiosis">Genome Shuffling</A></LI><LI><A href="https://2013.igem.org/Team:Newcastle/Project/plants">L-forms in Plants</A></LI><LI><A href="https://2013.igem.org/Team:Newcastle/Project/shape_shifting">Shape Shifting</A></LI><LI><A href="#" onclick="return false;">Parts</A></LI><LI><A href="https://2013.igem.org/Team:Newcastle/Parts/l_form_switch">L-form Switch</A></LI><LI><A href="https://2013.igem.org/Team:Newcastle/Parts/HBsu-fp">HBsu-xFP</A></LI><LI><A href="https://2013.igem.org/Team:Newcastle/Parts/e-coli_swich">E-coli Switch</A></LI><LI><A href="#" onclick="return false;">Models</A></LI><LI><A href="https://2013.igem.org/Team:Newcastle/Modelling/Introduction">Introduction</A></LI><LI><A href="https://2013.igem.org/Team:Newcastle/Modelling/BioNetGen">BioNetGen</A></LI><LI><A href="https://2013.igem.org/Team:Newcastle/Modelling/CellShapeModel">Cell Shape Model</A></LI><LI><A href="https://2013.igem.org/Team:Newcastle/Modelling/Cell_Fusion">Cell Fusion</A></LI><LI><A href="https://2013.igem.org/Team:Newcastle/Modelling/HBsu_Fusion_Protein">HBsu-xFP</A></LI><LI><A href="https://2013.igem.org/Team:Newcastle/Modelling/L-form_Switch">L-form Switch</A></LI><LI><A href="#" onclick="return false;">Notebook</A></LI><LI><A href="https://2013.igem.org/Team:Newcastle/Notebook/achievements">Achievements</A></LI><LI><A href="https://2013.igem.org/Team:Newcastle/Notebook/timeline">Timeline</A></LI><LI><A href="https://2013.igem.org/Team:Newcastle/Notebook/calendar">Calendar</A></LI><LI><A href="https://2013.igem.org/Team:Newcastle/Notebook/protocols">Protocols</A></LI><LI><A href="https://2013.igem.org/Team:Newcastle/Notebook/agenda">Meetings</A></LI><LI><A href="#" onclick="return false;">Implications</A></LI><LI><A href="https://2013.igem.org/Team:Newcastle/Outreach/Overview">Overview</A></LI><LI><A href="https://2013.igem.org/Team:Newcastle/Outreach/BioGame">BioGame</A></LI><LI><A href="https://2013.igem.org/Team:Newcastle/Outreach/uk_meet">YSB 1.0</A></LI><LI><A href="https://2013.igem.org/Team:Newcastle/Outreach/Workshop">Modelling Workshop</A></LI><LI><A href="https://2013.igem.org/Team:Newcastle/Outreach/Architecture">Architecture Workshop</A></LI><LI><A href="https://2013.igem.org/Team:Newcastle/HP/Safety">Safety</A></LI><LI><A href="https://2013.igem.org/Team:Newcastle/HP/Synthetic_Bio">Synthetic Biology</A></LI><LI><A href="https://2013.igem.org/Team:Newcastle/HP/Ethics">Ethics</A></LI><LI><A href="https://2013.igem.org/Team:Newcastle/HP/Law">Law</A></LI><LI><A href="#" onclick="return false;">Attributions</A></LI><LI><A href="https://2013.igem.org/Team:Newcastle/Collaborations">Collaborations</A></LI><LI><A href="https://2013.igem.org/Team:Newcastle/Attributions">Acknowledgments</A></LI><LI><A href="https://2013.igem.org/Team:Newcastle/Sponsors">Sponsors</A></LI><LI><A href="https://2013.igem.org/Team:Newcastle/Architecture">Architecture</A></LI><LI><A href="https://2013.igem.org/Main_Page">iGEM</A></LI></DIV><TABLE id="toc" class="toc"><TBODY><TR><TD><DIV id="toctitle"><H2>Contents</H2></DIV><UL><LI class="toclevel-1 tocsection-1"><A href="#Cell_Shape_Model"><SPAN class="tocnumber">1</SPAN><SPAN class="toctext">Cell Shape Model</SPAN></A></LI><LI class="toclevel-2 tocsection-2"><A href="#Background"><SPAN class="tocnumber">1.1</SPAN><SPAN class="toctext">Background</SPAN></A></LI><LI class="toclevel-2 tocsection-3"><A href="#Approach"><SPAN class="tocnumber">1.2</SPAN><SPAN class="toctext">Approach</SPAN></A></LI><LI class="toclevel-3 tocsection-4"><A href="#Unlimited_Cell_Growth_Model"><SPAN class="tocnumber">1.2.1</SPAN><SPAN class="toctext">Unlimited Cell Growth Model</SPAN></A></LI><LI class="toclevel-4 tocsection-5"><A href="#Physiological_basis"><SPAN class="tocnumber">1.2.1.1</SPAN><SPAN class="toctext">Physiological basis</SPAN></A></LI></UL><LI class="toclevel-3 tocsection-6"><A href="#Limitation_of_Growth_by_a_Set_Boundary"><SPAN class="tocnumber">1.2.2</SPAN><SPAN class="toctext">Limitation of Growth by a Set Boundary</SPAN></A></LI><LI class="toclevel-3 tocsection-7"><A href="#Methods"><SPAN class="tocnumber">1.2.3</SPAN><SPAN class="toctext">Methods</SPAN></A></LI><LI class="toclevel-2 tocsection-8"><A href="#Model"><SPAN class="tocnumber">1.3</SPAN><SPAN class="toctext">Model</SPAN></A></LI><LI class="toclevel-3 tocsection-9"><A href="#Discussion"><SPAN class="tocnumber">1.3.1</SPAN><SPAN class="toctext">Discussion</SPAN></A></LI><LI class="toclevel-3 tocsection-10"><A href="#Simulation"><SPAN class="tocnumber">1.3.2</SPAN><SPAN class="toctext">Simulation</SPAN></A></LI><LI class="toclevel-2 tocsection-11"><A href="#References"><SPAN class="tocnumber">1.4</SPAN><SPAN class="toctext">References</SPAN></A></LI></TD></TR></TBODY></TABLE><H1><SPAN class="mw-headline" id="Cell_Shape_Model">Cell Shape Model</SPAN></H1><H2><SPAN class="mw-headline" id="Background">Background</SPAN></H2><P><B><A href="https://2013.igem.org/Team:Newcastle/Project/shape_shifting" class="external text" rel="nofollow">One of our research themes</A> focuses on the ability of L-form bacteria to fit the mould and/or fill spaces by growing into them. We would like to explore the biophysical properties of the cell and the limits of the cell size the L-form can grow to before dividing by constructing a 2-dimensional cell model.</B></P><P>The shape of the bacterial cell is usually determined by the structures of the cell wall and bacterial cytoskeleton. The cell wall of <I>B.subtilis</I> is a thick layer of peptidoglycan which contains cell-wall proteins, techoic and lipotechoic acids. Peptidoglycan is a rigid structure which helps to protect the cell from osmotic pressure and mechanical damage. It also helps the cell to maintain a constant shape, while not specifically defining it. This role is played by the cytoskeleton. The bacterial cytoskeleton is largely similar to that of eukaryotes. It is comprised of three types of protein structures: FtsZ, MreB and crescentin (only present in certain species) which are homologous to eukaryotic tubulin, actin and intermediate filaments respectively. These proteins also perform other functions in the cell; usually they are involved in cell growth and division and therefore are often essential for the cell's survival (<A href="https://2013.igem.org/Team:Newcastle/Modelling/CellShapeModel#References" class="external text" rel="nofollow">Shih and Rothfield, 2006</A>). The communication between the two regulatory components is assured by a series of membrane-associated enzymes (penicillin binding proteins)(<A href="https://2013.igem.org/Team:Newcastle/Modelling/CellShapeModel#References" class="external text" rel="nofollow">Cabeen, Jacobs-Wagner, 2005</A>).
</P><P>Because L-forms lack the peptidoglycan cell wall, cells naturally adopt the most energetically favourable shape (i.e. sphere) regardless of the cytoskeleton structure. With protoplasts (cells with chemically digested cell wall), the growth, variation in shape and division is restricted due to the limited amount of the cell membrane which simply ruptures if the pressure inside the cell increases. The L-forms created with our BioBrick will have a mutation in the <I>MurE</I> gene which causes spontaneous increase in production of the <I>IspA-like</I> gene product which results in increased synthesis of the cell membrane. This stabilises the cell and allows for growth and division.
</P><H2><SPAN class="mw-headline" id="Approach">Approach</SPAN></H2><P>For the purposes of the study the complex model of the growing cell inside of the confined space can be broken down to simpler models of the system in three phases. The first phase would be a constantly growing cell, followed by a model of the cell when it touches the boundaries and therefore changes the trajectory of the membrane growth and finally starts adopting the shape of the boundaries.
</P><H3><SPAN class="mw-headline" id="Unlimited_Cell_Growth_Model">Unlimited Cell Growth Model</SPAN></H3><H4><SPAN class="mw-headline" id="Physiological_basis">Physiological basis</SPAN></H4><P>Since L-forms do not have the cell wall, the shape of the cell is determined by the laws of Newtonian physics and therefore will be as close to the shape of a sphere as possible. For the purposes of this model we will assume that the cell is spherical to start with and therefore will expand evenly in all directions. L-forms of <I>Bacillus subtilis</I> made with our BioBrick will have an induced mutation in the <I>IspA</I> gene, which allows the cell to constitutively produce cell membrane phospholipids and therefore the surface area of the membrane will increase over time as the cell grows. Provided that the cell has a sufficient nutrient supply, phospholipids will be produced at a constant rate <I>p</I>, and transported to the surface of the membrane at rate T. Once at the surface, they are inserted into the membrane at rate <I>i</I>, which is determined by the maximum velocity of an enzyme x. These processes will result in an increase of the sphere’s radius at a rate . In the paper written by <A href="https://2013.igem.org/Team:Newcastle/Modelling/CellShapeModel#References" class="external text" rel="nofollow">D. Fanelli and A.J. McKane</A> on &quot;Thermodynamics of vesicle growth and instability&quot; it is explained that the growth of a vesicle is quasistatic in nature, so an equation for growth of the area is given:
</P><P>However, this model seems physiologically unrealistic. As to allow for the exponential growth of the radius, the cell would have to produce lipids at an exponentially increasing rate, which is larger than the rate of increase of the radius by a factor of 2π. This seems highly improbable, as even at infinite concentration of source lipids inside the cell, the rate at which they are converted to phospholipids and transported to the membrane must be limited by the rate of diffusion, concentration of enzymes, and their turnover rate. While the rate of diffusion and the turnover rate of the enzyme are measurable constants, enzyme concentration is expected to be more or less constant to support homeostasis.
</P><P>Therefore we propose a simplistic model of growth which is based on following assumptions:
</P><UL><LI> the cell is a perfect sphere
</LI><LI> lipids from which the membrane is made of are uniformly distributed throughout the cell and the membrane has a constant thickness
</LI><LI> the cell is in a growth phase throughout the experiment
</LI><LI> rate of conversion into membrane phospholipids is constant
</LI><LI> the insertion of the synthesised membrane lipids is constant and spontaneous
</LI><LI> pressure inside and outside of the cell is quasistatic.
</LI><LI> there's sufficient nutrient supply to sustain growth
</LI></UL><H3><SPAN class="mw-headline" id="Limitation_of_Growth_by_a_Set_Boundary">Limitation of Growth by a Set Boundary</SPAN></H3><P>As the maximum volume of the L-forms cell before it divides is unknown, for the purposes of this modelling exercise we assume it's larger than the volume of the chamber and is negligible. Here we work under assumption that the cell grows uniformly in all directions as there's no evidence in the papers that would suggest otherwise. The cell membrane would not adhere to the sides of the chamber, therefore the new membrane would be redistributed towards the spare space, gradually filling the whole chamber. We used a square shape in our model.
</P><H3><SPAN class="mw-headline" id="Methods">Methods</SPAN></H3><P>Given that a cell is a sphere with a volume  which grows over time as , it has a surface area which is dependant on time .
Membrane precursor lipids are being constantly synthesised, and their total mass is equal to , where  is the initial mass of membrane lipids in the cell and  is a rate constant. Therefore the area of the cell will grow proportionally to the concentration of precursor lipids in the cell,  and a rate constant λ.
</P><P>Hence, , so
, or
, where </P><P>This can be solved for r(t):
</P><P>, where a A is an integration constant, determined by setting , so
</P><P>, so
</P><H2><SPAN class="mw-headline" id="Model">Model</SPAN></H2><P>The software of choice for this particular model was MATLAB as it allows a quick visualisation, graphing and mathematical operations.
The model was first constructed on paper, then adapted for MATLAB. 
</P><P>The meaning of each line is explained in green comments.
Some of the parameters used in this model were taken from the NCBI database, whereas others were estimated due to lack of information. Initial mass of membrane lipid was predicted to be relative to the general bacterial <A href="http://www.sciencedirect.com/science/article/pii/002228367090255X" class="external text" rel="nofollow">weight/size</A>. The rate of lipid synthesis and their conversion into membrane are identical as the integration is known to happen spontaneously.
</P><PRE class="brush: matlab">
%plots a time dependent circle with a growing radius which stops growing

	cellv=10^(-15); %mean cell volume
	t_end=70; %growth time (amount time for which the system is run, and does not equal time during which the cell fills the shape, this time is much smaller and can be determined from the model)
	m=100;
	lambda=10^(-10); % rate of conversion of mass into area
	M0=10^(-17); % initial mass of membrane lipids
	K=10^(-10); %rate of production of the membrane lipids
	a=10^(-4); %length of the side of a chamber
	n=100;
	tspan = linspace(0,t_end,m);
	
	r0=(cellv/pi*3/4)^(1/3); %radius of the cell
	alpha=(3*lambda)/(32*pi^2); % constant which defines the rate at which the radius of the cell grows
	
	x=zeros(n);
	y=zeros(n);
	
	x=linspace(-a,+a,n);
	y=linspace(-a,+a,n);
	
	figure(1)
	clf
	axis([-a +a -a +a])
	hold on
	
	r=r0;
	k=0;

	while r&lt;=a %below is the code to draw a series of circles representing the cell with a growing radius over the timespan until the cell touches the boundary
		
		k=k+1;
		t=tspan(k);
		r=(5*alpha*(M0*t+0.5*K*t^2)+r0^5)^(1/5); %radius which changes with time
		c(k)= 2*pi*r; %circumference of the cell

                rectangle('Position',[-r,-r,r*2,2r], 'Curvature',[1], 'Edgecolor',[0,0,1], Facecolour, [0,0,1]) %plots the circle with the new radius

		pause(0.5)
		radius(k)=r;

	end;

        hold all
        l=0;
        circ=c(k);
        crv=1 %curvature (represented by the fracture of the rectangle's side which is curved

        while crv&gt;0.15 %limit of curvature % the bit below codes for a cycle which draws the new cell outline
              
		k=k+1;
		t=tspan(k);
		r=(5*alpha*(M0*t+0.5*K*t^2)+r0^5)^(1/5);
		cn(k)= 2*pi*r; %new cell circumference

                l=((cn(k)-circ/4); %expansion of the line of coherence between the cell membrane and the chamber 
                crv=1-2*l/a;

                rectangle('Position',[-a,-a,a*2,2a], 'Curvature',[crv], 'Edgecolor',[0,0,1], Facecolour, [0,0,1]) %plots a rectangle with the side equal to the side of the chamber and curvature determined by the growth of the cell's circumference due to lipid synthesis and integration of lipids into the cell membrane.

		pause(0.5)
		radius(k)=r;

       end;

</PRE><H3><SPAN class="mw-headline" id="Discussion">Discussion</SPAN></H3><P>This model can be improved by conducting experiments which would allow us to find accurate parameters such as the rate of membrane synthesis and maximum membrane torsion. Freeing the model of a few assumptions would also be beneficial. For example if the effects of nutrient depletion were found, and if the maximum size the cell could grow to before dividing was derived, then these could be worked into the model. The model could also be expanded in terms of factoring in the bending energy of the membrane in a particular shape. This would allow us to compare the models of the cells adopting different shapes and to evaluate which shape would me more readily assumed. It would also be interesting to see whether the state of the cell (i.e. L- or rod form) before it enters the chamber has any bearing on the way it fills the space. Also, according to Svatina and Zeks(<A href="https://2013.igem.org/Team:Newcastle/Modelling/CellShapeModel#References" class="external text" rel="nofollow">2002</A>), different ratios between the surface area of the membrane and the cell volume yield different cell shapes due to the energy balance involved. We believe that this too may have implications on the cell's behaviour, which can be factored in in the future.
</P><H3><SPAN class="mw-headline" id="Simulation">Simulation</SPAN></H3><P>2D simulation of the Cell Growth model.
</P><H2><SPAN class="mw-headline" id="References">References</SPAN></H2><P><A href="http://www.ncbi.nlm.nih.gov/pubmed/16012516" class="external text" rel="nofollow">Cabeen, M.T. and Jacobs-Wagner, C. 2005 Bacterial cell shape. Nat Rev Microbiol. 3:601-10</A></P><P><A href="http://www.theory.physics.manchester.ac.uk/~ajm/fm08.pdf" class="external text" rel="nofollow">Fanelli D., McKane A.J. 2008. Thermodynamics of vesicle growth and instability. Physical Review E 78, 051406</A></P><P><A href="http://www.ncbi.nlm.nih.gov/pubmed/12382320" class="external text" rel="nofollow">Svetina S., Zeks B. 2002. Shape Behaviour of Lipid Vesicles as the Basis of Some Cellular Processes. The Anatomical Record 268:215-225</A></P><P><A href="http://www.ncbi.nlm.nih.gov/pubmed/16959967" class="external text" rel="nofollow">Shih Y.L., Rothfield L. 2006. The Bacterial Cytoskeleton. Microbiology and Molecular Biology Reviews p.729-754.</A></P><P><A href="http://www.sciencedirect.com/science/article/pii/002228367090255X" class="external text" rel="nofollow">Mindich L, 1970 Membrane Syntheis in <I>Bacillus subtilis</I> Journal of Molecular Biology</A></P><DIV class="printfooter">
Retrieved from &quot;<A href="http://2013.igem.org/Team:Newcastle/Modelling/CellShapeModel">http://2013.igem.org/Team:Newcastle/Modelling/CellShapeModel</A>&quot;</DIV></DIV></DIV><DIV id="footer-box" class="noprint"><DIV id="footer"><UL id="f-list"><LI id="t-recentchanges"><A href="/Special:RecentChanges" title="Recent changes">Recent changes</A></LI><LI id="t-whatlinkshere"><A href="/Special:WhatLinksHere/Team:Newcastle/Modelling/CellShapeModel" title="List of all wiki pages that link here [j]" accesskey="j">What links here</A></LI><LI id="t-recentchangeslinked"><A href="/Special:RecentChangesLinked/Team:Newcastle/Modelling/CellShapeModel" title="Recent changes in pages linked from this page [k]" accesskey="k">Related changes</A></LI><LI id="t-specialpages"><A href="/Special:SpecialPages" title="List of all special pages [q]" accesskey="q">Special pages</A></LI><LI><A href="/Special:Preferences">My preferences</A></LI></UL></DIV><DIV id="footer"><UL id="f-list"><LI id="t-print"><A href="/wiki/index.php?title=Team:Newcastle/Modelling/CellShapeModel&amp;printable=yes" title="Printable version of this page [p]" accesskey="p">Printable version</A></LI><LI id="t-permalink"><A href="/wiki/index.php?title=Team:Newcastle/Modelling/CellShapeModel&amp;oldid=349920" title="Permanent link to this revision of the page">Permanent link</A></LI><LI id="privacy"><A href="/2013.igem.org:Privacy_policy" title="2013.igem.org:Privacy policy">Privacy policy</A></LI><LI id="disclaimer"><A href="/2013.igem.org:General_disclaimer" title="2013.igem.org:General disclaimer">Disclaimers</A></LI></UL></DIV></DIV></DIV></DIV></BODY></HTML>