<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_GeorgiaTech_Background skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:GeorgiaTech/Background</SPAN></H1><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><DIV id="mainContainer"><DIV id="menuContainer" align="center"><UL><LI><A href="https://2015.igem.org/Team:GeorgiaTech">HOME</A></LI><LI><A href="#">PROJECT
            					</A></LI><LI><A href="https://2015.igem.org/Team:GeorgiaTech/Description">Description</A></LI><LI><A href="https://2015.igem.org/Team:GeorgiaTech/Background">Background</A></LI><LI><A href="https://2015.igem.org/Team:GeorgiaTech/Achievements">Achievements</A></LI></UL><LI><A href="#">PARTS
            					</A></LI><LI><A href="https://2015.igem.org/Team:GeorgiaTech/Parts">Team Parts</A></LI><LI><A href="#">BENCH WORK
            					</A></LI><LI><A href="https://2015.igem.org/Team:GeorgiaTech/Experiments">Experiments</A></LI><LI><A href="https://2015.igem.org/Team:GeorgiaTech/Protocols">Protocols</A></LI><LI><A href="https://2015.igem.org/Team:GeorgiaTech/Results">Results</A></LI><LI><A href="https://2015.igem.org/Team:GeorgiaTech/Notebook">Timeline</A></LI><LI><A href="https://2015.igem.org/Team:GeorgiaTech/Safety">Safety</A></LI><LI><A href="#">BEYOND THE BENCH
            					</A></LI><LI><A href="https://2015.igem.org/Team:GeorgiaTech/Collaborations">Collaborations</A></LI><LI><A href="https://2015.igem.org/Team:GeorgiaTech/Practices">Human Practices</A></LI><LI><A href="#">TEAM
            					</A></LI><LI><A href="https://2015.igem.org/Team:GeorgiaTech/Members">Members</A></LI><LI><A href="https://2015.igem.org/Team:GeorgiaTech/Attributions">Attributions</A></LI></DIV><DIV id="contentContainer"><DIV class="band"><H2>Background</H2><H3>Click Chemistry</H3><P>Nobel Laureate Barry Sharpless coined the term “click chemistry” in 1998 to describe a class of chemical reactions that revolutionized the field of organic chemistry by placing heavy emphasis on reactions that are “modular, wide in scope, give very high yields, generate only inoffensive by-products that can be removed by nonchromatographic methods, and be stereospecific (but not necessarily enantioselective)” (Sharpless, Finn, Kolb, 2001). Many click reactions are also insensitive to oxygen and water and require only readily available reagents and solvents. This new approach to organic synthesis presents opportunities for the advancement of materials science, drug design, bioconjugation. 
</P><P><B>Figure 1. Copper-catalyzed azide-alkyne cycloaddition reaction.</B> The colored components indicate the corresponding atoms and bonds through the transformation from reactants to products. A terminal alkyne in the presence of Cu will react with an azide to form a stable triazole product.</P><P>The copper-catalyzed azide-alkyne cycloaddition (CuAAC) reaction (Figure 1) is the ideal example of a click reaction, having all of the desirable characteristics of click chemistry. The CuAAC reaction is <B>modular</B> because the azide and alkyne groups easily attach to a wide variety of organic R groups, the “modules”, that need to be joined. This reaction has the power to simplify the synthesis of a diverse array of complex compounds. Furthermore, the azide and alkyne are <B>unobtrusive</B> and do not affect the function of their R1 and R2 groups. This reaction also exhibits the rare combination of producing a thermodynamic product but at an extremely slow reaction rate that can be dramatically accelerated with a catalyst. The azide and alkyne carry a lot a of potential energy and will reliably proceed towards the extremely stable heteroatom triazole ring, but this requires heat, time, and high concentration. When copper ions are present, however, the reaction is <B>nearly immediate</B> (Sharpless, Finn, Kolb, 2001). The CuAAC reaction is also <B>bioorthogonal</B>, meaning that the reaction does not interfere with any normally occurring biological reactions.  For these reasons, the CuAAC reaction has a high potential for applications in the <B>delivery of drugs with few side effects</B>. One possible application is a “pro drug” delivery method where a pharmaceutical would be delivered in two separate inactive “halves” or precursors that could be joined by an azide and alkyne at a desired location to form an active drug without any negative side-effects.</P><P>Before the CuAAC reaction can be implemented <I>in vivo</I>, we must address the scarcity of free copper(I) ions in cells. Even though copper(I) is a requirement for living systems, copper is <B>cytotoxic</B> at the concentrations required for CuAAC because it reacts with oxygen to form peroxide and superoxide radicals (Pham, et al., 2013) that are prone to initiate radical chain reactions that propagate and destroy proteins and cell membranes. As a result, free copper ions are only present at sub-picomolar concentrations in cells, and most of a cell’s copper is sequestered by the <B>copper-binding proteins</B> that need copper in order to perform their respective functions (Dean, Qin, Palmer, 2012). Although copper is not readily available to catalyze the CuAAC reaction within cells, we hypothesize that an “unnatural” enzyme, derived from a copper-binding protein, can be evolved through phage display.</P></DIV><DIV class="band" style="background-color:#eeb211"><A name="phage"><H3>Phage Biology</H3></A><P>A bacteriophage is a type of virus that infects bacteria, and bacteriophages are composed of the genetic material that encodes for the specific proteins that encapsulate the phage. Specifically, the M13 filamentous phage particles that we will use consist of a 6400-nucleotide circular ssDNA plasmid with “a dozen or fewer closely packed genes and an intergenic (IG) region that contains sequences necessary for DNA replication and encapsidation.” (Russel, Lowman, Clackson, 2004). This genetic material is wrapped in five types of coat proteins: pVII, pIX, pVIII, pIII, and pVI and the body of the phage is extremely small, about 6.5 nm wide and about 900 nm tall. </P><P>The phage reproductive cycle includes infection of an <I>E. coli</I> host cell, replication of the phage’s DNA, assembly of the progeny, and lysogenic secretion from the <I>E. coli</I>. First, the pIII coat protein binds to the pili of an <I>E. coli</I>, and Tol proteins Q, R, and A of the <I>E. coli</I> depolymerize the phage coat proteins, allowing translocation of the viral ssDNA into the bacterial cytoplasm (Russel, Lowman, Clackson, 2004). Once inside the <I>E. coli</I> host, the ssDNA is converted into dsDNA by host polymerases. Initially, only dsDNA is generated, but as more of the pV protein is synthesized, it becomes available to bind to newly made single strands and blocks host polymerase from assembling complementary base pairs. After all of the genes are expressed, the phage progeny exit the <I>E. coli</I> lysogenically.</P><H3>Phage Display</H3><P>Phage display is a well-established technique that uses bacteriophages to study protein-protein and protein-ligand interactions. “Foreign” genetic code is fused to the bacteriophage genome, causing the bacteriophage to display the corresponding “unnatural” peptides or proteins on its coat.</P><P>The pIII coat protein is the most commonly used protein for phage display and can be performed with commercially available kits. One important advantage of pIII phage display is its compatibility with a variety of proteins of different sizes. In addition, pIII protein is relatively easy to modify.</P><P><B>Figure 2. Phage display.</B> Phages with varying genetic sequences express varying proteins with differing binding affinities to the substrate. The non-binders are washed, and the binders are subjected to additional cycles of this process and ultimately analyzed (sequenced). Adapted from a figure created by Dr. Graham Beards.</P><P>Phage populations can be encoded with entire libraries of varying mutations of a genetic sequence, resulting in the display of a diverse protein collection that can be <B>screened for binding</B> (Figure 2). Non-binders are washed away while binders are eluted, and the selected variants can be further mutated. This cycle is repeated until the library is enriched with only the strongest binders. In the end, all the proteins that exhibit desired binding properties can be efficiently identified and sequenced, making phage display a powerful tool for a wide array of evolutionarily driven protein-discovery research efforts.</P><P>In addition to binding-based screening, phage display can also be used for <B>activity-based screening</B>. For example, our project will require a given displayed protein to successfully facilitate the CuAAC reaction if it is to avoid being washed away. More specifically, we hope to use the <I>in vitro</I> selection capabilities presented by phage display to artificially evolve naturally occurring copper-binding proteins into unnatural enzymes that can aid the CuAAC reaction in acquiring copper for catalysis in copper-deficient environments.</P></DIV><DIV class="band" style="background-color:#000245"><A name="starting"><H3 style="color:white">Starting Proteins</H3></A><P style="color:white">Our search for an unnatural enzyme began with research on <B>natural copper-binding proteins</B>. Our selection process demanded proteins that were <B>small</B> (less than 80 kDa) in order to be successfully displayed on the pIII coat protein (Russel, Lowman, Clackson, 2004). Additionally, these proteins had to have the ability to bind to either <B>one or two copper(I) ions</B> in order to satisfy the requirements of the CuAAC reaction (Fokin, Worrell, Malik, 2013). We selected the following six proteins: <B>Atx1, CusF, plastocyanin, tyrosinase, Ace1</B>, and the N-terminus domain of <B>MNK</B>. We also worked with <B>three additional tyrosinase homologues</B> which were 80%, 70%, and 70% homologous to wild-type tyrosinase. These homologous genes were necessary for DNA shuffling, as explained below. We hypothesized that using these candidate proteins’ copper binding geometry combined with further CuAAC reaction-specific optimization would allow us to more efficiently search the “protein universe” and generate more diverse libraries; therefore, we would increase our chances of discovering a functional “clickase”.</P></DIV><DIV class="band"><A name="library"><H3>Protein Library Generation</H3></A><P>A fundamental component of our evolution-directed experiment is to <B>generate a library of mutated proteins</B>. After selecting six starting proteins, our next task would be to generate a large number of derivatives with similar but different properties. An ideal mutant would retain the high binding affinity for copper while also being able to exchange copper with an alkyne to trigger the CuAAC reaction. We used two different techniques to generate used our library of mutants: error-prone polymerase chain reaction (PCR) and DNA shuffling.</P><P>Natural condition PCR has three main steps: denaturation, annealing, and extension. During denaturation, the reaction mixture (containing DNA, primer, dNTPs, buffer, and DNA polymerase) is heated, allowing the double-stranded DNA to “melt” into single-stranded DNA. Specialized primers anneal to specific sites (usually the beginning and end of the target sequence) on the single stranded “template” DNA in accordance with complementary base pairing. Then Taq polymerase (a heat-resistant DNA polymerase) begins assembling base pairs along the entire template DNA strand, extending it until a complete dsDNA molecule is formed. After the 3 steps are complete, the cycle repeats again to duplicate the two parent DNA strands. Each cycle theoretically doubles the amount of DNA present; PCR generally exhibits exponential growth patterns, allowing for the rapid generation of large amounts of DNA replicates.</P><P><B>Figure 3. Error-prone PCR.</B> Modifications to the standard PCR conditions are required to achieve different error rates of the PCR.</P><P><B>Error-prone PCR</B> (Figure 3) is essentially the same as canonical PCR, but with <B>different reaction conditions</B>, including varied ratios of available dNTPs, elevated concentrations of already-present magnesium chloride, and addition of manganese chloride. In small amounts, magnesium chloride acts as a cofactor for DNA polymerase, but in large amounts, magnesium chloride will stabilize non-complementary base pairing. Manganese chloride also stabilizes non-complementary base pairing (Cirino, Mayer, Umeno 2003).</P><P><B>Figure 4. DNA Shuffling.</B> This technique involves digestion and reassembly of homologous genes to form unnatural chimeric genes with possible new activities.</P><P>In addition to EP-PCR, <B>DNA shuffling</B> can also be used to generate a diverse protein library. DNA shuffling is another widely used molecular evolution technique developed by Willem Stemmer in 1994. The protocol calls for a set of naturally occurring homologous genes that code for the proteins of interest (Stemmer, 1994).</P><P>Figure 4 illustrates the general process of DNA shuffling. The genes are digested by DNase1, and the resulting fragments are sorted by gel electrophoresis. Fragments of the desired length are selected for subsequent purification. The purified fragments are subjected to a “PCR-like” reaction in the absence of primers (Stemmer, 1994). Instead homologous fragments to <B>anneal according to complementary base pairing</B>, resulting in the formation of chimeric gene sequences. Starting with naturally occurring homologous sequences leverages billions of years of evolution to combine the most useful mutations from individual genes.</P><P>While proteins with homologous sequences can be directly used in DNA shuffling, when homologous gene sequences are not readily available, a protein library can still be built by using a combination of EP-PCR and DNA shuffling.  These genes can first be subjected to EP-PCR to generate sequence diversity prior to DNA shuffling.</P></DIV><DIV class="band" style="background-color:#aaaaaa"><H3>Bibliography</H3><OL><LI>Cirino, P. C.; Mayer, K. M.; Umeno, D. <I>Methods in Molecular Biology</I> 2003 231: 3-9</LI><LI>Clackson, T.; Lowman, H. B. <I>Phage Display: A Practical Approach</I>; Oxford University Press: New York, 2004.</LI><LI>Dean, K. M.; Qin, Y.; Palmer, A. E. <I>Biochimica et Biophysica Acta</I> 2012 1823(9), 1406–1415.</LI><LI>Fokin, V.V.; Worrell, B. T.; Malik, J. A. <I>Science</I> 2013 340(6131), 457-460</LI><LI>Kolb, H. C.; Finn, M. G.; Sharpless, K. B. <I>Angew. Chem. Int. Ed.</I> 2001, 40: 2004–2021.</LI><LI>Pham, A. N.; Xing, G.; Miller, C. J.; Waite, T. D. <I>Journal of Catalysis</I> 2013 301: 54-64</LI><LI>Stemmer, W.C. <I>Letters to Nature</I> 1994 370</LI></OL></DIV></DIV></DIV></DIV></DIV></DIV></DIV></BODY></HTML>