<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_GeorgiaTech_Notebook skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:GeorgiaTech/Notebook</SPAN></H1><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><DIV id="mainContainer"><DIV id="menuContainer" align="center"><UL><LI><A href="https://2015.igem.org/Team:GeorgiaTech">HOME</A></LI><LI><A href="#">PROJECT
            					</A></LI><LI><A href="https://2015.igem.org/Team:GeorgiaTech/Description">Description</A></LI><LI><A href="https://2015.igem.org/Team:GeorgiaTech/Background">Background</A></LI><LI><A href="https://2015.igem.org/Team:GeorgiaTech/Achievements">Achievements</A></LI></UL><LI><A href="#">PARTS
            					</A></LI><LI><A href="https://2015.igem.org/Team:GeorgiaTech/Parts">Team Parts</A></LI><LI><A href="#">BENCH WORK
            					</A></LI><LI><A href="https://2015.igem.org/Team:GeorgiaTech/Experiments">Experiments</A></LI><LI><A href="https://2015.igem.org/Team:GeorgiaTech/Protocols">Protocols</A></LI><LI><A href="https://2015.igem.org/Team:GeorgiaTech/Results">Results</A></LI><LI><A href="https://2015.igem.org/Team:GeorgiaTech/Notebook">Timeline</A></LI><LI><A href="https://2015.igem.org/Team:GeorgiaTech/Safety">Safety</A></LI><LI><A href="#">BEYOND THE BENCH
            					</A></LI><LI><A href="https://2015.igem.org/Team:GeorgiaTech/Collaborations">Collaborations</A></LI><LI><A href="https://2015.igem.org/Team:GeorgiaTech/Practices">Human Practices</A></LI><LI><A href="#">TEAM
            					</A></LI><LI><A href="https://2015.igem.org/Team:GeorgiaTech/Members">Members</A></LI><LI><A href="https://2015.igem.org/Team:GeorgiaTech/Attributions">Attributions</A></LI></DIV><DIV id="contentContainer"><DIV class="band"><H2>Notebook</H2><H3 class="heading">May</H3><DIV class="content"><H3>Week 1 (5/11)</H3><UL><LI>Conducted background research on cloning techniques, phage display, and modern applications of phage display</LI><LI>Completed necessary safety trainings</LI><LI>Brainstormed and researched possible project ideas</LI></UL><H3>Week 2 (5/18)</H3><UL><LI>Conducted background research on click chemistry, specifically the CuAAC reaction</LI><LI>Conducted background research on copper-binding proteins including copper chaperones, copper regulators, and tyrosinase.</LI><LI>Memorized amino acids</LI><LI>Kellie worked on the website draft from here onward</LI><LI>Learned how to use Pymol for protein modeling</LI></UL><H3>Week 3 (5/25)</H3><UL><LI>Conducted in-depth research on and selected six different copper binding proteins</LI><LI>Continued memorizing amino acids and were quizzed on them</LI></UL></DIV><H3 class="heading">June</H3><DIV class="content"><H3>Week 4 (6/1)</H3><UL><LI>Conducted research on mutagenesis techniques, particularly error prone PCR and DNA shuffling
</LI></UL><H3>Week 5 (6/8)</H3><UL><LI>Prepared diagrams and figures to illustrate key aspects to our presentation</LI></UL><H3>Week 6 (6/15)</H3><UL><LI>Prepared a presentation for our graduate advisors</LI><LI>Continued creating figures</LI></UL><H3>Week 7 (6/22)</H3><UL><LI>Presented a preliminary project proposal to our head advisor (PI), M.G. Finn</LI><LI>Conducted background research on tagging proteins, specifically the His-6 tag.</LI><LI>Met with a grad student, Zak Costello, as part of our outreach activities</LI><LI>Brainstormed beyond the bench/human practices ideas</LI></UL></DIV><H3 class="heading">July</H3><DIV class="content"><H3>Week 8 (6/29)</H3><UL><LI>Reviewed protocols for DNA purification, PCR, DNA transformation, and cell/bacteria culture</LI></UL><H3>Week 9 (7/6)</H3><UL><LI>First week in lab!</LI><LI>Prepared LB media and LB agarose gel</LI><LI>Practiced plasmid transformation, purification, and characterization</LI></UL><H3>Week 10 (7/13)</H3><UL><LI>Hosted a meetup with Lambert High school; shared project ideas, we assisted them in focusing their project  on a smaller/more feasible goal and helped them with the first steps of designing their experiment</LI><LI>Used ApE to map plasmids for our six proteins</LI><LI>Designed forward and reverse primers for our six proteins</LI><LI>Prepared for next week’s lab work by reading about the TOPO vector and related protocols.</LI><LI>Researched error-prone PCR conditions</LI></UL><H3>Week 11 (7/20)</H3><P style="text-align:center"><UL><LI>Primers arrived!</LI><LI>Used “primer flanking” PCR to add SfiI and NotI restriction sites to our genes while also cloning them out of their pUC57 vectors</LI><LI>Purified the PCR products using gel purification and analyzed products with the NanoDrop spectrophotometer and with a second gel</LI><LI>Ligated PCR product to TOPO vectors</LI><LI>Transformed the TOPO plasmid into DH5a electocompetent cells and plated for overnight growth</LI><LI>Repeated plasmid transformation and plating after colony growth was unsuccessful</LI><LI>Cultured some of the transformed cell in LB liquid media (more nurturing growth media as a positive control)</LI><LI>Unsuccessful colony growth again; repeated TOPO ligation with lower TOPO:Insert ratio</LI><LI>Plated cells from five (#2, #3, #1, MNK, T60) samples that grew successfully in liquid media in attempt to isolate colonies (in preparation for colony PCR the following day)</LI><LI>Ultimately achieved colonies for Atx1, plastocyanin, CusF, T60, and T70 ligation/transformations.</LI><LI>Sent in purified plasmids for sequencing. All sequences were as expected except for Atx1, but we assumed a sequencing error.</LI></UL></P><H3>Week 12 (7/27)</H3><UL><LI>Successfully grew colonies for Ace1, T80, and tyrosinase ligation/transformations. Did not send these for sequencing but assumed they were “correct” sequences.</LI><LI>Prepared a stock of TOPO vectors for 8 of the 9 proteins (the missing one being MNK) through mini prep protocol, with extra tubes of Tyr, T80, T70, and T60 stocks in preparation for DNA shuffling</LI><LI>Digested TOPO products using SfiI and NotI restriction enzymes</LI><LI>Digested pET23b vectors</LI><LI>Collected and purified digestion products using gel purification</LI><LI>Ligated digestion products - joined pET23b with our genes</LI><LI>Transformed ligation products into chemically competent DH5a <I>E. coli</I> cells</LI><LI>Performed TOPO-gene and pET23b digestions again</LI></UL></DIV><H3 class="heading">August</H3><DIV class="content"><H3>Week 13 (8/3)</H3><P>Expression Testing</P><UL><LI>Ligated digestion products from 7/31 (from our second digestion reaction)</LI><LI>Transformed ligation products into chemically competent BL21 E. coli cells and plated cells for overnight culture</LI><LI>Inoculated plated colonies for overnight culture</LI><LI>Inoculated an aliquot of overnight culture for further growth in 5 mL fresh LB</LI><LI>At log phase of bacterial growth, induced protein expression with IPTG</LI><LI>After substantial time for protein expression, measured expression with Aglient Protein 80 kit</LI><LI>Found excellent expression of tyrosinase homologs, but poor expression of Ace1, plastocyanin, and CusF</LI><LI>Atx1 and MNK were not tested for expression</LI></UL><P>Error-prone PCR</P><UL><LI>Prepared stock solutions for dNTPs, MgCl2, and MnCl2</LI><LI>Prepared reaction mixtures and reacted in PCR thermocycler for about 2 hours</LI><LI>Gel purified EP-PCR products and characterized them through NanoDrop spectrophometery and gel electrophoresis</LI><LI>TOPO ligation reaction to insert EP-PCR products into TOPO cloning vectors</LI><LI>Transformed and plated TOPO ligation vectors for overnight growth</LI><LI>Inoculated 10 individual tyrosinase colonies</LI><LI>Various EP-PCR reaction mixtures
</LI><LI>Following protocols C and F in <I>Evolutionary Methods in Biotechnology</I>.
</LI><LI>Only protocol C yielded good EP-PCR products for Atx1, CusF, and Ace1</LI></UL><LI>Empirically modified version of protocol C (elevated MnCl2 concentrations)
</LI><LI>Good EP-PCR products for all genes with error rate (~1%)</LI><LI>Using commercially available Green Master Mix
</LI><LI>Good EP-PCR products for all genes with error rate (~1%)</LI><H3>Week 14 (8/10)</H3><UL><LI>Purified plasmids (miniprep) from liquid cultures harboring 10 mutants of EP-PCR TOPO ligation vectors</LI><LI>Performed PCR to amplify tyrosinase homologues</LI><LI>Designed iGEM Informational Pamphlet</LI></UL><H3>Week 15 (8/17)</H3><UL><LI>Met to discuss how we are going to accomplish the remaining medal requirements</LI><LI>Outlined a plan for making informational videos about synthetic biology techniques</LI><LI>Outlined our poster and PowerPoint presentations</LI><LI>Distributed iGEM Informational Pamphlet to students in biology and chemistry classes</LI><LI>Scripted and storyboarded iGEM Video Series</LI></UL><H3>Week 16 (8/24)</H3><UL><LI>Worked on figures and graphics for our presentation, poster, and Wiki</LI><LI>Worked on subcloning our genes into pSB1C3 vectors and submitting the BioBrick to the iGEM Parts Registry</LI><LI>Worked on filming iGEM Video Series</LI></UL><H3>Bibliography</H3><OL><LI>Brakmann, s.; Lindermann, B. F. <I>Evolutionary Methods in Biotechnology</I>; WILEY-VCH Verlag GmbH &amp; Co. KGaA: Weinheim, 2004.</LI></OL></DIV><H3 class="heading">September</H3><DIV class="content"><H3>Week 17 (8/31)</H3><UL><LI>Continued refining our presentation and figures</LI><LI>Revised content for the Wiki page</LI><LI>Uploaded iGEM Video Series to YouTube</LI><LI>Submitted our BioBricks to the iGEM Parts Registry</LI></UL><H3>Week 18 - 20 (9/7 - 9/21)</H3><UL><LI>Practiced our presentation in front of mentors from last year’s iGEM team</LI><LI>Worked on our poster</LI><LI>Worked on our Wiki page</LI></UL></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></BODY></HTML>