Team:GeorgiaTech/Experiments
HOME
PROJECT
Description
Background
Achievements
PARTS
Team Parts
BENCH WORK
Experiments
Protocols
Results
Timeline
Safety
BEYOND THE BENCH
Collaborations
Human Practices
TEAM
Members
Attributions
Experiments
Preparing Genes
We ordered genes for our six starting proteins, and transformed their plasmids into DH5α chemically competent cells. The following day, we isolated the plasmids and analyzed our products by taking concentration measurements in the NanoDrop 1000 spectrophotometer and by confirming their purity through gel electrophoresis. The process described is illustrated in Figure 1. The protocols described (and all other protocols) can be found on our experiments and protocols page.
Figure 1. Plasmid purification process. The starting genetic material was transformed in to E. Coli cells for culture, and the plasmids were isolated according to the QIA Prep Miniprep protocol.
Preliminary Expression Tests
The genes that we ordered arrived in pUC57 vectors, so our next goal was to subclone them into expression vectors to eventually test how efficiently these genes can be expressed. Before arriving at this final experiment, we achieved three intermediate goals: introducing SfiI and NotI (our two desired restriction sites), ligating these extended gene fragments into TOPO vectors (for high copy cloning), and lastly subcloning them into pET23b vectors (for expression). We got the first step done by using primers flanked with the two aforementioned restriction sites. After the PCR, this new fragment of interest was isolated by gel electrophoresis and subsequent purification. The purified products were analyzed by the NanoDrop spectrophotometer and gel electrophoresis.
The purified PCR products were then ligated into the TOPO vector following TOPO TA cloning protocol. We transformed our TOPO ligation products into electrocompetent DH5α E. coli cells, plated them on solid media containing 100 ug/mL ampicillin, and incubated the plate at 37 C overnight. The next day, we performed colony PCR to analyze the colonies, and the positive colonies were inoculated in liquid culture for overnight growth. The results from the colony PCR are displayed in Figure 2. The day after that, the plasmids from the liquid cultures were isolated and sequenced to confirm the integrity and accuracy. Once confirmed, we purified large quantities of plasmid for subcloning purposes.
Figure 2. Colony PCR. Colony PCR products were subject to gel electrophoresis for 30 minutes at 135 V in 1% agarose gel. The colonies in the lanes from left to right were transformed with Atx1, CusF and Tyrosinase (60% homology), with 3 colonies of each gene separated by lanes carrying a 1 kb DNA ladder for comparison. The lanes marked with a red "X" contain empty TOPO vectors, and those marked with a green check mark contain TOPO vectors that properly ligated to our genes. We moved forward with our next experiments using only colonies that contained the properly ligated genes.
Our next step was to transfer our genes from their TOPO vectors into pET23b vectors. pET23b is an expression vector containing the T7 inducible promoter, which allows for expression testing with positive and negative control groups. The TOPO plasmids and expression vector pET23B-SF were digested by SfiI and NotI to isolate the inserts and vector backbone. The digested products were gel purified and used for subsequent T4 ligation. The ligation product was transformed, and individual colonies were analyzed by colony PCR before plasmid isolation and sequencing. Once the sequencing confirmed the integrity and accuracy of the gene insertion, the plasmids were transformed into BL21(ED3) chemically competent cells. A single colony was used to inoculate a 5 mL overnight culture.
This overnight culture was used at a ratio of 1:100 to inoculate the expression culture. After the OD600 of the culture reached 0.4-0.6, , we induced protein expression with 1 mM IPTG and the expression was kept for another 4 hours. In parallel, uninduced cultures were used as controls. Finally, the protein expression test was performed using the Agilent Protein 80 kit and instrument.
Figure 3. Expression Test Results. Left: Induced BL21 cells harboring our tyrosinase plasmid successfully expressed tyrosinase, while the uninduced negative control did not. By contrast, induced cells harboring our Ace1 plasmid did not express Ace1. Right: Likewise, the induced cells successfully expressed plastocyanin while Ace1 again was not expressed.
As shown in Figure 3, the results of our expression tests revealed that tyrosinase and its homologues as well as plastocyanin were expressed well. After testing Ace1 twice and observing no successful expression, we hypothesized that Ace1 does not express well because its natural function is to be a "copper regulator".
Error Prone PCR
Another portion of our project revolved around error-prone PCR. Our goal was to mutate the genes for our six starting proteins so that we could search for one that might possibly bind copper for the CuAAC reaction.
The error prone PCR (EP-PCR) reaction is very similar to the standard PCR reaction, except we used elevated concentrations of MgCl2 with added MnCl2. We also altered the concentration ratios of the four dNTPs, depriving the reaction mixture of dATP and dGTP, to drive the nucleotide assembly toward mismatched pairing. After running EP-PCR reactions in the thermocycler for tyrosinase, its homologs, Ace1, and CusF, we performed gel purification to collect the mutated products.
We then ligated the linear EP-PCR products into TOPO high copy cloning vectors and transformed these plasmids into DH5a cells for plating. Colonies formed overnight, and they each contained plasmids with different mutations of the original genes. We selected some colonies from each plate, inoculated them into LB for overnight growth, and purified them through miniprep the next day. These purified plasmids were sent for sequencing (Figure 4) and the error rate was determined to be 0.2% - 0.5%.
Figure 4. Gene sequence alignment. Samples of wild-type (WT) plastocyanin, along with mutated genes (M1, M2, M3, M4) resulting from error-prone PCR were sequenced and aligned. Nucleotide assembly errors are highlighted.
