Team:Valencia UPV/Part Collection
Team
Team
Attributions
Collaborations
Gallery
Project
Description
Design
Experiments
Inter Lab
Safety
Improve
Demonstrate
Modeling
Hardware
Software
Graphic Design
Parts
Parts
Part Collection
Results
Results
Demonstrate
Improve
Human
Human Practices
Integrated Human
Public Engagement
Medals
Team
Team
Attributions
Collaborations
Gallery
Project
Description
Design
Experiments
Model
Hardware
Software
Inter Lab
Safety
Graphic Design
Results
Results
Improve
Demonstrate
Parts
Parts
Part Collection
Human
Human Practices
Integrated Human
Public Engagement
Awards
Medals
Awards
PRESS & CONTACT
Stack Multipurpose HTML Template
Index
Introduction
Basic Parts
Composite Parts
Plasmids
Introduction
Assembly methods using type IIS restriction enzymes (based on the Golden Gate technology) have a great advantage over those using type II: the reaction is performed in a single step, without the need for gel band purification, therefore increasing  the efficiency of the assembly and reducing the time consumed. In addition, the Golden Gate technology allows to assemble more than two pieces and the backbone in a single reaction. These characteristics allows the automation of the Golden Gate assembly method, what suits Printeria perfectly.
Despite these advantages of the Golden Gate technology, finding collections of parts in this standard optimized for the E. coli chassis and well characterized is very complicated. This is the reason why we decided to create our collection of basic parts, which is based on some of the most used parts in E. coli from the Registry of Standard Biological Parts.
To create it we used the GoldenBraid 3.0 standard , so they can be used both with the BioBrick and Golden Gate assembly methods . In addition, to give added value to this collection, we have made an exhaustive characterization of these parts.
Thus, we have included different types of basic parts so that, by combining them, we can create composite parts (transcriptional units) with different behaviors: constitutive or inducible, dependant of another genetic construction, single parts of genetic circuits such as oscillators, etc.
Basic Parts
All our basic parts are inserted in the plasmid BBa_P10500 and are compatible with BioBricks grammar, as they do not contain any illegal sites for RFC10. Moreover, they do not have sites for the type IIS restriction enzymes BsaI and BsmBI, used in the Golden Braid assembly. For information on how these parts have been designed, see our design page .
Promoters
With respect to promoters, we include three constitutive promoters of different stregths, a strong promoter for T7 phage DNA polymerase, and a promoter for the sigma 32 subunit of E. coli DNA polymerase.
In addition, we have included three promoters regulated by a transcription factor: the inducible pBad, positively regulated by AraC in the presence of L-arabinose, and the inducible and repressible lux promoters, positively and negatively regulated by LuxR in the presence of Acyl-homoserine-lactone, respectively.
Table 1. Our promoter Collection.
Part
Original BioBrick
Description
BBa_K2656005
BBa_J23102
Constitutive promoter (relative strength 1)
BBa_K2656004
BBa_J23106
Constitutive promoter (relative strength 0.26)
BBa_K2656007
BBa_J23101
Constitutive promoter (relative strength 0.94)
BBa_K2656000
BBa_I719005
Strong promoter for T7 phage DNA polymerase
BBa_K2656001
BBa_K338001
Promoter for the sigma 32 subunit of E. coli DNA polymerase
BBa_K2656006
BBa_K2442101
pBad minimal promoter: positively regulated by AraC in the presence of L-arabinose
BBa_K2656002
BBa_R0061
Lux repressible promoter:  negatively regulated by LuxR in the presence of Acyl-homoserine-lactone.
BBa_K2656003
BBa_R0062
Lux inducible promoter:  positively regulated by LuxR in the presence of Acyl-homoserine-lactone.
RBS
Regarding the RBS, we have selected five from the Registry of Standard Biological Parts, with different strengths.
Table 2. Our RBS Collection.
Part
Original BioBrick
Description
BBa_K2656009
BBa_B0030
Strong RBS (relative strength 1)
BBa_K2656011
BBa_B0034
Medium RBS (relative strength 0.38)
BBa_K2656010
BBa_B0032
Weak RBS (relative strength 0.047)
BBa_K2656008
BBa_J61100
Very weak RBS (relative strength 0.042)
BBa_K2656012
BBa_J61101
Very weak RBS (relative strength 0.032)
CDS
As E. coli is our bacterial chassis, all these coding sequences have been codon optimized for this chassis organism.
We have chosen different coding sequences of pigmented proteins, both fluoroproteins (sfGFP, mRFP1, YFP and GFPmut3b) and chromoproteins (amilCP), as they are completely useful to design genetic constructions where there are necessary protein reporters of gene expression.
Moreover, another choice we have made is the addition of the LVA degradation tag to the mRFP1, YFP and GFPmut3b sequences, respectively. The LVA tag causes an increase in protease activity and, therefore, a faster degradation of the reporter protein in the cell. Thus, comparison between protein expression with and without the tag is a useful information to integrate in complex genetic oscillators design.
On the other hand, we have also selected the BSMT1 enzyme coding sequence, which will allow our bacteria to smell like mint in the presence of salicylic acid, to demonstrate odor gene expression.
Additionally, this collection also includes the transcriptional factors AraC and LuxR, that control the regulable promoters previously explained, as well as the CDS of LuxI, that acts producing the Acyl-Homoserine-Lactone to which LuxR responds. For this last CDS, we had to codon optimize its sequence for E. coli.
Finally, we have picked a lysis gene from the enterobacteria phage phiX174, which allows us to create genetic circuits which functionality implies the death of part of the population.
Thus, all these coding sequences are a toolkit of useful basic pieces to supply the user with the necessary pieces to construct the basis of common bacterial genetic circuits.
Table 3. Our CDS Collection.
Part
Original BioBrick
Description
BBa_K2656013
BBa_I746916
Superfolder Green fluorescent Protein (sfGFP) coding sequence
BBa_K2656014
BBa_E1010
monomeric Red fluorescent protein (mRFP1) coding sequence
BBa_K2656021
BBa_K592101
Yellow fluorescent protein (YFP) coding sequence
BBa_K2656022
BBa_E0040
Green fluorescent protein (GFPmut3b) coding sequence
BBa_K2656018
BBa_K592009
amilCP (Blue chromoprotein) coding sequence
BBa_K2656024
BBa_K1399001
mRFP1 with the LVA tag coding sequence
BBa_K2656020
None
YFP with the LVA tag coding sequence
BBa_K2656023
BBa_K1399004
GFPmut3b with the LVA tag coding sequence
BBa_K2656025
BBa_J45004
BSMT1 coding sequence
BBa_K2656017
BBa_K2442103
AraC transcription factor coding sequence
BBa_K2656016
BBa_C0062
LuxR transcription factor coding sequence
BBa_K2656019
BBa_C0161
LuxI (Acyl-Homoserine-Lactone synthase) codon optimized coding sequence
BBa_K2656015
None
Lysis gene from the enterobacteria phage phiX174
Transcriptional terminators
Lastly, as far as terminators are concerned, we have only chosen one, as we do not think that introducing several would have an influence on the functioning of Printeria. The terminator chosen is B0015 because it is the most used in E. Coli.
Table 4. Our Transcriptional Terminator.
Part
Original BioBrick
Description
BBa_K2656026
BBa_B0015
Double transcriptional terminator
Composite Parts
In order to make the necessary measurements to characterize the basic parts and demonstrate that they are functional, we have built some composite parts by combining a promoter, an RBS, a CDS and the terminator in a GoldenBraid Alpha 1 vector with the Golden Gate assembly protocol .
Printone
With Printeria, one of our main goals is to promote the Microbial Art among society. Thus, we decided to provide the bioartists with a complete ART DNA toolkit for its use with Printeria. As they use living cells as primary material for their artworks, we wanted to produce an extense palette: Printone.
Printone is composed of a colection of transcriptional units assembled with each of our fluorescent and coloured basic parts. By changing the promoters and RBS strengths, a wide range of different tonalities were obtained. Moreover, the LVA tag allowed us to reduce the brightness and so include degraded tones.
Image 1. Our Printeria colors: Some of the Printone transcriptional units, with variable expression levels of mRFP, YFP and amilCP proteins
Table 5. Our  BioArt DNA toolkit palette: Printone.
Transcriptional unit
Intensity
CDS
BBa_K2656105
Medium: Low promoter, strong RBS
GFPmut3b
BBa_K2656106
Medium: low promoter, strong RBS
GFPmut3b
BBa_K2656107
Strong: Strong promoter, strong RBS
GFPmut3b
BBa_K2656117
Low: Low promoter, medium RBS
GFPmut3b
BBa_K2656108
Degradation tag: less colour intensity
GFPmut3b+LVA tag
BBa_K2656101
Medium: Low promoter, strong RBS
sfGFP
BBa_K2656100
Low: weak promoter, very weak RBS
sfGFP
BBa_K2656102
Low: weak promoter, weak RBS
sfGFP
BBa_K2656103
Medium: weak promoter, medium RBS
sfGFP
BBa_K2656104
Very low: weak promoter, very weak RBS
sfGFP
BBa_K2656109
Medium: Low promoter, strong RBS
mRFP1
BBa_K2656118
Low: Low promoter, weak RBS
mRFP1
BBa_K2656119
Medium: Low promoter, medium RBS
mRFP1
BBa_K2656110
Degradation tag: less colour intensity
mRFP1+LVA tag
BBa_K2656113
Medium: Low promoter, strong RBS
amilCP
BBa_K2656120
Medium: Low promoter, medium RBS
amilCP
BBa_K2656112
Medium: Low promoter, strong RBS
YFP
BBa_K2656111
Degradation tag: less colour intensity
YFP+LVA tag
Transcriptional units: complete table
This DNA Part Collection also includes non- constitutive transcriptional units and TU assemblies related with genetic circuits regulation, such as the HSP-GFPmut3b transcriptional unit, pLuxR-GFPmut3b transcriptional unit, LuxI constitutive transcriptional unit or AraC constitutive transcriptional unit.
Table 6. Our composite part complete collection
Composite Part
Promoter
RBS
CDS
Terminator
Description
BBa_K2656100
BBa_K2656004
BBa_K2656008
BBa_K2656013
BBa_K2656026
sfGFP transcriptional unit 1
BBa_K2656101
BBa_K2656004
BBa_K2656009
BBa_K2656013
BBa_K2656026
sfGFP transcriptional unit 2
BBa_K2656102
BBa_K2656004
BBa_K2656010
BBa_K2656013
BBa_K2656026
sfGFP transcriptional unit 3
BBa_K2656103
BBa_K2656004
BBa_K2656011
BBa_K2656013
BBa_K2656026
sfGFP transcriptional unit 4
BBa_K2656104
BBa_K2656004
BBa_K2656012
BBa_K2656013
BBa_K2656026
sfGFP transcriptional unit 5
BBa_K2656105
BBa_K2656004
BBa_K2656009
BBa_K2656022
BBa_K2656026
GFP transcriptional unit 1
BBa_K2656106
BBa_K2656005
BBa_K2656009
BBa_K2656022
BBa_K2656026
GFP transcriptional unit 2
BBa_K2656107
BBa_K2656007
BBa_K2656009
BBa_K2656022
BBa_K2656026
GFP transcriptional unit 3
BBa_K2656108
BBa_K2656004
BBa_K2656009
Bba_K2656023
BBa_K2656026
GFP_LVA transcriptional unit
BBa_K2656109
BBa_K2656004
BBa_K2656009
Bba_K2656014
BBa_K2656026
mRFP1 transcriptional unit 1
BBa_K2656110
BBa_K2656004
BBa_K2656009
Bba_K2656024
BBa_K2656026
mRFP1_LVA transcriptional unit 1
BBa_K2656111
BBa_K2656004
BBa_K2656009
BBa_K2656020
BBa_K2656026
YFP_LVA transcriptional unit 1
BBa_K2656112
BBa_K2656004
BBa_K2656009
BBa_K2656021
BBa_K2656026
YFP transcriptional unit 1
BBa_K2656113
BBa_K2656004
BBa_K2656009
BBa_K2656018
BBa_K2656026
amilCP transcriptional unit 1
BBa_K2656114
BBa_K2656004
BBa_K2656009
BBa_K2656016
BBa_K2656026
Constitutive LuxR transcriptional unit
BBa_K2656115
BBa_K2656001
BBa_K2656009
BBa_K2656022
BBa_K2656026
HSP-GFPmut3b transcriptional unit
BBa_K2656116
BBa_K2656002
BBa_K2656009
BBa_K2656022
BBa_K2656026
pLuxR-GFPmut3b transcriptional unit
BBa_K2656117
BBa_K2656004
BBa_K2656011
BBa_K2656022
BBa_K2656026
GFPmut3b transcriptional unit 4
BBa_K2656118
BBa_K2656004
BBa_K2656010
BBa_K2656014
BBa_K2656026
mRFP transcriptional unit 2
BBa_K2656119
BBa_K2656004
BBa_K2656011
BBa_K2656014
BBa_K2656026
mRFP transcriptional unit 3
BBa_K2656120
BBa_K2656004
BBa_K2656011
BBa_K2656018
BBa_K2656026
amilCP transcriptional unit 2
BBa_K2656121
BBa_K2656004
BBa_K2656011
BBa_K2656017
BBa_K2656026
AraC constitutive transcriptional unit
BBa_K2656122
BBa_K2656003
BBa_K2656011
BBa_K2656022
BBa_K2656026
pLux-GFPmut3b transcriptional unit
BBa_K2656123
BBa_K2656000
BBa_K2656010
BBa_K2656022
BBa_K2656026
pT7-GFPmut3b transcriptional unit
BBa_K2656124
BBa_K2656000
BBa_K2656011
BBa_K2656022
BBa_K2656026
pT7-GFPmut3b transcriptional unit 2
BBa_K2656125
BBa_K2656004
BBa_K2656009
BBa_K2656019
BBa_K2656026
LuxI constitutive transcriptional unit 1
BBa_K2656126
BBa_K2656004
BBa_K2656009
BBa_K2656025
BBa_K2656026
BMSTI transcriptional unit
Plasmids
Our transcriptional units were built using a GoldenBraid 3.0 alpha 1 vector which is not compatible with the BioBrick grammar. Therefore, we have created the BBa_K2656200 plasmid, adapted from the standard pSB1C3 backbone, to convert our GoldenBraid transcriptional units to BioBricks.
BBa_K2656200 is a modified version of BBa_P10500 , so it includes BsmBI restriction sites flanking the selection marker. When digesting this plasmid with BsmBI, the result is complementary sticky ends to those formed in a GoldenBraid alpha 1 plasmid when digesting with the same enzyme. By this way, the BioBrick compatibility for each composite part is easily achieved using this plasmid as the destination vector in a Golden Gate BsmBI one-pot reaction.
Screening marker
Using the BBa_P1050 plasmid, selection of positive transformants relies on the white-blue screening method, which is based on the α-complementation phenomena. To do so, the host strain must carry the ω-peptide, while the vector contains the α-peptide. If both subunits are expressed (when there is not insertion in the  vector), functional beta-galactosidase is reconstituted. Thus, blue-white screening is carry out by the addition of X-gal (D-lactose analog) and IPTG into the agar plates. In this way, if there is no insertion, colonies will be blue as a consequence of the X-gal hidrolysis. Although the principle is simple, this method requires the use of lacZ mutant strains and it is always necessary to add both reagents into the medium.
In order to avoid these limitations, we have decided to rely on a different visual method. Thus, we have inserted an mRFP1 transcriptional unit (BBa_K2656201) to act as the screening gene marker. By this way, cloning is achieved in a more reliable, efficient, low-cost way, so avoiding the necessity of adding chemical supplements and the possible non-function of these substances once on the solid medium. With this method, non-positive transformants will always express an intense red fluorescent protein, while non-recombinant transformants will not be able to express this protein.
Table 7. Our designed plasmids
Part
Description
BBa_K2656200
BioBricks compatible plasmid
CONTACT US igem.upv.2018@gmail.com
FOLLOW US @upvigem
CONTACT US igem.upv.2018@gmail.com
CONTACT US igem.upv.2018@gmail.com
