<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_Tel-Hai_Methodolodogy skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><DIV id="top_title"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:Tel-Hai/Methodolodogy</SPAN></H1></DIV><DIV id="HQ_page"><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><DIV id="pageContainer"><HEADER><NAV><UL id="menu-nav"><LI><A href="https://2016.igem.org/Team:Tel-Hai">Home</A></LI><LI><A href="#">Team</A></LI><LI><A href="https://2016.igem.org/Team:Tel-Hai/AboutUs">About Us</A></LI><LI><A href="https://2016.igem.org/Team:Tel-Hai/Attributions">Attributions</A></LI></UL><LI><A href="#">Project</A></LI><LI><A href="https://2016.igem.org/Team:Tel-Hai/Description">Description</A></LI><LI><A href="https://2016.igem.org/Team:Tel-Hai/Design">Applied Design</A></LI><LI><A href="https://2016.igem.org/Team:Tel-Hai/Proof">Proof of Concept</A></LI><LI><A href="https://2016.igem.org/Team:Tel-Hai/Achievements">Achievements</A></LI><LI><A href="https://2016.igem.org/Team:Tel-Hai/Entrepreneurship">Entrepreneurship</A></LI><LI><A href="#">Human Practice</A></LI><LI><A href="https://2016.igem.org/Team:Tel-Hai/Integrated_Practice">Human Practice</A></LI><LI><A href="https://2016.igem.org/Team:Tel-Hai/HP/Silver">HP Silver &amp; Gold</A></LI><LI><A href="https://2016.igem.org/Team:Tel-Hai/Engagement">Public Engagement</A></LI><LI><A href="https://2016.igem.org/Team:Tel-Hai/Collaborations">Collaborations</A></LI><LI><A href="https://2016.igem.org/Team:Tel-Hai/Ethics">Ethics</A></LI><LI><A href="#">Lab Work</A></LI><LI><A href="https://2016.igem.org/Team:Tel-Hai/Notebook">Notebook</A></LI><LI><A href="https://2016.igem.org/Team:Tel-Hai/Methodolodogy">Methodolodogy</A></LI><LI><A href="https://2016.igem.org/Team:Tel-Hai/Protocols">Protocols</A></LI><LI><A href="https://2016.igem.org/Team:Tel-Hai/Parts">Our Parts</A></LI></NAV><H2><SPAN>i</SPAN>GEM Tel-Hai 2016</H2></HEADER><H3>Materials &amp; Methodology</H3><H4>Bacterial strains and mammalian cell lines</H4><H5>Escherichia coli TOP10</H5>
   The bacterial strain we used as a base for all our transformations was TOP10
   <H5>Lung epithelial cells NCI-H 1650</H5>
   We used this cell line as a model line for epithelial lung cells
   <DIV style="display: flex;"><DIV style="width: 50%;"><H4>Buffers and medium</H4><UL><LI>LB 1L</LI><LI>10g Bacto-tryptone</LI><LI>5g yeast extract</LI><LI>CutSmart</LI><LI>RPMI</LI><LI>TAE</LI></UL><UL><LI><SPAN style="text-decoration: underline;">CCMB</SPAN></LI></UL><H4>Kits</H4><UL><LI>Miniprep kit</LI><LI>Maxiprep kit</LI></UL></DIV><DIV style="width: 50%;"><H4>Chemicals, Protein &amp; Enzymes</H4><UL><LI>Ladder</LI><LI>Hoescht</LI><LI>Chloramphenicol</LI><LI>EcoRI-HF and XbaI</LI><LI>agar</LI><LI>Cholera toxin subunit B (CTB) protein (SIGMA-ALDRICH)</LI><LI>FITC</LI><LI>FBS</LI><LI>PEN STREP</LI><LI>DMSO</LI><LI>Trypsin</LI><LI>Trypan blue</LI><LI>T4 ligase</LI><LI>Agarose</LI><LI>EtBr</LI><LI>BSA</LI></UL><H4>Machines</H4><UL><LI>FACS</LI><LI>PCR</LI><LI>Confocal microscope</LI><LI>Fume hood</LI><LI>Microbiological hood</LI><LI>nanodrop</LI></UL></DIV></DIV><H4>Methodology</H4><P>The bacterial strain we used as a base for all our transformations was TOP10 E.Coli strain by heat shock method. E. Coli were cultured on LB agar plates with the addition of antibiotic chloramphenicol and were incubated overnight at 37°C.</P><P>We used 5 parts: plasmid with GFP gene under mammalian promoter, two gRNA for CFTR ΔF508, one of donor part – CFTR exon 11, and the last is LTBD gene. The first four parts were received from IDT.</P><P>For the extraction of the plasmids we used a miniprep kit. for restriction and ligation we used EcoR1 and Pst1.</P><P> The pure cholera toxin subunit B protein was received by SIGMA-ALDRICH.</P><P>We performed two Methods to link the cholera toxin and plasmid:
   <OL><LI><STRONG>Chemical Binding</STRONG></LI></OL></P><P>The principle of chemical binding is that Residues of certain amino acids contain carboxylic acid that can be found in the toxin. For the first step, a chemical linker 3-amino-1,2 -propanediol is attached to the carboxylic acid by nucleophilic attack reaction. Then, the toxin is oxidized and mixed with the plasmid to form an imine bond with the amine group that can be found in the plasmid's nucleotides.</P><LI><STRONG>DNA Binding Domain</STRONG><P>A DNA-binding domain (DBD) is an independently folded protein domain that contains at least one structural motif that recognizes double or single-stranded DNA. A DBD can recognize a specific DNA sequence (a recognition sequence) or have a general affinity to DNA. Some DNA-binding domains may also include nucleic acids in their folded structure.</P></LI><P>To check the success of the plasmid binding to the protein, we performed a series of additional experiments.
      For checking the chemical binding method - we ran the complex (DNA+ protein) from the chemical method in agarose gel 3%, along with two negative controls (DNA alone, protein alone). We have seen at the results a gel shift, and deduced that the connection indeed succeeded.
   </P><P>To test the binding of the protein to the plasmid DNA binding domain method, we conducted an experiment where we ran in every well fixed DNA concentration (400 ng), and the protein at different concentration (first well 400ng of DNA, 30ng of protein. Second well 400ng of DNA, 60ng of protein, Etc.)  Indeed, we added more protein,the DNA band became narrower, and more protein bound to the DNA, therefore there is less free DNA to ran throw the gel..  We could see that in the concentration of 300ng protein, DNA had stopped running. This result proved to us that the connection between the plasmid and protein succedded, and thanks to this, we also learned which maximum protein concentration could be connected to DNA.</P><P>After we've been able to show that DNA binds to the protein, we tested the ability of the complex to enter into cells. We chose to continue working with the binding-domain because it is easier to perform and we received more accurate results.</P><P> We grew epithelial cells of the lung (NCI-H1650) until they reached a large enough quantity. Whe cells were incubated at 37 ° C and 5% CO2
      .
      When we had enough cells, we selected two methods to check whether the complex enters the cell or not- with the FACS, and with the Confocal microscope. We dyed the protein with FITC and the DNA with Hoechst.
   <OL><LI><STRONG>FACS:</STRONG></LI></OL></P><P>Fluorescence-activated cell sorting (FACS) is a specialized type of flow cytometry. It provides a method for sorting a heterogeneous mixture of biological cells into two or more containers, one cell at a time, based upon the specific light scattering and fluorescent characteristics of each cell.</P><P>After we dyed the complex (DNA with Hoechst protein with FITC) and incubated with NCI-H1650 cells, we looked at the results. The output could be concluded that the protein binds to cells, but we did not see the presence of the DNA with the cells.</P><LI><STRONG>Confocal microscope</STRONG><P>Confocal microscope is an optical imaging technique for increasing optical resolution and contrast of a micrograph by means of adding a spatial pinholeplaced at the confocal plane of the lens to eliminate out-of-focus light. It enables the reconstruction of three-dimensional structures from the obtained images by collecting sets of images at different depths (a process known as optical sectioning) within a thick object. This technique has gained popularity in the scientific and industrial communities and typical applications are in life sciences, semiconductor inspection and materials science.</P><P>We incubated the cells with the complex for 6 hours and we reviewed the results. We could see that the cells were painted in green which means that the protein was recocnized and entered the cells. However, we couldn’t see the DNA. We plan to conduct this experiment again to see the DNA in the nucleus. important to note that another group that we worked with conducted the same experiment under the same conditions and did in fact see the DNA in the nucleus. Therefore, we know that it does work.</P></LI></DIV></DIV></DIV></DIV></DIV></DIV></BODY></HTML>