<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_CUNY_Kingsborough_Light_Operon skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><DIV id="top_title"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:CUNY Kingsborough/Light Operon</SPAN></H1></DIV><DIV id="HQ_page"><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><DIV class="dropdown"><BUTTON class="dropbtn w3-round-xxlarge">Project</BUTTON><DIV class="dropdown-content dropdown-menu-center"><A href="https://2018.igem.org/Team:CUNY_Kingsborough/Description">Background</A><A href="https://2018.igem.org/Team:CUNY_Kingsborough/Design">Design</A><A href="https://2018.igem.org/Team:CUNY_Kingsborough/Experiments">Experiments</A><A href="https://2018.igem.org/Team:CUNY_Kingsborough/Results">Results</A></DIV></DIV><DIV class="dropdown"><BUTTON class="dropbtn w3-round-xxlarge">Modeling</BUTTON><DIV class="dropdown-content dropdown-menu-center"><A href="https://2018.igem.org/Team:CUNY_Kingsborough/Model">Overview</A><A href="https://2018.igem.org/Team:CUNY_Kingsborough/Modeling_101">Modeling 101</A><A href="https://2018.igem.org/Team:CUNY_Kingsborough/Light_Operon">Light Operon</A><A href="https://2018.igem.org/Team:CUNY_Kingsborough/Turing_Patterns">Turing Patterns</A><A href="https://2018.igem.org/Team:CUNY_Kingsborough/CRISPR-Cas13a">CRISPR-Cas13a</A></DIV></DIV><DIV class="dropdown"><BUTTON class="dropbtn w3-round-xxlarge">Human Practices</BUTTON><DIV class="dropdown-content"><A href="https://2018.igem.org/Team:CUNY_Kingsborough/Human_Practices">Human Practices</A><A href="https://2018.igem.org/Team:CUNY_Kingsborough/Public_Engagement">Public Engagement</A></DIV></DIV><DIV class="dropdown"><BUTTON class="dropbtn w3-round-xxlarge">Team</BUTTON><DIV class="dropdown-content dropdown-menu-center"><A href="https://2018.igem.org/Team:CUNY_Kingsborough/Team">Team Members</A><A href="https://2018.igem.org/Team:CUNY_Kingsborough/Collaborations">Collaborations</A><A href="https://2018.igem.org/Team:CUNY_Kingsborough/Attributions">Attributions</A></DIV></DIV><P><BUTTON onclick="topFunction()" id="scrollToTop" title="Go to top">Top</BUTTON></P><H1 class="title-padding">Light Operon</H1><H2 class="default-padding">1. Introduction</H2><P class="low-rise-padding">A deterministic model of differential equations over a continuous time interval is generally easy to implement using known numerical methods. However, the equations alone do not account for the variability in expression that we often observe at low molecular count. In other words, deterministic simulations fail to capture the actual physical basis of the reaction (Gillespie 1977; Wilkinson 2012). This is due to the intrinsic and extrinsic noise that occurs at low intracellular molecular counts; interactions do not follow traditionally defined constant parameters which limits the accurate characterization of a genetic system (Elowitz et al., 2002).</P><P class="low-rise-padding">It then becomes essential when characterizing a genetic system to have a deterministic approach (to capture the large average overtime) in addition to a stochastic approach. A stochastic approach allows one to model the probabilistic trajectory based on the initial conditions. One commonly used stochastic algorithm is the Gillespie Algorithm— a simple but powerful approach to simulation that takes into account the initial state of the reactants, the reaction rate and the number of molecules present for a reaction with each timed step of the reaction drawn from a probability distribution (Gillespie 1977).</P><P class="low-rise-padding">As a working example, we will be working with the pDawn and pDusk operon (BBa K161609 and BBa K1075044). Many teams have made either the pDawn/pDusk system as their inducer (IONIS Paris 2015, Wageningen 2016, NUS Singapore 2017, Cornell 2017, Kingsborough 2017). In 2017, NUS Singapore 2017 and Kingsborough 2017 used the pDawn/pDusk inducer to induce expression of <I>MazF</I> to induce cell death. To our knowledge, there is no literature or past iGEM project exploring a stochastic model of the pDawn/pDusk system at low molecule count.</P><H2 class="default-padding">2. Methods</H2><P class="low-rise-padding">In order to improve reproducibility and to build on past iGEM teams’ efforts, we used the differential equations and parameters described in the <A id="bodyLink" href="https://static.igem.org/mediawiki/2016/c/c2/T--Wageningen_UR--Detailed_Methods_Kill_Switch.pdf" target="_blank">2016 Wageningen iGEM Team’s kill switch design</A>. Using the “smfsb” package in R for stochastic simulations and the NDSolve function in Mathematica, we modeled the 3 light-sensitive states of the Yf1 homodimer over a period of 20 hours. Yll frequency simulation was done through the &quot;smfsb&quot; package in R and simulated a hundred times. Histograms were made through Excel.</P><H3 class="low-rise-padding" style="color: black;"><U>Equations, Constants, &amp; Parameters</U></H3><DIV class="no-rise-padding"><OL><LI>\(\frac{dy_{DD}}{dt}=k_1+ 2\cdot k_2 \cdot y_{DL,LD} - 2 \cdot (N\cdot k_3) \cdot y_{DD} - \beta_1 \cdot y_{DD}\)</LI><LI>\(\frac{dy_{DL,LD}}{dt}=2 \cdot (N \cdot k_3) \cdot y_{DD} + 2 \cdot k_2 \cdot y_{LL} - 2 \cdot k_2 \cdot y_{DL,LD} - 2 \cdot (N \cdot k_3) \cdot y_{DL,LD} - \beta_2 \cdot y_{DL,LD}\)</LI><LI>\(\frac{dy_{LL}}{dt} = 2 \cdot (N \cdot k_3) \cdot y_{DL,LD} - 2\cdot k_2 \cdot y_{LL} - \beta_3 \cdot y_{LL}\)</LI></OL></DIV><CENTER><TABLE id="param-table"><THEAD><TR><TH><SPAN>Constant/Parameter</SPAN></TH><TH><SPAN>Value</SPAN></TH><TH><SPAN>Description</SPAN></TH></TR></THEAD><TBODY><TR><TD>\(N\)</TD><TD>Variable of \(\frac{\mu \cdot mol}{m^2 \cdot h}\)</TD><TD>Concentration of light</TD></TR><TR><TD>\(k_1\)</TD><TD>\(2.6921 \frac{\mu \cdot mol}{hr}\)</TD><TD>Production rate of \(y_{DD}\)</TD></TR><TR><TD>\(k_2\)</TD><TD>\(0.0008 \frac{1}{hr}\)</TD><TD>Relaxtion rate of \(y_{DL,LD}\) and \(y_{LL}\)</TD></TR><TR><TD>\(k_3\)</TD><TD>\(0.4219 \frac{m^2}{\mu \cdot mol}\)</TD><TD>Conversion cross-section of light intensity activated production rate of \(y_{DL,LD}\) and \(y_{LL}\).</TD></TR><TR><TD>\(\beta_1\)</TD><TD>\(0.3049 \frac{1}{hr}\)</TD><TD>Degradation rate of \(y_{DD}\)</TD></TR><TR><TD>\(\beta_2\)</TD><TD>\(0.8406 \frac{1}{hr}\)</TD><TD>Degradation rate of \(y_{DL,LD}\)</TD></TR><TR><TD>\(\beta_3\)</TD><TD>\(0.1477 \frac{1}{hr}\)</TD><TD>Degradation rate of \(y_{DD}\)</TD></TR></TBODY></TABLE></CENTER><H3 class="low-rise-padding" style="color: black;"><U>Post-Reaction Pre-Reaction Matrix for Gillespie</U></H3><H2 class="default-padding">3. Results</H2><CENTER><P class="low-rise-padding"><I>Click on any of the images to view full size.</I></P></CENTER><P class="no-rise-padding">In the stochastic results, we observe a much greater variation in expression at 95 \(\mu mol\) compared to 948 \(\mu mol\). Contrast this to the deterministic model in which the low/high concentration plots are identical with respect to a scaling factor. Gillespie causes step-behavior in the concentration. Although the plot of  948 \(\mu mol\) appears to be much smoother, we would see the same “cliffs” that we observe at 95 \(\mu mol\) by stretching the y-axis enough. The impact of a fixed gain or loss in concentration is much more “felt” in a small system than it would be in a large system (here size is relative to the initial concentration of Yf1). If we only cared about modeling a large, stable concentration however, the deterministic model clearly suffices to capture the general behavior and is less computationally expensive.</P><OL><H3 class="default-padding" style="color: black">1. <U>Starting concentration of 95 Yf1 molecules (Gillespie)</U></H3><CENTER><DIV><SPAN style="color:red;">yDD (Dark-Dark state); </SPAN><SPAN style="color:blue;">yDL/LD (Dark-Light state); </SPAN><SPAN style="color:green;">yLL (Light-Light state)</SPAN></DIV></CENTER><H2 class="default-padding"> Figures</H2><P class="no-rise-padding">Histograms of the frequency of Yll given a starting concentration of Ydd after a simulation of 100 times. Light is produced at a constant amount-(1ɥmol and 10 ɥmol). Frequency distribution is obtained around the time of full activation.</P><H3 class="default-padding" style="color: black">2. <U>Starting concentration of 948 Yf1 molecules (Gillespie)</U></H3><CENTER><SPAN style="color:red;">yDD (Dark-Dark state); </SPAN><SPAN style="color:blue;">yDL/LD (Dark-Light state); </SPAN><SPAN style="color:green;">yLL (Light-Light state)</SPAN></CENTER><H3 class="default-padding" style="color: black">3. <U>Starting concentration of 95 and 948 Yf1 molecules (deterministic)</U></H3><CENTER><SPAN style="">(Left) Starting concentration of 95 Yf1 molecules; (Right) Starting concentration of 948 Yf1 molecules</SPAN><SPAN style="color:blue;">yDD (Dark-Dark state); </SPAN><SPAN style="color:orange;">yDL/LD (Dark-Light state); </SPAN><SPAN style="color:green;">yLL (Light-Light state)</SPAN></CENTER><H3 class="default-padding" style="color: black">5. <U>Frequency of Yll at Starting Concentration of 948 Ydd</U></H3><CENTER><SPAN style="">(Left) Constant light at 1 mumol; (Right) Constant light at 10 mumol</SPAN></CENTER><H3 class="default-padding" style="color: black">6. <U>Frequency of Yll at Starting Concentration of 95 Ydd</U></H3><CENTER><SPAN style="">(Left) Constant light at 1 mumol; (Right) Constant light at 10 mumol</SPAN></CENTER><H3 class="default-padding">Citations</H3><P class="no-rise-padding">
Elowitz, M. B., Levine, A. J., Siggia, E. D. &amp; Swain, P. S. (2002), ‘Stochastic gene expression in a single cell’, Science 297(5584), 1183–1186.
(Wilkinson 324-325)

Gillespie, D. T. (1977), ‘Exact stochastic simulation of coupled chemical reactions’, Journal of Physical Chemistry 81, 2340–2361.

Möglich, Andreas, Rebecca A. Ayers, and Keith Moffat. 2009. “Design and Signaling Mechanism of Light-Regulated Histidine Kinases.&quot; Journal of Molecular Biology 385(5):1433–44. Retrieved (http://dx.doi.org/10.1016/j.jmb.2008.12.017).

Ohlendorf, Robert, Roee R. Vidavski, Avigdor Eldar, Keith Moffat, and Andreas Möglich. 2012. “From Dusk till Dawn: One-Plasmid Systems for Light-Regulated Gene Expression.&quot; Journal of Molecular Biology 416(4):534–42. Retrieved (http://dx.doi.org/10.1016/j.jmb.2012.01.001) 

Wilkinson, D. J. (2011) Stochastic modelling for systems biology, second edition, Boca Raton, Florida: Chapman and Hall/CRC Press.

https://2016.igem.org/Team:Wageningen_UR/Description&gt;</P></OL></DIV></DIV></DIV></DIV></DIV></BODY></HTML>