<HTML xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en" dir="ltr">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki  ltr ns-0 ns-subject page-Team_ETHZ_Basel_Biology_Cloning"><DIV id="globalWrapper"><DIV id="top-section"><DIV id="p-logo"><A href="/Main_Page" title="Main Page">&quot;
	    </A></DIV><DIV id="menubar" class="left-menu noprint"><UL><LI class="selected"><A href="/Team:ETHZ_Basel/Biology/Cloning">Page               </A></LI><LI class="new"><A href="/wiki/index.php?title=Talk:Team:ETHZ_Basel/Biology/Cloning&amp;action=edit&amp;redlink=1">Discussion               </A></LI><LI><A href="/wiki/index.php?title=Team:ETHZ_Basel/Biology/Cloning&amp;action=edit">View source               </A></LI><LI><A href="/wiki/index.php?title=Team:ETHZ_Basel/Biology/Cloning&amp;action=history">History               </A></LI><LI style="color:#808080;cursor:default">teams</LI></UL></DIV><DIV class="right-menu noprint" id="menubar"><UL><LI id="pt-login"><A href="/wiki/index.php?title=Special:UserLogin&amp;returnto=Team:ETHZ_Basel/Biology/Cloning" title="You are encouraged to log in; however, it is not mandatory [o]" accesskey="o">Log in</A></LI></UL></DIV><DIV id="search-controls" class="noprint"><FORM action="/Special:Search" id="searchform">Â </FORM></DIV></DIV><DIV id="content"><H1 class="firstHeading">Team:ETHZ Basel/Biology/Cloning</H1><DIV id="bodyContent"><H3 id="siteSub" class="noprint">From 2010.igem.org</H3><DIV id="navigation"><UL id="simple-menu"><LI><A href="https://2010.igem.org/Team:ETHZ_Basel" class="introduction"><SPAN>Introduction</SPAN></A></LI><LI><A href="https://2010.igem.org/Team:ETHZ_Basel/Biology" class="biology_current"><SPAN>Biology &amp; Wet Laboratory</SPAN></A></LI><LI><A href="https://2010.igem.org/Team:ETHZ_Basel/Modeling" class="modeling"><SPAN>Mathematical Modeling</SPAN></A></LI><LI><A href="https://2010.igem.org/Team:ETHZ_Basel/InformationProcessing" class="information"><SPAN>Information Processing</SPAN></A></LI><LI><A href="https://2010.igem.org/Team:ETHZ_Basel/Achievements" class="achievements"><SPAN>Achievements</SPAN></A></LI><LI><A href="https://2010.igem.org/Team:ETHZ_Basel/Team" class="team"><SPAN>Team</SPAN></A></LI></UL></DIV><DIV id="subnavigation" class="biology"><UL id="sub-menu"><LI><A href="https://2010.igem.org/Team:ETHZ_Basel/Biology"><SPAN>Overview</SPAN></A></LI><LI><A href="https://2010.igem.org/Team:ETHZ_Basel/Biology/Molecular_Mechanism"><SPAN>Molecular Mechanism</SPAN></A></LI><LI><A href="https://2010.igem.org/Team:ETHZ_Basel/Biology/Archeal_Light_Receptor"><SPAN>Archeal Light Receptor</SPAN></A></LI><LI><A href="https://2010.igem.org/Team:ETHZ_Basel/Biology/Cloning"><SPAN>Cloning Strategy</SPAN></A></LI><LI><A href="https://2010.igem.org/Team:ETHZ_Basel/Biology/Implementation"><SPAN>Implementation</SPAN></A></LI><LI><A href="https://2010.igem.org/Team:ETHZ_Basel/Biology/Safety"><SPAN>Safety</SPAN></A></LI></UL></DIV><H1><SPAN class="mw-headline" id="Cloning_strategy_for_the_construction_of_our_Biobricks"> Cloning strategy for the construction of our Biobricks </SPAN></H1><DIV class="thumb tright"><DIV class="thumbinner" style="width:602px;"><DIV class="thumbcaption"><B>Schematic overview of the planned constructs.</B> In the planning phase, 81 constructs were considered.</DIV></DIV></DIV><P>When starting the project it was hard to know in advance which implementation strategy would be successful: Which type of anchor would work the most efficiently? Which Che protein should be chosen for the fusion and should it be C-terminally or N-terminally linked to the LSP (Light sensitive protein)? Could even the type of linker make a difference? Could this even have an influence on the functional properties of the whole biobrick?
</P><P>After reflection and literature research, we started to construct brickboxes for each type of brick (anchor, linker, che protein) that would enable for the generation of several fusion proteins. In paralell we started the model based experimental design to find out the best implementation strategy in the end. The picture on the right graphically represents all the fusion proteins we have in our wetlab production pipeline. 
</P><P>To ensure the efficiency of the cloning process of such a huge amount of fusion proteins, we decided to apply the cloning strategy BBF RFC28: A method for combinatorial multi-part assembly based on the Type IIs restriction enzyme AarI <A href="/Team:ETHZ_Basel/Biology/Cloning#References" title="Team:ETHZ Basel/Biology/Cloning">[1]</A>. The flow chart below shows the underlying mechanism. The advantage of this strategy is that we can simultaneously clone up to 3 different inserts into one single expression vector. In the following section we give an overview of how this was achieved.
</P><DIV class="center"><DIV class="thumb tnone"><DIV class="thumbinner" style="width:942px;"><DIV class="thumbcaption"><B>Overview of the cloning strategy BBF RFC28 and the cleavage mechanism of AarI.</B> All our bricks where designed according to this format in order to make the assembly of several fusion proteins achievable.</DIV></DIV></DIV></DIV><H2><SPAN class="mw-headline" id="1._Step:_Construction_of_brickboxes_which_enable_the_generation_of_fusion_proteins"> 1. Step: Construction of brickboxes which enable the generation of fusion proteins</SPAN></H2><TABLE border="0" align="center"><TBODY><TR valign="top"><TD><DIV class="center"><DIV class="thumb tnone"><DIV class="thumbinner" style="width:402px;"><DIV class="thumbcaption"><B>Chemotaxis brickbox.</B> Bricks from each box are compatible and can be assembled to generate the desired fusion proteins.</DIV></DIV></DIV></DIV></TD><TD><DIV class="center"><DIV class="thumb tnone"><DIV class="thumbinner" style="width:402px;"><DIV class="thumbcaption"><B>Anchor and Light sensor brickboxes.</B> Bricks from each box are compatible and can be assembled to generate the desired fusion proteins.</DIV></DIV></DIV></DIV></TD></TR><TR valign="top"><TD><DIV class="center"><DIV class="thumb tnone"><DIV class="thumbinner" style="width:202px;"><DIV class="thumbcaption"><B>Linkerbox.</B> Bricks from each box are compatible and can be assembled to generate the desired fusion proteins.</DIV></DIV></DIV></DIV></TD><TD><DIV class="center"><DIV class="thumb tnone"><DIV class="thumbinner" style="width:402px;"><DIV class="thumbcaption"><B>Expression vector box.</B> Bricks from each box are compatible and can be assembled to generate the desired fusion proteins.</DIV></DIV></DIV></DIV></TD></TR></TBODY></TABLE><P>The single bricks in the corresponding boxes above were generated by PCR. For amplification we used the primers specified in the BBF RFC28 manual. The purpose of this was to make all bricks compatible with this standard. The bricks were then subcloned into the storage vector pSEVA132 (Victor de Lorenzo's lab, KanR, pBBR1 ori) by blunt end ligation. The plasmid pSEVA132 enabled us to pre-screen the transformed colonies by blue white screening. This boosted the production of the brickbox parts, as we could simply pick the &quot;right&quot; colonies for the subsequent mini-prep procedure. Finally, the generated bricks were verified by AarI digest and sequencing. 
Due to the presence of rare codons in the sequence of PhyB and Pif3, these two genes had to be codon-optimized and synthesized by GeneArt.
As the successful implementation of our E. Lemming's pathway mainly relies on two protein fusions  (the &quot;anchor-light sensor&quot; and &quot;light sensor-che protein&quot; fusion), we constructed two expression vectors (we call them working vectors) with origins of replication from different compatibility classes which enables us to simultaneously express the two fusion proteins simultaneously. Working vector 1 is a derivative of pSEVA132 conferring resistance to ampicillin and replicating with ori pBB1. Working vector 2 is a derivative of pSEVA421 expressing a spectinomycin resistance cassette and replicating with ori RK2. The gene for the repressor AraC and the corresponding ParaBAD promotor/operator sites were introduced into both vectors followed by an insert, which is flanked by AarI-recognition sites. Digest with AarI releases this insert and generates a vector with assembly-compatible overhangs.
</P><P><B>All outlined assembly-conpatible bricks and working vector 1 were sent to the registry.</B></P><H2><SPAN class="mw-headline" id="2._Step:_Assembly_of_fusion_proteins_using_the_generated_brickboxes"> 2. Step: Assembly of fusion proteins using the generated brickboxes </SPAN></H2><P>The following image illustrates the assembly of a fusion protein. In the section <A href="/Team:ETHZ_Basel/Biology/Implementation" title="Team:ETHZ Basel/Biology/Implementation"><B>Implementation</B></A> the constructed fusion proteins, required for the implementation of the synthetic network, are described. 
</P><DIV class="center"><DIV class="thumb tnone"><DIV class="thumbinner" style="width:794px;"><DIV class="thumbcaption"><B>Overview on the contstruction of fusion proteins using the brickboxes.</B>.</DIV></DIV></DIV></DIV><H2><SPAN class="mw-headline" id="Linking_BBF_RFC28_to_the_Tom_Knight.27s_original_assembly_standard_.28OAS.29"> Linking BBF RFC28 to the Tom Knight's original assembly standard (OAS) </SPAN></H2><P><B>General scheme for the design and construction of Tom Knight's OAS-compatible fusion proteins</B></P><P>BBF RFC28 is a method for combinatorial multi-part assembly based on the Type II restriction enzyme AarI and it is a highly efficient method for constructing fusion proteins. As we have a great number of constructs to generate, efficiency in the cloning process is vital for the project. Unfortunately it is not compatible with the general iGEM standardized biobricks. As a consequence, the fusion proteins are not compatible with Tom Knight's original assembly standard as they don't contain the required prefix and suffix sequences. 
</P><P>During our project we learned a lot about the cloning of fusion proteins using the BBF RFC28 assembly method and  how to make it work efficiently. We would like to share our experience with you, especially as we worked out a general design method for the facilitated construction of Tom Knight's OAS  <A href="/Team:ETHZ_Basel/Biology/Cloning#References" title="Team:ETHZ Basel/Biology/Cloning">[2]</A> and BBF RFC28 compatible working vectors (the expression vector that is finally used for the assembly) that can be used for the final assembly step. 
Our vector design includes a GFP-mediated screening system for screening clones containing the fusion protein.
</P><P>As we had already started the cloning procedure before learning about the little difficulties of the assembly method and before making up our minds on how to improve it and link it to the Tom Knight's BBF, our fusion proteins were not cloned into a Tom Knight's OAS-compatible working vector. Nevertheless, the design scheme that is outlined below is under construction. 
</P><P>In general a working vector that can be used for efficient assembly needs to contain an insert that is flanked by AarI-sites. Those cleavage sites must be designed such that AarI-digestion releases the insert and generates a vector backbone with 5` overhangs compatible with the BBF RFC28 standard. The rational behind cloning an insert into the vector instead of simply adding two AarI sites is that AarI does not work with the same high efficiency on every cleavage site. Therefore, cleaved vector can be efficiently separated from uncut vector fractions by separation on a 1% agarose gel. By choosing a constitutively expressed reporter protein as insert, positive clones containing the assembled protein can be distinguished from clones containing the original insert (traces of uncut vector might still be present in the cleaved vector fraction). Constitutive expression of the reporter is very convenient as adding an inducer to the selection plates is not necessary. This gives total freedom of choice for the fusion protein expression system. 
By designing the primers for amplification of the insert such that the insert in the end contains the Tom Knight` OAS suffix and prefix sequences plus a ribosome binding site makes any available expression vector readily usable as standardized assembly vector. the insert simply needs to be cloned into the chosen expression vector. The standard primer sequences for the amplification of the insert and the general scheme for the working vector construction are outlined below.
</P><DIV class="center"><DIV class="thumb tnone"><DIV class="thumbinner" style="width:878px;"><DIV class="thumbcaption"><B>Update of BBF RFC28:</B> New technical standard to make BBF RFC28 compatible with Tom Knight's OAS.</DIV></DIV></DIV></DIV><P>Further, to contribute to the improvement of the registry, we screened it for a reporter that might fulfill the requirements of being constitutivly expressed, so that it could be used as the above outlined insert. We found the GFP generator BBa_K082034 as potential target and characterized it. You find the <A href="http://partsregistry.org/Part:BBa_K082034:Experience" class="external text" rel="nofollow">results</A> here.
</P><H2><SPAN class="mw-headline" id="References"> References </SPAN></H2><P>[1] <A href="http://dspace.mit.edu/handle/1721.1/46721" class="external text" rel="nofollow">BBF RFC 28: A method for combinatorial multi-part assembly based on the Type IIs restriction enzyme AarI</A></P><P>[2] <A href="http://dspace.mit.edu/handle/1721.1/21168" class="external text" rel="nofollow">Idempotent Vector Design for Standard Assembly of Biobricks</A></P><DIV class="printfooter">
Retrieved from &quot;<A href="http://2010.igem.org/Team:ETHZ_Basel/Biology/Cloning">http://2010.igem.org/Team:ETHZ_Basel/Biology/Cloning</A>&quot;</DIV></DIV></DIV><DIV id="footer-box" class="noprint"><DIV id="footer"><UL id="f-list"><LI id="t-recentchanges"><A href="/Special:RecentChanges" title="Recent changes">Recent changes</A></LI><LI id="t-whatlinkshere"><A href="/Special:WhatLinksHere/Team:ETHZ_Basel/Biology/Cloning" title="List of all wiki pages that link here [j]" accesskey="j">What links here</A></LI><LI id="t-recentchangeslinked"><A href="/Special:RecentChangesLinked/Team:ETHZ_Basel/Biology/Cloning" title="Recent changes in pages linked from this page [k]" accesskey="k">Related changes</A></LI><LI id="t-specialpages"><A href="/Special:SpecialPages" title="List of all special pages [q]" accesskey="q">Special pages</A></LI><LI><A href="/Special:Preferences">My preferences</A></LI></UL></DIV><DIV id="footer"><UL id="f-list"><LI id="t-print"><A href="/wiki/index.php?title=Team:ETHZ_Basel/Biology/Cloning&amp;printable=yes" title="Printable version of this page [p]" accesskey="p">Printable version</A></LI><LI id="t-permalink"><A href="/wiki/index.php?title=Team:ETHZ_Basel/Biology/Cloning&amp;oldid=207246" title="Permanent link to this revision of the page">Permanent link</A></LI><LI id="privacy"><A href="/2010.igem.org:Privacy_policy" title="2010.igem.org:Privacy policy">Privacy policy</A></LI><LI id="disclaimer"><A href="/2010.igem.org:General_disclaimer" title="2010.igem.org:General disclaimer">Disclaimers</A></LI></UL></DIV></DIV></DIV></BODY></HTML>