<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_Nottingham_Achievements skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><DIV id="top_title"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:Nottingham/Achievements</SPAN></H1></DIV><DIV id="HQ_page"><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><P><MENU class="main-menu"><MENUITEM class="header"><A href="/Team:Nottingham" style="color:inherit"><EM>Clostridium</EM> dTox</A></MENUITEM><MENU class="right menu"><MENUITEM class="dropdown dropdowntn"><SPAN class="link" data-url="/Team:Nottingham/Project">Project</SPAN></MENUITEM></MENU></MENU></P><DIV class="dropdown-content"><A href="/Team:Nottingham/Project#description">
			    Project description
			  </A><A href="/Team:Nottingham/Project#abstract">
			    Abstract
			  </A></DIV><MENUITEM class="dropdown dropdowntn"><SPAN class="link" data-url="/Team:Nottingham/Human_Practices">Human Practices</SPAN><DIV class="dropdown-content"><A href="/Team:Nottingham/Human_Practices#overview">
					Overview
				</A><A href="/Team:Nottingham/Human_Practices#silver">
					Silver
				</A><A href="/Team:Nottingham/Human_Practices#gold">
					Gold
				</A><A href="/Team:Nottingham/Human_Practices#safety">
					Safety and Ethics
				</A></DIV></MENUITEM><MENUITEM class="dropdown dropdowntn"><SPAN class="link" data-url="/Team:Nottingham/Public_Engagement">Public Engagement</SPAN><DIV class="dropdown-content"><A href="/Team:Nottingham/Public_Engagement#discovery">
					Discovery day
				</A><A href="/Team:Nottingham/Public_Engagement#school-workshops">
					School workshops
				</A><A href="/Team:Nottingham/Public_Engagement#media">
					Media
				</A></DIV></MENUITEM><MENUITEM class="dropdown dropdowntn"><SPAN class="link" data-url="/Team:Nottingham/Lab">Lab
			<DIV class="dropdown-content"><A href="/Team:Nottingham/Lab#ideas">
					Ideas
				</A><A href="/Team:Nottingham/Lab#design">
					Design
				</A><A href="/Team:Nottingham/Lab#results">
					Results
				</A><A href="/Team:Nottingham/Lab#conclusion">
					Conclusion
				</A><A href="/Team:Nottingham/Lab#future-work">
					Future Work
				</A><A href="/Team:Nottingham/Lab#interlab">
					Interlab
				</A><A href="/Team:Nottingham/Lab#labfolder">
					Labfolder
				</A></DIV></SPAN></MENUITEM><MENUITEM class="dropdown dropdowntn"><SPAN class="link" data-url="/Team:Nottingham/Model">Modelling
			<DIV class="dropdown-content" style="min-width:300px"><A href="/Team:Nottingham/Model#intro">
					Introduction
				</A><A href="/Team:Nottingham/Model#summary">
					Summary
				</A><A href="/Team:Nottingham/Model#lytic-phage-models">
					Lytic Phage Models
				</A><A href="/Team:Nottingham/Model#temperate-phage-models">
					Temporate Phage Models
				</A><A href="/Team:Nottingham/Model#comparison-of-models">
					Comparsion of all three models
				</A><A href="/Team:Nottingham/Model#methodology">
					Methodology
				</A><A href="/Team:Nottingham/Model#parameters">
					Parameters
				</A><A href="/Team:Nottingham/Model#references">
					References
				</A></DIV></SPAN></MENUITEM><MENUITEM class="dropdown dropdowntn"><SPAN class="link" data-url="/Team:Nottingham/Collaborations">Collaborations
			<DIV class="dropdown-content" style="min-width:300px"><A href="/Team:Nottingham/Collaborations#warwick">
					The University of Warwick 
				</A><A href="/Team:Nottingham/Collaborations#biomarvel">
					Team BioMarvel- Korea
				</A><A href="/Team:Nottingham/Collaborations#warwick-and-imperial">
					The University of Warwick and Imperial College London
				</A><A href="/Team:Nottingham/Collaborations#meetups">
					UK Meetups
				</A></DIV></SPAN></MENUITEM><MENUITEM class="dropdown dropdowntn"><SPAN class="link" data-url="/Team:Nottingham/Achievements">Achievements
			<DIV class="dropdown-content"><A href="/Team:Nottingham/Achievements#medal-criteria">
					Medal Criteria
				</A><A href="/Team:Nottingham/Achievements#parts">
					Parts
				</A><A href="/Team:Nottingham/Achievements#improved-parts">
					Improved parts
				</A><A href="/Team:Nottingham/Achievements#demonstrate">
					Demonstrate
				</A><A href="/Team:Nottingham/Achievements#conclusion">
					General conclusion
				</A></DIV></SPAN></MENUITEM><MENUITEM class="dropdown dropdowntn"><SPAN class="link" data-url="/Team:Nottingham/Team">Team
			<DIV class="dropdown-content" style="min-width:200px"><A href="/Team:Nottingham/Team#Students">
					Students
				</A><A href="/Team:Nottingham/Team#Supervisors">
					Supervisors
				</A><A href="/Team:Nottingham/Team#Principal-investigators">
					Principal Investigators
				</A></DIV></SPAN></MENUITEM><A href="/Team:Nottingham/Attributions"><MENUITEM class="dropdown dropdowntn">
			Attributions
		</MENUITEM></A><SECTION class="sidebar-content"><DIV class="main appear" style="opacity: 1; transform: none; width: 70%;"><DIV class="ui breadcrumb" style="padding-bottom:23px;"><A class="section">Home</A><DIV class="active section">Achievements</DIV></DIV><DIV class="ui segment"><H2>Medal criteria</H2><P>			
Nottingham iGEM team have been keeping the medal requirements in mind whilst working hard on their project. Below is a list of the completed criteria to earn each medal.
</P><DIV class="ui bulleted list"><DIV class="item">Models were created to give insights into the use of lytic vs temperate phage and to determine whether the treatment would be more successful over a prolonged period if the phage kept its ability to enter the lytic cycle rather than removing this ability to produce a strictly temperate phage. It was also used to study how these impacted on the titre of phage that would need to be administered.</DIV><DIV class="item">Integrated human practices: We consulted with experts from within the field from various universities across the UK, including University of Birmingham and University of Leeds, as well as holding a discussion group with non-scientists. This allowed us to inform the decisions we made in both the lab design and the modelling. The discussion group were concerned about the delivery of the phage and it was suggested by an expert that delivery should be by capsule. This was therefore chosen as the delivery method for the phage therapeutic, rather than using injection. The discussion group were also concerned about how elderly people would be informed about the new therapeutic therefore, a leaflet about phage therapy was created which could be given to patients to provide advice. Our integration is summarised in a flow diagram on our Wiki. We have also created a human practices gold guide to help future iGEM teams with their human practices.</DIV><DIV class="item">We have improved the characterisation of the part BBa_J23119 by the addition of an RBS and GFP, and characterised its fluorescence relative to iGEM InterLab control plasmids, using the calibration curves generated during the InterLab study. The new composite part used for this characterisation was BBa_K2715119. Additionally as part of our collaborative studies, we invited two other iGEM teams to characterise our composite parts using their equipment and calibration curves, and the observed reproducibility between laboratories further validated our observations. Furthermore this part has now been characterised in a Gram-negative and a Gram-positive non-model organism. When tested in Gram-positive organism it was driving expression of gusA.</DIV><DIV class="item">We demonstrated that our second antisense RNA construct [BBa_K2715008] was able to suppress the rate of production of toxin in <EM>C. difficile</EM> by 85%, validating our approach to reducing toxicity. </DIV></DIV><DIV class="ui bulleted list"><DIV class="item">We successfully validated the functionality of the following submitted parts: BBa_K2715001, BBa_K2715002 and BBa_K2715008.</DIV><DIV class="item">We collaborated with the University of Warwick and Imperial College London to validate our promoters and we collected a water sample for the University of Warwick. We also collaborated with Team BioMarvel, Korea to grammar check their Wiki pages.</DIV><DIV class="item">We held a discussion group with non-scientists to determine their thoughts and feelings on antibiotic resistance and alternative therapies. We also interviewed experts in the field from different universities in the UK. We have also compiled reports showing the impact of our project on the community and whether elderly people have difficulties taking their prescribed medication. We also mined the comments from a you tube video on phage therapy to explore the emotions people feel towards phage therapy.</DIV></DIV><DIV class="ui bulleted list"><DIV class="item">We have registered for the iGEM competition and will be attending the Giant Jamboree.</DIV><DIV class="item">We have completed the competition deliverables, to produce a Wiki, poster, presentation and complete the judging form.</DIV><DIV class="item">We have created an Attributions page showing the attributions for every aspect of our project.</DIV><DIV class="item">We participated and had our results accepted in the InterLab Measurement Study.</DIV></DIV></DIV><DIV class="ui segment"><H2>Parts</H2><TABLE class="ui celled table"><THEAD><TR><TH>﻿Unique part name</TH><TH>Part type</TH><TH>Description</TH></TR></THEAD><TBODY><TR><TD data-label="Name"><A href="http://parts.igem.org/Part:BBa_K2715009 ">BBa_K2715009</A></TD><TD data-label="Name">Basic</TD><TD data-label="Name"><EM>C. acetobutylicum</EM> thiolase gene RBS</TD></TR><TR><TD data-label="Name"><A href="http://parts.igem.org/Part:BBa_K2715010 ">BBa_K2715010</A></TD><TD data-label="Name">Basic</TD><TD data-label="Name">Constitutive promoter from <EM>C. acetobutylicum</EM> thiolase gene</TD></TR><TR><TD data-label="Name"><A href="http://parts.igem.org/Part:BBa_K2715011 ">BBa_K2715011</A></TD><TD data-label="Name">Basic</TD><TD data-label="Name">Constitutive promoter from <EM>C. sporogenes</EM> ferrodoxin gene</TD></TR><TR><TD data-label="Name"><A href="http://parts.igem.org/Part:BBa_K2715012 ">BBa_K2715012</A></TD><TD data-label="Name">Basic</TD><TD data-label="Name">Native promoter from <EM>C. difficile</EM> toxin A gene</TD></TR><TR><TD data-label="Name"><A href="http://parts.igem.org/Part:BBa_K2715013 ">BBa_K2715013</A></TD><TD data-label="Name">Basic</TD><TD data-label="Name">Native promoter from <EM>C. difficile</EM> toxin B gene</TD></TR><TR><TD data-label="Name"><A href="http://parts.igem.org/Part:BBa_K2715014 ">BBa_K2715014</A></TD><TD data-label="Name">Basic</TD><TD data-label="Name">HydA Terminator from <EM>C. acetobutylicum</EM></TD></TR><TR><TD data-label="Name"><A href="http://parts.igem.org/Part:BBa_K2715015 ">BBa_K2715015</A></TD><TD data-label="Name">Basic</TD><TD data-label="Name">Antisense RNA targeting <EM>tcdA</EM> 1</TD></TR><TR><TD data-label="Name"><A href="http://parts.igem.org/Part:BBa_K2715016 ">BBa_K2715016</A></TD><TD data-label="Name">Basic</TD><TD data-label="Name">Antisense RNA targeting <EM>tcdA</EM> 2</TD></TR><TR><TD data-label="Name"><A href="http://parts.igem.org/Part:BBa_K2715017 ">BBa_K2715017</A></TD><TD data-label="Name">Basic</TD><TD data-label="Name">Antisense RNA targeting <EM>tcdB</EM> 1</TD></TR><TR><TD data-label="Name"><A href="http://parts.igem.org/Part:BBa_K2715018 ">BBa_K2715018</A></TD><TD data-label="Name">Basic</TD><TD data-label="Name">Antisense RNA targeting <EM>tcdB</EM> 2</TD></TR><TR><TD data-label="Name"><A href="http://parts.igem.org/Part:BBa_K2715019 ">BBa_K2715019</A></TD><TD data-label="Name">Basic</TD><TD data-label="Name"><EM>C. acetobutylicum</EM> thiolase promoter 5' UTR</TD></TR><TR><TD data-label="Name"><A href="http://parts.igem.org/Part:BBa_K2715020 ">BBa_K2715020</A></TD><TD data-label="Name">Basic</TD><TD data-label="Name"><EM>C. sporogenes</EM> ferredoxin gene RBS</TD></TR><TR><TD data-label="Name"><A href="http://parts.igem.org/Part:BBa_K2715021 ">BBa_K2715021</A></TD><TD data-label="Name">Basic</TD><TD data-label="Name"><EM>C. difficile</EM> Toxin A promoter 5' UTR</TD></TR><TR><TD data-label="Name"><A href="http://parts.igem.org/Part:BBa_K2715022 ">BBa_K2715022</A></TD><TD data-label="Name">Basic</TD><TD data-label="Name"><EM>C. difficile</EM> Toxin A promoter RBS</TD></TR><TR><TD data-label="Name"><A href="http://parts.igem.org/Part:BBa_K2715023 ">BBa_K2715023</A></TD><TD data-label="Name">Basic</TD><TD data-label="Name"><EM>C. difficile</EM> Toxin B promoter 5' UTR without RBS</TD></TR><TR><TD data-label="Name"><A href="http://parts.igem.org/Part:BBa_K2715024 ">BBa_K2715024</A></TD><TD data-label="Name">Basic</TD><TD data-label="Name"><EM>C. difficile</EM> Toxin B promoter RBS</TD></TR><TR><TD data-label="Name"><A href="http://parts.igem.org/Part:BBa_K2715032 ">BBa_K2715032</A></TD><TD data-label="Name">Basic</TD><TD data-label="Name">Synthetic guide RNA targeting sequence 1 for <EM>tcdA</EM> promoter</TD></TR><TR><TD data-label="Name"><A href="http://parts.igem.org/Part:BBa_K2715033 ">BBa_K2715033</A></TD><TD data-label="Name">Basic</TD><TD data-label="Name">Synthetic guide RNA targeting sequence 2 for <EM>tcdA</EM> promoter</TD></TR><TR><TD data-label="Name"><A href="http://parts.igem.org/Part:BBa_K2715034 ">BBa_K2715034</A></TD><TD data-label="Name">Basic</TD><TD data-label="Name">Synthetic guide RNA targeting sequence 3 for <EM>tcdA</EM> promoter</TD></TR><TR><TD data-label="Name"><A href="http://parts.igem.org/Part:BBa_K2715035 ">BBa_K2715035</A></TD><TD data-label="Name">Basic</TD><TD data-label="Name">Synthetic guide RNA targeting sequence 4 for <EM>tcdA</EM> promoter</TD></TR><TR><TD data-label="Name"><A href="http://parts.igem.org/Part:BBa_K2715036 ">BBa_K2715036</A></TD><TD data-label="Name">Basic</TD><TD data-label="Name">Synthetic guide RNA targeting sequence 5 for <EM>tcdA</EM> promoter</TD></TR><TR><TD data-label="Name"><A href="http://parts.igem.org/Part:BBa_K2715037 ">BBa_K2715037</A></TD><TD data-label="Name">Basic</TD><TD data-label="Name">Synthetic guide RNA targeting sequence 6 for <EM>tcdA</EM> promoter</TD></TR><TR><TD data-label="Name"><A href="http://parts.igem.org/Part:BBa_K2715042 ">BBa_K2715042</A></TD><TD data-label="Name">Basic</TD><TD data-label="Name">Synthetic guide sequence for <EM>S. pyrogenes cas9</EM> - targeting sequence</TD></TR><TR><TD data-label="Name"><A href="http://parts.igem.org/Part:BBa_K2715043 ">BBa_K2715043</A></TD><TD data-label="Name">Basic</TD><TD data-label="Name">Alternative sigma factor TcdR regulating toxin expression in <EM>C. difficile</EM></TD></TR><TR><TD data-label="Name"><A href="http://parts.igem.org/Part:BBa_K2715001 ">BBa_K2715001</A></TD><TD data-label="Name">Composite</TD><TD data-label="Name">Constitutive clostridial thiolase promoter BBa_K2715010, strong RBS and GFP reporter</TD></TR><TR><TD data-label="Name"><A href="http://parts.igem.org/Part:BBa_K2715002 ">BBa_K2715002</A></TD><TD data-label="Name">Composite</TD><TD data-label="Name">Constitutive clostridial ferredoxin promoter BBa_K2715011, strong RBS and GFP reporter</TD></TR><TR><TD data-label="Name"><A href="http://parts.igem.org/Part:BBa_K2715003 ">BBa_K2715003</A></TD><TD data-label="Name">Composite</TD><TD data-label="Name">Clostridial promoter of <EM>tcdB</EM> with its native 5' UTR and RBS, driving a GFP reporter</TD></TR><TR><TD data-label="Name"><A href="http://parts.igem.org/Part:BBa_K2715004 ">BBa_K2715004</A></TD><TD data-label="Name">Composite</TD><TD data-label="Name">Clostridial promoter of tcdA with its native 5' UTR and RBS, driving a GFP reporter</TD></TR><TR><TD data-label="Name"><A href="http://parts.igem.org/Part:BBa_K2715005 ">BBa_K2715005</A></TD><TD data-label="Name">Composite</TD><TD data-label="Name"><EM>E.coli</EM> promoter BBa_J23119 driving synthetic guide 1 targeting <EM>tcdB</EM> promoter</TD></TR><TR><TD data-label="Name"><A href="http://parts.igem.org/Part:BBa_K2715006 ">BBa_K2715006</A></TD><TD data-label="Name">Composite</TD><TD data-label="Name"><EM>E.coli</EM> promoter BBa_J23119 driving synthetic guide 2 targeting <EM>tcdB</EM> promoter</TD></TR><TR><TD data-label="Name"><A href="http://parts.igem.org/Part:BBa_K2715007 ">BBa_K2715007</A></TD><TD data-label="Name">Composite</TD><TD data-label="Name">Antisense RNA targeting <EM>C. difficile</EM> toxins composite 1</TD></TR><TR><TD data-label="Name"><A href="http://parts.igem.org/Part:BBa_K2715008 ">BBa_K2715008</A></TD><TD data-label="Name">Composite</TD><TD data-label="Name">Antisense RNA targeting <EM>C. difficile</EM> toxins composite 2</TD></TR><TR><TD data-label="Name"><A href="http://parts.igem.org/Part:BBa_K2715025 ">BBa_K2715025</A></TD><TD data-label="Name">Composite</TD><TD data-label="Name">Constitutive clostridial promoter BBa_K2715010, strong RBS and GusA reporter</TD></TR><TR><TD data-label="Name"><A href="http://parts.igem.org/Part:BBa_K2715026 ">BBa_K2715026</A></TD><TD data-label="Name">Composite</TD><TD data-label="Name">Constitutive clostridial promoter BBa_K2715011, strong RBS and GusA reporter</TD></TR><TR><TD data-label="Name"><A href="http://parts.igem.org/Part:BBa_K2715027 ">BBa_K2715027</A></TD><TD data-label="Name">Composite</TD><TD data-label="Name">Clostridial promoter of <EM>tcdB</EM> with its native 5' UTR and RBS, driving a GusA reporter</TD></TR><TR><TD data-label="Name"><A href="http://parts.igem.org/Part:BBa_K2715028 ">BBa_K2715028</A></TD><TD data-label="Name">Composite</TD><TD data-label="Name">Clostridial promoter of <EM>tcdA</EM> with its native 5' UTR and RBS, driving a GusA reporter</TD></TR><TR><TD data-label="Name"><A href="http://parts.igem.org/Part:BBa_K2715029 ">BBa_K2715029</A></TD><TD data-label="Name">Composite</TD><TD data-label="Name">Constitutive <EM>E.coli</EM> promoter BBa_J23106, strong RBS and GusA reporter</TD></TR><TR><TD data-label="Name"><A href="http://parts.igem.org/Part:BBa_K2715030 ">BBa_K2715030</A></TD><TD data-label="Name">Composite</TD><TD data-label="Name">Constitutive <EM>E.coli</EM> promoter BBa_J23114, strong RBS and GusA reporter</TD></TR><TR><TD data-label="Name"><A href="http://parts.igem.org/Part:BBa_K2715031 ">BBa_K2715031</A></TD><TD data-label="Name">Composite</TD><TD data-label="Name">Constitutive <EM>E.coli</EM> promoter BBa_J23119, strong RBS and GusA reporter</TD></TR><TR><TD data-label="Name"><A href="http://parts.igem.org/Part:BBa_K2715038 ">BBa_K2715038</A></TD><TD data-label="Name">Composite</TD><TD data-label="Name"><EM>E.coli</EM> promoter BBa_J23119 driving synthetic guide 3 targeting <EM>tcdB</EM> promoter</TD></TR><TR><TD data-label="Name"><A href="http://parts.igem.org/Part:BBa_K2715039 ">BBa_K2715039</A></TD><TD data-label="Name">Composite</TD><TD data-label="Name"><EM>E.coli</EM> promoter BBa_J23119 driving synthetic guide 4 targeting <EM>tcdB</EM> promoter</TD></TR><TR><TD data-label="Name"><A href="http://parts.igem.org/Part:BBa_K2715040 ">BBa_K2715040</A></TD><TD data-label="Name">Composite</TD><TD data-label="Name"><EM>E.coli</EM> promoter BBa_J23119 driving synthetic guide 5 targeting <EM>tcdB</EM> promoter</TD></TR><TR><TD data-label="Name"><A href="http://parts.igem.org/Part:BBa_K2715041 ">BBa_K2715041</A></TD><TD data-label="Name">Composite</TD><TD data-label="Name"><EM>E.coli</EM> promoter BBa_J23119 driving synthetic guide 6 targeting <EM>tcdB</EM> promoter</TD></TR><TR><TD data-label="Name"><A href="http://parts.igem.org/Part:BBa_K2715044 ">BBa_K2715044</A></TD><TD data-label="Name">Composite</TD><TD data-label="Name">Constitutive expression of dCas9, and regulatory region of Toxin A in <EM>C. difficile</EM> driving GusA</TD></TR><TR><TD data-label="Name"><A href="http://parts.igem.org/Part:BBa_K2715106 ">BBa_K2715106</A></TD><TD data-label="Name">Composite</TD><TD data-label="Name">Constitutive <EM>E.coli</EM> promoter BBa_J23106, strong RBS and GFP reporter</TD></TR><TR><TD data-label="Name"><A href="http://parts.igem.org/Part:BBa_K2715114 ">BBa_K2715114</A></TD><TD data-label="Name">Composite</TD><TD data-label="Name">Constitutive <EM>E.coli</EM> promoter BBa_J23114, strong RBS and GFP reporter</TD></TR><TR><TD data-label="Name"><A href="http://parts.igem.org/Part:BBa_K2715119 ">BBa_K2715119</A></TD><TD data-label="Name">Composite</TD><TD data-label="Name">Constitutive <EM>E.coli</EM> promoter BBa_J23119, strong RBS and GFP reporter</TD></TR></TBODY></TABLE></DIV><DIV class="ui segment"><H2>Improved parts</H2><P>We have improved three promoters from the iGEM registry by adding a consensus ribosomal binding site (RBS) from the thiolase gene of <EM>Clostridium acetobutylicum</EM> to each, which functions in Gram-positive and Gram-negative bacteria. We then performed a robust characterisation through a GFP assay using a spectrophotometer in conjunction with calibration curves generated during the iGEM Interlab study, and expressed their strengths in standardised units of fluorescence.</P><P>Original promoters from iGEM registry:</P><DIV class="ui bulleted list"><DIV class="item"><A href="http://parts.igem.org/Part:BBa_J23119">BBa_J23119</A></DIV><DIV class="item"><A href="http://parts.igem.org/Part:BBa_J23114">BBa_J23114</A></DIV><DIV class="item"><A href="http://parts.igem.org/Part:BBa_J23106">BBa_J23106</A></DIV></DIV><P>Original promoters with RBS (Ribosomal binding site):</P><DIV class="ui bulleted list"><DIV class="item"><A href="http://parts.igem.org/Part:BBa_K2715119">BBa_K2715119</A></DIV><DIV class="item"><A href="http://parts.igem.org/Part:BBa_K2715114">BBa_K2715114</A></DIV><DIV class="item"><A href="http://parts.igem.org/Part:BBa_K2715106">BBa_K2715106</A></DIV></DIV><P>Additionally we have added four new promoters to the iGEM registry with the same ribosomal binding site from the thiolase gene of <EM>C. acetobutylicum</EM>. The four promoters are from <EM>C. acetobutylicum</EM> (Pcac_thl) [Bba_K2715010], <EM>Clostridium sporogenes</EM> (PCsp_fdx) [Bba_K2715011] and two from <EM>Clostridium difficile</EM> (PCdi_TcdA) [Bba_K2715012] and (PCdi_TcdB) [Bba_K2715013].</P><P>We characterized all seven promoters in both <EM>E.coli</EM> and <EM>C. difficile</EM>. <EM>C. difficile</EM> is a Gram-positive anaerobic organism with significant differences to the <EM>E. coli</EM> chassis for which existing characterisation was performed. The existing registry promoters BBa_J23114, BBa_J23106, and BBa_J23119 were characterised for expression strength using a GusA assay in <EM>C. difficile</EM>. The four novel registry parts were characterised alongside the existing registry promoters in a GFP assay in <EM>E. coli</EM> as well as in a GusA assay in <EM>C. difficile</EM>. The most remarkable conclusion from the <EM>E. coli</EM> GFP assay of these promoters is that both of the suspected strong <EM>C. difficile</EM> promoter PCsp_fdx and Pcac_thl were stronger than any of the three existing registry promoters we assayed; with Pcac_thl producing around three times the concentration of fluorescein (0.3235µM) as the positive control used in the InterLab studies (0.0958µM).</P><P>As part of our collaborative studies, we invited two other iGEM teams (Imperial College London and University of Warwick) to characterise our composite parts using their equipment and calibration curves, and the observed reproducibility between laboratories further validated our observations. Furthermore this part has now been characterised in a Gram-negative and a Gram-positive non-model organism. When tested in the Gram-positive organism it was driving expression of <EM>gusA</EM>.</P><P>Our main objective in characterising these promoters was to find a suitable pair of strong promoters to use in our subsequent dCas9 or asRNA projects. For this the GusA assay within <EM>C. difficile</EM> was most relevant since this is the chassis in which these constructs would be acting. The <EM>C. difficile</EM> GusA assay clearly showed that none of the three existing registry promoters from <EM>E. coli</EM> had any detectable activity in <EM>C. difficile</EM>. By far the strongest promoter we were able to measure was PCsp_fdx which was around 7.5 times stronger than the next strongest promoter we found (PCdi_TcdA). We were unable to clone the strongest promoter from the <EM>E. coli</EM>  GFP assay PCac_thl into a GusA reporter plasmid. This is likely because of the toxicity of the <EM>gusA</EM> gene in <EM>E. coli</EM> and since we know that  PCac_thl is the strongest of our promoters in <EM>E. coli</EM> it is unsurprising that this was the most problematic plasmid to clone. As a result we did not measure the strength of PCac_thl in <EM>C. difficile</EM>, but due to its measured strength in <EM>C. difficile</EM> as well as its widespread use for overexpression studies in Clostridia we decided to select it alongside PCsp_fdx as a promoter to use in the next stage of our project.</P><H6><STRONG>GFP assays.</STRONG> The graphs above represent the data accumulated from the GFP assay reproduced at Imperial College London and Warwick University. They all show similar levels of fluorescence.</H6></DIV><DIV class="ui segment"><H2>Demonstrate</H2><P>Project goal: to genetically modify a phage capable of suppressing toxin production in <EM>C. difficile</EM> by integrating our genetic constructs into its genome. We are using a phage known to infect strains of <EM>C. difficile</EM> called phiSBRC. The genome of phiSBRC will be packaged with constructs to suppress toxin production either via a CRIPSRi (dead-Cas9) approach or via asRNA.
The asRNA parts were created to target the two well characterised <EM>C. difficile</EM> toxins, TcdA and TcdB. These toxins are responsible for the characteristic symptoms of <EM>C. difficile</EM> infection. The parts were constructed in such a way that they targeted both toxins simultaneously because both toxins have been shown to have independent roles.</P><P>The asRNA parts are downstream of a PCac_thl promoter and a PCsp_fdx promoter. Between, the promoter-asRNA pairs, there is a transcriptional terminator (fdx). As a result of GusA and GFP assays, we determined that PCac_thl promoter was the strongest promoter, followed by PCsp_fdx promoter in <EM>C. difficile</EM>. Together, the basic parts produced our composite part.</P><P>Longer asRNAs give a higher degree of suppression but generally have more unwanted off-target effects which could be due to increased similarity with other mRNA transcripts. As the amount of DNA that phiSBRC can package is limited shorter asRNAs would be more desirable. As a result, we created two versions of the composite part which we named ‘asRNA Construct One’ (BBa_K2715007) and ‘asRNA Construct Two’ (BBa_K2715008). For each part we varied the length so that ‘asRNA Construct Two’ had 26 more base pairs complementary to the toxin gene mRNA than ‘asRNA Construct One’. ‘asRNA Construct One’ has a binding region of 24bp while ‘asRNA Construct Two’ has a binding region of 50bp.</P><P>We were able to directly measure the effect of our constructs on toxin production and toxicity of the <EM>C. difficile</EM> supernatant. We cloned the constructs into a plasmid vector suitable for transforming <EM>C. difficile</EM>. Supernatant cytotoxicity was compared between <EM>C. difficile</EM> containing the vectors with our constructs (coding for the asRNA) and wild-type <EM>C. difficile</EM> using African green monkey kidney epithelial ‘Vero’ cells. Supernatant samples from <EM>C. difficile</EM> cultures over five days were obtained for specific times points and the sterile supernatants applied to the Vero cells. Cytotoxicity was estimated using a lactate dehydrogenase (LDH) assay. Our results suggest that ‘asRNA Construct One’ reduces the rate of toxin production by 80% whilst asRNA Construct Two reduces it by 85% over wild type.</P><DIV class="ui two column grid"><DIV class="ui column"><H6><STRONG>Cytotoxicity of <EM>C. difficile</EM> supernatants.</STRONG>The graph shows supernatant cytotoxicity over a period of 120 hours. There is considerably less toxin production by <EM>C. difficile</EM> containing asRNA construct 1 and <EM>C. difficile</EM> containing asRNA construct 2 than by wild type <EM>C. difficile</EM>.</H6></DIV><DIV class="ui column"><H6><STRONG>Rate of toxin production.</STRONG><EM>C. difficile</EM> containing asRNA construct 1 and <EM>C. difficile</EM> containing asRNA construct 2 exhibit a significantly slower rate of toxin production than wild type <EM>C. difficile</EM>. Here we see an 80% reduction in the rate of toxin production by <EM>C. difficile</EM> containing asRNA construct 1 and an 85% reduction in the rate of toxin production by <EM>C. difficile</EM> containing asRNA construct 2.</H6></DIV></DIV></DIV><DIV class="ui segment"><H2>General conclusion</H2><P>
Our aim was to offer a treatment for <EM>Clostridium difficile</EM> infections which would not involve antibiotic treatment. Avoiding antibiotics is desirable because of the antibiotic resistance crisis and the risk of relapse associated with broad-spectrum antibiotic use in cases of <EM>C. difficile</EM> infection (CDI). The alternative treatment we focused on is based on a phage which specifically infects strains of <EM>C. difficile</EM>. As such our project is a phage therapy project, but instead of simply relying on the natural abilities of the phage ‘phiSBRC’ we have designed genetic constructs to enhance its function as a treatment for CDI. 
</P><P>
The phage we worked with (phiSBRC) is a temperate phage. This means that its normal lifecycle involves integration into the genome of <EM>C. difficile</EM> in which it can remain for many generations before excising from the genome to replicate and release many phage progeny into the environment. This natural lifecycle does not make it immediately amenable to serving as a therapeutic since many bacterial cells could escape the lethal effects of the phage initially as it integrates into the genome to lie dormant. This facet of its biology would essentially make the phage too slow acting to be a useful therapeutic against CDI. Our aim was to genetically engineer phiSBRC in such a way as to render it capable of suppressing toxin production within the <EM>C. difficile</EM> cells it integrates into. This would mean that even the non-lysed <EM>C. difficile</EM> cells would become non-pathogenic and would make the engineered phiSBRC phage a viable therapeutic to combat CDI. 
</P><P>
One consideration when choosing to work on a therapeutic for humans was the importance of consulting the relevant legislation, public perception and other human practices issues. It was thought that there may be a degree of concern about the idea of introducing genetically engineered phages into patients. For this reason one of our human practices goals was to gain data on public perceptions to our proposed therapeutic. This involved a focus group, expert interviews and an analysis of commentary of a phage themed online video. A key finding which we integrated into our project was the importance of the means of administration of the phage for influencing public perception. As a result of these investigations we opted for encapsulating our therapeutic; serendipitously this was in accordance with the scientific literature which we consulted on the means of phage delivery. Also, in accordance with the information we gained, we created a phage therapy information leaflet. It not only serves to educate the publics on phage therapy but can also be given by physicians to patients who are receiving the treatment. 
</P><P>
One approach we took to investigate the feasibility of our engineered phage against CDI was to create a mathematical model of the phage/bacterial cell lifecycle. The model was used to answer a question we encountered while designing the laboratory work. Our overall aim was to design, build and test a genetic construct which suppresses toxin production in <EM>C. difficile</EM> and subsequently modify the phage to express our construct. The plan to modify the phage involved using CRISPR/Cas9 genome engineering to integrate our construct into the prophage genome. As part of this approach a region of DNA must be removed from the target genome, as such we had the option to remove a region of the phage genome which may further enhance our end product. The specific gene we considered removing encoded a holin; this enzyme is responsible for lysing the bacterial cell at the end of the phage replication cycle. Without the holin the phage would be unable to lyse the cell it resides in. It was thought that this may be useful since in the gut our model suggests we would produce subpopulations of <EM>C. difficile</EM> some of which have been infected by our modified phage (lysogens) and are as such non-toxigenic while others have not been infected and are therefore capable of producing toxin and hence disease. By allowing the phage to lyse the cell it resides in we expected that this would impose a fitness burden onto the lysogen population, causing it to be eventually outcompeted by the wild type toxigenic strains. This possibility was analysed and the conclusion was that despite this potential fitness cost the fact that enough of the newly replicated phages could infect wild type toxigenic strains more than compensated. This means that overall it was beneficial to retain the phages’ ability to undergo cell lysis. It was seen that allowing cell lysis fulfilled a requirement highlighted by the human practices work which was to reduce the number of medications patients must take; this was in particular found to be an issue with elderly patients. A phage which could undergo lysis and spread to toxigenic bacteria would mean that a second dose of prepared phage would not be necessary. In addition by allowing phage-induced lysis the modelling results conclude that having an initially high dosage of phage particles is not crucial. This is in contrast to when a strictly temperate phage is used as a very high initial dose is required to ensure a high enough proportion of lysogens to toxigenic strains are produced. The preparation of a high titre of phage particles was one issue raised by the experts during our human practices interviews where it was noted that preparation can be arduous and costly. The conclusion therefore was that the holin gene should be left undisturbed in the final iteration of our device. 
</P><P>
In order to achieve our goal of engineering a toxin-suppressing phage specific to <EM>C. difficile</EM> we investigated two general approaches to gene suppression; dCas9 (CRISPRi) and antisense RNA (asRNA). Both of these approaches benefitted from knowledge of which promoters would give high expression within <EM>C. difficile</EM>. With this in mind a range of promoters were investigated, three from existing registry parts to which we added a new ribosome binding site part and four of which have been deposited into the registry as new parts entirely. The four parts we added are from three different Clostridial species. This series of promoters were assessed for expression in E. coli and in <EM>C. difficile</EM> using two separate assays. This allowed us to select two promoters which were predicted to have the highest expression levels in <EM>C. difficile</EM> to use in both the CRISPRi and asRNA approaches. Another key finding of the promoter work was that one of the new parts we added to the registry exhibited expression levels three times higher than the positive control which was supplied to teams participating in the 2018 iGEM InterLab studies. This result surprised us since this promoter is native to <ME>Clostridium acetobutylicum where it controls the expression of a gene encoding thiolase. The promoter was named PCac_thl and has the part number Bba_K2715010. To further confirm this result we sent our promoter series to collaborators from Imperial College London and University of Warwick asking them to repeat the experiments we did. To ensure there was no researcher bias to gain the same results as us we did not name the promoters. The results obtained from these collaborations were pleasing in that both teams produced similar results to ours with PCac_thl being the strongest promoter of the set. 
</ME></P><P>
Both the CRISPRi and asRNA approach subsequently used promoters selected from within the assessed promoter series. One aspect of optimisation of the CRISPRi approach was the selection of an appropriate guide RNA. Six guide RNAs designed against the promoter region of TcdA were assessed by placing the TcdA promoter upstream of a reporter gene. The degree to which reporter activity was reduced correlated with the effectiveness of the guide RNA. This assay was able to show that guide RNA six was the most reliably effective and as such would be selected in future CRISPRi work. 
</P><P>
Using antisense RNA to suppress toxin production in <EM>C. difficile</EM> progressed further. The necessary constructs were simpler to clone allowing us to directly measure the ability of our construct to reduce the toxicity of <EM>C. difficile</EM> supernatant. Two different constructs were made, both of which targeted the two <EM>C. difficile</EM> toxin genes tcdA and tcdB. We were interested in the effect of varying the length of the antisense RNA and as such made ‘asRNA Construct Two’ to contain 26bp more complementarily to the toxin gene mRNA than ‘asRNA Construct One’. The characterisation of these two constructs proceeded using African green monkey kidney epithelial ‘Vero’ cells. Sterile supernatants from <EM>C. difficile</EM>cultures were applied to the Vero cells and cytotoxicity was estimated using a lactate dehydrogenase (LDH) assay. Our results suggest that asRNA Construct One reduces the rate of toxin production by 80% whilst the slightly longer asRNA Construct Two reduces it by 85% over wild type. This demonstrates the validity of this approach for engineering a toxin-suppressing phage. It also suggests that the longer asRNA molecule may be beneficial, though it is also more likely to exhibit off-target effects. We designed CRISPR/Cas9 vectors to modify the phage genome in order to express our asRNA constructs. It was with regret that work on this project had to cease at this point.
</P><P>
Future work would focus on optimising toxin suppression further by fully assessing the strength of the CRISPRi approach through cytotoxicity assays in <EM>C. difficile</EM>. The possibility of combining the two approaches would be investigated with a view to more tightly repressing toxin production. The phage genome would be modified to integrate firstly each approach separately and then both approaches together. This would depend on the amount of phage DNA that can be packaged by the phage head and would be tested confirming the production of infectious phage particles. Currently the long term repression of toxin is unknown with these approaches therefore, cytotoxicity assays would be completed to determine whether long term repression is obtained when the approaches are used in combination. Finally, if the engineered phage containing the combination approach is successful, in vivo testing would be conducted to ensure that the therapy would work in the “real world”. At this stage the process for delivery of the phage therapeutic would be investigated and the production of a capsule that could withstand the harsh conditions of the stomach would begin.
</P></DIV></DIV><DIV class="ui sticky rail sidebar-list" style="left: 965.844px;"><DIV class="ui vertical fluid inverted green menu appear" style="opacity: 1; transform: none;"><A class="item" id="medal-criteria-button" href="#medal-criteria">
			    Medal Criteria
			  </A><A class="item" id="parts-button" href="#parts">
			    Parts
			  </A><A class="item" id="improved-parts-button" href="#improved-parts">
			    Improved parts
			  </A><A class="item" id="demonstrate-button" href="#demonstrate">
			    Demonstrate
			  </A><A class="item" id="conclusion-button" href="#conclusion">
			    General conclusion
			  </A></DIV></DIV></SECTION><DIV class="ui container PageCard_container_1ok2Drx5q2skrrX8ItRY6X"><A href="/Team:Nottingham/test/sponsors" class="ui fluid basic button">About our sponsors</A></DIV></DIV></DIV></DIV></DIV></DIV></BODY></HTML>