Team:Nottingham/Demonstrate
Clostridium dTox
Project
Project description Abstract
Human Practices
Overview Silver Gold Safety and Ethics
Public Engagement
Discovery day School workshops Media
Lab
Ideas Design Results Conclusion Future Work Interlab Labfolder
Modelling
Introduction Summary Lytic Phage Models Temporate Phage Models Comparsion of all three models Methodology Parameters References
Collaborations
The University of Warwick Team BioMarvel- Korea The University of Warwick and Imperial College London UK Meetups
Achievements
Medal Criteria Parts Improved parts Demonstrate General conclusion
Team
Students Supervisors Principal Investigators
Attributions
This is part of the Achievements page!
To meet medal criteria, and to make life for judges easier, we have extracted this part of the demonstrate page.
Original page
Demonstrate
Project goal: to genetically modify a phage capable of suppressing toxin production in C. difficile by integrating our genetic constructs into its genome. We are using a phage known to infect strains of C. difficile called phiSBRC. The genome of phiSBRC will be packaged with constructs to suppress toxin production either via a CRIPSRi (dead-Cas9) approach or via asRNA. The asRNA parts were created to target the two well characterised C. difficile toxins, TcdA and TcdB. These toxins are responsible for the characteristic symptoms of C. difficile infection. The parts were constructed in such a way that they targeted both toxins simultaneously because both toxins have been shown to have independent roles.
The asRNA parts are downstream of a PCac_thl promoter and a PCsp_fdx promoter. Between, the promoter-asRNA pairs, there is a transcriptional terminator (fdx). As a result of GusA and GFP assays, we determined that PCac_thl promoter was the strongest promoter, followed by PCsp_fdx promoter in C. difficile. Together, the basic parts produced our composite part.
Longer asRNAs give a higher degree of suppression but generally have more unwanted off-target effects which could be due to increased similarity with other mRNA transcripts. As the amount of DNA that phiSBRC can package is limited shorter asRNAs would be more desirable. As a result, we created two versions of the composite part which we named ‘asRNA Construct One’ (BBa_K2715007) and ‘asRNA Construct Two’ (BBa_K2715008). For each part we varied the length so that ‘asRNA Construct Two’ had 26 more base pairs complementary to the toxin gene mRNA than ‘asRNA Construct One’. ‘asRNA Construct One’ has a binding region of 24bp while ‘asRNA Construct Two’ has a binding region of 50bp.
We were able to directly measure the effect of our constructs on toxin production and toxicity of the C. difficile supernatant. We cloned the constructs into a plasmid vector suitable for transforming C. difficile. Supernatant cytotoxicity was compared between C. difficile containing the vectors with our constructs (coding for the asRNA) and wild-type C. difficile using African green monkey kidney epithelial ‘Vero’ cells. Supernatant samples from C. difficile cultures over five days were obtained for specific times points and the sterile supernatants applied to the Vero cells. Cytotoxicity was estimated using a lactate dehydrogenase (LDH) assay. Our results suggest that ‘asRNA Construct One’ reduces the rate of toxin production by 80% whilst asRNA Construct Two reduces it by 85% over wild type.
Cytotoxicity of C. difficile supernatants.The graph shows supernatant cytotoxicity over a period of 120 hours. There is considerably less toxin production by C. difficile containing asRNA construct 1 and C. difficile containing asRNA construct 2 than by wild type C. difficile.
Rate of toxin production.C. difficile containing asRNA construct 1 and C. difficile containing asRNA construct 2 exhibit a significantly slower rate of toxin production than wild type C. difficile. Here we see an 80% reduction in the rate of toxin production by C. difficile containing asRNA construct 1 and an 85% reduction in the rate of toxin production by C. difficile containing asRNA construct 2.
About our sponsors
