<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_Nottingham_Lab skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><DIV id="top_title"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:Nottingham/Lab</SPAN></H1></DIV><DIV id="HQ_page"><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><P><MENU class="main-menu"><MENUITEM class="header"><A href="/Team:Nottingham" style="color:inherit"><EM>Clostridium</EM> dTox</A></MENUITEM><MENU class="right menu"><MENUITEM class="dropdown dropdowntn"><SPAN class="link" data-url="/Team:Nottingham/Project">Project</SPAN></MENUITEM></MENU></MENU></P><DIV class="dropdown-content"><A href="/Team:Nottingham/Project#description">
			    Project description
			  </A><A href="/Team:Nottingham/Project#abstract">
			    Abstract
			  </A></DIV><MENUITEM class="dropdown dropdowntn"><SPAN class="link" data-url="/Team:Nottingham/Human_Practices">Human Practices</SPAN><DIV class="dropdown-content"><A href="/Team:Nottingham/Human_Practices#overview">
					Overview
				</A><A href="/Team:Nottingham/Human_Practices#silver">
					Silver
				</A><A href="/Team:Nottingham/Human_Practices#gold">
					Gold
				</A><A href="/Team:Nottingham/Human_Practices#safety">
					Safety and Ethics
				</A></DIV></MENUITEM><MENUITEM class="dropdown dropdowntn"><SPAN class="link" data-url="/Team:Nottingham/Public_Engagement">Public Engagement</SPAN><DIV class="dropdown-content"><A href="/Team:Nottingham/Public_Engagement#discovery">
					Discovery day
				</A><A href="/Team:Nottingham/Public_Engagement#school-workshops">
					School workshops
				</A><A href="/Team:Nottingham/Public_Engagement#media">
					Media
				</A></DIV></MENUITEM><MENUITEM class="dropdown dropdowntn"><SPAN class="link" data-url="/Team:Nottingham/Lab">Lab
			<DIV class="dropdown-content"><A href="/Team:Nottingham/Lab#ideas">
					Ideas
				</A><A href="/Team:Nottingham/Lab#design">
					Design
				</A><A href="/Team:Nottingham/Lab#results">
					Results
				</A><A href="/Team:Nottingham/Lab#conclusion">
					Conclusion
				</A><A href="/Team:Nottingham/Lab#future-work">
					Future Work
				</A><A href="/Team:Nottingham/Lab#interlab">
					Interlab
				</A><A href="/Team:Nottingham/Lab#labfolder">
					Labfolder
				</A></DIV></SPAN></MENUITEM><MENUITEM class="dropdown dropdowntn"><SPAN class="link" data-url="/Team:Nottingham/Model">Modelling
			<DIV class="dropdown-content" style="min-width:300px"><A href="/Team:Nottingham/Model#intro">
					Introduction
				</A><A href="/Team:Nottingham/Model#summary">
					Summary
				</A><A href="/Team:Nottingham/Model#lytic-phage-models">
					Lytic Phage Models
				</A><A href="/Team:Nottingham/Model#temperate-phage-models">
					Temporate Phage Models
				</A><A href="/Team:Nottingham/Model#comparison-of-models">	Comparsion of all three models
				</A><A href="/Team:Nottingham/Model#methodology">
					Methodology
				</A><A href="/Team:Nottingham/Model#parameters">
					Parameters
				</A><A href="/Team:Nottingham/Model#references">
					References
				</A></DIV></SPAN></MENUITEM><MENUITEM class="dropdown dropdowntn"><SPAN class="link" data-url="/Team:Nottingham/Collaborations">Collaborations
			<DIV class="dropdown-content" style="min-width:300px"><A href="/Team:Nottingham/Collaborations#warwick">
					The University of Warwick 
				</A><A href="/Team:Nottingham/Collaborations#biomarvel">
					Team BioMarvel- Korea
				</A><A href="/Team:Nottingham/Collaborations#warwick-and-imperial">
					The University of Warwick and Imperial College London
				</A><A href="/Team:Nottingham/Collaborations#meetups">
					UK Meetups
				</A></DIV></SPAN></MENUITEM><MENUITEM class="dropdown dropdowntn"><SPAN class="link" data-url="/Team:Nottingham/Achievements">Achievements
			<DIV class="dropdown-content"><A href="/Team:Nottingham/Achievements#medal-criteria">
					Medal Criteria
				</A><A href="/Team:Nottingham/Achievements#parts">
					Parts
				</A><A href="/Team:Nottingham/Achievements#improved-parts">
					Improved parts
				</A><A href="/Team:Nottingham/Achievements#demonstrate">
					Demonstrate
				</A><A href="/Team:Nottingham/Achievements#conclusion">
					General conclusion
				</A></DIV></SPAN></MENUITEM><MENUITEM class="dropdown dropdowntn"><SPAN class="link" data-url="/Team:Nottingham/Team">Team
			<DIV class="dropdown-content" style="min-width:200px"><A href="/Team:Nottingham/Team#Students">
					Students
				</A><A href="/Team:Nottingham/Team#Supervisors">
					Supervisors
				</A><A href="/Team:Nottingham/Team#Principal-investigators">
					Principal Investigators
				</A></DIV></SPAN></MENUITEM><A href="/Team:Nottingham/Attributions"><MENUITEM class="dropdown dropdowntn">
			Attributions
		</MENUITEM></A><SECTION class="sidebar-content"><DIV class="main appear" style="opacity: 1; transform: none; width: 70%;"><DIV class="ui breadcrumb" style="padding-bottom:23px;"><A class="section">Home</A><DIV class="active section">Lab</DIV></DIV><DIV class="ui segment"><H2>Ideas</H2><P>
				Immediately after meeting for the first time, we were tasked with finding a project that we would be working on over the summer. At first, the idea seemed daunting but before we knew it, amazing (sometimes outrageous) ideas were being suggested.
			</P><P>
				Almost a month later, after numerous presentations, we narrowed down our potential projects to five: 
			</P><DIV class="ui bulleted list"><DIV class="item">
					Exploiting the PET-degrading qualities of <EM>Ideonella sakaiensisto</EM> tackle the plastic crisis
				</DIV><DIV class="item">
					Creating a biosensor to detect harmful substances in water bottles
				</DIV><DIV class="item">
					using <EM>Bdellovibrio bacteriovorus</EM> (and its enzymes) to attack and kill harmful/unwanted bacteria in food processes
				</DIV><DIV class="item">
					using mRNA interference to silence <EM>Streptococcus mutanstoxins</EM></DIV><DIV class="item">
					using mRNA interference to silence <EM>Clostridium difficiletoxins</EM></DIV></DIV><H4><EM>Ideonella sakaiensis</EM></H4><P><EM>Ideonella sakaiensisis</EM> a bacterium that can degrade plastic (PET) using two enzymes. It was discovered outside a plastic recycling facility in Japan and subsequently isolated by research teams at Kyoto Institute of Technology and Keio University(Yoshida et al., 2016). The idea of being able to tackle the plastic crisis affecting the oceans and marine life was a particularly appealing one. However, we quickly realised that we didn’t know how to transform the organism and the time-scale for how long it would take. Also, our project was very similar to other research being conducted and so it would be difficult to come up with a novel way to address the crisis.
			</P><H4>Water bottle biosensor</H4><P>
				According to the House of Commons Environmental Audit Committee, in their first report of the season (2017-19), in the UK alone we use ‘13 billion plastic bottles every year’ and ‘only 7.5 billion are recycled’. This prompted the team to suggest a biosensor that detects harmful substance in plastic bottles. As a result, the average consumer would buy fewer plastic bottles and reuse their bottle. However, many solutions immediately were proposed that counteracted the need for the biosensor-this included the fact that many people buy permanent water bottles that they wash and reuse; it would be more cost effective to campaign recycling efforts than to market a biosensor and it is cheaper to just recycle a plastic bottle and buy a new one than to buy a biosensor. Indeed in the report by the Environmental Audit Committee (potential stakeholders), they suggest increasing the number of water fountains in open spaces and improving recycling through a number of methods including Deposit Return Schemes (which incentivise consumers).
			</P><H4><EM>Bdellovibrio bacteriovorus</EM></H4><P><EM>Bdellovibrio bacteriovorus</EM> is a predatory bacterium that preys on other (Gram-negative) bacterial species (Rendulic et al., 2004). Its enzymes and their mechanisms are being studied in order to better understand the bacterium for future use as a possible therapeutic agent or as a method to penetrate biofilms (Kadouri and O'Toole, 2005; Sockett and Lambert, 2004). Our team was interested in engineering the bacterium to use it (and its enzymes) as a ‘pathogen eating machine’in food processes-for example targeting <EM>Clostridium botulinum</EM>, which releases the botulinum toxin. On top of it being an interesting species to work on, we had a leading expert on the <EM>Bdellovibrio</EM> species at our university. However, the main bacterium we were targeting <EM>(Clostridium botulinum)</EM> is a Gram-positive bacterium so is not recognised by <EM>Bdellovibrio</EM>. Also, it would take many weeks to engineer <EM>Bdellovibrio</EM> for single gene mutations, which is difficult to transform using random transposon mutagenesis. This was a project that required many years which our team did not have!
			</P><H4>mRNA interference of <EM>Streptococcus mutans</EM></H4><P><EM>Streptococcus mutansis</EM> one of a number of bacteria involved in tooth decay and is the most prevalent. According to Public Health England, in the UK, ‘almost a quarter (24.7%) of 5 year olds have tooth decay’ of which 3 or 4 teeth are affected. Dental health problems also have a heavy financial burden on the NHS-it spends around £3.4 billion per year on dental care. What makes <EM>S. mutans</EM> particularly hard to deal wit his its ability to form biofilms regulated by glucosyltransferases which catalyse sucrose to adhesive glucan. In particular, GtfB and GtfB seem to be the most important in biofilm production-mutations in the gtfB and gtfC genes disrupted microcolony formation on saliva coated surfaces(Koo et al., 2010). Our idea was to use a bacteriophage (a virus that only infects bacteria) to deliver micro RNAs or small interfering RNAs to silence those genes. One of our supervisors works with phage so she would be able to guide the wet lab team. We opted to silence the toxins rather than kill the bacteria because we wanted a way of preventing glucan formation without disturbing the balance of the oral microbiome
			</P><H4>mRNA interference of <EM>Clostridium difficile</EM></H4><P><EM>Clostridium difficile</EM> is an anaerobic bacterium capable of forming spores (meaning it persists in the environment). <EM>Clostridium difficile</EM> infection(CDI) is a major infection which causes hospital and community acquired-diarrhoea. It particularly affects those who have long-term hospital stays (especially the elderly), are immunocompromised/immunodeficient (for example due to chemotherapy) and/or are on broad-spectrum antibiotics. CDI has a heavy financial burden-according to Zhang et al. (2017), the annual costs due to <EM>C. difficile</EM> infections in the US alone are an estimated $6.3 billion with almost 2.4 million days spent in hospitals. According to the Centers for Disease Control and Prevention, between 1999 and 2007 there was an increase in the estimated number of deaths due to CDI from 3,000 to 14,000 which was seen across Europe and Canada as well (McDonaldet al., 2012; Lessa et al., 2012). This has been linked to a hypervirulent, resistance strain of <EM>C. difficile</EM>.
			</P><P>
				Within the SBRC, our supervisors work with a wide range of Clostridial species in the Clostridia Research Group. Unlike with <EM>S. mutans</EM>, there was more experience within the team and so the supervisors would be able to work with us more closely and advise us on this project.
			</P><P>
				It was hard to decide between working with <EM>Ideonella sakaiensis</EM>, <EM>Streptococcus mutansor</EM> and <EM>C. difficile</EM>. But in the end, the team voted and chose mRNA interference of <EM>C. difficile</EM> which became our project; <EM>Clostridium</EM> dTox.
			</P><DIV class="ui button" onclick="$('.bib').toggle()">Bibliography</DIV><DIV class="ui attahced secondary segment bib" style="display: none;">
				(1)House of Commons. (2017). Plastic bottles: Turning Back the Plastic Tide. Environmental Audit Committee. [online] Available at:https://publications.parliament.uk/pa/cm201719/cmselect/cmenvaud/339/339.pdf. [Viewed1stAugust2018].(2) Kadouri DandO'Toole GA.2005. Susceptibility of biofilms to Bdellovibrio bacteriovorusattack.Applied and Environmental Microbiology. 71(7):4044-51.(3) Koo H, Xiao J, Klein MIet al. 2010. Exopolysaccharides produced by Streptococcus mutansglucosyltransferases modulate the establishment of microcolonies within multispecies biofilms.Journal of Bacteriology.192(12):3024-32.(4) Lessa FC,GouldCV andMcDonaldLC. 2012. Current Status of Clostridium difficileInfection Epidemiology. Clinical Infectious Diseases. 55(2): S65-S70.(5) McDonaldLC, LessaF, SievertD et al. 2012.Vital signs: preventing Clostridium difficileinfections.Morbidity and Mortality Weekly Report (MMWR). 61(9):157-62.(6) Public Health England. 2018. Child oral health: applying All Our Health. [online] Available at:https://www.gov.uk/government/publications/child-oral-health-applying-all-our-health/child-oral-health-applying-all-our-health. [Viewed 1stAugust 2018].(7) RendulicS, JagtapP,RosinusAet al. 2004.A Predator Unmasked: Life Cycle of Bdellovibrio bacteriovorusfrom a Genomic Perspective. Science. 303(5658): 689-92.(8) Sockett REand Lambert C.Bdellovibrioas therapeutic agents: a predatory renaissance?Nature Reviews Microbiology. 2(8):669-75.
				(9) YoshidaS, HiragaK, TakehanaT et al. 2016.A bacterium that degrades and assimilates poly(ethylene terephthalate). Science. 351(6278): 1196-99.(10) Zhang S, Palazuelos-Munoz S, Balsells E et al. 2016. Cost of hospital management of Clostridium difficileinfection in United States—a meta-analysis and modelling study. BMC Infectious Diseases. 16(1): 447.

			</DIV></DIV><DIV class="ui segment"><H2>Project design</H2><H3><EM>C. difficile</EM>&amp; phage characterisation</H3><P><EM>C. difficile</EM> strain SBRC 078 was isolated previously in the SBRC from clinical faecal samples and belongs to the hypervirulent PCR ribotype 078. The strain contains the genes <EM>tcdA</EM> and <EM>tcdB</EM> encoding for both toxins. Phage phiSBRC was previously isolated in the SBRC from an environmental sample and can infect and form plaques on <EM>C. difficile</EM> SBRC 078. A lysogenic version of <EM>C. difficile</EM> SBRC 078, which contains phage phiSBRC integrated into the bacterial chromosome, was created previously in the SBRC.
</P><H4><EM>C. difficile</EM> growth analysis</H4><P>
The growth profile of the wildtype version of <EM>C. difficile</EM> SBRC 078 was compared to the growth profile of the lysogenic version of this strain. To assess this the growth of both strains was monitored for 24 hours and the OD at 600 nm was measured and the maximum growth rate was calculated using the equation</P><CENTER>
(2.303x(log10t2)–(log10t1))/3
</CENTER><P>where t1 is the OD at the start of exponential phase and t2 is the OD at the end of exponential phase. This data was used to inform the model parameters and was required to ensure that in the human gut the lysogenic bacterial strains created in this project would grow in the same manner as the wild-type cells and therefore would outcompete them.
</P><H4>Phage burst size</H4><P>
Phage burst size was assessed to determine the number of infectious phage particles produced per bacterial cell during one infection cycle. This was determined by measuring the number of infectious phage particles (in plaque forming units per ml) produced over a time-course after infection of <EM>C. difficile</EM> SBRC 078 with phiSBRC. The titre of free phage at each time point was determined by enumeration of plaques using a plaque assay. <EM>C. difficile</EM> SBRC 078 was infected to a multiplicity of infection (MOI) of 1. The burst size was calculated as the
Final Phage Titre/(Infection Phage Titre – Titre of Unbound Phages).
This data was used to inform the model parameters.
</P><H3>Promoter library</H3><P>
Our project aims to supress toxin production in <EM>C. difficile</EM> and we chose two different strategies to pursue this aim. Briefly, these strategies involve either dead-Cas9 (dCas9) or antisense RNA (asRNA) to inhibit toxin production. Both strategies will require a careful consideration of the genetic parts involved in the device. Of particular importance is the choice of promoter employed to control expression of the dCas9, guide RNA or antisense RNA. Tailoring expression to an appropriate level is often an important design consideration in genetic engineering. In our case it was thought that the use of a strong promoter would be of greatest benefit for both strategies we pursued in order to maximise either the amount of guide RNA and dCas9 or the amount asRNA. Concentration of these components within the cell was expected to correlate with the degree of toxin suppression and since the objective was to supress toxin to the greatest extent possible, we aimed to find and characterise strong promoters within <EM>C. difficile</EM>.   
</P><P>
The promoters we chose to characterise were as follows:
</P><DIV class="ui bulleted list"><DIV class="item"><A href="http://parts.igem.org/Part:BBa_K2715001"> BBa_J23106 (from iGEM registry) </A></DIV><DIV class="item"><A href="http://parts.igem.org/Part:BBa_K2715114"> BBa_J23114 (from iGEM registry) </A></DIV><DIV class="item"><A href="http://parts.igem.org/Part:BBa_K2715119"> BBa_J23119 (from iGEM registry) </A></DIV><DIV class="item"><A href="http://parts.igem.org/Part:BBa_K2715002">PCsp_fdx (from <EM>Clostridium sporogenes</EM>)</A></DIV><DIV class="item"><A href="http://parts.igem.org/Part:BBa_K2715001">PCac_thl (from <EM>Clostridium acetobutylicum</EM>)</A></DIV><DIV class="item"><A href="http://parts.igem.org/Part:BBa_K2715004">PCdi_TcdA (from <EM>C. difficile</EM>)</A></DIV><DIV class="item"><A href="http://parts.igem.org/Part:BBa_K2715003">PCdi¬_TcdB (from <EM>C. difficile</EM>)</A></DIV></DIV><P>
All seven promoters were intended to be assessed in both <EM>E. coli</EM> and <EM>C. difficile</EM>. PCsp_fdx and PCac_thl were chosen since they have been used extensively in studies on <EM>C. difficile</EM> as well as related organisms and both are considered to be strong promoters (Heap, 2018, Heap <EM>et al</EM>., 2009). A comparison of the two suspected strong promoters was made with the native promoters controlling toxin expression in <EM>C. difficile</EM> PCdi_TcdA and PCdi¬_TcdB. It was thought to be interesting and potentially useful to discover the strength of the toxin promoters and potentially their variance in their expression in different conditions. Three existing iGEM registry promoters were also chosen to be assessed in <EM>C. difficile</EM>. This served two functions, firstly it improved the registry in terms of part characterisation as there is currently no data on their use in Gram-positive organisms. Secondly, since these promoters have been well documented in <EM>E. coli</EM> they could give a good indication of the strength of the Clostridial promoters when used in <EM>E. coli</EM>. 
</P><P>
Occasionally cloning dCas9 <EM>in E. coli</EM> can be problematic, potentially due to unwanted off-target effects of the protein, the DNA binding nature of the enzyme or due to the size of the gene itself. To facilitate cloning and yet maximise dCas9 activity in <EM>C. difficile</EM> the ideal promoter would have low expression in <EM>E. coli</EM> and yet high expression in <EM>C. difficile</EM>. The choice of promoters and decision to assay them in both <EM>E. coli</EM> and <EM>C. difficile</EM> was designed to help us choose the optimal promoters for the toxin suppression projects, characterise existing iGEM registry parts in novel contexts and add  potentially valuable Clostridial/Gram-positive promoters to the registry.
Two different assays were chosen to assess the promoters described above. In <EM>E. coli</EM>, a GFP assay was chosen due to its widespread use, ease, cost, precision and reliability. However, GFP assays have not been successfully used in Clostridia, due to the requirement of oxygen for GFP, and as such other reporter assays are commonly used. One such reporter assay is the GusA assay in which the expression of the reporter gene <EM>gusA</EM> can be accurately measured via the eventual release of a fluorescent compound 4-methylumberlliferone (4-MU). The assay relies on the fact that the protein encoded by <EM>gusA</EM> is a glucuronidase which converts the non-fluorescent 4-methylumberlliferyl glucuronide (4-MUG) into the fluorescent (4-MU). 
</P><P>
GusA assays can be performed in <EM>E. coli</EM> as well as Clostridia and so both GFP and GusA assays were used in <EM>E. coli</EM>. Our GFP assay was inspired by our InterLab experience as we thought it would be useful to use the protocols and calibration curves we obtained from the study to standardise our data. This would help us give context to the strength of the promoters by comparing them to the InterLab positive and negative controls, using the calibrations curves generated through our InterLab study to ensure that the results would be reproducible by any other laboratory using different equipment. 
</P><P>
Heap, J. T. (2018). Stringency of Synthetic Promoter Sequences in Clostridium Revealed and Circumvented by Tuning Promoter Library Mutation Rates. https://doi.org/10.1021/acssynbio.7b00398
</P><P>
Heap, J. T., Pennington, O. J., Cartman, S. T., &amp; Minton, N. P. (2009). A modular system for Clostridium shuttle plasmids. Journal of Microbiological Methods, 78(1), 79–85. https://doi.org/10.1016/j.mimet.2009.05.004
</P><H3>CRISPRi</H3><H4>The origin of the CRISPR/Cas9 based genome editing tool</H4><P>
The CRISPR/Cas system is a naturally occurring defence mechanism in bacteria. It confers adaptive immunity against mobile genetic elements (MGEs), like phages. CRISPR-Clustered Regularly Interspaced Short Palindromic Repeats (CRISPR)- are short segments of repetitive DNA that are identical to each other. Due to the palindromic nature of the sequence, RNA transcribed from these repeats forms hairpin turns. These small CRISPR RNAs (crRNAs) are interspaced with unique DNA sequences known as spacer DNA. These DNA regions encode sequences complementary to MGEs introduced to the cell previously by phages. Upstream of the CRISPR array one can usually find the <EM>cas</EM> genes. These genes encode the Cas proteins that play an essential role during the different stages of bacterial immunity against intruding DNA. When bacteria are infected by a virus or phage, the CRISPR array is transcribed and the product, crRNA, forms a complex with (multiple) Cas protein(s), called the Cas-crRNA ribonucleoprotein complex. Depending on the organism different proteins are involved in this process. The crRNA guides the Cas protein(s) to the invading DNA which leads to degradation of dangerous virus or phage DNA before the infection has started.  
</P><H4>The CRISPR ‘revolution’</H4><P>
To date the CRISPR system has been adapted to form a new revolutionary gene editing tool, mostly known under the name CRISPR/Cas9. This technique relies on two main components: the short guide RNA (sgRNA) and the Cas9 protein. The sgRNAs can be split into two parts: the “seed” region, which is a 20 bp sequence that can be easily modified to be complementary to target a gene of interest, and a “handle” region which facilitates the binding of Cas9 to the sgRNA. The sgRNA functions as a ‘guide’ to ensure Cas9 creates double-strand breakage at a specific place in the DNA. In eukaryotes these double-strand breaks are glued back together via non-homologous end-joining (NHEJ) During NHEJ, a few nucleotides are removed from the site of cleavage which leads to the disruption of the gene. This disruption might affect or even completely abolish the activity of the protein it encodes which could lead to phenotypic changes. On the other hand, most bacteria cannot perform NHEJ and therefore die when Cas9 cuts the DNA as their genome cannot be replicated. However, if the target region has been successfully modified, the sgRNA will no longer guide the Cas9 protein to this region of the genome thus preventing cleavage. This makes the CRISPR/Cas9 system an ideal selection tool when editing the genome of bacteria as wild type cells will be killed whereas mutants will survive.
</P><P>
Initially, the CRISPR/Cas9 tool was used to generate knock-out and knock-in genes in various bacterial species. It was quickly realised that this system could easily be altered and so can be used for a much wider range of applications. Genetic modifications of the Cas proteins has allowed scientists to use this system to purify specific DNA regions, image DNA in living cells, introduce specific mutations in the DNA code, and activate or repress target genes, the latter being the most interesting in light of this project (<EM>Clostridium</EM> dTOX). In order to use the CRIPSR/Cas9 tool for the repression of protein expression, the RuvC and HNH nuclease domains were inactivated in the Cas9 protein resulting in a catalytically inactive protein, nuclease dead-Cas9 (dCas9). Instead of creating double-strand breaks, dCas9 (temporary) binds to a sgRNA complementary DNA sequence thus preventing the mRNA polymerase from binding/continuing transcription and in doing so preventing transcription. This process of reducing protein expression levels with dCas9 is called CRISPRi.
</P><DIV class="ui two column grid"><DIV class="ui column"><H6><STRONG>Diagram explaining the process of transcription and translation in bacteria.</STRONG></H6></DIV><DIV class="ui column"><H6><STRONG>Diagram explaining CRISPRi</STRONG></H6></DIV></DIV><H4>Why using CRISPRi for <EM>Clostridium</EM> dTOX</H4><P>
Ultimately the success of <EM>Clostridium</EM> dTOX depends on how efficiently the <EM>C. difficile</EM> toxins are repressed thereby turning <EM>C. difficile</EM> from a pathogenic (toxic) strain into a non-toxigenic strain. Rather than killing the <EM>C. difficile</EM> population in the gut completely, the creation of competition between non-toxic and toxic &gt;em&gt;C. difficile can stop the infection as well as preserve natural gut microbiota to prevent other opportunistic bacteria from populating the gut. CRISPRi has been shown to effectively decrease the production of proteins in cells and can be easily engineered to target different DNA regions making it a good candidate to target the promoter region of TcdA (PtcdA) and the promoter region of TcdB (PtcdB) with the aim of reducing toxin production. Moreover, multiple sgRNAs can be expressed and directed to each promoter thereby improving the tightness of the system.
</P><H4>Initial planning/preparation</H4><P><EM>C. difficile</EM> infection (CDI) symptoms range from mild diarrhoea to life-threatening pseudo-membranous colitis and toxic megacolon. They are essentially caused by the production of two toxins: TcdA and TcdB. The two toxin genes are part of the PaLoc gene cluster encoding three accessory proteins: TcdR, TcdC and TcdE. TcdR is a sigma factor essential for the initiating the translation of <EM>tcdA</EM> and <EM>tcdB</EM>. Besides controlling toxin expression, TcdR is also involved in the regulation of many other genes in <EM>C. difficile</EM>. Repressing the expression of TcdR might have an effect on the general cell fitness, therefore it was decided to target the promoter regions upstream of <EM>tcdA</EM> and <EM>tcdB</EM> instead thus minimizing the effect on other cell processes. Moreover, it has been shown that Cas9 binding to the promoter to inhibit the initiation of translation results in stronger repression of expression when compared to prematurely ending translation by targeting Cas9 to the gene sequence. Therefore, in this project, the sgRNAs are designed so that they direct Cas9 to the promoter region to inhibit the initiation of translation. 
</P><P>
Potential dCas9 targets within the PtcdA and PtcB promoter sequence were selected using the Benchling platform and converted into sgRNA sequences. For PtcdA, six sgRNAs were designed and, for PtcdB, five sgRNAs were selected. In order to treat CDI, both toxins should be repressed simultaneously because each toxin individually can cause CDI symptoms. Therefore it was decided to first test the sgRNAs for toxin A and toxin B separately in <EM>E. coli</EM> after which the best repressing sgRNAs for each toxin would be taken forward and combined in <EM>C. difficile</EM>. In <EM>C. difficile</EM> only the expression of sgRNAs and dCas9  is required as all other components necessary for the system to work are already present. The overall toxin repression can then be measured with a cytotoxicity assay.
</P><H4>Vector design details</H4><P>
	After thorough consideration, plasmid pMTL84121 was chosen to express our future construct as it contained all the necessary parts for successful cloning:
</P><DIV class="ui bulleted list"><DIV class="ui item"><STRONG>pCD6 gram+ replicon</STRONG> – for stable maintenance of the vector in <EM>C. difficile</EM></DIV><DIV class="ui item"><STRONG>p15a+tra gram- replicon</STRONG> – for stable maintenance of the vector in <EM>E.coli</EM></DIV><DIV class="ui item"><STRONG>catP gene</STRONG> – gives chloramphenicol resistance to bacteria and is used as a selection marker</DIV><DIV class="ui item"><STRONG>Multiple cloning sites (MCS)</STRONG> - provides cloning sites for inserts</DIV></DIV><P>
	To test our experiment in E. coli and C. difficile, we constructed the following vector: pMTL84121-PthI-dCas9-Pfdx-tcdR-T10-PtcdA/PtcdB-gusA-T14 + gRNA. 

</P><TABLE class="ui celled table"><THEAD><TR><TH>What</TH><TH>Why</TH></TR></THEAD><TBODY><TR><TD>pthI</TD><TD>Promoter for dCas9.</TD></TR><TR><TD>dCas9</TD><TD>Cas protein that binds to DNA and blocks transcription due to steric hindrance.</TD></TR><TR><TD>Pfdx</TD><TD>Promoter for tcdR. Promoter team results showed the strongest activity of this promoter in C.difficile than any other chosen promoter aimed at <EM>E. coli</EM> (link to asRNA)</TD></TR><TR><TD>tcdR</TD><TD>An important regulatory gene in C.difficile, which affects synthesis of toxins A and B as well as other important chemical reactions</TD></TR><TR><TD>T10</TD><TD>Terminator for tcdR. It produces termination of tcdR transcription.</TD></TR><TR><TD>PtcdA/PtcdB</TD><TD>Promoters for toxin A and B respectively.</TD></TR><TR><TD>GusA</TD><TD>Reporter gene</TD></TR><TR><TD>T14</TD><TD>Terminator for reporter gene GusA. It produces termination of GusA transcription.</TD></TR><TR><TD>gRNA</TD><TD>Constructed gRNA complementary to specific region of PtcdA/PtcdB.</TD></TR></TBODY></TABLE><P><STRONG>Experimental plan:</STRONG>
	After carrying our initial planning of the components for our experiment, we came up with a following experimental proposal:
	</P><DIV class="ui ordered list"><DIV class="ui item">Cloning of PthI-dCas9 insert into pMTL84121: pMTLdCas9</DIV><DIV class="ui item">Cloning of ptcdA/ptcdB-GusA-TB0014 into pMTL84121-PthI-dCas9: pMTSdCas9-GusA</DIV><DIV class="ui item">Cloning of pfdx-tcdR-TB0010 into pMTL84121-PthI-dCas9-ptcdA/ptcdB-GusA-TB0014</DIV><DIV class="ui item">Cloning of gRNA (A1-A7, B1-B5) into pMTL84121-PthI-dCas9-Pfdx-tcdR-T10-PtcdA/PtcdB-gusA-T14: pMTLdCas9-GusA-sgRNA</DIV><DIV class="ui item">Transformation of full vector pMTL84121-PthI-dCas9-Pfdx-tcdR-T10-PtcdA/PtcdB-gusA-T14 + gRNA into E.coli</DIV><DIV class="ui item">Checking the repression of GusA by targeting ptcdA/ptcdB with respective gRNA via GusA assay.</DIV><DIV class="ui item">Transform potentially supressing gRNAs into <EM>C. difficile.</EM></DIV><DIV class="ui item">Test gRNA with signs of repression in <EM>C. difficile</EM> via cytotoxicity assay.</DIV></DIV><H3>Antisense RNA</H3><P>
Antisense RNA (asRNA) is another means of reducing gene function. In this strategy a piece of RNA is transcribed which is complementary to the coding strand of a target gene in the reverse orientation, in other words it is the antisense to it. This has the effect of sequestering the coding mRNA in an RNA-RNA duplex which is unable to effectively bind the ribosome meaning protein synthesis of that gene is inhibited. The RNA-RNA duplex molecule may also be targeted by RNases specific to double stranded RNAs meaning the target mRNA is degraded faster due to the presence of asRNA. 
</P><P>
Many studies have utilised an asRNA approach to genetic studies. Typically asRNA will not be capable of completely eliminating gene function since target mRNAs may not encounter the asRNA molecule within the cell before being translated. For this reason, asRNA has primarily been used when gene ‘knockdowns’ are desired rather than total gene knockouts. In knockdown strains the expression of the target gene is reduced rather than entirely eliminated. Non-model organisms which are less genetically accessible frequently do not have established methods for creating gene knockouts. In this case asRNA is often used to gain initial insights into gene function since all that is required for asRNA studies is knowledge of the target gene sequence, a plasmid capable of replication in the organism and a means of transformation. 
</P><DIV class="ui two column grid"><DIV class="ui column"><H6><STRONG>Diagram explaining the process of transcription and translation in bacteria.</STRONG></H6></DIV><DIV class="ui column"><H6><STRONG>Rate of toxin production.&gt;Diagram explaining antisense RNA</STRONG></H6></DIV></DIV><P>
Our project aims to create a phage which will suppress toxin production in <EM>C. difficile</EM> once integrated into its genome. To do this we will genetically modify a phage known to infect strains of <EM>C. difficile</EM> named phiSBRC. The phage will be modified to include constructs to suppress toxin production either via a CRISPRi (dead-Cas9) approach or via asRNA. There were several design considerations when approaching this problem. Firstly the construct should be capable of significantly suppressing toxin production, it was not known whether dead-Cas9 or asRNA would be superior in this respect. Secondly, the eventual genetic construct we choose should be sufficiently small that we can alter the phage genome without adversely affecting its normal function. One potential limitation was thought to be the total amount of DNA which the phage could package into its head. In this respect asRNA could have a significant advantage over a dead-Cas9 approach since the total size of the genetic construct can be much smaller. Antisense RNA constructs can simply consist of a promoter and a short asRNA region of less than 100bp while the <EM>cas9</EM> gene alone is more than 4kb long. However, since it was not known which approach would produce more effective toxin suppression both approaches were pursued. 
</P><P><EM>C. difficile</EM> has two well characterised toxins which cause epithelial cells to undergo apoptosis these are TcdA and TcdB. It was thought that each construct we create should aim to supress both of these toxins simultaneously since research has concluded that each can operate independently from the other (Kuehne <EM>et al</EM>., 2010). The general form of our constructs therefore is to have asRNA parts downstream of promoters with a transcriptional terminator between these promoter-asRNA pairs. It was thought to use two different promoters to as to avoid unwanted recombination events within our constructs. As discussed previously, the optimal promoter for expressing the asRNA parts was thought to be the strongest promoter possible. We were therefore looking for the two strongest promoters in <EM>C. difficile</EM> we could find. Our results from the promoter assays we performed indicate that PCsp_fdx¬ is the strongest promoter assayed in <EM>C. difficile</EM>. Unfortunately we were unable to clone the GusA reporter plasmid for PCac_thl which was the other Clostridial promoter expected to give strong expression. However, the PCac_thl GFP reporter plasmid was created and assayed in <EM>E. coli</EM> where it gave the highest expression of any promoter tested including the positive control used in the iGEM InterLab studies. This result was consistent in our laboratory as well as that of our two collaborating teams representing Imperial College London and University of Warwick. The Clostridial promoter P¬Cac_thl has been routinely used in Clostridial research and outperforms every other promoter in <EM>E. coli</EM>, therefore it can be hypothesised that P¬Cac_thl would exhibit the strongest levels of expression in the GusA promoter assay in <EM>C. difficile</EM>, had this plasmid been constructed. As such PCac_thl and PCsp_fdx were chosen as the two strong promoters from which to express our asRNA parts.
</P><P>      
When choosing the length of the antisense RNA we consulted the scientific literature. There is some contradicting advice on this topic with <EM>E. coli</EM> asRNA parts seeming to be significantly shorter than those used in the few asRNA studies we found performed in clostridia. There are important design considerations here since there is a compromise to be made. Longer asRNA parts seem to generally give a greater degree of suppression but are also more likely to give unwanted off-target effects. This is probably because they can bind the target mRNA more tightly but are also more likely to have regions of short similarity with other non-target mRNAs within the cell. With this in mind we chose to try two different lengths of asRNA binding to the coding region of the target gene as well as the entire region upstream of the gene expected to include the ribosome binding site. ‘Construct One’ has a binding region of 24bp, this is the length suggested by a recent review paper on this topic (Hoynes-O’Connor &amp; Moon, 2016). ‘Construct Two’ has a  binding region of 50bp, this is much longer though still significantly shorter than the hundreds of base pairs previously used in clostridial studies (Desai &amp; Papoutsakis, 1999; Fagan &amp; Fairweather, 2011). Both of these constructs target both of the toxin genes we are interested in. 
</P><H5>Construct One diagram</H5><H5>Construct Two diagram</H5><P>  
Desai, R. P., &amp; Papoutsakis, E. T. (1999). Antisense RNA strategies for metabolic engineering of Clostridium acetobutylicum. Applied and Environmental Microbiology, 65(3), 936–945. 
Fagan, R. P., &amp; Fairweather, N. F. (2011). Clostridium difficile has two parallel and essential sec secretion systems. Journal of Biological Chemistry, 286(31), 27483–27493. https://doi.org/10.1074/jbc.M111.263889 
Hoynes-O’Connor, A., &amp; Moon, T. S. (2016). Development of Design Rules for Reliable Antisense RNA Behavior in E. coli. ACS Synthetic Biology, 5(12), 1441–1454. https://doi.org/10.1021/acssynbio.6b00036 
Kuehne, S. A., Cartman, S. T., Heap, J. T., Kelly, M. L., Cockayne, A., &amp; Minton, N. P. (2010). The role of toxin A and toxin B in Clostridium difficile infection. Nature, 467(7316), 711–713. https://doi.org/10.1038/nature09397 </P></DIV><DIV class="ui segment"><H2>Results</H2><H3><EM>C. difficile</EM>&amp; phage characterisation</H3><H4><EM>C. difficile</EM> growth analysis</H4><P>
The growth of wild-type <EM>C. difficile</EM> SBRC 078 was compared with the growth of the lysogenic version of this strain. By measuring the OD at 600 nm of the two bacterial cultures over a 24-hour period. It was determined that the lysogenic strain has a slightly longer lag phase than the wild-type strain (Figure 1) but both strains reached the same maximum OD. The maximum growth rates for the two strains were similar with wild-type reporting a maximum growth rate of 0.26 µ/h and the lysogen measuring 0.23 µ/h. The negative control (broth containing no bacteria) reported a maximum growth rate of 0.007 µ/h showing no contamination over the time-period. The similar growth rates of the wild-type strain and lysogen showed that when lysogens were created in the gut as part of the therapy, the lysogens would be able to maintain their population over time in the same way as wild-type strains, therefore ensuring they are able to compete for nutrients and act as a probiotic to reduce colonisation of incoming toxigenic <EM>C. difficile strains</EM>.
</P><CENTER><H6 align="justify"><STRONG>Figure 1: Growth profile of wild-type <EM>C. difficile</EM> SBRC 078 and lysogenic <EM>C. difficile</EM> SBRC 078.</STRONG> The lysogenic strain has a slightly longer lag phase but both strains reach the same maximum OD. The negative control contains no bacteria and shows that no contamination has occurred over the time-period. OD was measured every hour for 24 hours in biological triplicate.
</H6></CENTER><H4>Phage burst size</H4><P>
The phage burst size was calculated to determine the number of infectious phage particles produced from one bacterial cell during one infection cycle. A culture of <EM>C. difficile</EM> SBRC 078 was infected with phiSBRC to a MOI of 1 (infection titre of phage of 1.38 x 106 pfu/ml) and incubated for 15 minutes to allow phages to adsorb. The culture was washed of any unbound phages and then incubated under anaerobic conditions for 80 minutes. The number of phages in the supernatant was monitored. Figure 2 shows the number of phages present at various intervals over the period. It was observed that between 65 and 70 minutes and then 75 and 80 minutes the phage titre seemed to plateau slightly, indicating the end of the first burst cycle. It was determined by plaque assay that the final phage titre at the end of the first burst cycle was 4.2 x 106 pfu/ml and the number of unbound phages after the 15-minute incubation was 1.05 x 104 pfu/ml. These values were used to calculate burst size which was determined as 33 phage particles per cell. The burst size was a useful parameter for the phage model and allowed the number of phages over time in the model to be more accurately determined.
</P><CENTER><H6 align="justify"><STRONG>Figure 2: Determination of phage phiSBRC burst size.</STRONG><EM>C. difficile</EM> SBRC 078 was infected with phiSBRC and the subsequent burst was measured over 80 minutes. The first burst cycle is deemed complete when the phage titre, measured in plaque forming units per ml, reaches a small plateau. The burst size was calculated and determined as 33 phage particles per cell.</H6></CENTER><H3>Promoter library</H3><H4>GFP assay in <I>E. coli</I></H4><P>Our first objective was to quantify promoter strength in <I>E. coli</I>. To do this we synthesised each promoter upstream of wild type <I>GFP</I> [BBa_E0040] with the biobrick prefix/suffix flanking the part. The terminal restriction enzymes for the prefix and suffix were then used to clone the part into an empty pMTL84151 vector. pMTL84151 contains the ColE1 origin of replication for use in Gram-negative organisms as well as a Gram-positive origin of replication so that it can be used as a shuttle plasmid between <I>E. coli </I>and <I>C. difficile</I>. </P><P>The method used for our GFP assay can be found in our Labfolder. Briefly, reporter plasmids were transformed into <I>E. coli </I>DH5α after which an overnight culture was grown. The overnight culture was used to inoculate test cultures which were normalised to the same starting optical density (OD). After six hours of growth at 37°C shaking at 220 RPM the optical density was measured and cell samples were taken. Fluorescence was calculated using excitation at 485 nm and measuring emission at 525 nm. This fluorescence value was then normalised to the optical density to give fluorescence per OD unit. The positive [BBa_I20270] and negative [BBa_R0040] controls used were the same as those used in the 2018 InterLab study. </P><P>The strongest promoter of our set was P<SUB>Cac_thl </SUB>(0.3235µM) which was around three times stronger than the positive control [BBa_I20270]. The second strongest clostridial promoter we tested (P<SUB>Csp_fdx</SUB>)was slightly less strong than the positive control. The remaining two clostridial promoter P<SUB>Cdi_tcdA </SUB>and P<SUB>Cdi_tcdB </SUB>only weakly expressed in <I>E. coli</I>, though both showed activity significantly higher than the negative control [BBa_R0040]. The three existing parts we characterised with a novel RBS showed a range of expression levels as expected [BBa_J23106, BBa_J23114, and BBa_J23119]. We conclude that these promoters are capatible with the novel RBS part [Bba_K2715009] we added to them. </P><CENTER><H6 align="justify">
Figure 1. Relative expression of GFP from each of the seven promoter constructs in <EM>E. coli</EM> from a six hour culture. The positive and negative controls are those used in the Interlab study, BBa_I20270 and BBa_R0040 respectively. The results represent the mean of four technical replicates with error bars representing the standard error of the mean. All values were calculated using calibration curves generated in the Interlab study, corrected for optical density and reported relative to the equivalent μM of fluorescein. 
</H6></CENTER><H4>GusA Assay in <I>C. difficile </I></H4><P>As we need to establish data for promoter strength in <I>C. difficile</I> as well as <I>E. coli</I>, we needed to find an alternative assay that would work for anaerobic organisms. The GFP protein requires oxygen in order to form properly and fluoresce meaning it is not suitable as a reporter gene in this context. The Β-Glucuronidase encoded by <I>gusA</I> functions as a very sensitive and specific reporter gene in anaerobic organisms. The seven plasmids used in the GFP assay were modified to replace the <I>gfp </I>gene with <I>gusA </I>using NdeI and SpeI. For six of the seven <I>gusA </I>reporter plasmids cloning was successful with sequencing of the <I>gusA </I>showing no mutations. However, every clone of P<SUB>Cac_thl </SUB>–<I>gusA </I>exhibited mutations within <I>gusA</I>. This is likely because of the toxin impact of <I>gusA </I>on <I>E. coli</I> and the very high expression levels of P<SUB>Cac_thl </SUB>as shown in the GFP assay. </P><P>The six successfully cloned <I>gusA </I>reporter plasmids were transformed into a conjugative strain of <I>E. coli </I>named Sexpress. Sexpress was then used to transfer the reporter plasmids into <I>C. difficile</I> where the gusA assay proceeded. </P><P>The gusA assay was performed in 96 well plates in a plate reader using a similar protocol to the interlab study, except that 16 hour cultures were used rather than 6 hour, and the cells were lysed using lysozyme and sonication. The substrate for the Β-Glucuronidase, 4-methylumbelliferyl glucuronide (4-MUG), was added to the cell lysate immediately before the assay, and the production of the fluorescent compound 4-methylumbelliferone (4-MU) was tracked over 30 minutes. The rate of change in absorbance over this time period was recorded by the plate reader, and the gradient of the linear portion of the curve was calculated in order to give an indication of the quantity of Β-Glucuronidase in each strain. The results of the gusA assay are shown in the graph below. </P><P>None of the three existing registry promoters (BBa_J23106, BBa_J23114, and BBa_J23119) showed any detectable level of expression. This is somewhat unsurprising given that they are native to the distantly related <I>E. coli</I>, though they do have a ribosome binding region from clostridia which was intended to allow them to function in <I>C. difficile</I>. Of the two toxin promoters native to <I>C. difficile </I>only P<SUB>Cdi_tcdA </SUB>showed any detectable activity and this was approximately seven times lower than the activity of P<SUB>Csp_fdx</SUB>. This could be due to the assay being performed in BHIS medium which contains compounds which suppress the promoter. A research paper suggests that these toxin promoters are subject to catabolite repression (Dupuy et al., 2011). The strongest promoter in <I>C. difficile </I>was clearly shown to be P<SUB>Csp_fdx</SUB>. </P><CENTER><H6 align="justify">
Figure 2. Relative expression of GusA from each of the six promoter constructs in <EM>C. difficile</EM> from a 16 hour mid-exponential culture. The negative control used was a <EM>C. difficile</EM> strain containing a modular vector pMTL84151 which contained no promoter. The results represent the mean of four technical replicates with error bars representing the standard error of the mean. Culture samples were harvested by centrifugation, cell pellets were resuspended in 500 μL of a suitable buffer, lysed by sonication, and 75 μL of the cell suspension was reacted with 28.4 μM of 4-methylumbelliferyl-β-D-glucuronide. Fluorescence intensity was monitored over a period of 30 min at 440-460 nm using an excitation wavelength of 355-375 nm, and the rate of change in intensity per minute was calculated as a measure of β-Glucuronidase activity, encoded by the protein GusA.  
</H6></CENTER><P>Antunes, A., Martin-verstraete, I., &amp; Dupuy, B. (2011). CcpA-mediated repression of Clostridium difficile toxin gene expression, 79(December 2010), 882–899. https://doi.org/10.1111/j.1365-2958.2010.07495.x</P><H3>CRISPRi</H3><P>
The part, pMTLdCas-GusA PthI-dCas9 was confirmed by conducting PCR and the DNA was extracted from <EM>E. coli</EM> for Sanger sequencing which confirmed that the inserted parts are correct (figure 1).
</P><CENTER><H6>
Figure 1. Sanger sequencing data of PthI-dCas9 inserted into pMTL84121. The red bars above the black middle line are sequencing reads mapped to the the dcas9 gene (light green) and Pthl promoter (dark green). The red bars show that there is no mutation with our inserts.
</H6></CENTER><P>
However, inserting PtcdB and the sgRNAs into the vector proved to be quite troublesome. It was repeatedly tried to insert PtcdB into the vector unfortunately sequencing results would always show mutations or rearrangements (Figure 2). Most likely these mutations can be attributed to the fact that high levels of <EM>gusA</EM> are toxic to <EM>E. coli</EM>. This has been previously observed with the promoter library construct Pthl-<EM>gusA</EM>.
</P><P>
In the original design the sgRNAs would be cloned downstream of the <EM>gusA</EM> gene. Unfortunately, several attempts to ligate sgRNAs with the vector pMTLdCas9-GusA were unsuccessful. Single digests were carried out and confirmed that the <EM>Asc</EM>I restriction enzyme did not cut the vector (figure 3). Repeating the same digestion with several <EM>Asc</EM>I restriction enzyme stocks did not resolve the problem, indicating that the <EM>Asc</EM>I restriction site has a mutation, this was later confirmed with Sanger sequencing.
</P><CENTER><H6>
Figure 2. Single digestion of pMTL84121 with <EM>Xho</EM>I or <EM>Asc</EM>I and it is proved that <EM>Asc</EM>I restriction site has a mutation.
</H6></CENTER>

Due to the mutated <EM>Asc</EM>I restriction site, a two-vector based approach was adopted. The sgRNAs were successfully cloned into two different vectors: pMTL82251 (<EM>erm</EM> resistance gene, high copy replicon) and pMTL71401 (<EM>kan</EM> resistance gene, low copy replicon). Subsequently both the dcas9-<EM>gusA</EM> vector and one of the sgRNA vectors were co-transformed into <EM>E. coli</EM>. In addition, the dcas9-<EM>gusA</EM> vector was co-transformed with the empty pMTL82251 or empty pMTL71401 vector. These strains (Ec and Kc respectively) were used as positive controls in the GusA assay. These controls were created because different antibiotics might impact the growth rate and thus the production of GusA in the various strains. In addition, the difference in copy number between the two vectors might affect the level of repression. 

<P>
In order to establish the repression efficiency of the various sgRNAs an enzymatic assay was performed. The assay was repeated twice, harvesting the cells at different stages of bacterial growth; exponential and stationary growth phase. Cells at exponential phase were harvested when the cultures reached an OD600 of 1.0 and cells at stationary phase were harvested after overnight growth. 
</P><CENTER><H6 align="justify">
Figure 3. Repression efficiency of sgRNA variants (A1-6) was measured using a β-Glucuronidase activity assay in <EM>E. coli</EM> strains harbouring pMTL82251-sgRNA (A, B) or pMTL71401-sgRNA (C, D) plasmid backbones. Strains harbouring vectors pMTL82251 –sgRNA (A, B) or pMTL71401-sgRNA (C, D) were used as controls, expressing dCas9 but no sgRNA. Briefly, strains were grown in media with the appropiate antibiotics and overnight cultures (A, C) or cultures at the mid-log growth phase [OD600 ≈ 1.0] (B, D) were harvested by centrifugation. Subsequently, cell pellets were re-suspended in 500 μL of a suitable buffer and 75 μL of the cell suspension were reacted with 28.4 μM of 4-methylumbelliferyl-β-D-glucuronide. Fluorescence intensity was monitored over a period of 10 min at 440-460 nm using an excitation wavelength of 355-375 nm. Data represent mean values of three technical replicates ± SD. Statistical analysis was carried out using one-way ANOVA with Dunnett’s test for multiple comparisons against the control strain (c); p-values are indicated as: 0.1234 (ns), 0.0332 (*), 0.0002 (***), &lt;0.0001 (****).
</H6></CENTER><P>
The sgRNAs tested exhibited different levels of repression for the PtcdA promoter (Figure 4). sgRNAs A2, A3, A4 and A6 all repressed <EM>gusA</EM> expression, although with different efficiencies. For the sgRNAs expressed from the pMTL82251 vector sgRNA A4 and A6 have the biggest effect on <EM>gusA</EM> expression, whereas sgRNAs A1 and A5 do not seem to repress (Figure 4-A, B). The repression efficiencies observed for the sgRNAs in the pMTL71401 vector have a similar profile as observed for the pMTL82251 vector; sgRNAs A3 and A4 significantly reduce the detected enzyme activity whereas sgRNAs A1 and A5 do not effect <EM>gusA</EM> expression (Figure 4-C, D). Moreover, no clear difference is observed between the samples harvested in the exponential growth phase and the samples harvested at the stationary growth phase. Suggesting that with the GusA reporter the level of repression is quite stable and not influenced by the accumulation of protein inside the cell. There is only one exception, pMTL82251 sgRNA A5 has a negative effect on the <EM>gusA</EM> activity measured at OD600 of 1.0 but not at the stationary phase. Furthermore, pMTL71401 sgRNA A5 has no effect on <EM>gusA</EM> expression. This suggest that the observed reduced expression of <EM>gusA</EM> for pMTL82251 sgRNA A5 during exponential growth is most likely due to an error in the assay. Only sgRNA A4 seems to behave differently when expressed from a different vector; showing strong repression in pMTL82251 and medium repression in pMTL71401. The observed discrepancy could be due to the difference between the copy number of the two vector backbones or a detrimental mutation in the PtcdA promoter or the <EM>gusA</EM> gene which inactivates the promoter or disrupts the <EM>gusA</EM> gene.
</P><P>
In the future we would like to repeat the GusA assay to characterise the activity of sgRNAs targeting PtcdB. Subsequently, the best sgRNAs to repress the PtcdA and PtcdB promoter can be expressed together with dCas9 in <EM>C. difficile</EM> to determine their effectiveness in repressing toxicity. In addition, different combinations of sgRNAs can be used in these assays to potentially increase the tightness of the system. Since we were not able to transform our construct into <EM>C. difficile</EM> but it has been proven that our construct shows the expected results, transforming the construct into <EM>C. difficile</EM> and conducting cytotoxicity assay will be ideal. 
</P><H3>Antisense RNA</H3><P>
The ultimate objective was to incorporate the described asRNA system suppressing two <EM>C. difficile</EM> toxins into the phiSBRC prophage of <EM>C. difficile</EM>. The edited prophage could then be prepared from a stock <EM>C. difficile</EM> culture and used as a phage therapy treatment on patients suffering from <EM>C. difficile</EM> infections. To first demonstrate the efficacy of the asRNA constructs at suppressing toxin production the two constructs we created were cloned into a plasmid vector suitable for transforming <EM>C. difficile</EM>. The <EM>C. difficile</EM> cultures harbouring asRNA plasmids were compared to wild type <EM>C. difficile</EM> in terms of supernatant cytotoxicity using African green monkey kidney epithelial cells of the ‘Vero’ lineage. <EM>C. difficile</EM> cultures were monitored over five days in terms of optical density as a read-out for bacterial growth and samples were taken, centrifuged and the supernatant filter sterilised in preparation for the cytotoxicity assay. 
</P><P>
Cell supernatants of <EM>C. difficile</EM> contain the two toxins of interest TcdA and TcdB which are capable of stimulating mammalian epithelial cells to undergo apoptosis. It was thought that the supernatants from cultures containing our two asRNA constructs would have a lower concentration of toxins and therefore produce lower cytotoxic effects on the vero cells. Vero cells were grown in a 96-well cell culture plate using Dulbecco’s modification of Eagle medium (DMEM) supplemented with 10% heat-inactivated fetal bovine serum (FBS). After a confluent monolayer of epithelial cells was formed the sterile <EM>C. difficile</EM> supernatant was applied and the cells incubated for 24 hours at 37°C with 5% CO2. After incubation the medium-supernatant solution was taken and added to the LDH master mix solution, incubated in the dark at room temperature for 30 minutes before the absorbance at 492 nm was measured. Absorbance at 492 nm is a readout for cell death due to the released lactate dehydrogenase from lysed cells reducing NAD+ to NADH/H+ which is then used to reduce a tetrazolium salt into formazan. The formazan dye produced gives an absorption maximum at 492 nm and since the concentration of formazan correlates with the amount of lactate dehydrogenase released by the cells it can be used as a measurement of cytotoxicity. 
</P><P>
Our results show that the supernatant toxicity of wild type <EM>C. difficile</EM> appears to plateau at 48 hours with no further increases observed. This plateau effect is likely produced by the concentration of toxin in the supernatant overcoming a threshold whereby the assay is no longer sensitive to any increases in toxicity. Both asRNA construct containing cultures take around 120 hours to reach this plateau of toxicity as their rate of toxin production is significantly lower. The rate of toxin production was taken as the OD-normalised LDH assay 492 nm absorbance value divided by the number of hours that the sample had been growing. Using this formula the wild type culture exhibited a toxin production rate of 0.0506 arbitrary units whilst construct one and two produced 0.0102 and 0.0074 respectively. Comparing these rates reveals that the asRNA construct one reduces the toxin production rate by 79.8% and construct two reduces the toxin production rate by 85.3%. 
</P><DIV class="ui two column grid"><DIV class="ui column"><H6><STRONG>Cytotoxicity of <EM>C. difficile</EM> supernatants.</STRONG>The graph shows supernatant cytotoxicity over a period of 120 hours. There is considerably less toxin production by <EM>C. difficile</EM> containing asRNA construct 1 and <EM>C. difficile</EM> containing asRNA construct 2 than by wild type <EM>C. difficile</EM>.</H6></DIV><DIV class="ui column"><H6><STRONG>Rate of toxin production.</STRONG><EM>C. difficile</EM> containing asRNA construct 1 and <EM>C. difficile</EM> containing asRNA construct 2 exhibit a significantly slower rate of toxin production than wild type <EM>C. difficile</EM>. Here we see an 80% reduction in the rate of toxin production by <EM>C. difficile</EM> containing asRNA construct 1 and an 85% reduction in the rate of toxin production by <EM>C. difficile</EM> containing asRNA construct 2.</H6></DIV></DIV></DIV><DIV class="ui segment"><H2>Conclusion</H2><P>
Our project aimed to show that <EM>C. difficile</EM> toxin production could be reduced with genetic constructs which could then be incorporated into a phage which targets strains of <EM>C. difficile</EM>. 
</P><P>
The first step toward this end was to characterise a recently discovered phage in terms of its infectivity parameters. Phage phiSBRC was demonstrated to infect the <EM>C. difficile</EM> SBRC 078 strain effectively with the plaque/burst size assay showing that 33 phage particles are released per <EM>C. difficile</EM> cell. This result was used as a parameter in our modelling work. Another important parameter needed for the model was the growth rate of <EM>C. difficile</EM> wild type compared with the <EM>C. difficile</EM> lysogen in which the phiSBRC phage has integrated into the <EM>C. difficile</EM> genome. The respective growth rates were calculated by tracking the growth of each culture. It was concluded that there was little difference in the growth rate between <EM>C. difficile</EM> and the lysogen. 
</P><P>
Having demonstrated that phiSBRC would be a suitable phage for infecting toxic <EM>C. difficile</EM> we next wanted to design a genetic construct which would be capable of suppressing toxin production. The two approaches we considered for this were dCas9 and asRNA. Both of these approaches required the use of strong, constitutive promoters. For this reason the next step for us was to characterise a range of promoters for strength in <EM>C. difficile</EM>. Whilst achieving this goal we also decided it would be beneficial to attempt to improve the characterisation of existing registry parts by measuring their expression in a novel organism. <EM>C. difficile</EM> is a Gram-positive anaerobic organism with significant differences to the <EM>E. coli</EM> chassis for which existing characterisation was performed. The existing registry promoters BBa_J23114, BBa_J23106, and BBa_J23119 were characterised for expression strength using a GusA assay in <EM>C. difficile</EM>. A new registry part which represents the ribosome binding region from the thiolase gene of <EM>Clostridium acetobutylicum</EM> was added to these promoter regions allowing them to be characterised in the context of having a different RBS than previously. In addition, four promoters have been added to the iGEM registry from <EM>C. acetobutylicum</EM> (Pcac_thl) [Bba_K2715010], <EM>C. sporogenes</EM> (PCsp_fdx) [Bba_K2715011] and two from <EM>C. difficile</EM> (PCdi_TcdA) [Bba_K2715012] and (PCdi_TcdB) [Bba_K2715013]. 
</P><P>
The four novel registry parts were characterised alongside the existing registry promoters in a GFP assay in E. coli as well as in a GusA assay in <EM>C. difficile</EM>. The most remarkable conclusion from the <EM>E. coli</EM> GFP assay of these promoters is that both of the suspected strong <EM>C. difficile</EM> promoter PCsp_fdx and Pcac_thl were stronger than any of the three existing registry promoters we assayed; with Pcac_thl producing around three times the concentration of fluorescein (0.3235µM) as the positive control used in the InterLab studies (0.0958µM). 
</P><P>
Our main objective in characterising these promoters was to find a suitable pair of strong promoters to use in our subsequent dCas9 or asRNA projects. For this the GusA assay within <EM>C. difficile</EM> was most relevant since this is the chassis in which these constructs would be acting. The <EM>C. difficile</EM> GusA assay clearly showed that none of the three existing registry promoters from <EM>E. coli</EM> had any detectable activity in <EM>C. difficile</EM>. By far the strongest promoter we were able to measure was PCsp_fdx which was around 7.5 times stronger than the next strongest promoter we found (PCdi_TcdA). We were unable to clone the strongest promoter from the <EM>E. coli</EM>  GFP assay PCdi_thl into a GusA reporter plasmid. This is likely because of the toxicity of the gusA gene in E. coli and since we know that  PCdi_thl is the strongest of our promoters in E. coli it is unsurprising that this was the most problematic plasmid to construct. As a result we did not measure the strength of PCdi_thl in <EM>C. difficile</EM>, but due to its measured strength in <EM>C. difficile</EM> as well as its widespread use for overexpression studies in Clostridia we decided to select it alongside PCsp_fdx as a promoter to use in the next stage of our project. 
</P><P>
Two asRNA constructs were cloned named ‘asRNA Construct One’ [Bba_K2715007] and ‘asRNA Construct Two’ [Bba_K2715008]. Both of these constructs target both toxin genes TcdA and TcdB with asRNA parts of varying length with asRNA Construct Two having longer regions of homology with 50bp of coding region verses 24bp for asRNA Construct One. These constructs were designed with the promoter results in mind, selecting the suspected two strongest promoters in <EM>C. difficile</EM>. Both constructs were assessed in terms of their ability to reduce <EM>C. difficile</EM> culture supernatant cytotoxicity on mammalian ‘Vero’ cells. The rate of toxin production was decreased by 80% and 85% by asRNA Construct One and asRNA Construct Two respectively. The main conclusion to draw from this result is that an asRNA strategy is viable for reducing <EM>C. difficile</EM> strain toxicity. Another conclusion of note is that having a longer region of homology with the target gene does seem to impact on the effectiveness of suppression significantly since asRNA Construct Two has a 5% greater effect whilst having 26bp extra of homology per toxin gene. 
</P><P>
The other approach to suppressing toxin production was via a nucleolytically inactive Cas9 (dCas9). Demonstration of this approach did not progress as far as with asRNA because the cloning stage of this project was more time-consuming. While asRNA demonstrated a <EM>C. difficile</EM> supernatant with reduced cytotoxicity, our dCas9 approach was only validated in <EM>E. coli</EM>. However, positive results were obtained and future work should continue to pursue this approach. Six guide RNAs were evaluated in terms of their ability to target dCas9 to the toxin promoter region for toxin A (PtcdA). PtcdA was placed in control of the reporter gene <EM>gusA</EM> allowing quantification of the effectiveness of each guide RNA. Out of the six guide RNAs tested guide RNA 6 displayed the most consistently promising results with significantly less Gus activity implying that this guide recruits dCas9 to the PtcdA promoter region most effectively. Therefore guide RNA 6 will be used in future work when the dCas9 approach is trialled within <EM>C. difficile</EM> for its ability to reduce toxin production. 
</P></DIV><DIV class="ui segment"><H2>Future Work</H2><P>			
Since we have demonstrated the effectiveness of asRNA at reducing <EM>C. difficile</EM> toxicity in this project, the obvious next step is to integrate our toxin suppressing construct into phiSBRC. This will involve taking the <EM>C. difficile</EM> lysogen with phiSBRC integrated into the genome and modifying it in the same way as we would modify the <EM>C. difficile</EM> genome normally. A recent paper (Wang et al., 2018) has described genome modification of <EM>C. difficile</EM> using Cas9 as a counter-selection mechanism forcing the cell to undergo homologous recombination with the delivered knockout plasmid to escape the lethal effects of Cas9. The recombination event which allows the cell to avoid the lethal double stranded break caused by Cas9 is directed by homology arms delivered on the knockout plasmid allowing researchers to delete genomic regions or introduce novel DNA into the genome. With this approach in mind we designed the plasmid pSBRC_Cas9_PhageIntegration_holin. This plasmid contains asRNA Construct Two which reduced toxin production by 85%, between homology arms directed at a gene within the phiSBRC prophage. The phiSBRC gene we chose to target was a holin gene which is thought to be responsible for cell lysis. Without this gene the progeny phage particles will not be able to burst out of the bacterial cell. This gene was chosen because it is one of the few areas of the genome which we are confident in ascribing function to and that function is not required to prepare more of the modified phage. Even without the phage being able to exit the bacterial cell it can still be induced and replicate itself and from there we can artificially extract phage particles ready for re-infection or delivery as a therapeutic. The other reason the holin gene was chosen is because of concerns around the size of phage genome which can be successfully packaged. It may be that the phage has evolved to be at or near to the limit of DNA which it can package. In this case replacing the holin gene which is of a similar size to asRNA Construct Two would mean that this is no longer an issue. 
</P><P>
After knocking out the holin gene whilst simultaneously introducing asRNA Construct Two we would have a lysogenic strain of <EM>C. difficile</EM> with the modified phiSBRC integrated within the <EM>C. difficile</EM> genome. The asRNA Construct Two should still be active within the genome, as it would be constitutively expressed, and have a similar toxin suppressing effect to that demonstrated on a replicative plasmid in our results section. The cytotoxicity assays performed earlier will have to be repeated with the modified phiSBRC prophage taking the place of the replicative vector to ensure that the toxin suppression effect remains. It may be the case that since the asRNA construct on the genome is at a lower copy number than on a replicative vector it no longer displays such powerful toxin suppressing effects. 
</P><P>
Having verified that the modified phiSBRC prophage retains its impact on toxin suppression the next stage would be to generate a second modified phiSBRC prophage which does not remove the phage holin gene. This is necessary because the modelling results suggest that having the phage able to occasionally enter the lytic cycle would be beneficial when put into practice. Instead of targeting the holin gene a region of non-coding DNA would be found and targeted with different homology arms to those used previously in pSBRC_Cas9_PhageIntegration_holin. Once the new modified prophage is created it would be necessary to ensure that the phage retains its ability to infect <EM>C. difficile</EM> and undergo the lytic cycle. For this reason a plaque assay would be performed as previously with the wild type phiSBRC and any difference in phage parameters would be re-entered into the mathematical model. 
</P><P>
After this research is complete we would have a <EM>C. difficile</EM> lysogen containing a modified prophage which has been demonstrated to suppress toxin. This lysogen could be used to generate pure infectious phage particles which could be used in phage therapy. The next factor to consider would be the means of delivery to patients. After consulting with experts and discussion groups as detailed in the human practices it was decided that a capsule would be the optimal delivery method. As such the final stage of research in future work would be optimisation of the encapsulation of phage particles ready for application to patients. 

</P><P>
Wang, S. et al. 2018. “Genome Engineering of Clostridium Difficile Using the CRISPR-Cas9 System.” Clinical Microbiology and Infection. https://doi.org/10.1016/j.cmi.2018.03.026.
</P></DIV><DIV class="ui segment"><H2>InterLab</H2><P>			
The aim of the iGEM InterLab study is to work towards a more reliable and repeatable measurement system to make synthetic biology an engineering biology. All participating laboratories first calibrated their instruments by obtaining standard curves using sodium fluorescein which was provided in our kits. This allowed us to fix settings such as top/bottom optic, gain and type of plate used so that all conditions were the same for our GFP and CFU protocols.
</P><CENTER><H6>
Table 1 shows data for the OD600 reference point.</H6></CENTER><CENTER><H6>
Table 2 shows data for the particle standard curve (standard curve below).</H6></CENTER><CENTER><H6>
Table 3 shows data for fluorescein standard curve (standard curve below).</H6></CENTER><P>
(Blue cells were raw data measurements, gold cells were calculated)
After the machine was calibrated fluorescence and absorbance of GFP in 8 different devices was measured using the standardised method provided by iGEM. The results gathered were suggestive of which device had the highest strength of gene expression. The data collection sheet provided converted our raw plate reader measurements into arbitrary fluorescein and abs600 values as well as calculating the µM fluorescein per OD as shown in the following tables. 
</P><CENTER><H6>
Tables 4 and 5 show calculated values of µM fluorescein per OD at 0 hour and 6 hour time points. This was calculated from OD measurements we took before beginning the assay.</H6></CENTER><CENTER><H6>
Tables 6 and 7 show calculated arbitrary values of net fluorescein at 0 hour and 6 hour time points.</H6></CENTER><CENTER><H6>
Tables 8 and 9 show net values of absorbance of light (600nm) at 0 hour and 6 hour time points.</H6></CENTER><P>
What did we gain from our InterLab experience? 
The InterLab study has allowed us to contribute to making synthetic biology more accurate through providing a series of standardised procedures of calibration and measurement tests. It helped our project as we were then able to use our new calibrated programmes to compare our promoters GFP assay data with iGEM standard data. 
</P></DIV><DIV class="ui segment"><H2>Labfolder</H2><P> Protocols </P><P> Lab book antisense RNA experiments </P><P> Lab book promoter experiments </P><P> Lab book dCas9 experiments </P></DIV></DIV><DIV class="ui sticky rail sidebar-list fixed top" style="left: 905.5px; width: 324.5px !important; height: 373px !important; margin-top: 30px; top: 0px;"><DIV class="ui vertical fluid inverted green menu appear" style="opacity: 1; transform: none;"><A class="item active" id="ideas-button" href="#ideas">
			    Ideas
			  </A><A class="item" id="design-button" href="#design">
			    Project design
			  </A><A class="item" id="results-button" href="#results">
			    Results
			  </A><A class="item" id="conclusion-button" href="#conclusion">
			    Conclusion
			  </A><A class="item" id="future-work-button" href="#future-work">
			    Future Work
			  </A><A class="item" id="interlab-button" href="#interlab">
			    Interlab
			  </A><A class="item" id="labfolder-button" href="#labfolder">
			    Labfolder
			  </A></DIV></DIV></SECTION><DIV class="ui container PageCard_container_1ok2Drx5q2skrrX8ItRY6X"><A href="/Team:Nottingham/test/sponsors" class="ui fluid basic button">About our sponsors</A></DIV></DIV></DIV></DIV></DIV></DIV></BODY></HTML>