<HTML xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en" dir="ltr">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki  ltr ns-0 ns-subject page-Team_St_Andrews_Omega-3-synthesis"><DIV id="globalWrapper"><DIV id="top-section"><DIV id="p-logo"><A href="/Main_Page" title="Main Page">&quot;
	    </A></DIV><DIV id="menubar" class="left-menu noprint"><UL><LI class="selected"><A href="/Team:St_Andrews/Omega-3-synthesis">Page               </A></LI><LI class="new"><A href="/wiki/index.php?title=Talk:Team:St_Andrews/Omega-3-synthesis&amp;action=edit&amp;redlink=1">Discussion               </A></LI><LI><A href="/wiki/index.php?title=Team:St_Andrews/Omega-3-synthesis&amp;action=edit">View source               </A></LI><LI><A href="/wiki/index.php?title=Team:St_Andrews/Omega-3-synthesis&amp;action=history">History               </A></LI><LI style="color:#808080;cursor:default">teams</LI></UL></DIV><DIV class="right-menu noprint" id="menubar"><UL><LI id="pt-login"><A href="/wiki/index.php?title=Special:UserLogin&amp;returnto=Team:St_Andrews/Omega-3-synthesis" title="You are encouraged to log in; however, it is not mandatory [o]" accesskey="o">Log in</A></LI></UL></DIV><DIV id="search-controls" class="noprint"><FORM action="/Special:Search" id="searchform"> </FORM></DIV></DIV><DIV id="content"><H1 class="firstHeading">Team:St Andrews/Omega-3-synthesis</H1><DIV id="bodyContent"><H3 id="siteSub" class="noprint">From 2012.igem.org</H3><DIV class="navbar navbar-fixed-top"><DIV class="navbar-inner"><DIV class="container"><A class="brand" href="/Team:St_Andrews">St Andrews iGEM '12</A><DIV class="nav-collapse"><UL class="nav"><LI class="dropdown"><A href="#" class="dropdown-toggle" data-toggle="dropdown">
                    Projects
                    </A></LI><LI><A href="/Team:St_Andrews/metal-binding">Metal binding protein</A></LI><LI><A href="/Team:St_Andrews/Omega-3-synthesis">Omega-3 fatty acid synthesis</A></LI><LI><A href="/Team:St_Andrews/Modelling">The mathematics of ω-3</A></LI><LI><A href="/Team:St_Andrews/Human-practices">Scientific impact of iGEM</A></LI></UL><LI><A href="/Team:St_Andrews/Parts">Biobricks</A></LI><LI class="dropdown"><A href="#" class="dropdown-toggle" data-toggle="dropdown">
                    Notebook
                    </A></LI><LI><A href="/Team:St_Andrews/Notebook">Journal</A></LI><LI><A href="/Team:St_Andrews/Lab-book">Lab book: Protocols</A></LI><LI><A href="/Team:St_Andrews/Safety">Safety</A></LI><LI><A href="/Team:St_Andrews/Public-outreach">Outreach</A></LI><LI class="dropdown"><A href="#" class="dropdown-toggle" data-toggle="dropdown">
                    Team
                    </A></LI><LI><A href="/Team:St_Andrews/Team">Members</A></LI><LI><A href="https://igem.org/Team.cgi?year=2012&amp;team_name=St_Andrews">Profile</A></LI><LI><A href="/Team:St_Andrews/Attributions">Attributions</A></LI></DIV></DIV></DIV></DIV><DIV class="container" id="content-container" style="background-image:url('https://static.igem.org/mediawiki/2012/thumb/6/60/Omega3logo2Watermark.jpg/281px-Omega3logo2Watermark.jpg');background-repeat:no-repeat;background-position:top right;background-size:35%"><HEADER class="jumbotron subhead" id="overview"><H1>Omega-3 fatty acid synthesis</H1><DIV class="subnav"><UL class="nav nav-pills"><LI><A href="#content-container">Introduction</A></LI><LI><A href="#project-description">Project description</A></LI><LI><A href="#Recreating-pathway">Recreating the pathway</A></LI><LI><A href="#methods">Methods</A></LI><LI><A href="#results">Results</A></LI><LI><A href="#biobricks">Biobricks</A></LI><LI><A href="#references">References</A></LI></UL></DIV></HEADER><SECTION id="introduction"><DIV class="page-header"><H1>Introduction</H1></DIV><P>ω-3 fatty acids are a key component of the human diet. Our team is recreating this synthetic pathway in <I>E. coli</I>, using genes from the cyanobacteria Synechocystis and the trypanosomatid Leishmania major. Combining the DNA code for elongase and desaturase enzymes, we are planning to convert a fatty acid with a single desaturation into highly valuable ω-3 fatty acids.</P></SECTION><SECTION id="project-description"><DIV class="page-header"><H1>Project description</H1></DIV><DIV class="row-fluid"><UL><LI class="span4"><DIV class="thumbnail"><DIV class="image-caption"><H5>Figure 1: <EM>&quot;Synechocystis sp.&quot;</EM></H5></DIV></DIV></LI><LI class="span4"><DIV class="thumbnail"><DIV class="image-caption"><H5>Figure 2: <EM>&quot;Trypanosome cruzi&quot;</EM></H5></DIV></DIV></LI><LI class="span4"><DIV class="thumbnail"><DIV class="image-caption"><H5>Figure 3: <I>&quot;Leishmania major&quot;</I></H5></DIV></DIV></LI></UL></DIV><P>Omega-3 fatty acids are an essential part of the human diet (Bender, Bender, 1999). Human beings, like all larger organisms, cannot synthesize ω-3 fatty acids. This is due to a lack of the enzyme Δ15 desaturase, which creates a double bond at the 15th carbon of a long-chain fatty acid. Certain micro-organisms, such as microalgae and cyanobacteria, do contain this desaturase and can thus directly synthesize ω-3 fatty acids (Arts <I>et al.</I>, 2009). ω-3 fatty acids then enter the food chain – algae are eaten by fish, and seafood is subsequently the main source of ω-3 for humans (Tonon et al., 2002).</P><P>However, the current economic policies of overfishing are a serious contributor to marine biodestruction. As the human population is estimated to rise to 9.1 billion by 2050 (Cohen, 2003), pressure on fish stock will increase. Additionally, global warming will reduce the availability of ω-3 (Arts et al., 2009):  in response to high temperatures, microalgae produce less ω-3 desaturated fatty acids and more saturated fatty acids, as desaturated carbon chains cause a lower melting temperature in membranes (Garwin, Cronan, 1980). Thus, the combination of declining fish stock and a decrease in overall ω-3 fatty acids is making the supply for human nutrition a relevant issue. </P><P>Harvesting algae directly is costly and ineffective (Borowitzka, 1997). There is much potential in expressing a metabolic pathway for ω-3 fatty acid synthesis in <I>E. coli</I>, which is cheaper and more accessible. </P><SECTION id="Recreating-pathway"><DIV class="page-header"><H1>Recreating the pathway</H1></DIV><DIV class="row"><DIV class="span6"><H5>Figure 4: <EM>&quot;The metabolic pathway to ω-3 fatty acids&quot;</EM></H5><DIV class="span5"><P>Figure 4 shows the elongation and desaturation enzymes necessary to convert an 18:1 fatty acid, into an poly-unsaturated fatty acid.</P><P><EM>modified from Livore et al., 2006</EM></P></DIV></DIV><DIV class="span6"><P><I>E. coli</I> naturally synthesize unsaturated fatty acids up to a carbon chain length of 18, with a single desaturation (18:1) (Marr, Ingraham, 1969). Valuable ω-3 fatty acids require a double bonds at the third carbon from the end of its carbon chain and can have &gt;20 carbons.</P><P>In order to have <I>E. coli</I> synthesize ω-3 fatty acids, we needed to introduce enzymes that could elongate and desaturate fatty acid substrates (cf. Fig. 4).</P><P>The genes for Δ12, Δ15 (ω6) and Δ6 were obtained from <I>Synechocystis sp.</I>, a cyanobacterium. The trypanosomatid <I>Leishmania major</I> provided the DNA for the ELO 6 gene. Additionally, we used <I>Trypanosome cruzi</I> as a secondary source of Δ12.</P><P>However, our first successful ligations of Δ12 did not provide us with the expected 18:2 fatty acid. We hypothesized that <I>E. coli</I>’s inherent 18-carbon chain fatty acid might not be suited as a substrate for Δ12 – the double bond is in a different position, the 11th. Therefore, we &quot;fed&quot; our cells with suitable 18:1, to then observe 18:2 fatty acid, and ultimately ω-3 desaturation, in the mass spec results!</P></DIV></DIV></SECTION><SECTION id="methods"><DIV class="page-header"><H1>Methods</H1></DIV><DIV class="row-fluid"><DIV class="span6"><P> The following genes were employed (please click for sequences and KEGG numbers):</P><DIV class="btn-toolbar"><A data-toggle="modal" href="#modal4" class="btn btn-primary btn-small"><FONT color="white">Δ6 Synechocystis</FONT></A><DIV class="modal hide" id="modal4"><DIV class="modal-header"><BUTTON type="button" class="close" data-dismiss="modal">×</BUTTON><H3>Δ6 Synechocystis</H3></DIV><DIV class="modal-body"><P>ATGCTAACAGCGGAAAGAATTAAATTTACCCAGAAACGGGGGTTTCGTCGGGTACTAAAC
CAACGGGTGGATGCCTACTTTGCCGAGCATGGCCTGACCCAAAGGGATAATCCCTCCATG
TATCTGAAAACCCTGATTATTGTGCTCTGGTTGTTTTCCGCTTGGGCCTTTGTGCTTTTT
GCTCCAGTTATTTTTCCGGTGCGCCTACTGGGTTGTATGGTTTTGGCGATCGCCTTGGCG
GCCTTTTCCTTCAATGTCGGCCACGATGCCAACCACAATGCCTATTCCTCCAATCCCCAC
ATCAACCGGGTTCTGGGCATGACCTACGATTTTGTCGGGTTATCTAGTTTTCTTTGGCGC
TATCGCCACAACTATTTGCACCACACCTACACCAATATTCTTGGCCATGACGTGGAAATC
CATGGAGATGGCGCAGTACGTATGAGTCCTGAACAAGAACATGTTGGTATTTATCGTTTC
CAGCAATTTTATATTTGGGGTTTATATCTTTTCATTCCCTTTTATTGGTTTCTCTACGAT
GTCTACCTAGTGCTTAATAAAGGCAAATATCACGACCATAAAATTCCTCCTTTCCAGCCC
CTAGAATTAGCTAGTTTGCTAGGGATTAAGCTATTATGGCTCGGCTACGTTTTCGGCTTA
CCTCTGGCTCTGGGCTTTTCCATTCCTGAAGTATTAATTGGTGCTTCGGTAACCTATATG
ACCTATGGCATCGTGGTTTGCACCATCTTTATGCTGGCCCATGTGTTGGAATCAACTGAA
TTTCTCACCCCCGATGGTGAATCCGGTGCCATTGATGACGAGTGGGCTATTTGCCAAATT
CGTACCACGGCCAATTTTGCCACCAATAATCCCTTTTGGAACTGGTTTTGTGGCGGTTTA
AATCACCAAGTTACCCACCATCTTTTCCCCAATATTTGTCATATTCACTATCCCCAATTG
GAAAATATTATTAAGGATGTTTGCCAAGAGTTTGGTGTGGAATATAAAGTTTATCCCACC
TTCAAAGCGGCGATCGCCTCTAACTATCGCTGGCTAGAGGCCATGGGCAAAGCATCGTGA</P><P>KEGG entry sll0541</P></DIV><DIV class="modal-footer"><A href="#" class="btn" data-dismiss="modal">Close</A></DIV></DIV><A data-toggle="modal" href="#modal2" class="btn btn-primary btn-small"><FONT color="white">Δ12 Synechocystis</FONT></A><DIV class="modal hide" id="modal2"><DIV class="modal-header"><BUTTON type="button" class="close" data-dismiss="modal">×</BUTTON><H3>Δ12 Synechocystis</H3></DIV><DIV class="modal-body"><P>ATGACTGCCACGATTCCCCCGTTGACACCAACGGTAACGCCCAGCAATCCCGATCGCCCG
ATTGCGGATCTCAAACTACAAGACATCATTAAAACCCTGCCCAAGGAATGCTTCGAGAAA
AAAGCGAGCAAAGCCTGGGCTTCTGTTTTGATTACCCTAGGGGCGATCGCCGTGGGCTAT
TTGGGCATTATTTATCTGCCCTGGTACTGCTTGCCCATTACCTGGATCTGGACAGGGACA
GCCTTAACGGGGGCCTTCGTTGTCGGCCATGACTGTGGCCATCGCTCCTTTGCTAAAAAA
CGCTGGGTCAATGATTTAGTGGGACATATCGCTTTTGCTCCCCTCATCTACCCTTTCCAT
AGCTGGCGCCTACTCCACGACCACCATCACCTCCACACCAACAAAATTGAGGTTGATAAC
GCCTGGGATCCCTGGAGTGTGGAAGCTTTCCAAGCCAGCCCGGCGATCGTCCGGCTTTTT
TATCGGGCCATCCGGGGTCCCTTCTGGTGGACTGGTTCCATTTTCCATTGGAGCTTAATG
CACTTCAAACTTTCCAACTTTGCCCAAAGGGACCGCAATAAAGTCAAATTATCCATTGCC
GTTGTCTTCCTGTTTGCGGCGATCGCCTTTCCTGCCCTAATTATCACCACAGGGGTGTGG
GGTTTCGTCAAATTTTGGCTAATGCCCTGGTTGGTGTATCACTTTTGGATGAGCACTTTT
ACCATTGTGCACCACACCATTCCCGAAATTCGTTTCCGTCCCGCCGCCGATTGGAGTGCC
GCCGAAGCCCAGTTAAATGGTACTGTTCACTGCGATTATCCCCGTTGGGTGGAAGTGCTC
TGCCATGACATCAACGTCCATATTCCCCACCACCTCTCCGTTGCCATCCCTTCCTATAAC
CTACGACTAGCCCACGGAAGTTTAAAAGAAAACTGGGGACCTTTTCTTTACGAGCGCACC
TTTAACTGGCAATTAATGCAACAAATTAGTGGGCAATGTCATTTATATGACCCCGAACAT
GGCTACCGCACCTTCGGCTCCCTGAAAAAAGTTTAA
</P><P>KEGG entry 429257.20</P></DIV><DIV class="modal-footer"><A href="#" class="btn" data-dismiss="modal">Close</A></DIV></DIV><A data-toggle="modal" href="#modal3" class="btn btn-primary btn-small"><FONT color="white">Δ15 Synechocystis</FONT></A><DIV class="modal hide" id="modal3"><DIV class="modal-header"><BUTTON type="button" class="close" data-dismiss="modal">×</BUTTON><H3>Δ15 Synechocystis</H3></DIV><DIV class="modal-body"><P>GTGCGTCTAGAAATTTCATCGCCTCAAACAAAGCTTCCTTACCCCAAAACTGAAGAATTA
CCATTTACCCTCCAAGAGCTCAGAAACGCTATTCCAGCGGATTGTTTTGAGCCATCGGTA
GTCCGGTCCTTGGGCTACTTTTTTTTGGATGTTGGTTTAATTGCCGGGTTTTATGCTCTA
GCGGCCTACCTTGATTCCTGGTTCTTCTATCCGATTTTTTGGTTAATTCAGGGAACCCTA
TTCTGGTCCCTGTTTGTGGTGGGCCATGATTGTGGCCATGGCTCCTTTTCCAAATCCAAA
ACCCTTAATAATTGGATTGGTCATCTCAGCCACACGCCAATTTTGGTGCCTTACCATGGC
TGGCGTATTAGTCATCGTACTCACCATGCCAACACGGGCAATATCGACACCGACGAAAGT
TGGTATCCAGTGTCGGAGCAAAAATATAACCAAATGGCCTGGTATGAAAAACTTCTACGT
TTTTACTTGCCTCTGATCGCCTACCCCATTTATCTATTTCGGCGATCGCCAAACCGGCAA
GGCTCCCATTTCATGCCCGGCAGTCCCCTATTCCGTCCCGGAGAAAAAGCAGCTGTTCTC
ACCAGCACCTTTGCCCTTGCAGCCTTTGTCGGCTTCCTTGGCTTTTTAACTTGGCAATTT
GGCTGGCTATTTTTGCTGAAATTTTATGTTGCCCCCTACCTCGTGTTTGTGGTGTGGTTA
GATTTGGTCACATTTTTACATCACACTGAAGACAATATCCCTTGGTATCGTGGTGATGAC
TGGTATTTTCTCAAAGGTGCCCTCTCCACCATTGATCGGGATTACGGCTTCATTAACCCC
ATTCACCATGACATTGGCACCCACGTCGCCCACCATATTTTCTCGAATATGCCCCACTAC
AAGTTACGCCGGGCGACTGAAGCCATCAAGCCCATTTTAGGGGAATATTATCGATATTCT
GACGAGCCAATTTGGCAAGCTTTTTTTAAGTCCTACTGGGCTTGCCATTTTGTTCCTAAT
CAAGGTTCAGGGGTCTATTACCAATCCCCATCCAATGGTGGATATCAAAAGAAACCTTAA
</P><P>KEGG entry s111441</P></DIV><DIV class="modal-footer"><A href="#" class="btn" data-dismiss="modal">Close</A></DIV></DIV><A data-toggle="modal" href="#modal5" class="btn btn-primary btn-small"><FONT color="white">ELO 6 L. major</FONT></A><DIV class="modal hide" id="modal5"><DIV class="modal-header"><BUTTON type="button" class="close" data-dismiss="modal">×</BUTTON><H3>ELO 6 L. major</H3></DIV><DIV class="modal-body"><P>ATGAAGGTCATCGTCGCTTCTGGCCCGGACGGTGCCCGCAAGCACGAGGTGGAGCTGGCA
GCCAACGCCACGCTCGCAGATCTGAAGAAGGCCTACCAACGGGGTGTGGACGTGCACCGC
AAGTCGTTCAAGGTTCCCAGCGCGGAGTCGCCGCTGCCAGGTGCGGATAGTGGCAAGCTG
CGCCCGAACCTCATTACTCTGTCAGATAAGGTGCCCCTGTCGCAGCAGGGGGTGAAGGAT
GGCTCGGTGATCACTTACAAGGACCTCGGCCCGCAGATCGGCTACCGCACGGTGTTCTAC
GTCGAGTATGCCGGCCCCATCGCCTTCATGCTGCTGTACGCCATGCGCCCTTCGCTCATC
TACGGCTCTGCCCCGATGCCGGCTTACGGCTACACGCAGAAGCTATACATTGGCCTCTTC
CTCGCCCACTTCTTAAAGCGCGAGCTCGAGTCCATGTTTGTGCACAAGTTCTCGCACCCA
ACGATGCCGATGCGCAACATCTTCAAGAACTGCATCTACTACTGGTCCTTCGCCGCCTTC
ATCGGCTACGTGCTGTGCAGTCCTTCATTCACGCCGACCAGCACCATGCAGTCAAACTTC
GGCGCCGTGGTCATGGTCATCAACGAGCTGCTGAACTTCGCGGTGCACTACCAGCTTAGC
ACGATGCGCAAGTCCGATGGTGACACCACCCGCAACGTGCCGCAAGGCCCTCTGTTCGCC
TTCGTCTCGTGCCCGAACTACTTCTTTGAGATTATGTCGTGGGTGTCCTTTTCCATCGGC
ACAAATATGTTATCCTCCTGGTTCTTCACACTCGCCGGTTTCGTGCAGATGGCGGACTGG
GCGAAGAAGAAGCACCGGGGCTACGTCAAGGCGGACCCGGCCAATAAAAAGAAGGCCGCC
ATTCTGCCCTTCATCATGTAG</P><P>KEGG entry LmjF32.1160</P></DIV><DIV class="modal-footer"><A href="#" class="btn" data-dismiss="modal">Close</A></DIV></DIV><A data-toggle="modal" href="#modal1" class="btn btn-primary btn-small"><FONT color="white">Δ12 T. Cruzi</FONT></A><DIV class="modal hide" id="modal1"><DIV class="modal-header"><BUTTON type="button" class="close" data-dismiss="modal">×</BUTTON><H3>Δ12 T. Cruzi</H3></DIV><DIV class="modal-body"><P>ATGTCGCGGGTTGACAATTTAACTGTCGCGCCGGGTCCACCGGATGTCATGAAGGCGGTG
CTTAAGTCTGGGCGCAACAAGGAGCACAATGTCATTGTGCCTTCCTCCGCACTGACCATTG
GAGAAATTCAGGAGAAAATCCCGGTAAAGTACTTTGAACGCAACACTACCCGGAGCGTCA
TGTTTCTTTTACGCGACCTCGCACAGGTGGCGTTGACGTACGCCATTATGTACGCCGTCGCC
TTGCCACTTGCCACTTCCATGGAAGTTTCTGCAGCAAGAACGTTGGCCGACGGTGGGGTGT
TATCATTGATGGCAGGTACAGCCATGACGACAGCGGCGTGGCTTTTAAAGGCAGTGTTGTG
GGCGGTGTTTTGGTTTGTACAGGGACTTAACGGCACTGCGCTTTGGGTCCTGGCTCACGAA
TGCGGCCATCAGGCGTTCAGCCCCATGAAAGGTTTGAATGACGCCGTTGGCATGATTTTGC
ACTCTGCGCTGCTCGTGCCGTACCACAGCTGGCGCATCACCCACGGCGGCCACCACAAACA
CACGAATCACCTCACGAAGGATCTTGTGTTTGTTCCAGACAAACGAAACGCGGTTGTGGAG
CTCGTGGAGGAGGCGCCGCTGGTGTTATTAATTCAATTATTGCTGATTTTTCTCTTTGGTTG
GCCGGCACATCTTCTTTTTAATGCCTCTGGACAGGAATTTGGCCGACTCGCGAGCCACTTTG
ACCCCGGCGCTCCATTTTTCCGCAGCGAAGACCGTCACGATATTGTCCTGTCGAATGTTGGG
ATTGTCAGCGCGTTATTTGTCATTTTCTCCAGCGTTTACCGCTTTGGTTTTACAAATGTTTTCT
GCTGGTACATTGTACCGTACCTCTGGGTGAACTTTTGGCTG
TGTACATTACATACCTGCAGCACACGGATATACGCATTCCTCACTACACACATGAGCACT
GGACGTTTGTTCGCGGTGCATTGGCGGCTGTGGACAGGGACTACGGCTTTGTCCTTAACA
CATGGCTCCATCACATCAATGATTCCCACGTGGTACATCACCTCTTTAGCAAAATGCCAC
ACTACAACGCAATCCAGGTGACAAGAAAGTACATTCGTGAGATACTGGGTGCCACATACA
TTACGGATGAGAGGTCACTGTGGAAGATGCTCTGGGAACAGCGTAGAGAGTGCCGCTATG
TTGTTCCCGCAGAGGGCGTCTGTGTCTTTCATGGGTAA</P><P>KEGG entry 429257.20</P></DIV><DIV class="modal-footer"><A href="#" class="btn" data-dismiss="modal">Close</A></DIV></DIV></DIV><P>These genes were amplified through PCR (Promega, GoTaq HotStart) at temperatures 48°C and 56°C.</P><P>The genes were initially ligated into pET-15b vector. After a number of expression attempts, some initial conclusions were reached. All experiments done on ELO6 failed. Also, Δ12 from <I>T. cruzi</I> gave overall weaker results than the same gene from <I>Synechocystis</I>. As such, latter work was only carried out on genes from <I>Synechocystis</I>.</P><P>Then, Δ6, Δ12, and Δ15 desaturases were successfully ligated into two distinct pET-Duet vectors (vectors with two multicloning sites). One vector contained Δ12 and Δ15 desaturases, and the other was ligated only with Δ6 desaturase.</P><P>Protein expression was clear after transformation into cell strain BL21(DE3) and induction by IPTG. However, functionality could not be established. It was hypothesized that the naturally-occuring 18:1 fatty acid in <I>E. coli</I> is the wrong substrate for the desaturases. This fatty acid has its desaturation at the 11th carbon, not at the 9th position required for a substrate. Thus, the <I>E. coli</I> were fed 18:1 (Δ9).</P><P>Full characterisatin and quantification of the fatty acid composition in transformed <I>E. coli</I> was performed by fatty acid conversion to the corresponding fatty acid methyl esters (FAMEs) followed by GC-MS analysis. In this way, lipid profiles of membrane assays and lipid extracts from cells were obtained.</P><P>After characterization, we ligated each of our desaturases into the submission vector pSB1C3.</P></DIV><DIV class="span6 well"><UL class="thumbnails"><H5>Fig. 5: <EM>&quot;UV photograph of PCR results&quot;</EM></H5><P>The figure shows the results of a PCR extraction of our genes of choice, done with GoTaq HotStart PCR kit at 2 different annealing temperatures: Δ12 (48°C) - Δ12 (56°C) - Δ15 (48°C) - Δ15 (56°C) - Δ6 (48°C) - Δ6 (56°C). </P></UL></DIV></DIV><H2>Primers</H2><P>All primers are notated 5' to 3'. Initially, we worked with NdeI and XhoI as the restriction sites.</P><DIV class="btn-toolbar"><DIV class="btn-group"><A class="btn btn-mini dropdown-toggle" data-toggle="dropdown" href="#">
   Δ12 T. Cruzi forward
    </A><UL class="dropdown-menu"><DIV class="span3">ATATCATATGTGGCGGGTTGACAATTTA 
  </DIV><DIV class="btn-group"><A class="btn btn-mini dropdown-toggle" data-toggle="dropdown" href="#">
    Δ12 T. Cruzi reverse
    </A><UL class="dropdown-menu"><DIV class="span3">ATATCTCGAGTTACCCATGAAAGACAC</DIV></UL></DIV><DIV class="btn-group"><A class="btn btn-mini dropdown-toggle" data-toggle="dropdown" href="#">
    Δ12 Synechocystis forward
    </A><UL class="dropdown-menu"><DIV class="span3">ATATCATATGACTGCCACGATTCCC</DIV></UL></DIV><DIV class="btn-group"><A class="btn btn-mini dropdown-toggle" data-toggle="dropdown" href="#">
    Δ12 Synechocystis reverse
    </A><UL class="dropdown-menu"><DIV class="span3">ATATCTCGAGTTAAACTTTTTTCAGGGAGC</DIV></UL></DIV><DIV class="btn-group"><A class="btn btn-mini dropdown-toggle" data-toggle="dropdown" href="#">
    Δ15 Synechocystis forward
    </A><UL class="dropdown-menu"><DIV class="span3">ATATCATATCGCTCTAGAAATTTCATC</DIV></UL></DIV><DIV class="btn-toolbar"><DIV class="btn-group"><A class="btn btn-mini dropdown-toggle" data-toggle="dropdown" href="#">
    Δ15 Synechocystis reverse
    </A><UL class="dropdown-menu"><DIV class="span3">ATATCACGACTTAAGGTTTCTTTTGATATC</DIV></UL></DIV><DIV class="btn-group"><A class="btn btn-mini dropdown-toggle" data-toggle="dropdown" href="#">
    Δ6 Synechocystis forward
    </A><UL class="dropdown-menu"><DIV class="span3">ATATCATATGCAAACAGCGGAAAGAATT</DIV></UL></DIV><DIV class="btn-group"><A class="btn btn-mini dropdown-toggle" data-toggle="dropdown" href="#">
    Δ6 Synechocystis reverse
    </A><UL class="dropdown-menu"><DIV class="span4">ATATCTCGAGTAACGATGCTTTGACCATGGCCTCTAG</DIV></UL></DIV><DIV class="btn-group"><A class="btn btn-mini dropdown-toggle" data-toggle="dropdown" href="#">
    ELO6 L. major forward
    </A><UL class="dropdown-menu"><DIV class="span4">ATATCATATGTTCTCTCTGGAGCAGGCCGAA</DIV></UL></DIV><DIV class="btn-group"><A class="btn btn-mini dropdown-toggle" data-toggle="dropdown" href="#">
    ELO6 L. major reverse
    </A><UL class="dropdown-menu"><DIV class="span4">ATATCTCGAGTCATCGACTGCCTGCCATGGCCTG</DIV></UL></DIV></DIV><P>When using the pET-duet vector, we needed additional primers for the alternative restriction sites and His-tags. For Δ6 and Δ12, we cut with HindIII and EcoRI; NdeI and XhoI were used for Δ15 and ELO6.</P><DIV class="btn-toolbar"><DIV class="btn-group"><A class="btn btn-mini dropdown-toggle" data-toggle="dropdown" href="#">
    ELO6 L. major forward
    </A><UL class="dropdown-menu"><DIV class="span6">TATCATATGGGAAGCAGCCATCATCATCATCATCACAGCATGGTCTCTCTGGAGCAGGCC</DIV></UL></DIV><DIV class="btn-group"><A class="btn btn-mini dropdown-toggle" data-toggle="dropdown" href="#">
    Δ15 Synechocystis sp. forward
    </A><UL class="dropdown-menu"><DIV class="span6">TATCATATGGGCAGCCATCATCATCATCATCACAGCATGCGTCTAGAAATTTCATCG</DIV></UL></DIV><DIV class="btn-group"><A class="btn btn-mini dropdown-toggle" data-toggle="dropdown" href="#">
    Δ6 Synechocystis sp. forward
    </A><UL class="dropdown-menu"><DIV class="span4">GGATCCGAATTCATCTAACAGCGGAAAGAATT</DIV></UL></DIV><DIV class="btn-group"><A class="btn btn-mini dropdown-toggle" data-toggle="dropdown" href="#">
    Δ6 Synechocystis sp. reverse
    </A><UL class="dropdown-menu"><DIV class="span4">GACAAGCTTTCACGATGCTTTGCCCATGGCCTCTAC</DIV></UL></DIV><DIV class="btn-group"><A class="btn btn-mini dropdown-toggle" data-toggle="dropdown" href="#">
    Δ12 Synechocystis sp. forward
    </A><UL class="dropdown-menu"><DIV class="span4"> GGATCCGAATTCATGACTGCCACGATTCCC</DIV></UL></DIV><DIV class="btn-group"><A class="btn btn-mini dropdown-toggle" data-toggle="dropdown" href="#">
    Δ12 Synechocystis sp. reverse
    </A><UL class="dropdown-menu"><DIV class="span3">GACAAGCTTTTAAACTTTTTTCAGGGAGC</DIV></UL></DIV><P>For the submission vector, we once again required new primers, using restriction sites EcoRI and PstI. For Δ6 and Δ12 forward, we used the primers from the duet vector (see above). </P><DIV class="btn-group"><A class="btn btn-mini dropdown-toggle" data-toggle="dropdown" href="#">
    Δ6 Synechocystis reverse
    </A><UL class="dropdown-menu"><DIV class="span5">GTTCCTGCAGTCACGATGCTTTGCCCATGG CTCTAG</DIV></UL></DIV><DIV class="btn-group"><A class="btn btn-mini dropdown-toggle" data-toggle="dropdown" href="#">
    Δ15 Synechocystis forward
    </A><UL class="dropdown-menu"><DIV class="span4">GGATCCGAATTCATGCGTCTAGAAATTTCATCG</DIV></UL></DIV><DIV class="btn-group"><A class="btn btn-mini dropdown-toggle" data-toggle="dropdown" href="#">
    Δ15 and Δ12 Synechocystis reverse
    </A><UL class="dropdown-menu"><DIV class="span3">GTTCCTGCAGTTAAACTTTTTTCAGGGAGC</DIV></UL></DIV></DIV><H2>Analysis</H2><DIV class="row-fluid"><DIV class="span2"><STRONG>Lipid Analysis</STRONG><P>Lipids were extracted from transformed <I>E. coli</I> using the Blight/Dyer method. (Please refer to the <A href="https://2012.igem.org/Team:St_Andrews/Lab-book">Lipid extraction lab book entry</A> for further detail.)</P></DIV><DIV class="span4"><STRONG>Mass Spectrometry</STRONG><P>Lipids were extracted from transformed <I>E. coli</I> using the Blight/Dyer method. (Please refer to the <A href="https://2012.igem.org/Team:St_Andrews/Lab-book">Lipid extraction lab book entry</A> for further detail.)</P><P>The fatty acid content of the transformed cells was measured to determine whether it differed from the previously analyzed background lipid composition. This was done using mass spectrometry.</P></DIV><DIV class="span2"><STRONG>Bradford protein assay</STRONG><P>A Bradford protein assay was carried out to assess protein concentration (Bradford, 1976).</P></DIV><DIV class="span2"><STRONG>Sequencing</STRONG><P>We sent off samples to GATC for sequencing.</P></DIV></DIV></UL></DIV></DIV></SECTION><SECTION id="results"><DIV class="page-header"><H1>Results</H1></DIV><P>The fatty acids were released by base hydrolysis followed by organic extraction, the resulting fatty acids were derivatised with diazomethane to the corresponding fatty acid methyl esters (FAMEs), together with fatty acid standards. The samples were analysed by GC-MS and the retention times and fragmentation patterns, compared with FAME standards. The results are shown in Charts 3,4 and 5.</P><P>By combining a number of genes, we were able to partially recreate the pathway described above (Fig. 4).</P><P>Due to time constraints, we were unable to submit a BioBrick containing the sequence of all of the necessary genes for the ω-3 biosynthetic pathway. However, we were able to express and characterise Δ12, Δ15 and Δ6 desaturases, the first three enzymes involved in ω-3 biosynthesis. As shown in charts 2, 3 and 4, the presence of alpha-linoleic acid (18:3<SUP>Δ9,12,15</SUP>), an omega-3 fatty acid, can be observed in <I>E. coli</I>. This does not occur naturally in their bacterial membrane.</P><H2>Mass Spectrometry</H2><DIV class="row-fluid"><DIV class="span12 well"><UL class="thumbnails"><LI class="span4"><A href="#fame-analysis" data-toggle="modal" class="thumbnail"><H5>Chart 2: <EM>&quot;Fatty acid methyl ester analysis&quot;</EM></H5><P>C18:3 refers to an 18 carbon backbone with 3 double bonds. C18:2 and C18:1 would have 2 double bonds and 1 respectively. The upper portion of this figure shows the peaks generated by the standard expected fatty acid methyl esters.The lower 4 panels show mass spec fingerprints that are indicative of our fatty acids of interest.</P></A></LI><DIV class="modal large hide" id="fame-analysis"><DIV class="modal-header"><BUTTON type="button" class="close" data-dismiss="modal">×</BUTTON><H3>Chart zoom: <EM>Fatty acid methyl ester analysis</EM></H3></DIV></DIV><LI class="span4"><A href="#lipid-extraction" data-toggle="modal" class="thumbnail"><H5>Chart 3: <EM>&quot;Lipid extraction analysis&quot;</EM></H5><P>All of the cellular lipids were extracted and their fatty acid methyl esters produced. These were analysed by mass spec. In the control cells no C18:2 is present. When Δ6 is expressed, a peak indicative of C18:2 is present. However, the primary fatty acid is still C18:1 - with the bouble bond present at position 9. With the expression of Δ12 and the addition of exogenous C18:1(Δ9) we get great abundance of C18:2 and negligible abundance of C18:1 indicating that the C18:1 is being processed successfully. The lowest graph shows that Δ12 expressed with Δ15 acts in a complementary fashion and the expected C18:3 - an Omega 3 - is produced. </P></A></LI><DIV class="modal large hide" id="lipid-extraction"><DIV class="modal-header"><BUTTON type="button" class="close" data-dismiss="modal">×</BUTTON><H3>Chart zoom: <EM>Fatty acid methyl ester analysis</EM></H3></DIV></DIV><LI class="span4"><A href="#membrane-assay" data-toggle="modal" class="thumbnail"><H5>Chart 4: <EM>&quot;Membrane assay&quot;</EM></H5><P>The cellular membranes were extracted and their lipids were analysed by mass spec. As in chart 2 it is confirmed that expression of Δ12 and Δ15 lead to the production of omega 3 fatty acids. This chart shows that these lipids locate to the plasma membrane of the bacteria</P></A></LI><DIV class="modal large hide" id="membrane-assay"><DIV class="modal-header"><BUTTON type="button" class="close" data-dismiss="modal">×</BUTTON><H3>Chart zoom: <EM>Fatty acid methyl ester analysis</EM></H3></DIV></DIV></UL></DIV></DIV><H2>Bradford protein assay</H2><DIV class="row-fluid"><DIV class="span6 well"><UL class="thumbnails"><H5>Chart 5: <EM>&quot;Bradford protein assay&quot;</EM></H5><P>This chart shows the standard results of a Bradford protein assay.  Measuring the absorbance at 595nm for set samples, a standard curve was calculated with the equation of y = 0.0006x + 0.0426.</P></UL></DIV><DIV class="span4 well"><UL class="thumbnails"><H5>Table 1: <EM>&quot;Bradford protein assay&quot;</EM></H5><P>Using the standard curve shown in chart 5, the protein concentrations of a number of transformed plasmids was calculated.</P></UL></DIV></DIV></SECTION><SECTION id="biobricks"><DIV class="page-header"><H1>Biobricks</H1></DIV><P>We have submitted 3 BioBricks involved in the ω-3 biosynthetic pathway to the Registry of Standard Parts:</P><TABLE class="table table-striped"><THEAD><TR><TH>Biobrick</TH><TH>Short name</TH><TH>Description</TH><TH>Length</TH></TR></THEAD><TBODY><TR><TD><A href="http://partsregistry.org/Part:BBa_K925000">BBa_K925000</A></TD><TD>Delta-12 desaturase</TD><TD>Desaturase introducing a double bond at the Δ-12 site in the hydrocarbon chain of oleic acid (18:1, Δ9) to give linoleic acid (18:2 Δ9,12)</TD><TD>1056</TD></TR><TR><TD><A href="http://partsregistry.org/Part:BBa_K925001">BBa_K925001</A></TD><TD>Delta-15 desaturase</TD><TD>Desaturase introducing a double bond at the Δ-15 site in the hydrocarbon chain of linoleic acid (18:2 ; Δ9,12) for its convertion into alpha-linoleic acid (18:3 ; Δ9,12,15), an ω -3 PUFA</TD><TD>1080</TD></TR><TR><TD><A href="http://partsregistry.org/Part:BBa_K925003">BBa_K925003</A></TD><TD>Delta-6 desaturase</TD><TD>Desaturase introducing a double bond at the Δ-12 site in the hydrocarbon chain of oleic acid (18:1, Δ9) to give linoleic acid (18:2 Δ9,12)</TD><TD>1080</TD></TR></TBODY></TABLE></SECTION><SECTION id="references"><DIV class="page-header"><H1>References</H1></DIV><P>Arts, <I>et al.</I>, 2009. Lipids in Aquatic Ecosystems. New York. Springer.</P><P>Bender D. A. and Bender, A. E, 1999. Benders' dictionary of nutrition and food technology. Cambridge: CRC Press.</P><P>Bradford, M., 1976. <A href="http://www.ncbi.nlm.nih.gov/pubmed/942051">A rapid and sensitive method for the quantitation of microgram quantities of protein utilizing the principle of protein-dye binding.</A> Anal Biochem, 7;72:248-54.</P><P>Cohen, J., 2003. <A href="http://www.sciencemag.org/content/302/5648/1172.abstract">Human Population: The Next Half Century.</A> Science, New Series, Vol. 302, No. 5648. Pp. 1172-1175.</P><P>Garwin J. L. and Cronan J. E. Jr, 1980. <A href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC293857/">Thermal modulation of fatty acid synthesis in <I>Escherichia coli</I> does not involve de novo enzyme synthesis.</A> J Bacteriol, 141(3): 1457–1459.</P><P>Livore V., Tripodi K., Utarro A., 2007. <A href="http://www.ncbi.nlm.nih.gov/pubmed/17222186">Elongation of polyunsaturated fatty acids in trypanosomatids.</A> FEBS Journal, 274: 264–274.</P><P>Marr A., Ingraham J., 1969.  <A href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC278056/">Effect of temperature on the composition of fatty acids in <I>Escherichia coli</I>.</A> J. Bactiol. 84(6). Pp.  1260–1267.</P><P>Tonon T., <I>et al.</I>, 2002. <A href="http://www.sciencedirect.com/science/article/pii/S0031942202002017">Long chain polyunsaturated fatty acid production and partitioning to triacylglycerols in four microalgae.</A> Phytochemistry, Vol 61 Iss 1. Pgs 15-24.</P></SECTION></SECTION></DIV><DIV class="container" id="content-container"><FOOTER class="footer"><P class="pull-right"><A href="#">Back to top</A></P><P>University of St Andrews, 2012.</P><P>Contact us: <A href="mailto:igem2012@st-andrews.ac.uk">igem2012@st-andrews.ac.uk</A>, <A href="https://twitter.com/#!/StAndrewsiGEM12">Twitter</A>, <A href="https://www.facebook.com/StAndrewsiGEM12">Facebook</A></P><P><SMALL>This iGEM team has been funded by the MSD Scottish Life Sciences Fund. The opinions expressed by this iGEM team are those of the team members and do not necessarily represent those of Merck Sharp &amp; Dohme Limited, nor its Affiliates.</SMALL></P></FOOTER></DIV><DIV class="printfooter">
Retrieved from &quot;<A href="http://2012.igem.org/Team:St_Andrews/Omega-3-synthesis">http://2012.igem.org/Team:St_Andrews/Omega-3-synthesis</A>&quot;</DIV></DIV></DIV><DIV id="footer-box" class="noprint"><DIV id="footer"><UL id="f-list"><LI id="t-recentchanges"><A href="/Special:RecentChanges" title="Recent changes">Recent changes</A></LI><LI id="t-whatlinkshere"><A href="/Special:WhatLinksHere/Team:St_Andrews/Omega-3-synthesis" title="List of all wiki pages that link here [j]" accesskey="j">What links here</A></LI><LI id="t-recentchangeslinked"><A href="/Special:RecentChangesLinked/Team:St_Andrews/Omega-3-synthesis" title="Recent changes in pages linked from this page [k]" accesskey="k">Related changes</A></LI><LI id="t-specialpages"><A href="/Special:SpecialPages" title="List of all special pages [q]" accesskey="q">Special pages</A></LI><LI><A href="/Special:Preferences">My preferences</A></LI></UL></DIV><DIV id="footer"><UL id="f-list"><LI id="t-print"><A href="/wiki/index.php?title=Team:St_Andrews/Omega-3-synthesis&amp;printable=yes" title="Printable version of this page [p]" accesskey="p">Printable version</A></LI><LI id="t-permalink"><A href="/wiki/index.php?title=Team:St_Andrews/Omega-3-synthesis&amp;oldid=223191" title="Permanent link to this revision of the page">Permanent link</A></LI><LI id="privacy"><A href="/2012.igem.org:Privacy_policy" title="2012.igem.org:Privacy policy">Privacy policy</A></LI><LI id="disclaimer"><A href="/2012.igem.org:General_disclaimer" title="2012.igem.org:General disclaimer">Disclaimers</A></LI></UL></DIV></DIV></DIV></BODY></HTML>