"
Page
Discussion
View source
History
teams
Log in
 
Team:St Andrews/metal-binding
From 2012.igem.org
St Andrews iGEM '12
Projects
Metal binding protein
Omega-3 fatty acid synthesis
The mathematics of ω-3
Scientific impact of iGEM
Biobricks
Notebook
Journal
Lab book: Protocols
Safety
Outreach
Team
Members
Profile
Attributions
Metal binding protein
Introduction
Synthesizing metal binding peptides
Biobricks
References
Introduction
Precious and toxic metals frequently find their way into the environment. As their names suggest, such leaks are wasteful and damaging respectively. St Andrews iGEM '12 plans to take on this challenge using synthetic biology.
Inspiration
Our project was inspired by the work done identifying the platinum and palladium particles present on roads, primarily emitted from catalytic converters (Deplanche et al. 2011).  In 2010, 50% of the world's platinum and palladium production was used for catalytic converters, with the largest use in Europe (Jollie 2010).  Platinum and palladium appear on road surfaces in small concentrations and in minute particles, < 3 μm (Prichard, Fisher 2012).  These precious elements are a finite resource!  However, their eventual exhaustion can be postponed by collecting and recycling them.
Collection
We have focused on the collection process.  We have worked in a similar way to Chung et al., in that we have produced a protein with a metal binding peptide on the terminus of an easily expressible protein (Chan Chung, Cao et al. 2008).  The difference is that we used glutathione S-transferase (GST tag) rather than ubiquitin, and instead of adding the binding peptide chemically to the protein we expressed both the protein and the peptide in E. coli BL21 (DE3) cells.  These cells are particularly suited to large scale protein production.  The peptides sequences were taken from various sources (Seker, Demir 2011, Bae, Chen et al. 2000, Song, Caguiat et al. 2004, White, Liljestrand et al. 2007) and codon optimised for E. coli.  The nucleotide sequence that code for the peptides were modified so they could be produced by E. coli using an online program Protein to DNA .
We decided to go down two different routes.  The first was making an array of peptides incorporated into GST-protein that bind specific metals, and the second was to make a protein with multiple metal binding sites for multiple metals.
Metal Binding Peptides
With the first of the two ideas we thought that it would be possible use a column to immobilise the protein by its GST tag.  A solution containing multiple metal ions would then be passed through the column, which would bind each metal ion specifically.  Initially, we decided to take a look at palladium, platinum and nickel (Seker, Demir 2011).  The reason for choosing these metals was twofold: firstly, because platinum and palladium are particularly precious metals, which provides an economic argument; and secondly, because nickel columns were readily available to test this idea.
gBlock Scavengers
The second route would produce a protein that would act as a general metal scavenging protein.  gBlocks (≤ 500 bps) were specifically designed and inserted into a pGEX-6P-1.  One of them was designed to bind toxic metals: cadmium, mercury and cobalt (Seker, Demir 2011, Bae, Chen et al. 2000, Song, Caguiat et al. 2004, White, Liljestrand et al. 2007).  This was done by inserting flexible linkers between the metal binding sites (Hu, Wang et al. 2007).  This prevented hairpins in the structure.  The second gBlock designed was for precious metals, not including platinum and palladium.  The corresponding gene sequences specific for gold, silver, aluminum and titanium peptides (Seker, Demir 2011) were used.  The design of this differed from the other gBlock as myoglobin was hijacked, the loops removed and replaced with our metal binding peptides.
Synthesizing Metal Binding Proteins
For detail on our laboratory procedures, please refer to our Protocols .
Metal binding peptide procedure
Sequences were found for metal binding peptides. The gene sequences for the production of the metal binding peptides were very short. Therefore we were able to have each peptide gene synthesised as two complementary oligonucleotides. We then annealed the primers together. The product of this reaction had the relevant sticky ends for insertion of the sequence into the plasmid vector.
Primers
All primers are notated 5' to 3'.
Ni1 forward
AATTCCATCATCACCATCACCACC
Ni2 forward
AATTCATGTGTACAACATGCGGTTGCGGTGAAGGCC
Pd forward
AGCGTGACCCAGAACAAATAT
Ni1 reverse
TCGAGGTGGTGATCGTGATGATGG
Ni2 reverse
TCGAGGCCTTCACCGCAACCGCATGTTGTACACATG
Pd reverse
ATATTTGTTCTGGGTCACGCT
For primer annealing in the PCR, the primer sequences were combined in the following way:
GST Forward and Ni1 Reverse
GST Forward and Ni2 Reverse
GST Forward and Pd Reverse
GST Forward and Pt Reverse
Vector
Our vector of choice was pGEX-6p-1, as it contains the genetic information needed to produce GST (258, 992) and is ampicillin-resistant.
pGEX was digested with EcoR1(955) and Xho1(970).
The primers designed to create our short Ni1, Ni2, Pd and Pt binding inserts were annealed.
The primer dimers were ligated into the cut pGEX vector on the N terminus of GST.
The ligated vector was transformed into DH5-α E. coli cells.
These cells were grown up and a miniprep was carried out.
The primer dimer insert was too small to detect using a diagnostic digest.
The miniprep product was used to transform the engineered pGEX vector into BL21 gold E. coli cells.
These cells were induced and GST+Ni1, Ni2, Pd or Pt was overexpressed.
GST+ Ni1 and Ni2 were used to prove functionality in our engineered protein.  They bound to nickel beads successfully, and further characterisation was carried out by the University Geology department using their Inductively coupled plasma mass spectrometer.
The Pd and Pt constructs were not fully characterised .
gBlock procedure
Two gBlock sequences were designed and ordered from IDT.
Precious metal scavenging
×
Precious metal scavenging
>500 bp
EcoR1 Au1 Ag1 Al1 Ti1 Au2 Ag2 Ni1 Xho1 Linker
GAATTC
GTTTCTGGTTCTTCTCCGGACTCT
GCGCTGGGCGGCATTCTGAAAAAAGGCCATCATGAAGCGGAAATTAAA
GCGTACTCTTCTGGTGCGCCGCCGATGCCGCCGTTC
CTGGAATTTATTTCCGAATGCATTATTCAG
GTTCCGTCTTCTGGTCCGCAGGACACCCGTACCACC
GCGGATGCGCAGGGCGCGATGAATAAAGCGCTGGAACTGTTTCGTAAAGATATGGCGTCC
CGTAAACTGCCGGACGCGCCGGGTATGCACACCTGG
TCCGAAGATGAAATGAAAGCGTCCGAAGATCTGAAAAAACATGGCGCG
ACCGGTACCTCTGTTCTGATCGCGACCCCGTACGTT
TCCATGCAGCTGTCCCAGCTGGAAGGCATT
ATCCGTCCGGCGATCCACATCATCCCGATCTCTCAC
TCCATGCAGCTGTCCCAGCTGGAAGGCATT
CATCATCACCATCACCAC
CTCGAG
Close
Toxic metal scavenging
×
Toxic metal scavenging
>500 bp
EcoR1 Hg1 Linker 1 Cd1 Linker 2 Co1 Linker 3 Hg1 Linker 1 Co2 Linker 2 Ni1 Xho1
GAATTC
ggcggcacccuggcggugccgggcaugaccugcgcggcgugcccgauuaccgugaaaaaaggcggcugg
TccaTgcagcTgTcccagcTggaaggcaTT
TTTGGATCCATGGAATGTGAATGTGAATGTGAATGTGAATGTGAATGTGAATGTGAGTGTGAATGTGAGTGCGAATGCGAA
CTGATTGCGTCCCTGCAGGGC
CACTCTGTTCGTTGGCTGCTGCCGGGTGCGCACCCG
ATGCCGCCGTCCCAGCTGGAAGGCATT
ggcggcacccuggcggugccgggcaugaccugcgcggcgugcccgauuaccgugaaaaaaggcggcugg
TccaTgcagcTgTcccagcTggaaggcaTT
AAACTGCACTCTTCTCCGCACACCCTGCCGGTTCAG
CTGATTGCGTCCCTGCAGGGC
CATCATCACCATCACCAC
CTCGAG
Close
The gBlocks alone were ligated into the iGEM pSB1C3 vector for submission.
The ligation was successful but the BioBricks remain uncharacterised.
Results
We have been able to submit two fully characterised BioBricks into the Registry of Standard Biological Parts as well as three functioning, yet uncharacterised, BioBricks.
We were first able to see the Ni1 and Ni2 proteins had been expressed as they were able to be purified on Ni beads.  This meant that any protein that was eluted from the beads would then be a Ni binding protein.  Both Ni binding proteins were seen after elution.
Using the results obtained by the UV-vis spectrum, seen on the graphs below, we were able to decipher that both of the Ni binding proteins, Ni1 and Ni2, had bound to Ni once again. This time it was shown using the shift in the λmax of the Ni solution.
As a further characterisation step we looked at the proteins using an ICP-MS.  This will allow us to see, in parts per billion, how much Ni was bound to the proteins.  The results from this method of characterisation are not yet available to us, but will be ready in time for the European iGEM Jamboree 2012 presentation and poster.
Graphs
Graph  1: NiCl2 solution
This graph shows the λmax of the NiCl2 solution in water with a peak at 394 nm.  The λmax of this peak will not change regardless of concentration.  The concentration of 25 mM was used so the absorbance was under one, and so that the amide bonds were not intruding with their absorbances, 190-230 nm.
Graph 2: Metal binding protein
This graph shows the absorbance of the metal binding protein Ni2.  We can see that in the regions where the NiCl2 will be unobstructed by any absorbance of the NiCl2.
Graph 3: Ni binding protein with Ni bound
This graph shows the change in the  λmax to 390 nm.  As stated previously, the λmax will not change with changes of concentrations, and there aren't any absorbances in this region of the Ni2 protein to obstruct the readings.  This, therefore, must mean the Ni is bound to the protein.
Biobricks
Biobrick
Short name
Description
Length
BBa_K925002
HisTagGST
This part codes for glutathione S-transferase (GST) protein with a histidine tag attached to the end. This provides a double ended tag protein and proof that the concept of attaching small metal binding peptides to the end of GST can create a protein with two active binding sites. This part is fully characterised.
669
BBa_K925004
NiTagGST
This part codes for GST and has a alternative nickel binding peptide attached to the end of the protein. This double ended protein binds to both GST and to nickel and is fully characterised.
681
BBa_K925005
PdTagGST
This part codes for GST with a palladium binding peptide attached to the end of the protein. The part is not yet characterised.
666
BBa_K925006
MyPrecious
This part codes for a gBlock that was designed by hijacking the structure of myoglobin and replacing the loops of the structure with precious metal binding peptides. The part was designed to bind gold, silver, aluminium and titanium. The part is not yet characterised.
479
BBa_K925007
MyToxic
This part codes for a gBlock that was designed by building flexible linker sequences around peptide binding sequences for toxic metals. The part was intended to scavenge cobalt, cadmium and mercury. The part is not yet characterised.
888
References
Bae, W., Chen, W., Mulchandani, A. and Mehra, R.K., 2000. Enhanced bioaccumulation of heavy metals by bacterial cells displaying synthetic phytochelatins. Biotechnology and bioengineering, 70(5), p.518-524.
Chan Chung, K.C., Cao, L., Dias, A.V., Pickering, I.J., George, G.N. and Zamble, D.B., 2008. A high-affinity metal-binding peptide from Escherichia coli HypB. Journal of the American Chemical Society, 130(43), p.14056-14057.
Deplanche, K., Murray, A., Mennan, C., Taylor, S. and Macaskie, L., Biorecycling of Precious Metals and Rare Earth Elements .
Hu, X., Wang, H., Ke, H. and Kuhlman, B., 2007. High-resolution design of a protein loop. Proceedings of the National Academy of Sciences, 104(45), p.17668.
Jollie, D., 2010. Platinum 2010. Johnson Matthey.
Pritchard, H.M. and Fisher, P.C., 2012. Identification of platinum and palladium particles emitted from vehicles and dispersed into the surface environment. Environmental science & technology.
Seker, U.O.S. and Demir, H.V., 2011. Material binding peptides for nanotechnology. Molecules, 16(2), p.1426-1451.
Song, L., Caguiat, J., Li, Z., Shokes, J., Scott, R.A., Olliff, L. and Summers, A.O., 2004. Engineered single-chain, antiparallel, coiled coil mimics the MerR metal binding site. Journal of Bacteriology, 186(6), p.1861-1868.
White, B.R., Liljestrand, H.M. and Holcombe, J.A., 2007. A ‘turn-on’FRET peptide sensor based on the mercury binding protein MerP. Analyst, 133(1), p.65-70.
×
Figure zoom: pGEX-6P-1 vector
×
Graph  zoom: NiCl2
×
Graph  zoom: NiCl2
×
Graph  zoom: Protein
Back to top
University of St Andrews, 2012.
Contact us: igem2012@st-andrews.ac.uk , Twitter , Facebook
This iGEM team has been funded by the MSD Scottish Life Sciences Fund. The opinions expressed by this iGEM team are those of the team members and do not necessarily represent those of Merck Sharp & Dohme Limited, nor its Affiliates.
Retrieved from " http://2012.igem.org/Team:St_Andrews/metal-binding "
Recent changes
What links here
Related changes
Special pages
My preferences
Printable version
Permanent link
Privacy policy
Disclaimers
