<HTML xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en" dir="ltr">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki  ltr ns-0 ns-subject page-Team_St_Andrews_Notebook"><DIV id="globalWrapper"><DIV id="top-section"><DIV id="p-logo"><A href="/Main_Page" title="Main Page">&quot;
	    </A></DIV><DIV id="menubar" class="left-menu noprint"><UL><LI class="selected"><A href="/Team:St_Andrews/Notebook">Page               </A></LI><LI class="new"><A href="/wiki/index.php?title=Talk:Team:St_Andrews/Notebook&amp;action=edit&amp;redlink=1">Discussion               </A></LI><LI><A href="/wiki/index.php?title=Team:St_Andrews/Notebook&amp;action=edit">View source               </A></LI><LI><A href="/wiki/index.php?title=Team:St_Andrews/Notebook&amp;action=history">History               </A></LI><LI style="color:#808080;cursor:default">teams</LI></UL></DIV><DIV class="right-menu noprint" id="menubar"><UL><LI id="pt-login"><A href="/wiki/index.php?title=Special:UserLogin&amp;returnto=Team:St_Andrews/Notebook" title="You are encouraged to log in; however, it is not mandatory [o]" accesskey="o">Log in</A></LI></UL></DIV><DIV id="search-controls" class="noprint"><FORM action="/Special:Search" id="searchform"> </FORM></DIV></DIV><DIV id="content"><H1 class="firstHeading">Team:St Andrews/Notebook</H1><DIV id="bodyContent"><H3 id="siteSub" class="noprint">From 2012.igem.org</H3><DIV class="navbar navbar-fixed-top"><DIV class="navbar-inner"><DIV class="container"><A class="brand" href="/Team:St_Andrews">St Andrews iGEM '12</A><DIV class="nav-collapse"><UL class="nav"><LI class="dropdown"><A href="#" class="dropdown-toggle" data-toggle="dropdown">
                    Projects
                    </A></LI><LI><A href="/Team:St_Andrews/metal-binding">Metal binding protein</A></LI><LI><A href="/Team:St_Andrews/Omega-3-synthesis">Omega-3 fatty acid synthesis</A></LI><LI><A href="/Team:St_Andrews/Modelling">The mathematics of ω-3</A></LI><LI><A href="/Team:St_Andrews/Human-practices">Scientific impact of iGEM</A></LI></UL><LI><A href="/Team:St_Andrews/Parts">Biobricks</A></LI><LI class="dropdown"><A href="#" class="dropdown-toggle" data-toggle="dropdown">
                    Notebook
                    </A></LI><LI><A href="/Team:St_Andrews/Notebook">Journal</A></LI><LI><A href="/Team:St_Andrews/Lab-book">Lab book: Protocols</A></LI><LI><A href="/Team:St_Andrews/Safety">Safety</A></LI><LI><A href="/Team:St_Andrews/Public-outreach">Outreach</A></LI><LI class="dropdown"><A href="#" class="dropdown-toggle" data-toggle="dropdown">
                    Team
                    </A></LI><LI><A href="/Team:St_Andrews/Team">Members</A></LI><LI><A href="https://igem.org/Team.cgi?year=2012&amp;team_name=St_Andrews">Profile</A></LI><LI><A href="/Team:St_Andrews/Attributions">Attributions</A></LI></DIV></DIV></DIV></DIV><DIV class="container" id="content-container"><HEADER class="jumbotron subhead" id="overview"><H1>Journal</H1><P class="lead">Our iGEM Story</P><DIV class="subnav"><UL class="nav nav-pills"><LI><A href="#december-march">December-March</A></LI><LI><A href="#june">June</A></LI><LI><A href="#july">July</A></LI><LI><A href="#august">August</A></LI></UL></DIV></HEADER><SECTION id="december-march"><H2>December-March</H2><H3>December 2011</H3><P>Team St Andrews forms, uniting nine students, seven world class researchers and four PhD advisors from disciplines as diverse as Computer Science and Physics, to Biology, Medicine and Chemistry.  </P><H3>January - March 2012</H3><P>Applications for sponsorship are made to specifically chosen businesses and organisations with an interest in advancing the Life Sciences.  Very quickly, &quot;BioSilta&quot;, &quot;GenScript&quot;, &quot;Clontech&quot;, &quot;Geneious&quot;, &quot;Integrated DNA Technologies&quot; and &quot;Thermo Fisher&quot; pledge their support and Team St Andrews grows...</P><H3>10 March 2012</H3><P>National Science and Engineering Week explodes in Fife with a regional &quot;Science Discovery Day&quot;.  Team St Andrews works to convey fundamental concepts in Genetic Engineering and Synthetic Biology to members of the public in new and exciting ways.  The interactive &quot;Codon Game&quot;, the 3 Dimensional visualisations of DNA and DNA polymerase 3 and display &quot;E. Coli: under the Microscope&quot; are all well received.  Children and adults alike are fascinated when DNA is extracted from bananas, using everyday kitchen utensils, before their eyes.</P><H3>April 2012</H3><P>Brainstorming sessions are held as the team researches project ideas.  Some promising titles include:</P><UL><LI><P>&quot;E. Coli and Omega 3: A Project to Feed the Minds of Our Generation&quot;</P></LI><LI><P>&quot;Enzymatic Methane Conversion in Cows: a Sweet Smelling Approach to Reducing Climate Change&quot;</P></LI><LI><P>&quot;Resurfacing Science, Resurfacing our Roads: Cell Factories and Metal Binding Proteins Recover Pavement Platinum&quot;</P></LI><LI><P>&quot;Project Bio-logic-al: Optimizing Soil Composition by Method of Biological Computation&quot;</P></LI></UL><H3>27 April 2012</H3><P>Team member Josi presents &quot;Spider Mutants and Bioterrorism - an Overview of Synthetic Biology as an Emerging Scientific Discipline&quot; to a “TEDx” audience of over eighty scientists and non - scientists akin.  Josi views the field as &quot;ground breaking&quot; and by the end of her talk, members of her audience too admit surprise at the wealth of possibilities that this new research area makes available.  There is excitement at the tantalizing proximity of reality of these ideas.</P><H2>May 2012</H2><P>Project ideas are discussed in greater detail and are filtered until only two research topics remain.  Those preferred ideas are: the production of Omega 3 Fatty Acids by E. Coli Cells (&quot;E. Coli and Omega 3: A Project to Feed the Minds of Our Generation&quot;) and the production of Metal – Binding Proteins (&quot;Resurfacing Science, Resurfacing our Roads: Cell Factories and Metal Binding Proteins Recover Pavement Platinum&quot;).</P></SECTION><SECTION id="june"><H2>June</H2><H2><EM>Week 1</EM></H2><H3>4 June 2012</H3><P>Full time work on Team St Andrews' iGEM Project finally begins!  After an initial Group Meeting, two thirds of our student members continue in depth research into those project ideas generated previously; the rest begin work on Team St Andrews' Wiki.</P><H3>5 June 2012</H3><P>&quot;CLC bio&quot; and &quot;Epoch Life Science&quot; are the latest businesses to offer support to Team St Andrews. The former promises CLC bio Main Workbenches to members of the team while the latter offers products and expertise in DNA/ RNA preparation for molecular manipulation.</P><DIV class="well metal"><H2>&quot;Metal Binding Protein&quot; Weekly Summary</H2><P>&quot;After a short meeting (which was dominated by the excitement of receiving our very own iGEM pins) we decided to split our iGEM family of nine into three groups.  Josi, Veronica and Yiwang were to look into Omega-3 production; Constantine, Michael and I were to investigate Metal Binding Proteins and Antti, Alexey and Zoe were to focus on setting up the wiki.</P><P>The wiki team duly produced a sophisticated template.</P><P>The research teams, however, were unanimously agreed that biology is hard.  Our motif quickly became standardised.  Read, Think, Re-read, Re-think, Repeat.</P><P>After three days, our previously nomadic team settled in the University’s new Biomedical Sciences Research Complex.  This proved to be a good move as it was followed by breakthroughs on all fronts.  Metal binding peptides were located in their tens and the decision was made to try and express them on the surface of cells in display proteins as well as cytosolically.  A membrane protein was found, preliminary peptides selected, and a primer design tutorial set up for the following Monday.  All this success left us with one very important and so far unanswered question- how exactly are we going to assay all this?&quot; (Team Member Hannah Taylor, Week One Report, 12/06/2012)</P></DIV><DIV class="well omega"><H2>&quot;ω-3&quot; Weekly Summary</H2><P>Research into polyunsaturated fatty acids has been active for over 15 years, with successes varying from creating transgenic plants enriched in ω-3 fatty acids to expressing an entire synthetic pathway from a gene clusters extracted from marine bacteria. We want to expand on this area of research by attempting to express a aerobic synthesis of unsaturated fatty acids in E.coli, which has never been done before. By combining the genes of the cyanobacteria Synechococcus and trypanosome T. brucei into E.coli, this vector should be able to synethise a unsaturated fatty acid up to at least 22-carbon length!</P><P>This week, our team has been focusing on preliminary research by reading relevant scientific papers and understanding the various pathways and methods of recombination. We’re focusing on groundwork research done in the early 1990s that appears to have fallen off the radar, and are excited to see where this path will take us!</P></DIV><H2><EM>Week 2</EM></H2><DIV class="well metal"><H2>&quot;Metal Binding Protein&quot; Weekly Summary</H2><P>Using E. Coli DH5-alpha: good at replicating vectors but not good at expression
Using pGEX-6P-1 vector with internal GST tag, selected with ampicillin
http://ecoliwiki.net/colipedia/index.php/pGEX-6P-1 (further information via the “source” link)
Purpose:
</P><P> First day: incorporate desired vectors into vector-replicating E.Coli and incubate them on selective plates to select out recombinant colonies with antibiotics</P><P>Second day: incubate the selected colonies in large quantities for vector harvesting</P><P>Third day: harvest vector from cell pellet thus pure vector miniprep.</P></DIV><DIV class="well omega"><H2>&quot;ω-3&quot; Weekly Summary</H2><P>This week, we took the first steps to put our grand scheme into laboratorial action! It also occurred to us that if we started calling ourselves DJ Omega 3 &amp; the Fatty Acids, we would intimidate the other squad.</P><P>First of all, we chose a vector to work in, pET-15B. We started looking into promoters suitable for both the vector and the chain of genes we want to express, and figured out how to use the Registry of Standard Parts. The next step was to design primers for all four of the genes of our pathway - and that was certainly a steep learning curve!</P><P>Preliminary tasks done, let’s head to the lab! We carried out a transformation, growing the vector in a DH5-α strain of E. coli with ampicillin as the antibiotic marker. Well, we attempted to carry out a transformation. It failed, much to the glee of the Metal Mickies. Fortunately, we were able to use some pET-15b grown in E. coli for a different purpose. The colonies were allowed to grow, and we then carried out a mini-prep to isolate the vectors. Visualization was done by running the DNA on an agarose gel and using UV light. Once we were sure the transformation was successful, it was time to add the restriction enzymes!</P><P>Also, a PCR was run with the genomic DNA of Leishmania major, the trypanosomatids that are providing us with the 4th gene of our synthesis pathway. We ran 5 samples total: 3 using high-fidelity PCR at different temperatures, and 2 of Clontech’s PCR kits at 2 temperatures. We added the primers for Δ6 elongase (GenBank LmjF32.1160). This gene is around 1.200 kb large. The results show that the Clontech kit run at 48° C gave the strongest result.</P></DIV><H2><EM>Week 3</EM></H2><DIV class="well metal"><H2>&quot;Metal Binding Protein&quot; Weekly Summary</H2><P>Having tried to digest our pGEX without diluting the buffer, and then running the DNA gel using dye instead of ladder, we made a second attempt at digesting our vector. Thankfully, this one went smoothly, and the pGEX was cleaned up using our epoch kit.  As our insert is so small, we were able to simply order complementary primers with the sequence we wanted in them and then anneal these to make a primer dimer. We ordered primers that contained peptide sequences to bind nickel, platinum and palladium. We then progressed to ligating our primer dimer into the clean cut pGEX vector. These ligations were transformed into DH5 cells and left overnight. The nickel transformation was successful and a PCR was done on the product to see if it did contain our nickel insert. This, however, was a disaster. But more on that later.  The platinum and palladium attempts yielded no colonies. More pGEX vector was grown up for miniprepping and digestion, and the nickel primer dimer was religated into the remaining original clean cut pGEX and transformed.</P></DIV><DIV class="well omega"><H2>&quot;ω-3&quot; Weekly Summary</H2><P>This week has been a succession of PCR, digestion, PCR, PCR, digestion, vector growth, more PCR.</P><P>We can confidently say that we now know how to set up and run agarose gels! Looking on the bright side of things, we made a lot of mistakes these last few days that we hope to evade in the future: buffers need to be diluted, instructions followed, labels clearly written, and don’t lick the metal poles in the -80° freezer. Just don’t.</P><P>In order to learn all these valuable lessons, we had to sacrifice one thing: results. Many of our PCRs were unsatisfactory, plasmids did not digest properly, E. coli wouldn’t take up the vectors, and it turns out we have a knack for mysteriously making PCRs disappear on agarose gels.</P><P>Not until Friday after lunchtime did we finally have some successes:  we found a PCR and the conditions that worked for the gene we are currently working on, the D6 elongase from L. major. Plus, we managed to grow a decent amount of vector and restrict it with enzymes. We hope that we’ve set a good foundation to build on next week – nowhere to go but up!</P></DIV><H2><EM>Week 4</EM></H2><DIV class="well metal"><H2>&quot;Metal Binding Protein&quot; Weekly Summary</H2><P>The clean cut pGEX was used to try and optimise our PCR conditions. However, all our preparations came back blank. After reviewing our protocol, and making sure we had done everything correctly, we turned to our primers. We had been using a primer that did not bind where we thought it did. We fixed this and started using our GST Forward primer in combination with the reverse primers that we had used to make our primer dimers. Taking a break from the lab, the team travelled through to Edinburgh to meet iGEM Dundee and iGEM Edinburgh to swap ideas and hear a few synthetic biology speakers.  On our return, and using the correct primers a colony PCR was run and bands were seen in lanes for platinum, palladium and nickel. We seem to be getting somewhere! Alongside this we made our first protein gels and prepared some uncut pGEX to run on them after inducing it with IPTG to see if our GST tag (with our primer dimer peptides ) would be over expressed. The gel will be run on Monday.</P></DIV><DIV class="well omega"><H2>&quot;ω-3&quot; Weekly Summary</H2><P>Squad Omega managed to create its very first preliminary BioBrick<SUP>TM</SUP>! We successfully inserted the L. Major D6 elongase into a pET-15a vector, and have sent it off to be sequenced over the weekend.  At the moment, we are working on creating an assay to monitor this gene’s function. In order to do this, we have grown three different types of protein expression E. coli: BL 21 BL 21 codon stop, BL 21 plyS. After growing them, we will be analysing their natural 18-1 lipid content, in order to create a “background”. Once we have inserted our foreign proteins into our expression vectors, we will be able to compare the types of lipids synthesised.
</P><P>We have also started working on another gene of our sequence, D15 desaturase from T. cruzi. Originally, we had planned to use this gene from the organisms Synechocystis; but as the genomic DNA is not due to arrive for several weeks, we have found a suitable, more accessible alternative. At the moment, we are running a PCR after designing primers to isolate the gene-of-choice. 
</P></DIV></SECTION><SECTION id="july"><H2>July</H2><H2><EM>Week 5</EM></H2><DIV class="well metal"><H2>&quot;Metal Binding Protein&quot; Weekly Summary</H2><P>This week we were able to see through our protein gel, loading it using what is here known as the ‘Excalibur method’ and letting it run, before staining it with coomasie over night and destaining it in the morning. The gel showed our GST protein band at 26 KDa, but without any over expression in comparison to the other cellular proteins. 
We had grown up the successful colonies from the colony PCRs done the previous week and minipreped them. Some of this miniprep was sent to GATC in Germany for sequencing and some was used to transform the plasmids with the insert in them into BL21 cells in order to over express them. This however was unsuccessful and investigation proved that our cells had not been lysing properly and releasing their plasmids during our minipreps. Some alteration in procedure and a further DNA gel showed provided a small amount of DNA, some of which was used for transformation into DH5.
Over the weekend we got colonies for the Ni, Pd and Pt and grew them up for expression on Monday. 
Somewhere along the way a second set of primers to make a nickel binding peptide was made and ordered, so we now have nickel 1 (Ni1) and nickel 2 (Ni2). The 2Ni primer insert was annealed and ligated into pGEX then transformed into DH5.</P></DIV><DIV class="well omega"><H2>&quot;ω-3&quot; Weekly Summary</H2><P>We received our sequencing results from our preliminary BioBrick<SUP>TM</SUP> this week, and ---- <I>nothing</I>. The insert has a 99% protein match when using BLAST; unfortunately, this match is to a mevalonate kinase of a trypanosome that we haven’t been using. Some sleuthing in slabs and moving of freezers uncovers the sad truth: the vector we have been using since week 1 was not in fact empty. So 4.5 weeks of work amount to 4.5 weeks of gaining lab experience, but no more. </P><P>There is some good news among the bad, however. The genomic data we ordered for <I>Synechocystis sp</I>. won’t arrive until the end of July, but Dr Markus Gierth at the University of Cologne was kind enough to send us some of his colonies. Excited as we were to receive green smears, it also gave us additional DNA sources.  After a Taq colony PCR, we ran high-fidelity PCRs to isolate our genes of choice. At the same time, we are also cloning D6 from <I>L. major</I> and <I>T. cruzi</I>. We hope that this simultaneous approach will give us fast results to make up for the time we have lost. </P></DIV><H2><EM>Week 6</EM></H2><DIV class="well metal"><H2>&quot;Metal Binding Protein&quot; Weekly Summary</H2><P>We checked out the optical density (OD) of the six colonies (three each, 1Ni, Pd and Pt) we had grown up over the weekend. When these were all between 0.5 and 0.8 we added IPTG to induce enhanced expression of GST in our plasmid. These were left for 3 hrs and then the results run on a protein gel and stained overnight before being destained in the morning. This presented prominent bands at 26KDa. 
This meant we were able to use the remainder of the lysed cells that had been run on the gel to run another one and this time attempt a western blot
As we did this we also used colony PCR to discover that our 2Ni colonies did not contain our insert, so these were re-ligated into the pGEX vector and transformed into DH5. 
We cut more pGEX with our restriction enzymes (Xho1 and EcoR1) and miniprepped more pGEX  and more of our nickel containing plasmid and run them on a gel getting positive results for both (eventually). 
</P></DIV><DIV class="well omega"><H2>&quot;ω-3&quot; Weekly Summary</H2><P>This week has been full of ups and downs for Squad Omega. We started off with a lot of hope when we found that the digestion of all necessary genes and plasmids worked. So we were ready to go for ligation and transformation into DH5-α cells. Much to our surprise, these transformations were then largely unsuccessful. However, we decided to pick some colonies from our plates and cultivate them to perform a miniprep. After analysis of our plasmids with numerous digestions and PCR reactions, and trying to re-do ligations, we finally were able to obtain some colonies containing our D15 gene, D6 and ELO6. D15 and D6 were sent off for sequencing, while for ELO6, we are still trying to confirm its presence in our colonies. Hopefully, next week we will be able to send off ELO6 and the missing D12 for sequencing, and start doing some protein expression. </P><P>Even though in the beggining of this week we felt like we needed a motivational speaker telling us not to give up, now we feel like we are finally getting somewhere and making progress.</P></DIV><H2><EM>Week 7</EM></H2><DIV class="well metal"><H2>&quot;Metal Binding Protein&quot; Weekly Summary</H2><P>We ran a PCR of our Ni 2 vectors using the GST forward and the Ni 2 reverse primer to see if they were correct inside the vector. The results showed that they were not present. 
We decided to do a giant colony PCR of the colonies on all the plates we have gathered over the last 6 weeks, intending to keep those that are of use and bin the others. This resulted in us finding plates with successfully ligated Ni1 and Ni2 inserts. These colonies were grown up in LB. However, successful Pd and Pt inserts were not found. The Pd and Pt inserts were religated into the pGEX vector. 
The sucesfull overnights of Ni1 and Ni2 were miniprepped and transformed into BL21 for protein expression trials. 
We decided to try using a gBlock synthesised by IDT to make a 500bp protein in which we would insert our own metal binding peptides. Two of these were designed, one for binding precious metals and one for binding toxic metals. This was designed.</P></DIV><DIV class="well omega"><H2>&quot;ω-3&quot; Weekly Summary</H2><P>This week one of our team members left for a conference in Slovenia, and took advantage of the opportunity to meet with the Slovenian iGEM team. Despite having one less person in the lab, the results obtained were quite encouraging! Standard lipid samples of BL21 with no insert prepared a few weeks ago were analysed using the Fatty Acid Methyl Esters (FAME) Analysis technique through Gas Chromatography-Mass Spectrometry(GC-MS). We showed BL21 cells produced some 18:1, but no polyunsaturated fatty acids…So we already have results that will act as a base reference to compare with transformed cells! After doing some reading, we also realised that the 18:1 the cells were producing was not the one we need (with a double bond at the Δ9 site), so in the future, the appropriate 18:1 will be fed to the cells for the Omega-3 pathway to occur. After many ligations, transformations, colony PCRs and incubations, it seemed like we finally ligated ELO6 and Δ12 into our vectors, but diagnostic tests have only shown to be positive for Δ12 desaturase. We transformed BL21 cells with the vector containing this gene so that next week protein expression and a lipid profile of these cells could be done. </P></DIV><H2><EM>Week 8</EM></H2><DIV class="well metal"><H2>&quot;Metal Binding Protein&quot; Weekly Summary</H2><P>BL21 cells containing Ni1 and two samples of Ni2-  Ni2(1) and Ni2(2 )- had been induced and grown overnight at 16 degrees. 
The resuspended cells were split into two eppendorfs, one was frozen. 
The other was used to prepare samples for running on a protein gel, each sample was run twice.
This was then transferred onto a membrane and blotted in 5% milk overnight in preparation for a western blot the following day 
The membrane was taken from the rocker in the cold room and left in 10% milk solution at room temperature for an hour. It was then cut in half so that each section contained the 3 different samples. 
This allowed us to do two different western blots, one with anti his antibodies, that should bind to our two new nickel peptides and anti GST which should bind to all our samples due to the GST tag our protein is attached to. 
However, the results of the western were proven to be inconclusive
BL21 cells that were frozen on Monday were defrosted and prepared to run on a protein gel. The gel was then stained with coomassie blue. Colony PCR was done in order to test Pt and Pd transformations. Colonies which showed positive results were grow n up and a miniprep was done. Minipreps were used to transform BL21 expression cells.</P></DIV><DIV class="well omega"><H2>&quot;ω-3&quot; Weekly Summary</H2><P>This week we have been, once again, puzzled by science. After insertion of our Synechocystis delta-12 desaturase into our vector, we sent it off for sequencing. Result? Nothing. However, while waiting for sequencing results, we did some protein expression and analysed the lipid profile of BL21 cells containing our desaturase. Protein expression showed that the desaturase was overexpressed in BL21 cells, and these cells also showed a different lipid profile than background BL21 cells, with clear indication of desaturation. Thus, even though we have shown the protein is being expressed and having its own desaturase activity, we are not able to submit a BioBrick<SUP>TM</SUP> because, according to sequencing, we have nothing.</P><P>We have also started working with pET-duet vectors. We have ligated 3 of our 4 genes into specific cloning sites in this vector, hoping that we will be able to insert 2 genes simultaneously in the same vector in the future.</P></DIV><SECTION id="august"><H2>August</H2><H2><EM>Week 9</EM></H2><DIV class="well metal"><H2>&quot;Metal Binding Protein&quot; Weekly Summary</H2><P>Platinum and palladium samples were used for DNA gel and it showed
positive results. Colony PCR for palladium colonies was done. It gave
negative results. More vector was cut and palladium and platinum
ligations were repeated. Toxic and precious G- block ligation was done
in to TOPO vector. All ligations were used in transformation of DH5α
cells.</P></DIV><DIV class="well omega"><H2>&quot;ω-3&quot; Weekly Summary</H2><P>“Week 9! ...Wait, what?  Week 9? Already? ” For some reason, we are unable to obtain a clean PCR of our Elongase 6, unlike in the first few weeks. Because we are already in week 9, and because this is the last gene to be used in our pathway we decided to forget about it (at least for now).  We ligated Δ12, Δ15 and Δ6 into our duet vectors. These were transformed into BL21 cells, which were induced with IPTG, fed with 18:1 fatty acid, and grown at 16C. These cells were used for analysis of protein expression and to obtain their lipid profile. We are also getting ready our duet vector containing Δ12 by digesting it with the appropriate restriction enzymes for ligation of Δ15 into the second cloning site, and in this way obtain the first two desaturases in our pathway in the same vector. After a lot of hard work this week, we are all looking forward to week 10 and obtaining results that will determine if our lab work these past 9 weeks was worth it or not!</P></DIV><H2><EM>Week 10</EM></H2><DIV class="well metal"><H2>&quot;Metal Binding Protein&quot; Weekly Summary</H2><P> Bl21 cells containing Ni 1 were induced and cell with Ni 2 insert
were lysed and Ni 2 protein was extracted using   Ni and GST beads.
Impurities were removed using spin columns and washed with PBS. Trial
run was done to optimize UV/Visible spectrum experiments.Ni1 and Ni2
protein solutions were mixed with NiCl2 solution for UV/Visible
spectrum experiments.   Colony PCR was done for Pt and Pd samples. 4
colonies with positive results were grown up in liquid medium and
miniprep was done. Mininipreps were used for DH5α transformation.
More expression cells were grown up and proteins were extracted for
UV/Visible spectrum experiments. Precious and toxic gBlockTM were
transform into BL21 expression cells.BL21 cell were induced.</P></DIV><DIV class="well omega"><H2>&quot;ω-3&quot; Weekly Summary</H2><P>A few words describing our final week of iGEM, by the Omega Squad: excitement, stress, happiness, frustration, impatience, back to happiness… So it seems like, except for Elongase 6, we were able to successfully ligate our genes into our duet vectors, and we have one duet vector both with Δ12 and Δ15. All of our minipreps were transformed into BL21 cells, proteins expressed, and the lipid extracts of cells in addition to extracts of membrane assays subjected to FAME analysis through GC-MS. We were really happy and satisfied with our results: (1) Cells expressing Δ12 desaturase showed 18:2 fatty acids when fed with 18:1. (2) Cells expressing Δ12 desaturase and Δ15 desaturase showed 18:3 fatty acids (our Omega-3!). (3) Cells expressing Δ6 desaturase showed showed 18:2 fatty acids when fed with 18:1. This enzyme was supposed to desaturate further 18:3 (the Δ15 desaturase product) in our pathway. However, because of time constraints we were unable to express this protein in the same cells as Δ12 and Δ15 desaturases were expressed, thus we couldn’t obtain the 18:4 fatty acid we expected in the beginning. However, we were able show the desaturation activity of Δ6 desaturase as 18:2 fatty acids were detected. The final step was inserting our 3 genes into the submission vectors, ready to be sent off as our new Omega-3 BioBricks<SUP>TM</SUP>! </P></DIV></SECTION></SECTION></DIV><DIV class="container" id="content-container"><FOOTER class="footer"><P class="pull-right"><A href="#">Back to top</A></P><P>University of St Andrews, 2012.</P><P>Contact us: <A href="mailto:igem2012@st-andrews.ac.uk">igem2012@st-andrews.ac.uk</A>, <A href="https://twitter.com/#!/StAndrewsiGEM12">Twitter</A>, <A href="https://www.facebook.com/StAndrewsiGEM12">Facebook</A></P><P><SMALL>This iGEM team has been funded by the MSD Scottish Life Sciences Fund. The opinions expressed by this iGEM team are those of the team members and do not necessarily represent those of Merck Sharp &amp; Dohme Limited, nor its Affiliates.</SMALL></P></FOOTER></DIV><DIV class="printfooter">
Retrieved from &quot;<A href="http://2012.igem.org/Team:St_Andrews/Notebook">http://2012.igem.org/Team:St_Andrews/Notebook</A>&quot;</DIV></DIV></DIV><DIV id="footer-box" class="noprint"><DIV id="footer"><UL id="f-list"><LI id="t-recentchanges"><A href="/Special:RecentChanges" title="Recent changes">Recent changes</A></LI><LI id="t-whatlinkshere"><A href="/Special:WhatLinksHere/Team:St_Andrews/Notebook" title="List of all wiki pages that link here [j]" accesskey="j">What links here</A></LI><LI id="t-recentchangeslinked"><A href="/Special:RecentChangesLinked/Team:St_Andrews/Notebook" title="Recent changes in pages linked from this page [k]" accesskey="k">Related changes</A></LI><LI id="t-specialpages"><A href="/Special:SpecialPages" title="List of all special pages [q]" accesskey="q">Special pages</A></LI><LI><A href="/Special:Preferences">My preferences</A></LI></UL></DIV><DIV id="footer"><UL id="f-list"><LI id="t-print"><A href="/wiki/index.php?title=Team:St_Andrews/Notebook&amp;printable=yes" title="Printable version of this page [p]" accesskey="p">Printable version</A></LI><LI id="t-permalink"><A href="/wiki/index.php?title=Team:St_Andrews/Notebook&amp;oldid=234201" title="Permanent link to this revision of the page">Permanent link</A></LI><LI id="privacy"><A href="/2012.igem.org:Privacy_policy" title="2012.igem.org:Privacy policy">Privacy policy</A></LI><LI id="disclaimer"><A href="/2012.igem.org:General_disclaimer" title="2012.igem.org:General disclaimer">Disclaimers</A></LI></UL></DIV></DIV></DIV></BODY></HTML>