"
Page
Discussion
View source
History
teams
Log in
 
Team:St Andrews/Omega-3-synthesis
From 2012.igem.org
St Andrews iGEM '12
Projects
Metal binding protein
Omega-3 fatty acid synthesis
The mathematics of ω-3
Scientific impact of iGEM
Biobricks
Notebook
Journal
Lab book: Protocols
Safety
Outreach
Team
Members
Profile
Attributions
Omega-3 fatty acid synthesis
Introduction
Project description
Recreating the pathway
Methods
Results
Biobricks
References
Introduction
ω-3 fatty acids are a key component of the human diet. Our team is recreating this synthetic pathway in E. coli, using genes from the cyanobacteria Synechocystis and the trypanosomatid Leishmania major. Combining the DNA code for elongase and desaturase enzymes, we are planning to convert a fatty acid with a single desaturation into highly valuable ω-3 fatty acids.
Project description
Figure 1: "Synechocystis sp."
Figure 2: "Trypanosome cruzi"
Figure 3: "Leishmania major"
Omega-3 fatty acids are an essential part of the human diet (Bender, Bender, 1999). Human beings, like all larger organisms, cannot synthesize ω-3 fatty acids. This is due to a lack of the enzyme Δ15 desaturase, which creates a double bond at the 15th carbon of a long-chain fatty acid. Certain micro-organisms, such as microalgae and cyanobacteria, do contain this desaturase and can thus directly synthesize ω-3 fatty acids (Arts et al., 2009). ω-3 fatty acids then enter the food chain – algae are eaten by fish, and seafood is subsequently the main source of ω-3 for humans (Tonon et al., 2002).
However, the current economic policies of overfishing are a serious contributor to marine biodestruction. As the human population is estimated to rise to 9.1 billion by 2050 (Cohen, 2003), pressure on fish stock will increase. Additionally, global warming will reduce the availability of ω-3 (Arts et al., 2009):  in response to high temperatures, microalgae produce less ω-3 desaturated fatty acids and more saturated fatty acids, as desaturated carbon chains cause a lower melting temperature in membranes (Garwin, Cronan, 1980). Thus, the combination of declining fish stock and a decrease in overall ω-3 fatty acids is making the supply for human nutrition a relevant issue.
Harvesting algae directly is costly and ineffective (Borowitzka, 1997). There is much potential in expressing a metabolic pathway for ω-3 fatty acid synthesis in E. coli, which is cheaper and more accessible.
Recreating the pathway
Figure 4: "The metabolic pathway to ω-3 fatty acids"
Figure 4 shows the elongation and desaturation enzymes necessary to convert an 18:1 fatty acid, into an poly-unsaturated fatty acid.
modified from Livore et al., 2006
E. coli naturally synthesize unsaturated fatty acids up to a carbon chain length of 18, with a single desaturation (18:1) (Marr, Ingraham, 1969). Valuable ω-3 fatty acids require a double bonds at the third carbon from the end of its carbon chain and can have >20 carbons.
In order to have E. coli synthesize ω-3 fatty acids, we needed to introduce enzymes that could elongate and desaturate fatty acid substrates (cf. Fig. 4).
The genes for Δ12, Δ15 (ω6) and Δ6 were obtained from Synechocystis sp., a cyanobacterium. The trypanosomatid Leishmania major provided the DNA for the ELO 6 gene. Additionally, we used Trypanosome cruzi as a secondary source of Δ12.
However, our first successful ligations of Δ12 did not provide us with the expected 18:2 fatty acid. We hypothesized that E. coli’s inherent 18-carbon chain fatty acid might not be suited as a substrate for Δ12 – the double bond is in a different position, the 11th. Therefore, we "fed" our cells with suitable 18:1, to then observe 18:2 fatty acid, and ultimately ω-3 desaturation, in the mass spec results!
Methods
The following genes were employed (please click for sequences and KEGG numbers):
Δ6 Synechocystis
×
Δ6 Synechocystis
ATGCTAACAGCGGAAAGAATTAAATTTACCCAGAAACGGGGGTTTCGTCGGGTACTAAAC CAACGGGTGGATGCCTACTTTGCCGAGCATGGCCTGACCCAAAGGGATAATCCCTCCATG TATCTGAAAACCCTGATTATTGTGCTCTGGTTGTTTTCCGCTTGGGCCTTTGTGCTTTTT GCTCCAGTTATTTTTCCGGTGCGCCTACTGGGTTGTATGGTTTTGGCGATCGCCTTGGCG GCCTTTTCCTTCAATGTCGGCCACGATGCCAACCACAATGCCTATTCCTCCAATCCCCAC ATCAACCGGGTTCTGGGCATGACCTACGATTTTGTCGGGTTATCTAGTTTTCTTTGGCGC TATCGCCACAACTATTTGCACCACACCTACACCAATATTCTTGGCCATGACGTGGAAATC CATGGAGATGGCGCAGTACGTATGAGTCCTGAACAAGAACATGTTGGTATTTATCGTTTC CAGCAATTTTATATTTGGGGTTTATATCTTTTCATTCCCTTTTATTGGTTTCTCTACGAT GTCTACCTAGTGCTTAATAAAGGCAAATATCACGACCATAAAATTCCTCCTTTCCAGCCC CTAGAATTAGCTAGTTTGCTAGGGATTAAGCTATTATGGCTCGGCTACGTTTTCGGCTTA CCTCTGGCTCTGGGCTTTTCCATTCCTGAAGTATTAATTGGTGCTTCGGTAACCTATATG ACCTATGGCATCGTGGTTTGCACCATCTTTATGCTGGCCCATGTGTTGGAATCAACTGAA TTTCTCACCCCCGATGGTGAATCCGGTGCCATTGATGACGAGTGGGCTATTTGCCAAATT CGTACCACGGCCAATTTTGCCACCAATAATCCCTTTTGGAACTGGTTTTGTGGCGGTTTA AATCACCAAGTTACCCACCATCTTTTCCCCAATATTTGTCATATTCACTATCCCCAATTG GAAAATATTATTAAGGATGTTTGCCAAGAGTTTGGTGTGGAATATAAAGTTTATCCCACC TTCAAAGCGGCGATCGCCTCTAACTATCGCTGGCTAGAGGCCATGGGCAAAGCATCGTGA
KEGG entry sll0541
Close
Δ12 Synechocystis
×
Δ12 Synechocystis
ATGACTGCCACGATTCCCCCGTTGACACCAACGGTAACGCCCAGCAATCCCGATCGCCCG ATTGCGGATCTCAAACTACAAGACATCATTAAAACCCTGCCCAAGGAATGCTTCGAGAAA AAAGCGAGCAAAGCCTGGGCTTCTGTTTTGATTACCCTAGGGGCGATCGCCGTGGGCTAT TTGGGCATTATTTATCTGCCCTGGTACTGCTTGCCCATTACCTGGATCTGGACAGGGACA GCCTTAACGGGGGCCTTCGTTGTCGGCCATGACTGTGGCCATCGCTCCTTTGCTAAAAAA CGCTGGGTCAATGATTTAGTGGGACATATCGCTTTTGCTCCCCTCATCTACCCTTTCCAT AGCTGGCGCCTACTCCACGACCACCATCACCTCCACACCAACAAAATTGAGGTTGATAAC GCCTGGGATCCCTGGAGTGTGGAAGCTTTCCAAGCCAGCCCGGCGATCGTCCGGCTTTTT TATCGGGCCATCCGGGGTCCCTTCTGGTGGACTGGTTCCATTTTCCATTGGAGCTTAATG CACTTCAAACTTTCCAACTTTGCCCAAAGGGACCGCAATAAAGTCAAATTATCCATTGCC GTTGTCTTCCTGTTTGCGGCGATCGCCTTTCCTGCCCTAATTATCACCACAGGGGTGTGG GGTTTCGTCAAATTTTGGCTAATGCCCTGGTTGGTGTATCACTTTTGGATGAGCACTTTT ACCATTGTGCACCACACCATTCCCGAAATTCGTTTCCGTCCCGCCGCCGATTGGAGTGCC GCCGAAGCCCAGTTAAATGGTACTGTTCACTGCGATTATCCCCGTTGGGTGGAAGTGCTC TGCCATGACATCAACGTCCATATTCCCCACCACCTCTCCGTTGCCATCCCTTCCTATAAC CTACGACTAGCCCACGGAAGTTTAAAAGAAAACTGGGGACCTTTTCTTTACGAGCGCACC TTTAACTGGCAATTAATGCAACAAATTAGTGGGCAATGTCATTTATATGACCCCGAACAT GGCTACCGCACCTTCGGCTCCCTGAAAAAAGTTTAA
KEGG entry 429257.20
Close
Δ15 Synechocystis
×
Δ15 Synechocystis
GTGCGTCTAGAAATTTCATCGCCTCAAACAAAGCTTCCTTACCCCAAAACTGAAGAATTA CCATTTACCCTCCAAGAGCTCAGAAACGCTATTCCAGCGGATTGTTTTGAGCCATCGGTA GTCCGGTCCTTGGGCTACTTTTTTTTGGATGTTGGTTTAATTGCCGGGTTTTATGCTCTA GCGGCCTACCTTGATTCCTGGTTCTTCTATCCGATTTTTTGGTTAATTCAGGGAACCCTA TTCTGGTCCCTGTTTGTGGTGGGCCATGATTGTGGCCATGGCTCCTTTTCCAAATCCAAA ACCCTTAATAATTGGATTGGTCATCTCAGCCACACGCCAATTTTGGTGCCTTACCATGGC TGGCGTATTAGTCATCGTACTCACCATGCCAACACGGGCAATATCGACACCGACGAAAGT TGGTATCCAGTGTCGGAGCAAAAATATAACCAAATGGCCTGGTATGAAAAACTTCTACGT TTTTACTTGCCTCTGATCGCCTACCCCATTTATCTATTTCGGCGATCGCCAAACCGGCAA GGCTCCCATTTCATGCCCGGCAGTCCCCTATTCCGTCCCGGAGAAAAAGCAGCTGTTCTC ACCAGCACCTTTGCCCTTGCAGCCTTTGTCGGCTTCCTTGGCTTTTTAACTTGGCAATTT GGCTGGCTATTTTTGCTGAAATTTTATGTTGCCCCCTACCTCGTGTTTGTGGTGTGGTTA GATTTGGTCACATTTTTACATCACACTGAAGACAATATCCCTTGGTATCGTGGTGATGAC TGGTATTTTCTCAAAGGTGCCCTCTCCACCATTGATCGGGATTACGGCTTCATTAACCCC ATTCACCATGACATTGGCACCCACGTCGCCCACCATATTTTCTCGAATATGCCCCACTAC AAGTTACGCCGGGCGACTGAAGCCATCAAGCCCATTTTAGGGGAATATTATCGATATTCT GACGAGCCAATTTGGCAAGCTTTTTTTAAGTCCTACTGGGCTTGCCATTTTGTTCCTAAT CAAGGTTCAGGGGTCTATTACCAATCCCCATCCAATGGTGGATATCAAAAGAAACCTTAA
KEGG entry s111441
Close
ELO 6 L. major
×
ELO 6 L. major
ATGAAGGTCATCGTCGCTTCTGGCCCGGACGGTGCCCGCAAGCACGAGGTGGAGCTGGCA GCCAACGCCACGCTCGCAGATCTGAAGAAGGCCTACCAACGGGGTGTGGACGTGCACCGC AAGTCGTTCAAGGTTCCCAGCGCGGAGTCGCCGCTGCCAGGTGCGGATAGTGGCAAGCTG CGCCCGAACCTCATTACTCTGTCAGATAAGGTGCCCCTGTCGCAGCAGGGGGTGAAGGAT GGCTCGGTGATCACTTACAAGGACCTCGGCCCGCAGATCGGCTACCGCACGGTGTTCTAC GTCGAGTATGCCGGCCCCATCGCCTTCATGCTGCTGTACGCCATGCGCCCTTCGCTCATC TACGGCTCTGCCCCGATGCCGGCTTACGGCTACACGCAGAAGCTATACATTGGCCTCTTC CTCGCCCACTTCTTAAAGCGCGAGCTCGAGTCCATGTTTGTGCACAAGTTCTCGCACCCA ACGATGCCGATGCGCAACATCTTCAAGAACTGCATCTACTACTGGTCCTTCGCCGCCTTC ATCGGCTACGTGCTGTGCAGTCCTTCATTCACGCCGACCAGCACCATGCAGTCAAACTTC GGCGCCGTGGTCATGGTCATCAACGAGCTGCTGAACTTCGCGGTGCACTACCAGCTTAGC ACGATGCGCAAGTCCGATGGTGACACCACCCGCAACGTGCCGCAAGGCCCTCTGTTCGCC TTCGTCTCGTGCCCGAACTACTTCTTTGAGATTATGTCGTGGGTGTCCTTTTCCATCGGC ACAAATATGTTATCCTCCTGGTTCTTCACACTCGCCGGTTTCGTGCAGATGGCGGACTGG GCGAAGAAGAAGCACCGGGGCTACGTCAAGGCGGACCCGGCCAATAAAAAGAAGGCCGCC ATTCTGCCCTTCATCATGTAG
KEGG entry LmjF32.1160
Close
Δ12 T. Cruzi
×
Δ12 T. Cruzi
ATGTCGCGGGTTGACAATTTAACTGTCGCGCCGGGTCCACCGGATGTCATGAAGGCGGTG CTTAAGTCTGGGCGCAACAAGGAGCACAATGTCATTGTGCCTTCCTCCGCACTGACCATTG GAGAAATTCAGGAGAAAATCCCGGTAAAGTACTTTGAACGCAACACTACCCGGAGCGTCA TGTTTCTTTTACGCGACCTCGCACAGGTGGCGTTGACGTACGCCATTATGTACGCCGTCGCC TTGCCACTTGCCACTTCCATGGAAGTTTCTGCAGCAAGAACGTTGGCCGACGGTGGGGTGT TATCATTGATGGCAGGTACAGCCATGACGACAGCGGCGTGGCTTTTAAAGGCAGTGTTGTG GGCGGTGTTTTGGTTTGTACAGGGACTTAACGGCACTGCGCTTTGGGTCCTGGCTCACGAA TGCGGCCATCAGGCGTTCAGCCCCATGAAAGGTTTGAATGACGCCGTTGGCATGATTTTGC ACTCTGCGCTGCTCGTGCCGTACCACAGCTGGCGCATCACCCACGGCGGCCACCACAAACA CACGAATCACCTCACGAAGGATCTTGTGTTTGTTCCAGACAAACGAAACGCGGTTGTGGAG CTCGTGGAGGAGGCGCCGCTGGTGTTATTAATTCAATTATTGCTGATTTTTCTCTTTGGTTG GCCGGCACATCTTCTTTTTAATGCCTCTGGACAGGAATTTGGCCGACTCGCGAGCCACTTTG ACCCCGGCGCTCCATTTTTCCGCAGCGAAGACCGTCACGATATTGTCCTGTCGAATGTTGGG ATTGTCAGCGCGTTATTTGTCATTTTCTCCAGCGTTTACCGCTTTGGTTTTACAAATGTTTTCT GCTGGTACATTGTACCGTACCTCTGGGTGAACTTTTGGCTG TGTACATTACATACCTGCAGCACACGGATATACGCATTCCTCACTACACACATGAGCACT GGACGTTTGTTCGCGGTGCATTGGCGGCTGTGGACAGGGACTACGGCTTTGTCCTTAACA CATGGCTCCATCACATCAATGATTCCCACGTGGTACATCACCTCTTTAGCAAAATGCCAC ACTACAACGCAATCCAGGTGACAAGAAAGTACATTCGTGAGATACTGGGTGCCACATACA TTACGGATGAGAGGTCACTGTGGAAGATGCTCTGGGAACAGCGTAGAGAGTGCCGCTATG TTGTTCCCGCAGAGGGCGTCTGTGTCTTTCATGGGTAA
KEGG entry 429257.20
Close
These genes were amplified through PCR (Promega, GoTaq HotStart) at temperatures 48°C and 56°C.
The genes were initially ligated into pET-15b vector. After a number of expression attempts, some initial conclusions were reached. All experiments done on ELO6 failed. Also, Δ12 from T. cruzi gave overall weaker results than the same gene from Synechocystis. As such, latter work was only carried out on genes from Synechocystis.
Then, Δ6, Δ12, and Δ15 desaturases were successfully ligated into two distinct pET-Duet vectors (vectors with two multicloning sites). One vector contained Δ12 and Δ15 desaturases, and the other was ligated only with Δ6 desaturase.
Protein expression was clear after transformation into cell strain BL21(DE3) and induction by IPTG. However, functionality could not be established. It was hypothesized that the naturally-occuring 18:1 fatty acid in E. coli is the wrong substrate for the desaturases. This fatty acid has its desaturation at the 11th carbon, not at the 9th position required for a substrate. Thus, the E. coli were fed 18:1 (Δ9).
Full characterisatin and quantification of the fatty acid composition in transformed E. coli was performed by fatty acid conversion to the corresponding fatty acid methyl esters (FAMEs) followed by GC-MS analysis. In this way, lipid profiles of membrane assays and lipid extracts from cells were obtained.
After characterization, we ligated each of our desaturases into the submission vector pSB1C3.
Fig. 5: "UV photograph of PCR results"
The figure shows the results of a PCR extraction of our genes of choice, done with GoTaq HotStart PCR kit at 2 different annealing temperatures: Δ12 (48°C) - Δ12 (56°C) - Δ15 (48°C) - Δ15 (56°C) - Δ6 (48°C) - Δ6 (56°C).
Primers
All primers are notated 5' to 3'. Initially, we worked with NdeI and XhoI as the restriction sites.
Δ12 T. Cruzi forward
ATATCATATGTGGCGGGTTGACAATTTA
Δ12 T. Cruzi reverse
ATATCTCGAGTTACCCATGAAAGACAC
Δ12 Synechocystis forward
ATATCATATGACTGCCACGATTCCC
Δ12 Synechocystis reverse
ATATCTCGAGTTAAACTTTTTTCAGGGAGC
Δ15 Synechocystis forward
ATATCATATCGCTCTAGAAATTTCATC
Δ15 Synechocystis reverse
ATATCACGACTTAAGGTTTCTTTTGATATC
Δ6 Synechocystis forward
ATATCATATGCAAACAGCGGAAAGAATT
Δ6 Synechocystis reverse
ATATCTCGAGTAACGATGCTTTGACCATGGCCTCTAG
ELO6 L. major forward
ATATCATATGTTCTCTCTGGAGCAGGCCGAA
ELO6 L. major reverse
ATATCTCGAGTCATCGACTGCCTGCCATGGCCTG
When using the pET-duet vector, we needed additional primers for the alternative restriction sites and His-tags. For Δ6 and Δ12, we cut with HindIII and EcoRI; NdeI and XhoI were used for Δ15 and ELO6.
ELO6 L. major forward
TATCATATGGGAAGCAGCCATCATCATCATCATCACAGCATGGTCTCTCTGGAGCAGGCC
Δ15 Synechocystis sp. forward
TATCATATGGGCAGCCATCATCATCATCATCACAGCATGCGTCTAGAAATTTCATCG
Δ6 Synechocystis sp. forward
GGATCCGAATTCATCTAACAGCGGAAAGAATT
Δ6 Synechocystis sp. reverse
GACAAGCTTTCACGATGCTTTGCCCATGGCCTCTAC
Δ12 Synechocystis sp. forward
GGATCCGAATTCATGACTGCCACGATTCCC
Δ12 Synechocystis sp. reverse
GACAAGCTTTTAAACTTTTTTCAGGGAGC
For the submission vector, we once again required new primers, using restriction sites EcoRI and PstI. For Δ6 and Δ12 forward, we used the primers from the duet vector (see above).
Δ6 Synechocystis reverse
GTTCCTGCAGTCACGATGCTTTGCCCATGG CTCTAG
Δ15 Synechocystis forward
GGATCCGAATTCATGCGTCTAGAAATTTCATCG
Δ15 and Δ12 Synechocystis reverse
GTTCCTGCAGTTAAACTTTTTTCAGGGAGC
Analysis
Lipid Analysis
Lipids were extracted from transformed E. coli using the Blight/Dyer method. (Please refer to the Lipid extraction lab book entry for further detail.)
Mass Spectrometry
Lipids were extracted from transformed E. coli using the Blight/Dyer method. (Please refer to the Lipid extraction lab book entry for further detail.)
The fatty acid content of the transformed cells was measured to determine whether it differed from the previously analyzed background lipid composition. This was done using mass spectrometry.
Bradford protein assay
A Bradford protein assay was carried out to assess protein concentration (Bradford, 1976).
Sequencing
We sent off samples to GATC for sequencing.
Results
The fatty acids were released by base hydrolysis followed by organic extraction, the resulting fatty acids were derivatised with diazomethane to the corresponding fatty acid methyl esters (FAMEs), together with fatty acid standards. The samples were analysed by GC-MS and the retention times and fragmentation patterns, compared with FAME standards. The results are shown in Charts 3,4 and 5.
By combining a number of genes, we were able to partially recreate the pathway described above (Fig. 4).
Due to time constraints, we were unable to submit a BioBrick containing the sequence of all of the necessary genes for the ω-3 biosynthetic pathway. However, we were able to express and characterise Δ12, Δ15 and Δ6 desaturases, the first three enzymes involved in ω-3 biosynthesis. As shown in charts 2, 3 and 4, the presence of alpha-linoleic acid (18:3Δ9,12,15), an omega-3 fatty acid, can be observed in E. coli. This does not occur naturally in their bacterial membrane.
Mass Spectrometry
Chart 2: "Fatty acid methyl ester analysis"
C18:3 refers to an 18 carbon backbone with 3 double bonds. C18:2 and C18:1 would have 2 double bonds and 1 respectively. The upper portion of this figure shows the peaks generated by the standard expected fatty acid methyl esters.The lower 4 panels show mass spec fingerprints that are indicative of our fatty acids of interest.
×
Chart zoom: Fatty acid methyl ester analysis
Chart 3: "Lipid extraction analysis"
All of the cellular lipids were extracted and their fatty acid methyl esters produced. These were analysed by mass spec. In the control cells no C18:2 is present. When Δ6 is expressed, a peak indicative of C18:2 is present. However, the primary fatty acid is still C18:1 - with the bouble bond present at position 9. With the expression of Δ12 and the addition of exogenous C18:1(Δ9) we get great abundance of C18:2 and negligible abundance of C18:1 indicating that the C18:1 is being processed successfully. The lowest graph shows that Δ12 expressed with Δ15 acts in a complementary fashion and the expected C18:3 - an Omega 3 - is produced.
×
Chart zoom: Fatty acid methyl ester analysis
Chart 4: "Membrane assay"
The cellular membranes were extracted and their lipids were analysed by mass spec. As in chart 2 it is confirmed that expression of Δ12 and Δ15 lead to the production of omega 3 fatty acids. This chart shows that these lipids locate to the plasma membrane of the bacteria
×
Chart zoom: Fatty acid methyl ester analysis
Bradford protein assay
Chart 5: "Bradford protein assay"
This chart shows the standard results of a Bradford protein assay.  Measuring the absorbance at 595nm for set samples, a standard curve was calculated with the equation of y = 0.0006x + 0.0426.
Table 1: "Bradford protein assay"
Using the standard curve shown in chart 5, the protein concentrations of a number of transformed plasmids was calculated.
Biobricks
We have submitted 3 BioBricks involved in the ω-3 biosynthetic pathway to the Registry of Standard Parts:
Biobrick
Short name
Description
Length
BBa_K925000
Delta-12 desaturase
Desaturase introducing a double bond at the Δ-12 site in the hydrocarbon chain of oleic acid (18:1, Δ9) to give linoleic acid (18:2 Δ9,12)
1056
BBa_K925001
Delta-15 desaturase
Desaturase introducing a double bond at the Δ-15 site in the hydrocarbon chain of linoleic acid (18:2 ; Δ9,12) for its convertion into alpha-linoleic acid (18:3 ; Δ9,12,15), an ω -3 PUFA
1080
BBa_K925003
Delta-6 desaturase
Desaturase introducing a double bond at the Δ-12 site in the hydrocarbon chain of oleic acid (18:1, Δ9) to give linoleic acid (18:2 Δ9,12)
1080
References
Arts, et al., 2009. Lipids in Aquatic Ecosystems. New York. Springer.
Bender D. A. and Bender, A. E, 1999. Benders' dictionary of nutrition and food technology. Cambridge: CRC Press.
Bradford, M., 1976. A rapid and sensitive method for the quantitation of microgram quantities of protein utilizing the principle of protein-dye binding. Anal Biochem, 7;72:248-54.
Cohen, J., 2003. Human Population: The Next Half Century. Science, New Series, Vol. 302, No. 5648. Pp. 1172-1175.
Garwin J. L. and Cronan J. E. Jr, 1980. Thermal modulation of fatty acid synthesis in Escherichia coli does not involve de novo enzyme synthesis. J Bacteriol, 141(3): 1457–1459.
Livore V., Tripodi K., Utarro A., 2007. Elongation of polyunsaturated fatty acids in trypanosomatids. FEBS Journal, 274: 264–274.
Marr A., Ingraham J., 1969. Effect of temperature on the composition of fatty acids in Escherichia coli. J. Bactiol. 84(6). Pp.  1260–1267.
Tonon T., et al., 2002. Long chain polyunsaturated fatty acid production and partitioning to triacylglycerols in four microalgae. Phytochemistry, Vol 61 Iss 1. Pgs 15-24.
Back to top
University of St Andrews, 2012.
Contact us: igem2012@st-andrews.ac.uk , Twitter , Facebook
This iGEM team has been funded by the MSD Scottish Life Sciences Fund. The opinions expressed by this iGEM team are those of the team members and do not necessarily represent those of Merck Sharp & Dohme Limited, nor its Affiliates.
Retrieved from " http://2012.igem.org/Team:St_Andrews/Omega-3-synthesis "
Recent changes
What links here
Related changes
Special pages
My preferences
Printable version
Permanent link
Privacy policy
Disclaimers
