Team:Calgary/Notebook
TOP
Project
Description CRISPR FLP-Beta CME Product Design Modelling Software Results
Notebook
Journals Protocols
Team
Team Members Attributions Sponsors Contact Us
Human Practices
Summary Silver Integrated Collaborations Public Engagement Safety
Judging
Medal Requirements Parts Improved Part Interlab
Team:Calgary/Notebook - 2018.igem.org
JOURNALS
Click a tab below to view the notes made by each subgroup.
Gene Integration
Gene Maintenance
Dry Lab
Below are notes regarding all Gene Integration experiments.
May
Week 1
We began our first week by performing a literature review on the key mechanisms involved in our section of the project, including the CRISPR-Cas9 wild type and Cas9 nickase mechanisms, as well as the FLP protein mechanism. Our literature search led us to design three plasmids to test the three main components of our subgroup project: ssDNA efficiency, FRT sequence insertion via Cas9n, and FLP recombinase efficiency. We decided that due to ease of access and high recombination frequency, wild-type CRISPR-Cas9 would be used in our initial ssDNA efficiency test. However, due to the fact that its increased specificity aligned with our project’s vision of safety, we decided to use Cas9n for the remainder of our project. As the week progressed, we continued to work out the foreseeable problems in our project and began to compile a list of faculty advisors whom we could contact for advice in shaping our project design. Throughout this planning process, we engaged in continuous discussion with the Gene Maintenance subgroup, to allow for our project design to be compatible and for our work to be done in parallel. Additionally, we attended the Laboratory Safety Training course held by the University of Calgary to prepare for our work in the laboratory.
Week 2
This week, we focused on meeting with faculty members and graduate students near our facility to discuss potential difficulties in our project. We met with graduate student, Taylor Goldsmith, to discuss our current project design and decided to make some changes, most notably by replacing our CRISPR-Cas9n plasmid with ribonucleoproteins, which is known to increase recombination efficiency. By running a cost analysis, we were able to determine that RNPs are more beneficial to our project. Additionally, we were able to lay out an updated timeline of our project, including our main focus of parts verification, and other side projects such as FLPo vs. FLPe characterization, circular vs. linear plasmid use for Cas9 insertion efficiency, and the use of a RNP-aptamer-ssDNA complex in facilitating Cas9 activity.  In our spare time, our team was able to begin our Human Practices work by beginning our preliminary research in building a curriculum change proposal to the International Baccalaureate Organization.
Week 3
This week, we focused on meeting with more faculty members to consult them and finalize our project design. We spoke to professors Wendy Anne Hutchins and Jonathon Lytton, who helped us troubleshoot our project. We also attended a "Methods Day" workshop hosted by the Department of Biochemistry and Molecular Biology, through which we were connected to some contacts who work with CRISPR as well as HEK cells. Meetings were set up with Iman Al Khatib, Justin Simms, and Tian Zhao, all graduate students at the University of Calgary working on similar projects. As the week progressed, we began putting together our plasmids on Benchling and communicating with the CME subgroup to ensure that our parts are compatible.
Week 4
The focus for this week was to deliver a project presentation held for our faculty mentors and graduate students who we would like to engage with for the duration of the project. We were also able to have a meeting with the IB coordinator and IB Biology teacher from Father Lacombe High School, to further our process of building an option module focusing on synthetic biology, bioethics and its applications for the upcoming 2022 IB Biology curriculum change.
Week 5
This week, we managed our part orders, including the finalization of part designs as well as ordering the parts through IDT, Genscript and Addgene for various parts. In addition, laboratory work started with wet lab members practicing transformations, overnights, minipreps and digest confirmations using registry parts that had potential for usefulness. PAH has importance in its mutation causing PKU, the AID gene has an enhanced GFP tag that could be used for reporting with Cas9, and we will be using a CMV promoter for transient expression of our parts. The Genscript and Promega funding applications were written and submitted.
June
Week 6
In waiting for parts to come in, this week was spent continuing laboratory practice using registry parts. We transformed superfolding-GFP for reporting, dCas-omega as a potential way to verify insertion of FRT sites by CRISPR once we have transitioned to HEK cell work, and the ECH and alpha-galactosidase genes for their implications in genetic disorders that may foreseeably be targets for our project. We considered using cDNA for the implementation of large genes to create functional proteins without splicing. In the lab, we started to experience low yields for minipreps. Through troubleshooting our process, we determined that the cause of this issue was likely the concentration of isopropanol that was used. Therefore, three parallel minipreps were done using 70%, 85%, and 100% isopropanol and it was found that there was a sharp increase in the concentration of isolated DNA when using increasingly higher concentrations of isopropanol, as a final concentration of 35% isopropanol is required in the solution.
Week 7
This week saw the beginning of our HEK cell training and further optimization of experimental practices. The team learned to check the confluency, passage, set cells with a hemocytometer and to transfect HEK cells using a calcium/phosphate approach. After securing access to the Core facility (Snyder Bioinstrumentations Labs) at Foothills Campus, we were given an orientation of the area. Work on bacteria was furthered by practicing the use of electroporation and comparing relative efficiencies with chemical transformations. Various methods of gel extraction were performed, the most successful was catching DNA run on a gel on filter paper after its identity was confirmed. However, it was determined that it would be most efficient to buy a gel extraction kit that has a relatively low cost. Apart from lab activities, the Geekstarter application for funding was worked on exhaustively and submitted.
Week 8
Parts started arriving this week, allowing for set-up for the larger project to begin. pCAG-FLPo (came in Stbl3) and pCAG-FLPe (came in DH5-α) were digest confirmed. Our FLPo part (for the registry) ordered from Genscript that will be used for submitting to the registry after ligation into a pSB1C3 backbone was digested and ligated for the first time. Electroporator testing was done by using varying concentrations of DNA. FRT-pUC57 sequence that is unfortunately incompatible with the other subgroup was transformed into DH5-α. Ligation transformations took two days to grow. Cas9 (pU6-(BbsI)_CBh-Cas9-T2A-mCherry) was linearized and Antarctic phosphatase treated to prepare for ligating sgRNAs into the plasmid once they are received.
Week 9
Work in the lab consisted of troubleshooting digestion and ligations for FLPo (for the registry). We received our FRT halves from IDT as they were not able to synthesize our sequence of both FRT sites together due to hairpinning.. We successfully inserted the FLPo sequence into pSB1C3, however after testing directionality, it was in backwards.
July
Week 10
This week, the sgRNA single-stranded oligos came in allowing for them to be annealed, ligated together and PNK treated. Our FRT 2nd half was not as successful as the first half in ligating into a pSB1C3 backbone, thus it was redone. Chemically competent DH5-α cells were made.
Week 11
This week was spent repeating and troubleshooting the digestion and ligations of sgRNAs and Cas9 (pU6-(BbsI)_CBh-Cas9-T2A-mCherry), as well as ligating mCherry into pCAG-FLPo to add the CMV promoter to the front of mCherry. Sequencing results confirmed that the sgRNA ligations into the backbone were not successful.
Week 12
Similar to last week, the focus this week was troubleshooting the sgRNA digestions as well as the mCherry digestion. Minor tweaks to concentrations were made and assays were re-attempted with different methods of purification, however, we had no success. sgRNA insertion is still not working.
Week 13
The same issues remain as in the last two weeks. FlpO-mCherry ligation was attempted and failed. The Cas9-sgRNA (pU6-(BbsI)_CBh-Cas9-T2A-mCherry) ligation was also re-attempted, and failed again. We determined that the issue was not BBsI, as this was shown to work, however, there may be another material problem resulting in our failed ligations.
August
Week 14
The main objectives for this week were to ligate mCherry into the pCAG plasmid, ligate the sgRNAs into the Cas9 plasmid (pU6-(BbsI)_CBh-Cas9-T2A-mCherry), and to ligate and transform the FRT 2nd half fragment ordered through IDT into pSB1C3. BBsI (the enzyme used for digestion of the Cas9 backbone for sgRNA insertion) was tested and shown to work. As such, we determined that this has not been the issue resulting in failed sgRNA transformation. No results were obtained with respect to FlpO and mCherry troubleshooting.
Week 15
The purpose of this week was to begin planning our transfection system and to continue attempting to ligate our sgRNA candidates into the pU6-Cas9 plasmid. Additionally, we attempted to clone mCherry into the pCAG-FlpO vector so that it would be attached to a promoter. No success was found with respect to sgRNAs or FlpO-mCherry. A homemade nucleofection buffer was made using sodium succinate based on literature review, which will be tested next week for effective nucleofection.
Week 16
One main success of this week was solving the problem of unligated sgRNAs into the Cas9 backbone (pU6-(BbsI)_CBh-Cas9-T2A-mCherry). This revelation originated from the particular focus on why sequencing results always showed undigested plasmid, despite the use of new restriction enzymes. Thus antarctic phosphatase that was used on the Cas9 plasmid was investigated as the sgRNAs were PNK treated and the backbone if not AP treated will not ligate together and will only allow for the cut-out section to re-ligate making it look like there is only undigested plasmid. Therefore, new AP was ordered and the old AP was tested. The testing of old AP was a comparison between a digestion and ligation of our FLPo (for the registry) with itself and a digestion, Antarctic phosphatase treatment and ligation of FLPo (for the registry) with itself. It was shown that the growth of both after transformation was the same verifying that the old AP was non-functional, as if the AP worked the AP treated trial would have had ~5% of the non-treated samples growth. After this, the sgRNAs were ligated into the Cas9 backbone with new AP and cPCR was used to verify insertion. Colony PCR was done using one of the complementary single-stranded oligos and the single primer sgRNA IV together showing that single-stranded oligos can be used as primers. From this, master plates were made and from cPCR results, the seemingly correct ones were overnighted and sent for sequencing. HEK cells were nucleofected with GFP using a home-made buffer and compared the usefulness of two types of cuvettes, finding which type of cuvettes to use in the future. FLPo (for the registry) was attempted to be reversed, but the directionality was not fixed. pCAG-mCherry work has continued to be troublesome due to pCAG-FLPo not running properly, but pCAG-FLPe was attempted for ligation as an alternative. It is also apparent that gel purification can still be done despite the nanodrop reading that there is no DNA due to running it on a gel and having a very clear and correct band.
Week 17
Further tests on the relative compatibility of the two brands of nucleofection cuvettes were assessed due to potential human error in the first run. The results of these tests confirmed that one type of cuvette was about 40% more efficiency for nucleofection. sgRNA sequencing results came in and showed successful sgRNA ligation in five of eight samples, thus those that did not work initially were re-ligated, transformed, cPCRed, overnighted, miniprepped and sent for sequencing. This second round of sgRNA sequencing revealed that we have seven out of eight sgRNAs ligated in. The missing one is CCR5 1-7R. The seven successful sgRNAs will be tested next week, while we attempt to clone CCR5 1-7R.
Week 18
This week, targeting efficiency of our various sgRNAs was tested and the relative effectiveness of Cas9-WT (wild-type) and Cas9n (nickase) were compared. This was accomplished by transfecting our sgRNA-Cas9 plasmids into HEK293 cells using Lonza nucleofection with the homemade buffer, which was tested in previous weeks. For the wild-type vs. nickase trial, HEK293 was transfected with a combination of Cas9n or Cas9-WT, as well as the corresponding sgRNA and ssDNA template for insertion. In both trials, a no transfection control and GFP-transfection control were used. In total, 7 sgRNA candidates were transfected, as well as Cas9n and Cas9-WT. However, after purifying gDNA (genomic DNA) from the transfected cells two days later and running them in a PCR reaction, no amplified fragments appeared.
September
Week 19
This week was used to troubleshoot the lack of gDNA PCR bands from last week. We predicted that the fault lay with our gDNA extraction technique, so a variety of gDNA extraction techniques were performed to test for the most effective. This included protocols from classes, workshops, proteinase K treatment, and a cell lysis buffer. However, after using the purified DNA for PCR, none of the extraction methods showed promise.
Week 20
A variety of methods of DNA isolation were attempted on the HEK 293T cells, in which it was found that the basic "strawberry extraction" technique worked to extract the DNA. This extracted DNA was ethanol precipitated and prepared to run a genomic PCR (gPCR) reaction. Outside of the lab, work on the aGEM project outline and video pitch was started and completed for the deadline on Friday. Past this, work on the formal presentation for aGEM began. Final designs for our hybrid FlpO-beta resolvase plasmid and the corresponding ssDNAs were made and the parts ordered.
Week 21
This week genomic PCR of the HEK293T cell genome was conducted, however, when run on a gel, bands were not seen. This leads us to troubleshoot our methods, in particular, optimizing the melting temperature of our primers, based on discrepancies between sources. HEK cells were thawed to prepare for future experiments to be done with the Flp/beta-resolvase hybrid part that was expected to come in soon. Preparation for the aGEM conference was a focus of the week on building and solidifying the presentation. Two mentors/advisors, Dr. Dalton and Rachelle Varga provided feedback before aGEM, helping focus the presentation on aspects that were most important.
Week 22
Using the control transfected HEK293T cells a genomic DNA extraction "miniprep" kit was tested, as a potential source of error in PCR (not having genomic DNA enough or sheared too heavily for primers to work effectively). The initial gPCR reactions run with these minipreps were unsuccessful much like the strawberry extraction method that was used, revealing only smears on the gel. Along with these, tests on various melting temperatures (53°C, 57°C) were done to attempt finding an optimal melting temperature.
October
Week 23
Troubleshooting tests continued on variables including DNA isolation from HEK293T cells, purification and melting temperatures. The genomic DNA extraction kit and strawberry extraction methods were tested again and nano-dropped. After ethanol precipitation of the kit miniprep, it was shown this sample was both more concentrated and pure than the strawberry extraction method. When these samples were gPCRed and run on a gel, the kit minipreps had bands that matched our expected lengths. The transfected and frozen HEK293T cells (for our sgRNA comparison and wild-type vs. nickase testing) were genomic DNA extracted for use once the optimal melting temperatures for each of the primers has been determined via control tests. Our FlpO-Beta resolvase site was transformed into psb1c3 for submission to the iGEM registry. Furthermore, the FlpO-beta resolvase plasmid was transfected into HEK293T in order to test the hybrid recombination system. The transfection involved four components: Cas9 nuclease RNP with the associated sgRNA, ssDNA template for the CRISPR knock-in, pCAG-FlpO for the expression of FlpO to mediate the initial recombination, and finally the plasmid containing beta resolvase which will be inserted into the genome using FlpO. Controls were also transfected, containing all of the above reagents minus Cas9, and minus FlpO. These cells were then maintained as they grew, and transferred to antibiotic media to screen transfected cells.
Week 24
This week saw the optimization of melting temperatures for our primers and successful gPCR for all of our experiments, allowing for transfection experiments related to the sgRNA comparisons and the Cas9 variant comparison to be sent for sequencing. In addition to these samples, the hybrid FLPo+β-resolvase and no-FLPo treatments of HEK293T transfections were sent for sequencing to analyze whether the CRISPR insertion worked and/or if the hybrid FLPo+β-resolvase worked. In preparation for sequencing, bands were excised from the gel and purified, then 3ul and 6ul of DNA was used in our sequencing mixture. The transfected cells with hybrid FLPo+β-resolvase were split into a pen-strep treated media and a puromycin treated media, in which pen-strep kills bacterial contamination and puromycin will kill any HEK293T cells that have not had integration into the genome. Along with this controls (no-FLPo, no-transfection, no-Cas9, GFP) were given puromycin treatment. Results later in the week revealed that the controls all died from puromycin treatment as expected and at the end of the week, two HEK293T cells were identified to be alive and adhered in the hybrid FLPo+β-resolvase (with Cas9 RNP, FLPo, ssDNA insert containing single FRT/beta site and the hybrid plasmid) treatment. These two cells act as a proof of concept for our core project. Beyond these results part of our sequencing results returned, allowing for chromatogram analysis of our sgRNA comparisons. DNA submission of hybrid FLPo+β-resolvase was also done. Work on writing up wiki content began.
Week 25
This final week before the dreaded wiki-freeze involved the analysis of sequencing results, finalizing content on the wiki and polishing our project as a whole for display on the iGEM website.
Below are notes regarding all Gene Maintenance experiments.
May
Week 1
We began an extensive literature search on the topic of insulators and other epigenetic regulatory elements. We determined that ubiquitous chromatin remodelling elements (UCOEs) would be effective at opening chromatin and will be very useful in producing a continuous high-level transcription of any transgenes that are inserted into the human cell genome via our CRISPR/FLP system.  While UCOEs will allow for our gene to enter an active transcriptional state, insulators will function to maintain this state. If flanking the transgene, insulators protect the gene from the spread of neighboring heterochromatic regions and from crosstalk of enhancers. In addition, we began a discussion about the testing platform that we will utilize. Working in parallel with the Gene Integration subgroup prevents us from using their integration scheme, so we are investigating other options for preliminary experiments. We also completed all necessary Lab Safety training which will allow us to work in the lab this summer.
Week 2
We continued with literature research specifically focused on ubiquitous chromatin remodelling elements (UCOEs) as well as insulators. The A2UCOE, CBX3-UCOE, and cSH4 insulator were selected for experimentation and their sequences were obtained. We began planning constructs which will eventually constitute the various cassettes during recombinase-mediated cassette exchange (RMCE) procedures. We continued the investigation of other options for preliminary experiments that will free us from dependence on the Gene Integration subgroup's FRT integration scheme. One promising approach would be to generate minicircles from triple-FRT containing FLP plasmids within prokaryotic cells, isolate and purify the minicircles, and co-transfect them with plasmids expressing FlpO (pCAG-FLPo) into a HEK cell line that already contains one FRT site, such as the Flp-In T-REx HEK293 cells available commercially. This would allow for the insertion of a second heterospecific FRT site, which makes RMCE possible. Therefore, we could conduct our experiments on UCOEs and insulators while the Gene Integration subgroup fine-tunes the CRISPR-mediated FRT integration scheme. We began a discussion about the process of obtaining the Flp-In T-REx HEK293 cells from internal contacts within the University of Calgary. Another promising possibility explored is to obtain a cell line that already contains heterospecific FRT sites. In this case, we could begin immediately with RMCE in order to investigate UCOE and insulator function. We contacted the principal investigator from that research group in regards to obtaining cells from the generated cell line and are awaiting a response.
Week 3
This week, we finalized our BioBrick plans including the restriction sites to be used. Further research on the characteristics of these restriction sites was also conducted. After meeting with Dr. Ray Wang, we secured access to the Flp-In T-REx HEK293 cell line as well as the commercial vectors pcDNA5 and pOG44 designed for use in the system. Dr. Ray Wang also offered to provide troubleshooting assistance to the team when using this system. We practiced laboratory techniques by transforming chemically competent DH5-α cells with a CMV promoter ( BBa_I712004 ) and a BGH terminator ( BBa_K1150012 ), and chloramphenicol resistance ( BBa_K143064 ). The University of Calgary also hosted a methods day, where we were able to attend presentations and speak to other researchers regarding certain laboratory techniques and protocols. In particular, we learned more about bisulphite sequencing, transfection techniques, and qPCR primer design. Our team also attended a welcome event for summer students, where we went bowling as a team-building activity. Finally, we helped to prepare slides and script for a presentation that will be given to mentors, industry professionals, and other interested general public attendees on May 23rd.
Week 4
We continued working on our transformed DH5-alpha cells for practice, including creating overnights, mini-prepping, running a digest, and a digest confirmation. In addition, we created approximately 70 aliquots of chemically competent DH5-alpha cells for later use. We transformed the parts required for Interlab into our chemically competent DH5-α cells and left them to grow over the weekend. Growth was slow, therefore we will re-transform the Interlab parts next week with an improved transformation protocol that will hopefully result in a normal growth rate.
We finalized the restriction sites that will be used in collaboration with the Gene Integration subgroup to ensure that all parts are compatible and that we are ordering sequences in the most efficient way possible. In addition to participating in the Innovate Calgary workshop, we also contributed to a team presentation in which we revealed our project idea and experimental plan to faculty experts, mentors, and members of the public. This event was very useful as we were able to identify some strengths and weaknesses of the project and plan as well as other general feedback from the audience.
Week 5
We received very generous samples of T-Rex Flp-In HEK293 cells and pcDNA5 and pOG44 vectors from Dr. Ray Wang, a researcher in Dr. Wayne Chen's lab here at the University of Calgary. The cells were stored at -80˚C while the pcDNA5 and pOG44 vectors were transformed into chemically competent DH5-α. We miniprepped several samples of pcDNA5 and pOG44 from the cells and have stored them in the freezer for use later in the summer. We also made glycerol stocks of both vectors for long-term storage.
We re-transformed all of the Interlab test parts in chemically competent DH5-α. During the transformation, we used a lower volume of recovery media and lengthened the recovery incubation. All of the transformations grew much quicker than last week, therefore we will use the new, faster-growing parts for the remainder of the Interlab study.
Addgene was offering a free synthesis of 2kb gene fragments. We took full advantage of this offer and ordered a few of our constructs: mCherry-BGH, which will be ligated to CMV ( BBa_K747096 ) from the distribution kit, and both CHS4 insulator parts. Before CMV is ligated to mCherry-BGH, we will use overhang PCR to add a SalI restriction site upstream of the promoter. This will improve the part and allow it to function with our construct system.
June
Week 6
We began the week by testing electroporation protocols using our electrocompetent cells and the competent cell test kit. The first protocol that we tried was very successful, and we saw very high transformation efficiencies. This allowed us to conclude that electroporation will be the method of choice for transforming more important parts into DH5-α cells. We also created a plasmid stock of the vectors pcDNA5 and pOG44 for later use in HEK293 Flp-in T-REx cells. We tried to miniprep and digest confirm the remaining Interlab test parts, but continued to get unexpected results. Therefore, we re-transformed test parts 1, 5, and 6 ( BBa_J364000 , BBa_J364008 , BBa_J364009 ). We designed PCR primers with overhangs to add restriction sites to CMV from the registry that will later be cloned with a mCherry-BGH construct to form our insert. We conducted more in-depth research into methods and protocols for qPCR and bisulphite sequencing. Since IDT was unable to synthesize our UCOEs, we contacted Genscript and they provided us with a quote. Finally, we investigated using Gibson assembly as an alternative method for assembling our constructs.
Week 7
We determined that the cloning method we will proceed with will be the traditional restriction enzyme cloning, while other members of the team may later pursue Gibson assembly if our method is not working. Having decided that we would PCR amplify the subpart of A2UCOE that constitutes CBX3-UCOE as a method for staying on budget, we ordered only A2UCOE from Genscript. We finished the CMV overhang-primer design on Monday and ordered them from the University of Calgary DNA services. We miniprepped pSB1C3-CMV that the Gene Integration subgroup had previously transformed and digest confirmed, and then ran PCR to generate CMV with restriction sites that we will need. The product was run on a gel and appears to be the correct size, so we proceeded with a digest. Our mCherry-BGH and one of the insulators that had been ordered as free samples from Genscript arrived on Friday and will be used early next week. For the Interlab, we miniprepped and digest confirmed the parts once more and decided to proceed. We attended an orientation at the Snyder Core Facility, which is where the fluorescent plate reader that we will need to use for the Interlab study (as well as the qPCR machine) is located. Following more in-depth training in the Snyder Core Facility, we will be able to complete the Interlab study. We helped train the Gene Integration subgroup in using electroporation. Finally, we participated in HEK cell training, including passaging and transfecting cell cultures.
Week 8
This week, our ordered parts arrived and we began to clone them into pSB1C3. We digested our mCherry DNA block, the CMV PCR products with XbaI and PstI, and a previously miniprepped backbone sample which we then ligated. Simultaneously, we digested both the cHS4 insulator DNA blocks, as well as more backbone with XbaI and SpeI then ligated. After some ligation troubleshooting, we proceeded to transform chemically competent DH5-α cells with the ligation products as well as electroporate some samples. However, as of Friday, these transformations were unsuccessful. Aside from the wet lab work, we collaborated with the dry lab team to discuss wiki content for our subgroup. In addition, we researched some protocols that will be used later this summer. Finally, we read the Thermofisher manuals for the T-REx HEK 293 cells that we will begin to use in the future.
Week 9
We began the week by screening colonies from the ligations of our parts into pSB1C3 using digest confirmations. The first colonies screened did not contain the correct inserts, but on the second attempt, we found some colonies that appeared to possibly contain our improved pSB1C3-CMV (figure 1) and pSB1C3-mCherry-BGH (figure 2). These samples were sent for sequencing and we are awaiting the results. As we had issues with trying to ligate the cHS4 insulators into pSB1C3 using XbaI and SpeI sticky ends, we repeated the ligation using a 7:1 insert to vector ratio. The ligation products were transformed into chemically competent DH5-α, and we screened some of the new colonies. Unfortunately, all the results were negative. Outside of the lab, we completed the iGEM Safety Form 1 and began writing content and creating figures for our wiki pages.
Figure 1. Digest confirmations of [A] pSB1C3-BBa_K2605001 (new CMV) and [B] pSB1C3-BBa_K747096 (old CMV) . The plasmid was double digested with SalII and SpeI or single digested with NotI  then run on a 1.5% agarose gel at 100 V for 35 minutes. The molecular ladder (L) is visible on the far left and the expected band sizes, obtained from Benchling Virtual Digest, are visible in the diagram on the right. Undigested plasmid (U) was used as a control.
Figure 2. Digest confirmations of pSB1C3-mCherry+BGH. The plasmid was single digested with NotI or double digested with NcoI and EcoRI then run on a 1.2% agarose gel at 100 V for 35 minutes. The molecular ladder (L) is visible on the far left and the expected band sizes, obtained from Benchling Virtual Digest, are visible in the diagram below. Undigested plasmid (U) was used as a control.
July
Week 10
Early in the week, we completed a large portion of the Interlab study by generating the required standard curves. We received the sequencing results for the suspected improved pSB1C3-CMV and pSB1C3-mCherry-BGH, but the sequences were not as expected. It appeared that what we suspected to be the backbone with the correct insert was actually residual undigested backbone, even though we had gel-extracted it before ligation. These parts were digested and ligated once more.
After using digest confirmations to screen approximately 60 colonies in the first half of the week, we discovered a colony PCR (cPCR) protocol that would optimize and streamline our colony screening process. We tested this protocol using the positive control from the Interlab study ( BBa_I20270 ) in order to determine which buffer and Taq polymerase combination works best. Having isolated the best combinations, we are confident in implementing this solution moving forward.
Week 11
We began the week by repeating some transformations as well as digests and ligations of the cHS4 insulators and the improved CMV product into pSB1C3. In addition to trying to ligate the insulators into pSB1C3 using XbaI and SpeI sites (as the Genscript part is missing the NotI site from the prefix and suffix), we also attempted to use the EcoRI and PstI sites. Using colony PCR (cPCR), we were able to screen approximately 130 colonies with ease. We found one colony with pSB1C3-mCherry-BGH, and ran an addition digest confirmation as well as sent a sample for sequencing. We were also able to find four colonies for the E/P-digested pSB1C3-cHS4(1) and one for E/P-digested pSB1C3-cHS4(2). The ligation of improved CMV into pSB1C3 produced no colonies, but we moved ahead with ligating the improved CMV into pSB1C3-mCherry-BGH and transforming the product. These colonies will be screened next week. In order to perform experiments to test the function of our CMV as an improved part, we also began steps to clone the original CMV promoter from the registry into pSB1C3-mCherry-BGH. Finally, as the E/P digest-ligations worked so much more efficiently than those with X/S, we began investigating alternative methods for correcting the NotI sites on the cHS4 insulators within the backbone.
Week 12
As the Gene Integration subgroup was mostly absent this week, we assumed their task of cloning together the two halves of the multiple cloning site (MCS). IDT was unable to synthesize the MCS as one part due to hairpins. Early in the week, we worked on digesting and recovering each half for ligation. The MCS first half was successfully digest confirmed, sequenced, and set aside for ligation. However, after several failed digests and sequencing, we concluded that the Gene Integration subgroup's 'MCS second half' glycerol stocks actually contained empty backbone. Before we can move forward with joining the two halves, we will need to ligate the IDT Biobrick MCS second half into pSB1C3.
Our pSB1C3-mCherry-BGH sequencing results came back early in the week, and the part is exactly as expected. We are confident in moving forward with cloning. We continued to screen pSB1C3-mCherry-BGH + improved CMV ligation colonies using cPCR, but so far we have not found a colony with the correct insert. As all of our improved CMV ligations have been unsuccessful, we began research into possible causes. We found that impurity of PCR products may be a factor, and therefore tried to gel purify our product. However, our gel purification kit failed once again. Next week, we plan to make new improved CMV and excise the correct band from LMP gel before digesting and ligation.
Sequencing results for both cHS4 insulators ligated using E/P showed that the correct part was inserted, but as suspected, the NotI site was mutated. This was further verified by a digest confirmation. In order to submit these parts, we will have to follow a two-step cloning process beginning with XbaI and PstI digestions. Colonies were screened using cPCR. Over the course of the week, we found only three colonies for the pSB1C3-cHS4(2) X/P ligation that appeared to have the correct insert. These will be sequenced next week.
As we intend to submit our new CMV as an improvement on CMV ( BBa_K747096 ) from the iGEM registry, we also worked on cloning this part into pSB1C3-mCherry-BGH. After experiments meant to quell some doubt regarding the identity of glycerol stocks made by the Gene Integration subgroup, we were able to digest pSB1C3-CMV with XbaI and SpeI and recover the insert via LMP gel extraction. Ligation into pSB1C3-mCherry-BGH will occur next week.
Week 13
We began the week with an attempt to ligate the second half of the MCS into the pSB1C3 backbone with two different methods. One of the methods produced no growth after ligation, while the other did produce colonies. These colonies were screened using cPCR, and the initial results were promising. Some of the colonies containing inserts of the correct size were miniprepped and sent for sequencing. However, a failed digest confirmation has caused us to doubt the part's identity. We also stockpiled many more samples of parts and backbones that we may need in the future
Overnights were made and miniprepped for insulator samples. These were digest-confirmed, and then we moved onto the next round of ligations with them in order to incorporate the NotI sites. We also digested each of the insulators with their corresponding MCS restriction enzymes and stored them in the freezer in preparation for future cloning.
We also continued trying to digest and ligate our CMV PCR product into the pSB1C3 backbone but had no definite success this week. Efforts will continue next week.
August
Week 14
As we suspected last week, the supposed pSB1C3-MCS was determined to be an incorrect part via sequencing and further digest confirmations. As it may not be entirely necessary to clone the second half into pSB1C3 individually, we moved on to cloning the IDT Biobrick into pSB1C3-MCS first half directly using NsiI and SpeI sites. However, we were not yet successful as of Friday.
We also continued to clone the CMV promoters into pSB1C3-mCherry-BGH but did not advance to the stage of screening colonies. Midway through the week, we discovered that the glycerol stocks of pSB1C3-CMV that the Gene Integration subgroup had created (the ones we have recently been using) may have been from a different registry part than the one we intended to use ( BBa_I712004 vs. BBa_K747096 ). Therefore we retransformed the pSB1C3-CMV that we had originally planned to use ( BBa_K747096 ), made new glycerol stocks, and miniprepped some samples. We then performed extension PCR on the new samples, the product of which was subsequently digested and prepared for ligation.
Finally, we continued with our attempts at cloning the cHS4 insulators into pSB1C3 so that they have both NotI sites. We believe that we were able to clone pSB1C3-cHS4(2), but we will digest confirm and sequence this sample next week.
Week 15
We began the week by repeating the ligation of MCS second half into pSB1C3-MCS first half. After cPCR screening and preliminary digest confirmations, we believe that we have successfully cloned the whole MCS into pSB1C3. More digest confirmations will be performed next week.
In addition, we believe that we have successfully ligated the MCS second half fragment into pSB1C3. We will confirm this next week.
We were also able to confirm via digests and sequencing that we have successfully inserted cHS4(2) (which has the same sequence as cHS4(1) aside from two flanking restriction sites) into pSB1C3 for submission to the registry.
Finally, we continued attempts to clone the improved CMV into pSB1C3 and pSB1C3-mCherry-BGH, as well as the original CMV ( BBa_K747096 ) into pSB1C3-mCherry-BGH. Though we have not yet had success with the improved CMV, cPCR results lead us to believe that we have successfully ligated the old CMV into pSB1C3-mCherry-BGH. This is essential for allowing us to test our improved part's function. The colonies suspected to contain this part will be characterized further next week.
Week 16
We were able to confirm that the suspected pSB1C3-MCS was indeed the correct part containing all of the necessary cloning sites using digest confirmations (figure 3). This part can now be used in future cloning and will also be submitted to the registry. Before any of the constructs can begin to be made, we first needed to clone our improved CMV into pSB1C3-mCherry-BGH. Therefore, the majority of our efforts this week focused on cloning these parts together, as well as cloning our improved CMV directly into pSB1C3. Though the former was not yet successful, we believe that we have successfully cloned improved CMV into pSB1C3. Samples were sent for sequencing and we are awaiting results. In parallel, we also set up digest confirmations of the miniprepped the improved pSB1C3-CMV samples using pSB1C3-CMV ( BBa_K747096 ) as a control. The results were exactly as expected.
We sent the supposed pSB1C3-MCS second half for sequencing early in the week. However, the results we received indicate another unsuccessful ligation.
In preparation for characterization of our improved part, we also digested pSB1C3-CMV-mCherry-BGH and pcDNA5 (our Flp-In vector that will be used in HEK 293 T-REx cells) so that we may ligate them together next week.
Figure 3. Digest confirmations of pSB1C3-MCS. The plasmid was single digested with SapI or AflII and  double digested with NcoI and FseI/MfeI/BmtI/BamHI/NsiI/SalI then run on a 1.2% agarose gel at 100 V for 35 minutes. The molecular ladder (L) is visible on the far left and the expected band sizes, obtained from Benchling Virtual Digest, are visible in the diagram below. Undigested plasmid (U) was used as a control.
Week 17
Early in the week, we received sequencing results for the improved pSB1C3-CMV and were able to determine that we had successfully ligated in the correct insert. We proceeded with ligating this promoter to our reporter, pSB1C3-mCherry-BGH. cPCR screening indicated that this ligation was also successful, so we sent samples for sequencing and began preparing to create our constructs by digesting this part for ligation into pSB1C3-MCS. However, the initial digestion of  pSB1C3-improved CMV-mCherry-BGH failed.
In order to characterize our improved parts, we also began working on cloning mCherry-BGH and CMV-mCherry-BGH into our eukaryotic T-REx HEK293 vector pcDNA5, though initial attempts were unsuccessful.
We finally received our pUC57-A2UCOE part from Genscript after many months, as the GC-rich sequence gave them lots of difficulty during synthesis. This part was transformed and digest confirmed, and we set up a digestion over the weekend in order to ligate it into pSB1C3.
Week 18
We continued the attempts at cloning the mCherry-BGH, CMV-mCherry-BGH, and improved CMV-mCherry-BGH parts into pcDNA5. However, after repeated attempts, we have still not yet been successful.
Early in the week, we received sequencing results that confirmed the identity of the pSB1C3-improved CMV-mCherry-BGH part (figure 4). We had to do some troubleshooting with the restriction enzymes that we were using to digest it, but even after a successful digestion, we had a failed ligation. It is beginning to look impractical to make all of our constructs in the remaining time before the competition.
We also ligated A2UCOE into pSB1C3 and after an initial cPCR screen, discovered one colony that contained the correct insert. This will be further verified next week.
Figure 4. Digest confirmations of [A] pSB1C3-BBa_K2605001-mCherry+BGH (new CMV) and [B] pSB1C3-BBa_K747096-mCherry+BGH (old CMV) . The plasmid was double digested with SalII and SpeI or single digested with NotI  then run on a 1.5% agarose gel at 100 V for 35 minutes. The molecular ladder (L) is visible on the far left and the expected band sizes, obtained from Benchling Virtual Digest, are visible in the diagram on the right. Undigested plasmid (U) was used as a control.
September
Week 19
As our fall semester of university began again this week, much of our time was spent cleaning up the lab. Aside from this, we continued with attempts at cloning the mCherry-BGH, CMV-mCherry-BGH, and improved CMV-mCherry-BGH parts into pcDNA5. We also prepared pSB1C3-A2UCOE for a digest confirmation.
Week 20
We continued to work on cloning the mCherry-BGH, CMV-mCherry-BGH, and improved CMV-mCherry-BGH parts into pcDNA5 using various samples and vector:insert ratios, but still did not have any success.
The pSB1C3-A2UCOE colony identified via cPCR two weeks past was digest confirmed, but the results were unclear. Therefore we repeated this ligation and transformed it.
Week 21
We began the week by once again attempting the ligation of mCherry-BGH, CMV-mCherry-BGH, and improved CMV-mCherry-BGH into pcDNA5. After once again failing, we decided to change our approach and use a new vector. Due to accessibility, we chose to work with pEGFP-N1, but had issues early on with this. We then investigated the possibility of characterizing our improved part in a prokaryotic chassis rather than eukaryotic, which proved possible. In order to do so, we transformed DH5-alpha with pSB1C3-mCherry ( BBa_J06602 ) from the distribution kit which contains an RBS for prokaryotic transcription.
We continued efforts to clone our new A2UCOE part into pSB1C3, and near the end of the week received positive results in a cPCR screen. These results will be confirmed further next week.
Over the weekend, we hosted aGEM, where we presented our project and participated in workshops along with 5 other iGEM teams from Alberta.
Week 22
Early in the week, we digest-confirmed the newly transformed pSB1C3-mCherry ( BBa_J06602 ) then moved forward with ligating both the old CMV ( BBa_K747096 ) and our improved CMV to this part. Initial cPCR screens indicated that these ligations were successful and many of the colonies on these plates turned red. We will further confirm these preliminary results next week.
A successful digest confirmation was run for the colony of pSB1C3-A2UCOE (figure 5) that was identified as having the correct insert via cPCR screening last week. We also sent this part for sequencing and were able to determine that the sequence is exactly as expected.
Finally, we began concrete planning for a collaboration with the Notre Dame iGEM team which was initially discussed last week at the aGEM meetup.
Figure 5. Digest confirmations of pSB1C3-A2UCOE. The plasmid was single digested with XhoI or double digested with SpeI and EcoRI then run on a 1.2% agarose gel at 100 V for 35 minutes. The molecular ladder (L) is visible on the far left and the expected band sizes, obtained from Benchling Virtual Digest, are visible in the diagram below. Undigested plasmid (U) was used as a control.
October
Week 23
Throughout the week, we cPCR-screened multiple red colonies from each of the  pSB1C3-CMV-mCherry ligations. We ran confirmation digests on colonies that appeared to have the correct insert and were able to determine that our ligation and part improvement was successful (figure 6).
We also began efforts to clone our team's final part, 'Flp/beta resolvase hybrid' into pSB1C3 in order to assist the Gene Integration subgroup.
Figure 6. Digest confirmations of [A] pSB1C3-BBa_J06504 fused to BBa_K2605001 and [B] pSB1C3-BBa_J06504 fused to BBa_K747096 . The plasmid was double digested with SalII and NcoI then run on a 1.5% agarose gel at 100 V for 35 minutes. The molecular ladder (L) is visible on the far left and the expected band sizes, obtained from Benchling Virtual Digest, are visible in the diagram on the right. Undigested plasmid (U) was used as a control.
Week 24
The main focus of this week was our collaboration with the Notre Dame iGEM team. We hosted a graphic design workshop and oriented them to our lab early in the week, and then ran spectrophotometry experiments with them later in the week using our spectrophotometer as they did not have access to one.
We finished cloning and confirming our new part, pSB1C3-Flp/beta-resolvase hybrid and submitted all of our new and improved parts to the registry.
Later in the week, we began more serious work on our wiki and presentation, as well as some other applications for local research symposiums.
Below are notes regarding all dry lab activities.
May
Week 1
The dry lab software group focused on learning the programming languages used in web development, looked into what makes a great website, planned out the general look of the team's wiki, and researched and experimented with internet bots.  In addition, past iGEM dry-lab members came by and gave the group advice on workflow and web development and design.
The dry lab engineering group started the week off by identifying specific components of what the "box" process would look like, and how those mechanisms would be integrated onto a microfluidic device. Those components were determined to be the cell complex emulsifier, a cell/particle sorting mechanism(s), and the electroporation mechanism. The work during this week was focused on journaling different emulsification techniques. By the end of the week, a preliminary design for a chip with parallelized droplet forming units (DFU) was developed. Characteristic equations and defining parameters of channel flow were also partially identified for modelling fluid flow using CFD. However, a late-week pivot from single DFUs to a parallel DFU system requires more work on deriving those relations.
Week 2
The dry lab software group finished the basic layout of the home page and the landing page of the wiki, created and tested the pages on the iGEM server, and worked on making the wiki responsive on differing screen sizes.
For the dry lab engineering group, this week mainly consisted of continued literature reviews on the identified components of the process system.
Week 3
The dry lab software group fixed several functionality issues on the team  wiki. Adjustments were also made to ensure cross-platform compatibility across the five most popular browsers. Styling and templates were developed for the team, landing, and journal pages.  Other functional elements, content, and styling were added to the team and landing pages. The styling of the footer and logo was completed. Research led to the creation of a basic web bot that grabs software wikis by URL and completes basic HTML/CSS parsing.
For the dry lab engineering group, this week started with a continuation of component function reviews, specifically reviews on Electroporation and cell sorting. However, after a meeting with Dr. Colin Dalton from the microsystems hub, it was decided that in order to determine the equipment, specific dimensions and proper designs are needed for the microfluidic devices. Due to limited funds, it was decided that the best way to move forward was to begin modelling as well as research the manufacturing processes needed, to ensure the best possible system design.
Following this, we contacted several grad students with experience in numerical simulation of microsystems and CFD. We also began work on learning CFD with the open source software OpenFOAM. We began to learn OpenFOAM using a 3-week tutorial course available from the available wiki.
We found existing work on microsystems using OpenFOAM, so we started literature reviews on those as well.
Week 4
This week the dry lab software group developed the first iteration of the software aggregator. Working on the Django platform, they were successful in pushing data from the paragraph parser to a simple web page. The group ultimately decided that they needed to use a database to store data from the parser in order to facilitate running the aggregator less frequently. MongoDB was chosen as the database for this process. The team used the remainder of the week to fine-tune the parser, ensuring that it could grab only the necessary information, and worked on building the code that defined the interaction between the parser, database, and web page.
Early this week the dry lab engineering group began literature reviews on the physical modelling of channel flow through our system. The group also identified potential CFD tools and concluded that the mass applicability of OpenFOAM would make it a very powerful tool.
Some critical feedback received during the faculty conference was the importance of analyzing the cell count efficiencies of the proposed system and comparing them to existing cell expansion and manufacturing methods. Following this, the group focused the workflow on tuning our current physical designs depending on the results from a cursory analysis.
Week 5
This week the dry lab software group revisited the graphic design and page formatting for the team wiki. Functionality was also adjusted and improved on several components. Work was completed on natural language processing and summary generation using the tool NLTK. This work will assist with parsing the information gathered by the bot from software pages on team wikis. Caspio was used to make a searchable database that is accessible from the internet, a form that can be used by the bot or users to populate the database, and an authentication method for input into the database.
Early this week the dry lab engineering group had the opportunity to meet with a graduate student, Milad Azarmanesh, about his experiences with CFD, OpenFOAM, and Microfluidic Device manufacture. After taking in his advice, the team modified the workflow process to focus on device manufacture and experimentation first, rather than fluid flow modelling. The reason for which being the fact that all of the channel geometry and designs that will be incorporated are already well established.
This week the team was able to finish a preliminary cursory analysis on our microfluidic process units; Unfortunately, it came to attention that some of the specific process components do not scale compared to existing methods. Following this, the group decided to take a different approach, involving Hydrogels, that would cut down our process time. The rest of the week focused on identifying relevant literature and reading.
June
Week 6
This week the dry lab software group continued to work on the software bot. After an investigation into the best algorithms for summarization, the machine learning summary tool was completed. It was programmed to pass team names, years, and descriptions into an excel file. The excel file was used to populate an online database in Caspio that is referenced by the online software tool.  The design of the webpage for the online search tool was established. The first, basic, working version of the web-based software tool has been built. Work was also completed on the locally based tool, which included creating a .exe file and threading to separate the scraper from the user interface.
This week the dry lab software group primarily focused on continued research into Hydrogel materials and the manufacture of microcarrier droplets using droplet formation units. Background research into Thiol-ene click chemistry as well as Michael-type addition was conducted and will be continued into the next week.
The rest of this week was dedicated to chip design and training on AutoCAD, as well as identifying learning material for the remaining modelling software. There was also some research done into chip manufacture technique, specifically for the design of multi-layer chips.
After the introductory meeting with Dr. Yesiloz, the group decided to focus on device design for the rest of the week.
Week 7
This week the dry lab software group analyzed and improved the efficiency and accuracy of the web scraper and text summarization tool. The group also worked on completing the user interface for both the web-based bot and the local executable file.
For the dry lab engineering group, this week was focused on continuing the preliminary/introductory research on Hydrogel materials and its application into microfluidic devices.
Aside from this, the technical drawings for the first iteration of the DFU were completed. The group began scouting 3D printers that were available through other labs to see if they were able to print with the definition that was needed for the 100micron width/height channels.
The group also had a meeting with Dr. Dalton, in which he offered the group the ability to use his COMSOL license to model the DFU design, and how the group was going to deliver equal flow distribution to the inlets on the device. He had also offered to help with chip manufacturing, as a graduate student of his was developing a way to manufacture glass microfluidic devices by laser-engravement. Following this, the group contacted his student, Dr. Ebenezer Owusu, to set up a meeting the following week.
Since the group had been given the access to COMSOL, the group made the decision to pivot our work to COMSOL rather than OpenFOAM because of its smaller learning curve.
Week 8
This week, the dry lab software group worked on the software aggregator, came up with a new software project, and discussed with the Gene Maintenance subgroup what information would be included on the wiki. For the software aggregator, only a few features and making the user interface better are left. As for the new software project, the group plans on making a Google Assistant/Alexa app to help synthetic biology labs activities.
For the dry lab engineering group, hydrogel research continued this week. Specifically, the group started developing a plan for encapsulating cells in PEG-hydrogel droplets. The group learned that PEG is highly biocompatible and can be easily functionalized with different functional groups.
In preparation for the meeting with Dr. Owusu, the group had completed two DFM (droplet formation module) designs, one with 4 flow focusing junctions, and another with 8. The meeting with Dr. Eben went extremely well, however, due to manufacturing time and complexity, he preferred it if we could finalize a design for the 4-junction DFM, rather than having multiple different designs. The group learned that to get the definition that they were looking for in the corners of the channel, the laser power had to be altered at different points in-run. He also asked the group to decide which substrate the group wanted the channels to be cut into. The rest of the week was focused on finalizing the AutoCAD files with the new design considerations and choosing what type of glass the group would like to use.
The group had also contacted Dr. Mark Ungrin, a researcher at the university who often worked with microfluidics to develop lab-on-a-chip applications. He had mentioned that Alginate hydrogels are in standard use and likely cheaper than PEG. He had also offered to set up a meeting with one of his graduate students to tour his lab and see if there was anything the group could use. Aside from this, he mentioned that the work on the microfluidic devices seemed too disconnected from the core project and recommended to rethink how the group is presenting the narrative for this sub-project.
The group also worked on online COMSOL tutorials whenever possible during the week.
Week 9
The dry lab software group worked on fixing bugs in the software aggregator, as well as implementing the search function. More research was done on Google Assistant and the applications that have already been produced for it, including a past iGEM team's software, which the group hopes to improve. Finally, a general description of the team's project was put up on the team wiki.
After finishing the finalized design for the 4-junction DFM, the dry lab engineering group continued the work on Hydrogels. The group had decided on a simple manufacture procedure for PEG Hydrogel droplets via a Michael-type addition of PEG. Taking the financial investment to begin development into this new subproject, the group decided to scout nearby labs, and see if anyone had some material the group could use to test our procedure.
The group continued the weeks’ work with COMSOL tutorials and began to realize that overcoming initial filling conditions of the channels was going to be a significant challenge, to reach a time-independent operating state. The group began to consider different ways to set up the model to address this challenge.
July
Week 10
The dry lab software group began working on human practices and contacting thought leaders. They swapped out generated summaries that did not make sense with manually curated summaries. Lastly, they worked on creating pages for the wiki and improving the scraper.
After failing to obtain materials for the hydrogel tests from the group's contacts, the dry lab engineering group decided to postpone this new component of the project due to the high associated cost.
The group had also heard back from Dr. Owusu, as he had finished manufacturing 3 chips with different power levels for the group to begin testing fluid distribution. Following this, the group had cleaned and begun to do a thorough analysis of the distinctions between the chips. With this information, the group hopes to help Dr. Owusu optimize this manufacturing technique.
The group had also returned to Dr. Mark Ungrin’ lab, to meet with his graduate student. The group assessed what kind of support they could obtain from the Ungrin lab in terms of manufacture but unfortunately concluded that none of their techniques would assist us in developing channels on 100micron scale.
Week 11
The dry lab software group completed the exchange of generated summaries with manual summaries. They continued work on SARA.exe and the search function.
Early in the week, the dry lab engineering group met with Dr. Dalton again to discuss testing protocols and initial characterization tests before achieving proper droplet formation. The group was looking to characterize the surface profile of the channels to get a sense of how well the laser can create a consistently flat surface. The group was also looking for obtaining syringe pumps to perform fluid and filling testing with the chips. To conclude this meeting, Dr. Dalton offered the group access to his microfluidic test lab, for general tools and consumables.
Later on, the group met with Dr. Owusu again to discuss a concept for another module that they had begun to develop. This was called the Droplet Electroporation Module, or DEM, which is essentially a microscale electroporator. However, the focus was instead on a more accessible way to manufacture and integrate the electrodes into the chip. The design that the group consulted Dr. Owusu with was vertical electrode pairs, that are holes drilled into the substrate, with a conductive filling. The channel cover acts as the ground electrode, which is a conductive PDMS mix. The group discussed that it may be difficult to manufacture the actual holes, but it was worth a try.
The rest of the week was devoted to finishing a design on AutoCAD to test the electrode holes, that would not require the full machine time. The group also finished optically characterizing the chips that they had received.
Week 12
The dry lab software group began working on and researching the synthetic biology google home app. They also continued working on and testing SARA.exe and developing the visuals. Lastly, they attended the North American iGEM Kick-Off, where they learned about wiki design and collaborated with other teams.
On Monday Dr. Dalton had given the dry lab engineering group a safety session for his microfluidics testing (MT) lab. The rest of the work in the MT lab this week had just been identifying and making an inventory of the available machines and consumables that had been made available to the group.
Dr. Owusu had contacted the group to say that he machined two additional DFMs, with a slightly different power level and engraving path that the group had initially proposed to him. After optically characterizing the grooves inside the channels, the group concluded that the power/path design did not properly work, as the depth in the junctions was far deeper than had been intended.
As the group waited for Dr. Owusu to fabricate the test design for the DEM, the group began to look into ways to optimize the difference between electrodes to ensure that cells could theoretically be transfected.
Week 13
The dry lab software group worked on planning a cohesive wiki and created a complete summary of the project for stakeholders (people involved with human practices and other mentors). They also discussed the future of the Lab Research Assistant (LARA) and took steps to achieve their goals.
This week the dry lab engineering group began to test delivering fluid to the chips. Initially, the group had set up a fluid splitter using syringe needles, pieces of PET tubing, and plastic valves. However, they were quick to realize that this setup concluded in very long fluid paths, which in turn creates very high-pressure drops. It was not possible to deliver fluid to the chips like this.
To address this, the group made a fluid splitter made from PDMS, with one inlet and four outlets. While this seemed to work decently, the group did not like how the tubing that was connected to the PDMS cover on the chip obstructed view from the channels. The group concluded that this setup was not robust enough and began to develop on-chip reservoirs that distribute fluid to the inlets equally and does not obstruct vision.    A preliminary model on COMSOL showed us that the design the group had in mind would work, so the group spent the rest of the week refining to our design requirements.
August
Week 14
The dry lab software group worked on implementing the plan for the wiki, specifically the info, journal, team, and project pages. They continued working on SARA.
For the dry lab engineering group, the design for the fluid reservoirs was finalized. It consists of concentric rings, whose fluids do not come into contact until they reach the flow-focusing junctions. The group decided to make these reservoirs out of acrylic sheets since it is a transparent material, very easy to source and machine it by laser cutting.
To fully assemble the devices, the group plasma-bonded a PDMS interlayer onto the glass chips. This interlayer contained eight fluid inlets and one central outlet, which correspond to the different rings on the acrylic rings. The reservoir is then aligned to the inlets on the interlayer and clamped to secure a tight seal. The top of the reservoir has two fluid inlets, for the continuous and dispersed phase, and an outlet for the droplets.
Once assembled the group began to test how fluid moves inside the reservoirs to reach steady-state conditions.
Week 15
The dry lab software group worked on making wiki mockups for various pages, including the team page, navigation bar, and project page. They also started creating all the pages required for the wiki.
Week 16
The dry lab software group worked on wiki pages and fixing bugs on pages. They completed the design for the project, home, team, and journal pages. They also designed the home and protocol page for SARA. Lastly, they designed apparel (t-shirt and crewneck).
Week 17
This week, the dry lab software group finished up the wiki framework and page structures. They worked on the ability of pages to resize. They began writing content for the software page.
For the dry lab engineering group, testing fluid filling of the reservoirs was achieved this week. It was possible to achieve fluid traversing from inlet to outlet of the chip; however, droplets were not observable, and fluid began to lead between the reservoirs, indicating that clamping the device was not enough in getting an appropriate seal.
It is also possible that droplets were not formed because the DFM’s that have been fully assembled did not have a uniform surface profile. However, Dr. Owusu had contacted the group, and let them know that he had been able to engrave consistent depths onto a glass slide by tuning the power output of the laser in-run, and by using the paths that we had designed for the flow-focusing junctions. This was also made possible by the new substrate material; a piece of borosilicate glass, which has a larger resistance to the laser over-etching a certain area. After optical characterization, the group was able to hypothesize that this new etch had the profile that would be needed to develop droplets.
Week 18
The dry lab software group implemented a submission form for SARA, completed writing up the software page for the wiki, and played around with the creation of a notebook and protocol tool for LARA and SARA.
To improve the sealing between the reservoir and the PDMS interlayer, the dry lab engineering group had tried to use a PDMS solution of 1:5 curing agent to monomer as a glue. They applied small amounts of the pre-cured polymer to the areas of interest and cured the whole device at 80 degrees for 4 hours. Sealing was achieved on the outer layer of the reservoir, but it was noticed that the inner rings had not bonded properly. This suggested that the surface area if the rings were not large enough to bond the two pieces, which lead to the whole structure failing.
The group concluded that a new reservoir design would have to be fabricated, in which the bonding areas were larger than before.
September
Week 19
The dry lab software group designed and created the team banner. They also finalized the team apparel design and order. They began compiling journal entries for each subgroup.
Early in the new design cycle the dry lab engineering group realized that it would be difficult to model what surface area for the new reservoir is needed, on COMSOL. Due to this, they made a number of physical acrylic models, to rapidly prototype reservoir filling and inlet/outlet locations, as well as the surface area needed to bond the components.
Week 20
The dry lab software group created, filmed, and edited a video summary of the project for the aGEM competition. They continued to develop functionality for SARA and decided to discontinue work on LARA, due to time constraints.
Using the reservoir prototypes helped to inform the design for an improved reservoir layer, the dry lab engineering group began to draw a new concept on AutoCAD, in which the fluid inlets do not feed directly into the glass channels, rather come from an inlet that is closer to the edge of the glass slide.
Week 21
The dry lab constructed the slideshow presentation for aGEM. They made decorative graphics and scientific diagrams to accompany the oral presentation.
This week the dry lab engineering group focused on helping the team prepare the presentation for aGEM.
Week 22
The dry lab software group identified hosting for SARA, which allowed for access to the tool online. They also created a slideshow presentation for a synthetic biology lesson plan aimed at high school students.
October
Week 23
The dry lab software group created a document of wiki editing guidelines for non-dry lab team members. They developed an algorithm for finding teams in the software track and scraping the project descriptions. This functionality is valuable and was not previously implemented due to the organization of software in iGEM's database.
The dry lab engineering group began to set up a new COMSOL model, to see if the new reservoir design could distribute pressure uniformly like before.
Week 24
The dry lab software group led a graphic design workshop and designed a logo for the Notre Dame Collegiate High School iGEM team. They taught the high school team how to make logos, diagrams, and presentations with various online tools.
