Team:Calgary/FLP-Beta
TOP
Project
Description CRISPR FLP-Beta CME Product Design Modelling Software Results
Notebook
Journals Protocols
Team
Team Members Attributions Sponsors Contact Us
Human Practices
Summary Silver Integrated Collaborations Public Engagement Safety
Judging
Medal Requirements Parts Improved Part Interlab
Team:Calgary/FLP-Beta - 2018.igem.org
FLP-BETA
FLP Recombinase & Beta Resolvase Use Overview
Flp recombinase and Beta resolvase were used to enable the insertion of large amounts of DNA into the genome, at the Six/FRT site previously inserted using CRISPR/Cas9 (see the CRISPR page for more details). FlpO was used to mediate recombination of our desired plasmid, which does for beta resolvase as well as provides the machinery necessary for beta-binding to the genome.  Beta resolvase was subsequently able to bind the genomic DNA. This allowed for a second recombination event, which removed junk plasmid sequences while locking the DNA insertion into the genome.
What is FLP Recombinase?
FlpO is a recombinase protein which mediates exchange of DNA sequences between two regions. FlpO functions by recognizing specialized Flp Target Recognition (FRT) sites in two different strands of DNA. Upon FRT recognition, FlpO cuts the DNA at the FRT sites and mediates the exchange of DNA. In the case where two FRT sites exist, the exchanged region is everything between the two FRt sites, and is known as a “cassette”. This process is known as Recombinase Mediated Cassette Exchange (RMCE).  In the case where only a single FRT site exists, one on each vector,  the vector is is fully integrated into the other vector. For example, a plasmid containing an FRT site would be completely integrated into a genome containing a matching FRT site.
What is Beta Resolvase?
Beta resolvase is a recombinase protein like Flp, however it harbours a few important differences; beta resolvase will only recombine sequences found on the same molecule, and the beta recognition sites (called “Six” sites) are broken into subsites. As a type of recombinase that only recombines sequences found on the same molecule (i.e. not between plasmids or between plasmid and genome), Beta's activity is irreversible and produces fewer unintended products. Unlike FLP, Beta recombines target sites (six sites) that are comprised of two subsites. The complex structural interplay between those subsites is necessary for Beta activity, and is the reason Beta does not recombine between molecules.
Our Hybrid Flp-Beta Approach
The use of recombinase mediated cassette exchange (RMCE) was initially our chosen method for genomic integration, but it was replaced by a customized hybrid FLP-Beta approach due to a key limitation with RMCE: the natural removal of integrated DNA from the genome by FLP. The issue with RMCE lies in the equilibrium reaction that favours the removal of the larger cassette from the genome, making us doubt that its efficiency could be high enough for reliable and lasting gene integration. Furthermore, RMCE requires two FRT sites in order to work. This increases the size of the initial CRISPR landing pad knock-in, and reduces efficiency. For these reasons, we shifted our focus to an experimental approach of our own design involving hybrid Flp-Beta activity.
At its core, our approach requires Beta's target subsites to be separated by an FRT site, which allows the two subsites to be separated or combined by FLP recombination. In other words, Beta recombination can be activated by FLP recombination. To this extent, a single subsite of a Six recognition site was attached to the FRT landing pad which was knocked into the genome using CRISPR. FlpO was then used to integrate a plasmid containing an FRT, the other half of the Six site, and any other genetic cargo that the user might want. For our purposes, this cargo was puromycin resistance, without an attached promoter.  Following Flp-integration of this plasmid, Beta recombination was also allowed to proceed due to the now-complete Six site. Beta recombination excises the sequences between the Six sites, effectively removing the unnecessary plasmid backbone which was integrated into the genome. More importantly, Beta recombination also removes a single FRT site from the genome, preventing removal via RMCE. For the purposes of our testing, Beta recombination also brought the puromycin resistance gene adjacent to a promoter enabling expression. Through this mechanism, puromycin resistance was only achieved when both Flp recombination and Beta recombination occurred, allowing us to select for modified cells. Below is a diagram showing the various recombination stages to help explain this sequential action:
Testing
Testing our recombination stage was a follow-up to the initial CRISPR event, and relied fully on the insertion of the Six/FRT landing pad in order to begin testing. Due to the difficulties we encountered with verifying CRISPR-mediated knock-in of our Six/FRT site, we were unable to exclusively test the effectiveness of our recombination system (See CRISPR for more details). With that in mind, following the transfection of our CRISPR/Cas9 system and Recombination system (plasmid containing the insert, Beta resolvase gene and the second Six subsite), we found a few HEK293T cells growing in puromycin media, while the control transfection which contained the CRISPR system but not the Recombination system was fully dead in puromycin. Based on the aforementioned design of our system, puromycin resistance would only be conferred following the activity of both FlpO and Beta resolvase. Therefore, these surviving cells could potentially have a functional CRISPR landing pad insert, as well as functional recombination insertion.
We are incredibly excited about this preliminary result, however it is far from conclusive. These surviving cells seems to indicate that our system worked as a whole, but this assertion must be validated through sequencing.  To this extent, we must wait for the few surviving cells to expand before we can can isolate them and analyze their DNA. Unfortunately, due to the extremely low number of surviving cells and the long doubling time of HEK293T cells (~24 hours) compared to bacterial chassis like E. coli, we will not have enough cells to perform a more thorough analysis before the Jamboree. Below are photos contrasting live versus dead cells, as well as the photos of our transfection plates in puromycin media.
Dead cells, no adherence
Living cells showing adherence to plate
No-FlpO Control Transfection (Dead)
Recombination System Transfection, with a single living cell
WORKS CITED
Dı́az, V., Rojo, F., Martínez-A, C., Alonso, J., & Bernad, A. (1999). The prokaryotic β-recombinase catalyzes site-specific recombination in mammalian cells. Journal Of Biological Chemistry, 274(10), 6634-6640.
Grønlund, J. T., Stemmer, C., Lichota, J., Merkle, T., & Grasser, K. D. (2007). Functionality of the β/six Site-Specific Recombination System in Tobacco and Arabidopsis: A Novel Tool for Genetic Engineering of Plant Genomes. Plant Molecular Biology, 63(4), 545–556.
Hubbard, E. J. A. (2014). FLP/FRT and Cre/lox recombination technology in C. elegans. Methods (San Diego, Calif.), 68(3), 417–424.
Long, D.-P., Zhao, A.-C., Chen, X.-J., Zhang, Y., Lu, W.-J., Guo, Q., … Xiang, Z.-H. (2012). FLP recombinase-mediated site-specific recombination in silkworm, Bombyx mori. PloS One, 7(6), e40150.
Rowland, S., Boocock, M., & Stark, W. (2005). Regulation of Sin recombinase by accessory proteins. Molecular Microbiology, 56(2), 371-382.
Subramaniam, S., Erler, A., Fu, J., Kranz, A., Tang, J., Gopalswamy, M., … Stewart, A. F. (2016). DNA annealing by Redβ is insufficient for homologous recombination and the additional requirements involve intra- and inter-molecular interactions. Scientific Reports, 6(1), 34525.
