Team:Calgary/Improve
TOP
Project
Description CRISPR FLP-Beta CME Product Design Modelling Software Results
Notebook
Journals Protocols
Team
Team Members Attributions Sponsors Contact Us
Human Practices
Summary Silver Integrated Collaborations Public Engagement Safety
Judging
Medal Requirements Parts Improved Part Interlab
Team:Calgary/Improve - 2018.igem.org
IMPROVED PART
Part Improvement
Part BBa_K2605001 is an                     improvement of Part BBa_K747096
Both parts are the constitutive mammalian cytomegalovirus (CMV) promoter,                 however our part is an improvement on BBa_K747096 by the addition of a SalI restriction site to                 the 5' end of the promoter. The addition of the SalI restriction site allows for the promoter to be                 used within our Multiple-Cloning Site flanked by FRT sites (more info on the Multiple Cloning-Site and                 its usage can be found here ). Although several                 CMV promoters exist in the registry, we chose to improve upon BBa_K747096 for two reasons; it was                 available in the 2018 iGEM distribution kit and it was relatively well-characterized.
The SalI restriction site (sequence: GTCGAC) was added to BBa_K747096 via                 overhang PCR. Overhang PCR is just like regular PCR except the primers that are used contain both a                 complementary annealing region and additional 5' or 3' nucleotides, which are not complementary to the                 target sequence. The "overhanging" 5' or 3' nucleotides are incorporated into the PCR product and this                 allows for the addition of nucleotides to the end(s) of your target sequence.
To create our improved part, BBa_K2605001, we first rehydrated the DNA of part                 BBa_K747096 in the 2018 iGEM distribution kit then transformed it into chemically competent E. coli DH5-alpha cells. The plasmid DNA was extracted and the SalI restriction site was added with overhang                 PCR. We used the forward and reverse primers pictured below (figure 1), which contain the SalI               restriction site in the overhang region of the forward primer.
Figure 1. Forward and reverse primers used in overhang PCR for the addition of a SalI restriction site. All nucleotides on the forward primer upstream of and including the SalI site are the non-complementary, "overhanging" portion of the primer.
Experimental Characterization
Restriction enzyme digest was used to confirm the addition of the SalI                 restriction site to the CMV promoter. pSB1C3-BBa_K2605001 and pSB1C3-BBa_K747096 were digested with                 either NotI (to confirm part size) or SalI/SpeI (to confirm the presence of the SalI restriction site).                 As expected, NotI cut both plasmids and produced a roughly 600kb insert while SalI/SpeI only produced a                 600kb insert from our improved part, pSB1C3-BBa_K2605001. Since pSB1C3-BBa_K747096 only contains a SpeI                 site, the plasmid was linearized and the insert band was not produced. The results of this restriction                 enzyme digest can be seen below (figure 2).
Figure 2. Digest confirmations of [A] pSB1C3-BBa_K2605001 and [B]pSB1C3-BBa_K747096 . The plasmid was double digested with SalII and SpeI or single digested with NotI  then run on a 1.5% agarose gel at 100 V for 35 minutes. The molecular ladder (L) is visible on the far left and the expected band sizes, obtained from Benchling Virtual Digest, are visible in the diagram on the right. Undigested plasmid (U) was used as a control.
To verify that our improved part could still function as a promoter, both parts                 were ligated to a red fluorescent protein (RFP) reporter gene optimized for bacteria ( BBa_J06504 ), which was found in the 2018 iGEM distribution kit. Although                 CMV promoters are normally used for mammalian expression, it also functions as a promoter in E. coli (Lewin et al., 2005), therefore it could be used to express the RFP reporter gene that was fused to it.
First, restriction enzyme digest was used to confirm that the ligation of our                 improved part (BBa_K2605001) and the old part (BBa_K747096) to the RFP reporter gene                 (pSB1C3-BBa_J06504). The constructs were digested with SalI/NcoI I to validate this. Again, both                 constructs produced different band sizes, due to the additional SalI site present on the construct with                 our improved part. The results of this restriction enzyme digest can be seen below (figure 3).
Figure 3. Digest confirmations of [A] pSB1C3-BBa_J06504 fused to BBa_K2605001 and [B]pSB1C3-BBa_J06504 fused to BBa_K747096 . The plasmid was double digested with SalII and NcoI then run on a 1.5% agarose gel at 100 V for 35 minutes. The molecular ladder (L) is visible on the far left and the expected band sizes, obtained from Benchling Virtual Digest, are visible in the diagram on the right. Undigested plasmid (U) was used as a control.
Then, the digest-confirmed CMV-RFP constructs and RFP alone (pSB1C3-BBa_J06504,                 negative control) were transformed into chemically competent E. coli DH5-alpha cells. Successful                 transformants were plated and imaged after 48h, at which time RFP production could be seen in both                 CMV-RFP constructs (figure 4, figure 5), while the negative control remained colourless (figure 4) as                 RFP could not be produced without a CMV promoter.
Figure 4. Streak plates of E. coli DH5-alpha containing TOP: pSB1C3-BBa_J06504 (RFP) fused to BBa_K747096 (CMV), RIGHT: pSB1C3-BBa_J06504 (RFP) fused to BBa_K2605001 (CMV) LEFT: negative control, pSB1C3-BBa_J06504 (RFP).
Figure 5. Streak plates of E. coli DH5-alpha containing TOP: pSB1C3-BBa_J06504 (RFP) fused to BBa_K747096 (CMV), BOTTOM: pSB1C3-BBa_J06504 (RFP) fused to BBa_K2605001 (CMV).
These characterization results indicate that the SalI restriction site was                 successfully added to the 5' end of part BBa_K747096, a CMV promoter, and that our improved part, BBa_K2605001,                 maintained its function as a promoter.
WORKS CITED
Lewin, A., Mayer, M., Chusainow, J., Jacob, D., Appel, B. (2005). Viral promoters can initiate                     expression of toxin genes introduced into Escherichia coli. Biomed Central Biotechnology, 5(19). doi:  10.1186/1472-6750-5-19.
