Team:IISER Pune/Projects
Your browser does not support the canvas tag.
Top
Project Background
Tuberculosis, an infectious pulmonary disease, is caused by Mycobacterium tuberculosis which spreads through air. The most common symptoms include cough lasting more than 3 weeks, weakness, fever, chest pain, coughing up blood/sputum, etc. The minimum duration of treatment is 6 months. If not treated properly, this disease can be fatal. According to the WHO Global TB Report 2014, India has the highest TB incidence. A large population combined with poor hygienic conditions, moderate temperatures, socio-economic conditions- all contribute to the high prevalence of TB in India.
The average duration from the appearance of symptoms to the beginning of treatment is 2-4 months. The reason for this large gap is that  M. tuberculosis is a slow growing pathogenic bacterium with a doubling time of approximately 24 hours in tissues. This property of M. tuberculosis hinders rapid detection in sputum samples obtained from patients. Also, the commonly used diagnostic techniques in India not only require a time period of 2-4 weeks but also cannot detect lower concentrations of mycobacteria in the sample from the patients.
Thus, there is a need for a diagnostic technique that is cost effective, yet is quick and can still detect mycobacteria in lower concentrations.
Project Overview
Tuberculosis is caused by Mycobacterium tuberculosis. In India, TB poses a great health threat with nearly 40% Indians having either latent or active TB. The most commonly used TB diagnostic techniques in India require a minimum of 2-3 weeks, thus causing a delay in treatment. Some of the techniques used require high-end instruments which many clinical labs may not have access to. Mycobacterium tuberculosis is a slow-growing bacteria and hence, the detection of this bacteria in patient samples using present techniques requires a longer duration. In the quest to improve TB diagnostics, we propose a technique that does away with the need of high-end instruments and complicated procedures. This technique aims to give an easy-to-identify output upon the addition of our genetic device to the sample.
Our genetic device consists of 3 modules - Hijack, Detection and Termination.
Hijack : This module of our project aims at hijacking the cell cycle to increase the growth rate of cells.The hijacking is achieved by means of a genetic oscillator that targets 2 proteins - dnaA and ftsZ.
Detection : As the number of cells increase, this module aims at detecting these cells by producing a colour output. For this purpose, three possibilities were explored - chromoproteins, enzyme-substrate reactions and carotenoid production in cells.
Terminator : Once the Hijack machinery takes over, the cells divide rapidly as their growth rate increases. The Terminator turns on the suicide switch at a particular cell density and prevents uncontrolled proliferation of these cells.
Detection Module:
The detection module focuses on minimizing the time required for the confirmation of a test. Production of a visible reporter will aid to achieve this goal without the usage of any sophisticated equipments or further processing and analysis of the results.
Three frontiers were explored for the same: Carotenoids Chromoproteins Enzyme-Substrate reactions
We were successfully able to express two carotenoids, namely, β-carotene and lycopene in E.coli. The characterization of these parts has been already carried out by our team using the biobrick provided by iGEM team Cambridge of 2009. Also, the cloning and expression of the other two strategies has been worked on simultaneously. The chromoprotein selected for expression was amilCP and β-galactosidase-IPTG reaction was worked upon in this module.
Further, we wish to express and characterise all the parts in M.smegmatis as a proof-of-principle for M.tuberculosis.
Hijack Module:
This Module of our project targets two proteins dnaA and ftsZ.
DnaA: DnaA is the initiator of DNA replication. It is a negative auto-regulator, which means that when concentration of dnaA goes up, dnaA itself inhibits its own transcription. It has 2 forms - the ATP-bound form and the ADP-bound form. The ATP-bound dnaA (ATP-dnaA) is the active form which initiates replication. It does this by binding to the dnaA box. The oriC has many such dnaA boxes. The binding of dnaA is the first step in the formation of the initiation complex.
In the replication process, it takes a constant time to replicate the chromosome.This time is not dependent on the actual growth rate of the cells. Once the replication is initiated, the actual DNA replication takes a constant time. Thus, the frequency of initiation of DNA replication determines the rate of DNA synthesis and hence will determine the growth rate of cells. In E.coli, overlapping rounds of replication enable the bacteria to have a doubling time less than the time required for DNA replication. This mechanism is not present naturally in mycobacteria.
It has been shown that a certain amount of DnaA-ATP must be present in the cell to initiate replication of DNA. The ratio DnaA-ATP : total genome length  must reach a threshold value to begin replication.
Thus, our device will increase the frequency of initiation of DNA replication by oscillating the levels of dnaA in the cell.
FtsZ: FtsZ, a tubulin like protein, is the initiator of cell division.It has 2 forms – GTP bound (which is active) and GDP form (which is inactive). It is the first protein to assemble at the mid-cell and forms a ring like structure called the Z-ring by assembling various FtsZ units and recruiting other protein units. This Z-ring structure forms the divisome complex which brings about the division of the cell into 2 daughter cells. It ensures temporal synchronization with the DNA replication.  Thus we wish to oscillate FtsZ along with DnaA to maintain synchronization with the increased frequency of initiation of DNA replication.
To hijack the cell cycle and enhance the growth rate of cells, we make use of a genetic oscillator. We explored many genetic oscillators of which we chose to further explore 2 oscillator designs:
� The Fast, Robust Tunable Oscillator  This oscillator was designed by Stricker et al.using a hybrid promoter plac/ara-1. The design behind this genetic oscillator is as follows:  Figure: Genetic Oscillator design
-
The molecule A activates its own transcription and the transcription of molecule B.
-
This causes an increase in levels of both molecule A and B.
-
As amount of B increases, B inhibits its own transcription and the transcription of molecule A.
-
This causes a decrease in levels of both A and B.
-
As levels of B go down, the inhibition action of B decreases and A again activates transcription of both molecules.
-
This cycle repeats leading to oscillations of concentrations of A and B in the system.
In the design by Stricker et al. (which we will also follow), A = araC and B= LacI.
The hybrid promoter is composed of:
Activation operator site from the araBAD promoter (araC protein binds to this site in the presence of arabinose to activate the promoter)
Repression operator sites from the lacZYA promoter (lacI binds to this site in the absence of IPTG to repress the promoter)
Thus, it requires both arabinose and IPTG to become active.
The complete construct of the oscillator uses GFP as a  reporter molecule to observe oscillations. We will also use GFP to test the working of the oscillator.
In our version of this oscillator, we replace GFP with dnaA and ftsZ.
We currently have successfully acquired all the parts necessary to build this oscillator and are assembling these parts, after which we will test the oscillator in E.coli and attempt to quantify the GFP oscillations.
Refrences:  Hett, Erik C., and Eric J. Rubin. "Bacterial growth and cell division: a mycobacterial perspective." Microbiology and Molecular Biology Reviews 72.1 (2008): 126-156. Grant, Matthew AA, et al. "DnaA and the timing of chromosome replication in Es-cherichia coli as a function of growth rate." BMC systems biology 5.1 (2011): 201. Messer, Walter. "The bacterial replication initiator DnaA. DnaA and oriC, the bacterial mode to initiate DNA replication." FEMS microbiology reviews 26.4 (2002): 355-374. Stricker, Jesse, et al. "A fast, robust and tunable synthetic gene oscillator."Nature 456.7221 (2008): 516-519. Lutz, Rolf, and Hermann Bujard. "Independent and tight regulation of transcriptional units in Escherichia coli via the LacR/O, the TetR/O and AraC/I1-I2 regulatory elements." Nucleic acids research 25.6 (1997): 1203-1210.
Terminator Module
The aim of this module is to build a self-sustained bacterial population control device.
The need for termination arises because controlled oscillations of cellular proteins, dnaA and ftsZ will lead to rapid division of the cells. We want to incorporate a self-destruct mechanism into our genetic device to prevent this division from getting out of control. Termination will thus stop replication of hijacked cells.
Termination should not occur before the pigment is detected. The pigment output produced by the cells directly correlates to the number of cells in the sample. Hence, termination should begin only after this critical cell density is reached. Our goal was to use quorum sensing to achieve gene regulation based on changes in cell density.
We incorporated the lux operon into our population growth control device. The major components of this operon are the autoinducer, lux promoters, and regulatory protein luxR.
The luxI gene codes for a synthase enzyme that produces the autoinducer. At high cell density, the autoinducer binds to luxR, making an activated complex. This complex then binds to the lux right promoter, allowing for the transcription of downstream genes. The gene coding for the CcdB toxin is our ‘killer’ gene. It poisons the DNA-Gyrase complex comprising of topoisomerase II, thus disrupting DNA replication. The cell eventually stops dividing and becomes filamentous, remaining intact. The pigment produced by the cell will thus remain concentrated within the boundaries of the cell membrane.
