Team:Tuebingen/Model
iGEM Tübingen 2017
Team Inspiration Results Human Practice Lab Attributions
model
Introduction
Theoretical Background
Procedure
Results
Modelling
Introduction
The typical drug discovery workflow includes the selection of a target molecule, the identification of potential drug candidates, their validation as lead compounds and optimization as well as preclinical and clinical studies. The successful development of one novel drug can take up to 15 years and requires millions of dollars. This process specifically the identification of hit and the optimization of lead compounds can be facilitated and/or accelerated through computational modeling techniques thereby significantly lowering the cost and time requirements.
Structure-based virtual screening (SBVS) tools have been extensively studied and proven to be successful in aiding modern drug development. The general principle behind SBVS is the docking of small compounds into the binding pocket of the target protein, e.g. a receptor or enzyme, and the subsequent selection of high-ranking compounds for further testing.  Since SBVS differentiates between active and inactive compounds by modeling and scoring the interaction between compound and protein target, it can also evaluate how modifications of the compound impact this interaction (Tetko 2005).
The aim of the modeling studies was to evaluate the interaction between the lead antibiotic clorobiocin and its target protein DNA Gyrase B specifically the 24 kDa ATP binding domain. Novobiocin  was used as a reference since the FDA approved antibiotic is highly similar in structure and well described in the literature. Two more stable compounds were generated based on the clorobiocin structure and the impact of these modifications was evaluated on the compound-target interactions.
Docking and scoring was performed through the Schrödinger small-molecule drug discovery suite.  Physicochemical properties of the modified compounds were computed based on quantitative structure-activity relationship models via chemoinformatic module CDK in KNIME (Beisken 2013)
Background
Both novobiocin and clorobiocin are described as inhibitors of Gyrase B activity by competitively binding to the ATP-binding site. The interaction between clorobiocin and Gyrase B has been described in detail by Lafitte et al. As seen in Figure 1 both polar and apolar interactions occur between clorobiocin and its target. Important polar interactions include the hydrogen bond between the Arg136 side chain within the binding site and 2-carbonyl oxygen of the ligand, the bond between the carbonyl oxygen of Asn46 and the hydroxyl of the 2’-noviose as well as the bond between the Asp73 side chain and the imino group of the pyrrole ring. Additionally, the ring is embedded between Thr165 and Asn46 and sits in a hydrophobic pocket generated by Val71, Val43, Val120, Val167 and Ile78. Another apolar interactions is the stacking of the Arg76 side chain against the coumarin ring. The binding site also includes two water molecules bridging between ligand and protein (Tsai 1997, Lafitte 2002).
Figure 1: Interaction diagram between clorobiocin and the binding pocket of Gyrase B (1KZN)
The interactions between novobiocin and Gyrase B at the same site are highly similar. However, the binding site contains two more water molecules. In case of a clorobiocin ligand those water molecules are displaced by the pyrrole ring.  Prior experiments provide evidence that clorobiocin has a higher affinity to Gyrase B than NOV. This can be attributed to the reorganization of the water molecules or the additional hydrophobic contacts formed by the pyrrole ring within the binding pocket (Lafitte 2002).
Results and discussion
Prediction of druglikeness
Assessment of the physicochemical properties of the drug candidates is an early step in the drug discovery pipeline. Candidates must be lipophilic enough to permeate membranes and reach their intended target. The octanol-water partition coefficient (logP) which is a measure of lipophilicity can be determined by measuring the distribution between a non-aqueous phase (octanol) and aqueous phase (water).  Based on the extended Lipinski’s rule of 5 which calls for logP values between -0.4 and 5.6 candidates outside this range are not viable drug candidates (Lipinski, 2001).
Another physicochemical property which affects bioavailability is the solubility in water (logS). Insufficient solubility limits the absorption of the compounds in the gastrointestinal tract. logS values of less than – 4 are desirable for candidate compounds.
Both logP and logS can be predicted based on quantitative structure-activity relationship (QSAR) models. QSAR models are developed by relating physicochemical and structural properties from well described compounds to their experimentally determined biological activity. A variety of tools are available to predict the activity of unknown compounds based on QSAR models. We used both the CDK toolkit provided by KNIME and the webtool ALOGPS 2.1 to determine logS and logP values to decide whether our modified compounds are viable candidates (Mannold 2009, Beisken 2013).
As seen in Table 1 clorobiocin does not fulfill the requirements for drug candidates regarding logP and logS values. Clorobiocin was modified to decrease lipophilicity and increase the solubility in water. The aim was to generate compounds with similar properties to the reference NOV and ideally fulfill the requirements of drug candidates. The introduction of the β-Lactam ring structure improved the properties. Based on the values predicted by ALOGPs the lipophilicity of the new compounds was significantly decreased, at the same time aqueous solubility was increased and even higher level compared to the reference NOV.
Table 1: partition coeffiecient (logP), predicted by KNIME CDK, AlogP, values are scaled according to NOV
KNIME
ALOGPs
ALOGPs
property
logP
logP
logS
novobiocin
4.10
4.10
-4.80
clorobiocin
6.35
6.38
-5.32
β-lactam clorobiocin
5.03
4.46
-4.49
intermediate
4.36
4.74
-4.58
activated TODO
5.29
4.74
-4.72
General screening
The aim of the initial screening procedure was to determine whether the alterations of the clorobiocin structure would impact the binding of the ligand to the gyrase B structure. Throughout the docking procedure the structure of the binding pocket was kept rigid. To compensate for this and to simulate a flexible binding pocket we selected various published structures of the 24 kDa ATP-binding domain from the protein data base and performed docking with each.
An overview of all docking results including the top docking score and the respective predicted binding energy are available for download in the supplementary files. In the great majority of all cases the glide docking procedure generated ligand poses with similar or higher docking scores for both Lactam-clorobiocin, the stable intermediate and the activated structure. Therefore, we propose that the introduction of the β-lactam ring structure does not impair the interaction between gyrase B subunit and the compounds.
Due to the missing structure of the outer loop consisting of the amino acids at position 108 to 120 it is difficult to draw a final conclusion regarding the position of para-hydroxy benzoic acid derivative. The antibiotic effect of the aminocoumarins can be attributed either the    competitive occupation of the ATP binding site or the prevention of the dimerization of the gyrase subunits by blocking the N-terminal section (Tsai 1997).
Modifications to the para-hydroxy benzoic derivative have a huge impact on the bioactivity of the aminocoumarin. Various in vivo studies have shown that a hydrophobic moiety with an carbon chain of n(C)=3 is needed for an effective inhibition of the gyrase. Furthermore the introduction of hydrophilic moieties significantly lower the antibiotic efficacy (Anderle 2008)
Supplementary files: all docking results.
Predicted interaction between decarboxylated Troiacin and gyrB
We selected a previously published gyrase B structure crystallized with clorobiocin to evaluate the interaction pattern of our newly designed compound decarboxylated Troiacin in detail. The interaction pattern of clorobiocin is depicted in Figure 1. Docking the decarboxylated Troiacin structure into the same binding pocket revealed a highly similar interaction as seen in Figure 2 and 3. Both clorobiocin and decarboxylated Troiacin formed hydrogen bonds to the residues Asp73 and Arg76 and Arg136. π-stacking occurred between the arginine residue at position 76 and the aminocoumarin ring of either compound. Similarly, the pyrrole ring was placed into the hydrophobic core of the binding pocket in both cases
.
Figure 2: Interaction diagram between decarboxylated Troiacin and the binding pocket of Gyrase B (1KZN)
Figure 3: decarboxylated Troiacin positioned in the binding pocket of Gyrase B (1KZN)
Docking poses of decarboxylated Troiacin scored better than both clorobiocin and novobiocin poses. The predicted free energy was lower as well. This suggests that decarboxylated Troiacin has a higher binding affinity to gyrase B than novobiocin and clorobiocin. Based on the in silico modeling of the interaction between decarboxylated Troiacin and gyrase B we propose that the modified compound has at least the same efficacy as clorobiocin and to proceed with both the synthesis and the evaluation of the compound in vitro and in vivo.
Predicted interaction between decarboxylated Troiacin and novobiocin-resistent gyrB
Mutations of residues within the binding pocket of gyrase B give rise to resistance against novobiocin. We selected a NOV-resistant gyrase B structure from PDB to determine whether modifications to the compound are able to reestablish the antibiotic effect.
Ligand docking was performed with our compounds and a novobiocin-resistant variant of gyrase B. The reported resistance was introduced by an exchange of Arg at position 136 with His. As seen in Figure 4 and 5 the compound was still placed within the binding pocket through the docking procedure but in a shifted position which potentially impairs the efficacy of the antibiotics.  Interestingly the amino-isobutyl-group of decarboxylated Troiacin was able to reposition the compound in the resistant structure as depicted in Figure 6 and 7. Therefore, we postulate that the rearrangement and stabilization of the compound at its original position through the interaction between its amino-isobutyl group and the Glu residue at position 137 will increase the efficacy of the compound.
.
Figure 4: Interaction diagram between novobiocin and the binding pocket of Gyrase B (1AJ6)
Figure 5: Interaction diagram between clorobiocin and the binding pocket of Gyrase B (1AJ6)
Figure 6: Interaction diagram between decarboxylated Troiacin and the binding pocket of Gyrase B (1AJ6)
Figure 7: decarboxylated Troiacin  positioned in binding pocket of Gyrase B (1AJ6)
Predicted interaction between β-lactam clorobiocin and the human topoisomerase type II
As of yet only NOV is approved as a therapeutic agent by the FDA. Unfortunately, toxicological studies have shown severe side effects since the human topoisomerase type II is confirmed an off-target (Sadiq 2010). We repeated the docking procedures with the human topoisomerase structures to exclude compounds with stronger interaction with the off-target and a stronger potential to cause tissue damage.
Figure 8: Interaction diagram between clorobiocin and the binding pocket of human topoisomerase type II (1ZXM)
Figure 9: Positioning of clorobiocin in the binding pocket of human topoisomerase type II (1ZXM)
Figure 10: Interaction diagram between Troiacin and the binding pocket of the human topoisomerase type II (1ZXM)
Figure 11: Positioning of Troiacin in the binding pocket of the human topoisomerase type II (1ZXM)
Interestingly the top scoring poses of β-lactam clorobiocin revealed an entirely new interaction pattern as seen in the Figures 10 and 11. The addition of the β lactam group allowed a reverse integration of the compound in the binding pocket. The predicted reverse poses of Troiacin had a score on the same level as clorobiocin and the predicted free binding energy was within the same range as well. The new position has unforeseeable consequences for the efficacy of the modified compound potentially rendering it ineffective in the human cell.
Material and Methods
Preparation of proteins and ligands
Both the target protein and the ligands of interest must be processed prior to the docking procedure.Proteins are acquired and optimized via the protein preparation wizard. Structures can be directly imported from the protein database (PDB). During the preprocessing step atoms and bonds are identified and allocated. This step also includes the identification of structural issues. While missing residues can be added automatically, duplicates must be selected manually.  If multiple protein chains and ligands are present those of interest can be selected and all other entries are removed. The next step optimizes the hydrogen-bonding network by adding and refining hydrogens, assigning bond orders, and correcting the aromaticity if need be. Generally, all water molecules have to be removed but those within the binding pocket can contribute to the interaction score. Therefore, waters with more than three bonds to non-waters are usually kept. And lastly the structure is refined through restrained minimization of the heavy atoms to relieve strain while mostly retaining the input geometry.
Ligands are prepared using the Maestro utility LigPrep. Both clorobiocin and novobiocin were sourced from the DrugBank database. The modified clorobiocin compounds were generated via a 2D Sketcher.  3D structures with low energy corresponding to the input files were generated and expanded through variation of the ionization state, tautomers, stereochemistry and ring conformations. Furthermore the energetic penalty for respective state war calculated and saved for the Glide docking procedure (Freisner 2004).
Grid generation
Grids representing the binding pocket or area within the ligands are placed were generated using the grid generation protocol provided by Maestro. To facilitate the procedure only protein structures with identified binding sites containing ligands were used. Therefore, present ligands had to be identified as such to prevent interference with the docking process. The grid itself was manually centered around the ligand in the binding pocket and generated based on the atoms present within the selected grid area.
The docking process was refined further by setting constraints during the grid generation process. This was done by defining areas within the binding pocket in which certain ligand atoms or interactions e.g. H-bonds were required to occur. The interaction between gyrase B and both novobiocin and clorobiocin has been described in detail and the constraints were set accordingly. Hence only ligand poses fitting the criteria were considered throughout the  docking process.
Docking
The interaction between the compounds and target proteins were analyzed using the Ligand Docking utility Glide included in Maestro.
For the docking procedure a collection of ligand conformations was generated from each input ligand by systematic enumeration of the ligand torsions. Each conformation was examined during the docking process. Both the shape and the properties of the receptor were represented by different sets of fields on the respective grid. These fields provided progressively more accurate scorings of the ligand pose.
Promising ligand poses were located during the initial screening phase. The selected ligand poses then were refined in torsional space using the OPLS3 force field with a distance-dependent dielectric model. During post-docking minimization only the highest scoring poses were minimized within the field of the rigid protein and at full ligand flexibility (Freisner 2004).
The Glide Extra Precision (XP) docking procedure was selected to increase specificity. Glide XP docking extended the standard protocol by an anchor-and-grow procedure. For this ring structures of the ligands were identified and kept rigid and planar while flexible bonds are added optimized according to the scoring function to the segment.  Furthermore, the procedure utilized a more sophisticated scoring function with greater restrictions on the ligand-receptor shape complementarity to reduce the number of false positive poses (Freisner 2006).
Evaluation of the docking poses
For each of the ligand the ten highest scoring poses were selected for further analysis. Interactions with the residues and placement within the binding pocket were visually examined. Furthermore the binding free-energy of each ligand was estimated using the MM-GBSA utility provided by Maestro. The free-energy is composed of the conformational energy term  and a solvent free-term. MM-GBSA utilizes the OPLS3 force fields to determine the potential energy of the selected structure. Any non-polar contributions to the solvation term are assumed based on the solvent accessible Surface area. Polar contributions more specifically the electrostatic potential around the solute are computed using the generalized Born approximation of the Poisson-Boltzmann equation (Genheden 2015).
References
Anderle, C., Stieger, M., Burrell, M., Reinelt, S., Maxwell, A., Page, M., Heide, L. (2008). Biological activities of novel gyrase inhibitors of the aminocoumarin class. Antimicrob Agents Chemother, 52(6), 1982-1990. doi:10.1128/AAC.01235-07
Beisken, S., Meinl, T., Wiswedel, B., de Figueiredo, L. F., Berthold, M., Steinbeck, C. (2013). KNIME-CDK: Workflow-driven cheminformatics. BMC Bioinformatics, 14, 257. doi:10.1186/1471-2105-14-257
Friesner, R. A., Banks, J. L., Murphy, R. B., Halgren, T. A., Klicic, J. J., Mainz, D. T., . . . Shenkin, P. S. (2004). Glide: a new approach for rapid, accurate docking and scoring. 1. Method and assessment of docking accuracy. J Med Chem, 47(7), 1739-1749. doi:10.1021/jm0306430
Friesner, R. A., Murphy, R. B., Repasky, M. P., Frye, L. L., Greenwood, J. R., Halgren, T. A., . . . Mainz, D. T. (2006). Extra precision glide: docking and scoring incorporating a model of hydrophobic enclosure for protein-ligand complexes. J Med Chem, 49(21), 6177-6196. doi:10.1021/jm051256o
Genheden, S., Ryde, U. (2015). The MM/PBSA and MM/GBSA methods to estimate ligand-binding affinities. Expert Opin Drug Discov, 10(5), 449-461. doi:10.1517/17460441.2015.1032936
Holdgate, G. A., Tunnicliffe, A., Ward, W. H., Weston, S. A., Rosenbrock, G., Barth, P. T., . . . Timms, D. (1997). The entropic penalty of ordered water accounts for weaker binding of the antibiotic novobiocin to a resistant mutant of DNA gyrase: a thermodynamic and crystallographic study. Biochemistry, 36(32), 9663-9673. doi:10.1021/bi970294+
Kontoyianni, M. (2017). Docking and Virtual Screening in Drug Discovery. Methods Mol Biol, 1647, 255-266. doi:10.1007/978-1-4939-7201-2_18
Lafitte, D., Lamour, V., Tsvetkov, P. O., Makarov, A. A., Klich, M., Deprez, P., . . . Gilli, R. (2002). DNA gyrase interaction with coumarin-based inhibitors: the role of the hydroxybenzoate isopentenyl moiety and the 5'-methyl group of the noviose. Biochemistry, 41(23), 7217-7223.
Lionta, E., Spyrou, G., Vassilatis, D. K., Cournia, Z. (2014). Structure-based virtual screening for drug discovery: principles, applications and recent advances. Curr Top Med Chem, 14(16), 1923-1938.
Lipinski, C. A., Lombardo, F., Dominy, B. W., Feeney, P. J. (2001). Experimental and computational approaches to estimate solubility and permeability in drug discovery and development settings. Adv Drug Deliv Rev, 46(1-3), 3-26.
Mannhold, R., Poda, G. I., Ostermann, C., Tetko, I. V. (2009). Calculation of molecular lipophilicity: State-of-the-art and comparison of log P methods on more than 96,000 compounds. J Pharm Sci, 98(3), 861-893. doi:10.1002/jps.21494
Sadiq, A. A., Patel, M. R., Jacobson, B. A., Escobedo, M., Ellis, K., Oppegard, L. M., . . . Kratzke, R. A. (2010). Anti-proliferative effects of simocyclinone D8 (SD8), a novel catalytic inhibitor of topoisomerase II. Invest New Drugs, 28(1), 20-25. doi:10.1007/s10637-008-9209-1
Tetko, I. V., Gasteiger, J., Todeschini, R., Mauri, A., Livingstone, D., Ertl, P., . . . Prokopenko, V. V. (2005). Virtual computational chemistry laboratory--design and description. J Comput Aided Mol Des, 19(6), 453-463. doi:10.1007/s10822-005-8694-y
Tsai, F. T., Singh, O. M., Skarzynski, T., Wonacott, A. J., Weston, S., Tucker, A., . . . Wigley, D. B. (1997). The high-resolution crystal structure of a 24-kDa gyrase B fragment from E. coli complexed with one of the most potent coumarin inhibitors, clorobiocin. Proteins, 28(1), 41-52.
© iGEM Team Tuebingen 2017
