<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_Tuebingen_Results_Biochemistry skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><DIV id="top_title"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:Tuebingen/Results/Biochemistry</SPAN></H1></DIV><DIV id="HQ_page"><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><P>&lt;htm
<TITLE> iGEM T√ºbingen 2017</TITLE></P><DIV class="Hauptnavigation"><A href="https://2017.igem.org/Team:Tuebingen/Team" style="margin-right:1.5em;">Team</A><A href="https://2017.igem.org/Team:Tuebingen/Inspiration" style="margin-right:1.5em;">Inspiration</A><A href="https://2017.igem.org/Team:Tuebingen/Demonstrate" style="margin-right:1.5em;">Results</A><A href="https://2017.igem.org/Team:Tuebingen/HP" style="margin-right:1.5em;">Human Practice</A><A href="https://2017.igem.org/Team:Tuebingen/Lab" style="margin-right:1.5em;">Lab</A><A href="https://2017.igem.org/Team:Tuebingen/Attributions" style="margin-right:1.5em;">Attributions</A></DIV><DIV style="max-width:100%;"><DIV style="position: absolute; text-align:right;
                                 margin-bottom: 10%;
                                 margin-left: 60%;
                                 margin-top:-4em;
                                 font-size: 3em;color:white;font-weight:light;letter-spacing: 3px;"> biochemistry
                </DIV></DIV><NAV class="Unternavigation-Team"><A href="#Introduction">Introduction </A><A href="#Theoretical Background">Theoretical Background </A><A href="#Procedure">Procedure</A><A href="#Results">Results</A><A href="#Discussion">Discussion</A></NAV><DIV class="Grid"><DIV id="Fliesstext1"><H1 id="Biochemistry">Biochemistry</H1><H2 id="Introduction" class="anchor">Introduction</H2><H3><I>Streptomyces coelicolor </I></H3><P><I>S.coelicolor </I> is a bacterium that belongs to the genus of <I>Streptomyces </I>. Over 500 species of <I>Streptomyces </I> are known which are mainly found in soil and decaying vegetation where they grow in mycelial manner and produce spores. One characteristic is their earthy smell  which is caused by the production of a volatile metabolite, Geosmin. <I>Streptomyces </I> are aerobic, gram-positive and have a genome with high GC-content. These little bacteria are most famous for their bioactive secondary metabolites that can be antifungal, antiviral, antitumoral, anti-hypertensive, immunosuppressant, and antibiotic.</P><H3>Application in Biotechnology </H3><P>Since <I>Streptomyces </I> are able to produce complex chemical components, they can be used for production of useful substances such as antibiotics. Furthermore, <I>Streptomyces </I> are used for heterologous protein expression as they have a few advantages compared to <I> E.coli </I>. Some proteins refuse to fold properly in <I> E.coli </I>, while they do in <I>Streptomyces </I>. Another advantage is their good secretion system which simplifies purification. </P><H3>Production of antibiotics by <I>Streptomyces </I></H3><P>One class of antibiotics produced by <I>Streptomyces </I> are aminocoumarins which inhibit bacterial DNA gyrase enzymes by binding tightly to subunit B competing with ATP for the ATP binding site and thereby blocking supercoiling. The therapeutic index is quite small because aminocoumarins also bind to ATP binding sites of human topoisomerases type II. </P><P>All aminocoumarins have a similar structure of three rings (A to C) and are produced in a similar fashion. The prenylated ring A is the first part on which ring B and C are attached.
				</P><FIGURE><FIGCAPTION>Figure 1: Biosynthesis of Clorobiocin.</FIGCAPTION></FIGURE><P>
						Our goal was to render the aminocoumarins specific to Œ≤-Lactam antibiotics resistant bacteria such as the dangerous <I>Methicillin resistant Staphylococcus aureus</I> (MRSA) which leads to many deaths every year. The new antibiotic should only target Œ≤-Lactam resistant bacteria for which reason one side important for the protein ligand interaction was masked by a Œ≤-Lactam ring. The resistant bacterium will cleave the Œ≤-lactam ring, activate the antibiotic and die. Therefore, our warhead, a Œ≤-lactam, was installed at the prenyl tail of Ring A. The bioinformatic analysis proofed our theoretical assumption that the binding of the closed  Œ≤-Lactam ring to GyrB and human topoisomerase type II (<A href="https://2017.igem.org/Team:Tuebingen/Model" style="font-size:1em;margin-left:-0.5em;">Model</A>) is weaker than the binding of clorobiocin. Furthermore it showed that the cleaved Œ≤-Lactam ring binds stronger to the target than clorobiocin. The modification does not only decrease the binding strength to its targets in the uncleaved state, but also increases the binding strength to its target as soon as it is cleaved by a Œ≤-Lactamase. Our modified aminocoumarin will be less toxic for every organism that is not resistant to Œ≤-Lactamase, but more toxic for the ones with resistance.
				</P></DIV><DIV id="Fliesstext2"><H2 id="Theoretical Background" class="anchor"> Theoretical Background </H2><H3>Key features</H3><P>In order to produce our new antibiotic we needed a production strain that lacks the biosynthetic pathway for Ring A and an amide synthase which accepts our substrate. Furthermore,  we have to close the  Œ≤-Lactam-ring using a Œ≤-Lactam synthetase (BLS).</P><H3>Ring A</H3><P>The key enzyme for the production of Ring A of clorobiocin is CloQ a prenyltransferase (Heide et al. 2003). A cloQ defective <I>S.coelicolor</I> M512 strain (Clo/SA02) with a recombinant biosynthetic cluster was made available to us by the group of Prof. Heide at University of Tuebingen.
					</P><H3>Amid synthase</H3><P>As the structure of the amide synthase (CloL) from the clorobiocin biosynthetic pathway is not available, we decided to use amid synthases from the related pathways of novobiocin (NovL) and coumermycin A1 (CouL) and co-express them in our production strain to increase the chance of successful incorporation. As shwon by Anderle (2007) CloL is the most promiscuous of the three enzymes (Anderle et al. 2007). Introduction of genes in <I>Streptomyces </I> is possible by conjugation, which means one bacteria transfers DNA over a dedicated transportation complex to another bacteria.</P><H3>Installation of the Œ≤-Lactam ring / warhead</H3><P>Although, many possibilities for formation of a Œ≤-Lactam ring by chemical synthesis are available, we chose a path via enzymatical synthesis. This bears the advantage that the enzyme can be co-expressed in <I>Streptomyces </I>. The enzyme Œ≤-Lactam synthetase (Bachmann et al. 1998) seemed to be useful as it uses a Œ≤-amino acid to form the Œ≤-Lactam ring. The synthesis of the needed Œ≤-amino acid was easily possible.
						Based on available crystal structures of the BLS from <I>S. clavuligerus</I> (1mbz,1mc1, 1mb9,1m1z,1jgt), we decided to change two amino acids within the active site to increase the probability that the enzyme accepts our substrate (based on the structure with the trapped intermediate (1MBZ)).
				</P><H2 id="Procedure" class="anchor">Procedure</H2><H3>Amid synthases</H3><P>For conjugation of amid synthase genes (CloL, CouL, SimL), the genes were cloned into a special plasmid pUWL-Apra-oriT which was supplied by the laboratory of Dr. Gust. The pUWL vector has origins of replication for propagation in <I> E.coli </I> and <I> S.coelicolor </I> and additional an origin of transfer (oriT), the startpoint of replication for conjugation and and a constitutive Streptomyces promoter (ermE up). In contrast to <I> E.coli </I><I>Streptomyces </I> do not need a RBS.<I>Streptomyces </I> have a strong methylation based restriction system, hence, all DNA passed a methylation (dam) defective <I> E.coli </I> strain like ET12567. This strain contains pUZ8002, which supplies the transfer function to oriT containing plasmids by expressing Tra which allows Tra defective vectors in trans mobilisation. pUZ8002 will not be transferred (Paget et al. 1999 and <A href="https://openwetware.org/wiki/Streptomyces:Protocols/Conjugation)" style="font-size:1em;margin-left:-0.5em;">OpenWetware</A>).
						
						Germinated <I>Streptomyces </I> recipient spores and ET12567 donor cells were plated together on Mannitol-Soya Flour (MS)-agar plates with 10 mM MgCl2. On the next day, the plate was overlaid with nalidixic acid (bacteriostatic to <I>E.coli</I>) and apramycin (plasmid selection). After three days, the first exconjugants appeared and a replica were prepared. Three colonies were picked and and diluted to gain single colonies. This step was repeated. After colonies appeared, one was picked and spread on a new plate. As soon as the plate was confluent grown two more plates were spread with each clon. When all three plates reached confluency, spores were harvested.</P><H3>Installation of the Œ≤-Lactam ring / warhead</H3><P>Both Œ≤-Lactam synthetases (BLS, wildtype (WT) and double mutant (DM)) were cloned into pSB1C3 (Wt: BBa_K2372002, DM: BBa_K2372003) and pSB1C3-pRha-RBS (Wt: BBa_K2372009, DM in pSB1K3-pRha-RBS: not submitted). The proteins were over-expressed in <I> E.coli </I> and purified using Nickel immobilized metal ion chromatography (Ni-IMAC). The synthesized substrate was supplied together with Mg<SUP>2+</SUP> and ATP to the reaction. The reaction is followed by using LC-MS, expecting a mass shift of 18.
					</P><H3>Feeding experiment</H3><P>A preculture in tryptic-soy-broth (TSB, 50mL) with antibiotics was prepared with spores. After 2-3 days the preculture was used for inoculation of chemically-defined medium (CDM).The preculture was checked for contaminants under the microscope, spun down and resuspended in CDM. The experiments were done in a 24 well bioreactor where each well contained 3 ml of <I>Streptomyces</I> culture.  Different substrate concentrations and <I>Streptomyces </I> were put in the reactor and left at 30¬∞C, 200 rpm for 7 days.
						1 mL per well was spun down at 17000g and the supernatant was cleaned by centrifugation. This is the aqueous phase sample. A second 1 mL per well was extracted using two times 1 mL ethyl acetate, which was dried down and resuspended in Methanol. This is the organic phase sample. The product formation was analyzed and confirmed by LC-MS.
				</P><FIGURE><FIGCAPTION>Figure 2: Workflow of the biochemistry part.</FIGCAPTION></FIGURE><H2 id="Results" class="anchor">Results</H2><H3> Feeding </H3><P>					
						Aqueous and organic phase samples were analyzed by LC-MS. The MS chromatograms were searched for expected masses indicated at the left top corner. The black trace indicates the mean ion count (MIC) in positive, the pink trace in negative mode.
					
						In order to examine if the correct culture conditions were used the clorobiocin producing strain Clo-BG1 and the clorobiocin standard were analyzed.
					<B>Clorobiocin standard (2.5¬µg):</B></P><FIGURE><FIGCAPTION>Figure 3: Chromatogram at 280 nm of clorobiocin standard.</FIGCAPTION></FIGURE><FIGURE><FIGCAPTION>Figure 4: UV chromatogram at 260 nm of clorobiocin standard.</FIGCAPTION></FIGURE><FIGURE><FIGCAPTION>Figure 5: MS chromatogram of clorobiocin standard.</FIGCAPTION></FIGURE><FIGURE><FIGCAPTION>Figure 6: MS spectrum at minute 13 (positive mode)</FIGCAPTION></FIGURE><FIGURE><FIGCAPTION>Figure 7: MS spectrum at minute 17 (positive mode)</FIGCAPTION></FIGURE><P>
					The most abundant ion is highlighted in cyan with a m/z of 391.2. This ion can be found in the organic phase of the non-fed Clo-BG1 strain:
                </P><FIGURE><FIGCAPTION>Figure 8: UV chromatogram of CloBG1 at 280nm.</FIGCAPTION></FIGURE><FIGURE><FIGCAPTION>Figure 9: UV chromatogram of CloBG1 at 280nm.</FIGCAPTION></FIGURE><FIGURE><FIGCAPTION>Figure 10: MS chromatogram of CloBG1.</FIGCAPTION></FIGURE><FIGURE><FIGCAPTION>Figure 11: MS spectrum of CloBG1 at 15min.</FIGCAPTION></FIGURE><P>
					As expected we did not find m/z signal of 391.2  in the CloQ-defective strain (organic phase, non-fed):
				</P><FIGURE><FIGCAPTION>Figure 12: UV chromatogram of SA02 at 280nm.</FIGCAPTION></FIGURE><FIGURE><FIGCAPTION>Figure 13: UV chromatogram of SA02 at 260nm.</FIGCAPTION></FIGURE><FIGURE><FIGCAPTION>Figure 14: MS chromatogram of SA02.</FIGCAPTION></FIGURE><FIGURE><FIGCAPTION>Figure 15: positive mode MS spectrum of SA02 at 15min.</FIGCAPTION></FIGURE><FIGURE><FIGCAPTION>Figure 16: negative mode MS spectrum of SA02 at 15min.</FIGCAPTION></FIGURE><P>
					The results indicate that the culture conditions were good. More ions fly with higher abundance in positive mode. Ions can be explained by fragmentation in the ionization process:
				</P><FIGURE><FIGCAPTION>Figure 17: Clorobiocin ions.</FIGCAPTION></FIGURE><P>	We find this ion (m/z = 391,1) again in the organic phase of fed samples (red trace):
				</P><FIGURE><FIGCAPTION>Figure 18: MS chromatogram (Cou2_320A)</FIGCAPTION></FIGURE><FIGURE><FIGCAPTION>Figure 19: MS chromatogram (Cou2_320A)</FIGCAPTION></FIGURE><P>
					But not in non-fed samples:
				</P><FIGURE><FIGCAPTION>Figure 20: MS chromatogram (Cou2_0A)</FIGCAPTION></FIGURE><P>
					The four peaks (minutes 8.0; 10.5; 11.25 and 12.0) have to be major metabolites [M+H]<SUP>+</SUP> ions, because the respective [M-H]<SUP>-</SUP> ions are also found. In the fed sample the first peak is much broader corresponding to the substrate and its methyl-esters:
				</P><FIGCAPTION>Figure 21: substrate and methylesters)</FIGCAPTION><P>
					The substrate can be detected much easier in aqueous phase samples:
				</P><FIGURE><FIGCAPTION>Figure 22: Chromatogram at 280nm of the aqueous phase of fed sample.</FIGCAPTION></FIGURE><FIGURE><FIGCAPTION>Figure 23: MS Chromatogram of the aqueous phase of fed sample</FIGCAPTION></FIGURE><FIGURE><FIGCAPTION>Figure 24: MS spectrum of the aqueous phase of fed sample at minute 5.5.</FIGCAPTION></FIGURE><P>
					[M+H]<SUP>+</SUP>-ion abundance was above detection limit, indicated by the overlaid red trace. The [methyl-M+H]<SUP>+</SUP> ion is shown as the green trace in the MS chromatogram. 
					
					We expect our product at m/z = 743.21 ([M+H]<SUP>+</SUP>) in the organic phase. Looking for it we found the corresponding mass:
				</P><FIGURE><FIGCAPTION>Figure 25: Structure of our new antibiotic with the open lactam ring.</FIGCAPTION></FIGURE><FIGURE><FIGCAPTION>Figure 26: MS chromatogram of organic phase of the fed, CouL conjugation.</FIGCAPTION></FIGURE><FIGURE><FIGCAPTION>Figure 27: MS chromatogram of organic phase of the fed, CouL conjugation zoomed in.</FIGCAPTION></FIGURE><FIGURE><FIGCAPTION>Figure 28: MS spectrum of organic phase of the fed, CouL Conjugationat minute 10.88.</FIGCAPTION></FIGURE><H3> Feeding </H3><P>
					Œí-Lactam synthetase (Wt: BBa_K2372009 and double mutant: BBa_K2372003 as pRha-RBS composite in pSB1K3) was expressed by induction with 1%(w/v) L-rhamnose and the 56 kDa protein was purified by Ni-IMAC. Buffer was exchanged using a centrifugal filter unit with a nominal molecular weight cut-off of 10 kDa.
				</P><FIGURE><FIGCAPTION>Figure 29: SDS-PAGE of BLS-WT purification. Supernatant (S), Flow-through (FT), Wash (W), fraction 3-13.</FIGCAPTION></FIGURE><FIGURE><FIGCAPTION>Figure 30: SDS-PAGE of BLS-DM purification. Supernatant (S), Flow-through (FT), Wash (W), fraction 3-13.</FIGCAPTION></FIGURE><P>
					0.73mg of protein were used for the enzyme reaction. Protein was removed by centrifugation in centrifugal filter units with a nominal molecular weight cut off of 10 kDa.
				</P><FIGURE><FIGCAPTION>Figure 31: BLS-WT chromatogram (260nm).</FIGCAPTION></FIGURE><FIGURE><FIGCAPTION>Figure 32: BLS-DM chromatogram (260nm).</FIGCAPTION></FIGURE><FIGURE><FIGCAPTION>Figure 33: BLS-DM chromatogram (260nm).</FIGCAPTION></FIGURE><P>
					The peaks at 1.6 and 2.0 min probably correspond to ATP. The peak at 5.6 min corresponds to the educt:
				</P><FIGURE><FIGCAPTION>Figure 34: Positive mode MS spectrum [M+H]<SUP>+</SUP>.</FIGCAPTION></FIGURE><FIGURE><FIGCAPTION>Figure 35: Negative mode MS spectrum [M-H]<SUP>-</SUP>.</FIGCAPTION></FIGURE><P>
					If the ùõÉ-Lactam ring is closed between the carboxylic acid and the secondary amin we would expect a mass loss of 18 corresponding to H<SUB>2</SUB>O:
				</P><FIGURE><FIGCAPTION>Figure 36: Ring closure reaction.</FIGCAPTION></FIGURE><P>
					The MS chromatogram of BLS-WT shows the expected mass at different retention times.
				</P><FIGURE><FIGCAPTION>Figure 37: BLS-WT MS chromatogram zoomed in.</FIGCAPTION></FIGURE><FIGURE><FIGCAPTION>Figure 38: BLS-DM MS chromatogram zoomed in.</FIGCAPTION></FIGURE><FIGURE><FIGCAPTION>Figure 39: MS chromatogram of the control samples zoomed in.</FIGCAPTION></FIGURE><P>
					In blue: [M+H]<SUP>+</SUP> = 236,08 and in brown: [M-H]<SUP>-</SUP> = 234.08.
					
					Unfortunately none of them is unique for the enzymatic reaction and the abundance is around the abundance of the ions of the background. A possible candidate ion might be eluting around minute 10: 
				</P><FIGURE><FIGCAPTION>Figure 40: Negative mode MS spectrum of BLS-WT at 9.78 min.</FIGCAPTION></FIGURE><FIGURE><FIGCAPTION>Figure 41: Negative mode MS spectrum of BLS-DM at 10.38  min.</FIGCAPTION></FIGURE><P>
					Negative mode MS spectrum of BLS-DM at 10.38 min
					
					Retention times are in general not perfectly reproducible. The control does not show a peak of this size around this retention time (min 9.5-10.5), but the [M+H]<SUP>+</SUP> ion is not present.
				</P><FIGURE><FIGCAPTION>Figure 42: MS chromatogram of the control samples zoomed in.</FIGCAPTION></FIGURE><H2 id="Discussion" class="anchor"> Discussion </H2><P>
						It seems like our synthesis product is accepted as substrate by the amidase CloL or CouL even tough with a very low yield. 
						The SDS-PAGE does not indicate a high yield of recombinant protein and shows a yet unexplained signal around 70kDa. The low quality of the run makes it difficult to analyze the gels. A repetition of the gels did not result in a better separation. We were not able to determine whether we purified the BLS and due to time constraints purification could not be repeated. Additionally the fractions were frozen overnight which might have resulted in the inactivation of the enzyme. 
						The MS spectra do not indicate a significant product formation, which might be due to the absence of active enzyme.
					</P><H2 id="Ref"> References </H2><P style="font-size:60%;">
				


						Anderle, C., Hennig, S., Kammerer, B., Li, S. M., Wessjohann, L., Gust, B., &amp; Heide, L. (2007). Improved mutasynthetic approaches for the production of modified aminocoumarin antibiotics. Chem Biol, 14(8), 955-967. doi:10.1016/j.chembiol.2007.07.014 
						Bachmann, B. O., Li, R., &amp; Townsend, C. A. (1998). beta-Lactam synthetase: a new biosynthetic enzyme. Proc Natl Acad Sci U S A, 95(16), 9082-9086.
						Paget, M. S., Chamberlin, L., Atrih, A., Foster, S. J., &amp; Buttner, M. J. (1999). Evidence that the extracytoplasmic function sigma factor sigmaE is required for normal cell wall structure in Streptomyces coelicolor A3(2). J Bacteriol, 181(1), 204-211. 
						Pojer F1, Wemakor E, Kammerer B, Chen H, Walsh CT, Li SM, Heide L. (2003). CloQ, a prenyltransferase involved in clorobiocin biosynthesis. Proc Natl Acad Sci U S A. 2003 Mar 4;100(5):2316-21.
                  </P></DIV></DIV><DIV id="ImpressumSponsoren"><DIV><P>¬© iGEM Team Tuebingen 2017</P></DIV></DIV></DIV></DIV></DIV></DIV></DIV></BODY></HTML>