<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_Tuebingen_Inspiration skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><DIV id="top_title"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:Tuebingen/Inspiration</SPAN></H1></DIV><DIV id="HQ_page"><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><P><TITLE> iGEM Tübingen 2017</TITLE></P><DIV class="Hauptnavigation"><A href="https://2017.igem.org/Team:Tuebingen/Team" style="margin-right:1.5em;">Team</A><A href="https://2017.igem.org/Team:Tuebingen/Inspiration" style="margin-right:1.5em;">Inspiration</A><A href="https://2017.igem.org/Team:Tuebingen/Demonstrate" style="margin-right:1.5em;">Results</A><A href="https://2017.igem.org/Team:Tuebingen/HP" style="margin-right:1.5em;">Human Practice</A><A href="https://2017.igem.org/Team:Tuebingen/Lab" style="margin-right:1.5em;">Lab</A><A href="https://2017.igem.org/Team:Tuebingen/Attributions" style="margin-right:1.5em;">Attributions</A></DIV><DIV style="max-width:100%;"><DIV style="position: absolute; text-align:right;
                                 margin-bottom: 10%;
                                 margin-left: 60%;
                                 margin-top:-4em;
                                 font-size: 3em;color:white;font-weight:light;letter-spacing: 3px;"> inspiration
                </DIV></DIV><NAV class="Unternavigation-Team"><A href="#Antibiotic Research at stake">Antibiotic Research at stake </A><A href="#Introducing the aminocoumarins">Introducing the aminocoumarins </A><A href="#Engineering the warhead">Engineering the warhead </A><A href="#Troiacin - Structural advancement">Troiacin - Structural advancement </A></NAV><DIV class="Grid"><DIV id="Fliesstext1"><H1 id="Inspiration">Inspiration</H1><H2 id="Antibiotic Research at stake" class="anchor">Nothing in pipeline? - 
				The problem of modern antibiotic research</H2><P>Worldwide globalisation and a rising world population creates a new problem for our society: 
				Pathogens acquire mutations and different resistance factors leaving us with less and less 
				effective antimicrobials. The World Health Organisation (WHO) considers the most problematic 
				organisms to be multi-resistant Mycobacterium tuberculosis, Acinetobacter baumannii, 
				Pseudomonas aeruginosa, Staphylococcus aureus, and many others (Geneva: World Health Organization 
				2017).</P><FIGURE><FIGCAPTION> Figure 1: Increase in antibiotic resistances in methicillin resistant 
				s.aureus, vancomycin resistant enterococci and fluoroquinolone resistant p.aeruginosa in the 
				last 30 years (source: Infectious disease society of America 2004) </FIGCAPTION></FIGURE><P>Besides fighting antibiotic misuse by raising awareness and health education to decrease 
				the number of new resistant bacteria, there is a necessity for new antibacterial substances 
				and therapies. This lies within the responsibility of universities and companies. In contrast,
				every year, the FDA approves less and less new antibacterial compounds. This is due to the fact,
				that research in the field of new antibiotics  for pharmaceutical companies is not attractive:
				New substances are usually only used as reserve antibiotics for patients suffering from infections
				with highly resistant pathogens and therefore provides in its patented period only a low budget 
				income.</P><FIGURE><FIGCAPTION> Figure 2: number of antibacterial new drug-applications by the FDA 
			  (source: CDC antibiotic/antimicrobial resistance report)
               </FIGCAPTION></FIGURE><P> Most new antibiotic substances are found in new bacterial strains, while (bio-)chemically
				modified compounds are rather rare. At the same time, it gets harder to identify new potent 
				candidates. We need new strategies to produce effective compounds that are specifically engineered
				to fight resistant pathogens and new methods to achieve these modifications. This is where antibiotic
				classes, that provide potency in theory, but fail in compliance and practical application, come into
				play. </P><H2 id="Introducing the aminocoumarins" class="anchor"> Introducing the aminocoumarins: potent 
				antibiotics with little relevance in clinical therapy </H2><P> Aminocoumarins are one of many substance classes with almost no use in the antibacterial therapy.
				So far, three aminocoumarins are known: Novobiocin, Clorobiocin and Coumermycin A1. (Heide 2014) 
				Furthermore, the structurally different Simocyclinone D8 has been described as a fourth aminocoumarin
				(Schimana et al. 2000). </P><FIGURE><FIGCAPTION> Figure 3: Basic structure of aminocoumarins (based on Heide 2014)
               </FIGCAPTION></FIGURE><P> All aminocoumarins produced by different streptomyces species have an aminocoumarin 
				(Figure 3 green) and a deoxyribose (Figure 3 blue) moiety in common. Aminocoumarins irreversibly bind 
				to and inhibit the B subunit of the bacterial gyrase and therefore prevent the cell from successful 
				mitosis (Lawson and Stevenson 2012). The human topoisomerase II is a known off-target, which can
				cause toxicity when highly dosed. Resistances are mediated by unspecific multidrug exporters (MDR)
				or mutations in the gyrase binding pockets. The latter are only found in aminocoumarin producing
				Streptomyces strains or bacteria that were forced to mutate by selection under low dose aminocoumarin
				application (Fujimoto-Nakamura et al. 2005).
                   
                    Disadvantages that prevent aminocoumarins from more frequent clinical use include path of
					administration (i.v. only), a narrow efficacy spectrum (no gram-negative bacteria are affected),
					unfavorable pharmacokinetics caused by a low water solubility or irritations at the injection 
					spot caused by toxicity due to a slow drug distribution in the blood (Grayson et al 2010).
					So far, Novobiocin is the only aminocoumarin approved by the FDA for antibiotic therapy.
					Lately, studies have found beneficial results for cancer therapies with topoisomerase II
					inhibitors in combination with Novobiocin.</P><H2 id="Engineering the warhead" class="anchor">Engineering the warhead: the β-lactam motive as an
				activator in resistant bacteria</H2><P> Our way to a new aminocoumarin antibiotic with an activation mechanism started with the question,
				which of the known aminocoumarins our substance should be based on. We decided to work with
				clorobiocin, as it is not used as antibiotic in its current form and provides perfect circumstances
				for different chemical modifications. 
				A basic aminocoumarin provides three sites open to (bio-)chemical modifications without loss
				of activity: While R1 allows little modification, as this part of the molecule is synthesised
				first (further involved enzymes wouldn't recognize their new substrate) and R2 can only be
				substituted with small chemical groups to retain the antibacterial properties, R3 provides
				the biggest variety in possible substrates. For an effective gyrase inhibition a nonpolar,
				hydrophobic moiety at R3 is preferred. Novobiocin and clorobiocin carry an isoprenyl moiety
				at this position. Therefore, we chose a modification at R3, that lowers the hydrophobic affinity
				in inactive state but keeps its linear, hydrophobic character after activation. R3 can be
				modified by knocking out genes in the antibiotic gene cluster that are involved in the synthesis
				of the para-hydroxy-benzoic acid derivative (Figure 3 red). Afterwards, chemically produced
				substances are fed to the producing Streptomyces which can then be used by the corresponding
				amide synthase to connect it with the residual molecule. 
				Many multiresistant pathogens acquired different resistances for many antibiotic classes.
				Frequently these pathogens integrated a kind of β-lactamase into their genome because of the high
				frequency of β-lactam motives in many different antibiotic classes. That's why we decided to use
				a β-lactam ring as warhead for our compound: Without cleavage by a β-lactamase, the aminocoumarin
				provides a slightly hydrophilic and bigger moiety then an isoprenyl at R3, not allowing an
				effective binding in the target's gyrase binding pocket, while after cleavage the opened β-lactam
				ring mimics a hydrophobic N-isobutyl moiety leading to an effective inhibition (Figure 5). These
				considerations lead us to our new aminocoumarin Troiacin. </P><FIGURE><FIGCAPTION> Figure 4: Chemical structure of Troiacin
               </FIGCAPTION></FIGURE><H2 id="Troiacin - Structural advancement" class="anchor"> Troiacin: Structural advancement leads
			to confined efficacy </H2><FIGURE><FIGCAPTION> Figure 5: Troiacin activation and antibacterial mechanism in β-lactam resistant pathogens			   
               </FIGCAPTION></FIGURE><P> Troiacin integrates many of our considerations listed above with our new semi-biosynthetic
			  synthesis. Troiacin has a β-lactam ring structure similar to the carbapenem β-lactam allowing an
			  effective cleavage by carbapenem resistant pathogens. While the expected pathogen spectrum remains
			  the same (mainly gram-positive bacteria including S. aureus, Actinobacteria, and M. tuberculosis),
			  we propose that Troiacin has a decreased off-target effect by decreased binding affinity to the human
			  topoisomerase II. Furthermore, it is better soluble in water then clorobiocin leading to a better
			  properties in pharmacokinetics, lower toxicity reactions at the infusion position and a better
			  distribution in the patient's body. Besides, the need of a lower antibiotic dose reduces the risks of
			  off-target effects in the human topoisomerase II even further. Therefore, Troiacin can be used for
			  a positive antibiotic selection in carbapenem resistant bacteria. </P><H2 id="Ref"> References </H2><P style="font-size:60%;"> Fujimoto-Nakamura, M., Ito, H., Oyamada, Y., Nishino, T., &amp; Yamagishi, J. (2005). Accumulation of mutations in both gyrB and parE genes is associated with high-level resistance to novobiocin in Staphylococcus aureus. Antimicrob Agents Chemother, 49(9), 3810-3815. doi:10.1128/AAC.49.9.3810-3815.2005 

Heide, L. (2014). New aminocoumarin antibiotics as gyrase inhibitors. Int J Med Microbiol, 304(1), 31-36. doi:10.1016/j.ijmm.2013.08.013 

Lawson, D. M., &amp; Stevenson, C. E. (2012). Structural and functional dissection of aminocoumarin antibiotic biosynthesis: a review. J Struct Funct Genomics, 13(2), 125-133. doi:10.1007/s10969-012-9138-2 

M Lindsay Grayson, S. M. C., James S McCarthy, John Mills, Johan W Mouton, S Ragnar Norrby, David L Paterson, Michael A Pfaller. (2010). Kucers' The Use of Antibiotics Sixth Edition: A Clinical Review of Antibacterial, Antifungal and Antiviral Drugs. 

Organization, G. W. H. (2017). Prioritization of pathogens to guide discovery, reserach and development of new antibiotics for drug-resistant bacterial infections, including tuberculosis. 

Schimana, J., Fiedler, H. P., Groth, I., Sussmuth, R., Beil, W., Walker, M., &amp; Zeeck, A. (2000). Simocyclinones, novel cytostatic angucyclinone antibiotics produced by Streptomyces antibioticus Tu 6040. I. Taxonomy, fermentation, isolation and biological activities. J Antibiot (Tokyo), 53(8), 779-787.
</P></DIV></DIV><DIV id="ImpressumSponsoren"><DIV><P>© iGEM Team Tuebingen 2017</P></DIV></DIV></DIV></DIV></DIV></DIV></DIV></BODY></HTML>