<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_Tuebingen_Attributions skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><DIV id="top_title"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:Tuebingen/Attributions</SPAN></H1></DIV><DIV id="HQ_page"><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><P><TITLE> iGEM Tübingen 2017</TITLE></P><DIV class="Hauptnavigation"><A href="https://2017.igem.org/Team:Tuebingen/Team" style="margin-right:1.5em;">Team</A><A href="https://2017.igem.org/Team:Tuebingen/Inspiration" style="margin-right:1.5em;">Inspiration</A><A href="https://2017.igem.org/Team:Tuebingen/Demonstrate" style="margin-right:1.5em;">Results</A><A href="https://2017.igem.org/Team:Tuebingen/HP" style="margin-right:1.5em;">Human Practice</A><A href="https://2017.igem.org/Team:Tuebingen/Lab" style="margin-right:1.5em;">Lab</A><A href="https://2017.igem.org/Team:Tuebingen/Attributions" style="margin-right:1.5em;">Attributions</A></DIV><DIV style="max-width:100%;"><DIV style="position: absolute; text-align:right;
                                 margin-bottom: 10%;
                                 margin-left: 60%;
                                 margin-top:-4em;
                                 font-size: 3em;color:white;font-weight:light;letter-spacing: 3px;"> attributions
                </DIV></DIV><NAV class="Unternavigation-Team"><A href="#Attributions">Attributions</A><A href="#Acknowledgement">Acknowledgment</A><A href="#ReferencesLink">References</A></NAV><DIV class="Grid"><DIV id="Fliesstext1"><H2 id="Attributions" class="anchor"> Attributions</H2><P> This year’s team was led by PD Dr. Bertolt Gust and PD Dr. Elisabeth Fuss. They provided the needed laboratory support and managed       main communication within the university. The student team was coordinated and organized by Brian Weidensee and Nikolas Layer as         student leaders; the original project idea was designed by PD Dr. Bertolt Gust and Nikolas Layer.

                    All student members participated in the project development with the special contribution in gene block and part design by Lukas Fuhs and Nikolas Layer. For laboratory execution, we divided the project into three parts: The chemical synthesis was done by Alexandra Haake, Marcel Conrady, Madeleine Heep, and Michael Krummhaar. Lisa Dussling contributed the in-silico analysis and the bioinformatics model while Marcel Conrady, Lukas Fuhs, Madeleine Heep, Mirjam Gneiting, Milena Krach, and Michael Krummhaar were involved in cloning, biochemical analysis, and compound production. 
                    Milena Krach, Brian Weidensee, and our intern Fan Zhang were responsible for procedure and data submission of the iGEM 2017 interlab study. 
                    For our this year’s YouTube collaboration project we want to thank all participating iGEM teams and Hannah Brasse for the realization and organization. Hannah Brasse, Lukas Fuhs, Mirjam Gneiting, Madeleine Heep and Milena Krach produced our own video for the channel. All educational projects embedded in our Human Practice project were mainly organized by Vic-Fabienne Schumann and Brian Weidensee with the engagement of the whole student team with special thanks to Marcel Conrady, Madeleine Heep, Milena Krach and Brian Weidensee for supervision of our internship student Fan Zhang. 
                    Our wiki’s structure was mainly designed by Mirjam Gneiting and Alexander Recktenwald; a working code was created and implemented by Alexander Recktenwald. 

                    All wiki content was created and uploaded by Hannah Brasse, Marcel Conrady, Lisa Dussling, Mirjam Gneiting, Alexandra Haake, Madeleine Heep, Milena Krach, Michael Krummhaar, Nikolas Layer, Vic-Fabienne Schumann, and Brian Weidensee.
                    Without a lot of fundraising, none of our work would have been possible: Hannah Brasse, Alexandra Haake, Madeleine Heep, Milena Krach, Michael Krummhaar, Nikolas Layer, Vic-Fabienne Schumann and Brian Weidensee contributed to this area. 
                    We want to especially thank Dr. H. Kalbacher for MALDI-TOF Analysis and crystallization experiments of our chemical compounds. </P><H2 id="Acknowledgement" class="anchor"> Acknowledgement</H2><P> AG Prof. Dr. O. Kohlbacher - Hardware and in silico support 
                    Prof. Dr. L.Heide - Clorobiocin Cluster and bacterial strains 
                    AG Prof. Dr. D.Schwarzer - chemical synthesis laboratory and support 
                    AG Prof. Dr. S. Grond - Laboratory material support 
                    AG Pr. Dr. Schulze-Osthoff - Laboratory material support 
                    AG Pr. Dr. A. Weber - Laboratory material support for interlab study 
                    Jakob Wendt - Wiki coding support </P><H2 id="ReferencesLink" class="anchor"> References </H2><DIV id="References"><P>
                    Anderle, C., Hennig, S., Kammerer, B., Li, S. M., Wessjohann, L., Gust, B., &amp; Heide, L. (2007). Improved mutasynthetic approaches for the production of modified aminocoumarin antibiotics. Chem Biol, 14(8), 955-967. doi:10.1016/j.chembiol.2007.07.014 

Anderle, C., Stieger, M., Burrell, M., Reinelt, S., Maxwell, A., Page, M., &amp; Heide, L. (2008). Biological activities of novel gyrase inhibitors of the aminocoumarin class. Antimicrob Agents Chemother, 52(6), 1982-1990. doi:10.1128/AAC.01235-07

Bachmann, B. O., Li, R., &amp; Townsend, C. A. (1998). beta-Lactam synthetase: a new biosynthetic enzyme. Proc Natl Acad Sci U S A, 95(16), 9082-9086.

Beisken, S., Meinl, T., Wiswedel, B., de Figueiredo, L. F., Berthold, M., &amp; Steinbeck, C. (2013). KNIME-CDK: Workflow-driven cheminformatics. BMC Bioinformatics, 14, 257. doi:10.1186/1471-2105-14-257

Chen, Y., Wendt-Pienkowski, E., Ju, J., Lin, S., Rajski, S. R., &amp; Shen, B. (2010). Characterization of FdmV as an amide synthetase for fredericamycin A biosynthesis in Streptomyces griseus ATCC 43944. J Biol Chem, 285(50), 38853-38860. doi:10.1074/jbc.M110.147744

Fasching, C. E., Tenover, F. C., Slama, T. G., Fisher, L. M., Sreedharan, S., Oram, M., . . . Peterson, L. R. (1991). gyrA mutations in ciprofloxacin-resistant, methicillin-resistant Staphylococcus aureus from Indiana, Minnesota, and Tennessee. J Infect Dis, 164(5), 976-979.

Friesner, R. A., Banks, J. L., Murphy, R. B., Halgren, T. A., Klicic, J. J., Mainz, D. T., . . . Shenkin, P. S. (2004). Glide: a new approach for rapid, accurate docking and scoring. 1. Method and assessment of docking accuracy. J Med Chem, 47(7), 1739-1749. doi:10.1021/jm0306430

Friesner, R. A., Murphy, R. B., Repasky, M. P., Frye, L. L., Greenwood, J. R., Halgren, T. A., . . . Mainz, D. T. (2006). Extra precision glide: docking and scoring incorporating a model of hydrophobic enclosure for protein-ligand complexes. J Med Chem, 49(21), 6177-6196. doi:10.1021/jm051256o

Fu, X. W., Pu, W. C., Zhang, G. L., &amp; Wang, C. (2015). Synthesis of salicylaldehydes from phenols via copper-mediated duff reaction. Research on Chemical Intermediates, 41(11), 8147-8158.

Fujimoto-Nakamura, M., Ito, H., Oyamada, Y., Nishino, T., &amp; Yamagishi, J. (2005). Accumulation of mutations in both gyrB and parE genes is associated with high-level resistance to novobiocin in Staphylococcus aureus. Antimicrob Agents Chemother, 49(9), 3810-3815. doi:10.1128/AAC.49.9.3810-3815.2005

Genheden, S., &amp; Ryde, U. (2015). The MM/PBSA and MM/GBSA methods to estimate ligand-binding affinities. Expert Opin Drug Discov, 10(5), 449-461. doi:10.1517/17460441.2015.1032936
                        
Heide, L. (2014). New aminocoumarin antibiotics as gyrase inhibitors. Int J Med Microbiol, 304(1), 31-36. doi:10.1016/j.ijmm.2013.08.013

Holdgate, G. A., Tunnicliffe, A., Ward, W. H., Weston, S. A., Rosenbrock, G., Barth, P. T., . . . Timms, D. (1997). The entropic penalty of ordered water accounts for weaker binding of the antibiotic novobiocin to a resistant mutant of DNA gyrase: a thermodynamic and crystallographic study. Biochemistry, 36(32), 9663-9673. doi:10.1021/bi970294+

Kontoyianni, M. (2017). Docking and Virtual Screening in Drug Discovery. Methods Mol Biol, 1647, 255-266. doi:10.1007/978-1-4939-7201-2_18

Lafitte, D., Lamour, V., Tsvetkov, P. O., Makarov, A. A., Klich, M., Deprez, P., . . . Gilli, R. (2002). DNA gyrase interaction with coumarin-based inhibitors: the role of the hydroxybenzoate isopentenyl moiety and the 5'-methyl group of the noviose. Biochemistry, 41(23), 7217-7223.

Lawson, D. M., &amp; Stevenson, C. E. (2012). Structural and functional dissection of aminocoumarin antibiotic biosynthesis: a review. J Struct Funct Genomics, 13(2), 125-133. doi:10.1007/s10969-012-9138-2

Lionta, E., Spyrou, G., Vassilatis, D. K., &amp; Cournia, Z. (2014). Structure-based virtual screening for drug discovery: principles, applications and recent advances. Curr Top Med Chem, 14(16), 1923-1938.
                        
Lipinski, C. A., Lombardo, F., Dominy, B. W., &amp; Feeney, P. J. (2001). Experimental and computational approaches to estimate solubility and permeability in drug discovery and development settings. Adv Drug Deliv Rev, 46(1-3), 3-26.

Luft, T., Li, S. M., Scheible, H., Kammerer, B., &amp; Heide, L. (2005). Overexpression, purification and characterization of SimL, an amide synthetase involved in simocyclinone biosynthesis. Arch Microbiol, 183(4), 277-285. doi:10.1007/s00203-005-0770-0

M Lindsay Grayson, S. M. C., James S McCarthy, John Mills, Johan W Mouton, S Ragnar Norrby, David L Paterson, Michael A Pfaller. (2010). Kucers' The Use of Antibiotics Sixth Edition: A Clinical Review of Antibacterial, Antifungal and Antiviral Drugs.

Mannhold, R., Poda, G. I., Ostermann, C., &amp; Tetko, I. V. (2009). Calculation of molecular lipophilicity: State-of-the-art and comparison of log P methods on more than 96,000 compounds. J Pharm Sci, 98(3), 861-893. doi:10.1002/jps.21494

Miller, M. T., Bachmann, B. O., Townsend, C. A., &amp; Rosenzweig, A. C. (2001). Structure of beta-lactam synthetase reveals how to synthesize antibiotics instead of asparagine. Nat Struct Biol, 8(8), 684-689. doi:10.1038/90394

Mutalik, V. K., Guimaraes, J. C., Cambray, G., Lam, C., Christoffersen, M. J., Mai, Q. A., ... &amp; Endy, D. (2013). Precise and reliable gene expression via standard transcription and translation initiation elements. Nature methods, 10(4), 354-360. 

Organization, G. W. H. (2017). Prioritization of pathogens to guide discovery, reserach and development of new antibiotics fro drug-resistant bacterial infections, including tuberculosis.
                        
Paget, M. S., Chamberlin, L., Atrih, A., Foster, S. J., &amp; Buttner, M. J. (1999). Evidence that the extracytoplasmic function sigma factor sigmaE is required for normal cell wall structure in Streptomyces coelicolor A3(2). J Bacteriol, 181(1), 204-211.

Sadiq, A. A., Patel, M. R., Jacobson, B. A., Escobedo, M., Ellis, K., Oppegard, L. M., . . . Kratzke, R. A. (2010). Anti-proliferative effects of simocyclinone D8 (SD8), a novel catalytic inhibitor of topoisomerase II. Invest New Drugs, 28(1), 20-25. doi:10.1007/s10637-008-9209-1

Schimana, J., Fiedler, H. P., Groth, I., Sussmuth, R., Beil, W., Walker, M., &amp; Zeeck, A. (2000). Simocyclinones, novel cytostatic angucyclinone antibiotics produced by Streptomyces antibioticus Tu 6040. I. Taxonomy, fermentation, isolation and biological activities. J Antibiot (Tokyo), 53(8), 779-787.
                        
Shaikh, S., Fatima, J., Shakil, S., Rizvi, S. M., &amp; Kamal, M. A. (2015). Antibiotic resistance and extended spectrum beta-lactamases: Types, epidemiology and treatment. Saudi J Biol Sci, 22(1), 90-101. doi:10.1016/j.sjbs.2014.08.002

Tetko, I. V., Gasteiger, J., Todeschini, R., Mauri, A., Livingstone, D., Ertl, P., . . . Prokopenko, V. V. (2005). Virtual computational chemistry laboratory--design and description. J Comput Aided Mol Des, 19(6), 453-463. doi:10.1007/s10822-005-8694-y

Tsai, F. T., Singh, O. M., Skarzynski, T., Wonacott, A. J., Weston, S., Tucker, A., . . . Wigley, D. B. (1997). The high-resolution crystal structure of a 24-kDa gyrase B fragment from E. coli complexed with one of the most potent coumarin inhibitors, clorobiocin. Proteins, 28(1), 41-52.</P></DIV></DIV></DIV><DIV id="ImpressumSponsoren"><DIV><P>© iGEM Team Tuebingen 2017</P></DIV></DIV></DIV></DIV></DIV></DIV></DIV></BODY></HTML>