<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_UChicago_Notebook skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><DIV id="top_title"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:UChicago/Notebook</SPAN></H1></DIV><DIV id="HQ_page"><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><DIV id="menucontainer"><DIV class="navmenubkg"><DIV id="navmenubox"><UL id="nav"><LI><A href="https://2016.igem.org/Team:UChicago/Description" title="Project Overview">PROJECT</A></LI><LI style="margin-left:3px"><A href="https://2016.igem.org/Team:UChicago/Description" title="Project Overview">PROJECT OVERVIEW</A></LI><LI><A href="https://2016.igem.org/Team:UChicago/Design" title="Design">DESIGN</A></LI><LI><A href="https://2016.igem.org/Team:UChicago/Results" title="Results">RESULTS</A></LI></UL></DIV><LI><A href="https://2016.igem.org/Team:UChicago/Parts" title="Parts">PARTS</A></LI><LI style="margin-left:70px;"><A href="https://2016.igem.org/Team:UChicago/Parts" title="Parts Overview">PARTS OVERVIEW</A></LI><LI><A href="https://2016.igem.org/Team:UChicago/Basic_Part" title="Basic Parts">BASIC PARTS</A></LI></DIV><LI><A href="https://2016.igem.org/Team:UChicago/Human_Practices" title="Human Practices">PRACTICES</A></LI><LI style="margin-left:80px"><A href="https://2016.igem.org/Team:UChicago/Human_Practices" title="Human Practices">HUMAN PRACTICES</A></LI></DIV><LI><A href="https://2016.igem.org/Team:UChicago/Team" title="Team">TEAM</A></LI><LI style="margin-left:380px;"><A href="https://2016.igem.org/Team:UChicago/Team" title="Team">TEAM MEMBERS</A></LI><LI><A href="https://2016.igem.org/Team:UChicago/Collaborations" title="Collaborations">COLLABORATIONS</A></LI></DIV><LI><A href="https://2016.igem.org/Team:UChicago/Safety" title="Documentation">DOCUMENTATION</A></LI><LI style="margin-left:570px;"><A href="https://2016.igem.org/Team:UChicago/Safety" title="Safety">SAFETY</A></LI><LI><A href="https://2016.igem.org/Team:UChicago/Notebook" title="Notebook">NOTEBOOK</A></LI></DIV><LI><A href="https://2016.igem.org/Team:UChicago/Attributions" title="Attributions">ATTRIBUTIONS</A></LI><LI style="margin-left:695px;"><A href="https://2016.igem.org/Team:UChicago/Attributions" title="Attributions">ATTRIBUTIONS</A></LI><LI><A href="https://2016.igem.org/Team:UChicago/References" title="References">REFERENCES</A></LI></DIV></DIV></DIV><DIV class="maindiv"><DIV class="container"><P><SPAN class="ptitle"> 4/21/16</SPAN></P><P><SPAN class="p">

Session Leader: Kevin

KY/SD/JB: We mixed up 10 mL of 50 mg/mL antibiotic stocks for kanamycin, ampicillin, and chloramphenicol, filtered them, and put them in our fridge. These should be aliquoted at 500 uL and frozen down in our -20 freezer some other time. We also autoclaved 500 mL of LB and made batches of chloramphenicol and ampicillin plates.

LB is on the shelf. The plates were poured but remain on the counter. KY will put them in the 
cold room tomorrow by lunchtime.</SPAN></P><P><SPAN class="ptitle"> 4/22/16</SPAN></P><P><SPAN class="p">
KY: The plates from yesterday were put in the cold room on the left, on the bottom part of a shelf on the right side of the room (shelf is labeled with green tape “iGEM Temp”).</SPAN></P><P><SPAN class="ptitle"> 4/27/16</SPAN></P><P><SPAN class="p">
Session Leader: Kevin

KY/JB/SD: Inoculated E. Coli in preparation for miniprep and glycerol stocks.</SPAN></P><P><SPAN class="ptitle"> 4/28/16</SPAN></P><P><SPAN class="p">
Session Leader: Kevin

KY/JB/SD: Mini-prepped pRS313-316, pRS423, 425, 426 plasmids; made glycerol stock for pRS313-316, pRS423, 425, 426 and placed them in -80 freezer. Made spreadsheet on google drive for glycerol stocks called plasmid inventory. Plasmids are stored in box labelled <U>iGEM 2016 Plasmids</U></SPAN></P><P><SPAN class="ptitle"> 4/30/16</SPAN></P><P><SPAN class="p">
Session Leader: Mansi

MC/CI/MP: Conducted transformations of competent cells with plasmids PRS 314, PRS 425, and PRS 426 (competent cells were from last year). Plated total of 350 microliters of each plasmid onto amp plates. Plates are kept in incubator. </SPAN></P><P><SPAN class="ptitle"> 5/3/16</SPAN></P><P><SPAN class="p">
Session Leader: Kevin

DG: Made LB and LB+AMP plates (put in cold room), and a 1L LB solution (left at bench). Divided up ampicillin, kanamycin, chloramphenicol originally in fridge into 1 mL aliquot tubes and put in freezer in a box labeled “Antibiotic Stocks”. Plated some competent cells from the -80 freezer in preparation for making competent cells on Thursday.</SPAN></P><P><SPAN class="ptitle"> 5/4/16</SPAN></P><P><SPAN class="p">
SD: Inoculated competent cells from yesterday and put them into the shaker to grow</SPAN></P><P><SPAN class="ptitle"> 5/5/16</SPAN></P><P><SPAN class="p">
Session Leader: Kevin

JB:
Made competent cells and stored them in the iGEM shelf of -80C freezer.

Made 10mM stocks of T3/T7 overhang primers, labelled with R1 and F1. They are stored in our freezer (along with undiluted stocks).

Used the R1/F1 10mM stock and Phusion master mix to PCR plasmids pRS313, 314, 315, 316, 423, 425, 426. Steve and Julia each did the 7, so in total we did 2 reactions for each plasmid.  We created a program titled “Phusion” on PCR machine #6.</SPAN></P><P><SPAN class="ptitle"> 5/6/16</SPAN></P><P><SPAN class="p">
JB: Got the PCR samples from the machine and put them in the fridge.</SPAN></P><P><SPAN class="ptitle"> 5/10/16</SPAN></P><P><SPAN class="p">
Session Leader: Benjamin 

AS: Made 1l of LB media, Cast 1 agarose gel.

Session Leader: Mansi

NM/CI: Today we ran the PCR samples, we did not know which ones were which so we arbitrarily labelled them 1-15. Below is a picture of the gel we ran, the UV machine was not working for us so we used the blue light instead. 

We also unpacked the items that were delivered to us. 
</SPAN></P><P><SPAN class="ptitle"> 5/11/16</SPAN></P><P><SPAN class="p">
JB: Received packages from NEB containing master mixes, DNA ladders, and competent cells. Master mixes (HiFi and Q5) are in -20 freezer, QL Purple 2-Log DNA Ladder is in -4 fridge, competent cells are labelled “iGEM 2016” and are in the -80 freezer

Session Leader: Benjamin

AS: We reran the PCR on the 7 plasmids (we put them in machine 6). Each tube is labeled 1-7, and we added corresponding labels to the plasmid containers in the -20 freezer. We also ran a transformation efficiency test on the 2016 competent cells. However, we accidentally plated them on the wrong plates, so nothing grew. This will need to be redone.

BH: Labels 1-7 correspond with labels on template; Plates should have been Amp instead of Kan.</SPAN></P><P><SPAN class="ptitle">5/12/16</SPAN></P><P><SPAN class="p">
Session Leader: Kevin

JB: Ran the 5/11 group’s PCR samples. Order of samples in image are: ladder, 1, 2, 3, 4, 5, 6, 7, 4. From looking at DNA ladder, it looks like only 1, 5, and 7 had the amplification we wanted.</SPAN></P>
We also made 0.1% adenine solution that needs to be autoclaved.
We autoclaved eppendorf tubes.
Set up new PCR reactions for all the plasmids, labelled 1, 2, 3, 4, 5, 6, 7. We set up 2 reactions for each sample -- one with Phusion master mix and the other with Q5 master mix. The samples are labelled “phusion” and “Q5”.
<P><SPAN class="ptitle"> 5/15/16</SPAN></P><P><SPAN class="p">
Session Leader: Kevin

DG: Ran samples on gel. Order of samples in image are: ladder, 1, 2, 3, 4, 5, 6, 7, 8, ladder for each half.</SPAN></P>
2nd row refers to BioBrick (sp) labeled tubes
<P><SPAN class="ptitle"> 5/18/16</SPAN></P><P><SPAN class="p">
OL: Cast 1% agarose gel. Autoclaved 0.1% adenine solution. Ran native agarose gel of “phusion” PCR products. 
</SPAN></P><P><SPAN class="ptitle"> 5/19/16</SPAN></P><P><SPAN class="p">
Session Leader: Kevin

JB: Made 50% PEG that needs to finish mixing, made YPDA plates (have red and black stripe and are in cold room). We also gel purified PCR samples but they got thrown out…… 
</SPAN></P><P><SPAN class="ptitle"> 5/24/16</SPAN></P><P><SPAN class="p">
JB got pipettes ready to be calibrated by Novamed
KY plated E Coli to be used for miniprep later in the week
</SPAN></P><P><SPAN class="ptitle"> 5/25/16</SPAN></P><P><SPAN class="p">
KY inoculated E. Coli in preparation for miniprep
</SPAN></P><P><SPAN class="ptitle"> 5/26/16</SPAN></P><P><SPAN class="p">
Session Leader: Kevin
JB: Mini-prepped pRS313-316. They are in iGEM 2016 Plasmids box in -20 freezer. 

We also started to test the competency of our competent cells. We conducted transformations with pRS313 and competent cells from both this year and last year. The plates are in the incubator and JB will check on them tomorrow.
</SPAN></P><P><SPAN class="ptitle"> 5/27/16</SPAN></P><P><SPAN class="p">
JB: Checked on competent cells. Calculated transformation efficiency:
950 CFU/(0.27 ug DNA) = 3519 CFU/ug
</SPAN></P><P><SPAN class="ptitle"> SUMMER BEGINS HERE!</SPAN></P><P><SPAN class="ptitle"> 6/24/16</SPAN></P><P><SPAN class="p">
MP:
Unpacked iGEM shipments. Put everything away in locations noted in 2016 inventory sheet.
Streaked out agar stabs for the BBa_K1680014-16 backbones on 3 separate plates and put them in the 37 deg incubator overnight.
</SPAN></P><P><SPAN class="ptitle"> 6/25/16</SPAN></P><P><SPAN class="p">
JB:
Went in to inoculate BBa_K1680014-16 bacteria but didn’t see any growth on the plates. Made new plates with same stabs.
</SPAN></P><P><SPAN class="ptitle"> 6/26/16</SPAN></P><P><SPAN class="p">
JB:
Went in to check on the new plates (made 6/25) but there was nothing on the plates again. We may need to reorder backbones from iGEM. Or we do have the backbones (pRS313-16, 423, 425, 426) without the prefix/suffix in glycerol stocks and can add the prefix/suffix with overhang PCR instead of reordering.
</SPAN></P><P><SPAN class="ptitle"> 7/6/16</SPAN></P><P><SPAN class="p">
SD/CI:
Made 1 L normal YPAUD media (to grow yeast) and 250 mL YPAUD media without glucose (to starve yeast). Left both on shelf above lab bench.
YPAUD is standard YPD yeast media with added adenosine sulfate and uracil. Glick says adding those things helps growth.
Made a 5 mL yeast culture and incubated at 30 degrees C overnight. The culture was made from a preexisting starter culture the Glick lab had in the cold room. The yeast strain is JK9 3DA.
Developed protocols for growing yeast and making yeast growth curves
Jason helped us do all of this stuff
</SPAN></P><P><SPAN class="ptitle"> 7/7/16</SPAN></P><P><SPAN class="p">
SD/CI:
Made a 0.25 dilution of last night’s 5 mL yeast culture and incubated it for 90 minutes. 
We expected the OD of the culture to double, but it only made it to 0.295, so we incubated it for 90 more minutes. After that, it was ~0.6.
Attempted to continue with the starvation assay protocol (see yeast protocols), but we ran out of culture
Made a 10 mL culture from the starter culture and put it in the 30 degree incubator to incubate overnight
</SPAN></P><P><SPAN class="ptitle"> 7/8/16</SPAN></P><P><SPAN class="p">
CI/JB:
Continued yeast starvation essay, starting at 6:30 (2 hours in)
Julia accidentally spilled the starvation culture ^-^
Added detail to yeast starvation assay protocol so it can be redone</SPAN></P><P><SPAN class="ptitle"> 7/10/16</SPAN></P><P><SPAN class="p">
JB:
Made starter culture for yeast JK9-3da so that starvation assay can be redone tomorrow. It’s in the 30 incubator.
Inoculated YIPLAC204 and YIPLAC128 bacteria from glycerol stock in LB/amp for miniprep tomorrow. It is in 37 incubator.
Streaked out YIPLAC204 and YIPLAC128 on LB/amp plates for good practice and to test our plates. They’re in the 37 incubator
</SPAN></P><P><SPAN class="ptitle"> 7/11/16</SPAN></P><P><SPAN class="p">
CI/SD/JB:
Another yeast starvation assay
Miniprep with bacteria Julia inoculated
Checked Julia’s plates from yesterday. They grew pretty well.</SPAN></P><P><SPAN class="ptitle"> 7/15/16</SPAN></P><P><SPAN class="p">
SD/JB/CI/MH
Starvation assay! Again!
PCR on yeast genome to extract terminator and promoter. Didn’t work.
Primers arrived yesterday! So did the G-blocks for the reporter construct
Other G-blocks are stuck in quality control :[
Made LB agar plates</SPAN></P><P><SPAN class="ptitle"> 7/19/16</SPAN></P><P><SPAN class="p">
MH/SD
ATP determination kit arrived in the mail! We need to test it, but Chinye knows the protocol and isn’t here today
After talking with Professor Glick in today’s lab meeting we decided to run our yeast starvation assay again, but this time we’re going to track cfu instead of OD
To that effect, Miranda made 1L of YPAUD agar
It yielded 40 plates!
Steve put yeast in the incubator to grow overnight in prep for the test
We’re also going to try PCRing the promoter and terminator out of the yeast genome again
We think the problem last time had to do with the annealing temp
We were using Phusion polymerase which has a higher annealing temp than most polymerases
We’re going to try with Herculase, see if it works better
Supposedly our primers have an unusually low annealing temp
This is with most polymerases
For Phusion the temp is always higher
The PCR worked! For the terminator
Worked with Phusion, less well with Herculase
Probably because the Herculase trial was run at ~50˚C, which is way low
Phusion trial was run at 62˚C, optimum temp for terminator
The promoter still didn’t work
Going to try again at 66˚C, which is the optimum temp for the promoter
Steve purified the terminator DNA from that run
Yield was 19.2 ng/μl
</SPAN></P><P><SPAN class="ptitle"> 7/20/16</SPAN></P><P><SPAN class="p">
SD/MH/CI
Attempted the starvation assay but the starvation media was contaminated so we only plated the original culture to test dilution range
The plates are incubating overnight
Made 1L of starvation media
Performed PCR to amplify promoter but it didn’t work. However, the problem was discovered! The template that we are using for the promoter has bases in front that aren’t present in yeast, so only half of our primer was annealing. Steve ordered new forward and reverse primers to get a better GC content and to fix the annealing problem. They should come tomorrow and he will run another PCR
Incubated culture overnight to perform starvation assay tomorrow
</SPAN></P><P><SPAN class="ptitle"> 7/21/16</SPAN></P><P><SPAN class="p">
SD/MH/CI/JB
Having remade the starvation media, we ran the assay
Dilutions were 10^-2, 10^-3, 10^-4, 10^-5, and 10^-6
Instead of putting each dilution on its own plate, we put a drop of each on one plate in a row
The idea for this was just to look for the timepoint when the colonies collapse
We plated 10µl of each dilution
This morning we got a notification that our Gal4-KaiCu_LexA-SasA G-block 1 was having problems in the quality control step of synthesis
Steve re-optimized it using the tools on the IDT website and re-ordered it
It should arrive Aug 2nd
We also met with Barry Aprison again to discuss outreach stuff
He showed us where he’s storing the prototype kiosk we built
We agreed to meet tomorrow to try to set it up with all its equipment (microscopes, iPad, etc.)
He also asked us to think about ideas for an interactive model of diffusion
Magnetic water molecules?
Something that moves, maybe you can introduce salt
Hopefully our primers will arrive tomorrow, so we’ll be able to PCR the yeast genome promoter
We also hope to test our ATP determination kit
</SPAN></P><P><SPAN class="ptitle"> 7/22/16</SPAN></P><P><SPAN class="p">
JB:
ATP determination assay
We are doing the assay using 0.5 mL of 0.4 OD culture, split into regular and starvation cultures. Since we didn’t prepare a starter culture for this assay last night, these cultures are being made from the “original” culture used in the starvation assay yesterday.
</SPAN></P><P><SPAN class="ptitle"> 7/25/16</SPAN></P><P><SPAN class="p">
SD/MH/CI
Performed PCR to amplify the promoter using primers F2 and new primer R2**
Performed 3 reactions
None of the reactions worked 
Will retry PCR tomorrow 
</SPAN></P><P><SPAN class="ptitle"> 7/26/16</SPAN></P><P><SPAN class="p">
SD:

Performed PCR to amplify promoter using primers F2* and R2**
The PCR worked!!! The yield was 6.7ng/μl
Will repeat tomorrow just so that we have more because we will use this a lot

MP/CI: 

Performed restriction digest using Kasey’s protocol.
We received 4nmol of “TetO”. Diluted it with 50ul to 4640ng/ul.
We received 1000ng of GFP reporter. Diluted it with 50ul to 20ng/ul.
ADH1 terminator was already purified -- 19.2ng/ul.
Cut TetO with BamH1; GFPR with Bg11 and BamH1; ADH1T with BglII
Quantified yield with Nanodot.
GFPR = 4.5 ng/ul. TetO = 1776.1ng/ul. Diluted with 1.5ml → 34.8ng/ul. ADH1T = 14.6ng/ul.
These are stored in top shelf of the -20.
Performed ligation using Jason’s protocol
This is now sitting in the Thermocycler. 
</SPAN></P><P><SPAN class="ptitle"> 7/27/16</SPAN></P><P><SPAN class="p">
SD/MP/CI/MH
The PCR worked yesterday, but the yield was still too low to be usable, so we are re-doing it under the same conditions (primers F2* and R2**, annealing temp of 71˚C, Phusion polymerase) in order to get more DNA
If yesterday’s ligation worked, the place where the two pieces were cut should now be an un-cuttable scar, since it is a ligation of a BamHI sticky end and a BglII sticky end
To test this, we are re-digesting with both BamHI and BglII and are then going to run the product on a gel
If the ligation didn’t work properly, the BamHI sticky ends will have ligated with other BamHI sticky ends (and same with the BglII sticky ends), and so the cut sites will still be intact and can be re-cut
If it did work properly, we expect to see a DNA piece of ~1900bp on the gel

MP:
Ran restriction digest on ligation from yesterday with Kasy method using 20ug sample (all of it)
Somewhere along the way we ended up with 200ul sample instead of 100ul which you would expect from a digest.
Ran sample on a gel--came up blank.

Re-started work from yesterday.
First, attempted to redo ligation in two parts using 50ng purified GFPR and 50ng purified “teto” from yesterday.
BUT, realized after the fact that because GFPR was already cut with Bgl and Bam, the promoter was going to preferentially ligate to the wrong side of the insert. 
This ligation is currently sitting in the Thermocycler but we suspect low yields if we attempt to digest/gel purify.

Started over again by doing a new restriction digestion!
Recut GFPR with just Bgl2 (Kasey method)
Spin purified -- very low yield -- 0.9ng/ul.
This is in top shelf of -20 labeled “GFPR2 purified”
Will attempt to recut and repurify GFPR tomorrow with hints from Fernando (let EB sit in column for 1 min before spinning; use higher initial concentration; do multiple digests in same column to concentrate final sample.)
Nothing works and everything is terrible.

SD:
Made 1L starvation media split between 2 500ml bottles
Ran a second PCR to amplify promoter in order to get more DNA
Primers F2* and R2** at 71 degrees
PCR worked but very poorly and there was not enough DNA to purify 
Did a second PCR included with PCR of G-Blocks
Received 8 G-blocks
REsuspended all G-blocks received
Performed PCR to amplify G-blocks Gal4-KaiCp_LexA-SasA 1 and Gal4-KaiCp_LexA-SasA 2 
Used primer pair 6 and 7 respectively 
Included the second PCR of promoter
Temperature was 68 degrees for all
Yield was as follows:
Part 1: 13.8ng/μl and 4.9ng/μl
Part 2: 4.8ng/μl
Promoter: 1.8ng/μl</SPAN></P><P><SPAN class="ptitle"> 7/28/16</SPAN></P><P><SPAN class="p">
MP:

Ran GFPR digest again with Bgl2, measured 16.6ng/ul (woo), and then re-attempted ligation to LexAO.
I think it’s in -20 in “GFPR2.2” if there’s any left
Gel purification turned up blank except for two very faint and possibly imaginary bands. Going to PCR GFPR (ordered primers) since we only have a little bit left and then start from the beginning – again – with a ton of starting material to help yield.

Gibson Assembled KaiCP-SasA1 and KaiCP-SasA2 → Gave to Steve to run a PCR.

CI
Prepared one glucose-starved yeast sample and one control yeast sample for ATP measurement. They were 4 mL each, but 1 mL from each was used for OD measurements.
Used yeast straight out of the cold room because I forgot to incubate a starter culture overnight.
Initially, both had ODs of 0.483. After two hours, the starved culture had an OD of 0.490, and the control had one of 0.449.
Took 12.5 uL of yeast from each culture, lysed the cells with 10% w/v  trichloroacetic acid, and added neutralization buffer (following ATP Measurement protocol).
Made 10 mL of standard reaction solution (without the luciferase or luciferin in it) for ATP measurement.
Made 500 nM ATP standard solutions with ATP stock from the ATP determination kit and some TE buffer from our bench.
10 1-mL eppendorf tubes full. 10 mL total. 9 are left.
Put everything necessary for the ATP determination assay on ice and carried it to Rust lab to meet with Justin
Justin told us that our kit only measures absolute ATP concentration. For our experiment, we want ATP/(ATP+ADP) ratios. That requires extra reagents to convert all ADP to ATP.
Justin is going to send us the Rust lab protocol for ATP/(ATP+ADP) measurement, and he’s going to help us follow the protocol (Friday 7/29 or Monday 8/1)
Threw out the lysates because they’re in the wrong buffer for Rust lab’s protocol
Put control culture in incubator (to use tomorrow if necessary). Threw out starvation culture.
Put ATP standard solutions and all of the ATP kit stuff back in the -20 freezer

SD
Ran PCR on assembled Gal4-KaiCp_LexA-SasA in order to get more of it and to add nucleotides after the cutsites to ensure that the enzymes would cut
Ran 3 reactions at 68 degrees and 35 cycles this time to get more DNA
2 of the lanes produced a streak of DNA instead of a band but there was some problem loading the DNA so I think that was the problem
Lane 3 produced a normal band so I isolated it for DNA purification
Purified the PCR product
Yield was quite high. The first time I got 259.1ng/uL which was so good I measured it again. The measurements were as follows:
259.1ng/uL
156ng/uL
164ng/uL
To be safe I took the 3rd yield as most accurate, so the yield was 164ng/uL (a personal best!) The sample is in the -20 fridge in box 3 next to the Tef promoter
Ordered primers for amplification of GFPr because we are running out of it
Ordered quick-change pcr primers</SPAN></P><P><SPAN class="ptitle"> 7/29/16</SPAN></P><P><SPAN class="p">
MP:


Digested Kai-CP-SasA with Bgl2 -- got 10.1ng/ul (in -20 labeled “KaiCPSasA purified”)
Digested Tef1 with BamH1 but lost all material during purification due to low starting concentration. Could not ligate. Transferred responsibility to Steve who is currently PCRing more Tef1.
SD:
Ran a PCR to amplify Tef1 promoter. Ran 2 reactions at 71 degrees and 35 cycles using fusion
PCR worked and I extracted the DNA and purified it. Yield was 19.6ng/uL
Digested the TEF1 and left it in incubator for CI to take out. Will ligate tomorrow
Ran a second PCR to amplify the GFP reporter as we are running out of that DNA. Ran 2 reactions with Herculase at 55 degrees using Kasey’s protocol. Also included the TEF promoter just because always use more dna and because I wanted to test it under herculase and use it as a control. LEft PCR in cold room to run on a gel tomorrow. 
CI:
Decided not to attempt Rust lab’s ATP determination protocol today because our luciferase isn’t suspended in the right buffer for that, and Glick said our kit may be fine. I asked him to elaborate, and I’m waiting for his response.
Made starved and control cultures from yesterday’s control culture
Originally 5 mL each at OD of 0.4, but I figured I’d need more, so I diluted both to 10 mL
Resuspended primers that came in today: F16-F19 and R16-R19
Made yeast pellets from starved and control cultures (for the ATP determination assay, whenever we figure out exactly how we’re going to do it)
Put 0.5 mL of yeast in each eppendorf tube, centrifuged at 3 xg (something between 5000 and 6000 rpm) for 5 minutes, and pipetted the supernatant out
Dropped the tubes into liquid nitrogen in the cold room, fished them out with a sieve-looking thing (it’s sitting on top of the container of liquid nitrogen), and stored them in a box labeled “iGEM Yeast” in the ThermoForma -80 freezer in CLSC 209
It was awesome
The eppendorf tubes are out of order in the box because I was rushing
The liquid nitrogen came from the 10th floor
2 pellets made from each culture at each time point. Did 4 time points, 1-hour intervals (Justin said 4 was fine for a trial run).
Put the starvation culture and the control culture in the cold room
Took Steve’s digest out of the 37 degree incubator at 6:25 and put it in our -20 freezer (“Enzymes and Buffers” box, or the one with words crossed out)
Just FYI, flash-frozen eppendorf tubes occasionally explode.
Why does deoxyglucose take so long to arrive, and why is it so expensive? It’s just sugar?
</SPAN></P><P><SPAN class="ptitle"> 7/30/16</SPAN></P><P><SPAN class="p">
SD

Ran a gel for the PCR product from the amplification of GFPr and TEF
PCR worked but the GFPr bands were very streaky
Purified the product and received following yields:
GFPr reaction 1: 48.6ng/uL
GFPr reaction 2: 45.3ng/uL
TEF promoter: 7.5ng/uL
The GFP has questionable purity 
</SPAN></P><P><SPAN class="ptitle"> 8/1/16</SPAN></P><P><SPAN class="p">
JB:
Re-digested KaiCP-SasA with BgIII because Steve and I couldn’t find the one that Morgan did in the -20 fridge. Ended up getting a very low yield (2.9 ng/uL), and also ended up finding the one Morgan did. So threw this one away and used Morgan’s for the ligation that I did next:
Ligated KaiCP-SasA and Tef1. Purified ligation product. I digested this product with BamHI and put it in the incubator. Steve will take this out later so it can be ligated tomorrow.
I also digested the terminator with BglII and put it in the incubator. Steve will also take this out later so it can be ligated to the Tef1-KaiCP-SasA part tomorrow.
SD:
Ran PCR of TEF1 and ADH1 from DNA isolated from previous PCR
Ran 2 reactions of TEF1 with herculase at 55 degrees
Ran 2 reactions of ADH1 with phusion at 66 degrees
Ran 1 reaction of TEF1 with phusion at 66 degrees
PCR worked and the bands were strong. Isolated them for purification
Yield was low for TEF and ok for ADH
TEF1: 4.6ng/uL
ADH: 22.2ng/uL</SPAN></P><P><SPAN class="ptitle"> 8/2/16</SPAN></P><P><SPAN class="p">
SD/CI

SD:
Purified the digestion of TEF+KaiCp-SasA and ADH
Yields were very low: 
2.3ng/uL for TEF+KaiCp-SasA
4.2ng/uL for ADH
Unable to ligate
Nothing works and I am very sad
Re-ligated KaiCp-SasA and TEF
Purified and obtained yield of 24.9ng/uL
Ran PCR to amplify TEF+KaiCp-SasA
3 reactions using phusion at 71 degrees with primers F2* and R7 and 35 cycles
2 reactions used the purified DNA
1 reaction used DNA from the ligation but before purification (to see what worked better)
The idea is the amplify the ligation so that we have more product when we try and digest the second time, and to troubleshoot ligation
Gel purified using GQ buffer mixed with 3M sodium acetate, using new elution buffer and different wash buffer. Results were great!
Yields were:
63.7ng/uL
53.5ng/uL
59.0ng/uL
All yields were labelled properly and placed in -20 in box labelled g-blocks
CI
Digested GFP reporter with BglII
Digested LexA operator (“TetO”) with BamHI
Purified both GFP reporter and BglII with nucleotide removal kit
Accidentally added 1 mL of Buffer PNI to the GFP reporter instead of 0.5 mL. Carried out the purification process in two columns and combined them at the elution step.
Put the digested and purified GFP reporter (labeled “GFP Dig Pur”) and LexA operator (labeled “TetO Dig Pur”) in iGEM 2016 GBlocks box in -20 freezer</SPAN></P><P><SPAN class="ptitle"> 8/3/16</SPAN></P><P><SPAN class="p">
CI:
Measured GFP reporter and LexA operator digest yields with Nanodrop
GFP: 4.4 ng/uL
LexA: 6.3 ng/uL
Ligated GFP and LexA with Quick Ligase
Picked up dNTPs from Fisher stockroom and made 10 mM stocks (9 tubes, 1 mL each). Put them in the bottom drawer of the -20 freezer.
Went with Miranda to see the RSO adviser , but he’s out again
SD:
Ran a PCR on LexAop-GFPr, TEF, and ADH
Reactions were as follows w/ phusion at 68 degrees and 35 cycles:
LexAop-GFPr (F15, R16) 
LexAop-GFPr (F15, R16) 
TEF (from previous PCR)
ADH (from genomic DNA)
ADH (from genomic DNA)
PCR worked for LexAop-GFPr and TEF but not ADH. Isolated LexAop-GFPr and TEF for purification
Yields:
LexAop-GFPr: 116.5ng/uL
TEF: 69.3ng/uL
Digested TEF+KaiCp-SasA and ADH
Purified and ligated them
Yield: 29.9ng/uL
Ran a PCR to amplify the full KaiCp construct (TEF+KaiCp-SasA+ADH)
3 reactions w/phusion at 72 degrees and 35 cycles
2 reactions were done with purified ligation,1 reaction was done with unpurified ligation
Left to run overnight 
</SPAN></P><P><SPAN class="ptitle"> 8/4/16</SPAN></P><P><SPAN class="p">
SD/CI/MH

SD: 
Ran PCR of full construct on a gel
PCR worked and I isolated the product of all 3 reactions to purify
Yields were:
22.1ng/uL
23.1ng/uL
27.7ng/uL
Tubes were labelled KaiCp Construct and placed in the -20 in the box labelled G-blocks
Ran PCR to amplify ADH (we ran out)
3 reactions from genomic DNA w/phusion at 66 degrees and 35 cycles
PCR did not work. Very dim bands and a lot of primer dimerization. Will try PCR again tomorrow from the KaiCp construct instead of genomic DNA.
CI:
Made an agarose gel
Digested LexA-GFP
Spin column purified LexA-GFP, labeled it “LexA GFP Dig Pur,” and put it in the GBlock box in the -20 freezer
MH:
Performed Gibson assembly with the Gal4-KaiCu G-blocks
I ran two Gibson assemblies because for the first one I accidentally put in twice the required master mix
Both are in the G-blocks box (G-blox?), labeled “KaiCu 8/4/16” on the top and, for the first one, “Gal4-KaiCu Gibson Assembly” on the side, and for the second “GAL4-KaiCu 2” — “GAL4-KaiCu 2” is the one with the correct reagent volume
</SPAN></P><P><SPAN class="ptitle"> 8/5/16</SPAN></P><P><SPAN class="p">
MH
PCR’ed the GAL4-KaiCu Gibson Assembly products

CI
Measured yield from LexA-GFP digest with Nanodrop. It was 10.3 ng/uL
Ligated LexA-GFP and ADH1 (terminator)
Put LexA-GFP-ADH1 in 2016 iGEM GBlocks box

SD

Ran PCR to amplify ADH
5 reactions at 66 degrees w/phusion and 35 cycles
3 reactions were from the KaiCp construct and 2 were with genomic DNA
PCR worked with reactions from construct but not well with genomic DNA
Isolated 3 bands corresponding to the KaiCp construct template reactions, and purified them. 
Total yield was 131.7ng/uL
</SPAN></P><P><SPAN class="ptitle"> 8/6/16</SPAN></P><P><SPAN class="p">
JB:
Gibson assembled the two Kai-LexA-SasA fragments
PCRed the assembled KaiA-LexA-SasA insert.
Made 1:10 dilution (5 uL primer stock, 45 uL water) of primers F10 and R11 and used these for the PCR
Split the 20uL mix from the Gibson assembly into 5 uL portions for 4 PCR reactions
Used Phusion and anneal temp 69
Kasey found 2 eppendorf tubes labeled “KaiCP construct” and “50 ng/ul A12 genomic DNA” at the bottom of an ice bucket and gave them to me. Someone must’ve accidentally left them in the bucket. I’m not sure if we still need them, but I put them in the -20 in the “iGEM 2016 g-Blocks” box.

JB/SD:
We ran the 4 KaiA-LexA-SasA PCR products on a gel and got this:</SPAN></P>
3 out of the 4 PCR reactions had worked, but for some reason they had 2 bands. We didn’t know which one contained the correct fragment, so decided to extract both. Steve did the top bands, and Julia did the bottom bands.
Got similar concentrations for both (~36-38 ng/uL). Labelled them “top” and “bottom” (corresponding to the bands) and placed them in the constructs gblocks box.

<P><SPAN class="ptitle"> 8/7/16</SPAN></P><P><SPAN class="p">
JB:
Digested Tef1 with BamHI and the 2 KaiA-LexA-SasA fragments from yesterday (“top” and “bottom” bands) with BglII
Labelled “Tef1 digest,” “Top digest,” and “Bottom digest” in the gblocks box.
They still need to be purified and quantified.
</SPAN></P><P><SPAN class="ptitle"> 8/8/16</SPAN></P><P><SPAN class="p">
CI:
Attempted to run PCR of LexA-GFP-ADH1 (complete construct), unpurified
3 tubes running simultaneously, each with 1 uL of DNA
Annealing temperature of 72 degrees C
Accidentally set annealing cycle time to 30 min instead of 30 s. Had to abort the PCR.
JB:
Spin column purified Tef1, Top, and Bottom digests
Ligated the purified Tef1 and Top, and Tef1 and Bottom.
Set aside 3 uL from each ligation and spin column purified the rest, as recommended by Steve. Ran 4 PCR reactions from the ligation products: unpurified Tef1/Top, unpurified Tef1/Bottom, purified Tef1/Top, and purified Tef1/Bottom
Meanwhile, as the PCR reaction was going, we talked to Kasey about getting 2 bands on 8/6/16. We decided that the best way to know what happened with the bands is to do a diagnostic digestion with ScaI. I digested the Top and Bottom with ScaI (incubated for 40 min) and ran them on a gel, but there was nothing on the gel
Made a new gel to save time later, and that’s also on the counter with the gel boxes. If someone wants to use it today they can.

SD:
Did Gibson assembly of KaiC_KaiB
Ran a PCR on the gibson product
4 reactions w/phusion at 68 degrees and 30 cycles
PCR worked, isolated and purified DNA
Yield: 28.8ng/uL
EDIT: Gibson assembly may have given me two products
</SPAN></P><P><SPAN class="ptitle"> 8/9/16</SPAN></P><P><SPAN class="p">
JB:
Ran the ligated Tef1+Top and Tef1+Bottom PCR products on a gel. From left to right, the lanes are Tef1+Top purified, Tef1+Bottom purified, Tef1+Top unpurified, Tef1+Bottom unpurified. Tef1+Top bands are very faint or nonexistent. If they are existent it’s weird because now the Top and Bottom ligation products are all the same size.
CI
PCRed LexA-GFP-ADH1
3 tubes, 1 uL DNA in each tube
Ligation was not purified beforehand
Ran PCR products on gel. 2 out of 3 showed up really well.
Cut out gel chunks w/ DNA, put them in 3 eppendorf tubes labelled “GFP Gel Ex.” Put them in the styrofoam holder in the cold room.

MH
Ran the PCR product from the Gibson-assembled KaiCu construct on a gel
Picture to come, but basically the result was a massive smear with a lot of DNA concentrated in the well
Extracted the DNA from the well area, will re-run on a gel tomorrow
Plans for tomorrow:
Re-run Gibson product on gel
Re-do Gibson assembly just in case
Test Gibson product via digestion

SD:
Made a 1:1000 dilution of Kanamyacin stock for making plates
Made LB plates but forgot to put Kanamyacin into the first batch, and the second batch solidified as I was pouring, so we have only have LB plates. Will make kanamyacin plates tomorrow.
Digested KaiC_KaiB with BamH1 and BglII
Digested INtermediary plasmid with BamHI and BglII
Purified Plasmid by spin column
Yield was 71.7ng/uL, 35uL total
Dephosphorylated plasmid
Gel purified dephosphorylated vector
Put in cold room to gel extract tomorrow
Gel purified digested KaiC_KaiB. Saw 2 distinct bands under the UV light. Isolated the top one because I thought it was correct (a good digestion couldn’t make our product longer)
Purified it but got very low yield (bands were very dim). 2.9ng/uL. Not enough to work with. Will start over from Gibson assembly.
I hate cloning with an intense passion
</SPAN></P><P><SPAN class="ptitle"> 8/10/16</SPAN></P><P><SPAN class="p">
CI:
Extracted the LexA-GFP-ADH1 construct from the gel
The chunks of gel in the eppendorf tubes were way too large, so I added less than the called-for amount of QG buffer, melted the chunks, and separated the solutions into more tubes. After that, I added more QG buffer to each tube.
The chunk of gel containing the faintest band wouldn’t melt (at 1.24 g, the chunk was so big that I couldn’t get enough QG buffer into the tube), so I threw it out
The other two chunks (labeled 1 and 3) were melted and divided into 3. These chunks of gel were 0.88 g and 0.81 g respectively. For the gel extraction protocol, each tube should contain no more than 0.4 g of gel.
Ended up with two tubes of construct, each containing roughly 30 uL. Their caps are labeled “GFP 1” and “GFP 3” (the numbers were arbitrarily assigned to bands on the gel), and their sides say “GFP Construct Gel Extract.” They are in the Constructs box in the -20 freezer.
“GFP 1” yield: 10.5 ng/uL
“GFP 3” yield: 9.6 ng/uL
JB:
Re-attempting the KaiA-LexA-SasA construct today. Gibson assembled the two g-blocks for the insert
PCRed the Gibson product -- one of the reactions turned blue...and one of them had significantly less than 50 uL -- I think the caps to the reaction tubes don’t fit that well
Ran the gel for PCR above and got 3 bands this time. Sizes are about ~3000, 2800, and 1300. 
The middle band is maybe the one we want, since the desired fragment is 2822 bp. To be safe I extract/purified the two top bands separately, like I did with Steve on 8/6. And then I got negative readings on the nanodrop
I also ran a bunch of my stuff on a gel to find out where everything went wrong/what is happening with all these random bands. From left to right, they are Tef1/KaiALexASasA top band ligation, KaiALexASasA top band, KaiALexASasA top band digest with BglII, Tef1/KaiALexASasA bottom band ligation, KaiALexASasA bottom band, KaiALexASasA bottom band digest with BglII
It looks like the KaiALexASasA Top was over 3000 and the Bottom was around 2000 so I have no idea what got assembled in the Gibson assembly

SD:
Made Kanamyacin plates and left them on the bench to get rid of condensation
Gibson assembled KaiC-KaiB sing hi-fidelity master mix
Ran a PCR on KaiC-KaiB and let it go overnight to gel purify tomorrow
4 rxns with phusion at 68 degrees and 35 cycles</SPAN></P><P><SPAN class="ptitle"> 8/11/16</SPAN></P><P><SPAN class="p">
MH
Upon removing my PCR tubes from the thermocycler, I saw that two of them had popped open at some point during the program and were empty due to evaporation
One of them still had liquid in it, but like… this sucks!
This keeps happening to people, it might be a problem with our new tubes
With the one tube that still had some reaction product in it, I performed another diagnostic digest (using BsaI and CutSmart buffer), and ran the rest on a gel by itself
Picture to follow— I got nothin’
Seriously, there were no bands from either the diagnostic digest of the PCR or the PCR itself, although the ladder showed up perfectly
Perhaps something happened with the PCR, even though there was still liquid in that tube
In any event, I took some of the remaining Gibson assembly product and PCR’ed it, this time taping the tubes shut
Hopefully that will work better!
CI:
PCR amplified KaiA-LexA-SasA
Primers F10 and R11; annealing temperature 69 C; 4 min extension cycle (30x)
2 out of 3 of the PCR tubes were open when I opened the thermocycler, and the solutions inside had evaporated. 
Threw the two empty tubes out
Ordered new tubes again. The new ones will have domed tops and come in strips of 8.
Ran the KaiA-LexA-SasA from the one closed tube on a gel. Despite having run the gel until the dye nearly reached the bottom, it seems that neither the ladder nor the sample migrated well.

Might have left the sample at room temperature for too long?
Perhaps PCR was still messed up by the tube, even though it remained closed.

SD:
Ran KaiC-KaiB PCR on a gel today
Got double bands again and smearing despite decrease in extension time. Glick suggested synthesis by overlap extension. Will look into it.

</SPAN></P><P><SPAN class="ptitle"> 8/13/16</SPAN></P><P><SPAN class="p">
JB:
Redoing Chinye’s PCR on the KaiA Gibson. Trying to figure out where stuff is going wrong, so trying a bunch of new things
I did 2 reactions, each with 5 uL of the Gibson product. I made a new 1:10 primer dilution for R11 and F10 and used those for one of the reactions, and the original R11 and F10 for the other one.
In hindsight I should’ve also done a PCR with the old KaiALexASasA Gibson as a control because I know that one PCRed well...oh well already started the machine 
I’m planning on running those on the gel later with the old “Top” or “Bottom” Gibson product (which I know from previous experience will make a nice band on the gel) to see if we are messing up the PCR or the gel

</SPAN></P><P><SPAN class="ptitle"> 8/14/16</SPAN></P><P><SPAN class="p">
SD:
Ran a synthesis by overlap extension (SOE) PCR using the new protocol I designed. Ran it on a gel
Still got double bands and lots of smearing despite decreased extension time.Talked to Kasey who wasn’t sure what the problem was. Suggested using all new reagents. Will try that.

</SPAN></P><P><SPAN class="ptitle"> 8/15/16</SPAN></P><P><SPAN class="p">
SD:
Ran Gibson assembly of KaiC-KaiB. Ran it on a gel to see if the gibson assembly was what was giving us two products, but there was not enough DNA to show up
Ran a PCR on the individual g-blocks of KaiC-KaiB to get them more pure and to see if I could work with those. The PCR still had a lot of smearing, even though I used all new reagents. Because of this, I could only extract g-block 1. Will purify tomorrow. </SPAN></P><P><SPAN class="ptitle"> 8/16/16</SPAN></P><P><SPAN class="p">
MP:
Resuspended well 4B on iGEM distribution plate#4 and placed in third drawer of -20bands
Basically, said construct didn’t actually get constructed
Also, I may have used the wrong G-blocks, as elaborated upon below
Since last time clearly didn’t work, I performed another Gibson assembly with the Gal4-KaiCu G-blocks

Used competent cell test kit (procedure on iGEM website); waiting for results tomorrow
Used 3ul of DNA rather than 1ul at Kevin’s suggestion
Poured Kan and chloroamp plates
Spin purified LexA-GFP-ADH1 ligation product
Warmed elution buffer to 50deg and let sit for 5 minutes
Nanodot: 19.4ng/ul (woo)
Digested LexA-GFP-ADH1 and “KaiCP construct” (Gal4KaicPLexASasA) with EcoR1 and PST1. Left in 37deg overnight.
SD:
Ran PCR (SOE) on KaiC_KaiB  and KaiA_LexA-SasA
Got double bands and unspecific binding for KaiC-KaiB, but no bands for KaiA_LexA-SasA, and the control had a correct band. Also, herculase seems to have prevented smearing.
I isolated the correct bands for KaiC_KaiB for purification but I will aslo PCR the individual g-blocks because I am not sure if the band I am isolating is actually what I think it is.
Yield was 37.9ng/uL
This is alpha
Ran a PCR on plasmid yIplac 128 and 204 and let that go overnight
Ran a PCR on the individual g-blocks of KaiC-KaiB and KaiA_LexA-SasA and let it go overnight

</SPAN></P><P><SPAN class="ptitle"> 8/17/16</SPAN></P><P><SPAN class="p">
MP:

Raw efficiency counts (rough estimates):


Calc:
Results
efficiency:
average
5.3E+07
weighted:
6.9E+07

Efficiency on the low end but Kasey said plates look fine.


Transformed well 4B on iGEM distribution plate#4 (submission plasmid).
Plated on chloroamp plates + control in 37deg incubator overnight.

Gel purified LexA-GFP-ADH1 and “KaiCP construct” (Gal4KaicPLexASasA) digests from yesterday.
Got bands for KaiCP construct but not LexA-GFP-ADH1. Gave to Steve to isolate.
Need to PCR more LexA-GFP-ADH1 because we have very little left.


LABEL: Ladder, KaiCP construct (2/3), GFP (4/5, empty). Removed top band for KaiCP.

Digested linearized submission backbone using iGEM protocol. In lower box in -20. “Backbone digest.”
Ideally would want to purify but worried about losing small volume; Kevin says don’t have to.

Diluted stock primer R15 in separate tube (5ul in 45ul dH2O).
Ran two PCR reactions on the GFP reporter using Steve’s protocol. Left in thermocycler.
Chinye to purify.
Digested Steve’s KaiC-KaiB alpha and beta. Left in 37deg.
Left all materials Steve gave me in “constructs #2” box in third drawer of -20.

SD:
Ran the individual g-block PCR on a gel. Smearing returned but I was able to isolate both g-blocks of KaiC-KaiB, and the second g-block of KaiA_LexA-SasA. Purified them and got the following yields:
KaiC-KaiB 1: 22.4ng/uL
KaiC-KaiB 2: 54.5ng/uL
KaiA_LexA-SasA: 39.2ng/uL
Performed Gibson assembly to make KaiC-KaiB. This is the beta.
Ran PCR on plasmids 128 and 204. Had smearing. Only the bands for 128 were present. Purified and isolated them and got a yield of 49.2ng/uL

</SPAN></P><P><SPAN class="ptitle"> 8/18/16</SPAN></P><P><SPAN class="p">
MP:
Transformation successful!
Innoculated 3x 5ml tubes w/ red colonies for glycerol stock and miniprep tomorrow.

Realized that alpha and beta were cut with wrong enzymes. Starting over.
Gibson’d KaiC-KaiB 1 and KaiC-KaiB 2 → “new beta”
Digested alpha and new beta.
Ran alpha and new beta on a gel.




LABEL: ladder, alpha (2/3), beta (4/5).
Not sure why there are double bands. Isolated the top ones. Very faint/hard to see so I cut them and then double checked with another pic. Pretty sure I got the right ones.




Purified and got basically nothing. Saved them as “alpha2 purified empty” and “beta2 purified empty” in -20.
Kevin says there should be a PCR step in there and to increase conc starting material.

CI:
Ran Steve’s LexA-GFP-ADH1 PCR product on a gel. The resulting bands were faint, but present. May have let them run a little too long.

Started a herculase PCR reaction with KaiA-LexA-SasA GBlock 1
2 tubes
Annealing temperature 55 C
I just mixed the reagents. Julia put them in the tubes in the machine. She used Glick lab’s herculase Thermocycler setting
Extracted the LexA-GFP-ADH1 from the gel
Yield: 7.5 ng/uL
Labeled top “LexA GFP ADH1” and side “8-18-16.” I put it in the Constructs box in the top drawer of the -20 freezer.
After doing the gel extraction, I looked at the picture of the gel again and realized that the DNA contained by the bands is way too short. No more than 1 kb. So… this may or may not be LexA-GFP-ADH1.
Story time: Yesterday, Morgan gel purified LexA-GFP-ADH1 for reasons I don’t quite understand (Steve told her to?). Then we ran out of that gel purified stuff, and Steve PCR amplified it. For some reason, he didn’t write about that in his lab journal entry. Today, I ran his PCR on a gel, and I got two bands of something that isn’t our construct, but it doesn’t matter, because we still have 2 whole tubes of LexA-GFP-ADH1 construct I made long ago labeled “GFP 1” and “GFP 3” (see my 8/10/2016 entry). They were gel purified, so they don’t need more purification. They are ready to be digested and ligated.
Digested my LexA-GFP-ADH1 with EcoRI and PstI (for the submission backbone). Labeled it “GFP Const. Digest” on top and “EcoRI PstI” on the side. Left it in the 37 C incubator.
Threw out the tube I labeled “LexA-GFP-ADH1”

JB:
Chinye did 2 PCR reactions on the first KaiASasALexA gblock with F10 and R10, annealing temp 55, extension time 1 min 30 sec, using Herculase and protocol “kd herc”
I ran it on a gel and got some weird bands...ideally we would want one band at 1370 bp but we didn’t get any of that size.

</SPAN></P><P><SPAN class="ptitle"> 8/19/16</SPAN></P><P><SPAN class="p">

MP:
Miniprepped 3x submission backbone using the PureYield system and protocol on the Promega website and stored in Construct #2 box in the -20.
Put the extra cells in the 4deg fridge.

Made 2x glycerol stock of the submission backbone. Put in the -80 in a fridge box that says “glycerol stocks” labeled “4B1” and “4B2” (because they came from well 4B)

Digested yiplac128 with EcoR1 and Pst1
Spin purified, dephosphorlyated, and gel purified yiplac218 (nanodot: 4ng/ul)
Ligated KaiCP to yiplac and submission backbones using Jason protocol. Left overnight in 4deg fridge (no 16deg?)




LABEL: Ladder, Yip128 (one band, correct size!)

SD:
Gibson assembled KaiC-KaiB
Ran a PCR to amplify KaiC-KaiB and KaiA-LexA 1
Left reactions in cold room
CI:
Measured concentrations of digested submission backbone and digested and purified LexA-GFP-ADH1 using the Nanodrop
Backbone: 12.7 ng/uL
LexA-GFP-ADH1 (digested and purified): 6.4 ng/uL
Used elution buffer as the blank for both. Morgan later told me that the submission backbone was dissolved in water, so my measurement may be inaccurate.
Ligated LexA-GFP-ADH1 and submission backbone
Labeled “GFP Const. Submission Backbone Ligation”
25 ng (2 uL) backbone, 75 ng (11.72 uL) construct, 3 uL ligase buffer, 1.5 uL T4 ligase
No water added
Based on iGEM ligation protocol (http://parts.igem.org/Help:Protocols/Ligation), but edited, because the construct solution was too dilute for me to use &lt;3 uL as was suggested

</SPAN></P><P><SPAN class="ptitle"> 8/20/16</SPAN></P><P><SPAN class="p">
MP:

Transformed GFP in sub backbone and KaiCIP in sub and yiplac128. Plates sitting in 37deg.
I forgot to do controls, so doing that now
Had enough yip backbone to do ligation right away (sitting in -20) but ran out of submission backbone, so digested more (5.8ng/ul).
Heat kill was accidentally 100deg instead of 80deg which evaporated some of the digestion. Wanted to use 8.6ul, ended up with ~6ul. Such is life.
Not perfect controls but I think it be ok for our purposes.

</SPAN></P><P><SPAN class="ptitle"> 8/21/16</SPAN></P><P><SPAN class="p">
MP:

All plates have colonies. GFP/sub and KaiCP/yip are fine; very few on KaiCP/sub.
Grew up 6x overnight cultures for each plate. Plan to run diagnostic digest tomorrow, then sequence ones with insert.
The neg controls won’t mean much at this point but I already defrosted the competent cells and feel bad about wasting them so I’ll go ahead anyway.
Left controls for Steve to plate.

SD:
Plated Morgan’s controls
Made Backbone and sequencing primers and sent them to Morgan to purchase
Made amp plates
Ran the PCR from friday on a gel
Everything got stuck in wells
Ran an SOE PCR on KaiC-KaiB and KaiA-LexA using the new 2-step protocol I designed from Kevin’s suggestions
Left to run overnight

</SPAN></P><P><SPAN class="ptitle"> 8/22/16</SPAN></P><P><SPAN class="p">
MP:

Control plates look alright.
Nothing growing on yip--although caveat that Steve messed up so we only have 20ul and 66ul plates. Something might’ve grown on a 200ul plate.
Sub plates *do* have significant growth, but, almost all the colonies are pink/red, whereas the experimental plates had white colonies.
Learned that we need to dephosphorylate sub backbone to prevent self-ligation.
All overnight cultures have growth.
One of the sub colonies is pink! Throwing it out (b/c functional RFP → no insert present).


I’m going to use this notation moving forward:
“A” = KaiCP+Yip128
“B” = KaiCP+Sub
“C” = GFP+Sub
Did a mega miniprep with Julia. High yields.
Now she’s doing a mega diagnostic digest and running a mega gel. Products with two bands will be sent for sequencing.

 

LABEL: Purpose of the diagnostic digest. Two bands = success. One band = no insert present.
JB:
Helped Morgan miniprep the GFP+sub plasmid.
Did mega diagnostic digest with following enzymes:
KaiCP+YIPlac (A) - ZraI + XmaI
KaiCP+sub (B) - NspI + HpaI
GFP+sub (C) - ZraI + BsaI
Control was for GFP+sub
Here’s the gel:

We messed up and didn’t realize the importance of controls for each backbone/insert/enzyme combo and also used weird enzymes so we will be redoing tomorrow with controls.

</SPAN></P><P><SPAN class="ptitle"> 8/23/16</SPAN></P><P><SPAN class="p">
MP:
Gibson’d KaiALexASasAKaiA 1,2&amp;3
Asked Miranda to leave in the -20
JB:
Redid diagnostic digest with enyzmes that work better from Jason’s experience and appropriate controls (empty circular backbone + the enzymes, which Fernando apparently thinks is a weird control. And then he said he never does controls for diagnostic digests).
KaiCP+YIPlac (A) - AflII + AfeI
KaiCP+sub (B) - MscI + AfeI
GFP+sub (C) - SphI + PvuII

Got some double bands for A and B but they are strange in size. And B6 is really weird. Band sizes are suspicious but we’ll transform A1, 3, 5 and B1, 3, 5 and send them for sequencing, since there’s not really a better option at this point
C is really strange, too, but it’s because the wrong buffer was used. We chose the enzymes/buffers from Snapgene, which said Cutsmart would be fine. But after these weird results I checked the poster on the fridge and it says “*may exhibit star activity” next to Cutsmart. So will redo with 2.1

CI:
Digested KaiA-LexA-SasA
It was from Steve’s latest attempt
Digested 6.1 uL KaiA-LexA-SasA with BglII
Labeled “KaiA Digest” on cap and “KaiA LexA SasA BglII Digest” on side. Placed in Constructs #2 box in -20 freezer.
Purified digested KaiA-LexA-SasA with nucleotide removal kit
Used 25 uL of EB to elute
Yield: 6.2 uL
Labeled “KaiA Dig Pur” on cap and “KaiA-LexA-SasA Digested Purified” on side. Placed in Constructs #2 box in -20 freezer.
Had issues quantifying with the Kovar lab Nanodrop originally. Kept getting negative, near-zero readings like Miranda did with KaiCu. Discovered that the EB that I used to elute the DNA registers differently on the Nanodrop than the EB by the Nanodrop, even though the containers are identical. Did someone do something special to increase yield? Is it contaminated?
Ligated Tef1 promoter and KaiA-LexA-SasA with Quick Ligase
8.0 uL KaiA-LexA-SasA, 10.2 uL Tef1; no water added
Labeled “KaiA Tef1 Lig” on cap and “KaiA-LexA-SasA Tef1 Ligation” on side. Placed in Constructs #2 box in -20 freezer.
We’re almost out of digested Tef1!

</SPAN></P><P><SPAN class="ptitle"> 8/24/16</SPAN></P><P><SPAN class="p">
MP:
PCR’d KaiALexASasAKaiA 123 w/ F12 and R14 w/ herculase
Gel purified above
Realized that the F12 primer binds in two places, on both of the KaiA’s, and in fact strongly prefers the second one, so PCR mostly amplified KaiA segment. Need to re-think strategy.
New plan: Amplify kaia-lexa-sasa-kaia part 1 (first of the 3 gblocks in the construct), digest, ligate to promoter, and then gibson to part 2 and 3 using F2* (binds to promoter-kaia rather than just kaia)
Transformed A1, A3, A5, B1, B3, and B5 with Julia. Left in 37 o/n.


LABEL: Ladder, 3x KaiALexASasAKaiA; larger band is probably KaiA + smaller bands likely unspecific binding (see explanation above). 

CI:
PCR amplified promoter-KaiA-LexA-SasA
3 reactions; 3 uL unpurified ligation added to each
Annealing temperature: 72 degrees C
Accidentally set annealing time to 45 sec per cycle; mean to do 1.5 min (45 sec/kbp, because Kevin said 1 min/kbp was too long)
Ran promoter-KaiA-LexA-SasA on a gel

Extracted promoter-KaiA-LexA-SasA from gel 
Yield: 19.4 ng/uL

JB:
Redid diagnostic digest for C (GFP+sub, with PvuII and SphI in buffer 2.1) and got this:


Got two bands but the bottom one is way too small. And the control got cut many times? Not sure how to proceed here. But it looks like 1-5 have the same stuff so maybe we can get it sequenced at least to solve the mystery (but does it even have the insert??)
</SPAN></P><P><SPAN class="ptitle"> 8/25/16</SPAN></P><P><SPAN class="p">
MP:
The transformation colonies totally overgrew all of the plates
Maybe because we added 3ul instead of 1ul of DNA?
It was impossible to select single colonies for o/n cultures, but given that we already know they’re correct from diagnostic digest, I think thats ok
Made overnight cultures of A1,3,5 and B1,3,5. 
Digested Tef1 with BamH1, KaiA-LexA-SasA-KaiA (hencefore “KaiA*”) with Bgl2, and GFP construct and Yip211 backbone with EcoR1 and Pst1. 
Spin purified above. Remainder is in constructs #1 box labeled “digested and purified.”
Ligated GFP and Yip211 and ligated KaiA* and Tef1. Left o/n in 4deg fridge.
Did not have enough yip211 to do a control ligation… and also forgot to dephosphorylate backbone. Whoops.
Transformed C1,3,&amp;5. Plated on chloroamp / left 37deg o/n.
JB:
Purified Miranda’s KaiCU digest but got negative readings on the nanodrop. Put it in the constructs 2 box, labelled “KaiCu digest purified” on the cap
Gel extracted Yiplac 211 PCR product. 

Size should be ~4300 bp. 
13.2 ng/uL
Morgan used most of this to digest, so I set up another PCR on what was left with primers F211, R211; program KD herc; polymerase herculase; anneal temp 55; extension time 6 min
Gel extracted PCRs of the 3 KaiASasALexAKaiA gblocks. They are in the iGEM 2016 constructs box, labelled KSLK 1, KSLK 2, and KSLK 3 on the caps

Good news is we got the general pattern we expected: 1st 2 gblocks are similar size, and the 3rd one is smaller
Bad news is those first two gblocks should both be around ~1400, but they look closer to 1000 to me. The 3rd gblock should be around 900, and that band could be ~900 but maybe it is smaller. UV lamp broke today so our picture was terrible and we have to go on 9th floor to take pictures of gels now
Gel extracted all three. But Chinye was helping me do that and we got some tubes mixed up so gblocks 2 and 3 got mixed up. I gave gblock 1 to Morgan to digest, and will be running all three on a gel again today. We can check sizes again and know which is which
Reran ~80-100 ng of each gblock on a gel to check which one is which (see above) and got everything fixed. Couldn’t take a picture but I looked at it over UV light on the 9th floor and sizes were actually perfect yay our gblocks are good

CI:
Digested promoter-KaiA-LexA-SasA with BamHI in 37 degree incubator
After 2 hrs, Julia took the sample out of the 37 degree incubator and put it in the cold room.
ATP assay stuff
Took out the yeast pellets I made 7/29/16 and resuspended each one in 1 mL of starvation YPAUD. 
Each pellet was made from 0.5 mL of 0.2 OD yeast, which is about 10^6 cells. Adding 1 mL YPAUD turned each pellet into a 10^3 cell/uL culture
Followed the ATP/ADP ratio kit protocol (in this folder: https://drive.google.com/drive/folders/0Bz_3wFx2wXHneWhFd2ItOEs1Sm8) in order to make an ATP reagent mixture and an ADP reagent mixture. Multiplied the values in the tables in the protocol by 16, for 16 samples.
Took the reagent mixtures and the yeast samples over to Rust lab with Morgan and went through the protocol with Justin.
Added ATP reagent mix to samples, incubated at room temp., measured, and got very low luminescence readings. I had two samples from each time point, so after measuring, I still had one sample from each time point left over.


Made standard ATP sample from ATP reagent mix and 200 mM ATP Justin had and measured the luminescence. It was about 200, which is bad. It’s supposed to be really high.
Went back to Glick lab, grabbed the substrate, cosubstrate, and ATP enzyme from the kit, and brought them back to Rust lab
Carefully re-made the ATP reagent mixture using the same protocol (only multiplied the amounts by 3 this time) while Justin watched. He emphasized mixing by pipeting up and down. And avoiding making air bubbles.
Luminescence reading for new ATP reagent mix was 1,618
Added 1 uL of 200 mM ATP to 90 uL of the new ATP reagent mix. Got a luminescence reading of 7.33 x 10^8, which Justin said was expected. This meant that the new ATP reagent mix was good and the old one was bad.
Took 1 uL of yeast from the 4 hr control sample and added it to 90 uL of the new ATP reagent mix . Incubated for a minute and measured the luminescence. Got a reading luminescence reading of 463,656 (measurement RLUa according to the protocol). Justin said this was good.
Waited 10 min and measured luminescence again (measurement RLUb). The reading was 440,623.
Added 5 uL of the ADP reagent mix I made in Glick lab to the same tube I’d added ATP reagent mix to,  vortexed, and measured luminescence (measurement RLUc). The reading was 1,322,355.
Calculated ATP/(ADP+ATP), which is RLUa/[(RLUc-RLUb)+RLUa)]
Derived from equation in protocol, which is for ATP/ADP
ATP/(ADP+ATP) was 0.345. The normal value for yeast on glucose is ~0.9. Ours was so low because the yeast were sitting in starvation YPAUD for over an hour. Yeast in starvation YPAUD tend to have a value of ~0.4.
Went back to Glick lab
Threw out the ATP reagent mix, ADP reagent mix, yeast samples, ATP standard, and 200 mM ATP, because all of this was for a trial run, and luminescence assays are very time sensitive
Put the cosubstrate, substrate, and ATP enzyme back in the kit’s red box and put the box back in the -20 freezer (3rd drawer?)
IMPORTANT: Next person doing ADP/ATP assays should do a trial run or two before doing the assay on important things. Also, these things are SUPER time sensitive (luminescence fades with time), so use a timer.
Took Tef1-KaiA-LexA-SasA digest out of cold room and spin purified with Qiagen nucleotide removal kit
Labeled “Tef1 KaiA Dig Pur” on cap and “Tef1-KaiA-LexA-SasA Digested Purified” on side. Placed in Constructs #2 box in -20 freezer.
Didn’t quantify with Nanodrop, because it was already like 6:30 pm. Someone should do that.
Flying to Georgia tomorrow. Bye! The next person doing ATP/ADP assay stuff needs to message me first!

</SPAN></P><P><SPAN class="ptitle"> 8/26/16</SPAN></P><P><SPAN class="p">
MP:
C1/3/5 transformations successful.
Made overnight cultures.
B1/3/5 overnight cultures successful. 
Made glycerol stocks and miniprepped. Handed off to Julia to deal with sequencing.
A1/3/5, however, did not grow.
Talked to Jason. He explained that because we had lawns of bacteria, colonies without amp resistence were able to grow around the ones that were secreting ampR enzymes.
Streaked out some of the lawn on new plates today (from Glick fridge) to see if any individual colonies pop up.
If that fails, will re-do the transformation with less DNA on different plates in case 1. transformation efficiency w/ miniprep is just too good 2. plates didn’t have right kind/amount of antibiotic.

 
Label: The consequences of lawns.



GFP Yip211 is ready for transformation but waiting for other stuff so don’t waste cells (in -20).
Made 1:10 primer dilutions from the original IDT tubes.
PCRed promoter-kaiA*.

JB:
Organized sequencing primers in a box labelled “igem 2016 sequencing primers”
Sent off KaiCP+sub (B) for sequencing!! (From Morgan’s miniprep above)
Used primers S5-S10 + vr + vf2 (Got vr and vf2 from 2015’s primer box. They’re now in our sequencing primers box)
Gel extracted Morgan’s promoter-kaiA*. Also ran Yiplac211 PCR on the same gel, but it got stuck at the wells:
Extracted the very top bands for promoter-kaiA*
Yield: 41.3 ng/ul. I put it in the 1st constructs box (top is labelled “promoter-KaiA* in blue marker)

Digested 3rd KaiA* gblock with BamHI and digested more ADH1 terminator with BglII
Super low yields. 0.1 ng/uL for terminator and 0.2 ng/uL for kaiA* gblock 3. These are pretty useless so I threw them away. I set up a PCR for kaiA gblock 3 so they can be digested tomorrow (using herculase, KD herc, anneal temp 55, extension time 1:45)</SPAN></P><P><SPAN class="ptitle"> 8/27/16</SPAN></P><P><SPAN class="p">
MP:

Made glycerol stocks for C1/3/5 and miniprepped with good yields. OK for sequencing.
Realized that I did the A1/3/5 overnight culture that didn’t work with the wrong antibiotic
This makes so much more sense
Re-did A1/3/5 overnight from the re-streak plates on the right antibiotic (amp, not kan)
Gel purified KaiA3* PCR from yesterday
Looks like it worked. One strong band ~1000bp, as expected.



Gel extraction: very low yield, 4.2ng/ul. There isn’t enough to do a good digestion (only 100-120ng).
Ran 3x PCRs on KaiA* gblock 3.  </SPAN></P><P><SPAN class="ptitle"> 8/28/16</SPAN></P><P><SPAN class="p">
MP:

A1/3/5 overnight--no growth again!
???
KaiA* gblock 3 PCRs worked. Gel purified. Some smearing.

Digested KaiA* gblock 3 with bamh1 and adh1 with bgl2. Left in -20.</SPAN></P><P><SPAN class="ptitle"> 8/29/16</SPAN></P><P><SPAN class="p">
JB:
Purified Morgan’s KaiA*3 digest and ADH1 digest and got
KaiA*3: 27.9 ng/uL
Checked this twice because it seemed too high and got same reading both times. 
ADH1: 1.8 ng/uL
At first I got 0.5 ng/uL, second time I got 1.0, then next two times in a row I got 1.8
Sent off C1, C3, C5 for sequencing


MP:

Performed two KaiA*3-terminator ligations, one using 1.7ul of KaiA*3 (amount you would need based on quantification) and another with 3ul (slightly more). 
NB: Forgot to purify ligation
Actually now that I think about it, the max amount we should have gotten for KaiA*3 was 6ng/ul. So realisitically I only added 18ng (probably less) to the second ligation.
I guess we’ll just see what happens but probably have to re-do this.
Set up 2x PCRs for the screwy ligations. Gel purified.
Results are terrible. Starting over. 
First re-checked nanodot for KaiA*3/adh1 digests. Got different numbers but same general range. Assuming something went wrong at the digest/purification stage. Re-did digests and left overnight.

Made a new overnight culture for A series using Jason’s lb/amp media.

New primer is here! Resuspended and made 1:10 dilution.
Set up 2x reactions for KaiA*2 overhang (henceforth “KaiA*2 OH”) PCR and 1x reaction for KaiA*2 regular PCR since I used up the last batch.

</SPAN></P><P><SPAN class="ptitle"> 8/30/16</SPAN></P><P><SPAN class="p">
MP:
A series didn’t grow, again. 
Apparently our lab and Glick lab use different color codes for antibiotic plates.
Re-streaked original lawn plates on actual LB/amp plates (red and blue stripes). Sitting in 37deg.
Will re-transform A series if that doesn’t work.

Spin purified digests
Got low yields but Jason says to go ahead with ligation anyway

Gel purified PCRs
Got sort of weird but probably ok bands



Isolated from gel and got good yields.
Digested KaiA*2 OH and ADHI with bamH1 and Bgl2 respectively.
Spin purified.

Did exhausting mega colony PCR fest (10 samples for x and z, 8 for y)
KaiCP + Yip128 = “X”
KaiCP + sub = “Y”
GFP + sub = “Z”

</SPAN></P><P><SPAN class="ptitle"> 8/31/16</SPAN></P><P><SPAN class="p">
JB:
Made gel for Morgan to run her mega colony PCRs on
Planned out on Snapgene the cloning for getting KaiA and KaiC from Justin and into our Yiplac backbones, and ordered primer “KaiC.KpnI.FOR” and “KaiC.ATG.EcoRI.FOR”
Digested yiplac211 and submission backbone with EcoRI-HF and PstI, and KaiCP and GFP inserts with EcoRI-HF and PstI, and TefI with BamHI
PCRed yiplac 211 and 128
Gibson assembled KaiCu gblocks

MP:

Ran colony PCRs on a gel. All came up negative. 



LABEL: ladder, x series, control, y series



LABEL: ladder, Z series, control (faint but present)

Re-streak plates were done incorrectly. Made new re-streak plates, but going to retransform A based on CPCR results.

Ligated KaiA*2 OH to ADHI and KaiA*3 to ADHI
PCRed both products
First PCR results indicated that annealing temp or timing was off (lots of smearing, below). Re-did PCR with phusion and better attention to details.</SPAN></P><P><SPAN class="ptitle"> 9/1/16</SPAN></P><P><SPAN class="p">
MP:

A series re-streak plates look ok but not doing a CPCR (yet if ever) since the B’s and C’s were all negative and we’re re-doing it anyway.
Ran yesterday’s PCRs on a gel. It was bad again.
Justin and Kasey think that the PCR isn’t working because there may only be a small amount of the desired product in the ligation mixture and the individual gblocks are competing for primers.



Going to try the following: do several ligations in parallel, combine them, and run the results on a gel without PCRing first. If correct band is visible, use that for gibson and then PCR.
Checked gel. Hard to see in picture but there are 3 bands in each well. I assume they correspond to the ligation and the two individual bits. Going to extract the longest ones.

Transformed yip+211 and sub+kaicp. Gave to Julia to plate.

JB:
PCRed the gibson product for KaiCu with F2* and R3 and phusion
Gel showed nothing except some bands at 250 bp
Quantified the digests from yesterday
yiplac211: 2.1 ng/ul
Submission backbone: 1.7 ng/ul
GFP construct: 0.1
KaiCP: 5.8
TEF1: 7.7
Dephosphorylated submission backbone and yiplac211, gave to Morgan to use for ligation/transformation
Gel extracted yiplac128 and 211 backbones that I PCRed yesterday

128: 42.5 ng/ul
211: 67.8 ng/ul
Made CAM plates
Plated Morgan’s transformations (Yip211 control, 211 + gfp, submission control, sub + kaicp)
PCRed KaiA from Justin’s plasmid
Phusion, anneal temp 72, extension time 30 sec

</SPAN></P><P><SPAN class="ptitle"> 9/2/16</SPAN></P><P><SPAN class="p">
MP:

Gibsoned then PCR’d KaiA* groups
Digested 128 backbone and sub backbone
Spin purified backbones. Came up empty. Will re-digest.

Gibsoned and then PCR’d KaiCP1/2.

JB:
Don’t have a lab entry for this date for some reason…
But I remember I wasn’t very productive this day
I think the KaiA PCR didn’t work/nothing on gel

</SPAN></P><P><SPAN class="ptitle"> 9/3/16</SPAN></P><P><SPAN class="p">
MP:

Gel purified all the PCRs
None of them are the right size
Going back a few steps again again again

Digested sub, 128, KaiA*2OH and ADH1
Spin purified. Low but ok yields.
Ligated KaiA*2OH and ADH1
We need a new primer to PCR this properly
Gibsoned then PCR’d P-K1-K2OH-T

PCR’d kaicp gblocks 1 and 2

Made new chloroamp and amp plates

</SPAN></P><P><SPAN class="ptitle"> 9/4/16</SPAN></P><P><SPAN class="p">
MP:

Ran PCRs on a gel
Kaicp gblock 1 didn't work. It amplified something really small, ~100-200bp.
Kaicp gblock 2 technically worked. There was some product of the right size (~2000bp), although the bands weren't very strong.
Isolated the bands
P-K1-K2OH-T used the wrong primers so accidentally just amplified the promoter. 
Isolated the bands
Re did the PCR with the right primers. 
Gave to Julia to run on a gel.

Tried runing 150ng kaicp1 by itself on a gel to purify before PCR
Totally blank. See lane #2. Gotta re-order.


PCRd yip128
Ran on gel
There was nothing? See above (lanes 3/4). We ended up having a good supply so didn’t re-do.

Plated sub backbone from glycerol stock. Left in 37deg.
It work!

Transformed GFP/yip211 and GFP/sub. Plated and left in 37deg.
Controls: lots of colonies. Experimental: 0 colonies. Probably something wrong witht the insert. Going to check those.

PCRed P-GFPr-T to make more just in case those do badly
Apparently did this with the wrong primers so no wonder it didn’t work
GFP is preceded by lexa operator, not tef1 promoter

JB:
PCRed KaiC from Justin’s plasmid. Herculase, anneal temp 55, extension time 2:15 -- this worked well
Ran the PCRed KaiC (1560 bp) on gel with Morgan’s GFP construct PCR. The GFP PCRs made some weird tiny products...

But I extracted KaiC and got 26.4 ng/uL
Also ran Morgan’s P-K1-K2OH-T on a gel and got this (second picture has somewhat better ladder). Extracted the top band (if you can call it that) because it was right size (~3500). Got low yield 2.1 ng/uL. PCRed the extracted product again with same conditions and primers to get more specific product</SPAN></P><P><SPAN class="ptitle"> 9/5/16</SPAN></P><P><SPAN class="p">
MP:

Realized the deadly R3 primer mistake.
Ordered a new R3* primer.
Going to do a PCR with F15 and R3* on both GFP and KLS (original and PCR1) with phusion at 67 degrees when it arrives.

KLS PCR2 didn’t work. Looks like possible primer dimers. Kasey recommends using better template. ^

Vortexing revealed that the missing KaiCP1 did have something in the container. Ran a PCR.

Made o/n cultures for sub backbone.  

JB:
PCRed KaiA from Justin’s plasmid
Pfu turbo, anneal temp 53, extension time 1 min
Ran PCRs of KaiA and Morgan’s KLS on gel:

Lanes in order are KaiA, KaiA control (no template), KLS, KLS, KLS control (no template). KLS didn’t work :(
Gel extracted KaiA and got 11.8 ng/uL. Did only one reaction of this PCR because I wanted to try out a new polyermase/protocol before using it on a ton of reactions. Set up 5 more reactions from this product to get more KaiA, which we need a lot of to digest with KpnI and BglII (this has to be done in two digestion reactions because of incompatible buffers, and the low yields of our purification steps requires us to start with a lot of dna). Will digest a ton of PCRed KaiA tomorrow
(reaction conditions were Pfu turbo, anneal temp 55, extension time 1 min -- this worked very well)
Digested KaiC with KpnI-HF and purified it. Got 5.1 ng/uL. Think this is low for another digestion and purification but set up the BglII digestion anyway since there’s not much else to do with it. 
Digested Yiplac128 (from Jason that has promoter and terminator) with KpnI-HF and BamHI-HF. Will ligate KaiC in when it’s ready
Set up 5 PCR reactions for KaiC to get a ton of it (like KaiA) in case the KaiC digestion today ends up having too low yield. Reaction conditions Pfu turbo, anneal temp 61, extension time 1:50. (see next day -- this one didn’t work so don’t use these reaction conditions)
KaiC digestion #2 with BglII ended up having yield 2.1 ng/ul
Set up ligation with double digested Jason’s Yip128 and KaiC, with control ligation of just double digested Jason’s Yip128. Left in 16 deg. overnight so should be ready for transformation tomorrow
Didn’t dephosphorylate backbone because I keep hearing it’s not necessary
Discovered weird problem with primer R3 today. The overhang that attaches the suffix to our terminator was wrong, so it was attaching some weird suffix-like sequence without the right cutsites to all our terminators. One explanation among hundreds of possible ones for why our digestions/ligations into backbones haven’t been making any useful colonies. Ordered new primers with the suffix overhang fixed. And we will PCR the inserts that we’ve been PCRing with the wrong primer with this new primer</SPAN></P><P><SPAN class="ptitle"> 9/6/16</SPAN></P><P><SPAN class="p">
MP:

Miniprepped 4B samples. Got lots of submission backbone. We should be set through the end of the summer on that one but the plate’s in the 4deg if we need it.

Ran vortex KaiCP1 PCR on a gel. Got terrible results.



As you can see, lots of different things got amplified, and none of them well. Isolated the top band and left it in QG buffer in the -20 (“kaicp1”) and then offiically gave up on it.

Ordered KaiCP from Genscript
Ordered KaiCP → KaiCU quickchange primers from IDT

JB:
Ran PCRs of KaiC (top) and KaiA (bottom). KaiC didn’t work but that’s ok because hopefully the ligation into yip128 worked and we won’t be needing more KaiC

Gel extracted KaiA bands
70.5 ng/uL
Digested KaiA with EcoRI-HF and spin column purified
18.0 ng/uL
Digested Jason’s yip204 with EcoRI-HF and BamHI-HF and purified
62.6 ng/uL
Dephosphorylated digested yip204 and gel purified it
Digested KaiA a second time with BglII. Purified and got negative readings on nanodrop :(
I proceeded to ligate the negative-reading KaiA digest to the dephsophorylated yip204 because what else would i do with the tube. Also set up a control ligation with just dephosphorylated yip204. Left it in 16 deg on 9th floor overnight

</SPAN></P><P><SPAN class="ptitle"> 9/7/16</SPAN></P><P><SPAN class="p">
MP:

Transformed kaia and kaic plus controls

</SPAN></P><P><SPAN class="ptitle"> 9/8/16</SPAN></P><P><SPAN class="p">
MP:

PCR’d KLS original and PCR1 and GFP
F15 and R3*, phusion at 67 degrees, 1 min 30 extension time (compromise between &lt;1min and 1min45)
Gel purified
KLS didn’t work
Ordered from Genscript
Also re-starting
GFP worked! Probably? Band looks vaguely correct. It’s a little small but I guess we’ll see post-transforming.



Got ~2~ full colonies for KaiA transformation, none on control
Grew overnight cultures.
Got a bunch for KaiC but the control plate also has a lot
Will CPCR a bunch of them even though they’re probably wrong
Going to re-do both just in case
PCR’d 5x KaiA from old PCR, PCR’d one round of KaiC from old PCR (barely 1ul left) and 4x from genomic DNA
Got nothing at all for KaiA and big beautiful bands for KaiC
Maybe I forgot something? Who knows. Will redo. 
Didn’t take a pic because I’m a rebel without a patience
Gel pur’d KaiCs and GFPs
Nanodropped
Digested KaiC with KpnI-HF and digested GFP and Yip211 with EcoR1-HF and PST1. Left overnight.
</SPAN></P><P><SPAN class="ptitle"> 9/9/16</SPAN></P><P><SPAN class="p">
MP:

KaiA transformation overnight cultures worked.
Miniprepped.

PCR’d KaiA and tried K2OH-T (even though Steve thinks it won’t work)

Spin purified digests (KaiC, GFP and Yip211)

Dephosphorylated Yip211 and 128
Gel purified K2OH-T and KaiA PCRs and gel purified dephospho 128 and 211
Behold, everything worked


Top: ladder, 3x K2OH-T, 128, 211 (too faint to see but a band was there)
Bottom: 4x KaiA, ladder</SPAN></P><P><SPAN class="ptitle"> 9/11/16</SPAN></P><P><SPAN class="p">
MP:

Gibsoned 2x P-K1 and K2-OH
PCR’d PK1K2OH (used 30/40ul, there’s 10ul left)
Herculase 55deg, 3 min and 25 sec

Digested 400ng KaiA
Realized that I accidentally used EcoRI-HF and BamHI-HF because those are the backbone cutsites, not the KaiA overhang cutsites. 
There is a Bamh1 cutsite in KaiA, so threw that out.
Re-did it, but then also realized that it was silly to do this before I checked the first round of transformations. Left in -20.

Picked 10 KaiC transformation colonies and ran a CPCR
Rest of the samples are chilling in the storage room
Nanodropped KaiA minipreps; good yields
Diagnostic digested KaiA transformations 1 and 2 with BamH1 and EcoRV in 3.1 buffer.

Ran PK1K2OH PCR and KaiA diag digests on a gel with good results



Top: ladder, 3x PK1K2OHT (exp ~3500bp)
Bottom: ladder, KaiA trans colony 1, KaiA trans colony 2 (exp ~1200 and ~3500bp)

PK1K2OHT has a few bands, like last time, including two large very similarly sized bands at the top (too close together to see in the pic). I ran for another 10 min and then separated the top/bottom ones near 3500bp separately. I’ll run following subsequent steps in duplicate. Left in QG in the -20.
KaiA 2 looks like a hit. Going to do an overnight culture tonight just to make more of it (innoculated from the original o/n) and send for sequencing tomorrow.

Ran yip211 on a gel. Really strong bands. Left in QG in the -20.



MH
PCRed yip211

</SPAN></P><P><SPAN class="ptitle"> 9/12/16</SPAN></P><P><SPAN class="p">
MP:

Sent KaiA for sequencing

Ran KaiC CPCRs on a gel. All of them had a band, some are fainter than others. Made overnight cultures of 5, 8, and 10 to miniprep and send for sequencing (darkest bands).



Digested Yip211 and KLS top/bottom with Eco/Pst
Spin purified
Dephosphorylated yip211, ran on gel, left in QG in the -20
Ligated KLS top/bottom and yip128. Left in 4deg.

</SPAN></P><P><SPAN class="ptitle"> 9/13/16</SPAN></P><P><SPAN class="p">
MP:

Sent KaiC transformation samples 5, 8, and 10 for sequencing

KaiA transformation colony 2 sequencing results confirmed positive!

Ran KLS top/bottom on a gel. Got lots of smearing/couldn’t decipher results. Will proceed with both ligations and try to purify the material stuck in the wells and re-run it to see if there’s anything to salvage.

Gel purified KLS top/bottom and yip211.
Put KLS top/bottom away for now--unimportant unless transformation fails and we need more.

Ligated yip211/gfp construct.

Transformed yip211/gfpc, yip128/KLS top, yip128/KLS bottom, and 128 and 211 controls

Made an overnight WT yeast culture for KaiA/KaiC transformations using Jason’s stock; 200ul in 25ml of YPAUD. Left in 30deg shaker.
Linearized KaiA/KaiC inserts by cutting KaiA/204 with EcoRV and KaiC/128 with Afl2

Took Miranda’s digests out of 37deg at 4:15 and put in the in -20.</SPAN></P><P><SPAN class="ptitle"> 9/14/16</SPAN></P><P><SPAN class="p">
MP:

Checked plates. There were some but very few colonies on all experimental plates and a surprisingly high number on the yip211 control plate compared to almost none on the yip218 control plate.
Ran a colony PCR on 3 GFP, 7 KLS top, and 8 KLS bottom (basically all available colonies).
Phusion, R15/F15, 71deg, 1min ext
Gel purified
Results were strange. All samples had a band, but in identical places. KLS should be much bigger than GFP. 



Basically going to assume these are all negatives because KLS is weird and GFP has more colonies on control than on experimental.

Digested new yip211. Left in -20.

Put 1ml overnight yeast culture in 24ml YPAUD and set timer for 2 hours
Jason says this is when the yeast will enter log phase
Spin purified the digests from yesterday
Transformed KaiA2 and KaiC10 into yeast
Forgot controls initially but then re-did them separately from step 8 onwards with same cells
Ordered CPCR primers to check them
KaiA.Rev and KaiC.Rev
Made a yeast starter culture from the overnight. Left in cold room.
Sequencing for KaiC’s came back negative for all 3 samples I sent for sequencing. 
Not sure what got amplified in the CPCR? It actually looks very similar to the weird one from today.
Should probably diagnostic digest from now on.
Asked Jason for new yip128 aliquot and digested with KpnI-HF and BamHI overnight.

According to my notes, I digested KaiC with KpnI-HF and purified it a while ago. Digested those tubes with Bgl2 overnight.

Kasey said that she used 5mM deoxyglucose w/ yeast before and Glick thinks that seems reasonable, so going to try 10, 5, 1, and 0.5mM concentrations for the starvation protocols
Since we need 5ml of starvation culture/sample, and I want to run this in triplicate, I made 25ml of each deoxyglucose concentration
10 millimoles / liter = x millimoles / 0.025 liters → x = 0.250 mmol
5 millimoles / liter = x millimoles / 0.025 liters → x = 0.125 mmol
1 millimole / liter = x millimoles / 0.025 liters → x = 0.0250 mmol
0.5 millimoles / liter = x millimoles / 0.025 liters → x = 0.0125 mmol
MW = 164.16 g/mol = 0.1642 g/mmol
0.1642 g/mmol * 0.250 mmol = 0.04105 grams deoxy
0.1642 g/mmol * 0.125 mmol = 0.02053 grams deoxy
0.1642 g/mmol * 0.0250 mmol = 0.004105 grams deoxy
0.1642 g/mmol * 0.0125 mmol = 0.002053 grams deoxy</SPAN></P><P><SPAN class="ptitle"> 9/15/16</SPAN></P><P><SPAN class="p">
MP:

Spin purified KaiC, Yip128, and Yip211 digests from yesterday
Desphosphorylated backbones
Gel purified backbones
Nanodropped various samples
Ligated GFP/yip211, KLS top/128, KLS bot/128, and KaiC/J-128 in 4deg fridge overnight

Made 2x glycerol stocks from KaiA transformation colony 2 overnight

Updated deoxy protocols

Miniprepped KLS T123 B456

</SPAN></P><P><SPAN class="ptitle"> 9/16/16</SPAN></P><P><SPAN class="p">
MP:

Quick ligase’d GFPc and sub backbone
Last minute addition
Didn’t bother with KLS t/b because I assume they’re bad.
Transformed GFP/sub, GFP/yip211, KLS top/128, KLS bot/128, and KaiC/J-128. Sitting in the 37deg.
Used all of the culture for all plates except did 200ul and 20ul plates for Gfp/sub and sub control b/c the sub backbone wasn’t dephosphorylated so I expect more background noise for those.

Made yeast overnight culture from starter culture for deoxyglucose assays.

Checked yeast plates––no colonies. 
Re did transformation with Jason’s supervision and pos and neg controls.

</SPAN></P><P><SPAN class="ptitle"> 9/17/16</SPAN></P><P><SPAN class="p">
MP:

GFP/SUB: I don’t think the sub backbone was digested correctly because almost, but not all, of the colonies were red. Will grow up white colonies from exp plates but not expecting much. Need to re-digest and then definitely dephosphorylate before ligaton.


KAIC/J128: Very successful. Very little background noise and overflowing plate but individual colonies still avail for overnights.



KLS/128 &amp; GFP/YIP: Experimental and controls look similar, so doesn’t tell us much other than to have low expectations.





Deoxyglucose Assay:
(ODs might be off since I only blanked with starvation media?)
</SPAN></P><P><SPAN class="ptitle"> 9/18/16</SPAN></P><P><SPAN class="p">
MP:

Checked KaiA transformation
Pos control: lots of colonies
KaiA: a few colonies!
Neg control: a few colonies? Jason says that’s weird but fine
Made 8x overnight cultures

Miniprepped yesterday’s overnights, with the exception of the GFP/sub group which were all red (ie backbone never got digested)
Good yields
Diagnostic digests:

KLS t/b: 
aat2 / econ1 in cutsmart
exp bands: ~2900, ~4700bp

kaic/j128:
bgl1 / pst1 in 3.1
exp bands: ~2200, ~4200

gfp/yip211:
ecorv / bgl2 in 3.1
exp bands: ~2500 and ~2900 (unfortunately didn’t have the snapgene file when I made the digest so the bands will be similar sizes)

KaiC looks correct. KLS t/b cut *something* but the bands aren’t the right size, and the top bands are all slightly different sizes, which isn’t a good sign. GFP is ambiguous. The bands are too close together to see if they got cut. 
Abandoning KLS.
Going to repeat GFP digest and also sequence the insert.

</SPAN></P><P><SPAN class="ptitle"> 9/19/16</SPAN></P><P><SPAN class="p">
MP:

Ran 6x PCRs for GFP 3 tube (should be full insert) with F15 and R3*, phusion at 67 degrees.
Ran on gel. Some nonsense but some strong bands in the correct spot. Isolated them.
Accidentally lost two of them due to dumb pipetting. 


Ladder, 6xGFP (top and bottom)

Did deoxyglucose assay and everything went horribly (see excel sheet for data).
Need to troubleshoot.

Made overnight cultures: WT yeast and KaiA yeast
Going to transform KaiC in tomorrow

Sent KaiC samples 1/2/3 for sequencing
Sent GFP construct linear fragment for sequencing

Did KaiA yeast CPCRs

</SPAN></P><P><SPAN class="ptitle"> 9/20/16</SPAN></P><P><SPAN class="p">
MP:

Re-did GFP/yip211 diagnostic digests with better enzyme pair. Still nothing.



Used KaiA yeast overnights to make glycerol stocks
.5mL 15% glycerol and .5mL culture
Also left 2mL of it in cold room as starter cultures
Note to self: throw these out

Put 1mL of KaiA and WT overnights in 24mL YPAUD and set timer for 2 hours for transformation protocol

Took Miranda’s YPAUDs out of the autoclave; labeled and stored.

Ran yesterday’s yeast KaiA CPCRs on a gel
Revealed that only sample 5 actually has the KaiA insert, so the ones I grew up are useless


Left WT sample in LiAc and grew a new culture with KaiA sample 5 from 11am to 6pm
Proceeded with KaiC transformation into WT and KaiA strains

</SPAN></P><P><SPAN class="ptitle"> 9/21/16</SPAN></P><P><SPAN class="p">
MP:
GFP sequencing came back negative
Ordered rush from Genscript
KaiC sequencing came back positive!
Yay good

Asked Miranda to make KaiA S5 glycerol stocks and starter culture

</SPAN></P><P><SPAN class="ptitle"> 9/22/16</SPAN></P><P><SPAN class="p">
MP:

KaiA/KaiC and KaiC yeast plates both have colonies!
Made 7x and 8x overnights respectively and put plates back in the incubator

Found KaiA Tr 2 e.coli from 9/11 entry and made a new overnight (to make glycerol stocks later)
NB: LB and lb/amp bottles were contaminated; threw them out
Can’t make KaiC e.coli b/c I minipreppped all of the culture and that would be too much effort. Will just settle for KaiC yeast.

</SPAN></P><P><SPAN class="ptitle"> 9/23/16</SPAN></P><P><SPAN class="p">
MP:

KaiA/KaiC overnight cultures grew a ton but KaiC barely grew/didn’t. Not sure why. Identical conditions.
Did first half of KaiA/KaiC western blot with Justin
Asked Miranda to make KaiA e.coli stocks and WT overnight culture

</SPAN></P><P><SPAN class="ptitle"> 9/26/16</SPAN></P><P><SPAN class="p">
MP:

Completed KaiA/KaiC western blot. Success! Pics of protein and ladder below. Ladder, experimental lanes, then cyanobacterial high conc and low conc, then ladder, then purified protein, then ladder.








KaiA (very high expression):





KaiC (moderate expression):



JB:
Did lysis + CPCR on KaiA/KaiC colonies 1-7 and KaiC colonies 1-8. CPCR didn’t work so will try again with better lysis protocol
JB:
Lysis + CPCR on KaiA/KaiC 3, 4, 5, and KaiC 1-8 using Kasey’s DNA purification protocol
Ran on gel and got strange results
Set up new cultures for KaiC 1-8 so we don’t run out and put them in shaker. (To be used for DNA purification + cPCR, since this went weird)
Set up new starter culture with KaiA starter culture for new lysis protocol trial run (https://docs.google.com/document/d/1hr0q6QiX1REvBYSSCGiHLPh_jTGn5zF_Easy2MkJxUk/edit) tomorrow

</SPAN></P><P><SPAN class="ptitle"> 9/28/16</SPAN></P><P><SPAN class="p">
JB:
Made new WT starter culture and put in incubator labelled “iGEM starter culture” (has my initials and date)
Made overnight cultures for Kai A 3, 4, 5 (to be made into glycerol stocks later)
Trial ran lysis protocol (https://docs.google.com/document/d/1hr0q6QiX1REvBYSSCGiHLPh_jTGn5zF_Easy2MkJxUk/edit) 
OD of starting culture 0.512
OD of control before incubation 0.549
OD of starvation before incubation 0.521
Found contamination in media in the middle so will be starting over

</SPAN></P><P><SPAN class="ptitle"> 9/29/16</SPAN></P><P><SPAN class="p">
MP:

Made KaiA/KaiC 3,4&amp;5 glycerol stocks
Spin purified Steve’s 128 and 204 digests
Dephosphorylated 128 and 204 at 9:20am. Ready for gel pur at 10:20am.
Digested 800ng of 211 and submission backbone at 9:13am. Ready for spin pur at 11:13am. 

JB:
Redid CPCRs with KaiCs

</SPAN></P><P><SPAN class="ptitle"> 9/30/16</SPAN></P><P><SPAN class="p">
MP:

Resuspended KaiCKaiB in 20ul
Transformed KCKB in original backbone
Digested and spin purified KCKB
Ligated to yip204 and submission backbones
Transformed KCKB in yip204 and sub
Didn’t have enough 204 to do a control

Ran Julia’s CPCRs on a gel. No hits. Most likely PCR isn’t working but insert is there.

</SPAN></P><P><SPAN class="ptitle"> 10/1/16</SPAN></P><P><SPAN class="p">
MP:

Tried and failed to do Western blot with Mansi and Julia
Made new KaiC and KaiA/KaiC overnights for re-trying

All plates grew. Some had lawns. Restreaked them. Made 5x overnights with the rest (controls had 0 colonies so assumed high efficiency).
SD:
Autoclaved large tips
Digested yip204
Spin purified yip204
Dephosphorylated yip204 and left in the -20 overnight to gel purify tomorrow
Made YPAUD (500ml)

</SPAN></P><P><SPAN class="ptitle"> 10/2/16</SPAN></P><P><SPAN class="p">
MP:

Moved overnights to cold room
Made new overnights from the re-streaks

SD:
Ran yip204 on a gel
Must have gotten messed up while loading because it looked very strange

Running a PCR on yip204
3 rxns with herculase at 55 degrees, extension time 4 minutes (3,000bp length)

JB:
Trial running new lysis protocol. Will save pellets in -80
OD of starting culture: 9.5
OD of 30 mL dilution: 0.526
Control 1 OD: 0.538
Control 2 OD (lysed): 0.509
Starvation 1 OD (blanked with starvation media): 0.545
Starvation 2 OD (blanked with starvation media, lysed): 0.508
Placed tubes in “igem 2016 glycerol stocks” box
Put beads in all 4 tubes (original protocol calls for just 2 tubes) -- control 1 and starvation 1 got 1 round of vortexing, and control 2 and starvation 2 got all 4 rounds like the protocol says
Made new LB

</SPAN></P><P><SPAN class="ptitle"> 10/3/16</SPAN></P><P><SPAN class="p">
MP:

Started new KaiC vs. KaiAKaiC western in Rust lab
Benjamin miniprepped all the kaickaib samples

SD:
Gel purified yip204
Concentration was 8.4ng/uL, total volume was 25uL (200ng)

</SPAN></P><P><SPAN class="ptitle"> 10/4/16</SPAN></P><P><SPAN class="p">
MP:

Continued KaiC vs. KaiAKaiC western in Rust lab
Set up diagnostic digests for yesterday’s minipreps
kaickaib/yip204: stu1 aat2 cutsmart
kaickaib/sub: aat2 mfe1 cutsmart
kaickaib/genscript: econ1 bsa1 cutsmart
Asked Mansi to run them on a gel


1-15 didnt show up. Nanodrop says miniprep yields were low. 16-20 have multiple weird bands for some reason. Redoing all. 
kaickaib/sub: aat2 bamh1 cutsmart
kaickaib/yip204: stu1 aat2 cutsmart
kaickaib/genscript: econ1 bsa1 cutsmart
NB: Accidentally added 150ng/ul*10ul=1500ng of DNA to 16-20 because I was using 10ul for the other ones to get 150ng. Will probably have undigested material.
These results were also weird. Jason pointed out why. When I cut KaiCKaiB out of the backbone, I spin purified instead of gel purified, which means that the other half of the plasmid was in there. Wo when I did the ligation, there were 3 possible products: re-ligation, ligation to the wrong half, and ligation to the right half.
Going to go back to digestion phase.

</SPAN></P><P><SPAN class="ptitle"> 10/5/16</SPAN></P><P><SPAN class="p">
SD:
Miniprepped yip204
589ng/uL
Realized I miniprepped a 204 sans prefix and suffix, and so I created an overnight culture from KCKB to miniprep tomorrow
Digested KCKB with ScaI, PstI, and EcorI-HF


Weird bands. Should have only gotten three. The two black bands are correct, while the two faint bands next to them are wrong. Isolated the top black band which was what we were looking for and put it in cold room overnight for gel extraction tomorrow.

</SPAN></P><P><SPAN class="ptitle"> 10/6/16</SPAN></P><P><SPAN class="p">
SD:
Miniprepped the correct yip204
89.5ng/uL, 50uL
Will gel purify to extract just plasmid and then dephosphorylate
Digested with EcorI and PstI
Gel purified KaiC-KaiB from yesterday
5.8ng/uL, 26uL

MP:

Ligated and transformed kckb/sub and kckb/204
Didn’t have enough backbone to do controls and had very little 204 in general
Made new overnight cultures for all the yeast KaiC samples
Digested 800ng of sub backbone w/ ecor1 and pst1

</SPAN></P><P><SPAN class="ptitle"> 10/7/16</SPAN></P><P><SPAN class="p">
SD:
Gel purified yip204 digest from yesterday

Very faint bands of too high molecular weight (underlined in yellow). Nothing to isolate. Wasn’t sure why I got this since I know I had lots of DNA in the digest. Decided to test original miniprep by running it on a gel



You can see the plasmid band circled in yellow. Although it is a bit faint, indicating that I didn’t have quite as much DNA as I thought, it is still present
Made a new digestion of yip204+KCKB with EcorI and PstI using remaining 27uL of miniprepped plasmid. Hopefully this will give us enough DNA.
Spin purified sub backbone
Dephosphorylated sub backbone 
Gel purified sub backbone
 
Got two bands when I was only supposed to get one. No explanation for why this is. Isolated the correct size band, the top one, but also stored the bottom band just in case.
Purifed and got 5.2ng/uL, 25uL

</SPAN></P><P><SPAN class="ptitle"> 10/8/16</SPAN></P><P><SPAN class="p">
MP:
Made overnight cultures for kaickaib/sub
Re-streaks for kaickaib/yip204 (lawns)
Made o/n WT yeast culture for xymolase lysis protocol
Asked Julia to do high purity yeast lysis on 10x samples for cpcrs

JB:
High purity lysis on KaiC yeast

</SPAN></P><P><SPAN class="ptitle"> 10/9/16</SPAN></P><P><SPAN class="p">
JB:
Moved cultures from 37 shaker into cold room
Made 8 mL overnight cultures in LB amp for the 5 KaiCKaiB cultures in genscript backbones. In 37 shaker
Made overnights from 8 colonies on the KaiCKaiB-yip204 plates. In 37 shaker

</SPAN></P><P><SPAN class="ptitle"> 10/10/16</SPAN></P><P><SPAN class="p">
JB:
CPCR on KaiC yeast. Not successful (positive control didn’t work either so something wrong with PCR) but will retry tomorrow



MP:

Made new overnights for kckb/genscript and kckb/yip204
Ones that Julia made didn’t grow. Don’t know why.
Miniprepped kckb/sub
Diagnostic digested kckb/sub
BamH1 and Sca1
Got a few likely hits
2,3,5,7
Gel ran slightly weird so sizes don’t exactly line up, but pretty close
Others might also be hits, just faint

</SPAN></P><P><SPAN class="ptitle"> 10/11/16</SPAN></P><P><SPAN class="p">
JB:
CPCR on KaiC with new reagents and more controls to find out where stuff is going wrong. Positive control with purified DNA didn’t work but with WT DNA did. So something is wrong with purified DNA

MP:
Made glycerol stocks from hits yesterday
Cultures didn’t grow again!
The plates must be wrong
Made restreaks from restreaks on Jason’s amp plates
Will likely need to redo transformations
SD:
Digested GFP in yip211 with EcorI and PstI
Gel purified GFP and yip204
Yip204 didn’t seem to be cut or the two bands are just too close in size. Will redigest again using a third enzyme next time which will cut the insert in half. 
Purified the GFP and got 5.2ng/uL, 25uL total
Made overnights 

</SPAN></P><P><SPAN class="ptitle"> 10/12/16</SPAN></P><P><SPAN class="p">
MP:
There actually were a handful of colonies on the restreak 2 plates
Made 5x overnight cultures

Did GFP/211 and GFP/sub ligations
Transformed:
GFP ori
GFP 211
GFP sub
KCKB ori
KCKB 204
Sub control
211 control
NB: was dumb and forgot to turn water bath on, so tubes were on ice for +30min

</SPAN></P><P><SPAN class="ptitle"> 10/14/16</SPAN></P><P><SPAN class="p">
MP:
Showed Steve how to re-streak KCKB/ori and GFP/ori

</SPAN></P><P><SPAN class="ptitle"> 10/15/16</SPAN></P><P><SPAN class="p">
JB:
Set up diagnostic digests expecting following results:



MP:
Various people miniprepped and diagnostic digested GFP 211, GFP sub, and KCKB 204.
Got hits for KCKB 204. Got weird results for both GFP constructs. Jason says should go back to the digest stage. 
Made KCKB/204 overnights for cultures 1,2,3 → for sequencing and glycerol stocks.
Transformed KCKB 204 sample 1 (from Steve’s batch, first check mark) into yeast.



Sent KCKB/sub 2, 3, and 5 for sequencing
Innoculated new LB with KCKB/sub samples because they were old → for glycerol stocks.

</SPAN></P><P><SPAN class="ptitle"> 10/16/16</SPAN></P><P><SPAN class="p">

MP:
Made KCKB/sub 2,3,5 and KCKB/204 1,2,3 glycerol stocks

</SPAN></P><P><SPAN class="ptitle"> 10/17/16</SPAN></P><P><SPAN class="p">
MP:
Picked new colonies from 9/20 KaiC plate and made 10ml overnights
Picked colonies from KCKB/204 plate (which was beautiful) and made 10ml overnights
Made a new 10ml overnight from KaiAKaiC yeast #3

NB: right now there are two rounds of KCKB/ori, 5x from the restreak 2 plates I made on 10/12 (ready for diagnostic digest in the -20), and another from the second transformation that I did which got made into overnights on 10/15 (ready to be miniprepped)

Miniprepped kckb/ori, gfp/ori, and kckb/204

</SPAN></P><P><SPAN class="ptitle"> 10/18/16</SPAN></P><P><SPAN class="p">
JB:
Digested Genscript GFP and Yip211 with PstI and EcoRI
Set up diagnostic diagests expecting these results:




MP:
Sequencing for KCKB/sub came back. Clone 5 is a winner. Ready to submit!
Sent KCKB/204 1,2,3 for sequencing

Ran Julia’s digests on a gel. Some hits and misses. Gfp/ori and kckb/ori hits should be sequenced and made into glycerol stocks.



Did high purity yeast lysis on 5x kckb/204s and 5x new kaics (alpha-epsilon)
Epsilon was lost along the way
Left DNA in cold room

</SPAN></P><P><SPAN class="ptitle"> 10/19/16</SPAN></P><P><SPAN class="p">
MP:

Dried down 10ul of 60ng/ul KCKB/sub #5

</SPAN></P></DIV></DIV></BODY></HTML>