Team:UChicago/Design
PROJECT
PROJECT OVERVIEW
DESIGN
RESULTS
PARTS
PARTS OVERVIEW
BASIC PARTS
PRACTICES
HUMAN PRACTICES
TEAM
TEAM MEMBERS
COLLABORATIONS
DOCUMENTATION
SAFETY
NOTEBOOK
ATTRIBUTIONS
ATTRIBUTIONS
REFERENCES
The KaiABC Oscillator
The lives of many organisms on earth, from cyanobacteria to humans, are governed by an endogenous circadian rhythm that is entrained to the environmental effects of the earth’s rotation. Each organism establishes such a rhythm through biological networks of varying complexity, but the simplest known circadian system is found in the cyanobacterium Synechococcus elongatus.
In S. elongatus, global circadian gene expression is regulated by the KaiABC system: three proteins, encoded by two genes, that as an autonomous unit drive each other through oscillatory phosphorylation states. More specifically, KaiA and KaiB regulate the rhythmic phosphorylation and dephosphorylation of KaiC, which regulates oscillatory downstream gene expression. During subjective dawn, KaiA binds to KaiC and promotes autophosphorylation at KaiC residues Thr432 and Ser431, respectively. As Thr432 is subsequently auto-dephosphorylated, KaiB increasingly binds to KaiC, and the KaiB-KaiC complex sequesters KaiA such that KaiA can no longer promote autophosphorylation of KaiC. As the cycle enters subjective dawn, KaiC continues to auto-dephosphorylate at both Thr432 and Ser431, releasing KaiB and subsequently KaiA such that all proteins are free to enter the cycle again. As it oscillates through its phosphoforms, KaiC phosphorylates downstream regulators of gene expression to produce a global circadian expression pattern (Rust 2007).
Of particular interest in the KaiABC system is its post-translational modularity. Organismal circadian clocks have traditionally been conceptualized as transcriptional feedback loops in which the translated product of a particular clock gene inhibits further transcription, until the concentration of the product is low enough to reactivate its transcription. On the contrary, the KaiABC system exhibits oscillations independently of transcriptional feedback – rhythms persist when the genes are expressed under constitutive promoters (Ditty et al. 2005), as well as in the presence of transcriptional or translational inhibitors (Tomita et al. 2005). Most significantly, the three purified proteins can sustain circadian rhythmicity in vitro, without transcriptional or other environmental input (Nakajima et al. 2005). The KaiABC system is thus remarkably modular, making it not only an invaluable model for the study of circadian systems, but also a potent tool in synthetic biology.
KaiABC In Synthetic Biology
In particular, KaiABC has the potential to repair dysfunctional circadian systems in heterologous organisms, which is significant because disrupted cycles are associated with increased risk for diseases such as cancers, cardiovascular disease, and obesity (Shanmugam et al. 2013). In mammals, the suprachiasmatic nucleus of the hypothalamus produces autonomous circadian oscillations through transcriptional-translational feedback loops, which coordinate the rhythms of peripheral oscillators throughout the body. If a component of the transcriptional loop is dysfunctional, perhaps a master transcription factor could be engineered to be driven by KaiABC. Alternatively, perhaps KaiABC could drive the production of a particular drug on a 24-hour time scale.
In exploration of these potential applications, the KaiABC system in 2015 was reconstituted for the first time in E. coli and successfully connected to a transcriptional output (Chen et al. 2015). Furthermore, the 2015 UChicago iGEM team designed a system for a circadian, KaiABC-driven release of a melatonin precursor using E. coli. These experimental milestones have been the first steps in a proof-of-concept that the KaiABC system is indeed functional in heterologous organisms, and further, that it can be engineered to drive an oscillatory transcriptional output.
This year, the UChicago iGEM team brings the KaiABC system one step closer to its potential for circadian rhythm repair by exploring the possibility of reconstitution in a eukaryotic organism. The 2016 team aims to express the KaiABC system in Saccharomyces cerevisiae, as well as design a readout system for an oscillatory transcriptional output.
Plasmid Design
Our first plasmid contains KaiC fused to a Gal4 activation domain, with a self-cleaving peptide linker, P2A, fusing it to KaiB. P2A cleaves itself during translation, causing the KaiC-Gal4AD and the KaiB to separate. We did this to allow us to control the exact ratio of KaiC, KaiB, and KaiA, and make sure it is constant and equal in every cell.  The second plasmid has KaiA linked by P2A to another fusion protein, SasA fused with the LexA binding domain. The third plasmid is our reporter plasmid, and contains GFP fused with a PEST degradation tag, and has a lexA operator region before the promoter. The degradation tag allows the amount of GFP in the cell to drop quickly enough once it’s no longer being expressed that we can use fluorescence assays to measure gene expression over time.  The lexA binding domain binds to the lexA operator region, causing the SasA-LexA fusion protein to bind to the lexA operator region on the reporter plasmid. In cyanobacteria, SasA binds to phosphorylated KaiC and activates transcription. In our system, this means that when KaiC is phosphorylated it will bind to SasA, and because it is fused to a Gal4 activation domain, it will promote transcription when it binds. This causes GFP to be expressed, which can be measured by fluorescence to test gene expression. In this way, SasA acts as the bait protein and KaiC as the prey, when using the analogy of the Y2K system.
KaiC is phosphorylated by KaiA and dephosphorylated by KaiB, on a periodic cycle lasting 24 hours. This means that KaiC is phosphorylated for 12 hours, causing GFP to be expressed. The next 12 hours it is dephosphorylated, doesn’t bind to SasA, and GFP isn’t expressed. Because the GFP is degradable, we will be able to see expression levels fall dramatically when KaiC is unphosphorylated.
This allows us to show that the KaiABC system can regulate gene expression in yeast, which serves as a proof of concept for placing eukaryotic gene expression under the control of an exogenous circadian clock.
Our strain of yeast is a TRP1, LEU2, and URA3 KO strain. Therefore our plasmids use those markers, with plasmid 1 having a TRP1 gene, plasmid 2 having a LEU2 gene, and plasmid 3 having a URA3 gene. Furthermore, plasmid 1 and 2 are integrated into the yeast genome, while plasmid 3 isn’t and relies on a MCS on the plasmid. This is because our strain of yeast isn’t integratable with URA3.
Our Goals
This summer and beyond, we aim to
1) show that the core oscillator proteins of the KaiABC system, KaiA, KaiB, and KaiC, are properly expressed in S. cerevisiae
2) show that KaiA promotes autophosphorylation of KaiC as in the KaiABC system, and
3) show that reconstituted KaiABC can be synchronized across cells by an external stimulus.
