<HTML xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en" dir="ltr">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki  ltr ns-0 ns-subject page-Team_WLC-Milwaukee_Cellulases"><DIV id="globalWrapper"><DIV id="top-section"><DIV id="p-logo"><A href="/Main_Page" title="Main Page">&quot;
	    </A></DIV><DIV id="menubar" class="left-menu noprint"><UL><LI class="selected"><A href="/Team:WLC-Milwaukee/Cellulases">Page               </A></LI><LI class="new"><A href="/wiki/index.php?title=Talk:Team:WLC-Milwaukee/Cellulases&amp;action=edit&amp;redlink=1">Discussion               </A></LI><LI><A href="/wiki/index.php?title=Team:WLC-Milwaukee/Cellulases&amp;action=edit">View source               </A></LI><LI><A href="/wiki/index.php?title=Team:WLC-Milwaukee/Cellulases&amp;action=history">History               </A></LI><LI style="color:white;cursor:default">teams</LI></UL></DIV><DIV class="right-menu noprint" id="menubar"><UL><LI id="pt-login"><A href="/wiki/index.php?title=Special:UserLogin&amp;returnto=Team:WLC-Milwaukee/Cellulases" title="You are encouraged to log in; however, it is not mandatory [o]" accesskey="o">Log in</A></LI></UL></DIV><DIV id="search-controls" class="noprint"><FORM action="/Special:Search" id="searchform"> </FORM></DIV></DIV><DIV id="content"><H1 class="firstHeading">Team:WLC-Milwaukee/Cellulases</H1><DIV id="bodyContent"><H3 id="siteSub" class="noprint">From 2014.igem.org</H3><DIV id="wlcHeader" class="navLinks" style="z-index:5;top:14px;left:0px;position:fixed;height:90px;width:100%;"><TABLE width="100%" align="center" style="background-color:#D1CAB0" cellspacing="0"><TBODY><TR height="90px"><TD id="HomeBox"><A id="homeLink" href="https://2014.igem.org/Team:WLC-Milwaukee" style="text-decoration:none">Home</A></TD><TD id="BackgroundBox"><A id="backgroundLink" href="https://2014.igem.org/Team:WLC-Milwaukee/Probiotics" style="text-decoration:none">Background</A></TD><TD id="ProjectBox"><A id="projectLink" href="https://2014.igem.org/Team:WLC-Milwaukee/Project" style="text-decoration:none">Project</A></TD><TD id="LabBox"><A id="labLink" href="https://2014.igem.org/Team:WLC-Milwaukee/Parts" style="text-decoration:none">Lab</A></TD><TD id="HumanPraticesBox"><A id="humanPraticesLink" href="https://2014.igem.org/Team:WLC-Milwaukee/Anthropology">Human Practices</A></TD><TD id="PeopleBox"><A id="peopleLink" href="https://2014.igem.org/Team:WLC-Milwaukee/Team">People</A></TD></TR></TBODY></TABLE></DIV><DIV class="mainCont" style="width: 85%"><P>
Our team selected three cellulases to begin the breakdown of cellulose: xynA, bgls, and yesZ. 
</P><H2>Enzyme xynA</H2><P> [2]
The enzyme xynA is a endo-1,4-beta xylanase or is referred to as beta xylanase. It catalyzes the hydrolysis of the xylan main chain in hemicellulose. Xylan is abundant in the cell wall structures of many plants. Lindner [1] investigated the regulation of xylanase within <I>Bacillus subtilis</I>. It was found that <I>Bacillus subtitis</I> had a slow rate of growth on xylan plates and did not grow on xylose plates. It was also found that the synthesis of xynA is not dependent on the environment, but instead was found to be highly synthesized during the exponential growth phase. This is a potential advantage to our project because the probiotic may not have time to reach the stationary phase in the ruminant. The presence of glucose also showed to have no effect on the presence of the mRNA sequence for xynA. It is now known that xynA has “glucose resistant synthesis”. This differs from most extracellular catabolic enzymes. Lindner also found that xynA is synthesized constitutively; meaning it is produced in constant amounts regardless of the surrounding environment of the bacterial cell. Banka [2] investigated xynA in <I>Bacillis subtilis</I> M015, and found that it was 99% identical to the amino acid sequence to <I>Bacillis subtilis</I> 168. They also found the temperature and pH profiles of recombinant xynA. This is displayed below. As the body temperature of ruminant animals is roughly 37 degrees Celsius to 40 degrees Celsius, the enzyme has roughly 60% activity. At a pH 6-7, which is the general pH of a ruminant, the xynA showed high activity. 

</P><H2>Enzyme bglS</H2><P>
The enzyme bglS hydrolyzes linked beta-D-glucans, which are commonly found in lichen and cellulose. YBeta-D-glucans are 5.5% of dry weight in grains, and 75% of carbohydrates in barley endospores. [3] They showed bglS to have an optimum pH of 6.0 and temperature of 50 degrees Celsius (active from 20 degrees Celcius to 70 degrees Celcius). BglS cleaves the beta-1,4 glycosidic linkage that is adjacent to the 3-0 substituted glucopyranose residues. This then releases trisaccharides or tetrasaccharides. [3] The active site for bglS is shown to have a &quot;EIDIEF&quot; motif. The two glutamic acid residues participate in acid and base nucleophile hydrolysis. 
</P><H3>Structure of bglS </H3><P>&quot;The active site cleft of the enzyme presents a negatively charged crevice surrounded by a number of aromatic residues (Fig. 2B)[3]. A single molecule of bis–tris-propane was found in the active site cleft, forming hydrogen bonds with the nucleophile Glu133 (2.62AÅ),the acid catalyst Glu137(2.56Å),Tyr151(2.83Å) and water mediated hydrogen bonds with Asn210, Asn56, Asn149, Gln147, Glu159 (Fig. 2C)[3]. On the opposite face of the active site, a single calcium ion (Fig. 2A)[3] was coordinated by backbone carbonyl oxygen atoms from Pro37, Gly73, Asp235, a carboxylate oxygen of Asp235, and two water molecules, as previously observed in the <I>B. licheniformis</I> homologue [18]. These amino-acids are located in the β1–β2 loop, in the β3–β4 loop and in the β15 strand respectively, showing that the stabilizing effect of the calcium stabilization is due to cross-linking of these regions.&quot; [3]</P><H2> Enzyme yesZ</H2><P> 
The enzyme yesZ is homologous to a beta-galactosidase. YesZ is known as a pectinase. Beta-galactosidases are found in four distinct CAZY (carbohydrate-active enzymes): GH1, GH2, GH35 and GH42. YesZ falls under GH42 and as it is in the clan GHA, it is is thought to have a two step, double displacement reaction. This forms a covalent glycosyl enzyme intermediate with is then hydrolyzed via a oxocarbenium like transition state. [4] The active site contains two key carboxylic type residues that are 5.5 Angstroms apart. This catalytic nucleophile attacks the sugar anomeric centre. This then covalently attaches to the galactosyl moiety as well as the acid/base residue that protonates a departing aglycone oxygen. The identification of the carboxylic acids that takes place in this is key to proposing this mechanism. Shaikh was able to identify the catalytic nucleophile as Glutamate-295. Glutamate-145 was also determined as a possible acid/base catalyst. Overall, it was determined that the peptide sequence ETSPSYAASL (residues 296-305) is also involved in the active site. [4]

</P><H2><B>Modeling</B></H2><H2>xynA</H2><P>The endo-1,4-beta-xylanase xynA is a globular protein that has two residues of interest the nucleophile and acid-base cleavage sites at the E residues 78 and 172 highlighted in red. [5]</P><H2>bglS</H2><P>The endo-1,3-1,4-glucanase bglS is a globular protein that that has two residues of interest. The putative nucleophile and acid-base cleavage sites at the E residues 133 and 137 highlighted in red. [5]</P><H2>yesZ</H2><P>The Beta-Galactosidase yesZ is globular and has three aspects that are important to its function. It contains a zinc ion binding site at the C residues 153, 155, and 158 in yellow. The putative nucleophile and acid base cleavage sites are at the E residues 155 and 296 in red.[5]</P><P><B>We are currently undergoing protein data modeling which describes the kinetics of our catalytic enzymes: xynA, bglS, and yesZ.</B></P>
Written by: Sierra Tackett
<DIV class="ref">
References
[1] Lindner, Stulke, Hecker. (1994) Regulation of xylanolytic enzymes in Bacillus subtilis. Microbiology 140, 753-757.  
[2] Banka, Guralp, Gulari. (2014) Secretory Expression and Characterization of Two Hemicellulases, Xylanase, and β-Xylosidase, Isolated from Bacillus Subtilis M015. Appl Biochem Biotechnol.  
[3] Furtado, Ribeiro, Santos, Tonoli, et al. (2011) Biochemical and structural characterization of a β-1,3–1,4-glucanase from Bacillus subtilis 168. Process Biochemistry 46, 1202-1206. 
[4] Shaikh, Mullegger, He, Withers. (2007) FEBS Letters 581, 2441-2446.  
[5] 2013 WLC Milwaukee Team: https://2013.igem.org/Team:WLC-Milwaukee/Modeling

</DIV></DIV></DIV><DIV class="printfooter">
Retrieved from &quot;<A href="http://2014.igem.org/Team:WLC-Milwaukee/Cellulases">http://2014.igem.org/Team:WLC-Milwaukee/Cellulases</A>&quot;</DIV></DIV></DIV><DIV id="footer-box" class="noprint"><DIV id="footer"><UL id="f-list"><LI id="t-recentchanges"><A href="/Special:RecentChanges" title="Recent changes">Recent changes</A></LI><LI id="t-whatlinkshere"><A href="/Special:WhatLinksHere/Team:WLC-Milwaukee/Cellulases" title="List of all wiki pages that link here [j]" accesskey="j">What links here</A></LI><LI id="t-recentchangeslinked"><A href="/Special:RecentChangesLinked/Team:WLC-Milwaukee/Cellulases" title="Recent changes in pages linked from this page [k]" accesskey="k">Related changes</A></LI><LI id="t-specialpages"><A href="/Special:SpecialPages" title="List of all special pages [q]" accesskey="q">Special pages</A></LI><LI><A href="/Special:Preferences">My preferences</A></LI></UL></DIV><DIV id="footer"><UL id="f-list"><LI id="t-print"><A href="/wiki/index.php?title=Team:WLC-Milwaukee/Cellulases&amp;printable=yes" title="Printable version of this page [p]" accesskey="p">Printable version</A></LI><LI id="t-permalink"><A href="/wiki/index.php?title=Team:WLC-Milwaukee/Cellulases&amp;oldid=400957" title="Permanent link to this revision of the page">Permanent link</A></LI><LI id="privacy"><A href="/2014.igem.org:Privacy_policy" title="2014.igem.org:Privacy policy">Privacy policy</A></LI><LI id="disclaimer"><A href="/2014.igem.org:General_disclaimer" title="2014.igem.org:General disclaimer">Disclaimers</A></LI></UL></DIV></DIV></BODY></HTML>