<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_Leiden_DNA_assembly skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><DIV id="top_title"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:Leiden/DNA assembly</SPAN></H1></DIV><DIV id="HQ_page"><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><P><TITLE>
   iGEM Leiden Wiki / DNA_assembly
  </TITLE></P><H1>
   DNA assembly Labjournal
  </H1><H2>
   Experimental setup
  </H2><P>
   In this track, the DNA received from IDT was assembled in pBluescript plasmids using Gibson Assemblies. Inserts were checked using colony-PCR on white colonies, after which frozen plasmid stocks were made and the constructs were checked in detail using sequencing.
				After this, the four BioBricks were placed in the right order using two subsequent rounds of 3A Assembly.
  </P><H2>
   IDTDNA solution, grow pBluescript E. coli
  </H2><P><I>
    Day1: 2016/07/04
   </I><B>
    Investigator(s): Sjoerd Seekles, Lisanne van Oosterhoud, Charlotte van de Velde
   </B><B>
    Procedure
   </B>
   :
   
   First labwork started. IDTDNA was delivered as powder. On the information sheet they recommend to disolve their DNA in 100ul water. Which will result in a concentration of 10 ng/µl (1000ng total in the tube).
   
   For this we spin down the tubes. Add (warmed up 60&amp;degC;) MQ water (fresh water, fresh pipet tips). Short vortexing (set the vortex at 1 pm). The DNA was put in 60&amp;degC; stove for 30 minutes. Then again spin down plus vortex. Put in the -20&amp;degC.;
   
   Took from -80&amp;degC; stock of Dennis some pBluescript to grow on LB+Xgal+IPTG+AMP. (20µl per plate, stock 0.1M IPTG, 40 mg/ml Xgal and 100 mg/ml Amp). Also grow pBluescript in liquid (add Amp 1:1000).
  </P><H2>
   Gibson pcrABCD, restriction, ligation, transformation QDH+DHC, cld, moaA
  </H2><P><I>
    Day2: 2016/07/06
   </I><B>
    Investigator(s): Sjoerd Seekles, Lisanne van Oosterhoud
   </B><B>
    Procedure
   </B>
   :
   
   Gibson assembly. Used pBlue 13.5 ng/µl linearized with EcoRV
   <B>
    pcrABCD
   </B>
   :
 <TABLE id="t01"><TBODY><TR><TH><B>
       Content
      </B></TH><TH><B>
       Volume
      </B></TH><TH><B>
       Amount (ng)
      </B></TH></TR><TR><TD>
      Vector
     </TD><TD>
      1 µl
     </TD><TD>
      13.5
     </TD></TR><TR><TD>
      Insert
     </TD><TD>
      2-2-2-2 µl
     </TD><TD>
      10-10-10-10
     </TD></TR><TR><TD>
      Gibsonmix
     </TD><TD>
      10 µl
     </TD><TD>
      -
     </TD></TR><TR><TD><B>
       Total
      </B></TD><TD>
      20 µl
     </TD><TD>
      -
     </TD></TR></TBODY></TABLE></P><P>
   Gibson reaction at 50&amp;degC; in heat block. 1 hour. Transform with super competent cells of kit. Followed transformation protocol of kit, but used everything of the gibson reaction to add to cells. And plate 50 µl, 250 µl and rest (~700µl).
  </P><P>
   Restriction was done on cld, QDH+DHC, MoaA and pBluescript(Wouter) with EcoRI and PstI. 1 hour, 37&amp;degC.; 
	The digestion was performed in a volume of 20 µl:
  </P><UL><LI>
    11 µl MilliQ
   </LI><LI>
    2 µl Orange buffer
   </LI><LI>
    5 µl IDTDNA
   </LI><LI>
    1 µl PstI
   </LI><LI>
    1 µl EcoRI
   </LI></UL><P>
   Ligation was done on the digestion. Ligation at room temperature for 10 minutes.
	The ligation was performed in 20 µl:
  </P><UL><LI>
    6 ul insert
   </LI><LI>
    3 ul 10x diluted vector (vector is 120 ng/µl pBlue Wouter)
   </LI><LI>
    4 ul Rapid ligation buffer (5x)
   </LI><LI>
    6 ul MQ
   </LI></UL><P>
   Transformation was done with Gibson protocol.
  </P><H2>
   Check IDTDNA
  </H2><P><I>
    Day2: 2016/07/06
   </I><B>
    Investigator(s): Sjoerd Seekles, Lisanne van Oosterhoud, Charlotte van de Velde
   </B><B>
    Procedure
   </B>
   : 
				The DNA was Nanodropped and checked on the gel.
  </P><P>
   Nanodrop results:
  </P><TABLE id="t01"><TBODY><TR><TH><B>
      Content
     </B></TH><TH><B>
      Nanodrop (ng/µl)
     </B></TH><TH><B>
      280/260
     </B></TH><TH><B>
      280/230
     </B></TH></TR><TR><TD>
     pcr1
    </TD><TD>
     8.6
    </TD><TD>
     2.09
    </TD><TD>
     2.28
    </TD></TR><TR><TD>
     pcr2
    </TD><TD>
     8.1
    </TD><TD>
     1.93
    </TD><TD>
     2.72
    </TD></TR><TR><TD>
     pcr3
    </TD><TD>
     3.9
    </TD><TD>
     1.48
    </TD><TD>
     2.28
    </TD></TR><TR><TD>
     pcr4
    </TD><TD>
     4.7
    </TD><TD>
     1.69
    </TD><TD>
     2.19
    </TD></TR><TR><TD>
     cld
    </TD><TD>
     6.4
    </TD><TD>
     1.66
    </TD><TD>
     1.58
    </TD></TR><TR><TD>
     QDH + DHC
    </TD><TD>
     7.5
    </TD><TD>
     1.97
    </TD><TD>
     2.17
    </TD></TR><TR><TD>
     MoaA
    </TD><TD>
     5.6
    </TD><TD>
     1.87
    </TD><TD>
     2.43
    </TD></TR></TBODY></TABLE><P>
   Gel was run with 5 ul IDTDNA and 5 ul pBluescript (linearized 13.5 and Wouter 120 variant, linearized)
  </P><H2>
   Gibson moaA
  </H2><P><I>
    Day3: 2016/07/08
   </I><B>
    Investigator(s): Sjoerd Seekles,
   </B><B>
    Procedure
   </B>
   : Repeat Gibson. This time with proper control. We saw on the gel that moaA had proper 1200 bps band.
   <B>
    moaA
   </B></P><TABLE id="t01"><TBODY><TR><TH><B>
      Content
     </B></TH><TH><B>
      Volume
     </B></TH><TH><B>
      Amount (ng)
     </B></TH></TR><TR><TD>
     Vector
    </TD><TD>
     2 µl
    </TD><TD>
     27
    </TD></TR><TR><TD>
     Insert
    </TD><TD>
     8 µl
    </TD><TD>
     40
    </TD></TR><TR><TD>
     Gibsonmix
    </TD><TD>
     10 µl
    </TD><TD>
     -
    </TD></TR><TR><TD><B>
      Total
     </B></TD><TD>
     20 µl
    </TD><TD>
     -
    </TD></TR></TBODY></TABLE><H2>
   Plasmid isolation from strains with moaA or cld
  </H2><P><I>
    Day4: 2016/07/13
   </I><B>
    Investigator(s): Valentijn Broeken, Wouter Liefting, Lisanne Oosterhoud, Lizah van der Aart
   </B><B>
    Procedure
   </B>
   : 
				The Plasmid isolation by Alkaline lysis protocol was used to isolate the plasmids from the white colonies grown in liquid LB medium. For 14 cultures, the anticipated insert was cld, and for 14 other cultures this was moaA. No chloroform/phenol extraction was performed.
   
   The isolated plasmids were then dissolved in 50 µl of demi water and the nucleic acid concentration was measured using a Nanodrop, together with the 260/280 ratio (protein contamination) and 260/230 ratio (contamination by salts and solvents).
   <B>
    Results:
   </B>
   The Nanodrop values were extremely high. In the follow-up experiment (putting the samples on gel), it turned out that most of it was RNA.
  </P><H2>
   Gel electrophoresis of the isolated moaA or cld plasmids
  </H2><P><I>
    Day5: 2016/07/14
   </I><B>
    Investigator(s): Valentijn Broeken
   </B><B>
    Procedure
   </B>
   : 
				To roughly check whether the anticipated inserts were present in the isolated plasmids, samples were digested with two restriction enzymes flanking the inserts (PstI and EcoRI) and subsequently put on a gel.
   
   For each insert, five samples with high 260/280 and 260/230 ratios (both &gt;1.9) and high nucleic acid concentrations (&gt;2500 ng/µl) were selected.
   
   The digestion was performed in a volume of 20 µl:
   <UL><LI>
     11 µl MilliQ
    </LI><LI>
     2 µl Restriction buffer 2.1 (NEB)
    </LI><LI>
     5 µl Isolated plasmid solution
    </LI><LI>
     1 µl PstI
    </LI><LI>
     1 µl EcoRI
    </LI></UL>
   The mixture was put at 37°C for 1 hour.
  </P><P>
   A gel containing 1% agarose was made by Lisanne the day before. Of each digestion mixture, 5 µl supplemented with 1 µl purple loading dye (NEB) was put on the gel. The gel run for 55 minutes at 100V. As a ladder, 5 µl of the Quick Load 2-log DNA ladder (NEB) was used.
   
   After running, the gel was stained 30 minutes in a 0.5 µg/ml ethidiumbromide in 1x TAE solution, after which it was washed during 30 minutes in 1x TAE (not necessary).
   
   Expectations After digestion with EcoRI and PstI, the following fragments are expected:
  </P><UL><LI>
    cld-BioBrick: 888 basepairs
   </LI><LI>
    moaA-BioBrick: 1011 basepairs
   </LI><LI>
    pBluescript plasmid: 2963 basepairs
   </LI></UL><P>
   Results The samples were loaded in the following order:
   <TABLE id="t01"><TBODY><TR><TH><B>
       Lane
      </B></TH><TH><B>
       Content
      </B></TH><TH><B>
       Lane
      </B></TH><TH><B>
       Content
      </B></TH></TR><TR><TD>
      7
     </TD><TD>
      Ladder
     </TD><TD>
      13
     </TD><TD>
      cld 1
     </TD></TR><TR><TD>
      8
     </TD><TD>
      moaA
     </TD><TD>
      14
     </TD><TD>
      cld 4
     </TD></TR><TR><TD>
      9
     </TD><TD>
      moaA 6
     </TD><TD>
      15
     </TD><TD>
      cld 6
     </TD></TR><TR><TD>
      10
     </TD><TD>
      moaA 7
     </TD><TD>
      16
     </TD><TD>
      cld 9
     </TD></TR><TR><TD>
      11
     </TD><TD>
      moaA 9
     </TD><TD>
      17
     </TD><TD>
      cld 12
     </TD></TR><TR><TD>
      12
     </TD><TD>
      moaA 13
     </TD><TD>
      18
     </TD><TD>
      Ladder
     </TD></TR></TBODY></TABLE></P><P><B>
    Conclusion
   </B>
   It is hard to draw conclusions from this gel, since RNA makes up most of the nucleic acid in the samples and no control pBluescript plasmid was added. The bands around 2-3kb could be pBluescript, but it is under debate why it would appear at two different heights then. There are really weak bands visible around 4kb, which could be the uncut plasmid containing the insert, but no fragments around 1 kb could be discovered - being the expected height for the digested inserts of moaA or cld.
   
   To be able to draw firmer conclusions, a colony PCR was performed.
  </P><H2>
   Gel electrophoresis of the isolated moaA or cld plasmids
  </H2><P><I>
    Day5: 2016/07/14
   </I><B>
    Investigator(s): Valentijn Broeken, Lizah van der Aart
   </B><B>
    Procedure
   </B>
   : 
				Since the digestion check wasn't successful in verifying whether inserts of the correct length were present in the pBluescript plasmids, a colony PCR was performed (see Q5 Colony PCR protocol with a 72&amp;degC; step of 45 sec). For each fragment, 1 blue and 7 white colonies were selected from the X-gal plates of the Gibson Assembly. Besides, an intact pBluescript was used as a control. 13M forward and reverse primers were used to amplify the insert in pBluescript. The PCR products were put on a gel to determine their length.
   <B>
    Expectations:
   </B><UL><LI>
     No insert: 231 bp
    </LI><LI>
     cld-insert: 1119 bp
    </LI><LI>
     moaA-insert: 1242 bp
    </LI></UL><B>
    Results
   </B><TABLE id="t01"><TBODY><TR><TH><B>
       Lane
      </B></TH><TH><B>
       Content
      </B></TH><TH><B>
       Lane
      </B></TH><TH><B>
       Content
      </B></TH></TR><TR><TD>
      7
     </TD><TD>
      Ladder
     </TD><TD>
      13
     </TD><TD>
      cld 1
     </TD></TR><TR><TD>
      8
     </TD><TD>
      moaA
     </TD><TD>
      14
     </TD><TD>
      cld 4
     </TD></TR><TR><TD>
      9
     </TD><TD>
      moaA 6
     </TD><TD>
      15
     </TD><TD>
      cld 6
     </TD></TR><TR><TD>
      10
     </TD><TD>
      moaA 7
     </TD><TD>
      16
     </TD><TD>
      cld 9
     </TD></TR><TR><TD>
      11
     </TD><TD>
      moaA 9
     </TD><TD>
      17
     </TD><TD>
      cld 12
     </TD></TR><TR><TD>
      12
     </TD><TD>
      moaA 13
     </TD><TD>
      18
     </TD><TD>
      Ladder
     </TD></TR></TBODY></TABLE></P><P>
   Colonies moaA #5, moaA #7, cld #6, cld #7 showed proper band sizes on the gel and were therefore used in the follow-up experiments.
  </P><H2>
   Gibson pcrABCD, QDH-DHC. Transformatie Backbones, Gibson en primers van kit, miniprep cld, moaA
  </H2><P><I>
    Day6: 2016/07/15
   </I><B>
    Investigator(s): Sjoerd Seekles, Wouter Liefting
   </B><B>
    Procedure
   </B>
   : 
				Cld and moaA colonies (moaA #5, moaA #7, cld #6, cld #7) were grown in liquid medium (13ml tubes) in LB+2xAMP at 37&amp;degC; 150rpm.
   
   Gibson assembly of pcrABCD in pBluescript and QDH+DHC in pBluescript was performed.
   <B>
    QDH
   </B><TABLE id="t01"><TBODY><TR><TH><B>
       Content
      </B></TH><TH><B>
       Volume
      </B></TH><TH><B>
       Amount (ng)
      </B></TH></TR><TR><TD>
      Vector
     </TD><TD>
      2 µl
     </TD><TD>
      27
     </TD></TR><TR><TD>
      Insert
     </TD><TD>
      8 µl
     </TD><TD>
      40
     </TD></TR><TR><TD>
      Gibsonmix
     </TD><TD>
      10 µl
     </TD><TD>
      -
     </TD></TR><TR><TD><B>
       Total
      </B></TD><TD>
      20 µl
     </TD><TD>
      -
     </TD></TR></TBODY></TABLE><B>
    pcrABCD:
   </B><TABLE id="t01"><TBODY><TR><TH><B>
       Content
      </B></TH><TH><B>
       Volume
      </B></TH><TH><B>
       Amount (ng)
      </B></TH></TR><TR><TD>
      Vector
     </TD><TD>
      2 µl
     </TD><TD>
      27
     </TD></TR><TR><TD>
      Insert
     </TD><TD>
      2-2-2-2 µl
     </TD><TD>
      40
     </TD></TR><TR><TD>
      Gibsonmix
     </TD><TD>
      10 µl
     </TD><TD>
      10-10-10-10
     </TD></TR><TR><TD><B>
       Total
      </B></TD><TD>
      20 µl
     </TD><TD>
      -
     </TD></TR></TBODY></TABLE><B>
    BioBricks ligation
   </B><TABLE id="t01"><TBODY><TR><TH><B>
       Content
      </B></TH><TH><B>
       Volume
      </B></TH></TR><TR><TD>
      Ligase
     </TD><TD>
      1 µl
     </TD></TR><TR><TD>
      Buffer 10x
     </TD><TD>
      2µl
     </TD></TR><TR><TD>
      Plasmid
     </TD><TD>
      5 µl
     </TD></TR><TR><TD>
      Water
     </TD><TD>
      12 µl
     </TD></TR><TR><TD><B>
       Total
      </B></TD><TD>
      20 µl
     </TD></TR></TBODY></TABLE></P><H2>
    Colony PCR for pcrABCD and QDH+DHC transformants
   </H2><P><I>
    Day7: 2016/07/18
   </I><B>
    Investigator(s): Valentijn Broeken
   </B><B>
    Procedure
   </B>
   : 
				White colonies resulting from the Gibson assembly of pcrABCD in pBluescript (10 colonies) and QDH + DHC in pBluescript (8 colonies) were taken and the plasmids were amplified using M13 primers. The method used is the Q5 Colony PCR protocol.
   <B>
    Expectations:
   </B><UL><LI>
     No insert: 231 bp
    </LI><LI>
     QDH+DHC insert: 1757 bp
    </LI><LI>
     pcrABCD insert: 5572 bp
    </LI></UL><B>
    Results
   </B><TABLE id="t01"><TBODY><TR><TH><B>
       Lane
      </B></TH><TH><B>
       Content
      </B></TH></TR><TR><TD>
      3
     </TD><TD>
      ladder
     </TD></TR><TR><TD>
      4-13
     </TD><TD>
      pcrABCD 1 - 10
     </TD></TR><TR><TD>
      14
     </TD><TD>
      pBluescript control
     </TD></TR><TR><TD>
      15-22
     </TD><TD>
      QDH+DHC 1 - 8
     </TD></TR></TBODY></TABLE></P><P>
   The same PCR programme as at 2016/07/14 was used. This meant that the 72&amp;degC; step turned out to be too short (only 45 sec) for the 5.6kb fragment to be amplified (which would require 168 sec), resulting in incomplete bands on the gel. Therefore, the strains showing bands were once again used for a colony PCR with these longer 72&amp;degC; steps of 2min48. The bacteria used for the second PCR were transferred from the streaks made this morning during the first colony PCR.
   
   These amplified samples were stored at 4&amp;degC; to put on a gel later.
  </P><P><I>
    Day8: 2016/07/20
   </I><B>
    Investigator(s): Lisanne Oosterhoud
   </B><B>
    Procedure
   </B>
   :
   <TABLE id="t01"><TBODY><TR><TH><B>
       Lane
      </B></TH><TH><B>
       Content
      </B></TH><TH><B>
       Lane
      </B></TH><TH><B>
       Content
      </B></TH></TR><TR><TD>
      1
     </TD><TD>
      ladder
     </TD><TD>
      15
     </TD><TD>
      ladder
     </TD></TR><TR><TD>
      12
     </TD><TD>
      pcrABCD culture 3
     </TD><TD>
      16
     </TD><TD>
      pcrABCD culture 9
     </TD></TR><TR><TD>
      13
     </TD><TD>
      pcrABCD culture 4
     </TD><TD>
      3=17
     </TD><TD>
      pcrABCD culture 10
     </TD></TR><TR><TD>
      14
     </TD><TD>
      pcrABCD culture 8
     </TD><TD>
      18
     </TD><TD>
      Empty pBluescript
     </TD></TR></TBODY></TABLE></P><P>
   The total fragment of pcrABCD amplified with M13 primers is 5,572 bp. There is however another binding site for the M13fw primer, at the bottom strand in pcrA (base 2277-2286), which can create a fragment of 2,401 bp. An empty pBluescript plasmid has a 231 bp fragment between the M13 primers.
  </P><H2>
   Massive DNA plasmid isolation from transformants with Biobricks, Backbones, Promotors and our own Gibsons, followed by restriction
  </H2><P><I>
    Day7: 2016/07/19
   </I><B>
    Investigator(s): Sjoerd Seekles, Charlotte van de Velde, Lisanne Oosterhoud
   </B><B>
    Procedure
   </B>
   :
   </P><H2>
    Isolate DNA from initially ungrown transformants, putting the digested samples on gel
   </H2><P><I>
    Day8: 2016/07/20
   </I><B>
    Investigator(s): Lisanne Oosterhoud, Lucie Delfos
   </B><B>
    Procedure
   </B>
   :
   
   The transformants that didn't grown in the liquid medium were miniprepped again. At 19/7 these strains were inoculated again from the plates.
   <B>
    Gel 1 Samples DNA plasmid isolation 19/7:
   </B><B>
    Gel 2 Samples DNA plasmid isolation 20/7:
   </B>
   Besides the ladders, no bands can be distinguished. For the samples on the first gel, a second Alkaline lysis buffer was used in which the SDS was precipitated, being a probable reason for its failure. For the second plasmid isolation + restriction, it is still unclear why no bands are visible.
  </P><P><I>
    Day8: 2016/07/21
   </I><B>
    Investigator(s): Valentijn Broeken
   </B><B>
    Procedure
   </B>
   :
   
   Since the staining for the former gels containing our digestion fragments wasn't optimal, all samples were loaded again and run in a gel that contained EtBr, so that staining afterwards wasn't necessary. Although the bands are more clear, besides the ladder and pBluescript (2,963 bp when digested with EcoRI and PstI) no isolated plasmids could be detected.
   <B>
    Gel 1 Samples DNA plasmid isolation 19/7:
   </B><B>
    Gel 2 Samples DNA plasmid isolation 20/7:
   </B>
   The first eight lanes of the second gel contain moaA and cld samples, but only RNA-bands are visible around 100bp.
  </P><P><I>
    Day9: 2016/07/22
   </I><B>
    Investigator(s): Charlotte Oosterhoud, Sjoerd Seekles
   </B><B>
    Procedure
   </B>
   :
   
   This day, miniprep was done in 2 different ways. The Quagen kit was used by Sjoerd, alkaline miniprep with chloroform-phenol steps by Charlotte. Miniprepping according to kit was done following their manual.
   
   Nanodrop results:
   <TABLE id="t01"><TBODY><TR><TH><B>
       Content
      </B></TH><TH><B>
       Nanodrop (ng/µl)
      </B></TH><TH><B>
       280/260
      </B></TH><TH><B>
       280/230
      </B></TH></TR><TR><TD>
      MoaA#5
     </TD><TD>
      128.8
     </TD><TD>
      1.92
     </TD><TD>
      1.88
     </TD></TR><TR><TD>
      MoaA#7
     </TD><TD>
      166.3
     </TD><TD>
      1.92
     </TD><TD>
      2.09
     </TD></TR><TR><TD>
      cld#6
     </TD><TD>
      112.8
     </TD><TD>
      1.92
     </TD><TD>
      2.06
     </TD></TR><TR><TD>
      cld#7
     </TD><TD>
      196.9
     </TD><TD>
      1.91
     </TD><TD>
      2.11
     </TD></TR><TR><TD>
      pcrABCD8/11
     </TD><TD>
      69.1
     </TD><TD>
      1.96
     </TD><TD>
      1.93
     </TD></TR><TR><TD>
      pcrABCD10/13
     </TD><TD>
      118.3
     </TD><TD>
      1.92
     </TD><TD>
      2.01
     </TD></TR><TR><TD>
      QDH-DHC
     </TD><TD>
      213.4
     </TD><TD>
      1.89
     </TD><TD>
      2.19
     </TD></TR><TR><TD>
      pSB1C3
     </TD><TD>
      82.7
     </TD><TD>
      1.98
     </TD><TD>
      1.95
     </TD></TR><TR><TD>
      pSB1A3
     </TD><TD>
      74.9
     </TD><TD>
      1.97
     </TD><TD>
      1.82
     </TD></TR></TBODY></TABLE></P></DIV></DIV></DIV></DIV></DIV></BODY></HTML>