<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_Leiden_Science skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><DIV id="top_title"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:Leiden/Science</SPAN></H1></DIV><DIV id="HQ_page"><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><P><TITLE>
   iGEM Leiden Wiki : Overview
  </TITLE></P><DIV id="body-wrapper"><DIV class="jumbotron text-center"><P style="font-size: 100px; color: white; text-align: center; font-family: Montserrat, sans-serif; font-weight: normal;">
     Overview
    </P></DIV></DIV><DIV id="Science behind the project" style="margin-top: -120px;" class="container-fluid bg-white"><DIV class="row"><DIV class="col-sm-8" style="text-align: justify;"><H2 style="margin-bottom: 20px;">
      The science behind the project
     </H2><P>
      Martian soil is contaminated with a toxic compound, perchlorate, making human life on mars impossible. For our project we use bacteria, which contain a perchlorate reduction system, to eliminate perchlorate from Martian soil. There are several bacteria that can already reduce perchlorate, for example Dechloromonas aromatica. However the model organism
      <I>
       E.coli
      </I>
      is a better suited bacterium to work with in this project, since it is easier to handle. Since E.coli does not contain a perchlorate reduction system, we transformed the E.coli cells from the bacterial strain DH5α with genes encoding for the perchlorate reduction system derived from Dechloromonas aromatica.
      
      Another aspect of our project is the influence of the gravity on mars on the bacterial growth. This is performed using an random positioning machine (RPM). The bacterial growth under normal gravity and the gravity on mars are compared.
     </P><P><I><B>
        Transformation of E.coli DH5α</B></I></P><P>
      Genes coding for perchlorate reduction were introduced in the E.coli DH5α strain. These genes used for the transformation are derived from Dechloromonas aromatica. With these genes these, perchlorate reducing bacteria like Dechloromonas aromatica can reduce perchlorate into chlorite (ClO
      <SUB>
       2
      </SUB>
      ) with perchlorate reductase
      <SUP>
       2
      </SUP>
      Subsequently, the bacteria can convert chlorite into oxygen (O
      <SUB>
       2
      </SUB>
      ) and chloride (Cl
      <SUP>
       -
      </SUP>
      ) with chlorite dismutase. The sequence coding for these genes were acquired from a gene database
      <SUP>
       1
      </SUP>
      and codon optimization for
      <I>
       E.coli
      </I>
      was performed.
     </P><P>
      The genes encoding for perchlorate reductase and chlorite dismutase from
      <I>
       Dechloromonas aromatica
      </I>
      are shown in the figure below:
     </P><B>
      Figure 1.
     </B><I>
      The perchlorate reduction genomic island of Dechloromonas aromatica. This includes all the genes necessary for the reduction of perchlorate
      <SUP>
       3
      </SUP>
      .
     </I><P>
      Upstream of the FNR/RpoN promoter a group of regulatory genes is present, these genes were not used. The genes downstream of the RpoN promotor are the genes with which the E. coli cells were transformed. The genes pcrA, B, C and D were cloned into one vector, with the use of Gibson assembly. Secondly the genes DHC and QDH, the membrane-associated proteins were cloned together into another vector. Finally, cld and moaA will be cloned separately in two vectors. First the E.coli cells were made competent and thereafter transformed with the plasmids containing the genes encoding for the perchlorate reduction system.
     </P><P>
      There are a lot of previously made BioBricks. We can choose from a variety of constitutive and inducible promoters to express the genes necessary for perchlorate reduction in E.coli. Next to these promotors other BioBricks may be helpful for our project. For example some BioBricks which can function as an O2 sensor or a promotor which is regulated by the presence of Oxygen.
     </P><P>
      We choose E.coli as a host because of the extensive molecular knowledge, vast amount of available BioBricks and phylogenetical similarities to the perchlorate reducing bacteria.
     </P><P>
      After the transformation, experiments were performed to determine whether the transformation had succeeded. This was performed by checking whether oxygen was formed by adding the transformed bacteria to perchlorate.
     </P><P><I><B>
        Random positioning machine (RPM)
       </B></I></P><P>
      Since many bacteria are known to behave differently in space, we are interested in the effects of simulated Martian gravity on gene expression. Various bacteria are known to grow faster and become (more) pathogenic in space and we wanted to know if this was also the case on Mars. As such, we measured gene expression in 3 different conditions: 1g (control), 0.37g (Martian gravity) and 0g (space, ISS). Of course, we cannot actually travel to Mars to experience its gravity, but instead, we used the Random Positioning Machine (RPM 2.0) by Airbus Defence and Space Netherlands. The Random Positioning Machine by Airbus Defence and Space Netherlands is a machine that allows us to do the impossible: experience Martian gravity here on Earth! The machine calculates a path over 3 axes, that will average the gravity exerted on whatever we put inside. This way, we can simulate any amount of gravity between 0g (e.g. space, ISS) and 1g (Earth). Martian gravity is approximately 0.37g and we performed many of our experiments with this amount of gravity. This way, we can accurately predict the behavior of our E. colinizer, using experimental conditions that would also be experienced on Mars!
      
      Four strains, Pseudomonas putida and three different E. coli strains, were used. These strains were spread on agar plates and these plates were kept under different gravities: the gravity on earth and the gravity on Mars.
      <I>
       RNA sequencing
      </I></P><P>
      Using differential gene-expression. For the iGEM competition, we would like to identify highly up and downregulated genes, so that their respective promoters can be used as gravity BioBricks. This will allow other contestants to use our promoters which will ensure their intended system also works under Martian gravity. Furthermore, we want to asses whether E. coli displays the same changes on Mars as observed in space. Perhaps the Martian gravity is sufficient to suppress potential pathogenicity and perhaps we will uncover that it will be just as dangerous as in space! Either way, this information is extremely valuable to Martian colonizers, since they will be able to anticipate to the expected changes.
     </P><P><B>
       References:
      </B></P><P><OL style="font-size: 8px;"><LI><A>
         NCBI
        </A></LI><LI><A>
         Bender, K. S., Shang, C., Chakraborty, R., Belchik, S. M., Coates, J. D., &amp; Achenbach, L. A. (2005). Identification, characterization, and classification of genes encoding perchlorate reductase. Journal of Bacteriology, 187(15), 5090-5096
        </A></LI><LI><A>
         Melnyk, R. A., Clark, I. C., Liao, A., &amp; Coates, J. D. (2014). Transposon and deletion mutagenesis of genes involved in perchlorate reduction in Azospira suillum PS. MBio, 5(1), e00769-13
        </A></LI></OL></P></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></BODY></HTML>