Team:Leiden/Notebook
iGEM Leiden Wiki : Lab Journal
Lab Journal
01/7 - 23/8 Sjoerd, Valentijn, Lisanne, Charlotte
We ordered DNA from IDTDNA. The goal was to use E. coli  optimized DNA from IDTDNA to transform our E. coli  DH5a cells. The following genes were codon optimized and ordered at IDTDNA: pcrA, pcrB, pcrC, pcrD, cld, QDH+DHC and moaA. The sequences of these genes are codon optimized versions of the Dechloromonas aromatica RCB strain. Unfortunately, the DNA received from our first order was not as expected, see figure 1.
Figure 1. DNA as received from IDTDNA.
The DNA received from IDTDNA turned out to be a smear (the 2 bands on the right are pBluescript controls). Fortunately, IDTDNA responded quickly and send us new DNA to work with.
Figure 2. New vs Old IDTDNA.  This gel shows the same ordered pieces new (first lane after ladder) vs old (second lane after ladder). This new vs old continues for 8 lanes.
The total amount of synthesized is 10.1kb and was ordered in several pieces. These pieces were assembled via Gibson Assembly. Gibson assembly worked fine for cld, moaA and qdh-dhc, but the perchlorate reducing genes A, B, C and D were not easily assembled. We ordered extra DNA at IDTDNA which let us clone pcrB, C and D into pSB1C3 via restriction and ligation. PcrA is too big to order in one piece, thus we were forced to try different Gibson Assembly methods to no avail.
23/8 Sjoerd, Charlotte
Miniprepped o/n grown cultures of cld, moaA and qdh-dhc (triplicates).
Nanodrop after miniprep
Sample
ng/ul DNA
260/280
260/230
cld 1
114,5
1,89
2,38
cld 2
118,3
1,89
2,32
cld 3
100,6
1,91
2,25
moaA 1
139,4
1,90
2,41
moaA 2
139,4
1,90
2,37
moaA 3
135,5
1,91
2,23
qdh-dhc 1
164,5
1,90
2,33
qdh-dhc 2
150,5
1,90
2,39
qdh-dhc 3
116,6
1,90
2,27
Digested samples o/n, cut with EcoRI + SpeI or EcoRI only.
24/8 Sjoerd, Charlotte
Gelelectroforese of single and double cut plasmids. Only cld 2 doesn’t give a band of 800 as expected when cut twice.
One sample of each construct was chosen and sent to Baseclear to be sequenced for verification (cld 1, moaA 1 and qdh-dhc 1). The Vf and Vr iGEM primers were used for this.
26/8 Sjoerd, Charlotte
Sequences of the DNA sent to baseclear showed that moaA and qdh-dhc did not have any mutations or deletions. These are the first parts to be registered into the database ( http://parts.igem.org/Part:BBa_K1938005  and http://parts.igem.org/Part:BBa_K1938006 ).
However, cld showed a point deletion in the coding part which would make a nonfunctional protein.
Miniprepped o/n grown cultures of pcrB, pcrC and pcrD
Nanodrop after miniprep
Sample
ng/ul DNA
260/280
260/230
pcrB
70,1
1,90
2,08
pcrC
49,9
1,95
2,59
pcrD
79,8
1,92
2,57
Samples were cut 1,5 hours with PstI and PstI + EcoRI
Gelelectroforese of single and double cut plasmids. Bands are better visible using the designated computer program. All expected bands were present.
All three samples were sent to Baseclear to be sequenced to check for mutations or deletions. The Vf and Vr iGEM primes were used.
30/8 Charlotte
The sequence of pcrB, pcrC and pcrD showed no mutations and/or deletions when compared to the designed DNA strand. This were the parts registred to the parts database as http://parts.igem.org/Part:BBa_K1938001 , http://parts.igem.org/Part:BBa_K1938002  and http://parts.igem.org/Part:BBa_K1938003 .
30/8 until 14/9. Sjoerd, Charlotte
We tried to repair the cld with the point-deletion by performing rolling-circle PCR, but this was in vain. The gels (not included) showed no or wrong-sized bands.
15/9 Sjoerd, Charlotte
Cld was ligated in pBluescript using Gibson assembly. White colonies were streaked on a plate and grown o/n at 30 ℃ to prepare them for the miniprep.
Nanodrop after miniprep
Sample
ng/ul DNA
260/280
260/230
1.7
66,1
1,83
1,87
1.9
26,0
1,80
1,80
1.10
76,2
1,81
1,90
2.2
14,5
1,61
2,05
3.4
106,2
1,78
0,99
6.9
38,2
1,88
1,95
Plasmids were cut with SpeI and XbaI and SpeI for 1 hour.
Gelelectroforese shows that only 1.7 and 1.10 have a band at the expected size (800), other gels not included.
1.7 and 1.10 were sent to Baseclear for sequencing with the M13 forward and M13 reverse primers.
23/9 Charlotte
Sequencing results show that both cld plasmids contain the right cld insert. We chose to make a part of 1.7, resulting in http://parts.igem.org/Part:BBa_K1938004 .
23/9 until 13/10 Sjoerd, Charlotte
The arabinose inducable promoter http://parts.igem.org/Part:BBa_K808000  and cld were ligated together using 3A assembly into pSB1C3. Several tests were done in order to determine if the construct works. We grew dilutions of our bacteria on several different agar plates containing a range of arabinose and chlorite concentrations (0 mM - 1mM). We observed no difference between the construct and our control but found out that the bacteria grow at concentrations of 0.25 mM chlorite but died at 0.5mM chlorite (see figure below. Left: 0.25mM arabinose, right 0.5mM arabinose).
We also checked the presence of proteins on a SDS-PAGE gel but no bands of the expected size were visible (figure not included).
14/10 Charlotte
All previous constructs didn’t contain a ribosomal binding site (RBS). 5 promoters from the iGEM depository kit plates were chosen:
1.3m: http://parts.igem.org/Part:BBa_K525998  (Promoter T7 and RBS)
1.3o: http://parts.igem.org/Part:BBa_K608002  (Strong promoter, strong RBS)
1.5a: http://parts.igem.org/Part:BBa_K608003  (Strong promoter, medium RBS)
1.5c: http://parts.igem.org/Part:BBa_K608004  (Strong promoter, weak RBS)
1.11:i http://parts.igem.org/Part:BBa_K541503  (Constitutive strong promoter, strong RBS and Tat signal peptide).
17/10 Sjoerd, Charlotte
Miniprepped o/n grown plasmids (triplicates) of the promoters mentioned above. 1.3c Didn’t grow and was left out of future experiments.
Nanodrop minprepped plasmids
Sample
ng/ul DNA
260/280
260/230
1.3m 1
71,8
1,88
2,20
1.3m 2
155,5
1,80
1,62
1.3m 3
126,8
1,79
1,50
1.3o 1
49,5
1,89
2,10
1.3o 2
234,2
1,85
1,95
1.3o 3
103,9
1,86
2,12
1.5a 1
65,5
1,87
1,99
1.5a 2
108,7
1,84
2,14
1.5a 3
84,9
1,87
1,99
1.11i 1
114,0
1,82
1,95
1.11i 2
102,9
1,82
1,99
1.11i 3
125,6
1,85
2,05
All plasmids were restricted with SpeI and PstI to ligate cld directly behind the promoter constructs. One of each type of plasmids was also cut with EcoRI and SpeI in order to perform 3A assembly with cld and the pSB1K3 backbone. Restriction reaction went on for 2 hours.
We ligated cld after the promoters (located in pSB1C3). We also performed 3A assembly of the promoters (in pSB1C3), cld (in pBluescript with ampicillin resistance) and pSB1K3.
Transformations were done in dh5alpha as well as Bl21 (only for the T7 promoter construct 1.5m).
18/10 Sjoerd, Charlotte, Valentijn
Transformations of Bl21 did not show growth. Almost all grown colonies of the 3A assembly were white. We chose 10 colonies for each different promoter and did a colony PCR to check for our construct (figure not included).
After colony PCR, we chose several colonies which showed promising bands on the PCR gel, to try their growth on chlorite (concentrations of 0.24 mM and 0.5 mM). These colonies were also incubated o/n to prepare them for miniprep to obtain composite parts and a protein SDS-PAGE gel. Empty backbones pSB1C3 and pSB1A3 were used as control.
19/10 Sjoerd, Charlotte, Valentijn, Vincent
On our chlorite plates, we did not observe any growth at 0.5 mM chlorite. A-C on the figure below shows the pSB1A3 control construct, without any insert. Growth is visible on 0 mM chlorite (A) and 0.25 mM chlorite (B, on the top of the plate). D-F show the construct with the strong promoter and medium RBS. Growth is visible on the plate with 0 mM chlorite (D) and Little growth is seen on the plate with 0.25 mM chlorite (E) and no growth is observed at 0.50 mM chlorite (F). All other constructs show a similar pattern, suggesting that either the construct does not work, or the cld protein is not present at all.
We performed SDS-PAGE electrophoresis using all chosen constructs (16 total). We expected to see a band at 28 kDa, which would indicate the presence of CLD. However, no clear band was seen except for our control constructs, the pSB1C3 and pSB1A3 which show a band just above 28 kDa indicating RFP (28-30 kDa).
The results of the SDS-PAGE gel confirmed our hypothesis of cld not being present in our E. coli .
Due to the time, we were not able to perform any further experiments to characterize our constructs even better. It might be useful to add hemin to the chlorite plates (as described in Streit et al.) since hemin might be a co-factor for cld. :5271–5280.
For future research, other E. coli strains or other promoters could be tried.
References:
Streit BR , DuBois JL . 2008. Chemical and steady-state kinetic analyses of a heterologously expressed heme dependent chlorite dismutase. Biochemistry 47
