<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_Leiden_Design skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><DIV id="top_title"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:Leiden/Design</SPAN></H1></DIV><DIV id="HQ_page"><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><P><TITLE>
   iGEM Leiden Wiki : Overview
  </TITLE></P><DIV id="body-wrapper"><DIV class="jumbotron text-center"><P style="font-size: 100px; color: white; text-align: center; font-family: Montserrat, sans-serif; font-weight: normal;">
     Design
    </P></DIV></DIV><DIV id="Science behind the project" style="margin-top: -120px;" class="container-fluid bg-white"><DIV class="row"><DIV class="col-sm-8" style="text-align: justify;"><H2 style="margin-bottom: 20px;">
      The science behind the project
     </H2><P>
      Martian soil is contaminated with a toxic compound, perchlorate, making human life on mars impossible. For our project we use bacteria, which contain a perchlorate reduction system, to eliminate perchlorate from Martian soil. There are several bacteria that can already reduce perchlorate, one example is
      <I>
       Dechloromonas aromatica
      </I>
      . In this research we use the model organism
      <I>
       E. coli
      </I>
      as a host for perchlorate reduction. Using
      <I>
       E. coli
      </I>
      has multiple advantages. Firstly, a lot is known about
      <I>
       E. coli
      </I>
      which makes it a preferable host to work with in the lab and in practise, for example a bioreactor. Secondly,
      <I>
       E. coli
      </I>
      grows easier and faster on less complicated resources which is not trivial when in outer space. Thirdly, the iGEM library is extensive when it comes to
      <I>
       E. coli
      </I>
      parts, making it favourable to use
      <I>
       E. coli
      </I>
      as a chassis for multiple functions including perchlorate reduction. This way a combination of genetic changes can easily be achieved to produce the workhorse for martian colonization. Fourthly,
      <I>
       E. coli
      </I>
      is also a Gram-negative bacterium like
      <I>
       Dechloromonas aromatica
      </I>
      and the process requires a periplasm, making other hosts such as Bacillus subtillus not an option.
     </P><P><B>
       -Since
       <I>
        E. coli
       </I>
       does not contain a perchlorate reduction system, our goal is to transform
       <I>
        E. coli
       </I>
       with genes encoding for the perchlorate reduction system derived from
       <I>
        Dechloromonas aromatica
       </I>
       .
      </B></P><P>
      Previous studies have shown that bacteria can behave radically different under microgravity. In our research, we do not only focus on building our genetic biobricks, but also on their function on the place of destination. Martian gravity is different from zero gravity and the gravity on earth. The impact of Martian gravity on
      <I>
       E. coli
      </I>
      and its capacity to express the build in parts are tested using an random positioning machine (RPM). The bacterial growth under normal gravity and the gravity on mars are compared.
     </P><P><B>
       -Since not much is known about gene transcription and bacterial metabolism under Martian gravity, our goal is to investigate differences in expression patterns of genes using the RPM and RNAseq.
      </B></P><H3>
      Introducing perchlorate reducing genes into
      <I>
       E. coli
      </I>
      DH5α
     </H3><P>
      We aim to introduce the genes coding for perchlorate reduction in the
      <I>
       E. coli
      </I>
      DH5α strain
      <SUP><A href="https://2016.igem.org/Team:Leiden/#references">
        1
       </A></SUP>
      . The genes responsible for perchlorate reduction were synthesised by IDTDNA and codon optimized for
      <I>
       E. coli
      </I>
      K12 by their Codon Optimization Tool. With these genes perchlorate reducing bacteria like
      <I>
       Dechloromonas aromatica
      </I>
      can reduce perchlorate into chlorite (ClO-2 ) utilizing perchlorate reductase2. Subsequently, the bacterium can convert chlorite into oxygen (O2 ) and chloride (Cl- ) utilizing chlorite dismutase. This process is visualized in figure 1:
     </P><P><B>
       Figure 1. The perchlorate reduction system in
       <I>
        Dechloromonas aromatica
       </I>
       .
      </B>
      This illustrates the mode of action and proteins required to reduce perchlorate into chloride ions and oxygen. Adapted from Bender et al. 2005 and elaborated.
      <SUP><A href="https://2016.igem.org/Team:Leiden/#references">
        2
       </A></SUP></P><P>
      The genes encoding for perchlorate reductase and chlorite dismutase from
      <I>
       Dechloromonas aromatica
      </I>
      are part of a perchlorate reducing island shown in the figure 2:
     </P><P><B>
       Figure 2.
      </B><I>
       The perchlorate reduction genomic island of
      </I><I>
       Dechloromonas
      </I>
      aromatica. This includes all the genes necessary for the reduction of perchlorate
      <SUP><A href="https://2016.igem.org/Team:Leiden/#references">
        1
       </A></SUP>
      . This figure was taken from Melnyk et al. 2014.
     </P><P>
      Upstream of pcrA a group of regulatory genes is present, these genes were not cloned into
      <I>
       E. coli
      </I>
      . We want to replace these regulatory genes by our own Martian gravity induced promoter, see RPM. Initially we chose to order the genes pcrA, B, C and D in such fashion, that they had to be cloned into one vector as a whole, with the use of Gibson assembly. Later we ordered new DNA from IDTDNA where the pcrA, pcrB, pcrC and pcrD genes can be cloned into a vector separately from each other. Also, the genes DHC and QDH, the membrane-associated proteins, were cloned together into a vector. Finally, cld and moaA were cloned separately in two vectors.
     </P><P>
      We choose from a variety of regular constitutive and inducible promoters to express the genes necessary for perchlorate reduction in
      <I>
       E. coli
      </I>
      from the iGEM database. Next to these promoters other BioBricks related to our project are tested. For example some BioBricks which can function as an O2 sensor, or a promoter which is regulated by the presence of oxygen and absence of oxygen.
     </P><P>
      The codon optimized pcr B, pcr C, pcr D, cld, QDH&amp;DHC; and MoaA are successfully Gibson assembled into pBluescript and transformed into
      <I>
       E. coli
      </I>
      DH5a. These genes are all sequenced by Baseclear and found to be 100% correct when aligning with Snapgene. Afterwards these genes are cloned into pSB1C3.
     </P><P>
      The gene cld codes for the protein chlorite dismutase. We characterized the functionality of the chlorite dismutase induced by an arabinose dependent promoter (BBa_K808000).
     </P><P><B>
       Random positioning machine (RPM)
      </B></P><P>
      Since many bacteria are known to behave differently in space, we are interested in the effects of simulated Martian gravity on gene expression. Various bacteria are known to grow faster and become (more) pathogenic in space and we wanted to know if this was also the case on Mars. As such, we measured gene expression in 3 different conditions: 1g (control), 0.37g (Martian gravity) and 0g (space, ISS). Of course, we cannot actually travel to Mars to experience its gravity, but instead, we used the Random Positioning Machine (RPM 2.0) by Airbus Defence and Space Netherlands. The Random Positioning Machine by Airbus Defence and Space Netherlands is a machine that allows us to do the impossible: experience Martian gravity here on Earth! The machine calculates a path over 3 axes, that will average the gravity exerted on whatever we put inside. This way, we can simulate any amount of gravity between 0g (e.g. space, ISS) and 1g (Earth). Martian gravity is approximately 0.37g and we performed many of our experiments with this amount of gravity. This way, we can accurately predict the behaviour of our
      <I>
       E. coli
      </I>
      nizer, using experimental conditions that would also be experienced on Mars!

            Four strains,
      <I>
       Pseudomonas putida
      </I>
      and three different
      <I>
       E. coli
      </I>
      strains, were used. These strains were spread on agar plates and these plates were kept under different gravities: the gravity on earth and the gravity on Mars.
     </P><P><B>
       RNA sequencing
      </B></P><P>
      Using differential gene-expression. For the iGEM competition, we would like to identify highly up and downregulated genes, so that their respective promoters can be used as gravity BioBricks. This will allow other contestants to use our promoters which will ensure their intended system also works under Martian gravity. Furthermore, we want to asses whether
      <I>
       E. coli
      </I>
      displays the same changes on Mars as observed in space. Perhaps the Martian gravity is sufficient to suppress potential pathogenicity and perhaps we will uncover that it will be just as dangerous as in space! Either way, this information is extremely valuable to Martian colonizers, since they will be able to anticipate to the expected changes.
     </P><P><B>
       References:
      </B></P><P><OL style="font-size: 8pt;" id="references"><LI>
        Melnyk, R. A., Clark, I. C., Liao, A., &amp; Coates, J. D. (2014). Transposon and deletion mutagenesis of genes involved in perchlorate reduction in Azospira suillum PS. MBio, 5(1), e00769-13
       </LI><LI>
        Bender, K. S., Shang, C., Chakraborty, R., Belchik, S. M., Coates, J. D., &amp; Achenbach, L. A. (2005). Identification, characterization, and classification of genes encoding perchlorate reductase. Journal of Bacteriology, 187(15), 5090-5096
       </LI></OL></P></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></BODY></HTML>