<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_Leiden_RNASeq skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><DIV id="top_title"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:Leiden/RNASeq</SPAN></H1></DIV><DIV id="HQ_page"><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><P><TITLE>
   iGEM Leiden Wiki : RNAseq
  </TITLE></P><DIV id="body-wrapper"><DIV class="jumbotron text-center"><P style="font-size: 100px; color: white; text-align: center; font-family: Montserrat, sans-serif; font-weight: normal;">
     RNAseq
    </P></DIV><DIV id="Science behind the project" style="margin-top: -120px;" class="container-fluid bg-white"><DIV class="row"><DIV class="col-sm-8" style="text-align: justify;"><H2 style="margin-bottom: 20px;">
       RNA Sequencing
      </H2><P><B>
        Abstract
       </B>
       We compared
       <I>
        E. coli
       </I>
       BW25113 bacteria under three different conditions: controls (1g) simulating Martian (± 0.4g) and microgravity (± 0g). Subsequently, RNA isolation and state-of-the-art RNAseq were performed, revealing significant down regulation of a translation initiation factor (yciH, ) and increased expression of two membrane proteins (yceK, hdeD, p_fdr &lt; 0.1). Furthermore, under simulated Martian gravity, 56 antisense RNAs were found to be significantly differentially expressed, constituting 56% of differentially expressed RNAs. Genome annotation (GO) of the genes inhibited by the affected antisense RNAs in the 0.4g treatment shows that
       <I>
        E. coli
       </I>
       specifically appears to suffer from carbon-deprivation and acid stress. These results are intuitive, given the reduced or absent convection in partial and microgravity, respectively, leading to build-up of waste products. We conclude that
       <I>
        E. coli
       </I>
       primarily responds to perceived changes in gravity on a translational level.
      </P><P><B>
        Introduction
       </B>
       We are interested in the effects of simulated Martian gravity on the expression of genes, as measured by RNAseq. Primarily, we would like to identify highly up and downregulated genes, so that their respective promoters can be used as gravity BioBricks in the iGEM competition. Furthermore, we want to characterize the transcriptomic changes in
       <I>
        Escherichia coli
       </I>
       (
       <I>
        E. coli
       </I>
       ) with respect to stress and potential virulence, when partial gravity is simulated. To this end, we isolated RNA from bacterial colonies grown in the Random Positioning Machine (RPM 2.0), made by Airbus Defence and Space Netherlands.
       <SUP><A href="https://2016.igem.org/Team:Leiden/#references">
         1
        </A></SUP></P><P><B><I>
         E. coli
        </I>
        in microgravity
       </B><U><A href="https://2016.igem.org/Team:Leiden/RPM">
         Previous Research
        </A></U>
       A concise description of previous research on bacteria in microgravity can be read here:
       <A href="https://microbewiki.kenyon.edu/index.php/Bacterial_response_to_microgravity">
        https://microbewiki.kenyon.edu/index.php/Bacterial_response_to_microgravity
       </A>
       .
                The effects of partial gravity on bacterial gene expression have not been studied in depth before.
      </P><P><B>
        Methods
       </B><I>
        Growth
       </I>
       We isolated RNA from
       <I>
        E. coli
       </I>
       BW25113 colonies grown overnight for 16h20m in a 37°C stove on LB agar plates without antibiotics. Prior to plating the cultures, the BW25113 strain was grown overnight in a thermal shaker, to maintain an exponential phase culture. Bacteria were grown in triplets under three different conditions: 1g (control), 0.37g (Martian gravity) and 0g (microgravity). The 1g controls were placed in the same stove, next to the RPM, to accurately mimic the conditions under which the 0.37g and 0g experiments were grown. Lighting conditions in the stove were near constant. Throughout this report, the 0.37g treatment is referred to as 0.4g for brevity.
       <I>
        RNA isolation
       </I>
       After growth, colonies on the LB agar plates were quickly dissolved by pipetting 2ml LB on the plates. RNA was then isolated after mixing 1ml of dissolved culture with 2ml RNAprotect, both according to Qiagen protocol. RNA quality and concentration were assessed using a Nanodrop photospectrometer. Care was taken to avoid RNase activity.
       <I>
        Sequencing
       </I>
       Isolated RNA was stored at -80°C and sent to BaseClear for RNAseq high-throughput sequencing. Samples were treated with rRNA depletion to filter out the expectedly large amounts of (uninteresting) ribosomal RNA.
       <I>
        Alignment
       </I>
       The resulting reads per sample (~3GB per library) were aligned to a reference BW25113 genome available from the NCBI,
       <SUP><A href="https://2016.igem.org/Team:Leiden/#references">
         2
        </A></SUP>
       using Rockhopper software.
       <SUP><A href="https://2016.igem.org/Team:Leiden/#references">
         3
        </A></SUP>
       For compatibility purposes, the genome feature file was converted to .ptt using gbk2ptt software.
       <SUP><A href="https://2016.igem.org/Team:Leiden/#references">
         4
        </A></SUP>
       Rockhopper aligns all sequenced RNA fragments to the genome and then counts the number of aligned sequences per gene, yielding raw count data. In total, reads were aligned to 7784 (predicted) RNAs.
       <I>
        Normalisation
       </I>
       Since raw counts are poorly comparable between and within samples, as discussed in Dillies et al. (2012), various normalization techniques are available. The most common method for normalization is reads per kilobase million (RPKM), which accounts for differences in gene length that is expected to positively correlate with read count. Though still widely used, RPKM values have on multiple occasions been reported to be biased and are deferred to other methods.
       <SUP><A href="https://2016.igem.org/Team:Leiden/#references">
         5
        </A></SUP>
       We used the trimmed mean of M-values (TMM), described in Robinson et al. (2010a) to normalize samples, while also accounting for total library size of each sample.
       <SUP><A href="https://2016.igem.org/Team:Leiden/#references">
         6
        </A></SUP>
       This technique was among the preferred methods in Dillies et al. (2012).
       <I>
        Statistical Analysis
       </I>
       Reads are count data, which can be assumed to follow a Poisson distribution. However, due to the many unknown sources of variance within and between samples, these counts are overdispersed with respect to a Poisson distribution. This implies that we can fit a GLM type regression model with a negative binomial error structure. The Bioconductor package edgeR has implemented this type of analysis and includes a built-in function to calculate TMM normalization.
       <SUP><A href="https://2016.igem.org/Team:Leiden/#references">
         6,7
        </A></SUP>
       , edgeR estimates tag-wise or common dispersion, depending on the assumptions made. We chose to estimate tag-wise dispersion, meaning each gene has its own dispersion estimate. Since RNAseq analysis involves considerable multiple testing (k = 7784, n = 3), uncorrected p-values would yield unacceptably high false positive rates. Therefore, false discovery rate (FDR) corrected p-values are reported in the results section. Furthermore, significance can be achieved without meaningful interpretation when the relative difference in expression is very low. Hence, ²log-fold changes are reported in addition to p-values.
       
       All analyses were with the statistical software R version 3.2.4 (R Development Core Team 2015).
       <SUP><A href="https://2016.igem.org/Team:Leiden/#references">
         8
        </A></SUP></P><P><B>
        Results
       </B><I>
        RNA Isolation
       </I>
       In table 1, the yields of each RNA isolation are shown, as reported by BaseClear’s quality control check. 8 out of 9 samples were successfully sequenced in the first run by BaseClear and the last sample (0g, replicate 2) was run again at a later time. Since this sample comes from a different batch, there may be larger dispersion in the 0g treatment than the other treatment.
       <B>
        Table 1.
       </B>
       RNA Isolation yields
       <TABLE class="frtable"><TBODY><TR><TD>
          ID
         </TD><TD>
          treatment
         </TD><TD>
          concentration (ng/microliter)
         </TD><TD>
          batch
         </TD></TR><TR><TD>
          1
         </TD><TD>
          1g (control)
         </TD><TD>
          1429
         </TD><TD>
          1
         </TD></TR><TR><TD>
          2
         </TD><TD>
          1g (control)
         </TD><TD>
          1316
         </TD><TD>
          1
         </TD></TR><TR><TD>
          3
         </TD><TD>
          1g (control)
         </TD><TD>
          2653
         </TD><TD>
          1
         </TD></TR><TR><TD>
          4
         </TD><TD>
          0g
         </TD><TD>
          1359
         </TD><TD>
          1
         </TD></TR><TR><TD>
          5
         </TD><TD>
          0g
         </TD><TD>
          1072
         </TD><TD>
          2
         </TD></TR><TR><TD>
          6
         </TD><TD>
          0g
         </TD><TD>
          1700
         </TD><TD>
          1
         </TD></TR><TR><TD>
          7
         </TD><TD>
          0.4g
         </TD><TD>
          8835
         </TD><TD>
          1
         </TD></TR><TR><TD>
          8
         </TD><TD>
          0.4g
         </TD><TD>
          4256
         </TD><TD>
          1
         </TD></TR><TR><TD>
          9
         </TD><TD>
          0.4g
         </TD><TD>
          2760
         </TD><TD>
          1
         </TD></TR></TBODY></TABLE></P><P><I>
        Differential Expression
       </I>
       Using a negative binomial GLM with tag-wise dispersion, 100 genes were differentially expressed in the 0.4g treatment at a false discovery rate of p_FDR &lt; 0.1, of which 67 remain significant at p_FDR &lt; 0.05. Contrastingly, the 0g treatment yielded only 3 significantly differentially expressed genes at p_FDR &lt; 0.1, of which none remained significant at the most stringent level. These genes are displayed in table
       <A href="https://static.igem.org/mediawiki/2016/7/7a/Team-Leiden-images-rnaseq-table_s1.xlsx">
        table S1
       </A>
       and include genome annotations of genes or affected genes in case of antisense RNA.
       </P><P><I>
        Antisense RNA
       </I>
       From the annotations included in
       <A href="https://static.igem.org/mediawiki/2016/7/7a/Team-Leiden-images-rnaseq-table_s1.xlsx">
        table S1
       </A>
       , it would appear that there is a particularly large number of antisense RNAs affected by the simulated reduced gravity. For the 0.4g treatment, we compared the number of significant antisense RNAs to the total number of significant RNAs (table 2). A simple chi^2-test for given probabilities shows that there is a larger number of significantly differentially expressed antisense RNAs than would be expected from the proportion of aligned antisense RNAs (p = 0.012).
       </P><P><B>
        Table 2.
       </B>
       Comparison of sense and antisense aligned RNAs
      </P><TABLE class="frtable"><TBODY><TR><TD>
         sense
        </TD><TD>
         antisense
        </TD><TD>
         unknown
        </TD><TD>
         total
        </TD></TR><TR><TD>
         total aligned
        </TD><TD>
         4131
        </TD><TD>
         3384
        </TD><TD>
         269
        </TD><TD>
         7764
        </TD></TR><TR><TD>
         p_FDR &lt; 0.1 (0.4g)
        </TD><TD>
         3
        </TD><TD>
         56
        </TD><TD>
         41
        </TD><TD>
         100
        </TD></TR><TR><TD>
         p_FDR &lt; 0.1 (0g)
        </TD><TD>
         1
        </TD><TD>
         2
        </TD><TD>
         0
        </TD><TD>
         3
        </TD></TR></TBODY></TABLE><P><B>
        Figure 1.
       </B>
       displays the (log) fold-changes in gene expression found to be significant plotted against their relative abundance (log counts per million). On the left, the 100 differentially expressed RNAs in the 0.4g treatment (p_FDR &lt; 0.1) are shown in red. On the right, the same is displayed for the 3 RNAs in the 0g treatment (p_FDR &lt; 0.1). Some genes are technically significant, but have such a low relative abundance that their relevance is questionable. These are shown as orange dots on the left side of either plot. Significantly differentially expressed genes compared to the 1g control (red). On the x-axis, the relative abundance is shown, expressed in (log) counts per million (CPM).On the y-axis, the (log) fold-change compared to the 1g control is shown. Genes that did not achieve statistical significance at the level of p_FDR &lt; 0.1 are shown as black dots. Genes that are statistically significant, albeit biologically uninteresting due to their extremely low expression are shown as orange dots. A label (mgtA) was added for a strongly underexpressed antisense RNA in both groups.
       </P><P><B>
        Discussion
       </B><I>
        General Discussion
       </I>
       Interestingly, it would appear in figure 1 that the fold-changes in the 0.4g treatment are symmetrically distributed with central tendency. The same cannot be said for the 0g treatment, which shows skew towards positive fold-changes. This discrepancy could be attributed to the sample that was run a second time (in a different batch) in the 0g replicates (table 1). We also believe this to be the main reason for the large difference in (the number of) significantly differentially expressed RNAs for either treatment (0.4g and 0g, respectively). Due to the time pressure of the competition in context of which this study was performed and the large costs of RNAseq, we were not able to redo the experiment. Despite this complication, an interesting observation can be made in the annotations of
       <A href="https://static.igem.org/mediawiki/2016/7/7a/Team-Leiden-images-rnaseq-table_s1.xlsx">
        table S1
       </A>
       . Namely, without regard for significance, it would appear that many of the differentially expressed genes under simulated Martian gravity display similar trends in the 0g treatment. Whether the result of reduced gravity or of the RPM’s movement, in either case this would suggest reproducibility of the experiment, since we expect similar changes to occur in partial and microgravity. A follow-up, using less expensive and time-consuming qPCR, will attempt to confirm the differentially expressed RNAs with a larger sample size.
       <I>
        E. coli is primarily affected on a translational level
       </I>
       As summarized in table 2, a large fraction of significantly differentially expressed RNAs are antisense RNAs. This could imply that
       <I>
        E. coli
       </I>
       copes with the perceived stress from reduced gravity primarily on a translational level, although changes on transcriptional or genomic level may follow after longer incubation time. Additional support for this conclusion comes from the fact that out of the three mRNAs affected, one is a translation initiation factor.
       <I>
        Stress from reduced gravity
       </I>
       Many of the affected antisense RNAs complement mRNAs related to stress response. For example, a near 16-fold decrease in expression of antisense mgtA was consistently observed among treatments (figure 1, labeled mgtA). This implies an increase in mgtA mRNA activity and thereby increased magnesium transport, which is a common stress response. Other stress related antisense RNAs observed in the 0.4g treatment include cold shock protein, heat shock protein and acid resistance membrane protein.
       <I>
        Difference between partial and microgravity
       </I>
       In the 0g treatment, multiple instances of antisense fliC showed trends towards overexpression, of which one was significant at p_FDR &lt; 0.1. Antisense fliC inhibits flic mRNA, which codes for a part of the bacterium’s flagellum. Interestingly, previous research by NASA led researchers to conclude that flagella do not work in microgravity. It would appear that
       <I>
        E. coli
       </I>
       quickly adapts by saving energy on flagella development in microgravity, but not in partial gravity.
       </P><P><B>
        Conclusion
       </B>
       Genome annotation (GO) of the genes inhibited by the affected antisense RNAs in the 0.4g treatment shows that
       <I>
        E. coli
       </I>
       specifically appears to suffer from carbon-deprivation and acid stress. This could be attributed to the reduced or absent convection in partial and microgravity, respectively, leading to build-up of waste products. Additional evidence for this claim can be seen in the affected outer membrane lipoproteins, which are both significantly underexpressed for four antisense RNAs (p_FDR &lt; 0.05) and significantly overexpressed in one mRNA (p_FDR &lt; 0.1).
      </P><P><B>
        References
       </B><OL id="references" style="font-size: 8pt"><LI><A href="https://2016.igem.org/Team:Leiden/www_partialgravity_com">
          www.partialgravity.com
         </A></LI><LI><A href="http://www.ncbi.nlm.nih.gov/nuccore/749300132">
          http://www.ncbi.nlm.nih.gov/nuccore/749300132
         </A></LI><LI><A href="http://cs.wellesley.edu/~btjaden/Rockhopper/index.html">
          http://cs.wellesley.edu/~btjaden/Rockhopper/index.html
         </A></LI><LI><A href="https://github.com/ajvilleg/gbk2ptt">
          https://github.com/ajvilleg/gbk2ptt
         </A></LI><LI>  Dillies et al. (2012): A comprehensive evaluation of normalization methods for Illumina high-throughput RNA sequencing data analysis
        </LI><LI>
         Robinson, MD, and Oshlack, A (2010a). A scaling normalization method for differential expression analysis of RNA-seq data. Genome Biology 11, R25.
        </LI><LI>
         Robinson MD, McCarthy DJ and Smyth GK (2010b). “edgeR: a Bioconductor package for differential expression analysis of digital gene expression data.” Bioinformatics, 26, pp. -1.
        </LI><LI>
         R Development Core Team. 2015. R: A Language and Environment for Statistical Computing. Vienna, Austria: R Foundation for Statistical Computing. http://www.R-project.org.
        </LI></OL></P></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></BODY></HTML>