"
Page
Discussion
View source
History
teams
Log in
 
Team:TU Darmstadt/Materials/Software
From 2012.igem.org
iGEM@TUD
Home
Project
Overview
1. Degradation
2. Transport
3. Metabolism
4. Material Science
5. Simulation
Ecology
Philosophy
Team
Members
Supporters
BioBricks
Documentation
Overview
Labjournal
Protocols
Materials
Modeling
Safety
Downloads
Human Practice
Sponsors
Overview
Benefits
Software
Visualistaion
Molecular Graphic and animations we used yasara:
Krieger, E., Koraimann, G., & Vriend, G. (2002). Increasing the precision of comparative model with YASARA NOVA--a self-parameterizing force field. Proteins, 47(3), 393-402. Wiley Online Library. Retrieved from http://www.ncbi.nlm.nih.gov/pubmed/11948792
Molecular Graphic for the Information Theory we used vmd  :
Humphrey, W., Dalke, A. and Schulten, K., "VMD - Visual Molecular Dynamics", J. Molec. Graphics, 1996, vol. 14, pp. 33-38.
Molecular Graphic for homology modeling we used Chimera  :
Pettersen, E. F., Goddard, T. D., Huang, C. C., Couch, G. S., Greenblatt, D. M., Meng, E. C., & Ferrin, T. E. (2004). UCSF Chimera--a visualization system for exploratory research and analysis. J Comput Chem, 25(13), 1605-1612. doi:10.1002/jcc.20084
Computation
For plots and calculations we use R  :
Team, R. D. C. (2008). R: A Language and Environment for Statistical Computing. Vienna, Austria.
For the GNM calculations we used :
Hoffgaard, F., Weil, P., & Hamacher, K. (2010). BioPhysConnectoR: Connecting sequence information and biophysical models. BMC Bioinformatics, 11, 199. doi:10.1186/1471-2105-11-199
For alignment creation we used Clustal omega
Sievers, F., Wilm, A., Dineen, D., Gibson, T. J., Karplus, K., Li, W., Lopez, R., et al. (2011). Fast, scalable generation of high-quality protein multiple sequence alignments using Clustal Omega. Molecular Systems Biology, 7(539), 539. Nature Publishing Group. doi:10.1038/msb.2011.75
For the sequnece search we used BLAST:
Altschul, S. F., Madden, T. L., Schäffer, A. A., Zhang, J., Zhang, Z., Miller, W., & Lipman, D. J. (1997). Gapped BLAST and PSI-BLAST: a new generation of protein database search programs. Nucleic Acids Res, 25(17), 3389-3402.
For molecular dynamics we used Yasara:
Krieger, E., Koraimann, G., & Vriend, G. (2002). Increasing the precision of comparative model with YASARA NOVA--a self-parameterizing force field. Proteins, 47(3), 393-402. Wiley Online Library. link
For docking simulation :
Automated Docking Using a Lamarckian Genetic Algorithm and and Empirical Binding Free Energy Function Morris GM, Goodsell DS, Halliday RS, Huey R, Hart WE, Belew RK and Olson AJ (1998), J.Comput.Chem. 19, 1639-1662
Retrieved from " http://2012.igem.org/Team:TU_Darmstadt/Materials/Software "
Recent changes
What links here
Related changes
Special pages
My preferences
Printable version
Permanent link
Privacy policy
Disclaimers
