<HTML xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en" dir="ltr">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki  ltr ns-0 ns-subject page-Team_TU_Darmstadt_Protocols_Information_theory"><DIV id="globalWrapper"><DIV id="top-section"><DIV id="p-logo"><A href="/Main_Page" title="Main Page">&quot;
	    </A></DIV><DIV id="menubar" class="left-menu noprint"><UL><LI class="selected"><A href="/Team:TU_Darmstadt/Protocols/Information_theory">Page               </A></LI><LI class="new"><A href="/wiki/index.php?title=Talk:Team:TU_Darmstadt/Protocols/Information_theory&amp;action=edit&amp;redlink=1">Discussion               </A></LI><LI><A href="/wiki/index.php?title=Team:TU_Darmstadt/Protocols/Information_theory&amp;action=edit">View source               </A></LI><LI><A href="/wiki/index.php?title=Team:TU_Darmstadt/Protocols/Information_theory&amp;action=history">History               </A></LI><LI style="color:#808080;cursor:default">teams</LI></UL></DIV><DIV class="right-menu noprint" id="menubar"><UL><LI id="pt-login"><A href="/wiki/index.php?title=Special:UserLogin&amp;returnto=Team:TU_Darmstadt/Protocols/Information_theory" title="You are encouraged to log in; however, it is not mandatory [o]" accesskey="o">Log in</A></LI></UL></DIV><DIV id="search-controls" class="noprint"><FORM action="/Special:Search" id="searchform"> </FORM></DIV></DIV><DIV id="content"><H1 class="firstHeading">Team:TU Darmstadt/Protocols/Information theory</H1><DIV id="bodyContent"><H3 id="siteSub" class="noprint">From 2012.igem.org</H3><DIV id="TUD"><DIV id="top"><UL><LI><A href="http://www.igem.tu-darmstadt.de/igem/projekt/index.en.jsp" title="iGEM">iGEM@TUD</A></LI><LI><A href="/Team:TU_Darmstadt" title="Home">Home</A></LI><LI><A href="/Team:TU_Darmstadt/Project" title="Project">Project</A></LI><LI><A href="/Team:TU_Darmstadt/Project" title="Team:Overview">Overview</A></LI><LI><A href="/Team:TU_Darmstadt/Project/Degradation" title="Degradation">1. Degradation</A></LI><LI><A href="/Team:TU_Darmstadt/Project/Transport" title="Transport">2. Transport</A></LI><LI><A href="/Team:TU_Darmstadt/Project/Metabolism" title="Metabolism">3. Metabolism</A></LI><LI><A href="/Team:TU_Darmstadt/Project/Material_Science" title="Material Science">4. Material Science</A></LI><LI><A href="/Team:TU_Darmstadt/Project/Simulation" title="Simulation">5. Simulation</A></LI><LI><A href="/Team:TU_Darmstadt/Project/Ecology" title="Ecology">Ecology</A></LI><LI><A href="/Team:TU_Darmstadt/Project/Philosophy" title="Philosophy">Philosophy</A></LI></UL><LI><A href="/Team:TU_Darmstadt/Team" title="Team:TU_Darmstadt/Team">Team</A></LI><LI><A href="/Team:TU_Darmstadt/Team" title="Team:TU_Darmstadt/Team">Members</A></LI><LI><A href="/Team:TU_Darmstadt/Supporters" title="Team:TU_Darmstadt/Supporters">Supporters</A></LI><LI><A href="/Team:TU_Darmstadt/Parts" title="BioBricks">BioBricks</A></LI><LI><A href="/Team:TU_Darmstadt/Documentation" title="Documentation">Documentation</A></LI><LI><A href="/Team:TU_Darmstadt/Documentation" title="Documentation">Overview</A></LI><LI><A href="/Team:TU_Darmstadt/Labjournal" title="Labjournal">Labjournal</A></LI><LI><A href="/Team:TU_Darmstadt/Protocols" title="Protocols">Protocols</A></LI><LI><A href="/Team:TU_Darmstadt/Materials" title="Materials">Materials</A></LI><LI><A href="/Team:TU_Darmstadt/Modeling" title="Modeling">Modeling</A></LI><LI><A href="/Team:TU_Darmstadt/Safety" title="Safety">Safety</A></LI><LI><A href="/Team:TU_Darmstadt/Downloads" title="Downloads">Downloads</A></LI><LI><A href="/Team:TU_Darmstadt/Human_Practice" title="Human Practice">Human Practice</A></LI><LI><A href="/Team:TU_Darmstadt/Sponsors" title="Sponsors">Sponsors</A></LI><LI><A href="/Team:TU_Darmstadt/Sponsors" title="Sponsors">Overview</A></LI><LI><A href="/Team:TU_Darmstadt/Contact" title="Contact">Benefits</A></LI></DIV><TABLE id="toc" class="toc"><TBODY><TR><TD><DIV id="toctitle"><H2>Contents</H2></DIV><UL><LI class="toclevel-1 tocsection-1"><A href="#Information_Theory"><SPAN class="tocnumber">1</SPAN><SPAN class="toctext">Information Theory</SPAN></A></LI><LI class="toclevel-2 tocsection-2"><A href="#Entropy"><SPAN class="tocnumber">1.1</SPAN><SPAN class="toctext">Entropy</SPAN></A></LI><LI class="toclevel-2 tocsection-3"><A href="#Mutual_Information"><SPAN class="tocnumber">1.2</SPAN><SPAN class="toctext">Mutual Information</SPAN></A></LI></UL><LI class="toclevel-1 tocsection-4"><A href="#Application_of_MI_to_sequence_Alignments"><SPAN class="tocnumber">2</SPAN><SPAN class="toctext">Application of MI to sequence Alignments</SPAN></A></LI><LI class="toclevel-2 tocsection-5"><A href="#Normalisation"><SPAN class="tocnumber">2.1</SPAN><SPAN class="toctext">Normalisation</SPAN></A></LI><LI class="toclevel-1 tocsection-6"><A href="#References"><SPAN class="tocnumber">3</SPAN><SPAN class="toctext">References</SPAN></A></LI></TD></TR></TBODY></TABLE><H2><SPAN class="mw-headline" id="Information_Theory"> Information Theory </SPAN></H2><H3><SPAN class="mw-headline" id="Entropy">Entropy</SPAN></H3><P>Claude Shannon created a new measurement approach of uncertainty of a random variable X. This measurement is called Shannon’s entropy H [1] which is measured in bit, if a logarithm to the base 2 is used. p(x) denotes the probability mass function of a random variable X.
</P><P>&lt;math&gt;H(X)=-\sum\limits_{x}p(x)\cdot \log_{2} p(x)&lt;math&gt;</P><H3><SPAN class="mw-headline" id="Mutual_Information">Mutual Information</SPAN></H3><P>In information theory, Mutual information (MI) is a correlations measure of two random variables X and Y . H(X) and H(Y ) are the Shannon entropy values of the random variables X and Y. H(X, Y ) is the two-point entropy. Moreover , the MI quantiﬁes the amount of information of variable X by knowing Y and vice versa. 
&lt;math&gt;MI(X;Y)=H(X)+H(Y)-H(X,Y)&lt;math&gt;</P><H2><SPAN class="mw-headline" id="Application_of_MI_to_sequence_Alignments">Application of MI to sequence Alignments</SPAN></H2><P>It is well known that the MI can be used to measure co-evolution signals in multiple sequence alignments (MSA)[2] [3] . An MSA serves as a comparison of three or more sequences used to investigate the functional or evolutionary homology of amino acid or nucleotide sequences. The MI of an MSA can be computed with the following equation derived from the Kullback-Leibler-Divergence:
</P><P>&lt;math&gt;MI(X;Y)= \sum\limits_{i=1}^{Q}\sum\limits_{j=1}^{Q}p(x_{i},y_{j})\cdot \log_{2}\left( \frac{p(x_{i},y_{j})}{p(x_{i})p(y_{j})}\right)&lt;math&gt;</P><P>with p(x) and p( y) being the frequency counts of symbols in column X and Y of the MSA. The joint frequency describe the occurrence for the amino acids xi and yj(p(x, y)) and Q is the set of  Symbols derived from the corresponding alphabet (DNA or Protein). The result of these calculations is a symmetric matrix M which includes all combined MI values for any two columns in an MSA. A dependency of two columns acids shows high MI values.
</P><H3><SPAN class="mw-headline" id="Normalisation">Normalisation</SPAN></H3><P>A standard score (Z-score) indicates how many standard deviations a value differs from the mean of a normal distribution. MI dependent Z-scores can be calculated with a shufﬂe-null model, where the symbols in MSA column are shufﬂed and every dependencies of the column pairs are eliminated. The expectation value for the shufﬂe-null model is described by E(Mi j) and its corresponding variance by Var(Mi j) [4].
</P><P>&lt;math&gt;Z_{ij}=\frac{M_{ij}-E(M_{ij})}{\sqrt{Var(M_{ij})}}&lt;math&gt;</P><H2><SPAN class="mw-headline" id="References">References</SPAN></H2><P>[1]	C. E. Shanon, “A Mathematical Theory of Communication,” The Bell System Technical Journal, vol. 27, pp. 379–423, 1948.
</P><P>[2]	K. Hamacher, “Relating sequence evolution of HIV1-protease to its underlying molecular mechanics.,” Gene, vol. 422, no. 1–2, pp. 30–36, 2008.
</P><P>[3]	F. H. K. H. P. Boba P. Weil, “Intra- and Inter-Molecular Co-Evolution: The Case of of HIV1 Protease and Reverse Transcriptase,” Springer Communications in Computer and Information Science, pp. 127:356–366, 2011.
</P><P>[4]	P. Weil, F. Hoffgaard, and K. Hamacher, “Estimating sufficient statistics in co-evolutionary analysis by mutual information.,” Comput Biol Chem, vol. 33, no. 6, pp. 440–444, 2009.
</P><DIV class="printfooter">
Retrieved from &quot;<A href="http://2012.igem.org/Team:TU_Darmstadt/Protocols/Information_theory">http://2012.igem.org/Team:TU_Darmstadt/Protocols/Information_theory</A>&quot;</DIV></DIV></DIV><DIV id="footer-box" class="noprint"><DIV id="footer"><UL id="f-list"><LI id="t-recentchanges"><A href="/Special:RecentChanges" title="Recent changes">Recent changes</A></LI><LI id="t-whatlinkshere"><A href="/Special:WhatLinksHere/Team:TU_Darmstadt/Protocols/Information_theory" title="List of all wiki pages that link here [j]" accesskey="j">What links here</A></LI><LI id="t-recentchangeslinked"><A href="/Special:RecentChangesLinked/Team:TU_Darmstadt/Protocols/Information_theory" title="Recent changes in pages linked from this page [k]" accesskey="k">Related changes</A></LI><LI id="t-specialpages"><A href="/Special:SpecialPages" title="List of all special pages [q]" accesskey="q">Special pages</A></LI><LI><A href="/Special:Preferences">My preferences</A></LI></UL></DIV><DIV id="footer"><UL id="f-list"><LI id="t-print"><A href="/wiki/index.php?title=Team:TU_Darmstadt/Protocols/Information_theory&amp;printable=yes" title="Printable version of this page [p]" accesskey="p">Printable version</A></LI><LI id="t-permalink"><A href="/wiki/index.php?title=Team:TU_Darmstadt/Protocols/Information_theory&amp;oldid=217856" title="Permanent link to this revision of the page">Permanent link</A></LI><LI id="privacy"><A href="/2012.igem.org:Privacy_policy" title="2012.igem.org:Privacy policy">Privacy policy</A></LI><LI id="disclaimer"><A href="/2012.igem.org:General_disclaimer" title="2012.igem.org:General disclaimer">Disclaimers</A></LI></UL></DIV></DIV></DIV></DIV></BODY></HTML>