<HTML xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en" dir="ltr">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki  ltr ns-0 ns-subject page-Team_TU_Darmstadt_Protocols_Metabolism"><DIV id="globalWrapper"><DIV id="top-section"><DIV id="p-logo"><A href="/Main_Page" title="Main Page">&quot;
	    </A></DIV><DIV id="menubar" class="left-menu noprint"><UL><LI class="selected"><A href="/Team:TU_Darmstadt/Protocols/Metabolism">Page               </A></LI><LI class="new"><A href="/wiki/index.php?title=Talk:Team:TU_Darmstadt/Protocols/Metabolism&amp;action=edit&amp;redlink=1">Discussion               </A></LI><LI><A href="/wiki/index.php?title=Team:TU_Darmstadt/Protocols/Metabolism&amp;action=edit">View source               </A></LI><LI><A href="/wiki/index.php?title=Team:TU_Darmstadt/Protocols/Metabolism&amp;action=history">History               </A></LI><LI style="color:#808080;cursor:default">teams</LI></UL></DIV><DIV class="right-menu noprint" id="menubar"><UL><LI id="pt-login"><A href="/wiki/index.php?title=Special:UserLogin&amp;returnto=Team:TU_Darmstadt/Protocols/Metabolism" title="You are encouraged to log in; however, it is not mandatory [o]" accesskey="o">Log in</A></LI></UL></DIV><DIV id="search-controls" class="noprint"><FORM action="/Special:Search" id="searchform"> </FORM></DIV></DIV><DIV id="content"><H1 class="firstHeading">Team:TU Darmstadt/Protocols/Metabolism</H1><DIV id="bodyContent"><H3 id="siteSub" class="noprint">From 2012.igem.org</H3><DIV id="TUD"><DIV id="top"><UL><LI><A href="http://www.igem.tu-darmstadt.de/igem/projekt/index.en.jsp" title="iGEM">iGEM@TUD</A></LI><LI><A href="/Team:TU_Darmstadt" title="Home">Home</A></LI><LI><A href="/Team:TU_Darmstadt/Project" title="Project">Project</A></LI><LI><A href="/Team:TU_Darmstadt/Project" title="Team:Overview">Overview</A></LI><LI><A href="/Team:TU_Darmstadt/Project/Degradation" title="Degradation">1. Degradation</A></LI><LI><A href="/Team:TU_Darmstadt/Project/Transport" title="Transport">2. Transport</A></LI><LI><A href="/Team:TU_Darmstadt/Project/Metabolism" title="Metabolism">3. Metabolism</A></LI><LI><A href="/Team:TU_Darmstadt/Project/Material_Science" title="Material Science">4. Material Science</A></LI><LI><A href="/Team:TU_Darmstadt/Project/Simulation" title="Simulation">5. Simulation</A></LI><LI><A href="/Team:TU_Darmstadt/Project/Ecology" title="Ecology">Ecology</A></LI><LI><A href="/Team:TU_Darmstadt/Project/Philosophy" title="Philosophy">Philosophy</A></LI></UL><LI><A href="/Team:TU_Darmstadt/Team" title="Team:TU_Darmstadt/Team">Team</A></LI><LI><A href="/Team:TU_Darmstadt/Team" title="Team:TU_Darmstadt/Team">Members</A></LI><LI><A href="/Team:TU_Darmstadt/Supporters" title="Team:TU_Darmstadt/Supporters">Supporters</A></LI><LI><A href="/Team:TU_Darmstadt/Parts" title="BioBricks">BioBricks</A></LI><LI><A href="/Team:TU_Darmstadt/Documentation" title="Documentation">Documentation</A></LI><LI><A href="/Team:TU_Darmstadt/Documentation" title="Documentation">Overview</A></LI><LI><A href="/Team:TU_Darmstadt/Labjournal" title="Labjournal">Labjournal</A></LI><LI><A href="/Team:TU_Darmstadt/Protocols" title="Protocols">Protocols</A></LI><LI><A href="/Team:TU_Darmstadt/Materials" title="Materials">Materials</A></LI><LI><A href="/Team:TU_Darmstadt/Modeling" title="Modeling">Modeling</A></LI><LI><A href="/Team:TU_Darmstadt/Safety" title="Safety">Safety</A></LI><LI><A href="/Team:TU_Darmstadt/Downloads" title="Downloads">Downloads</A></LI><LI><A href="/Team:TU_Darmstadt/Human_Practice" title="Human Practice">Human Practice</A></LI><LI><A href="/Team:TU_Darmstadt/Human_Practice/Panel_Discussion " title="Panel Discussion">Panel Discussion</A></LI><LI><A href="/Team:TU_Darmstadt/Human_Practice/Symposia" title="Symposia">Symposia</A></LI><LI><A href="/Team:TU_Darmstadt/Human_Practice/Classes" title="Classes">Classes</A></LI><LI><A href="/Team:TU_Darmstadt/Sponsors" title="Sponsors">Sponsors</A></LI><LI><A href="/Team:TU_Darmstadt/Sponsors" title="Sponsors">Overview</A></LI><LI><A href="/Team:TU_Darmstadt/Contact" title="Contact">Benefits</A></LI></DIV><P><SPAN style="font-size:200%;"><SPAN style="color:#00689D;">Protocols Metabolism</SPAN></SPAN></P><P>The following page gives an overview of the standard protocols used by the team Metabolism in the iGEM PET.erminators project 2012.
</P><TABLE id="toc" class="toc"><TBODY><TR><TD><DIV id="toctitle"><H2>Contents</H2></DIV><UL><LI class="toclevel-1 tocsection-1"><A href="#In_vivo"><SPAN class="tocnumber">1</SPAN><SPAN class="toctext">In vivo</SPAN></A></LI><LI class="toclevel-2 tocsection-2"><A href="#Production_of_chemically_competent_cells"><SPAN class="tocnumber">1.1</SPAN><SPAN class="toctext">Production of chemically competent cells</SPAN></A></LI><LI class="toclevel-2 tocsection-3"><A href="#Heat_shock_transformation_with_E._coli"><SPAN class="tocnumber">1.2</SPAN><SPAN class="toctext">Heat shock transformation with E. coli</SPAN></A></LI><LI class="toclevel-2 tocsection-4"><A href="#Glycerine_stock"><SPAN class="tocnumber">1.3</SPAN><SPAN class="toctext">Glycerine stock</SPAN></A></LI></UL><LI class="toclevel-1 tocsection-5"><A href="#In_vitro"><SPAN class="tocnumber">2</SPAN><SPAN class="toctext">In vitro</SPAN></A></LI><LI class="toclevel-2 tocsection-6"><A href="#PCR"><SPAN class="tocnumber">2.1</SPAN><SPAN class="toctext">PCR</SPAN></A></LI><LI class="toclevel-3 tocsection-7"><A href="#Colony_PCR_.28isolation_of_genomic_DNA.29"><SPAN class="tocnumber">2.1.1</SPAN><SPAN class="toctext">Colony PCR (isolation of genomic DNA)</SPAN></A></LI><LI class="toclevel-3 tocsection-8"><A href="#Colony_PCR_.28verification_of_transformation.29"><SPAN class="tocnumber">2.1.2</SPAN><SPAN class="toctext">Colony PCR (verification of transformation)</SPAN></A></LI><LI class="toclevel-3 tocsection-9"><A href="#PCR_on_a_DNA_template"><SPAN class="tocnumber">2.1.3</SPAN><SPAN class="toctext">PCR on a DNA template</SPAN></A></LI><LI class="toclevel-2 tocsection-10"><A href="#Restriction_digest"><SPAN class="tocnumber">2.2</SPAN><SPAN class="toctext">Restriction digest</SPAN></A></LI><LI class="toclevel-2 tocsection-11"><A href="#Dephosphorylation"><SPAN class="tocnumber">2.3</SPAN><SPAN class="toctext">Dephosphorylation</SPAN></A></LI><LI class="toclevel-2 tocsection-12"><A href="#Ligation"><SPAN class="tocnumber">2.4</SPAN><SPAN class="toctext">Ligation</SPAN></A></LI><LI class="toclevel-2 tocsection-13"><A href="#Plasmid_Preparation"><SPAN class="tocnumber">2.5</SPAN><SPAN class="toctext">Plasmid Preparation</SPAN></A></LI><LI class="toclevel-3 tocsection-14"><A href="#Plasmid_Miniprep"><SPAN class="tocnumber">2.5.1</SPAN><SPAN class="toctext">Plasmid Miniprep</SPAN></A></LI><LI class="toclevel-3 tocsection-15"><A href="#Plasmid_Midiprep"><SPAN class="tocnumber">2.5.2</SPAN><SPAN class="toctext">Plasmid Midiprep</SPAN></A></LI><LI class="toclevel-2 tocsection-16"><A href="#Agarose_gel_electrophoresis"><SPAN class="tocnumber">2.6</SPAN><SPAN class="toctext">Agarose gel electrophoresis</SPAN></A></LI><LI class="toclevel-2 tocsection-17"><A href="#Gel_and_PCR_Clean-Up"><SPAN class="tocnumber">2.7</SPAN><SPAN class="toctext">Gel and PCR Clean-Up</SPAN></A></LI><LI class="toclevel-1 tocsection-18"><A href="#Protein_methods"><SPAN class="tocnumber">3</SPAN><SPAN class="toctext">Protein methods</SPAN></A></LI><LI class="toclevel-2 tocsection-19"><A href="#Protein_overexpression"><SPAN class="tocnumber">3.1</SPAN><SPAN class="toctext">Protein overexpression</SPAN></A></LI><LI class="toclevel-2 tocsection-20"><A href="#SDS-PAGE"><SPAN class="tocnumber">3.2</SPAN><SPAN class="toctext">SDS-PAGE</SPAN></A></LI><LI class="toclevel-2 tocsection-21"><A href="#Protein_purification"><SPAN class="tocnumber">3.3</SPAN><SPAN class="toctext">Protein purification</SPAN></A></LI><LI class="toclevel-1 tocsection-22"><A href="#Measurements"><SPAN class="tocnumber">4</SPAN><SPAN class="toctext">Measurements</SPAN></A></LI><LI class="toclevel-2 tocsection-23"><A href="#Nanodrop"><SPAN class="tocnumber">4.1</SPAN><SPAN class="toctext">Nanodrop</SPAN></A></LI><LI class="toclevel-1 tocsection-24"><A href="#Enzyme_assays"><SPAN class="tocnumber">5</SPAN><SPAN class="toctext">Enzyme assays</SPAN></A></LI><LI class="toclevel-2 tocsection-25"><A href="#Assay_for_terephthalic_acid_1.2C2-dioxigenase_system_and_dihydrodiol_decarboxylase"><SPAN class="tocnumber">5.1</SPAN><SPAN class="toctext">Assay for  terephthalic acid 1,2-dioxigenase system and dihydrodiol decarboxylase</SPAN></A></LI><LI class="toclevel-2 tocsection-26"><A href="#Assay_for_AroY"><SPAN class="tocnumber">5.2</SPAN><SPAN class="toctext">Assay for AroY</SPAN></A></LI></TD></TR></TBODY></TABLE><H2><SPAN class="mw-headline" id="In_vivo"> In vivo </SPAN></H2><H3><SPAN class="mw-headline" id="Production_of_chemically_competent_cells"> Production of chemically competent cells </SPAN></H3><P>Chemically competent cells are needed for transformation with the heat shock method. We used CaCl<SUB>2</SUB> to produce them.
</P><P><B>Protocol</B></P><UL><LI> Inoculate 10 ml of LB-Media with an E. coli colony and incubate at 37 °C overnight
</LI><LI> Inoculate 200 ml of LB-Media with 2.5 ml of the overnight culture 
</LI><LI> Incubate the culture at 37 °C until an OD<SUB>600</SUB> of 0.5
</LI><LI> Cool the cells on ice for 5 min
</LI><LI> Centrifuge the cells for 10 min at 5000 rpm and 4 °C
</LI><LI> Resuspend the cell-pellet in 32 ml pre-cooled CaCl<SUB>2</SUB> solution (Do not vortex!)
</LI><LI> Add pre-cooled CaCl<SUB>2</SUB> solution to 200 ml
</LI><LI> Let the cells repose on ice for 1 hour
</LI><LI> Centrifuge the cells for 10 min at 5000 rpm and 4 °C
</LI><LI> Resuspend the pellet in 10 ml cryo-solution 
</LI><LI> Decant 200 µl of competent cells in a 1.5 ml tube
</LI><LI> Store the tube in an -80 °C freezer
</LI></UL><P><B>Solutions</B></P><UL><LI> CaCl<SUB>2</SUB></LI><LI>5.55 g CaCl<SUB>2</SUB></LI><LI>Add di H<SUB>2</SUB>O to 1 L
</LI><LI>Sterilize by autoclaving
</LI></UL><UL><LI>Cryo solution
</LI><LI> 0.278 g CaCl<SUB>2</SUB></LI><LI> 10 ml glycerin 
</LI><LI> Add di H<SUB>2</SUB>O to 50 ml 
</LI><LI> Sterilize by autoclave
</LI></UL><H3><SPAN class="mw-headline" id="Heat_shock_transformation_with_E._coli"> Heat shock transformation with E. coli </SPAN></H3><P>The heat shock method is an effective method to transform ligation mixes or plasmids into <I>E. coli</I> bacteria.
</P><P><B>Protocol</B></P><UL><LI> Thaw the chemically competent cell on ice
</LI><LI> Add 50 – 300 ng of the purified plasmid or 1-5 µL of the heat inactivated ligation mix to the cells (10 – 500 ng DNA)
</LI><LI> Invert the tube up to 6 times
</LI><LI> Incubate the cells on ice for 30 min
</LI><LI> Incubate the cells for 1 min at 42 °C
</LI><LI> Let the cells cool down on ice for 5 min
</LI><LI> Add 800 µl of SOC medium 
</LI><LI> Incubate the cells for 45 min at 37 °C
</LI><LI> Centrifuge for 5 min at 0.4 rcf
</LI><LI> Resuspend the pellet in 100 µl LB-Media and plate it on LB Agar with antibioticum
</LI></UL><H3><SPAN class="mw-headline" id="Glycerine_stock"> Glycerine stock </SPAN></H3><P>In order to have a permanent culture of cells glycerine stocks can be made.
</P><P><B>Protocol</B></P><UL><LI> Add 200 µl of sterilized glycerine to 800 µl cell culture and mix well
</LI><LI> Freeze the stock at -20 °C
</LI></UL><H2><SPAN class="mw-headline" id="In_vitro"> In vitro </SPAN></H2><H3><SPAN class="mw-headline" id="PCR"> PCR </SPAN></H3><P>The Polymerase Chain Reaction (PCR) is used to amplify DNA from bacterial colonies or DNA templates.
</P><H4><SPAN class="mw-headline" id="Colony_PCR_.28isolation_of_genomic_DNA.29">Colony PCR (isolation of genomic DNA)</SPAN></H4><UL><LI> Pick one colony with a sterile tip
</LI><LI> One reaction mix contains:
</LI><LI> 10 µL of 5x Phusion HF Buffer
</LI><LI> 1 µL of dNTPs (10 mM each)
</LI><LI> 0,5 µL of Phusion High-Fidelity Polymerase
</LI><LI> Forward primer (10 pmol)
</LI><LI> Reverse primer (10 pmol)
</LI><LI> 1,5 µL of DMSO
</LI><LI> DI water to 50 µL
</LI><LI> Colony template
</LI></UL><UL><LI> PCR program
</LI></UL><DL><DD><TABLE class="wikitable"><TBODY><TR><TH> # </TH><TH> Temperature </TH><TH> Time
</TH></TR><TR><TD> 1 </TD><TD> 98 °C </TD><TD> 00:02:00
</TD></TR><TR><TD> 2 </TD><TD> T<SUB>a</SUB></TD><TD> 00:01:00
</TD></TR><TR><TD> 3 </TD><TD> 72 °C </TD><TD> 00:01:00
</TD></TR><TR><TD> 4 </TD><TD> 98 °C </TD><TD> 00:01:00
</TD></TR><TR><TD> 5 </TD><TD> T<SUB>a</SUB></TD><TD> 00:01:00
</TD></TR><TR><TD> 6 </TD><TD> 72 °C </TD><TD> 00:01:00
</TD></TR><TR><TD> 7 </TD><TD> GO TO 4 </TD><TD> REPEAT 31x
</TD></TR><TR><TD> 8 </TD><TD> 98 °C </TD><TD> 00:01:00
</TD></TR><TR><TD> 9 </TD><TD> T<SUB>a</SUB></TD><TD> 00:01:00
</TD></TR><TR><TD> 10 </TD><TD> 72 °C </TD><TD> 00:06:00
</TD></TR><TR><TD> 11 </TD><TD> 4 °C </TD><TD> HOLD
</TD></TR></TBODY></TABLE></DD></DL><H4><SPAN class="mw-headline" id="Colony_PCR_.28verification_of_transformation.29">Colony PCR (verification of transformation)</SPAN></H4><UL><LI>Pick one colony with a sterile tip and suspend in 10 µL of DI H<SUB>2</SUB>O
</LI><LI>One reaction mix contains:
</LI><LI>2 µL of 10x Thermopol Reaction Buffer
</LI><LI>0,4 µL of dNTPs (10 mM each)
</LI><LI>0,3 µL of Taq DNA Polymerase
</LI><LI>VF2 (10 pmol)
</LI><LI>VR (10 pmol)
</LI><LI>0,6 µL of DMSO
</LI><LI>1 µL of colony suspension
</LI><LI>DI water to 20 µL
</LI></UL><UL><LI>PCR program
</LI></UL><DL><DD><TABLE class="wikitable"><TBODY><TR><TH> # </TH><TH> Temperature </TH><TH> Time
</TH></TR><TR><TD> 1 </TD><TD> 95 °C </TD><TD> 00:01:00
</TD></TR><TR><TD> 2 </TD><TD> 95 °C </TD><TD> 00:00:20
</TD></TR><TR><TD> 3 </TD><TD> 62 °C </TD><TD> 00:00:30
</TD></TR><TR><TD> 4 </TD><TD> 68 °C </TD><TD> 00:02:00
</TD></TR><TR><TD> 5 </TD><TD> GO TO 2 </TD><TD> REPEAT 30x
</TD></TR><TR><TD> 6 </TD><TD> 68 °C </TD><TD> 00:05:00
</TD></TR><TR><TD> 7 </TD><TD> 4 °C </TD><TD> HOLD
</TD></TR></TBODY></TABLE></DD></DL><UL><LI>For verification of longer DNA strands, the elongation time (step 4) was prolonged. It was assumed that 1 kb/min of template was replicated by the Taq DNA Polymerase.
</LI></UL><H4><SPAN class="mw-headline" id="PCR_on_a_DNA_template">PCR on a DNA template</SPAN></H4><UL><LI>One reaction mix contains:
</LI><LI>10 µL of 5x Phusion HF Buffer
</LI><LI>1 µL of dNTPs (10 mM each)
</LI><LI>0,5 µL of Phusion High-Fidelity Polymerase
</LI><LI>Forward primer (10 pmol)
</LI><LI>Reverse primer (10 pmol)
</LI><LI>1,5 µL of DMSO
</LI><LI>1 µL of template
</LI><LI>DI water to 50 µL
</LI></UL><LI>PCR program
</LI><DL><DD><TABLE class="wikitable"><TBODY><TR><TH> # </TH><TH> Temperature </TH><TH> Time
</TH></TR><TR><TD> 1 </TD><TD> 98 °C </TD><TD> 00:02:00
</TD></TR><TR><TD> 2 </TD><TD> T<SUB>a</SUB></TD><TD> 00:01:00
</TD></TR><TR><TD> 3 </TD><TD> 72 °C </TD><TD> 00:01:00
</TD></TR><TR><TD> 4 </TD><TD> 98 °C </TD><TD> 00:01:00
</TD></TR><TR><TD> 5 </TD><TD> T<SUB>a</SUB></TD><TD> 00:01:00
</TD></TR><TR><TD> 6 </TD><TD> 72 °C </TD><TD> 00:01:00
</TD></TR><TR><TD> 7 </TD><TD> GO TO 4 </TD><TD> REPEAT 31x
</TD></TR><TR><TD> 8 </TD><TD> 98 °C </TD><TD> 00:01:00
</TD></TR><TR><TD> 9 </TD><TD> T<SUB>a</SUB></TD><TD> 00:01:00
</TD></TR><TR><TD> 10 </TD><TD> 72 °C </TD><TD> 00:06:00
</TD></TR><TR><TD> 11 </TD><TD> 4 °C </TD><TD> HOLD
</TD></TR></TBODY></TABLE></DD></DL><H3><SPAN class="mw-headline" id="Restriction_digest">Restriction digest</SPAN></H3><P>A restriction digest is used to digest either vectors or inserts via restriction enzymes before ligation.
</P><P><B>Protocol</B></P><UL><LI>One reaction mix contains: 
</LI><LI>DNA template (up to 3µg)
</LI><LI>2 µL NEBuffer 4 (10x)
</LI><LI>0,5 µL of restriction enzyme 1
</LI><LI>0,5 µL of restriction enzyme 2
</LI><LI>0,2 µL of 100x BSA (only when cut with SpeI-HF)
</LI><LI>DI water to 20 µL
</LI></UL><LI>Incubate at 37° C for 1 hour
</LI><LI>Heat inactivate at 80° C for 25 minutes
</LI><H3><SPAN class="mw-headline" id="Dephosphorylation">Dephosphorylation</SPAN></H3><P>Antarctic Phosphatase catalyzes the removal of 5' phosphate groups of DNA and RNA and thus prevents re-ligation of cut vectors. It is used before ligation.
</P><P><B>Protocol</B></P><UL><LI>The reaction mix contains:
</LI><LI>up to 5 µg of digested DNA (no purification needed)
</LI><LI>1 µL of Antarctic Phosphatase
</LI><LI>1/10 of reaction volume of 10x Antarctic Phosphatase Reaction Buffer
</LI><LI>1 µL of Antarctic Phosphatase
</LI></UL><LI>Incubate at 37° C for 30 minutes (3' ends) or 60 minutes (5' ends)
</LI><LI>Heat inactivate at 65° C for 15 minutes
</LI><LI>Continue with ligation
</LI><H3><SPAN class="mw-headline" id="Ligation">Ligation</SPAN></H3><P>Ligation is the process of joining of DNA strands by catalyzing the formation of a phosphodiester bond. It is used to facilitate plasmid vectors.
</P><P><B>Protocol</B></P><UL><LI>One reaction mix contains:
</LI><LI>100 ng of vector DNA and x ng of insert DNA (vector:insert molar ratio is 1:5)
</LI><LI>2 µL of T4 DNA Ligase Buffer (10x)
</LI><LI>T4 DNA Ligase
</LI><LI>DI water to 20 µL
</LI></UL><LI>Put the reaction mix into a thermocycler using the following program:
</LI><DL><DD><TABLE class="wikitable"><TBODY><TR><TH> # </TH><TH> Temperature </TH><TH> Time
</TH></TR><TR><TD> 1 </TD><TD> 37 °C </TD><TD> 00:00:20
</TD></TR><TR><TD> 2 </TD><TD> 33 °C </TD><TD> 00:00:15
</TD></TR><TR><TD> 3 </TD><TD> 28 °C </TD><TD> 00:00:15
</TD></TR><TR><TD> 4 </TD><TD> 23 °C </TD><TD> 00:00:15
</TD></TR><TR><TD> 5 </TD><TD> 18 °C </TD><TD> 00:00:15
</TD></TR><TR><TD> 6 </TD><TD> 13 °C </TD><TD> 00:00:15
</TD></TR><TR><TD> 7 </TD><TD> 8 °C </TD><TD> 00:00:15
</TD></TR><TR><TD> 8 </TD><TD> 4 °C </TD><TD> 00:15:00
</TD></TR><TR><TD> 9 </TD><TD> 16 °C </TD><TD> 00:12:00
</TD></TR><TR><TD> 10 </TD><TD> 23 °C </TD><TD> 00:10:00
</TD></TR><TR><TD> 11 </TD><TD> 30 °C </TD><TD> 00:10:00
</TD></TR><TR><TD> 12 </TD><TD> 37 °C </TD><TD> 00:10:00
</TD></TR><TR><TD> 13 </TD><TD> 65 °C </TD><TD> 00:15:00
</TD></TR><TR><TD> 14 </TD><TD> 4 °C </TD><TD> HOLD
</TD></TR></TBODY></TABLE></DD></DL><H3><SPAN class="mw-headline" id="Plasmid_Preparation">Plasmid Preparation</SPAN></H3><P>Plasmid preparation is a method for extracting and purifying plasmid DNA from <I>E. coli cells</I>. The  Promega PureYield<SUP>TM</SUP> Miniprep and PureYield<SUP>TM</SUP> Miniprep System respectively was used.
</P><H4><SPAN class="mw-headline" id="Plasmid_Miniprep">Plasmid Miniprep</SPAN></H4><P>Minipreps were used for yields up to 15 µg of DNA
</P><P><B>Protocol</B></P><UL><LI>Centrifuge 1.5 mL of bacterial culture at 17,000x g for 1 minute and discard the supernatant. Repeat up to a total volume of bacterial culture of 9 mL
</LI><LI>Follow the steps 2. - 9. of the Promega PureYield<SUP>TM</SUP> Miniprep System <A href="https://www.promega.com/~/media/Files/Resources/ProtCards/PureYield%20Plasmid%20Miniprep%20System%20Quick%20Protocol.pdf" class="external text" rel="nofollow">protocol</A></LI><LI>Place the minicolumn on a 1.5 mL tube, add 30µL of Elution Buffer and let stand at room temperature for 1 minute. Then centrifuge at 17,000x g for 1 minute
</LI><LI>Add 20 µL of Elution Buffer and let stand at room temperature for 1 minute. Then centrifuge at 17,000x g for 1 minute
</LI></UL><H4><SPAN class="mw-headline" id="Plasmid_Midiprep">Plasmid Midiprep</SPAN></H4><P>Midipreps were used for yields up to 200 µg of DNA
</P><P><B>Protocol</B></P><UL><LI>Follow the steps for elution by applying vacuum of the Promega PureYieldTM Midiprep System <A href="http://www.promega.com/~/media/Files/Resources/ProtCards/PureYield%20Plasmid%20Midiprep%20System%20Quick%20Protocol.pdf" class="external text" rel="nofollow">protocol</A></LI></UL><H3><SPAN class="mw-headline" id="Agarose_gel_electrophoresis">Agarose gel electrophoresis</SPAN></H3><P>Agarose gel electrophoresis is used to separate DNA fragments of different lengths.
</P><P><B>Protocol</B></P><UL><LI>Fill 20 mL of 50x TAE buffer in 1L flask and add DI water to 1000 mL in order to have 1x TAE buffer
</LI><LI>Choose agarose concentration dependent on fragments size:
</LI></UL><DL><DD><TABLE class="wikitable"><TBODY><TR><TH> Agarose concentration [% (w/v)] </TH><TH> Fragment length [kbp]
</TH></TR><TR><TD><CENTER>0.8</CENTER></TD><TD><CENTER>0.5-11</CENTER></TD></TR><TR><TD><CENTER>1.2</CENTER></TD><TD><CENTER>0.4-7</CENTER></TD></TR><TR><TD><CENTER>1.5</CENTER></TD><TD><CENTER>0.2-3</CENTER></TD></TR><TR><TD><CENTER>2.5</CENTER></TD><TD><CENTER>0.05-2</CENTER></TD></TR></TBODY></TABLE></DD></DL><UL><LI>Measure the appropiate mass of agarose, add 1x TAE buffer to 100 mL in a flask and mix well
</LI><LI>Heat in a microwave at 450 W for 2-3 minutes until agarose is completely melted
</LI><LI>Let gel cool down to 50-60° C and pour into gel box 
</LI><LI>Add 10 µL of Roti®-Safe GelStain and let gel harden
</LI><LI>Put the gel box into the gel chamber filled with 1x TAE buffer
</LI><LI>Mix your DNA probe with 6x Loading Dye (1 µL Loading Dye per 5 µL DNA probe)
</LI><LI>Remove gel comb and load one pocket with 7 µL of DNA ladder
</LI><LI>Load the pockets with the prepared probes
</LI><LI>Run the gel at 120V for about 40 minutes until the yellowish OrangeG band is close to the bottom
</LI><LI>Place gel in UV box and capture gel image
</LI></UL><P><B>Media</B></P><UL><LI>Loading Dye (6x)
</LI></UL><DL><DD><TABLE class="wikitable"><TBODY><TR><TH> Contents </TH><TH> Concentration
</TH></TR><TR><TD><CENTER>Glycerol</CENTER></TD><TD><CENTER>50% (v/v)</CENTER></TD></TR><TR><TD><CENTER>EDTA (pH 8.0)</CENTER></TD><TD><CENTER>50 mmol/L</CENTER></TD></TR><TR><TD><CENTER>Bromophenol blue</CENTER></TD><TD><CENTER>0.05% (w/v)</CENTER></TD></TR><TR><TD><CENTER>Xylene cyanol</CENTER></TD><TD><CENTER>0.05% (w/v)</CENTER></TD></TR><TR><TD><CENTER>Orange G</CENTER></TD><TD><CENTER>0.05% (w/v)</CENTER></TD></TR></TBODY></TABLE></DD></DL><UL><LI>TAE buffer (50x)
</LI></UL><DL><DD><TABLE class="wikitable"><TBODY><TR><TH> Contents </TH><TH> Concentration
</TH></TR><TR><TD><CENTER>TRIS</CENTER></TD><TD><CENTER>2000 mmol/L</CENTER></TD></TR><TR><TD><CENTER>EDTA</CENTER></TD><TD><CENTER>50 mmol/L</CENTER></TD></TR><TR><TD><CENTER>Sodium acetate</CENTER></TD><TD><CENTER>250 mmol/L</CENTER></TD></TR><TR><TD><CENTER>pH 8.0</CENTER></TD><TD><CENTER>adjust with acetic acid</CENTER></TD></TR></TBODY></TABLE></DD></DL><H3><SPAN class="mw-headline" id="Gel_and_PCR_Clean-Up">Gel and PCR Clean-Up</SPAN></H3><P>The Wizard® SV Gel and PCR Clean-Up System was used for purifying gels and PCR products.
</P><P><B>Protocol</B></P><UL><LI>Follow steps 1. - 7. of the Wizard® SV Gel and PCR Clean-Up System <A href="http://www.promega.com/~/media/Files/Resources/ProtCards/Wizard%20SV%20Gel%20and%20PCR%20Clean-Up%20System%20Quick%20Protocol.pdf" class="external text" rel="nofollow">protocol</A></LI><LI>Add 30 µL of Nuclease-Free Water to the minicolumn and let stand for 1 minute at room temperature. Then centrifuge at 17,000x g for 1 minute
</LI><LI>Add 20 µL of Nuclease-Free Water to the minicolumn and let stand for 1 minute at room temperature. Then centrifuge at 17,000x g for 1 minute
</LI></UL><H2><SPAN class="mw-headline" id="Protein_methods">Protein methods</SPAN></H2><H3><SPAN class="mw-headline" id="Protein_overexpression">Protein overexpression</SPAN></H3><P>Protein overexpression is a subcomponent of gene overexpression and describes the massive production of a protein by an expression system, e. g. <I>E. coli</I>.
</P><P><B>Protocol</B></P><UL><LI> Inoculate 1 L LB-medium-antibiotica with 3 ml bacteria suspension (OD<SUB>600</SUB> = 1.0) and incubate over night
</LI><LI> Measure the OD<SUB>600</SUB>, take 1 mL of the culture and centrifuge the 1 mL sample at 17,000x g for 1 min
</LI><LI> Calculate the amount of diH<SUB>2</SUB>O and Hoechst buffer added to the sample
</LI><LI> Example: For an OD<SUB>600</SUB> = 1 you have to add 100 µl diH<SUB>2</SUB>O and 50 µl Hoechst buffer, for an OD<SUB>600</SUB> = 2 you have to add 200 µl diH<SUB>2</SUB>O and 100 µl Hoechst buffer and so on
</LI></UL><LI> After centrifugation discard the supernatant and add the diH<SUB>2</SUB>O and Hoechst buffer
</LI><LI> Freeze the sample at -20 °C
</LI><LI> Induce with 150 µl of 1 M IPTG Solution   
</LI><LI> Measure hourly the OD<SUB>600</SUB>, take 1 mL of the culture and centrifuge the 1 mL sample at 17,000x g for 1 min
</LI><LI> After centrifugation discard the supernatant and add the diH<SUB>2</SUB>O and Hoechst buffer
</LI><LI> Freeze the sample at -20 °C
</LI><LI> After 4 hours centrifuge the over expression culture at 4,600x g and resuspend the pellet with the wash buffer from <A href="https://2012.igem.org/Team:TU_Darmstadt/Protocols/Metabolism#Protein_purification" class="external text" rel="nofollow">protein purification</A></LI><LI>Storage the suspension at -20 °C or continue with <A href="https://2012.igem.org/Team:TU_Darmstadt/Protocols/Metabolism#Protein_purification" class="external text" rel="nofollow">protein purification</A></LI><LI>Perform an <A href="https://2012.igem.org/Team:TU_Darmstadt/Protocols/Metabolism#SDS-PAGE" class="external text" rel="nofollow">SDS-PAGE</A> with the frozen samples
</LI><H3><SPAN class="mw-headline" id="SDS-PAGE">SDS-PAGE</SPAN></H3><UL><LI>SDS-PAGE was done according to the following <A href="http://openwetware.org/wiki/Keating:Experimental_Protocols:SDS-PAGE" class="external text" rel="nofollow">protocol</A>.
</LI></UL><H3><SPAN class="mw-headline" id="Protein_purification">Protein purification</SPAN></H3><UL><LI>Protein purification was done with a <I>Strep</I>-Tactin® Superflow® high capacity column (Supplier: <A href="http://www.iba-lifesciences.com/Strep-tag_Strep-Tactin_Resins_Superflow_high_capacity_Technology.html" class="external text" rel="nofollow">IBA GmbH</A>) according to the following <A href="https://www.iba-lifesciences.com/isotope/2/2-1209-001-Short-Protocol-Strep-tag-purification.pdf" class="external text" rel="nofollow">protocol</A></LI></UL><H2><SPAN class="mw-headline" id="Measurements">Measurements</SPAN></H2><H3><SPAN class="mw-headline" id="Nanodrop">Nanodrop</SPAN></H3><P>DNA absorbs light at 260 nm and thus the concentration can be measured photometrically using the <A href="http://www.nanodrop.com/Library/nd-1000-v3.7-users-manual-8.5x11.pdf" class="external text" rel="nofollow">NanoDrop 1000 Spectrophotometer</A>.
</P><P><B>Protocol</B></P><UL><LI>Start the programme for the NanoDrop 1000 and click “Nucleic Acids”
</LI><LI>Pipette 1.5 µL of a water sample onto the lower measurement and click “OK”
</LI><LI>Load your blank and click “Blank”
</LI><LI>Load your DNA samples and click “Measure”. A 260/280 ration of 1.8 or above means the sample can be seen as pure
</LI><LI>At the end load your blank and click “Measure” to test the blank. If the measurement deviates more than 5 ng/µL from 0.0 ng/µL, load again the blank, click “Blank” and measure your DNA samples again
</LI></UL><H2><SPAN class="mw-headline" id="Enzyme_assays">Enzyme assays</SPAN></H2><H3><SPAN class="mw-headline" id="Assay_for_terephthalic_acid_1.2C2-dioxigenase_system_and_dihydrodiol_decarboxylase">Assay for  terephthalic acid 1,2-dioxigenase system and dihydrodiol decarboxylase</SPAN></H3><UL><LI> Mix 50 µM of each protein (TphA1, TphA2, TphA3, TphB) and add PBS (pH 7.4) up to 1 ml
</LI><LI> Incubate at 4 °C for 10 min (in order to build the TERDOS complex)
</LI><LI> Add 25 µM terephthalic acid 
</LI><LI> Incubate at 30 °C for 3 h
</LI><LI> Add 20 µl of XylE-protein suspension
</LI><LI> Incubate at 37 °C for 10 min
</LI><LI> Measure the solution at 380 nm in the UV/VIS-spectrometer (Prepare a fresh solution (for blank) without incubation)
</LI></UL><H3><SPAN class="mw-headline" id="Assay_for_AroY">Assay for AroY</SPAN></H3><UL><LI> Inoculate 50 ml LB-medium + amp and 25 mM formic acid with BL21[DE3]pLysS pPR-IBA2-aroY and incubate under <B>anaerobic conditions</B>  at 37 °C over night
</LI><LI> Add 15 µl of 1 M IPTG solution
</LI><LI> Incubate the culture <B>anaerobic</B> at 30 °C for 4 h
</LI><LI> Take 2 ml of the culture and centrifuge at 17,000 g at 4 °C 
</LI><LI> Resuspend the pellet in 1 ml PBS pH 7.4 and cover the solution with 300 µl highly fluid parafin
</LI><LI> Add 50 mM protocatechuic acid and incubate at 37 °C for 12 h 
</LI><LI> Add 50 µl XylE-Protein solution and incubate at 30 °C 10 min
</LI></UL><DIV class="printfooter">
Retrieved from &quot;<A href="http://2012.igem.org/Team:TU_Darmstadt/Protocols/Metabolism">http://2012.igem.org/Team:TU_Darmstadt/Protocols/Metabolism</A>&quot;</DIV></DIV></DIV><DIV id="footer-box" class="noprint"><DIV id="footer"><UL id="f-list"><LI id="t-recentchanges"><A href="/Special:RecentChanges" title="Recent changes">Recent changes</A></LI><LI id="t-whatlinkshere"><A href="/Special:WhatLinksHere/Team:TU_Darmstadt/Protocols/Metabolism" title="List of all wiki pages that link here [j]" accesskey="j">What links here</A></LI><LI id="t-recentchangeslinked"><A href="/Special:RecentChangesLinked/Team:TU_Darmstadt/Protocols/Metabolism" title="Recent changes in pages linked from this page [k]" accesskey="k">Related changes</A></LI><LI id="t-specialpages"><A href="/Special:SpecialPages" title="List of all special pages [q]" accesskey="q">Special pages</A></LI><LI><A href="/Special:Preferences">My preferences</A></LI></UL></DIV><DIV id="footer"><UL id="f-list"><LI id="t-print"><A href="/wiki/index.php?title=Team:TU_Darmstadt/Protocols/Metabolism&amp;printable=yes" title="Printable version of this page [p]" accesskey="p">Printable version</A></LI><LI id="t-permalink"><A href="/wiki/index.php?title=Team:TU_Darmstadt/Protocols/Metabolism&amp;oldid=222467" title="Permanent link to this revision of the page">Permanent link</A></LI><LI id="privacy"><A href="/2012.igem.org:Privacy_policy" title="2012.igem.org:Privacy policy">Privacy policy</A></LI><LI id="disclaimer"><A href="/2012.igem.org:General_disclaimer" title="2012.igem.org:General disclaimer">Disclaimers</A></LI></UL></DIV></DIV></DIV></DIV></BODY></HTML>