"
Page
Discussion
View source
History
teams
Log in
 
Team:TU Darmstadt/Protocols/Metabolism
From 2012.igem.org
iGEM@TUD
Home
Project
Overview
1. Degradation
2. Transport
3. Metabolism
4. Material Science
5. Simulation
Ecology
Philosophy
Team
Members
Supporters
BioBricks
Documentation
Overview
Labjournal
Protocols
Materials
Modeling
Safety
Downloads
Human Practice
Panel Discussion
Symposia
Classes
Sponsors
Overview
Benefits
Protocols Metabolism
The following page gives an overview of the standard protocols used by the team Metabolism in the iGEM PET.erminators project 2012.
Contents
1 In vivo
1.1 Production of chemically competent cells
1.2 Heat shock transformation with E. coli
1.3 Glycerine stock
2 In vitro
2.1 PCR
2.1.1 Colony PCR (isolation of genomic DNA)
2.1.2 Colony PCR (verification of transformation)
2.1.3 PCR on a DNA template
2.2 Restriction digest
2.3 Dephosphorylation
2.4 Ligation
2.5 Plasmid Preparation
2.5.1 Plasmid Miniprep
2.5.2 Plasmid Midiprep
2.6 Agarose gel electrophoresis
2.7 Gel and PCR Clean-Up
3 Protein methods
3.1 Protein overexpression
3.2 SDS-PAGE
3.3 Protein purification
4 Measurements
4.1 Nanodrop
5 Enzyme assays
5.1 Assay for  terephthalic acid 1,2-dioxigenase system and dihydrodiol decarboxylase
5.2 Assay for AroY
In vivo
Production of chemically competent cells
Chemically competent cells are needed for transformation with the heat shock method. We used CaCl2 to produce them.
Protocol
Inoculate 10 ml of LB-Media with an E. coli colony and incubate at 37 °C overnight
Inoculate 200 ml of LB-Media with 2.5 ml of the overnight culture
Incubate the culture at 37 °C until an OD600 of 0.5
Cool the cells on ice for 5 min
Centrifuge the cells for 10 min at 5000 rpm and 4 °C
Resuspend the cell-pellet in 32 ml pre-cooled CaCl2 solution (Do not vortex!)
Add pre-cooled CaCl2 solution to 200 ml
Let the cells repose on ice for 1 hour
Centrifuge the cells for 10 min at 5000 rpm and 4 °C
Resuspend the pellet in 10 ml cryo-solution
Decant 200 µl of competent cells in a 1.5 ml tube
Store the tube in an -80 °C freezer
Solutions
CaCl2
5.55 g CaCl2
Add di H2O to 1 L
Sterilize by autoclaving
Cryo solution
0.278 g CaCl2
10 ml glycerin
Add di H2O to 50 ml
Sterilize by autoclave
Heat shock transformation with E. coli
The heat shock method is an effective method to transform ligation mixes or plasmids into E. coli bacteria.
Protocol
Thaw the chemically competent cell on ice
Add 50 – 300 ng of the purified plasmid or 1-5 µL of the heat inactivated ligation mix to the cells (10 – 500 ng DNA)
Invert the tube up to 6 times
Incubate the cells on ice for 30 min
Incubate the cells for 1 min at 42 °C
Let the cells cool down on ice for 5 min
Add 800 µl of SOC medium
Incubate the cells for 45 min at 37 °C
Centrifuge for 5 min at 0.4 rcf
Resuspend the pellet in 100 µl LB-Media and plate it on LB Agar with antibioticum
Glycerine stock
In order to have a permanent culture of cells glycerine stocks can be made.
Protocol
Add 200 µl of sterilized glycerine to 800 µl cell culture and mix well
Freeze the stock at -20 °C
In vitro
PCR
The Polymerase Chain Reaction (PCR) is used to amplify DNA from bacterial colonies or DNA templates.
Colony PCR (isolation of genomic DNA)
Pick one colony with a sterile tip
One reaction mix contains:
10 µL of 5x Phusion HF Buffer
1 µL of dNTPs (10 mM each)
0,5 µL of Phusion High-Fidelity Polymerase
Forward primer (10 pmol)
Reverse primer (10 pmol)
1,5 µL of DMSO
DI water to 50 µL
Colony template
PCR program
#
Temperature
Time
1
98 °C
00:02:00
2
Ta
00:01:00
3
72 °C
00:01:00
4
98 °C
00:01:00
5
Ta
00:01:00
6
72 °C
00:01:00
7
GO TO 4
REPEAT 31x
8
98 °C
00:01:00
9
Ta
00:01:00
10
72 °C
00:06:00
11
4 °C
HOLD
Colony PCR (verification of transformation)
Pick one colony with a sterile tip and suspend in 10 µL of DI H2O
One reaction mix contains:
2 µL of 10x Thermopol Reaction Buffer
0,4 µL of dNTPs (10 mM each)
0,3 µL of Taq DNA Polymerase
VF2 (10 pmol)
VR (10 pmol)
0,6 µL of DMSO
1 µL of colony suspension
DI water to 20 µL
PCR program
#
Temperature
Time
1
95 °C
00:01:00
2
95 °C
00:00:20
3
62 °C
00:00:30
4
68 °C
00:02:00
5
GO TO 2
REPEAT 30x
6
68 °C
00:05:00
7
4 °C
HOLD
For verification of longer DNA strands, the elongation time (step 4) was prolonged. It was assumed that 1 kb/min of template was replicated by the Taq DNA Polymerase.
PCR on a DNA template
One reaction mix contains:
10 µL of 5x Phusion HF Buffer
1 µL of dNTPs (10 mM each)
0,5 µL of Phusion High-Fidelity Polymerase
Forward primer (10 pmol)
Reverse primer (10 pmol)
1,5 µL of DMSO
1 µL of template
DI water to 50 µL
PCR program
#
Temperature
Time
1
98 °C
00:02:00
2
Ta
00:01:00
3
72 °C
00:01:00
4
98 °C
00:01:00
5
Ta
00:01:00
6
72 °C
00:01:00
7
GO TO 4
REPEAT 31x
8
98 °C
00:01:00
9
Ta
00:01:00
10
72 °C
00:06:00
11
4 °C
HOLD
Restriction digest
A restriction digest is used to digest either vectors or inserts via restriction enzymes before ligation.
Protocol
One reaction mix contains:
DNA template (up to 3µg)
2 µL NEBuffer 4 (10x)
0,5 µL of restriction enzyme 1
0,5 µL of restriction enzyme 2
0,2 µL of 100x BSA (only when cut with SpeI-HF)
DI water to 20 µL
Incubate at 37° C for 1 hour
Heat inactivate at 80° C for 25 minutes
Dephosphorylation
Antarctic Phosphatase catalyzes the removal of 5' phosphate groups of DNA and RNA and thus prevents re-ligation of cut vectors. It is used before ligation.
Protocol
The reaction mix contains:
up to 5 µg of digested DNA (no purification needed)
1 µL of Antarctic Phosphatase
1/10 of reaction volume of 10x Antarctic Phosphatase Reaction Buffer
1 µL of Antarctic Phosphatase
Incubate at 37° C for 30 minutes (3' ends) or 60 minutes (5' ends)
Heat inactivate at 65° C for 15 minutes
Continue with ligation
Ligation
Ligation is the process of joining of DNA strands by catalyzing the formation of a phosphodiester bond. It is used to facilitate plasmid vectors.
Protocol
One reaction mix contains:
100 ng of vector DNA and x ng of insert DNA (vector:insert molar ratio is 1:5)
2 µL of T4 DNA Ligase Buffer (10x)
T4 DNA Ligase
DI water to 20 µL
Put the reaction mix into a thermocycler using the following program:
#
Temperature
Time
1
37 °C
00:00:20
2
33 °C
00:00:15
3
28 °C
00:00:15
4
23 °C
00:00:15
5
18 °C
00:00:15
6
13 °C
00:00:15
7
8 °C
00:00:15
8
4 °C
00:15:00
9
16 °C
00:12:00
10
23 °C
00:10:00
11
30 °C
00:10:00
12
37 °C
00:10:00
13
65 °C
00:15:00
14
4 °C
HOLD
Plasmid Preparation
Plasmid preparation is a method for extracting and purifying plasmid DNA from E. coli cells. The  Promega PureYieldTM Miniprep and PureYieldTM Miniprep System respectively was used.
Plasmid Miniprep
Minipreps were used for yields up to 15 µg of DNA
Protocol
Centrifuge 1.5 mL of bacterial culture at 17,000x g for 1 minute and discard the supernatant. Repeat up to a total volume of bacterial culture of 9 mL
Follow the steps 2. - 9. of the Promega PureYieldTM Miniprep System protocol
Place the minicolumn on a 1.5 mL tube, add 30µL of Elution Buffer and let stand at room temperature for 1 minute. Then centrifuge at 17,000x g for 1 minute
Add 20 µL of Elution Buffer and let stand at room temperature for 1 minute. Then centrifuge at 17,000x g for 1 minute
Plasmid Midiprep
Midipreps were used for yields up to 200 µg of DNA
Protocol
Follow the steps for elution by applying vacuum of the Promega PureYieldTM Midiprep System protocol
Agarose gel electrophoresis
Agarose gel electrophoresis is used to separate DNA fragments of different lengths.
Protocol
Fill 20 mL of 50x TAE buffer in 1L flask and add DI water to 1000 mL in order to have 1x TAE buffer
Choose agarose concentration dependent on fragments size:
Agarose concentration [% (w/v)]
Fragment length [kbp]
0.8
0.5-11
1.2
0.4-7
1.5
0.2-3
2.5
0.05-2
Measure the appropiate mass of agarose, add 1x TAE buffer to 100 mL in a flask and mix well
Heat in a microwave at 450 W for 2-3 minutes until agarose is completely melted
Let gel cool down to 50-60° C and pour into gel box
Add 10 µL of Roti®-Safe GelStain and let gel harden
Put the gel box into the gel chamber filled with 1x TAE buffer
Mix your DNA probe with 6x Loading Dye (1 µL Loading Dye per 5 µL DNA probe)
Remove gel comb and load one pocket with 7 µL of DNA ladder
Load the pockets with the prepared probes
Run the gel at 120V for about 40 minutes until the yellowish OrangeG band is close to the bottom
Place gel in UV box and capture gel image
Media
Loading Dye (6x)
Contents
Concentration
Glycerol
50% (v/v)
EDTA (pH 8.0)
50 mmol/L
Bromophenol blue
0.05% (w/v)
Xylene cyanol
0.05% (w/v)
Orange G
0.05% (w/v)
TAE buffer (50x)
Contents
Concentration
TRIS
2000 mmol/L
EDTA
50 mmol/L
Sodium acetate
250 mmol/L
pH 8.0
adjust with acetic acid
Gel and PCR Clean-Up
The Wizard® SV Gel and PCR Clean-Up System was used for purifying gels and PCR products.
Protocol
Follow steps 1. - 7. of the Wizard® SV Gel and PCR Clean-Up System protocol
Add 30 µL of Nuclease-Free Water to the minicolumn and let stand for 1 minute at room temperature. Then centrifuge at 17,000x g for 1 minute
Add 20 µL of Nuclease-Free Water to the minicolumn and let stand for 1 minute at room temperature. Then centrifuge at 17,000x g for 1 minute
Protein methods
Protein overexpression
Protein overexpression is a subcomponent of gene overexpression and describes the massive production of a protein by an expression system, e. g. E. coli.
Protocol
Inoculate 1 L LB-medium-antibiotica with 3 ml bacteria suspension (OD600 = 1.0) and incubate over night
Measure the OD600, take 1 mL of the culture and centrifuge the 1 mL sample at 17,000x g for 1 min
Calculate the amount of diH2O and Hoechst buffer added to the sample
Example: For an OD600 = 1 you have to add 100 µl diH2O and 50 µl Hoechst buffer, for an OD600 = 2 you have to add 200 µl diH2O and 100 µl Hoechst buffer and so on
After centrifugation discard the supernatant and add the diH2O and Hoechst buffer
Freeze the sample at -20 °C
Induce with 150 µl of 1 M IPTG Solution
Measure hourly the OD600, take 1 mL of the culture and centrifuge the 1 mL sample at 17,000x g for 1 min
After centrifugation discard the supernatant and add the diH2O and Hoechst buffer
Freeze the sample at -20 °C
After 4 hours centrifuge the over expression culture at 4,600x g and resuspend the pellet with the wash buffer from protein purification
Storage the suspension at -20 °C or continue with protein purification
Perform an SDS-PAGE with the frozen samples
SDS-PAGE
SDS-PAGE was done according to the following protocol .
Protein purification
Protein purification was done with a Strep-Tactin® Superflow® high capacity column (Supplier: IBA GmbH ) according to the following protocol
Measurements
Nanodrop
DNA absorbs light at 260 nm and thus the concentration can be measured photometrically using the NanoDrop 1000 Spectrophotometer .
Protocol
Start the programme for the NanoDrop 1000 and click “Nucleic Acids”
Pipette 1.5 µL of a water sample onto the lower measurement and click “OK”
Load your blank and click “Blank”
Load your DNA samples and click “Measure”. A 260/280 ration of 1.8 or above means the sample can be seen as pure
At the end load your blank and click “Measure” to test the blank. If the measurement deviates more than 5 ng/µL from 0.0 ng/µL, load again the blank, click “Blank” and measure your DNA samples again
Enzyme assays
Assay for  terephthalic acid 1,2-dioxigenase system and dihydrodiol decarboxylase
Mix 50 µM of each protein (TphA1, TphA2, TphA3, TphB) and add PBS (pH 7.4) up to 1 ml
Incubate at 4 °C for 10 min (in order to build the TERDOS complex)
Add 25 µM terephthalic acid
Incubate at 30 °C for 3 h
Add 20 µl of XylE-protein suspension
Incubate at 37 °C for 10 min
Measure the solution at 380 nm in the UV/VIS-spectrometer (Prepare a fresh solution (for blank) without incubation)
Assay for AroY
Inoculate 50 ml LB-medium + amp and 25 mM formic acid with BL21[DE3]pLysS pPR-IBA2-aroY and incubate under anaerobic conditions at 37 °C over night
Add 15 µl of 1 M IPTG solution
Incubate the culture anaerobic at 30 °C for 4 h
Take 2 ml of the culture and centrifuge at 17,000 g at 4 °C
Resuspend the pellet in 1 ml PBS pH 7.4 and cover the solution with 300 µl highly fluid parafin
Add 50 mM protocatechuic acid and incubate at 37 °C for 12 h
Add 50 µl XylE-Protein solution and incubate at 30 °C 10 min
Retrieved from " http://2012.igem.org/Team:TU_Darmstadt/Protocols/Metabolism "
Recent changes
What links here
Related changes
Special pages
My preferences
Printable version
Permanent link
Privacy policy
Disclaimers
