<HTML xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en" dir="ltr">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki  ltr ns-0 ns-subject page-Team_TU_Darmstadt_Labjournal_Transport"><DIV id="globalWrapper"><DIV id="top-section"><DIV id="p-logo"><A href="/Main_Page" title="Main Page">&quot;
	    </A></DIV><DIV id="menubar" class="left-menu noprint"><UL><LI class="selected"><A href="/Team:TU_Darmstadt/Labjournal/Transport">Page               </A></LI><LI class="new"><A href="/wiki/index.php?title=Talk:Team:TU_Darmstadt/Labjournal/Transport&amp;action=edit&amp;redlink=1">Discussion               </A></LI><LI><A href="/wiki/index.php?title=Team:TU_Darmstadt/Labjournal/Transport&amp;action=edit">View source               </A></LI><LI><A href="/wiki/index.php?title=Team:TU_Darmstadt/Labjournal/Transport&amp;action=history">History               </A></LI><LI style="color:#808080;cursor:default">teams</LI></UL></DIV><DIV class="right-menu noprint" id="menubar"><UL><LI id="pt-login"><A href="/wiki/index.php?title=Special:UserLogin&amp;returnto=Team:TU_Darmstadt/Labjournal/Transport" title="You are encouraged to log in; however, it is not mandatory [o]" accesskey="o">Log in</A></LI></UL></DIV><DIV id="search-controls" class="noprint"><FORM action="/Special:Search" id="searchform">Â </FORM></DIV></DIV><DIV id="content"><H1 class="firstHeading">Team:TU Darmstadt/Labjournal/Transport</H1><DIV id="bodyContent"><H3 id="siteSub" class="noprint">From 2012.igem.org</H3><DIV id="TUD"><DIV id="top"><UL><LI><A href="http://www.igem.tu-darmstadt.de/igem/projekt/index.en.jsp" title="iGEM">iGEM@TUD</A></LI><LI><A href="/Team:TU_Darmstadt" title="Home">Home</A></LI><LI><A href="/Team:TU_Darmstadt/Project" title="Project">Project</A></LI><LI><A href="/Team:TU_Darmstadt/Project" title="Team:Overview">Overview</A></LI><LI><A href="/Team:TU_Darmstadt/Project/Degradation" title="Degradation">1. Degradation</A></LI><LI><A href="/Team:TU_Darmstadt/Project/Transport" title="Transport">2. Transport</A></LI><LI><A href="/Team:TU_Darmstadt/Project/Metabolism" title="Metabolism">3. Metabolism</A></LI><LI><A href="/Team:TU_Darmstadt/Project/Material_Science" title="Material Science">4. Material Science</A></LI><LI><A href="/Team:TU_Darmstadt/Project/Simulation" title="Simulation">5. Simulation</A></LI><LI><A href="/Team:TU_Darmstadt/Project/Ecology" title="Ecology">Ecology</A></LI><LI><A href="/Team:TU_Darmstadt/Project/Philosophy" title="Philosophy">Philosophy</A></LI></UL><LI><A href="/Team:TU_Darmstadt/Team" title="Team:TU_Darmstadt/Team">Team</A></LI><LI><A href="/Team:TU_Darmstadt/Team" title="Team:TU_Darmstadt/Team">Members</A></LI><LI><A href="/Team:TU_Darmstadt/Supporters" title="Team:TU_Darmstadt/Supporters">Supporters</A></LI><LI><A href="/Team:TU_Darmstadt/Parts" title="BioBricks">BioBricks</A></LI><LI><A href="/Team:TU_Darmstadt/Documentation" title="Documentation">Documentation</A></LI><LI><A href="/Team:TU_Darmstadt/Documentation" title="Documentation">Overview</A></LI><LI><A href="/Team:TU_Darmstadt/Labjournal" title="Labjournal">Labjournal</A></LI><LI><A href="/Team:TU_Darmstadt/Protocols" title="Protocols">Protocols</A></LI><LI><A href="/Team:TU_Darmstadt/Materials" title="Materials">Materials</A></LI><LI><A href="/Team:TU_Darmstadt/Modeling" title="Modeling">Modeling</A></LI><LI><A href="/Team:TU_Darmstadt/Safety" title="Safety">Safety</A></LI><LI><A href="/Team:TU_Darmstadt/Downloads" title="Downloads">Downloads</A></LI><LI><A href="/Team:TU_Darmstadt/Human_Practice" title="Human Practice">Human Practice</A></LI><LI><A href="/Team:TU_Darmstadt/Sponsors" title="Sponsors">Sponsors</A></LI><LI><A href="/Team:TU_Darmstadt/Sponsors" title="Sponsors">Overview</A></LI><LI><A href="/Team:TU_Darmstadt/Contact" title="Contact">Benefits</A></LI></DIV><P><SPAN style="font-size:188%;"><SPAN style="color:#00689D;">Transport</SPAN></SPAN></P><TABLE id="toc" class="toc"><TBODY><TR><TD><DIV id="toctitle"><H2>Contents</H2></DIV><UL><LI class="toclevel-1 tocsection-1"><A href="#Overview"><SPAN class="tocnumber">1</SPAN><SPAN class="toctext">Overview</SPAN></A></LI><LI class="toclevel-2 tocsection-2"><A href="#How_we_constructed_our_Biobricks"><SPAN class="tocnumber">1.1</SPAN><SPAN class="toctext">How we constructed our Biobricks</SPAN></A></LI></UL><LI class="toclevel-1 tocsection-3"><A href="#Week_1_.2821.-25.05.12.29"><SPAN class="tocnumber">2</SPAN><SPAN class="toctext">Week 1 (21.-25.05.12)</SPAN></A></LI><LI class="toclevel-2 tocsection-4"><A href="#Results"><SPAN class="tocnumber">2.1</SPAN><SPAN class="toctext">Results</SPAN></A></LI><LI class="toclevel-1 tocsection-5"><A href="#week_2_.2828.05.-01.06.12.29"><SPAN class="tocnumber">3</SPAN><SPAN class="toctext">week 2 (28.05.-01.06.12)</SPAN></A></LI><LI class="toclevel-2 tocsection-6"><A href="#Results_2"><SPAN class="tocnumber">3.1</SPAN><SPAN class="toctext">Results</SPAN></A></LI><LI class="toclevel-1 tocsection-7"><A href="#week_3_.2804.-08.06.12.29"><SPAN class="tocnumber">4</SPAN><SPAN class="toctext">week 3 (04.-08.06.12)</SPAN></A></LI><LI class="toclevel-2 tocsection-8"><A href="#Results_3"><SPAN class="tocnumber">4.1</SPAN><SPAN class="toctext">Results</SPAN></A></LI><LI class="toclevel-1 tocsection-9"><A href="#week_4_.2811.-15-06.12.29"><SPAN class="tocnumber">5</SPAN><SPAN class="toctext">week 4 (11.-15-06.12)</SPAN></A></LI><LI class="toclevel-2 tocsection-10"><A href="#Results_4"><SPAN class="tocnumber">5.1</SPAN><SPAN class="toctext">Results</SPAN></A></LI><LI class="toclevel-1 tocsection-11"><A href="#week_5_.2818.-22.06.12.29"><SPAN class="tocnumber">6</SPAN><SPAN class="toctext">week 5 (18.-22.06.12)</SPAN></A></LI><LI class="toclevel-2 tocsection-12"><A href="#Results_5"><SPAN class="tocnumber">6.1</SPAN><SPAN class="toctext">Results</SPAN></A></LI><LI class="toclevel-1 tocsection-13"><A href="#week_6_.2825.-29.06.12.29"><SPAN class="tocnumber">7</SPAN><SPAN class="toctext">week 6 (25.-29.06.12)</SPAN></A></LI><LI class="toclevel-2 tocsection-14"><A href="#Results_6"><SPAN class="tocnumber">7.1</SPAN><SPAN class="toctext">Results</SPAN></A></LI><LI class="toclevel-1 tocsection-15"><A href="#week_7_.2802.-06.07.12.29"><SPAN class="tocnumber">8</SPAN><SPAN class="toctext">week 7 (02.-06.07.12)</SPAN></A></LI><LI class="toclevel-2 tocsection-16"><A href="#Results_7"><SPAN class="tocnumber">8.1</SPAN><SPAN class="toctext">Results</SPAN></A></LI><LI class="toclevel-1 tocsection-17"><A href="#week_8_.2809.-13.07.12.29"><SPAN class="tocnumber">9</SPAN><SPAN class="toctext">week 8 (09.-13.07.12)</SPAN></A></LI><LI class="toclevel-2 tocsection-18"><A href="#Results_8"><SPAN class="tocnumber">9.1</SPAN><SPAN class="toctext">Results</SPAN></A></LI><LI class="toclevel-1 tocsection-19"><A href="#week_9_.2816.-20.07.12.29"><SPAN class="tocnumber">10</SPAN><SPAN class="toctext">week 9 (16.-20.07.12)</SPAN></A></LI><LI class="toclevel-2 tocsection-20"><A href="#Results_9"><SPAN class="tocnumber">10.1</SPAN><SPAN class="toctext">Results</SPAN></A></LI><LI class="toclevel-1 tocsection-21"><A href="#week_10_.2823.-27.07.12.29"><SPAN class="tocnumber">11</SPAN><SPAN class="toctext">week 10 (23.-27.07.12)</SPAN></A></LI><LI class="toclevel-2 tocsection-22"><A href="#Results_10"><SPAN class="tocnumber">11.1</SPAN><SPAN class="toctext">Results</SPAN></A></LI><LI class="toclevel-1 tocsection-23"><A href="#week_11_.2830.07.-03.08.12.29"><SPAN class="tocnumber">12</SPAN><SPAN class="toctext">week 11 (30.07.-03.08.12)</SPAN></A></LI><LI class="toclevel-2 tocsection-24"><A href="#Results_11"><SPAN class="tocnumber">12.1</SPAN><SPAN class="toctext">Results</SPAN></A></LI><LI class="toclevel-1 tocsection-25"><A href="#week_12_.2806.-10.08.12.29"><SPAN class="tocnumber">13</SPAN><SPAN class="toctext">week 12 (06.-10.08.12)</SPAN></A></LI><LI class="toclevel-2 tocsection-26"><A href="#Results_12"><SPAN class="tocnumber">13.1</SPAN><SPAN class="toctext">Results</SPAN></A></LI><LI class="toclevel-1 tocsection-27"><A href="#week_13_.2813.-17.08.12.29"><SPAN class="tocnumber">14</SPAN><SPAN class="toctext">week 13 (13.-17.08.12)</SPAN></A></LI><LI class="toclevel-2 tocsection-28"><A href="#Results_13"><SPAN class="tocnumber">14.1</SPAN><SPAN class="toctext">Results</SPAN></A></LI><LI class="toclevel-1 tocsection-29"><A href="#week_14_.2820.-24.08.12.29"><SPAN class="tocnumber">15</SPAN><SPAN class="toctext">week 14 (20.-24.08.12)</SPAN></A></LI><LI class="toclevel-1 tocsection-30"><A href="#Week_15-19_charakterization_and_uptake_tests"><SPAN class="tocnumber">16</SPAN><SPAN class="toctext">Week 15-19 charakterization and uptake tests</SPAN></A></LI><LI class="toclevel-1 tocsection-31"><A href="#Uptake"><SPAN class="tocnumber">17</SPAN><SPAN class="toctext">Uptake</SPAN></A></LI></TD></TR></TBODY></TABLE><H3><SPAN class="mw-headline" id="Overview">Overview</SPAN></H3><P>This lab journal describes an isolation and characterisation of the putative terephtalate (TPA) uptake system from <A href="http://www.dsmz.de/catalogues/details/culture/DSM-14576.html?tx_dsmzresources_pi5%5BreturnPid%5D=304" class="external text" rel="nofollow"><I>Comamonas testosteroni KF-1</I></A> (<I>C. testosteroni</I>) in <I>Escherichia Coli</I> (<I>E.Coli</I>). The putative terephtalate system consist of three different domain, two alpha-helical transmembrane protein  (11-12 and 4-5  alpha-helical transmembrane protein) and one periplasmic terephtalate  binding protein. The strain was purchased from <A href="http://www.dsmz.de/" class="external text" rel="nofollow">DMSZ</A>-German Collection of Microorganism and Cell Cultures (DSMZ no.14576). We designed two putative TPA uptake operons and the intake of TPA was checked by photometry, gas chromatography-mass spectrometry (GC-MS) and energy dispersive X-ray spectroscopy (EDX). 
</P><H5><SPAN class="mw-headline" id="How_we_constructed_our_Biobricks">How we constructed our Biobricks</SPAN></H5><DIV class="thumb tnone"><DIV class="thumbinner" style="width:902px;"><DIV class="thumbcaption">Overview of Biobrick construction</DIV></DIV></DIV><H3><SPAN class="mw-headline" id="Week_1_.2821.-25.05.12.29">Week 1 (21.-25.05.12)</SPAN></H3><UL><LI> First <A href="https://2012.igem.org/Team:TU_Darmstadt/Protocols/Colony_PCR" class="external text" rel="nofollow">colony PCR</A> of <I>Comamonas testosteroni</I> for isolation of the following genes: <A href="http://partsregistry.org/wiki/index.php?title=Part:BBa_K808004" class="external text" rel="nofollow">tctA 505aa</A> (1), <A href="http://partsregistry.org/wiki/index.php?title=Part:BBa_K808005" class="external text" rel="nofollow">tctB 197aa</A> (2), <A href="http://partsregistry.org/wiki/index.php?title=Part:BBa_K808002" class="external text" rel="nofollow">tctA_503aa</A> (3), <A href="http://partsregistry.org/wiki/index.php?title=Part:BBa_K808003" class="external text" rel="nofollow">tctB 162aa</A> (4), <A href="http://partsregistry.org/wiki/index.php?title=Part:BBa_K808001" class="external text" rel="nofollow">tphC 322aa</A> (5). (Primers: <A href="https://2012.igem.org/Team:TU_Darmstadt/Materials/tctA503" class="external text" rel="nofollow">tctA_503 Biobrick</A>, <A href="https://2012.igem.org/Team:TU_Darmstadt/Materials/tctA505" class="external text" rel="nofollow">tctA_505 Biobrick</A>,  <A href="https://2012.igem.org/Team:TU_Darmstadt/Materials/tctB162" class="external text" rel="nofollow">tctB_162 Biobrick</A>, <A href="https://2012.igem.org/Team:TU_Darmstadt/Materials/tctB197" class="external text" rel="nofollow">tctB_197 Biobrick</A>, <A href="https://2012.igem.org/Team:TU_Darmstadt/Materials/tphC" class="external text" rel="nofollow">tphC Biobrick</A>). Analysis of <A href="http://en.wikipedia.org/wiki/Polymerase_chain_reaction" class="external text" rel="nofollow">PCR</A> products by <A href="https://2012.igem.org/Team:TU_Darmstadt/Protocols/Agarose_gel_electrophoresis" class="external text" rel="nofollow">Agarose gel electrophoresis</A></LI></UL><DIV class="thumb tnone"><DIV class="thumbinner" style="width:182px;"><DIV class="thumbcaption">successful isolation of the five genes form <I>C. testosteroni</I>. Marker: 1kb DNA ladder, NEB</DIV></DIV></DIV><UL><LI> PCR product purification using the Promega-Kit
</LI></UL><UL><LI> Nanodrop measurements of the purified PCR products:
</LI></UL><DL><DD>(1)Â : 19.5 ng/Âµl 260/280=1.58
</DD></DL><DL><DD>(2)Â : 61.9 ng/Âµl 260/280=1.75
</DD></DL><DL><DD>(3)Â : 75.1 ng/Âµl 260/280=1.75
</DD></DL><DL><DD>(4)Â : 93.3 ng/Âµl 260/280=1.85
</DD></DL><DL><DD>(5)Â : 91.3 ng/Âµl 260/280=1.58
</DD></DL><UL><LI><A href="https://2012.igem.org/Team:TU_Darmstadt/Protocols/Restriction_digest" class="external text" rel="nofollow">Restriction digest</A> of PCR products by SpeI and EcoRI, heat inactivation after digestion
</LI></UL><UL><LI> Analysis of restriction digest by <A href="https://2012.igem.org/Team:TU_Darmstadt/Protocols/Agarose_gel_electrophoresis" class="external text" rel="nofollow">Agarose gel electrophoresis</A></LI></UL><UL><LI> Purification of the bands gained form gel electrophoresis (Promega-Kit)
</LI></UL><UL><LI> Nanodrop measurements of the purified products:
</LI></UL><DL><DD>(1)Â : 8.3 ng/Âµl 260/280=1.94
</DD></DL><DL><DD>(2)Â : 8.9 ng/Âµl 260/280=1.80
</DD></DL><DL><DD>(3)Â : 13.0 ng/Âµl 260/280=1.57
</DD></DL><DL><DD>(4)Â : 1.5 ng/Âµl 260/280=3.08
</DD></DL><DL><DD>(5)Â : 3.7 ng/Âµl 260/280=2.26
</DD></DL><H4><SPAN class="mw-headline" id="Results">Results</SPAN></H4><UL><LI> We got the purified Genes from <I>C. testosteroni</I> but the yield after purification is to low for futher use
</LI></UL><H3><SPAN class="mw-headline" id="week_2_.2828.05.-01.06.12.29">week 2 (28.05.-01.06.12)</SPAN></H3><UL><LI> Purified products from 1. week were <A href="https://2012.igem.org/Team:TU_Darmstadt/Protocols/" class="external text" rel="nofollow">ligated</A> into pSB1A2 and subsequently <A href="https://2012.igem.org/Team:TU_Darmstadt/Protocols/Heat_Shock_Transformation" class="external text" rel="nofollow">transformed</A> into DH5a (in spite of low concentration)
</LI></UL><UL><LI> Overnight incubation on LB agar plates; no growth detectable
</LI></UL><UL><LI> Second approach to isolate the five genes from <I>Comamonas t.</I> using <A href="https://2012.igem.org/Team:TU_Darmstadt/Protocols/Colony_PCR" class="external text" rel="nofollow">colony-PCR</A> (Primers: <A href="https://2012.igem.org/Team:TU_Darmstadt/Materials/tctA503" class="external text" rel="nofollow">tctA_503 Biobrick</A>, <A href="https://2012.igem.org/Team:TU_Darmstadt/Materials/tctA505" class="external text" rel="nofollow">tctA_505 Biobrick</A>,  <A href="https://2012.igem.org/Team:TU_Darmstadt/Materials/tctB162" class="external text" rel="nofollow">tctB_162 Biobrick</A>, <A href="https://2012.igem.org/Team:TU_Darmstadt/Materials/tctB197" class="external text" rel="nofollow">tctB_197 Biobrick</A>, <A href="https://2012.igem.org/Team:TU_Darmstadt/Materials/tphC" class="external text" rel="nofollow">tphC Biobrick</A>)
</LI></UL><UL><LI> Analysis of PCR products by <A href="https://2012.igem.org/Team:TU_Darmstadt/Protocols/Agarose_gel_electrophoresis" class="external text" rel="nofollow">Agarose gel electrophoresis</A>; without results; modification of the PCR protocols: adding DMSO
</LI></UL><UL><LI> Third approach to isolate the five genes from <I>Comamonas t.</I> by <A href="https://2012.igem.org/Team:TU_Darmstadt/Protocols/Colony_PCR" class="external text" rel="nofollow">colony-PCR</A> (Primers: <A href="https://2012.igem.org/Team:TU_Darmstadt/Materials/tctA503" class="external text" rel="nofollow">tctA_503 Biobrick</A>, <A href="https://2012.igem.org/Team:TU_Darmstadt/Materials/tctA505" class="external text" rel="nofollow">tctA_505 Biobrick</A>,  <A href="https://2012.igem.org/Team:TU_Darmstadt/Materials/tctB162" class="external text" rel="nofollow">tctB_162 Biobrick</A>, <A href="https://2012.igem.org/Team:TU_Darmstadt/Materials/tctB197" class="external text" rel="nofollow">tctB_197 Biobrick</A>, <A href="https://2012.igem.org/Team:TU_Darmstadt/Materials/tphC" class="external text" rel="nofollow">tphC Biobrick</A>)
</LI></UL><UL><LI> PCR product purification using the Promega-Kit: (<A href="http://partsregistry.org/wiki/index.php?title=Part:BBa_K808005" class="external text" rel="nofollow">tctB 197aa</A> (2), <A href="http://partsregistry.org/wiki/index.php?title=Part:BBa_K808002" class="external text" rel="nofollow">tctA_503aa</A> (3), <A href="http://partsregistry.org/wiki/index.php?title=Part:BBa_K808003" class="external text" rel="nofollow">tctB 162aa</A> (4), <A href="http://partsregistry.org/wiki/index.php?title=Part:BBa_K808001" class="external text" rel="nofollow">tphC 322aa</A> (5))
</LI></UL><DIV class="thumb tnone"><DIV class="thumbinner" style="width:172px;"><DIV class="thumbcaption">(1): no PCR-product. Marker: 1kb DNA ladder, NEB</DIV></DIV></DIV><UL><LI> Nanodrop measurements of the purified PCR products:
</LI></UL><DL><DD>(2)Â : 56.1 ng/Âµl 260/280=1.87
</DD></DL><DL><DD>(3)Â : 66.8 ng/Âµl 260/280=1.92
</DD></DL><DL><DD>(4)Â : 72.8 ng/Âµl 260/280=1.87
</DD></DL><DL><DD>(5)Â : 68.8 ng/Âµl 260/280=1.82
</DD></DL><H4><SPAN class="mw-headline" id="Results_2">Results</SPAN></H4><UL><LI> Finally we got the genes 2, 3, 4 and 5 in acceptable concentrations
</LI></UL><H3><SPAN class="mw-headline" id="week_3_.2804.-08.06.12.29">week 3 (04.-08.06.12)</SPAN></H3><UL><LI><A href="https://2012.igem.org/Team:TU_Darmstadt/Protocols/Restriction_digest" class="external text" rel="nofollow">Restriction digest</A> of the purified PCR products and of <A href="http://partsregistry.org/wiki/index.php?title=Part:BBa_K808004" class="external text" rel="nofollow">tctA 505aa</A> (leftover of the 1. week) by SpeI and EcoRI
</LI></UL><UL><LI><A href="https://2012.igem.org/Team:TU_Darmstadt/Protocols/Restriction_digest" class="external text" rel="nofollow">Restriction digest</A> of pSB1A2 using SpeI and EcoRI
</LI></UL><UL><LI> Dephosphorylation of the restriction reactions by using antarctic Phosphatase
</LI></UL><UL><LI><A href="https://2012.igem.org/Team:TU_Darmstadt/Protocols/" class="external text" rel="nofollow">Ligation</A> of the five genes into pSB1A2 and subsequently <A href="https://2012.igem.org/Team:TU_Darmstadt/Protocols/Heat_Shock_Transformation" class="external text" rel="nofollow">transformation</A> into DH5a (incubation at 37Â°C)
</LI></UL><UL><LI><A href="https://2012.igem.org/Team:TU_Darmstadt/Protocols/Colony_PCR" class="external text" rel="nofollow">colony-PCR</A> of <A href="http://partsregistry.org/wiki/index.php?title=Part:BBa_K808004" class="external text" rel="nofollow">tctA 505aa</A> (1), 2 approaches (Primers:<A href="https://2012.igem.org/Team:TU_Darmstadt/Materials/tctA505" class="external text" rel="nofollow">tctA_505 Biobrick</A>)
</LI></UL><UL><LI> Analysis of PCR products by <A href="https://2012.igem.org/Team:TU_Darmstadt/Protocols/Agarose_gel_electrophoresis" class="external text" rel="nofollow">Agarose gel electrophoresis</A>. Only one of them showed the expected band. The band was excluded and purified via the Promega-Kit.
</LI></UL><DL><DD><DIV class="thumb tnone"><DIV class="thumbinner" style="width:152px;"><DIV class="thumbcaption">only (1.1) showed the expected band. Marker: 1kb DNA ladder, NEB</DIV></DIV></DIV></DD></DL><UL><LI> Nanodrop measurement of the purified product:
</LI></UL><DL><DD>(1.1)Â : 8.9 ng/Âµl 260/280= 2,22
</DD></DL><DL><DD>Because of low concentration and contamination the PCR was repeated and the product was purified again by the Promega-Kit.
</DD></DL><UL><LI> Performing a <A href="https://2012.igem.org/Team:TU_Darmstadt/Protocols/Colony_PCR" class="external text" rel="nofollow">colony-PCR</A>-screen of the transformed cells using a homemade Taq-Polymerase. Verification of 4 colonies per plate. (Primers: <A href="https://2012.igem.org/Team:TU_Darmstadt/Materials/tctA503" class="external text" rel="nofollow">tctA_503 Biobrick</A>, <A href="https://2012.igem.org/Team:TU_Darmstadt/Materials/tctA505" class="external text" rel="nofollow">tctA_505 Biobrick</A>,  <A href="https://2012.igem.org/Team:TU_Darmstadt/Materials/tctB162" class="external text" rel="nofollow">tctB_162 Biobrick</A>, <A href="https://2012.igem.org/Team:TU_Darmstadt/Materials/tctB197" class="external text" rel="nofollow">tctB_197 Biobrick</A>, <A href="https://2012.igem.org/Team:TU_Darmstadt/Materials/tphC" class="external text" rel="nofollow">tphC Biobrick</A>)
</LI></UL><DL><DD>PCR did not work
</DD></DL><UL><LI> Testing the homemade Taq-Polymerase for function. The Phu-Polymerase was used for comparison. Only the approach using the Phu-Polymerase showed the expected bands in <A href="https://2012.igem.org/Team:TU_Darmstadt/Protocols/Agarose_gel_electrophoresis" class="external text" rel="nofollow">Agarose gel electrophoresis</A>.
</LI></UL><DL><DD><DIV class="thumb tnone"><DIV class="thumbinner" style="width:148px;"><DIV class="thumbcaption">only the Phu-approach showed the expected bands. Marker: 1kb DNA ladder, NEB</DIV></DIV></DIV></DD></DL><UL><LI> Screening was repeated with Phu-Polymerase; no bands visible
</LI></UL><H4><SPAN class="mw-headline" id="Results_3">Results</SPAN></H4><UL><LI> We ligated the purified and restricted genes into pSB1A2 and transformed it into DH5a but the colony-PCR didn't work.
</LI><LI> We tried to isolate gene 1 again but after purification the concentration was too low
</LI></UL><H3><SPAN class="mw-headline" id="week_4_.2811.-15-06.12.29">week 4 (11.-15-06.12)</SPAN></H3><UL><LI> Inoculation of LB-media with DH5a_pSB1A2 and DH5a_pSB1C3; overnight incubation at 37Â°C
</LI></UL><UL><LI> Miniprep of DH5a_pSB1A2 and DH5a_pSB1C3 using the Promega-Kit
</LI></UL><UL><LI> Nanodrop measurements of the preparation:
</LI></UL><DL><DD>pSB1A2Â : 136.1 ng/Âµl 260/280= 1.93
</DD></DL><DL><DD>pSB1C3Â : 265.1 ng/Âµl 260/280= 1.89
</DD></DL><UL><LI><A href="https://2012.igem.org/Team:TU_Darmstadt/Protocols/Restriction_digest" class="external text" rel="nofollow">Restriction digest</A> of pSB1A2 and pSB1C3 with EcoRI and SpeI. Afterwards dephosphorylation by using Antarctic phosphatase.
</LI></UL><UL><LI> Analysis of the restriction products through <A href="https://2012.igem.org/Team:TU_Darmstadt/Protocols/Agarose_gel_electrophoresis" class="external text" rel="nofollow">Agarose gel electrophoresis</A>; for comparison the uncropped vectors were also analyzed.
</LI></UL><DL><DD><DIV class="thumb tnone"><DIV class="thumbinner" style="width:172px;"><DIV class="thumbcaption">Analysis of the restricted an unrestricted plasmids pSB1A2 and pSB1C3. Marker: 1kb DNA ladder, NEB</DIV></DIV></DIV></DD></DL><DL><DD>pSB1C3 shows several bands (Insert, vector without insert, linearized vector with insert); concentration of pSB1A2 is very low, only one band visible
</DD></DL><UL><LI> Approach to isolate the following genes from <I>Comamonas t.</I> by <A href="https://2012.igem.org/Team:TU_Darmstadt/Protocols/Colony_PCR" class="external text" rel="nofollow">colony-PCR</A>: <A href="http://partsregistry.org/wiki/index.php?title=Part:BBa_K808004" class="external text" rel="nofollow">tctA 505aa</A> (1), <A href="http://partsregistry.org/wiki/index.php?title=Part:BBa_K808005" class="external text" rel="nofollow">tctB 197aa</A> (2), <A href="http://partsregistry.org/wiki/index.php?title=Part:BBa_K808002" class="external text" rel="nofollow">tctA_503aa</A> (3), <A href="http://partsregistry.org/wiki/index.php?title=Part:BBa_K808003" class="external text" rel="nofollow">tctB 162aa</A> (4) and <A href="http://partsregistry.org/wiki/index.php?title=Part:BBa_K808001" class="external text" rel="nofollow">tphC 322aa</A> (5) by <A href="http://en.wikipedia.org/wiki/Polymerase_chain_reaction" class="external text" rel="nofollow">PCR</A> (Primers: <A href="https://2012.igem.org/Team:TU_Darmstadt/Materials/tctA503" class="external text" rel="nofollow">tctA_503 Biobrick</A>, <A href="https://2012.igem.org/Team:TU_Darmstadt/Materials/tctA505" class="external text" rel="nofollow">tctA_505 Biobrick</A>,  <A href="https://2012.igem.org/Team:TU_Darmstadt/Materials/tctB162" class="external text" rel="nofollow">tctB_162 Biobrick</A>, <A href="https://2012.igem.org/Team:TU_Darmstadt/Materials/tctB197" class="external text" rel="nofollow">tctB_197 Biobrick</A>, <A href="https://2012.igem.org/Team:TU_Darmstadt/Materials/tphC" class="external text" rel="nofollow">tphC Biobrick</A>)
</LI></UL><UL><LI> Analysis of PCR products by <A href="https://2012.igem.org/Team:TU_Darmstadt/Protocols/Agarose_gel_electrophoresis" class="external text" rel="nofollow">Agarose gel electrophoresis</A></LI></UL><DL><DD><DIV class="thumb tnone"><DIV class="thumbinner" style="width:168px;"><DIV class="thumbcaption">Isolation form the Genes 2, 3 and 4. The Isolation of 1 and 5 didn't work (not shown). Marker: 1kb DNA ladder, NEB</DIV></DIV></DIV></DD></DL><UL><LI> PCR product purification by using the Promega-Kit
</LI></UL><UL><LI> Nanodrop measurements of the purified PCR products:
</LI></UL><DL><DD>(2)Â : 18.3 ng/Âµl 260/280=2.03
</DD></DL><DL><DD>(3)Â : 21.2 ng/Âµl 260/280=1.95
</DD></DL><DL><DD>(4)Â : 23.9 ng/Âµl 260/280=2.05
</DD></DL><UL><LI><A href="https://2012.igem.org/Team:TU_Darmstadt/Protocols/Restriction_digest" class="external text" rel="nofollow">Restriction digest</A> of purified PCR products (2), (3) and (4) using SpeI and EcoRI
</LI></UL><UL><LI><A href="https://2012.igem.org/Team:TU_Darmstadt/Protocols/" class="external text" rel="nofollow">Ligation</A> of the digested genes ((2), (3), (4)) into pSB1A2 and pSB1C3
</LI></UL><UL><LI><A href="https://2012.igem.org/Team:TU_Darmstadt/Protocols/Heat_Shock_Transformation" class="external text" rel="nofollow">Transformation</A> of DH5a with pSB1A2_197, pSB1A2_503, pSB1A2_162, pSB1C3_197, pSB1C3_503, pSB1C3_162 (6 different transformation approaches). Overnight incubation at 37Â°C
</LI></UL><UL><LI> Screening of the transformants by <A href="https://2012.igem.org/Team:TU_Darmstadt/Protocols/Colony_PCR" class="external text" rel="nofollow">colony-PCR</A> (Primers: <A href="https://2012.igem.org/Team:TU_Darmstadt/Materials/tctA503" class="external text" rel="nofollow">tctA_503 Biobrick</A>, <A href="https://2012.igem.org/Team:TU_Darmstadt/Materials/tctA505" class="external text" rel="nofollow">tctA_505 Biobrick</A>,  <A href="https://2012.igem.org/Team:TU_Darmstadt/Materials/tctB162" class="external text" rel="nofollow">tctB_162 Biobrick</A>, <A href="https://2012.igem.org/Team:TU_Darmstadt/Materials/tctB197" class="external text" rel="nofollow">tctB_197 Biobrick</A>, <A href="https://2012.igem.org/Team:TU_Darmstadt/Materials/tphC" class="external text" rel="nofollow">tphC Biobrick</A>). Verification of 6 colonies for every transformation. Afterwards the PCR products were analyzed by <A href="https://2012.igem.org/Team:TU_Darmstadt/Protocols/Agarose_gel_electrophoresis" class="external text" rel="nofollow">Agarose gel electrophoresis</A></LI></UL><DL><DD><DIV class="thumb tnone"><DIV class="thumbinner" style="width:261px;"><DIV class="thumbcaption">colony-PCR-screen part I. Only the positive colonies are marked. Marker: Lambda DNA/Eco471 Maker</DIV></DIV></DIV></DD></DL><DL><DD><DIV class="thumb tnone"><DIV class="thumbinner" style="width:192px;"><DIV class="thumbcaption">colony-PCR-screen part II. Only the positive colonies are marked. Marker: Lambda DNA/Eco471 Maker</DIV></DIV></DIV></DD></DL><UL><LI><A href="https://2012.igem.org/Team:TU_Darmstadt/Protocols/Colony_PCR" class="external text" rel="nofollow">Colony-PCR</A> of <I>Comamonas t.</I> to isolate <A href="http://partsregistry.org/wiki/index.php?title=Part:BBa_K808004" class="external text" rel="nofollow">tctA 505aa</A> (1) und <A href="http://partsregistry.org/wiki/index.php?title=Part:BBa_K808001" class="external text" rel="nofollow">tphC 322aa</A> (5) (Primers: <A href="https://2012.igem.org/Team:TU_Darmstadt/Materials/tctA505" class="external text" rel="nofollow">tctA_505 Biobrick</A>, <A href="https://2012.igem.org/Team:TU_Darmstadt/Materials/tphC" class="external text" rel="nofollow">tphC Biobrick</A>)
</LI></UL><UL><LI> Analysis of PCR products (Colony-PCR) and the ligation approach by <A href="https://2012.igem.org/Team:TU_Darmstadt/Protocols/Agarose_gel_electrophoresis" class="external text" rel="nofollow">Agarose gel electrophoresis</A></LI></UL><DIV class="thumb tnone"><DIV class="thumbinner" style="width:252px;"><DIV class="thumbcaption">Colony-PCR products of gene 1 and 5. <A href="http://partsregistry.org/wiki/index.php?title=Part:BBa_K808004" class="external text" rel="nofollow">tctA 505aa</A> shows only the band of the undesired smaller product. Product of the ligation of 2,3 and 4 into pSB1A2 and pSC1C3</DIV></DIV></DIV><UL><LI><A href="https://2012.igem.org/Team:TU_Darmstadt/Protocols/Restriction_digest" class="external text" rel="nofollow">Restriction digest</A> of <A href="http://partsregistry.org/wiki/index.php?title=Part:BBa_K808001" class="external text" rel="nofollow">tphC 322aa</A> (5) and subsequent <A href="https://2012.igem.org/Team:TU_Darmstadt/Protocols/" class="external text" rel="nofollow">Ligation</A> into pSB1A2 and pSB1C3
</LI></UL><UL><LI><A href="https://2012.igem.org/Team:TU_Darmstadt/Protocols/Heat_Shock_Transformation" class="external text" rel="nofollow">Transformation</A> of DH5a with pSB1A2_322 und pSB1C3_322. <A href="http://en.wikipedia.org/wiki/Polymerase_chain_reaction" class="external text" rel="nofollow">PCR</A> (Primers: <A href="https://2012.igem.org/Team:TU_Darmstadt/Materials/tphC" class="external text" rel="nofollow">tphC Biobrick</A>) used to check for successful transformation
</LI></UL><UL><LI> Isolation of <I>C. testosteroni</I> genome, because it seems impossible to amplificate <A href="http://partsregistry.org/wiki/index.php?title=Part:BBa_K808004" class="external text" rel="nofollow">tctA 505aa</A> (1) from a fluid <I>Comamonas t.</I> culture.
</LI></UL><H4><SPAN class="mw-headline" id="Results_4">Results</SPAN></H4><UL><LI> We started an other approach to isolate the five genes. All but gene 1 worked, so we isolated the <I>C. testosteroni</I> genome
</LI><LI> We purified, restricted and ligated the genes 2-5 into pSB1A2 and pSB1C3 and transformed them into DH5a
</LI><LI> We checked the genes 2, 3 and 4 by colony-PCR. All genes had positive transformants.
</LI></UL><H3><SPAN class="mw-headline" id="week_5_.2818.-22.06.12.29">week 5 (18.-22.06.12)</SPAN></H3><UL><LI> Introduction of a nomenclature: A/C gene/colony: pSB1A2 = A; pSB1C3 = C; gene: tctA 505aa = 1, tctB 197aa = 2, tctA 503aa = 3, tctB 162aa = 4, tphC 322aa = 5; colony = 1...X
</LI></UL><UL><LI> Producing fluid cultures of the positive transformants 
</LI></UL><UL><LI> Miniprep of the fluid culures by using the Frementas-Kit. Afterwards Nanodrop measurements of the preparations
</LI></UL><DL><DD><DIV class="thumb tnone"><DIV class="thumbinner" style="width:352px;"><DIV class="thumbcaption">Miniprep of the fluid cultures. Marker: Lambda DNA/Eco471 Maker</DIV></DIV></DIV></DD></DL><DL><DD>A2.3Â : 125.9 ng/Âµl 260/280= 1.62
</DD></DL><DL><DD>A3.1Â : 103.3 ng/Âµl 260/280= 1.72
</DD></DL><DL><DD>A4.6Â : 147.2 ng/Âµl 260/280= 1.70
</DD></DL><DL><DD>A4.5Â : 108.7 ng/Âµl 260/280= 1.86
</DD></DL><DL><DD>A5.12Â : 88.0 ng/Âµl 260/280= 1.87
</DD></DL><DL><DD>A5.8Â : 85.01 ng/Âµl 260/280= 1.78
</DD></DL><DL><DD>A3.2Â : 87.3 ng/Âµl 260/280= 1.81
</DD></DL><DL><DD>C4.6Â : 70.5 ng/Âµl 260/280= 1.81
</DD></DL><DL><DD>C4.1Â : 41.2 ng/Âµl 260/280= 1.85
</DD></DL><DL><DD>C4.4Â : 104.2 ng/Âµl 260/280= 1.85
</DD></DL><DL><DD>C2.1Â : 132.2 ng/Âµl 260/280= 1.85
</DD></DL><DL><DD>C2.6Â : 188.1 ng/Âµl 260/280= 1.75
</DD></DL><UL><LI><A href="http://en.wikipedia.org/wiki/Polymerase_chain_reaction" class="external text" rel="nofollow">PCR</A> using a rest of purified <A href="http://partsregistry.org/wiki/index.php?title=Part:BBa_K808004" class="external text" rel="nofollow">tctA 505aa</A> (1) (Primers: <A href="https://2012.igem.org/Team:TU_Darmstadt/Materials/tctA505" class="external text" rel="nofollow">tctA_505 Biobrick</A>), subsequent <A href="https://2012.igem.org/Team:TU_Darmstadt/Protocols/Agarose_gel_electrophoresis" class="external text" rel="nofollow">Agarose gel electrophoresis</A></LI></UL><UL><LI> Another <A href="https://2012.igem.org/Team:TU_Darmstadt/Protocols/Colony_PCR" class="external text" rel="nofollow">colony-PCR</A> using the transformants (2)-(5) as template (Primers: <A href="https://2012.igem.org/Team:TU_Darmstadt/Materials/tctA503" class="external text" rel="nofollow">tctA_503 Biobrick</A>,  <A href="https://2012.igem.org/Team:TU_Darmstadt/Materials/tctB162" class="external text" rel="nofollow">tctB_162 Biobrick</A>, <A href="https://2012.igem.org/Team:TU_Darmstadt/Materials/tctB197" class="external text" rel="nofollow">tctB_197 Biobrick</A>, <A href="https://2012.igem.org/Team:TU_Darmstadt/Materials/tphC" class="external text" rel="nofollow">tphC Biobrick</A>); analyzing the PCR product by <A href="https://2012.igem.org/Team:TU_Darmstadt/Protocols/Agarose_gel_electrophoresis" class="external text" rel="nofollow">Agarose gel electrophoresis</A></LI></UL><DL><DD><DIV class="thumb tnone"><DIV class="thumbinner" style="width:387px;"><DIV class="thumbcaption">Colony-PCR screen. Only the positive colonies are marked.  3 positive transformants were picked for inoculation of fluid media (A34, A43; C513). Marker: Lambda DNA/Eco471 Maker</DIV></DIV></DIV></DD></DL><UL><LI> Miniprep (Fermentas Kit) of the fluid culures and Nanodrop measurements:
</LI></UL><DL><DD>A2.4Â : 202,1 ng/Âµl 260/280= 1,63
</DD></DL><DL><DD>A2.5Â : 126,5 ng/Âµl 260/280= 1,90
</DD></DL><DL><DD>A2.6Â : 107,5 ng/Âµl 260/280= 1,81
</DD></DL><DL><DD>A4.2Â : 133,8 ng/Âµl 260/280= 1,59
</DD></DL><DL><DD>C2.3Â : 187,8 ng/Âµl 260/280= 1,87
</DD></DL><DL><DD>C2.4Â : 270,7 ng/Âµl 260/280= 1,80
</DD></DL><DL><DD>C4.1Â : 103,8 ng/Âµl 260/280= 1,80
</DD></DL><UL><LI> The Miniprep products were <A href="https://2012.igem.org/Team:TU_Darmstadt/Protocols/Restriction_digest" class="external text" rel="nofollow">digested</A> by restriction enzymes, each was cut once with EcoRI and twice with EcoRI/SpeI. Analisys of the digested products by <A href="https://2012.igem.org/Team:TU_Darmstadt/Protocols/Agarose_gel_electrophoresis" class="external text" rel="nofollow">Agarose gel electrophoresis</A></LI></UL><DL><DD><DIV class="thumb tnone"><DIV class="thumbinner" style="width:484px;"><DIV class="thumbcaption">Restriction digest of miniprep. Marker: Lambda DNA/Eco471 Maker</DIV></DIV></DIV></DD></DL><H4><SPAN class="mw-headline" id="Results_5">Results</SPAN></H4><UL><LI> We ligated our genes 2, 3 and 4 successfully either in pSB1A2 or pSB1C3
</LI><LI> We couldn't get a positive colony-PCR of a transformant containing gene 5
</LI><LI> Isolation of gene 1 still didn't work
</LI></UL><H3><SPAN class="mw-headline" id="week_6_.2825.-29.06.12.29">week 6 (25.-29.06.12)</SPAN></H3><UL><LI> Further <A href="https://2012.igem.org/Team:TU_Darmstadt/Protocols/Colony_PCR" class="external text" rel="nofollow">colony-PCR</A> of <I>C.testosteroni.</I> (3 approaches) and <A href="http://en.wikipedia.org/wiki/Polymerase_chain_reaction" class="external text" rel="nofollow">PCR</A> with genomic <I>C. testosteroni.</I> DNA as template to obtain <A href="http://partsregistry.org/wiki/index.php?title=Part:BBa_K808004" class="external text" rel="nofollow">tctA 505aa</A> (1), 2 approaches each (Primers: <A href="https://2012.igem.org/Team:TU_Darmstadt/Materials/tctA505" class="external text" rel="nofollow">tctA_505 Biobrick</A>). Subsequent analisys of the PCR products by <A href="https://2012.igem.org/Team:TU_Darmstadt/Protocols/Agarose_gel_electrophoresis" class="external text" rel="nofollow">Agarose gel electrophoresis</A>. All PCRs of <A href="http://partsregistry.org/wiki/index.php?title=Part:BBa_K808004" class="external text" rel="nofollow">tctA 505aa</A> show only the band of the undesired smaller product.
</LI></UL><UL><LI><A href="https://2012.igem.org/Team:TU_Darmstadt/Protocols/" class="external text" rel="nofollow">Ligation</A> off the digested genes into pSB1A2 and <A href="https://2012.igem.org/Team:TU_Darmstadt/Protocols/Heat_Shock_Transformation" class="external text" rel="nofollow">transformation</A> into DH5a. Overnight incubation at 37Â°C
</LI></UL><UL><LI><A href="https://2012.igem.org/Team:TU_Darmstadt/Protocols/Colony_PCR" class="external text" rel="nofollow">Colony-PCR</A> of <I>C. testosteroni.</I> and <A href="http://en.wikipedia.org/wiki/Polymerase_chain_reaction" class="external text" rel="nofollow">PCR</A> with genomic DNA of <I>C. testosteroni.</I> as template to obtain <A href="http://partsregistry.org/wiki/index.php?title=Part:BBa_K808004" class="external text" rel="nofollow">tctA 505aa</A> (1). <A href="http://partsregistry.org/wiki/index.php?title=Part:BBa_K808005" class="external text" rel="nofollow">tctB 197aa</A> (2), <A href="http://partsregistry.org/wiki/index.php?title=Part:BBa_K808002" class="external text" rel="nofollow">tctA_503aa</A> (3), <A href="http://partsregistry.org/wiki/index.php?title=Part:BBa_K808003" class="external text" rel="nofollow">tctB 162aa</A> (4), <A href="http://partsregistry.org/wiki/index.php?title=Part:BBa_K808001" class="external text" rel="nofollow">tphC 322aa</A> (5) used as control (Primers:  <A href="https://2012.igem.org/Team:TU_Darmstadt/Materials/tctA503" class="external text" rel="nofollow">tctA_503 Biobrick</A>, <A href="https://2012.igem.org/Team:TU_Darmstadt/Materials/tctA505" class="external text" rel="nofollow">tctA_505 Biobrick</A>,  <A href="https://2012.igem.org/Team:TU_Darmstadt/Materials/tctB162" class="external text" rel="nofollow">tctB_162 Biobrick</A>, <A href="https://2012.igem.org/Team:TU_Darmstadt/Materials/tctB197" class="external text" rel="nofollow">tctB_197 Biobrick</A>, <A href="https://2012.igem.org/Team:TU_Darmstadt/Materials/tphC" class="external text" rel="nofollow">tphC Biobrick</A>)
</LI></UL><DL><DD><DIV class="thumb tnone"><DIV class="thumbinner" style="width:302px;"><DIV class="thumbcaption">Finally a colony-PCR of C. testosteroni showed a second band on agarose gel electrophoresis. Marker: Lambda DNA/Eco471 Maker</DIV></DIV></DIV></DD></DL><DL><DD>The upper bands were cut and purified by Promega kit. <A href="https://2012.igem.org/Team:TU_Darmstadt/Protocols/Restriction_digest" class="external text" rel="nofollow">Restriction digest</A> using SpeI and EcoRI.
</DD></DL><UL><LI> Miniprep and Nanodrop measurement of:
</LI></UL><DL><DD>A3.1Â : 111.6 ng/Âµl 260/280= 1.72
</DD></DL><DL><DD>A3.4Â : 79.5 ng/Âµl 260/280= 1.66
</DD></DL><DL><DD>C4.6Â : 69.6 ng/Âµl 260/280= 1.76
</DD></DL><DL><DD>A4.2Â : 96.9 ng/Âµl 260/280= 1.62
</DD></DL><DL><DD>A2.4Â : 109.7 ng/Âµl 260/280= 1.72
</DD></DL><DL><DD>A2.5Â : 160.8 ng/Âµl 260/280= 1.79
</DD></DL><DL><DD>A5.13Â : 188.7 ng/Âµl 260/280= 1.85
</DD></DL><DL><DD>A5.12Â : 114.5 ng/Âµl 260/280= 1.81
</DD></DL><UL><LI> Checking the Miniprep with <A href="http://en.wikipedia.org/wiki/Polymerase_chain_reaction" class="external text" rel="nofollow">PCR</A> (Primers: <A href="https://2012.igem.org/Team:TU_Darmstadt/Materials/tctA503" class="external text" rel="nofollow">tctA_503 Biobrick</A>,  <A href="https://2012.igem.org/Team:TU_Darmstadt/Materials/tctB162" class="external text" rel="nofollow">tctB_162 Biobrick</A>, <A href="https://2012.igem.org/Team:TU_Darmstadt/Materials/tctB197" class="external text" rel="nofollow">tctB_197 Biobrick</A>, <A href="https://2012.igem.org/Team:TU_Darmstadt/Materials/tphC" class="external text" rel="nofollow">tphC Biobrick</A>) and <A href="https://2012.igem.org/Team:TU_Darmstadt/Protocols/Agarose_gel_electrophoresis" class="external text" rel="nofollow">Agarose gel electrophoresis</A></LI></UL><DL><DD><DIV class="thumb tnone"><DIV class="thumbinner" style="width:298px;"><DIV class="thumbcaption">All PCRs of miniprep showed a product. Marker: 1kb DNA ladder, NEB</DIV></DIV></DIV></DD></DL><UL><LI><A href="https://2012.igem.org/Team:TU_Darmstadt/Protocols/Colony_PCR" class="external text" rel="nofollow">Colony-PCR</A>-Screen of the transformants (Primers: <A href="https://2012.igem.org/Team:TU_Darmstadt/Materials/tctA505" class="external text" rel="nofollow">tctA_505 Biobrick</A>) and analisys of the PCR products by <A href="https://2012.igem.org/Team:TU_Darmstadt/Protocols/Agarose_gel_electrophoresis" class="external text" rel="nofollow">Agarose gel electrophoresis</A></LI></UL><DL><DD>No bands visible
</DD></DL><UL><LI><A href="https://2012.igem.org/Team:TU_Darmstadt/Protocols/" class="external text" rel="nofollow">Ligation</A> of <A href="http://partsregistry.org/wiki/index.php?title=Part:BBa_K808004" class="external text" rel="nofollow">tctA 505aa</A> into pSB1C3 and subsequently <A href="https://2012.igem.org/Team:TU_Darmstadt/Protocols/Heat_Shock_Transformation" class="external text" rel="nofollow">transformation</A> into DH5a. No colonies
</LI></UL><UL><LI>  The following vectors were sequenced: pSB1A2_197.4, pSB1A2_197.5, psSB1A2_503.1, pSB1A2_503.4, pSB1A2_162.2, pSB1C3_162.6, pSB1A2_322.12, pSB1A2_322.13
</LI></UL><UL><LI><A href="https://2012.igem.org/Team:TU_Darmstadt/Protocols/Restriction_digest" class="external text" rel="nofollow">Restriction digest</A> of <A href="http://partsregistry.org/wiki/index.php?title=Part:BBa_K808004" class="external text" rel="nofollow">tctA 505aa</A> using EcoRI/SpeI. <A href="https://2012.igem.org/Team:TU_Darmstadt/Protocols/" class="external text" rel="nofollow">Ligation</A> into  pSB1C3 and subsequently <A href="https://2012.igem.org/Team:TU_Darmstadt/Protocols/Heat_Shock_Transformation" class="external text" rel="nofollow">transformation</A> into DH5a.
</LI></UL><UL><LI> Preperations for pure culture: <A href="https://2012.igem.org/Team:TU_Darmstadt/Protocols/Colony_PCR" class="external text" rel="nofollow">colony-PCR</A> of A2.4, A2.5, A3.1, A3.4, A4.2, C4.6, A5.12, A5.13 (2 colonies each, Primers: <A href="https://2012.igem.org/Team:TU_Darmstadt/Materials/tctA503" class="external text" rel="nofollow">tctA_503 Biobrick</A>,  <A href="https://2012.igem.org/Team:TU_Darmstadt/Materials/tctB162" class="external text" rel="nofollow">tctB_162 Biobrick</A>, <A href="https://2012.igem.org/Team:TU_Darmstadt/Materials/tctB197" class="external text" rel="nofollow">tctB_197 Biobrick</A>, <A href="https://2012.igem.org/Team:TU_Darmstadt/Materials/tphC" class="external text" rel="nofollow">tphC Biobrick</A>) and afterwards checking the PCR products by <A href="https://2012.igem.org/Team:TU_Darmstadt/Protocols/Agarose_gel_electrophoresis" class="external text" rel="nofollow">Agarose gel electrophoresis</A>. Positive colonies were picked, streaked onto fresh media and incubated again.
</LI></UL><UL><LI> As A5.12 and A5.13 are missing the right insert the <A href="https://2012.igem.org/Team:TU_Darmstadt/Protocols/Colony_PCR" class="external text" rel="nofollow">colony-PCR</A> was repeated (5 and 6 colonies were used, Primers: <A href="https://2012.igem.org/Team:TU_Darmstadt/Materials/tphC" class="external text" rel="nofollow">tphC Biobrick</A>). To provide additional monitoring the <A href="https://2012.igem.org/Team:TU_Darmstadt/Protocols/Colony_PCR" class="external text" rel="nofollow">colony-PCR</A> was repeated for A2.5_2, A3.1_2 und A4.2_2. 2 positive colonies were again streaked onto fresh media.
</LI></UL><UL><LI> The Arabinose-Promotor (<A href="http://partsregistry.org/wiki/index.php?title=Part:BBa_K808000" class="external text" rel="nofollow">Ara</A>) was isolated from pbad_turbo_gfp and colonies containing this plasmid (3 approaches each).
</LI></UL><DL><DD><DIV class="thumb tnone"><DIV class="thumbinner" style="width:252px;"><DIV class="thumbcaption">Isolation of Ara. Approach T2 is further in use. Marker: 1kb DNA ladder, NEB</DIV></DIV></DIV></DD></DL><H4><SPAN class="mw-headline" id="Results_6">Results</SPAN></H4><UL><LI> Finally we made it to isolate gene 1 form a colony of <I>C. testosteroni</I>, but transformation didn't work
</LI><LI> Miniprep of the other genes, and sequencing
</LI><LI> Isolation of Ara
</LI></UL><H3><SPAN class="mw-headline" id="week_7_.2802.-06.07.12.29">week 7 (02.-06.07.12)</SPAN></H3><UL><LI><A href="https://2012.igem.org/Team:TU_Darmstadt/Protocols/Restriction_digest" class="external text" rel="nofollow">Restriction digest</A> of <A href="http://partsregistry.org/wiki/index.php?title=Part:BBa_K808000" class="external text" rel="nofollow">Ara</A> and pXylE using EcoRI and SpeI. <A href="https://2012.igem.org/Team:TU_Darmstadt/Protocols/DNA_Ligation" class="external text" rel="nofollow">Ligation</A> of the restricted products and <A href="https://2012.igem.org/Team:TU_Darmstadt/Protocols/Heat_Shock_Transformation" class="external text" rel="nofollow">transformation</A> into  DH5a. No colonies
</LI></UL><DL><DD><DIV class="thumb tnone"><DIV class="thumbinner" style="width:252px;"><DIV class="thumbcaption">Restriction digest of Ara. Marker: 1kb DNA ladder, NEB</DIV></DIV></DIV></DD></DL><UL><LI><A href="https://2012.igem.org/Team:TU_Darmstadt/Protocols/Restriction_digest" class="external text" rel="nofollow">Restriction digest</A> of pSB1A2 using EcoRI and SpeI
</LI></UL><UL><LI> DH5a-cells containing the vectors A4.2_2, A3.1_2, A2.5_2, A5.13_5 were diluted in glycerol (20%) and freezed (-80Â°C).
</LI></UL><UL><LI><A href="https://2012.igem.org/Team:TU_Darmstadt/Protocols/mutagenic_PCR" class="external text" rel="nofollow">Mutagenic PCR</A> of A4.2, A5.13, A3.1 to eliminate the PstI restriction site (Primers: <A href="https://2012.igem.org/Team:TU_Darmstadt/Materials/tctA503#tctA503_Pst1" class="external text" rel="nofollow">tctA_503_PstI</A>, <A href="https://2012.igem.org/Team:TU_Darmstadt/Materials/tctB162#tctB162_Pst1" class="external text" rel="nofollow">tctB_162_PstI</A>, <A href="https://2012.igem.org/Team:TU_Darmstadt/Materials/tphC#tphC_Pst1" class="external text" rel="nofollow">tphC_PstI</A>). <A href="https://2012.igem.org/Team:TU_Darmstadt/Protocols/Heat_Shock_Transformation" class="external text" rel="nofollow">Transformation</A> into DH5a
</LI><LI> Checking the DH5a-cells for correct transformation with the vectors after the mutagenic PCR by <A href="https://2012.igem.org/Team:TU_Darmstadt/Protocols/Colony_PCR" class="external text" rel="nofollow">colony-PCR</A> (checking 7 colonies each, Primers: <A href="https://2012.igem.org/Team:TU_Darmstadt/Materials/tctA503" class="external text" rel="nofollow">tctA_503 Biobrick</A>, <A href="https://2012.igem.org/Team:TU_Darmstadt/Materials/tctB162" class="external text" rel="nofollow">tctB_162 Biobrick</A>, <A href="https://2012.igem.org/Team:TU_Darmstadt/Materials/tphC" class="external text" rel="nofollow">tphC Biobrick</A>). The PCR products were <A href="https://2012.igem.org/Team:TU_Darmstadt/Protocols/Restriction_digest" class="external text" rel="nofollow">digested</A> with PstI.
</LI></UL><DL><DD><DIV class="thumb tnone"><DIV class="thumbinner" style="width:502px;"><DIV class="thumbcaption">Restriction digest of the genes after mutagenic PCR. The gel was alternately loaded with digested unmutated and digested mutated PCR products. Marker: 1kb DNA ladder, NEB</DIV></DIV></DIV></DD></DL><UL><LI> The transformation of DH5a with pSB1C3_tctA_505aa was verified by <A href="https://2012.igem.org/Team:TU_Darmstadt/Protocols/Colony_PCR" class="external text" rel="nofollow">colony-PCR</A> (Primers: <A href="https://2012.igem.org/Team:TU_Darmstadt/Materials/tctA505" class="external text" rel="nofollow">tctA_505 Biobrick</A>). The positive colonies were used for Miniprep
</LI></UL><UL><LI> Miniprep (Fermentas Kit) and Nanodrop measurement:
</LI></UL><DL><DD>A5.13m_1Â : 105.2 ng/Âµl 260/280= 1.84
</DD></DL><DL><DD>A5.13m_2Â : 223.0 ng/Âµl 260/280= 1.87
</DD></DL><DL><DD>A5.13m_3Â : 110.6 ng/Âµl 260/280= 1.87
</DD></DL><DL><DD>A5.13m_4Â : 99.6 ng/Âµl 260/280= 1.90
</DD></DL><DL><DD>A5.13m_6Â : 326.6 ng/Âµl 260/280= 1.88
</DD></DL><DL><DD>A5.13m_7Â : 145.0 ng/Âµl 260/280= 1.84
</DD></DL><DL><DD>A3.1m_1Â : 241.7 ng/Âµl 260/280= 1.87
</DD></DL><DL><DD>A3.1m_2Â : 188.4 ng/Âµl 260/280= 1.89
</DD></DL><DL><DD>A3.1m_3Â : 112.4 ng/Âµl 260/280= 1.89
</DD></DL><DL><DD>A3.1m_4Â : 173.1 ng/Âµl 260/280= 1.85
</DD></DL><DL><DD>A3.1m_5Â : 111.6 ng/Âµl 260/280= 1.85
</DD></DL><DL><DD>A3.1m_6Â : 138.1 ng/Âµl 260/280= 1.87
</DD></DL><DL><DD>A3.1m_7Â : 178.9 ng/Âµl 260/280= 1.86
</DD></DL><DL><DD>A4.2m_1Â : 57.0 ng/Âµl 260/280= 1.84
</DD></DL><DL><DD>A4.2m_2Â : 46.5 ng/Âµl 260/280= 1.77
</DD></DL><DL><DD>A4.2m_3Â : 55.7 ng/Âµl 260/280= 1.88
</DD></DL><DL><DD>A4.2m_4Â : 111.3 ng/Âµl 260/280= 1.82
</DD></DL><DL><DD>A4.2m_5Â : 236.9 ng/Âµl 260/280= 1.89
</DD></DL><DL><DD>A4.2m_6Â : 69.3 ng/Âµl 260/280= 1.89
</DD></DL><DL><DD>A4.2m_7Â : 72.3 ng/Âµl 260/280= 1.85
</DD></DL><DL><DD>C1.2Â : 132.9 ng/Âµl 260/280= 1.72
</DD></DL><UL><LI> new <A href="https://2012.igem.org/Team:TU_Darmstadt/Protocols/Heat_Shock_Transformation" class="external text" rel="nofollow">transformation</A> of DH5a with pXylE_Ara
</LI></UL><H4><SPAN class="mw-headline" id="Results_7">Results</SPAN></H4><UL><LI> First transformation of Ara into DH5a didn't work
</LI><LI> Successfull mutagenic PCR of genes 3, 4 and 5 (gene 2 didn't has a PstI Restriction site)
</LI><LI> Successfull transformation of DH5a with tctA 505aa
</LI><LI> Second transformation of DH5a with pXylE_Ara
</LI></UL><H3><SPAN class="mw-headline" id="week_8_.2809.-13.07.12.29">week 8 (09.-13.07.12)</SPAN></H3><UL><LI> Checking the transformation of DH5a with pSB1C3_tctA_505aa again using <A href="https://2012.igem.org/Team:TU_Darmstadt/Protocols/Colony_PCR" class="external text" rel="nofollow">colony-PCR</A> (Primers: <A href="https://2012.igem.org/Team:TU_Darmstadt/Materials/tctA505" class="external text" rel="nofollow">tctA_505 Biobrick</A>) (data not shown). The colonies Nr. 2, 8 and 10 were picked and streaked onto fresh media.
</LI></UL><UL><LI><A href="https://2012.igem.org/Team:TU_Darmstadt/Protocols/Colony_PCR" class="external text" rel="nofollow">colony-PCR</A> of DH5a_pXylE_Ara (Primers: <A href="https://2012.igem.org/Team:TU_Darmstadt/Materials/Ara_RBS" class="external text" rel="nofollow">Ara_RBS</A>, no positve colonies, data not shown)
</LI><LI> Sequencing the following vectors: pSB1A2_4.2m_5, pSB1A2_4.2m_4, pSB1A2_5.13m_3, pSB1A2_5.13m_2, pSB1A2_3.1m_1, pSB1A2_3.1m_5, pSB1A2_3.1m_7
</LI></UL><UL><LI> Checking the transformation of DH5a with A1 (5 colonies) and C1 (7 colonies) using <A href="https://2012.igem.org/Team:TU_Darmstadt/Protocols/Colony_PCR" class="external text" rel="nofollow">colony-PCR</A> (Primers: <A href="https://2012.igem.org/Team:TU_Darmstadt/Materials/tctA505" class="external text" rel="nofollow">tctA_505 Biobrick</A>, data not shown)
</LI></UL><UL><LI> Checking the transformation of DH5a with pSB1C3_tctA_505aa again by <A href="https://2012.igem.org/Team:TU_Darmstadt/Protocols/Colony_PCR" class="external text" rel="nofollow">colony-PCR</A> (Primers: <A href="https://2012.igem.org/Team:TU_Darmstadt/Materials/tctA505" class="external text" rel="nofollow">tctA_505 Biobrick</A>). Only the colonies Nr. 2, 8 and 10 (see above) were checked (2 colonies each).
</LI></UL><DL><DD><DIV class="thumb tnone"><DIV class="thumbinner" style="width:252px;"><DIV class="thumbcaption">Colony-PCR of colonies 2,8 and 10 from transformation with tctA 505aa. Marker: 1kb DNA ladder, NEB</DIV></DIV></DIV></DD></DL><UL><LI> Another <A href="https://2012.igem.org/Team:TU_Darmstadt/Protocols/Colony_PCR" class="external text" rel="nofollow">colony-PCR</A> of DH5a_pXy19_Ara (Primers: <A href="https://2012.igem.org/Team:TU_Darmstadt/Materials/Ara_RBS" class="external text" rel="nofollow">Ara_RBS</A>). The positive colonies were used for fluid culture (data not shown).
</LI></UL><UL><LI> Miniprep (Fermentas Kit) and Nanodrop measurement of:
</LI></UL><DL><DD>A5.13m_2Â : 191.2 ng/Âµl 260/280= 1.84
</DD></DL><DL><DD>A5.13m_3Â : 168.1 ng/Âµl 260/280= 1.78
</DD></DL><DL><DD>A2.5Â : 193.0 ng/Âµl 260/280= 1.83
</DD></DL><DL><DD>A3.1m_5Â : 146.1 ng/Âµl 260/280= 1.82
</DD></DL><DL><DD>A3.1m_1Â : 250.1 ng/Âµl 260/280= 1.84
</DD></DL><DL><DD>A4.2m_4Â : 124.8 ng/Âµl 260/280= 1.85
</DD></DL><UL><LI><A href="https://2012.igem.org/Team:TU_Darmstadt/Protocols/Restriction_digest" class="external text" rel="nofollow">Restriction digest</A> of A5.13m_2, A5.13m_3, A3.1m_1, A3.1m_5, A2.5, A4.2m_4 with EcoRI and SpeI. The restriction was done to enable the <A href="https://2012.igem.org/Team:TU_Darmstadt/Protocols/" class="external text" rel="nofollow">ligation</A> of the mutated genes into pSB1C3. Afterwards DH5a was <A href="https://2012.igem.org/Team:TU_Darmstadt/Protocols/Heat_Shock_Transformation" class="external text" rel="nofollow">transformed</A> with the ligation product.
</LI></UL><UL><LI> Miniprep (Fermentas Kit) and Nanodrop measurement of:
</LI></UL><DL><DD>C505_2_2Â : 253.6 ng/Âµl 260/280= 1.60
</DD></DL><DL><DD>C505_2_3Â : 274.7 ng/Âµl 260/280= 1.88
</DD></DL><DL><DD>C505_8_2Â : 462.1 ng/Âµl 260/280= 1.38
</DD></DL><DL><DD>C505_8_3Â : 149.1 ng/Âµl 260/280= 1.84
</DD></DL><DL><DD>C505_10_3Â : 160.4 ng/Âµl 260/280= 1.98
</DD></DL><UL><LI><A href="https://2012.igem.org/Team:TU_Darmstadt/Protocols/Restriction_digest" class="external text" rel="nofollow">Restriction digest</A> of the Miniprep products with EcoRI and EcoRI/SpeI. Afterwards the restriction products were analyzed by <A href="https://2012.igem.org/Team:TU_Darmstadt/Protocols/Agarose_gel_electrophoresis" class="external text" rel="nofollow">Agarose gel electrophoresis</A>. 
</LI></UL><DL><DD><DIV class="thumb tnone"><DIV class="thumbinner" style="width:602px;"><DIV class="thumbcaption">Restriction of miniprep. Gel loading: uncut, cut once, cut twice. C505_2_3 und C505_8_2 were selected for mutagenic PCR. Marker: 1kb DNA ladder, NEB</DIV></DIV></DIV></DD></DL><UL><LI><A href="https://2012.igem.org/Team:TU_Darmstadt/Protocols/Restriction_digest" class="external text" rel="nofollow">Restriction digest</A> of pSB1C3 with EcoRI and SpeI
</LI></UL><UL><LI><A href="https://2012.igem.org/Team:TU_Darmstadt/Protocols/Restriction_digest" class="external text" rel="nofollow">Restriction digest</A> of pXylE with EcoRI and XbaI and of <A href="http://partsregistry.org/wiki/index.php?title=Part:BBa_K808000" class="external text" rel="nofollow">Ara</A> with EcoRI and SpeI. Dephosphorylation of pXylE. <A href="https://2012.igem.org/Team:TU_Darmstadt/Protocols/" class="external text" rel="nofollow">Ligation</A> of pXylE and <A href="http://partsregistry.org/wiki/index.php?title=Part:BBa_K808000" class="external text" rel="nofollow">Ara</A></LI></UL><H4><SPAN class="mw-headline" id="Results_8">Results</SPAN></H4><UL><LI> Miniprep of the DH5a-Colonies with the mutated genes and restriction analysis
</LI><LI> Sequencing of the mutated genes
</LI><LI> Amplification and restriction of gene 2, 3, 4 and 5 for Ligation into pSB1C3
</LI><LI> Successfull isolation of Ara, Ligation with XylE
</LI></UL><H3><SPAN class="mw-headline" id="week_9_.2816.-20.07.12.29">week 9 (16.-20.07.12)</SPAN></H3><UL><LI><A href="https://2012.igem.org/Team:TU_Darmstadt/Protocols/mutagenic_PCR" class="external text" rel="nofollow">mutagenic PCR</A> of C505_2_3 and C505_8_2. (Primers: <A href="https://2012.igem.org/Team:TU_Darmstadt/Materials/tctA505#tctA505_Pst1" class="external text" rel="nofollow">tctA_505_PstI</A>, did not work, data not shown)
</LI></UL><UL><LI><A href="http://en.wikipedia.org/wiki/Polymerase_chain_reaction" class="external text" rel="nofollow">PCR</A> of <A href="http://partsregistry.org/wiki/index.php?title=Part:BBa_K808003" class="external text" rel="nofollow">tct_B162m</A> and <A href="http://partsregistry.org/wiki/index.php?title=Part:BBa_K808005" class="external text" rel="nofollow">tct_B197m</A> with primer to add a RBS in front and behind the gene (Primers: <A href="https://2012.igem.org/Team:TU_Darmstadt/Materials/tctB162#tctB162_RBS" class="external text" rel="nofollow">tctB_162_RBS</A>, <A href="https://2012.igem.org/Team:TU_Darmstadt/Materials/tctB197#tctB197_RBS" class="external text" rel="nofollow">tctB_197_RBS</A>)
</LI></UL><UL><LI><A href="https://2012.igem.org/Team:TU_Darmstadt/Protocols/Restriction_digest" class="external text" rel="nofollow">Restriction digest</A> of the PCR products with EcoRI and SpeI for <A href="https://2012.igem.org/Team:TU_Darmstadt/Protocols/" class="external text" rel="nofollow">ligation</A> into pSB1C3_5.13m (tphC)
</LI></UL><UL><LI> Adding a RBS to one site of <A href="http://partsregistry.org/wiki/index.php?title=Part:BBa_K808000" class="external text" rel="nofollow">Ara</A> using <A href="http://en.wikipedia.org/wiki/Polymerase_chain_reaction" class="external text" rel="nofollow">PCR</A> (Primers: <A href="https://2012.igem.org/Team:TU_Darmstadt/Materials/Ara_RBS" class="external text" rel="nofollow">Ara_RBS</A>)
</LI></UL><UL><LI> Miniprep (Fermentas Kit) and Nanodrop measurement of:
</LI></UL><DL><DD>C3.1m_1Â : 278.9 ng/Âµl 260/280= 1.88
</DD></DL><DL><DD>C4.2m_4Â : 214.2 ng/Âµl 260/280= 1.86
</DD></DL><DL><DD>5.13m_3Â : 256.3 ng/Âµl 260/280= 1.88
</DD></DL><DL><DD>C2.5_2Â : 44.8 ng/Âµl 260/280= 2.00
</DD></DL><DL><DD>C5.13m_2Â : 60.8 ng/Âµl 260/280= 1.89
</DD></DL><H4><SPAN class="mw-headline" id="Results_9">Results</SPAN></H4><UL><LI> We added successfully a RBS in front and behind gene 2 and 4
</LI><LI> We added successfully a RBS on Ara
</LI><LI> First mutagenic PCR on gene 1 didn't work
</LI></UL><H3><SPAN class="mw-headline" id="week_10_.2823.-27.07.12.29">week 10 (23.-27.07.12)</SPAN></H3><UL><LI><A href="https://2012.igem.org/Team:TU_Darmstadt/Protocols/Restriction_digest" class="external text" rel="nofollow">Restriction digest</A> of pSB1C3_5.13m_3_2 and pSB1C3_5.13m_3_3 with EcoRI and XbaI. Afterwards the products are <A href="https://2012.igem.org/Team:TU_Darmstadt/Protocols/" class="external text" rel="nofollow">ligated</A> with RBS_tctB162m_RBS and RBS_tctB197_RBS each.
</LI></UL><UL><LI><A href="https://2012.igem.org/Team:TU_Darmstadt/Protocols/Restriction_digest" class="external text" rel="nofollow">Restriction digest</A> of A3.1m_1 for <A href="https://2012.igem.org/Team:TU_Darmstadt/Protocols/" class="external text" rel="nofollow">Ligation</A> with Ara
</LI></UL><UL><LI><A href="https://2012.igem.org/Team:TU_Darmstadt/Protocols/Restriction_digest" class="external text" rel="nofollow">Restriction</A> of pXylE for <A href="https://2012.igem.org/Team:TU_Darmstadt/Protocols/" class="external text" rel="nofollow">ligation</A> with Ara_RBS
</LI></UL><UL><LI><A href="http://en.wikipedia.org/wiki/Polymerase_chain_reaction" class="external text" rel="nofollow">PCR</A> of Ara_RBS and <A href="http://partsregistry.org/wiki/index.php?title=Part:BBa_K808000" class="external text" rel="nofollow">Ara</A> for <A href="https://2012.igem.org/Team:TU_Darmstadt/Protocols/" class="external text" rel="nofollow">ligation</A> into pXylE and pSB1C3 (Primers: <A href="https://2012.igem.org/Team:TU_Darmstadt/Materials/Ara_RBS" class="external text" rel="nofollow">Ara_RBS</A>). Afterwards <A href="https://2012.igem.org/Team:TU_Darmstadt/Protocols/Restriction_digest" class="external text" rel="nofollow">restriction digest</A> with restriction enzymes: Ara_RBS is cut with with EcoRI and SpeI and Ara is cut with EcoRI and XbaI
</LI></UL><UL><LI><A href="https://2012.igem.org/Team:TU_Darmstadt/Protocols/" class="external text" rel="nofollow">Ligation</A> of the restricted PCR-products to:  pSB1A2_tctB197_tphCm, pSB1C3_tctB162m_tphCm, pSB1C3_tct197_tphCm, pSB1A2_tctB162m_tphCm, pSB1A2_Ara_tctA503m, pSB1C3_Ara_tctA503m, pSB1A2_Ara_XylE and <A href="https://2012.igem.org/Team:TU_Darmstadt/Protocols/Heat_Shock_Transformation" class="external text" rel="nofollow">transformation</A> into  DH5a
</LI></UL><UL><LI><A href="https://2012.igem.org/Team:TU_Darmstadt/Protocols/mutagenic_PCR" class="external text" rel="nofollow">mutagenic PCR</A> of 505_8_2 (Primers: <A href="https://2012.igem.org/Team:TU_Darmstadt/Materials/tctA505#tctA505_Pst1" class="external text" rel="nofollow">tctA_505_PstI</A>)
</LI></UL><H4><SPAN class="mw-headline" id="Results_10">Results</SPAN></H4><UL><LI> Ligation of gene RBS_2_RBS and RBS_4_RBS with gene 5 and transformation into DH5a
</LI><LI> Ligation of gene 3 with Ara_RBS and transformation into DH5a
</LI><LI> Mutagenic PCR of gene 1
</LI></UL><H3><SPAN class="mw-headline" id="week_11_.2830.07.-03.08.12.29">week 11 (30.07.-03.08.12)</SPAN></H3><UL><LI> Checking <A href="https://2012.igem.org/Team:TU_Darmstadt/Protocols/Heat_Shock_Transformation" class="external text" rel="nofollow">transformation</A> of DH5a by <A href="https://2012.igem.org/Team:TU_Darmstadt/Protocols/Colony_PCR" class="external text" rel="nofollow">colony-PCR</A>: pSB1A2_tctB197_tphCm, pSB1C3_tctB162m_tphCm, pSB1C3_tct197_tphCm, pSB1A2_tctB162m_tphCm, pSB1A2_Ara_tctA503m, pSB1C3_Ara_tctA503m, pSB1A2_Ara_XylE (Primers: <A href="http://partsregistry.org/Part:BBa_G00100" class="external text" rel="nofollow">VF2</A>, <A href="http://partsregistry.org/Part:BBa_G00101" class="external text" rel="nofollow">VR</A>). Positive colonies are picked and used for inoculation of fluid media.
</LI></UL><UL><LI> Miniprep (Fermentas Kit) of fluid culture and Nanodrop measurement of:
</LI></UL><DL><DD>pSB1C3_503Â : 110.0 ng/Âµl 260/280= 1.82
</DD></DL><DL><DD>pSB1A2_tctB197_tphCmÂ : 32.0 ng/Âµl 260/280= 2.10
</DD></DL><DL><DD>pSB1A2_Ara_tctA503mÂ : 202.0 ng/Âµl 260/280= 1.78
</DD></DL><DL><DD>pSB1A2_tctB162m_tphCmÂ : 36.5  ng/Âµl 260/280= 1.91
</DD></DL><DL><DD>pSB1C3_Ara_tctA503mÂ : 133.2 ng/Âµl 260/280= 1.83
</DD></DL><DL><DD>pSB1C3_tctB162m_tphCmÂ : 35.2 ng/Âµl 260/280= 1.93
</DD></DL><DL><DD>pSB1C3_tct197_tphCÂ : 30.6 ng/Âµl 260/280= 1.88
</DD></DL><UL><LI><A href="https://2012.igem.org/Team:TU_Darmstadt/Protocols/Restriction_digest" class="external text" rel="nofollow">Restriction digest</A> of pSB1A2_Ara_tctA503m, pSB1C3_Ara_tctA503m, pSB1A2_tctB162m_tphCm and pSB1C3_tctB162m_tphCm and building the final constructs by <A href="https://2012.igem.org/Team:TU_Darmstadt/Protocols/" class="external text" rel="nofollow">ligation</A>: pSB1A2_tctA503m_tctB162m_tphCm und pSB1C3_tctA503m_tctB162m_tphCm. <A href="https://2012.igem.org/Team:TU_Darmstadt/Protocols/Heat_Shock_Transformation" class="external text" rel="nofollow">Transformation</A> of the final constructs into DH5a.
</LI></UL><UL><LI><A href="https://2012.igem.org/Team:TU_Darmstadt/Protocols/Heat_Shock_Transformation" class="external text" rel="nofollow">Transformation</A> of pSB1C3_503 into DH5a
</LI></UL><UL><LI>  Checking the mutagenic PCR product of 505_8_2 and the transformation of DH5a with pSB1C3_503 by <A href="http://en.wikipedia.org/wiki/Polymerase_chain_reaction" class="external text" rel="nofollow">PCR</A> (Primers: <A href="https://2012.igem.org/Team:TU_Darmstadt/Materials/tctA505" class="external text" rel="nofollow">tctA_505 Biobrick</A>, <A href="https://2012.igem.org/Team:TU_Darmstadt/Materials/tctA503" class="external text" rel="nofollow">tctA_503 Biobrick</A>)
</LI></UL><UL><LI><A href="https://2012.igem.org/Team:TU_Darmstadt/Protocols/Restriction_digest" class="external text" rel="nofollow">Restriction digest</A> of <A href="http://partsregistry.org/wiki/index.php?title=Part:BBa_K808000" class="external text" rel="nofollow">Ara</A> und C4.2_4_1 using EcoRI and PstI, building pSB1C3_Ara by <A href="https://2012.igem.org/Team:TU_Darmstadt/Protocols/" class="external text" rel="nofollow">ligation</A> and <A href="https://2012.igem.org/Team:TU_Darmstadt/Protocols/Heat_Shock_Transformation" class="external text" rel="nofollow">transformation</A> of the vector into DH5a
</LI></UL><UL><LI> Miniprep (Fermentas Kit) and Nanodrop measurement of:
</LI></UL><DL><DD>pSB1C3_AraÂ : 86.7 ng/Âµl 260/280= 1.83
</DD></DL><DL><DD>pSB1C3_505mÂ : 113.8 ng/Âµl 260/280= 1.85
</DD></DL><UL><LI><A href="https://2012.igem.org/Team:TU_Darmstadt/Protocols/Restriction_digest" class="external text" rel="nofollow">Restriction digest</A> of pSB1C3_505m with PstI. Checking if the restriction site has been successfully removed
</LI></UL><DL><DD><DIV class="thumb tnone"><DIV class="thumbinner" style="width:102px;"><DIV class="thumbcaption">Restriction digest of tctA 505aa after mutagenic PCR. The gel was alternately loaded with digested and uncut PCR products. Restriction site hasn't been removed. Marker: 1kb DNA ladder, NEB</DIV></DIV></DIV></DD></DL><UL><LI> Screening for transformation of DH5a with pSB1A2_tctA503m_tctB162m_tphCm and pSB1C3_tctA503m_tctB162m_tphCm using <A href="https://2012.igem.org/Team:TU_Darmstadt/Protocols/Colony_PCR" class="external text" rel="nofollow">colony-PCR</A> (Primers: <A href="http://partsregistry.org/Part:BBa_G00100" class="external text" rel="nofollow">VF2</A>, <A href="http://partsregistry.org/Part:BBa_G00101" class="external text" rel="nofollow">VR</A>)
</LI></UL><UL><LI> Checking the constructs pSB1A2/pSB1C3_Ara_503m and pSB1A2/pSB1C3_tctB162m_tphCm by <A href="http://en.wikipedia.org/wiki/Polymerase_chain_reaction" class="external text" rel="nofollow">PCR</A> (Primers: <A href="http://partsregistry.org/Part:BBa_G00100" class="external text" rel="nofollow">VF2</A>, <A href="http://partsregistry.org/Part:BBa_G00101" class="external text" rel="nofollow">VR</A>)
</LI></UL><UL><LI><A href="https://2012.igem.org/Team:TU_Darmstadt/Protocols/" class="external text" rel="nofollow">Ligation</A> of <A href="http://partsregistry.org/wiki/index.php?title=Part:BBa_K808000" class="external text" rel="nofollow">Ara</A> into pSB1C3
</LI></UL><UL><LI> Checking if pSB1A2_tctB162m_tphCm and pSB1C3_tctB162m_tphCm have the right Prefix/Suffix by digesting both with EcoRI and XbaI and with EcoRI and PstI [*]
</LI></UL><UL><LI><A href="https://2012.igem.org/Team:TU_Darmstadt/Protocols/" class="external text" rel="nofollow">Ligation</A> of pSB1A2_tctB162m_tphCm and pSB1C3_tctB162m_tphCm with Ara_503
</LI></UL><UL><LI>  Miniprep (Fermentas Kit) and Nanodrop measurement of:
</LI></UL><DL><DD>pSB1A2_tctB162m_tphCm_1Â : 45.3 ng/Âµl 260/280= 1.90
</DD></DL><DL><DD>pSB1A2_tctB162m_tphCm_2Â : 74.5 ng/Âµl 260/280= 1.82
</DD></DL><DL><DD>pSB1C3_tctB162m_tphCm_1Â : 95.8 ng/Âµl 260/280= 1.86
</DD></DL><DL><DD>pSB1C3_tctB162m_tphCm_2Â : 85.7 ng/Âµl 260/280= 1.81
</DD></DL><DL><DD>pSB1C3_tctB162m_tphCm_3Â : 100.1 ng/Âµl 260/280= 1.84
</DD></DL><DL><DD>pSB1C3_tctB162m_tphCm_4Â : 79.9 ng/Âµl 260/280= 1.88
</DD></DL><DL><DD>pSB1C3_tctB162m_tphCm_5Â : 89.8 ng/Âµl 260/280= 1.85
</DD></DL><UL><LI> Sequencing pSB1A2_tctB162m_tphCm_2 and pSB1C3_tctB162m_tphCm_5
</LI></UL><H4><SPAN class="mw-headline" id="Results_11">Results</SPAN></H4><UL><LI> Restriction of the construct build in week 10 and ligation to the final construct Operon 1. Transformation into DH5a
</LI><LI> We checked the mutagenic PCR of gene 1. The restriction site wasn't removed
</LI><LI> Ligation of gene 3 into pSB1C3
</LI><LI> Ligation of Ara into pSB1C3
</LI><LI> Sequencing pSB1A2_tctB162m_tphCm_2 and pSB1C3_tctB162m_tphCm_5
</LI></UL><H3><SPAN class="mw-headline" id="week_12_.2806.-10.08.12.29">week 12 (06.-10.08.12)</SPAN></H3><UL><LI> Miniprep (Fermentas Kit) and Nanodrop measurement of:
</LI></UL><DL><DD>pSB1A2_tctB162m_tphCm_2Â : 55.7 ng/Âµl 260/280= 1.81
</DD></DL><DL><DD>pSB1C3_tctB162m_tphCm_5Â : 101.2 ng/Âµl 260/280= 1.92
</DD></DL><UL><LI><A href="https://2012.igem.org/Team:TU_Darmstadt/Protocols/mutagenic_PCR" class="external text" rel="nofollow">Mutagenic PCR</A> of pSB1C3_505_8_2 (Primers: <A href="https://2012.igem.org/Team:TU_Darmstadt/Materials/tctA505#tctA505_Pst1" class="external text" rel="nofollow">tctA_505_PstI</A>) and subsequent <A href="https://2012.igem.org/Team:TU_Darmstadt/Protocols/Heat_Shock_Transformation" class="external text" rel="nofollow">transformation</A> into DH5a
</LI></UL><UL><LI><A href="https://2012.igem.org/Team:TU_Darmstadt/Protocols/" class="external text" rel="nofollow">Ligation</A> of Ara_503m with pSB1A2_tctB162m_tphCm/pSB1C3_tctB162m_tphCm and subsequent <A href="https://2012.igem.org/Team:TU_Darmstadt/Protocols/Heat_Shock_Transformation" class="external text" rel="nofollow">transformation</A> into DH5a. Screening the transformants using <A href="https://2012.igem.org/Team:TU_Darmstadt/Protocols/Colony_PCR" class="external text" rel="nofollow">colony-PCR</A> (Primers: <A href="http://partsregistry.org/Part:BBa_G00100" class="external text" rel="nofollow">VF2</A>, <A href="http://partsregistry.org/Part:BBa_G00101" class="external text" rel="nofollow">VR</A>)
</LI></UL><DL><DD><DIV class="thumb tnone"><DIV class="thumbinner" style="width:252px;"><DIV class="thumbcaption">Control of ligation (10 colonies tested). Only colony 1, 2 and 3 have the right insert. Marker: 1kb DNA ladder, NEB</DIV></DIV></DIV></DD></DL><UL><LI> Miniprep (Fermentas Kit) and Nanodrop measurement of:
</LI></UL><DL><DD>pSB1A2_Ara_tctA503mÂ : 278.0 ng/Âµl 260/280= 1.86
</DD></DL><DL><DD>pSB1C3_Ara_tctA503mÂ : 169.0 ng/Âµl 260/280= 1.87
</DD></DL><DL><DD>pSB1C3_Ara_1Â : 59.0 ng/Âµl 260/280= 1.90
</DD></DL><DL><DD>pSB1C3_Ara_4Â : 55.5 ng/Âµl 260/280= 1.80
</DD></DL><DL><DD>pSB1C3_Ara_tctA503m_tctB162m_tphCmÂ : 188.8 ng/Âµl 260/280= 1.86
</DD></DL><DL><DD>pSB1C3_Ara_tctA503m_tctB162m_tphCmÂ :168.8  ng/Âµl 260/280= 1.84
</DD></DL><DL><DD>pSB1C3_Ara_tctA503m_tctB162m_tphCmÂ :271.3  ng/Âµl 260/280= 1.85
</DD></DL><DL><DD>pSB1C3_Ara_XylEÂ : 149.8 ng/Âµl 260/280= 1.86
</DD></DL><DL><DD>pSB1C3_tctB197_tphCÂ : 95.8 ng/Âµl 260/280= 1.84
</DD></DL><DL><DD>pSB1A2_tctB197_tphCÂ : 69.0 ng/Âµl 260/280= 1.88
</DD></DL><UL><LI> Sequencing of 3 different pSB1C3_Ara_tctA503m_tctB162m_tphC
</LI></UL><UL><LI> Analysis of mutagenic PCR products of 505_8_2 and <A href="https://2012.igem.org/Team:TU_Darmstadt/Protocols/Restriction_digest" class="external text" rel="nofollow">restriction digest</A> with PstI
</LI></UL><UL><LI>Restriction digest of tctA 505aa after mutagenic PCR. The gel was alternately loaded with digested and uncut PCR products. Restriction site hasn't been removed. Marker: 1kb DNA ladder, NEB
</LI></UL><DL><DD><DIV class="thumb tnone"><DIV class="thumbinner" style="width:402px;"><DIV class="thumbcaption">Restriction digest of tctA 505aa after mutagenic PCR. The gel was alternately loaded with digested and uncut PCR products. Restriction site has successfully been removed. Marker: 1kb DNA ladder, NEB</DIV></DIV></DIV></DD></DL><UL><LI><A href="https://2012.igem.org/Team:TU_Darmstadt/Protocols/Restriction_digest" class="external text" rel="nofollow">Restriction digest</A> of Ara_RBS with EcoRI and SpeI and of pSB1C3_505m with EcoRI and XbaI. Building pSB1C3_Ara_505m by <A href="https://2012.igem.org/Team:TU_Darmstadt/Protocols/" class="external text" rel="nofollow">ligation</A> and subsequent <A href="https://2012.igem.org/Team:TU_Darmstadt/Protocols/Heat_Shock_Transformation" class="external text" rel="nofollow">transformation</A> into DH5a.
</LI></UL><H4><SPAN class="mw-headline" id="Results_12">Results</SPAN></H4><UL><LI> Successful mutagenic PCR of gene 1 and transformation into DH5a
</LI><LI> Ligation of Ara and gene 1 into pSB1C3 and transformation into DH5a
</LI></UL><H3><SPAN class="mw-headline" id="week_13_.2813.-17.08.12.29">week 13 (13.-17.08.12)</SPAN></H3><UL><LI><A href="https://2012.igem.org/Team:TU_Darmstadt/Protocols/Restriction_digest" class="external text" rel="nofollow">Restriction digest</A> of pSB1C3_197_tphCm with EcoRI and XbaI
</LI></UL><UL><LI> Checking the transformation of DH5a with pSB1C3_Ara_505m by using <A href="https://2012.igem.org/Team:TU_Darmstadt/Protocols/Colony_PCR" class="external text" rel="nofollow">colony-PCR</A> (Primers: <A href="http://partsregistry.org/Part:BBa_G00100" class="external text" rel="nofollow">VF2</A>, <A href="http://partsregistry.org/Part:BBa_G00101" class="external text" rel="nofollow">VR</A>)
</LI></UL><DL><DD><DIV class="thumb tnone"><DIV class="thumbinner" style="width:502px;"><DIV class="thumbcaption">Colony-PCR of DH5a with pSB1C3_Ara_505m. All Colonies are positive. Marker: 1kb DNA ladder, NEB</DIV></DIV></DIV></DD></DL><UL><LI> Sequencing pSB1C3_Ara_XylE, pSB1C3_Ara_tctA503m_tctB162m_tphCm (=Operon1), pSB1C3_505m
</LI></UL><UL><LI><A href="https://2012.igem.org/Team:TU_Darmstadt/Protocols/Restriction_digest" class="external text" rel="nofollow">Restriction digest</A> of Ara_RBS with EcoRI and SpeI and of pSB1C3_tphCm with EcoRI and XbaI. Building pSB1C3_Ara_tphCm by <A href="https://2012.igem.org/Team:TU_Darmstadt/Protocols/" class="external text" rel="nofollow">ligation</A> and subsequent <A href="https://2012.igem.org/Team:TU_Darmstadt/Protocols/Heat_Shock_Transformation" class="external text" rel="nofollow">transformation</A> into DH5a
</LI></UL><UL><LI><A href="https://2012.igem.org/Team:TU_Darmstadt/Protocols/Restriction_digest" class="external text" rel="nofollow">Restriction digest</A> of pSB1C3_Ara_tctA505m with EcoRI and SpeI. <A href="https://2012.igem.org/Team:TU_Darmstadt/Protocols/" class="external text" rel="nofollow">Ligation</A> of Ara_tctA505m and tctB197_tphCm into pSB1C3
</LI></UL><H4><SPAN class="mw-headline" id="Results_13">Results</SPAN></H4><UL><LI> Checking the Transdormation of pSB1C3_Ara_505m
</LI><LI> Sequencing pSB1C3_Ara_XylE, pSB1C3_Ara_tctA503m_tctB162m_tphCm (=Operon1), pSB1C3_505m 
</LI><LI> We created pSB1C3_Ara_tphCm by ligation
</LI></UL><H3><SPAN class="mw-headline" id="week_14_.2820.-24.08.12.29">week 14 (20.-24.08.12)</SPAN></H3><UL><LI> Checking the transformation of DH5a with pSB1C3_Ara_tphCm using <A href="https://2012.igem.org/Team:TU_Darmstadt/Protocols/Colony_PCR" class="external text" rel="nofollow">colony-PCR</A> (Primers: <A href="http://partsregistry.org/Part:BBa_G00100" class="external text" rel="nofollow">VF2</A>, <A href="http://partsregistry.org/Part:BBa_G00101" class="external text" rel="nofollow">VR</A>)
</LI></UL><DL><DD><DIV class="thumb tnone"><DIV class="thumbinner" style="width:302px;"><DIV class="thumbcaption">Colony-PCR of the transformation of DH5a with pSB1C3_Ara_tphCm. Colonies 1, 2, 3, 4, 5, 6, 9,10 and 11 are positive. Marker: 1kb DNA ladder, NEB</DIV></DIV></DIV></DD></DL><P><B>Result</B>
Ara_tphC was designed
</P><H3><SPAN class="mw-headline" id="Week_15-19_charakterization_and_uptake_tests">Week 15-19 charakterization and uptake tests</SPAN></H3><UL><LI><A href="https://2012.igem.org/Team:TU_Darmstadt/Protocols/Restriction_digest" class="external text" rel="nofollow">Restriction digest</A> of pSB1C3_Ara_tphCm and pSB1C3_Ara_tctA505m with PstI and SpeI
</LI></UL><UL><LI><A href="https://2012.igem.org/Team:TU_Darmstadt/Protocols/Restriction_digest" class="external text" rel="nofollow">Restriction digest</A> of pSB1C3_197_tphCm with XbaI and PstI.
</LI></UL><UL><LI> Building the construct pSB1C3_Ara_tctA505m_197_tphCm (= Operon2) by <A href="https://2012.igem.org/Team:TU_Darmstadt/Protocols/" class="external text" rel="nofollow">ligation</A>. Subsequent <A href="https://2012.igem.org/Team:TU_Darmstadt/Protocols/Heat_Shock_Transformation" class="external text" rel="nofollow">transformation</A> into DH5a
</LI></UL><UL><LI> Checking the transformation of DH5a with Operon2 using <A href="https://2012.igem.org/Team:TU_Darmstadt/Protocols/Colony_PCR" class="external text" rel="nofollow">colony-PCR</A> (Primers: <A href="http://partsregistry.org/Part:BBa_G00100" class="external text" rel="nofollow">VF2</A>, <A href="http://partsregistry.org/Part:BBa_G00101" class="external text" rel="nofollow">VR</A>)
</LI></UL><DL><DD><DIV class="thumb tnone"><DIV class="thumbinner" style="width:102px;"><DIV class="thumbcaption">Colony-PCR of transformation of DH5a with Operon2. Marker: 1kb DNA ladder, NEB</DIV></DIV></DIV></DD></DL><UL><LI><B>Overexpression of designed parts in <I>E. coli C43 (DE3)</I> to verificating the protein expression and isolation of membranes for SDS-PAGE</B></LI></UL><DIV class="thumb tright"><DIV class="thumbinner" style="width:402px;"><DIV class="thumbcaption"><B>Figure 1.</B> SDS-PAGE of isolated membranes  after overexpression <I>E Coli strain C43(DE3)</I> carrying <B>1.</B>putative terephtalate Operon1,<B>2.</B> putative terephtalate Operon2,<B>3.</B>AraC-Pbad ,<B>4.</B>AraC-Pbad_tctA_503 and <B>5.</B>AraC-Pbad_322 plasmids. <I>From left to right:</I><B>3.</B>(5ÂµL),<B>3.</B>(10ÂµL),<B>4.</B>(5ÂµL),<B>4.</B>(10ÂµL),Marker(Aplichem proteinmarker 3,<B>2.</B>(5ÂµL),<B>2.</B>(10ÂµL),<B>5.</B>(5ÂµL),<B>5.</B>(10ÂµL)/(xÂµL) membranes volume. The SDS PAGE was strained with Coomassie R-250</DIV></DIV></DIV><P>Three different overnight culture  of <I>E Coli strain C43(DE3)</I> carrying putative terephtalate Operon1  ), putative terephtalate Operon2 (BBa_K808007 ) and AraC-Pbad plasmids (BBa_K808000 ) was inoculated 1:100 into fresh 100 mL LB media. At the OD 600 of 0,8-1,0  0,02Â % L-(+)-Arabinose (inducer) was added  to the cultures  and incubated   at room tempreture (RT)and shaking at 230 rpm for 6 hours. After the incubation the culltures were centrifuged at 10.00xg and washed 2 times in 5 mL phosphate buffer saline buffer (PBS) and resuspenden in 5 mL PBS. Phenylmethylsulfonyl fluoride (PMSF,1 mM final concentration) and DNAse II (100 U/mL final concentration) was added to the resuspenden cultures and the cells were broked down for 2 times with french press at 10.000 psi. To remove of  unbroked cell and debris a centrifugation at 12.000xg for 20 min followed. The supernatan was transfered into a ultrazentrifugation tubes and centrifugated at 100.000xg for 45 min to collect membranes. The pelletts were resuspended in 300 ÂµL PBS and were used as a sample to perfomed a <A href="http://openwetware.org/wiki/Keating:Experimental_Protocols:SDS-PAGE" class="external text" rel="nofollow">SDS-PAGE.</A> (Fig.1) 
</P><P><B>Result</B></P><P>Figure 1 showes no  significant protein band. It suggest, that the SDS as a detergent was a wrong choice or the expression of the membrane proteins was to low to detect it, but itÂ´s well known that membrane proteins exhibit a low expression level and thus not able to detect with conventinal SDS-PAGE. For futher analyses we have to amplify the signal with a antibody staining (Conventional Western-Blot analysis).
</P><UL><LI><B>Charakterisation of the AraC-Pbad</B></LI></UL><P>The charakterization of the AraC-Pbad is described <A href="http://partsregistry.org/Part:BBa_K808000" class="external text" rel="nofollow">here</A></P><H3><SPAN class="mw-headline" id="Uptake"> Uptake</SPAN></H3><P>To verify the corellation between the designed putative terephtalate uptake operons and the uptake of TPA in <I>E. coli</I>, we used three strategies to detected the uptake.
</P><OL><LI><B>Photometry</B></LI><LI><B>Gas chromatographyâmass spectrometry (GC-MS)</B></LI><LI><B>Energy dispersive X-ray spectroscopy (EDX)</B></LI></OL><UL><LI><B>1. Photometric  measurement of TPA uptake in different  media at 240 nm</B></LI></UL>
Our first strategy to check our designed uptake system operon 1 and operon 2 was to checked  the absorbation at 240 nm. For that reason we developed a TPA standard curve in different media. <DIV class="thumb tright"><DIV class="thumbinner" style="width:302px;"><DIV class="thumbcaption"><B>Figure.1</B>In order to verify the uptake of TPA  into bacteria, we studied the influence of TPA on a solutionâs absorbance of light. A decrease in the concentration of TPA by time should result in lowering optical density at its wavelength of extinction. Scanning the spectrum of TPA (80 ÂµM in H2O) with an OD measuring device led to a single peak at a wavelength of approximately 240 nm. Therefore a serial dilution of TPA in H2O was inspected photometrically, ranging in concentration from 400 ÂµM to 50 nM. The measurementâs results are described by the graphs below. The chart shows a quite accurate linear correlation between TPA-concentration and optical density (series 1; linear section is highlighted in blue color). Note that the curve progression differs more and more from proportionality for concentrations above about 250 ÂµM TPA (red colored section of series 1). We assume the lack of precision of our measuring device when dealing with OD values higher than about 2.5 to be the reason for this stagnation.</DIV></DIV></DIV><P>In the following series differences between solutions of TPA in H2O and other fluids had to be found where upon M9-medium was of main interest, since this was planned to be used as medium for production of the eventually used modified E. coli. The results are represented in series 2 (green graph). The tight gap between the two curves led us to the decision that their differences was of no significance for our further measurements.
An overnight cultures of '<I>E. Colis DH5apha</I> carrying putative terephtalate <A href="http://partsregistry.org/wiki/index.php?title=Part:BBa_K808006" class="external text" rel="nofollow">Operon1</A>, putative terephtalate <A href="http://partsregistry.org/wiki/index.php?title=Part:BBa_K808007" class="external text" rel="nofollow">Operon2</A> were inoculated 1:100 in a freshly 10 mL M9 minimal medium cultivated shaking at 230 rpm at 30Â°C. At the OD600 of 0.5 0.001Â % L-(+)-Arabinose(inducer) and 100 ÂµM TPA was added and incubated for 6 hours at 30 Â°C and shaked at shaking 230 rpm. Every hour 500 ÂµL of the cultures was transfered into a 1.5 mL Eppendorf cup and centrifuged at 17xg to spin down the cells for 5 min. Afterwards the TPA concentration was measured at 240 nm. 
</P><P><B>Result</B></P><P>This strategy did not work. The M9 minimal medium had an absorbation at 240 nm after <I>E. colis</I>  were incubated until the OD600 of 0.5. Our assumption is that lyseted <I>E. colis DH5alpha</I> are responsible for the absorbation or  <I>E. colis</I> secreteting moleculs into the medium, which are absorbats at 240 nm. 
</P><UL><LI><B>2. Preparation of <I>E. coli</I> strain C43(DE3) carrying the putative terephtalate  Operon1, putative terephtalate  Operon2 and AraC-Pbad plasmids  for the Gas chromatographyâmass spectrometry (GC-MS)</B></LI></UL><P>Three different overnight culture  of <I>E. colis C43(DE3)</I> carrying putative terephtalate <A href="http://partsregistry.org/wiki/index.php?title=Part:BBa_K808006" class="external text" rel="nofollow">Operon1</A>,putative terephtalate <A href="http://partsregistry.org/wiki/index.php?title=Part:BBa_K808007" class="external text" rel="nofollow">Operon2</A> and AraC-Pbad plasmids were inoculated 1:100 into fresh 100 mL LB medium. 0,001% L-(+)-Arabinose (inducer) was added at the optical density  (OD600) of 1.3-1.4 and incubated at 30Â°C for 2 hour, shaked at 230 rpm. Afterwards  2 mL of each cultures were transferred to a  new 2 mL Epppendorf cup with each 50,-100,-150, -200,-250,-300,- ÂµM TPA and incubated on a heating-block at 30 Â°C, shaked at 1000 rpm for 5 hours. After the incubation the cultures were centrifuged at 17.000xg for 15 min. The culture with 100 ÂµM TPA cultures were chosed for GC-MC measurment.The supernatan was transferred into a new Eppendorf cup and the protocol <A href="https://2012.igem.org/Team:TU_Darmstadt/Protocols/GC-MS" class="external text" rel="nofollow">sample preparation for GC-MS </A> was accomplished.
</P><P><B>Result</B></P><P>We successfully showed that we are able to qualify and thus quantify TPA-di-methyl-ester via GC-MS. Furthermore, we quantified the operon 1 and operon 2 sample of the transport group. Summarising, we obtained the following peak-areas:
operon-1: 43113
operon-2: 38700
controlÂ : 32729
Interestingly, we measured a lower concentration of TPA in our control sample. The reason for this might be that the OD of our control sample was higher (0.85) than the OD of our biological samples. Notably, that our untransformed bacteria is able to take TPA up into the periplasma. Nonetheless, due to the the analysis, we measured significant differences in our peak areas. The delta of 4413 digits of Operon-1 to Operon-2 is may be result of a working TPA transport system. For further work we have to measure more samples (biger statistic), measure a control which contains only TPA in the cycl.hexan matrix and using constant ODs by the sample preparations. Moreover, the results have to be compared to HPLC-MS measurements. Unfortunately, the wiki freeze is today, so we are not able to measure again (this time.
</P><P>
For more details: <A href="https://2012.igem.org/Team:TU_Darmstadt/Protocols/GC-MS/Results" class="external free" rel="nofollow">https://2012.igem.org/Team:TU_Darmstadt/Protocols/GC-MS/Results</A></P><UL><LI><B>3. Preparation of <I>E. colis C43(DE3)</I> for the energy dispersive X-ray spectroscopy (EDX)</B></LI></UL><DIV class="thumb tright"><DIV class="thumbinner" style="width:302px;"><DIV class="thumbcaption"><B>Figure 2.</B> An electron microscope picture of embedded <I>E. colis C43(DE3)</I> carrying putative terephtalate Operon1</DIV></DIV></DIV><P>Three different overnight culture  of <I>E. colis C43(DE3)'</I> carrying putative terephtalate  <A href="http://partsregistry.org/wiki/index.php?title=Part:BBa_K808006" class="external text" rel="nofollow">Operon1</A>, putative terephtalate <A href="http://partsregistry.org/wiki/index.php?title=Part:BBa_K808007" class="external text" rel="nofollow">Operon2</A> and AraC-Pbad plasmids  was inoculated 1:100 into fresh 100 mL LB media. At the OD 600 of 0.8 - 1.0  0.02Â % L-(+)-Arabinose (inducer) was added  to the cultures  and incubated at room tempreture (RT) for 5 hours, shaking at 230 rpm. After the incubation the cultures were cenrifuged 15 min at 10.00xg and the pelletes were prepared for the energy dispersive X-ray spectroscopy (EDX). The cells had to be fixed, embedded in a SPURR-embedding medium (resin) and cut in to small rectangles smaller than 100 nm. For more details: <A href="https://2012.igem.org/Team:TU_Darmstadt/Protocols/SamplePreparationEDX" class="external text" rel="nofollow">EDX Preparation of Escherichia Coli for the energy dispersive X-ray spectroscopy (EDX)</A> The  embedded <I>E. Colis</I> was checked under an electron microscope
</P><P><B>Result</B></P><P>The spectra show a clear difference between background and sample measurement. In the background spectra no nitrogen (N) and phosphor (P) occurs. These elements are perfect markers for biological material. Although there is nor clear evidence for the uptake of TPA into the cells. The TAM-EDX experimental set up is a good method to achieve this goal. Unfortunately time is short in the iGEM competition, otherwise we would add a heavy atom like bromide or chloride to TPA.  As seen in the spectra non of this elements occur in our sample, which would make it possible to clearly identify the source of a Br/Cl peak as TPA. 
</P><P>For more details: <A href="https://2012.igem.org/Team:TU_Darmstadt/Protocols/TEM-EDX" class="external text" rel="nofollow">TEM-EDX measurment</A></P><DIV class="printfooter">
Retrieved from &quot;<A href="http://2012.igem.org/Team:TU_Darmstadt/Labjournal/Transport">http://2012.igem.org/Team:TU_Darmstadt/Labjournal/Transport</A>&quot;</DIV></DIV></DIV><DIV id="footer-box" class="noprint"><DIV id="footer"><UL id="f-list"><LI id="t-recentchanges"><A href="/Special:RecentChanges" title="Recent changes">Recent changes</A></LI><LI id="t-whatlinkshere"><A href="/Special:WhatLinksHere/Team:TU_Darmstadt/Labjournal/Transport" title="List of all wiki pages that link here [j]" accesskey="j">What links here</A></LI><LI id="t-recentchangeslinked"><A href="/Special:RecentChangesLinked/Team:TU_Darmstadt/Labjournal/Transport" title="Recent changes in pages linked from this page [k]" accesskey="k">Related changes</A></LI><LI id="t-specialpages"><A href="/Special:SpecialPages" title="List of all special pages [q]" accesskey="q">Special pages</A></LI><LI><A href="/Special:Preferences">My preferences</A></LI></UL></DIV><DIV id="footer"><UL id="f-list"><LI id="t-print"><A href="/wiki/index.php?title=Team:TU_Darmstadt/Labjournal/Transport&amp;printable=yes" title="Printable version of this page [p]" accesskey="p">Printable version</A></LI><LI id="t-permalink"><A href="/wiki/index.php?title=Team:TU_Darmstadt/Labjournal/Transport&amp;oldid=232494" title="Permanent link to this revision of the page">Permanent link</A></LI><LI id="privacy"><A href="/2012.igem.org:Privacy_policy" title="2012.igem.org:Privacy policy">Privacy policy</A></LI><LI id="disclaimer"><A href="/2012.igem.org:General_disclaimer" title="2012.igem.org:General disclaimer">Disclaimers</A></LI></UL></DIV></DIV></DIV></DIV></BODY></HTML>