"
Page
Discussion
View source
History
teams
Log in
 
Team:Imperial College/PCR
From 2008.igem.org
Home
Biofabricator Subtilis
Project Specifications Why B. subtilis? B. subtilis: Benefits vs Challenges Project Summary Example Application: Biocouture
Dry Lab
Dry Lab Hub Growth Curves Genetic Circuits Motility Analysis Appendices - Code etc.
Wet Lab
Wet Lab Hub Cloning Strategy Experiments & Protocols BioBricks Submitted Experimental Results
Notebook
Our Team
PCR
This protocol is desgined for use with the stratagene PfuUltra II Fusion DNA polymerase and is based in part on the PfuUltra II Fusion DNA polymerase usage manual. PfuUltra II Fusion Manual
Aim
To produce clones of two genes from B.subtilis that are too big to have synthesised by GeneArt; for use as an integration site and gene knockout (epsE) or for their original purpose as a transcriptional regulator (xylR).
To produce clones of sequences from vectors; for use as integration sites (amyE), antibiotic resistance (Spectinomycin - aad9) and as a transcriptional repressor (lacI).
A modified protocol for using Taq polymerase can be used if less fidelity is required, to obtainoptimal conditions for use with Pfu DNA polymerase, or to carry out single colony PCR for verification purposes (see below).
Equipment
Heated lid PCR machine
Thin walled PCR tube
Reagents
The reagents required for Pfu and Taq PCR are very similar. The regaents listed here are required for Pfu PCR, the minor changes required for Taq PCR are listed in the Taq PCR secton below.
When producing Biobrcik parts, the forward and reverse primers should contain the Biobrick prefix (forward primer) and the complementary sequence to the Biobrick suffix (reverse primer) 5' of the beginning of the annealing sequence.
Distilled H2O: 18.5μL
10Χ PfuUltra™II reaction buffer: 2.5μL
dNTP mix (10mM): 0.5μL
B.subtilis genomic DNA (100ng/μL): 1μL
Forward Primer (100ng/μL): 1μL
Reverse Primer (100ng/μL): 1μL
PfuUltra® II fusion HS DNA polymerase: 0.5μL
Total Reaction Volume: 25μL
Note: Template DNA should be diluted to 100ng/μL. If template DNA concentration is below 100ng/μL, 100ng of DNA should be added and the volume of H2O to be added should be adjusted to maintain a reaction volume of 25μL.
If a vector is used as the template, 5ng of plasmid DNA should be used instead and the volume of H2O to be added should be adjusted accordingly.
If PCR is proving difficult, particularly for denaturation, DMSO can be added to a final concentration of 10% to increase efficiency
Protocol
Add all the reagents in order (down the list) sequentially to the PCR tube, mxing after each addition.
Place the PCR tubes into the PCR machine and set the programme to the following set-up:
Initial Denaturation: 30 seconds at 95°C (longer for genomic DNA)
10 Cycles of:
30 second denaturation at 95°C
30 second annealing time at Primer Tm - ~3°C (complementary section of primer Tm)
15 seconds (+ 15 second for each additional kb) extending time at 68C (30 seconds for genomic templates)
20 - 30 Cycles of:
30 second denaturation at 95°C
30 second annealing time at Primer Tm - ~3°C (total primer Tm)
15 seconds (+ 15 second for each additional kb)extending time at 68°C (30 seconds for genomic templates)
Final Extension: 5 minutes at 68°C
The resulting solution can then be purified using a PCR purification column, by gel electrophoresis followed by spin purification or can simply be ligated ready for use.
Taq PCR
If using Taq DNA polymerase, the reaction mixture is effectively the same (with Taq buffer rather than Pfu buffer), 0.5μL of Taq should be used and care should be taken to ensure that a suitable amount(1.5-4 mM) of magnesium (usually as magnesium chloride) is present in the reaction, as some Taq buffers do not contain magnesium.
For Taq PCR, the temperature of the extension step should also be raised to 72°C and more time allowed for extension (60 seconds per kb usually works fine with Taq DNA polymerase)
Single Colony PCR
Single colony PCR is a useful tool for rapidly verifying if a colony contains the plasmid with the correct DNA after a ligation. The colony is replica plated, boiled and then used as a PCR template, potenetially with the registry VR and VF2 primers to obtain the approxiamte length of an insert.
The reagents are identical to that of a Taq PCR and the method only differs by template preparation and PCR programme.
Preparation of Samples
Further equipment needed: Heating block (or waterbath) capable of eaching 95°C
It is adviasble to replica plate andy samples taken to allow the cells to be grown up after the PCR, as sample preparation kills the cells.
Heat up heating block/waterbath to 95°C
Select distinct colonies from a plate for testing
Pipette 100μL of sterile H2O into eppendorf tubes (1 per colony)
Pick each colony in turn, replica plate the colony on a fresh plate and then mix the loop tip with the water in the eppendorf tube to leave cells as a sample
Be sure to use a fresh tube of water fo reach sample!
Boil the water and cells soltuion for 5 minutes in the heating block/waterbath
Be careful not to leave the sample in the heating block/waterbath too long as this may damage the DNA template
Each sample is now ready to be used as a DNA template in an PCR reaction
PCR
Single colony PCR requires the same set of reagents as standard Taq PCR (see above for reagents and amounts), the DNA template will be 1μL of the sample that was just created.
As with all Taq PCRs, ensure that your PCR reaction contains magnesium as this is vital for Taq DNA polymerase to work correctly.
The programme for the PCR machine is simpler when not producing Biobricks, PCR tubes should be placed into a pre-warmed machine(depending on make and model) and the machine set to;
Initial Denaturation: 30 seconds at 95°C (longer for a genomic DNA template (eg. checking integration))
30 Cycles of:
30 second denaturation at 95°C
30 second annealing time at Primer Tm - ~3°C
30 seconds (+60 seconds for each kb of insert) extending time at 72°C (longer for genomic templates)
Obtaining Results
5μL of each PCR reation should then be mixed with DNA loading buffer and all samples run on a 1% agarose gel with suitable marker and controls to check the length of inserts.
< Previous
Back to top
Next >
Retrieved from " http://2008.igem.org/Team:Imperial_College/PCR "
Recent changes
What links here
Related changes
Special pages
My preferences
Printable version
Permanent link
Privacy policy
Disclaimers
