Team:Bulgaria/experiments
Home Team
Project
Description Design Experiments Results Contribution Improve Second project Demonstrate Lab book Protocols
Human Practices
Silver Gold Survey Public Engagements   I, Engineer   Fission festival   International workshop Innovation corner   Sofia science festival   The European Researchers’ Night   Bloomberg TV Bulgaria   BNT1 TV Bulgaria   Agar Art - Bulgaria   BG Economics Magazine Project Yana Project Vyara
Achievements
Parts
Collaborations Safety Attributions
Experiments
Protocols
All used protocols were collected in a separate section of this Wiki. ( https://2017.igem.org/Team:Bulgaria/protocols )
Documentation of the development of our project
Our lab book is present in a separate section of this Wiki. ( https://2017.igem.org/Team:Bulgaria/lab-book )
Experiments
We initiated our experimental work with several optimization experiments for the TSS chemical competent cells preparation protocol (see it in the Protocols section). We found the optimal conditions for our major cloning strain (DH5alfa) achieving efficiencies up to 1.10^7 transformants for 1 ug pSB1C3 DNA.
Next we practiced the standart BioBrick assembly techniques – we built a number of different composite chromoprotein generators using the iGEM chromoprotein CDS collection plus all other needed elements. These constructs were used for the Bulgarian Agar Art Competition where our team won the first place.
After that we tested the site specific genomic integration with pOSIP vectors. We tried all 5 available integrases with the red color cassette from pSB1C3 as a test integration substrate. Unfortunately, despite all our efforts we were not able to find a successful integration event. After searching the literature for a possible explanation, the most logical one we found was the low transformation efficiency of our cells. Since we had no options to prepare or order better cells for that moment, we decided to stop our efforts with the clonetegration method and to look for some alternatives.
In the meantime the gRNA cloning cassette was synthesized by IDT as a gBlock fragment. It was cloned in pSB1K3 and many positive clones were identified via colony PCR. We selected the kanamycin standard vector pSB1K3 since the chloramphenicol resistance was already occupied by pdCas9 and the beta-lactamase gene in pSB1A3 contained an Eco31I target site. Two clones of gRNA-pSB1K3 were selected for the following experiments. Next we did a colony PCR with the primer pair VF2 + VR and both generated products were digested with Eco31I. The agarose gel electrophoresis showed that the Eco31I digestion functions as expected.
Then we took one of the gRNA vectors, purified a large amount from it and digested it with the Eco31I enzyme. Then we annealed the F and R oligos for each one of our selected gRNAs (see our oligo annealing protocol in the Protocols section) and cloned them into the vector. Positive clones for all these constructs were identified via colony PCR with the VR standard primer + the F oligo that corresponds to the given gRNA. Note: We do recommend the other iGEM teams to use not VR but a different primer on the vector backbone that will generate a product in size range 400-500 bp allowing easier detection of positive clones.
To validate the positive constructs we transformed them into cells with pCas9 plasmid. In case of working gRNA we observed no colonies on the petri dishes next morning – the combination of a wild type Cas9 + genome targeting gRNA without repair matrix is lethal for E. coli. This was simple, fast and efficient way to test our gRNA constructs.
Then we tested our dGas9 interference using gRNA for ftsZ. The corresponding gRNA vectors were transformed in our tunable CRISPRi strain and positive clones have been induced with L-arabinose and observed with a microscope to confirm the efficiency of our system.
Next thing was to assemble gRNAs for 3 selected genes – mutS, mutT and dnaQ (we selected them since these genes are mutated in the commercial mutator strain XL-1 red from Agilent) into one gRNA array. The assembly has been confirmed via colony PCR with primers VF2 and VR.
To finally prove our conception we transformed the tunable CRISPRi strain with these gRNA arrays, then selected positive clones were induced with L-arabinose and cells were plated on petri dishes with rifampicin, nalidixic acid or kanamycin.
Our Partners
