Team:Bulgaria/description
Home Team
Project
Description Design Experiments Results Contribution Improve Second project Demonstrate Lab book Protocols
Human Practices
Silver Gold Survey Public Engagements   I, Engineer   Fission festival   International workshop Innovation corner   Sofia science festival   The European Researchers’ Night   Bloomberg TV Bulgaria   BNT1 TV Bulgaria   Agar Art - Bulgaria   BG Economics Magazine Project Yana Project Vyara
Achievements
Parts
Collaborations Safety Attributions
Description
ACE of BASE - Accelerated Crispr-based EvolutiOn For BActerial Selection
Abstract
Directed evolution has been established as an effective strategy for improving or altering the function of biomolecules for industrial and research applications. One of the major pre-requirements for a successful directed evolution of proteins is a simple, safe, fast and cost-efficient method for generating genetic diversity. Many technologies have been used during the years to propagate mutations in different experimental setups. One of them is mutator strains of E. coli, which carry defects in one or more of their DNA repair genes. A large number of gene knock-out, transcriptional or translational silencing methods were applied for mutator strain generation. Regardless of its great potential, the CRISPR guided dCas9 targeting to transcriptional start sites of bacterial DNA repair genes was not among them. Our project is focused on adapting this promising system to manipulate the mutation levels in E. coli in an attempt to create a novel and efficient mutator strain with controllable mutation levels and high transformation efficiency.
Project description
Directed evolution of proteins is a valuable tool for the modern synthetic biology.[1] The evolutionary cycle is composed by three main stages - mutation, selection, and amplification. All of them can be imitated in the laboratory using proteins with industrial potential or scientific importance as starting templates. Random mutagenesis is a very useful technique that can be used to increase the genetic diversity of different sequences during the first stage of the evolutionary cycle.[2] One way to randomly mutagenize gene sequences cloned in plasmid vectors is through propagating these vectors in a mutator E. coli strain. This strain carries defects in one or more of its DNA repair genes and the plasmid propagation will produce randomly mutagenized plasmid libraries. Mutator strains offer a very simple, fast and economic way of introducing random point mutations throughout the sequence of interest with a reasonably high rate. Moreover, the number of sub-cloning steps is significantly greenuced compagreen to alternative methods like error-prone PCR and the final library is composed by supercoiled plasmids that can be retransformed into a different strain with a very high efficiency. Important note on the usage of mutator strains is the essential need to restore the natural low mutation rate of E. coli immediately after the mutation stage to prevent the accumulation of undesirable spontaneous mutations. This can be achieved by the use of an inducible system for transcriptional and/or translational repression in E. coli that targets different DNA repair genes. Very promising candidate for this role is the CRISPR guided dCas9 targeting to transcriptional start sites of bacterial genes.[3] According to our knowledge, no one has used it before for mutator strain generation. Nevertheless, this technology has been proven as an efficient tool for gene repression in E. coli in a number of publications.[3, 4] Additionally, it allows simple and efficient multiplexing, so repression of multiple DNA repair genes will not be an issue.[4] Moreover, the sequence-specific targeting compound is a group of fairly short gRNA molecules that can be easily re-cloned if one wants to change the repressed genes. The CRISPR guided dCas9 transcriptional silencing in bacteria can be easily controlled by using inducible promoters for expressing the gRNA molecules. This will allow fine control over our system and an increase in the mutation rate only upon a signal. All these properties make the CRISPR guided dCas9 targeting to transcriptional start sites a promising tool for a mutator strain generation.
Applications and Implications
CRISPR-dCas9 based mutator strain will be useful in all kinds of directed evolution experiments. Major advantages compagreen to other methods will be:                 • Low cost – our system can be adapted to any general cloning strain of E. coli. Cultivation media and arabinose is everything needed to use it.                 • Simple handling – no need of complicated, expensive or time consuming steps for random mutagenesis. • Safe – no use of chemical mutagens.                 • Supercoiled plasmids – mutagenized vectors can be isolated as supercoiled plasmids. This will increase re-transformation efficiency up to 2 fold eliminating the need of very expensive ultracompetent cells.                 • High transformation efficiency – due to the used E. coli strains, our system will have much higher transformation efficiency compagreen to the commercially available mutator strain XL1-green.
Why did we select ACE OF BACE
We decided to work on this particular project because we were fascinated by the CRISPR-Cas9 technology and its applications.  During the initial discussions we decided to create a new CRISPR based tool. This was motivated by the fact that the whole process is simple enough, not very expensive and we had all the equipment needed (plus an instructor with some CRISPR-Cas9 experience in eukaryotic systems). Moreover, the CRISPR-Cas9 system is popular enough even among the regular people in Bulgaria thanks to the different mediums. This gave us a chance to better explain our project to the society and to rise the interest in iGEM.                 From that point our main goal was to use the Cas9 and dCas9 in a new way to obtain a working device. We selected to focus on a mutator strain since it sounded like a great tool with a lot of applications. Moreover, our initial analysis showed that the final mutator strain (once it is made) will be very cheap to maintain, safe and with low requirements for specialized equipment for usage. This was very important for the final project selection since we wanted this strain to be used in both our universities as an educational tool. To ensure that we got a conversation with the Department of Genetics in University of Sofia “St. Kliment Ohridski”. The people there were open-minded and promised us that in case our mutator strain is really working as expected it will be integrated in different practical exercises.
References :
1. Packer, M.S. and D.R. Liu, Methods for the directed evolution of proteins. Nat Rev Genet, 2015. 16(7): p. 379-94.
2. Labrou, N.E., Random mutagenesis methods for in vitro directed enzyme evolution. Curr Protein Pept Sci, 2010. 11(1): p. 91-100.
3. Qi, L.S., et al., Repurposing CRISPR as an RNA-guided platform for sequence-specific control of gene expression. Cell, 2013. 152(5): p. 1173-83.
4. Cress, B.F., et al., CRISPathBrick: Modular Combinatorial Assembly of Type II-A CRISPR Arrays for dCas9-Mediated Multiplex Transcriptional Repression in E. coli. ACS Synth Biol, 2015. 4(9): p. 987-1000.
5. Horst, J.P., T.H. Wu, and M.G. Marinus, Escherichia coli mutator genes. Trends Microbiol, 1999. 7(1): p. 29-36. 6. Durfee, T., et al., The complete genome sequence of Escherichia coli DH10B: insights into the biology of a laboratory workhorse. J Bacteriol, 2008. 190(7): p. 2597-606.
7. St-Pierre, F., et al., One-step cloning and chromosomal integration of DNA. ACS Synth Biol, 2013. 2(9): p. 537-41.
8. Nakashima, N. and T. Tamura, Conditional gene silencing of multiple genes with antisense RNAs and generation of a mutator strain of Escherichia coli. Nucleic Acids Res, 2009. 37(15): p. e103.
Our Partners
