<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_Bulgaria_Design skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><DIV id="top_title"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:Bulgaria/Design</SPAN></H1></DIV><DIV id="HQ_page"><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><DIV class="container"><DIV class=" mynavbar"><A href="https://2017.igem.org/Team:Bulgaria">Home</A><A href="https://2017.igem.org/Team:Bulgaria/team">Team</A><DIV class="mydropdown"><BUTTON class=" mydropbtn">Project
                </BUTTON><DIV class=" mydropdown-content"><A href="https://2017.igem.org/Team:Bulgaria/description" style="border-bottom:1px solid black">Description</A><A href="https://2017.igem.org/Team:Bulgaria/design" style="border-bottom:1px solid black">Design</A><A href="https://2017.igem.org/Team:Bulgaria/experiments" style="border-bottom:1px solid black">Experiments</A><A href="https://2017.igem.org/Team:Bulgaria/results" style="border-bottom:1px solid black">Results</A><A href="https://2017.igem.org/Team:Bulgaria/contribution" style="border-bottom:1px solid black">Contribution</A><A href="https://2017.igem.org/Team:Bulgaria/improve" style="border-bottom:1px solid black">Improve</A><A href="https://2017.igem.org/Team:Bulgaria/second-project" style="border-bottom:1px solid black">Second project</A><A href="https://2017.igem.org/Team:Bulgaria/Demonstrate" style="border-bottom:1px solid black">Demonstrate</A><A href="https://2017.igem.org/Team:Bulgaria/lab-book" style="border-bottom:1px solid black">Lab book</A><A href="https://2017.igem.org/Team:Bulgaria/protocols">Protocols</A></DIV></DIV><DIV class=" mydropdown"><BUTTON class=" mydropbtn">Human Practices
                </BUTTON><DIV class=" mydropdown-content"><A href="https://2017.igem.org/Team:Bulgaria/HP/Silver" style="border-bottom:1px solid black">Silver</A><A href="https://2017.igem.org/Team:Bulgaria/HP/Gold_Integrated" style="border-bottom:1px solid black">Gold</A><A href="https://2017.igem.org/Team:Bulgaria/survey" style="border-bottom:1px solid black">Survey</A><A href="https://2017.igem.org/Team:Bulgaria/public-engagement">Public Engagements</A><A href="https://2017.igem.org/Team:Bulgaria/i-engineer" style="border-top:1px solid black">  I, Engineer</A><A href="https://2017.igem.org/Team:Bulgaria/fission-festival">  Fission festival</A><A href="https://2017.igem.org/Team:Bulgaria/international-workshop-innovation-corner">  International workshop Innovation corner</A><A href="https://2017.igem.org/Team:Bulgaria/sofia-science-festival">  Sofia science festival</A><A href="https://2017.igem.org/Team:Bulgaria/researchers-night">  The European Researchers’ Night</A><A href="https://2017.igem.org/Team:Bulgaria/bloomberg-tv-bulgaria">  Bloomberg TV Bulgaria</A><A href="https://2017.igem.org/Team:Bulgaria/bnt1-tv-bulgaria">  BNT1 TV Bulgaria</A><A href="https://2017.igem.org/Team:Bulgaria/agar-art-bulgaria">  Agar Art - Bulgaria</A><A href="https://2017.igem.org/Team:Bulgaria/bg-science-magazine" style="border-bottom:1px solid black">  BG Economics Magazine</A><A href="https://2017.igem.org/Team:Bulgaria/project-yana" style="border-bottom:1px solid black">Project Yana</A><A href="https://2017.igem.org/Team:Bulgaria/project-vyara">Project Vyara</A></DIV></DIV><DIV class=" mydropdown"><BUTTON class=" mydropbtn">Achievements
                </BUTTON><DIV class=" mydropdown-content"><A href="https://2017.igem.org/Team:Bulgaria/parts">Parts</A></DIV></DIV><A href="https://2017.igem.org/Team:Bulgaria/Collaborations">Collaborations</A><A href="https://2017.igem.org/Team:Bulgaria/safety">Safety</A><A href="https://2017.igem.org/Team:Bulgaria/Attributions">Attributions</A></DIV><DIV class="row"><DIV class="col-md-12"><H1 style="color:#1AA55E; font-family: Comic sans MS;">  Design</H1><H3><B>Principles</B></H3><P>In order to create a mutator strain the normal function of the DNA repair systems in <I>E. coli</I> should be disrupted. Multiple approaches like gene knockouts, expression of dominant negative mutant repair proteins and antisense RNA expression have already been used for this purpose and described in the literature. We decided to try something new – the CRISPR-dCas9 interference. The catalytically inactive dCas9 mutant can target promoter regions and CDS in the <I>E. coli</I> genome leading to a transcription silencing. This approach was perfect for our needs since it was simple, fast and easy for multiplexing. We just had to select key genes involved in the fidelity of the DNA replication and to suppress their expression via CRISPR interference. The most important part was to easily control the level of suppression making the mutator phenotype inducible.</P><H3><B> Step by Step explanation and discussion of our design</B></H3><P><B>Selection of a CRISPRi system</B> – we selected two CRISPR-dCAs9 systems:

        -	<B>pdCas9 plasmid based system </B>– it consists of the pdCas9 vector (#44249 from Addgene) and a separate gRNA expression vector. The dCas9 protein is under the control of a Tet inducible promoter allowing us to control the level of suppression. This is very a classical combination with a lot of options for customization. It could be used in any E. coli strain (that has no chloramphenicol resistance).
            
        -	<B>pdCas9 chromosome integrated system</B> - our second system was the Jun lab tunable CRISPRi strain (#86400 from Addgene). This strain already contains dCas9 integrated into its genome under the control of a fine tuned arabinose inducible promoter.</P><P>
        Both systems had their advantages and disadvantages. The plasmid based system is not limited to a given cell type and can be used in strains with very high transformation efficiencies like DH10B and DH5alfa. Its limitations are the requirement of two plasmids (this limits the available origins of replication and selection markers for other plasmids). The Jun lab tunable CRISPRi strain has both components integrated into its genome so no other plasmids are required. Moreover the Ara promoter and the additional strain modifications allow very fine tuning of the repression levels. The negative side is that this strain has a MG1655 genetic background that cannot be changed.</P><P>
        We selected the Jun lab tunable CRISPRi strain as our proof of principle system due to its simplicity (one need to clone only the gRNAs). Next we planned to use the pdCas9 vector system and to integrate it in selected genomic locations to minimize its downsides.</P><P><B>Design of gRNA expression vectors</B> – in order to clone gRNAs into the Jun lab tunable CRISPRi strain one needs to follow a specific procedure for oligo integration into the genome. It was not convenient and fast enough for a routine check of the large number of gRNAs that we needed for this project. Moreover our other plasmid based system needed a separate gRNA expression vector so we decided to create one.   The major points taken into consideration after consulting with a number of articles were:    - Constitutive promoter with fixed transcription start site – we selected BBa_J23119.
              -	Simple cloning method for different 20 bp long gRNAs – we integrated a stuffer fragment with two Eco31I target sites. The stuffer sequence was selected in a way that the Eco31I sites are not blocked by methylation in E. coli. After restriction one can clone gRNAs using simple annealed oligo cloning procedure. Both ends generated by this enzyme in our vector are not compatible so no alkaline phosphatase treatment and/ oligo phosphorylation are required.
              -	The 42 bp long gRNA backbone that is present immediately after the varying gRNA region – it is a standard element for each gRNA.
              -	The 40 bp S. pyogenes terminator after the gRNA body - another standard element found in different gRNA vectors.
              -	BioBrick compatibility – this allows easy construction of gRNA arrays following the rules for a standard BioBrick assembly. Of course one should have in mind that the cloned gRNAs do not contain EcoRi, SpeI, XbaI or PstI target sites.
        </P><P><B>Selection of gRNAs for genes involved in DNA repair</B> – all genes were selected based on literature mining. Only genes with experimentally confirmed increase in the DNA mutation rates were selected. The full gene list consist of <I>mutM, mutY, mutS, mutT, ndk, uvrD, mutL</I> and <I>dnaQ</I>.
        </P><P>
        All gRNAs for these genes were designed following a number of simple rules:
        </P><P><I>Note: All these steps were taken directly from the protocol article “CRISPR interference (CRISPRi) for sequence-specific control of gene expression” (doi:10.1038/nprot.2013.132) and are also described in detail in the protocol section of this wiki.</I></P><P>
                 1) Determine the DNA sequence of genes to be targeted using available genome databases.
              2) Select the DNA elements or loci to be targeted.
              3) Determine the sequence of the base-pairing region on the sgRNA.
              4) Check the specificity of sgRNA binding in the genome using the Basic Local Alignment Search Tool.
             Use only sgRNAs without predicted off-targets.
             5) Generate the full length of sgRNA by appending the dCas9 handle and predict the secondary structure folding of the dCas9 handle hairpin once it is appended to the sgRNA. If it is properly folded, keep the sgRNA and proceed.
        </P><P><B>Test of the observed mutation rates upon induction </B> – to test for an increased mutation rates we planned to monitor for the number of sporadic E. coli mutants with resistance for given antibiotics. This approach has the advantages of speed, simplicity and low cost.
        </P><P><B>Genomic integration of the plasmid based dCas9 system </B> – we planned to use the clonetegration technique for genomic integration of dCas9 and the gRNA expression cassette. To do this we ordered the Endy and Shearwin Lab pOSIP Plasmid Kit (Addgene Kit #1000000035) which contains 6 different site specific integrases cloned in easy to use vectors with unique EcoRI and PstI target sites.</P><H3><B>Experimental plan to test our design </B></H3><P>We designed a number of experiments to check all the different points in our system.
</P><P>    1)	<B>the Jun lab tunable CRISPRi strain</B> – it will be tested via cultivation on media supplemented with Tetracycline and Spectinomycin.
            2)	<B>pdCas9 plasmid</B> – it will be tested via analytical restriction with EcoRI.
        3)	<B>gRNA expression vector</B> – it will be synthesized as gBlock fragment by IDT and cloned via restriction with EcoRI and PstI in pSB1K3. Positive clones will be selected via colony PCR with a specific primer for the gRNA backbone + the VR standard primer. The Eco31I restriction sites will be tested via cassette amplification with VF2 and VR followed by an analytical digestion with Eco31I.
        4)	<B>gRNA positive clones</B> – the gRNA positive clones will be selected via colony PCR with the F oligo from the gRNA itself + the VR primer.
         5)	<B>gRNA expression and activity</B> – they will be tested via transformation of the gRNA vectors in E. coli cells that contain a Cas9 plasmid (with constitutive expression). If gRNAs work as expected no colonies will be obtain (wild type Cas9 + gRNA that targets E. coli genome is a lethal combination without a repair matrix).
         6)	<B>gRNA arrays assembly</B> – it will be tested via colony PCR with primers VF2 and VR.
         7)	<B>dCas9 gene repression</B> – we will clone a gRNA for the ftsZ gene. In case of a successful knockdown all cells should become filamentous and the cell division will stop.
         Note: this phenotype was described in “tCRISPRi: tunable and reversible, one-step control of gene expression” doi:10.1038/srep39076
         8)	<B>Increased mutation rates</B> – cells with both dCas9 and gRNAs will be cultivated for a different time periods after dCas9 induction and then plated on solid media with different antibiotics (rifampicin, nalidixic acid and kanamycin). The number of sporadic resistant clones on the dishes will be used as indicator for the mutation rates.
        </P></DIV></DIV></DIV><FOOTER id="footer"><DIV id="images" class="partners"><DIV class="row"><DIV class="col-md-12 text-center"><H2 style="font-family: Times New Roman">
                        Our Partners
                    </H2></DIV></DIV></DIV></FOOTER></DIV></DIV></DIV></DIV></DIV></BODY></HTML>