"
Page
Discussion
View source
History
teams
Log in
 
Team:Slovenia/METHODS and PARTS/notebook
From 2010.igem.org
Fun fact:
notebook - split/FRET
split/FRET oscillator violacein carotenoids proteins zinc-finger binding test
week 1  week 11
week 2  week 12
week 3  week 13
week 4  week 14
week 5  week 15
week 6  week 16
week 7  week 17
week 8  week 18
week 9  week 19
week 10week 20
6/17/2010 – 6/20/2010
PCR of C-terminal and N-terminal fragments of fluorescent proteins (termed split fluorescent proteins). mCerulean (cyan fluorescent protein) and mCitrine (yellow fluorescent protein) were used as  templates. N-splits were PCRed with reverse primer carrying stop codon, C-splits with forward primer carrying start codon and reverse primer carrying linker sequence.
reverse for N-splits: 5'-TGTACTGCAGGCGGCCGCACTAGTTTAGATGTTGTGGCGGATCTTG-3' 
forward for C-splits: 5'-ACTAGAATTCGCGGCCGCTCTAGAatgGCCGACAAGCAGAAGAACG-3' 
reverse for C-splits: 5'-TGTACTGCAGGCGGCCGCACTAGTGCTTCCCCCACTCCCACCGCCAGAGCCACCCTTGTACAGCTCGTCCATGC-3' 
EcoRI NotI XbaI SpeI PstI linker
 
6/21/2010 – 6/27/2010
The annealed linker was cloned first on the 3' end of ZNF HIVC and Gli1 DNA binding domains in order to proceed cloning of N-splits. C-splits with start codon and linker sequence on 3' end were cloned directly in front of opened vectors carrying ZNF HIVC and Gli1 DNA binding domain (parts BBaK165007 and BBaK165006).
6/28/2010 – 7/4/2010
Modification of pET19b, commercial low-copy plasmid for high protein expression to conform with BioBrick standard. Cloning of split/FRET tests with functional BioBricks.
Finished N-split constructs.
7/5/2010 – 7/11/2010
Planing of reconstitution of zinc finger-split fluorescent proteins on DNA program. Based on Gli1 size and structural and data concerning Gli1 DNA binding mode,  we decided to construct Gli1  with N-splits fluorescent proteins and ZNF HIVC with C-splitsfluorescent proteins.  PCR of whole mCerulean and mCitrine was done. We selected and ordered additional synthetic zinc fingers based on in silico data mining using our own software and data from the literature.
7/12/2010 – 7/18/2010
Primers for pET19b mutagenesis arrived. We started with Quickchange mutation PCR on this production vector.
PCR of T7 promoter with and without His tag and T7 terminator using iGEM team Slovenia 2009 production vector was done. PCR of T7 terminator was succesful, but we had some problems with amplifying T7 promoter. Restriction analysis proved successful cloning of mCerulean and mCitrine into pSB1AK3 backbone.
7/19/2010 – 7/25/2010
A plan was set for the fastest way to assemble of BioBrick split/FRET construct: Cloning of (His) stop T7 terminator into pSB1AK3; Cloning of T7 promoter with RBS, ATG and/or His tag in front of above mentioned constructs. Constructs with cloned T7 promoter were sent for sequencing. We cloned T7 terminator into back opened plasmids carrying split GFP – zinc finger fusion constructs.
7/26/2010 – 8/1/2010
We got the sequencing results of full zinc finger-split constructs.  However, T7 promotor  was not correct, therefore the procedure was repeated. Cloning strategy for synthetic zinc finger fusion constructs was selected. N
8/2/2010 – 8/8/2010
6 constructs were transformed into BL21 cells for protein production: CFP, YFP, Gli1_link_nCFP, Gli1_link_nYFP, cCFP_link_HIVC and cYFP_link_HIVC. Three-point ligation for FRET positive control (CFP_link_YFP, YFP_link_CFP) was done. Production of whole GFPs was observed under UV light. C-splits were detected on a Western blot. This was our first zinc finger fusion protein for SPR studies. Violacein constructs from Registry Kit Plate were transformed.  Primers for cloning Gli1 fusions into original pET19b were designed.
8/9/2010 – 8/15/2010
New primers for T7 promoter  were ordered. Three-point ligations of split fluorescent proteins and ordered synthetic zinc fingers: Blues, Jazz, Zif268, PBSII, Tyr456 were set. Registry Kit Plates were checked for GFPs reporter. Cloning of split fluorescent protein zinc finger fusions after T7 terminator.
8/16/2010 – 8/22/2010
Cloning of FRET positive controls and split GFP zinc finger fusions in front of (His)_stop_T7T. Emission spectra of bacterial cell lysates of CFP and YFP were determined using lab fluorimeter and  compared to those of split GFP fusions.  Annealed primers for T7 promotor were cloned into pSB1AK3 for the purpose of biosynthesis part of the project. Checking of constructs for T7 promoter by Colony PCR.
8/23/2010 – 8/29/2010
Colony PCR for T7 promoter was repeated and one positive clone was sent for sequencing. Annealed T7 promotor was cloned in front of split GFP zinc finger fusions with T7 terminator. Opening of mutated pET19b was set up (XbaI, PstI). Colony PCR with primers annealing to T7 promoter and T7 terminator was successful. Restriction analyses latter on proved correct inserts.
8/30/2010 – 9/5/2010
6 constructs Gli1 link nCFP, cCFP link HIVC, PBSII link nYFP, cYFP link Zif268, Blues link nCFP, cCFP link Jazz were selected, cloned into modified pET19b for protein production and purification.  .
DNA program
TYR456, JAZZ, BLUES, Zif268, PBSII, ZNF_HIVC, Gli1
Link to the part
Production failed.
Sequencing results of cloning of T7 promoter in the vectors above indicated several mutations.  Based on these results we decided for a new strategy:
     -Use of  pBAD promoter in biosynthesis constructs
    - Use of CMV for split FRET constructs (testing of reconstitution of Split fluorescent proteins and FRET effect in vivo). In addition, we chose to use a different plasmid vector - pSB1AK8 designed by previous Slovenian iGEM team 2009. pSB1C3 plasmid with a random insert was isolated and prepared  for cloning of all the BioBrick constructs for final part submission. 
9/6/2010 – 9/12/2010
Cloning of constructs for in vivo test of Split/Fret reconstitution into the CMV + pSB1AK8.  Emmision spectra of positive control for FRET was acquired by a fluorimeter.
Cloning of  ATG in front of N-split constructs and correct insertion confirmation by SacI restriction analyses with. In vitro split GFP test was performed with lysates from modified pET19b protein production vector but turned out negative.
9/13/2010 – 9/19/2010
pSB1C3 was cut with EcoRI and PstI in preparation for cloning of  the constructs for part submission. DNA Programs for assembly of zinc finger fusion proteins for biosynthesis of Carotenoids and Violacen arrived and were (ordered with EcoRI, SpeI overhangs)  cloned into pSB1C3. Screening of Gli1 link nCFP positive colony was peformed by running lysates of several bacterial cultures on SDS-PAGE. The clone with greatest production was picked out. Cloning of the DNA program for split/FRET tests directly into desired submission vector - pSB1C3.
9/20/2010 – 9/26/2010
In vivo reconstitution of Split-fluorescent proteins by confocal microscopy.There experiments (in vivo split GFP and FRET)  were done using 8-well microscopic slides. Plasmids (split GFPs and DNA program) were transfected into HEK293 cells and then fixed with paraformaldehyde, before the cells were observed under confocal microscope. FRET was measured with LAS AF software provided by the manufacturer (Laica). mCerulean was excited with 458 nm laser and emmision acquired between 470 and 500 nm. mCitrine was excited with 514 nm laser and emmision aquired between 525 and 580 nm. Reconstitution of  split GFPs was successful. The single split GFP bound to a DNA program did not give any signal.
In addition,  electrophoretic mobility shift assay (EMSA) of fusion zinc finger proteins was done on agarose gel.
Blunt end ligation of a DNA program to potentially ligate more program DNA molecules was performed and the constructs were cloned in pBluescript vector.
Various buffer conditions were tested for EMSA experiments.
9/27/2010 – 10/3/2010
Experiments of split GFP reconstitution experiment in 96 well plate with purified ZNF-split fluorescence proteins in various buffer conditions were not successful.  We decided to use lysates of HEK 293 cells transfected with split-fusion proteins for in vitro reconstitution of split proteins in vitro.
Alexa 546 primers with ZNF binding sites and different spacers was ordered in order to visualize exact position of split GFPs reconstituted in vivo.
Optimization of in vivo SPLIT/FRET experiments in 8-well microscopic slides. SPR studies were performed  showing binding modes of all zinc finger fusions inculed in the project.
10/4/2010 – 10/10/2010
Optimization of in vivo SPLIT/FRET experiments in 8-well microscopic slides.
Analysis of dependence of number of split YFP reconstituted cells on the amount of DNA program present in transfected cells.
Test of the distance between specific DNA binding sites on a DNA program that allow reconstitution of split fluorescent proteins. By transfecting cells with split GFPs having two zinc figer operators between their binding sites on a DNA program, we didn't observe any signal as predicted.
New DNA programs for assembly of biosynthesis pathways were ordered, this time with all BioBrick restriction sites.
In vitro reconstitution of SPLIT/FRET on program DNA
Cloning of parts to pSB1C3 backbone has stared.
10/11/2010 – 10/17/2010
Reconstitution of split GFP.  Cells cotransfected with split GFP systems seperately were lysed and incubated with 50 µl of DNA program at 4°C overnight. Fluorescence spectra that were acquired indicated successful reconstitution of split fluorescent proteins in the presence of program DNA.
10/18/2010 – 10/24/2010
Preparing parts for submission and testing of various in vitro conditions to improve split GFP reconstitution. Cloning of DNA programs for biosynthesis from pBluescript to PstI, HindIII cut pSB1C3 carrying PBSII zinc finger.
10/25/2010 – 10/27/10
HEK293tic last three days. It's Jamboree time
Retrieved from " http://2010.igem.org/Team:Slovenia/METHODS_and_PARTS/notebook "
Recent changes
What links here
Related changes
Special pages
My preferences
Printable version
Permanent link
Privacy policy
Disclaimers
