"
Page
Discussion
View source
History
teams
Log in
 
Team:Slovenia/PROJECT/biosynthesis
From 2010.igem.org
Chuck Norris facts:
biosynthetic pathways: background
Contents
1 Introduction
2 Scaffold-assisted biosynthetic pathway
3 Comparison od advantages and disadvantages between DNA- and protein-based biosynthetic scaffold
4 Selection of the biosynthetic pathways
5 Design of DNA-guided biosynthetic pathway
6 Estimation of the yield enhancement by DNA-guided biosynthesis
Introduction
Biosynthesis is an enzyme catalyzed process, occurring in living cells, by which simple substrate molecules are converted into more complex products. The process often consists of several steps, in which the product of one step is used as a substrate for the following step. In synthetic biology, research and engineering of biosynthetic pathways gains more and more attention every year. One of the first great stories in the field of synthetic biology was engineering the artificial biosynthetic pathway for antimalarial drug artemisinic acid production in Saccharomyces cerevisae yeast. Production of artemisinic acid in genetically modified yeasts was achieved modulating regulation of specific mevalonate pathway genes and by the introduction of genes for the biosynthetic pathway from Artemisia annua plant to yeast. Another great examples of biosynthetic pathway engineering are production of fatty esters (biodiesel), fatty alcohols, and waxes by genetically modified Escherichia coli. Genes from different organisms were combined into completely new pathway which was introduced to Escherichia coli  to produce useful fuel directly from plant biomass.
For industrial applications biosynthetic pathways composed of several enzymes should be engineered in a way to achieve high yield of the desired biosynthetic products. Various strategies for optimization have been undertaken so far, such as:
Increasing the pool of available substrate and/or overexpression of the enzymes of the limiting biosynthetic steps,
Introducing heterologous enzymes with preferred kinetic characteristics,
Blocking branching of biosynthetic pathway,
Compartmentalizing of biosynthetic pathways by directing enzymes of a particular biosynthetic pathway to a specific cell compartments or artificially made compartments (e.g. metabolosomes),
Increasing the proximity of enzymes by assembling metabolic pathways on a protein scaffold.
Scaffold-assisted biosynthetic pathway
The last approach is in some way similar to solutions that nature has already evolved. In some natural biosynthetic pathways enzymes form larger complexes, which results in faster transport of intermediates from one enzyme to another. This strategy enables an organism to produce higher amounts of the final product with lower metabolic burden. Unstable and toxic intermediates can be protected from decay or can be neutralized, since they are immediately used by the next enzyme in the pathway. Experimental data has already shown on a case of resveratrol biosynthesis (Zhang et al., 2006) where fusing two enzymes together improves the efficiency of this particular biosynthetic pathway. However, fusing more than two enzymes may prove to be difficult on account of maintaining the functionality of the fusion protein, which is the reason that scaffolding is a better approach. A protein scaffold has already been used to improve the mevalonate pathway yield (Dueber et al. 2009). While there are many advantages of protein-based scaffold, this approach has some disadvantages. Three dimensional arrangement of polypeptides is unpredictable due to the flexibility of and between dimerization domains as represented in the scheme. Additionally each protein dimerization domain has different conditions under which it folds and forms the functional interaction. And perhaps the most important argument, there is a limited number of available weel-behaved protein dimerization domains available, while the biosynthetic pathways may comprise ten or more enzymes and synthetic bioengineers will want to engineer several pathways simultaneously.
Figure 1: Schematic representation of advantages of DNA-guided biosynthetic scaffold. DNA imparts the linear order, while protein/polypeptide based scaffold predominantly clusters the biosynthetic enzymes without of particular order.
Keeping in mind all potential weaknesses of protein-based scaffolding, we came to an idea that DNA molecule could also be used as a scaffold for bringing biosynthetic pathway enzymes in the correct order. We further discussed the idea and fond out that DNA molecule in fact has several special advantages (see Table).
DNA molecule has highly predictable structure. Therefore scaffold based on it provides not only higher local concentration of biosynthetic pathway enzymes, but can also arrange them into a predefined order,
Because double helix of DNA molecule makes a turn every 10 nucleotides or so, the proper spatial orientation of bounded proteins can be achieved by varying the number of spacer nucleotides between the two binding sites for DNA-binding proteins,
A large number of different DNA binding proteins exists in nature. Some of them, such as zinc fingers or TAL effectors, have modular structure and can be engineered to bind any given nucleotide sequence. There are already more than 700 experimentally tested zinc fingers available to choose from. Therefore DNA scaffolding offers virually unlimited number of distinct combinations.
Comparison od advantages and disadvantages between DNA- and protein-based biosynthetic scaffold
 
DNA scaffold 
Protein scaffold 
Spatial orientation
Linear
Bundled
Order
Highly predictable order
Unpredictable
Scaffold : enzyme ratio
Difficult to achieve substantial amount of scaffold, ratio in favour of enzymes
Easy to achieve favorable ratio with gene expression regulation.
Scaffold: enzyme interactions
Similar well characterized, predictable interactions 
Variations in strength, limited number of well-characterized interactions
Variability, number of available elements
Large number of zinc finger and other DNA binding domains is readily available, engineered zinc finger domains
Limited number of protein dimerization domains
Interference with cellular metabolism
May bind to chromatin, selecti rom sequences that do not affect growh
Signal transduction domains usually do not interfere with bacteria
Selection of the biosynthetic pathways
Registry of standard  biological parts comprises several biosynthetic pathways. Two of the more characterised pathways are violacein and carotenoid biosynthetic pathway. Violacein pathway was chosen because the product  has many potential medical applications . Additionaly, violacein pathway is very convenient  choice due to the colored products. The carotenoid pathway  was selected because of a increasing significance of these compounds in food  and health industr and similar to violacein its products are colored, facilitating assays. By selecting different carotenoid pathways we wanted to point  out the possible application of our system at an industrial level. We  were surprised how easy it was find appropriate genes in a  Registry of standard biological parts and it was a the testimony of the usefulness of the  Registry as a practical and valuable tool in the field of  synthetic biology.
Design of DNA-guided biosynthetic pathway
After the biosynthetic pathways were chosen we fused all enzymes to DNA binding proteins. We decided to use zinc fingers as the DNA binding proteins, since there is a large number of experimentally available zinc fingers available.We introduced the obtained chimeric biosynthetic pathway enzymes into E. coli and tested if enzymes fused with zinc fingers are still functional, which was the case in all tested enzymes. We designed scaffold DNA molecule we called DNA program. Zinc finger binding sequences were arranged on a program DNA in a way that they enabled arrangement of chimeric biosynthetic pathway enzymes in the correct order and proper spatial orientation. We also designed the scrambled variant of DNA program, where all zinc finger binding sequences were still present but were not arranged in a correct order. We introduced program DNA, scrambled DNA and random DNA molecules into cells containing all chimeric biosynthetic pathway enzymes and determined the kinetics, productivity and yield of biosynthetic pathway products. To our great pleasure strategy proved to be successful. Yield of the product was significantly higher in cells that contained program DNA than in cells that contained scrambled one or no program DNA at all.
Estimation of the yield enhancement by DNA-guided biosynthesis
We wanted to estimate the acceleration of the biosynthetic pathway flux achieved by the ordered assembly of biosynthetic enzymes. In reality the overall reaction rate depends on many different factors. Each of them can be the rate limiting step. Diffusion is often the rate limiting step for biosynthesis of small molecules. Under this approximation we found that the reaction rate can be significantly accelerated if biosynthetic pathway enzymes are arranged on the DNA program. Acceleration of overall reaction kinetics can be under those approximations in direct correlation with the number of biosynthetic steps. The rationale is that the local concentration of the substrate is several orders of magnitude greater when enzymes are assembled on biosynthetic chains in comparison to soluble monomeric enzymes, which are typically present in the cell at submilimolar concentrations.
A simple way to perform an in silico simulation of the dynamics of quasi-random distributed enzymes in a cell, is to evaluate a few enzymatic reaction steps of the Michaelis-Menten kinetic law. To provide a more relevant view of enzyme and substrate dynamics, we designed a stochastic model that incorporates the mentioned law as the main kinetic constraint. To perform a valid representation of simulation’s results, we also included the Poisson distributed time delay as an essential part of the product (substrate) accumulation, accounting for the diffusion delay. The time delay variable was assumed to be inversely proportional to the average distance and enzyme/substrate local concentration. The following figure shows the simulated difference in product formation and corresponding reaction velocity for a three reaction biosynthetic pathway, assuming equal binding affinity and diffusion-limited reaction rate.
Figure 2: The comparison of a simulated product formation for a three reaction biosynthetic pathway. The "scaffold" graph shows a fast approaching of the steady state, where all the substrates are consumed and the accumulation of the final biosynthetic product reaches the maximal concentration. The graph corresponding to the contrarily, reaches the steady state much later.
Retrieved from " http://2010.igem.org/Team:Slovenia/PROJECT/biosynthesis "
Recent changes
What links here
Related changes
Special pages
My preferences
Printable version
Permanent link
Privacy policy
Disclaimers
