"
Page
Discussion
View source
History
teams
Log in
 
Team:Slovenia/PROJECT/introduction
From 2010.igem.org
Chuck Norris facts:
introduction
Contents
1 DNA as a program
2 Outline of the idea
3 Expansion of the number of tools for synthetic biology
4 Applications of DNA-guided programming
5 Conclusions
DNA as a program
As DNA molecule is so extensively characterized, being sort of a pop culture molecule, finding a completely novel way of using it presents quite a challenge. DNA is the basic linear carrier of information and we thought that it could represent an ideal scaffold for information processing. Instead of seeing DNA in a traditional way, where triplet codons guide the ribosome to incorporate a defined sequence of amino acids into a newly formed protein, we envisioned a completely new way of coding the information using DNA scaffold. Instead of triplets, binding sites for different DNA-binding proteins could be used as functional units. For example, we can select blocks of nine nucleotides that act as the sites for binding different DNA-binding proteins.
Figure 1: Schematic overview of the idea of DNA as a program.
Outline of the idea
How does this idea work? DNA sequence represents a program consisting of a series of blocks which determine the arrangement of DNA binding factors along the DNA. The main twist comes with the requirement that each of those DNA-binding proteins is fused to a different functional protein. Therefore, the sequence of target motifs encoded by DNA program defines also the arrangement of those functional proteins along with the order of DNA-binding factors. What is so powerful about this idea is the fact that by only changing the sequence of a DNA program, either switching positions or adding new target sequences, the outcome can be predicted in advance.
Expansion of the number of tools for synthetic biology
While there are 64 nucleotide triplets that in the nature code for 20 aminoacids, there could be as many as 262,144 different motifs consisting of nine nucleotides. Zinc-fingers and TAL elements can be designed to bind to almost any desired nucleotide sequence ranging from 9 to as many as 18 nucleotides. While this is sometimes a laborious process we don’t really need to do this. We can reverse the problem and select the target nucleotide sequence for each available DNA-binding factor. There are more than 700 experimentally determined zinc fingers in the database ZIFDB, offering a huge choice of tools for synthetic biology. Moreover zinc fingers have similar properties, such as binding affinity or stability, which is important since we don’t need to adjust properties of each separate part. DNA program therefore represents a modular approach for various synthetic biology applications. This could be an important step towards moving the synthetic biology from being a craft of tinkering peculiar biological elements towards becoming a truly engineering science. At the moment there is only one experimentally verified repressilator composed of three specific bacterial transcription factors, meaning a bacterial device that can count to three. Using the available natural elements those devices can never be extended above 5 or maximally 10 naturally available elements. Our approach therefore represents a completely new perspective of synthetic biology.
Applications of DNA-guided programming
What could be the specific implementations of this idea?
The first big application is to use the DNA program to arrange the sequence of biosynthetic reaction enzymes and therefore guide the biosynthetic flow towards the desired products. We decided to test this idea on a previously characterized biosynthetic pathway, demonstrating the advantages of DNA programming. One important advantage is to increase the reaction rate of the overall biosynthetic pathway but it can also guide the biosynthetic flow towards one or other product and prevent side reactions. We have demonstrated both effects on the biosynthesis of violacein, a five step reaction, which can, by the addition of enzymes, yield three different products. This approach offers several potential advantages in comparison to protein-based scaffolds.
Figure 2: Enhancing and directing biosynthetic flow using DNA program.
We can also envision the use of this approach to perform information processing that offers many advantages over traditional genetic switches, as the flow of information can be restricted to the reactions occurring along the linear DNA instead of throughout the cell. Attachment of the cascade of proteases, phosphatases or other enzymes/protein interaction domains to the DNA scaffold could result in a rapid information processing depending on the input of particular DNA sequence and initiation reaction.
Another application of the same principle is to use DNA-binding factors to extend and simplify the construction of oscillators. Modeling two different types of oscillators indicates that the use of synthetic DNA-binding proteins can provide tools to prepare orthogonal oscillators for both prokaryotic and eukaryotic cells, add new features to increase the robustness and independence of stochastic initial conditions and increase the selection of their frequency. For this purpose the basic type of building block was demonstrated to function in mammalian cells.
Conclusions
It can be said that our project has shuffled the way DNA is seen and regarded in the field of synthetic biology, meaning this view will never be the same again. To conclude, the perspective of sequential ordering of functional fusion proteins on a DNA scaffold on demand, could soon lead into novel computer-biotech hybrid science, changing fundamental paradigms of both fields, employing engineering principles of synthetic biology in highly predictive fashion using DNA program.
Retrieved from " http://2010.igem.org/Team:Slovenia/PROJECT/introduction "
Recent changes
What links here
Related changes
Special pages
My preferences
Printable version
Permanent link
Privacy policy
Disclaimers
