<HTML xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en" dir="ltr">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki  ltr ns-0 ns-subject page-Team_York_UK_Project_html"><DIV id="globalWrapper"><DIV id="top-section"><DIV id="p-logo"><A href="/Main_Page" title="Main Page">&quot;
	    </A></DIV><DIV id="menubar" class="left-menu noprint"><UL><LI class="selected"><A href="/Team:York_UK/Project.html">Page               </A></LI><LI class="new"><A href="/wiki/index.php?title=Talk:Team:York_UK/Project.html&amp;action=edit&amp;redlink=1">Discussion               </A></LI><LI><A href="/wiki/index.php?title=Team:York_UK/Project.html&amp;action=edit">View source               </A></LI><LI><A href="/wiki/index.php?title=Team:York_UK/Project.html&amp;action=history">History               </A></LI><LI style="color:white;cursor:default">teams</LI></UL></DIV><DIV class="right-menu noprint" id="menubar"><UL><LI id="pt-login"><A href="/wiki/index.php?title=Special:UserLogin&amp;returnto=Team:York_UK/Project.html" title="You are encouraged to log in; however, it is not mandatory [o]" accesskey="o">Log in</A></LI></UL></DIV><DIV id="search-controls" class="noprint"><FORM action="/Special:Search" id="searchform"> </FORM></DIV></DIV><DIV id="content"><H1 class="firstHeading">Team:York UK/Project.html</H1><DIV id="bodyContent"><H3 id="siteSub" class="noprint">From 2013.igem.org</H3><P><TITLE>iGEM York</TITLE></P><DIV id="header"><DIV id="navigation"><UL><LI><A href="https://2013.igem.org/Team:York_UK">Home</A></LI><LI><A href="https://2013.igem.org/Team:York_UK/Team.html">Team</A></LI><LI><A href="https://2013.igem.org/Team:York_UK/Project.html">Project</A></LI><LI><A href="https://2013.igem.org/Team:York_UK/Notebook.html">Notebook</A></LI><LI><A href="https://2013.igem.org/Team:York_UK/Outreach.html">Outreach</A></LI><LI id="acknowledgements"><A href="https://2013.igem.org/Team:York_UK/Acknowledgments.html">Acknowledgments</A></LI></UL></DIV><DIV id="nav2"><DIV id="team"><A href="https://2013.igem.org/Team:York_UK/Team.html?page=contacts">Contacts</A><A href="https://2013.igem.org/Team:York_UK/Team.html?page=university">The University</A><A href="https://2013.igem.org/Team:York_UK/Team.html?page=biography">Biography</A><A href="https://igem.org/Team.cgi?year=2013&amp;team_name=York_UK">Profile</A><A href="https://2013.igem.org/Team:York_UK/Team.html?page=overview">Overview</A></DIV><DIV id="project"><A href="https://2013.igem.org/Team:York_UK/Project.html?page=safety">Safety</A><A href="https://2013.igem.org/Team:York_UK/Project.html?page=gallery">Gallery</A><A href="https://2013.igem.org/Team:York_UK/Project.html?page=collaborations">Collaborations</A><A href="https://2013.igem.org/Team:York_UK/Project.html?page=achievements">Achievements</A><A href="https://2013.igem.org/Team:York_UK/Project.html?page=microbial">Microbial Fuel Cells</A><A href="https://2013.igem.org/Team:York_UK/Project.html?page=results">Results</A><A href="https://2013.igem.org/Team:York_UK/Project.html?page=parts">Parts</A><A href="https://2013.igem.org/Team:York_UK/Project.html?page=overview">Overview</A></DIV><DIV id="notebook"><A href="https://2013.igem.org/Team:York_UK/Notebook.html?page=modelling">Modelling</A><A href="https://2013.igem.org/Team:York_UK/Notebook.html?page=protocols">Protocols</A><A href="https://2013.igem.org/Team:York_UK/Notebook.html?page=labbook">Kobchai's Lab Book</A><A href="https://2013.igem.org/Team:York_UK/Notebook.html?page=wetlab">Jonas' Lab Book</A><A href="https://2013.igem.org/Team:York_UK/Notebook.html?page=drylab">Dry Lab</A></DIV><DIV id="outreach"><A href="https://2013.igem.org/Team:York_UK/Outreach.html?page=london">YSB 1.0</A><A href="https://2013.igem.org/Team:York_UK/Outreach.html?page=alumni">Alumni Event</A><A href="https://2013.igem.org/Team:York_UK/Outreach.html?page=lawrence">St. Lawrence's School</A><A href="https://2013.igem.org/Team:York_UK/Outreach.html?page=helen">St. Helen's Church</A></DIV><DIV id="acknowledgments"><A href="https://2013.igem.org/Team:York_UK/Acknowledgments.html?page=sponsors">Sponsors</A><A href="https://2013.igem.org/Team:York_UK/Acknowledgments.html?page=references">References</A><A href="https://2013.igem.org/Team:York_UK/Acknowledgments.html?page=attributions">Attributions</A></DIV></DIV></DIV><TITLE>iGEM York Team</TITLE><DIV id="container"><DIV id="navlocal"><UL><LI>Project</LI><LI><DIV id="overview"><A href="">Overview</A></DIV></LI><LI><DIV id="parts"><A href="">Parts</A></DIV></LI><LI><DIV id="gold"><A href="">Gold sensing</A></DIV></LI><LI><DIV id="goldpeptides"><A href="">Gold scavenging peptides</A></DIV></LI><LI><DIV id="electricity"><A href="">Electricity production</A></DIV></LI><LI><DIV id="hybrid"><A href="">The hybrid bistable switch</A></DIV></LI></UL><LI><DIV id="results"><A href="">Results</A></DIV></LI><LI><DIV id="microbial"><A href="">Microbial Fuel Cell</A></DIV></LI><LI><DIV id="achievements"><A href="">Achievements</A></DIV></LI><LI><DIV id="collaborations"><A href="">Collaborations</A></DIV></LI><LI><DIV id="gallery"><A href="">Gallery</A></DIV></LI><LI><DIV id="safety"><A href="">Safety</A></DIV></LI></DIV><DIV id="conlocal"><DIV id="overviewcontent" class="hiddenContent"><H2>Overview</H2><P>
Over the past decades, mankind has sought for a  new source of energy. It must be green, clean and 100% renewable. Microbial fuel cells (MFC) come to us as a plausible solution. The concept of tiny organisms converting waste materials into electricity has captured attention of scientists and publics around the world. However, MFC is inefficient and produces little electricity. In this project, we aim to improve MFC through decomposition of gold nanoparticles within the anodic chamber. What is more, our MFC must be bistable as well as tunable to work well under a wider range of conditions. 
</P><P>
We engineer the genetic circuit of <I>Escherichia coli</I>, such that the bacteria could sense gold and respond accordingly. Our new, engineered organism is called <B><I>Electricus aureus</I></B>. Our circuit is subdivided into three modules (Figure 1). The first is gold sensing. It is designed to recognize gold (HAuCl4) in its environment. GolS is a transcriptional activator of the promoter P(golTS). Upon binding with the gold, it dimerizes and binds to the recognition site within the promoter. The second is gold scavenging peptides. There are four variants of the fusion between MIDAS-2 and A3. We also include the N-terminal pelB tag for secretion to the periplasm and IM9 for higher stability. Note that the tag is likely to be removed during transportation process and potentially our peptides should be released to the extracellular space.  IM9 tag is relatively small (~9.5 kDa) and should not interfere with mineralization mediated by the peptides. Genes for the peptides are placed downstream of the gold sensing promoter, such that mineralization is conditional to concentrations of gold in the environment. The final module is hybrid bistable switch. 
</P><P>
The switch is an extended version of the quorum sensing from <I>Vibrio fischeri</I>. LuxI forms an indirect positive feedback loop through interactions with LuxR which is expressed by the constitutive promoter.  Upon gold induction, AiiA is expressed and degrade the quorum sensing signal AHL. In the other word, AiiA could disrupt the feedback loop. When gold is absent and population density is high, taR12 which is downstream of the Plux/lac is upregulated. In our project, taR12 is to transactivate the translation of mtrA upon binding to the crR12. Electricity is generated when all three mtr proteins are expressed and translocated to the right locations in cells. 
</P></DIV><DIV id="partscontent" class="hiddenContent"><H2>Parts</H2><P>All of the parts that we submitted to the igem registry:</P><TABLE border="1"><TBODY><TR><TH>Biobrick</TH><TH>Short name</TH><TH>Length</TH><TH>Characterized</TH></TR><TR><TD><A href="http://partsregistry.org/Part:BBa_K1127008">BBa_K1127008</A></TD><TD>PgolTS+LacZ+GolS</TD><TD>754 bp</TD><TD>Yes</TD></TR><TR><TD><A href="http://partsregistry.org/Part:BBa_K1127000">BBa_K1127000</A></TD><TD>pelB-A3-Flg</TD><TD>164 bp</TD><TD>No</TD></TR><TR><TD><A href="http://partsregistry.org/Part:BBa_K1127001">BBa_K1127001</A></TD><TD>pelB-A3-Flg-linker-IM9</TD><TD>437 bp</TD><TD>No</TD></TR><TR><TD><A href="http://partsregistry.org/Part:BBa_K1127002">BBa_K1127002</A></TD><TD>pelB-IM9-linker-A3-Flg</TD><TD>437 bp</TD><TD>No</TD></TR><TR><TD><A href="http://partsregistry.org/Part:BBa_K1127003">BBa_K1127003</A></TD><TD>pelB-MIDAS-2</TD><TD>137 bp</TD><TD>No</TD></TR><TR><TD><A href="http://partsregistry.org/Part:BBa_K1127006">BBa_K1127006</A></TD><TD>mtrABC</TD><TD>5168 bp</TD><TD>No</TD></TR><TR><TD><A href="http://partsregistry.org/Part:BBa_K1127016">BBa_K1127016</A></TD><TD>Gold and quorum sensing ultra complex</TD><TD>3708 bp</TD><TD>No</TD></TR><TR><TD><A href="http://partsregistry.org/Part:BBa_K1127018">BBa_K1127018</A></TD><TD>QS and gold sensing part for characterization</TD><TD>4674 bp</TD><TD>No</TD></TR></TBODY></TABLE><P>
For further explanations please refer to specific sections.
</P></DIV><DIV id="goldcontent" class="hiddenContent"><H2>Gold sensing</H2><P>
Some bacteria evolved to sense various toxic metals in the environment and produce certain responses. It involves transcription factors, different promoters and cascade activation reactions. One of the most characterised transcription factors belongs to MerR class. They are responsible for responding to stresses like antibiotics or heavy metals and the mechanism involves conserved N and C-terminal domains which contain DNA and metal binding motives (1, 2). <I>E. coli</I> has a MerR like transcriptional activator CueR, which allows the bacteria to sense copper ions and activate the transcription of specific genes. The same transcription factor can also allow the bacteria to sense the presence of gold ions in the environment (3). 
</P><P>
There are few problems involving CueR transcription activator for gold sensing. First of all; it is not entirely selective for gold ions and copper as well as silver ions can be involved in the activation of response genes. Secondly; it is a naturally present regulator of 2 genes in <I>E. coli</I> (4), therefore its use would involve some unsuspected behavior- we could not measure the exact response. The solution might involve mutating these 2 genes (which requires a lot of time) or finding alternative regulator system. 
</P><P><I>S. typhimurium</I> also contains a MerR like transcription activator GolS (6). It is responsible for expression of the gold detoxification system involving the transcription of golB (gold ion binding protein) from the promoter PgolB as well as golS (transcription of itself gene) and golT (gold ion transportation system) transcription from promoter PgolT/S (Fig 1a) (7). The deletion of these genes was shown to affect the bacteria in the presence of gold and increase mortality. Additionally, GolS is also able to control some other efflux systems, however it requires much higher levels than for the gol regulon and therefore would not be very sensitive for quantitative studies (9). 
By using this knowledge several attempts have been made to develop and characterise the gold sensing system from <I>S. typhimurium</I> (5, 8). Some of them involved PgolB and some PgolT/S promoters where the most common response molecule fluorescent proteins like RFP or GFP (Fig 1b). However, they faced some problems with basal expression levels of golS, which was causing some inconsistencies with their results.
</P><P>
Fig 1. Gol regulon found in <I>S. typhimurium</I> with all of the required components for detoxification system (a). The synthetic gol regulon with reporter molecule RFP under PgolB promoter (b).
</P><P>
We decided to use lacZα as a response molecule and make a gold sensing device containing PgolT/S-lacZα-golS in a standard iGEM plasmid. Also, the lacZα gene would have two restriction sites around it so it can be removed and exchanged with another response molecule or a response molecule that would transfer the signal to further downstream elements. The device would look like synthetic gol regulon Fig 1B, however the PgolB and RFP will not be present and between PgolT/S and golS the “replaceable” (SacII and SanDI restriction enzymes) lacZα will be present.
</P><P>
Two linear fragments were ordered as gBlocks containing PgolT/S-lacZα and golS with XbaI and SpeI restriction sites on the sides. However, later on we had to change the restriction sites by PCR, because the plasmids were only re-ligating during the cloning process and we were unable to transfer the genes and fuse these two parts together. Finally, a PgolT/S-lacZα-golS construct was made into pSB1C3. The activity was tested on LB plates containing X-gal as well as a β-galactosidase assay to best understand the response time and required gold ion concentration for induction.
</P><TABLE border="1"><TBODY><TR><TH>Biobrick</TH><TH>Short name</TH><TH>Length</TH><TH>Characterized</TH></TR><TR><TD>BBa_K1127008</TD><TD>PgolTS+LacZ+GolS</TD><TD>754 bp</TD><TD>Yes</TD></TR></TBODY></TABLE></DIV><DIV id="goldpeptidescontent" class="hiddenContent"><H2>Gold scavenging peptides</H2><P>
Gold nanoparticles (AuNP) differ from solid particles by their chemical and physical properties and have characteristics making them useful in: medicine as drug delivery agents (1), chemical industry as a catalyst (2) and various other areas (3). However, synthesis of AuNP with defined size range requires various toxic chemicals and usually traces of them are left and synthesized materials cannot be used for biological or other sensitive applications (4). 
</P><P>
Recently it was discovered that some bacteria like <I>Delftia acidovorans</I> is secreting small non-ribosomal peptides, which are able to reduce gold ions to solid gold. This forms a substrate to attach to and prevents the bacteria from gold ion induced toxicity (5). Interestingly, it was suggested that this species could be responsible for formation of small gold nuggets. However, commercial synthesis of delftibactin is too expensive due to modified side chains of the compound (Fig 1) as well as the formed gold particles are unstructured and can only be used for jewellery or electrochemical industry.
</P><P>
The alternatives like small peptide synthesis required prior knowledge, thus the phage display system was used and few AuNP binding dodacemer peptides like MIDAS-2, MIDAS-11 and A3 were generated (6, 7). Further experiments provided some evidences for nanoparticle formation. At first it was thought that not only MIDAS-2 and MIDAS-11 can reduce gold ions without external reducing agent, but also A3 and its fusion with Flg. AuNP almost showed no signs of aggregation in the presence of A3 peptide (Fig 2A) (8). In the latter paper there was lack of experimental controls and the experiments were carried only in one buffer. Even the conclusions were stated incorrectly predicting that A3 peptide directly interacts with gold ions, however no further evidences were found. Later on, it was showed that actually A3 requires external reducing agent HEPES buffer () and the peptide was described as nonreducing inhibitor, which caps the formed AuNP and prevents aggregation (9). Furthermore, modelling was used in order to understand the importance of each amino acid in A3 peptide for AuNP binding. Following simulations showed 4 regimes called diffusive, anchoring, crawling and binding. In addition specific amino acids responsible for each of them (except the diffusive regime, because it is thought to be random) were characterized (Fig 2B). Also, the comparison between other AuNP capping peptides revealed compositional similarities, which could be used later on for further designing of the active peptides (10). 
</P><P> 
Overall, all of the three mentioned peptides have low pI and are capable of stabilising AuNP by interacting with them and electrostatically repelling the other peptide covered nanoparticles at neutral pH (6, 8). This prevents them from aggregation; otherwise during the AuNP formation process random agglomeration would occur causing huge variations in sizes. Electrostatics could potentially be used to avoid interaction of nanoparticles with the bacterial plasma membrane, which was shown to be the key in toxicity process (13). 
</P><P>
Fig 1. The structure of delftibactin. Relatively complex molecule containing various ring structures as well as modified side chain and the only similarity with the peptides is amide bonds. 
</P><P>
Fig 2. The A3 peptide formed nanoparticles in HEPES buffer are dispersed and relatively small amounts of clumps are formed (A).  The simulation studies predicted the behaviour of the A3 peptide in the solution containing gold nanoparticles and showed the conformational changes during the binding process (B).
</P><P>
We decided to make various fusions of MIDAS-2 and A3 peptides (MIDAS-11 was left as alternative and not used) and express them only when the gold ions are present (incorporate as responsive element in the gold sensing part). Thus, in the presence of gold the transcription is induced of our synthetic operon controlled by promoter PgolT/S and production of the peptides begins. Additionally, N-therminal pelB secretion tag was introduced for the secretion to the periplasm. During the transportation process the pelB sequences is removed (14) and potentially our peptides should be released to the extracellular space.  This was shown to be the case for small antifreeze peptides (11) and we expect this to happen with our peptides. Due to small size the peptides might be degraded by various proteases in the cells, therefore we decided to fuse some of the constructs with IM9 tag. It is relatively small (~9.5 kDa), has similar pI to the peptides and was shown to stabilize the expressed proteins (15). 
Due to the short length sequences for the peptides were ordered as gBlocks – linear DNA fragments. These codon-optimized genes were cloned into pSB1C3 vector for submission. 
</P><H3>Biobricks</H3><TABLE border="1"><TBODY><TR><TH>Biobrick</TH><TH>Short name</TH><TH>Length</TH><TH>Characterized</TH></TR><TR><TD>BBa_K1127000</TD><TD>pelB-A3-Flg</TD><TD>164 bp</TD><TD>No</TD></TR><TR><TD>BBa_K1127001</TD><TD>pelB-A3-Flg-linker-IM9</TD><TD>437 bp</TD><TD>No</TD></TR><TR><TD>BBa_K1127002</TD><TD>pelB-IM9-linker-A3-Flg</TD><TD>437 bp</TD><TD>No</TD></TR><TR><TD>BBa_K1127003</TD><TD>pelB-MIDAS-2</TD><TD>137 bp</TD><TD>No</TD></TR></TBODY></TABLE><P>
For peptide activity testing we cloned each of the peptides under the constitutive promoter (BBa_J23100) and expressed in <I>E. coli</I> DH5 α in LB media overnight at +20 °C (better way would expressing in BL21 strain, however at that time we had no stocks of BL21). Later on the activity was tested upon mixing supernatant (cells removed by spinning at 5k rpm) with gold salt AuCl4 solution and HEPES buffer for A3 and its fusions. For supernatant containing MIDAS-2 the same conditions were applied except the HEPES buffer. The formed nanoparticles were detected by using the method called Nanoparticle Tracking Analysis (NTA) from NanoSight (12). The technique works by detecting the scattered light of moving nanoparticle and the motion is tracked. This produces data that is enough to determine the distribution of nanoparticle size.
</P><P>
Few words about nanosight:
“NanoSight’s “Nanoparticle Tracking Analysis” (NTA) technology detects and visualises nanoparticles in liquids down to 10nm. NanoSight are able to offer size distribution, concentration, zeta potential and fluorescence measurements, on a particle-by-particle basis.  This technology is being utilised in the development of drug delivery systems and viral vaccines, and in nanotoxicology. It also gives insight into the kinetics of protein aggregation and has a growing role in biodiagnostics, including the detection and speciation of exosomes and microvesicles. NanoSight has installed 600+ systems worldwide and its technology is validated by 700+ third party papers citing NanoSight results, consolidating NanoSight’s leading position in nanoparticle characterisation.”

More info can be found <A href="http://vimeo.com/62625839">here</A></P></DIV><DIV id="electricitycontent" class="hiddenContent"><H2>Electricity production</H2><P>
The increasing interest in electricity production by microbes is reflected by the conducted research amount and also publications not only in scientific journals, but also in more public ones (1, 2). The organisms that came to the discussion centre were Geobacter and Shewanella bacteria species, because of their unique cytochromes and ability to pump electrons through the membrane, which are generated during the respiration process. Some of the Geobacter species can easily colonize electrodes and start producing electricity, even from complex hydrocarbons containing aromatic rings. They also contain other characteristics required for good conductivity of the biofilm and it was characterized deeply in physiological and cellular level (3). But the most important feature for electron transfer in Geobacter species like <I>G. sulfurreducens</I> is heme group containing c type cytochromes (4). 
</P><P>
The problem is that these specific cytochromes are not well characterized and working with them might be problematic. Therefore, we decided to use c type cytochromes known as mtrCAB from <I>S. oneidensis</I>, which are homologous to omc cytochromes from <I>G. sulfurreducens</I>. These proteins were used before by several other teams in the iGEM competition and were shown to be active in <I>E. coli</I>. Additionally, some research groups managed to reconstitute the activity of these proteins in <I>E. coli</I> cells and described the activity by using metal reduction assays (5). 
</P><P>
The compartmentalization plays an important role in the electron transfer pathway as it provides location for various proteins. For example NapC protein that is naturally found in <I>E. coli</I> can reduce iron oxide and replace CymA from <I>S. oneidensis</I> (9) thus allowing transfer of electrons from cytosol to MtrA protein through the inner membrane. MtrA then acts as linker between MtrBC complex and NapC and transfers electrons further on. The MtrBC is then responsible for pumping electrons through the outer membrane. For the right folding of Mtr proteins cytochrome c maturation (ccm) (7) genes should be additionally expressed in <I>E. coli</I>, because under aerobic conditions the genes are not expressed (8) and usually during the introduction of mtr genes into <I>E. coli</I> additional copy of ccm is transferred too (5). This can be added on the additional plasmid (Fig 1). Another good reason for expression of mtr in <I>E. coli</I> K-12 is that it has ability to generate small amounts of current when grown on electrodes (6). 
</P><P>
Fig 1. The representation of plasmids on the left required for mtr protein activity testing. The pictures on the right indicate the position of mtr proteins in the cell membrane.
</P><P>
Additionally, Dr. James Moir offered us the plasmid containing ccm, so we decided to amplify and clone mtrBC with unchanged RBS originally from <I>S. oneidensis</I> genome and synthesize mtrA gene containing Plux-lac promoter and crr12 – RBS with complementary sequence for mRNA secondary structure formation. This is a double control mechanism for mtrA and single control mechanism for the mtrBC protein expression. However, due to non-sense mutations occurring during the PCR reaction with Phusion polymerase, we managed to clone only mtrABC construct, which contains no promoter and RBS for MtrA and MtrBC with RBS.
</P><TABLE border="1"><TBODY><TR><TH>Biobrick</TH><TH>Short name</TH><TH>Length</TH><TH>Characterized</TH></TR><TR><TD>BBa_K1127006</TD><TD>mtrABC</TD><TD>5168 bp</TD><TD>No</TD></TR></TBODY></TABLE></DIV><DIV id="hybridcontent" class="hiddenContent"><H2>The hybrid bistable switch</H2><P>
Biological systems usually exist in one of multiple states that are mutually exclusive. The switch for sporulation is a well known example. In <I>Bacillus subtilis</I>, this process is controlled by a complex network that involves more than 125 genes and interactions between proteins and other metabolites (1). Induced by starvation, feedback loops within the circuit compete for dictatorship to determine cell fate (2). <I>Vibrio fischeri</I> has a mutual relationship with the Hawaiian bobtail squid (<I>Euprymna scolopes</I>) . It lives in the squid's mantle cavity and produces luminous light to deter hosts from being eaten (3). Underlying process of the light production is quorum sensing. In contrary to sporulation, regulatory circuit of the quorum sensing is much simpler. In Figure 1, essential components of the circuit are LuxI and LuxR (4). LuxI is the synthase of the quorum sensing signal AHL. LuxR is the transcriptional activator of the promoter containing Lux box which is upstream of the LuxICDABE operon. This operon encodes enzymes - AHL synthase (LuxI), subunits of luciferase (LuxAB) and fatty acid reductase complex (LuxCDE). The last two are involved in light emission reactions. AHL is diffusible across the membrane, so that its concentration rises with the increase in population density. In addition, it increases stability of LuxR through cooperative dimerization. They form a positive feedback loop that synchronizes luminosity of the population. When population density is high, the autoinduction feedback loop responds to increasing AHL in the environment by switching cells from OFF state (no light emitting) to ON state (light emitting). 
</P><P>
These switches have capabilities to retain memories and make decisions. Such properties are increasingly interested by scientists around the globe, so that many have been trying to create systems with similar features (5, 6). Positive feedback loop is one of the basic elements in bistable switches. Multiple feedback loops are joined to enhance circuit regulation as well as robustness (7, 8). In Figure 2, positive feedback loops are joined to produce toggle switches that remember (9). The double toggle switch (bottom) displays strong bistability over a wider range of conditions (concentrations of the inducer). Other designs are also possible. Gardner TS. and colleagues constructed genetic toggle switches to determine conditions necessary for bistability in <I>Escherichia coli</I> (10). The circuits were made of two repressilators involving in transcription. Others designs include non-transcriptional components in the systems, and call it the hybrid bistable switches (11, 12). It is believed that they are robust and ultimately more tunable. For example, Daniel Huang and colleagues have demonstrated the usefulness of the sequestration-based bistability (6). In Figure 3 (left), the switch has σ^W which forms a positive feedback loop to itself, anti σ^W which sequestrates the sigma factor and a series of promoters for completing the feedback loop (P_(σ^W )) and for recognising the inducer (P_bad). P_tet is an optional component that adds another layer of regulation and makes their circuit even more tunable. In Figure 3 (right), the bifurcation plot shows transition between the states that is tuned by two inducer species (aTc and arabinose).
</P><P>
We aim to design a switch that is well regulated, tunable and robust. It must be bistable to synchronize current generation by the mtrABC complex with gold mineralization by the peptides, as well as to manage resource allocation with respect to fitness of the whole population. Quorum sensing came to us as a plausible solution. It is already a bistable switch as demonstrated by experiments (13). In addition, it has components that are transcriptional and enzymatic, so it could become tunable by multiple inducers with some extensions. 
</P><P>
We plan to assemble a genetic circuit from  Biobricks that have the same properties as the bistable switch in Figure 3. In theory, LuxI could form an indirect positive feedback loop through interactions with LuxR which is expressed by the constitutive promoter.  Upon gold induction, AiiA is expressed and degrade the quorum sensing signal AHL. In the other word, AiiA could disrupt the feedback loop. When gold is absent and population density is high, taR12 which is downstream of the Plux/lac is upregulated. In our project, taR12 is to transactivate the translation of mtrA upon binding to the crR12.
</P><TABLE border="1"><TBODY><TR><TH>Biobrick</TH><TH>Short name</TH><TH>Length</TH><TH>Characterized</TH></TR><TR><TD>BBa_K1127016</TD><TD>Gold and quorum sensing ultra complex</TD><TD>3708 bp</TD><TD>No</TD></TR><TR><TD>BBa_ K1127018</TD><TD>QS and gold sensing part for characterization</TD><TD>4674 bp</TD><TD>No</TD></TR></TBODY></TABLE></DIV><DIV id="resultscontent" class="hiddenContent"><H2>Results</H2><H3>Gold sensing device (BBa_K1127008)</H3><P>
This device is constructed to recognize gold (HAuCl4) in its environment. GolS is a transcriptional activator of the promoter P(golTS). Upon binding with the gold, it dimerizes and binds to the recognition site within the promoter. We also include LacZ alpha that is useful for quantification of the device activity via Beta-galactosidase assay. The mechanism is simplified and shown in Figure 1.
</P><P>
To determine activity of the gold sensing device, beta-galactosidase assays were run in triplicates using PNPG as the substrate. In all experiments, untransformed <I>E. coli</I> strain DH5 alpha or strain BL21 were used as our negative control. The protocols are described in more detail in the Protocols section.
</P><P>
We demonstrated that <I>E. coli</I> strain DH5 alpha transformed with BBa_K1127008 can respond to the gold. In Figure 2a, relative activity of the enzyme beta-galactosidase increased significantly in the presence of golS and promoter P(golTS) (DH5 alpha_AB) but reduced to the basal level when golS was missing (DH5 alpha_A). The results are well supported by statistics - Kruskal-Wallis X^2= 20.75, df = 7, p-value &lt; 0.01. This signifies the function of golS as a gold-dependent transcriptional activator.
</P><P>
In Figure 2b, <I>E. coli</I> strain BL21 has background expression of the enzyme. Strong enzyme activities were observed in the absence of lacZ alpha (BL21 and BL21_B), so that no differences were observed (Kruskal-Wallis X^2= 6.69, df = 3, p-value &gt; 0.05). This suggests that there could be beta-galactosidase in the chromosome of this <I>E. coli</I> strain. Further research has confirmed our hypothesis. Beta-galactosidase (accession no. = C6ELN6) is present in BL21 according to the gene database on uniprot.org.
</P><P>
We determined kinetics of the device. DH5 alpha were exposed to different gold concentrations for 16 hours to reach steady states. We found that the response was non-linear with distinct phases - exponential phase (0.0uM - 2.5uM AuCl4), stationary phase (2.5uM - 10.0uM AuCl4) and logarithmic decline phase (not shown). The data were plotted in Figure3a with simulation of the fitted logistic model. Please visit our Modelling section for more detail. As seen in the plot, the device is very sensitive to the gold as low as 0.5uM. Later on, it became saturated from about 10uM onwards. The saturation could be caused by several factor, such as maximum promoter activity, expression and stability of the proteins and diffusion limit of the gold. In Figure 3b,  we ran more assays on the subpart P(golTS)+LacZ alpha to confirm the function of golS. It is obvious that the promoter alone can't respond to the gold (Kruskal-Wallis X^2= 4.68, df = 6, p-value &gt; 0.5). 
</P><P><I>E. coli</I> strain DH5 alpha were transformed with the device or one of the subparts. They were streaked on the LB agar plates supplemented with chloramphenicol and Xgal. If LacZ alpha is expressed, the enzyme will degrade the substrate Xgal and turn colonies blue. In Figure 4, the colonies appear blue apart from those on the DH5 alpha+golS plate. This supports our hypothesis that P(golTS) is the minimal unit for the expression of LacZ alpha.
</P><H3>Gold scavenging peptides (BBa_K1127000-3)</H3><P>
We created four variants of the fusion between MIDAS-2 and A3. We also include the N-terminal pelB tag for secretion to the periplasm and IM9 for higher stability. Note that the tag should be removed during transportation process and potentially our peptides should be released to the extracellular space.  IM9 tag is relatively small (~9.5 kDa) and should not interfere with mineralization mediated by the peptides.
</P><P>
For peptide activity testing , we cloned the peptides under the constitutive promoter (BBa_J23100). Characterization using Nanoparticle Tracking Analyzer (NanoSight LM10 ®). The device allows us to visualize, measure and characterize any type of nanoparticles and any  size down to 10nm.  The technology is based on optical properties as well as Brownian motion of the particles.  Please visit http://www.nanosight.com/ for more information. 
</P><P>
We obtained a series of recordings and several analytical reports of the particles within samples from the overnight cultures of <I>E. coli</I> strain DH5 alpha transformed with the hybrids. Unfortunately, the results can't provide evidence for gold bio-mineralization because changes were subtle and not significant compared to the controls (untransformed <I>E. coli</I>). Please visit out Protocols section for more detail.
</P><P>Videos of nanoparticles present in cultures with bacteria transformed with a YG1, YG3, YG5, respectively, in HEPES.</P><VIDEO width="320" height="240" controls=""><SOURCE src="https://static.igem.org/mediawiki/2013/6/60/BBa_K1127000.mp4" type="video/mp4">
Your browser does not support videos, please view it on your computer. 
</SOURCE><VIDEO width="320" height="240" controls=""><SOURCE src="https://static.igem.org/mediawiki/2013/1/1b/BBa_K1127002.mp4" type="video/mp4">
Your browser does not support videos, please view it on your computer. 
</SOURCE><VIDEO width="320" height="240" controls=""><SOURCE src="https://static.igem.org/mediawiki/2013/a/a7/BBa_K1127003.mp4" type="video/mp4">
Your browser does not support videos, please view it on your computer. 
</SOURCE><A href="https://static.igem.org/mediawiki/2013/6/60/BBa_K1127000.mp4&quot;&quot;">To download Video 1</A><A href="https://static.igem.org/mediawiki/2013/1/1b/BBa_K1127002.mp4">To download Video 2</A><A href="https://static.igem.org/mediawiki/2013/a/a7/BBa_K1127003.mp4">To download Video 3</A></VIDEO></VIDEO></VIDEO></DIV><DIV id="microbialcontent" class="hiddenContent"><H2>Miniaturized microbial fuel cell (mMFC)</H2><P>
Traditional microbial fuel cells are bulky and inefficient. They are costly and less likely to be used in the industrial scale. We are fully aware of this issue, so we came up with the idea of miniaturized microbial fuel cell (mMFC).
</P><P>
According to our literature review, interests toward designs of the mMFC have been increasing over the last decades. Various models have been built and proved to be very successful [1, 2, 3]. For example, Shogo Inoue and colleagues have demonstrated that MFC can be as small as the size of one cent US coin (Figure 2) [1].  Even though the volume of their MFCs is smaller than 50 uL, the current produced is increased considerably  that up to 12 mW/100〖mm〗^3 was recorded regardless of modifications on both the MFCs and the bacteria.  In Figure 2, four MFC designs which are different by building materials and chamber volumes were compared [2]. There is no doubt that mMFCs can perform better. They produced about 300 times higher power densities than that of the bigger models.
</P><P>
Recently we came across the Bielefeld team who share the same interest and passion about MFC as a potential source of green energy. We get in touch and know that they also work on the mtrCAB complex, so we immediately see this as an opportunity for collaboration. We ask them to help us build the miniaturized MFC (Figure3) as they have facilities and materials for construction. We thank them for their generosity and wish them success in their iGEM project.
</P><P>
The MFC that they have sent us has 2 chambers, which in turn are divided by the cathode and the anode. Each chamber is 2.3 cm long, 2.3 cm wide and 1.5 cm high. The volume of each chamber is 7.9 cm3. The volume of the whole MFC is 15.8 cm3.
</P><P>Full Microbial Fuel Cell Assembly instructions by Bielefeld <A href="https://static.igem.org/mediawiki/2013/a/ae/Microbial_Fuel_Cell.pdf">here</A></P></DIV><DIV id="achievementscontent" class="hiddenContent"><H2>Achievements</H2><H3>Requirements for Bronze Medal</H3><UL><LI>Register the team, have a great summer, and plan to have fun at the Regional Jamboree.</LI><LI>
Successfully complete and submit this iGEM 2013 Judging form. <A href="https://igem.org/2013_Judging_Form?id=1127">Judging form</A></LI><LI>
Create and share a Description of the team's project using the iGEM wiki and the team's parts using the Registry of Standard Biological Parts. <A href="https://2013.igem.org/Team:York_UK/">Wiki</A></LI><LI>
Plan to present a Poster and Talk at the iGEM Jamboree.
</LI><LI>
Document at least one new standard BioBrick Part or Device used in your project/central to your project and submit this part to the iGEM Registry (submissions must adhere to the iGEM Registry guidelines). A new application of, and outstanding documentation (quantitative data showing the Part's/ Device's function), of a previously existing BioBrick part in the 'Experience' page of that part's Registry entry also counts. Please note you must submit this new part to the iGEM Registry.
(link)
</LI><LI>
Part Number(s): <A href="http://partsregistry.org/Part:BBa_K1127008"> BBa_K1127008</A>, <A href="http://partsregistry.org/Part:BBa_K1127000"> BBa_K1127000</A>, <A href="http://partsregistry.org/Part:BBa_K1127001"> BBa_K1127001</A>, <A href="http://partsregistry.org/Part:BBa_K1127002">BBa_K1127002</A>, <A href="http://partsregistry.org/Part:BBa_K1127003"> BBa_K1127003</A>, <A href="http://partsregistry.org/Part:BBa_K1127006">BBa_K1127006</A>, <A href="http://partsregistry.org/Part:BBa_K1127016"> BBa_K1127016</A>,<A href="http://partsregistry.org/Part:BBa_K1127018"> BBa_K1127018</A></LI><LI>
Description of function
</LI><LI>
Quantitative data showing the Part or Device function
</LI><LI>
Acknowledgment of sources and references
</LI></UL><H3>Additional requirements for Silver Medal:</H3><UL><LI>
Experimentally validate that at least one new BioBrick Part or Device of your own design and construction works as expected
</LI><LI>
Document the characterization of this part in the 'Main Page' of that Part's/Device's Registry entry (link)
</LI><LI>
Submit this new part to the iGEM Parts Registry (submissions must adhere to the iGEM Registry guidelines) <A href="http://parts.igem.org/cgi/dna_transfer/batch_list.cgi?group_id=1439">link to the message</A></LI><LI>
Part Number(s): <A href="http://partsregistry.org/Part:BBa_K1127008">BBa_K1127008</A></LI><LI>
Description of function
</LI><LI>
Quantitative data showing the Part or Device function
</LI><LI>
Acknowledgment of sources and references
</LI><LI>
Your project may have implications for the environment, security, safety and ethics and/or ownership and sharing. Describe one or more ways in which these or other broader implications have been taken into consideration in the design and execution of your project.
We are working in an isolated environment inside the MFC (anaerobic chamber); The gold ions that we are using in our experiments are potentially dangerous to the environment but we are using HEPES buffer to reduce the ions to nanoparticles, which pose no threat (add a link to the video of HEPES buffer reducing gold ions to nanoparticles by itself); <A href="https://2013.igem.org/Team:York_UK/Project.html?page=safety">Safety</A></LI><LI>
Also, we did outreach activities to inform people  about synthetic biology and allowed them to think about its uses, advantages and disadvantages.
</LI><LI>
Link to page on your team's wiki: <A href="https://2013.igem.org/Team:York_UK/Outreach.html">Team:York_UK/Outreach.html</A></LI></UL><H3>Additional Requirements for a Gold Medal: (one OR more)</H3><UL><LI>
Improve the function of an existing BioBrick Part or Device (created by another team or your own institution in a previous year), enter this information in the 'Experience' section of that Parts's Registry entry), create a new registry page for the improved part, and submit this part to the iGEM Registry (submissions must adhere to the iGEM Registry guidelines (look below for additional information below)
</LI><LI>
Part Number(s): <A href="http://partsregistry.org/Part:BBa_K1127008"> BBa_K1127008</A>, <A href="http://partsregistry.org/Part:BBa_K1127006"> BBa_K1127006</A></LI><LI>
Description of function
</LI><LI>
Quantitative data showing the Part or Device function (BBa_K1127008 only)
</LI><LI>
Acknowledgment of sources and references
</LI><LI>
Help any registered iGEM team from another school or institution by, for example, characterizing a part, debugging a construct, or modeling or simulating their system.  Link to Collaborations
</LI><LI>
Outline and detail a new approach to an issue of Human Practice in synthetic biology as it relates to your project, such as safety, security, ethics, or ownership, sharing, and innovation Link to church
</LI><LI>
We have established a novel link between science and religion, which could be explored in the future to improve the image of Synthetic Biology (And Science in general) in the media and amongst highly religious people, who disprove its significance. Also, there is still a big gap between science and religion, just because people think that these two areas cannot be combined. We discussed these problems and tried to show that both of them can be combined.
</LI></UL><H3>Additional information for improvement:</H3><P>
While doing some research to plan our constructs we used iGEM's library to find if any previous team had submitted information related to either the gol operon in <I>Salmonella enterica</I> or the mtr complex in Shewanella sp. 	
</P><P>
We had already read through literature (Soncini et al. 2007) and found the sequences for golB, golS and golT. This led us to identify PgolTS and PgolB, the promoters for these genes. This is the reason why when we found iGEM 2010 team UIUC Illinois' parts we were perplex. 
</P><P>
They had submitted PgolS (BBa_K310009) and their sequence did not match up the one we had identified after our research.  We followed their information and realized that what they had submitted as PgolS was actually a part of <I>Salmonella enterica</I> typhimurium serov. 14208s that included part of golS, PgolB and part of golB. 
</P><P>
Promoters of protein coding genes are usually upstream the genome therefore the possibility of this being the right sequence, in spite of what we had already discovered, was quite remote. (remota) Because they also submitted PgolS with ECFP reporter (BBa_K310015) and PgolS with EYFP reporter (BBa_K310016), these sequences are not correct either. In addition our latest assays prove our PgolsS construct to work efficiently.
</P><P>
After this discovery we checked other Illinois' gol operon related parts and realized they had submitted golB with what we believe is a typo within its sequence (BBa_K310011) and also in the parts that included it (BBa_K310014).
</P><P>
We suggest that these should be investigated by others teams too in order to prevent future participants from confusion. 
</P><P>
PgolS (BBa_K310009) was stated to be 500 bp, but we improved it by determining the right sequence cttgaccttcccacaatggcaagctttaggctttctgat. This was confirmed by activity assays <A href="http://partsregistry.org/Part:BBa_K1127008">BBa_K1127008</A></P><P>Checa, K., Espariz, M., Pérez Audero E., Botta P. E., Spinelli S. V. and Soncini F. C. (2007) ‘Bacterial sensing of and resistance to gold salts’ Molecular Microbiology 63, 5, pp.1307-1318 [ONLINE] Available at: http://onlinelibrary.wiley.com/doi/10.1111/j.1365-2958.2007.05590.x/full (Accesed: 10th June 2013)
</P></DIV><DIV id="collaborationscontent" class="hiddenContent"><H2>Collaborations</H2><H3>Collaboration with the iGEM Team Bielefeld</H3><P>
We have worked with the team from Bielefeld in order to create our MFC. They have kindly offered to help us with the construction of the parts for the MFC as they have the necessary equipment. We are grateful for their generosity and we wish them the best of luck with their project at the Jamboree.
</P><H3>Collaboration with the iGEM Team Manchester</H3><P>
We have joined forces with the iGEM Team Manchester to create a video for their series about modelling, which is very appropriate and useful for beginners.  In our video we are discussing in detail why we have chosen to do our modelling on R instead of other programs like Matlab, Python, COPASI and VCell. You can find it on <A href="http://www.youtube.com/watch?v=TltmKou8MW4">YouTube</A>.

</P><H3>Collaboration with the iGEM Team Copenhagen</H3><P>
We have also made a video for the iGEM Team Copenhagen's initiative to aid future iGEM teams by starting a YouTube series called Bricks of Knowledge. Our video is called &quot;How to insert a gene&quot; and explores the topic of cloning in broad terms. It provides a general overview of each step of the whole process as well as information about the things that you have to bear in mind while performing the experiments. It is suitable for people, who like us have struggled with cloning (that is why we have decided to share our wisdom on the topic). Find the video on <A href="http://www.youtube.com/watch?v=WsUne4Z5hpE">Youtube</A></P><H3>Collaboration with the iGEM team Purdue</H3><P>
We have participated in a Google+ chat with representatives  from many iGEM teams and we have discussed Purdue's idea of creating a universal standard, in which all new parts that are submitted to the iGEM registry should be characterised. We fully support this cause as we believe that it would be beneficial for the iGEM teams if we characterise all parts in the same way.

</P><H3>Collaboration with Microryza</H3><P>
We have joined forces with the people working behind the crowd funding platform called Microryza, who tried to promote our project and help us raise funds as well as awareness of our idea. We have also made a YouTube video about our project and iGEM in general in order to promote ourselves as well as the competition. Find it <A href="http://www.youtube.com/watch?v=XsFm4K20DCE">here</A></P></DIV><DIV id="gallerycontent" class="hiddenContent"><DIV class="gallery-section"><H2> Labs </H2></DIV><DIV class="gallery-section"><H2> Nanosight </H2></DIV><DIV class="gallery-section"><H2> Outreach - St. Helen's Church </H2></DIV><DIV class="gallery-section"><H2> Outreach - St. Lawrence's Primary School </H2></DIV></DIV><DIV id="safetycontent" class="hiddenContent"><H2>Safety</H2><H3>Safe Practices in the Lab</H3><P>
Our team conducted the research at the University of York, in the teaching laboratories of the Biology Department.  All work was conducted in a biosafety level (BSL) 1 laboratory. We used Escherichia coli DH5a, BL21 and K12 wild type, which are well characterised and do not constitute a threat to the environment. Shewanella oneidensis and Salmonella enterica typhimurium  serovar 14208s were not used (as we had the genes synthesised, instead of amplifying them from the genome). Before gaining access to the lab space, team members were required to:</P><OL><LI>Complete chemical waste disposal training</LI><LI>Attend an orientation with Dr Jen Lee and Nikki Begg, part of the technical staff of the labs, regarding the University of York Health &amp; Safety Regulations</LI><LI>Receive specific training from Dr Jen Lee and from Dr James Chong, our main supervisor, for the equipment in the lab</LI><LI>Complete a risk assessment form provided by Biological Safety Advisor (David Nelson)</LI><LI>Sign after-hours access agreements with the University, after having been instructed on the regulations and procedures which apply for working in the lab at weekends and between 18:00-08:00</LI></OL><P>
Our standard lab practices were in compliance with the World Health Organization’s Biosafety Level 1 guidelines. Safety forms were approved on September 22, 2013, by Evan Appleton of behalf of iGEM. Many of our basic lab practices are described below.
</P><UL><LI>Researchers were required to wear gloves while in the lab space, and to remove both gloves and wash their hands when going into the write-up area or in other lab common areas;</LI><LI>No gloves were allowed to leave the lab space, and no food or drink was allowed into the lab space, including the computer room;</LI><LI>Within the Department of Biology, microbiological samples were transported only via the internal service elevator and on lab-associated stairs which were accessed by card, to avoid contamination of public areas;</LI><LI>All flammable liquids were kept in a flammable storage cabinet;</LI><LI>The lab contained distinct waste containers for general waste, biohazard waste, biohazard sharps, and non-biohazard broken glassware;</LI><LI>Liquid bacterial waste was always autoclaved before being discharged into sanitary sewage;</LI><LI>Benchtops were decontaminated with ethanol before and after lab work was conducted;</LI><LI>Tools that came into contact with bacteria were soaked in ethanol and flame-treated before and after use;</LI><LI>An autoclave was used to decontaminate growth media, glassware, tubes, pipette tips, etc. All lab members were trained in preparing waste and media for being autoclaved. The actual autoclaving programs were run for us by the staff of the teaching labs, to avoid dangers to researcher safety;</LI><LI>An emergency shower, eyewash, and first aid kit were available within the lab space in case of emergency.</LI><LI>Dry ice was carried in iceboxes and thick gloves were used to take samples from them. Dry ice was left in the room to evaporate instead of disposing anywhere.</LI><LI>Microwave was used for melting agarose for agarose gels and LB agar for making LB agar plates. Thick gloves were used to take hot samples after heating. </LI><LI>Gel electrophoresis tanks were kept separately to avoid any electric shock.</LI><LI>All the unused isolated DNA was autoclaved in order to shear the genetic material that would not be used by bacteria in environment.</LI><LI>While using Bunsen burners no gloves were worn in order to prevent melting the gloves and causing high level of damage to the skin.</LI><LI>For adjusting the pH highly corrosive 1M NaOH was used. Thus gloves and protective glasses were worn in order to avoid any skin burning.</LI><LI>Phenol/Chloroform extraction of S. oneidensis genome was performed in the fume hood as both of the compounds are volatile and toxic.</LI><LI>β-mercaptoethanol was  used during the beta galactosidase activity assay. All the procedures performed in the fume hood to avoid intoxication of highly volatile compound.</LI><LI>As gold ions are toxic to most of the living organisms after each experiment we treated them with the excess of HEPES to form gold aggregates as well as washed with excess of water to dilute to non-toxic concentrations.</LI><P>For more information please refer to the safety forms:<A href="https://static.igem.org/mediawiki/2013/6/6f/York_Laboratory_Safety_Guidelines.pdf">Laboratory Safety Guidelines</A><A href="https://static.igem.org/mediawiki/2013/9/94/York_Safety_Form.pdf">Laboratory Safety Form</A>
Safety forms were approved on September 22, 2013 by Evan Appleton.
</P><P>
To minimise the risk one might be exposed to when working with Ethidium Bromide (which is a carcinogen) we used its safer alternative, SYBR Safe, for our gel electrophoresis.  Team members were protected from exposure to UV rays during gel reading with a UV shield. Facial shields were also available for use when needed. 
</P><H3>Safety Concerns for Our Project:</H3><P>
There are no serious safety concerns, as the genes or their products that we work on have no identified risk to human health, do not influence the environment and are specific for only our project.
</P><H3>Do any of the new BioBrick parts (or devices) that you made this year raise any safety issues?</H3><P>
The only safety issue associated with our new BioBrick parts is researcher safety in testing the parts. We have documented our standard operating procedures above, which other teams can reference if they want to use our parts, or test other parts using arsenic or naphthalene. 
We are also conducting extensive tests to assess the functionality of our BioBrick parts, to ensure that they behave as expected under different conditions. 
</P><H3>Safety Concerns for iGEM Competition:</H3><H3>Do you have any other ideas how to deal with safety issues that could be useful for future iGEM competitions? How could parts, devices and systems be made even safer through biosafety engineering?</H3><UL><LI>
use other selection forms like auxotrophic mutants for amino acids, not antibiotic resistance, to prevent release of resistant strains into the environment. However, because we wanted to try MFC with wastewater, this method would not be useful as in addition we would need to add amino acid, which would affect the metabolism.
</LI><LI>
usually the antibiotics that are used in the lab are different from medically common antibiotics and would potentially cause no influence towards MRSA and similar
</LI><LI>
make standard iGEM safety guidelines to be followed by all the teams
</LI></UL><DIV class="printfooter">
Retrieved from &quot;<A href="http://2013.igem.org/Team:York_UK/Project.html">http://2013.igem.org/Team:York_UK/Project.html</A>&quot;</DIV><DIV id="footer-box" class="noprint"><DIV id="footer"><UL id="f-list"><LI id="t-recentchanges"><A href="/Special:RecentChanges" title="Recent changes">Recent changes</A></LI><LI id="t-whatlinkshere"><A href="/Special:WhatLinksHere/Team:York_UK/Project.html" title="List of all wiki pages that link here [j]" accesskey="j">What links here</A></LI><LI id="t-recentchangeslinked"><A href="/Special:RecentChangesLinked/Team:York_UK/Project.html" title="Recent changes in pages linked from this page [k]" accesskey="k">Related changes</A></LI><LI id="t-specialpages"><A href="/Special:SpecialPages" title="List of all special pages [q]" accesskey="q">Special pages</A></LI><LI><A href="/Special:Preferences">My preferences</A></LI></UL></DIV><DIV id="footer"><UL id="f-list"><LI id="t-print"><A href="/wiki/index.php?title=Team:York_UK/Project.html&amp;printable=yes" title="Printable version of this page [p]" accesskey="p">Printable version</A></LI><LI id="t-permalink"><A href="/wiki/index.php?title=Team:York_UK/Project.html&amp;oldid=311627" title="Permanent link to this revision of the page">Permanent link</A></LI><LI id="privacy"><A href="/2013.igem.org:Privacy_policy" title="2013.igem.org:Privacy policy">Privacy policy</A></LI><LI id="disclaimer"><A href="/2013.igem.org:General_disclaimer" title="2013.igem.org:General disclaimer">Disclaimers</A></LI></UL></DIV></DIV></UL></DIV></DIV></DIV></DIV></DIV></DIV></BODY></HTML>