<HTML xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en" dir="ltr">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki  ltr ns-0 ns-subject page-Team_York_UK_Acknowledgments_html"><DIV id="globalWrapper"><DIV id="top-section"><DIV id="p-logo"><A href="/Main_Page" title="Main Page">&quot;
	    </A></DIV><DIV id="menubar" class="left-menu noprint"><UL><LI class="selected"><A href="/Team:York_UK/Acknowledgments.html">Page               </A></LI><LI class="new"><A href="/wiki/index.php?title=Talk:Team:York_UK/Acknowledgments.html&amp;action=edit&amp;redlink=1">Discussion               </A></LI><LI><A href="/wiki/index.php?title=Team:York_UK/Acknowledgments.html&amp;action=edit">View source               </A></LI><LI><A href="/wiki/index.php?title=Team:York_UK/Acknowledgments.html&amp;action=history">History               </A></LI><LI style="color:white;cursor:default">teams</LI></UL></DIV><DIV class="right-menu noprint" id="menubar"><UL><LI id="pt-login"><A href="/wiki/index.php?title=Special:UserLogin&amp;returnto=Team:York_UK/Acknowledgments.html" title="You are encouraged to log in; however, it is not mandatory [o]" accesskey="o">Log in</A></LI></UL></DIV><DIV id="search-controls" class="noprint"><FORM action="/Special:Search" id="searchform"> </FORM></DIV></DIV><DIV id="content"><H1 class="firstHeading">Team:York UK/Acknowledgments.html</H1><DIV id="bodyContent"><H3 id="siteSub" class="noprint">From 2013.igem.org</H3><P><TITLE>iGEM York</TITLE></P><DIV id="header"><DIV id="navigation"><UL><LI><A href="https://2013.igem.org/Team:York_UK">Home</A></LI><LI><A href="https://2013.igem.org/Team:York_UK/Team.html">Team</A></LI><LI><A href="https://2013.igem.org/Team:York_UK/Project.html">Project</A></LI><LI><A href="https://2013.igem.org/Team:York_UK/Notebook.html">Notebook</A></LI><LI><A href="https://2013.igem.org/Team:York_UK/Outreach.html">Outreach</A></LI><LI id="acknowledgements"><A href="https://2013.igem.org/Team:York_UK/Acknowledgments.html">Acknowledgments</A></LI></UL></DIV><DIV id="nav2"><DIV id="team"><A href="https://2013.igem.org/Team:York_UK/Team.html?page=contacts">Contacts</A><A href="https://2013.igem.org/Team:York_UK/Team.html?page=university">The University</A><A href="https://2013.igem.org/Team:York_UK/Team.html?page=biography">Biography</A><A href="https://igem.org/Team.cgi?year=2013&amp;team_name=York_UK">Profile</A><A href="https://2013.igem.org/Team:York_UK/Team.html?page=overview">Overview</A></DIV><DIV id="project"><A href="https://2013.igem.org/Team:York_UK/Project.html?page=safety">Safety</A><A href="https://2013.igem.org/Team:York_UK/Project.html?page=gallery">Gallery</A><A href="https://2013.igem.org/Team:York_UK/Project.html?page=collaborations">Collaborations</A><A href="https://2013.igem.org/Team:York_UK/Project.html?page=achievements">Achievements</A><A href="https://2013.igem.org/Team:York_UK/Project.html?page=microbial">Microbial Fuel Cells</A><A href="https://2013.igem.org/Team:York_UK/Project.html?page=results">Results</A><A href="https://2013.igem.org/Team:York_UK/Project.html?page=parts">Parts</A><A href="https://2013.igem.org/Team:York_UK/Project.html?page=overview">Overview</A></DIV><DIV id="notebook"><A href="https://2013.igem.org/Team:York_UK/Notebook.html?page=modelling">Modelling</A><A href="https://2013.igem.org/Team:York_UK/Notebook.html?page=protocols">Protocols</A><A href="https://2013.igem.org/Team:York_UK/Notebook.html?page=labbook">Kobchai's Lab Book</A><A href="https://2013.igem.org/Team:York_UK/Notebook.html?page=wetlab">Jonas' Lab Book</A><A href="https://2013.igem.org/Team:York_UK/Notebook.html?page=drylab">Dry Lab</A></DIV><DIV id="outreach"><A href="https://2013.igem.org/Team:York_UK/Outreach.html?page=london">YSB 1.0</A><A href="https://2013.igem.org/Team:York_UK/Outreach.html?page=alumni">Alumni Event</A><A href="https://2013.igem.org/Team:York_UK/Outreach.html?page=lawrence">St. Lawrence's School</A><A href="https://2013.igem.org/Team:York_UK/Outreach.html?page=helen">St. Helen's Church</A></DIV><DIV id="acknowledgments"><A href="https://2013.igem.org/Team:York_UK/Acknowledgments.html?page=sponsors">Sponsors</A><A href="https://2013.igem.org/Team:York_UK/Acknowledgments.html?page=references">References</A><A href="https://2013.igem.org/Team:York_UK/Acknowledgments.html?page=attributions">Attributions</A></DIV></DIV></DIV><TITLE>iGEM York Team</TITLE><DIV id="container"><DIV id="navlocal"><UL><LI>Team</LI><LI><DIV id="attributions"><A href="">Attributions</A></DIV></LI><LI><DIV id="references"><A href="">References</A></DIV></LI><LI><DIV id="sponsors"><A href="">Sponsors</A></DIV></LI></UL></DIV><DIV id="conlocal"><DIV id="sponsorscontent" class="hiddenContent"><H2>Sponsors</H2><DIV class="single-sponsor clearfix"><P>
The York Annual Fund grants funding opportunities for departments, colleges, and student clubs and societies, which support innovative projects that improve opportunities for students and help the university to contribute to the life of the wider community. The money is donated by alumni and friends of the University who are keen to invest in enhancing the opportunities and supporting current students. The York Annual Fund had a budget of roughly £50,000 but received applications for over £100,000. They generously gave us £8,000 to get our project up and running and for this we are incredibly grateful.
</P></DIV><DIV class="single-sponsor clearfix"><P>The Biology Department at the University of York offers a spectrum of modern biology; from ecology to biomedicine. It has the status of one of the UK’s most elite biological sciences departments as measured by its high rankings for both research and teaching. The department is currently ranked 2nd in the Guardian UK University Guide 2014 league table, 3rd in the Times Good University Guide 2013 and 4th in the 2014 Complete University Guide.

            For more information click <A href="http://www.york.ac.uk/biology/">here</A></P></DIV><DIV class="single-sponsor clearfix"><P>(We received money for the payment of our entry fees in iGEM. The Rapid Response Fund gave us £1000 in order to buy materials for our lab and start working.)
            The Rapid Response Fund of the University of York grants up to £3,000. They are available in support of small-scale short-term projects, initiatives or purchases to enhance the quality of learning and teaching by addressing a clearly-identified need or issue.

            For more information click <A href="http://www.york.ac.uk/">here</A></P></DIV><DIV class="single-sponsor clearfix"><P>
(They gave us samples of the most common restriction enzymes.) 
            Thermo Fisher Scientific Inc. serve customers within pharmaceutical and biotech companies, hospitals and clinical diagnostic labs, universities, research institutions and government agencies, as well as in environmental and process control industries. They create value for their key stakeholders through three premier brands, Thermo Scientific, Fisher Scientific and Unity™ Lab Services, which offer a unique combination of innovative technologies, convenient purchasing options and a single solution for laboratory operations management. Their products and services help their customers solve complex analytical challenges, improve patient diagnostics and increase laboratory productivity.

            For more information click <A href="http://www.thermofisher.com/global">here</A></P></DIV><DIV class="single-sponsor clearfix"><P>
At the forefront of the molecular biology revolution, more than 500,000 customers worldwide are using QIAGEN Sample &amp; Assay Technologies to achieve breakthroughs based on the building blocks of life - DNA, RNA and proteins. 
             Doctors are diagnosing diseases more precisely and quickly to guide treatment decisions. Scientists are conquering new frontiers in the understanding of life - and translating that knowledge into better medications. And experts are safeguarding the public with molecular technologies fur human identification, veterinary testing and food safety. A new generation of QIAGEN innovation is creating value in the global market with solutions not even imagined a few years ago. QIAGEN gave us kits for Gel Extraction and PCR purification without which our project would not have been successful.

            For more information click <A href="http://www.qiagen.com/">here</A></P></DIV><DIV class="single-sponsor clearfix"><P>
Founded in the mid-1970s as a collective of scientists committed to developing innovative products for the life sciences industry, New England Biolabs is now a recognized world leader in the discovery, development and commercialization of recombinant and native enzymes for genomic research. NEB gave us a free Gibson Assembly kit.

            For more information click <A href="https://www.neb.com/">here</A></P></DIV><DIV class="single-sponsor clearfix"><P>
The group has its headquarters in Nottingham, UK where it operates state of the art reference laboratory facilities. The LifeSciences division provides Sanger sequencing, Next Generation sequencing, bioinformatic analyses and a comprehensive portfolio of over 20 million clones and 200,000 antibodies. 
            The Healthcare division provides screening and reference laboratory diagnostic testing for cancer and other diseases and additional predictive testing for treatment optimisation for clinicians and patients. 
            The PharmaBiotech division supports the development and validation of biomarkers for use as companion diagnostics in new drug launches or as pharmacodynamic endpoints for clinical and research studies. 
            Source Bioscience gave us sequencing vouchers to help with the final stages of our project.

             For more information click <A href="http://www.sourcebioscience.com/">here</A></P></DIV><DIV class="single-sponsor clearfix"><P>
NanoSight’s “Nanoparticle Tracking Analysis” (NTA) technology detects and visualises nanoparticles in liquids down to 10nm. NanoSight are able to offer size distribution, concentration, zeta potential and fluorescence measurements, on a particle-by-particle basis.  This technology is being utilised in the development of drug delivery systems and viral vaccines, and in nanotoxicology. It also gives insight into the kinetics of protein aggregation and has a growing role in biodiagnostics, including the detection and speciation of exosomes and microvesicles. NanoSight has installed 600+ systems worldwide and its technology is validated by 700+ third party papers citing NanoSight results, consolidating NanoSight’s leading position in nanoparticle characterization.

            For more information click <A href="http://vimeo.com/62625839">here</A></P></DIV></DIV><DIV id="referencescontent" class="hiddenContent"><H2>References</H2><H3>Gold sensing</H3><OL><LI>
Brown, N. L., Stoyanov, J. V., Kidd, S. P. &amp; Hobman, J. L. The MerR family of transcriptional regulators. FEMS. Microbiol. Rev. 27, 145-163 (2003).
</LI><LI>
Newberry, K. J. &amp; Brennan, R. G. The structural mechanism for transcription activation by MerR family member multidrug transporter activation, N terminus. J. Biol. Chem. 279, 20356-20362 (2004).
</LI><LI>
Stoyanov, J. V. &amp; Brown, N. L. The Escherichia coli copper-responsive copA promoter is activated by gold. J. Biol. Chem. 278, 1407-1410 (2003).
</LI><LI>
Grass, G. &amp; Rensing, C. Genes Involved in Copper Homeostasis in Escherichia coli. J. Bacteriol. 183, 2145-2147 (2001).
</LI><LI>
Wei, W. et. al. Engineering a gold-specific regulon for cell-based visual detection and recovery of gold. Chem. Sci. 3, 1780-1784 (2012).
</LI><LI>
Hibman, J. L. MerR family transcription activators: similar designs, different speciﬁcities. Mol. Microbiol. 63, 1275-1278 (2007).
</LI><LI>
Checa, S. K., Espariz, M., Audero, M. E. P., Botta, P. E., Spinelli, S. V. &amp; Soncini, F. C. Bacterial sensing of and resistance to gold salts. Mol. Microbiol. 63, 1307-1318 (2007).
</LI><LI>
Cerminati, S., Soncini, F. C. &amp; Checa, S. K. Selective Detection of Gold Using GeneticallyEngineered Bacterial Reporters. Biotechnol. Bioeng. 108, 2553-2560 (2011).
</LI><LI>
Pontel, L. B., Audero, M. E. P., Espariz, M., Checa, S. K. &amp; Soncini, F. C. GolS controls the response to gold by the hierarchical induction of Salmonella-speciﬁc genes that include a CBA efflux-coding operon. Mol. Microbiol. 66, 814-825 (2007).
</LI></OL><H3>The hybrid bi-stable switch</H3><OL><LI>
Fawcett, P., P. Eichenberger, R. Losick &amp; P. Youngman. The trancriptional proﬁle of early to middle sporulation in Bacillus subtilis. Proc. Natl. Acad. Sci. USA. 97, 8063–8068 (2000).
</LI><LI>
Jong, H., Geiselmann, J., Batt, G., Hernandez, C. &amp; Page, M. Qualitative Simulation of the Initiation of Sporulation in Bacillus subtilis. B. Math. Biol. 66, 261–299 (2004).
</LI><LI>
Ng, W. L &amp; Bassler, B. L. Bacterial quorum-sensing network architecture. Annu. Rev. Genet. 43, 197-222 (2009).
</LI><LI>
Levskaya, A. et. al. Synthetic biology: Engineering Escherichia coli to see light. Nature. 438, 441–442 (2005).
</LI><LI>
Chen, D. &amp; Arkin, A. P. Sequestration-based bistability enables tuning of the switching boundaries and design of a latch. Mol. Syst. Biol. 620, 1-7 (2012).
</LI><LI>
Shah, N. A. &amp; Sarkar, C. A. Robust Network Topologies for Generating Switch-
Like Cellular Responses. PLoS Comput Biol. 7 (2011)
</LI><LI>
Pfeuty, B. &amp; Kaneko, K. The combination of positive and negative feedback loops confers exquisite flexibility to biochemical switches. Phys. Biol. 6, 046013 (2009) 
</LI><LI>
Chang, D. E., Leung, S., Atkinson, M. R., Reifler, A., Forger, D. &amp; Ninfa, A. J. Building biological memory by linking positive feedback loops. PNAS. 107 175–180 (2010).
</LI><LI>
Gardner, T. S., Cantor, C. R. &amp; Collins, J. J. Construction of a genetic toggle switch in Escherichia coli. Nature. 403, 339-342 (2000).
</LI><LI>
Huang, D., Holtz, W. J. &amp; Maharbiz, M. M. A genetic bistable switch utilizing nonlinear protein degradation. J. Biol. Eng. 6, 1-13 (2012).
</LI><LI>
Palani, S. &amp; Sarkar, C. A. Synthetic conversion of a graded receptor signal into a tunable, reversible switch. Mol. Syst. Biol. 7, 480 (2011).
</LI><LI>
Haseltine, E. L. &amp; Arnold, F. H. Implications of Rewiring Bacterial Quorum Sensing. Appl. Environ. Microbiol. 74, 437-445 (2008).
</LI></OL><H3>Microbial Fuel Cell</H3><OL><LI>
Shogo Inoue, Erika A. Parra, Adrienne Higa, Yingqi Jiang, Pengbo Wangc, Cullen R. Buied, John D. Coatese, Liwei Lin. Structural optimization of contact electrodes in microbial fuel cells for current density enhancements. Sensors and Actuators A 177, 30–36 (2012).
</LI><LI>
Fang Qian, Zhen Hec, Michael P. Thelen, Yat Li. A microﬂuidic microbial fuel cell fabricated by soft lithography. Bioresource Technology (2011)
</LI><LI>
BRADLEY R. RINGEISEN, EMILY HENDERSON, PETER K. WU, JEREMY PIETRON, RICKY RAY, BRENDA LITTLE, JUSTIN C. BIFFINGER, JOANNE M. JONES-MEEHAN. High Power sensity from a Miniature Microbial Fuel Cell Using Shewanella oneidensis DSP10. Environmental Science Technology 40, 2629-2634 (2006).
</LI></OL><H3>Gold scavenging peptides</H3><OL><LI>Dykman, L. A. &amp; Khlebstov, N. G. Gold nanoparticles in biology and medicine: recent advances and prospects. Acta. Naturae. 3, 34-55 (2011).</LI><LI>Mikami, Y., Dhakshinamoorthy, A., Alvaro, M. &amp; Garcia, H. Catalytic activity of unsupported gold nanoparticles. Catal. Sci. Technol. 3, 58-69 (2013).</LI><LI>Wilson, R. The use of gold nanoparticles in diagnostics and detection. Chem. Soc. Rev. 37, 2028-2045 (2008).</LI><LI>Grzelczak, M., Juste, J. P., Mulvaney, P. &amp; Marzan, L. M. L. Shape control in gold nanoparticle synthesis. Chem. Soc. Rev. 37, 1783-1791 (2008).</LI><LI>Johnston, C. W. et. al. Gold biomineralization by a metallophore from a gold-associated microbe. Nat. Chem. Biol. 9, 241-243 (2013).</LI><LI>Kim, J. et. al. Peptide mediated shape and size tunable synthesis of gold nanoparticles. Acta. Biomater. 6, 2681-2689 (2010).</LI><LI>Naik, R. R., Stringer, S. J., Agarwal, G., Jones, S. E. &amp; Stone, M. O. Biomimetic synthesis and patterning of silver nanoparticles. Nat. Mater. 1, 169-172 (2002).</LI><LI>Slocik, J. M., Stone, M. O. &amp; Naik, R. R. Synthesis of gold nanoparticles using multifunctional peptides. Small. 1, 1048-1051 (2005).</LI><LI>Stanley, S. K., Becker, M. L., Lin, E. K. &amp; Wu, W. L. Inhibitory effects of a phage derived peptife on Au Nanocrystal Nucleation and growth. Langmuir. 25, 10886-10892 (2009).</LI><LI>Yu, J., Becker, M. L. &amp; Carri, G. A. The influence of amino acid sequence and functionality on the binding process of peptides onto gold surfaces. Langmuir. 28, 1408-1417 (2012).</LI><LI>Tong, L. et. al. Extracellular expression, purification, and characterization of a winter flounder antifreeze polypeptide from Escherichia coli. Protein. Expr. Purif. 18, 175-181 (2000).</LI><LI>http://www.nanosight.com/</LI><LI>Zhou, Y., Kong, Y., Kundu, S., Cirillo, J. D. &amp; Liang, H. Antibacterial activities of gold and silver nanoparticles against Escherichia coli and bacillus Calmette-Guérin. J. Nanobiotechnology. 10, doi: 10.1186/1477-3155-10-19 (2012).</LI><LI>Huser, P. S. &amp; Ryan, R. O. Expressed protein ligation using an N-terminal cysteine containing fragment generated in vivo from a pelB fusion protein. Protein. Expre. Purif. 54, 227-233 (2007).</LI><LI>Spirin, A. S. &amp; Swartz, J. R. Cell-Free Protein Synthesis: Methods and Protocols. DOI: 10.1002/9783527622702 (2008).</LI></OL></DIV><DIV id="attributionscontent" class="hiddenContent"><H2>Attributions</H2><H3>Team</H3><P><B>Gintare Bucaite</B>: sponsorship (applied for departmental sponsorship, prepared promotional materials); website (non-scientific content); contributed to dry lab work (nanoparticle forming peptides); gave presentation at iGEM YSB London; budgeting (organised travel and accommodation for YSB and Lyon Jamboree; organised living stipends and payment of fees); wet lab work (worked in the lab for 9 weeks, DNA cloning and part characterisation).
</P><P><B>Ricardo Cañavate</B>: responsible for social media (facebook and twitter pages); organising meetings and keeping minutes; contributed to dry lab work (gold-sensing system); wet lab work for 8 weeks (nanoparticle forming peptides); made posters for iGEM presentations; sent sponsorship applications.
</P><P><B>Kobchai Duangrattanalert </B>: dry lab work (idea refinement; MFC technology; mtr complex; peptide design; quorum sensing system development; construct design); wet lab work (12 weeks on peptide cloning and characterisation, gold-sensing system assembly); organised Journal Clubs; gave scientific presentations; liason with academics at other institutions; logistics (procurement of materials); website (writing scientific content).
</P><P><B>Andrew Farthing</B>: did modelling; contributed to dry lab work (Quorum sensing, peptide design); raising sponsorship (applied for university funding, prepared promotional materials, crowdfunding); wet lab work for 12 weeks (gold-sensing system); poster design; writing website content; organising and participating in outreach events.
</P><P><B>Ivan Gyulev</B>: sponsorship (contacted pharma companies, prepared promotional materials); dry lab work (Quorum sensing and MTR complex); wet lab work for 12 weeks (nanoparticle forming peptides); gave presentation at iGEM YSB London; made scientific poster; organised Journal Clubs; student-academic staff liason person; participated in outreach events.
</P><P><B>Hannah Johnson</B>: PR person; wrote our press releases; contributed to dry lab work (reading into the kinetics of the GolS system); online advertising of the project.
</P><P><B>Evaldas Klumbys</B>: dry lab work (idea refinement; MFC technology; mtr complex; peptide design); wet lab work (12 weeks on MTR complex cloning); organised Journal Clubs; organised problem-solving workshops; liason with academics; website (writing scientific content); website development (liason with IT team); prepared presentation for Lyon Jamboree.
</P><P><B>Jonas Kondratavicius</B>: dry lab (idea refinement; peptide design; DNA construct design); wet lab work (worked for 12 weeks on mtr complex, gold sensing device, peptide cloning and their characterisation); website (wrote scientific content and helped with development); logistics (procurement of materials for lab work).
</P><P><B>Rebecca Miles</B>: prepared team videos (for funding and method presentation); dry lab work (idea refinement; gold-sensing system); wet lab work (6 weeks on DNA cloning).
</P><P><B>Andra Necula</B>: : team founder; organised iGEM workshops and brainstorming sessions; sponsorship (promotional materials; departmental funding); dry lab work (gold sensing system); wet lab work (gave lab induction to the rest of the team); prepared posters; website (wrote part of the content); organised supervisory meetings and kept minutes.
</P><P><B>Nikola Panayotov</B>: sponsorship (contacted companies and funding bodies, prepared promotional materials); dry lab work (sending the iGEM Registry parts); wet lab work for 12 weeks (peptide cloning, part characterisation); outreach event organiser and participant.
</P><P><B>Caroline Pearson</B>: organising outreach activities; raising sponsorship (contacted pharma companies and applied for university funding); contributed to dry lab work (reading into the Quorum Sensing system).
</P><P><B>Rudolfs Petrovs</B>: raising sponsorship (applied for university funding); contributed to dry lab work (reading into MFC technology, peptide design and analysis methods); organised google group for us.
</P><P><B>Alexandro Rizzo</B>: team founder; organised iGEM workshops and brainstorming sessions; promoted project within the department; sponsorship (promotional materials; contacted companies; applied for departmental funding); dry lab work (peptide design); wet lab work (worked in the lab for 8 weeks, part characterisation); website (wrote non-scientific content); prepared posters.
</P><H3>University of York Support</H3><P>The University of York iGEM team would like to thank and acknowledge the support of the university, as follows:
<UL><LI>The York Annual Fund, the Rapid Response Fund, the Internship Bureau and the Department of Biology for their financial support;
</LI><LI>
The teaching labs of the Department of Biology, for providing the space for us to work in over the summer and in September. We thank Jennifer Lee, Nikki Begg and Nicola Charltonfor assistance and guidance with our laboratory work;
</LI><LI>
Phil Roberts for advice on graphic design and poster creation;
</LI><LI>
Hilary Jones for help with student recruitment and Lorna Warnock for financial advice;
</LI><LI>
The Department of Biology as a whole for providing materials and facilities for us.
</LI></UL></P><H3>Instructor Support</H3><P>
All research work was performed by the undergraduate students on our team. Our instructors contributed support and ideas to the students and facilitated a safe and efficient laboratory environment. Special thanks to Dr James Chong, Dr Gavin Thomas, Prof Maggie Smith and Dr James Edwards, whose experience in bacterial work proved very useful for us. We are grateful to them for support in obtaining sponsorship, for helping us obtain all materials needed, for providing chemicals and for technical guidance and scientific advice throughout the project. 
</P><H3>Support from Colleagues</H3><P>We thank our colleague Bruna Cama, for designing and drawing our logo and the iGEM York comic story;</P><P>We are very grateful to Oswin Wan for designing our website and to Mantas Markevicius and Paulius Imbrasas for coding the website.</P><P>We thank Jamie Birch for technical advice and troubleshooting support.</P><H3>Support from Additional Professors and University Staff</H3><P>
We would like to acknowledge other professors in the Department of Biology including Dr Sean Sweeney, Dr Setareh Chong, Dr Thorunn Helgason, Dr James Moir, Prof Peter McGlynn, and Dr Jamie Wood from the Department of Biology for interest in our project and advice along the way. Dr Maria Chechik for providing the DNA of Shewanella oneidensis and Dr Marjan van der Woude for the DNA of Salmonella enterica.
</P><P>
Additionally we would like to thank Dr Martin Bees and Viktor Chechik from the Chemistry Department for advice with designing the microbial fuel cell and Sindhu Krishna for providing the gold solution necessary for supplying our system with gold ions.
</P><H3>Outside of University Research and Technical Support</H3><P>
We would like to thank and acknowledge the support of various individuals from other Universities. This includes Dr Chris French (University of Edinburgh) for supplying the DNA for genes encoding the MTR complex; Swati Sureka (Cornell iGEM team 2012) for offering to provide a S. oneidensis strain; Prof Paolo Magni from Universita di Pavia, for offering to provide BioBrick BBa_K300000; Jon Veal (NanoSight) for kindly lending us the equipment required to characterise our gold nano particles; Peter Hong (Waterloo iGEM TEAM 2012) for kindly providing parts BBa_K241001 - K241004 and unsubmitted part phiC31 attB CT; Edward Perello (AutoClone) for allowing us to use the beta version of their software.
</P><H3>Outside of University Additional Support</H3><P>
We would like to thank the Microryza team for advice on obtaining funding; Catherine Brophy (STEM) and St Lawrence School for helping us prepare our outreach event and hosting us; St Anne’s Church in York for allowing us to organise an outreach event; Jordan Yaron and Paula Salgado for promotion on social media; Matt Ravenhall for his help with press releases.
</P></DIV></DIV></DIV><DIV class="printfooter">
Retrieved from &quot;<A href="http://2013.igem.org/Team:York_UK/Acknowledgments.html">http://2013.igem.org/Team:York_UK/Acknowledgments.html</A>&quot;</DIV></DIV></DIV><DIV id="footer-box" class="noprint"><DIV id="footer"><UL id="f-list"><LI id="t-recentchanges"><A href="/Special:RecentChanges" title="Recent changes">Recent changes</A></LI><LI id="t-whatlinkshere"><A href="/Special:WhatLinksHere/Team:York_UK/Acknowledgments.html" title="List of all wiki pages that link here [j]" accesskey="j">What links here</A></LI><LI id="t-recentchangeslinked"><A href="/Special:RecentChangesLinked/Team:York_UK/Acknowledgments.html" title="Recent changes in pages linked from this page [k]" accesskey="k">Related changes</A></LI><LI id="t-specialpages"><A href="/Special:SpecialPages" title="List of all special pages [q]" accesskey="q">Special pages</A></LI><LI><A href="/Special:Preferences">My preferences</A></LI></UL></DIV><DIV id="footer"><UL id="f-list"><LI id="t-print"><A href="/wiki/index.php?title=Team:York_UK/Acknowledgments.html&amp;printable=yes" title="Printable version of this page [p]" accesskey="p">Printable version</A></LI><LI id="t-permalink"><A href="/wiki/index.php?title=Team:York_UK/Acknowledgments.html&amp;oldid=303035" title="Permanent link to this revision of the page">Permanent link</A></LI><LI id="privacy"><A href="/2013.igem.org:Privacy_policy" title="2013.igem.org:Privacy policy">Privacy policy</A></LI><LI id="disclaimer"><A href="/2013.igem.org:General_disclaimer" title="2013.igem.org:General disclaimer">Disclaimers</A></LI></UL></DIV></DIV></DIV></BODY></HTML>