<HTML xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en" dir="ltr">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki  ltr ns-0 ns-subject page-Team_York_UK_Notebook_html"><DIV id="globalWrapper"><DIV id="top-section"><DIV id="p-logo"><A href="/Main_Page" title="Main Page">&quot;
	    </A></DIV><DIV id="menubar" class="left-menu noprint"><UL><LI class="selected"><A href="/Team:York_UK/Notebook.html">Page               </A></LI><LI class="new"><A href="/wiki/index.php?title=Talk:Team:York_UK/Notebook.html&amp;action=edit&amp;redlink=1">Discussion               </A></LI><LI><A href="/wiki/index.php?title=Team:York_UK/Notebook.html&amp;action=edit">View source               </A></LI><LI><A href="/wiki/index.php?title=Team:York_UK/Notebook.html&amp;action=history">History               </A></LI><LI style="color:white;cursor:default">teams</LI></UL></DIV><DIV class="right-menu noprint" id="menubar"><UL><LI id="pt-login"><A href="/wiki/index.php?title=Special:UserLogin&amp;returnto=Team:York_UK/Notebook.html" title="You are encouraged to log in; however, it is not mandatory [o]" accesskey="o">Log in</A></LI></UL></DIV><DIV id="search-controls" class="noprint"><FORM action="/Special:Search" id="searchform"> </FORM></DIV></DIV><DIV id="content"><H1 class="firstHeading">Team:York UK/Notebook.html</H1><DIV id="bodyContent"><H3 id="siteSub" class="noprint">From 2013.igem.org</H3><P><TITLE>iGEM York</TITLE></P><DIV id="header"><DIV id="navigation"><UL><LI><A href="https://2013.igem.org/Team:York_UK">Home</A></LI><LI><A href="https://2013.igem.org/Team:York_UK/Team.html">Team</A></LI><LI><A href="https://2013.igem.org/Team:York_UK/Project.html">Project</A></LI><LI><A href="https://2013.igem.org/Team:York_UK/Notebook.html">Notebook</A></LI><LI><A href="https://2013.igem.org/Team:York_UK/Outreach.html">Outreach</A></LI><LI id="acknowledgements"><A href="https://2013.igem.org/Team:York_UK/Acknowledgments.html">Acknowledgments</A></LI></UL></DIV><DIV id="nav2"><DIV id="team"><A href="https://2013.igem.org/Team:York_UK/Team.html?page=contacts">Contacts</A><A href="https://2013.igem.org/Team:York_UK/Team.html?page=university">The University</A><A href="https://2013.igem.org/Team:York_UK/Team.html?page=biography">Biography</A><A href="https://igem.org/Team.cgi?year=2013&amp;team_name=York_UK">Profile</A><A href="https://2013.igem.org/Team:York_UK/Team.html?page=overview">Overview</A></DIV><DIV id="project"><A href="https://2013.igem.org/Team:York_UK/Project.html?page=safety">Safety</A><A href="https://2013.igem.org/Team:York_UK/Project.html?page=gallery">Gallery</A><A href="https://2013.igem.org/Team:York_UK/Project.html?page=collaborations">Collaborations</A><A href="https://2013.igem.org/Team:York_UK/Project.html?page=achievements">Achievements</A><A href="https://2013.igem.org/Team:York_UK/Project.html?page=microbial">Microbial Fuel Cells</A><A href="https://2013.igem.org/Team:York_UK/Project.html?page=results">Results</A><A href="https://2013.igem.org/Team:York_UK/Project.html?page=parts">Parts</A><A href="https://2013.igem.org/Team:York_UK/Project.html?page=overview">Overview</A></DIV><DIV id="notebook"><A href="https://2013.igem.org/Team:York_UK/Notebook.html?page=modelling">Modelling</A><A href="https://2013.igem.org/Team:York_UK/Notebook.html?page=protocols">Protocols</A><A href="https://2013.igem.org/Team:York_UK/Notebook.html?page=labbook">Kobchai's Lab Book</A><A href="https://2013.igem.org/Team:York_UK/Notebook.html?page=wetlab">Jonas' Lab Book</A><A href="https://2013.igem.org/Team:York_UK/Notebook.html?page=drylab">Dry Lab</A></DIV><DIV id="outreach"><A href="https://2013.igem.org/Team:York_UK/Outreach.html?page=london">YSB 1.0</A><A href="https://2013.igem.org/Team:York_UK/Outreach.html?page=alumni">Alumni Event</A><A href="https://2013.igem.org/Team:York_UK/Outreach.html?page=lawrence">St. Lawrence's School</A><A href="https://2013.igem.org/Team:York_UK/Outreach.html?page=helen">St. Helen's Church</A></DIV><DIV id="acknowledgments"><A href="https://2013.igem.org/Team:York_UK/Acknowledgments.html?page=sponsors">Sponsors</A><A href="https://2013.igem.org/Team:York_UK/Acknowledgments.html?page=references">References</A><A href="https://2013.igem.org/Team:York_UK/Acknowledgments.html?page=attributions">Attributions</A></DIV></DIV></DIV><TITLE>iGEM York Team</TITLE><DIV id="container"><DIV id="navlocal"><UL><LI>Notebook</LI><LI><DIV id="drylab"><A href="">Dry Lab</A></DIV></LI><LI><DIV id="wetlab"><A href="">Jonas' Lab Book</A></DIV></LI><LI><DIV id="labbook"><A href="">Kobchai's Lab Book</A></DIV></LI><LI><DIV id="niklabbook"><A href="">Nikola's Lab Book</A></DIV></LI><LI><DIV id="protocols"><A href="">Protocols</A></DIV></LI><LI><DIV id="modelling"><A href="">Modelling</A></DIV></LI></UL></DIV><DIV id="conlocal"><DIV id="wetlabcontent" class="hiddenContent"><H2>July 10th</H2><P>Purification of genomic DNA from <I>Shewanella oneidensis</I> and <I>Salmonella sp.</I></P><H2>July 25th</H2><P>After a long break we finally started working on the actual assembly of parts - the mtr complex. Today Jonas ran a trial PCR on mtrB and mtrC part of the complex both of which are around the size of 2kb. GoTaq polymerase (Promega) was used, this is the polymerase we generally use for trials and colony PCRs as fidelity is not the main priority.</P><P>This is the gel from the PCR. Temperatures ranged 46-71˚C. It appears that the primers are sufficiently specific at even low temperatures. 1% agarose, SYBR-Safe(Invitrogen), <A href="http://www.york-bio.com/QStep4.jpg">QStep4 ladder.</A></P><H2>July 26th</H2><P>Trial PCR again, this time for three small (~500bp) synthetic DNA fragments - gBlocks (IDT), these fragments are highlighted below on the diagram as well as the overall view of our mtr gene. </P><P>The gel. Three different temperatures (51-61˚C)- all work. 1% agarose dyed with SYBR-Safe, however, due to poor practice the ladder is indistinguishable.</P><P>At this point Jonas arrived at the realisation that as long as the primers are specific enough, there is no point in doing PCR trials (silly Jonas). Therefore most of the PCRs done this day forth use a universal 58˚C temperature.</P><H2>July 27th</H2><P>This is the first day that we actually got some positive outcome from our cloning. Or so we thought.</P><P>All the fragments of our mtr complex were once again amplified via PCR but this time using Phusion polymerase for high fidelity (Thermo Fisher). Gel confirmed that everything went fine and specifically. Using QIAquick PCR purification kit (Qiagen) we clean up the PCR product, then measure the concentrations using NanoDrop1000, do the required calculations and then proceed to the main reaction:</P><P>In 60 minutes at 50 ˚C we joined all of the 5 fragments using the Gibson Assembly Master Mix provided by NEB. Confirmed by a PCR.</P><P>The PCR was not fully specific but it is unclear whether these fragments formed during Gibson reaction (main fragment joining sites are identical RBS) or because the temperature used was too low (51˚C &amp; 58˚C). 3rd and 4th lanes are negative controls. 1% agarose dyed with SYBR-Safe, <A href="http://www.york-bio.com/QStep4.jpg">QStep4 ladder.</A></P><H2>Before August 11th</H2><P>During this time we tried putting all of our genes inside the pSB1C3 iGEM plasmid but with little luck. Everything seemed to be working fine up until the point when we realised that colony PCRs are not so trustworthy. It appeared clear that most of the colony PCRs came out as false positives after we decided to run the PCR on already purified plasmids. However, some samples were already on their way for sequencing (thinking that we actually got something within our pSB1C3 backbones we sent them). The sequencing results gave us an answer: we planned it all wrong! Graphical summary:</P><P>So all of our parts that were to be put inside the iGEM plasmid only had XbaI and SpeI cutting sites, but we were unaware that they actually had complementary sticky ends and it was nearly impossible to ligate something in between.</P><P>We fixed it with new primers which add the remaining part of the prefix and suffix. However, we lost 2 weeks worth of productive labwork.</P><H2>August 12th</H2><P>So after receiving the new primers we continue working. This time we're using Gibson's assembly to directly insert our mtr part into pSB1C3, this allows us to skip any problems that might occur during digestion or ligation.Note that during design of these primers we decided to produce two different mtr parts, one of which would lack the promoter and the first RBS(crR12) which we thought might not be as convenient for other teams.</P><P>So today we tried amplifying mtrAB(from the previous Gibson), mtrC(from previous PCR, should have used the genome) and pSB1C3 backbone(the linearised plasmid from iGEM HQ). Unfortunately, the backbone PCR failed.</P><P>We thought that perhaps dimers of the primers formed as the sequences of prefix and suffix are not that different. 1% agarose dyed with SYBR-Safe, <A href="http://tools.lifetechnologies.com/content/sfs/gallery/high/2852.jpg">1kb ladder(Invitrogen)</A>(on the right).</P><P>However, after playing a bit with DMSO and annealing temperatures we were still unable to amplify it until we changed the template. We took a random iGEM part we found in the fridge and tried the PCR on it and it worked. Conclusion: iGEM HQ should provide more information about their plasmid linearisation and how were the ends modified.</P><H2>August 13th</H2><P>2nd and 3rd Gibson Assembly. So we purified the assembly parts from previous PCR and did the calculations to maximise the amount of product. This reaction was also relatively simple as we reduced the number of fragments to 3 and it results in already circular plasmid.</P><P> So while we do the transformations, spending some usual late time in the labs, we also did a PCR using VF2 and VR primers. The gel confirms that reactions were successful. We can see the bands just above the 5kb mark,and as there is only about 130bp difference between these distinct mtr constructs it is impossible to see that on the gel. 1% agarose dyed with SYBR-Safe, <A href="http://tools.lifetechnologies.com/content/sfs/gallery/high/2852.jpg">1kb ladder(Invitrogen)</A>(on the right).</P><H2>August 14th</H2><P>So after having some sleep we head back to the lab and do the colony PCR on our transformations. But because Jonas is a very lazy person he used Phusion polymerase (which is as much as 4 times faster than GoTaq) and failed - there is nothing visible on the gel apart from ladder. It is not the first time this happens with Phusion colony PCR, so we try to avoid doing it, it appears though that Phusion requires relatively pure DNA template.</P><P> Nonetheless, we inoculate 2 cultures each of both mtr ORFs only and mtr+promoter constructs and grow them overnight. But it is not all for the day.</P><P>Also, today we got the new primers for the rest of the cloning, so we amplified the sequences of gold peptides and gold sensing device adding the rest of prefix and suffix. It all worked but the gel is really really hideous thus it will not appear up in here. We leave the rest for tomorrow.</P><H2>August 15th</H2><P>We extract the plasmids from overnight cultures using NucleoSpin Plasmid(Macherey-Nagel) and digest them with ThermoFisher's FastDigest enzymes:</P><P> Every triplet is: 1st the control, 2nd PstI digest and 3rd the EcoRI &amp; SpeI digest. It is obvious that there is nothing near the size of 5kb visible. However, the 1kb fragment that is visible in double digest shows us that it is actually the template we used for pSB1C3 backbone amplification. 1% agarose dyed with SYBR-Safe, <A href="http://tools.lifetechnologies.com/content/sfs/gallery/high/2852.jpg">1kb ladder(Invitrogen)</A>(on the right).</P><P>So the solution is simple: we digest the Gibson reactions we had with DpnI(Promega) thus removing the background and then repeat the transformations.</P><P>At the same time we continued working with other sequences. Amplified gBlocks (peptides,the gold device) were purified with QIAquick kit, digested with EcoRI &amp; PstI and then purified again using the same kit - this step frees our samples of small 2-6bp long fragments making the ligation easier. We also digest the linearised plasmid from iGEM HQ with the same enzymes and purify it as well. Even though this leads us to a very low concentration of pSB1C3, it is still enough for transformations as long as there are no problems with ligations. So we then make the reactions with T4 Ligase(Thermo Fisher) and proceed with the transformations.</P><H2>August 16th</H2><P>The day of GoTaq colony PCRs. So basically Gintare marched to the lab to do PCR of 44 colonies (and we are all aware of how annoying it is). Results were pretty straightforward with the peptides and gold sensing device parts. On the left gel below first four are <A href="http://parts.igem.org/wiki/index.php?title=Part:BBa_K1127000">BBa_K1127000</A>, then <A href="http://parts.igem.org/wiki/index.php?title=Part:BBa_K1127001">BBa_K1127001</A> and <A href="http://parts.igem.org/wiki/index.php?title=Part:BBa_K1127003">BBa_K1127003</A>. On the second gel: <A href="http://parts.igem.org/wiki/index.php?title=Part:BBa_K1127002">BBa_K1127002</A>, followed by two constituents (which we join together later) of <A href="http://parts.igem.org/wiki/index.php?title=Part:BBa_K1127008">BBa_K1127008</A> and then <A href="http://parts.igem.org/wiki/index.php?title=Part:BBa_K1127010">BBa_K1127010</A> which was not submitted.
Two colonies of each were cultivated overnight.</P><P>However, the results for mtr complexes were weird. So we picked a few cultures and grew them overnight for further analysis.</P><P>All gels were 1% agarose dyed with SYBR-Safe, <A href="http://tools.lifetechnologies.com/content/sfs/gallery/high/2852.jpg">1kb ladder(Invitrogen)</A>(on the right). We also noticed that something was not right with our gel viewing box as the images turned out to be very dark even at max exposure.</P><H2>August 17th</H2><P>Another busy day, so firstly about mtr complex. We purified the plasmids from yesterdays colonies using our usual NucleoSpin Plasmid kit and then digest them with FastDigest enzymes:</P><P>Triplet consists of: 1. Control 2. PstI digest 3. EcoRI &amp; SpeI digest. First three triplets are of the mtr ORFs only construct and the last one (a lot of it!) is the one with promoter. Even though the gel is faint we can see the most important thing: double digests separate the pSB1C3(2kb) backbone and our part (5.3kb). The PstI digest also confirms the prohibited PstI cutting site within mtrC sequence (present in the genome). 1% agarose dyed with SYBR-Safe, <A href="http://tools.lifetechnologies.com/content/sfs/gallery/high/2852.jpg">1kb ladder(Invitrogen)</A>(on the right).</P><P>Using the primers we designed earlier we do the Site-Directed Mutagenesis on each of the mtr constructs, which should remove the PstI cutting site from where it should not be present but still retaining the aminoacid sequence. We also finish it up with DpnI digestion to remove any background and then transform the cells.</P><P>In the meantime, we did similar to mtr digestions on the peptide parts we cloned earlier. Again on NucleoSpin purified plasmids with FastDigest enzymes.</P><P>Each plasmid was ran on the gel in triplets: 1. uncut control 2. XbaI &amp; SpeI digest 3. EcoRI &amp; PstI digest. Both gels show the same peptides mentioned earlier from different colonies picked yesterday. The gel shows us that the peptides are actually there within the pSB1C3 with working prefixes and suffixes (there was only a slight nuisance with the 3rd peptide). 1% agarose gels dyed with either ethidium bromide(left) or SYBR-Safe(right), <A href="http://www.york-bio.com/QStep4.jpg">QStep4 ladder.</A></P><H2>August 18th</H2><P>Today we do colony PCR (yet again!) on our SDM'd plasmid. A clever way we thought of to check if PstI cutting site is still there was to amplify the mtrC part of the colony and digest it immediately with PstI:</P><P> Although it was not so cleverly thought to the end: Jonas forgot the control for mtrC before the SDM. Basically, the first pair is the control on the SDM reagent from yesterday, the rest of them are our mtr constructs. First member of the pair is undigested product and the other one is PstI digested product. We can see the the bands are not at the equal sizes, however, that's an optical illusion caused by different volumes added to the lanes (yes, we checked under the manual UV lamp). 1% agarose dyed with SYBR-Safe(right), <A href="http://www.york-bio.com/QStep4.jpg">QStep4 ladder</A>(although irrelevant).</P><P> We also tried joining the two mentioned parts to produce <A href="http://parts.igem.org/wiki/index.php?title=Part:BBa_K1127008">BBa_K1127008</A> using <A href="http://www.jbioleng.org/content/5/1/17">Amplified Insert Assembly</A>.</P><H2>August 19th</H2><P>And it did not work for us.</P><P>This is the colony PCR with VF2&amp;VR primers and all the bands are clearly lower than the expected 900bp size, it actually fits the size of our recipient plasmid. 1% agarose dyed with SYBR-Safe, <A href="http://www.bioline.com/Images/HyperLadder/HyperLadder_1kb_550.jpg">Hyper ladder 1kb(Bioline).</A></P><P>We spent the remaining time designing primers for the sequencing of whole mtr complex.</P><H2>August 22nd</H2><P>After trying a few other techniques and failures of joining our two fragments together (note that they are now in biobrick format) we decided to test something new. Basically we joined the parts together before putting them inside the plasmid:</P><H2>August 23rd</H2><P>We select two random colonies from yesterday transformations and grow them overnight.</P><P>Also, we receive sequencing primers for mtr and send the samples to <A href="http://www.lifesciences.sourcebioscience.com/">Source BioScience.</A></P><H2>August 24th</H2><P>Purified plasmids from cultures are cut to confirm that we have the gold sensing particle ready.</P><P>Left and right of the marker are two plasmids: first the uncut control, XbaI digest, SpeI digest, EcoRI&amp;PstI double digest followed by SanDI digest. Double digest reveals the ~750bp long fragment which proves our method successful. 1% agarose dyed with SYBR-Safe, <A href="http://www.bioline.com/Images/HyperLadder/HyperLadder_1kb_550.jpg">Hyper ladder 1kb(Bioline).</A></P><H2>August 27th</H2><P>We receive our first sequencing results of mtr complexes. Unfortunately, we found a single base pair substitution at the very start of mtrA (changing the aminoacid lysine to glutamic acid) and a nonsense mutation in mtrC. However, the mtr construct with promoter contained an additional nonsense mutation within mtrA, this is the point where we decided to give up on this one and try and fix the other, more sensible biobrick.</P><P>So we ignore the single base pair substitution (as it is not part of the protein motive compared with other mtrA analogs) and order new set of primers for the Site-Directed Mutagenesis of mtrC.</P><H2>August 30th</H2><P>We receive the primers for mtr SDM from IDT and do it immediately the same way we did it last time.</P><P>Samples for sequencing of the peptides and gold sensing constructs were sent today.</P><H2>September 4th</H2><P>We do the β-galactosidase assay on the gold sensing device for which the results can be found elsewhere.</P><P>The mtr part was sent for sequecing again to see if the SDM was successful.</P><H2>September 10th</H2><P>Finally, DNA of the parts for submission is prepared and is finally sent to iGEM HQ on September 16th.</P></DIV><DIV id="drylabcontent" class="hiddenContent"><H2>February 21st</H2><P>Today the iGEM team divided to discuss about the two main ideas, chosen from over 100, after meeting with potential supervisors. </P><P>The first idea is making a mouthwash which contains harmless, live bacteria which normally reside in the mouth. These bacteria will be genetically engineered to secrete pleasant odours (e.g. menthol, rose, banana  - parts for which have already been submitted to the iGEM registry). The scope is to have the bacteria colonise the mouth; to prevent the smell being secreted at all times, their production could be controlled by controlling gene expression, depending on external signals (such as concentration of oxygen, glucose or stress). </P><P>Becca, Evaldas, Andy, Jonas, Andra, Hannah and Caroline looked into the idea in more depth. One of the major concerns is that safety issues would prevent the mouthwash from being regarded as a product which can be used, since it involves introducing live, genetically modified bacteria into humans. Also, the idea is not conceptually very challenging.</P><P>The second idea is to create an improved microbial fuel cell, a type of battery that uses live bacteria for current production. The bacteria releases H ions and electrons into the environment, when encapsulated in an anode chamber. These secreted particles contribute to energy formation, when electrons pass from the anode to the cathode of the battery. We would like to genetically engineer <I>E.coli</I> to secrete more electrons, by introducing the secretion system present in a different organism, <I>Shewanella oneidensis.</I> This has been attempted in iGEM before, but never achieved. In addition, we would try to deposit gold nanoparticles on the anode of the battery, to increase its conductivity. </P><P>Ivan, Kobchai, Nick, Ric, Gintare, Rudolfs and Alex looked into this idea. The main challenges are that, although nanoparticles have been created before, we are unsure which genes we would use for this process. More reading through literature is needed to identify them, before carrying on.</P><H2>February 27th</H2><P>Each subteam presented their latest findings for the two ideas. Kobchai went to discuss with people from the engineering department, who gave useful insight into how nanoparticles are made in vivo and which organisms and gene could be used. We found out that <I>Salmonella enterica</I> has a good gold-detection system, which we could use to control gene expression in the presence of gold ions. We have also identified a few papers which describe how bacteria secrete nanoparticles. We decided this would be a good starting point to carry on with the gold idea.</P><P>We have given ourselves two weeks to divide work into subgroups and do reading on several issues; 1st year students will be grouped with older students, to ensure they receive support in interpreting data and understanding the articles. To do list:
		        ● what genes would be involved in electron secretion?
		        ● what method of nanoparticle formation would we use?
		        ● how would we control gene expression?
		        ● are there any parts in the iGEM Registry we could use?</P><H2>March 13th</H2><P>Ivan gave a comprehensive presentation on our findings so far, in front of potential supervisors. We have established that the main objective of our project is to improve existing microbial fuel cell technology and make fuel cells more cost effective. The three aims which lead to this are:
		        1) Expressing the MTR protein complex from Shewanella on the surface of <I>E.coli</I> for electron transport;
		        2) Creating a MFC of approx 3cm diameter and engineer <I>E.coli</I> to deposit gold nanoparticles on the anode;
		        3) Use a quorum sensing system to control gene expression of genes involved in gold deposition.</P><P>Ideas which emerged during the meeting concerned the following:
		        ● would the electron transporter introduced affect bacterial metabolism?
		        ● would the bacteria form biofilms on the anode? Could this be avoided by expressing pilli?
		        ● what is the gold concetration needed and where would we take it from? Possible  solution: pharmaceutical waste rich in gold ions, which would also allow the safe removal of toxic ions from the environment
		        ● other metals have been discussed, as alternatives to gold, such as copper and silver. However, there are already genes which are involved in copper metabolism in bacteria, so their effects might interfere with our system.
		        ● two bacterial strains could be used, living together inside the MFC, one for the electron secretion and one for the quorum sensing</P><H2>May 1st</H2><P>After the spring holiday, during which the team has worked from  home on iGEM, we had a n important  meeting with our supervisors.</P><P>We are considering using a peptide which can bind elemental gold, such as MIDAS, to create nanoparticles. We have discussed how to ensure the nanoparticles are deposited on the anode. There were several variants proposed:
		        ● fuse the peptide to secreted protein and let the complex settle on the anode (by gravity or magnetism)
		        ● fuse the peptide to surface proteins (such as OpmA) and then as above
		        ● turn the whole chamber into a graphyte anode, to increase the chance of bacteria being trapped in the porous material and release nanoparticles there
		        ● lyse bacteria once they have captured gold ions internally, to release the peptide, and then repopolate the MFC with the second bacterial strain, which secretes the MTR complex
		        ● aqua regia was proposed as a source of gold ions for the initial stages of the project, as an alternative to pharmaceutical waste; the composition of the latter may vary between experiemtns and will prevent us from making controlled, accurate measurements between replicate experiments</P><H2>May 15th</H2><P>During today's meeting we discussed more the details of the peptides we want to create. We have looked into several secretion tags which we would like to fuse to the peptide. We have narrowed it down to two, for now. HlyA can transport small peptides outside bacteria (though our initial in silico simulations show the peptidic construct would not be very stable). We might need a special <I>E.coli </I>strain if we work with it. pelB is good for small peptides, easy to use and we have an available vector for it. Another option is to have a poly-peptide, made with the A3, nanoparticle-producing peptide, rather than MIDAS. This poly-A3 would be secreted and cleaved outside of the bacteria; this method would give more stability to the system.</P><P><B>Need to look more into:</B>        ● Physiology of bacteria in the MFC
		        ● What we will compare (e.g. two MFC, one improved and one normal)
		        ● We need to design a good, simple end-assay, which will allow us to test the effect of our peptide on electricity production even if the rest of the system does not work/is not ready yet (e.g., he suggests putting the peptide in <I>Shewanella</I>, if we're not ready closer to the finish, and show it produces NP in <I>Shewanella</I>).</P><H2>June 5th</H2><P>After exams, we divided into four groups and started doing extensive reading on genes, to start preparing the gene sequences we need to order. Ric, Hannah, Becca and Andra are looking into the gold sensing system existing in <I>Salmonella</I>, GolTS. Alex, Evaldas and Jonas are looking into peptide design and secretion. Ivan, Nick and Andy are looking into modelling and identifying genes in the iGEM Registry. Kobchai, Caroline and Rudolfs are looking into the quorum sensing system and system regulation as a whole.</P><P>Notable things so far: some of the sequences for the pGolTS system submitted by other teams to the Registry seem to be wrong; we need to look more into it during the following week.</P><P>We might use some of the parts from team Edinburgh, which were not submitted last year, but which they can send to us, for the mtr complex. </P><H2>June 12th</H2><P>Evaldas, Kobchai and Andra gave a presentation on what has been done so far.  Andy talked about the modelling he worked on, using the kinetic values obtained from scientific papers. . We need to look more into the details of the MFC we are planning on making, in particular what buffers we are planning on keeping the bacteria in,  whether we will grow the bacteria in an aerobic or anaerobic environment and what bacterial strains to choose.</P><H2>June 19th</H2><P>The team spent the week designing some of the constructs, but also looking into the details of the MFC construction. We came together to make some decisions regarding the latter. We have established that, to allow the bacteria to create biofilms, it would be better to grow them in an anaerobic environment in the final MFC. However, for the initial experiments, we will use an aerobic environment. The two bacterial strains we will use are K12 and W3110, as they have been used with MFC before, are well characterised and easily available. </P><P>We also met Prof. Smith, to discuss about methods of isolating the peptide once it is produced, for part characterisation. We chose to put a flag tag on the peptide, to be identified with antibodies by western blot.</P><H2>June 26th</H2><P>During our supervisory meeting we talked about which vectors are best to be used for our constructs, and which constructs should be put on what vectors. We have discussed the advantages of pET vs PUC vectors and other possibilities proposed. We decided to go with the 3 vectors of the pET duet, which have different antibiotic resistance and have compatible origins of replication. We would use one for the gold recognition, one for mtr and one for the quorum sensing. </P><P>We also showed the supervisors our defined sequences, hoping we will soon order the sequences. We presented the techniques we are planning to use for every part, to characterise it (beta galactosidase assays). Also, the methods of assembly were presented to the supervisors (amplified insert assembly, which they were unfamiliar with, and the formerly proposed Gibson assembly). We received positive feedback on our choices.</P><P>An issue was raised regarding the way in which we would isolate peptides from cells, to assess their presence (since they are too small to be run on a gel). We chose to perfor, pull down assays. Defining sequences. To be expanded. </P><H2>July 2nd</H2><P>We had a small discussion with Dr. Thomas, about ordering the sequences. We went over all the designed sequences, verified restriction sites and are now preparing to submit the online order for the sequences. We have received the ordered consumables and are now ready to start working in the lab. Our meetings from now on will be administrative meetings and troubleshooting meetings. </P><H2>July 16th</H2><P>Internal meeting to redistribute the work within the team</P><H2>July 24th</H2><P>Final meeting with Dr. Chong before he leaves for two weeks. Catch up with supervisors on how lab work is going.</P></DIV><DIV id="protocolscontent" class="hiddenContent"><H2>Rapid Assembly of Biobricks (RAB)</H2><P>We found this easy way of joining two or more biobricks together:</P><P>1. Amplify the parts using VF2&amp;VR primers (although specific primers make it easier).
	2. Digest one product with SpeI and the other one with XbaI. 
	3. We would usually gel extract the fragments we need, which makes the next step easier.
	4. Ligate the two fragments together.
	5. Now amplify with PCR again the whole fragment that should have formed.
	6. You can continue joining more parts together by repeating the above steps.
	7. Finally, insert your new part into the iGEM plasmid using the standard procedure.
	8. Transform.</P><P>Now many would argue that this continuous PCR would result in lots of mutations. However, polymerases with already high fidelity are being continuously improved to the point where their mutation rate is hard to detect statistically. Also, we sequence the parts anyway, so there is high potential in this assembly. We successfully joined our gold sensing device using this protocol. If you feel confused about the procedure, perhaps this picture will help to clarify the mechanism:</P><H2>Beta galactosidase assay for gold induction</H2><P><B>Growth:</B>1. Transfer one colony to LB with antibiotics and grow overnight at +37 °C with shaking.
	2. Next morning transfer bacteria to fresh LB with antibiotics up to OD600=0.1 and continue growing until exponential stage (OD600=0.6-0.8).
	3. Induce with required concentration of AuCl4 and continue growing at 30 °C or 37 °C depending on the experiment.
	4. Samples for measurements can be taken at time points or after overnight growth.</P><P><B>Measurements:</B>1. Take 200 µl of cell suspension and measure OD630 (due to machine that we used instead of OD600) in a 96 well plate.
	2. Take 100 µl of cell suspension for beta gal assay and mix with 100ul Z-buffer.
	3. Add 10 µl 0.1% SDS and 20 µl chloroform and incubate at 30 °C for 5 min to disrupt the cells.
	4. Add 50 µl 5mM PNPG and incubate at 30 °C for 30 min (incubation times can be extended up to 1h if the colour is not appearing, however the same incubation time should be applied for all of the related samples).
	5. To stop the reaction and increase the intensity of yellow colour add 100 µl of 1 M Na2CO3 and centrifuge and 13k rpm for 5 min.
	6. Measure the absorbance at 405 nm.
	7. The relative response (∅) was calculated according to the formula:
	</P><P><B>Reagents:</B></P><P>Z-buffer stock solution:
	4.27 g Na2HPO4
	2.75 g NaH2PO x 4H2O
	0.375 g KCl
	0.125 g MgSO4 x 7H2O
	Adjust to pH 7.0.
	Bring to 500 ml with dH2O. Do not autoclave. Store at 4°C.</P><P>For complete Z-buffer - Prior to daily use mix:
	50 ml Z-buffer
	0.14 ml ß-mercaptoethanol</P><P>1 M Na2CO3 (store at 4°C)
	5.3 g Na2CO3
	50 ml dH2O</P><H2>Bacterial growth media recipes:</H2><P><B>Luria-Bertani media</B></P><P>10g Tryptone
	5g Yeast Extract
	10g NaCl
	(15g Agar for solid media)
	dH2O to 1000ml. Autoclave.</P><P><B>M9 minimal media</B></P><P>6g NaHPO4
	3g KH2PO4
	0.5g NaCl
	Mili-Q water to 975ml. Autoclave.</P><P>Under sterile conditions:
	1ml 1000x vitamins
	1ml 1000x trace metals
	2ml 1M MgSO4 (autoclaved)
	0.2ml 1M CaCl2 (autoclaved)
	1ml 1000x antibiotic
	20ml 20% glucose (filtered via 0.2 micron filter)</P><H2>Preparation of chemically competent cells</H2><P><B>Solutions:</B></P><P>TFB1:
	30mM Potassium Acetate
	10mM CaCl2
	50mM MnCl2
	100mM RbCl
	15% glycerol
	Adjust pH to 5.8. Filter sterilise (0.2µm). Store at room temperature.</P><P>TFB2:
	10mM MOPS
	75mM CaCl2
	10mM RbCl
	15% glycerol
	Adjust pH to 6.5. Filter sterilise (0.2µm). Store at room temperature.</P><P><B>Notes:</B></P><P>Warm clothing is recommended for day 3.
	Prechill all equipment to be used in the cold room.
	When working from day 3 (step 4) keep the cells cold at all times and perform re-suspensions in the cold room. 
	Make sure to read the protocol thoroughly before starting.</P><P><B>Method:</B></P><P>1. Streak a non selective LB plate with a dilution of the previous batch of Competent Cells, incubate at 37°C overnight.
	2. The next day, inoculate a single colony from the plate in 2.5ml of LB broth. Incubate at 37°C overnight with shaking at approximately 225rpm.
	3. The next day, use the entire overnight culture to inoculate 250ml of LB medium containing 20mM MgSO4 (a 1/100 dilution of 2mM stock). Take 1ml of the culture and keep in a cuvette, as blank for OD measurement. Grow the cells in an autoclaved 1L flask until the OD600 reaches 0.4-0.6 (check after 2 hours then keep testing until within the range). A 1L flask is necessary for proper aeration and growth.
	4. Pellet the cells by centrifugation at 4500g for 5 minutes at 4°C. For a 250ml culture use two 250ml centrifuge bottles in a large rotor (eg. Beckman), or if not available use 50ml tubes in a bench top centrifuge at 3500rpm for 15 minutes at 4°C. Tubes need to be prechilled!
	5. Re-suspend the cell pellets in 0.4 volume (based on the original culture volume) of ice cold TFB1, using vortex, in the cold room. For a 250ml subculture, use 100ml of TFB1 (50ml/bottle). Combine the re-suspended cells in one bottle. For the remaining steps, keep the cells on ice and chill all pipettes, tubes and flasks.
	6. Incubate the re-suspended cells on ice for 5 minutes at 4°C.
	7. Pellet the cells by centrifugation as per step 4. When transporting cells to centrifuge, keep them on ice.
	8. Re-suspend the cells in 1/25 of the original culture volume of ice cold TFB2, using vortex, in the cold room. For a 250ml subculture use 10ml of TFB2.
	9. Incubate the cells on ice for 60 minutes and then aliquot 100µl/ Eppendorf tube for storage at -80°C.
	10.	Quick freeze the tubes in a dry ice/isopropanol bath (or liquid nitrogen). Cells should be stable for between 3 months to 1 year.</P><P><B>Transformation:</B></P><P>1. Thaw the competent cells on ice. Use 50µl per transformation. 
	2. Gently mix the cells with up to 5µl of DNA without pipetting up and down.
	3. Incubate on ice for 20 minutes.
	4. Heat shock the cells at 42°C for 45s.
	5. Place the cells on ice for 3-5 minutes.
	6. Add 200-950µl of room temperature LB or SOC media.
	7. Incubate the cells at 37°C whilst shaking them at 225rpm for 1hour (2 hours if plasmid contains chloramphenicol resistance).
	8. Spread up to 200µl on the selection plates and incubate at 37°C overnight.</P><H2>Colony PCR</H2><P>1. Combine the master mix for PCR on ice (20µl reactions are sufficient).
	2. Dip the pipet tip to the selected colony and now either:
	a) Pipet up and down directly into PCR mix aliquot and store the tip inside eppie in the fridge, or
	b) Pipet the bacteria into 10µl of H2O and pipet 1µl into the reaction, storing the rest in the fridge
	3.Continue the PCR as normal and run the gel, you can then grow the selected cultures from the bacteria you saved in the fridge.</P><H2>Site-Directed Mutagenesis</H2><P>1. Design the primers with mutation that meet end to end (there is software for that) and have them synthesized 5' phosphorylated (SDS-PAGE purification recommended).
	2. Perform a PCR as usual with high fidelity polymerase.
	3. Recircularise the amplified plasmid with DNA ligase.
	4. Digest with DpnI. This is optional if you use little amounts of template (as low as 10pg) and are going to send 3 or more samples for sequencing.
	5. Transform.</P></DIV><DIV id="modellingcontent" class="hiddenContent"><H2>Models for the gold sensing device <A href="http://parts.igem.org/Part:BBa_K1127008">(BBa_K1127008)</A></H2><P>Solutions to this problem are relatively simple. The law of mass action which is widely used in biochemistry, physiology and pharmacology is one of the straightforward approaches. However, it requires a massive amount of data at different points in time. This is very tricky to maintain accuracy and to spend less resource.  Fitting pre-existing models in our data came to us as a plausible solution. So we measured activities of the device when it reached steady states. Data were collected and fit into the nonlinear models for comparison.</P><P>We computed information Akaike Information Criterion (AIC) to determine quality of the models. Regardless of values of the goodness of fit, Logistic is the best model among the three. So model parameters of the Logistic are estimated and shown here.</P><P>The model is simulated and compared with the experimental data. As seen in the Figure 1, Logistic is well fit and could be a good reflection of reality.</P><P>The model is extended to include diffusion of the gold. 'e' is the diffusion constant. In the other word, it is the permeability of the bacterial envelope for the ion Au(III). We solved the differential equations and produced simulations to predict behaviour of the system in various conditions. </P><P>The model predicts multiple steady. As seen in the Figure 2, the system could reach higher steady states with increasing gold concentration.</P><P>In Figure 3, the system is fairly robust. It could approach the same steady state despite of differences in the diffusion constants.</P><P>For the sake of curiosity, we include expression of the synthetic peptide for gold bio-mineralization. Extension of the model is shown below.</P><P>Literature about this peptide is scarce, so we make following assumptions:
			1. is the change in expression of the peptide;
			2. Gold bio-mineralization is a linear process with the constant rate 'f';
			3. Secretion of the peptide is in the form of standard Hill equation  with Hill coefficient=1;
			4. The peptide is degraded through a linear process i[peptide]<SUB>out</SUB>.</P><P>The model is simulated and shown in Figure 4. It suggests that the system could undergo the decline phase.  The rates of reduction could vary depending on the rates of bio-mineralization.</P><P>Our models can't produce precise numerical solutions of the real world problems. However, they have capabilities to describe structures and to generate broad outcomes to what might happen in certain conditions. This qualitative approach is actually advantageous as it provides  guidance for experimental design and data analysis without overloading us with unnecessary information.</P><H2>R Source code</H2><PRE><CODE>#Model fitting
library(minpack.lm)
library(bbmle)
library(plyr)
library(ggplot2)

#Observed data between 0uM and 10uM
x&lt;-c(0,0,0,0.5,0.5,0.5,1,1,1,2.5,2.5,2.5,5,5,5,7.5,7.5,7.5,10,10,10)
y&lt;-c(0.002857143,0.00152381,0.00212963,0.011598174,0.009054726,0.011746032,0.021904762,0.00854902,0.016464646,0.038600823,
0.043166667,0.043508772,0.045142857,0.045367965,0.035365854,0.043474178,0.044122807,0.04971831,0.03822807,0.045063291,
0.043288889)
#0.043508772&lt;=0.053508772, 0.04971831&lt;=0.05971831, 0.03822807&lt;=0.033?
df&lt;-data.frame(x, y)

#HILL FUNCTION 
Hill&lt;-nlsLM(y~((a+b*x^d)/(c+x^d)), start=list(a=0.002, b=0.044, c=1, d=2), trace=TRUE)

#LOGISYIC FUNCTION 
Logt&lt;-nlsLM(y~(a+b/(1+exp(4*c*(d-x)/b+2))), start=list(a=0.001, b=0.044, c=0.025, d=0.1), trace=TRUE)

#GOMPERTZ FUNCTION 
Gomp&lt;-nlsLM(y~(a+b*exp(-exp(c*2.71828*(d-x)/b+1))), start=list(a=0.001, b=0.044, c=0.025, d=0.1), trace=TRUE)


AICtab(Hill, Logt, Gomp)
#     dAIC df
#Logt 0.0  5 
#Gomp 1.8  5 
#Hill 3.6  5 


summary(Logt)
#Formula: y ~ (a + b/(1 + exp(4 * c * (d - x)/b + 2)))

#Parameters:
#   Estimate Std. Error t value Pr(&gt;|t|)    
#a 0.0001397  0.0048826   0.029 0.977507    
#b 0.0434417  0.0053800   8.075 3.22e-07 ***
#c 0.0222537  0.0054435   4.088 0.000766 ***
#d 0.2392522  0.3978514   0.601 0.555535    

#Signif. codes:  0 ‘***’ 0.001 ‘**’ 0.01 ‘*’ 0.05 ‘.’ 0.1 ‘ ’ 1

#Residual standard error: 0.004108 on 17 degrees of freedom

#Number of iterations to convergence: 9 
#Achieved convergence tolerance: 1.49e-08
plot(x, y, lwd=2, xlab='AuCl4_uM', ylab='Relative_bGal')
xfit &lt;- seq(0, 10, 0.01)
yfit &lt;- (0.0001397+0.041717/(1+exp(4*0.0222537*(0.2392522-xfit)/ 0.0434417+2)))
lines(spline(xfit, yfit), lwd=2)

#The effect of gold concentration
iGEM&lt;-function(t, state, parms) {
with(as.list(c(state, parms)), {

if(Uin&lt;0) {Uin&lt;-0}
else {Uin&lt;-Uin}
if(Uout&lt;0) {Uout&lt;-0}
else {Uout&lt;-Uout}

dR &lt;- (a + b/(1 + exp(4 * c * (d - Uin)/b + 2)))-R
dUin &lt;- (e*(Uout-Uin))
dUout &lt;- (e*(Uin-Uout))

list(c(dR, dUin, dUout))
})
}

library(deSolve)

parms&lt;-c(
a=0.0001397,
b=0.0434417,
c=0.0222537,
d=0.2392522,
e=0.01)

times&lt;-seq(0, 120, 0.05)

state&lt;-c(
R=0.003464946,
Uin=0,
Uout=0)
out0 &lt;- ode(y = state, times = times, func = iGEM, parms = parms)


state&lt;-c(
R=0.003464946,
Uin=0,
Uout=0.5)
out1 &lt;- ode(y = state, times = times, func = iGEM, parms = parms)

state&lt;-c(
R=0.003464946,
Uin=0,
Uout=1)
out2 &lt;- ode(y = state, times = times, func = iGEM, parms = parms)

state&lt;-c(
R=0.003464946,
Uin=0,
Uout=2)
out3 &lt;- ode(y = state, times = times, func = iGEM, parms = parms)

state&lt;-c(
R=0.003464946,
Uin=0,
Uout=4)
out4 &lt;- ode(y = state, times = times, func = iGEM, parms = parms)

state&lt;-c(
R=0.003464946,
Uin=0,
Uout=6)
out5 &lt;- ode(y = state, times = times, func = iGEM, parms = parms)


state&lt;-c(
R=0.003464946,
Uin=0,
Uout=8)
out6 &lt;- ode(y = state, times = times, func = iGEM, parms = parms)


state&lt;-c(
R=0.003464946,
Uin=0,
Uout=10)
out7 &lt;- ode(y = state, times = times, func = iGEM, parms = parms)


out&lt;-rbind(out0, out1, out2, out3, out4, out5, out6, out7)
plot(out, pch='.')


#The effect of membrane permeability
iGEM&lt;-function(t, state, parms) {
with(as.list(c(state, parms)), {

if(Uin&lt;0) {Uin&lt;-0}
else {Uin&lt;-Uin}
if(Uout&lt;0) {Uout&lt;-0}
else {Uout&lt;-Uout}

dR &lt;- (a + b/(1 + exp(4 * c * (d - Uin)/b + 2)))-R
dUin &lt;- (e*(Uout-Uin))
dUout &lt;- (e*(Uin-Uout))

list(c(dR, dUin, dUout))
})
}
time&lt;-seq(0, 400, 1)

library(deSolve)
parms&lt;-c(
a=0.0001397,
b=0.0434417,
c=0.0222537,
d=0.2392522,
e=0.01)

state&lt;-c(
R=0.003464946,
Uin=0,
Uout=0)
out0 &lt;- ode(y = state, times = times, func = iGEM, parms = parms)

times&lt;-seq(0, 400, 0.1)

parms&lt;-c(
a=0.0001397,
b=0.0434417,
c=0.0222537,
d=0.2392522,
e=0.01)

state&lt;-c(
R=0.003464946,
Uin=0,
Uout=5)
out1 &lt;- ode(y = state, times = times, func = iGEM, parms = parms)

parms&lt;-c(
a=0.0001397,
b=0.0434417,
c=0.0222537,
d=0.2392522,
e=0.05)

state&lt;-c(
R=0.003464946,
Uin=0,
Uout=5)
out2 &lt;- ode(y = state, times = times, func = iGEM, parms = parms)

parms&lt;-c(
a=0.0001397,
b=0.0434417,
c=0.0222537,
d=0.2392522,
e=0.005)

state&lt;-c(
R=0.003464946,
Uin=0,
Uout=5)
out3 &lt;- ode(y = state, times = times, func = iGEM, parms = parms)


out&lt;-rbind(out0, out1, out2, out3)
plot(out, pch='.')


#The effect of gold mineralization peptides
#mineralisastion to peptide degradation ratio
iGEM&lt;-function(t, state, parms) {
with(as.list(c(state, parms)), {

if(Uin&lt;0) {Uin&lt;-0}
else {Uin&lt;-Uin}
if(Uout&lt;0) {Uout&lt;-0}
else {Uout&lt;-Uout}
if(Uout==0) {P&lt;-0}
else {P&lt;-P}

dR &lt;- (a + b/(1 + exp(4 * c * (d - Uin)/b + 2)))-R
dUin &lt;- (e*(Uout-Uin))
dUout &lt;- (e*(Uin-Uout)-f*P)
dP&lt;-(g*R/(h+R)-i*P)

list(c(dR, dUin, dUout, dP))
})
}

library(deSolve)

times&lt;-seq(0, 150, 0.1)

parms&lt;-c(
a=0.0001397,
b=0.0434417,
c=0.0222537,
d=0.2392522,
e=0.1,
f=0.1,
g=0.1,
h=0.1,
i=0.1)

state&lt;-c(
R=0.003464946,
Uin=0,
Uout=0,
P=0)
out0 &lt;- ode(y = state, times = times, func = iGEM, parms = parms)

state&lt;-c(
R=0.003464946,
Uin=0,
Uout=5,
P=0)

out1 &lt;- ode(y = state, times = times, func = iGEM, parms = parms)

parms&lt;-c(
a=0.0001397,
b=0.0434417,
c=0.0222537,
d=0.2392522,
e=0.1,
f=0.12,
g=1,
h=1,
i=0.1)
out2 &lt;- ode(y = state, times = times, func = iGEM, parms = parms)

parms&lt;-c(
a=0.0001397,
b=0.0434417,
c=0.0222537,
d=0.2392522,
e=0.1,
f=0.2,
g=1,
h=1,
i=0.1)
out3 &lt;- ode(y = state, times = times, func = iGEM, parms = parms)

out&lt;-rbind(out0, out1, out2, out3)
plot(out, pch='.')
</CODE></PRE></DIV><DIV id="labbookcontent" class="hiddenContent"><H2>August 22nd</H2><P>Assembly of the LuxR generator and the AiiA (BBa_K516022). We cleaved each part with XbaI and PstI (LuxR generator) or EcoRI and SpeI (AiiA). They were run on 1.2% agarose gels and extracted using QIAquick gel extraction kit. Through three way ligation, we managed to clone both inserts into the BBa_J61002 backbone that was precut with with EcoRI and PstI. We confirmed the insertion by amplifying the ligation products with GoTaq polymerase. The shift to 2Kb in lane D indicates the successful ligation reaction. We did transformations with <I>E. coli</I> strain DH5 alpha, and grew them on LB agar supplemented with 100ug/ml ampicilin.</P><H2>August 23rd</H2><P>We performed colony PCR to screen for the transformants. After 12 hours of incubation, colonies were small and appeared white, so we left them to grow for three more hours. Most colonies appeared to have the right plasmid as they remained unchanged. 14 white colonies were picked at random and used for the PCR. We observed the same band shift which is positive.</P><H2>August 24th</H2><P>We tried to assemble the gold sensing device (BBa_K1127008) with the LuxR/AiiA generators. Unfortunately, it was not successful. We think that the digestion could have been incomplete as bands were observed in the controls.</P><H2>August 27th</H2><P>Today, we tried to the assembly again. Instead of the three way ligation,  we used the Rapid Assembly of Biobricks (RAB). The parts were amplified with Phusion ® polymerase and digested by single restriction enzyme (SpeI for the BBa_K1127008, and XbaI for the LuxR/AiiA generators). The digests were ligated and subjected to PCR amplification. The band is shifted in lane A that indicates successful ligation. </P><H2>August 28th</H2><P>We returned to our favourite cloning techniques - the three way assembly.  This time we succeeded. For confirmation, we amplified the ligation products. The shift to about 1.5Kb is shown in the gel photo.</P><H2>September 9th</H2><P>For the purpose of characterization, we cloned a reporter gene (GFP or RFP generator) into our construct. Using the three way ligation, we managed to do it successfully. The gel photo shows how we cut off the gels for extraction. We also show the streaked colonies of the transformants. They are green which indicates the presence of GFP (DLig 13E).</P></DIV><DIV id="niklabbookcontent" class="hiddenContent"><H2>Nikola's lab book</H2><P>His lab book can be found online <A href="https://static.igem.org/mediawiki/2013/8/8f/Nikola%27s_lab_book.pdf">here.</A></P></DIV></DIV></DIV><DIV class="printfooter">
Retrieved from &quot;<A href="http://2013.igem.org/Team:York_UK/Notebook.html">http://2013.igem.org/Team:York_UK/Notebook.html</A>&quot;</DIV></DIV></DIV><DIV id="footer-box" class="noprint"><DIV id="footer"><UL id="f-list"><LI id="t-recentchanges"><A href="/Special:RecentChanges" title="Recent changes">Recent changes</A></LI><LI id="t-whatlinkshere"><A href="/Special:WhatLinksHere/Team:York_UK/Notebook.html" title="List of all wiki pages that link here [j]" accesskey="j">What links here</A></LI><LI id="t-recentchangeslinked"><A href="/Special:RecentChangesLinked/Team:York_UK/Notebook.html" title="Recent changes in pages linked from this page [k]" accesskey="k">Related changes</A></LI><LI id="t-specialpages"><A href="/Special:SpecialPages" title="List of all special pages [q]" accesskey="q">Special pages</A></LI><LI><A href="/Special:Preferences">My preferences</A></LI></UL></DIV><DIV id="footer"><UL id="f-list"><LI id="t-print"><A href="/wiki/index.php?title=Team:York_UK/Notebook.html&amp;printable=yes" title="Printable version of this page [p]" accesskey="p">Printable version</A></LI><LI id="t-permalink"><A href="/wiki/index.php?title=Team:York_UK/Notebook.html&amp;oldid=323255" title="Permanent link to this revision of the page">Permanent link</A></LI><LI id="privacy"><A href="/2013.igem.org:Privacy_policy" title="2013.igem.org:Privacy policy">Privacy policy</A></LI><LI id="disclaimer"><A href="/2013.igem.org:General_disclaimer" title="2013.igem.org:General disclaimer">Disclaimers</A></LI></UL></DIV></DIV></DIV></BODY></HTML>