<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_William_and_Mary_Description skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:William and Mary/Description</SPAN></H1><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><P><CFHEADER name="X-XSS-Protection" value="1"><TITLE>NOISE - W&amp;M iGEM</TITLE></CFHEADER></P><DIV class="container"><DIV class="navbar-header"><BUTTON id="menu-toggle" type="button" class="navbar-toggle" data-toggle="collapse" data-target="#example"><SPAN class="sr-only">Toggle navigation</SPAN></BUTTON><A href="https://2015.igem.org/Team:William_and_Mary" class="navbar-brand">W&amp;M iGEM 2015</A></DIV><DIV class="collapse navbar-collapse"><UL class="nav navbar-nav navbar-right navbar-uppercase"><LI><A href="https://2015.igem.org/Team:William_and_Mary"> 
                            Home
                            </A></LI><LI><A href="https://2015.igem.org/Team:William_and_Mary/Description"> 
                            Project Description
                            </A></LI><LI><A href="https://2015.igem.org/Team:William_and_Mary#parts">
                            Parts
                            </A></LI><LI><A href="https://2015.igem.org/Team:William_and_Mary/Measurement"> 
                            Results
                            </A></LI><LI><A href="https://2015.igem.org/Team:William_and_Mary/Modeling"> 
                            Modeling
                            </A></LI><LI><A href="https://2015.igem.org/Team:William_and_Mary/Interlab_Study"> 
                            Interlab Study
                            </A></LI><LI><A href="https://2015.igem.org/Team:William_and_Mary/Collaborations"> 
                            Collaboration
                            </A></LI><LI><A href="https://2015.igem.org/Team:William_and_Mary/Software"> 
                           Software
                            </A></LI><LI><A href="https://2015.igem.org/Team:William_and_Mary/Practices"> 
                            Human Practices
                            </A></LI><LI><A href="https://2015.igem.org/Team:William_and_Mary#ref"> 
                            References
                            </A></LI><LI><A href="https://2015.igem.org/Team:William_and_Mary/Medal_Criteria"> 
                            Medal Criteria
                            </A></LI><LI><A href="https://2015.igem.org/Team:William_and_Mary/Team"> 
                            Team
                            </A></LI></UL></DIV></DIV><DIV class="wrapper"><DIV class="section section-we-are-2" id="projectdesc"><A name="whatis"><DIV class="text-area"><DIV class="container"><DIV class="row"><DIV class="col-md-4"><DIV class="title"><P class="h2WM">What is noise?</P></DIV></DIV><DIV class="col-md-7 col-md-offset-1"><DIV class="description"><P>Cellular processes occur in a highly dynamic fashion. The variations resulting from the constantly shifting environmental context of the cell, as well as the inherent unpredictability that arises from the microscale kinetics of biomolecular reactions, compound together to ensure that the behavior of a cell can never truly be deterministic. 

                                                </P></DIV></DIV></DIV></DIV></DIV></A>The transcription of a gene is an example of such a process-- in prokaryotes, both the time between and the amount of mRNAs produced in each of the bursts of transcription are, though they are constrained by natural laws and limits, nonetheless random values. This inability to predict with complete accuracy and precision, regardless of the amount of information that an observer has about the cell and its environment, confers a level of irreducible variability on transcription.<P>

This variability, also commonly referred to as stochasticity or noise, in gene expression can be split into two components: extrinsic and intrinsic noise. Extrinsic factors, like differing RNA polymerase concentrations and nucleotide availability, affect transcription equally within the cell, regardless of what cis-regulatory elements drive transcription of the genes in question. Intrinsic properties, like DNA-protein binding kinetics, are promoter-specific contributions to noise.</P><P>

For a synthetic biologist designing a genetic network, it is critical to understand the level of noise present in the network’s components. On one hand, a high level of noise in a component could make the network more unstable, and hence less effective or even unsafe. On the other hand, noise in the network could be harnessed to occasionally cross otherwise energetically unfavorable thresholds and hence open up new possibilities for network functionality and applications. Either way, as the frontier of synthetic biology moves towards ever more complex and intricate networks, it is clearly not sustainable to remain ignorant of the level of noise intrinsic to each component part of a synthesized construct.</P><P>

Despite the extensive characterization of the average strength of the promoters available on the Biobrick registry, very few have information pertaining to the variability in their expression. Our project aims to tease apart the extrinsic and intrinsic components of gene expression noise, and characterize the noise inherent to commonly used promoters in synthetic biology.</P><P>
 
Original work done by Michael Elowitz investigated stochasticity in gene expression at the single cell level using a dual-reporter fluorescent system [1]. In this method two constructs are created that both drive expression of a fluorescent protein using the same promoter, RBS, and terminator. One of these reporter constructs uses YFP and the other CFP.  This dual reporter system is what allows the investigator to differentiate between the extrinsic factors of noise, which should affect both constructs equally, and the intrinsic factors, which will result in a difference in the ratio of CFP: YFP fluorescence. These constructs are then <A href="https://2015.igem.org/Team:William_and_Mary/Protocols:Integration">integrated into specific sites on the genome</A> and the fluorescence output measured using fluorescent imaging. </P><DIV><P>(ABOVE) Panel A shows an example of intrinsic and extrinsic noise; both the ratio of the fluorescence of the two reporters (intrinsic) and the absolute fluorescence of the two reporters (extrinsic) changes. Panel B shows an example of extrinsic noise only; the ratio of the fluorescence of the two reporters stays constant while the absolute fluorescence changes [2].</P></DIV><DIV><P>Once individual cell measurements have been taken for both CFP and YFP expression using confocal microscopy, <A href="https://2015.igem.org/Team:William_and_Mary/Measurement">the intrinsic noise measurements can be calculated and further analyzed</A>.</P></DIV><DIV><P>
(RIGHT) Schematic of dCas9/CRISPR system. gRNAs targeting the promoter region of the gene of interest will guide dCas9 binding. Bound dCas9 sterically hinders RNAP and transcription factors from binding and promoting transcription [3].</P><P>In addition to characterizing the noise of promoters in iGEM, we contributed additional tools for manipulating their expression. CRISPR/Cas9 has been used extensively in synthetic biology, both inside and out of iGEM. In particular, new functionalizations of the Cas9 protein that remove its catalytic nuclease domain while retaining its DNA-binding activity have allowed for novel methods in molecular biology. This catalytically inactive variant of Cas9, known as dCas9, can be used to repress gene expression by targeting the promoter region of a gene of interest. This repression is mediated by the CRISPR/Cas9 complex binding to the promoter region and can block RNAP binding or prevent transcriptional elongation. All of our gRNAs prevent RNAP binding and initiation of transcription.</P></DIV><DIV><P>We created a codon-optimized dCas9 variant for expression in E. coli  that, when used with an appropriate gRNA, reduced observed gene expression by 97%. In addition to making both a dCas9 coding region (BBa_K1795000), and a dCas9 operon for constitutive expression (BBa_K1795001), we also contribute 12 gRNAs that target commonly used promoters in iGEM, and a scrambled gRNA to be used as a negative control.</P><P>We believe that our contributions to iGEM 2015 will not only open the door for more noise-related iGEM projects in the future, but will allow for more sophisticated synthetic gene regulatory network design both through the additional promoter characterization we have performed and the availability of new negative regulators of gene expression.</P></DIV><DIV><P>References:</P><P>1: Elowitz, Michael B., et al. &quot;Stochastic gene expression in a single cell.&quot; Science 297.5584 (2002): 1183-1186.</P><P>2: Raser, Jonathan M., and Erin K. O'Shea. &quot;Noise in gene expression: origins, consequences, and control.&quot; Science 309.5743 (2005): 2010-2013.
</P><P>3: Qi, Lei S., et al. &quot;Repurposing CRISPR as an RNA-guided platform for sequence-specific control of gene expression.&quot; Cell 152.5 (2013): 1173-1183. </P></DIV></DIV></DIV></DIV></DIV><FOOTER class="footer footer-color-black" data-color="black"><DIV class="container"><NAV class="pull-left navbar-burger"><UL><LI><A href="https://2015.igem.org/Team:William_and_Mary">
                                                Home
                                            </A></LI><LI><A href="mailto:igem@wm.edu">
                                                Contact
                                            </A></LI></UL></NAV><DIV class="copyright">
                                    © 2015 William &amp; Mary iGEM
                                </DIV></DIV></FOOTER></DIV></DIV></BODY></HTML>