Team:NAWI-Graz/Results
NAWI-Graz @ iGem - Description
Petersgasse 12, Graz
NAWI Graz
Home
Team
Team
Collaborations
Project
Description
Design
Experiments
Proof of concept
Demonstration
Results
Notebook
Parts
Basic_Parts
Composite Part
Part Collection
Safety
Attributions
Human Practices
Human Practices
Silver
Sponsoring
Parts Verification
The first thing we had to do was to check if the gBlocks provided by IDT hat the correct sequence. For this we used the blunt end pjet1.2. cloning vector. For each gBlock we used 4 clones for sequencing. In case of the blue and the red gBlock we were quit sure we would get the correct sequence because we picked red and blue colonies. In case of the gBlocks containing the toxins we were not sure about the result. Red (Figure 1) and Blue (Figure 2) delivered like assumed correct sequences and also the mazF part (Figure 3) had the correct sequence. Only the ccdB part made problems.
Figure 1 Sequencing result for the gBlock encoding for the red chromoprotein and the mazE antitoxin.
Figure 2 Sequencing result for the gBlock encoding for the blue chromoprotein and the ccdA antitoxin.
Figure 3 Sequencing result for the gBlock encoding for the mazF toxin and the dam methylase.
When we got the results we saw, that all clones we used for sequencing looped out the sequence coding for the toxic ccdB (Figure 4). It seems that homologues region of 25bp is enough for E.coli to do recombination.
Figure 4 Negative sequencing result for the ccdB toxin dam methylase part. E.coli used the hologues RBS part to loop out the toxin part.
To solve the problem we designed a primer without the tac promoter and with a different RBS. The missing of the tac promoter was no problem because we had to use a pUC19 plasmid which already has a lac promoter downstream the MCS. After a successful PCR we used the product to do another pJET ligation. This time we did restricton analysis before sending for sequencing. We used EcoRI which hat to restriction sides and for this reason we got two fragments. One fragment should show a band at the size of 3000bp and the fragment a band at the size of about 1100bp. 3 clones showed the correct pattern on the gel. (Figure 5)
Figure 5 Restrcition analysis of the pjet2.1 palsmid with the ccdB gBlock. One band was expacted at 1100bp and one at 3000bp. Clone 1,3,5 were used for further working
The correct clones were sent for sequencing. This time the fragments showed the sequence we needed. (Figure 6)
Figure 6 Sequencing result for the ccdB toxin dam methylase part.
Plasmid Assembly
To assembly our final plasmids we used the correct sequenced pjet clones to perform a PCR. To see if we have the correct sequence we perform agarose gel and extracted the correct sized DNA band from the gel.
The next thing we did was to digest the different parts with the correct enzymes. After cutting we cleaned the product to be sure we can perform a perfect ligation.  After the ligation and transformation, we expected red and blue colonies.  To be sure the clones which turned blue and red had the correct plasmid we did a restriction digest of different colonies. For both clones we used EcoRI for restriction analysis. For the reason this enzyme has two restriction sites we expected two fragments one fragment with 1000bp and one with 4000bp. For the pUC19_red_mazF we checked 8 clones und 6 of them were positive (Figure 7). For the pUC19_blue_ccdB we checked only three clones. Two of the three showed the pattern we assumed (Figure 8).
Figure 7 Restriction analysis of the E.coli XL1 pUC19_red_mazF strain. By cutting with EcoRI we expected two bands. One at 1000bp and the other at 3000bp. Only clone 6 and 7 did not show the correct pattern.
Figure 8 Restriction analysis of the E.coli XL1 pUC19_blue_ccdB strain. By cutting with EcoRI we expacted two bands. One at 1000bp and the other at 3000bp. Only clone 6 and 7 did not show the correct pattern.
The correct clones were sequenced and after getting the sequenced verified we isolated the plasmid to be sure E.coli does not destroy it. We also picked colonies from the correct red and blue strains and plated them on three following days on plates and count each day how many colonies got colour and how many stayed with. This should say us how stabile the plasmid is.
Table 3 Analysis to see if the pUC19_red_mazF plasmid is stable over more then one generation. It can be seen that less the 1% of the E.coli XL1 cells lost the colour.
Table 4 Analysis to see if the pUC19_blue_ccdB plasmid is stable over more then one generation. It can be seen that less the 1% of the E.coli XL1 cells lost the colour.
The next thing we did was to use the correct sequenced clones and transform it again to receive fresh clones with no mutations for our final studies.
Figure 9 Coloured E.coli strains after successful plasmid assembling and transformation. a.) E.coli XL1 pUC19_blue_ccdB b.)E.coli XL1 pUC19_red_mazF
Proteomics
The check if the seven proteins, chromoprotein red/blue, ccdB, mazF, ccdA, mazE and dam methylase were expressed we used SDS-PAGE. It was also important to see, if the proteins especially the toxins were secreted to the media. The chromoproteins can be found at about 25kda, at the size of 32kDa we can find the dam methylase. The ccdB has a weight of about 11kDa.
Figure 10 SDS-PAGE useing the growthmedia from our strains. We used the pure solution and the sperarated media in lower 10kDA and bigger than 10kDA for analysis. We can a little lower than the 35kDA standard the dam methylase. At the size of 25kDA we find the chromoproteins from the differnen strains. At 11kDA the ccdB toxin of the blue strain can be detected.
The two antitoxins and mazE are smaller than 10kDA, when seeing the picture in real we could find small bands lower than 10kDA. Unfortinately no time was left for puring a higher concentrated SDS gel.
Although we cannot be 100% sure that all proteins we needed were expressed by E.coli we can be really sure that our insilico worked. For all composite parts one protein can be found on the gel. In case of the toxin part we found the dam methylase wich was located downstream the toxin. In case of the antitoxin composite part we saw the red chromoprotein on the SDS and also with the coloured colonies.
Microbiology
The last part of the project was the microbiological part. This final part would show us if our hypothesis was true or not.  The first thing we did was to set up two over night cultures from each strain. When we induced one ONC at an OD600 of 0.6 and we ecpacted different growth behaviour.  The E.coli with the mazF toxin had no changes in growing (Figure 12) while the one with the ccdB toxin did not grow as fast as the not induced, but still showed growth (figure 11) . We think this shows us that the toxin/antitoxin system works because there is growth but E.coli shows stress symptoms.
Figure 11 Growth behavior of E.coli XL1 pUC19_blue_ccdB with in iducing at OD600 of 0.6. It can be seen that after indcuing the induced strain growes slower than the not induced strain but has still growth.
Figure 12 Growth behavior of E.coli XL1 pUC19_red_mazF with in inducing at OD600 of 0.6. It can be seen that no changes in growth behavior happened
For the reason the red clone showed normal growth we decided to add IPTG from the beginning. What we saw was that the growth of the induced culture was about 50% slower than the not induced ONC but still showed growth (Figure 13). For us this proofed again that the system is working.
Figure 13 Growth behavior of E.coli XL1 pUC19_red_mazF induced from the beginning. After 3.5 hours the induced strain showed a 50% lower growth rate than the not induced strain. This led to the conclusion that the system works but puts our E.coli under pressure.
The next thing we did was to bring the different colonies together on a plate. First we tried without inducing to see what happened. Like we assumed nothing happened, to different strains grow next to each other. (Figure 14)
Figure 14 E.coli XL1 pUC19_blue_ccdB against E.coli XL1 pUC19_red_mazF on agar plate without IPTG. Plates show equal growth with no overgrowing or killing the other bacteria.
In the final test we used plates with 0.1M of IPTG. This time we used ab Impföse to bring the different strains together. After growing overnight we detected that the red strain killed or overgrew the blue strain on every single plate.
Figure 15 E.coli XL1 pUC19_blue_ccdB against E.coli XL1 pUC19_red_mazF on agar plate with IPTG. The plates show that the E.coli XL1 pUC19_red_mazF grows over E.coli XL1 pUC19_blue_ccdB. In a we let them grow straight to each other and mixed them in the middle. In b we did the same as in a just with 2 red strains. C shows that the red strain grows even when everything is full of blue E.coli. In d we used each strain and plated them over each other.
BioBricks
After we had our positive clones for the main project we used the positive sequenced pJET clones to assemble the BioBricks.  We decided to make four composite BioBricks. The composite parts are also the main parts of our project.  One BioBrick contains the MazE antitoxin and the red chromoprotein. (Figure 1)
Figure 16 Plasmid map of the composite part containing the red chromoprotein and the mazE antitoxin
One other BioBrick contains the mazF Toxin and the dam methylase. (Figure 2)
Figure 17 Plasmid map of the composite part containing the dam methylase and the mazF toxin
The third BioBrick contains the ccdA antitoxin for the ccdB toxin and the blue chromoprotein. (Figure 3)
Figure 18 Plasmid map of the composite part containing the blue chromoprotein and the ccdA antitoxin
The ccdB Toxin BioBrick did not work to assemble
Like told by the iGEM HQ we used the pSB1C3 plasmid as Backbone. Unfortunately the plasmid provided by the HQ did not work. So we had to use the plasmid from 12P well of plate 6. We transformed the plasmid in E.coli afterwards we isolated the plasmid cut with EcoRI and PstI. After electrophoreses we cut out the band at the size of 2000bp we purified the DNA and measured the concentration with the NanoDrop.  To get the Inserts we used PCR. As template we used like said before the positive sequenced pJET clones. We used an aliquot of the PCR product to check the correct size the other part of the PCR was used to purify. Then we cut the fragments with EcoRI and PstI. Next we calculated a molecular 1:1 ratio and performed ligation. Transformation and plating out followed.  After screening the plates for positive clones we performed dilution plating to get single colonies. Figure 4 showes the red-mazE BioBrick, Figure 5 the blue_ccdA BioBrick and Figure 6 the mazF_dam methylase BioBrick. It took over night to see the different colors of the blue and red constructs.
Figure 19 pSB1C3-red in E.coli XL1 after incubation over night
Figure 20 pSB1C3-blue in E.coli XL1 after incubation over night
Figure 21 pSB1C3-mazF in E.coli XL1 after incubation over night
To be sure we send the correct sequence to the registry we let it sequence. (Figure 7-9)
Figure 22 Sequencing results for the red_mazE composite part.
Figure 23 9 Sequencing results for the blue_ccdA composite part.
Figure 24 11 Sequencing results for the dam methylase_ mazF composite part.
For the reason we used the same parts for the BioBricks as we used for our main plasmids we did no further analysis in case of proteomics. You can find de results here!
Conclusions
After 3 months of working in the lab and planning everything on our own for the first time, we realized that synthetic biology can be quite challenging.  As it was the first time we did a project all alone, we faced many troubles we would never have imagined before. From getting a lab ready to work in to making time schedules and redoing a task again and again we had the chance to learn a lot. Before the iGEM competition our university laboratory exercises were planned and completely coordinated by professors with all their experience. We had no idea about how hard it will actually be to get a living organism to produce foreign toxic proteins, without mutating or killing itself. In theory we often got the impression that in science “everything is possible, there are no limits” but in real-life it is still living organisms that we are working with.
Above all, we got a better insight of real-life research with all its chances and challenges.
-->
Location
NAWI Graz
Petersgasse 12
8010 Graz, AUT
Mail address
igem.nawigraz@gmail.com
Social Media
© Copyright 2016. All Rights Reserved.
