Team:HSNU-TAIPEI/labnotes
HSNU
Menu
close
Project
Overview
Benzo[A]Pyrene
Cadmium
Lead
Aflatoxin
Product
Parts
Parts
Basic Part
Composite Part
Part Collection
Notebook
Human Practice
Overview
Organization
Education
Senior High
Junior High
Elementary
Kindergarten
Teacher
Government
Legislators
Ministry of Health and Welfare
NGO
Industry
Material
APP
PPT
Video
Poster
Questionnaires
Cooperation
Meet Up
Asian Conference
Community
Lawyer
Achievement
Team
Attributions
Lab Notes
Extract the plasmid
Researcher: Zhao Ming-Cheng, Su Chin-Fong
Place: NTU
Date: September 14th, 2015
Procedure
We set them according to the following.
FIRST
SECOND
THIRD
FORTH
Plasmid:Part1-QsrR(K1092000)-Terminator(B0015) 3μL
Plasmid:Part1-QsrR(K1092000)-Terminator(B0015) 3μL
Plasmid:Part2- RFP(E1010)-Terminator(B0015)
Plasmid:Part2- RFP(E1010)-Terminator(B0015)
Buffer:M 1μL
Buffer:M 1μL
Buffer:M 1μL
Buffer:M 1μL
Restriction enzyme: Xba1 0.25μL
Restriction enzyme: Xba1 and Spe1 0.25μL each
Restriction enzyme: Xba1 0.25μL
Restriction enzyme: Xba1 and Spe1 0.25μL each
DdH2O:5.75μL
DdH2O:5.50μL
DdH2O:5.75μL
DdH2O:5.50μL
And then do Gel electrophoresis.
Result
Show More
Break the Microcapsule and Serial Dilution
Researcher: Leng Yi-Yan, Chang Chun-chieh, Lee Ming-Jhen, Lin Sheng
Place: Fu Jen University Food Science Department
Date: September 13th, 2015
Material
Sodium citrate
Pipettes
Saline
Petri dish
Solid medium(LB agar)
Alcohol Burner
Procedure
Step1:  Add 9ml saline liquid into each tube. Each group has seven dilution ratio which dilute ten times each tube.
Step2: Add 5ml the microcapsule broken solution into each tube.
Step3: Put microcapsules into microcapsule broken solution and vortex for ten minutes until it decompose.
Step4: Draw 1ml from the tube which contains microcapsule broken solution and bacteria liquid. And add it into saline liquid then vortex it. Draw 1ml and add into next tube. Repeat these steps until it dilute seven times.
Step5: Draw 1ml the microcapsule broken solution and drop it onto the corresponding petri dish. Then pour the solid medium in and mix it well.
Step6: Until solid medium solidify, put them into thirty-seven degrees Celsiusincubator.
Step7:Repeat  each step until each immobililzation method and  control group finish the process of serial dilution.
By the way, we also design the product. And this is our blueprint.
Show More
Liquid culture
Researcher: Zhao Ming-Cheng, Su Chin-Fong
Place: NTU
Date: September 13th, 2015
Procedure
We take Part1-QsrR(K1092000)-Terminator(B0015) and Part2- RFP(E1010)-Terminator(B0015) which have been transformed to do liquid culture.
Show More
Ligation
Researcher: Zhao Ming-Cheng, Su Chin-Fong
Place: NTU
Date: September 12th, 2015
Procedure
We have to do ligation one more time.
promoter(J23113)-RBS(B0034)-Laccase(K1092104)-RBS(B0034)-QsrR(K1092000)-Terminator(B0015)
promoter(J23113)-QsrR binding site-RBS(B0034)-RFP(E1010)-Terminator(B0015)
That is  Part1-QsrR(K1092000)-Terminator(B0015)  Part2- RFP(E1010)-Terminator(B0015)
After the Ligation,and then transform.
Show More
Restriction Digest＆Ligation
Researcher: Chang Ko-Yu, Chu Yi-Chia
Place: Academia Sinica
Date: September 11th, 2015
Procedure
Restriction Digest
cyp1a2
10X buffer
3
DNA
5
SpeI
0.6
PstI
0.4
ddH2O
21
Total
30(μl)
cyp1a2
10X buffer
3
DNA
5
EcoRI-HF
0.5
PstI
0.5
ddH2O
21
Total
30(μl)
J23119
10X buffer
3
DNA
5
XbaI
0.5
PstI
0.5
ddH2O
21
Total
30(μl)
B0015
10X buffer
3
DNA
5
EcoRI-HF
0.5
PstI
0.5
ddH2O
21
Total
30(μl)
Run Gel(small well for check)
Run Gel(big well for elute)
Elute Fragment
Ligation
Psb1c3+cyp1a2
T4 DNA Ligase
1
Vector (psb1a2)
3
nsert(cyp1a2 compsite)
14
T4 DNA Ligase Buffer
2
ddH2O
0
Total
20μl
23119+cyp1a2
T4 DNA Ligase
1
Vector (B0015)
3
nsert(cyp1a2 )
14
T4 DNA Ligase Buffer
2
ddH2O
0
Total
20μl
Place all sample in room temperature for 2hr
Transform 10ul of the sample
Nothing on the plate
Result
From left to right:1kb ladder,B0015+cyp1a2(Colony1-7),Psb1c3+cyp1a2(Colony1-4)
Cyp1a2-1＆Cyp1a2-5 are fault.
We deside to choose B0015+cyp1a2 Colony-2 Psb1c3+cyp1a2 compsite-3
Show More
Break the Microcapsule and Serial Dilution
Researcher: Leng Yi-Yan, Chang Chun-chieh, Lee Ming-Jhen, Lin Sheng
Place: Fu Jen University Food Science Department
Date: September 11th, 2015
Material
Sodium citrate
Pipettes
Saline
Petri dish
Solid medium(LB agar)
Alcohol Burner
Procedure
Step1: Add 9ml saline liquid into each tube. Each group has seven dilution ratio which dilute ten times each tube.
Step2: Add 5ml the microcapsule broken solution into each tube.
Step3: Put microcapsules into microcapsule broken solution and vortex for ten minutes until it decompose.
Step4: Draw 1ml from the tube which contains microcapsule broken solution and bacteria liquid. And add it into saline liquid then vortex it. Draw 1ml and add into next tube. Repeat these steps until it dilute seven times.
Step5: Step5: Draw 1ml the microcapsule broken solution and drop it onto the corresponding petri dish. Then pour the solid medium in and mix it well.
Step6: Until solid medium solidify, put them into thirty-seven degrees Celsiusincubator.
Step7:Repeat  each step until each immobililzation method and  control group finish the process of serial dilution.
Show More
Extract the plasmid/Gel electrophoresis
Researcher: Zhao Ming-Cheng,Su Chin-Fong
Place: NTU
Date: September 11th, 2015
Procedure
Part1-QsrR(K1092000)-Terminator(B0015)
Part2- RFP(E1010)-Terminator(B0015)
After Extract the plasmid
We set them according to the following
FIRST
SECOND
THIRD
FORTH
Plasmid:Part1-QsrR(K1092000)-Terminator(B0015) 3μL
Plasmid:Part1-QsrR(K1092000)-Terminator(B0015) 3μL
Plasmid:Part2- RFP(E1010)-Terminator(B0015)
Plasmid:Part2- RFP(E1010)-Terminator(B0015)
Buffer:M 1μL
Buffer:M 1μL
Buffer:M 1μL
Buffer:M 1μL
Restriction enzyme: Xba1 0.25μL
Restriction enzyme: Xba1 and Spe1 0.25μL each
Restriction enzyme: Xba1 0.25μL
Restriction enzyme: Xba1 and Spe1 0.25μL each
DdH2O:5.75μL
DdH2O:5.50μL
DdH2O:5.75μL
DdH2O:5.50μL
And then do Gel electrophoresis.
Result
From left to right
1~4 is the first group
5~8 is the second group
9 is 1kb marker
10~13 is the third group
14~17 is the forth group
But the length is wrong and there are Impurities in it.
So,the result is failed
Show More
Liquid culture
Researcher: Zhao Ming-Cheng, Su Chin-Fong
Place: NTU
Date: September 10th, 2015
Procedure
We take Part1-QsrR(K1092000)-Terminator(B0015) and Part2- RFP(E1010)-Terminator(B0015) which have been transformed to do liquid culture.
Show More
Check(Mini plasmid ＆ Run Gel)
Researcher: Chang Ko-Yu, Chu Yi-Chia
Place: Academia Sinica
Date: September 10th, 2015
Procedure
Mini plasmid(14 colonies)
Restriction Digest
Psb1c3+cyp1a2 compsite(4sample)
10X buffer
1
DNA
8
EcoRI-HF
0.5
PstI
0.5
ddH2O
0
Total
10(μl)
B0015+cyp1a2 (10sample)
10X buffer
1
DNA
8
EcoRI-HF
0.5
PstI
0.5
ddH2O
0
Total
10(μl)
Run Gel
Result
From left to right:1kb ladder,B0015+cyp1a2(Colony1-7),Psb1c3+cyp1a2(Colony1-4)
Cyp1a2-1＆Cyp1a2-5 are fault.
We deside to choose B0015+cyp1a2 Colony-2 Psb1c3+cyp1a2 compsite-3
Show More
Pick Colonies＆Culture bacteria
Researcher: Chang Ko-Yu, Chu Yi-Chia
Place: Academia Sinica
Date: September 9th, 2015
Procedure
Pick different colonies to a new plate and a 5ml tube to culture them.
Place all plate and tube in 37℃ overnight
Result
plate
Ps1c3+cyp1a2compsite
B0015+cyp1a2
Original plate
Colonies we pick
tube × 14
Show More
Break the Microcapsule and Serial Dilution
Researcher: Leng Yi-Yan, Chang Chun-chieh, Lee Ming-Jhen, Lin Sheng
Place: Fu Jen University Food Science Department
Date: September 9th, 2015
Material
Sodium citrate
Pipettes
Saline
Petri dish
Solid medium(LB agar)
Alcohol Burner
Procedure
Step1:  Add 9ml saline liquid into each tube. Each group has seven dilution ratio which dilute ten times each tube.
Step2: Add 5ml the microcapsule broken solution into each tube.
Step3: Put microcapsules into microcapsule broken solution and vortex for ten minutes until it decompose.
Step4: Draw 1ml from the tube which contains microcapsule broken solution and bacteria liquid. And add it into saline liquid then vortex it. Draw 1ml and add into next tube. Repeat these steps until it dilute seven times.
Step5: Draw 1ml the microcapsule broken solution and drop it onto the corresponding petri dish. Then pour the solid medium in and mix it well.
Step6: Until solid medium solidify, put them into thirty-seven degrees Celsiusincubator.
Step7:Repeat  each step until each immobililzation method and  control group finish the process of serial dilution.
Show More
Ligation
Researcher: Zhao Ming-Cheng, Su Chin-Fong
Place: NTU
Date: September 9th, 2015
Procedure
Then take to ligaiton，according to our circuit design  is ：
promoter(J23113)-RBS(B0034)-Laccase(K1092104)-RBS(B0034)-QsrR(K1092000)-Terminator(B0015)
promoter(J23113)-QsrR binding site-RBS(B0034)-RFP(E1010)-Terminator(B0015)
That is,Part1-QsrR(K1092000)-Terminator(B0015)  Part2- RFP(E1010)-Terminator(B0015)
After the Ligation,and then transform it
Show More
Immobilization of E.coli
Researcher: Leng Yi-Yan, Chang Chun-chieh, Lee Ming-Jhen, Chen Szu-Hua, Chang Yu-Ting
Place: Fu Jen University Food Science Department
Date: September 8th, 2015
Material
3%SA(Alginate)
1.5%SA
1%CaCl2
0.3%Chitosan
Beaker
Syringe
E.coli Dh5α
LB broth
eppendorf
Procedure
Step1: Inoculate E.coli into LB broth for 4hr.
Step2: Sterilize everything before putting it into the hood.
Step3: Mark each eppendorf.
Step4: Mix the solution as the amount written below.
GROUP
BACTERIAL LIQUID
GEL
SA
15ml
3%SA 60mL
ACA
15ml
1.5%SA 60mL
NO(CONTROL)
15ml
LB Broth 60mL
Step5: Draw 0.5mL mixture by syringes and drop it into the forming solution (Immobilize E.coli by the method we try on September 1st, 2015).
Step6: Add the microcapsules into each eppendorf.
Step7: Keep the microcapsules in different temperatures (25℃/4℃/-18℃)
Result
Show More
Transformation
Researcher: Chang Ko-Yu, Chu Yi-Chia
Place: Academia Sinica
Date: September 8th, 2015
Procedure
Transform all sample(20μl) we ligase yesterday  (Ps1c3+cyp1a2compsite) (B0015+cyp1a2)
Result
Ps1c3+cyp1a2compsite
B0015+cyp1a2
Show More
Ligation
Researcher: Chang Ko-Yu, Chu Yi-Chia
Place: Academia Sinica
Date: September 7th, 2015
Procedure
Total:20μl
Vector:Insert=1:3
Psb1c3+cyp1a2 compsite
T4 DNA Ligase
1
Vector (psb1a2)
7
Insert(cyp1a2 compsite)
10
T4 DNA Ligase Buffer
2
ddH2O
0
Total
20(μl)
B0015+cyp1a2
T4 DNA Ligase
1
Vector (B0015)
7
Insert(cyp1a2)
10
T4 DNA Ligase Buffer
2
ddH2O
0
Total
20(μl)
Place the sample at 16℃ overnight
Show More
Elute DNA
Researcher: Chang Ko-Yu, Chu Yi-Chia
Place: Academia Sinica
Date: September 7th, 2015
Procedure
Add 500μl DF Buffer into the tube with gel.
55-60℃ for 10-15 min (the gel completely dissolve)
Cool in room temperature
Put the DF Colume in the 2ml Collection
Add 800μl sample in the  DF Colume
Centrifuge at 13000xg for 30sec
Discard the flow-throgh
Add 400μl W1 Buffer in the  DF Colume
Centrifuge at 13000xg for 30sec
Discard the flow-throgh
Add 600μl Wash Buffer in the  DF Colume
Centrifuge at 13000xg for 3min
Put DF Colume to the 1.5ml tube
Add 30μl Elution
Place it for 2 min
Centrifuge at 13000xg for 2min
Show More
Growth Curve
Researcher: Leng Yi-Yan, Chang Chun-chieh, Lee Ming-Jhen, Chen Szu-Hua, Chang Yu-Ting
Place: Fu Jen University Food Science Department
Date: September 6th, 2015
Purpose
To find out the time and temperature of E.coli’s culture to grow until our goal of bacterial count.
Material
E.coli
LB agar
0.85% saline
Tube
Dish
Procedure
Step1: Draw one-hundredth amount of bacteria liquid into liquid medium (Inoculation) and put it into 37℃ incubator.
Step2: Aliquot saline to each tube (9mL/tube).
Step3: Mark every dish.
Step4: Take out the bacteria liquid at 4 / 4.5 / 5 / 5.5 / 6 / 6.5 / 7 / 7.5 /8 hours and start serial dilution.
Step5: Draw 1mL bacteria liquid into the first tube and pipet it.
Step6: Exchange a tip, draw 1mL from the first tube and add it into the next tube.
Step7: Repeat Step6 until the seventh tube.
Step8: Exchange a tip and draw 1mL to a matched dish from the seventh to the first tube.
Step9: Pour LB agar to each dish (10mL/dish) and shake it by an infinity sign.
Step10: After LB agar’s solidification, place these dishes upside down in the 37℃ incubator.
Result
Show More
Activation of Culture
Researcher: Leng Yi-Yan, Chang Chun-chieh, Lee Ming-Jhen
Place: Fu Jen University Food Science Department
Date: September 4th, 2015
Material
E.coli(DH5α、CmR J22102)
Tube
LB broth
Inoculating loop
Alcohol Burner
Procedure
Step1:Draw 10mL LB broth with 5mL pipet into a tube (total4).
Step2: Scrape a colony with an inoculating loop sterilized by alcohol burner.
Step3: Add the colony into the tube (two for each strain).
Step4:Keep it in 37℃ incubator.
Show More
Restriction Digest
Researcher: Chang Ko-Yu, Chu Yi-Chia
Place: Academia Sinica
Date: September 4th, 2015
Procedure
Total:30μl
Enzyme:≤5%(1.5μl)
Buffer
U(DNA/mg in 50μl
Temper
EcoRI-HF
2.1[100]
20
37℃
SpeI
2.1[100]
10
37℃
PstI
3.1[100]
2.1[75]
20
37℃
XbaI
2.1[100]
20
37℃
We descide to use 2.1 Buffer.
B0015(75.5ng/μl)
10x buffer
3
DNA
13.5
EcoRⅠ-HF
0.5
XbaI
0.5
ddH2O
12.8
Total
30μl
B0034(208ng/μl)
10x buffer
3
DNA
4.8
EcoRⅠ-HF
0.5
XbaI
0.5
ddH2O
21.1
Total
30μl
B0032(350.5ng/μl)
10x buffer
3
DNA
2.9
EcoRⅠ-HF
0.5
XbaI
0.5
ddH2O
23.1
Total
30μl
E0040(188.5ng/μl)
10x buffer
3
DNA
5.3
EcoRⅠ-HF
0.5
XbaI
0.5
ddH2O
20.7
Total
30μl
CYP1A2(100ng/μl)
10x buffer
3
DNA
10
EcoRⅠ-HF
0.4
XbaI
0.6
ddH2O
16
Total
30μl
CYP1A2 COMPOSIE(10ng/μl)
10x buffer
3
DNA
10
EcoRⅠ-HF
0.5
XbaI
0.5
ddH2O
16
Total
30μl
J23119(220ng/μl)
10x buffer
3
DNA
4.5
EcoRⅠ-HF
0.5
XbaI
0.5
ddH2O
21.5
Total
30μl
E1010(162ng/μl)
10x buffer
3
DNA
6.2
EcoRⅠ-HF
0.4
XbaI
0.6
ddH2O
19.8
Total
30μl
B0032(350.5ng/μl)
10x buffer
3
DNA
2.9
EcoRⅠ-HF
0.5
XbaI
0.5
ddH2O
23.1
Total
30μl
Result
We only succeed B0015 ,B032, CYP1A2, and  CYP1A2 COMPOSIE
Show More
Survival
Researcher: Zhao Ming-Cheng, Su Chin-Fong
Place: NTU
Date: September 4th, 2015
Procedure
First we culture DH5α with LB only plate for 15hr. Then,pick one colony in the LB broth,and liquid culture for 15hr.
We divided two categories  A and B.
A:
Take 80μL into 2ml LB broth × 6 tubes and then culture 1 hr.
After 1hr,add 20μL benzo[a]pryene into three tubes(conc. Is 2000ppb(A thousand times the standard value))
And add 20μL DMSO into the other tubes.Then,culture for 3hr.
After 3hr,dilute the broth to 10-6
And take 200μL to spread the plate.
B:
Take 80μL into 2ml LB broth  in a tube And then culture 1 hr.
After 1hr, put them into 6 tubes equally.
Dilute the broth to 5×10-4
Add 0.4μL benzo[a]pryene(2×10-4) in three tubes.
Add 0.4μL DMSO in the other three tubes.
Go to 37 degree Celsius shaking for 10min.
Take 200μL to spread the plate.
Result:
According to the result, Beno[a]pryene does not affect E.coli’s survival.
But Category B is failed because its number of colony is too much.
Benzo[a]pryene Category A
Control Category A
Benzo[a]pryene  CategoryB
Control  CategoryB
Show More
Preprocess of Growth Curve
Researcher: Leng Yi-Yan, Chang Chun-chieh, Lee Ming-Jhen
Place: Fu Jen University Food Science Department
Date: September 3rd & 4th, 2015
Material
LB agar
0.85% saline
Tube
Dish
Procedure
Solid medium
Step1:Get certain weight of LB broth (25g/L) and agar (1.5%)
Step2: Add the powder into the serum bottle with a wash bottle.
Step3: Add certain amount of ddH2O into the serum bottle.
Step4:Dissolve the solution with a spinbar.
Step5:Sterilization for 1.5hr.
Step6:Distribute to each dish (10mL/dish).
Buffer solution
Step1: Get certain weight of saline (0.85%)
Step2: Add the powder into the serum bottle with a wash bottle.
Step3: Add certain amount of ddH2O into the serum bottle.
Step4:Sterilization for 1.5hr.
Step5:Distribute to each tube (9mL/tube).
Show More
Pre-test of Gel Entrapment
Researcher: Leng Yi-Yan, Chang Chun-chieh, Lee Ming-Jhen
Place: Fu Jen University Food Science Department
Date: August 31th & September 1st 2015
Purpose
Make sure if the procedure from others’ reference fits our experiment.
Compare the different effect of using syringe and pipet.
Material
PVA(Polyvinyl alcohol)
SA(Alginate)
BA(Boric acid)
CaCl2
Chitosan
Sodium citrate(Na3C6H5O7•2H2O)
Beaker
Syringe
Pipet
Procedure
PVA-SA
Step1:  Pour some solution into beakers.
Step2: Draw certain amount of 8%PVA-1%SA and add it into 3%BA-1%CaCl2.
SA
Step1:  Pour some solution into beakers.
Step2: Draw certain amount of 3%SA and add it into 1%CaCl2.
ACA
Step1:  Pour some solution into beakers.
Step2: Draw certain amount of 1.5%SA and add it into 100mmol/LCaCl2
Step3: Throw the microcapsule into 0.3%Chitosan.
Step4: Throw the microcapsule into 0.15%SA.
Step5: Use syringe to inject sodium citrate in the microcapsule.
Result
Different method of entrapment
PVA-SA
SA
ACA
White, Teardrop-shaped
colorless (bluish), sphere
Bluish, sphere
Step5 of ACA is infeasible, so we decide to delete this step.
Comparison of syringe and pipet:
syringe
pipet
Consistent size, faster, without bubble
Vary in size, slow, with bubble
Show More
Four-phase Streaking method
Researcher: Leng Yi-Yan, Chang Chun-chieh, Lee Ming-Jhen
Place: Fu Jen University Food Science Department
Date: August 31th, 2015
Purpose
Plate streaking is an important and essential technique in molecular biology. Streaking allows the selection of a single colony from an original mixture of colonies. It also allows for the selection of one single colony from an original plate to be grown up as many colonies on a new plate.
Material
E.coli(DH5α)
Solid medium(LB agar)
Inoculating loop
Alcohol Burner
Procedure
Step1: Sterilize the inoculating loop on an alcohol burner flame and allow it to cool for a few seconds.
Step2: Using the loop, streak the first section of the plate using tight sweeping lines that stay within that section.
Step3: Sterilize the loop and allow it to cool in the air for 15 seconds. Touch the loop to an unused edge of the agar surface to cool it completely before continuing.
Step4: Pull the loop through the previous streak one or two times to re-inoculate the loop with cells.
Step5: Streak section 2 of the plate, avoiding section 1 after the first 1-2 streaks and trying not to overlap the streaks.
Step6: Repeat Step3,4,5 for two more times.
Show More
Gel electrophoresis Gel extraction
Researcher: Zhao Ming-Cheng,Su Chin-Fong
Place: NTU
Date: August 30th, 2015
Procedure
Today,we get the plasmid which be sent from the United States.
They are:
part1:promoter(J23113)-RBS(B0034)-Laccase(K1092104)-RBS(B0034)-
Part2:promoter(J23113)-QsrR binding site-RBS(B0034)-
We set them according to the following
FIRST
SECOND
THIRD
FORTH
Plasmid:part1 4μL
Plasmid:part1 4μL
Plasmid:part2 4μL
Plasmid:part2 4μL
Buffer:M 4μL
Buffer:M 4μL
Buffer:10xM 4μL
Buffer:M 4μL
Restriction enzyme: Xba1 0.4μL
Restriction enzyme: Xba1 and Pst1 0.4μL each
Restriction enzyme: Xba1 0.4μL
Restriction enzyme: Xba1 and Pst1 0.4μL each
DdH2O:31.2μL
DdH2O:30.8μL
DdH2O:31.2μL
DdH2O:30.8μL
Put in 37  degree Celsius for 2 hr and then do Gel electrophoresis.
Result
Show More
Gel extraction
Researcher: Zhao Ming-Cheng,Su Chin-Fong
Place: NTU
Date: August 29th, 2015
Procedure
We take QsrR(K1092000)-Terminator(B0015) and RFP(E1010)-Ter(B0015) to do gel extraction.
Show More
Activation of Culture
Researcher: Chen Szu-Hua, Chang Yu-Ting
Place: Fu Jen University Food Science Department
Date: August 27th, 2015
Material
Chitosan
Acetic acid
Tube
Sonic oscillator
Procedure
Step1:Get certain weight (0.2g, 0.3g, 0.5g, 0.7g) of chitosan.
Step2: Add different concentration of acetic acid 10mL to dissolve chitosan.
Step3: Use sonic oscillator to fasten the dissolution.
Result
Limpid, with liquidity
Limpid at the upperlayer, with bubble at the lowerlayer
Dense,viscous ,with bubble
Dense,viscous ,with bubble
Show More
Plasmid Extraction
Researcher: Chang Ko-Yu, Chu Yi-Chia
Place: Academia Sinica
Date: August 27th, 2015
We extract the plasmid of B0015
Show More
Activation of Culture
Researcher: Leng Yi-Yan, Chang Chun-chieh, Lee Ming-Jhen, Chen Szu-Hua, Chang Yu-Ting
Place: Fu Jen University Food Science Department
Date: August 24th & 25th, 2015
Material
PVA(Polyvinyl alcohol)
SA(Alginate)
BA(Boric acid)
CaCl2
Chitosan
Sodium citrate(Na3C6H5O7•2H2O)
Electronic balance
Spatulas
Beaker
ddH2O
Volumetric flask
Mixing rod
Hotplate
Serum bottle
Procedure
Step1: Get certain weight of chemicals with spatulas by an electronic balance.
Step2: Use a volumetric flask to prepare a certain concentration of solution.
Step3: Dissolve the solution with a stirring rod. (if not dissolving, use the hotplate until the solution becomes clear)
Result
Show More
Send the DNA sequencing
Researcher: Chang Ko-Yu, Chu Yi-Chia
Place: Academia Sinica
Date: August 20th, 2015
Procedure
Add 223.3 μl and 291.1 μl water to two different primer, and the concentration of the two primer is 100uM. Then put the two primer at 55 degree Celsius for fifteen minutes
Diluted the two primer 10-fold to a concentration of 10μM for use
1μg of the plasmid (220ng/μl)
4.5(μl)
ddH2O
5.5(μl)
Total
10(μ)l
Primer
2(μl)
12(μ)l
Send the DNA sequencing
Show More
DH5α-Pretest
Researcher: Zhao Ming-Cheng, Su Chin-Fong
Place: NTU
Date: August 20th, 2015
Procedure
Because we must test E.coli’s Survival in the environment there is benzo[a]pryene by counting the colonies.
First we test how much concentration is the best.
culture (8/17)
STEP1:take 1μL DH5α to spread the plate(no Antibiotic)
STEP2:put in 37 degree Celsius 12~16hr
liquid culture(8/18)
(8/19)
STEP1:put 80μL into 2ml LB broth
STEP2:recovering
STEP3:After 2hr,dilute it to 10-4,10-5,10-6,10-7,and then go to spread the plate (no Antibiotic)
STEP4:After 4hr dilute it to 10-4,10-5,10-6,10-7,and then go to spread the plate (no Antibiotic),6hr and 8hr Similarly
STEP5:Take 200μL out from the tube and spread the plate(AMP+)
STEP6: put in  37 degree Celsius 12~16hr
RESULT:
The number of the colonies in the AMP+ plate is zero.
According to the result, 2hr 10-5 and 4hr 10-6 is the best.
2hr plate from left to right is 10-4,10-5,10-6,10-7
4hr plate from left to right is 10-4,10-5,10-6,10-7
AMP+ Plate
Show More
Extract the plasmid
Researcher: Zhao Ming-Cheng,Su Chin-Fong
Place: NTU
Date: August 19th, 2015
Procedure
Extract QsrR(K1092000)-Ter(B0015)
Show More
Gel electrophoresis
Researcher: Zhao Ming-Cheng,Su Chin-Fong
Place: NTU
Date: August 19th, 2015
Procedure
We take QsrR(K1092000)- Terminator(B0015) to gel electrophoresis
We set them according to the following
FIRST
SECOND
Plasmid:QsrR-Ter 3μL
Plasmid:QsrR-Ter 3μL
Buffer:cutsmart 1μL
Buffer:cutsmart 1μL
Restriction enzyme:Xbal1 0.25μL
Restriction enzyme:Xbal1 Pst1 0.25μL
DdH2O:5.75μL
DdH2O:5.50μL
RESULT:
Success!!
Show More
DH5α-Pretest
Researcher: Zhao Ming-Cheng,Su Chin-Fong
Place: NTU
Date: August 18th, 2015
Because we must test E.coli’s Survival in the environment there is benzo[a]pryene by counting the colonies,First we test how much concentration is the best.
culture (8/17)
STEP1:take 1μL DH5α to spread the plate(no Antibiotic)
STEP2:put in 37 degree Celsius 12~16hr
liquid culture(8/18)
(8/19)
STEP1:put 80μL into 2ml LB broth
STEP2:recovering
STEP3:After 2hr take 200μL out to spread the plate (no Antibiotic)
STEP4:After 4hr take 200μL out to spread the plate (no Antibiotic), 6hr and 8hr Similarly
STEP5:put in 37 degree Celsius 12~16hr
RESULT:
We hope the number of colonies between 30~300,but the result is the colonies is too much!!
Show More
Liquid culture
Researcher: Zhao Ming-Cheng,Su Chin-Fong
Place: NTU
Date: August 18th, 2015
Procedure
Culture QsrR(K1092000)-Ter(B0015)
Show More
Extract the plasmid
Researcher: Zhao Ming-Cheng,Su Chin-Fong
Place: NTU
Date: August 16th, 2015
Procedure
Extract RFP(E1010)-Ter(B0015)
Show More
Liquid culture
Researcher: Zhao Ming-Cheng,Su Chin-Fong
Place: NTU
Date: August 15th, 2015
Procedure
Culture RFP(E1010)-Ter(B0015)
Show More
8/14/2015 Cadmium
Researcher: Chang I, Chi Ying-Cheng
Place: NTNU
Date: August 14th, 2015
We successfully draw out the plasmid from E.coli.
Show More
Extract the plasmid
Researcher: Zhao Ming-Cheng,Su Chin-Fong
Place: NTU
Date: August 14th, 2015
Procedure
Extract QsrR(K1092000) plasmid
Show More
Ligation&Transform
Researcher: Zhao Ming-Cheng,Su Chin-Fong
Place: NTU
Date: August 14th, 2015
Procedure
Ligation and Transform RFP(E1010)-Terminator(B0015)
Show More
Transform
Researcher: Zhao Ming-Cheng,Su Chin-Fong
Place: NTU
Date: August 14th, 2015
Procedure
Transform QsrR(K1092000)
Show More
8/13/2015 Cadmium
Researcher: Chang I, Chi Ying-Cheng
Place: NTNU
Date: August 13th, 2015
We pick up the single column and incubate it.
Show More
Liquid culture
Researcher: Zhao Ming-Cheng,Su Chin-Fong
Place: NTU
Date: August 12th, 2015
Procedure
Culture RFP(E1010) and Terminator(B0015)
Show More
Transform
Researcher: Zhao Ming-Cheng,Su Chin-Fong
Place: NTU
Date: August 12th, 2015
Procedure
Transform QsrR(K1092000)
Show More
Gel electrophoresis
Researcher: Zhao Ming-Cheng,Su Chin-Fong
Place: NTU
Date: August 12th, 2015
Procedure
We take QsrR(K1092000) to gel electrophoresis
We set them according to the following
FIRST
SECOND
Plasmid:3μL
Plasmid:3μL
Restriction enzyme:EcoR1 0.25μL
Restriction enzyme:EcoR1 0.25μL Spel1 0.25μL
Buffer(smartcut):1μL
Buffer(smartcut):1μL
ddH2O:5.75μL
ddH2O:5.50μL
RESULT:
The picture is too blurry.
We think that it is because the concentration is too low.
So we must transform one more time.
Show More
Transformation
Researcher: Chang Ko-Yu, Chu Yi-Chia
Place: Academia Sinica
Date: August 12th, 2015
We transform J22106 this time.
Show More
8/12/2015 Cadmium
Researcher: Chang I, Chi Ying-Cheng
Place: NTNU
Date: August 12th, 2015
We get another RBS B0032 and transform it.
Show More
Send the second DNA sequencing
Researcher: Chang Ko-Yu, Chu Yi-Chia
Place: Academia Sinica
Date: August 11th, 2015
Procedure
Add 223.3 μl and 291.1 μl water to two different primer, and the concentration of the two primer is 100uM. Then put the two primer at 55 degree Celsius for fifteen minutes
Diluted the two primer 10-fold to a concentration of 10μM for use
1μg of the plasmid (220ng/μl)
4.5(μl)
ddH2O
5.5(μl)
Total
10(μ)l
Primer
2(μl)
12(μ)l
Send the DNA sequencing
Show More
8/11/2015 Cadmium
Researcher: Chang I, Chi Ying-Cheng
Place: NTNU
Date: August 11th, 2015
The transforming is failed.
Show More
Gel electrophoresis
Researcher: Zhao Ming-Cheng,Su Chin-Fong
Place: NTU
Date: August 11th, 2015
Procedure
STEP1:test RFP’s andTerminator’s concentration
STEP2:regulate their concentration to 1000np/μL
STEP3:We set them according to the following
FIRST
SECOND
THIRD
FORTH
Plasmid:RFP 4μL
Buffer:cutsmart 4μL
Restriction enzyme: EcoR1 0.4μL
DdH2O:31.2μL
Plasmid:RFP 4μL
Buffer:cutsmart 4μL
Restriction enzyme: EcoR1 and Spe1 0.4μL each
DdH2O:30.8μL
Plasmid:Ter 4μL
Buffer:cutsmart 4μL
Restriction enzyme: EcoR1 0.4μL
DdH2O:31.2μL
Plasmid:Ter 4μL
Buffer:cutsmart 4μL
Restriction enzyme: EcoR1 and Spe1 0.4μL each
DdH2O:30.8μL
RESULT:
Nothing
The result is failed
Show More
Extract the plasmid
Researcher: Zhao Ming-Cheng,Su Chin-Fong
Place: NTU
Date: August 11th, 2015
Procedure
Extract QsrR(K1092000) plasmid
Show More
Transform
Researcher: Zhao Ming-Cheng,Su Chin-Fong
Place: NTU
Date: August 11th, 2015
Procedure
Transform RFP(E1010) and Terminator(B0015)
Show More
Liquid culture
Researcher: Zhao Ming-Cheng,Su Chin-Fong
Place: NTU
Date: August 10th, 2015
Procedure
Culture QsrR(K1092000)
Show More
8/10/2015 Cadmium
Researcher: Chang I, Chi Ying-Cheng
Place: NTNU
Date: August 10th, 2015
We get the part B0034 and transform it.
Show More
Transform
Researcher: Zhao Ming-Cheng,Su Chin-Fong
Place: NTU
Date: August 9th, 2015
Procedure
Transform  QsrR(K1092000)
Show More
8/6/2015 Cadmium
Researcher: Chang I, Chi Ying-Cheng
Place: NTNU
Date: August 6th, 2015
Use the enzyme to check the part length correct or not one more time and determine that we success.
Show More
8/5/2015 Cadmium
Researcher: Chang I, Chi Ying-Cheng
Place: NTNU
Date: August 5th, 2015
Use the enzyme to check the part length correct or not.
Show More
Extract the plasmid
Researcher: Zhao Ming-Cheng,Su Chin-Fong
Place: NTU
Date: August 5th, 2015
Extract RFP(E1010) and Terminator(B0015) plasmid
Show More
Liquid culture
Researcher: Zhao Ming-Cheng,Su Chin-Fong
Place: NTU
Date: August 4th, 2015
Procedure
culture  RFP & Terminator(B0015)
Show More
Confirm the first DNA sequencing and Order the primer of second DNA sequencing
Researcher: Chang Ko-Yu, Chu Yi-Chia
Place: Academia Sinica
Date: August 4th, 2015
the primer of second DNA sequencing
TACAGAGTTCTTGAAGTGGTGGCC
TTTAAAGAAAAAGGGCAGGGTGGT
Show More
8/4/2015 Cadmium
Researcher: Chang I, Chi Ying-Cheng
Place: NTNU
Date: August 4th, 2015
We successfully draw out the plasmid.
Show More
Transform
Researcher: Zhao Ming-Cheng,Su Chin-Fong
Place: NTU
Date: August 3rd, 2015
Procedure
Transform  RFP & Terminator(B0015)
Show More
Plasmid Extraction
Researcher: Chang Ko-Yu, Chu Yi-Chia
Place: Academia Sinica
Date: July 30th, 2015
We extract the plasmid of B0034 and J23119 and test their concentration
B0034
J23119
concentration
208 ng/μl
220 ng/μl
Show More
7/29/2015 Cadmium
Researcher: Chang I, Chi Ying-Cheng
Place: NTNU
Date: July 29th, 2015
We fell to draw out the plasmid.
Show More
Transformation
Researcher: Chang Ko-Yu, Chu Yi-Chia
Place: Academia Sinica
Date: July 28th, 2015
We transform J23119 this time.
Show More
7/28/2015 Cadmium
Researcher: Chang I, Chi Ying-Cheng
Place: NTNU
Date: July 28th, 2015
We incubate our E.coli.
Show More
7/27/2015 Cadmium
Researcher: Chang I, Chi Ying-Cheng
Place: NTNU
Date: July 27th, 2015
We purify the gel , ligate the part, and transform into the E.coli.
Show More
7/24/2015 Cadmium
Researcher: Chang I, Chi Ying-Cheng
Place: NTNU
Date: July 24th, 2015
We use gel electrophoresis to cut our part. Than we purify our gel.
Show More
7/23/2015 Cadmium
Researcher: Chang I, Chi Ying-Cheng
Place: NTNU
Date: July 23rd, 2015
We draw out the plasmid from single column by plasmid mini kit.
And start to use restriction enzyme to cut the part E1010 and B0015.
Show More
7/22/2015 Cadmium
Researcher: Chang I, Chi Ying-Cheng
Place: NTNU
Date: July 22nd, 2015
We draw out the plasmid from single column by plasmid mini kit but failed.
Show More
7/21/2015 Cadmium
Researcher: Chang I, Chi Ying-Cheng
Place: NTNU
Date: July 21st, 2015
We get the part B0015.
Draw out the plasmid from single column by plasmid mini kit.
We are going to do the combine the part in time but our plasmid disappeared.
Show More
7/17/2015 Cadmium
Researcher: Chang I, Chi Ying-Cheng
Place: NTNU
Date: July 17th, 2015
Using gel electrophoresis again and check the part have problem.
Show More
7/16/2015 Cadmium
Researcher: Chang I, Chi Ying-Cheng
Place: NTNU
Date: July 16th, 2015
We use gel electrophoresis to cut our part, and check the part length and determine that the part B0030 which we have has a mistake.
Show More
7/15/2015 Cadmium
Researcher: Chang I, Chi Ying-Cheng
Place: NTNU
Date: July 15th, 2015
Draw out the plasmid from single column by plasmid mini kit{E0040,B0030,E1010,K896008}
Show More
Ag+ adsorb on Au-NPs affect with different heavy metal(Cu,Hg,Cd,Pb)
Researcher: Lin Sheng, Chu Wei-Min
Place: NTU Chemistry
Date: July 14th, 2015
Procedure
total:400 μL
material:
13nm Au-NPs 15mM,Cu 1mM100μM10μM,
Hg 1mM100μM10μM,pH7 Tris-Borate buffer 100mM,
AR 100μM,H2O2 1mM
Caclulate:
Pb2+
buffer
DI water
Au-NPs
Ag+
Mn+
AR
H2O2
target
50mM→5mM
15nM→7.5nM
100μM→10μM
100μM→10μM
1mM→100μM
Control
40μL
220μL
20μL
40μL
0μL
40μL
40μL
1mM
40μL
180μL
20μL
40μL
40μL
40μL
40μL
100μM
40μL
180μL
20μL
40μL
40μL
40μL
40μL
10μM
40μL
180μL
20μL
40μL
40μL(100μM→10μM)
40μL
40μL
1μM
40μL
180μL
20μL
40μL
40μL(10μM→1μM)
40μL
40μL
100nM
40μL
180μL
20μL
40μL
40μL
40μL
40μL
10nM
40μL
180μL
20μL
40μL
40μL(100μM→10μM)
40μL
40μL
1nM
40μL
180μL
20μL
40μL
40μL(10μM→1μM)
40μL
40μL
Cd2+
buffer
DI water
Au-NPs
Ag+
Mn+
AR
H2O2
target
50mM→5mM
15nM→7.5nM
100μM→10μM
100μM→10μM
1mM→100μM
Control
40μL
220μL
20μL
40μL
0μL
40μL
40μL
1mM
40μL
180μL
20μL
40μL
40μL
40μL
40μL
100μM
40μL
180μL
20μL
40μL
40μL
40μL
40μL
10μM
40μL
180μL
20μL
40μL
40μL(100μM→10μM)
40μL
40μL
1μM
40μL
180μL
20μL
40μL
40μL(10μM→1μM)
40μL
40μL
100nM
40μL
180μL
20μL
40μL
40μL
40μL
40μL
10nM
40μL
180μL
20μL
40μL
40μL(100μM→10μM)
40μL
40μL
1nM
40μL
180μL
20μL
40μL
40μL(10μM→1μM)
40μL
40μL
Hg+
buffer
DI water
Au-NPs
Ag+
Mn+
AR
H2O2
target
50mM→5mM
15nM→7.5nM
100μM→10μM
100μM→10μM
1mM→100μM
Control
40μL
220μL
20μL
40μL
0μL
40μL
40μL
1mM
40μL
180μL
20μL
40μL
40μL
40μL
40μL
100μM
40μL
180μL
20μL
40μL
40μL
40μL
40μL
10μM
40μL
180μL
20μL
40μL
40μL(100μM→10μM)
40μL
40μL
1μM
40μL
180μL
20μL
40μL
40μL(10μM→1μM)
40μL
40μL
100nM
40μL
180μL
20μL
40μL
40μL
40μL
40μL
10nM
40μL
180μL
20μL
40μL
40μL(100μM→10μM)
40μL
40μL
1nM
40μL
180μL
20μL
40μL
40μL(10μM→1μM)
40μL
40μL
Cu2+
buffer
DI water
Au-NPs
Ag+
Mn+
AR
H2O2
target
50mM→5mM
15nM→7.5nM
100μM→10μM
100μM→10μM
1mM→100μM
Control
40μL
220μL
20μL
40μL
0μL
40μL
40μL
1mM
40μL
180μL
20μL
40μL
40μL
40μL
40μL
100μM
40μL
180μL
20μL
40μL
40μL
40μL
40μL
10μM
40μL
180μL
20μL
40μL
40μL(100μM→10μM)
40μL
40μL
1μM
40μL
180μL
20μL
40μL
40μL(10μM→1μM)
40μL
40μL
100nM
40μL
180μL
20μL
40μL
40μL
40μL
40μL
10nM
40μL
180μL
20μL
40μL
40μL(100μM→10μM)
40μL
40μL
1nM
40μL
180μL
20μL
40μL
40μL(10μM→1μM)
40μL
40μL
Result
We put all the result into fluorescent reader.
Show More
7/14/2015 Cadmium
Researcher: Chang I, Chi Ying-Cheng
Place: NTNU
Date: July 14th, 2015
We pick single column, and incubate in another tube.
Show More
7/13/2015 Cadmium
Researcher: Chang I, Chi Ying-Cheng
Place: NTNU
Date: July 13th, 2015
We get “k89608” part and transform it.
Show More
Transformation
Researcher: Chang Ko-Yu, Chu Yi-Chia
Place: Academia Sinica
Date: July,2nd, 2015
Because  transformation we did on 6/11 was fail, this week, we did transformation again.
In order to find the cause, we did two control group. One is J23102 the other is the kit provided. And we transform B0034 this time.
Show More
Plasmid Purification
Researcher: Shen Yu-Chun, Chen Sheng-Yuan
Place: NTUCM
Date: June 18th, 2015
Material:
70 % Ethanol
Isopropanol
50 ml centrifuge tubes
Plasmid DNA Midi Kit
pSB1C3-BBa_B0030 in DH5α
pSB1C3-BBa_E1010 in DH5α
pSB1A2-BBa_E0040 in DH5α
pSB1A2-BBa_B0010 in DH5α
Notes:
Add RNase A to PL1 Buffer and store at 4℃
Warm PL2 Buffer in a 37℃ waterbath
Use 100~150 ml of bacterial culture for Midi Kit
Protocol:
Flick a PM Column 2~3 times, then place the PM Column on a conical flask.
Equilibrate the PM Column by applying 5 ml of PL4 Buffer by gravity flow
Harvest the bacterial culture 100 ml by centrifuge at 6000 x g for 10 mins
Add 5 ml of PL1 Buffer to resuspend the cell pellet by vortexing or pipetting
Add 5 ml of PL2 Buffer and mix gently by inverting the tube 10 times. DO NOT VORTEX!!!
Place for 10 mins at room temperature until lysate clears
Add 5 ml of PL3 Buffer and mix immediately by inverting the tube 10 times. DO NOT VORTEX!!!
Centrifuge at 15000g for 20 minutes or filtered with a filter paper
Apply the supernatant from step 8. to the equilibrated PM Column and allow it to flow by gravity flow.
Wash the PM Column by using PL5 Buffer 15 ml and allow the column empty by gravity flow.
Discard the filtrate.
Place PM Column in a clean centrifuge and apply 10 ml of PL6 Buffer to elute DNA by gravity flow
Using eluates from step 12. Add 7.5 ml of room temperature isopropanol. Vortex well and let the mixture sit for 2 minutes.
Centrifuge at 12500 x g for 30 minutes.
Remove the supernatant, then add 3 ml of 70% ethanol to wash.
Centrifuge at 12500 x g for 5 minutes, then dry ethanol at room temperature.
Apply 0.2 ml of TE Buffer to resuspend DNA.
Show More
6/18/2015 Cadmium
Researcher: Chang I, Chi Ying-Cheng
Place: NTNU
Date: June 18th, 2015
We get “B0030, E0040, E1010”part,and transform them.
Show More
Ag+ adsorb on Au-NPs affect with different heavy metal ( Cu, Hg )
Researcher: Lin Sheng, Chu Wei-Min
Place: NTU Chemistry
Date: June 18th, 2015
Procedure
total:400 μL
material:
13nm Au-NPs 1.5mM,Pb 1mM100μM10μM
Cd 1mM100μM10μM,H7 Tris-Borate buffer 100mM,
AR 100μM,H2O21mM
Caclulate:
Pb2+
buffer
DI water
Au-NPs
Ag+
Mn+
AR
H2O2
target
50mM→5mM
15mM→750μM
100μM→10μM
100μM→10μM
1mM→100μM
Control
40μL
220μL
20μL
40μL
0μL
40μL
40μL
100μM
40μL
180μL
20μL
40μL
40μL(1mM→100μM)
40μL
40μL
10μM
40μL
180μL
20μL
40μL
40μL(100μM→10μM)
40μL
40μL
1μM
40μL
180μL
20μL
40μL
40μL(10μM→1μM)
40μL
40μL
Cd2+
buffer
DI water
Au-NPs
Ag+
Mn+
AR
H2O2
target
50mM→5mM
15mM→750μM
100μM→10μM
100μM→10μM
1mM→100μM
Control
40μL
220μL
20μL
40μL
0μL
40μL
40μL
100μM
40μL
180μL
20μL
40μL
40μL(1mM→100μM)
40μL
40μL
10μM
40μL
180μL
20μL
40μL
40μL(100μM→10μM)
40μL
40μL
1μM
40μL
180μL
20μL
40μL
40μL(10μM→1μM)
40μL
40μL
Pb2+
buffer
DI water
Au-NPs
Ag+
Mn+
AR
H2O2
target
50mM→5mM
15mM→750μM
100μM→10μM
100μM→10μM
1mM→100μM
Control
40μL
260μL
20μL
0μL
0μL
40μL
40μL
100μM
40μL
220μL
20μL
0μL
40μL(1mM→100μM)
40μL
40μL
10μM
40μL
220μL
20μL
0μL
40μL(100μM→10μM)
40μL
40μL
1μM
40μL
220μL
20μL
0μL
40μL(10μM→1μM)
40μL
40μL
Cd2+
buffer
DI water
Au-NPs
Ag+
Mn+
AR
H2O2
target
50mM→5mM
15mM→750μM
100μM→10μM
100μM→10μM
1mM→100μM
Control
40μL
260μL
20μL
0μL
0μL
40μL
40μL
100μM
40μL
220μL
20μL
0μL
40μL(1mM→100μM)
40μL
40μL
10μM
40μL
220μL
20μL
0μL
40μL(100μM→10μM)
40μL
40μL
1μM
40μL
220μL
20μL
0μL
40μL(10μM→1μM)
40μL
40μL
Result:
We put all the result into fluorescent reader.
This one is stand for Pb+.
This one is stand for Cd2+.
Show More
6/15/2015 Cadmium
Researcher: Chang I, Chi Ying-Cheng
Place: NTNU
Date: June 15th, 2015
We do the ligation, and then transform the plasmid into E.coli
Mixing with LB broth, we smear the solid on the plate.
Show More
6/14/2015 Cadmium
Researcher: Chang I, Chi Ying-Cheng
Place: NTNU
Date: June 14th, 2015
This is our first time do the experience.
We are going to use the restriction enzyme to cut the sample part {E GFP} and add a part by ligation.
So first, we use enzyme to cut the plasmid, and use gel electrophoresis to cut our part.
Than we purify our gel.
Show More
Transformation
Researcher: Shen Yu-Chun, Chen Sheng-Yuan
Place: NTUCM
Date: June 11th, 2015
Material:
E.coli  DH5α (ps: E.coli should be kept on the ice at all time)
Plasmid : pSB1A2-BBa_E0040(Ampicillin-resistant)          GFP
Plasmid : pSB1C3-BBa_B0010(Chloramphenicol-resistant)     Terminator
Plasmid : pSB1C3-BBa_B0030(Chloramphenicol-resistant)     RBS
Plasmid : pSB1C3-BBa_E1010(Chloramphenicol-resistant)     RFP
LB broth (10 g /L Tryptone, 5 g/L Yeast extract, 5 g/L NaCl )
LB plate(LB broth + 1% agar)
Antibiotics : Ampicillin & Chloramphenicol (100μg/mL for each)
Protocol:
Add 0.5μL of plasmid into 100μL of E.coli and mix by tapping the tube.(ps: the quantity of plasmid is dependent on bacterial competency)
Put the tube on the ice for 20~30mins.
Remove from the ice and put the tube to the 42℃ hot bath for 45 sec.
After heat shock, put the tube into the ice IMMEDIATELY for 2 mins.
Add 500 μL of LB broth into the tube and incubate for 30~60 mins at 37℃
Take out the tube and put into 6000 rpm centrifuge for 5 mins.
Remove supertant and remain about 100μL of LB to resuspend
Spread onto LB plate containing the appropriate antibiotic
Incubate overnight for 12~16 hrs at 37℃.
Show More
Transformation
Researcher: Chang Ko-Yu, Chu Yi-Chia
Place: Academia Sinica
Date: June 11th, 2015
This week, we did transformation.
We transform B0034, B0015, I765001, I732005.
Show More
Ag+ adsorb on Au-NPs affect with different heavy metal ( Cu, Hg )
Researcher: Lin Sheng, Chu Wei-Min
Place: NTU Chemistry
Date: June 4th, 2015
Procedure
total:400 μL
material:
13nm Au-NPs 15mM,Cu 1mM100μM10μM
Hg 1mM100μM10μM,pH7 Tris-Borate buffer 100mM
AR 100μM,H2O2 1mM
Caclulate:
Cu2+
buffer
DI water
Au-NPs
Ag+
Mn+
AR
H2O2
target
50mM→5mM
15mM→750μM
100μM→10μM
100μM→10μM
1mM→100μM
Control
40μL
220μL
20μL
40μL
0μL
40μL
40μL
100μM
40μL
180μL
20μL
40μL
40μL(1mM→100μM)
40μL
40μL
10μM
40μL
180μL
20μL
40μL
40μL(100μM→10μM)
40μL
40μL
1μM
40μL
180μL
20μL
40μL
40μL(10μM→1μM)
40μL
40μL
Hg+
buffer
DI water
Au-NPs
Ag+
Mn+
AR
H2O2
target
50mM→5mM
15mM→750μM
100μM→10μM
100μM→10μM
1mM→100μM
Control
40μL
220μL
20μL
40μL
0μL
40μL
40μL
100μM
40μL
180μL
20μL
40μL
40μL(1mM→100μM)
40μL
40μL
10μM
40μL
180μL
20μL
40μL
40μL(100μM→10μM)
40μL
40μL
1μM
40μL
180μL
20μL
40μL
40μL(10μM→1μM)
40μL
40μL
Cu2+
buffer
DI water
Au-NPs
Ag+
Mn+
AR
H2O2
target
50mM→5mM
15mM→750μM
100μM→10μM
100μM→10μM
1mM→100μM
Control
40μL
260μL
20μL
0μL
0μL
40μL
40μL
100μM
40μL
220μL
20μL
0μL
40μL(1mM→100μM)
40μL
40μL
10μM
40μL
220μL
20μL
0μL
40μL(100μM→10μM)
40μL
40μL
1μM
40μL
220μL
20μL
0μL
40μL(10μM→1μM)
40μL
40μL
Hg+
buffer
DI water
Au-NPs
Ag+
Mn+
AR
H2O2
target
50mM→5mM
15mM→750μM
100μM→10μM
100μM→10μM
1mM→100μM
Control
40μL
260μL
20μL
0μL
0μL
40μL
40μL
100μM
40μL
220μL
20μL
0μL
40μL(1mM→100μM)
40μL
40μL
10μM
40μL
220μL
20μL
0μL
40μL(100μM→10μM)
40μL
40μL
1μM
40μL
220μL
20μL
0μL
40μL(10μM→1μM)
40μL
40μL
Result:
We put all the result into fluorescent reader.
The horizontal axis:1 means control, 2 means 100μM, 3 means 10μM, 4 means 1μM.And series1 represent no Ag+ adsorb on Au-NPs,series2 stand for the experiment with Ag+ adsorb on Au-NPs which had time mistake(there are one hour shock between adding Mn+ and AR), and the third series is the correct one with Ag+ adsorb on Au-NPs.
Show More
Au-NPs affect with different heavy metal (Cu, Pb, Hg,Zn)
Researcher: Lin Sheng, Chu Wei-Min
Place: NTU Chemistry
Date: May 28th, 2015
Procedure
total:500 μL
material:
3nm Au-NPs 1.5mM,Cu 10mM1mM100μM10μM
Cd 10mM1mM100μM10μM,Hg 10mM1mM100μM10μM
Zn10mM1mM100μM10μM,pH7 Pb buffer 100mM
Caclulate:
Cu
buffer
DI water
Au-NPs
Mn+
target
100mM→10mM
1.5mM→3μM
0.1x
Control
50μL
350μL
100μL
0
1mM
50μL
300μL
100μL
50μL(10mM→1mM)
100μM
50μL
300μL
100μL
50μL(1mM→100μM)
10μM
50μL
300μL
100μL
50μL(100μM→10μM)
1μM
50μL
300μL
100μL
50μL(10μM→1μM)
Cd
buffer
DI water
Au-NPs
Mn+
target
100mM→10mM
1.5mM→3μM
0.1x
Control
50μL
350μL
100μL
0
1mM
50μL
300μL
100μL
50μL(10mM→1mM)
100μM
50μL
300μL
100μL
50μL(1mM→100μM)
10μM
50μL
300μL
100μL
50μL(100μM→10μM)
1μM
50μL
300μL
100μL
50μL(10μM→1μM)
Hg
buffer
DI water
Au-NPs
Mn+
target
100mM→10mM
1.5mM→3μM
0.1x
Control
50μL
350μL
100μL
0
1mM
50μL
300μL
100μL
50μL(10mM→1mM)
100μM
50μL
300μL
100μL
50μL(1mM→100μM)
10μM
50μL
300μL
100μL
50μL(100μM→10μM)
1μM
50μL
300μL
100μL
50μL(10μM→1μM)
Zn
buffer
DI water
Au-NPs
Mn+
target
100mM→10mM
1.5mM→3μM
0.1x
Control
50μL
350μL
100μL
0
1mM
50μL
300μL
100μL
50μL(10mM→1mM)
100μM
50μL
300μL
100μL
50μL(1mM→100μM)
10μM
50μL
300μL
100μL
50μL(100μM→10μM)
1μM
50μL
300μL
100μL
50μL(10μM→1μM)
Result:
H1~H4 Hg1mM~1μM
H4~H8 Cd1mM~1μM
G1~G4 Cu1mM~1μM
G5~G8 Zn1mM~1μM
Show More
TRANSFORMATION
Researcher: Han Yun-An, Chang Yu-Ting, Lee Chang-Lung
Place: TMU lab
Date: May 28th, 2015
Materials
Resuspended DNA (B0015, E0240, E0840, E0420, E0430, I13001, J23119, B1006)
Competent cells (20μl DH5α per transformation)
Ice (in ice container)
2ml tube
42℃ water bath
Petri dishes with LB agar and appropriate antibiotic
Spreader
37℃ incubator
SOC Media (180μl per transformation)
Pipettman
Centrifuge
Procedure
Start thawing the competent cells on ice.
Add 20 μL of thawed competent cells into pre-chilled 2ml tube, labelled for your control.
Add 1 μL of the DNA to the 2ml tube. Pipet up and down a few times, gently. Make sure to keep the competent cells on ice.
Close tubes and incubate the cells on ice for 30 minutes.
Heat shock the cells by immersion in a pre-heated water bath at 42℃ for 60 seconds.
Incubate the cells on ice for 2 minutes for recover.
Add 180 μL of SOC Media(without antibiotic) and then incubate the cells at 37℃ for 1 hour while the tubes are shaking(200~250 rpm).
Centrifuge the cells at 2000 rpm for 2 minutes.
Remove 100 μL of the supernatant by the pipettmen.
For the control, label two petri dishes with LB agar and the appropriate antibiotic(chloramphenicol). Plate 100 μl of the transformation onto the dishes, and spread.
Incubate the plates at 37℃ for 12-16 hours, making sure the agar side of the plate is up.
Store the plate in 4 ℃.
Show More
TRANSFORMATION
Researcher: Han Yun-An, Chang Yu-Ting, Lee Chang-Lung
Place: TMU lab
Date: May 21th, 2015
Materials
Resuspended DNA (Resuspend well in 10μl dH20, pipette up and down several times, let sit for a few minutes)
Competent cells (100μl DH5α per transformation)
Ice (in ice container)
2ml tube (1 per transformation)
42℃ water bath
Petri dishes with LB agar and appropriate antibiotic (1 per transformation)
Spreader  (2 per transformation)
37℃ incubator
LB broth (900μl per transformation)
Pipettman
Centrifuge
Procedure
Start thawing the competent cells on ice.
Add 100 μL of thawed competent cells into pre-chilled 2ml tube, labelled for your control.
Add 1 μL of the DNA to the 2ml tube. Pipet up and down a few times, gently. Make sure to keep the competent cells on ice.
Close tubes and incubate the cells on ice for 30 minutes.
Heat shock the cells by immersion in a pre-heated water bath at 42℃ for 60 seconds.
Incubate the cells on ice for 2 minutes for recover.
Add 900 μL  of LB broth(without antibiotic) and thenincubate the cells at 37℃ for 1 hour while the tubes are shaking(200~250 rpm).
Centrifuge the cells at 2000 rpm for 2 minutes.
Remove 800 μL of the supernatant by the pipettmen.
For the control, label two petri dishes with LB agar and the appropriate antibiotic(s). Plate 150 μl of the transformation onto the dishes, and spread.
Incubate the plates at 37℃ for 12-16 hours, making sure the agar side of the plate is up.
After picking colonies, store the plate in 4 ℃.
Show More
TRANSFORMATION
Researcher: Han Yun-An, Chang Yu-Ting, Lee Chang-Lung
Place: TMU lab
Date: May 7th, 2015
Materials
Resuspended DNA (Resuspend well in 10μl ddH20, pipette up and down several times, let sit for a few minutes)
Competent cells (100μl DH5α per transformation)
Ice (in ice container)
2ml tube (1 per transformation)
42℃ water bath
Petri dishes with LB agar and appropriate antibiotic (1 per transformation)
Spreader  (1 per transformation)
37℃ incubator
LB broth (900μl per transformation)
Pipettman
Centrifuge
Procedure
Start thawing the competent cells on ice.
Add 100 μL of thawed competent cells into pre-chilled 2ml tube, labelled for your control.
Add 1 μL of the DNA to the 2ml tube. Pipet up and down a few times, gently. Make sure to keep the competent cells on ice.
Close tubes and incubate the cells on ice for 30 minutes.
Heat shock the cells by immersion in a pre-heated water bath at 42℃ for 60 seconds.
Incubate the cells on ice for 2 minutes for recover.
Add 900 μL  of LB broth(without antibiotic) and thenincubate the cells at 37℃ for 1 hour while the tubes are shaking(200~250 rpm).
Centrifuge the cells at 2000 rpm for 2 minutes.
Remove 800 μL of the supernatant by the pipettmen.
For the control, label two petri dishes with LB agar and the appropriate antibiotic(s). Plate 50 μl of the transformation onto the dishes, and spread.
Incubate the plates at 37℃ for 12-16 hours, making sure the agar side of the plate is up, and then store the plate in 4 ℃.
Show More
First-Strand cDNA Synthesis Using M-MLV RT
Researcher: Chang Ko-Yu, Chu Yi-Chia
Place: Academia Sinica
Date: April 30th, 2015
Material
randon primer 1㎕
2MUG RNA F10 1.95㎕
2MUG RNA m1 1.95㎕
10mM dNTP 1㎕
ddH2O
5× FSB 4㎕
0.1M DTT 2㎕
RNase out 1㎕
M-MLV RT 1㎕
eppendorf
Procedure
A 20-㎕ reaction volume can be used for 1 ng-5μg of total RNA or 1-500 ng of mRNA.
Add the following components to a nuclease-free microcentrifuge tube:
1㎕ oligo(dT)12-18(500μg/ml), or 50-250 ng random primers, or 2pmole gene -specific primer
1 ng to 5μG total RNA or 1 ng to 500ng of mRNA
1㎕ 10 mM dNTP Mix(10 mM each dATP,dGTP,dCTP and dTTP at neutral pH)
Sterile, distilled water to 12㎕
Heat mixture to 65℃ for 5 min an quick chill on ice. Collect the contents of the tube by brief centrifugation and add:
4㎕ 5X First-Strand Buffer
2㎕ 0.1 M DTT
1㎕ RNaseOUTTM Recombinant Ribonuclease Inhibitor (40 units/㎕)
(Note: When using less than 50 ng of starting RNA, the addition of RNaseOUTTM is essential.)
Mix contents of the tube gently and incubate at 37℃ for 2 min.
Add 1㎕(200 units) of M-MLV RT,a and mix by pipetting gently up and down. If using random primers, incubate tube at 25℃ for 10 min.
Incubate 50 min at 37℃.
Inactivate the reaction by heating at 70℃ for 15 min.
Result
Show More
TRANSFORMATION
Researcher: Han Yun-An, Chang Yu-Ting, Lee Chang-Lung
Place: TMU lab
Date: April 30th, 2015
Materials
Resuspended DNA (J04450)
Competent cells (100μl DH5α per transformation)
Ice (in ice container)
2ml tube
42℃ water bath
Petri dishes with LB agar and appropriate antibiotic
Spreader
37℃ incubator
LB broth (900μl per transformation)
Pipettman
Centrifuge
Procedure
Start thawing the competent cells on ice.
Add 100 μL of thawed competent cells into pre-chilled 2ml tube, labelled for your control.
Add 2 μL of the DNA to the 2ml tube. Pipet up and down a few times, gently. Make sure to keep the competent cells on ice.
Close tubes and incubate the cells on ice for 30 minutes.
Heat shock the cells by immersion in a pre-heated water bath at 42℃ for 60 seconds.
Incubate the cells on ice for 2 minutes for recover.
Add 900 μL  of LB broth(without antibiotic) and thenincubate the cells at 37℃ for 1 hour while the tubes are shaking(200~250 rpm).
Centrifuge the cells at 2000 rpm for 2 minutes.
Remove 800 μL of the supernatant by the pipettmen.
For the control, label two petri dishes with LB agar and the appropriate antibiotic(s). Plate 100 μl and 100 μl of the transformation onto the dishes, and spread.
Incubate the plates at 37℃ for 12-16 hours, making sure the agar side of the plate is up.
After picking colonies, store the plate in 4 ℃.
Show More
Plasmid Purification & Gel electrophoresis
Researcher: Shen Yu-Chun, Chen Sheng-Yuan
Place: NTUCM
Date: April 30th, 2015
Material:
Resuspension Buffer S1
Lysis Buffer S2
Neutralization Buffer S3
Equilibration Buffer N2
Wash Buffer N3
Elution Buffer N5
Overnight bacteria
Room-temperature isopropanol
1% agarose
TE buffer (using at Gel electrophoresis)
Protocol:(Plasmid Purification)
Harvest bacteria from an LB culture by centrifugation at 6000 rpm for 10 mins at 4℃
Resuspend the pellet of bacterial cells in Buffer S1.
Add Buffer S2 the suspension .Mix by inverting the tube for 6~8 times. Incubate the mixture at 18~25℃ for 2~3 mins.
Add Buffer S3 (pre-cooled at 4℃ ) to the suspension. Immediately mix the lysate by inverting the flask 6~8 times.
Equilibrate a Column with Buffer N2
Place the folded filter in a funnel of appropriate size. Wet the filter with a few drop of Buffer N2 and load the bacterial lysate onto the wet filter.
Load the cleared lysate from 6 onto the column. Allow the column to empty by gravity flow.
Wash the column with Buffer N3.
Elute the plasmid DNA with Buffer N5.
Add room-temperature isopropanol to precipitate the eluted plasmid DNA. Mix carefully and centrifuge at 12000 g for 30 mins at 4℃. Carefully discard the supertant.
Add room-temperature 70% ethanol to the pellet. Vortex briefly and centrifuge at 12000 g for 10 min at 18~25℃.
Carefully remove ethantol from the tube with a pipette tip. Allow the pellet to dry at 18~25℃ no iess than the indicated time.
Show More
Transform
Researcher: Zhao Ming-Cheng,Su Chin-Fong
Place: NTU
Date: April 30th, 2015
Procedure
Transform T7promoter-RBS-QsrR   BBa_K1092001
Show More
First-Strand cDNA Synthesis Using M-MLV RT
Researcher: Chang Ko-Yu, Chu Yi-Chia
Place: Academia Sinica
Date: April 30th, 2015
Preparing Samples
Homogenizing samples
Determine your sample type, and perform homogenization at room temperature according to the table below. The sample volume should not exceed 10% of the volume of TRIzol Reagent used for homogenization. Be sure to use the indicated amount of TRIzol Reagent, because an insufficient volume can result in DNA contamination of isolated RNA.
Sample Type
Action
Tissues
Add 1mL TRIzol Reagent per 50-100 mg of tissue sample.
Homogenize sample using a glass-Teflon or power homogenizer.
Note: Process or freeze tissue samples immediately upon collection.
RNA Isolation Procedurec
Always use the appropriate precautions to avoid RNase contamination when preparing and handling RNA.
RNA precipitation
(Optional) When precipitating RNA from small sample quantities (<106 cells or < 10 mg tissue), add 5-10μg of RNase-free glycogen as a carrier to the aqueous phase.
Note: Glycogen is co-precipitated with the RNA,but does not inhibit first-strand synthesis at concentrations ≤4mg/mL,and does not inhibit PCR.
Add 0.5mL of 100% isopropanol to the aqueous phase, per 1mL of TRIzol Reagent used for homogenization.
Incubate at room temperature for 10 minutes.
Centrifuge at 12,000 × g for 10 minutes at 4℃.
Note: The RNA is often invisible prior to centrifugation, and forms a gel-like pellet on the side and bottom of the tube.
Proceed to RNA wash.
Proceed to RNA wash.
Remove the supernatant from the tube,leaving only the RNA pellet.
Wash the pellet,with 1mL of 75% ethanol per 1mL of TRIzol Reagent used in the initial homogenization.
Note: The RNA can be stored in 75% ethanol at least 1 year at-20℃, or at least 1 week at 4℃.
Vortex the sample briefly,then centrifuge the tube at 7500 × g for 5 minutes at 4℃.Discard the wash.
Vacuum or air dry the RNA pellet for 5-10 minutes. Do not dry the pellet by vacuum centrifuge.
Note: Do not allow the RNA to dry completely,because  the  pellet  can  lose  solubility.Partially dissolved RNA samples have an A260/280 ratio<1.6.
Proceed to RNA resuspension.
RNA resuspension
Resuspend the RNA pellet in RNase-free water or 0.5% SDS solution(20-50㎕) by passing the solution up and down several times through a pipette tip.
Note: Do not dissolve the RNA in 0.5% SDS if it is to be used in subsequent enzymatic reactions.
Incubate in a water bath or heat block set at 55-60℃ for 10-15 minutes.
Proceed to downstream application, or store at -70℃.
Show More
Transformation
Researcher: Shen Yu-Chun, Chen Sheng-Yuan
Place: NTUCM
Date: April 13th, 2015
Material:
E.coli  DH5α (ps: E.coli should be kept on the ice at all time)
Plasmid pBR322(Ampicillin-resistant)
LB broth (10 g /L Tryptone, 5 g/L Yeast extract, 5 g/L NaCl )
LB plate(LB broth + 1% agar)
Antibiotics : Ampicillin(100μg/mL)
Protocol:
Add 5μL of DNA into 100μL of E.coli and mix by tapping the tube.    (ps: the quantity of DNA is dependent on bacterial competency)
Put the tube on the ice for 20~30mins.
Remove from the ice and put the tube to the 42℃ hot bath for 45 sec.
After heat shock, put the tube into the ice IMMEDIATELY for 2 mins.
Add 500 μL of LB broth into the tube and incubate for 30~60 mins at  37℃
Take out the tube and put into 6000 rpm centrifuge for 5 mins
Remove supertant and remain about 100μL of LB to resuspend
Spread onto LB plate containing the appropriate antibiotic
Incubate overnight for 12~16 hrs at 37℃ and you will see so many colonies on the LB plate!!!(A colony is a white dot on the picture below)
Show More
Plasmid Extraction
Researcher: Chang Ko-Yu, Chu Yi-Chia
Place: Academia Sinica
Date: April 2nd, 2015
Procedure
Harvesting
Transfer 1.5ml of cultured bacterial cells to a 1.5ml microcentrifuge tube. Centrifuge at 13000 x g for 1 minute at room temperature.
Resuspension
Add 200μl of PD1 Buffer to a new 1.5ml microcentrifuge tube. Add 2μl of TrueBlue Lysis Buffer to the same 1.5ml microcentrifuge tube then mix by shaking gently.
Cell Lysis
Add 200μl of PD2 to the resuspended sample then mix genlly by inverting the tube 10 times. Let stand at room temperature for at least 2 minutes to ensure the lysate is homogeneous. Do not exceed 5 minutes.
Neutralization
Add 300 μl of PD3 Buffer then mix immediately by inverting the tube 10 times. Centrifuge at 13000 x g for 3 minutes at room temperature.
DNA Binding
Transfer all of the supernatant to thePDH Columg. Centrifuge at 13000 x g for 30 seconds at room temperature then discard th flow-through. Place the PDH Column back in the 2 ml Collection Tube.
Wash
For Improved Downstream Sequencing Reactions
Add 400 μl of W1 Buffer into the PDH Column. Centrifuge at 13000 x g for 30 seconds. Discard the flow-through then place the PDH Column back in the 2 ml Collection Tube. Proceed with Wash Buffer addition.
For Standard Plasmid DNA Purification
Add 600 μl of Wash Buffer into the PDH column. Centrifuge at 13000 x g for 30 seconds at room temperature. Discard the flow through then place the PDH Column back in the 2 ml Collection Tube. Centrifuge at 13000 x g for 3 minutes at room temperature to dry the column matrix.
Show More
Colony PCR
Researcher: Han Yun-An, Chang Yu-Ting, Lee Chang-Lung
Place: TMU lab
Date: April 2nd, 2015
PCR mixer
10X PCR Buffer,-Mg
to 100μl
50mM MgCl2
10μl
10mM dNTP Mix
3μl
Taq DNA Polymerase(5U/μl)
0.4μl
10μM forward primer
5μl
10μM reverse primer
5μl
∗Mix briefly centrifuge the components.
Put the mix into each of ten tubes. Eight of them are colonies, and the others are positive and negative control.
Put all tubes into the PCR machine and set up like below:
Step
Temperature(℃)
Time
Initial Denaturation
95
5min
30 PCR Cycles
Denature
94
15sec
Anneal
55
15sec
Extend
72
30sec
Final Extension
72
7min
Hold
25
indefinitely
When it finished, we run gel.
Show More
Transformation
Researcher: Chang Ko-Yu, Chu Yi-Chia
Place: Academia Sinica
Date: March 26th, 2015
Material needed
70 % ethanol
Bba_J23102  (2λ)
Dn5α (20λ)
soc  (200λ)
Peni 50 mg/ml
chrlo  34mg/ml
ministart
dish x10
pipetman,tip
eppendorf
Add 2λ plasmid and 20λ E-coli into PCR tube in the ice bucket.
Place the tubes on ice for 30min
Heatshock 42℃ for 45secs
After Heatshock, put into the ice bucket for 30 min.
Add soc 200λ to repair the cell wall Culture in the 37℃ incubator for 1.5 hr
Finish smear dish in hood.
Result
Show More
Transform
Researcher: Zhao Ming-Cheng,Su Chin-Fong
Place: NTU
Date: March 26th, 2015
Procedure
Step1：Spread some culture media on the petri dish and heated in 37℃ for 20min in the oven.
STEP2：Take the plastid and bacterium out of the refrigerator(-80℃) and heated in 37℃ for 3min in the oven.
STEP3：Take the plastid and bacterium out and put it into the centrifuge.
STEP4：Use a L-type glass rod and spread the plastid and bacterium on the petri dish tell it become gluey.
STEP5：Cover up the petri dish and wait for about 16 to 24 hours to reform.
Show More
Ligation
Researcher: Han Yun-An, Chang Yu-Ting, Lee Chang-Lung
Place: TMU lab
Date: March 26th, 2015
To an autoclaved, 1.5ml microcentrifuge tube, add the following:
For Cohesive Ends
5X Ligase Reaction
4 μl
Vector D
3 to 30 fmol
Insert D
9 to 90 fmol
T4 DNA Ligase (
1 unit (in 1 μl)
Autoclaved distilled
to 20 μl
Mix gently. Centrifuge briefly the contents to the bottom of the tube.
Incubate at room temperature for 5 min.
Use 2 μl of the ligation reaction to transform 100 μl of competent cells.
Show More
3/19/2015
Researcher: Han Yun-An, Chang Yu-Ting, Lee Chang-Lung
Place: TMU
Date: Match 19th, 2015
Add restriction enzyme (EcoRI / SpeI)
Total:40ul
Plasmid DNA:500ng (77.7ng/ul, 500/77.7≒6.5ul)
ddH2O
22.5ul
10Xbfr(EcoRI)
4ul
10XBSA
4ul
Plasmid DNA
6.5ul
EcoRI
1.5ul
SpeI
1.5ul
Total
40ul
Show More
Gel Extraction
Researcher: Han Yun-An, Chang Yu-Ting, Lee Chang-Lung
Place: TMU lab
Date: March 19th, 2015
Excise the DNA fragment from the agarose gel with a clean, sharp scalpel.
Weigh the gel slice in a colorless tube. Add 3 volumes Buffer QG to 1 volume gel (100 mg gel~100 μl).The maximum amount of gel per spin column is 400 mg.For>2% agarose gels,add 6 volumes Buffer QG.
Incubate at 50℃ for 10 min (or until the gel slice has completely dissolved). Vortex the tube every 2–3 min to help dissolve gel. After the gel slice has dissolved completely, check that the color of the mixture is yellow (similar to Buffer QG without dissolved agarose). If the color of the mixture is orange or violet, add 10μl 3 M sodium acetate, pH 5.0, and mix. The mixture turns yellow.
Add 1 gel volume isopropanol to the sample and mix.
Place a QIAquick spin column in a provided 2 ml collection tube or into a vacuum manifold. To bind DNA, apply the sample to the QIAquick column and centrifuge for 1 min or apply vacuum to the manifold until all the samples have passed through the column. Discard flow-through and place the QIAquick column back into the same tube. For sample volumes of&62;800μl, load and spin/apply vacuum again.
If DNA will subsequently be used for sequencing, in vitro transcription, or microinjection, add 500 μl Buffer QG to the QIAquick column and centrifuge for 1 min or apply vacuum. Discard flow-through and place the QIAquick column back into the same tube.
To wash, add 750μl Buffer PE to QIAquick column and centrifuge for 1 min or apply vacuum. Discard flow-through and place the QIAquick column back into the same tube. Note: If the DNA will be used for salt-sensitive applications (e.g., sequencing, bluntended ligation), let the column stand 2–5 min after addition of Buffer PE. Centrifuge the QIAquick column in the provided 2 ml collection tube for 1 min to remove residual wash buffer.
Place QIAquick column into a clean 1.5 ml microcentrifuge tube.
To elute DNA, add 50 μl Buffer EB (10 mM Tris·Cl, pH 8.5) or water to the center of the QIAquick membrane and centrifuge the column for 1 min. For increased DNA concentration, add 30 μl Buffer EB to the center of the QIAquick membrane, let the column stand for 1 min, and then centrifuge for 1 min. After the addition of Buffer EB to the QIAquick membrane, increasing the incubation time to up to 4 min can increase the yield of purified DNA.
If purified DNA is to be analyzed on a gel, add 1 volume of Loading Dye to 5 volumes of purified DNA. Mix the solution by pipetting up and down before loading the gel.
Show More
High Copy Number Protocol
Researcher: Han Yun-An, Chang Yu-Ting, Lee Chang-Lung
Place: TMU lab
Date: March 18th, 2015
Harvesting
Transfer 1.5ml of bacterial culture to a microcentrifuge tube. Centrifuge for 2 min at 6,000 rpm in a microcentrifuge and discard the supernatant.
Resuspension
Add 200μl of PS1 Buffer(RNaseA added)to the tube and resuspend the cell pellet by vortexing or pipetting.
Lysis
Add 200μl of PS2 Buffer and mix gently by inverting the tube 10 times. Do not vortex, avoid shearing genomic DNA. Stand for 3 minutes at room temperature until lysate clears.
Neutralization
Add 300μl of PS3 Buffer and mix immediately by inverting the tube 10 times. Do not vortex. Until the precipitate formation, shake a few times to broken it. Incubate at room temperature for 5 minutes to digest RNA. Centrifuge for 5 minutes at 13,000 rpm.
DNA Binding
Place a PS Column in a Collection tube. Apply the clear lysate (supernatant) from Step 4 to the PS Column. Centrifuge at 13,000 rpm for 30 seconds.
Discard the flow-through and place the PS Column back into the Collection Tube.
Wash
Add 400μl of W1 Buffer in the PS Column. Centrifuge at 13,000 rpm for 30 seconds.
Discard the flow-through and place the PS Column back into the Collection Tube.
Add 600μl of W2 Buffer(ethanol added) in the PS Column. Centrifuge at 13,000 rpm for 30.
Discard the flow-through and place the PS Column back into the Collection Tube. Centrifuge again for 3 minutes at 13,000 rpm to dry the Column matrix.
DNA Elution
Transfer dried PS Column to a clean microcentrifuge tube. Add 50μl of distilled water into the center of the column matrix. Stand for 2 minutes until distilled water is absorbed by the matrix. Centrifuge for 2 minutes at 13,000 rpm to elute purified DNA.
If would like more production, repeat this elution step again.
Show More
TRANSFORMATION
Researcher: Han Yun-An, Chang Yu-Ting, Lee Chang-Lung
Place: Taipei Medical University
Date: March 12th, 2015
Materials
Resuspended DNA (Resuspend well in 10μl dH20, pipette up and down several times, let sit for a few minutes)
Competent cells (100μl DH5α per transformation)
Ice (in ice container)
2ml tube (1 per transformation)
42℃ water bath
Petri dishes with LB agar and appropriate antibiotic (2 per transformation)
Spreader (2 per transformation)
37℃ incubator
10pg/μl RFP Control (pSB1C3 w/ BBa_J23102)
LB broth (900μl per transformation)
Pipettman
Centrifuge
Procedure
Start thawing the competent cells on ice.
Add 100 μL of thawed competent cells into pre-chilled 2ml tube, labelled for your control.
Add 1 μL of the DNA with RFP Control to the 2ml tube. Pipet up and down a few times, gently. Make sure to keep the competent cells on ice.
Close tubes and incubate the cells on ice for 30 minutes.
Heat shock the cells by immersion in a pre-heated water bath at 42℃ for 60 seconds.
Incubate the cells on ice for 3 minutes for recover.
Add 900 μL of LB broth(without antibiotic) and thenincubate the cells at 37℃ for 1 hour while the tubes are shaking(200~250 rpm).
Centrifuge the cells at 2000 rpm for 2 minutes.
Remove 800 μL of the supernatant by the pipettmen.
For the control, label two petri dishes with LB agar and the appropriate antibiotic(s). Plate 50 μl and 150 μl of the transformation onto the dishes, and spread.
Incubate the plates at 37℃ for 12-16 hours, making sure the agar side of the plate is up.
After picking colonies, store the plate in 4 ℃.
Show More
September
August
July
June
May
April
March
HSNU | Taipei
