<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_Chalmers_Gothenburg_Project_Promoter_study skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><DIV id="top_title"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:Chalmers Gothenburg/Project/Promoter study</SPAN></H1></DIV><DIV id="HQ_page"><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><P><TITLE>Chalmers Gothenburg iGEM 2016</TITLE></P><DIV class="project"><DIV id="header_navbar_wrapper"><DIV id="header-nav-anchor"><NAV><UL class="header_navbar_ul"><LI id="header_navbar_home"><A href="https://2016.igem.org/Team:Chalmers_Gothenburg" id="home_link">Home</A></LI><LI class="dropdown"><A href="https://2016.igem.org/Team:Chalmers_Gothenburg/Team" id="team_link">Team</A><DIV class="dropdown-content"><A href="https://2016.igem.org/Team:Chalmers_Gothenburg/Team/Students">Students</A><A href="https://2016.igem.org/Team:Chalmers_Gothenburg/Team/Supervisors">Supervisors</A><A href="https://2016.igem.org/Team:Chalmers_Gothenburg/Attributions">Attributions</A><A href="https://2016.igem.org/Team:Chalmers_Gothenburg/Team/Sponsors">Sponsors</A><A href="https://2016.igem.org/Team:Chalmers_Gothenburg/Collaborations">Collaborations</A></DIV></LI><LI class="dropdown"><A href="https://2016.igem.org/Team:Chalmers_Gothenburg/Project" id="project_link">Project</A><DIV class="dropdown-content"><A href="https://2016.igem.org/Team:Chalmers_Gothenburg/Description">Description</A><A href="https://2016.igem.org/Team:Chalmers_Gothenburg/Project/Constructs">Constructs</A><A href="https://2016.igem.org/Team:Chalmers_Gothenburg/Project/Promoter_study">Promoter study</A><A href="https://2016.igem.org/Team:Chalmers_Gothenburg/Model">Model</A></DIV></LI><LI class="dropdown"><A href="https://2016.igem.org/Team:Chalmers_Gothenburg/Human_Practices" id="HP_link">Human Practices</A><DIV class="dropdown-content"><A href="https://2016.igem.org/Team:Chalmers_Gothenburg/HP/Silver">Human Practices</A><A href="https://2016.igem.org/Team:Chalmers_Gothenburg/HP/Gold">Integrated Human Practices</A></DIV></LI><LI class="dropdown"><A href="https://2016.igem.org/Team:Chalmers_Gothenburg/Achievements" id="achievements_link">Achievements</A><DIV class="dropdown-content"><A href="https://2016.igem.org/Team:Chalmers_Gothenburg/Results">Project results</A><A href="https://2016.igem.org/Team:Chalmers_Gothenburg/Parts">Parts</A></DIV></LI><LI class="dropdown"><A href="https://2016.igem.org/Team:Chalmers_Gothenburg/Experiments" id="notebook_link">Notebook</A><DIV class="dropdown-content"><A href="https://2016.igem.org/Team:Chalmers_Gothenburg/Safety">Safety</A><A href="https://2016.igem.org/Team:Chalmers_Gothenburg/Experiments/Lab_journal">Lab journal</A><A href="https://2016.igem.org/Team:Chalmers_Gothenburg/Experiments/Protocols">Protocols</A><A href="https://2016.igem.org/Team:Chalmers_Gothenburg/Experiments/Flowcharts">Flowcharts</A></DIV></LI></UL></NAV></DIV></DIV><DIV class="wrap"><DIV class="header"><DIV class="header-text">PROJECT</DIV><DIV class="header-subtitle">Promoter Study</DIV></DIV><ASIDE id="side-nav"><DIV id="nav-anchor"><NAV><A href="#abstract" class="active">Abstract</A><A href="#introduction">Introduction</A><A href="#method">Method</A><A href="#results">Results</A><A href="#conclusions">Conclusions</A><A href="#references">References</A></NAV></DIV></ASIDE><DIV class="wrap-content"><DIV class="target" id="abstract"><H3>Abstract</H3><P class="text">To make <I>Saccharomyces cerevisiae</I> compatible with the co-culture, the plan was to exchange two promoters involved in glutamine synthesis and excretion. Since one of the targeted genes, <I>GLN1</I>, is at a branching point in the <I>S. cerevisiae</I> amino acid metabolism, we were worried that a too strong promoter could cause auxotrophy for glutamate [1]. Hence, we were looking for intermediate strength promoters. Further, we were interested in how the promoters perform when cells grow on acetate, since the carbon source in our planned co-culture is acetate. While researching different promoters, we realized that information on different promoters’ characteristics on acetate is very sparse. Therefore, we planned and performed a promoter study where we investigated the expression of four promoter candidates compared to the native promoters we wanted to replace. GFP and the different promoters were inserted into a p416 backbone and the resulting plasmids were used to transform <I>S. cerevisiae</I>. The GFP expression was viewed with a fluorescence microscope and quantified spectrophotometrically in a plate reader. The promoter strengths were measured for cells grown on both acetate and glucose. Except for enabling a good choice of promoters to exchange in <I>S. cerevisiae</I>, this kind of information could be very useful to other parts of the iGEM community that needs information on promoters and thus serves a purpose in itself. The results of the promoter study are available here on our wiki and we encourage other iGEM teams as well as anyone else to use it for future projects.</P></DIV><DIV class="target" id="introduction"><H3>Introduction</H3><P class="text">For the promoter candidates a total of four promoters, pTEF1, pPYK2, pPCK1 and pFBP1 were initially chosen for testing. Along with this, the promoters native to the genes we wanted to overexpress, pAQR1 and pGLN1, were investigated. However, problems with the construction of the plasmid with pFBP1 resulted in it being dropped from the study due to time limitations. pTEF1 is generally regarded as a strong constitutive promoter [2], pPYK2 is repressed by glucose and thus derepressed in glucose free media [3], and pPCK1 is induced by acetate [4]. Even though pTEF1 is regarded as constitutive, we were able to find little or no work where its activity on acetate was investigated. More importantly, when data regarding acetate induced promoters was found, it was data regarding how much a promoter was induced compared to itself and comparisons between different promoters were not available.</P><P class="text">The choice to look into both constitutive, inducible and derepressed promoters had a few reasons. Firstly, constitutive promoters are convenient since they provide robust regulation and expression of the genes of interest. On the other hand, since many of the constitutive promoters are quite strong, we were worried that their expression levels would cause depletion of other amino acids. That could happen since the Gln1 protein coded by <I>GLN1</I> gene synthesizes glutamine from glutamate, which is also involved in the synthesis of other amino acids [5]. Inducible promoters might be a remedy for this, since their activities could be lower, especially at low levels of acetate. Further, a production and excretion of glutamine that is proportional to the acetate availability could be an efficient way to regulate the co-culture system, since high acetate availability in the case of this project indicates high growth and metabolism of the cyanobacteria, possibly meaning that they will require more glutamine.</P></DIV><DIV class="target" id="method"><H3>Method</H3><P class="text">The strain used for the promoter study was IMX581. It has Cas9 integrated into the genome and its parent strain is CEN.PK113-5D [6]. Just like its parent strain, it is auxotroph for uracil. It was chosen for the promoter study since the plan was to use it for the co-culture with the cyanobacteria.</P><P class="text">The plasmids used in the study was p416-TEF1-GFP and p416-TEF1 [7]. They are the same plasmid except that the p416-TEF1-GFP has GFP inserted after the TEF1 promoter while the p416-TEF1 does not have anything inserted after the TEF1 promoter, it’s “empty”. Both plasmids have ampicillin resistance and a gene for synthesizing uracil. They were supplied to us by the systems biology institution at Chalmers (SYSBIO). The p416-TEF1-GFP plasmid was used as it was for measuring expression levels of the pTEF1 promoter and saved us a bit of work with constructing the plasmids. For the rest of the study, p416-TEF1 was used.</P><P class="text">The promoters for the study were extracted from the genome of <I>S. cerevisiae</I> IMX581. The genomic extraction was performed according to “quick genomic extraction” protocol. The promoters were amplified with PCR from the genomic extraction. For primer sequences, see the <A href="https://2016.igem.org/Team:Chalmers_Gothenburg/Project/Constructs#sce">constructs page</A>.</P><P class="text">The p416-TEF1 plasmid backbone was cut with fast digest™ XbaI and SacI restriction enzymes from ThermoFisher according to “Assembly of promoter study” protocol When p416-TEF1 is cut with those enzymes, the pTEF1 promoter is removed from the rest of the plasmid backbone, which is then ready for insertion of the new promoter along with GFP. 
The plasmids were assembled using a three fragment Gibson assembly with the plasmid backbone, the GFP gene and the various promoters. The p416-GFP plasmids with the different promoters were transformed to chemically competent <I>E. coli</I> by heat shock. After growth over night colonies were picked from the <I>E. coli</I> plates. They were streaked onto new selection plates and inoculated into liquid LB media with ampicillin. After a second overnight growth, plasmids were extracted from the <I>E. coli</I>. Correct plasmid assembly was verified with restriction of the plasmids followed by gel electrophoresis. The Gibson assembly and the plasmid restriction verification was carried out according to the “Assembly of promoter study” protocol.
IMX581 was transformed with the plasmids using electroporation according to “yeast electroporation” protocol.</P><P class="text">The <I>S. cerevisiae</I> IMX581 with the fluorescence plasmids was taken from plate and cultured overnight in SD-ura media with 2 % (w/v) glucose. When measuring expression levels on acetate, the cultures that had grown overnight in glucose were centrifuged, washed and re-suspended in SD-ura acetate 0.5 % (w/v) and then left to grow for 24 hours. 
After overnight growth, samples were taken from the cultures and viewed with fluorescence microscopy. Excitation/emission was 485/520 nm. Pictures of the samples were taken (see results below).</P><P class="text">Approximately 3 hours before the fluorescence measurements, the OD of the cultures was measured and they were diluted to OD<SUB>600</SUB>=0.3. After growth for the remaining three hours, growth was halted by cooling the tubes on ice, OD was measured and samples were taken from the cultures for measurement of expression levels. Samples were loaded into 96-well plates in triplicates together with blank which was IMX581 transformed with the p416-TEF1 plasmid without GFP. Fluorescence levels was measured with a BMG Labtech FLUOstar Omega plate reader. Excitation/emission wavelength was 485/520 nm.</P></DIV><DIV class="target" id="results"><H3>Results</H3><P class="text">The raw data from the promoter study was corrected against OD<SUB>600</SUB> of that sample, and the mean value of the negative control (cells with p416tef without GFP) was subtracted. The results are shown in Table 1. Note that the fluorescence intensity should not be compared to other studies as it strongly depends of the settings of the machine used to make the measurements. Furthermore, the levels are depending on the cultivation time, which differed between the two conditions, and the results needs to be normalized before comparison. </P><FIGURE><FIGCAPTION class="figure_caption"><B>Table 1.</B> Fluorescent levels of GFP under the control of the promoters pAQR1, pGLN1, pPCK1, PYK2 and pTEF1 for cells cultivated in SD -URA media + 2 % glucose or 0.5 % acetate (n=3). </FIGCAPTION><TABLE><TBODY><TR><TH>pAQR1 (Fluorescent unit/OD<SUB>600</SUB>)</TH><TH>pGLN1 (Fluorescent unit/OD<SUB>600</SUB>)</TH><TH>pPCK1 (Fluorescent unit/OD<SUB>600</SUB>)</TH><TH>pPYK2 (Fluorescent unit/OD<SUB>600</SUB>)</TH><TH>pTEF1 (Fluorescent unit/OD<SUB>600</SUB>)</TH></TR><TR><TH>Acetate</TH><TD>63.4</TD><TD>425.8</TD><TD>1720.8</TD><TD>77.0</TD><TD>1398.7</TD></TR><TR><TH>Glucose</TH><TD>303.0</TD><TD>862.2</TD><TD>234.7</TD><TD>124.9</TD><TD>1314.2</TD></TR></TBODY></TABLE></FIGURE><P class="text">In Figure 1 the results are normalized against the expression level of the pTEF1 promoter. This allows comparison between the two conditions.</P><FIGURE><FIGCAPTION class="figure-caption"><B>Fig 1.</B> Fluorescent levels of GFP under the control of the promoters pAQR1, pGLN1, pPCK1, PYK2 and pTEF1 in glucose and acetate conditions relative the levels of pTEF1. Each sample was loaded into three different wells in the plate reader, and error bars are shown as confidence intervals with p = 0.05, using student's t-test.</FIGCAPTION></FIGURE><P class="text">The most striking feature of the results of the promoter study was the pPCK1 promoter, which showed a much higher expression when cells were grown on acetate than on glucose, which is consistent with previous studies [4]. It could be either induced by acetate or derepressed by the lack of glucose and in this study it was not determined which of the two. Another part of the results is how similar and high the activity of pTEF1 is on glucose and acetate. It further strengthens the reputation of pTEF1 as a true strong constitutive promoter. Still, the expression of the acetate induced pPCK1 is even higher than the pTEF1 expression. Therefore, pPCK1 might be a good choice for overexpression of <I>AQR1</I> in acetate conditions. pGLN1 seems to be a bit stronger on glucose than on acetate. In general, a clear variation in strength between the different promoters can be seen.</P><P class="text">The pictures taken with the fluorescence microscope are in general consistent with the data from the plate reader even though they are just a qualitative measurement of the promoter strength. A few of the pictures can be seen below in Figure 2 and 3. Figure 2 contains the pictures from the cultures on acetate, while Figure 3 is for the glucose cultures.</P><FIGURE><FIGCAPTION class="figure-caption"><B>Fig 2.</B> Pictures taken with fluorescence microscopy of the fluorescent yeast grown on acetate with plasmids with the different promoters. All pictures are an overlay of fluorescence and bright field channels.</FIGCAPTION></FIGURE><FIGURE><FIGCAPTION class="figure-caption"><B>Fig 3.</B> Pictures taken with fluorescence microscopy of the fluorescent yeast grown on glucose with plasmids with the different promoters. All pictures are an overlay of fluorescence and bright field channels. The pictures below are not of the same cultures as the ones that are presented in figure 1. In the picture of pAQR1, the settings of the microscope are slightly different compared to the other pictures which causes a small exaggeration of the apparent fluorescence.</FIGCAPTION></FIGURE></DIV><DIV class="target" id="conclusions"><H3>Conclusions</H3><P class="text">The promoter study was a great opportunity, which allowed a good choice of promoters to exchange in <I>S. cerevisiae</I> to moderately upregulate <I>AQR1</I> and <I>GLN1</I> expressions. For <I>GLN1</I>, the promoter pTEF1 could have been used, while pGLN1 could have been used to replace pAQR1. According to our results, this would have created a moderate overexpression of both the genes of interest. If something would have gone wrong with the promoter exchange, like poor growth of the yeast with new promoters, the results from the promoter study could also have helped with troubleshooting and guidance to what the next step should be.</P><P class="text">The information from the promoter study could also be of great use for other people interested in expression levels of different promoters, especially on acetate, where there is very little data comparing strengths of different promoters. Therefore, the promoter study serves a purpose in itself, separated from the rest of our project. Hopefully, the information gained will be used by future iGEM teams.</P></DIV><DIV class="target" id="references"><H3>References</H3><DIV class="reference_list"><UL><LI>[1]   Mitchell AP, Magasanik B. Biochemical and physiological aspects of glutamine synthetase inactivation in Saccharomyces cerevisiae. Journal of Biological Chemistry. 1984;259(19):12054-62.</LI><LI>[2]   Partow S, Siewers V, Bjørn S, Nielsen J, Maury J. Characterization of different promoters for designing a new expression vector in Saccharomyces cerevisiae. Yeast. 2010 Nov 1;27(11):955-64.</LI><LI>[3]   Boles E, Schulte F, Miosga T, Freidel K, Schlüter E, Zimmermann FK, et al. Characterization of a glucoserepressed pyruvate kinase (Pyk2p) in Saccharomyces cerevisiae that is catalytically insensitive to fructose-1,6-bisphosphate. Journal of Bacteriology. 1997;179(9):2987-93</LI><LI>[4]   Weinhandl K, Winkler M, Glieder A, Camattari A. Carbon source dependent promoters in yeasts. MICROBIAL CELL FACTORIES. 2014;13(1):5.</LI><LI>[5]   Ljungdahl PO, Daignan-Fornier B, Wenner-Grens institut, Stockholms universitet, Naturvetenskapliga fakulteten. Regulation of amino acid, nucleotide, and phosphate metabolism in Saccharomyces cerevisiae. Genetics. 2012;190(3):885-929.</LI><LI>[6]   Mans R, van Rossum HM, Wijsman M, Backx A, Kuijpers NGA, van den Broek M, et al. CRISPR/Cas9: a molecular Swiss army knife for simultaneous introduction of multiple genetic modifications in Saccharomyces cerevisiae. FEMS Yeast Research. 2015;15(2):1-15</LI><LI>[7]   Mumberg D, Müller R, Funk M. Yeast vectors for the controlled expression of heterologous proteins in different genetic backgrounds. Gene. 1995;156(1):119-22</LI></UL></DIV></DIV></DIV><FOOTER id="footer-top"><DIV id="footer-contact-row"><A href="https://www.facebook.com/chalmersigem/?fref=ts">facebook</A><A href="https://twitter.com/igemchalmers">twitter</A><A href="mailto:igemgothenburg@gmail.com">e-mail</A></DIV></FOOTER></DIV></DIV></DIV></DIV></DIV></DIV></DIV></BODY></HTML>