<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_Chalmers_Gothenburg_Project_Constructs skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><DIV id="top_title"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:Chalmers Gothenburg/Project/Constructs</SPAN></H1></DIV><DIV id="HQ_page"><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><P><TITLE>Chalmers Gothenburg iGEM 2016</TITLE></P><DIV class="project"><DIV id="header_navbar_wrapper"><DIV id="header-nav-anchor"><NAV><UL class="header_navbar_ul"><LI id="header_navbar_home"><A href="https://2016.igem.org/Team:Chalmers_Gothenburg" id="home_link">Home</A></LI><LI class="dropdown"><A href="https://2016.igem.org/Team:Chalmers_Gothenburg/Team" id="team_link">Team</A><DIV class="dropdown-content"><A href="https://2016.igem.org/Team:Chalmers_Gothenburg/Team/Students">Students</A><A href="https://2016.igem.org/Team:Chalmers_Gothenburg/Team/Supervisors">Supervisors</A><A href="https://2016.igem.org/Team:Chalmers_Gothenburg/Attributions">Attributions</A><A href="https://2016.igem.org/Team:Chalmers_Gothenburg/Team/Sponsors">Sponsors</A><A href="https://2016.igem.org/Team:Chalmers_Gothenburg/Collaborations">Collaborations</A></DIV></LI><LI class="dropdown"><A href="https://2016.igem.org/Team:Chalmers_Gothenburg/Project" id="project_link">Project</A><DIV class="dropdown-content"><A href="https://2016.igem.org/Team:Chalmers_Gothenburg/Description">Description</A><A href="https://2016.igem.org/Team:Chalmers_Gothenburg/Project/Constructs">Constructs</A><A href="https://2016.igem.org/Team:Chalmers_Gothenburg/Project/Promoter_study">Promoter study</A><A href="https://2016.igem.org/Team:Chalmers_Gothenburg/Model">Model</A></DIV></LI><LI class="dropdown"><A href="https://2016.igem.org/Team:Chalmers_Gothenburg/Human_Practices" id="HP_link">Human Practices</A><DIV class="dropdown-content"><A href="https://2016.igem.org/Team:Chalmers_Gothenburg/HP/Silver">Human Practices</A><A href="https://2016.igem.org/Team:Chalmers_Gothenburg/HP/Gold">Integrated Human Practices</A></DIV></LI><LI class="dropdown"><A href="https://2016.igem.org/Team:Chalmers_Gothenburg/Achievements" id="achievements_link">Achievements</A><DIV class="dropdown-content"><A href="https://2016.igem.org/Team:Chalmers_Gothenburg/Results">Project results</A><A href="https://2016.igem.org/Team:Chalmers_Gothenburg/Parts">Parts</A></DIV></LI><LI class="dropdown"><A href="https://2016.igem.org/Team:Chalmers_Gothenburg/Experiments" id="notebook_link">Notebook</A><DIV class="dropdown-content"><A href="https://2016.igem.org/Team:Chalmers_Gothenburg/Safety">Safety</A><A href="https://2016.igem.org/Team:Chalmers_Gothenburg/Experiments/Lab_journal">Lab journal</A><A href="https://2016.igem.org/Team:Chalmers_Gothenburg/Experiments/Protocols">Protocols</A><A href="https://2016.igem.org/Team:Chalmers_Gothenburg/Experiments/Flowcharts">Flowcharts</A></DIV></LI></UL></NAV></DIV></DIV><DIV class="wrap"><DIV class="header"><DIV class="header-text">PROJECT</DIV><DIV class="header-subtitle">Constructs</DIV></DIV><ASIDE id="side-nav"><DIV id="nav-anchor"><NAV><A href="#synechocystis" class="active">Synechocystis sp. PCC 6803</A><A class="sub-level" href="#koandins">Simultaneous knock-out and insertion</A><A class="sub-level" href="#ko">Knock-out</A><A href="#ecoli"><I>Escherechia coli</I></A><A href="#bs"><I>Bacillus subtilis</I></A><A href="#sce"><I>Saccharomyces cerevisiae</I></A><A href="#yar"><I>Yarrowia lipolytica</I></A><A href="#references">References</A></NAV></DIV></ASIDE><DIV class="wrap-content"><P class="text">All the constructs used in our project are summarized on this page. The DNA fragments of each construct were assembled with <A href="https://static.igem.org/mediawiki/2016/9/9d/T--Chalmers_Gothenburg--Protocols_gibson_assembly.pdf">Gibson assembly</A> after addition of overlapping sequences through PCR with primers containing overhangs to the adjacent DNA fragment (20-25 base pairs). Design of constructs and primers were performed in <A href="http://www.snapgene.com/">Snapgene</A> with the aid of the web tools <A href="http://nebuilder.neb.com/">NEBuilder</A> and <A href="http://tmcalculator.neb.com/">Tm calculator</A>. DNA sequences were obtained using <A href="https://www.ncbi.nlm.nih.gov/genbank/">Genbank,</A><A href="http://www.genome.jp/kegg/">KEGG</A> and <A href="https://www.addgene.org/">Addgene.</A></P><DIV class="target" id="synechocystis"><H2><I>Synechocystis sp. PCC 6803</I></H2><P class="text">The constructs for <I>Synechocystis</I> are based on three different knock-outs: argininosuccinate lyase <A href="http://www.genome.jp/dbget-bin/www_bget?syn:slr1133">(argH)</A>, glutamine synthetase <A href="http://www.genome.jp/dbget-bin/www_bget?syn:slr0288+syn:slr1756">(glnA)</A> and acetyl-CoA synthetase <A href="http://www.genome.jp/dbget-bin/www_bget?syn:sll0542">(acs)</A>. <I>ArgH</I> is knocked out when <I>Synechocystis</I> is going to be used in a consortium with <I>E. coli</I> or <I>B. subtilis</I>, while <I>glnA</I> is knocked out when <I>Y. lipolytica</I> or <I>S. cerevisiae</I> is the producing organism. However, <I>Synechocystis</I> can still synthesise glutamine even if <I>glnA</I> is knocked out due to existence of a different type of glutamine synthetase in the genome (<I>glnN</I>) [1]. Due to limits in available antibiotics and time limits for marker recycling [2], the expression of glnN was repressed by growing the cyanobacteria with NH<SUB>4</SUB><SUP>+</SUP> as the nitrogen source [1].</P><P class="text">In addition to constructing an auxotrophic cyanobacterium, <A href="http://www.genome.jp/dbget-bin/www_bget?syn:sll1299">ackA</A> and <A href="http://www.genome.jp/dbget-bin/www_bget?syn:slr2132">pta</A> are overexpressed in <I>Synechocystis</I>, along with knock-out of <A href="http://www.genome.jp/dbget-bin/www_bget?syn:sll0542">acs</A>, to increase the metabolic flux towards acetate through the pta-AckA pathway [3]. As a starting point, the mutant JA06 received by the Paul Hudson group at <A href="https://www.kth.se/en">KTH Royal Institute of Technology</A> was used as it already excreted some amounts of acetate [4].</P><FIGURE><DIV><B>Figure 1.</B> The plasmid pNF</DIV></FIGURE><P class="text">The homology regions for the knock-out constructs are designed to be around 1000 base pairs each. All <I>Synechocystis</I> constructs are modified versions of the pNF plasmid, shown in Figure 1, which contains the <I>E. coli</I> origin of replication pBR322, an ampicillin resistance marker and a kanamycin resistance marker (KmR). This plasmid was linearized with the primers in Table 1, which only amplified the origin of replication and amp marker, before Gibson assembly.</P><P class="text">One important thing to keep in mind when knocking out genes in <I>Synechocystis</I> is the fact that it has several copies of its genome [5]. To make sure that all of the copies of the specific amino acid synthesising gene is knocked out, transformants should be restreaked several times and checked through colony PCR [6].</P><FIGURE><FIGCAPTION class="table-caption"><B>Table 1.</B> Primers used for linearizing pNF</FIGCAPTION><TABLE><TBODY><TR><TH>Primers</TH><TH>Sequence (5’ → 3’)</TH></TR><TR><TD>pNF FW</TD><TD>GTTCACTGGCCGTCGTTTTAC</TD></TR><TR><TD>pNF RV</TD><TD>CATGCAAGCTTGGCGTAATC</TD></TR></TBODY></TABLE></FIGURE></DIV><DIV class="target" id="koandins"><H3>Constructs for simultaneous knock-out and insertion</H3><H3 class="sidetitle">1. Knock-out <I>argH</I> + insert <I>ackA</I></H3><P class="text">This construct consists of five parts: upstream homology region of <I>argH</I>, KmR cassette, pTrc promoter, <I>ackA</I> and the downstream homology region, Figure 2. This construct knocks out the <I>argH</I> gene when transformed into <I>Synechocystis</I>, making it an arginine auxotroph which is needed to make the cyanobacteria dependent on the producing organism. Furthermore, the endogenous <I>ackA</I> is inserted under the control of the strong consecutive promoter pTrc, for increased acetate production. By combining these two modifications into one construct, the number of constructs were reduced and thus the number of different markers needed.</P><FIGURE><DIV><B>Figure 2.</B> Construct for simultaneous knock-out of ArgH and insertion of AckA</DIV></FIGURE><P class="text">All the primers used for this construct are shown in Table 2. <I>AckA</I> and the flanking sequences of <I>argH</I> are amplified from genomic DNA of <I>Synechocystis</I>. The KmR cassette is amplified from the pNF plasmid and the pTrc promoter is amplified from the pCyJ2 plasmid.</P><FIGURE><FIGCAPTION class="table-caption"><B>Table 2.</B> Primers used for generating the fragments for the <I>argH</I> knock-out and <I>ackA</I> insertion. Uppercase characters hybridize to the template, while the lowercase characters contain the overlapping overhang to the fragment within the parenthesis in the primer name. The numbering of the primers indicates which constructs it can be used for.</FIGCAPTION><TABLE><TBODY><TR><TH>Primers</TH><TH>Sequence (5’ → 3’)</TH></TR><TR><TD>1,5 ArgH Up FW (pNF)</TD><TD>accatgattacgccaagcttgcatgCGCCTGTTATAACACCCC</TD></TR><TR><TD>1,5 ArgH Up RV (KmR)</TD><TD>gctctagagtgatagaattcGGGATTTCGTTGTGATAGTTAG</TD></TR><TR><TD>1 Kmr FW (argH)</TD><TD>ttgctaactatcacaacgaaatcccGAATTCTATCACTCTAGAGCCAGG</TD></TR><TR><TD>1 Kmr RV (pTrc)</TD><TD>attctgcctcgtgatacgcctaggtCGCTACTAGTACAACAAAGCCA</TD></TR><TR><TD>1,2, pTrc FW (KmR)</TD><TD>gctttgttgtactagtagcgACCTAGGCGTATCACGAGG</TD></TR><TR><TD>1,2, pTrc RV (AckA)</TD><TD>ttgagaatcaggaatttcatTCTAGATTTCTCCTCTTTTGTGTG</TD></TR><TR><TD>1 AckA FW (pTrc)</TD><TD>tcacacaaaagaggagaaatctagaATGAAATTCCTGATTCTCAATGC</TD></TR><TR><TD>1 AckA RV (argH)</TD><TD>aacctgggcatatctccacaccaatAGCCTAATTCAACATTTATCTTCAC</TD></TR><TR><TD>1 ArgH Dw FW (AckA) </TD><TD>gataaatgttgaattaggctATTGGTGTGGAGATATGCC</TD></TR><TR><TD>1,5 ArgH Dw RV (pNF)</TD><TD>cgttgtaaaacgacggccagtgaacCACCTTTTAAAAGAATGGCG</TD></TR></TBODY></TABLE></FIGURE><H3 class="sidetitle">2. Knock-out GlnA + insert AckA</H3><P class="text">The purpose of this five-piece construct (Figure 3) is to make <I>Synechocystis</I> glutamine auxotroph, while also overexpressing <I>ackA</I> for increased acetate production.</P><FIGURE><DIV><B>Figure 3.</B>Construct for simultaneous knock-out of<I>glnA</I> and insertion of <I>ackA</I>.</DIV></FIGURE><P class="text">All the primers used for this construct are shown in Table 3. <I>AckA</I> and the flanking sequences of <I>glnA</I> are amplified from genomic DNA of <I>Synechocystis</I>. The KmR cassette is amplified from the pNF plasmid and the pTrc promoter is amplified from the pCyJ2 plasmid.</P><FIGURE><FIGCAPTION class="table-caption"><B>Table 3.</B> Primers used for generating the fragments for the <I>glnA</I> knock-out and <I>ackA</I> insertion. Uppercase characters hybridize to the template, while the lowercase characters contain the overlapping overhang to the fragment within the parenthesis in the primer name. The numbering of the primers indicates which constructs it can be used for.</FIGCAPTION><TABLE><TBODY><TR><TH>Primers</TH><TH>Sequence (5’ → 3’)</TH></TR><TR><TD>2,6 glnA Up FW (pNF)</TD><TD>accatgattacgccaagcttgcatgAAAACGTCATGGCGATC</TD></TR><TR><TD>2,6 glnA Up RV (KmR)</TD><TD>gctctagagtgatagaattcTTTTTCTCCTTAGTGCAGTCAG</TD></TR><TR><TD>2 KmR FW (glnA)</TD><TD>tatctgactgcactaaggagaaaaaGAATTCTATCACTCTAGAGCCAGG</TD></TR><TR><TD>2 KmR RV (pTrc)</TD><TD>attctgcctcgtgatacgcctaggtCGCTACTAGTACAACAAAGCCA</TD></TR><TR><TD>1,2 pTrc FW (KmR)</TD><TD>gctttgttgtactagtagcgACCTAGGCGTATCACGAGG</TD></TR><TR><TD>1,2 pTrc RV (AckA)</TD><TD>ttgagaatcaggaatttcatTCTAGATTTCTCCTCTTTTGTGTG</TD></TR><TR><TD>2 AckA FW (pTrc)</TD><TD>tcacacaaaagaggagaaatctagaATGAAATTCCTGATTCTCAATGC</TD></TR><TR><TD>2 AckA RV (glnA)</TD><TD>ctaaaactgggtgagatggactggtAGCCTAATTCAACATTTATCTTCAC</TD></TR><TR><TD>2 glnA Dw FW (AckA)</TD><TD>gataaatgttgaattaggctACCAGTCCATCTCACCCA</TD></TR><TR><TD>2,6 glnA Dw RV (pNF)</TD><TD>cgttgtaaaacgacggccagtgaacGAGTGGATTTTAAAAACTCTTCGAC</TD></TR></TBODY></TABLE></FIGURE><H3 class="sidetitle">3. Knock-out Acs + insert RFP</H3><P class="text">This construct (Figure 4) knocks out <I>acs</I> to increase the acetate production [3], while also expressing mRFP1 to be able to quantify the levels of <I>Synechocystis</I> compared to the producing organism in the microbial consortia.</P><FIGURE><DIV><B>Figure 4.</B> Construct for simultaneous knock-out of <I>acs</I> and insertion of mRFP1</DIV></FIGURE><P class="text">All the primers used for this construct are shown in Table 4. The flanking sequences of <I>acs</I> are amplified from genomic DNA of <I>Synechocystis</I> and <I>mRFP1</I> is amplified from a plasmid supplied by the last year iGEM team of Chalmers Gothenburg. The SpR cassette and the pTrc promoter are amplified from the pCyJ2 plasmid. The terminator tB0015 is amplified from the pNF plasmid.</P><FIGURE><FIGCAPTION class="table-caption"><B>Table 4.</B> Primers used for generating the fragments for the <I>acs</I> knock-out and mRFP1 insertion. Uppercase characters hybridize to the template, while the lowercase characters contain the overlapping overhang to the fragment within the parenthesis in the primer name. The numbering of the primers indicates which constructs it can be used for.</FIGCAPTION><TABLE><TBODY><TR><TH>Primers</TH><TH>Sequence (5’ → 3’)</TH></TR><TR><TD>3,4,7 acs Up FW (pNF)</TD><TD>accatgattacgccaagcttgcatgCAAATTAGCCAAACCCACG</TD></TR><TR><TD>3,4,7 acs Up RV (SpR)</TD><TD>ccaccaattttctcttcagcTAGCGTGTTGGACAAATTACG</TD></TR><TR><TD>3,4 SpR FW (acs)</TD><TD>ctcccgtaatttgtccaacacgctaGCTGAAGAGAAAATTGGTGG</TD></TR><TR><TD>3,4 SpR RV (pTrc)</TD><TD>attctgcctcgtgatacgcctaggtTAAGAGGTTCCAACTTTCACC</TD></TR><TR><TD>3,4 pTrc FW (SpR)</TD><TD>gtgaaagttggaacctcttaACCTAGGCGTATCACGAGGC</TD></TR><TR><TD>3 pTrc RV (RFP1)</TD><TD>acgtcctcggaggaggCCATTCTAGATTTCTCCTCTTTTGTGTG</TD></TR><TR><TD>3 RFP FW (pTrc)</TD><TD>tcacacaaaagaggagaaatctagaATGGCCTCCTCCGAGG</TD></TR><TR><TD>3 RFP RV (tB0015)</TD><TD>tcgggtgggcctttctgcgtttataTTAGGCGCCGGTGGA</TD></TR><TR><TD>3 B0015 FW (RFP1)</TD><TD>gccactccaccggcgcctaaTATAAACGCAGAAAGGCCC</TD></TR><TR><TD>3 B0015 RV (acs Dw)</TD><TD>aaagactttgacggagaaccCCAGGCATCAAATAAAACG</TD></TR><TR><TD>3 acs Dw FW (tB0015)</TD><TD>gcctttcgttttatttgatgcctggGGTTCTCCGTCAAAGTCTTT</TD></TR><TR><TD>3,4,7 acs Dw RV (pNF)</TD><TD>cgttgtaaaacgacggccagtgaacTTTCCACTTCACTTGGTTTGT</TD></TR></TBODY></TABLE></FIGURE><H3 class="sidetitle">4. Knock-out <I>acs</I> + insert <I>pta</I></H3><P class="text">The purpose of the final knock-out and insertion construct (Figure 5) is to increase the acetate production in <I>Synechocystis</I> by knocking out <I>acs</I> and overexpressing <I>pta</I> [3].</P><FIGURE><DIV><B>Figure 5.</B> Construct for simultaneous knock-out of Acs and insertion of Pta.</DIV></FIGURE><P class="text">All the primers used for this construct are shown in Table 5. The fragments are almost the same as in construct 3, expect the overhangs of the fragments and the <I>pta</I> gene (amplified from genomic DNA of <I>Synechocystis</I>) which replaced the <I>mRFP1</I>.</P><FIGURE><FIGCAPTION class="table-caption"><B>Table 5.</B> Primers used for generating the fragments for the <I>acs</I> knock-out and <I>pta</I> insertion. Uppercase characters hybridize to the template, while the lowercase characters contain the overlapping overhang to the fragment within the parenthesis in the primer name. The numbering of the primers indicates which constructs it can be used for.</FIGCAPTION><TABLE><TBODY><TR><TH>Primers</TH><TH>Sequence (5’ → 3’)</TH></TR><TR><TD>3,4,7 acs Up FW (pNF)</TD><TD>accatgattacgccaagcttgcatgCAAATTAGCCAAACCCACG</TD></TR><TR><TD>3,4,7 acs Up RV (SpR)</TD><TD>ccaccaattttctcttcagcTAGCGTGTTGGACAAATTACG</TD></TR><TR><TD>3,4 SpR FW (acs)</TD><TD>ctcccgtaatttgtccaacacgctaGCTGAAGAGAAAATTGGTGG</TD></TR><TR><TD>3,4 SpR RV (pTrc)</TD><TD>attctgcctcgtgatacgcctaggtTAAGAGGTTCCAACTTTCACC</TD></TR><TR><TD>3,4 pTrc FW (SpR)</TD><TD>gtgaaagttggaacctcttaACCTAGGCGTATCACGAGGC</TD></TR><TR><TD>4 pTrc RV (pta)</TD><TD>aaataaagggaactcgtcatTCTAGATTTCTCCTCTTTTGTGTG</TD></TR><TR><TD>4 pta FW (pTrc)</TD><TD>tcacacaaaagaggagaaatctagaATGACGAGTTCCCTTTATTTAAGC</TD></TR><TR><TD>4 pta RV (acs Dw)</TD><TD>aaagactttgacggagaaccTTTGGAGGGCAAAATCAAG</TD></TR><TR><TD>4 acs Dw FW (pta)</TD><TD>aattaccttgattttgccctccaaaGGTTCTCCGTCAAAGTCTTT</TD></TR><TR><TD>3,4,7 acs Dw RV (pNF)</TD><TD>cgttgtaaaacgacggccagtgaacTTTCCACTTCACTTGGTTTGT</TD></TR></TBODY></TABLE></FIGURE></DIV><DIV class="target" id="ko"><H3>Constructs for knock-out only</H3><P class="text">The following constructs are simplified variants of the previous ones by limiting the modification to a knock-out, without inserting additional genes except the marker.</P><H3 class="sidetitle">5. Knock-out <I>argh</I></H3><P class="text">In this construct (Figure 6), <I>argH</I> is knocked out in <I>Synechocystis</I> by selection with kanamycin.</P><FIGURE><DIV><B>Figure 6.</B> Construct for knock-out of <I>argH</I>.</DIV></FIGURE><P class="text">All the primers used for this construct are shown in Table 6. The only difference in the fragments of this construct compared to construct 1 is the overlapping overhangs.</P><FIGURE><FIGCAPTION class="table-caption"><B>Table 6.</B> Primers used for generating the fragments for the <I>argH</I> knock-out. Uppercase characters hybridize to the template, while the lowercase characters contain the overlapping overhang to the fragment within the parenthesis in the primer name. The numbering of the primers indicates which constructs it can be used for.</FIGCAPTION><TABLE><TBODY><TR><TH>Primers</TH><TH>Sequence (5’ → 3’)</TH></TR><TR><TD>1,5 ArgH Up FW (pNF)</TD><TD>accatgattacgccaagcttgcatgCGCCTGTTATAACACCCC</TD></TR><TR><TD>1,5 ArgH Up RV (KmR)</TD><TD>gctctagagtgatagaattcGGGATTTCGTTGTGATAGTTAG</TD></TR><TR><TD>5,6 KmR FW</TD><TD>GAATTCTATCACTCTAGAGCCAGG</TD></TR><TR><TD>5,6 KmR RV</TD><TD>CGCTACTAGTACAACAAAGCCA</TD></TR><TR><TD>5 ArgH Dw FW (KmR)</TD><TD>acgtggctttgttgtactagtagcgATTGGTGTGGAGATATGC</TD></TR><TR><TD>1,5 ArgH Dw RV (pNF)</TD><TD>cgttgtaaaacgacggccagtgaacCACCTTTTAAAAGAATGGCG</TD></TR></TBODY></TABLE></FIGURE><H3 class="sidetitle">6. Knock-out <I>glnA</I></H3><P class="text">In this construct (Figure 7), <I>argH</I> is knocked out in <I>Synechocystis</I> by selection with kanamycin.</P><FIGURE><DIV><B>Figure 7.</B> Construct for knock-out of GlnA.</DIV></FIGURE><P class="text">All the primers used for this construct are shown in Table 7. The only difference in the fragments of this construct compared to construct 2 is the overlapping overhangs.</P><FIGURE><FIGCAPTION class="table-caption"><B>Table 7.</B> Primers used for generating the fragments for the <I>glnA</I> knock-out. Uppercase characters hybridize to the template, while the lowercase characters contain the overlapping overhang to the fragment within the parenthesis in the primer name. The numbering of the primers indicates which constructs it can be used for.</FIGCAPTION><TABLE><TBODY><TR><TH>Primers</TH><TH>Sequence (5’ → 3’)</TH></TR><TR><TD>2,6 glnA Up FW (pNF)</TD><TD>accatgattacgccaagcttgcatgAAAACGTCATGGCGATC</TD></TR><TR><TD>2,6 glnA Up RV (KmR)</TD><TD>gctctagagtgatagaattcTTTTTCTCCTTAGTGCAGTCAG</TD></TR><TR><TD>5,6 KmR FW</TD><TD>GAATTCTATCACTCTAGAGCCAGG</TD></TR><TR><TD>5,6 KmR RV</TD><TD>CGCTACTAGTACAACAAAGCCA</TD></TR><TR><TD>6 glnA Dw FW (KmR)</TD><TD>gctttgttgtactagtagcgACCAGTCCATCTCACCCA</TD></TR><TR><TD>2,6 glnA Dw RV (pNF)</TD><TD>cgttgtaaaacgacggccagtgaacGAGTGGATTTTAAAAACTCTTCGAC</TD></TR></TBODY></TABLE></FIGURE><H3 class="sidetitle">7. Knock-out of <I>acs</I></H3><P class="text">In this construct (Figure 8), <I>acs</I> is knocked out in <I>Synechocystis</I> by selection with spectinomycin.</P><FIGURE><DIV><B>Figure 8.</B> Construct for knock-out of <I>acs</I></DIV></FIGURE><P class="text">All the primers used for this construct are shown in Table 8. The only difference in the fragments of this construct compared to construct 3 and 4 is the overlapping overhangs.</P><FIGURE><FIGCAPTION class="table-caption"><B>Table 8.</B> Primers used for generating the fragments for the <I>acs</I> knock-out. Uppercase characters hybridize to the template, while the lowercase characters contain the overlapping overhang to the fragment within the parenthesis in the primer name. The numbering of the primers indicates which constructs it can be used for.</FIGCAPTION><TABLE><TBODY><TR><TH>Primers</TH><TH>Sequence (5’ → 3’)</TH></TR><TR><TD>3,4,7 acs Up FW (pNF)</TD><TD>accatgattacgccaagcttgcatgCAAATTAGCCAAACCCACG</TD></TR><TR><TD>3,4,7 acs Up RV (SpR)</TD><TD>ccaccaattttctcttcagcTAGCGTGTTGGACAAATTACG</TD></TR><TR><TD>7 SpR FW</TD><TD>GCTGAAGAGAAAATTGGTGG</TD></TR><TR><TD>7 SpR RV</TD><TD>TAAGAGGTTCCAACTTTCACC</TD></TR><TR><TD>7 acs Dw F (SpR)</TD><TD>gtgaaagttggaacctcttaGGTTCTCCGTCAAAGTCTTT</TD></TR><TR><TD>3,4,7 acs Dw RV (pNF)</TD><TD>cgttgtaaaacgacggccagtgaacTTTCCACTTCACTTGGTTTGT</TD></TR></TBODY></TABLE></FIGURE></DIV><DIV class="target" id="ecoli"><H2><I>Escherichia coli</I></H2><P class="text">The constructs for <I>E. coli</I> were designed to make it overproduce arginine by removing the product inhibition by arginine on <I>argA</I> and knocking out the repressor <I>argR</I>. This overproduction is assumed to make the cell secrete arginine without needing to add any additional transporters.</P><H2 class="sidetitle">1. Overexpression of point mutated <I>argA</I> + <I>GFP</I> insertion</H2><P class="text">To remove the feed-back inhibition of arginine, Tyrosine-19 in <I>argA</I> was mutated to Cysteine [7] through mutagenic PCR using a megaprimer method [8]. After generation of mutated <I>argA</I> fragment, the gene was assembled with Gibson assembly with the reporter gene <I>GFP</I> (for quantification of the ratio of E. coli to cyanobacteria in the co-culture), chloramphenicol resistance gene and upstream homology regions of <I>argA</I> to exchange the native promoter with the strong constitutive promoter J23119. The downstream homology region is included in the mutated <I>argA</I> fragments. The final construct is shown in Figure 9.</P><FIGURE><DIV><B>Figure 9.</B> Construct for overexpressing mutated <I>argA</I> and inserting GFP into the genome of <I>E. coli</I>.</DIV></FIGURE><P class="text">The primers used to assemble the fragments are shown in Table 9. The chloramphenicol resistance gene is amplified from the plasmid pKD3. The <I>GFP</I> gene is amplified from the <A href="http://parts.igem.org/Part:BBa_K584001">BioBrick BBa K584001</A>. <I>ArgA</I> is amplified from genomic DNA of <I>E. coli</I>. The new promoter to <I>argA</I>, along with a <A href="http://parts.igem.org/Part:BBa_B0032:Design">RBS</A>, was included as an overhang in the forward primer. The linear fragment produced was then inserted on the genome of E. coli via a <A href="https://static.igem.org/mediawiki/2016/b/b5/T--Chalmers_Gothenburg--E_coli_pRed_ET_mediated_point_mutation.pdf">pRED/ET</A> mediated method [9].</P><FIGURE><FIGCAPTION class="table-caption"><B>Table 9.</B> Primers used for generating the fragments for the overexpression of the point mutated <I>argA</I> and GFP insertion. Uppercase characters hybridize to the template, while the lowercase characters contain the overlapping overhang to the fragment within the parenthesis in the primer name.</FIGCAPTION><TABLE><TBODY><TR><TH>Primers</TH><TH>Sequence (5’ → 3’)</TH></TR><TR><TD>cat FW (IS)</TD><TD>caccctgcgaaaaaacagaataaaaatacactaatttcgaataatcatgc
aaagaggtgtgccGTGTAGGCTGGAGCTGCTTC</TD></TR><TR><TD>cat RV</TD><TD>ATGGGAATTAGCCATGGTCC</TD></TR><TR><TD>GFP FW (cat)</TD><TD>ggaccatggctaattcccatTTGACAGCTAGCTCAGTCCTAG</TD></TR><TR><TD>GFP RV (ArgA_mut)</TD><TD>aggactgagctagctgtcaaGTATATAAACGCAGAAAGGCCC</TD></TR><TR><TD>argA_mut FW</TD><TD>CGGTTCCCTgTATCAATACC</TD></TR><TR><TD>ArgA_mut RV</TD><TD>CTTGTGATACAGCGGTTCGT</TD></TR><TR><TD>ArgA_mut_amp FW</TD><TD>CACCCTGCGAAAAAACAG</TD></TR></TBODY></TABLE></FIGURE><H3 class="sidetitle">2. <I>ArgR</I> knock-out</H3><P class="text">The second construct is meant to increase the arginine production by knocking out the repressor <I>argR</I> which is induced by the presence of arginine [10]. The <I>argR</I>::Kan knock-out provided by Anne Farewell was transduced to a recipient strain via P1 transduction [11]. The P1 phage was also provided by Anne Farewell.</P></DIV><DIV class="target" id="bs"><H2><I>Bacillus subtilis</I></H2><P class="text">The purpose of the constructs in <I>B. subtilis</I> is to enable it to grow on acetate and overproduce arginine. Both constructs are based on the vectors from the bacillus box made by the <A href="https://2012.igem.org/Team:LMU-Munich">2012 LMU-Munich iGEM-team</A> [12]. </P><H3 class="sidetitle">1. Glyoxylate shunt</H3><P class="text"><I>B. subtilis</I> cannot utilize acetate as a carbon source, which can be solved by inserting the glyoxylate shunt from <I>Bacillus licheniformis</I> into <I>B. subtilis</I>, see Figure 10. The shunt consists of two enzymes in an operon, <I>aceA</I> and <I>aceB</I>, under the control of a native <I>B. licheniformis</I> promoter, which have been shown to be active in <I>B. subtilis</I> as well [13].</P><FIGURE><DIV><B>Figure 10.</B> Construct for the glyoxylate shunt</DIV></FIGURE><P class="text">The glyoxylate shunt was amplified from extracted genomic DNA from with the primers in Table 10. Both the product and the vector pBs4S were cut with XbaI and SpeI before ligation and transformation into <I>E. coli</I>. This was not optimal as there is a risk of vector-only clones due to overlaps of XbaI and SpeI sticky ends, but had to be done anyway because the shunt contained both EcoRI and PstI sites. They could have been removed through mutagenic PCR, but this was not done due to time limitations. </P><FIGURE><FIGCAPTION class="table-caption"><B>Table 10.</B> Primers used for generating the fragments glyoxylate shunt. Uppercase characters hybridize to the template, while the lowercase characters contain the BioBrick prefix or suffix.</FIGCAPTION><TABLE><TBODY><TR><TH>Primers</TH><TH>Sequence (5’ → 3’)</TH></TR><TR><TD>glyox op FW (prefix)</TD><TD>gaattcgcggccgcttctagaGAAAAATATGAACAAGCTATGAATAAAAAG</TD></TR><TR><TD>glyox op RV (suffix)</TD><TD>ctgcagcggccgctactagtaAACAAACAACAGGAATCATCAGAC</TD></TR></TBODY></TABLE></FIGURE><H3 class="sidetitle">2. <I>ahrC</I> knock-out</H3><P class="text">The second modification to <I>B. subtilis</I> was to make it overproduce arginine needed by the cyanobacteria in the co-culture. This was done by deleting the gene <I>ahrC</I>, which codes for a repressor of arginine biosynthesis [14]. Furthermore, the reporter gene <I>GFP</I> was inserted to be able to quantify the abundance of <I>B. subtilis</I> relative the cyanobacteria in the co-culture. The final construct can be seen in Figure 11.</P><FIGURE><DIV><B>Figure 11.</B> Construct for inserting GFP and knocking out the arginine repressor ahrC in <I>B. subtilis</I>.</DIV></FIGURE><P class="text">The fragments were amplified with the primers in Table 11. The homology regions up- and downstream of <I>ahrC</I> was amplified from genomic DNA extracted from <I>B. subtilis</I>. The reporter gene was assembled by ligation of the BioBricks <A href="http://partsregistry.org/Part:BBa_E0840">BBa_E0840</A> and <A href="http://parts.igem.org/Part:BBa_K823003">BBa_K823003</A>, which was then amplified with primers containing overhangs for Gibson assembly. The chloramphenicol and ampicillin resistance gene were amplified from the pBs1C plasmid. </P><FIGURE><FIGCAPTION class="table-caption"><B>Table 11.</B> Primers used for generating the fragments for knock-out of ahrC and the overexpression of GFP. Uppercase characters hybridize to the template, while the lowercase characters contain the overlapping overhang to the fragment within the parenthesis in the primer name.</FIGCAPTION><TABLE><TBODY><TR><TH>Primers</TH><TH>Sequence (5’ → 3’)</TH></TR><TR><TD>Dw ahrC FW (amp+Ori)</TD><TD>aaccattattatcCCGATTCGATTTCTTCGAG</TD></TR><TR><TD>Dw ahrC RV</TD><TD>CAGAAAATCTAACAAAGATAAGAGGTG</TD></TR><TR><TD>cat FW (Dw ahrC)</TD><TD>cctcttatctttgttagattttctgTGTCAATTCTCATGTTTGACAGC</TD></TR><TR><TD>cat RV (pVeg+GFP)</TD><TD>tcgggtgggcctttctgcgtttataCTCCTGCATTAGGAAGCAGC</TD></TR><TR><TD>pVeg+GFP FW</TD><TD>TATAAACGCAGAAAGGCCC</TD></TR><TR><TD>pVeg+GFP RV (Up ahrC)</TD><TD>ttggaaatagaggtgcttacGGAGTTCTGAGAATTGGTATGC</TD></TR><TR><TD>Up ahrC FW (pVeg+GFP)</TD><TD>aactccGTAAGCACCTCTATTTCCAAGC</TD></TR><TR><TD>Up ahrC RV (ori+ampR)</TD><TD>gagctggatacttCTCTGTCAAAGATAAAATTATGATTG</TD></TR><TR><TD>amp+Ori FW (Up ahrC)</TD><TD>atctttgacagagAAGTATCCAGCTCGAGGTCG</TD></TR><TR><TD>amp+Ori RV (Dw ahrC)</TD><TD>gaaatcgaatcggGATAATAATGGTTTCTTAGACGTCAGG</TD></TR></TBODY></TABLE></FIGURE></DIV><DIV class="target" id="sce"><H2><I>Saccharomyces cerevisiae</I></H2><P class="text">The constructs for <I>S. cerevisiae</I> are divided into two subprojects: the promoters study and the promoter exchange. The results of the promoter study are meant to guide the choice of promoters in the promoter exchange for increased production of glutamine in <I>S. cerevisiae</I>, which needed by the cyanobacteria in the co-culture.</P><H3 class="sidetitle">1. Promoter study</H3><P class="text">The constructs for the promoter study are based on the p416tef plasmid, Figure 12 [15]. The promoters used in this study are shown in Table 12 and were cloned into this plasmid together with <I>GFP</I> as the reporter gene. This was done using Gibson assembly with 20 bp overlaps after cutting the vector with SacI and XbaI.</P><P class="text">By using a replicative plasmid instead of chromosomal integration, a higher copy number can be achieved which will make sure that even weak promoters give a detectable signal, which will be compared against the signal from the TEF1 promoter.</P><FIGURE><DIV><B>Figure 12.</B> The plasmid p416tef</DIV></FIGURE><P class="text">All promoters were amplified from the genome of <I>S. cerevisiae</I> CEN.PK113-5D and <I>GFP</I> was amplified from the plasmid p416tef gfp. The primers used to amplify all parts are shown in Table 12.</P><FIGURE><FIGCAPTION class="table-caption"><B>Table 12.</B> Primers used for generating the fragments for the promoter study. Uppercase characters hybridize to the template, while the lowercase characters contain the overlapping overhang to the fragment within the parenthesis in the primer name.</FIGCAPTION><TABLE><TBODY><TR><TH>Primers</TH><TH>Sequence (5’ → 3’)</TH></TR><TR><TD>GFP FW (p416 TEF1)</TD><TD>gcccgggggatccactagttCTATTTGTATAGTTCATCCATGCCATG</TD></TR><TR><TD>GFP R</TD><TD>ATGCGAATCCCCGGGTTA</TD></TR><TR><TD>pAQR1 FW (p416)</TD><TD>attaacccggggattcgcatTGCTGATTCGACTTTCTGAA</TD></TR><TR><TD>pAQR1 RV (p416)</TD><TD>gggaacaaaagctggagctcGTTCTGTTGCCGTATGCTATC</TD></TR><TR><TD>pFBP1 FW (p416)</TD><TD>attaacccggggattcgcatATGTGTGGTAGTATGAGGGATG</TD></TR><TR><TD>pFBP1 RV (p416)</TD><TD>gggaacaaaagctggagctcATATAAAATGAAAATAATATCCAAAGAAAAA</TD></TR><TR><TD>pGLN1 FW (p416)</TD><TD>attaacccggggattcgcatTTTTGATTATATTATATTATATTATGTTTAATTTTTGTT</TD></TR><TR><TD>pGLN1 RV (p416)</TD><TD>gggaacaaaagctggagctcGTATAAATAGTTATATAGAGATGAACTCTAAGCTAGTG</TD></TR><TR><TD>pPCK1 FW (p416)</TD><TD>attaacccggggattcgcatGTTGTTATTTTATTATGGAATAATTAGTTGC</TD></TR><TR><TD>pPCK1 RV (p416)</TD><TD>gggaacaaaagctggagctcTCGTTCGTTGTACGTACATTTAC</TD></TR><TR><TD>pPYK2 FW (p416)</TD><TD>attaacccggggattcgcatCGATAGTGCTTTTGTTGTAATCTT</TD></TR><TR><TD>pPYK2 RV (p416)</TD><TD>gggaacaaaagctggagctcCGCTTTTATGAACATATTCCG</TD></TR></TBODY></TABLE></FIGURE><H3 class="sidetitle">2. Promoter exchange</H3><P class="text">The purpose of the promoter exchange is to fine tune the expression of Glutamine synthetase (GLN1) and Acids Quinidine Resistance (AQR1), which is an amino acid transporter [16], with the aim to increase the production and excretion of glutamine.</P><P class="text">This is done in the strain <I>S.cerevisiae</I> IMX581, which has CRISPR/Cas9 integrated in its genome [17]. The plasmid pMEL13 from the same paper is linearized with PCR with primers in Table 13 prior Gibson assembly with a gBlock (ordered from IDT) containing the gRNA targeting the promoter region of <I>GLN1</I> or <I>AQR1</I>, along with flanking regions of 50 basepair overlapping to pMEL13. The gRNA was designed with an online tool at <A href="https://benchling.com/">Benchling.</A></P><FIGURE><FIGCAPTION class="table-caption"><B>Table 13.</B> Primers used to linearize pMEL13.</FIGCAPTION><TABLE><TBODY><TR><TH>Primers</TH><TH>Sequence (5’ → 3’)</TH></TR><TR><TD>pMEL13 F</TD><TD>GTTTTAGAGCTAGAAATAGCAAGTTAAA</TD></TR><TR><TD>pMEL13 R</TD><TD>GATCATTTATCTTTCACTGCGG</TD></TR></TBODY></TABLE></FIGURE><P class="text">To exchange the promoter, the assembled pMEL13 was co-transformed with a linear DNA fragment containing the new promoter and homology regions to knock out either the promoter of <I>GLN1</I> or <I>AQR1</I>, which was generated through PCR with the primers in Table 14. As overexpression of GLN1 might lead to glutamate auxotrophy [18], the results of the promoter study will help guide the choice of promoter for GLN1</P><FIGURE><FIGCAPTION class="table-caption"><B>Table 14.</B> Primers used for generating the fragments for the promoter exchange. Uppercase characters hybridize to the template, while the lowercase characters contain the overlapping overhang to the fragment within the parenthesis in the primer name.</FIGCAPTION><TABLE><TBODY><TR><TH>Primers</TH><TH>Sequence (5’ → 3’)</TH></TR><TR><TD>pFBP1 FW (pAQR1 up)</TD><TD>ggcctgcattgttttcttcagacgagaagccgttccaacgtttctttttctcg
tcaccggATATAAAATGAAAATAATATCCAAAGAAA</TD></TR><TR><TD>pFBP1 RV (pAQR1 dw)</TD><TD>ctcattgtgagtaggatacatctcaatatcttctgtgtatatactgttacttc
gtgacatATGTGTGGTAGTATGAGGGATG</TD></TR><TR><TD>pPCK1 FW (pGLN1 up)</TD><TD>agcggtcaggtgtaagtagtaggcttgataatgaattaaagatgactccgacg
catattgTCGTTCGTTGTACGTACATTTAC</TD></TR><TR><TD>pPCK1 RV (pGLN1 dw)</TD><TD>ttggtccagttctagatatttttgtaaaatttgagtcttttcgatgcttgctt
cagccatGTTGTTATTTTATTATGGAATAATTAGTTG</TD></TR><TR><TD>pPYK1 FW (pGLN1 up)</TD><TD>agcggtcaggtgtaagtagtaggcttgataatgaattaaagatgactccgacg
catattgCGCTTTTATGAACATATTCCG</TD></TR><TR><TD>pPYK1 RV (pGLN1 dw)</TD><TD>ttggtccagttctagatatttttgtaaaatttgagtcttttcgatgcttgctt
cagccatCGATAGTGCTTTTGTTGTAATCTT</TD></TR><TR><TD>pTEF1 FW (pAQR1 up)</TD><TD>gcctgcattgttttcttcagacgagaagccgttccaacgtttctttttctcgt
caccggACAATGCATACTTTGTACGTTCA</TD></TR><TR><TD>pTEF1 RV (pAQR1 dw)</TD><TD>ctcattgtgagtaggatacatctcaatatcttctgtgtatatactgttacttc
gtgacatTTTGTAATTAAAACTTAGATTAGATTGCT</TD></TR></TBODY></TABLE></FIGURE></DIV><DIV class="target" id="yar"><H2><I>Yarrowia lipolytica</I></H2><P class="text">The constructs of <I>Y. lipolytica</I> are based on the two plasmids JMP1047 and JMP2563 [19], Figure 13.  Before Gibson assembly, both plasmids were cut with BamHI and AvrII. The overlaps on the fragments for the Gibson assembly were designed to be 25 base pairs and were generated through PCR. On the first plasmid <I>GFP</I> and the native Glutamine synthetase (GLN1) are expressed and on the second the amino acid transporter <I>AQR1</I> [16] from <I>S. cerevisiae</I>.</P><FIGURE><DIV><B>Figure 13.</B> The plasmids JMP1047 and JMP2563 used for the <I>Y. lipolytica</I> constructs</DIV></FIGURE><H3 class="sidetitle">1. Overexpression of GLN1 and <I>GFP</I></H3><P class="text">A similar strategy was used for <I>Yarrowia</I> as in <I>S. cerevisiae</I> to increase the glutamine production. The native GLN1 was overexpressed under the control of the native pTEF1 promoter, and <I>GFP</I> was expressed under the same promoter to allow quantification of <I>Y. lipolytica</I> in the co-culture. For the full construct, see Figure 14. This construct was integrated into the genome of <I>Y. lipolytica</I> JMY2101 by random ectopic integration through the NHEJ recombination pathway [20]. </P><FIGURE><DIV><B>Figure 14.</B> Construct for overexpression of GLN1 and GFP in <I>Y. lipolytica</I>.</DIV></FIGURE><P class="text">The fragments for the constructs were generated through PCR with the primers in Table 15. The gene <I>GLN1</I> and the terminators tXPR2 and tLIP2 were amplified from genomic DNA of <I>Y. lipolytica</I> JMY2900. The <I>GFP</I> gene was amplified from the <I>S. cerevisiae</I> plasmid p416tef gfp. The promoter driving the expression of <I>GFP</I> was amplified from the JMP1047 plasmid. </P><FIGURE><FIGCAPTION class="table-caption"><B>Table 15.</B> Primers used for generating the fragments for the overexpression of GLN1 and GFP. Uppercase characters hybridize to the template, while the lowercase characters contain the overlapping overhang to the fragment within the parenthesis in the primer name</FIGCAPTION><TABLE><TBODY><TR><TH>Primers</TH><TH>Sequence (5’ → 3’)</TH></TR><TR><TD>Gln1 F (JMP1047)</TD><TD>tctccttgtcaactcacacccgaagATGAACTGGGAATCCGAAC</TD></TR><TR><TD>Gln1 R (tXPR2)</TD><TD>aattgcTTAGACCGTCTCAATGTACCG</TD></TR><TR><TD>tXPR2 F (Glu1)</TD><TD>ggtacattgagacggtctaaGCAATTAACAGATAGTTTGCCG</TD></TR><TR><TD>tXPR2 R (tLip2)</TD><TD>tgaagttcatctgcgttgaaAGATCTGAGCGTGAATTATACGG</TD></TR><TR><TD>tLIP2 F (tXPR2)</TD><TD>agatctTTCAACGCAGATGAACTTCA</TD></TR><TR><TD>tLip2 R (GFP)</TD><TD>tggatgaactatacaaatagTATTTATCACTCTTTACAACTTCTACCTCAAC</TD></TR><TR><TD>GFP F (tLIP2)</TD><TD>taaataCTATTTGTATAGTTCATCCATGCCATG</TD></TR><TR><TD>GFP R (pTef)</TD><TD>ctcacacccgaagATGCGAATCCCCGGGTTA</TD></TR><TR><TD>pTef F (GFP)</TD><TD>cggggattcgcatCTTCGGGTGTGAGTTGACAAG</TD></TR><TR><TD>pTef R (JMP1047)</TD><TD>aaatagcttagataccacagacaccCGATAGAGACCGGGTTGG</TD></TR></TBODY></TABLE></FIGURE><H3 class="sidetitle">2. Overexpression of <I>AQR1</I></H3><P class="text">The gene coding for the amino acid transporter <I>AQR1</I> from <I>S. cerevisiae</I> CEN.PK113-5D was added to <I>Y. lipolytica</I> as we could not find a homolog in the genome of <I>Y. lipolytica</I>. It was inserted to the genome of the strain JMY330, through the same mechanisms as the previous construct, after Gibson assembly with the vector JMP2563.</P><FIGURE><DIV><B>Figure 15.</B> Construct for overexpression of <I>AQR1</I> in <I>Y. lipolytica</I>.</DIV></FIGURE><P class="text">The fragments for this construct was generated through PCR with the primers in Table 16. The gene <I>AQR1</I> was amplified from genomic DNA of <I>S. cerevisiae</I> CEN.PK113-5D and the terminator tXPR2 was amplified from genomic DNA of <I>Y. lipolytica</I> JMY2900.</P><FIGURE><FIGCAPTION class="table-caption"><B>Table 16.</B> Primers used for generating the fragments for the overexpression of <I>AQR1</I>. Uppercase characters hybridize to the template, while the lowercase characters contain the overlapping overhang to the fragment within the parenthesis in the primer name.</FIGCAPTION><TABLE><TBODY><TR><TH>Primers</TH><TH>Sequence (5’ → 3’)</TH></TR><TR><TD>AQR1 F (JMP2563)</TD><TD>caactcacacccgaaggatcATGTCACGAAGTAACAGTATATACACAG</TD></TR><TR><TD>AQR1 R (tXPR2)</TD><TD>gcaaactatctgttaattgcTTAATTATGATTATCGTTCTGGTCTC</TD></TR><TR><TD>tXPR2 F (AQR1)</TD><TD>agaacgataatcataattaaGCAATTAACAGATAGTTTGCCG</TD></TR><TR><TD>tXPR2 R (JMP2563)</TD><TD>agataccacagacaccctagAGATCTGAGCGTGAATTATACGG</TD></TR></TBODY></TABLE></FIGURE></DIV><DIV class="target" id="references"><H2>References</H2><DIV class="reference_list"><UL><LI>[1]   Reyes JC, Muro-Pastor MI, Florencio FJ. Transcription of glutamine synthetase genes (<I>glnA</I> and <I>glnN</I>) from the cyanobacterium <I>Synechocystis</I> sp. strain PCC 6803 is differently regulated in response to nitrogen availability. Journal of Bacteriology. 1997;179(8):2678-89</LI><LI>[2]   Cheah YE, Albers SC, Peebles CAM. A novel counter‐selection method for markerless genetic modification in <I>Synechocystis</I> sp. PCC 6803. Biotechnology Progress. 2013;29(1):23-30.</LI><LI>[3]   Valgepea K, Adamberg K, Nahku R, Lahtvee P, Arike L, Vilu R. Systems biology approach reveals that overflow metabolism of acetate in Escherichia coli is triggered by carbon catabolite repression of acetyl-CoA synthetase. BMC Systems Biology. 2010;4:166</LI><LI>[4]   Anfelt J, Kaczmarzyk D, Shabestary K, Renberg B, Rockberg J, Nielsen J, Uhlén M, Hudson EP. Genetic and nutrient modulation of acetyl-CoA levels in <I>Synechocystis</I> for n-butanol production. Microbial cell factories. 2015;14(1):1.</LI><LI>[5]   Griese M, Lange C, Soppa J. Ploidy in cyanobacteria. FEMS Microbiology Letters. 2011;323(2):124-31.</LI><LI>[6]   Proels R. Stable Transformation of Cyanobacterium <I>Synechocystis</I> sp. BIO-PROTOCOL. 2014;4(21).</LI><LI>[7]   Rajagopal BS, DePonte J, Tuchman M, Malamy MH. Use of Inducible Feedback-Resistan tN-Acetylglutamate Synthetase (<I>argA</I>) Genes for Enhanced Arginine Biosynthesis by Genetically Engineered Escherichia coli K-12 Strains. Applied and environmental microbiology. 1998;64(5):1805-11.</LI><LI>[8]   Lai R, Bekessy A, Chen CC, Walsh T, Barnard R. Megaprimer mutagenesis using very long primers. BioTechniques. 2003;34(1):52-6.</LI><LI>[9]   Heermann R, Zeppenfeld T, Jung K. Simple generation of site-directed point mutations in the <I>Escherichia coli</I> chromosome using Red®/ET® Recombination. Microbial cell factories. 2008;7(1):1.</LI><LI>[10]   Lu C. Pathways and regulation of bacterial arginine metabolism and perspectives for obtaining arginine overproducing strains. Applied Microbiology and Biotechnology. 2006;70(3):261-72</LI><LI>[11]   Thomason LC, Costantino N, Court DL. <I>E. coli</I> genome manipulation by P1 transduction. Current protocols in molecular biology. 2007:1-7.</LI><LI>[12]   Radeck J, Kraft K, Bartels J, Cikovic T, Durr F, Emenegger J, et al. The <I>Bacillus subtilis</I> BioBrick Box: generation and evaluation of essential genetic building blocks for standardized work with Bacillus subtilis. JOURNAL OF BIOLOGICAL ENGINEERING. 2013;7(1):29</LI><LI>[13]   Kabisch J, Pratzka I, Meyer H, Albrecht D, Lalk M, Ehrenreich A, et al. Metabolic engineering of <I>Bacillus subtilis</I> for growth on overflow metabolites. Microbial Cell Factories. 2013;12(1):72</LI><LI>[14]   Larsen R, Buist G, Kuipers OP, Kok J. ArgR and AhrC Are Both Required for Regulation of Arginine Metabolism in Lactococcus lactis. Journal of Bacteriology. 2004;186(4):1147-57.</LI><LI>[15]   Mumberg D, Müller R, Funk M. Yeast vectors for the controlled expression of heterologous proteins in different genetic backgrounds. Gene. 1995;156(1):119-22.</LI><LI>[16]   Tenreiro S, Nunes PA, Viegas CA, Neves MS, Teixeira MC, Cabral MG, et al. AQR1 Gene (ORF YNL065w) Encodes a Plasma Membrane Transporter of the Major Facilitator Superfamily That Confers Resistance to Short-Chain Monocarboxylic Acids and Quinidine in <I>Saccharomyces cerevisiae</I>. Biochemical and Biophysical Research Communications. 2002;292(3):741-8.</LI><LI>[17]   Mans R, van Rossum HM, Wijsman M, Backx A, Kuijpers NGA, van den Broek M, et al. CRISPR/Cas9: a molecular Swiss army knife for simultaneous introduction of multiple genetic modifications in <I>Saccharomyces cerevisiae</I>. FEMS Yeast Research. 2015;15(2):1-15.</LI><LI>[18]   Mitchell AP, Magasanik B. Biochemical and physiological aspects of glutamine synthetase inactivation in <I>Saccharomyces cerevisiae</I>. Journal of Biological Chemistry. 1984;259(19):12054-62.</LI><LI>[19]   Lazar Z, Rossignol T, Verbeke J, Crutz-Le Coq A, Nicaud J, Robak M. Optimized invertase expression and secretion cassette for improving <I>Yarrowia lipolytica</I> growth on sucrose for industrial applications. Journal of Industrial Microbiology &amp; Biotechnology. 2013;40(11):1273-83.</LI><LI>[20]   Madzak C. <I>Yarrowia lipolytica</I>: recent achievements in heterologous protein expression and pathway engineering. Applied Microbiology and Biotechnology. 2015;99(11):4559-77.</LI></UL></DIV></DIV></DIV><FOOTER id="footer-top"><DIV id="footer-contact-row"><A href="https://www.facebook.com/chalmersigem/?fref=ts">facebook</A><A href="https://twitter.com/igemchalmers">twitter</A><A href="mailto:igemgothenburg@gmail.com">e-mail</A></DIV></FOOTER></DIV></DIV></DIV></DIV></DIV></DIV></DIV></BODY></HTML>