Team:Chalmers Gothenburg/Project/Constructs
Chalmers Gothenburg iGEM 2016
Home
Team
Students
Supervisors
Attributions
Sponsors
Collaborations
Project
Description
Constructs
Promoter study
Model
Human Practices
Human Practices
Integrated Human Practices
Achievements
Project results
Parts
Notebook
Safety
Lab journal
Protocols
Flowcharts
PROJECT
Constructs
Synechocystis sp. PCC 6803 Simultaneous knock-out and insertion Knock-out Escherechia coli Bacillus subtilis Saccharomyces cerevisiae Yarrowia lipolytica References
All the constructs used in our project are summarized on this page. The DNA fragments of each construct were assembled with Gibson assembly after addition of overlapping sequences through PCR with primers containing overhangs to the adjacent DNA fragment (20-25 base pairs). Design of constructs and primers were performed in Snapgene with the aid of the web tools NEBuilder and Tm calculator . DNA sequences were obtained using Genbank, KEGG and Addgene.
Synechocystis sp. PCC 6803
The constructs for Synechocystis are based on three different knock-outs: argininosuccinate lyase (argH) , glutamine synthetase (glnA) and acetyl-CoA synthetase (acs) . ArgH is knocked out when Synechocystis is going to be used in a consortium with E. coli or B. subtilis, while glnA is knocked out when Y. lipolytica or S. cerevisiae is the producing organism. However, Synechocystis can still synthesise glutamine even if glnA is knocked out due to existence of a different type of glutamine synthetase in the genome (glnN) [1]. Due to limits in available antibiotics and time limits for marker recycling [2], the expression of glnN was repressed by growing the cyanobacteria with NH4+ as the nitrogen source [1].
In addition to constructing an auxotrophic cyanobacterium, ackA and pta are overexpressed in Synechocystis, along with knock-out of acs , to increase the metabolic flux towards acetate through the pta-AckA pathway [3]. As a starting point, the mutant JA06 received by the Paul Hudson group at KTH Royal Institute of Technology was used as it already excreted some amounts of acetate [4].
Figure 1. The plasmid pNF
The homology regions for the knock-out constructs are designed to be around 1000 base pairs each. All Synechocystis constructs are modified versions of the pNF plasmid, shown in Figure 1, which contains the E. coli origin of replication pBR322, an ampicillin resistance marker and a kanamycin resistance marker (KmR). This plasmid was linearized with the primers in Table 1, which only amplified the origin of replication and amp marker, before Gibson assembly.
One important thing to keep in mind when knocking out genes in Synechocystis is the fact that it has several copies of its genome [5]. To make sure that all of the copies of the specific amino acid synthesising gene is knocked out, transformants should be restreaked several times and checked through colony PCR [6].
Table 1. Primers used for linearizing pNF
Primers
Sequence (5’ → 3’)
pNF FW
GTTCACTGGCCGTCGTTTTAC
pNF RV
CATGCAAGCTTGGCGTAATC
Constructs for simultaneous knock-out and insertion
1. Knock-out argH + insert ackA
This construct consists of five parts: upstream homology region of argH, KmR cassette, pTrc promoter, ackA and the downstream homology region, Figure 2. This construct knocks out the argH gene when transformed into Synechocystis, making it an arginine auxotroph which is needed to make the cyanobacteria dependent on the producing organism. Furthermore, the endogenous ackA is inserted under the control of the strong consecutive promoter pTrc, for increased acetate production. By combining these two modifications into one construct, the number of constructs were reduced and thus the number of different markers needed.
Figure 2. Construct for simultaneous knock-out of ArgH and insertion of AckA
All the primers used for this construct are shown in Table 2. AckA and the flanking sequences of argH are amplified from genomic DNA of Synechocystis. The KmR cassette is amplified from the pNF plasmid and the pTrc promoter is amplified from the pCyJ2 plasmid.
Table 2. Primers used for generating the fragments for the argH knock-out and ackA insertion. Uppercase characters hybridize to the template, while the lowercase characters contain the overlapping overhang to the fragment within the parenthesis in the primer name. The numbering of the primers indicates which constructs it can be used for.
Primers
Sequence (5’ → 3’)
1,5 ArgH Up FW (pNF)
accatgattacgccaagcttgcatgCGCCTGTTATAACACCCC
1,5 ArgH Up RV (KmR)
gctctagagtgatagaattcGGGATTTCGTTGTGATAGTTAG
1 Kmr FW (argH)
ttgctaactatcacaacgaaatcccGAATTCTATCACTCTAGAGCCAGG
1 Kmr RV (pTrc)
attctgcctcgtgatacgcctaggtCGCTACTAGTACAACAAAGCCA
1,2, pTrc FW (KmR)
gctttgttgtactagtagcgACCTAGGCGTATCACGAGG
1,2, pTrc RV (AckA)
ttgagaatcaggaatttcatTCTAGATTTCTCCTCTTTTGTGTG
1 AckA FW (pTrc)
tcacacaaaagaggagaaatctagaATGAAATTCCTGATTCTCAATGC
1 AckA RV (argH)
aacctgggcatatctccacaccaatAGCCTAATTCAACATTTATCTTCAC
1 ArgH Dw FW (AckA)
gataaatgttgaattaggctATTGGTGTGGAGATATGCC
1,5 ArgH Dw RV (pNF)
cgttgtaaaacgacggccagtgaacCACCTTTTAAAAGAATGGCG
2. Knock-out GlnA + insert AckA
The purpose of this five-piece construct (Figure 3) is to make Synechocystis glutamine auxotroph, while also overexpressing ackA for increased acetate production.
Figure 3.Construct for simultaneous knock-out ofglnA and insertion of ackA.
All the primers used for this construct are shown in Table 3. AckA and the flanking sequences of glnA are amplified from genomic DNA of Synechocystis. The KmR cassette is amplified from the pNF plasmid and the pTrc promoter is amplified from the pCyJ2 plasmid.
Table 3. Primers used for generating the fragments for the glnA knock-out and ackA insertion. Uppercase characters hybridize to the template, while the lowercase characters contain the overlapping overhang to the fragment within the parenthesis in the primer name. The numbering of the primers indicates which constructs it can be used for.
Primers
Sequence (5’ → 3’)
2,6 glnA Up FW (pNF)
accatgattacgccaagcttgcatgAAAACGTCATGGCGATC
2,6 glnA Up RV (KmR)
gctctagagtgatagaattcTTTTTCTCCTTAGTGCAGTCAG
2 KmR FW (glnA)
tatctgactgcactaaggagaaaaaGAATTCTATCACTCTAGAGCCAGG
2 KmR RV (pTrc)
attctgcctcgtgatacgcctaggtCGCTACTAGTACAACAAAGCCA
1,2 pTrc FW (KmR)
gctttgttgtactagtagcgACCTAGGCGTATCACGAGG
1,2 pTrc RV (AckA)
ttgagaatcaggaatttcatTCTAGATTTCTCCTCTTTTGTGTG
2 AckA FW (pTrc)
tcacacaaaagaggagaaatctagaATGAAATTCCTGATTCTCAATGC
2 AckA RV (glnA)
ctaaaactgggtgagatggactggtAGCCTAATTCAACATTTATCTTCAC
2 glnA Dw FW (AckA)
gataaatgttgaattaggctACCAGTCCATCTCACCCA
2,6 glnA Dw RV (pNF)
cgttgtaaaacgacggccagtgaacGAGTGGATTTTAAAAACTCTTCGAC
3. Knock-out Acs + insert RFP
This construct (Figure 4) knocks out acs to increase the acetate production [3], while also expressing mRFP1 to be able to quantify the levels of Synechocystis compared to the producing organism in the microbial consortia.
Figure 4. Construct for simultaneous knock-out of acs and insertion of mRFP1
All the primers used for this construct are shown in Table 4. The flanking sequences of acs are amplified from genomic DNA of Synechocystis and mRFP1 is amplified from a plasmid supplied by the last year iGEM team of Chalmers Gothenburg. The SpR cassette and the pTrc promoter are amplified from the pCyJ2 plasmid. The terminator tB0015 is amplified from the pNF plasmid.
Table 4. Primers used for generating the fragments for the acs knock-out and mRFP1 insertion. Uppercase characters hybridize to the template, while the lowercase characters contain the overlapping overhang to the fragment within the parenthesis in the primer name. The numbering of the primers indicates which constructs it can be used for.
Primers
Sequence (5’ → 3’)
3,4,7 acs Up FW (pNF)
accatgattacgccaagcttgcatgCAAATTAGCCAAACCCACG
3,4,7 acs Up RV (SpR)
ccaccaattttctcttcagcTAGCGTGTTGGACAAATTACG
3,4 SpR FW (acs)
ctcccgtaatttgtccaacacgctaGCTGAAGAGAAAATTGGTGG
3,4 SpR RV (pTrc)
attctgcctcgtgatacgcctaggtTAAGAGGTTCCAACTTTCACC
3,4 pTrc FW (SpR)
gtgaaagttggaacctcttaACCTAGGCGTATCACGAGGC
3 pTrc RV (RFP1)
acgtcctcggaggaggCCATTCTAGATTTCTCCTCTTTTGTGTG
3 RFP FW (pTrc)
tcacacaaaagaggagaaatctagaATGGCCTCCTCCGAGG
3 RFP RV (tB0015)
tcgggtgggcctttctgcgtttataTTAGGCGCCGGTGGA
3 B0015 FW (RFP1)
gccactccaccggcgcctaaTATAAACGCAGAAAGGCCC
3 B0015 RV (acs Dw)
aaagactttgacggagaaccCCAGGCATCAAATAAAACG
3 acs Dw FW (tB0015)
gcctttcgttttatttgatgcctggGGTTCTCCGTCAAAGTCTTT
3,4,7 acs Dw RV (pNF)
cgttgtaaaacgacggccagtgaacTTTCCACTTCACTTGGTTTGT
4. Knock-out acs + insert pta
The purpose of the final knock-out and insertion construct (Figure 5) is to increase the acetate production in Synechocystis by knocking out acs and overexpressing pta [3].
Figure 5. Construct for simultaneous knock-out of Acs and insertion of Pta.
All the primers used for this construct are shown in Table 5. The fragments are almost the same as in construct 3, expect the overhangs of the fragments and the pta gene (amplified from genomic DNA of Synechocystis) which replaced the mRFP1.
Table 5. Primers used for generating the fragments for the acs knock-out and pta insertion. Uppercase characters hybridize to the template, while the lowercase characters contain the overlapping overhang to the fragment within the parenthesis in the primer name. The numbering of the primers indicates which constructs it can be used for.
Primers
Sequence (5’ → 3’)
3,4,7 acs Up FW (pNF)
accatgattacgccaagcttgcatgCAAATTAGCCAAACCCACG
3,4,7 acs Up RV (SpR)
ccaccaattttctcttcagcTAGCGTGTTGGACAAATTACG
3,4 SpR FW (acs)
ctcccgtaatttgtccaacacgctaGCTGAAGAGAAAATTGGTGG
3,4 SpR RV (pTrc)
attctgcctcgtgatacgcctaggtTAAGAGGTTCCAACTTTCACC
3,4 pTrc FW (SpR)
gtgaaagttggaacctcttaACCTAGGCGTATCACGAGGC
4 pTrc RV (pta)
aaataaagggaactcgtcatTCTAGATTTCTCCTCTTTTGTGTG
4 pta FW (pTrc)
tcacacaaaagaggagaaatctagaATGACGAGTTCCCTTTATTTAAGC
4 pta RV (acs Dw)
aaagactttgacggagaaccTTTGGAGGGCAAAATCAAG
4 acs Dw FW (pta)
aattaccttgattttgccctccaaaGGTTCTCCGTCAAAGTCTTT
3,4,7 acs Dw RV (pNF)
cgttgtaaaacgacggccagtgaacTTTCCACTTCACTTGGTTTGT
Constructs for knock-out only
The following constructs are simplified variants of the previous ones by limiting the modification to a knock-out, without inserting additional genes except the marker.
5. Knock-out argh
In this construct (Figure 6), argH is knocked out in Synechocystis by selection with kanamycin.
Figure 6. Construct for knock-out of argH.
All the primers used for this construct are shown in Table 6. The only difference in the fragments of this construct compared to construct 1 is the overlapping overhangs.
Table 6. Primers used for generating the fragments for the argH knock-out. Uppercase characters hybridize to the template, while the lowercase characters contain the overlapping overhang to the fragment within the parenthesis in the primer name. The numbering of the primers indicates which constructs it can be used for.
Primers
Sequence (5’ → 3’)
1,5 ArgH Up FW (pNF)
accatgattacgccaagcttgcatgCGCCTGTTATAACACCCC
1,5 ArgH Up RV (KmR)
gctctagagtgatagaattcGGGATTTCGTTGTGATAGTTAG
5,6 KmR FW
GAATTCTATCACTCTAGAGCCAGG
5,6 KmR RV
CGCTACTAGTACAACAAAGCCA
5 ArgH Dw FW (KmR)
acgtggctttgttgtactagtagcgATTGGTGTGGAGATATGC
1,5 ArgH Dw RV (pNF)
cgttgtaaaacgacggccagtgaacCACCTTTTAAAAGAATGGCG
6. Knock-out glnA
In this construct (Figure 7), argH is knocked out in Synechocystis by selection with kanamycin.
Figure 7. Construct for knock-out of GlnA.
All the primers used for this construct are shown in Table 7. The only difference in the fragments of this construct compared to construct 2 is the overlapping overhangs.
Table 7. Primers used for generating the fragments for the glnA knock-out. Uppercase characters hybridize to the template, while the lowercase characters contain the overlapping overhang to the fragment within the parenthesis in the primer name. The numbering of the primers indicates which constructs it can be used for.
Primers
Sequence (5’ → 3’)
2,6 glnA Up FW (pNF)
accatgattacgccaagcttgcatgAAAACGTCATGGCGATC
2,6 glnA Up RV (KmR)
gctctagagtgatagaattcTTTTTCTCCTTAGTGCAGTCAG
5,6 KmR FW
GAATTCTATCACTCTAGAGCCAGG
5,6 KmR RV
CGCTACTAGTACAACAAAGCCA
6 glnA Dw FW (KmR)
gctttgttgtactagtagcgACCAGTCCATCTCACCCA
2,6 glnA Dw RV (pNF)
cgttgtaaaacgacggccagtgaacGAGTGGATTTTAAAAACTCTTCGAC
7. Knock-out of acs
In this construct (Figure 8), acs is knocked out in Synechocystis by selection with spectinomycin.
Figure 8. Construct for knock-out of acs
All the primers used for this construct are shown in Table 8. The only difference in the fragments of this construct compared to construct 3 and 4 is the overlapping overhangs.
Table 8. Primers used for generating the fragments for the acs knock-out. Uppercase characters hybridize to the template, while the lowercase characters contain the overlapping overhang to the fragment within the parenthesis in the primer name. The numbering of the primers indicates which constructs it can be used for.
Primers
Sequence (5’ → 3’)
3,4,7 acs Up FW (pNF)
accatgattacgccaagcttgcatgCAAATTAGCCAAACCCACG
3,4,7 acs Up RV (SpR)
ccaccaattttctcttcagcTAGCGTGTTGGACAAATTACG
7 SpR FW
GCTGAAGAGAAAATTGGTGG
7 SpR RV
TAAGAGGTTCCAACTTTCACC
7 acs Dw F (SpR)
gtgaaagttggaacctcttaGGTTCTCCGTCAAAGTCTTT
3,4,7 acs Dw RV (pNF)
cgttgtaaaacgacggccagtgaacTTTCCACTTCACTTGGTTTGT
Escherichia coli
The constructs for E. coli were designed to make it overproduce arginine by removing the product inhibition by arginine on argA and knocking out the repressor argR. This overproduction is assumed to make the cell secrete arginine without needing to add any additional transporters.
1. Overexpression of point mutated argA + GFP insertion
To remove the feed-back inhibition of arginine, Tyrosine-19 in argA was mutated to Cysteine [7] through mutagenic PCR using a megaprimer method [8]. After generation of mutated argA fragment, the gene was assembled with Gibson assembly with the reporter gene GFP (for quantification of the ratio of E. coli to cyanobacteria in the co-culture), chloramphenicol resistance gene and upstream homology regions of argA to exchange the native promoter with the strong constitutive promoter J23119. The downstream homology region is included in the mutated argA fragments. The final construct is shown in Figure 9.
Figure 9. Construct for overexpressing mutated argA and inserting GFP into the genome of E. coli.
The primers used to assemble the fragments are shown in Table 9. The chloramphenicol resistance gene is amplified from the plasmid pKD3. The GFP gene is amplified from the BioBrick BBa K584001 . ArgA is amplified from genomic DNA of E. coli. The new promoter to argA, along with a RBS , was included as an overhang in the forward primer. The linear fragment produced was then inserted on the genome of E. coli via a pRED/ET mediated method [9].
Table 9. Primers used for generating the fragments for the overexpression of the point mutated argA and GFP insertion. Uppercase characters hybridize to the template, while the lowercase characters contain the overlapping overhang to the fragment within the parenthesis in the primer name.
Primers
Sequence (5’ → 3’)
cat FW (IS)
caccctgcgaaaaaacagaataaaaatacactaatttcgaataatcatgc aaagaggtgtgccGTGTAGGCTGGAGCTGCTTC
cat RV
ATGGGAATTAGCCATGGTCC
GFP FW (cat)
ggaccatggctaattcccatTTGACAGCTAGCTCAGTCCTAG
GFP RV (ArgA_mut)
aggactgagctagctgtcaaGTATATAAACGCAGAAAGGCCC
argA_mut FW
CGGTTCCCTgTATCAATACC
ArgA_mut RV
CTTGTGATACAGCGGTTCGT
ArgA_mut_amp FW
CACCCTGCGAAAAAACAG
2. ArgR knock-out
The second construct is meant to increase the arginine production by knocking out the repressor argR which is induced by the presence of arginine [10]. The argR::Kan knock-out provided by Anne Farewell was transduced to a recipient strain via P1 transduction [11]. The P1 phage was also provided by Anne Farewell.
Bacillus subtilis
The purpose of the constructs in B. subtilis is to enable it to grow on acetate and overproduce arginine. Both constructs are based on the vectors from the bacillus box made by the 2012 LMU-Munich iGEM-team [12].
1. Glyoxylate shunt
B. subtilis cannot utilize acetate as a carbon source, which can be solved by inserting the glyoxylate shunt from Bacillus licheniformis into B. subtilis, see Figure 10. The shunt consists of two enzymes in an operon, aceA and aceB, under the control of a native B. licheniformis promoter, which have been shown to be active in B. subtilis as well [13].
Figure 10. Construct for the glyoxylate shunt
The glyoxylate shunt was amplified from extracted genomic DNA from with the primers in Table 10. Both the product and the vector pBs4S were cut with XbaI and SpeI before ligation and transformation into E. coli. This was not optimal as there is a risk of vector-only clones due to overlaps of XbaI and SpeI sticky ends, but had to be done anyway because the shunt contained both EcoRI and PstI sites. They could have been removed through mutagenic PCR, but this was not done due to time limitations.
Table 10. Primers used for generating the fragments glyoxylate shunt. Uppercase characters hybridize to the template, while the lowercase characters contain the BioBrick prefix or suffix.
Primers
Sequence (5’ → 3’)
glyox op FW (prefix)
gaattcgcggccgcttctagaGAAAAATATGAACAAGCTATGAATAAAAAG
glyox op RV (suffix)
ctgcagcggccgctactagtaAACAAACAACAGGAATCATCAGAC
2. ahrC knock-out
The second modification to B. subtilis was to make it overproduce arginine needed by the cyanobacteria in the co-culture. This was done by deleting the gene ahrC, which codes for a repressor of arginine biosynthesis [14]. Furthermore, the reporter gene GFP was inserted to be able to quantify the abundance of B. subtilis relative the cyanobacteria in the co-culture. The final construct can be seen in Figure 11.
Figure 11. Construct for inserting GFP and knocking out the arginine repressor ahrC in B. subtilis.
The fragments were amplified with the primers in Table 11. The homology regions up- and downstream of ahrC was amplified from genomic DNA extracted from B. subtilis. The reporter gene was assembled by ligation of the BioBricks BBa_E0840 and BBa_K823003 , which was then amplified with primers containing overhangs for Gibson assembly. The chloramphenicol and ampicillin resistance gene were amplified from the pBs1C plasmid.
Table 11. Primers used for generating the fragments for knock-out of ahrC and the overexpression of GFP. Uppercase characters hybridize to the template, while the lowercase characters contain the overlapping overhang to the fragment within the parenthesis in the primer name.
Primers
Sequence (5’ → 3’)
Dw ahrC FW (amp+Ori)
aaccattattatcCCGATTCGATTTCTTCGAG
Dw ahrC RV
CAGAAAATCTAACAAAGATAAGAGGTG
cat FW (Dw ahrC)
cctcttatctttgttagattttctgTGTCAATTCTCATGTTTGACAGC
cat RV (pVeg+GFP)
tcgggtgggcctttctgcgtttataCTCCTGCATTAGGAAGCAGC
pVeg+GFP FW
TATAAACGCAGAAAGGCCC
pVeg+GFP RV (Up ahrC)
ttggaaatagaggtgcttacGGAGTTCTGAGAATTGGTATGC
Up ahrC FW (pVeg+GFP)
aactccGTAAGCACCTCTATTTCCAAGC
Up ahrC RV (ori+ampR)
gagctggatacttCTCTGTCAAAGATAAAATTATGATTG
amp+Ori FW (Up ahrC)
atctttgacagagAAGTATCCAGCTCGAGGTCG
amp+Ori RV (Dw ahrC)
gaaatcgaatcggGATAATAATGGTTTCTTAGACGTCAGG
Saccharomyces cerevisiae
The constructs for S. cerevisiae are divided into two subprojects: the promoters study and the promoter exchange. The results of the promoter study are meant to guide the choice of promoters in the promoter exchange for increased production of glutamine in S. cerevisiae, which needed by the cyanobacteria in the co-culture.
1. Promoter study
The constructs for the promoter study are based on the p416tef plasmid, Figure 12 [15]. The promoters used in this study are shown in Table 12 and were cloned into this plasmid together with GFP as the reporter gene. This was done using Gibson assembly with 20 bp overlaps after cutting the vector with SacI and XbaI.
By using a replicative plasmid instead of chromosomal integration, a higher copy number can be achieved which will make sure that even weak promoters give a detectable signal, which will be compared against the signal from the TEF1 promoter.
Figure 12. The plasmid p416tef
All promoters were amplified from the genome of S. cerevisiae CEN.PK113-5D and GFP was amplified from the plasmid p416tef gfp. The primers used to amplify all parts are shown in Table 12.
Table 12. Primers used for generating the fragments for the promoter study. Uppercase characters hybridize to the template, while the lowercase characters contain the overlapping overhang to the fragment within the parenthesis in the primer name.
Primers
Sequence (5’ → 3’)
GFP FW (p416 TEF1)
gcccgggggatccactagttCTATTTGTATAGTTCATCCATGCCATG
GFP R
ATGCGAATCCCCGGGTTA
pAQR1 FW (p416)
attaacccggggattcgcatTGCTGATTCGACTTTCTGAA
pAQR1 RV (p416)
gggaacaaaagctggagctcGTTCTGTTGCCGTATGCTATC
pFBP1 FW (p416)
attaacccggggattcgcatATGTGTGGTAGTATGAGGGATG
pFBP1 RV (p416)
gggaacaaaagctggagctcATATAAAATGAAAATAATATCCAAAGAAAAA
pGLN1 FW (p416)
attaacccggggattcgcatTTTTGATTATATTATATTATATTATGTTTAATTTTTGTT
pGLN1 RV (p416)
gggaacaaaagctggagctcGTATAAATAGTTATATAGAGATGAACTCTAAGCTAGTG
pPCK1 FW (p416)
attaacccggggattcgcatGTTGTTATTTTATTATGGAATAATTAGTTGC
pPCK1 RV (p416)
gggaacaaaagctggagctcTCGTTCGTTGTACGTACATTTAC
pPYK2 FW (p416)
attaacccggggattcgcatCGATAGTGCTTTTGTTGTAATCTT
pPYK2 RV (p416)
gggaacaaaagctggagctcCGCTTTTATGAACATATTCCG
2. Promoter exchange
The purpose of the promoter exchange is to fine tune the expression of Glutamine synthetase (GLN1) and Acids Quinidine Resistance (AQR1), which is an amino acid transporter [16], with the aim to increase the production and excretion of glutamine.
This is done in the strain S.cerevisiae IMX581, which has CRISPR/Cas9 integrated in its genome [17]. The plasmid pMEL13 from the same paper is linearized with PCR with primers in Table 13 prior Gibson assembly with a gBlock (ordered from IDT) containing the gRNA targeting the promoter region of GLN1 or AQR1, along with flanking regions of 50 basepair overlapping to pMEL13. The gRNA was designed with an online tool at Benchling.
Table 13. Primers used to linearize pMEL13.
Primers
Sequence (5’ → 3’)
pMEL13 F
GTTTTAGAGCTAGAAATAGCAAGTTAAA
pMEL13 R
GATCATTTATCTTTCACTGCGG
To exchange the promoter, the assembled pMEL13 was co-transformed with a linear DNA fragment containing the new promoter and homology regions to knock out either the promoter of GLN1 or AQR1, which was generated through PCR with the primers in Table 14. As overexpression of GLN1 might lead to glutamate auxotrophy [18], the results of the promoter study will help guide the choice of promoter for GLN1
Table 14. Primers used for generating the fragments for the promoter exchange. Uppercase characters hybridize to the template, while the lowercase characters contain the overlapping overhang to the fragment within the parenthesis in the primer name.
Primers
Sequence (5’ → 3’)
pFBP1 FW (pAQR1 up)
ggcctgcattgttttcttcagacgagaagccgttccaacgtttctttttctcg tcaccggATATAAAATGAAAATAATATCCAAAGAAA
pFBP1 RV (pAQR1 dw)
ctcattgtgagtaggatacatctcaatatcttctgtgtatatactgttacttc gtgacatATGTGTGGTAGTATGAGGGATG
pPCK1 FW (pGLN1 up)
agcggtcaggtgtaagtagtaggcttgataatgaattaaagatgactccgacg catattgTCGTTCGTTGTACGTACATTTAC
pPCK1 RV (pGLN1 dw)
ttggtccagttctagatatttttgtaaaatttgagtcttttcgatgcttgctt cagccatGTTGTTATTTTATTATGGAATAATTAGTTG
pPYK1 FW (pGLN1 up)
agcggtcaggtgtaagtagtaggcttgataatgaattaaagatgactccgacg catattgCGCTTTTATGAACATATTCCG
pPYK1 RV (pGLN1 dw)
ttggtccagttctagatatttttgtaaaatttgagtcttttcgatgcttgctt cagccatCGATAGTGCTTTTGTTGTAATCTT
pTEF1 FW (pAQR1 up)
gcctgcattgttttcttcagacgagaagccgttccaacgtttctttttctcgt caccggACAATGCATACTTTGTACGTTCA
pTEF1 RV (pAQR1 dw)
ctcattgtgagtaggatacatctcaatatcttctgtgtatatactgttacttc gtgacatTTTGTAATTAAAACTTAGATTAGATTGCT
Yarrowia lipolytica
The constructs of Y. lipolytica are based on the two plasmids JMP1047 and JMP2563 [19], Figure 13.  Before Gibson assembly, both plasmids were cut with BamHI and AvrII. The overlaps on the fragments for the Gibson assembly were designed to be 25 base pairs and were generated through PCR. On the first plasmid GFP and the native Glutamine synthetase (GLN1) are expressed and on the second the amino acid transporter AQR1 [16] from S. cerevisiae.
Figure 13. The plasmids JMP1047 and JMP2563 used for the Y. lipolytica constructs
1. Overexpression of GLN1 and GFP
A similar strategy was used for Yarrowia as in S. cerevisiae to increase the glutamine production. The native GLN1 was overexpressed under the control of the native pTEF1 promoter, and GFP was expressed under the same promoter to allow quantification of Y. lipolytica in the co-culture. For the full construct, see Figure 14. This construct was integrated into the genome of Y. lipolytica JMY2101 by random ectopic integration through the NHEJ recombination pathway [20].
Figure 14. Construct for overexpression of GLN1 and GFP in Y. lipolytica.
The fragments for the constructs were generated through PCR with the primers in Table 15. The gene GLN1 and the terminators tXPR2 and tLIP2 were amplified from genomic DNA of Y. lipolytica JMY2900. The GFP gene was amplified from the S. cerevisiae plasmid p416tef gfp. The promoter driving the expression of GFP was amplified from the JMP1047 plasmid.
Table 15. Primers used for generating the fragments for the overexpression of GLN1 and GFP. Uppercase characters hybridize to the template, while the lowercase characters contain the overlapping overhang to the fragment within the parenthesis in the primer name
Primers
Sequence (5’ → 3’)
Gln1 F (JMP1047)
tctccttgtcaactcacacccgaagATGAACTGGGAATCCGAAC
Gln1 R (tXPR2)
aattgcTTAGACCGTCTCAATGTACCG
tXPR2 F (Glu1)
ggtacattgagacggtctaaGCAATTAACAGATAGTTTGCCG
tXPR2 R (tLip2)
tgaagttcatctgcgttgaaAGATCTGAGCGTGAATTATACGG
tLIP2 F (tXPR2)
agatctTTCAACGCAGATGAACTTCA
tLip2 R (GFP)
tggatgaactatacaaatagTATTTATCACTCTTTACAACTTCTACCTCAAC
GFP F (tLIP2)
taaataCTATTTGTATAGTTCATCCATGCCATG
GFP R (pTef)
ctcacacccgaagATGCGAATCCCCGGGTTA
pTef F (GFP)
cggggattcgcatCTTCGGGTGTGAGTTGACAAG
pTef R (JMP1047)
aaatagcttagataccacagacaccCGATAGAGACCGGGTTGG
2. Overexpression of AQR1
The gene coding for the amino acid transporter AQR1 from S. cerevisiae CEN.PK113-5D was added to Y. lipolytica as we could not find a homolog in the genome of Y. lipolytica. It was inserted to the genome of the strain JMY330, through the same mechanisms as the previous construct, after Gibson assembly with the vector JMP2563.
Figure 15. Construct for overexpression of AQR1 in Y. lipolytica.
The fragments for this construct was generated through PCR with the primers in Table 16. The gene AQR1 was amplified from genomic DNA of S. cerevisiae CEN.PK113-5D and the terminator tXPR2 was amplified from genomic DNA of Y. lipolytica JMY2900.
Table 16. Primers used for generating the fragments for the overexpression of AQR1. Uppercase characters hybridize to the template, while the lowercase characters contain the overlapping overhang to the fragment within the parenthesis in the primer name.
Primers
Sequence (5’ → 3’)
AQR1 F (JMP2563)
caactcacacccgaaggatcATGTCACGAAGTAACAGTATATACACAG
AQR1 R (tXPR2)
gcaaactatctgttaattgcTTAATTATGATTATCGTTCTGGTCTC
tXPR2 F (AQR1)
agaacgataatcataattaaGCAATTAACAGATAGTTTGCCG
tXPR2 R (JMP2563)
agataccacagacaccctagAGATCTGAGCGTGAATTATACGG
References
[1]   Reyes JC, Muro-Pastor MI, Florencio FJ. Transcription of glutamine synthetase genes (glnA and glnN) from the cyanobacterium Synechocystis sp. strain PCC 6803 is differently regulated in response to nitrogen availability. Journal of Bacteriology. 1997;179(8):2678-89
[2]   Cheah YE, Albers SC, Peebles CAM. A novel counter‐selection method for markerless genetic modification in Synechocystis sp. PCC 6803. Biotechnology Progress. 2013;29(1):23-30.
[3]   Valgepea K, Adamberg K, Nahku R, Lahtvee P, Arike L, Vilu R. Systems biology approach reveals that overflow metabolism of acetate in Escherichia coli is triggered by carbon catabolite repression of acetyl-CoA synthetase. BMC Systems Biology. 2010;4:166
[4]   Anfelt J, Kaczmarzyk D, Shabestary K, Renberg B, Rockberg J, Nielsen J, Uhlén M, Hudson EP. Genetic and nutrient modulation of acetyl-CoA levels in Synechocystis for n-butanol production. Microbial cell factories. 2015;14(1):1.
[5]   Griese M, Lange C, Soppa J. Ploidy in cyanobacteria. FEMS Microbiology Letters. 2011;323(2):124-31.
[6]   Proels R. Stable Transformation of Cyanobacterium Synechocystis sp. BIO-PROTOCOL. 2014;4(21).
[7]   Rajagopal BS, DePonte J, Tuchman M, Malamy MH. Use of Inducible Feedback-Resistan tN-Acetylglutamate Synthetase (argA) Genes for Enhanced Arginine Biosynthesis by Genetically Engineered Escherichia coli K-12 Strains. Applied and environmental microbiology. 1998;64(5):1805-11.
[8]   Lai R, Bekessy A, Chen CC, Walsh T, Barnard R. Megaprimer mutagenesis using very long primers. BioTechniques. 2003;34(1):52-6.
[9]   Heermann R, Zeppenfeld T, Jung K. Simple generation of site-directed point mutations in the Escherichia coli chromosome using Red®/ET® Recombination. Microbial cell factories. 2008;7(1):1.
[10]   Lu C. Pathways and regulation of bacterial arginine metabolism and perspectives for obtaining arginine overproducing strains. Applied Microbiology and Biotechnology. 2006;70(3):261-72
[11]   Thomason LC, Costantino N, Court DL. E. coli genome manipulation by P1 transduction. Current protocols in molecular biology. 2007:1-7.
[12]   Radeck J, Kraft K, Bartels J, Cikovic T, Durr F, Emenegger J, et al. The Bacillus subtilis BioBrick Box: generation and evaluation of essential genetic building blocks for standardized work with Bacillus subtilis. JOURNAL OF BIOLOGICAL ENGINEERING. 2013;7(1):29
[13]   Kabisch J, Pratzka I, Meyer H, Albrecht D, Lalk M, Ehrenreich A, et al. Metabolic engineering of Bacillus subtilis for growth on overflow metabolites. Microbial Cell Factories. 2013;12(1):72
[14]   Larsen R, Buist G, Kuipers OP, Kok J. ArgR and AhrC Are Both Required for Regulation of Arginine Metabolism in Lactococcus lactis. Journal of Bacteriology. 2004;186(4):1147-57.
[15]   Mumberg D, Müller R, Funk M. Yeast vectors for the controlled expression of heterologous proteins in different genetic backgrounds. Gene. 1995;156(1):119-22.
[16]   Tenreiro S, Nunes PA, Viegas CA, Neves MS, Teixeira MC, Cabral MG, et al. AQR1 Gene (ORF YNL065w) Encodes a Plasma Membrane Transporter of the Major Facilitator Superfamily That Confers Resistance to Short-Chain Monocarboxylic Acids and Quinidine in Saccharomyces cerevisiae. Biochemical and Biophysical Research Communications. 2002;292(3):741-8.
[17]   Mans R, van Rossum HM, Wijsman M, Backx A, Kuijpers NGA, van den Broek M, et al. CRISPR/Cas9: a molecular Swiss army knife for simultaneous introduction of multiple genetic modifications in Saccharomyces cerevisiae. FEMS Yeast Research. 2015;15(2):1-15.
[18]   Mitchell AP, Magasanik B. Biochemical and physiological aspects of glutamine synthetase inactivation in Saccharomyces cerevisiae. Journal of Biological Chemistry. 1984;259(19):12054-62.
[19]   Lazar Z, Rossignol T, Verbeke J, Crutz-Le Coq A, Nicaud J, Robak M. Optimized invertase expression and secretion cassette for improving Yarrowia lipolytica growth on sucrose for industrial applications. Journal of Industrial Microbiology & Biotechnology. 2013;40(11):1273-83.
[20]   Madzak C. Yarrowia lipolytica: recent achievements in heterologous protein expression and pathway engineering. Applied Microbiology and Biotechnology. 2015;99(11):4559-77.
facebook twitter e-mail
