<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_Chalmers_Gothenburg_Description skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><DIV id="top_title"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:Chalmers Gothenburg/Description</SPAN></H1></DIV><DIV id="HQ_page"><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><P><TITLE>Chalmers Gothenburg iGEM 2016</TITLE></P><DIV class="project"><DIV id="header_navbar_wrapper"><DIV id="header-nav-anchor"><NAV><UL class="header_navbar_ul"><LI id="header_navbar_home"><A href="https://2016.igem.org/Team:Chalmers_Gothenburg" id="home_link">Home</A></LI><LI class="dropdown"><A href="https://2016.igem.org/Team:Chalmers_Gothenburg/Team" id="team_link">Team</A><DIV class="dropdown-content"><A href="https://2016.igem.org/Team:Chalmers_Gothenburg/Team/Students">Students</A><A href="https://2016.igem.org/Team:Chalmers_Gothenburg/Team/Supervisors">Supervisors</A><A href="https://2016.igem.org/Team:Chalmers_Gothenburg/Attributions">Attributions</A><A href="https://2016.igem.org/Team:Chalmers_Gothenburg/Team/Sponsors">Sponsors</A><A href="https://2016.igem.org/Team:Chalmers_Gothenburg/Collaborations">Collaborations</A></DIV></LI><LI class="dropdown"><A href="https://2016.igem.org/Team:Chalmers_Gothenburg/Project" id="project_link">Project</A><DIV class="dropdown-content"><A href="https://2016.igem.org/Team:Chalmers_Gothenburg/Description">Description</A><A href="https://2016.igem.org/Team:Chalmers_Gothenburg/Project/Constructs">Constructs</A><A href="https://2016.igem.org/Team:Chalmers_Gothenburg/Project/Promoter_study">Promoter study</A><A href="https://2016.igem.org/Team:Chalmers_Gothenburg/Model">Model</A></DIV></LI><LI class="dropdown"><A href="https://2016.igem.org/Team:Chalmers_Gothenburg/Human_Practices" id="HP_link">Human Practices</A><DIV class="dropdown-content"><A href="https://2016.igem.org/Team:Chalmers_Gothenburg/HP/Silver">Human Practices</A><A href="https://2016.igem.org/Team:Chalmers_Gothenburg/HP/Gold">Integrated Human Practices</A></DIV></LI><LI class="dropdown"><A href="https://2016.igem.org/Team:Chalmers_Gothenburg/Achievements" id="achievements_link">Achievements</A><DIV class="dropdown-content"><A href="https://2016.igem.org/Team:Chalmers_Gothenburg/Results">Project results</A><A href="https://2016.igem.org/Team:Chalmers_Gothenburg/Parts">Parts</A></DIV></LI><LI class="dropdown"><A href="https://2016.igem.org/Team:Chalmers_Gothenburg/Experiments" id="notebook_link">Notebook</A><DIV class="dropdown-content"><A href="https://2016.igem.org/Team:Chalmers_Gothenburg/Safety">Safety</A><A href="https://2016.igem.org/Team:Chalmers_Gothenburg/Experiments/Lab_journal">Lab journal</A><A href="https://2016.igem.org/Team:Chalmers_Gothenburg/Experiments/Protocols">Protocols</A><A href="https://2016.igem.org/Team:Chalmers_Gothenburg/Experiments/Flowcharts">Flowcharts</A></DIV></LI></UL></NAV></DIV></DIV><DIV class="wrap"><DIV class="header"><DIV class="header-text">PROJECT</DIV><DIV class="header-subtitle">Description</DIV></DIV><ASIDE id="side-nav"><DIV id="nav-anchor"><NAV><A href="#intro" class="active">Introduction</A><A href="#basic">The basic design</A><A href="#how">How will this work?</A><A href="#why">Why different production organisms?</A><A class="sub-level" href="#ecoli"><I>E. coli</I></A><A class="sub-level" href="#bsub"><I>B. subtilis</I></A><A class="sub-level" href="#ylip"><I>Y. lipolytica</I></A><A class="sub-level" href="#scer"><I>S. cerevisiae</I></A><A href="#promoter">Promoter study</A><A href="#char">Characterization of a previously existing BioBrick Part</A><A href="#references">References</A></NAV></DIV></ASIDE><DIV class="wrap-content"><DIV class="target" id="intro"><H2>Introduction</H2><P class="text">Current chemical synthesis based on a petroleum based platform have led to a disruption of environmental systems and continue to be a strong contributor to the emission of greenhouse gases. The accumulation of greenhouse gases, most notably carbon dioxide, is already creating dramatic changes in the worldwide temperatures and climate [1]. In 2016, our planet has reached the warmest global temperature ever recorded [2]. Human activity has been shown to be the main impacting factor of the increase of greenhouse gas concentrations during the last 150 years [3]. It has been proposed that even if the carbon dioxide emission stops immediately it would take 1000 years for the concentrations to return to normal level [4].</P><P class="text">Climate change is one the most important issue affecting our modern world, and we need to consider alternative ways of life to reduce our environmental impact. Several measures are being taken by scientists and industry to reduce the emission of greenhouse gas as much as possible, with carbon dioxide as the main focus. We have let this issue be an inspiration to us and came up with the main idea of our project. Inspired by nature, the idea is to utilize carbon dioxide as a carbon source for a system that could produce useful products while reusing excess carbon dioxide.</P><P class="text">Our project aims to use the carbon fixing ability of a photosynthetic microorganism to produce a carbon source that can be utilized by a wide range of microorganisms that are already used in industry for the production of chemical products today. For our photosynthetic microorganism we chose the cyanobacteria <I>Synechocystis</I>. We chose <I>Synechocystis</I> over, for example, microalgae because as a cyanobacteria it generally grows faster and have more tools available for genetic engineering [5].</P></DIV><DIV class="target" id="basic"><H2>The basic design</H2><P class="text">Since the carbon source is a major part of the cost for running industrial biosynthesis, utilizing carbon dioxide could be a major advantage not only from environmental perspective but also from an economical point of view. In order to achieve a stable co-culture, a dependent relationship between the two organisms is desired. This can be achieved in several ways, for example through quorum sensing [6] or metabolite exchange [7]. An exchange of metabolites was chosen due to the advantages of such a system, for example by providing additional fitness benefits due to the division of metabolic labor [8]. It has also been shown to be capable of purging cheaters from the system, i.e. cheaters do not seem to be able to outcompete the cooperating population [7]. Furthermore, in <A href="https://2015.igem.org/Team:Amsterdam">2015 the Amsterdam iGEM team</A> achieved a proof of concept in which they created a symbiotic culture between the cyanobacterium <I>Synechocystis</I> and <I>Escherichia coli</I> based on auxotrophy. This provided us with great inspiration on how to achieve our production system.</P><P class="text">The resulting system will allow for the direct conversion of atmospheric carbon dioxide into commercially used chemicals that are currently derived from fossil sources of carbon, as illustrated in Figure 1.</P><FIGURE><DIV><B>Figure 1.</B> General concept of the project with the cyanobacteria using light and carbon dioxide to produce acetate for a production organism</DIV></FIGURE><P class="text">The chosen production organisms were <I>Bacillus subtilis</I>, <I>Escherichia coli</I>, <I>Yarrowia lipolytica</I> and <I>Saccharomyces cerevisiae</I>. They were chosen since they have all proven to be capable of industrial level production. Furthermore, all of them have been the subject of extensive research, providing a good foundation for continued development.</P></DIV><DIV class="target" id="how"><H2>How will this work?</H2><P class="text">The modifications planned for each organism in the project are summarized in Figure 2. The project is divided into two systems: either with a prokaryotic or a eukaryotic production organism. For the prokaryotic organisms, arginine is overproduced and secreted. The cyanobacterium, which has its own gene coding for arginine biosynthesis knocked out, is therefore allowed to grow. The same strategy is used for the eukaryotic system, except glutamine is used instead of arginine. In both systems cyanobacteria will secrete acetate, acetate will be the only carbon source available for the production organisms thus leading to a co-dependency. This is achieved by overexpression and knock-out of several genes. For more details about the modifications, see the <A href="https://2016.igem.org/Team:Chalmers_Gothenburg/Project/Constructs">constructs page.</A></P><FIGURE><DIV><B>Figure 2.</B> Summary of all the genetic modifications planned in each organism.</DIV></FIGURE></DIV><DIV class="target" id="why"><H2>Why different production organisms?</H2><P class="text">With the use of organisms that have different strengths when it comes to product synthesis, a more efficient system can be obtained. Thus the production organism can easily be exchanged depending on the desired product. The result is a highly modular system with a wide range of production capabilities. Furthermore, there are several disadvantages when working with cyanobacteria alone. One of them is their slower growth rate than most chemotrophs [9], making genetic engineering of them slower. Since a future expansion of the biosynthesis field certainly will require extensive genetic engineering, working with this organism is not ideal. Leaving the product synthesis to faster growing organisms such as <I>Saccharomyces cerevisiae</I> and <I>Bacillus subtilis</I> will allow for faster implementation of new synthesis pathways.</P><P class="text">Additionally the use of photosynthetic microorganisms allows us to avoid several of the ethical and social issues that results from the current use of plant based substrate causes. For more information on this check out the <A href="https://2016.igem.org/Team:Chalmers_Gothenburg/HP/Gold">integrated human practices</A> part of our wiki.</P><DIV class="target" id="ecoli"><H3><I>Escherichia coli</I></H3><P class="text"><I>E. coli</I> is one of the most frequently used gram-negative prokaryotes in research and has been a model organism since the beginning of modern microbiology. There are many advantages to use it as one of the production organisms due to the abundant research available on it. Furthermore, it has a short generation time and can be easily transformed with altered plasmids which make it very easy to clone [10]. As a production organism <I>E. coli</I> was chosen for its ability to produce both heterologous and recombinant proteins [11].</P></DIV><DIV class="target" id="bsub"><H3><I>Bacillus subtilis</I></H3><P class="text">The Bacillus group of microorganisms, where <I>B. subtilis</I> being the most well known, are one of the most efficient and versatile groups of microorganisms at secreting proteins [12]. <I>B. subtilis</I> is mainly used on an industrial scale for production of enzymes that are used in a wide range of  applications such as medical care and laundry detergents [13]. Due to its wide usage in both industry and research, the <I>B. subtilis</I> genome is well annotated [14], facilitating further development of the organism for industrial use. Being a gram-positive bacteria, in contrast to <I>E. coli</I> which belongs to the gram-negative group, choosing <I>B. subtilis</I> will provide greater versatility to the project.</P></DIV><DIV class="target" id="ylip"><H3><I>Yarrowia lipolytica</I></H3><P class="text"><I>Yarrowia lipolytica</I> is one of the most used non-conventional yeasts, both in industry and research. It has the notable trait of being able to efficiently accumulate lipids, something that only a few microorganisms are capable of. The wild-type of this microorganism is able to accumulate lipids up to 10-15% of its biomass and with genetic engineering this ratio can reach over 90% [15]. <I>Y. lipolytica</I> is an obligate aerobic yeast with a wide substrate range, covering compounds such as alkanes and fatty acids [16]. Products from <I>Y. lipolytica</I> are diverse and include lipids, organic acids, proteases, and lysine. Most strains of <I>Y. lipolytica</I> can grow on acetate as a single carbon source, which is important for our project. Sodium acetate concentration of 0.4% is well tolerated, but higher concentrations weaken the growth rate and a concentration of 1% limits the growth [16].</P></DIV><DIV class="target" id="scer"><H3><I>Saccharomyces cerevisiae</I></H3><P class="text"><I>Saccharomyces cerevisiae</I> is one of the most widely used organisms in synthetic biology today. Thus, it has already been thoroughly investigated and thus there are many molecular biology tools available for it. As a eukaryote, its post-translational modification machinery gives it potential to synthesize proteins which are not available when using prokaryotes [17]. Because of those reasons, it was a reasonable choice for our project. On top of that, there is a lot of competence on <I>S. cerevisiae</I> at the synthetic biology department at Chalmers (the department where the project was performed).</P></DIV></DIV><DIV class="target" id="promoter"><H2>Promoter study</H2><P class="text">To be able to create a fine-tuned co-culture dependance between the cyanobacteria and <I>S. cerevisiae</I> a promoter for the glutamine production with a very specific strength is needed to avoid auxotrophy of glutamate in the <I>S. cerevisiae</I> [18]. A promoter study was developed as being a separate entity alongside of the project to provide aid for the choice of promoter for glutamine synthetase. In order to characterize the strength of the different promoters on BG-11 with 0.5% of acetate compared to 2% glucose we used GFP as reporter gene. The promoters characterized are shown in Table 1. The complete promoter study can be read in <A href="https://2016.igem.org/Team:Chalmers_Gothenburg/Project/Promoter_study">promoter study</A> part of wiki.</P><FIGURE><FIGCAPTION class="table-caption"><B>Table 1.</B> Description of the five promoters characterized in the promoter study.</FIGCAPTION><TABLE><TBODY><TR><TH>Promoters</TH><TH>Comments</TH></TR><TR><TD>pAQR1</TD><TD>Induced by amino acids [19]</TD></TR><TR><TD>pGLN1</TD><TD>Repressed by glutamine [20]</TD></TR><TR><TD>pPCK1</TD><TD>Repressed by glucose and 10 times increased expression by acetate [21]</TD></TR><TR><TD>pPYK2</TD><TD>Repressed by glucose [22]</TD></TR><TR><TD>pTEF1</TD><TD>Strong constitutive promoter [23]</TD></TR></TBODY></TABLE></FIGURE></DIV><DIV class="target" id="char"><H2>Characterization of a previously existing BioBrick Part</H2><P class="text">Through the promoter study, existing BioBricks of the native promoter pTEF1 in <I>S. cerevisiae</I> (<A href="http://parts.igem.org/Part:BBa_K319003">BBa_K319003</A> and <A href="http://parts.igem.org/Part:BBa_K563004">BBa_K563004</A>) were characterized further and the data was added to registry pages. The study was performed in S. cerevisiae via a replicate plasmid and the expression was quantified with GFP. The expression was investigated with two different carbon sources, glucose and acetate. More detailed description of the promoter study and its results can be also viewed in <A href="https://2016.igem.org/Team:Chalmers_Gothenburg/Project/Promoter_study">promoter study</A> part of wiki.</P></DIV><DIV class="target" id="references"><H3>References</H3><DIV class="reference_list"><UL><LI>[1]   Nordhaus WD. To slow or not to slow: the economics of the greenhouse effect. The economic journal. 1991 Jul 1;101(407):920-37.</LI><LI>[2]   Schmunk, RB; Schmidt, GA. NASA Goddard Institute for Space Studies. NASA Analysis Finds August 2016 Another Record Month (2016, September 12). Retrieved from: http://data.giss.nasa.gov/gistemp/news/20160912/</LI><LI>[3]   Solomon S, editor. Climate change 2007-the physical science basis: Working group I contribution to the fourth assessment report of the IPCC. Cambridge University Press; 2007 Sep 10.</LI><LI>[4]   Solomon S, Plattner GK, Knutti R, Friedlingstein P. Irreversible climate change due to carbon dioxide emissions. Proceedings of the national academy of sciences. 2009 Jan 28:pnas-0812721106.</LI><LI>[5]   Berla BM, Saha R, Immethun CM, Maranas CD, Moon TS, Pakrasi HB. Synthetic biology of cyanobacteria: Unique challenges and opportunities. Frontiers in Microbiology. 2013;4:246</LI><LI>[6]   Xavier JB, Kim W, Foster KR. A molecular mechanism that stabilizes cooperative secretions in <I>Pseudomonas aeruginosa</I>. Molecular microbiology. 2011 Jan 1;79(1):166-79</LI><LI>[7]   Waite AJ, Shou W. Adaptation to a new environment allows cooperators to purge cheaters stochastically. Proceedings of the National Academy of Sciences. 2012 Nov 20;109(47):19079-86.</LI><LI>[8]   Pande S, Merker H, Bohl K, Reichelt M, Schuster S, de Figueiredo LF, Kaleta C, Kost C. Fitness and stability of obligate cross-feeding interactions that emerge upon gene loss in bacteria. The ISME journal. 2014 May 1;8(5):953-62.</LI><LI>[9]   LÜRLING M, Eshetu F, Faassen EJ, Kosten S, Huszar VL. Comparison of cyanobacterial and green algal growth rates at different temperatures. Freshwater Biology. 2013 Mar 1;58(3):552-9.</LI><LI>[10]   Tortora GJ, Funke BR, Case CL. Cytomegalovirus Infections. Microbiology An Introduction, 10th ed. San Fransisco: Pearson Education Inc. 2010:658.</LI><LI>[11]   Russo E. The birth of biotechnology. Nature. 2003;421(6921):456-7.</LI><LI>[12]   Simonen M, Palva I. Protein secretion in Bacillus species. Microbiological Reviews. 1993 Mar 1;57(1):109-37</LI><LI>[13]   van Dijl JM, Hecker M. <I>Bacillus subtilis</I>: from soil bacterium to super-secreting cell factory. Microbial Cell Factories. 2013;12(1):3-.</LI><LI>[14]   Liu L, Liu Y, Shin H, Chen RR, Wang NS, Li J, et al. Developing Bacillus spp. as a cell factory for production of microbial enzymes and industrially important biochemicals in the context of systems and synthetic biology. Applied Microbiology and Biotechnology. 2013;97(14):6113-27.</LI><LI>[15]   Blazeck J, Hill A, Liu L, Knight R, Miller J, Pan A, et al. Harnessing <I>Yarrowia lipolytica</I> lipogenesis to create a platform for lipid and biofuel production. NATURE COMMUNICATIONS. 2014;5:3131.</LI><LI>[16]   Wolf K, SpringerLink (Online service), SpringerLink Archive (e-book collection). Nonconventional Yeasts in Biotechnology: A Handbook. Berlin, Heidelberg: Springer Berlin Heidelberg; 1996. ch. 10</LI><LI>[17]   Martínez JL, Liu L, Petranovic D, Nielsen J. Pharmaceutical protein production by yeast: towards production of human blood proteins by microbial fermentation. Current opinion in biotechnology. 2012 Dec;23(6):965-71.</LI><LI>[18]   Mitchell AP, Magasanik B. Biochemical and physiological aspects of glutamine synthetase inactivation in <I>Saccharomyces cerevisiae</I>. Journal of Biological Chemistry. 1984;259(19):12054-62.</LI><LI>[19]   Velasco I, Tenreiro S, Calderon IL, André B. <I>Saccharomyces cerevisiae</I> Aqr1 Is an Internal-Membrane Transporter Involved in Excretion of Amino Acids. Eukaryotic Cell. 2004;3(6):1492-503.</LI><LI>[20]   ter Schure EG, Silljé HH, Vermeulen EE, Kalhorn JW, Verkleij AJ, Boonstra J, Verrips CT. Repression of nitrogen catabolic genes by ammonia and glutamine in nitrogen-limited continuous cultures of <I>Saccharomyces cerevisiae</I>. Microbiology. 1998 May 1;144(5):1451-62.</LI><LI>[21]   Weinhandl K, Winkler M, Glieder A, Camattari A. Carbon source dependent promoters in yeasts. MICROBIAL CELL FACTORIES. 2014;13(1):5.</LI><LI>[22]   Boles E, Schulte F, Miosga T, Freidel K, Schlüter E, Zimmermann FK, et al. Characterization of a glucoserepressed pyruvate kinase (Pyk2p) in <I>Saccharomyces cerevisiae</I> that is catalytically insensitive to fructose-1,6-bisphosphate. Journal of Bacteriology. 1997;179(9):2987¬93</LI><LI>[23]   Partow S, Siewers V, Bjørn S, Nielsen J, Maury J. Characterization of different promoters for designing a new expression vRector in <I>Saccharomyces cerevisiae</I>. Yeast. 2010 Nov 1;27(11):955-64</LI></UL></DIV></DIV><FOOTER id="footer-top"><DIV id="footer-contact-row"><A href="https://www.facebook.com/chalmersigem/?fref=ts">facebook</A><A href="https://twitter.com/igemchalmers">twitter</A><A href="mailto:igemgothenburg@gmail.com">e-mail</A></DIV></FOOTER></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></BODY></HTML>