<HTML xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en" dir="ltr">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki  ltr ns-0 ns-subject page-Team_Cornell_Notebook"><DIV id="globalWrapper"><DIV id="top-section"><DIV id="p-logo"><A href="/Main_Page" title="Main Page">&quot;
	    </A></DIV><DIV id="menubar" class="left-menu"><UL><LI class="selected"><A href="/Team:Cornell/Notebook">Page               </A></LI><LI class="new"><A href="/wiki/index.php?title=Talk:Team:Cornell/Notebook&amp;action=edit&amp;redlink=1">Discussion               </A></LI><LI><A href="/wiki/index.php?title=Team:Cornell/Notebook&amp;action=edit">View source               </A></LI><LI><A href="/wiki/index.php?title=Team:Cornell/Notebook&amp;action=history">History               </A></LI><LI style="color:#808080;cursor:default">teams</LI></UL></DIV><DIV class="right-menu" id="menubar"><UL><LI id="pt-login"><A href="/wiki/index.php?title=Special:UserLogin&amp;returnto=Team:Cornell/Notebook" title="You are encouraged to log in; however, it is not mandatory [o]" accesskey="o">Log in</A></LI></UL></DIV><DIV id="search-controls"><FORM action="/Special:Search" id="searchform"> </FORM></DIV></DIV><DIV id="content"><H1 class="firstHeading">Team:Cornell/Notebook</H1><DIV id="bodyContent"><H3 id="siteSub">From 2009.igem.org</H3><TABLE align="justify"><TBODY><TR><TH align="center"><A href="/Team:Cornell" title="Team:Cornell">The Project</A></TH><TH align="center"><A href="/Team:Cornell/Project/Background" title="Team:Cornell/Project/Background">Background</A></TH><TH align="center"><A href="/Team:Cornell/Project/Chassis" title="Team:Cornell/Project/Chassis">Chassis</A></TH><TH align="center"><A href="/Team:Cornell/Project/Design" title="Team:Cornell/Project/Design">Design</A></TH><TH align="center"><A href="/Team:Cornell/Project/Standardization" title="Team:Cornell/Project/Standardization">Standardization</A></TH><TH align="center"><A href="/Team:Cornell/Parts" title="Team:Cornell/Parts">Parts Submitted to the Registry</A></TH><TH align="center"><STRONG class="selflink">Notebook</STRONG></TH><TH align="center"><A href="/Team:Cornell/Team" title="Team:Cornell/Team">The Team</A></TH></TR></TBODY></TABLE><P>6/26/09
Alyssa and Matt—made chloramphenicol, kanamycin, and tetracycline antibiotic stock solutions for use in agar plates.  See “Protocols” folder for antibiotic making protocol.
</P><P>
Antibiotic	Volume (mL):	Stock (mg/mL):	Amnt use (g):	Solvent
chloramphenicol	15	34(200x)	0.510	EtOH
kanamycin	15	10(200x)	0.150	H20
tetracycline	50	5(200x)	0.250	EtOH
</P><P>Comments:  
Please note that tetracycline does NOT fully dissolve in 100% EtOH—will see a yellow powder in a clear solution.
For all stock solutions of antibiotics dissolved in H20:  “sterilize by filtration through a 0.22um filter (Sambrook and Russel)”
</P><P>Xing—PCR cloning of GFP+ into Biobrick standard A
</P><P>6/29/09
</P><P>Malinka and Ben—Spread antibiotics on agar plates for ccdb amplification.  Created a table for the antibiotics we were going to use and then top spread antibiotics onto agar plates.  
</P><P>Comments:
The color code for the antibiotics: 
</P><PRE>                                     Amp – Black
</PRE><P>				        Chlor – Red
				        Kan – Green
				        Tet – Blue
Table they came up with is Ben’s notebook entry for the day.  Results are summarized as the following: 
--pSB1AK3 had colonies on each plate
water plates had no colonies, except for amp plates
pSB1AC3 and pSB1AT3 had no colonies, except for on the amp plates
Xing Agarose Gel Electrophoresis Analysis of GFP+ PCR Product to analyze products of PCR of GFP+
</P><P>
Lanes	1	2	3	4	5	6	7	8
PFx 1ng	5λ							
PFx 10ng		5λ						
PFx 100ng			5λ					
Pfu 1ng				5λ				
Pfu 10ng					5λ			
Pfu 100ng						5λ		
pGFP+ 1							5λ	
100bp ladder Fermentas								5λ
6X Loading dye	2λ	2λ	2λ	2λ	2λ	2λ	2λ	
Total Volume	7λ	7λ	7λ	7λ	7λ	7λ	7λ	5λ
</P><P>Comments:
The rxns using Pfx display product at the expected mobility ~750bp with increasing yields following increasing amount of template plasmid used. The rxns treated with Pfu displayed no product.
</P><P>6/30/09
</P><P>Xing—PCR Amplification of yvgY promoter &amp; GFP+.   The goal was to PCR Amplify yvgY promoter from chromosome into BioBrick standard and repeat GFP+ amplification since we used 10X more primer in the previous experiment than we should have.
</P><P>Malinka and Ben—extracted iGEM backbone plasmids from kit (wells 11c for pSB1AC3, 13e for pSB1AK3, and 15c for pSB1AT3) and transformed them.  Also grew B. subtilis strains to make frozen permanents. 
Comments:
Ben's plates (for pSB1AK3) had various numbers of colonies
Malinka’s plates (for pSB1AC3 and pSB1AT3)
Nothing grew on the 50 ul of only Tet or ChlorzNothing grew in the 50 ul or 250 ul of the Tet and amp and the Chlor and Amp
No colonies on water plates
Some cells grew on the only amp plates.  Thus amp not very effective, and there was a large “background”
</P><P>7/1 
Xing;  Agarose Gel Electrophoresis of GFP+/yvgY promoter PCR Products
GFP+ bands display expected mobility, with greater yield in the 1ng rxn than the previous rxn. No apparent product in rxns amplified by Pfu. yvgY shows up faintly for rxns using Pfx and Pfu with expected mobility &lt;100bp. 
Alyssa and Matt: Growth Curve Characterization of B Subtilis at Different Culture Scales To characterize the growth curves of B Subtilis in both ml scale tubes and in microtiter plates by measuring optical density at set intervals during growth.  (By finding when the strains enter logarithmic growth phase, we can better design our B galactosidase assay protocol.)  Calculations assume midlog phase.
Ben: Inoculating pSB1AK3 &amp; re-plating pSB1AC3 and pSB1AT3.  Results: Inoculating pSB1AK3
All tubes contain cells…SUCCESS!  Re-plating pSB1AC3 and pSB1AT3
None of the water plates grew cells, except for the tet lower concentration plate
The supposed plasmid containing cells also did not grow, except for some background on the lower concentration tet plate.
Malinka: Replating plasmids and making frozen permanents Replate the pSB1AC3 and pSB1AT3 and do frozen permanents
Results:  Plasmids didn’t grow, lawn on water
</P><P>7/2 
Xing:  Gel Electrophoresis of yvgY promoter PCR rxn with primers
Compare yvgY PCR rxns with primers to confirm or refute presence of product
Primers do not show up on gel, there is a band below 100bp for the PCR rxns. The expected product size is ~120 bp. The &lt;100 band could be due to primer dimerization. Run a primer titration down to 50nM primer to check next time.
Ben:  Re-transforming pSB1AC3 and pSB1AT3 and plating ccdB-resistant strain.  Results: Re-transforming pSB1AC3 and pSB1AT3-No plates contained cells, except for water amp, and puc19 (both had empty circle inside, may be problem with plates).  plating ccdB-resistant strain -2 lawns formed
Malinka: Re-transforming pSB1AC3 and pSB1AT3 and plating ccdB-resistant strain
Results: -No plates contained cells, except for water amp, and puc19 (both had empty circle inside, may be problem with plates) plating ccdB-resistant strain-2 lawns formed
</P><P>7/3 
Xing:  Preparation of Chemically Competent E.Coli. Part II
Xing:  Gel Analysis PCR of  yvgY and yvgW promoters primer titration; Run agarose gel electrophoresis to analyze PCR rxns yvgY rxns display the same band &lt;100bp but appears less intense for .1ng/uL primer rxn and even less for .05ng/uL primer rxn, supporting the hypothesis that these are primer dimerization products. yvgW does not display any products. Possibility of broken chromosomal DNA? Extract Bacillus DNA next time.
Xing:  PCR Amplification of yvgY and yvgW promoter PCR amplify yvgY and yvgW promoter into BB standard and titrate primers to see if proper product forms.
Xing:  Prepare Kanamycin and Tetracycline Plates
Ben: Preparation of Chemically Competent E.Coli.
Malinka: Prepare Chloramphenicol and Ampicillin Plates, and grow competent cells
</P><P>7/6
Ben and Malinka: Checking competency of cells/ if plates are working (by Malinka).  Bad results, redo experiment
7/7
Ben:  Re-re-transforming cells (LOL). Transform (hopefully) pSB1AC3 and pSB1AT3 in ccdB-resistant and DH5a strains.
--Results: -Tet plates (including water and both pSB1AT3 plates) contained lawns, reason unkown
-Chlor plates came out as predicted, nothing on water plates, dozens of colonies on 250uL plate, &lt;10 on 50uL plate, but colonies were not red still need to view them under UV light
-water controls were clean, except for Tet, as said above
-pUC19 1:10 dilution had &lt;10 colonies, 1:100 had none visible  replating at 1:1 to check competent cells’
Malinka: Replating plasmid backbones-- Playing pSB1AK3 and pSB1AC3 on new antibiotic plates
--Results: pSB1AT3 (ccdb) – Proportional amounts of single colonies had grown on my 50 and 250 ul plates. However a similar amount of colonies had grown on the water too. 
</P><P>pSB1AT3 (DH5a) – Ben had lawns on all his Tet plates.
</P><P>pSB1AC3 (ccdb) – I had nothing on all 3 of my chlor plate.
</P><P>pSB1AC3 (DH5a) – Proportional amounts of single colonies had grown on his 50 and 250 ul plates. Nothing had grown on his water. The cells weren’t red like we expected them to be but that was probably because they were too small.
</P><P>Water (1:1000) (ccdb and DH5a) – We had lots of single colonies on both our plates. They looked similar.
</P><P>pUC19 (ccdb) - 
</P><P>pUC19 (DH5a) - 
</P><P>
7/8
Xing:  PCR yvgY promoter &amp; yvgW temperature using a temperature gradient
Xing:  Preparation of mntR frozen permanents
Ben: Innoculating pSB1AC3 and pSB1AT3 (rfp versions) to see if we have transformed backbones.  
--Results: Chlor plasmids had growth, reddish in tube!-Tet did not grow, as expected
Malinka: culture backbone from the plates done yesterday
--Results: All of Ben’s pSB1AC3s grew and turned red. Only 1 of Malinka’s pSB1AT3s grew
Alyssa: Preparing B. subtilis cultures exposed to ZnCl2 for practice LacZ Assay-- To try out the B-Gal assay protocol mentioned on page 12 of Helmann’s “Genetic and physiological responses of B. subtilis to metal ion stress” paper
--Results: Whoa!  Heavy metals/ growth retardation in action!  According to the last OD600 readings, ZnCl2 seems to significantly (and negatively) affect the growth rates of the B. subtilis strains exposed to the highest concentrations of metal solution.
</P><P>7/9 
Xing:  Agarose Gel Electrophoresis of PCR yvgY yvgW temperature gradient to analyze PCR products-- yvgY rxn still forms primer dimmers, though concentration seems to decrease with increasing temperature. No products for yvgW rxn.
Matt and Alyssa:  Designed Zinc growth curve plate
Ben: Mini-prep pSB1AC3 (rfp) and pSB1AK3 (ccdB)
7/10
Xing:  Agarose Gel Electrophoresis of Temperature Gradient to analyze PCR products
Xing:  PCR temperature gradient of yvgY &amp; yvgW-- PCR yvgY &amp; yvgW promoters with positive controls.
7/13
Malinka: Digesting the plasmid backbones PSB1AC3 and PSB1AK3
</P><P>7/14
Xing:  Gel electrophoresis of varying [Mg2+] PCR
Xing:  PCR Amplification of yvgY &amp; yvgW w/ varying [Mg2+]-- Vary [Mg2+] in PCR rxn to see if products form
Malinka:  ran gels for backbones
7/15
Matt: Cadmium and Zinc Beta Galactosidase Assay-- To characterize the effects of Cadmium and Zinc concentration on the promoter for the CadA-lacZ fusion.
Malinka: Digestion - Removing inserts from the plasmid backbones PSB1AC3, PSB1AK3, PSB1AT3
--Results:  
</P><P>7/16
Ben:  DNA electrophoresis of doubly digested (X &amp; S) backbones and GFP+
7/17
Xing:  PNK treatment and annealing of complementary primers-- PNK treat primers to phosphorylate ends for ligation/anneal complementary primers
Alyssa:  CdCl2 soln growth curve plate design for use in TECAN machine
7/20
Ben:  Check of pSB1AK3 ccdB-gene effectiveness through transformation in DH5a
--Results: -waters were blank-puc19 had a few cells -sSB1AK3 plates both transformed, meaning the ccdB gene is broken, since the cells should have died
Malinka: Dephosphoralating and purifying backbone plasmids PSB1AC3, PSB1AK3, PSB1AT3
7/21
Matt and Alyssa:  Cadmium and Zinc Beta Galactosidase Assay-- To characterize the effects of Cadmium and Zinc concentration on the promoter for the CadA-lacZ fusion.
Ben: Ligation of promoters and GFP+
Malinka: Creating GFP and promoter biobricks—yvgW and yvgY
--Results: Based on yesterdays Tet plate results, we found that the Tet plates work. But the ccdb psb1AK3 does not work so we have only pSB1AC3 and pSB1AT3 as backbones.
7/22
Ben: Transformation of ligations yvgW and GFP+
Malinka: Redigestion - Removing inserts from the plasmid backbones
Results: Plated Ligated Biobricks 
</P><P>Controls:
The pUC19 on amp showed reasonable growth and the water on amp had nothing which meant that there was no background on the amp plates. As we still got good growth for pUC19, this showed that DH5 cells were alive and competent. The water on the Chlor plater had no growth. 
</P><P>The no insert and no enzyme ligation mixtures showed no growth.
</P><P>yvgW and yvgY:
There was little growth on the 2:1 ratio plates for yvgW and yvgY (maybe 5 colonies). There was more growth on the 6:1 ratio plates (maybe 15 colonies) – we plan to use 6:1 ratio from now on.
</P><P>GFP:
There was no growth on any of the GFP biobrick plates (2:1, 4:1 or 6:1).
</P><P>Undigested plasmid (pSB1AC3) would have shown up as red cells on the plate. However from previous experience, we knew that 24 hours of incubation wasn't enough for the cells to have grown large enough or long enough to be visibly red. So we weren't sure, of the plates with growth in them, which cells actually had been transformed with our biobricks as opposed to the undigested plasmid. 
</P><P>7/23
Ben: Transformation of pSB1K3 and pSB1T3 backbones
--Results: -Kan plates produced colonies (slightly red) -waters had nothing (except tet plates  bad plates) -pUC19 had colonies -tet plates all had lawns  bad plates
Malinka: To run gels on the newly digested plasmid backbones and cut GFP to make sure we had got them right.   To dephosphorylate the cut backbones and then purify them so that we had enough vector to ligate with our inserts later.
--Results: The gels looked good for Chlor, Tet and Kan. I was even able to see a the insert for Kan very low in the gel (it was very faint) – exactly where the iGEM gel showed it to be.
</P><P>I got nothing for the cut and uncut GFP. Ben had used uncut GFP from the same tube a few days ago and got results but he used 1 ul which is (600-700 ng) so perhaps I had too little. We need to do this again with more cut and uncut GFP.
</P><P>
Xing – To culture tubes with colonies from our transformed biobricks (with promoters). Also to culture 1 tube with pSB1AC3 (with RFP) because we were going to use RFP  (and GFP) for our project. 
</P><P>Ben and Matt – To make new Chlor and Tet plates and transform cells with new backbone plasmid from the kit (pSB1T3 &amp; pSB1K3) in case our next backbone doesn't work. 
7/24
Cadmium and Zinc Beta Galactosidase Assay in tubes: To characterize the effects of Cadmium and Zinc concentration on the promoter for the CadA-lacZ fusion. Couldn’t access spectrometer when the assay was complete, so tubes were stopped when they showed a clear yellow color and left to sit out over the weekend. On Monday all remaining tubes were stopped and OD420, OD550, and OD600 were measured in the spec.
Malinka:  Test--Digesting the pSB1AC3 and pSB1AT3 using 15 minute method and digesting purified GFP+
--Results: The 15 minute digest of pSB1AC3 and pSB1AT3 showed the same results as the digests of the same plasmids done overnight.
</P><P>We saw the uncut GFP but did not see the cut GFP. We think it’s because the GFP+ wasn’t purified.
</P><P>7/24
Xing:  Purification of GFP+ PCR Products
</P><P>7/27
Xing;  Purification of Additional GFP+ PCR Products
Alyssa and Malinka: Digesting GFP+ and running gels on it-- To test pure cut GFP+ digested for 6 hours, cut and then test GFP+ digested for 15 minutes, cut and then test new backbones, and cut and then test biobrick parts with promoters.
7/29
Ben: Mini-prep of GFP (E0040), CFP (E0020), and YFP (E0030)
Alyssa:  designed primers to extract constitutive mrgA promoter from Helmann strain of B. sub.
7/30
Alyssa and Xing:  Digest pSB1K3 &amp; ligate GFP+ part onto digest--Digestion of assembly vector pSB1K3, dephosphorylation &amp; ligation of GFP+ onto assembly vector. 
Ben: Digestion of GFP CFP and YFP-- Appropriate bands located in all wells: perfect!
</P><P>7/31
Xing: Transformation of GFP+ Ligation and selection with Kanamycin
Xing:  Transformation of pSB1AC3 using Amp selection--1 Colony for 50uL pSB1AC3
~  10 Colonies for 250 uL pSB1AC3, no colonies for H2O, pUC19 250 uL 0-5 colonies.
Alyssa and Ben: Ligation of assembly vector PSBK3 to GFP+ to create GFP+ biobrick parts
</P><P>8/3
</P><PRE>Xing: Digest &amp; gel analysis of yvgW, yvgY, GFP+ BB parts
</PRE><P>- No GFP+ parts, incomplete digestion for yvgW, yvgY parts.
Malinka: Mini-prepped GFP+ biobricks and pSB1AC3
8/4 Xing: Digest &amp; gel analysis of yvgW, yvgY BB parts
- Mobility as expected for for yvgW 2:1 1 and yvgY 6:1 3
Malinka: Transformed and plated pSB1AT3 (rfp and ccdb) from the kit.
</P><P>8/5 Xing: Testing effectiveness of tetracycline plates
- Lawn formed on all tet plates
Malinka: Cultured pSB1AT3 (DH5) cells, and yvgY (on pSB1AC3), yvgW (on pSB1AC3) and GFP+ (on pSB1AK3) biobricks.
Matt: Previous Tetracyclin plates have failed to kill strains without Tet resistance. Matt made another attempt to create working Tetracyclin using a new improved stock of 100X Tet. A new Tetracycline stock was created. Previous stocks were made using 100% Ethanol under the assumption that we were working with pure Tetracyline. However, our solid stock is actually Tetracyline Hydrochloride. This compound has difficulty suspending in pure Ethanol. Using a 70% Ethanol 30% water solvent the Tetracyline Hydrochloride dissolved almost instantly. Previously no precautions were made to counter Tetracyline’s light sensitivity. This time we wrapped the stock in tin foil and were careful to avoid too much light exposure.
</P><P>
Digest &amp; gel analysis of yvgW 2:1 1, yvgY 6:1 3, GFP+ 2:1 50 BB parts
- yvgW 2:1 1 sample with expected mobility, yvgY 6:1 3 sample incompletely digested, GFP+ 2:1 50 sample not as expected
</P><P>8/6 Matt characterize Tet’s effect on cells when top spreading and in liquid culture. All of the 0x, .1x, and 1x Tet tubes grew to saturation while neither of the 10x Tet tubes grew. Further testing is necessary to determine if a concentration can be found where Tet resistant strains grow but strains without Tet resistance do not grow. All of the plates grew. However, 1X Tet with Tet resistant strain PSB1AT3 in DH5alpha did not grow to a lawn but showed many (&gt;100) small single colonies.
</P><P>8/7 Matt: Previously we determined that both Tet resistant and unresistant cells grow in 1x Tet liquid culture but die in 10x liquid culture. Our purpose is to determine a Tet concentration between these two extremes where Tet resistant culture lives and Tet unresistant culture does not. Cultures were cloudy for pSB1AT3 &amp; pUC19: 0X tet, 2X tet. Cultures were clear for pSB1AT3 &amp; pUC 19: 4X tet, 6X tet, 8X tet. OD600 for pSB1AT3 2X tet(10X dilution): 0.24 OD600 for pUC19 2X tet(10X dilution): 0.18 Not a significant OD600 difference to select from. Test range from 2X-4X tet next.
</P><P>8/10 Xing: Transformation of BB_spoVG RBS &amp; BB_mrgApromoterrbs
Failed transformation
Alyssa: Made Excel template for ligation calculations. Phosphorylated, annealed, and ligated purchased oligonucleotides:
mrgA, mntH, spoVG1, spoVG2
</P><P>Malinka: Phosphorylated and annealed mrgA promoter, mrgA RBS, mrgA promoter-RBS, mntH promoter and spoVG RBS
Matt: Previously we determined that both Tet resistant and unresistant cells grow in 1x Tet liquid culture but die in 10x liquid culture. Our purpose is to determine a Tet concentration between these two extremes where Tet resistant culture lives and Tet unresistant culture does not. Using strains: PSB1AT3 in DH5alpha and PUC19 in DH5alpha at concentration of 0X, 2.5X, 3X, 3.5x, and 4X Tet
(1X Tet= 50ug/ml, stock is at 100X) Culture in 5ml of LB, inoculate from picking plates. At 2.5X PSB1At3 DH5alpha grew but PUC19 DH5alpha did not. Above this concentration nothing grew.
</P><P>8/11 Malinka: : Transformed and plated mrgA promoter, mrgA RBS, mrgA promoter-RBS, mntH promoter and spoVG RBS (all on pSB1AC3).
Alyssa: Ligated and transformed oligos of mntH promoter and mrgA promoter. Transformed and plated RBS and mnth promoter biobricks in DH5 cells
</P><P>8/12 Xing: Test transformation protocol and competency of cells	
Cells competent and transformation worked with pUC19
Alyssa: CdCl2 growth curve plate design
</P><P>
8/13 Xing: Tranformation of mntH promoter, mrgA promoter, mrgA rbs, BBa_K143001, BBa_K143002
Successful transformation
Ben: Inoculated spoVG transformants for mini-prepping
Malinka: Cultured amyE parts and made amp plates.
Alyssa:
Redid CdCl2 growth curves redo the CdCl2 growth curve experiment from 7/18, which resulted in biphasic growth curves and growth of mntR when exposed to Cd.  (This should not have happened due to its extreme Cd sensitivity.)
--Results:
No errors  
111 runs
Biphasic growth curves for all strains, though not as pronounced as 1st growth curve exp.
mntR shows more growth than WT, even at 5uM Cd 
</P><P>8/14 Xing: Inoculate cultures from transformant plates for mntH promoter, mrgA promoter, mrgA rbs, BBa_K143001, BBa_K143002
</P><P>PAGE purification of mrgApromrbs
Alyssa:
Made new 1000X Tet stock and measured concentration of PSB1K3-- to make a more potent Tet stock soln. and measure concentration of a biobrick backbone for calculation purposes in other experiments and procedures (such as ligation).
Analyzed data from 8/13 growth curves on Excel
Cut and dephosphorylated psB1K3 backbone
</P><P>
8/15 Xing: Digest and gel analysis of spoVG, BBa_K143001, BBa_K143002
spoVG migrates with expected mobility, BBa_K143001 &amp; 2 do not migrate as expected
</P><P>Miniprep mntH, mrgA promoter, mrgA rbs, BBa_K143001, BBa_K143002
</P><P>8/19 Alyssa: Digestion of mntH, mnrA promoter, mrgA + mrgA rbs, and mrgA rbs
Matt: digested Biobricks in Preparation for 3A Ligation and Gels.
</P><P>
8/20 Alyssa:
Phosphorylization, annealing, and ligation of mntH, mrgA, spoVG, yvgW, and amyE locus biobricks-- to make biobrick constructs out of the oligos that we ordered
Performed ligation calculations for mrgA promoter, mntH promoter, spoVG, amyE locus, and yvgW promoter
</P><P>8/30 Malinka: Made Kan plates.
</P><P>8/31 Xing: Ligation of mntH/mrgArbs, mntH/spoVG, mrgAprom/mrgARBS, mrgA/spoVG, yvgW/mrgArbs, yvgW/spoVG, and transformation.
</P><P>9/1 Matt: Inoculated mntH/mrgArbs, mntH/spoVG, mrgAprom/mrgARBS, mrgA/spoVG, yvgW/mrgArbs, and yvgW/spoVG to prepare for a miniprep. The plates he inncoculated from: No L 50ul control looks almost completely clear. No L 250ul control has very small speckling on surface. Both 50ul and 250ul no I controls grew dozens of singles. PUC19 grew. H20 control is clear. All samples grew.
</P><P>
9/2 Xing: Miniprep mntH/mrgA rbs, mntH/spoVG, mrgA/mrgArbs, mrgA/spoVG, yvgW/mrgArbs, yvgW/spoVG
</P><P>9/3 Alyssa: Digested mntH/ mrgArbs, mntH/ spoVG rbs, YFP, and GFP
</P><P>
9/4 Xing: Ligation of mntH-mrgA/YFP, mntH-spoVG/YFP, mrgAprom-mrgArbs/GFP and transformation
</P><P>9/7 Xing: Miniprep cultures of mntH/spoVG/YFP, mntH/mrgArbs/YFP, mrgAprom/spoVG, yvgW/mrgArbs, yvgW/spoVG 
</P><P>9/8 Malinka: Transforming and plating promoter + RBS bricks (yvgW and mrgAprom with spoVG or mrgArbs).
Matt tested to see if backbones took up desired sequences (mntH/mrgArbs, mntH/spoVG, mrgAprom/mrgARBS, mrgA/spoVG, yvgW/mrgArbs) by running a gel. Plasmids cut with E and P show slightly smaller bands than those cut with just E on both gels. Wells 6 and 7 corresponding to mrgApmrgA are very light.
</P><P>
9/9 Xing: Send constructs for sequencing
mrgA promoter, mrgA rbs proper sequence. mntH, yvgW, spoVG inserted inverted into plasmid.
</P><P>9/11: Miniprep and digestion of yvgW and mrgA constructs
</P><P>9/13 Xing: Digestion and Gel Analysis of mntH, yvgW, spoVG parts 
spoVG 6:1 2,3 migrate as expected, send for sequencing
Send spoVG 6:1 2,3 for sequencing
Both samples proper sequence
</P><P>9/15: Alyssa Miniprepped 9/14 inoculated cultures—yvgW, mntH, spoVG
</P><P>
9/16 Xing: Digestion and Gel Analysis of mntH, yvgW
No samples appear linearized
Matt prepared competent DH5alpha E Coli cells. Grew two 50ml cultures to .35 OD600 and .37 OD600.
Pellets re-suspended fine. About 80-85 100ul aliquots of competent cells were stored in -80C.
</P><P>9/21 Xing: Ligation of mntH, yvgW inserts into pSB1AC3 backbones
pUC19 control shows colonies, all other plates blank
</P><P>9/22 Matt checked to see that the newly transformed plates from 9/21 did not grow, so he inncoulated only from the old 8/13 and 7/22 plates. Taking from colonies that do not appear to have been sampled: 9 colonies from yvgW 6:1 buc 7/22 3 colonies from mntH 6:1 buc 8/13 3 colonies from mntH 6:1 buc 8/13 3 colonies from mntH 6:1 buc 8/13
</P><P>
9/24 Malinka: : Digestion (with X) and Gel Analysis of mntH, yvgW parts to determine which promoter biobricks were good and should be sequenced.
Alyssa: Miniprepped mntH, yvgW
</P><P>9/25 Malinka: : Preparing mntH, yvgW biobricks for sequencing.
</P><P>9/28 Xing: Digestion and Ligation of mntH/mrgArbs, mntH/spoVG, mrgAprom/mrgARBS, mrgA/spoVG, yvgW/mrgArbs, yvgW/spoVG, and transformation.
</P><P>9/29 Malinka: Culturing (mntH and yvgW) + RBS (spoVG) biobricks.
</P><P>9/30 Xing: Preparing promoter rbs constructs for sequencing
yvgW/spoVG 1 9/30 displays proper sequence
</P><P>10/3 Ben
Digested yvgW/spoVG, CFP, and pSB1AC3 (rfp insert)
Ligated yvgW/spoVG to CFP in pSB1AC3 following Ginkgo Bioworks protocol
</P><P>10/6 Matt innoculated from 8/31yvgW/mrgArbs and 10/5 yvgW/spovG/cfp. 10/5 yvgW/spovG/cfp grew on chlor all colonies white many colonies. The chlor controls were clear.
</P><P>
10/10 Malinka: Transforming and plating yvgW/spoVG/GFP biobricks and yvgW/spoVG biobricks	
Alyssa: 
Digested yvgW/spoVG, GFP, PSB1AC3
Ligated promoter/rbs and GFP to backbone
</P><P>10/11 Matt streaked 2 plates of B Subtillis Wild Type (CU1065) from frozen permanent stock to grow up for transformation. Nothing grew on either plate, the permanents must be bad. 
</P><P>10/12 Malinka: 3 way ligation to construct mrgAprom/mrgArbs parts. Re-digesting plasmids to construct yvgW/spoVG/GFP parts
Alyssa: Plated mrgA/ mrgA rbs on Kan plates
</P><P>
10/16 Xing: Preparation of Kan, Chlor, LB plates
</P><P>
10/17 Xing: Preparing samples for registry
Ben
Digested yvgW/spoVG, GFP, and pSB1AC3 (rfp insert)
Ligated yvgW/spoVG to GFP in pSB1AC3 following Ginkgo Bioworks protocol
Malinka: Preparing samples for registry.  Re-transforming and plating yvgW/spoVG/GFP
</P><P>10/20: Alyssa :
Miniprepped yvgW/spoVG/GFP on Chlor—pink pellets, so did not miniprep in full to obtain DNA
Inoculated six tubes of colonies plated on 10/17—yvgW/spoVG/GFP on chlor plate.
</P><DIV class="printfooter">
Retrieved from &quot;<A href="http://2009.igem.org/Team:Cornell/Notebook">http://2009.igem.org/Team:Cornell/Notebook</A>&quot;</DIV></DIV></DIV><DIV id="footer-box"><DIV id="footer"><UL id="f-list"><LI id="t-recentchanges"><A href="/Special:RecentChanges" title="Recent changes">Recent changes</A></LI><LI id="t-whatlinkshere"><A href="/Special:WhatLinksHere/Team:Cornell/Notebook" title="List of all wiki pages that link here [j]" accesskey="j">What links here</A></LI><LI id="t-recentchangeslinked"><A href="/Special:RecentChangesLinked/Team:Cornell/Notebook" title="Recent changes in pages linked from this page [k]" accesskey="k">Related changes</A></LI><LI id="t-specialpages"><A href="/Special:SpecialPages" title="List of all special pages [q]" accesskey="q">Special pages</A></LI><LI><A href="/Special:Preferences">My preferences</A></LI></UL></DIV><DIV id="footer"><UL id="f-list"><LI id="t-print"><A href="/wiki/index.php?title=Team:Cornell/Notebook&amp;printable=yes" title="Printable version of this page [p]" accesskey="p">Printable version</A></LI><LI id="t-permalink"><A href="/wiki/index.php?title=Team:Cornell/Notebook&amp;oldid=169141" title="Permanent link to this revision of the page">Permanent link</A></LI><LI id="privacy"><A href="/2009.igem.org:Privacy_policy" title="2009.igem.org:Privacy policy">Privacy policy</A></LI><LI id="disclaimer"><A href="/2009.igem.org:General_disclaimer" title="2009.igem.org:General disclaimer">Disclaimers</A></LI></UL></DIV></DIV></DIV></BODY></HTML>