"
Page
Discussion
View source
History
teams
Log in
 
Team:Valencia UPV/Project/results/constructs
From 2014.igem.org
Home
Team
Students
Advisors and Instructors
Official Team Profile
Attributions
Sponsors
Project
Overview
Modules
Results
Notebook
Modeling
Overview
Pheromone Production
Pheromone Diffusion and Moth Response
Safety
Policy and Practices
Overview
Activities
Outreach
News & Press
Contact Us
Interlab Study
Achievements
Medal Requirements
Parts
Achievements
Collaborations
Project > Results > Constructs
C
onstructs
Contents:
RESULTS–CONSTRUCTS-BIOSYNTHESIS
RESULTS-CONSTRUCTS-PHEROMONE RELEASE
RESULTS - CONSTRUCTS-SWITCH
RESULTS-CONSTRUCTS-BIOSAFETY
RESULTS–CONSTRUCTS-BIOSYNTHESIS
In order to engineer the insect sexual pheromone pathway in our Sexy plant, we had to isolate four genes from different organisms: a desaturase (AtrΔ11), a reductase (HarFAR), an acetyltransferase (EaDAcT) and finally an alcohol oxidase (FAO). As they were coming from very different and not easily accessible organisms (two moths and a plant from Asia), the coding sequences (CDS) of the first three enzymes were obtained by gene synthesis (Integrated DNA Technologies, IDT) after codon usage optimization for N. benthamiana. As for the fourth one (FAO) we tried to amplify it from the genomic DNA of the yeast Candida tropicalis, with no successful results. Nevertheless, the three synthetic genes were sufficient to produce at least two of our target pheromones, the alcohol (Z)-11-hexadecen-1-ol and the acetate (Z)-11-hexadecenyl acetate.
Figure 1. Engineered pheromone production pathway.
All three DNA sequences were domesticated; that is, standardized as GoldenBraid parts and cloned in the pUPD plasmid. Subsequently, each CDS was assembled with the strong constitutive Cauliflower mosaic virus promoter (P35S) and its terminator (T35S) respectively, in a multipartite assembly reaction. P35S is a strong plant constitutive promoter with high expression levels.  As result of these assemblies, we obtained three Transcriptional units (TU) ready for plant transformation.
To maximize the flow through of the pathway, we wanted to make sure that all three genes were co-delivered simultaneously. This is to ensure that each transformed cell receives a complete set of genes and that the expression of all three enzymes is balanced and coordinated. Co-delivery is achieved by creating a multigenic construct, where all three genes are assembled in a single plasmid.
We used the GoldenBraid assembly system to create the multigene assembly. After two binary reactions, the 3-genes construct was obtained. This construct was then transformed, by agroinfiltration, into N. benthamiana plants for pheromone production.
We used GoldenBraid 2.0 for all our cloning reactions. GoldenBraid and BioBrick parts are not directly exchangeable; however, we adapted the coding sequences of the three biosynthetic genes to BioBrick standards using a GoldenBraid-Biobricks translator developed by the NRP-UEA-Norwich team. These BioBricks have been submitted to the Parts Registry.
RESULTS-CONSTRUCTS-PHEROMONE RELEASE
To obtain a plant capable of releasing pheromones into the environment in an efficient way, we decided to use the glandular trichome specific promoter (PCPS2) ( see Pheromone Relase ).
We obtained this promoter from Nicotiana tabacum genome and tested its functionality assembling it with GFP ( see Results: Pheromone Release ).
As it proved to be an effective promoter, expressing GFP only at the glandular cells of the trichomes, we decided to assemble each of the pheromone biosynthetic genes with this promoter to reach improvements in the pheromones release ( see Biosynthesis ).
Finally, after two binary GoldenBraid assembly steps, we obtained a multigenic construct with all three transcriptional units with the PCPS2 promoter. This construct was ready to be transferred to the plant to test the release of the pheromones.
RESULTS - CONSTRUCTS-SWITCH
In order to have a tight control over the pheromone production in the plant, we implemented an inducible switch in the plant genetic circuit. This switch is induced with the presence of copper ions, activating the pheromone production pathway in the plant only when the insects mating season arrives. With this kind of inducible expression, the pheromones production metabolic cost for the plant will be reduced and the pheromone release will be always under control ( see Switch ).
This switch is composed of two different parts:
A constituvely expressed transcription factor (CUP2) that changes its conformation in the presence of a given concentration of cupper ions. With this change of conformation, it can bind a specific promoter and initiates the transcription of a  particular gene.
A chimeric promoter that includes an Upstream Activating Sequence (UAS) whith the transcription factor binding site.
This switch is naturally present in yeast, so it must be adapted to plants.
Therefore, the transcription factor CUP2, which was obtained from S. cerevisae genome, was assembled with de Cauliflower mosaic virus (CaMV) P35S promoter and T35S terminator to be constitutively expressed. In addition, an activator domain (GAL4 AD) was joined to the CUP2 sequence, in order to improve the transcription initiation.
On the other hand, we needed to assemble the chimeric promoter and the gene of interest. This chimeric promoter is composed of 3 different parts:
The aforementioned UAS, which is a 44bp region. Only 16bp of them are actually the binding site for CUP2 while the rest are spacer nucleotides.
A minimal promoter, miniP35S (-60). These are a reduced number of nucleotides from the CaMV P35S promoter required for starting transcription.
The 5'-UTR region of the tobacco mosaic virus (TMV), called omega sequence. This sequence functions as a translational enhancer in plants.
As we were going to perform a Luciferase assay to test the switch ( see Luciferase assay ) the gene of interest assembled with the chimeric promoter was the Luciferase. This is the obtained construct.
In order to have a control for the Luciferase assay, we needed another construct already available in the GoldenBraid collection. It included the Renilla and P19 genes.
Finally, we assembled all three constructs into a single multigenic construct that was introduced into the plant to perform the Luciferase assay.
RESULTS-CONSTRUCTS-BIOSAFETY
In order to obtain a safe plant we developed a biosafety module which avoids spread of the plant’s genetic material and allows easy identification of the modified plant.
In order to avoid spread of genetic material, we built a plasmid which constitutively expressed Barnase (an RNAse from Bacillus amyloliquefaciens) under the regulation of the tapetum specific promoter TA29. Barnase effect under the regulation of this promoter can’t be tested under the limitations of transient expression and time restrictions didn’t allow us to create the stable plant, but its function is well documented .
We worked together with NRP-UEA-Norwich team to introduce their chromoproteins (Yellow and Blue) in our system to be used as a strategy of identity preservation. We received their complete transcriptional units and tested them in plant. However, their activity was not strong enough to be detected by naked eye.
The complete biosafety module, containing chromoproteins as identity preservation agent, had already been constructed in biobricks format. Expression of Barnase specifically in tapetum cells and an identity preservation agent completes our biosafety module.
Since the provided chromoproteins did not show enough change in the plant’s colour to be detected by the naked eye, we tested an alternative strategy. We tested an anthocyanin production enhancing genetic construction in N.benthamiana. Anthocyanins are coloured compounds found in plants.
We obtained this construction from GoldenBraid 2.0 collection. This construction expresses two transcription factors (Ant1 and JAF13) and successfully enhanced the synthesis of anthocyanins. Therefore, we propose an identity preservation strategy based in anthocyanin accumulation as part of our biosafety module.
← Go to Methodology Go to Results Go to Trichome-Specific Expression →
Sitemap | Twitter | Facebook | Email
This wiki is designed and constructed by Valencia_UPV.
Retrieved from " http://2014.igem.org/Team:Valencia_UPV/Project/results/constructs "
Recent changes
What links here
Related changes
Special pages
My preferences
Printable version
Permanent link
Privacy policy
Disclaimers
