"
Page
Discussion
View source
History
teams
Log in
 
Team:Valencia UPV/Project/modules/methodology/gb
From 2014.igem.org
Home
Team
Students
Advisors and Instructors
Official Team Profile
Attributions
Sponsors
Project
Overview
Modules
Results
Notebook
Modeling
Overview
Pheromone Production
Pheromone Diffusion and Moth Response
Safety
Policy and Practices
Overview
Activities
Outreach
News & Press
Contact Us
Interlab Study
Achievements
Medal Requirements
Parts
Achievements
Collaborations
Project > Modules > Methodology > GoldenBraid
T
he
G
olden
B
raid
c
loning
s
trategy
Our Sexy Plant is a challenging project for many reasons; a very important one is that we use plants as chassis for engineering. Plants have eukaryotic gene structure, make use of plant-specific regulatory regions and require special T-vectors for transformation, among other special features. Consequently, DNA repositories and DNA assembly standards need certain adaptations to facilitate engineering using plant chassis. Without letting aside BioBricks, we decided to use the GoldenBraid system (GB) to build several of the intermediate genetic constructs employed in this project. GB is a DNA assembly system specially conceived to facilitate genetic engineering in Plant Synthetic Biology projects (visit gbcloning.org for more information).
As BioBricks, GB is a modular cloning strategy that allows the fabrication of new devices by the combination of prefabricated standard modules. A difference between both strategies is that BioBricks is based on type II enzymes and GB relies on the use of type IIS restriction enzymes.
Type IIS restriction enzymes, unlike type II enzymes; cleave DNA at a defined distance from their recognition sites, not requiring any specific sequence in the cleavage site. Since there are no sequence requirements in the cleavage sites, these can be defined by the user and adapted to serve as standard fusion sites to DNA parts. The enzymes used in GoldenBraid are BsaI and BsmBI, which cut out from their binding sites generating 4 base overhangs.
Figure 1. Comparison between type II and type IIS restriction enzymes
GoldenBraid step by step
1. GB Domestication
The first step in the GB cloning strategy is the adaptation of the DNA sequence to the GB standard. This process is called domestication and implies (1) the removal of internal restriction sites for the enzymes used in GB (BsaI, BsmBI and BtgZI) and (2) the addition of appropriate 4-nt flanking overhangs to convert the DNA sequence into a standard part (Gbpart). Gbparts are the minimal standard building blocks and they are classified in different categories according to their specific function.
There are three basic categories that define the most common parts making up a transcriptional unit. These part categories are PROM (GGAG-AATG), CDS (AATG-GCTT) and TER (GCTT-CGCT) and were the most used on this project.
Figure 2. Part categories of a basic GoldenBraid trancriptional unit. Promoter’s (PROM) prefix is GGAG and its suffix is AATG, which is the same as the coding region’s (CDS) prefix. The same happens with the CDS and the terminator (TER), which share the part identity overhang GCTT, the first one as its suffix and the second one as its prefix.
The mutagenesis procedure required to remove internal restriction sites is standardized and involves the amplification of the target DNA in separated fragments (GBpatches) using GB-adapted primers, which incorporate single mismatches to disrupt the enzyme target sites. Once amplified, GBpatches are reassembled together in a single-tube BsmBI restriction-ligation reaction into the universal entry vector (pUPD) to yield a domesticated GBpart
Figure 3. Domestication strategy with removal of internal restriction sites. Internal Type IIs recognition sites (exemplified here with the GGTCTC BsaI recognition site) are mutagenized during domestication following a standard procedure. In addition to the GB.F and GB.R primers that amplify the whole fragment, two other primers (M.F and M.R) are required for mutagenesis, which incorporate the flanking BsmBI overhangs and the single nucleotide change (C>M). Each primer pair is used to amplify a GBpatch by PCR, and the resulting fragments are assembled together in a BsmBI restriction-ligation reaction into pUPD. The resulting GBpart is free of internal recognition sites and can be released from pUPD using BsaI or BtgZI.
2. GB Multipartite assemblies
Domesticated GBparts can now be assembled together in a one-tube-one-step reaction to create a Transcriptional Unit (TU). GB uses the Golden Gate multipartite reaction to create transcriptional units (TU). By using special GB destination vectors in the reaction, we make sure that the resulting TUs can be subsequently used to build multigene constructs (constructs comprising several TUs within the same destination plasmid). GB destination vectors are T-plasmids, a special type of plasmids used for plant transformation. Therefore the new TUs assembled in GB vectors can be directly transferred into plants using Agrobacterium-mediated plant transformation.
Figure 4. Schema of a GoldenGate reaction. All the DNA parts composing a basic structure (PROM, CDS and TER) are mixed together in one tube with a GB destination plasmid, BsaI and T4 ligase. As result of the restriction-ligation reaction the correctly assembled transcriptional unit is obtained.
3. GB binary assemblies
Multigene constructs are important in Plant Synthetic Biology because they allow several TUs to be jointly transferred to the plant genome.  Co-transformed TUs are expressed coordinately and eventually inherited together, avoiding lengthy breeding strategies.
GB-made TUs can be combined using GB binary assemblies to create increasingly complex multigenic structures. The special orientation and disposition of the restriction sites in the GB destination vectors defines two levels of destination plasmids: the α plasmids used for BsaI reactions and the Ω plasmids, which are used for BsmBI GB reactions.
The double loop topology of the GB cloning strategy allows an indefinite growth of the constructs through iterative binary assembly steps. By choosing the appropriate combinations of expression and destination vectors, increasingly complex multigenic modules are created.
We used this strategy to build and transform in N. benthamiana a multigene construct encoding the last 3 enzymatic steps of the synthesis of a moth pheromone.
Figure 5. Schema of a GoldenBraid reaction. Two TUs assembled in complementary α-level plasmids are combined in a Ω-level destination vector to create a two genes module.
← Go to Switch Go to Methodology Go to Flowchart →
References
Engler C, Gruetzner R, Kandzia R, Marillonnet S (2009) Golden gate shuffling: a one-pot DNA shuffling method based on type IIs restriction enzymes. PLoS One 4 (5):e5553. doi:10.1371/journal.pone.0005553
Sarrion-Perdigones, A., Falconi, E.E., Zandalinas, S.I., Juarez, P., Fernandez-Del-Carmen, A., Granell, A. and Orzaez, D. (2011)  GoldenBraid: An Iterative Cloning System for Standardized Assembly of Reusable Genetic Modules. PLoS One 6, e21622
Sarrion-Perdigones A, Vazquez-Vilar M, Palaci J, Castelijns B, Forment J, Ziarsolo P, Blanca J, Granell A, Orzaez D (2013) GoldenBraid2.0: A comprehensive DNA assembly framework for Plant Synthetic Biology. Plant Physiol
Sitemap | Twitter | Facebook | Email
This wiki is designed and constructed by Valencia_UPV.
Retrieved from " http://2014.igem.org/Team:Valencia_UPV/Project/modules/methodology/gb "
Recent changes
What links here
Related changes
Special pages
My preferences
Printable version
Permanent link
Privacy policy
Disclaimers
