"
Page
Discussion
View source
History
teams
Log in
 
Team:Valencia UPV/prueba
From 2014.igem.org
Home
Team
Students
Advisors and Instructors
Official Team Profile
Attributions
Sponsors
Project
Overview
Modules
Results
Notebook
Modeling
Overview
Pheromone Production
Pheromone Diffusion and Moth Response
Safety
Policy and Practices
Overview
Activities
Outreach
News & Press
Contact Us
Interlab Study
Achievements
Medal Requirements
Parts
Achievements
Collaborations
Contents:
Biosynthesis under constitutive promoter
Expression in trichomes
Biosafety module
Measurement Interlab Study
Switch
Translator to BioBricks
Biosynthesis under constitutive promoter
06/09/2014
The enzymes involved in the biosynthesis pathways are AtrΔ11, HarFAR, FAO1, EaDAcT.
The design of the GBlocks was performed taking into account the following considerations:
Codon optimization
Inner restriction sites eliminations by finding synonymous mutations
Addition of GB endings
06/10/2014
Codes for IDT known. MEGAGEM2014 - 25% off one order, up to 800 USD
GBlocks designed to be compatible with BioBricks and GoldenBraid (GB).
06/11/2014
We ordered the following gBlocks and primers.
EaDAcT: Eunymus alatus (adapted for GB) 1127 bp
HarFAR: Helicoverpa armigera (adapted for GB) 1400 bp
AtrΔ11: Amyelois transitella (order primers for GB) 1000 bp
I14Jun03 AtrΔ11 F1
I14Jun04 AtrΔ11 R1
FAO1: N. benthamiana primers
I14Jun01 FAO1 F1
I14Jun02 FAO1 R1
Name
Sequence
Lenght
Tm (NTI)
Tm (Phusion)
I14Jun01_FAO1_F1
cgccgtctcgctcgaatggagaaaaagagccatcc
35
49.9
62.4
I14Jun02_FAO1_R1
cgccgtctcgctcgaagcttatcttgagaatttgccttcttttatc
46
54.5
63.7
I14Jun03Atr_D11_F1
gcgccgtctcgctcgaatggttcctaataag
31
54.5
65.3
I14Jun04Atr_D11_R1
gcgccgtctcgctcgaagctcaacgtttc
29
57
69.1
06/24/2014
We thought which parts of the GB collection could we use.
Strategy 1:
pP35S, pT35s (x2)
pAtUbq10, pTAtHSP18.2
Strategy 2:
pP35S, pT35s
pP35s, pTAtHSP18.2
pAtUbq10, pTAtHSP18.2
Strategy 3:
pP35S, pT35s
pP35s, pTTctp
pAtUbq10, pTAtHSP18.2
Pieces to take from GB2.0 colection:
pDGB2α1
GB0483
Kan
pDGB2α2
GB0484
Kan
pP35s
GB0030
Amp
pT35s
GB0036
Amp
pAtUbq10
GB0223
Amp
pTAtHSP18.2
GB0035
Amp
pTTctp
GB0081
Amp
pUPD
GB0317
Amp
Later we will need:
pDGB2Ω1
GB0487
Smp
pDGB2Ω2
GB0488
Smp
Prepare plaques with antibiotics Kan, Spm, Amp
06/25/2014
Grow the selected pieces from the GB collection in liquid medium (performed in laminar air flow cabinet).
06/26/2014
Culture in agar Petri dish. 2 plaques: Amp and Kan.
Minipreps with EZNA Plasmid DNA MiniKit I.
Expected digestions:
pP35s
GB0030
NotI
Buffer: Orange
2981, 1105
pT35s
GB0036
NotI
Buffer: Orange
2981, 304
pAtUbq10
GB0223
NotI
Buffer: Orange
2981, 714
pTAtHSP18.2
GB0035
NotI
Buffer: Orange
2981, 328
pTTctp
GB0081
NotI
Buffer: Orange
2981, 487
Electrophoresis analysis.
We got the expected bands in all cases.
07/01/2014
AtrΔ11 amplification by PCR with primers that contain extra nucleotides to introduce them in the sequence.
We made a PCR amplification using the AtrΔ11 gene as a template and the oligos: R +F
Reagents needed:
32.5 μL of H2O miliQ
10 μL HF buffer
2 μL dNTPs
2.5 μL Reverse primer
2.5 μL Forward primer
1 μL template (AtrΔ11 gene)
0.5 μL phusion (polimerase)
PCR parameters: The annealing temperature was 60°C and the extension temperature was 65°C.
Electrophoresis performed to check the PCR product, which was expected to be around 1 kb.
pUPD ligation of EaDAcT, HarFar and AtrΔ11.
Reagents needed for the reaction of ligation:
1 μL pUPD
1 μL PCR product/gblock product
1.2 μL buffer 10x
1 μL BsmBI
1 μL T4 ligase
6.8 μL H2O miliQ
Vfinal= 12 μL
Termocycler parameters: The ligase temperature was 16°C and the BsmBI temperature was 37°C.
As a result, there are obtained three different pUPD plasmids containing the genes EaDAcT, HarFAR and AtrΔ11.
07/02/2014
E. coli transformation. This step is performed in a laminar air flow cabinet (LAF). We have used an electrocompetent E. coli strain (DH5α) and a sample from each product of ligation made in the previous step (three pUPD plasmids, each of them containing one of the three genes), so transformation is made three times.
Reagents needed:
E. coli aliquot
1.5 μL of ligation in pUPD (for each gene: EaDAcT, HarFAR, AtrΔ11)
Each mix is introduced in a electroporation vial and electroporated at 1500 V, then 300 μL of SOC are added to each vial. All of them were incubated at 37°C for 1 hour.
After incubation, culture in Petri plates (always in a LAF).
2 cell-culture dishes per transformation (with Ampicillin), one with 50 μL and the other with the remaining volume.
Petri plates are incubated at 37°C for 16 h.
07/03/2014
Transformed colonies selection. The white ones are recultured in liquid medium. One colony of each transformation is picked and cultured in 3.5 mL LB and 7 μL Amp. This step is repeated three times:
3x 1 colony of EaDAcT in pUPD + LB + Amp
3x 1 colony of HarFAR in pUPD + LB + Amp
3x 1 colony of AtrΔ11 in pUPD + LB + Amp
All tubes are incubated at 37°C overnight in agitation.
07/04/2014
Digestions in silico using Vector NTI to check after minipreps if ligations are correct.
EaDAcT
NotI
2981, 1167
RsaI
1876, 1343, 532, 306, 91
HarFAR
NotI
2981, 1440
PvuII
2564, 1394, 463
AtrΔ11
NotI
2981, 1056
BanII
2570, 803, 351, 314
Digestion reagents:
0.5 μL restriction enzyme
2.5 μL buffer
21 μL H20 (miliQ)
1 μL sample
Preparation of master mixes
Master mix for NotI
5 μL NotI
25 μL Orange
210 μL H20
Master mix for RsaI
1.5 μL RsaI
7.5 μL Tango
63 μL H20
Master mix for PvuII
1.5 μL PvuII
7.5 μL Green
63 μL H20
Master mix for BanII
1.5 μL BanII
7.5 μL Tango
63 μL H20
Perform electrophoresis to check if the size of the fragments from the digestions is correct.
Comments:
We picked blue colonies instead of white by mistake. We need to pick colonies again but this time make sure we pick white colonies.
For the repetition we must find another enzyme instead of BanII as we found out that it doesn't cut very well.
07/06/2014
We picked again 3 colonies for each construction, and we made sure that we picked the WHITE ones. We cultivated them in a "double check" (name invented by us)  liquid medium. Those tubes contain:
3.5 mL LB
7 μL Amp
7 μL X-Gal
3.5 μL IPTG (turns the tube blue if the colonies picked were blue)
07/07/2014
We made minipreps of yesterday's culture. Thanks to our "double check" medium we found which colonies were well picked. Finally we had minipreps of tubes HarFAR 1, 2, 3; EaDAcT 3 and AtrΔ11 2, 3.
Once we had the minipreps, we perform the digestions to check which were correct and send them to sequencing. This time we selected RsaI instead of BanII. The in silico digestions were as follows.
EaDAcT
NotI
2981, 1167
RsaI
1876, 1343, 532, 306, 91
HarFAR
NotI
2981, 1440
PvuII
2564, 1394, 463
AtrΔ11
NotI
2981, 1056
RsaI
1879, 1310, 467, 327, 54
Preparation of master mixes
Master mix for NotI
3.5 μL NotI
17.5 μL Orange
147 μL H20
Master mix for RsaI
2 μL RsaI
10 μL Tango
84 μL H20
Master mix for PvuII
2 μL PvuII
10 μL Green
84 μL H20
We run the electrophoresis gel to check if this time we have done it correctly.
Everything was OK. We sent AtrΔ11 (3), HarFAR (3) and EaDAcT (3) to sequence.
07/08/2014
Now, while we wait for sequencing results, we go on as they were going to be correct in order to save time.
The next step is to build a transciptional unit (TU) with our sequences. A transcriptional unit is a structure composed by promoter, coding sequence (CDS) and terminator in an α or Ω vector.
Reagents needed for ligation:
1 μL promoter 75 ng/μL
1 μL terminator 75 ng/μL
1 μL CDS 75 ng/μL
1 μL vector α
1.2 μL ligase buffer 10x
1 μL T4
1 μL BsaI
4.2 μL H20
Total: 12 μL
Take into account that if we want to make binary constructions later (merge 2 TU in a same vector), we need to clone each TU in a different α vector.
Strategy Promoter-Terminator:
AtrΔ11
P35s
T35s
40.41
HarFAR
P35s
TatHSP
39.68
EaDAcT
PAtUbq
TatHSP
32.27
Adjust concentrations to 75 ng/μL for ligation reaction
Initial concentrations (nanodrop):
Piece
Concentrations
Volume
Volume of H20 to add
PAtUpb
442.6 ng/μL
34 μL
166.6 μL
pTatHSP
235.4 ng/μL
36 μL
77 μL
T35s
194.9 ng/μL
37.5 μL
60 μL
P35s
454.7 ng/μL
36 μL
182 μL
2α1
57.1 ng/μL
-
We will need to put 1.5 μL of this one
2α2
104.0 ng/μL
38 μL
14.7 μL
AtrΔ11
359.3 ng/μL
20 μL
75.8 μL
HarFAR
404.4 ng/μL
15 μL
65.9 μL
EaDAcT
155.6 ng/μL
10 μL
10.7 μL
Ligation reaction
P35s:AtrΔ11:T35s in 2α1
1 μL P35s
1 μL T35s
1 μL AtrΔ11
1.5 μL 2α1
1.2 μL ligase buffer 10x
1 μL T4
1 μL BsaI
3.7 μL H20
P35s:HarFAR:TatHSP in 2α2
1 μL P35s
1 μL TatHSP
1 μL HarFAR
1 μL 2α2
1.2 μL ligase buffer 10x
1 μL T4
1 μL BsaI
4.2 μL H20
PAtUbq:EaDAcT:TatHSP in 2α2
1 μL PAtUbq
1 μL TatHSP
1 μL EaDAcT
1 μL 2α2
1.2 μL ligase buffer 10x
1 μL T4
1 μL BsaI
4.2 μL H20
07/09/2014
Transformation of constructions in E. coli
We finally got the sequencing results from 07/07/2014.
Mutation in AtrΔ11 -> We threw away the colonies and transformed cells. We picked again white colonies.
HarFAR -> Sequencing correct
EaDAcT -> Synonim mutation in 601 (A -> T). This is a gBlock!
We took vectors 2Ω1 (GB0487) and 2Ω2 (GB0488) parts from the GB colection.
Worked in the LAF
Cultivated in a Petri dish with Spm
Let them grow for one day
Cultivate transformated cells in two Kan plaques (Kan matches vector α)
50 mL transformation in one plaque
Rest of the culture in another (250 μL aprox)
Let them grow for one day
07/10/2014
Pick colonies and grow them in liquid medium.
6 from AtrΔ11 (repetition because of mutation)
3.5 mL LB
7 μL Amp
7 μL X-gal
3.5 μL IPTG
1 colony from 2Ω1
3.5 mL LB
7 μL Spm
1 colony from 2Ω2
3.5 mL LB
7 μL Spm
3 colonies from P35s:HarFAR:TatHSP
3.5 mL LB
7 μL Kan
3 colonies from PAtUbq:EaDAcT:TatHSP
3.5 mL LB
7 μL Kan
Grow at 37°C in agitation overnight.
We have checked the promoters and terminators are both compatible with GB and BioBricks.
Only P35s and T35s work for both. pPnos could also work.
Pick 3 colonies of P35s:HarFAR:THsp and PAtUbq:EaDAcT:THsp. Culture in liquid medium with Kan.
07/11/2014
We made minipreps of yesterday's liquid culture. Thanks to our "double check" medium we found which colonies were well picked. Finally we had minipreps of tubes AtrΔ11 3 and 6; 2Ω1; 2Ω2; constructions P35s:HarFAR:TatHSP 1, 2, 3 and PAtUbq:EaDAcT:TatHSP 1, 2, 3.
Additionally, we have cultured each of the colonies named above to store them.
07/14/2014
We tested the minipreps made last friday by digestion. Once they were checked, we send the correct ones to sequencing. The in silico digestions were as follows.
Parts
Retriction enzyme
Expected Bands
PAtUbq:EaDAcT:TatHSP in 2α2
HindIII
6322, 1722, 736, 221
P35s:HarFAR:TatHSP in 2 α2
HindIII
6322, 1794, 221
AtrΔ11
NotI
2961, 1056
2Ω1
BamHI
6652, 382, 239
2Ω2
EcoRV
6652, 621
Preparation of master mixes
Master mix for HindIII
3.5 μL HindIII
17.5 μL Red
147 μL H20
Master mix for NotI
1.5 μL NotI
7.5 μL Orange
63 μL H20
Mix for EcoRV
0.5 μL EcoRV
2.5 μL Red
21 μL H20
Mix for BamHI
0.5 μL PvuII
2.5 μL Green
21 μL H20
We run the electrophoresis gel to check if this time we have done it correctly.
Everything was OK except the AtrΔ11 (3), which had some partial digestion. It was the reason we sent AtrΔ11 (6) to sequence.
07/15/2014
We got the sequencing results from yesterday and everything was OK, so we made the transcriptional units ligation.
Reagents needed for the reaction of ligation (Total volume = 12 μL):
P35s:AtrΔ11:T35s in 2α1
1 μL P35s
1 μL T35s
1 μL AtrΔ11
1.5 μL 2α1
1.2 μL ligase buffer 10x
1 μL T4
1 μL BsaI
3.7 μL H20
P35s:HarFAR:T35s in 2α2
1 μL P35s
1 μL T35s
1 μL HarFAR
1 μL 2α2
1.2 μL ligase buffer 10x
1 μL T4
1 μL BsaI
4.2 μL H20
P35s:EaDAcT:T35s in 2α2
1 μL P35s
1 μL T35s
1 μL EaDAcT
1 μL 2α2
1.2 μL ligase buffer 10x
1 μL T4
1 μL BsaI
4.2 μL H20
Note: Concentrations were previously adjusted to 75 ng/μL. Only the AtrΔ11 was adjusted from 250.2 ng/μL.
Finally, we prepared liquid cultures in order to store in glicerol. The tubes we used and their respective antibiotics were:
Amp
pAtrΔ11 (6)
pEaDAcT (3)
pHarFAR (3)
Kan
P35:HarFAR:TatHSP in 2α2 (3)
PPAtUbq:EaDAcT:TatHSP in 2apha2 (3)
07/16/2014
Storage in glycerol of the HarFAR (GB1018), AtrΔ11 (GB1019), EaDAcT (GB1020), P35s:HarFAR:TatHSP in 2α2 (GB1021) and PAtUbq:EaDAcT:TatHSP in 2α2 (GB1022). We made 3 tubes: one for us, one for the GB collenction and one for reserve.
The procedure is to mix 700 μL of culture with 300 μL of glycerol 50%, spin it and store it in the -80°C.
07/17/2014
Pick 3 colonies of P35s:AtrΔ11:T35s, P35s:HarFAR:T35s and P35s:EaDAcT:T35s. Culture in liquid medium with Kan.
Digestions in silico.
Transcriptional units
Restriction enzymes
Expected bands
P35s:AtrΔ11:T35s
EcoRI
6323, 2269
NcoI
390, 8202
P35s:HarFAR:T35s
HindIII
933, 6322, 1722
NcoI
8587, 390
P35s:EaDAcT:T35s
HindIII
6322, 2366
EcoRV
683, 8021
Preparation of reagents needed for genomic extraction of Candida tropicalis for FAO1.
07/18/2014
Mistake in P35s:AtrΔ11:T35s, P35s:HarFAR:T35s and P35s:EaDAcT:T35s minipreps. Repeat tomorrow.
07/19/2014
Minipreps of P35s:AtrΔ11:T35s, P35s:HarFAR:T35s and P35s:EaDAcT:T35s. Concentration measuments with nanodrop.
Transcriptional unit
DNA concentration
P35s:AtrΔ11:T35s (1)
164 ng/μL
P35s:AtrΔ11:T35s (2)
168 ng/μL
P35s:AtrΔ11:T35s (3)
147.4 ng/μL
P35s:HarFAR:T35s (1)
125.3 ng/μL
P35s:HarFAR:T35s (2)
114.5 ng/μL
P35s:HarFAR:T35s (3)
140.3 ng/μL
P35s:EaDAcT:T35s (1)
144.2 ng/μL
P35s:EaDAcT:T35s (2)
137.9 ng/μL
P35s:EaDAcT:T35s (3)
128.5 ng/μL
Stuffer fragment
135.5 ng/μL
2Ω1
196.8 ng/μL
2Ω2
175.4 ng/μL
Digestions of P35s:AtrΔ11:T35s, P35s:HarFAR:T35s and P35s:EaDAcT:T35s and gel electrophoresis to check if transciptional units have been assembled OK.
All digestions were correct except P35s:EaDAcT:T35s (2).
Ligation in Ω vectors.
P35s:AtrΔ11:T35s + P35s:HarFAR:T35s in 2Ω1
1 μL P35s:AtrΔ11:T35s (75 ng/μL)
1 μL P35s:HarFAR:T35s (75 ng/μL)
1 μL 2Ω1 (75 ng/μL)
1 μL BsmBI (5-10 ud)
1 μL T4 ligase (5-10 ud)
1 μL buffer ligase (3 ud)
4 μL H20
P35s:EaDAcT:T35s in 2Ω2
1 μL stuffer fragment (75 ng/μL)
1 μL P35s:EaDAcT:T35s (75 ng/μL)
1 μL 2Ω2 (75 ng/μL)
1 μL BsmBI (5-10 ud)
1 μL T4 ligase (5-10 ud)
1 μL buffer ligase (3 ud)
4 μL H20
Set the reaction: 25 cycles x (37°C 2 min, 16°C 5 min).
Omega vectors include a resistance to spectinomycin.
07/20/2014
Transform and grow in Petri dishes yesterday's ligations: P35S:AtrΔ11:T35S + P35S:HarFAR:T35S in 2Ω1 and P35S:EaDAcT:T35S in 2Ω2.
07/21/2014
Pick colonies of P35S:AtrΔ11:T35S + P35S:HarFAR:T35S in 2Ω1 (3) and P35S:EaDAcT:T35S in 2Ω2 (2).
07/22/2014
We made minipreps of yesterday's liquid culture. Selected tubes:
P35S:AtrΔ11:T35S + P35S:HarFAR:T35S in 2Ω1(Tubes 1, 2 and 3)
P35S:EaDAcT:T35S in 2Ω2 (Tubes 1 and 2)
Digestions in silico made to check the transcriptional units:
Transcriptional units
Restriction enzyme
Expected bands
P35S:AtrΔ11:T35S+P35S:HarFAR:T35S in 2Ω1
EcoRV
9307, 2251
BamHI
6652, 4906
P35S:EaDAcT:T35S in 2Ω2
EcoRV
6652, 1044, 817, 683
NcoI
8806, 390
Digestion master mixes:
Master mix for NotI
1.5 μL NotI
7.5 μL Orange buffer
63 μL H20
Master mix for NcoI
1.5 μL NcoI
7.5 μL Tango buffer
63 μL H20
Master mix for BamHI
2 μL BamHI
10 μL Green buffer
84 μL H20
Master mix for EcoRV
4 μL EcoRV
20 μL Red buffer
168 μL H20
Note: Trichome promoter digestion preparation included.
All digestions were correct except the transcriptional unit of EaDAcT in 2Ω2 (P35s:EaDAcT:T35S).
Miniprep quantification:
Piece
Tube
Concentration (ng/μL)
Volume (μL)
P35S:EaDAcT:T35S in 2Ω2
1
350.7
33
P35S:EaDAcT:T35S in 2Ω2
2
271.1
33
P35S:AtrΔ11:T35S + P35S:HarFAR:T35S in 2Ω1
1
306.3
31
P35S:AtrΔ11:T35S + P35S:HarFAR:T35S in 2Ω1
2
296.6
28
P35S:AtrΔ11:T35S + P35S:HarFAR:T35S in 2Ω1
3
246.0
33
All of the pieces named above were adjusted at 75 ng/μL.
Piece
Tube number
Final Volume (μL)
Volume to be added (μL)
P35S:EaDAcT:T35S in 2Ω2
1
154.30
121.3
P35S:EaDAcT:T35S in 2Ω2
2
119.30
86.30
P35S:AtrΔ11:T35S + P35S:HarFAR:T35S in 2Ω1
1
126.60
95.60
P35S:AtrΔ11:T35S + P35S:HarFAR:T35S in 2Ω1
2
110.70
82.70
P35S:AtrΔ11:T35S + P35S:HarFAR:T35S in 2Ω1
3
108.24
75.20
As the digestions of the transcriptional unit (TU) of EaDAcT were incorrect, we repeated the process from the ligation step.
We transformed the same TU in a E. coli competent strain (DH5α). Then, the transformants were cultured in LB media and Spm and stored at 37°C overnight.
Finally, in order to obtain the FAO1 gene, we want to extract the Candida tropicalis genome, so we have picked a colony of C. tropicalis. To check the extraction protocol, we used a yeast previously tested, Saccharomyces cerevisiae. We have cultured C. tropicalis in YPD media and S. cerevisiae in YPDA media at 28 °C (5 mL).
07/23/2014
Candida genome extraction
Saccharomyces cerevisiae is used as a control in order to see if we followed the protocol correctly. We aren't really sure if this protocol is going to work in Candida.
Cultures measured at 600 nm:
S. cerevisiae Abs = 1.07
C. tropicalis Abs = 0.39
S. cerevisiae is recultured with new media (1:2) because the previous media was saturated. 2.25 mL of YPD media were mixed with 2.25 mL of S. cerevisiae culture. The mix has to grow at 28 °C until the exponential phase is reached.
The absorbance was measured again:
S. cerevisiae Abs = 0.52
C. tropicalis Abs = 0.40
Buffers needed for the genome extraction were prepared freshly.The genome of both strains of yeast were extracted following the protocol:
Grow yeast in 2 or 5 mL YPD to exponential phase.
Collect cells in 1.5 mL eppendorf-cup (centrifugation 20 s, 6000 rpm).
Wash once with 1 mL sterile water.
Resuspend cells in 200 μL protoplast-buffer (100 mM Tris-HCl, pH 7.5, 10 mM EDTA, 1000 units Zymolyase/mL, 10 μL beta-mercaptoethanol/mL; prepare freshly!). Incubate at 37°C for 1-2 h and finally resuspend by turning the cups.
Add 200 μL of Lysis-Mix (0.2 M NaOH, 1% SDS) an mix carefully (Don't vortex!).
Incubate at 65 °C for 20 min and cool inmediately on ice.
Add 200 μL of 5 M KAc (pH 5.4) and mix carefully (Don't vortex!) and incubate 15 min on ice.
Centrifuge (13,000 rpm, 3 min) and transfer supernatant in a fresh cup.
Add 2 μL RNase A (10 mg/mL) and incubate at 37 °C for 30 min.
Add 600 μL isopropanol and mix carefully (Don't vortex!). Incubate at room temperature for 5 min ad centrifuge (13,000 rpm, 30 s).
Remove the supernatant and wash with 70% ethanol (10 min at room temperature).
Centrifuge (13,000 rpm, 30 s) and remove the supernatant.
Dry DNA pellet in a speed-vacuum (not longer than 3 min!) and resuspend in 50 μL TE-buffer.
Store chromosomal DNA at 4 °C (Don't freeze!). Concentration and quality can be checked in an agarose gel (loading 1/10 of the volume).
Genomic quantification:
Organism
Concentration
S. cerevisiae
72.2 ng/μL
C. tropicalis
1397.1 ng/μL
Electrophoresis made to check the extraction quality was correct.
We did not observe genomic from Candida because we used a very diluted sample. We will repeat the gel tomorrow.
We picked EaDAcT colonies.
07/24/2014
The genomic quality of both organisms (C. tropicalis and S. cerevisiae) was checked in an agarose gel again.
We got the Candida genome band, however, the Saccharomyces genome band was not present.
Additionally, minipreps of the liquid culture made yesterday were made and also recultured in solid agar plate.
Miniprep digestions are going to be done tomorrow.
07/25/2014
Digestions in silico made for checking yesterday's minipreps:
Pieces/TU
Restriction enzyme
Expected bands
P35S:EaDAcT:T35S in 2Ω2
EcoRV
6652, 1044, 817, 683
NcoI
8806, 390
Digestion master mixes:
Master mix for NotI
2 μL NotI
10 μL Orange buffer
84 μL H20
Master mix for NcoI
2 μL NcoI
10 μL Tango buffer
84 μL H20
Master mix for BglII
2 μL BglII
10 μL Orange buffer
84 μL H20
Master mix for EcoRV
1.5 μL EcoRV
7.5 μL Red buffer
63 μL H20
An agarose gel was made to check the transcriptional unit and the other pieces:
All pieces were correct except the TU corresponding to P35:EaDAcT:T35S.
07/28/2014
Once the Candida tropicalis genome DNA is obtained, the FAO1 gene can be amplified by PCR.
PCR reaction reagents:
FAO1-PCR1
Genomic 0.5 μL
Buffer HF (5X) 10.0 μL
dNTPs 2.0 μL
Oligo R (JUL06) 2.5 μL
Oligo F (JUL05) 2.5 μL
Phusion polymerase 0.5 μL
H2O 32.0 μL
FAO1-PCR2
Genomic 0.5 μL
Buffer HF (5X) 10.0 μL
dNTPs 2.0 μL
Oligo R (JUL08) 2.5 μL
Oligo F (JUL07) 2.5 μL
Phusion polymerase 0.5 μL
H2O 32.0 μL
Annealing temperatures
FAO1-PCR1: 59 °C
FAO1-PCR2: 64 °C
PCR products were checked using an electrophoresis. Expected bands:
FAO1-PCR1: 1157 bp
FAO1-PCR2: 1015 bp
Both FAO1 PCR products were not correct.
As the EaDAcT TU was not correct, ligation reaction was done again. The following table shows ligation details:
Reagent
Volume
Trichome promoter
1 μL
GFP
1 μL
TNos
1 μL
BsaI
1 μL
p2α2
1 μL
T4 ligase
1 μL
Ligase buffer
1 μL
H2O
3 μL
Total Volume
10 μL
07/29/2014
As the FAO1 PCR was not correct, we repeated the reaction. Below is a table showing the details:
Reagent
FAO1-PCR1
FAO1-PCR2
C. tropicalis genome
2.5 μL
2.5 μL
HF Buffer
30 μL
30 μL
dNTPs
10 μL
10 μL
Oligo R
12.5 μL
12.5 μL
Oligo F
12.5 μL
12.5 μL
Phusion polymerase
1.5 μL
1.5 μL
H2O
181 μL
181 μL
PCR temperatures, 25 cycles:
Step
Temperature (°C)
Time
Initialization
98
2 min
Denaturation
98
20 s
Annealing
50, 55, 60, 65
??
Extension
72
45 s
Final elongation
72
7 min
We made a mistake preparing the FAO1-PCR1 adding the wrong template, so we do not expect the correct FAO11-PCR1 product.
EaDAcT Transcriptional Unit (TU) transformation
Using an electrocompetent E. coli strain (DH5α) and 1.5 ul ligation (P35s:EaDAcT:T35s in 2Ω2), the mix is electroporated at 1500 V. Then, 300 μL of SOC are added and stored at 37°C with agitation.
07/30/2014
Transform P35s:AtrΔ11:T35+P35s:HarFAR:T35 and P35s:EaDAcT:T35s (in 2α2) in Agrobacterium tumefaciens strain C58. Introduce 1 μL of construction in a C58 aliquot. Electroporate at 1440V. Add 500 μL of LB in the LAF. Keep 2 hours in agitation at 28°C. Grow 20 μL and 200 μL in solid medium containing kanamicin and rifampicin. Incubate overnight at 28°C.
Pick colonies of P35s:EaDAcT:T35s in 2Ω2.
08/01/2014
Pick colonies from Agrobacterium tumefaciens and grow them in liquid medium for two days at 28°C. Liquid medium is composed by 5 mL LB, Rif (1:1000) and Kan (1:1000) for α vectors and 5 mL LB, Rif (1:1000) and Spm (1:1000) for Ω vectors.
07/31/2014
Minipreps of yesterday's culture were made, obtaining the transcripional unit: P35S:EaDAcT:T35S in 2Ω2
Additionally, we recultured in petri dish with its respective antibiotic (Spm).
Digestions in silico made for checking minipreps:
Pieces/TU
Restriction enzyme
Expected bands
P35S:EaDAcT:T35S in 2Ω2
NcoI
8806, 390
EcoRV
6652, 1044, 817, 683
Digestion mixes:
Master mix for EcoRV:
3 μL EcoRV
15 μL Red buffer
126 μL H20
Master mix for NcoI:
1.5 μL NcoI
7.5 μL Tango buffer
63 μL H2O
Note: We made master mixes because digestions were made simultaneously with the trichome promoter part.
An agarose gel was made to check the transcriptional unit.
Minipreps of P35s:EaDAcT:T35s in 2Ω2 (1) went correctly.
Miniprep results were quantified and then adjusted at 75 ng/μL:
Pieces/TU
Tube
Concentration (ug/μL)
Initial volume (μL)
Final Volume (μL)
P35S:EaDAcT:T35S in 2Ω2
(1)
141.4
35
31
P35S:EaDAcT:T35S in 2Ω2
2)
3.9
33
(Discarded)
Ligation of P35s:EaDAcT:T35s in 2Ω2 with P35s:AtrΔ11:T35+P35s:HarFAR:T35 in 2Ω1.
1 μL P35s:AtrΔ11:T35+P35s:HarFAR:T35 in 2Ω1
1 μL P35s:EaDAcT:T35s in 2Ω2
1 μL 2α1
1 μL BsaI
1 μL T4 ligase
1 μL ligase buffer
4 μL H20
08/04/2014
Transformation of P35s:EaDAcT:T35s in 2Ω2 P35s:AtrΔ11:T35+P35s:HarFAR:T35 in E. coli.
Agrobacterium liquid cultures (5 mL LB)
P35s:GFP:p19:Tnos (Spm, Tet, Rif)
Empty C58 Agrobacterium tumefaciens (Rif)
2x P35s:EaDAcT:T35s in 2α2 (Rif, Kan)
2x P35s:AtrΔ11:T35+P35s:HarFAR:T35 in 2Ω1 (Rif, Spm)
08/05/2014
Pick colonies from P35s:AtrΔ11:T35+P35s:HarFAR:T35+P35s:EaDAcT:T35s in 2α1.
Repeat PCR of FAO1.
FAO1-PCR1: 3 reactions at different temperatures (54, 59, 64°C)
1.75 μL Candida tropicalis genomic
35 μL HF buffer (5x)
7 μL dNTPs
8.75 μL oligo forward (Jul07)
8.75 μL oligo reverse (Jul08)
1.05 μL Phusion polymerase
112.7 H20
PCR temperatures, 35 cycles:
Step
Temperature (°C)
Time
Initialization
98
2 min
Denaturation
98
10 s
Annealing
54, 59, 64
55 s
Extension
72
55 s
Final elongation
72
7 min
FAO1-PCR2: touchdown PCR
0.5 μL Candida tropicalis genomic
10 μL HF buffer (5x)
2 μL dNTPs
2.5 μL oligo forward (Jul09)
2.5 μL oligo reverse (Jul10)
0.5 μL Phusion polymerase
32 μL H20
PCR temperatures, 35 cycles:
Step
Temperature (°C)
Time
Initialization
98
5 min
Denaturation
98
10 s
Annealing
69.5 (descending 0.5 per cycle)
20 s
Extension
72
55 s
Final elongation
72
7 min
It is not working yet. For the next time we are going to repeat the dilutions in case they weren't correctly done.
08/06/14
Minipreps of yesterday's culture were made:
TU AtrΔ11 + TU HarFAR + TU EaDAcT
Additionally, we made Agrobacterium' culture minipreps using a different kit (We used the QIAgen Miniprep kit 250, 27106)
TU AtrΔ11 + TU HarFAR in 2Ω1
P35S:EaDAcT:T35S in 2Ω2
FAO1 PCR was repeated (this time using a different primers aliquot).
FAO1-PCR1:
0.5 μL Candida tropicalis genomic
10 μL HF buffer (5x)
2 μL dNTPs
2.5 μL oligo forward (Jul07)
2.5 μL oligo reverse (Jul08)
0.5 μL Phusion polymerase
32 μL H20
FAO2-PCR1:
0.5 μL Candida tropicalis genomic
10 μL HF buffer (5x)
2 μL dNTPs
2.5 μL oligo forward (Jul09)
2.5 μL oligo reverse (Jul10)
0.5 μL Phusion polymerase
32 μL H20
PCR temperatures, 35 cycles:
Step
Temperature (°C)
Time
Initialization
98
2 min
Denaturation
98
10 s
Annealing
59 (PCR1)/ 64 (PCR2) (descending 0.5 per cycle)
20 s
Extension
72
55 s
Final elongation
72
7 min
Digestions made in silico to check minipreps:
E. coli
Pieces/TU
Resriction enzymes
Buffer
Expected Bands
TU AtrΔ11+ TU HarFAR + TU EaDAcT in 2α1
EcoRI
Orange
7428, 6323
TU AtrΔ11+ TU HarFAR + TU EaDAcT in 2α1
BglII
Orange
11175, 2576
A. tumefaciens
Pieces/TU
Resriction enzymes
Buffer
Expected Bands
TU AtrΔ11 + TU HarFAR in 2Ω1
EcoRV
Red
9307, 2251
TU AtrΔ11 + TU HarFAR in 2Ω1
BamHI
Green
6652, 4906
TU EaDAcT in 2α2
EcoRV
Red
8021, 683
TU EaDAcT in 2α2
HindIII
Red
6322, 2382
Digestion master mixes:
Master mix for NotI:
2.5 μL NotI
12.5 μL Orange buffer
105 μL H20
Master mix for RsaI:
2 μL NcoI
10 μL Tango buffer
84 μL H2O
Note: We made master mixes because digestions were made simultaneously with the switch part.
We made different mixes for Agrobacterium samples because we think that minipreps are not as good as it is expected.
Agrobacterium sample mix:
0.5 μL Restriction enzyme
2.5 μL Buffer
5 μL Miniprep sample
17 μL H2O
Digestions in A. tumefaciens.
FAO1 PCR product.
All digestions and TU AtrΔ11+ TU HarFAR + TU EaDAcT in 2α1 were correct. PCR products were not correct or absent again.
As digestions were correct, we recultured Agrobacterium in new media (LB) in order to have cultures in exponential phase for tomorrow. We mix in each tube 5 mL of LB with 40 μL of inoculum, XGal (2:1000), IPTG (1:1000)and the corresponding antibiotic (1:1000).
Culture
Antibiotic
P35:GFP:P19:TNos
Spm, Tet, Rif
Agrobacterium (as a control)
Rif
P35S:EaDAcT:T35S
Rif, Kan
P35S:AtrΔ11:T35S + P35S:HarFAR:T35S
Rif, Spm
Recultured media was grown at 28 °C overnight (around 16 h).
08/07/2014
Agroinfiltration in Nicotiana benthamiana.
Agrobacterium control culture and P35s:GFP:P19:Tnos (x2 forth true leaves)
TU AtrΔ11+TU HarFAR and P35s:GFP:P19:Tnos (x2 forth true leaves)
TU AtrΔ11+TU HarFAR and TU EaDAcT and P35s:GFP:P19:Tnos (x2 forth true leaves)
Agroinfiltration protocol consists of:
Centrifuge the cultures 15 min 3000 rpm and discard supernatant.
5 mL of agroinfiltration solution per culture. 100 mL of agroinfiltration solution were composed of 10 mL MES 100mM (pH 5.6), 1 mL MgCl2 1M and 100 μL acetosyringone solution 200 mM (19.62 mg, DMSO 500 μL; prepare freshly). Mix it with the vortex. If the culture is still turbid, add a bit more of agroinfiltration sollution. Put it in the (rodillos) for two hours.
Measure the OD. The optimum OD for agroinfiltration is 0.2. If it is too high adjust the concentration with more agroinfiltration solution.
Mix the cultures, keeping all of them in the same proportions.
Proceed to agroinfiltration.
08/08/2014
In order to have a control for the FAO1 PCR, which hasn't been very successful, Jesus Munoz provided us with 4 primers and 2 clones of Candida tropicalis (C981 ng/μL and pYEP C98 28.2 ng/μL). These primers amplify for the gene HSR1.
Name
Sequence
Tm
HSR1 RTRv+1149
TTTGTCTTGCAACAGGTCCA
56°C
HSR1 clone Fw+1
ATGAGTAAGAAAAGCAACAGTACC
54°C
HSR1 fw-BamHI+480
GCTGGATCCTTAGTAGTAGTGGATCAAGGAAT
49°C (annealing)
HSR1 clone RV+stop
CTAATTTTCTTCTTTTTCAATAGTAACTATCC
51°C
Possibility of primer combinations:
A
HSR1 fw-BamHI+480
HSR1 RTRv+1149
687
49°C
C
HSR1 clone Fw+1
HSR1 clone RV+stop
2187
-
B
HSR1 RTRv+1149
HSR1 clone Fw+1
1168
54°C
We amplified the genomic of C. tropicalis and the two clones (C98 and C98 pYep)with the primer combinations A and B with Taq polymerase at 2 different temperatures (49 and 52°C). C primer combination was not used due to the length of the spected band.
PCR parameters
94°C, 3 min
35 cycles
94°C, 30 s
49 or 52°C, 15 s
72°C, 90 s
72°C, 7 min
PCR products were not present. It probably did not work because we added to much buffer.
08/11/2014
We obtained a different plasmid (pUbiquitina HSRI-CDS col.6) as a positive control of PCR to check the quality of our Candida genome. We diluted them to obtain a final concentration of 30 ng/μL.
PCRs wih Taq polimerase:
1 μL Template
1.5 μL dNTPs
1 μL Forward primer
1 μL Reverse primer
1 μL Taq pol.
5 Buffer 10X
39.5 μL H2O
PCR
Template
F primer
R primer
1
pUbiquitina HSRI-CDS col.6
HSR1 BamHI + 480
HSR1 RTRev + 1149
2
pUbiquitina HSRI-CDS col.6
HSR1 RTRv + 1149
HSR1 Fw + 1
3
C. tropicalis genome
HSRI-CDS col.6
HSR1 BamHI + 480
HSR1 Rtrev + 1149
4
C. tropicalis genome
HSR1 RTRv + 1149
HSR1 Fw + 1
PCR conditions:
94°C 3 min
35 cycles
94°C 30 s
49°C 15 s
72°C 90 s
72°C 7 min
We had amplification in our positive controls. Our C. tropicalis genome may be wrong. Therefore Jesús Muñoz provided us with a new Candida tropicalis (NCYC 2512) culture and also a culture from a Candida tropicales genoteque made in E. coli.
08/12/2014
PHEROMONE ANALYSIS
PONER ENLACE DE LA WIKI
To begin with samples were obtained from the agroinfiltrated plants after 5 days. We collected 9 samples:
2 leaves from P35s:GFP:P19:Tnos
2 leaves from TU AtrΔ11+TU HarFAR and P35s:GFP:P19:Tnos
2 leaves from TU AtrΔ11+TU HarFAR and TU EaDAcT and P35s:GFP:P19:Tnos
1 leaf from a wild type plant
Each sample was stored in a vial and kept in liquid nitrogen. Leaves were mashed using a mortar and liquid nitrogen until powder from each leaf is obtained and stored in a vial .Samples must be always kept in liquid nitrogen or in a -80°C freezer . Afterwards the leaf powder was weighted and introduced in a 10 mL screwcap headspace vial.
94,6 mg of P35s:GFP:P19:Tnos leaf (replica 1)
97,0 mg of TU AtrΔ11+TU HarFAR and P35s:GFP:P19:Tnos leaf (replica 2)
118,7 mg of TU AtrΔ11+TU HarFAR and TU EaDAcT and P35s:GFP:P19:Tnos leaf (replica 1)
100,0 mg of wild type leaf
Then  150 μL of EDTA 500mM and 1 mL of a saturated solution of CaCl2 (5,7M) were added to each vial.
EDTA 500mM preparation:
Stock of solid EDTA Di-Sodium 372,24 Mw and a final solution of 50 mL, 500mM. 372,24*0,5*0,05=9,306 g in 50 mL.
After the addition of EDTA and CaCl2 the samples were sonicated dutring 5 minutes to disgregate the tissue and release the volatile compounds. Afterwards the samples were analysed by GC-MC following this procedure.
PONER LOS PASOS QUE SIGUE EL PARATO, provided by JOSE LUIS MAS ADELANTE: el protocolo entero est\E1 en la carpeta de protocolos como volatile analysis protocol
Analysis was performed overnight.
08/13/2014
First results of the analysis were obtained. The analysis proved that our plant was successfuly producing the desired pheromones in high concentration. As expected z-11-hexadecen-1-ol and z-11-hexadecen-1-ol acetate were being produced and also unexpectedly the z-11-hexadecenal.
As shown in the figure, the leaf agroinfiltrated with TU AtrΔ11+TU HarFAR and P35s:GFP:P19:Tnos (represented in black) shows a successful production of (Z)-11-hexadecen-1-ol compared with the negative control that only has P35s:GFP:P19:Tnos (represented in blue) and shows no expression.
In this figure, expression of (z)-11-hexadecen-1-ol and (z)-11-hexadecen-1-ol acetate is proved. The expression in the leaf infiltrated with TU AtrΔ11+TU HarFAR and TU EaDAcT and P35s:GFP:P19:Tnos is represented in black, and the negative control with P35s:GFP:P19:Tnos is represented in blue.
In this figure, an unexprected peak present in the leaf infiltrated with TU AtrΔ11+TU HarFAR and P35s:GFP:P19:Tnos (black) can be observed. Comparing its spectrum with the one provided from the database seems to be (z)-11-hexadecenal, a desired pheromone,  which is being produced by the plant itself using an endogenous alcohol oxidase. Nevertheless as it is produced with a low yield, the FAO1 of C. tropicalis search is still in progress.
The rest of the samples were prepared for the GC-MS analysis.
The samples were weighted, introduced in the vial and added with EDTA and CaCl2.
94,0 mg of P35s:GFP:P19:Tnos leaf (replica 2)
102,4 mg of TU AtrΔ11+TU HarFAR and P35s:GFP:P19:Tnos leaf (replica 1)
92,0 mg of TU AtrΔ11+TU HarFAR and TU EaDAcT and P35s:GFP:P19:Tnos leaf(replica 2)
Results of the replicae analysis are shown below:
In this replica, the sample with the TU AtrΔ11+TU HarFAR and P35s:GFP:P19:Tnos construction shows a huge production of (z)-11-hexadecen-1-ol.
In this replica, the sample with the TU AtrΔ11+TU HarFAR and TU EaDAcT and P35s:GFP:P19:Tnos shows a higher abundance of (z)-11-hexadecen-1-ol and z-11-hexadecen-1-ol acetate.
In order to verify that the analysed compounds are the desired pheromones, we acquired standards for (z)-11-hexadecen-1-ol and (z)-11-hexadecen-1-ol acetate and (z)-11-hexadecenal, and indeed, the analysed compunds were the right ones.
08/14/2014
We had problems to amplify the FAO1 gene, so in order to obtain it we performed a colony PCR. Using this method, it is possible to amplify a fragment directly from a colony rather than a DNA sample.
We made two different PCRs, one of them as a positive control and the other one to amplify our disered DNA fragment.
Colony PCR protocol (Taq Polimerase):
1 colony (C. tropicalis)
1.5 μL dNTPs
1 μL Forward Primer
1 μL Reverse Primer
1 μL Taq Polimerase
5 μL Buffer 10X
39.5 μL H2O miliQ
Primers used as a control: HSR1 + 480 and RTRv + 1149.
Primers used to amplify FAO1 gene: iGEMJUL07_FAO1_1F and iGEMJUL08_FAO1_1R.
Thermocycler conditions, 35 cycles:
Step
Temperature (°C)
Time
Initialization
94
3 min
Denaturation
94
30 s
Annealing
49
15 s
Extension
72
1 min
Final elongation
72
7 min
Starting from an agar plate containing a Candida gene bank, we add 1 mL of LB medium and we mix it. Then, the mix was transferred into a tube. We stored part of the culture in glycerine and another part (200 μL) was mixed with 5 mL of LB media and Amp (2:1000).
The tube containing the gene bank was grown at 28°C for 1 hour. Then, we made minipreps.
The electrophoresis gel shows that the colony PCR failed, even the control did not work. Additionally, we test the BbsI restriction enzyme and we found that it does not cut well.
08/15/2014
We transformed the whole pathway (P35S:AtrΔ11:T35S, P35S:HarFAR:T35S, P35S:EaDAcT:T35S in 2α1) into Agrobacterium tumefaciens (C58) and we cultured it in solid media with Kan (1:1000) and Rif (1:1000) during 2 days at 28°C.
08/18/2014
We repeated the colony PCR to obtain FAO1 gene and also control PCRs (using the gene bank minipreps made on 08/14/2014).
PCR conditions:
Colony PCR 1 (control):
1 colony (C. tropicalis)
1 μL dNTPs
1 μL HSR1 BamHI + 480
1 μL HSR1 RTRv + 1149
1 μL Tap pol.
5 μL Buffer 10x
39 μL H2O miliQ
Colony PCR 2:
1 colony (C. tropicalis)
1 μL dNTPs
1 μL iGEMJul09
1 μL iGEMJul10
1 μL Tap pol.
5 μL Buffer 10x
39 μL H2O miliQ
Gene bank PCR 3 (control):
1 μL C. tropicallis gene bank miniprep
1 μL dNTPs
1 μL HSR1 BamHI + 480
1 μL HSR1 RTRv + 1149
1 μL Tap pol.
5 μL Buffer 10x
39 μL H2O miliQ
Gene bank PCR 4:
1 μL C. tropicallis gene bank miniprep
1 μL dNTPs
1 μL iGEMJul09
1 μL iGEMJul10
1 μL Tap pol.
5 μL Buffer 10x
39 μL H2O miliQ
PCR conditions were the same as those used on 08/14/2014
08/20/2014
We were trying to obtain the FAO1 gene. We did a yeast colony PCR. Using an sterile tip, we picked one C. tropicalis colony and we introduced them into a vial containing 30 μL SDS 0.2 %. The vial was vortexed 15 seconds and then heated 4 minutes at 90°C. Next, it was centrifuged during 1 minute ans the supernatant was transferred to a new 1.5 μL vial. That was our PCR template.
We performed a control PCR employing control primers (HSRI Rtrv + 1149 and HSRI BamHI + 480)and the another PCR using FAO1 primers as previously done (iGEMJul09 and iGEMJul10).
PCR conditions using Phusion polimerase (35 cycles):
Step
Temperature (°C)
Time
Initialization
98
3 min
Denaturation
98
5 s
Annealing
49
10 s
Extension
72
20 s
Final elongation
72
7 min
We did not close the PCR tube properly so we found our PCR product had evaporated. The other PCR product (the control) was loaded and as it is shown in the gel electrophoresis, it didn't work.
08/22/2014
We did again a yeast genomic extraction (C. tropicalis), but this time we changed the protocol:
Pick 8 colonies of C. tropicalis growth in YPD media and resuspend them with 100 μL of solution (200 mM LiOAc and SDS 1%).
Incubate 15 min at 70°C.
Add 300 μL of ethanol 96%. Then, vortex the solution.
Centrifuge 3 min at 15000 xg.
Discard the superatant and resuspend the pellet (precipitated DNA) with 100 μL TE.
Centrifuge 1 min at 15000 xg.
Recover 1 μL of supernatant.
Using this genomic DNA as a template, we run a PCR (using Taq polimerase) with our primers and another one as a control.
Control PCR:
1 μL template
1 μL dNTPs
1 μL HSR1 clone Fw+1
1 μL HSR1 Rtrv + 1149
μL Taq polimerase
5 μL Buffer 10X
40 μL H2O
FAO1 PCR
1 μL template
1 μL dNTPs
1 μL iGEMJul09_FAO1_PCR2F
1 μL iGEMJul10_FAO1_PCR2R
μL Taq polimerase
5 μL Buffer 10X
40 μL H2O
PCR parameters (35 cycles):
Step
Temperature (°C)
Time
Initialization
94
3 min
Denaturation
94
30 s
Annealing
49
15 s
Extension
72
90 s
Final elongation
72
7 min
08/25/2014
We repeated the FAO1 colony PCR using a C. tropicalis gene bank in E. coli. We made 3 PCRs employing HSR1 primers and other 3 PCRs using our iGEM primers as follows:
PCR 1 (Annealing temperature 49°C)
HSR1 Fw_BamHI+480
HSR1 RTRv+1149
PCR 2 and 3 (Annealing temperature 54°C)
HSR1 clone Fw+1
HSR1 RTRv+1149
PCR 4 and 5 (Annealing temperature 50°C)
iGEMJul07
iGEMJul08
PCR 6 (Annealing temperature 56°C)
iGEMJul09
iGEMJul10
PCR conditions with Taq polimerase (35 cycles):
Step
Temperature (°C)
Time
Initialization
94
3 min
Denaturation
94
30 s
Annealing
49
1:30 min
Extension
72
1:30 min
Final elongation
72
7 min
The electrophoresis gel shows that PCRs have not yielded any product.
Expression in trichomes
07/03/2014
Genomic DNA extraction from Nicotiana tabacum. We need the genome of this organism because we want to obtain the trichome promoter from the NtCPS2 gene.
Obtain 100 mg of the tobacco leaves (5 disks made with a 1.5 mL vial). Made it twice.
Introduce the disks inside the tube.
Introduce the two tubes in liquid nitrogen.
Remove them from the liquid nitrogen and store at -80°C until use.
Remove one tube from -80°C and re-introduce them in liquid nitrogen.
Grind the disks.
Add 600 μL of CTAB (2%) buffer (pre-heat at 65°C.)
Grind the mixture.
Add RNAse (1.6 μL at M = 100 ug/μL for each mL of CTAB buffer).
Vortex it and maintain at 65°C for 45 min. Mix it by inversion 5-15 min.
Add 600 μL cloroform:isoamilic alcohol. Vortex it.
Centrifuge 15 min at 13000 rpm (or 10 min at 14500 rpm.
Recover the supernatant by aspiration (with a 200 μL pipet).
Repeat the last three steps.
Add one volume o isopropanol and mix well by inversion (10 times).
To precipitate, maintain 20 min on ice or at -80°C during 5 min.
Centrifuge 10 min at 13000 rpm (4°C).
Discard the supernatant by decantation (be carefull with the pellet).
Wash with 600 μL ethanol (80%).
Centrifuge 5 min at 13000 rpm.
Discard the ethanol by pipeting and let it dry a few minutes.
Resuspend it in 50-100 μL H2O miliQ or with TE buffer.
Store at -20°C.
Measurement of genomic concentration with nanodrop.
Tabacco 1: 182 ng/μL (Thrown away)
Tabacco 2: 620 ng/μL (Stored at -20°C)
Electrophoresis performed to check the genomic size of tobacco (to see if it is degradated).
It is correct.
07/10/2014
PCR of genomic extraction of tobacco in order to amplify the trichome promoter CPS2.
Ordered primers
IGEMJULO1
IGEMJULO2
Ajust primers to a 100 uM concentration:
IGEMJUL01 + 566 μL miliQ H2O
IGEMJUL02 + 691 μL miliQ H2O
Use a 1:10 alicuot for PCR (10 uM).
Reagents needed for PCR:
0.5 μL template
10 μL buffer HF 5x
2 μL dNTPs
2.5 μL oligo R
2.5 μL oligo F
0.5 μL Pfu
32 μL miliQ H2O
Final volume: 50 μL
Parameters:
98 °C (2 min)
35 cycles
98 °C (10 sec)
59 °C (15 sec)
72 °C (45 sec)
72 °C (7 min)
We didn't get PCR product.
07/11/2014
Repeat PCR with different parameters.
1
2
3
4
5
Template
0.5 μL
0.5 μL
0.5 μL
0.5 μL
0.5 μL
Buffer (5x)
0.5 μL
0.5 μL
0.5 μL
0.5 μL
0.5 μL
dNTPs
2 μL
2 μL
2 μL
2 μL
2 μL
Oligo R
2.5 μL
2.5 μL
2.5 μL
2.5 μL
2.5 μL
Oligo F
2.5 μL
2.5 μL
2.5 μL
2.5 μL
2.5 μL
Phu
0.5 μL
0.5 μL
0.5 μL
0.5 μL
0.5 μL
Buffer
32 μL
32 μL
32 μL
32 μL
32 μL
1, 2 and 5 contain buffer F; 3 and 4 contain buffer GC.
PCR parameters
98 °C (2 min)
35 cycles
98 °C (10 sec)
1, 3, 5 -> 59 °C (15 sec). 2, 4 -> 55 °C (15 sec)
72 °C (45 sec)
72 °C (7 min)
No PCR products again.
Repeat PCR again with other parameters.
Buffer HF
Buffer GC
Template
2 μL
2 μL
Buffer (5x)
40 μL
40 μL
dNTPs
8 μL
8 μL
Oligo R
10 μL
10 μL
Oligo F
10 μL
10 μL
Phu
2
2 μL μL
Buffer
128 μL
128 μL
Set 4 tubes with each buffer at different temperatures: 49, 52, 55, 60.
98 °C (2 min)
35 cycles
98 °C (10 sec)
49, 52, 55, 60 °C (15 sec)
72 °C (45 sec)
72 °C (7 min)
No PCR products again.
07/14/2014
Repeat PCR again with more genomic.
Buffer HF
Buffer GC
Template
5 μL
5 μL
Buffer (5x)
50 μL
50 μL
dNTPs
10 μL
10 μL
Oligo R
12.5 μL
12.5 μL
Oligo F
12.5 μL
12.5 μL
Phu
2.5 μL
2.5 μL
Buffer
107.5 μL
107.5 μL
Same parameters as before except annealing temperatures which are: 50, 53, 57, 59 °C.
We still without having any amplification.
07/18/2014
Repeat the PCR with other enzyme.
12.5 μL Q5 Master mix (2x).
1.25 μL forward primer 10 uM
1.25 μL reverse primer 10 uM
0.5 μL template 620 ng/μL
9.5 μL H2O
Set 4 reactions at 50, 53, 55, 59 °C.
98 °C (30 sec)
35 cycles
98 °C (10 sec)
50, 53, 55, 59 °C (15 sec)
72 °C (45 sec)
72 °C (2 min)
The DNA fragment of interest is around 1.5 kb so we see we finally obtained amplification at 55 and 59 °C reactions.
07/19/2014
Trichome promoter PCR product ligation in pUPD.
1 μL pUPD
1 μL PCR product
1 μL BsmBI (5-10 ud)
1 μL T4 ligase (5-10 ud)
1.2 μL buffer ligase (3 ud)
6.8 μL H20
Set the reaction: 25 cycles x (37°C 2 min, 16°C 5 min).
07/20/2014
Transform and grow in Petri dishes yesterday's ligation of the trichome promoter in pUPD.
07/21/2014
We picked colonies of the trichome promoter in pUPD and grown it in liquid culture.
07/22/2014
We made minipreps of yesterday's liquid culture. Additionally, we have recultured them in solid growth media.
Miniprep quantification:
Piece
Tube
Concentration (ng/μL)
Volume (μL)
Trichome promoter in pUPD
1
317.1
26
Trichome promoter in pUPD
3
354.8
32
Both minipreps were adjusted to 75 ng/μL.
Digestions in silico performed to check the insertion:
Piece
Restriction enzyme
Expected bands
Trichome Promoter in pUPD
NotI
2981, 1523
EcoRV
3942, 562
Note: To see further details of digestion master mixes, go to the biosynthesis part, date 07/22/2014.
Pieces taken from the GoldenBraid 2.0 collection were cultured in solid growth media:
pTnos (GB0037)
pGFP (GB0059)
pLuciferase (GB0096)
07/23/2014
Yesterday's digestions were correct, so the trichome promoter in pUPD was send to sequencing.
We picked colonies from pTnos, pGFP and pLuciferase.
07/24/2014
Results of sequencing the promoter were obtained:
Mutation
Position
T insertion
??
T insertion
??
Minipreps of pTnos, pGFP and pLuciferase.
07/28/2014
Piece
Concentration (ng/μL)
Initial Volume (μL)
Final Volume (μL)
GFP
318.8
35
148.8
Tnos
400.8
35
186.5
pLuciferase
NotI
2981, 1731
See master mix and gel digestion in Biosynthesis part. Pieces were obtained correctly and adjusted to 75 ng/μL.
The following table shows ligation details of the trichome promoter:
Reagent
Volume
CPS2
1 μL
GFP
1 μL
TNos
1 μL
BsaI
1 μL
p2α2
1 μL
T4 ligase
1 μL
Ligase buffer
1 μL
H2O
3 μL
Total Volume
10 μL
07/29/2014
Trichome Promoter transformation in E. coli.
Using an electrocompetent E. coli strain (DH5α) and 1.5 ul ligation (CPS2:GFP:TNos in 2α2), the mix is electroporated at 1500 V. Then, 300 μL of SOC are added and stored at 37 °C with agitation.
07/30/2014
Pick colonies of CPS2:GFP:TNos in 2α2.
07/31/2014
Minipreps of yesterday's culture were made, obtaining the transcripional unit: PCPS2:GFP:TNos in 2 α2
Additionally, we recultured in petri dish with its respective antibiotic (Kan).
Digestions in silico made for checking minipreps:
Pieces/TU
Restriction enzyme
Expected bands
CPS2:GFP:TNos in 2α2
HindIII
6322, 2694
EcoRV
8454, 562
Digestion mixes:
Master mix for EcoRV:
3 μL EcoRV
15 μL Red buffer
126 μL H20
Master mix for HindIII:
2 μL HindIII
10 μL Red buffer
84 μL H2O
Note: We made master mixes because digestions were made simultaneously with the biosynthesis part.
An agarose gel was made to check the transcriptional unit:
Minipreps of CPS2:GFP:TNos in 2α2 (1) went correctly.
Miniprep results were quantified and then adjusted at 75 ng/μL:
Pieces/TU
Tube
Concentration (ug/μL)
Initial volume (μL)
Final Volume (μL)
PCPS2:GFP:TNos in 2α2
1
128.5
33
56.5
PCPS2:GFP:TNos in 2α2
2
135.9
34
61.6
PCPS2:GFP:TNos in 2α2
3
126.2
35
58.9
08/05/2014
Transcriptional Unit (TU) PCPS2:GFP:TNos in 2α2 was transformed in Agrobacterium tumefaciens (C58) and cultured in liquid media with Kan and Rif at 1:1000 (2 days at 28°C).
Note: The scientific name has been updated to Rhizobium radiobacter.
08/08/2014
The TU (PCPS2:GFP:TNos) was recultured in liquid media. Additionally, P35S:GFP:p19:TNos TU was recultured in liquid media, using Spm and Rif as antibiotics.
08/11/2014
Agrobacterium cultures were refreshed in new liquid media. Additionally, we cultured them in solid media.
Minipreps of the TU PCPS2:GFP:TNos in Agrobacterium were made. and digestions were performed to check they were correct.
Digestions in silico performed to check the insertion:
Restriction enzyme
Expected bands
HindIII
6322, 2694
EcoRV
8454, 562
PCPS2:GFP:TNos (1) digestion was correct.
08/12/2014
A part containing P35S:P19:TNos construction was taken from the GoldenBraid collection (GB108) and cultured in solid media with Kanamycin 50 mg/mL. This part is not going to be used as a control but as a silencing supressor.
08/13/2014
One clony (P35S:P19:TNos) was recultured in liquid media.
08/14/2014
Minipreps and streaks of yesterday's culture were made.
The piece was checked by running a gel containing the digested fragment.
Digestions made in silico:
Piece
Restriction enzyme
Expected bands
P35S:P19:TNos
BanI
4256, 392
HindIII
788, 1287, 2563
The GB108 piece (P35S:P19:TNos) is digested as expected in silico.
08/15/2014
We transformed the piece (P35S:P19:T35S) into Agrobacterium tumefaciens (C58) and we cultured it in solid media with Kan (1:1000) and Rif (1:1000) at 28°C during 2 days.
08/17/2014
Agrobacterium containing the piece has not growm well, so we transformed the piece again and we cultured it in an agar plate following the same protocol as previously. In the mean time, we made agar plates.
We made ligations of the three enzymes that form the (Z)11-16:OAc (Z11-hexadecenyl acetate) pheromone but this time each TU will contain the trichome promoter.
Note: For further information about the PCPS2 promoter, please check the trichome promoter section.
Ligation reagents:
PCPS2:AtrΔ11:T35S
1 μL PCPS2
1 μL T35S
1 ul AtrΔ11
1.5 μL 2α1
1 μL Buffer ligase 10X
1 μL T4
1 μL BsaI
2.5 μL H2O
PCPS2:HarFAR:T35S and PCPS2:EaDAcT:T35S
1 μL PCPS2
1 μL T35S
1 ul AtrΔ11/EaDAcT
1 μL 2α2
1 μL Buffer ligase 10X
1 μL T4
1 μL BsaI
3 μL H2O
08/18/2014
TU containing the trichome promoter were transformed into E. coli.
PCPS2:AtrΔ11:T35S
PCPS2:HarFAR:T35S
PCPS2:EaDAcT:T35S
08/19/2014
Agrobacterium has not grown in agarose plates, so we made a transformation again.
E. coli colonies containing the TUs were recultured in liquid media:
PCPS2:AtrΔ11:T35S in 2α1
PCPS2:HarFAR:T35S in 2α2
PCPS2:EaDAcT:T35S in 2α2
08/20/2014
We made minipreps of yesterday's culture and to check them we made digestions in silico:
Piece
Enzyme
Expected bands
PCPS2:AtrΔ11:T35S
EcoRV
562, 8448
EcoRI
2687, 6323
PCPS2:HarFAR:T35S
HindIII
933, 2140, 6322
EcoRV
562, 8833
PCPS2:EaDAcT:T35S
HindIII
2800, 6322
EcoRV
7363, 1197, 562
Digestions were correct, but the PCPS2:HarFAR:T35S digestion 1 with HindIII resulted in more bands than expected, so we discarded that miniprep product and we used the other one.
We adjusted checked products to 75 ng/μL in order to use them in ligations.
We ligated the TUs containing the trichome promoter in Ω vectors as follows:
Ligation 1 (Vt = 10 μL):
1 μL PCPS2:AtrΔ11:T35S
1 μL PCPS2:HarFAR:T35S
1 μL 2Ω1
1 μL BsmBI
1 μL T4 ligase
1 μL Buffer ligase
4 μL H2O miliQ
Ligation 2 (Vt = 10 μL):
1 μL SF (Stuffer fragment)
1 μL PCPS2:EaDAcT:T35S
1 μL 2Ω2
1 μL BsmBI
1 μL T4 ligase
1 μL Buffer ligase
4 μL H2O miliQ
Reaction conditions: 25 cycles x (37°C 2 min, 16°C 5 min).
Then, we recultured E. coli in solid media.
TUs ligated previously were transformed in E. coli following the same protocol as it is usually used.
Finally, we obtained the control (Z)11-16Hexadecenl Acetate that will be used to check the peack in the GC-MS analysis.
08/21/2014
Agrobacterium cells containing P35S:P19:TNos did not grow, so we ask Marta for the glycerinated Agrobacterium culture.
The vector containing the TU was pGreen and we cultured them with Tetracycline, Rifampicin and Kanamycin.
We have confirmed our peak because the control sample has the same retention time and distribution pattern.
Additionally, we have recultured in liquid media TUs ligated yesterday.
08/22/2014
We made minipreps of yesterday's culture:
PCPS2:EaDAcT:T35S in 2Ω2
PCPS2:AtrΔ11:T35S_PCPS2:HarFAR:T35S in 2Ω1
Digestions in silico made to check minipreps:
Piece
Enzyme
Expected bands
PCPS2:AtrΔ11:T35S_PCPS2:HarFAR:T35S
BamHI
6652, 5742
NotI
9572, 1532, 1290
PCPS2:EaDAcT:T35S
NotI
6792, 1532, 1290
NdeI
7125, 2419
All digestions were not correct.
08/24/2014
We transformed in Agrobacterium the following TUs:
PCPS2:AtrΔ11:T35S_PCPS2:HarFAR:T35S in 2Ω1
PCPS2:EaDAcT:T35S in 2Ω2
We made minipreps of Agrobacterium culture: P35S:AtrΔ11:T35S_P35S:HarFAR:T35S_P35S:EaDAcT:T35S in 2α1.
Additionally, we refreshed Agrobacterium cultures with their corresponding antibiotic:
T35S:P19:TNos (Rif, Kan, Tet)
PCPS2:GFP:TNos (Rif, Kan)
T35S:P19:GFP:TNos (Rif, Smp, Tet)
TUs: P35S:AtrΔ11:T35S_P35S:HarFAR:T35S_P35S:EaDAcT:T35S in 2α1 (Rif, Kan)
08/25/2014
Digestions in silico made to check yesterday's minipreps:
Piece
Enzyme
Expected bands
P35S:AtrΔ11:T35S_P35S:HarFAR:T35S_P35S:EaDAcT:T35S
EcoRI
7428, 6323
BglII
2576, 11175
We repeated the Agroinfiltration protocol, but this time we infiltrated the following A. tumefaciens cultures:
T35S:P19:TNos
PCPS2:GFP:TNos + T35S:P19:TNos
T35S:P19:GFP:TNos
T35S:P19:GFP:TNos + P35S:AtrΔ11:T35S_P35S:HarFAR:T35S_P35S:EaDAcT:T35S
We picked colonies which were transformed yesterday and we recultured them in liquid media with Spm, IPTG and X-Gal.
Finally, we have trasplanted N. benthamiana into new flowerpots to have plants ready to infiltrate in the future.
08/26/2014
We made minipreps of yesterday's culture, but only for the TU PCPS2:AtrΔ11:T35S_PCPS2:HarFAR:T35S in 2Ω1 since the other tubes were blue colored.
Digestions in silico the check the minipreps:
Piece
Enzyme
Expected bands
PCPS2:AtrΔ11:T35S_PCPSS:HarFAR:T35S
BamHI
6652, 5742
BglII
8131, 2669, 1594
Biosafety module
07/22/2014
Pieces taken from the GoldenBraid 2.0 collection were cultured in solid growth media:
P35S:Rosea:TNos
TA29:Barnase:TNos (from GoldenBraid 1.0 collection)
We were told by our advisor that Rosea produces necrosis in N. benthamiana, so we must think of an alternative.
07/23/2014
We picked colonies from P35S:Rosea:TNos and TA29:Barnase:TNos.
07/24/2014
Minipreps of P35S:Rosea:TNos and TA29:Barnase:TNos.
07/25/2014
Digestions in silico made for checking yesterday's minipreps:
Pieces
Restriction enzyme
Expected bands
P35S:Rosea:Tnos
BglII
2495, 2302
NcoI
4407, 390
TA29:Barnase:Tnos
BglII
2825, 2245
07/28/2014
See master mix and gel digestion in Biosynthesis part. Pieces were obtained correctly and adjusted to 75 ng/μL.
07/31/2014
We talked with the NRP-UEA-Norwich team. We stablished a possible collaboration in developing the biosafety module together. They could send us their chromoproteins and we could send them our barnase and TA29 promoter.
08/05/2014
Order primers for TA29 and barnase:
Name
Sequence
T annealing
I14Ago01_TA29_F1
CGCCGTCTCGCTCGGGAGTAGCGAATGCAATTAATTTAGACAT
61.8°C
I14Ago02_TA29_R1
CGCCGTCTCGCTCGCATTTTTAGCTAATTTCTTTAAGTAAAAACTTTG
60.8°C
I14Ago03_barnase_F1
CGCCGTCTCGCTCGAATGGCACAGGTTATCAACACG
65.0°C
I14Ago04_barnase_R1
CGCCGTCTCGCTCGAAGCTTATCTGATTTTTGTAAAGGTCTGATAATG
63.4°C
08/07/2014
Primers received. PCR for barnase and TA29 performed.
TA29 PCR parameters
98°C, 2 min
35 cycles
98°C, 10 s
60°C, 18 s
72°C, 40 s
72°C, 7 min
Barnase PCR parameters
98°C, 2 min
35 cycles
98°C, 10 s
63°C, 18 s
72°C, 40 s
72°C, 7 min
We didn't obtain PCR product. There is a band for the barnase, but it should be around 330 bp.
08/08/2014
We repeat yesterday's PCR with 2 degrees less in the annealing step.
Results obtained are the same of yesterday's. We should think about charging something else.
08/11/2014
The previous PCR was repeated changing the annealing temperature to 61°C.
We still do not get PCR product.
08/12/2014
We forgot to adjust the TA29:Barnase:Tnos from GB 1.0 to 5 ng/μL. Maybe that's why PCRs don't work. We repeated again with the appropiate temperatures (60°C for TA29 and 63°C for barnase), but it still doesn't work!
08/18/2014
We transformed in E. coli the iGEM Barnase part (BBa_1716211), placed in Plate 3, 11o.
?? --> Se puso una PCR o no?
A PCR using Nicotiana tobacum genome as a template was made to obtain the Ta29 fragment. Primers used and also PCR conditions were the same as previous PCRs.
08/19/2014
E. coli colonies containing the iGEM Barnase part (BBa_I716.211) were recultured in liquid media.
08/20/2014
We made minipreps of yesterday's culture and to check them we made digestions in silico:
Piece
Enzyme
Expected bands
Barnase
NotI
2046, 357
BamHI
1558, 845
Digestions were correct, so we adjusted the product to 5 ng/μL in order to use them as a PCR template.
Adittionally, we made a ligation to obtain the TA29 piece in pUPD vector as follows:
1 μL pUPD
1 μL TA29
1 μL BsmBI
1.2 μL Buffer ligase
1 μL T4 ligase
6.8 μL H2O miliQ
Reaction conditions: 25 cycles x (37°C 2 min, 16°C 5 min).
We made a mistake predicting digetions in silico, so we repeated them, this time with the appropriate vector (pSB1C3).
Piece
Enzyme
Expected bands
Barnase
EcoRI and PstI
2029, 374
This double digestion was checked with an agarose gel showing that the resulting bands were the expected ones.
Additionally, TA29 in pUPD vector was transformed in E. coli. The protocol followed was the same as previously done.
08/21/2014
We did a PCR to obtain the Barnase as a product using the primers Bar_F1 and Bar_R1 and the template obtained yesterday.
PCR conditions (35 cycles):
Step
Temperature (°C)
Time
Initialization
98
1:30 min
Denaturation
98
10 s
Annealing
63
20 s
Extension
72
20 s
Final elongation
72
7 min
The agarose gel shows that the PCR product was correct, but we purified the band to get a better quality product using a QUIAGEN purification kit (QIAEXII Gel Extraction Kit 150, Cat. No: 20021).
We recultured in liquid media yesterday's TA29 culture.
08/22/2014
We made minipreps of yesterday's culture.
Piece
Restriction enzyme
Expected bands
TA29 in pUPD
EcoRI
2997, 817
PvuI
2818, 1096
Digestions were not correct. We picked again TA29 in pUPD colonies and recultured them in liquid media.
08/24/2014
We made minipreps of TA29 culture.
08/25/2014
In silico digestions made to check yesterday's minipreps.
Piece
Restriction enzyme
Expected bands
TA29 in pUPD
EcoRI
2997, 817
PvuI
2818, 1096
Resulting bands were as expected in silico, the piece is correct.
Measurement Interlab Study
08/20/2014
We transformed BBa_J23101, BBa_E0240 and BBa_J23115. All of the pieces share the vector pSB1C3, so we have cultured them in solid LB medium supplemented with chloramphenicol.
08/21/2014
Pick colonies and grow them in agitation at 37°C in liquid media supplemented with chloramphenicol.
08/22/2014
We made minipreps of yesterday's culture, except from BBa_E0240 culture, which has not grown.
Digestions in silico:
Part
Enzyme
Expected bands
BBa_J23101
RsaI
1567, 538
XhoI
1213, 892
BBa_23115
RsaI
1199, 538, 368
XhoI
1213, 892
All digestions were correct except BBa_23101 (1).
08/24/2014
BBa_E0240 and BBa_I20260 parts were transformed in E. coli DH5-α. BBa_E0240 is resistant to kanamycin and BBa_I20260 to chloramphenicol.
08/25/2014
Colonies did not grow so plates were left one more day at 37ºC.
08/26/2014
Pick colonies of BBa_E0240 and grow them in agitation at 37°C in liquid media supplemented with kanamycin.
Colonies of BBa_I20260 were not grown, so we performed transformation again.
08/27/2014
Pick colonies of BBa_I2026 grow them in agitation at 37°C in liquid media supplemented with kanamycin.
Minipreps and digestions of BBa_E0240.
Part
Enzyme
Expected bands
BBa_E0240
NcoI
1991, 955
Assembly protocol for BBa_J23101+BBa_E0240 and BBa_J23115+BBa_E0240:
Double digestions
250 ng of plasmid in 16 μL H20
2.5 μL NEBuffer
0.5 μL BSA
0.5 μL enzyme 1
0.5 μL enzyme 2
Final volume: 20 μL
Part
Enzymes
Size
BBa_J23101
SpeI, PstI
2.1 kb
BBa_J23115
SpeI, PstI
2.1 kb
BBa_E0240
XbaI, PstI
800 bp
Incubate 30 min at 37°C and 20 min more at 80°C.
Run digestions in an agarose gel and purify band using QIAEX II Gel Extraction Kit.
BioBricks ligations
2 μL part 1 (25 ng)
2 μL part 2 (25 ng)
1 μL T4 buffer 10X
0.5 μL T4
4 μL H20
Part 1
Part2
BBa_J23101
BBa_E0240
BBa_J23115
BBa_E0240
Incubate 30 min at 16°C and 20 min more at 80°C.
Transform both ligations (BBa_J23101+BBa_E0240 and BBa_J23115+BBa_E0240) and grow in solid plates supplemented with chloramphenicol.
08/28/2014
Colonies did not grow so plates were left one more day at 37ºC.
Minipreps and digestions of BBa_I2026.
Device
Enzyme
Expected bands
BBa_I20620
NotI
2726,   943
NcoI
3296, 373
There was some kind of trouble with the gel and bands where not clear. We repeat the digestion again other day.
08/29/2014
Pick colonies of BBa_J23101+BBa_E0240 and BBa_J23115+BBa_E0240 grow them in agitation at 37°C in liquid media supplemented with chloramphenicol.
08/30/2014
Minipreps of BBa_J23101+BBa_E0240 and BBa_J23115+BBa_E0240 and digestions. Repeat digestions of BBa_I20620.
Device
Enzyme
Expected bands
BBa_J23101+BBa_E0240
NotI
2046,   943
NcoI
1991,   998
BBa_J23115+BBa_E0240
NotI
2046,   943
NcoI
1991,   998
None of the digestions of BBa_J23101+BBa_E0240. Digestions BBa_J23115+BBa_E0240 (1) and (4) were correct and all of the colonies of BBa_I20620 were correct.
08/31/2014
Pick 5 more colonies of BBa_J23101+BBa_E0240.
09/01/2014
Minipreps and digestions of 5 more cultures of BBa_J23101+BBa_E0240.
BBa_J23101+BBa_E0240 (4) ligation is correct.
We noticed that, for some reason, the stry of BBa_J23115+BBa_E0240 was contaminated, so we picked 6 more colonies.
09/02/2014
Minipreps of BBa_J23115+BBa_E0240 and digestions.
All digestions are correct except BBa_J23115+BBa_E0240 (1).
We found out that the stry of BBa_J23101+BBa_E0240 was contaminated as well, so due to the low efficiency of this ligation (1/9) we decided to transform again with the correct miniprep.
09/03/2014
Pick one colony of BBa_J23101+BBa_E0240.
09/04/2014
Miniprep of BBa_J23101+BBa_E0240.
The digestion was correct. We have scheduled the GFP for next Wednesday.
09/09/2014
Pick colonies for Measurement Interlab Study. Three technical samples for each device and the negative control (Untransformed E.coli DH5-α)were picked. E. coli DH5-α cells were grown in 3.5 ml Luria-Bertani  broth supplied with the corresponding antibiotic at 37°C with shaking at 250 rpm for 16 hours.
09/10/2014
Today we measured GFP for the Measurement Interlab Study.
Cells were centrifuged at 4500 rpm for 5 minutes and resuspended in ten folds the culture volume with a phosphate buffered saline (58 mM Na2HPO4, 17 mM NaH2PO4, 68 mM NaCl), as performed by Scholz et al., 2000. Na2HPO4 and NaH2PO4 were purchased from Panreac. NaCl was purchased from Fisher Bioreagents.
A GloMax-Multi Detection System form Promega fluorometer configured with the Blue optical kit (&Lamda;ex=490 nm, &Lamda;em=510-575 nm) was used to measure fluorescence. For measuring fluorescence 250 μl of each sample were placed in a black 96-well plate. Each sample was measured three times and an average was displayed on the screen.
A Biowave CO 8000 from Biochrom spectophotometer was used to measure absorbance at 600 nm. For measuring absorbance 700 μl were placed in a cubet and measured one by one in the spectrophotometer.
Sample
Fluorescence*
Optical density
Fluorescence / Optical density
Negative control
(1)
1.085
0.38
2.854
(2)
1.036
0.35
2.959
(3)
1.076
0.39
2.759
BBa_I20260
(1)
4.907
0.36
13.632
(2)
4.754
0.34
13.981
(3)
3.494
0.26
13.439
BBa_J23101 + BBa_E0240
(1)
57.393
0.43
133.471
(2)
61.622
0.47
131.110
(3)
63.999
0.47
136.167
BBa_J23115 + BBa_E0240
(1)
1.389
0.37
3.754
(2)
1.353
0.37
3.656
(3)
1.370
0.33
4.151
*Fluorescence measurements were calculated subtracting the average value of fluorescence of three samples of phosphate buffer (286.1) to the value given for each sample by the fluorometer.
Sample
Fluorescence
Optical density
Fluorescence / Optical density
Negative control
1.065±0.026
0.373±0.021
2.857±0.100
BBa_I20260
4.385±0.775
0.320±0.053
13.684±0.275
BBa_J23101 + BBa_E0240
61.004±3.346
0.457±0.023
133.583±2.530
Bba_J23115 + BBa_E0240
1.370±0.018
0.357±0.023
3.854±0.262
Switch
07/04/2014
Adquisition of S. cerevisiae genomic DNA. (5 μL, stored in the fridge)
07/28/2014
We had the genome of S. cerevisiae, needed to extract the target genes that are going to be used to build the switch. However we finally used our genome extraction (see Biosynthesis part, date 07/23/2014 for further details).
Previously we have designed a cupple of primers to amplify the CUP1 and CUP2 genes present in the yeast.
PCR reaction reagents:
Reagent
CUP1-PCR1
CUP2-PCR2
Template
0.5 μL
0.5 μL
Buffer HF (5X)
10.0 μL
10.0 μL
dNTPs
2.0 μL
2.0 μL
Oligo R (JUL06)
2.5 μL
2.5 μL
Oligo F (JUL05)
2.5 μL
2.5 μL
Phusion polymerase
0.5 μL
0.5 μL
H2O
32.0 μL
32.0 μL
Annealing temperature: both 61 °C
PCR products were checked using an electrophoresis. Expected bands:
CUP1-PCR1: 386 bp
CUP2-PCR2: 348 bp
Both PCR products were correct.
07/30/2014
We repeated the PCR because we had to purify the bands CUP1-PCR1 and CUP2-PCR2.For this purpose we used the kit "QIAEX II Gel Extraction Kit".
Ligation of both parts of CUP2.
1 μL CUP1-PCR1
1 μL CUP1-PCR1
1 μL pUPD
1 μL BsmBI
1 μL T4 ligase
1 μL ligase buffer
4 μL H20
07/31/2014
CUP2 was transformed in pUPD and cultured in solid media (37°C).
08/04/2014
Grow the piece corresponding to Gal4 Activation Domain (GB0095) from the GB collection in solid medium.
08/05/2014
Pick colonies from CUP2 (3 colonies) and Gal4AD (1 colony).
08/06/14
Minipreps of yesterday's culture were made:
Gal4AD
CUP2
Digestions made in silico in order to check transcriptional units:
Pieces/TU
Resriction enzymes
Buffer
Expected Bands
CUP2 in pUPD
Not1
Orange
2981, 752
CUP2 in pUPD
RsaI
Tango
2457, 1276
Gal4AD in pUPD
Not1
Orange
2981, 330
Gal4AD
PuuI
Red
2215, 1096
CUP2 in pUPD is correct. RsaI restriction enzyme does not cut properly, as a result we obtained different bands from those ones expected.
Gal4AD piece is correct.
08/11/2014
Sequencing results of CUP2 piece were finally received and they were correct.
As the sequence was correct, we could continue with ligations.
Quantification
CUP2: 110.3 ng/μL
Gal4: 221.4 ng/μL
Samples were diluted to 75 ng/μL.
The following ligations were made:
P35S:CUP2:Gal4AD:T35S
1 μL P35S
1 μL CUP2
1 μL Gal4AD
1 μL T35S
1 μL BsaI
1 μL T4 ligase
1 μL Buffer ligase
1 μL 2α2 vector
2 μL H2O
PCPS2:CUP2:Gal4AD:T35S
1 μL PCPS2
1 μL CUP2
1 μL Gal4AD
1 μL T35S
1 μL BsaI
1 μL T4 ligase
1 μL Buffer ligase
1 μL 2α2 vector
2 μL H2O
08/12/2014
E. Coli transformation with the previous ligations and culture in solid medium (LB-agar with Kanamycin and X-Gal + IPTG) overnight.
08/13/2014
We recultured yesterday's colonies in liquid media with the same antibiotic (Kan) and X-Gal.
P35S:CUP2:Gal4AD:T35S in 2α2 (3 colonies)
PCPS2:CUP2:Gal4AD:T35S in 2α2 (3 colonies)
08/14/2014
Minipreps of yesterday's culture and streakes were made.
Digestions made in sililco to chceck the TU:
Pieces/TU
Resriction enzymes
Buffer
Expected Bands
PCPS2:CUP2:Gal4AD:T35S
HindIII
Red
6322, 2641
EcoRV
Red
562, 8401
P35S:CUP2:Gal4AD:T35S
HindIII
Red
6322, 2223
BclI
Green
476, 7137, 932
The agarose gel shows that P35S:CUP2:Gal4AD:T35S piece is not well build. Nevertheless, PCPS2:CUP2:Gal4AD:T35S piece is OK.
08/15/2014
P35S:CUP2:Gal4AD:T35S digestions made yesterday were repeated as follows:
Pieces/TU
Resriction enzymes
Buffer
Expected Bands
P35S:CUP2:Gal4AD:T35S
HindIII
Red
6322, 2223
NotI
Green
5723, 1290, 1532
After running the electrophoresis, the resulting bands show that there is something more than expected in the plasmid. Furthermore, we check that the extra part has been added in the part region. Ligation step has to be repeated.
08/17/2014
We repeated the P35S:CUP2:Gal4AD:T35S ligation.
Ligation reagents:
P35S:CUP2:Gal4AD:T35S:
1 μL PCPS2
1 μL T35S
1 ul Gal4AD
1 μL 2α2
1 μL CUP2
1 μL Buffer ligase 10X
1 μL T4
1 μL BsaI
2 μL H2O
08/18/2014
TU piece was transformed in E. coli (P35S:CUP2:Gal4AD:T35S) and cultured in solid media.
08/19/2014
E. coli colonies containing the TU (P35S:CUP2:Gal4AD:T35S in 2α2) were recultured in liquid media.
08/20/2014
Piece
Enzyme
Expected bands
P35S:CUP2:Gal4AD:T35S
HindIII
6322, 2223
NcoI
8155, 390
Digestions were correct.
Translator to BioBricks
08/07/2014
Ale's primers labeled A11Dic32 and M11Nov12 found.
Run PCR with the following templates and primers:
Template
Forward
Reverse
Expected lenght
P35s
iGEMJul11 A11Dic32
1086 bp
T35s
M11Nov12iGEM12Jul
284 bp
P35s PCR parameters
98°C, 2 min
35 cycles
98°C, 10 s
67°C, 18 s
72°C, 40 s
98°C, 7 min
T35s PCR parameters
98°C, 2 min
35 cycles
98°C, 10 s
65°C, 18 s
72°C, 40 s
98°C, 7 min
We didn't obtain PCR product.
08/08/2014
We repeat yesterday's PCR with 2 degrees less in the annealing step.
Now there is a band for P35s but it should not be there.
08/11/2014
The previous PCR was repeated changing the annealing temperature to 61°C.
We still do not get PCR product.
08/12/2014
We repeated the PCR once more, this time setting the annealing temperatures at (59°C for T35s and 61°C for P35s).
08/19/2014
We are trying another PCR strategy to obtain the PCR product.
PCR1: P35S template (as previously done)
PCR2: P35S:AtrΔ11:T35S template
PCR
Primers
Tm (°C)
1
iGEMJul11 and A11Dic32
62
2
M11Nov12 and iGEMJul12
65
We check the PCR products and only the T35S product was amplified correctly (the expected band was around 300 bp).
08/20/2014
As the PCR product was correct, we made a ligation to obtain the T35S piece in pUPD vector as follows:
1 μL pUPD
1 μL T35S_BB
1.2 μL Buffer ligase
1 μL BsmBI
1 μL T4 ligase
6.8 μL H20 miliQ
Reaction conditions: 25 cycles x (37°C 2 min, 16°C 5 min).
We repeated a PCR to obtain the P35S using the same template as previously and the following conditions (35 cycles):
Step
Temperature (°C)
Time
Initialization
98
1:30 min
Denaturation
98
10 s
Annealing
57/62/67
20 s
Extension
72
25 s
Final elongation
72
7 min
We checked the PCR product running a gel electrophoresis, but the PCR did not work again and the agarose gel did not show any band.
T35S in pUPD vector was transformed in E. coli and cultured in agar plates. The protocol followed was the same as it is usually done.
08/21/2014
We picked E. coli colonies and recultured them in liquid media with the apprpriate antibiotic, Amp (2:1000).
08/22/2014
We made minipreps of yesterday's culture and we made digestions to check them.
In silico digestions:
Piece
Restriction enzyme
Expected bands
T35S in pUPD
EcoRI
2997, 309
PvuI
2210, 1096
Digestions were correct.
We run a PCR with the TUs as templates (adjusted to 5 ng/μL) and using Jul11 and Jul12 as primers.
EaDAcT (2α2)
HarFAR (2α2)
AtrΔ11 (2α1)
Conditions for 35 cycles:
Step
Temperature (°C)
Time
Initialization
98
2 min
Denaturation
98
15 s
Annealing
65
20 s
Extension
72
45 s
Final elongation
72
7 min
We made another PCR to obtain P35S as a product. This time, we used Q5 High Fidelity polimerase.
Conditions for 35 cycles:
Step
Temperature (°C)
Time
Initialization
98
1:30 min
Denaturation
98
10 s
Annealing
55
20 s
Extension
72
25 s
Final elongation
72
7 min
The gel shows that the template is not there.
08/25/2014
We repeated the PCR made the previous day using TUs as a template and primers Jul11 and Jul12, but this time we changed the extension time to 1:30 min.
The gel showed that the PCR products were correct.
We repeated the FAO1 PCR
Pin it!
Great for pinned navigations — or just about anything you want to stick.
Developed by Webpop .
Sitemap | Twitter | Facebook | Email
This wiki is designed and constructed by Valencia_UPV.
Retrieved from " http://2014.igem.org/Team:Valencia_UPV/prueba "
Recent changes
What links here
Related changes
Special pages
My preferences
Printable version
Permanent link
Privacy policy
Disclaimers
