<HTML xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en" dir="ltr">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki  ltr ns-0 ns-subject page-Team_Valencia_UPV_otraprueba"><DIV id="globalWrapper"><DIV id="top-section"><DIV id="p-logo"><A href="/Main_Page" title="Main Page">&quot;
	    </A></DIV><DIV id="menubar" class="left-menu noprint"><UL><LI class="selected"><A href="/Team:Valencia_UPV/otraprueba">Page               </A></LI><LI class="new"><A href="/wiki/index.php?title=Talk:Team:Valencia_UPV/otraprueba&amp;action=edit&amp;redlink=1">Discussion               </A></LI><LI><A href="/wiki/index.php?title=Team:Valencia_UPV/otraprueba&amp;action=edit">View source               </A></LI><LI><A href="/wiki/index.php?title=Team:Valencia_UPV/otraprueba&amp;action=history">History               </A></LI><LI style="color:white;cursor:default">teams</LI></UL></DIV><DIV class="right-menu noprint" id="menubar"><UL><LI id="pt-login"><A href="/wiki/index.php?title=Special:UserLogin&amp;returnto=Team:Valencia_UPV/otraprueba" title="You are encouraged to log in; however, it is not mandatory [o]" accesskey="o">Log in</A></LI></UL></DIV><DIV id="search-controls" class="noprint"><FORM action="/Special:Search" id="searchform"> </FORM></DIV></DIV><DIV id="content"><H1 class="firstHeading">Team:Valencia UPV/otraprueba</H1><DIV id="bodyContent"><H3 id="siteSub" class="noprint">From 2014.igem.org</H3><DIV class="example"><DIV class="menu"><SPAN><UL id="nav"><LI><A href="https://2014.igem.org/Team:Valencia_UPV">Home</A></LI><LI><A href="#">Team</A></LI><LI><A href="https://2014.igem.org/Team:Valencia_UPV/Team/Students">Students</A></LI><LI><A href="https://2014.igem.org/Team:Valencia_UPV/Team/Advisors">Advisors and Instructors</A></LI><LI><A href="https://igem.org/Team.cgi?year=2014&amp;team_name=Valencia_UPV">Official Team Profile</A></LI><LI><A href="https://2014.igem.org/Team:Valencia_UPV/Team/Attributions">Attributions</A></LI><LI><A href="https://2014.igem.org/Team:Valencia_UPV/Sponsors">Sponsors</A></LI></UL></SPAN></DIV></DIV><LI><A href="#">Project</A></LI><LI><A href="https://2014.igem.org/Team:Valencia_UPV/Project/overview">Overview</A></LI><LI><A href="https://2014.igem.org/Team:Valencia_UPV/Project/modules">Modules</A></LI><LI><A href="https://2014.igem.org/Team:Valencia_UPV/Project/results">Results</A></LI><LI><A href="https://2014.igem.org/Team:Valencia_UPV/Project/notebook">Notebook</A></LI></DIV></DIV><LI><A href="#">Modeling</A></LI><LI><A href="https://2014.igem.org/Team:Valencia_UPV/Modeling/overview">Overview</A></LI><LI><A href="https://2014.igem.org/Team:Valencia_UPV/Modeling/fba">Pheromone Production</A></LI><LI><A href="https://2014.igem.org/Team:Valencia_UPV/Modeling/diffusion">Pheromone Diffusion and Moth Response</A></LI></DIV><LI><A href="https://2014.igem.org/Team:Valencia_UPV/safety">Safety</A></LI><LI><A href="#">Policy and Practices</A></LI><LI><A href="https://2014.igem.org/Team:Valencia_UPV/policy/overview">Overview</A></LI><LI><A href="https://2014.igem.org/Team:Valencia_UPV/policy/activities">Activities</A></LI><LI><A href="https://2014.igem.org/Team:Valencia_UPV/policy/outreach">Outreach</A></LI><LI><A href="https://2014.igem.org/Team:Valencia_UPV/policy/news">News &amp; Press</A></LI><LI><A href="https://2014.igem.org/Team:Valencia_UPV/policy/contact_us">Contact Us</A></LI><LI><A href="https://2014.igem.org/Team:Valencia_UPV/Project/interlab">Interlab Study</A></LI><LI><A href="#">Achievements</A></LI><LI><A href="https://2014.igem.org/Team:Valencia_UPV/Medal_requierement">Medal Requirements</A></LI><LI><A href="https://2014.igem.org/Team:Valencia_UPV/VUPV_Parts">Parts</A></LI><LI><A href="https://2014.igem.org/Team:Valencia_UPV/Achievements">Achievements</A></LI><LI><A href="https://2014.igem.org/Team:Valencia_UPV/Collaboration">Collaborations</A></LI><DIV id="cn-box-left" align="justify"><SECTION class="container clearfix"><DIV class="box_above_notebook">

Contents:
<UL style="margin-left: 1.5em;"><LI><A href="#Biosynthesis_under_constitutive_promoter">Biosynthesis under constitutive promoter</A></LI><LI><A href="#Expression_in_trichomes">Expression in trichomes</A></LI><LI><A href="#Biosafety_module">Biosafety module</A></LI><LI><A href="#Measurement_Interlab_Study">Measurement Interlab Study</A></LI><LI><A href="#Switch">Switch</A></LI><LI><A href="#Translator_to_BioBricks">Translator to BioBricks</A></LI></UL></DIV><H3 class="section_notebook">Biosynthesis under constitutive promoter</H3><H4 class="date_notebook">06/09/2014</H4><P class="p_notebook">The enzymes involved in the biosynthesis pathways are AtrΔ11, HarFAR, FAO1, EaDAcT.</P><P class="p_notebook">The design of the GBlocks was performed taking into account the following considerations:</P><UL class="ul_notebook"><LI>Codon optimization</LI><LI>Inner restriction sites eliminations by finding synonymous mutations</LI><LI>Addition of GB endings</LI></UL><H4 class="date_notebook">06/10/2014</H4><P class="p_notebook">Codes for IDT known. MEGAGEM2014 - 25% off one order, up to 800 USD</P><P class="p_notebook">GBlocks designed to be compatible with BioBricks and GoldenBraid (GB).</P><H4 class="date_notebook">06/11/2014</H4><P class="p_notebook">We ordered the following gBlocks and primers.</P><UL class="ul_notebook"><LI>EaDAcT: <I>Eunymus alatus</I> (adapted for GB) 1127 bp</LI><LI>HarFAR: <I>Helicoverpa armigera</I> (adapted for GB) 1400 bp</LI><LI>AtrΔ11: <I>Amyelois transitella</I> (order primers for GB) 1000 bp</LI><UL class="ul_ul_notebook"><LI>I14Jun03 AtrΔ11 F1</LI><LI>I14Jun04 AtrΔ11 R1</LI></UL><LI>FAO1: <I>N. benthamiana</I> primers</LI><UL class="ul_ul_notebook"><LI>I14Jun01 FAO1 F1</LI><LI>I14Jun02 FAO1 R1</LI></UL></UL><TABLE class="table_notebook"><TBODY><TR class="tr_notebook"><TD class="td_notebook">Name</TD><TD class="td_notebook">Sequence</TD><TD class="td_notebook">Lenght</TD><TD class="td_notebook">Tm (NTI)</TD><TD class="td_notebook">Tm (Phusion)</TD></TR><TR class="tr_notebook"><TD class="td_notebook">I14Jun01_FAO1_F1</TD><TD class="td_notebook">cgccgtctcgctcgaatggagaaaaagagccatcc</TD><TD class="td_notebook">35</TD><TD class="td_notebook">49.9</TD><TD class="td_notebook">62.4</TD></TR><TR class="tr_notebook"><TD class="td_notebook">I14Jun02_FAO1_R1</TD><TD class="td_notebook">cgccgtctcgctcgaagcttatcttgagaatttgccttcttttatc</TD><TD class="td_notebook">46</TD><TD class="td_notebook">54.5</TD><TD class="td_notebook">63.7</TD></TR><TR class="tr_notebook"><TD class="td_notebook">I14Jun03Atr_D11_F1</TD><TD class="td_notebook">gcgccgtctcgctcgaatggttcctaataag</TD><TD class="td_notebook">31</TD><TD class="td_notebook">54.5</TD><TD class="td_notebook">65.3</TD></TR><TR class="tr_notebook"><TD class="td_notebook">I14Jun04Atr_D11_R1</TD><TD class="td_notebook">gcgccgtctcgctcgaagctcaacgtttc</TD><TD class="td_notebook">29</TD><TD class="td_notebook">57</TD><TD class="td_notebook">69.1</TD></TR></TBODY></TABLE><H4 class="date_notebook">06/24/2014</H4><P class="p_notebook">We thought which parts of the GB collection could we use.</P><P class="p_notebook">Strategy 1:</P><UL class="ul_notebook"><LI>pP35S, pT35s (x2)</LI><LI>pAtUbq10, pTAtHSP18.2</LI></UL><P class="p_notebook">Strategy 2:</P><UL class="ul_notebook"><LI>pP35S, pT35s</LI><LI>pP35s, pTAtHSP18.2</LI><LI>pAtUbq10, pTAtHSP18.2</LI></UL><P class="p_notebook">Strategy 3:</P><UL class="ul_notebook"><LI>pP35S, pT35s</LI><LI>pP35s, pTTctp</LI><LI>pAtUbq10, pTAtHSP18.2</LI></UL><P class="p_notebook">Pieces to take from GB2.0 colection:</P><TABLE class="table_notebook"><TBODY><TR class="tr_notebook"><TD class="td_notebook">pDGB2α1</TD><TD class="td_notebook">GB0483</TD><TD class="td_notebook">Kan</TD></TR><TR class="tr_notebook"><TD class="td_notebook">pDGB2α2</TD><TD class="td_notebook">GB0484</TD><TD class="td_notebook">Kan</TD></TR><TR class="tr_notebook"><TD class="td_notebook">pP35s</TD><TD class="td_notebook">GB0030</TD><TD class="td_notebook">Amp</TD></TR><TR class="tr_notebook"><TD class="td_notebook">pT35s</TD><TD class="td_notebook">GB0036</TD><TD class="td_notebook">Amp</TD></TR><TR class="tr_notebook"><TD class="td_notebook">pAtUbq10</TD><TD class="td_notebook">GB0223</TD><TD class="td_notebook">Amp</TD></TR><TR class="tr_notebook"><TD class="td_notebook">pTAtHSP18.2</TD><TD class="td_notebook">GB0035</TD><TD class="td_notebook">Amp</TD></TR><TR class="tr_notebook"><TD class="td_notebook">pTTctp</TD><TD class="td_notebook">GB0081</TD><TD class="td_notebook">Amp</TD></TR><TR class="tr_notebook"><TD class="td_notebook">pUPD</TD><TD class="td_notebook">GB0317</TD><TD class="td_notebook">Amp</TD></TR></TBODY></TABLE><P class="p_notebook">Later we will need:</P><TABLE class="table_notebook"><TBODY><TR class="tr_notebook"><TD class="td_notebook">pDGB2Ω1</TD><TD class="td_notebook">GB0487</TD><TD class="td_notebook">Smp</TD></TR><TR class="tr_notebook"><TD class="td_notebook">pDGB2Ω2</TD><TD class="td_notebook">GB0488</TD><TD class="td_notebook">Smp</TD></TR></TBODY></TABLE><P class="p_notebook">Prepare plaques with antibiotics Kan, Spm, Amp</P><H4 class="date_notebook">06/25/2014</H4><P class="p_notebook">Grow the selected pieces from the GB collection in liquid medium (performed in laminar air flow cabinet).</P><H4 class="date_notebook">06/26/2014</H4><P class="p_notebook">Culture in agar Petri dish. 2 plaques: Amp and Kan.</P><P class="p_notebook">Minipreps with EZNA Plasmid DNA MiniKit I.</P><P class="p_notebook">Expected digestions:</P><TABLE class="table_notebook"><TBODY><TR class="tr_notebook"><TD class="td_notebook">pP35s </TD><TD class="td_notebook">GB0030</TD><TD class="td_notebook">NotI</TD><TD class="td_notebook">Buffer: Orange</TD><TD class="td_notebook">2981, 1105</TD></TR><TR class="tr_notebook"><TD class="td_notebook">pT35s </TD><TD class="td_notebook">GB0036</TD><TD class="td_notebook">NotI</TD><TD class="td_notebook">Buffer: Orange</TD><TD class="td_notebook">2981, 304</TD></TR><TR class="tr_notebook"><TD class="td_notebook">pAtUbq10 </TD><TD class="td_notebook">GB0223</TD><TD class="td_notebook">NotI</TD><TD class="td_notebook">Buffer: Orange</TD><TD class="td_notebook">2981, 714</TD></TR><TR class="tr_notebook"><TD class="td_notebook">pTAtHSP18.2 </TD><TD class="td_notebook">GB0035</TD><TD class="td_notebook">NotI</TD><TD class="td_notebook">Buffer: Orange</TD><TD class="td_notebook">2981, 328</TD></TR><TR class="tr_notebook"><TD class="td_notebook">pTTctp </TD><TD class="td_notebook">GB0081</TD><TD class="td_notebook">NotI</TD><TD class="td_notebook">Buffer: Orange</TD><TD class="td_notebook">2981, 487</TD></TR></TBODY></TABLE><P class="p_notebook">Electrophoresis analysis.</P><P class="p_notebook">We got the expected bands in all cases.</P><H4 class="date_notebook">07/01/2014</H4><P class="p_notebook">AtrΔ11 amplification by PCR with primers that contain extra nucleotides to introduce them in the sequence. </P><P class="p_notebook">We made a PCR amplification using the AtrΔ11 gene as a template and the oligos: R +F</P><P class="p_notebook">Reagents needed:</P><UL class="ul_notebook"><LI>32.5 μL of H2O miliQ</LI><LI>10 μL HF buffer </LI><LI>2 μL dNTPs</LI><LI>2.5 μL Reverse primer</LI><LI>2.5 μL Forward primer</LI><LI>1 μL template (AtrΔ11 gene)</LI><LI>0.5 μL phusion (polimerase)</LI></UL><P class="p_notebook">PCR parameters: The annealing temperature was 60°C and the extension temperature was 65°C. </P><P class="p_notebook">Electrophoresis performed to check the PCR product, which was expected to be around 1 kb. </P><P class="p_notebook">pUPD ligation of EaDAcT, HarFar and AtrΔ11.</P><P class="p_notebook">Reagents needed for the reaction of ligation:</P><UL class="ul_notebook"><LI>1 μL pUPD</LI><LI>1 μL PCR product/gblock product  </LI><LI>1.2 μL buffer 10x</LI><LI>1 μL BsmBI</LI><LI>1 μL T4 ligase</LI><LI>6.8 μL H2O miliQ</LI></UL><P class="p_notebook">Vfinal= 12 μL</P><P class="p_notebook">Termocycler parameters: The ligase temperature was 16°C and the BsmBI temperature was 37°C. </P><P class="p_notebook">As a result, there are obtained three different pUPD plasmids containing the genes EaDAcT, HarFAR and AtrΔ11.</P><H4 class="date_notebook">07/02/2014</H4><P class="p_notebook"><I>E. coli</I> transformation. This step is performed in a laminar air flow cabinet (LAF). We have used an electrocompetent <I>E. coli</I> strain (DH5α) and a sample from each product of ligation made in the previous step (three pUPD plasmids, each of them containing one of the three genes), so transformation is made three times.</P><P class="p_notebook">Reagents needed:</P><UL class="ul_notebook"><LI><I>E. coli</I> aliquot</LI><LI>1.5 μL of ligation in pUPD (for each gene: EaDAcT, HarFAR, AtrΔ11)</LI></UL><P class="p_notebook">Each mix is introduced in a electroporation vial and electroporated at 1500 V, then 300 μL of SOC are added to each vial. All of them were incubated at 37°C for 1 hour.</P><P class="p_notebook">After incubation, culture in Petri plates (always in a LAF).</P><P class="p_notebook">2 cell-culture dishes per transformation (with Ampicillin), one with 50 μL and the other with the remaining volume. </P><P class="p_notebook">Petri plates are incubated at 37°C for 16 h.</P><H4 class="date_notebook">07/03/2014</H4><P class="p_notebook">Transformed colonies selection. The white ones are recultured in liquid medium. One colony of each transformation is picked and cultured in 3.5 mL LB and 7 μL Amp. This step is repeated three times:</P><UL class="ul_notebook"><LI>3x 1 colony of EaDAcT in pUPD + LB + Amp</LI><LI>3x 1 colony of HarFAR in pUPD + LB + Amp</LI><LI>3x 1 colony of AtrΔ11 in pUPD + LB + Amp</LI></UL><P class="p_notebook">All tubes are incubated at 37°C overnight in agitation.</P><H4 class="date_notebook">07/04/2014</H4><P class="p_notebook">Digestions in silico using Vector NTI to check after minipreps if ligations are correct.</P><TABLE class="table_notebook"><TBODY><TR class="tr_notebook"><TD class="td_notebook">EaDAcT</TD><TD class="td_notebook">NotI</TD><TD class="td_notebook">2981, 1167</TD></TR><TR class="tr_notebook"><TD class="td_notebook">RsaI</TD><TD class="td_notebook">1876, 1343, 532, 306, 91</TD></TR><TR class="tr_notebook"><TD class="td_notebook">HarFAR</TD><TD class="td_notebook">NotI</TD><TD class="td_notebook">2981, 1440</TD></TR><TR class="tr_notebook"><TD class="td_notebook">PvuII</TD><TD class="td_notebook">2564, 1394, 463</TD></TR><TR class="tr_notebook"><TD class="td_notebook">AtrΔ11</TD><TD class="td_notebook">NotI</TD><TD class="td_notebook">2981, 1056</TD></TR><TR class="tr_notebook"><TD class="td_notebook">BanII</TD><TD class="td_notebook">2570, 803, 351, 314</TD></TR></TBODY></TABLE><P class="p_notebook">Digestion reagents:</P><UL class="ul_notebook"><LI>0.5 μL restriction enzyme</LI><LI>2.5 μL buffer</LI><LI>21 μL H20 (miliQ)</LI><LI>1 μL sample</LI></UL><P class="p_notebook">Preparation of master mixes</P><UL class="ul_notebook"><LI>Master mix for NotI</LI><UL class="ul_ul_notebook"><LI>5 μL NotI</LI><LI>25 μL Orange</LI><LI>210 μL H20</LI></UL><LI>Master mix for RsaI</LI><UL class="ul_ul_notebook"><LI>1.5 μL RsaI</LI><LI>7.5 μL Tango</LI><LI>63 μL H20</LI></UL><LI>Master mix for PvuII</LI><UL class="ul_ul_notebook"><LI>1.5 μL PvuII</LI><LI>7.5 μL Green</LI><LI>63 μL H20</LI></UL><LI>Master mix for BanII</LI><UL class="ul_ul_notebook"><LI>1.5 μL BanII</LI><LI>7.5 μL Tango</LI><LI>63 μL H20</LI></UL></UL><P class="p_notebook">Perform electrophoresis to check if the size of the fragments from the digestions is correct.</P><P class="p_notebook">Comments:</P><UL class="ul_notebook"><LI>We picked blue colonies instead of white by mistake. We need to pick colonies again but this time make sure we pick white colonies.</LI><LI>For the repetition we must find another enzyme instead of BanII as we found out that it doesn't cut very well.</LI></UL><H4 class="date_notebook">07/06/2014</H4><P class="p_notebook">We picked again 3 colonies for each construction, and we made sure that we picked the WHITE ones. We cultivated them in a &quot;double check&quot; (name invented by us)  liquid medium. Those tubes contain:</P><UL class="ul_notebook"><LI>3.5 mL LB</LI><LI>7 μL Amp</LI><LI>7 μL X-Gal</LI><LI>3.5 μL IPTG (turns the tube blue if the colonies picked were blue)</LI></UL><H4 class="date_notebook">07/07/2014</H4><P class="p_notebook">We made minipreps of yesterday's culture. Thanks to our &quot;double check&quot; medium we found which colonies were well picked. Finally we had minipreps of tubes HarFAR 1, 2, 3; EaDAcT 3 and AtrΔ11 2, 3.</P><P class="p_notebook">Once we had the minipreps, we perform the digestions to check which were correct and send them to sequencing. This time we selected RsaI instead of BanII. The in silico digestions were as follows.</P><TABLE class="table_notebook"><TBODY><TR class="tr_notebook"><TD class="td_notebook">EaDAcT</TD><TD class="td_notebook">NotI</TD><TD class="td_notebook">2981, 1167</TD></TR><TR class="tr_notebook"><TD class="td_notebook">RsaI</TD><TD class="td_notebook">1876, 1343, 532, 306, 91</TD></TR><TR class="tr_notebook"><TD class="td_notebook">HarFAR</TD><TD class="td_notebook">NotI</TD><TD class="td_notebook">2981, 1440</TD></TR><TR class="tr_notebook"><TD class="td_notebook">PvuII</TD><TD class="td_notebook">2564, 1394, 463</TD></TR><TR class="tr_notebook"><TD class="td_notebook">AtrΔ11</TD><TD class="td_notebook">NotI</TD><TD class="td_notebook">2981, 1056</TD></TR><TR class="tr_notebook"><TD class="td_notebook">RsaI</TD><TD class="td_notebook">1879, 1310, 467, 327, 54</TD></TR></TBODY></TABLE><P class="p_notebook">Preparation of master mixes</P><UL class="ul_notebook"><LI>Master mix for NotI</LI><UL class="ul_ul_notebook"><LI>3.5 μL NotI</LI><LI>17.5 μL Orange</LI><LI>147 μL H20</LI></UL><LI>Master mix for RsaI</LI><UL class="ul_ul_notebook"><LI>2 μL RsaI</LI><LI>10 μL Tango</LI><LI>84 μL H20</LI></UL><LI>Master mix for PvuII</LI><UL class="ul_ul_notebook"><LI>2 μL PvuII</LI><LI>10 μL Green</LI><LI>84 μL H20</LI></UL></UL><P class="p_notebook">We run the electrophoresis gel to check if this time we have done it correctly.</P><P class="p_notebook">Everything was OK. We sent AtrΔ11 (3), HarFAR (3) and EaDAcT (3) to sequence.</P><H4 class="date_notebook">07/08/2014</H4><P class="p_notebook">Now, while we wait for sequencing results, we go on as they were going to be correct in order to save time.</P><P class="p_notebook">The next step is to build a transciptional unit (TU) with our sequences. A transcriptional unit is a structure composed by promoter, coding sequence (CDS) and terminator in an α or Ω vector.</P><P class="p_notebook">Reagents needed for ligation:</P><UL class="ul_notebook"><LI>1 μL promoter 75 ng/μL</LI><LI>1 μL terminator 75 ng/μL</LI><LI>1 μL CDS 75 ng/μL</LI><LI>1 μL vector α</LI><LI>1.2 μL ligase buffer 10x</LI><LI>1 μL T4</LI><LI>1 μL BsaI</LI><LI>4.2 μL H20</LI></UL><P class="p_notebook">Total: 12 μL</P><P class="p_notebook">Take into account that if we want to make binary constructions later (merge 2 TU in a same vector), we need to clone each TU in a different α vector.</P><P class="p_notebook">Strategy Promoter-Terminator:</P><TABLE class="table_notebook"><TBODY><TR class="tr_notebook"><TD class="td_notebook">AtrΔ11</TD><TD class="td_notebook">P35s</TD><TD class="td_notebook">T35s</TD><TD class="td_notebook">40.41</TD></TR><TR class="tr_notebook"><TD class="td_notebook">HarFAR</TD><TD class="td_notebook">P35s</TD><TD class="td_notebook">TatHSP</TD><TD class="td_notebook">39.68</TD></TR><TR class="tr_notebook"><TD class="td_notebook">EaDAcT</TD><TD class="td_notebook">PAtUbq</TD><TD class="td_notebook">TatHSP</TD><TD class="td_notebook">32.27</TD></TR></TBODY></TABLE><P class="p_notebook">Adjust concentrations to 75 ng/μL for ligation reaction</P><P class="p_notebook">Initial concentrations (nanodrop):</P><TABLE class="table_notebook"><TBODY><TR class="tr_notebook"><TD class="td_notebook">Piece</TD><TD class="td_notebook">Concentrations</TD><TD class="td_notebook">Volume</TD><TD class="td_notebook">Volume of H20 to add</TD></TR><TR class="tr_notebook"><TD class="td_notebook">PAtUpb</TD><TD class="td_notebook">442.6 ng/μL</TD><TD class="td_notebook">34 μL</TD><TD class="td_notebook">166.6 μL</TD></TR><TR class="tr_notebook"><TD class="td_notebook">pTatHSP</TD><TD class="td_notebook">235.4 ng/μL</TD><TD class="td_notebook">36 μL</TD><TD class="td_notebook">77 μL</TD></TR><TR class="tr_notebook"><TD class="td_notebook">T35s</TD><TD class="td_notebook">194.9 ng/μL</TD><TD class="td_notebook">37.5 μL</TD><TD class="td_notebook">60 μL</TD></TR><TR class="tr_notebook"><TD class="td_notebook">P35s</TD><TD class="td_notebook">454.7 ng/μL</TD><TD class="td_notebook">36 μL</TD><TD class="td_notebook">182 μL</TD></TR><TR class="tr_notebook"><TD class="td_notebook">2α1</TD><TD class="td_notebook">57.1 ng/μL</TD><TD class="td_notebook">-</TD><TD class="td_notebook">We will need to put 1.5 μL of this one</TD></TR><TR class="tr_notebook"><TD class="td_notebook">2α2</TD><TD class="td_notebook">104.0 ng/μL</TD><TD class="td_notebook">38 μL</TD><TD class="td_notebook">14.7 μL</TD></TR><TR class="tr_notebook"><TD class="td_notebook">AtrΔ11</TD><TD class="td_notebook">359.3 ng/μL</TD><TD class="td_notebook">20 μL</TD><TD class="td_notebook">75.8 μL</TD></TR><TR class="tr_notebook"><TD class="td_notebook">HarFAR</TD><TD class="td_notebook">404.4 ng/μL</TD><TD class="td_notebook">15 μL</TD><TD class="td_notebook">65.9 μL</TD></TR><TR class="tr_notebook"><TD class="td_notebook">EaDAcT</TD><TD class="td_notebook">155.6 ng/μL</TD><TD class="td_notebook">10 μL</TD><TD class="td_notebook">10.7 μL</TD></TR></TBODY></TABLE><P class="p_notebook">Ligation reaction</P><UL class="ul_notebook"><LI>P35s:AtrΔ11:T35s in 2α1</LI><UL class="ul_ul_notebook"><LI>1 μL P35s</LI><LI>1 μL T35s</LI><LI>1 μL AtrΔ11</LI><LI>1.5 μL 2α1</LI><LI>1.2 μL ligase buffer 10x</LI><LI>1 μL T4</LI><LI>1 μL BsaI</LI><LI>3.7 μL H20</LI></UL></UL><UL class="ul_notebook"><LI>P35s:HarFAR:TatHSP in 2α2</LI><UL class="ul_ul_notebook"><LI>1 μL P35s</LI><LI>1 μL TatHSP</LI><LI>1 μL HarFAR</LI><LI>1 μL 2α2</LI><LI>1.2 μL ligase buffer 10x</LI><LI>1 μL T4</LI><LI>1 μL BsaI</LI><LI>4.2 μL H20</LI></UL></UL><UL class="ul_notebook"><LI>PAtUbq:EaDAcT:TatHSP in 2α2</LI><UL class="ul_ul_notebook"><LI>1 μL PAtUbq</LI><LI>1 μL TatHSP</LI><LI>1 μL EaDAcT</LI><LI>1 μL 2α2</LI><LI>1.2 μL ligase buffer 10x</LI><LI>1 μL T4</LI><LI>1 μL BsaI</LI><LI>4.2 μL H20</LI></UL></UL><H4 class="date_notebook">07/09/2014</H4><P class="p_notebook">Transformation of constructions in <I>E. coli</I></P><P class="p_notebook">We finally got the sequencing results from 07/07/2014.</P><UL class="ul_notebook"><LI>Mutation in AtrΔ11 -&gt; We threw away the colonies and transformed cells. We picked again white colonies.</LI><LI>HarFAR -&gt; Sequencing correct</LI><LI>EaDAcT -&gt; Synonim mutation in 601 (A -&gt; T). This is a gBlock!</LI></UL><P class="p_notebook">We took vectors 2Ω1 (GB0487) and 2Ω2 (GB0488) parts from the GB colection.</P><UL class="ul_notebook"><LI>Worked in the LAF</LI><LI>Cultivated in a Petri dish with Spm</LI><LI>Let them grow for one day</LI></UL><P class="p_notebook">Cultivate transformated cells in two Kan plaques (Kan matches vector α)</P><UL class="ul_notebook"><LI>50 mL transformation in one plaque</LI><LI>Rest of the culture in another (250 μL aprox)</LI><LI>Let them grow for one day</LI></UL><H4 class="date_notebook">07/10/2014</H4><P class="p_notebook">Pick colonies and grow them in liquid medium.</P><UL class="ul_notebook"><LI>6 from AtrΔ11 (repetition because of mutation)</LI><UL class="ul_ul_notebook"><LI>3.5 mL LB</LI><LI>7 μL Amp</LI><LI>7 μL X-gal</LI><LI>3.5 μL IPTG</LI></UL><LI>1 colony from 2Ω1</LI><UL class="ul_ul_notebook"><LI>3.5 mL LB</LI><LI>7 μL Spm</LI></UL></UL><UL class="ul_notebook"><LI>1 colony from 2Ω2</LI><UL class="ul_ul_notebook"><LI>3.5 mL LB</LI><LI>7 μL Spm</LI></UL><LI>3 colonies from P35s:HarFAR:TatHSP</LI><UL class="ul_ul_notebook"><LI>3.5 mL LB</LI><LI>7 μL Kan</LI></UL><LI>3 colonies from PAtUbq:EaDAcT:TatHSP</LI><UL class="ul_ul_notebook"><LI>3.5 mL LB</LI><LI>7 μL Kan</LI></UL></UL><P class="p_notebook">Grow at 37°C in agitation overnight.</P><P class="p_notebook">We have checked the promoters and terminators are both compatible with GB and BioBricks.</P><P class="p_notebook">Only P35s and T35s work for both. pPnos could also work.</P><P class="p_notebook">Pick 3 colonies of P35s:HarFAR:THsp and PAtUbq:EaDAcT:THsp. Culture in liquid medium with Kan.</P><H4 class="date_notebook">07/11/2014</H4><P class="p_notebook">We made minipreps of yesterday's liquid culture. Thanks to our &quot;double check&quot; medium we found which colonies were well picked. Finally we had minipreps of tubes AtrΔ11 3 and 6; 2Ω1; 2Ω2; constructions P35s:HarFAR:TatHSP 1, 2, 3 and PAtUbq:EaDAcT:TatHSP 1, 2, 3.</P><P class="p_notebook">Additionally, we have cultured each of the colonies named above to store them.</P><H4 class="date_notebook">07/14/2014</H4><P class="p_notebook">We tested the minipreps made last friday by digestion. Once they were checked, we send the correct ones to sequencing. The in silico digestions were as follows.</P><TABLE class="table_notebook"><TBODY><TR class="tr_notebook"><TD class="td_notebook">Parts</TD><TD class="td_notebook">Retriction enzyme</TD><TD class="td_notebook">Expected Bands</TD></TR><TR class="tr_notebook"><TD class="td_notebook">PAtUbq:EaDAcT:TatHSP in 2α2</TD><TD class="td_notebook">HindIII</TD><TD class="td_notebook">6322, 1722, 736, 221</TD></TR><TR class="tr_notebook"><TD class="td_notebook">P35s:HarFAR:TatHSP in 2 α2</TD><TD class="td_notebook">HindIII</TD><TD class="td_notebook">6322, 1794, 221</TD></TR><TR class="tr_notebook"><TD class="td_notebook">AtrΔ11</TD><TD class="td_notebook">NotI</TD><TD class="td_notebook">2961, 1056</TD></TR><TR class="tr_notebook"><TD class="td_notebook">2Ω1</TD><TD class="td_notebook">BamHI</TD><TD class="td_notebook">6652, 382, 239</TD></TR><TR class="tr_notebook"><TD class="td_notebook">2Ω2</TD><TD class="td_notebook">EcoRV</TD><TD class="td_notebook">6652, 621</TD></TR></TBODY></TABLE><P class="p_notebook">Preparation of master mixes</P><UL class="ul_notebook"><LI>Master mix for HindIII</LI><UL class="ul_ul_notebook"><LI>3.5 μL HindIII</LI><LI>17.5 μL Red</LI><LI>147 μL H20</LI></UL><LI>Master mix for NotI</LI><UL class="ul_ul_notebook"><LI>1.5 μL NotI</LI><LI>7.5 μL Orange</LI><LI>63 μL H20</LI></UL><LI>Mix for EcoRV</LI><UL class="ul_ul_notebook"><LI>0.5 μL EcoRV</LI><LI>2.5 μL Red</LI><LI>21 μL H20</LI></UL><LI>Mix for BamHI</LI><UL class="ul_ul_notebook"><LI>0.5 μL PvuII</LI><LI>2.5 μL Green</LI><LI>21 μL H20</LI></UL></UL><P class="p_notebook">We run the electrophoresis gel to check if this time we have done it correctly.</P><P class="p_notebook">Everything was OK except the AtrΔ11 (3), which had some partial digestion. It was the reason we sent AtrΔ11 (6) to sequence.</P><H4 class="date_notebook">07/15/2014</H4><P class="p_notebook">We got the sequencing results from yesterday and everything was OK, so we made the transcriptional units ligation. </P><P class="p_notebook">Reagents needed for the reaction of ligation (Total volume = 12 μL):</P><UL class="ul_notebook"><LI>P35s:AtrΔ11:T35s in 2α1</LI><UL class="ul_ul_notebook"><LI>1 μL P35s</LI><LI>1 μL T35s</LI><LI>1 μL AtrΔ11</LI><LI>1.5 μL 2α1</LI><LI>1.2 μL ligase buffer 10x</LI><LI>1 μL T4</LI><LI>1 μL BsaI</LI><LI>3.7 μL H20</LI></UL><LI>P35s:HarFAR:T35s in 2α2</LI><UL class="ul_ul_notebook"><LI>1 μL P35s</LI><LI>1 μL T35s</LI><LI>1 μL HarFAR</LI><LI>1 μL 2α2</LI><LI>1.2 μL ligase buffer 10x</LI><LI>1 μL T4</LI><LI>1 μL BsaI</LI><LI>4.2 μL H20</LI></UL><LI>P35s:EaDAcT:T35s in 2α2</LI><UL class="ul_ul_notebook"><LI>1 μL P35s</LI><LI>1 μL T35s</LI><LI>1 μL EaDAcT</LI><LI>1 μL 2α2</LI><LI>1.2 μL ligase buffer 10x</LI><LI>1 μL T4</LI><LI>1 μL BsaI</LI><LI>4.2 μL H20</LI></UL></UL><P class="p_notebook">Note: Concentrations were previously adjusted to 75 ng/μL. Only the AtrΔ11 was adjusted from 250.2 ng/μL.</P><P class="p_notebook">Finally, we prepared liquid cultures in order to store in glicerol. The tubes we used and their respective antibiotics were:</P><UL class="ul_notebook"><LI>Amp</LI><UL class="ul_ul_notebook"><LI>pAtrΔ11 (6)</LI><LI>pEaDAcT (3)</LI><LI>pHarFAR (3)</LI></UL><LI>Kan</LI><UL class="ul_ul_notebook"><LI>P35:HarFAR:TatHSP in 2α2 (3)</LI><LI>PPAtUbq:EaDAcT:TatHSP in 2apha2 (3)</LI></UL></UL><H4 class="date_notebook">07/16/2014</H4><P class="p_notebook">Storage in glycerol of the HarFAR (GB1018), AtrΔ11 (GB1019), EaDAcT (GB1020), P35s:HarFAR:TatHSP in 2α2 (GB1021) and PAtUbq:EaDAcT:TatHSP in 2α2 (GB1022). We made 3 tubes: one for us, one for the GB collenction and one for reserve. </P><P class="p_notebook">The procedure is to mix 700 μL of culture with 300 μL of glycerol 50%, spin it and store it in the -80°C.</P><H4 class="date_notebook">07/17/2014</H4><P class="p_notebook">Pick 3 colonies of P35s:AtrΔ11:T35s, P35s:HarFAR:T35s and P35s:EaDAcT:T35s. Culture in liquid medium with Kan.</P><P class="p_notebook">Digestions in silico.</P><TABLE class="table_notebook"><TBODY><TR class="tr_notebook"><TD class="td_notebook">Transcriptional units</TD><TD class="td_notebook">Restriction enzymes</TD><TD class="td_notebook">Expected bands</TD></TR><TR class="tr_notebook"><TD class="td_notebook">P35s:AtrΔ11:T35s</TD><TD class="td_notebook">EcoRI</TD><TD class="td_notebook">6323, 2269</TD></TR><TR class="tr_notebook"><TD class="td_notebook">NcoI</TD><TD class="td_notebook">390, 8202</TD></TR><TR class="tr_notebook"><TD class="td_notebook">P35s:HarFAR:T35s</TD><TD class="td_notebook">HindIII</TD><TD class="td_notebook">933, 6322, 1722</TD></TR><TR class="tr_notebook"><TD class="td_notebook">NcoI</TD><TD class="td_notebook">8587, 390</TD></TR><TR class="tr_notebook"><TD class="td_notebook">P35s:EaDAcT:T35s</TD><TD class="td_notebook">HindIII</TD><TD class="td_notebook">6322, 2366</TD></TR><TR class="tr_notebook"><TD class="td_notebook">EcoRV</TD><TD class="td_notebook">683, 8021</TD></TR></TBODY></TABLE><P class="p_notebook">Preparation of reagents needed for genomic extraction of <I>Candida tropicalis</I> for FAO1.</P><H4 class="date_notebook">07/18/2014</H4><P class="p_notebook">Mistake in P35s:AtrΔ11:T35s, P35s:HarFAR:T35s and P35s:EaDAcT:T35s minipreps. Repeat tomorrow.</P><H4 class="date_notebook">07/19/2014</H4><P class="p_notebook">Minipreps of P35s:AtrΔ11:T35s, P35s:HarFAR:T35s and P35s:EaDAcT:T35s. Concentration measuments with nanodrop.</P><TABLE class="table_notebook"><TBODY><TR class="tr_notebook"><TD class="td_notebook">Transcriptional unit </TD><TD class="td_notebook">DNA concentration</TD></TR><TR class="tr_notebook"><TD class="td_notebook">P35s:AtrΔ11:T35s (1)</TD><TD class="td_notebook">164 ng/μL</TD></TR><TR class="tr_notebook"><TD class="td_notebook">P35s:AtrΔ11:T35s (2)</TD><TD class="td_notebook">168 ng/μL</TD></TR><TR class="tr_notebook"><TD class="td_notebook">P35s:AtrΔ11:T35s (3)</TD><TD class="td_notebook">147.4 ng/μL</TD></TR><TR class="tr_notebook"><TD class="td_notebook">P35s:HarFAR:T35s (1)</TD><TD class="td_notebook">125.3 ng/μL</TD></TR><TR class="tr_notebook"><TD class="td_notebook">P35s:HarFAR:T35s (2)</TD><TD class="td_notebook">114.5 ng/μL</TD></TR><TR class="tr_notebook"><TD class="td_notebook">P35s:HarFAR:T35s (3)</TD><TD class="td_notebook">140.3 ng/μL</TD></TR><TR class="tr_notebook"><TD class="td_notebook">P35s:EaDAcT:T35s (1)</TD><TD class="td_notebook">144.2 ng/μL</TD></TR><TR class="tr_notebook"><TD class="td_notebook">P35s:EaDAcT:T35s (2)</TD><TD class="td_notebook">137.9 ng/μL</TD></TR><TR class="tr_notebook"><TD class="td_notebook">P35s:EaDAcT:T35s (3)</TD><TD class="td_notebook">128.5 ng/μL</TD></TR><TR class="tr_notebook"><TD class="td_notebook">Stuffer fragment</TD><TD class="td_notebook">135.5 ng/μL</TD></TR><TR class="tr_notebook"><TD class="td_notebook">2Ω1</TD><TD class="td_notebook">196.8 ng/μL</TD></TR><TR class="tr_notebook"><TD class="td_notebook">2Ω2</TD><TD class="td_notebook">175.4 ng/μL</TD></TR></TBODY></TABLE><P class="p_notebook">Digestions of P35s:AtrΔ11:T35s, P35s:HarFAR:T35s and P35s:EaDAcT:T35s and gel electrophoresis to check if transciptional units have been assembled OK.</P><P class="p_notebook">All digestions were correct except P35s:EaDAcT:T35s (2).</P><P class="p_notebook">Ligation in Ω vectors.</P><UL class="ul_notebook"><LI>P35s:AtrΔ11:T35s + P35s:HarFAR:T35s in 2Ω1</LI><UL class="ul_ul_notebook"><LI>1 μL P35s:AtrΔ11:T35s (75 ng/μL)</LI><LI>1 μL P35s:HarFAR:T35s (75 ng/μL)</LI><LI>1 μL 2Ω1 (75 ng/μL)</LI><LI>1 μL BsmBI (5-10 ud)</LI><LI>1 μL T4 ligase (5-10 ud)</LI><LI>1 μL buffer ligase (3 ud)</LI><LI>4 μL H20</LI></UL><LI>P35s:EaDAcT:T35s in 2Ω2</LI><UL class="ul_ul_notebook"><LI>1 μL stuffer fragment (75 ng/μL)</LI><LI>1 μL P35s:EaDAcT:T35s (75 ng/μL)</LI><LI>1 μL 2Ω2 (75 ng/μL)</LI><LI>1 μL BsmBI (5-10 ud)</LI><LI>1 μL T4 ligase (5-10 ud)</LI><LI>1 μL buffer ligase (3 ud)</LI><LI>4 μL H20</LI></UL></UL><P class="p_notebook">Set the reaction: 25 cycles x (37°C 2 min, 16°C 5 min).</P><P class="p_notebook">Omega vectors include a resistance to spectinomycin.</P><H4 class="date_notebook">07/20/2014</H4><P class="p_notebook">Transform and grow in Petri dishes yesterday's ligations: P35S:AtrΔ11:T35S + P35S:HarFAR:T35S in 2Ω1 and P35S:EaDAcT:T35S in 2Ω2.</P><H4 class="date_notebook">07/21/2014</H4><P class="p_notebook">Pick colonies of P35S:AtrΔ11:T35S + P35S:HarFAR:T35S in 2Ω1 (3) and P35S:EaDAcT:T35S in 2Ω2 (2).</P><H4 class="date_notebook">07/22/2014</H4><P class="p_notebook">We made minipreps of yesterday's liquid culture. Selected tubes: </P><UL class="ul_notebook"><LI>P35S:AtrΔ11:T35S + P35S:HarFAR:T35S in 2Ω1(Tubes 1, 2 and 3)</LI><LI>P35S:EaDAcT:T35S in 2Ω2 (Tubes 1 and 2)</LI></UL><P class="p_notebook">Digestions in silico made to check the transcriptional units:</P><TABLE class="table_notebook"><TBODY><TR class="tr_notebook"><TD class="td_notebook">Transcriptional units</TD><TD class="td_notebook">Restriction enzyme</TD><TD class="td_notebook">Expected bands</TD></TR><TR class="tr_notebook"><TD class="td_notebook">P35S:AtrΔ11:T35S+P35S:HarFAR:T35S in 2Ω1</TD><TD class="td_notebook">EcoRV</TD><TD class="td_notebook">9307, 2251</TD></TR><TR class="tr_notebook"><TD class="td_notebook">BamHI</TD><TD class="td_notebook">6652, 4906</TD></TR><TR class="tr_notebook"><TD class="td_notebook">P35S:EaDAcT:T35S in 2Ω2</TD><TD class="td_notebook">EcoRV</TD><TD class="td_notebook">6652, 1044, 817, 683</TD></TR><TR class="tr_notebook"><TD class="td_notebook">NcoI</TD><TD class="td_notebook">8806, 390</TD></TR></TBODY></TABLE><P class="p_notebook">Digestion master mixes:</P><UL class="ul_notebook"><LI>Master mix for NotI</LI><UL class="ul_ul_notebook"><LI>1.5 μL NotI</LI><LI>7.5 μL Orange buffer</LI><LI>63 μL H20</LI></UL><LI>Master mix for NcoI</LI><UL class="ul_ul_notebook"><LI>1.5 μL NcoI</LI><LI>7.5 μL Tango buffer</LI><LI>63 μL H20</LI></UL><LI>Master mix for BamHI</LI><UL class="ul_ul_notebook"><LI>2 μL BamHI</LI><LI>10 μL Green buffer</LI><LI>84 μL H20</LI></UL><LI>Master mix for EcoRV</LI><UL class="ul_ul_notebook"><LI>4 μL EcoRV</LI><LI>20 μL Red buffer</LI><LI>168 μL H20</LI></UL></UL><P class="p_notebook">Note: Trichome promoter digestion preparation included. </P><P class="p_notebook">All digestions were correct except the transcriptional unit of EaDAcT in 2Ω2 (P35s:EaDAcT:T35S). </P><P class="p_notebook">Miniprep quantification:</P><TABLE class="table_notebook"><TBODY><TR class="tr_notebook"><TD class="td_notebook">Piece</TD><TD class="td_notebook">Tube</TD><TD class="td_notebook">Concentration (ng/μL)</TD><TD class="td_notebook">Volume (μL)</TD></TR><TR class="tr_notebook"><TD class="td_notebook">P35S:EaDAcT:T35S in 2Ω2</TD><TD class="td_notebook">1</TD><TD class="td_notebook">350.7</TD><TD class="td_notebook">33</TD></TR><TR class="tr_notebook"><TD class="td_notebook">P35S:EaDAcT:T35S in 2Ω2</TD><TD class="td_notebook">2</TD><TD class="td_notebook">271.1</TD><TD class="td_notebook">33</TD></TR><TR class="tr_notebook"><TD class="td_notebook">P35S:AtrΔ11:T35S + P35S:HarFAR:T35S in 2Ω1</TD><TD class="td_notebook">1</TD><TD class="td_notebook">306.3</TD><TD class="td_notebook">31</TD></TR><TR class="tr_notebook"><TD class="td_notebook">P35S:AtrΔ11:T35S + P35S:HarFAR:T35S in 2Ω1</TD><TD class="td_notebook">2</TD><TD class="td_notebook">296.6</TD><TD class="td_notebook">28</TD></TR><TR class="tr_notebook"><TD class="td_notebook">P35S:AtrΔ11:T35S + P35S:HarFAR:T35S in 2Ω1</TD><TD class="td_notebook">3</TD><TD class="td_notebook">246.0</TD><TD class="td_notebook">33</TD></TR></TBODY></TABLE><P class="p_notebook">All of the pieces named above were adjusted at 75 ng/μL.</P><TABLE class="table_notebook"><TBODY><TR class="tr_notebook"><TD class="td_notebook">Piece </TD><TD class="td_notebook">Tube number</TD><TD class="td_notebook">Final Volume (μL)</TD><TD class="td_notebook">Volume to be added (μL)</TD></TR><TR class="tr_notebook"><TD class="td_notebook">P35S:EaDAcT:T35S in 2Ω2</TD><TD class="td_notebook">1</TD><TD class="td_notebook">154.30</TD><TD class="td_notebook">121.3</TD></TR><TR class="tr_notebook"><TD class="td_notebook">P35S:EaDAcT:T35S in 2Ω2</TD><TD class="td_notebook">2</TD><TD class="td_notebook">119.30</TD><TD class="td_notebook">86.30</TD></TR><TR class="tr_notebook"><TD class="td_notebook">P35S:AtrΔ11:T35S + P35S:HarFAR:T35S in 2Ω1</TD><TD class="td_notebook">1</TD><TD class="td_notebook">126.60</TD><TD class="td_notebook">95.60</TD></TR><TR class="tr_notebook"><TD class="td_notebook">P35S:AtrΔ11:T35S + P35S:HarFAR:T35S in 2Ω1</TD><TD class="td_notebook">2</TD><TD class="td_notebook">110.70</TD><TD class="td_notebook">82.70</TD></TR><TR class="tr_notebook"><TD class="td_notebook">P35S:AtrΔ11:T35S + P35S:HarFAR:T35S in 2Ω1</TD><TD class="td_notebook">3</TD><TD class="td_notebook">108.24</TD><TD class="td_notebook">75.20</TD></TR></TBODY></TABLE><P class="p_notebook">As the digestions of the transcriptional unit (TU) of EaDAcT were incorrect, we repeated the process from the ligation step. </P><P class="p_notebook">We transformed the same TU in a <I>E. coli</I> competent strain (DH5α). Then, the transformants were cultured in LB media and Spm and stored at 37°C overnight. </P><P class="p_notebook">Finally, in order to obtain the FAO1 gene, we want to extract the <I>Candida tropicalis</I> genome, so we have picked a colony of <I>C. tropicalis</I>. To check the extraction protocol, we used a yeast previously tested, <I>Saccharomyces cerevisiae</I>. We have cultured <I>C. tropicalis</I> in YPD media and <I>S. cerevisiae</I> in YPDA media at 28 °C (5 mL).</P><H4 class="date_notebook">07/23/2014</H4><P class="p_notebook">Candida genome extraction</P><P class="p_notebook"><I>Saccharomyces cerevisiae</I> is used as a control in order to see if we followed the protocol correctly. We aren't really sure if this protocol is going to work in Candida.</P><P class="p_notebook">Cultures measured at 600 nm:</P><UL class="ul_notebook"><LI><I>S. cerevisiae</I>	Abs = 1.07 </LI><LI><I>C. tropicalis</I> Abs = 0.39</LI></UL><P class="p_notebook"><I>S. cerevisiae</I> is recultured with new media (1:2) because the previous media was saturated. 2.25 mL of YPD media were mixed with 2.25 mL of <I>S. cerevisiae</I> culture. The mix has to grow at 28 °C until the exponential phase is reached. </P><P class="p_notebook">The absorbance was measured again:</P><UL class="ul_notebook"><LI><I>S. cerevisiae</I>	Abs = 0.52</LI><LI><I>C. tropicalis</I> Abs = 0.40</LI></UL><P class="p_notebook">Buffers needed for the genome extraction were prepared freshly.The genome of both strains of yeast were extracted following the protocol:</P><UL class="ul_notebook"><LI>Grow yeast in 2 or 5 mL YPD to exponential phase. </LI><LI>Collect cells in 1.5 mL eppendorf-cup (centrifugation 20 s, 6000 rpm).</LI><LI>Wash once with 1 mL sterile water.</LI><LI>Resuspend cells in 200 μL protoplast-buffer (100 mM Tris-HCl, pH 7.5, 10 mM EDTA, 1000 units Zymolyase/mL, 10 μL beta-mercaptoethanol/mL; prepare freshly!). Incubate at 37°C for 1-2 h and finally resuspend by turning the cups. </LI><LI>Add 200 μL of Lysis-Mix (0.2 M NaOH, 1% SDS) an mix carefully (Don't vortex!).</LI><LI>Incubate at 65 °C for 20 min and cool inmediately on ice.</LI><LI>Add 200 μL of 5 M KAc (pH 5.4) and mix carefully (Don't vortex!) and incubate 15 min on ice. </LI><LI>Centrifuge (13,000 rpm, 3 min) and transfer supernatant in a fresh cup.</LI><LI>Add 2 μL RNase A (10 mg/mL) and incubate at 37 °C for 30 min.</LI><LI>Add 600 μL isopropanol and mix carefully (Don't vortex!). Incubate at room temperature for 5 min ad centrifuge (13,000 rpm, 30 s). </LI><LI>Remove the supernatant and wash with 70% ethanol (10 min at room temperature). </LI><LI>Centrifuge (13,000 rpm, 30 s) and remove the supernatant.</LI><LI>Dry DNA pellet in a speed-vacuum (not longer than 3 min!) and resuspend in 50 μL TE-buffer. </LI><LI>Store chromosomal DNA at 4 °C (Don't freeze!). Concentration and quality can be checked in an agarose gel (loading 1/10 of the volume).</LI></UL><P class="p_notebook">Genomic quantification:</P><TABLE class="table_notebook"><TBODY><TR class="tr_notebook"><TD class="td_notebook">Organism</TD><TD class="td_notebook">Concentration </TD></TR><TR class="tr_notebook"><TD class="td_notebook"><I>S. cerevisiae</I></TD><TD class="td_notebook">72.2 ng/μL</TD></TR><TR class="tr_notebook"><TD class="td_notebook"><I>C. tropicalis</I></TD><TD class="td_notebook">1397.1 ng/μL</TD></TR></TBODY></TABLE><P class="p_notebook">Electrophoresis made to check the extraction quality was correct. </P><P class="p_notebook">We did not observe genomic from Candida because we used a very diluted sample. We will repeat the gel tomorrow.</P><P class="p_notebook">We picked EaDAcT colonies.</P><H4 class="date_notebook">07/24/2014</H4><P class="p_notebook">The genomic quality of both organisms (<I>C. tropicalis</I> and <I>S. cerevisiae</I>) was checked in an agarose gel again.</P><P class="p_notebook">We got the Candida genome band, however, the Saccharomyces genome band was not present.</P><P class="p_notebook">Additionally, minipreps of the liquid culture made yesterday were made and also recultured in solid agar plate. </P><P class="p_notebook">Miniprep digestions are going to be done tomorrow.</P><H4 class="date_notebook">07/25/2014</H4><P class="p_notebook">Digestions in silico made for checking yesterday's minipreps:</P><TABLE class="table_notebook"><TBODY><TR class="tr_notebook"><TD class="td_notebook">Pieces/TU</TD><TD class="td_notebook">Restriction enzyme</TD><TD class="td_notebook">Expected bands</TD></TR><TR class="tr_notebook"><TD class="td_notebook">P35S:EaDAcT:T35S in 2Ω2</TD><TD class="td_notebook">EcoRV</TD><TD class="td_notebook">6652, 1044, 817, 683</TD></TR><TR class="tr_notebook"><TD class="td_notebook">NcoI</TD><TD class="td_notebook">8806, 390</TD></TR></TBODY></TABLE><P class="p_notebook">Digestion master mixes:</P><UL class="ul_notebook"><LI>Master mix for NotI</LI><UL class="ul_ul_notebook"><LI>2 μL NotI</LI><LI>10 μL Orange buffer</LI><LI>84 μL H20</LI></UL><LI>Master mix for NcoI</LI><UL class="ul_ul_notebook"><LI>2 μL NcoI</LI><LI>10 μL Tango buffer</LI><LI>84 μL H20</LI></UL><LI>Master mix for BglII</LI><UL class="ul_ul_notebook"><LI>2 μL BglII</LI><LI>10 μL Orange buffer</LI><LI>84 μL H20</LI></UL><LI>Master mix for EcoRV</LI><UL class="ul_ul_notebook"><LI>1.5 μL EcoRV</LI><LI>7.5 μL Red buffer</LI><LI>63 μL H20</LI></UL></UL><P class="p_notebook">An agarose gel was made to check the transcriptional unit and the other pieces:</P><P class="p_notebook">All pieces were correct except the TU corresponding to P35:EaDAcT:T35S.</P><H4 class="date_notebook">07/28/2014</H4><P class="p_notebook">Once the <I>Candida tropicalis</I> genome DNA is obtained, the FAO1 gene can be amplified by PCR.</P><P class="p_notebook">PCR reaction reagents:</P><UL class="ul_notebook"><LI>FAO1-PCR1</LI><UL class="ul_ul_notebook"><LI>Genomic	0.5 μL</LI><LI>Buffer HF (5X)	10.0 μL</LI><LI>dNTPs	2.0 μL</LI><LI>Oligo R (JUL06)	2.5 μL</LI><LI>Oligo F (JUL05)	2.5 μL</LI><LI>Phusion polymerase	0.5 μL</LI><LI>H2O	32.0 μL</LI></UL><LI>FAO1-PCR2</LI><UL class="ul_ul_notebook"><LI>Genomic	0.5 μL</LI><LI>Buffer HF (5X)	10.0 μL</LI><LI>dNTPs	2.0 μL</LI><LI>Oligo R (JUL08)	2.5 μL</LI><LI>Oligo F (JUL07)	2.5 μL</LI><LI>Phusion polymerase	0.5 μL</LI><LI>H2O	32.0 μL</LI></UL></UL><P class="p_notebook">Annealing temperatures</P><UL class="ul_notebook"><LI>FAO1-PCR1: 59 °C</LI><LI>FAO1-PCR2: 64 °C</LI></UL><P class="p_notebook">PCR products were checked using an electrophoresis. Expected bands:</P><UL class="ul_notebook"><LI>FAO1-PCR1: 1157 bp</LI><LI>FAO1-PCR2: 1015 bp</LI></UL><P class="p_notebook">Both FAO1 PCR products were not correct.</P><P class="p_notebook">As the EaDAcT TU was not correct, ligation reaction was done again. The following table shows ligation details:</P><TABLE class="table_notebook"><TBODY><TR class="tr_notebook"><TD class="td_notebook">Reagent</TD><TD class="td_notebook">Volume</TD></TR><TR class="tr_notebook"><TD class="td_notebook">Trichome promoter</TD><TD class="td_notebook">1 μL</TD></TR><TR class="tr_notebook"><TD class="td_notebook">GFP</TD><TD class="td_notebook">1 μL</TD></TR><TR class="tr_notebook"><TD class="td_notebook">TNos</TD><TD class="td_notebook">1 μL</TD></TR><TR class="tr_notebook"><TD class="td_notebook">BsaI</TD><TD class="td_notebook">1 μL</TD></TR><TR class="tr_notebook"><TD class="td_notebook">p2α2</TD><TD class="td_notebook">1 μL</TD></TR><TR class="tr_notebook"><TD class="td_notebook">T4 ligase</TD><TD class="td_notebook">1 μL</TD></TR><TR class="tr_notebook"><TD class="td_notebook">Ligase buffer</TD><TD class="td_notebook">1 μL</TD></TR><TR class="tr_notebook"><TD class="td_notebook">H2O</TD><TD class="td_notebook">3 μL</TD></TR><TR class="tr_notebook"><TD class="td_notebook">Total Volume</TD><TD class="td_notebook">10 μL</TD></TR></TBODY></TABLE><H4 class="date_notebook">07/29/2014</H4><P class="p_notebook">As the FAO1 PCR was not correct, we repeated the reaction. Below is a table showing the details:</P><TABLE class="table_notebook"><TBODY><TR class="tr_notebook"><TD class="td_notebook">Reagent</TD><TD class="td_notebook">FAO1-PCR1</TD><TD class="td_notebook">FAO1-PCR2</TD></TR><TR class="tr_notebook"><TD class="td_notebook"><I>C. tropicalis</I> genome</TD><TD class="td_notebook">2.5 μL</TD><TD class="td_notebook">2.5 μL</TD></TR><TR class="tr_notebook"><TD class="td_notebook">HF Buffer</TD><TD class="td_notebook">30 μL</TD><TD class="td_notebook">30 μL</TD></TR><TR class="tr_notebook"><TD class="td_notebook">dNTPs</TD><TD class="td_notebook">10 μL</TD><TD class="td_notebook">10 μL</TD></TR><TR class="tr_notebook"><TD class="td_notebook">Oligo R</TD><TD class="td_notebook">12.5 μL</TD><TD class="td_notebook">12.5 μL</TD></TR><TR class="tr_notebook"><TD class="td_notebook">Oligo F</TD><TD class="td_notebook">12.5 μL</TD><TD class="td_notebook">12.5 μL</TD></TR><TR class="tr_notebook"><TD class="td_notebook">Phusion polymerase</TD><TD class="td_notebook">1.5 μL</TD><TD class="td_notebook">1.5 μL</TD></TR><TR class="tr_notebook"><TD class="td_notebook">H2O</TD><TD class="td_notebook">181 μL</TD><TD class="td_notebook">181 μL</TD></TR></TBODY></TABLE><P class="p_notebook">PCR temperatures, 25 cycles:</P><TABLE class="table_notebook"><TBODY><TR class="tr_notebook"><TD class="td_notebook">Step</TD><TD class="td_notebook">Temperature (°C)</TD><TD class="td_notebook">Time </TD></TR><TR class="tr_notebook"><TD class="td_notebook">Initialization</TD><TD class="td_notebook">98</TD><TD class="td_notebook">2 min</TD></TR><TR class="tr_notebook"><TD class="td_notebook">Denaturation</TD><TD class="td_notebook">98</TD><TD class="td_notebook">20 s</TD></TR><TR class="tr_notebook"><TD class="td_notebook">Annealing</TD><TD class="td_notebook">50, 55, 60, 65</TD><TD class="td_notebook">??</TD></TR><TR class="tr_notebook"><TD class="td_notebook">Extension</TD><TD class="td_notebook">72</TD><TD class="td_notebook">45 s</TD></TR><TR class="tr_notebook"><TD class="td_notebook">Final elongation</TD><TD class="td_notebook">72</TD><TD class="td_notebook">7 min</TD></TR></TBODY></TABLE><P class="p_notebook">We made a mistake preparing the FAO1-PCR1 adding the wrong template, so we do not expect the correct FAO11-PCR1 product. </P><P class="p_notebook">EaDAcT Transcriptional Unit (TU) transformation</P><P class="p_notebook">Using an electrocompetent <I>E. coli</I> strain (DH5α) and 1.5 ul ligation (P35s:EaDAcT:T35s in 2Ω2), the mix is electroporated at 1500 V. Then, 300 μL of SOC are added and stored at 37°C with agitation. </P><H4 class="date_notebook">07/30/2014</H4><P class="p_notebook">Transform P35s:AtrΔ11:T35+P35s:HarFAR:T35 and P35s:EaDAcT:T35s (in 2α2) in <I><I>Agrobacterium</I> tumefaciens</I> strain C58. Introduce 1 μL of construction in a C58 aliquot. Electroporate at 1440V. Add 500 μL of LB in the LAF. Keep 2 hours in agitation at 28°C. Grow 20 μL and 200 μL in solid medium containing kanamicin and rifampicin. Incubate overnight at 28°C.</P><P class="p_notebook">Pick colonies of P35s:EaDAcT:T35s in 2Ω2.</P><H4 class="date_notebook">08/01/2014</H4><P class="p_notebook">Pick colonies from <I><I>Agrobacterium</I> tumefaciens</I> and grow them in liquid medium for two days at 28°C. Liquid medium is composed by 5 mL LB, Rif (1:1000) and Kan (1:1000) for α vectors and 5 mL LB, Rif (1:1000) and Spm (1:1000) for Ω vectors.</P><H4 class="date_notebook">07/31/2014</H4><P class="p_notebook">Minipreps of yesterday's culture were made, obtaining the transcripional unit: P35S:EaDAcT:T35S in 2Ω2 </P><P class="p_notebook">Additionally, we recultured in petri dish with its respective antibiotic (Spm).</P><P class="p_notebook">Digestions in silico made for checking minipreps:</P><TABLE class="table_notebook"><TBODY><TR class="tr_notebook"><TD class="td_notebook">Pieces/TU</TD><TD class="td_notebook">Restriction enzyme</TD><TD class="td_notebook">Expected bands</TD></TR><TR class="tr_notebook"><TD class="td_notebook">P35S:EaDAcT:T35S in 2Ω2</TD><TD class="td_notebook">NcoI</TD><TD class="td_notebook">8806, 390</TD></TR><TR class="tr_notebook"><TD class="td_notebook">EcoRV</TD><TD class="td_notebook">6652, 1044, 817, 683</TD></TR></TBODY></TABLE><P class="p_notebook">Digestion mixes:</P><UL class="ul_notebook"><LI>Master mix for EcoRV:</LI><UL class="ul_ul_notebook"><LI>3 μL EcoRV</LI><LI>15 μL Red buffer</LI><LI>126 μL H20</LI></UL><LI>Master mix for NcoI:</LI><UL class="ul_ul_notebook"><LI>1.5 μL NcoI</LI><LI>7.5 μL Tango buffer</LI><LI>63 μL H2O</LI></UL></UL><P class="p_notebook">Note: We made master mixes because digestions were made simultaneously with the trichome promoter part. </P><P class="p_notebook">An agarose gel was made to check the transcriptional unit.</P><P class="p_notebook">Minipreps of P35s:EaDAcT:T35s in 2Ω2 (1) went correctly.</P><P class="p_notebook">Miniprep results were quantified and then adjusted at 75 ng/μL:</P><TABLE class="table_notebook"><TBODY><TR class="tr_notebook"><TD class="td_notebook">Pieces/TU</TD><TD class="td_notebook">Tube</TD><TD class="td_notebook">Concentration (ug/μL)</TD><TD class="td_notebook">Initial volume (μL)</TD><TD class="td_notebook">Final Volume (μL)</TD></TR><TR class="tr_notebook"><TD class="td_notebook">P35S:EaDAcT:T35S in 2Ω2</TD><TD class="td_notebook">(1)</TD><TD class="td_notebook">141.4</TD><TD class="td_notebook">35</TD><TD class="td_notebook">31</TD></TR><TR class="tr_notebook"><TD class="td_notebook">P35S:EaDAcT:T35S in 2Ω2</TD><TD class="td_notebook">2)</TD><TD class="td_notebook">3.9</TD><TD class="td_notebook">33</TD><TD class="td_notebook">(Discarded)</TD></TR></TBODY></TABLE><P class="p_notebook">Ligation of P35s:EaDAcT:T35s in 2Ω2 with P35s:AtrΔ11:T35+P35s:HarFAR:T35 in 2Ω1.</P><UL class="ul_notebook"><LI>1 μL P35s:AtrΔ11:T35+P35s:HarFAR:T35 in 2Ω1</LI><LI>1 μL P35s:EaDAcT:T35s in 2Ω2</LI><LI>1 μL 2α1</LI><LI>1 μL BsaI</LI><LI>1 μL T4 ligase</LI><LI>1 μL ligase buffer</LI><LI>4 μL H20</LI></UL><H4 class="date_notebook">08/04/2014</H4><P class="p_notebook">Transformation of P35s:EaDAcT:T35s in 2Ω2 P35s:AtrΔ11:T35+P35s:HarFAR:T35 in <I>E. coli</I>.</P><P class="p_notebook"><I>Agrobacterium</I> liquid cultures (5 mL LB)</P><UL class="ul_notebook"><LI>P35s:GFP:p19:Tnos (Spm, Tet, Rif)</LI><LI>Empty C58 <I><I>Agrobacterium</I> tumefaciens</I> (Rif)</LI><LI>2x P35s:EaDAcT:T35s in 2α2 (Rif, Kan)</LI><LI>2x P35s:AtrΔ11:T35+P35s:HarFAR:T35 in 2Ω1 (Rif, Spm)</LI></UL><H4 class="date_notebook">08/05/2014</H4><P class="p_notebook">Pick colonies from P35s:AtrΔ11:T35+P35s:HarFAR:T35+P35s:EaDAcT:T35s in 2α1.</P><P class="p_notebook">Repeat PCR of FAO1.</P><UL class="ul_notebook"><LI>FAO1-PCR1: 3 reactions at different temperatures (54, 59, 64°C)</LI><UL class="ul_ul_notebook"><LI>1.75 μL <I>Candida tropicalis</I> genomic</LI><LI>35 μL HF buffer (5x)</LI><LI>7 μL dNTPs</LI><LI>8.75 μL oligo forward (Jul07)</LI><LI>8.75 μL oligo reverse (Jul08)</LI><LI>1.05 μL Phusion polymerase</LI><LI>112.7 H20</LI></UL></UL><P class="p_notebook">PCR temperatures, 35 cycles:</P><TABLE class="table_notebook"><TBODY><TR class="tr_notebook"><TD class="td_notebook">Step</TD><TD class="td_notebook">Temperature (°C)</TD><TD class="td_notebook">Time </TD></TR><TR class="tr_notebook"><TD class="td_notebook">Initialization</TD><TD class="td_notebook">98</TD><TD class="td_notebook">2 min</TD></TR><TR class="tr_notebook"><TD class="td_notebook">Denaturation</TD><TD class="td_notebook">98</TD><TD class="td_notebook">10 s</TD></TR><TR class="tr_notebook"><TD class="td_notebook">Annealing</TD><TD class="td_notebook">54, 59, 64</TD><TD class="td_notebook">55 s</TD></TR><TR class="tr_notebook"><TD class="td_notebook">Extension</TD><TD class="td_notebook">72</TD><TD class="td_notebook">55 s</TD></TR><TR class="tr_notebook"><TD class="td_notebook">Final elongation</TD><TD class="td_notebook">72</TD><TD class="td_notebook">7 min</TD></TR></TBODY></TABLE><UL class="ul_notebook"><LI>FAO1-PCR2: touchdown PCR</LI><UL class="ul_ul_notebook"><LI>0.5 μL <I>Candida tropicalis</I> genomic</LI><LI>10 μL HF buffer (5x)</LI><LI>2 μL dNTPs</LI><LI>2.5 μL oligo forward (Jul09)</LI><LI>2.5 μL oligo reverse (Jul10)</LI><LI>0.5 μL Phusion polymerase</LI><LI>32 μL H20</LI></UL></UL><P class="p_notebook">PCR temperatures, 35 cycles:</P><TABLE class="table_notebook"><TBODY><TR class="tr_notebook"><TD class="td_notebook">Step</TD><TD class="td_notebook">Temperature (°C)</TD><TD class="td_notebook">Time </TD></TR><TR class="tr_notebook"><TD class="td_notebook">Initialization</TD><TD class="td_notebook">98</TD><TD class="td_notebook">5 min</TD></TR><TR class="tr_notebook"><TD class="td_notebook">Denaturation</TD><TD class="td_notebook">98</TD><TD class="td_notebook">10 s</TD></TR><TR class="tr_notebook"><TD class="td_notebook">Annealing</TD><TD class="td_notebook">69.5 (descending 0.5 per cycle) </TD><TD class="td_notebook">20 s</TD></TR><TR class="tr_notebook"><TD class="td_notebook">Extension</TD><TD class="td_notebook">72</TD><TD class="td_notebook">55 s</TD></TR><TR class="tr_notebook"><TD class="td_notebook">Final elongation</TD><TD class="td_notebook">72</TD><TD class="td_notebook">7 min</TD></TR></TBODY></TABLE><P class="p_notebook">It is not working yet. For the next time we are going to repeat the dilutions in case they weren't correctly done.</P><H4 class="date_notebook">08/06/14</H4><P class="p_notebook">Minipreps of yesterday's culture were made:</P><UL class="ul_notebook"><LI>TU AtrΔ11 + TU HarFAR + TU EaDAcT</LI></UL><P class="p_notebook">Additionally, we made <I>Agrobacterium</I>' culture minipreps using a different kit (We used the QIAgen Miniprep kit 250, 27106)</P><UL class="ul_notebook"><LI>TU AtrΔ11 + TU HarFAR in 2Ω1</LI><LI>P35S:EaDAcT:T35S in 2Ω2</LI></UL><P class="p_notebook">FAO1 PCR was repeated (this time using a different primers aliquot). </P><UL class="ul_notebook"><LI>FAO1-PCR1: </LI><UL class="ul_ul_notebook"><LI>0.5 μL <I>Candida tropicalis</I> genomic</LI><LI>10 μL HF buffer (5x)</LI><LI>2 μL dNTPs</LI><LI>2.5 μL oligo forward (Jul07)</LI><LI>2.5 μL oligo reverse (Jul08)</LI><LI>0.5 μL Phusion polymerase</LI><LI>32 μL H20</LI></UL></UL><UL class="ul_notebook"><LI>FAO2-PCR1: </LI><UL class="ul_ul_notebook"><LI>0.5 μL <I>Candida tropicalis</I> genomic</LI><LI>10 μL HF buffer (5x)</LI><LI>2 μL dNTPs</LI><LI>2.5 μL oligo forward (Jul09)</LI><LI>2.5 μL oligo reverse (Jul10)</LI><LI>0.5 μL Phusion polymerase</LI><LI>32 μL H20</LI></UL></UL><P class="p_notebook">PCR temperatures, 35 cycles:</P><TABLE class="table_notebook"><TBODY><TR class="tr_notebook"><TD class="td_notebook">Step</TD><TD class="td_notebook">Temperature (°C)</TD><TD class="td_notebook">Time </TD></TR><TR class="tr_notebook"><TD class="td_notebook">Initialization</TD><TD class="td_notebook">98</TD><TD class="td_notebook">2 min</TD></TR><TR class="tr_notebook"><TD class="td_notebook">Denaturation</TD><TD class="td_notebook">98</TD><TD class="td_notebook">10 s</TD></TR><TR class="tr_notebook"><TD class="td_notebook">Annealing</TD><TD class="td_notebook">59 (PCR1)/ 64 (PCR2) (descending 0.5 per cycle) </TD><TD class="td_notebook">20 s</TD></TR><TR class="tr_notebook"><TD class="td_notebook">Extension</TD><TD class="td_notebook">72</TD><TD class="td_notebook">55 s</TD></TR><TR class="tr_notebook"><TD class="td_notebook">Final elongation</TD><TD class="td_notebook">72</TD><TD class="td_notebook">7 min</TD></TR></TBODY></TABLE><P class="p_notebook">Digestions made in silico to check minipreps:</P><P class="p_notebook"><I>E. coli</I></P><TABLE class="table_notebook"><TBODY><TR class="tr_notebook"><TD class="td_notebook">Pieces/TU</TD><TD class="td_notebook">Resriction enzymes</TD><TD class="td_notebook">Buffer</TD><TD class="td_notebook">Expected Bands</TD></TR><TR class="tr_notebook"><TD class="td_notebook">TU AtrΔ11+ TU HarFAR + TU EaDAcT in 2α1</TD><TD class="td_notebook">EcoRI</TD><TD class="td_notebook">Orange</TD><TD class="td_notebook">7428, 6323</TD></TR><TR class="tr_notebook"><TD class="td_notebook">TU AtrΔ11+ TU HarFAR + TU EaDAcT in 2α1</TD><TD class="td_notebook">BglII</TD><TD class="td_notebook">Orange</TD><TD class="td_notebook">11175, 2576</TD></TR></TBODY></TABLE><P class="p_notebook"><I>A. tumefaciens</I></P><TABLE class="table_notebook"><TBODY><TR class="tr_notebook"><TD class="td_notebook">Pieces/TU</TD><TD class="td_notebook">Resriction enzymes</TD><TD class="td_notebook">Buffer</TD><TD class="td_notebook">Expected Bands</TD></TR><TR class="tr_notebook"><TD class="td_notebook">TU AtrΔ11 + TU HarFAR in 2Ω1</TD><TD class="td_notebook">EcoRV</TD><TD class="td_notebook">Red</TD><TD class="td_notebook">9307, 2251</TD></TR><TR class="tr_notebook"><TD class="td_notebook">TU AtrΔ11 + TU HarFAR in 2Ω1</TD><TD class="td_notebook">BamHI</TD><TD class="td_notebook">Green</TD><TD class="td_notebook">6652, 4906</TD></TR><TR class="tr_notebook"><TD class="td_notebook">TU EaDAcT in 2α2</TD><TD class="td_notebook">EcoRV</TD><TD class="td_notebook">Red</TD><TD class="td_notebook">8021, 683</TD></TR><TR class="tr_notebook"><TD class="td_notebook">TU EaDAcT in 2α2</TD><TD class="td_notebook">HindIII</TD><TD class="td_notebook">Red</TD><TD class="td_notebook">6322, 2382</TD></TR></TBODY></TABLE><P class="p_notebook">Digestion master mixes:</P><UL class="ul_notebook"><LI>Master mix for NotI:</LI><UL class="ul_ul_notebook"><LI>2.5 μL NotI</LI><LI>12.5 μL Orange buffer</LI><LI>105 μL H20</LI></UL><LI>Master mix for RsaI:</LI><UL class="ul_ul_notebook"><LI>2 μL NcoI</LI><LI>10 μL Tango buffer</LI><LI>84 μL H2O</LI></UL></UL><P class="p_notebook">Note: We made master mixes because digestions were made simultaneously with the switch part. </P><P class="p_notebook">We made different mixes for <I>Agrobacterium</I> samples because we think that minipreps are not as good as it is expected.</P><UL class="ul_notebook"><LI><I>Agrobacterium</I> sample mix:</LI><UL class="ul_ul_notebook"><LI>0.5 μL Restriction enzyme</LI><LI>2.5 μL Buffer</LI><LI>5 μL Miniprep sample</LI><LI>17 μL H2O</LI></UL></UL><P class="p_notebook">Digestions in <I>A. tumefaciens</I>.</P><P class="p_notebook">FAO1 PCR product.</P><P class="p_notebook">All digestions and TU AtrΔ11+ TU HarFAR + TU EaDAcT in 2α1 were correct. PCR products were not correct or absent again. </P><P class="p_notebook">As digestions were correct, we recultured <I>Agrobacterium</I> in new media (LB) in order to have cultures in exponential phase for tomorrow. We mix in each tube 5 mL of LB with 40 μL of inoculum, XGal (2:1000), IPTG (1:1000)and the corresponding antibiotic (1:1000). </P><TABLE class="table_notebook"><TBODY><TR class="tr_notebook"><TD class="td_notebook">Culture</TD><TD class="td_notebook">Antibiotic</TD></TR><TR class="tr_notebook"><TD class="td_notebook">P35:GFP:P19:TNos</TD><TD class="td_notebook">Spm, Tet, Rif</TD></TR><TR class="tr_notebook"><TD class="td_notebook"><I>Agrobacterium</I> (as a control)</TD><TD class="td_notebook">Rif</TD></TR><TR class="tr_notebook"><TD class="td_notebook">P35S:EaDAcT:T35S</TD><TD class="td_notebook"> Rif, Kan</TD></TR><TR class="tr_notebook"><TD class="td_notebook">P35S:AtrΔ11:T35S + P35S:HarFAR:T35S</TD><TD class="td_notebook">Rif, Spm</TD></TR></TBODY></TABLE><P class="p_notebook">Recultured media was grown at 28 °C overnight (around 16 h).</P><H4 class="date_notebook">08/07/2014</H4><P class="p_notebook">Agroinfiltration in <I>Nicotiana benthamiana</I>.</P><UL class="ul_notebook"><LI><I>Agrobacterium</I> control culture and P35s:GFP:P19:Tnos (x2 forth true leaves)</LI><LI>TU AtrΔ11+TU HarFAR and P35s:GFP:P19:Tnos (x2 forth true leaves)</LI><LI>TU AtrΔ11+TU HarFAR and TU EaDAcT and P35s:GFP:P19:Tnos (x2 forth true leaves)</LI></UL><P class="p_notebook">Agroinfiltration protocol consists of:</P><UL class="ul_notebook"><LI>Centrifuge the cultures 15 min 3000 rpm and discard supernatant.</LI><LI>5 mL of agroinfiltration solution per culture. 100 mL of agroinfiltration solution were composed of 10 mL MES 100mM (pH 5.6), 1 mL MgCl2 1M and 100 μL acetosyringone solution 200 mM (19.62 mg, DMSO 500 μL; prepare freshly). Mix it with the vortex. If the culture is still turbid, add a bit more of agroinfiltration sollution. Put it in the (rodillos) for two hours.</LI><LI>Measure the OD. The optimum OD for agroinfiltration is 0.2. If it is too high adjust the concentration with more agroinfiltration solution.</LI><LI>Mix the cultures, keeping all of them in the same proportions.</LI><LI>Proceed to agroinfiltration.</LI></UL><H4 class="date_notebook">08/08/2014</H4><P class="p_notebook">In order to have a control for the FAO1 PCR, which hasn't been very successful, Jesus Munoz provided us with 4 primers and 2 clones of <I>Candida tropicalis</I> (C981 ng/μL and pYEP C98 28.2 ng/μL). These primers amplify for the gene HSR1.</P><TABLE class="table_notebook"><TBODY><TR class="tr_notebook"><TD class="td_notebook">Name </TD><TD class="td_notebook">Sequence </TD><TD class="td_notebook">Tm</TD></TR><TR class="tr_notebook"><TD class="td_notebook">HSR1 RTRv+1149</TD><TD class="td_notebook">TTTGTCTTGCAACAGGTCCA</TD><TD class="td_notebook">56°C</TD></TR><TR class="tr_notebook"><TD class="td_notebook">HSR1 clone Fw+1 </TD><TD class="td_notebook">ATGAGTAAGAAAAGCAACAGTACC</TD><TD class="td_notebook">54°C</TD></TR><TR class="tr_notebook"><TD class="td_notebook">HSR1 fw-BamHI+480</TD><TD class="td_notebook">GCTGGATCCTTAGTAGTAGTGGATCAAGGAAT</TD><TD class="td_notebook">49°C (annealing)</TD></TR><TR class="tr_notebook"><TD class="td_notebook">HSR1 clone RV+stop</TD><TD class="td_notebook">CTAATTTTCTTCTTTTTCAATAGTAACTATCC</TD><TD class="td_notebook">51°C</TD></TR></TBODY></TABLE><P class="p_notebook">Possibility of primer combinations: </P><TABLE class="table_notebook"><TBODY><TR class="tr_notebook"><TD class="td_notebook">A</TD><TD class="td_notebook">HSR1 fw-BamHI+480</TD><TD class="td_notebook">HSR1 RTRv+1149</TD><TD class="td_notebook">687</TD><TD class="td_notebook">49°C</TD></TR><TR class="tr_notebook"><TD class="td_notebook">C</TD><TD class="td_notebook">HSR1 clone Fw+1</TD><TD class="td_notebook">HSR1 clone RV+stop</TD><TD class="td_notebook">2187</TD><TD class="td_notebook">-</TD></TR><TR class="tr_notebook"><TD class="td_notebook">B</TD><TD class="td_notebook">HSR1 RTRv+1149</TD><TD class="td_notebook">HSR1 clone Fw+1 </TD><TD class="td_notebook">1168</TD><TD class="td_notebook">54°C</TD></TR></TBODY></TABLE><P class="p_notebook">We amplified the genomic of <I>C. tropicalis</I> and the two clones (C98 and C98 pYep)with the primer combinations A and B with Taq polymerase at 2 different temperatures (49 and 52°C). C primer combination was not used due to the length of the spected band. </P><P class="p_notebook">PCR parameters</P><UL class="ul_notebook"><LI>94°C, 3 min</LI><LI>35 cycles</LI><UL class="ul_ul_notebook"><LI>94°C, 30 s</LI><LI>49 or 52°C, 15 s</LI><LI>72°C, 90 s</LI></UL><LI>72°C, 7 min</LI></UL><P class="p_notebook">PCR products were not present. It probably did not work because we added to much buffer.  </P><H4 class="date_notebook">08/11/2014</H4><P class="p_notebook">We obtained a different plasmid (pUbiquitina HSRI-CDS col.6) as a positive control of PCR to check the quality of our Candida genome. We diluted them to obtain a final concentration of 30 ng/μL.</P><P class="p_notebook">PCRs wih Taq polimerase:</P><UL class="ul_notebook"><LI>1 μL Template </LI><LI>1.5 μL dNTPs</LI><LI>1 μL Forward primer</LI><LI>1 μL Reverse primer</LI><LI>1 μL Taq pol.</LI><LI>5 Buffer 10X</LI><LI>39.5 μL H2O</LI></UL><TABLE class="table_notebook"><TBODY><TR class="tr_notebook"><TD class="td_notebook">PCR</TD><TD class="td_notebook">Template</TD><TD class="td_notebook">F primer</TD><TD class="td_notebook">R primer</TD></TR><TR class="tr_notebook"><TD class="td_notebook">1</TD><TD class="td_notebook">pUbiquitina HSRI-CDS col.6</TD><TD class="td_notebook">HSR1 BamHI + 480</TD><TD class="td_notebook">HSR1 RTRev + 1149</TD></TR><TR class="tr_notebook"><TD class="td_notebook">2</TD><TD class="td_notebook">pUbiquitina HSRI-CDS col.6</TD><TD class="td_notebook">HSR1 RTRv + 1149</TD><TD class="td_notebook">HSR1 Fw + 1</TD></TR><TR class="tr_notebook"><TD class="td_notebook">3</TD><TD class="td_notebook"><I>C. tropicalis</I> genome</TD><TD class="td_notebook">HSRI-CDS col.6</TD><TD class="td_notebook">HSR1 BamHI + 480</TD><TD class="td_notebook">HSR1 Rtrev + 1149</TD></TR><TR class="tr_notebook"><TD class="td_notebook">4</TD><TD class="td_notebook"><I>C. tropicalis</I> genome</TD><TD class="td_notebook">HSR1 RTRv + 1149</TD><TD class="td_notebook">HSR1 Fw + 1</TD></TR></TBODY></TABLE><P class="p_notebook">PCR conditions:</P><UL class="ul_notebook"><LI>94°C 3 min</LI><LI>35 cycles</LI><UL class="ul_ul_notebook"><LI>94°C 30 s</LI><LI>49°C 15 s</LI><LI>72°C 90 s</LI></UL><LI>72°C 7 min</LI></UL><P class="p_notebook">We had amplification in our positive controls. Our <I>C. tropicalis</I> genome may be wrong. Therefore Jesús Muñoz provided us with a new <I>Candida tropicalis</I> (NCYC 2512) culture and also a culture from a Candida tropicales genoteque made in <I>E. coli</I>.</P><H4 class="date_notebook">08/12/2014</H4><P class="p_notebook">PHEROMONE ANALYSIS</P><H4 class="date_notebook">PONER ENLACE DE LA WIKI</H4><P class="p_notebook">To begin with samples were obtained from the agroinfiltrated plants after 5 days. We collected 9 samples:</P><UL class="ul_notebook"><LI>2 leaves from P35s:GFP:P19:Tnos </LI><LI>2 leaves from TU AtrΔ11+TU HarFAR and P35s:GFP:P19:Tnos </LI><LI>2 leaves from TU AtrΔ11+TU HarFAR and TU EaDAcT and P35s:GFP:P19:Tnos </LI><LI>1 leaf from a wild type plant</LI></UL><P class="p_notebook">Each sample was stored in a vial and kept in liquid nitrogen. Leaves were mashed using a mortar and liquid nitrogen until powder from each leaf is obtained and stored in a vial .Samples must be always kept in liquid nitrogen or in a -80°C freezer . Afterwards the leaf powder was weighted and introduced in a 10 mL screwcap headspace vial.</P><UL class="ul_notebook"><LI>94,6 mg of P35s:GFP:P19:Tnos leaf (replica 1)</LI><LI>97,0 mg of TU AtrΔ11+TU HarFAR and P35s:GFP:P19:Tnos leaf (replica 2)</LI><LI>118,7 mg of TU AtrΔ11+TU HarFAR and TU EaDAcT and P35s:GFP:P19:Tnos leaf (replica 1)</LI><LI>100,0 mg of wild type leaf</LI></UL><P class="p_notebook">Then  150 μL of EDTA 500mM and 1 mL of a saturated solution of CaCl2 (5,7M) were added to each vial.</P><P class="p_notebook">EDTA 500mM preparation:</P><P class="p_notebook">Stock of solid EDTA Di-Sodium 372,24 Mw and a final solution of 50 mL, 500mM. 372,24*0,5*0,05=9,306 g in 50 mL.</P><P class="p_notebook">After the addition of EDTA and CaCl2 the samples were sonicated dutring 5 minutes to disgregate the tissue and release the volatile compounds. Afterwards the samples were analysed by GC-MC following this procedure.</P><H4 class="date_notebook">PONER LOS PASOS QUE SIGUE EL PARATO, provided by JOSE LUIS MAS ADELANTE: el protocolo entero est\E1 en la carpeta de protocolos como volatile analysis protocol</H4><P class="p_notebook">Analysis was performed overnight.</P><H4 class="date_notebook">08/13/2014</H4><P class="p_notebook">First results of the analysis were obtained. The analysis proved that our plant was successfuly producing the desired pheromones in high concentration. As expected z-11-hexadecen-1-ol and z-11-hexadecen-1-ol acetate were being produced and also unexpectedly the z-11-hexadecenal. </P><P class="p_notebook">As shown in the figure, the leaf agroinfiltrated with TU AtrΔ11+TU HarFAR and P35s:GFP:P19:Tnos (represented in black) shows a successful production of (Z)-11-hexadecen-1-ol compared with the negative control that only has P35s:GFP:P19:Tnos (represented in blue) and shows no expression. </P><P class="p_notebook">In this figure, expression of (z)-11-hexadecen-1-ol and (z)-11-hexadecen-1-ol acetate is proved. The expression in the leaf infiltrated with TU AtrΔ11+TU HarFAR and TU EaDAcT and P35s:GFP:P19:Tnos is represented in black, and the negative control with P35s:GFP:P19:Tnos is represented in blue.</P><P class="p_notebook">In this figure, an unexprected peak present in the leaf infiltrated with TU AtrΔ11+TU HarFAR and P35s:GFP:P19:Tnos (black) can be observed. Comparing its spectrum with the one provided from the database seems to be (z)-11-hexadecenal, a desired pheromone,  which is being produced by the plant itself using an endogenous alcohol oxidase. Nevertheless as it is produced with a low yield, the FAO1 of <I>C. tropicalis</I> search is still in progress.</P><P class="p_notebook">The rest of the samples were prepared for the GC-MS analysis.</P><P class="p_notebook">The samples were weighted, introduced in the vial and added with EDTA and CaCl2.</P><UL class="ul_notebook"><LI>94,0 mg of P35s:GFP:P19:Tnos leaf (replica 2)</LI><LI>102,4 mg of TU AtrΔ11+TU HarFAR and P35s:GFP:P19:Tnos leaf (replica 1)</LI><LI>92,0 mg of TU AtrΔ11+TU HarFAR and TU EaDAcT and P35s:GFP:P19:Tnos leaf(replica 2)</LI></UL><P class="p_notebook">Results of the replicae analysis are shown below:</P><P class="p_notebook">In this replica, the sample with the TU AtrΔ11+TU HarFAR and P35s:GFP:P19:Tnos construction shows a huge production of (z)-11-hexadecen-1-ol.</P><P class="p_notebook">In this replica, the sample with the TU AtrΔ11+TU HarFAR and TU EaDAcT and P35s:GFP:P19:Tnos shows a higher abundance of (z)-11-hexadecen-1-ol and z-11-hexadecen-1-ol acetate.</P><P class="p_notebook">In order to verify that the analysed compounds are the desired pheromones, we acquired standards for (z)-11-hexadecen-1-ol and (z)-11-hexadecen-1-ol acetate and (z)-11-hexadecenal, and indeed, the analysed compunds were the right ones.</P><H4 class="date_notebook">08/14/2014</H4><P class="p_notebook">We had problems to amplify the FAO1 gene, so in order to obtain it we performed a colony PCR. Using this method, it is possible to amplify a fragment directly from a colony rather than a DNA sample. </P><P class="p_notebook">We made two different PCRs, one of them as a positive control and the other one to amplify our disered DNA fragment.</P><P class="p_notebook">Colony PCR protocol (Taq Polimerase):</P><UL class="ul_notebook"><LI>1 colony (<I>C. tropicalis</I>)</LI><LI>1.5 μL dNTPs</LI><LI>1 μL Forward Primer </LI><LI>1 μL Reverse Primer</LI><LI>1 μL Taq Polimerase</LI><LI>5 μL Buffer 10X</LI><LI>39.5 μL H2O miliQ</LI></UL><P class="p_notebook">Primers used as a control: HSR1 + 480 and RTRv + 1149.</P><P class="p_notebook">Primers used to amplify FAO1 gene: iGEMJUL07_FAO1_1F and iGEMJUL08_FAO1_1R. </P><P class="p_notebook">Thermocycler conditions, 35 cycles:</P><TABLE class="table_notebook"><TBODY><TR class="tr_notebook"><TD class="td_notebook">Step</TD><TD class="td_notebook">Temperature (°C)</TD><TD class="td_notebook">Time </TD></TR><TR class="tr_notebook"><TD class="td_notebook">Initialization</TD><TD class="td_notebook">94</TD><TD class="td_notebook">3 min</TD></TR><TR class="tr_notebook"><TD class="td_notebook">Denaturation</TD><TD class="td_notebook">94</TD><TD class="td_notebook">30 s</TD></TR><TR class="tr_notebook"><TD class="td_notebook">Annealing</TD><TD class="td_notebook">49</TD><TD class="td_notebook">15 s</TD></TR><TR class="tr_notebook"><TD class="td_notebook">Extension</TD><TD class="td_notebook">72</TD><TD class="td_notebook">1 min</TD></TR><TR class="tr_notebook"><TD class="td_notebook">Final elongation</TD><TD class="td_notebook">72</TD><TD class="td_notebook">7 min</TD></TR></TBODY></TABLE><P class="p_notebook">Starting from an agar plate containing a Candida gene bank, we add 1 mL of LB medium and we mix it. Then, the mix was transferred into a tube. We stored part of the culture in glycerine and another part (200 μL) was mixed with 5 mL of LB media and Amp (2:1000). </P><P class="p_notebook">The tube containing the gene bank was grown at 28°C for 1 hour. Then, we made minipreps. </P><P class="p_notebook">The electrophoresis gel shows that the colony PCR failed, even the control did not work. Additionally, we test the BbsI restriction enzyme and we found that it does not cut well. </P><H4 class="date_notebook">08/15/2014</H4><P class="p_notebook">We transformed the whole pathway (P35S:AtrΔ11:T35S, P35S:HarFAR:T35S, P35S:EaDAcT:T35S in 2α1) into <I><I>Agrobacterium</I> tumefaciens</I> (C58) and we cultured it in solid media with Kan (1:1000) and Rif (1:1000) during 2 days at 28°C. </P><H4 class="date_notebook">08/18/2014</H4><P class="p_notebook">We repeated the colony PCR to obtain FAO1 gene and also control PCRs (using the gene bank minipreps made on 08/14/2014).</P><P class="p_notebook">PCR conditions:</P><UL class="ul_notebook"><LI>Colony PCR 1 (control):</LI><UL class="ul_ul_notebook"><LI>1 colony (<I>C. tropicalis</I>)</LI><LI>1 μL dNTPs</LI><LI>1 μL HSR1 BamHI + 480 </LI><LI>1 μL HSR1 RTRv + 1149  </LI><LI>1 μL Tap pol.</LI><LI>5 μL Buffer 10x</LI><LI>39 μL H2O miliQ</LI></UL></UL><UL class="ul_notebook"><LI>Colony PCR 2:</LI><UL class="ul_ul_notebook"><LI>1 colony (<I>C. tropicalis</I>)</LI><LI>1 μL dNTPs</LI><LI>1 μL iGEMJul09 </LI><LI>1 μL iGEMJul10   </LI><LI>1 μL Tap pol.</LI><LI>5 μL Buffer 10x</LI><LI>39 μL H2O miliQ</LI></UL></UL><UL class="ul_notebook"><LI>Gene bank PCR 3 (control):</LI><UL class="ul_ul_notebook"><LI>1 μL C. tropicallis gene bank miniprep </LI><LI>1 μL dNTPs</LI><LI>1 μL HSR1 BamHI + 480 </LI><LI>1 μL HSR1 RTRv + 1149  </LI><LI>1 μL Tap pol.</LI><LI>5 μL Buffer 10x</LI><LI>39 μL H2O miliQ</LI></UL></UL><UL class="ul_notebook"><LI>Gene bank PCR 4:</LI><UL class="ul_ul_notebook"><LI>1 μL C. tropicallis gene bank miniprep </LI><LI>1 μL dNTPs</LI><LI>1 μL iGEMJul09 </LI><LI>1 μL iGEMJul10   </LI><LI>1 μL Tap pol.</LI><LI>5 μL Buffer 10x</LI><LI>39 μL H2O miliQ</LI></UL></UL><P class="p_notebook">PCR conditions were the same as those used on 08/14/2014</P><H4 class="date_notebook">08/20/2014</H4><P class="p_notebook">We were trying to obtain the FAO1 gene. We did a yeast colony PCR. Using an sterile tip, we picked one <I>C. tropicalis</I> colony and we introduced them into a vial containing 30 μL SDS 0.2 %. The vial was vortexed 15 seconds and then heated 4 minutes at 90°C. Next, it was centrifuged during 1 minute ans the supernatant was transferred to a new 1.5 μL vial. That was our PCR template. </P><P class="p_notebook">We performed a control PCR employing control primers (HSRI Rtrv + 1149 and HSRI BamHI + 480)and the another PCR using FAO1 primers as previously done (iGEMJul09 and iGEMJul10).</P><P class="p_notebook">PCR conditions using Phusion polimerase (35 cycles):</P><TABLE class="table_notebook"><TBODY><TR class="tr_notebook"><TD class="td_notebook">Step</TD><TD class="td_notebook">Temperature (°C)</TD><TD class="td_notebook">Time </TD></TR><TR class="tr_notebook"><TD class="td_notebook">Initialization</TD><TD class="td_notebook">98</TD><TD class="td_notebook">3 min</TD></TR><TR class="tr_notebook"><TD class="td_notebook">Denaturation</TD><TD class="td_notebook">98</TD><TD class="td_notebook">5 s</TD></TR><TR class="tr_notebook"><TD class="td_notebook">Annealing</TD><TD class="td_notebook">49</TD><TD class="td_notebook">10 s</TD></TR><TR class="tr_notebook"><TD class="td_notebook">Extension</TD><TD class="td_notebook">72</TD><TD class="td_notebook">20 s</TD></TR><TR class="tr_notebook"><TD class="td_notebook">Final elongation</TD><TD class="td_notebook">72</TD><TD class="td_notebook">7 min</TD></TR></TBODY></TABLE><P class="p_notebook">We did not close the PCR tube properly so we found our PCR product had evaporated. The other PCR product (the control) was loaded and as it is shown in the gel electrophoresis, it didn't work. </P><H4 class="date_notebook">08/22/2014</H4><P class="p_notebook">We did again a yeast genomic extraction (<I>C. tropicalis</I>), but this time we changed the protocol:</P><UL class="ul_notebook"><LI>Pick 8 colonies of <I>C. tropicalis</I> growth in YPD media and resuspend them with 100 μL of solution (200 mM LiOAc and SDS 1%). </LI><LI>Incubate 15 min at 70°C.</LI><LI>Add 300 μL of ethanol 96%. Then, vortex the solution.</LI><LI>Centrifuge 3 min at 15000 xg.</LI><LI>Discard the superatant and resuspend the pellet (precipitated DNA) with 100 μL TE.</LI><LI>Centrifuge 1 min at 15000 xg. </LI><LI>Recover 1 μL of supernatant. </LI></UL><P class="p_notebook">Using this genomic DNA as a template, we run a PCR (using Taq polimerase) with our primers and another one as a control. </P><UL class="ul_notebook"><LI>Control PCR:</LI><UL class="ul_ul_notebook"><LI>1 μL template</LI><LI>1 μL dNTPs</LI><LI>1 μL HSR1 clone Fw+1 </LI><LI>1 μL HSR1 Rtrv + 1149</LI><LI>μL Taq polimerase </LI><LI>5 μL Buffer 10X</LI><LI>40 μL H2O</LI></UL></UL><UL class="ul_notebook"><LI>FAO1 PCR</LI><LI>1 μL template</LI><UL class="ul_ul_notebook"><LI>1 μL dNTPs</LI><LI>1 μL iGEMJul09_FAO1_PCR2F</LI><LI>1 μL iGEMJul10_FAO1_PCR2R</LI><LI>μL Taq polimerase </LI><LI>5 μL Buffer 10X</LI><LI>40 μL H2O</LI></UL></UL><P class="p_notebook">PCR parameters (35 cycles):</P><TABLE class="table_notebook"><TBODY><TR class="tr_notebook"><TD class="td_notebook">Step</TD><TD class="td_notebook">Temperature (°C)</TD><TD class="td_notebook">Time </TD></TR><TR class="tr_notebook"><TD class="td_notebook">Initialization</TD><TD class="td_notebook">94</TD><TD class="td_notebook">3 min</TD></TR><TR class="tr_notebook"><TD class="td_notebook">Denaturation</TD><TD class="td_notebook">94</TD><TD class="td_notebook">30 s</TD></TR><TR class="tr_notebook"><TD class="td_notebook">Annealing</TD><TD class="td_notebook">49</TD><TD class="td_notebook">15 s</TD></TR><TR class="tr_notebook"><TD class="td_notebook">Extension</TD><TD class="td_notebook">72</TD><TD class="td_notebook">90 s</TD></TR><TR class="tr_notebook"><TD class="td_notebook">Final elongation</TD><TD class="td_notebook">72</TD><TD class="td_notebook">7 min</TD></TR></TBODY></TABLE><H4 class="date_notebook">08/25/2014</H4><P class="p_notebook">We repeated the FAO1 colony PCR using a <I>C. tropicalis</I> gene bank in <I>E. coli</I>. We made 3 PCRs employing HSR1 primers and other 3 PCRs using our iGEM primers as follows:</P><UL class="ul_notebook"><LI>PCR 1 (Annealing temperature 49°C)</LI><UL class="ul_ul_notebook"><LI>HSR1 Fw_BamHI+480 </LI><LI>HSR1 RTRv+1149</LI></UL></UL><UL class="ul_notebook"><LI>PCR 2 and 3 (Annealing temperature 54°C)</LI><UL class="ul_ul_notebook"><LI>HSR1 clone Fw+1 </LI><LI>HSR1 RTRv+1149 </LI></UL></UL><UL class="ul_notebook"><LI>PCR 4 and 5 (Annealing temperature 50°C)</LI><UL class="ul_ul_notebook"><LI>iGEMJul07 </LI><LI>iGEMJul08</LI></UL></UL><UL class="ul_notebook"><LI>PCR 6 (Annealing temperature 56°C)</LI><UL class="ul_ul_notebook"><LI>iGEMJul09 </LI><LI>iGEMJul10</LI></UL></UL><P class="p_notebook">PCR conditions with Taq polimerase (35 cycles):</P><TABLE class="table_notebook"><TBODY><TR class="tr_notebook"><TD class="td_notebook">Step</TD><TD class="td_notebook">Temperature (°C)</TD><TD class="td_notebook">Time </TD></TR><TR class="tr_notebook"><TD class="td_notebook">Initialization</TD><TD class="td_notebook">94</TD><TD class="td_notebook">3 min</TD></TR><TR class="tr_notebook"><TD class="td_notebook">Denaturation</TD><TD class="td_notebook">94</TD><TD class="td_notebook">30 s</TD></TR><TR class="tr_notebook"><TD class="td_notebook">Annealing</TD><TD class="td_notebook">49</TD><TD class="td_notebook">1:30 min</TD></TR><TR class="tr_notebook"><TD class="td_notebook">Extension</TD><TD class="td_notebook">72</TD><TD class="td_notebook">1:30 min</TD></TR><TR class="tr_notebook"><TD class="td_notebook">Final elongation</TD><TD class="td_notebook">72</TD><TD class="td_notebook">7 min</TD></TR></TBODY></TABLE><P class="p_notebook">The electrophoresis gel shows that PCRs have not yielded any product.</P><H3 class="section_notebook">Expression in trichomes</H3><H4 class="date_notebook">07/03/2014</H4><P class="p_notebook">Genomic DNA extraction from Nicotiana tabacum. We need the genome of this organism because we want to obtain the trichome promoter from the NtCPS2 gene.</P><UL class="ul_notebook"><LI>Obtain 100 mg of the tobacco leaves (5 disks made with a 1.5 mL vial). Made it twice.</LI><LI>Introduce the disks inside the tube.</LI><LI>Introduce the two tubes in liquid nitrogen.</LI><LI>Remove them from the liquid nitrogen and store at -80°C until use.</LI><LI>Remove one tube from -80°C and re-introduce them in liquid nitrogen. </LI><LI>Grind the disks.</LI><LI>Add 600 μL of CTAB (2%) buffer (pre-heat at 65°C.)</LI><LI>Grind the mixture.</LI><LI>Add RNAse (1.6μL at M = 100 ug/μL for each mL of CTAB buffer). </LI><LI>Vortex it and maintain at 65°C for 45 min. Mix it by inversion 5-15 min.</LI><LI>Add 600 μL cloroform:isoamilic alcohol. Vortex it.</LI><LI>Centrifuge 15 min at 13000 rpm (or 10 min at 14500 rpm.</LI><LI>Recover the supernatant by aspiration (with a 200 μL pipet).</LI><LI>Repeat the last three steps.</LI><LI>Add one volume o isopropanol and mix well by inversion (10 times). </LI><LI>To precipitate, maintain 20 min on ice or at -80°C during 5 min.</LI><LI>Centrifuge 10 min at 13000 rpm (4°C).</LI><LI>Discard the supernatant by decantation (be carefull with the pellet).</LI><LI>Wash with 600 μL ethanol (80%).</LI><LI>Centrifuge 5 min at 13000 rpm. </LI><LI>Discard the ethanol by pipeting and let it dry a few minutes. </LI><LI>Resuspend it in 50-100 μL H2O miliQ or with TE buffer.</LI><LI>Store at -20°C.  </LI></UL><P class="p_notebook">Measurement of genomic concentration with nanodrop.</P><UL class="ul_notebook"><LI>Tabacco 1: 182 ng/μL (Thrown away)</LI><LI>Tabacco 2: 620 ng/μL (Stored at -20°C)</LI></UL><P class="p_notebook">Electrophoresis performed to check the genomic size of tobacco (to see if it is degradated).</P><P class="p_notebook">It is correct.</P><H4 class="date_notebook">07/10/2014</H4><P class="p_notebook">PCR of genomic extraction of tobacco in order to amplify the trichome promoter CPS2.</P><P class="p_notebook">Ordered primers</P><UL class="ul_notebook"><LI>IGEMJULO1</LI><LI>IGEMJULO2</LI></UL><P class="p_notebook">Ajust primers to a 100 uM concentration:</P><UL class="ul_notebook"><LI>IGEMJUL01 + 566 μL miliQ H2O</LI><LI>IGEMJUL02 + 691 μL miliQ H2O</LI></UL><P class="p_notebook">Use a 1:10 alicuot for PCR (10 uM).</P><P class="p_notebook">Reagents needed for PCR:</P><UL class="ul_notebook"><LI>0.5 μL template</LI><LI>10 μL buffer HF 5x</LI><LI>2 μL dNTPs</LI><LI>2.5 μL oligo R</LI><LI>2.5 μL oligo F</LI><LI>0.5 μL Pfu</LI><LI>32 μL miliQ H2O</LI></UL><P class="p_notebook">Final volume: 50 μL</P><P class="p_notebook">Parameters:</P><UL class="ul_notebook"><LI>98 °C (2 min)</LI><LI>35 cycles</LI><UL class="ul_ul_notebook"><LI>98 °C (10 sec)</LI><LI>59 °C (15 sec)</LI><LI>72 °C (45 sec)</LI></UL><LI>72 °C (7 min)</LI></UL><P class="p_notebook">We didn't get PCR product.</P><H4 class="date_notebook">07/11/2014</H4><P class="p_notebook">Repeat PCR with different parameters.</P><TABLE class="table_notebook"><TBODY><TR class="tr_notebook"><TD class="td_notebook">1 </TD><TD class="td_notebook">2 </TD><TD class="td_notebook">3 </TD><TD class="td_notebook">4 </TD><TD class="td_notebook">5</TD></TR><TR class="tr_notebook"><TD class="td_notebook">Template</TD><TD class="td_notebook">0.5 μL</TD><TD class="td_notebook">0.5 μL</TD><TD class="td_notebook">0.5 μL</TD><TD class="td_notebook">0.5 μL</TD><TD class="td_notebook">0.5 μL</TD></TR><TR class="tr_notebook"><TD class="td_notebook">Buffer (5x)</TD><TD class="td_notebook">0.5 μL</TD><TD class="td_notebook">0.5 μL</TD><TD class="td_notebook">0.5 μL</TD><TD class="td_notebook">0.5 μL</TD><TD class="td_notebook">0.5 μL</TD></TR><TR class="tr_notebook"><TD class="td_notebook">dNTPs</TD><TD class="td_notebook">2 μL</TD><TD class="td_notebook">2 μL</TD><TD class="td_notebook">2 μL</TD><TD class="td_notebook">2 μL</TD><TD class="td_notebook">2 μL</TD></TR><TR class="tr_notebook"><TD class="td_notebook">Oligo R</TD><TD class="td_notebook">2.5 μL</TD><TD class="td_notebook">2.5 μL</TD><TD class="td_notebook">2.5 μL</TD><TD class="td_notebook">2.5 μL</TD><TD class="td_notebook">2.5 μL</TD></TR><TR class="tr_notebook"><TD class="td_notebook">Oligo F</TD><TD class="td_notebook">2.5 μL</TD><TD class="td_notebook">2.5 μL</TD><TD class="td_notebook">2.5 μL</TD><TD class="td_notebook">2.5 μL</TD><TD class="td_notebook">2.5 μL</TD></TR><TR class="tr_notebook"><TD class="td_notebook">Phu</TD><TD class="td_notebook">0.5 μL</TD><TD class="td_notebook">0.5 μL</TD><TD class="td_notebook">0.5 μL</TD><TD class="td_notebook">0.5 μL</TD><TD class="td_notebook">0.5 μL</TD></TR><TR class="tr_notebook"><TD class="td_notebook">Buffer</TD><TD class="td_notebook">32 μL</TD><TD class="td_notebook">32 μL</TD><TD class="td_notebook">32 μL</TD><TD class="td_notebook">32 μL</TD><TD class="td_notebook">32 μL</TD></TR></TBODY></TABLE><P class="p_notebook">1, 2 and 5 contain buffer F; 3 and 4 contain buffer GC.</P><P class="p_notebook">PCR parameters</P><UL class="ul_notebook"><LI>98 °C (2 min)</LI><LI>35 cycles</LI><UL class="ul_ul_notebook"><LI>98 °C (10 sec)</LI><LI>1, 3, 5 -&gt; 59 °C (15 sec). 2, 4 -&gt; 55 °C (15 sec)</LI><LI>72 °C (45 sec)</LI></UL><LI>72 °C (7 min)</LI></UL><P class="p_notebook">No PCR products again.</P><P class="p_notebook">Repeat PCR again with other parameters.</P><TABLE class="table_notebook"><TBODY><TR class="tr_notebook"><TD class="td_notebook">Buffer HF </TD><TD class="td_notebook">Buffer GC</TD></TR><TR class="tr_notebook"><TD class="td_notebook">Template</TD><TD class="td_notebook">2 μL</TD><TD class="td_notebook">2 μL</TD></TR><TR class="tr_notebook"><TD class="td_notebook">Buffer (5x)</TD><TD class="td_notebook">40 μL</TD><TD class="td_notebook">40 μL</TD></TR><TR class="tr_notebook"><TD class="td_notebook">dNTPs</TD><TD class="td_notebook">8 μL</TD><TD class="td_notebook">8 μL</TD></TR><TR class="tr_notebook"><TD class="td_notebook">Oligo R</TD><TD class="td_notebook">10 μL</TD><TD class="td_notebook">10 μL</TD></TR><TR class="tr_notebook"><TD class="td_notebook">Oligo F</TD><TD class="td_notebook">10 μL</TD><TD class="td_notebook">10 μL</TD></TR><TR class="tr_notebook"><TD class="td_notebook">Phu</TD><TD class="td_notebook">2</TD><TD class="td_notebook">2 μL μL</TD></TR><TR class="tr_notebook"><TD class="td_notebook">Buffer</TD><TD class="td_notebook">128 μL</TD><TD class="td_notebook">128 μL</TD></TR></TBODY></TABLE><P class="p_notebook">Set 4 tubes with each buffer at different temperatures: 49, 52, 55, 60.</P><UL class="ul_notebook"><LI>98 °C (2 min)</LI><LI>35 cycles</LI><UL class="ul_ul_notebook"><LI>98 °C (10 sec)</LI><LI>49, 52, 55, 60 °C (15 sec)</LI><LI>72 °C (45 sec)</LI></UL><LI>72 °C (7 min)</LI></UL><P class="p_notebook">No PCR products again.</P><H4 class="date_notebook">07/14/2014</H4><P class="p_notebook">Repeat PCR again with more genomic.</P><TABLE class="table_notebook"><TBODY><TR class="tr_notebook"><TD class="td_notebook">Buffer HF </TD><TD class="td_notebook">Buffer GC</TD></TR><TR class="tr_notebook"><TD class="td_notebook">Template</TD><TD class="td_notebook">5 μL</TD><TD class="td_notebook">5 μL</TD></TR><TR class="tr_notebook"><TD class="td_notebook">Buffer (5x)</TD><TD class="td_notebook">50 μL</TD><TD class="td_notebook">50 μL</TD></TR><TR class="tr_notebook"><TD class="td_notebook">dNTPs</TD><TD class="td_notebook">10 μL</TD><TD class="td_notebook">10 μL</TD></TR><TR class="tr_notebook"><TD class="td_notebook">Oligo R</TD><TD class="td_notebook">12.5 μL</TD><TD class="td_notebook">12.5 μL</TD></TR><TR class="tr_notebook"><TD class="td_notebook">Oligo F</TD><TD class="td_notebook">12.5 μL</TD><TD class="td_notebook">12.5 μL</TD></TR><TR class="tr_notebook"><TD class="td_notebook">Phu</TD><TD class="td_notebook">2.5 μL</TD><TD class="td_notebook">2.5 μL</TD></TR><TR class="tr_notebook"><TD class="td_notebook">Buffer</TD><TD class="td_notebook">107.5 μL</TD><TD class="td_notebook">107.5 μL</TD></TR></TBODY></TABLE><P class="p_notebook">Same parameters as before except annealing temperatures which are: 50, 53, 57, 59  °C.</P><P class="p_notebook">We still without having any amplification.</P><H4 class="date_notebook">07/18/2014</H4><P class="p_notebook">Repeat the PCR with other enzyme.</P><UL class="ul_notebook"><LI>12.5 μL Q5 Master mix (2x).</LI><LI>1.25 μL forward primer 10 uM</LI><LI>1.25 μL reverse primer 10 uM</LI><LI>0.5 μL template 620 ng/μL</LI><LI>9.5 μL H2O</LI></UL><P class="p_notebook">Set 4 reactions at 50, 53, 55, 59 °C.</P><UL class="ul_notebook"><LI>98 °C (30 sec)</LI><LI>35 cycles</LI><UL class="ul_ul_notebook"><LI>98 °C (10 sec)</LI><LI>50, 53, 55, 59 °C (15 sec)</LI><LI>72 °C (45 sec)</LI></UL><LI>72 °C (2 min)</LI></UL><P class="p_notebook">The DNA fragment of interest is around 1.5 kb so we see we finally obtained amplification at 55 and 59 °C reactions.</P><H4 class="date_notebook">07/19/2014</H4><P class="p_notebook">Trichome promoter PCR product ligation in pUPD.</P><UL class="ul_notebook"><LI>1 μL pUPD</LI><LI>1 μL PCR product</LI><LI>1 μL BsmBI (5-10 ud)</LI><LI>1 μL T4 ligase (5-10 ud)</LI><LI>1.2 μL buffer ligase (3 ud)</LI><LI>6.8 μL H20</LI></UL><P class="p_notebook">Set the reaction: 25 cycles x (37°C 2 min, 16°C 5 min).</P><H4 class="date_notebook">07/20/2014</H4><P class="p_notebook">Transform and grow in Petri dishes yesterday's ligation of the trichome promoter in pUPD.</P><H4 class="date_notebook">07/21/2014</H4><P class="p_notebook">We picked colonies of the trichome promoter in pUPD and grown it in liquid culture.</P><H4 class="date_notebook">07/22/2014</H4><P class="p_notebook">We made minipreps of yesterday's liquid culture. Additionally, we have recultured them in solid growth media. </P><P class="p_notebook">Miniprep quantification:</P><TABLE class="table_notebook"><TBODY><TR class="tr_notebook"><TD class="td_notebook">Piece</TD><TD class="td_notebook">Tube</TD><TD class="td_notebook">Concentration (ng/μL)</TD><TD class="td_notebook">Volume (μL)</TD></TR><TR class="tr_notebook"><TD class="td_notebook">Trichome promoter in pUPD</TD><TD class="td_notebook">1</TD><TD class="td_notebook">317.1</TD><TD class="td_notebook">26</TD></TR><TR class="tr_notebook"><TD class="td_notebook">Trichome promoter in pUPD</TD><TD class="td_notebook">3</TD><TD class="td_notebook">354.8</TD><TD class="td_notebook">32</TD></TR></TBODY></TABLE><P class="p_notebook">Both minipreps were adjusted to 75 ng/μL.</P><P class="p_notebook">Digestions in silico performed to check the insertion:</P><TABLE class="table_notebook"><TBODY><TR class="tr_notebook"><TD class="td_notebook">Piece</TD><TD class="td_notebook">Restriction enzyme</TD><TD class="td_notebook">Expected bands</TD></TR><TR class="tr_notebook"><TD class="td_notebook">Trichome Promoter in pUPD</TD><TD class="td_notebook">NotI</TD><TD class="td_notebook">2981, 1523</TD></TR><TR class="tr_notebook"><TD class="td_notebook">EcoRV</TD><TD class="td_notebook">3942, 562</TD></TR></TBODY></TABLE><P class="p_notebook">Note: To see further details of digestion master mixes, go to the biosynthesis part, date 07/22/2014.</P><P class="p_notebook">Pieces taken from the GoldenBraid 2.0 collection were cultured in solid growth media:</P><UL class="ul_notebook"><LI>pTnos	(GB0037)</LI><LI>pGFP	(GB0059)</LI><LI>pLuciferase	(GB0096)</LI></UL><H4 class="date_notebook">07/23/2014</H4><P class="p_notebook">Yesterday's digestions were correct, so the trichome promoter in pUPD was send to sequencing.</P><P class="p_notebook">We picked colonies from pTnos, pGFP and pLuciferase.</P><H4 class="date_notebook">07/24/2014</H4><P class="p_notebook">Results of sequencing the promoter were obtained:</P><TABLE class="table_notebook"><TBODY><TR class="tr_notebook"><TD class="td_notebook">Mutation</TD><TD class="td_notebook">Position</TD></TR><TR class="tr_notebook"><TD class="td_notebook">T insertion</TD><TD class="td_notebook">??</TD></TR><TR class="tr_notebook"><TD class="td_notebook">T insertion</TD><TD class="td_notebook">??</TD></TR></TBODY></TABLE><P class="p_notebook">Minipreps of pTnos, pGFP and pLuciferase.</P><H4 class="date_notebook">07/28/2014</H4><TABLE class="table_notebook"><TBODY><TR class="tr_notebook"><TD class="td_notebook">Piece</TD><TD class="td_notebook">Concentration (ng/μL)</TD><TD class="td_notebook">Initial Volume (μL)</TD><TD class="td_notebook">Final Volume (μL)</TD></TR><TR class="tr_notebook"><TD class="td_notebook">GFP</TD><TD class="td_notebook">318.8</TD><TD class="td_notebook">35</TD><TD class="td_notebook">148.8</TD></TR><TR class="tr_notebook"><TD class="td_notebook">Tnos</TD><TD class="td_notebook">400.8</TD><TD class="td_notebook">35</TD><TD class="td_notebook">186.5</TD></TR><TR class="tr_notebook"><TD class="td_notebook">pLuciferase</TD><TD class="td_notebook">NotI</TD><TD class="td_notebook">2981, 1731</TD></TR></TBODY></TABLE><P class="p_notebook">See master mix and gel digestion in Biosynthesis part. Pieces were obtained correctly and adjusted to 75 ng/μL.</P><P class="p_notebook">The following table shows ligation details of the trichome promoter:</P><TABLE class="table_notebook"><TBODY><TR class="tr_notebook"><TD class="td_notebook">Reagent</TD><TD class="td_notebook">Volume</TD></TR><TR class="tr_notebook"><TD class="td_notebook">CPS2</TD><TD class="td_notebook">1 μL</TD></TR><TR class="tr_notebook"><TD class="td_notebook">GFP</TD><TD class="td_notebook">1 μL</TD></TR><TR class="tr_notebook"><TD class="td_notebook">TNos</TD><TD class="td_notebook">1 μL</TD></TR><TR class="tr_notebook"><TD class="td_notebook">BsaI</TD><TD class="td_notebook">1 μL</TD></TR><TR class="tr_notebook"><TD class="td_notebook">p2α2</TD><TD class="td_notebook">1 μL</TD></TR><TR class="tr_notebook"><TD class="td_notebook">T4 ligase</TD><TD class="td_notebook">1 μL</TD></TR><TR class="tr_notebook"><TD class="td_notebook">Ligase buffer</TD><TD class="td_notebook">1 μL</TD></TR><TR class="tr_notebook"><TD class="td_notebook">H2O</TD><TD class="td_notebook">3 μL</TD></TR><TR class="tr_notebook"><TD class="td_notebook">Total Volume</TD><TD class="td_notebook">10 μL</TD></TR></TBODY></TABLE><H4 class="date_notebook">07/29/2014</H4><P class="p_notebook">Trichome Promoter transformation in <I>E. coli</I>.</P><P class="p_notebook">Using an electrocompetent <I>E. coli</I> strain (DH5α) and 1.5 ul ligation (CPS2:GFP:TNos in 2α2), the mix is electroporated at 1500 V. Then, 300 μL of SOC are added and stored at 37 °C with agitation. </P><H4 class="date_notebook">07/30/2014</H4><P class="p_notebook">Pick colonies of CPS2:GFP:TNos in 2α2.</P><H4 class="date_notebook">07/31/2014</H4><P class="p_notebook">Minipreps of yesterday's culture were made, obtaining the transcripional unit: PCPS2:GFP:TNos in 2 α2</P><P class="p_notebook">Additionally, we recultured in petri dish with its respective antibiotic (Kan).</P><P class="p_notebook">Digestions in silico made for checking minipreps:</P><TABLE class="table_notebook"><TBODY><TR class="tr_notebook"><TD class="td_notebook">Pieces/TU</TD><TD class="td_notebook">Restriction enzyme</TD><TD class="td_notebook">Expected bands</TD></TR><TR class="tr_notebook"><TD class="td_notebook">CPS2:GFP:TNos in 2α2</TD><TD class="td_notebook">HindIII</TD><TD class="td_notebook">6322, 2694</TD></TR><TR class="tr_notebook"><TD class="td_notebook">EcoRV</TD><TD class="td_notebook">8454, 562</TD></TR></TBODY></TABLE><P class="p_notebook">Digestion mixes:</P><UL class="ul_notebook"><LI>Master mix for EcoRV:</LI><UL class="ul_ul_notebook"><LI>3 μL EcoRV</LI><LI>15 μL Red buffer</LI><LI>126 μL H20</LI></UL><LI>Master mix for HindIII:</LI><UL class="ul_ul_notebook"><LI>2 μL HindIII</LI><LI>10 μL Red buffer</LI><LI>84 μL H2O</LI></UL></UL><P class="p_notebook">Note: We made master mixes because digestions were made simultaneously with the biosynthesis part. </P><P class="p_notebook">An agarose gel was made to check the transcriptional unit:</P><P class="p_notebook">Minipreps of CPS2:GFP:TNos in 2α2 (1) went correctly.</P><P class="p_notebook">Miniprep results were quantified and then adjusted at 75 ng/μL:</P><TABLE class="table_notebook"><TBODY><TR class="tr_notebook"><TD class="td_notebook">Pieces/TU</TD><TD class="td_notebook">Tube</TD><TD class="td_notebook">Concentration (ug/μL)</TD><TD class="td_notebook">Initial volume (μL)</TD><TD class="td_notebook">Final Volume (μL)</TD></TR><TR class="tr_notebook"><TD class="td_notebook">PCPS2:GFP:TNos in 2α2</TD><TD class="td_notebook">1</TD><TD class="td_notebook">128.5</TD><TD class="td_notebook">33</TD><TD class="td_notebook">56.5</TD></TR><TR class="tr_notebook"><TD class="td_notebook">PCPS2:GFP:TNos in 2α2</TD><TD class="td_notebook">2</TD><TD class="td_notebook">135.9</TD><TD class="td_notebook">34</TD><TD class="td_notebook">61.6</TD></TR><TR class="tr_notebook"><TD class="td_notebook">PCPS2:GFP:TNos in 2α2</TD><TD class="td_notebook">3</TD><TD class="td_notebook">126.2</TD><TD class="td_notebook">35</TD><TD class="td_notebook">58.9</TD></TR></TBODY></TABLE><H4 class="date_notebook">08/05/2014</H4><P class="p_notebook">Transcriptional Unit (TU) PCPS2:GFP:TNos in 2α2 was transformed in <I><I>Agrobacterium</I> tumefaciens</I> (C58) and cultured in liquid media with Kan and Rif at 1:1000 (2 days at 28°C).</P><P class="p_notebook">Note: The scientific name has been updated to Rhizobium radiobacter. </P><H4 class="date_notebook">08/08/2014</H4><P class="p_notebook">The TU (PCPS2:GFP:TNos) was recultured in liquid media. Additionally, P35S:GFP:p19:TNos TU was recultured in liquid media, using Spm and Rif as antibiotics.</P><H4 class="date_notebook">08/11/2014</H4><P class="p_notebook"><I>Agrobacterium</I> cultures were refreshed in new liquid media. Additionally, we cultured them in solid media.</P><P class="p_notebook">Minipreps of the TU PCPS2:GFP:TNos in <I>Agrobacterium</I> were made. and digestions were performed to check they were correct.</P><P class="p_notebook">Digestions in silico performed to check the insertion:</P><TABLE class="table_notebook"><TBODY><TR class="tr_notebook"><TD class="td_notebook">Restriction enzyme</TD><TD class="td_notebook">Expected bands</TD></TR><TR class="tr_notebook"><TD class="td_notebook">HindIII</TD><TD class="td_notebook">6322, 2694</TD></TR><TR class="tr_notebook"><TD class="td_notebook">EcoRV</TD><TD class="td_notebook">8454, 562</TD></TR></TBODY></TABLE><P class="p_notebook">PCPS2:GFP:TNos (1) digestion was correct.</P><H4 class="date_notebook">08/12/2014</H4><P class="p_notebook">A part containing P35S:P19:TNos construction was taken from the GoldenBraid collection (GB108) and cultured in solid media with Kanamycin 50 mg/mL. This part is not going to be used as a control but as a silencing supressor.</P><H4 class="date_notebook">08/13/2014</H4><P class="p_notebook">One clony (P35S:P19:TNos) was recultured in liquid media.</P><H4 class="date_notebook">08/14/2014</H4><P class="p_notebook">Minipreps and streaks of yesterday's culture were made.</P><P class="p_notebook">The piece was checked by running a gel containing the digested fragment. </P><P class="p_notebook">Digestions made in silico:</P><TABLE class="table_notebook"><TBODY><TR class="tr_notebook"><TD class="td_notebook">Piece</TD><TD class="td_notebook">Restriction enzyme</TD><TD class="td_notebook">Expected bands</TD></TR><TR class="tr_notebook"><TD class="td_notebook">P35S:P19:TNos</TD><TD class="td_notebook">BanI</TD><TD class="td_notebook">4256, 392</TD></TR><TR class="tr_notebook"><TD class="td_notebook">HindIII</TD><TD class="td_notebook">788, 1287, 2563</TD></TR></TBODY></TABLE><P class="p_notebook">The GB108 piece (P35S:P19:TNos) is digested as expected in silico. </P><H4 class="date_notebook">08/15/2014</H4><P class="p_notebook">We transformed the piece (P35S:P19:T35S) into <I><I>Agrobacterium</I> tumefaciens</I> (C58) and we cultured it in solid media with Kan (1:1000) and Rif (1:1000) at 28°C during 2 days. </P><H4 class="date_notebook">08/17/2014</H4><P class="p_notebook"><I>Agrobacterium</I> containing the piece has not growm well, so we transformed the piece again and we cultured it in an agar plate following the same protocol as previously. In the mean time, we made agar plates.</P><P class="p_notebook">We made ligations of the three enzymes that form the (Z)11-16:OAc (Z11-hexadecenyl acetate) pheromone but this time each TU will contain the trichome promoter. </P><P class="p_notebook">Note: For further information about the PCPS2 promoter, please check the trichome promoter section. </P><P class="p_notebook">Ligation reagents:</P><UL class="ul_notebook"><LI>PCPS2:AtrΔ11:T35S</LI><UL class="ul_ul_notebook"><LI>1 μL PCPS2</LI><LI>1 μL T35S</LI><LI>1 ul AtrΔ11</LI><LI>1.5 μL 2α1</LI><LI>1 μL Buffer ligase 10X</LI><LI>1 μL T4</LI><LI>1 μL BsaI</LI><LI>2.5 μL H2O</LI></UL></UL><UL class="ul_notebook"><LI>PCPS2:HarFAR:T35S and PCPS2:EaDAcT:T35S</LI><UL class="ul_ul_notebook"><LI>1 μL PCPS2</LI><LI>1 μL T35S</LI><LI>1 ul AtrΔ11/EaDAcT</LI><LI>1 μL 2α2</LI><LI>1 μL Buffer ligase 10X</LI><LI>1 μL T4</LI><LI>1 μL BsaI</LI><LI>3 μL H2O</LI></UL></UL><H4 class="date_notebook">08/18/2014</H4><P class="p_notebook">TU containing the trichome promoter were transformed into <I>E. coli</I>. </P><UL class="ul_notebook"><LI>PCPS2:AtrΔ11:T35S</LI><LI>PCPS2:HarFAR:T35S</LI><LI>PCPS2:EaDAcT:T35S</LI></UL><H4 class="date_notebook">08/19/2014</H4><P class="p_notebook"><I>Agrobacterium</I> has not grown in agarose plates, so we made a transformation again.</P><P class="p_notebook"><I>E. coli</I> colonies containing the TUs were recultured in liquid media:</P><UL class="ul_notebook"><LI>PCPS2:AtrΔ11:T35S in 2α1</LI><LI>PCPS2:HarFAR:T35S in 2α2</LI><LI>PCPS2:EaDAcT:T35S in 2α2</LI></UL><H4 class="date_notebook">08/20/2014</H4><P class="p_notebook">We made minipreps of yesterday's culture and to check them we made digestions in silico:</P><TABLE class="table_notebook"><TBODY><TR class="tr_notebook"><TD class="td_notebook">Piece</TD><TD class="td_notebook">Enzyme</TD><TD class="td_notebook">Expected bands</TD></TR><TR class="tr_notebook"><TD class="td_notebook">PCPS2:AtrΔ11:T35S</TD><TD class="td_notebook">EcoRV</TD><TD class="td_notebook">562, 8448</TD></TR><TR class="tr_notebook"><TD class="td_notebook">EcoRI</TD><TD class="td_notebook">2687, 6323</TD></TR><TR class="tr_notebook"><TD class="td_notebook">PCPS2:HarFAR:T35S</TD><TD class="td_notebook">HindIII</TD><TD class="td_notebook">933, 2140, 6322</TD></TR><TR class="tr_notebook"><TD class="td_notebook">EcoRV</TD><TD class="td_notebook">562, 8833</TD></TR><TR class="tr_notebook"><TD class="td_notebook">PCPS2:EaDAcT:T35S</TD><TD class="td_notebook">HindIII</TD><TD class="td_notebook">2800, 6322</TD></TR><TR class="tr_notebook"><TD class="td_notebook">EcoRV</TD><TD class="td_notebook">7363, 1197, 562</TD></TR></TBODY></TABLE><P class="p_notebook">Digestions were correct, but the PCPS2:HarFAR:T35S digestion 1 with HindIII resulted in more bands than expected, so we discarded that miniprep product and we used the other one. </P><P class="p_notebook">We adjusted checked products to 75 ng/μL in order to use them in ligations. </P><P class="p_notebook">We ligated the TUs containing the trichome promoter in Ω vectors as follows:</P><UL class="ul_notebook"><LI>Ligation 1 (Vt = 10 μL):</LI><UL class="ul_ul_notebook"><LI>1 μL PCPS2:AtrΔ11:T35S</LI><LI>1 μL PCPS2:HarFAR:T35S</LI><LI>1 μL 2Ω1</LI><LI>1 μL BsmBI</LI><LI>1 μL T4 ligase</LI><LI>1 μL Buffer ligase</LI><LI>4 μL H2O miliQ</LI></UL></UL><UL class="ul_notebook"><LI>Ligation 2 (Vt = 10 μL):</LI><UL class="ul_ul_notebook"><LI>1 μL SF (Stuffer fragment)</LI><LI>1 μL PCPS2:EaDAcT:T35S</LI><LI>1 μL 2Ω2</LI><LI>1 μL BsmBI</LI><LI>1 μL T4 ligase</LI><LI>1 μL Buffer ligase</LI><LI>4 μL H2O miliQ</LI></UL></UL><P class="p_notebook">Reaction conditions: 25 cycles x (37°C 2 min, 16°C 5 min).</P><P class="p_notebook">Then, we recultured <I>E. coli</I> in solid media.</P><P class="p_notebook">TUs ligated previously were transformed in <I>E. coli</I> following the same protocol as it is usually used. </P><P class="p_notebook">Finally, we obtained the control (Z)11-16Hexadecenl Acetate that will be used to check the peack in the GC-MS analysis. </P><H4 class="date_notebook">08/21/2014</H4><P class="p_notebook"><I>Agrobacterium</I> cells containing P35S:P19:TNos did not grow, so we ask Marta for the glycerinated <I>Agrobacterium</I> culture.</P><P class="p_notebook">The vector containing the TU was pGreen and we cultured them with Tetracycline, Rifampicin and Kanamycin. </P><P class="p_notebook">We have confirmed our peak because the control sample has the same retention time and distribution pattern. </P><P class="p_notebook">Additionally, we have recultured in liquid media TUs ligated yesterday. </P><H4 class="date_notebook">08/22/2014</H4><P class="p_notebook">We made minipreps of yesterday's culture:</P><UL class="ul_notebook"><LI>PCPS2:EaDAcT:T35S in 2Ω2</LI><LI>PCPS2:AtrΔ11:T35S_PCPS2:HarFAR:T35S in 2Ω1</LI></UL><P class="p_notebook">Digestions in silico made to check minipreps:</P><TABLE class="table_notebook"><TBODY><TR class="tr_notebook"><TD class="td_notebook">Piece</TD><TD class="td_notebook">Enzyme</TD><TD class="td_notebook">Expected bands</TD></TR><TR class="tr_notebook"><TD class="td_notebook">PCPS2:AtrΔ11:T35S_PCPS2:HarFAR:T35S</TD><TD class="td_notebook">BamHI</TD><TD class="td_notebook">6652, 5742</TD></TR><TR class="tr_notebook"><TD class="td_notebook">NotI</TD><TD class="td_notebook">9572, 1532, 1290</TD></TR><TR class="tr_notebook"><TD class="td_notebook">PCPS2:EaDAcT:T35S</TD><TD class="td_notebook">NotI</TD><TD class="td_notebook">6792, 1532, 1290</TD></TR><TR class="tr_notebook"><TD class="td_notebook">NdeI</TD><TD class="td_notebook">7125, 2419</TD></TR></TBODY></TABLE><P class="p_notebook">All digestions were not correct. </P><H4 class="date_notebook">08/24/2014</H4><P class="p_notebook">We transformed in <I>Agrobacterium</I> the following TUs:</P><UL class="ul_notebook"><LI>PCPS2:AtrΔ11:T35S_PCPS2:HarFAR:T35S in 2Ω1</LI><LI>PCPS2:EaDAcT:T35S in 2Ω2</LI></UL><P class="p_notebook">We made minipreps of <I>Agrobacterium</I> culture: P35S:AtrΔ11:T35S_P35S:HarFAR:T35S_P35S:EaDAcT:T35S in 2α1.</P><P class="p_notebook">Additionally, we refreshed <I>Agrobacterium</I> cultures with their corresponding antibiotic:</P><UL class="ul_notebook"><LI>T35S:P19:TNos (Rif, Kan, Tet)</LI><LI>PCPS2:GFP:TNos (Rif, Kan)</LI><LI>T35S:P19:GFP:TNos (Rif, Smp, Tet)</LI><LI>TUs: P35S:AtrΔ11:T35S_P35S:HarFAR:T35S_P35S:EaDAcT:T35S in 2α1 (Rif, Kan)</LI></UL><H4 class="date_notebook">08/25/2014</H4><P class="p_notebook">Digestions in silico made to check yesterday's minipreps:</P><TABLE class="table_notebook"><TBODY><TR class="tr_notebook"><TD class="td_notebook">Piece</TD><TD class="td_notebook">Enzyme</TD><TD class="td_notebook">Expected bands</TD></TR><TR class="tr_notebook"><TD class="td_notebook">P35S:AtrΔ11:T35S_P35S:HarFAR:T35S_P35S:EaDAcT:T35S</TD><TD class="td_notebook">EcoRI</TD><TD class="td_notebook">7428, 6323</TD></TR><TR class="tr_notebook"><TD class="td_notebook">BglII</TD><TD class="td_notebook">2576, 11175</TD></TR></TBODY></TABLE><P class="p_notebook">We repeated the Agroinfiltration protocol, but this time we infiltrated the following <I>A. tumefaciens</I> cultures:</P><UL class="ul_notebook"><LI>T35S:P19:TNos </LI><LI>PCPS2:GFP:TNos + T35S:P19:TNos</LI><LI>T35S:P19:GFP:TNos</LI><LI>T35S:P19:GFP:TNos + P35S:AtrΔ11:T35S_P35S:HarFAR:T35S_P35S:EaDAcT:T35S</LI></UL><P class="p_notebook">We picked colonies which were transformed yesterday and we recultured them in liquid media with Spm, IPTG and X-Gal. </P><P class="p_notebook">Finally, we have trasplanted <I>N. benthamiana</I> into new flowerpots to have plants ready to infiltrate in the future. </P><H4 class="date_notebook">08/26/2014</H4><P class="p_notebook">We made minipreps of yesterday's culture, but only for the TU PCPS2:AtrΔ11:T35S_PCPS2:HarFAR:T35S in 2Ω1 since the other tubes were blue colored. </P><P class="p_notebook">Digestions in silico the check the minipreps:</P><TABLE class="table_notebook"><TBODY><TR class="tr_notebook"><TD class="td_notebook">Piece</TD><TD class="td_notebook">Enzyme</TD><TD class="td_notebook">Expected bands</TD></TR><TR class="tr_notebook"><TD class="td_notebook">PCPS2:AtrΔ11:T35S_PCPSS:HarFAR:T35S</TD><TD class="td_notebook">BamHI</TD><TD class="td_notebook">6652, 5742</TD></TR><TR class="tr_notebook"><TD class="td_notebook">BglII</TD><TD class="td_notebook">8131, 2669, 1594</TD></TR></TBODY></TABLE><H3 class="section_notebook">Biosafety module</H3><H4 class="date_notebook">07/22/2014</H4><P class="p_notebook">Pieces taken from the GoldenBraid 2.0 collection were cultured in solid growth media:</P><UL class="ul_notebook"><LI>P35S:Rosea:TNos</LI><LI>TA29:Barnase:TNos (from GoldenBraid 1.0 collection)</LI></UL><P class="p_notebook">We were told by our advisor that Rosea produces necrosis in <I>N. benthamiana</I>, so we must think of an alternative.</P><H4 class="date_notebook">07/23/2014</H4><P class="p_notebook">We picked colonies from P35S:Rosea:TNos and TA29:Barnase:TNos.</P><H4 class="date_notebook">07/24/2014</H4><P class="p_notebook">Minipreps of P35S:Rosea:TNos and TA29:Barnase:TNos.</P><H4 class="date_notebook">07/25/2014</H4><P class="p_notebook">Digestions in silico made for checking yesterday's minipreps:</P><TABLE class="table_notebook"><TBODY><TR class="tr_notebook"><TD class="td_notebook">Pieces</TD><TD class="td_notebook">Restriction enzyme</TD><TD class="td_notebook">Expected bands</TD></TR><TR class="tr_notebook"><TD class="td_notebook">P35S:Rosea:Tnos</TD><TD class="td_notebook">BglII</TD><TD class="td_notebook">2495, 2302</TD></TR><TR class="tr_notebook"><TD class="td_notebook">NcoI</TD><TD class="td_notebook">4407, 390</TD></TR><TR class="tr_notebook"><TD class="td_notebook">TA29:Barnase:Tnos</TD><TD class="td_notebook">BglII</TD><TD class="td_notebook">2825, 2245</TD></TR></TBODY></TABLE><H4 class="date_notebook">07/28/2014</H4><P class="p_notebook">See master mix and gel digestion in Biosynthesis part. Pieces were obtained correctly and adjusted to 75 ng/μL.</P><H4 class="date_notebook">07/31/2014</H4><P class="p_notebook">We talked with the NRP-UEA-Norwich team. We stablished a possible collaboration in developing the biosafety module together. They could send us their chromoproteins and we could send them our barnase and TA29 promoter.</P><H4 class="date_notebook">08/05/2014</H4><P class="p_notebook">Order primers for TA29 and barnase:</P><TABLE class="table_notebook"><TBODY><TR class="tr_notebook"><TD class="td_notebook">Name</TD><TD class="td_notebook">Sequence</TD><TD class="td_notebook">T annealing</TD></TR><TR class="tr_notebook"><TD class="td_notebook">I14Ago01_TA29_F1</TD><TD class="td_notebook">CGCCGTCTCGCTCGGGAGTAGCGAATGCAATTAATTTAGACAT</TD><TD class="td_notebook">61.8°C</TD></TR><TR class="tr_notebook"><TD class="td_notebook">I14Ago02_TA29_R1</TD><TD class="td_notebook">CGCCGTCTCGCTCGCATTTTTAGCTAATTTCTTTAAGTAAAAACTTTG</TD><TD class="td_notebook">60.8°C</TD></TR><TR class="tr_notebook"><TD class="td_notebook">I14Ago03_barnase_F1</TD><TD class="td_notebook">CGCCGTCTCGCTCGAATGGCACAGGTTATCAACACG</TD><TD class="td_notebook">65.0°C</TD></TR><TR class="tr_notebook"><TD class="td_notebook">I14Ago04_barnase_R1</TD><TD class="td_notebook">CGCCGTCTCGCTCGAAGCTTATCTGATTTTTGTAAAGGTCTGATAATG</TD><TD class="td_notebook">63.4°C</TD></TR></TBODY></TABLE><H4 class="date_notebook">08/07/2014</H4><P class="p_notebook">Primers received. PCR for barnase and TA29 performed.</P><UL class="ul_notebook"><LI>TA29 PCR parameters</LI><UL class="ul_ul_notebook"><LI>98°C, 2 min</LI><LI>35 cycles</LI><LI>98°C, 10 s</LI><LI>60°C, 18 s</LI><LI>72°C, 40 s</LI><LI>72°C, 7 min</LI></UL><LI>Barnase PCR parameters</LI><UL class="ul_ul_notebook"><LI>98°C, 2 min</LI><LI>35 cycles</LI><LI>98°C, 10 s</LI><LI>63°C, 18 s</LI><LI>72°C, 40 s</LI><LI>72°C, 7 min</LI></UL></UL><P class="p_notebook">We didn't obtain PCR product. There is a band for the barnase, but it should be around 330 bp.</P><H4 class="date_notebook">08/08/2014</H4><P class="p_notebook">We repeat yesterday's PCR with 2 degrees less in the annealing step.</P><P class="p_notebook">Results obtained are the same of yesterday's. We should think about charging something else.</P><H4 class="date_notebook">08/11/2014</H4><P class="p_notebook">The previous PCR was repeated changing the annealing temperature to 61°C. </P><P class="p_notebook">We still do not get PCR product.</P><H4 class="date_notebook">08/12/2014</H4><P class="p_notebook">We forgot to adjust the TA29:Barnase:Tnos from GB 1.0 to 5 ng/μL. Maybe that's why PCRs don't work. We repeated again with the appropiate temperatures (60°C for TA29 and 63°C for barnase), but it still doesn't work!</P><H4 class="date_notebook">08/18/2014</H4><P class="p_notebook">We transformed in <I>E. coli</I> the iGEM Barnase part (BBa_1716211), placed in Plate 3, 11o.</P><P class="p_notebook">?? --&gt; Se puso una PCR o no?</P><P class="p_notebook">A PCR using Nicotiana tobacum genome as a template was made to obtain the Ta29 fragment. Primers used and also PCR conditions were the same as previous PCRs. </P><H4 class="date_notebook">08/19/2014</H4><P class="p_notebook"><I>E. coli</I> colonies containing the iGEM Barnase part (BBa_I716.211) were recultured in liquid media.</P><H4 class="date_notebook">08/20/2014</H4><P class="p_notebook">We made minipreps of yesterday's culture and to check them we made digestions in silico:</P><TABLE class="table_notebook"><TBODY><TR class="tr_notebook"><TD class="td_notebook">Piece</TD><TD class="td_notebook">Enzyme</TD><TD class="td_notebook">Expected bands</TD></TR><TR class="tr_notebook"><TD class="td_notebook">Barnase</TD><TD class="td_notebook">NotI</TD><TD class="td_notebook">2046, 357</TD></TR><TR class="tr_notebook"><TD class="td_notebook">BamHI</TD><TD class="td_notebook">1558, 845</TD></TR></TBODY></TABLE><P class="p_notebook">Digestions were correct, so we adjusted the product to 5 ng/μL in order to use them as a PCR template. </P><P class="p_notebook">Adittionally, we made a ligation to obtain the TA29 piece in pUPD vector as follows:</P><UL class="ul_notebook"><LI>1 μL pUPD</LI><LI>1 μL TA29</LI><LI>1 μL BsmBI</LI><LI>1.2 μL Buffer ligase</LI><LI>1 μL T4 ligase</LI><LI>6.8 μL H2O miliQ</LI></UL><P class="p_notebook">Reaction conditions: 25 cycles x (37°C 2 min, 16°C 5 min).</P><P class="p_notebook">We made a mistake predicting digetions in silico, so we repeated them, this time with the appropriate vector (pSB1C3). </P><TABLE class="table_notebook"><TBODY><TR class="tr_notebook"><TD class="td_notebook">Piece</TD><TD class="td_notebook">Enzyme</TD><TD class="td_notebook">Expected bands</TD></TR><TR class="tr_notebook"><TD class="td_notebook">Barnase</TD><TD class="td_notebook">EcoRI and PstI</TD><TD class="td_notebook">2029, 374</TD></TR></TBODY></TABLE><P class="p_notebook">This double digestion was checked with an agarose gel showing that the resulting bands were the expected ones.</P><P class="p_notebook">Additionally, TA29 in pUPD vector was transformed in <I>E. coli</I>. The protocol followed was the same as previously done. </P><H4 class="date_notebook">08/21/2014</H4><P class="p_notebook">We did a PCR to obtain the Barnase as a product using the primers Bar_F1 and Bar_R1 and the template obtained yesterday.</P><P class="p_notebook">PCR conditions (35 cycles):</P><TABLE class="table_notebook"><TBODY><TR class="tr_notebook"><TD class="td_notebook">Step</TD><TD class="td_notebook">Temperature (°C)</TD><TD class="td_notebook">Time </TD></TR><TR class="tr_notebook"><TD class="td_notebook">Initialization</TD><TD class="td_notebook">98</TD><TD class="td_notebook">1:30 min</TD></TR><TR class="tr_notebook"><TD class="td_notebook">Denaturation</TD><TD class="td_notebook">98</TD><TD class="td_notebook">10 s</TD></TR><TR class="tr_notebook"><TD class="td_notebook">Annealing</TD><TD class="td_notebook">63</TD><TD class="td_notebook">20 s</TD></TR><TR class="tr_notebook"><TD class="td_notebook">Extension</TD><TD class="td_notebook">72</TD><TD class="td_notebook">20 s</TD></TR><TR class="tr_notebook"><TD class="td_notebook">Final elongation</TD><TD class="td_notebook">72</TD><TD class="td_notebook">7 min</TD></TR></TBODY></TABLE><P class="p_notebook">The agarose gel shows that the PCR product was correct, but we purified the band to get a better quality product using a QUIAGEN purification kit (QIAEXII Gel Extraction Kit 150, Cat. No: 20021).</P><P class="p_notebook">We recultured in liquid media yesterday's TA29 culture.</P><H4 class="date_notebook">08/22/2014</H4><P class="p_notebook">We made minipreps of yesterday's culture. </P><TABLE class="table_notebook"><TBODY><TR class="tr_notebook"><TD class="td_notebook">Piece</TD><TD class="td_notebook">Restriction enzyme</TD><TD class="td_notebook">Expected bands</TD></TR><TR class="tr_notebook"><TD class="td_notebook">TA29 in pUPD</TD><TD class="td_notebook">EcoRI</TD><TD class="td_notebook">2997, 817</TD></TR><TR class="tr_notebook"><TD class="td_notebook">PvuI</TD><TD class="td_notebook">2818, 1096</TD></TR></TBODY></TABLE><P class="p_notebook">Digestions were not correct. We picked again TA29 in pUPD colonies and recultured them in liquid media. </P><H4 class="date_notebook">08/24/2014</H4><P class="p_notebook">We made minipreps of TA29 culture.</P><H4 class="date_notebook">08/25/2014</H4><P class="p_notebook">In silico digestions made to check yesterday's minipreps.</P><TABLE class="table_notebook"><TBODY><TR class="tr_notebook"><TD class="td_notebook">Piece</TD><TD class="td_notebook">Restriction enzyme</TD><TD class="td_notebook">Expected bands</TD></TR><TR class="tr_notebook"><TD class="td_notebook">TA29 in pUPD</TD><TD class="td_notebook">EcoRI</TD><TD class="td_notebook">2997, 817</TD></TR><TR class="tr_notebook"><TD class="td_notebook">PvuI</TD><TD class="td_notebook">2818, 1096</TD></TR></TBODY></TABLE><P class="p_notebook">Resulting bands were as expected in silico, the piece is correct.</P><H3 class="section_notebook">Measurement Interlab Study</H3><H4 class="date_notebook">08/20/2014</H4><P class="p_notebook">We transformed BBa_J23101, BBa_E0240 and BBa_J23115. All of the pieces share the vector pSB1C3, so we have cultured them in solid LB medium supplemented with chloramphenicol. </P><H4 class="date_notebook">08/21/2014</H4><P class="p_notebook">Pick colonies and grow them in agitation at 37°C in liquid media supplemented with chloramphenicol.</P><H4 class="date_notebook">08/22/2014</H4><P class="p_notebook">We made minipreps of yesterday's culture, except from BBa_E0240 culture, which has not grown.</P><P class="p_notebook">Digestions in silico:</P><TABLE class="table_notebook"><TBODY><TR class="tr_notebook"><TD class="td_notebook">Part</TD><TD class="td_notebook">Enzyme</TD><TD class="td_notebook">Expected bands</TD></TR><TR class="tr_notebook"><TD class="td_notebook">BBa_J23101</TD><TD class="td_notebook">RsaI</TD><TD class="td_notebook">1567, 538</TD></TR><TR class="tr_notebook"><TD class="td_notebook">XhoI</TD><TD class="td_notebook">1213, 892</TD></TR><TR class="tr_notebook"><TD class="td_notebook">BBa_23115</TD><TD class="td_notebook">RsaI</TD><TD class="td_notebook">1199, 538, 368</TD></TR><TR class="tr_notebook"><TD class="td_notebook">XhoI</TD><TD class="td_notebook">1213, 892</TD></TR></TBODY></TABLE><P class="p_notebook">All digestions were correct except BBa_23101 (1). </P><H4 class="date_notebook">08/24/2014</H4><P class="p_notebook">BBa_E0240 and BBa_I20260 parts were transformed in <I>E. coli</I> DH5-α. BBa_E0240 is resistant to kanamycin and BBa_I20260 to chloramphenicol.</P><H4 class="date_notebook">08/25/2014</H4><P class="p_notebook">Colonies did not grow so plates were left one more day at 37ºC.</P><H4 class="date_notebook">08/26/2014</H4><P class="p_notebook">Pick colonies of BBa_E0240 and grow them in agitation at 37°C in liquid media supplemented with kanamycin.</P><P class="p_notebook">Colonies of BBa_I20260 were not grown, so we performed transformation again.</P><H4 class="date_notebook">08/27/2014</H4><P class="p_notebook">Pick colonies of BBa_I2026 grow them in agitation at 37°C in liquid media supplemented with kanamycin.</P><P class="p_notebook">Minipreps and digestions of BBa_E0240.</P><TABLE class="table_notebook"><TBODY><TR class="tr_notebook"><TD class="td_notebook">Part</TD><TD class="td_notebook">Enzyme</TD><TD class="td_notebook">Expected bands</TD></TR><TR class="tr_notebook"><TD class="td_notebook">BBa_E0240</TD><TD class="td_notebook">NcoI</TD><TD class="td_notebook">1991, 955</TD></TR></TBODY></TABLE><P class="p_notebook">Assembly protocol for BBa_J23101+BBa_E0240 and BBa_J23115+BBa_E0240:</P><P class="p_notebook">Double digestions</P><UL class="ul_notebook"><LI>250 ng of plasmid in 16 μL H20</LI><LI>2.5 μL NEBuffer</LI><LI>0.5 μL BSA</LI><LI>0.5 μL enzyme 1</LI><LI>0.5 μL enzyme 2</LI></UL><P class="p_notebook">Final volume: 20 μL</P><TABLE class="table_notebook"><TBODY><TR class="tr_notebook"><TD class="td_notebook">Part</TD><TD class="td_notebook">Enzymes</TD><TD class="td_notebook">Size</TD></TR><TR class="tr_notebook"><TD class="td_notebook">BBa_J23101</TD><TD class="td_notebook">SpeI, PstI</TD><TD class="td_notebook">2.1 kb</TD></TR><TR class="tr_notebook"><TD class="td_notebook">BBa_J23115</TD><TD class="td_notebook">SpeI, PstI</TD><TD class="td_notebook">2.1 kb</TD></TR><TR class="tr_notebook"><TD class="td_notebook">BBa_E0240</TD><TD class="td_notebook">XbaI, PstI</TD><TD class="td_notebook">800 bp</TD></TR></TBODY></TABLE><P class="p_notebook">Incubate 30 min at 37°C and 20 min more at 80°C.</P><P class="p_notebook">Run digestions in an agarose gel and purify band using QIAEX II Gel Extraction Kit.</P><P class="p_notebook">BioBricks ligations</P><UL class="ul_notebook"><LI>2 μL part 1 (25 ng)</LI><LI>2 μL part 2 (25 ng)</LI><LI>1 μL T4 buffer 10X</LI><LI>0.5 μL T4</LI><LI>4 μL H20</LI></UL><TABLE class="table_notebook"><TBODY><TR class="tr_notebook"><TD class="td_notebook">Part 1</TD><TD class="td_notebook">Part2</TD></TR><TR class="tr_notebook"><TD class="td_notebook">BBa_J23101</TD><TD class="td_notebook">BBa_E0240</TD></TR><TR class="tr_notebook"><TD class="td_notebook">BBa_J23115</TD><TD class="td_notebook">BBa_E0240</TD></TR></TBODY></TABLE><P class="p_notebook">Incubate 30 min at 16°C and 20 min more at 80°C.</P><P class="p_notebook">Transform both ligations (BBa_J23101+BBa_E0240 and BBa_J23115+BBa_E0240) and grow in solid plates supplemented with chloramphenicol.</P><H4 class="date_notebook">08/28/2014</H4><P class="p_notebook">Colonies did not grow so plates were left one more day at 37ºC.</P><P class="p_notebook">Minipreps and digestions of BBa_I2026.</P><TABLE class="table_notebook"><TBODY><TR class="tr_notebook"><TD class="td_notebook">Device</TD><TD class="td_notebook">Enzyme</TD><TD class="td_notebook">Expected bands</TD></TR><TR class="tr_notebook"><TD class="td_notebook">BBa_I20620</TD><TD class="td_notebook">NotI</TD><TD class="td_notebook">2726,   943</TD></TR><TR class="tr_notebook"><TD class="td_notebook">NcoI</TD><TD class="td_notebook">3296, 373</TD></TR></TBODY></TABLE><P class="p_notebook">There was some kind of trouble with the gel and bands where not clear. We repeat the digestion again other day.</P><H4 class="date_notebook">08/29/2014</H4><P class="p_notebook">Pick colonies of BBa_J23101+BBa_E0240 and BBa_J23115+BBa_E0240 grow them in agitation at 37°C in liquid media supplemented with chloramphenicol.</P><H4 class="date_notebook">08/30/2014</H4><P class="p_notebook">Minipreps of BBa_J23101+BBa_E0240 and BBa_J23115+BBa_E0240 and digestions. Repeat digestions of BBa_I20620.</P><TABLE class="table_notebook"><TBODY><TR class="tr_notebook"><TD class="td_notebook">Device</TD><TD class="td_notebook">Enzyme</TD><TD class="td_notebook">Expected bands</TD></TR><TR class="tr_notebook"><TD class="td_notebook">BBa_J23101+BBa_E0240</TD><TD class="td_notebook">NotI</TD><TD class="td_notebook">2046,   943</TD></TR><TR class="tr_notebook"><TD class="td_notebook">NcoI</TD><TD class="td_notebook">1991,   998</TD></TR><TR class="tr_notebook"><TD class="td_notebook">BBa_J23115+BBa_E0240</TD><TD class="td_notebook">NotI</TD><TD class="td_notebook">2046,   943</TD></TR><TR class="tr_notebook"><TD class="td_notebook">NcoI</TD><TD class="td_notebook">1991,   998</TD></TR></TBODY></TABLE><P class="p_notebook">None of the digestions of BBa_J23101+BBa_E0240. Digestions BBa_J23115+BBa_E0240 (1) and (4) were correct and all of the colonies of BBa_I20620 were correct.</P><H4 class="date_notebook">08/31/2014</H4><P class="p_notebook">Pick 5 more colonies of BBa_J23101+BBa_E0240.</P><H4 class="date_notebook">09/01/2014</H4><P class="p_notebook">Minipreps and digestions of 5 more cultures of BBa_J23101+BBa_E0240.</P><P class="p_notebook">BBa_J23101+BBa_E0240 (4) ligation is correct.</P><P class="p_notebook">We noticed that, for some reason, the stry of BBa_J23115+BBa_E0240 was contaminated, so we picked 6 more colonies.</P><H4 class="date_notebook">09/02/2014</H4><P class="p_notebook">Minipreps of BBa_J23115+BBa_E0240 and digestions.</P><P class="p_notebook">All digestions are correct except BBa_J23115+BBa_E0240 (1).</P><P class="p_notebook">We found out that the stry of BBa_J23101+BBa_E0240 was contaminated as well, so due to the low efficiency of this ligation (1/9) we decided to transform again with the correct miniprep.</P><H4 class="date_notebook">09/03/2014</H4><P class="p_notebook">Pick one colony of BBa_J23101+BBa_E0240.</P><H4 class="date_notebook">09/04/2014</H4><P class="p_notebook">Miniprep of BBa_J23101+BBa_E0240.</P><P class="p_notebook">The digestion was correct. We have scheduled the GFP for next Wednesday.</P><H4 class="date_notebook">09/09/2014</H4><P class="p_notebook">Pick colonies for Measurement Interlab Study. Three technical samples for each device and the negative control (Untransformed E.coli DH5-α)were picked. <I>E. coli</I> DH5-α cells were grown in 3.5 ml Luria-Bertani  broth supplied with the corresponding antibiotic at 37°C with shaking at 250 rpm for 16 hours.</P><H4 class="date_notebook">09/10/2014</H4><P class="p_notebook">Today we measured GFP for the Measurement Interlab Study.</P><P class="p_notebook">Cells were centrifuged at 4500 rpm for 5 minutes and resuspended in ten folds the culture volume with a phosphate buffered saline (58 mM Na2HPO4, 17 mM NaH2PO4, 68 mM NaCl), as performed by Scholz et al., 2000. Na2HPO4 and NaH2PO4 were purchased from Panreac. NaCl was purchased from Fisher Bioreagents.</P><P class="p_notebook">A GloMax-Multi Detection System form Promega fluorometer configured with the Blue optical kit (&amp;Lamda;ex=490 nm, &amp;Lamda;em=510-575 nm) was used to measure fluorescence. For measuring fluorescence 250 μl of each sample were placed in a black 96-well plate. Each sample was measured three times and an average was displayed on the screen.</P><P class="p_notebook">A Biowave CO 8000 from Biochrom spectophotometer was used to measure absorbance at 600 nm. For measuring absorbance 700 μl were placed in a cubet and measured one by one in the spectrophotometer.</P><TABLE class="table_notebook"><TBODY><TR class="tr_notebook"><TD class="td_notebook">Sample</TD><TD class="td_notebook">Fluorescence*</TD><TD class="td_notebook">Optical density</TD><TD class="td_notebook">Fluorescence / Optical density</TD></TR><TR class="tr_notebook"><TD class="td_notebook">Negative control</TD><TD class="td_notebook">(1)</TD><TD class="td_notebook">1.085</TD><TD class="td_notebook">0.38</TD><TD class="td_notebook">2.854</TD></TR><TR class="tr_notebook"><TD class="td_notebook">(2)</TD><TD class="td_notebook">1.036</TD><TD class="td_notebook">0.35</TD><TD class="td_notebook">2.959</TD></TR><TR class="tr_notebook"><TD class="td_notebook">(3)</TD><TD class="td_notebook">1.076</TD><TD class="td_notebook">0.39</TD><TD class="td_notebook">2.759</TD></TR><TR class="tr_notebook"><TD class="td_notebook">BBa_I20260</TD><TD class="td_notebook">(1)</TD><TD class="td_notebook">4.907</TD><TD class="td_notebook">0.36</TD><TD class="td_notebook">13.632</TD></TR><TR class="tr_notebook"><TD class="td_notebook">(2)</TD><TD class="td_notebook">4.754</TD><TD class="td_notebook">0.34</TD><TD class="td_notebook">13.981</TD></TR><TR class="tr_notebook"><TD class="td_notebook">(3)</TD><TD class="td_notebook">3.494</TD><TD class="td_notebook">0.26</TD><TD class="td_notebook">13.439</TD></TR><TR class="tr_notebook"><TD class="td_notebook">BBa_J23101 + BBa_E0240</TD><TD class="td_notebook">(1)</TD><TD class="td_notebook">57.393</TD><TD class="td_notebook">0.43</TD><TD class="td_notebook">133.471</TD></TR><TR class="tr_notebook"><TD class="td_notebook">(2)</TD><TD class="td_notebook">61.622</TD><TD class="td_notebook">0.47</TD><TD class="td_notebook">131.110</TD></TR><TR class="tr_notebook"><TD class="td_notebook">(3)</TD><TD class="td_notebook">63.999</TD><TD class="td_notebook">0.47</TD><TD class="td_notebook">136.167</TD></TR><TR class="tr_notebook"><TD class="td_notebook">BBa_J23115 + BBa_E0240</TD><TD class="td_notebook">(1)</TD><TD class="td_notebook">1.389</TD><TD class="td_notebook">0.37</TD><TD class="td_notebook">3.754</TD></TR><TR class="tr_notebook"><TD class="td_notebook">(2)</TD><TD class="td_notebook">1.353</TD><TD class="td_notebook">0.37</TD><TD class="td_notebook">3.656</TD></TR><TR class="tr_notebook"><TD class="td_notebook">(3)</TD><TD class="td_notebook">1.370</TD><TD class="td_notebook">0.33</TD><TD class="td_notebook">4.151</TD></TR></TBODY></TABLE><P class="p_notebook">*Fluorescence measurements were calculated subtracting the average value of fluorescence of three samples of phosphate buffer (286.1) to the value given for each sample by the fluorometer.</P><TABLE class="table_notebook"><TBODY><TR class="tr_notebook"><TD class="td_notebook">Sample</TD><TD class="td_notebook">Fluorescence</TD><TD class="td_notebook">Optical density</TD><TD class="td_notebook">Fluorescence / Optical density</TD></TR><TR class="tr_notebook"><TD class="td_notebook">Negative control</TD><TD class="td_notebook">1.065±0.026</TD><TD class="td_notebook">0.373±0.021</TD><TD class="td_notebook">2.857±0.100</TD></TR><TR class="tr_notebook"><TD class="td_notebook">BBa_I20260</TD><TD class="td_notebook">4.385±0.775</TD><TD class="td_notebook">0.320±0.053</TD><TD class="td_notebook">13.684±0.275</TD></TR><TR class="tr_notebook"><TD class="td_notebook">BBa_J23101 + BBa_E0240</TD><TD class="td_notebook">61.004±3.346</TD><TD class="td_notebook">0.457±0.023</TD><TD class="td_notebook">133.583±2.530</TD></TR><TR class="tr_notebook"><TD class="td_notebook">Bba_J23115 + BBa_E0240</TD><TD class="td_notebook">1.370±0.018</TD><TD class="td_notebook">0.357±0.023</TD><TD class="td_notebook">3.854±0.262</TD></TR></TBODY></TABLE><H3 class="section_notebook">Switch</H3><H4 class="date_notebook">07/04/2014</H4><P class="p_notebook">Adquisition of <I>S. cerevisiae</I> genomic DNA. (5 μL, stored in the fridge)</P><H4 class="date_notebook">07/28/2014</H4><P class="p_notebook">We had the genome of <I>S. cerevisiae</I>, needed to extract the target genes that are going to be used to build the switch. However we finally used our genome extraction (see Biosynthesis part, date 07/23/2014 for further details).</P><P class="p_notebook">Previously we have designed a cupple of primers to amplify the CUP1 and CUP2 genes present in the yeast. </P><P class="p_notebook">PCR reaction reagents:</P><TABLE class="table_notebook"><TBODY><TR class="tr_notebook"><TD class="td_notebook">Reagent</TD><TD class="td_notebook">CUP1-PCR1</TD><TD class="td_notebook">CUP2-PCR2</TD></TR><TR class="tr_notebook"><TD class="td_notebook">Template</TD><TD class="td_notebook">0.5 μL</TD><TD class="td_notebook">0.5 μL</TD></TR><TR class="tr_notebook"><TD class="td_notebook">Buffer HF (5X)</TD><TD class="td_notebook">10.0 μL</TD><TD class="td_notebook">10.0 μL</TD></TR><TR class="tr_notebook"><TD class="td_notebook">dNTPs</TD><TD class="td_notebook">2.0 μL</TD><TD class="td_notebook">2.0 μL</TD></TR><TR class="tr_notebook"><TD class="td_notebook">Oligo R (JUL06)</TD><TD class="td_notebook">2.5 μL</TD><TD class="td_notebook">2.5 μL</TD></TR><TR class="tr_notebook"><TD class="td_notebook">Oligo F (JUL05)</TD><TD class="td_notebook">2.5 μL</TD><TD class="td_notebook">2.5 μL</TD></TR><TR class="tr_notebook"><TD class="td_notebook">Phusion polymerase</TD><TD class="td_notebook">0.5 μL</TD><TD class="td_notebook">0.5 μL</TD></TR><TR class="tr_notebook"><TD class="td_notebook">H2O</TD><TD class="td_notebook">32.0 μL</TD><TD class="td_notebook">32.0 μL</TD></TR></TBODY></TABLE><P class="p_notebook">Annealing temperature: both 61 °C</P><P class="p_notebook">PCR products were checked using an electrophoresis. Expected bands:</P><UL class="ul_notebook"><LI>CUP1-PCR1: 386 bp</LI><LI>CUP2-PCR2: 348 bp</LI></UL><P class="p_notebook">Both PCR products were correct.</P><H4 class="date_notebook">07/30/2014</H4><P class="p_notebook">We repeated the PCR because we had to purify the bands CUP1-PCR1 and CUP2-PCR2.For this purpose we used the kit &quot;QIAEX II Gel Extraction Kit&quot;.</P><P class="p_notebook">Ligation of both parts of CUP2.</P><UL class="ul_notebook"><LI>1 μL CUP1-PCR1</LI><LI>1 μL CUP1-PCR1</LI><LI>1 μL pUPD</LI><LI>1 μL BsmBI</LI><LI>1 μL T4 ligase</LI><LI>1 μL ligase buffer</LI><LI>4 μL H20</LI></UL><H4 class="date_notebook">07/31/2014</H4><P class="p_notebook">CUP2 was transformed in pUPD and cultured in solid media (37°C).</P><H4 class="date_notebook">08/04/2014</H4><P class="p_notebook">Grow the piece corresponding to Gal4 Activation Domain (GB0095) from the GB collection in solid medium.</P><H4 class="date_notebook">08/05/2014</H4><P class="p_notebook">Pick colonies from CUP2 (3 colonies) and Gal4AD (1 colony).</P><H4 class="date_notebook">08/06/14</H4><P class="p_notebook">Minipreps of yesterday's culture were made:</P><UL class="ul_notebook"><LI>Gal4AD</LI><LI>CUP2</LI></UL><P class="p_notebook">Digestions made in silico in order to check transcriptional units:</P><TABLE class="table_notebook"><TBODY><TR class="tr_notebook"><TD class="td_notebook">Pieces/TU</TD><TD class="td_notebook">Resriction enzymes</TD><TD class="td_notebook">Buffer</TD><TD class="td_notebook">Expected Bands</TD></TR><TR class="tr_notebook"><TD class="td_notebook">CUP2 in pUPD</TD><TD class="td_notebook">Not1</TD><TD class="td_notebook">Orange</TD><TD class="td_notebook">2981, 752</TD></TR><TR class="tr_notebook"><TD class="td_notebook">CUP2 in pUPD</TD><TD class="td_notebook">RsaI</TD><TD class="td_notebook">Tango</TD><TD class="td_notebook">2457, 1276</TD></TR><TR class="tr_notebook"><TD class="td_notebook">Gal4AD in pUPD</TD><TD class="td_notebook">Not1</TD><TD class="td_notebook">Orange</TD><TD class="td_notebook">2981, 330</TD></TR><TR class="tr_notebook"><TD class="td_notebook">Gal4AD</TD><TD class="td_notebook">PuuI</TD><TD class="td_notebook">Red</TD><TD class="td_notebook">2215, 1096</TD></TR></TBODY></TABLE><P class="p_notebook">CUP2 in pUPD is correct. RsaI restriction enzyme does not cut properly, as a result we obtained different bands from those ones expected.</P><P class="p_notebook">Gal4AD piece is correct.</P><H4 class="date_notebook">08/11/2014</H4><P class="p_notebook">Sequencing results of CUP2 piece were finally received and they were correct. </P><P class="p_notebook">As the sequence was correct, we could continue with ligations. </P><P class="p_notebook">Quantification </P><UL class="ul_notebook"><LI>CUP2: 110.3 ng/μL</LI><LI>Gal4: 221.4 ng/μL</LI></UL><P class="p_notebook">Samples were diluted to 75 ng/μL.</P><P class="p_notebook">The following ligations were made:</P><UL class="ul_notebook"><LI>P35S:CUP2:Gal4AD:T35S</LI><UL class="ul_ul_notebook"><LI>1 μL P35S</LI><LI>1 μL CUP2</LI><LI>1 μL Gal4AD</LI><LI>1 μL T35S</LI><LI>1 μL BsaI</LI><LI>1 μL T4 ligase</LI><LI>1 μL Buffer ligase</LI><LI>1 μL 2α2 vector</LI><LI>2 μL H2O</LI></UL></UL><UL class="ul_notebook"><LI>PCPS2:CUP2:Gal4AD:T35S</LI><UL class="ul_ul_notebook"><LI>1 μL PCPS2</LI><LI>1 μL CUP2</LI><LI>1 μL Gal4AD</LI><LI>1 μL T35S</LI><LI>1 μL BsaI</LI><LI>1 μL T4 ligase</LI><LI>1 μL Buffer ligase</LI><LI>1 μL 2α2 vector</LI><LI>2 μL H2O	</LI></UL></UL><H4 class="date_notebook">08/12/2014</H4><P class="p_notebook">E. Coli transformation with the previous ligations and culture in solid medium (LB-agar with Kanamycin and X-Gal + IPTG) overnight.</P><H4 class="date_notebook">08/13/2014</H4><P class="p_notebook">We recultured yesterday's colonies in liquid media with the same antibiotic (Kan) and X-Gal. </P><UL class="ul_notebook"><LI>P35S:CUP2:Gal4AD:T35S in 2α2 (3 colonies)</LI><LI>PCPS2:CUP2:Gal4AD:T35S in 2α2 (3 colonies)</LI></UL><H4 class="date_notebook">08/14/2014</H4><P class="p_notebook">Minipreps of yesterday's culture and streakes were made. </P><P class="p_notebook">Digestions made in sililco to chceck the TU:</P><TABLE class="table_notebook"><TBODY><TR class="tr_notebook"><TD class="td_notebook">Pieces/TU</TD><TD class="td_notebook">Resriction enzymes</TD><TD class="td_notebook">Buffer</TD><TD class="td_notebook">Expected Bands</TD></TR><TR class="tr_notebook"><TD class="td_notebook">PCPS2:CUP2:Gal4AD:T35S</TD><TD class="td_notebook">HindIII</TD><TD class="td_notebook">Red</TD><TD class="td_notebook">6322, 2641</TD></TR><TR class="tr_notebook"><TD class="td_notebook">EcoRV</TD><TD class="td_notebook">Red</TD><TD class="td_notebook">562, 8401</TD></TR><TR class="tr_notebook"><TD class="td_notebook">P35S:CUP2:Gal4AD:T35S</TD><TD class="td_notebook">HindIII</TD><TD class="td_notebook">Red</TD><TD class="td_notebook">6322, 2223</TD></TR><TR class="tr_notebook"><TD class="td_notebook">BclI</TD><TD class="td_notebook">Green</TD><TD class="td_notebook">476, 7137, 932</TD></TR></TBODY></TABLE><P class="p_notebook">The agarose gel shows that P35S:CUP2:Gal4AD:T35S piece is not well build. Nevertheless, PCPS2:CUP2:Gal4AD:T35S piece is OK. </P><H4 class="date_notebook">08/15/2014</H4><P class="p_notebook">P35S:CUP2:Gal4AD:T35S digestions made yesterday were repeated as follows:</P><TABLE class="table_notebook"><TBODY><TR class="tr_notebook"><TD class="td_notebook">Pieces/TU</TD><TD class="td_notebook">Resriction enzymes</TD><TD class="td_notebook">Buffer</TD><TD class="td_notebook">Expected Bands</TD></TR><TR class="tr_notebook"><TD class="td_notebook">P35S:CUP2:Gal4AD:T35S</TD><TD class="td_notebook">HindIII</TD><TD class="td_notebook">Red</TD><TD class="td_notebook">6322, 2223</TD></TR><TR class="tr_notebook"><TD class="td_notebook">NotI</TD><TD class="td_notebook">Green</TD><TD class="td_notebook">5723, 1290, 1532</TD></TR></TBODY></TABLE><P class="p_notebook">After running the electrophoresis, the resulting bands show that there is something more than expected in the plasmid. Furthermore, we check that the extra part has been added in the part region. Ligation step has to be repeated. </P><H4 class="date_notebook">08/17/2014</H4><P class="p_notebook">We repeated the P35S:CUP2:Gal4AD:T35S ligation. </P><P class="p_notebook">Ligation reagents:</P><UL class="ul_notebook"><LI>P35S:CUP2:Gal4AD:T35S:</LI><UL class="ul_ul_notebook"><LI>1 μL PCPS2</LI><LI>1 μL T35S</LI><LI>1 ul Gal4AD</LI><LI>1 μL 2α2</LI><LI>1 μL CUP2</LI><LI>1 μL Buffer ligase 10X</LI><LI>1 μL T4</LI><LI>1 μL BsaI</LI><LI>2 μL H2O</LI></UL></UL><H4 class="date_notebook">08/18/2014</H4><P class="p_notebook">TU piece was transformed in <I>E. coli</I> (P35S:CUP2:Gal4AD:T35S) and cultured in solid media.</P><H4 class="date_notebook">08/19/2014</H4><P class="p_notebook"><I>E. coli</I> colonies containing the TU (P35S:CUP2:Gal4AD:T35S in 2α2) were recultured in liquid media.</P><H4 class="date_notebook">08/20/2014</H4><TABLE class="table_notebook"><TBODY><TR class="tr_notebook"><TD class="td_notebook">Piece</TD><TD class="td_notebook">Enzyme</TD><TD class="td_notebook">Expected bands</TD></TR><TR class="tr_notebook"><TD class="td_notebook">P35S:CUP2:Gal4AD:T35S</TD><TD class="td_notebook">HindIII</TD><TD class="td_notebook">6322, 2223</TD></TR><TR class="tr_notebook"><TD class="td_notebook">NcoI</TD><TD class="td_notebook">8155, 390</TD></TR></TBODY></TABLE><P class="p_notebook">Digestions were correct. </P><H3 class="section_notebook">Translator to BioBricks</H3><H4 class="date_notebook">08/07/2014</H4><P class="p_notebook">Ale's primers labeled A11Dic32 and M11Nov12 found.</P><P class="p_notebook">Run PCR with the following templates and primers:</P><TABLE class="table_notebook"><TBODY><TR class="tr_notebook"><TD class="td_notebook">Template</TD><TD class="td_notebook">Forward</TD><TD class="td_notebook">Reverse</TD><TD class="td_notebook">Expected lenght</TD></TR><TR class="tr_notebook"><TD class="td_notebook">P35s</TD><TD class="td_notebook">iGEMJul11 A11Dic32</TD><TD class="td_notebook">1086 bp</TD></TR><TR class="tr_notebook"><TD class="td_notebook">T35s</TD><TD class="td_notebook">M11Nov12iGEM12Jul</TD><TD class="td_notebook">284 bp</TD></TR></TBODY></TABLE><P class="p_notebook">P35s PCR parameters</P><UL class="ul_notebook"><LI>98°C, 2 min</LI><LI>35 cycles</LI><UL class="ul_ul_notebook"><LI>98°C, 10 s</LI><LI>67°C, 18 s</LI><LI>72°C, 40 s</LI></UL><LI>98°C, 7 min</LI></UL><P class="p_notebook">T35s PCR parameters</P><UL class="ul_notebook"><LI>98°C, 2 min</LI><LI>35 cycles</LI><UL class="ul_ul_notebook"><LI>98°C, 10 s</LI><LI>65°C, 18 s</LI><LI>72°C, 40 s</LI></UL><LI>98°C, 7 min</LI></UL><P class="p_notebook">We didn't obtain PCR product.</P><H4 class="date_notebook">08/08/2014</H4><P class="p_notebook">We repeat yesterday's PCR with 2 degrees less in the annealing step.</P><P class="p_notebook">Now there is a band for P35s but it should not be there.</P><H4 class="date_notebook">08/11/2014</H4><P class="p_notebook">The previous PCR was repeated changing the annealing temperature to 61°C. </P><P class="p_notebook">We still do not get PCR product.</P><H4 class="date_notebook">08/12/2014</H4><P class="p_notebook">We repeated the PCR once more, this time setting the annealing temperatures at (59°C for T35s and 61°C for P35s).</P><H4 class="date_notebook">08/19/2014</H4><P class="p_notebook">We are trying another PCR strategy to obtain the PCR product. </P><UL class="ul_notebook"><LI>PCR1: P35S template (as previously done)</LI><LI>PCR2: P35S:AtrΔ11:T35S template</LI></UL><TABLE class="table_notebook"><TBODY><TR class="tr_notebook"><TD class="td_notebook">PCR</TD><TD class="td_notebook">Primers</TD><TD class="td_notebook">Tm (°C)</TD></TR><TR class="tr_notebook"><TD class="td_notebook">1</TD><TD class="td_notebook">iGEMJul11 and A11Dic32</TD><TD class="td_notebook">62</TD></TR><TR class="tr_notebook"><TD class="td_notebook">2</TD><TD class="td_notebook">M11Nov12 and iGEMJul12</TD><TD class="td_notebook">65</TD></TR></TBODY></TABLE><P class="p_notebook">We check the PCR products and only the T35S product was amplified correctly (the expected band was around 300 bp).</P><H4 class="date_notebook">08/20/2014</H4><P class="p_notebook">As the PCR product was correct, we made a ligation to obtain the T35S piece in pUPD vector as follows:</P><UL class="ul_notebook"><LI>1 μL pUPD</LI><LI>1 μL T35S_BB</LI><LI>1.2 μL Buffer ligase</LI><LI>1 μL BsmBI</LI><LI>1 μL T4 ligase</LI><LI>6.8 μL H20 miliQ</LI></UL><P class="p_notebook">Reaction conditions: 25 cycles x (37°C 2 min, 16°C 5 min).</P><P class="p_notebook">We repeated a PCR to obtain the P35S using the same template as previously and the following conditions (35 cycles):</P><TABLE class="table_notebook"><TBODY><TR class="tr_notebook"><TD class="td_notebook">Step</TD><TD class="td_notebook">Temperature (°C)</TD><TD class="td_notebook">Time </TD></TR><TR class="tr_notebook"><TD class="td_notebook">Initialization</TD><TD class="td_notebook">98</TD><TD class="td_notebook">1:30 min</TD></TR><TR class="tr_notebook"><TD class="td_notebook">Denaturation</TD><TD class="td_notebook">98</TD><TD class="td_notebook">10 s</TD></TR><TR class="tr_notebook"><TD class="td_notebook">Annealing</TD><TD class="td_notebook">57/62/67</TD><TD class="td_notebook">20 s</TD></TR><TR class="tr_notebook"><TD class="td_notebook">Extension</TD><TD class="td_notebook">72</TD><TD class="td_notebook">25 s</TD></TR><TR class="tr_notebook"><TD class="td_notebook">Final elongation</TD><TD class="td_notebook">72</TD><TD class="td_notebook">7 min</TD></TR></TBODY></TABLE><P class="p_notebook">We checked the PCR product running a gel electrophoresis, but the PCR did not work again and the agarose gel did not show any band. </P><P class="p_notebook">T35S in pUPD vector was transformed in <I>E. coli</I> and cultured in agar plates. The protocol followed was the same as it is usually done. </P><H4 class="date_notebook">08/21/2014</H4><P class="p_notebook">We picked <I>E. coli</I> colonies and recultured them in liquid media with the apprpriate antibiotic, Amp (2:1000). </P><H4 class="date_notebook">08/22/2014</H4><P class="p_notebook">We made minipreps of yesterday's culture and we made digestions to check them. </P><P class="p_notebook">In silico digestions:</P><TABLE class="table_notebook"><TBODY><TR class="tr_notebook"><TD class="td_notebook">Piece</TD><TD class="td_notebook">Restriction enzyme</TD><TD class="td_notebook">Expected bands</TD></TR><TR class="tr_notebook"><TD class="td_notebook">T35S in pUPD</TD><TD class="td_notebook">EcoRI</TD><TD class="td_notebook">2997, 309</TD></TR><TR class="tr_notebook"><TD class="td_notebook">PvuI</TD><TD class="td_notebook">2210, 1096</TD></TR></TBODY></TABLE><P class="p_notebook">Digestions were correct. </P><P class="p_notebook">We run a PCR with the TUs as templates (adjusted to 5 ng/μL) and using Jul11 and Jul12 as primers.</P><UL class="ul_notebook"><LI>EaDAcT (2α2)</LI><LI>HarFAR (2α2)</LI><LI>AtrΔ11 (2α1)</LI></UL><P class="p_notebook">Conditions for 35 cycles:</P><TABLE class="table_notebook"><TBODY><TR class="tr_notebook"><TD class="td_notebook">Step</TD><TD class="td_notebook">Temperature (°C)</TD><TD class="td_notebook">Time </TD></TR><TR class="tr_notebook"><TD class="td_notebook">Initialization</TD><TD class="td_notebook">98</TD><TD class="td_notebook">2 min</TD></TR><TR class="tr_notebook"><TD class="td_notebook">Denaturation</TD><TD class="td_notebook">98</TD><TD class="td_notebook">15 s</TD></TR><TR class="tr_notebook"><TD class="td_notebook">Annealing</TD><TD class="td_notebook">65</TD><TD class="td_notebook">20 s</TD></TR><TR class="tr_notebook"><TD class="td_notebook">Extension</TD><TD class="td_notebook">72</TD><TD class="td_notebook">45 s</TD></TR><TR class="tr_notebook"><TD class="td_notebook">Final elongation</TD><TD class="td_notebook">72</TD><TD class="td_notebook">7 min</TD></TR></TBODY></TABLE><P class="p_notebook">We made another PCR to obtain P35S as a product. This time, we used Q5 High Fidelity polimerase. </P><P class="p_notebook">Conditions for 35 cycles:</P><TABLE class="table_notebook"><TBODY><TR class="tr_notebook"><TD class="td_notebook">Step</TD><TD class="td_notebook">Temperature (°C)</TD><TD class="td_notebook">Time </TD></TR><TR class="tr_notebook"><TD class="td_notebook">Initialization</TD><TD class="td_notebook">98</TD><TD class="td_notebook">1:30 min</TD></TR><TR class="tr_notebook"><TD class="td_notebook">Denaturation</TD><TD class="td_notebook">98</TD><TD class="td_notebook">10 s</TD></TR><TR class="tr_notebook"><TD class="td_notebook">Annealing</TD><TD class="td_notebook">55</TD><TD class="td_notebook">20 s</TD></TR><TR class="tr_notebook"><TD class="td_notebook">Extension</TD><TD class="td_notebook">72</TD><TD class="td_notebook">25 s</TD></TR><TR class="tr_notebook"><TD class="td_notebook">Final elongation</TD><TD class="td_notebook">72</TD><TD class="td_notebook">7 min</TD></TR></TBODY></TABLE><P class="p_notebook">The gel shows that the template is not there.</P><H4 class="date_notebook">08/25/2014</H4><P class="p_notebook">We repeated the PCR made the previous day using TUs as a template and primers Jul11 and Jul12, but this time we changed the extension time to 1:30 min.</P><P class="p_notebook">The gel showed that the PCR products were correct.</P><P class="p_notebook">We repeated the FAO1 PCR</P><DIV class="right-col"><DIV class="pinned note-container"><DIV class="note"><H3>Pin it!</H3><P>Great for pinned navigations — or just about anything you want to stick.</P><P>Developed by <A href="http://www.webpop.com">Webpop</A>.</P></DIV></DIV></DIV></SECTION></DIV><FOOTER><DIV id="yhteystiedot"><P><A href="https://static.igem.org/mediawiki/2014/7/73/Aalto_Helsinki_Wiki_Sitemap.png" target="_blank">Sitemap</A> | <A href="http://twitter.com/AaltoHelsinki">Twitter</A> | <A href="http://www.facebook.com/AaltoHelsinki">Facebook</A> | <A href="mailto:oskari.vinko@gmail.com" target="_top">Email</A>
      This wiki is designed and constructed by Valencia_UPV.
	</P></DIV></FOOTER><DIV class="printfooter">
Retrieved from &quot;<A href="http://2014.igem.org/Team:Valencia_UPV/otraprueba">http://2014.igem.org/Team:Valencia_UPV/otraprueba</A>&quot;</DIV><DIV id="footer-box" class="noprint"><DIV id="footer"><UL id="f-list"><LI id="t-recentchanges"><A href="/Special:RecentChanges" title="Recent changes">Recent changes</A></LI><LI id="t-whatlinkshere"><A href="/Special:WhatLinksHere/Team:Valencia_UPV/otraprueba" title="List of all wiki pages that link here [j]" accesskey="j">What links here</A></LI><LI id="t-recentchangeslinked"><A href="/Special:RecentChangesLinked/Team:Valencia_UPV/otraprueba" title="Recent changes in pages linked from this page [k]" accesskey="k">Related changes</A></LI><LI id="t-specialpages"><A href="/Special:SpecialPages" title="List of all special pages [q]" accesskey="q">Special pages</A></LI><LI><A href="/Special:Preferences">My preferences</A></LI></UL></DIV><DIV id="footer"><UL id="f-list"><LI id="t-print"><A href="/wiki/index.php?title=Team:Valencia_UPV/otraprueba&amp;printable=yes" title="Printable version of this page [p]" accesskey="p">Printable version</A></LI><LI id="t-permalink"><A href="/wiki/index.php?title=Team:Valencia_UPV/otraprueba&amp;oldid=203862" title="Permanent link to this revision of the page">Permanent link</A></LI><LI id="privacy"><A href="/2014.igem.org:Privacy_policy" title="2014.igem.org:Privacy policy">Privacy policy</A></LI><LI id="disclaimer"><A href="/2014.igem.org:General_disclaimer" title="2014.igem.org:General disclaimer">Disclaimers</A></LI></UL></DIV></DIV></BODY></HTML>