"
Page
Discussion
View source
History
teams
Log in
 
Team:UNAM Genomics Mexico/Notebook/Protocols
From 2012.igem.org
UNAM-Genomics_Mexico
Home
Project Our project
Overview Description Deeper Description Parts
Results Wet lab
AND OR Bacillus Subtilis Nanotubes!! Notebook
Human Practices
Who is Mrs. Cohnnie, PhD? BioSintetizARTE (ES) Biosintetizarte (EN) Outreach videos Talk and surveys Symposium Future Work Mexico Meeting Our Partners, Sponsors and Collaborations.
Modelling Model!!
Cross Talk Sweet AND Heavy Metal AND OR Parameters
Team& Requirements Everything  at one click
Team Official Team Unofficial Attributions Safety Notebook Parts
Protocols
E. Coli Protocols
Contents
1Protocols
2E. coli Protocols
2.1PCR Protocol  (50µl)
2.2Hot start PCR protocol
2.3Ligation
2.4Dephosphorylation protocol
2.5Transformation
2.6Liquid culture
2.7Digestion protocol (20 µl)
2.8Gel extraction protocol
2.9Lysis protocol
2.10Gel electrophoresis
2.11Competent cells protocol
2.12BUFFERS
2.13PCR purification
2.14GLYCEROL PROTOCOL
2.15PLASMID  EXTRACTION 2
3Bacillus subtilis Protocols
3.1Escherichia coli MC1061 competent cells protocol
3.2Escherichia coli MC1061 heat shock transformation protocol
3.3Two-step Bacillus subtilis Transformation Procedure
3.4SEM Analysis (Scanning Electron Microscope)[1]
Bacillus Protocols
E. coli Protocols
PCR Protocol  (50µl)
Buffer Pfu 5 µl
Upper primers 2.5  µl
Lower primers 2.5 µl
MgCL2 1 µl
DNA (depends on concentration, usually 0.5µl)
Enzime Taq Polymerase 1 µl
DNTP’s 8 µl
Water 29.5 µl
Cycles :)
95ºC 4 min
95ºC 1 min
55ºC :30 secs
70ºC 1 min
goto 2:4  30 times
72ºC 5 min
Hot start PCR protocol
Preheated 105ºC
Heated Lid ON
Pause OFF
In denat 94ºC 5 min.
Hot start OFF
30 cycles
94 ºC 30 secs.
55ºC 30 secs.
70ºC 45 secs.
72ºC 5 min
hold 10ºC
Ligation
These are done in 1 mL eppendorf tubes and normally have a T.V. of 20 µl.
•Add A µl of the insert (plasmid digested with corresponding enzymes).
•Add B µl of vector (plasmid digested with corresponding enzymes and complementary so that the ligation can be done properly) and properly DEPHOSPOHATED.
•Add 2 µl T4 DNA ligase buffer.
•Add 20-A-B-3-1 µl  H2O miliQ.
•Add 1 µl T4 DNA ligase enzyme.
Dephosphorylation protocol
-It is done to plasmids digested with the corresponding enzymes that will serve as a “vector” in a ligation.
1. Add 1 µl  alkaline phosphatase.
2. Add 2 µl buffer.
3.. Add 20 µl of Digestion.
4. Leave 1 hour at 37ºC.
5. Inactivate at 70ºC for 10 minutes.
Transformation
•Pipette the plasmid to a 1.3 mL eppendorf with competent cells and mix with the pipette.
•Incubate tubes on ice for 42 minutes.
•Heatshock at 42ºC for 2 minutes.
•Incubate for 5 minutes on ice.
•Add 1 mL LB broth to each tube.
•Mix by inversion 2-3 times.
•Incubate at 37ºC for an hour with shaking.
•During this hour prepare LB agar plates (30 mL) containing the appropriate antibiotics.
•After the hour Plate 100 µl.
•Centrifuge at 10000 rpm for 2 minutes.
•Drain 700-800 µl of supernatant, mix pellet with the rest and plate. Add to the plate.
•Let grow overnight at 37ºC.
Liquid culture
•Once the bacteria has grown, add 4 mL LB broth to a glass tube.
•Add the appropriate antibiotic.
•With a wooden toothpick take one colony per tube. Place the toothpick in the tube and vortex.
•Incubate the tube in the shaker 37ºC for at least 6 hours.
Digestion protocol (20 µl)
H2O 9.5 µl
Enzyme 0.5 µl
Buffer 10x  2 µl
DNA (it depends on the concentration but we usually put 8 µl)
37ºC at least 4 hours
For double digestions add 0.5 µl of the other enzyme and 0.5 µl less of H2O.
Gel extraction protocol
(Fermentas Gene Jet Gel extraction kit)
The extraction and purification of a DNA band from a gel electrophoresis is done when we digest and want to recover a specific fragment.
1. Cut the band of the fragment you wish to purify.
2. 400 µl of Binding Buffer, 10 min. at 65ºC with shaker so that the agarose melts.
3. 200 µl deisopropanol after 5 min in the shaker.
4. Pass through column and centrifuge. Throw out supernatant.
5. 700 µl wash buffer, centrifuge, throw it out, centrifuge.
6. Pass column to clean tube, add 40 µl de elution buffer.
Lysis protocol
1. Cultivate strain of interest in 5 µl LB with antibiotics.
2. Centrifuge culture at 14000rpm 2 minutes. Remove supernatant and repeat.
3. Add 200 µl solution I, incubate 5 minutes at room temperature. Mix well to dissolve pellet.
4. Add 400 µl  of solution II and incubate 5 minutes at room temperature. Mix by inversion 3-6 times.
5. Add 300 µl  of solution III. .
6. Mix by inversion and leave in ice 10 minutes. .
7. Centrifuge 10 minutes at 14000 rpm. .
8. Place supernatant in clean tubes and add 500µl 100% ethanol. Centrifuge 15 minutes at 14000 rpm and throw away supernatant. .
9. Add 1 ml 70% ethanol, centrifuge 5 minutes at 1200 rpm and throw out supernatant. Do this twice. .
10. Dry in SAVANT 5 minutes. .
11. Resuspend in 50 µl  water MQ and add 5 µl  RNAse. .
Gel electrophoresis
-Sample preparation
•Add 10 µl of DNA in each well.
•Add 3 µl  loading buffer in the same well
•There are special markers that contain a mix of molecules of known size to compare and determine the size of our sample.
•Add 1 µl  marker (ladder).
-Gel preparation and loading
•Add 1% agarose to an electrophoresis tank and let it become a gel.
•Introduce the tank in the chamber.
•In each well, with the pipette mix the content and load it in its corresponding well-
•Run with the appropriate Voltage, miliampers (mA) and time, corresponding to the size.
-Visualization
•Once completed, remove gel from chamber and place in special plastic container.
•Dye with ethidium bromide (5 minutes approximately).
•Rinse 10 minutes approximately with water.
Competent cells protocol
•Grow cells in LB all night.
•Dilute 100x in LB 50 µl  and let grow for 2 hours till an OD of 0.45-0.55 (40 ml X 4 tubes).
•Leave 10 minutes on ice, centrifuge 10 minutes at 6000 rpm at 4ºC.
•Resuspend in 16ml of ice cold TfbI.
•Leave 5 minutes on ice, centrifuge as before.
•Resuspend in 1.6 ml of cold TfbII.
•Leave 10 minutes on ice.
•Make 200 microliters aliquots and store at -70ºC.
BUFFERS
TfbI: Mw-> 100ml
30mM Kac    98.146-> 0.294
100mM RbCl 120.9->1.2092
10mM CaCl2 147.020->0.147
50mM MnCL2  197.9-> 0.9895
15% glycerol    15 ml (30ml glycerol at 50%)
-Adjust pH at 5.8 with acetic acid 0.2M (2.4ml p/300ml)
-Sterilize in autoclave
TfbI:100ml 10mM MOPS or PIPES   209.27->0.20927
75 mM CaCl2 110.99-> 0.8324->0.410
10mM RbCl 120.92->0.1292->.60
15% glycerol 15ml(gl 50%)
-Adjust pH at 6.5 with 1 M KOH
-Sterilize by filtration.
PCR purification
•Add 500 μl  solution I “green lid” from kit of PCR purification to eppendorf tube.
•Mix by inversion 5-6 times.
•Drain content to silicium column.
•Repeat band extraction protocol without heating to 65ºC since it is not necessary.
GLYCEROL PROTOCOL
•Add 500 μl  glycerol at 70% to a 1.5 mL eppendorf tube.
•Add 1 mL LB broth.
•Take respective colonie with a wooden toothpick.
•Place in eppendorf tube and mix with toothpick.
•Incubate at -80ºC and properly label tube.
PLASMID  EXTRACTION 2
Once grown in liquid culture:
Obtain the pellet:
•Add 1 mL liquid culture to a 1.5mL eppendorf.
•Centrifuge at 13000 rpm for 2 minutes.
•Drain supernatant.
•Repeat 4 times.
•If you will continue with the extraction continue, if not store pellets (1.5 mL eppendorf tubes) at -20ºC.
•Add 250 mL solution one “white lid” RNase.
•Vortex.
•Add 250 mL solution 2 “red lid” SDS NaOH.
•Mix gently by inversion:
-50 times slow inversion.
-Let rest 1 minute.
-Repeat 5 minutes.
•Add 350 mL solution 3 “green lid” KAc.
•Mix by inversion 20 secs.
•Incubate on ice 10 minutes.
•Centrifuge 13000 rpm for 15 minutes.
•During these 15 minutes place a silicium column inside a recollection tube for each eppendorf tube.
•Drain supernatant in the silicium column and close lid.
•Centrifuge at 13000 rpm for 1 minute.
•Drain supernatant from recollection tube.
•Add 700 mL solution 5 “blue lid” absolute ethanol in the center of the column and close.
•Centrifuge at 13000 rpm for a minute.
•Throw out column.
•Close tube and store -20ºC.
Bacillus subtilis Protocols
Escherichia coli MC1061 competent cells protocol
1.Plate the cells on LB, incubate at 37°C overnight.
2.Prepare the solutions.
3.Choose a colony and inoculate in 10 mL of LB media, incubate with vigorous shaking for about 16 hours.
4.Subculture in 1L  of LB media and incubate at 37°C until it reaches late logarithmic phase reading 0.4 OD600 nm.
5.Incubate the culture on ice immediately after the desire concentration is reached. 6.Note: Incubate on ice all the solutions too.
7.Spin-dry 1000 g & 4°C for 30 minutes. Decant the supernatant.
8.Resuspend gently on 250 mL of 0.1M MgCl2 solution.
9.Spin-dry 1000 g & 4°C for 25 minutes. Decant the supernatant.
10.Resuspend gently on 250 mL of 0.075M CaCl2 solution. Incubate on ice for 20 minutes.
11.Spin-dry 1000 g & 4°C for 25 minutes. Decant the supernatant.
12.Resuspend gently on 50 mL of 0.075M CaCl2+15% glycerol solution (vol/vol). Incubate on ice for 20 minutes.
13.Alicuote (100 microliters) in sterile and cold eppendorff tubes.
14.Store at -80°C.
Escherichia coli MC1061 heat shock transformation protocol
1.Add the DNA solution to the competent cells and let them 5-20 minutes on ice
. 2.Put the cells on a 42°C termoblock for 5 minutes.
3.Immediately get them back to ice for one minute.
4.Add 1 mL of LB, mix gently and incubate for one hour at 37°C & 250 rpm .
5.Plate 100 microliters onto selective media.
6.Pellet the cells 1 min at 1200 rpm and resuspend in 100 microliters.
7.Plate cells onto selective media.
Two-step Bacillus subtilis Transformation Procedure
Preparation of Bacillus subtilis competent cell
1.Streak out the strain to be made competent on a LB agar plate as a large patch and incubate overnight at 28°C
2.The following morning wash the cell growth of the plate with 3 ml of Spc and add 3 drops of the solution to inoculate fresh, pre-warmed, SpC medium (30 ml) in a klett flask and measure to give an OD600 reading of about 0.5 using a green filter.
3.Incubate the culture at 37°C with vigorous aeration and take periodic OD readings (OD600) to assess cell growth. (measure every 15 minutes)
4.When the rate of cell growth is seen to depart from exponential (i.e. no significant change in cell density ~5% over 15 min), dilute cell culture 1:10; inoculate 300 ml of pre-warmed, SpII medium with 30 ml of stationary-phase culture and  continue incubation at 37°C with slower aeration.
5.After 90 min incubation, pellet the cells by centrifugation (8,000 g, 10 min) at 4°C.
6.Carefully decant the supernatant into a sterile container and save.
7.Gently resuspend the cell pellet in 30 ml of the saved supernatant and add 3 ml of sterile glycerol to a 10% concentration; mix gently.
8.Aliquot the competent cell (0.35 ml) in sterile eppendorf tubes, freeze rapidly in liquid nitrogen bath and store -80°C.
Transformation
1.Add one volume of SpII + EGTA to the competent cells; mix gently
2.Add the DNA solution (500 & 1000 ng) and incubate at 37°C for 1 hour at 200 rpm.
3.Plate 100 microliters immediately onto selective media.
4.Pellet the cells 1 min at 1200 rpm and resuspend in 100 microliters.
5.Plate onto selective media.
NOTE: if you were not able to select you Bacillus, maybe it is because you kill them before they activate their resistance gene. So, made the 5th step different, plate but the 1/40 of the antibiotic, to active the gene and incubate for 1 hour. Then put soft-agar until the petri dish is cover and put the original amount of antibiotic. Leave them overnight (16-24 hours) to grow.
Media for two-step trasformation procedure
T base  per liter:
(NH4)2SO4    2 g
K2HPO4     13.96 g
KH2PO4     6 g
trisodium citrate•2H2O   1g
-> Autoclave
SpC   Made fresh on the day of use from the following sterile solution:
T base     20 ml
50% (w/v) glucose   0.2 ml
1.2% (w/v) MgSO4•3H2O    0.3 ml
10% (w/v) Bacto yeast extract   0.4 ml  1% (w/v) casamino acids   0.5 ml
SpII  Made fresh on the day of use from the following sterile solutions:
T base     200 ml
50% (w/v) glucose    2 ml
1.2% (w/v) MgSO4•3H2O    14 ml
10% (w/v) Bacto yeast extract   2 ml
1% (w/v) casamino acids   2 ml
0.1 M CaCl2    1 ml
SpII + EGTA  SpII (200 ml) with 4 ml EGTA (0.1 M, pH 8.0) but without CaCl2. SpII + EGTA can be frozen at -20  in small  ℃ aliquots, although repeated freeze-thawing should be avoided.
Note: filter EGTA.
SEM Analysis (Scanning Electron Microscope)[1]
Exponentially growing cells were plated on LB. Cells were incubated for 3hr at 37°C and then EM grids (FCF200-Cu) were placed on top of the growing cells. Plates were incubated for additional 3 hr and EM grids in the plate was fixed with 2% glutaraldehyde in sodium cacodylate buffer (0.2 M, [pH 7.2]) for 12 hr at 25°C. Cells attached to the grids were washed with 0.1 M sodium cacodylate buffer Na (CH3)2 AsO2 , 3H2O) (pH 7.2), 3X for 5 min each wash.
Next, cells were postfixed by incubation with 1% osmium tetroxide for 30 min at 25°C and then dehydrated by exposure to a graded series of ethanol washes (30%, 40%, 50%, 60%, 70%, 80%, 90%,; 5 min each and two times with: 100%; 10 min each).
Finally, the grid-attached cells were washed with a graded series of liquid CO2. The liquefied gas must be miscible with the solvent in the cells.  After sufficient equilibration time, the liquefied gas replaces the alcohol. The temperature of the pressure chamber is then raised above the critical point of the gas and the liquefied gas returns to the gaseous state without a change in volume or density and without surface tension forces. After drying, the specimens are mounted on a stub. Specimens were coated with gold-palladium with a Sputter Coater and cells observed with a SEM.
[1]Dunlap, M. Introduction to the Scanning Electron Microscope, 1997, FACILITY FOR ADVANCED INSTRUMENTATION
Retrieved from " http://2012.igem.org/Team:UNAM_Genomics_Mexico/Notebook/Protocols "
Recent changes
What links here
Related changes
Special pages
My preferences
Printable version
Permanent link
Privacy policy
Disclaimers
