<HTML xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en" dir="ltr">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki  ltr ns-0 ns-subject page-Team_UNAM_Genomics_Mexico_Modeling_Sweet_AND"><DIV id="globalWrapper"><DIV id="top-section"><DIV id="p-logo"><A href="/Main_Page" title="Main Page">&quot;
	    </A></DIV><DIV id="menubar" class="left-menu noprint"><UL><LI class="selected"><A href="/Team:UNAM_Genomics_Mexico/Modeling/Sweet_AND">Page               </A></LI><LI class="new"><A href="/wiki/index.php?title=Talk:Team:UNAM_Genomics_Mexico/Modeling/Sweet_AND&amp;action=edit&amp;redlink=1">Discussion               </A></LI><LI><A href="/wiki/index.php?title=Team:UNAM_Genomics_Mexico/Modeling/Sweet_AND&amp;action=edit">View source               </A></LI><LI><A href="/wiki/index.php?title=Team:UNAM_Genomics_Mexico/Modeling/Sweet_AND&amp;action=history">History               </A></LI><LI style="color:#808080;cursor:default">teams</LI></UL></DIV><DIV class="right-menu noprint" id="menubar"><UL><LI id="pt-login"><A href="/wiki/index.php?title=Special:UserLogin&amp;returnto=Team:UNAM_Genomics_Mexico/Modeling/Sweet_AND" title="You are encouraged to log in; however, it is not mandatory [o]" accesskey="o">Log in</A></LI></UL></DIV><DIV id="search-controls" class="noprint"><FORM action="/Special:Search" id="searchform"> </FORM></DIV></DIV><DIV id="content"><H1 class="firstHeading">Team:UNAM Genomics Mexico/Modeling/Sweet AND</H1><DIV id="bodyContent"><H3 id="siteSub" class="noprint">From 2012.igem.org</H3><P><TITLE>UNAM-Genomics_Mexico</TITLE></P><DIV id="container"><DIV class="header"><TABLE bgcolor="#000000" border="0px" height="95" width="10" id="tableabi"><TBODY><TR><TD><UL id="menu_unamgenomics" class="menu_unamgenomics"><LI><A href="https://2012.igem.org/Team:UNAM_Genomics_Mexico"><SPAN class="sdt_wrap"><SPAN class="menu_links">Home</SPAN></SPAN></A></LI><LI><A href="https://2012.igem.org/Team:UNAM_Genomics_Mexico/Project"><SPAN class="sdt_wrap"><SPAN class="menu_links">Project</SPAN><SPAN class="menu_descripcion">Our project</SPAN></SPAN></A><DIV class="menu_caja"><A href="https://2012.igem.org/Team:UNAM_Genomics_Mexico/Project/Overview">Overview</A><A href="https://2012.igem.org/Team:UNAM_Genomics_Mexico/Project/Description">Description</A><A href="https://2012.igem.org/Team:UNAM_Genomics_Mexico/Project/DeeperDescription">Deeper Description</A><A href="https://2012.igem.org/Team:UNAM_Genomics_Mexico/Parts">Parts</A></DIV></LI><LI><A href="https://2012.igem.org/Team:UNAM_Genomics_Mexico/Results"><SPAN class="sdt_wrap"><SPAN class="menu_links">Results</SPAN><SPAN class="menu_descripcion">Wet lab</SPAN></SPAN></A><DIV class="menu_caja"><A href="https://2012.igem.org/Team:UNAM_Genomics_Mexico/Results/AND
">AND</A><A href="https://2012.igem.org/Team:UNAM_Genomics_Mexico/Results/OR">OR</A><A href="https://2012.igem.org/Team:UNAM_Genomics_Mexico/Results/Bacillus">Bacillus Subtilis</A><A href="https://2012.igem.org/Team:UNAM_Genomics_Mexico/Results/Nanotubes">Nanotubes!!</A><A href="https://2012.igem.org/Team:UNAM_Genomics_Mexico/Notebook">Notebook</A></DIV></LI><LI><A href="https://2012.igem.org/Team:UNAM_Genomics_Mexico/HumanPractices"><SPAN class="sdt_wrap"><SPAN class="menu_links">Human Practices</SPAN></SPAN></A><DIV class="menu_caja"><A href="https://2012.igem.org/Team:UNAM_Genomics_Mexico/HumanPractices/Who_is_Mrs._Cohnnie_PhD?">Who is Mrs. Cohnnie, PhD?</A><A href="https://2012.igem.org/Team:UNAM_Genomics_Mexico/BioSintetizarte">BioSintetizARTE (ES)</A><A href="https://2012.igem.org/Team:UNAM_Genomics_Mexico/HumanPractices/BiosintetizarteEN">Biosintetizarte (EN)</A><A href="https://2012.igem.org/Team:UNAM_Genomics_Mexico/HumanPractices/Outreach videos">Outreach videos</A><A href="https://2012.igem.org/Team:UNAM_Genomics_Mexico/HumanPractices/Talk_and_surveys
">Talk and surveys</A><A href="https://2012.igem.org/Team:UNAM_Genomics_Mexico/HumanPractices/Symposium_on_Science_Communication
">Symposium</A><A href="https://2012.igem.org/Team:UNAM_Genomics_Mexico/HumanPractices/Future_Work
">Future Work</A><A href="https://2012.igem.org/Team:UNAM_Genomics_Mexico/HumanPractices/MexicoMeeting
">Mexico Meeting</A><A href="https://2012.igem.org/Team:UNAM_Genomics_Mexico/HumanPractices/Partners">Our Partners, Sponsors and Collaborations.</A></DIV></LI><LI><A href="https://2012.igem.org/Team:UNAM_Genomics_Mexico/Modeling"><SPAN class="sdt_wrap"><SPAN class="menu_links">Modelling</SPAN><SPAN class="menu_descripcion">Model!!</SPAN></SPAN></A><DIV class="menu_caja"><A href="https://2012.igem.org/Team:UNAM_Genomics_Mexico/Modeling/Cross_Talk">Cross Talk</A><A href="https://2012.igem.org/Team:UNAM_Genomics_Mexico/Modeling/Sweet_AND">Sweet AND</A><A href="https://2012.igem.org/Team:UNAM_Genomics_Mexico/Modeling/Heavy_Metal_AND">Heavy Metal AND</A><A href="https://2012.igem.org/Team:UNAM_Genomics_Mexico/Modeling/OR">OR</A><A href="https://2012.igem.org/Team:UNAM_Genomics_Mexico/Modeling/Parameters">Parameters</A></DIV></LI><LI><A href="#"><SPAN class="sdt_wrap"><SPAN class="menu_links">Team&amp; Requirements</SPAN><SPAN class="menu_descripcion">Everything  at one click</SPAN></SPAN></A><DIV class="menu_caja"><A href="https://igem.org/Team.cgi?year=2012">Team Official</A><A href="https://2012.igem.org/Team:UNAM_Genomics_Mexico/Team">Team Unofficial</A><A href="https://2012.igem.org/Team:UNAM_Genomics_Mexico/Attributions">Attributions</A><A href="https://2012.igem.org/Team:UNAM_Genomics_Mexico/Safety">Safety</A><A href="https://2012.igem.org/Team:UNAM_Genomics_Mexico/Notebook">Notebook </A><A href="https://2012.igem.org/Team:UNAM_Genomics_Mexico/Parts">Parts</A></DIV></LI></UL></TD></TR></TBODY></TABLE></DIV><DIV id="textContent"><CENTER><H1><SPAN class="mw-headline" id="AND.27s_Results"><B>AND's Results</B></SPAN></H1></CENTER><H1><SPAN class="mw-headline" id="Sweet_AND"><B>Sweet AND</B></SPAN></H1><P>Xylose and arabinose are sugars used by bacteria as carbon sources. XylR and AraC are
transcriptional regulators modulated by xylose and arabinose. For xylose sensing, our hybrid promoter
has as backbone made of the pBAD promoter plus a binding site for XylR. XylR represses 
transcription until xylose is present in the cell, a process similar to the metal sensor of ArsR and CzrA. As for arabinose, AraC represses transcription through DNA looping. When arabinose enters the cell, it dissolves the loop and transcription can take place. We conclude that arabinose and xylose are essential for the expression of genes under the pBAD/xylR promoter, thus acting as an AND Boolean operation. </P><P><B>Repressors</B></P><P><B>Inputs</B></P><P>The behavior of both ANDs predicted by our model is consistent with what we expected. From the perspective of the repressors, a higher level of PoPs (Polymerase Per Second, which is a measurement of the level of transcription) is reached when both repressor concentrations are 0. Reciprocally, PoPs start to fall when the concentration of repressors increases, reaching the highest level of repression (lower PoPs) at high concentration of both repressors. This behavior reminds us of the way a NOR behaves, because at low levels of repressor PoPs is high, but for repression, high concentration of one repressor is enough. However, we can't control (directly) the concentration of repressors, so the right way to look at the system is from the perspective of the Input: the metals or the sugars. 
If we looked PoPs as a function of the Input, it behaves as an AND logic gate, as can be seen in the continuous TRUTH TABLE. At lower concentrations of Inputs, the PoPs level is in a very repressed stage. If you keep increasing only one of the Inputs on only one of the axes (yes, axes is the plural of axis), there won’t be high activation until the addition of high concentrations on the other axis. PoPs will increase as the Input concentration increases in a direct proportion fashion. </P><H2><SPAN class="mw-headline" id="KAPPA"><B>KAPPA</B></SPAN></H2><P>Sometimes people wonder why we used Kappa, a stochastic approach, to model our system, which was also described by deterministic differential equations. First of all, Kappa helped us come up with the concentration ranges entered in the differential equations (here, an exclamation of awe is usually heard, and with good reason). How did this come to happen? Well, the simulation in kappa is driven by the binding events and is dependent on the species concentrations, so using general kinetic rates we were able to see the dynamic behavior given by the architecture of the network. This allowed us to approximate the final species concentration and look for parameters missing in our ODE system, which was something useful, since we were as lost as a three-legged dog in a rodeo dance. 
So, first we ran the <A href="https://static.igem.org/mediawiki/2012/f/fd/And_sugar.txt" class="external text" rel="nofollow">And Sugar kappa script</A> shifting the sugar initial concentrations in the single input burst and getting the average of 100 iterations to eliminate the noise in the simulation. After running some simulation time, we were happy to see that our &quot;Sweet AND&quot; behaved as expected, that is, as an AND. Besides, we were able to see the maximal concentration of the protein that would be downstream of our AND, and the time it took to reach it. As you can see, our simulation grid is not described in its entirety. This is because kappa can be computationally very expensive for non-atomic rules, so we chose to feed our ODE system with some parameters and scan the system in a broader manner instead of dying of old age waiting for Kappa (if only we had a supercluster to run things in...). Nevertheless, some behaviors can only be noticed by looking at the simulation grid. For example, the activation of P4 depending on the xylose and arabinose concentrations. As it becomes apparent through a meticulous and insightful analysis only doable by a really smart, handsome, and admirable people, the activation of P4 depends on the concentrations of both, xylose and arabinose. When only one of the two sugars has a high concentration, the activation is not as elevated as when the two have high concentrations. </P><P><TABLE border="1" cellpadding="2px" width="100%" height="600px"><TBODY><TR><TD rowspan="7">Xylose</TD><TD>500,000</TD></TR><TR><TD>350,000</TD></TR><TR><TD>100,000</TD></TR><TR><TD>75,000</TD></TR><TR><TD>50,000</TD></TR><TR><TD>25,000</TD></TR><TR><TD>0.0</TD></TR><TR><TD>0.0</TD><TD>25,000</TD><TD>50,000</TD><TD>75,000</TD><TD>100,000</TD><TD>350,000</TD><TD height="14px">500,000</TD></TR><TR><TD colspan="7" height="14px">Arabinose</TD></TR></TBODY></TABLE></P><P>
We can focus only in the dynamics of the downstream protein, instead of the dynamics of the whole system. This plot describes the time to reach steady state and maximal concentration of the AND downstream protein. It also describes what the maximal concentration of the AND protein is. All of this is based on a single input burst concentration.
</P><H2><SPAN class="mw-headline" id="Kind_of_input"><B>Kind of input</B></SPAN></H2><P>
The Endogenous system and Downstream protein dynamics is dependent on the kind of input. The inputs used by each AND have a unique effect in the response of the cell, because carbohydrates are degraded by the cell.  When we add sugars to the sweet AND system, if they are added just like a discrete and singular dose, the cell will react to the alteration through the activation of the degradation machinery, and when the sugar pool gets empty, the cell achieves a similar state to the one before the sugar was introduced, that is, a return to pre-induction steady state. When the sugar addition is constant, the cell has a complete phenotypic change, reaching a new steady state.</P><H2><SPAN class="mw-headline" id="Constitutive_VS_NO-Constitutive_XylR_expression"><B>Constitutive VS NO-Constitutive XylR expression</B></SPAN></H2><P>It has been reported that xylR has to be in high concentrations in order to repress in an effective manner. So we decided to look the dynamics of the AND with constitutive expression (The endogenous expression plus the pveg constitutive expression) of xylR vs No-Constitutive expression of xylR (just the endogenous expression). Our simulation says that for a finest logic behaviour it is needed the constitutive expression. Because when we compare the dynamics of the downstream protein (pink lines label with P4) when we add a single input burst of 100,000 arabinose molecules, the active araC concentration is depleted, so all the repression is given by xylR. But in the NO-constitutive expression AND the concentration of P4 goes significant higher than the AND with the constitutive expression. So we decided to express xylR constitutively in our constructions</P><H2><SPAN class="mw-headline" id="Transfer_Function"><B>Transfer Function</B></SPAN></H2><P>Our logic AND gates are fully dependent on the intracellular concentrations of the inducers (in this case, xylose and arabinose). To accomplish our duty in model, we therefore make use of the &quot;Transfer Function&quot;, which requires taking into account the regulation and dynamics of the endogenous Bacillus subtilis influx-efflux intracellular sugar regulation system. The &quot;Transfer Function&quot; is going to be the &quot;Black Box&quot; that will give you the dynamic
concentration through time of whatever is downstream of the &quot;AND&quot; when you
feed it with input data. It doesn't matter if it’s a single input burst or a
continuous input.

So, our first task was to reconstruct the regulatory network of B. subtilis for the influx-efflux system. The regulation data was retrieved from several papers and databases like <A href="http://bsubcyc.org/" class="external free" rel="nofollow">http://bsubcyc.org/</A></P><CENTER><TABLE border="0" cellspacing="15" bgcolor="transparent" id="tablecontentbg" cellpadding="10"><TBODY><TR><TD id="rightcolumn2" align="center"><P>Sweet AND gate: Arabinose and xylose are imported into the cell by the araE permease. The
</P><P>carbon metabolism system is highly regulated in order to optimize the consumption of sugars
using the least amount of energy. XylR and AraR are repressors responsible for the regulation of genes involved in the metabolism and intake of xylose and arabinose, respectively. Although XylR is not well characterized, it’s hypothesized that XylR represses itself; some studies suggest a correlation between XylR and the concentration of xylose inside the cell. AraR not only represses the production of genes like araE or araA, but it also represses itself. AraA and xylA are isomerases that convert L-arabinose into L-ribulose and D-xylose into D-xylulose. XylR from B.subtilis is also used by our construction as a repressor. Under that constitutive promoter, AraC is also produced for the regulation of the hybrid promoter. The output of the AND is the production of a transcription factor, LasR or P4, and RFP.
</P></TD></TR></TBODY></TABLE></CENTER><P>
One observation worthy of mention came when we realized that the
system was most repressed when there was scarce xylose in the system, not
when there was none. On the contrary, when there is no xylose in the system,
there is a very small expression gradient dependent on the concentration of
arabinose, yet this gradient does not surpass a threshold where we could
consider the AND as &quot;ON&quot;.</P><P>We are coupling an endogenous and an exogenous repressor to our AND. This
feature can easily be explained by the fact that we are expressing the
endogenous repressor XylR in a constitutive manner. This is due to the fact that
it has been reported that it has to be in high concentrations in order to repress
in an effective manner. When we add xylose to the medium, endogenous XylR
gene increases its expression. This makes
repression against the AND increase when xylose is found in the medium in
small amounts. After the repression threshold is surpassed, xylose stops being
a repressor and shifts to being an activator, so that an increase in its
concentration augments the AND's downstream protein expression.</P><P>
The suggested amount of each input that activates the gate according to our model is:</P><H2><SPAN class="mw-headline" id="Deterministic_model"><B>Deterministic model</B></SPAN></H2><P><B><A href="/Team:UNAM_Genomics_Mexico/Modeling/Parameters" title="Team:UNAM Genomics Mexico/Modeling/Parameters"> General assumptions</A></B>:</P><P>Xylose and arabinose are part of the pool of molecules that can be used by B. subtilis as carbon sources. Among membrane transporters, affinity for particular kinds of sugars is often high; nevertheless AraE is the main permease responsible for the intake of arabinose, xylose, and galactose monosaccharides. AraE is down-regulated by AraR, which also represses some other genes involved in arabinose metabolism. AraR is inactivated by arabinose and by itself [1]. These little details evidentiate the fact that arabinose is required for xylose cellular influx.
The concentrations of XylA and AraA depend on the concentration of their respective repressors, XylR and AraR, as well as their respective dissociation constants.</P><P>Intracellular xylose induces the production of genes involved in its metabolism, also through
the inactivation of their correspondent repressor, XylR. XylA is the enzyme that catalyzes the
conversion of D-xylose to D-xylulose. For arabinose, the isomerase that catalyzes the conversion of L-arabinose into L-ribulose is AraA [2].
In that way, the amount of intracellular arabinose is defined as the amount introduced by AraE minus the amount converted to L-ribulose by AraA; for xylose the process is similar but the degradation depends on xylA. We assume that the rates of intake of both sugars by AraE are similar. For the degradation rates by AraA and XylA, they are defined by Michaelis-Menten kinetics, since this kind of equation takes into account the energy consumption, and also because it is a simple reaction with one step and one substrate.</P><P>As for the amount of XylR presented in the system, we had two sources of production. One is the endogenous production in Bacillus subtilis, while the other is the production by our exogenous construct inserted into the genome under the Pveg constitutive promoter. This construction was introduced to counteract the leakiness of the promoter under XylR repression <A href="http://partsregistry.org/Part:BBa_K143036" class="external text" rel="nofollow">BBa_K143036</A>.</P><P>For AraR, which is produced only through the endogenous B. subtilis production, we considered the self-repression and described its production through a Hill function dependent on the concentration of the protein[3].</P><P>
The final TF involved in our system is AraC, from E. coli, which is produced in our construct by the Pveg promoter. Its concentration depends exclusively on the maximal rate of transcription, and the general degradation rate. Since the repression method used by AraC is DNA looping, we used the same approach as in [4], where the repression intensity of AraC was defined in terms of arabinose instead of AraC concentration. In this case, the Hill coefficient used was n=3, as reported in the same source as the best fitting value. Under the same assumption of independence between the two TF involved in the hybrid promoter, the total repression intensity is the repression given by AraC DNA looping plus the intensity of XylR, both modeled with a Hill function.</P><H2><SPAN class="mw-headline" id="Conclusions:"><B> Conclusions:</B></SPAN></H2><P>Complex mixture of inputs of varying natures, plus their collective properties and regulation of the designed system, yield very interesting behaviors. First, we want to highlight the existence of two unique steady states that are derived from the kind of addition of the inputs: one is reached when after an initial addition of sugars, these are degraded (in this case, there is an interval of time that shows the increment of the downstream protein and then its production rate decays as the sugars are degraded until they reach the state preceding the addition of sugars) and the other is  when the amount of sugars is constant in the environment (like when using a chemostat the cell reaches a new state where there’s an equilibrium between the sugar catabolism and the continuous production of the downstream TF in the AND).
The last remark is about the contribution to the AND by each input. We could see that when we had null or low levels of xylose, the repression remained strong. In the case of arabinose, its concentration contributes to the max level of expression of the downstream TF and has an impact on the way curve  of expression behaves. Our hypothesis is that the relevance of arabinose is due to the fact that it plays a role in the regulation of the transportes, which has as its job the introduction of both kind of sugars. Our hypothesis is based on the fact that we see in the gradient graphs the decay of the transporter, followed by a sudden increase of it.

References</P><P>[1]Krispin O, Allmansberger R. The Bacillus subtilis AraE protein displays a broad substrate specificity for several different sugars. J Bacteriol. 1998 June; 180(12): 3250–3252. PMCID: PMC107832.
[2]Gu Y, Ren C, Sun Z, Rodionov D A, Zhang W, Yang S, Yang C, Jiang W. Reconstruction of xylose utilization pathway and regulons in Firmicutes. BMC Genomics 2010, 11:255 doi:10.1186/1471-2164-11-25.
[3]Sá-Nogueira I, Nogueira T V, Soares S, Lencastre H. The Bacillus subtilis L-arabinose (ara) operon: nucleotide sequence, genetic organization and expression. Microbiology. 1997. 143:957-969.
[4]Megerle, Judith. Cell to cell variability of gene expression dynamics in inducible regulatory networks. Dissertation of Physics Faculty of Ludwig Maximilians University of Munich. January 2011.</P></DIV><DIV class="printfooter">
Retrieved from &quot;<A href="http://2012.igem.org/Team:UNAM_Genomics_Mexico/Modeling/Sweet_AND">http://2012.igem.org/Team:UNAM_Genomics_Mexico/Modeling/Sweet_AND</A>&quot;</DIV></DIV></DIV><DIV id="footer-box" class="noprint"><DIV id="footer"><UL id="f-list"><LI id="t-recentchanges"><A href="/Special:RecentChanges" title="Recent changes">Recent changes</A></LI><LI id="t-whatlinkshere"><A href="/Special:WhatLinksHere/Team:UNAM_Genomics_Mexico/Modeling/Sweet_AND" title="List of all wiki pages that link here [j]" accesskey="j">What links here</A></LI><LI id="t-recentchangeslinked"><A href="/Special:RecentChangesLinked/Team:UNAM_Genomics_Mexico/Modeling/Sweet_AND" title="Recent changes in pages linked from this page [k]" accesskey="k">Related changes</A></LI><LI id="t-specialpages"><A href="/Special:SpecialPages" title="List of all special pages [q]" accesskey="q">Special pages</A></LI><LI><A href="/Special:Preferences">My preferences</A></LI></UL></DIV><DIV id="footer"><UL id="f-list"><LI id="t-print"><A href="/wiki/index.php?title=Team:UNAM_Genomics_Mexico/Modeling/Sweet_AND&amp;printable=yes" title="Printable version of this page [p]" accesskey="p">Printable version</A></LI><LI id="t-permalink"><A href="/wiki/index.php?title=Team:UNAM_Genomics_Mexico/Modeling/Sweet_AND&amp;oldid=297029" title="Permanent link to this revision of the page">Permanent link</A></LI><LI id="privacy"><A href="/2012.igem.org:Privacy_policy" title="2012.igem.org:Privacy policy">Privacy policy</A></LI><LI id="disclaimer"><A href="/2012.igem.org:General_disclaimer" title="2012.igem.org:General disclaimer">Disclaimers</A></LI></UL></DIV></DIV></DIV></DIV></BODY></HTML>