Team:Purdue/Organisms
INTRODUCTION
BACKGROUND
The Problem
Organisms
Enzymes
PROJECT
Description
Project Development
Our Solution
PARTS
Team Parts
Basic Parts
Composite Parts
Part Collection
Parts Used
Parts Submitted
NOTEBOOK
ATTRIBUTIONS
OUR PROCESS
HUMAN PRACTICES
SAFETY
MODELING
Organisms
Yeast
Yeast as a Chassis
Escherichia coli is often used as the model prokaryotic organism. Similarly, Saccharomyces cerevisiae is often considered to be the model eukaryotic organism. S. cerevisiae is just one species of budding yeast, but it has been used for baking and fermentation since antiquity. Human use of yeast is thought to have begun over 7000 years ago¹.  In fact, S. cerevisiae is commonly known as baker’s yeast or brewer’s yeast, and the scientific name cerevisiae derives from an archaic word for beer.
S. cerevisiae (from now on referred to yeast for short) is classified as a fungus or mold. Although yeast is a single-celled organism, it shares many of the characteristics of other eukaryotes including a nucleus, cytoskeleton, and all of the major membrane-bound subcellular organelles. Also like E. coli, S. cerevisiae has been extensively characterized and, in 1996, its genome became the first fully sequenced eukaryotic genome.²
Figure 1: Budding yeast cells expressing different markers to track division. This process of division is called budding because when a yeast cell replicates, smaller daughter cells pinch, or bud, off the mother cell.¹
Below are some more advantages of using yeast as a chassis:
Yeast cells can be easily cultured and have a doubling time of around 90 minutes at 30°C.
Homologous recombination can be used to delete old genes or add new ones when transforming yeast.
Yeast has been adapted for use in both aerobic and anaerobic large scale culture.
Since yeast is a eukaryote, it shares many of the same complex internal structures found in plants and animals but without the high percentage of non-coding DNA found in more complicated eukaryotes.³
Our team decided to use yeast as our chassis for all the reasons listed above, but more specifically because several of the genes for the enzymes we chose to evaluate were isolated from the genome of termites and different fungi. Since we wanted to express genes taken from eukaryotic organisms, we decided to use yeast as our model system.
References
Duina, A. A., Miller, M.E., Keeney, J.B., 2014. Budding Yeast for Budding Geneticists: A Primer on the Saccharomyces cerevisiae Model System. Genetics 197(1): 33-48. doi: 10.1534/genetics.114.163188.
Goffeau, A., Barrell, B. G., Bussey, H., Davis, R. W., Dujon, B., et al., 1996. Life with 6000 Genes. Science 274(5287): 546, 563–567.
Lundblad, V. and Struhl, K. 2010. Yeast. Current Protocols in Molecular Biology. 92:13.0:13.0.1–13.0.4.
Termite
Biomimicry from Termites
Termites mainly feed on dead plant material and wood. The cellulose present in plant fiber has the potential to be a rich energy source, but it remains difficult to convert to sugars. Wood is mainly composed of 38-50% cellulose, 17-32% hemicellulose, and 15-30% lignin [1]. Similar to biofuel production, lignin acts as a significant obstacle to wood digestion.
Despite the challenges, termites have developed specialized digestive systems that help them process their woody food. A termite’s gut is essentially one of nature’s most efficient bioreactors, converting 95% of cellulose into simple sugars within 24 hours [2]. The picture below shows how termites ingest wood particles, break down the plant cell walls, and produce sugar [2].
Figure 1: US DOE. June 2007. Biofuels: Bringing Biological Solutions to Energy Challenges, US Department of Energy Office of Science.
Initially, researchers thought that the bacteria in the termites’ guts were what allowed them to digest the plant matter. However, although the bacteria aid in converting cellulose to sugars, most of the lignin degradation (which allows the cellulose to be digested) is actually done by the termite itself in combination with microorganisms called protists. By experimentally comparing the effects of diets containing varying degrees of lignin complexity and by looking at gene expression in the termite Reticulitermes flavipes, researchers identified two enzyme families responsible for aiding digestion of lignin rich diets: aldo-keto reductases and catalases [1]. These enzyme families are not involved in cellulose or hemicellulose digestion, yet their presence significantly enhances the amount of sugar recovered from plant matter.
Our team met with one of the researchers from this study, Dr. Michael Scharf, and he explained how these enzymes were located inside the lumen of the termite and served more as a pretreatment for the wood as it travels through the termite’s digestive system. Experiments that Dr. Scharf has done with the termite enzymes have shown that aldo-keto reductase can boost monosaccharide output by four times the original amount when using pre milled corn stalk for termite food.
Sources:
Sethi, A., Slack, J., Kovaleva, E., Buchman, G., Sharf, M. “Lignin-associated Metagene Expression in a Lignocellulose-digesting Termite.” Insect Biochemistry and Molecular Biology. Vol. 43, 2013, pp. 91-101.
US DOE. June 2007. Biofuels: Bringing Biological Solutions to Energy Challenges, US Department of Energy Office of Science.
White Rot Fungus: Biomimicry from Fungi
White rot fungi are a specific variety of fungus which are capable of decomposing various woody debris and living or dead trees by having the ability to break down lignin and cellulose found within trees, dead trees, and wooden debris. Although most species are known to digest the entirety of the wood they grow on, certain species of white rot are known to break down just the lignin, leaving characteristic white, string type patches that are almost mainly made up of cells containing high amounts of cellulose [1]. To achieve this effect, the fungus releases various lignin degrading enzymes (a variety of peroxidases) that oxidize aromatic rings within the lignin and break the structure of lignin into smaller subunits. At this point, cellulases and xylanases pass through the degraded lignin layer of the wood and extract the polysaccharides: the remaining cellulose and hemicellulose.The cell wall structure is shown in the second figure. These polysaccharides are then used by the fungus for nutrients, growth, and survival. Though white rot is very efficient at decaying the wood, the process of lignin degradation usually occurs on cells that are localized around the fungus only. Therefore, a majority of the infected wood will remain un-decayed until the fungus gradually spreads [1]. For the purposes of our project, we employed enzymes from the species Colletotrichum fioriniae PJ7 and Heterobasidion irregulare TC 32-1, specifically its genes for versatile peroxidase and manganese peroxidase respectively, though it produces an entire family of ligninases.
Sources:
http://www.wisconsinmushrooms.com/files/Trametes_pubescens__White_Rot_Fungus_.JPG
http://energy.mississippi.org/energycd/Report/ENERGYREPORT_files/image007.gif  http://genomea.asm.org/content/2/2/e00112-14.full
Blanchette, R. (2015). Microorganisms causing decay in trees and wood. Retrieved August 28, 2015, from http://forestpathology.cfans.umn.edu/microbes.htm
J4 Yeast
Background
Our initial obstacle was the inhibition of the enzyme beta-glucosidase by the products of lignin digestion. The current solution utilized by industries is to just add more enzymes to the mix; however, this requires extra expenditure. Our solution is to use a yeast strain developed by researchers at the University of Pennsylvania, the University of Illinois at Urbana-Champaign, and the University of California, Berkeley. The engineered strain, called J4-a,  combines multiple processes of microbial ethanol production into one-cohesive anaerobic-biological process. The J4-a strain specifically combines the processes of fermentation of cellobiose, xylose, and acetic acid into an intercellular co-fermentation process which, not only prevents the inhibition of beta-glucosidase, but also eliminates intermediate processes and overall expenses.
Image showing the J4-a yeast and the processes that go on inside of it.
Genes Utilized
Organism: Neurospora crassa (cellulolytic fungi)
Genes: cdt-1 (cellodextrin transporter) and gh1-1 (intracellular β-glucosidase)
Function: cellobiose consumption and prevents inhibition of glucose
Organism: Scheffersomyces stipitis
Genes: XYL1: XR (NAD(P)H-linked xylose reductase), XYL2: XDH (NAD+ linked xylitol dehydrogenase), XYL3: XK (xylulose kinase)
Function: Creates xylose assimilation pathway
Organism: E. coli
Genes: adhE (NADH-consuming acetylating acetaldehyde dehydrogenase ￼(AADH))
Function: Acetate reduction pathway to reduce acetate to ethanol. Also couples to XR-XDH pathway to use surplus NADH to reduce acetate and produce NAD+ for xylose assimilation pathway.
Combined genes: Yeast strain EJ3
Origins: Yeast strain SR8 (Constructed in lab by optimizing strain D452-2)
Optimized Genes: XYL1, XYL2, and XYL3. ALD6 (acetaldehyde dehydrogenase) was deleted to minimize acetate accumulation
EJ4
Yeast Strain EJ4-a was produced by serial subcultures of EJ3. EJ4 consumed cellobiose by a rate of  200% more than EJ3. Strain EJ4 had higher copies of gh1-1 (18) and cdt-1 (4) than strain EJ3 (6 copies of gh1-1 and 2 copies of cdt-1). Compared to the the control strain, EJ4-c, EJ4-a produced substantially more ethanol and lower accumulation of byproducts. EJ4-a and EJ4-c also had similar growth curves even though EJ4-a used ATP for the acetate reduction pathway.
Sources:
Wei, Na, Eun Joong Oh, Gyver Million, Jamie H. D. Cate, and Yoong-Su Jin. "Simultaneous Utilization of Cellobiose, Xylose, and Acetic Acid from Lignocellulosic Biomass for Biofuel Production by an Engineered Yeast Platform." ACS Publications (2015): n. pag. ACS Synthetic Biology. Web. 2 June 2015.
