<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_British_Columbia_Project_S-Layer_Cellulases skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><DIV id="top_title"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:British Columbia/Project/S-Layer/Cellulases</SPAN></H1></DIV><DIV id="HQ_page"><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><P><TITLE>Main CSS</TITLE><TITLE>Navbar CSS</TITLE></P><DIV class="container-fluid"><DIV class="collapse navbar-collapse"><UL class="nav navbar-nav navbar-right"><LI class="dropdown"><A href="#" class="dropdown-toggle" data-toggle="dropdown">Project
        </A></LI><LI><A href="https://2016.igem.org/Team:British_Columbia/Description">Description</A></LI><LI class="dropdown-submenu"><A href="#" class="dropdown-toggle" data-toggle="dropdown">S-Layer</A></LI><LI><A href="https://2016.igem.org/Team:British_Columbia/Project/S-Layer/Cellulases">Cellulases</A></LI><LI><A href="https://2016.igem.org/Team:British_Columbia/Project/S-Layer/Laccases">Laccases</A></LI></UL><LI><A href="https://2016.igem.org/Team:British_Columbia/Project/Bio-Pathways">Biosynthetic Pathways</A></LI><LI><A href="https://2016.igem.org/Team:British_Columbia/Project/Consortia">Consortia</A></LI><LI><A href="https://2016.igem.org/Team:British_Columbia/Proof">Proof</A></LI><LI><A href="https://2016.igem.org/Team:British_Columbia/Model">Modelling</A></LI><LI class="dropdown"><A href="#" class="dropdown-toggle" data-toggle="dropdown">Human Practices
        </A></LI><LI><A href="https://2016.igem.org/Team:British Columbia/HP/Gold">Introduction</A></LI><LI><A href="https://2016.igem.org/Team:British_Columbia/Human_Practices/Infographic">Infographic</A></LI><LI><A href="https://2016.igem.org/Team:British_Columbia/Human_Practices/Interview">Interview</A></LI><LI class="dropdown"><A href="#" class="dropdown-toggle" data-toggle="dropdown">Team
        </A></LI><LI><A href="https://2016.igem.org/Team:British_Columbia/Team/Bios">Bios</A></LI><LI><A href="https://2016.igem.org/Team:British_Columbia/Attributions">Attributions</A></LI><LI><A href="https://2016.igem.org/Team:British_Columbia/Team/Sponsors">Sponsors</A></LI><LI><A href="https://2016.igem.org/Team:British_Columbia/Notebook">Notebook</A></LI><LI class="dropdown"><A href="#" class="dropdown-toggle" data-toggle="dropdown">Achievements
        </A></LI><LI><A href="https://2016.igem.org/Team:British_Columbia/Collaborations">Collaborations</A></LI><LI><A href="https://2016.igem.org/Team:British_Columbia/Achievements/Medal_Criteria">Medal Criteria</A></LI><LI class="dropdown-submenu pull-left"><A href="#" class="dropdown-toggle" data-toggle="dropdown">Parts</A></LI><LI><A href="https://2016.igem.org/Team:British_Columbia/Parts">Parts</A></LI><LI><A href="https://2016.igem.org/Team:British_Columbia/Basic_Part">Basic Part</A></LI><LI><A href="https://2016.igem.org/Team:British_Columbia/Composite_Part">Composite Part</A></LI><LI><A href="https://2016.igem.org/Team:British_Columbia/Part_Collection">Parts Collection</A></LI></DIV></DIV><NAV class="navbar navbar-clear navbar-fixed-top"><DIV class="container-fluid"><DIV class="collapse navbar-collapse"><UL class="nav navbar-nav navbar-right"><LI class="dropdown"><A href="#" class="dropdown-toggle" data-toggle="dropdown">Project
        </A></LI><LI><A href="https://2016.igem.org/Team:British_Columbia/Description">Description</A></LI><LI class="dropdown-submenu"><A href="#" class="dropdown-toggle" data-toggle="dropdown">S-Layer</A></LI><LI><A href="https://2016.igem.org/Team:British_Columbia/Project/S-Layer/Cellulases">Cellulases</A></LI><LI><A href="https://2016.igem.org/Team:British_Columbia/Project/S-Layer/Laccases">Laccases</A></LI></UL><LI><A href="https://2016.igem.org/Team:British_Columbia/Project/Bio-Pathways">Biosynthetic Pathways</A></LI><LI><A href="https://2016.igem.org/Team:British_Columbia/Project/Consortia">Consortia</A></LI><LI><A href="https://2016.igem.org/Team:British_Columbia/Proof">Proof</A></LI><LI><A href="https://2016.igem.org/Team:British_Columbia/Model">Modelling</A></LI><LI class="dropdown"><A href="#" class="dropdown-toggle" data-toggle="dropdown">Human Practices
        </A></LI><LI><A href="https://2016.igem.org/Team:British Columbia/HP/Gold">Introduction</A></LI><LI><A href="https://2016.igem.org/Team:British_Columbia/Human_Practices/Infographic">Infographic</A></LI><LI><A href="https://2016.igem.org/Team:British_Columbia/Human_Practices/Interview">Interview</A></LI><LI class="dropdown"><A href="#" class="dropdown-toggle" data-toggle="dropdown">Team
        </A></LI><LI><A href="https://2016.igem.org/Team:British_Columbia/Team/Bios">Bios</A></LI><LI><A href="https://2016.igem.org/Team:British_Columbia/Attributions">Attributions</A></LI><LI><A href="https://2016.igem.org/Team:British_Columbia/Team/Sponsors">Sponsors</A></LI><LI><A href="https://2016.igem.org/Team:British_Columbia/Notebook">Notebook</A></LI><LI class="dropdown"><A href="#" class="dropdown-toggle" data-toggle="dropdown">Achievements
        </A></LI><LI><A href="https://2016.igem.org/Team:British_Columbia/Collaborations">Collaborations</A></LI><LI><A href="https://2016.igem.org/Team:British_Columbia/Achievements/Medal_Criteria">Medal Criteria</A></LI><LI class="dropdown-submenu pull-left"><A href="#" class="dropdown-toggle" data-toggle="dropdown">Parts</A></LI><LI><A href="https://2016.igem.org/Team:British_Columbia/Parts">Parts</A></LI><LI><A href="https://2016.igem.org/Team:British_Columbia/Basic_Part">Basic Part</A></LI><LI><A href="https://2016.igem.org/Team:British_Columbia/Composite_Part">Composite Part</A></LI><LI><A href="https://2016.igem.org/Team:British_Columbia/Part_Collection">Parts Collection</A></LI></DIV></DIV></NAV><TITLE>Sidebar CSS</TITLE><TITLE>Cellulases</TITLE><DIV class="cover"><DIV class="container-fluid"><DIV class="row" id="title"><DIV class="col-sm-12"><STRONG><P style="font-size: 3em">CellulaseS-Layer Engineering</P></STRONG></DIV></DIV></DIV></DIV><DIV id="breadcrumbs"><STRONG><A href="https://2016.igem.org/Team:British_Columbia">Home</A> /
<A href="https://2016.igem.org/Team:British_Columbia/Project/S-Layer/Cellulases">Project - Cellulase S-Layer Engineering</A></STRONG></DIV><DIV class="content-wrap"><DIV class="row"><NAV class="col-sm-3" id="side-menu"><UL class="nav sidebar-nav nav-pills nav-stacked"><LI class="active"><A href="#Abstract">Abstract</A></LI><LI><A href="#Key-Achievements">Key Achievements</A></LI><LI><A href="#Introduction">Introduction</A></LI><LI><A href="#Design">Design</A></LI><LI><A href="#Methods">Methods</A></LI><LI><A href="#Results">Results</A></LI><LI><A href="#Conclusion">Conclusion</A></LI><LI><A href="#References">References</A></LI></UL></NAV><DIV class="col-sm-9"><SECTION id="Abstract" class="anchor"><H2>Abstract</H2><P>Lignocellulosic biomass, the most abundantly available renewable resource in nature, represents a promising alternative to fossil fuels for sustainable production of chemicals, materials, and biofuels. The main technological obstacle for industrial exploitation of biomass is the recalcitrant nature of lignocellulosic polymers. One approach to harness the energy contained in biomass is to convert the lignocellulosic polymers into simple sugars, which can be further transformed in valuable compounds. Some anaerobic microorganisms have developed approaches to break down recalcitrant plant biomass using elaborate extracellular enzyme complexes, called cellulosomes, containing a scaffolding protein with many attached cellulolytic enzymes. The systems were adopted by scientists to generate artificial cellulosomes for biotechnological applications. These approaches have had their limitations due to inefficiencies with product yield, which  are inevitably incurred as a consequence of the metabolic strain experienced by single microbial strains that comprise most modern bioprocessing systems. </P><P> Our approach focuses on adapting the surface layer of <I>Caulobacter crescentus</I> for the high density display of cellulolytic enzymes. We specifically chose <I>C. crescentus</I> as it natively expresses a two dimensional crystal lattice protein called RsaA, which makes up the surface layer (S-Layer). The S-layer can be engineered for surface display of recombinant proteins. Our goal was to clone cellulolytic enzymes into the S-Layer protein to create a fusion protein for surface display, granting <I>C. crescentus</I> the ability to degrade cellulose to simple sugars. We successfully cloned, confirmed surface expression, and activity of the cellulases. Additionally, we demonstrated that surface expression of cellulases allowed <I>C. crescentus</I> to grow on cellulose as a sole carbon source, compared to wild-type <I>C. crescentus</I>, which is unable to do so. We believe that our approach will simplify the process of &quot;cellulosome&quot; engineering by exploiting the robust surface display system of <I>C. crescentus</I>. The developed platform can be easily adapted for the display of different proteins with different enzymatic activities.
                    </P></SECTION><SECTION id="Key-Achievements" style="font-size: 16px" class="anchor"><H2>Key Achievements</H2><P><LI>Cloned 5 different cellulases, such as β-1,4-endoglucanase (Endo5A), β-1,4-glucosidase (Gluc1C), β-1,4-exoglucanase (CEX) and 2 versions of  β-1,4-endoglucanase(E1) in p4A723 plasmid containing <I>rsaA</I> and transformed into <I>C. crescentus</I> for the display on cell surface.</LI><LI>Cloned 4 different cellulases - Endo5A, Gluc1C, E1, G12 into pSB1C3 and submitted as parts <A href="http://parts.igem.org/wiki/index.php?title=Part:BBa_K2139001"> BBa_K2139001</A>,<A href="http://parts.igem.org/wiki/index.php?title=Part:BBa_K2139002"> BBa_K2139002</A>, <A href="http://parts.igem.org/wiki/index.php?title=Part:BBa_K2139003"> BBa_K2139003, </A><A href="http://parts.igem.org/wiki/index.php?title=Part:BBa_K2139004"> BBa_K2139004</A>.</LI><LI>Confirmed surface expression of (rsaA)-cellulase fusion proteins with Endo5A, Gluc1C, E1_422, E1_399 constructs on <I>C. crescentus</I> by SDS-PAGE and Western Blot.</LI><LI>Confirmed cellulase activity of Endo5A, Gluc1C, E1_422, E1_399 cellulase constructs expressed on the surface of <I>C. crescentus</I>.</LI><LI>Confirmed growth of <I>C. crescentus</I> strains displaying cellulases on cellulose as a sole carbon source. </LI><LI>Codon-optimized β-1,4-endoglucanase (cenA) derived from Registry for expression in <I>C. crescentus</I>. Cloned the gene into p4A723 plasmid for the display on <I>C. crescentus</I> surface.</LI></P></SECTION><SECTION id="Introduction" class="anchor"><H2>Introduction</H2><P>Plant biomass is the most abundant renewable resource on Earth. Usually plant biomass consists of 30-50% of cellulose, 20-40% hemicellulose carbohydrates and 20-35% lignin heteropolymer, which are integrated in each other to form complex carbohydrate  structures recalcitrant to degradation. Cellulose represents a linear polymer of glucose molecules connected through β-1,4 glycoside bonds (Fig.1). In nature the degradation of cellulose into soluble sugars often involves the synergistic action of several different enzymes, including exoglucanases, endoglucanases and β-1,4-glucosidases.
                    </P><P><B>Figure 1.</B> Molecular structure of cellulose polymer.</P><P><I>C. crescentus</I> is a Gram negative non-pathogenic bacterium found in many freshwater and
                        soil environments (Poindexter,J.S. 1981). It displays, on its outer surface, a crystalline protein lattice called a surface layer (S-layer) which is made of multiple repetitions of the same RsaA protein encasing the whole surface of the cell (Fig.2)(Smit, J.1992). </P><P><B>Figure 2.</B> (left) Schematic top view on <I>C. crescentus</I> surface layer protein hexagonal arrangement
                        (right) Gram negative cell wall structure with the surface layer.</P><P>For our project we have used the RsaA fusion protein secretion system designed by the Smit lab at the University of British Columbia. We created a variety of cellulase-expressing <I>C. crescentus</I> strains using the S-layer as a surface expression system. We characterized expression and activity of the cellulases and tested growth of the strains on cellulose as a sole carbon source. We attempted to grow different combinations of strains to see if they could work in synergy to increase the rate of cellulose degradation and yield of monosaccharide release.</P></SECTION><SECTION id="Design" class="anchor"><H2>Design</H2><P>Engineering of the S-layer, encoded by the <I>rsaA</I> gene, has been made possible using a <I>C. crescentus</I> genomic rsaA knockout mutant, that can be complemented with a <I>rsaA</I> gene that contains an internal multiple cloning site. Using the multiple cloning site, genes encoding proteins of interest can be cloned in, in-frame, to create a construct that encodes a contiguous protein that contains the S-layer and internal protein of interest. </P><P>
In order to engineer cellulases onto the S-layer of <I>C. crescentus</I>, some key criteria had to be met for selection of candidate cellulases. This included that the cellulases had to be as small as possible, ideally less that 30 kDa, N- and C-termini within proximity of each other, and characterization in the literature for heterologous expression and activity. 

Once a candidate cellulase was identified, a BLAST search against the PDB was conducted to find a structure or structural homolog. Based on the structure or PHYRE2 (Protein Homology/analogY Recognition Engine V 2.0) homology modeling, and comparison with construct design in the literature, a construct was designed (Fig 3). </P><P><B>Figure 3.</B> Structures of cellulases candidates.  A) Model of CenA structure generated by PHYRE2 using PDB:1UOZ, endoglucanase Cel6 from Mycobacterium tuberculosis, as a structural model. The two sequences share 42% sequence identity and the model includes amino acids 162-447 and thiocellopentose (cyan) superimposed using the substrate bound 1UOZ template structure. Colored in pale blue are amino acid residues 162-189 that were excluded from the construct design for S-layer expression. The CenA construct for S-layer expression included amino acids 190-447 and N- and C-termini are colored blue and red, respectively. B) Structure of Gluc1C bound to thiocellobiose (purple) (PDB:2O9R_A). The structure includes amino acids 4-448, which is what we chose for our construct design for S-layer expression. N- and C-termini are within close proximity (colored blue and red, respectively).  </P><P>This process was done for all seven candidate cellulases. This meant that the carbohydrate domain was excluded and N-terminal regions were trimmed. Also, one cellulase, E1, constructs were designed to include (E1_422) or exclude (E1_399) the linker region between the cellulase and carbohydrate-binding domain, a P[S,T]-rich repeat region, based on activity described in the literature. Candidate cellulases were codon optimized for expression in C. crescentus and synthesized by IDT.</P><P>
Below is a summary of candidate cellulases that were cloned into the S-layer (Table 1). We searched for beta-1,4-glucosidase, endo-glucanase, and exoglucanases in hopes that we could grow a consortia capable of completely degrading cellulose to primary glucose units. </P><DIV class="scrollable"><TABLE><TBODY><TR><TH>Candidate cellulase </TH></TR></TBODY></TABLE></DIV></SECTION>Organism of originEnzyme typePDB structure (or homolog*)Amino acid residues included in constructLiterature referenceCEX<I>Cellulomonas fimi</I>exo-β-1,4-glucanase3CUF_A43-354Bingle, et al, 2000 Gluc1C<I>Paenibacillus sp.</I> MTCC 56391,4- β -glucosidase2O9R_A4-448Gupta, et al, 2013 Endo5a<I>Paenibacillus sp.</I> ICGEB2008endo- β -1,4-glucanase1ECE_A*, 1VRX_A*34-385Gupta, et al, 2013 E1_399<I>Acidothermus cellulolyticus</I>endo- β -1,4-glucanase1ECE42-399Linger, et al, 2010 E1_422<I>Acidothermus cellulolyticus</I>endo- β -1,4-glucanase1ECE42-422Linger, et al, 2010 G12<I>Acidothermus cellulolyticus</I>endo- β -1,4-glucanase1H0B_A*35-274Linger, et al, 2010 CenA<I>Cellulomonas fimi</I>endo- β -1,4-glucanase1UOZ*190-447iGEM repository
BBa_K118023
</DIV><P><B>Table 1.</B> Summary of candidate cellulases that were codon optimized for C. crescentus and synthesized by IDT for cloning into the S-layer for surface expression.</P><SECTION id="Methods" class="anchor"><H2>Methods</H2><P>
                        All <I>C. crescentus</I> cultures were grown in PYE media supplemented with 2 μg/ml chloramphenicol unless stated otherwise. All plasmid DNA extractions were performed with QIAprep Spin Miniprep Kit
                        (Qiagen). DNA purification from gels or PCR mixtures were done with NucleoSpin® Gel and PCR
                        Clean-up kit(Macherey-Nagel).</P><H3>Cloning of cellulases enzyme into rsaA plasmid in <I>C. crescentus</I></H3><P>The synthesized cellulase DNA was amplified using PCR with HF Phusion polymerase in 100 μl reactions. The following primers were used to amplify the selected region of cellulases genes and to add the BglII and PstI or NheI cut sites for cloning in p4A723 plasmid:</P><DIV class="scrollable"><TABLE><TBODY><TR><TD>Endo5A</TD><TD>Endo5a_fwd</TD><TD>5’-TCCAGATCTAGCGTCAAGGGGTATTACCAC</TD></TR><TR><TD>Endo5A</TD><TD>Endo5A_rev </TD><TD>5’-TCCATCTGCAGACACCGGCTTCATGATCCG </TD></TR><TR><TD>Gluc1C</TD><TD>Gluc1C_fwd </TD><TD>5’-TCCAGATCTAACACGTTCATCTTTCCGGC</TD></TR><TR><TD>Gluc1C</TD><TD>Gluc1C_rev </TD><TD>5’-TCCATCTGCAGAGAACCCGTTCTTGGCCAT</TD></TR><TR><TD>E1_399</TD><TD>E1_399_fwd </TD><TD>5’-TCCAGATCTGTTGCAGGCGGGGGTTATTG </TD></TR><TR><TD>E1_399</TD><TD> E1_399_rev </TD><TD>5’-TCCATCTGCAGAAACCGGGTCAAATATCGATGATTTTATC </TD></TR><TR><TD>E1_422</TD><TD>E1_422_fwd </TD><TD>5’-TCCAGATCTGTTGCAGGCGGGGGTTATTG </TD></TR><TR><TD>E1_422</TD><TD>E1_422_rev </TD><TD>5’-TCCATCTGCAGATGAGGGGGAGGGAGAC</TD></TR><TR><TD>G12</TD><TD>G12_fwd</TD><TD>5’-TCCAGATCTGCGACGACCTCCACG</TD></TR><TR><TD>G12</TD><TD>G12_rev</TD><TD>5’-TCCATCTGCAGATGAGGGGGTGGGAGTAG</TD></TR><TR><TD>CEX</TD><TD>CEX_fwd</TD><TD>5’-GGGAGATCTGCGACCACGCTCAAGGAGGCCGCC</TD></TR><TR><TD>CEX</TD><TD>CEX_Rev</TD><TD>5’-CTAGCTAGCCCCGGCCGGACCGGACGTCGG</TD></TR></TBODY></TABLE></DIV><P><B>Table 2</B> Primers for cloning cellulase gene inserts into <I>rsaA</I> P4A723 plasmid.</P><P>For the cloning of Endo5A, GlucIC, G12, E1_422 and E1_399 in P4A723 <I>rsaA</I> plasmid, the amplified
                        products and the isolated p4A723 plasmid DNA were digested with BglII and PstI restriction enzymes (NEB). For the cloning of CEX construct, the plasmid and amplified product were digested with BglII and NheI restriction enzymes. The plasmid digests were purified by agarose gel purification, while PCR product digests were purified by PCR purification. Purified digests were ligated using T4 ligase(NEB)
                        and the ligation mixes were then transformed in chemically competent DH5α <I>E. coli</I>.
                        Transformed colonies were then plated on and selected from a LB-CM plate and grown overnight in
                        LB-CM media. Recombinant plasmid was then isolated and sent for Sanger sequencing for confirmation.</P><P>
                        After sequence confirmation, the isolated construct DNA were electroporated into electrocompetent
                        <I>C. crescentus</I> with an RsaA Amber mutation and colonies were grown on a PYE-CM plate. After 3 days of growth one colony from each plate was selected and streaked
                        onto a fresh plate, which would be used for all future assays.</P><H3>
                        Cloning of cellulase enzymes into pSB1C3 plasmid with Ptac promoter and rbs
                    </H3><P>
                        The synthesized cellulases genes were amplified using PCR with HF Phusion polymerase in 100 μl reactions.
                        The following primers were used to amplify the selected region of cellulases genes and to add BioBrick Prefix and Suffix:</P><DIV class="scrollable"><TABLE><TBODY><TR><TD>Endo5A</TD><TD>Endo5A_fwd</TD><TD>5'-TCCGAATTCGCGGCCGCTTCTAGATGAAGAAGAAGGGC</TD></TR><TR><TD>Endo5A</TD><TD>Endo5A_rev </TD><TD>5'-TCCTACTAGTAGCGGCCGCTGCAGCTACTCCGCGGAGG</TD></TR><TR><TD>Gluc1C</TD><TD>Gluc1C_fwd </TD><TD>5'-TCCGAATTCGCGGCCGCTTCTAGATGAGCGAGAACACG</TD></TR><TR><TD>Gluc1C</TD><TD>Gluc1C_rev </TD><TD>5'-TCCTACTAGTAGCGGCCGCTGCAGTTAGAACCCGTTCTTG</TD></TR><TR><TD>E1</TD><TD>E1_fwd</TD><TD>5'-TCCGAATTCGCGGCCGCTTCTAGATGCCTCGCGCTCT</TD></TR><TR><TD>E1</TD><TD>E1_rev</TD><TD>5'-TCCTACTAGTAGCGGCCGCTGCAGCTAGGTGGGAGTTGGG</TD></TR><TR><TD>G12</TD><TD>G12_fwd</TD><TD>5'-TCCGAATTCGCGGCCGCTTCTAGATGTTAGTGTTAAGAGCC</TD></TR><TR><TD>G12</TD><TD>G12_rev</TD><TD>5'-TCCTACTAGTAGCGGCCGCTGCAGCTACGACGAAGTAGG</TD></TR></TBODY></TABLE></DIV><P><B>Table 3</B> Primers for cloning cellulase gene inserts in pSB1C3 plasmid.</P><P>
                        Purified pSB1C3 plasmid and the amplified Endo5A, Gluc1C, E1, G12 constructs were digested with
                        EcoRI  and SpeI restriction enzymes. The plasmid digests were purified by agarose gel purification,
                        while PCR product digests were purified by PCR purification. Purified digests were ligated using T4
                        ligase and the ligation mixes were then transformed in chemically competent DH5α <I>E. coli</I>.
                        Transformed colonies were then plated on LB-CM plate and the isolated colonies were selected and grown
                        overnight in LB-CM media. Recombinant plasmid was isolated and sent for Sanger sequencing for confirmation.
                    
                        pTAC was amplified from Bba_K868400 part using HF phusion PCR with the following primers to add the rbs (part Bba_B0034):

                        Ptac_rbs_F  5'-GTTTCTTCGAATTCGCGGCCGCTTCTAGAgagctgttgacaatta, Ptac_rbs_R 5'-GTTTCTTCCTGCAGCGGCCGCTACTAGTtttctcctctttaattgttatccgctca
                    
                        The amplified pTAC and rbs part was then digested with EcoRI and SpeI restriction enzymes.
                        pSB1C3 high copy assembly plasmid with the gene insert was digested using EcoRI and XbaI restriction enzymes.
                        The plasmid digest was then purified by agarose gel purification while the pTAC RBS digest was purified by PCR
                        purification. Both purified digest were ligated together using standard ligation protocol and ligation mix
                        was then transformed in chemically competent DH5α <I>E. coli</I>.
                    </P><H3>Surface layer fusion protein extraction</H3><P>             10 mL tubes of PYE-CM were inoculated with <I>C. crescentus</I> strains carrying the different
                        cellulase-rsaA fusions in p4A723 plasmid as well as a positive control carrying a plasmid with wild
                       type <I>rsaA</I> (P4A723) and a negative control of <I>rsaA</I> knock-out (ΔGCSS). Cultures were grown overnight at
                        30°C shaker. Cultures were taken out of incubator and optical density at 600 nm was measured. All
                        cultures were then normalized to the lowest OD by diluting the remaining culture with PYE.
                        Low pH extraction (Walker et al. 1992) was performed using different pHs of HEPES buffer to
                        extract only crystalline s-layer without lysing the cells. The procedure is illustrated on Fig 4. Proteins were then stored in -20°C freezer.
                        Remaining cell pellet was used to prepared a cell lysate by resuspending the pellet in 150 μl of
                        lysis buffer and boiling for 5 minutes.
                    </P><P><B>Figure 4.</B> Schematic presentation of low pH extraction procedure for removal of S-layer from <I>C. crescentus</I>.</P><H3> Surface layer fusion protein expression confirmation </H3><P>To confirm expression of surface layer fusion protein, <I>C. crescentus</I> cultures were grown and SDS-PAGE and western blot analysis were
                        performed on surface proteins from low pH extraction and on the cell lysates.
                        SDS-PAGE was done using 8% separating gels. Coomassie Brilliant Blue staining and western
                        immunoblotting was performed using protocols outlined in protocol section. Western blots were
                        probed with primary rabbit anti-RsaA polyclonal antibodies at 1/30,000 dilution. Goat anti-rabbit
                        IgG was used as secondary antibody at 1/50,000 dilution.
                        Fluorophore was detected by Odyssey Infrared Imaging System.</P><H3><I>C. crescentus</I> Surface Cellulase Activity Analysis</H3><P>
                        For cellulase enzyme activity measurement, triplicate 5mL PYE-CM starter cultures of p4A723 (Δ<I>rsaA C. crescentus </I> complemented with wildtype Δ<I>rsaA</I> in p4A723), E1_399, E1_422, Gluc1C and Endo5A <I>C. crescentus</I> strains were grown
                        in 10 mL tubes on a rotary shaker at 30°C for 2 days. Cultures were taken out of incubator and
                        OD600 was measured. All cultures were then normalized to the lowest OD 600 nm by diluting the
                        remaining culture with PYE. 1 mL of each sample was taken and resuspended with fresh PYE and
                        spin down 3 times to remove any lysed cell debris that may interfere with cellulase activity results. </P><P>
                        A protocol for cellulases activity assay was adopted from Mewis K.(2015) . 150 uL of each culture were aliquoted into a clear
                        96 well plate(Corning) and 150 μL of assay mix (0.1 mg/ml DNPC in 50 mM pH 5.5 potassium acetate buffer)
                        was added to the each well. OD 400 nm and OD 600 nm was measured every 30 minutes for several hours by
                        a VarioScan plate reader(Thermo). Between measurements the culture was incubating at 30°C.  Data was normalized further
                        by dividing the OD400nm measurement by the OD600nm measurement. </P><H3>
                        Growth of <I>C. crescentus</I> displaying cellulases on cellulose as a sole carbon source</H3><P>
                            Triplicate 5mL PYE-CM starter cultures of P4A723 (Δ<I>rsaA C. crescentus </I> complemented with wildtype Δ<I>rsaA</I> in p4A723), E1_399, E1_422, Gluc1C and Endo5A were grown
                            in 10 mL tubes on a rotary shaker at 30°C for 2 days. Cultures were taken out of incubator and
                            OD600 was measured. All cultures were then normalized to the lowest OD600 by diluting the remaining
                            cultures with PYE. 100-fold dilution was used to inoculate the cultures in 200 μL well clear plate
                            containing M2 with 0.2% w/v carboxymethylcellulose (CMC). For the co-cultures, equal-volume inoculations
                            were done to make final dilution of 100. The plate was incubate at 30C in a Tecan platereader
                            and OD600 was read every 30 minutes.
                    </P></SECTION><SECTION id="Results" class="anchor"><H2>Results</H2><H3>Cloning of cellulase enzymes into rsaA plasmid in <I>C.crescentus</I></H3><P>
                        Endo5A, Gluc1C, E1_422, and E1_399 were all successfully cloned into the p4A723 plasmid vector (Fig. 5) and
                        sequence confirmed by Sanger sequencing. Cloning was attempted on G12 several times, each time the
                        transformation step into DH5α <I>E.coli</I> failed, leading to the assumption that the gene product might
                        be toxic to the cell. As Endo5A, E1_422, and E1_399 are all endoglucanases, we decided to not proceed
                        with any troubleshooting on G12. CEX gene was successfully cloned into DH5α <I>E. coli</I> and sequence confirmed.
                        However after transforming the plasmid into <I>C. crescentus</I>, extreme cell clumping was observed, possibly a
                        result of the CEX fusion into the S-Layer. Due to the phenotype, we did not perform functional characterization of the CEX-rsaA fusion. 
                    </P><P><B>Figure 5.</B> Map of <I>rsaA</I> p4A723 plasmid.</P><H3>Surface layer fusion protein extraction expression confirmation</H3><P>Low pH extraction was done on Endo5A, Gluc1C, E1_399, and E1_422 RsaA fusion proteins as well as
                        P4A732 RsaA positive control and ΔGCSS (Δ<I>rsaA</I>) negative control. We also prepared a cell lysate from the
                        remaining cell pellet to compare the amount of protein within the cell as opposed to on the surface
                        of the cell. We ran two gels simultaneously staining one with Coomassie Brilliant blue and performing
                        western blot on the other.
                        
                        Thermo fisher prestained protein ladder was used to compare relative expected sizes of fusion proteins. Expected protein sizes were as followed: </P><DIV class="scrollable"><TABLE><TBODY><TR><TH>Protein </TH></TR></TBODY></TABLE></DIV></SECTION>SizeRsaA98 kDaltonsEndo5A-RsaA fusion138 kDaltonsGluc1C-RsaA fusion149 kDaltonsE1_399-RsaA fusion138 kDaltonsE1_422-RsaA fusion140 kDaltons</DIV><P>On the coomassie stained gel faint bands corresponding to cellulases-RsaA fusions were observed  in
                        low pH extracted proteins (Figure 6B). The cell lysate showed large amounts of intracellular proteins with poor
                        differentiation so no conclusions could be made from these. 
                        On the western blot analysis, RsaA fusion proteins were clearly observed at higher molecular
                        weight than wild type RsaA confirming successful expression of cellulases fused to the RsaA on the surface Figure 6A). No RsaA was detected in the negative control  ΔGCSS (Δ<I>rsaA</I>). The cell lysate results were comparable to the low pH extracted proteins,
                        with visibly more RsaA fusion proteins detectable in the cell lysate as opposed to the low pH
                        extracted proteins. This may be a sign that not all of the fusion protein can get out of the
                        cell and crystalize, likely due to the large size of the fusion protein. The higher visibility
                        of some constructs over others in the low pH extraction, such as with E1_399 vs. Gluc1C, is
                        indicative of higher expression and secretion outside of the cell. The smaller constructs such
                        as E1_399 and Endo5A were seen more clearly on both the coomassie stained gel and the western blot,
                        indicating that the smaller constructs might be easier to transport on the surface. RsaA and the fusion proteins ran about 30 kDa above expected their expected molecular weight. This could be due to incomplete denaturation of the proteins or an issue with original estimate of the size of RsaA expressed on the plasmid p4A732. 

                        Some fragments of lower molecular weight RsaA protein in the western, this is probably due to cleavage of the RsaA protein during the extraction. The blot was also performed with anti-serum, not purified monoclonal antibody.
                    </P><P><B>Figure 6.</B> (top) Western Blot of <I>C. crescentus</I> cellulase expressing strains, ran on SDS-PAGE and blotted with anti-RsaA. Left to right: (1) Thermofisher ladder, (2) Gluc1C cell lysate, (3) Endo5A cell lysate,
                        (4) E1_399 cell lysate, (5) E1_422 cell lysate, (6) negative control  ΔGCSS (Δ<I>rsaA</I>) cell lysate, (7) P4A723 (wildtype) cell lysate, (8) Thermofisher ladder, (9)
                        Gluc1C low pH extracted proteins, (10) Endo5A low pH extracted proteins, (11) E1_399 low pH extracted proteins, (12)
                        E1_422 low pH extracted proteins, (13) ΔGCSS (Δ<I>rsaA</I>) low pH extracted proteins, (14) P4A723 (wildtype) low pH extracted proteins. (bottom) SDS-PAGE stained with coomassie brilliant blue, samples loaded in the same order as above for the Western Blot. 

                    </P><H3><I>C. crescentus</I> cellulase activity analysis</H3><P>
                        Cellulase activity analysis was run using the protocols described in Method section for a period of 150 minutes. The assay mechanism is explained in Fig. 7. 2,4-dinitrophenylcellobiose (DNPC) consists of cellobiose bound to a 2,4dinitrophenol chromophore. If cellulase activity is present, the release of chromophore can be observed and measured at OD400</P><P><B>Figure 7.</B> Assay for cellulase activity with DNPC substrate.</P><P>
                        This assay was run on the <I>C. crescentus</I> cells expressing Endo5A, Gluc1C, E1_399, and E1_422 RsaA fusions using the wild type RsaA P4A723 protein as a control. From the first time point, cellulase-expressing <I>C. crescentus</I> demonstrated high cellulase activities, with E1_422 and E1_399 showing the highest activity level followed by Endo5A (Figure 8). Gluc1C had lower activity levels closer to that of the control, what can be explained by the glucosidase activity of Gluc1C. The enzyme cleaves a glucose molecule from DNPC first, not cellobiose what does not lead to the release of chromophore. Observation at later time points need to be taken to confirm the Gluc1C activity.
                    </P><P><B>Figure 8.</B> Cellulase activity assay results for <I>C. crescentus</I> displaying cellulases compared to <I>C. crescentus</I> expressing wildtype RsaA (P4A723).</P><H3><I>C. crescentus</I> growth on cellulose</H3><P><I>C. crescentus</I> cells expressing wild type RsaA, Endo5A, Gluc1C, E1_399, and E1_422 were grown on carboxymethylcellulose (CMC) as a sole carbon source to confirm the cellulase activities of the displayed enzymes. Endo5A and its co-cultures showed the best growth results on CMC (Fig. 9) As <I>C. crescentus</I> has intrinsic β-1,4-glucosidase, the displayed endoglucanase cleaves the cellulose in small disaccharide, which are further transformed by periplasmic glucosidase into glucose to sustain <I>C. crescentus</I> growth. E1_399 also demonstrates growth on cellulose and interestingly when co-cultured with Gluc1C it shows synergistic growth. The results suggest <I>C. crescentus</I> can be successfully used to transform cellulose in simple sugars and confirm that the first part of our consortia works! 
</P><P><B>Figure 9.</B> Growth of <I>C. crescentus</I> expressing wildtype RsaA (P4A723) compared to growth of <I>C. crescentus</I> strains expressing recombinant cellulases on carboxymethylcellulose (CMC) as a sole carbon source over 100 hours.</P><SECTION id="Conclusion" class="anchor"><H2>Conclusion</H2><P>For our project we employed robust expression and display system of <I>C. crescentus</I> S-layer for the generation of whole-cell biocatalysts. Previously, the S-layer was used to display small peptides and proteins. Our results are a first demonstration that <I>C. crescentus</I> can be successfully used for enzymes display. The cellulases enzymes used in the study were pre-selected based on their crystal structures and truncated to reduce the size without changing the catalytic activities. The expression of cellulases on cell surface was confirmed by Western blot and the cellulolytic activity was demonstrated in DNPC assay. Also we confirmed that the engineered <I>C. crescentus</I> strains can breakdown and grow on cellulose substrate.</P></SECTION><SECTION id="References" class="anchor"><H2>References</H2><OL>Amat, Fernando et al. “Analysis of the Intact Surface Layer of Caulobacter Crescentus by Cryo-Electron Tomography.” Journal of bacteriology 192.22 (2010): 5855–65.</OL><OL>Bingle, Wade H., John F. Nomellini, and John Smit. “Secretion of the Caulobacter Crescentus S-Layer Protein: Further Localization of the C-Terminal Secretion Signal and Its Use for Secretion of Recombinant Proteins.” Journal of Bacteriology 182.11 (2000): 3298–3301. </OL><OL>Farr, Christina et al. “Development of an HIV-1 Microbicide Based on Caulobacter Crescentus: Blocking Infection by High-Density Display of Virus Entry Inhibitors.” PLoS ONE 8.6 (2013): </OL><OL>Gandham, Lyngrace, John F Nomellini, and John Smit. “Evaluating Secretion and Surface Attachment of SapA, an S-Layer-Associated Metalloprotease of Caulobacter Crescentus.” Archives of microbiology 194.10 (2012): 865–77. </OL><OL>Gupta, Shefali, Nidhi Adlakha, and Syed Shams Yazdani. “2013-Efﬁcient Extracellular Secretion of an Endoglucanase and a B-Glucosidase in E. Coli.pdf.” 88 (2013): 20–25. </OL><OL>Linger, Jeffrey G., William S. Adney, and Al Darzins. “Heterologous Expression and Extracellular Secretion of Cellulolytic Enzymes by Zymomonas Mobilis.” Applied and Environmental Microbiology 76.19 (2010): 6360–6369. </OL><OL>Mewis, Keith, Marcus Taupp, and Steven J Hallam. “A High Throughput Screen for Biomining Cellulase Activity from Metagenomic Libraries.” Journal of visualized experiments : JoVE 48 (2011): n. pag. </OL><OL>Poindexter, J S. “The Caulobacters: Ubiquitous Unusual Bacteria.” Microbiological reviews 45.1 (1981): 123–79. </OL><OL>Presley, Gerald N. et al. “Extracellular Gluco-Oligosaccharide Degradation by Caulobacter Crescentus.” Microbiology (United Kingdom) 160.PART 3 (2014): 635–645.</OL><OL>Ratanakhanokchai, Khanok et al. “Paenibacillus Curdlanolyticus Strain B-6 Multienzyme Complex: A Novel System for Biomass Utilization.” Biomass Now - Cultivation and Utilization. N.p., 2013. 369–394.</OL><OL>Smit, J et al. “The S-Layer of Caulobacter Crescentus: Three-Dimensional Image Reconstruction and Structure Analysis by Electron Microscopy.” Journal of bacteriology 174.20 (1992): 6527–38.</OL><OL>Taylor, Publisher. “Anti-Tumor Effects of the Bacterium Caulobacter Crescentus.” (2006): 37–41. </OL><OL>Umelo-Njaka, Elizabeth et al. “Expression and Testing of Pseudomonas Aeruginosa Vaccine Candidate Proteins Prepared with the Caulobacter Crescentus S-Layer Protein Expression System.” Vaccine 19.11-12 (2001): 1406–1415. </OL><OL>Walker, S G, S H Smith, and J Smit. “Isolation and Comparison of the Paracrystalline Surface Layer Proteins of Freshwater Caulobacters.” Journal of bacteriology 174.6 (1992): 1783–92. </OL></SECTION></DIV></DIV></DIV><P id="read-more"><STRONG>Check out other parts of our project below!</STRONG></P><DIV id="up-next"><DIV class="row" style="max-width: 100%; margin: 0"><DIV class="col-sm-6" id="previous"><A href="https://2016.igem.org/Team:British_Columbia/Project/S-Layer/Laccases"><STRONG><FIGCAPTION>Laccases</FIGCAPTION></STRONG></A></DIV><DIV class="col-sm-6" id="next"><A href="https://2016.igem.org/Team:British_Columbia/Project/Bio-Pathways"><STRONG><FIGCAPTION>Biosynthetic Pathways</FIGCAPTION></STRONG></A></DIV></DIV></DIV></DIV></DIV></DIV></BODY></HTML>