<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_British_Columbia_Project_Bio-Pathways skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><DIV id="top_title"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:British Columbia/Project/Bio-Pathways</SPAN></H1></DIV><DIV id="HQ_page"><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><P><TITLE>Main CSS</TITLE><TITLE>Navbar CSS</TITLE></P><DIV class="container-fluid"><DIV class="collapse navbar-collapse"><UL class="nav navbar-nav navbar-right"><LI class="dropdown"><A href="#" class="dropdown-toggle" data-toggle="dropdown">Project
        </A></LI><LI><A href="https://2016.igem.org/Team:British_Columbia/Description">Description</A></LI><LI class="dropdown-submenu"><A href="#" class="dropdown-toggle" data-toggle="dropdown">S-Layer</A></LI><LI><A href="https://2016.igem.org/Team:British_Columbia/Project/S-Layer/Cellulases">Cellulases</A></LI><LI><A href="https://2016.igem.org/Team:British_Columbia/Project/S-Layer/Laccases">Laccases</A></LI></UL><LI><A href="https://2016.igem.org/Team:British_Columbia/Project/Bio-Pathways">Biosynthetic Pathways</A></LI><LI><A href="https://2016.igem.org/Team:British_Columbia/Project/Consortia">Consortia</A></LI><LI><A href="https://2016.igem.org/Team:British_Columbia/Proof">Proof</A></LI><LI><A href="https://2016.igem.org/Team:British_Columbia/Model">Modelling</A></LI><LI class="dropdown"><A href="#" class="dropdown-toggle" data-toggle="dropdown">Human Practices
        </A></LI><LI><A href="https://2016.igem.org/Team:British Columbia/HP/Gold">Introduction</A></LI><LI><A href="https://2016.igem.org/Team:British_Columbia/Human_Practices/Infographic">Infographic</A></LI><LI><A href="https://2016.igem.org/Team:British_Columbia/Human_Practices/Interview">Interview</A></LI><LI class="dropdown"><A href="#" class="dropdown-toggle" data-toggle="dropdown">Team
        </A></LI><LI><A href="https://2016.igem.org/Team:British_Columbia/Team/Bios">Bios</A></LI><LI><A href="https://2016.igem.org/Team:British_Columbia/Attributions">Attributions</A></LI><LI><A href="https://2016.igem.org/Team:British_Columbia/Team/Sponsors">Sponsors</A></LI><LI><A href="https://2016.igem.org/Team:British_Columbia/Notebook">Notebook</A></LI><LI class="dropdown"><A href="#" class="dropdown-toggle" data-toggle="dropdown">Achievements
        </A></LI><LI><A href="https://2016.igem.org/Team:British_Columbia/Collaborations">Collaborations</A></LI><LI><A href="https://2016.igem.org/Team:British_Columbia/Achievements/Medal_Criteria">Medal Criteria</A></LI><LI class="dropdown-submenu pull-left"><A href="#" class="dropdown-toggle" data-toggle="dropdown">Parts</A></LI><LI><A href="https://2016.igem.org/Team:British_Columbia/Parts">Parts</A></LI><LI><A href="https://2016.igem.org/Team:British_Columbia/Basic_Part">Basic Part</A></LI><LI><A href="https://2016.igem.org/Team:British_Columbia/Composite_Part">Composite Part</A></LI><LI><A href="https://2016.igem.org/Team:British_Columbia/Part_Collection">Parts Collection</A></LI></DIV></DIV><NAV class="navbar navbar-clear navbar-fixed-top"><DIV class="container-fluid"><DIV class="collapse navbar-collapse"><UL class="nav navbar-nav navbar-right"><LI class="dropdown"><A href="#" class="dropdown-toggle" data-toggle="dropdown">Project
        </A></LI><LI><A href="https://2016.igem.org/Team:British_Columbia/Description">Description</A></LI><LI class="dropdown-submenu"><A href="#" class="dropdown-toggle" data-toggle="dropdown">S-Layer</A></LI><LI><A href="https://2016.igem.org/Team:British_Columbia/Project/S-Layer/Cellulases">Cellulases</A></LI><LI><A href="https://2016.igem.org/Team:British_Columbia/Project/S-Layer/Laccases">Laccases</A></LI></UL><LI><A href="https://2016.igem.org/Team:British_Columbia/Project/Bio-Pathways">Biosynthetic Pathways</A></LI><LI><A href="https://2016.igem.org/Team:British_Columbia/Project/Consortia">Consortia</A></LI><LI><A href="https://2016.igem.org/Team:British_Columbia/Proof">Proof</A></LI><LI><A href="https://2016.igem.org/Team:British_Columbia/Model">Modelling</A></LI><LI class="dropdown"><A href="#" class="dropdown-toggle" data-toggle="dropdown">Human Practices
        </A></LI><LI><A href="https://2016.igem.org/Team:British Columbia/HP/Gold">Introduction</A></LI><LI><A href="https://2016.igem.org/Team:British_Columbia/Human_Practices/Infographic">Infographic</A></LI><LI><A href="https://2016.igem.org/Team:British_Columbia/Human_Practices/Interview">Interview</A></LI><LI class="dropdown"><A href="#" class="dropdown-toggle" data-toggle="dropdown">Team
        </A></LI><LI><A href="https://2016.igem.org/Team:British_Columbia/Team/Bios">Bios</A></LI><LI><A href="https://2016.igem.org/Team:British_Columbia/Attributions">Attributions</A></LI><LI><A href="https://2016.igem.org/Team:British_Columbia/Team/Sponsors">Sponsors</A></LI><LI><A href="https://2016.igem.org/Team:British_Columbia/Notebook">Notebook</A></LI><LI class="dropdown"><A href="#" class="dropdown-toggle" data-toggle="dropdown">Achievements
        </A></LI><LI><A href="https://2016.igem.org/Team:British_Columbia/Collaborations">Collaborations</A></LI><LI><A href="https://2016.igem.org/Team:British_Columbia/Achievements/Medal_Criteria">Medal Criteria</A></LI><LI class="dropdown-submenu pull-left"><A href="#" class="dropdown-toggle" data-toggle="dropdown">Parts</A></LI><LI><A href="https://2016.igem.org/Team:British_Columbia/Parts">Parts</A></LI><LI><A href="https://2016.igem.org/Team:British_Columbia/Basic_Part">Basic Part</A></LI><LI><A href="https://2016.igem.org/Team:British_Columbia/Composite_Part">Composite Part</A></LI><LI><A href="https://2016.igem.org/Team:British_Columbia/Part_Collection">Parts Collection</A></LI></DIV></DIV></NAV><TITLE>Sidebar CSS</TITLE><P><TITLE>Bio-Pathways</TITLE></P><DIV class="cover"><DIV class="container-fluid"><DIV class="row" id="title"><DIV class="col-sm-12"><STRONG><P style="font-size: 3em">Biosynthetic Pathways</P></STRONG></DIV></DIV></DIV></DIV><DIV id="breadcrumbs"><STRONG><A href="https://2016.igem.org/Team:British_Columbia">Home</A> /
<A href="https://2016.igem.org/Team:British Columbia/Project/Bio-Pathways">Project - Biosynthetic Pathways</A></STRONG></DIV><DIV class="content-wrap"><DIV class="row"><NAV class="col-sm-3" id="side-menu"><UL class="nav sidebar-nav nav-pills nav-stacked"><LI class="active"><A href="#Abstract">Abstract</A></LI><LI><A href="#Key-Achievements">Key Achievements</A></LI><LI><A href="#Introduction">Introduction</A></LI><LI><A href="#Methods">Methods</A></LI><LI><A href="#Results">Results</A></LI><LI><A href="#Conclusion">Conclusion</A></LI><LI><A href="#References">References</A></LI></UL></NAV><DIV class="col-sm-9"><SECTION id="Abstract" class="anchor"><H2>Abstract</H2><P>We attempted to engineer a pathway for β-carotene biosynthesis in <I>Escherichia coli</I>, as the second component of our consortium-based platform. We expect that glucose made available from degradation of cellulose by cellulase-expressing <I>Caulobacter crescentus</I> can be metabolized by our β-carotene-producing <I>E. coli</I>.  We first obtained growth curves for DH5α <I>E. coli</I> cells encoding  β-carotene biosynthetic genes in order to determine growth dynamics in minimal medium. We then acetone-extracted β-carotene from these cells to determine maximum yield of product at the stationary phase. Finally, we cloned the CrtEBIY operon under a constitutive promoter in pSB1C3 plasmid, to confer β-carotene-producing <I>E. coli</I> resistance to chloramphenicol, for co-culture with cellulase-expressing <I>Caulobacter crescentus</I> that has the same antibiotic resistance.</P></SECTION><SECTION id="Key-Achievements" style="font-size: 16px" class="anchor"><H2>Key Achievements</H2><LI>Established growth curves for DH5α <I>Escherichia coli</I> cells expressing β-carotene, in minimal medium supplemented with either different concentrations of glucose, cellobiose, or cellulose.</LI><LI>Characterized β-carotene production in  DH5α <I>E. coli</I> cells encoding β-carotene biosynthesis genes.</LI><LI>Cloned the operon encoding β-carotene production into pSB1C3.</LI></SECTION><SECTION id="Introduction" class="anchor"><H2>Introduction</H2><P><I>Escherichia coli</I> is commonly used as a chassis for the biosynthesis of valuable chemicals. Advantages to its use include fast growth kinetics, ease of acquiring high density cultures, ability to grow on many types of media that can be made from inexpensive reagents, and tractability for genetic manipulation (Rosano and Ceccarelli 2014). As such, many past iGEM teams have chosen to engineer biosynthetic pathways into <I>E. coli</I> for production of a wide range of commercially-desired products such as biofuels, violacein, astaxanthin, and its precursor β-carotene.	</P><P> This year, we intend to build a platform for producing valuable chemicals more sustainably by bridging lignocellulose processing directly to product manufacture. For our project, we decided to engineer E. coli to produce β-carotene in tandem to its growth with C.crescentus. In particular, we designed our system so that E. coli metabolizes the simple sugars released from cellulosic degradation accomplished by cellulase-expressing <I>C. crescentus</I>, to in turn produce β-carotene as a secondary metabolite. </P><P>β-carotene is a carotenoid found in many colored fruits and vegetables. It is the biosynthetic precursor to vitamin A which has roles in gene expression, vision, maintenance of body linings and skin, immune defenses, growth of the body, and normal development of cells (Webb, Sizer, and Whitney 2003). Aside from being a precursor, β-carotene also has functions as a potent quencher of singlet toxic oxygen species, and can act as an antioxidant that scavenges free radicals in human low density lipoprotein (LDL), high density lipoprotein (HDL), and cell membranes (Bendich 2004). </P><P>We chose to produce β-carotene as a proof-of-concept approach to validating the functionality of our system. However, our platform can be applied to the production of higher value chemicals as well. We anticipate that directly linking chemical production to treatment of lignocellulosic biomass, in the form of a microbial consortium, would result in both effectively reducing the costs associated with chemical production and valorizing lignocellulosic waste.</P></SECTION><SECTION id="Methods" class="anchor"><H2>Methods</H2><H3>Growth of DH5α <I>E. coli</I> cells expressing β-carotene in minimal medium</H3><P>Chemically competent DH5α <I>E. coli</I> cells were transformed with the 2009 Cambridge iGEM team’s construct, BBa_K274210, which encodes an operon for β-carotene production using pyruvate and glyceraldehyde-3-phosphate as substrates. Transformed cells were plated onto LB agar + CM plates and grown overnight at 37°C. Isolated colonies with yellow-orange color were then inoculated - in triplicates, into LB + CM and grown for 24 hours at 37°C while shaking, to serve as overnight cultures in subsequent experiments for growth curve generation. Likewise, untransformed DH5α E. coli cells that did not produce β-carotene were also inoculated - in triplicates, into LB+CM, to serve as a negative control.</P><P>To generate a curve for growth in minimal medium containing varying concentrations of glucose, overnight cultures of DH5α <I>E. coli</I> cells that either expressed or did not express β-carotene were inoculated to a starting optical density (OD600) of 0.01, in 100 ml of M9 minimal medium containing 100 μg/ml ampicillin, and either 0.05, 0.10 or 0.20% glucose, in triplicates. The cultures were incubated at 30°C and OD600 was subsequently measured at different times over the course of approximately 50 hours using a spectrophotometer.</P><P>To generate a curve for growth in minimal medium containing cellobiose or cellulose, overnight cultures of DH5α <I>E. coli</I> cells that either expressed or did not express β-carotene were inoculated to a starting optical density (OD600) of 0.01, in 100 ml of M9 minimal medium containing 100 μg/ml ampicillin, and either 0.20% cellobiose or 0.20% cellulose. The cultures were incubated at 30°C and OD600 was again subsequently measured at different times over the course of approximately 50 hours using a spectrophotometer.</P><H3>Characterizing β-carotene production in DH5α <I>E. coli</I> cells encoding β-carotene biosynthesis genes</H3><P>DH5α <I>E. coli</I> cells that either encode or do not encode for β-carotene biosynthetic genes were inoculated - in triplicates, into 50 ml of M2 minimal media supplemented with 100 μg/ml ampicillin and 0.20% glucose. These were grown for 48 hours at 30°C while shaking.</P><P>We then used a method adopted from the 2009 Cambridge iGEM team, for β-carotene extraction. After 48 hours, the aforementioned 50 ml culture of cells were initially pelleted at 4000 RPM using a Beckman Coulter centrifuge for 20 minutes, and the resulting supernatant was discarded. 5 ml of acetone was added to each pellet, heated for 10 minutes at 50°C, and then vortexed to lyse the cells. The samples were then centrifuged at 14,000 rpm for 1 minute, and the supernatant containing β-carotene in acetone was collected. Using a scanning spectrophotometer, absorbance at a range of wavelengths from 300-600 nm was measured for each of the acetone-extracted β-carotene samples. Absorbance at these wavelengths was also taken for laboratory grade β-carotene, which served as a positive control.</P><H3>Cloning CrtEBIY under a constitutive promoter into pSB1C3 plasmid</H3><P>All <I>Escherichia coli</I> cultures were grown in Luria Broth (LB) media supplemented with 12.5 μg/ml chloramphenicol (CM) unless stated otherwise. All plasmid DNA extractions were performed with QIAprep Spin Miniprep Kit (Qiagen). DNA purification from gels or PCR mixtures were done with NucleoSpin® Gel and PCR Clean-up kit (Macherey-Nagel).</P><P>The BBa_K274210 part was digested with EcoRI and SpeI restriction enzymes, gel purified and ligated using T4 ligase in  pSB1C3, which was also digested with EcoRI and SpeI enzymes, and gel purified. The ligation mixture was then transformed into chemically competent K12 (MG1655) and DH5α <I>E. coli</I>, and transformed cells were plated onto LB agar + CM. Several transformants were grown overnight in LB+CM, subjected to Miniprep, and extracted plasmids were sent for Sanger sequencing using the VF2 primer for confirmation of the correct insert.</P></SECTION><SECTION id="Results" class="anchor"><H2>Results</H2><H3>Growth curves for DH5α <I>E. coli</I> cells expressing β-carotene in minimal medium</H3><P>We performed growth experiments and obtained curves to determine differences in growth dynamics between DH5α <I>E. coli</I> that express or do not express β-carotene when grown in minimal media supplemented with either different concentrations of glucose, cellobiose, or cellulose. From figures 1a, 1b, and 1c, we see that β-carotene-expressing cells grow to similar optical densities as the non-β-carotene-expressing cells, indicating that the plasmid conferring β-carotene production phenotype has minimal impact on the growth rate of DH5α <I>E. coli</I>. As expected, cells grown in the presence of higher glucose concentrations also achieve higher densities. These cells initiate exponential phase at the 18 hour time point and reach stationary phase after 35 hours of growth. From figures 1d and 1e, we see that there is no appreciable growth by either β-carotene-expressing or non-β-carotene-expressing cells when grown in minimal medium containing cellulose or cellobiose. This is also expected since E. coli lacks enzymes capable of degrading cellulose or cellobiose into usable carbon sources, in the form of simple sugars. </P><P><B>Figure 1.</B> A. 0.05% glucose (yellow) with the control (blue), B. 0.05% glucose (yellow) with the control (blue), C. 0.1% glucose (orange) with the control (blue), D. 0.2% glucose (red) with the control (blue), E. 0.02% Cellulose (green) with the control (blue), F. 0.02% Cellobiose (green) with the control (blue). All controls have the same carbon content as parent flask. </P><H3>Characterization of β-carotene production in DH5α <I>E. coli</I> cells encoding β-carotene biosynthesis genes</H3><P>We were able to successfully produce and isolate β-carotene through acetone-extraction from DH5α E.coli. As seen from figure 2,  β-carotene-producing cells were able to produce a product that has peak absorbance at 452 nm, which is comparable to literature values for  β-carotene.</P><P><B>Figure 2.</B> Wavelength scan of beta carotene. Max Peak at 452nm.</P><H3>Cloning CrtEBIY under a constitutive promoter into pSB1C3 plasmid</H3><P>From Sanger sequencing results, we confirmed the presence of the CrtBEIY insert under BBa_R0011 constitutive promoter in psB1C3, and were able to obtain transformants containing this construct.  DH5α E.coli cells that now contain β-carotene biosynthetic genes and chloramphenicol resistance can be used in co-culture experiments with cellulase-expressing C. crescentus. Consortia work is elucidated on the corresponding wiki page. </P></SECTION><SECTION id="Conclusion" class="anchor"><H2>Conclusion</H2></SECTION><P>β-carotene-expressing cells grow to similar optical densities as the non-β-carotene-expressing cells, indicating that the plasmid conferring β-carotene production phenotype has minimal impact on the growth rate of DH5α <I>E. coli</I>. Furthermore,  no appreciable growth is seen when cells are grown in cellobiose or cellulose. This result is ideal for consortia work, as it implies that <I>E. coli</I> would not compete for growth with cellulase-expressing <I>C. crescentus</I> in the initial phases of our prototype bioreactor. We were able to successfully produce and isolate β-carotene through acetone-extraction from DH5α <I>E.coli</I>. We were also able to clone CrtBEIY insert under a constitutive promoter in psB1C3 and transform this into DH5α and K12 <I>E.coli</I>, for use in a consortium with C. crescentus. </P><SECTION id="References" class="anchor" style="height: 600px"><H2>References</H2><OL>Bendich, Adrianne. &quot;From 1989 To 2001: What Have We Learned About The “Biological Actions Of Beta-Carotene”?&quot;. American Society For Nutritional Sciences, vol 134, no. 1, 2004, pp. 225S-230S.</OL><OL>Rosano, Germain L. and Eduardo A. Ceccarelli. &quot;Recombinant Protein Expression In Escherichia Coli: Advances And Challenges&quot;. Front. Microbiol., vol 5, 2014, Frontiers Media SA, doi:10.3389/fmicb.2014.00172.</OL><OL>Webb, Frances Sizer and Eleanor Noss Whitney. Nutrition Concepts And Controversies. Australia, Wadsworth Thomson Learning, 2003.</OL></SECTION></DIV></DIV></DIV><P id="read-more"><STRONG>Check out other parts of our project below!</STRONG></P><DIV id="up-next"><DIV class="row" style="max-width: 100%; margin: 0"><DIV class="col-sm-6" id="previous"><A href="https://2016.igem.org/Team:British_Columbia/Project/Consortia"><STRONG><FIGCAPTION>Consortia</FIGCAPTION></STRONG></A></DIV><DIV class="col-sm-6" id="next"><A href="https://2016.igem.org/Team:British_Columbia/Model"><STRONG><FIGCAPTION>Modelling</FIGCAPTION></STRONG></A></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></BODY></HTML>