<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_British_Columbia_Project_S-Layer_Laccases skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><DIV id="top_title"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:British Columbia/Project/S-Layer/Laccases</SPAN></H1></DIV><DIV id="HQ_page"><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><P><TITLE>Main CSS</TITLE><TITLE>Navbar CSS</TITLE></P><DIV class="container-fluid"><DIV class="collapse navbar-collapse"><UL class="nav navbar-nav navbar-right"><LI class="dropdown"><A href="#" class="dropdown-toggle" data-toggle="dropdown">Project
        </A></LI><LI><A href="https://2016.igem.org/Team:British_Columbia/Description">Description</A></LI><LI class="dropdown-submenu"><A href="#" class="dropdown-toggle" data-toggle="dropdown">S-Layer</A></LI><LI><A href="https://2016.igem.org/Team:British_Columbia/Project/S-Layer/Cellulases">Cellulases</A></LI><LI><A href="https://2016.igem.org/Team:British_Columbia/Project/S-Layer/Laccases">Laccases</A></LI></UL><LI><A href="https://2016.igem.org/Team:British_Columbia/Project/Bio-Pathways">Biosynthetic Pathways</A></LI><LI><A href="https://2016.igem.org/Team:British_Columbia/Project/Consortia">Consortia</A></LI><LI><A href="https://2016.igem.org/Team:British_Columbia/Proof">Proof</A></LI><LI><A href="https://2016.igem.org/Team:British_Columbia/Model">Modelling</A></LI><LI class="dropdown"><A href="#" class="dropdown-toggle" data-toggle="dropdown">Human Practices
        </A></LI><LI><A href="https://2016.igem.org/Team:British Columbia/HP/Gold">Introduction</A></LI><LI><A href="https://2016.igem.org/Team:British_Columbia/Human_Practices/Infographic">Infographic</A></LI><LI><A href="https://2016.igem.org/Team:British_Columbia/Human_Practices/Interview">Interview</A></LI><LI class="dropdown"><A href="#" class="dropdown-toggle" data-toggle="dropdown">Team
        </A></LI><LI><A href="https://2016.igem.org/Team:British_Columbia/Team/Bios">Bios</A></LI><LI><A href="https://2016.igem.org/Team:British_Columbia/Attributions">Attributions</A></LI><LI><A href="https://2016.igem.org/Team:British_Columbia/Team/Sponsors">Sponsors</A></LI><LI><A href="https://2016.igem.org/Team:British_Columbia/Notebook">Notebook</A></LI><LI class="dropdown"><A href="#" class="dropdown-toggle" data-toggle="dropdown">Achievements
        </A></LI><LI><A href="https://2016.igem.org/Team:British_Columbia/Collaborations">Collaborations</A></LI><LI><A href="https://2016.igem.org/Team:British_Columbia/Achievements/Medal_Criteria">Medal Criteria</A></LI><LI class="dropdown-submenu pull-left"><A href="#" class="dropdown-toggle" data-toggle="dropdown">Parts</A></LI><LI><A href="https://2016.igem.org/Team:British_Columbia/Parts">Parts</A></LI><LI><A href="https://2016.igem.org/Team:British_Columbia/Basic_Part">Basic Part</A></LI><LI><A href="https://2016.igem.org/Team:British_Columbia/Composite_Part">Composite Part</A></LI><LI><A href="https://2016.igem.org/Team:British_Columbia/Part_Collection">Parts Collection</A></LI></DIV></DIV><NAV class="navbar navbar-clear navbar-fixed-top"><DIV class="container-fluid"><DIV class="collapse navbar-collapse"><UL class="nav navbar-nav navbar-right"><LI class="dropdown"><A href="#" class="dropdown-toggle" data-toggle="dropdown">Project
        </A></LI><LI><A href="https://2016.igem.org/Team:British_Columbia/Description">Description</A></LI><LI class="dropdown-submenu"><A href="#" class="dropdown-toggle" data-toggle="dropdown">S-Layer</A></LI><LI><A href="https://2016.igem.org/Team:British_Columbia/Project/S-Layer/Cellulases">Cellulases</A></LI><LI><A href="https://2016.igem.org/Team:British_Columbia/Project/S-Layer/Laccases">Laccases</A></LI></UL><LI><A href="https://2016.igem.org/Team:British_Columbia/Project/Bio-Pathways">Biosynthetic Pathways</A></LI><LI><A href="https://2016.igem.org/Team:British_Columbia/Project/Consortia">Consortia</A></LI><LI><A href="https://2016.igem.org/Team:British_Columbia/Proof">Proof</A></LI><LI><A href="https://2016.igem.org/Team:British_Columbia/Model">Modelling</A></LI><LI class="dropdown"><A href="#" class="dropdown-toggle" data-toggle="dropdown">Human Practices
        </A></LI><LI><A href="https://2016.igem.org/Team:British Columbia/HP/Gold">Introduction</A></LI><LI><A href="https://2016.igem.org/Team:British_Columbia/Human_Practices/Infographic">Infographic</A></LI><LI><A href="https://2016.igem.org/Team:British_Columbia/Human_Practices/Interview">Interview</A></LI><LI class="dropdown"><A href="#" class="dropdown-toggle" data-toggle="dropdown">Team
        </A></LI><LI><A href="https://2016.igem.org/Team:British_Columbia/Team/Bios">Bios</A></LI><LI><A href="https://2016.igem.org/Team:British_Columbia/Attributions">Attributions</A></LI><LI><A href="https://2016.igem.org/Team:British_Columbia/Team/Sponsors">Sponsors</A></LI><LI><A href="https://2016.igem.org/Team:British_Columbia/Notebook">Notebook</A></LI><LI class="dropdown"><A href="#" class="dropdown-toggle" data-toggle="dropdown">Achievements
        </A></LI><LI><A href="https://2016.igem.org/Team:British_Columbia/Collaborations">Collaborations</A></LI><LI><A href="https://2016.igem.org/Team:British_Columbia/Achievements/Medal_Criteria">Medal Criteria</A></LI><LI class="dropdown-submenu pull-left"><A href="#" class="dropdown-toggle" data-toggle="dropdown">Parts</A></LI><LI><A href="https://2016.igem.org/Team:British_Columbia/Parts">Parts</A></LI><LI><A href="https://2016.igem.org/Team:British_Columbia/Basic_Part">Basic Part</A></LI><LI><A href="https://2016.igem.org/Team:British_Columbia/Composite_Part">Composite Part</A></LI><LI><A href="https://2016.igem.org/Team:British_Columbia/Part_Collection">Parts Collection</A></LI></DIV></DIV></NAV><TITLE>Sidebar CSS</TITLE><P><TITLE>Laccases</TITLE></P><DIV class="cover"><DIV class="container-fluid"><DIV class="row" id="title"><DIV class="col-sm-12"><STRONG><P style="font-size: 3em">Laccase S-Layer Engineering</P></STRONG></DIV></DIV></DIV></DIV><DIV id="breadcrumbs"><STRONG><A href="https://2016.igem.org/Team:British_Columbia">Home</A> /
<A href="https://2016.igem.org/Team:British_Columbia/Project/S-Layer/Laccases">Project - Laccase S-Layer Engineering</A></STRONG></DIV><DIV class="content-wrap"><DIV class="row"><NAV class="col-sm-3" id="side-menu"><UL class="nav sidebar-nav nav-pills nav-stacked"><LI class="active"><A href="#Abstract">Abstract</A></LI><LI><A href="#Key-Achievements">Key Achievements</A></LI><LI><A href="#Introduction">Introduction</A></LI><LI><A href="#Methods">Methods</A></LI><LI><A href="#Results">Results</A></LI><LI><A href="#Conclusion">Conclusion</A></LI><LI><A href="#References">References</A></LI></UL></NAV><DIV class="col-sm-9"><SECTION id="Abstract" class="anchor"><H2>Abstract</H2><P> Lignin is a complex heteropolymer comprised of aromatic units linked to each other through a variety of bonds.This polymer combines with cellulosic fibers to form a highly recalcitrant structure, which limits the
the accessibility of cellulose to cellulolytic enzymes. For this part of our project, we attempted to engineer expression of lignin-modifying enzymes onto the <I>C. crescentus</I> S-layer as one approach to pretreat raw biomass and make it more accessible to cellulases. We expect that this &quot;delignification&quot; step will enhance hydrolysis rate and overall sugar yield. </P></SECTION><SECTION id="Key-Achievements" style="font-size: 16px" class="anchor"><H2>Key Achievements</H2><P><LI>Cloned small laccases (sLAC) in p4A723 plasmid containing rsaA protein and transformed into <I>C. crescentus</I> for display on cell surface.</LI><LI>Determined the concentration of copper that is toxic to <I>C. crescentus</I>.</LI><LI>Performed ABTS assay to test laccase activity of sLAC expressed on the <I>Caulobacter</I> cell surface.</LI></P></SECTION><SECTION id="Introduction" class="anchor"><H2>Introduction</H2><P>Current pretreatment and hydrolysis approaches to biomass depolymerization involve thermochemical methods that apply alkaline chemicals, in addition to high heat and pressure (Chundawat S. 2011). These methods enhance the fractionation of lignin and hemicellulose components from the plant cell wall, thereby allowing enzymes access to the cellulosic fraction for downstream processing. Although these methods are effective, their high cost to industries and to the environment requires that alternative methods to remove the more recalcitrant components of lignocellulosic biomass, are erected. We intend to produce a system that minimally relies on external thermochemical treatments to make the cellulosic component of biomass available for valuable-chemical production. To achieve this, we attempted to engineer surface layer expression of a small laccase (sLAC) derived from <I>Amycolatopsis sp.</I> 75iv2 shown to act on and degrade lignin (Majumdar S. 2014), in  <I>C. crescentus</I>.</P><P>Laccases belong to a superfamily of enzymes called multicopper oxidases (MCO), and are expressed by several rot-fungi and soil bacteria (Sirim D. 2011). As an MCO, they catalyze the one-electron oxidation of substrates through associated four-electron reductions of molecular oxygen to water, using four copper ions coordinated in designated copper centers (Jones Solomon E. 2015, Kunamneni A. 2007). Their broad phenolic and polyphenolic substrate specificity allows for direct transformation of lignin, or removal of toxic phenols that arise during lignocellulosic biomass pre-treatment (Kunamneni A. 2007). </P><P>
In this part of our project, we specifically aimed to fuse sLAC to the RsaA protein for display on the <I>C. crescentus</I> cell surface. We anticipate that the use of such laccase-RsaA fusion proteins expressed on the cell surface would contribute to raw biomass pretreatment, and perhaps alleviate the costs associated by thermochemical methods, by degrading the lignin component and allowing cellulases access to the cellulosic fraction of the lignocellulosic material.</P></SECTION><SECTION id="Methods" class="anchor"><H2>Methods</H2><P>
                        All <I>C. crescentus</I> cultures were grown in PYE media supplemented with 2 μg/ml chloramphenicol unless stated otherwise. All plasmid DNA extractions were performed with QIAprep Spin Miniprep Kit
                        (Qiagen). DNA purification from gels or PCR mixtures were done with NucleoSpin® Gel and PCR
                        Clean-up kit (Macherey-Nagel).</P><H3>Cloning of gene encoding the small laccase enzyme into RsaA plasmid and expression in <I>C. crescentus</I></H3><P> The small laccase gene from <I>Amycolatopsis sp.</I> 75iv2 ATCC 39116 was amplified using the following primers: sLAC_pSB1C3_F (5'-TCCgaattcgcggccgcttctagATGCAGGGCACGACCCGG-3') and sLAC_pSB1C3_R (5'-TCCtactagtagcggccgctgcagTCAGTGTTCGTGGACACC-3') to clone the construct in pSB1C3. The amplicon was digested with EcoRI and PstI, PCR purified and ligated into a pSB1C3, which was also digested with EcoRI and PstI and gel purified. Site-directed mutagenesis (SDM) was done to remove any forbidden cut-sites using a protocol outlined in the Nobebook section, and the following primers: SDM_sLAC_F (5'-GTACGCCGAGAAGATTTCCGACGAGCTGTAC-3') and SDM_sLAC_R(5'-GTACAGCTCGTCGGAAATCTTCTCGGCGTAC-3'). After the mutation was sequence confirmed, a select region of laccase gene was amplified to add the BglII and PstI cut sites using the following primers: (5'-GTTTCTTCAGATCTACGACCCGGCGGATCACG-3') and (5'-GTTTCTTCCTGCAGAGACACCGGCCGGCATCGT-3') . The sLAC amplicon was digested with BglII and PstI restriction enzymes, PCR purified and ligated into a p4A723 RsaA plasmid conferring chloramphenicol (CM) resistance. The ligation mix was transformed into chemically competent DH5α E. coli cells and transformed cells were plated onto LB + CM (12.5 µg/mL). Insertion of sLAC into p4A723 was preliminarily confirmed by colony PCR of selected transformants, using the aforementioned primers for sLAC, and checking for appropriately sized bands (850 bp). The selected colonies were sent for Sanger sequencing using the IRAT_F(5'-CGGAGCCGCCAGAACGGTCAGGCCGACATTCAC-3')  for confirmation.
After sequence confirmation, the isolated construct DNA were electroporated into electrocompetent <I>C.crescentus</I> with an RsaA Amber mutation and colonies were grown on a PYE-CM plate. One colony was selected and streaked onto a fresh plate. Colonies that spawn from this transformant would be used in all future assays.</P><H3>
                        Cloning of the small laccase gene into a pSB1C3 plasmid with pTAC promoter and rbs
                    </H3><P>
                        The laccase gene was amplified with sLAC_pSB1C3_F and sLAC_pSB1C3_R and digested using EcoRI and XbaI restriction enzymes. pTAC and rbs were amplified and digested as described in the Methods section for cellulases cloning.
 The plasmid digest was then purified by agarose gel purification while the pTAC RBS digest was purified by PCR
                        purification. Both purified digests were ligated together using standard ligation protocol and the ligation mix was then transformed into chemically competent DH5α <I>E. coli</I> cells.
</P><H3>Copper toxicity test</H3><P><I>C. crescentus</I> was inoculated - in triplicates, in PYE media supplemented with varying concentrations (0 to 10mM) of CuSO4. The cultures were incubated at 30C for 72 hours, while shaking. The OD600 was measured to determine concentrations of CuSO4 that would inhibit <I>Caulobacter</I> growth.

</P><H3> ABTS Assay to characterize sLAC activity</H3><P><I>C. crescentus</I> encoding sLAC in p4A723 and just p4A723 (as a negative control) were inoculated into flasks containing 15 mL cultures of PYE in the presence or absence of 300 µM CuSO4. The cultures were grown for 3 days, OD600 was normalized and ABTS assay was performed in 96-well plates. The protocols were derived from More et al(2011). Specifically, 176 µl of Sodium Acetate (pH 5), 10 µl of a culture, 4 µl of 0.5 M CuSO4 (10 mM CuSO4 final concentration) and 10 µl of 10 mM ABTS (0.5 mM ABTS final concentration) were added to each well. The plate was then incubated at 30C for an hour and absorbance at 420nm was measured using a Varioskan plate reader. 


</P></SECTION><SECTION id="Results" class="anchor"><H2>Results</H2><P> The small laccase (sLAC) was selected as a candidate to be expressed on the S-layer due to it small size (276aa). The region of 4-272aa was successfully cloned in p4A723 plasmid and transformed into <I>C. crescentus</I>. </P><P>Because sLAC requires CuSO4 for proper folding of its catalytic site (Majumdar S. 2014), we performed a CuSO4 toxicity test on p4A723 <I>Caulobacter</I> to determine the maximum concentration of copper that the bacterium could tolerate. First, we did a test with a wide range of copper concentrations. After 2 days of incubation we determined that the growth of <I>C. crescentus </I> falls off steeply when grown in the presence 0.5 mM of CuSO4 (Fig. 1A). Next, we performed the same assay with a more narrow range of copper concentrations to determine whether the bacterium would grow in the presence of between 0.1 mM to 0.5 mM CuSO4. For this assay, we inoculated <I>C. crescentus</I> expressing sLAC on its surface. From the growth results, we determined that 300 µM is the maximum concentration of CuSO4 that the cells could tolerate. Thus, we used this concentration for our future experiments(Fig. 1B). </P><P><B>Figure 1.</B> Copper toxicity test. A) Growth measurements of p4A723 <I>Caulobacter</I> cultures grown for 3 days - in triplicates, in PYE supplemented with 0, 0.0001, 0.001, 0.01, 0.5, 1 and 10mM CuSO4. B) Growth measurements of sLac <I>Caulobacter</I> in PYE supplemented with 0.1, 0.2, 0.3, 0.4, 0.5 mM CuSo4 to more precisely determine toxic concentration of copper.</P><P> To measure laccase activity on the surface of <I>C. crescentus</I>, we performed ABTS assay. The assay was performed in the presence and absence of copper. The sLAC-expressing <I>C. crescentus</I> demonstratated slightly higher activity than the control. Additional tests are required to confirm proper folding and activity of the sLAC enzyme.
</P><P><B>Figure 2.</B>Results of ABTS assay on sLac and p4A723 cultures. The test was performed with and without addition of copper.


  </P></SECTION><SECTION id="Conclusion" class="anchor"><H2>Conclusion</H2><P>Display of lignin-modifying enzymes on the cell surface is inherently more difficult than the display of cellulases due to their co-factor dependencies. Also, most laccases are homodimers or homotrimers which makes proper orientation and folding on the cell surface even more challenging. The sLAC we used in our study is a homotrimer, so its fusion to rsaA might not lead to a completely functional enzyme. Selecting smaller single-chain laccases is necessary to validate the approach and confirm lignin transformation activities leading to enhanced hydrolysis rate in our system.</P></SECTION><SECTION id="References" class="anchor"><H2>References</H2><OL>Chundawat S, Beckham G, Himmel M, Dale B. 2011. Deconstruction of Lignocellulosic Biomass to Fuels and Chemicals. Annual Review of Chemical and Biomolecular Engineering 2:121-145.</OL><OL>Sirim D, Wagner F, Wang L, Schmid R, Pleiss J. 2011. The Laccase Engineering Database: a classification and analysis system for laccases and related multicopper oxidases. Database 2011:bar006.</OL><OL>Jones Solomon E. 2015. Electron transfer and reaction mechanism of laccases. Cell. Mol. Life Sci. 72:869-883.</OL><OL>Reiss R, Ihssen J, Richter M, Eichhorn E, Schilling B, Thöny-Meyer L. 2013. Laccase versus Laccase-Like Multi-Copper Oxidase: A Comparative Study of Similar Enzymes with Diverse Substrate Spectra. PLoS ONE 8:e65633.</OL><OL>Kunamneni A, Ballesteros A, Plou FJ, Alcalde M. 2007. Fungal laccase – a versatile enzyme for biotechnological applications, p 233-245. In Mendez-Vilas A (ed). Communicating current research and educational topics and trends in applied microbiology. FORMATEX, Badajoz.</OL><OL>Majumdar S, Lukk T, Solbiati J, Bauer S, Nair S, Cronan J, Gerlt J. 2014. Roles of Small Laccases from <I>Streptomyces</I> in Lignin Degradation. Biochemistry 53:4047-4058.</OL><OL>More S, Malini S. 2011. Isolation, Purification, and Characterization of Fungal Laccase from <I>Pleurotus sp.</I> Enzyme Research 2011:1-7.</OL></SECTION></DIV></DIV></DIV><P id="read-more"><STRONG>Check out other parts of our project below!</STRONG></P><DIV id="up-next"><DIV class="row" style="max-width: 100%; margin: 0"><DIV class="col-sm-6" id="previous"><A href="https://2016.igem.org/Team:British_Columbia/Project/Bio-Pathways"><STRONG><FIGCAPTION>Biosynthetic Pathways</FIGCAPTION></STRONG></A></DIV><DIV class="col-sm-6" id="next"><A href="https://2016.igem.org/Team:British_Columbia/Project/Consortia"><STRONG><FIGCAPTION>Consortia</FIGCAPTION></STRONG></A></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></BODY></HTML>