<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_IIT_Delhi_Notebook skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><DIV id="top_title"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:IIT Delhi/Notebook</SPAN></H1></DIV><DIV id="HQ_page"><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><DIV class="menu_wrapper"><DIV class="collapsable_menu_control"> MENU ▤ </DIV><UL id="accordion" class="accordion"><LI class="menu_item"><A href="https://2016.igem.org/Team:IIT_Delhi">HOME </A></LI><LI class="menu_item"> TEAM
			</LI><LI><A href=" https://2016.igem.org/Team:IIT_Delhi/Team"> Team   </A></LI><LI><A href="https://2016.igem.org/Team:IIT_Delhi/Collaborations">★  Collaborations </A></LI><LI><A href="https://2016.igem.org/Team:IIT_Delhi/Attributes">★  Attributes </A></LI></UL><LI class="menu_item"><A href="https://2016.igem.org/Team:IIT_Delhi/Abstract">ABSTRACT </A></LI><LI class="menu_item"> PROJECT  
			</LI><LI><A href="https://2016.igem.org/Team:IIT_Delhi/Motivation"> ★  Motivation </A></LI><LI><A href="https://2016.igem.org/Team:IIT_Delhi/Cicuits"> ★ Oscillator Circuits </A></LI><LI><A href="https://2016.igem.org/Team:IIT_Delhi/Oscillator">★ The Danino Oscillator </A></LI><LI><A href="https://2016.igem.org/Team:IIT_Delhi/Modules"> ★ Our Modules </A></LI><LI><A href="https://2016.igem.org/Team:IIT_Delhi/Characterization"> ★ Characterization on Experimental Design </A></LI><LI class="menu_item"><A href="https://2016.igem.org/Team:IIT_Delhi/Modeling">MODELING</A></LI><LI class="menu_item"> PARTS  
			</LI><LI><A href="https://2016.igem.org/Team:IIT_Delhi/Parts">Parts </A></LI><LI><A href="https://2016.igem.org/Team:IIT_Delhi/Basic_Part"> ★ Basic Parts </A></LI><LI><A href="https://2016.igem.org/Team:IIT_Delhi/Composite_Part"> ★ Composite Parts </A></LI><LI><A href="https://2016.igem.org/Team:IIT_Delhi/Part_Collection"> ★ Part Collection </A></LI><LI class="menu_item"> RESULTS  
			</LI><LI><A href="https://2016.igem.org/Team:IIT_Delhi/Cloning">Cloning </A></LI><LI><A href="https://2016.igem.org/Team:IIT_Delhi/SDS_Page"> ★ SDS PAGE</A></LI><LI><A href="https://2016.igem.org/Team:IIT_Delhi/Florimetry"> ★ Florimetry </A></LI><LI><A href="https://2016.igem.org/Team:IIT_Delhi/Microfluidics"> ★ Microfluidics </A></LI><LI class="menu_item"><A href="https://2016.igem.org/Team:IIT_Delhi/Notebook">NOTEBOOK </A></LI><LI class="menu_item"> HUMAN PRACTICES  
			</LI><LI><A href="https://2016.igem.org/Team:IIT_Delhi/Workshop">★ SynBio Workshop </A></LI><LI><A href="https://2016.igem.org/Team:IIT_Delhi/SynTech"> ★ SynTech</A></LI><LI><A href="https://2016.igem.org/Team:IIT_Delhi/interactions"> ★ High School Interactions </A></LI><LI class="menu_item"><A href="https://2016.igem.org/Team:IIT_Delhi/References">REFERENCES </A></LI></DIV><DIV class="content_wrapper"><DIV class="column full_size"><H1>30th may to 7th june </H1><P> Discussion on our project and the proposal sent to iBEC (Indian biological engineering competition). </P><H2>31/05/16 </H2><P> Transformation of 4 plasmids using electroporation (Only 2microL of each part was used): </P><OL><LI>2H6 - BBa_R0062 - PLux</LI><LI>5D7 - BBa_K1399002 - RFP</LI><LI>2L4 - BBa_C0062 - LuxR</LI><LI>2J4 - BBa_C0061 - LuxI</LI></OL><P> 
Due to insufficient cell growth on 2 of the 4 plates transformed only 2 will be inoculated and the other 2 have to be redone. </P><P>This implies inadequate amount of plasmids were transformed.
For further processes amount of plasmid used would be around 5-6 microL</P><P> 
Computation:
Modelling of the Danino oscillator for a single cell.</P><H2>02/06/16</H2><P>Plasmid isolation of the 2 cell cultures. </P><H2>04/06/16</H2><P>Transformation using TSS protocol of '2 new plasmids' and '2 earlier used plasmids (insufficient cell growth)' respectively:</P><OL><LI>2L4 - BBa_C0062 - LuxR</LI><LI>2J4 - BBa_C0061 - LuxI</LI><LI>2H4 - BBa_C0060 - aiiA with degradation tag</LI><LI>2B4 - BBa_C0051 - lambda cI</LI></OL><H2>05/06/16</H2><P> Computation:
Discussion on Second order partial derivatives in delay differential equations. </P><P>
Lab:
All 4 plates show colonies, although on one of the plates (aiiA with degradation tag part), one of the colonies does not look like a typical E.coli colony, and is transparent instead of being opaque. Anyway, we inoculated colonies from all 4 plates into LB containing tubes. 
Hopefully all 4 will show growth tomorrow. That would be confirmation enough whether the colony is proper or not.
Also, we found the P+RBS (constitutive) from last year. We’ll transform that tomorrow to save it and then use it for cloning in our project.
We have also isolated a transcriptional terminator for our parts, which we will pick up from the kits tomorrow and use. </P><H2>06/06/16</H2><P><UL>
Transformation of 4 new plasmids:
1.	<LI>2L6 - BBa_R0065 - lambda cI and luxR</LI>
2.	<LI>3F3 - Bba_B0015 - Terminator</LI>
3.	<LI>P+RBS - J23119 and RBS</LI>
4.	<LI>5H1 - BBa_B0030 - RBS</LI></UL></P><P>
5.			
 			

</P><H1>From 8th june to 15th june </H1><H2>08/06/16</H2><OL>

Inoculation of 3 cell colonies having the following gene plasmids:
 
<LI>2L6 - BBa_R0065 - lambda cI and luxR promotor</LI><LI>3F3 - Bba_B0015 - Terminator</LI><LI>P+RBS - J23119 and RBS</LI></OL><P>The culture with the 4th plasmid didn't have sufficient growth. It has to be redone. </P><H2>09/06/16</H2>
 
Plasmid isolation of the following genes:
<OL><LI>2L6 - BBa_R0065 - λcI and luxR promotor</LI><LI>3F3 - Bba_B0015 - Terminator</LI><LI>P+RBS - J23119 and RBS</LI></OL><H2>10/06/16</H2><P>Gel run-</P><P>1% gel (Agarose in TAE) </P><P>Ethidium Bromide as staining dye for UV visibility</P><P>3 uL plasmid + 0.6 uL loading buffer (glycerol) </P><UL>
Plasmid + Backbone = Total size
1st row:                                                   
Ladder (1kb)
<LI>5D7 - BBa_K1399002 - RFP  </LI><LI>714 bp + 2070 bp   =  2784 bp</LI>
2B4 - BBa_C0051 - lambda cI    <LI>775 bp + 2070 bp   =  2845 bp</LI><LI>3F3 - Bba_B0015 - Terminator   </LI><LI>129 bp + 2070 bp   =  2199 bp</LI><LI>2H6 - BBa_R0062 - PLuxR         </LI><LI>55 bp + 2070 bp   =  2125 bp</LI><LI>2H4 - BBa_C0060 - aiiA           </LI><LI>814 bp + 2070 bp   =  2884 bp</LI><LI>2L6 - BBa_R0065 - P lux-λ       </LI><LI>97 bp + 2070 bp   =  2167 bp</LI><LI>2J4 - BBa_C0061 - LuxI              </LI><LI>643 bp + 2070 bp   =  2713 bp</LI>    
             
2nd row:
<LI>2L4 - BBa_C0062 - LuxR   </LI><LI>781 bp + 2070 bp   =  2851 bp</LI><LI>Bba_K880005 - J23119 and RBS   </LI><LI>55 bp + 2070 bp   =  2125 bp</LI><LI>PSB1C3 (Backbone) size:  2070 bp </LI></UL><P>All the plasmids except double terminator and LuxI showed results. This maybe because the colonies obtained of the double terminator were of green colour and the concentration of LuxI plasmid was too low.  </P><P>Digestion of all the plasmids was done except the double terminator. </P><H2>16/06/16</H2>
 
Transformation of the following plasmids:
 <OL><LI>Strong RBS	BBa_B0030	PSB1C3	Plasmid Present
</LI><LI>Double Terminator	BBa_B0015	PSB1C3	2015 Kit Plate 3</LI><LI>LuxR gene	BBa_C0062	PSB1C3	2015 Kit plate 2</LI><LI>Double Terminator	BBa_B0015	PSB1C3	2014 Kit Plate 3</LI><LI>LuxR gene	BBa_C0062	PSB1A2	2013 Kit Plate 5</LI>
Isolation of the 4 plasmids with their concentrations and absorbance: 
Gene          Concentration   A260/230   A260/280
Plux              108.2 ng/uL         1.95              2.24
Heme           120.6 ng/uL         1.90              2.06
CCas Ccar    165.8 ng/uL         1.94              2.28 
Pcpcg2         286.4 ng/uL         1.92              2.22
 
LuxI was also reisolated with new concentration 369 ng/uL.

<H2>18/06/16</H2>
 
Plates of the transformed cell culture have the following growth:<UL><LI>Strong RBS:             1000 colonies (lawn) </LI><LI>5O4  LuxR(2013):    No growth as expected (Ampicillin resistance) </LI><LI>2L4   LuxR(2105):    20 colonies</LI><LI>3F3  Double Term: 100 colonies, slightly better than last time (not green) </LI><LI>3F3  Double Term: Colonies there but medium ruptured</LI></UL><H2>20/06/16</H2>
 
Important discussion on the project.<P>The process of producing a cloned gene in a cell culture starting from the kit plates provided by iGEM was discussed in detail. Every possible limitation and benefit of the processes came up during the discussion. This gave us a clearer view of how and why theses steps were performed and also how to overcome a limitation or problem that was encountered in them. </P>
Then we discussed our progress till now and planned our tasks for the next few days.

<H2>21/06/16</H2>
 
 
Digestion of PSB1K3 and PSB1A3
 
Gel run: <UL>
•	<LI>Gel contents:  1% Agarose in TAE(100mL) and Ethidium Bromide(7.5uL)
 
•	</LI><LI>Loading dye 5X:   1uL dye + 4uL plasmid
 
•	</LI><LI>PLux                          55    </LI>
•	<LI>PLux-λ                    97</LI>
•	<LI>LuxI                         643</LI>
•	<LI>aiiA                          814</LI>
•	<LI>RFP                          714 </LI>
•	<LI>P+RBS                      55</LI>
•	<LI>Ladder                     1kb</LI>
•	<LI>RBS                          15</LI>
•	<LI>RBS                          15  </LI>
•	<LI>Terminator             129</LI>
•	<LI>Terminator             129</LI>
•<LI>Terminator             129</LI>
•	<LI>RFP                           714</LI></UL>
1)Promoter very faint band
2)backbone very faint( loading problem) <H2>22/06/16 </H2>
Calculation of amounts of plasmid content. 
 
 
Ligation of 12 plasmids in pairs as follows: 
PLux +  RBS
LuxI + T
J23104&amp;RBS + RFP	
Plux-λ + RBS
RFP + T
aiiA + T<H2>23/04/16</H2>
 
Made chloramphenicol
TAE(50X) and MilliQ prepared
Autoclaved apparatus and LA 
Transformation of the 6 cloned plasmids using TSS method. 
10 Amp, 11 Kana, 12 Cat plates prepared. 
Spreading of the 6 cultures. 
Plates kept in shaker at 37C for 16 hrs. <H2>24/06/16</H2>
 
3 colonies of each of the 6 plates inoculated to 18 tubes.   
Streaking of 3 samples (Kanamycin) to neomycin plates to check
if Kanamycin is working fine or not. 
50 colonies of J23104 + RFP streaked for confirmation of RFP. <H2>25/06/16</H2>
 
Plasmid isolation from the 18 tubes. 
After the isolation, 2 vials were removed due to loss of plasmid content
from them, namely aiiA + T1 (7th vial) and aiiA + T3 (18th vial), <H2>26/06/16</H2>

 
Digestion of the 6 clones from the 16 tubes. 
Gel run was done as follows: 
 
Gel run 3% for 15 min: 
 
1.	Empty
2.	PLux + RBS-1     (11) 
3.	PLux-λ + RBS-1 (12) 
4.	100 bp ladder
5.	PLux-λ + RBS-2 (14) 
6.	PLux-λ + RBS-3 (15) 
7.	50 bp ladder
8.	PLux + RBS-2     (16) 
9.	PLux + RBS-3     (17) 
10.	Empty
 
 
3% final
•	Loaded the samples (around 15uL) 
•	While loading, sample 12 (PLux-•	 			

<H2>27/06/16</H2>
 
Plasmid isolation of new aiiA + T
Digestion of aiiA + T and LuxR
 
Gel run 1%:
1.	LuxR c(undigested) 
2.	LuxR                             
3.	Ladder 
4.	aiiA + T
5.	aiiA + T c(aiiA) 
 
Gel run 3%
3.	T
4.	ladder
5.	T control<H2>29/06/16</H2>
 
Plasmid isolation of the following 8 genes: 
LuxI
Terminator(x2) 
PLux + RBS
PLux-λ + RBS
RFP + T
J23104-RBS
 
Gel run 1%
•	PLux + RBS and PLux-λ + RBS confirmed
•	LuxI showing a band of 9-10 kb
•	Terminator band not visible due to low concentration
 
Inoculation of 3 new colonies each of Terminator and LuxI <H2>30/06/16</H2>
 
Plasmid Isolation of T and LuxI
 
Gel run 1%
•	Terminator isolated from the kit was confirmed. 
•	All 3 LuxI gave a band of 8-10 kb. So LuxI is still unconfirmed.  
 
Inoculation of 4 colonies of aiiA (3 transparent + 1 white) 
 
Transformation of 2016 parts <H2>1/07/16</H2>
 
Culture of aiiA picked up from the white colony was of green colour. 
So aiiA was isolated from the remaining 3 cultures in a single falcon. 
 
Digestion of T, aiiA and PSB1A3
 
Gel run 1%
1.	T digested
2.	T control
3.	Ladder
4.	aiiA digested
5.	aiiA control
6.	PSB1A3
 
All 3 confirmed although band of aiiA was extremely faint.  
Ligation: aiiA + T
 
Inoculation of all 2016 parts: 
LuxI
RBS + LuxI + T
LuxR
aiiA
RBS + aiiA + T
PLux-λ + RBS
Terminator<H2>2/07/16</H2>
Isolation of 2016 parts(7) 
 
Gel run 1% of all the 2016 parts
•	LuxI and Lux R showing band of 9-10 kb (unconfirmed) 
•	RBS + LuxI + T , RBS + aiiA + T , aiiA  and  PLux-λ + RBS  confirmed! 
 
Transformation: aiiA + T<H2>3/07/16</H2>
 
Digestion of RBS + LuxI + T , RBS + aiiA + T , PLux  and  PLux-λ

<H2>4/07/16</H2>
 
Plasmid isolation of 2 PLux: small and big colonies
 
Nanodrop: 
PLux small - 243 and 2.00 , 2.10 ratios
PLux big -     311 and 1.99 , 2.19 ratios
 
Digestion of RBS + LuxI + T , RBS + aiiA + T , PLux(x2)  and  PLux-λ
 
Gel run 1%<H2>2.07.2016</H2>
•	Firstly, we did plasmid isolation ( 2 manually, 1kit) which was followed by Gel Run. 
•	Then we set up the digestion for 3 hours and then did the gel run for the digested plasmids. 
•	At the end of the day, we did ligation of the digested products. <H3>Plasmid Isolation </H3>

Plasmid	|Manual	|Kit(Concentration) 
PL+RBS|	0|	1(210) 
T(old)	|2	|1(700) 
RFP	|2	|1(222) 
RFP+T 	|2	|1(153) <H3>Gel Run</H3>
1.PL+RBS	
2.T(K) 
3.T(M1)	
4.T(M2)	
5.Ladder	
6.RFP+T(K)	
7..RFP+T(M1)	
8..RFP+T(M2)	
9.RFP(K)		
10.RFP(M1)	
11.RFP(M2)	
PL+RBS: confirmed
T(old): confirmed
RFP: confirmed
Pl=RFP+T: not confirmed<H3>Digestion</H3>
The total reaction volume was 20µl. 

Plasmid	Amount added in digestion (µl)
PL+RBS (E/S)	7.5
T(X/P)	2.3
T(E/S)	2.3
RFP (E/S)	7.5
RFP (X/P)	7.5
RFP+T (X/P)	10
J23104+RBS (X/P)	7.5<H3>Gel Run</H3>

Well 1		Confirmed.
Well 2		Confirmed.
3		Confirmed.
4		Confirmed.
5		Confirmed.
6		Saw its backbone.
7		ladder.
8		Saw its backbone.
9		Saw its backbone.
10		Saw its backbone.
11		Confirmed.
12		Confirmed.
13		Confirmed.<H3>Ligation</H3>
Total reaction volume was 30 µl and vector added was 2 µl.

Ligation Plasmid	Volume added.
(RFP) + (T) [kana]	3.2+2.6.
(PL+RBS)+(RFP+T) [chloram]	Not done because we did not have chloramphenicol.
(PL+RBS)+RFP [amp]	2.8+3.2.
T+(J23104+RBS) [amp]	2.6+2.3.<H2>4.07.2016</H2><H3>Digestions</H3>
Total reaction volume is 20µl.

Plasmid	|Amount added in digestion (µl) .
PL+RBS (E/S)	|3.5.
T(X/P)	|2.3.
T(E/S)	|2.3.
Chloramphenicol(E/P)|	10.
Ampecillin(E/P)	|10.<H3>LIGATION</H3>

Plamids	|Volume added.
(PL+RBS)+(RFP+T) [chloram] | 	2.8+3.6.<H2>5.07.2016</H2><H3>TRANFORMATION</H3>

Terminator+ (J23104 +RBS)	15 µL AMP
RFP+Terminator	15 µL KANA
(PLuxL+rbs)+RFP	15 µL AMP
Aiia+Terminator	15 µL AMP
(PLuxL+RBS)+(RFP+Terminator)	15 µL CAT
(Promoter + RBS+LuxI)+Terminator	15 µL AMP
(Promoter + LuxR+ Aiia)+Terminator	15 µL AMP	
17d	AMP
              17p	AMP
We saw lawns in all the amp plates. There is contamination in the plates. It could either be yeast .

<H1>WEEK – 7</H1>

•	Transformation – TSS
•	RFP(E/S-PM-Eluted)+T+psb1k3==&gt;Clone
•	T+psb1k3 ,transformation completed
•	Transformation results 
•	We observed 2 colonies in clone.RFP(E/S-PM-Eluted)+T+psb1k3.RTK3.1
•	No colonies were in negative control (without RFP) 

•	plasmid isolation of RTK3.1
•	Observed late growth in RTK3.1-2
•	 Digestion with ecor1
•	PRL(part) 

•	Plasmid isolation:- 
•	Vigoruous lysis==&gt; saw genomic DNA too
•	PCR

•	PRRT(part) <H2>24/7 </H2>
•	Plasmid isolation:- 
•	Vigoruous lysis==&gt; saw genomic DNA too
•	PCR

•	Gel run results:- 
•	Pst1 not working properly- PRL lot of undigested is present. 
•	PluxL+RBS digested is perfect. Saw a band below 250bp around 113bp. 
•	A lot of single digested is left after pst1 digestion. Xba1 not working properly after Pst1. 

•	Ligation  of RFP+T 
•	Ligation of RFP+T(psb1c3)+psb1k3
•	Digestion of psb1K3

•	PCR by shashank 
•	Reaction of (R+luxR+T) 
•	Remarks :- saw very little on gel. 

•	Digestion of psB1A3 and psB1T3
•	Psb1A3
•	Plasmid isolation through kit 
•	Gel of plasmid = 3ul (very intense bands)l
•	Digestion of j23100<H1>6/8/16 to 14/8/16</H1>

PLuxL+RBS<B>Plasmid isolation</B>
  
Nano drop reading ==&gt; 174.3ng/ul
260/280=1.99
260/230=2.09
Gel run of plasmid  
 
Digestion = 50ul
Plasmid = 20ul = 3.4ug
Enzymes -E/S=1ul glycerol = 1%
DNA conc. = 70ng/ul  (standard NEB = 50ng/ul)  

RFP+T <B>Plasmid isolation</B>
 
 
 
Nano drop reading ==&gt; 
260/280
260/230
280.6ng/ul
1.94 
2.23<B>Gel eluted </B>
Psb1k3 
<B>Ligation</B>
PluxL
1ul = 31mol
Ratios:2
Rfp+t
3ul = 17.6mol
1
1k3
1ul = 13.6 mol
1	




PluxL	
1ul = 31mol		
Rfp+t
3ul = 17.6mol
1k3
1ul = 13.6 mol<H1>22ND August to 29th August</H1>
 
 Plux + RLIT
Gel<UL><LI>Gel Elute RLIT</LI><LI>Nano Drop</LI><LI>Gel Check RLIT</LI><LI>Ligations</LI>
PλRAT(220ng/ul) + PRRT(224ng/ul) <LI>Plasmid isolation - 1600</LI><LI>Digestion</LI></UL> 

RLuRTP 
  Transformation()
  J23100 promoter plasmid/Disgested ask? And then verify
 			
 	 	Digestion 1k3 <B>PLRAT 220ng/ul</B>
 
Digestion = 50ul
Divide in 2 and add 0.5ul of enzymes.   
PLRAT E/S = 25ul (66ng/ul) 
PLRAT X/P = 25ul (66ng/ul) 
PRRT 220ng/ul
 
Digestion 
Divide in 2 and add 0.5ul of enzymes.  
 
PRRT E/S=25ul (66ng/ul) 
PRRT X/P = 25ul (66ng/ul) 

1.Ligation of plux+(RLIT)+1K3(iGEM) (4:2:1)
2.Ligation of plux+(RLIT)+1K3(iGEM)(1:1:1) 
3.Ligation of plux+(RLIT)+1k3(iGEM) (8:2:1) 
4.Ligation of plux+1k3(iGEM) Control (8:2:1) 
5.Ligation of plux+(RLIT)+1K3(GE) (1:1:1) 
7.Ligation of PλRAT+PRRT+1K3 (2:2:1) 
6.Ligation of PλRAT+PRRT+1K3(iGEM)(2:2:1) <H1>6/9 to 13/9</H1>
Digestion PL-PR-Elution

  Verify LuxR
 
  Gel elute PL-PR==&gt;E/S and X/P 
  PL-PR band visible with distinction of digested backbone.
Taking my history into account possible that E/S and X/P intermixed. 
After gel elution 5ul on gel - No bands visible 	
 
Ligations
•	LuxI and LuxR 
•	LuxI and PL-PR(with BB) 
•	LuxI 
 Strategy to characterize parts
Work on ADDGENE PARTS. 

 Ligation with pbluescript
•	LuxI with sk(+)
•	LuxR with ks(+)
•	LuxIR with RFP
 Ligation with 1C3
•	LuxI
•	LuxR
•	aiiA
•	RFP
•	LuxIR with 1C3
 Gel verification 
 aiiA
 RFP
 PL-PR E-S GE
 Pl-PR X-P GE 
Ligations with 1C3
Ligations with sk+ 
L10 CONTROL
 
Psl1180
Performed Electroporation of LuxIR
•	Pellet was alright
•	Poured 12.5ul in one and 8ul in another. 
•	One of them arched - not sure exactly which one. 
•	Already made plates of kana were used(assuming to be working). 
•	For some reason while spreading resistance came very early. 
•	Have also inoculated 5ul of recovery into LB. 
 
Results 1st batch of transformation failed. 
Reason ==&gt; 
•	1st time with gblocks. 
•	LuxI and LuxR were verified on gel. 
•	Problem with desalting possible. 
•	Enzymes are not working properly on DNA. 

Ligations with 1C3l 
Digestion LuxI - Verification+ligation
Digestion LuxR - Verification+ligation 
Digestion Aiia - Verification+ligation 
Digestion RFP - Verification+ligation
Results: - They are where they are supposed to be and conc
Was also what we expected. 

  Desalting of 10 ligations. 
  3 plates of chloramphenicol and 5 of amp. 
  Need some amp plate to verify amp plate. 
 
 			
 	Digestion of 1C3= 20ul
Digestion of LuxI = 20ul<H1>14/9 to 21/9</H1>

Sterilizing cuvettes
Wednesday, September 14, 2016
2:19 AM
10 times MQ
Fill with ethanol 10min
Dump it out
Put it in petri dish incubate in UV for 15min
Put them in oven for 5-6 hrs or overnight
  Kshitij did an experiment to check contamination of DNAses in tank. 
  In one ladder he poured 20mM of MgCl2. 
  Stefan sir were right smear was because of DNAses. 
  He said problem was because of tank. 
 	So from now put gel in 4c and use fresh TAE. 


LIGATION OF LuxIR with 1C3 
LIGATION OF LuxIR with 1C3 
LIGATION OF LuxIR with 1K3
LIGATION OF LuxIR with 1K3 

Ligations with 1K3
 	PL-PR with PRLIT
  Gel check PL-PR
  Digest Pl-PR with E/S- sequential
  Gel Check 
  Ligation
  Keep some for gel elution tomorrow		
 			
Gel Check
•	PRLIT and as control RLIT. 
PL-PR digested. <H1>22nd September to 29th September</H1>

  Make reagents for plasmid isolation - Ayush and Zode
  Autoclave of LB , MQ, and others. 
  Send parts for sequencing - give work to someone - K.Vinay all the clones send everything. 
  Digestion of luxI, LuxR and 1C3 and 1K3. 
  Gel verification
  12 colonies LuxR - 
  Digested LuxIR

Ligations
 
LuxIR(IDT) 
L1 1:1:1 kana
 
LuxI-E/S(IDT)[9ng/ul] = 1.2ul = 0.011
LuxR-X/P (IDT)[9ng/ul] = 1.2ul = 0.011
1K3 -E/P [12.5ng/ul] = 2ul = 0.011  
Ligase = 1ul
Buffer = 2ul
MQ = 12.6
 
L2 3:3:1 kana
LuxI-E/S(IDT)[9ng/ul] = 3.6ul = 0.033
LuxR-X/P (IDT)[9ng/ul] = 3.6ul = 0.033
1K3 -E/P [12.5ng/ul] = 2ul = 0.011  
Ligase = 1ul
Buffer = 3ul
MQ = 16.8ul
 
L3  CAT
PRLIT +LuxR
PRLIT = 2ul
LuxR = 1.2ul
1C3 = 2ul
Buffer = 2ul
Ligase = 1ul
Mq= 11.8ul
 
 
 
 
Ligations
 
PLIRT
L4 CAT
PRLIT E/S = 2 ul
PLURT X/P = 2 ul
1C3 = 2ul
Ligase = 1ul
Buffer = 3ul
MQ = 20ul
 
L5 CAT
PRLIT E/S = 4ul
PLURT X/P = 1 ul
1C3 = 2ul
Ligase = 1ul
Buffer = 3ul
MQ= 19ul
 
L6 CAT
PRLIT E/S = 2 ul
PLURT X/P = 2 ul
1C3 = ------
Ligase = 1ul
Buffer = 3ul
MQ = 20ul
 
 
 
L7 control for ligase   Amp
Pcks+ = 2ul
Ligase = 1ul
Buffer = 2ul
MQ = 15ul 
 
 
PRLIT is confirmed to be at the required size. 

Innouclate PRLIT and plasmid isolation using kit. 
Gel elution plURT and PRLIT	
Autoclave 100 LB, tips, MCT's 2ml -- Ayush 
Prepare own TSS -- Angad
Again transformation of remaining ligated mixture -- Shashank
luxIR2 1 and 3 ki digestion with 
 ask stefan for ligase 10ul<H1>28th September to 15th October </H1>

 			
 	  Digest 1k3 and 1c3 
  Send PRLIT for sequencing ASAP. 
  Transformations tomorrow. 
  1C3 and 1K3 verified on gel. Perfect picture. Important observation on gel
  PRLIT and PLuRT also verified on gel. 
  Everything is set for liagtions. 
 	To make TSS of my own<P>
Preparing LA plates by putting 2 flasks of 300ml failed 
Somehow heat was not enough for their boiling but enough for the charring. 
It took much longer to boil them and in one them because of improper mixing and late boiling (which ultimately helps in mixing) led to charring of chunk of mass. </P>
		 
 

Ligation of PRLIT and PLuRT


  Digestion of LIR(E/S) and PLPR(X/P) 
  Desalting of LIR and PL-PR
  Heat inactivate
  Digest with Sma1
  Gel Run 
  Gel elution
  Ligation of LIR and PLPR with 1K3 and 1C3. 
  Single digestion of same plasmids  with pst1 of LIR - 1 and 4 - to verify 
  Digestion of false clones to check what they really are. 
 			
 	Digestion = 40ul of LIR
Digestion = 40ul of PL-PR
  Tranformations
  PcstA+RBS+C1
  pBAD+RBS+CI
  LIR
  PL-PR
  Pacyc184

 			
 	There is turbidity in our buffer2.1. 
All this time problems might be because of bufffer2.1 
Digestion = 20ul 0f 1C3 and 1K3 
 			
 	5.Digestion of LIR = 40ul 
 There was no smear in the gel. 
 While eluting 1C3 some of the ladder was also taken
 6.Digestion of PL-PR = 40ul 
 7.Digestion of psb3t5 -E/P= 50ul
 </OL></DIV></DIV></DIV></DIV></DIV></DIV></DIV></BODY></HTML>