"
Page
Discussion
View source
History
teams
Log in
 
Team:Peking R/Notebook/HSY
From 2011.igem.org
Template:Https://2011.igem.org/Team:Peking R/bannerhidden Template:Https://2011.igem.org/Team:Peking R/back2 Template:Https://2011.igem.org/Team:Peking R/Notebookbackground
无标题文档
Hao Siyang's Notebook
 
 
I  am responsible for establishing RNA controllers which recognize theophylline as  a desired ligand. Furthermore, I test and optimize various kinds of RNA  controllers' responses in regulating gene expression.
 
 
 
Download her notebook
 
Contents:
Up  to 7.3
1.Positive transformation of the standard parts  from the 2011 distribution kit.
2.Pick  up three to five clones of each plate and incubate at 37℃ to amplify the bacteria. Use NanoDrop to measure the  concentration of miniprep products.
3.Measure the growth curve of E.coli DH5α  in LB liquid medium without any antibiotics.
4.Prepare for getting a tandem part of B0015-B0025. Access  50μl reaction system to double digest B0015 and B0025 separately. Unfortunately, electrophoresis enzyme digestive product of B0015 does not  result in objective stripes. Low efficiency of gel extraction of B0025  digestive product.
5.Prepare for getting a tandem part of  E0240-B0025 or E0840-B0025. Access 50μl reaction system to double digest E0240,  E0840 and B0025 respectively.
6.PCR primer design.
 
7.4-7.10
1. Taq DNA polymerase PCR to get;Promoter-1/2riboswitch/ribozyme” and “1/2riboswitch/ribozyme-Terminator” sequence respectively. PCR products are designed to conduct bluntend  ligation. However, Taq-PCR products contain poly A. Klenow should be used to  digest the poly A.
2. “Promoter-1/2riboswitch/ribozyme” and “1/2riboswitch/ribozyme-Terminator” sequence digest of double enzymes, and then link with backbone pSB1K3. Change  the resistance of backbone from ampicillin to kanamycin can avoid false  positives.
3. Use high fidelity pfu DNA polymerase to get  PCR products for linking without poly A.
4. Prepare theophylline solution. According to  its solubility in H2O, 10mM theophylline aqua and 10mM theophylline  LB liquid medium are available.
 
7.11-7.17
1. Measure the growth curve of E.coli DH5α in LB liquid  medium without any antibiotics.
2. Pick up three to five clones of each plate  which contains designed measurements. Send to  sequencing.  
3. Prepare pSB1K3, pSB1A3 and pSB1C3 as backbones for parts ligation. Digest backbones of  double enzymes.
4. Use Mutation Kit to get PCR  products without poly A.
5. Design ”Terminator-Promoter” part. PCR to get promoter J13453(pBAD), then digest of XbaI and PstI. Link the  segment with B0015 which has been digested of SpeI and PstI.
7.18-7.25
1. Try new method to get  designed RNA controllers. Pick up several clones of each plate, then PCR or  double digestion and agarose gel electrophoresis to test the preliminary  correctness of sequence.
2. Measure the ligand responsive  curve of theophylline hammerhead ribozyme designed by MU Tong with a theophylline  gradient. Take GFP fluorescence intensity as gene expression intensity. Add  theophylline before the inducer.
3. Design new primers to get  original RNA controller N8.3.
4. Sequencing result of “Terminator-Promoter” part is negative.
5. Send inocula of designed  theophylline responsive riboswitch measurements to sequencing.
7.26-8.4
1. Get positive results of three  theophylline responsive riboswitch measurements----1G1, 1G2 and 1G3. They both  consist of pBAD, riboswitch and E0840.
2. Measure the ligand responsive  curve of 1G1, 1G2 and 1G3 with a theophylline gradient. 1G1 shows the best  performance.
3. Design and receive the  primers of 1G1 fuse 36bp of C1434.
4. PCR to obtain N8.3 and  1G1-C1.
8.14-8.20
1. Get positive results of N8-3.
2. Measure the ligand  responsive curve of N8-3 with a theophylline gradient.
3. Measure the ligand  responsive curve of three groups of simultaneous 1G1 inocula with a  theophylline gradient. Record OD600nm when add the inducer.
4. Measure the ligand responsive  curve of TPP hammerhead ribozyme designed by ZHAO Yangyang with a TPP gradient.
5. Measure the ligand responsive  curve of AND GATE-1G1 with a theophylline gradient. Arabinose and salicylic  acid are inducers.
8.21-8.30
1. Measure the ligand responsive  curve of modified TPP hammerhead ribozyme with a TPP gradient.
2. Measure the ligand responsive  curve of AND GATE-1G1 with a theophylline gradient. Arabinose and salicylic  acid are inducers.
3. Design and receive the  primers of parental 1G1.
4. Use mutation kit to get  parental 1G1. Take 1G1 which was assembled in pSB1K3 as  template.
5. Mutate the RBS of AND GATE.
8.31-9.7
1. Design primers to obtain  solely original RBS of N8.3. Design primers to obtain solely original RBS of  parental 1G1.
2. Positive results of 1G1-C1,  P1G1, P1G1-OR, N8.3 and N8.3-OR.
3. Measure the ligand responsive  curve of parental 1G1 with a theophylline gradient. Record OD600nm  when add the inducer.
4. Measure the ligand responsive  curve of 1G1-C1 with a theophylline gradient. Record OD600nm when  add the inducer.
9.8-9.15
1. Measure the ligand responsive  curve of P1G1-OR and N8.3-OR with a theophylline gradient. Record OD600nm  when add the inducer.
2. Alert backbone from pSB1K3 to pSB1C3 for part submission. Digest pSB1C3 with E.coRI and PstI. Digest designed parts which are  assembled in pSB1K3 with E.coRI and PstI. Link  the segments and transform.
3. Send inocula to sequencing.
4. Design primers to obtain sole  RNA controllers which are expected inserted in pSB1C3.
9.16-9.23
1. Positive results of 1G1,  P1G1, P1G1-OR, N8.3 and N8.3-OR assembled in pSB1C3.
2. Remeasure the ligand responsive curve of TPP parts with a TPP gradient.
3. Use mutation kit to abandon promoter of P1G1 and 1G1. Send plasmid to sequencing. Get positive results.
4. Use mutation kit to insert sole RNA controller into backbone pSB1C3.
9.24-10.3
1. Part submission.
2. Growth curve measurement. Compare growth rate in the absence and presence of theophylline to test the toxicity of theophylline.
3. Part description. Enrich project and wiki.
 
==click here to her page==
==click here to return==
 
Notebook
>>
Personal Note
>>
Hao Siyang
Retrieved from " http://2011.igem.org/Team:Peking_R/Notebook/HSY "
Recent changes
What links here
Related changes
Special pages
My preferences
Printable version
Permanent link
Privacy policy
Disclaimers
