<HTML xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en" dir="ltr">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki  ltr ns-0 ns-subject page-Team_Peking_R_Project"><DIV id="globalWrapper"><DIV id="top-section"><DIV id="p-logo"><A href="/Main_Page" title="Main Page">&quot;
	    </A></DIV><DIV id="menubar" class="left-menu noprint"><UL><LI class="selected"><A href="/Team:Peking_R/Project">Page               </A></LI><LI class="new"><A href="/wiki/index.php?title=Talk:Team:Peking_R/Project&amp;action=edit&amp;redlink=1">Discussion               </A></LI><LI><A href="/wiki/index.php?title=Team:Peking_R/Project&amp;action=edit">View source               </A></LI><LI><A href="/wiki/index.php?title=Team:Peking_R/Project&amp;action=history">History               </A></LI><LI style="color:#808080;cursor:default">teams</LI></UL></DIV><DIV class="right-menu noprint" id="menubar"><UL><LI id="pt-login"><A href="/wiki/index.php?title=Special:UserLogin&amp;returnto=Team:Peking_R/Project" title="You are encouraged to log in; however, it is not mandatory [o]" accesskey="o">Log in</A></LI></UL></DIV><DIV id="search-controls" class="noprint"><FORM action="/Special:Search" id="searchform"> </FORM></DIV></DIV><DIV id="content"><H1 class="firstHeading">Team:Peking R/Project</H1><DIV id="bodyContent"><H3 id="siteSub" class="noprint">From 2011.igem.org</H3><P><A href="/wiki/index.php?title=Template:Https://2011.igem.org/Team:Peking_R/bannerhidden&amp;action=edit&amp;redlink=1" class="new" title="Template:Https://2011.igem.org/Team:Peking R/bannerhidden (page does not exist)">Template:Https://2011.igem.org/Team:Peking R/bannerhidden</A><A href="/wiki/index.php?title=Template:Https://2011.igem.org/Team:Peking_R/back2&amp;action=edit&amp;redlink=1" class="new" title="Template:Https://2011.igem.org/Team:Peking R/back2 (page does not exist)">Template:Https://2011.igem.org/Team:Peking R/back2</A><A href="/wiki/index.php?title=Template:Https://2011.igem.org/Team:Peking_R/Projectbackground&amp;action=edit&amp;redlink=1" class="new" title="Template:Https://2011.igem.org/Team:Peking R/Projectbackground (page does not exist)">Template:Https://2011.igem.org/Team:Peking R/Projectbackground</A><TITLE>无标题文档</TITLE></P><DIV id="apDiv1"><P class="notbookmaintitle" align="center">Softcoding of Genetic Program</P><P><STRONG>Background  of softcoding</STRONG></P><P>There are basically two design principles  in computer programming: hardcoding and softcoding. Hardcoding [1] refers to  the practice of embedding parameters and functions into the source code of a  program, whereas soft coding [2] obtains values and functions from external  source. Hardcoding would be convenient when no dynamic parameters are required  in the program, but the source code should be rewritten anytime the input data  or functions change. On the contrary, softcoding enables users to customize the  function of software to their needs by altering external input, without having  to edit the program’s source code time after time.</P><P>In the exciting field of synthetic biology,  various genetic programs have been developed to perform certain functions in  living organisms. For instance, a genetic toggle switch in vivo was developed  that could be switched between two states by chemical or thermal induction [3].  Another example is that an oscillatory network was constructed in which the  synthesis of green fluorescent protein was periodically induced [4]. Yet  genetic programs need optimization to achieve ideal performance, especially  when several genetic modules are coupled. Besides, optimization of metabolic  pathways is also a key issue in the field of metabolic engineering.</P><P>Traditional methods to optimize genetic  programs or metabolic pathways generally involve construction of libraries that  contain large amounts of mutants, and multi-round screening or iterative design  is usually required. Apart from the obvious drawback that the constructing and  screening procedures are laborious and time-consuming, these methods could only  generate mutants with a fixed configuration, and to fine-tune their performance  would require another round of mutagenesis and selection, which resembles  “hardcoding”. Therefore, a platform for “softcoding” of genetic programs is  urgently needed.</P><P class="mainbody"> </P><P><STRONG>Softcoding  Approach</STRONG></P><P>Based on the principle of softcoding, this  year our team established an extensible and versatile platform for softcoding  of genetic program, which is composed of “genetic rheostats” and an “RBS  calculator”: genetic rheostats consist of interoperable and truly modular  ligand-responsive riboswitches/ribozymes, while the RBS calculator is automated  design of synthetic ribosomal binding sites (RBS) with customized relative translation  strength. When combining them together, a quantitative correlation between the  concentration of specific ligand and synthetic RBS’ relative translation  strength can be established. Therefore, when tuning genetic program, customized  RBS’ relative translation strength at multiple sites can be high-throughputly  achieved without having to conduct laborious mutagenesis and characterization,  followed by easily determining the configuration of RBS(s)’ relative translation  strength. Then RBS sequences that meet this configuration will be automatically  designed via computer algorithms.</P><P class="mainbody"> </P><P><STRONG>Genetic Rheostats (<A href="https://2011.igem.org/Team:Peking_R/Project/RNAToolkit">Learn more...</A>)</STRONG></P><P>Recently, RNA devices have emerged as  powerful tools to regulate gene expression in vivo, and particularly,  ligand-responsive riboswitches enable us to manipulate translation strength of  specific genes upon different concentrations of ligands [5]. Ligand-responsive  riboswitches regulate the translation rate of downstream gene by changing  conformations, cleaving or splicing upon external addition of ligand. Compared  with transcriptional and post-translational regulation, riboswitches function  through allostery of RNA structure, which requires little or no assistance from  proteins, so the regulation mechanism is relatively simpler and their functions  are more decoupled from native biological activities. In our project, we  selected riboswitches as the genetic rheostats.</P><P>We characterized some existing  riboswitches, namely thiamine pyrophosphate (TPP)-responsive hammerhead  ribozymes and theophylline riboswitches. By altering the upstream promoter and  downstream coding sequence of the RNA controllers, we demonstrated that their  performance was independent from sequence context, which proved that these RNA  devices are modular.</P><P>To further extend the range of application  of our genetic rheostats, we created a ribozyme that functions with a different  mechanism, which has an extreme low level of backgrounds. We substituted the  aptamer domain of c-di-GMP group Intron to theophylline-responsive aptamer,  thus invented a group I intron that senses theophylline to perform splicing  function. Moreover, we provided a general method to evolve genetic rheostats that senses new ligands. By coupling an adenine aptamer with  hammerhead ribozyme and randomizing nucleotides in the linker domain, we  evolved a novel genetic rheostat through dual selection, whose self-cleavage  could be regulated by adenine. The method provided here is user-friendly. </P><P class="mainbody"> </P><P class="mainbody"> </P><P class="picturemark">Figure  1． A) The tertiary structure of natural TPP aptamer. B) Activation fold of translation of P1G1 and 1G1 under a gradient of TPP. C) The design of the  engineered group I intron with theophylline aptamer. D) Schematic illustration of  the dual genetic selection process of genetic rheostat</P><P><STRONG>RBS  Calculator (<A href="https://2011.igem.org/Team:Peking_R/Project/RBSAutomatedDesign">Learn more...</A>)</STRONG></P><P>Since the genetic rheostat tunes translation by decreasing or increasing the probability of RBS exposure, which is proportional to translation strength, we can achieve the mapping process merely by determining the translation strength of the fully exposed RBS. Conventionally, translation strength was calibrated with a library of RBS mutants. But this approach is not truly reliable, because the strength of RBS strongly depends on the surrounding sequence, for instance the coding sequence. Salis <EM>et.al.</EM> [6] used Gibbs energy  (∆G) of the “docked” state of the mRNA-30S ribosomal subunit complex to predict  the translation strength of RBS sequence. Based on their pioneering work, we  developed an RBS calculator that correlated the performance of the RNA  controllers under certain concentration of ligand to relative translation  strength met by corresponding RBS sequence. Combining this RBS calculator with  our genetic rheostats, we can generate an RBS sequence through automated design  once we achieved an ideal configuration of genetic programs through genetic  rheostats.</P><P class="mainbody"> </P><P class="picturemark"> figure 2  Schematic representation of genetic soft-coding approach and the role of RBS calculator. </P><P class="mainbody"> </P><P><STRONG>Demonstration of softcoding  platform (<A href="https://2011.igem.org/Team:Peking_R/Project/Application">learn more...</A>)</STRONG></P><P>We utilized the genetic rheostat and RBS calculator combination to fine-tune two genetic circuits, as proof of principle. The results demonstrate that, for given gene circuit, our genetic rheostat and RBS calculator combination can not only determine its function-translation strength map but also can precisely predict and implement any desired possible function.</P><P class="mainbody"><A href="https://2011.igem.org/Team:Peking_R/Project/Application/AG">AND gate</A></P><TABLE width="200" border="0" cellspacing="0" cellpadding="0"><TBODY><TR><TH scope="col"> </TH><TH scope="col"> </TH></TR><TR><TD> </TD><TD><P class="picturemark">Figure 3. AND gate performance modulated by  different concentration of thiamine pyrophosphate (TPP). The on/off ratio of  AND gate increases with ligand concentration, resulting in a well performed AND gate.</P></TD><TD> </TD></TR></TBODY></TABLE><P> </P><P><A href="https://2011.igem.org/Team:Peking_R/Project/Application/BS">Bistable switch</A></P><TABLE width="200" border="0" cellspacing="0" cellpadding="0"><TBODY><TR><TH scope="col"> </TH><TH scope="col"> </TH></TR><TR><TD> </TD><TD><P class="picturemark">Figure 4<STRONG>.</STRONG>  Fluorescence images of <EM>E.coli</EM> DH5α strain  populations bearing different bistable switch mutants. Each mutant contains different ribosome binding sites (RBSs) controlling the  expression of <EM>cI434 </EM>.  The ratiometric of green cells to red cells for each mutant bistable was precisely predicted by RBS calculator. </P></TD><TD> </TD></TR></TBODY></TABLE><P class="mainbody"><A href="https://2011.igem.org/Team:Peking_R/Project/Application/VIO">violacein biosynthetic pathway</A></P><P class="mainbody">We further applied this methodology  to  fine-tune violacein biosynthetic pathway. with genetic rheostat and RBS calculator, we could establish the quantitative correlation between the product purity, and the translation strength of VioE, and then find optimal RBS sequence to fix product purity at desired value. </P><TABLE width="200" border="0" cellspacing="0" cellpadding="0"><TBODY><TR><TH scope="col"> </TH><TH scope="col"> </TH></TR><TR><TD> </TD><TD><P>Figure 5.  <EM>E. coli</EM> producing pigments. The engineered <EM>E. coli </EM>produced dark-green pigments.  Upon addition of different concentration of thiamine pyrophosphate (TPP), the  color of the bacteria gradually shifted from dark-green to dark-brown.</P></TD><TD> </TD></TR></TBODY></TABLE><P> </P><P> </P><P class="mainbody"><SPAN class="Reference">Reference:</SPAN></P><P>[1]  <A href="http://en.wikipedia.org/wiki/Hard_coding">http://en.wikipedia.org/wiki/Hard_coding</A>
  [2] <A href="http://en.wikipedia.org/wiki/Softcoding">http://en.wikipedia.org/wiki/Softcoding</A></P><P><SPAN class="mainbody">[3].Gardner, Timothy S. <EM>et. al.</EM> (2000). Construction of a genetic toggleswitch in <EM>Escherichia coli</EM>. Nature 403, 339-342</SPAN> [4].Elowitz, Michael B. and  Leibler, Stanislas (2000). A synthetic oscillatory network of transcriptional  regulators. Nature 403, 335-338  [5].Breaker, Ronald R (2004).Natural and engineered nucleicacids as  tools to explore biology. Nature 432, 838-845
  [6].Salis, Howard M <EM>et.al.</EM> (2009). Automated design of synthetic ribosome binding sitesto control protein  expression. Nat. Biotech. 27, 946-950</P></DIV><DIV class="printfooter">
Retrieved from &quot;<A href="http://2011.igem.org/Team:Peking_R/Project">http://2011.igem.org/Team:Peking_R/Project</A>&quot;</DIV></DIV></DIV><DIV id="footer-box" class="noprint"><DIV id="footer"><UL id="f-list"><LI id="t-recentchanges"><A href="/Special:RecentChanges" title="Recent changes">Recent changes</A></LI><LI id="t-whatlinkshere"><A href="/Special:WhatLinksHere/Team:Peking_R/Project" title="List of all wiki pages that link here [j]" accesskey="j">What links here</A></LI><LI id="t-recentchangeslinked"><A href="/Special:RecentChangesLinked/Team:Peking_R/Project" title="Recent changes in pages linked from this page [k]" accesskey="k">Related changes</A></LI><LI id="t-specialpages"><A href="/Special:SpecialPages" title="List of all special pages [q]" accesskey="q">Special pages</A></LI><LI><A href="/Special:Preferences">My preferences</A></LI></UL></DIV><DIV id="footer"><UL id="f-list"><LI id="t-print"><A href="/wiki/index.php?title=Team:Peking_R/Project&amp;printable=yes" title="Printable version of this page [p]" accesskey="p">Printable version</A></LI><LI id="t-permalink"><A href="/wiki/index.php?title=Team:Peking_R/Project&amp;oldid=260879" title="Permanent link to this revision of the page">Permanent link</A></LI><LI id="privacy"><A href="/2011.igem.org:Privacy_policy" title="2011.igem.org:Privacy policy">Privacy policy</A></LI><LI id="disclaimer"><A href="/2011.igem.org:General_disclaimer" title="2011.igem.org:General disclaimer">Disclaimers</A></LI></UL></DIV></DIV></DIV></BODY></HTML>