"
Page
Discussion
View source
History
teams
Log in
 
Team:Peking R/Project/RNAToolkit3
From 2011.igem.org
Template:Https://2011.igem.org/Team:Peking R/bannerhidden Template:Https://2011.igem.org/Team:Peking R/back2 Template:Https://2011.igem.org/Team:Peking R/Projectbackground5
无标题文档
 
 
 
 
 
 
Basal-Level-Free Genetic Rheostats
Introduction
The relatively simple and repetitive  composition of RNA molecules makes them easy to synthesize and manipulate.  Evidence shows that RNA also has the structural sophistication needed for  dynamic interplay between functionally distinct catalytic and receptor  domains [1] . In our project, we tried to utilize these small genetic rheostats to  tune translation rate of target gene quantitatively.
Numerous riboswitches and hammerhead  ribozymes have been depicted previously [2,3] . However, they all function via  hiding or exposing the ribosomal binding site (RBS, which is a major determinant  of translation rate) through different mechanisms (Figure 1a and 1b) [3,4] . This  will probably lead to a relatively high basal level because of the conservation  of intact RBS. We speculated that, with the self-splicing Group I intron  ribozymes that will be further described later, we could interrupt the RBS  sequence via inserting a ribozyme to remove the basal level of translation rate  and manipulate translation rate via modulating fraction of ligand-induced  self-splicing (Figure 1c).
 
 
 
 
Figure 1. Predicted  mechanisms for different allosteric RNA-mediated gene expression control. (a) Allosteric riboswitch. In the absence of effector  molecules----theophylline in this example (left), the 5' region can adopt a  highly folded structure that extensively pairs the region that includes  ribosomal binding site to part of the aptamer sequence (shown in green). In the  presence of theophylline (right), the secondary structure shifts, such that the  ribosomal binding site is exposed [high]. (Sean A. Lynch, Shawn K. Desai, Hari  Krishna Sajja, and Justin P. Gallivan. (2006). A high-throughput screen for  synthetic riboswitches reveals mechanistic insights into their function.  Chemistry & Biology 14, 173-184) (b) Hammerhead aptazyme. Without the  inducer (left)----thiamine pyrophosphate (TPP), hammerhead ribozyme is  stabilized in a misfolded state with the ribosomal binding site sequestered by  the ribozyme stem I. The addition of TPP (right) unmasks the ribosomal binding  site through self-cleaving (Markus Wieland, Armin Benz, Benedikt Klauser, and JÖrg S. Hartig. (2009).  Artificial ribozyme switches containing natural riboswitch aptamer domains. RNA  technology 121, 2753-2756). (c) Group I intron aptazyme. Precursor  mRNA (right) with the ribosomal binding site separated by the group I intron.  RNA processed in the presence of c-di-GMP (left) creates a perfect ribosomal binding site (Elaine R.Lee, et al. (2010). An allosteric self-splicing ribozyme  triggered by a bacterial second messenger. Science 329, 845-848). Green: ribosomal binding site, black: open reading  frame (ORF), red arrow: hammerhead ribozyme cleavage site, 5'-ss and 3'-ss:  group I self-splicing ribozyme corresponding splicing site. Start codons and effector  molecule theophylline are indicated in the figure. Watson-Crick base pairs are  denoted by dashes, all other base pairs are indicated as dots. All of these  three elements are used in our project deriving from corresponding reference  listed after the legends. The structures shown upper might be a little  different from the original design in the listed paper, such us the start  codon----AUG instead of the atypical ones----UUG in (c).
 
In fact, this mechanism has been depicted  in the pathogenic bacterium Clostridium  difficile in nature with an allosteric group I ribozyme inside the RBS,  wherein self-splicing is regulated by a distinct aptamer class that senses the  bacterial second messenger c-di-GMP [5] . Here, we regard this ribozyme as a  prototype to evolve ligand-regulated self-splicing ribozymes. Before those, we  will first make a brief introduction about group I intron ribozyme.
Group I intron ribozymes
Group I intron ribozymes consist of two  parts----an aptamer which directly senses the small effector molecule and an  intron which self-splices in response to the conformational change transmitted  from the aptamer.
An aptamer is a piece of nucleic acid or  peptide that binds to specific ligand. Here, we refer to RNA aptamers. Various  kinds of aptamers have been discovered, among which is the one responses to 3'-5'-cyclic  diguanylic acid (c-di-GMP). C-di-GMP is a second messenger signaling molecule  that regulates many vital processes within the bacterial kingdom including  motility, pathogenesis and biofilm formation [6,7] . Its corresponding aptamers  can be categorized into two classes and the binding mechanism of class I is  illustrated in Figure 2 and Figure 3 [7] .
 
 
 
Figure  2. Comparison of the consensus sequences and  structural models for class I (left) and class II (right) c-di-GMP aptamers.  (Elaine R.Lee, et al. (2010). An allosteric self-splicing ribozyme triggered by  a bacterial second messenger. Science 329, 845-848)
 
 
 
 
 
Figure  3.  Structure of the class I c-di-GMP  aptamer. (a) Secondary structure representation of the crystallized c-di-GMP aptamer.  Helices P1, P2, and P3 are colored purple, blue, and green respectively. C-di-GMP  is shown in red. Nucleotides that directly contact c-di-GMP are shown in  orange. The U1A cocrystallization protein is shown in gray. Watson-Crick base  pairs are denoted by dashes, all other base pairs are indicated using the  nomenclature of Westhof and Leontis. A* indicates a Watson-Crick base pair in  the tertiary structure. The two guanines of c-di-GMP are denoted Gα and Gβ  for clarity. (b) Representation of  crystal structure of the Vc2 c-di-GMP riboswitch aptamer from V. cholera. Coloring is the same as used  in (a). (Kathryn D Smith, Sarah V  Lipchock, Tyler D Ames, Jimin Wang, Ronald R Breaker, and Scott A Strobel.  (2009). Structural basis of ligand binding by a c-di-GMP riboswitch. Nat Struct  Mol Biol. 16, 1218-1223)
 
An intron is a non-coding sequence found  within the genes of all three domains of life. Introns are classified into four  groups based on splicing mechanisms. Group I and group II introns are able to  self-splice using different mechanisms [8] . However, assistance from proteins  is required for in vivo splicing for Group II intron. All Group I introns share  a conserved secondary structure consists of nine paired regions (P1-P9 shown in  Figure 4) [9] . Splicing of Group I introns is processed by two sequential  ester-transfer reactions. The exogenous guanosine or guanosine nucleotide first  docks onto the active G-binding site located in P7, and its 3'-OH is aligned to  attack the phosphodiester bond at the 5' splice site located in P1, resulting  in a free 3'-OH group at the upstream exon and the exoG being attached to the 5'  end of the intron. Then the terminal G (ΩG) of the intron swaps the exoG and occupies the G-binding site to  organize the second ester-transfer reaction, the 3'-OH group of the upstream  exon in P1 is aligned to attacks the 3' splice site in P10, leading to the  ligation of the adjacent upstream and downstream exons and free of the catalytic  intron [10] (Figure 5).
 
 
 
Figure  4. Overall  secondary and tertiary structure of the Azoarcus pre-2S intron splicing  complex. (a) Intron secondary structure. The intron sequences, exon  sequences and structural elements (P and J elements) are depicted in the  colours. The ligation reaction catalyzed by this complex is shown by black  arrows. The RNA transcript (UP62) comprising the majority of the intron is  shown with capital letters, while residues derived from the two chimaeric  oligonucleotides dCIRC (intron/3'-exon segment) and CAT (5'-exon segment) are  shown in lower-case letters. The break between dCIRC and UP62 is indicated with  a jagged line. (b) Overall structure  of the intron. The backbone is depicted with a ribbon and individual bases  depicted as cylinders. The two tetraloop-tetraloop receptor interactions on the  periphery of the intron are indicated. (Peter L. Adams, Mary R. Stahley, Anne  B. Kosek, Jimin Wang, and Scott A. Strobel. (2004). Crystal structure of a  self-splicing group I intron with both exons. Nature 430, 45-50)
 
 
 
 
 
Figure  5. Mechanism of group I intron splicing. 5' and 3'  exons are in red and blue, respectively. ΩG,  terminal intron guanine. G*, exogenous guanosine. (Step I) Nucleophilic attack  on the 5' splice site by the 3'-OH of G* in guanosine-binding site (GBS). (Step  2) Nucleophilic attack on the 3' splice site by the free 3'-OH of the 5' exon.  (Step 3) Free intron and spliced exons. (Rahul Raghavan  and Michael F. Minnick (2009). Group I Introns and Inteins: Disparate Origins  but Convergent Parasitic Strategies. American Society for Microbiology 191,  6193-6202)
 
Engineering of  Ligand-Responsive Self-splicing
We firstly examined the utility of  naturally emerging c-di-GMP ribozyme in E.coli with a circuit design shown in  Figure 6. It was anticipated that the self-splicing could be chemically induced  by c-di-GMP in vivo and the expression level of GFP would increase with  increasing ligand concentration.
 
 
 
 
Figure  6. The plasmid used in this project (left) and key  features of the c-di-GMP aptamer-ribozyme construct (right), including  validated splice and GTP attack sites. GTP1 and GTP2 indicate two different GTP  attacking sites. Alternative base-pair interactions guiding allosteric function  are shaded blue and green. (Elaine R.Lee, et al. (2010). An allosteric  self-splicing ribozyme triggered by a bacterial second messenger. Science 329,  845-848)
 
However, no change in GFP expression was  observed during in vivo chemical induction (Figure 7). This is probably due to  impenetrability of the second messenger c-di-GMP when it diffuses through cell  membrane. Therefore, it's not proper to use c-di-GMP as a ligand to carry experiments in vivo.
 
 
 
Diagram  1. C-di-GMP inducing experiment. Activity of  c-di-GMP-dependent allosteric ribozyme reported by green fluorescent protein (GFP).  All are incubated for 120 minutes in the presence of various concentrations of  the second messenger c-di-GMP and 1mM arabinose(Only the highest concentration  used in this work is shown). C-di-GMP ribozyme used here has been described in  Figure 6a previously. The numbers over the column is corresponding c-di-GMP  concentration used. All are excitated at 470nm with investigating their  absorption at 509nm.
 
We then swapped c-di-GMP aptamer with a  theophylline hammerhead ribozyme which was found by Ronald R. Breaker in 2000 [3] .  We designed three alternative base-pairing structures, the ribozyme P1 stem (Figure  7 dark blue boxing), hammerhead stem I (orange shading) and the extending P10 stem (blue shading), which may explain theophylline inducing  control. Misfolded aptamer formation disrupts base pairing of the ribozyme P1 stem and further affects the structure of the group I intron. Because theophylline binding stabilizes aptamer substructures, presence of theophylline should favor the formation of hammerhead ribozyme stem I and induced the cleavage of the hammerhead. This process induced the correct formation of the group I intron thus endow it with the self-splicing activity which finally brings on the formation of RBS----AGGAGG.
 
 
 
 
Figure  7. Rational design of engineered theophylline ribozyme.  Proposed mechanism for engineered theophylline ribozyme-mediated gene control (a and b). (a-b) In the absence of theophylline, the correct structure of group I intron is disrupted. As theophylline binding stabilizes aptamer substructures, presence of theophylline should favor the formation of hammerhead ribozyme stem I. This leads to cleavage of the hammerhead ribozyme and further splicing of the group I intron which finally brings on the formation of RBS----AGGAGG (shown in red letters separated). (c) Proposed secondary structure of the positive control in  theophylline inducing experiment. (d) Map of the plasmid using in this part of the project. Orange: hammerhead stem  I, blue: ribozyme P1 stem, black: open reading frame (ORF), dark blue box: anti  RBS stem, red letters separated: ribosomal binding site (RBS), 5'ss and 3'ss:  group I self-splicing ribozyme corresponding splicing site, red arrow:  hammerhead ribozyme cleavage site, theophylline aptamer, hammerhead ribozyme  and the elements in plasmid map are illustrated in this figure. Watson-Crick  base pairs are denoted by dashes, all other base pairs are indicated as dots.
 
 
As the Diagram 2 shows below, addition of  theophylline indeed facilitated the expression of GFP, which implies that the  domain swapping succeeded.
 
 
 
 
Diagram  2. Theophylline inducing experiment. Activity of  theophylline-dependent engineered group I intron reported by green fluorescent  protein (GFP). All are incubated for 120 minutes in the presence or absence of  10mM theophylline and 1mM arabinose. The plasmid used in this work is theophylline-dependent  engineered group I intron described in Figure 7a,7b,7d. * is the positive  control ---- hammerhead-group I ribozyme without specific response described in  Figure 7c. All are excitated at 470nm with investigation their absorption at  509nm.
 
Discussion
Genetic rheostats are promising gene  regulatory elements for efficient operation of cellular processes due to their  modularity and pre-translational fast-response. More and more RNA regulatory  elements have been studied and an easier approach to combine them and cluster  their properties is need urgently. Breaker and his co-workers have previously  demonstrated that the cleavage activity of hammerhead ribozyme can be coupled  with the binding of molecular effectors to corresponding aptamer by simple  expedient of appending aptamers to non-essential stem regions [3,11,12]. However, in addition to hammerhead ribozyme, there are still lots of available  RNA regulatory elements with complicated structures that are difficult to  engineer [13] .
Here, we proposed a design in which the  activity of other complicated ribozyme, including group I intron, can be easily  modulated, combining with engineered hammerhead aptazymes. Hammerhead ribozyme  here was used as an “Adapter” to link the binding of effector molecules to  specific aptamer to the self-splicing activity of group I intron as Figure 8  and Figure 9 shows previously in this project. Under this design, we can easily  swap the hammerhead aptazyme, the adapter, to construct engineered group I  introns that self-splice in response to different effector molecules, without  further in vitro selection which is  time consuming and labor resume. In summary, this design is fast, affordable  and more predictable with general interest.
 
 
Reference:
Ronald R Breaker. (2002).  Engineered allosteric ribozymes as biosensor components. Current Opinion in  Biotechnology 13, 31-39
Markus Wieland, Armin Benz,  Benedikt Klauser, and JÖrg S. Hartig. (2009). Artificial ribozyme switches containing  natural riboswitch aptamer domains. RNA technology 121, 2753-2756
Garrett A. Soukup, Gail A. M.  Emilsson and Ronald R. Breaker. (2000). Altering molecular recognition of RNA  aptamers by allosteric selection. J. Mol. Biol. 298, 623-632
Sean A. Lynch, Shawn K. Desai,  Hari Krishna Sajja, and Justin P. Gallivan. (2006). A high-throughput screen  for synthetic riboswitches reveals mechanistic insights into their function.  Chemistry & Biology 14, 173-184
Elaine R.Lee, et al. (2010). An  allosteric self-splicing ribozyme triggered by a bacterial second messenger.  Science 329, 845-848
Regine Hengge. (2009).  Principles of c-di-GMP signaling in bacteria. Nature 7, 263-273
Kathryn D Smith, Sarah V  Lipchock, Tyler D Ames, Jimin Wang, Ronald R Breaker, and Scott A Strobel.  (2009). Structural basis of ligand binding by a c-di-GMP riboswitch. Nat Struct  Mol Biol. 16, 1218-1223
Rahul Raghavan and Michael F.  Minnick (2009). Group I Introns and Inteins: Disparate Origins but Convergent  Parasitic Strategies. American Society for Microbiology 191, 6193-6202
Sarah A Woodson. (2005).  Structure and assembly of group I introns. Sciencedirect 15, 324-330
Peter L. Adams, Mary R.  Stahley, Anne B. Kosek, Jimin Wang, and Scott A. Strobel. (2004). Crystal  structure of a self-splicing group I intron with both exons. Nature 430, 45-50
Michael P. Robertson and Andrew  D. Ellington. (1999). Design and optimization of effector-activated ribozyme  ligases. Nucleic Acids Research 28,  1751-1759
Michael P. Robertson and Andrew  D. Ellington. (1999). In vitro selection of an allosteric ribozyme that  transduces analytes to amplicons. Nature Biotech 17, 62-66
Kristin M Thompson, Heather A  Syrett, Scott M Knudsen and Andrew D Ellington. (2002). Group I aptazymes as  genetic regulatory switches. BMC biotechnology 2, 21
 
 
Existing Natural Genetic Rheostat
 
TPP-regulated hammerhead ribozyme
 
Theophylline-responsive riboswitch
Extension
 
Engineered group I intron with a theophylline hammerhead ribozyme
 
Adenine hammerhead ribozyme obtained from screening
Retrieved from " http://2011.igem.org/Team:Peking_R/Project/RNAToolkit3 "
Recent changes
What links here
Related changes
Special pages
My preferences
Printable version
Permanent link
Privacy policy
Disclaimers
