"
Page
Discussion
View source
History
teams
Log in
 
Team:Peking R/Project/RNAToolkit
From 2011.igem.org
Template:Https://2011.igem.org/Team:Peking R/bannerhidden Template:Https://2011.igem.org/Team:Peking R/back2 Template:Https://2011.igem.org/Team:Peking R/Projectbackground2
无标题文档
 
 
 
Overview
In recent years, RNA devices have emerged  as powerful tools to regulate gene expression in vivo, and particularly,  ligand-responsive riboswitches/ribozymes enable us to manipulate translation  strength of specific genes upon different concentrations of ligands.  Ligand-responsive riboswitches/ribozymes regulate the translation rate of  downstream gene by changing conformations, cleaving or splicing upon external  addition of ligand. Compared with transcriptional and post-translational  regulation, riboswitches/ribozymes function through allostery of RNA structure,  which requires little or no assistance from proteins, so the regulation  mechanism is relatively simpler and their functions are more decoupled from  native biological activities.
In order to fulfill the goal of  establishing an extensible and versatile methodology for softcoding of genetic  program, our team reached out to a great extent to search for available  ligand-responsive riboswitches/ribozymes that meet our criteria and selected  them as our genetic rheostats.
Candidates for genetic rheostats must meet  two basic criteria: firstly, they should possess a relatively plain  dose-response curve, which would allow for precise translation strength  modulation within a wide range of ligand concentration; secondly, ligands they  recognize should be genetically and biochemically orthogonal to the host cells,  in our case, E.coli cells, as much as  possible.
Two candidates emerged as promising genetic  rheostats that satisfied our requirements: thiamine pyrophosphate  (TPP)-responsive hammerhead ribozymes and theophylline-responsive riboswitches.  By altering the upstream promoter and downstream coding sequence of the genetic  rheostats, we demonstrated that their performance was independent of sequence  context, which proved that our genetic rheostats are modular.
To further extend the repertoire of our genetic rheostats, we created a ribozyme that functions with a different  mechanism, which has an extreme low basal level. We substituted the aptamer  domain of c-di-GMP group I intron to theophylline-responsive aptamer, thus invented a group I intron that senses theophylline to perform splicing  function.
Moreover, we introduced a general method to  evolve genetic rheostat that senses a new ligand. By coupling an adenine  aptamer with hammerhead ribozyme and randomizing nucleotides in the linker  domain, we evolved new genetic rheostat through dual selection, whose  self-cleavage could be regulated by adenine.
 
Existing Natural Genetic Rheostats
 
TPP-regulated hammerhead ribozyme
 
Theophylline-responsive riboswitch
Engineered Genetic Rheostat
 
Engineered group I intron with a theophylline hammerhead ribozyme
 
Adenine hammerhead ribozyme obtained from screening
 
Retrieved from " http://2011.igem.org/Team:Peking_R/Project/RNAToolkit "
Recent changes
What links here
Related changes
Special pages
My preferences
Printable version
Permanent link
Privacy policy
Disclaimers
