"
Page
Discussion
View source
History
teams
Log in
 
Team:Peking R/Project/RBSAutomatedDesign
From 2011.igem.org
Template:Https://2011.igem.org/Team:Peking R/bannerhidden Template:Https://2011.igem.org/Team:Peking R/back2 Template:Https://2011.igem.org/Team:Peking R/Projectbackground7
无标题文档
Thermodynamics-based RBS Calculator
Introduction
Through fine-tuning the tranlation of core genes using the genetic rheostat, we can easily  modulate the behavior of genetic devices and complex gene circuits with ligands at various concentrations and finally configure an automated-designed RBS to meet the corresponding translation strength, thus optimal performance of gene circuits.
To  achieve such a goal, we employed two sets of tools to rationalize the process  of genetic devices/gene circuit modulation–genetic rheostats(ligand-responsive RNA elements) to quantitatively modulate translation process, and the RBS Calculator to predict the rate of translation initiation and automatically design a synthetic RBS  sequence that meets the needed relative translation rate determined by the genetic rheostat. A critical  step, therefore, is to map the translation strength of a particular sequence to  that of a genetic rheostat-modulated messenger RNA (mRNA). Since the genetic rheostat tunes translation by decreasing or increasing the probability of RBS exposure, which is proportional to translation strength, we can achieve the mapping process merely by determining the translation strength of the fully exposed RBS.
Conventionally, trl. str. was calibrated with a library of RBS mutants. But this approach is not truly reliable, because the strength of RBS strongly depends on the surrounding sequence, for instance the coding sequence, as is shown in Figure 2. Salis and Voigt have derived a model for RBS str. prediction that takes sequence context into consideration.
According to the model, the bridging factor between the strength of translation and a particular RBS sequence is Gibbs free energy change for translation initiation determined by RBS Calculator.A schematic representation of our modeling for the  platform is shown in the diagram below.
 
 
 
 
Figure 1 Schematic  representation of genetic soft-coding approach and the role of RBS calculator.In  a two-step strategy (green boxes connected by curved arrows) to engineer  biological systems, a ligand-responsive RNA controller("genetic rheostat") is used to easily  determine an appropriate translation strength from varying concentrations of  the ligand, followed by replacement of the genetic rheostat(together with its  native RBS) with a fixed synthetic RBS that meets the previously determined  translation strength. The role of the RBS Calculator, therefore, is to map a value  of translation strength (expressed in terms of △G) to a specific RBS sequence.
 
Overview of  Components of the Prokaryotic Ribosome
 
 
 
Figure 3  Components of a prokaryotic ribosome.(Alberts et al., 2008).
 
It  can be seen from Figure 3 that ribosome in prokaryotes consists of two subunits,  one larger with a Svedberg sedimentation coefficient of 50, and a smaller  subunit with a Svedberg sedimentation coefficient of 301. Both  subunits consist of several proteins and several strands of ribosomal RNA (rRNA)  in different lengths. In the process of translation initiation, the 30S subunit  first binds to ribosome binding site (RBS) of the messenger RNA (mRNA), in  which the 16S rRNA of the 30S subunit has a critical role of recognizing and  binding to RBS in mRNA. After recognizing and binding to the RBS, the 30S  subunit then recognizes the start codon and recruits the transfer RNA(tRNA)that  carries a fMet residue corresponding to the start codon, followed by the 50S  large subunit. The formation of a complete ribosome-mRNA complex then “fires  off” from the start codon and starts translation.
 
 
 
Figure 4 The  postulated process of translation initiation. (a) Single-chain mRNA in its  native folded form, with RBS and start codon indicated in yellow and red  respectively. (b) Binding of the 30S ribosomal subunit to the RBS (yellow  ribbon), excluding secondary structures in a region surrounding it. The last  nine nucleotides of the 16S rRNA in the 30S subunit are given. (c) Binding of  the 30S subunit to the start codon (red ribbon) and recruitment of the first  tRNA and 50S subunit to start translation.
 
Based  on the above assumptions, ∆Gtot is composed of five additive Gibbs  free energy terms:
∆Gtot = ∆GmRNA:rRNA  + ∆Gstart + ∆Gspacing- ∆Gstandby- ∆GmRNA
where:
∆GmRNA:rRNAis  the energy change brought by binding of the 16S rRNA tail to the specific  binding region in the 5' UTR of the mRNA.
∆Gstart  is the free energy change that results from binding of the 16S-rRNA to the  start codon(it is treated as a constant for all mRNAs with AUG as the start  codon.
∆Gspacing  is the free energy change brought by the spacing sequence between the RBS and  the start codon, determined by the sequence length using an empirical formula  derived from experimental data.
∆Gstandby  is the energy needed to expose the 16S-rRNA binding site to the 16S-rRNA.
∆GmRNA  is the free energy change for a free mRNA sequence in solution to spontaneously  fold into a native secondary structure.
The  results in the original work agree well with that derived from another probabilistic  model constructed by Na et al. (2010)3.
For  RBS Calculator used in our platform, we optimized parameters and algorithms in  the original work by Salis et al. to produce a more general tool for predicting  the translation rate of mRNAs that encode different proteins. The program  showed fair performance with the green fluorescent protein (GFP) that we used,  but not as satisfactory as it did with red fluorescent protein (RFP) and its  fusion proteins in the original work. Therefore, we attempted to optimize the  program for equal performance on different protein-coding mRNAs. Since the free  energy terms to a great extent depend on a set of thermodynamic parameters used  for secondary structure prediction3, the thermodynamic scoring  matrix for base pairing in RNA was changed from rna1999 parameters used in the  original program to rna1995. Besides, as the sequence surrounding the start  codon has been shown to significantly affect the accuracy of prediction1,the  cutoff value, which is the number of nucleotides up- and downstream of the  start codon used for calculating free energy terms, was increased from 35 to  51, which produced a better fit with experimental measurements (see results  below).
In  order to evaluate the accuracy of the model, we exploited green fluorescent protein  (GFP) as a reporter gene to measure relative translation initiation rates of a library  of mRNAs with synthetic RBSs. The output fluorescence measured by flow  cytometry and/or spectrophotometry was fitted to the corresponding prediction  by RBS Calculator software package that integrates the separate calculations of  the five energy terms described above into ∆Gtot and provides an arbitrary expression level using  equation (1) and fitted parameters for the scaling coefficient K (~2500) and β(~0.45).Results for GFP before  and after changing the parameters are shown in Fig 5a and Fig 5b respectively.  It can be seen that altering the parameters used in the algorithm significantly  improved the fitting results.
 
(a)
(b)
 
 
Figure  5 Fitness of experimental measurements to computational predictions by RBS  Calculator. Horizontal axis indicates the △Gtot determined by RBS  Calculator, and vertical axis corresponds to logarithm of expression level(in  terms of GFP fluorescence normalized for host bacteria growth). (a) Original parameters  (rna1999, cutoff=35nt) produced unsatisfactory fitting with low square R (Pearson  correlation coefficient). (b) Changing thermodynamic scoring matrix and cutoff  value to rna1995 and 51nt respectively resulted in better fitting. Note that  the improvement primarily occurred at the two ends – high and low △G.
 
After confirming the reliability of the  RBS Calculator in determining translation strength, we are able to employ the  computational results to predict a synthetic RBS sequence that matches the  desired relative translation rate (forward engineering). The prediction curve is shown  in Figure 6.
 
 
 
Figure 6 Using RBS Calculator to find a sequence with desired translation strength. It can be seen that the RBS Calculator also performs well in finding the right sequence under our optimized parameter combination.
 
 
Finally, we can figure out the trl. str of the fully exposed RBS sequence and then convert the probabilities of RBS exposure to a range of deltaG for translation. Besides, any given deltaG for trl can be met by an RBS sequence calculated from the RBS Calc. Therefore we can convert any given TPP conc. into an RBS sequence.
 
 
 
Figure 7 Mapping of ligand concentrations and fixed RBS sequences. Using the response curve of RBS exposure probability to TPP(ligand) concentration, in combination with the mapping of RBS exposure probabilities to RBS sequences through translation strengths(deltaG) determined by the RBS Calculator, we may find a corresponding RBS sequence for any given TPP concentration.
 
 
 
Reference:
[1].Alberts, B., Johnson, A., Lewis, J., Raff,  M., Roberts, K., and Walter, P.(2008). Molecular Biology of the Cell. Garland  Science, Fifth Edition, 373-376.
[2]. Salis, H.M.,  Mirsky, E.A., and Voigt, C.A.(2009). Automated design of synthetic ribosome  binding sites to control protein expression. Nature Biotechnology 27, 946-952.
[3]. Na, D., Lee, S., and L, D.(2010).  Mathematical modeling of translation initiation for the estimation of its  efficiency to computationally design mRNA sequences with desired expression  levels in prokaryotes. BMC Systems Biology 4,71.
 
 
 
 
 
 
 
Retrieved from " http://2011.igem.org/Team:Peking_R/Project/RBSAutomatedDesign "
Recent changes
What links here
Related changes
Special pages
My preferences
Printable version
Permanent link
Privacy policy
Disclaimers
