"
Page
Discussion
View source
History
teams
Log in
 
Team:Peking R/Notebook/LYP
From 2011.igem.org
Template:Https://2011.igem.org/Team:Peking R/bannerhidden Template:Https://2011.igem.org/Team:Peking R/back2 Template:Https://2011.igem.org/Team:Peking R/Notebookbackground
无标题文档
Liu Yunpeng's Notebook
 
 
I took part in the task of constructing the platform for translationinitiation rate prediction, and modeling the AND gate and bistableswitch modules.
 
 
 
Download his note
 
Contents:
6.28-7.20
Task:
1. Generate a mutation library for a given  sequence.
I used a traverse algorithm to enumerate every  possible sequence with different mutations in a 7-nt region near the  prospective RBS. For a given sequence, 47(16384) mutant sequences  are produced.
2. Determine △Gtot  for a given mutant.
I first tried to simplify the model produced  in the work by Voigt et al(2009). According to the free energy model brought  forward by Voigt et al(2009), the △G  required for translation initiation consists of five major terms:
△GmRNA(the folding free  energy change for a free mRNA sequence in solution)
△GmRNA:rRNA(the energy  change brought by the binding of the 16S rRNA tail to the specific binding  region in the 5’ UTR of the mRNA)
△Gspacing(the free energy  change brought by the spacing sequence between the RBS and the start codon,  determined by the sequence length using an empirical formula derived from  experimental data)
△Gstandby(the energy needed  to expose the 16S-rRNA binding site to the 16S-rRNA. However, according to  calculations by Voigt et al, this term is almost always approximately zero,  therefore it is omitted in our model)
△Gstart, which is the free  energy change from binding of the 16S-rRNA to the start codon(it is treated as  a constant for all mRNAs with AUG as the start codon)
In my own calculations, the UNAfold package is  used. hybrid-ss-min.exe calculates △GmRNA,  and UNAfold.pl calculates △GmRNA:rRNA. With the help  of a friend proficient in UNIX and programming, I was able to determine the  optimal combination of △GmRNA:rRNA and △Gspacing  to minimize their sum, and run the programs in a batch for all the above terms  to determine △Gtot for every sequence in  the library.
However, the results are unsatisfactory as  they are significantly different from the data produced in Voigt’s work. After  close examination of the software package provided by Voigt et al, I found that  other than over-simplification in our model, the model described in the paper  has significant deviations from that used in the software’s algorithms.  Besides, the software employs NuPACK instead of UNAFold for the energy  calculations.
Therefore, we decided to use the provided  software package to generate data for our own sequences. I had to tackle  problems including properly running the software package and processing our  sequences in a batch before I finally obtained △G  data for experimental verification. Great acknowledgement must be paid to  Qiu Yunjiang for the help given in programming and data processing.
07.21
Constructed and repeated the computational  modeling results of the AND Gate module in iGEM 2009 Team Peking’s Project,  using Matlab to solve the ordinary differential equations(ODEs).
Simplified the sequential logic circuit  containing a bistable switch module constructed by iGEM 2007 Team Peking,  modeling the bistable switch to analysis the effects of translation initiation  rates on the switch’s properties.
07.25
Constructed a coarse phase diagram of the  bistable module to reflect ratios of red and green fluorescence, thus  indirectly reflecting the strength of the ribosome binding site(RBS).
07.31
Platform for high-throughput Gibbs free energy  calculation to predict translation initiation complete. First batch of  sequences containing computationally mutated RBSs are entered into the software  package provided by C. Voigt et al. and calculated for gene expression  strengths.
08.03-08.09
Running the prediction platform on several  batches of sequence libraries to obtain sufficient amounts of data for further  experimental validation of our fine-tuning knob.
08.14
Rewrote the first draft for Human Practice  page, integrating information collected from other teammates and summarizing  everyone’s views on biosafety issues in synthetic biology.
Improved the structure and design of the Human  Practice page so that it could convey more information in a more efficient way.
08.15
Collected data for the survey conducted by Xiao  Qingyang and other teammates and presented the statistical data, drawing  conclusions and providing suggestions for ensuring biosafety in laboratories  using antibiotic-resistant bacteria.
08.18
Modeling of the AND Gate module reveals  critical parameters that establish mapping between RBS strength(△G)  and fold-changes in output when input turns from 0 0 to 1 1. This was  accomplished by quantitatively studying the “transfer function” derived in  previous work by Voigt et al..
08.19
Fitted the calculated RBS strengths to  experimental measurements of actual expression levels(using green fluorescent protein,  GFP). Results were partially disappointing as the two sets of data showed much  less correlation than expected. The calculation and fitting were repeated  changing sequence cutoff values(length of sequence chosen for calculation of △G,  but no improvements were observed. Planned to conduct experiment repeats to  verify the reliability of measurements. However, one of the calculated  expression level data sets showed excellent linear correlation with  experimental values, with Pearson coefficient Square R≈0.9
08.20
After double-checking with Robin on previous  sequence data, a mistake in programming was found that altered a few  nucleotides in the protein coding sequence of GFP. The mistake resulted in  unexpectedly low correlation in some sequence libraries but high in some  others. After correcting the mistake, most sequence libraries showed fairly  good correlation.
08.22
Together with Robin, we tried to further validate  the prediction of gene expression levels by fitting parameters β in the  empirical equation: Expression level∝exp{-β△G}.  Theoretical value of β is around 0.4, and fitted (to experimental measurements)  values were between 0.2 and 0.4, implying that some corrections to the  prediction method need to be made.
08.24-08.27
Calculated libraries for RBSs with lengths  from 1nt~7nt, with a total of over 30000 rounds of △G  calculations.
This has consumed most of my laptop’s  computation powers those days!
08.29
Modelled the theoretical fold-change in output  of the AND gate module when input turns from 0 0 to 1 1. Results showed that as  the RBS of the T7ptag coding sequence becomes stronger, the circuit displays  decreasing levels of AND gate property, in accordance with both our expectation  and experiment results.
09.03
After summarizing previous calculations of  sequence libraries, we decided on a cutoff value of 51nt, that is, only  calculating the sequence 51nt upstream and downstream of the start codon, which  provided satisfactory fitting with experiment measurements for most sequence  libraries. Thermodynamic paramters of RNA1995 was also employed to improve  accuracy of prediction.
09.05-09.12
New semester starts. Busy with school affairs.
09.14
Plotted the colormap of the AND Gate phase  diagram with input levels as horizontal and vertical axes. Parameters from work  by Voigt et al. were used. The procedure was repeated on RBSs with different  strengths. This was accomplished by replacing original parameters in the  transfer function with new ones that proportionally changes with △G  values of the RBSs. The results qualitatively described the behaviour of the  AND gate module as RBS strength changes, but not in good accordance with  experiment results using the module constructed by ourselves. Further  experiments that measure input levels more precisely are needed.
09.17-09.20
Establish a quantitative correlation between  fold-change of output upon induction of the AND gate module and △G  values of the RBS, thus enabling the selection of appropriate RBSs for the  circuit to perform particular output functions as determined by dosage response  to ribozyme ligands.
9.23
Planned  the Wiki outline, including documentation of the RBS Calculator platform and  the mathematical modelling for the AND gate and bistable switch modules.
9.25
Discussed  modeling with Professor Ouyang during JC. The bistable switch should be  verified using a stochastic algorithm instead of a steady-state one.
9.26~9.28
Optimized  details to the AND gate modeling using new experimental data: quantification of  the AND gate experimental results by employing the fold-changes in output under  different inducer combinations. The fold-changes were plotted versus parameter a to show its correlation with  translation rates.
10.1~10.4
Wrote  first and second drafts of the modeling part of the wiki. After revising the  text, figures and images were sequentially designed and added to the text. The  wiki draft was reformatted into an academic style and Appendices were written  for more details of our models for the AND gate and bistable switch, ready for  the final round of revision.
 
==click here to his page==
==click here to return==
 
Notebook
>>
Personal Note
>>
Liu Yunpeng
Retrieved from " http://2011.igem.org/Team:Peking_R/Notebook/LYP "
Recent changes
What links here
Related changes
Special pages
My preferences
Printable version
Permanent link
Privacy policy
Disclaimers
