"
Page
Discussion
View source
History
teams
Log in
 
Team:Peking R/Project/RNAToolkit1
From 2011.igem.org
Template:Https://2011.igem.org/Team:Peking R/bannerhidden Template:Https://2011.igem.org/Team:Peking R/back2 Template:Https://2011.igem.org/Team:Peking R/Projectbackground3
无标题文档
Existing Natural Genetic Rheostat
 
TPP-regulated hammerhead ribozyme
 
Theophylline-responsive riboswitch
Extension
 
Engineered group I intron with a theophylline hammerhead ribozyme
 
Adenine hammerhead ribozyme obtained from screening
 
 
 
 
High-performance Genetic Rheostats
Regulated By TPP
Introduction
Genetic rheostat aims to fast, flexibly and precisely regulate gene expression. As you can see here, there are two possible approaches. One is regulating transcription strength using inducible promoters. However, due to the lacking of computational tools to effectively describe DNA-protein interaction, it is not ideal. The other approach is regulating translation strength with riboswitches. Because translation process mainly concerns about nucleotide base-paring and RNA unfolding it can be easily simulated and predicted. So we chose to regulate translation strength with riboswitches.
Riboswitches are structured non-coding RNA elements that control the translation of downstream gene. Hammerhead ribozymes are small self-cleaving RNAs, first discovered in satellite RNAs of plant viruses that catalyze a specific phosphodiester bond isomerization reaction in the course of rolling-circle replication [1] . More recently a full-length hammerhead ribozyme from mansoni is being more frequently utilized for application. As shown in Fig.1-A, this hammerhead ribozyme can be truncated to a minimal, catalytically active motif consisting of (marked in colors) flanking a central core of 15 mostly invariant nucleotides (marked in frame). And the conserved central bases are essential for the hammerhead ribozyme's catalytic activity [1] . Fig.1-B indicates that the secondary structure of the Schistosoma hammerhead ribozyme can be distorted into a uridine turn because of distant loop/bulge interaction which induces changes in stem II while simultaneously unwinding stem I. For the basic catalytic function of hammerhead ribozyme, the active site for self-cleaving of Schistosoma hammerhead ribozyme resides between stem III and stem I, as shown in Fig.1-A.
It has been reported previously that mRNAs encoding enzymes involved in thiamine (vitamin B1) biosynthesis in Escherichia coli can bind derivative without the assistance from protein cofactors [2] . These ligand-binding mRNAs actually possess thiamine or pyrophosphate binding domain, called aptamer, in which the binding event can bring about a conformational change which is important for genetic control. This natural thiamine pyrophosphate (TPP) aptamer can bind to TPP specifically and a defined structure is stabilized. As shown in Fig.1-C, upon addition of TPP, TPP can bind loop in green. Fig.1-D shows the tertiary structure of natural TPP aptamer binding to TPP.
 
 
 
 
Figure.1 The schematic structures of ribozyme and natural TPP aptamer. A) The secondary structure of Schistosoma hammerhead ribozyme. Three base-pairing stems are shaded in colors. The part in yellow represents stem III, which is later modified to be the linker between hammerhead ribozyme and aptamer. The part in blue represents stem II, and the purple and red ones stand for two parts of stem I. The sequence in frame represents for the conserved nucleotides. The red arrow points to the scissile bond. B)The tertiary structure of Schistosoma hammerhead ribozyme. The cyan part indicates the fragment of mRNA after cleavage. The red one indicates the active site for self-cleavage. The linker between hammerhead ribozyme and aptamer is shown in yellow.C) The secondary structure of natural TPP aptamer. TPP can bind to loop in green through non-covalent bond and the part marked in yellow indicates the linker between hammerhead ribozyme and apatamer. D) The tertiary structure of natural TPP aptamer. The three-dimensional segment in blue is TPP, and the yellow part represents the linker between Schistosoma hammerhead ribozyme and natural TPP aptamer. Nucleotides that bind to TPP are shown in green.
In fact, the natural aptamer domain of the TPP riboswitch can be exploited to construct very efficient that regulate gene expression, demonstrated by Markus Wieland et.al. [3] . To couple the natural TPP aptamer riboswitch with Schistosoma hammerhead ribozyme, stem III of Schistosoma hammerhead ribozyme and yellow shaded stem of TPP aptamer in Fig.1-C were modified to construct linker between hammerhead ribozyme and aptamer. The resulting artificial ribozymes functioned with high performance, whose highest fold reached  1000.
Markus Wieland et.al. have created several mutants of the constructed ribozyme-based TPP-responsive artificial ribozyme switches (TPP ribozyme) [3] . We chose two of the mutants in our project, one of which can activate downstream gene expression upon adding TPP, numbered 1.20, and the other would inhibit downstream gene expression when TPP added, numbered 2.5. The secondary structure of TPP ribozyme 1.20 and 2.5 are shown in Fig.2. The lower part of the structure is natural TPP aptamer riboswitch, and the upper part is Schistosoma hammerhead ribozyme. Stem III in green indicates  ribozyme, the pairing nucleotides of which is the only distinction between TPP ribozyme 1.20 and 2.5.
 
 
 
 
Figure.2 Secondary structure of artificial thiamine pyrophosphate (TPP) ribozymes. Left: TPP ribozyme 1.20; Right: TPP  ribozyme 2.5. The natural TPP aptamer domain (blue) is fused to stem III of the Schistosomahammerhead ribozyme. The linker between aptamer and hammerhead ribozyme is shown in green. Stems are indicated by roman numerals; rate-enhancing interaction between stem I and stem II are shown as gray lines; the cleavage site is marked by a red arrow. RBS is shaded in pink and the translation start code (AUG) is shaded in black. The figure was modified from [3] .
The ribosomal binding site (RBS) of TPP ribozyme locates at the extended stem (shaded red in Fig.2).  The Schistosoma hammerhead domain in TPP ribozyme could perform self-cleavage when posed in an appropriate conformation, and upon self-cleavage the RBS would be released from pairing, thus ribosome could get access to RBS and initiate translation of the downstream gene. Though similar in secondary structure, TPP ribozyme 1.20 and 2.5 undergo different mechanisms to regulate the translation of downstream gene. Upon addition of TPP, the aptamer domain would bind to TPP; while TPP ribozyme 1.20 would change to a conformation that is suitable for hammerhead domain to cleave itself, TPP ribozyme 2.5 would undergo a conformational change that would decelerate the self-cleaving rate of hammerhead domain. Therefore, upon adding TPP, TPP ribozyme 1.20 would facilitate the translation of downstream gene, whereas TPP ribozyme 2.5 would decrease the translation strength of downstream gene.
 
 
 
 
Figure.3 Allosteric mechanism of TPP ribozymes. Upon self-cleavage the RBS would be released from pairing, thus ribosome could get access to RBS and initiate translation of the downstream gene. A) Mechanism for TPP ribozyme 1.20. Upon addition of TPP, TPP ribozyme 1.20 adopts a conformation that facilitates the self-cleavage of hammerhead domain. B) Mechanism for TPP ribozyme 2.5. When TPP added, TPP ribozyme 2.5 would change to a conformation that hinders the self-cleavage of hammerhead domain. Red: RBS sequence, blue: natural TPP aptamer, green: linker between aptamer and hammerhead domain, black: Schistosoma hammerhead ribozyme, red arrow: self-cleavage site. The figure is modified from [3] .
 
We designed several experiments to demonstrate that TPP ribozymes are truly modular RNA controllers, independent of sequence context. The plasmids kindly provided by Prof. Hartig's lab contain TPP ribozyme 1.20/2.5 with an upstream T7 promoter, and the downstream coding sequence is a copy of GFP gene, followed by a T7 terminator (Fig. 4-A). We termed these two constructs T7-1.20 and T7-2.5 respectively. To prove that the performance of TPP ribozymes is not relevant to the downstream coding sequence, we inserted first 36 base pairs of the coding sequence from Part BBa_E0040 ahead of the GFP coding sequence in the plasmid (Fig. 4-B). The two constructs were termed 36-1.20/36-2.5 respectively.
 
 
 
 
Figure.4 Scheme of constructs designed to demonstrate the modularity of TPP ribozymes. A) T7-1.20/T7-2.5 consists of T7 promoter, TPP ribozyme, GFP and T7 terminator with native RBS. B) 36-1.20/36-2.5 were constructed by inserting the first 36 base pairs of BBa_E0040 between TPP ribozyme and GFP. C) pBAD-1.20/pBAD-2.5 consists of pBAD promoter, TPP ribozyme, BBa_E0040 and BBa_B0015 with native RBS (AAGGAGAT). D) CI-1.20/CI-2.5 were constructed by adding the first 36 base pairs of CI ahead of coding sequence of E0040. E) TPP-RBS consists of pBAD promoter, native RBS, BBa_E0040 and BBa_B0015.
 
After transforming T7-1.20, T7-2.5, 36-1.20, 36-2.5 into E. coli DH5αcells, the bacteria were characterized in M9 medium with TPP concentration gradient. The result turned out that the corresponding working curves were similar, despite of different downstream coding sequence (Fig.5 & Fig.6, grids and curves in black and in red).
To further investigate whether the performance of TPP ribozymes is influenced by upstream promoter and downsteam terminator, we constructed two plasmids that contain TPP ribozyme 1.20/2.5 with an upstream pBAD promoter, and the downstream coding sequence is a GFP gene from Part BBa_E0040, followed by a terminator (Part:BBa_B0015) (Fig. 4-C). These two constructs were named pBAD-1.20 and pBAD-2.5 respectively. Similarly, we characterized these two constructs, and the working curves were also similar to those of T7-1.20/T7-2.5 (Fig.5 & Fig.6, grids and curves in black and in blue). Another two plasmids were constructed by inserting first 36 base pairs of CI gene ahead of GFP coding sequence in pBAD-1.20/pBAD-2.5 (Fig. 4-D). These are named CI-1.20 and CI-2.5 respectively. The working curves of these two constructs overlapped to a large extent with those of other constructs (Fig.5 & Fig.6, grids and curves in green), further demonstrating that the performance of TPP ribozyme 1.20 and 2.5 was independent of their coding sequence context, rendering them truly modular RNA devices to regulate gene expression.
 
 
 
Figure.5 Working curves of TPP ribozyme 1.20 in different constructs. The activation ratio is fluorescence intensity under given TPP concentrations compared to that of without TPP. Constructed plasmids were transformed into E. coli DH5a cells and characterized in M9 medium with a TPP concentration gradient of 0.001, 0.003, 0.01, 0.03, 0.1, 0.3, 1, 3mM. T7-1.20 and 36-1.20 were induced by 1mM IPTG. pBAD-1.20 and CI-1.20 were induced by 1mM arabinose.
 
 
 
 
 
Figure.6 Working curves of TPP ribozyme 2.5 in different constructs. The inhibition ratio is fluorescence intensity under given TPP concentrations compared to that of without TPP. Constructed plasmids were transformed into E. coli DH5a cells and characterized in M9 medium with a TPP concentration gradient of 0.001, 0.003, 0.01, 0.03, 0.1, 0.3, 1, 3mM. T7-2.5 and 36-2.5 were induced by 1mM IPTG. pBAD-2.5 and CI-2.5 were induced by 1mM arabinose.
 
Additionally, to further confirm that the rise or drop of the working curves in Fig.5 or Fig.6 on different TPP concentrations was indeed the contribution of TPP ribozymes, we constructed a plasmid as control by substituting TPP ribozyme in pBAD-1.20/pBAD-2.5 to the native RBS (AAGGAGAT) of TPP ribozyme, termed TPP-RBS (Fig.4-E). Similar characterization was performed, and the result showed that the fluorescence intensity produced by TPP-RBS fluctuated, yet not significant enough to show a trend to increase or decrease when TPP concentration went up, compared with the obvious fluorescence intensity change produced by pBAD-1.20/pBAD-2.5 (Fig.7). Therefore, we can reach the conclusion that TPP ribozyme 1.20 and 2.5 functioned modularly to regulate downstream gene's translation strength upon different concentrations of TPP.
 
 
 
Figure.7 The fluorescence intensity of TPP-RBS, pBAD-1.20 and pBAD-2.5 under different TPP concentrations. Ordinate axis indicates the fluorescence intensity normalized by cell density. Constructed plasmids were transformed into E. coli DH5a cells and characterized in M9 medium with a TPP concentration gradient of 0, 0.001, 0.003, 0.01, 0.03, 0.1, 0.3, 1, 3mM, with induction by 1mM arabinose.
 
 
Reference:
[1] Monika Martick and William G.Scott.  (2006). Tertiary Contacts Distant from
the Active Site Prime a  Ribozyme for Catalysis. Cell 126, 309-320
[2] Wade Winkler, Ali Nahvi Ronald R. Breaker. (2002). Thiamine  Derivatives Bind Messenger RNAs Directly to Regulate Bacterial Gene Expression.  Nature 419, 952-956
[3] Markus Wieland, Armin Benz, Benedikt Klauser, and Jörg S. Hartig. (2009). Artificial Ribozyme Switches Containing Natural  Riboswitch Aptamer Domains. Angew. Chem. 121, 2753-2756
Retrieved from " http://2011.igem.org/Team:Peking_R/Project/RNAToolkit1 "
Recent changes
What links here
Related changes
Special pages
My preferences
Printable version
Permanent link
Privacy policy
Disclaimers
