<HTML xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en" dir="ltr">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki  ltr ns-0 ns-subject page-Team_Peking_R_Project_RNAToolkit2"><DIV id="globalWrapper"><DIV id="top-section"><DIV id="p-logo"><A href="/Main_Page" title="Main Page">&quot;
	    </A></DIV><DIV id="menubar" class="left-menu noprint"><UL><LI class="selected"><A href="/Team:Peking_R/Project/RNAToolkit2">Page               </A></LI><LI class="new"><A href="/wiki/index.php?title=Talk:Team:Peking_R/Project/RNAToolkit2&amp;action=edit&amp;redlink=1">Discussion               </A></LI><LI><A href="/wiki/index.php?title=Team:Peking_R/Project/RNAToolkit2&amp;action=edit">View source               </A></LI><LI><A href="/wiki/index.php?title=Team:Peking_R/Project/RNAToolkit2&amp;action=history">History               </A></LI><LI style="color:#808080;cursor:default">teams</LI></UL></DIV><DIV class="right-menu noprint" id="menubar"><UL><LI id="pt-login"><A href="/wiki/index.php?title=Special:UserLogin&amp;returnto=Team:Peking_R/Project/RNAToolkit2" title="You are encouraged to log in; however, it is not mandatory [o]" accesskey="o">Log in</A></LI></UL></DIV><DIV id="search-controls" class="noprint"><FORM action="/Special:Search" id="searchform"> </FORM></DIV></DIV><DIV id="content"><H1 class="firstHeading">Team:Peking R/Project/RNAToolkit2</H1><DIV id="bodyContent"><H3 id="siteSub" class="noprint">From 2011.igem.org</H3><P><A href="/wiki/index.php?title=Template:Https://2011.igem.org/Team:Peking_R/bannerhidden&amp;action=edit&amp;redlink=1" class="new" title="Template:Https://2011.igem.org/Team:Peking R/bannerhidden (page does not exist)">Template:Https://2011.igem.org/Team:Peking R/bannerhidden</A><A href="/wiki/index.php?title=Template:Https://2011.igem.org/Team:Peking_R/back2&amp;action=edit&amp;redlink=1" class="new" title="Template:Https://2011.igem.org/Team:Peking R/back2 (page does not exist)">Template:Https://2011.igem.org/Team:Peking R/back2</A><A href="/wiki/index.php?title=Template:Https://2011.igem.org/Team:Peking_R/Projectbackground4&amp;action=edit&amp;redlink=1" class="new" title="Template:Https://2011.igem.org/Team:Peking R/Projectbackground4 (page does not exist)">Template:Https://2011.igem.org/Team:Peking R/Projectbackground4</A><TITLE>无标题文档</TITLE></P><DIV id="apDiv1"><P class="notbookmaintitle" align="center"> </P><P> </P><P> </P><P> </P><P> </P><P> </P><P align="left" class="TITLE2"><STRONG>High-performance Theophylline </STRONG></P><P align="left" class="TITLE2"><STRONG>Responsive Riboswitches</STRONG></P><P class="mainbody">Riboswitches are structural domains embedded within the non-coding sequences of certain mRNAs' 5'untranslated regions. Riboswitches perform as RNA-based genetic control elements that regulate gene expression in a ligand-dependent manner through direct, small molecule-RNA interactions without help of proteins. Over past several years, a number of natural riboswitches have been reported in literatures, sensing small molecules like coenzyme B1 and B12, FMN, SAM, lysine, guanine, adenine, TPP and theophylline<A href="#r1"> [1]</A>. Riboswitches are typically comprised of an aptamer domain, which recognizes a specific ligand with high specificity and affinity as a precise binding pocket<A href="#r2"> [2]</A>. Aptamers' complex formation is often accompanied by structural changes and/or decreased rigidity. Such complexes thus become an obstacle for ribosome binding or movement by utilizing Watson-Crick base pairing to seal ribosomal binding  site (RBS) in the absence of the ligand. As a result, gene expression will be  switched off in the absence or presence of the ligand, but proceeds in its presence, and vice versa. In another word, the ligand-induced allosteric interconversion between structural states causes altered translation strength. </P><P class="mainbody">In our project, we attempted to develop genetic rheostats first. Genetic rheostat candid-ates must meet several criteria: firstly, they should possess a relatively plain dose-response curve, thus to allow a wide range for precise translation strength modulation; secondly, their cognate ligands should  be genetically and biochemically orthogonal to the host cell, in our case, <EM>E.coil</EM> cell, as much as possible. </P><P class="mainbody">Theophylline responsive riboswitches have been widely studied in recent years. Theo-phylline is a small molecule which is passable through bacterial cytomembrane. It is nei-ther a precursor of metabolic macromolecules nor a cofactor of enzymes. In addition, it is not a component of LB or M9 liquid medium, consistent with the fact that theophylline is not an essential nutrient to bacteria. Although theophylline has moderate toxicity that causes obvious bacterial growth inhibition in logarithmic phase with 15mM theophylline in LB medium, this theophylline responsive riboswitch shows an approving regulation curve under 10mM theophylline concentration with negligible growth inhibition in M9 liquid medium. (Fig.1)</P><TABLE width="200" border="0" cellspacing="0" cellpadding="0"><TBODY><TR><TH scope="col"> </TH><TH scope="col"> </TH></TR><TR><TD> </TD><TD><P class="picturemark"><STRONG>Figure.1 </STRONG>Growth curves of <EM>E.coli</EM> under treatments of different theophylline concentrations. Add theophylline into the medium after incubating at 37 degrees in shaker for 8 hours, consistent with induce experiment. Under 10mM theophylline concentration, there is negligible growth inhibition.</P><P class="picturemark"> </P></TD><TD> </TD></TR></TBODY></TABLE><P class="mainbody">We adopted the original theophylline responsive riboswitch, named Parental 1G1 (P1G1) in our project, designed by Beatrix Suess on Nucleic Acids  Research, as a candidate of our genetic rheostat. Stereo representation of the three-dimensional structure of theophylline aptamer is schematized in Fig.2. </P><TABLE width="200" border="0" cellspacing="0" cellpadding="0"><TBODY><TR><TH scope="col"> </TH><TH scope="col"> </TH></TR><TR><TD> </TD><TD><P><STRONG>Figure.2</STRONG> Schematic of the theophylline aptamer structure. Red: theophylline; yellow: theophylline binding  site; blue: linker. </P></TD><TD> </TD></TR></TBODY></TABLE><P class="mainbody">Theophylline responsive riboswitch P1G1 integrates the theophylline aptamer with a helical communication module based on a ligand–dependent one-nucleotide slipping mechanism<A href="#r3"> [2]</A>. This structural element lies at a position just interfering with translation in the non ligand-bound form. Addition of the ligand then shifts the inhibitory secondary structure element that is closed to RBS to a distance that permits resulted efficient translation. This slippage mechanism to control gene expression makes it different from natural riboswtiches those are based on sequenstration or antitermination<A href="#r4"> [3]</A>.</P><P class="mainbody">We constructed a device via PCR and standard assembly. It composes of RNA controller P1G1 which contains original RBS sequence and GFP as reporter gene (Fig.3). A graph of predicted secondary structure (by mFold) of translation initiation region with introduced regulatory element P1G1 containing theophylline aptamer (boxed) and stem-loop structure is shown in Fig.4. The  proposed theophylline-mediated slipping mechanism of the bridge domain is shown schematically as well. </P><TABLE width="200" border="0" cellspacing="0" cellpadding="0"><TBODY><TR><TH scope="col"> </TH><TH scope="col"> </TH></TR><TR><TD> </TD><TD><P class="picturemark"><STRONG>Figure.3</STRONG> Scheme of constructs designed to demonstrate the modularity of theophylline riboswitches. P1G1 composes of pBAD promoter, theophylline responsive riboswitch, GFP  BBa_E0040 and Terminator BBa_B0015 with original RBS (UGGA). 1G1 composes of pBAD promoter, theophylline responsive riboswitch, GFP BBa_E0040 and Terminator BBa_B0015 with optimized RBS (AGGAGGU).</P></TD><TD> </TD></TR></TBODY></TABLE><P> </P><TABLE width="200" border="0" cellspacing="0" cellpadding="0"><TBODY><TR><TH scope="col"> </TH><TH scope="col"> </TH></TR><TR><TD> </TD><TD><P><STRONG>Figure.4</STRONG> Predicted secondary structure of P1G1 in the absence and presence of theophylline. Theophylline aptamer is boxed. 1nt slipping mechanism is shown in yellow. UGGA is shown in red as RBS sequence. AUG initiation codon is in black.</P></TD><TD> </TD></TR></TBODY></TABLE><P class="mainbody">We used GFP fluorescence intensity to present the ability of P1G1 riboswitches to modulate downstream gene expression. GFP fluorescence intensity measured and with increasing concentrations of theophylline is shown in Fig.5. P1G1 displays a 6-fold activation ratio of the absence to presence of 10mM theophylline (Fig.6). Inspection of the highest translation rate reveals that although high concentration of ligand fully motivates translation, the GFP fluorescence intensity is still low, indicating relatively low translation strength. However, we expect wider translation strength available for theophylline riboswitch. Thus, the performance of P1G1 needs to be genetically optimized. </P><TABLE width="200" border="0" cellspacing="0" cellpadding="0"><TBODY><TR><TH scope="col"> </TH><TH scope="col"> </TH></TR><TR><TD> </TD><TD><P class="picturemark"><STRONG>Figure.5</STRONG> Dose responses of P1G1 riboswitch and 1G1 riboswitch. (A) Photo shows GFP fluorescence intensity controlled by gradients of theophylline concentration. The most left well denotes induction in the absence of theophylline. GFP fluorescence intensity increases with the concentration of theophylline. (B) Dose responsive curves of 1G1 and P1G1 with a theophylline concentration gradient in the presence of arabinose. RBS sequences and corresponding dG are also presented in the graph. Arrow indicates the global increase of translation strength variation range. </P><P> </P></TD><TD> </TD></TR></TBODY></TABLE><TABLE width="200" border="0" cellspacing="0" cellpadding="0"><TBODY><TR><TH scope="col"> </TH><TH scope="col"> </TH></TR><TR><TD> </TD><TD><P><STRONG>Figure.6</STRONG> The ability of P1G1 and 1G1 to activate translation strength under induction by a theophylline  concentration gradient. </P></TD><TD> </TD></TR></TBODY></TABLE><P class="mainbody">On account of the circumscribed property of genetic rheostats, insertion of genetic rheostat probably makes even the highest expression level lower than sole RBS sequence upstream AUG start codon. Insertion of genetic rheostat could expediently down-regulate downstream gene translation, but it is impossible to up-regulate downstream gene translation rate up to or exceed that of sole RBS. Therefore, we mutated the RBS sequence of P1G1 to AGGAGGU. This is predicted to provide a more flexible absolute translation rate with a fixed modulating range. </P><P class="mainbody">RBS sequence is considerably important to mRNA translation initiation. The length of consensus RBS sequence is 7nt. Consensus RBS sequence which entirely pairs with anti-Ribosome-Binding-Site sequence is considered to be the most robust one. The less base-pairing between RBS and anti-RBS, the weaker the RBS is. A strong RBS is thought to proportionally shift the modulating range/activation ratio of RNA controllers without changing the width to a higher level, as they are turned gradually from completely OFF state to fully ON state in the  presence of ligand. Therefore, we optimized original RBS sequence of P1G1 to AGGAGGU, the consensus RBS sequence predicted to initiate high rate of translation according to the RBS calculator. Simultaneously, as we positioned genetic rheostat 1G1 upstream of GFP coding sequence by standard assembly, the spacing between RBS and AUG strat codon is ACUAG (scar generated during standard assembly). The modification of RBS sequence and spacing between RBS and AUG start codon had inappreciable influence on the core secondary stem-loop structure of functional original riboswitch P1G1 and almost preserves the quondam base-pairing region.  Stem-loop structure of 1G1 is schematized in Fig.7. 1G1 shares the same slipping mechanism with P1G1.</P><P> </P><TABLE width="200" border="0" cellspacing="0" cellpadding="0"><TBODY><TR><TH scope="col"> </TH><TH scope="col"> </TH></TR><TR><TD> </TD><TD><P><STRONG>Figure.7 </STRONG>Predicted secondary structure of 1G1 in the absence and presence of theophylline. Theophylline aptamer is boxed. 1nt slipping mechanism is shown in yellow. UGGA is shown in red as RBS sequence. AUG start codon is in black. </P></TD><TD> </TD></TR></TBODY></TABLE><P> </P><P class="mainbody">Dose response curve is shown in Fig.5. 1G1 displays a 10-fold activation ratio of the absence to presence of 10mM theophylline (Fig.6). However, under a concentration of 5mM theophylline, 1G1 shares similar activation ratio with P1G1, consistent with the prediction that alteration of RBS provides more flexible absolute translation strength with a fixed regulative range. Inspection of these two version of theophylline riboswitch reveals that 1G1 with the highest level of expression in the presence of theophylline not only possesses a longer RBS sequence (AGGAGGU) than P1G1, but also a more optimal spacing (5nt upstream to AUG start codon). Comparison of basal level of P1G1 and 1G1 suggests that 1G1 holds a more significant leakage in the absence of theophylline than P1G1, coincident with the holistic high expression of 1G1. According to our experiment data, engineered riboswitch possesses a more optimal absolute translation rate range, consistent with the prediction that a stronger RBS provides the genetic rheostat with a stronger regulation capability. </P><P class="mainbody">In summary, our semi-rational method to modify genetic rheostat (in our case, theophylline riboswitch) offers a promising clue for the optimization of performance of RBS-related riboswitches. </P><P> </P><P align="center" class="TITLE2"> </P><P class="mainbody"><SPAN class="Reference">Reference:</SPAN></P><P>[1]Nahvi,A., Sudarsan,N., Ebert,M.S., Zou,X., Brown,K.L. and Breaker  ,R.R.(2002) Genetic conrrol by a metabolite binding mRNA. <EM>Chem. Biol.</EM>, <STRONG>9</STRONG>, 1043-1049.
  [2]Beatrix Suess, Barbara Fink, Christian Berens, Régis Stentz and Wolfgang Hillen. (2004) A theophylline responsive riboswitch based on helix  slipping controls gene expression <EM>in vivo</EM>. <EM>Nucleic Acids Research</EM>, <STRONG>32</STRONG>, 1610-1614. 
  [3]Sean A. Lynch, Shawn K. Desai, Hari Krishna Sajja, and Justin P. Gallivan. (2006) A High-Throughput Screen for Synthetic Riboswitches Reveals Mechanistic Insights into Their Function. <EM>Chem. Biol.</EM>, <STRONG>14</STRONG>, 173-184.</P><P align="left" class="mainbody"> </P></DIV><DIV id="apDiv2"><TABLE width="673" border="0" cellspacing="0" cellpadding="0"><TBODY><TR><TH height="27" colspan="2" align="left" class="exist1" scope="col">Existing Natural Genetic Rheostat</TH></TR><TR><TD width="41" height="27" align="left"> </TD><TD width="447" align="left" class="TPP1"><A href="https://2011.igem.org/Team:Peking_R/Project/RNAToolkit1">TPP-regulated hammerhead ribozyme</A></TD></TR><TR><TD height="28" align="left"> </TD><TD align="left" class="TPP2">Theophylline-responsive riboswitch</TD></TR><TR><TD colspan="2" align="left" class="exist1"><P>Extension</P></TD></TR><TR><TD height="27" align="left"> </TD><TD align="left" class="TPP1"><A href="https://2011.igem.org/Team:Peking_R/Project/RNAToolkit3">Engineered group I intron with a theophylline hammerhead ribozyme</A></TD></TR><TR><TD height="33" align="left"> </TD><TD align="left" class="TPP1"><A href="https://2011.igem.org/Team:Peking_R/Project/RNAToolkit4">Adenine hammerhead ribozyme obtained from screening</A></TD></TR></TBODY></TABLE></DIV><DIV class="printfooter">
Retrieved from &quot;<A href="http://2011.igem.org/Team:Peking_R/Project/RNAToolkit2">http://2011.igem.org/Team:Peking_R/Project/RNAToolkit2</A>&quot;</DIV></DIV></DIV><DIV id="footer-box" class="noprint"><DIV id="footer"><UL id="f-list"><LI id="t-recentchanges"><A href="/Special:RecentChanges" title="Recent changes">Recent changes</A></LI><LI id="t-whatlinkshere"><A href="/Special:WhatLinksHere/Team:Peking_R/Project/RNAToolkit2" title="List of all wiki pages that link here [j]" accesskey="j">What links here</A></LI><LI id="t-recentchangeslinked"><A href="/Special:RecentChangesLinked/Team:Peking_R/Project/RNAToolkit2" title="Recent changes in pages linked from this page [k]" accesskey="k">Related changes</A></LI><LI id="t-specialpages"><A href="/Special:SpecialPages" title="List of all special pages [q]" accesskey="q">Special pages</A></LI><LI><A href="/Special:Preferences">My preferences</A></LI></UL></DIV><DIV id="footer"><UL id="f-list"><LI id="t-print"><A href="/wiki/index.php?title=Team:Peking_R/Project/RNAToolkit2&amp;printable=yes" title="Printable version of this page [p]" accesskey="p">Printable version</A></LI><LI id="t-permalink"><A href="/wiki/index.php?title=Team:Peking_R/Project/RNAToolkit2&amp;oldid=259933" title="Permanent link to this revision of the page">Permanent link</A></LI><LI id="privacy"><A href="/2011.igem.org:Privacy_policy" title="2011.igem.org:Privacy policy">Privacy policy</A></LI><LI id="disclaimer"><A href="/2011.igem.org:General_disclaimer" title="2011.igem.org:General disclaimer">Disclaimers</A></LI></UL></DIV></DIV></DIV></BODY></HTML>