<HTML xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en" dir="ltr">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki  ltr ns-0 ns-subject page-Team_Peking_R_Project_RNAToolkit4"><DIV id="globalWrapper"><DIV id="top-section"><DIV id="p-logo"><A href="/Main_Page" title="Main Page">&quot;
	    </A></DIV><DIV id="menubar" class="left-menu noprint"><UL><LI class="selected"><A href="/Team:Peking_R/Project/RNAToolkit4">Page               </A></LI><LI class="new"><A href="/wiki/index.php?title=Talk:Team:Peking_R/Project/RNAToolkit4&amp;action=edit&amp;redlink=1">Discussion               </A></LI><LI><A href="/wiki/index.php?title=Team:Peking_R/Project/RNAToolkit4&amp;action=edit">View source               </A></LI><LI><A href="/wiki/index.php?title=Team:Peking_R/Project/RNAToolkit4&amp;action=history">History               </A></LI><LI style="color:#808080;cursor:default">teams</LI></UL></DIV><DIV class="right-menu noprint" id="menubar"><UL><LI id="pt-login"><A href="/wiki/index.php?title=Special:UserLogin&amp;returnto=Team:Peking_R/Project/RNAToolkit4" title="You are encouraged to log in; however, it is not mandatory [o]" accesskey="o">Log in</A></LI></UL></DIV><DIV id="search-controls" class="noprint"><FORM action="/Special:Search" id="searchform"> </FORM></DIV></DIV><DIV id="content"><H1 class="firstHeading">Team:Peking R/Project/RNAToolkit4</H1><DIV id="bodyContent"><H3 id="siteSub" class="noprint">From 2011.igem.org</H3><P><A href="/wiki/index.php?title=Template:Https://2011.igem.org/Team:Peking_R/bannerhidden&amp;action=edit&amp;redlink=1" class="new" title="Template:Https://2011.igem.org/Team:Peking R/bannerhidden (page does not exist)">Template:Https://2011.igem.org/Team:Peking R/bannerhidden</A><A href="/wiki/index.php?title=Template:Https://2011.igem.org/Team:Peking_R/back2&amp;action=edit&amp;redlink=1" class="new" title="Template:Https://2011.igem.org/Team:Peking R/back2 (page does not exist)">Template:Https://2011.igem.org/Team:Peking R/back2</A><A href="/wiki/index.php?title=Template:Https://2011.igem.org/Team:Peking_R/Projectbackground6&amp;action=edit&amp;redlink=1" class="new" title="Template:Https://2011.igem.org/Team:Peking R/Projectbackground6 (page does not exist)">Template:Https://2011.igem.org/Team:Peking R/Projectbackground6</A><TITLE>无标题文档</TITLE></P><DIV id="apDiv1"><P class="notbookmaintitle" align="center"> </P><P> </P><P> </P><P> </P><P> </P><P> </P><P align="left" class="TITLE22"><STRONG>An ON-OFF Selection for Genetic Rheostat Evolving</STRONG></P><P class="mainbody">It is easy to understand that the number of  sites we can control depends on the genetic rheostats we have. As the available genetic rheostats are limited and existing gene switches that  function in living cells are challenging due to the inherent complexity of themselves  and complex intracellular milieu, it is necessary to develop a general method to construct new genetic rheostats for the purpose of optimizing advanced  RNA-based regulation. Though several strategies for the design or construction of  artificial genetic rheostats have been developed, they are either too limited or too complicated. We aim to develop a novel method of genetic rheostat design and construction, which has general interest and can be widely used. As there have already been diversities of RNA sequences binding to various kinds of ligands, we propose that, by swapping these aptamers with those of existing regulatory RNA, more novel genetic rheostats regulated by various ligands can be implemented. Reassuringly, Jörg S. Hartig et al. presented that it is possible to design novel artificial riboswitches by coupling a natural aptamer domain to a self-cleaving ribozyme <A href="#r1">[7]</A>. On the other hand, Yohei Yokobayashi et al., established an efficient platform for genetic switch selection<A href="#r2"> [1]</A>. Inspired by these two methodologies, we combined them to develop a standard  method to synthesize new genetic rheostats.</P><P class="mainbody">As mentioned above, Jörg S. Hartig et al. demonstrated that the natural aptamer domain could be used to construct artificial ribozyme-based genetic switches<A href="#r3"> [7]</A>.  First of all, they coupled the thiM aptamer domain of thethiamine pyrophosphate  (TPP) riboswitch (from <EM>E. coli</EM>) to the stem III of a fast- cleaving hammerhead ribozyme(from <EM>Schistosoma mansonii</EM>). Hammerhead ribozyme is one of the simplest formats of ribozymes with clearly depicted mechanism. Its stem I/stem II tertiary interaction leads to the closure of the ribozyme and stabilizes the active conformation, accompanying the cis-cleavage reaction. Therefore, the  binding of TPP molecule to its aptamer stimulates the liberation of the ribosome binding site (RBS) by TPP-dependent, fast cleavage. Next, to erase the  trade-off between the specificity of the aptamer and the cleavage activity of the hammerhead ribozyme as much as possible, they randomized six nucleotides of the region connecting the aptamer and the ribozyme to  perform an in vivo screeningto select functional RNA switches. After screened  4000 clones, they found an expected riboswitch with high performance.</P><P class="mainbody">Besides, Yohei Yokobayashi et al. established an efficient selection and screen platform for genetic switches. The most critical  component in this platform is a fusion protein in which tetA, which encodes a  tetracycline/H+ antiporter as both a positive and a negative selection marker,  was fused with a green fluorescent protein (gfpuv) via a flexible peptide  linker. This fusing protein is used as a dual selection and screening marker  for laboratory evolution of genetic controller: when the genetic switch (in our case, the riboswitches) is open, TetA will be expressed and endow resistance to  tetracycline (ON selection) to bacterial cell. However, in turn the  overexpression of this membrane-bound protein (TetA-GFP) makes cells more sensitive to toxic metal salts such as NiCl2 (OFF selection). </P><P class="mainbody">In details, this dual genetic selection  process includes four steps (Fig.1A): Firstly, a genetic switch mutant library  was established; Secondly, after incubated with ligand under nonselective conditions to stabilize TetA-GFPuv expression, ON cells were selected by addition of tetracycline; Thirdly, the selected cells were grown under  nonselective conditions to readjust TetA-GFPuv expression levels and then OFF cells are selected by addition of NiCl2. After one round of the genetic selection, selected individual bacterial clones are inoculated and  quantitatively assayed for the gene switch activity by measuring GFPuv ﬂuorescence.</P><TABLE width="200" border="0" cellspacing="0" cellpadding="0"><TBODY><TR><TH scope="col"> </TH><TH scope="col"> </TH></TR><TR><TD> </TD><TD><P align="left" class="picturemark"><STRONG>Figure  1.  </STRONG>Dual genetic selection process of gene  switches.<STRONG> (a)</STRONG> Schematic illustration  of the dual genetic selection process. a)A gene switch library is established in <EM>E. coli</EM> b) The library cells are  grown for 8 h under nonselective-ON conditions (e.g. with ligand) in M9 medium to stabilize TetA-GFPuv expression. c) ON selection: ON cells are selected by  addition of tetracycline d) The selected cells are washed once and grown for 8 h under nonselective-OFF conditions (e.g. without ligand) in M9 medium to  readjust TetA-GFPuv expression levels. e)OFF selection: OFF cells are selected  by addition of NiCl2. Selected cells are identiﬁed by GFPuv ﬂuorescence  measurement. <STRONG>(b)</STRONG> Mock selections of  TPP riboswitches from model libraries. GFPuv ﬂuorescence of the unselected (white bars) and selected (dark bars) cell populations were  measured under nonselective conditions in the absence(OFF) and presence (ON,100 mM) of TPP. Two independent selections were performed for each dilution. </P></TD><TD> </TD></TR></TBODY></TABLE><P class="mainbody">With this ON-OFF selection, researchers can  remarkably simplify the selection process by omitting plasmid isolation step  and following transformation. According to their data, single-round (sequential  positive and negative selections) enrichment efficiencies of 7000 were observed  in mock selections of model libraries containing functional riboswitches in  liquid culture. (In their mock selection, 96 clones from the 10–4  dilution library (–ThiMwt cells were diluted in the background of other cells)  selection were randomly tested by tetracycline-growth assay. 75% of them were  found to exhibit the phenotype consistent with –ThiMwt. This equates to a 7500-fold enrichment in a single cycle of dual genetic selection.) </P><P class="mainbody">We applied this selection platform to develop our translation tuner design and construction methodology. </P><P class="mainbody">To verify this platform, we conducted a mock selection, in which cells bearing a functional TPP riboswitch (+ThiM#2) were diluted in the background of a mixture of cells bearing always-ON and always-OFF riboswitch. After performing one round selection, the selected and unselected cells are cultured in the presence (100 μM) and absence of thiamine under nonselective conditions followed by ﬂuorescence measurement (Fig.1B). Near half of the survived cells exhibited thiamine-up-regulate GFPuv expression while none of the cells without selection showed the same response, in accordance with the results of Yohei Yokobayashi et al. <A href="#r4">[1]</A>
    After the validation of the genetic switch selection platform, we then selected an adenine aptamer to conduct aptamer swapping, as a proof of concept. As is known, the specificity of a genetic rheostate is decided by the aptamer to large extent. Besides, the pattern of aptamer's conformational change after ligand binding should be taken into consideration as it might influent the cleavage of hammerhead ribozyme. The adenine aptamer was first found in <EM>V. vulniﬁcus </EM><A href="#r5">[2]</A> and shares the similar fashion of mechanism to TPP aptamer (Fig.2)<A href="#r6"> [3]</A>.</P><TABLE width="200" border="0" cellspacing="0" cellpadding="0"><TBODY><TR><TH scope="col"> </TH><TH scope="col"> </TH></TR><TR><TD> </TD><TD align="left"><P class="picturemark"><STRONG>Figure   2.</STRONG> Sequence and conformational change of adenine aptamer<STRONG> (a) </STRONG>Sequence and secondary structure of the adenine-sensing aptamer we used. The ligand-binding pocket is indicated by roman numberals. Linker domain is boxed in green.<STRONG> (b) </STRONG>The 3D structure of the adenine aptamer with adenine, green: linker domain; blue: aptamer; yellow: adenine  binding site. <STRONG>(c)</STRONG> the pattern of  TPP-sensing aptamer's conformational change after TPP binding. If TPP is not  present (GENE ON), an antisequestor hairpin (blue) forms that does not interfere with gene expression. When binding TPP (GENE OFF), P1 is stabilized, thereby allowing a hairpin to form that sequesters the Shine-Dalgarno sequence  (SD) and prevents the ribosome from translating the ORF. (figure is from Kenneth F Blount &amp; Ronald R Breaker<A href="#R7"> [4]</A>)<STRONG> (d)</STRONG> the pattern of adenine-sensing aptamer's conformational change after adenine binding. When binding adenine (ON state), stem P1 (green) developed. Therefore formation of the terminator stem-loop is prevented and the SD sequence and the AUG codon are accessible to the ribosome. If adenine is not present (OFF state), the transcription terminator stem-loop is stabilized and prevents the ribosome from translating the ORF</P></TD><TD> </TD></TR></TBODY></TABLE><P class="mainbody">After determining the core sequence of adenine aptamer<A href="#r8"> [5][6]</A>, we introduced this aptamer into stem III of wildtype hammerhead and randomized six to twelve nucleotides of the linker domain (Fig. 3 A,B). Then we conducted selection and screening on this library. Cells were selected under the optimized selection conditions (75 μg/ml tetracycline, 1.0mM NiCl2, 0.1mg/ml adenine <A href="#r9">[2]</A>). Selecting from about 1500 clones and screening 50 selected clones via the protocol described above, we obtained four adenine up-regulate hammerhead ribozymes and further measurement recommend one with excellent performance. Most interesting is that the sequencing result show that three base-pair within the linker near the aptamer domain are the same as the these in the wild tyoe adenine aptamer (Tab.1).Then we introduced this ribozyme upstream GFP coding sequence via PCR and measure the performance of this ribozyme under induction with various concentrations of adenine in vivo in M9 medium (Fig.4). As we speculated that high concentration of adenine might be harmful to the growth of bacteria, we measured the growth curve of <EM>E.coli</EM> with 0.05mg/mL and 0.1mg/mL of  adenine added in the culture to determine the cytotoxity of adenine.(Fig.5) There was apparent difference between the growth of control and that of chemically induced sample with 0.1mg/mL adenine and measurement of fluorescence  showed a significant decrease when the concentration of adenine exceeded  0.1mg/mL. However, in the presence of 0.05mg/mL adenine when the translation  strength of reporter gene reaches relatively high level, the growth of control and that of chemically induced sample is similar. So the working-concentration adenine  won't influence <EM>E.coli</EM>'s growth.</P><TABLE width="200" border="0" cellspacing="0" cellpadding="0"><TBODY><TR><TH scope="col"> </TH><TH scope="col"> </TH></TR><TR><TD> </TD><TD><P class="picturemark"><STRONG>Figure  3</STRONG>. Representation of artificial adenine riboswitche. The natural adenine aptamer domain (blue) was fused to stem III of the Schistosoma mansonii hammerhead ribozyme. Stems are indicated with roman numerals; rate-enhancing  stem I/stem II interactions are shown as gray lines; the cleavage site is  marked by a red arrowhead. The extended stem I of the ribozyme masks the  Shine–Dalgarno sequence (red). Upon self-cleavage of the activated ribozyme,  the ribosome-binding site is liberated, and gene expression is turned on. <STRONG>(a)</STRONG> Sequence of the artificial adenine-responsive  riboswitches; red: Shine–Dalgarno sequence, blue: adenine aptamer, green box:  nucleotide positions randomized for the screening of adenine-responsive  sequences <STRONG>(b)</STRONG> adenine-activated  riboswitches increase ribozyme cleavage and hence gene expression upon the external  addition of adenine.  </P></TD><TD> </TD></TR></TBODY></TABLE><P> </P><TABLE width="200" border="0" cellspacing="0" cellpadding="0"><TBODY><TR><TH scope="col"> </TH><TH scope="col"> </TH><TH scope="col"> </TH></TR><TR><TD> </TD><TD><P class="picturemark"><STRONG>Figure   4</STRONG>D<STRONG>Figure 4.</STRONG>ependence of gene expression on adenine concentration. Left: ﬂuorescence image of cells in the  absence and presence ( 0.05 mg/ml ) of adenine. Right: gene expression level responses to various  dosage of adenine. The eGFP was used as a reporter.</P></TD><TD> </TD></TR></TBODY></TABLE><P> </P><TABLE width="200" border="0" cellspacing="0" cellpadding="0"><TBODY><TR><TH scope="col"> </TH><TH scope="col"> </TH></TR><TR><TD> </TD><TD><P class="picturemark"><STRONG>Figure  5</STRONG>  Growth curves for <EM>E. coli</EM> in the presence of varying concentrations of adenine. <EM>E.coli</EM>was grown in M9 minimal medium containing  0.1% casamino acid and 0.8% glycerol as the carbon source supplemented with 50  mg/ml carbenicillin and 0.05 or 0.1 mg/mL adenine. </P></TD><TD> </TD></TR></TBODY></TABLE><P align="left"><STRONG>Table 1</STRONG> The sequence of linker of the wild type and four selected ribozyme </P><DIV align="center"><TABLE border="1" cellspacing="0" cellpadding="0"><TBODY><TR><TD width="121" valign="top"><P> </P></TD><TD width="263"><P align="center">Sequence(5'-&gt;3')</P></TD><TD width="184" valign="top"><P>Activation fold (ON/OFF)</P></TD></TR><TR><TD width="121" valign="top"><P>     Wild type</P></TD><TD width="263"><P align="center">TTC<SPAN class="ata">ATA</SPAN>…(aptamer)...<SPAN class="ata">TAT</SPAN>GAA</P></TD><TD width="184" valign="top"><P align="center">2.5<A href="#r13"> [8] </A></P></TD></TR><TR><TD width="121" valign="top"><P>      NO.1.1</P></TD><TD width="263"><P align="center">CCC<SPAN class="ata">ATA</SPAN>…(aptamer)...<SPAN class="ata">TAT</SPAN>ATG</P></TD><TD width="184" valign="top"><P align="center">4</P></TD></TR><TR><TD width="121" valign="top"><P>      NO.1.4</P></TD><TD width="263"><P align="center">CAA<SPAN class="ata">ATA</SPAN>…(aptamer)...<SPAN class="ata">TAT</SPAN>CTT</P></TD><TD width="184" valign="top"><P align="center">2.5*</P></TD></TR><TR><TD width="121" valign="top"><P>      NO.2.1</P></TD><TD width="263"><P align="center">CTG<SPAN class="ata">ATA</SPAN>…(aptamer)...<SPAN class="ata">TAT</SPAN>GTA</P></TD><TD width="184" valign="top"><P align="center">1.5*</P></TD></TR><TR><TD width="121" valign="top"><P>      NO.2.2</P></TD><TD width="263"><P align="center">ATG<SPAN class="ata">ATA</SPAN>…(aptamer)...<SPAN class="ata">TAT</SPAN>TTA</P></TD><TD width="184" valign="top"><P align="center">1.5</P></TD></TR></TBODY></TABLE></DIV><P class="mainbody">*result from measuring the fluorescence of tetA-GFP， two independent measurements were made for each clone.
    The successful construction of the adenine up-regulate genetic rheostat reflects the potential of our developed method. It is easily to imagine that various high-performance aptamers can be exploited to synthesize genetic rheostat with different ligands specificity via this standard method.</P><P class="mainbody">This method possesses general interest, however, there are some weak points in it. The construction of the library is promptly, but the selection might be a long process. Additionally, the performance of the new genetic rheostat highly depends on the character of the aptamer. We also utilized an aptamer responsive to 3-methylxanthine to synthesize corresponding genetic rheostat, but we did not get an effective genetic rheostat probably because that the binding affinity of this aptamer to its  ligand is relatively low. </P><P> </P><P class="mainbody"><SPAN class="Reference">Reference:</SPAN></P><OL><P class="mainbody">[1] Yohei Yokobayashi et al.(2009) An  efficient platform for genetic selection and screening of gene switches in  Escherichia coli. Nucleic Acids Research. 37:NO.5
      [2] Maumita Mandal &amp; Ronald R  Breaker.(2004)Adenine riboswitches and gene activation by disruption of a  transcription terminator. Nature structural&amp; molecular biology 
      [3] Alexander Serganov and Dinshaw J. Patel.  (2007). Ribozymes, riboswitches and beyond: regulation of gene expression  without proteins. Nature. 8:776-790
      [4] Kenneth F Blount &amp; Ronald R Breaker  (2006) Riboswitches as antibacterial drug targets, nature biology, 24:1558-1564
      [5]Jane N. Kim and Ronald R. Breaker.  (2008). Purine sensing by riboswitches, Biol. Cell. 100, 1–11
      [6]Jean-FranÇois Lemay and Daniel A.  Lafontaine.(2007) Core requirements of the adenine riboswitch aptamer for  ligand binding. RNA. 13: 339-350
      [7]Jörg S. Hartig et al. (2008)  Artificial Ribozyme Switches Containing Natural Riboswitch Aptamer Domains. Angew.  Chem. 121: 2753 –2756
      [8] Renate Rieder, Kathrin Lang, Dagmar  Graber, and Ronald Micura, (2007) Ligand-Induced Folding of the Adenosine  Deaminase A-Riboswitch and Implications on Riboswitch Translational Control,  ChemBioChem 8: 896 – 902
      [9] Pascale Legault et al. (2010)  Riboswitch structure: an internal residue mimicking the purine ligand. Nucleic  Acids Research. 6:2057–2068
      [10] Eric Masse, Daniel A. Lafontaine et  al. (2011), Comparative Study between Transcriptionally- and  Translationally-Acting Adenine Riboswitches Reveals Key Differences in  Riboswitch Regulatory Mechanisms, PLoS Genetics, 7: e1001278</P></OL><P align="left" class="TITLE21"> </P><P align="left" class="mainbody"> </P></DIV><DIV id="apDiv2"><TABLE width="673" border="0" cellspacing="0" cellpadding="0"><TBODY><TR><TH height="27" colspan="2" align="left" class="exist1" scope="col">Existing Natural Genetic Rheostat</TH></TR><TR><TD width="41" height="27" align="left"> </TD><TD width="447" align="left" class="TPP1"><A href="https://2011.igem.org/Team:Peking_R/Project/RNAToolkit1">TPP-regulated hammerhead ribozyme</A></TD></TR><TR><TD height="28" align="left"> </TD><TD align="left" class="TPP2"><A href="https://2011.igem.org/Team:Peking_R/Project/RNAToolkit2">Theophylline-responsive riboswitch</A></TD></TR><TR><TD colspan="2" align="left" class="exist1"><P>Extension</P></TD></TR><TR><TD height="27" align="left"> </TD><TD align="left" class="TPP1"><A href="https://2011.igem.org/Team:Peking_R/Project/RNAToolkit3">Engineered group I intron with a theophylline hammerhead ribozyme</A></TD></TR><TR><TD height="33" align="left"> </TD><TD align="left" class="TPP1">Adenine hammerhead ribozyme obtained from screening</TD></TR></TBODY></TABLE></DIV><DIV class="printfooter">
Retrieved from &quot;<A href="http://2011.igem.org/Team:Peking_R/Project/RNAToolkit4">http://2011.igem.org/Team:Peking_R/Project/RNAToolkit4</A>&quot;</DIV></DIV></DIV><DIV id="footer-box" class="noprint"><DIV id="footer"><UL id="f-list"><LI id="t-recentchanges"><A href="/Special:RecentChanges" title="Recent changes">Recent changes</A></LI><LI id="t-whatlinkshere"><A href="/Special:WhatLinksHere/Team:Peking_R/Project/RNAToolkit4" title="List of all wiki pages that link here [j]" accesskey="j">What links here</A></LI><LI id="t-recentchangeslinked"><A href="/Special:RecentChangesLinked/Team:Peking_R/Project/RNAToolkit4" title="Recent changes in pages linked from this page [k]" accesskey="k">Related changes</A></LI><LI id="t-specialpages"><A href="/Special:SpecialPages" title="List of all special pages [q]" accesskey="q">Special pages</A></LI><LI><A href="/Special:Preferences">My preferences</A></LI></UL></DIV><DIV id="footer"><UL id="f-list"><LI id="t-print"><A href="/wiki/index.php?title=Team:Peking_R/Project/RNAToolkit4&amp;printable=yes" title="Printable version of this page [p]" accesskey="p">Printable version</A></LI><LI id="t-permalink"><A href="/wiki/index.php?title=Team:Peking_R/Project/RNAToolkit4&amp;oldid=252609" title="Permanent link to this revision of the page">Permanent link</A></LI><LI id="privacy"><A href="/2011.igem.org:Privacy_policy" title="2011.igem.org:Privacy policy">Privacy policy</A></LI><LI id="disclaimer"><A href="/2011.igem.org:General_disclaimer" title="2011.igem.org:General disclaimer">Disclaimers</A></LI></UL></DIV></DIV></DIV></BODY></HTML>