"
Page
Discussion
View source
History
teams
Log in
 
Team:Peking R/Project/Application/VIO
From 2011.igem.org
Template:Https://2011.igem.org/Team:Peking R/bannerhidden Template:Https://2011.igem.org/Team:Peking R/back2 Template:Https://2011.igem.org/Team:Peking R/Projectbackground11
无标题文档
Fine-tuning Biosynthetic Pathway Using Soft-coding of Genetic Program
AND gate
Bistable switch
Violacein synthetic pathway
Abstract
Achieving higher yields of desired products and optimizing the efficiency of metabolic pathways have long been key goals of metabolic engineering. Recent breakthroughs in synthetic biology that microorganisms could produce valuable chemicals originally from heterologous organisms make the demand for a versatile method to optimize biosynthetic pathway more urgent. Existing methods, including rearranging the order of genes in the gene clusters[3], directed evolution of enzymes to alter their catalytic functions[4], and combinatorial alteration of intergenic regions[5], proved to be laborious and time-consuming. Here we present a brand-new approach to fine-tune the translation strength of enzymes by replacing ribosomal binding sites (RBS) of genes of interest with riboswsitches that respond to thiamine pyrophosphate (TPP). After characterization on the performance of riboswitches, the RBS could be altered to an automated-designed one with the help of RBS calculator developed in our project. This approach would significantly save work on laborious library constructing and screening, and accelerate the process of pathway optimization with computer-aided design.
Introduction
Biosynthesis is a biological process that involves several enzymes to convert specific molecules to other functional products. The process often goes in a stepwise manner, in which the product of an upstream enzyme is used as a substrate of a downstream enzyme.
Marked advancements in biosynthesis have been achieved in recent years. For example, artemisinic acid, an anti-malarial drug precursor, was produced by inserting an artificial biosynthetic pathway into Saccharomyces cerevisae (beer yeast)[1]. Another example is that production of butanol, a prospective new biofuel, is achieved in high yield in E. coli by producing enzymes with preferred kinetic characteristics from other organisms[2].
These breakthroughs have spurred even more interest in optimizing biosynthetic pathways in microorganisms. To achieve high yield of the desired products, various strategies have emerged to optimize biosynthetic pathways composed of several enzymes. For instance, the order of five genes encoding enzymes that are involved in carotenoid biosynthesis were rearranged using Ordered Gene Assembly in Bacillus subtilis (OGAB) method, and the carotenoid production varied largely among different constructs[3]. Besides, directed evolution was carried out to screen for rate-limiting enzyme mutants with enhanced catalytic activities, so that the production yield could be improved[4]. More recently, Pfleger et. al.[5]generated libraries of tunable intergenic regions (TIGRs) to vary the relative expression levels of genes in an operon, resulting in a sevenfold increase in mevalonate production.
However, these methods possess obvious drawbacks. Each of the methods mentioned above would require construction of libraries in different forms, like constructing many similar plasmids with different gene arrangement, or generating a library of coding sequence mutants, or assembling a library of TIGR sequences. The library needs further screening process, and often than not, mutants that survived one-round selection cannot meet the requirement, so multi-round library constructing and screening are often indispensible. In summary, these methods require large human input and are time-consuming.
Recently, RNA devices have emerged as powerful tools to regulate gene expression in vivo, and particularly, ligand-responsive riboswitches enable us to manipulate translation strength of specific genes upon different concentration of ligands[6]. Moreover, with ribosomal binding site (RBS) computer-aided designer developed in our project, the translation strength of a ligand-responsive riboswitch under certain concentration of corresponding ligand could be met by a designed RBS sequence. Therefore, combining these two methods would enable us to fine-tune biosynthetic pathway without laborious library constructing and screening procedure.
To testify our platform for genetic softcoding, we chose to apply it to a segment of violacein biosynthetic pathway, because violacein and its precursors have notable antibacterial, antiviral, antiparasite and many other biological activities, rendering them having great potential in medical applications. Besides, this biosynthetic pathway is more characterized, including the intermediates, the shunt products, the mechanism of each reaction and so on. Moreover, the products in violacein biosynthetic pathway are often colored, so it's relatively easier to pick positive colonies and to verify the experiment primarily by naked eyes. And most importantly, the enzymes involved in this pathway are readily available in the Registry of Standard Biological Parts.
Violacein originates from a soil bacterium Chromobacterium violaceum, and genes for violacein biosynthesis are arranged in an operon consisting of vioA, vioB, vioC, vioD and vioE. These genes have been successfully transformed to E. coli to produce violacein[7]. In the segment of the pathway investigated in our project, four enzymes are involved (Fig. 1). VioA is an FAD dependent L-tryptophan oxidase, which transforms L-tryptophan to an IPA imine. VioB would further convert the IPA imine into a dimer. VioEis responsible for the indole shift that converts the IPA imine dimer to prodeoxyviolacein, which can be taken over by VioD to form proviolacein. In E.coli an additional side reaction occurs, producing a green pigment called deoxychromoviridans, which is produced by condensing two prodeoxyviolacein molecules.
Fig.  1 Scheme of a segment of violacein biosynthetic pathway. VioA, VioB, VioE  function sequentially to convert L-tryptophan to prodeoxyviolacein, which would  either form deoxychromoviridans due to intrinsic reactions in E. coli, or be converted to proviolacein  by VioD.
 
Design of constructs to fine-tune violacein biosynthetic pathway
After analyzing the metabolic flux in violacein biosynthetic pathway, we decided to fine-tune the translation strength of enzymes VioE and VioD, which have direct links to the side products, as shown in Fig.1. Therefore, we substituted the RBS of vioE or vioD with thiamine pyrophosphate (TPP)-responsive hammerhead ribozyme variants, which were previously characterized in our project. Decreasing the amount of VioE may lower the pool of prodeoxyviolacein, hence lowering the production of deoxychromoviridans, thus we inserted a TPP down-regulated hammerhead ribozyme (TPP ribozyme 2.5) in front of vioE (Fig.2A), so that the purity of the product proviolacein may increase as TPP is added to the culture(Fig.2B). Similarly, increasing the amount of VioD may increase the yield of proviolacein, thus we inserted a TPP up-regulated hammerhead ribozyme (TPP ribozyme 1.20) upstream of vioD (Fig.2A), so that upon adding TPP into the culture, the purity of the products may increase(Fig.2C).
Fig.2 (A) Design of constructs to fine-tune violacein biosynthetic pathway. Hexagon: Stem-loop terminator (Part:BBa_B0015); Bent arrow: pBAD promoter (Part: BBa_I13453); Oval: Ribosomal binding site; Straight arrow: Coding sequence originated from Part: BBa_K274003; ribbon shape: TPP-responsive ribozyme. (B) Decreasing the amount of VioE may lower the pool of prodeoxyviolacein, hence lowering the production of deoxychromoviridans, thus we inserted a TPP down-regulated hammerhead ribozyme (TPP ribozyme 2.5) in front of vioE, and the metabolic flux would favor proviolacein. Dash arrow: down-regulated. (C) Increasing the amount of VioD may increase the yield of proviolacein, thus we inserted a TPP up-regulated hammerhead ribozyme (TPP ribozyme 1.20) upstream of vioD, and the metabolic flux would favor proviolacein. Dash arrow: down-regulated; Filled arrow: up-regulated.
 
Results
We have successfully constructed a plasmid that would enable us to fine-tune the translation strength of vioE by TPP ribozyme 2.5 (Part: BBa_K598019). The plasmid was transformed to E. coli, and after induction by arabinose and culturing in different concentrations of TPP, the bacteria produced some pigments that were visible by naked eyes (Fig.3). After lysating the bacteria with 10% SDS and extracting the pigments with ethylacetate, the extraction samples were analyzed by HPLC (Agilent systems 1200 series, Spursil C18 5um column, mobile phase 50% methanol, 50% water, monitor wavelength 650nm). The two peaks eluted at 0.7-1.0 min were confirmed as the products of the bacteria that were extracted into ethylacetate(Fig.4), and we tentatively assumed that the two peaks corresponded to proviolacein (left) and deoxychromoviridans (right) (Fig.4).
Fig.3 E. coli producing pigments. When induced by arabinose, the engineered E. coli produced dark-green pigments (second tube comparing to first one). Upon addition of different concentration of thiamine pyrophosphate (TPP), the color of the bacteria gradually shifted from dark-green to dark-brown (from second tube to sixth one).
To see if adding TPP could affect the purity of the products extracted, we induced the bacteria under different concentrations of TPP. The samples were prepared as described above. The pelleted cells produced some pigments, and the color changed as the concentration of TPP increased (Fig. 3). The HPLC results turned out that upon adding TPP into the cultures, the amount of deoxychromoviridans decreased, and the ratio between proviolacein and deoxychromoviridans increased upon increasing TPP concentration (Fig.5), until deoxychromoviridans was not detectable when TPP concentration reached to 10uM. These results proved that our genetic rheostate had the ability to fine-tune the biosynthetic pathway and increased the purity of the products as we desired.
Fig.5 HPLC results of the bacterial extraction when different concentrations of TPP were presented in the cultures. When induced with arabinose, the bacterial extraction produced two peaks eluted at 0.7-1.0 min ((b) comparing to (a)), and we tentatively assumed that the two peaks corresponded to proviolacein (left) and deoxychromoviridans (right). Upon adding TPP into the cultures, the amount of deoxychromoviridans decreased, and the ratio between proviolacein and deoxychromoviridans increased upon increasing TPP concentration ((b) to (f)), until deoxychromoviridans was not detectable (f).
Fig.6 The area ratio between peaks of proviolacein and deoxychromoviridans on HPLC when different concentrations of TPP were presented in the cultures. Note that deoxychromoviridans was not detectable when TPP concentration reached to 10uM, so the ratio was not presented in this figure.
Due to time limit, we have not performed calculated RBS sequence substitution yet, but further experiment that would fix the RBS sequence of vioE designed by our RBS calculator will enable us to settle down the optimized biosynthetic pathway. This brand-new approach greatly saved us from time-consuming library construction and screening, which facilitated our experiment progress to a large extent. The versatile and extensible platform for softcoding of genetic program has proved its applicability in fine-tuning biosynthetic pathway, and it will prove its robustness in more bioengineering fields.
 
Reference:
1.Ro, D K et al. (2006). Production of the antimalarial drug precursor artemisinic acid in engineered yeast. Nature 440, 940-943.
2.Bond-Watts, Brooks B et al. (2011). Enzyme mechanism as a kinetic control element for designing synthetic biofuel pathways. Nat. Chem. Biol. 7, 222-227
3.Nishizaki, Tomoko et. al. (2007) Metabolic Engineering of Carotenoid Biosynthesis in Escherichia coliby Ordered Gene Assembly in Bacillus subtilis. Appl. Environ. Microbiol. 73, 1355-1361
4.Wang, Chia-wei et.al.(2000) Directed Evolution of Metabolically ngineered Escherichia coli forCarotenoid Production.Biotechnol. Prog. 16, 922-926
5.Pfleger, Brian F et. al. (2006) Combinatorial engineering of intergenic regions inoperons tunes expression of multiple genes. Nat. Biotech. 24, 1027-1032
6.Breaker, Ronald R (2004). Natural and engineered nucleicacids as tools to explore biology. Nature 432, 838-845
7.Balibar CJ, Walsh CT. (2006). In vitro biosynthesis of violacein from L-tryptophan by the enzymes VioA-E from Chromobacterium violaceum, Biochemistry 45, 15444-57.
 
 
 
 
 
 
Retrieved from " http://2011.igem.org/Team:Peking_R/Project/Application/VIO "
Recent changes
What links here
Related changes
Special pages
My preferences
Printable version
Permanent link
Privacy policy
Disclaimers
