<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_ETH_Zurich_Tar skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><DIV id="top_title"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:ETH Zurich/Tar</SPAN></H1></DIV><DIV id="HQ_page"><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><DIV class="container-nav-menu"><DIV class="nav-menu"><DIV class="nav-menu-left "><DIV class="nav-item" id="nav-item1"><SPAN>Project</SPAN><UL class="submenu"><LI><A href="https://2018.igem.org/Team:ETH_Zurich/Description">Description</A></LI><LI><A href="https://2018.igem.org/Team:ETH_Zurich/Applied_Design">Applied Design</A></LI><LI><A href="https://2018.igem.org/Team:ETH_Zurich/Team">Team Members</A></LI><LI><A href="https://2018.igem.org/Team:ETH_Zurich/Attributions">Attributions</A></LI><LI><A href="https://2018.igem.org/Team:ETH_Zurich/Sponsoring">Sponsoring</A></LI><LI><A href="https://2018.igem.org/Team:ETH_Zurich/Notebook">Notebook</A></LI><LI><A href="https://2018.igem.org/Team:ETH_Zurich/Safety">Safety</A></LI><LI><A href="https://2018.igem.org/Team:ETH_Zurich/Download">Downloads</A></LI></UL></DIV><DIV class="nav-item" id="nav-item2"><SPAN>Wet Lab</SPAN><UL class="submenu"><LI><A href="https://2018.igem.org/Team:ETH_Zurich/Tar">Tar Receptor Evolution</A></LI><LI><A href="https://2018.igem.org/Team:ETH_Zurich/ApproachA">Approach A</A></LI><LI><A href="https://2018.igem.org/Team:ETH_Zurich/ApproachB">Approach B</A></LI><LI><A href="https://2018.igem.org/Team:ETH_Zurich/Experiments">Experiments</A></LI><LI><A href="https://2018.igem.org/Team:ETH_Zurich/InterLab">InterLab</A></LI></UL></DIV><DIV class="nav-item" id="nav-item3"><SPAN>Hardware</SPAN><UL class="submenu"><LI><A href="https://2018.igem.org/Team:ETH_Zurich/Hardware">Hardware Overview</A></LI><LI><A href="https://2018.igem.org/Team:ETH_Zurich/Microscope">Microscope</A></LI><LI><A href="https://2018.igem.org/Team:ETH_Zurich/Mechanics">Mechanics</A></LI><LI><A href="https://2018.igem.org/Team:ETH_Zurich/SyringePump">Syringe Pump</A></LI><LI><A href="https://2018.igem.org/Team:ETH_Zurich/MicrofluidicChip">Microfluidic Chip</A></LI><LI><A href="https://2018.igem.org/Team:ETH_Zurich/bubbling">Bubbling</A></LI></UL></DIV><DIV class="nav-item" id="nav-item4"><SPAN>Software</SPAN><UL class="submenu"><LI><A href="https://2018.igem.org/Team:ETH_Zurich/Software">Software Overview</A></LI><LI><A href="https://2018.igem.org/Team:ETH_Zurich/Motility_Readout">Motility Readout</A></LI><LI><A href="https://2018.igem.org/Team:ETH_Zurich/ImageAnalysis">Luminescence Algorithm</A></LI><LI><A href="https://2018.igem.org/Team:ETH_Zurich/RobotControls">Robot Controls</A></LI><LI><A href="https://2018.igem.org/Team:ETH_Zurich/Path_Planning">Path Planning</A></LI></UL></DIV></DIV><DIV class="nav-menu-middle "><DIV class="nav-item" id="home"><A href="https://2018.igem.org/Team:ETH_Zurich"><TITLE id="title6">Logo Website</TITLE></A></DIV></DIV><DIV class="nav-menu-right "><DIV class="nav-item" id="nav-item5"><SPAN>Model</SPAN><UL class="submenu"><LI><A href="https://2018.igem.org/Team:ETH_Zurich/Model">Model Overview</A></LI><LI><A href="https://2018.igem.org/Team:ETH_Zurich/integratedModel">Integrated Models</A></LI><LI><A href="https://2018.igem.org/Team:ETH_Zurich/motilityModel">Motility Model</A></LI><LI><A href="https://2018.igem.org/Team:ETH_Zurich/splitModel">Split Luciferase Model</A></LI><LI><A href="https://2018.igem.org/Team:ETH_Zurich/diffusionModel">Diffusion Model</A></LI><LI><A href="https://2018.igem.org/Team:ETH_Zurich/bubbelingModel">Bubbling Model</A></LI><LI><A href="https://2018.igem.org/Team:ETH_Zurich/HolographicModel">Holographic Imaging Model</A></LI><LI><A href="https://2018.igem.org/Team:ETH_Zurich/Screening">Screening Assay Model</A></LI><LI><A href="https://2018.igem.org/Team:ETH_Zurich/Parameter">Model Parameters</A></LI></UL></DIV><DIV class="nav-item" id="nav-item6"><SPAN>Human Practices</SPAN><UL class="submenu"><LI><A href="https://2018.igem.org/Team:ETH_Zurich/Human_Practices">Human Practices</A></LI><LI><A href="https://2018.igem.org/Team:ETH_Zurich/Public_Engagement">Education &amp; Engagement</A></LI></UL></DIV><DIV class="nav-item" id="nav-item7"><SPAN>Achievements</SPAN><UL class="submenu"><LI><A href="https://2018.igem.org/Team:ETH_Zurich/Results">Results</A></LI><LI><A href="https://2018.igem.org/Team:ETH_Zurich/Parts">Parts Overview</A></LI><LI><A href="https://2018.igem.org/Team:ETH_Zurich/Demonstrate">Demonstrate</A></LI><LI><A href="https://2018.igem.org/Team:ETH_Zurich/Collaborations">Collaborations</A></LI></UL></DIV></DIV></DIV><DIV class="container-content"><DIV id="tongue1" class="content-tongue"><DIV class="tongue-div" id="tongue-div1">Content</DIV></DIV><DIV class="content-text"><SPAN href="section-Introduction" class="content-item">Introduction</SPAN><SPAN href="container_table" class="content-item">Table</SPAN></DIV></DIV></DIV><DIV class="main-menu"><A href="https://2018.igem.org/Team:ETH_Zurich"><DIV class="home">
        Home
      </DIV></A><DIV class="main-titel"><A href="https://2018.igem.org/Team:ETH_Zurich">
        A<SPAN>.</SPAN>R<SPAN>.</SPAN>O<SPAN>.</SPAN>M<SPAN>.</SPAN>A<SPAN class="main-letterSpacing">.</SPAN></A></DIV><A href="https://2018.igem.org/Team:ETH_Zurich"><DIV class="menu-logo"><TITLE id="title6">Logo Website</TITLE></DIV></A></DIV><DIV class="main-submenu"><DIV class="alt-main-submenu-item" id="alt-nav-item1" href="#project"><SPAN>Project</SPAN><UL class="alt-submenu"><LI><A href="https://2018.igem.org/Team:ETH_Zurich/Description">Description</A></LI><LI><A href="https://2018.igem.org/Team:ETH_Zurich/Applied_Design">Applied Design</A></LI><LI><A href="https://2018.igem.org/Team:ETH_Zurich/Team">Team Members</A></LI><LI><A href="https://2018.igem.org/Team:ETH_Zurich/Attributions">Attributions</A></LI><LI><A href="https://2018.igem.org/Team:ETH_Zurich/Sponsoring">Sponsoring</A></LI><LI><A href="https://2018.igem.org/Team:ETH_Zurich/Notebook">Notebook</A></LI><LI><A href="https://2018.igem.org/Team:ETH_Zurich/Safety">Safety</A></LI><LI><A href="https://2018.igem.org/Team:ETH_Zurich/Download">Downloads</A></LI></UL></DIV><DIV class="alt-main-submenu-item" id="alt-nav-item2" href="#parts"><SPAN>Wet Lab</SPAN><UL class="alt-submenu"><LI><A href="https://2018.igem.org/Team:ETH_Zurich/Tar">Tar Receptor Evolution</A></LI><LI><A href="https://2018.igem.org/Team:ETH_Zurich/ApproachA">Approach A</A></LI><LI><A href="https://2018.igem.org/Team:ETH_Zurich/ApproachB">Approach B</A></LI><LI><A href="https://2018.igem.org/Team:ETH_Zurich/Experiments">Experiments</A></LI><LI><A href="https://2018.igem.org/Team:ETH_Zurich/InterLab">InterLab</A></LI></UL></DIV><DIV class="alt-main-submenu-item" id="alt-nav-item3" href="#modelling"><SPAN>Hardware</SPAN><UL class="alt-submenu"><LI><A href="https://2018.igem.org/Team:ETH_Zurich/Hardware">Hardware Overview</A></LI><LI><A href="https://2018.igem.org/Team:ETH_Zurich/Microscope">Microscope</A></LI><LI><A href="https://2018.igem.org/Team:ETH_Zurich/Mechanics">Mechanics</A></LI><LI><A href="https://2018.igem.org/Team:ETH_Zurich/SyringePump">Syringe Pump</A></LI><LI><A href="https://2018.igem.org/Team:ETH_Zurich/MicrofluidicChip">Microfluidic Chip</A></LI><LI><A href="https://2018.igem.org/Team:ETH_Zurich/bubbling">Bubbling</A></LI></UL></DIV><DIV class="alt-main-submenu-item" id="alt-nav-item4"><SPAN>Software</SPAN><UL class="alt-submenu"><LI><A href="https://2018.igem.org/Team:ETH_Zurich/Software">Software Overview</A></LI><LI><A href="https://2018.igem.org/Team:ETH_Zurich/Motility_Readout">Motility Readout</A></LI><LI><A href="https://2018.igem.org/Team:ETH_Zurich/ImageAnalysis">Luminescence Algorithm</A></LI><LI><A href="https://2018.igem.org/Team:ETH_Zurich/RobotControls">Robot Controls</A></LI><LI><A href="https://2018.igem.org/Team:ETH_Zurich/Path_Planning">Path Planning</A></LI></UL></DIV><DIV class="alt-main-submenu-item" id="alt-nav-item5"><SPAN>Model</SPAN><UL class="alt-submenu"><LI><A href="https://2018.igem.org/Team:ETH_Zurich/Model">Model Overview</A></LI><LI><A href="https://2018.igem.org/Team:ETH_Zurich/integratedModel">Integrated Models</A></LI><LI><A href="https://2018.igem.org/Team:ETH_Zurich/motilityModel">Motility Model</A></LI><LI><A href="https://2018.igem.org/Team:ETH_Zurich/splitModel">Split Luciferase Model</A></LI><LI><A href="https://2018.igem.org/Team:ETH_Zurich/diffusionModel">Diffusion Model</A></LI><LI><A href="https://2018.igem.org/Team:ETH_Zurich/bubbelingModel">Bubbling Model</A></LI><LI><A href="https://2018.igem.org/Team:ETH_Zurich/Screening">Screening Assay Model</A></LI><LI><A href="https://2018.igem.org/Team:ETH_Zurich/HolographicModel">Holographic Imaging Model</A></LI></UL></DIV><DIV class="alt-main-submenu-item" id="alt-nav-item6"><SPAN>Human Practices</SPAN><UL class="alt-submenu"><LI><A href="https://2018.igem.org/Team:ETH_Zurich/Human_Practices">Human Practices</A></LI><LI><A href="https://2018.igem.org/Team:ETH_Zurich/Public_Engagement">Education &amp; Engagement</A></LI></UL></DIV><DIV class="alt-main-submenu-item" id="alt-nav-item7"><SPAN>Achievements</SPAN><UL class="alt-submenu"><LI><A href="https://2018.igem.org/Team:ETH_Zurich/Results">Results</A></LI><LI><A href="https://2018.igem.org/Team:ETH_Zurich/Parts">Parts Overview</A></LI><LI><A href="https://2018.igem.org/Team:ETH_Zurich/Demonstrate">Demonstrate</A></LI><LI><A href="https://2018.igem.org/Team:ETH_Zurich/Collaborations">Collaborations</A></LI></UL></DIV></DIV><DIV class="mobileMenuBg"><DIV class="container-titel"><DIV class="titel" id="titel">
        Tar Receptor
        Evolution<SPAN>.</SPAN></DIV><DIV class="scroll">Scroll</DIV></DIV><DIV class="sectionh1 contentTable contentIntro" contenttable="Introduction" contentref=".contentIntro">Introduction</DIV><DIV class="text">
      The natural ligand of the Tar receptor is aspartate. While this molecule is useful for bacteria, localizing aspartate sources is less interesting from a human perspective. For this reason, it is necessary to generate libraries of Tar receptor
      mutants from which to select the ones showing selectivity for new, noteworthy compounds. To this end, it is equally important to create libraries with sufficient diversity and establish a fast and effective assay to screen them. We thus aimed
      at developing robust protocols to obtain mutation libraries and a simple and fast experimental set up for the selection process.
      The Tar receptor structure is well characterized [1], and the sensory domain is defined at the N terminus, between M1 and F189. This modularity allows to restrict the target area of diversity generation to this region, consequently requiring
      fewer mutants and resulting in a higher library quality, as the other regions are not varied and definitely remain functional.
      Moreover, since modifications in the sequence of Tar ultimately result in changing of the motility [2,3], a simple assay can be designed. Differences in the chemotactic behavior are a rather convenient phenotype, since it physically separates
      responding mutants from not-responding ones.

    </DIV><DIV class="sectionh2">Template</DIV><DIV class="text">
      We used BBa_K777001 as a template to optimize the protocol for library generation. The BioBrick contains the strong J23100 Anderson promoter and the tar gene, however it lacks an RBS. Imaging tethered ∆tar∆tsr bacteria responding to aspartate,
      they can be seen modifying their spinning behavior, suggesting that the protein is, to some level, expressed and functional. Nevertheless, this BioBrick should not be used as a starting point for a stable expression of the Tar receptor. A good
      improvement would be to express the gene under an inducible promoter and inserting a weak RBS.
    </DIV><DIV class="sectionh1 contentTable contentlib" contenttable="Libraries Generation" contentref=".contentlib">Libraries Generation</DIV><DIV class="sectionh2">Two types of libraries</DIV><DIV class="text">
      The Tar receptor has already been modified with the aim of expanding its specificity to other small molecules. The three main approaches to achieve this are: swapping the sensory domain for the one of another species (e.g. Pseudomonas putida
      [3]), mutations around the two ligand contacting sites (Goettingen iGEM team 2012) and random mutations over the whole sensory domain [2].
      Since the first approach requires multiple iterations before tuning appropriately the cutting and fusing sites, we decided to base our experiments on the last two methods, that have the quality of a more straight-forward workflow.
    </DIV><DIV class="text">
      As previously stated, to change the specificity of the Tar receptor, the interesting region to focus on are the first 189 amino acids on the N terminus. In particular, it is possible to identify the residues that localize in the ligand binding
      pocket (Y149 to T154 and R69 to R73) [4]. We generated two types of libraries, one that only has mutated residues around the ligand binding sites and one that contained random mutations on the whole sensory domain.
    </DIV><DIV class="sectionh2">Site-directed mutagenesis
    </DIV><DIV class="text">
      To create a library with mutations at the ligand binding sites, we designed degenerated primers that anneal on the targeted sites in BBa_K777001. The oligonucleotides were designed to have an annealing region without mutations at the 3’ end,
      either one or two NNK codons in the middle and another annealing region to the template at the 5’ end, used in the following ligation step as overlapping region. The first pair of primers amplifies the region between the two ligand binding
      sites and includes the degenerated bases at the 5’. The second pair amplifies the region external to the two ligand binding sites and includes the degenerated bases at the 3’.
    </DIV><DIV class="container_figure"><DIV class="figure_caption caption_center">Workflow overview for the generation of the site directed mutagenesis library</DIV></DIV><DIV class="text">
      The two fragments have been amplified by standard PCR with Phusion polymerase (refer to protocol) and, after digestion of the template with DpnI, ligated using a homemade Gibson assembly mix (refer to protocol), transformed in TOP10 chemically
      competent cells and plated on LB agar + Chloramphenicol 34 plates. The DpnI digestion has been proven to be crucial in order to retrieve mutants and not only wild type plasmids.
    </DIV><DIV class="text">
      Once the colonies were grown, in order to conserve a copy, the colonies were transferred onto fresh plates by replica plating. Then the colonies were scraped off and resuspended in LB medium. The DNA from the culture was then extracted and
      sent for sequencing. The sequencing analysis showed variability around the two ligand binding sites.
    </DIV><DIV class="text">
      Given that we targeted 5 amino acids , the number of clones that would be needed to cover 95% of the library can be estimated with Kille et al., 2012 [5] algorithm:
    </DIV><DIV class="text math">
      \begin{equation}
      L=-V\cdot \ln (1-F)=-64^5\cdot \ln (1-0.95) = 3.2\cdot 10^9
      \end{equation}
    </DIV><DIV class="text">
      Where L is the number of samples; V is the total number of variants X<SUP>n</SUP> (X being the number of possible codons and n being the number of saturated residues); F is the fractional library completeness.
      While this calculation gives a ballpark on the numbers of mutants that should be screened, smaller libraries have been shown to equally serve the purpose.
      It is important to note that the dimension of the library could be in the way of an effective screening process, however the assay that we established is suitable for these library sizes and offers a handy set up to select interesting mutants.
    </DIV><DIV class="sectionh2">Random mutagenesis</DIV><DIV class="text">
      The second approach that we aimed to explore is the construction of a library with random mutations on the whole sensory domain, as reported in Derr et al. [2]. For this reason, we designed two pairs of primers, one that amplifies the
      interesting part of the gene and the other one that amplifies the remaining portion of the plasmid. Once again BBa_K777001 was used as a template.
    </DIV><DIV class="text">
      The fragment containing the backbone was amplified using high fidelity Phusion polymerase. The fragment containing the sensory domain was amplified using Taq polymerase with addition of MnCl<SUB>2</SUB>, which competes with MgCl<SUB>2</SUB>
      and interferes with the
      polymerase activity to obtain an error-prone amplification. 
      The protocol was optimized for both fragments, for the first one the most effective way to obtain such a long fragment (&gt;3 kb), was to add 2.5% glycerol and perform a touch-up PCR going from 65 °C to 72 °C, increasing the temperature by 1 °C
      every 5 cycles.
      For the error-prone PCR multiple protocols were tested, using either Taq or Pfu polymerase.
      Finally, the most successful conditions were found to be using Taq polymerase, a biased proportion of dNTPs (0.2 mM dATP and dGTP, 1 mM dCTP and dTTp), 0.1 mM of MnCl<SUB>2</SUB> and 2.5 mM of MgCl<SUB>2</SUB>. Refer to the protocol for a more
      detailed description
      of the steps and the PCR mix assembly.
    </DIV><DIV class="text">
      Once a pool of mutated sensory domains is generated, it is crucial to loose as little diversity as possible in the successive steps in order to have a high-quality library. For this reason, the assembly and the transformation need to be as
      efficient as possible.
      After DpnI digestion and PCR clean-up of the two fragments, the two were ligated using the high-fidelity commercial Gibson assembly mix and transformed in DH5alpha chemically competent cells.
      Many different conditions were tested to obtain the best coverage possible (i.e. the maximum number of clones with mutations). We tried adding a desalting step after Gibson, using electroporation instead of chemical transformation, we compared
      different ratios of fragments in the Gibson assembly mix and different strains of E. coli were assayed. The method that resulted in the highest number of mutants was transforming 5 µL of the Gibson assembly (assembled with 2:1 ratio of
      fragment to backbone, with a total final amount of 0.2 pmole of DNA) into 50 µL of chemically competent DH5alpha cells.
    </DIV><DIV class="text">
      The diversity of the library was estimated by picking 10 transformants and sequencing the sensory domain. The average mutation rate was 2.8 per insert, with a maximum number of 5 point-mutations and only one insertion (which disrupts the
      function of the protein). Having retrieved around 4000 clones, the coverage was drastically low, however, practical experience shows that a number as low as 5000 clones can sometimes be sufficient to find a hit.
    </DIV><DIV class="container_figure"><DIV class="figure_caption caption_center">Mutation frequency</DIV></DIV><DIV class="sectionh2">Conclusion</DIV><DIV class="text">
      We established two protocols to generate two different kinds of libraries, however due to time constraints, we could not iterate through the procedure enough to get to a satisfying coverage. However, this optimized pipeline can constitute a
      good starting point which the iGEM community can build upon to create a diverse Tar receptor libraries and evolve this protein towards new and interesting specificities.
    </DIV><DIV class="sectionh1 contentTable contentScreen" contenttable="Screening assay " contentref=".contentScreen">Screening assay </DIV><DIV class="text">
      One of the main advantages of screening for mutants of the Tar receptor is the presence of a handy phenotype like bacterial movement. It is indeed relatively straightforward to select mutants that are attracted to a compound by physically
      separating them from all the other variants. However, while the concept of such an assay can appear simple, in practice a lot of factors need to be optimized. We took care of tuning some meaningful parameters in order to provide a robust
      experimental set up.
    </DIV><DIV class="text">
      The key concept of this assay is the generation of a gradient of attractant on a semisolid agar plate, in the concentration range of the Tar sensitivity, that can be followed by specific mutants that will then separate themselves from the
      group of non-responding bacteria.

    </DIV><DIV class="sectionh2">The strain</DIV><DIV class="text">
      Common laboratory strains of E. coli have all five main genes responsible for chemotaxis. This is an issue when trying to express mutants of the same protein, since Tar and Tsr receptor exhibit cross-talk, this could lead to aspecific movement
      towards other molecules. While the proof-of-concept of this assay has been conducted with wild type MG1655, the experiments on the libraries should be transformed and screened into a ∆tar∆tsr strain. We were kindly provided with this strain by
      the Panke laboratory, however since it was not used in crucial experiments we will not go into the details of its characteristics.
    </DIV><DIV class="sectionh2">The plate</DIV><DIV class="text">
      We performed our experiment using rectangular 9x15 cm plates for two reasons: they allow to easily plate a long straight line of transformants that do not inhibit each other’s growth while moving towards the source (as opposed to a circular
      disposition where the movement is towards the center), and they allow to make easy unidimensional and unidirectional assumptions when calculating the formation of the gradient. 
      The medium composition has been set after multiple iterations, combining different protocols [2], also referring to the Goettingen iGEM team from 2012. Since LB contains aspartate it cannot be used for this experiment, indeed the base for the
      swimming medium needs to be a minimal medium. We decided to use a modified version of M9, with 0.3% agar to allow the swimming movement. Since the plasmid has a pSB1C3 backbone, chloramphenicol was added when the agar was cooled but still
      liquid. A critical step is to pour the medium when it is still completely liquid, to achieve an homogeneous distribution and thus a homogeneous movement of the bacteria.
      The attractant was applied on the plates by pipetting small drops of a liquid solution (10 mM) on Whatman paper stripes and then delicately placing them on one of the borders of the plate. It is recommended not to soak the stripes directly in
      the solution as they would tend to form ooze on the plate.
    </DIV><DIV class="sectionh2">The Gradient</DIV><DIV class="text">
      The formation of a useful gradient depends on the initial concentration, on the position of the source and its dimension, on the time passed, on the composition of the diffusing medium and on the diffusion coefficient of the small molecule
      itself.
    </DIV><DIV class="text">
      We thus found it very helpful to build a model that can integrate all this parameters and provide a guideline the plating time and position with respect to the source.
      The model simulates the diffusion of the selected molecule, provided the diffusion coefficient in a 0.3% agar plate, given a source of customizable concentrations and dimensions (meaning how wide the stripe is). The indicated concentrations
      are representative of an upper bound of the real one, as the model only consider one dimensional diffusion.
    </DIV><DIV class="text">
      We simulated the diffusion of 10 mM aspartate diluted in M9, on a stripe that covers slightly more than one third of the plate. In the laboratory, we set up the corresponding experiment by placing a stripe soaked in 10 mM aspartate solution in
      M9 on the border of a swimming plate; we immediately plated several drops of MG1655 overnight culture on different positions on the plate. The simulation is in accordance with the experiment and it is possible to distinguish three different
      behaviors between the groups. The first group, starting from the upper part of the plate, shows strong symmetrical movement, this can be explained by the fact that the concentration in that portion of the plate is higher than the sensitivity
      threshold of the Tar receptor, thus the bacteria cannot distinguish the direction of the gradient and expand equally in all directions.
      The second group shows a biased movement towards the stripe, this is in accordance to the predicted position of an appropriate window of aspartate concentrations.
      The remaining four groups do not exhibit any clear swimming activity, which, once again, is predicted in the model by the fact that there is no aspartate yet in that portion of the plate.
    </DIV><DIV class="container_figure"><DIV class="figure_caption caption_center">Model simulation of the diffusion of aspartate, compared to the swimming assay visualizing MG1655 chemotactic behavior towards aspartate.</DIV></DIV><DIV class="text">
      As negative control we used the same set up but with a stripe soaked in M9 [Figure 4.1] plating MG1655, and stripe soaked in 10 mM aspartate in M9 plating ∆tar∆tsr strain (multiple plating positions were tested, all giving the same
      result)[Figure 4.2]. In
      both negative control experiments no directional movement was observed.
    </DIV><DIV class="container_figure"><DIV class="figures_container"><DIV class="figure"><DIV class="subfigure_caption">Negative control with ∆tar∆tsr strain that does not show any specific movement towards aspartate</DIV></DIV><DIV class="figure"><DIV class="subfigure_caption">Negative control with wild type MG1655 that shows no specific movement towards aspartate</DIV></DIV></DIV><DIV class="figure_caption caption_center">Negative controls</DIV></DIV><DIV class="sectionh2">The screening experimental set up
    </DIV><DIV class="text">
      Based on our experiments, swimming plate assays can be used for library screening as follows. The bacterial culture needs to be grown in M9 at 30°C for at least one night before plating on swimming plates. After observing the desired movement
      behavior, the responding bacteria can be selected by simply pipetting out of the plate the semi solid agar where a movement halo is visible and diluting it in LB. The culture then needs to be diluted and grown in M9 to get rid of any residual
      attractant and can then be re-plated, starting the next round of selection.
    </DIV><DIV class="sectionh2">Conclusion
    </DIV><DIV class="text">
      We established a simple, inexpensive and relatively fast assay to screen large libraries of chemotactic mutants; complementing it with a model that can support the process of tuning multiple parameters.
    </DIV><DIV class="sectionh1 contentTable contentRef" contenttable="References" contentref=".contentRef">References</DIV><DIV class="references"><OL><LI>
          [1] Park, Hahnbeom, Wonpil Im, and Chaok Seok. “Transmembrane Signaling of Chemotaxis Receptor Tar: Insights from Molecular Dynamics Simulation Studies.” Biophysical Journal 100.12 (2011): 2955–2963. PMC. Web. 17 Oct. 2018.
        </LI><LI>
          [2] Derr, Paige, et al. “Changing the Specificity of a Bacterial Chemoreceptor.” Journal of Molecular Biology, vol. 355, no. 5, 2006, pp. 923–932., doi:10.1016/j.jmb.2005.11.025.
        </LI><LI>
          [3] Bi, Shuangyu et al. “Engineering Hybrid Chemotaxis Receptors in Bacteria.” ACS Synthetic Biology 5.9 (2016): 989–1001. Web.

        </LI><LI>
          [4] Yeh, Joanne I. et al. “High-Resolution Structures of the Ligand Binding Domain of the Wild-Type Bacterial Aspartate Receptor.” Journal of Molecular Biology 262.2 (1996): 186–201. Web.

        </LI><LI>
          [5] Kille, Sabrina et al. “Reducing Codon Redundancy and Screening Effort of Combinatorial Protein Libraries Created by Saturation Mutagenesis.” ACS Synthetic Biology 2.2 (2013): 83–92. Web.

        </LI></OL></DIV></DIV><DIV class="mobileMenuBg"><FOOTER><DIV class="sponsor-title">Sponsors</DIV><DIV class="multimedia-title">Contact Us</DIV></FOOTER></DIV></DIV></DIV></DIV></DIV></DIV></BODY></HTML>