<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_ETH_Zurich_ApproachA skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><DIV id="top_title"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:ETH Zurich/ApproachA</SPAN></H1></DIV><DIV id="HQ_page"><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><DIV class="container-nav-menu"><DIV class="nav-menu"><DIV class="nav-menu-left "><DIV class="nav-item" id="nav-item1"><SPAN>Project</SPAN><UL class="submenu"><LI><A href="https://2018.igem.org/Team:ETH_Zurich/Description">Description</A></LI><LI><A href="https://2018.igem.org/Team:ETH_Zurich/Applied_Design">Applied Design</A></LI><LI><A href="https://2018.igem.org/Team:ETH_Zurich/Team">Team Members</A></LI><LI><A href="https://2018.igem.org/Team:ETH_Zurich/Attributions">Attributions</A></LI><LI><A href="https://2018.igem.org/Team:ETH_Zurich/Sponsoring">Sponsoring</A></LI><LI><A href="https://2018.igem.org/Team:ETH_Zurich/Notebook">Notebook</A></LI><LI><A href="https://2018.igem.org/Team:ETH_Zurich/Safety">Safety</A></LI><LI><A href="https://2018.igem.org/Team:ETH_Zurich/Download">Downloads</A></LI></UL></DIV><DIV class="nav-item" id="nav-item2"><SPAN>Wet Lab</SPAN><UL class="submenu"><LI><A href="https://2018.igem.org/Team:ETH_Zurich/Tar">Tar Receptor Evolution</A></LI><LI><A href="https://2018.igem.org/Team:ETH_Zurich/ApproachA">Approach A</A></LI><LI><A href="https://2018.igem.org/Team:ETH_Zurich/ApproachB">Approach B</A></LI><LI><A href="https://2018.igem.org/Team:ETH_Zurich/Experiments">Experiments</A></LI><LI><A href="https://2018.igem.org/Team:ETH_Zurich/InterLab">InterLab</A></LI></UL></DIV><DIV class="nav-item" id="nav-item3"><SPAN>Hardware</SPAN><UL class="submenu"><LI><A href="https://2018.igem.org/Team:ETH_Zurich/Hardware">Hardware Overview</A></LI><LI><A href="https://2018.igem.org/Team:ETH_Zurich/Microscope">Microscope</A></LI><LI><A href="https://2018.igem.org/Team:ETH_Zurich/Mechanics">Mechanics</A></LI><LI><A href="https://2018.igem.org/Team:ETH_Zurich/SyringePump">Syringe Pump</A></LI><LI><A href="https://2018.igem.org/Team:ETH_Zurich/MicrofluidicChip">Microfluidic Chip</A></LI><LI><A href="https://2018.igem.org/Team:ETH_Zurich/bubbling">Bubbling</A></LI></UL></DIV><DIV class="nav-item" id="nav-item4"><SPAN>Software</SPAN><UL class="submenu"><LI><A href="https://2018.igem.org/Team:ETH_Zurich/Software">Software Overview</A></LI><LI><A href="https://2018.igem.org/Team:ETH_Zurich/Motility_Readout">Motility Readout</A></LI><LI><A href="https://2018.igem.org/Team:ETH_Zurich/ImageAnalysis">Luminescence Algorithm</A></LI><LI><A href="https://2018.igem.org/Team:ETH_Zurich/RobotControls">Robot Controls</A></LI><LI><A href="https://2018.igem.org/Team:ETH_Zurich/Path_Planning">Path Planning</A></LI></UL></DIV></DIV><DIV class="nav-menu-middle "><DIV class="nav-item" id="home"><A href="https://2018.igem.org/Team:ETH_Zurich"><TITLE id="title6">Logo Website</TITLE></A></DIV></DIV><DIV class="nav-menu-right "><DIV class="nav-item" id="nav-item5"><SPAN>Model</SPAN><UL class="submenu"><LI><A href="https://2018.igem.org/Team:ETH_Zurich/Model">Model Overview</A></LI><LI><A href="https://2018.igem.org/Team:ETH_Zurich/integratedModel">Integrated Models</A></LI><LI><A href="https://2018.igem.org/Team:ETH_Zurich/motilityModel">Motility Model</A></LI><LI><A href="https://2018.igem.org/Team:ETH_Zurich/splitModel">Split Luciferase Model</A></LI><LI><A href="https://2018.igem.org/Team:ETH_Zurich/diffusionModel">Diffusion Model</A></LI><LI><A href="https://2018.igem.org/Team:ETH_Zurich/bubbelingModel">Bubbling Model</A></LI><LI><A href="https://2018.igem.org/Team:ETH_Zurich/HolographicModel">Holographic Imaging Model</A></LI><LI><A href="https://2018.igem.org/Team:ETH_Zurich/Screening">Screening Assay Model</A></LI><LI><A href="https://2018.igem.org/Team:ETH_Zurich/Parameter">Model Parameters</A></LI></UL></DIV><DIV class="nav-item" id="nav-item6"><SPAN>Human Practices</SPAN><UL class="submenu"><LI><A href="https://2018.igem.org/Team:ETH_Zurich/Human_Practices">Human Practices</A></LI><LI><A href="https://2018.igem.org/Team:ETH_Zurich/Public_Engagement">Education &amp; Engagement</A></LI></UL></DIV><DIV class="nav-item" id="nav-item7"><SPAN>Achievements</SPAN><UL class="submenu"><LI><A href="https://2018.igem.org/Team:ETH_Zurich/Results">Results</A></LI><LI><A href="https://2018.igem.org/Team:ETH_Zurich/Parts">Parts Overview</A></LI><LI><A href="https://2018.igem.org/Team:ETH_Zurich/Demonstrate">Demonstrate</A></LI><LI><A href="https://2018.igem.org/Team:ETH_Zurich/Collaborations">Collaborations</A></LI></UL></DIV></DIV></DIV><DIV class="container-content"><DIV id="tongue1" class="content-tongue"><DIV class="tongue-div" id="tongue-div1">Content</DIV></DIV><DIV class="content-text"><SPAN href="section-Introduction" class="content-item">Introduction</SPAN><SPAN href="container_table" class="content-item">Table</SPAN></DIV></DIV></DIV><DIV class="main-menu"><A href="https://2018.igem.org/Team:ETH_Zurich"><DIV class="home">
        Home
      </DIV></A><DIV class="main-titel"><A href="https://2018.igem.org/Team:ETH_Zurich">
        A<SPAN>.</SPAN>R<SPAN>.</SPAN>O<SPAN>.</SPAN>M<SPAN>.</SPAN>A<SPAN class="main-letterSpacing">.</SPAN></A></DIV><A href="https://2018.igem.org/Team:ETH_Zurich"><DIV class="menu-logo"><TITLE id="title6">Logo Website</TITLE></DIV></A></DIV><DIV class="main-submenu"><DIV class="alt-main-submenu-item" id="alt-nav-item1" href="#project"><SPAN>Project</SPAN><UL class="alt-submenu"><LI><A href="https://2018.igem.org/Team:ETH_Zurich/Description">Description</A></LI><LI><A href="https://2018.igem.org/Team:ETH_Zurich/Applied_Design">Applied Design</A></LI><LI><A href="https://2018.igem.org/Team:ETH_Zurich/Team">Team Members</A></LI><LI><A href="https://2018.igem.org/Team:ETH_Zurich/Attributions">Attributions</A></LI><LI><A href="https://2018.igem.org/Team:ETH_Zurich/Sponsoring">Sponsoring</A></LI><LI><A href="https://2018.igem.org/Team:ETH_Zurich/Notebook">Notebook</A></LI><LI><A href="https://2018.igem.org/Team:ETH_Zurich/Safety">Safety</A></LI><LI><A href="https://2018.igem.org/Team:ETH_Zurich/Download">Downloads</A></LI></UL></DIV><DIV class="alt-main-submenu-item" id="alt-nav-item2" href="#parts"><SPAN>Wet Lab</SPAN><UL class="alt-submenu"><LI><A href="https://2018.igem.org/Team:ETH_Zurich/Tar">Tar Receptor Evolution</A></LI><LI><A href="https://2018.igem.org/Team:ETH_Zurich/ApproachA">Approach A</A></LI><LI><A href="https://2018.igem.org/Team:ETH_Zurich/ApproachB">Approach B</A></LI><LI><A href="https://2018.igem.org/Team:ETH_Zurich/Experiments">Experiments</A></LI><LI><A href="https://2018.igem.org/Team:ETH_Zurich/InterLab">InterLab</A></LI></UL></DIV><DIV class="alt-main-submenu-item" id="alt-nav-item3" href="#modelling"><SPAN>Hardware</SPAN><UL class="alt-submenu"><LI><A href="https://2018.igem.org/Team:ETH_Zurich/Hardware">Hardware Overview</A></LI><LI><A href="https://2018.igem.org/Team:ETH_Zurich/Microscope">Microscope</A></LI><LI><A href="https://2018.igem.org/Team:ETH_Zurich/Mechanics">Mechanics</A></LI><LI><A href="https://2018.igem.org/Team:ETH_Zurich/SyringePump">Syringe Pump</A></LI><LI><A href="https://2018.igem.org/Team:ETH_Zurich/MicrofluidicChip">Microfluidic Chip</A></LI><LI><A href="https://2018.igem.org/Team:ETH_Zurich/bubbling">Bubbling</A></LI></UL></DIV><DIV class="alt-main-submenu-item" id="alt-nav-item4"><SPAN>Software</SPAN><UL class="alt-submenu"><LI><A href="https://2018.igem.org/Team:ETH_Zurich/Software">Software Overview</A></LI><LI><A href="https://2018.igem.org/Team:ETH_Zurich/Motility_Readout">Motility Readout</A></LI><LI><A href="https://2018.igem.org/Team:ETH_Zurich/ImageAnalysis">Luminescence Algorithm</A></LI><LI><A href="https://2018.igem.org/Team:ETH_Zurich/RobotControls">Robot Controls</A></LI><LI><A href="https://2018.igem.org/Team:ETH_Zurich/Path_Planning">Path Planning</A></LI></UL></DIV><DIV class="alt-main-submenu-item" id="alt-nav-item5"><SPAN>Model</SPAN><UL class="alt-submenu"><LI><A href="https://2018.igem.org/Team:ETH_Zurich/Model">Model Overview</A></LI><LI><A href="https://2018.igem.org/Team:ETH_Zurich/integratedModel">Integrated Models</A></LI><LI><A href="https://2018.igem.org/Team:ETH_Zurich/motilityModel">Motility Model</A></LI><LI><A href="https://2018.igem.org/Team:ETH_Zurich/splitModel">Split Luciferase Model</A></LI><LI><A href="https://2018.igem.org/Team:ETH_Zurich/diffusionModel">Diffusion Model</A></LI><LI><A href="https://2018.igem.org/Team:ETH_Zurich/bubbelingModel">Bubbling Model</A></LI><LI><A href="https://2018.igem.org/Team:ETH_Zurich/Screening">Screening Assay Model</A></LI><LI><A href="https://2018.igem.org/Team:ETH_Zurich/HolographicModel">Holographic Imaging Model</A></LI></UL></DIV><DIV class="alt-main-submenu-item" id="alt-nav-item6"><SPAN>Human Practices</SPAN><UL class="alt-submenu"><LI><A href="https://2018.igem.org/Team:ETH_Zurich/Human_Practices">Human Practices</A></LI><LI><A href="https://2018.igem.org/Team:ETH_Zurich/Public_Engagement">Education &amp; Engagement</A></LI></UL></DIV><DIV class="alt-main-submenu-item" id="alt-nav-item7"><SPAN>Achievements</SPAN><UL class="alt-submenu"><LI><A href="https://2018.igem.org/Team:ETH_Zurich/Results">Results</A></LI><LI><A href="https://2018.igem.org/Team:ETH_Zurich/Parts">Parts Overview</A></LI><LI><A href="https://2018.igem.org/Team:ETH_Zurich/Demonstrate">Demonstrate</A></LI><LI><A href="https://2018.igem.org/Team:ETH_Zurich/Collaborations">Collaborations</A></LI></UL></DIV></DIV><DIV class="mobileMenuBg" style="visibility: hidden"><DIV class="container-titel"><DIV class="titel" id="titel">
        Approach A: Chemotaxis based<SPAN>.</SPAN></DIV><DIV class="scroll">Scroll</DIV></DIV><DIV class="sectionh1 contentTable contentFast" contenttable="Fast Biosensing" contentref=".contentFast">Fast Biosensing</DIV><DIV class="sectionh2">Introduction</DIV><DIV class="text">
      Our approach to creating a fast biosensor is based on post-translational processes to circumvent slow acting mechanisms like gene expression currently employed in most cell-based biosensors. To this end, we exploit the naturally occurring and
      highly adaptive Tar receptor that can be readily engineered to respond to new ligands. The use of phosphorylation and protein-protein interactions allows us to generate a readout in the time scale of seconds. In our project, we propose two
      such approaches for a biosensor reacting to molecules of interest.

    </DIV><DIV class="sectionh1 contentTable contentChemo" contenttable="Approach A: Chemotaxis" contentref=".contentChemo">Approach A: Chemotaxis</DIV><DIV class="sectionh2">Concept</DIV><DIV class="text clearfix counterFigure"><DIV class="textpicture-right textpicture40"><DIV class="figure_caption caption_left">This figure shows a cartoon representation of the chemotaxis pathway, which we utilize for our biosensor. The Tar receptor on the left, senses a specific molecule and induces a fast phosphorylation cascade. This results in a change in the rotational bias of the flagellum, i.e. the proportion of CW vs CCW rotations.</DIV></DIV>

      The underlying pathway of the first approach is the natural bacterial chemotaxis pathway. Chemotaxis is the mechanism underlying the movement of bacteria along a concentration gradient. E. coli swim mainly with their flagella, which can adopt two separate modes of rotation: counterclockwise (CCW) and clockwise (CW), resulting in a straight run forward or a tumble to change direction, respectively. The binding of the ligand to the Tar receptor induces a fast phosphorylation cascade via CheA and CheY followed by a decrease in the bacterium‚Äôs natural tumbling frequency, i.e. its bias towards the CCW rotation (see Fig.1).
    To utilize this process as a biosensor, we tethered the bacteria specifically at their flagella to a glass surface.[1] For this, we coated the glass surface with an antibody specific for the flagellar protein FliC. The antibody binds to the surface due to intermolecular interactions with the silica. [2] The tethering results in a translation of the flagellar rotation into a rotation of the whole bacterial body (see Fig. 2). Hence, a change in ligand concentration directly influences the rotational bias of the tethered bacteria.
    </DIV><DIV class="text clearfix counterFigure"><DIV class="textpicture-left textpicture20"><DIV class="figure_caption caption_left">This figure shows a cartoon of a tethered bacterium. The cell is specifically tethered to the glass surface at its flagellum. This translates the flagellar rotation into a spinning of the whole bacterial body.</DIV></DIV>
    The natural chemotaxis pathway contains an adaptation system, consisting of the proteins CheR and CheB, which respectively methylate or demethylate the Tar receptor, depending on the ligand concentration in the environment. This system allows the bacteria to adjust their receptor sensitivity continuously. CheR and CheB react in the order of a few seconds. Hence, the expected signal of rotational bias is a continuous CCW rotation right after induction, which persists for several seconds to a few minutes depending on the size of the step change of the concentration and re-adapts to the initial value (see Model Link).  Through image analysis, this information can be directly used as the output of the biosensor.
    </DIV><DIV class="sectionh2">Imaging</DIV><DIV class="text clearfix counterFigure"><DIV class="textpicture-left textpicture20"><DIV class="figure_caption caption_left">Measurement</DIV></DIV>
      The bacteria are tethered within a microfluidic channel to confine them locally and to conduct specific medium exchanges. The rotational bias resulting from the concentration difference is imaged by acquiring a video of several spinning
      bacteria. To avoid aliasing when capturing the rotation, the acquisition frame rate is 28 frames per second to safely resolve the rotation frequency of 2-9 Hz[1]. The images are acquired with a light microscope, developed by our team.
      By utilizing a microfluidic chip, we confine the bacteria to a known location. This has the advantage that we can image their response to multiple subsequent concentration changes.

    </DIV><DIV class="sectionh2">Processing</DIV><DIV class="text">
      To read out the rotational bias of the bacteria, a thorough in silico analysis is conducted for every video. First, rotating cells are selected, while completely fixed and freely moving ones are disregarded. From the small selection of
      spinning bacteria, only ones rotating nicely around a clear axis are used for analysis. Then, the bacteria‚Äôs rotation is analyzed frame by frame by reading out the angle between the bacteria‚Äôs position for subsequent frames. The rotational
      bias is then computed by comparing the bacteria‚Äôs natural tumbling frequency, which is 1 Hz on average [3], to its frequency right after the concentration change. This yields the desired output of the biosensor.


    </DIV><DIV class="sectionh2">Chemotaxis as a biosensor <SPAN class="italic">to control the robot</SPAN></DIV><DIV class="text">
      The increase in ligand concentration will directly lead to a change in the rotational bias of the bacterium - an increase yields longer CCW rotation, while a decrease yields more tumbles. This information is directly gathered from the imaging
      data. Since the bacteria quickly adapt to the new concentrations and restore their original tumbling frequency, the field of view can be kept on the same population of bacteria, which will adapt to the concentration change and therefore can be
      used for subsequent measurements.

    </DIV><DIV class="sectionh1 contentTable contentExperiments" contenttable="Experiments Approach A: Chemotaxis" contentref=".contentExperiments">Experiments Approach A: Chemotaxis
    </DIV><DIV class="text">
      The first try to image the reaction of the bacteria to a ligand was done by simply image the motility of freely moving bacteria on a coverslip. The bacteria showed very high activity, however, it quickly turned out to be impossible to track
      the cells on the coverslip since they are moving in three dimensions, i.e. in and out of focus.
      Afterwards, they were observed using a counting slide, which is confined in the z-direction. There, the first aspartate experiments were done by simply pipetting in the desired concentration into the counting slide. The experiment was done in
      M9 medium, which did not contain any aspartate. This allowed us to have control over the final aspartate concentration. Even if the problem of bacteria moving out of the focal plane was no longer existing, there were still several other
      problems persisting:
    </DIV><DIV class="sectionh2">Imaging bacterial motility
    </DIV>


      The first try to image the reaction of the bacteria to a ligand was done by simply image the motility of freely moving bacteria on a coverslip. The bacteria showed very high activity, however, it quickly turned out to be impossible to track
      the cells on the coverslip since they are moving in three dimensions, i.e. in and out of focus.
      Afterwards, they were observed using a counting slide, which is confined in the z-direction. There, the first aspartate experiments were done by simply pipetting in the desired concentration into the counting slide. The experiment was done in
      M9 medium, which did not contain any aspartate. This allowed us to have control over the final aspartate concentration. Even if the problem of bacteria moving out of the focal plane was no longer existing, there were still several other
      problems persisting:

      <OL><LI>
          The bacteria were flushed away, whenever medium containing aspartate was added
        </LI><LI>
          Aspartate was mixed with the already existing medium. This does not create a homogeneous mixture as desired for a reproducible response.

        </LI><LI>
          Tracking bacteria is difficult and they are moving out of the field of view

        </LI><LI>
          Subsequent measurements of the same cell are not possible

        </LI></OL>

      From these problems, the medium exchange combined with the measurement of the same bacteria were regarded as the most important criteria to have in our biosensor, since it allows a fast and reliable output. We considered using valves for a
      microfluidic system, magnets for ferritin expressing bacteria or even attaching magnetic beads to the cells to hold them in place. However, the most simple solution seemed to be to specifically tether the bacteria to the surface at their
      flagella.[1,4] This would prevent all the previously stated problems, i.e. it allows the medium exchange with a measurement of the same bacteria. Additionally, the readout of a rotation is much simpler than tracking bacteria in 3D.

    </DIV><DIV class="sectionh2">Tethering bacteria to a glass surface
    </DIV><DIV class="text">
      This experiment is both simple to set up and analyze. The cells were grown, as before, in M9 medium at 30¬∞C, which yields an average of 1.5 flagella per cell[5]. This is crucial to tether most of the bacteria at only one flagellum. For highest
      motility, cells were harvested during the mid to late exponential growth phase and resuspended in a specific tethering medium. To truncate the flagella, the bacteria were pushed through a very thin needle. This is a necessary step so the
      bacteria bind to the antibody close to the membrane, which guarantees nice circular rotations.
      The microfluidic chip was prepared by incubating it with the antibody specific for the flagellar protein FliC. The antibody attaches to the glass surface, due to intermolecular forces with the glass‚Äô silica surface. The bacteria with the
      sheared off flagella were then also incubated inside the channels overnight at 4¬∞C to prevent cell division (see <A href="https://2018.igem.org/Team:ETH_Zurich/MicrofluidicChip">Microfluidic Chip</A>).
    </DIV><DIV class="sectionh2">Imaging the apartate response
    </DIV><DIV class="text">
      The procedure described in this section was the proof-of-concept experiment for the viability of ‚ÄúApproach A: Chemotaxis‚Äù as a biosensor. It showed the bacteria reacting to step changes in concentration of the natural Tar ligand aspartate. The
      readout was acquired by imaging many bacteria with a 20x magnification phase contrast microscope. The low magnification allowed us to have enough resolution to resolve single spinning bacteria, while guaranteeing a high enough frame rate (25
      frames per seconds) to also resolve the bacteria‚Äôs natural rotational frequency (2-9 Hz). As seen in (video of spinning bacteria) only about 5-10% were spinning, a value which agrees with literature [citation needed].
      Several dilutions of aspartate ranging from 0.1 ŒºM to 1 mM were prepared. These concentrations correspond to the natural sensing range of the Tar receptor.[citation needed] To expose the tethered cells to the different concentrations, the
      medium in the reservoir of the microfluidic chip was exchanged manually with a pipette. With the optimal flow rate of 10 ŒºL/min (see <A href="https://2018.igem.org/Team:ETH_Zurich/MicrofluidicChip">Microfluidic Chip</A>) the cells were induced
      resulting in the described change in rotational bias. We measured many different concentration changes; we increased from zero to low and high concentration and we made multiple stepwise increases or decreases.
    </DIV><DIV class="text clearfix counterVideo"><DIV class="textpicture-right textpicture40"><VIDEO controls=""><DIV class="video_caption caption_left">The video shows an overview of multiple bacteria spinning inside the microfluidic chip.</DIV></VIDEO></DIV>

      Originally this experiment was conducted by analyzing individual cells by eye.[3] We developed an image analysis algorithm (with the help of Corey Dominick from the University of Pittsburgh Link to attributions/integrated HP) to read out the
      rotation of the bacteria. This is achieved by determining the position of the individual bacteria in each frame of the video, which yields the rotational angle for subsequent frames (more details Link Software/Motility analysis). From this
      data the adaptation times for different changes can be read out directly. This data is important for tuning our model to our data (see Model).
      To yield the data in a form, which can be used to control the robot, the tumbling frequency is read out before and after induction. To this end, we created an algorithm, which reads out the number of tumbles over a certain amount of time using
      a ‚Äúsliding window‚Äù over the raw data.
    </DIV><DIV class="container_video" id="vid1"><DIV class="videos_container"><DIV class="video"><DIV class="subvideo_caption">Before</DIV></DIV><DIV class="video"><DIV class="subvideo_caption">AFTER</DIV></DIV><DIV class="video"><DIV class="subvideo_caption">ADAPTED</DIV></DIV></DIV><DIV class="video_caption caption_center">Measurement</DIV></DIV><DIV class="container_figure" id="fig1"><DIV class="figures_container"><DIV class="figure"><DIV class="subfigure_caption">Particle Standard Curve Log Scale</DIV></DIV></DIV><DIV class="figure_caption caption_center">This figure shows the response of one bacterium to a concentration change from 0 to 100 ŒºM. Top: This graph depicts the step increase of the aspartate concentration, which happens at 90 s after measurement start. After the medium exchange the concentration stays at this level. Middle: The graph shows the data acquired from the analysis of the bacterial rotation. The bacteria, which have been tethered to the glass surface spin either in CCW or CW direction. At the beginning of the time series the Tar receptor is fully adapted to the environmental concentration (which is 0 in this case), so it spins with its natural tumbling frequency (tumble = CW rotation). After 70 seconds the flow starts to increase for the medium exchange resulting in a forced stop in rotation. The medium containing the 100 ŒºM aspartate reaches the bacteria at 100 s (marked with the red arrow), after which the flow rate decreases over the next 10 seconds. This results in the start of the expected rotational behavior at 110 s. Towards the end of the graph the bacteria have adapted to the new concentration. Bottom: This graph is generated from the one in the middle by calculating the CW rate using a ‚Äúmoving window‚Äù. This data is used as the output of the biosensor.</DIV></DIV><DIV class="container_figure" id="fig1"><DIV class="figures_container"><DIV class="figure"><DIV class="subfigure_caption">Particle Standard Curve Log Scale</DIV></DIV></DIV><DIV class="figure_caption caption_center">This figure shows the chemotactic response of the tethered bacteria to a aspartate concentration change of 0 - 10 ŒºM. The information on how to read the graphs is described in figure 4. From the bottom graph it becomes clear that the adaptation time is a lot shorter for 10 ŒºM than for 100 ŒºM (compare figure 4).</DIV></DIV><DIV class="sectionh2">Follow-up experiment with ùù∞-methyl-aspartate
    </DIV><DIV class="text">
      Considering that aspartate is a natural nutrient for E. coli, we continued the response experiment with the related compound ùù∞-methyl-aspartate. This molecule has structural similarity to aspartate and is therefore also sensed by the Tar
      receptor. However, it is not degraded by the bacteria. Although no response was observed for very low concentrations of ùù∞-methyl-aspartate (around 1 ŒºM), we were still able to detect a rotational bias similar to aspartate. This proves the
      hypothesis that other non-nutrient molecules can be sensed by the Tar receptor.
    </DIV><DIV class="sectionh2">Sticky Flagella</DIV><DIV class="text">
      After promising result, we expanded on the idea of tethering bacteria to a glass surface by creating a flagellum, which sticks to objects, without an antibody as an intermediate molecule. This would make the setup even easier and cheaper to produce. The idea was to introduce a mutation in the flagellar gene fliC, which generates a fully functional but sticky version of the protein.[8,9,10] We generated the BioBrick ‚ÄúSticky fliC‚Äù (BBa_K2845000) as a basic part as described in literature, as well as a composite part with the Sticky fliC under a constitutive promoter plus RBS (BBa_K2845001).
    </DIV><DIV class="sectionh2">Conclusion</DIV><DIV class="text">
      The readout of the chemotactic response of bacteria turns out to be a viable approach as a fast cell-based biosensor. The post-translational pathway allowed us to measure a signal seconds after induction with different aspartate concentrations. The big advantage of this approach is that it does not contain any artificial protein interactions but is purely based on the natural pathway of the Tar receptor, which has been optimized over millions of years of evolution. This promises the speed required for our biology-electronics interface.



    <DIV class="sectionh2">References</DIV><DIV class="references"><UL><LI>
          [1] Silverman, Michael, and Melvin Simon. &quot;Flagellar rotation and the mechanism of bacterial motility.&quot; Nature 249.5452 (1974): 73.

        </LI><LI>
          [2] Guha, Suvajyoti, et al. &quot;Characterizing the adsorption of proteins on glass capillary surfaces using electrospray-differential mobility analysis.&quot; Langmuir 27.21 (2011): 13008-13014.

        </LI><LI>
          [3] Block, Steven M., Jeffrey E. Segall, and Howard C. Berg. &quot;Impulse responses in bacterial chemotaxis.&quot; Cell 31.1 (1982): 215-226.

        </LI><LI>
          [4] Berg, Howard C., and P. M. Tedesco. &quot;Transient response to chemotactic stimuli in Escherichia coli.&quot; Proceedings of the National Academy of Sciences 72.8 (1975): 3235-3239.
        </LI><LI>
          [5] Larsen, Steven H., et al. &quot;Change in direction of flagellar rotation is the basis of the chemotactic response in Escherichia coli.&quot; Nature 249.5452 (1974): 74.

        </LI><LI>
          [6] Kuwajima, G. O. R. O. &quot;Construction of a minimum-size functional flagellin of Escherichia coli.&quot; Journal of Bacteriology 170.7 (1988): 3305-3309.

        </LI><LI>
          [7] Berg, Howard C., and Linda Turner. &quot;Torque generated by the flagellar motor of Escherichia coli.&quot; Biophysical journal 65.5 (1993): 2201-2216.

        </LI><LI>
          [8] Scharf, Birgit E., et al. &quot;Control of direction of flagellar rotation in bacterial chemotaxis.&quot; Proceedings of the National Academy of Sciences 95.1 (1998): 201-206.

        </LI></UL></DIV></DIV><DIV class="mobileMenuBg"><FOOTER><DIV class="sponsor-title">Sponsors</DIV><DIV class="multimedia-title">Contact Us</DIV></FOOTER></DIV></DIV></DIV></DIV></DIV></DIV></BODY></HTML>