<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_ETH_Zurich_Human_Practices skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><DIV id="top_title"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:ETH Zurich/Human Practices</SPAN></H1></DIV><DIV id="HQ_page"><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><DIV class="container-nav-menu"><DIV class="nav-menu"><DIV class="nav-menu-left "><DIV class="nav-item" id="nav-item1"><SPAN>Project</SPAN><UL class="submenu"><LI><A href="https://2018.igem.org/Team:ETH_Zurich/Description">Description</A></LI><LI><A href="https://2018.igem.org/Team:ETH_Zurich/Applied_Design">Applied Design</A></LI><LI><A href="https://2018.igem.org/Team:ETH_Zurich/Team">Team Members</A></LI><LI><A href="https://2018.igem.org/Team:ETH_Zurich/Attributions">Attributions</A></LI><LI><A href="https://2018.igem.org/Team:ETH_Zurich/Sponsoring">Sponsoring</A></LI><LI><A href="https://2018.igem.org/Team:ETH_Zurich/Notebook">Notebook</A></LI><LI><A href="https://2018.igem.org/Team:ETH_Zurich/Safety">Safety</A></LI><LI><A href="https://2018.igem.org/Team:ETH_Zurich/Download">Downloads</A></LI></UL></DIV><DIV class="nav-item" id="nav-item2"><SPAN>Wet Lab</SPAN><UL class="submenu"><LI><A href="https://2018.igem.org/Team:ETH_Zurich/Tar">Tar Receptor Evolution</A></LI><LI><A href="https://2018.igem.org/Team:ETH_Zurich/ApproachA">Approach A</A></LI><LI><A href="https://2018.igem.org/Team:ETH_Zurich/ApproachB">Approach B</A></LI><LI><A href="https://2018.igem.org/Team:ETH_Zurich/Experiments">Experiments</A></LI><LI><A href="https://2018.igem.org/Team:ETH_Zurich/InterLab">InterLab</A></LI></UL></DIV><DIV class="nav-item" id="nav-item3"><SPAN>Hardware</SPAN><UL class="submenu"><LI><A href="https://2018.igem.org/Team:ETH_Zurich/Hardware">Hardware Overview</A></LI><LI><A href="https://2018.igem.org/Team:ETH_Zurich/Microscope">Microscope</A></LI><LI><A href="https://2018.igem.org/Team:ETH_Zurich/Mechanics">Mechanics</A></LI><LI><A href="https://2018.igem.org/Team:ETH_Zurich/SyringePump">Syringe Pump</A></LI><LI><A href="https://2018.igem.org/Team:ETH_Zurich/MicrofluidicChip">Microfluidic Chip</A></LI><LI><A href="https://2018.igem.org/Team:ETH_Zurich/bubbling">Bubbling</A></LI></UL></DIV><DIV class="nav-item" id="nav-item4"><SPAN>Software</SPAN><UL class="submenu"><LI><A href="https://2018.igem.org/Team:ETH_Zurich/Software">Software Overview</A></LI><LI><A href="https://2018.igem.org/Team:ETH_Zurich/Motility_Readout">Motility Readout</A></LI><LI><A href="https://2018.igem.org/Team:ETH_Zurich/ImageAnalysis">Luminescence Algorithm</A></LI><LI><A href="https://2018.igem.org/Team:ETH_Zurich/RobotControls">Robot Controls</A></LI><LI><A href="https://2018.igem.org/Team:ETH_Zurich/Path_Planning">Path Planning</A></LI></UL></DIV></DIV><DIV class="nav-menu-middle "><DIV class="nav-item" id="home"><A href="https://2018.igem.org/Team:ETH_Zurich"><TITLE id="title6">Logo Website</TITLE></A></DIV></DIV><DIV class="nav-menu-right "><DIV class="nav-item" id="nav-item5"><SPAN>Model</SPAN><UL class="submenu"><LI><A href="https://2018.igem.org/Team:ETH_Zurich/Model">Model Overview</A></LI><LI><A href="https://2018.igem.org/Team:ETH_Zurich/integratedModel">Integrated Models</A></LI><LI><A href="https://2018.igem.org/Team:ETH_Zurich/motilityModel">Motility Model</A></LI><LI><A href="https://2018.igem.org/Team:ETH_Zurich/splitModel">Split Luciferase Model</A></LI><LI><A href="https://2018.igem.org/Team:ETH_Zurich/diffusionModel">Diffusion Model</A></LI><LI><A href="https://2018.igem.org/Team:ETH_Zurich/bubbelingModel">Bubbling Model</A></LI><LI><A href="https://2018.igem.org/Team:ETH_Zurich/HolographicModel">Holographic Imaging Model</A></LI><LI><A href="https://2018.igem.org/Team:ETH_Zurich/Screening">Screening Assay Model</A></LI><LI><A href="https://2018.igem.org/Team:ETH_Zurich/Parameter">Model Parameters</A></LI></UL></DIV><DIV class="nav-item" id="nav-item6"><SPAN>Human Practices</SPAN><UL class="submenu"><LI><A href="https://2018.igem.org/Team:ETH_Zurich/Human_Practices">Human Practices</A></LI><LI><A href="https://2018.igem.org/Team:ETH_Zurich/Public_Engagement">Education &amp; Engagement</A></LI></UL></DIV><DIV class="nav-item" id="nav-item7"><SPAN>Achievements</SPAN><UL class="submenu"><LI><A href="https://2018.igem.org/Team:ETH_Zurich/Results">Results</A></LI><LI><A href="https://2018.igem.org/Team:ETH_Zurich/Parts">Parts Overview</A></LI><LI><A href="https://2018.igem.org/Team:ETH_Zurich/Demonstrate">Demonstrate</A></LI><LI><A href="https://2018.igem.org/Team:ETH_Zurich/Collaborations">Collaborations</A></LI></UL></DIV></DIV></DIV><DIV class="container-content"><DIV id="tongue1" class="content-tongue"><DIV class="tongue-div" id="tongue-div1">Content</DIV></DIV><DIV class="content-text"><SPAN href="section-Introduction" class="content-item">Introduction</SPAN><SPAN href="container_table" class="content-item">Table</SPAN></DIV></DIV></DIV><DIV class="main-menu"><A href="https://2018.igem.org/Team:ETH_Zurich"><DIV class="home">
        Home
      </DIV></A><DIV class="main-titel"><A href="https://2018.igem.org/Team:ETH_Zurich">
        A<SPAN>.</SPAN>R<SPAN>.</SPAN>O<SPAN>.</SPAN>M<SPAN>.</SPAN>A<SPAN class="main-letterSpacing">.</SPAN></A></DIV><A href="https://2018.igem.org/Team:ETH_Zurich"><DIV class="menu-logo"><TITLE id="title6">Logo Website</TITLE></DIV></A></DIV><DIV class="main-submenu"><DIV class="alt-main-submenu-item" id="alt-nav-item1" href="#project"><SPAN>Project</SPAN><UL class="alt-submenu"><LI><A href="https://2018.igem.org/Team:ETH_Zurich/Description">Description</A></LI><LI><A href="https://2018.igem.org/Team:ETH_Zurich/Applied_Design">Applied Design</A></LI><LI><A href="https://2018.igem.org/Team:ETH_Zurich/Team">Team Members</A></LI><LI><A href="https://2018.igem.org/Team:ETH_Zurich/Attributions">Attributions</A></LI><LI><A href="https://2018.igem.org/Team:ETH_Zurich/Sponsoring">Sponsoring</A></LI><LI><A href="https://2018.igem.org/Team:ETH_Zurich/Notebook">Notebook</A></LI><LI><A href="https://2018.igem.org/Team:ETH_Zurich/Safety">Safety</A></LI><LI><A href="https://2018.igem.org/Team:ETH_Zurich/Download">Downloads</A></LI></UL></DIV><DIV class="alt-main-submenu-item" id="alt-nav-item2" href="#parts"><SPAN>Wet Lab</SPAN><UL class="alt-submenu"><LI><A href="https://2018.igem.org/Team:ETH_Zurich/Tar">Tar Receptor Evolution</A></LI><LI><A href="https://2018.igem.org/Team:ETH_Zurich/ApproachA">Approach A</A></LI><LI><A href="https://2018.igem.org/Team:ETH_Zurich/ApproachB">Approach B</A></LI><LI><A href="https://2018.igem.org/Team:ETH_Zurich/Experiments">Experiments</A></LI><LI><A href="https://2018.igem.org/Team:ETH_Zurich/InterLab">InterLab</A></LI></UL></DIV><DIV class="alt-main-submenu-item" id="alt-nav-item3" href="#modelling"><SPAN>Hardware</SPAN><UL class="alt-submenu"><LI><A href="https://2018.igem.org/Team:ETH_Zurich/Hardware">Hardware Overview</A></LI><LI><A href="https://2018.igem.org/Team:ETH_Zurich/Microscope">Microscope</A></LI><LI><A href="https://2018.igem.org/Team:ETH_Zurich/Mechanics">Mechanics</A></LI><LI><A href="https://2018.igem.org/Team:ETH_Zurich/SyringePump">Syringe Pump</A></LI><LI><A href="https://2018.igem.org/Team:ETH_Zurich/MicrofluidicChip">Microfluidic Chip</A></LI><LI><A href="https://2018.igem.org/Team:ETH_Zurich/bubbling">Bubbling</A></LI></UL></DIV><DIV class="alt-main-submenu-item" id="alt-nav-item4"><SPAN>Software</SPAN><UL class="alt-submenu"><LI><A href="https://2018.igem.org/Team:ETH_Zurich/Software">Software Overview</A></LI><LI><A href="https://2018.igem.org/Team:ETH_Zurich/Motility_Readout">Motility Readout</A></LI><LI><A href="https://2018.igem.org/Team:ETH_Zurich/ImageAnalysis">Luminescence Algorithm</A></LI><LI><A href="https://2018.igem.org/Team:ETH_Zurich/RobotControls">Robot Controls</A></LI><LI><A href="https://2018.igem.org/Team:ETH_Zurich/Path_Planning">Path Planning</A></LI></UL></DIV><DIV class="alt-main-submenu-item" id="alt-nav-item5"><SPAN>Model</SPAN><UL class="alt-submenu"><LI><A href="https://2018.igem.org/Team:ETH_Zurich/Model">Model Overview</A></LI><LI><A href="https://2018.igem.org/Team:ETH_Zurich/integratedModel">Integrated Models</A></LI><LI><A href="https://2018.igem.org/Team:ETH_Zurich/motilityModel">Motility Model</A></LI><LI><A href="https://2018.igem.org/Team:ETH_Zurich/splitModel">Split Luciferase Model</A></LI><LI><A href="https://2018.igem.org/Team:ETH_Zurich/diffusionModel">Diffusion Model</A></LI><LI><A href="https://2018.igem.org/Team:ETH_Zurich/bubbelingModel">Bubbling Model</A></LI><LI><A href="https://2018.igem.org/Team:ETH_Zurich/Screening">Screening Assay Model</A></LI><LI><A href="https://2018.igem.org/Team:ETH_Zurich/HolographicModel">Holographic Imaging Model</A></LI></UL></DIV><DIV class="alt-main-submenu-item" id="alt-nav-item6"><SPAN>Human Practices</SPAN><UL class="alt-submenu"><LI><A href="https://2018.igem.org/Team:ETH_Zurich/Human_Practices">Human Practices</A></LI><LI><A href="https://2018.igem.org/Team:ETH_Zurich/Public_Engagement">Education &amp; Engagement</A></LI></UL></DIV><DIV class="alt-main-submenu-item" id="alt-nav-item7"><SPAN>Achievements</SPAN><UL class="alt-submenu"><LI><A href="https://2018.igem.org/Team:ETH_Zurich/Results">Results</A></LI><LI><A href="https://2018.igem.org/Team:ETH_Zurich/Parts">Parts Overview</A></LI><LI><A href="https://2018.igem.org/Team:ETH_Zurich/Demonstrate">Demonstrate</A></LI><LI><A href="https://2018.igem.org/Team:ETH_Zurich/Collaborations">Collaborations</A></LI></UL></DIV></DIV><DIV class="mobileMenuBg" style="visibility: hidden"><DIV class="container-titel"><DIV class="titel">
        Human
        Practices<SPAN>.</SPAN></DIV><DIV class="scroll">Scroll</DIV></DIV><DIV class="section-Introduction contentTable" contenttable="Introduction" contentref=".section-Introduction"><P>
        For Synthetic Biology in particular and scientific projects in general, we believe societal considerations should be upfront. Here we show how we went on and explored safety, responsibility, ethics and scientific relevance of our project.
        Furthermore, we demonstrate how we integrated gained insights throughout the design and execution of our project.
      </P></DIV><DIV class="sectionh1 contentTable contentOverview" contenttable="Overview" contentref=".contentOverview">Overview</DIV><DIV class="container-hpOveview"><DIV id="hp1" class="container-hpItem"><DIV class="hp_pictogram">Experts</DIV><DIV class="hp_content"><DIV class="hp-header">Interview with Experts</DIV><DIV class="text">
            We met with Auke Jan Ijspeert (Head of Biorobotics Laboratory, EPFL) and Jan Roelof van der Meer (Head of Department for Fundamental Microbiology, Université de Lausanne) to interview them about their project “Envirobot” (<A href="https://biorob.epfl.ch/envirobot" target="blank">biorob.epfl.ch/envirobot)
            </A>, biosensors and their general application in robotics.

          </DIV><DIV class="interview-keypoints"><DIV class="keypoints-titel">
              Keypoints of the Interview
            </DIV><DIV class="keypoints-list"><UL><LI>“Envirobot” had the aim to develop an eel-like robot with built-in biosensors that can either go on predefined missions or autonomously react to molecule concentrations in water.
                </LI><LI>The project was carried out over 4 years with more than 10 collaborators from 5 different research institutes in Switzerland. The material value of the final robot was about 10.000 USD.
                </LI><LI>When building the robot, main obstacles were the robustness and speed of conventional biosensors.
                </LI><LI>Due to legal regulations and missing contamination in swiss lakes, the robot could not be tested in the field and was therefore trained with modeling data or collected samples.
                </LI><LI>In general, biosensors outcompete electronic sensors regarding the number of detectable molecules, thus enabling their use for environmental monitoring.

                </LI></UL></DIV></DIV><DIV class="hp-header">Consulting Experts</DIV><DIV class="text">
            From the idea, over the design to the execution of our project, we consulted experts and were able to benefit from the knowledge in their respective research areas.

          </DIV><DIV class="container-consulting"><DIV class="consulting-topic"><DIV class="consulting-pictogram"><DIV>Uptake</DIV></DIV><DIV class="consulting-persons"><DIV class="consulting-person"><DIV class="person-box"><DIV class="hp-integrated">Integrated</DIV></DIV><DIV class="person-text"><SPAN>Prof. Petra Dittrich</SPAN><SPAN class="institute">ETH Zurich Biosystems Science and Engineering</SPAN>
                    Petra Dittrich is Associate Professor for Bioanalytics at the Department of Biosystems Science and Engineering since 2014. Her research in the field of lab-on-chip-technologies focuses on the miniaturization of high-sensitivity
                    devices for chemical and biological analyses, and microfluidic-aided organization of materials.

                  </DIV></DIV><DIV class="consulting-person"><DIV class="person-box"><DIV class="hp-integrated">Integrated</DIV></DIV><DIV class="person-text"><SPAN>Renaud Dubè</SPAN><SPAN class="institute">PhD at Autonomous Systems Lab (ASL) at ETH Zurich
                    </SPAN>
                    Renaud helped us in finding a suitable robotic platform for our project. He especially informed us about the most commonly used robotic platforms which were very important to us as they served as a reference when we decided to go
                    for our own design.
                  </DIV></DIV></DIV></DIV><DIV class="consulting-topic"><DIV class="consulting-pictogram"><DIV>Sensing</DIV></DIV><DIV class="consulting-persons"><DIV class="consulting-person"><DIV class="person-box"><DIV class="hp-integrated">Integrated</DIV></DIV><DIV class="person-text"><SPAN>Prof. Sarah Maddocks</SPAN><SPAN class="institute"> Cardiff Metropolitan University</SPAN>
                    In a publication from 2008 (Maddocks and Oyston), Sarah reviewed the structure and function of LysR-type transcriptional regulator (LTTR) family proteins. Since they are the largest family of prokaryotic DNA-binding proteins, we
                    wanted to know whether our idea of visualizing transcription factor binding to DNA could be realized with them. LTTRs bind constitutively to DNA and activate transcription by changing the DNA bending. We discussed possible
                    options of visualizing this DNA bending but Sarah recommended not to use them for our project but rather look for a system where DNA binding is induced by a signal molecule, which we found in the EnvZ/OmpR system.
                    

                    Maddocks, Sarah E., and Petra C. F. Oyston. “Structure and Function of the LysR-Type Transcriptional Regulator ( LTTR ) Family Proteins.” Microbiology, vol. 154, 2008, pp. 3609–23, doi:10.1099/mic.0.2008/022772-0.

                  </DIV></DIV><DIV class="consulting-person"><DIV class="person-box"><DIV class="hp-integrated">Integrated</DIV></DIV><DIV class="person-text"><SPAN>Prof. Urs Jenal</SPAN><SPAN class="institute">Head of the research group for microbiology at the Biozentrum Basel</SPAN>
                    Prof. Urs Jenal’s research addresses basic molecular principles of bacterial signal transduction. His in-depth knowledge about microbiology helped us building the foundation of our project, which focuses on the bacterial
                    chemotaxis pathway. We gained a lot of insight and received valuable advice about the chemotactic signal processing in bacteria.

                  </DIV></DIV><DIV class="consulting-person"><DIV class="person-box"><DIV class="hp-integrated">Integrated</DIV></DIV><DIV class="person-text"><SPAN>Dr. Hannes Link
                    </SPAN><SPAN class="institute">Research group leader at the Max Planck Institute for Terrestrial Microbiology in Marburg</SPAN>
                    Hannes works on engineering allosteric enzymes for biotechnology. We consulted him and pitched our project idea of visualizing transcription factor binding to DNA. He confirmed that our system could in theory work and suggested
                    to test various amino acid linkers. Based on this advice, we designed our approach in a modular manner with different amino acid linkers consisting of mixtures of flexible and rigid repeats.


                  </DIV></DIV></DIV></DIV><DIV class="consulting-topic"><DIV class="consulting-pictogram"><DIV>Imaging</DIV></DIV><DIV class="consulting-persons"><DIV class="consulting-person"><DIV class="person-text"><SPAN>Dr. Laurence Wilson
                    </SPAN><SPAN class="institute">Lecturer in biophysics at the University of York
                    </SPAN>
                    Dr. Laurence Wilson as well as the York iGEM team of 2017 provided us advice on our lensless microscopy setup. They advised us to go for a monochromatic light source and pointed out that for resolving extremely small object with
                    very little contrast (like E.coli) an additional magnification might be necessary.

                  </DIV></DIV><DIV class="consulting-person"><DIV class="person-box"><DIV class="hp-integrated">Integrated</DIV></DIV><DIV class="person-text"><SPAN>Tom Lummen
                    </SPAN><SPAN class="institute">Microscopy expert at the Single Cell Facility (SCF), ETH Zurich
                    </SPAN>
                    Tom supported us a lot in all questions concerning microscopy. He was the one who advised us to actually go for a brightfield microscope and therefore greatly influenced the successful outcome of our imaging solution. Furthermore
                    he helped us by providing microscopy training and during the assembly of our own robot.
                  </DIV></DIV></DIV></DIV><DIV class="consulting-topic"><DIV class="consulting-pictogram"><DIV>Processing</DIV></DIV><DIV class="consulting-persons"><DIV class="consulting-person"><DIV class="person-text"><SPAN>Andreas Cuny
                    </SPAN><SPAN class="institute">PhD at the Computational Systems Biology Group, ETH Zurich</SPAN>
                    Supported us in questions concerning the algorithmic design of our single-cell tracking algorithm as well as our luminescence imaging readout.
                  </DIV></DIV><DIV class="consulting-person"><DIV class="person-text"><SPAN>Aaron Ponti
                    </SPAN><SPAN class="institute">Data Management and Image Analysis Expert at the Single Cell Facility (SCF), ETH Zurich
                    </SPAN>
                    Supported us in questions concerning the algorithmic design of our single-cell tracking algorithm as well as our luminescence imaging readout.
                  </DIV></DIV></DIV></DIV></DIV></DIV></DIV><DIV id="hp2" class="container-hpItem"><DIV class="hp_pictogram">Safety</DIV><DIV class="hp_content"><DIV class="hp-header">Collaboration with the ETH Departement for Safety, Security, Health and Environment</DIV><DIV class="text"><DIV class="hp-integrated">Integrated</DIV>
            The main concern raised during our Human Practices efforts was the biocontainment of our engineered bacteria. For the identification of critical components in our robotic design, we cooperated with the ETH Department for Safety, Health
            and Environment. As a result of this discussen we revised the design of our robot and included safety precautions to prevent
            contamination of the environment by the genetically modified organisms.

            Have a look a the before and after design in the picture below.
          </DIV><DIV class="interview-keypoints"><DIV class="keypoints-titel">
              Keypoints of the Interview
            </DIV><DIV class="keypoints-list"><UL><LI>Robot parts containing engineered bacteria should be made from non-fragile material to avoid breaking in case of an accident.
                  </LI><LI>
                      On the robot, bacteria are either operated in standard plastic lab ware or specially 3D-printed parts made from Acrylonitrile Butadiene Styrene, a durable polymer known from Lego bricks.
                    </LI></UL><LI>Bacteria need to be stopped from escaping from the robot. Mechanically this could be solved by placing a tub, which contains an absorbent or antimicrobial reagent under the robotic platform.
                  </LI><LI>
                      We designed a 3D-printable double-walled case for all parts on the robot containing bacteria. The case has only one opening for the influx of bubbled medium, where a unidirectional valve can be placed. The compartment between
                      the two walls of the case can be filled with any antimicrobial reagent to ensure the elimination of any bacteria in the unlikely case of a container breakage.
                    </LI><LI>Before moving the robot outside of the lab, removal of any operated GMOs is required. Especially when bringing our robot to the U.S., we were recommended not to bring any material that was brought in contact with GMOs. For
                  cleaning, we were recommended to use a strong antiseptic like H2O2 as it causes no damage to the electronics.

                  </LI><LI>
                      When bringing our robot to the U.S., we are going to replace all parts that were possibly brought in contact with engineered bacteria. All other non-replaceable parts are going to be cleaned with H<SUB>2</SUB>O<SUB>2</SUB>.


                    </LI><LI>For our project, we were interested in finding out which components would be interesting to sense but primarily, which ones would be easy to use in a biosafety level one laboratory like ours. We were recommended to work with
                  non-toxic compounds in a proof of concept manner to minimize any risk to human health and the environment.
                  </LI><LI>
                      For the experimental part, we worked with the natural aspartate sensing system from E. coli. Our attempts in evolving the Tar receptor were carried out with vanillin, a volatile, non-hazardous food additive and with toluene
                      which was handled with all necessary safety measurements under a fume hood.
                    </LI><LI>During our meeting, we discussed the issue of dual use research, i. e. the potential risk of misusing the knowledge gained from our project by third parties.
                  </LI><LI>
                      To address this issue, we participated in the dual use initiative from the iGEM team Bielefeld and organized a presentation where we informed about the potential risks of dual use research. Although dual use can never be
                      excluded, the experiments we demonstrate with our robot do not exhibit a direct potential for being misused.

                    </LI><LI>In previous mechanical engineering projects at our university, teams were required to develop a safety concept.d
                  </LI><LI>
                      We voluntarily outlined a safety concept for the manufacturing and operation of our robot and sent it back to the ETH safety department for evaluation.
                    </LI></DIV></DIV><DIV class="ba-slider"><DIV class="img-before">Before</DIV><DIV class="img-after">After</DIV><DIV class="select-view"><SPAN view="view1">View 1</SPAN><SPAN view="view2">View 2</SPAN><SPAN view="view3">View 3</SPAN><SPAN view="view4">View 4</SPAN></DIV></DIV></DIV></DIV><DIV id="hp3" class="container-hpItem"><DIV class="hp_pictogram">Responsibility</DIV><DIV class="hp_content"><DIV class="hp-header">Interview with Greenpeace</DIV><DIV class="text">
            For the societal and environmental consideration of our project, we deemed it important to engage with critical audiences outside of our scientific niche. Greenpeace is one of the biggest non-governmental environmental organizations
            worldwide and has traditionally a rather critical opinion on genetic engineering. We interviewed Philippe Schenkel, Sustainable Agriculture Campaigner at Greenpeace Switzerland, and discussed the general position of Greenpeace on this
            topic, the iGEM competition, our project and the communication between scientists and NGOs.
          </DIV><DIV class="interview-keypoints"><DIV class="keypoints-titel">
              Keypoints of the Interview
            </DIV><DIV class="keypoints-list"><UL><LI>Greenpeace does not have a general position against genetic engineering, especially not in the research sector or for medical reasons. In agriculture however, they have a clear position against the use of GMOs as there is more
                  research needed to study the long time effects.
                </LI><LI>As long as safety standards are met and bacteria are kept inside the lab, the iGEM competition does not represent a conflict of interest with the principles of Greenpeace.

                </LI><LI>For our robot it is equally important that biocontainment is ensured. This can not only be supported by mechanical devices but also by the implementation of biological mechanisms like kill-switches or auxotrophies.

                </LI><LI>Already there is some knowledge exchange between Greenpeace and scientists who tend to be rather critical. Philippe encourages researchers to expand this dialogue in order to increase the understanding

                </LI></UL></DIV></DIV><DIV class="text"><DIV class="hp-integrated">Integrated</DIV>
            Our project exploits the ability of engineered bacteria to localize the source of a volatile molecule. Depending on the mechanical specifications and legal framework for our robot,  GMOs would be considered to be brought outside the laboratory.
            be brought outside the laboratory.
          </DIV><DIV class="text">
         

In the interview with Philippe Schenkel from Greenpeace, we discussed possibilities for biological mechanisms ensuring biocontainment. Instead of MG1655, the strain we used in our experiments, we propose to use the GRO (Genetically Recoded Organism) generated in the study from DJ Mandell et al. 2015, which uses the re-assigned codon UAG to code for L-4,49-biphenylalanine (bipA). The latter is a non-standard amino acid (NSAA), which cannot be found in nature. However, its is compatible with protein cores and is integrated into essential enzymes, causing NSAA auxotrophy.
In this way, even if the bacteria were to escape the mechanical barriers on our robot, they would not survive due to the artificially induced auxotrophy. Moreover, their genetic material could not be transmitted to other strains, since the information would be that of a truncated and therefore non-functional protein due to the recoding of the stop codon.


          </DIV><DIV class="text">

We decided to choose this novel approach as a biological containment strategy, since it shows unparalleled robustness to genetic drift. In the future, the realization of devices like AROMA will depend on legal frameworks defining what can be considered as an “extension of the laboratory”. For this, synthetic auxotrophies alongside mechanical barriers represent a useful tool to ensure biocontainment of GMOs.
          </DIV></DIV></DIV></DIV></DIV><DIV class="mobileMenuBg"><FOOTER><DIV class="sponsor-title">Sponsors</DIV><DIV class="multimedia-title">Contact Us</DIV></FOOTER></DIV></DIV></DIV></DIV></DIV></DIV></BODY></HTML>