Team:ETH Zurich/ApproachB
Project
Description
Applied Design
Team Members
Attributions
Sponsoring
Notebook
Safety
Downloads
Wet Lab
Tar Receptor Evolution
Approach A
Approach B
Experiments
InterLab
Hardware
Hardware Overview
Microscope
Mechanics
Syringe Pump
Microfluidic Chip
Bubbling
Software
Software Overview
Motility Readout
Luminescence Algorithm
Robot Controls
Path Planning
Logo Website
Model
Model Overview
Integrated Models
Motility Model
Split Luciferase Model
Diffusion Model
Bubbling Model
Holographic Imaging Model
Screening Assay Model
Model Parameters
Human Practices
Human Practices
Education & Engagement
Achievements
Results
Parts Overview
Demonstrate
Collaborations
Content
Introduction Table
Home
A.R.O.M.A.
Logo Website
Project
Description
Applied Design
Team Members
Attributions
Sponsoring
Notebook
Safety
Downloads
Wet Lab
Tar Receptor Evolution
Approach A
Approach B
Experiments
InterLab
Hardware
Hardware Overview
Microscope
Mechanics
Syringe Pump
Microfluidic Chip
Bubbling
Software
Software Overview
Motility Readout
Luminescence Algorithm
Robot Controls
Path Planning
Model
Model Overview
Integrated Models
Motility Model
Split Luciferase Model
Diffusion Model
Bubbling Model
Screening Assay Model
Holographic Imaging Model
Human Practices
Human Practices
Education & Engagement
Achievements
Results
Parts Overview
Demonstrate
Collaborations
DNA binding visualization with split-proteins.
Scroll
Introduction
Goal of this project
The aim of this series of experiments was the creation of a multi-component reporter system that allows the detection of an output signal in a timescale of seconds to minutes after the input molecule becomes present. To achieve this goal we exploit different aspects of biomolecular engineering such as chimeric receptors, fusion proteins, and split protein complementation to detect an early step of bacterial response to the ligand: the binding of transcription factors to their regulatory DNA elements.
Disadvantages of transcription-based biosensors
Among other biosensing approaches, cell-based strategies benefit the most from the tools offered by synthetic biology and genetic engineering. Based on extensively studied biomolecular processes that take place within the cell upon encounter with a ligand, they provide several advantages compared to other systems: modularity of engineered gene circuits, high selectivity of protein-ligand interactions, a wide sensitivity range, natural signal amplification, and the possibility of adapting the system through artificial evolution.
Despite these advantages, a common weakness of cell-based biosensing devices is their slow response compared to bioelectronic devices, since their output is normally based on imaging a reporter gene, expressed as a result of the presence of the analyte.  Molecular processes (mRNA transcription and maturation, protein translation and folding, post-translational modifications) taking place after analyte detection slow down output generation to range of hours. This timescale renders cell-based biosensors unsuitable for any application that requires an immediate readout. In the field, AROMA navigates autonomously while only reacting to its integrated biosensors, which therefore need to be as fast as possible to ensure the operation in a reasonable timescale.
Thinking fast biosensors
Gene expression involves the interaction of a transcription factor (TF) with a DNA sequence known as binding site to induce or repress transcription. The activity of a TF can be directly or indirectly modulated by a ligand. Although the expression of a reporter gene can take hours, the effect of the ligand on the TF-DNA interaction occurs in a matter of seconds. We aimed at exploiting this fast kinetics to circumvent the transcriptional/translational lag other biosensors, by coupling the TF-DNA binding to the likewise fast kinetics of split-protein complementation. Split proteins are fragmented reporter proteins that in order to produce a readable output, need to be reconstituted by bringing their components to close proximity. This is usually achieved by fusing them to proteins that co-localize under certain  circumstances. Split fluorescent proteins like split-GFP require a time-consuming maturation step after reconstitution (Cabantous et al.). Split luciferases, in contrast, do not require a folding step and start to emit light right after reassembly, making them suitable for our fast biosensing application. We chose a genetically modified version of the firefly luciferase from Photinus pyralis, named EPIC luciferase , which exhibits up to 10-fold higher bioluminescence than its natural version (Fujii et al.). For our approach, we split it as previously described by Luker et al. (2004), generating an N-terminal (NLuc: BBa_K2845003 ) and a C-terminal (CLuc: BBa_K2845002 ) fragment.
For the DNA binding itself, we looked at the two-component regulatory system OmpR/EnvZ. EnvZ is a membrane protein capable of sensing an increase of osmolarity, in which case it phosphorylates OmpR into OmpR-P, which then binds its operator sequence called ompC C1 to upregulate gene expression. Controlling the phosphorylation status of OmpR, its binding kinetics to DNA can be modified. This can be achieved by using the chimeric receptor Tar-EnvZ (or Taz) (Yoshida et al., 2007), which enables the activation of EnvZ histidine kinase activity in response of extracellular aspartate.
The extracellular part of Tar-EnvZ comes from Tar, one of the many chemotaxis receptors that have been widely studied for their potential application in biosensing, due to the wide range of different molecules they can sense.
Putting all these pieces together, we devised a multi-component biosensing system composed of a membrane receptor to sense an input ligand,  a pair of fusion proteins and their corresponding binding sites.
Diagram of the split-protein approach for the fast biosensing of a ligand.
Experimental design
After deciding on the strategy to follow for our fast biosensing system, we needed to turn our  ideas into a specific genetic circuit with the appropriate biomolecular pieces that allow us to achieve the desired specifications.
Because fusion protein design is not straightforward, we decided to test a variety of linkers for our gene constructs (more details in the following section). In addition to the linker used, the orientation and separation of TF operator sequences can also affect split protein complementation. To facilitate testing multiple combinations we used a cloning approach with 3 plasmids with compatible antibiotic resistances and origins of replication. However, different origins of replication usually imply dissimilar plasmid copy number. Therefore, we made the following design choices:
CLuc-TetR in pSEVA471 : since TetR has a much higher binding affinity for its operator sequence (tetO) it should be the one in the lower copy number plasmid.
OmpR-NLuc in pSB3C5 : OmpR has lower affinity for its operator sequence (ompC1), compared to TetR. Also, it is the molecule that will mediate the transduction of the input into the final signal. Therefore it was convenient to have it in higher copy number than TetR.
Binding sites in pSB1A3 : The operator sequences for TetR and OmpR (tetO and ompC1, respectively) were cloned into the highest copy number plasmid with the aim of maximizing signal amplification and avoiding saturation. Since cloning repetitive and palindromic sequences is challenging, we only cloned one repeat of each binding site in two different orientations  (tetO-ompC1 or ompC1-tetO), and placed 10, 15, 20 and 25 bp away. This resulted in 8 different binding site constructs.
Design of the cloning strategy for the fusion proteins and binding sites.
Experiments and results
Fusion protein design concepts
In order to use TF-DNA binding as a trigger for split protein complementation, each of these two proteins needs to be fused to one of the luciferase fragments, that will produce light when both proteins bind to DNA next to each other.
The design of fusion proteins that work as expected is challenging. The fusion protein needs to be expressed and fold correctly, while allowing its individual parts to keep their original functions. In these two aspects the choice of the correct amino acid linker sequence is critical. The mechanical properties of a linker depend on its amino acid sequence, and in general they can  flexible linkers which provide more movement freedom for the individual parts (i.e. easier complementing with the other fragment of the luciferase), or rigid linkers that prevent interference between the parts correct folding and function (Chen et al., 2013).
We aimed to find linkers that preserve both the ability of the transcription factors OmpR and TetR to bind their corresponding operator sequences, and the ability of the N and C terminal fragments of EPIC luciferase to complement each other and emit light. We decided to screen a variety of linkers with different combinations of flexible (GGGGS) and rigid (EAAAK) repeats. We nicknamed them by their sequence of flexible (F) and rigid (R) repeats.
We chose to test linkers made of 3 or 4 repeats since they have been reported to work more frequently than more complex linker structures, and they are also easier to clone than longer repetitive sequences. Since our screening capacity is limited by the timespan of the project, before tackling the wet lab experiments an in silico simulation was carried out for the docking on DNA of OmpR-NLuc and CLuc-TetR, both joined by a triple flexible linker. The results show that reconstitution is possible with these lengths of linkers.
For the fusion proteins that were successfully cloned, we measured to what extent the activity of the transcription factors and the luciferase fragments was conserved.
CLuc-TetR fusion protein characterization
The performance of the different CLuc-TetR fusion proteins was assessed by measuring their ability to repress mRFP expression. To this end, they were expressed together with mRFP under control of the pTet promoter. Our goal was to see high levels of repression indicating the retention of a strong binding affinity for the tetO operator sequence. We also tested wether our constructs retained the ability to be de-repressed by aTc (anhydrotetracycline).
With this idea in mind, we designed a multi-component biosensing system composed of a binding site array, a pair of fusion proteins and membrane receptor to sense the input ligand. Learn more of our system by scrolling         down to the details.
Experimental procedure
First, E. coli DH5a competent cells were transformed with the corresponding pSEVA471 plasmids carrying the CLuc-TetR fusion protein. A negative control without TetR fusion protein was transformed with an empty pSEVA471 plasmid. All bacteria were simultaneously transformed with a pSB1C3 plasmid carrying mRFP under pTet control, taken from the Distribution Kit ( BBa_I13521 ). Colonies were picked from the plates and grown overnight in M9+glucose media with the corresponding antibiotics. The next day, 1:100 dilutions of each culture were grown in triplicate until an OD600 of 0.5 and transferred to a black 96-well plate with transparent bottom for measurement. Measurements were carried out using a TECAN M1000 plate reader with the typical mRFP settings, first without aTc and then with 30 nM aTc. All measurements are normalized for OD600.
mRFP fluorescence before and after aTC addition for the different CLuc-TetR fusion proteins. Control strain does not express a TetR fusion protein. TetR strain expresses a constitutively active TetR protein. Error bars represent standard deviation (n=3).
As can be observed in Figure 3, there is diversity in the results, with most of the linkers allowing some degree of repression.  No particular rule for the content of rigid and flexible linkers can be deduced from the results. The best repression was achieved the CLuc-FFFF-TetR hybrid. This fusion protein was chosen as the most promising CLuc-TetR version in terms of DNA binding and therefore a strong candidate for further characterization.
Repression performance for CLuc-TetR fusion proteins. Fold change is the ratio of mRFP fluorescence before and after aTc addition. Maximum fluorescence is the one measured for the non-repressed mRFP control.
Fold change repression
% of maximum fluorescence
FFF
1.09
87.84
FRF
1.71
54.28
RFR
1.54
60.77
FFFF
4.33
17.8
RFFF
2.03
42.68
FFFR
1.72
56.02
RFFR
1.97
49
RFRF
0.99
92.44
FRFR
1.66
60.38
OmpR-NLuc fusion protein characterization
In a similar fashion to the previous experiments, we wanted to measure OmpR-NLuc fusion proteins ability to keep its function as a transcriptional up-regulator, since to act as such it needs to bind to DNA as a prerequisite. For this we used GFP under control of pOmpC promoter containing the C1 operator sequence for OmpR ( BBa_K1012005 ). This promoter can can be induced by subjecting the cells to high osmolarity conditions such as high sucrose concentration, activating EnvZ signaling. In case of retaining its transcriptional activator properties, a higher luminescence emission should be observed after osmolarity induction.
Experimental procedure
Experimental procedure: E. coli DH5a were co-transformed with the  corresponding OmpR-NLuc fusion proteins with different linkers (plus the empty plasmid pSB3C5 as a negative control) and pOmpC-GFP.  An OmpR knock-out strain (ΔOmpR) was transformed with pOmpC-GFP too as a negative control for OmpR transcriptional upregulation. Colonies were picked from the LB plates and grown in liquid LB media overnight with the corresponding antibiotics. Then, each culture was diluted 100 times in minimal M9 media + 1% glucose and grown at 37 °C until they reached OD600 of 0.5. Then, they were centrifuged and resuspended in M9 media with 20% (w/v) sucrose. After 1 hour, they were centrifuged, resuspended in PBS, centrifuged and resuspended in M9 media for measurement in a black flat bottom plate. Luminescence measurements were carried out using a TECAN M1000 plate reader. All measurements are normalized for OD600.
GFP fluorescence for different OmpR-NLuc constructs before and after induction of pOmpC with 20% sucrose. Error bars represent standard deviation (n=3).
The results shown in Figure 4 indicate that some of the OmpR-NLuc linkers allowed the fused OmpR to retain a significant amount of upregulation capability, specifically FFF, RFFF, and RFFR linkers. However, for these constructs the high expression of GFP in uninduced conditions highlights the leakiness of the pOmpC promoter that has been reported by previous iGEM teams (BBa_R0082). In line with expectations, for most constructs GFP fluorescence increased upon sucrose induction, albeit in a very low amount (34% in the best case for RFFR linker).
Split-luciferase background emission measurement
Even if fused to TetR and OmpR regulator proteins, the N and C-terminal fragments of firefly luciferase will float freely in the cytoplasm, which raises the concern of high background noise due to spontaneous reconstitution. In order to check the amount of background signal and at the same time test the potential of the split luciferase to emit light, we expressed both fragments in the same E. coli strain, either in a single polycistronic mRNA (BBa_K2845006) or in two different plasmids with separate promoters (BBa_K2845004 and BBa_K2845005)
Luminescence read-out before and after 20 mM luciferin addition for different combinations of split and full luciferase. Errors are standard deviation (n=3) except for the full luciferase (n=1).
The results (Figure 5) show an increase in light emission upon addition of luciferin, for both split luciferase expressed from one plasmid (12-fold increase) or two separate ones (23-fold increase). This increase constitutes a clear signal over the random background noise (under 100 relative luminescence units) measured with the negative controls containing only one part of the protein, allowing us to conclude that the split protein reassembles successfully, and also providing us with a measurement of the baseline luminescence that can be expected for the split complementation assays. For reference, we also included a strain expressing the full firefly luciferase, which as expected produced a substantially higher signal than its split version, increasing its brightness 375 times after substrate addition.
Conclusion
In conclusion, we can state that we made a big step towards our goal of visualizing binding of a transcription factor to DNA via split protein complementation. The next step will be to add up all single parts that showed  promising results, and to test the three components of the system together in the same E. coli strain. Right now we are working on carrying out the last cloning steps and hopefully we will already be able to present final results for this approach at the Giant Jamboree.
References
Cabantous, Stéphanie, et al. “A New Protein-Protein Interaction Sensor Based on Tripartite Split-GFP Association.” Scientific Reports, vol. 3, 2013, pp. 1–9, doi:10.1038/srep02854.
Chen, Xiaoying, Jennica L. Zaro, and Wei-Chiang Shen. "Fusion protein linkers: property, design and functionality." Advanced drug delivery reviews 65.10 (2013): 1357-1369.
Fujii, Hiroya, et al. Increase in Bioluminescence Intensity of Firefly Luciferase Using Genetic Modification. Vol. 366, 2007, pp. 131–36, doi:10.1016/j.ab.2007.04.018.
Kamiyama, Daichi, et al. “Versatile Protein Tagging in Cells with Split Fluorescent Protein.” Nature Communications, vol. 7, Nature Publishing Group, 2016, pp. 1–9, doi:10.1038/ncomms11046.
Luker, Kathryn E., et al. “Kinetics of Regulated Protein – Protein Interactions Revealed with Firefly Luciferase Complementation Imaging in Cells and Living Animals.” PNAS, vol. 101, no. 33, 2004.
Ozawa, Takeaki, et al. “Split Luciferase as an Optical Probe for Detecting Protein - Protein Interactions in Mammalian Cells Based on Protein Splicing.” Analytical Chemistry, vol. 73, no. 11, 2001, pp. 5187–92, doi:10.1021/ac0013296.
Sponsors
Contact Us
