<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_ETH_Zurich_splitModel skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><DIV id="top_title"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:ETH Zurich/splitModel</SPAN></H1></DIV><DIV id="HQ_page"><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><DIV class="container-nav-menu"><DIV class="nav-menu"><DIV class="nav-menu-left "><DIV class="nav-item" id="nav-item1"><SPAN>Project</SPAN><UL class="submenu"><LI><A href="https://2018.igem.org/Team:ETH_Zurich/Description">Description</A></LI><LI><A href="https://2018.igem.org/Team:ETH_Zurich/Applied_Design">Applied Design</A></LI><LI><A href="https://2018.igem.org/Team:ETH_Zurich/Team">Team Members</A></LI><LI><A href="https://2018.igem.org/Team:ETH_Zurich/Attributions">Attributions</A></LI><LI><A href="https://2018.igem.org/Team:ETH_Zurich/Sponsoring">Sponsoring</A></LI><LI><A href="https://2018.igem.org/Team:ETH_Zurich/Notebook">Notebook</A></LI><LI><A href="https://2018.igem.org/Team:ETH_Zurich/Safety">Safety</A></LI><LI><A href="https://2018.igem.org/Team:ETH_Zurich/Download">Downloads</A></LI></UL></DIV><DIV class="nav-item" id="nav-item2"><SPAN>Wet Lab</SPAN><UL class="submenu"><LI><A href="https://2018.igem.org/Team:ETH_Zurich/Tar">Tar Receptor Evolution</A></LI><LI><A href="https://2018.igem.org/Team:ETH_Zurich/ApproachA">Approach A</A></LI><LI><A href="https://2018.igem.org/Team:ETH_Zurich/ApproachB">Approach B</A></LI><LI><A href="https://2018.igem.org/Team:ETH_Zurich/Experiments">Experiments</A></LI><LI><A href="https://2018.igem.org/Team:ETH_Zurich/InterLab">InterLab</A></LI></UL></DIV><DIV class="nav-item" id="nav-item3"><SPAN>Hardware</SPAN><UL class="submenu"><LI><A href="https://2018.igem.org/Team:ETH_Zurich/Hardware">Hardware Overview</A></LI><LI><A href="https://2018.igem.org/Team:ETH_Zurich/Microscope">Microscope</A></LI><LI><A href="https://2018.igem.org/Team:ETH_Zurich/Mechanics">Mechanics</A></LI><LI><A href="https://2018.igem.org/Team:ETH_Zurich/SyringePump">Syringe Pump</A></LI><LI><A href="https://2018.igem.org/Team:ETH_Zurich/MicrofluidicChip">Microfluidic Chip</A></LI><LI><A href="https://2018.igem.org/Team:ETH_Zurich/bubbling">Bubbling</A></LI></UL></DIV><DIV class="nav-item" id="nav-item4"><SPAN>Software</SPAN><UL class="submenu"><LI><A href="https://2018.igem.org/Team:ETH_Zurich/Software">Software Overview</A></LI><LI><A href="https://2018.igem.org/Team:ETH_Zurich/Motility_Readout">Motility Readout</A></LI><LI><A href="https://2018.igem.org/Team:ETH_Zurich/ImageAnalysis">Luminescence Algorithm</A></LI><LI><A href="https://2018.igem.org/Team:ETH_Zurich/RobotControls">Robot Controls</A></LI><LI><A href="https://2018.igem.org/Team:ETH_Zurich/Path_Planning">Path Planning</A></LI></UL></DIV></DIV><DIV class="nav-menu-middle "><DIV class="nav-item" id="home"><A href="https://2018.igem.org/Team:ETH_Zurich"><TITLE id="title6">Logo Website</TITLE></A></DIV></DIV><DIV class="nav-menu-right "><DIV class="nav-item" id="nav-item5"><SPAN>Model</SPAN><UL class="submenu"><LI><A href="https://2018.igem.org/Team:ETH_Zurich/Model">Model Overview</A></LI><LI><A href="https://2018.igem.org/Team:ETH_Zurich/integratedModel">Integrated Models</A></LI><LI><A href="https://2018.igem.org/Team:ETH_Zurich/motilityModel">Motility Model</A></LI><LI><A href="https://2018.igem.org/Team:ETH_Zurich/splitModel">Split Luciferase Model</A></LI><LI><A href="https://2018.igem.org/Team:ETH_Zurich/diffusionModel">Diffusion Model</A></LI><LI><A href="https://2018.igem.org/Team:ETH_Zurich/bubbelingModel">Bubbling Model</A></LI><LI><A href="https://2018.igem.org/Team:ETH_Zurich/HolographicModel">Holographic Imaging Model</A></LI><LI><A href="https://2018.igem.org/Team:ETH_Zurich/Screening">Screening Assay Model</A></LI><LI><A href="https://2018.igem.org/Team:ETH_Zurich/Parameter">Model Parameters</A></LI></UL></DIV><DIV class="nav-item" id="nav-item6"><SPAN>Human Practices</SPAN><UL class="submenu"><LI><A href="https://2018.igem.org/Team:ETH_Zurich/Human_Practices">Human Practices</A></LI><LI><A href="https://2018.igem.org/Team:ETH_Zurich/Public_Engagement">Education &amp; Engagement</A></LI></UL></DIV><DIV class="nav-item" id="nav-item7"><SPAN>Achievements</SPAN><UL class="submenu"><LI><A href="https://2018.igem.org/Team:ETH_Zurich/Results">Results</A></LI><LI><A href="https://2018.igem.org/Team:ETH_Zurich/Parts">Parts Overview</A></LI><LI><A href="https://2018.igem.org/Team:ETH_Zurich/Demonstrate">Demonstrate</A></LI><LI><A href="https://2018.igem.org/Team:ETH_Zurich/Collaborations">Collaborations</A></LI></UL></DIV></DIV></DIV><DIV class="container-content"><DIV id="tongue1" class="content-tongue"><DIV class="tongue-div" id="tongue-div1">Content</DIV></DIV><DIV class="content-text"><SPAN href="section-Introduction" class="content-item">Introduction</SPAN><SPAN href="container_table" class="content-item">Table</SPAN></DIV></DIV></DIV><DIV class="main-menu"><A href="https://2018.igem.org/Team:ETH_Zurich"><DIV class="home">
        Home
      </DIV></A><DIV class="main-titel"><A href="https://2018.igem.org/Team:ETH_Zurich">
        A<SPAN>.</SPAN>R<SPAN>.</SPAN>O<SPAN>.</SPAN>M<SPAN>.</SPAN>A<SPAN class="main-letterSpacing">.</SPAN></A></DIV><A href="https://2018.igem.org/Team:ETH_Zurich"><DIV class="menu-logo"><TITLE id="title6">Logo Website</TITLE></DIV></A></DIV><DIV class="main-submenu"><DIV class="alt-main-submenu-item" id="alt-nav-item1" href="#project"><SPAN>Project</SPAN><UL class="alt-submenu"><LI><A href="https://2018.igem.org/Team:ETH_Zurich/Description">Description</A></LI><LI><A href="https://2018.igem.org/Team:ETH_Zurich/Applied_Design">Applied Design</A></LI><LI><A href="https://2018.igem.org/Team:ETH_Zurich/Team">Team Members</A></LI><LI><A href="https://2018.igem.org/Team:ETH_Zurich/Attributions">Attributions</A></LI><LI><A href="https://2018.igem.org/Team:ETH_Zurich/Sponsoring">Sponsoring</A></LI><LI><A href="https://2018.igem.org/Team:ETH_Zurich/Notebook">Notebook</A></LI><LI><A href="https://2018.igem.org/Team:ETH_Zurich/Safety">Safety</A></LI><LI><A href="https://2018.igem.org/Team:ETH_Zurich/Download">Downloads</A></LI></UL></DIV><DIV class="alt-main-submenu-item" id="alt-nav-item2" href="#parts"><SPAN>Wet Lab</SPAN><UL class="alt-submenu"><LI><A href="https://2018.igem.org/Team:ETH_Zurich/Tar">Tar Receptor Evolution</A></LI><LI><A href="https://2018.igem.org/Team:ETH_Zurich/ApproachA">Approach A</A></LI><LI><A href="https://2018.igem.org/Team:ETH_Zurich/ApproachB">Approach B</A></LI><LI><A href="https://2018.igem.org/Team:ETH_Zurich/Experiments">Experiments</A></LI><LI><A href="https://2018.igem.org/Team:ETH_Zurich/InterLab">InterLab</A></LI></UL></DIV><DIV class="alt-main-submenu-item" id="alt-nav-item3" href="#modelling"><SPAN>Hardware</SPAN><UL class="alt-submenu"><LI><A href="https://2018.igem.org/Team:ETH_Zurich/Hardware">Hardware Overview</A></LI><LI><A href="https://2018.igem.org/Team:ETH_Zurich/Microscope">Microscope</A></LI><LI><A href="https://2018.igem.org/Team:ETH_Zurich/Mechanics">Mechanics</A></LI><LI><A href="https://2018.igem.org/Team:ETH_Zurich/SyringePump">Syringe Pump</A></LI><LI><A href="https://2018.igem.org/Team:ETH_Zurich/MicrofluidicChip">Microfluidic Chip</A></LI><LI><A href="https://2018.igem.org/Team:ETH_Zurich/bubbling">Bubbling</A></LI></UL></DIV><DIV class="alt-main-submenu-item" id="alt-nav-item4"><SPAN>Software</SPAN><UL class="alt-submenu"><LI><A href="https://2018.igem.org/Team:ETH_Zurich/Software">Software Overview</A></LI><LI><A href="https://2018.igem.org/Team:ETH_Zurich/Motility_Readout">Motility Readout</A></LI><LI><A href="https://2018.igem.org/Team:ETH_Zurich/ImageAnalysis">Luminescence Algorithm</A></LI><LI><A href="https://2018.igem.org/Team:ETH_Zurich/RobotControls">Robot Controls</A></LI><LI><A href="https://2018.igem.org/Team:ETH_Zurich/Path_Planning">Path Planning</A></LI></UL></DIV><DIV class="alt-main-submenu-item" id="alt-nav-item5"><SPAN>Model</SPAN><UL class="alt-submenu"><LI><A href="https://2018.igem.org/Team:ETH_Zurich/Model">Model Overview</A></LI><LI><A href="https://2018.igem.org/Team:ETH_Zurich/integratedModel">Integrated Models</A></LI><LI><A href="https://2018.igem.org/Team:ETH_Zurich/motilityModel">Motility Model</A></LI><LI><A href="https://2018.igem.org/Team:ETH_Zurich/splitModel">Split Luciferase Model</A></LI><LI><A href="https://2018.igem.org/Team:ETH_Zurich/diffusionModel">Diffusion Model</A></LI><LI><A href="https://2018.igem.org/Team:ETH_Zurich/bubbelingModel">Bubbling Model</A></LI><LI><A href="https://2018.igem.org/Team:ETH_Zurich/Screening">Screening Assay Model</A></LI><LI><A href="https://2018.igem.org/Team:ETH_Zurich/HolographicModel">Holographic Imaging Model</A></LI></UL></DIV><DIV class="alt-main-submenu-item" id="alt-nav-item6"><SPAN>Human Practices</SPAN><UL class="alt-submenu"><LI><A href="https://2018.igem.org/Team:ETH_Zurich/Human_Practices">Human Practices</A></LI><LI><A href="https://2018.igem.org/Team:ETH_Zurich/Public_Engagement">Education &amp; Engagement</A></LI></UL></DIV><DIV class="alt-main-submenu-item" id="alt-nav-item7"><SPAN>Achievements</SPAN><UL class="alt-submenu"><LI><A href="https://2018.igem.org/Team:ETH_Zurich/Results">Results</A></LI><LI><A href="https://2018.igem.org/Team:ETH_Zurich/Parts">Parts Overview</A></LI><LI><A href="https://2018.igem.org/Team:ETH_Zurich/Demonstrate">Demonstrate</A></LI><LI><A href="https://2018.igem.org/Team:ETH_Zurich/Collaborations">Collaborations</A></LI></UL></DIV></DIV><DIV class="mobileMenuBg" style="visibility: hidden"><DIV class="container-titel"><DIV class="titel">
        Split Luciferase
        Model<SPAN>.</SPAN></DIV><DIV class="scroll">Scroll</DIV></DIV><DIV class="sectionh1 contentTable contentDocking" contenttable="In silico Protein-DNA Docking" contentref="contentDocking">
      In silico Protein-DNA Docking
    </DIV><DIV class="sectionh2">Aim</DIV><DIV class="text">
      For our approach B, DNA is used as a scaffold to bring two fusion-proteins together. Both proteins carry one half of the luciferase protein (either N-luc or C-luc), and upon DNA binding of the fusion-proteins the two parts of the luciferase
      are able to interact, reconstituting the complete and functional luciferase. To investigate how the fusion of N-luc and C-luc to OmpR and TetR, respectively, affect their structure and DNA binding, which distances between the two binding sites
      results in a close binding of the two proteins and which type of linkers facilitate the luciferase complementation, we simulated DNA-protein docking in silico.

      The graphs shown here were obtained with a distance of 10 bp between the two binding sites and a linker containing 3 repeats of flexible linker element.

    </DIV><DIV class="sectionh2">Protein structure prediction of C-luc-FFF-TetR and OmpR-FFF-N-luc
    </DIV><DIV class="text">
      To obtain a 3D structure of our fusion proteins, we used the protein structure prediction server I-TASSER [1,2,3]. This program uses a combination of both threading, in order to find template proteins of similar folds from the Protein Data
      Bank (PDB), and Monte Carlo-based simulations, which is needed for assembling the final protein structure. The program requires the amino acid of the fusion proteins as an input and allows specifying a template PDB file. The predicted proteins
      are displayed below. Comparing C-luc-TetR with PDB structures of the <A href="https://www.rcsb.org/structure/1lci">luciferase</A> and <A href="https://www.rcsb.org/structure/1qpi">TetR</A> only, the parts of the fusion protein resembles the structure of the separate
      proteins. The same applies to the <A href="http://www.rcsb.org/structure/1ODD">OmpR-NLuc</A> fusion protein.
    </DIV><DIV class="container_molecule"><DIV class="molecules_container"><DIV class="molecule"><DIV class="submolecule_caption">Cluc-FFF-TetR</DIV></DIV><DIV class="molecule"><DIV class="submolecule_caption">OmpR-FFF-Nluc</DIV></DIV></DIV><DIV class="mol_viewer_caption">
        3D structure of the fusion proteins obtained by the program I-tasser.
      </DIV></DIV><DIV class="sectionh2">Evaluation and further improvement of C-luc-FFF-TetR and OmpR-FFF-N-luc 3D protein structures
    </DIV><DIV class="text">
      We then analyzed both structures with the programmes ProSA [4, 5] and Procheck [6] . ProSA gives protein structures a Z-score, which is an overall quality score in the context of known protein structures. A plot is shown with the ranges of
      Z-scores for proteins of a similar size. A Z-score obtained outside of this range points towards an erroneous structure, while a z-score within this range indicates that the proteins were folded correctly. The results are displayed in figure
      1 , both showing a z-score that lays within the ideal range.
    </DIV><DIV class="container_figure"><DIV class="figures_container"><DIV class="figure"><DIV class="subfigure_caption caption_center">Cluc-FFF-TetR
          </DIV></DIV><DIV class="figure"><DIV class="subfigure_caption caption_center">OmpR-FFF-NLuc
          </DIV></DIV></DIV><DIV class="figure_caption caption_center">Z-scores obtained for the fusion protein </DIV></DIV><DIV class="text">
      As a next step protein structures were analysed with Procheck which gives us a Ramachandran plot of the fusion proteins. This plot visualizes energetically allowed regions for the backbone dihedral angles (psi) against amino acid residues
      (phi). As our proteins contained multiple residues in ‘disallowed’ regions, we decided to use the program ModRefiner [7] to perform energy minimization in order to increase the quality of our models. The Ramachandran plots of the improved
      structures are displayed below.
    </DIV><DIV class="container_figure"><DIV class="figures_container"><DIV class="figure"><DIV class="subfigure_caption caption_center">Cluc-FFF-TetR
          </DIV></DIV><DIV class="figure"><DIV class="subfigure_caption caption_center">OmpR-FFF-NLuc
          </DIV></DIV></DIV><DIV class="figure_caption caption_left">Ramachandran plots visualize energetically allowed regions for the backbone dihedral angles (psi) against amino acid residues (phi). Most of our residues can be found in the energetically favored
        regions.
      </DIV></DIV><DIV class="text">
      The plots show a percentage of 88 and 82 % for residues in most favourable regions. Visualizing the structures using Pymol did not show drastic changes in 3D structure. Moreover, our scores closely resemble the 90% score that is needed to be
      called a good quality model as determined by the program. We can therefore continue with other fusion proteins to perform protein-DNA docking.
    </DIV><DIV class="sectionh2">Protein – DNA docking
    </DIV><DIV class="text">
      With the improved structures we separately performed Protein-DNA docking. A DNA strand containing the binding sites was created using C-DART, after which it served as an input, together with the refined structures, for the docking program
      HADDOCK [8] . HADDOCK exploits a data-driven approach, where information from NMR experiments, mutagenesis experiments as well as bioinformatic predictions are combined to find the optimal docking structures. The top 10 structures were checked
      and the most promising structures were fused to obtain an image of both structures docked to the same DNA strand. The best two options are shown below:
    </DIV><DIV class="container_video" id="vid1"><DIV class="videos_container"><DIV class="video"><DIV class="subvideo_caption">Final result 1</DIV></DIV><DIV class="video"><DIV class="subvideo_caption">Final result 2</DIV></DIV></DIV><DIV class="video_caption caption_center">Docking of Cluc-FFF-TetR and OmpR-FFF-Nluc to the constructed DNA scaffolds</DIV></DIV><DIV class="text">
      The first result shows a reconstitution of luciferase. However, when looking at previous pdb files of TetR bound to DNA ((https://www.rcsb.org/structure/1qpi) and PhoB (https://www.rcsb.org/structure/1gxp ) (an OmpR homologue, as no file of
      OmpR exists) it can be questioned if OmpR is likely to dock in this way. The second result has a docking profile that resembles both the TetR and PhoB docking, and is more likely to be correct. This means that we need to include linkers in our
      screening process that are longer than the one taking into account here, to increase the change of luciferase reconstitution.
    </DIV><DIV class="sectionh2 contentTable contentConclusion1" contentref=".contentConclusion1" contenttable="Conclusion In silico Protein-DNA Docking"> Conclusion/Recommendations</DIV><DIV class="text">
      Altogether, the in silico docking of our fusion proteins to the DNA showed us that reconstitution of the split luciferase is possible within our set-up. However, it is recommended to increase the number of repeats within the linkers as this
      will facilitate luciferase reconstitution.
    </DIV><DIV class="sectionh1 contentTable contentSplitKinetics" contenttable="Split-luciferase kinetics" contentref="contentDocking">
        Kinetic model of split-luciferase
      </DIV><DIV class="sectionh2">Goals</DIV><DIV class="text">
        Besides Protein-DNA docking, we also constructed an ODE model to guide
        the experimental design of our biosensor based on luciferase
        complementation. Simulating our system for different expression levels
        of transcription factor and receptor can help us to maximize the binding
        of our transcription factor to the DNA.
      </DIV><DIV class="sectionh2">Model</DIV><DIV class="text"><DIV class="container_figure"><DIV class="figures_container"><DIV class="figure"><DIV class="subfigure_caption caption_center">Schematic
                representation of the split-luciferase kinetic model.
                </DIV></DIV></DIV></DIV>

      Tar-Envz (Taz) autophosphorylates itself upon ligand binding. This
      phosphate is quickly transferred to the transcription factor OmpR which
      facilitates its dimerization and binding to the DNA. As OmpR carries one
      half of the split luciferase protein, this part can complement the other
      half that is bound to the repressor TetR which is localized on the DNA
      next to the binding site of OmpR. Removal of the phosphate group of
      OmpR$-p$ happens by  binding to an unphosphorylated Taz receptor.
    </DIV><DIV class="sectionh2">Reactions</DIV><DIV class="text">
        The reactions below represent the split-protein complementation system.
        As our reactions are happening on a post-translational level, we can
        assume that we don't have to take any protein production and degradation
        into account.

        <DIV class="text math">
            $Taz + L \leftrightarrow L-Taz$ 
            $L-Taz   \rightarrow   L-Taz_{p}$   
            $L-Taz_{p} + OmpR  \leftrightarrow  L-Taz_{p}-OmpR$ 
            $L-Taz_{p}-OmpR    \rightarrow   L-Taz  + OmpR_{p}$ 
            $OmpR_{p}  + Taz   \leftrightarrow   Taz-OmpR_{p}$ 
            $Taz-OmpR_{p}  \rightarrow  OmpR +  Taz$ 
            $OmpR + OmpR  \leftrightarrow OmpR_{2}$ 
            $OmpR_{p} +  OmpR_{p} \leftrightarrow OmpR_{p2}$ 
            $OmpR_{p2} + PompC \leftrightarrow OmpR_{p2}*$ 

            $TetR + TetR \leftrightarrow TetR-2$ 
            $TetR-2 + PTetR \leftrightarrow TetR*$ 
            $TetR*   + OmpR_{p2}* \leftrightarrow Luminescence$ </DIV>

        From the reactions it follows that TetR repression can be tuned independently
        from the Envz/OmpR pathway: It is important to obtain a high concentration of
        TetR bound to the DNA in order to increase the luminsecent signal. Therefore
        our focus point for this ODE model will be the maximization of OmpR$_{p2}$
        binding to the DNA.
 
    </DIV><DIV class="sectionh2">Equations</DIV><DIV class="text math">
        \begin{equation}
        \frac{dL}{dt} = k_{-1} \cdot [L-Taz] - k_1 \cdot [L] \cdot [Taz]
        \end{equation} 
    </DIV><DIV class="text math">
        \begin{equation}
        \frac{dTaz}{dt} = k_{-1} \cdot [L-Taz] - k_1 \cdot [L] \cdot [Taz] +  k_{-5} \cdot [Taz-OmpR_p] - k_5 \cdot [OmpR] \cdot [Taz] + k_6 \cdot [Taz-OmpR_p]
        \end{equation} 
    </DIV><DIV class="text math">
        \begin{equation}
        \frac{dL-Taz}{dt} = k_1 \cdot [L] \cdot [Taz] - k_{-1} \cdot [L-Taz] - k_2 \cdot [L-Taz] + k_4 \cdot [L-Taz_p-OmpR]
        \end{equation} 
    </DIV><DIV class="text math">
        \begin{equation}
        \frac{dL-Taz_p}{dt} = k_2 \cdot [L-Taz] - k_3 \cdot [L-Taz_p] \cdot [OmpR] + k_{-3}*[L-Taz_p-OmpR]
        \end{equation} 
    </DIV><DIV class="text math">
        \begin{equation}
        \frac{dOmpR}{dt} = - k_3 \cdot [L-Taz_p] \cdot [OmpR] + k_{-3}*[L-Taz_p-OmpR] + k_6 \cdot [Taz-OmpR_p] - 2 \cdot k_{dim1} \cdot [OmpR]^2 + 2 \cdot k_{-dim1}*[OmpR_2] 
        \end{equation} 
    </DIV><DIV class="text math">
        \begin{equation}
        \frac{dL-Taz_p-OmpR}{dt} = k_3 \cdot [L-Taz_p] \cdot [OmpR] - k_{-3}*[L-Taz_p-OmpR]
        \end{equation} 
    </DIV><DIV class="text math">
        \begin{equation}
        \frac{dOmpR_p}{dt} = k_4 \cdot [L-Taz_p-OmpR]  - 2 \cdot k_{dim2} \cdot [OmpR_p]^2 + 2 \cdot k_{-dim2}*[OmpR_{p2}] + k_{-5} \cdot [Taz-OmpR_p] - k_5 \cdot [OmpR] \cdot [Taz]
        \end{equation} 
    </DIV><DIV class="text math">
        \begin{equation}
        \frac{dTaz-OmpR_p}{dt} = - k_{-5} \cdot [Taz-OmpR_p] + k_5 \cdot [OmpR] \cdot [Taz] - k_6 \cdot [Taz-OmpR_p]
        \end{equation} 
    </DIV><DIV class="text math">
        \begin{equation}
        \frac{dPompC}{dt} = -k_8 \cdot [OmpR_{p2}] \cdot [PompC] + k_{-8} \cdot [OmpR_{p2}*] 
        \end{equation} 
    </DIV><DIV class="text math">
        \begin{equation}
        \frac{dOmpR_2}{dt} = k_{dim1} \cdot [OmpR]^2 - k_{-dim1}*[OmpR_2] 
        \end{equation} 
    </DIV><DIV class="text math">
        \begin{equation}
        \frac{dOmpR_{p2}*}{dt} = k_8 \cdot [OmpR_{p2}] \cdot [PompC] -k_{-8} \cdot [OmpR_{p2}*]
        \end{equation} 
    </DIV><DIV class="text math">
        \begin{equation}
        \frac{dOmpR_{p2}}{dt} =  k_{dim2} \cdot [OmpR_p]^2 - k_{-dim2}*[OmpR_{p2}] - k_8 \cdot [OmpR_{p2}] \cdot [PompC] + k_{-8} \cdot [OmpR_{p2}*]
        \end{equation} 
     </DIV><DIV class="sectionh2">Parameters</DIV><DIV class="text">
        The parameters used within this model can be found here (<A href="https://2018.igem.org/Team:ETH_Zurich/Parameter">parameter
        page</A>). As this ODE model only provided qualitative insights
        for guiding the experimental set-up, no experimental data was
        incorporated.
    </DIV><DIV class="sectionh2">Results</DIV><DIV class="text">
        With the equations presented above, the influence of different Taz and
        OmpR protein levels on the binding of OmpR_p to the DNA was examined.
        The figure below shows the result for the induction of the pathway
        using the aspartate concentrations 10uM, 0.1mM an 1mM respectively. As Taz is
        a membrane receptor that is expressed in low amounts in the cell, we
        took a concentration range from 0 to 0.33 uM into account during the
        simulation. This responds to a maximum of 200 moleculs of Taz. For OmpR,
        we took a range from 0 to 4000 molecules, as the normal expression level
        of OmpR lays around 3000 molecules. 
        
        <DIV class="container_figure"><DIV class="figure_caption caption_left">Concentration of
            phosphorilated OmpR bound to DNA as a function of the Taz and OmpR
            concentrations for different levels of ligand concentration.
            </DIV></DIV>

        All three figures show that level of Taz is critical for the amount of
        OmpR_p2 bound to the DNA. The higher the expression level of Taz,
        the more OmpR_p2 locates to the DNA. However, as overexpression of
        Taz leads to cell death, its best to put Taz under an inducible
        promotor. This will allow to tune the expression precisely.
        
            
        For OmpR a different situation applies. Already for a relative low
        expression of OmpR the maximum amount of OmpR_p2 bound to the DNA
        for a specific ligand concentration is achieved.
        
            
        Last, we see that the amount of OmpR$_p2 bound to the DNA
        increases with an increasing ligand concentration, confirming the
        possiblity of turning this pathway into a biosensor.
            
    </DIV><DIV class="sectionh2">Conclusions</DIV><DIV class="text">
        Our split-protein ODE model suggest to put the Taz receptor under an
        inducible promoter. This way the expression level can be tuned,
        obtaining a balance between a high protein expression and a cell that
        does keeps proliferating.  
    </DIV><DIV class="sectionh2">References</DIV><DIV class="references"><OL><LI>
          Y Zhang. I-TASSER server for protein 3D structure prediction. BMC Bioinformatics, 9: 40 (2008). doi: 10.1186/1471-2105-9-40.

        </LI><LI>
          J Yang, R Yan, A Roy, D Xu, J Poisson, Y Zhang. The I-TASSER Suite: Protein structure and function prediction. Nature Methods, 12: 7-8 (2015). doi:10.1038/nmeth.3213

        </LI><LI>
          Yang J, Yan R, Roy A, Xu D, Poisson J, Zhang Y. 2015. The I-TASSER Suite: protein structure and function prediction. Nature Methods. 12(1): 7-8. doi.org/10.1038/nmeth.3213

        </LI><LI>
          Wiederstein &amp; Sippl (2007) ProSA-web: interactive web service for the recognition of errors in three-dimensional structures of proteins. Nucleic Acids Research 35, W407-W410.
        </LI><LI>
          Sippl, M.J. (1993) Recognition of Errors in Three-Dimensional Structures of Proteins. Proteins 17, 355-362
        </LI><LI>
          Laskowski R A, MacArthur M W, Moss D S, Thornton J M (1993). PROCHECK - a program to check the stereochemical quality of protein structures. J. App. Cryst., 26, 283-291

        </LI><LI>
          Dong Xu and Yang Zhang. Improving the Physical Realism and Structural Accuracy of Protein Models by a Two-step Atomic-level Energy Minimization. Biophysical Journal, vol 101, 2525-2534 (2011)

        </LI><LI>
          Van Zundert, G. C. P., et al. &quot;The HADDOCK2. 2 web server: user-friendly integrative modeling of biomolecular complexes.&quot; Journal of molecular biology 428.4 (2016): 720-725.

        </LI></OL></DIV></DIV><DIV class="mobileMenuBg"><FOOTER><DIV class="sponsor-title">Sponsors</DIV><DIV class="multimedia-title">Contact Us</DIV></FOOTER></DIV></DIV></DIV></DIV></DIV></DIV></BODY></HTML>