<HTML xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en" dir="ltr">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki  ltr ns-0 ns-subject page-Team_Concordia_Notebook"><DIV id="globalWrapper"><DIV id="top-section"><DIV id="p-logo"><A href="/Main_Page" title="Main Page">&quot;
	    </A></DIV><DIV id="menubar" class="left-menu noprint"><UL><LI class="selected"><A href="/Team:Concordia/Notebook">Page               </A></LI><LI class="new"><A href="/wiki/index.php?title=Talk:Team:Concordia/Notebook&amp;action=edit&amp;redlink=1">Discussion               </A></LI><LI><A href="/wiki/index.php?title=Team:Concordia/Notebook&amp;action=edit">View source               </A></LI><LI><A href="/wiki/index.php?title=Team:Concordia/Notebook&amp;action=history">History               </A></LI><LI style="color:white;cursor:default">teams</LI></UL></DIV><DIV class="right-menu noprint" id="menubar"><UL><LI id="pt-login"><A href="/wiki/index.php?title=Special:UserLogin&amp;returnto=Team:Concordia/Notebook" title="You are encouraged to log in; however, it is not mandatory [o]" accesskey="o">Log in</A></LI></UL></DIV><DIV id="search-controls" class="noprint"><FORM action="/Special:Search" id="searchform"> </FORM></DIV></DIV><DIV id="content"><H1 class="firstHeading">Team:Concordia/Notebook</H1><DIV id="bodyContent"><H3 id="siteSub" class="noprint">From 2013.igem.org</H3><DIV id="myContent"><DIV class="wrapper"><DIV class="site-menu col-md-2"><DIV class="site-menu-content"><H2>Sitemap</H2><UL><LI><A href="https://2013.igem.org/Team:Concordia">Home</A></LI><LI><A href="https://2013.igem.org/Team:Concordia/Project">Project</A></LI><UL class="project-list"><LI><A href="https://2013.igem.org/Team:Concordia/GasClock">Gas Clock</A></LI><LI><A href="https://2013.igem.org/Team:Concordia/Logic">Logic</A></LI><LI><A href="https://2013.igem.org/Team:Concordia/Interface">Interface</A></LI><LI><A href="https://2013.igem.org/Team:Concordia/Memory">Memory</A></LI></UL><LI><A href="https://2013.igem.org/Team:Concordia/Team">Team</A></LI><LI><A href="https://igem.org/Team.cgi?year=2013&amp;team_name=Concordia">Official Profile</A></LI><LI><A href="https://2013.igem.org/Team:Concordia/Parts">Registry</A></LI><LI><A href="https://2013.igem.org/Team:Concordia/HumanPractices">Human Practices</A></LI><LI><A href="https://2013.igem.org/Team:Concordia/Modeling">Modeling</A></LI><LI><A href="https://2013.igem.org/Team:Concordia/Notebook">Notebook</A></LI><LI><A href="https://2013.igem.org/Team:Concordia/Safety">Safety</A></LI><LI><A href="https://2013.igem.org/Team:Concordia/Attributions">Attributions</A></LI></UL></DIV></DIV><DIV class="content col-md-10"><DIV id="top-section"><DIV class="left-menu noprint"><UL><LI class="selected"><A href="/Team:Concordia">Page               </A></LI><LI class="new"><A href="/wiki/index.php?title=Talk:Team:Concordia&amp;action=edit&amp;redlink=1">Discussion               </A></LI><LI><A href="/wiki/index.php?title=Team:Concordia&amp;action=edit">Edit               </A></LI><LI><A href="/wiki/index.php?title=Team:Concordia&amp;action=history">History               </A></LI><LI><A href="/Special:MovePage/Team:Concordia">Move               </A></LI><LI><A href="/wiki/index.php?title=Team:Concordia&amp;action=watch">Watch               </A></LI><LI style="color:white;cursor:default">teams</LI></UL></DIV><DIV class="right-menu noprint"><UL><LI id="pt-preferences"><A title="My account" href="https://igem.org/User_Information">My account</A></LI><LI id="pt-logout"><A title="Log out" href="/wiki/index.php?title=Special:UserLogout&amp;returnto=Team:Concordia">Log out</A></LI></UL><DIV class="noprint" id="search-controls"><FORM id="searchform" action="/Special:Search"> </FORM></DIV></DIV></DIV><DIV class="container"><H1>Notebook</H1><DIV class="log col-md-9 log-right"><DIV class="log-title-wrap"><H2 class="log-title">Gene Regulatory Network Completed</H2><H4 class="log-time">June, 27th</H4></DIV><DIV class="log-desc"><P class="log-long-desc">After 2 months of meetings, discussion and research we have finalized our genetic regulatory network. Next stop… wet lab!</P></DIV></DIV><DIV class="log col-md-9 log-left"><DIV class="log-title-wrap"><H2 class="log-title">Amplification of ETR1 and EnvZ Fragments Successful</H2><H4 class="log-time">July, 26th</H4></DIV><DIV class="log-desc"><P class="log-long-desc">We successfully amplified truncated fragments of ETR1 and EnvZ from A. thaliana cDNA (provided by Patrick Gulick, PhD) and E. coli genomic DNA (provided by Michelle Harvey, Tech). These truncated fragments will be used to generate site specific fusion proteins.</P></DIV></DIV><DIV class="log col-md-9 log-right"><DIV class="log-title-wrap"><H2 class="log-title">Generated Fusions</H2><H4 class="log-time">July, 29th</H4></DIV><DIV class="log-desc"><P class="log-long-desc">We successfully extracted the amplicons (from 07-26) from 1% Agarose gel to be used in subsequent overlapping PCR experiments. This was a fundamental step as any residual primers within the sample will make it difficult to perform the overlapping PCR.</P></DIV></DIV><DIV class="log col-md-9 log-left"><DIV class="log-title-wrap"><H2 class="log-title">Generated Fusions</H2><H4 class="log-time">Aug, 1st</H4></DIV><DIV class="log-desc"><P class="log-long-desc">Overlapping PCR was attempted using a troubleshooting technique. Fragments used came from E.coli (EnvZ) and A. thaliana (ETR1). One nanogram of ETR1 and EnvZ fragments, corresponding to their proper fusion domains (DOM – domain, CON- conserved, and LIT-literature), were loaded using the standard contents necessary for PCR minus the primers. Four differing DNA polymerases were tested; PhireHot Start Polymerase, Phusion polymerase, Crimson Taq polymerase, and Q5 polymerase.  Primers were added after 10 cycles out of a total of 30 cycles. PCR samples were then verified using a 1% agarose gel containing EtBr, and tested against ETR1 and EnvZ fragments (positive controls). Electrophoresis was conducted for 50 minutes at 100volts. Visualization revealed that out of the four polymerases used, only one was able to generated the correctly sized fusion proteins; Phire Hot Start DNA polymerase.</P></DIV></DIV><DIV class="log col-md-9 log-right"><DIV class="log-title-wrap"><H2 class="log-title">Amplified EnvZ, but No LIT fusion</H2><H4 class="log-time">Aug, 6th</H4></DIV><DIV class="log-desc"><P class="log-long-desc">Full length ETR1 and EnvZ fragments were needed for Nested Deletion protocol, also the LIT fusion protein required increased yield. Amplification PCR of all of these samples was conducted. Standard PCR components were loaded, including correct primers, with 1 ng of genomic DNA (for EnvZ), 3ng of A. thaliana cDNA (for ETR1), and 1 ng of previously generated LIT fusion. PCR samples were run for 30 cycles, and then tested via Gel Electrophoresis, using a 1% agarose gel containing EtBr, at 100 volts for 50 minutes.   For out Nested Deletions samples; ENVz was successfully Amplified from E.coli genomic DNA, but ETR1 was not amplified from A. thaliana cDNA. Also, our LIT fusion amplification, using DNA template from the previously successful overlapping PCR, was not achieved.</P></DIV></DIV><DIV class="log col-md-9 log-left"><DIV class="log-title-wrap"><H2 class="log-title">Ethanol purification of EnvZ, and Fusions DOM/CON</H2><H4 class="log-time">Aug, 7th</H4></DIV><DIV class="log-desc"><P class="log-long-desc">Any further purification of samples was suggested to be done using an ethanol purification protocol. Previously amplified Nested Deletion EnvZ, and Fusions proteins DOM and CON were Ethanol purified. Ethanol purification was conducted, using all of the remaining sample, which was precipitated using sodium acetate and ice cold 99% ethanol, followed by 70% ethanol and re-suspension in Ultra Pure Distilled water.  Ethanol Purification products were then verified and quantified using a 1% agarose gel containing EtBr run at 100 volts for 50 min. Concentrations, determined through gel quantification, are estimated as; 630 ng/ul for EnvZ, 450ng/ul for DOM fusion, and 105 ng/ul for CON fusion. Additionally, amplification PCR for ETR1 and LIT Fusion were attempted again but were unsuccessful.</P></DIV></DIV><DIV class="log col-md-9 log-right"><DIV class="log-title-wrap"><H2 class="log-title">Amplification/Ethanol Purification of TetR and RhlR. Ethanol Purification of PompC-GFP</H2><H4 class="log-time">Aug, 13th</H4></DIV><DIV class="log-desc"><P class="log-long-desc">PCR amplification of; interface gene RhlR, and repressor protein TetR was conducted. Standard PCR components, including specific and correct primers, were added to 200-600 pg of target DNA. Ethanol Purification of RhlR, TetR, and PompC-GFP was conducted, using all of the remaining sample, which was precipitated using sodium acetate and ice cold 99% ethanol, followed by 70% ethanol and re-suspension in Ultra Pure Distilled water. Products were then verified and quantified using a 1% agarose gel containing EtBr run at 100 volts for 50 min. Concentrations, determined through gel quantification, are estimated as; 360 ng/ul for RhlR, 216 ng/ul for TetR, and 270 ng/ul for PompC-GFP.</P></DIV></DIV><DIV class="log col-md-9 log-left"><DIV class="log-title-wrap"><H2 class="log-title">Amplification of ETR1, LIT fusion, PLac, and Pomc-GFP. </H2><H4 class="log-time">Aug, 21st</H4></DIV><DIV class="log-desc"><P class="log-long-desc">Temperature Gradient Amplification PCR was attempted for the ETR1 and Literature Fusion, since they were proving difficult to amplify. PCR was conduct using standard components, including specific primers, and 1 ng of Lit Fusion DNA, and 3ng of ETR1 cDNA. A positive control of EnvZ was also run in order to rule out potential discrepancies. The temperature gradient was set from 58 degrees Celsius to 62 degrees Celsius, and number of cycles was 35. Additionally, normal amplification PCR was conducted for 200-600 pg of Plac, and 1 ng of PompC-GFP DNA using standard protocol. Products were then tested using a 1% agarose gel containing EtBr run at 100 volts for 50 min. Visualization of the temperature gradient revealed that ETR1 products were amplified at the correct size for temperatures 60 to 63, and Lit fusion products were amplified correctly at 62 and 64. Also, Plac and PompC-GFP were also amplified properly.</P></DIV></DIV><DIV class="log col-md-9 log-right"><DIV class="log-title-wrap"><H2 class="log-title">Ethanol Purification of ETR1, PLac, and PompC-GFP</H2><H4 class="log-time">Aug, 22nd</H4></DIV><DIV class="log-desc"><P class="log-long-desc">Previously amplified ETR1, PLac and PompC-GFP were Ethanol purified. Ethanol purification was conducted, using all of the remaining sample, which was precipitated using sodium acetate and ice cold 99% ethanol, followed by 70% ethanol and re-suspension in Ultra Pure Distilled water.  Ethanol Purification products were then verified using a 1% agarose gel containing EtBr run at 100 volts for 50 min. Visualization shows that ETR1 was purified at various concentrations depending on which sample was used (where 61 degrees was the best), PLac and PompC-GFP were also purified accurately.</P></DIV></DIV><DIV class="log col-md-9 log-left"><DIV class="log-title-wrap"><H2 class="log-title">Amplification of RhlI, LuxR, and LuxI. Purification of all Fusion proteins and Plac.</H2><H4 class="log-time">Sept, 3rd</H4></DIV><DIV class="log-desc"><P class="log-long-desc">Troubleshooting PCR amplification of; interface genes RhlR, LuxR, and LuxI was conducted. RhlI was used as a positive control since its amplification had been successful in numerous other PCR reactions. Standard PCR components, including specific and correct primers, were added to 200-600 pg of target DNA. PCR reactions tested common troubleshooting techniques such as; reduced primers, DMSO, added Magnesium content, and Taq polymerase. Ethanol Purification of DOM, CON and LIT fusions as well as Plac was conducted, using all of the remaining sample, which was precipitated using sodium acetate and ice cold 99% ethanol, followed by 70% ethanol and re-suspension in Ultra Pure Distilled water. PCR products were then verified using a 1% agarose gel containing EtBr run at 100 volts for 50 min. Out of the four parameters tested (plus one normal reaction); only added magnesium gave a visible result for LuxI and Lux R. However, the bands obtained were weak, therefore this reaction must be attempted again.</P></DIV></DIV><DIV class="log col-md-9 log-right"><DIV class="log-title-wrap"><H2 class="log-title">Mini-Prep of PompC transformed E.coli cells. </H2><H4 class="log-time">Sept, 10th</H4></DIV><DIV class="log-desc"><P class="log-long-desc">All cells appeared red in color after transformation, but mini prep was still attempted. All samples (10 in total) were mini-preped according to kit procedures.</P></DIV></DIV><DIV class="log col-md-9 log-left"><DIV class="log-title-wrap"><H2 class="log-title">GFP amplified in mini-prep samples</H2><H4 class="log-time">Sept, 12th</H4></DIV><DIV class="log-desc"><P class="log-long-desc">Since samples were red, and not white, presence of PompC-GFP (proof of a successful ligation) was questionable. All samples were tested for PompC using restriction digest and PCR amplification. Standard PCR components, including specific and correct primers, were added to ~1ng of target DNA. Restriction Digest was conducted using standard Biobricks cut sites. PCR products and Restriction digest products, were visualized using a 1% agarose gel containing EtBr run at 100 volts for 50 min. Visualization revealed that PCR samples yielded a positive result for PompC-GFP in mini-preped samples, while restriction digest products also cut out a fragment the same size as PompC-GFP.</P></DIV></DIV></DIV></DIV></DIV></DIV><DIV class="printfooter">
Retrieved from &quot;<A href="http://2013.igem.org/Team:Concordia/Notebook">http://2013.igem.org/Team:Concordia/Notebook</A>&quot;</DIV></DIV></DIV><DIV id="footer-box" class="noprint"><DIV id="footer"><UL id="f-list"><LI id="t-recentchanges"><A href="/Special:RecentChanges" title="Recent changes">Recent changes</A></LI><LI id="t-whatlinkshere"><A href="/Special:WhatLinksHere/Team:Concordia/Notebook" title="List of all wiki pages that link here [j]" accesskey="j">What links here</A></LI><LI id="t-recentchangeslinked"><A href="/Special:RecentChangesLinked/Team:Concordia/Notebook" title="Recent changes in pages linked from this page [k]" accesskey="k">Related changes</A></LI><LI id="t-specialpages"><A href="/Special:SpecialPages" title="List of all special pages [q]" accesskey="q">Special pages</A></LI><LI><A href="/Special:Preferences">My preferences</A></LI></UL></DIV><DIV id="footer"><UL id="f-list"><LI id="t-print"><A href="/wiki/index.php?title=Team:Concordia/Notebook&amp;printable=yes" title="Printable version of this page [p]" accesskey="p">Printable version</A></LI><LI id="t-permalink"><A href="/wiki/index.php?title=Team:Concordia/Notebook&amp;oldid=237389" title="Permanent link to this revision of the page">Permanent link</A></LI><LI id="privacy"><A href="/2013.igem.org:Privacy_policy" title="2013.igem.org:Privacy policy">Privacy policy</A></LI><LI id="disclaimer"><A href="/2013.igem.org:General_disclaimer" title="2013.igem.org:General disclaimer">Disclaimers</A></LI></UL></DIV></DIV></DIV></BODY></HTML>