"
Page
Discussion
View source
History
teams
Log in
 
Team:Concordia/Logic
From 2013.igem.org
Sitemap
Home
Project
Gas Clock
Logic
Interface
Memory
Team
Official Profile
Registry
Human Practises
Modeling
Notebook
Safety
Attributions
Page
Discussion
Edit
History
Move
Watch
teams
My account
Log out
 
Logic
Just like in any cellular automaton, a set of rules must be established for each cell to specifically recognize their inputs and subsequently produce a desired output in a consistent fashion. We chose a particular set of rules, described in the truth table below, where QL is the present state of the left neighbor cell, QT is the present state of the cell being investigated, QR is the present state of the right neighbor cell, and QT+1 is the next state of the investigated cell, determined by the pace of the gas clock.
Ql
Qt
Qr
Qt+1
0
0
0
0
0
0
1
0
0
1
0
0
0
1
1
1
1
0
0
1
1
0
1
1
1
1
0
1
1
1
1
0
We found that these rules were simple enough to be implemented into a biological system and were computationally interesting. As can be seen in the simulation below, when a single cell is stimulated and changed into the “ON” state, this change in state will propagate itself around the circular cellular automaton one cell at a time in a clockwise orientation. When two neighbor cells are initially stimulated, an ever-growing and chaotic pattern arises that engulfs the entire cellular automaton, demonstrating how each cell within the system, made aware of its surroundings, can react in a predictable and contrasting way.
There are many functions that could have been used to execute these truth table rules.  We devised a very simple function that allowed for a significant decrease in the number of logic gates, as seen in the diagram below.
QT+1= NOT[(QT AND QR) XOR QL ]
In our biological cellular automata, this function is encoded into the genome. Our logic circuit involves RNA-based mechanisms like ribozyme binding, cleavage and inactivation with small RNAs. RNA-based logic was chosen over transcriptional regulation with proteins due its general speed and robustness, its simple in silico design, and its flexibility in creating novel logic gates such as an XOR gate.
A simple layout of the essential components of the logic processing can be found below. It can be seen that the promoters for the 3 AHL systems we are employing are acting as inputs directly into the logic function. To establish perceived directionality from one cell to another, the three input AHL promoters will all shift positions by one in each cell type of the repeating-triplet circular array of strains that composes our biological cellular automaton.
Two Y-type hammerhead ribozymes are involved in the cleavage of one target mRNA. Due to their particular structure as discussed later on, when only one of these ribozymes is present on its own it is able to cleave specific sites within the coding region of the target mRNA.
An RNA-based AND gate was implemented within the network in order to comply with the logic function. This particular logic gate was accomplished by introducing small RNA sequences that are complementary along a significant portion of a targeted ribozyme’s flanking and extended arms, but does not contain a cut site, deemed a silencer. This allows for the effective inactivation of the ribozyme by preventing it from binding to the intended target mRNA. Anti-silencer sequences were also introduced in order to complete the functional AND gate by being expressed in excess and being perfectly complementary to the silencer sequences, allowing for the sequestering of the silencer sequences to anti-silencer sequences and thus relieving the repression of the ribozyme, as can be seen in the above diagram.
A ribozyme-based XOR gate was also necessary to be designed in order to properly implement our desired logic function. As stated earlier, each individual ribozyme can cleave their intended target only when they are not present in conjunction with the other. This mechanism was accomplished by careful in silico design.
These Y-type hammerhead ribozymes have target-binding flanking arms adjacent to the catalytic core and synthetic extended arms located towards the 5' and 3' ends. The catalytic core allows for the sequence-specific cleavage of sites within the coding sequence of the target RNA. Hammerhead ribozymes are able to cleave at any NUH within an RNA sequence, where H represents any nucleotide except guanine (Mir et al, 2001), although with GUC as the hammerhead consensus cleaving site (Lieber et al, 1995). The binding of the flanking arms adjacent to the hammerhead catalytic core favors the recognition and cleavage of only one target site. Additional secondary structure within the flanking arms was detected to a small degree by using IDT OligoAnalyzer's Hairpin tool (Owczarzy et al, 2008) as is depicted below.
The synthetic extended arms were designed to bind to the complementary extended arms of another similar ribozyme within the same logic circuit but do not contribute to any additional secondary structure as confirmed via the Hairpin tool in IDT's OligoAnalyzer (Owczarzy et al, 2008), as both arms of a particular ribozyme only contain a purine and pyrimidine pair that do not bind to each other (in this case, A and G). The two-nucleotide extended arms are also staggered in number to avoid slippage in binding and bind to its complement pair at significantly higher melting temperatures than the adjacent flanking arms. This feature allows for the sequestering of the two ribozymes when both are present and significantly decreasing any ribozyme activity, as shown in the image below. Regions of complementarity between the two target-binding regions of both ribozymes are also present to ensure that the functional catalytic cores are not able to cleave their mRNA targets, even when bound.
The characteristics of this expressed pair of ribozymes were considered to create a functional RNA-based XOR logic gate.
The simple in silico design of this ribozyme allows for the cleavage of virtually any RNA sequence containing a cleavage site (NUH). In order to properly implement this logic function in an optimal fashion, it was conceptualized that the targets of this pair of ribozymes and thus the outputs of our logic function will be the mRNAs of the phage protein repressor CI434 and the PenI repressor from Bacillus licheniformis, which is able to consequently negatively regulate the CI repressor. These two mRNAs were chosen to be targets in order to tilt the bistable memory of each cell in either the “ON” or “OFF” state at each processing cycle depending on the presence of the various AHL inputs.
Parts
Reference
Name
BBa_K1165002
PenI-cleaving Ribozyme 1 - XOR Gate 1
BBa_K1165003
PenI-cleaving Ribozyme 2 - XOR Gate 1
BBa_K1165004
PenI-cleaving Ribozyme 1 - XOR Gate 2
BBa_K1165005
PenI-cleaving Ribozyme 2 - XOR Gate 2
References
Lieber, A. & Strauss, M. Selection of efficient cleavage sites in target RNAs by using a ribozyme expression library. Mol. Cell. Biol. 15, 540–551 (1995).
Mir, A. A., Lockett, T. J. & Hendry, P. Identifying ribozyme-accessible  sites using NUH triplet-targeting gapmers. Nucleic Acids Res 29, 1906–1914 (2001).
Owczarzy, R. et al. IDT SciTools: a suite for analysis and design of nucleic acid oligomers. Nucleic Acids Res 36, W163–W169 (2008).
Retrieved from " http://2013.igem.org/Team:Concordia/Logic "
Recent changes
What links here
Related changes
Special pages
My preferences
Printable version
Permanent link
Privacy policy
Disclaimers
