<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_Peking_Model skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><DIV id="top_title"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:Peking/Model</SPAN></H1></DIV><DIV id="HQ_page"><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><DIV class="mdl-layout mdl-js-layout mdl-layout--fixed-header"><HEADER class="mdl-layout__header" style="position:fixed; margin-top: -7px; !important;"><DIV class="mdl-layout__header-row"><A class="mdl-navigation__link" href="https://2017.igem.org/Team:Peking" style="color: #000000; font-size: x-large"><STRONG>Peking iGEM </STRONG> 2017</A><NAV class="mdl-navigation mdl-layout--large-screen-only"><A class="mdl-navigation__link" href="https://2017.igem.org/Team:Peking">Home</A><A class="mdl-navigation__link" href="https://2017.igem.org/Team:Peking/Project#Introduction">Project</A><A class="mdl-navigation__link" href="https://2017.igem.org/Team:Peking/Model#Overview" style="color: #000; font-weight: 500 ">Modelling</A><A class="mdl-navigation__link" href="https://2017.igem.org/Team:Peking/Software">Software</A><A class="mdl-navigation__link" href="https://2017.igem.org/Team:Peking/Hardware">Hardware</A><A class="mdl-navigation__link" href="https://2017.igem.org/Team:Peking/Lab">Lab</A><A class="mdl-navigation__link" href="https://2017.igem.org/Team:Peking/HP">Practices</A><A class="mdl-navigation__link" href="https://2017.igem.org/Team:Peking/Parts#Overview">Parts</A><A class="mdl-navigation__link" href="https://2017.igem.org/Team:Peking/Team">Team</A></NAV></DIV></HEADER></DIV><MAIN class="docs-layout-content mdl-layout__content mdl-color-text--grey-600" style="width: 100%; padding-bottom: 0px;"><DIV class="content mdl-grid mdl-grid--no-spacing" id="content"><DIV class="mdl-components mdl-js-components mdl-cell mdl-cell--12-col" style="padding-top: 60px; background-color: #FAFAFA"><ASIDE class="mdl-components__nav docs-text-styling mdl-shadow--4dp" style="width: 220px"><A href="https://2017.igem.org/Team:Peking/Model#Overview" class="mdl-components__link mdl-component FF"><SPAN class="mdl-components__link-text" style="font-size: medium">Overview</SPAN></A><A href="https://2017.igem.org/Team:Peking/Model#Flip-flop" class="mdl-components__link mdl-component FF"><SPAN class="mdl-components__link-text" style="font-size: medium">Flip-flop</SPAN></A><A href="https://2017.igem.org/Team:Peking/Model#Controller" class="mdl-components__link mdl-component CU"><SPAN class="mdl-components__link-text" style="font-size: medium">Controller</SPAN></A><A href="https://2017.igem.org/Team:Peking/Model#Clock" class="mdl-components__link mdl-component Clock"><SPAN class="mdl-components__link-text" style="font-size: medium">Clock</SPAN></A><A href="https://2017.igem.org/Team:Peking/Model#Supplementary" class="mdl-components__link mdl-component Prospect"><SPAN class="mdl-components__link-text" style="font-size: medium">Supplementary</SPAN></A></ASIDE><SECTION id="index-section" class="mdl-components-index mdl-components__page"><DIV class="mdl-components-index-text mdl-grid"><DIV class="mdl-components-text docs-text-styling about-panel about-panel--text mdl-cell mdl-cell--12-col"><DL class="mdl-components-dl"><DT class="mdl-components-dt">MODELING</DT><DD class="mdl_components-dd">
                                Our model is integrated and tightly connected with experiments.
                                We applied ODE-system
                                simulation to assist in recombinase characterization. The behavior of control units
                                under recombinase-mediated sequence inversion was predicted with stochastic simulation.
                                Furthermore, the generation of control unit design candidates was implemented with
                                evolutionary algorithm. Screening of candidates was done with an original set of scoring
                                rules. All these parts work closely together to realize the rational design of genetic
                                sequential logic. Also, the experimental viability of the design should be
                                confirmed. We
                                base our model on substantiated experiment observations. The sensitive parameters in ODE
                                system for recombinase characterization were estimated with experiment data. In this
                                process, the modeling and design are directed by experiments. Also, we predicted the
                                operation range thus specifying RBS strength for expression of each recombinase. In
                                turn, the modeling results aided in experiment design. Also, filtering criteria of
                                control unit candidates have roots in experimental findings.
                            </DD></DL></DIV></DIV></SECTION><SECTION id="Overview" class=" mdl-components__page mdl-grid"><DIV class="demo-card-wide mdl-card mdl-shadow--2dp"><DIV class="mdl-card__supporting-text"><H1>Overview</H1>
                        Our model is integrated and tightly connected with experiments.

                        We applied ODE-system simulation to assist in recombinase characterization. The behavior of
                        control units under recombinase-mediated sequence inversion was predicted with stochastic
                        simulation. Furthermore, the generation of control unit design candidates was implemented with
                        evolutionary algorithm. Screening of candidates was done with an original set of scoring rules.
                        All these parts work closely together to realize the rational design of genetic sequential
                        logic.

                        Also, the experimental viability of the design should be confirmed. We base our model on
                        substantiated experiment observations. The sensitive parameters in ODE system for recombinase
                        characterization were estimated with experiment data. In this process, the modeling and design
                        are directed by experiments.

                        We predicted the operation range thus specifying RBS strength for expression of each
                        recombinase. In turn, the modeling results aided in experiment design. Also, filtering criteria
                        of control unit candidates have roots in experimental findings.<A class="mdl-button mdl-js-button mdl-button--raised mdl-button--accent mdl-js-ripple-effect" href="#top" style="background-color: #2D8EE8; color: white;">
                        Back to top
                    </A></DIV></DIV></SECTION><SECTION id="Flip-flop" class="mdl-components__page mdl-grid"><DIV class="demo-card-wide mdl-card mdl-shadow--2dp"><DIV class="mdl-card__supporting-text"><H1>Flip-flop</H1><SECTION class="docs-toc docs-text-styling" style="margin-left: 0px"><NAV class="section-content" style="margin-left: 0px"><UL style="margin-left: 0px;"><LI><A href="#FF_Basis">Biological Basis</A></LI><LI><A href="#FF_Compositions">Model Composition</A></LI><LI><A href="#FF_Results">Simulation Results</A></LI><LI><A href="#FF_Reference">References</A></LI></UL></NAV></SECTION><H2>Intention of the Model</H2>
                        According to our design of biological flip-flops, a recombinase-based device must be
                        constructed, characterized and standardized. Recombinases can operate on sequences with
                        recombination sites thus altering the direction of transcription. In this section we propose a
                        model that characterize the process of gene recombination with ordinary differential equations
                        (ODEs). Each reaction in the process is described with Hill or mas action kinetics. The aims of
                        this model are listed as follows:<OL start="1"><LI>Estimate parameters in the model from experimental data and
                                fine-tune the model.
                            </LI><LI>Simulate the sequence recombination process via recombinases. Show the
                                changing of quantities of original and flipped sequences over time. Prove that the
                                device can
                                work as expected.
  </LI><LI>Analyze the effects of varying recombinase production and degradation rates
                                on the overall performance of the device.
                            </LI><LI>Assist in designing of the genetic circuit.</LI></OL></DIV></DIV><DIV id="FF_Basis" class="demo-card-wide mdl-card mdl-shadow--2dp"><DIV class="mdl-card__supporting-text"><H2>Biological Basis and Assumptions</H2>
                        The sequence-flipping device is based on the integrase family of recombinases. On this page we
                        focus only on the Bxb1 serine integrase, but the same dynamics model and analysis applies to two
                        other types TP901 and PhiC31. Mechanism of serine integrases is well recorded in
                        literature.[1]
                        <H3 id="set-process">SET Process</H3>
                        After transcription and translation of the Bxb1 recombinase gene, the proteins are expressed.
                        Two Bxb1 monomers form a dimer. In dimerized form, the recombinase proteins can bind to attL
                        and attR binding sites around the constitutive promoter J23119 on the reporter sequence.
                        When both sites are occupied, synapse between the two sites can happen. This can enable
                        flipping of the sequence and alter the direction of the promoter. The flipping process
                        changes the direction of the reporter promoter and transforms the attB and attP sites in
                        attL and attR. In its initial direction, the promoter starts GFP transcription; after
                        flipping, it turns to RFP. By measuring cell fluorescence with flow cytometry, we can
                        quantify the ratio of flipped sequences (attL/R ratio). The data-fitting procedure relies on
                        this key indicator.
                        <H3 id="reset-process-with-integrase-rdf-fusion-protein">RESET Process with Integrase-RDF Fusion
                            Protein</H3>
                        Another kind of recombinases - excisionases are able to recognize and bind attL and attR
                        sequences. With the help of excisionases, the state transitions become reversible.
                        Recombination Directionality Factors (RDF) are a kind of protein that achieves reverse
                        flipping to recover the sequence flipped by recombinases. By co-expression of the integrase
                        with the corresponding RDF, or expression of the integrase-RDF fusion protein, recombination
                        between the attL and attR sites can be induced. Thus, the flip-flop can restore to the
                        previous state.
                        </DIV></DIV><DIV id="FF_Compositions" class="demo-card-wide mdl-card mdl-shadow--2dp"><DIV class="mdl-card__supporting-text"><H2 id="model-composition">Model Composition</H2>
                        The following part describes species and reactions involved in different components of the
                        model.
                        <H3 id="pBAD promoter">Arabinose-Induced Protein Expression</H3>
                        We have decided to use an arabinose-induced promoter (pBAD) for the expression of
                        recombinases on ColE1 vector. This promoter can be modelled as the following chemical
                        system. <SPAN class="math display">\((AraB:AraC) \xrightarrow[]{K_{transcription}} mRNA \xrightarrow[]{K_{translation}} Recombinase\)</SPAN><SPAN class="math display">\(mRNA \xrightarrow[]{\gamma_{mRNA}} \emptyset\)</SPAN><SPAN class="math display">\(Recombinase \xrightarrow[]{\gamma_{prot}} \emptyset\)</SPAN>
                        The promoter pBAD binds to AraC and this represses transcription of mRNA. Arabinose will
                        bind to AraC and form the Arab:AraC compound, allowing transcription to occur.
                        
                        We assume that:<OL start="1"><LI>AraC is always in large concentration</LI><LI>AraC binding to arabinose occurs on a faster time scale to transcription</LI></OL>


                        Therefore it is not necessary to consider individual concentrations of arabinose and AraC. We
                        need only include the concentration of (Arab:AraC) complex in the model. The transcription
                        rate constant K on the schematic diagram above is not a simple constant but described as a
                        Michaelis-Mentin kinetics[4] equation below.
                        <SPAN class="math display">\(\frac{\mathrm{d}[mRNA]}{\mathrm{d}t}=K_{max}\frac{[Arab:AraC]^n}{[Arab:AraC]^n+K_{half}^n}-\gamma_{mRNA}[mRNA]\)</SPAN>
                        Although pBAD system is known for rigidity[2], we observed leaky expression in the
                        experiments. So we add a basal translation level to describe leakiness. The basal expression
                        level should be several orders of magnitude lower than the maximal level.
                        <SPAN class="math inline">\(\frac{\mathrm{d}[mRNA]}{\mathrm{d}t}=K_{basal}+K_{max}\frac{[Arab:AraC]^n}{[Arab:AraC]^n+K_{half}^n}-\gamma_{mRNA}[mRNA]\)</SPAN>
                        The mRNA degradation and dilution rates are combined into a single parameter. Protein
                        (recombinase) synthesis and degradation can be modeled with the equation below.
                        <SPAN class="math inline">\(\frac{\mathrm{d}[P]}{\mathrm{d}t}=\alpha_{Int}[mRNA]-\gamma_{Int}[P]\)</SPAN>
                        Again, we combine dilution and degradation rate of recombinase into a single parameter.
                        <DIV align="center"><STRONG>Figure1. Recombinase expression mechanism schematic drawing
                        </STRONG></DIV><H3 id="recombinase-functioning">Recombinase Functioning</H3>
                        Here we make the following assumptions about the reporter vector:

                        <OL start="1"><LI>The total concentration of
                                the reporter vector in cell remains constant during the whole reaction. This is based on
                                the
                                fact that reporter fluorescent proteins are on the stringent pSC101 vector, whose copy
                                number is
                                strictly controlled to around 5~10 nM[3]. We assume this concentration to be 5nM.
                            </LI><LI>The reporter sequences are divided into 4 states: B0F, B1F, B2F and R. Here F and R
                                denote
                                the
                                direction of the constitutive promoter. F stands for “Forward”, corresponding to the
                                initial
                                direction (GFP direction) before flipping. R stands for “Reverse”, corresponding to the
                                direction after the flipping (RFP direction). B0, B1 and B2 denote the number of
                                recombination
                                sites bound by the recombinase dimer.
                            </LI><LI>After 12h of reaction, the RFP expressed by the
                                promoter in reverse direction have all matured. The efficiency of the recombinase
                                flipping
                                mechanism is defined as the cell population fraction expressing RFP in FACS data.
                                <A href="https://2017.igem.org/Team:Peking/Project#Flip-flop" target="blank" style="color: #226CE4"><STRONG>See Experiment Part for detailed calculation</STRONG></A></LI></OL>
                        The following diagram summarizes the mechanism and key reactions in the recombinase flipping
                        mechanism.
                        <DIV align="center"><STRONG>Figure2. Recombinase flipping mechanism schematic drawing
                        </STRONG></DIV>
                        The table below are reactions and parameters involved in this model.
                        <DIV align="center"><STRONG>Table1. Recombinase flipping mechanism reaction and parameter table
                            Simbiology</STRONG></DIV>
                        These reactions, together with kinetic laws and reaction rate parameters, can be transformed
                        into an ordinary differential equation (ODE) system.
                        <H3 id="recombinase-rdf-fusion-protein-functioning">Recombinase-RDF Fusion Protein
                            Functioning</H3>
                        Similar mechanisms can describe recombinase-RDF fusion protein functioning. The recombinases
                        themselves cannot flip back the sequence flanked between attL and attR sites. They have to work
                        together with recombination directionality factors (RDFs) to accomplish this task.[5]
                        Several proposals on genetically engineered RDF expression were suggested by Bonnet and
                        colleagues. [6] In our experiments, we implemented the fusion protein construction,
                        which was also reported to be functional in other work. [7]
                        
                        We make the following assumptions about recombinase-RDF fusion proteins:<OL start="1"><LI>The recombinase-RDF fusion proteins have the similar transcription and translation
                                behavior
                                as the recombinases
                            </LI><LI>The binding and unbinding of recombinase-RDF fusion proteins to attL and attR sites are
                                reversible reactions
                            </LI><LI>Flipping of the target sequence is also reversible</LI></OL></DIV></DIV><DIV id="FF_Results" class="demo-card-wide mdl-card mdl-shadow--2dp"><DIV class="mdl-card__supporting-text"><H2 id="simulation-results">Simulation Results</H2><H3 id="simulation-tool-and-methods">Simulation Tool and Methods</H3>
                        We used MATLAB Simbiology to convert the chemical reaction system into an ODE system. This can
                        be done through building the reaction pathway in a graphical interface and specify the
                        parameters. Tasks as simulation, sensitivity analysis and parameter scanning can then be
                        deployed. Below is the diagram that represents our ODE system model. Yellow circles represent
                        reactions and blue ovals represent species.
                        <DIV align="center"><STRONG>Fig.3. Graphical representation of ODE system in MATLAB
                            Simbiology</STRONG></DIV>
                        Reaction rate parameters for pBAD transcription, mRNA degradation, integrase dimerization and
                        degradation are recorded in literature. Our goal is to use experimental data to estimate the
                        following parameters:<OL start="1"><LI>Association constant between Integrase dimer and attB/P recombination sites: <SPAN class="math inline">\(k_{attBP}, k_{DattBP}\)</SPAN> (We assume that two binding
                                parameters are the same, <SPAN class="math inline">\(k_{attBP1}=k_{DattBP2}\)</SPAN>)
                            </LI><LI>Flip reaction rate (forward and reverse) <SPAN class="math inline">\(k_{flip}, k_{flipr}\)</SPAN></LI><LI>Integrase translation rate <SPAN class="math inline">\(\alpha_{Int}\)</SPAN>. This
                                parameter is directly correlated with the RBS of the integrase, which is tuned through
                                experiment（<A href="https://2017.igem.org/Team:Peking/Project#Flip-flop" target="blank" style="color: #226CE4"><STRONG>See here for details</STRONG></A>).
                            </LI></OL>
                        Thus, we arrive at a 14-parameter ODE system, with 5 parameters sensitive. We started from
                        values described in literature and used constraint optimization using derivative-based method
                        (fmincon) to do further estimation. Simulation was done through the ODE23t solver (Mod.
                        Stiff/Trapezoidal) in MATLAB.
                        <H3 id="recombinase-dynamics">Recombinase Dynamics</H3>
                        Here we show the time-course variation of the species concentration under two conditions: 0 and
                        10nM Arabinose induction. Experimental data of Bxb1_F8 listed below.
                        <DIV align="center"><STRONG>Table2. Experiment observation of recombinase flipping reaction</STRONG></DIV><DIV align="center"><STRONG>Fig.4. Simulation result of no arabinose induction</STRONG></DIV>
                        We expect rather low expression of recombinase protein, which is shown in the simulation result.
                        Almost no sequence has both sites bound with recombinase dimers. Sequence of type R has final
                        concentration 0.23 nM, which is identical to the experimental results.
                        <DIV align="center"><STRONG>Fig.5. Simulation result of 10nM arabinose induction</STRONG></DIV>
                        Under 10nM arabinose induction, sequence of type R has final concentration 4.916 nM, which also
                        fits with experimental results. Sequence states stabilize after approximately 3 hours. After
                        nearly 2 hours, sequence in B2F state has all been flipped.
                        <H3 id="sensitivity-analysis">Sensitivity Analysis</H3>
                        In order to determine which parameters of the system can be tuned for our purpose, we perform a
                        sensitivity analysis towards the possible sensitive parameters. This allows us to indentify the
                        parameters that have the greatest influence on the final state of the system.<DIV align="center"><STRONG>Fig.6. Sensitivity of the promoter switching with respect to the main
                                parameters</STRONG></DIV>

                        The integrase RBS strength (<SPAN class="math inline">\(\alpha_{Int}\)</SPAN>) and the strength
                        of the integrase flipping rate (<SPAN class="math inline">\(k_{flip}\)</SPAN>) are biologically
                        tunable parameters. Flipping rate can be altered by selection of integrase. These parameters
                        have significant influence on the response of the switch to activation.<DIV align="center"><STRONG> Fig.7. Time integral results of sensitivity
                            analysis.</STRONG></DIV><H3 id="translation-and-degradation-operative-range">Translation and Degradation Operative
                            Range</H3>
                        The aim of the experiment in recombinase characteristics and standardization is to find the best
                        RBS-recombinase pair. In the model, this means determining the operative range of two
                        parameters: the recombinase flipping rate and translation rate.
                        We made the heat map of the final R state concentration under varying $\alpha_{Int}$ and
                        $k_{flip}$. Results show that recombinase devices have a rather wide functional range. Failure
                        to generate enough flipped sequences can happen only when one of the two parameters is low.<DIV align="center"><STRONG>Fig.8. Functional regions of the parameter space with
                            efficiency</STRONG></DIV>
                        A cross is put on the point of
                        Bxb1-F8 experiment in the above simulations
                        <H3 id="assembly">Assembly</H3>
                        We assembled the two recombinases and recombinase-RDF fusion proteins as the structure of our
                        design. The bio-flip-flop is constructed as two sequences each with an invertible promoter
                        flanked between recombination sites and expressing recombinase or recombinase-RDF fusion protein
                        mediating inversion of another sequence.
                        <DIV align="center"><STRONG>Fig.9. Schematic drawing of bio-flip-flop structure. Recombinases, recombinase-RDF fusion
                                proteins and their corresponding sites are in the same color. X_F denotes the fraction of
                                forward direction (right) of the upper promoter pBAD. Y_F denotes the fraction of forward
                                direction (right) of the lower promoter pTAC.</STRONG></DIV>
                        Then we did simulation of a whole process of bio-flip-flop with four flipping reactions
                        happening in sequential order induced by four temporally adjacent inducing events. Parameters
                        were estimated from experimental data. Inducing of the system is in the form of square waves to
                        simulate manual induction like changing of medium.
                        <DIV align="center"><STRONG>Fig.10. Simulation result of the whole bio-flip-flop process</STRONG></DIV>
                        Induction on X promoter (pBAD)
                        happens at 60min and 540min, lasting 180min each. Induction on Y promoter (pTAC) happens at
                        300min and 780min, lasting 180min each. Every state is held for 1h.
                        </DIV></DIV><DIV id="FF_Reference" class="demo-card-wide mdl-card mdl-shadow--2dp"><DIV class="mdl-card__supporting-text"><H2>References</H2><OL start="1"><LI>Singh, S., Ghosh, P., &amp; Hatfull, G. F. (2013). Attachment site selection and
                                identity
                                in Bxb1 serine integrase-mediated site-specific recombination. PLoS genetics, 9(5),
                                e1003490.
                            </LI><LI>Manen, D., &amp; Caro, L. (1991). The replication of plasmid pSC101. Molecular
                                microbiology,
                                5(2), 233-237.
                            </LI><LI>Sourjik, V.,&amp; Berg, H. C. (2004). Functional interactions between receptors in
                                bacterial
                                chemotaxis. Nature, 428(6981), 437.
                            </LI><LI>Rosano, G. L., &amp; Ceccarelli, E. A. (2014). Recombinant protein expression in
                                Escherichia
                                coli: advances and challenges. Frontiers in microbiology, 5.
                            </LI><LI>Lewis, J. A., &amp; Hatfull, G. F. (2001). Control of directionality in
                                integrase-mediated
                                recombination: examination of recombination directionality factors (RDFs) including Xis
                                and Cox proteins. Nucleic acids research, 29(11), 2205-2216.
                            </LI><LI>Bonnet, J., Subsoontorn, P., &amp; Endy, D. (2012). Rewritable digital data storage in
                                live
                                cells via engineered control of recombination directionality. Proceedings of the
                                National Academy of Sciences, 109(23), 8884-8889.
                            </LI><LI>Olorunniji, F. J., McPherson, A. L., Rosser, S. J., Smith, M. C., Colloms, S. D., &amp;
                                Stark, W. M. (2017). Control of serine integrase recombination directionality by fusion
                                with the directionality factor. Nucleic acids research, 45(14), 8635-8645.
                            </LI></OL><A class="mdl-button mdl-js-button mdl-button--raised mdl-button--accent mdl-js-ripple-effect" href="#top" style="background-color: #2D8EE8; color: white;">
                        Back to top
                    </A></DIV></DIV></SECTION><SECTION id="Controller" class="mdl-components__page mdl-grid"><DIV class="demo-card-wide mdl-card mdl-shadow--2dp"><DIV class="mdl-card__supporting-text"><H1>Controller</H1><SECTION class="docs-toc docs-text-styling" style="margin-left: 0px"><NAV class="section-content" style="margin-left: 0px"><UL style="margin-left: 0px;"><LI><A href="#Controller_Basis">Biological Basis</A></LI><LI><A href="#Controller_Composition">Model Composition and Assumptions</A></LI><LI><A href="#Controller_Results">Simulation Results</A></LI><LI><A href="#Controller_Reference">References</A></LI></UL></NAV></SECTION><H2>Intention of the Model</H2>
                        In this model, we aim to describe the sequence modification process of the control unit sequence
                        with recombination sites. Different recombinases are expressed in series to alter the direction
                        of terminators between corresponding recombination sites on the control unit circuit. The
                        combination of unidirectional terminators in different directions (reverse or foward) determines
                        the state of RNA polymerase flux, thus controlling expression of the downstream gene.

                        Though we have done simulation of recombinase dynamics in the previous section, it is necessary
                        to consider two key concepts of biological processes:
                        1. population behavior
                        2. stochastic process

                        The reason for modeling cell population behavior is that deployment of synthetic
                        bacterial
                        devices would certainly involve populations of cells. Furthermore, we demonstrate our design
                        through continuous imaging of entire cell cultures. This is a theory-experiment interaction
                        procedure. The reason for stochastic modeling is self-evident. Cellular biological processes,
                        especially gene expression, are innately noisy and stochastic.[1] Previous work has
                        demonstrated the effectiveness of the integration of population-level dynamics and stochastic
                        cellular responses to inputs.[2]It is important to explain how stochastic single-cell
                        responses affect overall population-level distributions and behavior.

                        This section describes a Markov model of recombinase-based DNA flipping and a Gillespie
                        stochastic simulation method[3] to observe theoretical sequence state evolution. We
                        will focus on our demonstration design, which can be generalized to other designs.</DIV></DIV><DIV id="Controller_Basis" class="demo-card-wide mdl-card mdl-shadow--2dp"><DIV class="mdl-card__supporting-text"><H2>Biological Basis</H2>
                        The control units are implemented experimentally by transforming a plasmid into an E. coli. This
                        plasmid has similar structure as the reporter plasmid in the previous section. However, in this
                        situation the sequence fragments being inverted are terminators. Using experimentally screened
                        uni-directional terminators, the transcription or polymerase flux4 can be precisely controlled.
                        The figure below illustrates the result of the design <A href="https://2017.igem.org/Team:Peking/Software" target="blank" style="color: #226CE4"><STRONG>software</STRONG></A>. 

                        In the seven-segment decoder tube demonstration, if we would like to display “1-2-3” counting, 3
                        kinds of control units have to be transformed into 3 strains of bacteria. 2 of them has only one
                        pair of recombination sites, and only undergoes recombinase inversion once. The other one has
                        two nested recombination sites. Successive inversion results in the same state as the initial
                        one.</DIV></DIV><DIV id="Controller_Composition" class="demo-card-wide mdl-card mdl-shadow--2dp"><DIV class="mdl-card__supporting-text"><H2 id="equations-and-parameters">Model Composition and Assumptions</H2>

                        A Markov process is defined upon a specific state space. We define that each state occupies a
                        specific state. Then we can use Markov models to track cell state changes over time. This
                        “state” consists of two parts: the state of recombination sites on the control unit sequence and
                        the concentration or molecule number of two types of recombinases A and B. This definition was
                        first proposed by Victoria Hsiao and colleagues to model a population-based event detector.5

                        The state of a single cell is defined by the DNA state and the copy numbers of integrases A and
                        B. We denote the state of a single cell by (DNA, IntA, IntB).

                        <H3 id="cell-states">Cell States</H3>
                        All of the four possible DNA states are: the initial state (<SPAN class="math inline">\(S_{init}\)</SPAN>),
                        the IntA inversion only state (<SPAN class="math inline">\(S_a\)</SPAN>), the IntB inversion
                        only state (<SPAN class="math inline">\(S_b\)</SPAN>), and the IntA then IntB inversion
                        state (<SPAN class="math inline">\(S_{ab}\)</SPAN>). We omit the state IntB then IntA
                        inversion state (<SPAN class="math inline">\(S_{ba}\)</SPAN>) because in our design and
                        experiments, integrase expression is always induced in order. IntB expression can never be
                        triggered before that of IntA. These states and transitions by recombinase inversions are
                        illustrated below.Our implementation requires that the control unit sequence on a stringent, low copy
                        number
                        plasmid. Thus we assume that each cell only has one copy of the control unit.
                        Uni-directional terminators are screened by the criterion that the forward terminator
                        strength[6] exceeds the reverse one by a few orders of magnitude. This allows us to
                        assume that when the terminator is in the forward direction, it can completely block RNA
                        polymerase, while in the reverse direction, it cannot function(<A href="https://2017.igem.org/Team:Peking/Project#Controller" target="blank" style="color: #226CE4"><STRONG>Click for more details</STRONG></A>).

                        <H3 id="integrase-production-and-tetramerization">Integrase Production and Tetramerization</H3>
                        Serine integrases are produced as monomers that form dimers, search for specific attB and attP
                        sequences, and, once both attB and attP sites are occupied, form a tetramer (dimer of dimers)
                        that digests, flips and re-ligates the DNA[7].
                        In this model we also consider leaky expression of the recombinases, because of observation of
                        such phenomena in recombinase characterization experiments. Furthermore, induction of
                        recombinase expression happens in order and we assume that at each time point, only one of the
                        two inducers (a and b) exists. Thus we define the following table of state transitions and
                        parameters:<SPAN class="math inline">\(x \in \{init, a, b, ab\})\)</SPAN>,<SPAN class="math inline"><SPAN class="math inline">\(\gamma_A(t)\)</SPAN> and <SPAN class="math inline">\(\gamma_B(t)\)</SPAN> are defined as follows: <SPAN class="math display">\[
\gamma_A(t) =
\begin{cases}
k_{prodA} + k_{leakA},  &amp; \text{if inducer a exists} \\
k_{leakB}, &amp; \text{otherwise}
\end{cases}
\]</SPAN><SPAN class="math display">\[
\gamma_B(t) =
\begin{cases}
k_{prodB} + k_{leakB},  &amp; \text{if inducer a exists} \\
k_{leakB}, &amp; \text{otherwise}
\end{cases}
\]</SPAN>

                        The serine integrases need to tetramerize prior to DNA recombination. Each DNA binding site
                        (attB and attP) is bound by two copies of integrase monomers in an independent manner to form a
                        dimer. Once both of the binding sites on both DNA attachment sites are
                        occupied, a dimer of the dimers (tetramer) is formed and recombination can occur. Here we deduce
                        the reaction propensity of the tetramerization process. We only focus on the number of integrase
                        molecules inside the cell, and ignore the detailed process of transcription and translation, and
                        represent the process with a single parameter.

 <SPAN class="math inline">\(S_x:Int_i (i = 1,2,3,4) (x \in \{init, a, b, ab\})\)</SPAN>     denote
                        the DNA-integrase complexes where <SPAN class="math inline">\(i\)</SPAN> denote the number of
                        integrase molecules bound to DNA sequence. DNA recombination occurs when the complex state is at
                        <SPAN class="math inline">\(S_x:Int_4\)</SPAN>, which means the tetramerization of integrases on
                        the DNA. Because of independent binding of DNA sequence with integrase dimers, the transition of
                        states can be modeled by the following equations: <SPAN class="math display">\[
S_x \rightleftarrows S_x:Int_1 \rightleftarrows S_x:Int_2 \rightleftarrows S_x:Int_3 \rightleftarrows S_x:Int_4
\]</SPAN>where the binding rate of DNA recombination sites (attB, attP) and integrase dimers is <SPAN class="math inline">\(k_{bind}\)</SPAN> (reaction from left to right side) and unbinding
                        rate is <SPAN class="math inline">\(k_{unbind}\)</SPAN>. The dynamics can be modeled by the
                        following ordinary differential equations. <SPAN class="math inline">\(\mathbb{P}_t(S_x:Int_i, n_*) (* = A, B)\)</SPAN>
                        is the joint probability distribution.<DIV style="font-size:0.74em"><SPAN class="math display">\[
\frac{d}{dt}\mathbb{P}_t(S_x, n_*) = -k_{bind}n_*\mathbb{P}_t(S_x, n_*) + k_{unbind}\mathbb{P}_t(S_x:Int_1, n_*)
\]</SPAN><SPAN class="math display">\[
\frac{d}{dt}\mathbb{P}_t(S_x:Int_1, n_*-1) = -(k_{bind}(n_*-1)+k_{unbind})\mathbb{P}_t(S_x:Int_1, n_*-1) + k_{bind}n_*\mathbb{P}_t(S_x, n_*) + k_{unbind}\mathbb{P}_t(S_x:Int_2,n_*-2)
\]</SPAN><SPAN class="math display">\[
\frac{d}{dt}\mathbb{P}_t(S_x:Int_2, n_*-2) = -(k_{bind}(n_*-2)+k_{unbind})\mathbb{P}_t(S_x:Int_2, n_*-2) + k_{bind}(n_*-1)\mathbb{P}_t(S_x:Int_1, n_*-1) + k_{unbind}\mathbb{P}_t(S_x:Int_3,n_*-3)
\]</SPAN><SPAN class="math display">\[
\frac{d}{dt}\mathbb{P}_t(S_x:Int_1, n_*-3) = -(k_{bind}(n_*-3)+k_{unbind})\mathbb{P}_t(S_x:Int_3, n_*-3) + k_{bind}(n_*-2)\mathbb{P}_t(S_x:Int_2, n_*-2) + k_{unbind}\mathbb{P}_t(S_x:Int_4,n_*-4)
\]</SPAN><SPAN class="math display">\[
\frac{d}{dt}\mathbb{P}_t(S_x:Int_4, n_*-4) = -k_{unbind}n_*\mathbb{P}_t(S_x:Int_4, n_*-4) + k_{bind}(n_*-3)\mathbb{P}_t(S_x:Int_3, n_*-3)
\]</SPAN></DIV>
                        We assume that binding and unbinding of integrase molecules to DNA equilibrate fast enough compared to the dynamics of DNA recombination, and the production and degradation of integrases (See Parameter Supplemental) and hence the above equations converge to an equilibrium while <SPAN class="math inline">\(n_*\)</SPAN> remain constant (equilibrium approximation). Let the above equations all equal to zero and solve in terms of <SPAN class="math inline">\(\frac{d}{dt}\mathbb{P}_t(S_x:Int_4, n_*-4)\)</SPAN>,    we can write:<SPAN class="math display" style="font-size: small">\[
\frac{d}{dt}\mathbb{P}_t(S_x:Int_4, n_*-4) = \frac{n_* (n_*-1)(n_*-2) (n_*-3)}{K_{d*}^4 + K_{d*}^3 n_* + K_{d*}^2 n_* (n_*-1) + K_{d*}n_*(n_*-1)(n_*-2) + n_* (n_*-1)(n_*-2)(n_*-3)}
\]</SPAN> (<SPAN class="math display">\(K_{d*} = k_{unbind}/k_{bind}\)</SPAN> )<H3 id="continuous-time-markov-process">Continuous-time Markov Process</H3>
Rather than accounting fro all individual DNA-integrase interactions, we have created a minimal model of stochastic transitions. Here only the final DNA states <SPAN class="math inline">\((S_o, S_a, S_b, S_{ab})\)</SPAN> and the number of integrase monomer molecules (IntA, IntB) are tracked and all integrase activity is encompassed in the <SPAN class="math inline">\(k_{flip*} (*=A,B)\)</SPAN> term. Because no cooperativity has been observed in Bxb1 or TP901-1 DNA binding, e.g. occupation of attB does not increase the probability of attP binding, we assume that the flipping activity is zero unless at least four molecules are present. Then, the propensity function for state transitions is a function of integrase concentration (equivalent to protein molecule number in cell,
                        if cell volume is assumed to be constant): <SPAN class="math display" style="font-size: small">\[
\alpha_i(Int_*) = k_{flip*}(\frac{n_* (n_*-1)(n_*-2) (n_*-3)}{K_{d*}^4 + K_{d*}^3 n_* + K_{d*}^2 n_* (n_*-1) + K_{d*}n_*(n_*-1)(n_*-2) + n_* (n_*-1)(n_*-2)(n_*-3)})
\]</SPAN><SPAN class="math inline">\(i = 1, 2, 3; * = A, B\)</SPAN> Then the table of state transition and intensities can be completed. We consider three kinds of state transitions as described previously: <SPAN class="math inline">\(S_{init}\)</SPAN> to <SPAN class="math inline">\(S_a\)</SPAN>, <SPAN class="math inline">\(S_{init}\)</SPAN> to <SPAN class="math inline">\(S_b\)</SPAN>, and <SPAN class="math inline">\(S_a\)</SPAN> to <SPAN class="math inline">\(S_{ab}\)</SPAN>. <SPAN class="math inline">\(S_{init}\)</SPAN> to <SPAN class="math inline">\(S_b\)</SPAN> is due to missing of integrase A flipping. This transition denotes the system’s ineffective fraction.Numbers of cells occupying each state are calculated by the summing equation below: <SPAN class="math display">\[
S_{i} = \sum_{n_A=0}^{\infty}\sum_{n_B=0}^{\infty} S(i,n_A,n_B), i \in \{S_{init},S_a,S_b\}
\]</SPAN> because we only concentrate on the cell numbers, not the specific protein copy numbers in cells.


                    </SPAN></DIV></DIV><DIV id="Controller_Results" class="demo-card-wide mdl-card mdl-shadow--2dp"><DIV class="mdl-card__supporting-text"><H2 id="simulation-results">Simulation Results</H2><H3>Simulation Setting</H3>
                        In the simulation in this section,
                        we consider induction with step inputs, where the inducers are either present or not
                        present. Concentrations of the inducers when they are ‘’on’’ will be held constant. Also, it
                        is important to note that inducer <SPAN class="math inline">\(a\)</SPAN> is still present
                        during and after time <SPAN class="math inline">\(\Delta t\)</SPAN> when inducer <SPAN class="math inline">\(b\)</SPAN> is introduced. We predict qualitative circuit
                        behavior with biologically plausible parameters simplified to a minimal set: integrase
                        production, degradation, tetramerization and DNA flipping. The integrase monomer
                        dissociation constant <SPAN class="math inline">\(K_{d*} (*=intA, intB)\)</SPAN> was
                        estimated from Singh’s work. For more details of parameter setting for simulation please see
                        our supplement page.

                        <H3>Whole Process and Analysis: Guidance for experimentalists</H3>
                        Qualitative prediction of the population behavior of 3000 or 1000 cells was simulated. We
                        first limit the simulator to the situation of only one pair of recombination sites on the
                        circuit. From the results below we can see that after approximately 10h the recombinase intA
                        complete the flipping and the reaction is rather efficient.<DIV align="center"><STRONG>Fig.1.
                            Simulation result of only one pair of recombination sites on control unit circuit</STRONG></DIV>


                        The results for two pairs of recombination sites are shown below. We did two simulations. The
                        first one (shown in the left figure), with two recombinases induced with no time separation,
                        the system acts in an unsatisfactory way and approximately half of the cells are in the
                        undesired state <SPAN class="math inline">\(S_b\)</SPAN>. In the second one, inducing of
                        intA and intB are separated with 5 hours. Then the system performs rather stably, with
                        approximately 90% cells in desired state. This provides guidance for team members working in
                        the lab. <DIV align="center"><STRONG>Fig.2. Simulation result of two pairs
                            of recombination sites</STRONG></DIV>


                        The left figure shows state evolution with no time separation in
                        inducing events A and B. The right figure shows state evolution with time separation of
                        <SPAN class="math inline">\(\Delta t = 5h\)</SPAN>.
                        Finally we show specific cell transitions in Gillespie simulation. 100 cells are sampled from
                        the population. Each vertical line denotes a cell state transition event (from <SPAN class="math inline">\(S_{init}\)</SPAN> to <SPAN class="math inline">\(S_a\)</SPAN>,
                        <SPAN class="math inline">\(S_{init}\)</SPAN> to <SPAN class="math inline">\(S_b\)</SPAN>,
                        from <SPAN class="math inline">\(S_a\)</SPAN> to <SPAN class="math inline">\(S_{ab}\)</SPAN>).
                        The last two subplots show the simulation results of protein expression in single cell.


                        <DIV align="center"><STRONG>Fig.3. Specification of state transitions in
                            Gillespie simulation using 100 sampled cells.</STRONG></DIV></DIV></DIV><DIV id="Controller_Reference" class="demo-card-wide mdl-card mdl-shadow--2dp"><DIV class="mdl-card__supporting-text"><H2>References</H2><OL start="1"><LI>Elowitz, M. B., Levine, A. J., Siggia, E. D., &amp; Swain, P. S. (2002). Stochastic gene
                                expression in a single cell. Science, 297(5584), 1183-1186.
                            </LI><LI>Ruess, J., Parise, F., Milias-Argeitis, A., Khammash, M., &amp; Lygeros, J. (2015).
                                Iterative experiment design guides the characterization of a light-inducible gene
                                expression circuit. Proceedings of the National Academy of Sciences, 112(26), 8148-8153.
                            </LI><LI>Gillespie, D. T. (1977). Exact stochastic simulation of coupled chemical reactions. J.
                                phys. Chem, 81(25), 2340-2361.
                            </LI><LI>Darzacq, X., Shav-Tal, Y., De Turris, V., Brody, Y., Shenoy, S. M., Phair, R. D., &amp;
                                Singer, R. H. (2007). In vivo dynamics of RNA polymerase II transcription. Nature
                                structural &amp; molecular biology, 14(9), 796-806.
                            </LI><LI>Hsiao, V., Hori, Y., Rothemund, P. W., &amp; Murray, R. M. (2016 ). A population‐based
                                temporal logic gate for timing and recording chemical events. Molecular systems biology,
                                12(5), 869.
                            </LI><LI>Chen, Y. J., Liu, P., Nielsen, A. A., Brophy, J. A., Clancy, K., Peterson, T., &amp;
                                Voigt,
                                C. A. (2013). Characterization of 582 natural and synthetic terminators and
                                quantification of their design constraints. Nature methods, 10(7), 659-664.
                            </LI><LI>Rutherford, K., Yuan, P., Perry, K., Sharp, R., &amp; Van Duyne, G. D. (2013).
                                Attachment
                                site recognition and regulation of directionality by the serine integrases. Nucleic
                                acids research, 41(17), 8341-8356.
                            </LI><LI>Singh, S., Ghosh, P., &amp; Hatfull, G. F. (2013). Attachment site selection and
                                identity in
                                Bxb1 serine integrase-mediated site-specific recombination. PLoS genetics, 9(5),
                                e1003490.
                            </LI></OL><A class="mdl-button mdl-js-button mdl-button--raised mdl-button--accent mdl-js-ripple-effect" href="#top" style="background-color: #2D8EE8; color: white;">
                        Back to top
                    </A></DIV></DIV></SECTION><SECTION id="Clock" class="mdl-components__page mdl-grid"><DIV class="demo-card-wide mdl-card mdl-shadow--2dp"><DIV class="mdl-card__supporting-text"><H1>Clock</H1><SECTION class="docs-toc docs-text-styling" style="margin-left: 0px"><NAV class="section-content" style="margin-left: 0px"><UL style="margin-left: 0px;"><LI><A href="#Clock_Basis">Biological Basis and Assumptions</A></LI><LI><A href="#Clock_Equations">Equations and Parameters</A></LI><LI><A href="#Clock_Results">Simulation Results</A></LI><LI><A href="#Clock_Reference">References</A></LI></UL></NAV></SECTION><H2>Intention of the Model</H2>
                        We attempts to develop a framework of biological sequential circuits that are programmable.
                        The envisioned circuit is capable of both storing states in DNA and automatically running a
                        series of instructions in a specific order. According to our design, our bio-flip-flop has
                        two promoters, each triggered by an inducing signal. Therefore, if we would like to build an
                        automatic triggering system for the bio-flip-flop, we need at least two signals in our
                        clock. It is then self-evident to incorporate the classic structure in synthetic biology:
                        the repressilator. This model serves as a proof of concept for such implementation. We
                        first
                        constructed an ordinary differential equation (ODE) system to describe the repressilator,
                        then we used two repressor proteins expressed by the repressilator to regulate the two
                        promoters. Finally we adjusted parameters and gave a simulation result as a proof of the
                        possibility of repressilator-driven bio-flip-flop.

                    </DIV></DIV><DIV id="Clock_Basis" class="demo-card-wide mdl-card mdl-shadow--2dp"><DIV class="mdl-card__supporting-text"><H2>Biological Basis and Assumptions</H2>
                        The
                        repressilator is a synthetic genetic regulatory network consisting of a ring-oscillator with
                        three genes, each expressing a protein that represses the next gene in the loop.[1]
                        This network was designed from scratch to exhibit a stable oscillation which is reported via
                        the expression of green fluorescent protein, and hence acts like an electrical oscillator
                        system with fixed time periods. The repressilator consists of three genes connected in a
                        feedback loop, such that each gene represses the next gene in the loop, and is repressed by
                        the previous gene.<DIV align="center"><STRONG>Fig.1. The genetic structure of the repressilator network</STRONG></DIV>
                        We decided to implement our repressilator-triggered bio-flip-flop with two repressor proteins
                        regulating the two promoters’ transcription. The basis of ODE system construction is
                        illustrated as follows:<DIV align="center"><STRONG>Fig.2. Schematic
                                drawing of the repressilator-driven bio-flip-flop </STRONG></DIV>
                        The upper promoter (X) is <SPAN class="math inline">\(P_{LtetO}\)</SPAN> and is repressed by tetR protein. The lower
                        promoter (Y) is <SPAN class="math inline">\(P_{R}\)</SPAN> and is repressed by cI
                        protein.Simulation of bio-flip-flop starts at a later time than repressilator because the
                        expression levels of the repressor proteins at the beginning is not high enough.

                    </DIV></DIV><DIV id="Clock_Equations" class="demo-card-wide mdl-card mdl-shadow--2dp"><DIV class="mdl-card__supporting-text"><H2 id="equations-and-parameters">Equations and Parameters</H2><H3 id="repressilator">Repressilator</H3>
                        The following table lists the reactions in the repressilator network. X, Y, Z represent LacI,
                        TetR and cI mRNAs, and PX, PY, PZ represent the corresponding proteins.


                        Calculation of degradation rates are described as follows: <SPAN class="math display">\[
k_{dmRNA} = \frac{ln(2)}{\tau_{mRNA}}
\]</SPAN><SPAN class="math display">\[
k_{dprot} = \frac{ln(2)}{\tau_{protein}}
\]</SPAN><SPAN class="math inline">\(\tau_{mRNA}\)</SPAN> and <SPAN class="math inline">\(\tau_{protein}\)</SPAN> are
                        half lives of mRNA and protein. Calculation of maximum transcription rate: <SPAN class="math display">\[
a_{tr} = 60(ps_a - ps_0)
\]</SPAN><SPAN class="math inline">\(ps_a\)</SPAN> is “tps active” and <SPAN class="math inline">\(ps_0\)</SPAN> is
                        “tps_repressive”. Calculation of translation rate: <SPAN class="math display">\[
k_{tl} = \frac{eff}{t_{ave}}
\]</SPAN><SPAN class="math inline">\(eff\)</SPAN> is translation efficiency and <SPAN class="math inline">\(t_{ave}\)</SPAN> is average mRNA life time. <SPAN class="math inline">\(t_{ave} = \frac{\tau_{mRNA}}{ln(2)}\)</SPAN><H3>Transcription in bio-flip-flop</H3> The transcription kinetics in the bio-flip-flop were
                        modified to be repressed by repressors. These can be described by the following
                        equations:

                        </DIV></DIV><DIV id="Clock_Results" class="demo-card-wide mdl-card mdl-shadow--2dp"><DIV class="mdl-card__supporting-text"><H2 id="simulation-results">Simulation Results</H2>
                        We conducted our simulation using Tellurium[3], a Python-based simulation platform.
                        We first adjusted the parameters protein half life and mRNA half life (equivalently
                        transcription and translation of repressor proteins) to adjust oscillation period as the one
                        observed in the experiment (<A href="https://2017.igem.org/Team:Peking/Project#Clock" target="blank" style="color: #226CE4"><STRONG>See experiment</STRONG></A>). <DIV align="center"><STRONG>Fig.1. Repressilator simulation results of tetR and cI protein</STRONG></DIV>
                        Then we incorporate the bio-flip-flop into the model, and we got the following simulation
                        results: <DIV align="center"><STRONG>Fig.2. Repressilator-driven
                                frequency divider simulation with two trigger signal groups</STRONG>X_F and Y_F have
                            the same
                            meaning as those described in <A href="https://2017.igem.org/Team:Peking/Model#Flip-flop" target="blank" style="color: #226CE4"><STRONG>Flip-flop</STRONG></A> section.
                        </DIV>


                        In this simulation
                        experiment, the flip-flop lost approximately 10% of its initial state after two trigger
                        signal groups.

                    </DIV></DIV><DIV id="Clock_Reference" class="demo-card-wide mdl-card mdl-shadow--2dp"><DIV class="mdl-card__supporting-text"><H2>References</H2><OL start="1"><LI>Elowitz, M. B., &amp; Leibler, S. (2000). A synthetic oscillatory network of
                                transcriptional
                                regulators. Nature, 403(6767), 335-338.
                            </LI><LI>https://wwwdev.ebi.ac.uk/biomodels/BIOMD0000000012</LI><LI>http://tellurium.analogmachine.org/</LI></OL><A class="mdl-button mdl-js-button mdl-button--raised mdl-button--accent mdl-js-ripple-effect" href="#top" style="background-color: #2D8EE8; color: white;">
                        Back to top
                    </A></DIV></DIV></SECTION><SECTION id="Supplementary" class="mdl-components__page mdl-grid"><DIV class="demo-card-wide mdl-card mdl-shadow--2dp"><DIV class="mdl-card__supporting-text"><H1>Supplementary</H1>
                        The following are the parameters and tools we used in modelling and simulation. We share them in the interest of reproducibility.

                        </DIV></DIV></SECTION></DIV></DIV></MAIN></DIV></DIV><FOOTER class="mdl-mega-footer" style="height: 260px; margin-bottom: 0px;"><DIV class="mdl-mega-footer__middle-section"><DIV class="mdl-mega-footer__drop-down-section"><H1 class="mdl-mega-footer__heading">Acknowledgement</H1></DIV><DIV class="mdl-mega-footer__drop-down-section"><H1 class="mdl-mega-footer__heading">Connection</H1><UL class="mdl-mega-footer__link-list" style="margin-left: 0px;&gt;
                    &lt;li&gt;&lt;a href=" https:=""><LI><A href="https://www.facebook.com/pekingigem">Facebook</A></LI><LI><A href="https://www.youtube.com/channel/UCefEYVCVqKJLmoJrSxivgZQ">Youtube</A></LI><LI><A href="https://twitter.com/iGEM_Peking">Twitter</A></LI><LI><A href="https://github.com/PekingiGEM2017">Github</A></LI></UL></DIV><DIV class="mdl-mega-footer__drop-down-section"><H1 class="mdl-mega-footer__heading">Contact</H1><UL class="mdl-mega-footer__link-list" style="margin-left: 0px;&gt;
                    &lt;li&gt;&lt;a href=" http:=""><LI><A href="http://www.synbiowiki.com">SynBioWiki.com</A></LI><LI><A href="mailto:PekingIGEM@gmail.com">Mail</A></LI></UL></DIV><DIV class="mdl-mega-footer__drop-down-section"><H2 style="color: white; "><STRONG>Peking iGEM</STRONG> 2017</H2><UL class="mdl-mega-footer__link-list" style="margin-left: 0px;"><LI style="color: white; line-height: 1.88em;">Peking University, No.5 Yiheyuan Road Haidian
                        District, Beijing, P.R.China100871
                    </LI></UL></DIV></DIV></FOOTER></DIV></DIV></DIV></BODY></HTML>