<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_Peking_Protocols skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><DIV id="top_title"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:Peking/Protocols</SPAN></H1></DIV><DIV id="HQ_page"><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><DIV class="mdl-layout mdl-js-layout mdl-layout--fixed-header"><HEADER class="mdl-layout__header" style="position:fixed; margin-top: -7px; !important;"><DIV class="mdl-layout__header-row"><A class="mdl-navigation__link" href="https://2017.igem.org/Team:Peking" style="color: #000000; font-size: x-large"><STRONG>Peking iGEM </STRONG> 2017</A><NAV class="mdl-navigation mdl-layout--large-screen-only"><A class="mdl-navigation__link" href="https://2017.igem.org/Team:Peking">Home</A><A class="mdl-navigation__link" href="https://2017.igem.org/Team:Peking/Project#Introduction">Project</A><A class="mdl-navigation__link" href="https://2017.igem.org/Team:Peking/Model#Overview">Modelling</A><A class="mdl-navigation__link" href="https://2017.igem.org/Team:Peking/Software">Software</A><A class="mdl-navigation__link" href="https://2017.igem.org/Team:Peking/Hardware">Hardware</A><A class="mdl-navigation__link" href="https://2017.igem.org/Team:Peking/Lab" style="color: #000; font-weight: 500;">Lab</A><A class="mdl-navigation__link" href="https://2017.igem.org/Team:Peking/HP">Practices</A><A class="mdl-navigation__link" href="https://2017.igem.org/Team:Peking/Parts#Overview">Parts</A><A class="mdl-navigation__link" href="https://2017.igem.org/Team:Peking/Team">Team</A></NAV></DIV></HEADER></DIV><MAIN class="docs-layout-content mdl-layout__content mdl-color-text--grey-600" style="width: 100%; padding-bottom: 0px; background-color: #FAFAFA;"><DIV class="demo-card-wide mdl-card mdl-shadow--2dp"><DIV class="mdl-card__title" style="background: url('https://static.igem.org/mediawiki/2017/f/fb/Peking_Protocols_Background.jpg') center / cover; padding-left: 50px;"><DIV class="mdl-card__title-text"><H1 style="font-size: xx-large; color: white;text-shadow:2px 2px 8px #070707;"><STRONG>Protocols</STRONG></H1></DIV></DIV><DIV class="mdl-card__supporting-text" style="line-height: 2em;text-align: justify; color: #3A3A3A; padding-left: 50px; padding-top: 50px; padding-bottom:50px"><SECTION class="docs-toc docs-text-styling"><NAV class="section-content"><UL style="margin-left: 0px;"><LI><A href="#p1">1. Ligation</A></LI><LI><A href="#p2">2. Chemical Transformation</A></LI><LI><A href="#p3">3. Agarose Gel Electrophoresis</A></LI><LI><A href="#p4">4. Plasmid Extraction</A></LI><LI><A href="#p5">5. Gel Extraction</A></LI><LI><A href="#p6">6. Gibson assembly</A></LI><LI><A href="#p7">7. PCR using Q5 high-Fidelity DNA polymerase</A></LI><LI><A href="#p8">8. Golden Gate Assembly</A></LI><LI><A href="#p9">9. Growing Overnight cultures</A></LI><LI><A href="#p10">10. Colony PCR</A></LI><LI><A href="#p11">11. Preparation of LB Broth, LB Ager, and M9 culture medium</A></LI><LI><A href="#p12">12. Making Competent cells</A></LI><LI><A href="#p13">13. Cell sorting for recombination efficiency characterization</A></LI><LI><A href="#p14">14. Imaging Oscillation of Repressilator</A></LI></UL></NAV></SECTION><H2 id="p1">Ligation</H2><H3>Materials:</H3>
            1 µL T4 DNA Ligase 
            2 µL 10X T4 DNA ligase buffer 
            50ng Vector Plasmid 
            Insert DNA (Required molar ratio of 1:3 :: vector:insert) 
            Sterile water (Make reaction up to 20 µL)<H3>Methods:</H3>
            Incubate at room temperature for 30 - 60 min or at 16℃ for 60min.<H2 id="p2">Chemical Transformation</H2><H3>Methods:</H3><OL start="1"><LI>Thaw 50µL DH10B competent E. coli cells on ice.</LI><LI>Add 5 µl DNA from a ligation reaction mix or 10-100ng DNA of a plasmid.</LI><LI>Place the mixture on ice for 20-30 minutes.</LI><LI>Heat shock at 42°C for 90 seconds.</LI><LI>Place on ice for 5 minutes.</LI><LI>Pipette 200 µl of room temperature LB media into the mixture.</LI><LI>Incubate at 37°C and 220 rpm for 60 minutes.</LI><LI>Add 50-100 µl of the transformed cells to the selection plate.</LI></OL><H2 id="p3">Agarose Gel Electrophoresis</H2><H3>Methods:</H3><OL start="1"><LI>Prepare 10% w/v solution of agarose powder in 1× TAE buffer .Weigh agarose powder and TAE buffer and add them to a flask.</LI><LI>Melt the mixture in a microwave until the agarose is completely dissolved.</LI><LI>Add SybrSafe®（1/10000 v/v solution of the mixture）to the solution.</LI><LI>Pour the solution into the gel casting tray with appreciate comb.</LI><LI>Let the gel cool until it is solid and then pull out the comb.</LI><LI>Place the gel in the electrophoresis chamber and add enough TAE Buffer.</LI><LI>Pipette DNA samples mixed with appreciate amount of loading dye (6×) into wells on the gel and add the DNA ladder.</LI><LI>Run the gel at 100V for about half an hour.</LI></OL><H2 id="p4">Plasmid Extraction</H2>
            Minipreps were performed according to the Axygen Miniprep Kit, we also make some improvement.
            <H3>Methods :</H3><OL start="1"><LI>Prepare 5mL bacteria solution grown overnight in LB broth, centrifuge at 12,000×g for 1min. Discard the supernatant.</LI><LI>Add 250µL Buffer S1 to resuspend bacteria cells and transfer to a 1.5mL centrifuge tube.</LI><LI>Add 250µL Buffer S2 and gently invert the tube 4-6 times to mix.</LI><LI>Add 350µL Buffer S3 and gently invert the tube immediately but gently 6-8 times.</LI><LI>Centrifuge for 10min at 12,000×g in a microcentrifuge.</LI><LI>Apply the supernatants from step 5 to the Axygen spin column by pipetting.</LI><LI>Centrifuge for 1min. Discard the flow-through.</LI><LI>Wash the Axygen spin column by adding 500µL Buffer W1, and centrifuging for 1min. Discard the flow-through.</LI><LI>Wash the Axygen spin column by adding 700µL Buffer W2, and centrifuging for 1min. Discard the flow-through.</LI><LI>Repeat the step 9</LI><LI>Centrifuging for an additional 2min to remove residual wash buffer.</LI><LI>Drying the Axygen spin column for about 15min at 65°C to keep the value of 260/280 in a normal level.</LI><LI>Place the Axygen spin column in a clean 1.5mL microcentrifuge tube. Add 30-50µL ddH2O (65°C) to the center of the Axygen spin column, let stand for 2min, and centrifuge for 1min.</LI></OL><H2 id="p5">Gel Extraction</H2>
            Was performed according to the Tiangem Gel Extraction Kit.<H2 id="p6">Gibson assembly</H2><H3>Methods :</H3><OL start="1"><LI>Keep 15µL Gibson mix on ice.</LI><LI>Add 5µL DNA fragment (Vector fragment: less than 15ng; small fragment: 20-50ng).</LI><LI> Incubate at 50℃ to 60min.</LI><LI>Transformation.</LI></OL><TABLE class="mdl-data-table mdl-js-data-table"><THEAD><TR><TH class="mdl-data-table__cell--non-numeric">Gibson Mix</TH><TH>1.2 mL</TH></TR></THEAD><TBODY><TR><TD class="mdl-data-table__cell--non-numeric">5×ISO buffer</TD><TD>320 µL</TD></TR><TR><TD class="mdl-data-table__cell--non-numeric">10U/µL T5 exo</TD><TD>0.64 µL</TD></TR><TR><TD class="mdl-data-table__cell--non-numeric">20U/µL phusion</TD><TD>20 µL</TD></TR><TR><TD class="mdl-data-table__cell--non-numeric">40U/µL Taq ligase</TD><TD>16 µL</TD></TR><TR><TD class="mdl-data-table__cell--non-numeric">ddH2O</TD><TD>843.4 µL</TD></TR></TBODY></TABLE>
            Store in -20℃ aliquot 15µL aliquot 15µL<TABLE class="mdl-data-table mdl-js-data-table"><THEAD><TR><TH class="mdl-data-table__cell--non-numeric">5×ISO buffer</TH><TH>6 mL</TH></TR></THEAD><TBODY><TR><TD class="mdl-data-table__cell--non-numeric">1M Tris-HCl pH=7.5</TD><TD>3 mL</TD></TR><TR><TD class="mdl-data-table__cell--non-numeric">2M MgCl2</TD><TD>150 µL</TD></TR><TR><TD class="mdl-data-table__cell--non-numeric">100mM dATP</TD><TD>60 µL</TD></TR><TR><TD class="mdl-data-table__cell--non-numeric">100mM dGTP</TD><TD>60 µL</TD></TR><TR><TD class="mdl-data-table__cell--non-numeric">100mM dCTP</TD><TD>60 µL</TD></TR><TR><TD class="mdl-data-table__cell--non-numeric">100mM dTTP</TD><TD>60 µL</TD></TR><TR><TD class="mdl-data-table__cell--non-numeric">1M DTT</TD><TD>300 µL</TD></TR><TR><TD class="mdl-data-table__cell--non-numeric">PEG-8000</TD><TD>1.5 g</TD></TR><TR><TD class="mdl-data-table__cell--non-numeric">100mM NAD</TD><TD>300 µL</TD></TR></TBODY></TABLE>
            Add ddH2O to 6mL, store in -20℃
            Aliquot 100µL<H2 id="p7">PCR using Q5 high-Fidelity DNA polymerase</H2><H3>Materials:</H3><TABLE class="mdl-data-table mdl-js-data-table"><THEAD><TR><TH class="mdl-data-table__cell--non-numeric"><STRONG>Component</STRONG></TH><TH><STRONG>50 μl Reaction</STRONG></TH></TR></THEAD><TBODY><TR><TD class="mdl-data-table__cell--non-numeric">5×Q5 Reaction Buffer</TD><TD>10 μl</TD></TR><TR><TD class="mdl-data-table__cell--non-numeric">10 mM dNTPs</TD><TD>1 μl</TD></TR><TR><TD class="mdl-data-table__cell--non-numeric">10 μM Forward Primer</TD><TD>2.5 μl</TD></TR><TR><TD class="mdl-data-table__cell--non-numeric">10 μM Reverse Primer</TD><TD>2.5 µL</TD></TR><TR><TD class="mdl-data-table__cell--non-numeric">100mM dCTP</TD><TD>60 µL</TD></TR><TR><TD class="mdl-data-table__cell--non-numeric">100mM dTTP</TD><TD>60 µL</TD></TR><TR><TD class="mdl-data-table__cell--non-numeric">Template DNA</TD><TD>Variable</TD></TR><TR><TD class="mdl-data-table__cell--non-numeric">Q5 high-Fidelity DNA polymerase</TD><TD>0.5 μl</TD></TR><TR><TD class="mdl-data-table__cell--non-numeric">5×Q5 High GC Enhancer (optional)</TD><TD>(10 μl)</TD></TR><TR><TD class="mdl-data-table__cell--non-numeric">ddH2O</TD><TD>to 50 μl</TD></TR></TBODY></TABLE><H3>Methods:</H3><OL start="1"><LI>In a PCR tube on ice, combine 1-10 ng of template DNA, 2.50 μl of 10 μM forward primer,
                    2.50 μl of 10 μM reverse primer, 10 μl of 5×Q5 Reaction Buffer, and sterile water to 50 μl.</LI><LI>Gently mix the reaction.</LI><LI>Collect all liquid to the bottom of the tube if necessary.</LI><LI>Transfer PCR tubes to a PCR machine and begin thermocycling.</LI></OL><H4>Thermocycling conditions for a Routine PCR:</H4><TABLE class="mdl-data-table mdl-js-data-table"><THEAD><TR><TH class="mdl-data-table__cell--non-numeric">Step</TH><TH>Temperature (℃)</TH><TH>Time</TH></TR></THEAD><TBODY><TR><TD class="mdl-data-table__cell--non-numeric">Initial denaturation</TD><TD>98</TD><TD>30 seconds</TD></TR><TR><TD class="mdl-data-table__cell--non-numeric">25-35 cycles</TD><TD>98 (denaturation)
                        50-72 (annealing)
                        72 (extension)</TD><TD>5-10 seconds
                        10-30 seconds
                        15-30 seconds</TD></TR><TR><TD class="mdl-data-table__cell--non-numeric">Final extension</TD><TD>72</TD><TD>2-5 minites</TD></TR><TR><TD class="mdl-data-table__cell--non-numeric">Hold</TD><TD>4</TD><TD>Indefinitely</TD></TR></TBODY></TABLE><H2 id="p8">Golden Gate Assembly</H2><H3>Material:</H3>
            10×T4 ligase buffer
            T4 Polynucleotide Kinase
            BsaⅠ/ BsmB Ⅰ
            T4 ligase
            Sterile water<H3>Methods:</H3><H4>Oligo Anealing</H4><TABLE class="mdl-data-table mdl-js-data-table"><THEAD><TR><TH class="mdl-data-table__cell--non-numeric">Component</TH><TH>50 μl Reaction</TH></TR></THEAD><TBODY><TR><TD class="mdl-data-table__cell--non-numeric">Forward Primer</TD><TD>5 μl</TD></TR><TR><TD class="mdl-data-table__cell--non-numeric">Reverse Primer</TD><TD>5μl</TD></TR><TR><TD class="mdl-data-table__cell--non-numeric">Sterile water</TD><TD>40 μl</TD></TR></TBODY></TABLE>
            Denaturation at 95 ℃ for 5 minutes, then hold at 4 ℃ indefinitely. Dilute annealed Oligo with sterile water after that
            <H4>Oligo phosphorylation</H4><TABLE class="mdl-data-table mdl-js-data-table"><THEAD><TR><TH class="mdl-data-table__cell--non-numeric">Component</TH><TH>20 μl Reaction</TH></TR></THEAD><TBODY><TR><TD class="mdl-data-table__cell--non-numeric">Oligo</TD><TD>6 μl</TD></TR><TR><TD class="mdl-data-table__cell--non-numeric">T4 ligase buffer</TD><TD>2 μl</TD></TR><TR><TD class="mdl-data-table__cell--non-numeric">T4 polynucleotide kinase</TD><TD>2 μl</TD></TR><TR><TD class="mdl-data-table__cell--non-numeric">Sterile water</TD><TD>10 μl</TD></TR></TBODY></TABLE><STRONG>Note:</STRONG>
            Use the diluted Oligo mentioned.
            React at 37 ℃ for 1 hour, then hold at 4 ℃ indefinitely.
            <H4>Golden Gate Assembly</H4><TABLE class="mdl-data-table mdl-js-data-table"><THEAD><TR><TH class="mdl-data-table__cell--non-numeric">Component</TH><TH>20 μl Reaction</TH></TR></THEAD><TBODY><TR><TD class="mdl-data-table__cell--non-numeric">Backbone</TD><TD>30 ng</TD></TR><TR><TD class="mdl-data-table__cell--non-numeric">Oligo</TD><TD>1 μl</TD></TR><TR><TD class="mdl-data-table__cell--non-numeric">T4 ligase buffer</TD><TD>2 μl</TD></TR><TR><TD class="mdl-data-table__cell--non-numeric">Bsa Ⅰ/BsmB Ⅰ</TD><TD>1 μl</TD></TR><TR><TD class="mdl-data-table__cell--non-numeric">T4 ligase</TD><TD>1 μl</TD></TR><TR><TD class="mdl-data-table__cell--non-numeric">Sterile water</TD><TD>to 20 μl</TD></TR></TBODY></TABLE><H4>Thermocycling conditions for a Routine</H4><TABLE class="mdl-data-table mdl-js-data-table"><THEAD><TR><TH class="mdl-data-table__cell--non-numeric">Step</TH><TH>Temperature (℃)</TH><TH>Time</TH></TR></THEAD><TBODY><TR><TD class="mdl-data-table__cell--non-numeric">10 cycles</TD><TD>37/42 (37 for BsaⅠ,42 for BsmB Ⅰ )
                        16 (assembly)
                    </TD><TD>5 minutes
                        10 minutes
                    </TD></TR><TR><TD class="mdl-data-table__cell--non-numeric">Final react</TD><TD>37/42
                        50
                        80
                    </TD><TD>15 minutes
                        5 minutes
                        5 minutes</TD></TR><TR><TD class="mdl-data-table__cell--non-numeric">Hold</TD><TD>4</TD><TD>Indefinitely</TD></TR></TBODY></TABLE><H4>Transfer the product immediately.</H4><H2 id="p9">Growing Overnight cultures</H2><H3>Materials:</H3>
            5ml LB broth
            5μl antibiotic
            Tips
            12 ml culture tube<H3>Methods:</H3>
            Overnight cultures are prepare under sterile conditions using a Bunsen burner<OL start="1"><LI>Add 5ml LB broth into 12 ml culture tubes</LI><LI>Add 5μl antibiotic into the broth</LI><LI>Pick a single colony by a sterile tip and inoculate the culture by dipping the tip into the LB broth.</LI><LI>Seal the tubes and incubate overnight at 37℃ shaking at 200-250rpm.</LI></OL><H2 id="p10">Colony PCR</H2><H3>Materials:</H3><TABLE class="mdl-data-table mdl-js-data-table"><THEAD><TR><TH class="mdl-data-table__cell--non-numeric">Component</TH><TH>25 μl Reaction</TH></TR></THEAD><TBODY><TR><TD class="mdl-data-table__cell--non-numeric">2×EasyTaq Mix</TD><TD>12.5 μl</TD></TR><TR><TD class="mdl-data-table__cell--non-numeric">Forward Primer</TD><TD>1 μl</TD></TR><TR><TD class="mdl-data-table__cell--non-numeric">Reverse Primer</TD><TD>1 μl</TD></TR><TR><TD class="mdl-data-table__cell--non-numeric">E. coli colony</TD><TD>1 </TD></TR><TR><TD class="mdl-data-table__cell--non-numeric">Sterile water</TD><TD>10.5 μl</TD></TR></TBODY></TABLE><H3>Methods:</H3><OL start="1"><LI>Mix the materials mentioned expect E. coli colony in PCR tubes</LI><LI>Pick 1 E. coli colony and add it into the reaction mixture</LI><LI>Gently mix the reaction</LI><LI>Transfer the PCR tubes to a PCR machine and begin thermocycling</LI></OL><H4>Thermocycling</H4><STRONG>The PCR machine should be set to run the following steps:</STRONG><TABLE class="mdl-data-table mdl-js-data-table"><THEAD><TR><TH class="mdl-data-table__cell--non-numeric">Step</TH><TH>Temperature (℃)</TH><TH>Time</TH></TR></THEAD><TBODY><TR><TD class="mdl-data-table__cell--non-numeric">Initial denaturation</TD><TD>98</TD><TD>30 seconds</TD></TR><TR><TD class="mdl-data-table__cell--non-numeric">25-35 cycles</TD><TD>98 (denaturation)
                        45-72 (annealing)
                        68 (extension)
                    </TD><TD>5-10 seconds
                        10-30 seconds
                        15-30 seconds
                    </TD></TR><TR><TD class="mdl-data-table__cell--non-numeric">Final extension</TD><TD>72</TD><TD>5-10 minites</TD></TR><TR><TD class="mdl-data-table__cell--non-numeric">Hold</TD><TD>4</TD><TD>Indefinitely</TD></TR></TBODY></TABLE><STRONG>Note:</STRONG>
            If loading on a gel, the EasyTaq Mix contains loading dye, so don’t add anything else.<H2 id="p11">Preparation of LB Broth, LB Ager, and M9 culture medium</H2><H2>LB Broth:</H2><H3>Materials:</H3><TABLE class="mdl-data-table mdl-js-data-table"><THEAD><TR><TH class="mdl-data-table__cell--non-numeric">Component</TH><TH>500 mL</TH></TR></THEAD><TBODY><TR><TD class="mdl-data-table__cell--non-numeric">Tryptone</TD><TD>5 g</TD></TR><TR><TD class="mdl-data-table__cell--non-numeric">Yeast Extract</TD><TD>2.5 g</TD></TR><TR><TD class="mdl-data-table__cell--non-numeric">NaCl</TD><TD>5 g</TD></TR><TR><TD class="mdl-data-table__cell--non-numeric">Sterile water</TD><TD>to 500 mL</TD></TR></TBODY></TABLE><H3>Methods:</H3><OL start="1"><LI>Mix the components listed above</LI><LI>Autoclave</LI></OL><H2>LB Ager</H2><H3>Materials:</H3><TABLE class="mdl-data-table mdl-js-data-table"><THEAD><TR><TH class="mdl-data-table__cell--non-numeric">Component</TH><TH>500 mL</TH></TR></THEAD><TBODY><TR><TD class="mdl-data-table__cell--non-numeric">Ager</TD><TD>7.5 g</TD></TR><TR><TD class="mdl-data-table__cell--non-numeric">Tryptone</TD><TD>5 g</TD></TR><TR><TD class="mdl-data-table__cell--non-numeric">Yeast Extract</TD><TD>2.5 g</TD></TR><TR><TD class="mdl-data-table__cell--non-numeric">NaCl</TD><TD>5 g</TD></TR><TR><TD class="mdl-data-table__cell--non-numeric">Sterile water</TD><TD>to 500 mL</TD></TR></TBODY></TABLE><H3>Methods:</H3><OL start="1"><LI>Mix the components listed above</LI><LI>Autoclave</LI></OL><H2>M9 culture medium</H2><H3>Materials:</H3><TABLE class="mdl-data-table mdl-js-data-table"><THEAD><TR><TH class="mdl-data-table__cell--non-numeric">Component</TH><TH>500 mL</TH></TR></THEAD><TBODY><TR><TD class="mdl-data-table__cell--non-numeric">5×M9 salt solution</TD><TD>100 mL</TD></TR><TR><TD class="mdl-data-table__cell--non-numeric">10% glucose</TD><TD>10 mL</TD></TR><TR><TD class="mdl-data-table__cell--non-numeric">1 M MgSO4</TD><TD>1 mL</TD></TR><TR><TD class="mdl-data-table__cell--non-numeric">1 M CaCl2</TD><TD>50 μl</TD></TR><TR><TD class="mdl-data-table__cell--non-numeric">10% Casein amino acid (optional)</TD><TD>10 ml</TD></TR><TR><TD class="mdl-data-table__cell--non-numeric">10 mg/ml vitamin B1 (optional)</TD><TD>17ml</TD></TR><TR><TD class="mdl-data-table__cell--non-numeric">Sterile water</TD><TD>to 500 mL</TD></TR></TBODY></TABLE><STRONG>Note:</STRONG>
            Add MgSO4 and CaCl2 last.
            <H3>Methods:</H3><OL start="1"><LI>Add 100ml 5×M9 salt solution and sterile water to about 480ml</LI><LI>Add 10ml 10% glucose and 1ml 1M MgSO4</LI><LI>Add 50μl 1M CaCl2, keep shaking the container at the same time</LI><LI>Treat with ultrasonator until no sediment observed</LI><LI>Filtrate with a 0.22μm filter</LI></OL><H2 id="p12">Making Competent cells</H2><H3>Materials:</H3>
            sterilized, stored at 4℃ before use
            <OL start="1"><LI>Several 5 mL liquid LB Medium.</LI><LI>500 ml Conical flask with 100 ml LB Medium (ensure sufficient oxygen for bacteria growing).</LI><LI>Several clean 50 ml centrifuge tubes with 2/3 sterilized water.</LI><LI>Enough 200 μl and 1 ml pipette tips.</LI><LI>Enough 1.5ml Eppendorf tubes.</LI><LI>Solution Ⅰ:：80 mM MgCl2 + 20 mM CaCl2, solute in sterilized water.</LI><LI>Solution Ⅱ: 100 mM CaCl2 + 10% glycerol (aq)</LI></OL><H3>Preparation:</H3><OL start="1"><LI>Pick a single colony from fresh plate, or take 50 μL of a fresh
                    overnight culture and transfer into 5 mL liquid LB medium, incubate at 37℃
                    (98.6℉) with vigorous shake overnight (less than 16 h). </LI><LI>Next morning, transfer 200~500 μL culture from Step 1. into 100 ml clean LB
                    medium and incubate 1.5~2 h till it is slightly turbid (the cell concentration
                    reaches around 5 × 107 cells/mL Do not over incubate) .</LI><LI>	In clean bench, transfer whole culture into precooled 50 mL centrifuge tubes,
                    then bath them in ice-water mixture for 20 minutes.</LI><LI>	Centrifuge cooled culture at 4℃(39.2℉) 4,000 rpm for 10 minutes to collect
                    cells, carefully pour off the medium in clean bench.</LI><LI>	Add 5 mL precooled Solution Ⅰfor every 50 mL centrifuge tube in Step 3. and
                    suspend the cells.</LI><LI>	Collect cells like Step 4. Then suspend cells from every 50 mL centrifuge tube
                    in Step 3. with precooled 2 mL Solution Ⅱ .</LI><LI>	Bath in ice-water for 4 h.</LI></OL>
            Aliquot 100 μL to every pre-iced 1.5 mL Eppendorf tube on ice-water, store at -80℃(-112℉)<H2 id="p13">Cell sorting for recombination efficiency characterization</H2><H3>Materials:</H3>
            Shaker
            Induced cells
            96 wells plate
            Flow cytometer
            PBS
            Media of choice<H3>Methods:</H3><H4>Cell preparation:</H4><OL start="1"><LI>E. coli strains are grown overnight in LB broth with appropriate antibiotics.</LI><LI>Transfer to M9 culture medium, which contains a certain concentration of inducer (IPTG/Ara).
                    Incubate cells and take samples at desired time points.</LI><LI>Dilute culture sample by 10 fold to 100 fold in PBS</LI></OL><H4>Flow cytometry</H4><OL start="1"><LI>Set up parameter on a 96 wells plate</LI><LI>Run wells</LI><LI>Use FlowJo to define the gates</LI></OL><H2 id="p14">Imaging Oscillation of Repressilator</H2><H3>Microfluidic master fabrication</H3>
            Fabrication of the E. coli mother machine was carried out using standard UV photolithography
            in a clean room environment. The device was designed using L Edit and quartz-chrome photomasks. <STRONG>Note:</STRONG> For all spin coater steps described below, the following shorthand notation is used: speed
            (rpm)/acceleration (rpm/sec)/time (sec).
            <STRONG>A. First Layer: Cell Channels</STRONG>
            This set of steps lays down the channels that house the cells in the final device. The tolerances for this
            layer are very stringent; the exposure dose and contact between mask and wafer must be optimized.
            We recommend trying a range of exposure parameters to ensure that a useful device is obtained.
            We also stress the importance of the very long post exposure bake time in the process below.<OL start="1"><LI>Place a new 3′′ Si wafer (we used 380 µm TEST grade wafers from University Wafer)
                    in a dish of fresh acetone. Sonicate at high power for 5 minutes.</LI><LI>Sequentially rinse the wafer with streams of methyl alcohol (MeOH), isopropyl alcohol (IPA) and
                    H2O (∼ 10 seconds per solvent).</LI><LI>Place wafer on 2′′ spin chuck and spin seconds at 500 rpm.</LI><LI>While spinning, sequentially rinse the wafer with streams of MeOH, IPA and H2O.</LI><LI>Spin wafer 1 minute at 3,000 rpm to dry. </LI><LI>Dehydrate wafer with smooth side upwards above clean filter paper for 5 minutes on a hot plate set
                    to 190◦C，then cool it at r.t..</LI><LI>Place the dehydrated wafer onto the spin coater chuck and dispense a small (cover ∼2/3 of the wafer
                    surface) amount of Su8 3005 photoresist (Microchem ) with a disposable pipette.
                    Run the spin program（Set spin program to: Step 1: 500/100/10, Step 2: 1400/300/36）.
                    This should result in a coat of ∼1.5 µm.</LI><LI>Soft bake wafer for 2 minute 95◦C.</LI><LI>Expose wafer for 40~50 seconds (3.7 mW/cm2) through cell channel mask in vacuum contact mode.</LI><LI>Post exposure bake the wafer (in order) for 2 minute at 95◦C（according to the thickness of this
                    layer），then cool it at r.t.</LI></OL><STRONG>B. Second Layer: Feeding Channels</STRONG>
            This layer of the device forms the medium flow channels. The dimensions of these features are not critical:
            we have used feeding channels of widely varying dimension to similar effect. The alignment is sensitive to
            large errors, however. The alignment between feeding channels and cell channels must be accurate
            (down to a couple of microns) in order to ensure that the cell channels are of the desired final length.<OL start="1"><LI>Set spin program to: Step 1: 500/100/10, Step 2:1200/300/40.</LI><LI>Place the wafer onto the spin coater chuck and dispense a small (cover ∼ 2/3 of the wafer surface)
                    amount of Su8 3005 photoresist with a pipette being careful not to introduce bubbles.
                    Run the spin program. This should result in a coat of ∼ 9 µm.</LI><LI>Soft bake the wafer (in order) for 8 minute at 95◦C.</LI><LI>With an Su8-developer-soaked swab, clean the newly-deposited photoresist off the alignment marks
                    to make them visible for the alignment process.</LI><LI>Align feeding channel mask to the alignment marks on the wafer. Apply vacuum contact and check
                    alignment again. If the vacuum application skewed the alignment, repeat the alignment process.</LI><LI>Expose wafer for 30 seconds (3.7 mW/cm2,) through aligned feeding channel mask.</LI><LI>Bake wafer for 10 minute at 95◦C.</LI><LI>Develop wafer for 2 minutes in Su8 Developer with mild agitation.</LI><LI>Rinse wafer for 10 seconds in IPA. Check to ensure that the development is finished. If undesired
                    photoresist remains, develop again for 20 seconds.</LI><LI>Verify channel height using a profilometer. The expected height is 10.5 µm. If the channel
                    dimensions lie outside of your expected tolerance bounds, the process must be repeated
                    with modified spin coating parameters.</LI></OL><H3>Chip preparation</H3>
            Dimethyl siloxane monomer (Sylgard 184 ) was mixed in a 10:1 ratio with curing agent, poured onto the
            silicon wafer, defoamed, degassed for 1 h and cured at 75 °C for 1 h. Individual chips were then cut the
            inlets and outlets were punched with a puncher (inner diameter : 0.9 mm, outer diameter : 1.3mm), and
            cleaned with Scotch tape. Bonding to water-cleaned and nitrogen-dried coverslips was ensured using oxygen
            plasma treatment (60 s at LOW and O2 pressure at 170 mTorr) on the day the experiments start. Attach treated
            surfaces of chip and coverslip together gently in case of cavity-clapse.The chips were then incubated at
            75 °C overnight to reinforce the bonding.<H3>Cell preparation</H3>
            Escherichia coli strains were grown overnight in LB with appropriate antibiotics and
            diluted 1:100 approximately 2–3 h before the beginning of the experiments in imaging
            media, consisting of M9 salts, 10% (v/v) LB, 0.2% (w/v) glucose, 2 mM MgSO4, 0.1 mM CaCl2,
            1.5 μM thiamine hydrochloride (Sigma Aldrich, included as a passivating agent).
            The cells were centrifuged on a holder that could fit into a standard table-top
            centrifuge at 4,000g for 10 min to insert them into the single straight channels.
            The feeding channels were connected to syringes filled with imaging media using
            Tygon tubing (VWR), and media was pumped using syringe pumps (New Era Pump System)
            initially at a high rate of 100 μl min−1 for 5 min, to clear the inlets and outlets.
            The media was then pumped at 2.5-5 μl min−1 for the duration of the experiment and cells
            were allowed to adapt to the device for several hours before imaging was started.<H3>IPTG synchronization </H3>
            To synchronize the phase of the oscillators in the population, we diluted the strains in imaging medium
            supplemented with appropriate antibiotics and 1 mM IPTG so that they would be early
            exponential (OD600 = 0.2) 8 h later (∼1 × 10−6) at 37 °C.<H3>Microscopy and image acquisition</H3>
            Images were acquired using a Nikon ECLIPSE Ti inverted microscope equipped with a temperature-controlled
            incubator, an Orca R2 CCD camera (Hamamatsu), a 60× Plan Apo oil objective (numerical
            aperture (NA) 1.4, Nikon), an automated xy-stage (Ludl) and Nikon HG Precentered Fiber
            Illuminator (INTENSILIGHT C-HGFIE). All experiments were performed at 37 °C. Typical
            exposure was low (50–100 ms) to reduce photobleaching, and the reporter channels were
            acquired using 2 × 2 binning (CCD chip dimension of 1,344 × 1,024 pixels, effective pixel
            size of 129 × 129 nm). Then 16-bit TIFF images were taken every 5–8 min, and focal drift
            was controlled via the Nikon PerfectFocus system, as well as a custom routine based on
            z-stack images of a sacrificial position (a position that was not used for further
            analysis). The following filter sets were used for acquisition: GFP (Nikon GFPHQ), RFP
            (Nikon TxRed), YFP (Nikon YFPHQ) and CFP (Nikon CFPHQ).<A class="mdl-button mdl-js-button mdl-button--raised mdl-js-ripple-effect" href="https://2017.igem.org/Team:Peking/Lab" style="background-color: #CB2C32; color: white;">
                Back &lt; Lab
        </A><A id="demo-menu-top-left" class="mdl-button mdl-button--accent mdl-js-button mdl-button--raised mdl-js-ripple-effect" style="background-color:#FD3C48; color: white;">
                Goto &gt; Lab/...
            </A><UL class="mdl-menu mdl-menu--top-left mdl-js-menu mdl-js-ripple-effect" data-mdl-for="demo-menu-top-left" style="margin-left: 0px"><A class="mdl-menu__item" href="https://2017.igem.org/Team:Peking/Notebook" style="color: #000;">Notebook</A><A class="mdl-menu__item" href="https://2017.igem.org/Team:Peking/Safety" style="color: #000;">Safety</A><A class="mdl-menu__item" href="https://2017.igem.org/Team:Peking/InterLab" style="color: #000;">InterLab</A></UL></DIV><DIV class="mdl-card__menu"><A class="mdl-button mdl-button--icon mdl-js-button mdl-js-ripple-effect"><I class="material-icons">opacity</I></A></DIV></DIV><FOOTER class="mdl-mega-footer" style="height: 260px; margin-bottom: 0px;"><DIV class="mdl-mega-footer__middle-section"><DIV class="mdl-mega-footer__drop-down-section"><H1 class="mdl-mega-footer__heading">Acknowledgement</H1></DIV><DIV class="mdl-mega-footer__drop-down-section"><H1 class="mdl-mega-footer__heading">Connection</H1><UL class="mdl-mega-footer__link-list" style="margin-left: 0px;&gt;
                    &lt;li&gt;&lt;a href=" https:=""><LI><A href="https://www.facebook.com/pekingigem">Facebook</A></LI><LI><A href="https://www.youtube.com/channel/UCefEYVCVqKJLmoJrSxivgZQ">Youtube</A></LI><LI><A href="https://twitter.com/iGEM_Peking">Twitter</A></LI><LI><A href="https://github.com/PekingiGEM2017">Github</A></LI></UL></DIV><DIV class="mdl-mega-footer__drop-down-section"><H1 class="mdl-mega-footer__heading">Contact</H1><UL class="mdl-mega-footer__link-list" style="margin-left: 0px;&gt;
                    &lt;li&gt;&lt;a href=" http:=""><LI><A href="http://www.synbiowiki.com">SynBioWiki.com</A></LI><LI><A href="mailto:PekingIGEM@gmail.com">Mail</A></LI></UL></DIV><DIV class="mdl-mega-footer__drop-down-section"><H2 style="color: white; "><STRONG>Peking iGEM</STRONG> 2017</H2><UL class="mdl-mega-footer__link-list" style="margin-left: 0px;"><LI style="color: white; line-height: 1.88em;">Peking University, No.5 Yiheyuan Road Haidian District, Beijing, P.R.China100871</LI></UL></DIV></DIV></FOOTER></MAIN></DIV></DIV></DIV></DIV></DIV></BODY></HTML>