Team:Peking/Project
Peking iGEM 2017
Home Project Modelling Software Hardware Lab Practices Parts Team
Introduction
Flip-flop
Controller
Clock
Demonstrate
Background
Cells are responsive to a myriad signals under most conditions[1]. Upon receiving this                         cornucopia of signals, i.e. inputs from the external and internal environment, cells must                         adjust their own internal mechanisms in response in order to survive. This adjustment depends                         not only on processing a combination of current environmental signal inputs[1], but also on                         determining the cell’s current state, which is a result of a series of past inputs. In                         dig-ital circuit theory, this operating mode is known as sequential logic[3]. Sequential logic is a type of logic in which output is a function of the present value                             of inputs and the system’s internal state.
The field of synthetic biology aims to design biological systems to perform tasks to better                         understand analogous natural systems and to develop direct applications for research and                         medicine. Nowadays, a wide va-riety of tasks can be performed by synthetically engineered                         genetic circuits, mostly constructed using combinational logic. The output of combinational                         logic (also referred to as time-independent logic) is a function of the present input only[4].                         In contrast to sequential logic, it has no internal state as an influence of the history of                         inputs. Without a memory of state, it is difficult to perform functions in a specific order,                         which has limited the widespread implementation of such systems[5].
The ability of sequential logic circuits to store modest amounts of information within living                         systems and to act upon them would enable new ap-proaches to the study and control of biological                         processes[6]. Indeed, the attention paid to this field has progressed far beyond scarce. For                         example, genetic toggle switches have proved to be effective in building synthetic memory                         modules[7]. In 2010, a bistable push-on push-off switch was constructed as a memory module by                         Lou et al[8][9]. Five years later, Pakpoom Subsoontorn et al. designed a toggle flip-flop switch                         whose state can be changed by the same signal[10]. Moreover, in 2012, Jerome Bonnet                         demon-strated                         a rewriteable recombinase-addressable data (RAD) module[11]. Re-cently, a state machine with two                         inputs and five states depending on the order of the input was realized in E. coli by Roquet,                         N. et al[12].
These studies indicate that more functions could be accomplished with a memory module embedded                         within the circuit[13]. Once the memory system is able to scale up to more states, a cell can be                         designed to do work that is more complex. In other words, we can reach a new dimensionality in                         designing synthetic life–time.
Introduction
This year, the Peking iGEM team is attempting to develop a frame of biological sequential                         circuits that are programmable. The envisioned circuit is capable of both storing states in                         DNA and automatically running a series of instructions in a specific order. More specifically,                         the sequential logic (or a real-state machine) that consists of a clock (trigger signal), flip                         flop (remembering device) and control unit (functional part) in bacteria. The clock is an                         oscillator with a repeated signal cycle that is utilized like a metronome to trigger actions                         of sequential logic circuits. Flip-flop is a memory device that can remember a state. Paired                         with a clock signal, it can realize state transition. The control unit is a functional part                         which can convert a signal from flip-flop into complex functions. With such a design,                         historical events are recorded and influence current cell behavior.
This work tries to point the way toward building large computational systems from modular                         biological parts—basic sequential computing devices in living cells, and ultimately, programming                         complex biological functions. Computers have thus become "alive". A unicellular organism itself                         cannot pack much computational power, but considered as a modular building block, its potential                         is impressive.
References
Hansen, J., & Benenson, Y. (2016). Synthetic biology of cell signaling. Natural                                 Computing, 15(1), 5-13.
Densmore, D., & Anderson, J. C. (2009, May). Combinational logic design in synthetic                                 biology. In Circuits and Systems, 2009. ISCAS 2009. IEEE International Symposium on (pp.                                 301-304). IEEE.
Cavanagh, Joseph (2006). Sequential Logic: Analysis and Synthesis. CRC Press. pp. ix.                                 ISBN 0849375649.
C.J. Savant, Jr.; Martin Roden; Gordon Carpenter. "Electronic Design: Circuits and                                 Systems". 1991. ISBN 0-8053-0285-9 p. 682
Lou, C. et al. Synthesizing a novel genetic sequential logic circuit: a push-on push-off                                 switch. Mol. Syst. Biol. 6, 350 (2010).
Zhang, H., Lin, M., Shi, H., Ji, W., Huang, L., Zhang, X., ... & Yang, Z. (2014).                                 Programming a Pavlovian-like conditioning circuit in Escherichia coli. Nature                                 communications, 5, 3102.
Subsoontorn P, Endy D (2012) Design and analysis of genetically encoded counters.                                 Procedia Comput Sci 11:43–54. doi:10.1016/j.procs.2012.09.006CrossRefGoogle Scholar
J. Bonnet, P. Subsoontorn, D. Endy, Rewritable digital data storage in live cells via                                 engineered control of recombination directionality. Proc. Natl. Acad. Sci. U.S.A. 109,                                 8884–8889 (2012). 10.1073/pnas.1202344109pmid:22615351 doi:10.1073/pnas.1202344109
Roquet, N., Soleimany, A.P., Ferris, A.C., Aaronson, S. & Lu, T.K. Synthetic                                 recombinase-based state machines in living cells. Science 353, aad8559 (2016).
Burrill, D. R., & Silver, P. A. (2010). Making cellular memories. Cell, 140(1), 13-18.
Back to top
star
Flip-flop
Background
Description
Key Achievements
Our Approach and Designs
Standard Characterization & Optimization Workflow
Construction of bio-flip-flops
Results
References
Background
The basic building blocks of sequential logic circuits are flip-flops, which are devices that                         can store information. A flip-flop has two stable states (see more on " state ")                         and its state                         can be changed by signals of arriving pulses. Thus, it and can store one bit of information                         and alternate its state following a trigger. It is the basic storage element in sequential                         logic. Click here to get a more specific                             description
When designing genetic-circuit counterparts of flip-flops, we should consider the following                         three criteria:
The device should have at least two stable states;
State changes are triggered by an identical signal;
The flip-flop should hold its state and remain unaltered until the next signal                                     arrives.
In a living cell, DNA is the natural medium for storing cell-state information and encoding                         functions. Recombinases, especially a subset called serine integrases and excisionases, are                         enzymes that can flip or excise specific fragments of DNA. They have been proved to be able to                         stably modify DNA sequences , which is the biological basis of our genetic flip-flop                         construct.
Large serine integrases reliably and irreversibly flip or excise unique fragments of DNA . DNA                         cleavage and re-ligation occur at the central crossover region at a pair of recombinase                         recognition sites (attB and attP), which allows the sequence to be flipped, excised, or inserted                         between recognition sites . After recombination, the original attB and attP sequences become                         reconstructed sequences - attL and attR. The resulting attL and attR sequences cannot be                         recognized and bound by integrases alone, so the state after integration is stable.
Another kind of recombinases, excisionases, are able to recognize and bind attL and attR                         sequences. With the help of excisionases, the state transitions become reversible.
Recombination Directionality Factors (RDF) are proteins which mediate reverse flipping to                         recover the sequence flipped by recombinases. By co-expressing the integrase with the                         corresponding RDF, or expressing the integrase-RDF fusion protein, recombination between the                         attL and attR sites can be induced. Thus, the flip-flop can be restored to the previous                         state.
Figure 1. Site-specific recombination either integrates, deletes or reverses a DNA                             sequence.
star
Aims
We would like to prove the feasibility of bio-flip-flops and optimize their function as parts.                         In order to do this, we have to:
Select effective recombinases and their corresponding RDFs; characterize and                                     optimize                                     their function.
Construct half of the flip-flop structure and express it in E. coli.
We would first search for known recombinase sequences in the literature, construct the                         corresponding vectors, and select the best ones to make standardized parts. After confirming                         their expression and function, we would optimize these devices through the standard workflow of                         vector and RBS substitution. We would also prove integrase-RDF fusion protein function. Furthermore, we would con-struct half of the bio-flip-flop structure and experimentally show                         that it can function.
This would serve as a proof of concept for a novel design strategy for bio-flip-flops.
Key Achievements
Test and optimize recombinase and recombinase-RDF fusion protein function.
Standardization into BioBrick devices.
Introduction of the forward latch of the bio-flip-flop into cells and proof of                                     function                                     in vivo.
Our Approach and Designs
Design of the bio-flip-flop
Using a pair of integrases and their corresponding excisionases, we designed the bio-flip-flop                         device to store state information as follows:
The device has two promoters, controlled by input inducers X and Y, respectively.                                     The                                     input can either promote or inhibit transcription.
In the schematic mechanism figure below, recombination sites with colors                                     corresponding                                     to their binding integrases are placed on both sides of the promoter in inverse                                     directions. Thus, integrases can flip the promoters, changing the direction of                                     transcription.
We define the combination of the input series X, and then Y as a single signal.                                     Thus,                                     the bio-flip-flop can change the circuit state each time a signal arrives, and keep                                     this                                     state until the next signal pulse.
Figure 2. Schematic diagram of the bio-flip-flop mechanism.
Selection of Available Integrases
We first did a literature search and sequence alignment to mine the databases for available                         integrases for device development. Here we list the names and sources of the collected                         recombinases.
We finally chose to use TP901-1, Bxb1 and phiC31. These are well-known serine integrases, and                         all of them have been used to demonstrate static-input logic gates. Some have cofactors                         (equivalent to RDFs) that can reverse directionality. Previous work has shown that there exists                         a new set of 11 orthogonal integrases, greatly expanding the availability of integrases that can                         be formed into standardized parts. We hope that given enough time and resources, it is possible                         to construct more parts using our standard workflow described below.
Standard Characterization & Optimization Workflow
The viability of a bio-flip-flop relies on the performance of two integrases and their                         corresponding excisionases. To select integrases for the bio-flip-flop, we constructed                         expression vectors for different recombinases and tested their performance individually. Here                         we introduce a standard workflow to accomplish this task.
Figure 3. The workflow of bio-flip-flop optimization.
Different replication origins affect the copy number of plasmids, and it is highly probable that                         they also influence the expression levels of their encoded recombinases. To select integrases                         for the bio-flip-flop, we constructed expression vectors for different recombinases and tested                         their performance individually. This was carried out by co-transformation of E. coli Top10 with the expression vector and a reporter plasmid. The reporter plasmid expresses differen                         t fluorescent proteins before and after flipping of a constitutive promoter.
Figure 4. The standard genetic structure used to characterize the recombinases.
According to our design of the bio-flip-flop, the two inputs X and Y comprise a signal.                         Consequently, two distinct induction systems need to be constructed. For this reason,                         recombinases were cloned onto two different backbones, under regulation of differently induced                         promoters. The expression vectors were constructed by Gibson assembly. Plasmid type 1 is cloned                         from the repressor generator (RPG), which has a p15A replication origin, ampicillin resistance                         gene, lacI, an IPTG-induced pTac promoter, and a RiboJ insulator after pTac. The backbone of                         plasmid type 2 was cloned from pIntegrase_13, which was a gift from Christopher Voigt                         (Addgene plasmid # 60584). It has a ColE1 replication origin, kanamycin resistance gene, araC,                         and an arabinose-induced pBad promoter. RiboJ was added to Plasmid2 by cloning RiboJ and the                         recombinase from Plasmid1, and assembled after pBad.
Figure 5. Integrase expression vectors with different replication origins. The                         one shown on top has a relaxed ColE1 replication origin and recombinase expression is induced by                         arabinose via a pBad induction system. The one shown in the bottom picture has a relaxed p15A                         replication origin and recombinase expression is induced by IPTG.
For a given backbone and inducible promoter, RBS strength affects the expression level                         post-transcription. For this reason, we tested a series of RBS for different recombinases in                         different backbones.
In order to tune the RBS, an intermediate vector with a 20bp random sequence (45% GC), flanked                         by a pair of BsmbI recognition sites (introduced by PCR) was prepared for each recombinase.                         Vector construction was completed by adding RBS via Golden Gate assembly. The RBS calculator                         was used to generate RBS sequences with a wide range of translation initiation rates. Other                         sequences (B0029, B0030, B0032, B0034, B0035) were obtained from the iGEM website. The RBS                         oligos were obtained from the annealing of two complementary primers. Expression vectors for                         each recom-binase with different RBS were tested by flow cytometry after co-transformation with                         the reporter vec-tor. For each recombinase, an RBS correlating with low leakage (small RFP                         subset in the cultures with no inducer) and high recombination efficiency after induction was                         selected.
Measurement of Recombination Efficiency
We used a microplate reader to roughly measure the efficiency of the selected integrases. We                         used flow cytometry to conduct a more accurate characterization.
Microplate Readers
Microplate readers are instruments used to detect biological, chemical or physical events in                         samples in microtiter plates. We used a microplate reader to detect optical density and                         fluorescence intensity follwing the procedure as below:
Pick and inoculate single colony into 1ml of LB media with antibiotics in a V-bottom                                 96-well plate. Grow the culture overnight (about 12 hours) at 37°C, 220 rpm in a Thermo                                 VARIOSKAN FLASH shaker.
Then, an aliquot comprising 2 μL of the culture was transferred into 1ml LB media with                                 antibiotics and inducer (1mM IPTG or 10mM arabinose). Culture overnight (about 12 hours)                                 at 37°C and 220 rpm.
Aliquot 1 μl of the cultures in the deep-well plates, then copy them to the 96-well                                 plate filled with 200μl PBS. Don’t forget to leave at least 12 wells with only PBS in                                 it as your negative conctrol.
Measure the samples (OD and Fluorescence). The former represents the density of                                 bacteria,                                 and the latter implies the efficiency of recombination. GFP/OD is calculated to measure                                 recombination efficiency.
Flow Cytometry
For more detailed measurements,flow cytometry were used to evaluate the recombination                         efficiency.                         Our procedures are as follows:
Single colonies were picked and used to inoculate into 1ml of LB media with antibiotics                                 in a V-bottom 96-well plate.
The cultures were grown at 37°C and 1000 RPM for 12h. Subsequently, an aliquot                                 comprising 2 μL of the culture was transferred into 1ml of M9 glucose media with                                 antibiotics and inducer (1mM IPTG or 10mM arabinose for RBS tuning, gradient                                 concentration for transfer curve) in a V-bottom 96-well plate.
The cultures were grown at 37°C and 1000 RPM for 15h. An aliquot comprising 2μL of the                                 culture was transferred into 198 μL of phosphate buffered saline (PBS) containing 2                                 mg/mL kanamycin in a 96-well plate. This mixture was incubated for one hour at room                                 temperature                                 before testing. Two lasers were used to excite GFP and RFP simultaneously. Single-cell                                 fluorescence distribution at both emission wavelengths was recorded.
The counted cells were gated to eliminate the population which showed no fluorescence. The                         remaining cells were divided into two subsets by a diagonal: RFP sub-set and GFP subset. The                         recombination efficiency was estimated from the proportion of the RFP subset in the total                         fluorescent population.
Figure 6. A example of Gating the RFP and GFP subsets. Change of fluorescence after                             induction was seen. Left: no inducer. Right: 10 mM Ara for 15h. Change of                         fluorescence after induction was seen. Left: no inducer. Right: 10 mM Ara for 15h.
For the vector with ColE1 replication origin, we found proper RBS in a list of calculated RBSs                         for TP901-1.
We observed a shift in fluorescence level and wavelength in several trails, which indicated that                         recombinase can flip the promoter region and alter gene expression in our design. However,                         problems such as leakage and insufficient recombination efficiency were obvious. Tuning of our expression system seemed necessary before assembly into the bio-flip-flop device.
Characterization of Excisionases
To make sure that the sequence-reversing process is efficient and complete, we used two                         strategies: one encompasses combining the integrase and its corresponding RDF into a                         polycistronic structure, and the other is to construct fusion proteins comprising the integrases                         and RDFs in one polypeptide chain. We have been trying to improve the recombination efficiency                         by replacing the expression vec-tor with different replication origins or inducible promoters                         and changing the RBS sequence before the coding sequence.
Figure 7. The two strategies used to characterize integrases and their RDFs. UP. The polycistronic structure containing the integrase and its corresponding RDF. Bottom. The                         fusion                         protein structure of integrase with its corresponding RDF.
The method is similar to the integrase test method. We constructed an input plasmid to express                         integrases and RDFs, and an output plasmid on which the promoter J23119 is located between the                         attL and attR sequences. The input plasmid expresses GFP before induction of integrase and RDF,                         and if an inversion occurs after induction, the output plasmid can express RFP. By comparing the                         changes in fluorescence types and intensities before and after induction, the efficiency of                         inversion can be estimated.
Figure 8.The standard genetic structure used to characterize the                             recombinases.
Construction of Bio-flip-flops
After having already optimized the expression systems of integrases and excisionases, we set out                         to construct a Bio-Flip-Flop. Since it is unknown whether the Bio-Flip-Flop is practical, we                         considered developing two hierarchical execution units supporting our Bio-Flip-Flop. We called                         these two units the forward latch and the backward latch. The state transition process of each                         unit consists of two stable and irreversible states. Nevertheless, the state transition proces                         s of both units consists of four stable and cyclic states. Such units support the verification                         of the feasibility of our Bio-Flip-Flop and further allow us to construct a complete and                         functional Bio-Flip-Flop.
Forward Latch
The structure of the forward latch consists of two plasmids with different induction systems of                         integrases (Figure 9). The state transition process of the forward latch consists of two stable                         and irreversible states (Figure 10).
Figure 9. The structure of the forward latch. Two plasmids constitute the                         forward latch. Each plasmid contains an integrase gene (Bxb1-gp35 or TP901) and a reporter gene                         (GFP or mRFP). To standardize the input, we used the pBAD promoter to regulate the transcription                         of Bxb1 and GFP, and the pTAC promoter to regulate the transcription of TP901 and mRFP.                         Moreover,                         each integrase recognizes and converts the attB and attP sites on the other plasmid.
Figure 10. The state transition process of the Forward latch. After induction                         with arabinose, the expression of integrase Bxb1 and GFP is initiated. The integrase Bxb1                         recognizes and converts the attB and attP sites flanking the pTAC promoter, which leads to a                         change of orientation of the pTAC promoter. If we input IPTG next, the expression of integrase                         TP901 and mRFP will begin. The orientation of the pBAD promoter will change. As the GFP is                         degraded and mRFP is produced, the ratio of mRFP/GFP fluorescence rises.
Backward Latch
The structure of the Backward latch consists of two plasmids with different induction systems                         driving excisionases (Figure 11). The state transition process of the Backward latch consists                         of two stable and irreversible states (Figure 12). Because excisionases recognize attL and attR                         sites and convert them into attB and attP sites, the backward latch is capable of resetting the                         state of the Forward latch.
Figure 11. The structure of the Backward latch. Two plasmids constitute the                         Backward latch. Each plasmid contains an excisionase gene (Bxb1-gp47 or TP901) and a reporter                         gene (GFP or mRFP). To standardize the input, we used the pBAD promoter to regulate the                         transcription of Bxb1 and GFP, and the pTAC promoter to control the transcription of TP901 and                         mRFP. Moreover, each excisionase recognizes and converts the attL and attR sites on the other                         plasmid.
Figure 12. The state transition process of the Backward latch. After induction                         with arabinose, the expression of excisionase Bxb1 and GFP is initiated. The excisionase Bxb1                         recognizes and converts the attL and attR sites flanking the pTAC promoter, which leads to a                         change of the orientation of the pTAC promoter. If we input IPTG next, the expression of                         excisionase TP901 and mRFP will begin. The ori-entation of the pBAD promoter will change. As GFP                         is degraded and mRFP is produced, the ratio of mRFP/GFP fluorescence rises.
Experiment
Because our final goal is to realize the state transition of our Bio-Flip-Flop, it is                         significant                         to characterize the efficiency of the primary state transition of the forward latch. We                         quantified the amounts of GFP and mRFP and considered the ratio of mRFP/GFP fluorescence as a                         direct measure of the efficiency of the state transition.
For all characterization experiments, co-transformed cells were first grown and subcultured                         overnight, after which we induced the cells with arabinose and IPTG in that order. Finally, we                         quantified the fluorescence strength using a flow cytometer.
Results
Primary Proof of Recombination
To prove the function of our integrase-RDF fusion protein in an illustrative way, IPTG was                         applied to the petri dish                         where E. coli Top10 co-transferred with expression and reporter plasmid were cultured. Compared                         to the RFP control (transferred with RFP expressing reporter plasmid only), green fluorescence                         was obvious when excited by visible light. Half red, half green colonies were seen, indicate                         incomplete recombination.
Figure 13. (left) RFP control. (middle) expression vector of TP901-RDF fusion                         protein co-transferred with                         reporter. White arrows: colonies with green fluorescence indicates recombination. (right) A                         example of half red, half green colonies, indicating incomplete recombination.
In a similar trail, Arabinose was applied to the petri dish where E. coli Top10 co-transferred                         with TP901-1 integrase expression and reporter plasmid were cultured. Compared to the GFP                         control (transferred with GFP expressing reporter plasmid and expression vector, but no inducer                         was added), We can see a transition of colony color from green (GFP expressin) to red (RFP                         expression). When excited by visible light, it’s distinguishable that the colonies nearly                         completely cease to express GFP. The expression of RFP and shut down of GFP expression shows                         that recombination has happened.
Figure 14. (Up, left) GFP control, colonies in green. (Up, right) GFP control,                         green                         fluorescence can be seen when excited. (Bottom, left) Expression vector of TP901-1 integrase                         co-transferred with reporter, red color instead of green indicates recombination. (Bottom,                         right) Expression vector of TP901-1 integrase co-transferred with reporter, green fluorescence                         is nearly completely shut down, indicating desirable recombination efficiency.
Fine-tuning of Recombination Efficiency
Selection of RBS for different replication origin
For the vector with ColE1 replication origin, we found proper RBS in a list of                         calculated RBSs for TP901-1 and Bxb1 gp35.
Figure 15. Bxb1 gp35 recombination efficiency with a variety of calculated RBS and RBS                             sequences from iGEM (B0030∼B0035).
T.I.R = Translation Initiation Rate
Figure 16. TP901-1 recombination efficiency with a variety of calculated RBS. A. At the vector with p15A.
B. At the vector with colE1.
T.I.R = Translation                         Initiation Rate
For the expression vector with p15A replication origin, a proper RBS for TP901-1 was                         selected.
Figure 16_b. TP901-1 recombination efficiency with a variety of RBS from iGEM (B0030∼B0035).
T.I.R = Translation Initiation Rate
Transfer curves
We evaluated the performance of phiC31 and Bxb1 gp35 on ColE1 expression vectors under a series                         of inducer concentration gradients. The results enabled us to determine the appropriate inducer                         concentration.
A
B
Figure 17. The transfer curves of the recombinases we utilized. A. The transfer                         curve of integrase phiC31. B. The transfer curve of integrase Bxb1.
Measurement of the Flipping Efficiency of Excisionases
We have tested the flipping efficiency of our Integrase-RDF fusion proteins with a group pf RBS.                         Flipping performance varies with different RBS. For TP901-RDF fusion proteins, recombination                         seems not obvious in some cases (RBS B0032, B0034, B0035, phiC 10000, Bxb 1000, int2 3000),                         while in others both flipping efficiency and leakage and significant.
Figure 18. pTac TP901-RDF fusion protein flipping efficiency with ten RBS. Proportion of GFP subset indicates flipping efficiency.
A second test was supplemented. Similar results support that our Integrase-RDF fusion proteins                         are able to flip the sequence between attL and attR site of the reporter, but much improvement                         are needed to reduce the leakage as well as increase the efficiency.
Figure 19. pTac TP901-RDF fusion protein flipping efficiency with two RBS. Proportion of GFP subset indicates flipping efficiency.
Integrase-RDF for Bxb1 and phiC31 were also constructed, tested. High flipping efficiency and                         leakage were seen.
Figure 20. Flipping efficiency for pTac integrase-RDF fusion protein of phiC31 and                             Bxb1. Proportion of GFP subset indicates flipping efficiency.
To conclude, we found that the Integrase-RDF fusion proteins are able to flip the sequence                         between attL and attR site (indicated by observed GFP subset), although tuning work remains to                         be done to construct a controllable, functional bio-flip-flop device.
Reference
Yang, L., Nielsen, A. A., Fernandez-Rodriguez, J., McClune, C. J., Laub, M. T., Lu, T.                                 K., & Voigt, C. A. (2014). Permanent genetic memory with> 1-byte capacity. Nature methods, 11(12), 1261-1266.
Bonnet, J., Yin, P., Ortiz, M. E., Subsoontorn, P., & Endy, D. (2013). Amplifying                                 genetic logic gates. Science, 340(6132), 599-603.
Baker, T. A., Bell, S. P., Gann, A., Levine, M., & Losick, R. (1970). Molecular biology                                 of the gene.
Olorunniji, F. J., McPherson, A. L., Rosser, S. J., Smith, M. C., Colloms, S. D., & Stark, W. M. (2017). Control of serine integrase recombination directionality by fusion                                 with the directionality factor. Nucleic acids research, 45(14), 8635-8645.
Siuti, P., Yazbek, J., & Lu, T. K. (2013). Synthetic circuits integrating logic and                                 memory in living cells. Nature biotechnology, 31(5), 448-452.
Khaleel, T., Younger, E., McEwan, A. R., Varghese, A. S., & Smith, M. (2011). A phage                                 protein that binds φC31 integrase to switch its directionality. Molecular microbiology,                                 80(6), 1450-1463.
Salis, H. M., Mirsky, E. A., & Voigt, C. A. (2009). Automated design of synthetic                                 ribosome binding sites to control protein expression. Nature biotechnology, 27(10),                                 946-950.
Espah Borujeni, A., Channarasappa, A. S., & Salis, H. M. (2013). Translation rate is                                 controlled by coupled trade-offs between site accessibility, selective RNA unfolding and sliding at upstream standby sites. Nucleic acids research, 42(4), 2646-2659.
T Flip-Flop
Basically, a real flip flop has two inputs. One input is a control input. For a T flip flop, the                         control input is labelled T. The other input is the clock. Positive edge-triggered flip flops                         can only change output values when the clock is at a positive edge. While negative edge                         triggered flip flops change on a negative edge, and level-triggered flip flops, that change only                         when the value is 1. When the clock is not at a positive edge, then the output value is held.                         That is, it does not change. A flip flop’s output is the bit it stored. Thus, the flip flop is                         always outputting the one bit of information.
Here's the characteristic table for a T flip flop.
T
Q
Q+
Operation
0
0
0
Hold
0
1
1
Hold
1
0
1
Toggle
1
1
0
Tohgle
The T flip flop characteristic table has 3 columns. The first column is the value of T, a                         control input. The second column is the current state, that is the current value being output by                         Q. The third column is the next state, that is, the value of Q at the next positive edge. It's                         labelled with Q and the superscript, + (the plus sign).
The T flip flop has two possible values. When T = 0, the flip flop does a hold. A hold means                         that the output, Q is kept the same. When T = 1, the flip flop does a toggle, which means the                         output Q is negated. Thus, in a T flip flop, you can either maintain the current state's value                         for another cycle, or you can toggle the value (negate it).
Why isn't it XOR?
First, XOR has two inputs, and one output. A T flip flop essentially has a control input and a control output. Second, the second column and the third column are really the same output, but                         at different points in time.
Difference between Flip flop and gate
The biggest difference between a flip flop and a gate is that a flip flop can hold its value.                         Even though holding a value is something very simple, it makes it different from logic gates,                         and allows us to design circuits that have cycles in them (i.e., feedback).
State
There is a generic definition of state:
State is a quantity that stores the previous history of what's happened in such a way that                             you can predict the output, given any future input.
From the definition, we can get two things. First, state refers to some recorded information. We                         can consider state as a kind of memory. For sequential logic circuits, this is some finite                         number of bits. Second, there is always a notion of time when we talk about state. Usually, we                         think of time being involved with state, because it may change over time.
Let’s think of an object, it starts off in an initial state, and over time, that state changes.                         For example, if the object is a piece of paper, when we are drawing we may draw or erase                         something (add or remove elements) over time. Our picture’s state may not keep track of all the                         things that have happened to it, nevertheless, the picture changes from state to state, as                         various operations are performing object.
Learn More
Back to top
Controller
Background
Aim
Key Achievements
Our Approach and Designs
Results
References
Background
In our design, after having designed the flip flop, the device can remember the information of                         its state, the next step is to transform the state into an actual function. To achieve this                         transformation, we first needed a “reader” to read out the current state. At this point the                         control unit comes into play. A control unit is a DNA sequence with recombinase sites whose                         expression is controlled by recom-binase and RDF by reversing or deleting a promoter and/or a                         terminator. To make the control unit reliable and predictable, we first need to be able to                         predict the behaviors of its “building blocks” (or “elements” in electrical engineering), from                         which we weave our engineer’s perspective into the biolog-ical system. However, we need to make                         some adaptations and adjustments to these “elements” to make them usable.
star
Aim
To characterize a group of terminators and measure their strength in both                                     orientations.
To select unidirectional terminators that are able to serve as switches in genetic                                     circuits.
To measure the strength of terminators flanked by one set of attB/P sites, and the                                     inversion efficiency of recombinases.
To measure the terminator strength of terminators flanked by two sets of attB/P                                     sites,                                     and the inversion efficiency of different recombinases.
To construct a 7-segment display which is able to count from 1 to 3.
Key Achievements
In our project, after we have made these constructive modifications to terminators                                     and                                     recombinases to meet our requirements of robust, reliable and predictable behaviors,                                     which will be presented in the form of a seven-segment display. This will improve                                     our                                     quantitative understanding of natural phenomena and foster an engineering discipline                                     for obtaining new complex cell behaviors.
We measured the strengths of a total of 24 terminators in both directions, and we                                     elected 6 suitable unidirectional terminators.
We measured the strengths of several terminators paired with attB/P sites and the                                     inversion efficiency of 3 recombinases.
We succeeded to construct the transformation of the pattern from ”2017” to ”PKU”.
Our Approach and Designs
Characterization of Terminators
Terminators are responsible for disengaging the RNA polymerase from the DNA and stopping gene                         transcription in vivo. This characteristic makes terminators desirable as switches in synthetic                         genetic circuits. The terminators that are used to realize this function should be                         unidirectional.                         That is, they should be able to block the movement of RNA polymerase effectively in one                         orientation and hardly influence its movement in the other. With this “reader”, advantages                         include a shorter sequence, per-manently recorded message in the DNA and greater number of                         design plans. Consequently, it is es-sential that we characterize the different terminators.                         We first chose 24 terminators whose forward orientation strength is high, as previously                         reported.                         We then measured their strength in both the forward and reverse orientations in our testing                         system.                         Finally, we sifted out a set of six reliable unidirectional terminators without potential                         cryptic promoters in both orientations.
Given that the same part can perform differently in different genetic circuits, previous testing                         systems and results of terminators may not fit our project. We therefore constructed a rather                         simple but practical reporter plasmid (as shown below), based on which terminator strength can                         be measured via fluorescence intensity and simple calculations.
Vector construction
Figure 1. Vector construction to measure the terminator strength
As shown above (Figure 1.), the circuit used to measure terminator strength was constructed on                         the pSB4C5 vector. The terminator to be tested is located downstream of the Ptac promoter and                         followed successively by RiboJ, RBS, and sfGFP. To enable high throughput, we introduced a pair                         of BsaI cutting sites between Ptac and RiboJ so that Golden Gate assembly can be used to add                         different terminators efficiently. In addition, there is a random sequence (from a previous                         associated research, the author of which had designed it using the Sequence Manipulation Suite)                         between the two Bsal sites flanked by two RiboJ, which was constructed as a positive                         control.
The terminators to be tested were selected from the literature.
RiboJ acts as an insulator between the RBS and its upstream sequence in the genetic circuit. The                         insulating function of RiboJ is directly associated with its biochemical function. RiboJ can be                         defined as the DNA sequence containing a ribozyme called sTRSV- ribozyme and a 23-nt hairpin.                         The downstream hairpin helps to expose the RBS. mRNA can be cleaved by sTRSV autocatalytically                         at a defined residue. As a result, extraneous RNA leaders that arise from transcription from                         promoters with different start sites will be cleaved. Moreover, to make sure the sequence                         upstream of the RBS will not affect RiboJ’s efficiency, the transcripts of our genetic circuits                         were designed to have the same 5’-UTR sequence, to avoid the effect called “part junction                         interference”, in which neighboring sequences may alter the part’s character. For example, a                         barcode part was found to contain a sequence similar to the –10 region of a constitutive                         promoter.
Flow Cytometry
Vectors constructed with different terminators in both directions were tested via flow                         cytometry.
E. coli strains were grown for 12h in LB media containing antibiotics in culture tubes                         at 37°C and 250 rpm in an incubator. After 12h, the cultures were diluted 1:100 with M9 medium                         with antibiotics in 96-deep-well plates and grown for 2h at 37°C and 1150rpm in a shaker. Next,                         the cultures were diluted 1:100 with fresh M9 medium with antibiotics and different                         concentrations of IPTG. These cultures were grown for 6 h and then diluted 1:5 with                         phosphate-buffered saline (PBS) containing 2 mg/ml kanamycin or 35 ug/ml chloramphenicol to                         arrest protein production, and fluorescence was measured using a flow cytometry.
Cells were analyzed by flow cytometry using a BD biosciences Fortessa flow cytometer with blue                         (488-nm) laser. An injection volume of 15 μl and a flow rate of 1 μl/s were used. Cytometry data                         were analyzed using Flowjo. The gated populations consisted of at least 10 000 cells. The median                         fluorescence of the gated populations was calculated using Fowjo and used for all reporting.                         Autofluorescence of white cells was subtracted from all fluorescence measurements.
The Mathematical Formula
As shown below, a mathematical formula was also used to characterize the strength of the                         terminators. ＂Ts＂ is the abbreviation of ＂terminator strength＂. [GFP]random sequence denotes the fluorescence strength of the plasmid with the random sequence.                         [GFP]terminator denotes the fluorescence strength of the plasmid with a                         terminator.
Ideal unidirectional terminators should fit the following standards:
The Ts of both directions are larger than 0.9 (i.e. there is no cryptic promoter                                 potential)
The Ts ratios (Ts of the forward direction/Ts of the reverse direction) are larger than                                 10
The larger Ts between the forward and reverse directions has to be larger than 10 (which                                 ensures the blocking strength to be high enough)
The smaller Ts between the forward and reverse directions has to be no larger than 1.2                                 (i.e. terminators in reverse orientation no longer block the transcription process).
To remeasure the Ts of the unidirectional terminators
After the preliminary sifting of the terminators, a set of desirable unidirectional terminators were sifted out and re-measured using the plate reader to refine the parameters.
TOP10 cells were co-transformed with the system containing the terminator and one pair of attB/P                         sites and the expressing system. Cells then were cultured overnight on LB plates.
Colonies were picked and cultured in liquid media with antibiotics and inducer. The bacterial                         liquid culture was then diluted at the ratio of 1:100 into M9 media with both antibiotics and                         inducer into 96-well plate, cultured for 12-24 hours in the incubator. After the induction,                         OD600 and fluorescence were measured using the plate reader. Terminator strength was then                         calculated according to the formula above.
Characterization of Inversion Efficiency
After finishing these measurements, the main protagonist of our project, the recombinase, will                         come into play. When terminators are flanked by a pair of attB/P sites, the synthetic circuit                         will transit between on and off state, which is achieved by controlling the expression of the                         respective recombinases. Here the question becomes whether the attB/P sites flanking the                         terminators may alter their strength due to some unknown interaction. Thus, we needed to                         remeasure their strength. Additionally, the inversion efficiency of respective recombinases                         need to be characterized.
We introduced the testing system and expression system of the corresponding recombinases into                         the same cell to see if inversions happened, and if leaky expression of recombinases would                         impact the system. In the end, couples of terminators and recombinases with near-complete                         inversions and minimal leakage were selected.
Vector construction
Figure 2. Terminator flanked by a pair of attB/P sites
As shown above, we selected terminators for both directions that were sandwiched between                         attB/attP recognition sites, on the basis of the aforementioned terminator strength testing                         system. The constructed terminators are shown in the table below.
Recombinase Recognition Site                                         (attB/attP)
Terminators
TP901-1(AG)
855 F/R,836 F/R,435 F/R,309R,221 F/R
TP901-1(TC)
836 F/R,435 F/R,322R
Bxb1 (GT)
836 F/R,435 F/R,322F/R
Int2
855 F/R,836 F,435 FR,322 R
ΦC31
855 F/R,836 F/R,435 F/R,309 R,221 FR
Those letters in pairs inside the brackets stand for different two-base core inside the                             sequence of recombinase recognition sites, which remained unchanged after                             inversion. The expression system includes the recombinase Φc31, Bxb1, TP901-1, or int2 under the control of                         the inducible pBAD promoter and optimal RBS, which can simultaneously ensure high efficiency and                         low leakage.
As a part working in genetic circuits, the terminator flanked with a pair of att B/P sites is a                         brand new terminator in a sense. It’s necessary for us developers to measure the Ts of the new                         terminator.
Remeasurement of Ts of the new terminators
The plate reader is which we use to remeasure. The protocol and the formula are same as the page                         above.
Characterization of inversion efficiency
Activation and induction Escherichia coli colonies were picked and grown for 12h in LB media containing                         antibiotics in culture tubes at 37°C and 250 rpm in an incubator. Next, the bacterial cultures                         were diluted at a ratio of 1:100 with fresh M9 medium with antibiotics and inducers, after which                         the bacteria were grown for another 12h.
Plate reader Fluorescence was measured using Varioskan© Flash. We measured the OD600 and green fluorescence                         (the excitation wave length is 485nm and the emission wave length is 510nm). The ratio of                         fluorescence intensity/OD600 was calculated using the net fluorescence intensity and net OD600.                         The transcription barrier strength of an attB/P-Terminator can be characterized quantitatively                         using the Ts formula.
The inversion efficiency was calculated according to the below formula.
Fterstands for the fluorescence intensity of the system containing both the                         terminator and the recombinase elements.
Fcontrol stands for the fluorescence intensity of the system containing only the                         terminator element.
Fforward stands for the fluorescence intensity of the system containing both the                         forward terminator and the recombinase elements.
Freverse stands for the fluorescence intensity of the system containing both the                         reverse terminator and the recombinase elements.
The difference between the fluorescence intensity of the system expressing a recombinase and the                         system not with recombinase expression, divided by the difference between the fluorescence                         intensity of the system containing the terminator in the forward direction and the system                         containing the terminator in the reverse direction.
Characterization of Terminator Flanked with Two Pairs of attBP Sites
Vector Construction
Figure 3. Terminator flanked by two pairs of attB/P sites
As shown above(Figure 3.), the terminator is sandwiched between two different pairs of attB/attP                         sites, under the control of a constitutive J23119 promoter.
Two sets of expression systems , induced by IPTG and                         arabinose, were constructed. Same as before, a mathematical formula was also used to                         characterize the strength of the terminators. [GFP]random sequence denotes the                         fluorescence strength of the plasmid with the random sequence. [GFP]terminator denotes the fluorescence strength of the plasmid with a terminator.
The inversion efficiency calculation formula is same as above.
Construction of Seven-Segment Display
After constructing two reliable elements with predictable behaviors, we aimed to directly                         implement and arrange them according to sequential logic, which comes in the form of a visible                         “1→2→3” transition on the seven-segment display. In this display, GFP expression is used instead                         of an LCD (liquid crystal display) to light up the panel. In our case, three simple and                         previously characterized parts were placed in segments except F. The system yields the                         predictable expression of recombinases which invert terminators between corresponding attB/attP                         sites either once (A,D,G) or twice (C,E).
Figure 4. The schematic of Seven-segment Display
Construction of pattern“2017”→“PKU”in 96-well plate
Cells were placed according to the pattern of “2017”and “PKU”, and cultured in the 96-deep-well                         plate. Then cells were transferred into clear-96-well plate, OD600 and fluorescence were                         measured using the plate reader. The transition from the pattern of “2017” to “PKU” was achieved                         using the inversion effect.
Figure 5. The schematic of “2017”→“PKU”
The holes marked “a” is always light. The ones marked “b” need to transform from the light state                         to the dark state. And the ones marked “c” perform in contrary.
Results
Characterization of Terminators
A total of 24 terminators were tested in both orientations (their code names[1] and                             abbreviations are shown in the table below). And a gradient of different concentrations of                             IPTG (0, 0.01, 0.1, 1mM) was used for induction. We found that the calculated Ts values                             varied                             with IPTG concentration. Commonly, the Ts value for the forward orientation increased with                             IPTG concentration and reached the highest value at 0.1mM, after which it declined slightly.                             In the reverse orientation there were no obvious trends. Consequently, 0.1mM of IPTG was the                             most suitable concentration for induction. After the test, we chose 6 terminators                             (221,836,435,                             322,855 and 309), among which 435 and 322 showed the best performance and conformed best to                             the                             4 standards described above.
(The code names and their abbreviations.)
Characterization of Inversion Efficiency
Ts of the different attB/P+terminator combinations
Transformation with the testing system plasmid alone yielded the following results and                             conclusions:
Sole attB/P sites (without a terminator) can block the transcription to a certain extent.                             Such barrier strength varied with the types of the sites. Among the 5 pairs of attB/P sites                             tested, int2 had the strongest intrinsic barrier effect ("Ts"= 23.0), even exceeding that of                             the terminator 221F ("Ts"=16.6). The other sites had weaker barrier effects: int2＞ΦC31                             ("Ts"=2.00)＞TP901-1(AG)("Ts"=1.42).
Combination of different attB/P sites and terminators can produce unexpected                                 results
Int2 attB/P and 435F had the strongest barrier strengths among the recognition sites and                             terminators, respectively. However, the barrier strength of their combination was the                             weakest                             (“Ts”=5.37) among all combinations.
TP901-1(AG), attB/P and 221F had the weakest barrier strength among recognition sites and                             terminators, respectively, but the barrier strength of their combination was the                             highest.
In addition to the barrier effect, the combination of attB/P sites and terminators can also                             lead to the formation of cryptic promoters, which was the case for the TP901-1(AG) 221R                             combination.
The most excellent combinations of attB/P sites and terminators selected based on the four                             standards above were (with the corresponding “Ts” ratios in brackets): int2 221 (35.3), Φc31                             435 (28.3), TPAG 221 (17.7), TPAG 435 (17.3), int2 855 (17.2), and BxGT435 (21.1).
Considering that the expression systems of int2 and TP901-1 are induced by IPTG, which                             causes a                             conflict with the reporting system, we used BxGT435 and Φc31 435, whose systems are induced                             by                             Ara, to conduct the reversal experiments.
Reversal Experiment
Co-transformation of E. coli with the testing system plasmid and corresponding                                 recombinase expression system We observed successful reversal events after inducing the expression of the recombinase. As                             shown in the picture of the 96-well plate, the fluorescence difference is visible by naked                             eye.                             Using the microplate reader and analysis, we confirmed the existence of a significant                             fluorescence difference. In addition, we directly used the bacterial solution to conduct a                             PCR,                             and the sequencing results showed double peaks, which substantially proves that the reversal                             occurred.
Characterization of Terminators Flanked by Two Pairs of attB/P Sites
We built the following constructs:
Two sets of expression systems, induced by IPTG and arabinose, were constructed.
Ts of double attB/P+terminator combinations
Due to the introduction of double attB/P sites, which substantially extended the distance                             from                             the promoter to the GFP coding sequence, together with the barrier effect of each pair of                             attB/P                             sites, it became impractical to expect the Ts of the attB/P with a reverse terminator to be                             around 1. We there-fore lowered the standard and laid more emphasis on the ratio of Ts for                             both                             orientations. Based on the test results above, we chose TP901-1 BxGT435 and Φc31 BxbGT 435 to                             conduct the reversal experiment.
Reversal Experiment
Image Transition Demos
The E. coli groups with “Bxb1 T+B435F” were placed at the position c, and the E.                             coli groups with “Bxb1 T+B435R” were placed at the position b. The a position there are                             groups which always expressed sfGFP.
The E. coli colonies were picked and grown for 12h in LB media containing antibiotics                             in culture tubes at 37°C and 250 rpm in an incubator. Next, the bacterial cultures were                             diluted at a ratio of 1:100 with fresh M9 medium with antibiotics and without inducers,                             after which the bacteria were grown for another 12h. We measured the fluorescence intensity                             and OD600. Then the bacterial cultures were diluted with fresh M9 medium with antibiotics                             and 10mM arabinose at a ratio of 1:100 once again and were grown for another 12h. Finally,                             the fluorescence intensity and OD600 were measured once again.
The ratio of fluorescence intensity/OD600 was calculated using the net fluorescence                             intensity and net OD600. We regarded the ratio greater than 100 as “light” and the one less                             than 100 as “dark”. The ratio of every well is shown below.
A
Animation in 96-well Plate
Animation in 96-well Plate
B
Animation in 96-well Plate
Animation in 96-well Plate
Figure 6.The pattern which transformed “2017” to “PKU”.A. When the culture                             had begun for 12h, we got the pattern “2017”. B. When the culture had begun for 24h, we got                             the pattern “PKU”.
We have confirmed that our genetic circuits can work as designed through the “ 2017-PKU”                             pattern transition. Next we constructed the system named “7 segment display”. The details of                             the circuits in segments are shown in the table below.
Figure 7.The design of "1-2-3" transition
We got the figure below on cloud nine. The numbers in figure were fluorescence intensity of                             corresponding wells, which were measured using the microplate reader mentioned above and                             adjusted as the same methods. We defined the wells as “on” when its fluorescence intensity                             exceeds 50; otherwise defined as “off”. As plain as the nose on your face, we managed to                             make it. We realized the sequential logic!
Figure 8.The experiment result of "1-2-3" transition
References
Dyakonov T, Muir A, Nasri H, et al. Characterization of 582 natural and synthetic                                 terminators and quantification of their design constraints[J]. Nature Methods, 2013,                                 10(7):659-664.
Brophy J A, Voigt C A. Antisense transcription as a tool to tune gene expression[J].                                 Molecular Systems Biology, 2016, 12(1):854.
Yang L, Voigt CA, et al. Permanent genetic memory with >1-byte capacity. Nat Methods.                                 2014 Dec; 11(12):1261-6.
Stothard P. The sequence manipulation suite: JavaScript programs for analyzing and                                 formatting protein and DNA sequences. Biotechniques. 2000 Jun; 28(6):1102, 1104.
Lou C, Stanton B, Chen YJ, Munsky B, Voigt CA. Ribozyme-based insulator parts buffer                                 synthetic circuits from genetic context. Nat Biotechnol. 2012 Nov; 30(11):1137-42.                                 [PMID:23034349]
Back to top
Clock
Background
Aims
Key Achievements
Our Approach
Results
References
Background
Clock Signal
A clock signal is a particular type of signal that oscillates between a high and a low state                         repeatedly. This signal is emitted by a clock generator and is utilized like a metronome to                         trigger actions of a sequential logic circuit. In one clock signal cycle, the state of the                         circuit, which is the basis of sequential logic circuits, remains unchanged after shifting. A sequential logic circuit designed with an inner clock can change its state automatically, even                         without inputs. Additionally, the time a state lasts can be controlled by the clock signal period.
Figure 1. Typical clock signal and fluorescence signals of the repressilator.
Repressilator Introduction
The repressilator is a synthetic genetic regulatory network consisting of a ring-oscillator                         with three genes -- tetR from the Tn10 transposon, cI from bacteriophage λ and lacI from the E. coli lactose operon, each expressing a protein that represses the next gene in the loop.
Figure 2. The structure of the original repressilator. It is encoded by a                         low-copy plasmid with three repressor genes and a high-copy plasmid containing a GFP gene                         driven by PLtetO1.
The Repressilator as an Internal Clock
In our project, we aimed to use the repressilator to generate a clock signal. We chose it for                         the following reasons:
It can generate up to three oscillatory signals in different phases; two different                                 signal peaks can drive the flip-flop to shift its state serving as a component of a period.
The period and amplitude of the oscillation can stay stable and robust through many                                 generations.
The period of the oscillation is adjustable. We can modify the oscillation period by                                 simply chang-ing the bacterial generation period and oscillator components.
It has the potential to be connected to a downstream circuit. Because of the oscillatory                                 changes in promoter strength, we can add the promoters (PR ,PLtetO1 ,PLlacO1) to the downstream circuit.
star
Aims
Compared with natural biological systems, synthetic circuits such as the repressilator have a                         lower accuracy of oscillation. There have been many efforts to enhance the robustness of                         oscillation of the original repressilator, such as improvements in the core cellular processes                         and the reporter systems . Thus, if we want to use the repressilator as an internal clock, we                         have to determine whether our construct would have a significant influence on the oscillation                         itself. To answer this question, we have to do the following experiments:
Evaluate the repressilator system in our own experimental environment.
Construct an oscillator-induced protein expression system and compare its oscillation                                     period with the parent repressilator.
Key Achievements
Test the parent repressilator published in the 2016 paper by Potvin-Trottier et al.
Construct a recombinase oscillator and T7 oscillator.
Our Approach
Reproduction of the Oscillation of the Original Repressilator
We used a stereoscope, a microfluidic device and a plate reader to reproduce the oscillation                         experi-ments of the parent repressilator (single reporter oscillator and three-reporter                         oscillator). The main strains we used in the oscillation-related experiments are listed in                         Table 1. The clpXP genes in these strains were deleted to obtain more robust oscillations.
Table1. The strains used in the oscillation-related experiments.
Strain
Plasmid
Genotype (MC4100)
Recombinase Oscillator
plpt20-gp35
plpr41-reporter
ΔclpXP
Recombinase Amplifier
plpt20-t7
plpt41-gp35-reporter
ΔclpXP
T7 Polymerase Oscillator
plpt20-t7
plpt41-t7 test
ΔclpXP
Triple Reporter Oscillator (LPT117)
plpt107
plpt43
ΔclpXP
Single Reporter Oscillator
plpt20
Plpt41
ΔclpXP
Since one of the repressors is under the control of PLlacO1, we can use IPTG                         (Isopropyl β-D-Thiogalactoside) to synchronize the oscillation in different cells. We also                         compared the synchronized oscillation and the unsynchronized oscillation. (see Results figure)
If we want to look at dynamic changes of the oscillation, population-based experiments are not                         enough, so we also designed a microfluidic device of the same type as the "mother machine"                         used in our lab to house single cells and their offspring. In this device, cells are trapped                         in narrow channels and newborn cells are washed away by fresh medium. (see Results video1)
Figure 3. Design of the "Mother Machine". It consists of dozens of thin channels                         holding single cells, and can be used to track individual cell line for generations.
Design of the Recombinase Oscillator
In order to test whether our modifications of the repressilator would severely affect its                         oscillation pattern, we combined the repressilator with recombinases (Bxb1 gp35) to obtain an                         oscillator-activated recombinase expression system. By adjusting this system, we can also                         determine whether the oscillation period is compatible with the recombination time.
Figure 4. Schematic diagram of the recombinase oscillator.
For the expression system of the recombinase oscillator, we made use of the plpt20 plasmid                         (Laurent et.al., 2016), and we cloned Bxb1 gp35 downstream of mVenus in plpt20. By doing                         this, the expression of recombinase was coupled to the expression of mVenus, which is under the                         control of the promoter PLtetO1.
As for the reporter system used to detect recombinase expression, we constructed a revertible                         promoter surrounded by a Bxb1 attB site and a Bxb1 attP site on the sponge plasmid. When the                         recombinase is expressed, it inverts the promoter sequence and mRFP is expressed.
Both the expression and reporter systems of the recombinase oscillator were constructed                         using Gibson assembly.
A
B
Figure 5. Plasmids of the recombinase oscillator. (A) Plasmid encoding the oscillatory recombinase expression system (plpt20-gp35).                         (B) The plasmid encoding the oscillatory recombinase reporter system (plpt41-reporter).
Possible improvements
mRFP is a widely used fluorescent protein in mammalian cells, but it has its                                     drawbacks. As we have discovered in our experiments, mRFP takes a rather long time                                     to mature and emit red fluorescence. Thus, we may have to exchange the fluorescent                                     reporter protein for a fast-maturing one such as mKate.
The oscillation period is not compatible with the duration of recombinase-induced                                     promoter reversal and mRFP maturation. In order to adjust the oscillation period,                                     we can use different liquid media to prolong the oscillation period. Previous work                                     has shown that the oscillation period is proportional to the cell division time. Thus,                                     by using a liquid medium such as M9, we can increase the cell division time and                                     prolong the oscillation period.
Design of the T7-coupled Oscillator
The repressilator is constantly working as an internal clock, but under most circumstances                         we would like to start the internal clock in a specific moment. Using IPTG to synchronize                         the internal clocks of all the cells in a population and keep them in the same phase seems                         plausible. However, through our experiments with the recombinase oscillator we found that                         using IPTG cannot prevent signal leakage of the internal clock. Thus, we would like to add an                         "AND" gate behind the clock signal, so that we can turn on or turn off the internal clock.
T7 RNA polymerase has been previously used to implement an "AND" gate. In that design, the                         first promoter controls the expression of a T7 RNA polymerase gene with two internal amber                         stop codons. The second promoter controls the amber suppressor tRNA supD. When both promoters                         are activated, T7 RNA polymerase is expressed and can act specifically on the downstream T7                         promoter. What's more, T7 RNA polymerase can be used as an amplifier to increase the signal                         amplitude.
Our aim is to prove that T7 RNA polymerase can be combined with the repressilator and can serve                         as a potential on-off switch and amplifier. Therefore, we designed a T7 oscillator and                         recombinase amplifier.
Figure6. Schematic diagram of the recombinase oscillator.
A
B
C
Figure 7. Plasmids of the T7 polymerase oscillator. (A) The plasmid encoding the T7 oscillator expression system (plpt20-T7). (B) The                         plasmid encoding the T7 oscillator reporter system (plpt41-T7 test). (C) The plasmid encoding                         the recombinase amplifier system.
For the expression system of the T7 oscillator, we also used the plpt20 plasmid (Laurent et.al. , 2016). We cloned the T7 RNA polymerase right behind mVenus in plpt20. To detect the                         expression of T7 RNA polymerase, we added a T7 promoter before mRFP in the plpt41 plasmid.                         In order to prove that T7 polymerase can be used as an amplifier, we also constructed a                         recombinase amplifier. The re-porter system of the recombinase amplifier was adapted from the                         plpt41-reporter; we added a Bxb1-gp35 activated by a T7 promoter in the plpt41-reporter                         plasmid.
Microfluidic Image Analysis
Firstly, we opened the files from the microscope generated in nd2 format in NIS element viewer                         or image J plugins NDi6d. We were then able to change them to videos or pictures in Tag Image                         File Format. Finally, we used image J to analyze the intensity of the images in TIF format.
Results
Recurrence of the Oscillation in the Repressilator
Figure 8. Stereoscopic image of colony oscillation. Triple reporter Plpt107+titration sponge.                         We repeated the experiments of the original repressilator (John et al., 2016,) using the LPT117                         strain. By using a stereoscope, we discovered concentric yellow fluorescence circles in the                         colony of LPT117. This spatial pattern is the reflection of temporal pattern of the oscillation.                         It is formed when the fluorescence signal oscillates during cell division, and inner parts of                         cells in the colony cannot divide and remain the fluorescence signal.
We repeated the experiments of the original repressilator (John et al., 2016,) using the LPT117                         strain. By using a stereoscope, we discovered concentric green fluorescence circles in the                         colony of LPT117.
A
B
Figure 9. Typical time trace of a single cell for original repressilator. A. Time trace of LPT117                         repressilator (plpt107+titration sponge). B. Time trace of single reporter repressilator                         (plpt20+titration sponge). Both traces were normalized to their means. The oscillation period of                         LPT117 is around 510 minutes in LB medium,14 generations ( cell division period is about 35 minutes). The oscillation period of the single reporter oscillator                         is also around 15 generations, 450 minutes (cell division period is about 30 minutes).
Video 1. Microfluidic videos of LPT117 oscillations in the mother machine.
Recombinase Oscillator
Video 2. Dynamic changes of a bacteria colony of recombinase oscillator. At the beginning, the colony was green because of the expression of GFP.When the colony had grown to a certain size, Bxb1-gp35 was expressed leading the expression of mRFP..
Figure 10. Optical microscopic views of a bacteria colony with a ' disk-like ' pattern. An optical                         microscopic view of a bacterial colony with a  ' disk-like ' pattern, the red parts of the colony are distributed in wedge                         form. When recombinase Bxb1-gp35 expresses, the promoter of mRFP surrounded by Bxb1 attB and attP sites is inverted, which leads to the                         expression of mRFP.Some cells shift from yellow to red because of the expression of Bxb1-gp35, and its offsping, which are distributed in wedge form, kept expressing mRFP..
Figure 11. Time trace of a single cell for recombinase oscillator. The oscillation period of                         recombinase oscillator is around 300 minutes, 14 gene in LB medium (cell division period is 20 minutes), which is similar to the oscillation                         period of LPT117 and single reporter repressilator. These results shows that our modification                         on the parent repressilator (combine recombinase with repressilator) will not affect the clock                         itself and clock signal can be passed from the repressilator to downstream structure (invert of                         mRFP promoter).
Video 3. Microfluidic video of Recombinase Oscillator.
References
Elowitz, M. B., & Leibler, S. (2000). A synthetic oscillatory network of transcriptional                                 regulators. Na-ture, 403(6767), 335-338.
Potvin-Trottier, L., Lord, N. D., Vinnicombe, G., & Paulsson, J. (2016). Synchronous                                 long-term oscilla-tions in a synthetic gene circuit. Nature, 538(7626), 514-517.
Wang, P., Robert, L., Pelletier, J., Dang, W., Taddei, F., Wright, A., & Jun, S. (2010).                                 Robust Growth of Escherichia coli. Current Biology, 20(12), 1099–1103.
Back to top
Demostrate
Background & Design
Methods
Results
Discussion
Potential Applications
Background & Design
We attempts to develop a framework of biological sequential circuits that are programmable. The                         envisioned                         circuit is capable of both storing states in DNA and automatically running a series of                         instructions in a                         specific order. More specifically, the sequential logic (or a real-state machine) that consists                         of a clock                         (trigger signal), flip flop (remembering device) and control unit (functional part) in bacteria.                         The clock                         is an oscillator with a repeated signal cycle that is utilized like a metronome to trigger                         actions of                         sequential logic circuits. Flip-flop is a memory device that can remember a state. Paired with a                         clock                         signal, it can realize state transition. The control unit is a functional part which can convert                         a signal                         from flip-flop into complex functions. With such a design, historical events are recorded and                         influence                         current cell behavior.
star
Methods
The bio-flip-flop is constructed as two sequences each with an invertible promoter flanked                         between                         recombination sites and expressing recombinase or recombinase-RDF fusion protein mediating                         inversion of                         another sequence.
Fig.1. Bio-flip-flop construction.
We first tested and optimized recombinase and recombinase-RDF fusion protein functions and we                         standardized                         these parts into BioBrick devices. Then we separated the function of bio-flip-flops and built                         two circuits:                         the Forward and Backward latch. Each of them can implement state change when receives a group of                         trigger                         signal consisting of two inducing events. We then introduced these latches in vivo and proved                         their                         functionality.
Check the FLIP-FLOP module for more details.
Fig.2. Construction of Forward and Backward latches of the bio-flip-flop and their                             function.
Unidirectional terminators whose directions are changed by recombinases or recombinase-RDF                         fusion proteins                         are used to control transcription of the target gene. We measured the strengths of 24                         terminators in both                         directions, and we elected six suitable unidirectional terminators. We measured the strengths of                         several                         terminators paired with attB/P sites and the inversion efficiency of three recombinases.
Check the CONTROLLER module for more details.
We used repressilator, a classic genetic oscillator, as an internal clock to drive our                         bio-flip-flop. To do                         so, we first observed and evaluated the repressilator system in our own experimental                         environment. We then                         constructed an oscillator-induced recombinase expression system and observed promoter inversion,                         which shows                         that repressilator has the possibility to act as an internal oscillatory trigger signal                         generator.
Check the CLOCK module for more details.
Results
Following the methodology of modularity in synthetic biology, we divided the system into three                         key                         components. We showed the feasibility of their assembly into a whole system by a set of                         simulation                         results.
First, an ordinary differential equation system describing recombinase and RDF function was                         built.                         Parameters were estimated from experimental data. Then we assembled four components (two                         recombinases and                         two fusion proteins) as the one in flip-flop design. Inducing of the system is in the form of                         square waves                         to simulate manual induction like changing of medium.
Fig.3. Simulation of the whole flip-flop function. X_F denotes the fraction of                         forward direction (right) of                         the upper promoter pBAD. Y_F denotes the fraction of forward direction (right) of the lower                         promoter pTAC.
We also proved that the system can also be triggered by repressilator as internal clock signal.                         Substituting                         the upper promoter (repressed by tetR protein) to PLtetO1 and the lower one (repressed by cI                         protein) to PR,                         we then did another simulation of forward latch and proved that the flip-flop can change its                         state and then                         recover back to initial state after arrival of two adjacent trigger signal groups.
Discussion
In this simulation experiment, the flip-flop lost approximately 10% of its initial state after                         two trigger                         signal groups. If we use manual and discrete induction, the loss is roughly 5%. These results                         have reached                         the expectations. However, in practical experiment, there are still many problems to solve and                         further                         research is needed. Though we constructed and configured a set of recombinase and                         recombinase-RDF fusion                         protein tools, we observed leaky expression and other problems when assembled. Further                         modification of                         repressilator (adding one more recombinase) has to be tested to satisfy the need of in vivo                         automatic state                         transition.
Potential Applications
This work tries to point the way toward building large computational systems from modular                         biological                         parts—basic sequential computing devices in living cells—and ultimately, programming complex                         biological                         functions. We also illustrated the possibility of self-triggered state transition. Possible                         application                         would be an automatic cell factory that can complete a series of manufacturing procedures with                         just one                         trigger at the start. Furthermore, our bio-flip-flop can act as an in vivo memory device and aid                         in                         recording cellular events, which can provide deep insight into cell differentiation and                         development.
Back to top
Acknowledgement
Connection
Facebook
Youtube
Twitter
Github
Contact
SynBioWiki.com
Mail
Peking iGEM 2017
Peking University,
No.5 Yiheyuan Road Haidian                         District, Beijing, P.R.China
100871
