Team:Peking/Protocols
Peking iGEM 2017
Home Project Modelling Software Hardware Lab Practices Parts Team
Protocols
1. Ligation
2. Chemical Transformation
3. Agarose Gel Electrophoresis
4. Plasmid Extraction
5. Gel Extraction
6. Gibson assembly
7. PCR using Q5 high-Fidelity DNA polymerase
8. Golden Gate Assembly
9. Growing Overnight cultures
10. Colony PCR
11. Preparation of LB Broth, LB Ager, and M9 culture medium
12. Making Competent cells
13. Cell sorting for recombination efficiency characterization
14. Imaging Oscillation of Repressilator
Ligation
Materials:
1 µL T4 DNA Ligase
2 µL 10X T4 DNA ligase buffer
50ng Vector Plasmid
Insert DNA (Required molar ratio of 1:3 :: vector:insert)
Sterile water (Make reaction up to 20 µL)
Methods:
Incubate at room temperature for 30 - 60 min or at 16℃ for 60min.
Chemical Transformation
Methods:
Thaw 50µL DH10B competent E. coli cells on ice.
Add 5 µl DNA from a ligation reaction mix or 10-100ng DNA of a plasmid.
Place the mixture on ice for 20-30 minutes.
Heat shock at 42°C for 90 seconds.
Place on ice for 5 minutes.
Pipette 200 µl of room temperature LB media into the mixture.
Incubate at 37°C and 220 rpm for 60 minutes.
Add 50-100 µl of the transformed cells to the selection plate.
Agarose Gel Electrophoresis
Methods:
Prepare 10% w/v solution of agarose powder in 1× TAE buffer .Weigh agarose powder and TAE buffer and add them to a flask.
Melt the mixture in a microwave until the agarose is completely dissolved.
Add SybrSafe®（1/10000 v/v solution of the mixture）to the solution.
Pour the solution into the gel casting tray with appreciate comb.
Let the gel cool until it is solid and then pull out the comb.
Place the gel in the electrophoresis chamber and add enough TAE Buffer.
Pipette DNA samples mixed with appreciate amount of loading dye (6×) into wells on the gel and add the DNA ladder.
Run the gel at 100V for about half an hour.
Plasmid Extraction
Minipreps were performed according to the Axygen Miniprep Kit, we also make some improvement.
Methods :
Prepare 5mL bacteria solution grown overnight in LB broth, centrifuge at 12,000×g for 1min. Discard the supernatant.
Add 250µL Buffer S1 to resuspend bacteria cells and transfer to a 1.5mL centrifuge tube.
Add 250µL Buffer S2 and gently invert the tube 4-6 times to mix.
Add 350µL Buffer S3 and gently invert the tube immediately but gently 6-8 times.
Centrifuge for 10min at 12,000×g in a microcentrifuge.
Apply the supernatants from step 5 to the Axygen spin column by pipetting.
Centrifuge for 1min. Discard the flow-through.
Wash the Axygen spin column by adding 500µL Buffer W1, and centrifuging for 1min. Discard the flow-through.
Wash the Axygen spin column by adding 700µL Buffer W2, and centrifuging for 1min. Discard the flow-through.
Repeat the step 9
Centrifuging for an additional 2min to remove residual wash buffer.
Drying the Axygen spin column for about 15min at 65°C to keep the value of 260/280 in a normal level.
Place the Axygen spin column in a clean 1.5mL microcentrifuge tube. Add 30-50µL ddH2O (65°C) to the center of the Axygen spin column, let stand for 2min, and centrifuge for 1min.
Gel Extraction
Was performed according to the Tiangem Gel Extraction Kit.
Gibson assembly
Methods :
Keep 15µL Gibson mix on ice.
Add 5µL DNA fragment (Vector fragment: less than 15ng; small fragment: 20-50ng).
Incubate at 50℃ to 60min.
Transformation.
Gibson Mix
1.2 mL
5×ISO buffer
320 µL
10U/µL T5 exo
0.64 µL
20U/µL phusion
20 µL
40U/µL Taq ligase
16 µL
ddH2O
843.4 µL
Store in -20℃ aliquot 15µL aliquot 15µL
5×ISO buffer
6 mL
1M Tris-HCl pH=7.5
3 mL
2M MgCl2
150 µL
100mM dATP
60 µL
100mM dGTP
60 µL
100mM dCTP
60 µL
100mM dTTP
60 µL
1M DTT
300 µL
PEG-8000
1.5 g
100mM NAD
300 µL
Add ddH2O to 6mL, store in -20℃
Aliquot 100µL
PCR using Q5 high-Fidelity DNA polymerase
Materials:
Component
50 μl Reaction
5×Q5 Reaction Buffer
10 μl
10 mM dNTPs
1 μl
10 μM Forward Primer
2.5 μl
10 μM Reverse Primer
2.5 µL
100mM dCTP
60 µL
100mM dTTP
60 µL
Template DNA
Variable
Q5 high-Fidelity DNA polymerase
0.5 μl
5×Q5 High GC Enhancer (optional)
(10 μl)
ddH2O
to 50 μl
Methods:
In a PCR tube on ice, combine 1-10 ng of template DNA, 2.50 μl of 10 μM forward primer,                     2.50 μl of 10 μM reverse primer, 10 μl of 5×Q5 Reaction Buffer, and sterile water to 50 μl.
Gently mix the reaction.
Collect all liquid to the bottom of the tube if necessary.
Transfer PCR tubes to a PCR machine and begin thermocycling.
Thermocycling conditions for a Routine PCR:
Step
Temperature (℃)
Time
Initial denaturation
98
30 seconds
25-35 cycles
98 (denaturation)
50-72 (annealing)
72 (extension)
5-10 seconds
10-30 seconds
15-30 seconds
Final extension
72
2-5 minites
Hold
4
Indefinitely
Golden Gate Assembly
Material:
10×T4 ligase buffer
T4 Polynucleotide Kinase
BsaⅠ/ BsmB Ⅰ
T4 ligase
Sterile water
Methods:
Oligo Anealing
Component
50 μl Reaction
Forward Primer
5 μl
Reverse Primer
5μl
Sterile water
40 μl
Denaturation at 95 ℃ for 5 minutes, then hold at 4 ℃ indefinitely. Dilute annealed Oligo with sterile water after that
Oligo phosphorylation
Component
20 μl Reaction
Oligo
6 μl
T4 ligase buffer
2 μl
T4 polynucleotide kinase
2 μl
Sterile water
10 μl
Note:
Use the diluted Oligo mentioned.
React at 37 ℃ for 1 hour, then hold at 4 ℃ indefinitely.
Golden Gate Assembly
Component
20 μl Reaction
Backbone
30 ng
Oligo
1 μl
T4 ligase buffer
2 μl
Bsa Ⅰ/BsmB Ⅰ
1 μl
T4 ligase
1 μl
Sterile water
to 20 μl
Thermocycling conditions for a Routine
Step
Temperature (℃)
Time
10 cycles
37/42 (37 for BsaⅠ,42 for BsmB Ⅰ )
16 (assembly)
5 minutes
10 minutes
Final react
37/42
50
80
15 minutes
5 minutes
5 minutes
Hold
4
Indefinitely
Transfer the product immediately.
Growing Overnight cultures
Materials:
5ml LB broth
5μl antibiotic
Tips
12 ml culture tube
Methods:
Overnight cultures are prepare under sterile conditions using a Bunsen burner
Add 5ml LB broth into 12 ml culture tubes
Add 5μl antibiotic into the broth
Pick a single colony by a sterile tip and inoculate the culture by dipping the tip into the LB broth.
Seal the tubes and incubate overnight at 37℃ shaking at 200-250rpm.
Colony PCR
Materials:
Component
25 μl Reaction
2×EasyTaq Mix
12.5 μl
Forward Primer
1 μl
Reverse Primer
1 μl
E. coli colony
1
Sterile water
10.5 μl
Methods:
Mix the materials mentioned expect E. coli colony in PCR tubes
Pick 1 E. coli colony and add it into the reaction mixture
Gently mix the reaction
Transfer the PCR tubes to a PCR machine and begin thermocycling
Thermocycling
The PCR machine should be set to run the following steps:
Step
Temperature (℃)
Time
Initial denaturation
98
30 seconds
25-35 cycles
98 (denaturation)
45-72 (annealing)
68 (extension)
5-10 seconds
10-30 seconds
15-30 seconds
Final extension
72
5-10 minites
Hold
4
Indefinitely
Note:
If loading on a gel, the EasyTaq Mix contains loading dye, so don’t add anything else.
Preparation of LB Broth, LB Ager, and M9 culture medium
LB Broth:
Materials:
Component
500 mL
Tryptone
5 g
Yeast Extract
2.5 g
NaCl
5 g
Sterile water
to 500 mL
Methods:
Mix the components listed above
Autoclave
LB Ager
Materials:
Component
500 mL
Ager
7.5 g
Tryptone
5 g
Yeast Extract
2.5 g
NaCl
5 g
Sterile water
to 500 mL
Methods:
Mix the components listed above
Autoclave
M9 culture medium
Materials:
Component
500 mL
5×M9 salt solution
100 mL
10% glucose
10 mL
1 M MgSO4
1 mL
1 M CaCl2
50 μl
10% Casein amino acid (optional)
10 ml
10 mg/ml vitamin B1 (optional)
17ml
Sterile water
to 500 mL
Note:
Add MgSO4 and CaCl2 last.
Methods:
Add 100ml 5×M9 salt solution and sterile water to about 480ml
Add 10ml 10% glucose and 1ml 1M MgSO4
Add 50μl 1M CaCl2, keep shaking the container at the same time
Treat with ultrasonator until no sediment observed
Filtrate with a 0.22μm filter
Making Competent cells
Materials:
sterilized, stored at 4℃ before use
Several 5 mL liquid LB Medium.
500 ml Conical flask with 100 ml LB Medium (ensure sufficient oxygen for bacteria growing).
Several clean 50 ml centrifuge tubes with 2/3 sterilized water.
Enough 200 μl and 1 ml pipette tips.
Enough 1.5ml Eppendorf tubes.
Solution Ⅰ:：80 mM MgCl2 + 20 mM CaCl2, solute in sterilized water.
Solution Ⅱ: 100 mM CaCl2 + 10% glycerol (aq)
Preparation:
Pick a single colony from fresh plate, or take 50 μL of a fresh                     overnight culture and transfer into 5 mL liquid LB medium, incubate at 37℃                     (98.6℉) with vigorous shake overnight (less than 16 h).
Next morning, transfer 200~500 μL culture from Step 1. into 100 ml clean LB                     medium and incubate 1.5~2 h till it is slightly turbid (the cell concentration                     reaches around 5 × 107 cells/mL Do not over incubate) .
In clean bench, transfer whole culture into precooled 50 mL centrifuge tubes, then bath them in ice-water mixture for 20 minutes.
Centrifuge cooled culture at 4℃(39.2℉) 4,000 rpm for 10 minutes to collect                     cells, carefully pour off the medium in clean bench.
Add 5 mL precooled Solution Ⅰfor every 50 mL centrifuge tube in Step 3. and                     suspend the cells.
Collect cells like Step 4. Then suspend cells from every 50 mL centrifuge tube                     in Step 3. with precooled 2 mL Solution Ⅱ .
Bath in ice-water for 4 h.
Aliquot 100 μL to every pre-iced 1.5 mL Eppendorf tube on ice-water, store at -80℃(-112℉)
Cell sorting for recombination efficiency characterization
Materials:
Shaker
Induced cells
96 wells plate
Flow cytometer
PBS
Media of choice
Methods:
Cell preparation:
E. coli strains are grown overnight in LB broth with appropriate antibiotics.
Transfer to M9 culture medium, which contains a certain concentration of inducer (IPTG/Ara).                     Incubate cells and take samples at desired time points.
Dilute culture sample by 10 fold to 100 fold in PBS
Flow cytometry
Set up parameter on a 96 wells plate
Run wells
Use FlowJo to define the gates
Imaging Oscillation of Repressilator
Microfluidic master fabrication
Fabrication of the E. coli mother machine was carried out using standard UV photolithography             in a clean room environment. The device was designed using L Edit and quartz-chrome photomasks.
Note: For all spin coater steps described below, the following shorthand notation is used: speed             (rpm)/acceleration (rpm/sec)/time (sec).
A. First Layer: Cell Channels
This set of steps lays down the channels that house the cells in the final device. The tolerances for this             layer are very stringent; the exposure dose and contact between mask and wafer must be optimized.             We recommend trying a range of exposure parameters to ensure that a useful device is obtained.             We also stress the importance of the very long post exposure bake time in the process below.
Place a new 3′′ Si wafer (we used 380 µm TEST grade wafers from University Wafer)                     in a dish of fresh acetone. Sonicate at high power for 5 minutes.
Sequentially rinse the wafer with streams of methyl alcohol (MeOH), isopropyl alcohol (IPA) and                     H2O (∼ 10 seconds per solvent).
Place wafer on 2′′ spin chuck and spin seconds at 500 rpm.
While spinning, sequentially rinse the wafer with streams of MeOH, IPA and H2O.
Spin wafer 1 minute at 3,000 rpm to dry.
Dehydrate wafer with smooth side upwards above clean filter paper for 5 minutes on a hot plate set                     to 190◦C，then cool it at r.t..
Place the dehydrated wafer onto the spin coater chuck and dispense a small (cover ∼2/3 of the wafer                     surface) amount of Su8 3005 photoresist (Microchem ) with a disposable pipette.                     Run the spin program（Set spin program to: Step 1: 500/100/10, Step 2: 1400/300/36）.                     This should result in a coat of ∼1.5 µm.
Soft bake wafer for 2 minute 95◦C.
Expose wafer for 40~50 seconds (3.7 mW/cm2) through cell channel mask in vacuum contact mode.
Post exposure bake the wafer (in order) for 2 minute at 95◦C（according to the thickness of this                     layer），then cool it at r.t.
B. Second Layer: Feeding Channels
This layer of the device forms the medium flow channels. The dimensions of these features are not critical:             we have used feeding channels of widely varying dimension to similar effect. The alignment is sensitive to             large errors, however. The alignment between feeding channels and cell channels must be accurate             (down to a couple of microns) in order to ensure that the cell channels are of the desired final length.
Set spin program to: Step 1: 500/100/10, Step 2:1200/300/40.
Place the wafer onto the spin coater chuck and dispense a small (cover ∼ 2/3 of the wafer surface)                     amount of Su8 3005 photoresist with a pipette being careful not to introduce bubbles.                     Run the spin program. This should result in a coat of ∼ 9 µm.
Soft bake the wafer (in order) for 8 minute at 95◦C.
With an Su8-developer-soaked swab, clean the newly-deposited photoresist off the alignment marks                     to make them visible for the alignment process.
Align feeding channel mask to the alignment marks on the wafer. Apply vacuum contact and check                     alignment again. If the vacuum application skewed the alignment, repeat the alignment process.
Expose wafer for 30 seconds (3.7 mW/cm2,) through aligned feeding channel mask.
Bake wafer for 10 minute at 95◦C.
Develop wafer for 2 minutes in Su8 Developer with mild agitation.
Rinse wafer for 10 seconds in IPA. Check to ensure that the development is finished. If undesired                     photoresist remains, develop again for 20 seconds.
Verify channel height using a profilometer. The expected height is 10.5 µm. If the channel                     dimensions lie outside of your expected tolerance bounds, the process must be repeated                     with modified spin coating parameters.
Chip preparation
Dimethyl siloxane monomer (Sylgard 184 ) was mixed in a 10:1 ratio with curing agent, poured onto the             silicon wafer, defoamed, degassed for 1 h and cured at 75 °C for 1 h. Individual chips were then cut the             inlets and outlets were punched with a puncher (inner diameter : 0.9 mm, outer diameter : 1.3mm), and             cleaned with Scotch tape. Bonding to water-cleaned and nitrogen-dried coverslips was ensured using oxygen             plasma treatment (60 s at LOW and O2 pressure at 170 mTorr) on the day the experiments start. Attach treated             surfaces of chip and coverslip together gently in case of cavity-clapse.The chips were then incubated at             75 °C overnight to reinforce the bonding.
Cell preparation
Escherichia coli strains were grown overnight in LB with appropriate antibiotics and             diluted 1:100 approximately 2–3 h before the beginning of the experiments in imaging             media, consisting of M9 salts, 10% (v/v) LB, 0.2% (w/v) glucose, 2 mM MgSO4, 0.1 mM CaCl2,             1.5 μM thiamine hydrochloride (Sigma Aldrich, included as a passivating agent).             The cells were centrifuged on a holder that could fit into a standard table-top             centrifuge at 4,000g for 10 min to insert them into the single straight channels.             The feeding channels were connected to syringes filled with imaging media using             Tygon tubing (VWR), and media was pumped using syringe pumps (New Era Pump System)             initially at a high rate of 100 μl min−1 for 5 min, to clear the inlets and outlets.             The media was then pumped at 2.5-5 μl min−1 for the duration of the experiment and cells             were allowed to adapt to the device for several hours before imaging was started.
IPTG synchronization
To synchronize the phase of the oscillators in the population, we diluted the strains in imaging medium             supplemented with appropriate antibiotics and 1 mM IPTG so that they would be early             exponential (OD600 = 0.2) 8 h later (∼1 × 10−6) at 37 °C.
Microscopy and image acquisition
Images were acquired using a Nikon ECLIPSE Ti inverted microscope equipped with a temperature-controlled             incubator, an Orca R2 CCD camera (Hamamatsu), a 60× Plan Apo oil objective (numerical             aperture (NA) 1.4, Nikon), an automated xy-stage (Ludl) and Nikon HG Precentered Fiber             Illuminator (INTENSILIGHT C-HGFIE). All experiments were performed at 37 °C. Typical             exposure was low (50–100 ms) to reduce photobleaching, and the reporter channels were             acquired using 2 × 2 binning (CCD chip dimension of 1,344 × 1,024 pixels, effective pixel             size of 129 × 129 nm). Then 16-bit TIFF images were taken every 5–8 min, and focal drift             was controlled via the Nikon PerfectFocus system, as well as a custom routine based on             z-stack images of a sacrificial position (a position that was not used for further             analysis). The following filter sets were used for acquisition: GFP (Nikon GFPHQ), RFP             (Nikon TxRed), YFP (Nikon YFPHQ) and CFP (Nikon CFPHQ).
Back < Lab Goto > Lab/...
Notebook Safety InterLab
opacity
Acknowledgement
Connection
Facebook
Youtube
Twitter
Github
Contact
SynBioWiki.com
Mail
Peking iGEM 2017
Peking University,
No.5 Yiheyuan Road Haidian District, Beijing, P.R.China
100871
