<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_Valencia_UPV_Elements skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:Valencia UPV/Elements</SPAN></H1><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><P><TITLE>Valencia UPV iGEM 2015</TITLE></P><DIV id="page-wrapper"><HEADER id="header" class="alt"><H1><A href="https://2015.igem.org" target="_blank">iGEM 2015</A> | <A href="https://2015.igem.org/Team:Valencia_UPV">Valencia UPV</A></H1><NAV id="nav"><UL><LI><A href="https://2015.igem.org/Team:Valencia_UPV">Home</A></LI><LI><A href="https://2015.igem.org/Team:Valencia_UPV/Description" class="icon fa-angle-down">Project</A></LI><LI><A href="https://2015.igem.org/Team:Valencia_UPV/Description">Overview</A></LI><LI><A href="https://2015.igem.org/Team:Valencia_UPV/Circuit">Circuit</A></LI><LI><A href="https://2015.igem.org/Team:Valencia_UPV/Components">Components</A></LI><LI><A href="https://2015.igem.org/Team:Valencia_UPV/Modeling">Modeling</A></LI><LI><A href="https://2015.igem.org/Team:Valencia_UPV/Results">Results</A></LI><LI><A href="https://2015.igem.org/Team:Valencia_UPV/Notebook">Notebook</A></LI></UL><LI><A href="https://2015.igem.org/Team:Valencia_UPV/Modeling" class="icon fa-angle-down">Modeling</A></LI><LI><A href="https://2015.igem.org/Team:Valencia_UPV/Modeling">Overview</A></LI><LI><A href="https://2015.igem.org/Team:Valencia_UPV/Modeling/DetModel">Deterministic model</A></LI><LI><A href="https://2015.igem.org/Team:Valencia_UPV/Modeling/Simulations">Simulations</A></LI><LI><A href="https://2015.igem.org/Team:Valencia_UPV/Modeling/AlaDNA2.0">Light control</A></LI><LI><A href="https://2015.igem.org/Team:Valencia_UPV/Modeling/Conclusions">Conclusions</A></LI><LI><A href="https://2015.igem.org/Team:Valencia_UPV/Practices">Policy and Practicies</A></LI><LI><A href="https://2015.igem.org/Team:Valencia_UPV/Practices">Overview</A></LI><LI><A href="https://2015.igem.org/Team:Valencia_UPV/Practices/Minecraft">SynBiocraft</A></LI><LI><A href="https://2015.igem.org/Team:Valencia_UPV/Practices/Safety">Safety</A></LI><LI><A href="https://2015.igem.org/Team:Valencia_UPV/Collaborations">Collaborations</A></LI><LI><A href="https://2015.igem.org/Team:Valencia_UPV/Design" class="icon fa-angle-down">Magic Lamp</A></LI><LI><A href="https://2015.igem.org/Team:Valencia_UPV/Design">Overview</A></LI><LI><A href="https://2015.igem.org/Team:Valencia_UPV/Design#scroll2">Device: Description</A></LI><LI><A href="https://2015.igem.org/Team:Valencia_UPV/Design#scroll3">Device: Electronics</A></LI><LI><A href="https://2015.igem.org/Team:Valencia_UPV/Design#scroll4">How To</A></LI><LI><A href="https://2015.igem.org/Team:Valencia_UPV/OurTeam" class="icon fa-angle-down">Team</A></LI><LI><A href="https://2015.igem.org/Team:Valencia_UPV/OurTeam">Members</A></LI><LI><A href="https://2015.igem.org/Team:Valencia_UPV/Attributions">Attributions</A></LI><LI><A href="https://2015.igem.org/Team:Valencia_UPV/Attributions#scroll3">Acknowledgments</A></LI><LI><A href="https://2015.igem.org/Team:Valencia_UPV/Parts" class="icon fa-angle-down">Parts</A></LI><LI><A href="https://2015.igem.org/Team:Valencia_UPV/Achievements">Achievements</A></LI><LI><A href="https://2015.igem.org/Team:Valencia_UPV/Achievements#scroll1">Medals</A></LI><LI><A href="https://2015.igem.org/Team:Valencia_UPV/Achievements#scroll2">Achievements</A></LI></NAV></HEADER></DIV><H2 style="font-weight:bold">Valencia UPV Team</H2><P>Be patient, we are under construction</P><UL class="actions"><LI><A href="#main" id="button01" class="button">Abstract</A></LI><LI><A href="https://www.youtube.com/watch?v=D9Nx-kyEsO0" id="button02" class="button" target="blank">Video</A></LI></UL><SECTION id="main" class="container"><DIV class="row"><DIV class="12u"><SECTION class="box"><H3 style="color:green">5 June 2015</H3><P>We had 2 cultures from the last day, corresponding to other 2 colonies of ligation. </P><P><I>Agrobacterium</I> culture of promoter less: Luciferase + Renilla </P><P>Minipreps</P><P>Digestion with BamHI and EcoRV</P><P>Agarose gel 1%</P><P>How to ask and make primers?</P><UL><LI>Select the sequence to amplify and save in FASTA format.</LI><LI>gbCloning, go to Tools-Domesticator-1º Category</LI><LI>Add FASTA and select parts.</LI><LI>On the protocol we have the primers </LI><LI>The oligos they give us:</LI><UL class="ul_2"><LI>4 first nucleotides: so the enzyme can recognize without problems</LI><LI>6 following bingind sites.</LI><LI>1 extra nucleotide.</LI><LI>4 overhangs. </LI></UL></UL><P>Meeting with Daniel Ramón (Biopolis). </P><P>Ligation with part 2 and 24 of task sheet.</P><DIV class="table-wrapper"><TABLE class="alt"><TBODY><TR><TD>PIF6 + PhyB; ?1</TD><TD>Etr8 CMV_Bxb1_T35S</TD></TR><TR><TD>1µL 892 (PIF α1)</TD><TD>1µL 1097 (Etr8 CMV) Pupd2</TD></TR><TR><TD>1µL 88E (Phy α2)</TD><TD>1µL Bxb1 (PuPD)</TD></TR><TR><TD>1µL ?1 </TD><TD>1µL Tnos PuPD</TD></TR><TR><TD>1.2µL Buffer ligase</TD><TD>1µL α1</TD></TR><TR><TD>1µL Bsmb1</TD><TD>5.8µL H2O</TD></TR><TR><TD>6.8µL H2O</TD></TR></TBODY></TABLE><P>If we make a digestion of 160 (35S:Renilla:tNOS-35S:P19:tNOS) with EcoRV, we obtain: 2475, 381, 4601 pb.</P><P>If we make a digestion of 896 (Luc:PIF6:PhyB)with EcoRV, we obtain: 11608, 3942 pb.</P><H3 style="color:green">June 2015</H3><P>Transform to E.coli from PIF+Phy and Bxb1</P><UL><LI>1.5µL of ligation</LI><UL class="ul_2"><LI>Cuvette on ice</LI><LI>Competent cells + 1.5µL of ligation</LI><LI>Pulse (electroporator) at 1500V</LI><LI>Add 300µL shock medium and put Eppendorf 1h at 37ºC</LI></UL></UL><P>Culture on petri dishes the ligations.</P><P>Digest of 160, 289 and the two ligations, PIF+phy and Etr8+BxbI. </P><P>Agarose gel. </P><H3 style="color:green">7 June 2015</H3><P>We’ve got white colonies! (from PIF+Phy and Bxb1)</P><P>Pick two colonies from each construction.</P><P>4 tubes</P><UL><LI>3.5µL LB each tube</LI></UL><P>2)	2 tubes + 3.5µL Kanamycin (K)</P><H3 style="color:green">8 June 2015</H3><P>Minipreps of the 4 liquid cultures and digestion to see the band patterns.</P><P>Digestion:</P><DIV class="table-wrapper"><TABLE class="alt"><TBODY><TR><TD>Etr8(CMV):Bxb1:Tnos; ?1</TD><TD>EcoRI</TD><TD>6345, 238</TD></TR><TR><TD>EPIF6 + PhyB-PV16; ?1</TD><TD>BamHI</TD><TD>6686, 1439, 2685, 2237</TD></TR></TBODY></TABLE><P>Agarose gel was made:</P><DIV class="table-wrapper"><TABLE class="alt"><TBODY><TR><TD>Bxb1 (C1)</TD><TD>Bxb1 (C2)</TD><TD>EPIF6 + PhyB-PV16 (C1)</TD><TD>EPIF6 + PhyB-PV16 (C2)</TD></TR><TR><TD>ok</TD><TD>ok</TD></TR></TBODY></TABLE><P>Repeat digestion (errors).</P><P>We don’t have sure the toggle, so we decide to repeat the digestion with other enzyme tomorrow, noticing that the colony 2 has better bands pattern.</P><P>Optimized ligation of PIF-Phy-Lac-Renilla-P19</P><UL><LI>1 µL vector</LI><LI>0.8 µL dilution ½ 160</LI><LI>1.7 µL big part</LI><LI>1.2 µL BSA</LI><LI>1.2 µL buffer</LI><LI>1 µL BsbmI</LI><LI>1 µL ligase</LI><LI>4.15 µL H2O</LI><LI>Ratio 1:2 vector insert</LI></UL><P>As Bxb1 was good at the digestion we put 1 µL of LB and 1 µL of Kanamicyne on the tube where it had grown and store at 37ºC to glycerinate later.</P><P>We design primers to binding domain (BD) and PIF.</P><UL><LI>Problem: domesticator is introduced in an old pUPD. The new one has different bases. </LI><LI>Change manually the pUPD bases in the program (Benchling).</LI></UL><H3 style="color:green">9 June 2015</H3><P>Digestion of the ligation of yesterday containing: EPIF6-PhyB-VP16 (C1 y C2)</P><DIV class="table-wrapper"><TABLE class="alt"><TBODY><TR><TD>EPIF6-PhyB-VP16</TD><TD>PvuII (green buffer)</TD><TD>3663, 9472pb</TD></TR></TBODY></TABLE><P>Agarose gel 1%:</P><DIV class="table-wrapper"><TABLE class="alt"><TBODY><TR><TD>EPIF6-PhyB-VP16 (C1)</TD><TD>EPIF6-PhyB-VP16 (C2)</TD><TD>EPIF6-PhyB-VP16 (C3)</TD></TR><TR><TD>no</TD><TD>no</TD><TD>no</TD></TR></TBODY></TABLE><P>We see three bands: 7000, 4000, 1900pb</P><P>Transform optimized ligation (yesterday 8/6)</P><UL><LI>3.5 µL ligation product (EPIF6-PhyB-VP16 + Luciferase + Renille + P19)</LI><LI>40 µL electrocompetent cells.</LI><LI>Pulse of 1500V</LI><LI>Store 1h at 37ºC</LI><LI>Plate culture at agar with Spectinomycin: 50 µL of the transformation.</LI></UL><H3 style="color:green">10 June 2015</H3><UL><LI>Check the primers and order LexA, Gal4, PIF6, LacI, Dronpa.</LI><LI>Check linker VP16 (88E) and make a primer for it.</LI><LI>Take out glycerinate of Ω2.</LI></UL><P>Alfredo’s part is not working.</P><UL><LI>Pick colonies of transformation and make liquid culture of E:PIF6:PhyB:VP16:luc:ren (C1-C3).</LI></UL><UL><LI>Minipreps of liquid culture (PIF + Phy), colonies C3, C4, C5, C6</LI><LI>Digestion:</LI></UL><DIV class="table-wrapper"><TABLE class="alt"><TBODY><TR><TD>PIF + Phy:VP16</TD><TD>PvuII (buffer green 10x)</TD><TD>3663, 9472</TD></TR><TR><TD>PIF + Phy:VP16</TD><TD>BamHI</TD><TD>1939, 2685, 2337, 6674</TD></TR></TBODY></TABLE><UL><LI>Agarose gel 1% (10 samples, 8 + 2 molecular markers)</LI></UL><P>PIF + Phy (PvuII) C3	PIF + Phy (PvuII) C4	PIF + Phy (PvuII) C5	PIF + Phy (PvuII) C6</P><P>no	ok	no	No</P><P>PIF + Phy (BamHI) C3	PIF + Phy (BamHI) C4	PIF + Phy (BamHI) C5	PIF + Phy (BamHI) C6</P><P>no	ok	no	No</P><UL><LI>Transformation of <I>Agrobacterium</I> (C58) of the 896 construction (EPIF6-PhyB-VP16 + luciferase). We are not going to have the positive control and we won’t be able to quantify (we don’t have Renilla + P19).</LI></UL><H3 style="color:green">11 June 2015</H3><UL><LI>Minipreps of the culture:</LI></UL><UL><LI>Digestion:</LI></UL><DIV class="table-wrapper"><TABLE class="alt"><TBODY><TR><TD>E:PIF6:PhyB:VP16:luc:ren</TD><TD>BamHI</TD><TD>4209, 3756, 6100, 6674</TD></TR><TR><TD>EcoRV</TD><TD>3942, 2989, 2475, 381, 10952</TD></TR></TBODY></TABLE><P>Gel:</P><DIV class="table-wrapper"><TABLE class="alt"><TBODY><TR><TD>PIF6:PhyB:VP16:luc:</TD></TR><TR><TD>ren C1 (BamHI)</TD><TD>PIF6:PhyB:VP16:luc:</TD></TR><TR><TD>ren C3 (BamHI)</TD><TD>PIF6:PhyB:VP16:luc:ren</TD></TR><TR><TD>C1 (EcoRV)</TD><TD>PIF6:PhyB:VP16:luc:ren </TD></TR><TR><TD>C3 (EcoRV)</TD></TR><TR><TD>??</TD></TR></TBODY></TABLE><P>Transformation in <I>Agrobacterium</I> of Renilla (160) due to that we could not join this with PIF:phyB and so we will do a cotransfection of both plasmids. Make petri dish culture with kanamicyn and rifampicin.</P><H3 style="color:green">12 June 2015</H3><P>The petri dish with PIF:phy:luc was taken out the 37ºC room and put into the fridge to pick  colonies tomorrow.</P><H3 style="color:green">13 June 2015</H3><UL><LI>Pick colonies to make liquid culture:</LI><UL class="ul_2"><LI>Renilla in agrobacterium: just one colony, it was made liquid culture but check carefully the gel.</LI><LI>It was noticed that the piece 160, renilla, needs a pSub plasmid to replicate itself so we will transform 160 into a agrobacterium with this plasmid (C58 pSub).</LI></UL><LI>Ligation:</LI></UL><DIV class="table-wrapper"><TABLE class="alt"><TBODY><TR><TD>BxbI; alfa1+phyB; alfa2</TD></TR><TR><TD>1µl BxbI</TD></TR><TR><TD>1 µl phyB</TD></TR><TR><TD>1 µl ?2</TD></TR><TR><TD>4.6 µl H2O</TD></TR></TBODY></TABLE><UL><LI>Transform renilla (160) into agrobacterium and make </LI></UL><H3 style="color:green">15 June 2015</H3><UL><LI>Repeat the ligation BxbI+35S:E-PIF6:tnos because PIF was ?2</LI></UL><DIV class="table-wrapper"><TABLE class="alt"><TBODY><TR><TD>BxbI + 35S:E-PIF6:tnos; ?1</TD></TR><TR><TD>1µl BxbI</TD></TR><TR><TD>1 µl phyB</TD></TR><TR><TD>1 µl ?1</TD></TR><TR><TD>4.6</TD><TD>µl H2O</TD></TR></TBODY></TABLE><UL><LI>KDronpa has arrived:</LI><UL class="ul_2"><LI>Centrifuge it 2-5sec at max velocity.</LI><LI>Add 50 µl to have a concentration of 20ng/µl</LI><LI>Mix it with the vortex and spin.</LI></UL><LI>Ligation: en la libreta pone que se liga a pUPD2</LI></UL><DIV class="table-wrapper"><TABLE class="alt"><TBODY><TR><TD>KDronpa; pUPD2</TD></TR><TR><TD>1 µl KDronpa</TD></TR><TR><TD>1 µl pUPD2</TD></TR><TR><TD>5.6 µl H2O</TD></TR></TBODY></TABLE><UL><LI>It was not possible to pick colonies of the <I>Agrobacterium</I> transformed because they did not grow. Maybe the problem is that with tetraciclyn bacterias grow slowly. Wait 1 day more.</LI><LI>Transformation of the ligation, BxbI + 35S:E-PIF6:tnos; ?1, into E.coli.</LI><LI>Make an agar culture in petri dish and let grow 16h at 37ºC.</LI></UL><H3 style="color:green">16 June 2015</H3><UL><LI>Transformation of the ligation, KDronpa, into E.coli.</LI><LI>Pick colonies of BxbI:E-PIF6 and make liquid culture (C1-C3).</LI><LI>Primers had arrived, it has been done the resuspension (dilution 1:10)of all of them.</LI></UL><DIV class="table-wrapper"><TABLE class="alt"><TBODY><TR><TD>Primers</TD><TD>Code </TD><TD>Template</TD><TD>Working temperature? ºC</TD></TR><TR><TD>LacI F</TD><TD>1</TD><TD>LacI (858)</TD><TD>69.7</TD></TR><TR><TD>LacI R </TD><TD>2</TD></TR><TR><TD>Gal4 F</TD><TD>3</TD><TD>We did not take out the glicerynate.</TD><TD>63.2</TD></TR><TR><TD>Gal4 </TD><TD>4</TD></TR><TR><TD>LexA F</TD><TD>5</TD><TD>LexA (732)</TD><TD>62.7</TD></TR><TR><TD>LexA R</TD><TD>6</TD></TR><TR><TD>PIF:VP16 F</TD><TD>7</TD><TD>PIF6 (288)</TD><TD>60.1</TD></TR><TR><TD>PIFVP16 R</TD><TD>8</TD></TR><TR><TD>NDronpa F1</TD><TD>9</TD><TD>Kdronpa</TD><TD>67.7</TD></TR><TR><TD>NDronpa R1</TD><TD>10</TD></TR><TR><TD>Dronpa F2</TD><TD>11</TD><TD>58.5</TD></TR><TR><TD>NDronpa R2</TD><TD>12</TD></TR></TBODY></TABLE><UL><LI>A PCR with all the primers and the fragments was done, the samples were put in order with the temperature.</LI><UL class="ul_2"><LI>The templates were in dilution 1:50, exception of KDronpa that was dilution 1:5 and the primers 1:10.</LI></UL></UL><DIV class="table-wrapper"><TABLE class="alt"><TBODY><TR><TD>PCR Fusion Taq (50µl)</TD></TR><TR><TD>DNA template (10 µg/µl)</TD></TR><TR><TD>0.5 µl fusion taq</TD></TR><TR><TD>2.5 µl primer F</TD></TR><TR><TD>2.5 µl primer R</TD></TR><TR><TD>2 µl NTPs</TD></TR><TR><TD>31.5 µl H2O</TD></TR><TR><TD>17 June 2015</TD></TR></TBODY></TABLE><UL><LI>Pick colonies and make liquid culture of:</LI><UL class="ul_2"><LI>KDronpa (C1-C4)</LI><LI>Ligations with the PCR’s products:</LI><LI>Templates PCR: 1+2, 5+6, 7+8PIF, 7+8VP16, 9+10, 11+12.</LI></UL></UL><DIV class="table-wrapper"><TABLE class="alt"><TBODY><TR><TD>Template PCR; pUPD2</TD></TR><TR><TD>0.5µl template</TD></TR><TR><TD>1µl pUPD2</TD></TR><TR><TD>6.1µl H2O</TD></TR></TBODY></TABLE><UL><LI>Minipreps of liquid cultures:</LI><UL class="ul_2"><LI>BxbI:E-PIF6 (C1-C3)</LI></UL><LI>Agarose gel with the PCRs:</LI></UL><DIV class="table-wrapper"><TABLE class="alt"><TBODY><TR><TD>Template</TD><TD>1+2</TD><TD>5+6</TD><TD>7+8PIF</TD><TD>7+8VP16</TD><TD>9+10</TD><TD>11+12</TD></TR><TR><TD>Band pattern</TD><TD>1017</TD><TD>284</TD><TD>391</TD><TD>478</TD><TD>464</TD><TD>290</TD></TR><TR><TD>Gel result</TD><TD>ok</TD><TD>ok</TD><TD>ok</TD><TD>ok</TD><TD>No DNA</TD><TD>ok</TD></TR></TBODY></TABLE><UL><LI>Transformation in E.coli of the correct ligations:</LI><UL class="ul_2"><LI>1+2, 5+6, 7+8PIF, 7+8VP16, 11+12</LI><LI>Put in cloranfenicol petri dishes. </LI></UL></UL><H3 style="color:green">18 June 2015</H3><UL><LI>Minipreps of the liquid cultrures:</LI><UL class="ul_2"><LI>KDronpa (C1-C4)	</LI></UL><LI>Digestions:</LI></UL><P>KDronpa	EcoRI	2800</P><UL><LI>Gel:</LI></UL><DIV class="table-wrapper"><TABLE class="alt"><TBODY><TR><TD>Kdronpa C1</TD><TD>Kdronpa C2</TD><TD>Kdronpa C3</TD><TD>KdronpaC4</TD></TR><TR><TD>no</TD><TD>no</TD><TD>ok</TD><TD>no</TD></TR><TR><TD>Etr8:BxbI:phyB C1</TD><TD>Etr8:BxbI:phyB C2</TD><TD>Etr8:BxbI:phyB C3</TD></TR><TR><TD>No</TD><TD>no</TD><TD>no</TD></TR></TBODY></TABLE><P>We discovered that the construction with BxbI did not go well because our lab college gives us the wrong piece. </P><UL><LI>Take glicerynates out:</LI><UL class="ul_2"><LI>Gal4; pUPD (731)</LI><LI>?2</LI><LI>PromotersinATG (GB00552)</LI><LI>Renilla (160)(159)(109)</LI></UL><LI>PCR:</LI></UL><DIV class="table-wrapper"><TABLE class="alt"><TBODY><TR><TD>NDronpa</TD></TR><TR><TD>2.5 µl (9+10) primer F</TD></TR><TR><TD>2.5 µl (11+12) primer R</TD></TR><TR><TD>2 µl NTPs</TD></TR><TR><TD>0.2 µl Taq</TD></TR><TR><TD>10 µl Buffer</TD></TR><TR><TD>31.5</TD><TD>µl H2O</TD></TR></TBODY></TABLE><UL><LI>Ligations:</LI></UL><DIV class="table-wrapper"><TABLE class="alt"><TBODY><TR><TD>Etr8:BxbI:T35S; α1</TD><TD>Template PCR; pUPD2</TD></TR><TR><TD>1 µlEtr8</TD><TD>0.5µl template</TD></TR><TR><TD>1 µl BxbI</TD><TD>1µl pUPD2</TD></TR><TR><TD>1 µl T35S</TD><TD>6.1µl H2O</TD></TR><TR><TD>5.8 µl H2O</TD></TR><DIV class="table-wrapper"><TABLE class="alt"><TBODY><TR><TD>Templates PCR: 1+2, 5+6, 7+8PIF, 7+8VP16</TD></TR><H3 style="color:green">19 June 2015</H3><UL><LI>We do a PCR with the normal Taq polymerase.</LI><UL class="ul_2"><LI>1µl of DNA’s template (9+10, 9+12 and 11+12)</LI><LI>2µl of specific buffer</LI><LI>2µl of NTPs</LI><LI>1µl primer forward</LI><LI>1µl primer reverse</LI><LI>0.5 µl of Taq</LI><LI>12.5 µl H2O</LI><LI>These quantities multiplied by 3.</LI></UL><LI>Minipreps of the yesterday’s glycerinated cultures.</LI><LI>To do the digestions we add in each eppendorf.</LI></UL><DIV class="table-wrapper"><TABLE class="alt"><TBODY><TR><TD>Minipreps:</TD><TD>Enzime</TD><TD>Band pattern</TD></TR><TR><TD>159 pDGB1_?2 renilla</TD><TD>EcoRV</TD><TD>2909, 2475,882, 812, 381</TD></TR><TR><TD>Entry vector, ?2</TD><TD>EcoRV</TD><TD>6652, 621</TD></TR><TR><TD>552 pP35s NoATG, pUPD</TD><TD>EcoRI</TD><TD>2997, 1090</TD></TR><TR><TD>160 renilla pDGB1, a2 </TD><TD>EcoRV</TD><TD>4601, 2475, 381</TD></TR><TR><TD>731 pUPD pGal4BD (CDS)</TD><TD>EcoRI</TD><TD>2997, 2493</TD></TR><TR><TD>109</TD><TD>GB1_a1 355:renilla:Tnos</TD><TD>EcoRI</TD><TD>2580, 2493</TD></TR></TBODY></TABLE><UL><LI>We make an agarose gel with the digestions made before and the PCR of KDronpa. </LI></UL><DIV class="table-wrapper"><TABLE class="alt"><TBODY><TR><TD>159</TD><TD>160</TD><TD>?2</TD><TD>552</TD><TD>731</TD><TD>109</TD><TD>9+10</TD><TD>9+12</TD><TD>11+12</TD></TR><TR><TD>ok</TD><TD>ok</TD><TD>ok</TD><TD>ok</TD><TD>ok</TD><TD>ok</TD><TD>no</TD><TD>ok</TD><TD>ok</TD></TR></TBODY></TABLE><UL><LI>Transformation of the ligations:</LI><UL class="ul_2"><LI>50 µl of electrocompetent cell</LI><LI>1.5 µl of ligation</LI><LI>Put 1 min before the cubettes in ice </LI><LI>Make the transformation (1500V)</LI><LI>Put the transformed cells 1h at 37ºC</LI></UL></UL><UL><LI>We made an stack of Cloranfenicol petri dishes</LI><UL class="ul_2"><LI>250ml  LB aga</LI><LI>X-gal (1:500): 500 µl</LI><LI>Iptg (1:1000): 250 µl</LI><LI>Cloranfenicol (1:2000): 125 µl</LI></UL></UL><H3 style="color:green">20 june 2015</H3><P>We have White colonies of renilla! Also of Etr8 + Bxb1</P><P>We have also pUPD colonies but they are so close to the blue ones that we can’t pick anyone.So we make strakes in another plates to have the colonies separated.</P><UL><LI>We make a liquid culture of Agro of Renilla.</LI><UL class="ul_2"><LI>5ml of LB agar</LI><LI>5 µl rifampicine</LI><LI>5 µl of kanamicine</LI><LI>5 µl tetracycline</LI><LI>Incubate for 2 days (48h) at 28ºC.</LI></UL></UL><H3 style="color:green">21 June 2015</H3><UL><LI>Pick colonies of and put into a liquid medium of 3 ml of LB agar, 3 µl of each antibiotic (kanamicine, spectomicine, cloranfenicol):</LI><UL class="ul_2"><LI>Plates (17/06/15): PIF (C1, C2) (with cloranfenicol), VP16 (C1, C3) (with cloranfenicol), LacI (C1, C2, C3) (with cloranfenicol)</LI><LI>Plates (19/06/15): Bxb1 (C1, C2, C3) (with kanamicine), VP16 (C4, C5) (with cloranfenicol), LacI (C1, C2) (with C1, C2) (with cloranfenicol), PIF (C1, C2, C3, C4, C5) (with cloranfenicol), LexA (C1, C2) (with cloranfenicol).</LI></UL><LI>We take out two glicerynates of GFP and BFP (of the Alfredo’s box)</LI></UL><H3 style="color:green">22 June 2015</H3><UL><LI>We made minipreps of the liquid culture of the day before:</LI><UL class="ul_2"><LI>LacIBD, pUPD (C1-C5)</LI><LI>LexABD, pUPD (C1, C2)</LI><LI>Etr8(CMV):Bxb1 (C1-C3)</LI><LI>PIF6,pUPD (C1-C5)</LI><LI>VP16, pUPD (C1, C4, C5)</LI></UL></UL><UL><LI>Make the digestions of all the minipreps:</LI></UL><DIV class="table-wrapper"><TABLE class="alt"><TBODY><TR><TD>LacIBD, pUPD</TD><TD>NotI</TD><TD>2046, 1053</TD></TR><TR><TD>LexABD, pUPD </TD><TD>NotI</TD><TD>2046, 321</TD></TR><TR><TD>Etr8(CMV):Bxb1 </TD><TD>NotI</TD><TD>1532, 1290, 5896</TD></TR><TR><TD>PIF6,pUPD </TD><TD>NotI</TD><TD>2046, 407</TD></TR><TR><TD>VP16, pUPD </TD><TD>NotI</TD><TD>2046, 500</TD></TR></TBODY></TABLE><UL><LI>Viral system:……….</LI><LI>We received the reported Bxb1!</LI><UL class="ul_2"><LI>500ng of sample</LI><LI>Centrifuge at 3000rpm for 5 seconds (spin).</LI><LI>Add 50 µl H2O</LI><LI>Shake it and let at 50ºC for 20min</LI><LI>Make a PCR of Gal4 and NDrompa (9-10), the primers of NDrompa are aliquoted</LI></UL><LI>…..</LI></UL><P>Make the gel with all the digestions writed before.</P><DIV class="table-wrapper"><TABLE class="alt"><TBODY><TR><TD>Lac1</TD><TD>Lac2</TD><TD>Lac3</TD><TD>Lac4</TD><TD>Lac5</TD><TD>Lex1</TD><TD>Lex2</TD><TD>Bxb1,1</TD><TD>Bxb2, 2</TD><TD>Bxb1,3</TD></TR><TR><TD>Ok</TD><TD>ok</TD><TD>ok</TD><TD>ok</TD><TD>ok</TD><TD>no</TD><TD>no</TD><TD>ok</TD><TD>ok</TD><TD>no</TD></TR><TR><TD>PIF1</TD><TD>PIF2</TD><TD>PIF3</TD><TD>PIF4</TD><TD>PIF5</TD><TD>VP16,1</TD><TD>VP16,4</TD><TD>VP16,5</TD></TR><TR><TD>No</TD><TD>no</TD><TD>-</TD><TD>ok</TD><TD>ok</TD><TD>ok</TD><TD>ok</TD><TD>ok</TD></TR><TR><TD>PCRS</TD><TD>…</TD><TD>…</TD><TD>…</TD></TR></TBODY></TABLE><UL><LI>We make ligations of:</LI><UL class="ul_2"><LI>Etr8(CMV):BxbI in a1 + PhyB:P16 in a2, ?1</LI><LI>LacIBD in pUPD2 + PIFBDless+promoter+terminator, a1</LI><LI>KDrompa in pUPD2 + LacBD+promoter+terminator, a1</LI><LI>Gal4BD, pUPD2</LI><LI>Reporter of BxbI, pUPD2</LI></UL></UL><UL><LI>Tomorrow we have to take out pUPD of constitutive promoters, terminators and GFP (CDS).</LI></UL><H3 style="color:green">23 June 2015</H3><UL><LI>Transformations in E.Coli of the 5 ligations done yesterday and two more transformations of 5+6(1) and 5+6(2) which are the ligations in pUPD of the 18/06. </LI><LI>We put 50 µl of the transformations in the plates. Put in 37ºC.</LI><UL class="ul_2"><LI>Etr8(CMV):Bxb1 in a1 + PhyB:P16 in a2, ?1</LI><LI>LacIBD in pUPD2 + PIFBDless+promoter+terminator, a1</LI><LI>KDrompa in pUPD2 + LacBD+promoter+terminator, a1</LI><LI>Gal4BD, pUPD2</LI><LI>Reporter of Bxb2, pUPD</LI><LI>LexABD (5+6(1)), pUPD</LI></UL></UL><UL><LI>We have taken out of the -80ºC fridge the glycerinate of GFP, pUPD/ampicilina/GB0059.</LI><LI>The liquid culture of Renilla (rifampicina/kanamicia/tetraciclina) doesn’t grow before the two days required. So we decide to put two new colonies in une tube with the three antibiotics and another with rifampicina and kanamicine. Asun say that the tetracycline slow down the growth of Agro.</LI><LI>The 4 liquid cultures of LexA+IPTG/+gal are all blue: throw them.</LI><LI>We ordered again the primer nº10 (NDronpa R1). Changing one codon in 3’ and delete another in 5’.</LI></UL><H3 style="color:green">24 June 2015</H3><P>Pick colonies of the plates done yesterday and pass them into a liquid media: 3 µl of LB, 3 µl of antibiotics (K, Spect, Clor).</P><UL><LI>LacIBD+PIF, a1 (C1, C2)</LI><LI>Gal4BD, pUPD2 (C1)</LI><LI>RepBxb1, pUPD2 (C1, C2, C3)</LI><LI>LacIBD+KDonpa, a1 (C1, C2)</LI><LI>Etr8(CMV)+Bxb1+phyB+VP16, ?1 (C1)</LI><LI>LexABD1, pUPD (C1-C4)</LI><LI>LexABD2, pUPD. No colonies.</LI></UL><P>The viral systems cultures of Agro for the color mosaics are ready before 2 days at 28ºC. We can make the Agroinfiltration.</P><P>Buffers of Agroinfiltration:</P><UL><LI>First we have to prepare and ajust the pH of the buffer MES and the buffer MgCl.</LI><LI>MES (10x), 100nM; ph=5,6 (adding NaOH). Make 1L.</LI><LI>MgCl (100x), 1M. Make 100ml.</LI><LI>Solution to agroinfiltration: 10ml of MES(10x) + 1ml of MgCl (100x) + 100 µl of DMSO+acetosiningona and finally “enrasar” to 100ml.</LI><LI> 19.6mg of acetosiningona for 500 µl of DMSO</LI></UL><UL><LI>Ligation:</LI></UL><DIV class="table-wrapper"><TABLE class="alt"><TBODY><TR><TD>ETR8(CMV):Bxb1(a1)+phyB+VP16(a2); ?1 </TD><TD>Gal4BD(pcr) + pUPD2</TD></TR><TR><TD>1.5 µl etr8:Bxb1</TD><TD>1 µl Gal4 PCR</TD></TR><TR><TD>1.5 µl 88E (phyB+VP16)</TD><TD>1 µl pUPD2</TD></TR><TR><TD>1 µl ?1</TD><TD>1.2 buffer T4 ligase</TD></TR><TR><TD>1.2 µl buffer T4 ligase</TD><TD>1 µl ligase</TD></TR><TR><TD>1 µl ligase</TD><TD>1.2 µl BSA</TD></TR><TR><TD>1.2 µl BSA</TD><TD>1 µl BsmbI</TD></TR><TR><TD>1 µl BsmbI</TD><TD>5,6 µl H2O</TD></TR><TR><TD>3.6</TD><TD>µl H2O</TD></TR></TBODY></TABLE><P>Quantification of DNA:</P><UL><LI>pUPD GFF (0059): 249 ng/µl</LI><LI>?2: 238 ng/µl</LI><LI>Alfredo’s pUPD2, domesticator: 102 ng/µl</LI><LI> iGEM704: 405 ng/µl</LI><LI>IGEM735: 403 ng/µl</LI><LI>552 AMP 35S noATG: 45 ng/µl</LI><LI>PIF (C5), pUPD2: 119 ng/µl</LI><LI>pD6B3, ?2 (22/06): 158 ng/µl</LI><LI>LacIBD (C1), pUPD (22/06): 129 ng/µl</LI><LI>109k renillaDC: 49 ng/µl</LI><LI>IGEM 534: 13.6 ng/µl</LI><LI>VP16 (C1), pUPD2:102 ng/µl</LI><LI>IGEM 1097: 409 ng/µl</LI><LI>K-donpa (C3), pUPD2 (18/06): 174 ng/µl</LI><LI>IGEM 858: 487 ng/µl</LI><LI>731AMP Gal4 (19/06): 81 ng/µl</LI><LI>IGEM pUPD2 domesticator: 87 ng/µl</LI><LI>PIF+phy8 (c1) (08/06): 108 ng/µl</LI><LI>160 renilla, a2 (19/06): 46 ng/µl</LI><LI>159 renilla, ?2 (19/06): 149 ng/µl</LI><LI>Etr8:Bxb1 (C1)(22/06): 149 ng/µl</LI><LI>IGEM 732: 422 ng/µl</LI></UL><P>Measurement of the OD:</P><UL><LI>first we have to </LI></UL><H3 style="color:green">25 June 2015</H3><P>Minipreps of the liquid culture:</P><UL><LI>We don’t observed growth in LacIBD+PIF and LaciBD+K-donpa.</LI></UL><P>Digestion of the minipreps and do the gel:</P><DIV class="table-wrapper"><TABLE class="alt"><TBODY><TR><TD>Gal4BD, pUPD2</TD><TD>NotI</TD><TD>2046, 282</TD></TR><TR><TD>RepBxb1, pUPD2</TD><TD>NotI</TD><TD>2046, 460</TD></TR><TR><TD>Etr8(CMV):Bxb1 + phyB,a1</TD><TD>BamHI</TD><TD>6674, 2237, 2806, 1174</TD></TR><TR><TD>LexABD, pUPD2</TD><TD>NotI</TD><TD>2046, 321</TD></TR><TR><TD>9+10, pUPD2</TD><TD>NotI</TD><TD>464</TD></TR></TBODY></TABLE><P>Gel:</P><DIV class="table-wrapper"><TABLE class="alt"><TBODY><TR><TD>Etr8:Bxb1</TD><TD>Lex1</TD><TD>Lex2</TD><TD>Lex3</TD><TD>Lex4</TD><TD>Rep1</TD><TD>Rep2</TD><TD>Rep3</TD><TD>Gal4 C1</TD><TD>PCR 9+10</TD></TR><TR><TD>no</TD><TD>no</TD><TD>no</TD><TD>no</TD><TD>no</TD><TD>ok</TD><TD>ok</TD><TD>ok</TD><TD>no</TD><TD>ok</TD></TR></TBODY></TABLE><UL><LI>We make a PCR of the Fusion Taq pH (proof-reading) to prove that the primer received number 10. This new one works! Amplify the sequence of N-donpa (R1).</LI><LI>Refresh the cultures of Agro (28ºC). We pick 7.5 µl of the previous culture. And put into a new liquid media with Rifampicine and Kanamicine for 2 days and 28ºC.</LI><LI>Ligations:</LI></UL><DIV class="table-wrapper"><TABLE class="alt"><TBODY><TR><TD>N-dronpa</TD><TD>Rep GFP</TD><TD>Gal4</TD><TD>LexA</TD></TR><TR><TD>1 µl PCR 9+10</TD><TD>1 µl Rep Bxb1</TD><TD>1 µl PCR 3+4</TD><TD>1 µl PCR 5+6</TD></TR><TR><TD>1 µl PCR11+12</TD><TD>1 µl promoter without ATG</TD><TD>1 µl pUPD2</TD><TD>1 µl pUPD2</TD></TR><TR><TD>1 µl pUPD2</TD><TD>1 µl Tnos</TD></TR><TR><TD>1.2 µl buffer ligase</TD><TD>1.2 µl buffer ligase</TD><TD>1.2 µl buffer ligase</TD><TD>1.2 µl buffer ligase</TD></TR><TR><TD>1.2 µl BSA</TD><TD>1.2 µl BSA</TD><TD>1.2 µl BSA</TD><TD>1.2 µl BSA</TD></TR><TR><TD>1 µl BsmbI</TD><TD>1 µl BsaI</TD><TD>1 µl BsmbI</TD><TD>1 µl BsmbI</TD></TR><TR><TD>1 µl T4 ligase</TD><TD>1 µl T4 ligase</TD><TD>1 µl T4 ligase</TD><TD>1 µl T4 ligase</TD></TR><TR><TD>4,6 µl H2O</TD><TD>3,6 µl H2O</TD><TD>5,6 µl H2O</TD><TD>5,6 µl H2O</TD></TR><TR><TD>1 µl a2</TD></TR><TR><TD>Etr8:Bxb1+phyB</TD></TR><TR><TD>1 µl Etr8:Bxb1</TD></TR><TR><TD>1 µl 88E</TD></TR><TR><TD>1µl ?1</TD></TR><TR><TD>1.2 µl buffer ligase</TD></TR><TR><TD>1.2 µl BSA</TD></TR><TR><TD>1 µl BsmbI</TD></TR><TR><TD>1 µl T4 ligase</TD></TR><TR><TD>3,6 µl H2O</TD></TR></TBODY></TABLE><P>We transform the ligations in E.Coli ant pass them into agar plates with cloranfenicol for all of them except the ligation of Etr8:Bxb1+phy that goes with bhnfnfg</P><H3 style="color:green">26 June 2015</H3><P>We do again the two ligations that didn’t gone?</P><DIV class="table-wrapper"><TABLE class="alt"><TBODY><TR><TD>Rep GFP</TD><TD>LacI BD+PIF6</TD></TR><TR><TD>1 µl Rep Bxb1</TD><TD>1 µl LACIBD, pUPD</TD></TR><TR><TD>1 µl promoter without ATG</TD><TD>1 µl PIF6, pUPD</TD></TR><TR><TD>1 µl Tnos</TD><TD>1 µl promoter</TD></TR><TR><TD>1.2 µl buffer ligase</TD><TD>1 µl T35</TD></TR><TR><TD>1.2 µl BSA</TD><TD>1 µl a1</TD></TR><TR><TD>1 µl BsaI</TD><TD>1.2 µl buffer BsaI</TD></TR><TR><TD>1 µl T4 ligase</TD><TD>1.2 µlBSA</TD></TR><TR><TD>2.6 µl H2O</TD><TD>1 µl BsaI</TD></TR><TR><TD>1 µl a2</TD><TD>1 µl ligase</TD></TR><TR><TD>1µl GFP (0059)</TD><TD>2.6 µl H2O</TD></TR></TBODY></TABLE><P>We make a gel of LAcI+PIF6 (C1 and C2). Both of them present the fragment of the vector at 6000 pb but none of them at 2000bp which is the insert one.</P><DIV class="table-wrapper"><TABLE class="alt"><TBODY><TR><TD>LacIBD+PIF; a1</TD><TD>EcoRI</TD><TD>6345, 1997, 641</TD></TR></TBODY></TABLE><DIV class="table-wrapper"><TABLE class="alt"><TBODY><TR><TD>LacIBD+PIF C1</TD><TD>LacIBD+PIF C2</TD></TR><TR><TD>no</TD><TD>no</TD></TR></TBODY></TABLE><P>Measurement of the ODs of phyB+PIF+luc and renilla+P19.</P><UL><LI>phyB+PIF+luc: o.35 (1:2)</LI><LI>Ren+P19: 0.34 (1:2)</LI><LI>Final volume= 2</LI><LI>CCo= 0.35*2</LI><LI>CCf= 0.2</LI><LI>Ecuation=&gt; Vo*CCo=Vf*CCf</LI><LI>So we obtain that Vo(LacI+PIF6)=1.429 µl and Vo(ren)= 1.412 µl.</LI></UL><UL><LI>Ligation of: </LI></UL><DIV class="table-wrapper"><TABLE class="alt"><TBODY><TR><TD>LacIBD,pUPD + K-donpa, pUPD</TD></TR><TR><TD>1 µl 35S</TD></TR><TR><TD>1 µl LacIBD,pUPD2</TD></TR><TR><TD>1 µl K-dronpa, pUPD</TD></TR><TR><TD>1 µl T35S</TD></TR><TR><TD>1 µl a1</TD></TR><TR><TD>1.2 µl buffer T4 ligase</TD></TR><TR><TD>1 µl BsaI</TD></TR><TR><TD>1 µl T4 ligase</TD></TR><TR><TD>2.6 µl H2O</TD></TR></TBODY></TABLE><UL><LI>We make the agorinfiltration of (…..) to start checking if the plant react to the different ..</LI></UL><H3 style="color:green">27 June 2015</H3><P>Transformation in E.coli of LacIBD+K-Dronpa, a1</P><P>We Put into plates LexABD and Etr8(CMV):Bxb1:GFP again and also LAcIBD+K-Dronpa.</P><P>We make liquid culture of:</P><UL><LI>RepBxb1:GFP (C1-C4)</LI><LI>LacIBD+PIF (C1-C5)</LI><LI>N-Dronpa (C1-C4)</LI><LI>Gal4BD (C1-C5)</LI><LI>LexA</LI></UL><H3 style="color:green">28 June 2015</H3><P>We do the minipreps of the liquid cultures that have grown.</P><UL><LI>RepBxb1:GFP (C1 and C2)</LI><LI>LacIBD+PIF (C1-C4)</LI><LI>N-Dronpa (C1-C4)</LI><LI>Gal4BD (C1-C5)</LI><LI>LexA: didn’t grow</LI></UL><P>Do the digestions of the minipreps:</P><DIV class="table-wrapper"><TABLE class="alt"><TBODY><TR><TD>LacIBD+PIF, a1</TD><TD>EcoRI</TD><TD>6345, 1997, 641</TD></TR><TR><TD>RepBxb1:GFP, ?2</TD><TD>HindIII</TD><TD>6345, 2683</TD></TR><TR><TD>Gal4BD; pUPD2</TD><TD>NotI</TD><TD>2681, 644</TD></TR><TR><TD>N-Dronpa; pUPD2</TD><TD>NotI</TD><TD>2046, 744</TD></TR></TBODY></TABLE><P>Make the gel.</P><DIV class="table-wrapper"><TABLE class="alt"><TBODY><TR><TD>RepBxb1:GFP C1</TD><TD>RepBxb1:GFP C2</TD><TD>LacIBD+PIF C1</TD><TD>LacIBD+PIF C2</TD><TD>LacIBD+PIF C3</TD><TD>LacIBD+PIF C4</TD></TR><TR><TD>no</TD><TD>no</TD><TD>no</TD><TD>no</TD><TD>no</TD><TD>No</TD></TR><TR><TD>Gal4BD C1</TD><TD>Gal4BD C2</TD><TD>Gal4BD C3</TD><TD>Gal4BD C4</TD><TD>Gal4BD C5</TD><TD>N-Dronpa C1</TD></TR><TR><TD>ok</TD><TD>ok</TD><TD>ok</TD><TD>ok</TD><TD>ok</TD><TD>ok</TD></TR><TR><TD>N-Dronpa C2</TD><TD>N-Dronpa C3</TD><TD>N-Dronpa C4</TD></TR><TR><TD>no</TD><TD>ok</TD><TD>ok</TD></TR></TBODY></TABLE><P>Take glycerinated:</P><UL><LI>GB0030: p35S</LI><LI>GB0036: T35S</LI></UL><UL><LI>Make liquid culture of LexABD (C1-C4).</LI><LI>We transform again LacIBD+K-Dronpa and RepBxb1:GFP, adding to the agar plates 100 µl of each transformation. </LI></UL><H3 style="color:green">29 June 2015</H3><P>Do the minipreps of the 4 colonies of LexABD and both glycerinates, 35S and T35S.</P><P>Do the digestion of the 4 colonies of LexA and both glycerinates:</P><DIV class="table-wrapper"><TABLE class="alt"><TBODY><TR><TD>LexABD; pPPD2</TD><TD>NotI</TD><TD>2358, 312</TD></TR><TR><TD>35S; pUPD2</TD><TD>NotI</TD><TD>2981, 1074</TD></TR><TR><TD>T35S; pPUD2</TD><TD>NotI</TD><TD>2981, 304</TD></TR></TBODY></TABLE><P>Make the gel:</P><DIV class="table-wrapper"><TABLE class="alt"><TBODY><TR><TD>LexA C1</TD><TD>LexA C2</TD><TD>LexA C3</TD><TD>LexA C4</TD><TD>P35S</TD><TD>T35S</TD></TR><TR><TD>ok</TD><TD>ok</TD><TD>ok</TD><TD>ok</TD><TD>Ok?</TD><TD>Ok?</TD></TR></TBODY></TABLE><P>Make ligations:</P><DIV class="table-wrapper"><TABLE class="alt"><TBODY><TR><TD>LacIBD+K-Dronpa+promoter+termi; a1</TD><TD>Gal4BD+K-Donpa+prom+ter; a1</TD><TD>LexABD+K-Dronpa+prom+term; a1</TD></TR><TR><TD>1 µl LacI, pUPD2</TD><TD>1 µl Gal4, pUPD2</TD><TD>1 µl Gal4, pUPD2</TD></TR><TR><TD>1 µl K-Dronpa, pUPD2</TD><TD>1 µl K-Dronpa, pUPD2</TD><TD>1 µl K-Dronpa, pUPD2</TD></TR><TR><TD>1 µl 35S (GB0030)</TD><TD>1 µl 35S (GB0030)</TD><TD>1 µl 35S (GB0030)</TD></TR><TR><TD>1 µl T35S (GB0036)</TD><TD>1 µl T35S (GB0036)</TD><TD>1 µl T35S (GB0036)</TD></TR><TR><TD>1.2 µl  buffer ligase</TD><TD>1.2 µl  buffer ligase</TD><TD>1.2 µl  buffer ligase</TD></TR><TR><TD>1.2 µl BSA 10x</TD><TD>1.2 µl BSA 10x</TD><TD>1.2 µl BSA 10x</TD></TR><TR><TD>1 µl BsaI</TD><TD>1 µl BsaI</TD><TD>1 µl BsaI</TD></TR><TR><TD>1 µl ligase T4</TD><TD>1 µl ligase T4</TD><TD>1 µl ligase T4</TD></TR><TR><TD>2.6 µl H2O</TD><TD>2.6 µl H2O</TD><TD>2.6 µl H2O</TD></TR><TR><TD>1 µl a1</TD><TD>1 µl a1</TD><TD>1 µl a1</TD></TR><TR><TD>N-Dronpa+VP16; a2</TD><TD>Gal4BD+PIF6; a1</TD><TD>LacIBD+PIF6; a1</TD></TR><TR><TD>1 µl N-Dronpa, pUPD2</TD><TD>1 µl Gal4BD, pUPD2</TD><TD>1 µl LacIBD, pUPD2</TD></TR><TR><TD>1 µl VP16, pUPD2</TD><TD>1 µl PIF6, pUPD2</TD><TD>1 µl PIF6, pUPD2</TD></TR><TR><TD>1 µl 35S (GB0030)</TD><TD>1 µl 35S (GB0030)</TD><TD>1 µl 35S (GB0030)</TD></TR><TR><TD>1 µl T35S (GB0036)</TD><TD>1 µl T35S (GB0036)</TD><TD>1 µl T35S (GB0036)</TD></TR><TR><TD>1.2 µl  buffer ligase</TD><TD>1.2 µl  buffer ligase</TD><TD>1.2 µl  buffer ligase</TD></TR><TR><TD>1.2 µl BSA 10x</TD><TD>1.2 µl BSA 10x</TD><TD>1.2 µl BSA 10x</TD></TR><TR><TD>1 µl BsaI</TD><TD>1 µl BsaI</TD><TD>1 µl BsaI</TD></TR><TR><TD>1 µl ligase T4</TD><TD>1 µl ligase T4</TD><TD>1 µl ligase T4</TD></TR><TR><TD>2.6 µl H2O</TD><TD>2.6 µl H2O</TD><TD>2.6 µl H2O</TD></TR><TR><TD>1 µl a2</TD><TD>1 µl a1</TD><TD>1 µl a1</TD></TR><TR><TD>LexABD+PIF6; a1</TD></TR><TR><TD>1 µl LexABD, pUPD2</TD></TR><TR><TD>1 µl PIF, pUPD2</TD></TR><TR><TD>1 µl 35S (GB0030)</TD></TR><TR><TD>1 µl T35S (GB0036)</TD></TR><TR><TD>1.2 µl  buffer ligase</TD></TR><TR><TD>1.2 µl BSA 10x</TD></TR><TR><TD>1 µl BsaI</TD></TR><TR><TD>1 µl ligase T4</TD></TR><TR><TD>2.6 µl H2O</TD></TR><TR><TD>1 µl a2</TD></TR></TBODY></TABLE><UL><LI>Transform all the ligations into E.Coli.Gal4BD+K-Dronpa and LacIBD+K-Dronpa goes wrong and we have to do it again. </LI><LI>Put into a plate the colonies before let stay them 1h at 37ºC.</LI></UL><P>Quantification of DNA:</P><UL><LI>RepBxb1:GFP (C1): 163.8 ng/µl</LI><LI>N-Dronpa, pUPD2 (C4):113.1 ng/µl</LI><LI>N-Dronpa (C3): 83.2 ng/µl</LI><LI>NDonpa (C1): 116.6 ng/µl</LI><LI>Gal4BD (C1): 95.2 ng/µl</LI><LI>Gal4BD (C2): 120.7 ng/µl</LI><LI>RepBxb1:GFP (C2): 170.6 ng/µl</LI><LI>RepBxb1 (C1): 80.6 ng/µl</LI></UL><H3 style="color:green">30 June 2015</H3><P>Transform Gal4+K-Dronpa and LacI+K-Dronpa</P><P>Miniprep of:</P><UL><LI>RepBxb1+GFP (C1-C3)</LI></UL><DIV class="table-wrapper"><TABLE class="alt"><TBODY><TR><TD>RepBxb1+GFP</TD><TD>HindIII</TD><TD>6345, 2683</TD></TR></TBODY></TABLE><P>Gel:</P><DIV class="table-wrapper"><TABLE class="alt"><TBODY><TR><TD>RepBxb1+GFP C1</TD><TD>RepBxb1+GFP C2</TD><TD>RepBxb1+GFP C3</TD></TR><TR><TD>No</TD><TD>no</TD><TD>no</TD></TR></TBODY></TABLE><P>We pick more colonies and make other liquid cultures.</P><P>Save the last samples of the leaves of the plants that were under the first experiment, next to the glycerinates in the freezer (-80ºC). </P><P>Put in plates the transformations of Gal4+K-Dronpa and LacI+K-Dronpa (50 µl of bacteria in SOC medium).</P><P>Make the gel:</P><UL><LI>Add to the digestions 2 µl of loading buffer (6x) because for 5 µl of digestion we put 1 µl of the buffer.</LI></UL><P>Make liquid culture of two colonies for each plates of the transformations done yesterday, 10 tubes in total.</P><P>Take out a glycerinate 35S:Luciferase:Tnos (GB0227) and do a miniprep.</P><H3 style="color:green">1 July 2015</H3><P>Do the minipreps of the 10 liquid cultures.</P><P>Do the digestions:</P><DIV class="table-wrapper"><TABLE class="alt"><TBODY><TR><TD>LexABD+K-Dronpa;a1</TD><TD>EcoRI</TD><TD>6345, 2296</TD></TR><TR><TD>N-Donpa+VP16; a2</TD><TD>HindIII</TD><TD>6345, 2427</TD></TR><TR><TD>Gal4BD+PIF6; a1</TD><TD>EcoRI</TD><TD>6345, 1867</TD></TR><TR><TD>LacI+PIF; a1</TD><TD>EcoRI</TD><TD>6345, 2638</TD></TR><TR><TD>LexABD+PIF6; a1</TD><TD>EcoRI</TD><TD>6345, 1906</TD></TR></TBODY></TABLE><P>Do the gel:</P><DIV class="table-wrapper"><TABLE class="alt"><TBODY><TR><TD>Gal4+PIF C1</TD><TD>Gal4+PIF C2</TD><TD>LexA+PIF C1</TD><TD>LexA+PIF C2</TD><TD>LexA+KDronpa C1</TD></TR><TR><TD>ok</TD><TD>ok</TD><TD>ok</TD><TD>ok</TD><TD>Ok</TD></TR><TR><TD>LexA+Kdronpa C2</TD><TD>LacI+PIF C1</TD><TD>LacI+PIF C2</TD><TD>Ndonpa+VP16 C1</TD><TD>Ndronpa+VP16 C2</TD></TR><TR><TD>ok</TD><TD>ok</TD><TD>ok</TD><TD>ok</TD><TD>ok</TD></TR></TBODY></TABLE><UL><LI>Prepare liquid culture of:</LI></UL><UL><LI>LexA+PIF; ?1 (C1)</LI><LI>LacI+PIF; ?1 (C1)</LI><LI>LexA+K-Dronpa (C1)</LI><LI>Gal4+PIF; ?1 (C1)</LI><LI>VP16, pUPD2 (C1)</LI><LI>LexABD, pUPD2 (C2)</LI><LI>PIF6, pUPD2 (C5)</LI><LI>LacIBD, pUPD2 (C1)</LI></UL><UL><LI>Pick colonies and make liquid culture of:</LI><UL class="ul_2"><LI>Gal4+K-Dronpa (C1 and C2)</LI><LI>LacI+K-Dronpa (C1 and C2)</LI><LI>RepBxb1+GFP (C4-C6)</LI></UL><LI>We sent to sequence:</LI><UL class="ul_2"><LI>Code:</LI><LI>210.08-249: pUPD2, KDronpa C3</LI><LI>210.08-250: pUPD2, NDronpa C1</LI><LI>210.08-251: pUPD2, NDronpa C3</LI><LI>210.08-252: pUPD2, NDronpa C4</LI><LI>The solution have: 10µl of miniprep + 5µl (dilution 1:3) of primers.</LI></UL></UL><UL><LI>Data of the luciferase essay with the sample plat RED at 24h (replica2): we obtain a value of 7.4E6.</LI></UL><H3 style="color:green">2 July 2015</H3><P>Minipreps of:</P><UL><LI>Gal4+K-Dronpa (C1 and C2)</LI><LI>LacI+K-Dronpa (C1 and C2)</LI><LI>RepBxb1+GFP (C4-C6)</LI></UL><DIV class="table-wrapper"><TABLE class="alt"><TBODY><TR><TD>Gal4+K-Dronpa</TD><TD>EcoRI</TD><TD>6345, 3028</TD></TR><TR><TD>LacI+K-Dronpa</TD><TD>EcoRI</TD><TD>6345, 2257</TD></TR><TR><TD>RepBxb1+GFP</TD><TD>HindIII</TD><TD>6300, 2400</TD></TR></TBODY></TABLE><P>Do the gel:</P><DIV class="table-wrapper"><TABLE class="alt"><TBODY><TR><TD>LacI+K-Dronpa C1</TD><TD>LacI+K-Dronpa C2</TD><TD>Gal4+K-Dronpa C1</TD><TD>Gal4+K-Dronpa C2</TD></TR><TR><TD>??</TD></TR><TR><TD>RepBxb1+GFP C4</TD><TD>RepBxb1+GFP C5</TD><TD>RepBxb1+GFP C6</TD></TR></TBODY></TABLE><P>Luciferase essay: </P><UL><LI>During the whole experiment we lost three samples: T16/FarRed/1; T24/Red/2 and T0/FarRed/1</LI></UL><P>Results:</P><P>Things to keep in mind for the next experiment:</P><UL><LI>The luminimeter (machine to measure the luminescence) has to be ready before start adding the reactants to the samples because it needs 10min to be ready.</LI><LI>Set the timer (10min) with the first sample of luciferase and add the reactant to the other samples as quick as possible. </LI><LI>We have to let the renilla stay before putting it in the luminimeter the same time as the luciferase, in this case 15min because with the luciferase we didn’t manage well the time. Theoretically we have to wait 10min.</LI></UL><P>Make ligations:</P><DIV class="table-wrapper"><TABLE class="alt"><TBODY><TR><TD>LexA:Kdonpa+N-Dronpa; ?1</TD><TD>LacI:Kdronpa+N-Dronpa; ?1</TD><TD>Gal4:Kdronpa+N-Dronpa; ?1</TD></TR><TR><TD>1 µl LexA:Kdronpa</TD><TD>1 µl LacI:Kdronpa</TD><TD>1 µl Gal4:Kdronpa</TD></TR><TR><TD>1 µl N-Dronpa</TD><TD>1 µl N-Dronpa</TD><TD>1 µl N-Dronpa</TD></TR><TR><TD>1 µl ?1</TD><TD>1 µl ?1</TD><TD>1 µl ?1</TD></TR><TR><TD>1.2 µl buffer ligase</TD><TD>1.2 µl buffer ligase</TD><TD>1.2 µl buffer ligase</TD></TR><TR><TD>1.2 µl BSA</TD><TD>1.2 µl BSA</TD><TD>1.2 µl BSA</TD></TR><TR><TD>1 µl BsmbI</TD><TD>1 µl BsmbI</TD><TD>1 µl BsmbI</TD></TR><TR><TD>1 µl BsaI</TD><TD>1 µl BsaI</TD><TD>1 µl BsaI</TD></TR><TR><TD>4.6 µl H2O</TD><TD>4.6 µl H2O</TD><TD>4.6 µl H2O</TD></TR><TR><TD>LexA:PIF+PhyB:VP16; ?1</TD><TD>LacI:PIF+PhyB:VP16; ?1</TD><TD>Gal4:PIF+PhyB:VP16; ?1</TD></TR><TR><TD>1 µl LexA:PIF</TD><TD>1 µl LacI:PIF</TD><TD>1 µl Gal4:PIF</TD></TR><TR><TD>1 µl PhyB:VP16 (88E)</TD><TD>1 µl PhyB:VP16</TD><TD>1 µl PhyB:VP16</TD></TR><TR><TD>1 µl ?1</TD><TD>1 µl ?1</TD><TD>1 µl ?1</TD></TR><TR><TD>1.2 µl buffer ligase</TD><TD>1.2 µl buffer ligase</TD><TD>1.2 µl buffer ligase</TD></TR><TR><TD>1.2 µl BSA</TD><TD>1.2 µl BSA</TD><TD>1.2 µl BSA</TD></TR><TR><TD>1 µl BsmbI</TD><TD>1 µl BsmbI</TD><TD>1 µl BsmbI</TD></TR><TR><TD>1 µl BsaI</TD><TD>1 µl BsaI</TD><TD>1 µl BsaI</TD></TR><TR><TD>4.6 µl H2O</TD><TD>4.6 µl H2O</TD><TD>4.6 µl H2O</TD></TR><TR><TD>35S:Bxb1+RepBxb1:GFP; ?1</TD><TD>E-PIF+phyB+luc+ren; ?1</TD></TR><TR><TD>1 µl 35s:Bxb1:T35S (alfredo’s)</TD><TD>0.5 µl 896 (PIF+phy+luc)</TD></TR><TR><TD>1 µl PromsinATG:RepBxb1:GFP</TD><TD>1 µl 160 (renilla)</TD></TR><TR><TD>1 µl ?1</TD><TD>1 µl ?1</TD></TR><TR><TD>1.2 µl buffer ligase</TD><TD>1.2 µl buffer ligase</TD></TR><TR><TD>1.2 µl BSA</TD><TD>1.2 µl BSA</TD></TR><TR><TD>1 µl BsmbI</TD><TD>1 µl BsmbI</TD></TR><TR><TD>1 µl BsaI</TD><TD>1 µl BsaI</TD></TR><TR><TD>4.6 µl H2O</TD><TD>4.6 µl H2O</TD></TR></TBODY></TABLE><P>The samples that we sent to sequence have arrived:</P><UL><LI>210.08-249: pUPD2, KDronpa C3------ok</LI><LI>210.08-250: pUPD2, NDronpa C1------ok</LI><LI>210.08-251: pUPD2, NDronpa C3------ok</LI><LI>210.08-252: pUPD2, NDronpa C4------ok</LI></UL><P>The sequences of N-Dronpa have the desired mutation.</P><UL><LI>Transformation in E.Coli of the 8 ligations.</LI><LI>Put the transformation into plates, put at 37ºC.</LI><LI>Refresh the Agro’s cultures (Renilla and PIF+phy+luc):</LI><LI>Add in 5ml of LB, 5 µl of rifampicine, 5 µl kanamicine, 5 µl of culture.</LI></UL><H3 style="color:green">4 July 2015</H3><P>Make the 2nd refresh of the culture of <I>Agrobacterium</I>. Put 5ml of LB, 5 µl of rifampicine, 5 µl kanamicine, 5 µl of culture.</P><P>Make liquid cultures (4ml of LB and 4 µl of spectomicine) of the 8 colonies of E.Coli. </P><UL><LI>Red toggle (C1)</LI><LI>Gal4:Kdronpa+N-Dronpa (C1 and C2)</LI><LI>LexA:Kdonpa+N-Dronpa (C1 and C2)</LI><LI>LacI:Kdronpa+N-Dronpa (C1 and C2)</LI><LI>LexA:PIF+PhyB:VP16 (C1 and C2)</LI><LI>35S:Bxb1+RepBxb1:GFP (C1 and C2)</LI><LI>This colonies didn’t grow: LacI:PIF+PhyB:VP16; Gal4:PIF+PhyB:VP16 and E-PIF+phyB+luc+ren. Tomorrow we will repeat the ligations.</LI></UL><H3 style="color:green">5 July 2015</H3><P>Ligations:</P><DIV class="table-wrapper"><TABLE class="alt"><TBODY><TR><TD>LacI:PIF+PhyB:VP16; ?1</TD><TD>Gal4:PIF+PhyB:VP16; ?1</TD></TR><TR><TD>1 µl LacI:PIF</TD><TD>1 µl Gal4:PIF</TD></TR><TR><TD>1 µl PhyB:VP16</TD><TD>1 µl PhyB:VP16</TD></TR><TR><TD>1 µl ?1</TD><TD>1 µl ?1</TD></TR><TR><TD>1.2 µl buffer ligase</TD><TD>1.2 µl buffer ligase</TD></TR><TR><TD>1.2 µl BSA</TD><TD>1.2 µl BSA</TD></TR><TR><TD>1 µl BsmbI</TD><TD>1 µl BsmbI</TD></TR><TR><TD>1 µl T4 ligase</TD><TD>1 µl T4 ligase</TD></TR><TR><TD>4.6</TD><TD>µl H2O</TD><TD>4.6 µl H2O</TD></TR></TBODY></TABLE><P>Minipreps of the liquid cultures.</P><P>Digestion of the minipreps.</P><DIV class="table-wrapper"><TABLE class="alt"><TBODY><TR><TD>LacI:Kdronpa+N-Dronpa</TD><TD>BamHI</TD><TD>6674, 5437</TD></TR><TR><TD>35S:Bxb1+RepBxb1:GFP</TD><TD>BamHI</TD><TD>6674, 3859, 1782</TD></TR><TR><TD>Gal4:Kdronpa+N-Dronpa</TD><TD>BamHI</TD><TD>6674, 4666</TD></TR><TR><TD>LexA:Kdonpa+N-Dronpa</TD><TD>BamHI</TD><TD>6674, 4705</TD></TR><TR><TD>LexA:PIF+PhyB:VP16</TD><TD>BamHI</TD><TD>6674, 3513, 2337</TD></TR></TBODY></TABLE><UL><LI>Agarose gel (1%): </LI></UL><DIV class="table-wrapper"><TABLE class="alt"><TBODY><TR><TD>LacKN C1</TD><TD>LacIKN C2</TD><TD>Bxb1RepGFP C1</TD><TD>Bxb1RepGFP C2</TD><TD>Gal4KN C1</TD><TD>Gal4KN C2</TD></TR><TR><TD>ok</TD><TD>ok</TD><TD>ok</TD><TD>ok</TD><TD>ok</TD><TD>ok</TD></TR><TR><TD>LexA:KN C1</TD><TD>LexA:KN C2</TD><TD>LexAPIFPhy C1</TD><TD>LexAPIFPhy C2</TD><TD>Red toggle</TD></TR><TR><TD>ok</TD><TD>ok</TD><TD>no</TD><TD>no</TD><TD>no</TD></TR></TBODY></TABLE><P>We have to repeat the digestion of: LexA+PIF:phy+VP16.</P><UL><LI>Take out the glycerinate 88C (1098): Etr8:luc:Tnos. We will use it like a negative control in the second luciferase essat.</LI><LI>Calculation of the ODs:</LI><UL class="ul_2"><LI>Dilution of both samples 1:10.</LI><LI>Renilla:=0.22---182 µl of sample + 1.818 ml MES</LI><LI>PIF+PhyB+Luc=0.26--- 154 µl of sample + 1.646 ml MES</LI></UL></UL><P>Agroinfiltration of the two samples with renilla and luciferase+PIF+phy in three plants with 4 spots in each leaf and 2 leaf in each plant.</P><P>Let the plants 2 days in the darkness till agrobacterium infects the plant. They have to be in the dark because we are trying our red toggle ant it activates with light.</P><H3 style="color:green">6 July 2015</H3><P>Transform the negative control into agrobacterium.</P><UL><LI>Etr8:luc:Tnos</LI><LI>Bxb1:reporterBxb1:GFP</LI></UL><P>We were doing dry lab preparing the power point to present our project to the rector and biotecs companies.</P><H3 style="color:green">7 July 2015</H3><P>Luciferase essay: Copy the protocol.</P><UL><LI>Add 150 µl of the lisis buffer and 800µl of MiliQ water, dilution 1:5.</LI><LI>We centrifuge both cultures of agrobacterium at 2900rpm for 10min and remove the supernatant. </LI><LI>We prepare the stock of MES (10ml of MES + 1ml MgCl + 100µl of “Acetosiningona” and level with H2O until 100ml.</LI><LI>Resuspend the pellet of bacteria with 5ml of MES and let grow 2h.</LI></UL><UL><LI>Renilla=0.29 (dilution 1:10): 2.9</LI><LI>PhyB+PIF+luc=0.69 (dilution 1:4):2.76</LI><LI>Solution to agroinflitrate:</LI><LI>Renilla: 0.138ml of sample + 1.862ml of MES</LI><LI>PhyB+PIF+luc: 0.145ml of sample + 1.855ml of MES</LI><LI>We make the infiltration of both samples mix together in 3 different plants, 2 leaf per plant and 4 spots per leaf. Explicacion del experiment (tiempo en oscuridad… luces…)</LI></UL><P>Digestions:</P><DIV class="table-wrapper"><TABLE class="alt"><TBODY><TR><TD>Red toggle</TD><TD>Enzyme?</TD></TR><TR><TD>LexA+PIF+phy</TD><TD>BamHI</TD><TD>3518, 5855, 6674</TD></TR></TBODY></TABLE><P>Gel:</P><DIV class="table-wrapper"><TABLE class="alt"><TBODY><TR><TD>Red toggle</TD><TD>LexA+PIF+phy C1</TD><TD>LexA+PIF+phy C2</TD></TR><TR><TD>No</TD><TD>Ok</TD><TD>no</TD></TR></TBODY></TABLE><P>It has arrived a new construction: AsLOVpep.</P><UL><LI>Suspended with 50µl of H2O.</LI></UL><P>Ligations:</P><DIV class="table-wrapper"><TABLE class="alt"><TBODY><TR><TD>AsLOVpep; pUPD2</TD><TD>Red Toggle</TD><TD>LacI:Kdronpa:Ndronpa:VP16+</TD></TR><TR><TD>35S:renilla:Tnos-35S:P19:Tnos(GB159)</TD><TD>Gal4:Kdronpa:Ndronpa:VP16+GB159</TD></TR><TR><TD>1 µl AsLOVpep</TD><TD>1 µl GB846</TD><TD>1 µl LacI:KNdronpa:VP16</TD><TD>1 µl Gal4:KNdronpa</TD></TR><TR><TD>1 µl pUPD2</TD><TD>1 µl GB160</TD><TD>1 µl GB159</TD><TD>1 µl GB159</TD></TR><TR><TD>1.2 µl buffer</TD><TD>1 µl ?1</TD><TD>1 µl a1</TD><TD>1 µl a1</TD></TR><TR><TD>1.2 µl BSA</TD><TD>1.2 µl buffer</TD><TD>1.2 µl buffer</TD><TD>1.2 µl buffer</TD></TR><TR><TD>1 µl BsmbI</TD><TD>1.2 µl BSA</TD><TD>1.2 µl BSA</TD><TD>1.2 µl BSA</TD></TR><TR><TD>1 µl T4 ligase</TD><TD>1 µl BsmbI</TD><TD>1 µl BsmbI</TD><TD>1 µl BsmbI</TD></TR><TR><TD>5.6 µl H2O</TD><TD>1 µl T4 ligase</TD><TD>1 µl T4 ligase</TD><TD>1 µl T4 ligase</TD></TR><TR><TD>4.6 µl H2O</TD><TD>4.6 µl H2O</TD><TD>4.6 µl H2O</TD></TR><TR><TD>LexA:Kdronpa:Ndronpa:VP16+GB159</TD><TD>LexA:PIF:Phy:VP16+ GB159</TD></TR><TR><TD>1 µl LexA:Kdronpa:Ndronpa:VP16</TD><TD>1 µl LexA:PIF:Phy:VP16</TD></TR><TR><TD>1 µl GB159</TD><TD>1 µl GB159</TD></TR><TR><TD>1 µl a1</TD><TD>1 µl a1</TD></TR><TR><TD>1.2 µl buffer</TD><TD>1.2 µl buffer</TD></TR><TR><TD>1.2 µl BSA</TD><TD>1.2 µl BSA</TD></TR><TR><TD>1 µl BsmbI</TD><TD>1 µl BsmbI</TD></TR><TR><TD>1 µl T4 ligase</TD><TD>1 µl T4 ligase</TD></TR><TR><TD>4.6 µl H2O</TD><TD>4.6 µl H2O</TD></TR></TBODY></TABLE><H3 style="color:green">8 July 2015</H3><UL><LI>Experiment to study the piece PhyB+PIF.</LI></UL><P>	The plants in red are exposed at the light intensity of the leds (we can’t regulate it) and the plants in far red are 100% with far red light and 0% of white light.</P><P>	How the machines work: (hace falta?)</P><P>	The machine with red light has the switch …</P><P>	The 1st sample its at 8:00am and we spend 1h till the machines were working correctly because we didn’t know exactly how they work.</P><P>	Then, before 12h (21:00), we take the 2nd samples; obtainin 3 discs of each condition (red an far red). </P><P>We transform:</P><UL><LI>AsLOVpep; pUPD2 in DHSa an let incubate 1h at 37ºC.</LI><LI>The last ligations in E.coli. Put in plates. Red toggle with Spectomicine+IPTG+XGal ann the rest (a1) with kanamicine+IPTG+XGal.</LI></UL><H3 style="color:green">9 July 2015</H3><P>Pick colonies:</P><UL><LI>AsLOVpep colonies didn’t grow. Repeat.</LI><LI>Red toggle colonies are all blue. Repeat.</LI><LI>The other 4 colonies had grown. we pick them and male liquid culture.</LI><LI>LacI:Kdronpa:Ndronpa:VP16+renilla (C1-C3)</LI><LI>Gal4:Kdronpa:Ndronpa:VP16+renilla (C1-C3)</LI><LI>LexA:Kdronpa:Ndronpa:VP16+GB159 (C1 and C2)</LI><LI>LexA:PIF:Phy:VP16+ GB159 (C1 and C2)</LI></UL><P>Repeat the ligations.</P><UL><LI>AsLOVpep, as before.</LI></UL><DIV class="table-wrapper"><TABLE class="alt"><TBODY><TR><TD>Red toggle (PIF+PhyB+luc+ren)</TD></TR><TR><TD>0.5 µl PIF+phy+luc (896)</TD></TR><TR><TD>1 µl renilla (160)</TD></TR><TR><TD>1 µl ?1</TD></TR><TR><TD>1.2 µl buffer</TD></TR><TR><TD>1.2 µl BSA</TD></TR><TR><TD>1 µl BsmbI</TD></TR><TR><TD>1 µl T4 ligase</TD></TR><TR><TD>5.1 µl H2O</TD></TR></TBODY></TABLE><P>We do the luciferase essay:</P><UL><LI>We didn’t obtain goo results, we can’t observed a significative difference between the on(red samples) and off (far red samples). It seems that the red toggle it has been activated. Graphics and tables</LI></UL><P>Transform the ligations of AsLOVpepe and red toggle. </P><UL><LI>Add SOC medium and let incubate 1h at 37ºC. Then we make an spin to the red toggle cells to concentrate them and see if we can obtain a colonies in the plates.</LI></UL><H3 style="color:green">10 July 2015</H3><P>Minipreps of:</P><UL><LI>LexABD:PIF:phyB:VP16+Renilla; a1 (C1,C2)</LI><LI>LexA:Kdronpa:Ndronpa+renilla; a1C1, C2)</LI><LI>Gal4:Kdronpa:Ndronpa+renilla; a1 (C1-C3)</LI><LI>LacI:Kdronpa:Ndronpa+renilla; a1 (C1,C2)</LI></UL><P>Digestions of:</P><DIV class="table-wrapper"><TABLE class="alt"><TBODY><TR><TD>LexABD:PIF:phyB:VP16+Renilla; a1 (C1,C2)</TD><TD>EcoRI</TD><TD>6345, 5487, 4891</TD></TR><TR><TD>LexA:Kdronpa:Ndronpa+renilla; a1C1, C2)</TD><TD>EcoRI</TD><TD>9333, 6345</TD></TR><TR><TD>Gal4:Kdronpa:Ndronpa+renilla; a1 (C1-C3)</TD><TD>EcoRI</TD><TD>6345, 9194</TD></TR><TR><TD>LacI:Kdronpa:Ndronpa+renilla; a1 (C1,C2)</TD><TD>EcoRI</TD><TD>6345, 9965</TD></TR><TR><TD>LacIBD+PIF+phyB; Ω1 (C1)</TD><TD>BamHI</TD><TD>6674, 4245, 2337</TD></TR><TR><TD>Gal4BD+PIF+phyB; Ω 1 (C1)</TD><TD>BamHI</TD><TD>6674, 3474, 2337</TD></TR></TBODY></TABLE><P>Make the gel:</P><DIV class="table-wrapper"><TABLE class="alt"><TBODY><TR><TD>LacIBD+PIF+phyB</TD><TD>Gal4BD+PIF+phyB</TD><TD>LexA:PIF:phyB:VP16+Renilla C1</TD><TD>LexA:PIF:phyB:VP16+Renilla C2</TD></TR><TR><TD>Ok</TD><TD>Ok</TD><TD>ok</TD><TD>ok</TD></TR><TR><TD>LexA:KNdronpa+renilla C1</TD><TD>LexA:KNdronpa+renilla C2</TD><TD>Gal4:KNdronpa+renilla C1</TD><TD>Gal4:KNdronpa+renilla C2</TD></TR><TR><TD>Ok</TD><TD>Ok</TD><TD>ok</TD><TD>Ok</TD></TR><TR><TD>Gal4:KNdronpa+renilla C3</TD><TD>LacI:Kdronpa:Ndronpa+renilla C1</TD><TD>LacI:Kdronpa:Ndronpa+renilla C2</TD></TR><TR><TD>Ok</TD><TD>Ok</TD><TD>ok</TD></TR></TBODY></TABLE><P>We received two constructions:</P><UL><LI>CDS: phiC31. Resuspended with 100µl.</LI><LI>Reporter phi31. Resuspended with 50 µl</LI></UL><P>Pick colonies and make liquid culture of:</P><UL><LI>AsLOVpep; pUPD2 (C1 and C2)</LI><LI>Red toggle (C1 and C2). In both liquid cultures we ad YPTG and Xgal to make sure that the colonies are correct, if not the medium will change to blue color.</LI></UL><P>Ligations:</P><DIV class="table-wrapper"><TABLE class="alt"><TBODY><TR><TD>phyC31;pUPD2</TD><TD>Reporter phiC31; pUPD2</TD><TD>LacI:PIF:Phy:VP16+ren; a1</TD></TR><TR><TD>1 µl phiC31</TD><TD>1 µl rep phiC31</TD><TD>1 µl LacI:PIF:Phy:VP16</TD></TR><TR><TD>1 pUPD2</TD><TD>1 pUPD2</TD><TD>1 µl renilla (159)</TD></TR><TR><TD>1.2 µl buffer</TD><TD>1.2 µl buffer</TD><TD>1.2 µl buffer</TD></TR><TR><TD>1.2 µl BSA</TD><TD>1.2 µl BSA</TD><TD>1.2 µl BSA</TD></TR><TR><TD>1 µl T4 ligase</TD><TD>1 µl T4 ligase</TD><TD>1 µl T4 ligase</TD></TR><TR><TD>1 µl BsmbI</TD><TD>1 µl BsmbI</TD><TD>1 µl BSAI</TD></TR><TR><TD>5.6 µl H2O</TD><TD>5.6 µl H2O</TD><TD>4.6 µl H2O</TD></TR><TR><TD>1 µl a1</TD></TR><TR><TD>Gal4:PIF:phy:VP16+ren; a1</TD><TD>LexA:PIF:phy:ren+opLex:luc; ?1</TD><TD>LexA:KNdronpa:ren+OpLex:Luc; ?1</TD></TR><TR><TD>1 µl Gal4:PIF:phy:VP16</TD><TD>1 µl LexA:PIF:phy:ren</TD><TD>1 µl LexA:KNdronpa:ren</TD></TR><TR><TD>1 µl renilla (159)</TD><TD>1 µl opLex:luc (151)</TD><TD>1 µl OpLex:Luc (151)</TD></TR><TR><TD>1.2 µl buffer</TD><TD>1.2 µl buffer</TD><TD>1.2 µl buffer</TD></TR><TR><TD>1.2 µl BSA </TD><TD>1.2 µl BSA</TD><TD>1.2 µl BSA</TD></TR><TR><TD>1 µl T4 ligase</TD><TD>1 µl T4 ligase</TD><TD>1 µl T4 ligase</TD></TR><TR><TD>1 µl BSAI</TD><TD>1 µl BsmbI</TD><TD>1 µl BsmbI</TD></TR><TR><TD>4.6 µl H2O</TD><TD>4.6 µl H2O</TD><TD>4.6 µl H2O</TD></TR><TR><TD>1 µl a1</TD><TD>1 µl ?1</TD><TD>1 µl ?1</TD></TR><TR><TD>Gal4:KNdronpa:ren+UAS:luc; ?1</TD><TD>LacI:KNdronpa:ren+OpLacI:luc; ?1</TD></TR><TR><TD>1 µl Gal4:KNdronpa:ren</TD><TD>1 µl LacI:KNdronpa:ren</TD></TR><TR><TD>1 µl UAS:luc (227)</TD><TD>1 µl OpLacI:luc (152)</TD></TR><TR><TD>1.2 µl buffer</TD><TD>1.2 µl buffer</TD></TR><TR><TD>1.2 µl BSA </TD><TD>1.2 µl BSA</TD></TR><TR><TD>1 µl T4 ligase</TD><TD>1 µl T4 ligase</TD></TR><TR><TD>1 µl BsmbI</TD><TD>1 µl BsmbI</TD></TR><TR><TD>4.6 µl H2O</TD><TD>4.6 µl H2O</TD></TR><TR><TD>1 µl ?1</TD><TD>1 µl ?1</TD></TR></TBODY></TABLE><H3 style="color:green">11 July 2015</H3><P>Prepare glycerinates of:</P><UL><LI>Bxb1+Rep:GFP; ?1</LI><LI>N-Dronpa; pUPD2</LI><LI>Bxb1:Etr8; pUPD2</LI><LI>Etr8(CMV):Bxb1:T35S; a1</LI></UL><P>Pick up the liquid cultures of AsLOVpep and red toggle. We observed that one tube of a red toggle colony is blue, we discard it.</P><P>Do miniprep of:</P><UL><LI>AsLOVpep (C1 and C2)</LI><LI>Red toggle (C2)</LI></UL><P>Digestions:</P><DIV class="table-wrapper"><TABLE class="alt"><TBODY><TR><TD>Red toggle</TD><TD>BamHI</TD><TD>6674, 6100 / 4209, 3756</TD></TR><TR><TD>AsLOVpep</TD><TD>NotI</TD><TD>2558, 512</TD></TR></TBODY></TABLE><P>Me falta el resultado del gel!!</P><DIV class="table-wrapper"><TABLE class="alt"><TBODY><TR><TD>AsLOVpep C1</TD><TD>AsLOVpep C2</TD><TD>Red toggle C2</TD></TR><TR><TD>¿?</TD></TR></TBODY></TABLE><P>Do transformation of DHSa and yesterday ligations:</P><UL><LI>phyC31;pUPD2</LI><LI>Reporter phiC31; pUPD2</LI><LI>LacI:PIF:Phy:VP16+ren; a1</LI><LI>Gal4:PIF:phy:VP16+ren; a1</LI><LI>LexA:PIF:phy:ren+opLex:luc; ?1</LI><LI>LexA:KNdronpa:ren+OpLex:Luc; ?1</LI><LI>Gal4:KNdronpa:ren+UAS:luc; ?1</LI><LI>LacI:KNdronpa:ren+OpLacI:luc; ?1</LI><LI>The pUPD2 in plates with cloranfenicol+IPTG+XGal. The a1 in plates with kanamicyn+IPTG+XGal. The ?1 in plates with spectinmicyn+IPTG+XGal.</LI></UL><H3 style="color:green">12 July 2015</H3><P>Pick colonies and make liquid culture:</P><UL><LI>phyC31;pUPD2 (C1-C3)</LI><LI>Reporter phiC31; pUPD2 (C1-C3)</LI><LI>LacI:PIF:Phy:VP16+ren; a1 (C1-C3)</LI><LI>Gal4:PIF:phy:VP16+ren; a1  All blue colonies.</LI><LI>LexA:PIF:phy:ren+opLex:luc; ?1 (C1)</LI><LI>LexA:KNdronpa:ren+OpLex:Luc; ?1(C1-C3)</LI><LI>Gal4:KNdronpa:ren+UAS:luc; ?1 (C1)</LI><LI>LacI:KNdronpa:ren+OpLacI:luc; ?1 All blue colonies.</LI><LI>We have to repeat the transformations or do again ligations.</LI></UL><P>Ligations were repeated:</P><DIV class="table-wrapper"><TABLE class="alt"><TBODY><TR><TD>Gal4:PIF:phy:VP16+ren; a1</TD><TD>LacI:KNdronpa:ren+OpLacI:luc; ?1</TD></TR><TR><TD>1 µl Gal4:PIF:phy:VP16</TD><TD>1 µl LacI:KNdronpa:ren</TD></TR><TR><TD>1 µl renilla (159)</TD><TD>1 µl OpLacI:luc (152)</TD></TR><TR><TD>1.2 µl buffer</TD><TD>1.2 µl buffer</TD></TR><TR><TD>1.2 µl BSA </TD><TD>1.2 µl BSA</TD></TR><TR><TD>1 µl T4 ligase</TD><TD>1 µl T4 ligase</TD></TR><TR><TD>1 µl BSAI</TD><TD>1 µl BsmbI</TD></TR><TR><TD>4.6 µl H2O</TD><TD>4.6 µl H2O</TD></TR><TR><TD>1 µl a1</TD><TD>1 µl ?1</TD></TR></TBODY></TABLE><P>Refresh the liquid cultures of <I>Agrobacterium</I>:</P><UL><LI>Bxb1:GFP and Etr8:tnos. They were 2 days at 28ºC.</LI><LI>Pnos, it was at the fridge (-4ºC).</LI></UL><H3 style="color:green">13 July 2015</H3><P>All the liquid cultures have grown, do minipreps.</P><P>Do digestions:</P><DIV class="table-wrapper"><TABLE class="alt"><TBODY><TR><TD>phyC31;pUPD2 (C1-C3)</TD><TD>NotI</TD><TD>2046, 1899</TD></TR><TR><TD>Reporter phiC31; pUPD2 (C1-C3)</TD><TD>NotI</TD><TD>2046, 475</TD></TR><TR><TD>LacI:PIF:Phy:VP16+ren; a1 (C1-C3)</TD></TR><TR><TD>EcoRI</TD><TD>6345, 5623, 5487</TD></TR><TR><TD>LexA:PIF:phy:ren+opLex:luc; ?1 (C1)</TD></TR><TR><TD>BamHI</TD><TD>9431, 6674, 3531</TD></TR><TR><TD>LexA:KNdronpa:ren+OpLex:Luc; ?1(C1-C3)</TD></TR><TR><TD>BamHI</TD><TD>1199, 6674</TD></TR><TR><TD>Gal4:KNdronpa:ren+UAS:luc; ?1 (C1)</TD></TR><TR><TD>BamHI</TD><TD>11582, 6674</TD></TR></TBODY></TABLE><P>Agarose gel:</P><DIV class="table-wrapper"><TABLE class="alt"><TBODY><TR><TD>phyC31 C1</TD><TD>phyC31 C2</TD><TD>phyC31 C3</TD><TD>RepPhiC31 C1</TD></TR><TR><TD>no</TD><TD>no</TD><TD>no</TD><TD>ok</TD></TR><TR><TD>RepPhiC31 C2 </TD><TD>LacI:PIF:Phy:VP16+ren C1</TD><TD>LacI:PIF:Phy:VP16+ren C2</TD><TD>LacI:PIF:Phy:VP16+ren C3</TD></TR><TR><TD>no</TD><TD>no</TD><TD>ok</TD><TD>ok</TD></TR><TR><TD>LexA:PIF:phy:ren+opLex:luc</TD><TD>LexA:KNdronpa:ren+OpLex:Luc C1</TD><TD>LexA:KNdronpa:ren+OpLex:Luc C2</TD><TD>LexA:KNdronpa:ren+OpLex:Luc C3</TD></TR><TR><TD>no</TD><TD>ok</TD><TD>ok</TD><TD>ok</TD></TR><TR><TD>Gal4:KNdronpa:ren+UAS:luc C1</TD></TR><TR><TD>no</TD></TR></TBODY></TABLE><P>The <I>Agrobacterium</I> cultures refreshed yesterday were store in the fridge.</P><P>It is made another culture of 35S:Bxb1+reporterBxb1:GFP to keep it in the fridge. It had 5ml of LB medium, 5 µl rifampicin and 5 µl of spectinomicyn an 1 µl of the culture. </P><P>Mesurement of the OD’s:</P><UL><LI>35S:Bxb1+reporterBxb1:GFP: 0.28 (dilution 1:10)</LI><LI>143 µl of culture+1857 µl of MES/acetosiningon solution.</LI><LI>With this preparation 2 plants were infiltrated and let in natural light to see the normal activity of the recombinase.</LI></UL><P>Transformation of the ligation: Gal4:PIF:phy:VP16+ren; a1 and LacI:KNdronpa:ren+OpLacI:luc; ?1.</P><P>The liquid cultures of this constructions were repeated:</P><UL><LI>phyC31;pUPD2 (C4 and C5)</LI><LI>Reporter phiC31; pUPD2 (C4)</LI><LI>LacI:PIF:Phy:VP16+ren; a1 (C4)</LI><LI>LexA:PIF:phy:ren+opLex:luc; ?1 (C1)</LI><LI>LexA:KNdronpa:ren+OpLex:Luc; ?1(C1-C3)</LI><LI>Gal4:KNdronpa:ren+UAS:luc; ?1 (C1)</LI><LI>AsLOVpep; pUPD2 (C3)</LI></UL><H3 style="color:green">14 July 2015</H3><P>Do minipreps of:</P><UL><LI>phyC31;pUPD2 (C4 and C5)</LI><LI>Reporter phiC31; pUPD2 (C4)</LI><LI>LacI:PIF:Phy:VP16+ren; a1 (C4)</LI><LI>LexA:PIF:phy:ren+opLex:luc; ?1 (C1)</LI><LI>LexA:KNdronpa:ren+OpLex:Luc; ?1(C1-C3)</LI><LI>Gal4:KNdronpa:ren+UAS:luc; ?1 (C1)</LI><LI>AsLOVpep; pUPD2 (C3)</LI></UL><P>Do digestions:</P><DIV class="table-wrapper"><TABLE class="alt"><TBODY><TR><TD>phyC31;pUPD2</TD><TD>NotI</TD><TD>2046, 1899</TD></TR><TR><TD>Reporter phiC31; pUPD2</TD><TD>NotI</TD><TD>2046, 475</TD></TR><TR><TD>LacI:PIF:Phy:VP16+ren; a1</TD><TD>EcoRI</TD><TD>6345, 5623, 5487</TD></TR><TR><TD>LexA:PIF:phy:ren+opLex:luc, ?1</TD><TD>BamHI</TD><TD>9431, 6674, 3531</TD></TR><TR><TD>LexA:KNdronpa:ren+OpLex:Luc; ?1</TD><TD>BamHI</TD><TD>1199, 6674</TD></TR><TR><TD>Gal4:KNdronpa:ren+UAS:luc; ?1</TD><TD>BamHI</TD><TD>11582, 6674</TD></TR><TR><TD>AsLOVpep; pUPD2 </TD><TD>NotI</TD><TD>2558, 512</TD></TR></TBODY></TABLE><P>Make an agarose gel:</P><DIV class="table-wrapper"><TABLE class="alt"><TBODY><TR><TD>phyC31 (C4)</TD><TD>phyC31 (C5)</TD><TD>AsLOVpep (C4)</TD><TD>LexA:PIF:phy:ren+opLex:luc (C1)</TD></TR><TR><TD>no</TD><TD>no</TD><TD>No</TD><TD>No</TD></TR><TR><TD>LexA:KNdronpa:ren+OpLex:Luc (C3)</TD><TD>Gal4:KNdronpa:ren+UAS:luc (C1)</TD><TD>Gal4:KNdronpa:ren+UAS:luc(C2)</TD><TD>Gal4:KNdronpa:ren+UAS:luc (C3)</TD></TR><TR><TD>Ok</TD><TD>no</TD><TD>no</TD><TD>No</TD></TR><TR><TD>LacI:PIF:Phy:VP16+ren</TD><TD>Reporter phiC31 (C1)</TD></TR><TR><TD>no</TD><TD>Ok</TD></TR></TBODY></TABLE><P>Only LexA:KNdronpa:ren+OpLex:Luc and Reporter phiC31 were correct. Repeat the ligations because is the second digestion of this construction that were made.</P><UL><LI>Measurement of DNA concentration:</LI><UL class="ul_2"><LI>Reporter:phyC31: 13.6 ng/µl</LI><LI>PhyC31: 4.6 ng/µl </LI><LI>AsLOVpep: 30.3 ng/µl</LI><LI>Igem151 (op:LexAluc): 40 ng/µl</LI><LI>Gal4:PIF:phyB:ren (C1): 124.4 ng/µl</LI><LI>LacI:PIF:phyB:VP16 (C1): 126 ng/µl</LI><LI>Igem 159 (renilla): 42 ng/µl</LI><LI>Gal4:Kdronpa:Ndronpa:renilla (C3): 172.8 ng/µl</LI><LI>Igem 227 (op:UAS:luc): 6.3 ng/µl</LI></UL></UL><UL><LI>Pick colonies and make liquid culture of LacI:KDronpa:NDronpa:ren:luc (C4 and C5). The colonies of Gal4:PIF:phy:VP16+ren were all blue.</LI></UL><H3 style="color:green">15 July 2015</H3><UL><LI>Miniprep of LacI:KDronpa:NDronpa:ren:luc (C4 and C5). </LI><LI>Digestion:</LI></UL><DIV class="table-wrapper"><TABLE class="alt"><TBODY><TR><TD>LacI:KDronpa:NDronpa:ren:luc</TD><TD>EcoRI</TD><TD>6345, 9965</TD></TR></TBODY></TABLE><UL><LI>Make the gel:</LI></UL><DIV class="table-wrapper"><TABLE class="alt"><TBODY><TR><TD>LacI:K:NDronpa:ren:luc C4</TD><TD>LacI:K:NDronpa:ren:luc C5</TD></TR><TR><TD>no</TD><TD>no</TD></TR></TBODY></TABLE><UL><LI>Measurement of DNA concentrations:</LI><UL class="ul_2"><LI>Gal4BD:PIF:PhyB:VP16: 125.6 ng/µl</LI><LI>LexA:PIF:phyB:VP16:ren: 322.6 ng/µl</LI><LI>LacI:Kdronpa:NDronpa:ran: 209.0 ng/µl</LI></UL></UL><UL><LI>We decided to repeat the ligations due to that we make twice the digestions and we didn’t obtain good results. </LI></UL><DIV class="table-wrapper"><TABLE class="alt"><TBODY><TR><TD>phyC31;pUPD2</TD><TD>AsLOVpep; pUPD2</TD><TD>LacI:PIF:Phy:VP16+ren; a1</TD></TR><TR><TD>1 µl phiC31</TD><TD>1 µl AsLOVpep</TD><TD>1.5 µl LacI:PIF:Phy:VP16</TD></TR><TR><TD>1 pUPD2</TD><TD>1 pUPD2</TD><TD>2.5 µl renilla (159)</TD></TR><TR><TD>1.2 µl buffer</TD><TD>1.2 µl buffer</TD><TD>1.2 µl buffer</TD></TR><TR><TD>1.2 µl BSA</TD><TD>1.2 µl BSA</TD><TD>1.2 µl BSA</TD></TR><TR><TD>1 µl T4 ligase</TD><TD>1 µl T4 ligase</TD><TD>1 µl T4 ligase</TD></TR><TR><TD>1 µl BsmbI</TD><TD>1 µl BsmbI</TD><TD>1 µl BSAI</TD></TR><TR><TD>5.6 µl H2O</TD><TD>5.6 µl H2O</TD><TD>4.6 µl H2O</TD></TR><TR><TD>0.5 µl a1</TD></TR><TR><TD>Gal4:PIF:phy:VP16+ren; a1</TD><TD>LexA:PIF:phy:ren+opLex:luc; ?1</TD><TD>LacI:KNdronpa:ren+OpLex:Luc; ?1</TD></TR><TR><TD>1.5 µl Gal4:PIF:phy:VP16</TD><TD>1 µl LexA:PIF:phy:ren</TD><TD>1 µl LacI:KNdronpa:ren</TD></TR><TR><TD>2 µl renilla (159)</TD><TD>2.5 µl opLex:luc (151)</TD><TD>3 µl OpLac:Luc (152)</TD></TR><TR><TD>1.2 µl buffer</TD><TD>1.2 µl buffer</TD><TD>1.2 µl buffer</TD></TR><TR><TD>1.2 µl BSA </TD><TD>1.2 µl BSA</TD><TD>1.2 µl BSA</TD></TR><TR><TD>1 µl T4 ligase</TD><TD>1 µl T4 ligase</TD><TD>1 µl T4 ligase</TD></TR><TR><TD>1 µl BSAI</TD><TD>1 µl BsmbI</TD><TD>1 µl BsmbI</TD></TR><TR><TD>2.6 µl H2O</TD><TD>2.6 µl H2O</TD><TD>3 µl H2O</TD></TR><TR><TD>1 µl a1</TD><TD>0.5 µl ?1</TD><TD>0.5 µl ?1</TD></TR><TR><TD>Gal4:KNdronpa:ren+OpLex:Luc; ?1</TD><TD>PhyB:VP16+PIF6; ?1</TD></TR><TR><TD>1 µl Gal4:KNdronpa:ren</TD><TD>1 µl PhyB:VP16 (88E)</TD></TR><TR><TD>4 µl OpUAS:Luc (227)</TD><TD>2 µl PIF6 (170)</TD></TR><TR><TD>1.2 µl buffer</TD><TD>1.2 µl buffer</TD></TR><TR><TD>1.2 µl BSA</TD><TD>1.2 µl BSA</TD></TR><TR><TD>1 µl T4 ligase</TD><TD>1 µl T4 ligase</TD></TR><TR><TD>1 µl BsmbI</TD><TD>1 µl BsmbI</TD></TR><TR><TD>3 µl H2O</TD><TD>3.6 µl H2O</TD></TR><TR><TD>0.5 µl ?1</TD><TD>1 µl ?1</TD></TR></TBODY></TABLE><UL><LI>These <I>Agrobacterium</I> cultures have been refreshed:</LI><UL class="ul_2"><LI>PIF-phyB-luc</LI><LI>Renilla</LI><LI>Pnos</LI><LI>Etr8</LI></UL></UL><H3 style="color:green">16 July 2015</H3><UL><LI>Yesterday ligations have been transformed into <I>E. coli</I>:</LI><UL class="ul_2"><LI>phyC31;pUPD2</LI><LI>AsLOVpep; pUPD2</LI><LI>LacI:PIF:Phy:VP16+ren; a1</LI><LI>Gal4:PIF:phy:VP16+ren; a1</LI><LI>LexA:PIF:phy:ren+opLex:luc; ?1</LI><LI>LacI:KNdronpa:ren+OpLex:Luc; ?1</LI><LI>Gal4:KNdronpa:ren+OpLex:Luc; ?1</LI><LI>PhyB:VP16+PIF6; ?1</LI></UL></UL><UL><LI>The agroinfiltrated leaf with BxbI:rep:GFP has been observed in the magnifying glass with fluorescent lights. The efficiency of the recombinase is lower and it can not been observed a lot of green spots. Tomorrow another leaf will be seen to check again the construction.</LI></UL><UL><LI>Second refresh of the <I>Agrobacterium</I> cultures:</LI><UL class="ul_2"><LI>PIF-phyB-luc</LI><LI>Renilla</LI><LI>Pnos</LI><LI>Etr8</LI></UL></UL><UL><LI>Miniprep of these cultures.</LI><LI>Digestion of the minipreps:</LI></UL><DIV class="table-wrapper"><TABLE class="alt"><TBODY><TR><TD>PIF-phyB-luc; ?1</TD><TD>EcoRI</TD><TD>??</TD></TR><TR><TD>Renilla; ?2</TD><TD>HindIII</TD><TD>No lo encuentro</TD></TR><TR><TD>Pnos; ?1</TD><TD>EcoRI</TD><TD>2997, 353</TD></TR><TR><TD>Etr8; ?1</TD><TD>EcoRI</TD></TR></TBODY></TABLE><P>The digestions had positive controls that were included in the gel to compare the results obtained.</P><P>The digestions are left overnight in the working table.</P><H3 style="color:green">17 July 2015</H3><P>Do the gel:</P><DIV class="table-wrapper"><TABLE class="alt"><TBODY><TR><TD>No se lo que pone?</TD></TR></TBODY></TABLE><UL><LI>Liquid culture of the colonies in the agar plates have been made, 3 colonies for each construction.</LI><LI>OD’s mesurement for the agroinfiltration.</LI><UL class="ul_2"><LI>Dilution 1:10 in MES.</LI></UL></UL><DIV class="table-wrapper"><TABLE class="alt"><TBODY><TR><TD>PIF:phyB:luc</TD><TD>0.47</TD><TD>41 µl/ml</TD><TD>630 µl</TD></TR><TR><TD>Renilla</TD><TD>0.28</TD><TD>71 µl/ml</TD><TD>1065 µl</TD></TR><TR><TD>Etr8:luc</TD><TD>0.32</TD><TD>52 µl/ml</TD><TD>930 µl</TD></TR><TR><TD>Pnos</TD><TD>0.34</TD><TD>59 µl/ml</TD><TD>885 µl</TD></TR></TBODY></TABLE><UL><LI>Red toggle experiement:</LI></UL><P>The plants were coinfiltrated with the red toggle (PIF+phyB) and renilla. Also with two controls, Pnos (positive control) and Etr8 (negative control).</P><P>14 plants were used with 3 infiltrated leafs for each plants and two spots per leaf.</P><P>The controls have been infiltrated in leafs of the plants like is shown in the picture. Pnos in the right part and Etr8 in the left part.</P><P>Two plants were infiltrated with both controls in the three leafs and they were in natural light during all the experiment. </P><P>Another 4 plants were infiltrated with controls following the same pattern in the procedure. </P><P>The remaining 8 plants were infiltrated with the red toggle.</P><P>Immediately after the infiltration the plants were distributed in three different conditions: natural light, darkness and far red.</P><P>So after the agroinfiltration 2 control plants stay in natural light all the experiment; 2 control plants and 4 red toggle went into the far red chamber and the same amount of control and red toggle plan (2+4) went put in darkness.</P><P>This conditions were maintained 3 days and then all the infiltrated leafs were cut into small discs and put into a special plates with water. </P><P>In this moment was set the time 0 (21/07/2015 at 19:30) and take the first samples.</P><P>Then in time 0 some of the dark and far red samples were put in red conditions to activate the red toggle.</P><P>This scheme represent the distribution of samples and the times that were taken the samples.</P><P>Hacer el esquema!!! Cuando este mas espabilada ;)</P><P>It was observed another leaf with the recombinase BxbI with GFP in the magnifying glass with fluorescent lights. It was not observed a lot of spots, the efficiency is very low.</P><H3 style="color:green">18 July 2015</H3><P>23 minipreps of the liquid cultrures. LexA:PIF:phtB:ren:luc (C3) has not grown.</P><P>Digestions of the minipreps:</P><DIV class="table-wrapper"><TABLE class="alt"><TBODY><TR><TD>phyC31;pUPD2</TD><TD>NotI</TD><TD>2046, 1899</TD></TR><TR><TD>AsLOVpep; pUPD2</TD><TD>NotI</TD></TR><TR><TD>2046, 521</TD></TR><TR><TD>LacI:PIF:Phy:VP16+ren; a1</TD><TD>EcoRI</TD><TD>6345, 5623, 5487</TD></TR><TR><TD>Gal4:PIF:phy:VP16+ren; a1</TD><TD>EcoRI</TD><TD>6345, 5487, 4852</TD></TR><TR><TD>LexA:PIF:phy:ren+opLex:luc; ?1</TD><TD>BamHI</TD><TD>9431, 6674, 3513</TD></TR><TR><TD>LacI:KNdronpa:ren+OpLex:Luc; ?1</TD><TD>BamHI</TD><TD>12632, 6574</TD></TR><TR><TD>Gal4:KNdronpa:ren+OpLex:Luc; ?1</TD><TD>BamHI</TD><TD>11582, 6674</TD></TR><TR><TD>PhyB:VP16+PIF6; ?1</TD><TD>BamHI</TD><TD>6674, 2685, 2337, 1439</TD></TR></TBODY></TABLE><P>Gel has been done:</P><DIV class="table-wrapper"><TABLE class="alt"><TBODY><TR><TD>phyC31 C1</TD><TD>phyC31 C2</TD><TD>phyC31 C3</TD><TD>AsLOVpep C1</TD></TR><TR><TD>no</TD><TD>no</TD><TD>no</TD><TD>ok</TD></TR><TR><TD>AsLOVpep C2</TD><TD>AsLOVpep C3</TD><TD>Gal4:PIF:phy:VP16:ren C1</TD><TD>Gal4:PIF:phy:VP16:ren C2</TD></TR><TR><TD>no</TD><TD>no</TD><TD>no</TD><TD>no</TD></TR><TR><TD>Gal4:PIF:phy:VP16:ren C3</TD><TD>LacI:PIF:phy:ren C1</TD><TD>LacI:PIF:phy:ren C2</TD><TD>LacI:PIF:phy:ren C3</TD></TR><TR><TD>no</TD><TD>ok</TD><TD>no</TD><TD>No</TD></TR><TR><TD>LexA:PIF:phy:ren:luc C1</TD><TD>LexA:PIF:phy:ren:luc C2</TD><TD>Gal4:KNdronpa:ren:luc C1</TD><TD>Gal4:KNdronpa:ren:luc C2</TD></TR><TR><TD>no</TD><TD>no</TD><TD>ok</TD><TD>No</TD></TR><TR><TD>Gal4:KNdronpa:ren:luc C3</TD><TD>LacI:KNdronpa:ren:luc C1</TD><TD>LacI:KNdronpa:ren:luc C2</TD><TD>LacI:KNdronpa:ren:luc C3</TD></TR><TR><TD>no</TD><TD>no</TD><TD>ok</TD><TD>No</TD></TR><TR><TD>PhyB:VP16:PIF6 C1</TD><TD>PhyB:VP16:PIF6 C2</TD><TD>PhyB:VP16:PIF6 C3</TD></TR><TR><TD>ok</TD><TD>no</TD><TD>no</TD></TR></TBODY></TABLE><P>Pick more colonies of:</P><UL><LI>Gal4:PIF:phy:VP16+ren; a1</LI><LI>LexA:PIF:phy:ren+opLex:luc; ?1</LI></UL><P>New digestions with new enzymes:</P><DIV class="table-wrapper"><TABLE class="alt"><TBODY><TR><TD>phyC31;pUPD2</TD><TD>XhoI (buffer red)</TD><TD>2119, 934, 894</TD></TR><TR><TD>LacI:PIF:Phy:VP16+ren; a1</TD><TD>NEB4</TD><TD>5949, 5653, 3610, 2246</TD></TR><TR><TD>LacI:PIF:Phy:VP16+ren; a1</TD><TD>HindIII</TD><TD>11568, 5587</TD></TR></TBODY></TABLE><P>After 3 digestions of LacI:PIF:Phy:VP16+ren; a1 is accepted the construction.</P><P>It was observed another leaf with the recombinase BxbI with GFP in the magnifying glass with fluorescent lights. It was not observed a lot of spots, the efficiency is very low. This is the 4th day…</P><H3 style="color:green">19 July 2015</H3><H3 style="color:green">20 July 2015</H3><H3 style="color:green">21 July 2015</H3><UL><LI>Red toggle experiment:</LI></UL><P>Time lapses: </P><P>-19:00=t0</P><P>-1:00=t1</P><P>-7:00= t2</P><P>-19:00= t3 (it was taken the controls in natural light)</P><UL><LI>Minipreps of the colonies that were in 37ºC.</LI><UL class="ul_2"><LI>LexA:PIF:Phy:ren:luc (C1 and C2)</LI><LI>PhyC31 (C1, C2, C4 and C5)</LI></UL></UL><DIV class="table-wrapper"><TABLE class="alt"><TBODY><TR><TD>LexA:PIF:Phy:ren:luc</TD><TD>BamHI</TD><TD>9431, 6674, 3513</TD></TR><TR><TD>PhyC31</TD><TD>NotI</TD><TD>2046, 1899</TD></TR></TBODY></TABLE><P>Gel with the digestions:</P><DIV class="table-wrapper"><TABLE class="alt"><TBODY><TR><TD>PhyC31 C1</TD><TD>PhyC31 C2</TD><TD>PhyC31 C4</TD><TD>PhyC31 C5</TD></TR><TR><TD>Ok?</TD><TD>ok</TD><TD>ok</TD><TD>ok</TD></TR><TR><TD>LexA:PIF:Phy:ren:luc C1</TD><TD>LexA:PIF:Phy:ren:luc C2</TD></TR><TR><TD>No DNA</TD><TD>No DNA</TD></TR></TBODY></TABLE><P>We had problems with some colonies because in the digestion did not appear DNA. The minipreps will be made with a better kit.</P><P>Transform in <I>Agrobacterium</I> this cultures:</P><UL><LI>LexABD:KDronpa:NDronpa:ren:luc</LI><LI>Gal4BD:KDronpa:NDronpa:ren:luc</LI><LI>LacIBD:KDronpa:NDronpa:ren:luc</LI><LI>OpLexA:luc (151)</LI><LI>OpUAS:luc</LI><LI>OpLacI:luc (152)</LI></UL><P>It was made liquid culture of Gal4:PIF:phyB:ren; ?1 (C1-C5)</P><P>It was made ligations:</P><DIV class="table-wrapper"><TABLE class="alt"><TBODY><TR><TD>35S:Gal4:AsLOVpep:T35S; ?1</TD><TD>35S:LacI:AsLOVpep:T35S; ?1</TD><TD>35S:LexA:AsLOVpep:T35S; ?1</TD></TR><TR><TD>1 µl 35S (0030)</TD><TD>1 µl 35S (0030)</TD><TD>1 µl 35S (0030)</TD></TR><TR><TD>1 µl Gal4BD</TD><TD>1 µl LacI</TD><TD>1 µl LexA</TD></TR><TR><TD>1 µl AsLOVpep </TD><TD>1 µl AsLOVpep </TD><TD>1 µl AsLOVpep </TD></TR><TR><TD>1 µl T35S (0036)</TD><TD>1 µl T35S (0036)</TD><TD>1 µl T35S (0036)</TD></TR><TR><TD>1 µl ?1 </TD><TD>1 µl ?1 </TD><TD>1 µl ?1 </TD></TR><TR><TD>2.6 µl H2O</TD><TD>2.6 µl H2O</TD><TD>2.6 µl H2O</TD></TR><TR><TD>PsinATG:RepPhiC31:GFP:T35S; ?2</TD><TD>LacI:PIF:PhyB:ren+luc; ?1</TD><TD>PIF:PhyB+renilla; ?2</TD></TR><TR><TD>1 µl PsinATG (552)</TD><TD>1.5 µl LacI:PIF:PhyB:ren; ?1</TD><TD>1.5 µl PIF:PhyB</TD></TR><TR><TD>1 µl ReporterPhyC31</TD><TD>3 µl OpLacI:luc (152); ?2</TD><TD>2.5 µl renilla (159)</TD></TR><TR><TD>1 µl GFP (0059)</TD><TD>0.5 µl ?1</TD><TD>0.5 µl ?2</TD></TR><TR><TD>1 µl T35S (0036)</TD><TD>2.6 H2O</TD><TD>3 µl H2O</TD></TR><TR><TD>1 µl ?2 </TD></TR><TR><TD>2.6 µl H2O</TD></TR></TBODY></TABLE><P>Luciferase essay with all the samples collected in the last three days.</P><UL><LI>Sampling: 25 samples in total.</LI><LI>Passive lysis 1x (200µl/sample x 25 samples)= 5.000 µl</LI><LI>Crush samples.</LI><LI>Add 150 µl of passive lysis buffer and mix in the vortex.</LI><LI>Centrifuge at 13200rpm for 15’.</LI></UL><DIV class="table-wrapper"><TABLE class="alt"><TBODY><TR><TD>E8/li</TD><TD>E8/li</TD><TD>E8/li</TD><TD>P/li</TD><TD>P/li</TD><TD>P/li</TD><TD>TR/Fr</TD><TD>TR/Fr</TD><TD>TR/Fr</TD><TD>TR/D</TD><TD>TR/D</TD><TD>TR/D</TD></TR><TR><TD>TR/Fr</TD></TR><TR><TD>Red</TD><TD>TR/Fr</TD></TR><TR><TD>Red</TD><TD>TR/r</TD></TR><TR><TD>Red</TD><TD>TR/D</TD></TR><TR><TD>Red</TD><TD>TR/D</TD></TR><TR><TD>Red</TD><TD>TR/D</TD></TR><TR><TD>Red</TD></TR></TBODY></TABLE><P>We used 14.4 µl of Stop and Glow. 705.6 destilled H2O</P><H3 style="color:green">22 July 2015</H3><P>Miniprep of Gal4:PIF:PhyB:ren (C1-C3) C4 and C5 did not grow.</P><P>Digestion of the miniprep:</P><DIV class="table-wrapper"><TABLE class="alt"><TBODY><TR><TD>Gal4:PIF:PhyB:ren</TD><TD>BamHI</TD><TD>16684</TD></TR><TR><TD>EcoRI</TD><TD>4852, 5487, 6345</TD></TR><TR><TD>EcoRV</TD><TD>1849, 3942, 2475, 381, 8037</TD></TR></TBODY></TABLE><P>Gel was made:</P><DIV class="table-wrapper"><TABLE class="alt"><TBODY><TR><TD>Gal4… (BamHI)</TD><TD>Gal4… (EcoRI)</TD><TD>Gal4… (EcoRV)</TD></TR><TR><TD>no</TD><TD>no</TD><TD>no</TD></TR></TBODY></TABLE><P>After doing several digestions with different enzyme all with wrong band patterns, it was decided to revise each part making digestions. The parts are: </P><UL><LI>PhyC31; pUPD2</LI><LI>Gal4:PIF:phyB; </LI><LI>Renilla (GB159)</LI></UL><P>Made liquid culture of the ReporterBxbI:GFP in <I>Agrobacterium</I>.</P><P>Transform the ligations into <I>E. coli</I>:</P><UL><LI>35S:Gal4:AsLOVpep:T35S; ?1</LI><LI>35S:LacI:AsLOVpep:T35S; ?1</LI><LI>35S:LexA:AsLOVpep:T35S; ?1</LI><LI>PsinATG:RepPhiC31:GFP:T35S; ?2</LI><LI>LexA:PIF:PhyB:ren+luc; ?1</LI><LI>Add 300 µl of SOC medium and put into a agar plate with the corresponding antibiotics.</LI></UL><P>Luciferase essay:</P><UL><LI>Sampling: samples that have been in red and natural light 48h, 3 samples.</LI><LI>200 µl/sample x 3 sample= 600 µl of passive lissis buffer (5x)</LI><LI>Passive lissis buffer 1x= 120 µl+480 µl water.</LI><LI>2.4 µl of Stop and glow</LI><LI>118 µl of  buffer.</LI></UL><H3 style="color:green">23 July 2015</H3><P>We decided to infiltrate soybean sprouts. First we decided to infiltrate with dye to observe the characteristics and the capacity of absorption.</P><P>Mesurement of the ODs to agroinfiltrate:</P><P>The samples are:</P><UL><LI>Citoplasm=0.27</LI><LI>DsRed=0.27</LI><LI>GFP=0.36</LI><LI>Recombinase=0.43</LI></UL><P>Vi= (Vf*Absf)/(Absi*10)</P><P>Absi=0.1 (because is a viral system)</P><P>Vf=120 µl</P><P>Make 16 liquid culture of the white colonies in the agar plates.</P><H3 style="color:green">24 July 2015</H3><P>Minipreps of the 13 liquid culture. LacI:PIF:phyB:ren cultures did not grow.</P><P>Digestion of the minipreps:</P><DIV class="table-wrapper"><TABLE class="alt"><TBODY><TR><TD>Gal4:AsLOVpep</TD><TD>EcoRI</TD><TD>6345, 1972</TD></TR><TR><TD>LacI:AsLOVpep</TD><TD>EcoRI</TD><TD>6345, 2743</TD></TR><TR><TD>LexA:AsLOVpep</TD><TD>EcoRI</TD><TD>6345, 2011</TD></TR><TR><TD>PsinATG:RepPhiC31:GFP</TD><TD>HindIII</TD><TD>6345, 2691</TD></TR><TR><TD>LexA:PIF:PhyB:ren+luc</TD><TD>BamHI</TD><TD>9431, 6674, 3513</TD></TR><TR><TD>PhyC31; pUPD2</TD><TD>NotI</TD><TD>2046, 1899</TD></TR><TR><TD>Gal4:PIF:phyB</TD><TD>BamHI</TD><TD>6674, 3474, 2337</TD></TR><TR><TD>Renilla (GB159)</TD><TD>EcoRV</TD><TD>2909, 2475, 882, 812, 381</TD></TR></TBODY></TABLE><P>It was done 2 gels with ligations:</P><DIV class="table-wrapper"><TABLE class="alt"><TBODY><TR><TD>Gal4:AsLOV C1</TD><TD>Gal4:AsLOV C2</TD><TD>Gal4:AsLOV C3</TD><TD>PsinATG:RepPhi</TD></TR><TR><TD>C31:GFP C1</TD><TD>PsinATG:RepPhi</TD></TR><TR><TD>C31:GFP C2</TD><TD>PsinATG:RepPhi</TD></TR><TR><TD>C31:GFP C3</TD></TR><TR><TD>ok</TD><TD>ok</TD><TD>ok</TD><TD>ok</TD><TD>no</TD><TD>No</TD></TR><TR><TD>Mw/ladder</TD><TD>LacI:AsLOV C1</TD><TD>LacI:AsLOV C2</TD><TD>LexA:AsLOV C1</TD><TD>LexA:AsLOV C2</TD><TD>LexA:AsLOV C3</TD></TR><TR><TD>-</TD><TD>ok</TD><TD>No</TD><TD>no</TD><TD>ok</TD><TD>no</TD></TR><TR><TD>LexA:PIF:PhyB:ren+luc C1</TD><TD>LexA:PIF:PhyB:ren+luc C2</TD></TR><TR><TD>no</TD><TD>Ok</TD></TR></TBODY></TABLE><DIV class="table-wrapper"><TABLE class="alt"><TBODY><TR><TD>PhyC31 (C1)</TD><TD>PhyC31 (C2)</TD><TD>PhyC31 (C3)</TD><TD>PhyC31 (C4)</TD><TD>PhyC31 (C5)</TD></TR><TR><TD>no</TD><TD>ok</TD><TD>ok</TD><TD>ok</TD><TD>no</TD></TR><TR><TD>Gal4:PIF:phyB</TD><TD>Renilla (GB159)</TD></TR><TR><TD>ok</TD><TD>Ok</TD></TR></TBODY></TABLE><H3 style="color:green">26 July 2015</H3><P>Liquid cultures of <I>Agrobacterium</I> were refreshed:</P><UL><LI>TsinATG:BxbIreporter:GFP</LI><LI>TsinATG:BxbI:reporterBxbI:GFP</LI><LI>Viral vector??no se cual es</LI></UL><H3 style="color:green">27 July 2015</H3><P>Sent to sequence:</P><UL><LI>210.08.256: LacIBD; pUPD2 (C1)</LI><LI>210.08258: Gal4BD; pUPD2 (C2)</LI><LI>210.08.259:LexABD; pUPD2 (C1) </LI><LI>210.08.260: PIF6; pUPD2 (C5)</LI><LI>210.08.261: VP16; pUPD2 (C1)</LI><LI>210.08.262: PhiC31; pUPD2 (C2)</LI><LI>210.08.264: PhiC31; pUPD2 (C3)</LI><LI>210.08.266: PhiC31; pUPD2 (C4)</LI><LI>210.08.268: ReporterBxbI; pUPD2 (C1)</LI><LI>210.08.269: ReporterPhiC31; pUPD2 (C1)</LI><LI>210.08.270: AsLOVpep; pUPD2 (C1)</LI></UL><P>The sample had to have 10µl of miniprep (200ng/µl aprox) + 5 µl of primer (dilution 1:3)</P><P>Ligations:</P><DIV class="table-wrapper"><TABLE class="alt"><TBODY><TR><TD>Gal4:PIF:phiB + ren; ?1</TD><TD>LacI:PIF:phiB:ren + luc; ?2</TD><TD>PIF:PhyB+renilla; ?2</TD></TR><TR><TD>1.5 µl Gal4:PIF:phiB</TD><TD>1.5 µl LacI:PIF:phiB:ren</TD><TD>1.5 µl PIF:phiB</TD></TR><TR><TD>2 µl Renilla (GB159)</TD><TD>2 µl OpLacI:luc</TD><TD>2.5 µl Renilla (GB159)</TD></TR><TR><TD>0.5 µl ?1</TD><TD>0.5 µl ?2</TD><TD>0.5 µl ?2</TD></TR><TR><TD>3.6 µl H2O</TD><TD>2.6 µl H2O</TD><TD>3 µl H2O</TD></TR></TBODY></TABLE><P>OD’s measurements to prepare the agroinfiltrates:</P><DIV class="table-wrapper"><TABLE class="alt"><TBODY><TR><TD>Cytoplasm: 0.39 (viral)</TD><TD>0.25ml</TD></TR><TR><TD>Integrase: 0.35 (viral)</TD><TD>0.28ml</TD></TR><TR><TD>GFP: 0.32 (viral)</TD><TD>0.31ml</TD></TR><TR><TD>Dsred: 0.31 (viral)</TD><TD>0.32ml</TD></TR><TR><TD>BxbI:reporter: 0.41</TD><TD>0.49ml</TD></TR><TR><TD>Reporter: 0.26</TD><TD>0.77ml</TD></TR></TBODY></TABLE><P>3 plants were infiltrated with BxbI and the reporter of BxbI (one leaf with the control an the other with the recombinase)?? No entiendo lo de la libreta </P><P>Ligations to join the negative controls with renilla:</P><DIV class="table-wrapper"><TABLE class="alt"><TBODY><TR><TD>Etr8:luc+staffer (SF)</TD><TD>OpLexA:luc+SF</TD><TD>OpLacI:luc+SF</TD><TD>UAS:luc+SF</TD></TR><TR><TD>1.5 µl Etr8:luc (88C o 1098)</TD><TD>1.5 µl OpLexA:luc (151)</TD><TD>1.5 µl OpLacI:luc (152)</TD><TD>1.5 µl UAS:luc (227)</TD></TR><TR><TD>1 µl SF; ?2</TD><TD>1 µl SF; ?2</TD><TD>1 µl SF; ?2</TD><TD>1 µl SF; ?2</TD></TR><TR><TD>1 µl ?1</TD><TD>1 µl ?1</TD><TD>1 µl ?1</TD><TD>1 µl ?1</TD></TR><TR><TD>6.1 µl H2O</TD><TD>6.1 µl H2O</TD><TD>6.1 µl H2O</TD><TD>6.1 µl H2O</TD></TR></TBODY></TABLE><P>Transformation of <I>E. coli</I> of the ligations:</P><UL><LI>Gal4:PIF:phiB + ren; ?1</LI><LI>LacI:PIF:phiB:ren + luc; ?2</LI><LI>PIF:PhyB+renilla; ?2</LI></UL><P>Make cultures in agar petri dishes of the transformations with the corresponding antibiotics.</P><P>Transformation into <I>Agrobacterium</I> with the constructions:</P><UL><LI>Gal4:KDronpa:NDronpa:ren:luc</LI><LI>LacI:KDronpa:NDronpa:ren:luc</LI><LI>LexA:KDronpa:NDronpa:ren:luc</LI><LI>OpLexA:luc (GB 151)</LI><LI>OpLacI:luc (GB 152)</LI><LI>UAS:luc (GB 227)</LI></UL><P>Make cultures in agar petri dishes of the transformations with the corresponding antibiotics.</P><P>It was received a new piece (ePDZ) which is part of the blue toggle (plan A). It arrived in E.coli so it was made a lquid culture and let it grow at 37ºC overnight.</P><H3 style="color:green">28 July 2015</H3><P>Miniprep of the liquid culture: ePDZ.</P><P>Transformation into E.coli of the ligations:</P><UL><LI>Etr8:luc+staffer (SF)</LI><LI>OpLexA:luc+SF</LI><LI>OpLacI:luc+SF</LI><LI>UAS:luc+SF</LI><LI>Gal4:PIF:phyB:ren</LI></UL><P>It was observed the soybean sprouts that were infiltrated with a dye with green light and red filter. It was not observed nothing significant, moreover, the damage is evident. </P><P>Transformation in <I>Agrobacterium</I> the ReporterPhiC31:GFP.</P><P>Pick colonies and make liquid cultures adding X-Gal and IPTG because the colonies were little and we can not observe clearly if they were white or blue.</P><UL><LI>Gal4:PIF:phiB + ren. Did not grow any colony.</LI><LI>LacI:PIF:phiB:ren + luc (C1-C3)</LI><LI>PIF:PhyB+renilla (C1- C3)</LI></UL><P>The medicine LTB (heat labile toxin B subunit) which is part of an enterotoxin of Echerichia coli homologous to the same toxin in Vibrio cholerae that causes diarrhea. </P><P>We add 50µl to have a final concentration of 10ng/µl.</P><P>Ligation:</P><DIV class="table-wrapper"><TABLE class="alt"><TBODY><TR><TD>LTB; pUPD2</TD></TR><TR><TD>1 µl LTB</TD></TR><TR><TD>1 µl pUPD2</TD></TR><TR><TD>5.6 µl H2O</TD></TR></TBODY></TABLE><H3 style="color:green">29 July 2015</H3><P>Miniprep of yesterday liquid culture:</P><UL><LI>LacI:PIF:phiB:ren + luc (C1 and C3) C2 turn into blue.</LI><LI>PIF:PhyB+renilla (C2) C1 and C3 did not grow.</LI></UL><P>Digestion:</P><DIV class="table-wrapper"><TABLE class="alt"><TBODY><TR><TD>LacI:PIF:phiB:ren:luc</TD><TD>EcoRV</TD><TD>882, 968, 1652, 3942, 2475, 381, 3477, 6674</TD></TR><TR><TD>PIF:PhyB+renilla</TD><TD>HindIII</TD><TD>4316, 5887, 788, 6345</TD></TR></TBODY></TABLE><P>Gel:</P><DIV class="table-wrapper"><TABLE class="alt"><TBODY><TR><TD>LacI:PIF:phiB:ren:luc (C1)</TD><TD>LacI:PIF:phiB:ren:luc (C3)</TD><TD>PIF:PhyB+renilla (C2)</TD></TR><TR><TD>no</TD><TD>no</TD><TD>no</TD></TR></TBODY></TABLE><P>Pick colonies and make liquid culture of:</P><UL><LI>Etr8:luc:staffer(SF) (C1-C3)</LI><LI>OpLexA:luc:SF (C1-C3)</LI><LI>OpLacI:luc:SF (C1-C3)</LI><LI>UAS:luc:SF (C1-C3)</LI><LI>Gal4:PIF:phyB:ren (C1-C3)</LI></UL><P>Transformation in E.coli of:</P><UL><LI>LTB; pUPD2</LI></UL><H3 style="color:green">30 July 2015</H3><P>Minipreps have been done:</P><UL><LI>Etr8:luc:staffer(SF) (C1-C3)</LI><LI>OpLexA:luc:SF (C1 and C3) C2 did not grow.	</LI><LI>OpLacI:luc:SF (C1-C3)</LI><LI>UAS:luc:SF (C1-C3)</LI><LI>Gal4:PIF:phyB:ren (C1-C3)</LI></UL><UL><LI>LacI:PIF:phy:ren:luc (C1-C3)</LI><LI>PIF:phyB:ren (C1-C3)</LI></UL><DIV class="table-wrapper"><TABLE class="alt"><TBODY><TR><TD>Etr8:luc:staffer(SF)</TD><TD>BamHI</TD><TD>6674, 2766</TD></TR><TR><TD>OpLexA:luc:SF</TD><TD>BamHI</TD><TD>6674, 2746</TD></TR><TR><TD>OpLacI:luc:SF</TD><TD>BamHI</TD><TD>6674, 2847</TD></TR><TR><TD>UAS:luc:SF</TD><TD>BamHI</TD><TD>6674, 2568</TD></TR><TR><TD>Gal4:PIF:phyB:ren</TD><TD>EcoRI</TD><TD>6345, 5487, 4852</TD></TR><TR><TD>LacI:PIF:phy:ren:luc</TD><TD>BamHI</TD><TD>20451</TD></TR><TR><TD>PIF:phyB:ren</TD><TD>HindIII</TD><TD>6345, 5887, 4316, 788</TD></TR></TBODY></TABLE><P>Do the gel:</P><P>No se el orden!! Preguntar </P><P>Primers have arrived:</P><UL><LI>ePDZ reverse and forward and phyC31.</LI></UL><P>Make a PCR with ePDZ and its primers to obtain the desired fragment and put it in pUPD2.</P><DIV class="table-wrapper"><TABLE class="alt"><TBODY><TR><TD>ePDZ PCR</TD></TR><TR><TD>10µl Buffer HF</TD></TR><TR><TD>31.5 µl H2O</TD></TR><TR><TD>2 µl dNTPs</TD></TR><TR><TD>2.5 µl  Primer forward</TD></TR><TR><TD>2.5 µl primer reverse</TD></TR><TR><TD>1 µl ePDZ (dilution 1:50)</TD></TR><TR><TD>0.5 µl Taq phunion</TD></TR></TBODY></TABLE><P>Pick 2 colonies of phiC31:GFP in <I>Agrobacterium</I> and make liquid culture.</P><P>Ligations:</P><DIV class="table-wrapper"><TABLE class="alt"><TBODY><TR><TD>ePDZ; pUPD2</TD><TD>PIF:phyB+ren; ?2</TD><TD>OpUAS:luc:SF+ren; ?1</TD><TD>OpLexA:luc:SF+ren; ?1</TD></TR><TR><TD>1 µl ePDZ</TD><TD>1 µl PIF:phyB</TD><TD>1 µl OpUAS:luc:SF</TD><TD>1 µl OpLexA:luc:SF</TD></TR><TR><TD>1 µl pUPD2</TD><TD>1 µl ren (159)</TD><TD>1 µl ren</TD><TD>1 µl ren</TD></TR><TR><TD>5.6 µl H2O</TD><TD>1 µl ?2</TD><TD>1 µl ?1</TD><TD>1 µl ?1</TD></TR><TR><TD>4.6 µl H2O</TD><TD>4.6 µl H2O</TD><TD>4.6 µl H2O</TD></TR><TR><TD>OpEtr8:luc:SF+ren; ?1</TD><TD>Op:LacI:luc:SF+ren; ?1</TD></TR><TR><TD>1 µl OpEtr8:luc:SF</TD><TD>1 µl OpLacI:luc:SF</TD></TR><TR><TD>1 µl ren</TD><TD>1 µl ren</TD></TR><TR><TD>1 µl ?1</TD><TD>1 µl ?1</TD></TR><TR><TD>4.6 µl H2O</TD><TD>4.6 µl H2O</TD></TR></TBODY></TABLE><P>After 3 days till the agroinfiltration we have observed the leaf discs that have BxbI+repBxbI and the repBxbI (negative control). </P><P>We could see that the level of GFP expression was higer in the infiltration with the recombinase but there was also expression in the negative control. We think that this can be a contamination due to that we had infiltrated both constructions in he same leaf and we did not change the gloves between agroinfiltrations. FOTOO!</P><P>Make liquid culture (E.coli) of:</P><UL><LI>LTB; pUPD2 (C1-C3)</LI></UL><H3 style="color:green">31 July 2015</H3><P>Ligation:</P><DIV class="table-wrapper"><TABLE class="alt"><TBODY><TR><TD>LacI:PIF:phyB:ren+OpLacI:luc; ?2</TD></TR><TR><TD>1.5 µl LacI:PIF:phyB:ren</TD></TR><TR><TD>3 µl OpLacI:luc</TD></TR><TR><TD>0.5 µl ?2</TD></TR><TR><TD>2.6 µl H2O</TD></TR></TBODY></TABLE><P>After 4 days till the agroinfiltration we have observed again the samples with BxbI and the negative control but was the same. The negative control expressed GFP but less than the recombinase. Foto!</P><P>Minipreps of liquid culture LTB (C1 and C2)</P><P>Digestion:</P><DIV class="table-wrapper"><TABLE class="alt"><TBODY><TR><TD>LTB; pUPD2</TD><TD>NotI</TD><TD>2046, 474</TD></TR></TBODY></TABLE><P>Gel:</P><DIV class="table-wrapper"><TABLE class="alt"><TBODY><TR><TD>LTB (C1)</TD><TD>LTB (C2)</TD><TD>PCR ePDZ</TD></TR><TR><TD>??</TD></TR></TBODY></TABLE><P>Take out glicerynates of Paloma, lab mate:</P><UL><LI>Sip rotavirus CH2</LI><LI>Sip rotavirus CH2-CH3</LI></UL><P>For E.coli and <I>Agrobacterium</I>.</P><P>Pick a colony of Asun interferon (IFN) in Agro, make liquid culture.</P><P>We had miniprep of IFN in pUPD. Make ligations.</P><P>Transform in E.coli the ligations and make petri dishes cultures:</P><UL><LI>ePDZ; pUPD2</LI><LI>PIF:phyB+ren; ?2</LI><LI>OpUAS:luc:SF+ren; ?1</LI><LI>OpLexA:luc:SF+ren; ?1</LI><LI>OpEtr8:luc:SF+ren; ?1</LI><LI>Op:LacI:luc:SF+ren; ?1</LI><LI>LacI:PIF:phyB:ren+OpLacI:luc; ?2</LI></UL><P>Refresh agro cultures to agroinfiltrate tomorrow:</P><UL><LI>PhiC31 (viral system)</LI><LI>ReporterPhiC31</LI><LI>BxbI+reporterBxbI</LI><LI>ReporterBxbI</LI><LI>Gal4:KDronpa:NDronpa:luc:ren (brue toggle)</LI><LI>PIF:phi:luc</LI><LI>Renilla</LI><LI>P19</LI><LI>Pnos</LI></UL><P>New experiment with Nicotiana bentamiana. PROTOCOL</P><P>Next constructions were agroinfiltrated, 2 or 3 leafs for each plant:</P><P>PhiC31: one leaf for PhiC31+RepPhiC31+P19 (coinfiltration) and the other with RepPhiC31+P19 which is the negative control. 2 plants were infiltrated.</P><P>BxbI: one leaf with BxbI:RepBxbI+P19 and the other with RepBxbI+P19 which is negative control. 2 plants were infiltrated.</P><P>Red toggle: PIF6:phyB:luc+ren (coinfiltration). They were infiltrated 3 plants with 3 leafs for each.</P><P>Blue toggle: Gal4:NDronpa:KDronpa:luc:ren. They were infiltrated 3 plants with 3 leafs for each.</P><P>Pnos, the positive control. There are 4 plantas, 3 leafs per plant. </P><P>So we will take samples in time 0, 12, 24 and 36h for each construction and control. After 2 days of the agroinfiltration (during this period all the plants were in dark, it is set time 0 we make discs os leaf and put in a plate with water and we change the conditions that were before. </P><P>Red toggle plants: 9 discs stay in dark, 9 went to red and 9 went to natural light.</P><P>Blue toggle: the same as red but went to ultraviolet instead of red light.</P><P>Pnos: one plant were in natural light during all the experiment. The other discs will go, after the 2 days in black, to red, ultraviolet and stay in dark.</P><P>Recombinases: they are in natural light during all the experiment.</P><H3 style="color:green">1 August 2015</H3><P>Prepare the <I>Agrobacterium</I> cultures for agroinfiltration:</P><P>Gal4B:KDronpa:NDronpa:luc:ren, PhiC31 and its control, BxbI and its control and PIF:PhyB and its control were centrifugated 10 min at 3000g. The sobrenadant was discarded and the bacterias were resuspended with the agroinfiltration medium.</P><P>Agroinfiltration medium had: 10ml MES, 1ml MgCl2, 89ml H2O, 100ml DMSO+acetosiningone.</P><P>Incubate in 2h.</P><P>Mesurement oof the ODs:</P><DIV class="table-wrapper"><TABLE class="alt"><TBODY><TR><TD>Ren+P19</TD><TD>0.09</TD><TD>888.9 µl</TD></TR><TR><TD>RepPhiC31</TD><TD>0.69</TD><TD>115.94 µl</TD></TR><TR><TD>BxbI</TD><TD>0.46</TD><TD>174 µl</TD></TR><TR><TD>RepBxbI</TD><TD>0.15</TD><TD>533.3 µl</TD></TR><TR><TD>Gal4:KNDronpa:ren:luc</TD><TD>0.51</TD><TD>156.9 µl</TD></TR><TR><TD>Pnos</TD><TD>0.29</TD><TD>275.9 µl</TD></TR><TR><TD>PIF:phy:luc</TD><TD>0.17</TD><TD>470.6 µl</TD></TR><TR><TD>PhiC31</TD><TD>0.32</TD><TD>250 µl</TD></TR></TBODY></TABLE><P>We went to the greenhouse and infiltrate the 13 plants.</P><P>Look the petri dishes culture:</P><UL><LI>PIF:phyB+ren; ?2</LI><LI>OpLexA:luc:SF+ren; ?1</LI><LI>Op:LacI:luc:SF+ren; ?1</LI></UL><P>This colonies did not grow, the rest were left in the fridge.</P><UL><LI>ePDZ; pUPD2</LI><LI>OpUAS:luc:SF+ren; ?1</LI><LI>OpEtr8:luc:SF+ren; ?1</LI><LI>LacI:PIF:phyB:ren+OpLacI:luc; ?2</LI></UL><H3 style="color:green">3 August 2015</H3><P>Miniprep of the liquid culture:</P><UL><LI>Sip rotavirus CH2</LI><LI>Sip rotavirus CH2-CH3</LI></UL><P>Measure of DNA concentration of:</P><UL><LI>OpLexA:luc:SF=169ng/µl</LI><LI>OpacI:luc:SF=291ng/µl</LI></UL><P>Pick colonies gb159and make liquid culture of:</P><UL><LI>ePDZ; pUPD2 (C1-C3)</LI><LI>OpUAS:luc:SF+ren; ?1(C1-C3)</LI><LI>OpEtr8:luc:SF+ren; ?1(C1-C3)</LI><LI>LacI:PIF:phyB:ren+OpLacI:luc; ?2 (C1-C3) Added X-gal and IPTG.</LI></UL><P>Repeat ligations:</P><DIV class="table-wrapper"><TABLE class="alt"><TBODY><TR><TD>OpLexA:luc:SF+ren; ?1</TD><TD>Op:LacI:luc:SF+ren; ?1</TD><TD>E-PIF:phyB+ren; ?2</TD></TR><TR><TD>1 µl OpLexA:luc:SF</TD><TD>1 µl OpLacI:luc:SF</TD><TD>1 µl E-PIF:phy</TD></TR><TR><TD>3 µl ren</TD><TD>3 µl ren</TD><TD>3 µl ren</TD></TR><TR><TD>1 µl ?1</TD><TD>1 µl ?1</TD><TD>1 µl ?1</TD></TR><TR><TD>2.6 µl H2O</TD><TD>2.6 µl H2O</TD><TD>2.6 µl H2O</TD></TR></TBODY></TABLE><P>Refresh agrobacterium cultures of:</P><UL><LI>Interferon</LI><LI>SIP-CH2</LI><LI>SIP-CH2-CH3</LI></UL><P>Take the glycerinate of Renilla; ?2 (GB159) and make liquid culture.</P><P>We have made all the leaf discs and put in order in the plates with 300 µl of water, also we put each plate in the conditions light that they have to be. We have taken the samples of time0 (13:00).</P><P>So as T0= 13:00, T12=1:00, T24= 13:00 and T36= 1:00.</P><P>Imagenes de la colocacion en los platos? Hace falta?</P><H3 style="color:green">4 August 2015</H3><P>We do a Western blot with Asun so we can learn how to do it. Lo copio? De verdad!!?</P><P>Miniprep of the liquid cultures:</P><UL><LI>ePDZ; pUPD2 (C1-C3)</LI><LI>OpUAS:luc:SF+ren; ?1(C1-C3)</LI><LI>OpEtr8:luc:SF+ren; ?1(C1-C3)</LI><LI>LacI:PIF:phyB+ren; ?2 (C2 and C3) C1 was blue.</LI></UL><P>Digestions:</P><DIV class="table-wrapper"><TABLE class="alt"><TBODY><TR><TD>ePDZ; pUPD2</TD><TD>NotI</TD><TD>2046, 642</TD></TR><TR><TD>OpUAS:luc:SF+ren; ?1</TD><TD>EcoRI</TD><TD>6345, 7096</TD></TR><TR><TD>OpEtr8:luc:SF+ren; ?1</TD><TD>EcoRI</TD><TD>6345, 7294</TD></TR><TR><TD>LacI:PIF:phyB:ren+OpLacI:luc; ?2</TD><TD>EcoRV</TD><TD>6674, 3477, 381, 2475, 3942, 1652, 968, 882</TD></TR><TR><TD>Renilla 159</TD><TD>EcoRV</TD><TD>2909, 2475, 882, 812, 381</TD></TR></TBODY></TABLE><P>Gel:</P><DIV class="table-wrapper"><TABLE class="alt"><TBODY><TR><TD>ePDZ C1</TD><TD>ePDZ C2</TD><TD>ePDZ C3</TD><TD>LacI:PIF:phyB+ren </TD></TR><TR><TD>Ok</TD><TD>ok</TD><TD>ok</TD></TR><TR><TD>Renilla 159</TD><TD>OpUAS:luc:SF+ren C1</TD><TD>OpUAS:luc:SF+ren C2</TD><TD>OpUAS:luc:SF+ren C3</TD></TR><TR><TD>Ok</TD><TD>¿?</TD></TR><TR><TD>OpEtr8:luc:SF+ren C1</TD><TD>OpEtr8:luc:SF+ren C2</TD><TD>OpEtr8:luc:SF+ren C3</TD></TR></TBODY></TABLE><P>Transformation into E.coli of yesterday ligations:</P><UL><LI>OpLexA:luc:SF+ren; ?1</LI><LI>Op:LacI:luc:SF+ren; ?1</LI><LI>E-PIF:phyB+ren; ?2</LI></UL><P>Make petri dishes culture with the indicate antibiotic.</P><P>Ligations:</P><DIV class="table-wrapper"><TABLE class="alt"><TBODY><TR><TD>35S+ePDZ+VP16+T35S;?2</TD><TD>35S+LTB+T35S; ?1</TD></TR><TR><TD>1µl ePDZ</TD><TD>1µl 35S (30)</TD></TR><TR><TD>1 µl VP16</TD><TD>1µl T35S (36)</TD></TR><TR><TD>1 µl 35S (30)</TD><TD>1µl LTB</TD></TR><TR><TD>1 µl T35S (36)</TD><TD>1µl ?1</TD></TR><TR><TD>1 µl ?2</TD><TD>3.6 µl H2O</TD></TR><TR><TD>2.6 µl H2O</TD></TR></TBODY></TABLE><P>Refresh agrobacterium cultures of:</P><UL><LI>Interferon</LI><LI>SIP-CH2</LI><LI>SIP-CH2-CH3</LI></UL><H3 style="color:green">5 August 2015</H3><P>Transform ligations:</P><UL><LI>35S+ePDZ+VP16+T35S; ?2</LI><LI>35S+LTB+T35S; ?1</LI></UL><P>Pick colonies and make liquid cultures cultures of:</P><UL><LI>OpLexA:luc:SF+ren; ?1 (C1-C3)</LI><LI>Op:LacI:luc:SF+ren; ?1 (C1-C3)</LI><LI>E-PIF:phyB+ren; ?2 (C1)</LI><UL class="ul_ul_ul_notebook"><LI>35S+ePDZ+VP16+T35S; ?2 (C1 and C2)</LI></UL><LI>35S+LTB+T35S; ?1 (C1 and C2)</LI></UL><P>Make luciferase essay of red and blue toggle:</P><P>Number of samples=75 (a lot)</P><P>1.	Prepare lisis buffer for 80 samples.</P><P>200 µl/sample * 80sample = 16ml</P><P>Buffer (5x)—3.2ml of buffer + 12.8ml H2O miliQ: lysis buffer (1x)</P><P>2.	Crushed samples+ 150 µl of lysis buffer.</P><P>3.	Centrifuge 15min at 13200rpm in cold.</P><P>4.	Dilution 2:3 (Add 36 µl lysis buffer + 24 µl sample)</P><P>5.	Luciferase: 40 µl/sample. Prepare 2.88ml (3 substrate tubes)</P><P>6.	Renilla:….?</P><H3 style="color:green">6 August 2015</H3><P>Miniprep of the liquid cultures.</P><P>Digestion:</P><DIV class="table-wrapper"><TABLE class="alt"><TBODY><TR><TD>OpLexA:luc:SF+ren</TD><TD>EcoRI</TD><TD>6345, 7274</TD></TR><TR><TD>Op:LacI:luc:SF+ren</TD><TD>EcoRI</TD><TD>6345, 7375</TD></TR><TR><TD>E-PIF:phyB+ren</TD><TD>HindIII</TD><TD>6345, 788, 5887, 4316</TD></TR><TR><TD>35S+ePDZ+VP16+T35S</TD><TD>HindIII</TD><TD>6345, 2316</TD></TR><TR><TD>35S+LTB+T35S</TD><TD>EcoRI</TD><TD>6345, 1684</TD></TR></TBODY></TABLE><P>Gel:</P><DIV class="table-wrapper"><TABLE class="alt"><TBODY><TR><TD>PIF:phyB+ren</TD><TD>OpLexA:luc:SF+ren C1</TD><TD>OpLexA:luc:SF+ren C2</TD><TD>OpLexA:luc:SF+ren C3</TD></TR><TR><TD>No</TD><TD>Ok, repeat</TD><TD>No</TD><TD>Ok, repeat</TD></TR><TR><TD>Op:LacI:luc:SF+ren C1</TD><TD>Op:LacI:luc:SF+ren C2</TD><TD>Op:LacI:luc:SF+ren C3</TD><TD>35S+LTB+T35S C1</TD></TR><TR><TD>Ok, repeat</TD><TD>Ok</TD><TD>Ok</TD><TD>ok</TD></TR><TR><TD>35S+LTB+T35S C2</TD><TD>35S+ePDZ+VP16+T35S C1</TD><TD>35S+ePDZ+VP16+T35S C2</TD></TR><TR><TD>Ok</TD><TD>ok</TD><TD>ok</TD></TR></TBODY></TABLE><P>Make colony PCR with 6 colonies of PhiC31:</P><UL><LI>DNA- </LI><LI>JM1: 2 µl (dilution 1:10)</LI><LI>JM2: 2 µl (dilution 1:10)</LI><LI>Buffer Taq (with Mg): 2 µl</LI><LI>dNTPs: 2.5 µl</LI><LI>Taq: 0.5 µl</LI><LI>H2O: 10 µl</LI><LI>Program: 96ºC-2min; 96ºC-30min; 55ºC-30min; 72ºC-30min</LI></UL><P>Make a gel with the PCR but we did not see the correct bands.</P><P>Repeat the colony PCR:</P><UL><LI>DNA- </LI><LI>JM1: 2.5 µl (dilution 1:10)</LI><LI>JM2: 2.5 µl (dilution 1:10)</LI><LI>Buffer Taq (with Mg): 10 µl</LI><LI>dNTPs: 2 µl</LI><LI>Taq: 0.5 µl</LI><LI>H2O: 7.5 µl</LI><LI>Program: : 96ºC-10min; 96ºC-30min; 55ºC-30min; 72ºC-30min</LI></UL><P>Ligations:</P><DIV class="table-wrapper"><TABLE class="alt"><TBODY><TR><TD>LexA:AsLOVpep+ePDZ; ?1</TD><TD>LacI:AsLOVpep+ePDZ; ?1</TD><TD>Gal4:AsLOVpep+ePDZ; ?1</TD></TR><TR><TD>1 µl LexA:AsLOVpep; ?1</TD><TD>1 µl LacI:AsLOVpep; ?1</TD><TD>1 µl Gal4:AsLOVpep; ?1</TD></TR><TR><TD>1 µl ePDZ; ?2</TD><TD>1 µl ePDZ; ?2</TD><TD>1 µl ePDZ; ?2</TD></TR><TR><TD>1 µl BsmBI</TD><TD>1 µl BsmBI</TD><TD>1 µl BsmBI</TD></TR><TR><TD>1 µl ?1</TD><TD>1 µl ?1</TD><TD>1 µl ?1</TD></TR><TR><TD>4.6 µl H2O</TD><TD>4.6 µl H2O</TD><TD>4.6 µl H2O</TD></TR></TBODY></TABLE><P>Refresh Agro cultures to agroinfiltrate:</P><UL><LI>PhiC31:RepPhiC31:GFP</LI><LI>RepPhiC31:GFP</LI><LI>BxbI:RepBxbI:GFP</LI><LI>RepBxbI:GFP</LI><LI>IFN (interferon)</LI><LI>Sip-CH2</LI><LI>Sip-CH2-CH3</LI><LI>Pnos</LI><LI>P19 (this culture ha 10ml of LB + 10 µl antibiotics + 5 µl culture)</LI></UL><H3 style="color:green">7 August 2015</H3><P>Transformation into <I>Agrobacterium</I>:</P><UL><LI>35S:LTB:VP16:T35S</LI></UL><P>Transformation into E.coli and make petri dishes cultures:</P><UL><LI>LexA:AsLOVpep+ePDZ</LI><LI>LacI:AsLOVpep+ePDZ</LI><LI>Gal4:AsLOVpep+ePDZ</LI></UL><P>Make a gel with the colonies PCRs:</P><DIV class="table-wrapper"><TABLE class="alt"><TBODY><TR><TD>C7</TD><TD>C8</TD><TD>C9</TD><TD>C10</TD><TD>C11</TD><TD>C12</TD><TD>C13</TD><TD>C14</TD></TR><TR><TD>No</TD><TD>no</TD><TD>no</TD><TD>no</TD><TD>no</TD><TD>no</TD><TD>no</TD><TD>no</TD></TR></TBODY></TABLE><P>There was no DNA, there is a problem with the cells or the procedure, it have to be revised.</P><P>Ligation:</P><DIV class="table-wrapper"><TABLE class="alt"><TBODY><TR><TD>PhyC31; pUPD2</TD></TR><TR><TD>3 µl PhiC31</TD></TR><TR><TD>1 µl pUPD2</TD></TR><TR><TD>1 µl BsmBI</TD></TR><TR><TD>3.6 µl H2O</TD></TR></TBODY></TABLE><P>Refresh agrobacterium cultures for tomorrow infiltration:</P><UL><LI>Sip-CH2</LI><LI>Sip-CH2-CH3</LI><LI>Pnos</LI><LI>Red toggle (PIF6:PhyB:luc )</LI><LI>Renilla</LI><LI>Blue toggle (….:KDronpa:NDronpa:ren:luc)</LI></UL><UL><LI>A new experiment was started:</LI><UL class="ul_2"><LI>We are going to check again the recombinase BxbI and its reporter (negative control). </LI></UL></UL><UL><LI>Test the recombinase PhiC31. Due to that we did not obtain yet the complete recombinase, we will coinfiltrate the PhiC31 and the RepPhiC31:GFP following the same scheme as BxbI. 2 plants with 2 leafs, one of them with PhiC31 + RepPhiC31:GFP and the other with only RepPhiC31:GFP.</LI><LI>Infiltration in 2 plants with 2 leafs each of them with interferon.</LI></UL><P>Measure f the ODs:</P><DIV class="table-wrapper"><TABLE class="alt"><TBODY><TR><TD>Construction</TD><TD>OD</TD><TD>Volume (ml)</TD></TR><TR><TD>IFN</TD><TD>0.13</TD><TD>1.54</TD></TR><TR><TD>RepBxbI:GFP</TD><TD>0.13</TD><TD>1.54</TD></TR><TR><TD>BxbI:RepBxbI:GFP</TD><TD>0.33</TD><TD>0.61</TD></TR><TR><TD>PhiC31</TD><TD>0.19</TD><TD>1.05</TD></TR><TR><TD>RepPhiC31:GFP</TD><TD>0.22</TD><TD>0.91</TD></TR></TBODY></TABLE><H3 style="color:green">8 August 2015</H3><P>Transform the ligation into E.coli and make petri dishes cultures:</P><UL><LI>PhiC31; pUPD2.</LI></UL><UL><LI>LexA:AsLOVpep+ePDZ (C1-C5)</LI><LI>LacI:AsLOVpep+ePDZ (C1-C4)</LI><LI>Gal4:AsLOVpep+ePDZ (C1-C4)</LI></UL><P>New experiment:</P><P>It will be tested again the red toggle (PIF6:PhyB:luc + ren), the blue toggle (LacI:KDronpa:NDronpa:ren:luc), pnos  (positive control to check the ratio renilla luciferasa) and the production of antirotavirus that are SIP rotavirus CH2 and SIP rotavirus CH2-CH3.</P><P>2 plants with 3 leafs ache one were infiltrated with pnos.</P><P>The same with the red(4) and blue toggle(4) and sip rativurs I DON’T KNOW</P><P>Measurement of the ODs:</P><DIV class="table-wrapper"><TABLE class="alt"><TBODY><TR><TD>Construction</TD><TD>OD</TD><TD>Volume (ml)</TD></TR><TR><TD>Sip-CH2</TD><TD>0.04</TD><TD>6.667</TD></TR><TR><TD>Sip-CH2-CH3</TD><TD>0.04</TD><TD>6.667</TD></TR><TR><TD>Red toggle (PIF6:PhyB:luc)</TD><TD>0.09</TD><TD>15</TD></TR><TR><TD>Blue toggle (LacI:KDronpa:NDronpa:ren:luc)</TD><TD>0.09</TD><TD>15.00</TD></TR><TR><TD>Renilla</TD><TD>0.04</TD><TD>6.667</TD></TR><TR><TD>Pnos</TD><TD>0.04</TD><TD>6.667</TD></TR></TBODY></TABLE><H3 style="color:green">9 August 2015</H3><P>Make a colony PCRs with the 10 white colonies of PhiC31 that grow in the petri dishes.</P><UL><LI>DNA- colony</LI><LI>JM1: 2 µl (dilution 1:10)</LI><LI>JM2: 2 µl (dilution 1:10)</LI><LI>Buffer Taq (with Mg): 2 µl</LI><LI>dNTPs: 2.5 µl</LI><LI>Taq: 0.5 µl</LI><LI>H2O: 1 µl</LI></UL><P>Minipreps of the liquid cultures:</P><UL><LI>LexA:AsLOVpep+ePDZ (C1-C4) C5 has turn into blue color.</LI><LI>LacI:AsLOVpep+ePDZ (C1-C4)</LI><LI>Gal4:AsLOVpep+ePDZ (C1-C4)</LI></UL><P>Digestion of the minipreps:</P><DIV class="table-wrapper"><TABLE class="alt"><TBODY><TR><TD>LexA:AsLOVpep+ePDZ</TD><TD>BamHI</TD><TD>6674, 4309</TD></TR><TR><TD>LacI:AsLOVpep+ePDZ</TD><TD>BamHI</TD><TD>6674, 5041</TD></TR><TR><TD>Gal4:AsLOVpep+ePDZ</TD><TD>BamHI</TD><TD>6674, 4270</TD></TR></TBODY></TABLE><P>Make the gel:</P><DIV class="table-wrapper"><TABLE class="alt"><TBODY><TR><TD>LexA:AsLOVpep+ePDZ (C1)</TD><TD>LexA:AsLOVpep+ePDZ (C2)</TD><TD>LexA:AsLOVpep+ePDZ (C3)</TD><TD>LexA:AsLOVpep+ePDZ (C4)</TD></TR><TR><TD>Ok</TD><TD>ok</TD><TD>ok</TD><TD>ok</TD></TR><TR><TD>oLacI:AsLOVpep+ePDZ (C1)</TD><TD>LacI:AsLOVpep+ePDZ (C2)</TD><TD>LacI:AsLOVpep+ePDZ (C3)</TD><TD>LacI:AsLOVpep+ePDZ (C4)</TD></TR><TR><TD>Ok</TD><TD>ok</TD><TD>ok</TD><TD>ok</TD></TR><TR><TD>Gal4:AsLOVpep+ePDZ (C1)</TD><TD>Gal4:AsLOVpep+ePDZ (C2)</TD><TD>Gal4:AsLOVpep+ePDZ (C3)</TD><TD>Gal4:AsLOVpep+ePDZ (C4)</TD></TR><TR><TD>Ok</TD><TD>ok</TD><TD>ok</TD><TD>Ok</TD></TR></TBODY></TABLE><P>We keep in our inventory the colony C1 of each construction</P><P>Make a gel of the colonies PCRs of PhiC31: we did not observed any DNA. Still can’t fix the problem.	</P><P>Make liquid culture of renilla and PIF6:phyB:luc in <I>Agrobacterium</I> because the last time that we did an experiment they have grown slowly.</P><P>Store in the 4ºC fridge an agrobacterium culture with P19.</P><P>Ligations:</P><DIV class="table-wrapper"><TABLE class="alt"><TBODY><TR><TD>LexA:AsLOVpep+ePDZ+ren; ?1</TD><TD>LacI:AsLOVpep+ePDZ+ren; ?1</TD><TD>Gal4:AsLOVpep+ePDZ+ren; ?1</TD></TR><TR><TD>1µl LexA:AsLOVpep+ePDZ</TD><TD>1µl LacI:AsLOVpep+ePDZ</TD><TD>1µl Gal4:AsLOVpep+ePDZ</TD></TR><TR><TD>1 µl renilla (159)</TD><TD>1 µl renilla (159)</TD><TD>1 µl renilla (159)</TD></TR><TR><TD>1 µl ?1</TD><TD>1 µl ?1</TD><TD>1 µl ?1</TD></TR><TR><TD>4.6 µl H2O</TD><TD>4.6 µl H2O</TD><TD>4.6 µl H2O</TD></TR></TBODY></TABLE><H3 style="color:green">10 August 2015</H3><P>Transform into E.coli this ligations and make petri dishes cultures:</P><UL><LI>LexA:AsLOVpep+ePDZ+ren; ?1</LI><LI>LacI:AsLOVpep+ePDZ+ren; ?1</LI><LI>Gal4:AsLOVpep+ePDZ+ren; ?1</LI><LI>(we add into the tubes X-gal and IPTG)</LI></UL><P>Make liquid culture of 11 colonies of PhiC31 due to that this construction was giving us problems to obtain.</P><P>Refresh the agrobacterium cultures of:</P><UL><LI>BxbI:Rep:BxbI:GFP</LI><LI>Rep:BxbI:GFP</LI><LI>PhiC31</LI><LI>RepPhiC31:GFP</LI><LI>P19</LI><LI>Citoplasm</LI><LI>Integrase</LI><LI>Dsred</LI><LI>GFP</LI></UL><P>Take the samples of the agroinfiltrated plants that were in darkness for 2 days and make discs of them to change the conditions.</P><P>Make liquid culture of:</P><UL><LI>LexA:AsLOVpep+ePDZ+ren (C1 and C2)</LI><LI>LacI:AsLOVpep+ePDZ+ren (C1-C4)</LI><LI>Gal4:AsLOVpep+ePDZ+ren (C1 and C2)</LI><LI>(we add X-Gal and IPTG because they  are small)</LI></UL><H3 style="color:green">11 August 2015</H3><P>Primers have arrived, they have been resuspended. They were used to eliminate the recognitions sites of the enzymes in BioBricks. This will make our constructions ready to be send to the iGEM</P><P>Minipreps of the liquid cultures done yesterday:</P><UL><LI>PhiC31 (C6-C16)</LI><LI>LacI:AsLOVpep+ePDZ+ren (C1-C4)</LI><LI>Gal4:AsLOVpep+ePDZ+ren (C1 and C2)</LI><LI>(LexA:AsLOVpep+ePDZ+rencolonies were all blue)</LI></UL><P>Make PCRs with all the primers and constructions:</P><P>All of them follow the same composition which is:</P><DIV class="table-wrapper"><TABLE class="alt"><TBODY><TR><TD>10 µl </TD><TD>Buffer HF</TD></TR><TR><TD>26.5 µl</TD><TD>H2O</TD></TR><TR><TD>2 µl</TD><TD>dNTPs</TD></TR><TR><TD>0.5 µl</TD><TD>Taq Phusion</TD></TR><TR><TD>1 µl </TD><TD>DNA (dilution 1:10)</TD></TR><TR><TD>2.5 µl</TD><TD>Primer forward (F) (dilution 1:10)</TD></TR><TR><TD>2.5 µl</TD><TD>Primer reverse (R) (dilution 1:10)</TD></TR></TBODY></TABLE><P>*green ones are the only specify.</P><DIV class="table-wrapper"><TABLE class="alt"><TBODY><TR><TD>IFN (1)</TD><TD>IFN (2)</TD><TD>AsLOVpep (1)</TD><TD>AsLOVpep (2)</TD></TR><TR><TD>IFN</TD><TD>IFN</TD><TD>AsLOVpep</TD><TD>AsLOVpep</TD></TR><TR><TD>Mys int F</TD><TD>IFN domBB R</TD><TD>AsLOVpep F</TD><TD>AsLOVpep Fint</TD></TR><TR><TD>Mys int R</TD><TD>IFN domBB F</TD><TD>AsLOVpep Rint</TD><TD>AsLOVpep R</TD></TR><TR><TD>AsLOVpep (nested)</TD><TD>PhiC31 (1)</TD><TD>PhiC31 (2)</TD><TD>PhiC31 (3)</TD></TR><TR><TD>AsLOVpep</TD><TD>PhiC31</TD><TD>PhiC31</TD><TD>PhiC31</TD></TR><TR><TD>AsLOVpep nested</TD><TD>PhiC31 Fint 1</TD><TD>PhiC31 Fint 2</TD><TD>PhiC31 Fint 3</TD></TR><TR><TD>AsLOVpep</TD><TD>PhiC31 Rint 2</TD><TD>PhiC31 Rint 3</TD><TD>PhiC31 R</TD></TR><TR><TD>PhiC31</TD></TR><TR><TD>PhiC31</TD></TR><TR><TD>PhiC31 F</TD></TR><TR><TD>PhiC31 Rint</TD></TR></TBODY></TABLE><P>Digestions of the minipreps:</P><DIV class="table-wrapper"><TABLE class="alt"><TBODY><TR><TD>PhiC31</TD><TD>NotI</TD><TD>2046, 1899</TD></TR><TR><TD>LacI:AsLOVpep+ePDZ+ren </TD><TD>EcoRI</TD><TD>6345, 8798</TD></TR><TR><TD>Gal4:AsLOVpep+ePDZ+ren </TD><TD>EcoRI</TD><TD>6345, 9569</TD></TR></TBODY></TABLE><P>Make the gel:</P><DIV class="table-wrapper"><TABLE class="alt"><TBODY><TR><TD>PhiC31 (C6)</TD><TD>C7…C16</TD><TD>LacI:AsLOVpep:ePDZ:ren (C1)</TD><TD>C2</TD><TD>C3…C4</TD></TR><TR><TD>no</TD><TD>no</TD><TD>ok</TD><TD>no</TD><TD>ok</TD></TR><TR><TD>Gal4:AsLOVpep+ePDZ+ren (C1)</TD><TD>Gal4:AsLOVpep+ePDZ+ren (C2)</TD></TR><TR><TD>ok</TD><TD>ok</TD></TR></TBODY></TABLE><P>Mesurement of the ODs:</P><DIV class="table-wrapper"><TABLE class="alt"><TBODY><TR><TD>Citoplasm</TD><TD>0.21</TD><TD>7.35 ml</TD></TR><TR><TD>RepPhiC31</TD><TD>0.26</TD><TD>9.1ml</TD></TR><TR><TD>35S:LTB:T35S</TD><TD>0.27</TD><TD>9.45ml</TD></TR><TR><TD>PhiC31</TD><TD>0.27</TD><TD>9.45ml</TD></TR><TR><TD>GFP</TD><TD>0.20</TD><TD>10ml</TD></TR><TR><TD>RepBxbI</TD><TD>0.33</TD><TD>11.55ml</TD></TR><TR><TD>PsinATG:RepBxbI:GFP</TD><TD>0.36</TD><TD>12.6ml</TD></TR><TR><TD>PhiC31</TD><TD>0.34</TD><TD>11.9ml</TD></TR><TR><TD>DsRed</TD><TD>0.26</TD><TD>9.1ml</TD></TR><TR><TD>P19</TD><TD>0.28</TD><TD>9.8ml</TD></TR></TBODY></TABLE><H3 style="color:green">12 August 2015</H3><P>New digestions to check if the negative control constructions are correct and we can transformed it in agro</P><DIV class="table-wrapper"><TABLE class="alt"><TBODY><TR><TD>OpLexA:luc:SF:ren</TD><TD>EcoRI</TD><TD>6345, 7274</TD></TR><TR><TD>Op:UAS:luc:SF:ren</TD><TD>EcoRI</TD><TD>6345, 7096</TD></TR><TR><TD>OpEtr8.luc:SF:ren</TD><TD>EcoRI</TD><TD>6345, 7294</TD></TR></TBODY></TABLE><P>Gel:</P><DIV class="table-wrapper"><TABLE class="alt"><TBODY><TR><TD>Op:UAS:luc:SF:ren C1</TD><TD>Op:UAS:luc:SF:ren C2</TD><TD>Op:UAS:luc:SF:ren C3</TD><TD>OpEtr8.luc:SF:ren</TD></TR><TR><TD>no</TD><TD>no</TD><TD>no</TD><TD>no</TD></TR><TR><TD>OpLexA:luc:SF:ren C1</TD><TD>OpLexA:luc:SF:ren C2</TD><TD>Partner dna</TD><TD>Partner dna</TD></TR><TR><TD>no</TD><TD>no</TD></TR></TBODY></TABLE><P>Transform in Agro and make petri dishes cultures:</P><UL><LI>LexA:AsLOVpep:ePDZ</LI><LI>Gal4:AsLOVpep:ePDZ</LI><LI>LacI:AsLOVpep:ePDZ</LI><LI>LexA:PIF6:phyB:VP16</LI><LI>Gal4:PIF6:phyB:VP16</LI><LI>LacI:PIF6:phyB:VP16</LI><LI>All of them are in ?1.</LI></UL><P>Ligations:</P><DIV class="table-wrapper"><TABLE class="alt"><TBODY><TR><TD>LexA:AsLOVpep:ePDZ+ren; ?1</TD><TD>Gal4:AsLOVpep:ePDZ:ren+OpUAS:luc; ?1</TD><TD>LacI:AsLOVpep:ePDZ:ren+OpLacI:luc; ?1</TD></TR><TR><TD>1 µl LexA:AsLOVpep:ePDZ</TD><TD>1 µl Gal4:AsLOVpep:ePDZ:ren</TD><TD>1 µl LacI:AsLOVpep:ePDZ</TD></TR><TR><TD>1 µl renilla (159)</TD><TD>1 µl OpUAS:luc</TD><TD>1 µl OpLacI:luc</TD></TR><TR><TD>1 µl ?1</TD><TD>1 µl ?1</TD><TD>1 µl ?1</TD></TR><TR><TD>4.6 µl H2O</TD><TD>4.6 µl H2O</TD><TD>4.6 µl H2O</TD></TR></TBODY></TABLE></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></TBODY></TABLE></DIV></TBODY></TABLE></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></SECTION></DIV></DIV></SECTION><SECTION id="footb"><FOOTER id="foot"><SECTION><DIV id="contact"><H3>Contact</H3></DIV><DIV id="sponsor"><H3>Sponsors</H3></DIV></SECTION></FOOTER></SECTION></DIV></DIV></DIV></DIV></BODY></HTML>