Team:Valencia UPV/Notebook/Content
Valencia UPV iGEM 2015
iGEM 2015 | Valencia UPV
Home
Project
Overview
Circuit
Components
Modeling
Results
Notebook
Modeling
Overview
Deterministic model
Simulations
Light control
Conclusions
Policy and Practicies
Overview
SynBiocraft
Safety
Collaborations
Magic Lamp
Overview
Device: Description
Device: Electronics
How To
Team
Members
Attributions
Acknowledgments
Parts
Achievements
Medals
Achievements
Notebook
Protocols
Daily notebook
Daily notebook
Here we present you all the procedures we did to develop our project. On this page you can find the notebook contents. If preferred, you can go directly to the protocols by pressing in the buttons above or below. We hope you enjoy reading our incredible journey!
June
27 May 2015
Starts our work in the lab!
Marta, a lab mate gives us a construction, the red toggle swich (E:PIF6:PhyB:VP16:Etr8:luc), we just have to add the renilla; α2 (GB160) to test it.
Make the ligation (step 2 in the protocol):
E:PIF6:PhyB:VP16:Etr8:luc+ren; Ω1
1µL E:PIF6:PhyB:VP16:Etr8:luc; α1
1µL renilla; apha2
1µL Ω1
6,8µL H2O
28 May 2015
Electroporation (step 3 in the protocol) of E. coli to insert our first construction.
Make a petri dish culture with a LB-Agar plate with streptomycin.
29 May 2015
There is no white colonies, we electroporate again and make petri dish culture.
30 May 2015
There was just one white colony, make the ligation again.
E:PIF6:PhyB:VP16:Etr8:luc+ren; Ω1
0.5µL E:PIF6:PhyB:VP16:Etr8:luc; α1
1µL renilla; apha2
1µL Ω1
7.2µL H2O
1 June 2015
Electroporation of the new ligation.
2 June 2015
There are white colonies. Make 2 liquid cultures of them (Step 4 in the protocol). Add 3.5ml of LB and 3.5µL of spectomycin.
Make liquid culture of just some glycerinates:
α1
α2
Ω1
Ω2
pUPD2
E:PIF6:NLS; pUPD2 (GB0288)
E:PIF6:NLS; α1 (GB892)
E:PIF6:NLS; Ω2 (GB893)
E:PIF6:NLS:luc:PhyB; α1 (GB896)
Luc:PhyB; Ω1 (GB890)
PhyB:VP16; pUPD2 (GB289)
PhyB:VP16; α2 (GB88E)
Etr8:CMVmini; pUPD2 (GB1097)
OpLexA:mini35S; pUPD2 (GB733)
OpLexA:mini35S:luc:Tnos; α2
LexABD; pUPD2 (GB0732)
LacI for N-Tfusion; pUPD2 (GB858)
Linker:LacIBD; pUPD2 (GB704)
OpLacI:mini35S:luc:Tnos; α2 (GB152)
OpLacI:mini35S; pPUD2 (GB534)
3 June 2015
Al cultures have grown except for Ω2. Make minipreps (Step 5 in the protocol).
Digestion of the minipreps (Step 6 of the protocol).
α1
-
-
α2
-
-
Ω1
-
-
pUPD2
-
-
E:PIF6:NLS; pUPD2 (GB0288)
EcoRI
3000, 1000
E:PIF6:NLS; α1 (GB892)
EcoRI
6300, 2500
E:PIF6:NLS; Ω2 (GB893)
EcoRV
1800, 6600, 900
E:PIF6:NLS:luc:PhyB; α1 (GB896)
EcoRI
3600, 6300, 5600
Luc:PhyB; Ω1 (GB890)
BamHI
2300, 6300, 4200
PhyB:VP16; pUPD2 (GB289)
EcoRI
3000, 2000, 500
PhyB:VP16; α2 (GB88E)
HindIII
2100, 6300, 1800
Etr8:CMVmini; pUPD2 (GB1097)
EcoRI
3000, 480
OpLexA:mini35S; pUPD2 (GB733)
EcoRI
3000, 460
OpLexA:mini35S:luc:Tnos; α2 (GB151)
HindIII
2500
LexABD; pUPD2 (GB0732)
EcoRI
3000, 300
LacI for N-Tfusion; pUPD2 (GB858)
EcoRI
3000, 1000
Linker:LacIBD; pUPD2 (GB704)
EcoRI
3000, 1000
OpLacI:mini35S:luc:Tnos; α2 (GB152)
HindIII
2500, 2600
OpLacI:mini35S; pPUD2 (GB534)
EcoRI
3000, 560
E:PIF6:PhyB:VP16:Etr8:luc+ren; Ω1
BamHI
3700, 6100, 6600, 4200
E:PIF6:PhyB:VP16:Etr8:luc+ren; Ω1
EcoRV
11000, 400, 2500, 3000, 4000
Make the gel:
pUPD2
Alf
Alpha1
288
289
534
ok
ok
ok
ok
ok
?
704
732
733
858
892
896
ok
ok
ok
ok
ok
ok
1097
Alpha2
88E
151
152
Omega1
ok
ok
ok
ok
ok
ok
890
893
Red toggle (C1) (EcoRV)
Red toggle (C1) (BamHI)
Red toggle (C2) (EcoRV)
Red toggle (C2) (BamHI)
ok
ok
ok
no
no
no
4 June 2015
Ask for the NDronpa sequence. This will be part of our blue toggle.
This piece is known by reading the paper ‘Reversible photoswichable Dronpa-1 monitors nucleocytoplasmic transport of an RNA-binding protein in transgenic plants?(Doi: 10.111/j.1600-0854.2011.01180.lambda.).
The sequence of NDronpa is plant optimised and avoid cryptic sequences. We have domesticated this sequence with a linker in N-terminal to allow us to join it to a binding domain and also we had a NLS in the C-terminal to transport itself to the nucleus. It is domesticated as B5 part for Golden Braid assembling.
After obtaining the sequence we compare the protein in Uniprot and we can observed that our sequence add a V in the position 2. We compare this results with other papers and none of them has this addition. When we compare this sequence with the paper ?Optical control protein activity by fluorescent protein domains?(Doi: 10.1126/science.1226854)  we observed that our position 146 is the position 145 and as what we want is the interaction caused by the N145-K145, we eliminate the V. We also eliminate a pair of amino acids at the end of the sequence following the same criteria.
Once obtained both variants of Dronpa, we decided to add the binding domain to KDronpa and the activation to NDronpa as this last one tetramerizes and all operator sequence are repeated in our promoters.
5 June 2015
We had 2 cultures from the last day, corresponding to other 2 colonies of ligation.
Agrobacterium culture of promoter less: Luciferase + Renilla
Minipreps
Digestion with BamHI and EcoRV
Agarose gel 1%
How to ask and make primers?
Select the sequence to amplify and save in FASTA format.
gbCloning, go to Tools-Domesticator-1?Category
Add FASTA and select parts.
On the protocol we have the primers
The oligos they give us:
4 first nucleotides: so the enzyme can recognize without problems
6 following bingind sites.
1 extra nucleotide.
4 overhangs.
Meeting with Daniel Ramón (Biopolis).
Ligations:
PIF6 + PhyB; Ω1
Etr8(CMV)+BxbI:T35S; α1
1µL (GB892) PIF; α1
1µL (GB1097) Etr8(CMV); pUPD2
1µL (GB88E) PhyB; α2
1µL BxbI; pUPD2
1µL Ω1
1µL Tnos pUPD2
6.8µL H2O
1µL α1
5.8µL H2O
Digestions:
(GB160) 35S:Renilla:tNOS-35S:P19:tNOS
EcoRV
2475, 381, 4601
(GB896) Luc:PIF6:PhyB
EcoRV
11608, 3942
6 June 2015
Transform to E. coli from PIF+Phy and BxbI and make petri dish cultures.
Digest of 160, 289 and the two ligations, PIF+phy and Etr8+BxbI.
Agarose gel.
GB160
289
PIF+PhyB
BxbI
ok
no
?
?
7 June 2015
We’ve got white colonies from PIF+Phy and BxbI!
Pick two colonies from each construction.
8 June 2015
Minipreps of the 4 liquid cultures and digestion to see the band patterns.
Digestion:
Etr8(CMV):Bxb1:Tnos; α1
EcoRI
6345, 238
EPIF6 + PhyB-PV16; Ω1
BamHI
6686, 1439, 2685, 2237
Agarose gel was made:
BxbI (C1)
BxbI (C2)
E:PIF6+PhyB-VP16 (C1)
E:PIF6+PhyB-PV16 (C2)
ok
ok
no
no
Repeat digestion because we are not sure of the last digestions.
We don’t have sure the toggle, so we decide to repeat the digestion with other enzyme tomorrow, noticing that the colony 2 has better bands pattern.
Optimized ligation:
PIF-PhyB-Luc-Renilla-P19
1 µL vector
0.8 µL dilution ?GB160
1.7 µL PIF:PhyB
4.15 µL H2O
Ratio 1:2 vector insert
As BxbI was good at the digestion we put 1 µL of LB and 1 µL of Kanamicyne on the tube where it had grown and store at 37ºC to glycerinate later.
We design primers to binding domain (BD) and PIF.
Problem: domesticator is introduced in an old pUPD2. The new one has different bases.
Change manually the pUPD2 bases in the program (Benchling).
9 June 2015
Digestion of the ligation of yesterday containing: EPIF6-PhyB-VP16 (C1 y C2)
EPIF6-PhyB-VP16
PvuII (green buffer)
3663, 9472pb
Agarose gel 1%:
EPIF6-PhyB-VP16 (C1)
EPIF6-PhyB-VP16 (C2)
no
no
We see three bands: 7000, 4000, 1900pb
Transform optimized ligation PIF-Phy-Luc-Renilla-P19 and make petri dish cultures.
10 June 2015
Check the primers and order LexA, Gal4, PIF6, LacI, Dronpa.
Check linker VP16 (88E) and make a primer for it.
Take out glycerinate of Ω2.
Alfredo’s part is not working.
Make liquid culture of E:PIF6:PhyB:VP16:luc:ren (C1-C3).
Minipreps of liquid culture (PIF + Phy), colonies C3, C4, C5, C6
Digestion:
PIF+Phy:VP16
PvuII (buffer green 10x)
3663, 9472
PIF+Phy:VP16
BamHI
1939, 2685, 2337, 6674
Agarose gel 1%:
PIF + Phy (PvuII) C3
PIF + Phy (PvuII) C4
PIF + Phy (PvuII) C5
PIF + Phy (PvuII) C6
no
ok
no
No
PIF + Phy (BamHI) C3
PIF + Phy (BamHI) C4
PIF + Phy (BamHI) C5
PIF + Phy (BamHI) C6
no
ok
no
No
Transformation into AgrobacteriumEPIF6-PhyB-VP16 + luciferase (GB896) and make petri dish culture. We are not going to have the positive control (renilla+P19) and we won’t be able to quantify and make ratios.
11 June 2015
Minipreps of the culture:
Digestion:
E:PIF6:PhyB:VP16:luc:ren
BamHI
4209, 3756, 6100, 6674
EcoRV
3942, 2989, 2475, 381, 10952
Gel:
PIF6:PhyB:VP16:luc:ren C1 (BamHI)
PIF6:PhyB:VP16:luc:ren C3 (BamHI)
PIF6:PhyB:VP16:luc:ren C1 (EcoRV)
PIF6:PhyB:VP16:luc:ren C3 (EcoRV)
no
no
no
no
Transformation into Agrobacteriumof Renilla (GB160) because we could not join this construction with PIF:PhyB and so we will do a cotransfection of both plasmids.Make petri dish culture.
12 June 2015
The petri dish with PIF:PhyB:luc was taken out the 37ºC room and put into the fridge to pick colonies tomorrow.
13 June 2015
Pick colonies to make liquid culture:
Renilla in agrobacterium: just one colony, it was made liquid culture but check carefully the gel.
It was noticed that the piece 160, renilla, needs a pSub plasmid to replicate itself so we will transform 160 into a agrobacterium with this plasmid (C58 pSub).
BxbI; α1+PhyB; α2
1µl BxbI
1 µl PhyB
1 µl Ω2
4.6 µl H2O
Transform renilla (160) with pSub plasmid into agrobacterium and make petri dish culture.
15 June 2015
Repeat the ligation BxbI+35S:E-PIF6:tnos because PIF was Ω2
BxbI + 35S:E-PIF6:tnos; Ω1
1µl BxbI
1 µl PhyB
1 µl Ω1
4.6
µl H2O
KDronpa has arrived:
Centrifuge it 2-5sec at maximum velocity.
Add 50 µl to have a concentration of 20ng/µl
Mix it with the vortex and spin.
Ligation:
KDronpa; pUPD2
1 µl KDronpa
1 µl pUPD2
5.6 µl H2O
It was not possible to pick colonies of the Agrobacterium transformed with renilla because they did not grow. Maybe the problem is that with tetraciclyn bacterias grow slowly. Wait 1 day more.
Transformation of the ligation, BxbI+35S:E-PIF6:tnos; Ω1, into E. coli.Make petri dish culture.
16 June 2015
Transformation of the ligation, KDronpa, into E. coli.
Pick colonies of BxbI:E-PIF6 and make liquid culture (C1-C3).
Primers had arrived, it has been done the resuspension (dilution 1:10) of all of them.
Primers
Code
Template
Working temperature (ºC)
LacI F
1
LacI (858)
69.7
LacI R
2
Gal4 F
3
We did not take out the glicerynate.
63.2
Gal4
4
LexA F
5
LexA (732)
62.7
LexA R
6
PIF:VP16 F
7
PIF6 (288)
60.1
PIFVP16 R
8
NDronpa F1
9
Kdronpa
67.7
NDronpa R1
10
Dronpa F2
11
58.5
NDronpa R2
12
A PCR with all the primers and the fragments was done, the samples were put in order following the temperature gradient.
The templates were in dilution 1:50, exception of KDronpa that was dilution 1:5 and the primers 1:10.
PCR Fusion Taq (50µl)
DNA template (10 µg/µl)
0.5 µl fusion taq
2.5 µl primer F
2.5 µl primer R
2 µl NTPs
31.5 µl H2O
17 June 2015
Pick colonies and make liquid culture of:
KDronpa (C1-C4)
Ligations with the PCR’s products:
Templates PCR: 1+2, 5+6, 7+8PIF, 7+8VP16, 9+10, 11+12.
Template PCR; pUPD2
0.5µl template
1µl pUPD2
6.1µl H2O
Minipreps of liquid cultures:
BxbI:E-PIF6 (C1-C3)
Agarose gel with the PCRs:
Template
1+2
5+6
7+8PIF
7+8VP16
9+10
11+12
Band pattern
1017
284
391
478
464
290
Gel result
ok
ok
ok
ok
No DNA
ok
Transformation in E. coli of the correct ligations and make petri dishes cultures:
1+2, 5+6, 7+8PIF, 7+8VP16, 11+12
18 June 2015
Minipreps of the liquid cultures:
KDronpa (C1-C4)
Digestions:
KDronpa EcoRI 2800
Gel:
Kdronpa C1
Kdronpa C2
Kdronpa C3
Kdronpa C4
no
no
ok
no
Etr8:BxbI:phyB C1
Etr8:BxbI:phyB C2
Etr8:BxbI:phyB C3
No
no
no
We discovered that the construction with BxbI did not go well because our lab college gives us the wrong piece. Thanks Alfredo :)
Take glicerynates out:
Gal4; pUPD2 (GB731)
Ω2
NoATGPromoter (GB00552)
Renilla (GB160)(GB159)(GB109)
PCR:
NDronpa
2.5 µl (9+10) primer F
2.5 µl (11+12) primer R
2 µl NTPs
0.2 µl Taq
10 µl Buffer
31.5
µl H2O
Ligations:
Etr8:BxbI:T35S; α1
Template PCR; pUPD2
1 µlEtr8
0.5µl template
1 µl BxbI
1µl pUPD2
1 µl T35S
6.1µl H2O
1 µl α1
5.8 µl H2O
Templates PCR: 1+2, 5+6, 7+8PIF, 7+8VP16
19 June 2015
We do a PCR with the normal Taq polymerase.
1µl of DNA’s template (9+10, 9+12 and 11+12)
2µl of specific buffer
2µl of NTPs
1µl primer forward
1µl primer reverse
0.5 µl of Taq
12.5 µl H2O
These quantities multiplied by 3.
Minipreps of the yesterday’s glycerinated cultures.
Gal4; pUPD2 (GB731)
Ω2
NoATGPromoter (GB00552)
Renilla (GB160)(GB159)(GB109)
Do the glycerinates digestions:
Minipreps:
Enzime
Band pattern
(GB159) pDGB1_Ω2 renilla
EcoRV
2909, 2475,882, 812, 381
Entry vector, Ω2
EcoRV
6652, 621
(GB552) pP35s NoATG; pUPD2
EcoRI
2997, 1090
(GB160) renilla pDGB1, α2
EcoRV
4601, 2475, 381
(GB731) Gal4BD (CDS); pUPD2
EcoRI
2997, 2493
(GB109)
355:renilla:Tnos; α1
EcoRI
2580, 2493
We make an agarose gel with the digestions made before and the PCR of KDronpa.
159
160
Ω2
552
731
109
9+10
9+12
11+12
ok
ok
ok
ok
ok
ok
no
ok
ok
Transformation into E. coli of ligations:
1+2, 5+6, 7+8PIF, 7+8VP16 all in pUPD2
We made an stack of Cloranfenicol petri dishes
250ml  LB agar
X-Gal (1:500): 500 µl
IPTG (1:1000): 250 µl
Cloranfenicol (1:2000): 125 µl
20 june 2015
We have white colonies of renilla! Also of Etr8+BxbI; α1
We have also pUPD2 colonies but they are so close to the blue ones that we can’t pick anyone.So we make strakes.
We make a liquid culture of Agrobacteriumof Renilla (rif/kan/tetr).
21 June 2015
Pick colonies and make liquid culure of (all colonies are in pUPD2):
Plates : PIF (17/06/15) (C1 and C2)
VP16 (C1 and C3)
LacI (C1-C3)
Plates (19/06/15): BxbI (C1, C2, C3),
VP16 (C4, C5)
LacI (C1, C2)
PIF (C1-C5)
LexA (C1, C2)
We take out two glicerynates of GFP and BFP (of the Alfredo’s box)
22 June 2015
We made minipreps of the liquid culture of the day before:
LacIBD; pUPD2 (C1-C5)
LexABD; pUPD2 (C1, C2)
Etr8(CMV):Bxb1 (C1-C3)
PIF6; pUPD2 (C1-C5)
VP16; pUPD2 (C1, C4, C5)
Make the digestions of all the minipreps:
LacIBD, pUPD2
NotI
2046, 1053
LexABD, pUPD2
NotI
2046, 321
Etr8(CMV):Bxb1
NotI
1532, 1290, 5896
PIF6,pUPD2
NotI
2046, 407
VP16, pUPD2
NotI
2046, 500
Refresh the viral system that a lab mate borrow to us. This is going to be use to agroinfiltrate some plants to make some cool draws to sent to a TV programm so they can watch what are we doing. This cultures consist of three parts divided in three Agrobacteriumcolonies. They are the citoplasm, the fluerescent protein (GFP, DsRed or YFP) and the integrase, in our case PhiC31.
We received the reporter BxbI (RepBxbI)!
500ng of sample
Centrifuge at 3000rpm for 5 seconds (spin).
Add 50 µl H2O
Shake it and let at 50ºC for 20min
Make a PCR of Gal4 and NDronpa (9-10), the primers of NDronpa are aliquoted.
Make an agarose gel with all the digestions:
LacI C1
LacI C2
LacI C3
LacI C4
LacI C5
LexA C1
LexA C2
BxbI C1
BxbI C2
BxbI C3
Ok
ok
ok
ok
ok
no
no
ok
ok
no
PIF C1
PIF C2
PIF C3
PIF C4
PIF C5
VP16 C1
VP16 C4
VP16 C5
No
no
-
ok
ok
ok
ok
ok
Gal4
NDronpa 1
NDronpa 2
FOTO
We make ligations of:
Etr8(CMV):BxbI; α1 + PhyB:VP16;α2; Ω1
LacIBD;pUPD2 + PIF6BDPless; pUPD2; α1
KDronpa;pUPD2 + LacIBD; pUPD2; α1
Gal4BD; pUPD2
Reporter of BxbI; pUPD2
Tomorrow we have to take out pUPD2 of constitutive promoters, terminators and GFP (CDS).
23 June 2015
Transform into E.Coli the 5 ligations done yesterday and two more transformations of 5+6(1) and 5+6(2) which are the ligations in pUPD2 of the 18/06.
Etr8(CMV):BxbI; α1 + PhyB:VP16;α2; Ω1
LacIBD;pUPD2 + PIF6BDPless; pUPD2; α1
KDronpa;pUPD2 + LacIBD; pUPD2; α1
Gal4BD; pUPD2
Reporter of BxbI; pUPD2
LexABD (5+6), pUPD2 (1 and 2)
We have taken out of the -80ºC fridge the glycerinate of GFP; pUPD2 (GB0059)/ampicilin.
The liquid culture of Renilla (ryfampicin/kanamycin/tetracyclin) does not grow after the two days required. So we decide to refresh two new colonies, one of them in a tube with the three antibiotics and another with rifampicina and kanamicine. Asun says that the tetracycline slow down the growth of Agro.
The 4 liquid cultures of LexA+IPTG/+gal are all blue: throw them.
We ordered again the primer n?0 (NDronpa R1). Changing one codon in 3?and delete another in 5?
24 June 2015
Pick colonies of the plates done yesterday and pass them into a liquid medium:
LacIBD+PIF; α1 (C1, C2)
Gal4BD; pUPD2 (C1)
RepBxb1; pUPD2 (C1-C3)
LacIBD+KDonpa; α1 (C1, C2)
Etr8(CMV)+BxbI+PhyB+VP16; Ω1 (C1)
LexABD1; pUPD2 (C1-C4)
LexABD2; pUPD2. No colonies.
The viral systems of Agrobacteriumcultures to make the color mosaics are ready after 2 days at 28ºC. We can make the agroinfiltration.
Protocol to prepare solution to agroinfiltrate in the protocols notebook part.
Ligation:
ETR8(CMV):BxbI; α1+PhyB:VP16; α2; Ω1
Gal4BD(pcr) + pUPD2
1.5 µl Etr8:BxbI
1 µl Gal4 PCR
1.5 µl 88E (PhyB:VP16)
1 µl pUPD2
1 µl Ω1
5,6 µl H2O
3.6µl H2O
Quantification of DNA:
GFP (GB0059); pUPD2: 249 ng/µl
Ω2: 238 ng/µl
Alfredo’s pUPD2, domesticator: 102 ng/µl
iGEM704: 405 ng/µl
iGEM735: 403 ng/µl
552 AMP 35S noATG: 45 ng/µl
PIF (C5), pUPD2: 119 ng/µl
pD6B3, Ω2 (22/06): 158 ng/µl
LacIBD (C1); pUPD2 (22/06): 129 ng/µl
109 renillaDC: 49 ng/µl
IGEM 534: 13.6 ng/µl
VP16 (C1); pUPD2:102 ng/µl
IGEM 1097: 409 ng/µl
KDronpa (C3); pUPD2 (18/06): 174 ng/µl
IGEM 858: 487 ng/µl
731AMP Gal4 (19/06): 81 ng/µl
IGEM pUPD2 domesticator: 87 ng/µl
PIF+PhyB (C1) (08/06): 108 ng/µl
160 renilla, α2 (19/06): 46 ng/µl
159 renilla, Ω2 (19/06): 149 ng/µl
Etr8:BxbI (C1)(22/06): 149 ng/µl
IGEM 732: 422 ng/µl
25 June 2015
Minipreps of the liquid culture:
We don’t observed growth in LacIBD+PIF and LacIBD+KDronpa.
Digestion of the minipreps and do the gel:
Gal4BD; pUPD2
NotI
2046, 282
RepBxbI; pUPD2
NotI
2046, 460
Etr8(CMV):BxbI:PhyB; α1
BamHI
6674, 2237, 2806, 1174
LexABD; pUPD2
NotI
2046, 321
9+10; pUPD2
NotI
464
Gel:
Etr8:BxbI
LexA C1
LexA C2
LexA C3
LexA C4
RepBxbI C1
RepBxbI C2
RepBxbI C3
Gal4 C1
PCR 9+10
no
no
no
no
no
ok
ok
ok
no
ok
We make a PCR of the Fusion Taq pH (proof-reading) to prove that the primer received number 10. This new one works! Amplify the sequence of NDronpa (R1).
Refresh the cultures of Agrobacteriumwith the viral system. Add only ryfampicin and kanamycin.
Ligations:
N-dronpa; pUPD2
RepBxbI; α1
Gal4BD, pUPD2
LexABD; pUPD2
1 µl PCR 9+10
1 µl Rep Bxb1
1 µl PCR 3+4
1 µl PCR 5+6
1 µl PCR11+12
1 µl Promoter without ATG
1 µl pUPD2
1 µl pUPD2
1 µl pUPD2
1 µl Tnos
1 µl α1
4,6 µl H2O
3,6 µl H2O
5,6 µl H2O
5,6 µl H2O
Etr8:BxbI+PhyB; Ω1
1 µl Etr8:BxbI
1 µl 88E
1µl Ω1
3,6 µl H2O
Transform ligations into E.Coli and make petri dish cultures with cloranfenicol for all of them except the ligation of Etr8:Bxb1+PhyB that goes with streptomycin.
26 June 2015
Do ligations:
RepBxbI+GFP; α2
LacIBD+PIF6; α1
1 µl RepBxbI
1 µl LacIBD, pUPD2
1 µl promoter without ATG
1 µl PIF6, pUPD2
1 µl Tnos
1 µl promoter
1µl GFP (0059)
1 µl T35
1 µl α2
1 µl α1
2.6 µl H2O
2.6 µl H2O
Digestion:
LacIBD+PIF6; α1
EcoRI
6345, 1997, 641
Gel:
LacIBD+PIF C1
LacIBD+PIF C2
no
no
Both of them present the fragment of the vector at 6000 pb but none of them at 2000bp which is the insert one.
Measurement of the ODs of PhyB:PIF6:luc and renilla+P19.
PhyB:PIF6:luc: 0.35 (1:2)
0.35
1.429 µl
Ren+P19: 0.34 (1:2)
0.34
1.412 µl
Ligation of:
LacIBD; pUPD2+KDronpa; pUPD2; α1
1 µl 35S
1 µl LacIBD;pUPD2
1 µl KDronpa; pUPD
1 µl T35S
1 µl α1
2.6 µl H2O
1?EXPERIMENT. Red toggle. E:PIF6:PhyB and renilla. For more info, click here.
27 June 2015
Transformation into E. coli of LacIBD+KDronpa; α1 and make petri dish culture.
Make petri dish culture of LexABD and Etr8(CMV):Bxb1:GFP.
We make liquid culture of:
RepBxbI:GFP (C1-C4)
LacIBD+PIF6 (C1-C5)
NDronpa (C1-C4)
Gal4BD (C1-C5)
LexABD (C1-C3)
28 June 2015
Do the minipreps of the liquid cultures that have grown.
RepBxbI:GFP (C1 and C2)
LacIBD+PIF6 (C1-C4)
NDronpa (C1-C4)
Gal4BD (C1-C5)
LexA: didn’t grow
Do the digestions of the minipreps:
LacIBD+PIF; α1
EcoRI
6345, 1997, 641
RepBxbI:GFP; Ω2
HindIII
6345, 2683
Gal4BD; pUPD2
NotI
2681, 644
NDronpa; pUPD2
NotI
2046, 744
Make the gel.
RepBxbI:GFP C1
RepBxbI:GFP C2
LacIBD+PIF C1
LacIBD+PIF C2
LacIBD+PIF C3
LacIBD+PIF C4
no
no
no
no
no
No
Gal4BD C1
Gal4BD C2
Gal4BD C3
Gal4BD C4
Gal4BD C5
N-Dronpa C1
ok
ok
ok
ok
ok
ok
N-Dronpa C2
N-Dronpa C3
N-Dronpa C4
no
ok
ok
Take glycerinated:
GB0030: p35S
GB0036: T35S
Make liquid culture of LexABD (C1-C4).
We transform again LacIBD:KDronpa and RepBxb1:GFP, adding to the agar plates 100 µl of each transformation.
29 June 2015
Do the minipreps of the 4 colonies of LexABD and both glycerinates, 35S and T35S.
Do the digestion of the minipreps:
LexABD; pPPD2
NotI
2358, 312
35S; pUPD2
NotI
2981, 1074
T35S; pPUD2
NotI
2981, 304
Make the gel:
LexA C1
LexA C2
LexA C3
LexA C4
P35S
T35S
ok
ok
ok
ok
Ok?
Ok?
Make ligations:
LacIBD+KDronpa+promoter+termi; α1
Gal4BD+KDonpa+prom+ter; α1
LexABD+KDronpa+prom+term; α1
1 µl LacI; pUPD2
1 µl Gal4; pUPD2
1 µl Gal4; pUPD2
1 µl KDronpa; pUPD2
1 µl KDronpa; pUPD2
1 µl KDronpa; pUPD2
1 µl 35S (GB0030)
1 µl 35S (GB0030)
1 µl 35S (GB0030)
1 µl T35S (GB0036)
1 µl T35S (GB0036)
1 µl T35S (GB0036)
2.6 µl H2O
2.6 µl H2O
2.6 µl H2O
1 µl α1
1 µl α1
1 µl α1
NDronpa+VP16; α2
Gal4BD+PIF6; α1
LacIBD+PIF6; α1
1 µl NDronpa; pUPD2
1 µl Gal4BD; pUPD2
1 µl LacIBD; pUPD2
1 µl VP16; pUPD2
1 µl PIF6; pUPD2
1 µl PIF6; pUPD2
1 µl 35S (GB0030)
1 µl 35S (GB0030)
1 µl 35S (GB0030)
1 µl T35S (GB0036)
1 µl T35S (GB0036)
1 µl T35S (GB0036)
2.6 µl H2O
2.6 µl H2O
2.6 µl H2O
1 µl α2
1 µl α1
1 µl α1
LexABD+PIF6; α1
1 µl LexABD; pUPD2
1 µl PIF6; pUPD2
1 µl 35S (GB0030)
1 µl T35S (GB0036)
2.6 µl H2O
1 µl α2
Transform all the ligations into E.Coli. Gal4BD+K-Dronpa and LacIBD+K-Dronpa went wrong and we have to do it again.
Sent N-Dronpa with the primers 9 and 12 to sequence to check if the codon that synthetize for the amino acid K has change to the amino acid N.
Quantification of DNA:
RepBxbI:GFP (C1): 163.8 ng/µl
NDronpa; pUPD2 (C4):113.1 ng/µl
NDronpa (C3): 83.2 ng/µl
NDronpa (C1): 116.6 ng/µl
Gal4BD (C1): 95.2 ng/µl
Gal4BD (C2): 120.7 ng/µl
RepBxbI:GFP (C2): 170.6 ng/µl
RepBxbI (C1): 80.6 ng/µl
30 June 2015
Transform Gal4+KDronpa and LacI+KDronpa and make petri dish culture.
Miniprep of:
RepBxbI+GFP (C1-C3)
RepBxb1+GFP; Ω2
HindIII
6345, 2683
Gel:
RepBxbI+GFP C1
RepBxbI+GFP C2
RepBxbI+GFP C3
No
no
no
We pick more colonies of RepBxb1+GFP, Ω2 and make liquid cultures.
Make liquid culture of:
LexABD+KDronpa+prom+term; α1 (C1 and C2)
NDronpa+VP16; α2 (C1 and C2)
Gal4BD+PIF6; α1 (C1 and C2)
LacIBD+PIF6; α1 (C1 and C2)
LexABD+PIF6; α1 (C1 and C2)
Take out a glycerinate 35S:Luciferase:Tnos (GB0227) and do a miniprep.
July
1 July 2015
Do the minipreps of the 10 liquid cultures.
Do the digestions:
LexABD+KDronpa;α1
EcoRI
6345, 2296
NDonpa+VP16; α2
HindIII
6345, 2427
Gal4BD+PIF6; α1
EcoRI
6345, 1867
LacI+PIF; α1
EcoRI
6345, 2638
LexABD+PIF6; α1
EcoRI
6345, 1906
Do the gel:
Gal4+PIF C1
Gal4+PIF C2
LexA+PIF C1
LexA+PIF C2
LexA+KDronpa C1
ok
ok
ok
ok
Ok
LexA+Kdronpa C2
LacI+PIF C1
LacI+PIF C2
Ndonpa+VP16 C1
Ndronpa+VP16 C2
ok
ok
ok
ok
ok
Prepare liquid culture of:
LexA+PIF; Ω1 (C1)
LacI+PIF; Ω1 (C1)
LexA+K-Dronpa (C1)
Gal4+PIF; Ω1 (C1)
VP16, pUPD2 (C1)
LexABD, pUPD2 (C2)
PIF6, pUPD2 (C5)
LacIBD, pUPD2 (C1)
EXPERIMENT 1.
Luciferase essay.
Pick colonies and make liquid culture of:
Gal4+K-Dronpa (C1 and C2)
LacI+K-Dronpa (C1 and C2)
RepBxbI+GFP (C4-C6)
Code:
210.08-249: pUPD2, KDronpa C3
210.08-250: pUPD2, NDronpa C1
210.08-251: pUPD2, NDronpa C3
210.08-252: pUPD2, NDronpa C4
2 July 2015
Minipreps of:
Gal4+KDronpa (C1 and C2)
LacI+KDronpa (C1 and C2)
RepBxbi+GFP (C4-C6)
Gal4+KDronpa
EcoRI
6345, 3028
LacI+KDronpa
EcoRI
6345, 2257
RepBxbI+GFP
HindIII
6300, 2400
Do the gel:
LacI+KDronpa C1
LacI+KDronpa C2
Gal4+KDronpa C1
Gal4+KDronpa C2
ok
ok
ok
ok
RepBxb1+GFP C4
RepBxb1+GFP C5
RepBxb1+GFP C6
ok
ok
ok
Make ligations:
LexA:Kdonpa+NDronpa; Ω1
LacI:Kdronpa+NDronpa; Ω1
Gal4:Kdronpa+NDronpa; Ω1
1 µl LexA:Kdronpa
1 µl LacI:Kdronpa
1 µl Gal4:Kdronpa
1 µl NDronpa
1 µl NDronpa
1 µl NDronpa
1 µl Ω1
1 µl Ω1
1 µl Ω1
4.6 µl H2O
4.6 µl H2O
4.6 µl H2O
LexA:PIF+PhyB:VP16; Ω1
LacI:PIF+PhyB:VP16; Ω1
Gal4:PIF+PhyB:VP16; Ω1
1 µl LexA:PIF
1 µl LacI:PIF
1 µl Gal4:PIF
1 µl PhyB:VP16 (88E)
1 µl PhyB:VP16
1 µl PhyB:VP16
1 µl Ω1
1 µl Ω1
1 µl Ω1
4.6 µl H2O
4.6 µl H2O
4.6 µl H2O
35S:BxbI+RepBxbI:GFP; Ω1
E-PIF+phyB+luc+ren; Ω1
1 µl 35s:Bxb1:T35S (alfredo’s)
0.5 µl PIF:PhyB:luc (GB0896)
1 µl NoATGProm:RepBxbI:GFP
1 µl Renilla GB0160
1 µl Ω1
1 µl Ω1
4.6 µl H2O
4.6 µl H2O
The samples that we sent to sequence have arrived:
210.08-249: pUPD2, KDronpa C3 - ok
210.08-250: pUPD2, NDronpa C1 - ok
210.08-251: pUPD2, NDronpa C3 - ok
210.08-252: pUPD2, NDronpa C4 - ok
The sequences of NDronpa have the desired mutation.
Transformation in E.Coli of the 8 ligations and make petri dish cultures.
Refresh the Agro’s cultures Renilla and PIF:phy:luc.
4 July 2015
Make the 2nd refresh of the culture of Agrobacterium.
Make liquid cultures of the 8 colonies of E.Coli.
Red toggle (C1)
Gal4:Kdronpa+NDronpa (C1 and C2)
LexA:Kdonpa+NDronpa (C1 and C2)
LacI:Kdronpa+NDronpa (C1 and C2)
LexA:PIF+PhyB:VP16 (C1 and C2)
35S:BxbI+RepBxbI:GFP (C1 and C2)
5 July 2015
Ligations:
LacI:PIF+PhyB:VP16; Ω1
Gal4:PIF+PhyB:VP16; Ω1
1 µl LacI:PIF
1 µl Gal4:PIF
1 µl PhyB:VP16
1 µl PhyB:VP16
1 µl Ω1
1 µl Ω1
4.6
µl H2O
4.6 µl H2O
Minipreps of the liquid cultures.
Digestion of the minipreps:
LacI:Kdronpa+NDronpa
BamHI
6674, 5437
35S:BxbI+RepBxbI:GFP
BamHI
6674, 3859, 1782
Gal4:Kdronpa+NDronpa
BamHI
6674, 4666
LexA:Kdonpa+NDronpa
BamHI
6674, 4705
LexA:PIF+PhyB:VP16
BamHI
6674, 3513, 2337
E:PIF6:PhyB:VP16
BamHI
4209, 3756, 6100, 6674
Agarose gel (1%):
LacI:KNDronpa C1
LacI:KNDronpa C2
BxbI:Rep:GFP C1
BxbI:Rep:GFP C2
Gal4:KNDronpa C1
Gal4:KNDronpa C2
ok
ok
ok
ok
ok
ok
LexA:KNDronpa C1
LexA:KNDronpa C2
LexA:PIF:Phy C1
LexA:PIF:Phy C2
Red toggle
ok
ok
no
no
no
We have to repeat the digestion of: LexA+PIF:phy+VP16.
Take out the glycerinate 88C (GB1098), Etr8:luc:Tnos. We will use it like a negative control in the second luciferase essay.
Calculation of the ODs:
Renilla
0.22
182 µl of sample + 1.818 ml MES
E:PIF6:PhyB:Luc
0.26
154 µl of sample + 1.646 ml MES
EXPERIMENT 2.
Agroinfiltration of renilla and PIF6:PhyB:luc. For more info, click here.
6 July 2015
Transform the negative control into agrobacterium and make petri dish culture of:
Etr8:luc:Tnos
BxbI:ReporterBxbI:GF
We were doing dry lab preparing the power point to present our project to the rector and biotecs companies.
7 July 2015
Prepare this cultures for agroinfiltration:
Renilla
E:PIF6:PhyB:Luc
Renilla
0.29
0.138ml of sample + 1.862ml of MES
PhyB+PIF+luc
0.69
0.145ml of sample + 1.855ml of MES
EXPERIMENT 3. Start. Agroinfiltrate PhyB+PIF+luc and Renilla
Digestions:
Red toggle
BamHI
4209, 3756, 6100, 6674
LexA+PIF+phy
BamHI
3518, 5855, 6674
Gel:
Red toggle
LexA+PIF+Phy C1
LexA+PIF+Phy C2
No
Ok
no
It has arrived a new construction: AsLOVpep.
Resuspended with 50µl of H2O.
Ligations:
AsLOVpep; pUPD2
Red Toggle
LacI:Kdronpa:Ndronpa:VP16+renilla; α1
Gal4:Kdronpa:Ndronpa:VP16+renilla; α1
1 µl AsLOVpep
1 µl GB846
1 µl LacI:KNdronpa:VP16
1 µl Gal4:KNdronpa
1 µl pUPD2
1 µl GB160
1 µl GB159
1 µl GB159
5.6 µl H2O
1 µl Ω1
1 µl α1
1 µl α1
4.6 µl H2O
4.6 µl H2O
4.6 µl H2O
LexA:Kdronpa:Ndronpa:VP16+renilla; α1
LexA:PIF:Phy:VP16+renilla; α1
1 µl LexA:Kdronpa:Ndronpa:VP16
1 µl LexA:PIF:Phy:VP16
1 µl GB159
1 µl GB159
1 µl α1
1 µl α1
4.6 µl H2O
4.6 µl H2O
8 July 2015
EXPERIMENT 2. Change the ligth conditions of the plants.
Transform into E. coli last ligations.
9 July 2015
Pick colonies and make liquid culture of:
AsLOVpep; pUPD2 colonies didn’t grow. Repeat.
Red toggle colonies are all blue. Repeat.
LacI:Kdronpa:Ndronpa:VP16+renilla (C1-C3)
Gal4:Kdronpa:Ndronpa:VP16+renilla (C1-C3)
LexA:Kdronpa:Ndronpa:VP16+renilla (C1 and C2)
LexA:PIF:Phy:VP16+renilla (C1 and C2)
Repeat the ligations.
AsLOVpep, as before.
Red toggle (PIF+PhyB+luc+ren)
0.5 µl PIF+phy+luc (896)
1 µl renilla (160)
1 µl Ω1
5.1 µl H2O
EXPERIMENT 2.
Luciferase essay.
Transform the ligations of AsLOVpepe and red toggle. This time we make a spin to concentrate the cells to make petri dish culture.
10 July 2015
Minipreps of:
LexABD:PIF:
PhyB:VP16+Renilla; α1 (C1,C2)
LexA:Kdronpa:Ndronpa+renilla; α1 C1, C2)
Gal4:Kdronpa:Ndronpa+renilla; α1 (C1-C3)
LacI:Kdronpa:Ndronpa+renilla; α1 (C1,C2)
Digestions of:
LexABD:PIF:PhyB:VP16+Renilla; α1
EcoRI
6345, 5487, 4891
LexA:Kdronpa:Ndronpa+renilla; α1
EcoRI
9333, 6345
Gal4:Kdronpa:Ndronpa+renilla; α1
EcoRI
6345, 9194
LacI:Kdronpa:Ndronpa+renilla; α1
EcoRI
6345, 9965
LacIBD+PIF+PhyB; Ω1
BamHI
6674, 4245, 2337
Gal4BD+PIF+PhyB; Ω1
BamHI
6674, 3474, 2337
Make the gel:
LacIBD+PIF+PhyB
Gal4BD+PIF+PhyB
LexA:PIF:PhyB:VP16+Ren C1
LexA:PIF:PhyB:VP16+Ren C2
Ok
Ok
ok
ok
LexA:KNdronpa+ren C1
LexA:KNdronpa+ren C2
Gal4:KNdronpa+ren C1
Gal4:KNdronpa+ren C2
Ok
Ok
ok
Ok
Gal4:KNdronpa+ren C3
LacI:Kdronpa:Ndronpa+ren C1
LacI:Kdronpa:Ndronpa+ren C2
Ok
Ok
ok
We received two constructions:
CDS: phiC31. Resuspended with 100µl.
Reporter Phi31. Resuspended with 50 µl.
Pick colonies and make liquid culture of:
AsLOVpep; pUPD2 (C1 and C2)
Red toggle (C1 and C2).
Ligations:
PhiC31; pUPD2
Reporter PhiC31; pUPD2
LacI:PIF:PhyB:VP16+ren; α1
1 µl PhiC31
1 µl RepPhiC31
1 µl LacI:PIF:PhyB:VP16
1µl pUPD2
1µl pUPD2
1 µl renilla (GB159)
5.6 µl H2O
5.6 µl H2O
1 µl α1
4.6 µl H2O
Gal4:PIF:Phy:VP16+ren; α1
LexA:PIF:PhyB:ren+OpLex:luc; Ω1
LexA:KNdronpa:ren+OpLex:Luc; Ω1
1 µl Gal4:PIF:phy:VP16
1 µl LexA:PIF:phy:ren
1 µl LexA:KNdronpa:ren
1 µl renilla (GB159)
1 µl opLex:luc (GB151)
1 µl OpLex:Luc (GB151)
1 µl α1
1 µl Ω1
1 µl Ω1
4.6 µl H2O
4.6 µl H2O
4.6 µl H2O
Gal4:KNdronpa:ren+UAS:luc; Ω1
LacI:KNdronpa:ren+OpLacI:luc; Ω1
1 µl Gal4:KNdronpa:ren
1 µl LacI:KNdronpa:ren
1 µl OpUAS:luc (GB227)
1 µl OpLacI:luc (GB152)
1 µl Ω1
1 µl Ω1
4.6 µl H2O
4.6 µl H2O
11 July 2015
Prepare glycerinates of:
BxbI+Rep:GFP; Ω1
NDronpa; pUPD2
BxbI:Etr8; pUPD2
Etr8(CMV):BxbI:T35S; α1
Do miniprep of:
AsLOVpep (C1 and C2)
Red toggle (C2) (C1 was blue)
Digestions:
E:PIF6:PhyB:luc:ren
BamHI
6674, 6100, 4209, 3756
AsLOVpep
NotI
2558, 512
AsLOVpep C1
AsLOVpep C2
Red toggle C2
no
no
no
Do transformation of DHSa and yesterday ligations:
phyC31;pUPD2
Reporter PhiC31; pUPD2
LacI:PIF:PhyB:VP16+ren; α1
Gal4:PIF:phy:VP16+ren; α1
LexA:PIF:phy:ren+opLex:luc; Ω1
LexA:KNdronpa:ren+OpLex:luc; Ω1
Gal4:KNdronpa:ren+OpUAS:luc; Ω1
LacI:KNdronpa:ren+OpLacI:luc; Ω1
12 July 2015
Pick colonies and make liquid culture:
PhiC31;pUPD2 (C1-C3)
Reporter PhiC31; pUPD2 (C1-C3)
LacI:PIF:PhyB:VP16+ren; α1 (C1-C3)
Gal4:PIF:phy:VP16+ren; α1  All blue colonies.
LexA:PIF:phy:ren+OpLex:luc; Ω1 (C1)
LexA:KNdronpa:ren+OpLex:Luc; Ω1(C1-C3)
Gal4:KNdronpa:ren+UAS:luc; Ω1 (C1)
LacI:KNdronpa:ren+OpLacI:luc; Ω1 All blue colonies.
Ligations that were repeated:
Gal4:PIF:phy:VP16+ren; α1
LacI:KNdronpa:ren+OpLacI:luc; Ω1
1 µl Gal4:PIF:phy:VP16
1 µl LacI:KNdronpa:ren
1 µl renilla (GB159)
1 µl OpLacI:luc (GB152)
4.6 µl H2O
4.6 µl H2O
1 µl α1
1 µl Ω1
Refresh the liquid cultures of Agrobacterium:
BxbI:GFP and Etr8:Tnos.
Pnos, it was at the fridge (-4ºC).
13 July 2015
All the liquid cultures have grown, do minipreps.
Do digestions:
phyC31;pUPD2
NotI
2046, 1899
Reporter phiC31; pUPD2
NotI
2046, 475
LacI:PIF:Phy:VP16+ren; α1
EcoRI
6345, 5623, 5487
LexA:PIF:phy:ren+opLex:luc; Ω1
BamHI
9431, 6674, 3531
LexA:KNdronpa:ren+OpLex:Luc; Ω1
BamHI
1199, 6674
Gal4:KNdronpa:ren+UAS:luc; Ω1
BamHI
11582, 6674
Agarose gel:
PhiC31 C1
PhiC31 C2
PhiC31 C3
RepPhiC31 C1
no
no
no
ok
RepPhiC31 C2
LacI:PIF:PhyB:VP16+ren C1
LacI:PIF:PhyB:VP16+ren C2
LacI:PIF:PhyB:VP16+ren C3
no
no
ok
ok
LexA:PIF:phy:ren+OpLex:luc
LexA:KNdronpa:ren+OpLex:luc C1
LexA:KNdronpa:ren+OpLex:Luc C2
LexA:KNdronpa:ren+OpLex:Luc C3
no
ok
ok
ok
Gal4:KNdronpa:ren+OpUAS:luc C1
no
The Agrobacterium cultures refreshed yesterday were store in the fridge.
It is made another culture of 35S:BxbI+reporterBxbI:GFP to keep it in the fridge.
EXPERIMENT 4. Recombinase BxbI.
Mesurement of the OD’s:
35S:BxbI+reporterBxbI:GFP: 0.28
143 µl of culture+1857 µl of MES
Transformation of the ligation: Gal4:PIF:phy:VP16+ren; α1 and LacI:KNdronpa:ren+OpLacI:luc; Ω1.
The liquid cultures of this constructions were repeated:
phyC31;pUPD2 (C4 and C5)
Reporter phiC31; pUPD2 (C4)
LacI:PIF:PhyB:VP16+ren; α1 (C4)
LexA:PIF:phy:ren+OpLex:luc; Ω1 (C1)
LexA:KNdronpa:ren+OpLex:Luc; Ω1(C1-C3)
Gal4:KNdronpa:ren+UAS:luc;Ω1 (C1)
AsLOVpep; pUPD2 (C3)
14 July 2015
Do minipreps of:
phyC31;pUPD2 (C4 and C5)
Reporter phiC31; pUPD2 (C4)
LacI:PIF:PhyB:VP16+ren; α1 (C4)
LexA:PIF:phy:ren+OpLex:luc; Ω1 (C1)
LexA:KNdronpa:ren+OpLex:Luc; Ω1(C1-C3)
Gal4:KNdronpa:ren+UAS:luc;Ω1 (C1)
AsLOVpep; pUPD2 (C3)
Do digestions:
phyC31;pUPD2
NotI
2046, 1899
Reporter phiC31; pUPD2
NotI
2046, 475
LacI:PIF:Phy:VP16+ren; α1
EcoRI
6345, 5623, 5487
LexA:PIF:phy:ren+opLex:luc, Ω1
BamHI
9431, 6674, 3531
LexA:KNdronpa:ren+OpLex:Luc; Ω1
BamHI
1199, 6674
Gal4:KNdronpa:ren+UAS:luc; Ω1
BamHI
11582, 6674
AsLOVpep; pUPD2
NotI
2558, 512
Make an agarose gel:
phyC31 (C4)
phyC31 (C5)
AsLOVpep (C4)
LexA:PIF:phy:ren+opLex:luc (C1)
no
no
No
No
LexA:KNdronpa:ren+OpLex:Luc (C3)
Gal4:KNdronpa:ren+OpUAS:luc (C1)
Gal4:KNdronpa:ren+UAS:luc(C2)
Gal4:KNdronpa:ren+UAS:luc (C3)
Ok
no
no
No
LacI:PIF:Phy:VP16+ren
Reporter phiC31 (C1)
no
Ok
Only LexA:KNdronpa:ren+OpLex:Luc and Reporter phiC31 were correct. Repeat the ligations because is the second digestion of this construction that were made.
Measurement of DNA concentration:
Reporter:phyC31: 13.6 ng/µl
PhiC31: 4.6 ng/µl
AsLOVpep: 30.3 ng/µl
OpLexA:luc (GB151): 40 ng/µl
Gal4:PIF:PhyB:ren (C1): 124.4 ng/µl
LacI:PIF:PhyB:VP16 (C1): 126 ng/µl
Renilla (GB159): 42 ng/µl
Gal4:Kdronpa:Ndronpa:renilla (C3): 172.8 ng/µl
OpUAS:luc (GB227): 6.3 ng/µl
Pick colonies and make liquid culture of LacI:KDronpa:NDronpa:ren:luc (C4 and C5). The colonies of Gal4:PIF:phy:VP16+ren were all blue.
15 July 2015
Miniprep of LacI:KDronpa:NDronpa:ren:luc (C4 and C5).
Digestion:
LacI:KDronpa:NDronpa:ren:luc
EcoRI
6345, 9965
Make the gel:
LacI:K:NDronpa:ren:luc C4
LacI:K:NDronpa:ren:luc C5
no
no
Measurement of DNA concentrations:
Gal4BD:PIF:PhyB:VP16: 125.6 ng/µl
LexA:PIF:PhyB:VP16:ren: 322.6 ng/µl
LacI:Kdronpa:NDronpa:ren: 209.0 ng/µl
Repeat the ligations:
PhiC31;pUPD2
AsLOVpep; pUPD2
LacI:PIF:PhyB:VP16+ren; α1
1 µl PhiC31
1 µl AsLOVpep
1.5 µl LacI:PIF:PhyB:VP16
1µl pUPD2
1µl pUPD2
2.5 µl renilla (159)
5.6 µl H2O
5.6 µl H2O
0.5 µl α1
4.6 µl H2O
Gal4:PIF:phy:VP16+ren; α1
LexA:PIF:phy:ren+opLex:luc; Ω1
LacI:KNdronpa:ren+OpLex:Luc; Ω1
1.5 µl Gal4:PIF:phy:VP16
1 µl LexA:PIF:phy:ren
1 µl LacI:KNdronpa:ren
2 µl renilla (159)
2.5 µl opLex:luc (151)
3 µl OpLac:Luc (152)
2.6 µl H2O
2.6 µl H2O
3 µl H2O
1 µl α1
0.5 µl Ω1
0.5 µl Ω1
Gal4:KNdronpa:ren+OpLex:Luc; Ω1
PhyB:VP16+PIF6; Ω1
1 µl Gal4:KNdronpa:ren
1 µl PhyB:VP16 (88E)
4 µl OpUAS:Luc (227)
2 µl PIF6 (170)
3 µl H2O
3.6 µl H2O
0.5 µl Ω1
1 µl Ω1
These Agrobacterium cultures have been refreshed:
E:PIF:PhyB:luc
Renilla
Pnos
Etr8
16 July 2015
Yesterday ligations have been transformed into E. coli:
phyC31;pUPD2
AsLOVpep; pUPD2
LacI:PIF:Phy:VP16+ren; α1
Gal4:PIF:phy:VP16+ren; α1
LexA:PIF:phy:ren+opLex:luc; Ω1
LacI:KNdronpa:ren+OpLex:Luc; Ω1
Gal4:KNdronpa:ren+OpLex:Luc; Ω1
PhyB:VP16+PIF6; Ω1
EXPERIMENT 4.
The agroinfiltrated leaf with BxbI:rep:GFP has been observed in the magnifying glass.
Second refresh of the Agrobacterium cultures:
E:PIF:PhyB:luc
Renilla
Pnos
Etr8
Miniprep of these cultures.
Digestion of the minipreps:
E:PIF:PhyB:luc; Ω1
EcoRI
??
Renilla; Ω2
HindIII
7453
Pnos; α1
EcoRI
2997, 353
Etr8; Ω1
EcoRI
2742
The digestions had positive controls that were included in the gel to compare the results obtained.
The digestions are left overnight in the working table.
17 July 2015
Do the gel:
Pnos
Etr8:luc
Etr8:luc C+
PIF6:PhyB:luc
no
ko
ok
ok
PIF6:PhyB:luc C+
renilla
renilla C+
ok
no
ok
Make liquid cultures of yesterday ligations, three colonies per cultures.
OD’s mesurement for the agroinfiltration.
PIF:PhyB:luc
0.47
41 µl/ml
630 µl
Renilla
0.28
71 µl/ml
1065 µl
Etr8:luc
0.32
52 µl/ml
930 µl
Pnos
0.34
59 µl/ml
885 µl
EXPERIMENT 5. Red toggle experiement:
PIF:PhyB:luc
Renilla
Etr8:luc (C-)
Pnos (C+)
EXPERIMENT 4.
It was observed another leaf with the recombinase BxbI with GFP in the magnifying glass with fluorescent lights.
18 July 2015
23 minipreps of the liquid cultrures. LexA:PIF:PhyB:ren:luc (C3) has not grown.
Digestions of the minipreps:
PhiC31;pUPD2
NotI
2046, 1899
AsLOVpep; pUPD2
NotI
2046, 521
LacI:PIF:PhyB:VP16+ren; α1
EcoRI
6345, 5623, 5487
Gal4:PIF:phy:VP16+ren; α1
EcoRI
6345, 5487, 4852
LexA:PIF:phy:ren+OpLex:luc; Ω1
BamHI
9431, 6674, 3513
LacI:KNdronpa:ren+OpLex:Luc; Ω1
BamHI
12632, 6574
Gal4:KNdronpa:ren+OpLex:Luc; Ω1
BamHI
11582, 6674
PhyB:VP16+PIF6; Ω1
BamHI
6674, 2685, 2337, 1439
Gel has been done:
phiC31 C1
phiC31 C2
phiC31 C3
AsLOVpep C1
no
no
no
ok
AsLOVpep C2
AsLOVpep C3
Gal4:PIF:phy:VP16:ren C1
Gal4:PIF:phy:VP16:ren C2
no
no
no
no
Gal4:PIF:phy:VP16:ren C3
LacI:PIF:phy:ren C1
LacI:PIF:phy:ren C2
LacI:PIF:phy:ren C3
no
ok
no
No
LexA:PIF:phy:ren:luc C1
LexA:PIF:phy:ren:luc C2
Gal4:KNdronpa:ren:luc C1
Gal4:KNdronpa:ren:luc C2
no
no
ok
No
Gal4:KNdronpa:ren:luc C3
LacI:KNdronpa:ren:luc C1
LacI:KNdronpa:ren:luc C2
LacI:KNdronpa:ren:luc C3
no
no
ok
No
PhyB:VP16:PIF6 C1
PhyB:VP16:PIF6 C2
PhyB:VP16:PIF6 C3
ok
no
no
Pick more colonies of:
Gal4:PIF:phy:VP16+ren; α1
LexA:PIF:phy:ren+opLex:luc; Ω1
New digestions with new enzymes:
phiC31;pUPD2
XhoI (buffer red)
2119, 934, 894
LacI:PIF:Phy:VP16+ren; a1
NEB4
5949, 5653, 3610, 2246
LacI:PIF:Phy:VP16+ren; a1
HindIII
11568, 5587
After 3 digestions of LacI:PIF:Phy:VP16+ren; α1 is accepted the construction.
It was observed another leaf with the recombinase BxbI with GFP in the magnifying glass with fluorescent lights. It was not observed a lot of spots, the efficiency is very low. This is the 4th day…
FOTO
19 July 2015
The incredible mistery of the lost notebook papers...
20 July 2015
Another day...
21 July 2015
EXPERIMENT 5. Red toggle.
It was picked disc samples: 3 in far red, 3 in darkness and 6 in red (3 of them were in far red and the other 3 in darkness).
Time lapses:
-19:00=t0
-1:00=t1
-7:00= t2
-19:00= t3 (it was taken the controls in natural light)
Minipreps of the colonies that were in 37ºC.
LexA:PIF:Phy:ren:luc (C1 and C2)
PhyC31 (C1, C2, C4 and C5)
LexA:PIF:Phy:ren:luc
BamHI
9431, 6674, 3513
PhiC31
NotI
2046, 1899
Gel with the digestions:
PhiC31 C1
PhiC31 C2
PhiC31 C4
PhiC31 C5
Ok?
ok
ok
ok
LexA:PIF:PhyB:ren:luc C1
LexA:PIF:PhyB:ren:luc C2
No DNA
No DNA
We had problems with some colonies because in the digestion did not appear DNA. The minipreps will be made with a better kit.
Transform in Agrobacterium this cultures:
LexABD:KDronpa:NDronpa:ren:luc
Gal4BD:KDronpa:NDronpa:ren:luc
LacIBD:KDronpa:NDronpa:ren:luc
OpLexA:luc (151)
OpUAS:luc (227)
OpLacI:luc (152)
It was made liquid culture of Gal4:PIF:phyB:ren; Ω1 (C1-C5)
It was made ligations:
35S:Gal4:AsLOVpep:T35S; α1
35S:LacI:AsLOVpep:T35S; α1
35S:LexA:AsLOVpep:T35S; Ω1
1 µl 35S (0030)
1 µl 35S (0030)
1 µl 35S (0030)
1 µl Gal4BD
1 µl LacI
1 µl LexA
1 µl AsLOVpep
1 µl AsLOVpep
1 µl AsLOVpep
1 µl T35S (0036)
1 µl T35S (0036)
1 µl T35S (0036)
1 µl α1
1 µl α1
1 µl α1
2.6 µl H2O
2.6 µl H2O
2.6 µl H2O
PsinATG:RepPhiC31:GFP:T35S; α2
LacI:PIF:PhyB:ren+luc; Ω1
PIF:PhyB+renilla; α2
1 µl PsinATG (552)
1.5 µl LacI:PIF:PhyB:ren; α1
1.5 µl PIF:PhyB
1 µl ReporterPhiC31
3 µl OpLacI:luc (152); α2
2.5 µl renilla (159)
1 µl GFP (0059)
0.5 µl Ω1
0.5 µl α2
1 µl T35S (0036)
2.6 H2O
3 µl H2O
1 µl α2
2.6 µl H2O
EXPERIMENT 5.
Luciferase essay.
22 July 2015
Miniprep of Gal4:PIF:PhyB:ren (C1-C3) C4 and C5 did not grow.
Digestion of the miniprep:
Gal4:PIF:PhyB:ren
BamHI
16684
EcoRI
4852, 5487, 6345
EcoRV
1849, 3942, 2475, 381, 8037
Gel was made:
Gal4:PIF:PhyB:ren (BamHI)
Gal4:PIF:PhyB:ren (EcoRI)
Gal4:PIF:PhyB:ren (EcoRV)
no
no
no
After doing several digestions with different enzyme all with wrong band patterns, it was decided to revise each part making digestions. The parts are:
PhiC31; pUPD2
Gal4:PIF:PhyB
Renilla (GB159)
Made liquid culture of the ReporterBxbI:GFP in Agrobacterium.
Transform the ligations into E. coli:
35S:Gal4:AsLOVpep:T35S; α1
35S:LacI:AsLOVpep:T35S; α1
35S:LexA:AsLOVpep:T35S; α1
PsinATG:RepPhiC31:GFP:T35S; α2
LacI:PIF:PhyB:ren+luc; Ω1
EXPERIMENT 6.
Luciferase essay:
Sampling: samples that have been in red and natural light 48h, 3 samples.
200 µl/sample x 3 sample= 600 µl of passive lissis buffer (5x)
Passive lissis buffer 1x= 120 µl+480 µl water.
2.4 µl of Stop and glow
118 µl of  buffer.
23 July 2015
EXPERIMENT 7.
We decided to infiltrate soybean sprouts. First we decided to infiltrate with dye to observe the characteristics and the capacity of absorption.
FOTO
Make liquid cultures of yesterday transformations.
24 July 2015
Minipreps of the 13 liquid culture. LacI:PIF:phyB:ren cultures did not grow.
Digestion of the minipreps:
Gal4:AsLOVpep
EcoRI
6345, 1972
LacI:AsLOVpep
EcoRI
6345, 2743
LexA:AsLOVpep
EcoRI
6345, 2011
PsinATG:RepPhiC31:GFP
HindIII
6345, 2691
LexA:PIF:PhyB:ren+luc
BamHI
9431, 6674, 3513
PhyC31; pUPD2
NotI
2046, 1899
Gal4:PIF:phyB
BamHI
6674, 3474, 2337
Renilla (GB159)
EcoRV
2909, 2475, 882, 812, 381
It was done 2 gels with ligations:
Gal4:AsLOV C1
Gal4:AsLOV C2
Gal4:AsLOV C3
PsinATG:RepPhiC31:GFP C1
PsinATG:RepPhiC31:GFP C2
ok
ok
ok
ok
no
PsinATG:RepPhiC31:GFP C3
LacI:AsLOV C1
LacI:AsLOV C2
LexA:AsLOV C1
LexA:AsLOV C2
no
ok
No
no
ok
LexA:AsLOV C3
LexA:PIF:PhyB:ren+luc C1
LexA:PIF:PhyB:ren+luc C2
no
no
Ok
PhiC31 (C1)
PhiC31 (C2)
PhiC31 (C3)
PhiC31 (C4)
PhiC31 (C5)
no
ok
ok
ok
no
Gal4:PIF:phyB
Renilla (GB159)
ok
Ok
26 July 2015
Liquid cultures of Agrobacterium were refreshed:
TsinATG:BxbIreporter:GFP
TsinATG:BxbI:reporterBxbI:GFP
Viral vectors (citoplasm, integrase)
GFP
Dsred
27 July 2015
Sent to sequence:
210.08.256: LacIBD; pUPD2 (C1)
210.08.258: Gal4BD; pUPD2 (C2)
210.08.259:LexABD; pUPD2 (C1)
210.08.260: PIF6; pUPD2 (C5)
210.08.261: VP16; pUPD2 (C1)
210.08.262: PhiC31; pUPD2 (C2)
210.08.264: PhiC31; pUPD2 (C3)
210.08.266: PhiC31; pUPD2 (C4)
210.08.268: ReporterBxbI; pUPD2 (C1)
210.08.269: ReporterPhiC31; pUPD2 (C1)
210.08.270: AsLOVpep; pUPD2 (C1)
The sample had to have 10µl of miniprep (200ng/µl aprox) + 5 µl of primer (dilution 1:3)
Ligations:
Gal4:PIF:phiB + ren; α1
LacI:PIF:phiB:ren + luc; Ω2
PIF:PhyB+renilla; α2
1.5 µl Gal4:PIF:PhyB
1.5 µl LacI:PIF:PhyB:ren
1.5 µl PIF:PhyB
2 µl Renilla (GB159)
2 µl OpLacI:luc
2.5 µl Renilla (GB159)
0.5 µl α1
0.5 µl Ω2
0.5 µl α2
3.6 µl H2O
2.6 µl H2O
3 µl H2O
OD’s measurements to agroinfiltrate:
Cytoplasm: 0.39 (viral)
0.25ml
Integrase: 0.35 (viral)
0.28ml
GFP: 0.32 (viral)
0.31ml
Dsred: 0.31 (viral)
0.32ml
BxbI:reporter: 0.41
0.49ml
Reporter: 0.26
0.77ml
EXPERIMENT 8.
BxbI:RepBxbI:GFP
RepBxbI:GFP
EXPERIMENT 9.
Infiltrate at vaccum the soyabeans sprouts with viral system and GFP.
Ligations to join the negative controls with renilla:
Etr8:luc+staffer(SF)
OpLexA:luc+SF
OpLacI:luc+SF
OpUAS:luc+SF
1.5 µl Etr8:luc (88C o 1098)
1.5 µl OpLexA:luc (151)
1.5 µl OpLacI:luc (152)
1.5 µl UAS:luc (227)
1 µl SF; α2
1 µl SF; α2
1 µl SF; α2
1 µl SF;α2
1 µl Ω1
1 µl Ω1
1 µl Ω1
1 µl Ω1
6.1 µl H2O
6.1 µl H2O
6.1 µl H2O
6.1 µl H2O
Transformation of E. coli of the ligations and make petri dish cultures:
Gal4:PIF:PhyB + ren; α1
LacI:PIF:PhyB:ren + luc; Ω2
PIF:PhyB+renilla; α2
Transformation into Agrobacterium with the constructions and make petri dish cultures:
Gal4:KDronpa:NDronpa:ren:luc
LacI:KDronpa:NDronpa:ren:luc
LexA:KDronpa:NDronpa:ren:luc
OpLexA:luc (GB151)
OpLacI:luc (GB152)
OpUAS:luc (GB227)
It was received a new piece (ePDZ) which is part of the blue toggle (plan A). It arrived in E. coli so it was made a lquid culture and let it grow at 37ºC overnight.
28 July 2015
Miniprep of the liquid culture: ePDZ.
Transformation into E. coli of the ligations:
Etr8:luc+staffer(SF)
OpLexA:luc+SF
OpLacI:luc+SF
UAS:luc+SF
Gal4:PIF:PhyB:ren
EXPERIMENT 9.
It was observed the soybean sprouts that were infiltrated with Agrobacteriumwith green light and red filter. It was not observed nothing significant, moreover, the damage is evident.
FOTO
Transformation in Agrobacterium the ReporterPhiC31:GFP.
Pick colonies and make liquid cultures adding X-Gal and IPTG because the colonies were little and we can not observe clearly if they were white or blue.
Gal4:PIF:phiB+ren. Did not grow any colony.
LacI:PIF:PhyB:ren+luc (C1-C3)
PIF:PhyB+renilla (C1- C3)
The medicine LTB (heat labile toxin B subunit) which is part of an enterotoxin of Echerichia coli homologous to the same toxin in Vibrio cholerae that causes diarrhea.
We add 50µl to have a final concentration of 10ng/µl.
Ligation:
LTB; pUPD2
1 µl LTB
1 µl pUPD2
5.6 µl H2O
29 July 2015
Miniprep of yesterday liquid culture:
LacI:PIF:phiB:ren+luc (C1 and C3) C2 turn into blue.
PIF:PhyB+renilla (C2) C1 and C3 did not grow.
Digestion:
LacI:PIF:phiB:ren:luc
EcoRV
882, 968, 1652, 3942, 2475, 381, 3477, 6674
PIF:PhyB+renilla
HindIII
4316, 5887, 788, 6345
Gel:
LacI:PIF:phiB:ren:luc (C1)
LacI:PIF:phiB:ren:luc (C3)
PIF:PhyB+renilla (C2)
no
no
no
Pick colonies and make liquid culture of:
Etr8:luc:staffer(SF) (C1-C3)
OpLexA:luc:SF (C1-C3)
OpLacI:luc:SF (C1-C3)
UAS:luc:SF (C1-C3)
Gal4:PIF:phyB:ren (C1-C3)
Transformation in E. coli of:
LTB; pUPD2
30 July 2015
Minipreps have been done:
Etr8:luc:staffer(SF) (C1-C3)
OpLexA:luc:SF (C1 and C3) C2 did not grow.
OpLacI:luc:SF (C1-C3)
OpUAS:luc:SF (C1-C3)
Gal4:PIF:phyB:ren (C1-C3)
LacI:PIF:phy:ren:luc (C1-C3)
PIF:PhyB:ren (C1-C3)
Etr8:luc:staffer(SF)
BamHI
6674, 2766
OpLexA:luc:SF
BamHI
6674, 2746
OpLacI:luc:SF
BamHI
6674, 2847
OpUAS:luc:SF
BamHI
6674, 2568
Gal4:PIF:phyB:ren
EcoRI
6345, 5487, 4852
LacI:PIF:phy:ren:luc
BamHI
20451
PIF:PhyB:ren
HindIII
6345, 5887, 4316, 788
Do the gel:
Primers have arrived:
ePDZ reverse and forward and phiC31.
Make a PCR with ePDZ and its primers to obtain the desired fragment and put it in pUPD2.
ePDZ PCR
10µl Buffer HF
31.5 µl H2O
2 µl dNTPs
2.5 µl  Primer forward
2.5 µl primer reverse
1 µl ePDZ (dilution 1:50)
0.5 µl Taq phunion
Pick 2 colonies of PhiC31:GFP in Agrobacterium and make liquid culture.
Ligations:
ePDZ; pUPD2
PIF:phyB+ren; α2
OpUAS:luc:SF+ren; α1
OpLexA:luc:SF+ren; α1
1 µl ePDZ
1 µl PIF:phyB
1 µl OpUAS:luc:SF
1 µl OpLexA:luc:SF
1 µl pUPD2
1 µl ren (159)
1 µl ren
1 µl ren
5.6 µl H2O
1 µl α2
1 µl α1
1 µl α1
4.6 µl H2O
4.6 µl H2O
4.6 µl H2O
OpEtr8:luc:SF+ren; α1
Op:LacI:luc:SF+ren; α1
1 µl OpEtr8:luc:SF
1 µl OpLacI:luc:SF
1 µl ren
1 µl ren
1 µl α1
1 µl aplha1
4.6 µl H2O
4.6 µl H2O
Make liquid culture (E. coli) of:
LTB; pUPD2 (C1-C3)
31 July 2015
Ligation:
LacI:PIF:PhyB:ren+OpLacI:luc; Ω2
1.5 µl LacI:PIF:phyB:ren
3 µl OpLacI:luc
0.5 µl Ω2
2.6 µl H2O
EXPERIMENT 8.
After 4 days till the agroinfiltration we have observed again the samples with BxbI and the negative control.
Minipreps of liquid culture LTB (C1 and C2)
Digestion:
LTB; pUPD2
NotI
2046, 474
Gel:
LTB (C1)
LTB (C2)
PCR ePDZ
ok
ok
ok
Take out glicerynates of Paloma, lab mate:
Sip rotavirus CH2
Sip rotavirus CH2-CH3
For E. coli and Agrobacterium.
Pick a colony of Asun interferon (IFN) in Agro, make liquid culture.
We had miniprep of IFN in pUPD. Make ligations.
Transform in E. coli the ligations and make petri dishes cultures:
ePDZ; pUPD2
PIF:phyB+ren; α2
OpUAS:luc:SF+ren; α1
OpLexA:luc:SF+ren; α1
OpEtr8:luc:SF+ren; α1
Op:LacI:luc:SF+ren; α1
LacI:PIF:phyB:ren+OpLacI:luc; Ω2
Refresh agro cultures to agroinfiltrate tomorrow:
PhiC31 (viral system)
ReporterPhiC31
BxbI+reporterBxbI
ReporterBxbI
Gal4:KDronpa:NDronpa:luc:ren (blue toggle)
EPIF:phi:luc
Renilla
P19
Pnos
August and September
1 August 2015
Prepare the Agrobacterium cultures for agroinfiltration:
Gal4B:KDronpa:NDronpa:luc:ren, PhiC31 and its control, BxbI and its control and PIF:PhyB and its control.
Mesurement oof the ODs:
Ren+P19
0.09
888.9 µl
RepPhiC31
0.69
115.94 µl
BxbI
0.46
174 µl
RepBxbI
0.15
533.3 µl
Gal4:KNDronpa:ren:luc
0.51
156.9 µl
Pnos
0.29
275.9 µl
PIF:phy:luc
0.17
470.6 µl
PhiC31
0.32
250 µl
EXPERIMENT 9.
PhiC31
RepPhiC31:GFP
BxbI:RepBxbI:GFP
RepBxbI:GFP
Gal4:NDronpa:KDronpa:luc:ren.
Pnos
Look the petri dishes culture:
PIF:phyB+ren; α2
OpLexA:luc:SF+ren; α1
Op:LacI:luc:SF+ren; α1
ePDZ; pUPD2
OpUAS:luc:SF+ren; α1
OpEtr8:luc:SF+ren; α1
LacI:PIF:PhyB:ren+OpLac:luc; α2
3 August 2015
Miniprep of the liquid culture:
Sip rotavirus CH2
Sip rotavirus CH2-CH3
Measure of DNA concentration of:
OpLexA:luc:SF=169ng/µl
OpacI:luc:SF=291ng/µl
Pick colonies and make liquid culture of:
ePDZ; pUPD2 (C1-C3)
OpUAS:luc:SF+ren; α1(C1-C3)
OpEtr8:luc:SF+ren; α1(C1-C3)
LacI:PIF:phyB:ren+OpLacI:luc; Ω2 (C1-C3) Added X-gal and IPTG.
Repeat ligations:
OpLexA:luc:SF+ren; α1
Op:LacI:luc:SF+ren; α1
E-PIF:phyB+ren; α2
1 µl OpLexA:luc:SF
1 µl OpLacI:luc:SF
1 µl E-PIF:phy
3 µl ren
3 µl ren
3 µl ren
1 µl α1
1 µl α1
1 µl α2
2.6 µl H2O
2.6 µl H2O
2.6 µl H2O
Refresh agrobacterium cultures of:
Interferon
SIP-CH2
SIP-CH2-CH3
Take the glycerinate of Renilla; Ω2 (GB159) and make liquid culture.
EXPERIMENT 9.
We have made all the leaf discs and put in order in the plates.
4 August 2015
We do a Western blot with Asun so we can learn how to do it. Lo copio? De verdad!!?
Miniprep of the liquid cultures:
ePDZ; pUPD2 (C1-C3)
OpUAS:luc:SF+ren; ?1(C1-C3)
OpEtr8:luc:SF+ren; ?1(C1-C3)
LacI:PIF:phyB+ren; ?2 (C2 and C3) C1 was blue.
Digestions:
ePDZ; pUPD2
NotI
2046, 642
OpUAS:luc:SF+ren; α1
EcoRI
6345, 7096
OpEtr8:luc:SF+ren; α1
EcoRI
6345, 7294
LacI:PIF:phyB:ren+OpLacI:luc; Ω2
EcoRV
6674, 3477, 381, 2475, 3942, 1652, 968, 882
Renilla 159
EcoRV
2909, 2475, 882, 812, 381
Gel:
ePDZ C1
ePDZ C2
ePDZ C3
LacI:PIF:PhyB+ren
Ok
ok
ok
Renilla 159
OpUAS:luc:SF+ren C1
OpUAS:luc:SF+ren C2
OpUAS:luc:SF+ren C3
Ok
¿?
OpEtr8:luc:SF+ren C1
OpEtr8:luc:SF+ren C2
OpEtr8:luc:SF+ren C3
Transformation into E. coli of yesterday ligations and make petri dish culture:
OpLexA:luc:SF+ren; α1
Op:LacI:luc:SF+ren; α1
E-PIF:phyB+ren; α2
Ligations:
35S+ePDZ+VP16+T35S;α2
35S+LTB+T35S; α1
1µl ePDZ
1µl 35S (30)
1 µl VP16
1µl T35S (36)
1 µl 35S (30)
1µl LTB
1 µl T35S (36)
1µl α1
1 µl α2
3.6 µl H2O
2.6 µl H2O
Refresh agrobacterium cultures of:
Interferon
SIP-CH2
SIP-CH2-CH3
5 August 2015
Transform ligations:
35S+ePDZ+VP16+T35S; α2
35S+LTB+T35S; α1
Pick colonies and make liquid cultures cultures of:
OpLexA:luc:SF+ren; α1 (C1-C3)
Op:LacI:luc:SF+ren; α1 (C1-C3)
E-PIF:phyB+ren; α2 (C1)
35S+ePDZ+VP16+T35S; α2 (C1 and C2)
35S+LTB+T35S; α1 (C1 and C2)
EXPERIMENT 9.
Make luciferase essay of red and blue toggle:
6 August 2015
Miniprep of the liquid cultures.
Digestion:
OpLexA:luc:SF+ren
EcoRI
6345, 7274
Op:LacI:luc:SF+ren
EcoRI
6345, 7375
E-PIF:phyB+ren
HindIII
6345, 788, 5887, 4316
35S+ePDZ+VP16+T35S
HindIII
6345, 2316
35S+LTB+T35S
EcoRI
6345, 1684
Gel:
PIF:phyB+ren
OpLexA:luc:SF+ren C1
OpLexA:luc:SF+ren C2
OpLexA:luc:SF+ren C3
No
Ok, repeat
No
Ok, repeat
Op:LacI:luc:SF+ren C1
Op:LacI:luc:SF+ren C2
Op:LacI:luc:SF+ren C3
35S+LTB+T35S C1
Ok, repeat
Ok
Ok
ok
35S+LTB+T35S C2
35S+ePDZ+VP16+T35S C1
35S+ePDZ+VP16+T35S C2
Ok
ok
ok
Make colony PCR with 6 colonies of PhiC31:
DNA-
JM1: 2 µl (dilution 1:10)
JM2: 2 µl (dilution 1:10)
Buffer Taq (with Mg): 2 µl
dNTPs: 2.5 µl
Taq: 0.5 µl
H2O: 10 µl
Program: 96ºC-2min; 96ºC-30min; 55ºC-30min; 72ºC-30min
Make a gel with the PCR but we did not see the correct bands.
Repeat the colony PCR:
DNA-
JM1: 2.5 µl (dilution 1:10)
JM2: 2.5 µl (dilution 1:10)
Buffer Taq (with Mg): 10 µl
dNTPs: 2 µl
Taq: 0.5 µl
H2O: 7.5 µl
Program: : 96ºC-10min; 96ºC-30min; 55ºC-30min; 72ºC-30min
Ligations:
LexA:AsLOVpep+ePDZ; Ω1
LacI:AsLOVpep+ePDZ;Ω1
Gal4:AsLOVpep+ePDZ; Ω1
1 µl LexA:AsLOVpep; α1
1 µl LacI:AsLOVpep; α1
1 µl Gal4:AsLOVpep;α1
1 µl ePDZ; α2
1 µl ePDZ; α2
1 µl ePDZ; α2
1 µl BsmBI
1 µl BsmBI
1 µl BsmBI
1 µl Ω1
1 µl Ω1
1 µl Ω1
4.6 µl H2O
4.6 µl H2O
4.6 µl H2O
Refresh Agrobacteriumcultures to agroinfiltrate:
PhiC31:RepPhiC31:GFP
RepPhiC31:GFP
BxbI:RepBxbI:GFP
RepBxbI:GFP
IFN (interferon)
Sip-CH2
Sip-CH2-CH3
Pnos
P19 (this culture has 10ml of LB + 10 µl antibiotics + 5 µl culture)
7 August 2015
Transformation into Agrobacterium:
35S:LTB:VP16:T35S
Transformation into E. coli and make petri dishes cultures:
LexA:AsLOVpep+ePDZ
LacI:AsLOVpep+ePDZ
Gal4:AsLOVpep+ePDZ
Make a gel with the colonies PCRs of PhiC31:
C7
C8
C9
C10
C11
C12
C13
C14
No
no
no
no
no
no
no
no
There was no DNA, there is a problem with the cells or the procedure, it have to be revised.
Ligation:
PhyC31; pUPD2
3 µl PhiC31
1 µl pUPD2
1 µl BsmBI
3.6 µl H2O
Refresh agrobacterium cultures for tomorrow infiltration:
Sip-CH2
Sip-CH2-CH3
Pnos
Red toggle (PIF6:PhyB:luc)
Renilla
Blue toggle (LacI:KDronpa:NDronpa:ren:OpLacI:luc)
EXPERIMENT 10.
PhiC31:RepPhiC31:GFP
RepPhiC31:GFP
BxbI:RepBxbI:GFP
RepBxbI:GFP
IFN (interferon)
Measure f the ODs:
Construction
OD
Volume (ml)
IFN
0.13
1.54
RepBxbI:GFP
0.13
1.54
BxbI:RepBxbI:GFP
0.33
0.61
PhiC31
0.19
1.05
RepPhiC31:GFP
0.22
0.91
8 August 2015
Transform the ligation into E. coli and make petri dishes cultures:
PhiC31; pUPD2.
LexA:AsLOVpep+ePDZ (C1-C5)
LacI:AsLOVpep+ePDZ (C1-C4)
Gal4:AsLOVpep+ePDZ (C1-C4)
EXPERIMENT 11.
Sip-CH2
Sip-CH2-CH3
Pnos
Red toggle (PIF6:PhyB:luc)
Renilla
Blue toggle (LacI:KDronpa:NDronpa:ren:OpLacI:luc)
Measurement of the ODs:
Construction
OD
Volume (ml)
Sip-CH2
0.04
6.667
Sip-CH2-CH3
0.04
6.667
Red toggle (PIF6:PhyB:luc)
0.09
15
Blue toggle (LacI:KDronpa:NDronpa:ren:luc)
0.09
15.00
Renilla
0.04
6.667
Pnos
0.04
6.667
9 August 2015
Make a colony PCRs with the 10 white colonies of PhiC31 that grow in the petri dishes.
DNA- colony
JM1: 2 µl (dilution 1:10)
JM2: 2 µl (dilution 1:10)
Buffer Taq (with Mg): 2 µl
dNTPs: 2.5 µl
Taq: 0.5 µl
H2O: 1 µl
Minipreps of the liquid cultures:
LexA:AsLOVpep+ePDZ (C1-C4) C5 has turn into blue color.
LacI:AsLOVpep+ePDZ (C1-C4)
Gal4:AsLOVpep+ePDZ (C1-C4)
Digestion of the minipreps:
LexA:AsLOVpep+ePDZ
BamHI
6674, 4309
LacI:AsLOVpep+ePDZ
BamHI
6674, 5041
Gal4:AsLOVpep+ePDZ
BamHI
6674, 4270
Make the gel:
LexA:AsLOVpep+ePDZ (C1)
LexA:AsLOVpep+ePDZ (C2)
LexA:AsLOVpep+ePDZ (C3)
LexA:AsLOVpep+ePDZ (C4)
Ok
ok
ok
ok
oLacI:AsLOVpep+ePDZ (C1)
LacI:AsLOVpep+ePDZ (C2)
LacI:AsLOVpep+ePDZ (C3)
LacI:AsLOVpep+ePDZ (C4)
Ok
ok
ok
ok
Gal4:AsLOVpep+ePDZ (C1)
Gal4:AsLOVpep+ePDZ (C2)
Gal4:AsLOVpep+ePDZ (C3)
Gal4:AsLOVpep+ePDZ (C4)
Ok
ok
ok
Ok
We keep in our inventory the colony C1 of each construction
Make a gel of the colonies PCRs of PhiC31: we did not observed any DNA. Still can’t fix the problem.
Make liquid culture of renilla and PIF6:PhyB:luc in Agrobacterium because the last time that we did an experiment they have grown slowly.
Store in the 4ºC fridge an agrobacterium culture with P19.
Ligations:
LexA:AsLOVpep+ePDZ+ren; α1
LacI:AsLOVpep+ePDZ+ren; α1
Gal4:AsLOVpep+ePDZ+ren; α1
1µl LexA:AsLOVpep+ePDZ
1µl LacI:AsLOVpep+ePDZ
1µl Gal4:AsLOVpep+ePDZ
1 µl renilla (159)
1 µl renilla (159)
1 µl renilla (159)
1 µl α1
1 µl α1
1 µl α1
4.6 µl H2O
4.6 µl H2O
4.6 µl H2O
10 August 2015
Transform into E. coli this ligations and make petri dishes cultures:
LexA:AsLOVpep+ePDZ+ren; α1
LacI:AsLOVpep+ePDZ+ren; α1
Gal4:AsLOVpep+ePDZ+ren; α1
Make liquid culture of 11 colonies of PhiC31 due to that this construction was giving us problems to obtain.
Refresh the agrobacterium cultures of:
BxbI:Rep:BxbI:GFP
Rep:BxbI:GFP
PhiC31
RepPhiC31:GFP
P19
Citoplasm
Integrase
Dsred
GFP
EXPERIMENT 10.
Take the samples of the agroinfiltrated plants that were in darkness to change the conditions.
Make liquid culture of:
LexA:AsLOVpep+ePDZ+ren (C1 and C2)
LacI:AsLOVpep+ePDZ+ren (C1-C4)
Gal4:AsLOVpep+ePDZ+ren (C1 and C2)
11 August 2015
Primers have arrived, they have been resuspended. They were used to eliminate the recognitions sites of the enzymes in BioBricks. This will make our constructions ready to be send to the iGEM
Minipreps of the liquid cultures done yesterday:
PhiC31 (C6-C16)
LacI:AsLOVpep+ePDZ+ren (C1-C4)
Gal4:AsLOVpep+ePDZ+ren (C1 and C2)
LexA:AsLOVpep+ePDZ+ren colonies were all blue
Make PCRs with all the primers and constructions:
All of them follow the same composition which is:
10 µl
Buffer HF
26.5 µl
H2O
2 µl
dNTPs
0.5 µl
Taq Phusion
1 µl
DNA (dilution 1:10)
2.5 µl
Primer forward (F) (dilution 1:10)
2.5 µl
Primer reverse (R) (dilution 1:10)
*The three last lines are the only specify bellow.
IFN (1)
IFN (2)
AsLOVpep (1)
AsLOVpep (2)
IFN
IFN
AsLOVpep
AsLOVpep
Mys int F
IFN domBB R
AsLOVpep F
AsLOVpep Fint
Mys int R
IFN domBB F
AsLOVpep Rint
AsLOVpep R
AsLOVpep (nested)
PhiC31 (1)
PhiC31 (2)
PhiC31 (3)
AsLOVpep
PhiC31
PhiC31
PhiC31
AsLOVpep nested
PhiC31 Fint 1
PhiC31 Fint 2
PhiC31 Fint 3
AsLOVpep
PhiC31 Rint 2
PhiC31 Rint 3
PhiC31 R
PhiC31
PhiC31
PhiC31 F
PhiC31 Rint
Digestions of the minipreps:
PhiC31
NotI
2046, 1899
LacI:AsLOVpep+ePDZ+ren
EcoRI
6345, 8798
Gal4:AsLOVpep+ePDZ+ren
EcoRI
6345, 9569
Make the gel:
PhiC31 (C6)
C7…C16
LacI:AsLOVpep:ePDZ:ren (C1)
LacI:AsLOVpep:ePDZ:renC2
LacI:AsLOVpep:ePDZ:ren(C3)
LacI:AsLOVpep:ePDZ:ren(C4)
no
no
ok
no
ok
Gal4:AsLOVpep+ePDZ+ren (C1)
Gal4:AsLOVpep+ePDZ+ren (C2)
ok
ok
Mesurement of the ODs:
Citoplasm
0.21
7.35 ml
RepPhiC31
0.26
9.1ml
35S:LTB:T35S
0.27
9.45ml
PhiC31
0.27
9.45ml
GFP
0.20
10ml
RepBxbI
0.33
11.55ml
PsinATG:RepBxbI:GFP
0.36
12.6ml
PhiC31
0.34
11.9ml
DsRed
0.26
9.1ml
P19
0.28
9.8ml
EXPERIMENT 11.
Take the samples of the agroinfiltrated plants that were in darkness to change the conditions.
EXPERIMENT 12.
Citoplasm
RepPhiC31
35S:LTB:T35S
PhiC31
GFP
RepBxbI
PsinATG:RepBxbI:GFP
PhiC31
DsRed
P19
12 August 2015
New digestions to check if the negative control constructions are correct and we can transformed it in agro
OpLexA:luc:SF:ren
EcoRI
6345, 7274
Op:UAS:luc:SF:ren
EcoRI
6345, 7096
OpEtr8.luc:SF:ren
EcoRI
6345, 7294
Gel:
Op:UAS:luc:SF:ren C1
Op:UAS:luc:SF:ren C2
Op:UAS:luc:SF:ren C3
OpEtr8.luc:SF:ren
no
no
no
no
OpLexA:luc:SF:ren C1
OpLexA:luc:SF:ren C2
Partner dna
Partner dna
no
no
Transform in Agrobacteriumand make petri dishes cultures:
LexA:AsLOVpep:ePDZ
Gal4:AsLOVpep:ePDZ
LacI:AsLOVpep:ePDZ
LexA:PIF6:phyB:VP16
Gal4:PIF6:phyB:VP16
LacI:PIF6:phyB:VP16
All of them are in Ω1.
Ligations:
LexA:AsLOVpep:ePDZ+ren; α1
Gal4:AsLOVpep:ePDZ:ren+OpUAS:luc; Ω1
LacI:AsLOVpep:ePDZ:ren+OpLacI:luc; Ω1
1 µl LexA:AsLOVpep:ePDZ
1 µl Gal4:AsLOVpep:ePDZ:ren
1 µl LacI:AsLOVpep:ePDZ
1 µl renilla (159)
1 µl OpUAS:luc
1 µl OpLacI:luc
1 µl α1
1 µl Ω1
1 µl Ω1
4.6 µl H2O
4.6 µl H2O
4.6 µl H2O
PROTOPLAST 1: Do for the first time. PROTOCOL, click here.
13 August 2015
Analysis of the AsLOVpep PCR’s products that were set to sequencing:
The primer AsLOVpep F (forward) works well.
The primer AsLOVpep Ri (reverse intern): it is possible that occurs a deletion in the position 147.
The primer AsLOVpep Fi (forward intern): change in one base A to C in the position 446, last base in the codon, the amino acid (alanine) still the same.
The primer AsLOVpep R (reverse): it is ok but the las 50bp are no sequenced.
The primer PhiC31 Fi (forward intern): did not work well. 4 consecutive deletions and changes in the nucleotides from 470 to 990.
The primer PhiC1 Ri2 (reverse intern 2): corrects the errors in the sequencing of the las primer. There is not deletion in the zone between 470 till 990 but is diffuse.
The primer PhiC31 Fi2 (forward intern 2): possible deletion in 1063.
The primer PhiC31 Fi3 (forward inter 3): ok
The primer PhiC31 R (reverse): ok
Transform into E. coli yesterday ligations and make petri dishes cultures:
LexA:AsLOVpep:ePDZ+ren; ?1
Gal4:AsLOVpep:ePDZ:ren+OpUAS:luc; ?1
LacI:AsLOVpep:ePDZ:ren+OpLacI:luc; ?1
PROTOPLAST 1:We try to obtain the protoplast but we make a mistake, repeat.
PROTOPLAST 2. Start the procedure again.
EXPERIMENT 11. Luciferase essay of blue and red toggle.
Ligations of the PCR products:
AsLOVpep (BB); pUPD2
1µl AsLOVpep1
1µl AsLOVpep2
1µl pUPD2
1µl BsmbI
4.6µl H2O
14 August 2015
Repeat the digestions of the negative controls:
OpLexA:luc:SF:ren
SacII
13619
PouI
2495, 2055, 5937, 3152
OpUAS:luc:SF:ren
PouI
2475, 2055, 5937, 2474
OpEtr8.luc:SF:ren
PouI
2637, 2055, 5937, 3010
OpLacI:luc:SF:ren
SacII
12930, 790
OpLacI:luc:SF:ren
PouI
2475, 2055, 5937, 3253
Make the gel:
OpUAS:luc:SF:ren C1
OpUAS:luc:SF:ren C2
OpUAS:luc:SF:ren C3
OpEtr8.luc:SF:ren
ok
no
ok
ok
OpLacI:luc:SF:ren (PouI)
OpLacI:luc:SF:ren (SacII)
OpLexA:luc:SF:ren (PouI)
OpLexA:luc:SF:ren (SacII)
ok
ok
ok
ok
Transform into E. coli AsLOVpep (BB) and make petri dish culture.
Make liquid cultures of:
LexA:AsLOVpep:ePDZ+ren; ?1 (C1-C4)
Gal4:AsLOVpep:ePDZ:ren+OpUAS:luc; ?1 (C1-C4)
LacI:AsLOVpep:ePDZ:ren+OpLacI:luc; ?1 (C1-C4)
Make liquid cultures of agrobacterium cultures in petri dishes:
LexA:AsLOVpep:ePDZ
Gal4:AsLOVpep:ePDZ
LacI:AsLOVpep:ePDZ
Gal4:PIF6:phyB:VP16
LacI:PIF6:phyB:VP16
We make triple strake method to obtain isolated colonies of LexA:PIF6:PhyB:VP16.
PROTOPLAST 2:
Continue with the procedure to obtain protoplasts. We finally see in the microscope the protoplasts.
Transform protoplasts. How? Click here.
E:PIF6:PhyB:luc+ren
E:PIF6:PhyB:luc+ren
EXPERIMENT 12.
Prepare samples of both recombinases and its negative control to see them in the confocal microscope. We see samples of plants agroinfiltrated 7 days ago and other agroinfiltrated 5 days ago with the viral silencing repressor, P19.
PROTOPLAST 2:
Take samples at time 8h of protoplasts.
Samples of both toggles and in all the conditions.
15 August 2015
Minipreps of:
LexA:AsLOVpep:ePDZ+ren; α1
Gal4:AsLOVpep:ePDZ:ren+OpUAS:luc; Ω1
LacI:AsLOVpep:ePDZ:ren+OpLacI:luc; Ω1
Minipreps digestions:
LexA:AsLOVpep:ePDZ+ren; α1
EcoRI
8837, 6345
Gal4:AsLOVpep:ePDZ:ren+OpUAS:luc; Ω1
BamHI
6674, 11186
LacI:AsLOVpep:ePDZ:ren+OpLacI:luc; Ω1
BamHI
6674, 12236
Make the gel:
LexA:AsLOVpep:ePDZ+ren C1
LexA:AsLOVpep:ePDZ+ren C2
LexA:AsLOVpep:ePDZ+ren C3
LexA:AsLOVpep:ePDZ+ren C4
ok
ok
ok
Ok
Gal4:AsLOVpep:ePDZ:ren+OpUAS:luc C1
Gal4:AsLOVpep:ePDZ:ren+OpUAS:luc C2
Gal4:AsLOVpep:ePDZ:ren+OpUAS:luc C3
Gal4:AsLOVpep:ePDZ:ren+OpUAS:luc C4
no
no
no
No
LacI:AsLOVpep:ePDZ:ren+OpLacI:luc C1
LacI:AsLOVpep:ePDZ:ren+OpLacI:luc C2
LacI:AsLOVpep:ePDZ:ren+OpLacI:luc C3
LacI:AsLOVpep:ePDZ:ren+OpLacI:luc C4
ok
ok
ok
ok
We keep LexA:AsLOVpep:ePDZ+ren C1 and LacI:AsLOVpep:ePDZ:ren+OpLacI:luc C4.
Make liquid cultures of (added X-gal + IPTG):
AsLOvpep (BB); pUPD2 (C1-C4)
PROTOPLASTS 3.
We cut 3 leafs into strips and let this overnight shaking at minimum.
PROTOPLAST 2.
Take protoplasts samples at time 16h. Samples of both toggles and in all the conditions.
Take also time 24h.
16 August 2015
All liquid cultures of AsLOvpep (BB); pUPD2 (C1-C4) were blue.
Pick 4 more colonies also with added X-gal + IPTG
Ligations:
Gal4:AsLOVpep:ePDZ:ren+OpUAS:luc; Ω1
AsLOVpep; pUPD2
LexA:AsLOVpep:ePDZ:ren+OpLexA:luc; Ω1
1µl Gal4:AsLOVpep:ePDZ:ren
1µl AsLOVpep1
1µl Lex!:AsLOVpep:ePDZ:ren
1µl OpUAS:luc (227)
1µl AsLOVpep2
1µl OpLexA:luc (151)
1µl Ω1
1µl pUPD2
1µl Ω1
4.6µl H2O
4.6µl H2O
4.6µl H2O
PROTOPLASTS 3.
Finish the protoplasts started yesterday, we obtain good concentrations.
Transform them:
LacI:AsLOVpep:ePDZ:ren:OpLacI:luc
LacI:KDronpa:NDronpa:ren:OpLacI:luc
OpLacI:luc:ren
PROTOPLAST 2. Luciferase essay.
Do a luciferase essay with protoplasts, click here.
Transform into agro and make petri dish culture of:
LacI:AsLOVpep:ePDZ:ren:OpLacI:luc; Ω1
Refresh agrobacterium liquid culture of:
Viral system (cytoplasm, integrase, Dsred).
Pnos, to make a miniprep tomorrow and obtain the DNA construction.
17 August 2015
PROTOPLASTS 3.
Take protoplasts samples at time 16h. Big disapoint here, we dscovered hat they were cloroplasts... ha, ha.
Liquid cultures of AsLOVpep (BB); pUPD2 were all blue. The ligation was made again.
Miniprep of Pnos in Agrobacterium.
Transform the ligations in E. coli:
Gal4:AsLOVpep:ePDZ:ren+OpUAS:luc; Ω1
AsLOVpep; pUPD2
LexA:AsLOVpep:ePDZ:ren+OpLexA:luc; Ω1
EXPERIMENT 13.
Prepare the viral system in agrobacterium cultures to infiltrate into sunflowers sprouts by vacuum.
Measurement of the ODs:
DsRed
0.14
71.4ml
Integrase
0.14
71.4ml
Citoplasm
0.12
83.3ml
Make 60ml of medium for each infiltration.
New pieces have arrived:
1st part Lactoferrina
Primer PhiC31 F
EXPERIMENT 11 and 12.
Take samples of leads agroinfiltrated with LTB, IFN and sip rotavirus CH2 and CH2-CH3.
18 August 2015
Make PCRs of PhiC31 and AsLOVpep to eliminate the enzyme recognition sites of Biobricks (BB).
PhiC31 (BB)
AsLOVpep (BB)
1 µl PhiC31
1 µl AsLOVpep
2.5 µl PhiC31 F (1:10)
2.5 µl AsLOVpep nested
2.5 µl PhiC31 Rint (1:10)
2.5 µl AsLOVpep Fint
Tm=44ºC
Tm=59ºC
Program: 96ºC-10’; 96ºC-30’’; Tm-30’’ and 72ºC-30’’
Ligation of the PCRs products:
PhiC31 (BB); pUPD2
AsLOVpep (BB); pUPD2
1µl Phi1
1µl AsLOVpep1
1µl Phi2
1µl AsLOVpep nested
1µl Phi3
1µl pUPD2
1µl Phi4
4.6µl H2O
1µl pUPD2
2.6µl H2O
Transform in Agrobacterium:
OpLexA:luc:SF:ren; α1
OpUAS:luc:SF:ren; α1
OpEtr8:luc:SF:ren; α1
OpLacI:luc:SF:ren; α1
Make liquid culture of:
Gal4:AsLOVpep:ePDZ:ren+OpUAS:luc; Ω1 (C1 and C2)
LexA:AsLOVpep:ePDZ:ren+OpLexA:luc; Ω1 (C1-C3)
We start to do the western to see if any drugs had been produced in our plants.
The gel order is:
IFN 1
IFN2
LTB1 (10µL)
LTB1 (10 µL)
LTB2 (30 µL)
Sip 1 CH2
Sip2 CH2
Sip 1 CH2-CH3
Sip 2 CH2-CH3
Make a gel with the PCRs products:
PhiC31 and AsLOVpep modified for BioBricks. NO SE LAS BANDAS ESPERADAS.
We change a little bit the protocol to make protoplasts, now we follow a protocol found in a Nature paper. The important changes are that during the digestion of the cell wall there is no agitation.
Transform into E. coli the PCRs products:
PhiC31 (BB)
AsLOVpep (BB)
Prepare liquid culture of:
LTB; α1
Gal4:AsLOVpep:ePDZ; Ω1
LacI:AsLOVpep:ePDZ; Ω1
LexA:AsLOVpep:ePDZ; Ω1
ePDZ; pPUD2
AsLOVpep; pUPD2
NDronpa; pUPD2
Gal4:AsLOVpep; α1
RepPhiC31; pPUPD2
LexA:AsLOVpep; α1
19 August 2015
Miniprep of the liquid cultures:
Gal4:AsLOVpep:ePDZ:ren+OpUAS:luc; Ω1 (C1 and C2), did not grow.
Digestions:
LTB; α1
EcoRI
6345, 1684
Gal4:AsLOVpep:ePDZ; Ω1
BamHI
6674, 4270
LacI:AsLOVpep:ePDZ; Ω1
BamHI
6674, 5041
LexA:AsLOVpep:ePDZ; Ω1
BamHI
6674, 4309
ePDZ; pPUD2
NotI
2046, 642
AsLOVpep; pPUD2
NotI
2046, 521
NDronpa; pPUD2
NotI
2046, 735
Gal4:AsLOVpep; α1
EcoRI
6345, 1972
RepPhiC31: pPUD2
NotI
2046, 475
LexA:AsLOVpep; α1
EcoRI
6345, 2016
LexA:AsLOVpep:ePDZ:ren+OpLexA:luc; Ω1
BamHI
6674, 11403
Gel:
LTB; α1
NDronpa; pPUD2
RepPhiC31: pPUD2
AsLOVpep; pPUD2
ok
ok
ok
ok
LexA:AsLOVpep
Gal4:AsLOVpep
LTB; α1
LexA:AsLOVpep; α1
ok
ok
ok
Ok
LacI:AsLOVpep:ePDZ
Gal4:AsLOVpep:ePDZ
LexA:AsLOVpep:ePDZ:ren+OpLexA:luc C2
LexA:AsLOVpep:ePDZ:ren+OpLexA:luc C3
ok
ok
ok
Ok
Make liquid culture of:
PhiC31 (BB) (C1-C9)
AsLOVpep (BB) (C1-C3)
Also had grown the liquid cultures:
LTB; pUPD2
LacI:AsLOVpep:ePDZ:ren; α1 (C1 and C2)
Gal4;pUPD2 and RepBxbI; α2 did not grow.
20 August 2015
All the liquid cultures of AsLOVpep (BB) and PhiC31 (BB) had grown. Do 12 minipreps.
Digestions of the minipreps:
AsLOVpep (BB)
NotI
2046, 518
PhiC31 (BB)
NotI
2046, 1899
Do the gel:
AsLOV C1
AsLOV C2
AsLOV C3
Phi C1
Phi C2
Phi C3
Ok
ok
ok
no
no
no
Phi C4
Phi C5
Phi C6
Phi C7
Phi C8
Phi C9
No
no
Ok?
Ok?
no
No
We keep the three colonies of AsLOVpep but we make another digestion with different enzymes of PhiC31 colonies C6 and C7.
Optimized ligation:
Gal4:AsLOVpep:ePDZ:ren+OpUAS:luc; Ω1
2.5 µl Gal4:AsLOVpep:ePDZ:ren
6 µl OpUAS:luc
0.5 µl Ω1
0 µl H2O
Make glycerinates of:
ePDZ; pUPD2
NDronpa; pUPD2
RepPhiC31; pUPD2
AsLOVpep; pUPD2
LexA:AsLOVpep; α1
Gal4:AsLOVpep; α1
LexA:AsLOVpep:ePDZ; Ω1
LacI:AsLOVpep:ePDZ; Ω1
Gal4:AsLOVpep:ePDZ; Ω1
LexA:AsLOVpep:ePDZ:ren:OpLexA:luc; Ω1
Make liquid culture of positive controls in Agro:
OpUAS:luc:ren
OpLexA:luc:ren
OpLacI:luc:ren
OpEtr8:luc:ren
Prepare the liquid cultures of Agrobacteriumto agroinfiltrate:
IFN
0.26
77 µl/ng
LTB
0.22
91 µl/ng
Sip CH2-CH3
0.12
167 µl/ng
Sip CH2
0.09
223 µl/ng
The total volume of preparation will be 6ml each drug.
EXPERIMENT 14.
IFN
LTB
Sip CH2-CH3
Sip CH2
Refresh more colonies to make more glycerinates and pick two colonies of the petri dish culture:
Gal4:KDronpa:NDronpa:ren:OpUAS:luc; Ω1
LexA:KDronpa:NDronpa:ren:OpLexA:luc; Ω1
LTB; pUPD2
LTB; α1
Gal4:AsLOVpep:ePDZ:ren:OpUAS:luc; Ω1 (C1 and C2)
Miniprep of the liquid culture:
LacI:AsLOVpep; α1 (C1 and C2)
Digestion:
LacI:AsLOVpep
EcoRI
8345, 2743
New digestion to PhiC31 (BB):
PhiC31 (BB)
AatII
2635, 1310
PhiC31 (BB)
BanI
3497, 448
Make the gel:
PhiC31(BB) C6 (banI)
PhiC31(BB) C7 (BanI)
PhiC31(BB) C6 (AatII)
PhiC31(BB) C7 (AatII)
Ok
no
ok
no
Transformation in Agrobacteriumand make petri dishe culture of :
LexA:AsLOVpep:ePDZ:ren:OpLexA:luc; Ω1
21 August 2015
Miniprep of:
Gal4:KDronpa:NDronpa:ren:OpUAS:luc
LexA:KDronpa:NDronpa:ren:OpLexA:luc
LTB; pUPD2
LTB; α1
Gal4:AsLOVpep:ePDZ:ren:OpUAS:luc; Ω1 (C1 and C2)
Digestions of the minipreps:
Gal4:KDronpa:NDronpa:ren:OpUAS:luc; Ω1
BamHI
6674, 11582
LexA:KDronpa:NDronpa:ren:OpLexA:luc; Ω1
BamHI
6674, 11799
LTB; pUPD2
NotI
2046, 474
LTB; α1
EcoRI
1684, 6345
Gal4:AsLOVpep:ePDZ:ren:OpUAS:luc; Ω1
BamHI
6674, 11186
Do the gel:
LTB; pUPD2
LTB; α1
LTB; α1
Gal4:KDronpa:NDronpa:ren:OpUAS:luc; Ω1
No
no
no
ok
LexA:KDronpa:NDronpa:ren:OpLexA:luc; Ω1
Gal4:AsLOVpep:ePDZ:ren:OpUAS:luc; Ω1 C1
Gal4:AsLOVpep:ePDZ:ren:OpUAS:luc; Ω1 C2
Ok
no
no
We make glicerynates of:
Gal4:KDronpa:NDronpa:ren:OpUAS:luc; Ω1
LexA:KDronpa:NDronpa:ren:OpLexA:luc; Ω1
Transform into E. coli and make Petri dish culture:
Gal4:AsLOVpep:ePDZ:ren:OpUAS:luc; Ω1
Refresh this cultures to agroinfiltrate:
Pnos
LacI:AsLOVpep:ePDZ:ren:OpLacI:luc
Still trying how to make protoplasts… We will win this battle!
22 August 2015
Make liquid culture of the culture of Gal4:AsLOVpep:ePDZ:ren:OpUAS:luc; Ω1. There was only one white colony, nothing hopeful…
We fount that the culture of Pnos in Agrobacterium did not grow, so we decided not to carry on with the experiment because it was our control. Refresh again two different cultures of Pnos that were found in the fridge and do the experiment tomorrow.
Transform into Agrobacteriumthe following constructions:
E:PIF6:PhyB
LexA:PIF6:PhyB
LacI:PIF6:PhyB
Gal4:PIF6:PhyB
Miniprep of:
Pnos1 (agro)
Pnos2 (Agro)
PhiC31 (BB); pUPD2 (C6)
Digestion:
PhiC31 (BB); pUPD2
NotI
2046, 1899
Pnos
EcoRI
7317, 634, 11
LTB; pUPD2
NotI
2046, 474
LTB; α1
EcoRI
1684, 6345
Gel:
LTB; α1
LTB; pUPD2
Pnos 1
Pnos 2
PhiC31 (BB)
Ok
no
?
?
Ok
As the Pnos construction was given to us form a lab partner, we do not surely that the expected bands were correct because we choose the construction at the golden braid register parts.
We can not make any glicerynate because it was not the necessary amount of liquid culture to make it. Refresh again: PhiC31 (BB); pUPD2; LTB; α1.
23 August 2015
Prepare the Agrobacteriumcultures to agroinfiltrate:
Pnos and LacI:AsLOVpep:ePDZ:ren:OpLacI:luc.
Pnos
0.30
1ml/15ml
LacI:AsLOVpep:ePDZ:ren:OpLacI:luc
0.34
OpLacI:luc:ren
0.47
EXPERIMENT 15.
Pnos
LacI:AsLOVpep:ePDZ:ren:OpLacI:luc.
OpLacI:luc
PhiC31 (BB); pUPD2
Gal4:AsLOVpep:ePDZ:ren:OpUAS:luc; Ω1
LTB; α1 did not grow.
Make liquid culture of agro colonies:
LexA:AsLOVpep:ePDZ:ren:OpLexA:luc; Ω1
24 August 2015
Digestion of yesterday minipreps:
Gal4:AsLOVpep:ePDZ:ren:OpUAS:luc
BamHI
11186, 6674
PhiC31 (BB)
NotI
2096, 1899
Do a gel:
Gal4:AsLOVpep:ePDZ:ren:OpUAS:luc
PhiC31 (BB)
Ok
ok
It had arrived the second part of the lactoferrin.
Resuspend them 100 µl.
Ligation into pUPD2 of both parts of the lactoferrin and other ligation:
Lactoferrina; pUPD2
PhiC31; α1
1 µl Part1 Lactoferrin
1 µl PhiC31
1 µl Part2 Lactoferrin
1 µl 35S
1 µl pUPD2
1 µl T35S
4.6 µl H2O
1 µl α1
3.6 µl H2O
Transform into E. coli the ligations.
25 August 2015
Refresh PhiC31 (BB) and Gal4:AsLOVpep:ePDZ:ren:OpUAS:luc to make glycerinates.
Transfrorm into Agrobacteriumthe construction:
Gal4:AsLOVpep:ePDZ:ren:OpUAS:luc; Ω1
Keep trying protoplasts, this time we see them in the microscope without coverslip. We can see them alive, maybe this was the error. Also we decided to leave the digestion enzymes overnight because after only 3 hours the amount of protoplasts was very little.
EXPERIMENT 14.
Take leafs agroinflitrated with the drugs
EXPERIMENT 15.
Do discs of the agroinfiltrated lefs with the blue toggle (LacI:AsLOVpep:ePDZ:ren:OpLacI:luc) and the control Pnos.
FOTO DE LA PLACA
Make liquid cultures of yesterday transformations:
Lactoferrina; pUPD2 (C1-C3)
PhiC31; α1 (C1-C3)
26 August 2015
Minipreps of:
Lactoferrina; pUPD2 (C1-C3)
PhiC31; α1 (C1-C3)
PhiC31; pUPD2 (BB)
Gal4:AsLOVpep:ePDZ:ren:OpUAS:luc
Digestions of the minnipeps:
Lactoferrina; pUPD2
NotI
2046, 2142
PhiC31; α1
EcoRI
6345, 3127
PhiC31; pUPD2 (BB)
NotI
2046, 1899
Gal4:AsLOVpep:ePDZ:ren:OpUAS:luc
BamHI
6674, 11191
Gel:
PhiC31; α1 C1
PhiC31; α1 C2
PhiC31; α1 C3
PhiC31; α1 C4
Lactoferrina C1
ok
no
ok
ok
ok
Lactoferrina C2
Lactoferrina C3
Lactoferrina C4
PhiC31; pUPD2
Gal4:AsLOVpep:ePDZ:
ren:OpUAS:luc
ok
ok
ok
ok
ok
Do glycerinates of:
Gal4:AsLOVpep:ePDZ:ren:OpUAS:luc; Ω1
PhiC31; pUPD2 (BB)
Ligations:
PhiC31+RepPhiC31:GFP; Ω1
35S+Lactoferrin+T35S; α1
1µl PhiC31
1µl 35S
1µl RepPhiC31:GFP
1µl T35S
1µl Ω1
1µl Lactoferrin; pUPD2
4.6µl H2O
1µl α1
3.6µl H2O
Take samples of the glycerinates:
35S:GFP:T35S (this is necessary to transform into protoplasts and see if the expressed the constructions that we introduce to them)
Alpha1
Omega1
Refresh the Agrobacteriumculture to make agroinfiltration tomorrow of:
Gal4:KDronpa:NDronpa:ren:OpUAS:luc
27 August 2015
Make liquid culture of the Agrobacteriumcolony of Gal4:AsLOVpep:ePDZ:ren:OPUAS:luc; Ω1.
Minipreps of:
Gal4:KDronpa:NDronpa:ren:OpUAS:luc
35S:GFP:T35S
Alpha1
Omega1
Digestions:
Gal4:KDronpa:NDronpa:ren:OpUAS:luc
BamHI
6674, 11191
35S:GFP:T35S
EcoRI
2580, 2274
Alpha1
Nothing
6953
Omega1
Nothing
7295
Gel:
Alpha1
Omega1
35S:GFP:T35S
Gal4:KDronpa:NDronpa:
ren:OpUAS:luc
ok
ok
ok
Ok
Transform into E. coli yesterday ligations:
PhiC31+RepPhiC31:GFP; Ω1
35S+Lactoferrin+T35S; α1
Take samples of the glycerinates and make liquid culture:
Signal peptide (GB0393), in E. coli
Make a glycerinates of Gal4:KDronpa:NDronpa:ren:OpUAS:luc; Ω1
EXPERIMENT 15.
Take samples at 48h and store them at -80ºC in the box “The rebellion of AsLOV”.
PROTOPLASTS 4.
Do protoplasts to transform them tomorrow.
28 August 2015
Miniprep of: 35S:GFP:Tnos.
Digestion of the miniprep:
35S:GFP:Tnos
NotI
2981, 140
Gel:
Correct.
Ligation:
Lactoferrina; α1
1µl Lactoferrina
1µl 35S
1µl T35S
1µl GB393
2.6µl H2O
Measure the OD of GFP: 157.5. It the right curve.
Make liquid culture of the only white colony of PhiC31+RepPhiC31:GFP; Ω1
PROTOPLASTS 4.
Finish the preparation of protoplasts and transform them with 40 µl of 35S:GFP:Tnos at a concentration of 157.5ng/ml.
After the transformation of the protoplast we see them at the microscope and there were no protoplasts.
Take samples of glycerinates and make liquid cultures of:
OpLexA:mini35S (GB0733)
OpLacI:mini35S (GB534)
OpUAS:mini35S (GB0672)
DsRed (GB0672)
Transform and make petri dish culture of Lactoferrin; α1 ligation.
29 August 2015
Miniprep of the glycerinates’ liquid cultures:
OpLexA:mini35S (GB0733)
OpLacI:mini35S (GB534)
OpUAS:mini35S (GB0672)
DsRed (GB0672)
35S:GFP:Tnos; α1
Refresh the culture of Gal4:KDronpa:NDronpa:ren:OpUAS:luc; Ω1 due to that in the last transformation of protoplasts we run out all the DNA.
Refresh and make liquid culture of
Dsred (GB672) in case the DNA concentration of the miniprep was low.
Staffer fragment in Ω2.
Digestion of the minipreps:
OpLexA:mini35S (GB0733); pUPD2
NotI
2981, 477
OpLacI:mini35S (GB534); pUPD2
NotI
2981, 878
OpUAS:mini35S (GB0672); pUPD2
NotI
2981, 477
DsRed (GB0672); α1
EcoRI
2580, 2274
35S:GFP:Tnos; α1
EcoRI
2580, 2274
Do the gel:
OpUAS:mini35S (GB0672); pUPD2
OpLexA:mini35S (GB0733); pUPD2
OpLacI:mini35S (GB534); pUPD2
OpLacI:mini35S (GB534); pUPD2
35S:GFP:Tnos; α1
ok
ok
ok
ok
ok
Make liquid culture of Lactoferrin; α1 (C1-C4)
30 August 2015
Miniprep of the liquid cultures, lactoferrin C1 and C4 did not grow. We do 5 minipreps.
Digestion of the minipreps:
Gal4:KDronpa:NDronpa:ren:OpUAS:luc; Ω1
BamHI
6674, 11191
Dsred (GB672); α 1
EcoRI
Staffer fragment in Ω2
EcoRV
Lactoferrin; α1
EcoRI
6345, 3436
Gel(we repeat it because yesterday gel was very bad):
OpLacI:mini35S (GB534); pUPD2
OpLexA:mini35S (GB0733); pUPD2
OpUAS:mini35S (GB0672); pUPD2
DsRed (GB0672); pUPD2
Mini35S:Dsred; α1
ok
ok
ok
ok
Ok
Lactoferrin; α1 (C2)
Lactoferrin; α1 (C3)
Gal4:KDronpa:NDronpa:ren:OpUAS:luc; Ω1
Staffer fragment in Ω2
no
no
ok
ok
Ligations: (we make this because the pieze Operator+mini35S can not be ligated with the other constructions in Benchlig so we will join them all in this new ligations).
OpLacI:mini35S+Dsred+T35S; α2
OpLexA:mini35S+DsRed+T35S; α2
OpUAS:mini35S+DsRed+T35S; α2
1µl OpLacI
1µl OpLexA
1µl OpUAS
1µl Dsred
1µl Dsred
1µl Dsred
1µl T35S
1µl T35S
1µl T35S
1µl mini35S
1µl mini35S
1µl mini35S
1µl α2
1µl α2
1µl α2
3.6 µl H2O
3.6 µl H2O
3.6 µl H2O
Gal4:KDronpa:NDronpa+SF(Ω2); α1
LexA:KDronpa:NDronpa+SF(Ω2); α1
LacI:KDronpa:NDronpa+SF(Ω2); α1
1µl Gal4:KDronpa:NDronpa
1µl LexA:KDronpa:NDronpa
1µl LacI:KDronpa:NDronpa
1µl SF(Ω2)
1µl SF(Ω2)
1µl SF(Ω2)
1µl α1
1µl α1
1µl α1
4.6µl H2O
4.6µl H2O
4.6µl H2O
Lactoferrin; α1
1µl Lactoferrin
1µl 35S
1µl T35S
1µl α1
2.6 µl H2O
31 August 2015
Transform the ligations into E. coli adding IPTG and X-Gal.
Ligation:
PhiC31+RepPhiC31:GFP; Ω1
1µl PhiC31 (BB); α1
1µl RepPhiC31:GFP; α2
1µl Ω1
4.6µl H2O
The collaborations part of the Norwich iGEM team have arrived:
This are the Mofippers (cloranfenicol resistance):
0408
Promoters
224 ng/µl
7 µl
0409
CDs
77 ng/µl
3 µl
0410
terminator
159 ng/µl
10 µl
0411
TUs
158 ng/µl
10 µl
Añadir una foto de las placas en la wiki y enlace de la wiki de los the norwich
This are the constructions (carbenicillin resistance) that we are going to prove in our plant.
584-MAA with YFP
596-RV3034c with YFP
598-RV3037c with YFP
600-RV3030 with YFP
Transform the Moflippers into E. coli to have later more DNA to make the ligations and be able to send the parts with the necessary characteristics to the BioBricks requirements.
Transform the functional constructions into Agrobacteriumand make petri dishes cultures.
Refresh liquid cultures of Agrobacteriumto make an experiment with red toggle:
LexA:PIF6:PhyB
Gal4:PIF6:PhyB
LacI:PIF6:PhyB
OpLexA:ren:luc
OpLacI:ren:luc
OpUAS:ren:luc
Pnos
1 September 2015
Make liquid culture of the Moflippers, 2 colonies for each culture.
Make liquid culture of yesterday transformations:
OpLacI:mini35S+Dsred+T35S; α2
OpLexA:mini35S+DsRed+T35S; α2
OpUAS:mini35S+DsRed+T35S; α2
Gal4:KDronpa:NDronpa+SF(Ω2); α1
LexA:KDronpa:NDronpa+SF(Ω2); α1
LacI:KDronpa:NDronpa+SF(Ω2); α1
Lactoferrin; α1
Do the ligations in Moflippers with the miniprep that had arrived. This constructions are the ones that go with our plasmid α.
35S:BxbI:T35S
35S:PhiC31:T35S
PsinATG:RepBxbI:GFP:T35S
PsinATG:RepPhiC31:GFP:T35S
1µl 35S
1µl 35S
1µl PsinATG
1µl PsinATG
1µl BxbI
1µl PhiC31
1µl RepBxbI
1µl RepPhiC31
1µl T35S
1µl T35S
1µl GFP
1µl GFP
1µl Mo411
1µl Mo411
1µl T35S
1µl T35S
4.6 µl H2O
4.6 µl H2O
1µl Mo411
1µl Mo411
4.6 µl H2O
4.6 µl H2O
35S:Lactoferrin:T35S
35S:LTB:T35S
Gal4:KDronpa
LacI:KDronpa
1µl 35S
1µl 35S
1µl 35S
1µl 35S
1µl Lactoferrin
1µl LTB
1µl Gal4
1µl LacI
1µl T35S
1µl T35S
1µl T35S
1µl T35S
1µl Mo411
1µl Mo411
1µl KDronpa
1µl KDronpa
4.6 µl H2O
4.6 µl H2O
1µl Mo411
1µl Mo411
4.6 µl H2O
4.6 µl H2O
LexA:KDronpa
NDronpa:VP16
Gal4:AsLOVpep
LacI:AsLOVpep
1µl 35S
1µl 35S
1µl 35S
1µl 35S
1µl LexA
1µl Gal4
1µl Gal4
1µl LacI
1µl T35S
1µl T35S
1µl T35S
1µl T35S
1µl KDronpa
1µl KDronpa
1µl AsLOVpep
1µl AsLOVpep
1µl Mo411
1µl Mo411
1µl Mo411
1µl Mo411
4.6 µl H2O
4.6 µl H2O
4.6 µl H2O
4.6 µl H2O
LexA:AsLOVpep
ePDZ:VP16
1µl 35S
1µl 35S
1µl LexA
1µl ePDZ
1µl T35S
1µl T35S
1µl AsLOVpep
1µl VP16
1µl Mo411
1µl Mo411
4.6 µl H2O
4.6 µl H2O
2 September 2015
Do the minipreps of the liquid cultures:
OpLacI:mini35S+Dsred+T35S; α2
OpLexA:mini35S+DsRed+T35S; α2
OpUAS:mini35S+DsRed+T35S; α2
Gal4:KDronpa:NDronpa+SF(Ω2); α1
LexA:KDronpa:NDronpa+SF(Ω2); α1
LacI:KDronpa:NDronpa+SF(Ω2); α1
Lactoferrin; α1
Mo0408
Mo0409
Mo0410
Mo0411
Gal4:KNDronpa:VP16:SF; Mo
LacI:KNDronpa:VP16:SF; Mo
LexA:KNDronpa:VP16:SF; Mo
1µl Gal4:KNDronpa:VP16
1µl LacI:KNDronpa:VP16
1µl LexA:KNDronpa:VP16
1µl SF
1µl SF
1µl SF
1µl Mo
1µl Mo
1µl Mo
4.6 µl H2O
4.6 µl H2O
4.6 µl H2O
Gal4:AsLOVpep:ePDZ:VP16:SF; Mo
LacI:AsLOVpep:ePDZ:VP16:SF; Mo
LexA:AsLOVpep:ePDZ:VP16:SF; Mo
1µl Gal4:AsLOVpep:ePDZ:VP16
1µl LacI:AsLOVpep:ePDZ:VP16
1µl LexA:AsLOVpep:ePDZ:VP16
1µl SF
1µl SF
1µl SF
1µl Mo
1µl Mo
1µl Mo
4.6 µl H2O
4.6 µl H2O
4.6 µl H2O
Gal4:PIF6:PhyB:VP16:SF; Mo
LacI:PIF6:PhyB:VP16:SF; Mo
LexA:PIF6:PhyB:VP16:SF; Mo
1µl Gal4:PIF6:PhyB:VP16
1µl LacI:PIF6:PhyB:VP16
1µl LexA:PIF6:PhyB:VP16
1µl SF
1µl SF
1µl SF
1µl Mo
1µl Mo
1µl Mo
4.6 µl H2O
4.6 µl H2O
4.6 µl H2O
35S:BxbI:RepBxbI:GFP:SF
35S:PhiC31+RepPhiC31:GFP
1µl 35S:BxbI:RepBxbI:GFP
1µl 35S:PhiC31
1µl SF
1µl RepPhiC31:GFP
1µl Mo
1µl Ω1
4.6 µl H2O
4.6 µl H2O
Make liquid culture of the Norwich constructions in Agro:
584-MAA with YFP
596-RV3034c with YFP
598-RV3037c with YFP
600-RV3030 with YFP
Transform and make petri dish culture of yesterday ligations that have Moflipper411, the α constructions.
35S:BxbI:T35S
35S:PhiC31:T35S
PsinATG:RepBxbI:GFP:T35S
PsinATG:RepPhiC31:GFP:T35S
35S:Lactoferrin:T35S
35S:LTB:T35S
Gal4:KDronpa
LacI:KDronpa
LexA:KDronpa
NDronpa:VP16
Gal4:AsLOVpep
LacI:AsLOVpep
LexA:AsLOVpep
ePDZ:VP16
3 September 2015
Transform the 11 ligations  into E. coli and make petri dish culture.
Gal4:KNDronpa:VP16:SF; Mo
LacI:KNDronpa:VP16:SF; Mo
LexA:KNDronpa:VP16:SF; Mo
Gal4:AsLOVpep:ePDZ:VP16:SF; Mo
LacI:AsLOVpep:ePDZ:VP16:SF; Mo
LexA:AsLOVpep:ePDZ:VP16:SF; Mo
Gal4:PIF6:PhyB:VP16:SF; Mo
LacI:PIF6:PhyB:VP16:SF; Mo
LexA:PIF6:PhyB:VP16:SF; Mo
35S:BxbI:RepBxbI:GFP:SF
35S:PhiC31+RepPhiC31:GFP; Ω1
Make liquid culture of the 12 petri dishes cultures with the Moflippers construction, 2 colonies for each construction.
Put the protoplasts in the determine conditions to run a new experiment.
4 September 2015
Make liquid culture of yesterday cultures.
Minipreps of yesterday liquid cultures:
Digestion:
35S:BxbI:T35S
EcoRI+PstI
2959
35S:PhiC31:T35S
EcoRI+PstI
3105
PsinATG:RepBxbI:GFP:T35S
EcoRI+PstI
2388
PsinATG:RepPhiC31:GFP:T35S
EcoRI+PstI
2669
35S:Lactoferrin:T35S
EcoRI+PstI
3141
35S:LTB:T35S
EcoRI+PstI
1662
Gal4:KDronpa
EcoRI+PstI
1966
LacI:KDronpa
EcoRI+PstI
3006
LexA:KDronpa
EcoRI+PstI
2274
NDronpa:VP16
EcoRI+PstI
2403
Gal4:AsLOVpep
EcoRI+PstI
2235
LacI:AsLOVpep
EcoRI+PstI
2721
LexA:AsLOVpep
EcoRI+PstI
1989
ePDZ:VP16
EcoRI+PstI
2292
The band write in the digestion table only show the approximately legth of the DNA construction. We can not make a proper digestion in Benchling because we do not have the sequence of the Moflippers.
This digestions are special ones due to that we have to use BioBricks enzimes because the plasmid is a Moflipper. So the mix is:
1µl DNA
1µl Buffer O
0.5µl EcoRI
0.5µl PstI
7 µl H2O
Gel:
LexA:KNDronpa (C1)
LexA:KNDronpa (C2)
Gal4:KNDronpa (C1)
Gal4:KNDronpa (C2)
LacI:KNDronpa (C1)
no
no
no
no
no
LacI:KNDronpa (C2)
RepBxbI:GFP (C1)
RepBxbI:GFP (C2)
Gal4:AsLOV (C1)
Gal4:AsLOV (C2)
no
no
no
No
no
LacI:AsLOV (C1)
LacI:AsLOV (C2)
LexA:AsLOV (C1)
LexA:AsLOV (C2)
NDronpa:VP16 (C1)
no
no
no
no
no
NDronpa:VP16 (C2)
LTB (C1)
LTB (C2)
ePDZ:VP16 (C1)
ePDZ:VP16 (C2)
no
no
no
no
no
Lactoferrin (C1)
Lactoferrin (C2)
BxbI (C1)
BxbI (C2)
RepPhiC31 (C1)
no
no
no
no
no
RepPhiC31 (C2)
PhiC31 (C1)
PhiC31 (C2)
LexA:AsLOV (C1)
LexA:AsLOV (C2)
no
no
no
no
no
We found a problem because the length of the construction is very similar to the Moflipper length, so we can not distinguish very well the bands. Moreover the total weight do not match with the expected for the plasmid.
5 September 2015
Prepare to agroinfiltrate the Norwich constructions:
584-MAA with YFP 0.27 1.11ml
596-RV3034c with YFP 0.3 1ml
598-RV3037c with YFP 0.28 1.07
600-RV3030 with YFP 0.25 1.2
We infiltrate 1 plant per construction, 2 leafs per plant.
Miniprep of the last moflippers constructions, the supposed Ω ones.
Digestion:
Gal4:KNDronpa:VP16:SF; Mo
EcoRI+PstI
4804
LacI:KNDronpa:VP16:SF; Mo
EcoRI+PstI
5580
LexA:KNDronpa:VP16:SF; Mo
EcoRI+PstI
4843
Gal4:AsLOVpep:ePDZ:VP16:SF; Mo
EcoRI+PstI
-
LacI:AsLOVpep:ePDZ:VP16:SF; Mo
EcoRI+PstI
-
LexA:AsLOVpep:ePDZ:VP16:SF; Mo
EcoRI+PstI
-
Gal4:PIF6:PhyB:VP16:SF; Mo
EcoRI+PstI
-
LacI:PIF6:PhyB:VP16:SF; Mo
EcoRI+PstI
-
LexA:PIF6:PhyB:VP16:SF; Mo
EcoRI+PstI
-
35S:BxbI:RepBxbI:GFP:SF
EcoRI+PstI
2654
35S:PhiC31+RepPhiC31:GFP; Ω1
BamHI
1860, 3550, 6674, 86, 30, 10
We realized that AsLOVpep and PhyB have recognition sites for BioBricks enzyes so we can not sent them to the iGEM.
Gel:
LacI:KNDronpa (C1)
LacI:KNDronpa (C2)
Gal4:KNDronpa (C1)
Gal4:KNDronpa (C2)
LexA:KNDronpa (C1)
ok
ok
ok
ok
Ok
Gal4:AsLOVpep:ePDZ (C1)
Gal4:AsLOVpep:ePDZ (C2)
LexA:AsLOVpep:ePDZ (C2)
Gal4:PIF6:PhyB (C1)
Gal4:PIF6:PhyB (C2)
No
no
no
no
no
LacI:PIF6:PhyB (C1)
LacI:PIF6:PhyB (C2)
LexA:PIF6:PhyB (C1)
LexA:PIF6:PhyB (C2)
BxbI:RepBxbI:GFP (C1)
no
no
no
no
Ok
BxbI:RepBxbI:GFP (C2)
PhiC31+RepPhiC31:GFP (C1)
PhiC31+RepPhiC31:GFP (C2)
ok
ok
ok
We keep and send to the iGEM parts:
Gal4:KNDronpa:VP16:SF; Mo
LacI:KNDronpa:VP16:SF; Mo
LexA:KNDronpa:VP16:SF; Mo
35S:BxbI:RepBxbI:GFP:SF; Mo
35S:PhiC31:RepPhiC31:GFP+SF
1µl PhiC31:RepPhiC31:GFP
1µl SF (466)
1µl Mo411
4.6 µl H2O
6 September 2015
Transform in AgrobacteriumPhiC31:RepPhiC31:GFP; Ω1
Transnsform into E. coli 35S:PhiC31:RepPhiC31:GFP+SF; Mo.
Make liquid culture again of the colonies that went wrong last time.
Prepare to agroinfilrate:
OpLacI:luc:ren
0.19
200 µl/1.9ml
OpUAS:luc:ren
0.18
211 µl/1.9ml
OpLexA:luc:ren
0.17
224 µl/1.9ml
LexA:AsLOVpep:ePDZ:luc:ren
0.18
211 µl/1.9ml
LacI:AsLOVpep:ePDZ:luc:ren
0.15
253 µl/1.9ml
Gal4:AsLOVpep:ePDZ:luc:ren
0.21
181 µl/1.9ml
LexA:KNDronpa:ren:luc
0.04
950 µl/1.9ml
Pnos
0.77
494 µl/1.9ml
PROTOPLASTS 5.
There were infiltrated:
OpLacI:luc:ren
OpUAS:luc:ren
OpLexA:luc:ren
LexA:AsLOVpep:ePDZ:luc:ren
LacI:AsLOVpep:ePDZ:luc:ren
Gal4:AsLOVpep:ePDZ:luc:ren
LexA:KNDronpa:ren:luc
Pnos
Mesurement of concentrations:
Gal4:KNDronpa:VP16:OpUAS:luc:ren: 327.6 ng/µl
LexA:KNDronpa:VP16:OpLexA:luc:ren: 300.5 ng/µl
Gal4:KNDronpa:VP16:SF; Mo: 183.25 ng/µl
LacI:KNDronpa:VP16:SF; Mo: 221.9 ng/µl
LexA:KNDronpa:VP16:SF; Mo: 266.7 ng/µl
35S:BxbI:RepBxbI:GFP:SF; Mo: 187.5 ng/µl
Prepare the samples to sent to igem. 3µl of each constructions.
7 September 2015
Miniprep of the liquid cultures.
Digestion and gel, that goes wrong again.
Do the ligations again with some new things to optimizied it. So we will boil first the plasmid for 10 minutes adding DTT because BsaI cuts better with an unrolled plasmid. Then we will add ATP into the mix to improve the efficiency of the enzyme. General mix is:
DNA construction
1µl DNA1
1µl DNA2
1µl DNA3
0.5µl Mo411
1µl DTT
1µl ATP
1µl T4ligase
1µl BsaI
1.2µl BSA 10x
1.2µl buffer ligase
2.4µl H20
DNA constructions that were ligated with this technique:
35S:BxbI:T35S
35S:PhiC31:T35S
PsinATG:RepBxbI:GFP:T35S
PsinATG:RepPhiC31:GFP:T35S
35S:Lactoferrin:T35S
35S:LTB:T35S
Gal4:KDronpa
LacI:KDronpa
LexA:KDronpa
NDronpa:VP16
Gal4:AsLOVpep
LacI:AsLOVpep
LexA:AsLOVpep
ePDZ:VP16
Put the agroinfiltrated leafs to digest the membrane, let them overnight. We did 9 preparation in total.
8 September 2015
PROTOPLASTS 5.
Obtained the protoplasts and put into the corresponding plate to make the experiment.
10 September 2015
The results of the luciferase essay are not going well, we thougth that this could be caused because the time spent between the agroinfiltration and the take of samples is too long. So we decided to make a isolated experiment only with Pnos, to see its activity.
We agroinfilrate 2 plants..
Go to Protocols
Scroll
