Team:Valencia UPV/Elements
Valencia UPV iGEM 2015
iGEM 2015 | Valencia UPV
Home
Project
Overview
Circuit
Components
Modeling
Results
Notebook
Modeling
Overview
Deterministic model
Simulations
Light control
Conclusions
Policy and Practicies
Overview
SynBiocraft
Safety
Collaborations
Magic Lamp
Overview
Device: Description
Device: Electronics
How To
Team
Members
Attributions
Acknowledgments
Parts
Achievements
Medals
Achievements
Valencia UPV Team
Be patient, we are under construction
Abstract
Video
5 June 2015
We had 2 cultures from the last day, corresponding to other 2 colonies of ligation.
Agrobacterium culture of promoter less: Luciferase + Renilla
Minipreps
Digestion with BamHI and EcoRV
Agarose gel 1%
How to ask and make primers?
Select the sequence to amplify and save in FASTA format.
gbCloning, go to Tools-Domesticator-1º Category
Add FASTA and select parts.
On the protocol we have the primers
The oligos they give us:
4 first nucleotides: so the enzyme can recognize without problems
6 following bingind sites.
1 extra nucleotide.
4 overhangs.
Meeting with Daniel Ramón (Biopolis).
Ligation with part 2 and 24 of task sheet.
PIF6 + PhyB; ?1
Etr8 CMV_Bxb1_T35S
1µL 892 (PIF α1)
1µL 1097 (Etr8 CMV) Pupd2
1µL 88E (Phy α2)
1µL Bxb1 (PuPD)
1µL ?1
1µL Tnos PuPD
1.2µL Buffer ligase
1µL α1
1µL Bsmb1
5.8µL H2O
6.8µL H2O
If we make a digestion of 160 (35S:Renilla:tNOS-35S:P19:tNOS) with EcoRV, we obtain: 2475, 381, 4601 pb.
If we make a digestion of 896 (Luc:PIF6:PhyB)with EcoRV, we obtain: 11608, 3942 pb.
June 2015
Transform to E.coli from PIF+Phy and Bxb1
1.5µL of ligation
Cuvette on ice
Competent cells + 1.5µL of ligation
Pulse (electroporator) at 1500V
Add 300µL shock medium and put Eppendorf 1h at 37ºC
Culture on petri dishes the ligations.
Digest of 160, 289 and the two ligations, PIF+phy and Etr8+BxbI.
Agarose gel.
7 June 2015
We’ve got white colonies! (from PIF+Phy and Bxb1)
Pick two colonies from each construction.
4 tubes
3.5µL LB each tube
2) 2 tubes + 3.5µL Kanamycin (K)
8 June 2015
Minipreps of the 4 liquid cultures and digestion to see the band patterns.
Digestion:
Etr8(CMV):Bxb1:Tnos; ?1
EcoRI
6345, 238
EPIF6 + PhyB-PV16; ?1
BamHI
6686, 1439, 2685, 2237
Agarose gel was made:
Bxb1 (C1)
Bxb1 (C2)
EPIF6 + PhyB-PV16 (C1)
EPIF6 + PhyB-PV16 (C2)
ok
ok
Repeat digestion (errors).
We don’t have sure the toggle, so we decide to repeat the digestion with other enzyme tomorrow, noticing that the colony 2 has better bands pattern.
Optimized ligation of PIF-Phy-Lac-Renilla-P19
1 µL vector
0.8 µL dilution ½ 160
1.7 µL big part
1.2 µL BSA
1.2 µL buffer
1 µL BsbmI
1 µL ligase
4.15 µL H2O
Ratio 1:2 vector insert
As Bxb1 was good at the digestion we put 1 µL of LB and 1 µL of Kanamicyne on the tube where it had grown and store at 37ºC to glycerinate later.
We design primers to binding domain (BD) and PIF.
Problem: domesticator is introduced in an old pUPD. The new one has different bases.
Change manually the pUPD bases in the program (Benchling).
9 June 2015
Digestion of the ligation of yesterday containing: EPIF6-PhyB-VP16 (C1 y C2)
EPIF6-PhyB-VP16
PvuII (green buffer)
3663, 9472pb
Agarose gel 1%:
EPIF6-PhyB-VP16 (C1)
EPIF6-PhyB-VP16 (C2)
EPIF6-PhyB-VP16 (C3)
no
no
no
We see three bands: 7000, 4000, 1900pb
Transform optimized ligation (yesterday 8/6)
3.5 µL ligation product (EPIF6-PhyB-VP16 + Luciferase + Renille + P19)
40 µL electrocompetent cells.
Pulse of 1500V
Store 1h at 37ºC
Plate culture at agar with Spectinomycin: 50 µL of the transformation.
10 June 2015
Check the primers and order LexA, Gal4, PIF6, LacI, Dronpa.
Check linker VP16 (88E) and make a primer for it.
Take out glycerinate of Ω2.
Alfredo’s part is not working.
Pick colonies of transformation and make liquid culture of E:PIF6:PhyB:VP16:luc:ren (C1-C3).
Minipreps of liquid culture (PIF + Phy), colonies C3, C4, C5, C6
Digestion:
PIF + Phy:VP16
PvuII (buffer green 10x)
3663, 9472
PIF + Phy:VP16
BamHI
1939, 2685, 2337, 6674
Agarose gel 1% (10 samples, 8 + 2 molecular markers)
PIF + Phy (PvuII) C3 PIF + Phy (PvuII) C4 PIF + Phy (PvuII) C5 PIF + Phy (PvuII) C6
no ok no No
PIF + Phy (BamHI) C3 PIF + Phy (BamHI) C4 PIF + Phy (BamHI) C5 PIF + Phy (BamHI) C6
no ok no No
Transformation of Agrobacterium (C58) of the 896 construction (EPIF6-PhyB-VP16 + luciferase). We are not going to have the positive control and we won’t be able to quantify (we don’t have Renilla + P19).
11 June 2015
Minipreps of the culture:
Digestion:
E:PIF6:PhyB:VP16:luc:ren
BamHI
4209, 3756, 6100, 6674
EcoRV
3942, 2989, 2475, 381, 10952
Gel:
PIF6:PhyB:VP16:luc:
ren C1 (BamHI)
PIF6:PhyB:VP16:luc:
ren C3 (BamHI)
PIF6:PhyB:VP16:luc:ren
C1 (EcoRV)
PIF6:PhyB:VP16:luc:ren
C3 (EcoRV)
??
Transformation in Agrobacterium of Renilla (160) due to that we could not join this with PIF:phyB and so we will do a cotransfection of both plasmids. Make petri dish culture with kanamicyn and rifampicin.
12 June 2015
The petri dish with PIF:phy:luc was taken out the 37ºC room and put into the fridge to pick  colonies tomorrow.
13 June 2015
Pick colonies to make liquid culture:
Renilla in agrobacterium: just one colony, it was made liquid culture but check carefully the gel.
It was noticed that the piece 160, renilla, needs a pSub plasmid to replicate itself so we will transform 160 into a agrobacterium with this plasmid (C58 pSub).
Ligation:
BxbI; alfa1+phyB; alfa2
1µl BxbI
1 µl phyB
1 µl ?2
4.6 µl H2O
Transform renilla (160) into agrobacterium and make
15 June 2015
Repeat the ligation BxbI+35S:E-PIF6:tnos because PIF was ?2
BxbI + 35S:E-PIF6:tnos; ?1
1µl BxbI
1 µl phyB
1 µl ?1
4.6
µl H2O
KDronpa has arrived:
Centrifuge it 2-5sec at max velocity.
Add 50 µl to have a concentration of 20ng/µl
Mix it with the vortex and spin.
Ligation: en la libreta pone que se liga a pUPD2
KDronpa; pUPD2
1 µl KDronpa
1 µl pUPD2
5.6 µl H2O
It was not possible to pick colonies of the Agrobacterium transformed because they did not grow. Maybe the problem is that with tetraciclyn bacterias grow slowly. Wait 1 day more.
Transformation of the ligation, BxbI + 35S:E-PIF6:tnos; ?1, into E.coli.
Make an agar culture in petri dish and let grow 16h at 37ºC.
16 June 2015
Transformation of the ligation, KDronpa, into E.coli.
Pick colonies of BxbI:E-PIF6 and make liquid culture (C1-C3).
Primers had arrived, it has been done the resuspension (dilution 1:10)of all of them.
Primers
Code
Template
Working temperature? ºC
LacI F
1
LacI (858)
69.7
LacI R
2
Gal4 F
3
We did not take out the glicerynate.
63.2
Gal4
4
LexA F
5
LexA (732)
62.7
LexA R
6
PIF:VP16 F
7
PIF6 (288)
60.1
PIFVP16 R
8
NDronpa F1
9
Kdronpa
67.7
NDronpa R1
10
Dronpa F2
11
58.5
NDronpa R2
12
A PCR with all the primers and the fragments was done, the samples were put in order with the temperature.
The templates were in dilution 1:50, exception of KDronpa that was dilution 1:5 and the primers 1:10.
PCR Fusion Taq (50µl)
DNA template (10 µg/µl)
0.5 µl fusion taq
2.5 µl primer F
2.5 µl primer R
2 µl NTPs
31.5 µl H2O
17 June 2015
Pick colonies and make liquid culture of:
KDronpa (C1-C4)
Ligations with the PCR’s products:
Templates PCR: 1+2, 5+6, 7+8PIF, 7+8VP16, 9+10, 11+12.
Template PCR; pUPD2
0.5µl template
1µl pUPD2
6.1µl H2O
Minipreps of liquid cultures:
BxbI:E-PIF6 (C1-C3)
Agarose gel with the PCRs:
Template
1+2
5+6
7+8PIF
7+8VP16
9+10
11+12
Band pattern
1017
284
391
478
464
290
Gel result
ok
ok
ok
ok
No DNA
ok
Transformation in E.coli of the correct ligations:
1+2, 5+6, 7+8PIF, 7+8VP16, 11+12
Put in cloranfenicol petri dishes.
18 June 2015
Minipreps of the liquid cultrures:
KDronpa (C1-C4)
Digestions:
KDronpa EcoRI 2800
Gel:
Kdronpa C1
Kdronpa C2
Kdronpa C3
KdronpaC4
no
no
ok
no
Etr8:BxbI:phyB C1
Etr8:BxbI:phyB C2
Etr8:BxbI:phyB C3
No
no
no
We discovered that the construction with BxbI did not go well because our lab college gives us the wrong piece.
Take glicerynates out:
Gal4; pUPD (731)
?2
PromotersinATG (GB00552)
Renilla (160)(159)(109)
PCR:
NDronpa
2.5 µl (9+10) primer F
2.5 µl (11+12) primer R
2 µl NTPs
0.2 µl Taq
10 µl Buffer
31.5
µl H2O
Ligations:
Etr8:BxbI:T35S; α1
Template PCR; pUPD2
1 µlEtr8
0.5µl template
1 µl BxbI
1µl pUPD2
1 µl T35S
6.1µl H2O
5.8 µl H2O
Templates PCR: 1+2, 5+6, 7+8PIF, 7+8VP16
19 June 2015
We do a PCR with the normal Taq polymerase.
1µl of DNA’s template (9+10, 9+12 and 11+12)
2µl of specific buffer
2µl of NTPs
1µl primer forward
1µl primer reverse
0.5 µl of Taq
12.5 µl H2O
These quantities multiplied by 3.
Minipreps of the yesterday’s glycerinated cultures.
To do the digestions we add in each eppendorf.
Minipreps:
Enzime
Band pattern
159 pDGB1_?2 renilla
EcoRV
2909, 2475,882, 812, 381
Entry vector, ?2
EcoRV
6652, 621
552 pP35s NoATG, pUPD
EcoRI
2997, 1090
160 renilla pDGB1, a2
EcoRV
4601, 2475, 381
731 pUPD pGal4BD (CDS)
EcoRI
2997, 2493
109
GB1_a1 355:renilla:Tnos
EcoRI
2580, 2493
We make an agarose gel with the digestions made before and the PCR of KDronpa.
159
160
?2
552
731
109
9+10
9+12
11+12
ok
ok
ok
ok
ok
ok
no
ok
ok
Transformation of the ligations:
50 µl of electrocompetent cell
1.5 µl of ligation
Put 1 min before the cubettes in ice
Make the transformation (1500V)
Put the transformed cells 1h at 37ºC
We made an stack of Cloranfenicol petri dishes
250ml  LB aga
X-gal (1:500): 500 µl
Iptg (1:1000): 250 µl
Cloranfenicol (1:2000): 125 µl
20 june 2015
We have White colonies of renilla! Also of Etr8 + Bxb1
We have also pUPD colonies but they are so close to the blue ones that we can’t pick anyone.So we make strakes in another plates to have the colonies separated.
We make a liquid culture of Agro of Renilla.
5ml of LB agar
5 µl rifampicine
5 µl of kanamicine
5 µl tetracycline
Incubate for 2 days (48h) at 28ºC.
21 June 2015
Pick colonies of and put into a liquid medium of 3 ml of LB agar, 3 µl of each antibiotic (kanamicine, spectomicine, cloranfenicol):
Plates (17/06/15): PIF (C1, C2) (with cloranfenicol), VP16 (C1, C3) (with cloranfenicol), LacI (C1, C2, C3) (with cloranfenicol)
Plates (19/06/15): Bxb1 (C1, C2, C3) (with kanamicine), VP16 (C4, C5) (with cloranfenicol), LacI (C1, C2) (with C1, C2) (with cloranfenicol), PIF (C1, C2, C3, C4, C5) (with cloranfenicol), LexA (C1, C2) (with cloranfenicol).
We take out two glicerynates of GFP and BFP (of the Alfredo’s box)
22 June 2015
We made minipreps of the liquid culture of the day before:
LacIBD, pUPD (C1-C5)
LexABD, pUPD (C1, C2)
Etr8(CMV):Bxb1 (C1-C3)
PIF6,pUPD (C1-C5)
VP16, pUPD (C1, C4, C5)
Make the digestions of all the minipreps:
LacIBD, pUPD
NotI
2046, 1053
LexABD, pUPD
NotI
2046, 321
Etr8(CMV):Bxb1
NotI
1532, 1290, 5896
PIF6,pUPD
NotI
2046, 407
VP16, pUPD
NotI
2046, 500
Viral system:……….
We received the reported Bxb1!
500ng of sample
Centrifuge at 3000rpm for 5 seconds (spin).
Add 50 µl H2O
Shake it and let at 50ºC for 20min
Make a PCR of Gal4 and NDrompa (9-10), the primers of NDrompa are aliquoted
…..
Make the gel with all the digestions writed before.
Lac1
Lac2
Lac3
Lac4
Lac5
Lex1
Lex2
Bxb1,1
Bxb2, 2
Bxb1,3
Ok
ok
ok
ok
ok
no
no
ok
ok
no
PIF1
PIF2
PIF3
PIF4
PIF5
VP16,1
VP16,4
VP16,5
No
no
-
ok
ok
ok
ok
ok
PCRS
…
…
…
We make ligations of:
Etr8(CMV):BxbI in a1 + PhyB:P16 in a2, ?1
LacIBD in pUPD2 + PIFBDless+promoter+terminator, a1
KDrompa in pUPD2 + LacBD+promoter+terminator, a1
Gal4BD, pUPD2
Reporter of BxbI, pUPD2
Tomorrow we have to take out pUPD of constitutive promoters, terminators and GFP (CDS).
23 June 2015
Transformations in E.Coli of the 5 ligations done yesterday and two more transformations of 5+6(1) and 5+6(2) which are the ligations in pUPD of the 18/06.
We put 50 µl of the transformations in the plates. Put in 37ºC.
Etr8(CMV):Bxb1 in a1 + PhyB:P16 in a2, ?1
LacIBD in pUPD2 + PIFBDless+promoter+terminator, a1
KDrompa in pUPD2 + LacBD+promoter+terminator, a1
Gal4BD, pUPD2
Reporter of Bxb2, pUPD
LexABD (5+6(1)), pUPD
We have taken out of the -80ºC fridge the glycerinate of GFP, pUPD/ampicilina/GB0059.
The liquid culture of Renilla (rifampicina/kanamicia/tetraciclina) doesn’t grow before the two days required. So we decide to put two new colonies in une tube with the three antibiotics and another with rifampicina and kanamicine. Asun say that the tetracycline slow down the growth of Agro.
The 4 liquid cultures of LexA+IPTG/+gal are all blue: throw them.
We ordered again the primer nº10 (NDronpa R1). Changing one codon in 3’ and delete another in 5’.
24 June 2015
Pick colonies of the plates done yesterday and pass them into a liquid media: 3 µl of LB, 3 µl of antibiotics (K, Spect, Clor).
LacIBD+PIF, a1 (C1, C2)
Gal4BD, pUPD2 (C1)
RepBxb1, pUPD2 (C1, C2, C3)
LacIBD+KDonpa, a1 (C1, C2)
Etr8(CMV)+Bxb1+phyB+VP16, ?1 (C1)
LexABD1, pUPD (C1-C4)
LexABD2, pUPD. No colonies.
The viral systems cultures of Agro for the color mosaics are ready before 2 days at 28ºC. We can make the Agroinfiltration.
Buffers of Agroinfiltration:
First we have to prepare and ajust the pH of the buffer MES and the buffer MgCl.
MES (10x), 100nM; ph=5,6 (adding NaOH). Make 1L.
MgCl (100x), 1M. Make 100ml.
Solution to agroinfiltration: 10ml of MES(10x) + 1ml of MgCl (100x) + 100 µl of DMSO+acetosiningona and finally “enrasar” to 100ml.
19.6mg of acetosiningona for 500 µl of DMSO
Ligation:
ETR8(CMV):Bxb1(a1)+phyB+VP16(a2); ?1
Gal4BD(pcr) + pUPD2
1.5 µl etr8:Bxb1
1 µl Gal4 PCR
1.5 µl 88E (phyB+VP16)
1 µl pUPD2
1 µl ?1
1.2 buffer T4 ligase
1.2 µl buffer T4 ligase
1 µl ligase
1 µl ligase
1.2 µl BSA
1.2 µl BSA
1 µl BsmbI
1 µl BsmbI
5,6 µl H2O
3.6
µl H2O
Quantification of DNA:
pUPD GFF (0059): 249 ng/µl
?2: 238 ng/µl
Alfredo’s pUPD2, domesticator: 102 ng/µl
iGEM704: 405 ng/µl
IGEM735: 403 ng/µl
552 AMP 35S noATG: 45 ng/µl
PIF (C5), pUPD2: 119 ng/µl
pD6B3, ?2 (22/06): 158 ng/µl
LacIBD (C1), pUPD (22/06): 129 ng/µl
109k renillaDC: 49 ng/µl
IGEM 534: 13.6 ng/µl
VP16 (C1), pUPD2:102 ng/µl
IGEM 1097: 409 ng/µl
K-donpa (C3), pUPD2 (18/06): 174 ng/µl
IGEM 858: 487 ng/µl
731AMP Gal4 (19/06): 81 ng/µl
IGEM pUPD2 domesticator: 87 ng/µl
PIF+phy8 (c1) (08/06): 108 ng/µl
160 renilla, a2 (19/06): 46 ng/µl
159 renilla, ?2 (19/06): 149 ng/µl
Etr8:Bxb1 (C1)(22/06): 149 ng/µl
IGEM 732: 422 ng/µl
Measurement of the OD:
first we have to
25 June 2015
Minipreps of the liquid culture:
We don’t observed growth in LacIBD+PIF and LaciBD+K-donpa.
Digestion of the minipreps and do the gel:
Gal4BD, pUPD2
NotI
2046, 282
RepBxb1, pUPD2
NotI
2046, 460
Etr8(CMV):Bxb1 + phyB,a1
BamHI
6674, 2237, 2806, 1174
LexABD, pUPD2
NotI
2046, 321
9+10, pUPD2
NotI
464
Gel:
Etr8:Bxb1
Lex1
Lex2
Lex3
Lex4
Rep1
Rep2
Rep3
Gal4 C1
PCR 9+10
no
no
no
no
no
ok
ok
ok
no
ok
We make a PCR of the Fusion Taq pH (proof-reading) to prove that the primer received number 10. This new one works! Amplify the sequence of N-donpa (R1).
Refresh the cultures of Agro (28ºC). We pick 7.5 µl of the previous culture. And put into a new liquid media with Rifampicine and Kanamicine for 2 days and 28ºC.
Ligations:
N-dronpa
Rep GFP
Gal4
LexA
1 µl PCR 9+10
1 µl Rep Bxb1
1 µl PCR 3+4
1 µl PCR 5+6
1 µl PCR11+12
1 µl promoter without ATG
1 µl pUPD2
1 µl pUPD2
1 µl pUPD2
1 µl Tnos
1.2 µl buffer ligase
1.2 µl buffer ligase
1.2 µl buffer ligase
1.2 µl buffer ligase
1.2 µl BSA
1.2 µl BSA
1.2 µl BSA
1.2 µl BSA
1 µl BsmbI
1 µl BsaI
1 µl BsmbI
1 µl BsmbI
1 µl T4 ligase
1 µl T4 ligase
1 µl T4 ligase
1 µl T4 ligase
4,6 µl H2O
3,6 µl H2O
5,6 µl H2O
5,6 µl H2O
1 µl a2
Etr8:Bxb1+phyB
1 µl Etr8:Bxb1
1 µl 88E
1µl ?1
1.2 µl buffer ligase
1.2 µl BSA
1 µl BsmbI
1 µl T4 ligase
3,6 µl H2O
We transform the ligations in E.Coli ant pass them into agar plates with cloranfenicol for all of them except the ligation of Etr8:Bxb1+phy that goes with bhnfnfg
26 June 2015
We do again the two ligations that didn’t gone?
Rep GFP
LacI BD+PIF6
1 µl Rep Bxb1
1 µl LACIBD, pUPD
1 µl promoter without ATG
1 µl PIF6, pUPD
1 µl Tnos
1 µl promoter
1.2 µl buffer ligase
1 µl T35
1.2 µl BSA
1 µl a1
1 µl BsaI
1.2 µl buffer BsaI
1 µl T4 ligase
1.2 µlBSA
2.6 µl H2O
1 µl BsaI
1 µl a2
1 µl ligase
1µl GFP (0059)
2.6 µl H2O
We make a gel of LAcI+PIF6 (C1 and C2). Both of them present the fragment of the vector at 6000 pb but none of them at 2000bp which is the insert one.
LacIBD+PIF; a1
EcoRI
6345, 1997, 641
LacIBD+PIF C1
LacIBD+PIF C2
no
no
Measurement of the ODs of phyB+PIF+luc and renilla+P19.
phyB+PIF+luc: o.35 (1:2)
Ren+P19: 0.34 (1:2)
Final volume= 2
CCo= 0.35*2
CCf= 0.2
Ecuation=> Vo*CCo=Vf*CCf
So we obtain that Vo(LacI+PIF6)=1.429 µl and Vo(ren)= 1.412 µl.
Ligation of:
LacIBD,pUPD + K-donpa, pUPD
1 µl 35S
1 µl LacIBD,pUPD2
1 µl K-dronpa, pUPD
1 µl T35S
1 µl a1
1.2 µl buffer T4 ligase
1 µl BsaI
1 µl T4 ligase
2.6 µl H2O
We make the agorinfiltration of (…..) to start checking if the plant react to the different ..
27 June 2015
Transformation in E.coli of LacIBD+K-Dronpa, a1
We Put into plates LexABD and Etr8(CMV):Bxb1:GFP again and also LAcIBD+K-Dronpa.
We make liquid culture of:
RepBxb1:GFP (C1-C4)
LacIBD+PIF (C1-C5)
N-Dronpa (C1-C4)
Gal4BD (C1-C5)
LexA
28 June 2015
We do the minipreps of the liquid cultures that have grown.
RepBxb1:GFP (C1 and C2)
LacIBD+PIF (C1-C4)
N-Dronpa (C1-C4)
Gal4BD (C1-C5)
LexA: didn’t grow
Do the digestions of the minipreps:
LacIBD+PIF, a1
EcoRI
6345, 1997, 641
RepBxb1:GFP, ?2
HindIII
6345, 2683
Gal4BD; pUPD2
NotI
2681, 644
N-Dronpa; pUPD2
NotI
2046, 744
Make the gel.
RepBxb1:GFP C1
RepBxb1:GFP C2
LacIBD+PIF C1
LacIBD+PIF C2
LacIBD+PIF C3
LacIBD+PIF C4
no
no
no
no
no
No
Gal4BD C1
Gal4BD C2
Gal4BD C3
Gal4BD C4
Gal4BD C5
N-Dronpa C1
ok
ok
ok
ok
ok
ok
N-Dronpa C2
N-Dronpa C3
N-Dronpa C4
no
ok
ok
Take glycerinated:
GB0030: p35S
GB0036: T35S
Make liquid culture of LexABD (C1-C4).
We transform again LacIBD+K-Dronpa and RepBxb1:GFP, adding to the agar plates 100 µl of each transformation.
29 June 2015
Do the minipreps of the 4 colonies of LexABD and both glycerinates, 35S and T35S.
Do the digestion of the 4 colonies of LexA and both glycerinates:
LexABD; pPPD2
NotI
2358, 312
35S; pUPD2
NotI
2981, 1074
T35S; pPUD2
NotI
2981, 304
Make the gel:
LexA C1
LexA C2
LexA C3
LexA C4
P35S
T35S
ok
ok
ok
ok
Ok?
Ok?
Make ligations:
LacIBD+K-Dronpa+promoter+termi; a1
Gal4BD+K-Donpa+prom+ter; a1
LexABD+K-Dronpa+prom+term; a1
1 µl LacI, pUPD2
1 µl Gal4, pUPD2
1 µl Gal4, pUPD2
1 µl K-Dronpa, pUPD2
1 µl K-Dronpa, pUPD2
1 µl K-Dronpa, pUPD2
1 µl 35S (GB0030)
1 µl 35S (GB0030)
1 µl 35S (GB0030)
1 µl T35S (GB0036)
1 µl T35S (GB0036)
1 µl T35S (GB0036)
1.2 µl  buffer ligase
1.2 µl  buffer ligase
1.2 µl  buffer ligase
1.2 µl BSA 10x
1.2 µl BSA 10x
1.2 µl BSA 10x
1 µl BsaI
1 µl BsaI
1 µl BsaI
1 µl ligase T4
1 µl ligase T4
1 µl ligase T4
2.6 µl H2O
2.6 µl H2O
2.6 µl H2O
1 µl a1
1 µl a1
1 µl a1
N-Dronpa+VP16; a2
Gal4BD+PIF6; a1
LacIBD+PIF6; a1
1 µl N-Dronpa, pUPD2
1 µl Gal4BD, pUPD2
1 µl LacIBD, pUPD2
1 µl VP16, pUPD2
1 µl PIF6, pUPD2
1 µl PIF6, pUPD2
1 µl 35S (GB0030)
1 µl 35S (GB0030)
1 µl 35S (GB0030)
1 µl T35S (GB0036)
1 µl T35S (GB0036)
1 µl T35S (GB0036)
1.2 µl  buffer ligase
1.2 µl  buffer ligase
1.2 µl  buffer ligase
1.2 µl BSA 10x
1.2 µl BSA 10x
1.2 µl BSA 10x
1 µl BsaI
1 µl BsaI
1 µl BsaI
1 µl ligase T4
1 µl ligase T4
1 µl ligase T4
2.6 µl H2O
2.6 µl H2O
2.6 µl H2O
1 µl a2
1 µl a1
1 µl a1
LexABD+PIF6; a1
1 µl LexABD, pUPD2
1 µl PIF, pUPD2
1 µl 35S (GB0030)
1 µl T35S (GB0036)
1.2 µl  buffer ligase
1.2 µl BSA 10x
1 µl BsaI
1 µl ligase T4
2.6 µl H2O
1 µl a2
Transform all the ligations into E.Coli.Gal4BD+K-Dronpa and LacIBD+K-Dronpa goes wrong and we have to do it again.
Put into a plate the colonies before let stay them 1h at 37ºC.
Quantification of DNA:
RepBxb1:GFP (C1): 163.8 ng/µl
N-Dronpa, pUPD2 (C4):113.1 ng/µl
N-Dronpa (C3): 83.2 ng/µl
NDonpa (C1): 116.6 ng/µl
Gal4BD (C1): 95.2 ng/µl
Gal4BD (C2): 120.7 ng/µl
RepBxb1:GFP (C2): 170.6 ng/µl
RepBxb1 (C1): 80.6 ng/µl
30 June 2015
Transform Gal4+K-Dronpa and LacI+K-Dronpa
Miniprep of:
RepBxb1+GFP (C1-C3)
RepBxb1+GFP
HindIII
6345, 2683
Gel:
RepBxb1+GFP C1
RepBxb1+GFP C2
RepBxb1+GFP C3
No
no
no
We pick more colonies and make other liquid cultures.
Save the last samples of the leaves of the plants that were under the first experiment, next to the glycerinates in the freezer (-80ºC).
Put in plates the transformations of Gal4+K-Dronpa and LacI+K-Dronpa (50 µl of bacteria in SOC medium).
Make the gel:
Add to the digestions 2 µl of loading buffer (6x) because for 5 µl of digestion we put 1 µl of the buffer.
Make liquid culture of two colonies for each plates of the transformations done yesterday, 10 tubes in total.
Take out a glycerinate 35S:Luciferase:Tnos (GB0227) and do a miniprep.
1 July 2015
Do the minipreps of the 10 liquid cultures.
Do the digestions:
LexABD+K-Dronpa;a1
EcoRI
6345, 2296
N-Donpa+VP16; a2
HindIII
6345, 2427
Gal4BD+PIF6; a1
EcoRI
6345, 1867
LacI+PIF; a1
EcoRI
6345, 2638
LexABD+PIF6; a1
EcoRI
6345, 1906
Do the gel:
Gal4+PIF C1
Gal4+PIF C2
LexA+PIF C1
LexA+PIF C2
LexA+KDronpa C1
ok
ok
ok
ok
Ok
LexA+Kdronpa C2
LacI+PIF C1
LacI+PIF C2
Ndonpa+VP16 C1
Ndronpa+VP16 C2
ok
ok
ok
ok
ok
Prepare liquid culture of:
LexA+PIF; ?1 (C1)
LacI+PIF; ?1 (C1)
LexA+K-Dronpa (C1)
Gal4+PIF; ?1 (C1)
VP16, pUPD2 (C1)
LexABD, pUPD2 (C2)
PIF6, pUPD2 (C5)
LacIBD, pUPD2 (C1)
Pick colonies and make liquid culture of:
Gal4+K-Dronpa (C1 and C2)
LacI+K-Dronpa (C1 and C2)
RepBxb1+GFP (C4-C6)
We sent to sequence:
Code:
210.08-249: pUPD2, KDronpa C3
210.08-250: pUPD2, NDronpa C1
210.08-251: pUPD2, NDronpa C3
210.08-252: pUPD2, NDronpa C4
The solution have: 10µl of miniprep + 5µl (dilution 1:3) of primers.
Data of the luciferase essay with the sample plat RED at 24h (replica2): we obtain a value of 7.4E6.
2 July 2015
Minipreps of:
Gal4+K-Dronpa (C1 and C2)
LacI+K-Dronpa (C1 and C2)
RepBxb1+GFP (C4-C6)
Gal4+K-Dronpa
EcoRI
6345, 3028
LacI+K-Dronpa
EcoRI
6345, 2257
RepBxb1+GFP
HindIII
6300, 2400
Do the gel:
LacI+K-Dronpa C1
LacI+K-Dronpa C2
Gal4+K-Dronpa C1
Gal4+K-Dronpa C2
??
RepBxb1+GFP C4
RepBxb1+GFP C5
RepBxb1+GFP C6
Luciferase essay:
During the whole experiment we lost three samples: T16/FarRed/1; T24/Red/2 and T0/FarRed/1
Results:
Things to keep in mind for the next experiment:
The luminimeter (machine to measure the luminescence) has to be ready before start adding the reactants to the samples because it needs 10min to be ready.
Set the timer (10min) with the first sample of luciferase and add the reactant to the other samples as quick as possible.
We have to let the renilla stay before putting it in the luminimeter the same time as the luciferase, in this case 15min because with the luciferase we didn’t manage well the time. Theoretically we have to wait 10min.
Make ligations:
LexA:Kdonpa+N-Dronpa; ?1
LacI:Kdronpa+N-Dronpa; ?1
Gal4:Kdronpa+N-Dronpa; ?1
1 µl LexA:Kdronpa
1 µl LacI:Kdronpa
1 µl Gal4:Kdronpa
1 µl N-Dronpa
1 µl N-Dronpa
1 µl N-Dronpa
1 µl ?1
1 µl ?1
1 µl ?1
1.2 µl buffer ligase
1.2 µl buffer ligase
1.2 µl buffer ligase
1.2 µl BSA
1.2 µl BSA
1.2 µl BSA
1 µl BsmbI
1 µl BsmbI
1 µl BsmbI
1 µl BsaI
1 µl BsaI
1 µl BsaI
4.6 µl H2O
4.6 µl H2O
4.6 µl H2O
LexA:PIF+PhyB:VP16; ?1
LacI:PIF+PhyB:VP16; ?1
Gal4:PIF+PhyB:VP16; ?1
1 µl LexA:PIF
1 µl LacI:PIF
1 µl Gal4:PIF
1 µl PhyB:VP16 (88E)
1 µl PhyB:VP16
1 µl PhyB:VP16
1 µl ?1
1 µl ?1
1 µl ?1
1.2 µl buffer ligase
1.2 µl buffer ligase
1.2 µl buffer ligase
1.2 µl BSA
1.2 µl BSA
1.2 µl BSA
1 µl BsmbI
1 µl BsmbI
1 µl BsmbI
1 µl BsaI
1 µl BsaI
1 µl BsaI
4.6 µl H2O
4.6 µl H2O
4.6 µl H2O
35S:Bxb1+RepBxb1:GFP; ?1
E-PIF+phyB+luc+ren; ?1
1 µl 35s:Bxb1:T35S (alfredo’s)
0.5 µl 896 (PIF+phy+luc)
1 µl PromsinATG:RepBxb1:GFP
1 µl 160 (renilla)
1 µl ?1
1 µl ?1
1.2 µl buffer ligase
1.2 µl buffer ligase
1.2 µl BSA
1.2 µl BSA
1 µl BsmbI
1 µl BsmbI
1 µl BsaI
1 µl BsaI
4.6 µl H2O
4.6 µl H2O
The samples that we sent to sequence have arrived:
210.08-249: pUPD2, KDronpa C3------ok
210.08-250: pUPD2, NDronpa C1------ok
210.08-251: pUPD2, NDronpa C3------ok
210.08-252: pUPD2, NDronpa C4------ok
The sequences of N-Dronpa have the desired mutation.
Transformation in E.Coli of the 8 ligations.
Put the transformation into plates, put at 37ºC.
Refresh the Agro’s cultures (Renilla and PIF+phy+luc):
Add in 5ml of LB, 5 µl of rifampicine, 5 µl kanamicine, 5 µl of culture.
4 July 2015
Make the 2nd refresh of the culture of Agrobacterium. Put 5ml of LB, 5 µl of rifampicine, 5 µl kanamicine, 5 µl of culture.
Make liquid cultures (4ml of LB and 4 µl of spectomicine) of the 8 colonies of E.Coli.
Red toggle (C1)
Gal4:Kdronpa+N-Dronpa (C1 and C2)
LexA:Kdonpa+N-Dronpa (C1 and C2)
LacI:Kdronpa+N-Dronpa (C1 and C2)
LexA:PIF+PhyB:VP16 (C1 and C2)
35S:Bxb1+RepBxb1:GFP (C1 and C2)
This colonies didn’t grow: LacI:PIF+PhyB:VP16; Gal4:PIF+PhyB:VP16 and E-PIF+phyB+luc+ren. Tomorrow we will repeat the ligations.
5 July 2015
Ligations:
LacI:PIF+PhyB:VP16; ?1
Gal4:PIF+PhyB:VP16; ?1
1 µl LacI:PIF
1 µl Gal4:PIF
1 µl PhyB:VP16
1 µl PhyB:VP16
1 µl ?1
1 µl ?1
1.2 µl buffer ligase
1.2 µl buffer ligase
1.2 µl BSA
1.2 µl BSA
1 µl BsmbI
1 µl BsmbI
1 µl T4 ligase
1 µl T4 ligase
4.6
µl H2O
4.6 µl H2O
Minipreps of the liquid cultures.
Digestion of the minipreps.
LacI:Kdronpa+N-Dronpa
BamHI
6674, 5437
35S:Bxb1+RepBxb1:GFP
BamHI
6674, 3859, 1782
Gal4:Kdronpa+N-Dronpa
BamHI
6674, 4666
LexA:Kdonpa+N-Dronpa
BamHI
6674, 4705
LexA:PIF+PhyB:VP16
BamHI
6674, 3513, 2337
Agarose gel (1%):
LacKN C1
LacIKN C2
Bxb1RepGFP C1
Bxb1RepGFP C2
Gal4KN C1
Gal4KN C2
ok
ok
ok
ok
ok
ok
LexA:KN C1
LexA:KN C2
LexAPIFPhy C1
LexAPIFPhy C2
Red toggle
ok
ok
no
no
no
We have to repeat the digestion of: LexA+PIF:phy+VP16.
Take out the glycerinate 88C (1098): Etr8:luc:Tnos. We will use it like a negative control in the second luciferase essat.
Calculation of the ODs:
Dilution of both samples 1:10.
Renilla:=0.22---182 µl of sample + 1.818 ml MES
PIF+PhyB+Luc=0.26--- 154 µl of sample + 1.646 ml MES
Agroinfiltration of the two samples with renilla and luciferase+PIF+phy in three plants with 4 spots in each leaf and 2 leaf in each plant.
Let the plants 2 days in the darkness till agrobacterium infects the plant. They have to be in the dark because we are trying our red toggle ant it activates with light.
6 July 2015
Transform the negative control into agrobacterium.
Etr8:luc:Tnos
Bxb1:reporterBxb1:GFP
We were doing dry lab preparing the power point to present our project to the rector and biotecs companies.
7 July 2015
Luciferase essay: Copy the protocol.
Add 150 µl of the lisis buffer and 800µl of MiliQ water, dilution 1:5.
We centrifuge both cultures of agrobacterium at 2900rpm for 10min and remove the supernatant.
We prepare the stock of MES (10ml of MES + 1ml MgCl + 100µl of “Acetosiningona” and level with H2O until 100ml.
Resuspend the pellet of bacteria with 5ml of MES and let grow 2h.
Renilla=0.29 (dilution 1:10): 2.9
PhyB+PIF+luc=0.69 (dilution 1:4):2.76
Solution to agroinflitrate:
Renilla: 0.138ml of sample + 1.862ml of MES
PhyB+PIF+luc: 0.145ml of sample + 1.855ml of MES
We make the infiltration of both samples mix together in 3 different plants, 2 leaf per plant and 4 spots per leaf. Explicacion del experiment (tiempo en oscuridad… luces…)
Digestions:
Red toggle
Enzyme?
LexA+PIF+phy
BamHI
3518, 5855, 6674
Gel:
Red toggle
LexA+PIF+phy C1
LexA+PIF+phy C2
No
Ok
no
It has arrived a new construction: AsLOVpep.
Suspended with 50µl of H2O.
Ligations:
AsLOVpep; pUPD2
Red Toggle
LacI:Kdronpa:Ndronpa:VP16+
35S:renilla:Tnos-35S:P19:Tnos(GB159)
Gal4:Kdronpa:Ndronpa:VP16+GB159
1 µl AsLOVpep
1 µl GB846
1 µl LacI:KNdronpa:VP16
1 µl Gal4:KNdronpa
1 µl pUPD2
1 µl GB160
1 µl GB159
1 µl GB159
1.2 µl buffer
1 µl ?1
1 µl a1
1 µl a1
1.2 µl BSA
1.2 µl buffer
1.2 µl buffer
1.2 µl buffer
1 µl BsmbI
1.2 µl BSA
1.2 µl BSA
1.2 µl BSA
1 µl T4 ligase
1 µl BsmbI
1 µl BsmbI
1 µl BsmbI
5.6 µl H2O
1 µl T4 ligase
1 µl T4 ligase
1 µl T4 ligase
4.6 µl H2O
4.6 µl H2O
4.6 µl H2O
LexA:Kdronpa:Ndronpa:VP16+GB159
LexA:PIF:Phy:VP16+ GB159
1 µl LexA:Kdronpa:Ndronpa:VP16
1 µl LexA:PIF:Phy:VP16
1 µl GB159
1 µl GB159
1 µl a1
1 µl a1
1.2 µl buffer
1.2 µl buffer
1.2 µl BSA
1.2 µl BSA
1 µl BsmbI
1 µl BsmbI
1 µl T4 ligase
1 µl T4 ligase
4.6 µl H2O
4.6 µl H2O
8 July 2015
Experiment to study the piece PhyB+PIF.
The plants in red are exposed at the light intensity of the leds (we can’t regulate it) and the plants in far red are 100% with far red light and 0% of white light.
How the machines work: (hace falta?)
The machine with red light has the switch …
The 1st sample its at 8:00am and we spend 1h till the machines were working correctly because we didn’t know exactly how they work.
Then, before 12h (21:00), we take the 2nd samples; obtainin 3 discs of each condition (red an far red).
We transform:
AsLOVpep; pUPD2 in DHSa an let incubate 1h at 37ºC.
The last ligations in E.coli. Put in plates. Red toggle with Spectomicine+IPTG+XGal ann the rest (a1) with kanamicine+IPTG+XGal.
9 July 2015
Pick colonies:
AsLOVpep colonies didn’t grow. Repeat.
Red toggle colonies are all blue. Repeat.
The other 4 colonies had grown. we pick them and male liquid culture.
LacI:Kdronpa:Ndronpa:VP16+renilla (C1-C3)
Gal4:Kdronpa:Ndronpa:VP16+renilla (C1-C3)
LexA:Kdronpa:Ndronpa:VP16+GB159 (C1 and C2)
LexA:PIF:Phy:VP16+ GB159 (C1 and C2)
Repeat the ligations.
AsLOVpep, as before.
Red toggle (PIF+PhyB+luc+ren)
0.5 µl PIF+phy+luc (896)
1 µl renilla (160)
1 µl ?1
1.2 µl buffer
1.2 µl BSA
1 µl BsmbI
1 µl T4 ligase
5.1 µl H2O
We do the luciferase essay:
We didn’t obtain goo results, we can’t observed a significative difference between the on(red samples) and off (far red samples). It seems that the red toggle it has been activated. Graphics and tables
Transform the ligations of AsLOVpepe and red toggle.
Add SOC medium and let incubate 1h at 37ºC. Then we make an spin to the red toggle cells to concentrate them and see if we can obtain a colonies in the plates.
10 July 2015
Minipreps of:
LexABD:PIF:phyB:VP16+Renilla; a1 (C1,C2)
LexA:Kdronpa:Ndronpa+renilla; a1C1, C2)
Gal4:Kdronpa:Ndronpa+renilla; a1 (C1-C3)
LacI:Kdronpa:Ndronpa+renilla; a1 (C1,C2)
Digestions of:
LexABD:PIF:phyB:VP16+Renilla; a1 (C1,C2)
EcoRI
6345, 5487, 4891
LexA:Kdronpa:Ndronpa+renilla; a1C1, C2)
EcoRI
9333, 6345
Gal4:Kdronpa:Ndronpa+renilla; a1 (C1-C3)
EcoRI
6345, 9194
LacI:Kdronpa:Ndronpa+renilla; a1 (C1,C2)
EcoRI
6345, 9965
LacIBD+PIF+phyB; Ω1 (C1)
BamHI
6674, 4245, 2337
Gal4BD+PIF+phyB; Ω 1 (C1)
BamHI
6674, 3474, 2337
Make the gel:
LacIBD+PIF+phyB
Gal4BD+PIF+phyB
LexA:PIF:phyB:VP16+Renilla C1
LexA:PIF:phyB:VP16+Renilla C2
Ok
Ok
ok
ok
LexA:KNdronpa+renilla C1
LexA:KNdronpa+renilla C2
Gal4:KNdronpa+renilla C1
Gal4:KNdronpa+renilla C2
Ok
Ok
ok
Ok
Gal4:KNdronpa+renilla C3
LacI:Kdronpa:Ndronpa+renilla C1
LacI:Kdronpa:Ndronpa+renilla C2
Ok
Ok
ok
We received two constructions:
CDS: phiC31. Resuspended with 100µl.
Reporter phi31. Resuspended with 50 µl
Pick colonies and make liquid culture of:
AsLOVpep; pUPD2 (C1 and C2)
Red toggle (C1 and C2). In both liquid cultures we ad YPTG and Xgal to make sure that the colonies are correct, if not the medium will change to blue color.
Ligations:
phyC31;pUPD2
Reporter phiC31; pUPD2
LacI:PIF:Phy:VP16+ren; a1
1 µl phiC31
1 µl rep phiC31
1 µl LacI:PIF:Phy:VP16
1 pUPD2
1 pUPD2
1 µl renilla (159)
1.2 µl buffer
1.2 µl buffer
1.2 µl buffer
1.2 µl BSA
1.2 µl BSA
1.2 µl BSA
1 µl T4 ligase
1 µl T4 ligase
1 µl T4 ligase
1 µl BsmbI
1 µl BsmbI
1 µl BSAI
5.6 µl H2O
5.6 µl H2O
4.6 µl H2O
1 µl a1
Gal4:PIF:phy:VP16+ren; a1
LexA:PIF:phy:ren+opLex:luc; ?1
LexA:KNdronpa:ren+OpLex:Luc; ?1
1 µl Gal4:PIF:phy:VP16
1 µl LexA:PIF:phy:ren
1 µl LexA:KNdronpa:ren
1 µl renilla (159)
1 µl opLex:luc (151)
1 µl OpLex:Luc (151)
1.2 µl buffer
1.2 µl buffer
1.2 µl buffer
1.2 µl BSA
1.2 µl BSA
1.2 µl BSA
1 µl T4 ligase
1 µl T4 ligase
1 µl T4 ligase
1 µl BSAI
1 µl BsmbI
1 µl BsmbI
4.6 µl H2O
4.6 µl H2O
4.6 µl H2O
1 µl a1
1 µl ?1
1 µl ?1
Gal4:KNdronpa:ren+UAS:luc; ?1
LacI:KNdronpa:ren+OpLacI:luc; ?1
1 µl Gal4:KNdronpa:ren
1 µl LacI:KNdronpa:ren
1 µl UAS:luc (227)
1 µl OpLacI:luc (152)
1.2 µl buffer
1.2 µl buffer
1.2 µl BSA
1.2 µl BSA
1 µl T4 ligase
1 µl T4 ligase
1 µl BsmbI
1 µl BsmbI
4.6 µl H2O
4.6 µl H2O
1 µl ?1
1 µl ?1
11 July 2015
Prepare glycerinates of:
Bxb1+Rep:GFP; ?1
N-Dronpa; pUPD2
Bxb1:Etr8; pUPD2
Etr8(CMV):Bxb1:T35S; a1
Pick up the liquid cultures of AsLOVpep and red toggle. We observed that one tube of a red toggle colony is blue, we discard it.
Do miniprep of:
AsLOVpep (C1 and C2)
Red toggle (C2)
Digestions:
Red toggle
BamHI
6674, 6100 / 4209, 3756
AsLOVpep
NotI
2558, 512
Me falta el resultado del gel!!
AsLOVpep C1
AsLOVpep C2
Red toggle C2
¿?
Do transformation of DHSa and yesterday ligations:
phyC31;pUPD2
Reporter phiC31; pUPD2
LacI:PIF:Phy:VP16+ren; a1
Gal4:PIF:phy:VP16+ren; a1
LexA:PIF:phy:ren+opLex:luc; ?1
LexA:KNdronpa:ren+OpLex:Luc; ?1
Gal4:KNdronpa:ren+UAS:luc; ?1
LacI:KNdronpa:ren+OpLacI:luc; ?1
The pUPD2 in plates with cloranfenicol+IPTG+XGal. The a1 in plates with kanamicyn+IPTG+XGal. The ?1 in plates with spectinmicyn+IPTG+XGal.
12 July 2015
Pick colonies and make liquid culture:
phyC31;pUPD2 (C1-C3)
Reporter phiC31; pUPD2 (C1-C3)
LacI:PIF:Phy:VP16+ren; a1 (C1-C3)
Gal4:PIF:phy:VP16+ren; a1  All blue colonies.
LexA:PIF:phy:ren+opLex:luc; ?1 (C1)
LexA:KNdronpa:ren+OpLex:Luc; ?1(C1-C3)
Gal4:KNdronpa:ren+UAS:luc; ?1 (C1)
LacI:KNdronpa:ren+OpLacI:luc; ?1 All blue colonies.
We have to repeat the transformations or do again ligations.
Ligations were repeated:
Gal4:PIF:phy:VP16+ren; a1
LacI:KNdronpa:ren+OpLacI:luc; ?1
1 µl Gal4:PIF:phy:VP16
1 µl LacI:KNdronpa:ren
1 µl renilla (159)
1 µl OpLacI:luc (152)
1.2 µl buffer
1.2 µl buffer
1.2 µl BSA
1.2 µl BSA
1 µl T4 ligase
1 µl T4 ligase
1 µl BSAI
1 µl BsmbI
4.6 µl H2O
4.6 µl H2O
1 µl a1
1 µl ?1
Refresh the liquid cultures of Agrobacterium:
Bxb1:GFP and Etr8:tnos. They were 2 days at 28ºC.
Pnos, it was at the fridge (-4ºC).
13 July 2015
All the liquid cultures have grown, do minipreps.
Do digestions:
phyC31;pUPD2 (C1-C3)
NotI
2046, 1899
Reporter phiC31; pUPD2 (C1-C3)
NotI
2046, 475
LacI:PIF:Phy:VP16+ren; a1 (C1-C3)
EcoRI
6345, 5623, 5487
LexA:PIF:phy:ren+opLex:luc; ?1 (C1)
BamHI
9431, 6674, 3531
LexA:KNdronpa:ren+OpLex:Luc; ?1(C1-C3)
BamHI
1199, 6674
Gal4:KNdronpa:ren+UAS:luc; ?1 (C1)
BamHI
11582, 6674
Agarose gel:
phyC31 C1
phyC31 C2
phyC31 C3
RepPhiC31 C1
no
no
no
ok
RepPhiC31 C2
LacI:PIF:Phy:VP16+ren C1
LacI:PIF:Phy:VP16+ren C2
LacI:PIF:Phy:VP16+ren C3
no
no
ok
ok
LexA:PIF:phy:ren+opLex:luc
LexA:KNdronpa:ren+OpLex:Luc C1
LexA:KNdronpa:ren+OpLex:Luc C2
LexA:KNdronpa:ren+OpLex:Luc C3
no
ok
ok
ok
Gal4:KNdronpa:ren+UAS:luc C1
no
The Agrobacterium cultures refreshed yesterday were store in the fridge.
It is made another culture of 35S:Bxb1+reporterBxb1:GFP to keep it in the fridge. It had 5ml of LB medium, 5 µl rifampicin and 5 µl of spectinomicyn an 1 µl of the culture.
Mesurement of the OD’s:
35S:Bxb1+reporterBxb1:GFP: 0.28 (dilution 1:10)
143 µl of culture+1857 µl of MES/acetosiningon solution.
With this preparation 2 plants were infiltrated and let in natural light to see the normal activity of the recombinase.
Transformation of the ligation: Gal4:PIF:phy:VP16+ren; a1 and LacI:KNdronpa:ren+OpLacI:luc; ?1.
The liquid cultures of this constructions were repeated:
phyC31;pUPD2 (C4 and C5)
Reporter phiC31; pUPD2 (C4)
LacI:PIF:Phy:VP16+ren; a1 (C4)
LexA:PIF:phy:ren+opLex:luc; ?1 (C1)
LexA:KNdronpa:ren+OpLex:Luc; ?1(C1-C3)
Gal4:KNdronpa:ren+UAS:luc; ?1 (C1)
AsLOVpep; pUPD2 (C3)
14 July 2015
Do minipreps of:
phyC31;pUPD2 (C4 and C5)
Reporter phiC31; pUPD2 (C4)
LacI:PIF:Phy:VP16+ren; a1 (C4)
LexA:PIF:phy:ren+opLex:luc; ?1 (C1)
LexA:KNdronpa:ren+OpLex:Luc; ?1(C1-C3)
Gal4:KNdronpa:ren+UAS:luc; ?1 (C1)
AsLOVpep; pUPD2 (C3)
Do digestions:
phyC31;pUPD2
NotI
2046, 1899
Reporter phiC31; pUPD2
NotI
2046, 475
LacI:PIF:Phy:VP16+ren; a1
EcoRI
6345, 5623, 5487
LexA:PIF:phy:ren+opLex:luc, ?1
BamHI
9431, 6674, 3531
LexA:KNdronpa:ren+OpLex:Luc; ?1
BamHI
1199, 6674
Gal4:KNdronpa:ren+UAS:luc; ?1
BamHI
11582, 6674
AsLOVpep; pUPD2
NotI
2558, 512
Make an agarose gel:
phyC31 (C4)
phyC31 (C5)
AsLOVpep (C4)
LexA:PIF:phy:ren+opLex:luc (C1)
no
no
No
No
LexA:KNdronpa:ren+OpLex:Luc (C3)
Gal4:KNdronpa:ren+UAS:luc (C1)
Gal4:KNdronpa:ren+UAS:luc(C2)
Gal4:KNdronpa:ren+UAS:luc (C3)
Ok
no
no
No
LacI:PIF:Phy:VP16+ren
Reporter phiC31 (C1)
no
Ok
Only LexA:KNdronpa:ren+OpLex:Luc and Reporter phiC31 were correct. Repeat the ligations because is the second digestion of this construction that were made.
Measurement of DNA concentration:
Reporter:phyC31: 13.6 ng/µl
PhyC31: 4.6 ng/µl
AsLOVpep: 30.3 ng/µl
Igem151 (op:LexAluc): 40 ng/µl
Gal4:PIF:phyB:ren (C1): 124.4 ng/µl
LacI:PIF:phyB:VP16 (C1): 126 ng/µl
Igem 159 (renilla): 42 ng/µl
Gal4:Kdronpa:Ndronpa:renilla (C3): 172.8 ng/µl
Igem 227 (op:UAS:luc): 6.3 ng/µl
Pick colonies and make liquid culture of LacI:KDronpa:NDronpa:ren:luc (C4 and C5). The colonies of Gal4:PIF:phy:VP16+ren were all blue.
15 July 2015
Miniprep of LacI:KDronpa:NDronpa:ren:luc (C4 and C5).
Digestion:
LacI:KDronpa:NDronpa:ren:luc
EcoRI
6345, 9965
Make the gel:
LacI:K:NDronpa:ren:luc C4
LacI:K:NDronpa:ren:luc C5
no
no
Measurement of DNA concentrations:
Gal4BD:PIF:PhyB:VP16: 125.6 ng/µl
LexA:PIF:phyB:VP16:ren: 322.6 ng/µl
LacI:Kdronpa:NDronpa:ran: 209.0 ng/µl
We decided to repeat the ligations due to that we make twice the digestions and we didn’t obtain good results.
phyC31;pUPD2
AsLOVpep; pUPD2
LacI:PIF:Phy:VP16+ren; a1
1 µl phiC31
1 µl AsLOVpep
1.5 µl LacI:PIF:Phy:VP16
1 pUPD2
1 pUPD2
2.5 µl renilla (159)
1.2 µl buffer
1.2 µl buffer
1.2 µl buffer
1.2 µl BSA
1.2 µl BSA
1.2 µl BSA
1 µl T4 ligase
1 µl T4 ligase
1 µl T4 ligase
1 µl BsmbI
1 µl BsmbI
1 µl BSAI
5.6 µl H2O
5.6 µl H2O
4.6 µl H2O
0.5 µl a1
Gal4:PIF:phy:VP16+ren; a1
LexA:PIF:phy:ren+opLex:luc; ?1
LacI:KNdronpa:ren+OpLex:Luc; ?1
1.5 µl Gal4:PIF:phy:VP16
1 µl LexA:PIF:phy:ren
1 µl LacI:KNdronpa:ren
2 µl renilla (159)
2.5 µl opLex:luc (151)
3 µl OpLac:Luc (152)
1.2 µl buffer
1.2 µl buffer
1.2 µl buffer
1.2 µl BSA
1.2 µl BSA
1.2 µl BSA
1 µl T4 ligase
1 µl T4 ligase
1 µl T4 ligase
1 µl BSAI
1 µl BsmbI
1 µl BsmbI
2.6 µl H2O
2.6 µl H2O
3 µl H2O
1 µl a1
0.5 µl ?1
0.5 µl ?1
Gal4:KNdronpa:ren+OpLex:Luc; ?1
PhyB:VP16+PIF6; ?1
1 µl Gal4:KNdronpa:ren
1 µl PhyB:VP16 (88E)
4 µl OpUAS:Luc (227)
2 µl PIF6 (170)
1.2 µl buffer
1.2 µl buffer
1.2 µl BSA
1.2 µl BSA
1 µl T4 ligase
1 µl T4 ligase
1 µl BsmbI
1 µl BsmbI
3 µl H2O
3.6 µl H2O
0.5 µl ?1
1 µl ?1
These Agrobacterium cultures have been refreshed:
PIF-phyB-luc
Renilla
Pnos
Etr8
16 July 2015
Yesterday ligations have been transformed into E. coli:
phyC31;pUPD2
AsLOVpep; pUPD2
LacI:PIF:Phy:VP16+ren; a1
Gal4:PIF:phy:VP16+ren; a1
LexA:PIF:phy:ren+opLex:luc; ?1
LacI:KNdronpa:ren+OpLex:Luc; ?1
Gal4:KNdronpa:ren+OpLex:Luc; ?1
PhyB:VP16+PIF6; ?1
The agroinfiltrated leaf with BxbI:rep:GFP has been observed in the magnifying glass with fluorescent lights. The efficiency of the recombinase is lower and it can not been observed a lot of green spots. Tomorrow another leaf will be seen to check again the construction.
Second refresh of the Agrobacterium cultures:
PIF-phyB-luc
Renilla
Pnos
Etr8
Miniprep of these cultures.
Digestion of the minipreps:
PIF-phyB-luc; ?1
EcoRI
??
Renilla; ?2
HindIII
No lo encuentro
Pnos; ?1
EcoRI
2997, 353
Etr8; ?1
EcoRI
The digestions had positive controls that were included in the gel to compare the results obtained.
The digestions are left overnight in the working table.
17 July 2015
Do the gel:
No se lo que pone?
Liquid culture of the colonies in the agar plates have been made, 3 colonies for each construction.
OD’s mesurement for the agroinfiltration.
Dilution 1:10 in MES.
PIF:phyB:luc
0.47
41 µl/ml
630 µl
Renilla
0.28
71 µl/ml
1065 µl
Etr8:luc
0.32
52 µl/ml
930 µl
Pnos
0.34
59 µl/ml
885 µl
Red toggle experiement:
The plants were coinfiltrated with the red toggle (PIF+phyB) and renilla. Also with two controls, Pnos (positive control) and Etr8 (negative control).
14 plants were used with 3 infiltrated leafs for each plants and two spots per leaf.
The controls have been infiltrated in leafs of the plants like is shown in the picture. Pnos in the right part and Etr8 in the left part.
Two plants were infiltrated with both controls in the three leafs and they were in natural light during all the experiment.
Another 4 plants were infiltrated with controls following the same pattern in the procedure.
The remaining 8 plants were infiltrated with the red toggle.
Immediately after the infiltration the plants were distributed in three different conditions: natural light, darkness and far red.
So after the agroinfiltration 2 control plants stay in natural light all the experiment; 2 control plants and 4 red toggle went into the far red chamber and the same amount of control and red toggle plan (2+4) went put in darkness.
This conditions were maintained 3 days and then all the infiltrated leafs were cut into small discs and put into a special plates with water.
In this moment was set the time 0 (21/07/2015 at 19:30) and take the first samples.
Then in time 0 some of the dark and far red samples were put in red conditions to activate the red toggle.
This scheme represent the distribution of samples and the times that were taken the samples.
Hacer el esquema!!! Cuando este mas espabilada ;)
It was observed another leaf with the recombinase BxbI with GFP in the magnifying glass with fluorescent lights. It was not observed a lot of spots, the efficiency is very low.
18 July 2015
23 minipreps of the liquid cultrures. LexA:PIF:phtB:ren:luc (C3) has not grown.
Digestions of the minipreps:
phyC31;pUPD2
NotI
2046, 1899
AsLOVpep; pUPD2
NotI
2046, 521
LacI:PIF:Phy:VP16+ren; a1
EcoRI
6345, 5623, 5487
Gal4:PIF:phy:VP16+ren; a1
EcoRI
6345, 5487, 4852
LexA:PIF:phy:ren+opLex:luc; ?1
BamHI
9431, 6674, 3513
LacI:KNdronpa:ren+OpLex:Luc; ?1
BamHI
12632, 6574
Gal4:KNdronpa:ren+OpLex:Luc; ?1
BamHI
11582, 6674
PhyB:VP16+PIF6; ?1
BamHI
6674, 2685, 2337, 1439
Gel has been done:
phyC31 C1
phyC31 C2
phyC31 C3
AsLOVpep C1
no
no
no
ok
AsLOVpep C2
AsLOVpep C3
Gal4:PIF:phy:VP16:ren C1
Gal4:PIF:phy:VP16:ren C2
no
no
no
no
Gal4:PIF:phy:VP16:ren C3
LacI:PIF:phy:ren C1
LacI:PIF:phy:ren C2
LacI:PIF:phy:ren C3
no
ok
no
No
LexA:PIF:phy:ren:luc C1
LexA:PIF:phy:ren:luc C2
Gal4:KNdronpa:ren:luc C1
Gal4:KNdronpa:ren:luc C2
no
no
ok
No
Gal4:KNdronpa:ren:luc C3
LacI:KNdronpa:ren:luc C1
LacI:KNdronpa:ren:luc C2
LacI:KNdronpa:ren:luc C3
no
no
ok
No
PhyB:VP16:PIF6 C1
PhyB:VP16:PIF6 C2
PhyB:VP16:PIF6 C3
ok
no
no
Pick more colonies of:
Gal4:PIF:phy:VP16+ren; a1
LexA:PIF:phy:ren+opLex:luc; ?1
New digestions with new enzymes:
phyC31;pUPD2
XhoI (buffer red)
2119, 934, 894
LacI:PIF:Phy:VP16+ren; a1
NEB4
5949, 5653, 3610, 2246
LacI:PIF:Phy:VP16+ren; a1
HindIII
11568, 5587
After 3 digestions of LacI:PIF:Phy:VP16+ren; a1 is accepted the construction.
It was observed another leaf with the recombinase BxbI with GFP in the magnifying glass with fluorescent lights. It was not observed a lot of spots, the efficiency is very low. This is the 4th day…
19 July 2015
20 July 2015
21 July 2015
Red toggle experiment:
Time lapses:
-19:00=t0
-1:00=t1
-7:00= t2
-19:00= t3 (it was taken the controls in natural light)
Minipreps of the colonies that were in 37ºC.
LexA:PIF:Phy:ren:luc (C1 and C2)
PhyC31 (C1, C2, C4 and C5)
LexA:PIF:Phy:ren:luc
BamHI
9431, 6674, 3513
PhyC31
NotI
2046, 1899
Gel with the digestions:
PhyC31 C1
PhyC31 C2
PhyC31 C4
PhyC31 C5
Ok?
ok
ok
ok
LexA:PIF:Phy:ren:luc C1
LexA:PIF:Phy:ren:luc C2
No DNA
No DNA
We had problems with some colonies because in the digestion did not appear DNA. The minipreps will be made with a better kit.
Transform in Agrobacterium this cultures:
LexABD:KDronpa:NDronpa:ren:luc
Gal4BD:KDronpa:NDronpa:ren:luc
LacIBD:KDronpa:NDronpa:ren:luc
OpLexA:luc (151)
OpUAS:luc
OpLacI:luc (152)
It was made liquid culture of Gal4:PIF:phyB:ren; ?1 (C1-C5)
It was made ligations:
35S:Gal4:AsLOVpep:T35S; ?1
35S:LacI:AsLOVpep:T35S; ?1
35S:LexA:AsLOVpep:T35S; ?1
1 µl 35S (0030)
1 µl 35S (0030)
1 µl 35S (0030)
1 µl Gal4BD
1 µl LacI
1 µl LexA
1 µl AsLOVpep
1 µl AsLOVpep
1 µl AsLOVpep
1 µl T35S (0036)
1 µl T35S (0036)
1 µl T35S (0036)
1 µl ?1
1 µl ?1
1 µl ?1
2.6 µl H2O
2.6 µl H2O
2.6 µl H2O
PsinATG:RepPhiC31:GFP:T35S; ?2
LacI:PIF:PhyB:ren+luc; ?1
PIF:PhyB+renilla; ?2
1 µl PsinATG (552)
1.5 µl LacI:PIF:PhyB:ren; ?1
1.5 µl PIF:PhyB
1 µl ReporterPhyC31
3 µl OpLacI:luc (152); ?2
2.5 µl renilla (159)
1 µl GFP (0059)
0.5 µl ?1
0.5 µl ?2
1 µl T35S (0036)
2.6 H2O
3 µl H2O
1 µl ?2
2.6 µl H2O
Luciferase essay with all the samples collected in the last three days.
Sampling: 25 samples in total.
Passive lysis 1x (200µl/sample x 25 samples)= 5.000 µl
Crush samples.
Add 150 µl of passive lysis buffer and mix in the vortex.
Centrifuge at 13200rpm for 15’.
E8/li
E8/li
E8/li
P/li
P/li
P/li
TR/Fr
TR/Fr
TR/Fr
TR/D
TR/D
TR/D
TR/Fr
Red
TR/Fr
Red
TR/r
Red
TR/D
Red
TR/D
Red
TR/D
Red
We used 14.4 µl of Stop and Glow. 705.6 destilled H2O
22 July 2015
Miniprep of Gal4:PIF:PhyB:ren (C1-C3) C4 and C5 did not grow.
Digestion of the miniprep:
Gal4:PIF:PhyB:ren
BamHI
16684
EcoRI
4852, 5487, 6345
EcoRV
1849, 3942, 2475, 381, 8037
Gel was made:
Gal4… (BamHI)
Gal4… (EcoRI)
Gal4… (EcoRV)
no
no
no
After doing several digestions with different enzyme all with wrong band patterns, it was decided to revise each part making digestions. The parts are:
PhyC31; pUPD2
Gal4:PIF:phyB;
Renilla (GB159)
Made liquid culture of the ReporterBxbI:GFP in Agrobacterium.
Transform the ligations into E. coli:
35S:Gal4:AsLOVpep:T35S; ?1
35S:LacI:AsLOVpep:T35S; ?1
35S:LexA:AsLOVpep:T35S; ?1
PsinATG:RepPhiC31:GFP:T35S; ?2
LexA:PIF:PhyB:ren+luc; ?1
Add 300 µl of SOC medium and put into a agar plate with the corresponding antibiotics.
Luciferase essay:
Sampling: samples that have been in red and natural light 48h, 3 samples.
200 µl/sample x 3 sample= 600 µl of passive lissis buffer (5x)
Passive lissis buffer 1x= 120 µl+480 µl water.
2.4 µl of Stop and glow
118 µl of  buffer.
23 July 2015
We decided to infiltrate soybean sprouts. First we decided to infiltrate with dye to observe the characteristics and the capacity of absorption.
Mesurement of the ODs to agroinfiltrate:
The samples are:
Citoplasm=0.27
DsRed=0.27
GFP=0.36
Recombinase=0.43
Vi= (Vf*Absf)/(Absi*10)
Absi=0.1 (because is a viral system)
Vf=120 µl
Make 16 liquid culture of the white colonies in the agar plates.
24 July 2015
Minipreps of the 13 liquid culture. LacI:PIF:phyB:ren cultures did not grow.
Digestion of the minipreps:
Gal4:AsLOVpep
EcoRI
6345, 1972
LacI:AsLOVpep
EcoRI
6345, 2743
LexA:AsLOVpep
EcoRI
6345, 2011
PsinATG:RepPhiC31:GFP
HindIII
6345, 2691
LexA:PIF:PhyB:ren+luc
BamHI
9431, 6674, 3513
PhyC31; pUPD2
NotI
2046, 1899
Gal4:PIF:phyB
BamHI
6674, 3474, 2337
Renilla (GB159)
EcoRV
2909, 2475, 882, 812, 381
It was done 2 gels with ligations:
Gal4:AsLOV C1
Gal4:AsLOV C2
Gal4:AsLOV C3
PsinATG:RepPhi
C31:GFP C1
PsinATG:RepPhi
C31:GFP C2
PsinATG:RepPhi
C31:GFP C3
ok
ok
ok
ok
no
No
Mw/ladder
LacI:AsLOV C1
LacI:AsLOV C2
LexA:AsLOV C1
LexA:AsLOV C2
LexA:AsLOV C3
-
ok
No
no
ok
no
LexA:PIF:PhyB:ren+luc C1
LexA:PIF:PhyB:ren+luc C2
no
Ok
PhyC31 (C1)
PhyC31 (C2)
PhyC31 (C3)
PhyC31 (C4)
PhyC31 (C5)
no
ok
ok
ok
no
Gal4:PIF:phyB
Renilla (GB159)
ok
Ok
26 July 2015
Liquid cultures of Agrobacterium were refreshed:
TsinATG:BxbIreporter:GFP
TsinATG:BxbI:reporterBxbI:GFP
Viral vector??no se cual es
27 July 2015
Sent to sequence:
210.08.256: LacIBD; pUPD2 (C1)
210.08258: Gal4BD; pUPD2 (C2)
210.08.259:LexABD; pUPD2 (C1)
210.08.260: PIF6; pUPD2 (C5)
210.08.261: VP16; pUPD2 (C1)
210.08.262: PhiC31; pUPD2 (C2)
210.08.264: PhiC31; pUPD2 (C3)
210.08.266: PhiC31; pUPD2 (C4)
210.08.268: ReporterBxbI; pUPD2 (C1)
210.08.269: ReporterPhiC31; pUPD2 (C1)
210.08.270: AsLOVpep; pUPD2 (C1)
The sample had to have 10µl of miniprep (200ng/µl aprox) + 5 µl of primer (dilution 1:3)
Ligations:
Gal4:PIF:phiB + ren; ?1
LacI:PIF:phiB:ren + luc; ?2
PIF:PhyB+renilla; ?2
1.5 µl Gal4:PIF:phiB
1.5 µl LacI:PIF:phiB:ren
1.5 µl PIF:phiB
2 µl Renilla (GB159)
2 µl OpLacI:luc
2.5 µl Renilla (GB159)
0.5 µl ?1
0.5 µl ?2
0.5 µl ?2
3.6 µl H2O
2.6 µl H2O
3 µl H2O
OD’s measurements to prepare the agroinfiltrates:
Cytoplasm: 0.39 (viral)
0.25ml
Integrase: 0.35 (viral)
0.28ml
GFP: 0.32 (viral)
0.31ml
Dsred: 0.31 (viral)
0.32ml
BxbI:reporter: 0.41
0.49ml
Reporter: 0.26
0.77ml
3 plants were infiltrated with BxbI and the reporter of BxbI (one leaf with the control an the other with the recombinase)?? No entiendo lo de la libreta
Ligations to join the negative controls with renilla:
Etr8:luc+staffer (SF)
OpLexA:luc+SF
OpLacI:luc+SF
UAS:luc+SF
1.5 µl Etr8:luc (88C o 1098)
1.5 µl OpLexA:luc (151)
1.5 µl OpLacI:luc (152)
1.5 µl UAS:luc (227)
1 µl SF; ?2
1 µl SF; ?2
1 µl SF; ?2
1 µl SF; ?2
1 µl ?1
1 µl ?1
1 µl ?1
1 µl ?1
6.1 µl H2O
6.1 µl H2O
6.1 µl H2O
6.1 µl H2O
Transformation of E. coli of the ligations:
Gal4:PIF:phiB + ren; ?1
LacI:PIF:phiB:ren + luc; ?2
PIF:PhyB+renilla; ?2
Make cultures in agar petri dishes of the transformations with the corresponding antibiotics.
Transformation into Agrobacterium with the constructions:
Gal4:KDronpa:NDronpa:ren:luc
LacI:KDronpa:NDronpa:ren:luc
LexA:KDronpa:NDronpa:ren:luc
OpLexA:luc (GB 151)
OpLacI:luc (GB 152)
UAS:luc (GB 227)
Make cultures in agar petri dishes of the transformations with the corresponding antibiotics.
It was received a new piece (ePDZ) which is part of the blue toggle (plan A). It arrived in E.coli so it was made a lquid culture and let it grow at 37ºC overnight.
28 July 2015
Miniprep of the liquid culture: ePDZ.
Transformation into E.coli of the ligations:
Etr8:luc+staffer (SF)
OpLexA:luc+SF
OpLacI:luc+SF
UAS:luc+SF
Gal4:PIF:phyB:ren
It was observed the soybean sprouts that were infiltrated with a dye with green light and red filter. It was not observed nothing significant, moreover, the damage is evident.
Transformation in Agrobacterium the ReporterPhiC31:GFP.
Pick colonies and make liquid cultures adding X-Gal and IPTG because the colonies were little and we can not observe clearly if they were white or blue.
Gal4:PIF:phiB + ren. Did not grow any colony.
LacI:PIF:phiB:ren + luc (C1-C3)
PIF:PhyB+renilla (C1- C3)
The medicine LTB (heat labile toxin B subunit) which is part of an enterotoxin of Echerichia coli homologous to the same toxin in Vibrio cholerae that causes diarrhea.
We add 50µl to have a final concentration of 10ng/µl.
Ligation:
LTB; pUPD2
1 µl LTB
1 µl pUPD2
5.6 µl H2O
29 July 2015
Miniprep of yesterday liquid culture:
LacI:PIF:phiB:ren + luc (C1 and C3) C2 turn into blue.
PIF:PhyB+renilla (C2) C1 and C3 did not grow.
Digestion:
LacI:PIF:phiB:ren:luc
EcoRV
882, 968, 1652, 3942, 2475, 381, 3477, 6674
PIF:PhyB+renilla
HindIII
4316, 5887, 788, 6345
Gel:
LacI:PIF:phiB:ren:luc (C1)
LacI:PIF:phiB:ren:luc (C3)
PIF:PhyB+renilla (C2)
no
no
no
Pick colonies and make liquid culture of:
Etr8:luc:staffer(SF) (C1-C3)
OpLexA:luc:SF (C1-C3)
OpLacI:luc:SF (C1-C3)
UAS:luc:SF (C1-C3)
Gal4:PIF:phyB:ren (C1-C3)
Transformation in E.coli of:
LTB; pUPD2
30 July 2015
Minipreps have been done:
Etr8:luc:staffer(SF) (C1-C3)
OpLexA:luc:SF (C1 and C3) C2 did not grow.
OpLacI:luc:SF (C1-C3)
UAS:luc:SF (C1-C3)
Gal4:PIF:phyB:ren (C1-C3)
LacI:PIF:phy:ren:luc (C1-C3)
PIF:phyB:ren (C1-C3)
Etr8:luc:staffer(SF)
BamHI
6674, 2766
OpLexA:luc:SF
BamHI
6674, 2746
OpLacI:luc:SF
BamHI
6674, 2847
UAS:luc:SF
BamHI
6674, 2568
Gal4:PIF:phyB:ren
EcoRI
6345, 5487, 4852
LacI:PIF:phy:ren:luc
BamHI
20451
PIF:phyB:ren
HindIII
6345, 5887, 4316, 788
Do the gel:
No se el orden!! Preguntar
Primers have arrived:
ePDZ reverse and forward and phyC31.
Make a PCR with ePDZ and its primers to obtain the desired fragment and put it in pUPD2.
ePDZ PCR
10µl Buffer HF
31.5 µl H2O
2 µl dNTPs
2.5 µl  Primer forward
2.5 µl primer reverse
1 µl ePDZ (dilution 1:50)
0.5 µl Taq phunion
Pick 2 colonies of phiC31:GFP in Agrobacterium and make liquid culture.
Ligations:
ePDZ; pUPD2
PIF:phyB+ren; ?2
OpUAS:luc:SF+ren; ?1
OpLexA:luc:SF+ren; ?1
1 µl ePDZ
1 µl PIF:phyB
1 µl OpUAS:luc:SF
1 µl OpLexA:luc:SF
1 µl pUPD2
1 µl ren (159)
1 µl ren
1 µl ren
5.6 µl H2O
1 µl ?2
1 µl ?1
1 µl ?1
4.6 µl H2O
4.6 µl H2O
4.6 µl H2O
OpEtr8:luc:SF+ren; ?1
Op:LacI:luc:SF+ren; ?1
1 µl OpEtr8:luc:SF
1 µl OpLacI:luc:SF
1 µl ren
1 µl ren
1 µl ?1
1 µl ?1
4.6 µl H2O
4.6 µl H2O
After 3 days till the agroinfiltration we have observed the leaf discs that have BxbI+repBxbI and the repBxbI (negative control).
We could see that the level of GFP expression was higer in the infiltration with the recombinase but there was also expression in the negative control. We think that this can be a contamination due to that we had infiltrated both constructions in he same leaf and we did not change the gloves between agroinfiltrations. FOTOO!
Make liquid culture (E.coli) of:
LTB; pUPD2 (C1-C3)
31 July 2015
Ligation:
LacI:PIF:phyB:ren+OpLacI:luc; ?2
1.5 µl LacI:PIF:phyB:ren
3 µl OpLacI:luc
0.5 µl ?2
2.6 µl H2O
After 4 days till the agroinfiltration we have observed again the samples with BxbI and the negative control but was the same. The negative control expressed GFP but less than the recombinase. Foto!
Minipreps of liquid culture LTB (C1 and C2)
Digestion:
LTB; pUPD2
NotI
2046, 474
Gel:
LTB (C1)
LTB (C2)
PCR ePDZ
??
Take out glicerynates of Paloma, lab mate:
Sip rotavirus CH2
Sip rotavirus CH2-CH3
For E.coli and Agrobacterium.
Pick a colony of Asun interferon (IFN) in Agro, make liquid culture.
We had miniprep of IFN in pUPD. Make ligations.
Transform in E.coli the ligations and make petri dishes cultures:
ePDZ; pUPD2
PIF:phyB+ren; ?2
OpUAS:luc:SF+ren; ?1
OpLexA:luc:SF+ren; ?1
OpEtr8:luc:SF+ren; ?1
Op:LacI:luc:SF+ren; ?1
LacI:PIF:phyB:ren+OpLacI:luc; ?2
Refresh agro cultures to agroinfiltrate tomorrow:
PhiC31 (viral system)
ReporterPhiC31
BxbI+reporterBxbI
ReporterBxbI
Gal4:KDronpa:NDronpa:luc:ren (brue toggle)
PIF:phi:luc
Renilla
P19
Pnos
New experiment with Nicotiana bentamiana. PROTOCOL
Next constructions were agroinfiltrated, 2 or 3 leafs for each plant:
PhiC31: one leaf for PhiC31+RepPhiC31+P19 (coinfiltration) and the other with RepPhiC31+P19 which is the negative control. 2 plants were infiltrated.
BxbI: one leaf with BxbI:RepBxbI+P19 and the other with RepBxbI+P19 which is negative control. 2 plants were infiltrated.
Red toggle: PIF6:phyB:luc+ren (coinfiltration). They were infiltrated 3 plants with 3 leafs for each.
Blue toggle: Gal4:NDronpa:KDronpa:luc:ren. They were infiltrated 3 plants with 3 leafs for each.
Pnos, the positive control. There are 4 plantas, 3 leafs per plant.
So we will take samples in time 0, 12, 24 and 36h for each construction and control. After 2 days of the agroinfiltration (during this period all the plants were in dark, it is set time 0 we make discs os leaf and put in a plate with water and we change the conditions that were before.
Red toggle plants: 9 discs stay in dark, 9 went to red and 9 went to natural light.
Blue toggle: the same as red but went to ultraviolet instead of red light.
Pnos: one plant were in natural light during all the experiment. The other discs will go, after the 2 days in black, to red, ultraviolet and stay in dark.
Recombinases: they are in natural light during all the experiment.
1 August 2015
Prepare the Agrobacterium cultures for agroinfiltration:
Gal4B:KDronpa:NDronpa:luc:ren, PhiC31 and its control, BxbI and its control and PIF:PhyB and its control were centrifugated 10 min at 3000g. The sobrenadant was discarded and the bacterias were resuspended with the agroinfiltration medium.
Agroinfiltration medium had: 10ml MES, 1ml MgCl2, 89ml H2O, 100ml DMSO+acetosiningone.
Incubate in 2h.
Mesurement oof the ODs:
Ren+P19
0.09
888.9 µl
RepPhiC31
0.69
115.94 µl
BxbI
0.46
174 µl
RepBxbI
0.15
533.3 µl
Gal4:KNDronpa:ren:luc
0.51
156.9 µl
Pnos
0.29
275.9 µl
PIF:phy:luc
0.17
470.6 µl
PhiC31
0.32
250 µl
We went to the greenhouse and infiltrate the 13 plants.
Look the petri dishes culture:
PIF:phyB+ren; ?2
OpLexA:luc:SF+ren; ?1
Op:LacI:luc:SF+ren; ?1
This colonies did not grow, the rest were left in the fridge.
ePDZ; pUPD2
OpUAS:luc:SF+ren; ?1
OpEtr8:luc:SF+ren; ?1
LacI:PIF:phyB:ren+OpLacI:luc; ?2
3 August 2015
Miniprep of the liquid culture:
Sip rotavirus CH2
Sip rotavirus CH2-CH3
Measure of DNA concentration of:
OpLexA:luc:SF=169ng/µl
OpacI:luc:SF=291ng/µl
Pick colonies gb159and make liquid culture of:
ePDZ; pUPD2 (C1-C3)
OpUAS:luc:SF+ren; ?1(C1-C3)
OpEtr8:luc:SF+ren; ?1(C1-C3)
LacI:PIF:phyB:ren+OpLacI:luc; ?2 (C1-C3) Added X-gal and IPTG.
Repeat ligations:
OpLexA:luc:SF+ren; ?1
Op:LacI:luc:SF+ren; ?1
E-PIF:phyB+ren; ?2
1 µl OpLexA:luc:SF
1 µl OpLacI:luc:SF
1 µl E-PIF:phy
3 µl ren
3 µl ren
3 µl ren
1 µl ?1
1 µl ?1
1 µl ?1
2.6 µl H2O
2.6 µl H2O
2.6 µl H2O
Refresh agrobacterium cultures of:
Interferon
SIP-CH2
SIP-CH2-CH3
Take the glycerinate of Renilla; ?2 (GB159) and make liquid culture.
We have made all the leaf discs and put in order in the plates with 300 µl of water, also we put each plate in the conditions light that they have to be. We have taken the samples of time0 (13:00).
So as T0= 13:00, T12=1:00, T24= 13:00 and T36= 1:00.
Imagenes de la colocacion en los platos? Hace falta?
4 August 2015
We do a Western blot with Asun so we can learn how to do it. Lo copio? De verdad!!?
Miniprep of the liquid cultures:
ePDZ; pUPD2 (C1-C3)
OpUAS:luc:SF+ren; ?1(C1-C3)
OpEtr8:luc:SF+ren; ?1(C1-C3)
LacI:PIF:phyB+ren; ?2 (C2 and C3) C1 was blue.
Digestions:
ePDZ; pUPD2
NotI
2046, 642
OpUAS:luc:SF+ren; ?1
EcoRI
6345, 7096
OpEtr8:luc:SF+ren; ?1
EcoRI
6345, 7294
LacI:PIF:phyB:ren+OpLacI:luc; ?2
EcoRV
6674, 3477, 381, 2475, 3942, 1652, 968, 882
Renilla 159
EcoRV
2909, 2475, 882, 812, 381
Gel:
ePDZ C1
ePDZ C2
ePDZ C3
LacI:PIF:phyB+ren
Ok
ok
ok
Renilla 159
OpUAS:luc:SF+ren C1
OpUAS:luc:SF+ren C2
OpUAS:luc:SF+ren C3
Ok
¿?
OpEtr8:luc:SF+ren C1
OpEtr8:luc:SF+ren C2
OpEtr8:luc:SF+ren C3
Transformation into E.coli of yesterday ligations:
OpLexA:luc:SF+ren; ?1
Op:LacI:luc:SF+ren; ?1
E-PIF:phyB+ren; ?2
Make petri dishes culture with the indicate antibiotic.
Ligations:
35S+ePDZ+VP16+T35S;?2
35S+LTB+T35S; ?1
1µl ePDZ
1µl 35S (30)
1 µl VP16
1µl T35S (36)
1 µl 35S (30)
1µl LTB
1 µl T35S (36)
1µl ?1
1 µl ?2
3.6 µl H2O
2.6 µl H2O
Refresh agrobacterium cultures of:
Interferon
SIP-CH2
SIP-CH2-CH3
5 August 2015
Transform ligations:
35S+ePDZ+VP16+T35S; ?2
35S+LTB+T35S; ?1
Pick colonies and make liquid cultures cultures of:
OpLexA:luc:SF+ren; ?1 (C1-C3)
Op:LacI:luc:SF+ren; ?1 (C1-C3)
E-PIF:phyB+ren; ?2 (C1)
35S+ePDZ+VP16+T35S; ?2 (C1 and C2)
35S+LTB+T35S; ?1 (C1 and C2)
Make luciferase essay of red and blue toggle:
Number of samples=75 (a lot)
1. Prepare lisis buffer for 80 samples.
200 µl/sample * 80sample = 16ml
Buffer (5x)—3.2ml of buffer + 12.8ml H2O miliQ: lysis buffer (1x)
2. Crushed samples+ 150 µl of lysis buffer.
3. Centrifuge 15min at 13200rpm in cold.
4. Dilution 2:3 (Add 36 µl lysis buffer + 24 µl sample)
5. Luciferase: 40 µl/sample. Prepare 2.88ml (3 substrate tubes)
6. Renilla:….?
6 August 2015
Miniprep of the liquid cultures.
Digestion:
OpLexA:luc:SF+ren
EcoRI
6345, 7274
Op:LacI:luc:SF+ren
EcoRI
6345, 7375
E-PIF:phyB+ren
HindIII
6345, 788, 5887, 4316
35S+ePDZ+VP16+T35S
HindIII
6345, 2316
35S+LTB+T35S
EcoRI
6345, 1684
Gel:
PIF:phyB+ren
OpLexA:luc:SF+ren C1
OpLexA:luc:SF+ren C2
OpLexA:luc:SF+ren C3
No
Ok, repeat
No
Ok, repeat
Op:LacI:luc:SF+ren C1
Op:LacI:luc:SF+ren C2
Op:LacI:luc:SF+ren C3
35S+LTB+T35S C1
Ok, repeat
Ok
Ok
ok
35S+LTB+T35S C2
35S+ePDZ+VP16+T35S C1
35S+ePDZ+VP16+T35S C2
Ok
ok
ok
Make colony PCR with 6 colonies of PhiC31:
DNA-
JM1: 2 µl (dilution 1:10)
JM2: 2 µl (dilution 1:10)
Buffer Taq (with Mg): 2 µl
dNTPs: 2.5 µl
Taq: 0.5 µl
H2O: 10 µl
Program: 96ºC-2min; 96ºC-30min; 55ºC-30min; 72ºC-30min
Make a gel with the PCR but we did not see the correct bands.
Repeat the colony PCR:
DNA-
JM1: 2.5 µl (dilution 1:10)
JM2: 2.5 µl (dilution 1:10)
Buffer Taq (with Mg): 10 µl
dNTPs: 2 µl
Taq: 0.5 µl
H2O: 7.5 µl
Program: : 96ºC-10min; 96ºC-30min; 55ºC-30min; 72ºC-30min
Ligations:
LexA:AsLOVpep+ePDZ; ?1
LacI:AsLOVpep+ePDZ; ?1
Gal4:AsLOVpep+ePDZ; ?1
1 µl LexA:AsLOVpep; ?1
1 µl LacI:AsLOVpep; ?1
1 µl Gal4:AsLOVpep; ?1
1 µl ePDZ; ?2
1 µl ePDZ; ?2
1 µl ePDZ; ?2
1 µl BsmBI
1 µl BsmBI
1 µl BsmBI
1 µl ?1
1 µl ?1
1 µl ?1
4.6 µl H2O
4.6 µl H2O
4.6 µl H2O
Refresh Agro cultures to agroinfiltrate:
PhiC31:RepPhiC31:GFP
RepPhiC31:GFP
BxbI:RepBxbI:GFP
RepBxbI:GFP
IFN (interferon)
Sip-CH2
Sip-CH2-CH3
Pnos
P19 (this culture ha 10ml of LB + 10 µl antibiotics + 5 µl culture)
7 August 2015
Transformation into Agrobacterium:
35S:LTB:VP16:T35S
Transformation into E.coli and make petri dishes cultures:
LexA:AsLOVpep+ePDZ
LacI:AsLOVpep+ePDZ
Gal4:AsLOVpep+ePDZ
Make a gel with the colonies PCRs:
C7
C8
C9
C10
C11
C12
C13
C14
No
no
no
no
no
no
no
no
There was no DNA, there is a problem with the cells or the procedure, it have to be revised.
Ligation:
PhyC31; pUPD2
3 µl PhiC31
1 µl pUPD2
1 µl BsmBI
3.6 µl H2O
Refresh agrobacterium cultures for tomorrow infiltration:
Sip-CH2
Sip-CH2-CH3
Pnos
Red toggle (PIF6:PhyB:luc )
Renilla
Blue toggle (….:KDronpa:NDronpa:ren:luc)
A new experiment was started:
We are going to check again the recombinase BxbI and its reporter (negative control).
Test the recombinase PhiC31. Due to that we did not obtain yet the complete recombinase, we will coinfiltrate the PhiC31 and the RepPhiC31:GFP following the same scheme as BxbI. 2 plants with 2 leafs, one of them with PhiC31 + RepPhiC31:GFP and the other with only RepPhiC31:GFP.
Infiltration in 2 plants with 2 leafs each of them with interferon.
Measure f the ODs:
Construction
OD
Volume (ml)
IFN
0.13
1.54
RepBxbI:GFP
0.13
1.54
BxbI:RepBxbI:GFP
0.33
0.61
PhiC31
0.19
1.05
RepPhiC31:GFP
0.22
0.91
8 August 2015
Transform the ligation into E.coli and make petri dishes cultures:
PhiC31; pUPD2.
LexA:AsLOVpep+ePDZ (C1-C5)
LacI:AsLOVpep+ePDZ (C1-C4)
Gal4:AsLOVpep+ePDZ (C1-C4)
New experiment:
It will be tested again the red toggle (PIF6:PhyB:luc + ren), the blue toggle (LacI:KDronpa:NDronpa:ren:luc), pnos  (positive control to check the ratio renilla luciferasa) and the production of antirotavirus that are SIP rotavirus CH2 and SIP rotavirus CH2-CH3.
2 plants with 3 leafs ache one were infiltrated with pnos.
The same with the red(4) and blue toggle(4) and sip rativurs I DON’T KNOW
Measurement of the ODs:
Construction
OD
Volume (ml)
Sip-CH2
0.04
6.667
Sip-CH2-CH3
0.04
6.667
Red toggle (PIF6:PhyB:luc)
0.09
15
Blue toggle (LacI:KDronpa:NDronpa:ren:luc)
0.09
15.00
Renilla
0.04
6.667
Pnos
0.04
6.667
9 August 2015
Make a colony PCRs with the 10 white colonies of PhiC31 that grow in the petri dishes.
DNA- colony
JM1: 2 µl (dilution 1:10)
JM2: 2 µl (dilution 1:10)
Buffer Taq (with Mg): 2 µl
dNTPs: 2.5 µl
Taq: 0.5 µl
H2O: 1 µl
Minipreps of the liquid cultures:
LexA:AsLOVpep+ePDZ (C1-C4) C5 has turn into blue color.
LacI:AsLOVpep+ePDZ (C1-C4)
Gal4:AsLOVpep+ePDZ (C1-C4)
Digestion of the minipreps:
LexA:AsLOVpep+ePDZ
BamHI
6674, 4309
LacI:AsLOVpep+ePDZ
BamHI
6674, 5041
Gal4:AsLOVpep+ePDZ
BamHI
6674, 4270
Make the gel:
LexA:AsLOVpep+ePDZ (C1)
LexA:AsLOVpep+ePDZ (C2)
LexA:AsLOVpep+ePDZ (C3)
LexA:AsLOVpep+ePDZ (C4)
Ok
ok
ok
ok
oLacI:AsLOVpep+ePDZ (C1)
LacI:AsLOVpep+ePDZ (C2)
LacI:AsLOVpep+ePDZ (C3)
LacI:AsLOVpep+ePDZ (C4)
Ok
ok
ok
ok
Gal4:AsLOVpep+ePDZ (C1)
Gal4:AsLOVpep+ePDZ (C2)
Gal4:AsLOVpep+ePDZ (C3)
Gal4:AsLOVpep+ePDZ (C4)
Ok
ok
ok
Ok
We keep in our inventory the colony C1 of each construction
Make a gel of the colonies PCRs of PhiC31: we did not observed any DNA. Still can’t fix the problem.
Make liquid culture of renilla and PIF6:phyB:luc in Agrobacterium because the last time that we did an experiment they have grown slowly.
Store in the 4ºC fridge an agrobacterium culture with P19.
Ligations:
LexA:AsLOVpep+ePDZ+ren; ?1
LacI:AsLOVpep+ePDZ+ren; ?1
Gal4:AsLOVpep+ePDZ+ren; ?1
1µl LexA:AsLOVpep+ePDZ
1µl LacI:AsLOVpep+ePDZ
1µl Gal4:AsLOVpep+ePDZ
1 µl renilla (159)
1 µl renilla (159)
1 µl renilla (159)
1 µl ?1
1 µl ?1
1 µl ?1
4.6 µl H2O
4.6 µl H2O
4.6 µl H2O
10 August 2015
Transform into E.coli this ligations and make petri dishes cultures:
LexA:AsLOVpep+ePDZ+ren; ?1
LacI:AsLOVpep+ePDZ+ren; ?1
Gal4:AsLOVpep+ePDZ+ren; ?1
(we add into the tubes X-gal and IPTG)
Make liquid culture of 11 colonies of PhiC31 due to that this construction was giving us problems to obtain.
Refresh the agrobacterium cultures of:
BxbI:Rep:BxbI:GFP
Rep:BxbI:GFP
PhiC31
RepPhiC31:GFP
P19
Citoplasm
Integrase
Dsred
GFP
Take the samples of the agroinfiltrated plants that were in darkness for 2 days and make discs of them to change the conditions.
Make liquid culture of:
LexA:AsLOVpep+ePDZ+ren (C1 and C2)
LacI:AsLOVpep+ePDZ+ren (C1-C4)
Gal4:AsLOVpep+ePDZ+ren (C1 and C2)
(we add X-Gal and IPTG because they  are small)
11 August 2015
Primers have arrived, they have been resuspended. They were used to eliminate the recognitions sites of the enzymes in BioBricks. This will make our constructions ready to be send to the iGEM
Minipreps of the liquid cultures done yesterday:
PhiC31 (C6-C16)
LacI:AsLOVpep+ePDZ+ren (C1-C4)
Gal4:AsLOVpep+ePDZ+ren (C1 and C2)
(LexA:AsLOVpep+ePDZ+rencolonies were all blue)
Make PCRs with all the primers and constructions:
All of them follow the same composition which is:
10 µl
Buffer HF
26.5 µl
H2O
2 µl
dNTPs
0.5 µl
Taq Phusion
1 µl
DNA (dilution 1:10)
2.5 µl
Primer forward (F) (dilution 1:10)
2.5 µl
Primer reverse (R) (dilution 1:10)
*green ones are the only specify.
IFN (1)
IFN (2)
AsLOVpep (1)
AsLOVpep (2)
IFN
IFN
AsLOVpep
AsLOVpep
Mys int F
IFN domBB R
AsLOVpep F
AsLOVpep Fint
Mys int R
IFN domBB F
AsLOVpep Rint
AsLOVpep R
AsLOVpep (nested)
PhiC31 (1)
PhiC31 (2)
PhiC31 (3)
AsLOVpep
PhiC31
PhiC31
PhiC31
AsLOVpep nested
PhiC31 Fint 1
PhiC31 Fint 2
PhiC31 Fint 3
AsLOVpep
PhiC31 Rint 2
PhiC31 Rint 3
PhiC31 R
PhiC31
PhiC31
PhiC31 F
PhiC31 Rint
Digestions of the minipreps:
PhiC31
NotI
2046, 1899
LacI:AsLOVpep+ePDZ+ren
EcoRI
6345, 8798
Gal4:AsLOVpep+ePDZ+ren
EcoRI
6345, 9569
Make the gel:
PhiC31 (C6)
C7…C16
LacI:AsLOVpep:ePDZ:ren (C1)
C2
C3…C4
no
no
ok
no
ok
Gal4:AsLOVpep+ePDZ+ren (C1)
Gal4:AsLOVpep+ePDZ+ren (C2)
ok
ok
Mesurement of the ODs:
Citoplasm
0.21
7.35 ml
RepPhiC31
0.26
9.1ml
35S:LTB:T35S
0.27
9.45ml
PhiC31
0.27
9.45ml
GFP
0.20
10ml
RepBxbI
0.33
11.55ml
PsinATG:RepBxbI:GFP
0.36
12.6ml
PhiC31
0.34
11.9ml
DsRed
0.26
9.1ml
P19
0.28
9.8ml
12 August 2015
New digestions to check if the negative control constructions are correct and we can transformed it in agro
OpLexA:luc:SF:ren
EcoRI
6345, 7274
Op:UAS:luc:SF:ren
EcoRI
6345, 7096
OpEtr8.luc:SF:ren
EcoRI
6345, 7294
Gel:
Op:UAS:luc:SF:ren C1
Op:UAS:luc:SF:ren C2
Op:UAS:luc:SF:ren C3
OpEtr8.luc:SF:ren
no
no
no
no
OpLexA:luc:SF:ren C1
OpLexA:luc:SF:ren C2
Partner dna
Partner dna
no
no
Transform in Agro and make petri dishes cultures:
LexA:AsLOVpep:ePDZ
Gal4:AsLOVpep:ePDZ
LacI:AsLOVpep:ePDZ
LexA:PIF6:phyB:VP16
Gal4:PIF6:phyB:VP16
LacI:PIF6:phyB:VP16
All of them are in ?1.
Ligations:
LexA:AsLOVpep:ePDZ+ren; ?1
Gal4:AsLOVpep:ePDZ:ren+OpUAS:luc; ?1
LacI:AsLOVpep:ePDZ:ren+OpLacI:luc; ?1
1 µl LexA:AsLOVpep:ePDZ
1 µl Gal4:AsLOVpep:ePDZ:ren
1 µl LacI:AsLOVpep:ePDZ
1 µl renilla (159)
1 µl OpUAS:luc
1 µl OpLacI:luc
1 µl ?1
1 µl ?1
1 µl ?1
4.6 µl H2O
4.6 µl H2O
4.6 µl H2O
Contact
Sponsors
