<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_Valencia_UPV_Results skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:Valencia UPV/Results</SPAN></H1><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><P><TITLE>Valencia UPV iGEM 2015</TITLE></P><DIV id="page-wrapper"><HEADER id="header" class="alt"><H1><A href="https://2015.igem.org" target="_blank">iGEM 2015</A> | <A href="https://2015.igem.org/Team:Valencia_UPV">Valencia UPV</A></H1><NAV id="nav"><UL><LI><A href="https://2015.igem.org/Team:Valencia_UPV">Home</A></LI><LI><A href="https://2015.igem.org/Team:Valencia_UPV/Description" class="icon fa-angle-down">Project</A></LI><LI><A href="https://2015.igem.org/Team:Valencia_UPV/Description">Overview</A></LI><LI><A href="https://2015.igem.org/Team:Valencia_UPV/Circuit">Circuit</A></LI><LI><A href="https://2015.igem.org/Team:Valencia_UPV/Components">Components</A></LI><LI><A href="https://2015.igem.org/Team:Valencia_UPV/Modeling">Modeling</A></LI><LI><A href="https://2015.igem.org/Team:Valencia_UPV/Results">Results</A></LI><LI><A href="https://2015.igem.org/Team:Valencia_UPV/Notebook">Notebook</A></LI></UL><LI><A href="https://2015.igem.org/Team:Valencia_UPV/Modeling" class="icon fa-angle-down">Modeling</A></LI><LI><A href="https://2015.igem.org/Team:Valencia_UPV/Modeling">Overview</A></LI><LI><A href="https://2015.igem.org/Team:Valencia_UPV/Modeling/DetModel">Deterministic model</A></LI><LI><A href="https://2015.igem.org/Team:Valencia_UPV/Modeling/Simulations">Simulations</A></LI><LI><A href="https://2015.igem.org/Team:Valencia_UPV/Modeling/AlaDNA2.0">Light control</A></LI><LI><A href="https://2015.igem.org/Team:Valencia_UPV/Modeling/Conclusions">Conclusions</A></LI><LI><A href="https://2015.igem.org/Team:Valencia_UPV/Practices">Policy and Practicies</A></LI><LI><A href="https://2015.igem.org/Team:Valencia_UPV/Practices">Overview</A></LI><LI><A href="https://2015.igem.org/Team:Valencia_UPV/Practices/Minecraft">SynBiocraft</A></LI><LI><A href="https://2015.igem.org/Team:Valencia_UPV/Practices/Safety">Safety</A></LI><LI><A href="https://2015.igem.org/Team:Valencia_UPV/Collaborations">Collaborations</A></LI><LI><A href="https://2015.igem.org/Team:Valencia_UPV/Design" class="icon fa-angle-down">Magic Lamp</A></LI><LI><A href="https://2015.igem.org/Team:Valencia_UPV/Design">Overview</A></LI><LI><A href="https://2015.igem.org/Team:Valencia_UPV/Design#scroll2">Device: Description</A></LI><LI><A href="https://2015.igem.org/Team:Valencia_UPV/Design#scroll3">Device: Electronics</A></LI><LI><A href="https://2015.igem.org/Team:Valencia_UPV/Design#scroll4">How To</A></LI><LI><A href="https://2015.igem.org/Team:Valencia_UPV/OurTeam" class="icon fa-angle-down">Team</A></LI><LI><A href="https://2015.igem.org/Team:Valencia_UPV/OurTeam">Members</A></LI><LI><A href="https://2015.igem.org/Team:Valencia_UPV/Attributions">Attributions</A></LI><LI><A href="https://2015.igem.org/Team:Valencia_UPV/Attributions#scroll3">Acknowledgments</A></LI><LI><A href="https://2015.igem.org/Team:Valencia_UPV/Parts" class="icon fa-angle-down">Parts</A></LI><LI><A href="https://2015.igem.org/Team:Valencia_UPV/Achievements">Achievements</A></LI><LI><A href="https://2015.igem.org/Team:Valencia_UPV/Achievements#scroll1">Medals</A></LI><LI><A href="https://2015.igem.org/Team:Valencia_UPV/Achievements#scroll2">Achievements</A></LI></NAV></HEADER></DIV><H2>Results</H2><P>A summary of all we obtained</P><UL class="actions"><LI><A href="#scroll1" class="button">Overview</A></LI><LI><A href="#scroll2" class="button">Toggle Switches</A></LI><LI><A href="#scroll3" class="button">Recombinases</A></LI><LI><A href="#scroll4" class="button">Drug Production</A></LI><LI><A href="#scroll5" class="button">Seedlings</A></LI><LI><A href="#scroll6" class="button">Funny Work</A></LI><LI><A href="#scroll7" class="button">Constructions</A></LI></UL><SECTION id="main" class="container"><DIV id="scroll1" class="row" style="font-size:initial"><DIV class="12u"><SECTION class="box"><HEADER class="major"><H2>Overview</H2></HEADER><P>Our first achievement was the design of a biological 2N decoder capable to produce 2N outputs using only N different inputs arranged in different combinations. This design will solve the accessibility issue creating a system capable of storing information and processing it efficiently. Many complications were faced in the development of the idea, as the usage of a non material inductor in order to make it independent of matter transport or the need of memory to make the sequence of light pulses a determinant factor. To learn more about the circuit design click <A href="https://2015.igem.org/Team:Valencia_UPV/Circuit">here!</A></P><P>The designed biological circuit incorporated two toggle switches (including one de novo design!) and two recombinases that provided memory to the system. In order to prove its feasibility, the whole circuit was modelled <I> in silico</I>. The deterministic model showed excellent and promising results and predicted the optimal pulse sequence for selective production. The modeling also solved important questions as the importance of Dronpa tetramerization in the toggle implementation. In order to know more about modeling results click <A href="https://2015.igem.org/Team:Valencia_UPV/Modeling">here!</A></P><P>We also created the perfect environment for our decoder, a portable bioreactor which provide the inputs needed for correct working. To see more about our magic lamp click <A href="https://2015.igem.org/Team:Valencia_UPV/Design">here!</A></P><P>Once the circuit was designed and validated modeling, the different control elements of the circuit needed to be tested independently.You can find a description of our main results in the following sections. </P></SECTION></DIV></DIV><DIV id="scroll2" class="row" style="font-size:initial"><DIV class="12u"><SECTION class="box"><HEADER class="major"><H2>Toggle Switches</H2></HEADER><P>As we mentioned in the circuit design, essential components for our biological decoder are the blue and red light-dependent switches. These light inducible systems allow high spatiotemporal control over gene expression in order to transmit the information through the distinct circuit levels. </P><DIV style="float:right"><A href="https://2015.igem.org/Team:Valencia_UPV/Components#scrollred" class="button alt">See component</A></DIV><HEADER class="major"><H3 style="text-align:left">Red/Far-Red toggle switch</H3></HEADER><P>The red system is based on the protein Phytochrome B (PhyB) from <I>Arabidopsis thaliana</I> and its interacting factor PIF6. It is activated by light around 660 nm and it can be reversible deactivated by light around 740 nm. As previously described by Müller <I>et al</I>., the Red/Far-Red system was implemented in transient assays in tobacco protoplasts resulting in a strong induction. Thus, as a first step, we wanted to analyze its functionality employing distinct DNA binding domains (DBD) for controlling gene expression.</P><P>To test the function of the red light switch we adapted the macrolide gene expression tool based on the macrolide repressor protein (referred as E) in plant system with three different DBD, LexABD, LacIBD and Gal4BD and their operator sequences, OpLexA, OpLacI and OpUAS, repectively.</P><P>We used the GoldenBraid multipartite assembly reaction to create a construct composed by the P35S promoter, the DBD fused to PIF6 and the T35S terminator. As a result, we obtained three Transcriptional Units (TUs), each one containing a DBD mentioned above. </P><P>Simultaneously, the P35S promoter, PhyB fused to the VP16 activator domain and the T35S terminator was assembled in a multipartite reaction. After a binary assembly step for each DBD, we obtained three multigenic constructs with both transcriptional units. </P><P>On the other hand, we needed to assemble the chimeric promoter and the gene of interest for each DBD as a transcriptional unit. The chimeric promoter is composed of 3 different parts:</P><P><UL><LI>The aforementioned DBD operators (OpLexA, OpLacI and OpUAS), which are the binding sites of the corresponding binding domain.</LI><LI>A minimal promoter, miniP35S (-60), required for starting transcription.</LI><LI>The 5’-UTR region of the tobacco mosaic virus (TMV), called omega sequence. It functions as a translational enhancer in plants.</LI></UL></P><P>As we were going to perform a <A href="https://2015.igem.org/Team:Valencia_UPV/Notebook#scroll1">Luciferase assay</A> to test the switch, the gene of interest assembled with the chimeric promoter was the Luciferase.</P><P>Both constructs, containing the red toggle switch with LexA as a binding domain, and the luciferase protein as a reporter gene were codelivered into the <I>Nicotiana benthamiana</I> plant <A href="https://2015.igem.org/Team:Valencia_UPV/Notebook#scroll1">(see agroinfiltration)</A> and after two hours protoplasts were made <A href="https://2015.igem.org/Team:Valencia_UPV/Notebook#scroll1">(see protoplasts obtaining!)</A>.</P><P>The firefly luciferase reporter was followed over a time-course of 24 h by quantifying its luminescence. (Figure 1)</P><DIV style="text-align: center;"><H5><B>Figure 1. kinetics of the Red/Far-Red light-regulated gene expression system in <I>N. benthamiana</I> protoplasts. </B>Protoplasts were obtained from <I>Agrobacterium</I>-transfected leaves for light-responsive expression of firefly luciferase. Two hours after transformation, protoplast samples were illuminated with 660 nm red light and in white light during 24 hours. Control cells were incubated in the dark for the entire experiment. Firefly luciferase luminescence was quantified at the indicated points in time. </H5></DIV><P>As it can be observe, samples transfected only with the chimeric promoter and the luciferase gene construct showed basal expression, which it is maintained during all time course. In contrast, samples that continuously received 660 nm illumination began to significantly increase the luciferase luminescence after 9 hours. However, they showed a decrease in the luminescence signal after 16 hours.</P><P>Although the previous figure 1 showed that there is an increase in luciferase expression in <I>N. benthamiana</I> protoplasts, we wanted to go further and explore if this system would also work in transiently transfected plants.</P><P>To do so, the construct containing the E-PIF6 and PhyB-VP16 transcriptional units was co-transformed, together with the chimeric promoter nearby the luciferase gene transcriptional unit, by <I>Agrobacterium</I> into <I>N. benthamiana</I> leaves. After 2 days, samples were taken and the <A href="https://2015.igem.org/Team:Valencia_UPV/Notebook#scroll1">luciferase assay</A> expression was measured (Figure 2).</P><DIV style="text-align: center;"><H5><B>Figure 2. kinetics of the Red/Far-Red light-regulated gene expression system in <I>N. benthamiana</I> leaves for light-responsive expression of firefly luciferase. </B>
				Two days after transformation, discs were made and samples were illuminated with 660 nm red light and in white light during 24 hours. Control discs were incubated in the dark for the entire experiment. Firefly luciferase luminescence was quantified at the indicated points in time </H5></DIV><DIV style="float:right"><A href="https://2015.igem.org/Team:Valencia_UPV/Components#scrollblue" class="button alt">See component</A></DIV><HEADER class="major"><H3 style="text-align:left">Blue light-controlled switch</H3></HEADER><P>The blue light-dependant gene expression system based on AsLOVpep and ePDZ interaction would also be responsible to transmit the information through the decoder <A href="https://2015.igem.org/Team:Valencia_UPV/Circuit">(see our circuit model!)</A>. As for the previous described red switch, we also wanted to analyze its functionality in plants, employing the same DNA binding domains (DBD) to control gene expression, i.e., LexABD, LacIBD and Gal4BD and their operator sequences, OpLexA, OpLacI and OpUAS, repectively. </P><P>Employing the GoldenBraid assembly system we were able to create a construct composed by the Cauliflower Mosaic Virus promoter (P35S), the DBD fused to AsLOVpep and the T35S terminator. We obtained three TUs, each one containing a distinct DBD. Simultaneously, it was also assembled in a multipartite reaction a TU composed by the P35S promoter, ePDZ fused to the VP16 transcriptional activator and the T35S terminator.</P><P>Finally, after a binary assembly step with the two previously described TUs, we obtained three multigenic constructs.</P><P>As we were going to perform a </P><P>Finally, after one more GoldenBraid binary assembly step, we obtained three multigenic constructs.</P><P>In order to test the blue light-dependant switch, we perform a <A href="https://2015.igem.org/Team:Valencia_UPV/Notebook#scroll1">Luciferase assay</A>. For this purpose, the construct containing the LacI DNA binding domain was used to transform <I>N. benthamiana</I> leaves. After two days, discs were made from the leaves. And the expression levels of the luciferase reporter gene were measured by a luminometer (Figure 3). As a negative control, plants were infiltrated with LacI operator fused to the luciferase and renilla genes.</P><DIV style="text-align: center;"><H5><B>Figure 3. Luciferase assay showing the luciferase/renilla ratios of the AsLOV-ePDZ blue light-regulated gene expression system in <I>N. benthamiana</I> leaves. </B>Two days after Agrobacterium-mediated transformation, discs were made and samples were illuminated with 470 nm blue light and in white light during 48 hours. Control discs were incubated in blue light for the entire experiment. Firefly luciferase luminescence was quantified at the indicated points in time.</H5></DIV><P>It can be observed that the blue switch composed mainly by AsLOVpep and ePDZ appears to be active upon illumination with blue light and white light, when comparing it with the negative control at 48 hours. A greater induction occurs in the samples illuminated with light than in those samples illuminated with blue light. Further investigation and experiments should be done to confirm its activation in plants.</P><DIV style="float:right"><A href="https://2015.igem.org/Team:Valencia_UPV/Components#scrollcyan" class="button alt">See component</A></DIV><HEADER class="major"><H3 style="text-align:left">Violet/Cyan light-controlled switch</H3></HEADER><P>Our <I>“de novo”</I> designed blue toggle switch is based in DronpaK, a blue light–sensitive protein able to change its conformation depending on the wavelength irradiated. Upon the introduction of a K145N substitution, a homotetrameric complex is formed. When it is irradiated with 390 nm the complex monomerizes (switch on) and when irradiated with 490nm they tetramerizes again (switch off).</P><P>In order to test the functionality of our <I>”de novo”</I> designed blue light-dependent toggle switch in plants; we used GoldenBraid multipartite assembly to create a construct composed by the P35S promoter, the DBD fused to dronpaK and the T35S. Since we used three different binding domains, we obtained three different Transcriptional Units. Simultaneously, the P35S promoter, DronpaK fused to the activator domain VP16 and the T35S terminator were assembled in a multipartite reaction.</P><P>After a binary assembly stapp for each DBD, we obtained three multigenic constructs with both transcriptional units.</P><P>Moreover, we needed to assemble the chimeric promoter and the gene of firefly Luciferase, since we studied the luminescence of each sample with the luminometer.</P><P>Next, we needed to ligate the chimeric promoter and the gene of interest for each DBD as a transcriptional unit. This chimeric promoter is composed of 3 different parts:</P><P><UL><LI>The DBD operators of the binding domains above mentioned (OpLexA, OpLacI and OpUAS), which are the binding sites of the corresponding binding domain.</LI><LI>A minimal promoter, miniP35S(-60), required for starting transcription.</LI><LI>The 5’-UTR region of the tobacco mosaic virus (TMV), called omega sequence. It functions as a translational enhancer in plants.</LI></UL></P><P>The gene of interest assembled with the chimeric promoter was the Luciferase since we were going to perform a Luciferase assay to test the switch,
Both constructs, containing the blue toggle switch with LexA as a binding domain, and the luciferase protein as a reporter gene were made <A href="https://2015.igem.org/Team:Valencia_UPV/Notebook#scroll1">see protoplasts obtaining!</A>. Unfortunatelly results were not observed due to a lack of activation. In order to discard  the possibility of lack of expression, a sample of protoplasts transformed with the entire construction of the blue toggle was observed in a confocal microscope irradiating with wavelengths of 405 nm (Dronpa activation), 488nm (Dronpa deactivation).</P><DIV style="text-align: center;"><H5><B>Figure 4. Fluorescence of Dronpa protein in <I>N. benthamiana</I> protoplasts.</B>Protoplasts were obtained from <I>Agrobacterium</I>-transfected leaves. Two hours after transformation, protoplast sample was collected and observed in a confocal microscope
As it can be observed when applying a 405nm laser, fluorescence inside nucleus is observed, besides some fluorescence appears as time goes by. Nevertheless, after irradiating with 488nm the protoplast has lost the majority of its cytoplasm fluorescence as well as its nucleus fluorescence. Finally, after applying a 405nm laser again nucleus fluorescence is reestablished.</H5></DIV></SECTION></DIV></DIV><DIV id="scroll3" class="row" style="font-size:initial"><DIV class="12u"><SECTION class="box"><HEADER class="major"><H2>Recombinases</H2></HEADER> Recombinases are one of the key points of our ciruit. In order to demonstrate their correct function in the circuit, transient expression experiments in <I>N. benthamiana</I> were carried out.<DIV style="float:right"><A href="https://2015.igem.org/Team:Valencia_UPV/Components#scrollbxb1" class="button alt">See component</A></DIV><HEADER class="major"><H3 style="text-align:left">Bxb1
					</H3></HEADER><P>Since we wanted to prove that the codon-optimized recombinase Bxb1 works in <I>Nicotiana benthamiana</I>, we designed a reporter element that consists on Cauliflower Mosaic Virus terminator (T35S) flanked with bxb1 attachment sites (attB:T35S:attP:omegaUTR) which allows recognition by the recombinases and excision of the fragment flanked. </P><P>After a final binary assembly, the composite construct formed by Bxb1 recombinase TU with its reporter element TU (P35S:bxb1:T35S-P35S-attB:T35S:attP:omegaUTR-GFP:T35S),  was transformed into <I>Agrobacterium tumefaciens</I> and agroinfiltrated in <I>N. Benthamiana</I> leaves. </P><P>In addition, this construction was coinfiltrated with P19, a viral silencing repressor in order to increase the signal of the samples. Moreover we agroinfiltrated another plant just with the reporter ligated with GFP as negative control (P35S-attP:T35S:attB:omegaUTR-GFP:T35S).</P><DIV style="text-align: center;"><H5><B>Figure 5. Expression levels of GFP in <I>N. benthamiana</I> leaves. </B> A) Plant leaf transformed with the Bxb1 reporter element assembled with the promoter, GFP and the terminator. B) <I>Agrobacterium</I>-mediated transformation with the multigenic construct <A href="http://parts.igem.org/Part:BBa_K1742009">BBa_K1742009</A>, that works as a Bxb1 reporter. </H5></DIV><P>Five days after the infiltration some discs leaves were taken and observed with a confocal microscope. As it could be seen in the images above, bxb1 works in Nicotiana Benthamiana although negative control presents some basal expression.</P><DIV style="float:right"><A href="https://2015.igem.org/Team:Valencia_UPV/Components#scrollphi" class="button alt">See component</A></DIV><HEADER class="major"><H3 style="text-align:left">PhiC31
					</H3></HEADER><P>Since we wanted to prove that the codon-optimized recombinase phiC31 works in Nicotiana Benthamina, we designed a reporter element that consists on Cauliflower Mosaic Virus terminator (T35S) flanked with phiC31 attachment sites (attP:T35S:attB:omegaUTR) which allows recognition by the recombinases and excision of the fragment flanked .   </P><P>After a final binary assembly, the composite construct formed by PhiC31 recombinase TU with its reporter element TU (P35S:PhiC31:T35S-P35S-attP:T35S:attB:omegaUTR-GFP:T35S),  was transformed into <I>Agrobacterium tumefaciens</I> and agroinfiltrated in <I>N. Benthamiana</I> leaves. </P><P>In addition, this construction was coinfiltrated with P19, a viral silencing repressor in order to increase the signal of the samples. Moreover we agroinfiltrated another plant just with the reporter ligated with GFP as negative control (P35S-attP:T35S:attB:omegaUTR-GFP:T35S).</P><DIV style="text-align: center;"><H5><B>Figure 6. Expression levels of GFP in <I>N. benthamiana</I> leaves. </B> A) Agrobacterium-mediated transformation with the multigenic construct <A href="http://parts.igem.org/Part:BBa_K1742013">BBa_K1742013</A>. B) Plant leaf transformed with the PhiC31 reporter element assembled with the promoter, GFP and the terminator.</H5></DIV><P>Five days after infiltration we take some discs of leaves and observe them with a confocal microscope. As it could be seen in the images above, results were positive, PhiC31 works in <I>Nicotiana Benthamiana</I> although the negative control presents some basal expression since some fluorescent cells were spotted.</P></SECTION></DIV></DIV><DIV id="scroll4" class="row" style="font-size:initial"><DIV class="12u"><SECTION class="box"><HEADER class="major"><H2>Drug production</H2></HEADER><P>The concrete aim of AladDNA in this occasion is the production of some of the most needed drugs in the remote areas of the world. The production of our compounds was achive by transient expression in <I>Nicotiana benthamiana</I> leaves. </P><DIV style="float:right"><A href="https://2015.igem.org/Team:Valencia_UPV/Components#scrollrota" class="button alt">See component</A></DIV><HEADER class="major"><H3 style="text-align:left">Rotavirus Sip
					</H3></HEADER><P>The sSIP constructions were kindly given to us by Juarez P. We tested two different constructions carrying the light and heavy chains variable regions against rotavirus fused to either the CH2 or the CH2-CH3 domains of the constant region of human IgA heavy chain. </P><H5 style="margin-left:7em"><B>Figure 7.SIP simplificated structure</B><P style="margin-left:2em">Source:Alonso CMA <I> et al,</I>2010</P></H5><P>Two leaves of <B><I>Nicotiana benthamiana</I></B> were agroinfiltrated for each SIP version and after three days samples were collected and a <A href="https://2015.igem.org/Team:Valencia_UPV/Notebook#scroll1">western blot inmunoassay</A> was performed. Western membranes were probed with a peroxidase labelled antibody against human IgA.</P><P>As it can be observed in Fig. 8 only the SIP with the CH2 constant domain was produced at detectable levels. The recombinant protein migrated as two distinct bands with estimated molecular weights of 43 and 25 kDa. The higher molecular weight band corresponds to the complete protein. Whereas the lower molecular weight is probably the result of protein degradation that occurs either in the plant or during the extraction process. 
</P><DIV style="text-align:right;"><H5><B>Figure 8. Detection of anti-rotavirus SIP</B></H5></DIV><DIV style="float:right"><A href="https://2015.igem.org/Team:Valencia_UPV/Components#scrolllacto" class="button alt">See component</A></DIV><HEADER class="major"><H3 style="text-align:left">Lactoferrin
					</H3></HEADER><P>The lactoferrin construction was designed by us. This protein consist in two homologous domains with iron chelant capability. Although it is known that the desired antimicrobial effect is caused by the N-terminus domain, the lactoferrin has many other beneficial effects as modulator of the immune system. Therefore we decided to produce the full length protein in our device. </P><H5><B style="margin-left:9em">Figure 9. Lactoferrin 3D Structure</B></H5><P>For that purpose, we optimize the lactoferrin coding sequence for <I>N. benthamiana</I> codon usage and added a 6xHis Tag at the 3' end for subsequent protein detection. The sequence was synthesised by IDT (Integrated DNA Technologies) as two gBlocks. As a first step in the construction of the transcriptional unit, the two gBlocks were successfully assembled in a pUPD2 vector as confirmed by restriction enzyme digestion.  </P><P>The lactoferrin transcriptional unit was then built in a Golden Gate multipartite reaction using a 35S promoter, a signal peptide for secretion (tomato pectate lyase signal peptide) and a 35S terminator. Unfortunately we could not manage to complete the construction in time.</P><DIV style="float:right"><A href="https://2015.igem.org/Team:Valencia_UPV/Components#scrollinterf" class="button alt">See component</A></DIV><HEADER class="major"><H3 style="text-align:left">Interferon
					</H3></HEADER><P>The human interferon alpha 2a coding sequence was optimised for <I>N. benthamiana</I> codon usage in order to improve recombinant protein expression. Additionally, a 6xHis tag was added at the 3' end to facilitate protein detection. The optimised sequence of the recombinant protein was synthesised by IDT (Integrated DNA Technologies).  </P><P>For plant expression, the CDS was assembled with 35S promoter,  the tomato pectate lyase signal peptide (to direct it to the secretory pathway), and the 35S terminator (obtained from the Golden Braid 2.0 collection) in a multipartite assembly reaction. This construct was transformed into C58 <I>Agrobacterium tumefaciens</I> strain and used for transiently expression in <I>N. benthamiana</I>.</P><P>Infiltrated leaves were collected five days after infiltration. Production of the recombinant protein was assessed by <A href="https://2015.igem.org/Team:Valencia_UPV/Notebook#scroll1">western blot inmunoassay</A> using an anti-His antibody. As it can be seen in Figure 10, a band with the expected molecular weight of the recombinant protein (20kDa) can be detected by the anti-His antibody, confirming the identity of the band as recombinant interferon.</P><DIV style="text-align: right;"><H5><B>Figure 10.Interferon Detection by inmuno asssay</B></H5></DIV><DIV style="float:right"><A href="https://2015.igem.org/Team:Valencia_UPV/Components#scrollchol" class="button alt">See component</A></DIV><HEADER class="major"><H3 style="text-align:left">Cholera Vaccine
					</H3></HEADER><P>In order to produce a Cholera Vaccine we looked for edible vaccines capable to be produced in plants. In this research we found a study from 2004 in which a cholera vaccine was produced immunizing with a labile entherobacteria toxin from <I>E. coli.</I> This LTB is produced in plants by Kang <I>et al</I>, 2004. Using their sequence we order to synthesis a gBlock and with golden braid assembling we obtained a transcriptional unit for plants. </P><P>The LTB was also tagged with His so it was detected by <A href="https://2015.igem.org/Team:Valencia_UPV/Notebook#scroll1">inmuno assay</A> with anti His and a labeled secondary antibody. We obtained a huge expression of this protein inside the plant at the expected molecular weight as it is observed in the Figure 11. </P><DIV style="text-align: right;"><H5><B>Figure 11. Inmuno detection of cholera vaccine produced in plant leaves </B></H5></DIV><HEADER class="major"><H3 style="text-align:left">Conclusion</H3></HEADER><P>Drug production using plants as biofactories, is a real achievement. This chasis allows high production rates in an eco-friendly manner. It is also very interesting specifically in case of biodrugs as they might need some post translational modifications imposible to be achived by other species. An other huge advantage in plant productions is that they are free of human and animal viruses and diseases so there are less risks in the purification! </P><P>We achived the constructions of three from four of the designed transcriptional units for drug production. The three constructions were finely expressed in Nicothiana. However further studies are required in order to prove the biological activity of all of them.</P></SECTION></DIV></DIV><DIV id="scroll5" class="row" style="font-size:initial"><DIV class="12u"><SECTION class="box"><HEADER class="major"><H2>Seedlings</H2></HEADER><P>The circuit was designed for plant implementation so we wanted to asses which species would be better for that porpoise. In order to achive that we transciently transformed three different plant species:</P><HEADER class="major"><H3 style="text-align:left">Soy Beans</H3></HEADER><P>Soy would be an ideal chasis as they grow very fast and are known worldwide. They are approve by human consumption and what is more important germination only last one day! In our tray to agroinfiltrate them we firstly do it with comassie colorant in order to see if liquid when seedling are submitted to vacuum. As it is observed in Figure 12, the external part of the epidermis is coloured but when sections were analyzed the inner part of the stem was not coloured. </P><DIV style="text-align: center;"><H5><B>Figure 12. Soy seedling infiltrated wth Comassie blue after infliction of dermis injuries.</B></H5></DIV><P>Although the colorant did not work we tried to agroinfiltrate with <I>Agrobacterium</I> with fluorescent protein construction (DsRed) by vacuum. Seedling were observed under the Leica magnifier but no fluorescence was observed.</P><HEADER class="major"><H3 style="text-align:left">Sunflowers</H3></HEADER><P>Sunfloers are very beautifull plants known by their appereance in many famouse paintings. Sunflowers produce edible fruits with great consumption in Spanish population, however there is no tradition of sunflowers in the places where AladDNA is needed. In spite of that, the germination process is slower than soy but still short (4-5 days). </P><P>Seedlings were agroinfiltrated with DsRed fluorescent protein. Some signal was observed after three days of infiltration but very high adquisitions times were needed in order to make it visible.</P><DIV style="text-align: center;"><H5><B>Figure 13. Fluorescence observed in sunflowers after agroinfiltration with DsRed </B></H5></DIV><HEADER class="major"><H3 style="text-align:left">Spinach</H3></HEADER><P>Spinach are the terror of childs at lunch time, but they are also the favourite Popeye food. So if spinachs can turn Popeye into a strong man why not use spinach to save their lives?</P><P>They have big leaves so high expression levels could be reached. However, they present a big problem, they last 10 days in order to germinate and they were more delicated than the others.</P><P>We agoinfiltrated this seeedlings with DrRed we could see a huge amount of it, mainly concentrated in the rowels. Three days after that, samples were evaluated <IN vivo=""> in the magnifier and fluorescence was observed (Figure 14).


				</IN></P><DIV style="text-align: center;"><H5><B>Figure 14. Fluorescence observed in spinach after agroinfiltration with DsRed </B></H5></DIV><HEADER class="major"><H3 style="text-align:left">Biorreactor</H3></HEADER><P>In this part we also wanted to test the method more adeccuated for a fast and easy germination. With this porpoise, Control seed were grown in petri dishes with cotton and seed were kept in different devices with different characteristics. Some seed were submerged in water, others were kept with very very little water in the bottom of the cup and others were subjected into a device capable to keep them in the water surface. This last device was the one with better results not only for germination but also for agitation. Agitation was also tested because in the magic lamp we need it to adquire a complete irradiation of light by switches. Then the agitation system of our lamp also works in the same way as our Chocolate cup maker .</P><DIV style="text-align: center;"><H5><B>Figure 15. Seedling growth. </B>Image in the left is taken from the winner biorreactor in which seeds growth is higher and faster. Image in the right is the positive control for normal growing.</H5></DIV></SECTION></DIV></DIV><DIV id="scroll6" class="row" style="font-size:initial"><DIV class="12u"><SECTION class="box"><HEADER class="major"><H2>Funny work</H2></HEADER><P>We wanted to achieve all medal requirements so… what about having a great summer? We used agroinfiltration to free the artistic inspiration inside us! We wrote with fluorescence in leaves creating words and wonderful mosaics. </P></SECTION></DIV></DIV><DIV id="scroll7" class="row" style="font-size:initial"><DIV class="12u"><SECTION class="box"><HEADER class="major"><H2>Constructions</H2></HEADER><DIV class="table-wrapper"><TABLE class="alt"><TBODY><TR style="background-color:green;font-weight:bold"><TD>Constructions</TD><TD style="width:20em;text-align:center">Kind of part</TD></TR><TR><TD>35s:Gal4:Kdronpa:T35s</TD><TD>TU</TD></TR><TR><TD>35s:LexA:Kdronpa:T35s</TD><TD>TU</TD></TR><TR><TD>35s:LacI:Kdronpa:T35s</TD><TD>TU</TD></TR><TR><TD>35s:Ndronpa:VP16:T35s</TD><TD>TU</TD></TR><TR><TD>35s:Gal4:AsLOVpep:T35s</TD><TD>TU</TD></TR><TR><TD>35s:LexA:AsLOVpep:T35s</TD><TD>TU</TD></TR><TR><TD>35s:LacI:AsLOVpep:T35s</TD><TD>TU</TD></TR><TR><TD>35s:ePDZ:VP16:T35s</TD><TD>TU</TD></TR><TR><TD>35s:Gal4:PIF6:T35s</TD><TD>TU</TD></TR><TR><TD>35s:LexA:PIF6:T35s</TD><TD>TU</TD></TR><TR><TD>35s:LacI:PIF6:T35s</TD><TD>TU</TD></TR><TR><TD>PsinATG:RepBxbI:GFP:T35S</TD><TD>TU</TD></TR><TR><TD>PsinATG:Rep:PhiC31:GFP:T35S</TD><TD>TU</TD></TR><TR><TD>P35S:PhiC31:T35S</TD><TD>TU</TD></TR><TR><TD>P35S:PhiC31CodonOptimized:T35S</TD><TD>TU</TD></TR><TR><TD>P35S:SignalPeptide:LTB:T35S</TD><TD>TU</TD></TR><TR><TD>P35S:SignalPeptide:IFN:T35S</TD><TD>TU</TD></TR><TR><TD>P35S:LacI:Kdronpa:T35S-P35S:Ndronpa-VP16:T35S</TD><TD>Device</TD></TR><TR><TD>P35S:LexA:Kdronpa:Tnos-P35S:Ndronpa:VP16:T35S</TD><TD>Device</TD></TR><TR><TD>P35S:Gal4:Kdronpa:Tnos-P35S:Ndronpa:VP16:T35S</TD><TD>Device</TD></TR><TR><TD>P35S:LacI:Kdronpa:T35S-P35S:Ndronpa-VP16:T35S</TD><TD>Device</TD></TR><TR><TD>P35S:LacI:AsLOVpep:T35S-P35S:ePDZ:VP16:T35S</TD><TD>Device</TD></TR><TR><TD>P35S:LexA:AsLOVpep:T35S-P35S:ePDZ:VP16:T35S</TD><TD>Device</TD></TR><TR><TD>P35S:Gal4:AsLOVpep:T35S-P35S:ePDZ:VP16:T35S</TD><TD>Device</TD></TR><TR><TD>P35S:Gal4:PIF6:T35S-P35S:PhyB:VP16:T35S</TD><TD>Device</TD></TR><TR><TD>P35S:LexA:PIF6:T35S-P35S:PhyB:VP16:T35S</TD><TD>Device</TD></TR><TR><TD>P35S:LacI:PIF6:T35S-P35S:PhyB:VP16:T35S</TD><TD>Device</TD></TR><TR><TD>P35S:BxbI:T35S-PsinATG:RepBxbI:GFP:T35S</TD><TD>Device</TD></TR><TR><TD>P35S:PhiC31:T35S-PsinATG:RepPhiC31:GFP:T35S</TD><TD>Device</TD></TR><TR><TD>P35S:PhiC31CodonOptimized:T35S-PsinATG:RepPhiC31:GFP:T35S</TD><TD>Device</TD></TR><TR><TD>P35S:LexA:Kdronpa:T35S-P35S:Ndronpa-VP16:T35S-P35S:Renilla:Tnos-P35S:P19:Tnos-

				P35S:OpLexA:Luciferase:Tnos</TD><TD>Composite</TD></TR><TR><TD>P35S:Gal4:Kdronpa:T35S-P35S:Ndronpa-VP16:T35S-P35S:Renilla:Tnos-P35S:P19:Tnos-

				P35S:OpLacI:Luciferase:Tnos</TD><TD>Composite</TD></TR><TR><TD>P35S:LacI:AsLOVpep:T35S-P35S:ePDZ:VP16:T35S-P35S:Renilla:Tnos-P35S:P19:Tnos-

				OpLacI:35s:Luciferase:T35S</TD><TD>Composite</TD></TR><TR><TD>P35S:LexA:AsLOVpep:T35S-P35S:ePDZ:T35S-P35S:Renilla:Tnos-P35S:P19:Tnos-

				OpLexA:35S:luciferase:T35S</TD><TD>Composite</TD></TR><TR><TD>P35S:Gal4:AsLOVpep:T35S-P35S:ePDZ:T35S-P35S:Renilla:Tnos-P35S:P19:Tnos-

				OpGal4:35S:luciferase:T35S</TD><TD>Composite</TD></TR></TBODY></TABLE><UL class="actions" style="text-align:right"><LI><A href="https://2015.igem.org/Team:Valencia_UPV/Modeling" class="button alt">Modeling</A></LI><LI><A href="https://2015.igem.org/Team:Valencia_UPV/Design" class="button alt">Magic Lamp</A></LI><LI><A href="https://2015.igem.org/Team:Valencia_UPV/Parts" class="button alt">Parts</A></LI></UL></DIV></SECTION></DIV></DIV></SECTION><SECTION id="footb"><FOOTER id="foot"><DIV id="sponsor"><H3>Sponsors</H3></DIV><DIV id="sponsor"><H3>Contact</H3></DIV></FOOTER><A href="#" class="scrollup">Scroll</A></SECTION></DIV></DIV></DIV></DIV></BODY></HTML>