<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_UCL_Safety skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><DIV id="top_title"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:UCL/Safety</SPAN></H1></DIV><DIV id="HQ_page"><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><P><TITLE>UCL SETA - Safety</TITLE></P><DIV class="background_img_igem"><DIV class="navbarzindex"><DIV class="topnav_igem" id="myTopnav_igem"><DIV class="floatright_igem"><A href="javascript:void(0);" style="font-size:21px;" class="icon_igem" onclick="myFunction()">☰</A><DIV class="dropdown_igem"><BUTTON class="dropbtn_igem" onclick="People()">People

    </BUTTON><DIV class="dropdown-content_igem" id="dropdownpeople_igem"><A href="https://2018.igem.org/Team:UCL/Team">Team</A><A href="https://2018.igem.org/Team:UCL/Attributions">Attributions</A><A href="https://2018.igem.org/Team:UCL/Attributions#Sponsors">Sponsors</A></DIV></DIV><DIV class="dropdown_igem"><BUTTON class="dropbtn_igem" onclick="Project()">Project

    </BUTTON><DIV class="dropdown-content_igem" id="dropdownproject_igem"><A href="https://2018.igem.org/Team:UCL/Description">Description</A><A href="https://2018.igem.org/Team:UCL/Results">Results</A><A href="https://2018.igem.org/Team:UCL/Demonstrate">Demonstrate</A><A href="https://2018.igem.org/Team:UCL/InterLab">InterLab</A><A href="https://2018.igem.org/Team:UCL/Safety">Safety</A></DIV></DIV><DIV class="dropdown_igem"><BUTTON class="dropbtn_igem" onclick="Parts()">Parts

    </BUTTON><DIV class="dropdown-content_igem" id="dropdownparts_igem"><A href="https://2018.igem.org/Team:UCL/Design">Part Design</A><A href="https://2018.igem.org/Team:UCL/Parts">Parts</A><A href="https://2018.igem.org/Team:UCL/Improve">Improved Part</A></DIV></DIV><DIV class="dropdown_igem"><BUTTON class="dropbtn_igem" onclick="Modelling()">Modelling

    </BUTTON><DIV class="dropdown-content_igem" id="dropdownmodelling_igem"><A href="https://2018.igem.org/Team:UCL/Model">Modelling</A></DIV></DIV><DIV class="dropdown_igem"><BUTTON class="dropbtn_igem" onclick="Humanpractices()">Human Practices

    </BUTTON><DIV class="dropdown-content_igem" id="dropdownhumanpractices_igem"><A href="https://2018.igem.org/Team:UCL/Human_Practices">Human Practices</A><A href="https://2018.igem.org/Team:UCL/Public_Engagement">Public Engagement</A><A href="https://2018.igem.org/Team:UCL/Collaborations">Collaborations</A></DIV></DIV><DIV class="dropdown_igem"><BUTTON class="dropbtn_igem" onclick="Notebook()">Notebook

    </BUTTON><DIV class="dropdown-content_igem" id="dropdownnotebook_igem"><A href="https://2018.igem.org/Team:UCL/Notebook">Notebook</A><A href="https://2018.igem.org/Team:UCL/Achievements">Achievements</A><A href="https://2018.igem.org/Team:UCL/Experiments">Experiments</A></DIV></DIV></DIV></DIV></DIV><DIV><DIV class="container"><DIV class="container"><DIV class="h1_igem_subpage">Safety</DIV></DIV><DIV class="container containter-fluid container_igem"><DIV id="sidenav" class="sidenav wow fadeInUp"><A href="#Incorporated_Safety">Incorporated Safety</A><A href="#Personal_Data">Personal Data &amp; Surveys</A><A href="#Dual_Use">Dual Use &amp; Misuse</A><A href="#Ethics">Ethics</A><A href="#Users">Users</A><A href="#Bibliography">Bibliography</A></DIV><SECTION id="Incorporated_Safety"><DIV class="card card_igem wow fadeInUp" data-wow-duration="2s" data-wow-delay="0.1s"><DIV class="card-title text-center"><H4>Incorporated Safety</H4></DIV><DIV class="card-body text-justify"><DIV class="container"><DIV class="row"><DIV class="col-sm-10 mmmm mrfont"><P class="card-text card-text_igem text-justify"><OL><LI>
                                                    The genes included in our BioBricks do not have a role in virulence and do not pose a growth advantage to the E. coli strains used. With an exception of NickR, which on its own may aid virulence, yet is not involved directly in infection process.                                                </LI><LI>
                                                    Our chassis organisms are attenuated E. coli strains DH5-α and BL21 (DE3) commonly used in biosafety level 1 (BSL-1) laboratories and commercially available to order. With correct incubation and handling (such as wearing gloves, lab coats and goggles), they do not pose direct threat to the health of our researchers. All samples containing our chassis must be correctly isolated and disposed of to prevent release to the environment, as they could induce unwanted growth in unintended locations. To achieve this, we always dispose our cultures in sealed bin bags in the correct biocontainment bin according to the regulations of the laboratories at UCL department of Biochemical engineering.
                                                </LI><LI>
                                                    We have considered an additional level of biocontainment to our project and we  highly recommend it to the future users of our technology. We proposed using the synthetic nucleotide base chlorouracil, from the strain discovered by Marliere’s team (1) to contain our genes. This nucleotide is not found in genomes of bacteria that grow in our environment and cannot be incorporated into the genome, unless the strain has the right polymerases. As a result, genes encoded with this base cannot be transferred between strains.
                                                </LI><LI>
                                                    We have assessed our project for any additional threats and determined that the misuse of our biomaterial is highly unlikely.
                                                </LI><LI>
                                                    Our project reduces the need to use high order chassis organisms such as goats, mice, or mammalian cells to produce biomaterials, and improves insertion of synthetic genes into bacteria. Animals are used to express proteins of human or other non-bacterial origin. Spider silk proteins have been previously expressed in mammary gland cells of transgenic goats by a team led by Prof R. V. Lewis (2). However, due to the costly nature of keeping large numbers of goats and the negative public perception of transgenic mammals, we believe that bacteria are a much better chassis for our proteins.
                                                </LI><LI>
                                                    Our entire team has successfully completed lab safety training with Dr O’Brian Sullivan according to the guidelines of the UCL Biochemical Engineering department. The training included; risk assessment, fire alarm, waste disposal, safety within lab and pilot plant area, and incident reporting. Every member of our team has followed the rules thoroughly during their work within the lab for iGEM.
                                                </LI></OL></P></DIV></DIV></DIV></DIV></DIV></SECTION><SECTION id="Personal_Data"><DIV class="card card_igem wow fadeInUp" data-wow-duration="2s" data-wow-delay="0.1s"><DIV class="card-title text-center"><H4>Personal Data &amp; Surveys</H4></DIV><DIV class="card-body text-justify"><DIV class="container"><DIV class="row"><DIV class="col-sm-10 mmmm mrfont"><P class="card-text card-text_igem text-justify">
                                            All of the data were collected under with the consent of the person filling out the survey. These data are protected by the Data Protection Act 1998 (DPA) in the United Kingdom. We have followed the guidelines of the UCL Psychology department in the generation our surveys. The data collected were anonymous.
                                        </P></DIV></DIV></DIV></DIV></DIV></SECTION><SECTION id="Dual_Use"><DIV class="card card_igem wow fadeInUp" data-wow-duration="2s" data-wow-delay="0.1s"><DIV class="card-title text-center"><H4>Dua Use and Misuse of Our BioBricks</H4></DIV><DIV class="card-body text-justify"><DIV class="container"><DIV class="row"><DIV class="col-sm-10 mmmm mrfont"><P class="card-text card-text_igem text-justify">
                                            Our BioBrick parts were designed to be as safe as possible for future users. Our biomaterial, spider silk, is derived from major ampullate silk protein 1 (MaSp1) from N. clavices species. The sequence for the repetitive core of the silk protein was kindly provided by Dr R. V. Lewis (3). Additionally, we have obtained minispidroin sequences (4) for the N- and C-termini of the protein to allow production of small monomeric units of the silk at a large scale. Both Lewis’ and Andersons’ teams have previously expressed dragline spider silk proteins in E. coli on a small scale, without generating any additional safety risks than other BSL-1 organisms. The MaSp1 protein does not impose any health or biosecurity risks, including growth advantage for the strain, as it cannot be used by the chassis itself. The same goes for the genes for our flourescent proteins, such as mScarlet, which have previously been expressed in the same chassis. The major component of our BioBrick 2.0 lacZ genes are commonly found in E. coli strains and do not play a role in virulence. Inteins are parasitic genetic elements, which when expressed as proteins can self-excise and ligate the host protein without disrupting its function. We used inteins as means of polymerisation of spider silk from small monomers and platform for synthesis of other long biomaterials. Since inteins most often affect the host endonuclease, they decrease the fitness of the host and therefore are detrimental to the growth of our chassis E. coli (5).
                                        </P><P>
                                            An important consideration to the potential use of our modified chimeric silk is the addition of other proteins for functionalisation of the biomaterial. For example, proteins such as growth factors can be attached to the silk polymer for tissue regeneration. Human proteins expressed in E. coli, including estrogen binding peptide in one of our BioBricks, do not pose additional safety risks for the safety as they cannot be used by the attenuated strain for pathogenicity. In fact, recombinant human proteins expressed in E. coli, including growth factors and hormones are widely used to make biopharmaceuticals. As of 2015, nearly 400 recombinant proteins have been FDA approved and over 1300 have entered clinical or pre-clinical trials, showing that bacterial production of human proteins is very safe (6).  Nonetheless, we strongly advise to lyse the bacterial cells first before purifying the proteins to prevent contamination of human tissue with bacteria. Another protein which can be added to spider silk to give a function is NickR, which can be used to generate water filter for transition metals. NickR is a nickel-binding transcription factor, commonly found in E. coli and repressively regulates expression of the nickel ion transporter to maintain balance of nickel ions to avoid toxicity and aid hydrogenase synthesis (7). However, this protein is commonly used for virulence by E. coli and several other species including Helicobacter pyroli, which causes ulcers (8). As addition of this protein to an attenuated strain, e.g. BL21, may potentially aid in restoration of virulence, and the gene may be horizontally transferred to other species, we strongly suggest that any lab handling this protein will refer to Biosafety Level 2. This includes wearing gloves, lab coat and goggles to avoid any contact with the strain expressing this protein. To prevent containment of the environment with the strain, any equipment in contact with the strain should be autoclaved, and any colonies should be isolated and grown selectively.
                                        </P><P>
                                            Lastly, one of the  risks posed by the strain used within our lab is the vector plasmid pSB1C3, which carries a chloramphenicol resistance that can be passed on to another bacterial strain via horizontal gene transfer. To prevent this from happening, we designed an additional layer of security; we proposed the use of a strain that incorporates the synthetic nucleotide base chlorouracil into its genome. This synthetic base enables DNA replication and translation only using modified DNA and RNA polymerases, and therefore prevents the use of any gene obtained from another strain (1).   We have also selectively cultured our strains in medium with chloramphenicol to prevent the growth of unwanted strains alongside ours and to reduce the chance of horizontal gene transfer.
                                        </P></DIV></DIV></DIV></DIV></DIV></SECTION><SECTION id="Ethics"><DIV class="card card_igem wow fadeInUp" data-wow-duration="2s" data-wow-delay="0.1s"><DIV class="card-title text-center"><H4>Ethics of Our Work</H4></DIV><DIV class="card-body text-justify"><DIV class="container"><DIV class="row"><DIV class="col-sm-10 mmmm mrfont"><P class="card-text card-text_igem text-justify">
                                            To assess the ethics and impact of our technology and genetic engineering on the society, we have collaborated with 2debate platform to create an online debate “Should genetic engineering be accessible to the general public?” (9). The recording is more than an hour long and introduces genetic engineering in a friendly manner to the general public. During the debate many aspects of genetic engineering were discussed including biocontainment and contamination of environment with engineered organisms (GMOs), public perception of GMO products and policies across different countries to regulate science. This debate has educated our team in terms of how science can have a big influence on the society.  Even if a technology holds no risks, it may still take time for it to be approved by general public and incorporated into daily use. This debate has encouraged us to do more human integrated practices to show that science is for the society.
                                        </P><P>
                                            The aim of our project was to generate an easily accessible platform for the production of biomaterials and to reduce the need for animals or plants in their production. We have chosen Escherichia coli as our chassis, as its protein expression has been well studied and commercial strains are easily accessible for any wet lab. In 1959 Russell and Burch introduced their principles of the 3Rs regarding animal experimentation; replace, refine and reduce. Since then, scientists aimed to find alternative methods to reduce the use of animals for experimentation and production of proteins. The general public agrees with the notion that the use of animals in laboratories should be limited and their living conditions should be designed to prevent senseless suffering (10). Thanks to the expression of spider silk in bacteria, we prevent the need for organisms such as mammals and human cells to produce biomaterial proteins.  The choice of chassis for protein expression is very important. Higher order organisms often are protected by tight regulations to prevent unnecessary pain and suffering; they also need special conditions and care, which costs money and time (10). Spiders are difficult organisms to culture due to their territorial behaviour. Furthermore, the process of silking spiders directly is impractical, time consuming, and requires large amounts of manual labour. To obtain an 11-foot by 4-foot textile of spider silk, it would take 4 years, hundreds of people and over 1 million N. clavispes spiders. Immobilisation of the spiders is required for silking, which can cause discomfort and stress for the animal (11).  One of the best alternatives is the use of non-sentient organisms, and E. coli is perfect for such purpose (4). E. coli can be cultured at much larger numbers than mammals and other multicellular organisms, and therefore can produce a much greater amount of spider silk. In a single culture of 200 ml, there are over billion bacterial cells depending on the growth of the population.
                                        </P></DIV></DIV></DIV></DIV></DIV></SECTION><SECTION id="Users"><DIV class="card card_igem wow fadeInUp" data-wow-duration="2s" data-wow-delay="0.1s"><DIV class="card-title text-center"><H4>Users of Our BioBricks</H4></DIV><DIV class="card-body text-justify"><DIV class="container"><DIV class="row"><DIV class="col-sm-10 mmmm mrfont"><P class="card-text card-text_igem text-justify">
                                            Our Biobrick 2.0, Intein Passenger, and silk biobricks were designed to increase accessibility of synthetic biology cloning techniques and enable large scale production of biomaterials. Our intended direct users of BioBricks are future iGEM teams and synthetic biology engineers at scientific institutes. We highly recommend that these users follow the safety procedures for biosafety levels 1 and 2, to prevent contamination of the environment with the modified chassis. Future users should take full responsibility of their application of our platform to produce biomaterials; we cannot predict all possible applications and therefore cannot deduce all preventional methods effectively. For example, if our spider silk will be used to treat patients for tissue regeneration, then patients should be fully informed about the material and, if they wish to choose alternatives, should be given a free choice.                                        </P></DIV></DIV></DIV></DIV></DIV></SECTION><SECTION id="Bibliography"><DIV class="card card_igem wow fadeInUp" data-wow-duration="2s" data-wow-delay="0.1s"><DIV class="card-title text-center"><H4>Bibliography</H4></DIV><DIV class="card-body text-justify"><DIV class="container"><DIV class="row"><DIV class="col-sm-10 mmmm mrfont"><P class="card-text card-text_igem text-justify"><OL><LI>
                                                    Marliere P, Patrouix J, Doring V, Herdewijn P, Tricot S, Cruveiller S, et al. Chemical evolution of a bacterium's genome. Angew Chem Int Ed Engl. 2011;50(31):7109-14.
                                                </LI><LI>
                                                    Rutherforfd A. The goats with spider genes and silk in their milk BBC News: BBC News; 2012 [Available from: https://www.bbc.co.uk/news/av/science-environment-16554357/the-goats-with-spider-genes-and-silk-in-their-milk.
                                                </LI><LI>
                                                    Teule F, Cooper AR, Furin WA, Bittencourt D, Rech EL, Brooks A, et al. A protocol for the production of recombinant spider silk-like proteins for artificial fiber spinning. Nat Protoc. 2009;4(3):341-55.
                                                </LI><LI>
                                                    Andersson M, Jia Q, Abella A, Lee XY, Landreh M, Purhonen P, et al. Biomimetic spinning of artificial spider silk from a chimeric minispidroin. Nat Chem Biol. 2017;13(3):262-4.
                                                </LI><LI>
                                                    Gogarten JP, Hilario E. Inteins, introns, and homing endonucleases: recent revelations about the life cycle of parasitic genetic elements. BMC Evol Biol. 2006;6:94.
                                                </LI><LI>
                                                    Sanchez-Garcia L, Martin L, Mangues R, Ferrer-Miralles N, Vazquez E, Villaverde A. Recombinant pharmaceuticals from microbial cells: a 2015 update. Microb Cell Fact. 2016;15:33.
                                                </LI><LI>
                                                    Sheila C. Wang AVD, Stephanie L. Bloom, and Deborah B. Zamble. Selectivity of Metal Binding and Metal-Induced Stability of Escherichia coli NikR. Biochemistry. 2004;43(1):10018-28.
                                                </LI><LI>
                                                    Higgins KA, Carr CE, Maroney MJ. Specific metal recognition in nickel trafficking. Biochemistry. 2012;51(40):7816-32.
                                                </LI><LI>
                                                    Primbs ST-CD. 2D54 – GENETIC ENGINEERING SHOULD BE ACCESSIBLE TO THE PUBLIC! 2018 [Online Debate Podcast]. Available from: <A class="sshh" href="https://2debate.eu/genetic-engineering-should-be-accessible-to-the-public/">2debate</A></LI><LI>
                                                    Replacement, Reduction and Refinement [press release]. Altex: Altex2002.
                                                </LI><LI>
                                                    Perkel J. Cell culture's spider silk road. BioTechniques. 2018;53(6):284-8.
                                                </LI></OL></P></DIV></DIV></DIV></DIV></DIV></SECTION></DIV></DIV></DIV><DIV class="footer"><DIV class="container"><DIV class="row"><DIV class="col-lg-4"><CENTER><DIV class="h2_footer_igem">Our Location</DIV>
                        University College London
      Gower St, BloomsburyLondon WC1E 6BT
                        </CENTER></DIV><DIV class="col-lg-4"><CENTER><DIV class="h2_footer_igem">Contact Us</DIV>
                      ucl.igem@ucl.ac.uk
                        </CENTER></DIV><DIV class="col-lg-4"><CENTER><DIV class="h2_footer_igem">Follow Us</DIV></CENTER></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></BODY></HTML>