<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_UCL_Design skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><DIV id="top_title"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:UCL/Design</SPAN></H1></DIV><DIV id="HQ_page"><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><P><TITLE>UCL SETA - Part Design</TITLE></P><DIV class="background_img_igem"><DIV class="navbarzindex"><DIV class="topnav_igem" id="myTopnav_igem"><DIV class="floatright_igem"><A href="javascript:void(0);" style="font-size:21px;" class="icon_igem" onclick="myFunction()">☰</A><DIV class="dropdown_igem"><BUTTON class="dropbtn_igem" onclick="People()">People

                        </BUTTON><DIV class="dropdown-content_igem" id="dropdownpeople_igem"><A href="https://2018.igem.org/Team:UCL/Team">Team</A><A href="https://2018.igem.org/Team:UCL/Attributions">Attributions</A><A href="https://2018.igem.org/Team:UCL/Attributions#Sponsors">Sponsors</A></DIV></DIV><DIV class="dropdown_igem"><BUTTON class="dropbtn_igem" onclick="Project()">Project

                        </BUTTON><DIV class="dropdown-content_igem" id="dropdownproject_igem"><A href="https://2018.igem.org/Team:UCL/Description">Description</A><A href="https://2018.igem.org/Team:UCL/Results">Results</A><A href="https://2018.igem.org/Team:UCL/Demonstrate">Demonstrate</A><A href="https://2018.igem.org/Team:UCL/InterLab">InterLab</A><A href="https://2018.igem.org/Team:UCL/Safety">Safety</A></DIV></DIV><DIV class="dropdown_igem"><BUTTON class="dropbtn_igem" onclick="Parts()">Parts

                        </BUTTON><DIV class="dropdown-content_igem" id="dropdownparts_igem"><A href="https://2018.igem.org/Team:UCL/Design">Part Design</A><A href="https://2018.igem.org/Team:UCL/Parts">Parts</A><A href="https://2018.igem.org/Team:UCL/Improve">Improved Part</A></DIV></DIV><DIV class="dropdown_igem"><BUTTON class="dropbtn_igem" onclick="Modelling()">Modelling

                        </BUTTON><DIV class="dropdown-content_igem" id="dropdownmodelling_igem"><A href="https://2018.igem.org/Team:UCL/Model">Modelling</A></DIV></DIV><DIV class="dropdown_igem"><BUTTON class="dropbtn_igem" onclick="Humanpractices()">Human Practices

                        </BUTTON><DIV class="dropdown-content_igem" id="dropdownhumanpractices_igem"><A href="https://2018.igem.org/Team:UCL/Human_Practices">Human Practices</A><A href="https://2018.igem.org/Team:UCL/Public_Engagement">Public Engagement</A><A href="https://2018.igem.org/Team:UCL/Collaborations">Collaborations</A></DIV></DIV><DIV class="dropdown_igem"><BUTTON class="dropbtn_igem" onclick="Notebook()">Notebook

                        </BUTTON><DIV class="dropdown-content_igem" id="dropdownnotebook_igem"><A href="https://2018.igem.org/Team:UCL/Notebook">Notebook</A><A href="https://2018.igem.org/Team:UCL/Achievements">Achievements</A><A href="https://2018.igem.org/Team:UCL/Experiments">Experiments</A></DIV></DIV></DIV></DIV></DIV><DIV class="container container_igem"><DIV class="container container_igem"><DIV class="h1_igem_subpage">Part Design</DIV></DIV><DIV class="container container_igem"><DIV id="sidenav" class="sidenav wow fadeInUp" data-wow-duration="2s" data-wow-delay="0.1s"><A href="#BB">BioBrick 2.0</A><A href="#IM">Intein Monomers</A><A href="#SC">Spidroin Constructs</A><A href="#IP">Intein Passenger</A><A href="#NIKR">NikR</A><A href="#ER">ER</A></DIV><SECTION id="BB"><DIV class="card card_igem wow fadeInUp" data-wow-duration="2s" data-wow-delay="0.1s"><DIV class="card-title text-center"><H4>BioBrick 2.0</H4></DIV><DIV class="card-body text-justify"><DIV class="container"><DIV class="row"><DIV class="col-sm-10 mmmm mrfont"><DIV class="card-title text-center"><DIV class="h4_cards">Overview</DIV></DIV><P class="card-text card-text_igem text-justify">
                                        While developing our modular platform, we conceived a BioBrick-compatible standard with improved flexibility that enables the integration of conventional cloning methods into iGEM’s workflow. This construct, once inserted into a backbone, allows cloning through Golden Gate assembly and Gibson assembly. At the same time, our construct has a LacZ reporter which can be used to screen plates for successful colonies. We are proposing our BioBrick2.0 as a new standard, which could facilitate an improved cloning process for future iGEM teams.
                                    </P><DIV class="card-title text-center"><DIV class="h4_cards">Reporter</DIV></DIV><P class="card-text card-text_igem text-justify">
                                        The lacZ reporter allows for blue-white screening. It can be displaced by compatible constructs through BioBrick assembly, Golden Gate assembly or Gibson assembly. Blue-white screening is based on the inability of commercial E. coli to metabolise galactose or structurally similar substrates like X-Gal because of a lacZ deletion mutation. Once lacZα is provided through a plasmid, X-Gal can be metabolised to an easily detectable blue compound. This screening can also be used for subsequent cloning where the displacement of lacZ results in white colonies.
                                    </P><DIV class="card-title text-center"><DIV class="h4_cards">Compatibility</DIV></DIV><P class="card-text card-text_igem text-justify">For Golden Gate assembly, BsaI cut sites and defined overhangs were added. The BsaI overhangs are based on Vladimir et al. 2018 [1]. CCAA was chosen for upstream and ACGG for downstream. For Gibson assembly, a defined area of homology was designed. BsaI can be used to open the plasmid, removing the reporter and exposing the Gibson overhangs. </P><DIV class="card-title text-center"><DIV class="h4_cards">BioBrick2.0 standardisation</DIV></DIV><P class="card-text card-text_igem text-justify">One of the oldest problems of biological research which persists nowadays is the lack of standardization in experimental design, which impacts reproducibility and therefore reliability of results. Especially with the advent of DNA cloning, sequencing and omics techniques, there has been a growing need for a connecting thread which would simplify the bundle of data generated by these new experimental techniques. Arkin and Endy from MIT were the first ones who tried to address the issue by publishing a standard registry of biological parts which didn’t have considerable success [1]. However, Tom Knight, one of their colleagues, who at the time was working in an artificial intelligence laboratory, proposed in 2003 a new standard called Biobrick [2]. BioBricks are DNA sequences which conform to a restriction enzyme assembly standard, building blocks which can be assembled in larger synthetic biological circuits from individual parts.</P><P class="card-text card-text_igem text-justify">Over the years the Biobrick assembly has shown various flaws. First of all, it doesn’t allow for assembly of multiple parts in one reaction and it cannot be used for fusion proteins. Secondly, it produces a scar sequence which can be harmful for E. coli [3]. Even though, some of these issues have been addressed by improved methodologies such as the Silver or the 3A [4], [5], they are unnecessarily laborious as they require additional pre-screenings or, in the case of 3A, three different antibiotics.
                                    </P><P class="card-text card-text_igem text-justify">Our BioBrick2.0 construct integrates different cloning techniques, such as Gibson assembly and Golden Gate assembly, into the iGEM standard. When cloned into a backbone through BioBrick Assembly, it enables the other cloning methods to be used inside the BioBrick2.0 sequence. The Gibson overlapping regions allow for high-fidelity cloning of constructs up to thousands of kilobases, as shown previously [6]. The Golden Gate recognition sites allow for seamless cloning and, applying methods such as MoClo and Golden Braid, this procedure can be carried out with multiple fragments in one single reaction [7] [8].</P><P class="card-text card-text_igem text-justify">The purpose of the BioBrick is not to create a new standard because, as it happened before in synthetic biology, an excess of attempts at creating new standards can become detrimental [9]. As a consequence, our brick is meant to be a platform to BioBrick standardize constructs cloned using different methods to accommodate different labs’ preferences and boost the iGEM workflow.
                                    </P></DIV></DIV></DIV></DIV></DIV></SECTION><SECTION id="IM"><DIV class="card card_igem wow fadeInUp" data-wow-duration="2s" data-wow-delay="0.1s"><DIV class="card-title text-center"><H4>Intein Monomers</H4></DIV><DIV class="card-body text-justify"><DIV class="container"><DIV class="row"><DIV class="col-sm-10 mmmm mrfont"><DIV class="card-title text-center"><DIV class="h4_cards">Intein Monomer 1 and Intein Monomer 2</DIV></DIV><P class="card-text card-text_igem text-justify">


                                        Initially we planned to clone constructs that would allow us to produce long spider silk proteins which could then be functionalised with proteins of interest using SpyTag/SpyCatcher technology. We later realised that instead of restricting ourselves to just one protein - spider silk protein, we could develop a much broader platform  to polymerise any protein of interest. To achieve this we thought about using split inteins for  polymerisation. We designed two constructs ,Bba_K2842680 and Bba_K2842690 that could work together to polymerise proteins via intein trans-splicing. Each construct is made of an Intein C termini, a fluorescent reporter protein, a StrepTag and an intein N termini.
                                        We introduced Type II restriction sites sites that could be used to introduce any protein of interest into the construct for polymerisation.
                                    </P><DIV class="card-title text-center"><DIV class="h4_cards">Design Details</DIV></DIV><P class="card-text card-text_igem text-justify"><B>Inteins</B>: Each construct contains orthogonal inteins on each side preventing self reaction and circularisation. The orthogonal inteins mean that polymerisation will only occur when both constructs are expressed together or when the two proteins are combined.</P><P class="card-text card-text_igem text-justify"><B>Reporters</B>: One construct contains GFP and the other mRFP1. This will enable easy identification of cross polymerisation by fluorescence measurements.</P><P class="card-text card-text_igem text-justify"><B>Modular design</B>: SapI sites were placed immediately upstream and downstream of the reporter proteins. The codons used were designed to be as innocuous as possible.</P><P class="card-text card-text_igem text-justify"><B>Other elements</B></P><P class="card-text card-text_igem text-justify">Promotor: A T7 promoter was used
                                        RBS: We used a RBS based on predictions by sallis lab <SUP>[1,2]</SUP>
                                        Regulatory elements: We introduced a lac operator to further control gene expression if the T7 promoter was too strong or detrimental to cell health.
                                    </P></DIV></DIV></DIV></DIV></DIV></SECTION><SECTION id="SC"><DIV class="card card_igem wow fadeInUp" data-wow-duration="2s" data-wow-delay="0.1s"><DIV class="card-title text-center"><H4>Spidroin Constructs</H4></DIV><DIV class="card-body text-justify"><DIV class="container"><DIV class="row"><DIV class="col-sm-10 mmmm mrfont"><DIV class="card-title text-center"><DIV class="h4_cards">Spidroin Constructs</DIV></DIV><P class="card-text card-text_igem text-justify">
                                        A spidroin is a protein belonging to the family of scleroproteins, which includes collagen and keratin. It is the main building block of spiders’ dragline silk; its tensile properties exceed steel and kevlar and, at the same time, it is more flexible than both of these material as it can be stretched up to 135% of its original length.
                                    </P><P class="card-text card-text_igem text-justify">The amino acid composition is largely dominated by alanine, found in the crystalline regions of the protein, and glycine blocks found in the amorphous matrix. The alternation of the crystalline and amorphous regions, formed by helical structures and beta turns, is what gives spidersilk its incredible properties. However, the protein by itself is a monomer, therefore in order to form a strong fiber, it has to form an oligomer. The C-termini in solution associate by covalent bond and with a change in pH, which in vivo is carried by transport into a different gland The N-termini form hydrogen bonds and complete the oligomerization.
                                    </P><P class="card-text card-text_igem text-justify">Our construct, which we split into two for simplicity of assembly, is composed by the spidroin core with four total repetitive regions, two in one construct and two in the other, and the C-and N-termini, distinguishing the two constructs. Moreover, to enable characterisation of the fiber, we flanked the C-terminus of the sequence with the N-terminus of the AceL-TerL intein which would splice out when associated with its compatible counterpart.
                                    </P><P class="card-text card-text_igem text-justify">The assembly of this construct can be carried out through Gibson assembly or Golden Gate into our BioBrick2.0 construct to standardise it for Bobrick assembly. Finally, the fibre can also be modified to test different properties through assembly of additional repetitive regions into the core of the sequence. This is made possible by our SapI overhangs which are compatible with the ones on our Spidroin Core construct.  </P></DIV></DIV></DIV></DIV></DIV></SECTION><SECTION id="IP"><DIV class="card card_igem wow fadeInUp" data-wow-duration="2s" data-wow-delay="0.1s"><DIV class="card-title text-center"><H4>Intein Passenger</H4></DIV><DIV class="card-body text-justify"><DIV class="container"><DIV class="row"><DIV class="col-sm-10 mmmm mrfont"><DIV class="card-title text-center"><DIV class="h4_cards">Overview of Intein Passenger</DIV></DIV><P class="card-text card-text_igem text-justify">
                                        The intein passenger construct is the framework used to insert functional proteins into our highly modular platform. The design of the Biobrick allows it to flank any protein with inteins and, therefore, to functionalise biomaterials with a high degree of freedom. Design notes:

                                    </P><DIV class="card-title text-center"><DIV class="h4_cards"><B>Intein</B></DIV></DIV><P class="card-text card-text_igem text-justify">The choice of intein is important because we need to leave as small a fragment as possible at the sister construct.  We picked the AceL-TerL intein (DOI 10.1002/anie.201307969) since its C-terminal section is 125 amino acids long and the N-terminal section only 11. </P><DIV class="card-title text-center"><DIV class="h4_cards"><B>Reporter</B></DIV></DIV><P class="card-text card-text_igem text-justify">mScarlet was chosen as it is one of the brightest RFPs available and it is a fast folder.</P><DIV class="card-title text-center"><DIV class="h4_cards"><B>Golden Gate Set-Up</B></DIV></DIV><P class="card-text card-text_igem text-justify">To make it compatible with the Golden Gate assembly approach incorporated in the BioBrick2.0, we decided to use SapI. The chosen design allows the lowest impact on the linker sequence. An advantage of using SapI is that any downstream design will be simpler by maintaining the overhang in frame. SapI sites were coded as GRA and SSS. It uses GGC (upstream) and GAG (downstream) overhangs.</P><DIV class="card-title text-center"><DIV class="h4_cards"><B>Promoter
                                        </B></DIV></DIV><P class="card-text card-text_igem text-justify">We needed high-level expression but too much expression would make  the spider silk insoluble, so we used a T7 promote with a middle of the road RBS. This is based on part BBa_K525998 (T7 promoter and RBS).</P><DIV class="card-title text-center"><DIV class="h4_cards"><B>Terminator</B></DIV></DIV><P class="card-text card-text_igem text-justify">The BBa_B1006 terminator was chosen because of its termination probability above 90%  and the many reported cases.
                                    </P><DIV class="card-title text-center"><DIV class="h4_cards"><B>Additional regulation
                                        </B></DIV></DIV><P class="card-text card-text_igem text-justify">We picked a lac operator for improved silencing and reducing leaky expression situated downstream of the T7 promoter.</P><DIV class="card-title text-center"><DIV class="h4_cards"><B>Spacers
                                        </B></DIV></DIV><P class="card-text card-text_igem text-justify">Spacers were added based on estimates of translation calculated on Salis Lab RBS calculator.
                                        </P></DIV></DIV></DIV></DIV></DIV></SECTION><SECTION id="NIKR"><DIV class="card card_igem wow fadeInUp" data-wow-duration="2s" data-wow-delay="0.1s"><DIV class="card-title text-center"><H4>NikR</H4></DIV><DIV class="card-body text-justify"><DIV class="container"><DIV class="row"><DIV class="col-sm-10 mmmm mrfont"><P class="card-text card-text_igem text-justify">
                                            Nickel is a naturally occurring substance in ground water; in very low concentrations it is beneficial to the growth of plants. Factory waste and sewage disposal can result in leakage of this substance, which increases the concentration in water making it dangerous to humans. As a solution, we hereby propose a Nickel binding construct integrated with our project.
                                        </P><P class="card-text card-text_igem text-justify">A hexahistidine tag is the most obvious Nickel binding protein available but it does not have a particularly high affinity. In solution, Ni-NTA has an affinity of 10μM for a single hexahistidine tag. A double histidine tag (6xHis + short spacer + 6xHis) showed an affinity nearly 10 times higher. However, an even higher binding affinity is achieved by the enzyme NikR which is a transcription activator that responds to Nickel. Additionally, E. coli has a NikR enzyme that  is very <A href="https://www.uniprot.org/uniprot/P0A6Z6">well characterised</A>.
                                        </P><P class="card-text card-text_igem text-justify">NikR is a small 133 amino acid long molecule of known structure [1] with an affinity of 6.8 pM. It has two domains and the C-terminal domain alone, which is made of 86 residues from Gly48 to Asp133, is sufficient to bind nickel tightly at 2.2 pM [2].
                                        </P><P class="card-text card-text_igem text-justify">We inserted NikR in our BioBrick2.0 construct to standardise it. This can now be used to modularly characterise biomaterials through our innovative system. Materials functionalised with our NikR construct could be applied as filters to purify water from Nickel which is a very common pollutant. Together with our endocrine disruptor construct, these are an example of how our plug and play split inteins system is not only a method that could greatly help in research, but it is also aimed at real world applications that could affect millions of people.
                                        </P></DIV></DIV></DIV></DIV></DIV></SECTION><SECTION id="ER"><DIV class="card card_igem wow fadeInUp" data-wow-duration="2s" data-wow-delay="0.1s"><DIV class="card-title text-center"><H4>Estrogen Receptor</H4></DIV><DIV class="card-body text-justify"><DIV class="container"><DIV class="row"><DIV class="col-sm-10 mmmm mrfont"><P class="card-text card-text_igem text-justify">
                                            Endocrine disruptors are chemical substances that can interact with parts of the endocrine systems and, at high concentrations, compromise their functions. Dangerous quantities of these chemicals have been registered multiple times in sources of water that are accessible to the public like rivers and springs. The concentration of estradiol 2 (17 β-estradiol) in the river Ahr at Blankenheim is on the order of 4–12 ng/L and 122–631 ng/L in Rio de la Plata, Argentina[1]. A concentration of 15.1 ± 0.3 ng/L of E2 was measured in a sample obtained from a food production process, and 8.8 ± 0.7 ng/L in a sample from the Adriatic Sea[2].
                                        </P><P class="card-text card-text_igem text-justify">In order to deal with this health and environmental risk, we developed an endocrine disruptor binder that works with our intein passenger system. The construct is based on the M421F mutated version of the estrogen receptor protein which was shown have higher binding affinities and stability than the normal enzyme[3]. These characteristics make it a well performing potential bionsensor as suggested by Ferrero et al. </P><P class="card-text card-text_igem text-justify">We inserted ER in our Final Brick to standardise it and this can now be used to modularly characterise biomaterials through our innovative system. Materials characterised with our ER construct could be applied as filters to purify water from dangerous quantities of estrogen which is a very common pollutant. Together with our nickel binding construct, these are an example of how our plug and play split inteins system is not only a method that could greatly help in research, but it is also aimed at real world applications which could affect millions of people.
                                        </P></DIV></DIV></DIV></DIV></DIV></SECTION></DIV></DIV></DIV></DIV><DIV class="footer"><DIV class="container"><DIV class="row"><DIV class="col-lg-4"><CENTER><DIV class="h2_footer_igem">Our Location</DIV>
                    University College London
                    Gower St, BloomsburyLondon WC1E 6BT
                    </CENTER></DIV><DIV class="col-lg-4"><CENTER><DIV class="h2_footer_igem">Contact Us</DIV>
                    ucl.igem@ucl.ac.uk
                    </CENTER></DIV><DIV class="col-lg-4"><CENTER><DIV class="h2_footer_igem">Follow Us</DIV></CENTER></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></BODY></HTML>