<HTML xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en" dir="ltr">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki  ltr ns-0 ns-subject page-Team_BIT-China_modeling_html"><DIV id="globalWrapper"><DIV id="top-section"><DIV id="p-logo"><A href="/Main_Page" title="Main Page">&quot;
	    </A></DIV><DIV id="menubar" class="left-menu noprint"><UL><LI class="selected"><A href="/Team:BIT-China/modeling.html">Page               </A></LI><LI class="new"><A href="/wiki/index.php?title=Talk:Team:BIT-China/modeling.html&amp;action=edit&amp;redlink=1">Discussion               </A></LI><LI><A href="/wiki/index.php?title=Team:BIT-China/modeling.html&amp;action=edit">View source               </A></LI><LI><A href="/wiki/index.php?title=Team:BIT-China/modeling.html&amp;action=history">History               </A></LI><LI style="color:white;cursor:default">teams</LI></UL></DIV><DIV class="right-menu noprint" id="menubar"><UL><LI id="pt-login"><A href="/wiki/index.php?title=Special:UserLogin&amp;returnto=Team:BIT-China/modeling.html" title="You are encouraged to log in; however, it is not mandatory [o]" accesskey="o">Log in</A></LI></UL></DIV><DIV id="search-controls" class="noprint"><FORM action="/Special:Search" id="searchform"> </FORM></DIV></DIV><DIV id="content"><H1 class="firstHeading">Team:BIT-China/modeling.html</H1><DIV id="bodyContent"><H3 id="siteSub" class="noprint">From 2013.igem.org</H3><P><TITLE>MODELING | BIT-CHINA IGEM2013</TITLE></P><DIV id="wrapper"><DIV id="navigation"><DIV class="navigation"><DIV class="centering"><UL class="topnav" id="topnav"><LI class="navitem1"><A href="./index.html">HOME</A></LI><LI class="navitem2"><A href="./project.html">PROJECT</A></LI><LI><A href="./project.html#overview">Overview</A></LI><LI><A href="./project.html#HR">Heat-resistant system</A></LI><LI><A href="./project.html#QC">Quorum-control system</A></LI><LI><A href="./project.html#parts">Parts</A></LI><LI><A href="./project.html#co">Cooperation</A></LI><LI><A href="./project.html#safety">Safety</A></LI></UL><LI class="navitem3"><A href="./team.html">TEAM</A></LI><LI><A href="./team.html#team_members">Members</A></LI><LI><A href="https://igem.org/Team.cgi?id=1117">Official Profile</A></LI><LI><A href="./team.html#About_the_University">About the University</A></LI><LI class="navitem4"><A href="./modeling.html">MODELING</A></LI><LI><A href="./modeling.html#overview">Overview</A></LI><LI><A href="./modeling.html#Noise_in_the_cell">Noise in the cell</A></LI><LI><A href="./modeling.html#Quorum_Control_system">Quorum control system</A></LI><LI><A href="./modeling.html#Heat_Resistant_system">Heat Resistant system</A></LI><LI class="navitem5"><A href="./humanpractice.html">HUMAN PRACTICE</A></LI><LI><A href="./humanpractice.html#Meaning">Meaning</A></LI><LI><A href="./humanpractice.html#C">Communication</A></LI><LI><A href="./humanpractice.html#pro">Promotion</A></LI><LI class="navitem6"><A href="./notebook.html">NOTEBOOK</A></LI><LI><A href="./notebook.html#">Journal</A></LI><LI class="navitem7"><A href="./sponser.html">SPONSER</A></LI><LI class="navitem8"><A href="https://2013.igem.org">IGEM</A></LI></DIV></DIV></DIV></DIV><DIV id="width_wrapper"><DIV id="sidebar" style="left: 0px"><DIV class="list" id="modeling"><DIV class="temper"><DIV class="model_height"><DIV class="temp-body-modeling"><DIV class="color-modeling"><DIV id="mercury"><DIV class="temp tem37 heartbeat">37℃</DIV><DIV class="temp tem40 heartbeat">40℃</DIV><DIV class="temp tem43 heartbeat">43℃</DIV></DIV></DIV></DIV></DIV></DIV><DIV class="navis"><H2><A href="javascript:void(0);" onclick="temper(0, '#modeling')">Modeling</A></H2><UL><LI><A href="javascript:void(0);" onclick="temper(1,'#overview')">Overview</A></LI><LI><A href="javascript:void(0);" onclick="temper(2,'#Noise_in_the_cell')">Noise in the cell</A></LI><LI><A href="javascript:void(0);" onclick="temper(3,'#Quorum_Control_system')">Quorum Control system</A></LI><LI><A href="javascript:void(0);" onclick="temper(5,'#qsd')">Quorum sensing device</A></LI><LI><A href="javascript:void(0);" onclick="temper(6,'#rd')"> Oscillator device</A></LI><LI><A href="javascript:void(0);" onclick="temper(7,'#pc')">Programmed cell death device</A></LI></UL><LI><A href="javascript:void(0);" onclick="temper(9,'#Heat_Resistant_system')">Heat Resistant system</A></LI><LI><A href="javascript:void(0);" onclick="temper(10,'#des')">Description</A></LI><LI><A href="javascript:void(0);" onclick="temper(11,'#si')">Simulation results</A></LI><LI><A href="javascript:void(0);" onclick="temper(12,'#re')">Reference</A></LI></DIV></DIV></DIV><DIV class="section pagetitle modeling"><H1>Overview</H1></DIV><DIV class="section hero"><DIV class="content"><DIV class="word-title">Why modelling?</DIV><P>Computational and mathematical models are useful and effective tools for understanding biology. Molecular biology experiments in vitro are very expensive and time consuming, building models of biological processes as a preliminary step helps to circumvent some of the drawbacks of performing hypothesis-testing in the wet lab. Using models could provide a faster and cheaper shortcut for biologists to gain a better understanding of the newly engineered system.Particularly with the extent of synthetic biology, many of the biological systems that are being researched could not be found in nature because they are genetically engineered, so their behaviors are unknown and need to be characterized.</P></DIV></DIV><DIV class="section text"><DIV class="content"><DIV class="word-title">Gene Regulatory Network</DIV><P>A gene regulatory network (GRN) is a collection of DNA segments in a cell which interact with each other indirectly (through their RNA and protein expression products) and with other substances in the cell, thereby governing the expression levels of mRNA and proteins. Some proteins serve only to activate or repress other genes, and these are the transcription factors that are the main players in regulatory networks or cascades. By binding to the promoter region at the start of other genes they turn them on or turn them off, initiating the production of another protein, and so on.</P></DIV></DIV><DIV class="section hero"><DIV class="content"><DIV class="word-title">GRN of I’M HeRE</DIV><P>For one green world, an Intelligent Microbial Heat Regulating Engine (I’M HeRE), as shown in figure 1, was designed to satisfy our needs described in the project. The engine consists of 2 systems namely the heat-resistant system and the quorum-control system.  </P><DIV><SMALL>Fig.1 The gene regulatory network of the intelligent microbial heat regulating . (a) shows the regulatory relationship of these genes. (b) is the detailed diagram of the GRN of the quorum control system.</SMALL></DIV><P>When the number of cells increased to a certain extent, the gene regulatory network was started to control the number of live cells.</P></DIV></DIV><DIV class="section text"><DIV class="content"><DIV class="word-title">Method description</DIV><P>Currently, there are a variety of models, such as the directed graph model, Bayesian network model, Boolean network model, ODE models and stochastic differential equations models, could be applied to the analysis, modeling and simulation of gene regulatory networks.</P><P>ODE models implicitly assumes that the underlying quantities, i.e., concentrations or molecule numbers, vary in a continuous and deterministic fashion. However, at the molecular level these assumptions are mere approximations which might not accurately reflect the underlying dynamics, especially when the number of molecules involved is &quot;small&quot;, such as is the case for transcription factors, which, in certain circumstances, can be expressed at low levels, i.e., a few tens of molecules, or for chromosomal DNA, for which a single copy exists in the cell. </P><P>Intrinsic stochasticity at the molecular level may not be neglected any more when small numbers of molecules are involved (since in this case the &quot;averaging&quot; out of stochastic effects due to the application of the law of large numbers does not hold any more). Therefore, stochastic models have to be considered.</P><P>The system may contain rare, discrete, but critical events and continuously occurring deterministic or stochastic transitions.</P></DIV></DIV><DIV class="section hero"><DIV class="content"><DIV class="word-title">Stochastic simulation of protein degradation</DIV><UL class="list-unstyled"><LI>A is the chemical species of interest, and the number of molecules of chemical species A at time t is simply denoted as A(t)</LI><LI>The symbol φ denotes chemical species which are of no further interest in what follows.</LI><LI>k is the rate constant of the reaction and is defined so that k*dt gives the probability that a randomly chosen molecule of chemical species A reacts (is degraded) during the time interval [t, t + dt)</LI><LI>the probability that r &lt; A(t)k∆t is equal to A(t)k∆t, since r is a random number uniformly distributed in the interval (0, 1)</LI><LI>A(t)*kdt gives the probability that exactly one reaction occurs during the infinitesimal time interval [t, t+dt)</LI></UL><P>The goal is to compute the number of molecules A(t) for times t &gt; 0. </P><DIV><P>With k = 0.1, protein (0) = 20 and ∆t = 0.005.</P></DIV><DIV><SMALL>Fig.2 Stochastic simulation of protein degradation for k = 0.1, protein (0) = 20 and ∆t = 0.005. Number of molecules of proteins as a function of time for two realizations</SMALL></DIV><DIV><SMALL>Fig.3 Stochastic simulation of protein degradation for k = 0.1, protein (0) = 20 and ∆t = 0.005. Number of molecules of proteins as a function of time for ten realizations. The blue dash line shows the mean value of the stochastic results.</SMALL></DIV><UL class="list-unstyled" style="font-size:30px;"><LI> Stochastic simulation of production and degradation</LI></UL><DIV><P>With Protein (0) = 0, k1 =0.1 and k2= 1</P></DIV><DIV><SMALL>Fig.4 two realizations</SMALL></DIV><DIV><SMALL>Fig. 5 ten realizations</SMALL></DIV><DIV><SMALL>Fig.6 stationary distribution of the molecules, it’s analyzed to be poison disribution</SMALL></DIV><UL class="list-unstyled" style="font-size:30px;"><LI>Stochastic simulation of dimer formation</LI></UL><DIV><P>With A(0)=0, B(0)=0, k1=10-3,k2=10-2, k3=1.2 and k4=1.</P></DIV><DIV><SMALL>Fig.7 number of A molecules</SMALL></DIV><DIV><SMALL>Fig.8 number of B molecules</SMALL></DIV></DIV></DIV><DIV class="section hero"><DIV class="content"><DIV class="word-title">Modelling the gene regulatory network</DIV><P>The GRN is too complicated to conducted biological experiments. Thus dynamic simulation of the GRN by mathematical modelling is particularly important. Before the wet experiments were conducted, mathematical models were built to predict the behavior of our heat regulating engine. Such bottom-up design of synthetic gene circuit will not only speed up the biology research, but provide us a valuable insight into the system as well. We want to know:</P><UL class="list-unstyled"><LI>In a particular state which genes were expressed?</LI><LI>What’s the influence of the gene product on cell physiological activities?</LI><LI>To provide a guidance on the design of experiments by further analysis of the dynamic behavior of gene regulatory networks.</LI></UL></DIV></DIV><DIV class="section pagetitle modeling"><H1>Noise in the cell</H1></DIV><DIV class="section text"><DIV class="content"><P>The cellular environment is abuzz with noise. A key source of this “intrinsic” noise is the randomness that characterizes the motion of cellular constituents at the molecular level. Cellular noise not only results in random ﬂuctuations (over time) within individual cells, but it is also a source of phenotypic variability among clonal cellular populations. Researchers are just now beginning to understand that the richness of stochastic phenomena in biology depends directly upon the interactions of dynamics and noise and upon the mechanisms through which these interactions occur.</P></DIV></DIV><DIV class="section pagetitle modeling"><H1>Quorum Control system</H1></DIV><DIV class="section text"><DIV class="content"><P>The quorum control system was composed of 3 device.</P><UL class="list-unstyled"><P>Goals:</P><LI> We want a better understanding of the process of gene regulation by analyzing the dynamical properties of the gene regulatory networks.</LI><P>Assumption:</P><LI> In this project, the simulation was performed in only a single cell. </LI></UL></DIV></DIV><DIV class="section hero"><DIV class="content"><P>Bacteria communicate with each other using chemical signaling molecules. The process of producing, releasing, detecting, and responding to the signaling molecules in bacteria is referred to as quorum sensing. Quorum sensing enables bacteria to obtain information from their environment and alter their behavior in a population size that is scaled in response to changes the number of bacteria present in their community. Thus, this method of intercellular communication allows populations of bacteria to work together, in contrast to the traditional unicellular view of prokaryotes.</P><P>In general, quorum-sensing bacteria produce and release signaling molecules into their environments which are called auto-inducers. The concentrations of auto-inducers increase in the bacteria environment as the bacteria population density grows. The bacteria detect the accumulation of a minimal threshold concentration of the auto-inducer and regulate their gene expression in a population-wide scale in response. As a result, using these signaling systems, all the bacteria in a community alter their behavior at the same time in response to an external signal, and therefore function similar to a multicellular organism.(indexed from Wikipedia)</P><P>Many different quorum-sensing systems have been discovered. Some of them are extremely specific for a particular species and some of them are common in all bacteria. In this work, we sought to establish a synthetic auto-inducer (AHL) signaling system.</P><P>It is assumed that AHL was produced and degraded at a certain rate to maintain a certain concentration in a single cell. That is to say the number of AHL molecules within a single cell is constant. Therefore, there is a linear relationship between the AHL molecule concentration and cell concentration. The regulating engine was started when the number of AHL molecules reaches a certain threshold, or when the number of cells reaches a certain threshold. </P><DIV><SMALL>Fig.9 the GRN of the quorum sensing device. The right table shows the chemical reactions of the GRN.</SMALL></DIV><DIV><SMALL>Fig.10 stochastic simulation of the GRN in a single cell</SMALL></DIV><P>As we could see from the simulation result in fig.10. AHL was produced and degraded at a certain rate to maintain a certain concentration in a single cell. A AHL*LuxR complex was formed once LuxR was produced. Thus the concentration of LuxR was low.</P></DIV></DIV><DIV class="section hero"><DIV class="content"><P>The oscillator is a synthetic biology construct composed of three genes which mutually repress each other in sequence according to a ring structure. </P><DIV><SMALL>Fig.11 detailed GRN of the oscillator</SMALL></DIV><DIV><SMALL>Fig.12 ODE simulation of the oscillator</SMALL></DIV><DIV>An experiment showing the oscillator was shown here <A href="http://www.youtube.com/watch?v=HFwSnqYC5LA">http://www.youtube.com/watch?v=HFwSnqYC5LA</A></DIV></DIV></DIV><DIV class="section hero"><DIV class="content"><DIV><SMALL>Fig.13 the GRN of the PCD device. The right table shows the chemical reactions of the GRN.</SMALL></DIV><DIV><SMALL>Fig.14  Stochastic simulation of the PCD</SMALL></DIV></DIV></DIV><DIV class="section pagetitle modeling"><H1>Heat Resistant system</H1></DIV><DIV class="section text"><DIV class="content"><P>The heat resistant system was made up by a RNA thermometer and a heat shock protein expression device. </P><P>Questions</P><UL class="list-unstyled"><LI>What’s the shape of the growth curve under normal growth condition?</LI><LI>What the impact if the lethal effect is considered?</LI><LI>What is the time to turn on the RNA temperature switch?</LI><LI>What happened if the cooling system was removed?</LI></UL><UL class="list-unstyled"><LI>Build the normal cell growth model</LI><LI>Build the growth model with the lethal effect considered</LI><LI>Build a model which can be used to predict the temperature change tendency</LI></UL></DIV></DIV><DIV class="section hero"><DIV class="content"><DIV><SMALL>Fig. 14 the red curve shows the result with the lethal effect considered, the blue curve shows the normal logistic growth result.</SMALL></DIV><P>The initial inoculation amount of this model is 1 unit. The proposed intrinsic rate is 0.05. And we set the maximum number of cells that the environment can support is 100 units. As can be seen from the simulation , from 0 to 200 units of time, the number of cells are in a growth state in both cases, no lethal effect is greater than the growth rate of cells in the presence simulation lethal effect of simulations; after 200 units of time, two kinds of simulations have tended to steady state, the growth of non-lethal effects of simulated results of the final number of the environment can accommodate the largest state, while the presence of the number of lethal effect of significantly less than the maximum number of steady-state simulation results is about 90 units. </P><DIV><SMALL>Fig.15 time evolution of heat accumulated in the fermenter</SMALL></DIV><P>We could get the information that the heat in the fermenter will accumulate over time without a cooling system from the above figure. Once the cooling system was turned on, a steady state could be achieved as shown in the figure above. </P></DIV></DIV><DIV class="section pagetitle modeling"><H1>Reference</H1></DIV><DIV class="section text"><DIV class="content"><P>[1] Esmaeili A, Yazdanbod I, Jacob C. A Model of the Quorum Sensing System in Genetically Engineered E. coli Using Membrane Computing[J]. International Genetically Engineered Machine (iGEM) competition, 2009.</P><P>[2] Paustian K, Pankhurst C E, Doube B M, et al. Modelling soil biology and biochemical processes for sustainable agriculture research[J]. Soil biota: management in sustainable farming systems., 1994: 182-193.</P><P>[3] Erban R, Chapman J, Maini P. A practical guide to stochastic simulations of reaction-diffusion processes[J]. arXiv preprint arXiv:0704.1908, 2007.</P><P>[4] Fournier T. Stochastic models of a self regulated gene network[D]. , 2008.</P><P>[5] Johansen S H. Modelling and Analysis of a Synthetic Bistable Genetic Switch[D]. Norwegian University of Science and Technology, 2011.</P><P>[6] Scott M. Tutorial: Genetic circuits and noise[J].</P></DIV></DIV><DIV id="footer_bg"><DIV id="footer">
                    © Copyright BIT iGEM Team 2013, All Rights Reserved.
                </DIV></DIV><DIV id="widget" style="visibility:hidden;opacity:0;"><DIV id="updown"><UL id="nav-arrow"><LI class="first"><A href="javascript:void(0);" class="up disabled" onclick="flipup()">Up</A></LI><LI class="last"><A href="javascript:void(0);" class="down disabled" onclick="flipdown()">Down</A></LI></UL></DIV></DIV></DIV><DIV class="printfooter">
Retrieved from &quot;<A href="http://2013.igem.org/Team:BIT-China/modeling.html">http://2013.igem.org/Team:BIT-China/modeling.html</A>&quot;</DIV></DIV></DIV><DIV id="footer-box" class="noprint"><DIV id="footer"><UL id="f-list"><LI id="t-recentchanges"><A href="/Special:RecentChanges" title="Recent changes">Recent changes</A></LI><LI id="t-whatlinkshere"><A href="/Special:WhatLinksHere/Team:BIT-China/modeling.html" title="List of all wiki pages that link here [j]" accesskey="j">What links here</A></LI><LI id="t-recentchangeslinked"><A href="/Special:RecentChangesLinked/Team:BIT-China/modeling.html" title="Recent changes in pages linked from this page [k]" accesskey="k">Related changes</A></LI><LI id="t-specialpages"><A href="/Special:SpecialPages" title="List of all special pages [q]" accesskey="q">Special pages</A></LI><LI><A href="/Special:Preferences">My preferences</A></LI></UL></DIV><DIV id="footer"><UL id="f-list"><LI id="t-print"><A href="/wiki/index.php?title=Team:BIT-China/modeling.html&amp;printable=yes" title="Printable version of this page [p]" accesskey="p">Printable version</A></LI><LI id="t-permalink"><A href="/wiki/index.php?title=Team:BIT-China/modeling.html&amp;oldid=241277" title="Permanent link to this revision of the page">Permanent link</A></LI><LI id="privacy"><A href="/2013.igem.org:Privacy_policy" title="2013.igem.org:Privacy policy">Privacy policy</A></LI><LI id="disclaimer"><A href="/2013.igem.org:General_disclaimer" title="2013.igem.org:General disclaimer">Disclaimers</A></LI></UL></DIV></DIV></DIV></BODY></HTML>