<HTML xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en" dir="ltr">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki  ltr ns-0 ns-subject page-Team_BIT-China_project_html"><DIV id="globalWrapper"><DIV id="top-section"><DIV id="p-logo"><A href="/Main_Page" title="Main Page">&quot;
	    </A></DIV><DIV id="menubar" class="left-menu noprint"><UL><LI class="selected"><A href="/Team:BIT-China/project.html">Page               </A></LI><LI class="new"><A href="/wiki/index.php?title=Talk:Team:BIT-China/project.html&amp;action=edit&amp;redlink=1">Discussion               </A></LI><LI><A href="/wiki/index.php?title=Team:BIT-China/project.html&amp;action=edit">View source               </A></LI><LI><A href="/wiki/index.php?title=Team:BIT-China/project.html&amp;action=history">History               </A></LI><LI style="color:white;cursor:default">teams</LI></UL></DIV><DIV class="right-menu noprint" id="menubar"><UL><LI id="pt-login"><A href="/wiki/index.php?title=Special:UserLogin&amp;returnto=Team:BIT-China/project.html" title="You are encouraged to log in; however, it is not mandatory [o]" accesskey="o">Log in</A></LI></UL></DIV><DIV id="search-controls" class="noprint"><FORM action="/Special:Search" id="searchform"> </FORM></DIV></DIV><DIV id="content"><H1 class="firstHeading">Team:BIT-China/project.html</H1><DIV id="bodyContent"><H3 id="siteSub" class="noprint">From 2013.igem.org</H3><P><TITLE>PROJECT | BIT-CHINA IGEM2013</TITLE></P><DIV id="wrapper"><DIV id="navigation"><DIV class="navigation"><DIV class="centering"><UL class="topnav" id="topnav"><LI class="navitem1"><A href="./index.html">HOME</A></LI><LI class="navitem2"><A href="./project.html">PROJECT</A></LI><LI><A href="./project.html#overview">Overview</A></LI><LI><A href="./project.html#CT">CT system</A></LI><LI><A href="./project.html#QC">Quorum-control system</A></LI><LI><A href="./project.html#parts">Parts</A></LI><LI><A href="./project.html#co">Cooperation</A></LI><LI><A href="./project.html#safety">Safety</A></LI></UL><LI class="navitem3"><A href="./team.html">TEAM</A></LI><LI><A href="./team.html#team_members">Members</A></LI><LI><A href="https://igem.org/Team.cgi?id=1117">Official Profile</A></LI><LI><A href="./team.html#About_the_University">About the University</A></LI><LI class="navitem4"><A href="./modeling.html">MODELING</A></LI><LI><A href="./modeling.html#overview">Overview</A></LI><LI><A href="./modeling.html#Noise_in_the_cell">Noise in the cell</A></LI><LI><A href="./modeling.html#Quorum_Control_system">Quorum-control system</A></LI><LI><A href="./modeling.html#Heat_Resistant_system">Heat Resistant system</A></LI><LI class="navitem5"><A href="./humanpractice.html">HUMAN PRACTICE</A></LI><LI><A href="./humanpractice.html#Meaning">Meaning</A></LI><LI><A href="./humanpractice.html#C">Communication</A></LI><LI><A href="./humanpractice.html#pro">Promotion</A></LI><LI class="navitem6"><A href="./notebook.html">NOTEBOOK</A></LI><LI><A href="./notebook.html#">Journal</A></LI><LI class="navitem7"><A href="./sponser.html">SPONSER</A></LI><LI class="navitem8"><A href="https://2013.igem.org">IGEM</A></LI></DIV></DIV></DIV></DIV><DIV id="width_wrapper"><DIV id="sidebar" style="left: 0px"><DIV class="list" id="project"><DIV class="temper"><DIV class="project_height"><DIV class="temp-body-project"><DIV class="color-project"><DIV id="mercury"><DIV class="temp tem37 heartbeat">37℃</DIV><DIV class="temp tem40 heartbeat">40℃</DIV><DIV class="temp tem43 heartbeat">43℃</DIV></DIV></DIV></DIV></DIV></DIV><DIV class="navis"><H2><A href="javascript:void(0);" onclick="temper(0, '#project')">PROJECT</A></H2><UL><LI><A href="javascript:void(0);" onclick="temper(1, '#overview')">Overview</A></LI><LI><A href="javascript:void(0);" onclick="temper(2,'#mab')">Motivation and Background</A></LI></UL><LI><A href="javascript:void(0);" onclick="temper(4,'#CT')">CT-system</A></LI><LI><A href="javascript:void(0);" onclick="temper(5,'#rt')">RNA thermometer</A></LI><LI><A href="javascript:void(0);" onclick="temper(6,'#hsps')">HSPs</A></LI><LI><A href="javascript:void(0);" onclick="temper(7,'#QC')">QC-system</A></LI><LI><A href="javascript:void(0);" onclick="temper(8,'#QS')">Quorum-sensing device</A></LI><LI><A href="javascript:void(0);" onclick="temper(10,'#Iv')">Frequency Self-regulating oscillator</A></LI><LI><A href="javascript:void(0);" onclick="temper(13,'#PCD')">PCD device</A></LI><LI><A href="javascript:void(0);" onclick="temper(14,'#parts')">Parts</A></LI><A href="javascript:void(0);" onclick="temper(15,'#co')">Cooperation</A><LI><A href="javascript:void(0);" onclick="temper(17, '#safety')">Safety</A></LI></DIV></DIV></DIV><DIV class="section pagetitle project"><H1>Overview</H1></DIV><DIV class="section hero"><DIV class="content"><P>
                In modern fermentation process, cooling system, aiming at keeping the cells in a good condition, plays an important role. However, that results in a great consumption of energy. According to a research carried by COFCO, the leader of bio-industry in China, once the fermentation temperature limit can be risen by 1 centidegree, 100 million degrees of electricity can be saved all around China. Moreover, thermal electric generation accounts for a large proportion in China. Let’s do a calculation about the effectiveness (Figure 1). The result is striking and the emission brought an enormous impact on environment such as acid rain, global warming and so on.

                </P><DIV><SMALL>Figure 1: if the temperature of fermentation raise 1℃，all fermentation industry in China can save nearly 1 billion yuan per year. According to the electric price of industry，1 billion equal to 173310225(kW•h) power. In China, 78% electric energy was produced with coal in 2012. That means we can save 135181975(kW•h) power which made by coal every year. To generate those electricity need consume 50017 tuns of coal! That is a big number! So we can shrift those coal and we can reduce 134776 tuns of carbon dioxide emissions per year. </SMALL></DIV><DIV style="float:right;width:300px;"><DIV> Figure 2:I’MHeRE include two systems: the customized thermo-tolerance and the intelligent quorum regulating system</DIV></DIV><P>Inspired by this, our team managed an intelligent microbial heat regulating engine (I’MHeRE) in E.coli with the methods of synthetic biology. In this engine, two complementary systems (figure 2) were designed. The first one is the customized thermo-tolerance system used to improve E.coli’s heat resistant ability and the other is the intelligent quorum regulating system used for controlling the cell density in an appropriate degree. Moreover, the engine begin to work only when it is properly to do so, ensuring the productive efficiency.</P><P>The chassis host with I’MHeRE used in the industry may make the fermentation process less depend on the cooling system and decrease the pollutant emission. On the other hand, cells can live as comfortable as in their optimum temperature, because we extend the because we extend the range of their optimum living temperature and make them live in the optimizing density. Owing to this, the high activity of the enzymes in cell could be kept in a wide range of temperature and the efficiency of microbial metabolism can be improved.</P></DIV></DIV><DIV class="section pagetitle project"><H1 style="font-size:60px;">Customize thermotolerance system</H1></DIV><DIV class="section hero"><DIV class="content"><P>Different strength promoters control different expression level of different heat shock proteins (HSPs) at different temperature to achieve the goal of Hierarchy Heat-resistant.</P><DIV>Figure 3: Hierarchy Heat-resistant.</DIV><DIV style="width:600px;float:left;margin:30px;">
                Figure 4:Overexpression of HSP will saddle the cell growth
                </DIV><P>We found that overexpressing a protein may affect the growth of the host to a certain degree. With the help of RNA thermometers, they can not only be expressed in a hierarchy way, but decrease the burden of overexpressing HSPs at low temperature. That is the answer why we use RNA thermometers.</P></DIV></DIV><DIV class="section hero"><DIV class="content"><P>Bacteria use complex strategies to control gene expression in response to ambient temperature changing. Many genes encoding heat shock proteins and virulence factors are regulated by temperature sensing RNA sequences, known as RNA thermometers (RNATs), which are located in the 5’untranslated region of the mRNA folding into secondary structure that shed RBS to influence translational efficiency in low temperature. But when temperature is increased to a certain temperature, the RNATS release RBS and the blocked genes express.</P><P>Most of gene expression responding to temperature shock measure not the temperature itself but the consequences of temperature-induced damage. Those reaction are effected by temperature indirectly. However, RNA thermometers response to temperature immediately, precisely, and controllably. So using RNA thermometers to control our heat shock proteins expression is a good choice. The RNA thermometers can be available included two types, the natural RNATS and the synthetic RNATS.</P><H3>Natural RNATS</H3><DIV style="width:320px;float:right;margin-left:20px;margin-top:20px;"><SMALL>Figure 6: ROSE RNATS</SMALL></DIV><H2>ROSE family</H2><P><B>ROSE RNATS</B> were found in numerous alphaproteobacteria and gammaproteobacteria. All known ROSE elements control the expression of small heat shock genes. ROSE-type structures are composed of two, three or four individual hairpins. The 5ʹ-most hairpins remain stable under heat shock conditions, but the 3ʹ-most hairpin that pairs with the SD sequence is only stable at low temperatures. Temperature-induced local melting in this hairpin exposes the SD sequence, facilitating ribosome binding. The SD region is occluded at 30 °C, partial melting occurs at 37 °C, whereas an increase to 42 °C facilitates the mRNA–ribosome interaction owing to full liberation of the SD and AUG start codon. This type feeds our needs. It may not initiate translation at a certain temperature completely, but the level of expression increase alone with the temperature increasing.</P><H2>Four U family</H2><DIV style="float:right;margin-left:20px;width:350px;"><SMALL>Figure 7:the structre of Four U RNATS</SMALL></DIV><P>Another family of RNATs is a stretch of four uridines that pairs with AGGA in the SD sequence. FourU elements exist in the 5ʹ UTRs of several heat shock and virulence genes. It has two distinct hairpins. The first hairpin is heat stable, however the second hairpin is temperature sensitive, which hide the SD region through the fourU. The formation of the ternary translation initiation complex occurs at high (45 °C) but not at low (30 °C) temperatures.</P><H3>Synthetic RNA thermometers</H3><P>Obvious, RNA thermometers provide us superb genetic tools to induce or repress gene expression. However, most natural RNA thermometers are relatively large. They fold into rather complex secondary structures and have been suggested to undergo gradual conformational changes in response to changes in temperature. Therefore, we want to construct smaller and more convenient synthetic RNA thermometers to achieve the goal of hierarchy Heat-resistant regulation. Under the following principle we construct a series of synthetic RNA thermometers.</P><P>Stem stability can be influenced by (i) changes in the size and/or GC content of a perfectly matched stem and (ii) introduction of mismatches at different positions.</P><DIV style="width:800px;"><SMALL>Figure 8: The structures of Synthetic RNA thermometers simulated using mFold</SMALL></DIV></DIV></DIV><DIV class="section text"><DIV class="content"><P>Based on literature, we know that it’s a practical and effective way to improve E.coli’s heat resistant ability by importing exogenous HSPs into it. Heat shock proteins (HSPs) are a group of proteins induced by heat shock, the most prominent members of this group are a class of functionally related proteins named chaperones which are involved in the folding and unfolding of other proteins. The amount of HSPs' expression is the key point of cell's heat resistant ability. </P><DIV><SMALL>Figure 9: Heat shock protein family </SMALL><SMALL>(From: http://pdslab.biochem.iisc.ernet.in/hspir/chaperone.php)</SMALL></DIV><P>However, HSPs exist in E.coli can’t improve the heat resistant ability much even when being highly expressed. As a result, we paid our attention to other hot-spring bacteria. After several strains were compared, we chose Tengcongensis MB4 which contains abundant kinds of HSPs. Here is a summary: </P><P>After further literature, we verified the function of GroEL, GroES, DnaK, DnaJ, TTE, IbpA, FliA, RpoE3 and RpoE7. Here’re the results:</P><DIV><SMALL>Figure 10 The functional verification of HSPs</SMALL></DIV><P>As showed above, GroEL, GroES, DnaK and FliA perform better and we chose these four to construct heat-resistance part.</P></DIV></DIV><DIV class="section pagetitle project"><H1 style="font-size:60px;">Intelligent quorum regulating system</H1></DIV><DIV class="section hero"><P>We designed an Intelligent quorum regulating system contains three devices to make cell density decrease moderately and control the apoptosis happen in a certain part of cell. The first device is for quorum sensing. The second device is for moderation and targeting and the third device leads to programmed cell death.</P><P>The first one is quorum sensing device. Quorum-sensing molecules called AHLs can cross cell membrane freely, which means that it can be used as a symbol of cell density. When the density of AHL is high enough, AHL will combine with a protein called LuxR and become a complex. This complex can inhibit downstream gene, TetR. So the next device will not be inhibited anymore and then begin to work.</P><DIV><SMALL>Figure 11: Quorum-sensing device gene pathway</SMALL></DIV></DIV><DIV class="section text"><P>We added an oscillating circuit as the moderate device. In this circuit, three proteins inhibit the expression of each other, forming an oscillation as showed in figure 12.</P><DIV><SMALL>Figure 12:Oscillator gene pathway</SMALL></DIV><P>The function of oscillator is outputting the CI periodically which leads the accumulation of mazeF. In other words, the accumulation of mazeF is moderated. At the same time, different cells response to this signal differently in the beginning so that the cells responded quicker will die first (more detail in third device). With the decreasing of the concentration of AHL, the signal will disappear and the cells responded slower will survived.</P><DIV style="width:100px;">Figure 13</DIV></DIV><DIV class="section hero"><DIV><SMALL>Figure 14: MazEF system</SMALL></DIV><P>Programmed cell death (PCD) is an active process that results in cell suicide and is an essential mechanism in multicellular organisms. Generally, PCD is required for the elimination of superfluous or potentially harmful cells. In E.coli, there are several toxin-antitoxin systems, including mazEF, chpBIK, relBE, yefM, yoeB. The mazEF is one of the well-studied systems. MazEF system is a toxin-antitoxin system. MazF, which has RNA cleavage ability, is a kind of toxin. MazE, on the other hand, is the antidote of MazF. In the nature, many bacteria will sacrifice themselves and leave nutrition to the left when facing threats from dangerous environment, such as heat, lack of food and so on.</P><P>As we all know, the more E.coli, the more biological heat would release. The excessive heat is not good for cells growth. What’s more, the overhigh density of cell is an obstacle for E.coli to produce Secondary metabolites like antibiotics. So we use the PCD device to control the density of cell. </P><P>The CI expressed in oscillation device will activate PCD device. Once CI is expressed, the expression of MazE will be inhibited. No more hexamer will be formed, and MazF will show its power and the density of cell will decrease. When the density is too low to activate the system, then there will be no CI. After that, the MazE will be expressed again, and then it becomes a circle (Figure 15).</P><DIV><SMALL>Figure 15: The program cell death system</SMALL></DIV><P>In order to test whether this device is work, we use the T7 promoter to control the expression of CI. If the cell density decrease when CI is expressed with induction of T7 promoter, this device is successful (Figure 16). </P><DIV><DIV style="float:left;width:400px;"><SMALL>Figure 16: Verification of the program Cell death system</SMALL></DIV></DIV></DIV><DIV class="section hero"><DIV class="content"><DIV class="word-title">Reference</DIV><P>[1]	Nocker A, Hausherr T, Balsiger S, et al. A mRNA-based thermosensor controls expression of rhizobial heat shock genes[J]. Nucleic acids research, 2001, 29(23): 4800-4807.</P><P>[2]	Neupert J, Karcher D, Bock R. Design of simple synthetic RNA thermometers for temperature-controlled gene expression in Escherichia coli[J]. Nucleic acids research, 2008, 36(19): e124-e124.</P><P>[3]	Neupert J, Bock R. Designing and using synthetic RNA thermometers for temperature-controlled gene expression in bacteria[J]. Nature protocols, 2009, 4(9): 1262-1273.</P><P>[4]	Kortmann J, Narberhaus F. Bacterial RNA thermometers: molecular zippers and switches[J]. Nature Reviews Microbiology, 2012, 10(4): 255-265.</P><P>[5]	Narberhaus F. Control of Bacterial Heat Shock and Virulence Genes by RNA Thermometers[M]//Regulatory RNAs in Prokaryotes. Springer Vienna, 2012: 183-193.</P><P>[6]	Rinnenthal J, Klinkert B, Narberhaus F, et al. Modulation of the stability of the Salmonella fourU-type RNA thermometer[J]. Nucleic acids research, 2011, 39(18): 8258-8270.</P><P>[7]	Danino T, Mondragón-Palomino O, Tsimring L, et al. A synchronized quorum of genetic clocks[J]. Nature, 2010, 463(7279): 326-330.</P><P>[8]	Elowitz M B, Leibler S. A synthetic oscillatory network of transcriptional regulators[J]. Nature, 2000, 403(6767): 335-338.</P><P>[9]	You L, Cox R S, Weiss R, et al. Programmed population control by cell–cell communication and regulated killing[J]. Nature, 2004, 428(6985): 868-871.</P><P>[10]	Lila M. Gierasch ,FoldEco: a model for proteostasis in E. coli.Cell Reports, 265–276, March 29, 2012</P><P>[11]	FU Hartl Molecular, chaperones in protein folding and proteostasis,NATURE, VOL 475, 21 JULY ,2011</P><P>[12]	Hanna Engelberg-Kulka,Bacterial Programmed Cell Death and Multicellular Behavior in Bacteria,PLoS Genetics,October 2006 ,Volume 2 ,Issue 10</P><P>[13]	BLOWER T R，SALMOND G P C，LUISI B F． Balancing at survival’sedge: the structure and adaptive benefits of prokaryotic toxin – antitoxin partners［J］． Curr Opin Struct Biol，2011，21( 1) : 109-118．</P></DIV></DIV><DIV class="section pagetitle project"><H1>Parts</H1></DIV><DIV class="section hero"><DIV class="content"><DIV class="component"><DIV class="items">1</DIV><H3>New Parts</H3><H6>BBa_K1117001</H6><H6>DnaK</H6><P>DnaK is one of the Hsp70 proteins expressed in prokaryotes. By binding to partially peptide sequences, it can prevent proteins from aggregating and being rendered unfunctional.</P><P><A href="http://parts.igem.org/Part:BBa_K1117001">http://parts.igem.org/Part:BBa_K1117001</A></P><H6>BBa_K1117002</H6><H6>GroEL</H6><P>GroEL(Hsp60) belongs to the chaperonin family of molecular chaperonescan. It could help proteins to fold properly with the presence of protein complex GroES(Hsp10).</P><P><A href="http://parts.igem.org/Part:BBa_K1117002">http://parts.igem.org/Part:BBa_K1117002</A></P><H6>BBa_K1117003</H6><H6>GroES</H6><P>GroES(Hsp10) is also known as chaperonin 10 (cpn10) or early-pregnancy factor (EPF). It could help proteins to fold properly with the presence of GroEL(Hsp60).</P><P><A href="http://parts.igem.org/Part:BBa_K1117003">http://parts.igem.org/Part:BBa_K1117003</A></P><H6>BBa_K1117004</H6><H6>ThiF</H6><P>ThiF acts like an E1 ubiquitin-activating enzyme for ThiS, which is a ubiquitin-like protein in prokaryotes. The process of ubiquitination can target denatured proteins and stimulate their degradation.</P><P><A href="http://parts.igem.org/Part:BBa_K1117004">http://parts.igem.org/Part:BBa_K1117004</A></P><H6>BBa_K1117005</H6><H6>J23119-K115001-DnaK</H6><P>A strong constitutive promoter J23119 linked to the 42℃ RNA thermometer K115001 and heat shock protein DnaK K1117001.</P><P><A href="http://parts.igem.org/Part:BBa_K1117005">http://parts.igem.org/Part:BBa_K1117005</A></P><H6>BBa_K1117006</H6><H6>J23119-K115001-GroEL</H6><P>A strong constitutive promoter J23119 linked to the 42℃ RNA thermometer K115001 and heat shock protein GroEL K1117002.</P><P><A href="http://parts.igem.org/Part:BBa_K1117006">http://parts.igem.org/Part:BBa_K1117006</A></P><H6>BBa_K1117007</H6><H6>J23119-K115001-ThiF</H6><P>A strong constitutive promoter J23119 linked to the 42℃ RNA thermometer K115001 and enzyme ThiF K1117004.</P><P><A href="http://parts.igem.org/Part:BBa_K1117007">http://parts.igem.org/Part:BBa_K1117007</A></P><H6>BBa_K1117008</H6><H6>J23110-K115002-DnaK</H6><P>A medium strength constitutive promoter J23110 linked to the 37℃ RNA thermometer K115002 and enzyme DnaK K1117001.</P><P><A href="http://parts.igem.org/Part:BBa_K1117008">http://parts.igem.org/Part:BBa_K1117008</A></P><H6>BBa_K1117009</H6><H6>J23119-K115001</H6><P>A strong constitutive promoter J23119 linked to the 42℃ RNA thermometer K115001.</P><P><A href="http://parts.igem.org/Part:BBa_K1117009">http://parts.igem.org/Part:BBa_K1117009</A></P><H6>BBa_K1117010</H6><H6>J23110-K115002</H6><P>A medium strength constitutive promoter J23110 combined with 37℃ RNA thermometer K115002.</P><P><A href="http://parts.igem.org/Part:BBa_K1117010">http://parts.igem.org/Part:BBa_K1117010</A></P><H6>BBa_K1117011</H6><H6>J23119-B0034-C0062-B0015</H6><P>We change the Constitutive Promoter II of BBa_I9020 to make it more efficiency to feed our need.</P><P><A href="http://parts.igem.org/Part:BBa_K1117011">http://parts.igem.org/Part:BBa_K1117011</A></P><H6>BBa_K1117012</H6><H6>I719005-P0451</H6><P>Inducible promoter T7 controls the expression of CI.</P><P><A href="http://parts.igem.org/Part:BBa_K1117012">http://parts.igem.org/Part:BBa_K1117012</A></P></DIV><DIV class="component"><DIV class="items">2</DIV><H3>Used Parts</H3><TABLE class="table1 table1-hover"><THEAD><TR><TH>NAME</TH><TH>DESCRIPTION</TH></TR></THEAD><TBODY><TR><TD>J23117</TD><TD>Promoter</TD></TR><TR><TD>B0034</TD><TD>  RBS</TD></TR><TR><TD>C0062</TD><TD>luxR repressor/activator</TD></TR><TR><TD>B0010</TD><TD>Terminator</TD></TR><TR><TD>B0012</TD><TD>Terminator</TD></TR><TR><TD>J23103</TD><TD>Promoter</TD></TR><TR><TD>C0061</TD><TD>Autoinducer synthetase for AHL</TD></TR><TR><TD>R0063</TD><TD>Promoter</TD></TR><TR><TD>C0040</TD><TD>tetracycline repressor from transposon Tn10 (+LVA)</TD></TR><TR><TD>C0012</TD><TD>lacI repressor from E. coli (+LVA)</TD></TR><TR><TD>C0051</TD><TD>cI repressor from E. coli phage lambda (+LVA)</TD></TR><TR><TD>R0051</TD><TD>Promoter (lambda cI regulated)</TD></TR></TBODY></TABLE></DIV></DIV></DIV><DIV class="section pagetitle project"><H1>Cooperation</H1></DIV><DIV class="section hero"><DIV class="content"><P>To verify our work, we find Tianjin team, which have cooperation relationship with us. We verified a part of their work in return.
The initial device they designed is BBa_K1020004. During the cooperation,We noticed that the bacteria with this device had a slow growth rate. We thought this was because the leakage of this promoter (J23100) was too strong, for alkR was constitutively expressed and RFP had a high expression level even in the absence of alkanes, the inducible compounds. We assumed that the over-expression of alkR caused negative influence on the growth of cells. To reduce the expression level of the constitutively expressed alkR, eliminating the influence, we designed a modified device BBa_K1020005, and replace J23100 with J23103. It turned out that RFP showed almost no expression without alkanes. If we add C8 as inducible compounds, the red color can be observed and the rate of cell growth is largely increased.
</P><P>The initial device they designed is BBa_K1020004. During the cooperation, We noticed that the bacteria with this device had a slow growth rate. We think this is because the leakage of this promoter (J23100) was quite strong, for alkR was constitutively expressed and RFP had a high expression level even in the absence of alkanes as inducible compounds. We assumed that over-expression of alkR had a negative influence on the growth of cells. To eliminate the influence, we designed a modified device BBa_K1020005, which replaced J23100 with J23103, in order to reduce the expression level of the constitutively expressed alkR. It turned out that RFP showed almost no expression without alkanes. If we add C8 as inducible compounds, the red colour of the cells can be observed and the growth rate of cells is largely increased.</P><DIV><P style="text-align:center;">Figure 1: without alkanes as inducible compounds (BBa_K1020004&amp; BBa_K1020005)</P></DIV><DIV><P style="text-align:center;">Figure 2: modified BBa_K1020005（with and without alkane as inducible compounds）</P></DIV><P><A href="jhttps://2013.igem.org/Team:Tianjin/Project/Experiment">https://2013.igem.org/Team:Tianjin/Project/Experiment</A></P></DIV></DIV><DIV class="section pagetitle project"><H1>Safety</H1></DIV><DIV class="section text"><DIV class="content"><DIV class="component"><DIV class="items">1</DIV><DIV class="left"><H3>Organism &amp; Experiment safety</H3><P>
The organisms used in our experiment conclude:
    <UL><LI>Escherichia coli BL21(DE3)</LI><LI>Escherichia coli K12</LI></UL></P><P>
Although all of the organisms are in line with biosafety level 1 (BSL), we have treated it as a potential affection source and created following rules to minimize the danger:
<UL><LI>Wearing appropriate protective equipment like gloves and lab coats. </LI><LI>Don’t eat any food or store it in the lab.</LI><LI>Avoid wearing shorts or shoes that make your legs or toes exposed.</LI><LI>Washing hands before leaving the laboratory.</LI></UL></P></DIV></DIV><DIV class="component"><DIV class="items">2</DIV><DIV class="left"><H3>Safety training</H3><P>
There is no Biosafety Committee at our university, only a bio-safety officer. Nevertheless, we have iGEM instructors, Dr. Chun Li and Dr. Shuyuan Guo, who trained us about the project progress regularly and is responsible for the bio-safety of our project. The general rule at our university is that every student has to take part in specific safety training before their first experiment. Further, once a year, every member of the lab is required to take a refreshment course in lab safety, bio safety, fire safety, waste management, handling gas cylinders and working with hazardous materials. All of these treatments ensure our lab safety.
</P></DIV></DIV><DIV class="component"><DIV class="items">3</DIV><DIV class="left"><H3>Public &amp; Environment safety</H3><P>
The model organisms used in our project are non-virulent strains which are commonly used in microbiology laboratories. The organism has been provided various resistance of antibiotics such as chloramphenicol, ampicillin and kanamycin. Without the circumstance which contained specific antibiotic, our organism has no advantage in the competition with the wild one. Besides, although we have improve its resistance of higher temperature(42~45℃), it can still be denatured via heat. In a word, it can work well in laboratory environment and will be eliminated through competition when exposing in the wild.
</P></DIV></DIV></DIV></DIV><DIV id="footer_bg"><DIV id="footer">
            © Copyright BIT iGEM Team 2013, All Rights Reserved.
        </DIV></DIV><DIV id="widget" style="visibility:hidden;opacity:0;"><DIV id="updown"><UL id="nav-arrow"><LI class="first"><A href="javascript:void(0);" class="up disabled" onclick="flipup()">Up</A></LI><LI class="last"><A href="javascript:void(0);" class="down disabled" onclick="flipdown()">Down</A></LI></UL></DIV></DIV></DIV></DIV><DIV class="printfooter">
Retrieved from &quot;<A href="http://2013.igem.org/Team:BIT-China/project.html">http://2013.igem.org/Team:BIT-China/project.html</A>&quot;</DIV></DIV></DIV><DIV id="footer-box" class="noprint"><DIV id="footer"><UL id="f-list"><LI id="t-recentchanges"><A href="/Special:RecentChanges" title="Recent changes">Recent changes</A></LI><LI id="t-whatlinkshere"><A href="/Special:WhatLinksHere/Team:BIT-China/project.html" title="List of all wiki pages that link here [j]" accesskey="j">What links here</A></LI><LI id="t-recentchangeslinked"><A href="/Special:RecentChangesLinked/Team:BIT-China/project.html" title="Recent changes in pages linked from this page [k]" accesskey="k">Related changes</A></LI><LI id="t-specialpages"><A href="/Special:SpecialPages" title="List of all special pages [q]" accesskey="q">Special pages</A></LI><LI><A href="/Special:Preferences">My preferences</A></LI></UL></DIV><DIV id="footer"><UL id="f-list"><LI id="t-print"><A href="/wiki/index.php?title=Team:BIT-China/project.html&amp;printable=yes" title="Printable version of this page [p]" accesskey="p">Printable version</A></LI><LI id="t-permalink"><A href="/wiki/index.php?title=Team:BIT-China/project.html&amp;oldid=335485" title="Permanent link to this revision of the page">Permanent link</A></LI><LI id="privacy"><A href="/2013.igem.org:Privacy_policy" title="2013.igem.org:Privacy policy">Privacy policy</A></LI><LI id="disclaimer"><A href="/2013.igem.org:General_disclaimer" title="2013.igem.org:General disclaimer">Disclaimers</A></LI></UL></DIV></DIV></BODY></HTML>