Team:Madrid-OLM/AptamerProtocols
Madrid-OLM
Team
Members
Attribution
Sponsors
Project
Description
Contribution
Improve
Demonstrate
Judging Form
Aptamers
Overview
Discovery
Characterization
Computational
Improvement
Binding to Electrode
Device
Overview
First Prototype
Final Prototype
Overview
Microfluidics
Electronics
Experiments
Interlab
Protocols
Aptamer Related
Manufacturing
Human Practices
Integrated Human Practice
Education and
Engagements
Collaborations
Awards
Hardware
Applied Design
Entrepreneurship
Measurement
Medals
Aptamer`s Protocols
Aptamer's Protocols
Aptamers offer an endless number of possibilities, however, in iGEM hasn't settled as a main tool.  So far, the cost and difficulty to work with them have been the bottleneck.
We offer the workflow that we had been successful and relatively cheaper than the others techniques
Aptamer Discovery
SELEX
SELEX
Bill Of Materials: You could see a complete BoM here.
Amount of time: 1 day
Total  costs: 40 € (with iGEM sponsor).
DIY nitrocellulose column manufacture
Download the columns of the stl files from our github repository .
3D print the stl models in PETG. For more information about the reasons why we choose this material see the results page.
ADVICE
: We have found the following parameters as the optimal ones printing with a Prusa i3 machine:
-Filaments diameter of 1.75mm
-Nozzle at 230ºC. Base 80ºC with Nelly hairspray. (CAUTION: The brand of the headspray must be Nelly.
Separate the 3D printed structures from the printer base. Remove the excess of printed material.
Treat the columns with dichloromethane until the surface gets smooth.
ADVICE
: For us it have worked putting the columns in glass jar, above a cardboard pedestal. Then cover the base of the jar with dichloromethane without touching the 3D printed files. Put the jard on the 3D printed hotbed at 80ºC for 20 minutes.
Wash the columns three times in deionized water to clean them from dicloromethane.
Put the columns in sterilizing solution (0,1N NaOH, 1% (m/v) EDTA) to inactivate DNAses and remove other pollutants. Keep overnight at room temperature.
Keep in milliQ water until its use.
Designing and ordering the initial library
Design your library as a DNA of 30-40 random nucleotides flanked by constant extremes of 12-18 nucleotides. Use HPLC purification. Also order the primers for this constant edges.
ADVICE
: For us, IDT have worked well as a DNA provider. They are also iGEM sponsor at our year, so this libraries could be free for igem teams.
ADVICE
: The following sequence have fit well to us:
START SELEX CICLE
Prepare the library pool
Resuspend 2 nmol de la library pool on 200 µl of Binding Buffer (Tris-HCl PH= 7,4 20 mM; MgCl21mM; NaCl 150mM; KCl 5 mM).
Denatured the library by heating it at 90ºC for 10 min and immediately cold it on ice for another 10 min.
Wash in distilled water and  mount the nitrocellulose column by cutting a small square of the membrane and then pre-wet it with the BB.
CAUTION
: The colums break easily, so do not aplyy too much force on them.
To get rid of the DNA that unespecifically binds to the system, apply the library through  a nitrocellulose  membrane and centrifuge 1 min at 8000 rpm. Quantify the DNA that does   not bind unspeficically and note it as the initial DNA.
Protein-Aptamer incubation
Incubate the flowthrough with the protein of interest during 1 hour.
Apply the DNA to a new nitrocellulose  membrane as in step 11.
Wash the membrane four times with 300 µl of BB, like on step 11.
Recover the membrane and transfer it to a new Eppendorf  tube.
CAUTION
: Do not let the membrane dry, as it becomes fragile and the mollecules inside it could be damage.
Denatured the protein and elute the selected DNAs
Add 400µL of FES and 500 µL of phenol and mix in a thermomixer/ vortex at 1.400 rpm for 10 min.
Transfer the liquid to a new tube and repeat step 8 but this time with 200 µl of each regeant.
Mix the two samples and add 200 µl of Milli-Q wáter to allow the phase separation and centrifuge 10 min at 16100 g.
Transfer the aqueous phase (upper) to a new 2 ml tube and made a PCI or Qiagen (link) columns to extract the DNA. Resuspend the purified DNA in 30 ul of Milli-Q water.
ADVICE
: Qiagen colums recover more DNA and also reduced the time of the purification, but are more expensive.
PAUSE POINT
:You can leave the PCI precipitation overnight (see PCI protocol), or the Qiagen Purified DNA in the fridge at 4ºC.
Library amplification
Prepare the PCR mixture for a final volume of 50 µl per reaction and a final primer concentration of 0,8 µM. For the first round use all the template recover after the incubation.                                                 For the next rounds use 20 ul of template and adjust the rest according to the reagent you use.
Perform the amplification with the following amplification conditions. Adjust the annealing temperature according to the primers used, and the hotstart to the specifications of your polymerase:
Prepare an agarose gel at 3%. Load the samples and perform the electrophoresis at 90V for 50 min.
ADVICE
: We strongly recommend to quantify the DNA by gel molecular mass marker instead other methods like nanodrop. Add in this step to the first line of your gel  if you decide to use this method.
ADVICE
: For revealing the gel bands, GelRed have fits correctly to our purpose. We have put the GelRed before the gel polymerization step inside the mixture, following the product specifications.
Remove the gel and observe the bands under UV light.
It is needed at least 1 ug to continue with the next round. If it not accomplish, a further amplification is needed (continue reading). If you succeed amplifying with 10 cyclis this amount of DNA, skip the next steps and continue repeating this steps to do the next SELEX round.
PAUSE POINT
:The library can be stored at -20ºC
CAUTION
: We strongly recommend you to keep a little portion of each round of selection as a backup plan in case that you lost your DNA in further rounds. Keep in mind this when you amplify your DNA, because you will need more that the 1ug of DNA used in the next SELEX round.
Determination the optimal number of amplification cycles:
The total PCR reaction mixture volume for each tube is 50 µl using as template 0,5 µl of the library amplified before, for each tube, and a final primers concentration of 0.8µM.  Choose PCR samples at the following cycles:5, 10, 15, 20, 25. Also a negative control tube at the twentieth cycle.
Perform the PCR amplification with the same condition as step 22 and take the samples at the specified cycles
Prepared an agarose gel at 3%.
Perform the electrophoresis gel at 90V for 50 min.
Select the maximum number of cycles where you can a see a clear band without unspecific products.
PAUSE POINT
: You can store the DNA at -20ºC
ADVICE
: If you always have secondary bands, it means that concatemers are forming in your PCR reactions. Consider reducing the template and/or the cycles you are performing.
ADVICE
: Select the rounds that have the maximum amount of DNA that fits to your needs without secondary bands. Its more important to have the correct purity if you already are going to have the necessary amount. If secondary structures are always forming in your PCR, consider purifying the correct bands from your gel with a kit.
Preparative PCR:
Prepare a 200 µL PCR. Use as template 2 µL of the library amplified before and a final primer concentration of 0.8 µM..
Use the same programme but with the cycles chosen before
Perform a new electrophoresis gel to ensure that the amplification was successful. Purified the DNA and stored it at -20ºC.
END SELEX CICLE
If the cicle is mutiple of 3, do the qPCR (explaind in the next step) to check if the selection is done right.
qPCR
qPCR
Bill Of Materials: You could see a complete BoM here.
Amount of time: 4 hours.
Total  costs: 94 € (price of the genomic service of our university).
Prepare a 1:10 dilution of each round you want to check.
ADVICE
: If one of the rounds is very concentrated, make a 1:10 and a 1:100 dilution of it.
Prepare a 20 µl PCR for each well following these specifications:
CAUTION
: qPCR are extremely sensible. To avoid pipetting errors, make the mixture, except the template, multiplying x 1,5 your number os samples (including duplicates).
Divided the mixture into different tubes. As many as different rounds, you want to check.
Add 2 µl of template for each well into the mixtures. Pipette 20 µl for well.
The plaque will look like this:
Perform the amplification with the following amplification conditions for 25 cycles. Adjust the annealing temperature according to the primers used, and the hot start to the specifications of your polymerase:
As you perform each round of selection and enrich your library with the bound sequences, the graphic on the PCR would change from reaching a maximum and then decreasing the fluorencend to a sigmoid curve. This means the number of sequences is significally reduced in comparison with the initial library ( 106 different sequences):
Manual PCI Purification
PCI Extraction and ethanol  precipitation
Bill Of Materials: You could see a complete BoM here.
Amount of time: 2 dias
Total  costs: 0 €.
PCI Extraction:
Add an equal volume of PCI  (phenol: chloroform: isoamyl alcohol 25:24:1) to the digested DNA solution to be purified in a 1.5-ml microcentrifuge tube.
Mix gently for 5 min (rocking platform or vortex) and microcentrifuge 10 min at 10,000 rpm at room temperature.
CAUTION
: work with all the reagents in an extration hood.
Remove the top (aqueous) phase containing the DNA and transfer to a new tube. Repeat steps 1-3.
Add an equal volume of CI ( chloroform: isoamyl alcohol 24:1). Mix gently for 2 min and centrifuge for 1 min at 10,000
Remove the top (aqueous) phase containing the DNA and transfer to a new tube.
Ethanol Precipitation:
Add 3 volumes of ice-cold 100 ethanol and 1/10 volumes of 3M Sodium acetate. Invert the tube and place in -20 ºC overnight or in -70ºC for 1 hour.
Spin 30 min in a fixed-angle microcentrifuge at 16 100g and 4ºC. Remove the supernatant.
Add 1 ml of room-temperature 70% ethanol ( if the DNA molecules are very small, less than 200 pb, use 95% ethanol)  and only wash the pellet. microcentrifuge as in step 2.
Spin 10 min at 16 100g and remove the supernatant
Let the pellet air dry for 20 min.
CAUTION
: Wash the pellet carefully. Invert the tube gently.
Quiagen Purification
Quiagen Purification
Bill Of Materials: this link. .
Amount of time: 1 hour
Total  costs: 100 €.
Add 5 volumes of Buffer PB to 1 volume of the PCR sample, and then mix. It is not necessary to remove mineral oil or kerosene. For example, add 500 μl of Buffer PB to 100 μl PCR sample (not including oil).
If pH indicator I has been added to Buffer PB, check that the mixture’s color is yellow. If the color of the mixture is orange or violet, add 10 μl of 3 M sodium acetate, pH 5.0, and mix. The color of the mixture will turn yellow.
Place a QIAquick spin column in a provided 2 ml collection tube.
To bind DNA, apply the sample to the QIAquick column and centrifuge for 30–60 s.
Discard flow-through. Place the QIAquick column back into the same tube. Collection tubes are reused to reduce plastic waste.
To wash, add 0.75 ml Buffer PE to the QIAquick column and centrifuge for 30–60 s.
Discard flow-through and place the QIAquick column back into the same tube. Centrifuge the column for an additional 1 min
CAUTION
: Residual ethanol from Buffer PE will not be completely removed unless the flow-through is discarded before this a Ensure that the elution buffer is dispensed directly onto the QIAquick membrane for complete elution of bound DNA. dditional centrifugation
Place QIAquick column in a clean 1.5 ml microcentrifuge tube.
To elute DNA, add 50 μl Buffer EB (10 mM Tris•Cl, pH 8.5) or water (pH 7.0–8.5) to the center of the QIAquick membrane and centrifuge the column for 1 min. Alternatively, for increased DNA concentration, add 30 μl elution buffer to the center of the QIAquick membrane, let the column stand for 4 min, and then centrifuge.
CAUTION
: Residual ethanol from Buffer PE will not be completely removed unless the flow-through is discarded before this a Ensure that the elution buffer is dispensed directly onto the QIAquick membrane for complete elution of bound DNA. dditional centrifugation
Back to Dicovery Protocol Index
Aptamer Characterization
DIG Labelling
DIG Labelling
Bill Of Materials: You could see a complete BoM here.
Amount of time: 5 hours.
Total  costs: 454,56 € (depending on the PCR reagents and without being sponsored.
CAUTION
: You can do the DIG labelling the same day as the day one of the ELONA assay.
ADVICE
: For us, IDT have worked fine and was easy to make the modifications needed.
Order the same primers you use for the PCR amplification but adding in the 5’ extreme the Digoxigenin molecule.
To determine, the number of cycles needed to label the aptamers, make a PCR reaction mixture of a final volume of 50 µl, for each round to want to check. Add the following reagents to PCR tubes as shown below:
ADVICE
: You can use a different tube for each cycle, or a single tube and extract 5 µl for each cycle.
CAUTION
: Digoxigenin tends to bind with herself and  a band will appear in the negative control. Make a 3 negative control and take them at the cycles 10, 15 and 20.
Perform the PCR amplification with the same conditions you use in the SELEX protocol and choose the PCR samples at the following cycles: 5, 10, 15 and 20.
Order the same primers you use for the PCR amplification but adding in the 5’ extreme the Digoxigenin molecule.
ADVICE
: You can use a different tube for each cycle, or a single tube and extract 5 µl for each cycle.
ADVICE
: You can use a different tube for each cycle, or a single tube and extract 5 µl for each cycle.
Order the same primers you use for the PCR amplification but adding in the 5’ extreme the Digoxigenin molecule.
Order the same primers you use for the PCR amplification but adding in the 5’ extreme the Digoxigenin molecule.
Elona
Elona
Bill Of Materials: You could see a complete BoM here.
Amount of time: 2 days.
Total  costs: 314 € (depending on the PCR reagents and without being sponsored.
DAY 1
Coat a NUNC96-well plate with the protein of interest and BSA (negative control) in aptamer buffer or coating buffer with 100 ng/well ( 2ng/μl, 100 μl each well). Incubate overnight  4ºC with agitation (260rpm).
ADVICE
: The protein will bind to the well surface by itself, so the use of coating solution it is not necessary.
DAY 2
Wash 3x200 µl with PBS 1x-Tween 0,1%. Remove the drops after the last wash.
CAUTION
: Be careful to do not touch the well and remove the protein.
Block the plate with 200 µl PBS 1x BSA 5% for 1 hour (260 rpm).
ADVICE
: We recomend to use a multichannel pipette.
Structure 2 µg/µl, 1.5 µg/µl and 0.5 µg/µl of the aptamers (the population you want to check as well as the initial population) marked with digoxigenin as you usually do in the buffer you have done the selection.
ADVICE
: You do not need to purify the PCR labelling to perform the assay. As we recommend you it is better to measure directly from the gel.
Wash 3x200 µl with PBS 1x tween 0,1%. Remove the drops after the last wash.
CAUTION
: Be careful to do not touch the well and remove the protein.
Add 100 µl/well of the structured aptamers. Incubated for 1 hour..
Wash 3x200 µl with PBS 1x BB. Remove the drops after the last wash.
Add anti-body antidigoxigenin (100µL/well) preparing 1:1000 dilution in Aptamer buffer-BSA 0,2%. Incubate at room temperature for 1h.
CAUTION
: We use the selection buffer because it already contains Mg. If you use a different one for the Selex protocol dilution of the aptamer is PBS 1X-Mg 0,2%BSA
Wash 3 x 200µL with PBS 1x Tween 0,1%.
Add 100 µL/wall of ABTS. Read the absorbance (405 nm) every 10 min for 1h.
ADVICE
: We recomend you to buy the ABTS than comes diluted and with the oxygene peroxide.
The plaque would look like this
CAUTION
: Do not add aptamers to the negative control wells.
Aptamer electrode
Sinthesis of the electrode
Bill Of Materials: You could see a complete BoM here .
Amount of time: 2 day
Total  costs: 220€ (with sponsors).
Selecting the electrode
There are so many scaffolds to join the Aptamers (or the DNA). Our choice was based on the kind of measuring hardware that we have used, a potentiostat. For this variety of measuring system you need a 3-electrodes system (working, reference and counter electrodes). The other parameters of the electrode was choose as follows:
We choose Dropsens as our provider, because they are one of the standards in the field, and they are relatively near to our laboratory.
The material of the working electrode was choose as carbon, modified to include gold nanoparticles. The carbon have better electrochemical window than gold or silver (check this post for more information) and gold are the ideal substrate to join DNA (It only have to be thiolated).
Ordering the DNA
To run the first Proof of Concept  we ordered a commercial Thrombin aptamer. Some tips have been took into account for the aptamer adaptation to electrode binding:
Between the DNA and its thiolation in its 5’, we have include a 6 carbon chain after the thiol modification and 15 thymes before the aptamer sequence. The purpose of this modifications was to separate the aptamer from the electrode surface aiming to ensure enough conformational flexibility of the molecule.
ADVICE
: The IDT code for this modification is  /5ThioMC6-D/
We have order the aptamers to Integrated DNA Technologies as they are one of the competition sponsors.
CAUTION
: As the thiolated ends are considerably unstable, they are shipped as they oxidized form. To treat your electrodes with this aptamers you need to reduce them with DTT or TCEP. You could find a complete protocol of this process here.
Aptamer Bounding
[Optional] Depending on your electrodes, it needs to be pre-treated to ensure the correct aptamer binding. For this purpose pipette 50 uL of  H2SO4  0.5M until the electrode are covered and perform 10 cyclic voltammograms from 0V to 1.25V at 100 mV/s of scan rate.
ADVICE
: TWith Dropsens electrodes there is no need to perform this step.
Wash the electrodes three times with deionized water and let them dry under an extraction hood air flow.
Follow the protocol to structure the aptamers in their individual binding buffers. If you have follow our SELEX protocol, check the buffers and their own structuration steps in this protocol. Make sure that you have enough concentration for the next step.
Drop 10 uL of the 5uM solution of aptamer in its own Binding Buffer (if you have selected the aptamer with our protocol check the SELEX protocol) above the working electrode.
Incubate overnight in an humidity chamber.Incubate overnight in an humidity chamber.
PAUSE POINT
: Let the electrodes incubating overnight
Wash the electrodes three times with deionized water and let them dry under an extraction hood air flow.
To remove the excess of DNA, treat the electrodes with 10 uL of β-Mercaptoethanol for 50 minutes in a humidity chamber.
Wash the electrodes three times with deionized water and let them dry under an extraction hood air flow.
CAUTION
: When incubating the different solutions and buffers with the electrode, do NOT let the solution evaporate. Be sure of making the step in a humidified chamber.
Electrodes testing with cyclic voltammetry
First of all you must calibrate the ideal concentration of the electrode donor solution. For this purpose ferricyanide redox couple  (K3Fe(CN)6and K4Fe(CN)6) was used above a raw electrode without aptamer. After our experiments the optimal concentration was found to be 5mM of each chemical in a 0.1 KCl solution.
ADVICE
: In our experience, this concentration could be different depending on things like the quality of your reactives or your electrodes. We encourage you to adjust this value experimentally making some dilutions (0.5X, 2X…).
Cover the electrode with a ferricyanide droplet and connect it to the potentiostat.
Run a preliminar cyclic voltammetry test with a stardart parameters. The ones that have fits better with our hardware (Rodeostat) was one cycle from -0.3V to 0.3V, with a current limit of 1000 uA, a sample rate of 100 Hz and a scan rate of 0.05 mV/s.
After the test have finished, adjust the parameters (voltage range and current limit) to fit the complete curve in your range. A typical Cyclic Voltammetry curve may have a shape similar to a duck.
Once you have calibrated the test for raw electrode is time to compare the results between itself and the electrode with aptamer bonded. If your binding process have succeed you must find a decreasement between the current peak of the electrode with aptamers compared to the raw one. This decreasement is proportional to the quantity of aftamer bonded to the electrode surface as they are obstructing the electrons flow through the electrode surface.
To calibrate the minimum quantity of aptamer that you need to achieve your detection limits, you may carry out the same experiment but with different concentrations of the aptamer.  At the end of this experiment you will be able to correlate the quantity of the aptamer bonded to your electrode with the cyclic voltammetry peak.
Now your electrode is prepared to test it with your protein. You may set an incubation time and temperature depending on the individual interaction between each aptamer and protein. There is no protocol here. This will be a part of your individual results and its your work from now to adapt this protocol to your individual case. If your experiment succeed you will see a decreasement of current in the electrode with higher concentrations of aptamers. To see the outcome of our experiments check the results in the device section.
Troubleshooting
If after the measurement you can’t see any kind of signal or your noise is too high compared to the signal the problem may be caused by different sources. You should analyze the following things:
Check if you don’t have enough current (current under 30 uA in Rodeostat) or  you have current but so much noise. In the first case you should check the wiring because your circuit isn’t close. In the second case you should check possible noise sources near your system (like magnets or fluorescent lights).
ADVICE
: We have faced this kind of issue when we have connected the electrodes through alligator clips. The connection was not stable enough and the system have low current and too much noise.  The solution was substituting the clips with oscilloscope probes. You could also solder the electrode to copper wires or consider to buy a commercial electrode adaptor.
Check if in the electrode are appearing air bubbles when you run the measurement. This is a symptom of a wrong electrode connection. Check if you haven’t switch the connections of reference and counter electrode.
If no current decreasing have achieve in the electrode with aptamers compared to the raw electrode check if you have followed correctly reduced the aptamers before following the binding protocol. Also check the concentration of aptamers that you have cast above the electrode.
