Team:Madrid-OLM/AptCharacterization
Madrid-OLM
Team
Members
Attribution
Sponsors
Project
Description
Contribution
Improve
Demonstrate
Judging Form
Aptamers
Overview
Discovery
Characterization
Computational
Improvement
Binding to Electrode
Device
Overview
First Prototype
Final Prototype
Overview
Microfluidics
Electronics
Experiments
Interlab
Protocols
Aptamer Related
Manufacturing
Human Practices
Integrated Human Practice
Education and
Engagements
Collaborations
Awards
Hardware
Applied Design
Entrepreneurship
Measurement
Medals
Aptamer Characterization
Biochemical characterization of the aptamers
One of the most important steps when you are working with aptamers, especially if you are looking for aptamers for a downstream application, is to demonstrate that aptamers have high affinity, specificity and selectivity for its substrate. It is logical to think that any aptamer with flexible conformational structure would also demonstrate interaction with many off-targets having similar motifs. However, aptamers with a defined ground state would bind only to specific targets with high affinity.
Affinity is a term that makes reference to the strength of the interaction that exists between a molecule (aptamer in this case) and its target. The key variable to measure if you want to assess the binding capacity of an aptamer is the dissociation constant (Kd).
Aptamers that show low dissociation constants have strong interactions with their targets. In this case, we have developed aptamers from scratch. Because of this, it was very important to know if we ended up with a specific aptamer against his target and which was their affinity.
To solve this problem, we decided to attempt to do an ELONA (Enzyme-Linked Oligonucleotide Assay). ELONA is a biochemical method based on enzyme-linked immunosorbent assay (ELISA). You have a plate with your target protein linked in the surface and you test different concentrations of your aptamer instead of doing so with a first antibody (like in an ELISA assay).
It has been described different ELONA formats for aptamer-based protein detection. We have chosen one of them, which uses an anti-digoxigenin antibody to recognize an aptamer previously labelled with digoxigenin. This antibody is conjugated with a peroxidase enzyme, and once it adds ABTS with hydrogen peroxide, it will be responsible for the colourimetric reaction which will be detected.
ELONA is a quantitative experiment and allows to calculate the Kd of the aptamers tested. This method improves the ones than previous iGEM teams have used to measure the affinity of aptamers like the Lyon team that uses polyacrylamide gels. This is a qualitative experiment that only tells if the aptamer binds to the target protein but does not give you further information about the interaction ( Kd),  neither allow you to compare between different sequences, to choose the one with the best affinity.
DIG-Labelling
We choose to mark our aptamers with digoxigenin, so the second antibody could detect the aptamers binding to the protein, as it would do in a normal ELISA assay. We choose this method because it is performed as a normal PCR, but with modified primers. We followed the steps described in the “ELONA Protocol” and successfully labelled the sixth round of OLE-E1:
Figure 1: Agarose gel after 15 cycles of amplification: line 1 negative control; line 2 initial population; line 3 round 6 of OLE-E1, line 4 round 6 of OLE-E1.
ELONA
During the DIG-labelling, we only obtained enough nanograms of aptamers to test only one concentration, 1 ng/µL. Because of this, we did a triplicate, instead of a duplicate like in our protocol. We also didn’t use thrombin as positive control, because we run out of it while tuning up the protocol. We used another proven aptamer ceded from the Victor González research group.
Elona Protocol
Results
We performed a successful ELONA assay, as it can be seen in the positive control. As the positive control is an aptamer that we know that has a high affinity, it shows a high absorbance, proving that we were able to measure aptamers binding to their target.
The absorbance from round 6 increased in comparison to the initial population. This fact showed that we were able to recover the sequences that actually binds as well as discard the ones that don’t.
The results also tell, together with the qPCR ones,  that a few more rounds will be needed to achieve the desired affinity. Because if you compare the absorbance of our round 6 to the positive control, it is still low.
We did an additional round of SELEX, restricting the time to half an hour to force the selection, but we did not repeat an ELONA with the new round because of the lack of time.
Figure 2: Absorbance of the initial population against negative control (BSA), round 6 of OLE-E1 against negative control, noise and positive control.
