<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_Madrid-OLM_Computationaimprovement skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><DIV id="top_title"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:Madrid-OLM/Computationaimprovement</SPAN></H1></DIV><DIV id="HQ_page"><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><P><TITLE>Madrid-OLM</TITLE></P><DIV class="nav-container"><NAV id="menu1" class="bar bar--sm bar-1 bar--absolute bar--dark" data-scroll-class="100px:pos-fixed" style="background-color: rgba(0, 0, 0, 0); box-shadow: 0px 0px 25px 0px rgba(0, 0, 0, 0) !important;"><DIV class="container"><DIV class="row"><DIV class="col-lg-12 col-md-12 text-left-xs text-left-sm" style=" background: #686868; opacity: 0.9;"><DIV class="bar__module text-right"><UL class="menu-horizontal text-left"><LI class="dropdown"><SPAN class="dropdown__trigger subttt">Team</SPAN></LI><LI class=""><A class="subtt" href="https://2018.igem.org/Team:Madrid-OLM/Team">Members</A></LI><LI class=""><A class="subtt" href="https://2018.igem.org/Team:Madrid-OLM/Attributions">Attribution</A></LI><LI class=""><A class="subtt" href="https://2018.igem.org/Team:Madrid-OLM/Sponsor">Sponsors</A></LI></UL></DIV></DIV></DIV></DIV><LI class="dropdown"><SPAN class="dropdown__trigger subttt">Project</SPAN></LI><LI class=""><A class="subtt" href="https://2018.igem.org/Team:Madrid-OLM/Description">Description</A></LI><LI class=""><A class="subtt" href="https://2018.igem.org/Team:Madrid-OLM/Contribution">Contribution</A></LI><LI class=""><A class="subtt" href="https://2018.igem.org/Team:Madrid-OLM/Improve">Improve</A></LI><LI class=""><A class="subtt" href="https://2018.igem.org/Team:Madrid-OLM/Demonstrate">Demonstrate</A></LI><LI class=""><A class="subtt" href="https://igem.org/2018_Judging_Form?team=Madrid-OLM">Judging Form</A></LI></NAV></DIV></DIV></DIV></DIV><LI class="dropdown"><SPAN class="dropdown__trigger subttt">Aptamers</SPAN></LI><LI class=""><A class="subtt" href="https://2018.igem.org/Team:Madrid-OLM/Aptamer">Overview</A></LI><LI class=""><A class="subtt" href="https://2018.igem.org/Team:Madrid-OLM/AptDiscovery">Discovery</A></LI><LI class=""><A class="subtt" href="https://2018.igem.org/Team:Madrid-OLM/AptCharacterization">Characterization</A></LI><LI class=""><A class="subtt" href="https://2018.igem.org/Team:Madrid-OLM/Computationaimprovement">Computational Improvement</A></LI><LI class=""><A class="subtt" href="https://2018.igem.org/Team:Madrid-OLM/ElectrodeIntegration">Binding to Electrode </A></LI></DIV></DIV><LI class="dropdown"><SPAN class="dropdown__trigger subttt">Device</SPAN></LI><LI><A class="subtt" href="https://2018.igem.org/Team:Madrid-OLM/OurHardware">
                                                                    Overview
                                                                </A></LI><LI><A class="subtt" href="https://2018.igem.org/Team:Madrid-OLM/FirstPrototype">                                                                   First Prototype
                                                                </A></LI><LI class="dropdown"><SPAN class="dropdown__trigger subttt">Final Prototype</SPAN></LI><LI><A class="subtt" href="https://2018.igem.org/Team:Madrid-OLM/FinalPrototype">
                                                                                            Overview
                                                                                        </A></LI><LI><A class="subtt" href="https://2018.igem.org/Team:Madrid-OLM/HardawareMicrofluidics">
                                                                                            Microfluidics
                                                                                        </A></LI><LI><A class="subtt" href="https://2018.igem.org/Team:Madrid-OLM/HardwareElectronics">
                                                                                            Electronics
                                                                                        </A></LI><LI class="dropdown"><SPAN class="dropdown__trigger subttt">Experiments</SPAN></LI><LI class=""><A class="subtt" href="https://2018.igem.org/Team:Madrid-OLM/InterLab">Interlab</A></LI><LI class="dropdown"><SPAN class="dropdown__trigger subttt">Protocols</SPAN></LI><LI><A class="subtt" href="https://2018.igem.org/Team:Madrid-OLM/AptamerProtocols">
                                                                                            Aptamer Related
                                                                                        </A></LI><LI><A class="subtt" href="https://2018.igem.org/Team:Madrid-OLM/ProManufacturing">
                                                                                            Manufacturing
                                                                                        </A></LI><LI class="dropdown"><SPAN class="dropdown__trigger subttt">Human Practices</SPAN></LI><LI class=""><A class="subtt" href="https://2018.igem.org/Team:Madrid-OLM/Human_Practices">Integrated Human Practice</A></LI><LI class=""><A class="subtt" href="https://2018.igem.org/Team:Madrid-OLM/Public_Engagement">Education and Engagements </A></LI><LI class=""><A class="subtt" href="https://2018.igem.org/Team:Madrid-OLM/Collaborations">Collaborations</A></LI><LI class="dropdown"><SPAN class="dropdown__trigger subttt">Awards</SPAN></LI><LI><A class="subtt" href="https://2018.igem.org/Team:Madrid-OLM/Hardware">
                                                                    Hardware
                                                                </A></LI><LI class=""><A class="subtt" href="https://2018.igem.org/Team:Madrid-OLM/Applied_Design">Applied Design</A></LI><LI class=""><A class="subtt" href="https://2018.igem.org/Team:Madrid-OLM/Entrepreneurship">Entrepreneurship</A></LI><LI class=""><A class="subtt" href="https://2018.igem.org/Team:Madrid-OLM/Measurement">Measurement</A></LI><LI class="dropdown"><SPAN><A href="https://2018.igem.org/Team:Madrid-OLM/Medals" style="font-size: 0.857142857142857em; line-height: 2.166666666666667em; text-transform: uppercase; font-weight: 600; letter-spacing: .5px; color: white; text-decoration: none;">Medals</A></SPAN></LI><TITLE>Computational improvement of the aptamer</TITLE><DIV class="main-container"><SECTION id="home" class="tittle-secc text-center switchable feature-large"><DIV class="container"><DIV class="row justify-content-around"><DIV class="col-md-10 col-lg-8 "><H1 id="Teamtittle">Computational improvement of the aptamer</H1><P class="lead">
                                The target of this section is to improve the affinity of the union between the protein and the aptamer, obtained by the process that has been explained previously, by bioinformatics methods.
                            </P><P class="lead">
                                To carry out this affinity improvement, you can start from only 2 elements: the DNA aptamer sequence and the name or ,preferably, the amino acid protein sequence. However, any extra information that one could find in previous studies could be so helpful as to save time and reduce the margin of acumulative error. The complete path has four different sections:  
                            </P><UL><LI><P class="lead">•	Obtaining the 3D protein structure 
                                    </P></LI><LI><P class="lead">•	Obtaining the 3D aptamer structure (most critical point).
                                    </P></LI><LI><P class="lead">•	A Search of the binding site between protein and aptamer through a docking process. 
                                    </P></LI><LI><P class="lead">•	Study of the union and proposal of mutation in the sequence.
                                    </P></LI></UL><P class="lead">It is very important to understand that this process does not guarantee a better result. However it gives the opportunity to improve the aptamer obtained. Although there are many works with proteins structures and the interaction between them, there are almost no studies that work with nucleic acids in simulation or structure prediction of the structures beyond double strand helix. Therefore any final results obtained should be checked in the laboratory, either to confirm or discard the new sequence obtained. Important to understand that this process does not guarantee a better result. However it gives the opportunity to improve the aptamer obtained. 
                            </P><P class="lead">You could see all the server that we will mention in the link section:
                            </P><A class="btn btn--primary-2 type--uppercase " href="#links" class="inner-link"><SPAN class="btn__text">
                                    Links of interest
                                </SPAN></A></DIV></DIV></DIV></SECTION><SECTION id="protein" class="text-center"><DIV class="container"><DIV class="row"><DIV class="col-md-10 col-lg-8 boxed boxed--border bg--secondary boxed--lg box-shadow"><H2>Obtaining the 3D protein structure</H2><P class="lead">
                                The final result of this section should be a file in .pdb format where we can find the coordinates of every single atom of the protein. That file could be interpreted by bioinformatics programs like Phymol or Chimera and represent the protein in different graphical ways. To obtain it, there are two different paths depending on one fact: has the protein been previously crystallographed?
                            </P><H4>A cristallographed structure exists</H4><P class="lead">
                                We will simply use this proven structure. This will be the most reliable method as it is the experimental result of any research group. To check if there is any previous cristallographed structure we recommend the following simple steps:
                            </P><UL><LI><P class="lead">1.	Access the UniProt server and search for the protein of interest.
                                    </P></LI><LI><P class="lead">2.	Search for the structure description area and check if it has an entry to the Protein Data Bank (PDB) database.
                                    </P></LI><LI><P class="lead">3.	In the positive case, access that PDB entry and download the file that is needed.
                                    </P></LI><LI><P class="lead">4.	In the case of no entry, the steps explained in the next section must be performed.
                                </P></LI></UL><P class="lead">Clarification, the PDB is a universal database where are the 3D structure of all proteins which have been obtained its structure experimentally. That is, any structure that we find there is quite reliable and proven. However, predictions of structures are not included, owing to this procedure is not experimental.
                            </P><H4>A cristallographed structure does not exist</H4><P class="lead">
                                In this case it is necessary to start with a structure prediction process. Although this is not the optimal method because we could start to accumulate error, we can obtain very reliable results. Several different methods can be used for this purpose. We recommend the following two and in this order:
                            </P><UL><LI><P class="lead">1.	<B>Homology methods:</B> This method looks for sequences of proteins, that are included in the PDB, that have a high similarity with our protein and adopt its 3D structure. If the result shows a similarity of more than 30%  in the entire sequence, we could be certain about the structure. For this step, we recommend the Swiss-Model server. Otherwise, if it gives an inferior value, we should move on to the next step.
                                    </P></LI><LI><P class="lead">2.	<B>I-Tasser (Iterative Threading ASSEmbly Refinement): </B> This server uses several methods, as they define themselves: “a hierarchical approach to protein structure and function prediction. It first identifies structural templates from the PDB by multiple threading approach LOMETS, with full-length atomic models constructed by iterative template fragment assembly simulations.” This method is currently the most reliable and has won the last 6 editions of the CAP tests.
                                    </P></LI><LI><P class="lead">3.	As an extra option we mention <B>Robetta: </B> It is an ab-initio simulation method, whose aim is to create the protein from scratch, using physical simulation and brute force. It works better for short sequences as it may take a long time to get reliable results.
                                    </P></LI></UL><H4>Energy minimization.</H4><P class="lead">
                                This last step is completely recommendable whatever the origin of the file with the structure is. It is necessary to have the structure in a state, as stable as possible. To minimize the structure we can use two strategies:
                            </P><UL><LI><P class="lead">1.	<B>Quick vacuum minimization</B> This is the simplest method. For this purpose, we will use the Chimera program and  we will choose the &quot;Energy minimization&quot; option. Then we will adjust the parameters that suit our requirements better.
   </P></LI><LI><P class="lead">2.	<B>Minimization in aqueous medium: </B> It is a complex method. We recommend to look for a tutorial to understand how to use the programs and some of the basic concepts involved. We will use the VMD program (Visual Molecular Dynamics). It will be necessary to build a structure around the protein with the &quot;Automatic PSF builder&quot; tool, which includes water molecules and ions. Subsequently, a simulation of energy minimization will be carried out with the tool &quot;AutoIMD&quot; which in turn calls an external program (NAND) that must be installed on the PC. Finally, the resulting file should be saved and opened in Chimera to eliminate all water molecules and leave only the structure of the protein.

                                    </P></LI></UL><P class="lead" style="margin-left:10%; margin-right:10%;">Figure 1. Comparison of the structure of Ole E1 before (purple) and after (blue) an energy minimization process.
                            </P><P class="lead" style="margin-left:10%; margin-right:10%;">Figure 2. Ole E1 after an energy minimization process represented as ribbon and as ASA (Accessible Surface Area). Made by Chimera tool.
                            </P></DIV></DIV></DIV></SECTION><SECTION id="aptamer" class="text-center"><DIV class="container"><DIV class="row"><DIV class="col-md-10 col-lg-8 boxed boxed--border bg--secondary boxed--lg box-shadow"><H2>Obtaining the 3D aptamer structure</H2><P class="lead">
                                Previously there were two options, that the protein had a known structure or not. In this case it is a DNA chain that we have personally obtained in the laboratory, so the possibility of finding a structure published for that exact sequence is almost zero. 
                            </P><P class="lead">
                                The only possible way is to try to predict its structure. Nowadays, this action presents a series of drawbacks. While for proteins there are numerous methods and servers with high reliability, for nucleic acids there are hardly any servers and prediction methods.  We must consider that he structure of proteins has always been very important and varied depending on its function. On the other hand, the most important part of the genetic material has always been its code, whether in the form of DNA or RNA.
                            </P><P class="lead">
                                For the case of RNAs, there is more than one function, that is the reason why in certain occasions  the need to know their structures has been taken into account, although it has not been developed correctly. 
                            </P><P class="lead">
                                We propose a series of steps to predict the structure of the aptamer. Although this predicted structure will not be reliable and it is important to understand this, it can fulfill  our objective of proposing an improvement in the initial sequence. 
                     </P><H4>Step 1: Obtaining the secondary structure</H4><P class="lead">
                                As there is no single server, as in proteins, to obtain a final result giving only the sequence, we have to divide the process into different stages: In this first stage, we need to obtain a secondary structure of the single strand of DNA. The server we recommend is the MFold server of the University of Albany. It provides both a graphic map of the links between bases and the secondary structure in text format (in the Vienna format that we need later), as we show in this example:
                            </P><P class="lead" style="text-align: center;">
                                GTGACGTAGGTTGGTGTGGTTGGGGCGTCAC 
                                (((((((.................)))))))
                            </P><P class="lead" style="margin-left:20%; margin-right:10%;">Figure 3. Example image of the links between base pairs that the server provides
                            </P><H4>Step 2: Prediction of 3D structure</H4><P class="lead">
                                The next step is to predict the tertiary structure. The only server capable of doing that nowadays is ROSIE and it requires the secondary structure that we have previously obtained in Vienna format in addition to the sequence. This server predicts RNA structures (not DNA), so it will be necessary to change the T for the U in the sequence.
                            </P><P class="lead">
                                The result obtained will be a series of three-dimensional RNA structure proposals in .pdb format, so a posteriori it will be necessary to make file modifications to eliminate the excess atoms and modify the links so it can be identified as a DNA chain.
                            </P><P class="lead">
                                We must take into account that the algorithm used by this server is not as refined as in proteins and usually commits some deviations in the ab initio simulation. It is particularly very important to know if our aptamer has incorporated some ions because this method does not take into account aptamers that incorporate external molecules. In these cases, the predicted structure deviates a lot from the real one.
                            </P><P class="lead">
                                Subsequently, it will be necessary to perform a process of energy minimization as explained in the previous section, choosing the method that is preferred to have the most stable molecule.
                            </P><P class="lead" style="margin-left:10%; margin-right:10%;">Figure 4. Thrombin aptamer structure comparison between the crystallographed in a lab (blue) and the predicted (grey). We can see that the blue one has an ion that conditions the structure.
                            </P><P class="lead"><B>Fact: </B>We tried to fine-tune this step with an aptamer already characterized by thrombin. Although theoretically, the protocol was logical, the results obtained with the set-up led us to conclude that we did not have it ready yet due to the high error rate. 
                            </P></DIV></DIV></DIV></SECTION><SECTION id="dockin" class="text-center"><DIV class="container"><DIV class="row"><DIV class="col-md-10 col-lg-8 boxed boxed--border bg--secondary boxed--lg box-shadow"><H2>Docking procces and mutation proposal</H2><P class="lead">
                                After obtaining the structures of the protein and the aptamer, we need to know in which area and with which orientation the interaction between them takes place. To do so, we will study which pairs of bases and which amino acids are involved in this interaction, and then we will propose a change of 1 or 2 bases in order to make this interaction stronger.
                            </P><H4>Docking</H4><P class="lead">
                                The docking is the computational simulation method in which the ideal orientation of an organic molecule is calculated when interacting with another one forming a stable complex (if it exists). Generally this process is designed for protein-protein interactions. This is the reason why most servers are optimized for these cases. However, we recommend using the NPDock server. This server allows to perform the simulation of the interaction between a protein and a single-stranded nucleic acid molecule, having to identify whether it is DNA or RNA before sending the work.
                            </P><P class="lead" style="margin-left:10%; margin-right:10%;">Figure 5. The interaction between thrombin and an aptamer simulated by the server NPDock.
                            </P><H4>Proposal of mutation</H4><P class="lead">                           With the .pdb file obtained from the previous step, we only have the most subjective step. In this case we have to perform a study of the interaction and see which atoms interact with each others. For this purpose, the most useful thing is to use a visualizer like the ones that we mentioned at the beginning (Chimera or Phymol). With the evaluation of the different options we will have to interpret if these interactions could improve by making any change in the aptamer.
                            </P><P class="lead">
                                The proposed mutation cannot be accepted blindly. It will be necessary to synthesize the proposed aptamer and check in the laboratory if its affinity is superior to that of the original aptamer.
                            </P></DIV></DIV></DIV></SECTION><SECTION id="links" class="text-center"><DIV class="container"><DIV class="row"><DIV class="col-md-10 col-lg-8 boxed boxed--border bg--secondary boxed--lg box-shadow"><H2>Links of interest</H2><UL><LI><P class="lead"><A href="http://www.uniprot.org/">UniProt:</A> Database with all proteins and the related information.
                                    </P></LI><LI><P class="lead"><A href="http://www.rcsb.org/">PDB:</A>D atabase with all 3D shapes of protein, nucleic acids, and complex assemblies.
                                    </P></LI><LI><P class="lead"><A href="http://pymol.org/2/">Phymol:</A> Is a user-sponsored molecular visualization system on an open-source foundation, maintained and distributed by Schrödinger.
                                    </P></LI><LI><P class="lead"><A href="http://www.cgl.ucsf.edu/chimera/"> Chimera:</A> A highly extensible program for interactive visualization and analysis of molecular structures and related data, including density maps, supramolecular assemblies, sequence alignments, docking results, trajectories, and conformational ensembles
                                    </P></LI><LI><P class="lead"><A href="http://www.ks.uiuc.edu/Research/vmd/"> VMD:</A> A molecular visualization program for displaying, animating, and analyzing large biomolecular systems using 3-D graphics and built-in scripting.
                                    </P></LI><LI><P class="lead"><A href="http://swissmodel.expasy.org/"> Swiss-Model:</A> A fully automated protein structure homology-modelling server, accessible via the ExPASy web server, or from the program DeepView (Swiss Pdb-Viewer).
                                    </P></LI><LI><P class="lead"><A href="http://zhanglab.ccmb.med.umich.edu/I-TASSER/"> I-tasser:</A> A hierarchical approach to protein structure and function prediction.
                                    </P></LI><LI><P class="lead"><A href="http://robetta.bakerlab.org/"> Robbeta:</A> Ab-inition full chain protein structures prediction server.
                                </P></LI><LI><P class="lead"><A href="http://unafold.rna.albany.edu/?q=mfold/DNA-Folding-Form"> Albany University MFold:</A> Web server for the prediction of secondary structures in DNA single-strand chains.
                                    </P></LI><LI><P class="lead"><A href="http://rosie.rosettacommons.org/rna_denovo/submit"> Rossie:</A> Web server for ab-initio RNA single-stand structure prediction.
                                    </P></LI><LI><P class="lead"><A href="http://genesilico.pl/NPDock"> NPDock:</A> Web server for modelling of RNA-protein and DNA-protein complex structures. 
                                </P></LI></UL></DIV></DIV></DIV></SECTION></DIV></BODY></HTML>