Team:Tec-Monterrey/Description
Tec de Monterrey
Project
Description Design Interlab Notebook Safety Demonstrate Medals
Parts
Parts Improved parts Parts list
People
Team Collaborations Attribution Sponsors
Human Practices
Human Practices Public & Educational Engagement
Modeling
Overview D.E. System Quasi-Steady State Protein Production
Home
PROJECT
PARTS
HUMAN PRACTICES
MODELING
PEOPLE
Description
Overview
System
References
Design
Background
Module 1
Module 2
System
Adaptation
InterLab
Procedure
Results
Conclusion
Notebook
Logbook
Protocols
Safety
Biosafety
Our Lab
Project Safety
Demonstrate
Medals
Parts
Overview
Basic Parts
Composite Parts
Collection
Improved Parts
Previous Parts
Improved Parts
Parts List
Team
Header
Team Members
PI 1
PI 2
Instructor
Advisors
Collaborators
Collaborations
Collabs
Meetups
Attributions
What we have done
General Support
HP Support
Design/Wiki
Fund raising
Thanks
Sponsors
Human Practices
Header
Intro
Action Plan
Journey
Integrated
World
Public & Educational Engagement
Magazine articles
Presentations
Social Media
Congress
Tour
Workshops
Seminar
Overview
Introduction
Biological System
D.E. System
Kinetic Diagram
Equations
Parameters
Quasi-Steady State
Equations
Protein Production
Equations
Experimentation
Parameters
Header
Overview
System
References
Overview
CRISPR-Cas technology has the capability of storing information. This year, iGEM team Tec-Monterrey aims to use the CRISPR-Cas system to store specific DNA sequences in the genome of E. coli in order to save information about the environment surrounding the bacteria. To make this possible, Cas1-Cas2 proteins, which create the protospacer acquisition in the CRISPR system, are used to insert a synthetic DNA sequence in the CRISPR array within the genome of the bacteria. This synthetic sequence is produced by a second system, called SCRIBE. The final step of our project is reading out the inserted DNA sequence. Specific primers for polymerase chain reaction (PCR) are used to amplify a section of the CRISPR array where the sequence is inserted. Taking together both systems, our project intends to act as a biological tape recorder capable of sensing external stimuli in the environment and storing their presence in the genome.
System Functionality
The construct shown in Figure 1 codes for the SCRIBE system adaptation.
This system generates single-stranded DNA inside of living cells in response to gene regulatory signals.
It is composed of 3 main components:
A reverse transcriptase protein
One
msr
RNA moiety which acts as a primer for the RT.
A second RNA moiety, called
msd
, which represents a template for the reverse transcriptase. This sequence contains the
message of interest
.
Figure 2: Retrotranscriptase action
This hybrid
RNA-ssDNA
molecule contains the
message of interest
.
This message has a PAM sequence, a crucial component for the identification by the CRISPR-Cas complex. (Figure 3)
Figure 4: Cas 1 and Cas 2 construct
These proteins (Figure 5) have to detect the PAM sequence in order to enable their function as endonucleases, in other words, their activity to work as scissors that cut DNA sequences.
Figure 6: Message insertion
Figure 1: Construct for SCRIBE system
Then, the retrotranscriptase acts on the
msd
(RNA) sequence and produces a hybrid RNA-ssDNA molecule called
msDNA
, as shown in Figure 2.
Figure 3: Message acquisition by Cas proteins
Here is where our beloved
CRISPR-Cas
complex comes into place. (Figure 4)
CRISPR-Cas
is a prokaryotic immune system that protects bacteria from phages and plasmids. With this complex, foreign DNA, in the form of spacers, is incorporated into the bacteria’s genome, specifically at the  CRISPR locus. This acquisition of new DNA generates a memory of each “infection”.
Several Cas proteins are involved in this system, but Cas1 and Cas2 are the only proteins needed for acquisition of new spacers.
Figure 5: Cas 1 and Cas 2 complex
Once the PAM sequence is detected, the Cas1-Cas2 complex cuts the target sequence and integrates it into the bacteria’s genome, as shown in Figure 6.
References
Sheth, R. U., Yim, S. S., Wu, F. L. & Wang, H. H. Science. 358, 1457–1461 (2017).
A. Levy et al. Nature. 520, 505–510 (2015).
S. L. Shipman et al. Science. 353, aaf1175 (2016).
E. S. Lander. Cell. 164, 18–28 (2016).
Nuñez, J. K., Kranzusch, P. J., Noeske, J., Wright, A., Davies, C. & Doudna, J. Nat. Struct. Mol. Biol. 21, 528–534 (2014).
Díez-Villaseñor, C., Guzmán, N., Almendros, C., García-Martínez, J. & Mojica, F. J. RNA Biology. 10, 792-802 (2013).
Yosef, I., Goren, M. & Qimron, U. Nucleic Acids Research. 40, 5569–5576 (2012).
Yosef, I., Shitrit, D., Goren, M., Burstein, D., Pupko, T. & Qimron U. Proceedings of the National Academy of Sciences. 110, 14396-14401 (2013).
Nuñez, J. K. Mechanism of CRISPR–Cas Immunological Memory. (2016). Doctoral dissertation, UC Berkeley
Xue, C., Whitis, N., Sashital, D. Molecular Cell. 64, 826–834 (2016).
Church, G., Gao, Y., Kosuri, S. Science. 337, 1628 (2012).
Panda, D. et al. 3 Biotech. 8, 239 (2018).
Farzadfard, F., Lu, T. Science. 346 (2014).
Contact Us
Av. Eugenio Garza Sada 2501
64849 Monterrey, Mexico
Sponsors
