"
Page
Discussion
View source
History
teams
Log in
 
Team:UQ-Australia/Project
From 2011.igem.org
Home
UQ-Australia
iGEM 2011
Team
Students
Supervisors
Project
Parts
Data
Modelling
Kinetics
Synchronisation
Microscopy
Notebook
Protocols
April
May
June
July
August
September
Safety
Human Practices
Patenting
Outreach
Collaboration
Attributions
The human circadian rhythm drives many important processes in the body in accordance with the sleep/wake cycle. A characteristic of this biological clock is the periodic oscillation of gene expression. Current parts in the Registry designed to regulate periodic oscillations of gene expression have shown limited success.
Here we demonstrate the feasibility of a biological clock being standardised as a set of BioBrick parts.
Our network is controlled by an engineered promoter, Plac/ara, which features both an activator and a repressor domain. This controls the production of downstream genes to activate other inducible promoters, pBAD and GlnAp2, eventually leading to the production of a repressor protein, lacI, which inhibits Plac/ara, resulting in oscillatory expression. This project shows the feasibility of standardising the biological clock in E. coli and grounds further development for applications in regulated drug/hormone delivery and ion channel control.
Project Details
The project has been split into categories:
Development of BioBricks
Experimental methods to be fully recorded in the Notebook
Modelling of the circuit
Modelling of the kinetics of the oscillating cells
Modelling of the synchronisation of oscillating cells
Thorough evaluation of the safety issues regarding UQ-Autralia's entry in iGEM
Human practices
Raising awareness of synthetic biology
Providing a solution to the patenting issue that iGEM is facing
Together, this forms the UQ-Australia project for the 2011 iGEM.
Motivation and Background
In humans, the circadian rhythm is controlled by several core genes that operate via a series of feedback loops (Figure 1). A transcription–translation negative-feedback loop powers the system, with a delay between the transcription of these genes and the negative feedback being a key factor that allows the system to oscillate [1]. A 'master clock' located in the Suprachiasmatic Nucleus coordinates the timing of the rhythm, but external factors such as light exposure play a large role in regulating the exact
Figure 1: The gene network responsible for establishing the circadian rhythm in humans [1]
There has been much effort put into reconstructing and determining the exact nature of this system, as the impact the circadian rhythm has on our lifestyle and cellular processes are still not very well understood. In particular, it is believed the circadian rhythm could exert an effect on everything from body temperature, feeding behaviour and appetite, hormone secretion and metabolism, glucose homeostasis, and cell-cycle progression [2].
Consequently, there have been a number of efforts to reconstruct this clock in a mammalian system for further study. In particular, both Hong et al. [3] and Tigges et al. [4] utilize the inducible tTa system to drive the expression and oscillation of genes in their synthetic networks.
It was our initial plan to construct a similar oscillatory system ourselves using standardized parts which could be added to the registry and then utilized by other iGEM teams working in mammalian cells. Such as system could be 'plugged in' to any number of different outputs and would allow for the timely and regular expression of the genes it drives. However, we encountered a number of issues around the intellectual property protecting of certain elements we wanted to use and so decided to switch to a simpler system in E. coli.
Our System
In prokaryotic systems, there are a number of tools available for synthetic biologists to construct gene networks such as the circadian clock we hope to replicate. In particular, we are going to make use of the inducible promoters glnAp2 and pBAD and their inducers glnG and araC. These components have been used in synthetic oscillators from both Atkinson et al. [5] and Stricker et al. [6]. In addition, we plan on using an engineered promoter denoted as Plac/ara, which features a pBAD activation domain as well as several lacO sites, meaning it is both inducible and repressible depending on the input. This promoter was first engineered by Lutz & Bujard [7].
Based on the work from these previous studies, we designed our own system using components that have been previously utilized to construct other synthetic clocks (Figure 2). We decided to make a 3 component system rather that a dual component system as the issues encountered by some other synthetic clocks suggest that there might not have been enough time between the transcription of various components [8]. As such, including this extra step to add a delay should ensure our network is able to reach an oscillatory expression pattern.
Figure 2: The genetic system we are aiming to construct in order to produce an oscillating gene network. An engineered promoter, Plac/ara, which features both an activator and a repressor domain controls the production of downstream genes to activate other inducible promoters, pBAD and GlnAp2. These eventually lead to the production of a repressor protein, lacI, which inhibits Plac/ara, resulting in oscillatory expression.
Once our system has been constructed, we hope to be able to make it even more stable by integrating the necessary components into the E coli chromosome. In this manner, we will be able to remove any interfering factors from our system, and will also be able to control the gene dosage level of our system. This has been a particular problem for other synthetic oscillators, with researchers being unable to control for the uptake of plasmid and therefore having cell-to-cell differences in the rate of oscillation [4, 6].
Kuhlman & Cox [9] have developed a method for integrating large synthetic constructs into the bacterial chromosome, and kindly provided us with the necessary plasmids should we reach this stage. Furthermore, Dr. Alex Ninfa [5] provided us with E. coli strain 3.300L*G, which has both glnG and lacI knocked out so as not to interfere with the network we construct.
Details on the modelling behind out circuit can be found on the Modelling page.
Experimental Outline & Goals
Our main aim for the experimental work is to produce each component as a standardized and characterised BioBrick part that can be re-used by other teams in future to build their own oscillators. In the long run, we would hope to produce an E. Coli strain incorporating the three main 'modules' of our oscillator. In this manner, other teams could use this strain and simply transform the bacteria with a plasmid driven by one of our inducible promoters in order to obtain regular, oscillating expression.
To achieve these goals, we must first source all the components we require and standardize them to include BioBrick sites. Standard BioBrick assembly can then be used to construct our main components, before we will utilize a pre-determined system and protocols [9] to attempt to integrate our modules into the chromosome. In the process of developing our parts and constructing the system, we hope to characterise all our parts. In particular, confirming the inducibility and kinetics of our promoters will allow us to refine our models.
Other activities
Our safety analysis is on the Safety page, while the Human Practices discussion is in the Human Practices section.
References
[1] Gallego, M & Virshup, DM 2007. "Post-translational modifications regulate the ticking of the circadian clock", Molecular Cell Biology, vol. 8, pp. 139-148.
[2] Takahashi, JS, Hong, HK, Ko, CH & McDearmon, EL 2008. "The genetics of mammalian circadian order and disorder: implications for physiology and disease", Genetics, vol. 9, pp. 764-775.
[3] Hong, HK, Chong, JL, Song, W, Song, EJ & Jyawook, AA et al. 2007. "Inducible and reversible Clock gene expression in brain using the tTa system for the study of circadian behaviour", PLoS Genetics, vol. 3, no. 2, 324-338.
[4] Tigges, M, Marques-Lago, TT, Stelling, J & Fusseneger, M 2009. "A tunable synthetic mammalian oscillator", Nature, vol. 457, pp. 309-312.
[5] Atkinson, MR, Savageau, MA, Myers, JT & Ninfa, AJ 2003. "Development of genetic circuitry exhibiting toggle switch or oscillatory behaviour in Escherichia coli", Cell, vol. 113, pp. 597-607.
[6] Stricker, J, Cookson, S, Bennett, MR, Mather, WH, Tsimring, LS & Hasty, J 2008. "A fast, robust and tunable synthetic oscillator", Nature, vol. 456, pp. 516-520.
[7] Lutz, R & Bujard H 1997. "Independent and tight regulation of transcriptional units in Escherichia coli via the LacR/O, the TetR/O and araC/I1-I2 regulatory elements", Nucleic Acids Research, vol. 25, no. 6, pp. 1203-1210.
[8] Purcell, O, Savery, NJ, Grierson, CS & di Bernardo, M 2010. "A comparative analysis of synthetic genetic oscillators", Journal of the Royal Society, vol. 7, pp. 1503-1524.
[9] Kuhlman, TE & Cox, EC 2010. "Site-specific chromosomal integration of large synthetic constructs", Nucleic Acids Research, vol. 38, no. 6, pp. e92.
Retrieved from " http://2011.igem.org/Team:UQ-Australia/Project "
Recent changes
What links here
Related changes
Special pages
My preferences
Printable version
Permanent link
Privacy policy
Disclaimers
