<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_HokkaidoU_Japan_other-systems skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:HokkaidoU Japan/other-systems</SPAN></H1><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><DIV id="menuContainer"><DIV class="mainContainer clearfix"><UL><LI><A href="https://2015.igem.org/Team:HokkaidoU_Japan">HOME</A></LI><LI><A href="https://2015.igem.org/Team:HokkaidoU_Japan/Description">MICROBUSTERS
                        </A></LI><LI><A href="https://2015.igem.org/Team:HokkaidoU_Japan/Description">Project Description</A></LI><LI><A href="https://2015.igem.org/Team:HokkaidoU_Japan/ag43">Ag43 Secretion System</A></LI><LI><A href="https://2015.igem.org/Team:HokkaidoU_Japan/other-systems">Other Systems</A></LI><LI><A href="https://2015.igem.org/Team:HokkaidoU_Japan/future">Future Work</A></LI><LI><A href="https://2015.igem.org/Team:HokkaidoU_Japan/Modeling">Modeling</A></LI></UL><LI><A href="https://2015.igem.org/Team:HokkaidoU_Japan/Parts">PARTS
			</A></LI><LI><A href="https://2015.igem.org/Team:HokkaidoU_Japan/Parts">Team Parts</A></LI><LI><A href="https://2015.igem.org/Team:HokkaidoU_Japan/Composite_Part">Composite Part</A></LI><LI><A href="https://2015.igem.org/Team:HokkaidoU_Japan/Notebook">NOTEBOOK
			</A></LI><LI><A href="https://2015.igem.org/Team:HokkaidoU_Japan/Notebook/protocol">Protocol</A></LI><LI><A href="https://2015.igem.org/Team:HokkaidoU_Japan/Notebook/ecoli"><I>E. coli</I></A></LI><LI><A href="https://2015.igem.org/Team:HokkaidoU_Japan/Notebook/lcasei"><I>L. casei</I></A></LI><LI><A href="https://2015.igem.org/Team:HokkaidoU_Japan/Practices">HUMAN PRACTICES</A></LI><LI><A href="https://2015.igem.org/Team:HokkaidoU_Japan/Team">ABOUT US
			</A></LI><LI><A href="https://2015.igem.org/Team:HokkaidoU_Japan/Team">Team</A></LI><LI><A href="https://2015.igem.org/Team:HokkaidoU_Japan/Attributions">Attributions</A></LI><LI><A href="https://2015.igem.org/Team:HokkaidoU_Japan/Collaborations">Collaborations</A></LI><LI><A href="https://2015.igem.org/Team:HokkaidoU_Japan/Safety">Safety</A></LI></DIV></DIV><DIV id="bannerContainer"><DIV class="mainContainer"><DIV style="float:left; margin-left:45px;"><P class="bannerWord">Microbusters</P></DIV></DIV></DIV><DIV class="mainContainer"><DIV id="sidebarContainer"><UL class="clearfix"><LI><A href="https://2015.igem.org/Team:HokkaidoU_Japan/Description">Project Description</A></LI><LI class="sub"><A href="https://2015.igem.org/Team:HokkaidoU_Japan/Description#overview">Overview</A></LI><LI class="sub"><A href="https://2015.igem.org/Team:HokkaidoU_Japan/Description#background">Background</A></LI></UL><UL class="clearfix"><LI><A href="https://2015.igem.org/Team:HokkaidoU_Japan/ag43">Ag43 Secretion System</A></LI><LI class="sub"><A href="https://2015.igem.org/Team:HokkaidoU_Japan/ag43#overview">Overview</A></LI><LI class="sub"><A href="https://2015.igem.org/Team:HokkaidoU_Japan/ag43#design">Design</A></LI><LI class="sub"><A href="https://2015.igem.org/Team:HokkaidoU_Japan/ag43#experiments">Experiments</A></LI><LI class="sub"><A href="https://2015.igem.org/Team:HokkaidoU_Japan/ag43#result">Result</A></LI><LI class="sub"><A href="https://2015.igem.org/Team:HokkaidoU_Japan/ag43#conclusion">Conclusion</A></LI></UL><UL class="clearfix"><LI><A href="https://2015.igem.org/Team:HokkaidoU_Japan/other-systems">Other Systems</A></LI><LI class="sub"><A href="https://2015.igem.org/Team:HokkaidoU_Japan/other-systems#his-tag">His-tag Inactivation System</A></LI><LI class="sub"><A href="https://2015.igem.org/Team:HokkaidoU_Japan/other-systems#co-expression">Co-expression System</A></LI><LI class="sub"><A href="https://2015.igem.org/Team:HokkaidoU_Japan/other-systems#discussion">Discussion</A></LI></UL><UL class="clearfix"><LI><A href="https://2015.igem.org/Team:HokkaidoU_Japan/future">Future Work</A></LI></UL><UL class="clearfix"><LI><A href="https://2015.igem.org/Team:HokkaidoU_Japan/Modeling">Modeling</A></LI></UL></DIV><DIV id="contentContainer"><H1>Other Systems</H1><H2>Overview</H2><P>Other than Ag43 secretion system, we tried several other ways to mass-produce AMPs (Antimicrobial-peptides) in bacterial cells. We documented them below. One system utilizes 10x His-tag and the other system utilizes HLA (Human Lactalbumin) Family. Both of them utilize the inclusion bodies which is naturally equipped in <I>E. coli</I>.</P><P> We were not able to succeed in producing thanatin in these systems. Detailed discussions are on the last part of this page.</P><H2 id="his-tag">His-tag Inactivation System</H2><H3>Overview</H3><P>One of the ways to produce a large quantity of a recombinant toxic protein in <I>E. coli</I> is to aggregate them as inclusion bodies (IBs) (※1). IBs are insoluble protein aggregates observed in bacteria overexpressing heterologous genes whose products fail to attain a solubility (de Groot <I>et al.</I> 2008). Protein aggregation can lose the activity of the protein to some extent. So we optimized inclusion bodies to achieve large amounts of the toxic peptide, thanatin.</P><P>It’s known that using a histidine tag as a fusion protein for protein expression did affect the formation of inclusion bodies. In our project, we intended to express thanatin with a 10x His-tag (※2) at the N terminus as an inclusion body, then cleave of a His-tag with some protease (TEV protease (※3)) leading successfully to express thanatin in a soluble form. </P><UL style="list-style-type: none; list-style-image: none;"><LI>(※1) Inclusion bodies (IBs)</LI><P>Inclusion bodies are major protein aggregates commonly occurring in recombinant bacteria when protein of inserted gene is overexpression.</P><LI>(※2) His-tag</LI><P>His-tag (histidine-tag) is an amino acid motif consisting of successive histidine residues, often fused at the N- or C-terminus of the target protein. His-tag is often used for affinity purification of tagged recombinant proteins expressed in <I>Escherichia coli</I>.</P><LI>(※3) TEV protease</LI><P>TEV protease is a highly sequence-specific cysteine protease derived from Tobacco Etch Virus (TEV). TEV protease uses a cysteine as its catalytic nucleophile. The native cut sites of this protease is Glu-Asn-Leu-Tyr-Phe-Gln-/-(Gly/Ser) where ‘/’ is the cleaved peptide bond.</P></UL><H3>Design</H3><P>First, we made the construct shown in Fig. 1 (P<SUB>lac</SUB> - B003x - Thanatin - dT on 1C3). Thanatin is under control of inducible promoters; P<SUB>lac</SUB> promoter. In the absence of IPTG, lacI binds to lac operator region and represses the transcription, therefore expression of gene downstream  of P<SUB>lac</SUB> is negatively controlled. By adding IPTG, this inducer interacts with lacI and changes lacI's conformation.. Finally, lacI is released from promoter region, leading recruiting of RNA polymerase and initiation of transcription. We predict that <I>E. coli</I> could be difficult to grow due to leak of P<SUB>lac</SUB> (Even if we don’t activated P<SUB>lac</SUB> by IPTG, gene downstream would be transcribed a little and thanatin could kill host cells).</P><P>Second, we made the construct shown in Fig. 2 (P<SUB>lac</SUB> - B0034 - 10x His-tag - Thanatin - dT on 1C3).We designed TEV cleave site between 10x His-tag and Thanatin.</P><P class="caption">Fig. 1 The simple construct of BBa_K1714002</P><P class="caption">Fig. 2 The simple construct of BBa_K1714003</P><H2 id="experiments">Experiments</H2><P>We tried to test that 10x His-tag fused with thanatin and formation of inclusion body can surely keep thanatin inactive and that TEV protease can cleave His-tag and get thanatin to be reactive. </P><H4>How assay</H4><P>First of all, we made sure that the constructs have correct sequence by sequencing them.If we get the expected constructs, we transform it into JM109 and compare the growth curve among different inductive conditions.</P><OL><LI>Activity test for Thanatin and 10x His-tag - thanatin</LI>
→Comparing growth curve between the samples one is induced by IPTG and the other is not induced. Diluting the bacterial culture which is enough pre-cultured by 0.1 OD<SUB>600</SUB> (Optical Density in λ=600 nm) in 2 mL LBC(Lysogeny Broth liquid culture including chloramphenicol). 
 <LI>Adding 2 µL of IPTG into one and DW into the other.</LI><LI>Measuring OD<SUB>600</SUB> of these samples once in an hour. </LI></OL><H4>Reactivation of thanatin by TEV proteinase</H4><OL><LI>Giving damage to the bacteria body by ultrasonic treatment.</LI><LI>Adding TEV protease and buffer to the homogenate</LI><LI>Centrifuge 5,000 rpm for 2 min</LI><LI>Conducting MIC test for the supernatant</LI></OL><H3>Result</H3><P>We made two constructs, <A href="http://parts.igem.org/wiki/index.php?title=Part:BBa_K1714002" target="blank">BBa_K1714002</A> (P<SUB>lac</SUB> - B0034 - thanatin - dT on 1C3) (Fig. 1) and <A href="http://parts.igem.org/wiki/index.php?title=Part:BBa_K1714003" target="blank">BBa_K1714003</A> (P<SUB>lac</SUB> - B0034 - 10x His-tag - TEV - thanatin - dT) (Fig. 2). But we made sure that these constructs have correct sequence only by colony PCR.</P><P>We also made <I>E. coli</I> (DH5α) which includes <A href="http://parts.igem.org/wiki/index.php?title=Part:BBa_K1714002" target="blank">BBa_K1714002</A> and <A href="http://parts.igem.org/wiki/index.php?title=Part:BBa_K1714003" target="blank">BBa_K1714003</A> plasmid for activity test of thanatin and 10x His-tag - thanatin and then attempted to carry out the activity test (How Assay 1). However, we couldn’t subculture colony of the LBC plate to liquid culture and colony in the LBC couldn’t survive in liquid culture.</P><H3>Conclusion</H3><P>Conclusion
 we could make sure that <A href="http://parts.igem.org/wiki/index.php?title=Part:BBa_K1714001" target="blank">BBa_K1714001</A> has almost correct sequence only by colony PCR. And we managed to registry the possible expected construct which is P<SUB>lac</SUB> - B0034 - thanatin - dT and P<SUB>lac</SUB> - B0034 - 10x His-tag - thanatin - dT on 1C3 (like Fig. 1,2 ), however we couldn’t analysis sequence of these. The main reason for it is that we couldn’t subculture colony of the LBC plate to liquid culture, it is possibly because <A href="http://parts.igem.org/wiki/index.php?title=Part:BBa_K1714002" target="blank">BBa_K1714002</A> and <A href="http://parts.igem.org/wiki/index.php?title=Part:BBa_K1714003" target="blank">BBa_K1714003</A> are too toxic to survive. In case that the bacteria with these parts can survive they may lose ability to express them for some reasons such as nonsense mutation. Unfortunately, that means production of AMPs by <I>E. coli</I> in large scale is hard through the method of IPTG induction and His-tag inactivation system.  
</P><H2 id="co-expression">Co-expression System</H2><H3>Overview</H3><P>If we need antimicrobial peptides (AMPs) (※4), we usually get them by chemical synthesis method because its amino acid residues are less than          another peptide. However chemical synthesis method is unsuitable for the case that a large amount of isotopically labeled peptides are essential for structural study by NMR. In this case, we’ll choose the recombinant synthesis method but, of course, AMPs are harmful to their host bacteria. To prevent host toxicity, there is the solution which utilizes expression with inclusion bodies. In addition, in case that it is not promising to form inclusion bodies, we use a partner protein which has the high potential to form inclusion bodies with co-expressed protein of interest.</P><UL style="list-style-type: none; list-style-image: none;"><LI>(※4) ABF-2</LI><P>ABF-2 is one of the AMPs derived from <I>Caenorhabditis elegans</I>. ABF-2 belongs to the CSαβ superfamily which has both α-helix and antiparallel β-sheet stabilized by some disulfide bonds and interacts to cell membrane like thanatin. It is difficult for ABF-2 to form inclusion bodies alone. So the recombinant synthesis method by co-expression system may be effective.</P><LI>Lactalbumin family</LI><P>We choose four Lactalbumin family proteins, concretely HLA (Human α-Lactalbumin), BLA (Bovine α-Lactalbumin), HLZ (Human Lysozyme), BLZ (Bovine Lysozyme), as partner proteins. Because these proteins have negative charge and ABF-2 have positive charge like thanatin, they will interact and form inclusion bodies. There is study which used these proteins for formation of inclusion bodies with ABF-2. (2013. S. Tomisawa et al.)</P><LI>Application of co-expression system to Thanatin</LI><P>Because, as referred to above, thanatin also has positive charge, it is projected that thanatin interacts Lactalbumin family proteins which have negative charge. For reference, the isoelectric point (pI) of ABF-2 is 9.1, on the other hand, pI of thanatin is 10.47. On the other hand, pI of HLA is 4.7, BLA is 4.8, HLZ is 9.3 and BLZ is 6.5. We think the value of pI of thanatin is enough similar with ABF-2 to apply this method for ABF-2 to thanatin. </P></UL><H3>Design</H3><UL><LI>Constructs</LI><P>First of all, we made the construct like Fig. 3 (P<SUB>BAD</SUB> - B003x - HLA family - dT on 1C3). </P><P class="caption">Fig. 3 The construct of partner proteins</P><P>On the other hand, we also made the construct like Fig. 4 (P<SUB>lac</SUB> - B0034 - AMPs - dT on 1C3).</P><P class="caption">Fig. 4 The construct of AMPs</P><P>Then, we picked up the P<SUB>lac</SUB> - B0034 - AMPs - dT as fragment from the latter construct and introduce it upstream shown in Fig. 5.</P><P class="caption">Fig. 5 The complete construct of Co-expression system</P><P>Finally, we can induce only thanatin or ABF-2 (AMPs) by IPTG (glucose) and only partner proteins (HLA family).</P></UL><H3>Experiments</H3><P>We research which partner proteins is the most suitable for formation of inclusion body with thanatin by using the constructs described above. (HLA, BLA, HLZ or BLZ) And we also research the effect of intensity of RBS by changing it. (B_0032, B_33 or B_0034)
 In addition, we can get the activation form of thanatin by heating or ionic strengths. </P><UL><LI>How assay</LI><P>First of all, we make sure that the constructs have correct sequence by sequencing.</P><P>If we get the expected constructs, we transform it into <I>E. coli</I> (in this case, we used DH5α) and compare the growth curve among different inductive condition.</P><OL><LI>Activity test for ABF-2 and thanatin</LI>
→Comparing growth curve between the samples one is induced by IPTG and the other is not induced.
   <OL><LI>Diluting the bacterial culture which is enough pre-cultured by 0.1 OD<SUB>600</SUB> 	(Optical Density in λ=600nm) in 2 mL LBC.(Lysogeny Broth liquid culture 	including chloramphenicol) </LI><LI>Adding 2 µL of IPTG into one and DW into the other.</LI><LI>Measuring OD<SUB>600</SUB> of these samples once in an hour. </LI></OL><P>*In this experiment, we should also make sure that the AMPs induced by IPTG are surely effective to growth curve by changing inductive timing or amount of IPTG.</P><LI>Repressing test of AMPs’ activity by co-expression system</LI>
→Comparing growth curve among the different induction patterns, induced by IPTG,  arabinose or not.
   <OL><LI>Diluting the bacterial culture which is enough pre-cultured by 0.1 OD<SUB>600</SUB> (Optical Density in λ=600nm) in 2 mL LBC.(Lysogeny Broth liquid culture	including chloramphenicol)</LI><LI>Adding 2 µL of IPTG and 200 µL of arabinose into sample 1, 2 µL of DW and 200 µL of arabinose into sample 2, 2 µL of IPTG and 200 µL of DW into sample 3 and 202 µL of DW into sample 4.</LI><LI>Measuring OD<SUB>600</SUB> of these samples once in an hour.</LI></OL><P>*In this experiment, we should change different patterns of constructs, for example changing strength of RBS or kind of partner proteins.</P><H3>Result</H3><P>We could make only two constructs,
      <A href="http://parts.igem.org/wiki/index.php?title=Part:BBa_K1714004" target="blank">BBa_K1714004</A>
      (P<SUB>BAD</SUB>-B0034-BLA family-dT on 1C3) (Fig. 6) and 
      <A href="http://parts.igem.org/wiki/index.php?title=Part:BBa_K1714001" target="blank">BBa_K1714001</A>
      (P<SUB>lac</SUB>-B0034-ABF-2-dT on 1C3) (Fig. 7). We made sure that these constructs have correct sequence but 
      <A href="http://parts.igem.org/wiki/index.php?title=Part:BBa_K1714001" target="blank">BBa_K1714001</A> 
      is not sequenced correctly in some part of it.</P><P class="caption">Fig. 6 The simple construct of <A href="http://parts.igem.org/wiki/index.php?title=Part:BBa_K1714004" target="blank">BBa_K1714004</A></P><P class="caption">Fig. 7 The simple construct of <A href="http://parts.igem.org/wiki/index.php?title=Part:BBa_K1714001" target="blank">BBa_K1714001</A></P><P>We also made E.coli (DH5α) which includes 
   <A href="http://parts.igem.org/wiki/index.php?title=Part:BBa_K1714001" target="blank">BBa_K1714001</A>
   plasmid for activity test of ABF-2 and carried out the activity test. (How Assay 1) There is a result of the test as Fig. 8.</P><P class="caption">Fig. 8 The growth curve of <I>E. coli</I> including <A href="http://parts.igem.org/wiki/index.php?title=Part:BBa_K1714001" target="blank">BBa_K1714001</A> with IPTG or not </P><P>As you can see, we can’t say that ABF-2 in 
   <A href="http://parts.igem.org/wiki/index.php?title=Part:BBa_K1714001" target="blank">BBa_K1714001</A>
   has antimicrobial activity.</P><H3>Conclusion</H3><P>However we could make sure that 
   <A href="http://parts.igem.org/wiki/index.php?title=Part:BBa_K1714001" target="blank">BBa_K1714001</A>
   has almost correct sequence, we couldn’t make sure that ABF-2 in 
   <A href="http://parts.igem.org/wiki/index.php?title=Part:BBa_K1714001" target="blank">BBa_K1714001</A> 
   has antimicrobial activity. It means that the ABF-2 doesn’t express appropriately by any cause. As one of the supposable reason, because ABF-2 is toxic to <I>E. coli</I>, the mutated one which is not toxic to <I>E. coli</I> is selected in transfer <I>E. coli</I> into new culture again and again. We can also imagine that this is something to do with that the sequence of 
   <A href="http://parts.igem.org/wiki/index.php?title=Part:BBa_K1714001" target="blank">BBa_K1714001</A> 
   wasn’t perfectly correct or we could get the colonies of <I>E. coli</I> including AMPs by DH5α which is less repressive efficiency of lac promotor than JM109. (We couldn’t get constructs including thanatin by DH5α)</P><P>We couldn’t registry the expected construct which is P<SUB>lac</SUB> - B0034 - ABF-2 - dT - P<SUB>BAD</SUB> - B0034 - BLA family - dT on 1C3 (like Fig. 3) but we can co-express these two constructs in another way. It’s transferring one or the other construct to another plasmid which has another origin and transforming them together. However there is a potential of fail to repress the toxicity by co-expression system because of the difference of copy number of plasmids. For this problem, we should find solution experimentally through trial and error for example changing the strength of RBS, the amount of Inductive materials or the timing of induction.</P><P>In this year, because of the lack of time, we couldn’t accomplish the expected experiments in this project. However if we accomplished all experiments we expected and found the best pattern to make inclusion body for thanatin, we may consider so many things. According to the paper (2013 S. Tomisawa et al.), BLA is the best partner protein to make inclusion body for ABF-2. If the value of isotonic point (pI) is main factor to form inclusion body, we can expect that BLA is also the best one for thanatin because the value of pI of thanatin is similar with ABF-2. On the other hand, if we find another protein the best one to form inclusion body for thanatin, the relation with interest protein and partner protein is not only due to the value of pI but other factors for example structure and so on. Originally, because there are so many proteins which have positive charge, it is difficult to form inclusion body only by pI factor. So, if we made all expected constructs, we can’t surely repress the toxicity of thanatin by co-expression system of these four kinds of Lactalbumin family proteins.</P><P>Anyway, we should experiment for longer time and get any result about co-expression system. By doing this, not only we can standardize the expression of proteins which have host toxicity and make smooth it, but also we may found something new about AMPs which have many question marks in structural part.</P><H2 id="discussion">Discussion</H2><P>Both systems have a process in which AMPs are controlled by only P<SUB>lac</SUB>. This may be one of the causes of the failure because P<SUB>lac</SUB> is leaky promoter. In that process, AMPs may be expressed because of leak of P<SUB>lac</SUB>. </P><P>As AMPs have so strong antimicrobial-activity, if there is the slightest leak of AMPs, it’s considered to kill <I>E.coli</I> from both inside and outside of the cell.</P><P>Even though you can get the probable colony consisted by <I>E. coli</I> which have objective plasmid DNA including AMPs controlled by only P<SUB>lac</SUB> and get the mini-prep products of the plasmid DNA, it’s likely mutated one, because it’s considered that the toxicity to host cells places selective pressures on mutation of loss of toxicity.</P><P>We found the expression of AMPs with simple idea so difficult through these experiments.</P><H4>Reference</H4><OL><LI><P>Tomisawa, S., Hojo, E., Umetsu, Y., Ohki, S., Kato, Y., Miyazawa, M., ... &amp; Aizawa, T. (2013). Overexpression of an antimicrobial peptide derived from C. elegans using an aggregation-prone protein coexpression system. <I>AMB Express, 3</I>(1), 45.</P></LI></OL><P style="text-align:left; float:left;"><A href="https://2015.igem.org/Team:HokkaidoU_Japan/ag43">←Ag43 Secretion System</A></P><P style="text-align:right;"><A href="https://2015.igem.org/Team:HokkaidoU_Japan/future">Future Work→</A></P></OL></UL></DIV></DIV></DIV></DIV></DIV></DIV></BODY></HTML>