<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_Cork_Ireland_Notebook skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:Cork Ireland/Notebook</SPAN></H1><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><P><TITLE>Notebook</TITLE></P><DIV id="globalWrapper"><DIV id="top_menu_14" class="noprint">Loading menubar.....</DIV><HEADER><NAV><UL><LI><A href="https://2015.igem.org/Team:Cork_Ireland">Home</A></LI><LI><A href="https://2015.igem.org/Team:Cork_Ireland/Project">Our Project</A></LI><LI><A href="https://2015.igem.org/Team:Cork_Ireland/Design">Design</A></LI><LI><A href="https://2015.igem.org/Team:Cork_Ireland/Practices">Outreach</A></LI><LI><A href="https://2015.igem.org/Team:Cork_Ireland/Team">Team</A></LI><LI><A href="https://2015.igem.org/Team:Cork_Ireland/Notebook">Notebook</A></LI><LI><A href="https://2015.igem.org/Team:Cork_Ireland/Parts">Parts</A></LI><LI><A href="https://2015.igem.org/Team:Cork_Ireland/Collaborations">Collaborations</A></LI><LI><A href="https://2015.igem.org/Team:Cork_Ireland/InterlabStudy">Interlab Study</A></LI><LI><A href="https://2015.igem.org/Team:Cork_Ireland/Attributions">Attributions</A></LI><LI><A href="https://2015.igem.org/Team:Cork_Ireland/Achievements">Achievements</A></LI></UL></NAV></HEADER><DIV id="content"><DIV id="main_col"><DIV id="notebook"><H2>Wetlab Progress</H2><DIV><H3 id="w1">Week 1: 25-May-2015</H3><P>
			After many hours of pooling ideas together the team decided on the project idea of optimising and further developing a molecular 
			diagnostic tool in March. Team members entered the lab in late May to begin training in molecular biology and microbiological laboratory 
			techniques such as PCR, bacterial transformations, gel electrophoresis  and so on.
			</P></DIV><DIV><H3 id="w2">Week 2: 1-June-2015</H3><P>
			The team entered the lab and spent a great deal of time investigating the best way to approach the project to ensure maximum productivity 
			with a limited amount of time. Cost of the project was also greatly considered. The following parts from the distribution kit were transformed 
			into DH5α cells.
			</P><UL><LI>BBa_K592026</LI><LI>BBa_K592100</LI><LI>BBa_K952003</LI><LI>BBa_J06702</LI><LI>BBa_K1357009</LI><LI>BBa_K395602</LI></UL><P>
			The team ran into trouble on the first large transformation, so it was repeated over the course of the week. 
			Liquid culture of the following parts were prepared:
			</P><UL><LI>BBa_K1357008</LI><LI>BBa_J23119</LI><LI>BBa_K59602</LI><LI>BBa_K592100</LI><LI>BBa_K592026</LI></UL><P>
			Plasmid preps of the above parts was also performed. A restriction digest using XbaI and PstI was performed to see if the 
			plasmid backbone and the respective parts were present. During the course of the week the first biobrick assembly of 
			parts BBa_J23119 (Promotor) &amp; BBa_K1357008 (Purple fluorescent protein) was attempted after a restriction digest had 
			been performed. The different assembly steps were repeated over the week in order to try achieve a successful assembly.
			</P></DIV><DIV><H3 id="w3">Week 3: 8-June-2015</H3><P>
			This week was spent experimenting with different ligations such as; BBa_J23119 (Promotor) as the backbone &amp; 
			part BBa_K1357008 (Purple fluorescent protein) as the insert. These ligations were performed to work on the colour readout 
			of the system. The various ligations were transformed into competent cells to see if the ligations were successful, if successful 
			the CFUs would glow the colour as determined by the biobrick part. Primers for the detector plasmid construction were designed by the 
			team members and ordered from IDT. This allowed the detector to be constructed using an additional PCR step.
			</P><P>
			The ligated plasmids were extracted from the transformed cells, digested and ran on a gel to see if the ligation was successful. 
			This was done in an attempt to understand the logistics of the ligations and the <I>E.coli</I> cells. The products of detector construction 
			were ran on agarose gels at various stages in construction to determine if the construction was successful at every stage.
			</P><P> 
			Also this week the sensitivity of the Digested-HPV55 detector was tested. Detectors constructed by PCR and a quadruple digest 
			were tested to see if different detector construction could influence the sensitivity of the system. The PCR detectors were tested 
			after a minimum of a Dpn1 digest, PCR clean-up &amp; quadruple digest. Further steps in the detector construction included a decoy reaction 
			to remove all template DNA and another PCR clean-up.
			</P></DIV><DIV><H3 id="w4">Week 4: 15-June-2015</H3><P>
			Further ligations of colour producing biobrick parts were performed to optimise the colour readout of the system. 3A assembly with 
			Kanamycin and Chloramphenicol was performed as an alternative to digestion, gel extraction and ligation.
			</P><P>The transformation efficiency of the DH5α cells was tested this week using the iGEM Transformation efficiency kit. </P><P>
			This week marked the beginning of the testing of the specificity of the HPV-55 detector plasmid. 
			Both the Digested-HPV55 and the PCR-HPV55 were tested. This was done by setting up a detector reaction using a series of 
			mutated oligos that were designed by the team and synthesised by IDT.
			</P><P>
			The GFP plasmid was tested to see if any background was present, this allowed the results from the specificity testing to 
			be analysed with a higher degree of confidence. 
			</P><P>
			After each detector construction, the functionality of the detectors were tested by gel electrophoresis 
			(to check for detector presence at right MW) and transformations were carried out using both positive and 
			negative controls.
			</P></DIV><DIV><H3 id="w5">Week 5: 22-June-2015</H3><P>The team signed up to participate in the Interlab study.</P><P>
			A HPV-30 base pair detector was constructed and a series of mutated oligos were designed and synthesised in order 
			to test the specificity of this detector.
			</P><P>
			It was investigated to see if increased or decreased target length would affect the specificity of the detector 
			using oligos illustrated in figure below.
			</P><P>HPV-55 detector was reacted with the 30bp top strand target, HPV-30 was reacted with the 55bp top strand target. </P><P>
			For the colour readout of the system, the RFP from the transformation efficiency kit was purified from transformed 
			cells and implemented into the detector reaction protocol as a transformation efficiency control. RFP in dilutions of 1:100, 
			1:200, 1:1000 were tested.
			</P><P>
			The interlab study was started this week, the different parts needed were taken from the distribution kit and transformed. 
			After successful transformations of the three promoters and the GFP generating device, plasmid preps were done on the devices 
			in order to prepare them for cloning.
			</P></DIV><DIV><H3 id="w6">Week 6: 29-June-2015</H3><P>The HPV-30bp detector was tested with the series of mutated oligos in order to see if it was more specific that the HPV-55 detector.</P><P>
			While investigating the specificity of the system, the different temperatures of the detector reaction were 
			investigated and calculated in order to minimise hybridisation (see WHATEVER SECTION WILL HAVE THOSE PROTOCOLS OR PUT IN LINK). 
			This was investigated using the HPV-55bp detector. This week the work into developing differently timed detector reactions with 
			different temperatures to minimise non-specific binding of DNA to the target site were tested in the plasmid. This was investigated thoroughly 
			over the course of the next few weeks.
			</P><P>
			A great variation in transformation efficiency was noted between different aliquots of cells. 
			This lead to the development of a new transformation protocol. For the construction of the devices for the interlab study, 
			two methods of assembly were trialed due to difficulties in cloning, the methods were 3A assembly and the traditional 
			restriction digest/ligation method. Two of the devices i.e Device 1 and 2 were constructed in the initial stages but Device 3 
			proved to be more difficult to clone successfully.
			</P></DIV><DIV><H3 id="w7">Week 7: 6-July-2015</H3><P>
			Experimentation into the various detector reaction temperatures continued. The IDT Oligo Analyser was used to calculate the melting 
			temperature based on [Mg+2] of 0.1 and [Na] of 0. A diluted amount of target was also tested, 1 microlitres of a 1 in 10 dilution instead 
			of 1μl of an undiluted target.
			</P><P>
			Overnight liquid cultures were prepared for the constructed devices 1 and 2, the overnights were made in triplicate i.e three biological 
			replicates were made. Colony PCR was done on the non-green fluorescent colonies present on the  plates where the ligated device 3 was transformed, 
			this was done using the VF2 and VR2 primers, the colony PCR done revealed that the correct size device was present in some of the colonies that 
			were screened. Later this week, following more ligations, colonies that showed green fluorescence were evident on the plates, these colonies were 
			taken grown up overnight in a liquid culture and re-transformed to reveal more green-fluorescent colonies from which the three biological replicates 
			were taken.
			</P></DIV><DIV><H3 id="w8">Week 8: 13-July-2015</H3><P>
			Over the last few weeks, a significant variation in transformation efficiency was observed. For this reason, detector reactions 
			on the RFP without the detector plasmid was performed to investigate if the different temperatures had an effect on the RFP.
			</P><P>Inconclusive results were drawn from the different temperatures in the detector reaction so investigation was ceased.</P><P>The team commenced in developing a new transformation protocol. This was investigated this week.</P><P>
			As an alternative to a quadruple digest, detector construction was attempted on an agarose gel. This idea was inspired by reading many 
			papers on the subject. Plasmid preps were made of all overnights that were successful and these were subsequently diluted 1:1000 and re-transformed 
			whereby a greater proportion of the colonies showed green fluorescence. 3 biological replicates were taken from these and by the 14th July, 
			fluorescence was ready to be measures using the plate reader. As controls, colonies without promoters but containing the GFP generator I13504 were 
			grown overnight and a LB media inoculated with antibiotic was  left overnight at 37°C with the rest of the samples.
			</P><P>
			Each of the biological replicates were tested three different times using the plate reader. The average and standard deviation of both technical 
			replicates and biological replicates was obtained. This average was then divided by the OD at 600nm.
			</P><P>
			Digests were performed on all the devices to show that they were the correct size by restriction mapping. These liquid cultures were diluted and 
			re-transformed again to give biological replicates whose fluorescence data was closer in value.
			</P></DIV><DIV><H3 id="w9">Week 9: 20-July-2015</H3><P>
			The specificity of the HPV-30bp detector was further investigated. The different detector reaction temperatures were still being 
			experimented on using the HPV-55bp detector. Plasmid mini-preps were performed on non-specific colonies. These were mismatched oligos 
			that gave a positive result in the testing process. The plasmid mini-preps were performed in order to prepare the samples for sequencing. 
			This was done in an effort to understand what was happening with the detector plasmid and mismatched oligo to indicate a positive result. 
			The team hoped that the results would reveal what the repair mechanisms of the <I>E.coli</I> cells were doing to the detector plasmid.
			</P><P>
			Detector reactions were carried out with phosphorylated and nonphosphorylated oligos and detector plasmids to understand whether or not 
			any difference would be evident in the number of CFUs, this was carried out as a method to detect if <I>E.coli</I> would naturally phosphorylate 
			the oligos  when they’re transformed into <I>E.coli.</I></P></DIV><DIV><H3 id="w10">Week 10: 27-July-2015</H3><P>This week the HPV-55bp and the HPV-30bp detectors were reacted with male mouse genomic DNA samples to assess the background of the detectors.</P><P>
				When the results of the specificity testing for the detectors were analysed, a smaller target yielded higher specificity. 
				For this reason, a HPV-24bp detector was constructed using PCR.
				</P></DIV><DIV><H3 id="w11">Week 11: 3-August-2015</H3><P>A selection of detectors were constructed and their functionality was investigated by gel electrophoresis and detector reactions.</P><P>The week was spent perfecting the different detectors. Both male and female mouse genomic DNA was digested using a variety of 
				enzymes and it was then ran on a gel to see if the digest was successful, however complications arose as DNA was not seen on gel.</P><P>This digested DNA was used to investigate the background levels from the new detectors, thus investigating their specificity.</P></DIV><DIV><H3 id="w12">Week 12: 10-August-2015</H3><P>More male and female mouse genomic DNA was digested using different enzymes to investigate the background levels from the new detectors.</P></DIV><DIV><H3 id="w13">Week 13: 17-August-2015</H3><P>
				More genomic DNA need to be extracted from mice liver, this genomic DNA was verified to be there by using a nanodrop and running it on 
				a gel. Both male and female samples were digested with PstI. Detector reactions were then done using the HPV 24 base pair detector and the 
				digested genomic DNA samples to detect for background due to unspecific hybridisation. This was performed twice in triplicate using normal 
				and high transformation efficiency cells. Four more HPV 30 base pair detectors were also partly constructed this week. The sequences needed 
				for the mycobacterium detector were identified and study was done to construct this.
				</P></DIV><DIV><H3 id="w14">Week 14: 24-August-2015</H3><P>
				Detector reactions were carried out on the SRY 32 base pair detector using PstI digested genomic DNA, this would show whether 
				the detector could be used in a real world application. The data was finalised and entered for the interlab study. 4 more HPV 30 
				base pair detectors were completed early this week. The oligos and primers for the mycobacterium detector were ordered this week.
				</P></DIV><DIV><H3 id="w15">Week 15: 31-August-2015</H3><P>
				The mycobacterium detector was constructed this week, this involved amplifying it by PCR, digesting the PCR product with DPNI, 
				doing a PCR clean-up, digesting the constructed detector with BamHI, KPNI, NtBspQ1,NbBtsI, reacting the product with decoys and doing 
				a PCR clean-up. The detector was then tested by doing detector reactions.
				</P><P>
				The SRY detector was again tested with genomic male and female DNA using highly competent cells. 
				Complications again arose as transformation efficiency was poor and controls did not work.
				</P></DIV><DIV><H3 id="w16">Week 16: 7-September-2015</H3><P>
			Construction of the SRY 32 bp detector was started this week. The initial PCR amplification didn’t work and so the PCR amplification steps were changed to accommodate this. The mycobacterium detector with GFP was transformed. 
			</P></DIV><DIV><H3 id="w17">Week 17: 14-September-2015</H3><P>
			Work was done to test the SRY 32bp detector and the Mycobacterium detector constructed. Biobricks were assembled and prepared for submission to the Registry All results were gathered and finalised for presentation on the wiki.
			</P><P>
			Preparation for Giant Jamboree!
			</P></DIV><DIV><P><A href="https://static.igem.org/mediawiki/2015/8/88/ProtocolsCork.pdf">Click here for our protocols! 
</A></P></DIV></DIV></DIV><DIV class="col"><UL id="fixed_menu"><LI>Quick Links:</LI><LI><A href="#w1">Week 1-2</A></LI><LI><A href="#w3">Week 3</A></LI><LI><A href="#w4">Week 4</A></LI><LI><A href="#w5">Week 5</A></LI><LI><A href="#w6">Week 6</A></LI><LI><A href="#w7">Week 7-8</A></LI><LI><A href="#w9">Week 9-10</A></LI></UL><UL id="fixed_menu2"><LI><A href="#w11">Week 11</A></LI><LI><A href="#w12">Week 12-13</A></LI><LI><A href="#w14">Week 14-15</A></LI><LI><A href="#w16">Week 16</A></LI><LI><A href="#w17">Week 17</A></LI></UL></DIV></DIV><FOOTER><DIV><UL><LI><A href="https://2015.igem.org/Team:Cork_Ireland">Home</A></LI><LI><A href="https://2015.igem.org/Team:Cork_Ireland/Project">Our Project</A></LI><LI><A href="https://2015.igem.org/Team:Cork_Ireland/Design">Design</A></LI><LI><A href="https://2015.igem.org/Team:Cork_Ireland/Practices">Outreach</A></LI><LI><A href="https://2015.igem.org/Team:Cork_Ireland/Team">Team</A></LI><LI><A href="https://2015.igem.org/Team:Cork_Ireland/Notebook">Notebook</A></LI><LI><A href="https://2015.igem.org/Team:Cork_Ireland/Parts">Parts</A></LI><LI><A href="https://2015.igem.org/Team:Cork_Ireland/Collaborations">Collaborations</A></LI><LI><A href="https://2015.igem.org/Team:Cork_Ireland/InterlabStudy">Interlab Study</A></LI><LI><A href="https://2015.igem.org/Team:Cork_Ireland/Attributions">Attributions</A></LI><A href="https://2015.igem.org/Team:Cork_Ireland/Achievements">Achievements</A></UL></DIV><DIV><P>Cork iGEM 2015</P></DIV></FOOTER></DIV></DIV></DIV></DIV></DIV></BODY></HTML>