<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_Cork_Ireland_Project skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:Cork Ireland/Project</SPAN></H1><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><P><TITLE>Our Project</TITLE></P><DIV id="globalWrapper"><DIV id="top_menu_14" class="noprint">Loading menubar.....</DIV><HEADER><NAV><UL><LI><A href="https://2015.igem.org/Team:Cork_Ireland">Home</A></LI><LI><A href="https://2015.igem.org/Team:Cork_Ireland/Project">Our Project</A></LI><LI><A href="https://2015.igem.org/Team:Cork_Ireland/Design">Design</A></LI><LI><A href="https://2015.igem.org/Team:Cork_Ireland/Practices">Outreach</A></LI><LI><A href="https://2015.igem.org/Team:Cork_Ireland/Team">Team</A></LI><LI><A href="https://2015.igem.org/Team:Cork_Ireland/Notebook">Notebook</A></LI><LI><A href="https://2015.igem.org/Team:Cork_Ireland/Parts">Parts</A></LI><LI><A href="https://2015.igem.org/Team:Cork_Ireland/Collaborations">Collaborations</A></LI><LI><A href="https://2015.igem.org/Team:Cork_Ireland/InterlabStudy">Interlab Study</A></LI><LI><A href="https://2015.igem.org/Team:Cork_Ireland/Attributions">Attributions</A></LI><A href="https://2015.igem.org/Team:Cork_Ireland/Achievements">Achievements</A></UL></NAV></HEADER><DIV id="content"><DIV id="main_col"><DIV id="our_project"><DIV id="Summary"><H2>Summary</H2><P>
				Our project is a novel bacterial method of DNA detection. We have developed a customisable, linearised, double 
				stranded plasmid with two sticky overhangs. When the sticky overhangs come into contact with a target sequence, the 
				binding of the DNA sequence to the overhangs circularises the plasmid. The circularised plasmid is then transformed 
				into competent <I>E. coli</I> cells. Bacterial growth of green fluorescent colonies indicates a positive result, therefore 
				the complementary DNA target sequence was present. This system could act as a cheap alternative to both digital and 
				real time PCR, as target DNA fragments are amplified in living cells without the use of a costly PCR machine. This 
				system could potentially be used as a diagnostic or screening tool for viral and/or bacterial infection such as Human 
				Papilloma Virus, <I>Mycobacterium tuberculosis</I>. By improving sensitivity and specificity this system could also be used 
				for the detection of genetic mutations resulting in disease such as cystic fibrosis.</P><P>Over the summer we have:</P><UL><LI>Standardized the readout of the Basehunter system.</LI><LI>Developed a new method for detector plasmid preparation with the goal of reducing false positive results improving sensitivity</LI><LI>Characterized the specificity of the Basehunter system</LI><LI>Developed detectors for shorter target sequences to improve specificity</LI><LI>Designed a new mycobacterium detector based on community consultation</LI><LI>Created a prototype of the Basehunter system that was shipped to and successfully used by a collaborating iGEM team.</LI></UL></DIV><DIV id="proj_desc"><H2>Our Project</H2><P>
				Basehunter is a bacterial-based DNA detection system. This project aims to further develop the specificity, sensitivity 
				and colour readout of the system. Building on last years work, we have designed customizable plasmids that could be modified 
				to detect any specific short DNA sequence. Bacterial growth is a readout for the presence of target DNA.This system potentially 
				provides a cheap, simple alternative to both digital and real time PCR as target DNA fragments are amplified in living cells. 
				Thus making this system applicable in poorly resourced labs in both the developed and developing world as PCR machines are 
				expensive to purchase and can be costly to run.
				</P><P>
				The system works by interaction of the target sequence with the plasmid, and upon subsequent transformation into competent E.coli, 
				results in bacterial growth and the production of a fluorescent signal. The plasmid is linearised with overhanging sequences that 
				are complementary to the target sequence. Upon annealing to the target, the plasmid becomes circularised and functional in a bacterial 
				cell. The presence of this fluorescent signal indicates that the target DNA is present.
				In this project we have been investigating different possible methods of detector (plasmid) construction. The sensitivity of the 
				different detector types has been tested thoroughly. We have also been improving the colour readout of the system.
				</P><P>
				Three different types of DNA detector have been constructed; SRY detector, that detects a DNA sequence only present on the Y 
				chromosome. This is used to investigate the systems ability of detecting target sequences in genomic samples. A HPV detector 
				that detects a sequence in the L1 gene that encodes the capsid of the Human Papilloma Virus 16 &amp; 18. We are investigating different 
				target and detector lengths to see which is both the most sensitive and specific. Lastly, we have also made a detector for <I>Mycobacterium tuberculosis</I>.
				</P><P>
				The specificity of these detectors has been studied by different detector reactions using mutated targets. Increasing the specificity 
				is important as in order to be used as a diagnostic test, the Basehunter system must have a high specificity to avoid a false positive result.
				</P><P>
				The colour readout of the system allows us to identify if the target DNA was present or not. Two different fluorescent colours are 
				present, one acting as a positive readout if the DNA target is present. Another colour acts as a transformation efficiency control. 
				The colour readout was altered by producing different functional fluorescent proteins by a series of ligations. These were then tested 
				and used, thereby significantly improving the colour readout of the system.
				</P><P>
				Cork iGEM aim to further optimise the Basehunter system to detect other DNA sequences from bacterial or viral pathogens, such as 
				<I>Mycobacterium tuberculosis</I>, and genetic mutations that cause diseases such as cystic fibrosis. We also hope to continue to test the 
				sensitivity and specificity of the detector plasmids in order to optimise the system and render it suitable as a diagnostic test. 
				Other potential applications may include using the system to estimate copy number variation in oncogene amplification and also in multiplex 
				detection to detect a number of targets at one time.
				</P><P>
				In our development of this diagnostic system, we have consulted with expert biomedical scientists in both the developed and developing 
				world to discuss the specificity and sensitivity of the test. We also got advice on what diseases lack a simple diagnostic test, to see 
				where our project could be implemented.
				</P><P>
				In addition to this we have investigated the role iGEM has had in emerging synthetic biology start up companies through interviews and 
				surveys conducted with companies taking part in this years IndieBio Start up Accelerator Programme.
				</P><P>
				Basehunter - aims to create a system for bacterial-based DNA detection. Last year, the UCC Ireland 2014 Team designed customizable plasmids 
				that can be activated to detect any specific short DNA sequence. This system potentially provides digital quantification of target DNAs 
				– representing an alternative to digital and real-time PCR. Interaction of the target sequence with the plasmid and subsequent transformation 
				into <I>E. coli</I> results in bacterial cell growth and production of a visible fluorescent signal. As proof-of-principle they also designed 
				a HPV-detector plasmid that detects a short sequence from the human papilloma virus. This could be a rapid and cheap diagnostic tool for 
				pathogenic DNA, and a revolution in resource poor hospital labs in developing countries or in an agricultural or industrial setting.
				</P></DIV><DIV id="standard"><H2>Standardization of Basehunter Readout</H2><H3>Introduction to Results</H3><H3>Standardization of Basehunter readout</H3><H3>Summary:</H3><P>Basehunter relies on transformation of detector/target complexes into <I>E. coli</I> with subsequent colony growth as the readout. 
					The efficiency of bacterial transformation is thus a key variable in the Basehunter protocol. 
					We developed a method to have a built-in control for transformation efficiency in every Basehunter detector reaction as outlined below. 
					This allows results to be reported in standard units that we refer to as <SPAN class="bold">Controlled Colony Count (CCC):</SPAN></P><H3>The problem:</H3><P>
					Transformation efficiency varied greatly between different batches of cells, different experiments and even within experiments. 
					As seen in figure 1, on occasion CFUs achieved did not correlate with concentration of plasmid transformed even within the same 
					experiment. Clearly variation in transformation efficiency will have a significant impact on results obtained with Basehunter 
					detectors and must be controlled for.
					</P><FIGURE><FIGCAPTION>
					Variability of transformation efficiency observed using different concentrations of RFP expressing plasmid. 
					Dilutions of plasmid: Left = 1:100 Middle = 1:200 Right = 1:1000.  The number of colonies obtained does not correlate with plasmid dilution.
					</FIGCAPTION></FIGURE><H2>The solution:</H2><P>
					We reasoned that a plasmid that produces colonies that are distinguishable from those from our detector reaction could be used as a built 
					in transformation efficiency control.  Thus detectors were generated from plasmids encoding constitutively expressed GFP and used in 
					combination with a known amount of a control (uncut) plasmid encoding RFP.  GFP+ and RFP+ colonies could be distinguished by eye, or with 
					the aid of a blue LED lightbox.  In some cases, a basehunter detector without a fluorescent marker was used in combination with a GFP 
					expressing control plasmid, but the principle is the same (e.g. SRY detector). The results of simultaneous transformation of the detector 
					and control plasmid may be seen in figure 2.
					</P><FIGURE><FIGCAPTION>
					Visualization of detector-derived colonies (Green) and control colonies (Red) plasmids using a blue LED lightbox. 
					Left plate = GFP detector + target (green and red colonies present) 
					Right plate = GFP detector without target (only red colonies present)
					</FIGCAPTION></FIGURE><P>
					Since a known amount of the control plasmid is simultaneously transformed with the detector we can calculate the transformation efficiency 
					of each test and express the results in standard units that that acccount of variations in transformation efficiencies. The calculated 
					transformation efficiency is used to amend the number of detector CFUs achieved to a constant transformation efficiency (10^6 CFU per ug DNA).
					We refer to this as the <SPAN class="bold">Controlled Colony Count (CCC)</SPAN>.  Hence results of all experiments could be compared. Calculations for all experimental 
					results were performed as outlined in Table 1 and then plotted as bar graphs.
					</P><H2>Sample calculation of the Controlled Colony Count:</H2><UL><LI>A 1:1000 dilution of control plasmid was used of concentration 114ng/ul, ie. 0.114ng/ul.</LI><LI>1 ul of this dilution was added to the transformation, meaning 0.114ng were transformed and all of this was plated.</LI><LI>Average control derived CFU achieved was 1429.33</LI></UL><UL><LI>Transformation Efficiency (TE) = (colonies on plate) / ng of DNA plated x 1000ng/µg (iGEM Competent Cell Test Kit Protocol, 2015)</LI><LI>TE = 1429.33/0.114 X 1000ng</LI><LI>TE = 1.2 x 10^6</LI></UL><FIGURE><FIGCAPTION>
					Calculation of the Controlled Colony Count (CCC) by correction to a Standard Transformation Efficiency using sample experimental results.
					</FIGCAPTION></FIGURE></DIV><DIV><H2>Selection &amp; Design of Basehunter Detectors</H2><P>In the course of this project we used detectors that are designed to detect three different targets:</P><OL><LI>Human Papilloma Virus (HPV) (K1698001)</LI><LI>The Sry gene located on the Y chromosome (K1698002) &amp; (K1698003)</LI><LI>Mycobacterium species (K1698004)</LI></OL><P>
				These targets represent the major applications that we forsee for Basehunter – namely the detection on infectious agents including 
				(viruses and bacteria) and the diagnosis of genetic conditions (inherited mutations, copy number variations etc.).  Further information 
				about each these targets is provided below.
				</P><H2>HPV Detector</H2><H3>Human Papilloma virus</H3><P>
				HPV is a common pathogen worldwide. Moloecular diagnostics has helped saved over 275,000 lives every year and our team believes our 
				device could be of use in this setting. The human papilloma virus (HPV) is a small double-stranded DNA virus with a genome of 8000 base 
				pairs. It infects the epithelial cells of skin and mucosa (Cutts et al., 2007). Theses epithelial surfaces (squamous cells) include all 
				areas covered by skin and/or mucosa such as the mouth interior, throat, tongue, tonsils, vagina, cervix, vulva, penis (the urethra - the opening), 
				and anus. Transmission of the virus occurs when these areas come into contact with a virus, allowing it to transfer between epithelial 
				cells. Over 170 HPV types have been identified and, each HPV strain is denoted by a number. Of all the HPV variants, HPV16 is one of the 
				most carcinogenic &quot;high-risk&quot; HPV stains. HPV16 is implicated in oropharyngeal (throat) cancer, while it also plays a role 
				in cervical cancer (Muñoz et al., 2003).
				</P><P>
				It has been postulated that HPV16 infects epithelial tissues through micro-abrasions of epitheliums; here HPV16 partners with receptors 
				located on the basement membrane such as alpha integrins and laminins. This leads to entry of the virus into basement membrane cells. 
				HPV16 accomplishes entry via clathrin- and caveolin-mediated endocytosis (Laniosz et al., 2009) . At this point, the viral genome is 
				transported to the nucleus where it establishes itself at a copy number between 10-200 viral genomes per cell. HPV Type 16, causes the 
				most incidences of cervical cancer cases worldwide, hence a target from this genome (Ref: NC_001526) was chosen.
				</P><P>
				HPV16’S L1 gene codes for the L1 protein, the major capsid protein of HPV16. By spontaneous self-assembly, five of these L1 proteins 
				form a homopentamer called a capsomere (Heino et al., 1995).72 of these capsomeres come together to form a capsids. The capsomeres 
				are held together by disulphide bonds and the minor capsid protein L2, which stabilise the overall capsid structure.
				</P><FIGURE><FIGCAPTION>
				Figure 1-55 bp target on the L1 gene
				</FIGCAPTION></FIGURE><P>
				A 55bp target sequence that is released from the L1 gene by digestion with HaeIII restriction enzyme was chosen as the basis for a 
				HPV Detector (Figure 1).  Some of us working as part of a secondary <A href="https://2014.igem.org/Team:UCC_Ireland/Projects_SeeDNA.html">project of the 2014</A>,
				UCC iGEM team demonstrated that this HPV Detector was capable of detecting a single stranded 55bp Target as well as a double stranded 55bp target.
				A double stranded target could also be detected in a context of where other HaeIII  DNA fragments was present in the form of fragments 
				of a HaeIII digested plasmid indicating a degree of specificity for the correct target sequence.  This “55bp HPV detector” has been 
				further studied and optimized in this years project in order to submit a well characterised Biobrick Part to the Registry (BBa_K1698001). 
				(No detector parts were submitted in 2014)
				</P><H3>References</H3><UL><LI>
				Cutts, F. T., Franceschi, S., Goldie, S., Castellsague, X., De Sanjose, S., Garnett, G., Edmunds, W. J., Claeys, P., Goldenthal, K. L., Harper, 
				D. M. &amp; Markowitz, L. 2007. Human papillomavirus and HPV vaccines: a review. Bull World Health Organ, 85, 719-26.
				</LI><LI>
				Heino, P., Dillner, J. &amp; Schwartz, S. 1995. Human papillomavirus type 16 capsid proteins produced from recombinant Semliki Forest virus assemble 
				into virus-like particles. Virology, 214, 349-59.
				</LI><LI>
				Laniosz, V., Dabydeen, S. A., Havens, M. A. &amp; Meneses, P. I. 2009. Human papillomavirus type 16 infection of human keratinocytes requires 
				clathrin and caveolin-1 and is brefeldin a sensitive.J Virol, 83, 8221-32.
				</LI><LI>
				Muñoz, N., Bosch, F. X., De Sanjosé, S., Herrero, R., Castellsagué, X., Shah, K. V., Snijders, P. J. F. &amp; Meijer, C. J. L. M. 2003. 
				Epidemiologic Classification of Human Papillomavirus Types Associated with Cervical Cancer. New England Journal of Medicine, 348, 518-527.
				</LI></UL></DIV><DIV id="sry_detect"><H2>SRY Detectors</H2><P>
				Sry is a gene located on the Y chromosome. It codes for sex-determining region Y (SRY) protein which is also known as Testis-determining 
				factor (TDF) (Kashimada and Koopman, 2010). SRY protein is transcription factor that is responsible for male sex determination in mammals.
				</P><P>
				A target located on the Y chromosome was chosen to test whether our Basehunter detector could be used for real world applications because 
				it would allow us to test both the sensitivity and specificity of the system in detecting a target sequence from genomic DNA. The ability 
				to detect and quantify target sequences from a genomic DNA sample would provide evidence that the Basehunter detector could be used for genetic 
				testing of either inherited diseases for example. By choosing a target located on the Y chromosome, it will be possible to test the detector 
				with genomic DNA from a male and to use genomic DNA from a female as a negative control. This would be a very good test of the sensitivity 
				and specificity of the Basehunter detection system.
				</P><P>
				Two Sry detectors were used in the project:
				</P><UL><LI>
					A detector specific for a 62bp target that is generated from the Sry upon HaeIII digestion was developed by some of us working as part 
					of a secondary <A href="https://2014.igem.org/Team:UCC_Ireland/Projects_SeeDNA.html">project of the 2014</A>, UCC iGEM.  This detector had already 
					been shown to work in principle but has been further characterised this year and submitted to the Registry as Part Number (BBa_K1698002).
					</LI><LI>
					A second Sry detector was also developed that recognises a 32bp PstI fragment from the Sry gene has also been developed Part Number (BBa_K1698003).
					</LI></UL><H3>References</H3><UL><LI>
					Kashimada, K., Koopman, P. (2010). Sry: the master switch in mammalian sex determination.Development. 2010 Dec;137(23):3921-30
					</LI></UL></DIV><DIV id="mycobac"><H2>Mycobacterium detector</H2><P>
			From our policy and practice work during the summer we learned of the growing need for a detector to quickly analyse whether or not a patient was infected 
			with a mycobacterium like M.Tuberculosis which is still a major cause of death in many third world countries. TB is second to aids/HIV as the most deadly 
			disease caused by an infectious pathogen.  We have known of the existence of the pathogen behind TB since the beginning of the 20th century , however to 
			this day every person with active TB will go on to infect 10 to 15 more people ( World health organisation 2010 ). Mycobacterial culture on solid 
			Lowenstein-Jensen (LJ) medium is considered the gold standard isolation method (Conde et al 2009). Although the limitation of this method is the long 
			incubation period (2-8 weeks), it is used by most developing countries because of its low cost. Techniques such as nucleic acid amplification and automated 
			liquid culture systems are costly and depend on sophisticated tools, which prevents their routine use in poor countries. TB is diagnosed currently using acid 
			fast stains which is based on the binding of mycobacterium like M.Tuberculosis to fuchsin which is very selective. Our detector serves as a novel way to 
			amplify DNA in ecoli and the results of the amplification are evident in the number of colonies that form on the plate.In saying that statistics show that over 
			37 million lives are saved every year due to effective diagnosis. We constructed a detector which would act as a negative screening device for TB. 
			We decided to this based on the reported growing need which we identified in our policies and practices throughout the summer. 
			However proper treatment is available and we decided to construct our detector to allow for a cheap diagnosis of the M. Tuberculosis pathogen.
			</P><H3>Target Identification</H3><P>
			Mycobacterium are unique among bacteria in that they possess several unique proteins involved in fatty acid synthesis and metabolism. By identifying such 
			proteins which are unique to <I>mycobacterium tuberculosis</I> (Beile Gao et al 2006), we were able to obtain the gene sequence for the protein and establish a 
			detector target from this sequence. One protein which contained a 24 base pair target flanked by Sma1 restrictions sites was identified, ML0319 is a 
			lipoprotein uniquely found in Mycobacteria and also present in Norcardia. The detector construct designed to detect this target is shown in the figure below. 
			The nucleotide base sequence for this gene has already been sequence and we hoped to be able to diagnose TB using our detector. Due to the high GC nature 
			of the 24 base pair target sequence, we identified that BamHI and KpnI would be the restriction sites to be used in the actual detector. The proposed 
			target nucleotide was checked to see if it would have a likely chance to form hairpin structures or if we would experience any problems with annealing 
			or dimerization. Even though the target has a high GC content ( 68 %), it will likely to not form hairpin structures spontaneously.
			</P><FIGURE><FIGCAPTION>
			Mycobacterium detector sequence indicating the biobrick prefix and suffix, the left and right target sequences and the nicking and restriction enzyme 
			cut sites used to generate the “activated” detector.  The oligonucleotides used for cloing the detector between the SpeI and PstI sites of the prefix 
			and suffix are indicated as are the primers that can be used to generate and activated detector by the PCR amplification method (see Section XXX).
			</FIGCAPTION></FIGURE><H3>Construction and Testing the Mycobacterium detector</H3><P>
			The Mycobacterium detector was constructed after the Mycobacterium detector sequence was cloned into the pSB1-C3 containing the GFP generating biobrick 
			part K584001. It was also assembled by cloning straight into the digested linearized plasmid pSB1C3, oligonucleotides were ordered from IDT and they were 
			first phosphorylated and then annealed together (SEE PROTOCOL SECTION). This annealed detector sequence was then ligated into the digested linearized 
			plasmid backbone and transformed. A four enzyme digest was done i.e the enzymes Nt.BspQI, Nb.BtsI, BamHI and KpnI were used as described in the protocols 
			section. Following the digest, the mixture was ran and a gel and a gel extraction was done to remove the top strand i.e detector plasmid, which could then 
			be used in the detector reaction.
			</P><H3>Results</H3><FIGURE><FIGCAPTION>
			The TB detector showed promise as a diagnostic tool as a negative screen. The detector was tested using a complementary top strand oligonucleotide ordered from IDT.
			</FIGCAPTION></FIGURE><H3>References</H3><UL><LI>
			World Health Organization. Global tuberculosis control report 2010. Geneva: World Heath Organization; 2010.
			</LI><LI>
			Conde MB, Melo FA, Marques AM, Cardoso NC, Pinheiro VG, Dalcin Pde T et al. III Brazilian Thoracic Association Guidelines on tuberculosis. J Bras Pneumol. 2009;35(10):1018-48.
			</LI><LI>
			Gao B, Paramanathan R, Gupta RS. Signature proteins that are distinctive
			characteristics of Actinobacteria and their subgroups. Antonie Van Leeuwenhoek.
			2006 Jul;90(1):69-91. Epub 2006 May 3. PubMed PMID: 16670965.
			</LI></UL></DIV><DIV id="sensit"><DIV id="sens"><H2>Results</H2><H2>Part One</H2><H3>Optimising Basehunter Sensitivity</H3><H3>A. Alternative Detector Construction by PCR to reduce detector background
</H3><H3>Summary:</H3><P>
				The initial method for detector preparation by digestion of plasmid containing the detector part with restriction and nicking enzymes was found to give a low 
				level of background colonies, presumably due to incomplete digestion of the plasmid.  Such false positives are not acceptable in a diagnostic test.  Also the 
				level of such false positives will increase if higher transformations efficiencies are employed to boost the sensitivity of the system.  We therefore devised 
				a method to amplify the detector by PCR. Preparation of the the activated detector by dgestion of linear PCR products largely eliminated background colonies.
				</P><P>
				UCC iGEM 2014 developed the first method of detector construction by digestion using 4 enzymes (Kpn I, HindIII &amp; nicking enzymes, NtBsI &amp; NtBsQI) 
				(click <A href="https://static.igem.org/mediawiki/2015/8/88/ProtocolsCork.pdf">here</A> to read our protocols). Results of this method of construction were generally positive as positive control plates achieved 
				colonies TNTC and negative control plates showed &lt;10 CFUs. As seen in figure 1 and 2, showing detector reactions using digested detector made by UCC 
				iGEM 2014, tested in June 2015 &amp; September 2014, negative plates showed CFUs. This may have been due to the presence of undigested detector that transformed 
				without target needing to be present. To improve on this, Cork iGEM 2015, created a new protocol for detector construction by PCR.
				</P><FIGURE><FIGCAPTION>
				Results generated by UCC iGEM 2014 relating to HPV Detector constructed by digestion. Negative plates show a number of CFUs.
				</FIGCAPTION></FIGURE><P>
				The new protocol (click <A href="https://static.igem.org/mediawiki/2015/8/88/ProtocolsCork.pdf">here</A> to read our protocols) involves using primers specific to the detector part and amplifying this part on a plasmid 
				containing GFP biobrick and chloramphenicol resistance. These primers are illustrated in Figure 2. Further modification of the detector is then carried out to activate. 
				</P><P>
				A DpnI digest after PCR ensures that template DNA is digested, being rich in GATC restriction sites. A subsequent clean up step ensures that template DNA is removed 
				to minimise re-annealing to detector. Next, an enzyme digest was carried out using the same 4 enzymes as the original protocol (NtBsQI, NtBsI, KpnI &amp; HindIII), shown 
				in Figure 2. This was done to create single stranded overhangs complementary to the target. The detector was then reacted with decoy oligos (complementary to the top 
				strand of the detector, which is to be removed) in excess to yield the single strands of the detector. Another cleanup was then carried out to remove the decoys. Further 
				experimentation showed that the decoy oligo step was unnecessary and the detector was ready for use after the 4 enzyme digest and PCR cleanup.
				</P><FIGURE><FIGCAPTION>
				Part shown with Primers for the creation of 30bp HPV Detector and 55bp HPV Detector. Enzymes used to digest the detector in construction are also shown.
				</FIGCAPTION></FIGURE><P>
				Table 1 - Results of detector reactions using the PCR constructed detector showed less background overall. However, the number of CFUs achieved on positive control plates 
				also showed a reduction. This may have been due to DNA concentration in the final detector sample being less after multiple clean ups during construction. 
				This may be seen in Table 1 which shows a comparison of detector reactions carried out simultaneously using digested and PCR constructed detectors.
				</P><FIGURE><FIGCAPTION>
				Figure 5 - Results of transformation using highly competent cells and detectors (PCR Constructed &amp; Digested detector)
				</FIGCAPTION></FIGURE><FIGURE><FIGCAPTION>
				Figure 6 - Results of transformation using highly competent cells and detectors (PCR Constructed &amp; Digested detector)
				</FIGCAPTION></FIGURE><H2>B. Detection of Target in Genomic DNA</H2><P>
				Our gold standard for the sensitivity and specificity of the Base hunter system would be that it should be able to detect a sequence from a sample of restriction 
				digested genomic DNA.  Here we employed the SRY detector, specific to the SRY gene on the Y chromosome.  Digested female genomic DNA serves as a very stringent 
				negative control.  Our results show some signs that the Sry target is being detected, but also that non-specific interactions are yielding positive colonies from female DNA.
				</P><H3>Sry 55bp Detector</H3><P>
				The Sry 55bp detector was tested with single stranded, double stranded and double stranded targets with background DNA present in 2014 but the detector had yet to be reacted 
				with genomic DNA. This was initially carried out by Srijita Banarjee as a final year research project, and again repeated by Cork iGEM 2015 using the detector constructed by 
				the new PCR amplification protocol (click <A href="https://static.igem.org/mediawiki/2015/8/88/ProtocolsCork.pdf">here</A> to read our protocols).
				</P><FIGURE><FIGCAPTION>
				Results of reaction of SRY Detector made by Digestion with genomic DNA digested with HaeIII.
				</FIGCAPTION></FIGURE><FIGURE><FIGCAPTION>
				Results of reaction of SRY Detector made by PCR with genomic DNA digested with HaeIII.
				</FIGCAPTION></FIGURE><P>
				As shown in figures 2 the digested detector appeared to yield more colonies with male genomic DNA, however the oppositie result was seen in figure 3.  Results appeared 
				inconclusive and it emerged that the detector faces a major challenge in use with samples with a lot of background DNA, such as a patient genomic sample.
				Sequencing of colonies from these experiments however showed that colonies from the male genomic DNA had correctly picked up the Sry target, while does from the female 
				genomic DNA had deletions in the region of the detector.  A better understanding of the mechanisms of how non-specific interactions from female DNA results in plasmid 
				recircularization may allow this background to be eliminated.
				</P><H3>Sry 32bp detector</H3><P>
				To improve on this result, a smaller SRY detector was constructed (32bp). A smaller detector may have been more specific to its target and hence give less false positives. Also, the 32bp target site was nested in 2 PstI restriction sites in the genome. This meant that digestion of a genomic sample with PstI yielded fewer fragments as a 6 cutter enzyme, than HaeIII which was previously used. Unfortunately due to time constraints the SRY 32bp Detector was not tested with genomic DNA.

				</P></DIV><DIV id="selec"><H2>Part Two</H2><H2>Assessment &amp; Optimisation of Specificity</H2><H3>A. Design of Detectors for Shorter Targets</H3><H3>Summary</H3><P>
				We reasoned that detectors design to recognize shorter targets might exhibit more sequence specificity.  We therefore tested variants of the HPV  detector as well as a new 
				Sry detector designed to detect target with a size of 30-32bp.  The performance of these detectors was similar to the longer 55 and 62 bp detectors that we orginally designed. 
				Thus Basehunter can work for target sequences in this shorter size range.
				</P><H3>Development of detectors for shorter target sequences:</H3><P>
				In 2014, UCC iGEM demonstrated that the HPV detector was capable of detecting a single stranded 55bp Target, a double stranded 55bp target and a double stranded target where 
				background DNA was present in the form of fragments of a digested plasmid.
				</P><P>
				Our aim this year was to further test this detector in order to try and optimise the system. This involved changing the target length and creating several HPV detectors and 
				targets. Our thoughts were that a smaller “non-target” or non-complementary nucleotide would have a smaller chance of binding unspecifically even if some of the bases were 
				complementary in that there would be a smaller amount of hybridisation between the two DNA strands i.e detector and target. We ordered primers so that we could amplify the 
				detector by PCR to create detectors that would bind to targets of different lengths e.g. complementary oligonucleotides. 
				</P><P>
				30bp and 24 bp variants of the HPV detector were also constructed and assessed to see if they were more selective than a larger detector. Results shown in figure 1 comparing 
				the HPV 30 bp and 55 bp detectors suggest that the 55bp Detector gave slightly more positive colonies and somewhat fewer colonies on negative control plates.  Thus the 55 bp 
				detector size worked better somwhat better.  Nevertheless the 30bp detector did work quite well and number of colonies on the control plates made be more related to variation 
				in the preparation of the detector (incomplete digestion) rather than length of the target per se.  Further testing could be carried out to determine the exact optimal length of a detector region.
				</P><H3>Assembly of a new Sry detector</H3><P>
Giving our thoughts that a shorter detector might be more useful, we constructed a 32 bp Sry detector. This detector was PCR amplified using primers ordered from IDT and the original template of the 62 bp Sry detector. This detector was tested using genomic DNA digested with Pst1 since digest with the six cutter will result in fewer smaller DNA fragments being made that might hybridise on unspecifically when compared to the target being made after the genomic DNA was digested with HaeIII.</P><P>Studies were done to investigate digested genomic DNA as a target for the detector using both the Sry 62 bp detector and the Sry 32 bp detector. The Sry detector was tested and checked to see if it worked using a positive target that was ordered from IDT. The results show that the detector that was created worked. 
				</P><P>
				To evaluate the reliability of the detector to select the target of the correct length, targets of various size (with deletions in true target sequence) were reacted with the detector. 
				First tested was the 30bp Detector with targets ranging in size from 18bp to 30bp, with missing sequence at ends of 30bp target. Figure 1 shows that the detector reacted significantly 
				more with the target of the correct length than those of smaller length, in spite of complementary sequence to the middle region of the detector.
				</P><FIGURE><FIGCAPTION>
				Results of 30bp Detector reaction with Targets of Various Size shorter than 30bp
				</FIGCAPTION></FIGURE><P>
				Figure 2 shows the effect of targets longer than 30bp on the detector reaction.</P><P>Results suggest that a longer target, although giving fewer CFU than an accurately sized target, gives more CFU than a target that is too short (see Figure 1). In addition, random 
				sequences which are either accurately sized or too short are not detected by the detector (Figure 3).
				</P><FIGURE><FIGCAPTION>
				Results of 30bp Detector reaction with Targets of Various Size longer than 30bp. N=2 where results shown are an average of sample results
				</FIGCAPTION></FIGURE><FIGURE><FIGCAPTION>
				Figure 3 - Results of transformation of 30bp and 55bp HPV detectors with targets that were oversized or undersized in target length
				</FIGCAPTION></FIGURE><H2>C. Testing Detector with Mutated Targets</H2><FIGURE><FIGCAPTION>
				Figure 1 - Illustration of mutated targets designed and tested with HPV 55bp Detector
				</FIGCAPTION></FIGURE><P>
				An important aspect of a reliable diagnostic test is its ability to differentiate the true target and other components in the sample. 
				This eliminates false positives and is referred to as “Specificity”. To assess the system’s specificity, it was reacted with a number of targets 
				which resembled the true target in size and sequence. The mutated targets varied either by a deletion, insertion or the presense of 
				incorrect bases in part or all of the target (shown in figure 1).
				The 55bp Detector was first tested with the mutated targets. Results are as shown in figure 2.
				</P><FIGURE><FIGCAPTION>
				Figure 2 - Results of reactions of 55bp HPV Detector with mutated targets
				</FIGCAPTION></FIGURE><P>
				Figure 2 suggests that more colonies are achieved when the correct, non-mutated target is present (Wild Type (WT)). However, CFUs were still seen 
				where mutated targets were present. This was especially true where the mutation occurred at the ends of the target (deletions @bases 1-3, insertions 
				at base 55). Where a mutation occurred in the middle region of the target (base change at 26-28bp) no colonies were seen, suggesting the middle region 
				of the detector is highly specific, while the detector is inefficient at detecting targets with mutated ends.
				</P></DIV><DIV id="FurCh"><H2>Part Three</H2><H2>Further Characterisation</H2><H3>A. Assessing need for Phosphorylation of Detectors &amp; Target</H3><P>We also carried out experimentation using the Sry detector to try and understand some of the mechanistic aspects of the detector. One of the experiments that we carried out was to see whether the detector reaction results would be impacted if non-phosphorylated DNA was used as target and detector, we hypothesized that doing the detector reaction with phosphorylated target and detector should give more CFUs than if both non-phosphorylated target and detector were used.</P><P>As shown by the data there wasn’t quite a big difference in corrected colony count between the three tests and the positive. We hypothesized therefore that DNA polymerase I in E.coli might be chewing back on the non-phosphorylated DNA because of its exonuclease activity in recognizing a possible error and the polymerase actively filling in nucleotides that are phosphorylated for ligase activity (Khare &amp; Eckert, 2002 ). This could be probable since the DH5 alpha cells used in our lab don’t have polymerase I knocked out. The results show that the three tests on average do not differ significantly from the positive control.</P><H3>References</H3><P>Khare V, Eckert KA. The proofreading 3'--&gt;5' exonuclease activity of DNA polymerases: a kinetic barrier to translesion DNA synthesis. Mutat Res. 2002 Dec 29;510(1-2):45-54. Review. PubMed PMID: 12459442.</P><H3>B. Genomic DNA- unspecific digested targets</H3><H2>Genomic DNA</H2><P>
				We also tested the selectivity of the detector system using the three HPV detectors (see selectivity section). Experiments were trialled where digested 
				genomic DNA was used as a target, any small cut pieces of DNA were hypothesized to be unspecific. 
				</P><P>
				We digested the genomic DNA with the four cutter Haelll and then carried out the detector reaction using this digested DNA. We hypothesized that HaeIII 
				would cut frequently enough throughout the genome so that nucleotides of appropriate length would be available to anneal on if the detector system was 
				unspecific. The results show that the detector was specific enough so that there was very little background. 
				</P><P>
				This experiment was repeated using the HPV 30 base pair detector using high efficiency cells to test whether or not there would be no background still. High efficiency cells were commercially bought from NEB.
				</P><P>This experiment was repeated using the HPV 30 base pair detector using high efficiency cells to test whether or not there would be no background still. High efficiency cells were commercially bought from NEB. </P></DIV><DIV id="con"><H2>Conclusion</H2><P>The detectors constructed work efficiently to differentiate samples containing target sequence and those without DNA. Where background DNA is present in the sample, such as genomic samples, the detectors work less specifically and can return false positives. Efforts to improve this, such as constructing shorter detectors and using enzymes that cut less frequently to digest genomic samples were began but results were unclear at the time of submitting project. Further work may be done to improve this.</P><P>Also noted was the ability of the detector to accurately select targets which were longer than the detector itself but not those which were shorter. This feature may allow the detector to be constructed shorter than the intended target sequence. This is favourable as shorter detectors tended to perform better and return higher CFU counts.</P><P>The customizable-aspect of the detector design appears to hold true as detectors for mycobacterium species and different sized targets for HPV and SRY gene were quickly and easily constructed. In a matter of days, a unique detector can be designed and constructed at minimal cost for virtually any target. These detectors appear to work equally well. This suggests that the detector holds promise as a diagnostic tool applicable to any DNA-detecting purpose.</P><P>Interpretation of results was improved significantly this year as the standardisation to a specific transformation efficiency and use of a control RFP plasmid in all tests ensures reliability of results and the ability to compare with those obtained at different times or in different labs.</P></DIV></DIV></DIV></DIV><DIV class="col"><UL id="fixed_menu"><LI>Quick Links</LI><LI><A href="#Summary">Summary</A></LI><LI><A href="#proj_desc">Project Description</A></LI><LI><A href="#standard">Introduction To Results</A></LI><LI><A href="#sens">Sensitivity</A></LI><LI><A href="#selec">Selectivity</A></LI><LI><A href="#FurCh">Further Characterization</A></LI><LI><A href="#con">Conclusion</A></LI></UL></DIV><FOOTER><DIV><UL><LI><A href="https://2015.igem.org/Team:Cork_Ireland">Home</A></LI><LI><A href="https://2015.igem.org/Team:Cork_Ireland/Project">Our Project</A></LI><LI><A href="https://2015.igem.org/Team:Cork_Ireland/Design">Design</A></LI><LI><A href="https://2015.igem.org/Team:Cork_Ireland/Practices">Outreach</A></LI><LI><A href="https://2015.igem.org/Team:Cork_Ireland/Team">Team</A></LI><LI><A href="https://2015.igem.org/Team:Cork_Ireland/Notebook">Notebook</A></LI><LI><A href="https://2015.igem.org/Team:Cork_Ireland/Parts">Parts</A></LI><LI><A href="https://2015.igem.org/Team:Cork_Ireland/Collaborations">Collaborations</A></LI><LI><A href="https://2015.igem.org/Team:Cork_Ireland/InterlabStudy">Interlab Study</A></LI><LI><A href="https://2015.igem.org/Team:Cork_Ireland/Attributions">Attributions</A></LI><A href="https://2015.igem.org/Team:Cork_Ireland/Achievements">Achievements</A></UL></DIV><DIV><P>Cork iGEM 2015</P></DIV></FOOTER></DIV></DIV></DIV></DIV></DIV></DIV></BODY></HTML>