Team:TecCEM/STUDY&RESULTS
HOME
TEAM
PROJECT
Description
Experiments & Protocols
Results
PARTS
Team Parts
Basic Parts
Composite Parts
Part Collection
NOTEBOOK
ATTRIBUTIONS
COLLABORATIONS
HUMAN PRACTICES
SAFETY
MODELING
INTERLAB
STUDY & RESULTS
ANNEX
2015 InterLab Protocol
Ligation
Each psB1C3 containing a promoter was digested with SpeI and PstI. Reagents were added as follows: 12.5 μl of molecular biology grade water, 4 μl of NEB® Buffer 2.1, 0.5 μl of BSA, 20 μl of DNA (BioBricks K823005, K823008 and K823013 in pSB1C3 backbone), 1.5 μl of SpeI and 1.5 μl of PstI. After gently mixing, the tube was placed in a Thermoblock at 37°C for 75 minutes. Then, the tube was placed in a water bath at 80°C for 20 minutes to inactivate the enzymes.  Finally, the digestion product was stored at -20°C.
The GFP cassette I13504 (contained in a pSB1A2 plasmid) was obtained by digestion with XbaI and PstI. The same procedure was used, but now 20 μl of DNA, 1.5 μl of XbaI enzyme, and 1.5 μl of PstI enzyme were added to the tube. The digestion product was also stored at -20°C.
GFP cassette (I13504) was ligated to each digested pSB1C3 containing a different promoter (J23101, J23106, J23117 called K823005, K823008 and K823013 in pSB1C3 backbone).  Fragments were ligated as follows: 2 μl of water for molecular biology, 2 μl of NEB® T4 Ligase buffer, 8 μl of the first digestion product (BBa_K823005, K823008 or K823013 in pSB1C3 backbone), 8 μl of the second digestion product (I13504), and 1 μl of NEB® T4 Ligase were added to a 0.5 ml PCR tube. The mixtures were incubated at 16°C for 24 hours, then inactivated for 10 minutes at 65°C and finally stored at -20°C. The BioBrick ligation was confirmed by electrophoresis, only one possible combination for each BioBrick ligation existed because of the cohesive ends generated by the restriction enzymes.  The first device (plasmid 1) contained J23101 + I13504, in pSB1C3 backbone; the second device (plasmid 2) contained J23106 + I13504, in pSB1C3 backbone; and the third device (plasmid 3) contained J23117 + I13504, in pSB1C3 backbone.
Transformation Protocol
DH5α competent cells were prepared by chemical protocol, with calcium chloride (0.1M) and glicerol (15%).  Then cells were transformed with the assembled BioBricks.  NEB ®’s transformation protocol (3) was used: 50 μl of competent cells and 5 μl of each previously assembled device were added to microtubes and they were placed on ice for 10 minutes. Samples were submitted to 45 seconds of a 42°C heat shock followed by placing on ice for 5 minutes. After that, 300 μl of SOC medium were added to each tuve and they were placed at 37°C and 250 rpm for 30 minutes.  200 μl of each sample were plated into warm, solid LB media with 0.1% v/v of antibiotic (chloramphenicol 35 mg/ml).  After 12 hours of incubation at 37°C 300 rpm, a single colony was isolated from each plate and cultured overnight in LB broth with the previously stated concentration of antibiotic.
Plasmid extraction
Miniprep plasmid-DNA extraction was performed.  10 ml of the transformed culture were centrifuged at 13500 rpm for 30 seconds and cells were resuspended in 350 μl of STET buffer. The mixture was transferred to a 1.5 ml microtube, where 5 μl of lysozyme (10 mg/ml) were added. After incubation for 5 minutes, the tubes were transferred to a boiling water bath for 2 minutes in order to inactivate the enzyme. Samples were centrifuged at 13000 rpm for 10 minutes, bacterial pellet was taken out of the liquid using a sterile micropipette, and 10 μl of RNase A (2 mg/ml) were added as well as 100 of μl MB grade water. After incubating for 8 minutes at room temperatura (RT), 75 μl of sodium acetate (3M), and 400 μl of isopropanol were added. The mixture was gently stirred and incubated for 10 minutes at RT. Then, it was centrifuged for 10 minutes at 13000 rpm, the supernatant was discarded, and the pellet was washed 2 times with 1 ml of ethanol 70% v/v. The DNA was resuspended in 200 μl of MB grade water, quantified by spectrophotometry and stored at -20°C.
Verifying the cloning
The results were validated by restriction mapping and DNA sequencing. For the restriction mapping, digestion reaction was performed as follows.  Microtubes were filled with H20°BM (6 μl), buffer NEB 2.1 (2 μl), plasmid (10 μl), enzymes XbaI and PstI (1 μl each); the mix was spin 5 minutes and incubated 1 hr at 37°C. Enzymes were inactivated at 80°C 20 minutes and an electrophoresis gel was run, figure is shown below.  Several colonies were restricted, then three colonies were selected to perform the fluorescence measurements.
Fig.1. Results of DNA electrophoresis in 0.8% agarose gel (90 V, 40 min) for devices constructed 1 and 2 (A) and 3 (B) from different colonies. DNA ladder is included and each lane is labelled with the device and colony number, enzymes used to digest are also specified.
For all plasmids obtained from different colonies, after digesting with PstI and XbaI, two fragments were obtained corresponding to the expected ones (2044bp and 944bp), confirming with this the success of the devices construction. Undigested plasmids showed three isoforms common to plasmids.
DNA Sequencing In order to determine the similarity of theoretical sequences and sequenced DNA of the plasmids built by the team, an alignment was performed with Geneious (Biomatters Limited, 2005-2015).   Identities were high, containing GFP and the respective promoters. An exception was plasmid 3 which chromatogram peaks overlapped; this could be due to impurities in the DNA samples or, most probably, because primers were unspecific.
Fig.2. Sequencing alignment (sequenced DNA and theoretical sequence). Device 1.
Fig.3. Sequencing alignment (sequenced DNA and theoretical sequence). Device 2.
Fig.4. Sequencing alignment (sequenced DNA and theoretical sequence). Device 3. This third sequencing could not describe the actual template. The plasmid quality was not optimal.
Fluorescence measurement procedure
We set up biological replicates in triplícate, this was done by measuring the fluorescence from three different colonies containing the same device. 1) One plate was streaked per device and control, the plates were incubated overnight (18 hrs) at 37 C.  As positive control it was used BBa_I20270, a GFP expression device in the pSB1C3 backbone (chloramphenicol resistant).  As negative controls we used BBa_R0040 (pTetR in pSB1C3) (an empty vector transformed).  Cells were grown in LB Agar supplemented with Chloramphenicol at 35 μg/ml. 2) Liquid cultured was inoculated with our experimental devices and controls: Device 1: J23101+I13504, Device 2: J23106+I13504, Device 3: J23117+I13504, positive and negative control. Cells were grown in 250 ml smooths flasks.  The tubes were oriented in the incubator at 37 C with shaking at 300 rpm for 16-18 hours.  The volume was 10 ml of media.  OD600 of the overnight cultures were measured and samples were diluted to an OD600 of 0.5, the dilution was calculated and samples were re-measured to be confirm and OD600 of 0.5.
I. Process flowchart.
Fig.5. Process flowchart 1 for fluorescence determination
The next procedure was performed to obtain the dilutions: (OD) * (10) = Total OD. (Total OD) * (V1) = (Dilution: 0.25) (150µL). Where V1 stands for the volume of culture to be used. The volume of H2O °MB for each sample will be complementary to 150µL. After that, the distribution of the samples in the plaques was defined.
Fig.6. Process flowchart 2 for fluorescence determination
Fig 7 shows the fluorescence measurements of cultures transformed with constructed plasmids 1, 2, 3; negative control is included.
Fig.7. Samples fluorescence measurements
II. Graph and analyze data.
This headland will describe the graph procedure for three experiments. Three sets of results were obtained from three different instruments. Each set of results were measured by triplicate and each sample used was prepared by triplicate.
1) First measurement.
A)All the data has to be captured in order to analyze it. See Annex 1.1.
a. Two blank samples are used because the plaques couldn´t hold all of the samples at once.
b. A, B, C and D stand for Dilutions 0.25, 0.5, 0.75 and 1.
B)The average and standard deviation are calculated for every strain using data from the three repetitions, each                          plaque measured three times. See Annex 1.2.
C)To facilitate the graphical procedure, the average and standard deviation tables are transposed. See Annex 1.3.
D)The final graph will illustrate the Average Fluorescence values for each series of samples versus the four Optical Densities used as shown below. The calculated standard deviation was also included for each average.
Fig.8. Graphic of Fluorescence VS OD. Measurement 1/3. One colony for Device.
Fig.9. Graphic of Fluorescence for every biological triplicate, CNTRL+ and CTRL-. Measurement 1/3.
2) Second measurement
The same protocol was performed but with a different instrument.
A) Captured data is available in Annex 2.1.
B) Average and standard deviation calculations are in Annex 2.2.
C) Transposed tables are in Annex 2.3
D) Final graph is showed below.
Fig.10. Graphic of Fluorescence VS OD. Measurement 2/3. One colony for Device.
3) Third measurement.
A) Captured data in Annex 3.1
B) Average and standard deviation calculations. Available in Annex 3.2.
C) Transposed tables are in Annex 3.3.
D) Final graph.
Fig.11. Graphic of Fluorescence VS OD. Measurement 3/3. One colony for Device.
Conclusion
Device 1 showed higher fluorescence in the three samples (biological and technical replicates).  A deeper statistical analysis is pertinent for this measures, but the data in the graphics clearly shows that Device 1 promotes the highest GFP expression. The possible main reason for the GFP regulation may be linked to the TATTA sequences in each promoter. After analysing the sequences of the promoters, we noticed that the differences are within the TATTA nucleotides.
