"
Page
Discussion
View source
History
teams
Log in
 
Team:Alberta/Parts
From 2012.igem.org
Home
Team
Project
Overview
The Colours
Spatial gradients
Control elements
Controling copy number
Integrated design
Achievements
Parts
Parts Submitted
Attributions
Human Practice
A social experiment
Outreach
Collaborations
Safety
Attributions
Safety
Sponsors
About Sponsors
   Team Alberta
Part usage and description
General description of parts
The parts we have made are composed of protein generating open reading frames, ribosomal binding sites, promoters, and copy-number controlled vectors. We describe each of these classes of part in turn.
Parts submitted
<groupparts>iGEM012 Alberta</groupparts>
Protein parts and ribosomal binding sites
We have made and tested parts for producing colors and repressors. The color proteins are red, based on E1010, yellow, based on amilGFP K592010, and blue, based on amilCP K592009. Repressor proteins are lambda CI, LacI, and TetR, based on C0051, C0012, and C0040 respectively. The sequences are unmodified except for removal of restriction sites, removal of degradation tags, and replacement of stop codons. The sequences were synthesized along with their RBSes.
Our original intent was to create a series of RBS sequences of similar strength that were each tailored to a particular ORF, and that showed significant sequence variation between each other to minimize the chances of intra-molecular recombination when used in combination. We used the online Salis RBS calculator (Salis et al, Nat Biotech 27: 946 2009) to calculate RBSes for each protein coding sequence with a target translation initiation strength set to a value of 50,000 for each of the ORFs shown above using a common length constraint that began with the common transcriptional start site (position -1 of the common promoter downstream region- Fig.1 below). When driven by Pr-2, the colour genes showed marked differences in expression and growth and were therefore not ideally suited to our purposes. In response we created versions of the colour genes that each used the popular registry RBS, B0034. Although the LacI and TetR RBSes were never directly compared, they both functioned adequately in subsequent experiments.
Fig 1. Information of RBS protein parts
Promoter parts
                         
Fig 2. The list of promoters that were used in our project to regulate and tune colour expression. 1. Promoters Pr-1 to Pr-5 are variants of the Anderson promoters J23114, J23105, J23108, J23100, and J23119 (wt) that are described in the Registry. The five bp between the AvrII and NheI sites have been varied to minimize recombination events when used in combination. Strength refers to the original fluorescence measurements made using  RFP as a reporter. Note that each promoter is roughly double the strength of the one that follows it in the list.  Pr-0 is a negative control sequence that was designed from the complement of the sequence of Pr-1. The sequences for Tet-Pr, Lac-Pr and lCI-Pr are as originally reported in Cox et. Al (Molecular systems Biology 2007) where each operator sequence has been placed between the -35 and -10 regions. The downstream region indicated for each promoter is discussed below. Promoter DNA used in plasmid constructions was made by annealing plus and minus strand synthetic oligonucleotides (IDT). 2. Status indicates “made (+)/tested (+)/functional (+) as the best-case scenario. In our hands Pr-5 (wt) failed to produce viable colonies with RFP but worked when linked to LacI This may explain the N.D. result reported for its J23119 analog since strong colour development seems to be associated with slow growth.
Copy control origin vectors
                         
Fig 3. Origin promoter sequences: Shown are synthetic promoters that were used to replace the native RNA II origin promoter (upper sequence) of the pSB1C3-based plasmids that constitutively express either [Pr-3]LacI (in the case of  LacI-regulated RNA II) or [Pr-3]TetR (in the case of TetR-regulated RNA II). Promoters and their accompanying NsiI sites were incorporated into each repressor plasmid by appending each sequence to the 5’ end of a pcr primer whose annealing region immediately followed the -10 region of the native promoter and which synthesized towards the suffix.  The reverse primer incorporated the NsiI site at its 5’ end and synthesized towards the prefix. The resulting product was then cut with NsiI, ligated and transformed into competent cells. Since the Inducer concentration required for optimal growth was unknown. Each transformation mix was equally divided between culture tubes with different concentration of the appropriate inducer (IPTG: 0mM, 0.002mM, 0.004 mM, 0.008 mM, 0.016mM and 0.032, 0.64 mM; ATC: same numerical series but at mg/L). Notably no growth was observed in the absence of inducer. LacI-Pr* and TetR-Pr* were our first attempt at this strategy. It was discovered after sequence analysis that the promoter primer contained a 2-bp addition that shifted the transcriptional start site 2-bp to the right. These errors were corrected with LacI-Pr and TetR-Pr.
These parts consist of a vector backbone with a pMB1 origin, modified to add a lactose or tetracycline operator sequence between the -10 and -35 boxes in the promoter for the RNA II gene. The RNA II gene is the primer which serves to begin replication of the plasmid, thus controlling its expression is an effective means of controlling plasmid reproduction and copy number.
The part is provided with an insert consisting of an RFP cassette and the appropriate repressor, so that the vector is functional out of the box. This repressor has the additional benefit of functioning as a positive selection marker when replacing the supplied insert: if the clones are plated on a plate lacking the appropriate inducer, the copy number of vectors not receiving the new insert will be repressed, leading to selection in favor of the user’s new insert. This necessitates plating in presence of the inducer during normal transformation and propagation, however.
Uses:
smoothly controllable copy number
Positive selection marker, analogous to BBA_P1010 (ccdB).
Note that this could be superior positive selection marker, since typical positive selection markers are disabled by any single mutation.
Positive selection enables plasmid shuffling in E. coli
Construction
We began with the criterion of assembling our ORFs with a variety of promoters and regulatory sequences. ORF sequences were sourced from the Registry, modified to remove KpnI sites, degradation tags, and replace TAA stops with TGA. As noted above, RBSes were engineered for each ORF using the Salis RBS calculator, with a target TIR of 50000. For promoters, we were inspired by the work of Elowitz (EMBO, 2007), and selected several constitutive promoters based on the Anderson collection, plus LambdaCI, LacI, and TetR repressible promoters. Since we were uncertain as to the appropriate expression level for the colors in our system, we wanted to flexibly assemble ORFs together with promoters and possible upstream regulatory regions (as used by Elowitz). We therefore obtained most of our parts as gBlocks (IDT) and cloned them into an entry vector with a weak promoter (Pr-2) surrounded by BsaI cut sites (Fig 4, plasmid 1). After verifying the gene sequence, we cut with BsaI, leaving a pair of standardized four basepair nonpalindromic overhangs (X and Y), which allow us to ligate in synthesized sequences for the promoter (Pr) and upstream region (USR). The Y site comes from the downstream region K879092 shown in the promoter diagram above, and the X site is included with the upstream region K879091. In the end, we used only one upstream region.
Initially, all RBS-ORFs used in this project were preceded by the common promoter cassette shown above (their construction is described below). Subsequent cleavage with the off-set cutter BsaI produces the two different asymmetric overhangs X and Y (indicated in pink). Creating different USR-promoter pairs first involved ligating USR and promoter  fragments at high concentration  via the asymmetric Z ends that were appended to the sequences of each part (Z=  5’-ACAA, Z’= 5’-TTGT). At equimolar concentrations the reaction goes virtually to completion. An aliquot of the ligation reaction was added to a fresh ligation reaction containing cut plasmid (at a 2:1 molar excess of fragment  to plasmid). The USR-Pr insertion is facilitated by the Y’ end appended to the 3’ side of the promoter part, and the X  end appended to the 5’ side of the USR part.
                         
Fig 4. Typical assembly strategy for assembling genes with a variety of promoters.
                         
Fig 5. Construction of ORFs and entry cassettes from synthetic linear fragments
All of our protein expression plasmids with the exception of LacI were created by ligating two IDT GeneBlocks together at a centrally located unique restriction sites that varied depending on the ORF as shown above. The restriction sites  used for each ORF are as follows: RFP- AflIII, amilGFP- Bsm1,  amilCP- BspHI, lCI- Tsp451, TetR- BsaXI. LacI, based on its size (1.1 Kbp), would have required three G-Blocks so instead we opted for a pcr-based strategy that incorporated the promoter cassette sequence within an ultramer.
Retrieved from " http://2012.igem.org/Team:Alberta/Parts "
Recent changes
What links here
Related changes
Special pages
My preferences
Printable version
Permanent link
Privacy policy
Disclaimers
